
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="24" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB703F5430F7F10000000000F6CAF1" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="23" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="24" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="25" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="24" /></div>
            <div><h2 class="result_count">Results: 2301 to 2400 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2301.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3122959/figure/F1/" ref="ordinalpos=2301&amp;ncbi_uid=3577231&amp;link_uid=PMC3122959" href="/pmc/articles/PMC3122959/"><img src="/pmc/articles/instance/3122959/bin/nihms283807f1.gif" src-large="/pmc/articles/instance/3122959/bin/nihms283807f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3122959/" ref="ordinalpos=2301&amp;ncbi_uid=3577231&amp;link_uid=PMC3122959" image-link="/pmc/articles/PMC3122959/figure/F1/" class="imagepopup">Figure 1. A simplified model for Hh <span class="highlight" style="background-color:">signaling</span> in mammalian cells.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> in skin cancers. </a></p><div class="supp"><p class="details"><div>SMO is the key signal transducer of the Hh <span class="highlight" style="background-color:">pathway</span>. <b>A</b>, In the absence of Hh ligands, the PTCH receptor at the base of the primary cilium suppresses the function of SMO by preventing its entry into the cilium. Without SMO activation, Gli proteins are processed with Su(Fu), GSK3β, PKA, and CKI into a repressor form (Gli-R), which disable the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>B</b>, In the presence of Hh, it binds to PTCH. The binding can be repressed by HIP and supported by Cdo, Gas1 and Boc. Upon binding, the Hh/PTCH complex becomes internalized in endosomes and later degraded. Without inhibition from PTCH, SMO becomes activated and facilitated Gli activation n (GliA), which stimulates Hh target gene expression.</div></p></div><div class="aux"><div class="resc">Chengxin Li, et al. Cell Signal. ;23(8):1235-1243.</div><p class="links"><a class="dblinks" href="/pubmed/21397013" ref="ordinalpos=2301&amp;ncbi_uid=3577231&amp;link_uid=21397013">Citation</a><a class="dblinks" href="/pmc/articles/PMC3122959/" ref="ordinalpos=2301&amp;ncbi_uid=3577231&amp;link_uid=PMC3122959">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2302.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC85713/figure/F7/" ref="ordinalpos=2302&amp;ncbi_uid=16495&amp;link_uid=PMC85713" href="/pmc/articles/PMC85713/"><img src="/pmc/articles/instance/85713/bin/mb1100060007.gif" src-large="/pmc/articles/instance/85713/bin/mb1100060007.jpg" alt="FIG. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC85713/" ref="ordinalpos=2302&amp;ncbi_uid=16495&amp;link_uid=PMC85713" image-link="/pmc/articles/PMC85713/figure/F7/" class="imagepopup">FIG. 7.  From: Repression of Ribosome and tRNA Synthesis in Secretion-Defective Cells Is Signaled by a Novel Branch of the Cell Integrity <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Model of the secretory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediating transcriptional repression of ribosome and tRNA synthesis. Turgor pressure generated by ongoing protein synthesis and a reduction or blockage in the delivery of secretory vesicles to the plasma membrane is proposed to activate the <span class="highlight" style="background-color:">pathway</span>. The guanine nucleotide exchange factor Rom2 and the Rho1 GTPase are possible upstream components of the secretory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (between Wsc and Pkc1) based on studies of actin depolarization (5). The number of steps downstream of Pkc1 is unknown. Arrows indicate activation; bars indicate repression.</div></p></div><div class="aux"><div class="resc">Yun Li, et al. Mol Cell Biol. 2000 June;20(11):3843-3851.</div><p class="links"><a class="dblinks" href="/pubmed/10805727" ref="ordinalpos=2302&amp;ncbi_uid=16495&amp;link_uid=10805727">Citation</a><a class="dblinks" href="/pmc/articles/PMC85713/" ref="ordinalpos=2302&amp;ncbi_uid=16495&amp;link_uid=PMC85713">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2303.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4114558/figure/pone-0096441-g004/" ref="ordinalpos=2303&amp;ncbi_uid=6459805&amp;link_uid=PMC4114558" href="/pmc/articles/PMC4114558/"><img src="/pmc/articles/instance/4114558/bin/pone.0096441.g004.gif" src-large="/pmc/articles/instance/4114558/bin/pone.0096441.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4114558/" ref="ordinalpos=2303&amp;ncbi_uid=6459805&amp;link_uid=PMC4114558" image-link="/pmc/articles/PMC4114558/figure/pone-0096441-g004/" class="imagepopup">Figure 4. Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promoting invasion/migration of pancreatic cancer cells through mediating S100A4..  From: Sonic Hedgehog-Gli1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulates the Epithelial Mesenchymal Transition (EMT) by Mediating a New Target Gene, S100A4, in Pancreatic Cancer Cells. </a></p><div class="supp"><p class="details"><div>A: The relative transcription levels of Shh, Gli1, S100A4, E-cadherin and vimentin were regulated by L-Gli1i/L-Shh transduction. B: The relative transcription levels of S100A4, E-cadherin and vimentin in L-Shh transfected cells were reversed by siS100A4 transduction. C: The expression levels of Gli1, S100A4, E-cadherin and vimentin proteins regulated by L-Gli1i/L-Shh transduction. D: The expression levels of S100A4, E-cadherin and vimentin proteins were reversed by siS100A4 transduction. E: The invasion/migration of pancreatic cancer cells regulated by L-Gli1i/L-Shh transduction were analyzed by transwell assays. F: The invasion/migration of L-Shh transfected cells were reversed by siS100A4 transduction. * <i>P</i>&lt;0.05, ** <i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Xuanfu Xu, et al. PLoS One. 2014;9(7):e96441.</div><p class="links"><a class="dblinks" href="/pubmed/25072505" ref="ordinalpos=2303&amp;ncbi_uid=6459805&amp;link_uid=25072505">Citation</a><a class="dblinks" href="/pmc/articles/PMC4114558/" ref="ordinalpos=2303&amp;ncbi_uid=6459805&amp;link_uid=PMC4114558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2304.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3995778/figure/pone-0095413-g005/" ref="ordinalpos=2304&amp;ncbi_uid=6062706&amp;link_uid=PMC3995778" href="/pmc/articles/PMC3995778/"><img src="/pmc/articles/instance/3995778/bin/pone.0095413.g005.gif" src-large="/pmc/articles/instance/3995778/bin/pone.0095413.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3995778/" ref="ordinalpos=2304&amp;ncbi_uid=6062706&amp;link_uid=PMC3995778" image-link="/pmc/articles/PMC3995778/figure/pone-0095413-g005/" class="imagepopup">Figure 5. Overrepresented glutamate <span class="highlight" style="background-color:">signaling</span> components in the functional synapse..  From: Overrepresentation of Glutamate <span class="highlight" style="background-color:">Signaling</span> in Alzheimer's Disease: Network-Based <span class="highlight" style="background-color:">Pathway</span> Enrichment Using Meta-Analysis of Genome-Wide Association Studies. </a></p><div class="supp"><p class="details"><div>The original version of the hsa04724 KEGG <span class="highlight" style="background-color:">pathway</span> (Glutamatergic synapse) is shown. Black arrows denote direct molecular interaction or relation between gene products (green squares) or other types of molecules (unfilled circles), while black arrows with dashed lines denote an indirect effect between the each node. The relationship with other KEGG pathways is shown with the presence of white round rectangles. Gene symbols in components belonging to Meta-GW and META-GATES SN1 are denoted in red.</div></p></div><div class="aux"><div class="resc">Eduardo Pérez-Palma, et al. PLoS One. 2014;9(4):e95413.</div><p class="links"><a class="dblinks" href="/pubmed/24755620" ref="ordinalpos=2304&amp;ncbi_uid=6062706&amp;link_uid=24755620">Citation</a><a class="dblinks" href="/pmc/articles/PMC3995778/" ref="ordinalpos=2304&amp;ncbi_uid=6062706&amp;link_uid=PMC3995778">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2305.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2174193/figure/F5/" ref="ordinalpos=2305&amp;ncbi_uid=1149925&amp;link_uid=PMC2174193" href="/pmc/articles/PMC2174193/"><img src="/pmc/articles/instance/2174193/bin/zmk0010883780005.gif" src-large="/pmc/articles/instance/2174193/bin/zmk0010883780005.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2174193/" ref="ordinalpos=2305&amp;ncbi_uid=1149925&amp;link_uid=PMC2174193" image-link="/pmc/articles/PMC2174193/figure/F5/" class="imagepopup">Figure 5. From: Large-Scale Analysis of Yeast Filamentous Growth by Systematic Gene Disruption and Overexpression. </a></p><div class="supp"><p class="details"><div>The mitochondrial retrograde <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and filamentous growth. (A) Overview of the <span class="highlight" style="background-color:">pathway</span> and relevant proteins. (B) Phenotypic analysis of RTG <span class="highlight" style="background-color:">pathway</span> mutants. Gene deletion and <i>rho</i><sup>0</sup> mutants were analyzed for filamentous growth phenotypes as indicated. Colony morphology was assessed on low-nitrogen solid growth medium supplemented with 1% butanol; cell morphology was assessed in rich liquid medium supplemented with 1% butanol. (C) The RTG <span class="highlight" style="background-color:">pathway</span> inhibits filamentous growth. Retrograde <span class="highlight" style="background-color:">signaling</span> acts as part of the mechanism by which mitochondrial dysfunction is transduced into a signal affecting filamentous growth.</div></p></div><div class="aux"><div class="resc">Rui Jin, et al. Mol Biol Cell. 2008 January;19(1):284-296.</div><p class="links"><a class="dblinks" href="/pubmed/17989363" ref="ordinalpos=2305&amp;ncbi_uid=1149925&amp;link_uid=17989363">Citation</a><a class="dblinks" href="/pmc/articles/PMC2174193/" ref="ordinalpos=2305&amp;ncbi_uid=1149925&amp;link_uid=PMC2174193">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2306.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3638576/figure/fig3/" ref="ordinalpos=2306&amp;ncbi_uid=4883242&amp;link_uid=PMC3638576" href="/pmc/articles/PMC3638576/"><img src="/pmc/articles/instance/3638576/bin/ECAM2013-219472.003.gif" src-large="/pmc/articles/instance/3638576/bin/ECAM2013-219472.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3638576/" ref="ordinalpos=2306&amp;ncbi_uid=4883242&amp;link_uid=PMC3638576" image-link="/pmc/articles/PMC3638576/figure/fig3/" class="imagepopup">Figure 3.  From:  Ganoderma tsugae Extract Inhibits Growth of HER2-Overexpressing Cancer Cells via Modulation of HER2/PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effect of GTE on HER2/PI3K/Akt and Ras/MAPK <span class="highlight" style="background-color:">signaling</span> cascades in HER2-overexpressing cancer cells. (a) SKOV-3 cells were treated with various concentrations of GTE (0, 0.1, 0.25, 0.5, 0.75, and 1 mg/mL) for 24 h. The expression of HER2 protein was measured by Western blotting. (b) SKOV-3 cells were treated with 0.5 mg/mL GTE for 6, 12, 24, 48, and 72 h. The protein level of HER2 was determined by Western blotting. (c) Treatment of SKOV-3 cells with GTE (0, 0.25, or 0.5 mg/mL) for 24 h inhibited HER2/PI3K/Akt but not HER2/Erk <span class="highlight" style="background-color:">signaling</span>. (d) Treatment of SKBR-3 and BT-474 cells with GTE (0, 0.25, 0.5 mg/mL) for 24 h led to inhibition of the HER2/Akt but not the HER2/Erk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The results are expressed as the mean ± SD of three independent experiments. *<i>P</i> &lt; 0.05; **<i>P</i> &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Han-Peng Kuo, et al. Evid Based Complement Alternat Med. 2013;2013:219472.</div><p class="links"><a class="dblinks" href="/pubmed/23662119" ref="ordinalpos=2306&amp;ncbi_uid=4883242&amp;link_uid=23662119">Citation</a><a class="dblinks" href="/pmc/articles/PMC3638576/" ref="ordinalpos=2306&amp;ncbi_uid=4883242&amp;link_uid=PMC3638576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2307.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3131414/figure/F9/" ref="ordinalpos=2307&amp;ncbi_uid=3329199&amp;link_uid=PMC3131414" href="/pmc/articles/PMC3131414/"><img src="/pmc/articles/instance/3131414/bin/nihms299388f9.gif" src-large="/pmc/articles/instance/3131414/bin/nihms299388f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3131414/" ref="ordinalpos=2307&amp;ncbi_uid=3329199&amp;link_uid=PMC3131414" image-link="/pmc/articles/PMC3131414/figure/F9/" class="imagepopup">Figure 9. MnSOD in the mitochondrial matrix is at the bifurcation point between 1-electron and 2-electron <span class="highlight" style="background-color:">signaling</span>.  From: Superoxide Dismutase in Redox Biology: The roles of superoxide and hydrogen peroxide. </a></p><div class="supp"><p class="details"><div>The different <span class="highlight" style="background-color:">signaling</span> cascades provide a feedback mechanism for the high-flux metabolic circuit as well as low flux circuits that lead to gene expression, such as with the HIF <span class="highlight" style="background-color:">pathway</span> as presented in Fig. (2).</div></p></div><div class="aux"><div class="resc">Garry R. Buettner. Anticancer Agents Med Chem. ;11(4):341-346.</div><p class="links"><a class="dblinks" href="/pubmed/21453242" ref="ordinalpos=2307&amp;ncbi_uid=3329199&amp;link_uid=21453242">Citation</a><a class="dblinks" href="/pmc/articles/PMC3131414/" ref="ordinalpos=2307&amp;ncbi_uid=3329199&amp;link_uid=PMC3131414">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2308.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3246677/figure/F2/" ref="ordinalpos=2308&amp;ncbi_uid=4062847&amp;link_uid=PMC3246677" href="/pmc/articles/PMC3246677/"><img src="/pmc/articles/instance/3246677/bin/nihms-345848-f0002.gif" src-large="/pmc/articles/instance/3246677/bin/nihms-345848-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3246677/" ref="ordinalpos=2308&amp;ncbi_uid=4062847&amp;link_uid=PMC3246677" image-link="/pmc/articles/PMC3246677/figure/F2/" class="imagepopup">Figure 2.  From: Viperin: a multifunctional, interferon-inducible protein that regulates virus replication. </a></p><div class="supp"><p class="details"><div>Regulation of viperin expression. Viperin induction is mediated by both the classical IFN-stimulated gene induction <span class="highlight" style="background-color:">pathway</span> and the IFN-independent <span class="highlight" style="background-color:">pathway</span>. The IFN-mediated viperin gene expression is regulated by ISGF3 (left panel), while IFN-independent viperin gene expression is regulated by IRF1 and IRF3, which can be activated by viral factors or by the peroxisomal MAVS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jun-Young Seo, et al. Cell Host Microbe. ;10(6):534-539.</div><p class="links"><a class="dblinks" href="/pubmed/22177558" ref="ordinalpos=2308&amp;ncbi_uid=4062847&amp;link_uid=22177558">Citation</a><a class="dblinks" href="/pmc/articles/PMC3246677/" ref="ordinalpos=2308&amp;ncbi_uid=4062847&amp;link_uid=PMC3246677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2309.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2658594/figure/F4/" ref="ordinalpos=2309&amp;ncbi_uid=1710646&amp;link_uid=PMC2658594" href="/pmc/articles/PMC2658594/"><img src="/pmc/articles/instance/2658594/bin/nihms97377f4.gif" src-large="/pmc/articles/instance/2658594/bin/nihms97377f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2658594/" ref="ordinalpos=2309&amp;ncbi_uid=1710646&amp;link_uid=PMC2658594" image-link="/pmc/articles/PMC2658594/figure/F4/" class="imagepopup">Figure 4. Mechanistic model.  From: AHR-dependent misregulation of Wnt <span class="highlight" style="background-color:">signaling</span> disrupts tissue regeneration. </a></p><div class="supp"><p class="details"><div>Activation of the AHR <span class="highlight" style="background-color:">pathway</span> results in the improper expression of R-Spondin1, which acts through LRP6 to activate Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. <b>A)</b> Over-activation of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> causes the stabilization of β-catenin, resulting in the misexpression of various Wnt target genes. <b>B)</b> β-catenin could also be regulated at the post-transcriptional level by direct interactions with Sox9 controlling the expression of these proteins. The inhibitory effect on regeneration after AHR activation could be due to an interplay between these events.</div></p></div><div class="aux"><div class="resc">Lijoy K. Mathew, et al. Biochem Pharmacol. ;77(4):498-507.</div><p class="links"><a class="dblinks" href="/pubmed/18938144" ref="ordinalpos=2309&amp;ncbi_uid=1710646&amp;link_uid=18938144">Citation</a><a class="dblinks" href="/pmc/articles/PMC2658594/" ref="ordinalpos=2309&amp;ncbi_uid=1710646&amp;link_uid=PMC2658594">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2310.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3392028/figure/F3/" ref="ordinalpos=2310&amp;ncbi_uid=4107696&amp;link_uid=PMC3392028" href="/pmc/articles/PMC3392028/"><img src="/pmc/articles/instance/3392028/bin/nihms219131f3.gif" src-large="/pmc/articles/instance/3392028/bin/nihms219131f3.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3392028/" ref="ordinalpos=2310&amp;ncbi_uid=4107696&amp;link_uid=PMC3392028" image-link="/pmc/articles/PMC3392028/figure/F3/" class="imagepopup">FIGURE 3.  From: <span class="highlight" style="background-color:">Signaling</span> Networks that Control the Lineage Commitment and Differentiation of Bone Cells. </a></p><div class="supp"><p class="details"><div>Seven important <span class="highlight" style="background-color:">signaling</span> networks of osteoblast differentiation. Binding of Wnt to the FZD receptor induces β-catenin accumulation, which translocates to the nucleus to activate target gene transcription. Several transcription factors have been found crucial for osteoblast differentiation downstream of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, such as Runx2, Osterix, and ATF4. They are essential for differentiation of mesenchymal stem cells into differentiated osteoblasts and also function in the transcription of osteoblast-specific genes.</div></p></div><div class="aux"><div class="resc">Carrie S. Soltanoff, et al. Crit Rev Eukaryot Gene Expr. ;19(1):1-46.</div><p class="links"><a class="dblinks" href="/pubmed/19191755" ref="ordinalpos=2310&amp;ncbi_uid=4107696&amp;link_uid=19191755">Citation</a><a class="dblinks" href="/pmc/articles/PMC3392028/" ref="ordinalpos=2310&amp;ncbi_uid=4107696&amp;link_uid=PMC3392028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2311.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3194817/figure/pone-0025847-g009/" ref="ordinalpos=2311&amp;ncbi_uid=3296728&amp;link_uid=PMC3194817" href="/pmc/articles/PMC3194817/"><img src="/pmc/articles/instance/3194817/bin/pone.0025847.g009.gif" src-large="/pmc/articles/instance/3194817/bin/pone.0025847.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3194817/" ref="ordinalpos=2311&amp;ncbi_uid=3296728&amp;link_uid=PMC3194817" image-link="/pmc/articles/PMC3194817/figure/pone-0025847-g009/" class="imagepopup">Figure 9. Wnt <span class="highlight" style="background-color:">signaling</span> inhibition “rescues” GSK-3β null embryo osteogenic gene expression and Hedgehog <span class="highlight" style="background-color:">signaling</span> activity..  From: Role of GSK-3β in the Osteogenic Differentiation of Palatal Mesenchyme. </a></p><div class="supp"><p class="details"><div>(A–B) e13.5 <i>GSK-3β</i> +/+ and −/− embryo palate cultures treated for 2 days with DMEM F12 +/− supplementation with Dkk-1 (100 ng/mL). (A) qRT-PCR for the osteogenic gene markers <i>Alp</i>, <i>Runx2</i>, <i>Ocn</i>, and <i>Col1a1</i>. N = 3, *&lt;0.01. (B) qRT-PCR for members of the <i>Hedgehog</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>Ihh</i>, <i>Shh</i>, <i>Ptch1</i>, and <i>Gli1</i>). N = 3, *&lt;0.01.</div></p></div><div class="aux"><div class="resc">Emily R. Nelson, et al. PLoS One. 2011;6(10):e25847.</div><p class="links"><a class="dblinks" href="/pubmed/22022457" ref="ordinalpos=2311&amp;ncbi_uid=3296728&amp;link_uid=22022457">Citation</a><a class="dblinks" href="/pmc/articles/PMC3194817/" ref="ordinalpos=2311&amp;ncbi_uid=3296728&amp;link_uid=PMC3194817">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2312.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3792569/figure/F1/" ref="ordinalpos=2312&amp;ncbi_uid=5375164&amp;link_uid=PMC3792569" href="/pmc/articles/PMC3792569/"><img src="/pmc/articles/instance/3792569/bin/nihms512757f1.gif" src-large="/pmc/articles/instance/3792569/bin/nihms512757f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3792569/" ref="ordinalpos=2312&amp;ncbi_uid=5375164&amp;link_uid=PMC3792569" image-link="/pmc/articles/PMC3792569/figure/F1/" class="imagepopup">Fig. 1.  From: Down-Regulation of Notch-1 Is Associated With Akt and FoxM1 in Inducing Cell Growth Inhibition and Apoptosis in Prostate Cancer Cells. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PCa cell lines. The baseline expression of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was determined in a panel of PCa cell lines using real-time RT-PCR (A,C) and Western blotting analysis (B,D), respectively.</div></p></div><div class="aux"><div class="resc">Zhiwei Wang, et al. J Cell Biochem. 2011 January;112(1):78-88.</div><p class="links"><a class="dblinks" href="/pubmed/20658545" ref="ordinalpos=2312&amp;ncbi_uid=5375164&amp;link_uid=20658545">Citation</a><a class="dblinks" href="/pmc/articles/PMC3792569/" ref="ordinalpos=2312&amp;ncbi_uid=5375164&amp;link_uid=PMC3792569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2313.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694829/figure/F6/" ref="ordinalpos=2313&amp;ncbi_uid=1939724&amp;link_uid=PMC2694829" href="/pmc/articles/PMC2694829/"><img src="/pmc/articles/instance/2694829/bin/1755-8794-2-27-6.gif" src-large="/pmc/articles/instance/2694829/bin/1755-8794-2-27-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694829/" ref="ordinalpos=2313&amp;ncbi_uid=1939724&amp;link_uid=PMC2694829" image-link="/pmc/articles/PMC2694829/figure/F6/" class="imagepopup">Figure 6.  From: Prediction of HIV-1 virus-host protein interactions using virus and host sequence motifs. </a></p><div class="supp"><p class="details"><div><b>HIV-1 TAT T cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The KEGG T cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is colored for TAT <i>HHP </i>(blue) and <i>HHE </i>(red), and their overlap (purple). Green boxes have proteins not involved in infection, while white boxes do not have human proteins.</div></p></div><div class="aux"><div class="resc">Perry Evans, et al. BMC Med Genomics. 2009;2:27-27.</div><p class="links"><a class="dblinks" href="/pubmed/19450270" ref="ordinalpos=2313&amp;ncbi_uid=1939724&amp;link_uid=19450270">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694829/" ref="ordinalpos=2313&amp;ncbi_uid=1939724&amp;link_uid=PMC2694829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2314.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3723617/figure/fig10/" ref="ordinalpos=2314&amp;ncbi_uid=6213260&amp;link_uid=PMC3723617" href="/pmc/articles/PMC3723617/"><img src="/pmc/articles/instance/3723617/bin/pc_112896_f10.gif" src-large="/pmc/articles/instance/3723617/bin/pc_112896_f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3723617/" ref="ordinalpos=2314&amp;ncbi_uid=6213260&amp;link_uid=PMC3723617" image-link="/pmc/articles/PMC3723617/figure/fig10/" class="imagepopup">Figure 10. From: A Spatio-Temporal Understanding of Growth Regulation during the Salt Stress Response in Arabidopsis[W]. </a></p><div class="supp"><p class="details"><div>Model for Spatio-Temporal Dynamics in Hormone <span class="highlight" style="background-color:">Signaling</span> and Growth Regulation during the Salt Stress Response.<br />Diagram shows the inferred timing and spatial pattern of activity for ABA, JA, GA, and BR <span class="highlight" style="background-color:">signaling</span> during the salt stress response time course. ABA levels peak during the ST and QU phases of the salt response, but lower levels present during the RE and HO phases promote growth recovery. Based on experiments using tissue-specific suppression of ABA <span class="highlight" style="background-color:">signaling</span>, the inferred site of ABA in growth regulation is the END/PER tissue layers; however, regulation of gene transcription appears to occur through <span class="highlight" style="background-color:">signaling</span> in all tissue layers. JA levels are predicted to rise during the RE and HO phases, which are when the <i>jai3-1</i> mutant has an effect on growth rates and when JA-associated transcriptional programs are induced during the salt response. GA and BR <span class="highlight" style="background-color:">signaling</span> is coregulated with peak levels occurring under nonstress conditions. Based on the expression pattern of GA and BR-associated transcriptional targets and dynamic changes in fluorescent-based reporters of <span class="highlight" style="background-color:">pathway</span> activity, we infer that GA and BR <span class="highlight" style="background-color:">signaling</span> recovers during the RE and HO phases of the salt stress response. Reestablishment of <span class="highlight" style="background-color:">pathway</span> activity is necessary to promote growth recovery.</div></p></div><div class="aux"><div class="resc">Yu Geng, et al. Plant Cell. 2013 June;25(6):2132-2154.</div><p class="links"><a class="dblinks" href="/pubmed/23898029" ref="ordinalpos=2314&amp;ncbi_uid=6213260&amp;link_uid=23898029">Citation</a><a class="dblinks" href="/pmc/articles/PMC3723617/" ref="ordinalpos=2314&amp;ncbi_uid=6213260&amp;link_uid=PMC3723617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2315.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2172815/figure/fig7/" ref="ordinalpos=2315&amp;ncbi_uid=1129375&amp;link_uid=PMC2172815" href="/pmc/articles/PMC2172815/"><img src="/pmc/articles/instance/2172815/bin/200304105f7.gif" src-large="/pmc/articles/instance/2172815/bin/200304105f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2172815/" ref="ordinalpos=2315&amp;ncbi_uid=1129375&amp;link_uid=PMC2172815" image-link="/pmc/articles/PMC2172815/figure/fig7/" class="imagepopup">Figure 7. From: Differential ?v integrin-mediated Ras-ERK <span class="highlight" style="background-color:">signaling</span> during two pathways of angiogenesis. </a></p><div class="supp"><p class="details"><div><b>bFGF/αvβ3 and VEGF/αvβ5 <span class="highlight" style="background-color:">signaling</span> pathways.</b> A summary of the <span class="highlight" style="background-color:">signaling</span> pathways outlined in this report as it relates to EC cell survival as recently described in Alavi et al. (2003). Evidence presented here reveals that bFGF/αvβ3 and VEGF/αvβ5 differentially activate Ras-Raf-ERK <span class="highlight" style="background-color:">signaling</span>. This, together with our recent work (Alavi et al., 2003), allows us to propose a model whereby each of these <span class="highlight" style="background-color:">signaling</span> pathways accounts for protection of EC from distinct mediators of apoptosis. The αvβ3 <span class="highlight" style="background-color:">pathway</span> promotes an ERK-independent survival mechanism preventing stress-mediated death based on Raf coupling to the mitochondria, whereas the αvβ5 <span class="highlight" style="background-color:">pathway</span> prevents receptor-mediated death in an ERK-dependent manner. In addition, ERK is likely playing a general role in both pathways of angiogenesis because it regulates gene transcription, cell cycle progression, and cell migration, which are critical to the growth and differentiation of new blood vessels.</div></p></div><div class="aux"><div class="resc">John D. Hood, et al. J Cell Biol. 2003 September 1;162(5):933-943.</div><p class="links"><a class="dblinks" href="/pubmed/12952943" ref="ordinalpos=2315&amp;ncbi_uid=1129375&amp;link_uid=12952943">Citation</a><a class="dblinks" href="/pmc/articles/PMC2172815/" ref="ordinalpos=2315&amp;ncbi_uid=1129375&amp;link_uid=PMC2172815">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2316.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2956880/figure/F4/" ref="ordinalpos=2316&amp;ncbi_uid=2523742&amp;link_uid=PMC2956880" href="/pmc/articles/PMC2956880/"><img src="/pmc/articles/instance/2956880/bin/nihms239224f4.gif" src-large="/pmc/articles/instance/2956880/bin/nihms239224f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2956880/" ref="ordinalpos=2316&amp;ncbi_uid=2523742&amp;link_uid=PMC2956880" image-link="/pmc/articles/PMC2956880/figure/F4/" class="imagepopup">Figure 4.  From: Proneurotrophins, Seizures, and Neuronal Apoptosis. </a></p><div class="supp"><p class="details"><div>Schematic diagram showing the interaction of the p75<sup>NTR</sup> and TrkB <span class="highlight" style="background-color:">signaling</span> pathways. Activation of p75<sup>NTR</sup> blocks <span class="highlight" style="background-color:">signaling</span> from TrkB downstream of the receptor, allowing the apoptotic <span class="highlight" style="background-color:">pathway</span> to proceed.</div></p></div><div class="aux"><div class="resc">Wilma J. Friedman. Neuroscientist. ;16(3):244-252.</div><p class="links"><a class="dblinks" href="/pubmed/20360602" ref="ordinalpos=2316&amp;ncbi_uid=2523742&amp;link_uid=20360602">Citation</a><a class="dblinks" href="/pmc/articles/PMC2956880/" ref="ordinalpos=2316&amp;ncbi_uid=2523742&amp;link_uid=PMC2956880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2317.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4037577/figure/fig4/" ref="ordinalpos=2317&amp;ncbi_uid=6197722&amp;link_uid=PMC4037577" href="/pmc/articles/PMC4037577/"><img src="/pmc/articles/instance/4037577/bin/MI2014-978463.004.gif" src-large="/pmc/articles/instance/4037577/bin/MI2014-978463.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4037577/" ref="ordinalpos=2317&amp;ncbi_uid=6197722&amp;link_uid=PMC4037577" image-link="/pmc/articles/PMC4037577/figure/fig4/" class="imagepopup">Figure 4.  From: Obesity, Inflammation, and Lung Injury (OILI): The Good. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of SFRP5, a decoy for WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which further activates <i>β</i>-catenin and then JNK. Activated JNK promotes proinflammatory cytokines TNF<i>α</i>, IL-6, and MCP-1. Under obese state, the production of SFRP5 was reduced and thus the decoying effect was weak, which is translated into the increased proinflammation and insulin resistance.</div></p></div><div class="aux"><div class="resc">Cheryl Wang. Mediators Inflamm. 2014;2014:978463.</div><p class="links"><a class="dblinks" href="/pubmed/24899788" ref="ordinalpos=2317&amp;ncbi_uid=6197722&amp;link_uid=24899788">Citation</a><a class="dblinks" href="/pmc/articles/PMC4037577/" ref="ordinalpos=2317&amp;ncbi_uid=6197722&amp;link_uid=PMC4037577">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2318.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3141988/figure/fig06/" ref="ordinalpos=2318&amp;ncbi_uid=3651276&amp;link_uid=PMC3141988" href="/pmc/articles/PMC3141988/"><img src="/pmc/articles/instance/3141988/bin/pnas.1103345108fig06.gif" src-large="/pmc/articles/instance/3141988/bin/pnas.1103345108fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3141988/" ref="ordinalpos=2318&amp;ncbi_uid=3651276&amp;link_uid=PMC3141988" image-link="/pmc/articles/PMC3141988/figure/fig06/" class="imagepopup">Fig. 6. From: E-cadherin mediates contact inhibition of proliferation through Hippo <span class="highlight" style="background-color:">signaling</span>-<span class="highlight" style="background-color:">pathway</span> components. </a></p><div class="supp"><p class="details"><div>A model for an E-cadherin–mediated Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Homophilic binding of E-cadherin between two cells stimulates the Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which control proliferation by inhibiting the activity of YAP in the nucleus. Broken arrows indicate steps unresolved by the present study (see <i>Discussion</i> for more details).</div></p></div><div class="aux"><div class="resc">Nam-Gyun Kim, et al. Proc Natl Acad Sci U S A. 2011 July 19;108(29):11930-11935.</div><p class="links"><a class="dblinks" href="/pubmed/21730131" ref="ordinalpos=2318&amp;ncbi_uid=3651276&amp;link_uid=21730131">Citation</a><a class="dblinks" href="/pmc/articles/PMC3141988/" ref="ordinalpos=2318&amp;ncbi_uid=3651276&amp;link_uid=PMC3141988">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2319.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2735923/figure/F10/" ref="ordinalpos=2319&amp;ncbi_uid=2038549&amp;link_uid=PMC2735923" href="/pmc/articles/PMC2735923/"><img src="/pmc/articles/instance/2735923/bin/JCI38079.f10.gif" src-large="/pmc/articles/instance/2735923/bin/JCI38079.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2735923/" ref="ordinalpos=2319&amp;ncbi_uid=2038549&amp;link_uid=PMC2735923" image-link="/pmc/articles/PMC2735923/figure/F10/" class="imagepopup">Figure 10. Model of a Wnt/Tnc/Pdgfr <span class="highlight" style="background-color:">signaling</span> axis regulating lung SMC development..  From: Wnt <span class="highlight" style="background-color:">signaling</span> regulates smooth muscle precursor development in the mouse lung via a tenascin C/PDGFR <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Airway epithelium (green) expresses Wnt7b, which acts on mesenchymal SMC precursors (yellow). Wnt7b and β-catenin <span class="highlight" style="background-color:">signaling</span> directly activates Tnc expression and in turn promotes airway SMC (blue) and VSMC (red) precursor proliferation and development.</div></p></div><div class="aux"><div class="resc">Ethan David Cohen, et al. J Clin Invest. 2009 September 1;119(9):2538-2549.</div><p class="links"><a class="dblinks" href="/pubmed/19690384" ref="ordinalpos=2319&amp;ncbi_uid=2038549&amp;link_uid=19690384">Citation</a><a class="dblinks" href="/pmc/articles/PMC2735923/" ref="ordinalpos=2319&amp;ncbi_uid=2038549&amp;link_uid=PMC2735923">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2320.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2764916/figure/F1/" ref="ordinalpos=2320&amp;ncbi_uid=2105740&amp;link_uid=PMC2764916" href="/pmc/articles/PMC2764916/"><img src="/pmc/articles/instance/2764916/bin/zpq9990997160001.gif" src-large="/pmc/articles/instance/2764916/bin/zpq9990997160001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2764916/" ref="ordinalpos=2320&amp;ncbi_uid=2105740&amp;link_uid=PMC2764916" image-link="/pmc/articles/PMC2764916/figure/F1/" class="imagepopup">Fig. 1. From: An evolutionary conserved function of the JAK-STAT <span class="highlight" style="background-color:">pathway</span> in anti-dengue defense. </a></p><div class="supp"><p class="details"><div>The canonical JAK-STAT <span class="highlight" style="background-color:">signaling</span> cascade. (<i>A</i>) Core JAK-STAT <span class="highlight" style="background-color:">pathway</span> factors identified in Drosophila and their putative roles in the cascade. (<i>B</i>) Evolutionary conservation of the JAK-STAT immune <span class="highlight" style="background-color:">signaling</span> in insects and humans. Insect orthologs are indicated with solid black circles at the branch node. Genes that are transcriptionally regulated by the insect STAT are indicated with a star. The <span class="highlight" style="background-color:">pathway</span> core intracellular machinery is highly conserved in human and insects. The upstream ligands and effectors are less conserved, which may suggest a host–pathogen-specific response. Accession numbers are presented in Fig. S1.</div></p></div><div class="aux"><div class="resc">Jayme A. Souza-Neto, et al. Proc Natl Acad Sci U S A. 2009 October 20;106(42):17841-17846.</div><p class="links"><a class="dblinks" href="/pubmed/19805194" ref="ordinalpos=2320&amp;ncbi_uid=2105740&amp;link_uid=19805194">Citation</a><a class="dblinks" href="/pmc/articles/PMC2764916/" ref="ordinalpos=2320&amp;ncbi_uid=2105740&amp;link_uid=PMC2764916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2321.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2772411/figure/f5/" ref="ordinalpos=2321&amp;ncbi_uid=2061062&amp;link_uid=PMC2772411" href="/pmc/articles/PMC2772411/"><img src="/pmc/articles/instance/2772411/bin/zek0110934250005.gif" src-large="/pmc/articles/instance/2772411/bin/zek0110934250005.jpg" alt="FIG. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2772411/" ref="ordinalpos=2321&amp;ncbi_uid=2061062&amp;link_uid=PMC2772411" image-link="/pmc/articles/PMC2772411/figure/f5/" class="imagepopup">FIG. 5. From: Our Paths Might Cross: the Role of the Fungal Cell Wall Integrity <span class="highlight" style="background-color:">Pathway</span> in Stress Response and Cross Talk with Other Stress Response Pathways . </a></p><div class="supp"><p class="details"><div>Nitrogen limitation inhibits TOR <span class="highlight" style="background-color:">signaling</span> and the subsequent activation of the CWI <span class="highlight" style="background-color:">pathway</span>. <i>S. cerevisiae</i> carries the genes <i>TOR1</i> and <i>TOR2</i>. TOR <span class="highlight" style="background-color:">signaling</span> is activated in response to nitrogen or inhibited due to cellular exposure to nitrogen limitations or synthetically to the agent rapamycin. Rapamycin binds to the prolyl-isomerase enzyme FKBP12. Binding of FKBP12 and thus inhibition of Tor leads to Slt2/Mpk1 phosphorylation. In this process, the Tor/FKBP12 complex prevents Tap42, a type 2A phosphatase-associated protein that binds to Sit4, a type 2A-related serine threonine phosphatase, from inhibiting Rom2. As a likely recipient of TOR <span class="highlight" style="background-color:">signaling</span>, Rom2 stimulates the CWI <span class="highlight" style="background-color:">pathway</span> in a PKC <span class="highlight" style="background-color:">pathway</span>-dependent manner that leads to Slt2/Mpk1 activation.</div></p></div><div class="aux"><div class="resc">Beth Burgwyn Fuchs, et al. Eukaryot Cell. 2009 November;8(11):1616-1625.</div><p class="links"><a class="dblinks" href="/pubmed/19717745" ref="ordinalpos=2321&amp;ncbi_uid=2061062&amp;link_uid=19717745">Citation</a><a class="dblinks" href="/pmc/articles/PMC2772411/" ref="ordinalpos=2321&amp;ncbi_uid=2061062&amp;link_uid=PMC2772411">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2322.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3877388/figure/pone-0085737-g006/" ref="ordinalpos=2322&amp;ncbi_uid=5669095&amp;link_uid=PMC3877388" href="/pmc/articles/PMC3877388/"><img src="/pmc/articles/instance/3877388/bin/pone.0085737.g006.gif" src-large="/pmc/articles/instance/3877388/bin/pone.0085737.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3877388/" ref="ordinalpos=2322&amp;ncbi_uid=5669095&amp;link_uid=PMC3877388" image-link="/pmc/articles/PMC3877388/figure/pone-0085737-g006/" class="imagepopup">Figure 6. Syndecan-1 modulates triple-negative breast cancer stem cell properties via the IL-6/STAT3, NFkB and Wnt <span class="highlight" style="background-color:">signaling</span> pathways..  From: Syndecan-1 (CD138) Modulates Triple-Negative Breast Cancer Stem Cell Properties via Regulation of LRP-6 and IL-6-Mediated STAT3 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>siRNA-mediated knockdown of Syndecan-1 expression results in decreased expression of IL-6, the IL-6R and the chemokine CCL20, possibly due to reduced activation of NFkB. Reduced expression of components of the IL-6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> results in decreased constitutive activation of STAT3 in MDA-MB-231 cells. Low expression of the Wnt-coreceptor LRP-6 in Syndecan-1-deficient cells may reduce responsiveness to Wnt <span class="highlight" style="background-color:">signaling</span>. The attenuation of mutiple stemness-related <span class="highlight" style="background-color:">signaling</span> events results in a reduction of the SP and ALDH-activity, two surrogate parameters of stem cell activity. The reduction of the CD44(+)/CD24(-/low) phenotype may have implications for novel Syndecan-1-centered therapeutic approaches of basal-like breast cancer.</div></p></div><div class="aux"><div class="resc">Sherif A. Ibrahim, et al. PLoS One. 2013;8(12):e85737.</div><p class="links"><a class="dblinks" href="/pubmed/24392029" ref="ordinalpos=2322&amp;ncbi_uid=5669095&amp;link_uid=24392029">Citation</a><a class="dblinks" href="/pmc/articles/PMC3877388/" ref="ordinalpos=2322&amp;ncbi_uid=5669095&amp;link_uid=PMC3877388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2323.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3567737/figure/fig6/" ref="ordinalpos=2323&amp;ncbi_uid=5760024&amp;link_uid=PMC3567737" href="/pmc/articles/PMC3567737/"><img src="/pmc/articles/instance/3567737/bin/467fig6.gif" src-large="/pmc/articles/instance/3567737/bin/467fig6.jpg" alt="Figure 6 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3567737/" ref="ordinalpos=2323&amp;ncbi_uid=5760024&amp;link_uid=PMC3567737" image-link="/pmc/articles/PMC3567737/figure/fig6/" class="imagepopup">Figure 6 .  From: Dysfunctional Mitochondria Modulate cAMP-PKA <span class="highlight" style="background-color:">Signaling</span> and Filamentous and Invasive Growth of Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>Model showing the role of mitochondrial function in the regulation of cAMP-PKA <span class="highlight" style="background-color:">signaling</span>. Mitochondrial dysfunction triggers downregulation of the cAMP-PKA <span class="highlight" style="background-color:">pathway</span> in Σ1278b cells. This leads to a defective filamentous response despite the wild-type level activity of the RTG and FG MAPK pathways. Downregulation of the cAMP-PKA <span class="highlight" style="background-color:">pathway</span> results in modulation of several cellular processes in addition to filamentation in respiratory-deficient mutants. Activation is shown by arrows and inhibition by T-bars.</div></p></div><div class="aux"><div class="resc">Anu Aun, et al. Genetics. 2013 February;193(2):467-481.</div><p class="links"><a class="dblinks" href="/pubmed/23172851" ref="ordinalpos=2323&amp;ncbi_uid=5760024&amp;link_uid=23172851">Citation</a><a class="dblinks" href="/pmc/articles/PMC3567737/" ref="ordinalpos=2323&amp;ncbi_uid=5760024&amp;link_uid=PMC3567737">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2324.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2377155/figure/pcbi-1000086-g003/" ref="ordinalpos=2324&amp;ncbi_uid=1358561&amp;link_uid=PMC2377155" href="/pmc/articles/PMC2377155/"><img src="/pmc/articles/instance/2377155/bin/pcbi.1000086.g003.gif" src-large="/pmc/articles/instance/2377155/bin/pcbi.1000086.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2377155/" ref="ordinalpos=2324&amp;ncbi_uid=1358561&amp;link_uid=PMC2377155" image-link="/pmc/articles/PMC2377155/figure/pcbi-1000086-g003/" class="imagepopup">Figure 3. A representative integrated module in S. cerevisiae..  From: Dynamic Analysis of Integrated <span class="highlight" style="background-color:">Signaling</span>, Metabolic, and Regulatory Networks. </a></p><div class="supp"><p class="details"><div>A representative integrated module in <i>S. cerevisiae</i>, the high osmolarity glycerol (HOG) <span class="highlight" style="background-color:">pathway</span>, is illustrated. <span class="highlight" style="background-color:">Signaling</span> reactions appear in the upper left of the figure, metabolic reactions in the upper right, and regulatory reactions in the bottom. The components and reactions within these networks are based on published literature on the <i>S. cerevisiae</i> HOG <span class="highlight" style="background-color:">pathway</span>. Detailed components and reactions are presented in Text S2.</div></p></div><div class="aux"><div class="resc">Jong Min Lee, et al. PLoS Comput Biol. 2008 May;4(5):e1000086.</div><p class="links"><a class="dblinks" href="/pubmed/18483615" ref="ordinalpos=2324&amp;ncbi_uid=1358561&amp;link_uid=18483615">Citation</a><a class="dblinks" href="/pmc/articles/PMC2377155/" ref="ordinalpos=2324&amp;ncbi_uid=1358561&amp;link_uid=PMC2377155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2325.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3772095/figure/pone-0073553-g005/" ref="ordinalpos=2325&amp;ncbi_uid=5335925&amp;link_uid=PMC3772095" href="/pmc/articles/PMC3772095/"><img src="/pmc/articles/instance/3772095/bin/pone.0073553.g005.gif" src-large="/pmc/articles/instance/3772095/bin/pone.0073553.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3772095/" ref="ordinalpos=2325&amp;ncbi_uid=5335925&amp;link_uid=PMC3772095" image-link="/pmc/articles/PMC3772095/figure/pone-0073553-g005/" class="imagepopup">Figure 5. Number of multipath genes per KEGG <span class="highlight" style="background-color:">pathway</span>..  From: Identification of Multipath Genes Differentially Expressed in <span class="highlight" style="background-color:">Pathway</span>-Targeted Microarrays in Zebrafish Infected and Surviving Spring Viremia Carp Virus (SVCV) Suggest Preventive Drug Candidates. </a></p><div class="supp"><p class="details"><div>The number of multipath genes having differential expression folds &lt;0.66 or &gt; 1.5 (differentially expressed multipath genes of Table 1) in each of the 20 studied KEGG pathways, were represented.<br />Black bar, Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Hatched bars, rest of the pathways.</div></p></div><div class="aux"><div class="resc">Paloma Encinas, et al. PLoS One. 2013;8(9):e73553.</div><p class="links"><a class="dblinks" href="/pubmed/24069208" ref="ordinalpos=2325&amp;ncbi_uid=5335925&amp;link_uid=24069208">Citation</a><a class="dblinks" href="/pmc/articles/PMC3772095/" ref="ordinalpos=2325&amp;ncbi_uid=5335925&amp;link_uid=PMC3772095">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2326.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2885101/figure/f2-ijms-11-02188/" ref="ordinalpos=2326&amp;ncbi_uid=2306930&amp;link_uid=PMC2885101" href="/pmc/articles/PMC2885101/"><img src="/pmc/articles/instance/2885101/bin/ijms-11-02188f2.gif" src-large="/pmc/articles/instance/2885101/bin/ijms-11-02188f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2885101/" ref="ordinalpos=2326&amp;ncbi_uid=2306930&amp;link_uid=PMC2885101" image-link="/pmc/articles/PMC2885101/figure/f2-ijms-11-02188/" class="imagepopup">Figure 2. From: Apoptotic Effects of Chrysin in Human Cancer Cell Lines. </a></p><div class="supp"><p class="details"><div>The PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Chrysin is likely to act via activation of caspases and inactivation of Akt <span class="highlight" style="background-color:">signaling</span> in leukemia cells. (•) depicts chrysin.</div></p></div><div class="aux"><div class="resc">Boon Yin Khoo, et al. Int J Mol Sci. 2010;11(5):2188-2199.</div><p class="links"><a class="dblinks" href="/pubmed/20559509" ref="ordinalpos=2326&amp;ncbi_uid=2306930&amp;link_uid=20559509">Citation</a><a class="dblinks" href="/pmc/articles/PMC2885101/" ref="ordinalpos=2326&amp;ncbi_uid=2306930&amp;link_uid=PMC2885101">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2327.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847167/figure/F4/" ref="ordinalpos=2327&amp;ncbi_uid=5579785&amp;link_uid=PMC3847167" href="/pmc/articles/PMC3847167/"><img src="/pmc/articles/instance/3847167/bin/1750-1172-8-87-4.gif" src-large="/pmc/articles/instance/3847167/bin/1750-1172-8-87-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847167/" ref="ordinalpos=2327&amp;ncbi_uid=5579785&amp;link_uid=PMC3847167" image-link="/pmc/articles/PMC3847167/figure/F4/" class="imagepopup">Figure 4.  From: LINS, a modulator of the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, is involved in human cognition. </a></p><div class="supp"><p class="details"><div><b>Lines/LINS plays a putative dual role in WNT canonical <span class="highlight" style="background-color:">pathway</span>.</b> In WNT canonical <span class="highlight" style="background-color:">pathway</span>, the absence of a signal leads to the hyperphosphorylation of arm/CTNNB1 leading to its ubiqitination and degradation by the proteasome in the cytoplasm. Binding of wg/WNT1 ligand to a Fz and arr/LRP receptor complex leads to stabilization of hypophosphorylated arm/CTNNB1, translocating it to the nucleus. In the nucleus, arm/CTNNB1 competes with and displaces gro/TLE interacting with pan/TCF proteins to activate transcription. In Drosophila, lines/LINS was found to act as a modulator of wg/WNT canonical <span class="highlight" style="background-color:">pathway</span> acting in parallel with or downstream of arm/CTNNB1 in response to wg/WNT1 <span class="highlight" style="background-color:">signaling</span> to enhance or represses the transcription of target genes. Frizzled (Fz), arrow (arr), LDL receptor-related protein (LRP), armallido (Arm), β-catenin (CTNNB1), groucho (gro), transducin-like enhancer of split (TLE), pangolin (pan), T-cell factor (TCF).</div></p></div><div class="aux"><div class="resc">Nadia A Akawi, et al. Orphanet J Rare Dis. 2013;8:87-87.</div><p class="links"><a class="dblinks" href="/pubmed/23773660" ref="ordinalpos=2327&amp;ncbi_uid=5579785&amp;link_uid=23773660">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847167/" ref="ordinalpos=2327&amp;ncbi_uid=5579785&amp;link_uid=PMC3847167">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2328.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3043160/figure/F4/" ref="ordinalpos=2328&amp;ncbi_uid=3644495&amp;link_uid=PMC3043160" href="/pmc/articles/PMC3043160/"><img src="/pmc/articles/instance/3043160/bin/nihms244054f4.gif" src-large="/pmc/articles/instance/3043160/bin/nihms244054f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3043160/" ref="ordinalpos=2328&amp;ncbi_uid=3644495&amp;link_uid=PMC3043160" image-link="/pmc/articles/PMC3043160/figure/F4/" class="imagepopup">Figure 4. Mechanosensory cilia, cytosolic calcium and Wnt <span class="highlight" style="background-color:">signaling</span>.  From: Calcium-mediated mechanisms of cystic expansion. </a></p><div class="supp"><p class="details"><div>Bending or activation of the cilia results in maintaining cytosolic calcium (Ca<sup>2+</sup>), followed by activation of inversin. Calcium and inversin function as the molecular switches for Wnt <span class="highlight" style="background-color:">signaling</span> pathways. The Wnt canonical <span class="highlight" style="background-color:">pathway</span> involves β-catenin. Both β-catenin and E-cadherin regulate cell differentiation and proliferation. The Wnt non-canonical <span class="highlight" style="background-color:">pathway</span> involves small GTPase rho and JNK, both of which play an important role in planar cell polarity.</div></p></div><div class="aux"><div class="resc">Shakila Abdul-Majeed, et al. Biochim Biophys Acta. ;1812(10):1281-1290.</div><p class="links"><a class="dblinks" href="/pubmed/20932898" ref="ordinalpos=2328&amp;ncbi_uid=3644495&amp;link_uid=20932898">Citation</a><a class="dblinks" href="/pmc/articles/PMC3043160/" ref="ordinalpos=2328&amp;ncbi_uid=3644495&amp;link_uid=PMC3043160">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2329.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2840099/figure/fig07/" ref="ordinalpos=2329&amp;ncbi_uid=2449228&amp;link_uid=PMC2840099" href="/pmc/articles/PMC2840099/"><img src="/pmc/articles/instance/2840099/bin/pnas.1000300107fig07.gif" src-large="/pmc/articles/instance/2840099/bin/pnas.1000300107fig07.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2840099/" ref="ordinalpos=2329&amp;ncbi_uid=2449228&amp;link_uid=PMC2840099" image-link="/pmc/articles/PMC2840099/figure/fig07/" class="imagepopup">Fig. 7. From: Regulation of PI3-kinase/Akt <span class="highlight" style="background-color:">signaling</span> by muscle-enriched microRNA-486. </a></p><div class="supp"><p class="details"><div>Proposed model of miR-486 regulation and function in muscle cells. miR-486 transcription is activated by MyoD and MRTF-A. miR-486 directly represses the translation of PTEN and Foxo1a, two crucial negative regulators of PI3K/Akt <span class="highlight" style="background-color:">signaling</span>, resulting in the phosphorylation of Akt and the activation of the <span class="highlight" style="background-color:">pathway</span>. Activated Akt results in the phosphorylation of GSK3β and the inhibition of Foxo1a activity.</div></p></div><div class="aux"><div class="resc">Eric M. Small, et al. Proc Natl Acad Sci U S A. 2010 March 2;107(9):4218-4223.</div><p class="links"><a class="dblinks" href="/pubmed/20142475" ref="ordinalpos=2329&amp;ncbi_uid=2449228&amp;link_uid=20142475">Citation</a><a class="dblinks" href="/pmc/articles/PMC2840099/" ref="ordinalpos=2329&amp;ncbi_uid=2449228&amp;link_uid=PMC2840099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2330.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC150971/figure/F1/" ref="ordinalpos=2330&amp;ncbi_uid=241814&amp;link_uid=PMC150971" href="/pmc/articles/PMC150971/"><img src="/pmc/articles/instance/150971/bin/JCI0215644.f1.gif" src-large="/pmc/articles/instance/150971/bin/JCI0215644.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC150971/" ref="ordinalpos=2330&amp;ncbi_uid=241814&amp;link_uid=PMC150971" image-link="/pmc/articles/PMC150971/figure/F1/" class="imagepopup">Figure 1.  From: Series Introduction: JAK-STAT <span class="highlight" style="background-color:">signaling</span> in human disease. </a></p><div class="supp"><p class="details"><div>The JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Upon binding ligand, receptor-associated JAKs become activated and mediate phosphorylation of specific receptor tyrosine residues. This leads to the recruitment of specific STATs, which are then also tyrosine-phosphorylated. Activated STATs are released from the receptor, dimerize, translocate to the nucleus, and bind to members of the γ-activated site (GAS) family of enhancers.</div></p></div><div class="aux"><div class="resc">Christian W. Schindler. J Clin Invest. 2002 May 1;109(9):1133-1137.</div><p class="links"><a class="dblinks" href="/pubmed/11994400" ref="ordinalpos=2330&amp;ncbi_uid=241814&amp;link_uid=11994400">Citation</a><a class="dblinks" href="/pmc/articles/PMC150971/" ref="ordinalpos=2330&amp;ncbi_uid=241814&amp;link_uid=PMC150971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2331.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3842013/figure/F2/" ref="ordinalpos=2331&amp;ncbi_uid=5525297&amp;link_uid=PMC3842013" href="/pmc/articles/PMC3842013/"><img src="/pmc/articles/instance/3842013/bin/nihms387699f2.gif" src-large="/pmc/articles/instance/3842013/bin/nihms387699f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3842013/" ref="ordinalpos=2331&amp;ncbi_uid=5525297&amp;link_uid=PMC3842013" image-link="/pmc/articles/PMC3842013/figure/F2/" class="imagepopup">Figure 2. Novel role of adenosine <span class="highlight" style="background-color:">signaling</span> in normal and sickle erythrocytes.  From: Adenosine <span class="highlight" style="background-color:">signaling</span> in normal and sickle erythrocytes and beyond. </a></p><div class="supp"><p class="details"><div>Extracellular levels of adenosine increase in response to hypoxia (see Figure 1). Elevated adenosine activates ADORA2B adenosine receptors on erythrocytes, thereby activating downstream <span class="highlight" style="background-color:">signaling</span> pathways resulting in increased intracellular 2,3-bisphosphoglycerate (2,3-BPG), an allosteric regulator of hemoglobin (Hb) that reduces oxygen-binding affinity. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is beneficial to for normal individuals leading to increased oxygen release to hypoxic tissues. However, this process is detrimental for individuals with sickle cell disease by promoting the release of oxygen from sickle hemoglobin (HbS), resulting in increased concentrations of deoxy-HbS, increased deoxy-HbS polymerization and sickling.</div></p></div><div class="aux"><div class="resc">Yujin Zhang, et al. Microbes Infect. ;14(10):10.1016/j.micinf.2012.05.005.</div><p class="links"><a class="dblinks" href="/pubmed/22634345" ref="ordinalpos=2331&amp;ncbi_uid=5525297&amp;link_uid=22634345">Citation</a><a class="dblinks" href="/pmc/articles/PMC3842013/" ref="ordinalpos=2331&amp;ncbi_uid=5525297&amp;link_uid=PMC3842013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2332.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2782488/figure/F1/" ref="ordinalpos=2332&amp;ncbi_uid=2161871&amp;link_uid=PMC2782488" href="/pmc/articles/PMC2782488/"><img src="/pmc/articles/instance/2782488/bin/nihms-107421-f0001.gif" src-large="/pmc/articles/instance/2782488/bin/nihms-107421-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2782488/" ref="ordinalpos=2332&amp;ncbi_uid=2161871&amp;link_uid=PMC2782488" image-link="/pmc/articles/PMC2782488/figure/F1/" class="imagepopup">Figure 1. Cytokine <span class="highlight" style="background-color:">signaling</span> in T cell development and function.  From: Cytokine <span class="highlight" style="background-color:">signaling</span> modules in inflammatory responses. </a></p><div class="supp"><p class="details"><div>(A) Stat5 <span class="highlight" style="background-color:">signaling</span> from cytokines that use γc is essential for T cell development. Mice or humans lacking key components of this <span class="highlight" style="background-color:">pathway</span> (γc, Jak3, Stat5) fail to develop T cells. (B) Stat5 <span class="highlight" style="background-color:">signaling</span> controls the development of FoxP3-positive T regs. (C) Stat3 is crucial to the development and function of Th17 cells. IL-23 and IL-6 enforce Th17 development via the direct or indirect induction of <i>Rorc</i> and <i>Rora</i> expression. Stat3 also regulates the expression of IL-17-encoding genes and <i>Il21</i>, which acts in an autocrine/paracrine way to regulate Th17 cells. Socs3 is an important inhibitor of cytokines that use gp130 (IL-23R and IL-6R) while Socs1 is anticipated to inhibit any cytokines that use γc (IL-7, IL-21 as shown).</div></p></div><div class="aux"><div class="resc">John J. O’Shea, et al. Immunity. ;28(4):477-487.</div><p class="links"><a class="dblinks" href="/pubmed/18400190" ref="ordinalpos=2332&amp;ncbi_uid=2161871&amp;link_uid=18400190">Citation</a><a class="dblinks" href="/pmc/articles/PMC2782488/" ref="ordinalpos=2332&amp;ncbi_uid=2161871&amp;link_uid=PMC2782488">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2333.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4116751/figure/F7/" ref="ordinalpos=2333&amp;ncbi_uid=6430929&amp;link_uid=PMC4116751" href="/pmc/articles/PMC4116751/"><img src="/pmc/articles/instance/4116751/bin/nihms605701f7.gif" src-large="/pmc/articles/instance/4116751/bin/nihms605701f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4116751/" ref="ordinalpos=2333&amp;ncbi_uid=6430929&amp;link_uid=PMC4116751" image-link="/pmc/articles/PMC4116751/figure/F7/" class="imagepopup">Figure 7. Model for the crosstalk between LIN-3/LET-23 (ligand/receptor tyrosine kinase) <span class="highlight" style="background-color:">signaling</span> and LIN-12/Notch <span class="highlight" style="background-color:">signaling</span> in specification of vulval cell fate.  From: Dissection of Genetic Pathways in C. elegans. </a></p><div class="supp"><p class="details"><div>Through LET-23/EGFR receptor, the graded inductive signal, LIN-3/EGF, secreted from the anchor cell (AC) promotes the 1° fate and activates expression of genes encoding ligands for the LIN-12 receptor in P6.p. Lateral <span class="highlight" style="background-color:">signaling</span> mediated by these ligands via LIN-12 promotes gene expression specific for the 2° fate in P5.p and P7.p, including <i>lst</i> (lateral signal target) genes. The <i>lst</i> genes that encode inhibitors of the LIN-3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> antagonize the 1°-fate inductive effects of LIN-3 on P5.p and P7.p.</div></p></div><div class="aux"><div class="resc">Zheng Wang, et al. Methods Cell Biol. ;106:113-157.</div><p class="links"><a class="dblinks" href="/pubmed/22118276" ref="ordinalpos=2333&amp;ncbi_uid=6430929&amp;link_uid=22118276">Citation</a><a class="dblinks" href="/pmc/articles/PMC4116751/" ref="ordinalpos=2333&amp;ncbi_uid=6430929&amp;link_uid=PMC4116751">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2334.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4293489/figure/Fig1/" ref="ordinalpos=2334&amp;ncbi_uid=7088214&amp;link_uid=PMC4293489" href="/pmc/articles/PMC4293489/"><img src="/pmc/articles/instance/4293489/bin/18_2014_1762_Fig1_HTML.gif" src-large="/pmc/articles/instance/4293489/bin/18_2014_1762_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4293489/" ref="ordinalpos=2334&amp;ncbi_uid=7088214&amp;link_uid=PMC4293489" image-link="/pmc/articles/PMC4293489/figure/Fig1/" class="imagepopup">Fig. 1.  From: Co-operation of TLR4 and raft proteins in LPS-induced pro-inflammatory <span class="highlight" style="background-color:">signaling</span> . </a></p><div class="supp"><p class="details"><div>Activation of TLR4 by LPS. LBP facilitates transfer of LPS monomers to CD14 with the help of LBP and CD14 subsequently shifts the endotoxin to TLR4/MD-2 complex. Dimerization of the receptor complex induces the assembly of TIRAP, MyD88, and IRAK kinases in a myddosome at the TIR domain of TLR4 inducing a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to production of pro-inflammatory cytokines. After endocytosis, TRAM and TRIF associate with TLR4 triggering a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> which controls production of type I interferons and some other cytokines. The presence of leucine-rich repeats in CD14 and TLR4 is marked by <i>ellipses</i>. The molecules are drawn not to scale</div></p></div><div class="aux"><div class="resc">Agnieszka Płóciennikowska, et al. Cell Mol Life Sci. 2015;72:557-581.</div><p class="links"><a class="dblinks" href="/pubmed/25332099" ref="ordinalpos=2334&amp;ncbi_uid=7088214&amp;link_uid=25332099">Citation</a><a class="dblinks" href="/pmc/articles/PMC4293489/" ref="ordinalpos=2334&amp;ncbi_uid=7088214&amp;link_uid=PMC4293489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2335.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2729340/figure/fig6/" ref="ordinalpos=2335&amp;ncbi_uid=1970481&amp;link_uid=PMC2729340" href="/pmc/articles/PMC2729340/"><img src="/pmc/articles/instance/2729340/bin/DEV034595F6.gif" src-large="/pmc/articles/instance/2729340/bin/DEV034595F6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2729340/" ref="ordinalpos=2335&amp;ncbi_uid=1970481&amp;link_uid=PMC2729340" image-link="/pmc/articles/PMC2729340/figure/fig6/" class="imagepopup">Fig. 6. From: Somatic cAMP <span class="highlight" style="background-color:">signaling</span> regulates MSP-dependent oocyte growth and meiotic  maturation in C. elegans. </a></p><div class="supp"><p class="details"><div><b>A model for the control of oocyte meiotic maturation by  G</b>α<b><sub>s</sub>-adenylate cyclase <span class="highlight" style="background-color:">signaling</span> in the gonadal sheath  cells.</b> MSP is proposed to trigger Gα<sub>s</sub>-adenylate cyclase  <span class="highlight" style="background-color:">signaling</span> resulting in PKA activation. Based on genetic epistasis analysis,  Gα<sub>s</sub>-adenylate cyclase <span class="highlight" style="background-color:">signaling</span> is proposed to antagonize at  least two pathways that inhibit meiotic maturation, one involving the germline  innexins INX-14 and INX-22, and an uncharacterized <span class="highlight" style="background-color:">pathway</span> defined by  mutations (X?) that suppress <i>acy-4(lf)</i> (S.K., J.A.G. and D.G.,  unpublished). See text for details.</div></p></div><div class="aux"><div class="resc">J. Amaranath Govindan, et al. Development. 2009 July 1;136(13):2211-2221.</div><p class="links"><a class="dblinks" href="/pubmed/19502483" ref="ordinalpos=2335&amp;ncbi_uid=1970481&amp;link_uid=19502483">Citation</a><a class="dblinks" href="/pmc/articles/PMC2729340/" ref="ordinalpos=2335&amp;ncbi_uid=1970481&amp;link_uid=PMC2729340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2336.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC18851/figure/F6/" ref="ordinalpos=2336&amp;ncbi_uid=65214&amp;link_uid=PMC18851" href="/pmc/articles/PMC18851/"><img src="/pmc/articles/instance/18851/bin/pq2204103006.gif" src-large="/pmc/articles/instance/18851/bin/pq2204103006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC18851/" ref="ordinalpos=2336&amp;ncbi_uid=65214&amp;link_uid=PMC18851" image-link="/pmc/articles/PMC18851/figure/F6/" class="imagepopup">Figure 6.  From: Amplification of dopaminergic <span class="highlight" style="background-color:">signaling</span> by a positive  feedback loop. </a></p><div class="supp"><p class="details"><div>Model illustrating the regulation of dopamine <span class="highlight" style="background-color:">signaling</span> by  Cdk5/phospho-Thr-75 DARPP-32 and regulation of the  Cdk5/phospho-Thr-75 DARPP-32 <span class="highlight" style="background-color:">pathway</span> by dopamine. The Cdk5 <span class="highlight" style="background-color:">signaling</span>  and PKA <span class="highlight" style="background-color:">signaling</span> pathways, through a positive feedback loop, are  mutually antagonistic. (<i>Left</i>) Under basal conditions,  phosphorylation of DARPP-32 at Thr-75 by Cdk5 causes, successively,  inhibition of PKA, inactivation of PP-2A, and decreased  dephosphorylation of phospho-Thr-75 DARPP-32. Inhibition of PKA also  results in decreased phosphorylation of DARPP-32-Thr-34 and, therefore,  in activation of PP-1. Inhibition of PKA and activation of PP-1  synergistically reduce the phosphorylation of various substrates.  (<i>Right</i>) Dopamine, by sequentially activating dopamine D1  receptors, PKA and PP-2A, reduces the level of phospho-Thr-75 DARPP-32.  Dephosphorylation of DARPP-32 at Thr-75 by PP-2A removes the inhibition  of PKA. Activation of PKA also results in increased phosphorylation of  DARPP-32-Thr-34 and in inhibition of PP-1. Activation of PKA and  inhibition of PP-1 synergistically increase the phosphorylation of  various substrates.</div></p></div><div class="aux"><div class="resc">Akinori Nishi, et al. Proc Natl Acad Sci U S A. 2000 November 7;97(23):12840-12845.</div><p class="links"><a class="dblinks" href="/pubmed/11050161" ref="ordinalpos=2336&amp;ncbi_uid=65214&amp;link_uid=11050161">Citation</a><a class="dblinks" href="/pmc/articles/PMC18851/" ref="ordinalpos=2336&amp;ncbi_uid=65214&amp;link_uid=PMC18851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2337.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3700017/figure/f4/" ref="ordinalpos=2337&amp;ncbi_uid=6446068&amp;link_uid=PMC3700017" href="/pmc/articles/PMC3700017/"><img src="/pmc/articles/instance/3700017/bin/fig-4.gif" src-large="/pmc/articles/instance/3700017/bin/fig-4.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3700017/" ref="ordinalpos=2337&amp;ncbi_uid=6446068&amp;link_uid=PMC3700017" image-link="/pmc/articles/PMC3700017/figure/f4/" class="imagepopup">FIG. 4. From: Does Scavenging of Mitochondrial Superoxide Attenuate Cancer Prosurvival <span class="highlight" style="background-color:">Signaling</span> Pathways?. </a></p><div class="supp"><p class="details"><div><b>Proposed scheme of attenuation of cancer prosurvival <span class="highlight" style="background-color:">signaling</span> pathways by mitochondria-targeted antioxidants.</b> Mitochondrial ROS decrease the Δψ, activates cytoplasmic redox-dependent Akt/Erk1/2 <span class="highlight" style="background-color:">signaling</span> promoting cell growth and survival, and increases expression of HIF1-α leading to PDH inactivation and activation of the glycolytic <span class="highlight" style="background-color:">pathway</span>. Blocking mitochondrial ROS by mitochondria-targeted antioxidants prevents Akt/Erk activation and decreases HIF1-α expression, leading to inhibition of glycolysis and reduced cancer cell growth and survival.</div></p></div><div class="aux"><div class="resc">Rafal R. Nazarewicz, et al. Antioxid Redox Signal. 2013 August 1;19(4):344-349.</div><p class="links"><a class="dblinks" href="/pubmed/23373855" ref="ordinalpos=2337&amp;ncbi_uid=6446068&amp;link_uid=23373855">Citation</a><a class="dblinks" href="/pmc/articles/PMC3700017/" ref="ordinalpos=2337&amp;ncbi_uid=6446068&amp;link_uid=PMC3700017">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2338.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3348409/figure/F1/" ref="ordinalpos=2338&amp;ncbi_uid=4851113&amp;link_uid=PMC3348409" href="/pmc/articles/PMC3348409/"><img src="/pmc/articles/instance/3348409/bin/nihms-357949-f0001.gif" src-large="/pmc/articles/instance/3348409/bin/nihms-357949-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3348409/" ref="ordinalpos=2338&amp;ncbi_uid=4851113&amp;link_uid=PMC3348409" image-link="/pmc/articles/PMC3348409/figure/F1/" class="imagepopup">Figure 1.  From: Noise in cellular <span class="highlight" style="background-color:">signaling</span> pathways: causes and effects. </a></p><div class="supp"><p class="details"><div>Classical linear <span class="highlight" style="background-color:">signaling</span> versus the high-connectivity interactome. The high number and intricacy of protein-protein interactions revealed by the human interactome <sup>8-10</sup> (Figure taken from http://www.unihi.org/) (right panel) and kinome <sup>12</sup> is in contrast to the traditionally depicted linear <span class="highlight" style="background-color:">signaling</span> cascades, where each component of the <span class="highlight" style="background-color:">pathway</span> interacts with only a few others (left panel), from the membrane-localized receptor (elongated blue oval) to the nucleus (gray oval).</div></p></div><div class="aux"><div class="resc">John E. Ladbury, et al. Trends Biochem Sci. ;37(5):173-178.</div><p class="links"><a class="dblinks" href="/pubmed/22341496" ref="ordinalpos=2338&amp;ncbi_uid=4851113&amp;link_uid=22341496">Citation</a><a class="dblinks" href="/pmc/articles/PMC3348409/" ref="ordinalpos=2338&amp;ncbi_uid=4851113&amp;link_uid=PMC3348409">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2339.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4129017/figure/fig03/" ref="ordinalpos=2339&amp;ncbi_uid=6489264&amp;link_uid=PMC4129017" href="/pmc/articles/PMC4129017/"><img src="/pmc/articles/instance/4129017/bin/ijcep0007-4039-f3.gif" src-large="/pmc/articles/instance/4129017/bin/ijcep0007-4039-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4129017/" ref="ordinalpos=2339&amp;ncbi_uid=6489264&amp;link_uid=PMC4129017" image-link="/pmc/articles/PMC4129017/figure/fig03/" class="imagepopup">Figure 3.  From: miR-218 modulate hepatocellular carcinoma cell proliferation through PTEN/AKT/PI3K <span class="highlight" style="background-color:">pathway</span> and HoxA10. </a></p><div class="supp"><p class="details"><div>miR-218 regulated PTEN/AKT/PI3K <span class="highlight" style="background-color:">pathway</span> in HCC. (A) MHC97L cells were transfected with miR-218-mimics and miR-218-inhibitor, along with their corresponding control siRNAs. Western blotting analysis was used to examine the cell cycle proteins Cyclin D1 and p21, and (B) PTEN/AKT/PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related proteins, including AKT/phosphorylated-AKT, GSK-3β phosphorylated-GSK-3β PTEN.</div></p></div><div class="aux"><div class="resc">Zhong-Di Xiao, et al. Int J Clin Exp Pathol. 2014;7(7):4039-4044.</div><p class="links"><a class="dblinks" href="/pubmed/25120782" ref="ordinalpos=2339&amp;ncbi_uid=6489264&amp;link_uid=25120782">Citation</a><a class="dblinks" href="/pmc/articles/PMC4129017/" ref="ordinalpos=2339&amp;ncbi_uid=6489264&amp;link_uid=PMC4129017">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2340.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3749252/figure/F1/" ref="ordinalpos=2340&amp;ncbi_uid=5873668&amp;link_uid=PMC3749252" href="/pmc/articles/PMC3749252/"><img src="/pmc/articles/instance/3749252/bin/nihms-489580-f0001.gif" src-large="/pmc/articles/instance/3749252/bin/nihms-489580-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3749252/" ref="ordinalpos=2340&amp;ncbi_uid=5873668&amp;link_uid=PMC3749252" image-link="/pmc/articles/PMC3749252/figure/F1/" class="imagepopup">Figure 1.  From: Oxysterol binding to the extracellular domain of Smoothened in Hedgehog <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>22-azacholesterol inhibits vertebrate Hh <span class="highlight" style="background-color:">signaling</span><br />(a) Structure of 22(S)-azacholesterol (22-NHC, <b>1</b>) and 20(S)-hydroxycholesterol (20(S)-OHC).<br />(b) Shh Light II cells were treated with various concentrations of Shh, in the presence of increasing amounts of 22-NHC, and Hh <span class="highlight" style="background-color:">pathway</span> activation was measured by luciferase assay. Error bars represent standard deviation (n=4 independent experiments). 22-NHC inhibits Hh <span class="highlight" style="background-color:">pathway</span> activation by Shh but does not significantly change the EC50 of Shh.<br />(c) As in (b) but Hh <span class="highlight" style="background-color:">signaling</span> was activated by various concentrations of the 20-OHC analog, 20-OHC-Pent. 22-NHC inhibits Hh <span class="highlight" style="background-color:">pathway</span> activation by 20-OHC-Pent, without significantly changing the EC50.<br />(d) As in (b) but Hh <span class="highlight" style="background-color:">signaling</span> was activated by various concentrations of SAG. 22-NHC does not inhibit Hh <span class="highlight" style="background-color:">pathway</span> activation by SAG but decreases the EC50 for SAG.<br />(e) Smo−/− MEFs were rescued by stable expression of mSmo or the constitutively active mutant mSmoM2. Transcription of the Hh target gene, Gli1, was measured by Q-PCR in the absence or presence of 22-NHC (20 μM) or SANT1 (2 μM). Error bars indicate standard deviation (n=3 independent experiments). 22-NHC does not inhibit SmoM2.<br />(f) Shh Light II cells were stimulated with Shh, in the presence of increasing amounts of 22-NHC, with the addition of cylopamine or cyclopamine-KAAD. Hh <span class="highlight" style="background-color:">pathway</span> activity was assayed as in (b)<br />(g) As in (f) but with addition of SANT1, GDC0449 or itraconazole.</div></p></div><div class="aux"><div class="resc">Daniel Nedelcu, et al. Nat Chem Biol. ;9(9):557-564.</div><p class="links"><a class="dblinks" href="/pubmed/23831757" ref="ordinalpos=2340&amp;ncbi_uid=5873668&amp;link_uid=23831757">Citation</a><a class="dblinks" href="/pmc/articles/PMC3749252/" ref="ordinalpos=2340&amp;ncbi_uid=5873668&amp;link_uid=PMC3749252">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2341.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3513982/figure/f6/" ref="ordinalpos=2341&amp;ncbi_uid=5535609&amp;link_uid=PMC3513982" href="/pmc/articles/PMC3513982/"><img src="/pmc/articles/instance/3513982/bin/fig-6.gif" src-large="/pmc/articles/instance/3513982/bin/fig-6.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3513982/" ref="ordinalpos=2341&amp;ncbi_uid=5535609&amp;link_uid=PMC3513982" image-link="/pmc/articles/PMC3513982/figure/f6/" class="imagepopup">FIG. 6. From: Reconstruction of Biological Networks by Incorporating Prior Knowledge into Bayesian Network Models. </a></p><div class="supp"><p class="details"><div>RAF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reconstruction. (<b>a</b>) Network structure recovered from RAF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> data and direct-link prior knowledge. (<b>b</b>) Network structure recovered with additional ordering prior knowledge. Black solid edges are true positive edges, and colored dashed edges are false positive edges.</div></p></div><div class="aux"><div class="resc">Baikang Pei, et al. J Comput Biol. 2012 December;19(12):1324-1334.</div><p class="links"><a class="dblinks" href="/pubmed/23210479" ref="ordinalpos=2341&amp;ncbi_uid=5535609&amp;link_uid=23210479">Citation</a><a class="dblinks" href="/pmc/articles/PMC3513982/" ref="ordinalpos=2341&amp;ncbi_uid=5535609&amp;link_uid=PMC3513982">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2342.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2837985/figure/F9/" ref="ordinalpos=2342&amp;ncbi_uid=2443813&amp;link_uid=PMC2837985" href="/pmc/articles/PMC2837985/"><img src="/pmc/articles/instance/2837985/bin/zek9990934960009.gif" src-large="/pmc/articles/instance/2837985/bin/zek9990934960009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2837985/" ref="ordinalpos=2342&amp;ncbi_uid=2443813&amp;link_uid=PMC2837985" image-link="/pmc/articles/PMC2837985/figure/F9/" class="imagepopup">Fig. 9. From: Comparative Transcriptome Analysis Reveals Novel Roles of the Ras and Cyclic AMP <span class="highlight" style="background-color:">Signaling</span> Pathways in Environmental Stress Response and Antifungal Drug Sensitivity in Cryptococcus neoformans   . </a></p><div class="supp"><p class="details"><div>Proposed model for the regulatory mechanisms of the Ras and cAMP <span class="highlight" style="background-color:">signaling</span> pathways in stress response and antifungal drug susceptibility in C. neoformans. The Ras1-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promotes cell survival under diverse environmental stresses, such as cell wall destabilization, osmotic shock, oxidative and genotoxic stresses, and high temperature, in a manner mediated by Cdc24 and Cdc42. Ras1 appears to affect resistance to antifungal drugs by maintaining cell wall and membrane integrity. Along with Ras1, Ras2 plays some minor role in controlling antifungal and oxidative stress sensitivity. The cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in responses to stress from heavy metals and toxic metabolites. Notably, small heat shock proteins, Hsp12 and Hsp122, are positively regulated by the cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and appears to affect resistance against AmpB treatment. However, expression of <i>HSP12</i> and <i>HSP122</i> is much more significantly regulated by the HOG <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Shinae Maeng, et al. Eukaryot Cell. 2010 March;9(3):360-378.</div><p class="links"><a class="dblinks" href="/pubmed/20097740" ref="ordinalpos=2342&amp;ncbi_uid=2443813&amp;link_uid=20097740">Citation</a><a class="dblinks" href="/pmc/articles/PMC2837985/" ref="ordinalpos=2342&amp;ncbi_uid=2443813&amp;link_uid=PMC2837985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2343.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3085438/figure/F5/" ref="ordinalpos=2343&amp;ncbi_uid=3383904&amp;link_uid=PMC3085438" href="/pmc/articles/PMC3085438/"><img src="/pmc/articles/instance/3085438/bin/JCS084194F5.gif" src-large="/pmc/articles/instance/3085438/bin/JCS084194F5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3085438/" ref="ordinalpos=2343&amp;ncbi_uid=3383904&amp;link_uid=PMC3085438" image-link="/pmc/articles/PMC3085438/figure/F5/" class="imagepopup">Fig. 5. From: CCN6 (WISP3) decreases ZEB1-mediated EMT and invasion by attenuation of IGF-1 receptor <span class="highlight" style="background-color:">signaling</span> in breast cancer. </a></p><div class="supp"><p class="details"><div><b>Working model for the role of CCN6 in breast cancer invasion.</b> Our data support the hypothesis that CCN6 regulates the levels and activity of IGF-1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mammary epithelial cells. Low levels of CCN6, as found in aggressive invasive carcinomas of the breast, result in activation of IGF-1R <span class="highlight" style="background-color:">signaling</span> and increased ZEB1 levels and function.</div></p></div><div class="aux"><div class="resc">Guadalupe Lorenzatti, et al. J Cell Sci. 2011 May 15;124(10):1752-1758.</div><p class="links"><a class="dblinks" href="/pubmed/21525039" ref="ordinalpos=2343&amp;ncbi_uid=3383904&amp;link_uid=21525039">Citation</a><a class="dblinks" href="/pmc/articles/PMC3085438/" ref="ordinalpos=2343&amp;ncbi_uid=3383904&amp;link_uid=PMC3085438">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2344.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3839121/figure/fig5/" ref="ordinalpos=2344&amp;ncbi_uid=5567321&amp;link_uid=PMC3839121" href="/pmc/articles/PMC3839121/"><img src="/pmc/articles/instance/3839121/bin/MI2013-651207.005.gif" src-large="/pmc/articles/instance/3839121/bin/MI2013-651207.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3839121/" ref="ordinalpos=2344&amp;ncbi_uid=5567321&amp;link_uid=PMC3839121" image-link="/pmc/articles/PMC3839121/figure/fig5/" class="imagepopup">Figure 5.  From: The Role of EGFR/PI3K/Akt/cyclinD1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Acquired Middle Ear Cholesteatoma. </a></p><div class="supp"><p class="details"><div>EGFR/PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is required for cyclinD1 upregulation in EGF-stimulated EACKs. Cells were treated with 15 ng/mL of EGF for the indicated periods of time ((a), (b), (e)) or incubated with or without AG1478 or wortmannin for 1 h and then exposed to 15 ng/mL of EGF for the indicated periods of time ((c), (d), (f)). After treatment, cells were harvested for Western blotting analysis using specific antibodies as indicated, and <i>β</i>-actin was used as an equal loading control.</div></p></div><div class="aux"><div class="resc">Wei Liu, et al. Mediators Inflamm. 2013;2013:651207.</div><p class="links"><a class="dblinks" href="/pubmed/24311896" ref="ordinalpos=2344&amp;ncbi_uid=5567321&amp;link_uid=24311896">Citation</a><a class="dblinks" href="/pmc/articles/PMC3839121/" ref="ordinalpos=2344&amp;ncbi_uid=5567321&amp;link_uid=PMC3839121">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2345.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775578/figure/F1/" ref="ordinalpos=2345&amp;ncbi_uid=5309654&amp;link_uid=PMC3775578" href="/pmc/articles/PMC3775578/"><img src="/pmc/articles/instance/3775578/bin/nihms298539f1.gif" src-large="/pmc/articles/instance/3775578/bin/nihms298539f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775578/" ref="ordinalpos=2345&amp;ncbi_uid=5309654&amp;link_uid=PMC3775578" image-link="/pmc/articles/PMC3775578/figure/F1/" class="imagepopup">Figure 1. The Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Novel investigational drugs for basal cell carcinoma. </a></p><div class="supp"><p class="details"><div><b>A.</b> In the absence of Hh ligand, Ptch1 in the primary cilium represses Smo function, resulting in the proteolytic processing of Gli-activator (Gli1-A) (bound to SuFu, a negative regulator of Hh <span class="highlight" style="background-color:">signaling</span>) to Gli-repressor (Gli-R). The latter then binds to the promoters of Hh target genes to repress transcription. <b>B.</b> In the presence of Hh, Ptch1 translocates out of the cilium and is degraded, allowing Smo to enter the cilium and activate the Hh <span class="highlight" style="background-color:">pathway</span> by preventing cleavage of Gli proteins to its repressor form. Gli-A enters the nucleus and activates Hh-target gene (e.g., <i>Gli1, Ptch1, Bcl2, FoxM1</i>) transcription.</div></p></div><div class="aux"><div class="resc">Po-Lin So, et al. Expert Opin Investig Drugs. 2010 September;19(9):1099-1112.</div><p class="links"><a class="dblinks" href="/pubmed/20662553" ref="ordinalpos=2345&amp;ncbi_uid=5309654&amp;link_uid=20662553">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775578/" ref="ordinalpos=2345&amp;ncbi_uid=5309654&amp;link_uid=PMC3775578">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2346.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2213558/figure/F5/" ref="ordinalpos=2346&amp;ncbi_uid=1145377&amp;link_uid=PMC2213558" href="/pmc/articles/PMC2213558/"><img src="/pmc/articles/instance/2213558/bin/nihms37241f5.gif" src-large="/pmc/articles/instance/2213558/bin/nihms37241f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2213558/" ref="ordinalpos=2346&amp;ncbi_uid=1145377&amp;link_uid=PMC2213558" image-link="/pmc/articles/PMC2213558/figure/F5/" class="imagepopup">Figure 5. A model for the roles of TGFβ and Notch <span class="highlight" style="background-color:">signaling</span> pathways in patterning the early M lineage.  From: Dorsoventral patterning of the C. elegans postembryonic mesoderm requires both LIN-12/Notch and TGF-beta <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>A proposed model for how SMA-9 and the Sma/Mab TGFβ and LIN-12/Notch <span class="highlight" style="background-color:">signaling</span> pathways function in the M lineage in wild-type animals. Gray boxes represent cells of the M lineage. We propose that in the dorsal side of the M lineage, the Sma/Mab TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated and promotes the SM cell fate. SMA-9 antagonizes this action, resulting in the formation of CCs. LIN-12 activity is restricted to the ventral M-derived cells by the activity of LAG-2, APX-1 and DSL-1, and it also promotes the SM cell fate. Although SMA-9 is present in the ventral cells, it does not antagonize LIN-12 function, due to either the lack of a SMA-9 co-factor(s) or the lack of binding sites in the promoters of LIN-12 target genes. At present, we do not know whether the Sma/Mab TGFβ and the LIN-12/Notch <span class="highlight" style="background-color:">signaling</span> pathways promote the SM fate by activating the SM fate determinant(s) or by activating a repressor(s) of the CC fate determinant(s).</div></p></div><div class="aux"><div class="resc">Marisa L. Foehr, et al. Dev Biol. ;313(1):256-266.</div><p class="links"><a class="dblinks" href="/pubmed/18036582" ref="ordinalpos=2346&amp;ncbi_uid=1145377&amp;link_uid=18036582">Citation</a><a class="dblinks" href="/pmc/articles/PMC2213558/" ref="ordinalpos=2346&amp;ncbi_uid=1145377&amp;link_uid=PMC2213558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2347.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2819343/figure/F3/" ref="ordinalpos=2347&amp;ncbi_uid=2216833&amp;link_uid=PMC2819343" href="/pmc/articles/PMC2819343/"><img src="/pmc/articles/instance/2819343/bin/nihms-173131-f0003.gif" src-large="/pmc/articles/instance/2819343/bin/nihms-173131-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2819343/" ref="ordinalpos=2347&amp;ncbi_uid=2216833&amp;link_uid=PMC2819343" image-link="/pmc/articles/PMC2819343/figure/F3/" class="imagepopup">Fig. 3.  From: Death-associated protein kinase (DAPK) and signal transduction: fine-tuning of autophagy in Caenorhabditis elegans homeostasis. </a></p><div class="supp"><p class="details"><div>Schematic diagram of autophagy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i>. Autophagy inducing- and inhibiting <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i>. Plain lines and dashed lines indicate known and possible regulatory pathways, respectively. See the main text for details.</div></p></div><div class="aux"><div class="resc">Chanhee Kang, et al. FEBS J. ;277(1):66.</div><p class="links"><a class="dblinks" href="/pubmed/19878311" ref="ordinalpos=2347&amp;ncbi_uid=2216833&amp;link_uid=19878311">Citation</a><a class="dblinks" href="/pmc/articles/PMC2819343/" ref="ordinalpos=2347&amp;ncbi_uid=2216833&amp;link_uid=PMC2819343">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2348.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC135641/figure/f10/" ref="ordinalpos=2348&amp;ncbi_uid=297823&amp;link_uid=PMC135641" href="/pmc/articles/PMC135641/"><img src="/pmc/articles/instance/135641/bin/mb1820558010.gif" src-large="/pmc/articles/instance/135641/bin/mb1820558010.jpg" alt="FIG. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC135641/" ref="ordinalpos=2348&amp;ncbi_uid=297823&amp;link_uid=PMC135641" image-link="/pmc/articles/PMC135641/figure/f10/" class="imagepopup">FIG. 10. From: I-mfa Domain Proteins Interact with Axin and Affect Its Regulation of the Wnt and c-Jun N-Terminal Kinase <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Diagram of the effects of HIC and I-mfa on Axin-mediated <span class="highlight" style="background-color:">signaling</span>. (A) Effects on the Wnt <span class="highlight" style="background-color:">Pathway</span>. (Top panel) In the absence of HIC or I-mfa, Axin binds free β-catenin or β-catenin that is released from E-cadherin and promotes phosphorylation of β-catenin by GSK-3β and β-catenin degradation. TCF- or LEF-mediated transcription is not activated. (Middle panel) In the presence of HIC, the Axin-mediated decrease in the free β-catenin level is inhibited due to effects on Axin complex formation or GSK-3β activity. Axin inhibition of β-catenin activation of TCF- or LEF-mediated transcription is decreased. (Bottom panel) In the presence of I-mfa, the Axin-mediated decrease in the free β-catenin level is inhibited, and β-catenin activation of TCF- or LEF-mediated transcription is inhibited. (B) Effects on the JNK <span class="highlight" style="background-color:">pathway</span>. (Top panel) In the absence of GSK-3β, Axin binds MEKK, leading to activation of the JNK <span class="highlight" style="background-color:">pathway</span>. (Middle panel) HIC binding to Axin does not affect the binding of GSK-3β, and the activation of JNK is decreased. (Bottom panel) I-mfa binding to Axin does not affect the binding of GSK-3β, and the activation of JNK is decreased.</div></p></div><div class="aux"><div class="resc">Shuichi Kusano, et al. Mol Cell Biol. 2002 September;22(18):6393-6405.</div><p class="links"><a class="dblinks" href="/pubmed/12192039" ref="ordinalpos=2348&amp;ncbi_uid=297823&amp;link_uid=12192039">Citation</a><a class="dblinks" href="/pmc/articles/PMC135641/" ref="ordinalpos=2348&amp;ncbi_uid=297823&amp;link_uid=PMC135641">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2349.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3856107/figure/f3-ijms-14-23086/" ref="ordinalpos=2349&amp;ncbi_uid=5583571&amp;link_uid=PMC3856107" href="/pmc/articles/PMC3856107/"><img src="/pmc/articles/instance/3856107/bin/ijms-14-23086f3.gif" src-large="/pmc/articles/instance/3856107/bin/ijms-14-23086f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3856107/" ref="ordinalpos=2349&amp;ncbi_uid=5583571&amp;link_uid=PMC3856107" image-link="/pmc/articles/PMC3856107/figure/f3-ijms-14-23086/" class="imagepopup">Figure 3.  From: Let-7 in Cardiovascular Diseases, Heart Development and Cardiovascular Differentiation from Stem Cells. </a></p><div class="supp"><p class="details"><div>Roles and <span class="highlight" style="background-color:">signaling</span> pathways of let-7 in hypoxia- and neural stem/progenitor cells (NSPCs)-induced angiogenesis. Hypoxia treatment on endothelial cells (ECs) induced angiogenesis through hypoxia-inducible factor 1α (HIF1α)/let-7s/(AGO1)/VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In ECs co-incubated with NSPCs, pro-angiogenic factors released by the NSPCs induced angiogenesis through SMAD2 (SMAD3), and p53/IGF-1R/p-mTOR <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Mei-Hua Bao, et al. Int J Mol Sci. 2013 November;14(11):23086-23102.</div><p class="links"><a class="dblinks" href="/pubmed/24284400" ref="ordinalpos=2349&amp;ncbi_uid=5583571&amp;link_uid=24284400">Citation</a><a class="dblinks" href="/pmc/articles/PMC3856107/" ref="ordinalpos=2349&amp;ncbi_uid=5583571&amp;link_uid=PMC3856107">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2350.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC98275/figure/F5/" ref="ordinalpos=2350&amp;ncbi_uid=36192&amp;link_uid=PMC98275" href="/pmc/articles/PMC98275/"><img src="/pmc/articles/instance/98275/bin/ii0511683005.gif" src-large="/pmc/articles/instance/98275/bin/ii0511683005.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC98275/" ref="ordinalpos=2350&amp;ncbi_uid=36192&amp;link_uid=PMC98275" image-link="/pmc/articles/PMC98275/figure/F5/" class="imagepopup">FIG. 5.  From: Apoptotic <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activated by Helicobacter pylori Infection and Increase of Apoptosis-Inducing Activity under Serum-Starved Conditions. </a></p><div class="supp"><p class="details"><div>Possible mechanism for the <i>H. pylori</i>-induced apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The apoptosis-triggering protein(s) attaches to the host cell surface and then stimulates the death receptors through unknown systems. Stimulation of the death receptors induces apoptosis through sequential induction of caspase-8 activity, proapoptotic proteins Bad and Bid, caspase-9 activity, and caspase-3 activity. Under the serum-supplemented condition, expression of antiapoptotic factors of <i>H. pylori</i> would presumably increase, leading to activation of antiapoptotic pathways of the host cell. Antiapoptotic substances, such as growth factors, would also be responsible for inhibition of <i>H. pylori</i>-mediated apoptosis.</div></p></div><div class="aux"><div class="resc">Keigo Shibayama, et al. Infect Immun. 2001 May;69(5):3181-3189.</div><p class="links"><a class="dblinks" href="/pubmed/11292739" ref="ordinalpos=2350&amp;ncbi_uid=36192&amp;link_uid=11292739">Citation</a><a class="dblinks" href="/pmc/articles/PMC98275/" ref="ordinalpos=2350&amp;ncbi_uid=36192&amp;link_uid=PMC98275">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2351.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3684781/figure/F2/" ref="ordinalpos=2351&amp;ncbi_uid=5006392&amp;link_uid=PMC3684781" href="/pmc/articles/PMC3684781/"><img src="/pmc/articles/instance/3684781/bin/fpls-04-00204-g002.gif" src-large="/pmc/articles/instance/3684781/bin/fpls-04-00204-g002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3684781/" ref="ordinalpos=2351&amp;ncbi_uid=5006392&amp;link_uid=PMC3684781" image-link="/pmc/articles/PMC3684781/figure/F2/" class="imagepopup">FIGURE 2.  From: Multiple control levels of root system remodeling in arbuscular mycorrhizal symbiosis. </a></p><div class="supp"><p class="details"><div><b>Pre-symbiotic induction of lateral root formation in arbuscular mycorrhiza.</b> Germinating spore exudates (GSE) contain Myc-LCOs and possibly phytohormone-like compounds. Perception of Myc-LCOs leads to lateral root induction in <i>Medicago truncatula</i>, which requires the common symbiosis <span class="highlight" style="background-color:">signaling</span> components DMI1, DMI2, and DMI3 (brown <span class="highlight" style="background-color:">pathway</span>). The green <span class="highlight" style="background-color:">pathway</span> hypothesizes phytohormone-like <span class="highlight" style="background-color:">signaling</span> to operate either downstream or independent of common symbiosis <span class="highlight" style="background-color:">signaling</span> in <i>M. truncatula</i> and rice, respectively.</div></p></div><div class="aux"><div class="resc">Caroline Gutjahr, et al. Front Plant Sci. 2013;4:204.</div><p class="links"><a class="dblinks" href="/pubmed/23785383" ref="ordinalpos=2351&amp;ncbi_uid=5006392&amp;link_uid=23785383">Citation</a><a class="dblinks" href="/pmc/articles/PMC3684781/" ref="ordinalpos=2351&amp;ncbi_uid=5006392&amp;link_uid=PMC3684781">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2352.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3502106/figure/F5/" ref="ordinalpos=2352&amp;ncbi_uid=4383389&amp;link_uid=PMC3502106" href="/pmc/articles/PMC3502106/"><img src="/pmc/articles/instance/3502106/bin/1742-9994-9-16-5.gif" src-large="/pmc/articles/instance/3502106/bin/1742-9994-9-16-5.jpg" alt="Figure 5 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3502106/" ref="ordinalpos=2352&amp;ncbi_uid=4383389&amp;link_uid=PMC3502106" image-link="/pmc/articles/PMC3502106/figure/F5/" class="imagepopup">Figure 5 .  From: Comparative molecular developmental aspects of the mammalian- and the avian lungs, and the insectan tracheal system by branching morphogenesis: recent advances and future directions. </a></p><div class="supp"><p class="details"><div><b>Figure 8: Feed-forward <span class="highlight" style="background-color:">signaling</span> in which FGF-9 controls Wnt-2a expression and mesenchymal Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>, and in which mesenchymal Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> is requisite for mesenchymal FGFR expression and mesenchymal responsiveness to FGF-9.</b> In absence of either FGF-9 or canonical Wnt ligands, the mesenchymal FGF-9-Wnt/β-catenin feed-forward network breaks up, occasioning loss of mesenchymal FGFR expression and FGF responsiveness. Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> has to be preserved in order to induce or maintain FGFR expression, FGF-9 responsiveness, and continuance of FGF-Wnt/β-catenin feed-forward <span class="highlight" style="background-color:">signaling</span>. Loss of mesenchymal FGF-Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> increases Noggin expression in both the subepithelial and submesothelial mesenchyme. Lack of either <span class="highlight" style="background-color:">pathway</span> decreases mesenchymal proliferation. The change in the density of the shading of the epithelium shows the proximal-distal extent of the lung bud. Details can be acquired from Yin et al. [33], Rajagopal et al. [118], Yi et al. [119] De Langhe et al. [32], and Ornitz and Yin [18]. Modified after Ornitz and Yin [18].</div></p></div><div class="aux"><div class="resc">John N Maina. Front Zool. 2012;9:16-16.</div><p class="links"><a class="dblinks" href="/pubmed/22871018" ref="ordinalpos=2352&amp;ncbi_uid=4383389&amp;link_uid=22871018">Citation</a><a class="dblinks" href="/pmc/articles/PMC3502106/" ref="ordinalpos=2352&amp;ncbi_uid=4383389&amp;link_uid=PMC3502106">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2353.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4053266/figure/F3/" ref="ordinalpos=2353&amp;ncbi_uid=6277643&amp;link_uid=PMC4053266" href="/pmc/articles/PMC4053266/"><img src="/pmc/articles/instance/4053266/bin/1471-2105-15-162-3.gif" src-large="/pmc/articles/instance/4053266/bin/1471-2105-15-162-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4053266/" ref="ordinalpos=2353&amp;ncbi_uid=6277643&amp;link_uid=PMC4053266" image-link="/pmc/articles/PMC4053266/figure/F3/" class="imagepopup">Figure 3.  From: A multivariate approach to the integration of multi-omics datasets. </a></p><div class="supp"><p class="details"><div><b>Integrative <span class="highlight" style="background-color:">pathway</span> analysis. (A)</b> Shows the coordinates of proteins from the proteomics dataset (see as Figure 2F), where proteins from the leukemia tail are highlighted. In contrast to the microarray data, leukemia features are clearly represented in the proteome dataset. <b>(B)</b> The leukocyte extravasation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is significantly enriched in analysis resulting from integration of leukemia features from all platforms. Colors indicate features from different datasets. <b>(C)</b> Melanocyte development and pigmentation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was enriched in melanoma genes.</div></p></div><div class="aux"><div class="resc">Chen Meng, et al. BMC Bioinformatics. 2014;15:162-162.</div><p class="links"><a class="dblinks" href="/pubmed/24884486" ref="ordinalpos=2353&amp;ncbi_uid=6277643&amp;link_uid=24884486">Citation</a><a class="dblinks" href="/pmc/articles/PMC4053266/" ref="ordinalpos=2353&amp;ncbi_uid=6277643&amp;link_uid=PMC4053266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2354.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3767596/figure/pone-0073706-g008/" ref="ordinalpos=2354&amp;ncbi_uid=5301185&amp;link_uid=PMC3767596" href="/pmc/articles/PMC3767596/"><img src="/pmc/articles/instance/3767596/bin/pone.0073706.g008.gif" src-large="/pmc/articles/instance/3767596/bin/pone.0073706.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3767596/" ref="ordinalpos=2354&amp;ncbi_uid=5301185&amp;link_uid=PMC3767596" image-link="/pmc/articles/PMC3767596/figure/pone-0073706-g008/" class="imagepopup">Figure 8. A schematic model illustrating CD9-dependent regulation of LPS-induced inflammatory <span class="highlight" style="background-color:">signaling</span>..  From: Statins Decrease Lung Inflammation in Mice by Upregulating Tetraspanin CD9 in Macrophages. </a></p><div class="supp"><p class="details"><div>CD9 expression is downregulated by either inactivation of HDAC (e.g., by LPS or cigarette smoke exposure) or activation of the mevalonate <span class="highlight" style="background-color:">pathway</span>. The loss of CD9 causes the transfer of the LPS-<span class="highlight" style="background-color:">signaling</span> mediator CD14 from tetraspanin-enriched microdomains to lipid rafts and thereby leads to augmentation of LPS-induced inflammatory <span class="highlight" style="background-color:">signaling</span> in macrophages. Theophylline and dexamethasone (which activate HDACs) or statins (which inhibit the mevalonate <span class="highlight" style="background-color:">pathway</span>) upregulate CD9 and may reverse this cascade in inflammatory lung diseases, including COPD.</div></p></div><div class="aux"><div class="resc">Yingji Jin, et al. PLoS One. 2013;8(9):e73706.</div><p class="links"><a class="dblinks" href="/pubmed/24040034" ref="ordinalpos=2354&amp;ncbi_uid=5301185&amp;link_uid=24040034">Citation</a><a class="dblinks" href="/pmc/articles/PMC3767596/" ref="ordinalpos=2354&amp;ncbi_uid=5301185&amp;link_uid=PMC3767596">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2355.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4287412/figure/F9/" ref="ordinalpos=2355&amp;ncbi_uid=7048704&amp;link_uid=PMC4287412" href="/pmc/articles/PMC4287412/"><img src="/pmc/articles/instance/4287412/bin/nihms640187f9.gif" src-large="/pmc/articles/instance/4287412/bin/nihms640187f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4287412/" ref="ordinalpos=2355&amp;ncbi_uid=7048704&amp;link_uid=PMC4287412" image-link="/pmc/articles/PMC4287412/figure/F9/" class="imagepopup">Fig. 9.  From: Role of non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> in osteoblast maturation on microstructured titanium surfaces. </a></p><div class="supp"><p class="details"><div>Schematic of regulation of Wnt canonical and calcium-dependent <span class="highlight" style="background-color:">pathway</span> molecules in microstructured Ti surfaces. Comparison of regulation of canonical (left) and calcium-dependent (right) Wnt <span class="highlight" style="background-color:">signaling</span> pathways in cells grown on PT and modSLA surfaces. Green arrow indicates upregulation and red arrow indicates downregulation of expression on modSLA surfaces in comparison to PT surfaces.</div></p></div><div class="aux"><div class="resc">Rene Olivares-Navarrete, et al. Acta Biomater. ;7(6):2740-2750.</div><p class="links"><a class="dblinks" href="/pubmed/21352958" ref="ordinalpos=2355&amp;ncbi_uid=7048704&amp;link_uid=21352958">Citation</a><a class="dblinks" href="/pmc/articles/PMC4287412/" ref="ordinalpos=2355&amp;ncbi_uid=7048704&amp;link_uid=PMC4287412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2356.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4094471/figure/pone-0101929-g006/" ref="ordinalpos=2356&amp;ncbi_uid=6375453&amp;link_uid=PMC4094471" href="/pmc/articles/PMC4094471/"><img src="/pmc/articles/instance/4094471/bin/pone.0101929.g006.gif" src-large="/pmc/articles/instance/4094471/bin/pone.0101929.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4094471/" ref="ordinalpos=2356&amp;ncbi_uid=6375453&amp;link_uid=PMC4094471" image-link="/pmc/articles/PMC4094471/figure/pone-0101929-g006/" class="imagepopup">Figure 6. Model of DBL-1 <span class="highlight" style="background-color:">pathway</span>-mediated cuticular phenotypes..  From: Regulation of Extracellular Matrix Organization by BMP <span class="highlight" style="background-color:">Signaling</span> in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>Model of how DBL-1 controls organization and composition of the cuticle, which affects body length, permeability barrier function, and worm-star formation in animals with normal (A), increased (B), and decreased (C) DBL-1 <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. Cuticle layers are indicated on the left.</div></p></div><div class="aux"><div class="resc">Robbie D. Schultz, et al. PLoS One. 2014;9(7):e101929.</div><p class="links"><a class="dblinks" href="/pubmed/25013968" ref="ordinalpos=2356&amp;ncbi_uid=6375453&amp;link_uid=25013968">Citation</a><a class="dblinks" href="/pmc/articles/PMC4094471/" ref="ordinalpos=2356&amp;ncbi_uid=6375453&amp;link_uid=PMC4094471">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2357.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847316/figure/fig5/" ref="ordinalpos=2357&amp;ncbi_uid=5558888&amp;link_uid=PMC3847316" href="/pmc/articles/PMC3847316/"><img src="/pmc/articles/instance/3847316/bin/cddis2013438f5.gif" src-large="/pmc/articles/instance/3847316/bin/cddis2013438f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847316/" ref="ordinalpos=2357&amp;ncbi_uid=5558888&amp;link_uid=PMC3847316" image-link="/pmc/articles/PMC3847316/figure/fig5/" class="imagepopup">Figure 5.  From: Dichloroacetate induces protective autophagy in LoVo cells: involvement of cathepsin D/thioredoxin-like protein 1 and Akt-mTOR-mediated <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>DCA inhibited the Akt-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in LoVo cells. Cell lysates from LoVo cells either treated with PBS (control) or 29 mM DCA for 48 h were subjected to western blot analysis. <i>β</i>-Actin was used as a loading control</div></p></div><div class="aux"><div class="resc">F Gong, et al. Cell Death Dis. 2013 November;4(11):e913.</div><p class="links"><a class="dblinks" href="/pubmed/24201812" ref="ordinalpos=2357&amp;ncbi_uid=5558888&amp;link_uid=24201812">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847316/" ref="ordinalpos=2357&amp;ncbi_uid=5558888&amp;link_uid=PMC3847316">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2358.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3648501/figure/F3/" ref="ordinalpos=2358&amp;ncbi_uid=4888500&amp;link_uid=PMC3648501" href="/pmc/articles/PMC3648501/"><img src="/pmc/articles/instance/3648501/bin/1742-4682-10-27-3.gif" src-large="/pmc/articles/instance/3648501/bin/1742-4682-10-27-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3648501/" ref="ordinalpos=2358&amp;ncbi_uid=4888500&amp;link_uid=PMC3648501" image-link="/pmc/articles/PMC3648501/figure/F3/" class="imagepopup">Figure 3.  From: Mathematical models for the Notch and Wnt <span class="highlight" style="background-color:">signaling</span> pathways and the crosstalk between them during somitogenesis. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The diagram was created using CellDesigner. The light green rectangle represents protein; the bottle green parallelogram represents mRNA; the white rectangle that contains a light green rectangle represents a complex; ∅ represents the resultant of a degradation reaction or reactant of a combination reaction. The arrow represents the reaction.</div></p></div><div class="aux"><div class="resc">Hong-yan Wang, et al. Theor Biol Med Model. 2013;10:27-27.</div><p class="links"><a class="dblinks" href="/pubmed/23602012" ref="ordinalpos=2358&amp;ncbi_uid=4888500&amp;link_uid=23602012">Citation</a><a class="dblinks" href="/pmc/articles/PMC3648501/" ref="ordinalpos=2358&amp;ncbi_uid=4888500&amp;link_uid=PMC3648501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2359.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3213172/figure/pcbi-1002264-g003/" ref="ordinalpos=2359&amp;ncbi_uid=3420628&amp;link_uid=PMC3213172" href="/pmc/articles/PMC3213172/"><img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.g003.gif" src-large="/pmc/articles/instance/3213172/bin/pcbi.1002264.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3213172/" ref="ordinalpos=2359&amp;ncbi_uid=3420628&amp;link_uid=PMC3213172" image-link="/pmc/articles/PMC3213172/figure/pcbi-1002264-g003/" class="imagepopup">Figure 3. Network schematics for the coherent and incoherent feedforward loops..  From: Stochastic Delay Accelerates <span class="highlight" style="background-color:">Signaling</span> in Gene Networks. </a></p><div class="supp"><p class="details"><div>Each <span class="highlight" style="background-color:">pathway</span> in the networks has an associated <span class="highlight" style="background-color:">signaling</span> threshold (<img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.e149.jpg" class="inline-graphic" style="border-style: none;" />) and mean delay time (<img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.e150.jpg" class="inline-graphic" style="border-style: none;" />). The random time between the initiation of transcription of gene <img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.e151.jpg" class="inline-graphic" style="border-style: none;" /> to the full formation of a total of <img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.e152.jpg" class="inline-graphic" style="border-style: none;" /> proteins <img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.e153.jpg" class="inline-graphic" style="border-style: none;" /> is denoted <img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.e154.jpg" class="inline-graphic" style="border-style: none;" />, which is an implicit function of <img src="/pmc/articles/instance/3213172/bin/pcbi.1002264.e155.jpg" class="inline-graphic" style="border-style: none;" />.</div></p></div><div class="aux"><div class="resc">Krešimir Josić, et al. PLoS Comput Biol. 2011 November;7(11):e1002264.</div><p class="links"><a class="dblinks" href="/pubmed/22102802" ref="ordinalpos=2359&amp;ncbi_uid=3420628&amp;link_uid=22102802">Citation</a><a class="dblinks" href="/pmc/articles/PMC3213172/" ref="ordinalpos=2359&amp;ncbi_uid=3420628&amp;link_uid=PMC3213172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2360.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4128657/figure/pone-0104202-g004/" ref="ordinalpos=2360&amp;ncbi_uid=6491097&amp;link_uid=PMC4128657" href="/pmc/articles/PMC4128657/"><img src="/pmc/articles/instance/4128657/bin/pone.0104202.g004.gif" src-large="/pmc/articles/instance/4128657/bin/pone.0104202.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4128657/" ref="ordinalpos=2360&amp;ncbi_uid=6491097&amp;link_uid=PMC4128657" image-link="/pmc/articles/PMC4128657/figure/pone-0104202-g004/" class="imagepopup">Figure 4. Genes in the pattern recognition receptor <span class="highlight" style="background-color:">pathway</span> are associated with IFN responsiveness..  From: An Interferon Response Gene Signature Is Associated with the Therapeutic Response of Hepatitis C Patients. </a></p><div class="supp"><p class="details"><div>The gene network was generated using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software. Genes upregulated in nonresponders (shown in Figure 1) were highlighted. As expected, several antiviral <span class="highlight" style="background-color:">signaling</span> pathways were overrepresented in these genes.</div></p></div><div class="aux"><div class="resc">Lawrence M. Pfeffer, et al. PLoS One. 2014;9(8):e104202.</div><p class="links"><a class="dblinks" href="/pubmed/25111807" ref="ordinalpos=2360&amp;ncbi_uid=6491097&amp;link_uid=25111807">Citation</a><a class="dblinks" href="/pmc/articles/PMC4128657/" ref="ordinalpos=2360&amp;ncbi_uid=6491097&amp;link_uid=PMC4128657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2361.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1064894/figure/F3/" ref="ordinalpos=2361&amp;ncbi_uid=524251&amp;link_uid=PMC1064894" href="/pmc/articles/PMC1064894/"><img src="/pmc/articles/instance/1064894/bin/ar1469-3.gif" src-large="/pmc/articles/instance/1064894/bin/ar1469-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1064894/" ref="ordinalpos=2361&amp;ncbi_uid=524251&amp;link_uid=PMC1064894" image-link="/pmc/articles/PMC1064894/figure/F3/" class="imagepopup">Figure 3.  From: Toll-like receptor downstream <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of Toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> pathways. All TLRs except for TLR3 are thought to share the MyD88-dependent <span class="highlight" style="background-color:">pathway</span> that activates NF-κB and mitogen-activated protein (MAP) kinases, leading to the induction of inflammatory cytokine genes. Interleukin-1 receptor-associated kinases (IRAKs) and TRAF6 are located downstream of MyD88. TIRAP is involved in the MyD88-dependent <span class="highlight" style="background-color:">pathway</span> downstream of TLR2 and TLR4. TRIF is utilized in the TLR3-mediated and TLR4-mediated activation of interferon regulatory factor (IRF)3 and the subsequent induction of IRF3-dependent gene expression such as interferon-β (IFN-β). TRAM is specifically involved in the activation of IRF3 in TLR4 <span class="highlight" style="background-color:">signaling</span>. The complex of TBK1/IκB kinase-i (IKK-i) is responsible for the activation of IRF3 downstream of TRIF in TLR3 and TLR4 <span class="highlight" style="background-color:">signaling</span>. TRAF6 is also involved in the TRIF-dependent activation of NF-κB and MAP kinases. Receptor-interacting protein (RIP) mediates TRIF-dependent NF-κB activation. DD, death domain.</div></p></div><div class="aux"><div class="resc">Taro Kawai, et al. Arthritis Res Ther. 2005;7(1):12-19.</div><p class="links"><a class="dblinks" href="/pubmed/15642149" ref="ordinalpos=2361&amp;ncbi_uid=524251&amp;link_uid=15642149">Citation</a><a class="dblinks" href="/pmc/articles/PMC1064894/" ref="ordinalpos=2361&amp;ncbi_uid=524251&amp;link_uid=PMC1064894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2362.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4017424/figure/F0004/" ref="ordinalpos=2362&amp;ncbi_uid=6134656&amp;link_uid=PMC4017424" href="/pmc/articles/PMC4017424/"><img src="/pmc/articles/instance/4017424/bin/GHFBB-4-127-g004.gif" src-large="/pmc/articles/instance/4017424/bin/GHFBB-4-127-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4017424/" ref="ordinalpos=2362&amp;ncbi_uid=6134656&amp;link_uid=PMC4017424" image-link="/pmc/articles/PMC4017424/figure/F0004/" class="imagepopup">Figure 4.  From: Colorectal cancer and NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Activation of IKK <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Parichehr Hassanzadeh. Gastroenterol Hepatol Bed Bench. 2011 Summer;4(3):127-132.</div><p class="links"><a class="dblinks" href="/pubmed/24834170" ref="ordinalpos=2362&amp;ncbi_uid=6134656&amp;link_uid=24834170">Citation</a><a class="dblinks" href="/pmc/articles/PMC4017424/" ref="ordinalpos=2362&amp;ncbi_uid=6134656&amp;link_uid=PMC4017424">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2363.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4290619/figure/F4/" ref="ordinalpos=2363&amp;ncbi_uid=7065969&amp;link_uid=PMC4290619" href="/pmc/articles/PMC4290619/"><img src="/pmc/articles/instance/4290619/bin/1471-2164-15-S9-S6-4.gif" src-large="/pmc/articles/instance/4290619/bin/1471-2164-15-S9-S6-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4290619/" ref="ordinalpos=2363&amp;ncbi_uid=7065969&amp;link_uid=PMC4290619" image-link="/pmc/articles/PMC4290619/figure/F4/" class="imagepopup">Figure 4.  From: Transcriptome alterations of mitochondrial and coagulation function in schizophrenia by cortical sequencing analysis. </a></p><div class="supp"><p class="details"><div><b>The crosstalk of PID candidate pathways from over- and under-expressed candidate genes for schizophrenia</b>. The enrichment pathways with over- and under-expressed candidate genes indicated the <span class="highlight" style="background-color:">pathway</span> involving biological function including transcription activity, <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, cancer-related <span class="highlight" style="background-color:">pathway</span>, tumor suppression, coagulation, insulin secretion, cell cycle, cell differentiation and apoptosis.</div></p></div><div class="aux"><div class="resc">Kuo-Chuan Huang, et al. BMC Genomics. 2014;15(Suppl 9):S6-S6.</div><p class="links"><a class="dblinks" href="/pubmed/25522158" ref="ordinalpos=2363&amp;ncbi_uid=7065969&amp;link_uid=25522158">Citation</a><a class="dblinks" href="/pmc/articles/PMC4290619/" ref="ordinalpos=2363&amp;ncbi_uid=7065969&amp;link_uid=PMC4290619">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2364.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4261972/figure/f05/" ref="ordinalpos=2364&amp;ncbi_uid=6951343&amp;link_uid=PMC4261972" href="/pmc/articles/PMC4261972/"><img src="/pmc/articles/instance/4261972/bin/gmb-37-716-g005.gif" src-large="/pmc/articles/instance/4261972/bin/gmb-37-716-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4261972/" ref="ordinalpos=2364&amp;ncbi_uid=6951343&amp;link_uid=PMC4261972" image-link="/pmc/articles/PMC4261972/figure/f05/" class="imagepopup">Figure 5.  From: PCI-24781 down-regulates EZH2 expression and then promotes glioma apoptosis by suppressing the PIK3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>EZH2 gene silencing attenuated the expression of proteins involved in the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. M – blank control (mock), NC – normal control (cells with scrambled siRNA), SIR – cells treated with specific siRNA. The blots are representative of 3 experiments.</div></p></div><div class="aux"><div class="resc">Wei Zhang, et al. Genet Mol Biol. 2014 October;37(4):716-724.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2364&amp;ncbi_uid=6951343&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4261972/" ref="ordinalpos=2364&amp;ncbi_uid=6951343&amp;link_uid=PMC4261972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2365.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3647219/figure/F1/" ref="ordinalpos=2365&amp;ncbi_uid=4880208&amp;link_uid=PMC3647219" href="/pmc/articles/PMC3647219/"><img src="/pmc/articles/instance/3647219/bin/fonc-03-00108-g001.gif" src-large="/pmc/articles/instance/3647219/bin/fonc-03-00108-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3647219/" ref="ordinalpos=2365&amp;ncbi_uid=4880208&amp;link_uid=PMC3647219" image-link="/pmc/articles/PMC3647219/figure/F1/" class="imagepopup">Figure 1.  From: Targeting PI3K in Cancer: Any Good News?. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of PI3K <span class="highlight" style="background-color:">pathway</span> and sites of action of PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors in solid tumors and hematological malignancies</b>.</div></p></div><div class="aux"><div class="resc">Miriam Martini, et al. Front Oncol. 2013;3:108.</div><p class="links"><a class="dblinks" href="/pubmed/23658859" ref="ordinalpos=2365&amp;ncbi_uid=4880208&amp;link_uid=23658859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3647219/" ref="ordinalpos=2365&amp;ncbi_uid=4880208&amp;link_uid=PMC3647219">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2366.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2962631/figure/pone-0013558-g004/" ref="ordinalpos=2366&amp;ncbi_uid=2555409&amp;link_uid=PMC2962631" href="/pmc/articles/PMC2962631/"><img src="/pmc/articles/instance/2962631/bin/pone.0013558.g004.gif" src-large="/pmc/articles/instance/2962631/bin/pone.0013558.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2962631/" ref="ordinalpos=2366&amp;ncbi_uid=2555409&amp;link_uid=PMC2962631" image-link="/pmc/articles/PMC2962631/figure/pone-0013558-g004/" class="imagepopup">Figure 4. miR-20b targeting several repressive genes in the BMPs/Runx2 <span class="highlight" style="background-color:">pathway</span> induces miR-20b-mediated osteogenesis in hMSCs..  From: miRNA-Mediated Functional Changes through Co-Regulating Function Related Genes. </a></p><div class="supp"><p class="details"><div>(A) miR-20b enhanced osteogenesis of hMSCs by repressing the expression of inhibitors of the BMP/Runx2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in different levels. (B) Comparison of ARS staining between miR-20b-transfected hMSCs and siRNA-transfected hMSCs (designed to target miR-20b's target genes, PPARr, Bambi and Crami). (C) Des-staining and OD measurement of ARS staining samples.</div></p></div><div class="aux"><div class="resc">Jie He, et al. PLoS One. 2010;5(10):e13558.</div><p class="links"><a class="dblinks" href="/pubmed/21042576" ref="ordinalpos=2366&amp;ncbi_uid=2555409&amp;link_uid=21042576">Citation</a><a class="dblinks" href="/pmc/articles/PMC2962631/" ref="ordinalpos=2366&amp;ncbi_uid=2555409&amp;link_uid=PMC2962631">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2367.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3293036/figure/F1/" ref="ordinalpos=2367&amp;ncbi_uid=3839384&amp;link_uid=PMC3293036" href="/pmc/articles/PMC3293036/"><img src="/pmc/articles/instance/3293036/bin/1476-4598-11-7-1.gif" src-large="/pmc/articles/instance/3293036/bin/1476-4598-11-7-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3293036/" ref="ordinalpos=2367&amp;ncbi_uid=3839384&amp;link_uid=PMC3293036" image-link="/pmc/articles/PMC3293036/figure/F1/" class="imagepopup">Figure 1.  From: Wnt <span class="highlight" style="background-color:">signaling</span> pathways in urological cancers: past decades and still growing. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancer cells</b>. In the presence of active Wnt, β-catenin accumulates in the cytoplasm, then localizes to the nucleus, and activates transcription together with TCF/LEF transcription factors. Negative regulators are depicted in red and positive regulators in green. Activation of the mTOR <span class="highlight" style="background-color:">pathway</span> is also directly regulated by Wnt-dependent downregulation of GSK3 kinase activity which is depicted in yellow.</div></p></div><div class="aux"><div class="resc">Shahana Majid, et al. Mol Cancer. 2012;11:7-7.</div><p class="links"><a class="dblinks" href="/pubmed/22325146" ref="ordinalpos=2367&amp;ncbi_uid=3839384&amp;link_uid=22325146">Citation</a><a class="dblinks" href="/pmc/articles/PMC3293036/" ref="ordinalpos=2367&amp;ncbi_uid=3839384&amp;link_uid=PMC3293036">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2368.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2698645/figure/f2/" ref="ordinalpos=2368&amp;ncbi_uid=1852663&amp;link_uid=PMC2698645" href="/pmc/articles/PMC2698645/"><img src="/pmc/articles/instance/2698645/bin/zdg0040984510002.gif" src-large="/pmc/articles/instance/2698645/bin/zdg0040984510002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2698645/" ref="ordinalpos=2368&amp;ncbi_uid=1852663&amp;link_uid=PMC2698645" image-link="/pmc/articles/PMC2698645/figure/f2/" class="imagepopup">Fig. 2. From: Nutritional and contractile regulation of human skeletal muscle protein synthesis and mTORC1 <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>A simplified schematic representation of the interaction between the mTORC1 and ERK1/2 <span class="highlight" style="background-color:">signaling</span> pathways following muscle contraction and/or insulin stimulation. mTORC1 <span class="highlight" style="background-color:">pathway</span> is indicated in blue, while the ERK1/2 <span class="highlight" style="background-color:">pathway</span> is in purple. MEK1/2, mitogen-activated protein kinase kinase-1/2; ERK1/2, extracellular signal-regulated kinase 1/2; MNK1, MAP kinase-interacting kinase 1; PI3K, phosphatidylinositol 3-kinase; RSK1, p90 ribosomal protein S6 kinase.</div></p></div><div class="aux"><div class="resc">Micah J. Drummond, et al. J Appl Physiol (1985). 2009 April;106(4):1374-1384.</div><p class="links"><a class="dblinks" href="/pubmed/19150856" ref="ordinalpos=2368&amp;ncbi_uid=1852663&amp;link_uid=19150856">Citation</a><a class="dblinks" href="/pmc/articles/PMC2698645/" ref="ordinalpos=2368&amp;ncbi_uid=1852663&amp;link_uid=PMC2698645">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2369.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3905307/figure/F1/" ref="ordinalpos=2369&amp;ncbi_uid=6447034&amp;link_uid=PMC3905307" href="/pmc/articles/PMC3905307/"><img src="/pmc/articles/instance/3905307/bin/nihms491378f1.gif" src-large="/pmc/articles/instance/3905307/bin/nihms491378f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3905307/" ref="ordinalpos=2369&amp;ncbi_uid=6447034&amp;link_uid=PMC3905307" image-link="/pmc/articles/PMC3905307/figure/F1/" class="imagepopup">Figure 1.  From: Rational Combination of a MEK Inhibitor, Selumetinib, and the Wnt/Calcium <span class="highlight" style="background-color:">Pathway</span> Modulator, Cyclosporin A, in Preclinical Models of Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>Canonical <b>(A)</b> and noncanonical <b>(B)</b> Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> represented by KEGG analysis. Red ovals represent core genes in the Wnt <span class="highlight" style="background-color:">pathway</span> with increased expression in CRC cells resistant to selumetinib. Purple and pink ovals represent synthetically lethal genes for SW480 and RKO CRC cell lines respectively. Blue and green ovals represent top-scoring genes in both GSEA and SLS analyses for SW480 and RKO respectively.</div></p></div><div class="aux"><div class="resc">Anna Spreafico, et al. Clin Cancer Res. ;19(15):4149-4162.</div><p class="links"><a class="dblinks" href="/pubmed/23757356" ref="ordinalpos=2369&amp;ncbi_uid=6447034&amp;link_uid=23757356">Citation</a><a class="dblinks" href="/pmc/articles/PMC3905307/" ref="ordinalpos=2369&amp;ncbi_uid=6447034&amp;link_uid=PMC3905307">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2370.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711309/figure/F6/" ref="ordinalpos=2370&amp;ncbi_uid=6365108&amp;link_uid=PMC3711309" href="/pmc/articles/PMC3711309/"><img src="/pmc/articles/instance/3711309/bin/zbc0321355500006.gif" src-large="/pmc/articles/instance/3711309/bin/zbc0321355500006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711309/" ref="ordinalpos=2370&amp;ncbi_uid=6365108&amp;link_uid=PMC3711309" image-link="/pmc/articles/PMC3711309/figure/F6/" class="imagepopup">FIGURE 6. From: TLR-TRIF <span class="highlight" style="background-color:">Pathway</span> Enhances the Expression of KSHV Replication and Transcription Activator. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the TRIF and RTA interaction.</b> Upon KSHV infection of a permissive cell, the TLR3 <span class="highlight" style="background-color:">pathway</span> is activated, and RTA is also expressed. RTA is able to degrade the critical adaptor molecule, TRIF, for TLR3 <span class="highlight" style="background-color:">signaling</span>, and consequently blocks TLR3-mediated innate immunity directed against KSHV. At the same time, the virus is able to use the TLR3-TRIF <span class="highlight" style="background-color:">pathway</span> to enhance the expression of RTA, and to expedite the degradation process of TRIF, and furthermore to block the TLR3-mediated inhibitory effects on KSHV replication.</div></p></div><div class="aux"><div class="resc">Florencia Meyer, et al. J Biol Chem. 2013 July 12;288(28):20435-20442.</div><p class="links"><a class="dblinks" href="/pubmed/23723066" ref="ordinalpos=2370&amp;ncbi_uid=6365108&amp;link_uid=23723066">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711309/" ref="ordinalpos=2370&amp;ncbi_uid=6365108&amp;link_uid=PMC3711309">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2371.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3494369/figure/fig1-1947601912455324/" ref="ordinalpos=2371&amp;ncbi_uid=4365164&amp;link_uid=PMC3494369" href="/pmc/articles/PMC3494369/"><img src="/pmc/articles/instance/3494369/bin/10.1177_1947601912455324-fig1.gif" src-large="/pmc/articles/instance/3494369/bin/10.1177_1947601912455324-fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3494369/" ref="ordinalpos=2371&amp;ncbi_uid=4365164&amp;link_uid=PMC3494369" image-link="/pmc/articles/PMC3494369/figure/fig1-1947601912455324/" class="imagepopup">Figure 1. From: Using Mouse Models to Explore MDM-p53 <span class="highlight" style="background-color:">Signaling</span> in Development, Cell Growth, and Tumorigenesis. </a></p><div class="supp"><p class="details"><div>The Mdm2-MdmX-p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>Mdm2</i> and <i>MdmX</i> expression is upregulated by p53 due to the presence of intragenic p53 promoter response elements (triangles). The subsequent increase in MDM proteins results in the inhibition of p53 functions due to p53 and Mdm2/MdmX complex formation.</div></p></div><div class="aux"><div class="resc">Hugh S. Gannon, et al. Genes Cancer. 2012 March;3(3-4):209-218.</div><p class="links"><a class="dblinks" href="/pubmed/23150754" ref="ordinalpos=2371&amp;ncbi_uid=4365164&amp;link_uid=23150754">Citation</a><a class="dblinks" href="/pmc/articles/PMC3494369/" ref="ordinalpos=2371&amp;ncbi_uid=4365164&amp;link_uid=PMC3494369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2372.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2885864/figure/F1/" ref="ordinalpos=2372&amp;ncbi_uid=2312601&amp;link_uid=PMC2885864" href="/pmc/articles/PMC2885864/"><img src="/pmc/articles/instance/2885864/bin/eplasty10e42_fig1.gif" src-large="/pmc/articles/instance/2885864/bin/eplasty10e42_fig1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2885864/" ref="ordinalpos=2372&amp;ncbi_uid=2312601&amp;link_uid=PMC2885864" image-link="/pmc/articles/PMC2885864/figure/F1/" class="imagepopup">Figure 1.  From: Nitric Oxide <span class="highlight" style="background-color:">Signaling</span> and Neural Stem Cell Differentiation in Peripheral Nerve Regeneration. </a></p><div class="supp"><p class="details"><div>Nitric oxide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. NO indicates nitric oxide; sGC, soluble guanylate cyclase; GTP, Guanine triphosphate; cGMP, cyclic Guanine monophosphate; PDE, phosphodiesterase; PKG, protein kinase G.</div></p></div><div class="aux"><div class="resc">Jessica Tao Li, et al. Eplasty. 2010;10:e42.</div><p class="links"><a class="dblinks" href="/pubmed/20563304" ref="ordinalpos=2372&amp;ncbi_uid=2312601&amp;link_uid=20563304">Citation</a><a class="dblinks" href="/pmc/articles/PMC2885864/" ref="ordinalpos=2372&amp;ncbi_uid=2312601&amp;link_uid=PMC2885864">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2373.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC24103/figure/F5/" ref="ordinalpos=2373&amp;ncbi_uid=154838&amp;link_uid=PMC24103" href="/pmc/articles/PMC24103/"><img src="/pmc/articles/instance/24103/bin/pq2262603005.gif" src-large="/pmc/articles/instance/24103/bin/pq2262603005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC24103/" ref="ordinalpos=2373&amp;ncbi_uid=154838&amp;link_uid=PMC24103" image-link="/pmc/articles/PMC24103/figure/F5/" class="imagepopup">Figure 5.  From: Convergent pathways for lipochitooligosaccharide and auxin  <span class="highlight" style="background-color:">signaling</span> in tobacco cells. </a></p><div class="supp"><p class="details"><div>Schematic interpretation of the data. Induction  of <i>AXI1</i> by LCOs involves a <span class="highlight" style="background-color:">signaling</span> route independent  from the auxin <span class="highlight" style="background-color:">pathway</span>. LCOs and auxin require the presence of  cytokinin to stimulate maximal <i>AXI</i> transcription that  correlates with cell division. The exogenously applied concentrations  of growth factors are indicated.</div></p></div><div class="aux"><div class="resc">Horst Röhrig, et al. Proc Natl Acad Sci U S A. 1996 November 12;93(23):13389-13392.</div><p class="links"><a class="dblinks" href="/pubmed/11038524" ref="ordinalpos=2373&amp;ncbi_uid=154838&amp;link_uid=11038524">Citation</a><a class="dblinks" href="/pmc/articles/PMC24103/" ref="ordinalpos=2373&amp;ncbi_uid=154838&amp;link_uid=PMC24103">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2374.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4377161/figure/F3/" ref="ordinalpos=2374&amp;ncbi_uid=7339430&amp;link_uid=PMC4377161" href="/pmc/articles/PMC4377161/"><img src="/pmc/articles/instance/4377161/bin/nihms672512f3.gif" src-large="/pmc/articles/instance/4377161/bin/nihms672512f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4377161/" ref="ordinalpos=2374&amp;ncbi_uid=7339430&amp;link_uid=PMC4377161" image-link="/pmc/articles/PMC4377161/figure/F3/" class="imagepopup">Figure 3. Leelamine inhibits three key <span class="highlight" style="background-color:">signaling</span> pathways regulating melanoma development.  From: Targeting Multiple Key <span class="highlight" style="background-color:">Signaling</span> Pathways in Melanoma using Leelamine. </a></p><div class="supp"><p class="details"><div><b>3A, 3B and 3C.</b> Treatment with 3 to 6 µmol/L of leelamine for 3 to 24 hours decreased PI3K/Akt (<b>3A</b>), MAPK (<b>3B</b>) and STAT3 pathways (<b>3C</b>). <span class="highlight" style="background-color:">Signaling</span> inhibition was observed for both UACC 903 and 1205 Lu cell lines, with inhibition of PI3 and MAP kinase pathways occurring at 3 to 6 hours while inhibition of the STAT3 <span class="highlight" style="background-color:">pathway</span> occurred from 12 hours of treatment. Alpha-enolase served as a control for equal protein loading.</div></p></div><div class="aux"><div class="resc">Raghavendra Gowda, et al. Mol Cancer Ther. ;13(7):1679-1689.</div><p class="links"><a class="dblinks" href="/pubmed/24688050" ref="ordinalpos=2374&amp;ncbi_uid=7339430&amp;link_uid=24688050">Citation</a><a class="dblinks" href="/pmc/articles/PMC4377161/" ref="ordinalpos=2374&amp;ncbi_uid=7339430&amp;link_uid=PMC4377161">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2375.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3841341/figure/F2/" ref="ordinalpos=2375&amp;ncbi_uid=5523855&amp;link_uid=PMC3841341" href="/pmc/articles/PMC3841341/"><img src="/pmc/articles/instance/3841341/bin/nihms394192f2.gif" src-large="/pmc/articles/instance/3841341/bin/nihms394192f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3841341/" ref="ordinalpos=2375&amp;ncbi_uid=5523855&amp;link_uid=PMC3841341" image-link="/pmc/articles/PMC3841341/figure/F2/" class="imagepopup">Figure 2.  From: Bioprospecting the Bibleome: Adding Evidence to Support the Inflammatory Basis of Cancer. </a></p><div class="supp"><p class="details"><div>This figure depicts the connections between the top 200 novel cancer genes and the Acute Phase Response <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>. Nine of the genes that have metabolic functions associated with multiple cancers are found in this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Peter L. Elkin, et al. Metabolomics (Los Angel). ;2(112):6451.</div><p class="links"><a class="dblinks" href="/pubmed/24294537" ref="ordinalpos=2375&amp;ncbi_uid=5523855&amp;link_uid=24294537">Citation</a><a class="dblinks" href="/pmc/articles/PMC3841341/" ref="ordinalpos=2375&amp;ncbi_uid=5523855&amp;link_uid=PMC3841341">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2376.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC99013/figure/F16/" ref="ordinalpos=2376&amp;ncbi_uid=52540&amp;link_uid=PMC99013" href="/pmc/articles/PMC99013/"><img src="/pmc/articles/instance/99013/bin/mr0400030016.gif" src-large="/pmc/articles/instance/99013/bin/mr0400030016.jpg" alt="FIG. 16" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC99013/" ref="ordinalpos=2376&amp;ncbi_uid=52540&amp;link_uid=PMC99013" image-link="/pmc/articles/PMC99013/figure/F16/" class="imagepopup">FIG. 16.  From: Signal Transduction Cascades Regulating Fungal  Development and Virulence. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways regulating virulence of the rice blast fungus <i>M. grisea</i>. Appressorium formation and virulence of the rice blast fungus <i>M. grisea</i> involve both conserved MAP kinase <span class="highlight" style="background-color:">signaling</span> cascades and the cAMP-PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Klaus B. Lengeler, et al. Microbiol Mol Biol Rev. 2000 December;64(4):746-785.</div><p class="links"><a class="dblinks" href="/pubmed/11104818" ref="ordinalpos=2376&amp;ncbi_uid=52540&amp;link_uid=11104818">Citation</a><a class="dblinks" href="/pmc/articles/PMC99013/" ref="ordinalpos=2376&amp;ncbi_uid=52540&amp;link_uid=PMC99013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2377.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2581770/figure/F3/" ref="ordinalpos=2377&amp;ncbi_uid=1574618&amp;link_uid=PMC2581770" href="/pmc/articles/PMC2581770/"><img src="/pmc/articles/instance/2581770/bin/nihms57617f3.gif" src-large="/pmc/articles/instance/2581770/bin/nihms57617f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2581770/" ref="ordinalpos=2377&amp;ncbi_uid=1574618&amp;link_uid=PMC2581770" image-link="/pmc/articles/PMC2581770/figure/F3/" class="imagepopup">Figure 3.  From: The TNF receptor and Ig superfamily members form an integrated <span class="highlight" style="background-color:">signaling</span> circuit controlling dendritic cell homeostasis. </a></p><div class="supp"><p class="details"><div>TNFR and LTβR <span class="highlight" style="background-color:">signaling</span> pathways for NFκB activation. <span class="highlight" style="background-color:">Signaling</span> through TNFR activates the canonical RelA NFκB <span class="highlight" style="background-color:">pathway</span>, which involves the α, β and γ (NEMO) subunits of the IκB kinase (IKK) complex, which leads to proteosomal degradation of IκB, releasing RelA/p50 dimer from its cytoplasmic constraint, locate to the nucleus and binding κB transcriptional target sequences. LTβR ligation activates the serine kinases NIK and IKKα (also a part of the canonical <span class="highlight" style="background-color:">pathway</span>). The phosphorylation and ubiquitin-dependent proteasomal degradation of p100 results in the processing of the precursor to the p52 subunit and accumulation of p52-RelB heterodimers to the nucleus. <span class="highlight" style="background-color:">Signaling</span> through the LTβR also leads to the activation of the classical NF-κB <span class="highlight" style="background-color:">pathway</span>. TNFR <span class="highlight" style="background-color:">signaling</span> rapidly activates a large number of proinflammatory genes, whereas LTβR activation of RelB takes several hours but sustained expression in the nucleus leads to different sets of activated genes involved in homeostasis, such as the chemokine, CCL21.</div></p></div><div class="aux"><div class="resc">Carl De Trez, et al. Cytokine Growth Factor Rev. ;19(3-4):277-284.</div><p class="links"><a class="dblinks" href="/pubmed/18511331" ref="ordinalpos=2377&amp;ncbi_uid=1574618&amp;link_uid=18511331">Citation</a><a class="dblinks" href="/pmc/articles/PMC2581770/" ref="ordinalpos=2377&amp;ncbi_uid=1574618&amp;link_uid=PMC2581770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2378.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3079764/figure/F1/" ref="ordinalpos=2378&amp;ncbi_uid=3393317&amp;link_uid=PMC3079764" href="/pmc/articles/PMC3079764/"><img src="/pmc/articles/instance/3079764/bin/nihms275650f1.gif" src-large="/pmc/articles/instance/3079764/bin/nihms275650f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3079764/" ref="ordinalpos=2378&amp;ncbi_uid=3393317&amp;link_uid=PMC3079764" image-link="/pmc/articles/PMC3079764/figure/F1/" class="imagepopup">FIGURE 1.  From: Transforming Growth Factor-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activation in Keratoconus. </a></p><div class="supp"><p class="details"><div>Immunohistochemistry for transforming growth factor-β (TGFβ) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> markers. TGFβ2, pSmad2, and TGFβ (total) were assessed in normal human corneas and corneas with keratoconus. Horseradish peroxidase (HRP)-positive signal was quantified as described in Methods. Error bars indicate standard deviation. *<i>P</i> &lt; .05; **<i>P</i> &lt; .01; ***<i>P</i> &lt; .001; <sup>+</sup><i>P</i> ≥ .05; Mann–Whitney <i>U</i> test. Original magnification ×200. (A and B): Immunohistochemistry for TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> markers (TGFβ2 and phosphorylated Smad2 [pSmad2]) in (Top left) normal human corneas and (Top right) in patients with mild and (Bottom left) severe keratoconus. (Bottom right) Signal quantification. (C): Immunohistochemistry results for TGFβ (total) in (Left) normal human corneas and (Middle) patients with severe keratoconus. (Right) Signal quantification.</div></p></div><div class="aux"><div class="resc">CHRISTOPH ENGLER, et al. Am J Ophthalmol. ;151(5):752-759.e2.</div><p class="links"><a class="dblinks" href="/pubmed/21310385" ref="ordinalpos=2378&amp;ncbi_uid=3393317&amp;link_uid=21310385">Citation</a><a class="dblinks" href="/pmc/articles/PMC3079764/" ref="ordinalpos=2378&amp;ncbi_uid=3393317&amp;link_uid=PMC3079764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2379.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2982790/figure/F9/" ref="ordinalpos=2379&amp;ncbi_uid=2578266&amp;link_uid=PMC2982790" href="/pmc/articles/PMC2982790/"><img src="/pmc/articles/instance/2982790/bin/nihms247427f9.gif" src-large="/pmc/articles/instance/2982790/bin/nihms247427f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2982790/" ref="ordinalpos=2379&amp;ncbi_uid=2578266&amp;link_uid=PMC2982790" image-link="/pmc/articles/PMC2982790/figure/F9/" class="imagepopup">Fig. 9.  From: Akt-dependent NF-?B activation is required for bile acids to rescue colon cancer cells from stress-induced apoptosis. </a></p><div class="supp"><p class="details"><div>Model of deoxycholyltaurine-induced post-EGFR <span class="highlight" style="background-color:">signaling</span> in human colon cancer cells. <span class="highlight" style="background-color:">Signaling</span> via the MEK/ERK <span class="highlight" style="background-color:">pathway</span> stimulates cell proliferation. <span class="highlight" style="background-color:">Signaling</span> via the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> stimulates nuclear translocation and transcriptional activity of NF-κB, thereby protecting cells from stress-induced apoptosis. Dashed arrow between deoxycholyltaurine and EGFR indicates a mechanism involving release of the EGFR ligand HB-EGF [8,9]. Circled P indicates key phosphorylations.</div></p></div><div class="aux"><div class="resc">Jasleen Shant, et al. Exp Cell Res. ;315(3):432-450.</div><p class="links"><a class="dblinks" href="/pubmed/19056378" ref="ordinalpos=2379&amp;ncbi_uid=2578266&amp;link_uid=19056378">Citation</a><a class="dblinks" href="/pmc/articles/PMC2982790/" ref="ordinalpos=2379&amp;ncbi_uid=2578266&amp;link_uid=PMC2982790">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2380.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2774304/figure/F9/" ref="ordinalpos=2380&amp;ncbi_uid=2084596&amp;link_uid=PMC2774304" href="/pmc/articles/PMC2774304/"><img src="/pmc/articles/instance/2774304/bin/1742-2094-6-30-9.gif" src-large="/pmc/articles/instance/2774304/bin/1742-2094-6-30-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2774304/" ref="ordinalpos=2380&amp;ncbi_uid=2084596&amp;link_uid=PMC2774304" image-link="/pmc/articles/PMC2774304/figure/F9/" class="imagepopup">Figure 9.  From: Estrogen treatment following severe burn injury reduces brain inflammation and apoptotic <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Proposed model of estrogen's anti-inflammatory and pro-survival actions in the brain following severe burn injury</b>. In this study, we propose that following severe burn injury, acute estrogen treatment increases the activity of the anti-apoptotic Akt <span class="highlight" style="background-color:">signaling</span> protein and decreases the activity of pro-apoptotic factors such as caspase-3. Additionally, estrogen decreases PARP cleavage and ultimately decrease the levels of apoptosis (<span class="highlight" style="background-color:">Pathway</span> 1). Following injury, estrogen increases the activity of ERK through a parallel <span class="highlight" style="background-color:">pathway</span>, and subsequently decreases the levels of inflammatory cytokines (<span class="highlight" style="background-color:">Pathway</span> 2).</div></p></div><div class="aux"><div class="resc">Joshua W Gatson, et al. J Neuroinflammation. 2009;6:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/19849845" ref="ordinalpos=2380&amp;ncbi_uid=2084596&amp;link_uid=19849845">Citation</a><a class="dblinks" href="/pmc/articles/PMC2774304/" ref="ordinalpos=2380&amp;ncbi_uid=2084596&amp;link_uid=PMC2774304">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2381.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3388731/figure/fig05/" ref="ordinalpos=2381&amp;ncbi_uid=4103266&amp;link_uid=PMC3388731" href="/pmc/articles/PMC3388731/"><img src="/pmc/articles/instance/3388731/bin/ijbmb0003-0117-f5.gif" src-large="/pmc/articles/instance/3388731/bin/ijbmb0003-0117-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3388731/" ref="ordinalpos=2381&amp;ncbi_uid=4103266&amp;link_uid=PMC3388731" image-link="/pmc/articles/PMC3388731/figure/fig05/" class="imagepopup">Figure 5.  From: Epithelial-mesenchymal transition, the tumor microenvironment, and metastatic behavior of epithelial malignancies. </a></p><div class="supp"><p class="details"><div>Canonical Hedgehog <span class="highlight" style="background-color:">signaling</span>. Reproduced with permission from Prasad CP, The Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Breast Cancer. CML Breast Cancer 2012; 22.</div></p></div><div class="aux"><div class="resc">Lindsay J Talbot, et al. Int J Biochem Mol Biol. 2012;3(2):117-136.</div><p class="links"><a class="dblinks" href="/pubmed/22773954" ref="ordinalpos=2381&amp;ncbi_uid=4103266&amp;link_uid=22773954">Citation</a><a class="dblinks" href="/pmc/articles/PMC3388731/" ref="ordinalpos=2381&amp;ncbi_uid=4103266&amp;link_uid=PMC3388731">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2382.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4145387/figure/fig2/" ref="ordinalpos=2382&amp;ncbi_uid=6552030&amp;link_uid=PMC4145387" href="/pmc/articles/PMC4145387/"><img src="/pmc/articles/instance/4145387/bin/BMRI2014-637308.002.gif" src-large="/pmc/articles/instance/4145387/bin/BMRI2014-637308.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4145387/" ref="ordinalpos=2382&amp;ncbi_uid=6552030&amp;link_uid=PMC4145387" image-link="/pmc/articles/PMC4145387/figure/fig2/" class="imagepopup">Figure 2.  From: E-Cadherin and Gastric Cancer: Cause, Consequence, and Applications. </a></p><div class="supp"><p class="details"><div>E-cadherin regulated <span class="highlight" style="background-color:">signaling</span> pathways involved in gastric cancer, including Wnt/<i>β</i>-catenin <span class="highlight" style="background-color:">pathway</span>, Rho GTPases, NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span>, EGFR, and Rac-MAPK <span class="highlight" style="background-color:">signaling</span>. Activation of these pathways leads to increase in cell proliferation, decrease in cell apoptosis, cell migration, and inflammation associated gastric cancer development.</div></p></div><div class="aux"><div class="resc">Xin Liu, et al. Biomed Res Int. 2014;2014:637308.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2382&amp;ncbi_uid=6552030&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4145387/" ref="ordinalpos=2382&amp;ncbi_uid=6552030&amp;link_uid=PMC4145387">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2383.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2600414/figure/F1/" ref="ordinalpos=2383&amp;ncbi_uid=1562479&amp;link_uid=PMC2600414" href="/pmc/articles/PMC2600414/"><img src="/pmc/articles/instance/2600414/bin/nihms-81117-f0001.gif" src-large="/pmc/articles/instance/2600414/bin/nihms-81117-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2600414/" ref="ordinalpos=2383&amp;ncbi_uid=1562479&amp;link_uid=PMC2600414" image-link="/pmc/articles/PMC2600414/figure/F1/" class="imagepopup">Figure 1. PTEN loss disrupts the circuitry controlling oncogenic cell growth in T-ALL downstream of NOTCH1 in T-ALL.  From: The role of the PTEN/AKT <span class="highlight" style="background-color:">Pathway</span> in NOTCH1-induced leukemia. </a></p><div class="supp"><p class="details"><div><b>A.</b> NOTCH1 controls leukemic cell growth through multiple overlapping mechanisms including activation of target genes, upregulation of <i>MYC</i> and downregulation of <i>PTEN</i> via HES1. <b>B.</b> Mutational loss of <i>PTEN</i> induces constitutive activation of AKT and uncouples NOTCH1 <span class="highlight" style="background-color:">signaling</span> from the PI3K-AKT <span class="highlight" style="background-color:">pathway</span>. Thus, PTEN-null T-ALL cells become insensitive to inhibition of NOTCH1 <span class="highlight" style="background-color:">signaling</span> with GSIs.</div></p></div><div class="aux"><div class="resc">Teresa Palomero, et al. Cell Cycle. ;7(8):965-970.</div><p class="links"><a class="dblinks" href="/pubmed/18414037" ref="ordinalpos=2383&amp;ncbi_uid=1562479&amp;link_uid=18414037">Citation</a><a class="dblinks" href="/pmc/articles/PMC2600414/" ref="ordinalpos=2383&amp;ncbi_uid=1562479&amp;link_uid=PMC2600414">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2384.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2582834/figure/F7/" ref="ordinalpos=2384&amp;ncbi_uid=1588296&amp;link_uid=PMC2582834" href="/pmc/articles/PMC2582834/"><img src="/pmc/articles/instance/2582834/bin/nihms59759f7a.gif" src-large="/pmc/articles/instance/2582834/bin/nihms59759f7a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2582834/figure/F7/" ref="ordinalpos=2384&amp;ncbi_uid=1588296&amp;link_uid=PMC2582834" href="/pmc/articles/PMC2582834/"><img src="/pmc/articles/instance/2582834/bin/nihms59759f7b.gif" src-large="/pmc/articles/instance/2582834/bin/nihms59759f7b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2582834/" ref="ordinalpos=2384&amp;ncbi_uid=1588296&amp;link_uid=PMC2582834" image-link="/pmc/articles/PMC2582834/figure/F7/" class="imagepopup">Figure 7.  From: Killing of Human Melanoma Cells Induced by Activation of Class I Interferon-Regulated <span class="highlight" style="background-color:">Signaling</span> Pathways via MDA-7/IL-24. </a></p><div class="supp"><p class="details"><div>MDA7/IL-24 differentially regulates genes on <span class="highlight" style="background-color:">signaling</span> and apoptosis. <b>A,</b> MeWo melanoma cells were exposed to either a dose titration of affinity-purified MDA7/IL-24 or cocultured with MDA7/IL-24–transfected HEK-293 cells. After 48 h of treatment, cells were assessed for their apoptotic and <span class="highlight" style="background-color:">signaling</span> pathways in gene regulation with commercially available gene array analysis. <b>B,</b> MeWo melanoma cells were exposed to either a dose titration of affinity-purified MDA7/IL-24 or cocultured with MDA7/IL-24–transfected HEK-293 cells. After 48 h of treatment, lysates were taken and proteins were precipitated and washed in 1× PBS, resolved by SDS-PAGE, transferred to nitrocellulose, and Western blotted for apoptotic and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> markers</div></p></div><div class="aux"><div class="resc">Suhendan Ekmekcioglu, et al. Cytokine. ;43(1):34-44.</div><p class="links"><a class="dblinks" href="/pubmed/18511292" ref="ordinalpos=2384&amp;ncbi_uid=1588296&amp;link_uid=18511292">Citation</a><a class="dblinks" href="/pmc/articles/PMC2582834/" ref="ordinalpos=2384&amp;ncbi_uid=1588296&amp;link_uid=PMC2582834">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2385.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3334867/figure/F5/" ref="ordinalpos=2385&amp;ncbi_uid=4815633&amp;link_uid=PMC3334867" href="/pmc/articles/PMC3334867/"><img src="/pmc/articles/instance/3334867/bin/nihms367719f5.gif" src-large="/pmc/articles/instance/3334867/bin/nihms367719f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3334867/" ref="ordinalpos=2385&amp;ncbi_uid=4815633&amp;link_uid=PMC3334867" image-link="/pmc/articles/PMC3334867/figure/F5/" class="imagepopup">Figure 5. Multiple receptors activate the STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Inborn Errors of Human JAKs and STATs. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> and inhibitions of STAT3 are shown, with areas of special emphasis for the dominant negative mutants shown. Crosshatched STAT3 depicts the AD-mutant form of the molecule, with each heterodimer in which it participates being inhibited from function. The thunderbolts are to indicate where a normal function is being inhibited. STAT3 is both involved in IL-10 signal transduction and in IL-10 expression, both of which are affected in Job’s syndrome.</div></p></div><div class="aux"><div class="resc">Jean-Laurent Casanova, et al. Immunity. ;36(4):515-528.</div><p class="links"><a class="dblinks" href="/pubmed/22520845" ref="ordinalpos=2385&amp;ncbi_uid=4815633&amp;link_uid=22520845">Citation</a><a class="dblinks" href="/pmc/articles/PMC3334867/" ref="ordinalpos=2385&amp;ncbi_uid=4815633&amp;link_uid=PMC3334867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2386.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3676467/figure/pone-0065448-g006/" ref="ordinalpos=2386&amp;ncbi_uid=4990133&amp;link_uid=PMC3676467" href="/pmc/articles/PMC3676467/"><img src="/pmc/articles/instance/3676467/bin/pone.0065448.g006.gif" src-large="/pmc/articles/instance/3676467/bin/pone.0065448.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676467/" ref="ordinalpos=2386&amp;ncbi_uid=4990133&amp;link_uid=PMC3676467" image-link="/pmc/articles/PMC3676467/figure/pone-0065448-g006/" class="imagepopup">Figure 6. Kif3a controls expression of a Fgf8-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during nephron formation..  From: Kif3a Controls Murine Nephron Number Via GLI3 Repressor, Cell Survival, and Gene Expression in a Lineage-Specific Manner. </a></p><div class="supp"><p class="details"><div>A. Schematic of <i>Fgf8/Wnt4/Lim1</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> required during nephron formation. (B–G) <i>In situ</i> RNA hybridization in kidney tissue sections generated from E15.5 embryos. <i>Fgf8</i> (B, C), <i>Wnt4</i> (D, E) and <i>Lim1</i> (F, G) are expressed in nephrogenic progenitors in WT mice. <i>Fgf8</i> expression is barely detectable in <i>Kif3a<sup>−/−MM</sup></i> mice (C, C’). Expression of <i>Wnt4</i> (E) is markedly decreased as is expression of <i>Lim1</i> (G). (H, I) Histological sections from P0 kidney tissue. There are fewer S-shaped bodies (arrows) in <i>Kif3a<sup>−/−MM</sup></i> mice (I) than in control mice (H). Scale bars: 25 micrometer.</div></p></div><div class="aux"><div class="resc">Lijun Chi, et al. PLoS One. 2013;8(6):e65448.</div><p class="links"><a class="dblinks" href="/pubmed/23762375" ref="ordinalpos=2386&amp;ncbi_uid=4990133&amp;link_uid=23762375">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676467/" ref="ordinalpos=2386&amp;ncbi_uid=4990133&amp;link_uid=PMC3676467">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2387.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4358870/figure/Fig3/" ref="ordinalpos=2387&amp;ncbi_uid=7281810&amp;link_uid=PMC4358870" href="/pmc/articles/PMC4358870/"><img src="/pmc/articles/instance/4358870/bin/12885_2015_1124_Fig3_HTML.gif" src-large="/pmc/articles/instance/4358870/bin/12885_2015_1124_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4358870/" ref="ordinalpos=2387&amp;ncbi_uid=7281810&amp;link_uid=PMC4358870" image-link="/pmc/articles/PMC4358870/figure/Fig3/" class="imagepopup">Figure 3.  From: Astrocyte elevated gene-1(AEG-1) induces epithelial-mesenchymal transition in lung cancer through activating Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div> <b>AEG-1 was associated with CKIδand modulated the GSK-3β/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> Slu-01 cells stably overexpressing AEG-1 were established. GSK-3β was immunoprecipitated from cell lysates, and its expression was confirmed by immunoblotting with the indicated antibodies. <b>(B)</b> CKIδ was critical for AEG-1-mediated regulation of GSK-3β/β-catenin <span class="highlight" style="background-color:">signaling</span> and EMT. AEG-1 complexes were associated with both GSK-3β and CKIδ. <b>(C)</b> CKIδplayed a role in AEG-mediated regulation of EMT and Ser-9 phosphorylation of GSK-3β. Slu-01 cells transfected with pcDNA3.1-AEG-1 were co-transfected with CKIδ-specific siRNA. Cell lysates were then subjected to Western blotting.</div></p></div><div class="aux"><div class="resc">Weiling He, et al. BMC Cancer. 2015;15:107.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2387&amp;ncbi_uid=7281810&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4358870/" ref="ordinalpos=2387&amp;ncbi_uid=7281810&amp;link_uid=PMC4358870">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2388.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2987346/figure/fig2/" ref="ordinalpos=2388&amp;ncbi_uid=2989845&amp;link_uid=PMC2987346" href="/pmc/articles/PMC2987346/"><img src="/pmc/articles/instance/2987346/bin/ejhg2009189f2.gif" src-large="/pmc/articles/instance/2987346/bin/ejhg2009189f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2987346/" ref="ordinalpos=2388&amp;ncbi_uid=2989845&amp;link_uid=PMC2987346" image-link="/pmc/articles/PMC2987346/figure/fig2/" class="imagepopup">Figure 2.  From: Coffin-Lowry syndrome. </a></p><div class="supp"><p class="details"><div>The RSK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Patricia Marques Pereira, et al. Eur J Hum Genet. 2010 June;18(6):627-633.</div><p class="links"><a class="dblinks" href="/pubmed/19888300" ref="ordinalpos=2388&amp;ncbi_uid=2989845&amp;link_uid=19888300">Citation</a><a class="dblinks" href="/pmc/articles/PMC2987346/" ref="ordinalpos=2388&amp;ncbi_uid=2989845&amp;link_uid=PMC2987346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2389.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517588/figure/pone-0049812-g002/" ref="ordinalpos=2389&amp;ncbi_uid=4442912&amp;link_uid=PMC3517588" href="/pmc/articles/PMC3517588/"><img src="/pmc/articles/instance/3517588/bin/pone.0049812.g002.gif" src-large="/pmc/articles/instance/3517588/bin/pone.0049812.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517588/" ref="ordinalpos=2389&amp;ncbi_uid=4442912&amp;link_uid=PMC3517588" image-link="/pmc/articles/PMC3517588/figure/pone-0049812-g002/" class="imagepopup">Figure 2. <span class="highlight" style="background-color:">Pathway</span> analysis of the significantly lesion- vs. sham-regulated genes in aged and young animals..  From: Age-Dependent Modulation of Cortical Transcriptomes in Spinal Cord Injury and Repair. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis at different time points was done using the Ingenuity curated database of gene interactions of over 23,900 human, rat, and mouse genes. In this analysis, genes were tested for significant association in specific cell functional or <span class="highlight" style="background-color:">signaling</span> pathways versus random chance association in a total of gene interactions using right-tailed Fisher's exact test (Ingenuity Systems). Significance was assessed by testing the number of genes that were regulated by sprouting neurons in a specific <span class="highlight" style="background-color:">pathway</span> versus total number of genes in this database for that <span class="highlight" style="background-color:">pathway</span> (blue columns). The red line in the left graph indicates the threshold for a significant association, the −log (0.05). For example, if the <span class="highlight" style="background-color:">pathway</span> has a <i>P-score</i> of 10, the odds of this <span class="highlight" style="background-color:">pathway</span> being generated at random are less than 1 out of 10<sup>10</sup>.</div></p></div><div class="aux"><div class="resc">Anne Jaerve, et al. PLoS One. 2012;7(12):e49812.</div><p class="links"><a class="dblinks" href="/pubmed/23236355" ref="ordinalpos=2389&amp;ncbi_uid=4442912&amp;link_uid=23236355">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517588/" ref="ordinalpos=2389&amp;ncbi_uid=4442912&amp;link_uid=PMC3517588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2390.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3241725/figure/fig3/" ref="ordinalpos=2390&amp;ncbi_uid=4055976&amp;link_uid=PMC3241725" href="/pmc/articles/PMC3241725/"><img src="/pmc/articles/instance/3241725/bin/JCB_201108081_RGB_Fig3.gif" src-large="/pmc/articles/instance/3241725/bin/JCB_201108081_RGB_Fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3241725/" ref="ordinalpos=2390&amp;ncbi_uid=4055976&amp;link_uid=PMC3241725" image-link="/pmc/articles/PMC3241725/figure/fig3/" class="imagepopup">Figure 3. From: Cell death and infection: A double-edged sword for host and pathogen survival. </a></p><div class="supp"><p class="details"><div><b>Bacterial manipulation of host pro-survival <span class="highlight" style="background-color:">pathway</span>.</b> <i>M. tuberculosis</i> activates host anti-apoptotic <span class="highlight" style="background-color:">signaling</span> by up-regulating the anti-apoptosis genes <i>Mcl-1</i>, <i>bfl1</i>, and <i>FLIP</i>. NuoG dampens Nox2-mediated host <span class="highlight" style="background-color:">signaling</span>, resulting in the inhibition of apoptosis. <i>Legionella</i> deliver LegK1 and LnaB, which enhance the pro-survival activity of NF-κB. <i>Shigella</i> invasion of the epithelium results in mitochondria permeability transition (MPT)−dependent ROS production and necrosis-like cell death, which is counterbalanced by the NOD1−RIP2−NF-κB−Bcl2 pro-survival <span class="highlight" style="background-color:">pathway</span>. PGN, peptidoglycan. <i>Salmonella</i> deliver SopB and AvrA via the T3SS, and they inhibit apoptosis by activating the PI3K–Akt pro-survival <span class="highlight" style="background-color:">pathway</span> via inositol phosphate activity and by preventing JNK pro-cell death <span class="highlight" style="background-color:">signaling</span> via acetyltransferase activity, respectively. EPEC also targets JNK <span class="highlight" style="background-color:">signaling</span> by delivering NleD, which uses its metalloprotease activity to cleave JNK, thereby inhibiting cell death.</div></p></div><div class="aux"><div class="resc">Hiroshi Ashida, et al. J Cell Biol. 2011 December 12;195(6):931-942.</div><p class="links"><a class="dblinks" href="/pubmed/22123830" ref="ordinalpos=2390&amp;ncbi_uid=4055976&amp;link_uid=22123830">Citation</a><a class="dblinks" href="/pmc/articles/PMC3241725/" ref="ordinalpos=2390&amp;ncbi_uid=4055976&amp;link_uid=PMC3241725">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2391.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1698511/figure/f3/" ref="ordinalpos=2391&amp;ncbi_uid=833139&amp;link_uid=PMC1698511" href="/pmc/articles/PMC1698511/"><img src="/pmc/articles/instance/1698511/bin/zmr0040621400003.gif" src-large="/pmc/articles/instance/1698511/bin/zmr0040621400003.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1698511/" ref="ordinalpos=2391&amp;ncbi_uid=833139&amp;link_uid=PMC1698511" image-link="/pmc/articles/PMC1698511/figure/f3/" class="imagepopup">FIG. 3. From: Impact of Protein Kinase PKR in Cell Biology: from Antiviral to Antiproliferative Action. </a></p><div class="supp"><p class="details"><div>PKR is an intermediary component in TLR <span class="highlight" style="background-color:">signaling</span>. PKR is implicated in the LPS/TLR4-mediated <span class="highlight" style="background-color:">pathway</span> probably recruited by the TIRAP complex. PKR is also involved during the dsRNA/TLR3 <span class="highlight" style="background-color:">pathway</span>, recruited by a TAK1-containing complex. Several proteins act downstream from PKR, such as TRAF6, which has been identified downstream of PKR in the <span class="highlight" style="background-color:">signaling</span> cascades triggered by TLR3 and TLR4. Moreover, TRAF3 is also involved in PKR downstream events during the activation of TLR3. The activation of these pathways in turn triggers the induction of proinflammatory cytokines. RIGI/MDA5 in response to dsRNA <span class="highlight" style="background-color:">signaling</span> is indicated.</div></p></div><div class="aux"><div class="resc">M. A. García, et al. Microbiol Mol Biol Rev. 2006 December;70(4):1032-1060.</div><p class="links"><a class="dblinks" href="/pubmed/17158706" ref="ordinalpos=2391&amp;ncbi_uid=833139&amp;link_uid=17158706">Citation</a><a class="dblinks" href="/pmc/articles/PMC1698511/" ref="ordinalpos=2391&amp;ncbi_uid=833139&amp;link_uid=PMC1698511">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2392.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3132010/figure/F2/" ref="ordinalpos=2392&amp;ncbi_uid=3223206&amp;link_uid=PMC3132010" href="/pmc/articles/PMC3132010/"><img src="/pmc/articles/instance/3132010/bin/ar3255-2.gif" src-large="/pmc/articles/instance/3132010/bin/ar3255-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3132010/" ref="ordinalpos=2392&amp;ncbi_uid=3223206&amp;link_uid=PMC3132010" image-link="/pmc/articles/PMC3132010/figure/F2/" class="imagepopup">Figure 2.  From: The role of the Notch <span class="highlight" style="background-color:">pathway</span> in healthy and osteoarthritic articular cartilage: from experimental models to ex vivo studies. </a></p><div class="supp"><p class="details"><div><b>Canonical and non-canonical (Abl) Notch <span class="highlight" style="background-color:">signaling</span> pathways</b>. A, co-activator; CSL, CBF, Su(H), Lag3; DSL, Delta, Serrate, Lag2; R, co-repressor; S1, S2, S3 and S4, Notch cleavage sites in the canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; TACE, tumor necrosis factor alpha converting enzyme.</div></p></div><div class="aux"><div class="resc">Nadia Sassi, et al. Arthritis Res Ther. 2011;13(2):208-208.</div><p class="links"><a class="dblinks" href="/pubmed/21457519" ref="ordinalpos=2392&amp;ncbi_uid=3223206&amp;link_uid=21457519">Citation</a><a class="dblinks" href="/pmc/articles/PMC3132010/" ref="ordinalpos=2392&amp;ncbi_uid=3223206&amp;link_uid=PMC3132010">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2393.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4069606/figure/F1/" ref="ordinalpos=2393&amp;ncbi_uid=6290675&amp;link_uid=PMC4069606" href="/pmc/articles/PMC4069606/"><img src="/pmc/articles/instance/4069606/bin/nihms-596860-f0001.gif" src-large="/pmc/articles/instance/4069606/bin/nihms-596860-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4069606/" ref="ordinalpos=2393&amp;ncbi_uid=6290675&amp;link_uid=PMC4069606" image-link="/pmc/articles/PMC4069606/figure/F1/" class="imagepopup">Fig. 1.  From: Dynamically reshaping <span class="highlight" style="background-color:">signaling</span> networks to program cell fate via genetic controllers. </a></p><div class="supp"><p class="details"><div>Molecular network diverters and key <span class="highlight" style="background-color:">pathway</span> control points. <b>(A)</b> General architecture of a molecular network diverter. Transducer: RNA-based controller responsive to an environmental signal; modulator: promoter that controls level and mode of expression (feedback, non-feedback); <span class="highlight" style="background-color:">pathway</span> regulator: protein that modulates <span class="highlight" style="background-color:">pathway</span> activity. <b>(B)</b> Diagram of molecular network diverters reshaping native <span class="highlight" style="background-color:">signaling</span> networks in response to environmental signals. <b>(C)</b> Schematic of the yeast mating <span class="highlight" style="background-color:">pathway</span>. Pheromone (α-factor) binds to a transmembrane receptor (Ste2), where the signal is relayed through G-proteins (Ste4), an adaptor protein (Ste50), and a scaffold (Ste5)-bound three-tiered MAPK cascade. Phosphorylated Fus3 (dephosphorylated by Msg5) is translocated to the nucleus, resulting in activation of mating genes. <span class="highlight" style="background-color:">Pathway</span> activity and mating response are monitored through a <i>pFUS1-GFP</i> reporter and pheromone-induced cell cycle arrest, respectively.</div></p></div><div class="aux"><div class="resc">Kate E. Galloway, et al. Science. ;341(6152):1235005-1235005.</div><p class="links"><a class="dblinks" href="/pubmed/23950497" ref="ordinalpos=2393&amp;ncbi_uid=6290675&amp;link_uid=23950497">Citation</a><a class="dblinks" href="/pmc/articles/PMC4069606/" ref="ordinalpos=2393&amp;ncbi_uid=6290675&amp;link_uid=PMC4069606">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2394.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634257/figure/F1/" ref="ordinalpos=2394&amp;ncbi_uid=1709762&amp;link_uid=PMC2634257" href="/pmc/articles/PMC2634257/"><img src="/pmc/articles/instance/2634257/bin/org0402_0116_fig001.gif" src-large="/pmc/articles/instance/2634257/bin/org0402_0116_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634257/" ref="ordinalpos=2394&amp;ncbi_uid=1709762&amp;link_uid=PMC2634257" image-link="/pmc/articles/PMC2634257/figure/F1/" class="imagepopup">Figure 1.  From: Wnt <span class="highlight" style="background-color:">signaling</span> in breast organogenesis. </a></p><div class="supp"><p class="details"><div>Simplified schemes of the three Wnt <span class="highlight" style="background-color:">signaling</span> cascades. (A) Canonical <span class="highlight" style="background-color:">Pathway</span>. A Wnt ligand binds to a Frizzled receptor and a co-receptor of low density lipoprotein (Lrp) family. Dishevelled (Dvl) and β-catenin are required to transduce the Wnt signal, leading to a transcriptional response mediated by transcription factors of the Tcf/Lef1 family. (B) Planar cell Polarity <span class="highlight" style="background-color:">pathway</span>. In this <span class="highlight" style="background-color:">pathway</span>, Dvl also transduces signals although independent of β-catenin activating JNK and Rho kinase cascades. (C) The Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> signals via heterotrimeric G-proteins to mobilize intracellular Ca<sup>2+</sup> and stimulate protein kinase C (PKC). The requirement for Dvl is controversial. In vertebrates this <span class="highlight" style="background-color:">pathway</span> is activated by the same ligands as (B). Dapper, a Dvl-associated antogonist, antagonizes both (A and B) pathways.</div></p></div><div class="aux"><div class="resc">Kata Boras-Granic, et al. Organogenesis. 2008 Apr-Jun;4(2):116-122.</div><p class="links"><a class="dblinks" href="/pubmed/19279723" ref="ordinalpos=2394&amp;ncbi_uid=1709762&amp;link_uid=19279723">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634257/" ref="ordinalpos=2394&amp;ncbi_uid=1709762&amp;link_uid=PMC2634257">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2395.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3880986/figure/F7/" ref="ordinalpos=2395&amp;ncbi_uid=5681663&amp;link_uid=PMC3880986" href="/pmc/articles/PMC3880986/"><img src="/pmc/articles/instance/3880986/bin/ijmsv11p0007g010.gif" src-large="/pmc/articles/instance/3880986/bin/ijmsv11p0007g010.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3880986/" ref="ordinalpos=2395&amp;ncbi_uid=5681663&amp;link_uid=PMC3880986" image-link="/pmc/articles/PMC3880986/figure/F7/" class="imagepopup">Fig 7.  From: 17?-Estradiol Inhibits Mesenchymal Stem Cells-Induced Human AGS Gastric Cancer Cell Mobility via Suppression of CCL5- Src/Cas/Paxillin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b> A schematic representation showing 17β-estradiol inhibition of cell mobility via suppression of CCL5-Src/Cas/Paxillin pathways in human AGS gastric cancer cells.</b> CCL-5 is mainly secreted from HBMMSCs, which is great enhanced when co-culturing with AGS cells. Soluble CCL-5 induces the activation of Src, Cas and paxillin, then contributing to the changes in organization of cellular actin cytoskeleton, which thus promotes cell mobility events in human AGS gastric cancer cells. Treatment of 17β-estradiol significantly inhibits HBMMSCs-induced cell mobility by suppressing activation of CCL5-Src/Cas/paxillin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chao-Hung Kuo, et al. Int J Med Sci. 2014;11(1):7-16.</div><p class="links"><a class="dblinks" href="/pubmed/24396281" ref="ordinalpos=2395&amp;ncbi_uid=5681663&amp;link_uid=24396281">Citation</a><a class="dblinks" href="/pmc/articles/PMC3880986/" ref="ordinalpos=2395&amp;ncbi_uid=5681663&amp;link_uid=PMC3880986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2396.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938097/figure/F9/" ref="ordinalpos=2396&amp;ncbi_uid=3059917&amp;link_uid=PMC2938097" href="/pmc/articles/PMC2938097/"><img src="/pmc/articles/instance/2938097/bin/zjw0071036280009.gif" src-large="/pmc/articles/instance/2938097/bin/zjw0071036280009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938097/" ref="ordinalpos=2396&amp;ncbi_uid=3059917&amp;link_uid=PMC2938097" image-link="/pmc/articles/PMC2938097/figure/F9/" class="imagepopup">Fig. 9. From: Studies of Phosphoproteomic Changes Induced by Nucleophosmin-Anaplastic Lymphoma Kinase (ALK) Highlight Deregulation of Tumor Necrosis Factor (TNF)/Fas/TNF-related Apoptosis-induced Ligand <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in ALK-positive Anaplastic Large Cell Lymphoma. </a></p><div class="supp"><p class="details"><div><b>Validation of phosphoproteins involved in TNF/Fas/TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> using ALK<sup>+</sup> ALCL cell lines.</b> The immunoprecipitation (<i>IP</i>) experiments were performed using Karpas 299 and SU-DH-L1, two well studied ALK<sup>+</sup> ALCL cell lines, with anti-TRAP1 antibody (<i>A</i>), anti-RIP antibody (<i>B</i>), and anti-FAF1 antibody (<i>C</i>). The phosphorylation statuses of these three proteins were detected by using anti-phosphotyrosine antibody. The immunoprecipitation experiments without antibody were used as control experiments. <i>IB</i>, immunoblot; <i>phos</i>, phospho.</div></p></div><div class="aux"><div class="resc">Fang Wu, et al. Mol Cell Proteomics. 2010 July;9(7):1616-1632.</div><p class="links"><a class="dblinks" href="/pubmed/20393185" ref="ordinalpos=2396&amp;ncbi_uid=3059917&amp;link_uid=20393185">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938097/" ref="ordinalpos=2396&amp;ncbi_uid=3059917&amp;link_uid=PMC2938097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2397.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3073839/figure/F2/" ref="ordinalpos=2397&amp;ncbi_uid=3402055&amp;link_uid=PMC3073839" href="/pmc/articles/PMC3073839/"><img src="/pmc/articles/instance/3073839/bin/nihms234689f2.gif" src-large="/pmc/articles/instance/3073839/bin/nihms234689f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3073839/" ref="ordinalpos=2397&amp;ncbi_uid=3402055&amp;link_uid=PMC3073839" image-link="/pmc/articles/PMC3073839/figure/F2/" class="imagepopup">Figure 2. Activation of the Arf promoter by HER2 and the <span class="highlight" style="background-color:">signaling</span> pathways that link HER2 and p53.  From: Critical roles of DMP1 in HER2/neu-Arf-p53 <span class="highlight" style="background-color:">signaling</span> and breast cancer development. </a></p><div class="supp"><p class="details"><div><i>A</i>, activation of the <i>Arf</i> promoter by HER2 is dependent on Dmp1. Reporter assays were conducted in 3T3 cells (left and middle panels) or in <i>Dmp1<sup>−/−</sup></i> cells (right panel). The baseline (without HER2 expression vector) was set at 1.0. <i>B</i>, binding of endogenous Dmp1 to the <i>Arf</i> promoter in four different MMTV-<i>neu</i> tumors. Tissue ChIP was conducted with formalin-fixed tumors from wild-type MMTV-<i>neu</i> mice. The Dmp1 protein was detectable on the <i>Arf</i> promoter with two different antibodies to Dmp1 (RAX for T1 and T2, and RAD for T3 and T4). <i>C</i>, proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that links HER2/neu overexpression and p53 activation. Activation of the <i>Dmp1</i> promoter is mediated by PI3K-Akt-NF-κB <span class="highlight" style="background-color:">signaling</span>, and induction of <i>Arf</i> by HER2/neu is dependent on Dmp1. This diverts toxic hyperproliferative <span class="highlight" style="background-color:">signaling</span> from HER2/neu to a p53-dependent cell cycle arrest or apoptosis.</div></p></div><div class="aux"><div class="resc">Pankaj Taneja, et al. Cancer Res. ;70(22):9084-9094.</div><p class="links"><a class="dblinks" href="/pubmed/21062982" ref="ordinalpos=2397&amp;ncbi_uid=3402055&amp;link_uid=21062982">Citation</a><a class="dblinks" href="/pmc/articles/PMC3073839/" ref="ordinalpos=2397&amp;ncbi_uid=3402055&amp;link_uid=PMC3073839">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2398.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4332478/figure/pcbi.1003983.g003/" ref="ordinalpos=2398&amp;ncbi_uid=7211643&amp;link_uid=PMC4332478" href="/pmc/articles/PMC4332478/"><img src="/pmc/articles/instance/4332478/bin/pcbi.1003983.g003.gif" src-large="/pmc/articles/instance/4332478/bin/pcbi.1003983.g003.jpg" alt="Fig 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4332478/" ref="ordinalpos=2398&amp;ncbi_uid=7211643&amp;link_uid=PMC4332478" image-link="/pmc/articles/PMC4332478/figure/pcbi.1003983.g003/" class="imagepopup">Fig 3. Glioblastoma <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> alterations for top hub regulators..  From: Integrative Multi-omics Module Network Inference with Lemon-Tree. </a></p><div class="supp"><p class="details"><div>Copy number alterations for a selection of predicted hub regulators are indicated for canonical glioblastoma <span class="highlight" style="background-color:">signaling</span> pathways p53, RB and RTK/PI3K. Genes selected by the algorithm are indicated in black boxes, while light grey boxes depict genes that were not selected by the algorithm but are key factors for the <span class="highlight" style="background-color:">pathway</span>. Purple hexagons indicate phenotypes. Percentage of copy gain or loss is indicated by value and by color shades of red for gene gains and green for gene losses. The values are taken from GISTIC putative calls for low-levels gains or single-copy losses on 563 glioblastoma samples (data from the Broad institute).</div></p></div><div class="aux"><div class="resc">Eric Bonnet, et al. PLoS Comput Biol. 2015 February;11(2):e1003983.</div><p class="links"><a class="dblinks" href="/pubmed/25679508" ref="ordinalpos=2398&amp;ncbi_uid=7211643&amp;link_uid=25679508">Citation</a><a class="dblinks" href="/pmc/articles/PMC4332478/" ref="ordinalpos=2398&amp;ncbi_uid=7211643&amp;link_uid=PMC4332478">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2399.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3712869/figure/F4/" ref="ordinalpos=2399&amp;ncbi_uid=5101048&amp;link_uid=PMC3712869" href="/pmc/articles/PMC3712869/"><img src="/pmc/articles/instance/3712869/bin/nihms-214943-f0004.gif" src-large="/pmc/articles/instance/3712869/bin/nihms-214943-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3712869/" ref="ordinalpos=2399&amp;ncbi_uid=5101048&amp;link_uid=PMC3712869" image-link="/pmc/articles/PMC3712869/figure/F4/" class="imagepopup">Figure 4. Proposed Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in bladder mesenchymal cells.  From: Smooth Muscle Differentiation and Patterning in the Urinary Bladder. </a></p><div class="supp"><p class="details"><div>Upon binding Shh, Ptc1 releases the inhibition of Smo. The effect of Shh on smooth muscle differentiation and patterning is mediated by <i>Gli2</i>, which, upon translocation into the nucleus, regulates downstream genes including BMP4. BMP4 may inhibit cellular proliferation and is a prime candidate for mediating the effect of Shh on smooth muscle patterning during bladder development. Other <span class="highlight" style="background-color:">signaling</span> molecules, such as TGF-β and SRF, likely have a role in bladder smooth muscle differentiation and may interact with the Shh <span class="highlight" style="background-color:">pathway</span>. The mechanism through which Shh affects cellular proliferation in the bladder is uncertain but MPF has been shown to be a Shh-responsive complex that regulates mitosis in other systems.</div></p></div><div class="aux"><div class="resc">Gregory Tasian, et al. Differentiation. ;80(0):106-117.</div><p class="links"><a class="dblinks" href="/pubmed/20541860" ref="ordinalpos=2399&amp;ncbi_uid=5101048&amp;link_uid=20541860">Citation</a><a class="dblinks" href="/pmc/articles/PMC3712869/" ref="ordinalpos=2399&amp;ncbi_uid=5101048&amp;link_uid=PMC3712869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2400.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3310225/figure/F5/" ref="ordinalpos=2400&amp;ncbi_uid=4750459&amp;link_uid=PMC3310225" href="/pmc/articles/PMC3310225/"><img src="/pmc/articles/instance/3310225/bin/nihms357092f5.gif" src-large="/pmc/articles/instance/3310225/bin/nihms357092f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3310225/" ref="ordinalpos=2400&amp;ncbi_uid=4750459&amp;link_uid=PMC3310225" image-link="/pmc/articles/PMC3310225/figure/F5/" class="imagepopup">Figure 5. Modulation of the PKA <span class="highlight" style="background-color:">pathway</span> identified PKA-2 as a regulator of tubule morphogenesis.  From: A Protein Kinase A and Wnt-dependent network regulating an intermediate stage in epithelial tubulogenesis during kidney development. </a></p><div class="supp"><p class="details"><div>Spinal cord-induced MM was treated with PKA <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> modulators and observed for defects in tubule morphogenesis. (A) Activation of PKA by stimulation of cAMP production (forskolin) or by adding cAMP analogues (dibutyryl-cAMP) prevented tubular morphogenesis. Activation of PKA-1 (8-PIP-cAMP and 8-HA-cAMPS) and Epac (8-CPT-2-O-Me-cAMP) did not inhibit tubulogenesis while activation of PKA-2 (6-MBC-cAMP and Sp-5,6-DCl-cBIMPS) prevented tubule formation. Inhibition of PKA with the Walsh PKA inhibitory peptide, 14–22 stimulated tubulogenesis. (B) Cartoon demonstrating the PKA <span class="highlight" style="background-color:">pathway</span> and the effects of modulation of specific parts on tubulogenesis demonstrates activation of the PKA <span class="highlight" style="background-color:">pathway</span> at different levels prevents tubulogenesis.</div></p></div><div class="aux"><div class="resc">Thomas F. Gallegos, et al. Dev Biol. ;364(1):11-21.</div><p class="links"><a class="dblinks" href="/pubmed/22290330" ref="ordinalpos=2400&amp;ncbi_uid=4750459&amp;link_uid=22290330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3310225/" ref="ordinalpos=2400&amp;ncbi_uid=4750459&amp;link_uid=PMC3310225">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="23" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="24" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="25" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:34:07-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB703F5430F7F10000000000F6CAF1;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB703F5430F7F10000000000F6CAF1" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>