
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="28" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB1FC7543107C10000000000571741" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="27" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="28" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="29" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="28" /></div>
            <div><h2 class="result_count">Results: 2701 to 2800 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2701.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3814639/figure/F3/" ref="ordinalpos=2701&amp;ncbi_uid=6032992&amp;link_uid=PMC3814639" href="/pmc/articles/PMC3814639/"><img src="/pmc/articles/instance/3814639/bin/2179fig3.gif" src-large="/pmc/articles/instance/3814639/bin/2179fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3814639/" ref="ordinalpos=2701&amp;ncbi_uid=6032992&amp;link_uid=PMC3814639" image-link="/pmc/articles/PMC3814639/figure/F3/" class="imagepopup">Figure 3. From: The emerging role of the Nrf2-Keap1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancer. </a></p><div class="supp"><p class="details"><div>Cross-talk between Nrf2 and other proteins. The substrate adaptor sequestosome 1 protein (p62) modulates the Nrf2â€“Keap1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by directly interacting with Keap1. p62 sequesters Keap1 in autophagosomes, which results in a decrease in Nrf2 ubiquitination, an increase in Nrf2 stability, and activation of Nrf2 target genes. Nrf2 is stabilized in response to p21 up-regulation. p21 associates with the DLG motif on Nrf2, thereby disrupting the ability of Keap1 to properly bind and ubiquitinate Nrf2. DPP3 binds to Keap1, inhibits Nrf2 ubiquitination, and drives Nrf2-dependent transcription in cancer cells. (E) ETGE; (S) STGE; (D) DLG.</div></p></div><div class="aux"><div class="resc">Melba C. Jaramillo, et al. Genes Dev. 2013 October 15;27(20):2179-2191.</div><p class="links"><a class="dblinks" href="/pubmed/24142871" ref="ordinalpos=2701&amp;ncbi_uid=6032992&amp;link_uid=24142871">Citation</a><a class="dblinks" href="/pmc/articles/PMC3814639/" ref="ordinalpos=2701&amp;ncbi_uid=6032992&amp;link_uid=PMC3814639">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2702.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3563200/figure/fig02/" ref="ordinalpos=2702&amp;ncbi_uid=4616858&amp;link_uid=PMC3563200" href="/pmc/articles/PMC3563200/"><img src="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f2.gif" src-large="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3563200/" ref="ordinalpos=2702&amp;ncbi_uid=4616858&amp;link_uid=PMC3563200" image-link="/pmc/articles/PMC3563200/figure/fig02/" class="imagepopup">Figure 2.  From: Indoleamine 2,3-dioxygenase-1 (IDO1) enhances survival and invasiveness of endometrial stromal cells via the activation of JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>IDO1 activates JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ESCs. Normal ESCs from endometriosis-free women were treated with plasmid pEGFP-N1-IDO1 or SD11-IDO1 shRNA, and vector-only plasmid as control. The efficiency of plasmid transfection (A), as well as the phosphorylation level of Erk1/2 (B), JNK (C) and p38 (D) was demonstrated by in-cell Western analysis. Herein, vector control: normal ESCs transfected with vector-only plasmid; IDO1 overexpression: normal ESCs transfected with plasmid pEGFP-N1-IDO1; IDO1 interference: normal ESCs transfected with SD11-IDO1 shRNA. *P&lt;.01, vs. normal group, **P&lt;.01, vs. vector control group. Representative data as well as the mean Â± SD of 12 different experiments were shown.</div></p></div><div class="aux"><div class="resc">Jie Mei, et al. Int J Clin Exp Pathol. 2013;6(3):431-444.</div><p class="links"><a class="dblinks" href="/pubmed/23411497" ref="ordinalpos=2702&amp;ncbi_uid=4616858&amp;link_uid=23411497">Citation</a><a class="dblinks" href="/pmc/articles/PMC3563200/" ref="ordinalpos=2702&amp;ncbi_uid=4616858&amp;link_uid=PMC3563200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2703.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3875894/figure/F3/" ref="ordinalpos=2703&amp;ncbi_uid=5641249&amp;link_uid=PMC3875894" href="/pmc/articles/PMC3875894/"><img src="/pmc/articles/instance/3875894/bin/2043-9113-3-20-3.gif" src-large="/pmc/articles/instance/3875894/bin/2043-9113-3-20-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3875894/" ref="ordinalpos=2703&amp;ncbi_uid=5641249&amp;link_uid=PMC3875894" image-link="/pmc/articles/PMC3875894/figure/F3/" class="imagepopup">Figure 3.  From: A systematic analysis of a mi-RNA inter-<span class="highlight" style="background-color:">pathway</span> regulatory motif. </a></p><div class="supp"><p class="details"><div><b>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (#hsa04150).</b> TOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (#hsa04150) including the identified PPL (3/31/09 release). (1) The <span class="highlight" style="background-color:">pathway</span> host-gene (RSK) expresses a protective intragenic miRNA (miR-1976) which (2) acts against the expression of the MLL transcription factor. (3) MLL is responsible for HOXA9 transcription, that (4) leads to the expression of miR-196b which, if expressed, (5) would target RICTOR, dysregulating the mTORC2 complex in which RICTOR plays a central role.</div></p></div><div class="aux"><div class="resc">Stefano Di Carlo, et al. J Clin Bioinforma. 2013;3:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/24152805" ref="ordinalpos=2703&amp;ncbi_uid=5641249&amp;link_uid=24152805">Citation</a><a class="dblinks" href="/pmc/articles/PMC3875894/" ref="ordinalpos=2703&amp;ncbi_uid=5641249&amp;link_uid=PMC3875894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2704.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3628745/figure/fig-2/" ref="ordinalpos=2704&amp;ncbi_uid=4868918&amp;link_uid=PMC3628745" href="/pmc/articles/PMC3628745/"><img src="/pmc/articles/instance/3628745/bin/peerj-01-49-g002.gif" src-large="/pmc/articles/instance/3628745/bin/peerj-01-49-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3628745/" ref="ordinalpos=2704&amp;ncbi_uid=4868918&amp;link_uid=PMC3628745" image-link="/pmc/articles/PMC3628745/figure/fig-2/" class="imagepopup">Figure 2.  From: A new 12-gene diagnostic biomarker signature of melanoma revealed by integrated microarray analysis. </a></p><div class="supp"><p class="details"><div>A new <span class="highlight" style="background-color:">signaling</span> network for melanoma. The <span class="highlight" style="background-color:">signaling</span> network is based on the complex interactions of the 12 signature genes (labeled in red) and the 4 melanoma driver genes (<i>BRAF, cKit, NRAS, MITF</i>) in 3 <span class="highlight" style="background-color:">signaling</span> pathways (MAPK, Ca<sup>2+</sup> and WNT). Nine of these 12 genes (i.e., <i>EGFR, FGFR2, FGFR3, IL8, PTPRF, CXCL13, TNC, COL11A1, and SHC4</i>) closely interact with three driver genes (<i>NRAS, BRAF, and MITF</i>) in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: the remaining 3 genes include <i>WNT4, PPP2R2C and CHP2</i>, which also play important roles in WNT and Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Wanting Liu, et al. Peerj. 2013;1:e49.</div><p class="links"><a class="dblinks" href="/pubmed/23638386" ref="ordinalpos=2704&amp;ncbi_uid=4868918&amp;link_uid=23638386">Citation</a><a class="dblinks" href="/pmc/articles/PMC3628745/" ref="ordinalpos=2704&amp;ncbi_uid=4868918&amp;link_uid=PMC3628745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2705.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3814314/figure/pgen-1003852-g005/" ref="ordinalpos=2705&amp;ncbi_uid=5477969&amp;link_uid=PMC3814314" href="/pmc/articles/PMC3814314/"><img src="/pmc/articles/instance/3814314/bin/pgen.1003852.g005.gif" src-large="/pmc/articles/instance/3814314/bin/pgen.1003852.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3814314/" ref="ordinalpos=2705&amp;ncbi_uid=5477969&amp;link_uid=PMC3814314" image-link="/pmc/articles/PMC3814314/figure/pgen-1003852-g005/" class="imagepopup">Figure 5. Zic3-regulated genes in the Nodal and Wnt <span class="highlight" style="background-color:">signaling</span> pathways..  From: Genome Wide Analysis Reveals Zic3 Interaction with Distal Regulatory Elements of Stage Specific Developmental Genes in Zebrafish. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the Nodal/TGF-Î² and Wnt <span class="highlight" style="background-color:">signaling</span> pathways generated by the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis Software. Genes differentially regulated by Zic3 are shown in colour â€“ red for upregulation and green for downregulation of their expression patterns in the microarray data.</div></p></div><div class="aux"><div class="resc">Cecilia L. Winata, et al. PLoS Genet. 2013 October;9(10):e1003852.</div><p class="links"><a class="dblinks" href="/pubmed/24204288" ref="ordinalpos=2705&amp;ncbi_uid=5477969&amp;link_uid=24204288">Citation</a><a class="dblinks" href="/pmc/articles/PMC3814314/" ref="ordinalpos=2705&amp;ncbi_uid=5477969&amp;link_uid=PMC3814314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2706.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2957366/figure/F2/" ref="ordinalpos=2706&amp;ncbi_uid=2523983&amp;link_uid=PMC2957366" href="/pmc/articles/PMC2957366/"><img src="/pmc/articles/instance/2957366/bin/nihms188624f2.gif" src-large="/pmc/articles/instance/2957366/bin/nihms188624f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2957366/" ref="ordinalpos=2706&amp;ncbi_uid=2523983&amp;link_uid=PMC2957366" image-link="/pmc/articles/PMC2957366/figure/F2/" class="imagepopup">Fig. 2.  From: Gene expression profiling reveals signatures characterizing histologic subtypes of hepatoblastoma and global deregulation in cell growth and survival pathways. </a></p><div class="supp"><p class="details"><div>Analysis of genes differentially expressed between fetal liver and all HBs shows significant activation of <span class="highlight" style="background-color:">signaling</span> pathways including (A) wnt <span class="highlight" style="background-color:">signaling</span>, cell cycle, etc. The fold change for genes which are components of the (B) cell cycle, (C) wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, (D) TGF <i>Î²</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and (F) apoptosis, as well as deregulated genes shared with other specific malignancies such as (E) small cell lung cancer and (G) prostate cancer, is shown.</div></p></div><div class="aux"><div class="resc">Adekunle M. Adesina, et al. Hum Pathol. ;40(6):843-853.</div><p class="links"><a class="dblinks" href="/pubmed/19200578" ref="ordinalpos=2706&amp;ncbi_uid=2523983&amp;link_uid=19200578">Citation</a><a class="dblinks" href="/pmc/articles/PMC2957366/" ref="ordinalpos=2706&amp;ncbi_uid=2523983&amp;link_uid=PMC2957366">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2707.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2834863/figure/F1/" ref="ordinalpos=2707&amp;ncbi_uid=2737597&amp;link_uid=PMC2834863" href="/pmc/articles/PMC2834863/"><img src="/pmc/articles/instance/2834863/bin/nihms160855f1.gif" src-large="/pmc/articles/instance/2834863/bin/nihms160855f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2834863/" ref="ordinalpos=2707&amp;ncbi_uid=2737597&amp;link_uid=PMC2834863" image-link="/pmc/articles/PMC2834863/figure/F1/" class="imagepopup">Figure 1. General overview of TGF-Î² <span class="highlight" style="background-color:">signaling</span>.  From: Transforming growth factor beta (TGF-?) and inflammation in cancer. </a></p><div class="supp"><p class="details"><div>TGF-Î² can be liberated from latent complexes by Î±<sub>v</sub>Î²<sub>6</sub> integrin, calpain, cathepsin D, chymase, elastase, endoglycosidase F, kallikrein, matrix metalloproteinase 9 (MMP-9), neuraminidase, plasmin or thrombospondin 1. Once activated TGF-Î² is able to bind type II TGF-Î² receptor homo-dimers (TÎ²RII) thereby permitting the efficient transactivation of type I TGF-Î² receptor homo-dimers (TÎ²RI). As a result of receptor activation by TGF-Î², downstream SMAD dependent and SMAD independent <span class="highlight" style="background-color:">signaling</span> is initiated. However, the level of downstream <span class="highlight" style="background-color:">pathway</span> activation is dependent the abundance of receptor associated TGF-Î² <span class="highlight" style="background-color:">signaling</span> repressors such as SMAD7, STRAP, YAP65, SMURF1, SMURF2, GADD34 and PP1. SMAD dependent <span class="highlight" style="background-color:">signaling</span> has been primarily associated with activation of SMAD2 or SMAD3, however it has now been shown that SMAD1 and SMAD5 can be activated by TGF-Î². Once SMAD1, SMAD2, SMAD3 or SMAD5 are activated they are able to associate with SMAD4 to co-activate or co-repress transcription. Importantly, SMAD dependent <span class="highlight" style="background-color:">signaling</span> can also be repressed by complex association with other transcription factors, co-activators or co-repressors such the c-Ski or SnoN proto-oncogenes. The SMAD independent pathways are known to include ShcA, RHO, RAC/CDC42, RAS, TRAF6, TAK1, PI3K, PAR6, MAP3K1, DAXX and PP2A. Together, the balance of SMAD dependent <span class="highlight" style="background-color:">signaling</span>, SMAD independent <span class="highlight" style="background-color:">signaling</span> and the presence or absence of <span class="highlight" style="background-color:">signaling</span> repressors ultimately determines the response to TGF-Î² <i>in vitro</i> or <i>in vivo</i>.</div></p></div><div class="aux"><div class="resc">Brian Bierie, et al. Cytokine Growth Factor Rev. ;21(1):49-59.</div><p class="links"><a class="dblinks" href="/pubmed/20018551" ref="ordinalpos=2707&amp;ncbi_uid=2737597&amp;link_uid=20018551">Citation</a><a class="dblinks" href="/pmc/articles/PMC2834863/" ref="ordinalpos=2707&amp;ncbi_uid=2737597&amp;link_uid=PMC2834863">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2708.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4053448/figure/pone-0099137-g006/" ref="ordinalpos=2708&amp;ncbi_uid=6259754&amp;link_uid=PMC4053448" href="/pmc/articles/PMC4053448/"><img src="/pmc/articles/instance/4053448/bin/pone.0099137.g006.gif" src-large="/pmc/articles/instance/4053448/bin/pone.0099137.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4053448/" ref="ordinalpos=2708&amp;ncbi_uid=6259754&amp;link_uid=PMC4053448" image-link="/pmc/articles/PMC4053448/figure/pone-0099137-g006/" class="imagepopup">Figure 6. Western blot and immunocytochemistry of canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Dose-Dependent Effect of Estrogen Suppresses the Osteo-Adipogenic Transdifferentiation of Osteoblasts via Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>a:O14+A14+E2; b: O14+A14+E2+ICI; c:O14+A14+E2+DKK-1; d:O14+A14+E2+ICI+Wnt-3a. A: western blot of Gsk3Î² and Î²-catenin; B: Immunocytochemistry of Î²-catenin and GSK-3Î² protein (size barâ€Š=â€Š10 Âµm). Green: Î²-catenin; Red: GSK-3Î²; blue: DAPI staining of nuclei. All photomicrographs were recorded under identical exposure and magnification conditions. C: Percentage of Î²-catenin positive cells (T) and cells positive for Î²-catenin in the nucleus (N). Data are represented as meanÂ±SD of 3 independent experiments.</div></p></div><div class="aux"><div class="resc">Bo Gao, et al. PLoS One. 2014;9(6):e99137.</div><p class="links"><a class="dblinks" href="/pubmed/24918446" ref="ordinalpos=2708&amp;ncbi_uid=6259754&amp;link_uid=24918446">Citation</a><a class="dblinks" href="/pmc/articles/PMC4053448/" ref="ordinalpos=2708&amp;ncbi_uid=6259754&amp;link_uid=PMC4053448">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2709.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3371765/figure/F8/" ref="ordinalpos=2709&amp;ncbi_uid=4054188&amp;link_uid=PMC3371765" href="/pmc/articles/PMC3371765/"><img src="/pmc/articles/instance/3371765/bin/nihms381869f8.gif" src-large="/pmc/articles/instance/3371765/bin/nihms381869f8.jpg" alt="Fig 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3371765/" ref="ordinalpos=2709&amp;ncbi_uid=4054188&amp;link_uid=PMC3371765" image-link="/pmc/articles/PMC3371765/figure/F8/" class="imagepopup">Fig 8.  From: Pathogenesis of Barrett's esophagus: Bile acids inhibit the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with induction of CDX2 gene expression in human esophageal cells. </a></p><div class="supp"><p class="details"><div>A simplified scheme of the correlation between Notch <span class="highlight" style="background-color:">signaling</span> components and CDX2 amounts determined in the study with and without inhibition by DCA and DAPT. <i>Left, Î³</i>-secretase inhibitor DAPT decreases the amounts of Notch1C, inhibits Hrt/Hes, increases the amounts of Hath1 and CDX2 and probably leads to goblet cells production. <i>Right,</i> inhibition of the <span class="highlight" style="background-color:">pathway</span> by DCA (an unknown mechanism) decreases the total amounts of Notch receptors, thereby reducing the amounts of Notch1C, inhibits production of Hes/Hrt, increases amounts of Hath1 and CDX2, and may lead to goblet cells. An arrow indicates activation and a blocked arrow indicates inhibition. <i>Broken arrow,</i> suggested processes.</div></p></div><div class="aux"><div class="resc">David J. Morrow, et al. Surgery. ;146(4):714-722.</div><p class="links"><a class="dblinks" href="/pubmed/19789031" ref="ordinalpos=2709&amp;ncbi_uid=4054188&amp;link_uid=19789031">Citation</a><a class="dblinks" href="/pmc/articles/PMC3371765/" ref="ordinalpos=2709&amp;ncbi_uid=4054188&amp;link_uid=PMC3371765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2710.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3307138/figure/F2/" ref="ordinalpos=2710&amp;ncbi_uid=3877684&amp;link_uid=PMC3307138" href="/pmc/articles/PMC3307138/"><img src="/pmc/articles/instance/3307138/bin/fphys-03-00053-g002.gif" src-large="/pmc/articles/instance/3307138/bin/fphys-03-00053-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3307138/" ref="ordinalpos=2710&amp;ncbi_uid=3877684&amp;link_uid=PMC3307138" image-link="/pmc/articles/PMC3307138/figure/F2/" class="imagepopup">Figure 2.  From: Differential Regulation of TGF-?/Smad <span class="highlight" style="background-color:">Signaling</span> in Hepatic Stellate Cells between Acute and Chronic Liver Injuries. </a></p><div class="supp"><p class="details"><div><b>Differential regulation of fibrogenic phospho-Smad <span class="highlight" style="background-color:">signaling</span> between HSC during acute liver injury and MFB during chronic liver injury</b>. <b>(A)</b> Phospho-Smad <span class="highlight" style="background-color:">signaling</span> involved in tissue repair. TGF-Î² inhibits HSC growth by down-regulating c-Myc expression by pSmad2C and pSmad3C pathways (left); TGF-Î² <span class="highlight" style="background-color:">signaling</span> in turn enhances HSC growth and collagen synthesis via the CDK4-dependent pSmad2L/C and pSmad3L/C pathways induced by PDGF signal (right). However, Smad7 induced by pSmad3L/C signal terminates the fibrogenic phospho-Smad <span class="highlight" style="background-color:">signaling</span>. This negative-feedback mechanism of the fibrogenic TGF-Î²/PDGF signal results in a transient collagen synthesis in the activated HSC, which may thus contribute to tissue repair. <b>(B)</b> Phospho-Smad <span class="highlight" style="background-color:">signaling</span> involved in fibrogenesis. In MFB, PDGF activates JNK, which phosphorylates Smad2L and Smad3L (left). The JNK-mediated Smad3L phosphorylation leads to hetero-complex of Smad3 with Smad4 in the nucleus where the complex stimulates MFB growth by upregulation of c-Myc transcription. After COOH-tail phosphorylation of cytoplasmic pSmad2L by TGF-Î² signal, pSmad2L/C translocates to the nucleus where it binds to the pSmad3L and Smad4 complex, which then stimulates plasminogen activator inhibitor (PAI)-1 transcription (right). In contrast of Smad7 induction in HSC via pSmad3C <span class="highlight" style="background-color:">pathway</span>, pSmad3L cannot induce Smad7 in MFB (left). Under a low level of Smad7, the fibrogenic phospho-Smad <span class="highlight" style="background-color:">signaling</span> can constitutively promote ECM deposition by MFB, which may eventually develop into accelerated liver fibrosis.</div></p></div><div class="aux"><div class="resc">Katsunori Yoshida, et al. Front Physiol. 2012;3:53.</div><p class="links"><a class="dblinks" href="/pubmed/22457652" ref="ordinalpos=2710&amp;ncbi_uid=3877684&amp;link_uid=22457652">Citation</a><a class="dblinks" href="/pmc/articles/PMC3307138/" ref="ordinalpos=2710&amp;ncbi_uid=3877684&amp;link_uid=PMC3307138">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2711.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4070524/figure/F1/" ref="ordinalpos=2711&amp;ncbi_uid=6289863&amp;link_uid=PMC4070524" href="/pmc/articles/PMC4070524/"><img src="/pmc/articles/instance/4070524/bin/nihms547262f1.gif" src-large="/pmc/articles/instance/4070524/bin/nihms547262f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4070524/" ref="ordinalpos=2711&amp;ncbi_uid=6289863&amp;link_uid=PMC4070524" image-link="/pmc/articles/PMC4070524/figure/F1/" class="imagepopup">Figure 1. MOR <span class="highlight" style="background-color:">signaling</span> networks and the MAPK <span class="highlight" style="background-color:">pathway</span> are dysregulated in human heroin abusers.  From: ELK1 transcription factor linked to dysregulated striatal mu opioid receptor <span class="highlight" style="background-color:">signaling</span> network and OPRM1 polymorphism in human heroin abusers. </a></p><div class="supp"><p class="details"><div>(<i>A,B)</i> Protein levels of MOR, Î²-arrestin2, MEK1, ERK1, ERK2, ELK1, and pELK1 in the putamen of human control subjects and heroin abusers. N=15 control, N=32 heroin. Representative bands are shown next to the histograms; Mean Â± SEM. * <i>p</i> &lt; 0.05, ** <i>p</i> &lt; 0.01. (<i>C</i>) Schematic depiction of the MOR <span class="highlight" style="background-color:">signaling</span> networks. <i>(D)</i> Ratio of phosphorylated versus total ELK1 protein in human subjects. Abbreviations: C, CTL, control; H, HER, heroin.</div></p></div><div class="aux"><div class="resc">Stephanie E. Sillivan, et al. Biol Psychiatry. ;74(7):511-519.</div><p class="links"><a class="dblinks" href="/pubmed/23702428" ref="ordinalpos=2711&amp;ncbi_uid=6289863&amp;link_uid=23702428">Citation</a><a class="dblinks" href="/pmc/articles/PMC4070524/" ref="ordinalpos=2711&amp;ncbi_uid=6289863&amp;link_uid=PMC4070524">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2712.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2709272/figure/F1/" ref="ordinalpos=2712&amp;ncbi_uid=1996339&amp;link_uid=PMC2709272" href="/pmc/articles/PMC2709272/"><img src="/pmc/articles/instance/2709272/bin/nihms36363f1.gif" src-large="/pmc/articles/instance/2709272/bin/nihms36363f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2709272/" ref="ordinalpos=2712&amp;ncbi_uid=1996339&amp;link_uid=PMC2709272" image-link="/pmc/articles/PMC2709272/figure/F1/" class="imagepopup">Figure 1. Wnt/Î²-catenin <span class="highlight" style="background-color:">signaling</span> is a central <span class="highlight" style="background-color:">pathway</span> regulating adipogenesis.  From: Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> in adipogenesis and metabolism. </a></p><div class="supp"><p class="details"><div>Wnt/Î²-catenin <span class="highlight" style="background-color:">signaling</span> in preadipocytes is initiated by expression of Wnt10b, Wnt10a, and Wnt6. Binding of these Wnts to transmembrane frizzled receptors and LRP coreceptors inhibits GSK3Î² leading to hypophosphorylation and stabilization of Î²-catenin in the cytoplasm. Î²-catenin is then translocated to the nucleus where it binds TCF/LEF transcription factors and activates downstream targets to inhibit preadipocyte differentiation. Factors from other inhibitory pathways converge on the Wnt/Î²-catenin <span class="highlight" style="background-color:">pathway</span> to block adipocyte conversion. TNFÎ± signals through its receptor, TNFÎ± receptor-1, to stabilize Î²-catenin and activate downstream pathways. Testosterone inhibits adipogenesis in part by stimulating interactions between androgen receptor and Î²-catenin. This complex travels to the nucleus to promote Î²-catenin-mediated gene transcription. DKK1 and Wnt5b are transiently induced during adipogenesis and stimulate preadipocyte differentiation. DKK1 prevents Wnt <span class="highlight" style="background-color:">signaling</span> by inhibiting LRP coreceptors, while Wnt5b promotes differentiation through an unknown mechanism. As adipogenesis proceeds, the expression of another Wnt inhibitor, Cby, is induced. Cby binds Î²-catenin in the nucleus and prevents coactivation of TCF/LEF transcription factors. Finally, binding of Î²-catenin to PPARÎ³ leads to rapid degradation of Î²-catenin through a mechanism that involves the proteosome. (âˆ’) indicates factors that inhibit preadipocyte differentiation and (+) denotes factors that stimulate the process.</div></p></div><div class="aux"><div class="resc">Tyler C. Prestwich, et al. Curr Opin Cell Biol. ;19(6):612-617.</div><p class="links"><a class="dblinks" href="/pubmed/17997088" ref="ordinalpos=2712&amp;ncbi_uid=1996339&amp;link_uid=17997088">Citation</a><a class="dblinks" href="/pmc/articles/PMC2709272/" ref="ordinalpos=2712&amp;ncbi_uid=1996339&amp;link_uid=PMC2709272">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2713.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3760587/figure/F2/" ref="ordinalpos=2713&amp;ncbi_uid=5265719&amp;link_uid=PMC3760587" href="/pmc/articles/PMC3760587/"><img src="/pmc/articles/instance/3760587/bin/nihms366709f2.gif" src-large="/pmc/articles/instance/3760587/bin/nihms366709f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3760587/" ref="ordinalpos=2713&amp;ncbi_uid=5265719&amp;link_uid=PMC3760587" image-link="/pmc/articles/PMC3760587/figure/F2/" class="imagepopup">Figure 2.  From: Chemokines: key players in cancer progression and metastasis. </a></p><div class="supp"><p class="details"><div>Summary of major signal transduction pathways downstream of chemokine receptors. This simplified figure depicts <span class="highlight" style="background-color:">signaling</span> molecules activated following chemokine receptor activation. The binding of chemokines to their respective receptors results in the activation of three main downstream <span class="highlight" style="background-color:">signaling</span> pathways to various cellular responses. The classical G-protein <span class="highlight" style="background-color:">pathway</span> activates various downstream systems including PLC, PI3K, FAK and MAPK, leading to cell adhesion, polarization and chemotaxis. Inactivating GiÎ± using PTX can be inhibite this <span class="highlight" style="background-color:">pathway</span>. The activation of the JAK/STAT <span class="highlight" style="background-color:">pathway</span> being initiated following the tyrosine phosphorylation of activated chemokine receptors has been documented. The <span class="highlight" style="background-color:">signaling</span> pathways initiated by phosphorylation at serine/threonine residues on the COOH-terminus region involve the activities of GRKs and Î²-arrestins. This <span class="highlight" style="background-color:">pathway</span> is responsible for internalization and recycling/degradation of the activated receptors. Arrows indicate â€œactivated downstreamâ€. Question marks indicate unclear downstream <span class="highlight" style="background-color:">signaling</span> pathways. FAK focal adhesion kinase, PLC phospholipase C, PI3K phosphatidylinositol 3-kinase, MAPK mitogen-activated protein kinase, PTX pertussis toxin, JAK/STAT Janus kinase/Signal transducer and activator of transcription, GRKs G-protein-coupled receptor kinases.</div></p></div><div class="aux"><div class="resc">Rajesh Singh, et al. Front Biosci (Schol Ed). 2011 June 1;3:1569-1582.</div><p class="links"><a class="dblinks" href="/pubmed/21622291" ref="ordinalpos=2713&amp;ncbi_uid=5265719&amp;link_uid=21622291">Citation</a><a class="dblinks" href="/pmc/articles/PMC3760587/" ref="ordinalpos=2713&amp;ncbi_uid=5265719&amp;link_uid=PMC3760587">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2714.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3787240/figure/Fig2/" ref="ordinalpos=2714&amp;ncbi_uid=6299736&amp;link_uid=PMC3787240" href="/pmc/articles/PMC3787240/"><img src="/pmc/articles/instance/3787240/bin/12248_2013_9504_Fig2_HTML.gif" src-large="/pmc/articles/instance/3787240/bin/12248_2013_9504_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3787240/" ref="ordinalpos=2714&amp;ncbi_uid=6299736&amp;link_uid=PMC3787240" image-link="/pmc/articles/PMC3787240/figure/Fig2/" class="imagepopup">Fig. 2.  From: Phytochemicals from Cruciferous Vegetables, Epigenetics, and Prostate Cancer Prevention. </a></p><div class="supp"><p class="details"><div>Selected non-epigenetic effects of sulforaphane and I3C/DIM on prostate cancer cells. Sulforaphane (<i>SFN</i>) and I3C/DIM inhibit the Akt <span class="highlight" style="background-color:">signaling</span> axis, a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> often hyperactive in prostate cancer. Inhibition of this <span class="highlight" style="background-color:">pathway</span> decreases pro-survival <span class="highlight" style="background-color:">signaling</span> by <i>mTOR</i>, <i>Akt</i>, and <i>NFkB</i>. Sulforaphane and I3C/DIM treatment also lead to changes in gene expression (<i>blue arrow</i>) that trigger growth arrest and apoptosis. The expression of proteins controlling the cell cycle (e.g., <i>p21</i>, <i>p27</i>, and <i>CDK6</i>) is altered to effect growth arrest, and apoptosis is finally induced through the mitochondrial <span class="highlight" style="background-color:">pathway</span>. Abbreviations: <i>AR</i> androgen receptor, <i>CDK6</i> cyclin-dependant kinase 6, <i>IAP</i> inhibitors of apoptosis</div></p></div><div class="aux"><div class="resc">Gregory W. Watson, et al. AAPS J. 2013 October;15(4):951-961.</div><p class="links"><a class="dblinks" href="/pubmed/23800833" ref="ordinalpos=2714&amp;ncbi_uid=6299736&amp;link_uid=23800833">Citation</a><a class="dblinks" href="/pmc/articles/PMC3787240/" ref="ordinalpos=2714&amp;ncbi_uid=6299736&amp;link_uid=PMC3787240">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2715.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2912210/figure/F1/" ref="ordinalpos=2715&amp;ncbi_uid=2395789&amp;link_uid=PMC2912210" href="/pmc/articles/PMC2912210/"><img src="/pmc/articles/instance/2912210/bin/nihms216665f1.gif" src-large="/pmc/articles/instance/2912210/bin/nihms216665f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2912210/" ref="ordinalpos=2715&amp;ncbi_uid=2395789&amp;link_uid=PMC2912210" image-link="/pmc/articles/PMC2912210/figure/F1/" class="imagepopup">Figure 1. The RAS-RAF-MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Targeting the mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span>: physiological feedback and drug response. </a></p><div class="supp"><p class="details"><div>The classical MAP kinase <span class="highlight" style="background-color:">pathway</span> is activated in human tumors by several mechanisms including the binding of ligand to receptor tyrosine kinases (RTKs), mutational activation of an RTK, by loss of the tumor suppressor NF1, or by mutations in RAS, BRAF and MEK1. Phosphorylation and thus activation of ERK regulates transcription of target genes which promote cell cycle progression and tumor survival. The ERK <span class="highlight" style="background-color:">pathway</span> contains a classical feedback loop in which the expression of feedback elements such as SPRY and DUSP family proteins are regulated by the level of ERK activity. Loss of expression of SPRY and DUSP family members due to promoter methylation or deletion is thus permissive for persistently elevated <span class="highlight" style="background-color:">pathway</span> output. In the case of tumors with <sup>V600E</sup>BRAF expression, <span class="highlight" style="background-color:">pathway</span> output is enhanced by impaired upstream feedback regulation.</div></p></div><div class="aux"><div class="resc">Christine A. Pratilas, et al. Clin Cancer Res. ;16(13):3329-3334.</div><p class="links"><a class="dblinks" href="/pubmed/20472680" ref="ordinalpos=2715&amp;ncbi_uid=2395789&amp;link_uid=20472680">Citation</a><a class="dblinks" href="/pmc/articles/PMC2912210/" ref="ordinalpos=2715&amp;ncbi_uid=2395789&amp;link_uid=PMC2912210">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2716.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4211160/figure/fig3/" ref="ordinalpos=2716&amp;ncbi_uid=6803775&amp;link_uid=PMC4211160" href="/pmc/articles/PMC4211160/"><img src="/pmc/articles/instance/4211160/bin/MI2014-708746.003.gif" src-large="/pmc/articles/instance/4211160/bin/MI2014-708746.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4211160/" ref="ordinalpos=2716&amp;ncbi_uid=6803775&amp;link_uid=PMC4211160" image-link="/pmc/articles/PMC4211160/figure/fig3/" class="imagepopup">Figure 3.  From: A3 Adenosine Receptor Allosteric Modulator Induces an Anti-Inflammatory Effect: In Vivo Studies and Molecular Mechanism of Action. </a></p><div class="supp"><p class="details"><div>LUF6000 downregulates the NF-<i>Îº</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via the A<sub>3</sub>AR. Protein analysis was performed using Western blot (WB) analysis in both PAW and PBMCs. LUF6000 downregulates A<sub>3</sub>AR expression levels in both PAW and PBMCs (a). PI3K, IKK, I<i>Îº</i>B, and NF-<i>Îº</i>B, all members of the NF-<i>Îº</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, were downregulated upon LUF6000 administration (b).</div></p></div><div class="aux"><div class="resc">Shira Cohen, et al. Mediators Inflamm. 2014;2014:708746.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2716&amp;ncbi_uid=6803775&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4211160/" ref="ordinalpos=2716&amp;ncbi_uid=6803775&amp;link_uid=PMC4211160">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2717.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2946353/figure/F4/" ref="ordinalpos=2717&amp;ncbi_uid=2525618&amp;link_uid=PMC2946353" href="/pmc/articles/PMC2946353/"><img src="/pmc/articles/instance/2946353/bin/1471-2105-11-469-4.gif" src-large="/pmc/articles/instance/2946353/bin/1471-2105-11-469-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2946353/" ref="ordinalpos=2717&amp;ncbi_uid=2525618&amp;link_uid=PMC2946353" image-link="/pmc/articles/PMC2946353/figure/F4/" class="imagepopup">Figure 4.  From: MetNetGE: interactive views of biological networks and ontologies. </a></p><div class="supp"><p class="details"><div><b>Visualize one <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The <span class="highlight" style="background-color:">pathway</span> 'ethylene <span class="highlight" style="background-color:">signaling</span>' is viewed using the yFiles Organic layout in Cytoscape (left) and the 3D tiered layout in MetNetGE (right). The 3D tiered layout reveals several interesting features which can not be easily seen from the 'Organic' layout. For example, there are two metabolites (ethylene and ATP) that regulate many proteins, and one protein (erf1) that activates many RNAs.</div></p></div><div class="aux"><div class="resc">Ming Jia, et al. BMC Bioinformatics. 2010;11:469-469.</div><p class="links"><a class="dblinks" href="/pubmed/20849585" ref="ordinalpos=2717&amp;ncbi_uid=2525618&amp;link_uid=20849585">Citation</a><a class="dblinks" href="/pmc/articles/PMC2946353/" ref="ordinalpos=2717&amp;ncbi_uid=2525618&amp;link_uid=PMC2946353">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2718.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766920/figure/F4/" ref="ordinalpos=2718&amp;ncbi_uid=2129893&amp;link_uid=PMC2766920" href="/pmc/articles/PMC2766920/"><img src="/pmc/articles/instance/2766920/bin/nihms148407f4.gif" src-large="/pmc/articles/instance/2766920/bin/nihms148407f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766920/" ref="ordinalpos=2718&amp;ncbi_uid=2129893&amp;link_uid=PMC2766920" image-link="/pmc/articles/PMC2766920/figure/F4/" class="imagepopup">Figure 4.  From: Pharmacologic intervention targeting glycolytic-related pathways protects against retinal injury due to ischemia and reperfusion. </a></p><div class="supp"><p class="details"><div>PANTHER classification by <span class="highlight" style="background-color:">pathway</span> (A), biological process (B) and molecular function (C). (Heterotrimeric G-protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>-1* is G<sub>i</sub> alpha and G<sub>s</sub> alpha mediated; Heterotrimeric G-protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>-2* is G<sub>q</sub> alpha and G<sub>o</sub> alpha mediated; Heterotrimeric G-protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>-3* involves in rod outer segment phototransduction)</div></p></div><div class="aux"><div class="resc">Ling Zheng, et al. Proteomics. ;9(7):1869-1882.</div><p class="links"><a class="dblinks" href="/pubmed/19288518" ref="ordinalpos=2718&amp;ncbi_uid=2129893&amp;link_uid=19288518">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766920/" ref="ordinalpos=2718&amp;ncbi_uid=2129893&amp;link_uid=PMC2766920">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2719.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3093809/figure/F1/" ref="ordinalpos=2719&amp;ncbi_uid=2953710&amp;link_uid=PMC3093809" href="/pmc/articles/PMC3093809/"><img src="/pmc/articles/instance/3093809/bin/cbt1007_0658_fig001.gif" src-large="/pmc/articles/instance/3093809/bin/cbt1007_0658_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3093809/" ref="ordinalpos=2719&amp;ncbi_uid=2953710&amp;link_uid=PMC3093809" image-link="/pmc/articles/PMC3093809/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Signaling</span> pathways in renal cell carcinoma. </a></p><div class="supp"><p class="details"><div>Schematic representation of selected <span class="highlight" style="background-color:">signaling</span> pathways and the current targeted therapies related to RCC. Angiogenic and cell proliferating <span class="highlight" style="background-color:">signaling</span> cascades are upregulated in RCC tumor cells. VEGF and other related growth factors secreted by tumor cells stimulate angiogenic <span class="highlight" style="background-color:">signaling</span> in the surrounding vascular endothelial cells. In response to growth factor <span class="highlight" style="background-color:">signaling</span> mediated through VEGF, PDGF and KIT receptors, PI-3-kinase and Ras effectors activate HIF transcription factors, which in turn switch on gene expression needed for angiogenesis and cell proliferation in endothelial cells. In addition to the angiogenic <span class="highlight" style="background-color:">pathway</span>, the Wnt and HGF pathways are also upregulated in RCC tumor cells. While VEGF-targeted therapy for metastatic RCC is focused on blocking angiogenic <span class="highlight" style="background-color:">signaling</span> in vascular endothelial cells surrounding the tumor, mTOR inhibitors act directly on the tumor cells to suppress growth.</div></p></div><div class="aux"><div class="resc">Gowrishankar Banumathy, et al. Cancer Biol Ther. 2010 October 1;10(7):658-664.</div><p class="links"><a class="dblinks" href="/pubmed/20814228" ref="ordinalpos=2719&amp;ncbi_uid=2953710&amp;link_uid=20814228">Citation</a><a class="dblinks" href="/pmc/articles/PMC3093809/" ref="ordinalpos=2719&amp;ncbi_uid=2953710&amp;link_uid=PMC3093809">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2720.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3496878/figure/F4/" ref="ordinalpos=2720&amp;ncbi_uid=4370679&amp;link_uid=PMC3496878" href="/pmc/articles/PMC3496878/"><img src="/pmc/articles/instance/3496878/bin/WJG-18-6005-g004.gif" src-large="/pmc/articles/instance/3496878/bin/WJG-18-6005-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3496878/" ref="ordinalpos=2720&amp;ncbi_uid=4370679&amp;link_uid=PMC3496878" image-link="/pmc/articles/PMC3496878/figure/F4/" class="imagepopup">Figure 4.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>/molecular targets and new targeted agents under development in hepatocellular carcinoma. </a></p><div class="supp"><p class="details"><div>Phosphoinositide 3-kinase/Akt/mammalian target of rapamycin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cell proliferation in solid cancer. PI3K: Phosphoinositide 3-kinase; mTOR: Mammalian target of rapamycin; PTEN: Phosphatase and tensin homolog; TSC: Tuberous sclerosis; FKBP12: FK506-binding protein 12.</div></p></div><div class="aux"><div class="resc">Masatoshi Kudo. World J Gastroenterol. 2012 November 14;18(42):6005-6017.</div><p class="links"><a class="dblinks" href="/pubmed/23155330" ref="ordinalpos=2720&amp;ncbi_uid=4370679&amp;link_uid=23155330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3496878/" ref="ordinalpos=2720&amp;ncbi_uid=4370679&amp;link_uid=PMC3496878">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2721.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2834833/figure/F3/" ref="ordinalpos=2721&amp;ncbi_uid=2792063&amp;link_uid=PMC2834833" href="/pmc/articles/PMC2834833/"><img src="/pmc/articles/instance/2834833/bin/nihms153308f3.gif" src-large="/pmc/articles/instance/2834833/bin/nihms153308f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2834833/" ref="ordinalpos=2721&amp;ncbi_uid=2792063&amp;link_uid=PMC2834833" image-link="/pmc/articles/PMC2834833/figure/F3/" class="imagepopup">Figure 3. Hypothetical depiction of three GSK pools and their possible involvement in regulation of diverse <span class="highlight" style="background-color:">signaling</span> pathways.  From: GSK3: a multifaceted kinase in Wnt <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Three pools of GSK3 are hypothesized to exist. One pool is associated with AXIN and probably regulated by LRP5/6. Another pool is regulated by phosphorylation at Ser9 or Ser-27 by the PI3K-AKT <span class="highlight" style="background-color:">pathway</span>. Additionally, there might be an AXIN-independent pool of GSK3, which is also regulated by Wnt. However, the mechanisms by which this pool would be regulated are not clear.</div></p></div><div class="aux"><div class="resc">Dianqing Wu, et al. Trends Biochem Sci. ;35(3):161-168.</div><p class="links"><a class="dblinks" href="/pubmed/19884009" ref="ordinalpos=2721&amp;ncbi_uid=2792063&amp;link_uid=19884009">Citation</a><a class="dblinks" href="/pmc/articles/PMC2834833/" ref="ordinalpos=2721&amp;ncbi_uid=2792063&amp;link_uid=PMC2834833">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2722.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC514948/figure/f8/" ref="ordinalpos=2722&amp;ncbi_uid=483965&amp;link_uid=PMC514948" href="/pmc/articles/PMC514948/"><img src="/pmc/articles/instance/514948/bin/7600287f8.gif" src-large="/pmc/articles/instance/514948/bin/7600287f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC514948/" ref="ordinalpos=2722&amp;ncbi_uid=483965&amp;link_uid=PMC514948" image-link="/pmc/articles/PMC514948/figure/f8/" class="imagepopup">Figure 8.  From: Anchoring of both PKA and 14-3-3 inhibits the Rho-GEF activity of the AKAP-Lbc <span class="highlight" style="background-color:">signaling</span> complex. </a></p><div class="supp"><p class="details"><div>Model for the 14-3-3-mediated inhibition of AKAP-Lbc. (<b>A</b>) AKAP-Lbc is activated in response to serum or LPA stimulation through a GÎ±12-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) Elevation of the cellular concentration of cAMP activates the PKA holoenzyme anchored to AKAP-Lbc. The catalytic subunit released from the PKA holoenzyme can phosphorylate AKAP-Lbc on serine 1565. This induces the recruitment of 14-3-3, which inhibits the Rho-GEF activity of AKAP-Lbc.</div></p></div><div class="aux"><div class="resc">Dario Diviani, et al. EMBO J. 2004 July 21;23(14):2811-2820.</div><p class="links"><a class="dblinks" href="/pubmed/15229649" ref="ordinalpos=2722&amp;ncbi_uid=483965&amp;link_uid=15229649">Citation</a><a class="dblinks" href="/pmc/articles/PMC514948/" ref="ordinalpos=2722&amp;ncbi_uid=483965&amp;link_uid=PMC514948">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2723.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4141691/figure/pone-0101725-g005/" ref="ordinalpos=2723&amp;ncbi_uid=6531522&amp;link_uid=PMC4141691" href="/pmc/articles/PMC4141691/"><img src="/pmc/articles/instance/4141691/bin/pone.0101725.g005.gif" src-large="/pmc/articles/instance/4141691/bin/pone.0101725.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4141691/" ref="ordinalpos=2723&amp;ncbi_uid=6531522&amp;link_uid=PMC4141691" image-link="/pmc/articles/PMC4141691/figure/pone-0101725-g005/" class="imagepopup">Figure 5. Differentially expressed miRNAs and their target genes in the PI3k/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Genome-Wide Analysis of miRNA Signature in the APPswe/PS1?E9 Mouse Model of Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>Joint analysis of KEGG and different algorithms was performed to identify the putative and experimentally validated targets for the most differently expressed miRNAs in two groups. Part of the <span class="highlight" style="background-color:">pathway</span> was displayed here. MiRNAs marked red are up-regulated while the blue ones are down-regulated.</div></p></div><div class="aux"><div class="resc">Hongxue Luo, et al. PLoS One. 2014;9(8):e101725.</div><p class="links"><a class="dblinks" href="/pubmed/25148207" ref="ordinalpos=2723&amp;ncbi_uid=6531522&amp;link_uid=25148207">Citation</a><a class="dblinks" href="/pmc/articles/PMC4141691/" ref="ordinalpos=2723&amp;ncbi_uid=6531522&amp;link_uid=PMC4141691">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2724.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3655553/figure/F6/" ref="ordinalpos=2724&amp;ncbi_uid=4903348&amp;link_uid=PMC3655553" href="/pmc/articles/PMC3655553/"><img src="/pmc/articles/instance/3655553/bin/nihms264626f6.gif" src-large="/pmc/articles/instance/3655553/bin/nihms264626f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3655553/" ref="ordinalpos=2724&amp;ncbi_uid=4903348&amp;link_uid=PMC3655553" image-link="/pmc/articles/PMC3655553/figure/F6/" class="imagepopup">Fig. 6. Mechanism of AC5-mediated regulation of longevity.  From: Modulation of ?-Adrenergic Receptor <span class="highlight" style="background-color:">Signaling</span> in Heart Failure and Longevity: Targeting Adenylyl Cyclase Type 5. </a></p><div class="supp"><p class="details"><div>Disruption of AC5 activates the MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [179]. The activation of ERK leads to increased expression of manganese superoxide dismutase (MnSOD), which results in inhibition of apoptosis. On the other hand, the ERK <span class="highlight" style="background-color:">signaling</span> activates the cell survival <span class="highlight" style="background-color:">signaling</span> (ref). Taken together, these mechanisms contribute to increased longevity in AC5KO mice. Western blots demonstrate the changes of key enzymes in the heart by disruption of AC5 [179]. Figures used and modified with permission</div></p></div><div class="aux"><div class="resc">David Ho, et al. Heart Fail Rev. 2010 September;15(5):495-512.</div><p class="links"><a class="dblinks" href="/pubmed/20658186" ref="ordinalpos=2724&amp;ncbi_uid=4903348&amp;link_uid=20658186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3655553/" ref="ordinalpos=2724&amp;ncbi_uid=4903348&amp;link_uid=PMC3655553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2725.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4079829/figure/F9/" ref="ordinalpos=2725&amp;ncbi_uid=6323108&amp;link_uid=PMC4079829" href="/pmc/articles/PMC4079829/"><img src="/pmc/articles/instance/4079829/bin/1744-8069-10-36-9.gif" src-large="/pmc/articles/instance/4079829/bin/1744-8069-10-36-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4079829/" ref="ordinalpos=2725&amp;ncbi_uid=6323108&amp;link_uid=PMC4079829" image-link="/pmc/articles/PMC4079829/figure/F9/" class="imagepopup">Figure 9.  From: Peripheral administration of morphine attenuates postincisional pain by regulating macrophage polarization through COX-2-dependent <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of morphine-induced analgesia representing the molecular pathways in macrophages.</b> Peripherally administered morphine alters macrophage polarity through a COX2/PGE2-dependent <span class="highlight" style="background-color:">pathway</span>. Morphine-induced changes in macrophage polarity decrease IL-1Î² levels, and increase PGE2 and enkephalin in the late phase. HO-1 might be upstream of Î¼ opioid receptor <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Kohei Godai, et al. Mol Pain. 2014;10:36-36.</div><p class="links"><a class="dblinks" href="/pubmed/24928142" ref="ordinalpos=2725&amp;ncbi_uid=6323108&amp;link_uid=24928142">Citation</a><a class="dblinks" href="/pmc/articles/PMC4079829/" ref="ordinalpos=2725&amp;ncbi_uid=6323108&amp;link_uid=PMC4079829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2726.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2988350/figure/F8/" ref="ordinalpos=2726&amp;ncbi_uid=3030909&amp;link_uid=PMC2988350" href="/pmc/articles/PMC2988350/"><img src="/pmc/articles/instance/2988350/bin/zbc0501039980008.gif" src-large="/pmc/articles/instance/2988350/bin/zbc0501039980008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2988350/" ref="ordinalpos=2726&amp;ncbi_uid=3030909&amp;link_uid=PMC2988350" image-link="/pmc/articles/PMC2988350/figure/F8/" class="imagepopup">FIGURE 8. From: Mutant Huntingtin Alters Cell Fate in Response to Microtubule                     Depolymerization via the GEF-H1-RhoA-ERK <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Model of the cell survival <span class="highlight" style="background-color:">pathway</span> activated in mutant htt                                         cells upon MT depolymerization.</b> MT depolymerization                                     releases GEF-H1 that activates RhoA by inducing exchange of GTP                                     for GDP on RhoA. GTP-bound RhoA activates downstream ROCK, which                                     up-regulates CTGF and activates ERK survival <span class="highlight" style="background-color:">signaling</span>. It is                                     possible that mediators (<i>X</i>), in addition to                                     CTGF, link GEF-H1-Rho-ROCK to ERK.</div></p></div><div class="aux"><div class="resc">Hemant Varma, et al. J Biol Chem. 2010 November 26;285(48):37445-37457.</div><p class="links"><a class="dblinks" href="/pubmed/20858895" ref="ordinalpos=2726&amp;ncbi_uid=3030909&amp;link_uid=20858895">Citation</a><a class="dblinks" href="/pmc/articles/PMC2988350/" ref="ordinalpos=2726&amp;ncbi_uid=3030909&amp;link_uid=PMC2988350">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2727.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267377/figure/F3/" ref="ordinalpos=2727&amp;ncbi_uid=4588697&amp;link_uid=PMC3267377" href="/pmc/articles/PMC3267377/"><img src="/pmc/articles/instance/3267377/bin/nihms334766f3.gif" src-large="/pmc/articles/instance/3267377/bin/nihms334766f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267377/" ref="ordinalpos=2727&amp;ncbi_uid=4588697&amp;link_uid=PMC3267377" image-link="/pmc/articles/PMC3267377/figure/F3/" class="imagepopup">Fig. 3. AopP does not inhibit MAPK <span class="highlight" style="background-color:">pathway</span> activation in Drosophila.  From: Aeromonas salmonicida secreted protein AopP is a potent inducer of apoptosis in a mammalian and a Drosophila model. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Phenotypes of adult <i>Drosophila</i> eyes constitutively expressing AopP or VopP in combination with activators of ERK (Rolled (<i>rl</i>) in <i>Drosophila</i>) <span class="highlight" style="background-color:">pathway</span>. Note, the AopP induced rough eye phenotype is not rescued by the activation of the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, whereas the VopA induced rough eye is completely rescued by the activation of the ERK <span class="highlight" style="background-color:">pathway</span> at the level of Rolled.<br /><b>(B)</b> Phenotypes of adult <i>Drosophila</i> eyes resulting from constitutive expression of the JNK <span class="highlight" style="background-color:">pathway</span> activator Eiger (<i>Drosophila</i> homolouge of mammalian TNF), and from a co-expression of Eiger with the indicated bacterial protein. Note that AopP only results in a slight increase in eye size whereas AvrA, a potent inhibitor of JNK activity, completely rescues the Eiger-induced small eye phenotype. Also, note appearance of black necrotic-like tissue the eye where Eiger and AopP are co-expressed.</div></p></div><div class="aux"><div class="resc">Rheinallt M. Jones, et al. Cell Microbiol. ;14(2):274-285.</div><p class="links"><a class="dblinks" href="/pubmed/22040305" ref="ordinalpos=2727&amp;ncbi_uid=4588697&amp;link_uid=22040305">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267377/" ref="ordinalpos=2727&amp;ncbi_uid=4588697&amp;link_uid=PMC3267377">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2728.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4053524/figure/pone-0099728-g003/" ref="ordinalpos=2728&amp;ncbi_uid=6263817&amp;link_uid=PMC4053524" href="/pmc/articles/PMC4053524/"><img src="/pmc/articles/instance/4053524/bin/pone.0099728.g003.gif" src-large="/pmc/articles/instance/4053524/bin/pone.0099728.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4053524/" ref="ordinalpos=2728&amp;ncbi_uid=6263817&amp;link_uid=PMC4053524" image-link="/pmc/articles/PMC4053524/figure/pone-0099728-g003/" class="imagepopup">Figure 3. Evaluation of the effects of uridine on liver heme biosynthesis and insulin <span class="highlight" style="background-color:">signaling</span> activity..  From: Uridine Affects Liver Protein Glycosylation, Insulin <span class="highlight" style="background-color:">Signaling</span>, and Heme Biosynthesis. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Western blots of proteins participating in the regulation of heme biosynthesis and catabolism. (<b>B</b>) Quantitative analysis of protein expression levels of Western blot data presented in (<b>A</b>). (<b>C</b>) Western blots of proteins participating in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>D</b>) Quantitative analysis of protein expression and phosphorylation levels of Western blot data presented in (<b>C</b>). Error bars are standard deviations across 9 liver samples evaluated. Asterisks indicate P&lt;0.05 versus untreated control C57BL/6J mice.</div></p></div><div class="aux"><div class="resc">Yasuyo Urasaki, et al. PLoS One. 2014;9(6):e99728.</div><p class="links"><a class="dblinks" href="/pubmed/24918436" ref="ordinalpos=2728&amp;ncbi_uid=6263817&amp;link_uid=24918436">Citation</a><a class="dblinks" href="/pmc/articles/PMC4053524/" ref="ordinalpos=2728&amp;ncbi_uid=6263817&amp;link_uid=PMC4053524">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2729.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4253612/figure/Fig3/" ref="ordinalpos=2729&amp;ncbi_uid=6903857&amp;link_uid=PMC4253612" href="/pmc/articles/PMC4253612/"><img src="/pmc/articles/instance/4253612/bin/13059_2014_533_Fig3_HTML.gif" src-large="/pmc/articles/instance/4253612/bin/13059_2014_533_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4253612/" ref="ordinalpos=2729&amp;ncbi_uid=6903857&amp;link_uid=PMC4253612" image-link="/pmc/articles/PMC4253612/figure/Fig3/" class="imagepopup">Figure 3.  From: Integrated analyses of DNA methylation and hydroxymethylation reveal tumor suppressive roles of ECM1, ATF5, and EOMES in human hepatocellular carcinoma. </a></p><div class="supp"><p class="details"><div> <b>Genes in cancer-related pathways.</b> Genes in the â€˜Focal adhesionâ€™, â€˜Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>â€™ and â€˜MAPK <span class="highlight" style="background-color:">signaling</span>â€™ <span class="highlight" style="background-color:">pathway</span>. The cyan rectangle indicates differentially expressed genes (DE genes), and the red rectangle indicates genes with a differentially methylated region (DMR gene).</div></p></div><div class="aux"><div class="resc">Fei Gao, et al. Genome Biol. 2014;15(12):533.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2729&amp;ncbi_uid=6903857&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4253612/" ref="ordinalpos=2729&amp;ncbi_uid=6903857&amp;link_uid=PMC4253612">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2730.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3890531/figure/F4/" ref="ordinalpos=2730&amp;ncbi_uid=5704230&amp;link_uid=PMC3890531" href="/pmc/articles/PMC3890531/"><img src="/pmc/articles/instance/3890531/bin/1471-2164-14-904-4.gif" src-large="/pmc/articles/instance/3890531/bin/1471-2164-14-904-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3890531/" ref="ordinalpos=2730&amp;ncbi_uid=5704230&amp;link_uid=PMC3890531" image-link="/pmc/articles/PMC3890531/figure/F4/" class="imagepopup">Figure 4.  From: An overview of gene expression dynamics during early ovarian folliculogenesis: specificity of follicular compartments and bi-directional dialog. </a></p><div class="supp"><p class="details"><div><b>IGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Gene expression involved in the IGF1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes in red are over expressed in oocytes. Genes in green are over expressed in GCs. Relative expression of IGF1 and IGF1R from RNAseq data in the graph. The Y axis corresponds to RNA-seq normalized read counts.</div></p></div><div class="aux"><div class="resc">Agnes Bonnet, et al. BMC Genomics. 2013;14:904-904.</div><p class="links"><a class="dblinks" href="/pubmed/24350644" ref="ordinalpos=2730&amp;ncbi_uid=5704230&amp;link_uid=24350644">Citation</a><a class="dblinks" href="/pmc/articles/PMC3890531/" ref="ordinalpos=2730&amp;ncbi_uid=5704230&amp;link_uid=PMC3890531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2731.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3282946/figure/f4/" ref="ordinalpos=2731&amp;ncbi_uid=3815016&amp;link_uid=PMC3282946" href="/pmc/articles/PMC3282946/"><img src="/pmc/articles/instance/3282946/bin/srep00282-f4.gif" src-large="/pmc/articles/instance/3282946/bin/srep00282-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3282946/" ref="ordinalpos=2731&amp;ncbi_uid=3815016&amp;link_uid=PMC3282946" image-link="/pmc/articles/PMC3282946/figure/f4/" class="imagepopup">Figure 4.  From: Identification of links between small molecules and miRNAs in human cancers based on transcriptional responses. </a></p><div class="supp"><p class="details"><div>(a) miR-23b and tretinoin, which was associated with this miRNA in thyroid cancer, shared the common target genes, ALDH1A1 and ALDH1A2. (b) The MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that was enriched with the common targets of miR-19a, miR-19b and miR-23b. The red nodes represented these common targets. (c) The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that was enriched with the common targets of miR-19a, miR-19b and miR-23b. The red nodes represented these common targets.</div></p></div><div class="aux"><div class="resc">Wei Jiang, et al. Sci Rep. 2012;2:282.</div><p class="links"><a class="dblinks" href="/pubmed/22355792" ref="ordinalpos=2731&amp;ncbi_uid=3815016&amp;link_uid=22355792">Citation</a><a class="dblinks" href="/pmc/articles/PMC3282946/" ref="ordinalpos=2731&amp;ncbi_uid=3815016&amp;link_uid=PMC3282946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2732.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC282807/figure/fig7/" ref="ordinalpos=2732&amp;ncbi_uid=317153&amp;link_uid=PMC282807" href="/pmc/articles/PMC282807/"><img src="/pmc/articles/instance/282807/bin/015685f7.gif" src-large="/pmc/articles/instance/282807/bin/015685f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC282807/" ref="ordinalpos=2732&amp;ncbi_uid=317153&amp;link_uid=PMC282807" image-link="/pmc/articles/PMC282807/figure/fig7/" class="imagepopup">Figure 7. From: Ethylene Regulates Arabidopsis Development via the Modulation of DELLA Protein Growth Repressor Function. </a></p><div class="supp"><p class="details"><div>DELLA Proteins Integrate the Effects on Growth of the Ethylene, Auxin, and GA <span class="highlight" style="background-color:">Signaling</span> Pathways.<br />Scheme of growth regulation via the ethylene (ETR-CTR1) and auxin <span class="highlight" style="background-color:">signaling</span> pathways, showing how DELLA proteins integrate information from both pathways and act as regulators of growth. The DELLA proteins restrain growth, and growth is relieved of restraint by GA-promoted DELLA protein degradation. Ethylene (perhaps via an auxin-dependent <span class="highlight" style="background-color:">pathway</span>) and auxin (via an ethylene-independent <span class="highlight" style="background-color:">pathway</span>) both alter the abundance/stability of DELLA.</div></p></div><div class="aux"><div class="resc">Patrick Achard, et al. Plant Cell. 2003 December;15(12):2816-2825.</div><p class="links"><a class="dblinks" href="/pubmed/14615596" ref="ordinalpos=2732&amp;ncbi_uid=317153&amp;link_uid=14615596">Citation</a><a class="dblinks" href="/pmc/articles/PMC282807/" ref="ordinalpos=2732&amp;ncbi_uid=317153&amp;link_uid=PMC282807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2733.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4383490/figure/fig2/" ref="ordinalpos=2733&amp;ncbi_uid=7417398&amp;link_uid=PMC4383490" href="/pmc/articles/PMC4383490/"><img src="/pmc/articles/instance/4383490/bin/MI2015-807976.002.gif" src-large="/pmc/articles/instance/4383490/bin/MI2015-807976.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4383490/" ref="ordinalpos=2733&amp;ncbi_uid=7417398&amp;link_uid=PMC4383490" image-link="/pmc/articles/PMC4383490/figure/fig2/" class="imagepopup">Figure 2.  From: Role of IL-38 and Its Related Cytokines in Inflammation. </a></p><div class="supp"><p class="details"><div>Receptor and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of IL-38.</div></p></div><div class="aux"><div class="resc">Xianli Yuan, et al. Mediators Inflamm. 2015;2015:807976.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2733&amp;ncbi_uid=7417398&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4383490/" ref="ordinalpos=2733&amp;ncbi_uid=7417398&amp;link_uid=PMC4383490">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2734.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4248487/figure/F3/" ref="ordinalpos=2734&amp;ncbi_uid=6913829&amp;link_uid=PMC4248487" href="/pmc/articles/PMC4248487/"><img src="/pmc/articles/instance/4248487/bin/IJD-59-531-g003.gif" src-large="/pmc/articles/instance/4248487/bin/IJD-59-531-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4248487/" ref="ordinalpos=2734&amp;ncbi_uid=6913829&amp;link_uid=PMC4248487" image-link="/pmc/articles/PMC4248487/figure/F3/" class="imagepopup">Figure 3.  From: Novel Medical Strategies Combating Nonmelanoma Skin Cancer. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Prasan R Bhandari, et al. Indian J Dermatol. 2014 Nov-Dec;59(6):531-546.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2734&amp;ncbi_uid=6913829&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4248487/" ref="ordinalpos=2734&amp;ncbi_uid=6913829&amp;link_uid=PMC4248487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2735.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3784586/figure/fig7/" ref="ordinalpos=2735&amp;ncbi_uid=6449536&amp;link_uid=PMC3784586" href="/pmc/articles/PMC3784586/"><img src="/pmc/articles/instance/3784586/bin/pc_113886_f7.gif" src-large="/pmc/articles/instance/3784586/bin/pc_113886_f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3784586/" ref="ordinalpos=2735&amp;ncbi_uid=6449536&amp;link_uid=PMC3784586" image-link="/pmc/articles/PMC3784586/figure/fig7/" class="imagepopup">Figure 7. From: Arabidopsis Histone Methylase CAU1/PRMT5/SKB1 Acts as an Epigenetic Suppressor of the Calcium <span class="highlight" style="background-color:">Signaling</span> Gene CAS to Mediate Stomatal Closure in Response to Extracellular Calcium[W]. </a></p><div class="supp"><p class="details"><div>Analysis of Expression Levels of Ca<sup>2+</sup> Homeostasis or <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Genes.<br /><b>(A)</b> Quantitative determination of expression levels of genes involved in the IP3 <span class="highlight" style="background-color:">pathway</span> or Ca<sup>2+</sup> homeostasis.<br /><b>(B)</b> Expression of <i>CAX1</i>, <i>CAX3</i>, <i>PLC3</i>, and <i>PLC4</i> in <i>cau1</i>, <i>cas-1</i>, and <i>cau1 cas-1</i> plants. Plants were grown to 28 d old in soil.<br />The <i>y</i> axes show RNA levels normalized to those of <i>ACTIN2</i>. Values are mean Â± <span style="font-variant: small-caps" class="small-caps">se</span>, <i>n</i> = 3.</div></p></div><div class="aux"><div class="resc">Yan-Lei Fu, et al. Plant Cell. 2013 August;25(8):2878-2891.</div><p class="links"><a class="dblinks" href="/pubmed/23943859" ref="ordinalpos=2735&amp;ncbi_uid=6449536&amp;link_uid=23943859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3784586/" ref="ordinalpos=2735&amp;ncbi_uid=6449536&amp;link_uid=PMC3784586">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2736.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3162305/figure/F4/" ref="ordinalpos=2736&amp;ncbi_uid=3258992&amp;link_uid=PMC3162305" href="/pmc/articles/PMC3162305/"><img src="/pmc/articles/instance/3162305/bin/tlo0405_0301_fig004.gif" src-large="/pmc/articles/instance/3162305/bin/tlo0405_0301_fig004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3162305/" ref="ordinalpos=2736&amp;ncbi_uid=3258992&amp;link_uid=PMC3162305" image-link="/pmc/articles/PMC3162305/figure/F4/" class="imagepopup">Figure 4.  From: Expression Profiling of Vulvar Carcinoma: Clues for Deranged Extracellular Matrix Remodeling and Effects on Multiple <span class="highlight" style="background-color:">Signaling</span> Pathways Combined with Discrete Patient Subsets. </a></p><div class="supp"><p class="details"><div>Oncostatin M <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Within this <span class="highlight" style="background-color:">pathway</span>, the network of cell adhesion and extracellular remodeling is the most significantly upregulated network in VSCC. The latter was generated from critical genes most upregulated in the tumor samples, depicting the major nodes of MMPs 1, 2, and 13 and their interactors. Red depicts the genes that are upregulated in tumor samples. The intensity of the colors indicates the magnitude of regulation.</div></p></div><div class="aux"><div class="resc">Kalliopi I Pappa, et al. Transl Oncol. 2011 October;4(5):301-313.</div><p class="links"><a class="dblinks" href="/pubmed/21966547" ref="ordinalpos=2736&amp;ncbi_uid=3258992&amp;link_uid=21966547">Citation</a><a class="dblinks" href="/pmc/articles/PMC3162305/" ref="ordinalpos=2736&amp;ncbi_uid=3258992&amp;link_uid=PMC3162305">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2737.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1201572/figure/fig5/" ref="ordinalpos=2737&amp;ncbi_uid=620536&amp;link_uid=PMC1201572" href="/pmc/articles/PMC1201572/"><img src="/pmc/articles/instance/1201572/bin/zpq0340592800005.gif" src-large="/pmc/articles/instance/1201572/bin/zpq0340592800005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1201572/" ref="ordinalpos=2737&amp;ncbi_uid=620536&amp;link_uid=PMC1201572" image-link="/pmc/articles/PMC1201572/figure/fig5/" class="imagepopup">Fig. 5. From: Regulation of hunger-driven behaviors by neural ribosomal S6 kinase in Drosophila. </a></p><div class="supp"><p class="details"><div>A model for the hunger regulation of adaptive feeding behaviors in <i>Drosophila</i> larvae. Our results suggest that DILP2 neurons, and possibly together with DILP4 neurons, negatively regulate two downstream hunger-responsive feeding systems: an NPF/NPFR1-dependent <span class="highlight" style="background-color:">pathway</span> specialized for motivated feeding and an NPF/NPFR1-independent <span class="highlight" style="background-color:">pathway</span> for general enhancement of feeding rate. In DILP neurons, dS6K activity is likely to positively regulate DILP synthesis and/or release. In fed larvae, a relatively high level of DILP <span class="highlight" style="background-color:">signaling</span> suppresses the two downstream pathways. In fasted animals, hunger stimuli down-regulate dS6K activity in DILP neurons, which in turn leads to decreased DILP <span class="highlight" style="background-color:">signaling</span> and therefore disinhibition of the two pathways. The former overrides the high threshold set by the default <span class="highlight" style="background-color:">pathway</span>, enabling hungry animals to engage in motivated foraging and food selection. The latter enhances feeding rate, allowing animals to compete effectively for limited food sources. DILPs negatively regulate the NPFR1 <span class="highlight" style="background-color:">pathway</span> through the dInR/dPI3K/dS6K <span class="highlight" style="background-color:">pathway</span> in NPFR1 neurons. Our data also implicate the presence of a separate default <span class="highlight" style="background-color:">pathway</span> for ad libitum feeding of higher-quality foods (baseline feeding) by fed larvae. This default <span class="highlight" style="background-color:">pathway</span> may be largely insensitive to DILP or NPF <span class="highlight" style="background-color:">signaling</span>, because overexpression of dS6K, DILPs, or NPFR1 in fed larvae did not affect baseline feeding in the presence of the liquid food (see Figs. 1, 2, 3 and data not shown).</div></p></div><div class="aux"><div class="resc">Qi Wu, et al. Proc Natl Acad Sci U S A. 2005 September 13;102(37):13289-13294.</div><p class="links"><a class="dblinks" href="/pubmed/16150727" ref="ordinalpos=2737&amp;ncbi_uid=620536&amp;link_uid=16150727">Citation</a><a class="dblinks" href="/pmc/articles/PMC1201572/" ref="ordinalpos=2737&amp;ncbi_uid=620536&amp;link_uid=PMC1201572">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2738.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678733/figure/F1/" ref="ordinalpos=2738&amp;ncbi_uid=6159685&amp;link_uid=PMC3678733" href="/pmc/articles/PMC3678733/"><img src="/pmc/articles/instance/3678733/bin/nihms462970f1.gif" src-large="/pmc/articles/instance/3678733/bin/nihms462970f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678733/" ref="ordinalpos=2738&amp;ncbi_uid=6159685&amp;link_uid=PMC3678733" image-link="/pmc/articles/PMC3678733/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Pathway</span> Databases: Making Chemical and Biological Sense of the Genomic Data Flood. </a></p><div class="supp"><p class="details"><div>Human Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Kinase cascade components STK, LATS, MOB, and SAV1 interact, leading to the phosphorylation and cytosolic sequestration of YAP and WWTR1 transcription factors. Caspase 3 may activate this process by cleaving STK3, increasing its kinase activity. Cytosolic sequestration of YAP and WWTR1 may also be mediated by AMOT and ZO proteins.</div></p></div><div class="aux"><div class="resc">Peter Dâ€™Eustachio. Chem Biol. ;20(5):629-635.</div><p class="links"><a class="dblinks" href="/pubmed/23706629" ref="ordinalpos=2738&amp;ncbi_uid=6159685&amp;link_uid=23706629">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678733/" ref="ordinalpos=2738&amp;ncbi_uid=6159685&amp;link_uid=PMC3678733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2739.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2269037/figure/Fig2/" ref="ordinalpos=2739&amp;ncbi_uid=1317751&amp;link_uid=PMC2269037" href="/pmc/articles/PMC2269037/"><img src="/pmc/articles/instance/2269037/bin/11568_2008_9018_Fig2_HTML.gif" src-large="/pmc/articles/instance/2269037/bin/11568_2008_9018_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2269037/" ref="ordinalpos=2739&amp;ncbi_uid=1317751&amp;link_uid=PMC2269037" image-link="/pmc/articles/PMC2269037/figure/Fig2/" class="imagepopup">Fig. 2.  From: 5?-androstane-3?,17?-diol selectively activates the canonical PI3K/AKT <span class="highlight" style="background-color:">pathway</span>: a bioinformatics-based evidence for androgen-activated cytoplasmic <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Venn diagram of overlapping genes in three overrepresented canonical pathways. Genes that are shared between and among apoptosis <span class="highlight" style="background-color:">signaling</span>, PI3K/AKT <span class="highlight" style="background-color:">signaling</span>, and death receptor <span class="highlight" style="background-color:">signaling</span> pathways are shown. Bold, normal, and italic characters represent genes that are up-regulated, down-regulated, and BNT, respectively, from the array analysis</div></p></div><div class="aux"><div class="resc">Mikhail G. Dozmorov, et al. Genomic Med. 2007 September;1(3-4):139-146.</div><p class="links"><a class="dblinks" href="/pubmed/18923939" ref="ordinalpos=2739&amp;ncbi_uid=1317751&amp;link_uid=18923939">Citation</a><a class="dblinks" href="/pmc/articles/PMC2269037/" ref="ordinalpos=2739&amp;ncbi_uid=1317751&amp;link_uid=PMC2269037">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2740.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4249599/figure/Fig7/" ref="ordinalpos=2740&amp;ncbi_uid=6895271&amp;link_uid=PMC4249599" href="/pmc/articles/PMC4249599/"><img src="/pmc/articles/instance/4249599/bin/12943_2014_1387_Fig7_HTML.gif" src-large="/pmc/articles/instance/4249599/bin/12943_2014_1387_Fig7_HTML.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4249599/" ref="ordinalpos=2740&amp;ncbi_uid=6895271&amp;link_uid=PMC4249599" image-link="/pmc/articles/PMC4249599/figure/Fig7/" class="imagepopup">Figure 7.  From: Combined targeting of mTOR and c-MET <span class="highlight" style="background-color:">signaling</span> pathways for effective management of epithelioid sarcoma. </a></p><div class="supp"><p class="details"><div> <b>RAD001-induced activation of AKT and ERK is dependent on c-MET <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in EpS. A)</b> Effects of RAD001 on phosphorylation of c-MET and its downstream effectors in Asra-EPS and VAESBJ cells. Cells were treated with 1 nM RAD001 for 5â€“360 minutes. <b>B)</b> Effects of RAD001 on phosphorylation of AKT and ERK in Asra-EPS and VAESBJ cells transfected with anti-c-MET siRNAs or a control siRNA. Cells transfected with siRNAs were treated with 1 nM RAD001 for 1 hour.</div></p></div><div class="aux"><div class="resc">Yoshinori Imura, et al. Mol Cancer. 2014;13(1):185.</div><p class="links"><a class="dblinks" href="/pubmed/25098767" ref="ordinalpos=2740&amp;ncbi_uid=6895271&amp;link_uid=25098767">Citation</a><a class="dblinks" href="/pmc/articles/PMC4249599/" ref="ordinalpos=2740&amp;ncbi_uid=6895271&amp;link_uid=PMC4249599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2741.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3979993/figure/F6/" ref="ordinalpos=2741&amp;ncbi_uid=6008778&amp;link_uid=PMC3979993" href="/pmc/articles/PMC3979993/"><img src="/pmc/articles/instance/3979993/bin/ijbsv10p0404g006.gif" src-large="/pmc/articles/instance/3979993/bin/ijbsv10p0404g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3979993/" ref="ordinalpos=2741&amp;ncbi_uid=6008778&amp;link_uid=PMC3979993" image-link="/pmc/articles/PMC3979993/figure/F6/" class="imagepopup">Figure 6.  From: Andrographolide Suppress Tumor Growth by Inhibiting TLR4/NF-?B <span class="highlight" style="background-color:">Signaling</span> Activation in Insulinoma. </a></p><div class="supp"><p class="details"><div><b> TLR4/NF-ÎºB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is the dominant functional target of andrographolide</b>. (A) TLR4 deficient could only slightly inhibit the expression of p65 and cannot suppress the expression of MyD88. (B) The inhibition of MyD88 result the reduction of p65 but not TLR4 (C) p65 deficient could inhibit the expression of MyD88 but not TLR4. (D) The inhibition of TLR4, MyD88 and p65 can both suppressed cell proliferation. n=12, *<i> P</i> &lt; 0.05, **<i> P</i> &lt; 0.01 and ***<i> P</i> &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Qian-Qian Zhang, et al. Int J Biol Sci. 2014;10(4):404-414.</div><p class="links"><a class="dblinks" href="/pubmed/24719558" ref="ordinalpos=2741&amp;ncbi_uid=6008778&amp;link_uid=24719558">Citation</a><a class="dblinks" href="/pmc/articles/PMC3979993/" ref="ordinalpos=2741&amp;ncbi_uid=6008778&amp;link_uid=PMC3979993">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2742.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3496878/figure/F5/" ref="ordinalpos=2742&amp;ncbi_uid=4370678&amp;link_uid=PMC3496878" href="/pmc/articles/PMC3496878/"><img src="/pmc/articles/instance/3496878/bin/WJG-18-6005-g005.gif" src-large="/pmc/articles/instance/3496878/bin/WJG-18-6005-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3496878/" ref="ordinalpos=2742&amp;ncbi_uid=4370678&amp;link_uid=PMC3496878" image-link="/pmc/articles/PMC3496878/figure/F5/" class="imagepopup">Figure 5.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>/molecular targets and new targeted agents under development in hepatocellular carcinoma. </a></p><div class="supp"><p class="details"><div>Mammalian target of rapamycin/hypoxia-inducible factor-1/vascular endothelial growth factor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in angiogenesis in solid cancer. HIF: Hypoxia-inducible factor; VEGF: Vascular endothelial growth factor; mTOR: Mammalian target of rapamycin; FKBP12: FK506-binding protein 12.</div></p></div><div class="aux"><div class="resc">Masatoshi Kudo. World J Gastroenterol. 2012 November 14;18(42):6005-6017.</div><p class="links"><a class="dblinks" href="/pubmed/23155330" ref="ordinalpos=2742&amp;ncbi_uid=4370678&amp;link_uid=23155330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3496878/" ref="ordinalpos=2742&amp;ncbi_uid=4370678&amp;link_uid=PMC3496878">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2743.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4261510/figure/F0001/" ref="ordinalpos=2743&amp;ncbi_uid=6955599&amp;link_uid=PMC4261510" href="/pmc/articles/PMC4261510/"><img src="/pmc/articles/instance/4261510/bin/BMB-47-540-g0001.gif" src-large="/pmc/articles/instance/4261510/bin/BMB-47-540-g0001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4261510/" ref="ordinalpos=2743&amp;ncbi_uid=6955599&amp;link_uid=PMC4261510" image-link="/pmc/articles/PMC4261510/figure/F0001/" class="imagepopup">Fig. 1.Wnt/Î²-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Schematic diagram for the core components and signal transduction of Wnt/Î²-catenin <span class="highlight" style="background-color:">pathway</span>. (A) In the absence of Wnt, GSK3Î² and CK1 phosphorylate Î²-catenin degradation complex which includes APC and Axin. The phosphorylated Î²-catenin is recognized by Î²-TrCP and subsequently degraded by proteasomal <span class="highlight" style="background-color:">pathway</span>. As a result, TCF/LEF1 suppresses the expression of target genes. (B) In the presence of Wnt, binding of Wnt to Fz and its co-receptor LRP5/6 leads to phosphorylation of LRP6. Axin, itself alone or whole Î²-catenin degradation complex including Axin, translocates to the phosphorylated LRP5/6, which leads to stabilization of cytoplasmic Î²-catenin. The stabilized Î²-catenin translocates into the nucleus and interacts with TCF/LEF1, which in turn enhances the expression of target genes..  From: Cross-talk between Wnt/Î²-catenin and Hippo <span class="highlight" style="background-color:">signaling</span> pathways: a brief review. </a></p><div class="aux"><div class="resc">Minseong Kim, et al. BMB Rep. 2014 October;47(10):540-545.</div><p class="links"><a class="dblinks" href="/pubmed/25154721" ref="ordinalpos=2743&amp;ncbi_uid=6955599&amp;link_uid=25154721">Citation</a><a class="dblinks" href="/pmc/articles/PMC4261510/" ref="ordinalpos=2743&amp;ncbi_uid=6955599&amp;link_uid=PMC4261510">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2744.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3287363/figure/F7/" ref="ordinalpos=2744&amp;ncbi_uid=4576972&amp;link_uid=PMC3287363" href="/pmc/articles/PMC3287363/"><img src="/pmc/articles/instance/3287363/bin/zh10031264290007.gif" src-large="/pmc/articles/instance/3287363/bin/zh10031264290007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3287363/" ref="ordinalpos=2744&amp;ncbi_uid=4576972&amp;link_uid=PMC3287363" image-link="/pmc/articles/PMC3287363/figure/F7/" class="imagepopup">Fig. 7. From: Gestational-neonatal iron deficiency suppresses and iron treatment reactivates IGF <span class="highlight" style="background-color:">signaling</span> in developing rat hippocampus. </a></p><div class="supp"><p class="details"><div>A working model of ID/anemia-suppressed hippocampal IGF <span class="highlight" style="background-color:">signaling</span>. Hippocampal ID-lowered IGF/PI3K/Akt <span class="highlight" style="background-color:">signaling</span> (downward light gray arrows) preferentially leads to reduced neurogenesis and synaptogenesis. In contrast, PI3K-independent activation of the Erk1/2 <span class="highlight" style="background-color:">pathway</span> (upward black arrows) may overcome its PI3K-dependent repression. In an adaptive sense, this may have a stimulatory effect on promoting cellular survival. mTOR, mammalian target of rapamycin; FoxO1, forkhead box O1; Hif1Î±, hypoxia-inducible factor 1Î±.</div></p></div><div class="aux"><div class="resc">Phu V. Tran, et al. Am J Physiol Endocrinol Metab. 2012 February;302(3):E316-E324.</div><p class="links"><a class="dblinks" href="/pubmed/22068601" ref="ordinalpos=2744&amp;ncbi_uid=4576972&amp;link_uid=22068601">Citation</a><a class="dblinks" href="/pmc/articles/PMC3287363/" ref="ordinalpos=2744&amp;ncbi_uid=4576972&amp;link_uid=PMC3287363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2745.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2925626/figure/F3/" ref="ordinalpos=2745&amp;ncbi_uid=2454097&amp;link_uid=PMC2925626" href="/pmc/articles/PMC2925626/"><img src="/pmc/articles/instance/2925626/bin/zh89990943790003.gif" src-large="/pmc/articles/instance/2925626/bin/zh89990943790003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2925626/" ref="ordinalpos=2745&amp;ncbi_uid=2454097&amp;link_uid=PMC2925626" image-link="/pmc/articles/PMC2925626/figure/F3/" class="imagepopup">Figure 3.  From: Plasmodium falciparum-infected erythrocytes induce NF-?B regulated inflammatory pathways in human cerebral endothelium. </a></p><div class="supp"><p class="details"><div><b>NF-ÎºB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was the most significant canonical <span class="highlight" style="background-color:">pathway</span> in HBMEC up-regulation</b>. Genes, differentially regulated in HBMECs in response to Pf-IRBC coincubation, were clustered into canonical pathways using Ingenuity tool (Ingenuity Systems). The NF-ÎºB <span class="highlight" style="background-color:">pathway</span> was statistically most significant (<i>P</i> = .001). Colored genes were up-regulated in the presence of Pf-IRBC.</div></p></div><div class="aux"><div class="resc">Abhai K. Tripathi, et al. Blood. 2009 November 5;114(19):4243-4252.</div><p class="links"><a class="dblinks" href="/pubmed/19713460" ref="ordinalpos=2745&amp;ncbi_uid=2454097&amp;link_uid=19713460">Citation</a><a class="dblinks" href="/pmc/articles/PMC2925626/" ref="ordinalpos=2745&amp;ncbi_uid=2454097&amp;link_uid=PMC2925626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2746.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1431720/figure/F7/" ref="ordinalpos=2746&amp;ncbi_uid=750769&amp;link_uid=PMC1431720" href="/pmc/articles/PMC1431720/"><img src="/pmc/articles/instance/1431720/bin/gb-2006-7-2-r11-7.gif" src-large="/pmc/articles/instance/1431720/bin/gb-2006-7-2-r11-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1431720/" ref="ordinalpos=2746&amp;ncbi_uid=750769&amp;link_uid=PMC1431720" image-link="/pmc/articles/PMC1431720/figure/F7/" class="imagepopup">Figure 7.  From: Identification of <span class="highlight" style="background-color:">signaling</span> components required for the prediction of cytokine release in RAW 264.7 macrophages. </a></p><div class="supp"><p class="details"><div>Topologies of <span class="highlight" style="background-color:">signaling</span> networks leading to cytokine releases derived from PCR minimal models and ANOVA analysis. In each panel, nodes in the upper row represent ligands that significantly regulate respective cytokines (ANOVA). Nodes in the middle row represent significant pathways identified by PCR minimal models. Edges between top and middle rows represent significant <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulation by the given ligands (ANOVA). Edges between top and bottom rows, or middle and bottom rows, represent significant participation identified by PCR minimal models. Weak activation of <span class="highlight" style="background-color:">signaling</span> pathways is indicated by dashed edges. Light gray: pathways demonstrated in the literature to not play any role (false positives).</div></p></div><div class="aux"><div class="resc">Sylvain Pradervand, et al. Genome Biol. 2006;7(2):R11-R11.</div><p class="links"><a class="dblinks" href="/pubmed/16507166" ref="ordinalpos=2746&amp;ncbi_uid=750769&amp;link_uid=16507166">Citation</a><a class="dblinks" href="/pmc/articles/PMC1431720/" ref="ordinalpos=2746&amp;ncbi_uid=750769&amp;link_uid=PMC1431720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2747.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4001537/figure/F2/" ref="ordinalpos=2747&amp;ncbi_uid=6097874&amp;link_uid=PMC4001537" href="/pmc/articles/PMC4001537/"><img src="/pmc/articles/instance/4001537/bin/JCI71194.f2.gif" src-large="/pmc/articles/instance/4001537/bin/JCI71194.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4001537/" ref="ordinalpos=2747&amp;ncbi_uid=6097874&amp;link_uid=PMC4001537" image-link="/pmc/articles/PMC4001537/figure/F2/" class="imagepopup">Figure 2. S1P1 activation in MEFs..  From: Sphingosine-1-phosphate receptor 1 reporter mice reveal receptor activation sites in vivo. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Validation of modified S1P1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in MEFs. S1P1 GFP <span class="highlight" style="background-color:">signaling</span> MEFs were cultured for 16 hours in medium containing 10% charcoal-stripped FBS and received either S1P (10<sup>â€“7</sup> M) or vehicle (4 mg/ml BSA in PBS). Nuclei were stained with Hoechst, and MEF cultures were imaged under an inverted laser-scanning confocal microscope. The experiment was repeated twice in duplicate, and a representative result is shown. (<b>B</b>) Treatment of MEFs with RP-001. S1P1 GFP <span class="highlight" style="background-color:">signaling</span> MEFs were cultured for 16 hours in medium containing 10% charcoal-stripped FBS and received either S1P or RP-001. After 24 hours, nuclei were stained with Hoechst, and MEF cultures were imaged under an inverted laser-scanning confocal microscope. The experiment was performed in duplicate, and a representative result is shown. (<b>C</b>) Flow cytometry analysis of S1P1 GFP <span class="highlight" style="background-color:">signaling</span> MEFs. S1P1 GFP <span class="highlight" style="background-color:">signaling</span> MEFs were cultured for 16 hours in medium containing 10% charcoal-stripped FBS and various concentrations of S1P were added. After 24 hours, the number of GFP<sup>+</sup> cells was determined by flow cytometry. The experiment was repeated twice. Data represent mean Â± SEM (<i>n</i> = 3). (<b>D</b>) Endogenous S1P1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in MEFs. S1P1 GFP <span class="highlight" style="background-color:">signaling</span> MEFs were cultured for 16 hours in medium containing 0.1% FBS and received 1 Î¼M of S1P1 receptor ligands (RP-001, S1P, or SEW2871) or vehicle (4 mg/ml BSA in PBS). After 10 minutes, the cell lysate was harvested and then Akt and phospho-Akt were identified by Western blotting (see complete unedited blot in the supplemental material). The experiment was performed in triplicate, and a representative result is shown. Scale bars: 100 Î¼m. </div></p></div><div class="aux"><div class="resc">Mari Kono, et al. J Clin Invest. 2014 May 1;124(5):2076-2086.</div><p class="links"><a class="dblinks" href="/pubmed/24667638" ref="ordinalpos=2747&amp;ncbi_uid=6097874&amp;link_uid=24667638">Citation</a><a class="dblinks" href="/pmc/articles/PMC4001537/" ref="ordinalpos=2747&amp;ncbi_uid=6097874&amp;link_uid=PMC4001537">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2748.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3042643/figure/fig1/" ref="ordinalpos=2748&amp;ncbi_uid=2826422&amp;link_uid=PMC3042643" href="/pmc/articles/PMC3042643/"><img src="/pmc/articles/instance/3042643/bin/PRI2011-932932.001.gif" src-large="/pmc/articles/instance/3042643/bin/PRI2011-932932.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3042643/" ref="ordinalpos=2748&amp;ncbi_uid=2826422&amp;link_uid=PMC3042643" image-link="/pmc/articles/PMC3042643/figure/fig1/" class="imagepopup">Figure 1.  From: EGFR <span class="highlight" style="background-color:">Signaling</span> in Colorectal Carcinoma. </a></p><div class="supp"><p class="details"><div>EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Ligand binding induces dimerization and activates the EGFR. Subsequent autophosphorylation of tyrosine residues activates downstream <span class="highlight" style="background-color:">signaling</span>. In the Ras-Raf-MEK-MAPK, one axis of the EGFR <span class="highlight" style="background-color:">signaling</span> cascade, an adaptor protein complex composed of growth factor receptor-bound protein 2 adapter protein (Grb2), which harbors a tyrosine phosphate-docking site, and son of sevenless (SOS), a Ras GDP/GTP exchange factor, then activates the Ras GTPase. After activation, Ras (i.e., KRas) recruits and activates the serine protein Raf (i.e., B-Raf), and subsequent phosphorylation and activation of MEK and then MAPK occurs, resulting in activation of transcription factors in the cell nucleus. The Ras-Raf-MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is thought to control cell growth, differentiation, and survival (?apoptosis). The other axis of the EGFR <span class="highlight" style="background-color:">signaling</span> cascade that is important in colorectal carcinogenesis is the PI3K-AKT <span class="highlight" style="background-color:">pathway</span>. Once the EFGR tyrosine residues are phosphorylated, PI3K is translocated to the cell membrane and binds to tyrosine phosphate (through its adaptor subunit p85) which triggers the PI3K catalytic subunit p110 to produce phosphatidylinositol-3,4,5-triphosphate (PIP3). PI3K then promotes AKT activation. Activated AKT (p-AKT), present within the cytoplasm, then activates various targets that result in cell growth, proliferation, and survival (paralleling the Ras-Raf-MEK-MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>). Importantly, these two axes are closely related and have some overlap. For example, the p110 subunit of PI3K can also be activated via interaction with Ras. Of note, phosphatase with tensin homology (PTEN) is a phosphatase that converts PIP3 back to phosphatidylinositol (4, 5) bisphosphate (PIP2), thereby negatively regulating the PI3K-AKT <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Alyssa M. Krasinskas. Patholog Res Int. 2011;2011:932932.</div><p class="links"><a class="dblinks" href="/pubmed/21403829" ref="ordinalpos=2748&amp;ncbi_uid=2826422&amp;link_uid=21403829">Citation</a><a class="dblinks" href="/pmc/articles/PMC3042643/" ref="ordinalpos=2748&amp;ncbi_uid=2826422&amp;link_uid=PMC3042643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2749.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2172008/figure/fig5/" ref="ordinalpos=2749&amp;ncbi_uid=1117677&amp;link_uid=PMC2172008" href="/pmc/articles/PMC2172008/"><img src="/pmc/articles/instance/2172008/bin/200403110f5.gif" src-large="/pmc/articles/instance/2172008/bin/200403110f5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2172008/" ref="ordinalpos=2749&amp;ncbi_uid=1117677&amp;link_uid=PMC2172008" image-link="/pmc/articles/PMC2172008/figure/fig5/" class="imagepopup">Figure 5. From: STAT3 noncell-autonomously controls planar cell polarity during zebrafish convergence and extension. </a></p><div class="supp"><p class="details"><div><b>Noncell-autonomous activity of STAT3 <span class="highlight" style="background-color:">signaling</span> is essential for the correct activation of PCP <span class="highlight" style="background-color:">signaling</span> through Dishevelled.</b> Lateral hypoblast cells expressing Î”N-Dishevelled-GFP in control embryos (A, LWR = 2.17, MLA = 70%, <i>n</i> = 100), STAT3-MO injected embryos (B, LWR = 2.31, MLA = 13%, <i>n</i> = 100), STAT3-MO injected embryos receiving shield transplantation from <i>STAT3</i> mRNA and STAT3-MO coinjected donor (C, LWR = 2.13, MLA = 72%, <i>n</i> = 100), STAT3-MO injected embryos receiving shield transplantation from <i>LIV1</i> mRNA and STAT3-MO coinjected donor (D, LWR = 2.05, MLA = 18%, <i>n</i> = 100), STAT3-MO injected embryos receiving shield transplantation from LIV1-MO injected donor embryos (E, LWR = 2.00, MLA = 73%, <i>n</i> = 100), and Strabismus-MO injected embryos (F, LWR = 1.17, MLA = 32%, <i>n</i> = 100); animal pole is up, and dorsal is to the right. Experiments were performed at least 10 times for each experiment, and similar results were obtained. (G), Model for the establishment of PCP during zebrafish convergence and extension by noncell-autonomous activity of STAT3 <span class="highlight" style="background-color:">signaling</span>. In gastrula organizer cells, STAT3 induces an as-yet-unidentified <span class="highlight" style="background-color:">signaling</span> molecule (X) that is able to noncell-autonomously activate Dishevelled-RhoA <span class="highlight" style="background-color:">signaling</span> in the PCP <span class="highlight" style="background-color:">pathway</span> in neighboring cells to establish the PCP. The converging neighboring cells may sense the extracellular gradient of <span class="highlight" style="background-color:">signaling</span> molecules through the receptor Frizzled (Fz), and translate this signal into the transmembrane protein Strabismus (Stbm), an intracellular modulator Dishevelled (Dsh), and its downstream effector RhoA. Alternatively, non-Frizzled receptors (Y) such as PTK7/CCK-4 may sense the gradient of <span class="highlight" style="background-color:">signaling</span> molecules, which are expressed in gastrula organizer by STAT3 <span class="highlight" style="background-color:">signaling</span> and noncell-autonomously activate PCP <span class="highlight" style="background-color:">signaling</span> in neighboring cells during zebrafish convergence and extension movements.</div></p></div><div class="aux"><div class="resc">Chiemi Miyagi, et al. J Cell Biol. 2004 September 27;166(7):975-981.</div><p class="links"><a class="dblinks" href="/pubmed/15452141" ref="ordinalpos=2749&amp;ncbi_uid=1117677&amp;link_uid=15452141">Citation</a><a class="dblinks" href="/pmc/articles/PMC2172008/" ref="ordinalpos=2749&amp;ncbi_uid=1117677&amp;link_uid=PMC2172008">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2750.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3919687/figure/F5/" ref="ordinalpos=2750&amp;ncbi_uid=5799946&amp;link_uid=PMC3919687" href="/pmc/articles/PMC3919687/"><img src="/pmc/articles/instance/3919687/bin/nihms547222f5.gif" src-large="/pmc/articles/instance/3919687/bin/nihms547222f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3919687/" ref="ordinalpos=2750&amp;ncbi_uid=5799946&amp;link_uid=PMC3919687" image-link="/pmc/articles/PMC3919687/figure/F5/" class="imagepopup">Fig. 5.  From: Comparative kinetic analyses of gene profiles of na?ve CD4+ and CD8+ T cells from young and old animals reveal novel age-related alterations. </a></p><div class="supp"><p class="details"><div>Kinetic changes in gene expression of selected components of the Wnt/Î²-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in aging CD8<sup>+</sup> T cells. Wnt/Î²-catenin <span class="highlight" style="background-color:">pathway</span> was generated by Ingenuity <span class="highlight" style="background-color:">Pathway</span> analysis. Adjacent to each step of the <span class="highlight" style="background-color:">pathway</span> are the gene profile plots of old and young CD8<sup>+</sup> T cells. Axis labels are the same as in Fig. 2. Red indicates up-regulated genes, blue down-regulated genes.</div></p></div><div class="aux"><div class="resc">Noweeda Mirza, et al. Aging Cell. ;10(5):853-867.</div><p class="links"><a class="dblinks" href="/pubmed/21711441" ref="ordinalpos=2750&amp;ncbi_uid=5799946&amp;link_uid=21711441">Citation</a><a class="dblinks" href="/pmc/articles/PMC3919687/" ref="ordinalpos=2750&amp;ncbi_uid=5799946&amp;link_uid=PMC3919687">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2751.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2820418/figure/F12/" ref="ordinalpos=2751&amp;ncbi_uid=2227171&amp;link_uid=PMC2820418" href="/pmc/articles/PMC2820418/"><img src="/pmc/articles/instance/2820418/bin/zmk0041093580012.gif" src-large="/pmc/articles/instance/2820418/bin/zmk0041093580012.jpg" alt="Figure 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2820418/" ref="ordinalpos=2751&amp;ncbi_uid=2227171&amp;link_uid=PMC2820418" image-link="/pmc/articles/PMC2820418/figure/F12/" class="imagepopup">Figure 12. From: Plasticity-related Gene 5 (PRG5) Induces Filopodia and Neurite Growth and Impedes Lysophosphatidic Acid- and Nogo-A-mediated Axonal Retraction. </a></p><div class="supp"><p class="details"><div>PRG5 <span class="highlight" style="background-color:">signaling</span> in axon growth and neurite retraction. PRG5 induces filopodia and axon growth independent of Cdc42 and impedes RhoA-dependent neurite retraction. (A) Proposed model of PRG5 action on filopodia and axon growth. (B) Summary and proposed model of PRG5 <span class="highlight" style="background-color:">signaling</span> in the context of neurite growth inhibitors. The common neurite growth inhibitors Nogo-A and LPA (underscored light blue) act on their receptors, and their <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> converge at the RhoA-Rho kinase <span class="highlight" style="background-color:">pathway</span>. PRG5 is localized in the plasma membrane and attenuates LPA and Nogo-Aâ€”induced neurite retraction and alleviates basal RhoA activation, whereas direct RhoA, ROCK or PIP5KÎ² activation overcomes PRG5 induced neurites. Thus, PRG5 may act upstream of RhoA interfering with Nogo-A and LPA receptor <span class="highlight" style="background-color:">signaling</span> or alternatively PRG5 directly balances or impacts RhoA activation.</div></p></div><div class="aux"><div class="resc">Thomas Broggini, et al. Mol Biol Cell. 2010 February 15;21(4):521-537.</div><p class="links"><a class="dblinks" href="/pubmed/20032306" ref="ordinalpos=2751&amp;ncbi_uid=2227171&amp;link_uid=20032306">Citation</a><a class="dblinks" href="/pmc/articles/PMC2820418/" ref="ordinalpos=2751&amp;ncbi_uid=2227171&amp;link_uid=PMC2820418">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2752.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2869521/figure/A000109F1/" ref="ordinalpos=2752&amp;ncbi_uid=2358215&amp;link_uid=PMC2869521" href="/pmc/articles/PMC2869521/"><img src="/pmc/articles/instance/2869521/bin/cshperspect-NFK-000109_F1.gif" src-large="/pmc/articles/instance/2869521/bin/cshperspect-NFK-000109_F1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2869521/" ref="ordinalpos=2752&amp;ncbi_uid=2358215&amp;link_uid=PMC2869521" image-link="/pmc/articles/PMC2869521/figure/A000109F1/" class="imagepopup">Figure 1. From: Oncogenic Activation of NF-?B. </a></p><div class="supp"><p class="details"><div>Role of BCR <span class="highlight" style="background-color:">signaling</span> to NF-ÎºB in ABC DLBCL. (<i>A</i>) Schematic of BCR <span class="highlight" style="background-color:">signaling</span> to NF-ÎºB. Recurrent genetic alterations in ABC DLBCL that result in constitutive NF-ÎºB activation are indicated by the gray boxes. Proximal <span class="highlight" style="background-color:">signaling</span> by the BCR is initiated by SRC-family kinases (SFK; e.g., LYN, FYN, FGR, and BLK), which phosphorylate the ITAM motifs in the CD79A and CD79B components of the BCR receptor. SYK is recruited to the phosphorylated ITAMs and activated to phosphorylate many downstream proteins. The PI(3) kinase <span class="highlight" style="background-color:">pathway</span> is activated by SRC-family kinase phosphorylation of the BCR coreceptor CD19. The generation of PIP3 by PI(3) kinase recruits BTK and associated BLNK and phospholipase CÎ³2 (PLCÎ³2) to the plasma membrane. PLCÎ³2 generates inositol triphosphate (IP3), which leads to opening of the capacitative calcium channel, thereby activating the NF-AT <span class="highlight" style="background-color:">pathway</span>. Diacylglycerol (DAG) is also generated, which activates protein kinase CÎ² (PKCÎ²). PKCÎ² phosphorylates the latent form of CARD11 (CARD-off) in the cytoplasm, causing it to adopt an active conformation (CARD11-on), translocate to the plasma membrane, and recruit the <span class="highlight" style="background-color:">signaling</span> adapters BCL10 and MALT1. MALT1 binds TRAF6, causing TRAF6 to catalyze K63-linked polyubiquitination of the IKKÎ³ subunit, MALT1 and itself. TAB2 recognizes the polyubiquitin chains on TRAF6, leading to phosphorylation of IKKÎ² in its activation loop by TAK1 kinase. Ubiquitination of IKKÎ³ and phosphorylation of IKKÎ² activate IKKÎ² to phosphorylate IÎºBÎ±. Phosphorylated IÎºBÎ± is ubiquitinated by the ubiquitin ligase Î²TrCP, leading to its proteasomal degradation. Nuclear NF-ÎºB heterodimers activate multiple target genes, including A20. A20 terminates NF-ÎºB <span class="highlight" style="background-color:">signaling</span> by removing K63-linked ubiquitin chains from IKKÎ³, TRAF6, and MALT1, and attaching K48-linked ubiquitin chains. (<i>B</i>) The role of chronic active BCR <span class="highlight" style="background-color:">signaling</span> in the pathogenesis of ABC DLBCL. Genetic mutations in CD79B and CD79A ITAM regions in ABC DLBCLs suggest that BCR <span class="highlight" style="background-color:">signaling</span> is key to the pathogenesis of ABC DLBCL. CD79B and CD79A ITAM mutations in the mouse cause hyperactive BCR <span class="highlight" style="background-color:">signaling</span>, suggesting that CD79 ITAM mutations in ABC DLBCL may amplify antigen-stimulated BCR <span class="highlight" style="background-color:">signaling</span>. CD79 ITAM mutations increase surface BCR expression and decrease activation of LYN kinase, a negative regulator of BCR <span class="highlight" style="background-color:">signaling</span>, potentially resulting in increased <span class="highlight" style="background-color:">signaling</span> to NF-ÎºB and greater clonal expansion. A separate step leading to chronic active BCR <span class="highlight" style="background-color:">signaling</span> in ABC DLBCL is the acquisition of spontaneous BCR clustering, which is not caused by the ITAM mutations. The BCR clustering phenotype could theoretically be acquired either before or after the CD79 mutations. Finally, ABC DLBCLs must acquire additional oncogenic hits to become fully malignant.</div></p></div><div class="aux"><div class="resc">Louis M. Staudt. Cold Spring Harb Perspect Biol. 2010 June;2(6):a000109.</div><p class="links"><a class="dblinks" href="/pubmed/20516126" ref="ordinalpos=2752&amp;ncbi_uid=2358215&amp;link_uid=20516126">Citation</a><a class="dblinks" href="/pmc/articles/PMC2869521/" ref="ordinalpos=2752&amp;ncbi_uid=2358215&amp;link_uid=PMC2869521">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2753.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3317229/figure/fig5/" ref="ordinalpos=2753&amp;ncbi_uid=3956479&amp;link_uid=PMC3317229" href="/pmc/articles/PMC3317229/"><img src="/pmc/articles/instance/3317229/bin/JST2012-463617.005.gif" src-large="/pmc/articles/instance/3317229/bin/JST2012-463617.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3317229/" ref="ordinalpos=2753&amp;ncbi_uid=3956479&amp;link_uid=PMC3317229" image-link="/pmc/articles/PMC3317229/figure/fig5/" class="imagepopup">Figure 5.  From: The Role of MAPK in Drug-Induced Kidney Injury. </a></p><div class="supp"><p class="details"><div>Alternative <span class="highlight" style="background-color:">signaling</span> in the presence of cyclosporine A. In normal, <span class="highlight" style="background-color:">signaling</span> calcineurin binds to and dephosphorylates NFAT, allowing it to enter the nucleus where it induces IL-2 expression. In the presence of cyclosporine this <span class="highlight" style="background-color:">pathway</span> is blocked by the binding of the cyclosporine/cyclophilin complex to calcineurin, which prevents the subsequent binding to NFAT. Alternative <span class="highlight" style="background-color:">signaling</span> occurs via the induction of a MAPK cascade, resulting in the expression of IL-2.</div></p></div><div class="aux"><div class="resc">Hilary Cassidy, et al. J Signal Transduct. 2012;2012:463617.</div><p class="links"><a class="dblinks" href="/pubmed/22523682" ref="ordinalpos=2753&amp;ncbi_uid=3956479&amp;link_uid=22523682">Citation</a><a class="dblinks" href="/pmc/articles/PMC3317229/" ref="ordinalpos=2753&amp;ncbi_uid=3956479&amp;link_uid=PMC3317229">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2754.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3913696/figure/pone-0087810-g008/" ref="ordinalpos=2754&amp;ncbi_uid=5791170&amp;link_uid=PMC3913696" href="/pmc/articles/PMC3913696/"><img src="/pmc/articles/instance/3913696/bin/pone.0087810.g008.gif" src-large="/pmc/articles/instance/3913696/bin/pone.0087810.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3913696/" ref="ordinalpos=2754&amp;ncbi_uid=5791170&amp;link_uid=PMC3913696" image-link="/pmc/articles/PMC3913696/figure/pone-0087810-g008/" class="imagepopup">Figure 8. CK prevents inflammatory responses via inhibiting NF-ÎºB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation..  From: Ginsenoside Metabolite Compound K Promotes Recovery of Dextran Sulfate Sodium-Induced Colitis and Inhibits Inflammatory Responses by Suppressing NF-?B Activation. </a></p><div class="supp"><p class="details"><div>CK exerts therapy effects on the progression of DSS-induced colitis in mice, and the potential mechanisms may contribute to regulating the innate immune cells such as macrophages activities through modulating NF-ÎºB <span class="highlight" style="background-color:">signaling</span> pathways activation.</div></p></div><div class="aux"><div class="resc">Juan Li, et al. PLoS One. 2014;9(2):e87810.</div><p class="links"><a class="dblinks" href="/pubmed/24504372" ref="ordinalpos=2754&amp;ncbi_uid=5791170&amp;link_uid=24504372">Citation</a><a class="dblinks" href="/pmc/articles/PMC3913696/" ref="ordinalpos=2754&amp;ncbi_uid=5791170&amp;link_uid=PMC3913696">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2755.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3397954/figure/pone-0040853-g005/" ref="ordinalpos=2755&amp;ncbi_uid=4127423&amp;link_uid=PMC3397954" href="/pmc/articles/PMC3397954/"><img src="/pmc/articles/instance/3397954/bin/pone.0040853.g005.gif" src-large="/pmc/articles/instance/3397954/bin/pone.0040853.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3397954/" ref="ordinalpos=2755&amp;ncbi_uid=4127423&amp;link_uid=PMC3397954" image-link="/pmc/articles/PMC3397954/figure/pone-0040853-g005/" class="imagepopup">Figure 5. Kinetics and dynamics of BCR-ABL <span class="highlight" style="background-color:">signaling</span> upon tyrosine-kinase inhibition..  From: Intracellular Retention of ABL Kinase Inhibitors Determines Commitment to Apoptosis in CML Cells. </a></p><div class="supp"><p class="details"><div>K562 cells were treated with increasing concentrations of tyrosine kinase inhibitors. Untreated cells served as a control. For analysis of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibition cells were either lysed for Western Blot analysis or fixed for FACS. (<b>A</b>): <i>Left panel</i>: Western Blot analysis of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibition after exposure of Ba/F3-BCR-ABL cells to imatinib for 2 h. <i>Right panel</i>: FACS analysis of STAT5 and CRKL inhibition using phosphorylation-specific antibodies. To assess <span class="highlight" style="background-color:">signaling</span> dynamics, this analysis was performed at three different time-points (0.3, 2, and 12 h). (<b>B</b>) The same analysis was performed using the second-generation BCR-ABL tyrosine kinase inhibitor dasatinib. Experiments were performed at least in triplicate and one representative experiment is shown.</div></p></div><div class="aux"><div class="resc">Daniel B. Lipka, et al. PLoS One. 2012;7(7):e40853.</div><p class="links"><a class="dblinks" href="/pubmed/22815843" ref="ordinalpos=2755&amp;ncbi_uid=4127423&amp;link_uid=22815843">Citation</a><a class="dblinks" href="/pmc/articles/PMC3397954/" ref="ordinalpos=2755&amp;ncbi_uid=4127423&amp;link_uid=PMC3397954">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2756.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3339976/figure/F6/" ref="ordinalpos=2756&amp;ncbi_uid=4796707&amp;link_uid=PMC3339976" href="/pmc/articles/PMC3339976/"><img src="/pmc/articles/instance/3339976/bin/zbc0181205740006.gif" src-large="/pmc/articles/instance/3339976/bin/zbc0181205740006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3339976/" ref="ordinalpos=2756&amp;ncbi_uid=4796707&amp;link_uid=PMC3339976" image-link="/pmc/articles/PMC3339976/figure/F6/" class="imagepopup">FIGURE 6. From: Involvement of Nuclear Factor ?B (NF-?B) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Regulation of Cardiac G Protein-coupled Receptor Kinase 5 (GRK5) Expression. </a></p><div class="supp"><p class="details"><div><b>Activation of NF-ÎºB enhances <i>GRK5</i> transcription and subsequent expression in myocytes.</b> Cardiac hypertrophic agonists activating Gq-coupled receptors activate PKC (mimicked by PMA in our study), which can lead to activation of NF-ÎºB through ubiquitination, phosphorylation, and degradation of IÎºBÎ±. Activated NF-ÎºB-p50 and NF-ÎºB-p65 proteins in cytoplasm are now free to move into nucleus where they can bind to the promoter of <i>GRK5</i>. This activated protein complex may interact with other transcription factors and coactivators in the promoter region, thereby enhancing the transcription of GRK5. This event can be blocked by inhibition of activation of the NF-ÎºB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, such as by antioxidants.</div></p></div><div class="aux"><div class="resc">Kazi N. Islam, et al. J Biol Chem. 2012 April 13;287(16):12771-12778.</div><p class="links"><a class="dblinks" href="/pubmed/22389501" ref="ordinalpos=2756&amp;ncbi_uid=4796707&amp;link_uid=22389501">Citation</a><a class="dblinks" href="/pmc/articles/PMC3339976/" ref="ordinalpos=2756&amp;ncbi_uid=4796707&amp;link_uid=PMC3339976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2757.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2893682/figure/F7-8673/" ref="ordinalpos=2757&amp;ncbi_uid=3050998&amp;link_uid=PMC2893682" href="/pmc/articles/PMC2893682/"><img src="/pmc/articles/instance/2893682/bin/zjh0071086730007.gif" src-large="/pmc/articles/instance/2893682/bin/zjh0071086730007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2893682/" ref="ordinalpos=2757&amp;ncbi_uid=3050998&amp;link_uid=PMC2893682" image-link="/pmc/articles/PMC2893682/figure/F7-8673/" class="imagepopup">Figure 7.  From: Sonic Hedgehog Acts as a Negative Regulator of ?-Catenin <span class="highlight" style="background-color:">Signaling</span> in the Adult Tongue Epithelium. </a></p><div class="supp"><p class="details"><div>Interaction scheme of Shh and activated Î²-catenin <span class="highlight" style="background-color:">signaling</span>. Schematic illustration of the hypothesized inhibition of Î²-catenin by canonical Shh <span class="highlight" style="background-color:">signaling</span> together with Jagged2-Notch2. <b>Left side:</b> If Shh binds to Ptch, it looses its inhibitory effect on SMO. This activates the Gli proteins for target gene translation (left side). The APC mutation in APC<sup>min/+</sup> leads to accumulation of cytosolic and nuclear Î²-catenin and leads to Wnt-specific target gene translation (<b>middle</b>). The activation of the Shh <span class="highlight" style="background-color:">signaling</span> is able to inhibit Î²-catenin downstream of the degradation complex. It is unknown whether this inhibition mechanism works through a novel <span class="highlight" style="background-color:">pathway</span> initiated by the receptor complex Ptch/SMO or takes place further downstream. <b>Right side:</b> JAG2 might be a second control mechanism for Î²-catenin via an unknown mechanism, as canonical Notch <span class="highlight" style="background-color:">signaling</span> is not required.</div></p></div><div class="aux"><div class="resc">Fabian T. Schneider, et al. Am J Pathol. 2010 July;177(1):404-414.</div><p class="links"><a class="dblinks" href="/pubmed/20508033" ref="ordinalpos=2757&amp;ncbi_uid=3050998&amp;link_uid=20508033">Citation</a><a class="dblinks" href="/pmc/articles/PMC2893682/" ref="ordinalpos=2757&amp;ncbi_uid=3050998&amp;link_uid=PMC2893682">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2758.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3642497/figure/F6/" ref="ordinalpos=2758&amp;ncbi_uid=4883443&amp;link_uid=PMC3642497" href="/pmc/articles/PMC3642497/"><img src="/pmc/articles/instance/3642497/bin/fncel-07-00056-g006.gif" src-large="/pmc/articles/instance/3642497/bin/fncel-07-00056-g006.jpg" alt="FIGURE 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3642497/" ref="ordinalpos=2758&amp;ncbi_uid=4883443&amp;link_uid=PMC3642497" image-link="/pmc/articles/PMC3642497/figure/F6/" class="imagepopup">FIGURE 6.  From: Methylene blue induces macroautophagy through 5? adenosine monophosphate-activated protein kinase <span class="highlight" style="background-color:">pathway</span> to protect neurons from serum deprivation. </a></p><div class="supp"><p class="details"><div><b>Methylene blue induces activation of AMPK <span class="highlight" style="background-color:">pathway</span> in mouse brain.</b> <b>(A,B)</b> Representative Western blots and quantitative analysis showed that protein levels of AMPKÎ±, AMPKÎ²1 and AMPKÎ²2, but not mTOR <span class="highlight" style="background-color:">signaling</span>, were increased at cortex in methylene blue-treated groups. Methylene blue did not significantly induce AMPK <span class="highlight" style="background-color:">signaling</span> activation in SVZ <b>(C)</b> *, <i>p</i> &lt; 0.05; **, <i>p</i> &lt; 0.01. U, vehicle control; L, low dose methylene blue; H, high dose methylene blue.</div></p></div><div class="aux"><div class="resc">Luokun Xie, et al. Front Cell Neurosci. 2013;7:56.</div><p class="links"><a class="dblinks" href="/pubmed/23653592" ref="ordinalpos=2758&amp;ncbi_uid=4883443&amp;link_uid=23653592">Citation</a><a class="dblinks" href="/pmc/articles/PMC3642497/" ref="ordinalpos=2758&amp;ncbi_uid=4883443&amp;link_uid=PMC3642497">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2759.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1574111/figure/fig3/" ref="ordinalpos=2759&amp;ncbi_uid=805269&amp;link_uid=PMC1574111" href="/pmc/articles/PMC1574111/"><img src="/pmc/articles/instance/1574111/bin/140-0705522f3.gif" src-large="/pmc/articles/instance/1574111/bin/140-0705522f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1574111/" ref="ordinalpos=2759&amp;ncbi_uid=805269&amp;link_uid=PMC1574111" image-link="/pmc/articles/PMC1574111/figure/fig3/" class="imagepopup">Figure 3.  From: FK506 potentiates NGF-induced neurite outgrowth via the Ras/Raf/MAP kinase <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed mechanism for FK506-potentiated neurite outgrowth. TrkA activation by NGF leads to activation of multiple <span class="highlight" style="background-color:">signaling</span> pathways, including Ras/Raf/MAPK. FK506 interacts with constituents of the mature steroid receptor complex (HSP90 <i>via</i> FKBP52; Gold <i>et al</i>., 1999a), which putatively enhances the Ras/Raf/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Line patterns indicate results from our experiments. Hashed arrows are putative links; thin arrows indicate pathways established by other studies (Huang &amp; Reichardt, 2001).</div></p></div><div class="aux"><div class="resc">Raymond D Price, et al. Br J Pharmacol. 2003 November;140(5):825-829.</div><p class="links"><a class="dblinks" href="/pubmed/14559856" ref="ordinalpos=2759&amp;ncbi_uid=805269&amp;link_uid=14559856">Citation</a><a class="dblinks" href="/pmc/articles/PMC1574111/" ref="ordinalpos=2759&amp;ncbi_uid=805269&amp;link_uid=PMC1574111">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2760.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3759892/figure/f5-ijms-14-15885/" ref="ordinalpos=2760&amp;ncbi_uid=5263515&amp;link_uid=PMC3759892" href="/pmc/articles/PMC3759892/"><img src="/pmc/articles/instance/3759892/bin/ijms-14-15885f5.gif" src-large="/pmc/articles/instance/3759892/bin/ijms-14-15885f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759892/" ref="ordinalpos=2760&amp;ncbi_uid=5263515&amp;link_uid=PMC3759892" image-link="/pmc/articles/PMC3759892/figure/f5-ijms-14-15885/" class="imagepopup">Figure 5.  From: Angiogenesis-Related Pathways in the Pathogenesis of Ovarian Cancer. </a></p><div class="supp"><p class="details"><div>An overview of the Ang <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nikos G. Gavalas, et al. Int J Mol Sci. 2013 August;14(8):15885-15909.</div><p class="links"><a class="dblinks" href="/pubmed/23903048" ref="ordinalpos=2760&amp;ncbi_uid=5263515&amp;link_uid=23903048">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759892/" ref="ordinalpos=2760&amp;ncbi_uid=5263515&amp;link_uid=PMC3759892">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2761.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1503662/figure/F7/" ref="ordinalpos=2761&amp;ncbi_uid=710993&amp;link_uid=PMC1503662" href="/pmc/articles/PMC1503662/"><img src="/pmc/articles/instance/1503662/bin/neo0406_0486_fig007.gif" src-large="/pmc/articles/instance/1503662/bin/neo0406_0486_fig007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1503662/" ref="ordinalpos=2761&amp;ncbi_uid=710993&amp;link_uid=PMC1503662" image-link="/pmc/articles/PMC1503662/figure/F7/" class="imagepopup">Figure 7.  From: Ultraviolet Radiation-Induced Apoptosis is Mediated by Daxx. </a></p><div class="supp"><p class="details"><div>Model of UV-induced apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Shiyong Wu, et al. Neoplasia. 2002 November;4(6):486-492.</div><p class="links"><a class="dblinks" href="/pubmed/12407442" ref="ordinalpos=2761&amp;ncbi_uid=710993&amp;link_uid=12407442">Citation</a><a class="dblinks" href="/pmc/articles/PMC1503662/" ref="ordinalpos=2761&amp;ncbi_uid=710993&amp;link_uid=PMC1503662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2762.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3742220/figure/f5-ijms-14-13826/" ref="ordinalpos=2762&amp;ncbi_uid=5193132&amp;link_uid=PMC3742220" href="/pmc/articles/PMC3742220/"><img src="/pmc/articles/instance/3742220/bin/ijms-14-13826f5.gif" src-large="/pmc/articles/instance/3742220/bin/ijms-14-13826f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3742220/" ref="ordinalpos=2762&amp;ncbi_uid=5193132&amp;link_uid=PMC3742220" image-link="/pmc/articles/PMC3742220/figure/f5-ijms-14-13826/" class="imagepopup">Figure 5.  From: Knockdown of FABP3 Impairs Cardiac Development in Zebrafish through the Retinoic Acid <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>FABP3-MO alters the retinoic acid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Raldh2 expression was not significantly different in FABP3-MO injected zebrafish, as visualized by <i>in situ</i> hybridization at 14 hpf (<b>B</b>) and 24 hpf (<b>D</b>), compared with Wt controls at 14 hpf (<b>A</b>) and 24 hpf (<b>C</b>), respectively. Stable expression levels of Raldh2 were also confirmed by real-time PCR (<b>E</b>). Cyp26a1 expression was significantly upregulated in FABP3-MO injected zebrafish, as visualized by in situ hybridization at 14 hpf (<b>G</b>) and 24 hpf (<b>I</b>), compared with Wt controls at 14 hpf (<b>F</b>) and 24 hpf (<b>H</b>), respectively. Increased expression levels of Cyp26a1 were confirmed by real-time PCR (<b>J</b>).* <i>p</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Xuejie Wang, et al. Int J Mol Sci. 2013 July;14(7):13826-13841.</div><p class="links"><a class="dblinks" href="/pubmed/23823803" ref="ordinalpos=2762&amp;ncbi_uid=5193132&amp;link_uid=23823803">Citation</a><a class="dblinks" href="/pmc/articles/PMC3742220/" ref="ordinalpos=2762&amp;ncbi_uid=5193132&amp;link_uid=PMC3742220">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2763.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4003867/figure/F1/" ref="ordinalpos=2763&amp;ncbi_uid=6068200&amp;link_uid=PMC4003867" href="/pmc/articles/PMC4003867/"><img src="/pmc/articles/instance/4003867/bin/nihms401967f1.gif" src-large="/pmc/articles/instance/4003867/bin/nihms401967f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4003867/" ref="ordinalpos=2763&amp;ncbi_uid=6068200&amp;link_uid=PMC4003867" image-link="/pmc/articles/PMC4003867/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Signaling</span> pathways of IL-22.  From: Emerging role of Interleukin-22 in autoimmune diseases. </a></p><div class="supp"><p class="details"><div>IL-22 binds as a homodimer at a receptor complex composed of the IL-10R2 and the IL-22R1. IL-22BPacts as a natural, soluble antagonist. Receptor <span class="highlight" style="background-color:">signaling</span> induces the phosphorylation of Jak-1 and Tyk-2, which activates the transcription factor STAT1, and depending on the cell type, also STAT3 and STAT5. MAPKpathway (including MEK1/2, ERK1/2, JNK, and p38) are also activated through a distinct <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hai-Feng Pan, et al. Cytokine Growth Factor Rev. 2013 February;24(1):51-57.</div><p class="links"><a class="dblinks" href="/pubmed/22906768" ref="ordinalpos=2763&amp;ncbi_uid=6068200&amp;link_uid=22906768">Citation</a><a class="dblinks" href="/pmc/articles/PMC4003867/" ref="ordinalpos=2763&amp;ncbi_uid=6068200&amp;link_uid=PMC4003867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2764.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2597721/figure/ppat-1000247-g006/" ref="ordinalpos=2764&amp;ncbi_uid=1550141&amp;link_uid=PMC2597721" href="/pmc/articles/PMC2597721/"><img src="/pmc/articles/instance/2597721/bin/ppat.1000247.g006.gif" src-large="/pmc/articles/instance/2597721/bin/ppat.1000247.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2597721/" ref="ordinalpos=2764&amp;ncbi_uid=1550141&amp;link_uid=PMC2597721" image-link="/pmc/articles/PMC2597721/figure/ppat-1000247-g006/" class="imagepopup">Figure 6. Model for NIK1-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Regulated Nuclear Trafficking of rpL10A Mediated by NIK1 Represents a Defense Strategy of Plant Cells against Virus. </a></p><div class="supp"><p class="details"><div>Stress-induced oligomerization of the extracellular domain of NIK1 brings the intracellular kinase domains into proximity and allows them to transphosphorylate and activate one another. Upon activation, NIK1 phosphorylates rpL10A, promoting its translocation to the nucleus where it may mount a defense strategy that prevents virus proliferation of spread. Conversely, binding of NSP to the NIK1 kinase domain (A-Loop) inhibits autophosphorylation of NIK1 and, thus, prevents receptor kinase activation and <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Claudine M. Carvalho, et al. PLoS Pathog. 2008 December;4(12):e1000247.</div><p class="links"><a class="dblinks" href="/pubmed/19112492" ref="ordinalpos=2764&amp;ncbi_uid=1550141&amp;link_uid=19112492">Citation</a><a class="dblinks" href="/pmc/articles/PMC2597721/" ref="ordinalpos=2764&amp;ncbi_uid=1550141&amp;link_uid=PMC2597721">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2765.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4367691/figure/f1/" ref="ordinalpos=2765&amp;ncbi_uid=7319499&amp;link_uid=PMC4367691" href="/pmc/articles/PMC4367691/"><img src="/pmc/articles/instance/4367691/bin/tlcr-03-03-122-f1.gif" src-large="/pmc/articles/instance/4367691/bin/tlcr-03-03-122-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4367691/" ref="ordinalpos=2765&amp;ncbi_uid=7319499&amp;link_uid=PMC4367691" image-link="/pmc/articles/PMC4367691/figure/f1/" class="imagepopup">Figure 1.  From: ROR1 as a novel therapeutic target for EGFR-mutant non-small-cell lung cancer patients with the EGFR T790M mutation. </a></p><div class="supp"><p class="details"><div>ROR1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ROR1 is required to sustain balance between pro-survival and pro-apoptotic <span class="highlight" style="background-color:">signaling</span> through sustainment of EGFR-ERBB3 and c-Src activation. ROR1, orphan receptor 1.</div></p></div><div class="aux"><div class="resc">Niki Karachaliou, et al. Transl Lung Cancer Res. 2014 June;3(3):122-130.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2765&amp;ncbi_uid=7319499&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4367691/" ref="ordinalpos=2765&amp;ncbi_uid=7319499&amp;link_uid=PMC4367691">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2766.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2783518/figure/F1/" ref="ordinalpos=2766&amp;ncbi_uid=2170589&amp;link_uid=PMC2783518" href="/pmc/articles/PMC2783518/"><img src="/pmc/articles/instance/2783518/bin/nihms80646f1.gif" src-large="/pmc/articles/instance/2783518/bin/nihms80646f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783518/" ref="ordinalpos=2766&amp;ncbi_uid=2170589&amp;link_uid=PMC2783518" image-link="/pmc/articles/PMC2783518/figure/F1/" class="imagepopup">Figure 1. The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Wnt <span class="highlight" style="background-color:">signaling</span> and senescence: a tug of war in early neoplasia?. </a></p><div class="supp"><p class="details"><div>Canonical Wnt ligands bind to their cognate transmembrane receptors, comprised of a Frizzled family member and the LRP5/6 co-receptor. <span class="highlight" style="background-color:">Signaling</span> through Dishevelled inhibits GSK3-mediated phosphorylation of b-catenin. Phosphorylation is promoted by two associated proteins, APC and axin. Phosphorylated b-catenin is targeted for degradation. Thus, Wnt ligands act to stabilize the cytoplasmic pool of b-catenin. B-catenin translocates to the cell nucleus where it binds to the TCF family of DNA-binding proteins. This complex activates transcription of genes, including proliferation-promoting genes such as c-jun, cyclin D1 and c-myc. Aqua: activators, red: inhibitors.</div></p></div><div class="aux"><div class="resc">Peter D. Adams, et al. Cancer Biol Ther. ;7(11):1706-1711.</div><p class="links"><a class="dblinks" href="/pubmed/18836285" ref="ordinalpos=2766&amp;ncbi_uid=2170589&amp;link_uid=18836285">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783518/" ref="ordinalpos=2766&amp;ncbi_uid=2170589&amp;link_uid=PMC2783518">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2767.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2744869/figure/pone-0007178-g004/" ref="ordinalpos=2767&amp;ncbi_uid=2030192&amp;link_uid=PMC2744869" href="/pmc/articles/PMC2744869/"><img src="/pmc/articles/instance/2744869/bin/pone.0007178.g004.gif" src-large="/pmc/articles/instance/2744869/bin/pone.0007178.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2744869/" ref="ordinalpos=2767&amp;ncbi_uid=2030192&amp;link_uid=PMC2744869" image-link="/pmc/articles/PMC2744869/figure/pone-0007178-g004/" class="imagepopup">Figure 4. SOC2 siRNA affects LPS inducible gene expression both in the MyD88-dependent and -independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: SOCS2 Influences LPS Induced Human Monocyte-Derived Dendritic Cell Maturation. </a></p><div class="supp"><p class="details"><div>Enriched monocytes were cultured in the presence of GM-CSF and IL-4 for 6 days. iDCs were transfected with SOCS2 siRNA or control siRNA and incubated for 24 hours. Cells were further incubated with LPS (200 ng/ml) for additional 2 h. LPS inducible gene mRNA expressions were measured by real-time PCR. (A). LPS inducible gene expression of TNF-Î±, IL-6, IL-1Î² and CCL-4 via the MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) LPS inducible gene expression of IFN-Î², CXCL-9, CXCL-10 via the MyD88-independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Data shown are the mean of triplicate sample, and are representative of at least two donors with similar results. Values for the control were set to 1. Error bars illustrate s.d. * P&lt;0.05; ** P&lt;0.01.</div></p></div><div class="aux"><div class="resc">Jin Hu, et al. PLoS One. 2009;4(9):e7178.</div><p class="links"><a class="dblinks" href="/pubmed/19779605" ref="ordinalpos=2767&amp;ncbi_uid=2030192&amp;link_uid=19779605">Citation</a><a class="dblinks" href="/pmc/articles/PMC2744869/" ref="ordinalpos=2767&amp;ncbi_uid=2030192&amp;link_uid=PMC2744869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2768.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC300841/figure/F1/" ref="ordinalpos=2768&amp;ncbi_uid=336160&amp;link_uid=PMC300841" href="/pmc/articles/PMC300841/"><img src="/pmc/articles/instance/300841/bin/JCI0009871.f1.gif" src-large="/pmc/articles/instance/300841/bin/JCI0009871.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC300841/" ref="ordinalpos=2768&amp;ncbi_uid=336160&amp;link_uid=PMC300841" image-link="/pmc/articles/PMC300841/figure/F1/" class="imagepopup">Figure 1.  From: Reovirus as a novel oncolytic agent. </a></p><div class="supp"><p class="details"><div>Usurpation of the Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by reovirus. In untransformed, reovirus-resistant cells, double-stranded RNA structures in reovirus transcripts activate PKR, which subsequently phosphorylates eIF-2Î±, inhibiting translation initiation of viral genes. In cells with an activated Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, however, PKR phosphorylation in response to viral transcripts is inhibited and viral translation proceeds unimpeded.</div></p></div><div class="aux"><div class="resc">Kara L. Norman, et al. J Clin Invest. 2000 April 15;105(8):1035-1038.</div><p class="links"><a class="dblinks" href="/pubmed/10772645" ref="ordinalpos=2768&amp;ncbi_uid=336160&amp;link_uid=10772645">Citation</a><a class="dblinks" href="/pmc/articles/PMC300841/" ref="ordinalpos=2768&amp;ncbi_uid=336160&amp;link_uid=PMC300841">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2769.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2963311/figure/F8/" ref="ordinalpos=2769&amp;ncbi_uid=2531534&amp;link_uid=PMC2963311" href="/pmc/articles/PMC2963311/"><img src="/pmc/articles/instance/2963311/bin/nihms237509f8.gif" src-large="/pmc/articles/instance/2963311/bin/nihms237509f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2963311/" ref="ordinalpos=2769&amp;ncbi_uid=2531534&amp;link_uid=PMC2963311" image-link="/pmc/articles/PMC2963311/figure/F8/" class="imagepopup">Figure 8.  From: FMRFamide neuropeptides and acetylcholine synergistically inhibit egg-laying by C. elegans. </a></p><div class="supp"><p class="details"><div>EGL-6 <span class="highlight" style="background-color:">signaling</span> inhibits egg-laying behavior redundantly with acetylcholine. (<b>a</b>) Loss of function in both EGL-6 <span class="highlight" style="background-color:">signaling</span> and acetylcholine <span class="highlight" style="background-color:">signaling</span> decreases egg retention time. The distribution of developmental stages of embryos laid by <i>unc-17</i> mutants (defective in vesicular acetylcholine transport) is shown together with the distributions of stages of eggs laid by worms mutant for <i>unc-17</i> and genes in the <i>egl-6</i> <span class="highlight" style="background-color:">pathway</span> or <i>unc-17</i> mutants lacking BAG neurons. BAG neurons were ablated with a laser microbeam in <i>unc-17</i> mutants. An asterisk denotes the presence of the <i>gcy-33âˆ·gfp</i> transgene <i>nIs242</i>, used in some strains to facilitate identification of BAG neurons. The <i>unc-17</i> allele used was <i>e113</i>. The <i>egl-6, flp-10</i>, and <i>flp-17</i> alleles used were <i>n4536, n4543</i> and <i>n4894</i>, respectively. (<b>b</b>) Loss of function in the <i>egl-6</i> <span class="highlight" style="background-color:">pathway</span> suppresses the egg-laying defect of mutants with excess acetylcholine <span class="highlight" style="background-color:">signaling</span>. Shown are distributions of developmental stages of eggs laid by <i>ace-2; ace-1</i> double mutants (defective in acetylcholinesterase function) and by worms multiply mutant for acetylcholinesterase-encoding genes and genes in the <i>egl-6</i> <span class="highlight" style="background-color:">pathway</span>. The <i>ace-1</i> allele used was <i>p1000</i>. The <i>ace-2</i> allele used was <i>g72</i>. The <i>egl-6, flp-10</i>, and <i>flp-17</i> alleles used were <i>n4536, n4543</i> and <i>n4894</i>, respectively.</div></p></div><div class="aux"><div class="resc">Niels Ringstad, et al. Nat Neurosci. ;11(10):1168-1176.</div><p class="links"><a class="dblinks" href="/pubmed/18806786" ref="ordinalpos=2769&amp;ncbi_uid=2531534&amp;link_uid=18806786">Citation</a><a class="dblinks" href="/pmc/articles/PMC2963311/" ref="ordinalpos=2769&amp;ncbi_uid=2531534&amp;link_uid=PMC2963311">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2770.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3209053/figure/F1/" ref="ordinalpos=2770&amp;ncbi_uid=3446438&amp;link_uid=PMC3209053" href="/pmc/articles/PMC3209053/"><img src="/pmc/articles/instance/3209053/bin/zek9990938040001.gif" src-large="/pmc/articles/instance/3209053/bin/zek9990938040001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3209053/" ref="ordinalpos=2770&amp;ncbi_uid=3446438&amp;link_uid=PMC3209053" image-link="/pmc/articles/PMC3209053/figure/F1/" class="imagepopup">Fig. 1. From: Vacuolar Protein Sorting Genes Regulate Mat Formation in Saccharomyces cerevisiae by Flo11p-Dependent and -Independent Mechanisms  . </a></p><div class="supp"><p class="details"><div>Class E vps mutants that affect the Rim101p <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (class E-1) cause defects in mat formation and invasive growth, but class E vps mutants that do not affect the Rim101p <span class="highlight" style="background-color:">pathway</span> (class E-2) disrupt mat formation but not invasive growth. (A) The class E vps mutants <i>vps27</i>Î”, <i>vps28</i>Î” (ESCRT-I), <i>vps25</i>Î” (ESCRT-II), <i>vps20</i>Î” (ESCRT-III), and <i>vps4</i>Î” were subjected to the invasive growth assay (WT, wild type); (B) members of the Rim101p <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, <i>rim101</i>Î”, <i>rim9</i>Î”, and <i>rim8</i>Î”, were subjected to the invasive growth assay; (C and D) representative members of the class E-1 and E-2 vps mutants (C) and the Rim101p signal transduction <span class="highlight" style="background-color:">pathway</span> (D) were subjected to the overlay adhesion assay.</div></p></div><div class="aux"><div class="resc">Neha Sarode, et al. Eukaryot Cell. 2011 November;10(11):1516-1526.</div><p class="links"><a class="dblinks" href="/pubmed/21908597" ref="ordinalpos=2770&amp;ncbi_uid=3446438&amp;link_uid=21908597">Citation</a><a class="dblinks" href="/pmc/articles/PMC3209053/" ref="ordinalpos=2770&amp;ncbi_uid=3446438&amp;link_uid=PMC3209053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2771.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4063677/figure/pcbi-1003655-g001/" ref="ordinalpos=2771&amp;ncbi_uid=6288924&amp;link_uid=PMC4063677" href="/pmc/articles/PMC4063677/"><img src="/pmc/articles/instance/4063677/bin/pcbi.1003655.g001.gif" src-large="/pmc/articles/instance/4063677/bin/pcbi.1003655.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4063677/" ref="ordinalpos=2771&amp;ncbi_uid=6288924&amp;link_uid=PMC4063677" image-link="/pmc/articles/PMC4063677/figure/pcbi-1003655-g001/" class="imagepopup">Figure 1. Core Notch-Delta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: An Expanded Notch-Delta Model Exhibiting Long-Range Patterning and Incorporating MicroRNA Regulation. </a></p><div class="aux"><div class="resc">Jerry S. Chen, et al. PLoS Comput Biol. 2014 June;10(6):e1003655.</div><p class="links"><a class="dblinks" href="/pubmed/24945987" ref="ordinalpos=2771&amp;ncbi_uid=6288924&amp;link_uid=24945987">Citation</a><a class="dblinks" href="/pmc/articles/PMC4063677/" ref="ordinalpos=2771&amp;ncbi_uid=6288924&amp;link_uid=PMC4063677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2772.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2943518/figure/F7/" ref="ordinalpos=2772&amp;ncbi_uid=2608786&amp;link_uid=PMC2943518" href="/pmc/articles/PMC2943518/"><img src="/pmc/articles/instance/2943518/bin/jin0002-0043-f07.gif" src-large="/pmc/articles/instance/2943518/bin/jin0002-0043-f07.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2943518/" ref="ordinalpos=2772&amp;ncbi_uid=2608786&amp;link_uid=PMC2943518" image-link="/pmc/articles/PMC2943518/figure/F7/" class="imagepopup">Fig. 7.  From: A PTP4A3 Peptide PIMAP39 Modulates TNF-Alpha Levels and Endotoxic Shock. </a></p><div class="supp"><p class="details"><div>Diagram of a putative LPS-induced PTP/LITAF/TNF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Xiaoren Tang, et al. J Innate Immun. 2009 December;2(1):43-55.</div><p class="links"><a class="dblinks" href="/pubmed/20375622" ref="ordinalpos=2772&amp;ncbi_uid=2608786&amp;link_uid=20375622">Citation</a><a class="dblinks" href="/pmc/articles/PMC2943518/" ref="ordinalpos=2772&amp;ncbi_uid=2608786&amp;link_uid=PMC2943518">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2773.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3722525/figure/btt335-F7/" ref="ordinalpos=2773&amp;ncbi_uid=5127717&amp;link_uid=PMC3722525" href="/pmc/articles/PMC3722525/"><img src="/pmc/articles/instance/3722525/bin/btt335f7p.gif" src-large="/pmc/articles/instance/3722525/bin/btt335f7p.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3722525/" ref="ordinalpos=2773&amp;ncbi_uid=5127717&amp;link_uid=PMC3722525" image-link="/pmc/articles/PMC3722525/figure/btt335-F7/" class="imagepopup">Fig. 7. From: Joint network and node selection for <span class="highlight" style="background-color:">pathway</span>-based genomic data analysis. </a></p><div class="supp"><p class="details"><div>Examples of part of identified pathways. (<b>a</b>) The antigen processing and presentation <span class="highlight" style="background-color:">pathway</span> for DLBCL; (<b>b</b>) the cell cycle <span class="highlight" style="background-color:">pathway</span> for CRC; (<b>c</b>) the TGF-<img src="/pmc/articles/instance/3722525/bin/btt335i147.jpg" class="inline-graphic" style="border-style: none;" /> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for PDAC. Shaded and unshaded boxes indicate selected and not selected genes, respectively</div></p></div><div class="aux"><div class="resc">Shandian Zhe, et al. Bioinformatics. 2013 August 15;29(16):1987-1996.</div><p class="links"><a class="dblinks" href="/pubmed/23749986" ref="ordinalpos=2773&amp;ncbi_uid=5127717&amp;link_uid=23749986">Citation</a><a class="dblinks" href="/pmc/articles/PMC3722525/" ref="ordinalpos=2773&amp;ncbi_uid=5127717&amp;link_uid=PMC3722525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2774.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3520483/figure/F1/" ref="ordinalpos=2774&amp;ncbi_uid=4457751&amp;link_uid=PMC3520483" href="/pmc/articles/PMC3520483/"><img src="/pmc/articles/instance/3520483/bin/fphys-03-00468-g0001.gif" src-large="/pmc/articles/instance/3520483/bin/fphys-03-00468-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3520483/" ref="ordinalpos=2774&amp;ncbi_uid=4457751&amp;link_uid=PMC3520483" image-link="/pmc/articles/PMC3520483/figure/F1/" class="imagepopup">Figure 1.  From: Modulation of myocardial contraction by peroxynitrite. </a></p><div class="supp"><p class="details"><div><b>Physiological <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of low peroxynitrite.</b> This is a diagrammatic representation of low peroxynitrite <span class="highlight" style="background-color:">signaling</span> and various proposed end targets within the cardiomyocyte. The endogenous production of low peroxynitrite can occur via NOS1/xanthine oxidase.</div></p></div><div class="aux"><div class="resc">Mark J. Kohr, et al. Front Physiol. 2012;3:468.</div><p class="links"><a class="dblinks" href="/pubmed/23248603" ref="ordinalpos=2774&amp;ncbi_uid=4457751&amp;link_uid=23248603">Citation</a><a class="dblinks" href="/pmc/articles/PMC3520483/" ref="ordinalpos=2774&amp;ncbi_uid=4457751&amp;link_uid=PMC3520483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2775.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2806576/figure/F8/" ref="ordinalpos=2775&amp;ncbi_uid=2207122&amp;link_uid=PMC2806576" href="/pmc/articles/PMC2806576/"><img src="/pmc/articles/instance/2806576/bin/nihms-168566-f0008.gif" src-large="/pmc/articles/instance/2806576/bin/nihms-168566-f0008.jpg" alt="FIGURE 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2806576/" ref="ordinalpos=2775&amp;ncbi_uid=2207122&amp;link_uid=PMC2806576" image-link="/pmc/articles/PMC2806576/figure/F8/" class="imagepopup">FIGURE 8.  From: Cholinergic Neural Signals to the Spleen Down-Regulate Leukocyte Trafficking via CD11b. </a></p><div class="supp"><p class="details"><div>Proposed model for the role of the spleen in regulating leukocyte trafficking via cholinergic <span class="highlight" style="background-color:">signaling</span>. Collectively, our findings reveal a centralized <span class="highlight" style="background-color:">pathway</span> for controlling leukocyte trafficking to sites of inflammation via modulation of neutrophil CD11b levels through a spleen-dependent neural cholinergic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> can be manipulated by cholinergic stimulation via electrical stimulation of the vagus nerve or by pharmacological cholinergic agonists to attenuate excess leukocyte accumulation at the site of inflammation and reduce tissue damage.</div></p></div><div class="aux"><div class="resc">Jared M. Huston, et al. J Immunol. ;183(1):552-559.</div><p class="links"><a class="dblinks" href="/pubmed/19542466" ref="ordinalpos=2775&amp;ncbi_uid=2207122&amp;link_uid=19542466">Citation</a><a class="dblinks" href="/pmc/articles/PMC2806576/" ref="ordinalpos=2775&amp;ncbi_uid=2207122&amp;link_uid=PMC2806576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2776.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3659771/figure/F2/" ref="ordinalpos=2776&amp;ncbi_uid=4913237&amp;link_uid=PMC3659771" href="/pmc/articles/PMC3659771/"><img src="/pmc/articles/instance/3659771/bin/nihms468458f2.gif" src-large="/pmc/articles/instance/3659771/bin/nihms468458f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3659771/" ref="ordinalpos=2776&amp;ncbi_uid=4913237&amp;link_uid=PMC3659771" image-link="/pmc/articles/PMC3659771/figure/F2/" class="imagepopup">Figure 2. Model for genetic regulation of artery-vein differentiation by the Notch and PLCÎ³/MAPK pathways.  From: Notch <span class="highlight" style="background-color:">signaling</span> in the vasculature. </a></p><div class="supp"><p class="details"><div>An arterial endothelial cell is shown on the left, while a venous endothelial cell is depicted on the right. Two main <span class="highlight" style="background-color:">signaling</span> pathways operate downstream of VEGF-A to induce arterial differentiation: the Notch <span class="highlight" style="background-color:">pathway</span> and the Phospholipase CÎ³ (PLCÎ³) / Mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">pathway</span>. The transcription factors Foxc1 and Foxc2 directly induce <i>Dll4</i> and <i>Hey2</i> transcription. VEGF-A <span class="highlight" style="background-color:">signaling</span>, by an unknown mechanism, augments Foxc1/Foxc2 induction of <i>Dll4</i> and <i>Hey2</i> gene expression. During venous differentiation, two different mechanisms inhibit artery differentiation. The orphan nuclear receptor COUP-TFII suppresses neuropilin1 expression, thereby suppressing reception of the VEGF-A signal and activation of Notch <span class="highlight" style="background-color:">signaling</span>. In addition, activation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> antagonizes promotion of arterial cell differentiation by blocking ERK activation.</div></p></div><div class="aux"><div class="resc">Thomas Gridley. Curr Top Dev Biol. ;92:277-309.</div><p class="links"><a class="dblinks" href="/pubmed/20816399" ref="ordinalpos=2776&amp;ncbi_uid=4913237&amp;link_uid=20816399">Citation</a><a class="dblinks" href="/pmc/articles/PMC3659771/" ref="ordinalpos=2776&amp;ncbi_uid=4913237&amp;link_uid=PMC3659771">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2777.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3554567/figure/pgen-1003109-g004/" ref="ordinalpos=2777&amp;ncbi_uid=4562117&amp;link_uid=PMC3554567" href="/pmc/articles/PMC3554567/"><img src="/pmc/articles/instance/3554567/bin/pgen.1003109.g004.gif" src-large="/pmc/articles/instance/3554567/bin/pgen.1003109.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3554567/" ref="ordinalpos=2777&amp;ncbi_uid=4562117&amp;link_uid=PMC3554567" image-link="/pmc/articles/PMC3554567/figure/pgen-1003109-g004/" class="imagepopup">Figure 4. Model of the opposing roles for Nodal and Bmp <span class="highlight" style="background-color:">signaling</span> during asymmetric cardiac morphogenesis..  From: Integration of Nodal and BMP Signals in the Heart Requires FoxH1 to Create Left-Right Differences in Cell Migration Rates That Direct Cardiac Asymmetry. </a></p><div class="supp"><p class="details"><div>1: Nodal <span class="highlight" style="background-color:">signaling</span> to the myocardium through the transcription factor FoxH1 is required to asymmetrically increase cardiac migration rates specifically on the left of the cardiac cone. 2: The Bmp <span class="highlight" style="background-color:">pathway</span> is bilaterally activated in the endocardium but only effectively limits migration rates in the absence of strong inductive cues from Nodal <span class="highlight" style="background-color:">signaling</span> on the right of the cardiac cone. This leads to a non-cell autonomous decrease in myocardial migration rates on the right of the cone. 3: The Nodal <span class="highlight" style="background-color:">pathway</span> generally limits Bmp <span class="highlight" style="background-color:">pathway</span> activation in the heart, potentially through competition for access to proteins common to both pathways. 4: FoxH1 is also required in a Nodal-independent manner in the heart to mediate cardiac cell responsiveness to Bmp signals.</div></p></div><div class="aux"><div class="resc">Kari F. Lenhart, et al. PLoS Genet. 2013 January;9(1):e1003109.</div><p class="links"><a class="dblinks" href="/pubmed/23358434" ref="ordinalpos=2777&amp;ncbi_uid=4562117&amp;link_uid=23358434">Citation</a><a class="dblinks" href="/pmc/articles/PMC3554567/" ref="ordinalpos=2777&amp;ncbi_uid=4562117&amp;link_uid=PMC3554567">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2778.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4145548/figure/fig3/" ref="ordinalpos=2778&amp;ncbi_uid=6551458&amp;link_uid=PMC4145548" href="/pmc/articles/PMC4145548/"><img src="/pmc/articles/instance/4145548/bin/BMRI2014-509149.003.gif" src-large="/pmc/articles/instance/4145548/bin/BMRI2014-509149.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4145548/" ref="ordinalpos=2778&amp;ncbi_uid=6551458&amp;link_uid=PMC4145548" image-link="/pmc/articles/PMC4145548/figure/fig3/" class="imagepopup">Figure 3.  From: FSim: A Novel Functional Similarity Search Algorithm and Tool for Discovering Functionally Related Gene Products. </a></p><div class="supp"><p class="details"><div>Hierarchical clustering analysis of the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes with FSim functional similarity scores. The Entrez gene ID is shown for each gene in the p53 <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> genes are clustered based on the functional similarity distances between each two genes, as calculated by FSim. A subcluster of 13 genes were highlighted in red rectangle. Eleven of these 13 genes were in the same apoptosis <span class="highlight" style="background-color:">pathway</span> except for TSC2 (Entrez ID 7249) and GADD45A (Entrez ID 1647).</div></p></div><div class="aux"><div class="resc">Qiang Hu, et al. Biomed Res Int. 2014;2014:509149.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2778&amp;ncbi_uid=6551458&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4145548/" ref="ordinalpos=2778&amp;ncbi_uid=6551458&amp;link_uid=PMC4145548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2779.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3511325/figure/pone-0050874-g009/" ref="ordinalpos=2779&amp;ncbi_uid=4426947&amp;link_uid=PMC3511325" href="/pmc/articles/PMC3511325/"><img src="/pmc/articles/instance/3511325/bin/pone.0050874.g009.gif" src-large="/pmc/articles/instance/3511325/bin/pone.0050874.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3511325/" ref="ordinalpos=2779&amp;ncbi_uid=4426947&amp;link_uid=PMC3511325" image-link="/pmc/articles/PMC3511325/figure/pone-0050874-g009/" class="imagepopup">Figure 9. Proposed Model for SOCS3 Contribution to Excitoxicity in Neurons. A.  From: Protective Role of STAT3 in NMDA and Glutamate-Induced Neuronal Death: Negative Regulatory Effect of SOCS3. </a></p><div class="supp"><p class="details"><div>, IL-6/R or OSM activates the JAK/STAT3 <span class="highlight" style="background-color:">pathway</span>, which increases Bcl-xL expression and causes activation of the Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in a STAT3-dependent manner. Increased Bcl-xL expression and binding to Bax, as well as Akt activation inhibits NMDA or glutamate-induced neuronal death. <b>B</b>, IL-6/R and OSM also induce SOCS3 expression in a STAT3-dependent fashion. SOCS3, in turn, modulates IL-6/R or OSM induced Bcl-xL expression and Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in a negative regulatory manner, which contributes, in part, to NMDA or glutamate-induced neuronal death.</div></p></div><div class="aux"><div class="resc">Keun W. Park, et al. PLoS One. 2012;7(11):e50874.</div><p class="links"><a class="dblinks" href="/pubmed/23226414" ref="ordinalpos=2779&amp;ncbi_uid=4426947&amp;link_uid=23226414">Citation</a><a class="dblinks" href="/pmc/articles/PMC3511325/" ref="ordinalpos=2779&amp;ncbi_uid=4426947&amp;link_uid=PMC3511325">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2780.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4168072/figure/F2/" ref="ordinalpos=2780&amp;ncbi_uid=6648708&amp;link_uid=PMC4168072" href="/pmc/articles/PMC4168072/"><img src="/pmc/articles/instance/4168072/bin/WJG-20-12381-g002.gif" src-large="/pmc/articles/instance/4168072/bin/WJG-20-12381-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4168072/" ref="ordinalpos=2780&amp;ncbi_uid=6648708&amp;link_uid=PMC4168072" image-link="/pmc/articles/PMC4168072/figure/F2/" class="imagepopup">Figure 2.  From: Smad3 phospho-isoform <span class="highlight" style="background-color:">signaling</span> in hepatitis C virus-related chronic liver diseases. </a></p><div class="supp"><p class="details"><div>Smad3 phospho-isoform <span class="highlight" style="background-color:">signaling</span>: Tumor suppressive TÎ²RI/pSmad3C <span class="highlight" style="background-color:">pathway</span> <i>vs</i> carcinogenic JNK/pSmad3L <span class="highlight" style="background-color:">pathway</span>. A: In normal hepatocytes, transforming growth factor (TGF)-Î² activates the TGF-Î² typeâ€…Iâ€…receptor (TÎ²RI), leading to the direct phosphorylation of the C-terminal region of Smad3. The C-terminal phosphorylated Smad3 (pSmad3C) moves to the nucleus and transmits cytostatic signals by up-regulating p21<sup>WAF1</sup>. The TÎ²RI/pSmad3C <span class="highlight" style="background-color:">pathway</span> is crucial for homeostasis and is a tumor suppressor; B: In hepatocytes affected by mitogenic stimuli, pro-inflammatory cytokines activate c-Jun N-terminal kinase (JNK), drastically altering the Smad3 <span class="highlight" style="background-color:">signaling</span> to induce the phosphorylation of Smad3 at its linker region. The linker-phosphorylated Smad3 (pSmad3L) undergoes translocation to the nucleus to stimulate c-Myc transcription, resulting in a proliferative state. During hepatitis C virus-related chronic liver disease progression, the JNK/pSmad3L <span class="highlight" style="background-color:">pathway</span> is constitutively activated and promotes carcinogenesis.</div></p></div><div class="aux"><div class="resc">Takashi Yamaguchi, et al. World J Gastroenterol. 2014 September 21;20(35):12381-12390.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2780&amp;ncbi_uid=6648708&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4168072/" ref="ordinalpos=2780&amp;ncbi_uid=6648708&amp;link_uid=PMC4168072">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2781.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4066587/figure/F2/" ref="ordinalpos=2781&amp;ncbi_uid=6347305&amp;link_uid=PMC4066587" href="/pmc/articles/PMC4066587/"><img src="/pmc/articles/instance/4066587/bin/fpls-05-00297-g002.gif" src-large="/pmc/articles/instance/4066587/bin/fpls-05-00297-g002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4066587/" ref="ordinalpos=2781&amp;ncbi_uid=6347305&amp;link_uid=PMC4066587" image-link="/pmc/articles/PMC4066587/figure/F2/" class="imagepopup">FIGURE 2.  From: <span class="highlight" style="background-color:">Signaling</span> to stomatal initiation and cell division. </a></p><div class="supp"><p class="details"><div><b>The stomatal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> EPF1, EPF2, and STOMAGEN/EPFL9 signals are received by TMMâ€“ERf leading to transduction and the YODAâ€“MKKâ€“MPK cascade. Light regulates stomatal production via COP1 which is upstream of YODA. In stems or hypocotyls TMM dampens CHALf peptide binding with the ERf receptor. BR signals received by BRI1â€“BAK1 inhibit BIN2 activity. BIN2 suppresses SPCH by directly phosphorylating or indirectly thorough YODA or MKK4/5.</div></p></div><div class="aux"><div class="resc">Jie Le, et al. Front Plant Sci. 2014;5:297.</div><p class="links"><a class="dblinks" href="/pubmed/25002867" ref="ordinalpos=2781&amp;ncbi_uid=6347305&amp;link_uid=25002867">Citation</a><a class="dblinks" href="/pmc/articles/PMC4066587/" ref="ordinalpos=2781&amp;ncbi_uid=6347305&amp;link_uid=PMC4066587">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2782.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634812/figure/F3/" ref="ordinalpos=2782&amp;ncbi_uid=1715422&amp;link_uid=PMC2634812" href="/pmc/articles/PMC2634812/"><img src="/pmc/articles/instance/2634812/bin/nihms70868f3.gif" src-large="/pmc/articles/instance/2634812/bin/nihms70868f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634812/" ref="ordinalpos=2782&amp;ncbi_uid=1715422&amp;link_uid=PMC2634812" image-link="/pmc/articles/PMC2634812/figure/F3/" class="imagepopup">Figure 3.  From: Competition and collaboration: GATA-3, PU.1, and Notch <span class="highlight" style="background-color:">signaling</span> in early T-cell fate determination. </a></p><div class="supp"><p class="details"><div>Stepwise respecification of ETP and DN2 cells by GATA-3 overexpression. Early, intermediate, and late stages in the reprogramming of ETP or DN2 cells by forced expression of GATA-3. The stages where Notch <span class="highlight" style="background-color:">signaling</span> exerts inhibition are shown. Adapted from Supplementary Figure 6, ref. [36]. Note that susceptibility of cells to these interactions is limited to the stages before DN3, when an unidentified commitment function blocks access to the mast-cell <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ellen V. Rothenberg, et al. Semin Immunol. ;20(4):236-246.</div><p class="links"><a class="dblinks" href="/pubmed/18768329" ref="ordinalpos=2782&amp;ncbi_uid=1715422&amp;link_uid=18768329">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634812/" ref="ordinalpos=2782&amp;ncbi_uid=1715422&amp;link_uid=PMC2634812">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2783.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2600878/figure/F6/" ref="ordinalpos=2783&amp;ncbi_uid=1569957&amp;link_uid=PMC2600878" href="/pmc/articles/PMC2600878/"><img src="/pmc/articles/instance/2600878/bin/nihms26144f6.gif" src-large="/pmc/articles/instance/2600878/bin/nihms26144f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2600878/" ref="ordinalpos=2783&amp;ncbi_uid=1569957&amp;link_uid=PMC2600878" image-link="/pmc/articles/PMC2600878/figure/F6/" class="imagepopup">Figure 6. Model of LFA-1 activation during neutrophil rolling.  From: Syk <span class="highlight" style="background-color:">signaling</span> is necessary for E-selectin-induced LFA-1-ICAM-1 association and rolling but not arrest. </a></p><div class="supp"><p class="details"><div>E-selectin binding to PSGL-1 leads, through Syk (arrows), to partial LFA-1 (open curved arrow) activation that supports rolling on ICAM-1 (shown as dimer). CXCR2 engagement by CXCL1 leads, through Gai, to full LFA-1 activation, resulting in arrest on ICAM-1 (filled curved arrow). Straight arrows indicate <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, but interactions may be indirect. LFA-1 conformations from Takagi et al (Takagi and Springer, 2002).</div></p></div><div class="aux"><div class="resc">Alexander Zarbock, et al. Immunity. ;26(6):773-783.</div><p class="links"><a class="dblinks" href="/pubmed/17543554" ref="ordinalpos=2783&amp;ncbi_uid=1569957&amp;link_uid=17543554">Citation</a><a class="dblinks" href="/pmc/articles/PMC2600878/" ref="ordinalpos=2783&amp;ncbi_uid=1569957&amp;link_uid=PMC2600878">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2784.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3756181/figure/F7/" ref="ordinalpos=2784&amp;ncbi_uid=5263447&amp;link_uid=PMC3756181" href="/pmc/articles/PMC3756181/"><img src="/pmc/articles/instance/3756181/bin/aair-5-329-g007.gif" src-large="/pmc/articles/instance/3756181/bin/aair-5-329-g007.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3756181/" ref="ordinalpos=2784&amp;ncbi_uid=5263447&amp;link_uid=PMC3756181" image-link="/pmc/articles/PMC3756181/figure/F7/" class="imagepopup">Fig. 7.  From: Suppression of Heme Oxygenase-1 by Prostaglandin E2-Protein Kinase A-A-Kinase Anchoring Protein <span class="highlight" style="background-color:">Signaling</span> Is Central for Augmented Cyclooxygenase-2 Expression in Lipopolysaccharide-Stimulated RAW 264.7 Macrophages. </a></p><div class="supp"><p class="details"><div>Schematic representation of the proposed mechanism by which PGE<sub>2</sub> <span class="highlight" style="background-color:">signaling</span> amplifies COX-2 expression through an autocrine manner in LPS-stimulated macrophages. Upon LPS binding to the toll-like receptor 4 (TLR4), the NF-ÎºB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> becomes activated, leading to an increase in COX-2 expression, which in turn activates cAMP/PKA/AKAP through the production of PGE<sub>2</sub>. The PKA-AKAP complex blocks HO-1 activity, which results in enhanced translocation of NF-ÎºB p65 to the nucleus.</div></p></div><div class="aux"><div class="resc">Jae-Hyoung Lee, et al. Allergy Asthma Immunol Res. 2013 September;5(5):329-336.</div><p class="links"><a class="dblinks" href="/pubmed/24003391" ref="ordinalpos=2784&amp;ncbi_uid=5263447&amp;link_uid=24003391">Citation</a><a class="dblinks" href="/pmc/articles/PMC3756181/" ref="ordinalpos=2784&amp;ncbi_uid=5263447&amp;link_uid=PMC3756181">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2785.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4159007/figure/evu158-F7/" ref="ordinalpos=2785&amp;ncbi_uid=6603327&amp;link_uid=PMC4159007" href="/pmc/articles/PMC4159007/"><img src="/pmc/articles/instance/4159007/bin/evu158f7p.gif" src-large="/pmc/articles/instance/4159007/bin/evu158f7p.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 7.â€”" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4159007/" ref="ordinalpos=2785&amp;ncbi_uid=6603327&amp;link_uid=PMC4159007" image-link="/pmc/articles/PMC4159007/figure/evu158-F7/" class="imagepopup">F ig . 7.â€”.  From: Epigenetics and the Evolution of Darwin's Finches. </a></p><div class="supp"><p class="details"><div>Epimutation-associated genes and correlated BMP <span class="highlight" style="background-color:">pathway</span>. The genes having associated epimutations in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> presented for the different species are identified as FUL (purple), SCA (green), PAR (blue), and CRA (red) colored boxed genes.</div></p></div><div class="aux"><div class="resc">Michael K. Skinner, et al. Genome Biol Evol. 2014 August;6(8):1972-1989.</div><p class="links"><a class="dblinks" href="/pubmed/25062919" ref="ordinalpos=2785&amp;ncbi_uid=6603327&amp;link_uid=25062919">Citation</a><a class="dblinks" href="/pmc/articles/PMC4159007/" ref="ordinalpos=2785&amp;ncbi_uid=6603327&amp;link_uid=PMC4159007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2786.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4159007/figure/evu158-F9/" ref="ordinalpos=2786&amp;ncbi_uid=6603325&amp;link_uid=PMC4159007" href="/pmc/articles/PMC4159007/"><img src="/pmc/articles/instance/4159007/bin/evu158f9p.gif" src-large="/pmc/articles/instance/4159007/bin/evu158f9p.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 9.â€”" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4159007/" ref="ordinalpos=2786&amp;ncbi_uid=6603325&amp;link_uid=PMC4159007" image-link="/pmc/articles/PMC4159007/figure/evu158-F9/" class="imagepopup">F ig . 9.â€”.  From: Epigenetics and the Evolution of Darwin's Finches. </a></p><div class="supp"><p class="details"><div>Epimutation-associated genes and correlated melanogenesis <span class="highlight" style="background-color:">pathway</span>. The genes having associated epimutations in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> presented for the different species are identified as FUL (purple), SCA (green), PAR (blue), and CRA (red) colored boxed genes.</div></p></div><div class="aux"><div class="resc">Michael K. Skinner, et al. Genome Biol Evol. 2014 August;6(8):1972-1989.</div><p class="links"><a class="dblinks" href="/pubmed/25062919" ref="ordinalpos=2786&amp;ncbi_uid=6603325&amp;link_uid=25062919">Citation</a><a class="dblinks" href="/pmc/articles/PMC4159007/" ref="ordinalpos=2786&amp;ncbi_uid=6603325&amp;link_uid=PMC4159007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2787.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3694890/figure/pone-0067634-g008/" ref="ordinalpos=2787&amp;ncbi_uid=5062026&amp;link_uid=PMC3694890" href="/pmc/articles/PMC3694890/"><img src="/pmc/articles/instance/3694890/bin/pone.0067634.g008.gif" src-large="/pmc/articles/instance/3694890/bin/pone.0067634.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3694890/" ref="ordinalpos=2787&amp;ncbi_uid=5062026&amp;link_uid=PMC3694890" image-link="/pmc/articles/PMC3694890/figure/pone-0067634-g008/" class="imagepopup">Figure 8. <span class="highlight" style="background-color:">Pathway</span> interactome of upregulated GRB2-pY proteome interactome in breast tumor cells (MCF7 and T47D)..  From: A Novel Phosphopeptide Microarray Based Interactome Map in Breast Cancer Cells Reveals Phosphoprotein-GRB2 Cell <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>The GRB2-pY proteome interactions imply cross-talk between various <span class="highlight" style="background-color:">signaling</span> pathways that are functional to initiate various cellular responses. The GBR2 interactome revealed connections of RTK pathways with both classical GRB2-SOS-RAS cascade with PI3K-IRS1 cascade leading to AKT and ERK activation. The crosstalk between growth factor and cytokine <span class="highlight" style="background-color:">signaling</span> pathways as revealed by major receptors and adaptors and other downstream proteins. Upregulation of negative regulation of immune <span class="highlight" style="background-color:">signaling</span> pathways in tumor cells indicate the preparedness of tumor cells to fight and survive attack from host immune <span class="highlight" style="background-color:">signaling</span> mechanisms. Novel interactions are marked by red arrows.</div></p></div><div class="aux"><div class="resc">Srinivasan Krishnamoorthy, et al. PLoS One. 2013;8(6):e67634.</div><p class="links"><a class="dblinks" href="/pubmed/23826330" ref="ordinalpos=2787&amp;ncbi_uid=5062026&amp;link_uid=23826330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3694890/" ref="ordinalpos=2787&amp;ncbi_uid=5062026&amp;link_uid=PMC3694890">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2788.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3357369/figure/F3/" ref="ordinalpos=2788&amp;ncbi_uid=4095990&amp;link_uid=PMC3357369" href="/pmc/articles/PMC3357369/"><img src="/pmc/articles/instance/3357369/bin/psb-7-53-g3.gif" src-large="/pmc/articles/instance/3357369/bin/psb-7-53-g3.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3357369/" ref="ordinalpos=2788&amp;ncbi_uid=4095990&amp;link_uid=PMC3357369" image-link="/pmc/articles/PMC3357369/figure/F3/" class="imagepopup">Figure. From: SCI1, the first member of the tissue-specific inhibitors of CDK (TIC) class, is probably connected to the auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 3.</b> Proposed model for SCI1 as a signal transducer in the auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the cell proliferation/differentiation control during pistil development. Pistil developmental signals switch on SCI1 expression which is potentially regulated by auxin- and cell cycle-dependent factors at the transcriptional level. SCI1 controls cell proliferation/differentiation in specific tissues, promoting proper pistil development. 1  This control is very probably exerted by SCI1 interaction with cyclins and/or CDKs, which causes the inhibition of the cell cycle. Conversely, SCI1 influences the transcript levels of auxin-responsive genes, probably through a feedback mechanism in the auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The SCI1 inhibition on cell proliferation helps to provide the fine-tuning of the auxin response to cell proliferation/differentiation in stigmas/styles. (Arrow bars, induction, promotion; ball ended bars, induction or repression; interrupted bar, inhibition; dotted line, correlation, probably through an unknown factor (?).</div></p></div><div class="aux"><div class="resc">Henrique DePaoli, et al. Plant Signal Behav. 2012 January 1;7(1):53-58.</div><p class="links"><a class="dblinks" href="/pubmed/22301969" ref="ordinalpos=2788&amp;ncbi_uid=4095990&amp;link_uid=22301969">Citation</a><a class="dblinks" href="/pmc/articles/PMC3357369/" ref="ordinalpos=2788&amp;ncbi_uid=4095990&amp;link_uid=PMC3357369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2789.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3190874/figure/F1/" ref="ordinalpos=2789&amp;ncbi_uid=3290767&amp;link_uid=PMC3190874" href="/pmc/articles/PMC3190874/"><img src="/pmc/articles/instance/3190874/bin/zbc0411179700001.gif" src-large="/pmc/articles/instance/3190874/bin/zbc0411179700001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3190874/" ref="ordinalpos=2789&amp;ncbi_uid=3290767&amp;link_uid=PMC3190874" image-link="/pmc/articles/PMC3190874/figure/F1/" class="imagepopup">FIGURE 1. From: Immunoglobulin E Receptor <span class="highlight" style="background-color:">Signaling</span> and Asthma. </a></p><div class="supp"><p class="details"><div><b>FcÏµRI <span class="highlight" style="background-color:">signaling</span> in mast cells proceeds through a network of <span class="highlight" style="background-color:">signaling</span> molecules and adaptor proteins, ultimately leading to effects on cell migration and adhesion, growth and survival, degranulation, eicosanoid production, and cytokine and chemokine production.</b> FcÏµRI in mast cells is a tetrameric complex consisting of an Î±-subunit, a Î²-subunit, and two disulfide-bonded Î³-subunits (<i>blue</i>). Proximal FcÏµRI <span class="highlight" style="background-color:">signaling</span> is mediated through Src family kinases and Syk (<i>yellow</i>). Adaptor proteins include LAT, NTAL/LAB/LAT2, Grb2, Gads, and SLP76 (<i>aqua</i>). Lipid <span class="highlight" style="background-color:">signaling</span> pathways are mediated by PI3K, SphK, PLD, and PLCÎ³ (<i>purple</i>). Calcium <span class="highlight" style="background-color:">signaling</span> proceeds through a two-step process, consisting of the initial release of intracellular ER calcium stores, followed by extracellular calcium influx (<i>green</i>). Additional <span class="highlight" style="background-color:">signaling</span> molecules and pathways include Btk, which links PI3K activation to PLCÎ³ activation (<i>brown</i>); Ras/MAPK pathways (<i>orange</i>); the PDK1/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> (<i>gray</i>); PKCs (<i>white</i>); and NF-ÎºB (<i>pink</i>). <i>PA</i>, phosphatidic acid; <i>IP3R</i>, inositol 1,4,5-trisphosphate receptor.</div></p></div><div class="aux"><div class="resc">Lawren C. Wu. J Biol Chem. 2011 September 23;286(38):32891-32897.</div><p class="links"><a class="dblinks" href="/pubmed/21799019" ref="ordinalpos=2789&amp;ncbi_uid=3290767&amp;link_uid=21799019">Citation</a><a class="dblinks" href="/pmc/articles/PMC3190874/" ref="ordinalpos=2789&amp;ncbi_uid=3290767&amp;link_uid=PMC3190874">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2790.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3101770/figure/F1/" ref="ordinalpos=2790&amp;ncbi_uid=2968344&amp;link_uid=PMC3101770" href="/pmc/articles/PMC3101770/"><img src="/pmc/articles/instance/3101770/bin/nihms-260063-f0001.gif" src-large="/pmc/articles/instance/3101770/bin/nihms-260063-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3101770/" ref="ordinalpos=2790&amp;ncbi_uid=2968344&amp;link_uid=PMC3101770" image-link="/pmc/articles/PMC3101770/figure/F1/" class="imagepopup">Figure 1.  From: Specificity and <span class="highlight" style="background-color:">signaling</span> in the Drosophila immune response. </a></p><div class="supp"><p class="details"><div>Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its multiple modes of activation in the <i>Drosophila</i> immune response. Three distinct mechanisms for microbial recognition, leading to the cleavage of SpÃ¤tzle and activation of Toll, are illustrated. The mechanisms include peptidoglycan detection, by PGRP-SA, PGRP-SD and GNBP1, Î²-glucan detection through GNBP3, and protease activity sensing via the serine protease Persephone. All these detection modalities lead to the activation of the SpÃ¤tzle processing enzyme (SPE) which converts this cytokine into its active form, for binding and activating Toll. The intracellular signal transduction downstream of Toll is very similar to the MyD88-dependent <span class="highlight" style="background-color:">pathway</span>, which functions downstream of most mammalian TLRs. The key features of this <span class="highlight" style="background-color:">pathway</span> include a trimeric receptor associated complex, containing MyD88, Tube and Pelle, which ultimately lead to the phosphorylation and degradation of the IÎºB homology Cactus and the nuclear translocation of NF-ÎºB homologs Dif and Dorsal.</div></p></div><div class="aux"><div class="resc">N Silverman, et al. Invertebrate Surviv J. ;6(2):163-174.</div><p class="links"><a class="dblinks" href="/pubmed/21625362" ref="ordinalpos=2790&amp;ncbi_uid=2968344&amp;link_uid=21625362">Citation</a><a class="dblinks" href="/pmc/articles/PMC3101770/" ref="ordinalpos=2790&amp;ncbi_uid=2968344&amp;link_uid=PMC3101770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2791.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412266/figure/F1/" ref="ordinalpos=2791&amp;ncbi_uid=4171767&amp;link_uid=PMC3412266" href="/pmc/articles/PMC3412266/"><img src="/pmc/articles/instance/3412266/bin/fimmu-03-00241-g001.gif" src-large="/pmc/articles/instance/3412266/bin/fimmu-03-00241-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412266/" ref="ordinalpos=2791&amp;ncbi_uid=4171767&amp;link_uid=PMC3412266" image-link="/pmc/articles/PMC3412266/figure/F1/" class="imagepopup">Figure 1.  From: Th1/Th2 Differentiation and B Cell Function by the Atypical PKCs and Their Regulators. </a></p><div class="supp"><p class="details"><div><b>Atypical PKC <span class="highlight" style="background-color:">signaling</span> pathways leading to lymphocyte activation and/or differentiation</b>. <b>(A)</b> <span class="highlight" style="background-color:">Signaling</span> through B cell antigen receptor (BCR) in B-lymphocytes. PKCÎ¶ is required for the activation of ERK after antigen challenge and necessary for IÎºB transcription and for the activation and nuclear translocation of c-Fos. The transcription of NF-ÎºB-dependent genes IL-6 and Bcl-X<sub>L</sub>, and the secretion of T-dependent immunoglobulins depend on PKCÎ¶-mediated <span class="highlight" style="background-color:">signaling</span>. The aPKC inhibitor Par-4 blocks PKCÎ¶ activation and B cell proliferation. <b>(B)</b> <span class="highlight" style="background-color:">Signaling</span> pathways in T lymphocytes. After antigen presentation, downstream <span class="highlight" style="background-color:">signaling</span> through T cell receptor (TCR) takes place. Atypical PKCÎ¶ and PKCÎ»/É© are important for IKK activation and NF-ÎºB nuclear translocation, together with p62 scaffold protein that links the aPKCs with TCR activation through MALT-1 and TRAF-6 proteins. Moreover PKCÎ»/É© activates JNK and NFATc1 nuclear translocation, required for Th2 differentiation of T cells. <span class="highlight" style="background-color:">Signaling</span> through IL-4 receptor (IL-4R) requires both atypical PKCs to fully activate Jak1/Stat6 <span class="highlight" style="background-color:">pathway</span> and therefore to Th2 differentiation. Par 4 inhibits TCR and IL-4R <span class="highlight" style="background-color:">signaling</span> pathways blocking the activation of aPKCs.</div></p></div><div class="aux"><div class="resc">Pilar Martin, et al. Front Immunol. 2012;3:241.</div><p class="links"><a class="dblinks" href="/pubmed/22888333" ref="ordinalpos=2791&amp;ncbi_uid=4171767&amp;link_uid=22888333">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412266/" ref="ordinalpos=2791&amp;ncbi_uid=4171767&amp;link_uid=PMC3412266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2792.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4385934/figure/fig3/" ref="ordinalpos=2792&amp;ncbi_uid=7392845&amp;link_uid=PMC4385934" href="/pmc/articles/PMC4385934/"><img src="/pmc/articles/instance/4385934/bin/cddis201556f3.gif" src-large="/pmc/articles/instance/4385934/bin/cddis201556f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4385934/" ref="ordinalpos=2792&amp;ncbi_uid=7392845&amp;link_uid=PMC4385934" image-link="/pmc/articles/PMC4385934/figure/fig3/" class="imagepopup">Figure 3.  From: Fgf9 inhibition of meiotic differentiation in spermatogonia is mediated by Erk-dependent activation of Nodal-Smad2/3 <span class="highlight" style="background-color:">signaling</span> and is antagonized by Kit Ligand. </a></p><div class="supp"><p class="details"><div>Expression and activity of the Cripto-Nodal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in postnatal mouse spermatogonia. (<b>a</b>) Semiquantitative RT-PCR analysis of mRNA expression of the indicated elements of the Cripto-Nodal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Kit+ and Kit- spermatogonia. (<b>b</b>) Western blot analysis of Smad2 phosphorylation in cultured untreated Kit+ spermatogonia and in the same cells treated overnight with recombinant Nodal, in the presence or absence of the Alk4/7 selective inhibitor SB431542. Densitometric analysis of western blots from three independent experiments is shown below. Bars represent the mean Â±S.D.</div></p></div><div class="aux"><div class="resc">V Tassinari, et al. Cell Death Dis. 2015 March;6(3):e1688.</div><p class="links"><a class="dblinks" href="/pubmed/25766327" ref="ordinalpos=2792&amp;ncbi_uid=7392845&amp;link_uid=25766327">Citation</a><a class="dblinks" href="/pmc/articles/PMC4385934/" ref="ordinalpos=2792&amp;ncbi_uid=7392845&amp;link_uid=PMC4385934">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2793.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3832994/figure/F2/" ref="ordinalpos=2793&amp;ncbi_uid=5551590&amp;link_uid=PMC3832994" href="/pmc/articles/PMC3832994/"><img src="/pmc/articles/instance/3832994/bin/nihms-412631-f0002.gif" src-large="/pmc/articles/instance/3832994/bin/nihms-412631-f0002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3832994/" ref="ordinalpos=2793&amp;ncbi_uid=5551590&amp;link_uid=PMC3832994" image-link="/pmc/articles/PMC3832994/figure/F2/" class="imagepopup">FIGURE 2.  From: Role of Epidermal Growth Factor-Triggered PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> in the Migration of Medulloblastoma-Derived Cells. </a></p><div class="supp"><p class="details"><div>EGF-R phosphorylation levels and localization are mediated by EGF <span class="highlight" style="background-color:">pathway</span> stimulation and inhibition in MB-derived cells. Localization of total EGF-R (iEGF-R) in non-stimulated cells (a), in cells stimulated with 1000 ng/mL EGF (b) and in cells in which EGF <span class="highlight" style="background-color:">signaling</span> was inhibited by 1 <i>Î¼</i>M Tarceva (c). (d) Levels of iEGF-R are not changed due to activation/inactivation of the EGF <span class="highlight" style="background-color:">pathway</span>. Localization of phosphorylated EGF-R (pEGF-R) in non-stimulated cells (e), in cells stimulated with 1000 ng/mL EGF (f) and in cells in which EGF <span class="highlight" style="background-color:">signaling</span> was inhibited by 1 <i>Î¼</i>M Tarceva (g). (h) Levels of pEGF-R are significantly increased due to activation of the EGF <span class="highlight" style="background-color:">pathway</span>. (i) iEGF-R (green) (i1) accumulates in Transferrin (Tf)-positive vesicles (purple) (i2) in EGF-stimulated cells; merged channels (i3). Insert shows higher magnification of the boxed area and co-localization between iEGF-R and Tf. (j) Co-localization between iEGF-R and Tf-positive vesicles, and between iEGF-R and CT-B-positive vesicles in EGF-stimulated cells. Scale bars, 20 <i>Î¼</i>m. * <i>p</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Veronica Dudu, et al. Cell Mol Bioeng. ;5(4):502-413.</div><p class="links"><a class="dblinks" href="/pubmed/24273611" ref="ordinalpos=2793&amp;ncbi_uid=5551590&amp;link_uid=24273611">Citation</a><a class="dblinks" href="/pmc/articles/PMC3832994/" ref="ordinalpos=2793&amp;ncbi_uid=5551590&amp;link_uid=PMC3832994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2794.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1839894/figure/F4/" ref="ordinalpos=2794&amp;ncbi_uid=889625&amp;link_uid=PMC1839894" href="/pmc/articles/PMC1839894/"><img src="/pmc/articles/instance/1839894/bin/1752-0509-1-4-4.gif" src-large="/pmc/articles/instance/1839894/bin/1752-0509-1-4-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1839894/" ref="ordinalpos=2794&amp;ncbi_uid=889625&amp;link_uid=PMC1839894" image-link="/pmc/articles/PMC1839894/figure/F4/" class="imagepopup">Figure 4.  From: Qualitative networks: a symbolic approach to analyze biological <span class="highlight" style="background-color:">signaling</span> networks. </a></p><div class="supp"><p class="details"><div><b>Visualization of a single cell</b>. The components of the Wnt <span class="highlight" style="background-color:">pathway</span> are represented in red and the components of the Notch <span class="highlight" style="background-color:">pathway</span> in blue. The canonical Wnt <span class="highlight" style="background-color:">pathway</span> starts from the extracellular level of the short range <span class="highlight" style="background-color:">signaling</span> Wnt protein, which we represent by the <i>Wnt</i><sub>ext </sub>variable. This short range molecule binds to the <i>Frizzled </i>receptor, which leads to an increase in the intracellular level of <i>DSH</i>. We therefore represent the interaction between these molecules by an activation from Wnt<sub><i>ext </i></sub>to Frizzled and then to DSH. The level of the scaffolding protein <i>Axin </i>is dependent on the level of DSH which acts as an inhibitor and Casein Kinase 1 <i>Î± </i>(represented by the variable <i>Cask1Î±</i>), which is assumed to be an activator. <i>Î²-Catenin </i>is phosphorylated by a complex composed, amongst others, by Axin. We separately represent the expression level of <i>Î²</i>-Catenin, <i>Î²-Catenin</i><sub><i>exp</i></sub>. <i>Î²</i>-Catenin is activated by <i>Î²</i>-Catenin<sub><i>exp </i></sub>and inhibited by Axin. The family of downstream target genes of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, (<i>GT1</i>), are transcriptionally activated by <i>Î²</i>-Catenin. The Notch <span class="highlight" style="background-color:">pathway</span> is based on the cell-cell interaction between the Notch receptor (whose level is represented by the variable <i>Notch</i>) and the ligands expressed on the membrane of the immediate neighboring cells (<i>Ligand</i><sub><i>in</i></sub>). Since a successful binding between the receptor and the ligand leads to the cleavage of the intracellular part of Notch (<i>Notch-IC</i>) we use the minimum target function to model this interaction. We represent the level of the downstream target genes of Notch <span class="highlight" style="background-color:">signaling</span> as a single variable (<i>GT2</i>), which is activated by Notch-IC. Amongst these targets is the protein <i>p21</i>, which is activated by Notch-IC and inhibits <i>Wnt</i>. Wnt is an external variable of the cell. The downstream targets of Notch <span class="highlight" style="background-color:">signaling</span> activate the <i>Jagged </i>ligand.</div></p></div><div class="aux"><div class="resc">Marc A Schaub, et al. BMC Syst Biol. 2007;1:4-4.</div><p class="links"><a class="dblinks" href="/pubmed/17408511" ref="ordinalpos=2794&amp;ncbi_uid=889625&amp;link_uid=17408511">Citation</a><a class="dblinks" href="/pmc/articles/PMC1839894/" ref="ordinalpos=2794&amp;ncbi_uid=889625&amp;link_uid=PMC1839894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2795.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3947389/figure/F6/" ref="ordinalpos=2795&amp;ncbi_uid=5900145&amp;link_uid=PMC3947389" href="/pmc/articles/PMC3947389/"><img src="/pmc/articles/instance/3947389/bin/nihms4165f6.gif" src-large="/pmc/articles/instance/3947389/bin/nihms4165f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3947389/" ref="ordinalpos=2795&amp;ncbi_uid=5900145&amp;link_uid=PMC3947389" image-link="/pmc/articles/PMC3947389/figure/F6/" class="imagepopup">Figure 6.  From: Microarray Analysis Identifies Changes in Inflammatory Gene Expression in Response to Amyloid-Î² Stimulation of Cultured Human Retinal Pigment Epithelial Cells. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) software program. Note that the six top-ranked pathways, leukocyte extravasation <span class="highlight" style="background-color:">signaling</span>, natural killer cell <span class="highlight" style="background-color:">signaling</span>, hepatic cholestasis, <i>FXR-RXR</i> activation, <i>IL-10</i> <span class="highlight" style="background-color:">signaling</span> and the complement system, reached the criteria set at <i>P</i> &lt; 0.05. All six are associated with inflammation.</div></p></div><div class="aux"><div class="resc">Khaliq H. Kurji, et al. Invest Ophthalmol Vis Sci. 2010 February;51(2):1151-1163.</div><p class="links"><a class="dblinks" href="/pubmed/19797223" ref="ordinalpos=2795&amp;ncbi_uid=5900145&amp;link_uid=19797223">Citation</a><a class="dblinks" href="/pmc/articles/PMC3947389/" ref="ordinalpos=2795&amp;ncbi_uid=5900145&amp;link_uid=PMC3947389">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2796.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2486832/figure/F3/" ref="ordinalpos=2796&amp;ncbi_uid=1499842&amp;link_uid=PMC2486832" href="/pmc/articles/PMC2486832/"><img src="/pmc/articles/instance/2486832/bin/nihms55797f3.gif" src-large="/pmc/articles/instance/2486832/bin/nihms55797f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2486832/" ref="ordinalpos=2796&amp;ncbi_uid=1499842&amp;link_uid=PMC2486832" image-link="/pmc/articles/PMC2486832/figure/F3/" class="imagepopup">Figure 3. Insulin/Igf1-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: SIRT6 IN DNA REPAIR, METABOLISM, AND AGEING. </a></p><div class="supp"><p class="details"><div>For details see text.</div></p></div><div class="aux"><div class="resc">David B. Lombard, et al. J Intern Med. ;263(2):128-141.</div><p class="links"><a class="dblinks" href="/pubmed/18226091" ref="ordinalpos=2796&amp;ncbi_uid=1499842&amp;link_uid=18226091">Citation</a><a class="dblinks" href="/pmc/articles/PMC2486832/" ref="ordinalpos=2796&amp;ncbi_uid=1499842&amp;link_uid=PMC2486832">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2797.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC324551/figure/F3/" ref="ordinalpos=2797&amp;ncbi_uid=432088&amp;link_uid=PMC324551" href="/pmc/articles/PMC324551/"><img src="/pmc/articles/instance/324551/bin/JCI0420986.f3.gif" src-large="/pmc/articles/instance/324551/bin/JCI0420986.f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC324551/" ref="ordinalpos=2797&amp;ncbi_uid=432088&amp;link_uid=PMC324551" image-link="/pmc/articles/PMC324551/figure/F3/" class="imagepopup">Figure 3.  From: Central role of the P2Y12 receptor in platelet activation. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> events downstream of the P2Y<sub>1</sub> and P2Y<sub>12</sub> receptors. ADP binds to the P2Y<sub>12</sub> receptor and causes a number of intracellular <span class="highlight" style="background-color:">signaling</span> events downstream of the G<sub>i</sub> <span class="highlight" style="background-color:">pathway</span> that contribute to fibrinogen receptor activation and platelet aggregation. The P2Y<sub>12</sub> receptorâ€“mediated inhibition of adenylyl cyclase is not directly responsible for fibrinogen receptor activation. Potassium channels and PI3K are also activated by the P2Y<sub>12</sub> receptor. Both Rap1b and Akt are <span class="highlight" style="background-color:">signaling</span> mediators that contribute to platelet aggregation and are activated in a PI3K-dependent manner. Other mediators of P2Y<sub>12</sub> <span class="highlight" style="background-color:">signaling</span> remain to be elucidated.</div></p></div><div class="aux"><div class="resc">Robert T. Dorsam, et al. J Clin Invest. 2004 February 1;113(3):340-345.</div><p class="links"><a class="dblinks" href="/pubmed/14755328" ref="ordinalpos=2797&amp;ncbi_uid=432088&amp;link_uid=14755328">Citation</a><a class="dblinks" href="/pmc/articles/PMC324551/" ref="ordinalpos=2797&amp;ncbi_uid=432088&amp;link_uid=PMC324551">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2798.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1456257/figure/fig3/" ref="ordinalpos=2798&amp;ncbi_uid=628444&amp;link_uid=PMC1456257" href="/pmc/articles/PMC1456257/"><img src="/pmc/articles/instance/1456257/bin/GEN1722943f3.gif" src-large="/pmc/articles/instance/1456257/bin/GEN1722943f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1456257/" ref="ordinalpos=2798&amp;ncbi_uid=628444&amp;link_uid=PMC1456257" image-link="/pmc/articles/PMC1456257/figure/fig3/" class="imagepopup">Figure 3. From: Presynaptic UNC-31 (CAPS) Is Required to Activate the G?s <span class="highlight" style="background-color:">Pathway</span> of the Caenorhabditis elegans Synaptic <span class="highlight" style="background-color:">Signaling</span> Network. </a></p><div class="supp"><p class="details"><div>UNC-31 (CAPS) null mutants and mutants lacking a neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> share similar phenotypes. Note that for purposes of clarity, GÎ±<sub>q</sub>, GÎ±<sub>s</sub>, and GÎ±<sub>o</sub> are at times referred to as Gq, Gs, and Go in this and subsequent figures. (A) <i>unc-31</i> null mutants look similar to mutants that lack a neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> when viewed on culture plates through a stereomicroscope. Images compare a typical N2 (wild-type) animal with a typical <i>unc-31(e928)</i> null mutant and a mutant that lacks a neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span>. The latter's genotype is <i>acy-1(pk1279); ceEx108</i> [<i>myo-3âˆ·acy-1(+)</i> cDNA], which expresses <i>acy-1</i> (adenylyl cyclase) only in muscle cells and thus is null for <i>acy-1</i> in the nervous system (Reynolds et al. 2005). All three animals are feeding in a lawn of bacteria. Note the similar straight posture associated with the paralysis of the <i>unc-31</i> and neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> null mutants. The image â€œMutant lacking a neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span>â€ is reprinted with permission from Reynolds et al. (2005) for comparison. (B) <i>unc-31</i> and neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> null mutants exhibit a similar degree of paralysis. Shown are the mean locomotion rates, expressed as body bends per minute, of the indicated strains. An <i>egl-30</i> (GÎ±<sub>q</sub>) reduction-of-function mutant (<i>ad805</i>) (Brundage et al. 1996) that is paralyzed to a similar extent as mutants lacking a neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> or UNC-31 is shown for comparison. <i>unc-31</i> null mutants are only slightly less paralyzed than animals that lack a neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span>. The bar on the right demonstrates that a transgenic array containing the <i>unc-31 (+)</i> cDNA expressed in the nervous system fully rescues the locomotion defect of the <i>unc-31(e928)</i> null. Error bars represent the standard error of the mean for populations of 10 animals. Transgenic arrays are <i>ceEx108</i> [<i>myo-3âˆ·acy-1(+)</i> cDNA (expressed only in muscle)] and <i>ceEx117</i> [<i>rab-3âˆ·unc-31(+)</i> cDNA (expressed only in the nervous system)]. The value for <i>acy-1(pk1279); ceEx108</i> is reprinted with permission from Reynolds et al. (2005) for comparison. (C) <i>unc-31</i> and neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> null mutants respond normally to the cholinergic agonist levamisole. The graph depicts the contraction of individual animals after a 20-min exposure to various concentrations of levamisole. Error bars represent the standard errors for six individual animals at each concentration. <i>acy-1(pk1279); ceEx108</i> (a transgenic muscle-rescued <i>acy-1</i> null mutant) was used as the neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> null. Note that the levamisole sensitivity of <i>unc-31(e928)</i> is indistinguishable from wild type, while the neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> null appears only slightly hypersensitive to levamisole (although this is not statistically significant at any concentration). We did not investigate the extent to which overexpression of the <i>acy-1(+)</i> transgene cDNA in muscle may have contributed to the slight levamisole hypersensitivity, as has been previously shown to occur with muscle-specific expression of <i>acy-1(gf)</i> (Schade et al. 2005). Statistical significance tests used the unpaired <i>t</i>-test with Welch correction, with the level of statistical significance set at <i>P</i> = 0.05. The data for N2 and the neuronal Gs <span class="highlight" style="background-color:">pathway</span> null are reprinted with permission from Reynolds et al. (2005) for comparison and were generated as part of the same experiment with the <i>unc-31</i> null. (D) <i>unc-31</i> and neuronal GÎ±<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> null mutants exhibit near-normal aldicarb sensitivity. Shown are the population growth rates of strains on various concentrations of the acetylcholinesterase inhibitor aldicarb. One hundred percent represents the number of progeny produced from a starting population of L1 larvae over a 96-hr period in the absence of aldicarb. Curves are representative of duplicate experiments. Complete genotypes are as follows: <i>unc-31</i> null, <i>unc-31(e928)</i>; GÎ±<sub>q</sub> reduction-of-function mutant, <i>egl-30(ad805)</i> (B<span style="font-variant: small-caps" class="small-caps">rundage</span> <i>et al</i>. 1996); and neuronal Gs <span class="highlight" style="background-color:">pathway</span> null, <i>acy-1(pk1279); ceEx108</i> (a transgenic muscle-rescued <i>acy-1</i> null mutant) (Reynolds et al. 2005). The data for N2, <i>egl-30(ad805)</i>, and the neuronal Gs <span class="highlight" style="background-color:">pathway</span> null are reprinted with permission from Reynolds et al. (2005) for comparison and were generated as part of the same experiment with the <i>unc-31</i> null.</div></p></div><div class="aux"><div class="resc">Nicole K. Charlie, et al. Genetics. 2006 February;172(2):943-961.</div><p class="links"><a class="dblinks" href="/pubmed/16272411" ref="ordinalpos=2798&amp;ncbi_uid=628444&amp;link_uid=16272411">Citation</a><a class="dblinks" href="/pmc/articles/PMC1456257/" ref="ordinalpos=2798&amp;ncbi_uid=628444&amp;link_uid=PMC1456257">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2799.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657945/figure/DDP032F1/" ref="ordinalpos=2799&amp;ncbi_uid=1707030&amp;link_uid=PMC2657945" href="/pmc/articles/PMC2657945/"><img src="/pmc/articles/instance/2657945/bin/ddp03201.gif" src-large="/pmc/articles/instance/2657945/bin/ddp03201.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657945/" ref="ordinalpos=2799&amp;ncbi_uid=1707030&amp;link_uid=PMC2657945" image-link="/pmc/articles/PMC2657945/figure/DDP032F1/" class="imagepopup">Figure 1. From: Tuberous sclerosis complex, implication from a rare genetic disease to common cancer treatment. </a></p><div class="supp"><p class="details"><div>A model of the TSC-mTOR <span class="highlight" style="background-color:">signaling</span>. Phosphorylation-dependent <span class="highlight" style="background-color:">signaling</span> cascades of the TSC-mTOR <span class="highlight" style="background-color:">pathway</span> are depicted. The kinases and their target phosphorylation are denoted with the same color.</div></p></div><div class="aux"><div class="resc">Ken Inoki, et al. Hum Mol Genet. 2009 April 15;18(R1):R94-R100.</div><p class="links"><a class="dblinks" href="/pubmed/19297407" ref="ordinalpos=2799&amp;ncbi_uid=1707030&amp;link_uid=19297407">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657945/" ref="ordinalpos=2799&amp;ncbi_uid=1707030&amp;link_uid=PMC2657945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2800.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3745454/figure/pone-0070956-g009/" ref="ordinalpos=2800&amp;ncbi_uid=5221561&amp;link_uid=PMC3745454" href="/pmc/articles/PMC3745454/"><img src="/pmc/articles/instance/3745454/bin/pone.0070956.g009.gif" src-large="/pmc/articles/instance/3745454/bin/pone.0070956.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3745454/" ref="ordinalpos=2800&amp;ncbi_uid=5221561&amp;link_uid=PMC3745454" image-link="/pmc/articles/PMC3745454/figure/pone-0070956-g009/" class="imagepopup">Figure 9. Schematic diagram depicting protective <span class="highlight" style="background-color:">signaling</span> of GSRd in MI/R-induced apoptosis..  From: Ginsenoside Rd Attenuates Myocardial Ischemia/Reperfusion Injury via Akt/GSK-3? <span class="highlight" style="background-color:">Signaling</span> and Inhibition of the Mitochondria-Dependent Apoptotic <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>GSRd inhibits the apoptotic <span class="highlight" style="background-color:">signaling</span> cascades initiated by MI/R-generated ROS. Arrows (â†’) indicate activation or induction, and segments ending with a (âŠ¢) indicate inhibition/blockade. Solid lines (â€”) indicate mechanisms strongly supported by the current study, and dotted lines (--) indicate hypothesized connections requiring further investigations.</div></p></div><div class="aux"><div class="resc">Yang Wang, et al. PLoS One. 2013;8(8):e70956.</div><p class="links"><a class="dblinks" href="/pubmed/23976968" ref="ordinalpos=2800&amp;ncbi_uid=5221561&amp;link_uid=23976968">Citation</a><a class="dblinks" href="/pmc/articles/PMC3745454/" ref="ordinalpos=2800&amp;ncbi_uid=5221561&amp;link_uid=PMC3745454">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="27" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="28" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="29" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:36:22-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB1FC7543107C10000000000571741;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB1FC7543107C10000000000571741" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>