
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="25" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC55A0543105E100000000001AD71C" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="24" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="25" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="26" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="25" /></div>
            <div><h2 class="result_count">Results: 2401 to 2500 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2401.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3369616/figure/F1/" ref="ordinalpos=2401&amp;ncbi_uid=4936873&amp;link_uid=PMC3369616" href="/pmc/articles/PMC3369616/"><img src="/pmc/articles/instance/3369616/bin/zh89991291470001.gif" src-large="/pmc/articles/instance/3369616/bin/zh89991291470001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3369616/" ref="ordinalpos=2401&amp;ncbi_uid=4936873&amp;link_uid=PMC3369616" image-link="/pmc/articles/PMC3369616/figure/F1/" class="imagepopup">Figure 1.  From: Heme-regulated eIF2? kinase activated Atf4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oxidative stress and erythropoiesis. </a></p><div class="supp"><p class="details"><div><b>Induction of the Hri-eIF2αP-Atf4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and protection against oxidative stress in FL erythroblasts.</b> (A) ROS levels. <i>Hri</i><sup>+/+</sup> FL cells were treated with 25μM sodium arsenite. At times indicated, cellular ROS levels were measured by DCF fluorescence. (B) Hyperphosphorylation and activation of Hri. Hyperphosphorylation of Hri was examined by the slower migration in SDS-PAGE. eIF2α was used as a loading control. (C) Induction of the Hri-dependent Atf4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cells were treated with 5μM As. At times indicated, samples were analyzed for expression of eIF2αP, Atf4, Chop, and eIF2α proteins. (D) Relative expression of mRNAs of the Hri-Atf4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components. <i>Atf4</i>, <i>Chop</i>, <i>Gadd34</i>, and <i>Crep</i> mRNA levels from FL cells treated with 5μM As for 3 hours were analyzed by qPCR. Expression of each of the mRNA in untreated <i>Hri</i><sup>+/+</sup> cells was defined as 1. Data are presented as mean ± SD (n = 3). <i>P</i> values denote the comparison between arsenite-treated <i>Hri</i><sup>+/+</sup> and <i>Hri</i><sup>−/−</sup> samples; <i>P</i> &lt; .001 for <i>Atf4</i> and <i>Gadd34</i>; <i>P</i> &lt; .005 for <i>Chop</i>. (E) ROS levels. <i>Hri</i><sup>+/+</sup> and <i>Hri</i><sup>−/−</sup> FL cells treated with 5 or 25μM arsenite for 9 hours before ROS measurement.</div></p></div><div class="aux"><div class="resc">Rajasekhar N. V. S. Suragani, et al. Blood. 2012 May 31;119(22):5276-5284.</div><p class="links"><a class="dblinks" href="/pubmed/22498744" ref="ordinalpos=2401&amp;ncbi_uid=4936873&amp;link_uid=22498744">Citation</a><a class="dblinks" href="/pmc/articles/PMC3369616/" ref="ordinalpos=2401&amp;ncbi_uid=4936873&amp;link_uid=PMC3369616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2402.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4260770/figure/f3-cin-suppl.2-2014-113/" ref="ordinalpos=2402&amp;ncbi_uid=6969889&amp;link_uid=PMC4260770" href="/pmc/articles/PMC4260770/"><img src="/pmc/articles/instance/4260770/bin/cin-suppl.2-2014-113f3.gif" src-large="/pmc/articles/instance/4260770/bin/cin-suppl.2-2014-113f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4260770/" ref="ordinalpos=2402&amp;ncbi_uid=6969889&amp;link_uid=PMC4260770" image-link="/pmc/articles/PMC4260770/figure/f3-cin-suppl.2-2014-113/" class="imagepopup">Figure 3.  From: Bayesian Joint Selection of Genes and Pathways: Applications in Multiple Myeloma Genomics. </a></p><div class="supp"><p class="details"><div>The 95% posterior credible intervals of the coefficients for the gene variables in the selected pathways: (<b>A</b>) galactose metabolism <span class="highlight" style="background-color:">pathway</span>; (<b>B</b>) cell cycle <span class="highlight" style="background-color:">pathway</span>; (<b>C</b>) Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Lin Zhang, et al. Cancer Inform. 2014;13(Suppl 2):113-123.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2402&amp;ncbi_uid=6969889&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4260770/" ref="ordinalpos=2402&amp;ncbi_uid=6969889&amp;link_uid=PMC4260770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2403.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3918904/figure/F1/" ref="ordinalpos=2403&amp;ncbi_uid=5801605&amp;link_uid=PMC3918904" href="/pmc/articles/PMC3918904/"><img src="/pmc/articles/instance/3918904/bin/nihms549146f1.gif" src-large="/pmc/articles/instance/3918904/bin/nihms549146f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3918904/" ref="ordinalpos=2403&amp;ncbi_uid=5801605&amp;link_uid=PMC3918904" image-link="/pmc/articles/PMC3918904/figure/F1/" class="imagepopup">Figure 1. The levels of factors encoded by putative target transcripts of miRNAs donwregulated by resveratrol increase correspondingly.  From: Resveratrol modulates the levels of microRNAs targeting genes encoding tumor-suppressors and effectors of TGF? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in SW480 cells. </a></p><div class="supp"><p class="details"><div>The levels in SW480 cells of tumor suppressors as well as key effectors of TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were assessed by Western blotting using the indicated antibodies. The relative levels following resveratrol treatment are given in percent of the corresponding control vehicle sample. Panels A to D were from different gels.</div></p></div><div class="aux"><div class="resc">Esmerina Tili, et al. Biochem Pharmacol. ;80(12):2057-2065.</div><p class="links"><a class="dblinks" href="/pubmed/20637737" ref="ordinalpos=2403&amp;ncbi_uid=5801605&amp;link_uid=20637737">Citation</a><a class="dblinks" href="/pmc/articles/PMC3918904/" ref="ordinalpos=2403&amp;ncbi_uid=5801605&amp;link_uid=PMC3918904">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2404.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2258146/figure/pone-0001761-g002/" ref="ordinalpos=2404&amp;ncbi_uid=1234041&amp;link_uid=PMC2258146" href="/pmc/articles/PMC2258146/"><img src="/pmc/articles/instance/2258146/bin/pone.0001761.g002.gif" src-large="/pmc/articles/instance/2258146/bin/pone.0001761.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2258146/" ref="ordinalpos=2404&amp;ncbi_uid=1234041&amp;link_uid=PMC2258146" image-link="/pmc/articles/PMC2258146/figure/pone-0001761-g002/" class="imagepopup">Figure 2. Mutants of the Hpo <span class="highlight" style="background-color:">pathway</span> disrupt PFC differentiation..  From: The Hippo <span class="highlight" style="background-color:">Pathway</span> Promotes Notch <span class="highlight" style="background-color:">Signaling</span> in Regulation of Cell Differentiation, Proliferation, and Oocyte Polarity. </a></p><div class="supp"><p class="details"><div>The PFC markers <i>pnt-lacZ</i> (A) and <i>667/9-lacZ</i> (C) are specifically expressed in the PFC after stage 6 in wild-type egg chambers. (B and D) <i>hpo</i> PFC clones fail to express <i>pnt-lacZ</i> (B) or <i>667/9-lacZ</i> (D), in a cell-autonomous manner (arrows). (E) Activation of JAK-STAT <span class="highlight" style="background-color:">signaling</span> in the PFC (arrow) can be marked by the expression of <i>dome-lacZ</i>. (F) In <i>sav</i> PFC clones, expression of <i>dome-lacZ</i> is not affected (arrow). The AFC markers <i>slbo-lacZ</i> (G) and <i>dpp-lacZ</i> (I) are expressed in the AFCs in stage-9 wild-type egg chambers. In <i>sav</i> mutant PFC, no misexpression of <i>slbo-lacZ</i> (H) or <i>dpp-lacZ</i> (J) was detected.</div></p></div><div class="aux"><div class="resc">Jianzhong Yu, et al. PLoS ONE. 2008;3(3):e1761.</div><p class="links"><a class="dblinks" href="/pubmed/18335037" ref="ordinalpos=2404&amp;ncbi_uid=1234041&amp;link_uid=18335037">Citation</a><a class="dblinks" href="/pmc/articles/PMC2258146/" ref="ordinalpos=2404&amp;ncbi_uid=1234041&amp;link_uid=PMC2258146">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2405.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC420486/figure/F5/" ref="ordinalpos=2405&amp;ncbi_uid=402678&amp;link_uid=PMC420486" href="/pmc/articles/PMC420486/"><img src="/pmc/articles/instance/420486/bin/1476-4598-3-15-5.gif" src-large="/pmc/articles/instance/420486/bin/1476-4598-3-15-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC420486/" ref="ordinalpos=2405&amp;ncbi_uid=402678&amp;link_uid=PMC420486" image-link="/pmc/articles/PMC420486/figure/F5/" class="imagepopup">Figure 5.  From: Phospholipase C ?-4 overexpression upregulates ErbB1/2 expression, Erk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and proliferation in MCF-7 cells. </a></p><div class="supp"><p class="details"><div><b>Overexpression of PLCδ4 activates Erk1/2 <span class="highlight" style="background-color:">pathway</span> in MCF-7 cells</b>. Western blot analysis of MCF-7 cell extracts derived from 4 independent isolates overexpressing PLCδ4 (left 4 lanes) and 2 isolates transfected with control vectors (right 2 lanes) using antibodies against various phosphorylated forms of kinases and non-phosphorylated Erk1/2. Antibodies were from Cell <span class="highlight" style="background-color:">Signaling</span> Technology (Beverly, MA). Blotting was done using the ECL Plus™ detection reagents from Amersham Biosciences (Piscataway, NJ).</div></p></div><div class="aux"><div class="resc">David W Leung, et al. Mol Cancer. 2004;3:15-15.</div><p class="links"><a class="dblinks" href="/pubmed/15140260" ref="ordinalpos=2405&amp;ncbi_uid=402678&amp;link_uid=15140260">Citation</a><a class="dblinks" href="/pmc/articles/PMC420486/" ref="ordinalpos=2405&amp;ncbi_uid=402678&amp;link_uid=PMC420486">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2406.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC95007/figure/F5/" ref="ordinalpos=2406&amp;ncbi_uid=152024&amp;link_uid=PMC95007" href="/pmc/articles/PMC95007/"><img src="/pmc/articles/instance/95007/bin/jb0411265005.gif" src-large="/pmc/articles/instance/95007/bin/jb0411265005.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC95007/" ref="ordinalpos=2406&amp;ncbi_uid=152024&amp;link_uid=PMC95007" image-link="/pmc/articles/PMC95007/figure/F5/" class="imagepopup">FIG. 5.  From: New Family of Regulators in the Environmental <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Which Activates the General Stress Transcription Factor ?B of Bacillus subtilis. </a></p><div class="supp"><p class="details"><div>Analyses of triple and quadruple mutants indicate that YojH and YqhA act in concert with RsbR and YkoB in the environmental <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A <i>ctc-lacZ</i> transcriptional fusion was used to measure ς<sup>B</sup> activity in unstressed cells during logarithmic growth and also during entry into stationary phase. (A) PB198 (wild type, ○); PB531 (<i>rsbRΔ1 ykoBΔ1</i>::<i>kan</i>, ▵); PB653 (<i>rsbRΔ1 ykoBΔ1</i>::<i>kan yojHΔ1</i>::<i>ery</i>, ●); PB530 (<i>rsbRΔ1 ykoBΔ1</i>::<i>kan yqhAΔ1</i>::<i>spc</i>, ■); and quadruple mutant PB629 (<i>rsbRΔ1 ykoBΔ1</i>::<i>kan yojHΔ1</i>::<i>ery yqhAΔ1</i>::<i>spc</i>, ▴). (B) PB198 (wild type, ○); quadruple mutant PB629 (<i>rsbRΔ1 ykoBΔ1</i>::<i>kan yojHΔ1</i>::<i>ery yqhAΔ1</i>::<i>spc</i>, ▴); PB639 (<i>rsbRΔ1 yojHΔ1</i>::<i>ery yqhAΔ1</i>::<i>spc</i>, ▾); and PB654 (<i>rsbRΔ1 rsbTΔ1 ykoBΔ1</i>::<i>kan yojHΔ1</i>::<i>ery yqhAΔ1</i>::<i>spc</i>, ⧫)</div></p></div><div class="aux"><div class="resc">Samina Akbar, et al. J Bacteriol. 2001 February;183(4):1329-1338.</div><p class="links"><a class="dblinks" href="/pubmed/11157946" ref="ordinalpos=2406&amp;ncbi_uid=152024&amp;link_uid=11157946">Citation</a><a class="dblinks" href="/pmc/articles/PMC95007/" ref="ordinalpos=2406&amp;ncbi_uid=152024&amp;link_uid=PMC95007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2407.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3627483/figure/F2/" ref="ordinalpos=2407&amp;ncbi_uid=4810357&amp;link_uid=PMC3627483" href="/pmc/articles/PMC3627483/"><img src="/pmc/articles/instance/3627483/bin/nihms451254f2.gif" src-large="/pmc/articles/instance/3627483/bin/nihms451254f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3627483/" ref="ordinalpos=2407&amp;ncbi_uid=4810357&amp;link_uid=PMC3627483" image-link="/pmc/articles/PMC3627483/figure/F2/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Pathway</span>-Specific Analysis of Gene Expression Data Identifies the PI3K/Akt <span class="highlight" style="background-color:">Pathway</span> as a Novel Therapeutic Target in Cervical Cancer. </a></p><div class="supp"><p class="details"><div>GSEA enrichment plot of KEGG endometrial cancer <span class="highlight" style="background-color:">pathway</span> genes in PET<sup>+</sup> versus PET<sup>−</sup> tumors. Genes in the KEGG endometrial cancer <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showed significant enrichment in PET<sup>+</sup> versus PET<sup>−</sup> tumors (FDR <i>q</i> value = 0.006). The top portion of the figure plots the enrichment scores (ES) for each gene, whereas the bottom portion of the plot shows the value of the ranking metric moving down the list of ranked genes. The table enumerates the genes in the <span class="highlight" style="background-color:">pathway</span> for which a majority of probe sets were significantly enriched and upregulated in PET-positive versus PET-negative tumors. FDR <i>q</i> value (FDR, false discovery rate); FWER <i>p</i> value (FWER, family wise-error rate).</div></p></div><div class="aux"><div class="resc">Julie K. Schwarz, et al. Clin Cancer Res. 2012 March 1;18(5):1464-1471.</div><p class="links"><a class="dblinks" href="/pubmed/22235101" ref="ordinalpos=2407&amp;ncbi_uid=4810357&amp;link_uid=22235101">Citation</a><a class="dblinks" href="/pmc/articles/PMC3627483/" ref="ordinalpos=2407&amp;ncbi_uid=4810357&amp;link_uid=PMC3627483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2408.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3648501/figure/F1/" ref="ordinalpos=2408&amp;ncbi_uid=4888502&amp;link_uid=PMC3648501" href="/pmc/articles/PMC3648501/"><img src="/pmc/articles/instance/3648501/bin/1742-4682-10-27-1.gif" src-large="/pmc/articles/instance/3648501/bin/1742-4682-10-27-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3648501/" ref="ordinalpos=2408&amp;ncbi_uid=4888502&amp;link_uid=PMC3648501" image-link="/pmc/articles/PMC3648501/figure/F1/" class="imagepopup">Figure 1.  From: Mathematical models for the Notch and Wnt <span class="highlight" style="background-color:">signaling</span> pathways and the crosstalk between them during somitogenesis. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The diagram was created using CellDesigner. The light green rectangle represents protein; the bottle green parallelogram represents mRNA; the white rectangle that contains a light green rectangle represents a complex; ∅ represents the resultant of a degradation reaction or reactant of a combination reaction. The arrow represents the reaction.</div></p></div><div class="aux"><div class="resc">Hong-yan Wang, et al. Theor Biol Med Model. 2013;10:27-27.</div><p class="links"><a class="dblinks" href="/pubmed/23602012" ref="ordinalpos=2408&amp;ncbi_uid=4888502&amp;link_uid=23602012">Citation</a><a class="dblinks" href="/pmc/articles/PMC3648501/" ref="ordinalpos=2408&amp;ncbi_uid=4888502&amp;link_uid=PMC3648501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2409.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3062567/figure/pone-0018059-g008/" ref="ordinalpos=2409&amp;ncbi_uid=2848141&amp;link_uid=PMC3062567" href="/pmc/articles/PMC3062567/"><img src="/pmc/articles/instance/3062567/bin/pone.0018059.g008.gif" src-large="/pmc/articles/instance/3062567/bin/pone.0018059.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3062567/" ref="ordinalpos=2409&amp;ncbi_uid=2848141&amp;link_uid=PMC3062567" image-link="/pmc/articles/PMC3062567/figure/pone-0018059-g008/" class="imagepopup">Figure 8. Schematic model for the regulation of GPCR-mediated <span class="highlight" style="background-color:">signaling</span> by SLC39A14..  From: The Zinc Transporter SLC39A14/ZIP14 Controls G-Protein Coupled Receptor-Mediated <span class="highlight" style="background-color:">Signaling</span> Required for Systemic Growth. </a></p><div class="supp"><p class="details"><div>SLC39A14 regulates the basal cAMP level by suppressing PDE activity, through either the direct provision of Zn to PDE or the indirect provision via unidentified molecular chaperone(s) (Protein X). This system may facilitate the GPCR-cAMP-CREB <span class="highlight" style="background-color:">pathway</span> in endocrine-system reactions.</div></p></div><div class="aux"><div class="resc">Shintaro Hojyo, et al. PLoS One. 2011;6(3):e18059.</div><p class="links"><a class="dblinks" href="/pubmed/21445361" ref="ordinalpos=2409&amp;ncbi_uid=2848141&amp;link_uid=21445361">Citation</a><a class="dblinks" href="/pmc/articles/PMC3062567/" ref="ordinalpos=2409&amp;ncbi_uid=2848141&amp;link_uid=PMC3062567">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2410.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3434772/figure/F5/" ref="ordinalpos=2410&amp;ncbi_uid=5248081&amp;link_uid=PMC3434772" href="/pmc/articles/PMC3434772/"><img src="/pmc/articles/instance/3434772/bin/zjw0081242390005.gif" src-large="/pmc/articles/instance/3434772/bin/zjw0081242390005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3434772/" ref="ordinalpos=2410&amp;ncbi_uid=5248081&amp;link_uid=PMC3434772" image-link="/pmc/articles/PMC3434772/figure/F5/" class="imagepopup">Fig. 5. From: Rapid Phosphoproteomic and Transcriptomic Changes in the Rhizobia-legume Symbiosis. </a></p><div class="supp"><p class="details"><div><b>Proposed model depicting putative LCO-receptor kinases on the plasma membrane and the cross-talks between different symbiotic <span class="highlight" style="background-color:">signaling</span> pathways in Medicago.</b> NF are perceived with high stringency by NF receptor (NFP) and its putative heteromultimeric partner, an NFR1 homolog. This accounts for majority of signal transduction events. However, NF-induced, but NFP-independent <span class="highlight" style="background-color:">signaling</span> responses suggest the existence of other receptor complexes. Differential transcriptional regulation of these candidate genes by both NF and GSE in NFP-independent manner suggests the existence of Myc factor receptor, which is capable of perceiving NF. Interestingly, NFP-independent, but NF-specific transcriptional responses suggest the existence of NF-specific “cryptic” <span class="highlight" style="background-color:">signaling</span> receptor in Medicago. Although a majority of these transcriptional responses are dependent on DMI3, GSE-induced <i>DMI3</i>-independent transcriptional regulation of candidate genes, suggest the existence of parallel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is independent of well established <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> comprising <i>DMI</i> genes.</div></p></div><div class="aux"><div class="resc">Christopher M. Rose, et al. Mol Cell Proteomics. 2012 September;11(9):724-744.</div><p class="links"><a class="dblinks" href="/pubmed/22683509" ref="ordinalpos=2410&amp;ncbi_uid=5248081&amp;link_uid=22683509">Citation</a><a class="dblinks" href="/pmc/articles/PMC3434772/" ref="ordinalpos=2410&amp;ncbi_uid=5248081&amp;link_uid=PMC3434772">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2411.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136937/figure/pone-0021977-g004/" ref="ordinalpos=2411&amp;ncbi_uid=3090718&amp;link_uid=PMC3136937" href="/pmc/articles/PMC3136937/"><img src="/pmc/articles/instance/3136937/bin/pone.0021977.g004.gif" src-large="/pmc/articles/instance/3136937/bin/pone.0021977.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136937/" ref="ordinalpos=2411&amp;ncbi_uid=3090718&amp;link_uid=PMC3136937" image-link="/pmc/articles/PMC3136937/figure/pone-0021977-g004/" class="imagepopup">Figure 4. Major <span class="highlight" style="background-color:">signaling</span> network 1 associated with the proteins differentially expressed in MB-231-Br compared with MB-231-Pa cells identified by the Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis..  From: Identification and Analysis of <span class="highlight" style="background-color:">Signaling</span> Networks Potentially Involved in Breast Carcinoma Metastasis to the Brain. </a></p><div class="supp"><p class="details"><div>The network consists of 35 nodes and includes 11 out of 12 differentially expressed proteins (shaded green). Note the interactions of NASP, RPS10, HSPA8, and HSPA1B with TRAF6 (TNF receptor-associated factor 6), a key factor acting upon the TNFα/TGFβ <span class="highlight" style="background-color:">signaling</span> axis, as well as the involvement of NF<span style="font-variant: small-caps" class="small-caps">κ</span>B-, HSP-70-, TP53-, and IFNγ-associated pathways.</div></p></div><div class="aux"><div class="resc">Feng Li, et al. PLoS One. 2011;6(7):e21977.</div><p class="links"><a class="dblinks" href="/pubmed/21779361" ref="ordinalpos=2411&amp;ncbi_uid=3090718&amp;link_uid=21779361">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136937/" ref="ordinalpos=2411&amp;ncbi_uid=3090718&amp;link_uid=PMC3136937">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2412.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4349055/figure/F2/" ref="ordinalpos=2412&amp;ncbi_uid=7297633&amp;link_uid=PMC4349055" href="/pmc/articles/PMC4349055/"><img src="/pmc/articles/instance/4349055/bin/fphys-06-00066-g0002.gif" src-large="/pmc/articles/instance/4349055/bin/fphys-06-00066-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4349055/" ref="ordinalpos=2412&amp;ncbi_uid=7297633&amp;link_uid=PMC4349055" image-link="/pmc/articles/PMC4349055/figure/F2/" class="imagepopup">Figure 2.  From: Good and bad sides of TGFβ-<span class="highlight" style="background-color:">signaling</span> in myocardial infarction. </a></p><div class="supp"><p class="details"><div><b>MicroRNA <span class="highlight" style="background-color:">signaling</span> in TGFβ-induced apoptosis and fibrosis</b>. Apoptosis induction after myocardial infarction has been shown to be mediated via TGFβ —microRNA—SMAD <span class="highlight" style="background-color:">signaling</span>. Influence of microRNAs on fibrosis has been shown via enhancement of collagen synthesis and EndMT. Red arrows indicate enhancement or reduction of the respective <span class="highlight" style="background-color:">signaling</span> molecule. <img src="/pmc/articles/instance/4349055/bin/fphys-06-00066-i0001.jpg" class="inline-graphic" style="border-style: none;" /> Indicates negative/reducing influence of this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Gerhild Euler. Front Physiol. 2015;6:66.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2412&amp;ncbi_uid=7297633&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4349055/" ref="ordinalpos=2412&amp;ncbi_uid=7297633&amp;link_uid=PMC4349055">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2413.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4009148/figure/fig3/" ref="ordinalpos=2413&amp;ncbi_uid=6130782&amp;link_uid=PMC4009148" href="/pmc/articles/PMC4009148/"><img src="/pmc/articles/instance/4009148/bin/JIR2014-602846.003.gif" src-large="/pmc/articles/instance/4009148/bin/JIR2014-602846.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4009148/" ref="ordinalpos=2413&amp;ncbi_uid=6130782&amp;link_uid=PMC4009148" image-link="/pmc/articles/PMC4009148/figure/fig3/" class="imagepopup">Figure 3.  From: Potential Involvement of IL-17F in Asthma. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> mechanism of IL-17F expression in bronchial epithelial cells. Bronchial epithelial cells are a novel cell source of IL-17F. The expression of IL-17F is induced by IL-33. IL-33 binds to its receptor, ST2, and then the ERK1/2-MSK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated. However, other stimuli inducing IL-17F and their <span class="highlight" style="background-color:">signaling</span> pathways are largely unknown.</div></p></div><div class="aux"><div class="resc">Kyoko Ota, et al. J Immunol Res. 2014;2014:602846.</div><p class="links"><a class="dblinks" href="/pubmed/24829928" ref="ordinalpos=2413&amp;ncbi_uid=6130782&amp;link_uid=24829928">Citation</a><a class="dblinks" href="/pmc/articles/PMC4009148/" ref="ordinalpos=2413&amp;ncbi_uid=6130782&amp;link_uid=PMC4009148">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2414.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4353798/figure/f6-etm-09-04-1265/" ref="ordinalpos=2414&amp;ncbi_uid=7283570&amp;link_uid=PMC4353798" href="/pmc/articles/PMC4353798/"><img src="/pmc/articles/instance/4353798/bin/ETM-09-04-1265-g05.gif" src-large="/pmc/articles/instance/4353798/bin/ETM-09-04-1265-g05.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4353798/" ref="ordinalpos=2414&amp;ncbi_uid=7283570&amp;link_uid=PMC4353798" image-link="/pmc/articles/PMC4353798/figure/f6-etm-09-04-1265/" class="imagepopup">Figure 6.  From: Cbl-b-regulated extracellular signal-regulated kinase <span class="highlight" style="background-color:">signaling</span> is involved in the shikonin-induced apoptosis of lung cancer cells in vitro. </a></p><div class="supp"><p class="details"><div>Effects of shikonin (SK) on the phosphoinositide 3-kinase (PI3K)/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in NCI-H460 cells. NCI-H460 cells were treated with 2.6 μM SK for the indicated times. The expression levels of phospho (p)-Akt and Akt proteins were then analyzed by western blotting.</div></p></div><div class="aux"><div class="resc">DAN QU, et al. Exp Ther Med. 2015 April;9(4):1265-1270.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2414&amp;ncbi_uid=7283570&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4353798/" ref="ordinalpos=2414&amp;ncbi_uid=7283570&amp;link_uid=PMC4353798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2415.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2715766/figure/f9/" ref="ordinalpos=2415&amp;ncbi_uid=1971747&amp;link_uid=PMC2715766" href="/pmc/articles/PMC2715766/"><img src="/pmc/articles/instance/2715766/bin/zjv0160922090009.gif" src-large="/pmc/articles/instance/2715766/bin/zjv0160922090009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2715766/" ref="ordinalpos=2415&amp;ncbi_uid=1971747&amp;link_uid=PMC2715766" image-link="/pmc/articles/PMC2715766/figure/f9/" class="imagepopup">FIG. 9. From: Activation of Intracellular <span class="highlight" style="background-color:">Signaling</span> Pathways by the Murine Cytomegalovirus G Protein-Coupled Receptor M33 Occurs via PLC-?/PKC-Dependent and -Independent Mechanisms  . </a></p><div class="supp"><p class="details"><div>G protein-dependent and G protein-independent <span class="highlight" style="background-color:">signaling</span> activities of M33. Use of pharmacological and genetic inhibitors at multiple steps along the G<sub>q/11</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reveals that CREB activation by the MCMV GPCR M33 is mediated by coupling to heterotrimeric G<sub>q/11</sub> proteins and PLC-β activity. Moreover, while not affected by IP<sub>3</sub> and Ca<sup>2+</sup> levels, CREB activity is dependent on PKC activity and is augmented by increased levels of DAG. Conversely, M33 stimulation of p38 (which may partially contribute to M33-induced CREB activity) and NF-κB occurs in the absence of G<sub>q/11</sub> <span class="highlight" style="background-color:">signaling</span> through an as-yet-defined mechanism(s).</div></p></div><div class="aux"><div class="resc">Joseph D. Sherrill, et al. J Virol. 2009 August;83(16):8141-8152.</div><p class="links"><a class="dblinks" href="/pubmed/19494016" ref="ordinalpos=2415&amp;ncbi_uid=1971747&amp;link_uid=19494016">Citation</a><a class="dblinks" href="/pmc/articles/PMC2715766/" ref="ordinalpos=2415&amp;ncbi_uid=1971747&amp;link_uid=PMC2715766">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2416.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3174480/figure/F3/" ref="ordinalpos=2416&amp;ncbi_uid=3234430&amp;link_uid=PMC3174480" href="/pmc/articles/PMC3174480/"><img src="/pmc/articles/instance/3174480/bin/fly0404_0288_fig003.gif" src-large="/pmc/articles/instance/3174480/bin/fly0404_0288_fig003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3174480/" ref="ordinalpos=2416&amp;ncbi_uid=3234430&amp;link_uid=PMC3174480" image-link="/pmc/articles/PMC3174480/figure/F3/" class="imagepopup">Figure 3.  From: Lgl/aPKC and Crb regulate the Salvador/Warts/Hippo                     <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) Lgl/aPKC regulates Yki activity by promoting the formation of Hpo/RASSF                         complexes. Lgl/aPKC regulate the SWH <span class="highlight" style="background-color:">pathway</span> by favoring the formation of                         inactive Hpo/Rassf over active Hpo/Sav protein complexes. Hpo/Rassf are                         unable to phophorylate Yki. Consequently, pYki levels are reduced and                         dephosphorylated Yki enters the nucleus to form a complex with Scalloped                         (Sd) and activate SWH <span class="highlight" style="background-color:">pathway</span> targets. Perhaps the mislocalization and                         formation of Hpo/Rassf complexes is due to disruptions in endocytosis. (B).                         Crb regulates Yki activity by reducing the levels of Ex/Yki complexes. Crb                         regulates the SWH <span class="highlight" style="background-color:">pathway</span> through direct interactions with Ex. Crb/Ex/Yki                         form a tripartite complex and Yki is sequestered in the cytoplasm and                         prevented from entering the nucleus by binding to Expanded (Ex). The                         overexpression of Crb reduces Ex/Yki complexes, freeing Yki to enter the                         nucleus, interact with Scalloped (Sd) and activate SWH target genes. It is                         possible that Ex/Yki complexes are regulated by ubiquitination and/or                         proteolysis. <i>crb</i> mutant tissue also shows overgrowth due to                         deregulation of Notch endocytosis but the influence of Notch <span class="highlight" style="background-color:">signaling</span> on                         SWH <span class="highlight" style="background-color:">pathway</span> targets remains to be determined.</div></p></div><div class="aux"><div class="resc">Linda M Parsons, et al. Fly (Austin). 2010 Oct-Dec;4(4):288-293.</div><p class="links"><a class="dblinks" href="/pubmed/20798605" ref="ordinalpos=2416&amp;ncbi_uid=3234430&amp;link_uid=20798605">Citation</a><a class="dblinks" href="/pmc/articles/PMC3174480/" ref="ordinalpos=2416&amp;ncbi_uid=3234430&amp;link_uid=PMC3174480">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2417.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2262896/figure/F6/" ref="ordinalpos=2417&amp;ncbi_uid=1260473&amp;link_uid=PMC2262896" href="/pmc/articles/PMC2262896/"><img src="/pmc/articles/instance/2262896/bin/1471-2164-9-71-6.gif" src-large="/pmc/articles/instance/2262896/bin/1471-2164-9-71-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2262896/" ref="ordinalpos=2417&amp;ncbi_uid=1260473&amp;link_uid=PMC2262896" image-link="/pmc/articles/PMC2262896/figure/F6/" class="imagepopup">Figure 6.  From: Activation of counter-regulatory mechanisms in a rat renal acute rejection model. </a></p><div class="supp"><p class="details"><div><b>Interferon <span class="highlight" style="background-color:">signaling</span></b>. Significant up-regulation of genes was also observed in the canonical interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Up-regulated genes are highlighted in red.</div></p></div><div class="aux"><div class="resc">Bayram Edemir, et al. BMC Genomics. 2008;9:71-71.</div><p class="links"><a class="dblinks" href="/pubmed/18261221" ref="ordinalpos=2417&amp;ncbi_uid=1260473&amp;link_uid=18261221">Citation</a><a class="dblinks" href="/pmc/articles/PMC2262896/" ref="ordinalpos=2417&amp;ncbi_uid=1260473&amp;link_uid=PMC2262896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2418.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3316753/figure/Fig4/" ref="ordinalpos=2418&amp;ncbi_uid=3886669&amp;link_uid=PMC3316753" href="/pmc/articles/PMC3316753/"><img src="/pmc/articles/instance/3316753/bin/12263_2012_281_Fig4_HTML.gif" src-large="/pmc/articles/instance/3316753/bin/12263_2012_281_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3316753/" ref="ordinalpos=2418&amp;ncbi_uid=3886669&amp;link_uid=PMC3316753" image-link="/pmc/articles/PMC3316753/figure/Fig4/" class="imagepopup">Fig. 4.  From: Dietary compounds as potent inhibitors of the signal transducers and activators of transcription (STAT) 3 regulatory network. </a></p><div class="supp"><p class="details"><div>JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> targeted by different inhibitors. These inhibitors target the following points in the JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: constitutive activation of JAK2, STAT3 phosphorylation, formation of the STAT dimer and binding activity of STAT3 to DNA. In contrast, the phosphatase SHP-1 can be activated</div></p></div><div class="aux"><div class="resc">Anne Trécul, et al. Genes Nutr. 2012 April;7(2):111-125.</div><p class="links"><a class="dblinks" href="/pubmed/22274779" ref="ordinalpos=2418&amp;ncbi_uid=3886669&amp;link_uid=22274779">Citation</a><a class="dblinks" href="/pmc/articles/PMC3316753/" ref="ordinalpos=2418&amp;ncbi_uid=3886669&amp;link_uid=PMC3316753">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2419.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2698925/figure/F4/" ref="ordinalpos=2419&amp;ncbi_uid=1856745&amp;link_uid=PMC2698925" href="/pmc/articles/PMC2698925/"><img src="/pmc/articles/instance/2698925/bin/1471-2407-9-166-4.gif" src-large="/pmc/articles/instance/2698925/bin/1471-2407-9-166-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2698925/" ref="ordinalpos=2419&amp;ncbi_uid=1856745&amp;link_uid=PMC2698925" image-link="/pmc/articles/PMC2698925/figure/F4/" class="imagepopup">Figure 4.  From: Chondromodulin-1 directly suppresses growth of human cancer cells. </a></p><div class="supp"><p class="details"><div><b>The effect of ChM1 on the downstream <span class="highlight" style="background-color:">pathway</span> of extracellular matrix-integrin <span class="highlight" style="background-color:">signaling</span></b>. Western blots showing phosphorylation levels of Erk, Akt, and GSK3β, the downstream molecules of the extracellular matrix-integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ChM1 had no effect on phosphorylation levels of these proteins at 8 and 24 hours after adenovirus infection.</div></p></div><div class="aux"><div class="resc">Hisashi Mera, et al. BMC Cancer. 2009;9:166-166.</div><p class="links"><a class="dblinks" href="/pubmed/19480713" ref="ordinalpos=2419&amp;ncbi_uid=1856745&amp;link_uid=19480713">Citation</a><a class="dblinks" href="/pmc/articles/PMC2698925/" ref="ordinalpos=2419&amp;ncbi_uid=1856745&amp;link_uid=PMC2698925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2420.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1838573/figure/F1/" ref="ordinalpos=2420&amp;ncbi_uid=1025757&amp;link_uid=PMC1838573" href="/pmc/articles/PMC1838573/"><img src="/pmc/articles/instance/1838573/bin/nihms-12454-0001.gif" src-large="/pmc/articles/instance/1838573/bin/nihms-12454-0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1838573/" ref="ordinalpos=2420&amp;ncbi_uid=1025757&amp;link_uid=PMC1838573" image-link="/pmc/articles/PMC1838573/figure/F1/" class="imagepopup">Figure 1.  From: Blockade of Apoptosis as a Rational Therapeutic Strategy for the Treatment of Sepsis. </a></p><div class="supp"><p class="details"><div>Apoptotic <span class="highlight" style="background-color:">signaling</span> pathways as seen through death receptor ligation of TNFR or Fas (extrinsic <span class="highlight" style="background-color:">signaling</span>, type I cells) or through activation of the mitochondrial <span class="highlight" style="background-color:">pathway</span> through Bcl-2 family members (intrinsic <span class="highlight" style="background-color:">signaling</span>, type II cells).</div></p></div><div class="aux"><div class="resc">Alfred Ayala, et al. Novartis Found Symp. ;280:37-164.</div><p class="links"><a class="dblinks" href="/pubmed/17380787" ref="ordinalpos=2420&amp;ncbi_uid=1025757&amp;link_uid=17380787">Citation</a><a class="dblinks" href="/pmc/articles/PMC1838573/" ref="ordinalpos=2420&amp;ncbi_uid=1025757&amp;link_uid=PMC1838573">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2421.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3112285/figure/F2/" ref="ordinalpos=2421&amp;ncbi_uid=3374971&amp;link_uid=PMC3112285" href="/pmc/articles/PMC3112285/"><img src="/pmc/articles/instance/3112285/bin/nihms293426f2.gif" src-large="/pmc/articles/instance/3112285/bin/nihms293426f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3112285/" ref="ordinalpos=2421&amp;ncbi_uid=3374971&amp;link_uid=PMC3112285" image-link="/pmc/articles/PMC3112285/figure/F2/" class="imagepopup">Figure 2.  From: The Ras-ERK and PI3K-mTOR Pathways: Cross-talk and Compensation. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> Crosstalk. The Ras-MAPK and PI3K-mTORC1 pathways regulate each other via cross-inhibition (red) and cross-activation (green). Each <span class="highlight" style="background-color:">pathway</span> has a mechanism to negatively feed onto the other: ERK phosphorylation of GAB and AKT phosphorylation of Raf. Components of the Ras-ERK <span class="highlight" style="background-color:">pathway</span> (Ras, Raf, ERK, and RSK) also positively regulate the PI3K-mTORC1 <span class="highlight" style="background-color:">pathway</span>. TSC2 and mTORC1 are key integration points that receive many inputs from both the Ras-ERK and PI3K <span class="highlight" style="background-color:">signaling</span>. Positive regulation of the substrate protein is shown as an arrow. Negative regulation of the substrate protein is depicted as a blunt-ended line.</div></p></div><div class="aux"><div class="resc">Michelle C. Mendoza, et al. Trends Biochem Sci. ;36(6):320-328.</div><p class="links"><a class="dblinks" href="/pubmed/21531565" ref="ordinalpos=2421&amp;ncbi_uid=3374971&amp;link_uid=21531565">Citation</a><a class="dblinks" href="/pmc/articles/PMC3112285/" ref="ordinalpos=2421&amp;ncbi_uid=3374971&amp;link_uid=PMC3112285">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2422.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2846157/figure/DDQ014F4/" ref="ordinalpos=2422&amp;ncbi_uid=2891672&amp;link_uid=PMC2846157" href="/pmc/articles/PMC2846157/"><img src="/pmc/articles/instance/2846157/bin/ddq01404.gif" src-large="/pmc/articles/instance/2846157/bin/ddq01404.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2846157/" ref="ordinalpos=2422&amp;ncbi_uid=2891672&amp;link_uid=PMC2846157" image-link="/pmc/articles/PMC2846157/figure/DDQ014F4/" class="imagepopup">Figure 4. From: Mouse Prkar1a haploinsufficiency leads to an increase in tumors in the Trp53+/− or Rb1+/− backgrounds and chemically induced skin papillomas by dysregulation of the cell cycle and Wnt <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><i>Wnt</i> <span class="highlight" style="background-color:">pathway</span> expression in sarcomas, papillomas, pituitary and thyroid tumors. (<b>A</b>) <i>Wnt</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes were over-expressed in <i>Prkar1a</i><sup>+/−</sup> sarcomas, pituitary and thyroid tumors, papillomas from all three experiments, as demonstrated by a qRT–PCR array study that included 84 genes from this <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) Strong staining for Ctnnb1 in <i>Prkar1a</i><sup><i>+/−</i></sup> papillomas, <i>Prkar1a</i><sup><i>+/−</i></sup> <i>Trp53</i><sup><i>+/−</i></sup> sarcomas and <i>Prkar1a</i><sup>+/−</sup> <i>Rb1</i><sup>+/−</sup> pituitary tumors. (<b>C</b>) Wnt3 immunoreactivity was higher in <i>Prkar1a</i><sup><i>+/−</i></sup> papillomas than in WT papillomas. (<b>D</b>) Lrp5 protein was over-expressed in <i>Prkar1a</i><sup><i>+/−</i></sup> papillomas and double heterozygous tumors. *<i>P</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Madson Q. Almeida, et al. Hum Mol Genet. 2010 April 15;19(8):1387-1398.</div><p class="links"><a class="dblinks" href="/pubmed/20080939" ref="ordinalpos=2422&amp;ncbi_uid=2891672&amp;link_uid=20080939">Citation</a><a class="dblinks" href="/pmc/articles/PMC2846157/" ref="ordinalpos=2422&amp;ncbi_uid=2891672&amp;link_uid=PMC2846157">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2423.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4295872/figure/pone.0115352.g008/" ref="ordinalpos=2423&amp;ncbi_uid=7082186&amp;link_uid=PMC4295872" href="/pmc/articles/PMC4295872/"><img src="/pmc/articles/instance/4295872/bin/pone.0115352.g008.gif" src-large="/pmc/articles/instance/4295872/bin/pone.0115352.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4295872/" ref="ordinalpos=2423&amp;ncbi_uid=7082186&amp;link_uid=PMC4295872" image-link="/pmc/articles/PMC4295872/figure/pone.0115352.g008/" class="imagepopup">Figure 8. Mechanistic model of PSMC3IP and EPSTI1 as putative apoptotic factors..  From: Breast Cancer Genes PSMC3IP and EPSTI1 Play a Role in Apoptosis Regulation. </a></p><div class="supp"><p class="details"><div>The extrinsic apoptosis <span class="highlight" style="background-color:">pathway</span> is initiated by the ligand binding to the death receptor, forming the DISC complex and leading to the activation of the caspase <span class="highlight" style="background-color:">signaling</span> cascade. The overexpression of EGFR and HER2 or activation of components of their downstream <span class="highlight" style="background-color:">signaling</span> pathways (i.e. PI3K <span class="highlight" style="background-color:">pathway</span>) induce an anti-apoptotic <span class="highlight" style="background-color:">signaling</span> through PI3K-Akt <span class="highlight" style="background-color:">pathway</span> in endocrine resistant breast cancer cells. Based on our findings, we suggest that PSMC3IP and EPSTI1 may regulate the apoptotic <span class="highlight" style="background-color:">pathway</span> via the physical interaction mainly with the apoptosis initiator CASP8, but also with AKT1 and BCAR3. Pro-apoptotic proteins are displayed in red and anti-apoptotic in green. Candidate proteins are displayed in purple.</div></p></div><div class="aux"><div class="resc">Eva Capdevila-Busquets, et al. PLoS One. 2015;10(1):e0115352.</div><p class="links"><a class="dblinks" href="/pubmed/25590583" ref="ordinalpos=2423&amp;ncbi_uid=7082186&amp;link_uid=25590583">Citation</a><a class="dblinks" href="/pmc/articles/PMC4295872/" ref="ordinalpos=2423&amp;ncbi_uid=7082186&amp;link_uid=PMC4295872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2424.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3000836/figure/F1/" ref="ordinalpos=2424&amp;ncbi_uid=2628763&amp;link_uid=PMC3000836" href="/pmc/articles/PMC3000836/"><img src="/pmc/articles/instance/3000836/bin/1745-7580-6-10-1.gif" src-large="/pmc/articles/instance/3000836/bin/1745-7580-6-10-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3000836/" ref="ordinalpos=2424&amp;ncbi_uid=2628763&amp;link_uid=PMC3000836" image-link="/pmc/articles/PMC3000836/figure/F1/" class="imagepopup">Figure 1.  From: DC-ATLAS: a systems biology resource to dissect receptor specific signal transduction in dendritic cells. </a></p><div class="supp"><p class="details"><div><b>Comparison of the DC-ATLAS Toll Like Receptor (TLR) 3 <span class="highlight" style="background-color:">pathway</span> with other <span class="highlight" style="background-color:">pathway</span> databases</b>. (A) Representation of the KEGG Toll-like receptor (TLR) <span class="highlight" style="background-color:">pathway</span>. The TLR3 signal is highlighted in red. (B) Representation of the KEGG TLR3 <span class="highlight" style="background-color:">pathway</span>, displaying only the reactions proven in human DCs. The curation led to the validation of about 50% of the genes previously belonging to public TLR3 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Duccio Cavalieri, et al. Immunome Res. 2010;6:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/21092113" ref="ordinalpos=2424&amp;ncbi_uid=2628763&amp;link_uid=21092113">Citation</a><a class="dblinks" href="/pmc/articles/PMC3000836/" ref="ordinalpos=2424&amp;ncbi_uid=2628763&amp;link_uid=PMC3000836">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2425.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2891025/figure/F6/" ref="ordinalpos=2425&amp;ncbi_uid=2947697&amp;link_uid=PMC2891025" href="/pmc/articles/PMC2891025/"><img src="/pmc/articles/instance/2891025/bin/nihms-198850-f0006.gif" src-large="/pmc/articles/instance/2891025/bin/nihms-198850-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2891025/" ref="ordinalpos=2425&amp;ncbi_uid=2947697&amp;link_uid=PMC2891025" image-link="/pmc/articles/PMC2891025/figure/F6/" class="imagepopup">Figure 6. <span class="highlight" style="background-color:">Signaling</span> networks of Gαq-regulated transcripts.  From: Deep mRNA Sequencing for In Vivo Functional Analysis of Cardiac Transcriptional Regulators: Application to G?q. </a></p><div class="supp"><p class="details"><div>Ingenuity Pathways Analysis software (http://www.ingenuity.com) was used to depict potential <span class="highlight" style="background-color:">signaling</span> pathways between (<b>A</b>) high-abundance and (<b>B</b>) low-abundance, Gαq-regulated gene products. Lines with arrowheads, molecule acts on a target; lines without arrowheads, binding only. Solid lines, direct interaction; dotted lines, indirect interaction. Blue background, high-abundance genes; green background, low-abundance genes; white background, member of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> but not regulated in Gαq hearts.</div></p></div><div class="aux"><div class="resc">SJ Matkovich, et al. Circ Res. ;106(9):1459-1467.</div><p class="links"><a class="dblinks" href="/pubmed/20360248" ref="ordinalpos=2425&amp;ncbi_uid=2947697&amp;link_uid=20360248">Citation</a><a class="dblinks" href="/pmc/articles/PMC2891025/" ref="ordinalpos=2425&amp;ncbi_uid=2947697&amp;link_uid=PMC2891025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2426.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1783693/figure/F3/" ref="ordinalpos=2426&amp;ncbi_uid=877543&amp;link_uid=PMC1783693" href="/pmc/articles/PMC1783693/"><img src="/pmc/articles/instance/1783693/bin/RevDiabeticStud-01-165-003.gif" src-large="/pmc/articles/instance/1783693/bin/RevDiabeticStud-01-165-003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1783693/" ref="ordinalpos=2426&amp;ncbi_uid=877543&amp;link_uid=PMC1783693" image-link="/pmc/articles/PMC1783693/figure/F3/" class="imagepopup">Figure 3. The effects of the JNK <span class="highlight" style="background-color:">pathway</span> on insulin resistance and glucose tolerance.  From: Involvement of Oxidative Stress and the JNK <span class="highlight" style="background-color:">Pathway</span> in Glucose Toxicity. </a></p><div class="supp"><p class="details"><div>The JNK <span class="highlight" style="background-color:">pathway</span> in the liver is activated by factors such as oxidative stress, free fatty acids (FFAs), and TNF-α. The levels of these factors are elevated under obese diabetic conditions. Serine phosphorylation of insulin receptor substrate-1 (IRS-1) inhibits insulin-stimulated tyrosine phosphorylation of IRS-1 and leads to an increase in insulin resistance. Expression of wild type JNK in the liver of normal mice decreased insulin sensitivity. Suppression of the JNK <span class="highlight" style="background-color:">pathway</span> by the treatment with Ad-DN-JNK enhances insulin <span class="highlight" style="background-color:">signaling</span> which leads to a decrease in gluconeogenesis and amelioration of glucose tolerance.</div></p></div><div class="aux"><div class="resc">Hideaki Kaneto, et al. Rev Diabet Stud. 2004 Winter;1(4):165-174.</div><p class="links"><a class="dblinks" href="/pubmed/17491701" ref="ordinalpos=2426&amp;ncbi_uid=877543&amp;link_uid=17491701">Citation</a><a class="dblinks" href="/pmc/articles/PMC1783693/" ref="ordinalpos=2426&amp;ncbi_uid=877543&amp;link_uid=PMC1783693">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2427.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2638772/figure/DDN339F9/" ref="ordinalpos=2427&amp;ncbi_uid=1762650&amp;link_uid=PMC2638772" href="/pmc/articles/PMC2638772/"><img src="/pmc/articles/instance/2638772/bin/ddn33909.gif" src-large="/pmc/articles/instance/2638772/bin/ddn33909.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2638772/" ref="ordinalpos=2427&amp;ncbi_uid=1762650&amp;link_uid=PMC2638772" image-link="/pmc/articles/PMC2638772/figure/DDN339F9/" class="imagepopup">Figure 9. From: FGFR3 promotes synchondrosis closure and fusion of ossification centers through the MAPK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Model whereby increased Fgfr3 <span class="highlight" style="background-color:">signaling</span> in chondrocytes causes premature synchondrosis closure and unification of ossification centers. Increased Fgfr3 <span class="highlight" style="background-color:">signaling</span> induces premature exit from the cell cycle and accelerates the transition into hypertrophic chondrocytes. Increased Fgfr3 <span class="highlight" style="background-color:">signaling</span> also causes upregulation of Vegf in chondrocytes, promoting vascular invasion. Increased secretion of Bmp ligands and decreased secretion of Bmp antagonists result in enhanced osteoblast differentiation of osteoprogenitors in the periosteum, promoting bone formation and fusion of ossification centers. The upregulation of Vegf and Bmps and downregulation of Bmp antagonists are at least partially mediated by the MAPK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Takehiko Matsushita, et al. Hum Mol Genet. 2009 January 15;18(2):227-240.</div><p class="links"><a class="dblinks" href="/pubmed/18923003" ref="ordinalpos=2427&amp;ncbi_uid=1762650&amp;link_uid=18923003">Citation</a><a class="dblinks" href="/pmc/articles/PMC2638772/" ref="ordinalpos=2427&amp;ncbi_uid=1762650&amp;link_uid=PMC2638772">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2428.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3923659/figure/pcbi-1003470-g003/" ref="ordinalpos=2428&amp;ncbi_uid=5826774&amp;link_uid=PMC3923659" href="/pmc/articles/PMC3923659/"><img src="/pmc/articles/instance/3923659/bin/pcbi.1003470.g003.gif" src-large="/pmc/articles/instance/3923659/bin/pcbi.1003470.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3923659/" ref="ordinalpos=2428&amp;ncbi_uid=5826774&amp;link_uid=PMC3923659" image-link="/pmc/articles/PMC3923659/figure/pcbi-1003470-g003/" class="imagepopup">Figure 3. Structures of protein-protein complexes mapped to the IL-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: The Structural <span class="highlight" style="background-color:">Pathway</span> of Interleukin 1 (IL-1) Initiated <span class="highlight" style="background-color:">Signaling</span> Reveals Mechanisms of Oncogenic Mutations and SNPs in Inflammation and Cancer. </a></p><div class="supp"><p class="details"><div><b>A.</b> Overall distribution of experimental and predicted complexes on the <span class="highlight" style="background-color:">pathway</span>. Dark blue interactions represent the experimentally determined complex structures in the PDB, red interactions represent the predicted complexes with predicted binding energies lower than −10 energy units and yellow interactions represent the interactions for which neither experimental nor computational data exists. <b>B.</b> Predicted structures (Template PDB code+Target PDB codes+Energy value): IL1α-IL1R1 (1itbAB 2l5x 4depB −71.92); IL1α-IL1RAP (1itbAB 2l5x 4depC −49.25); IL1R1-MYD88 (1gylAB IL1R1 (model) 2z5v −23.8); IL1R1-TOLLIP (1oh0AB IL1R1 (model) 1wgl −18.85); IL1RAP-MYD88 (1p65AB IL1RAP (model) 2z5v −31.72); MYD88-TOLLIP (1yrlAC 3mopA 1wgl −11.04); MYD88-TRAF6 (1vjlAB 2js7 1lb6 −37.01); TRAF6-IRF7 (1g8tAB 2o61 3hct −25.83). The blue color represents the proteins that precede its partners in the information flow.</div></p></div><div class="aux"><div class="resc">Saliha Ece Acuner Ozbabacan, et al. PLoS Comput Biol. 2014 February;10(2):e1003470.</div><p class="links"><a class="dblinks" href="/pubmed/24550720" ref="ordinalpos=2428&amp;ncbi_uid=5826774&amp;link_uid=24550720">Citation</a><a class="dblinks" href="/pmc/articles/PMC3923659/" ref="ordinalpos=2428&amp;ncbi_uid=5826774&amp;link_uid=PMC3923659">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2429.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3069098/figure/pone-0018373-g005/" ref="ordinalpos=2429&amp;ncbi_uid=2883626&amp;link_uid=PMC3069098" href="/pmc/articles/PMC3069098/"><img src="/pmc/articles/instance/3069098/bin/pone.0018373.g005.gif" src-large="/pmc/articles/instance/3069098/bin/pone.0018373.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3069098/" ref="ordinalpos=2429&amp;ncbi_uid=2883626&amp;link_uid=PMC3069098" image-link="/pmc/articles/PMC3069098/figure/pone-0018373-g005/" class="imagepopup">Figure 5. The effects of Wnt5a on elongation are mediated through Rac1 of the                             non-canonical Wnt/PCP <span class="highlight" style="background-color:">pathway</span>..  From: Wnt5a Regulates Midbrain Dopaminergic Axon Growth and                     Guidance. </a></p><div class="supp"><p class="details"><div>Select antagonists were used to identify the downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>                             responsible for mediating the effects of Wnt5a on DA neurites.                             Antagonism of the canonical Wnt <span class="highlight" style="background-color:">pathway</span> (using Dkk1 recombinant                             protein), the canonical and non-canonical Wnt pathways (using casein                             kinase 1 inhibitor, D4476) and even more selectively the non-canonical                             PCP <span class="highlight" style="background-color:">pathway</span> (using Rac1 inhibitor, NSC23766) revealed that the effect of                             Wnt5a on (<b>A</b>) neurite length and (<b>B</b>) neurite                             number were mediated by the non-canonical PCP <span class="highlight" style="background-color:">pathway</span> (D4476 and                             NSC23766 both significantly antagonizing the effects of Wnt5a).                                 (<b>C</b>–<b>F</b>) Examples of changes in DA                             neuron morphology following treatment with Wnt5a ± selective                             antagonists. Scale bar  = 50 µm. Data                             represents mean ± s.e.m., n = 4–5                             cultures; ** p&lt;0.01, *** p&lt;0.001.</div></p></div><div class="aux"><div class="resc">Brette D. Blakely, et al. PLoS One. 2011;6(3):e18373.</div><p class="links"><a class="dblinks" href="/pubmed/21483795" ref="ordinalpos=2429&amp;ncbi_uid=2883626&amp;link_uid=21483795">Citation</a><a class="dblinks" href="/pmc/articles/PMC3069098/" ref="ordinalpos=2429&amp;ncbi_uid=2883626&amp;link_uid=PMC3069098">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2430.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3986937/figure/F8/" ref="ordinalpos=2430&amp;ncbi_uid=6024738&amp;link_uid=PMC3986937" href="/pmc/articles/PMC3986937/"><img src="/pmc/articles/instance/3986937/bin/1742-2094-11-59-8.gif" src-large="/pmc/articles/instance/3986937/bin/1742-2094-11-59-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3986937/" ref="ordinalpos=2430&amp;ncbi_uid=6024738&amp;link_uid=PMC3986937" image-link="/pmc/articles/PMC3986937/figure/F8/" class="imagepopup">Figure 8.  From: Curcumin attenuates acute inflammatory injury by inhibiting the TLR4/MyD88/NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in experimental traumatic brain injury. </a></p><div class="supp"><p class="details"><div><b>Curcumin suppressed the expression of proteins in the microglial TLR4/MyD88/NF-</b>κ<b>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in co-culture system. (A)</b> TLR4 and MyD88 protein expression in microglia was significantly lower in the curcumin-treated group than in the vehicle-treated group at 24 hours after lipopolysaccharide (LPS) stimulation. <b>(B)</b> NF-κB p65 and p-IκB-α protein expression in microglia was also significantly lower in the curcumin-treated group than in the vehicle-treated group at 24 hours following LPS stimulation. In contrast, IκB-α protein expression was significantly higher in the curcumin-treated group than in the vehicle-treated group after LPS stimulation. Values (mean ± SD) are representative of three independent experiments. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Hai-tao Zhu, et al. J Neuroinflammation. 2014;11:59-59.</div><p class="links"><a class="dblinks" href="/pubmed/24669820" ref="ordinalpos=2430&amp;ncbi_uid=6024738&amp;link_uid=24669820">Citation</a><a class="dblinks" href="/pmc/articles/PMC3986937/" ref="ordinalpos=2430&amp;ncbi_uid=6024738&amp;link_uid=PMC3986937">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2431.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3552989/figure/F5/" ref="ordinalpos=2431&amp;ncbi_uid=4561615&amp;link_uid=PMC3552989" href="/pmc/articles/PMC3552989/"><img src="/pmc/articles/instance/3552989/bin/1472-6882-12-154-5.gif" src-large="/pmc/articles/instance/3552989/bin/1472-6882-12-154-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3552989/" ref="ordinalpos=2431&amp;ncbi_uid=4561615&amp;link_uid=PMC3552989" image-link="/pmc/articles/PMC3552989/figure/F5/" class="imagepopup">Figure 5.  From: The roots of Atractylodes japonica Koidzumi promote adipogenic differentiation via activation of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 3T3-L1 cells. </a></p><div class="supp"><p class="details"><div><b>Effect of ARA extract on PI3K/IRS-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 3T3-L1 cells.</b> The cells were induced to differentiate into adipocytes with MDI medium with or without ARA (100, 200 and 500 mg/mL) for 8 days. The expression of PI3K and the phosphorylation of IRS-1 were analyzed by western blot. Relative density was calculated as the ratio of PI3K and IRS-1 expression to actin expression. Each value represents the mean ± S.D. (<i>n</i> = 3). *<i>p</i> &lt; 0.05 vs. control. C: control, 100: ARA 100 μg/mL, 250: ARA 250 μg/mL and 500: ARA 500 μg/mL.</div></p></div><div class="aux"><div class="resc">Yunkyung Han, et al. BMC Complement Altern Med. 2012;12:154-154.</div><p class="links"><a class="dblinks" href="/pubmed/22978376" ref="ordinalpos=2431&amp;ncbi_uid=4561615&amp;link_uid=22978376">Citation</a><a class="dblinks" href="/pmc/articles/PMC3552989/" ref="ordinalpos=2431&amp;ncbi_uid=4561615&amp;link_uid=PMC3552989">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2432.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3367558/figure/A008045F2/" ref="ordinalpos=2432&amp;ncbi_uid=6185744&amp;link_uid=PMC3367558" href="/pmc/articles/PMC3367558/"><img src="/pmc/articles/instance/3367558/bin/cshperspect-WNT-008045_F2.gif" src-large="/pmc/articles/instance/3367558/bin/cshperspect-WNT-008045_F2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3367558/" ref="ordinalpos=2432&amp;ncbi_uid=6185744&amp;link_uid=PMC3367558" image-link="/pmc/articles/PMC3367558/figure/A008045F2/" class="imagepopup">Figure 2. From: Wnt <span class="highlight" style="background-color:">Signaling</span> in Neuromuscular Junction Development. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> at vertebrate neuromuscular junctions. (<i>A</i>) Mouse Wnt3 and zebrafish Wnt11r function during prepatterning. Wnt11r <span class="highlight" style="background-color:">signaling</span> through Unplugged/MuSK before innervation leads to aneural AChR clustering and also via a parallel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to changes in extracellular matrix modifications (EMMs), possibly via chondroitin sulfate proteoglycans (CSPGs) that eventually influence incoming motor neuron axonal growth cone guidance. Wnt3 <span class="highlight" style="background-color:">signaling</span> leads to the rapid and transient activation of Rac1 and AChR microcluster formation, which (<i>B</i>) in the presence of neural agrin at the time of innervation are stabilized in exact opposition to the presynaptic terminal.</div></p></div><div class="aux"><div class="resc">Kate Koles, et al. Cold Spring Harb Perspect Biol. 2012 June;4(6):a008045.</div><p class="links"><a class="dblinks" href="/pubmed/22510459" ref="ordinalpos=2432&amp;ncbi_uid=6185744&amp;link_uid=22510459">Citation</a><a class="dblinks" href="/pmc/articles/PMC3367558/" ref="ordinalpos=2432&amp;ncbi_uid=6185744&amp;link_uid=PMC3367558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2433.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634110/figure/F2/" ref="ordinalpos=2433&amp;ncbi_uid=1708624&amp;link_uid=PMC2634110" href="/pmc/articles/PMC2634110/"><img src="/pmc/articles/instance/2634110/bin/psb0303_0166_fig002.gif" src-large="/pmc/articles/instance/2634110/bin/psb0303_0166_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634110/" ref="ordinalpos=2433&amp;ncbi_uid=1708624&amp;link_uid=PMC2634110" image-link="/pmc/articles/PMC2634110/figure/F2/" class="imagepopup">Figure 2.  From: The role of phytohormone <span class="highlight" style="background-color:">signaling</span> in ozone-induced cell death in plants. </a></p><div class="supp"><p class="details"><div>Proposed pathways for ethylene biosynthesis and <span class="highlight" style="background-color:">signaling</span> in ozone-exposed plants. Activity of ACC synthase (ACS), the key enzyme of this <span class="highlight" style="background-color:">pathway</span>, is regulated with MPK6 in <i>Arabidopsis</i> or SIPK in tobacco, which activity is controlled with cytosolic free Ca<sup>2+</sup>. Ethylene binds to ethylene receptors (e.g., ETR), and ethylene signal is transmitted via EIN2.</div></p></div><div class="aux"><div class="resc">Masanori Tamaoki. Plant Signal Behav. 2008 March;3(3):166-174.</div><p class="links"><a class="dblinks" href="/pubmed/19513211" ref="ordinalpos=2433&amp;ncbi_uid=1708624&amp;link_uid=19513211">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634110/" ref="ordinalpos=2433&amp;ncbi_uid=1708624&amp;link_uid=PMC2634110">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2434.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3524194/figure/F1/" ref="ordinalpos=2434&amp;ncbi_uid=4453288&amp;link_uid=PMC3524194" href="/pmc/articles/PMC3524194/"><img src="/pmc/articles/instance/3524194/bin/cc-11-4097-g1.gif" src-large="/pmc/articles/instance/3524194/bin/cc-11-4097-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3524194/" ref="ordinalpos=2434&amp;ncbi_uid=4453288&amp;link_uid=PMC3524194" image-link="/pmc/articles/PMC3524194/figure/F1/" class="imagepopup">Figure. From: Upstream regulators of the Hippo <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> GPCR <span class="highlight" style="background-color:">signaling</span> regulates the Hippo <span class="highlight" style="background-color:">pathway</span>. Gα<sub>12/13</sub>-, Gα<sub>q/11</sub>- and Gα<sub>i/o</sub>-coupled receptors and ligands activate Rho GTPases, inhibit Lats1/2 and induce YAP/TAZ. Gα<sub>s</sub>-coupled receptors and ligands induce Lats1/2, leading to inhibition of YAP/TAZ activity. Mechanical cues may also modulate YAP/TAZ activity by regulating Rho GTPases. YAP/TAZ regulates a transcriptional program to control organ size, tumorigenesis and stem cell maintenance.</div></p></div><div class="aux"><div class="resc">Fa-Xing Yu, et al. Cell Cycle. 2012 November 15;11(22):4097-4098.</div><p class="links"><a class="dblinks" href="/pubmed/23075495" ref="ordinalpos=2434&amp;ncbi_uid=4453288&amp;link_uid=23075495">Citation</a><a class="dblinks" href="/pmc/articles/PMC3524194/" ref="ordinalpos=2434&amp;ncbi_uid=4453288&amp;link_uid=PMC3524194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2435.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4061726/figure/F1/" ref="ordinalpos=2435&amp;ncbi_uid=6259302&amp;link_uid=PMC4061726" href="/pmc/articles/PMC4061726/"><img src="/pmc/articles/instance/4061726/bin/emss-59125-f0001.gif" src-large="/pmc/articles/instance/4061726/bin/emss-59125-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4061726/" ref="ordinalpos=2435&amp;ncbi_uid=6259302&amp;link_uid=PMC4061726" image-link="/pmc/articles/PMC4061726/figure/F1/" class="imagepopup">Figure 1. Wnt <span class="highlight" style="background-color:">signaling</span> cascades..  From: Targeting Wnt <span class="highlight" style="background-color:">signaling</span> at the neuroimmune interface for dopaminergic neuroprotection/repair in Parkinson's disease. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) In canonical Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span>, binding of Wnts (‘Wnt on’) to a receptor complex composed of Fzd and LRP family members inhibits the APC/GSK-3β destruction complex and blockade of β-catenin (β-cat) by GSK-3β. β-cat then accumulates in the cytoplasm and translocates to the nucleus where it regulates target gene expression with TCF/LEF family of transcription factors. In the absence of Wnt ligand (‘Wnt off’), β-cat is targeted for proteolytic degradation by the APC/GSK-3β destruction complex. The proteins Norrin and R-spondin (Rspo) are unrelated to Wnt and act as Wnt agonists, whereas Dkk1, Wif, and sFRPs act as antagonists. (<b>B</b>) In noncanonical Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span>, the binding of Wnts promotes Fzd-mediated activation of G proteins, leading to the release of Ca<sup>2+</sup> from intracellular stores and consequent activation of Ca<sup>2+</sup>-dependent effector molecules. Several Ca<sup>2+</sup>-sensitive targets, i.e. PKC, CamKII, and calcineurin, have been identified downstream of the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span>. Targets of the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> appear to interact with the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> at multiple points. Additionally, Fzd receptors in association with Kny, Ror2, or Ryk receptors can activate JNK, promoting target gene expression through AP-1. In noncanonical Wnt/PCP <span class="highlight" style="background-color:">pathway</span>, the binding of Wnts activates RhoA/B, Cdc42, or Rac1. Dvl activates Rac1, which can activate JNK to signal through the NFAT <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Francesca L’Episcopo, et al. J Mol Cell Biol. ;6(1):13-26.</div><p class="links"><a class="dblinks" href="/pubmed/24431301" ref="ordinalpos=2435&amp;ncbi_uid=6259302&amp;link_uid=24431301">Citation</a><a class="dblinks" href="/pmc/articles/PMC4061726/" ref="ordinalpos=2435&amp;ncbi_uid=6259302&amp;link_uid=PMC4061726">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2436.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3824689/figure/fig3/" ref="ordinalpos=2436&amp;ncbi_uid=5464493&amp;link_uid=PMC3824689" href="/pmc/articles/PMC3824689/"><img src="/pmc/articles/instance/3824689/bin/cddis2013380f3.gif" src-large="/pmc/articles/instance/3824689/bin/cddis2013380f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3824689/" ref="ordinalpos=2436&amp;ncbi_uid=5464493&amp;link_uid=PMC3824689" image-link="/pmc/articles/PMC3824689/figure/fig3/" class="imagepopup">Figure 3.  From: IBP-mediated suppression of autophagy promotes growth and metastasis of breast cancer cells via activating mTORC2/Akt/FOXO3a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>IBP-mediated inhibition of autophagy is dependent on the mTORC2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in breast cancer cells. (<b>a</b> and <b>b</b>) MDA-MB-231 and MDA-MB-468 cells with different levels of IBP were treated without or with LY294002 (25 <i>μ</i>m), rapamycin (10nm), OSI-027 (10 <i>μ</i>m). Immunoblotting detected the levels of autophagy. (<b>c</b> and <b>d</b>) Cells were incubated with either rapamycin (10 nm) or OSI-027 (10 <i>μ</i>m) or combined the two for 2 h. Immunoblotting detected the level of p-mTOR Ser2481, p-Akt Ser473, p-FOXO3a Thr32 and LC3-II</div></p></div><div class="aux"><div class="resc">S Chen, et al. Cell Death Dis. 2013 October;4(10):e842.</div><p class="links"><a class="dblinks" href="/pubmed/24113176" ref="ordinalpos=2436&amp;ncbi_uid=5464493&amp;link_uid=24113176">Citation</a><a class="dblinks" href="/pmc/articles/PMC3824689/" ref="ordinalpos=2436&amp;ncbi_uid=5464493&amp;link_uid=PMC3824689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2437.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3563200/figure/fig05/" ref="ordinalpos=2437&amp;ncbi_uid=4616855&amp;link_uid=PMC3563200" href="/pmc/articles/PMC3563200/"><img src="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f5.gif" src-large="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3563200/" ref="ordinalpos=2437&amp;ncbi_uid=4616855&amp;link_uid=PMC3563200" image-link="/pmc/articles/PMC3563200/figure/fig05/" class="imagepopup">Figure 5.  From: Indoleamine 2,3-dioxygenase-1 (IDO1) enhances survival and invasiveness of endometrial stromal cells via the activation of JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>IDO1 stimulates COX-2, MMP-9 expression in ESCs through activation of JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IDO1 in normal ESCs was overexpressed or blockaded by plasmid transfection, and treated with or without JNK inhibitor (SP600125, 20μM). The expression of invasion-related proteins MMP-2 (A), MMP-9 (B), TIMP-1 (C) and COX-2 (D) were determined by in-cell Western. Data were shown as the mean ± SD calculus of 8 different experiments; *P&lt;.05 vs. vector control transfected ESCs without SP600125; **P&lt;.01 vs. vector control transfected ESCs without SP600125; #P&lt;.05, IDO1-overexpressing ESCs with SP600125 vs. IDO1-overexpressing ESCs without SP600125; ##P&lt;.01, IDO1-overexpressing ESCs with SP600125 vs. IDO1-overexpressing ESCs without SP600125.</div></p></div><div class="aux"><div class="resc">Jie Mei, et al. Int J Clin Exp Pathol. 2013;6(3):431-444.</div><p class="links"><a class="dblinks" href="/pubmed/23411497" ref="ordinalpos=2437&amp;ncbi_uid=4616855&amp;link_uid=23411497">Citation</a><a class="dblinks" href="/pmc/articles/PMC3563200/" ref="ordinalpos=2437&amp;ncbi_uid=4616855&amp;link_uid=PMC3563200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2438.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3656027/figure/F1/" ref="ordinalpos=2438&amp;ncbi_uid=4924126&amp;link_uid=PMC3656027" href="/pmc/articles/PMC3656027/"><img src="/pmc/articles/instance/3656027/bin/cib-6-e24214-g1.gif" src-large="/pmc/articles/instance/3656027/bin/cib-6-e24214-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3656027/" ref="ordinalpos=2438&amp;ncbi_uid=4924126&amp;link_uid=PMC3656027" image-link="/pmc/articles/PMC3656027/figure/F1/" class="imagepopup">Figure. From: SIMPLE: A new regulator of endosomal trafficking and <span class="highlight" style="background-color:">signaling</span> in health and disease. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Endosomal trafficking and <span class="highlight" style="background-color:">signaling</span> dysregulation as a potential pathogenic mechanism in CMT and other diseases that involve SIMPLE dysfunction. (<b>A</b>) Domain structure of SIMPLE and mutations found in CMT1C patients. PSAP, predicted TSG101-binding site; PPSY, predicted NEDD4-binding site; C-rich domain, cysteine-rich domain; TMD, predicted transmembrane domain. The locations of CMT1C-linked SIMPLE mutations are indicated on the domain structure. (<b>B</b>) Potential pathogenic roles of SIMPLE dysfunction in CMT, cancer, obesity and inflammatory bowel diseases. Our recent work 24  suggests a pathogenic <span class="highlight" style="background-color:">pathway</span> by which CMT1C-linked SIMPLE mutations 2 <sup>-</sup> 6  cause demyelinating peripheral neuropathy by disrupting endosome-to-lysosome trafficking and <span class="highlight" style="background-color:">signaling</span> attenuation of NRG1-activated ErbB2/ErbB3 receptors and consequently prolonging their <span class="highlight" style="background-color:">signaling</span> to downstream pathways in Schwann cells (colored in pink). Downregulation of SIMPLE expression found in several types of cancer 11 <sup>-</sup> 14  may contribute to the process of malignant transformation by impairing endosome-to-lysosome trafficking and <span class="highlight" style="background-color:">signaling</span> attenuation of mitogenic <span class="highlight" style="background-color:">signaling</span> receptors (colored in lavender). Upregulation of SIMPLE expression found in obesity 15  and inflammatory bowel diseases 16  may contribute to the pathogenesis or progression of these diseases by altering endosomal trafficking and intracellular <span class="highlight" style="background-color:">signaling</span> (colored in blue).</div></p></div><div class="aux"><div class="resc">Lih-Shen Chin, et al. Commun Integr Biol. 2013 May 1;6(3):e24214.</div><p class="links"><a class="dblinks" href="/pubmed/23713142" ref="ordinalpos=2438&amp;ncbi_uid=4924126&amp;link_uid=23713142">Citation</a><a class="dblinks" href="/pmc/articles/PMC3656027/" ref="ordinalpos=2438&amp;ncbi_uid=4924126&amp;link_uid=PMC3656027">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2439.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3821797/figure/F1/" ref="ordinalpos=2439&amp;ncbi_uid=5478607&amp;link_uid=PMC3821797" href="/pmc/articles/PMC3821797/"><img src="/pmc/articles/instance/3821797/bin/nihms525686f1.gif" src-large="/pmc/articles/instance/3821797/bin/nihms525686f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3821797/" ref="ordinalpos=2439&amp;ncbi_uid=5478607&amp;link_uid=PMC3821797" image-link="/pmc/articles/PMC3821797/figure/F1/" class="imagepopup">Figure 1.  From: B cell survival. </a></p><div class="supp"><p class="details"><div>TRAF2 and TRAF3 cooperatively and constitutively repress the alternative NF-κB <span class="highlight" style="background-color:">pathway</span>, with BAFF-R <span class="highlight" style="background-color:">signaling</span> relieving this suppression. In all reports describing deficient BAFF-R <span class="highlight" style="background-color:">signaling</span>, blunted naive mature B cell survival and development is the result (<b>a</b>). When BAFF-R <span class="highlight" style="background-color:">signaling</span> is deficient, TRAF3 and TRAF2 cooperate to constitutively degrade NIK and thus block activation of the alternative NK-κB <span class="highlight" style="background-color:">pathway</span> (<b>a</b>). It is thought that both TRAF2 and TRAF3 bind to NIK and cooperate in its degradation, with the heterocomplex of TRAF2/TRAF3/NIK depicted in (<b>a</b>) purely hypothetical. Upon ligation of BAFF-R by BAFF (<b>b</b>), TRAF3 is recruited to an atypical TRAF3-binding site located in the cytoplasmic tail of BAFF-R (depicted by the amino acid sequence PVPAT).<sup>6</sup> TRAF3 is then targeted for degradation in a TRAF2-dependent manner. Owing to depleted TRAF3 levels, NIK is no longer the subject of continual degradation and can activate the alternative NF-κB <span class="highlight" style="background-color:">pathway</span>. Through NF-κB2 <span class="highlight" style="background-color:">signaling</span>, B cell survival and the transitional B cell developmental checkpoints occur.</div></p></div><div class="aux"><div class="resc">Micah J Benson, et al. Immunol Cell Biol. 2008 Aug-Sep;86(6):10.1038/icb.2008.35.</div><p class="links"><a class="dblinks" href="/pubmed/18490932" ref="ordinalpos=2439&amp;ncbi_uid=5478607&amp;link_uid=18490932">Citation</a><a class="dblinks" href="/pmc/articles/PMC3821797/" ref="ordinalpos=2439&amp;ncbi_uid=5478607&amp;link_uid=PMC3821797">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2440.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3589551/figure/F2/" ref="ordinalpos=2440&amp;ncbi_uid=4680267&amp;link_uid=PMC3589551" href="/pmc/articles/PMC3589551/"><img src="/pmc/articles/instance/3589551/bin/nihms2731f2.gif" src-large="/pmc/articles/instance/3589551/bin/nihms2731f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3589551/" ref="ordinalpos=2440&amp;ncbi_uid=4680267&amp;link_uid=PMC3589551" image-link="/pmc/articles/PMC3589551/figure/F2/" class="imagepopup">Figure 2.  From: Dysregulated Src upregulation of NMDA receptor activity: a common link in chronic pain and schizophrenia. </a></p><div class="supp"><p class="details"><div>A model for regulation of LTP induction in the hippocampus by NRG1-ErbB4 <span class="highlight" style="background-color:">signaling</span> through suppressing Src enhancement of NMDARs. The cartoon illustrates induction of LTP by theta burst stimulations under physiological conditions (right), suppressed NRG1-ErbB4 <span class="highlight" style="background-color:">signaling</span> (top) and enhanced NRG1-<span class="highlight" style="background-color:">signaling</span> (bottom). Under physiological conditions, the NRG1-ErbB4 <span class="highlight" style="background-color:">pathway</span> is a partial brake on the induction of LTP as genetic or pharmacological suppression of this <span class="highlight" style="background-color:">pathway</span> increases the magnitude of the potentiation. Gain-of-function of NRG1-ErbB4 <span class="highlight" style="background-color:">signaling</span> may block induction of LTP.</div></p></div><div class="aux"><div class="resc">Michael W. Salter, et al. FEBS J. 2012 January;279(1):2-11.</div><p class="links"><a class="dblinks" href="/pubmed/21985289" ref="ordinalpos=2440&amp;ncbi_uid=4680267&amp;link_uid=21985289">Citation</a><a class="dblinks" href="/pmc/articles/PMC3589551/" ref="ordinalpos=2440&amp;ncbi_uid=4680267&amp;link_uid=PMC3589551">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2441.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC524028/figure/F2/" ref="ordinalpos=2441&amp;ncbi_uid=544210&amp;link_uid=PMC524028" href="/pmc/articles/PMC524028/"><img src="/pmc/articles/instance/524028/bin/1743-7075-1-3-2.gif" src-large="/pmc/articles/instance/524028/bin/1743-7075-1-3-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC524028/" ref="ordinalpos=2441&amp;ncbi_uid=544210&amp;link_uid=PMC524028" image-link="/pmc/articles/PMC524028/figure/F2/" class="imagepopup">Figure 2.  From: Amino acids as regulators of gene expression. </a></p><div class="supp"><p class="details"><div>Regulation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is controlled through various upstream kinases (e.g. AMPK, AKT, and MK2) that converge on the tuberous sclerosis complex, TSC1•TSC2. TSC2 is a GTPase-activator protein for Rheb which is a positive effector of <span class="highlight" style="background-color:">signaling</span> through mTOR. mTOR signals to downstream targets such as 4E-BP1 and S6K1 as a complex with the regulatory proteins raptor and GβL as described in detail in the text.</div></p></div><div class="aux"><div class="resc">Scot R Kimball, et al. Nutr Metab (Lond). 2004;1:3-3.</div><p class="links"><a class="dblinks" href="/pubmed/15507151" ref="ordinalpos=2441&amp;ncbi_uid=544210&amp;link_uid=15507151">Citation</a><a class="dblinks" href="/pmc/articles/PMC524028/" ref="ordinalpos=2441&amp;ncbi_uid=544210&amp;link_uid=PMC524028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2442.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3092259/figure/fig1-1947601910385449/" ref="ordinalpos=2442&amp;ncbi_uid=3024751&amp;link_uid=PMC3092259" href="/pmc/articles/PMC3092259/"><img src="/pmc/articles/instance/3092259/bin/10.1177_1947601910385449-fig1.gif" src-large="/pmc/articles/instance/3092259/bin/10.1177_1947601910385449-fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3092259/" ref="ordinalpos=2442&amp;ncbi_uid=3024751&amp;link_uid=PMC3092259" image-link="/pmc/articles/PMC3092259/figure/fig1-1947601910385449/" class="imagepopup">Figure 1. From: Reduction of Human Embryonal Rhabdomyosarcoma Tumor Growth by Inhibition of the Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>E-RMS cells express HH <span class="highlight" style="background-color:">signaling</span> target genes and components of the <span class="highlight" style="background-color:">pathway</span>. (<b>A</b>) Real-time RT-PCR analysis of the HH <span class="highlight" style="background-color:">pathway</span> target genes PTCH1, HHIP, and SFRP1 expression in the E-RMS cell lines JR-1, RD, Rh36, CCA, and CT-TC and the EWS cell line A673. (<b>B</b>) Real-time RT-PCR analysis of GLI1, GLI2, GLI3, and SMO expression in the E-RMS cell lines and the EWS cell line. For both (<b>A</b> and <b>B</b>), the expression levels relative to human fetal skeletal muscle cells (HFSMC), after normalization for the housekeeping gene GAPDH, are shown.</div></p></div><div class="aux"><div class="resc">Ulrica Tostar, et al. Genes Cancer. 2010 September;1(9):941-951.</div><p class="links"><a class="dblinks" href="/pubmed/21779473" ref="ordinalpos=2442&amp;ncbi_uid=3024751&amp;link_uid=21779473">Citation</a><a class="dblinks" href="/pmc/articles/PMC3092259/" ref="ordinalpos=2442&amp;ncbi_uid=3024751&amp;link_uid=PMC3092259">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2443.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3617064/figure/F4/" ref="ordinalpos=2443&amp;ncbi_uid=6091022&amp;link_uid=PMC3617064" href="/pmc/articles/PMC3617064/"><img src="/pmc/articles/instance/3617064/bin/nihms445882f4.gif" src-large="/pmc/articles/instance/3617064/bin/nihms445882f4.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3617064/" ref="ordinalpos=2443&amp;ncbi_uid=6091022&amp;link_uid=PMC3617064" image-link="/pmc/articles/PMC3617064/figure/F4/" class="imagepopup">Fig 4. LD053 inhibits oncogenic <span class="highlight" style="background-color:">signaling</span> mediated by Akt and c-Raf.  From: Inhibition of gastric tumor growth by a novel HSP90 inhibitor. </a></p><div class="supp"><p class="details"><div>(A) Schematic representative of the oncogenic <span class="highlight" style="background-color:">signaling</span> mediated by c-Raf and Akt. (B, C) BCG823 cells were treated with varying amounts of LD053 for 48 h, or 10 μM of LD053 for different time, and subjected to Western blotting for phosphorylation levels of proteins involved in the c-Raf <span class="highlight" style="background-color:">pathway</span>. (D, E) BCG823 cells were treated as in (B, C), and tested for proteins involved in the Akt <span class="highlight" style="background-color:">pathway</span>. Relative protein levels were quantified by densitometry and indicated under each blot. Relative protein levels were quantified by densitometry and shown in the histograms.</div></p></div><div class="aux"><div class="resc">Chunwan Lu, et al. Biochem Pharmacol. 2013 May 1;85(9):1246-1256.</div><p class="links"><a class="dblinks" href="/pubmed/23415900" ref="ordinalpos=2443&amp;ncbi_uid=6091022&amp;link_uid=23415900">Citation</a><a class="dblinks" href="/pmc/articles/PMC3617064/" ref="ordinalpos=2443&amp;ncbi_uid=6091022&amp;link_uid=PMC3617064">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2444.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3328762/figure/f2/" ref="ordinalpos=2444&amp;ncbi_uid=4853753&amp;link_uid=PMC3328762" href="/pmc/articles/PMC3328762/"><img src="/pmc/articles/instance/3328762/bin/fig-2.gif" src-large="/pmc/articles/instance/3328762/bin/fig-2.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3328762/" ref="ordinalpos=2444&amp;ncbi_uid=4853753&amp;link_uid=PMC3328762" image-link="/pmc/articles/PMC3328762/figure/f2/" class="imagepopup">FIG. 2. From: Efficient Differentiation of Human Pluripotent Stem Cells into Mesenchymal Stem Cells by Modulating Intracellular <span class="highlight" style="background-color:">Signaling</span> Pathways in a Feeder/Serum-Free System. </a></p><div class="supp"><p class="details"><div>Effect of inhibition of each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on the activation of mesoderm-lineage genes in hESCs. Effects of <b>(A)</b> TGF-β <span class="highlight" style="background-color:">signaling</span> inhibitor (SB431542, SB), <b>(B)</b> WNT <span class="highlight" style="background-color:">signaling</span> inhibitor [dickkopf-related protein 1 (DKK1), DKK], and <b>(C)</b> BMP <span class="highlight" style="background-color:">signaling</span> inhibitor (Noggin, Ng) on the expression patterns of mesoderm-lineage genes in hESCs. After 3 days of treatment, real-time PCR was used to measure the expression levels of mesoderm-lineage genes (<i>T</i>, <i>MIXL1</i>, and <i>WNT3a</i>) in the chemical-treated hESCs. Untreated hESCs and ABB-treated hESCs were used as the negative control and positive control, respectively. Statistical significance among samples was evaluated by ANOVA (*<i>P</i>&lt;0.05). ANOVA, analysis of variance.</div></p></div><div class="aux"><div class="resc">Ngoc-Tung Tran, et al. Stem Cells Dev. 2012 May 1;21(7):1165-1175.</div><p class="links"><a class="dblinks" href="/pubmed/21793661" ref="ordinalpos=2444&amp;ncbi_uid=4853753&amp;link_uid=21793661">Citation</a><a class="dblinks" href="/pmc/articles/PMC3328762/" ref="ordinalpos=2444&amp;ncbi_uid=4853753&amp;link_uid=PMC3328762">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2445.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1458946/figure/F5/" ref="ordinalpos=2445&amp;ncbi_uid=641623&amp;link_uid=PMC1458946" href="/pmc/articles/PMC1458946/"><img src="/pmc/articles/instance/1458946/bin/zpq0160619370005.gif" src-large="/pmc/articles/instance/1458946/bin/zpq0160619370005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1458946/" ref="ordinalpos=2445&amp;ncbi_uid=641623&amp;link_uid=PMC1458946" image-link="/pmc/articles/PMC1458946/figure/F5/" class="imagepopup">Fig. 5. From: Cell-cell <span class="highlight" style="background-color:">signaling</span> in Xanthomonas campestris involves an HD-GYP domain protein that functions in cyclic di-GMP turnover. </a></p><div class="supp"><p class="details"><div>Reconstruction of the Rpf/DSF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>P. aeruginosa</i>. Expression of RpfG and RpfC together (from the <i>rpfGHC</i> operon) partially restores swarming motility to the <i>rsmA</i> mutant of <i>P. aeruginosa</i> (upper left). Addition of DSF to the medium in increasing amounts allows a full restoration to wild-type motility.</div></p></div><div class="aux"><div class="resc">Robert P. Ryan, et al. Proc Natl Acad Sci U S A. 2006 April 25;103(17):6712-6717.</div><p class="links"><a class="dblinks" href="/pubmed/16611728" ref="ordinalpos=2445&amp;ncbi_uid=641623&amp;link_uid=16611728">Citation</a><a class="dblinks" href="/pmc/articles/PMC1458946/" ref="ordinalpos=2445&amp;ncbi_uid=641623&amp;link_uid=PMC1458946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2446.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3950190/figure/F2/" ref="ordinalpos=2446&amp;ncbi_uid=5913018&amp;link_uid=PMC3950190" href="/pmc/articles/PMC3950190/"><img src="/pmc/articles/instance/3950190/bin/dmj-38-13-g002.gif" src-large="/pmc/articles/instance/3950190/bin/dmj-38-13-g002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3950190/" ref="ordinalpos=2446&amp;ncbi_uid=5913018&amp;link_uid=PMC3950190" image-link="/pmc/articles/PMC3950190/figure/F2/" class="imagepopup">Fig. 2.  From: The Hijacking of Cellular <span class="highlight" style="background-color:">Signaling</span> and the Diabetes Epidemic: Mechanisms of Environmental Disruption of Insulin Action and Glucose Homeostasis. </a></p><div class="supp"><p class="details"><div>Endocrine disrupting chemical (EDC) modulation of glucocorticoid activity. The effects of multiple studies examining the effects of EDCs on the glucocorticoid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are summarized, including effects at various concentrations and in different model systems. As such, these studies should not be interpreted to mean that each EDCs operates similarly in all tissues; however, pooling of the data suggests potential mechanisms of synergy among various EDCs that modulate activity of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. 11β-HSD, 11β-hydroxysteroid dehydrogenase; BPA, bisphenol A; PCB, polychlorinated biphenyl; GR-GRE, glucocorticoid receptor-glucocorticoid response element.</div></p></div><div class="aux"><div class="resc">Robert M. Sargis. Diabetes Metab J. 2014 February;38(1):13-24.</div><p class="links"><a class="dblinks" href="/pubmed/24627823" ref="ordinalpos=2446&amp;ncbi_uid=5913018&amp;link_uid=24627823">Citation</a><a class="dblinks" href="/pmc/articles/PMC3950190/" ref="ordinalpos=2446&amp;ncbi_uid=5913018&amp;link_uid=PMC3950190">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2447.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC85825/figure/F9/" ref="ordinalpos=2447&amp;ncbi_uid=18518&amp;link_uid=PMC85825" href="/pmc/articles/PMC85825/"><img src="/pmc/articles/instance/85825/bin/mb1301596009.gif" src-large="/pmc/articles/instance/85825/bin/mb1301596009.jpg" alt="FIG. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC85825/" ref="ordinalpos=2447&amp;ncbi_uid=18518&amp;link_uid=PMC85825" image-link="/pmc/articles/PMC85825/figure/F9/" class="imagepopup">FIG. 9.  From: Mapping of Atypical Protein Kinase C within the Nerve Growth Factor <span class="highlight" style="background-color:">Signaling</span> Cascade: Relationship to Differentiation  and Survival of PC12 Cells. </a></p><div class="supp"><p class="details"><div>Model illustrating the relative positioning of aPKC within the NGF <span class="highlight" style="background-color:">signaling</span> cascades for survival and differentiation. aPKC can activate MEK and lies upstream of JNK. Both Ras and Src lie upstream of aPKC, as does PI3K, whereas aPKC lies upstream of NF-κB, directly interacting with IKK. Differentiation requires components of both the Src and Ras pathways. The differentiation <span class="highlight" style="background-color:">pathway</span> overlaps with components of the survival <span class="highlight" style="background-color:">pathway</span> through Src, aPKC, and PI3K. Survival <span class="highlight" style="background-color:">signaling</span> is Ras independent but dependent on PI3K, Src, aPKC, and NF-κB. Our findings support a model whereby aPKC occupies a critical node, capable of interacting with Ras-MAPK through modulation of MEK and JNK and upstream of NF-κB. NGFR, NGF receptor.</div></p></div><div class="aux"><div class="resc">Marie W. Wooten, et al. Mol Cell Biol. 2000 July;20(13):4494-4504.</div><p class="links"><a class="dblinks" href="/pubmed/10848576" ref="ordinalpos=2447&amp;ncbi_uid=18518&amp;link_uid=10848576">Citation</a><a class="dblinks" href="/pmc/articles/PMC85825/" ref="ordinalpos=2447&amp;ncbi_uid=18518&amp;link_uid=PMC85825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2448.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3260721/figure/F1/" ref="ordinalpos=2448&amp;ncbi_uid=3783571&amp;link_uid=PMC3260721" href="/pmc/articles/PMC3260721/"><img src="/pmc/articles/instance/3260721/bin/psb0608_1198_fig001.gif" src-large="/pmc/articles/instance/3260721/bin/psb0608_1198_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3260721/" ref="ordinalpos=2448&amp;ncbi_uid=3783571&amp;link_uid=PMC3260721" image-link="/pmc/articles/PMC3260721/figure/F1/" class="imagepopup">Figure 1.  From: Auxin modulation of salt stress <span class="highlight" style="background-color:">signaling</span> in Arabidopsis seed germination. </a></p><div class="supp"><p class="details"><div>Figure 1. Schematic working model for NTM2 function in seed germination. High salinity induces <i>NTM2</i> gene transcription and NTM2 protein release from the plasma membranes. The transcriptionally active NTM 2 transcription factor binds directly to the <i>IAA30</i> gene promoter. In this <span class="highlight" style="background-color:">signaling</span> crosstalk, the NTM2 transcription factor links salt signals into an auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> functioning in seed germination.</div></p></div><div class="aux"><div class="resc">Jae-Hoon Jung, et al. Plant Signal Behav. 2011 August;6(8):1198-1200.</div><p class="links"><a class="dblinks" href="/pubmed/21757997" ref="ordinalpos=2448&amp;ncbi_uid=3783571&amp;link_uid=21757997">Citation</a><a class="dblinks" href="/pmc/articles/PMC3260721/" ref="ordinalpos=2448&amp;ncbi_uid=3783571&amp;link_uid=PMC3260721">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2449.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2137898/figure/fig4/" ref="ordinalpos=2449&amp;ncbi_uid=1192998&amp;link_uid=PMC2137898" href="/pmc/articles/PMC2137898/"><img src="/pmc/articles/instance/2137898/bin/jem2040467f04.gif" src-large="/pmc/articles/instance/2137898/bin/jem2040467f04.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2137898/" ref="ordinalpos=2449&amp;ncbi_uid=1192998&amp;link_uid=PMC2137898" image-link="/pmc/articles/PMC2137898/figure/fig4/" class="imagepopup">Figure 4. From: Smad4 is critical for self-renewal of hematopoietic stem cells. </a></p><div class="supp"><p class="details"><div><b>Model for the regulation of HSC self-renewal by the Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The canonical Smad <span class="highlight" style="background-color:">pathway</span> inhibits HSC expansion. This inhibition can be relieved upon enforced expression of the inhibitory Smad7. Recently, TIF1γ has been shown to compete with Smad4 for binding of R-Smads, leading to differentiation of primitive cells into erythrocytes. In addition to its inhibitory role in HSCs as a critical partner in the canonical Smad <span class="highlight" style="background-color:">pathway</span>, Smad4 has a positive effect on self-renewal, possibly as a transducer of unknown regulators outside this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Göran Karlsson, et al. J Exp Med. 2007 March 19;204(3):467-474.</div><p class="links"><a class="dblinks" href="/pubmed/17353364" ref="ordinalpos=2449&amp;ncbi_uid=1192998&amp;link_uid=17353364">Citation</a><a class="dblinks" href="/pmc/articles/PMC2137898/" ref="ordinalpos=2449&amp;ncbi_uid=1192998&amp;link_uid=PMC2137898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2450.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3469665/figure/F5/" ref="ordinalpos=2450&amp;ncbi_uid=5384251&amp;link_uid=PMC3469665" href="/pmc/articles/PMC3469665/"><img src="/pmc/articles/instance/3469665/bin/zh60191280360005.gif" src-large="/pmc/articles/instance/3469665/bin/zh60191280360005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3469665/" ref="ordinalpos=2450&amp;ncbi_uid=5384251&amp;link_uid=PMC3469665" image-link="/pmc/articles/PMC3469665/figure/F5/" class="imagepopup">Fig. 5. From: Differences in oxidative stress status and expression of MKP-1 in dorsal medulla of transgenic rats with altered brain renin-angiotensin system. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> mechanisms regulating mean arterial pressure (MAP) and baroreflex function (BRS) within the brain dorsal medulla of transgenic rats with long-term alterations in brain renin-angiotensin system. <i>A</i>: oxidative stress but not MAPK <span class="highlight" style="background-color:">signaling</span> may be associated with hypertension and impaired autonomic function in transgenic (mRen2)27 rats with higher ANG II actions in the brain. ANG II produces reactive oxygen species (ROS) through NADPH oxidase and mitochondria. Both ANG II and ROS can activate MAPKs, which can further stimulate ROS production through activation of NADPH oxidase. Increased ROS or MAPK <span class="highlight" style="background-color:">signaling</span> can cause higher MAP and impaired BRS directly or through altered redox <span class="highlight" style="background-color:">signaling</span> and downstream gene expression changes leading to hypertension. ROS <span class="highlight" style="background-color:">pathway</span> is predominant in (mRen2)27 rats in the dorsal medullary region. <i>B</i>: reduced MAPK <span class="highlight" style="background-color:">signaling</span> may contribute to lower resting MAP and enhanced BRS in transgenic ASrAOGEN rats with higher ANG-(1–7) actions in the brain. ANG-(1–7) through upregulation of MAP kinase phosphatase (MKP-1) negatively regulates MAPK <span class="highlight" style="background-color:">signaling</span>. Whether unchanged levels of ROS in ASrAOGEN rats is through direct actions of ANG-(1–7) on NADPH oxidase or mitochondria, or through reduced MAPK <span class="highlight" style="background-color:">signaling</span> is not known (?). Reduced MAPK <span class="highlight" style="background-color:">signaling</span> within the brain dorsal medulla of ASrAOGEN rats may contribute to lower MAP and enhanced BRS through altered redox <span class="highlight" style="background-color:">signaling</span> and downstream gene expression changes.</div></p></div><div class="aux"><div class="resc">Manisha Nautiyal, et al. Am J Physiol Regul Integr Comp Physiol. 2012 October 15;303(8):R799-R806.</div><p class="links"><a class="dblinks" href="/pubmed/22914751" ref="ordinalpos=2450&amp;ncbi_uid=5384251&amp;link_uid=22914751">Citation</a><a class="dblinks" href="/pmc/articles/PMC3469665/" ref="ordinalpos=2450&amp;ncbi_uid=5384251&amp;link_uid=PMC3469665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2451.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3337715/figure/F1/" ref="ordinalpos=2451&amp;ncbi_uid=3969652&amp;link_uid=PMC3337715" href="/pmc/articles/PMC3337715/"><img src="/pmc/articles/instance/3337715/bin/nihms-368835-f0001.gif" src-large="/pmc/articles/instance/3337715/bin/nihms-368835-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3337715/" ref="ordinalpos=2451&amp;ncbi_uid=3969652&amp;link_uid=PMC3337715" image-link="/pmc/articles/PMC3337715/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of PI 3-kinase, Akt and mTOR.  From: Phosphoinositide-3 kinase <span class="highlight" style="background-color:">signaling</span> in cardiac hypertrophy and heart failure. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways are depicted schematically. FKBP: FK506-binding protein, GLUT: glucose transporter, GPCR: G protein-coupled receptor, GSK: Glycogen synthase kinase, IGF: Insulin-like growth factor, PDK: phosphoinositide-dependent kinase, PI3K: phosphoinositide 3-kinase, PtdIns: phosphatidylinositol, PTEN: phosphatase and tensin homolog deleted on chromosome ten, p70S6K: p70S6 kinase, TSC: tuberous sclerosis complex, eIF4: eukaryotic initiation factor 4.</div></p></div><div class="aux"><div class="resc">Toshinori Aoyagi, et al. Curr Pharm Des. ;17(18):1818-1824.</div><p class="links"><a class="dblinks" href="/pubmed/21631421" ref="ordinalpos=2451&amp;ncbi_uid=3969652&amp;link_uid=21631421">Citation</a><a class="dblinks" href="/pmc/articles/PMC3337715/" ref="ordinalpos=2451&amp;ncbi_uid=3969652&amp;link_uid=PMC3337715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2452.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4319960/figure/pone.0116995.g007/" ref="ordinalpos=2452&amp;ncbi_uid=7188566&amp;link_uid=PMC4319960" href="/pmc/articles/PMC4319960/"><img src="/pmc/articles/instance/4319960/bin/pone.0116995.g007.gif" src-large="/pmc/articles/instance/4319960/bin/pone.0116995.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4319960/" ref="ordinalpos=2452&amp;ncbi_uid=7188566&amp;link_uid=PMC4319960" image-link="/pmc/articles/PMC4319960/figure/pone.0116995.g007/" class="imagepopup">Figure 7. Schematic representation of TGF-β1-activated <span class="highlight" style="background-color:">signaling</span> pathways mediating mesenchymal cell growth -suppressive and -promoting effects..  From: SMAD-Independent Down-Regulation of Caveolin-1 by TGF-β: Effects on Proliferation and Survival of Myofibroblasts. </a></p><div class="supp"><p class="details"><div>TGF-β1 activates the cell surface TGF-β receptor(s) complex that leads to rapid activation of the canonical SMAD <span class="highlight" style="background-color:">pathway</span> as well as the SMAD-<i>in</i>dependent p38 MAPK <span class="highlight" style="background-color:">pathway</span>. Activation of the SMAD <span class="highlight" style="background-color:">pathway</span> is required for the induction of a cellular program of growth-arrest and myofibroblast differentiation. In contrast, activation of the p38 MAPK <span class="highlight" style="background-color:">pathway</span>, independently of SMAD2/3, is required the down-regulation of Cav-1 by TGF-β1. Down-regulation of Cav-1 by TGF-β1 “primes” differentiated myofibroblasts for enhanced proliferative responses to mitogens and resistance to apoptosis. These divergent TGF-β <span class="highlight" style="background-color:">signaling</span> pathways may explain, in part, the contextual effects of TGF-β1 as both a growth-inhibitor and –promoter on the same target (mesenchymal) cells.</div></p></div><div class="aux"><div class="resc">Yan Y. Sanders, et al. PLoS One. 2015;10(2):e0116995.</div><p class="links"><a class="dblinks" href="/pubmed/25658089" ref="ordinalpos=2452&amp;ncbi_uid=7188566&amp;link_uid=25658089">Citation</a><a class="dblinks" href="/pmc/articles/PMC4319960/" ref="ordinalpos=2452&amp;ncbi_uid=7188566&amp;link_uid=PMC4319960">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2453.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3632376/figure/F7/" ref="ordinalpos=2453&amp;ncbi_uid=5890115&amp;link_uid=PMC3632376" href="/pmc/articles/PMC3632376/"><img src="/pmc/articles/instance/3632376/bin/nihms-450723-f0007.gif" src-large="/pmc/articles/instance/3632376/bin/nihms-450723-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3632376/" ref="ordinalpos=2453&amp;ncbi_uid=5890115&amp;link_uid=PMC3632376" image-link="/pmc/articles/PMC3632376/figure/F7/" class="imagepopup">Figure 7. Proposed model of <span class="highlight" style="background-color:">signaling</span> molecules involved in LTPGABA.  From: Kappa opioid receptors regulate stress-induced cocaine-seeking and synaptic plasticity. </a></p><div class="supp"><p class="details"><div>(A), an <i>in vivo</i> exposure to stress blocks LTP<sub>GABA</sub> through activation of kappa opioid receptors (KORs) with subsequent modulation of the NO <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at a point downstream of NO production. Presynaptic terminals expressing LTP<sub>GABA</sub> may originate from local interneurons or extrinsic GABAergic afferents. (B), <i>in vivo</i> exposure to morphine alters LTP<sub>GABA</sub> through activation of μ-opioid receptors (μOR), inhibiting the cGMP <span class="highlight" style="background-color:">pathway</span> at the level of sGC (soluble guanylate cyclase)(Niehaus et al., 2010). (C), Forskolin potentiates GABAergic synapses by activating adenylyl cyclase (AC), which in turn activates the cAMP/PKA <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> is unaffected by <i>in vivo</i> manipulations with morphine or stress (Nugent et al., 2009).</div></p></div><div class="aux"><div class="resc">Nicholas M. Graziane, et al. Neuron. 2013 March 6;77(5):942-954.</div><p class="links"><a class="dblinks" href="/pubmed/23473323" ref="ordinalpos=2453&amp;ncbi_uid=5890115&amp;link_uid=23473323">Citation</a><a class="dblinks" href="/pmc/articles/PMC3632376/" ref="ordinalpos=2453&amp;ncbi_uid=5890115&amp;link_uid=PMC3632376">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2454.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2491599/figure/F2/" ref="ordinalpos=2454&amp;ncbi_uid=1510816&amp;link_uid=PMC2491599" href="/pmc/articles/PMC2491599/"><img src="/pmc/articles/instance/2491599/bin/1755-7682-1-10-2.gif" src-large="/pmc/articles/instance/2491599/bin/1755-7682-1-10-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2491599/" ref="ordinalpos=2454&amp;ncbi_uid=1510816&amp;link_uid=PMC2491599" image-link="/pmc/articles/PMC2491599/figure/F2/" class="imagepopup">Figure 2.  From: Second hit in cervical carcinogenesis process: involvement of wnt/beta catenin <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Wnt canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. A. When the wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not active, β-catenin is bound to the degradation complex composed by APC, axin, and the serine/theronine kinases CK1 and GSK3. The main role of the degradation complex is to phosphorylate β-catenin leading to its degradation by means of the proteasome-ubiquitin <span class="highlight" style="background-color:">pathway</span>. There are several negative regulators that operate at the receptor-ligand level; such as, Cer1, DKK, WIF1 and sFRP, whose function is to modulate positive signals induced by Wnts. B. The contact of wnt with its receptors leads to the stabilization of β-catenin and its accumulation in cytoplasm and nucleus. β-catenin displaces the transcriptional repressor groucho from the LEF/TCF complex, leading to the activation of target genes, such as c-myc and cyclinD1, which are involved in cell proliferation and cell cycle progression.</div></p></div><div class="aux"><div class="resc">Carlos Perez-Plasencia, et al. Int Arch Med. 2008;1:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/18606007" ref="ordinalpos=2454&amp;ncbi_uid=1510816&amp;link_uid=18606007">Citation</a><a class="dblinks" href="/pmc/articles/PMC2491599/" ref="ordinalpos=2454&amp;ncbi_uid=1510816&amp;link_uid=PMC2491599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2455.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3547545/figure/f1-ebo-9-2013-001/" ref="ordinalpos=2455&amp;ncbi_uid=4564328&amp;link_uid=PMC3547545" href="/pmc/articles/PMC3547545/"><img src="/pmc/articles/instance/3547545/bin/ebo-9-2013-001f1.gif" src-large="/pmc/articles/instance/3547545/bin/ebo-9-2013-001f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3547545/" ref="ordinalpos=2455&amp;ncbi_uid=4564328&amp;link_uid=PMC3547545" image-link="/pmc/articles/PMC3547545/figure/f1-ebo-9-2013-001/" class="imagepopup">Figure 1.  From: The Molecular Evolutionary Patterns of the Insulin/FOXO <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>An illustration of the IIF <span class="highlight" style="background-color:">pathway</span> genes.<br /><b>Notes:</b> Arrows indicate the direction of signal transduction. The numbers on the left side represent the position of the IIF <span class="highlight" style="background-color:">pathway</span> genes.</div></p></div><div class="aux"><div class="resc">Minghui Wang, et al. Evol Bioinform Online. 2013;9:1-16.</div><p class="links"><a class="dblinks" href="/pubmed/23362368" ref="ordinalpos=2455&amp;ncbi_uid=4564328&amp;link_uid=23362368">Citation</a><a class="dblinks" href="/pmc/articles/PMC3547545/" ref="ordinalpos=2455&amp;ncbi_uid=4564328&amp;link_uid=PMC3547545">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2456.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4060082/figure/f5-bt-22-3-200/" ref="ordinalpos=2456&amp;ncbi_uid=6358624&amp;link_uid=PMC4060082" href="/pmc/articles/PMC4060082/"><img src="/pmc/articles/instance/4060082/bin/bt-22-3-200f5.gif" src-large="/pmc/articles/instance/4060082/bin/bt-22-3-200f5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4060082/" ref="ordinalpos=2456&amp;ncbi_uid=6358624&amp;link_uid=PMC4060082" image-link="/pmc/articles/PMC4060082/figure/f5-bt-22-3-200/" class="imagepopup">Fig. 5. From: N-(p-Coumaryol)-Tryptamine Suppresses the Activation of JNK/c-Jun <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in LPS-Challenged RAW264.7 Cells. </a></p><div class="supp"><p class="details"><div>Effect of N-(<i>p</i>-coumaryol) tryptamine on LPS-induced activation of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RAW264.7 macrophage cells. (A) representative immunoblots, (B) quantitative analysis of immunoblots. Cellswere challenged with 200 ng/ml LPS in the absence or presence of <i>N</i>-(<i>p</i>-coumaryol) tryptamine. LPS-induced increased phosphorylation of JNK and c-jun was significantly attenuated with <i>N</i>-(<i>p</i>-coumaryol) tryptamine treatment. However, phosphorylation of ERK and p38 was not affected with <i>N</i>-(<i>p</i>-coumaryol) tryptamine treatment, suggesting that JNK/c-jun <span class="highlight" style="background-color:">signaling</span> might play a key role in the LPS-induced activation of RAW264.7 cells. Images are representative of three independent experiments that shows reproducible results. The values are expressed as mean ± SD for three independent experiments. <sup>*</sup><i>p</i>&lt;0.05 and <sup>**</sup><i>p</i>&lt;0.01 indicate statistically significant differences from treatments with LPS alone. CT stands for <i>N</i>-(<i>p</i>-coumaryol) tryptamine.</div></p></div><div class="aux"><div class="resc">Van Anh Vo, et al. Biomol Ther (Seoul). 2014 May;22(3):200-206.</div><p class="links"><a class="dblinks" href="/pubmed/25009700" ref="ordinalpos=2456&amp;ncbi_uid=6358624&amp;link_uid=25009700">Citation</a><a class="dblinks" href="/pmc/articles/PMC4060082/" ref="ordinalpos=2456&amp;ncbi_uid=6358624&amp;link_uid=PMC4060082">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2457.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2858442/figure/f3/" ref="ordinalpos=2457&amp;ncbi_uid=2277719&amp;link_uid=PMC2858442" href="/pmc/articles/PMC2858442/"><img src="/pmc/articles/instance/2858442/bin/msb20106-f3.gif" src-large="/pmc/articles/instance/2858442/bin/msb20106-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2858442/" ref="ordinalpos=2457&amp;ncbi_uid=2277719&amp;link_uid=PMC2858442" image-link="/pmc/articles/PMC2858442/figure/f3/" class="imagepopup">Figure 3.  From: Dynamics within the CD95 death-inducing <span class="highlight" style="background-color:">signaling</span> complex decide life and death of cells. </a></p><div class="supp"><p class="details"><div>Model of CD95-mediated <span class="highlight" style="background-color:">signaling</span>. A graphic representation of the complete model illustrating the process of DISC formation and subsequent <span class="highlight" style="background-color:">signaling</span> of the apoptotic (depicted in red) and NF-κB <span class="highlight" style="background-color:">pathway</span> (depicted in green). Abbreviations: L, CD95 ligand; R, CD95 receptor; F, FADD; C8, procaspase-8 (p55/p53); C8<sup>*</sup>, active caspase-8; C3, procaspase-3; C3<sup>*</sup>, active caspase-3; C6, procaspase-6; C6<sup>*</sup>, active caspase-6; FL, c-FLIP<sub>L</sub>; FS, c-FLIP<sub>S</sub>; X, C8, FL or FS; p43/p41, 1st cleavage product of procaspase-8; and p43-FLIP, cleavage product of c-FLIP<sub>L</sub>.</div></p></div><div class="aux"><div class="resc">Leo Neumann, et al. Mol Syst Biol. 2010;6:352-352.</div><p class="links"><a class="dblinks" href="/pubmed/20212524" ref="ordinalpos=2457&amp;ncbi_uid=2277719&amp;link_uid=20212524">Citation</a><a class="dblinks" href="/pmc/articles/PMC2858442/" ref="ordinalpos=2457&amp;ncbi_uid=2277719&amp;link_uid=PMC2858442">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2458.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3208761/figure/F5/" ref="ordinalpos=2458&amp;ncbi_uid=3486815&amp;link_uid=PMC3208761" href="/pmc/articles/PMC3208761/"><img src="/pmc/articles/instance/3208761/bin/nihms328128f5a.gif" src-large="/pmc/articles/instance/3208761/bin/nihms328128f5a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3208761/figure/F5/" ref="ordinalpos=2458&amp;ncbi_uid=3486815&amp;link_uid=PMC3208761" href="/pmc/articles/PMC3208761/"><img src="/pmc/articles/instance/3208761/bin/nihms328128f5b.gif" src-large="/pmc/articles/instance/3208761/bin/nihms328128f5b.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3208761/figure/F5/" ref="ordinalpos=2458&amp;ncbi_uid=3486815&amp;link_uid=PMC3208761" href="/pmc/articles/PMC3208761/"><img src="/pmc/articles/instance/3208761/bin/nihms328128f5c.gif" src-large="/pmc/articles/instance/3208761/bin/nihms328128f5c.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3208761/" ref="ordinalpos=2458&amp;ncbi_uid=3486815&amp;link_uid=PMC3208761" image-link="/pmc/articles/PMC3208761/figure/F5/" class="imagepopup">Fig 5. Gene expression analysis of BMP and Wnt <span class="highlight" style="background-color:">signaling</span> pathways in hMSCs on topographic surfaces and planar control at day 3.  From: Modulation of osteogenic differentiation in hMSCs cells by submicron topographically-patterned ridges and grooves. </a></p><div class="supp"><p class="details"><div>Expression of <i>ID1</i>, a downstream gene of the BMP <span class="highlight" style="background-color:">pathway</span> (A), and <i>TCF3</i> (B) and <i>AXIN2</i> (C), downstream genes of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, was measured by quantitative PCR. The expression of each gene was first normalized to the expression of endogenous control <i>TBP</i>, and then normalized to the planar control. No significant increase in BMP or Wnt <span class="highlight" style="background-color:">signaling</span> pathways was noted in hMSCs under growth medium condition. All data are presented as the mean ± SEM.</div></p></div><div class="aux"><div class="resc">Shinya Watari, et al. Biomaterials. ;33(1):128-136.</div><p class="links"><a class="dblinks" href="/pubmed/21982295" ref="ordinalpos=2458&amp;ncbi_uid=3486815&amp;link_uid=21982295">Citation</a><a class="dblinks" href="/pmc/articles/PMC3208761/" ref="ordinalpos=2458&amp;ncbi_uid=3486815&amp;link_uid=PMC3208761">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2459.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2133067/figure/F8/" ref="ordinalpos=2459&amp;ncbi_uid=1181517&amp;link_uid=PMC2133067" href="/pmc/articles/PMC2133067/"><img src="/pmc/articles/instance/2133067/bin/JCB9807101.f8.gif" src-large="/pmc/articles/instance/2133067/bin/JCB9807101.f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2133067/" ref="ordinalpos=2459&amp;ncbi_uid=1181517&amp;link_uid=PMC2133067" image-link="/pmc/articles/PMC2133067/figure/F8/" class="imagepopup">Figure 8.  From: Tumor Suppressor PTEN Inhibits Integrin- and Growth Factor-mediated Mitogen-activated Protein (MAP) Kinase <span class="highlight" style="background-color:">Signaling</span> Pathways . </a></p><div class="supp"><p class="details"><div>Working model of  the PTEN regulatory network. Integrins and growth  factors can separately stimulate the Ras/ERK/MAP kinase <span class="highlight" style="background-color:">pathway</span> or function  collaboratively. Integrin receptor engagement with FN  stimulates FAK, Shc and c-Src,  and they activate the <span class="highlight" style="background-color:">pathway</span> as described (Schlaepfer  et al., 1998). Growth factors  can activate the <span class="highlight" style="background-color:">pathway</span>  through autophosphorylation of growth factor receptors. PTEN reduces Shc phosphorylation and its interaction with Grb2, as well as the downstream Ras/ ERK common MAP kinase <span class="highlight" style="background-color:">pathway</span> for integrin- and growth factor–mediated <span class="highlight" style="background-color:">signaling</span>, and then affects cell spreading and focal  contact formation. Activated MEK1 can partially reverse the latter effects of PTEN.</div></p></div><div class="aux"><div class="resc">Jianguo Gu, et al. J Cell Biol. 1998 November 30;143(5):1375-1383.</div><p class="links"><a class="dblinks" href="/pubmed/9832564" ref="ordinalpos=2459&amp;ncbi_uid=1181517&amp;link_uid=9832564">Citation</a><a class="dblinks" href="/pmc/articles/PMC2133067/" ref="ordinalpos=2459&amp;ncbi_uid=1181517&amp;link_uid=PMC2133067">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2460.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3148081/figure/F6/" ref="ordinalpos=2460&amp;ncbi_uid=3269406&amp;link_uid=PMC3148081" href="/pmc/articles/PMC3148081/"><img src="/pmc/articles/instance/3148081/bin/nihms289444f6.gif" src-large="/pmc/articles/instance/3148081/bin/nihms289444f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3148081/" ref="ordinalpos=2460&amp;ncbi_uid=3269406&amp;link_uid=PMC3148081" image-link="/pmc/articles/PMC3148081/figure/F6/" class="imagepopup">Figure 6. Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis reveals conserved response patterns between human and rhesus MPBSCs (see also Tables S4-7).  From: Prostaglandin E2 enhances engraftment of human cord blood stem cells and shows long-term safety in preclinical non-human primate transplant models. </a></p><div class="supp"><p class="details"><div>Microarray data were subjected to Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis.<br /><b>A,B</b>-Venn diagram illustrations of the overlap of significantly up- and down-regulated <span class="highlight" style="background-color:">signaling</span> pathways between humans and rhesus macaques MPBSCs.<br /><b>C,D</b>-List of significantly up- and down-regulated pathways common between human and rhesus dmPGE2 treated MPBSCs; the ratio indicates the fraction of affected genes of the number of total genes included in each <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Wolfram Goessling, et al. Cell Stem Cell. ;8(4):445-458.</div><p class="links"><a class="dblinks" href="/pubmed/21474107" ref="ordinalpos=2460&amp;ncbi_uid=3269406&amp;link_uid=21474107">Citation</a><a class="dblinks" href="/pmc/articles/PMC3148081/" ref="ordinalpos=2460&amp;ncbi_uid=3269406&amp;link_uid=PMC3148081">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2461.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3763490/figure/F2/" ref="ordinalpos=2461&amp;ncbi_uid=5269896&amp;link_uid=PMC3763490" href="/pmc/articles/PMC3763490/"><img src="/pmc/articles/instance/3763490/bin/nihms503012f2a.gif" src-large="/pmc/articles/instance/3763490/bin/nihms503012f2a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3763490/figure/F2/" ref="ordinalpos=2461&amp;ncbi_uid=5269896&amp;link_uid=PMC3763490" href="/pmc/articles/PMC3763490/"><img src="/pmc/articles/instance/3763490/bin/nihms503012f2b.gif" src-large="/pmc/articles/instance/3763490/bin/nihms503012f2b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3763490/" ref="ordinalpos=2461&amp;ncbi_uid=5269896&amp;link_uid=PMC3763490" image-link="/pmc/articles/PMC3763490/figure/F2/" class="imagepopup">Figure 2. <span class="highlight" style="background-color:">Pathway</span> analysis.  From: Rapid and dynamic alterations of gene expression profiles of adult porcine bone marrow derived stem cell in response to hypoxia. </a></p><div class="supp"><p class="details"><div>(A). Pie graphs illustrate the proportions of the regulated genes enriched in cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B). Illustration of the hypoxia induced genes employed in the cell cycle <span class="highlight" style="background-color:">pathway</span> (KEGG database) The Red and blue symbol represent the genes up- or down-regulated in 6hr H group, respectively.</div></p></div><div class="aux"><div class="resc">Suna Wang, et al. Stem Cell Res. 2010 March;4(2):117-128.</div><p class="links"><a class="dblinks" href="/pubmed/20172499" ref="ordinalpos=2461&amp;ncbi_uid=5269896&amp;link_uid=20172499">Citation</a><a class="dblinks" href="/pmc/articles/PMC3763490/" ref="ordinalpos=2461&amp;ncbi_uid=5269896&amp;link_uid=PMC3763490">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2462.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3209670/figure/F3/" ref="ordinalpos=2462&amp;ncbi_uid=3340138&amp;link_uid=PMC3209670" href="/pmc/articles/PMC3209670/"><img src="/pmc/articles/instance/3209670/bin/1471-2164-12-496-3.gif" src-large="/pmc/articles/instance/3209670/bin/1471-2164-12-496-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3209670/" ref="ordinalpos=2462&amp;ncbi_uid=3340138&amp;link_uid=PMC3209670" image-link="/pmc/articles/PMC3209670/figure/F3/" class="imagepopup">Figure 3.  From: Nutrigenomics in honey bees: digital gene expression analysis of pollen's nutritive effects on healthy and varroa-parasitized bees. </a></p><div class="supp"><p class="details"><div><b>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> affected by pollen feeding or varroa parasitism</b>. Genes from the <i>Drosophila </i>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that were differentially transcribed after pollen feeding or varroa parasitism. The <span class="highlight" style="background-color:">pathway</span> was modified from KEGG: Kyoto Encyclopedia of Genes and Genomes http://www.kegg.com/. Grey indicates the presence of genes in the bee genome. Squares and circles indicate genes and enriched functions that were up- and downregulated between two treatments, respectively.</div></p></div><div class="aux"><div class="resc">Cédric Alaux, et al. BMC Genomics. 2011;12:496-496.</div><p class="links"><a class="dblinks" href="/pubmed/21985689" ref="ordinalpos=2462&amp;ncbi_uid=3340138&amp;link_uid=21985689">Citation</a><a class="dblinks" href="/pmc/articles/PMC3209670/" ref="ordinalpos=2462&amp;ncbi_uid=3340138&amp;link_uid=PMC3209670">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2463.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3135232/figure/fig1/" ref="ordinalpos=2463&amp;ncbi_uid=3089830&amp;link_uid=PMC3135232" href="/pmc/articles/PMC3135232/"><img src="/pmc/articles/instance/3135232/bin/JST2011-603290.001.gif" src-large="/pmc/articles/instance/3135232/bin/JST2011-603290.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3135232/" ref="ordinalpos=2463&amp;ncbi_uid=3089830&amp;link_uid=PMC3135232" image-link="/pmc/articles/PMC3135232/figure/fig1/" class="imagepopup">Figure 1.  From: Functional Diversity of the Schistosoma mansoni Tyrosine Kinases. </a></p><div class="supp"><p class="details"><div>      <i>S. mansoni</i> phosphotyrosine-<span class="highlight" style="background-color:">signaling</span> network. In the phosphotyrosine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the tyrosine kinase (TK), phosphotyrosine phosphatase (PTP), and Src Homology 2 (SH2) domains form a highly interdependent <span class="highlight" style="background-color:">signaling</span> network. At the moment, 81 protein members of the phosphotyrosine <span class="highlight" style="background-color:">signaling</span> network on <i>S. mansoni</i> genome were identified. This <span class="highlight" style="background-color:">signaling</span> network serves as the “<i>writer</i>”, “<i>eraser</i>”, and “<i>reader</i>” domains, respectively, for processing phosphotyrosine targets.</div></p></div><div class="aux"><div class="resc">Lívia G. A. Avelar, et al. J Signal Transduct. 2011;2011:603290.</div><p class="links"><a class="dblinks" href="/pubmed/21776387" ref="ordinalpos=2463&amp;ncbi_uid=3089830&amp;link_uid=21776387">Citation</a><a class="dblinks" href="/pmc/articles/PMC3135232/" ref="ordinalpos=2463&amp;ncbi_uid=3089830&amp;link_uid=PMC3135232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2464.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4349708/figure/pcbi.1004088.g005/" ref="ordinalpos=2464&amp;ncbi_uid=7293551&amp;link_uid=PMC4349708" href="/pmc/articles/PMC4349708/"><img src="/pmc/articles/instance/4349708/bin/pcbi.1004088.g005.gif" src-large="/pmc/articles/instance/4349708/bin/pcbi.1004088.g005.jpg" alt="Fig 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4349708/" ref="ordinalpos=2464&amp;ncbi_uid=7293551&amp;link_uid=PMC4349708" image-link="/pmc/articles/PMC4349708/figure/pcbi.1004088.g005/" class="imagepopup">Fig 5. Crosstalk between host pathways targeted by B. mallei virulence factors..  From: Mining Host-Pathogen Protein Interactions to Characterize Burkholderia mallei Infectivity Mechanisms. </a></p><div class="supp"><p class="details"><div>A) The number of shared proteins and shared <span class="highlight" style="background-color:">pathway</span> protein-protein interactions (PPIs) among human proteins interacting with <i>B. mallei</i> that appeared in the focal adhesion <span class="highlight" style="background-color:">pathway</span> and in up to eight other molecular pathways that shared proteins (partially overlapped) with this <span class="highlight" style="background-color:">pathway</span>. The number of shared proteins across pathways was smaller than the number of shared <span class="highlight" style="background-color:">pathway</span> PPIs. B) The location and number of crosstalk interactions affected by <i>B. mallei</i> centered around the focal adhesion <span class="highlight" style="background-color:">pathway</span> and appear as arrows with line thicknesses proportional to the number of shared PPIs. The identity and number of virulence factors that target each <span class="highlight" style="background-color:">pathway</span> are illustrated using a word cloud. By preferentially targeting <span class="highlight" style="background-color:">signaling</span> pathways, the effect of one protein modulated through interaction with a virulence factor could propagate and disproportionally influence a larger number of biological processes.</div></p></div><div class="aux"><div class="resc">Vesna Memišević, et al. PLoS Comput Biol. 2015 March;11(3):e1004088.</div><p class="links"><a class="dblinks" href="/pubmed/25738731" ref="ordinalpos=2464&amp;ncbi_uid=7293551&amp;link_uid=25738731">Citation</a><a class="dblinks" href="/pmc/articles/PMC4349708/" ref="ordinalpos=2464&amp;ncbi_uid=7293551&amp;link_uid=PMC4349708">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2465.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3978385/figure/F4/" ref="ordinalpos=2465&amp;ncbi_uid=6006523&amp;link_uid=PMC3978385" href="/pmc/articles/PMC3978385/"><img src="/pmc/articles/instance/3978385/bin/nihms-570892-f0004.gif" src-large="/pmc/articles/instance/3978385/bin/nihms-570892-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3978385/" ref="ordinalpos=2465&amp;ncbi_uid=6006523&amp;link_uid=PMC3978385" image-link="/pmc/articles/PMC3978385/figure/F4/" class="imagepopup">Figure 4. Transactivation of Raf <span class="highlight" style="background-color:">signaling</span> by BRAF inhibitors in Ras mutant melanoma.  From: NRAS mutant melanoma: biological behavior and future strategies for therapeutic management. </a></p><div class="supp"><p class="details"><div><b>A.</b> Typically, <i>NRAS</i> mutant melanomas rely on CRAF-mediated MAPK activation, with BRAF locked into an inactive conformation. <b>B.</b> BRAF inhibitors relieve the auto-inhibition of BRAF leading to its recruitment to the plasma membrane. This then leads to the dimerization of BRAF and CRAF leading to enhanced MAPK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> and proliferation.</div></p></div><div class="aux"><div class="resc">Inna V. Fedorenko, et al. Oncogene. 2013 June 20;32(25):3009-3018.</div><p class="links"><a class="dblinks" href="/pubmed/23069660" ref="ordinalpos=2465&amp;ncbi_uid=6006523&amp;link_uid=23069660">Citation</a><a class="dblinks" href="/pmc/articles/PMC3978385/" ref="ordinalpos=2465&amp;ncbi_uid=6006523&amp;link_uid=PMC3978385">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2466.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3288004/figure/pone-0030701-g001/" ref="ordinalpos=2466&amp;ncbi_uid=3835488&amp;link_uid=PMC3288004" href="/pmc/articles/PMC3288004/"><img src="/pmc/articles/instance/3288004/bin/pone.0030701.g001.gif" src-large="/pmc/articles/instance/3288004/bin/pone.0030701.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3288004/" ref="ordinalpos=2466&amp;ncbi_uid=3835488&amp;link_uid=PMC3288004" image-link="/pmc/articles/PMC3288004/figure/pone-0030701-g001/" class="imagepopup">Figure 1. Affymetrix GeneChip microarray and Ingenuity <span class="highlight" style="background-color:">Pathway</span> analyses of lal−/− MDSCs..  From: Gene Profile of Myeloid-Derived Suppressive Cells from the Bone Marrow of Lysosomal Acid Lipase Knock-Out Mice. </a></p><div class="supp"><p class="details"><div><b>A</b>). Affymetrix GeneChip microarray analysis of Ly6G<sup>+</sup> MDSCs from the bone marrow of <i>lal+/+</i> mice and lal−/− mice. Numbers represent percentages of changed genes in each category vs total changed genes between <i>lal−/−</i> mice and <i>lal+/+</i> mice; <b>B</b>). Differential gene expression of Ly6G<sup>+</sup> MDSCs from the bone marrow of <i>lal+/+</i> mice and <i>lal−/−</i> mice was analyzed by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. Changed genes in the PPARγ <span class="highlight" style="background-color:">pathway</span>, cholesterol biosynthesis and mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are presented; <b>C</b>) Flow cytometry of mTOR downstream effectors S6 and 4E-BP1 in Ly6G<sup>+</sup> MDSCs from the bone marrow of <i>lal+/+</i> mice and <i>lal−/−</i> mice. n = 4.</div></p></div><div class="aux"><div class="resc">Cong Yan, et al. PLoS One. 2012;7(2):e30701.</div><p class="links"><a class="dblinks" href="/pubmed/22383970" ref="ordinalpos=2466&amp;ncbi_uid=3835488&amp;link_uid=22383970">Citation</a><a class="dblinks" href="/pmc/articles/PMC3288004/" ref="ordinalpos=2466&amp;ncbi_uid=3835488&amp;link_uid=PMC3288004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2467.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3936914/figure/F4/" ref="ordinalpos=2467&amp;ncbi_uid=5850846&amp;link_uid=PMC3936914" href="/pmc/articles/PMC3936914/"><img src="/pmc/articles/instance/3936914/bin/1476-4598-13-33-4.gif" src-large="/pmc/articles/instance/3936914/bin/1476-4598-13-33-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3936914/" ref="ordinalpos=2467&amp;ncbi_uid=5850846&amp;link_uid=PMC3936914" image-link="/pmc/articles/PMC3936914/figure/F4/" class="imagepopup">Figure 4.  From: Brain microRNAs and insights into biological functions and therapeutic potential of brain enriched miRNA-128. </a></p><div class="supp"><p class="details"><div><b>Biological relevance of miRNA-128 as revealed by bioinformatic analysis: The biological pathways affected by miRNA-128 were revealed by the PANTHER and Gene Codis analysis using the list of common ninet y targets as input.</b> Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and chemokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were the enriched categories in both PANTHER and GeneCodis analysis (p-value &lt; 0.05).</div></p></div><div class="aux"><div class="resc">Yogita K Adlakha, et al. Mol Cancer. 2014;13:33-33.</div><p class="links"><a class="dblinks" href="/pubmed/24555688" ref="ordinalpos=2467&amp;ncbi_uid=5850846&amp;link_uid=24555688">Citation</a><a class="dblinks" href="/pmc/articles/PMC3936914/" ref="ordinalpos=2467&amp;ncbi_uid=5850846&amp;link_uid=PMC3936914">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2468.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1838971/figure/F10/" ref="ordinalpos=2468&amp;ncbi_uid=1029903&amp;link_uid=PMC1838971" href="/pmc/articles/PMC1838971/"><img src="/pmc/articles/instance/1838971/bin/zmk0040780130010.gif" src-large="/pmc/articles/instance/1838971/bin/zmk0040780130010.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1838971/" ref="ordinalpos=2468&amp;ncbi_uid=1029903&amp;link_uid=PMC1838971" image-link="/pmc/articles/PMC1838971/figure/F10/" class="imagepopup">Figure 10. From: Nerve Growth Factor-mediated Neurite Outgrowth via Regulation of Rab5. </a></p><div class="supp"><p class="details"><div>Model of NGF/TrkA-mediated Rab5 inactivation and establishment of <span class="highlight" style="background-color:">signaling</span> endosomes. NGF binds TrkA and induces its endocytosis. TrkA-associated RabGAP5 inactivates Rab5 by promoting its GTP hydrolysis. As a result, the TrkA-containing endocytic vesicles cannot fuse with early endosomes and enter the endocytic <span class="highlight" style="background-color:">pathway</span>. This population of TrkA-containing endocytic vesicles is thus diverted from the conventional endocytic <span class="highlight" style="background-color:">pathway</span> and specializes to become the <span class="highlight" style="background-color:">signaling</span> endosomes.</div></p></div><div class="aux"><div class="resc">Jay Liu, et al. Mol Biol Cell. 2007 April;18(4):1375-1384.</div><p class="links"><a class="dblinks" href="/pubmed/17267689" ref="ordinalpos=2468&amp;ncbi_uid=1029903&amp;link_uid=17267689">Citation</a><a class="dblinks" href="/pmc/articles/PMC1838971/" ref="ordinalpos=2468&amp;ncbi_uid=1029903&amp;link_uid=PMC1838971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2469.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1578677/figure/F7/" ref="ordinalpos=2469&amp;ncbi_uid=828656&amp;link_uid=PMC1578677" href="/pmc/articles/PMC1578677/"><img src="/pmc/articles/instance/1578677/bin/2539fig7.gif" src-large="/pmc/articles/instance/1578677/bin/2539fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1578677/" ref="ordinalpos=2469&amp;ncbi_uid=828656&amp;link_uid=PMC1578677" image-link="/pmc/articles/PMC1578677/figure/F7/" class="imagepopup">Figure 7. From: The C-terminal tail of the Hedgehog receptor Patched regulates both localization and turnover. </a></p><div class="supp"><p class="details"><div>Model of Ptc homo- and heteromultimers. Ptc monomers can multimerize to form stable trimers that repress the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, unless bound by Hh. Ptc monomers with a mutant CTD can form stable trimers that consist entirely of the mutant protein or include wild-type Ptc. Trimers containing CTD mutations can bind Hh, are stable, but are incapable of inactivating the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Xingwu Lu, et al. Genes Dev. 2006 September 15;20(18):2539-2551.</div><p class="links"><a class="dblinks" href="/pubmed/16980583" ref="ordinalpos=2469&amp;ncbi_uid=828656&amp;link_uid=16980583">Citation</a><a class="dblinks" href="/pmc/articles/PMC1578677/" ref="ordinalpos=2469&amp;ncbi_uid=828656&amp;link_uid=PMC1578677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2470.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1877091/figure/F15/" ref="ordinalpos=2470&amp;ncbi_uid=937222&amp;link_uid=PMC1877091" href="/pmc/articles/PMC1877091/"><img src="/pmc/articles/instance/1877091/bin/zmk0060780720015.gif" src-large="/pmc/articles/instance/1877091/bin/zmk0060780720015.jpg" alt="Figure 15." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1877091/" ref="ordinalpos=2470&amp;ncbi_uid=937222&amp;link_uid=PMC1877091" image-link="/pmc/articles/PMC1877091/figure/F15/" class="imagepopup">Figure 15. From: PDCD10 Interacts with Ste20-related Kinase MST4 to Promote Cell Growth and Transformation via Modulation of the ERK <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>PDCD10/MST4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and CCM protein function. The CCM1/2 complex may modulate integrin-mediated cell adhesion via association with ICAP1 in the cytoplasm, as well as organize a p38 MAPK <span class="highlight" style="background-color:">signaling</span> complex (Zawistowski et al., 2005). However, it has been reported that MST4 activated MEK-1/ERK via a Ras/Raf independent <span class="highlight" style="background-color:">pathway</span> (Lin et al., 2001). These data, and ours, confirmed that PDCD10 (CCM3) interacted with MST4 and modulated the ERK MAPK cascade, but not P38 and JNK, resulting in growth induction and cellular transformation.</div></p></div><div class="aux"><div class="resc">Xi Ma, et al. Mol Biol Cell. 2007 June;18(6):1965-1978.</div><p class="links"><a class="dblinks" href="/pubmed/17360971" ref="ordinalpos=2470&amp;ncbi_uid=937222&amp;link_uid=17360971">Citation</a><a class="dblinks" href="/pmc/articles/PMC1877091/" ref="ordinalpos=2470&amp;ncbi_uid=937222&amp;link_uid=PMC1877091">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2471.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3100315/figure/pone-0020041-g006/" ref="ordinalpos=2471&amp;ncbi_uid=2971283&amp;link_uid=PMC3100315" href="/pmc/articles/PMC3100315/"><img src="/pmc/articles/instance/3100315/bin/pone.0020041.g006.gif" src-large="/pmc/articles/instance/3100315/bin/pone.0020041.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3100315/" ref="ordinalpos=2471&amp;ncbi_uid=2971283&amp;link_uid=PMC3100315" image-link="/pmc/articles/PMC3100315/figure/pone-0020041-g006/" class="imagepopup">Figure 6. Ontology analysis reveals many genes on DGE lists are related to p53                             <span class="highlight" style="background-color:">signaling</span>..  From: Glioblastoma Models Reveal the Connection between Adult Glial                     Progenitors and the Proneural Phenotype. </a></p><div class="supp"><p class="details"><div>(A) 104 members of DGE list were connected in Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis.                             Out of these, 8 members of core p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are highlighted as                             red. The shapes on the left indicate the functions and relationships of                             the genes. (B) The biological processes and disease pathways are                             significantly enriched with genes on the DGE list. The yellow line                             indicates the threshold to significance.</div></p></div><div class="aux"><div class="resc">Liang Lei, et al. PLoS One. 2011;6(5):e20041.</div><p class="links"><a class="dblinks" href="/pubmed/21625383" ref="ordinalpos=2471&amp;ncbi_uid=2971283&amp;link_uid=21625383">Citation</a><a class="dblinks" href="/pmc/articles/PMC3100315/" ref="ordinalpos=2471&amp;ncbi_uid=2971283&amp;link_uid=PMC3100315">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2472.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3235159/figure/pone-0028846-g005/" ref="ordinalpos=2472&amp;ncbi_uid=3714243&amp;link_uid=PMC3235159" href="/pmc/articles/PMC3235159/"><img src="/pmc/articles/instance/3235159/bin/pone.0028846.g005.gif" src-large="/pmc/articles/instance/3235159/bin/pone.0028846.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3235159/" ref="ordinalpos=2472&amp;ncbi_uid=3714243&amp;link_uid=PMC3235159" image-link="/pmc/articles/PMC3235159/figure/pone-0028846-g005/" class="imagepopup">Figure 5. Increased activation of the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during vessel remodeling in macLRP1-/- mice..  From: Macrophage LRP1 Suppresses Neo-Intima Formation during Vascular Remodeling by Modulating the TGF-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Representative images from contralateral (a,d) and ligated vessels (b,c,e,f) stained for TGF-β2 (a,b,c) or phospho-Smad 2/3 (d,e,f). Internal elastic lamina (IEL) is marked by a black arrow. Arrowheads indicate a cell positive for antibody staining while L marks the lumen. (g) five fields from each mouse were assessed for the number of phospho-Smad2/3 positive nuclei for LRP1+/+ (n = 2 mice) and macLRP1-/- (n = 2 mice). *p&lt;0.0001, students <i>t-</i>test.</div></p></div><div class="aux"><div class="resc">Selen Catania Muratoglu, et al. PLoS One. 2011;6(12):e28846.</div><p class="links"><a class="dblinks" href="/pubmed/22174911" ref="ordinalpos=2472&amp;ncbi_uid=3714243&amp;link_uid=22174911">Citation</a><a class="dblinks" href="/pmc/articles/PMC3235159/" ref="ordinalpos=2472&amp;ncbi_uid=3714243&amp;link_uid=PMC3235159">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2473.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1522081/figure/F7/" ref="ordinalpos=2473&amp;ncbi_uid=724626&amp;link_uid=PMC1522081" href="/pmc/articles/PMC1522081/"><img src="/pmc/articles/instance/1522081/bin/1933fig7.gif" src-large="/pmc/articles/instance/1522081/bin/1933fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1522081/" ref="ordinalpos=2473&amp;ncbi_uid=724626&amp;link_uid=PMC1522081" image-link="/pmc/articles/PMC1522081/figure/F7/" class="imagepopup">Figure 7. From: The role of microtubule actin cross-linking factor 1 (MACF1) in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>A model for the involvement of MACF1 in the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) Without Wnt, MACF1 (red) binds to a protein complex containing Axin (purple), APC (yellow), β-catenin (green), and GSK3β (light blue). There is no expression of the TCF (blue) controlled genes due to the degradation of β-catenin after phosphorylation by GSK3β. (<i>B</i>) Wnt activates its receptor frizzled and its coreceptor LRP-5/6. MACF1 is involved in the translocation of the complex containing Axin, β-catenin, and GSK3β but not APC from the cytosol to the cell membrane, where Axin and MACF1 bind to LRP-5/6. Subsequently, GSK3β is inactivated by phosphorylation, Axin is degraded, and β-catenin is released and enters the nucleus, where it can activate the Wnt-responsive genes.</div></p></div><div class="aux"><div class="resc">Hui-Jye Chen, et al. Genes Dev. 2006 July 15;20(14):1933-1945.</div><p class="links"><a class="dblinks" href="/pubmed/16815997" ref="ordinalpos=2473&amp;ncbi_uid=724626&amp;link_uid=16815997">Citation</a><a class="dblinks" href="/pmc/articles/PMC1522081/" ref="ordinalpos=2473&amp;ncbi_uid=724626&amp;link_uid=PMC1522081">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2474.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3831179/figure/F15/" ref="ordinalpos=2474&amp;ncbi_uid=6329963&amp;link_uid=PMC3831179" href="/pmc/articles/PMC3831179/"><img src="/pmc/articles/instance/3831179/bin/nihms-440651-f0015.gif" src-large="/pmc/articles/instance/3831179/bin/nihms-440651-f0015.jpg" alt="Figure 15" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3831179/" ref="ordinalpos=2474&amp;ncbi_uid=6329963&amp;link_uid=PMC3831179" image-link="/pmc/articles/PMC3831179/figure/F15/" class="imagepopup">Figure 15.  From: Integrating In Silico Resources to Map a <span class="highlight" style="background-color:">Signaling</span> Network. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> Commons (accessed through Cytoscape). <b>(A)</b> BMP (bone morphogenetic protein) <span class="highlight" style="background-color:">pathway</span> – ‘File’ -&gt; ‘Import’ -&gt; ‘Network from Web Services…’ -&gt; use ‘Smad1_human’ as input -&gt; select ‘BMP receptor <span class="highlight" style="background-color:">signaling</span>’ from generated list -&gt; double click. <b>(B)</b> SMAD1-centered Sub-network – type ‘smad1_human” into toolbar search box -&gt; ‘Select’ -&gt; ‘Nodes’ -&gt; ‘First Neighbors of Selected Nodes’ -&gt; click ‘Create new network from selected nodes, all edges’ icon on toolbar. (Some nodes were manually rearranged).</div></p></div><div class="aux"><div class="resc">Hanqing Liu, et al. Methods Mol Biol. ;1101:10.1007/978-1-62703-721-1_11.</div><p class="links"><a class="dblinks" href="/pubmed/24233784" ref="ordinalpos=2474&amp;ncbi_uid=6329963&amp;link_uid=24233784">Citation</a><a class="dblinks" href="/pmc/articles/PMC3831179/" ref="ordinalpos=2474&amp;ncbi_uid=6329963&amp;link_uid=PMC3831179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2475.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3566645/figure/F7/" ref="ordinalpos=2475&amp;ncbi_uid=4605215&amp;link_uid=PMC3566645" href="/pmc/articles/PMC3566645/"><img src="/pmc/articles/instance/3566645/bin/nihms347889f7.gif" src-large="/pmc/articles/instance/3566645/bin/nihms347889f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3566645/" ref="ordinalpos=2475&amp;ncbi_uid=4605215&amp;link_uid=PMC3566645" image-link="/pmc/articles/PMC3566645/figure/F7/" class="imagepopup">Figure 7. The proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of Tat-induced nuclear localization of ZO-1 in human brain endothelial cells.  From: HIV-1 Tat triggers nuclear localization of ZO-1 via Rho <span class="highlight" style="background-color:">signaling</span> and CREB activation. </a></p><div class="supp"><p class="details"><div>HIV-1 protein Tat activates RhoA and downstream ROCK kinase, leading to phosphorylation of MLC, which then can mediate cross-linking F-actin into stress fibers and influence nuclear ZO-1 translocation. Stimulation of the Rho <span class="highlight" style="background-color:">pathway</span> results in phosphorylation of CREB, a transcriptional activator of ZO-1. In addition, exposure to HIV-1 or Tat protein stimulates matrix metalloproteinases and proteasome, which are responsible for degradation of TJ proteins, including ZO-1 (Huang et al., 2009). Nuclear translocation may protect this pool of ZO-1 against proteolytic degradation, while the membrane pool remains vulnerable to MMP and proteasome proteolysis. Because membrane ZO-1 is the major form of this TJ protein, its decrease results in diminished levels of total ZO-1. In addition to stimulation of CREB phosphorylation by the Rho <span class="highlight" style="background-color:">signaling</span>, CREB also influences activation of MLC, indicating a close cross-talk between these two <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Yu Zhong, et al. J Neurosci. ;32(1):143-150.</div><p class="links"><a class="dblinks" href="/pubmed/22219277" ref="ordinalpos=2475&amp;ncbi_uid=4605215&amp;link_uid=22219277">Citation</a><a class="dblinks" href="/pmc/articles/PMC3566645/" ref="ordinalpos=2475&amp;ncbi_uid=4605215&amp;link_uid=PMC3566645">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2476.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3252343/figure/pone-0029653-g004/" ref="ordinalpos=2476&amp;ncbi_uid=3750340&amp;link_uid=PMC3252343" href="/pmc/articles/PMC3252343/"><img src="/pmc/articles/instance/3252343/bin/pone.0029653.g004.gif" src-large="/pmc/articles/instance/3252343/bin/pone.0029653.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3252343/" ref="ordinalpos=2476&amp;ncbi_uid=3750340&amp;link_uid=PMC3252343" image-link="/pmc/articles/PMC3252343/figure/pone-0029653-g004/" class="imagepopup">Figure 4. Functional Annotation networks from IPA (Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis) that show documented gene relationships among the genes in the hypothesized eight gene STAT1/IFN gene set (A) and that show the functional relationships among these genes as they relate to Interferon <span class="highlight" style="background-color:">signaling</span> (B) (genes in eight gene signature are shaded)..  From: Expression Signature of IFN/STAT1 <span class="highlight" style="background-color:">Signaling</span> Genes Predicts Poor Survival Outcome in Glioblastoma Multiforme in a Subtype-Specific Manner. </a></p><div class="aux"><div class="resc">Christine W. Duarte, et al. PLoS One. 2012;7(1):e29653.</div><p class="links"><a class="dblinks" href="/pubmed/22242177" ref="ordinalpos=2476&amp;ncbi_uid=3750340&amp;link_uid=22242177">Citation</a><a class="dblinks" href="/pmc/articles/PMC3252343/" ref="ordinalpos=2476&amp;ncbi_uid=3750340&amp;link_uid=PMC3252343">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2477.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3009900/figure/F6/" ref="ordinalpos=2477&amp;ncbi_uid=3604249&amp;link_uid=PMC3009900" href="/pmc/articles/PMC3009900/"><img src="/pmc/articles/instance/3009900/bin/zbc0051145420006.gif" src-large="/pmc/articles/instance/3009900/bin/zbc0051145420006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3009900/" ref="ordinalpos=2477&amp;ncbi_uid=3604249&amp;link_uid=PMC3009900" image-link="/pmc/articles/PMC3009900/figure/F6/" class="imagepopup">FIGURE 6. From: Agonist-induced Localization of Gq-coupled Receptors and G Protein-gated Inwardly Rectifying K+ (GIRK) Channels to Caveolae Determines Receptor Specificity of Phosphatidylinositol 4,5-Bisphosphate <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of <span class="highlight" style="background-color:">signaling</span> microdomain for specific PIP<sub>2</sub> signal.</b> Stimulation of M2AChR/GIRK <span class="highlight" style="background-color:">pathway</span> and ET<sub>A</sub>R induce the recruitments of each component to Cav-3, which allow ET<sub>A</sub>R to interact with GIRK channels (<i>upper</i>). In contrast, B<sub>2</sub>R are dissociated from Cav-3 during B<sub>2</sub>R stimulation. By losing scaffolding network, they cannot interact with GIRK channel complexes any more (<i>lower</i>).</div></p></div><div class="aux"><div class="resc">Shanyu Cui, et al. J Biol Chem. 2010 December 31;285(53):41732-41739.</div><p class="links"><a class="dblinks" href="/pubmed/21041306" ref="ordinalpos=2477&amp;ncbi_uid=3604249&amp;link_uid=21041306">Citation</a><a class="dblinks" href="/pmc/articles/PMC3009900/" ref="ordinalpos=2477&amp;ncbi_uid=3604249&amp;link_uid=PMC3009900">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2478.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2215765/figure/f1-mol14_5p0346/" ref="ordinalpos=2478&amp;ncbi_uid=1331648&amp;link_uid=PMC2215765" href="/pmc/articles/PMC2215765/"><img src="/pmc/articles/instance/2374726/bin/mol14_5p0346f1.gif" src-large="/pmc/articles/instance/2374726/bin/mol14_5p0346f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2215765/" ref="ordinalpos=2478&amp;ncbi_uid=1331648&amp;link_uid=PMC2215765" image-link="/pmc/articles/PMC2215765/figure/f1-mol14_5p0346/" class="imagepopup">Figure 1.  From: Functional Consequences of Toll-like Receptor 4 Polymorphisms. </a></p><div class="supp"><p class="details"><div>Schematic overview of the LPS TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Recognition of LPS is established by the interplay between lipopolysaccharide-binding proteins (LBPs) and CD14 that transfer the LPS to the MD-2/TLR4 complex. Dimerization of TLR4 then initiates downstream intracellular <span class="highlight" style="background-color:">signaling</span> transduction through Toll-interleukin-1 receptor (TIR) domains and several adaptor molecules. Eventually, the intracellular <span class="highlight" style="background-color:">signaling</span> results in the activation of specific pro-inflammatory cytokine profiles by the MYD88-dependent or TRIF-dependent <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Bart Ferwerda, et al. Mol Med. 2008 May-Jun;14(5-6):346-352.</div><p class="links"><a class="dblinks" href="/pubmed/18231573" ref="ordinalpos=2478&amp;ncbi_uid=1331648&amp;link_uid=18231573">Citation</a><a class="dblinks" href="/pmc/articles/PMC2215765/" ref="ordinalpos=2478&amp;ncbi_uid=1331648&amp;link_uid=PMC2215765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2479.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3310050/figure/pone-0032920-g012/" ref="ordinalpos=2479&amp;ncbi_uid=3881586&amp;link_uid=PMC3310050" href="/pmc/articles/PMC3310050/"><img src="/pmc/articles/instance/3310050/bin/pone.0032920.g012.gif" src-large="/pmc/articles/instance/3310050/bin/pone.0032920.g012.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3310050/" ref="ordinalpos=2479&amp;ncbi_uid=3881586&amp;link_uid=PMC3310050" image-link="/pmc/articles/PMC3310050/figure/pone-0032920-g012/" class="imagepopup">Figure 12. Schematic presentation of proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by folate targeted dual drug loaded nanoparticles..  From: Folate Decorated Dual Drug Loaded Nanoparticle: Role of Curcumin in Enhancing Therapeutic Potential of Nutlin-3a by Reversing Multidrug Resistance. </a></p><div class="supp"><p class="details"><div>Folate conjugated NPs loaded with nutlin-3a (o) and curcumin (•) synergistically activates major apototic proteins or down regulates major antiapoptotic proteins involved in apoptosis <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Manasi Das, et al. PLoS One. 2012;7(3):e32920.</div><p class="links"><a class="dblinks" href="/pubmed/22470431" ref="ordinalpos=2479&amp;ncbi_uid=3881586&amp;link_uid=22470431">Citation</a><a class="dblinks" href="/pmc/articles/PMC3310050/" ref="ordinalpos=2479&amp;ncbi_uid=3881586&amp;link_uid=PMC3310050">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2480.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012266/figure/cjc-30-10-655-g002/" ref="ordinalpos=2480&amp;ncbi_uid=6135793&amp;link_uid=PMC4012266" href="/pmc/articles/PMC4012266/"><img src="/pmc/articles/instance/4012266/bin/cjc-30-10-655-g002.gif" src-large="/pmc/articles/instance/4012266/bin/cjc-30-10-655-g002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012266/" ref="ordinalpos=2480&amp;ncbi_uid=6135793&amp;link_uid=PMC4012266" image-link="/pmc/articles/PMC4012266/figure/cjc-30-10-655-g002/" class="imagepopup">Figure 2.The autophagy process and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Modulating autophagy: a strategy for cancer therapy. </a></p><div class="supp"><p class="details"><div>Autophagy occurs sequentially in phases: induction, phagophore formation, autophagosome elongation, autolysome formation, and then degradation. Cellular stresses trigger the autophagic <span class="highlight" style="background-color:">pathway</span> by several phosphorylation events within mTOR and drive Atg1-Atg13-Atg17 complex formation. Atgs, such as Vps15, Atg6, and Vps34, then accumulate in the PAS. During the autophagosome elongation phase, two conjugation systems, Atg12 and Atg8/LC, are indispensable for the formation of the autophagosome.</div></p></div><div class="aux"><div class="resc">Jun-Lin Li, et al. Chin J Cancer. 2011 October;30(10):655-668.</div><p class="links"><a class="dblinks" href="/pubmed/21959043" ref="ordinalpos=2480&amp;ncbi_uid=6135793&amp;link_uid=21959043">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012266/" ref="ordinalpos=2480&amp;ncbi_uid=6135793&amp;link_uid=PMC4012266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2481.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807212/figure/fig2/" ref="ordinalpos=2481&amp;ncbi_uid=2213195&amp;link_uid=PMC2807212" href="/pmc/articles/PMC2807212/"><img src="/pmc/articles/instance/2807212/bin/cb-2009-00249y_0007.gif" src-large="/pmc/articles/instance/2807212/bin/cb-2009-00249y_0007.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807212/" ref="ordinalpos=2481&amp;ncbi_uid=2213195&amp;link_uid=PMC2807212" image-link="/pmc/articles/PMC2807212/figure/fig2/" class="imagepopup">Figure 2.  From: Controlling Destiny through Chemistry: Small-Molecule Regulators of Cell Fate. </a></p><div class="supp"><p class="details"><div>Hh and Wnt <span class="highlight" style="background-color:">signaling</span> pathways. <span class="highlight" style="background-color:">Signaling</span> proteins associated with Hh (a) and Wnt (b) <span class="highlight" style="background-color:">pathway</span> regulation are shown. Components or processes that currently can be targeted by small molecules to achieve selective <span class="highlight" style="background-color:">pathway</span> control are labeled with green (agonist) and red (antagonist) hexagons. Direct small molecule−protein interactions are depicted when known.</div></p></div><div class="aux"><div class="resc">Ari J. Firestone, et al. ACS Chem Biol. 2010 January 15;5(1):15-34.</div><p class="links"><a class="dblinks" href="/pubmed/20000447" ref="ordinalpos=2481&amp;ncbi_uid=2213195&amp;link_uid=20000447">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807212/" ref="ordinalpos=2481&amp;ncbi_uid=2213195&amp;link_uid=PMC2807212">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2482.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2556273/figure/F8/" ref="ordinalpos=2482&amp;ncbi_uid=1526671&amp;link_uid=PMC2556273" href="/pmc/articles/PMC2556273/"><img src="/pmc/articles/instance/2556273/bin/1660fig8.gif" src-large="/pmc/articles/instance/2556273/bin/1660fig8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2556273/" ref="ordinalpos=2482&amp;ncbi_uid=1526671&amp;link_uid=PMC2556273" image-link="/pmc/articles/PMC2556273/figure/F8/" class="imagepopup">Figure 8. From: Use of shotgun proteomics for the identification, confirmation, and correction of C. elegans gene annotations. </a></p><div class="supp"><p class="details"><div>Peptides identified in the insulin/insulin-like growth factor 1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be used as proteotypic peptides in future targeted analyses. Shown here are the major proteins involved in the insulin/insulin-like growth factor 1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> along with peptides identified from the respective proteins.</div></p></div><div class="aux"><div class="resc">Gennifer E. Merrihew, et al. Genome Res. 2008 October;18(10):1660-1669.</div><p class="links"><a class="dblinks" href="/pubmed/18653799" ref="ordinalpos=2482&amp;ncbi_uid=1526671&amp;link_uid=18653799">Citation</a><a class="dblinks" href="/pmc/articles/PMC2556273/" ref="ordinalpos=2482&amp;ncbi_uid=1526671&amp;link_uid=PMC2556273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2483.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2689953/figure/fig1/" ref="ordinalpos=2483&amp;ncbi_uid=1876778&amp;link_uid=PMC2689953" href="/pmc/articles/PMC2689953/"><img src="/pmc/articles/instance/2689953/bin/pp1500951f01.gif" src-large="/pmc/articles/instance/2689953/bin/pp1500951f01.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2689953/" ref="ordinalpos=2483&amp;ncbi_uid=1876778&amp;link_uid=PMC2689953" image-link="/pmc/articles/PMC2689953/figure/fig1/" class="imagepopup">Figure 1. From: VTC4 Is a Bifunctional Enzyme That Affects Myoinositol and Ascorbate Biosynthesis in Plants. </a></p><div class="supp"><p class="details"><div>Myoinositol synthesis and metabolism <span class="highlight" style="background-color:">pathway</span>. De novo synthesis of myoinositol (i.e. the Loewus <span class="highlight" style="background-color:">pathway</span>) is catalyzed by myoinositol phosphate synthase (MIPS) and IMP, where its immediate precursor is <span style="font-variant: small-caps" class="small-caps">d</span>-Ins 3-P = <span style="font-variant: small-caps" class="small-caps">l</span>-Ins 1-P. IMP also regenerates myoinositol from the second messenger <span style="font-variant: small-caps" class="small-caps">d</span>-Ins(1,4,5)P<sub>3</sub>. Oxidation of inositol by myoinositol oxygenase (MIOX) produces <span style="font-variant: small-caps" class="small-caps">d</span>-GlcUA (<span style="font-variant: small-caps" class="small-caps">d</span>-GlcA), which is a possible entry point into ascorbate synthesis. The major route to ascorbate in plants is the Smirnoff-Wheeler <span class="highlight" style="background-color:">pathway</span> and utilizes GDP-<span style="font-variant: small-caps" class="small-caps">d</span>-Man. VTC4 has homology to the animal IMPs and has been shown to catalyze the conversion of <span style="font-variant: small-caps" class="small-caps">l</span>-Gal 1-P to <span style="font-variant: small-caps" class="small-caps">l</span>-Gal in the Smirnoff-Wheeler <span class="highlight" style="background-color:">pathway</span>. Inositol is also the precursor for the synthesis of several compounds indicated in gray. The asterisk indicates the inositol <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Javad Torabinejad, et al. Plant Physiol. 2009 June;150(2):951-961.</div><p class="links"><a class="dblinks" href="/pubmed/19339506" ref="ordinalpos=2483&amp;ncbi_uid=1876778&amp;link_uid=19339506">Citation</a><a class="dblinks" href="/pmc/articles/PMC2689953/" ref="ordinalpos=2483&amp;ncbi_uid=1876778&amp;link_uid=PMC2689953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2484.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2817017/figure/f2-0950179/" ref="ordinalpos=2484&amp;ncbi_uid=2198022&amp;link_uid=PMC2817017" href="/pmc/articles/PMC2817017/"><img src="/pmc/articles/instance/2817017/bin/95179.fig2.gif" src-large="/pmc/articles/instance/2817017/bin/95179.fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2817017/" ref="ordinalpos=2484&amp;ncbi_uid=2198022&amp;link_uid=PMC2817017" image-link="/pmc/articles/PMC2817017/figure/f2-0950179/" class="imagepopup">Figure 2. From: Molecular basis of juvenile myelomonocytic leukemia. </a></p><div class="supp"><p class="details"><div>RAS-RAF-MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Andrica C.H. de Vries, et al. Haematologica. 2010 February;95(2):179-182.</div><p class="links"><a class="dblinks" href="/pubmed/20139388" ref="ordinalpos=2484&amp;ncbi_uid=2198022&amp;link_uid=20139388">Citation</a><a class="dblinks" href="/pmc/articles/PMC2817017/" ref="ordinalpos=2484&amp;ncbi_uid=2198022&amp;link_uid=PMC2817017">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2485.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3940872/figure/pone-0090275-g006/" ref="ordinalpos=2485&amp;ncbi_uid=5895740&amp;link_uid=PMC3940872" href="/pmc/articles/PMC3940872/"><img src="/pmc/articles/instance/3940872/bin/pone.0090275.g006.gif" src-large="/pmc/articles/instance/3940872/bin/pone.0090275.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3940872/" ref="ordinalpos=2485&amp;ncbi_uid=5895740&amp;link_uid=PMC3940872" image-link="/pmc/articles/PMC3940872/figure/pone-0090275-g006/" class="imagepopup">Figure 6. α-Br-TMC inhibits the STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in both a JAK2-dependent and -independent manner..  From: The Synthetic ?-Bromo-2?,3,4,4?-Tetramethoxychalcone (?-Br-TMC) Inhibits the JAK/STAT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Ba/F3-1*6 and K562 cells were treated for 60 minutes with the indicated compounds. Protein whole cell extracts were analyzed by Western-Blot using antibodies specific for pSTAT5, STAT5A, STAT5B, STAT5A and B and α-tubulin as a loading control. (<b>B</b>) Ba/F3 cells were pre-treated 30 minutes with 10 µM α-Br-TMC and stimulated with 5 ng/mL IL-3 for 5 minutes. Ba/F3-1*6 and K562 cells were treated with 10 µM α-Br-TMC for 90 minutes. DMSO (Veh.) was adjusted to 0.01% in all conditions. Protein whole cell extracts were analyzed by Western-Blot using antibodies specific for pSTAT5, pJAK2, STAT5A, STAT5B, STAT5A and B, JAK2 and α-tubulin (loading control).</div></p></div><div class="aux"><div class="resc">Sophia Pinz, et al. PLoS One. 2014;9(3):e90275.</div><p class="links"><a class="dblinks" href="/pubmed/24595334" ref="ordinalpos=2485&amp;ncbi_uid=5895740&amp;link_uid=24595334">Citation</a><a class="dblinks" href="/pmc/articles/PMC3940872/" ref="ordinalpos=2485&amp;ncbi_uid=5895740&amp;link_uid=PMC3940872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2486.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2647780/figure/pcbi-1000287-g001/" ref="ordinalpos=2486&amp;ncbi_uid=1701566&amp;link_uid=PMC2647780" href="/pmc/articles/PMC2647780/"><img src="/pmc/articles/instance/2647780/bin/pcbi.1000287.g001.gif" src-large="/pmc/articles/instance/2647780/bin/pcbi.1000287.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2647780/" ref="ordinalpos=2486&amp;ncbi_uid=1701566&amp;link_uid=PMC2647780" image-link="/pmc/articles/PMC2647780/figure/pcbi-1000287-g001/" class="imagepopup">Figure 1. Block diagram of <span class="highlight" style="background-color:">signaling</span> network..  From: <span class="highlight" style="background-color:">Signaling</span> Logic of Activity-Triggered Dendritic Protein Synthesis: An mTOR Gate But Not a Feedback Switch. </a></p><div class="supp"><p class="details"><div>Inputs are located at the top. The primary input to the <span class="highlight" style="background-color:">pathway</span> is BDNF. We use Ca<sup>2+</sup> influx as a surrogate for glutamate input, and AKT activation as equivalent to input from the mGluR5 <span class="highlight" style="background-color:">pathway</span>, since it is still incompletely understood. Dashed arrows represent such virtual inputs. Regular arrows represent positive interactions and T-s represent inhibitory inputs. The various modules in the model are represented as gray blocks, and their names are in rotated text.</div></p></div><div class="aux"><div class="resc">Pragati Jain, et al. PLoS Comput Biol. 2009 February;5(2):e1000287.</div><p class="links"><a class="dblinks" href="/pubmed/19242559" ref="ordinalpos=2486&amp;ncbi_uid=1701566&amp;link_uid=19242559">Citation</a><a class="dblinks" href="/pmc/articles/PMC2647780/" ref="ordinalpos=2486&amp;ncbi_uid=1701566&amp;link_uid=PMC2647780">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2487.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC545831/figure/fig8/" ref="ordinalpos=2487&amp;ncbi_uid=460607&amp;link_uid=PMC545831" href="/pmc/articles/PMC545831/"><img src="/pmc/articles/instance/545831/bin/zpq0040570230008.gif" src-large="/pmc/articles/instance/545831/bin/zpq0040570230008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC545831/" ref="ordinalpos=2487&amp;ncbi_uid=460607&amp;link_uid=PMC545831" image-link="/pmc/articles/PMC545831/figure/fig8/" class="imagepopup">Fig. 8. From: Glutamate regulation of DARPP-32 phosphorylation in neostriatal neurons involves activation of multiple <span class="highlight" style="background-color:">signaling</span> cascades. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> cascades activated by glutamate NMDA/AMPA and mGlu receptors. Glutamate activates multiple <span class="highlight" style="background-color:">signaling</span> cascades by activation of NMDA/AMPA and mGlu1/5 receptors (<i>A</i>), leading to the modulation of DARPP-32 phosphorylation at Thr-34 (<i>B</i>) and Thr-75 (<i>C</i>) and the regulation of PP-1 and PKA activities. Pathways leading to increased phospho-Thr-34 DARPP-32 and increased PP-1 inhibition are shown in blue, and those leading to decreased phospho-Thr-34 DARPP-32 and decreased PP-1 inhibition are shown in red. Inhibition of PP-1 by phospho-Thr-34 DARPP-32 and inhibition of PKA by phospho-Thr-75 DARPP-32 are shown in orange. (<i>B</i>) In the initial 15–30 s of incubation, activation of NMDA/AMPA and mGlu5 receptors in nNOS-positive interneurons stimulates the synthesis and release of NO, leading to activation in medium spiny neurons of PKG and DARPP-32 Thr-34 phosphorylation. At 1–5 min of incubation, activation of NMDA/AMPA receptors induces Ca<sup>2+</sup>-dependent activation of PP-2B, resulting in the dephosphorylation of Thr-34. At 5–10 min of incubation, there is a gradual increase in DARPP-32 Thr-34 phosphorylation. This increase is due, in part, to activation of a <span class="highlight" style="background-color:">pathway</span> involving mGlu5 receptor/PLC/CK1/phospho-Ser-130 DARPP-32, resulting in inhibition of dephosphorylation of phospho-Thr-34 by PP-2B, and, in part, to activation of <span class="highlight" style="background-color:">signaling</span> through the adenosine A<sub>2A</sub> receptor <span class="highlight" style="background-color:">pathway</span>. A synergistic activation of adenosine A<sub>2A</sub> receptor <span class="highlight" style="background-color:">signaling</span> involves (<i>i</i>) activation of an mGlu5 receptor/ERK <span class="highlight" style="background-color:">pathway</span> and, through some yet unknown mechanism, increased efficiency of A<sub>2A</sub> receptor-induced cAMP formation, and (<i>ii</i>) increased extracellular adenosine levels arising from both medium spiny neurons and presynaptic terminals (15). (<i>C</i>) In the initial 5 min of incubation, activation of NMDA/AMPA receptors induces Ca<sup>2+</sup>-dependent activation of PP-2A, resulting in the dephosphorylation of Thr-75. Subsequently, activation of a <span class="highlight" style="background-color:">pathway</span> involving mGlu1 receptor/PLC/CK1/Cdk5 increases the phosphorylation of Thr-75.</div></p></div><div class="aux"><div class="resc">Akinori Nishi, et al. Proc Natl Acad Sci U S A. 2005 January 25;102(4):1199-1204.</div><p class="links"><a class="dblinks" href="/pubmed/15657149" ref="ordinalpos=2487&amp;ncbi_uid=460607&amp;link_uid=15657149">Citation</a><a class="dblinks" href="/pmc/articles/PMC545831/" ref="ordinalpos=2487&amp;ncbi_uid=460607&amp;link_uid=PMC545831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2488.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3503165/figure/AS-fig01/" ref="ordinalpos=2488&amp;ncbi_uid=4890411&amp;link_uid=PMC3503165" href="/pmc/articles/PMC3503165/"><img src="/pmc/articles/instance/3503165/bin/pnas.1206004S109fig01.gif" src-large="/pmc/articles/instance/3503165/bin/pnas.1206004S109fig01.jpg" alt="Fig. P1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3503165/" ref="ordinalpos=2488&amp;ncbi_uid=4890411&amp;link_uid=PMC3503165" image-link="/pmc/articles/PMC3503165/figure/AS-fig01/" class="imagepopup">Fig. P1. From: CDX2 is an amplified lineage-survival oncogene in colorectal cancer. </a></p><div class="supp"><p class="details"><div><i>CDX2</i> is frequently amplified in colorectal cancer, where it promotes Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. (<i>Upper</i>) Plot of significant DNA amplifications in colorectal cancers reveals amplification on chromosome 13 pinpointing <i>CDX2</i>. (<i>Lower</i>) Schematic model summarizing data indicating that <i>CDX2</i>, when amplified, promotes Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> by regulating transcript levels of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes (<i>Left</i>) and by enhancing expression of Wnt <span class="highlight" style="background-color:">pathway</span> target genes by β-catenin (<i>Right</i>).</div></p></div><div class="aux"><div class="resc">Keyan Salari, et al. Proc Natl Acad Sci U S A. 2012 November 13;109(46):E3196-E3205.</div><p class="links"><a class="dblinks" href="/pubmed/23112155" ref="ordinalpos=2488&amp;ncbi_uid=4890411&amp;link_uid=23112155">Citation</a><a class="dblinks" href="/pmc/articles/PMC3503165/" ref="ordinalpos=2488&amp;ncbi_uid=4890411&amp;link_uid=PMC3503165">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2489.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3028588/figure/F2/" ref="ordinalpos=2489&amp;ncbi_uid=2992221&amp;link_uid=PMC3028588" href="/pmc/articles/PMC3028588/"><img src="/pmc/articles/instance/3028588/bin/nihms260952f2.gif" src-large="/pmc/articles/instance/3028588/bin/nihms260952f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3028588/" ref="ordinalpos=2489&amp;ncbi_uid=2992221&amp;link_uid=PMC3028588" image-link="/pmc/articles/PMC3028588/figure/F2/" class="imagepopup">Figure 2. Oncogenic kinases for cancer therapy.  From: Validating cancer drug targets through chemical genetics. </a></p><div class="supp"><p class="details"><div>Receptor tyrosine kinases (RTKs) are transmembrane proteins that dimerize when bound to ligand. EGFR and ErbB2/Her2 are examples of oncogenic RTKs. Downstream mitogenic <span class="highlight" style="background-color:">signaling</span> cascades include the canonical Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the PI3K <span class="highlight" style="background-color:">pathway</span>. These protein kinases typically phosphorylate serine/threonine residues. Here, mitogenic <span class="highlight" style="background-color:">signaling</span> protein kinases are boxed. Oncogenic mutations have been found in AKT and Raf among the protein kinases.</div></p></div><div class="aux"><div class="resc">Mark E. Burkard, et al. Biochim Biophys Acta. ;1806(2):251-257.</div><p class="links"><a class="dblinks" href="/pubmed/20708654" ref="ordinalpos=2489&amp;ncbi_uid=2992221&amp;link_uid=20708654">Citation</a><a class="dblinks" href="/pmc/articles/PMC3028588/" ref="ordinalpos=2489&amp;ncbi_uid=2992221&amp;link_uid=PMC3028588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2490.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1592896/figure/f7/" ref="ordinalpos=2490&amp;ncbi_uid=850434&amp;link_uid=PMC1592896" href="/pmc/articles/PMC1592896/"><img src="/pmc/articles/instance/1592896/bin/zmb0190662410007.gif" src-large="/pmc/articles/instance/1592896/bin/zmb0190662410007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1592896/" ref="ordinalpos=2490&amp;ncbi_uid=850434&amp;link_uid=PMC1592896" image-link="/pmc/articles/PMC1592896/figure/f7/" class="imagepopup">FIG. 7. From: DRONC Coordinates Cell Death and Compensatory Proliferation . </a></p><div class="supp"><p class="details"><div>Schematic diagram showing the proposed <span class="highlight" style="background-color:">signaling</span> cascade that regulates apoptosis and compensatory proliferation. First, apoptotic stimuli given to the cell activates RHG proteins. RHG proteins activate DRONC through inhibition of DIAP1 function. The <span class="highlight" style="background-color:">signaling</span> cascade bifurcates at DRONC, one <span class="highlight" style="background-color:">pathway</span> leading to apoptosis through cleavage of DRICE by DRONC. The other <span class="highlight" style="background-color:">pathway</span> leads to compensatory proliferation through cleavage of an as-yet-unknown factor (denoted as “X”) by DRONC. Cleavage of factor “X” activates JNK <span class="highlight" style="background-color:">signaling</span>, which induces compensatory proliferation through transcriptional upregulation of mitogens including DPP.</div></p></div><div class="aux"><div class="resc">Shu Kondo, et al. Mol Cell Biol. 2006 October;26(19):7258-7268.</div><p class="links"><a class="dblinks" href="/pubmed/16980627" ref="ordinalpos=2490&amp;ncbi_uid=850434&amp;link_uid=16980627">Citation</a><a class="dblinks" href="/pmc/articles/PMC1592896/" ref="ordinalpos=2490&amp;ncbi_uid=850434&amp;link_uid=PMC1592896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2491.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3983080/figure/pcbi-1003546-g001/" ref="ordinalpos=2491&amp;ncbi_uid=6023055&amp;link_uid=PMC3983080" href="/pmc/articles/PMC3983080/"><img src="/pmc/articles/instance/3983080/bin/pcbi.1003546.g001.gif" src-large="/pmc/articles/instance/3983080/bin/pcbi.1003546.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3983080/" ref="ordinalpos=2491&amp;ncbi_uid=6023055&amp;link_uid=PMC3983080" image-link="/pmc/articles/PMC3983080/figure/pcbi-1003546-g001/" class="imagepopup">Figure 1. Generalized illustration of the TCR-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through the Erk-MAPK cascade..  From: Multiple Model-Informed Open-Loop Control of Uncertain Intracellular <span class="highlight" style="background-color:">Signaling</span> Dynamics. </a></p><div class="supp"><p class="details"><div>Arrow- and diamond-heads denote activation and inhibition of substrate molecules, respectively. TCR stimulation is achieved through αCD3 (green) binding. Subsequent Erk activation (black) controlled using small molecule inhibitors sanguinarine (blue) and U0126 (red).</div></p></div><div class="aux"><div class="resc">Jeffrey P. Perley, et al. PLoS Comput Biol. 2014 April;10(4):e1003546.</div><p class="links"><a class="dblinks" href="/pubmed/24722333" ref="ordinalpos=2491&amp;ncbi_uid=6023055&amp;link_uid=24722333">Citation</a><a class="dblinks" href="/pmc/articles/PMC3983080/" ref="ordinalpos=2491&amp;ncbi_uid=6023055&amp;link_uid=PMC3983080">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2492.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2409381/figure/F6/" ref="ordinalpos=2492&amp;ncbi_uid=1450137&amp;link_uid=PMC2409381" href="/pmc/articles/PMC2409381/"><img src="/pmc/articles/instance/2409381/bin/zpq0170803920006.gif" src-large="/pmc/articles/instance/2409381/bin/zpq0170803920006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2409381/" ref="ordinalpos=2492&amp;ncbi_uid=1450137&amp;link_uid=PMC2409381" image-link="/pmc/articles/PMC2409381/figure/F6/" class="imagepopup">Fig. 6. From: Control of endothelial cell proliferation and migration by VEGF <span class="highlight" style="background-color:">signaling</span> to histone deacetylase 7. </a></p><div class="supp"><p class="details"><div>A model for the control of EC gene expression by VEGF <span class="highlight" style="background-color:">signaling</span> to HDAC7. In this model, VEGF induces phosphorylation and nuclear export of HDAC7 via a PKC/PKD1 <span class="highlight" style="background-color:">pathway</span>. HDAC7 controls VEGF-induced EC proliferation and migration by regulating VEGF responsive gene expression.</div></p></div><div class="aux"><div class="resc">Shusheng Wang, et al. Proc Natl Acad Sci U S A. 2008 June 3;105(22):7738-7743.</div><p class="links"><a class="dblinks" href="/pubmed/18509061" ref="ordinalpos=2492&amp;ncbi_uid=1450137&amp;link_uid=18509061">Citation</a><a class="dblinks" href="/pmc/articles/PMC2409381/" ref="ordinalpos=2492&amp;ncbi_uid=1450137&amp;link_uid=PMC2409381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2493.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2195651/figure/F2/" ref="ordinalpos=2493&amp;ncbi_uid=1230827&amp;link_uid=PMC2195651" href="/pmc/articles/PMC2195651/"><img src="/pmc/articles/instance/2195651/bin/JEM991494.f2.gif" src-large="/pmc/articles/instance/2195651/bin/JEM991494.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2195651/" ref="ordinalpos=2493&amp;ncbi_uid=1230827&amp;link_uid=PMC2195651" image-link="/pmc/articles/PMC2195651/figure/F2/" class="imagepopup">Figure 2.  From: Inhibition of Death Receptor <span class="highlight" style="background-color:">Signaling</span> by Flice-Inhibitory Protein as a Mechanism for Immune Escape of Tumors. </a></p><div class="supp"><p class="details"><div>Inhibition of death receptor <span class="highlight" style="background-color:">signaling</span> as a mechanism of tumor immune escape. CTLs use two major pathways (perforin/granzyme granule exocytosis and Fas) to induce target cell (tumor) death by apoptosis after TCR–MHC–peptide complex engagement. By downregulating Fas surface expression or by producing FLIP, tumor cells can escape immune destruction mediated by the Fas death receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in vivo, despite the persistence of a functional perforin/granzyme granule exocytosis <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Lars E. French, et al. J Exp Med. 1999 October 4;190(7):891-894.</div><p class="links"><a class="dblinks" href="/pubmed/10510078" ref="ordinalpos=2493&amp;ncbi_uid=1230827&amp;link_uid=10510078">Citation</a><a class="dblinks" href="/pmc/articles/PMC2195651/" ref="ordinalpos=2493&amp;ncbi_uid=1230827&amp;link_uid=PMC2195651">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2494.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2754635/figure/fig7/" ref="ordinalpos=2494&amp;ncbi_uid=2135239&amp;link_uid=PMC2754635" href="/pmc/articles/PMC2754635/"><img src="/pmc/articles/instance/2754635/bin/pp1510669f07.gif" src-large="/pmc/articles/instance/2754635/bin/pp1510669f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2754635/" ref="ordinalpos=2494&amp;ncbi_uid=2135239&amp;link_uid=PMC2754635" image-link="/pmc/articles/PMC2754635/figure/fig7/" class="imagepopup">Figure 7. From: BRASSINOSTEROID UPREGULATED1, Encoding a Helix-Loop-Helix Protein, Is a Novel Gene Involved in Brassinosteroid <span class="highlight" style="background-color:">Signaling</span> and Controls Bending of the Lamina Joint in Rice. </a></p><div class="supp"><p class="details"><div>BR <span class="highlight" style="background-color:">signaling</span> model in rice. Induction of <i>BU1</i> by BL is partially inhibited in <i>OsBRI1</i> mutant (<i>d61</i>-<i>2</i>) and <i>RGA1</i> mutant (<i>d1</i>). In addition, <i>BU1</i> is the primary response (up-regulated) gene to BR <span class="highlight" style="background-color:">signaling</span>. Therefore, BU1 may be a novel mediator of a distinct <span class="highlight" style="background-color:">pathway</span> from OsBZR1, because OsBZR1 functions as the transcription repressor. Moreover, regulation of downstream genes by BU1 may be through a complex mechanism like the regulation of human Id protein.</div></p></div><div class="aux"><div class="resc">Atsunori Tanaka, et al. Plant Physiol. 2009 October;151(2):669-680.</div><p class="links"><a class="dblinks" href="/pubmed/19648232" ref="ordinalpos=2494&amp;ncbi_uid=2135239&amp;link_uid=19648232">Citation</a><a class="dblinks" href="/pmc/articles/PMC2754635/" ref="ordinalpos=2494&amp;ncbi_uid=2135239&amp;link_uid=PMC2754635">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2495.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC155330/figure/F7/" ref="ordinalpos=2495&amp;ncbi_uid=310148&amp;link_uid=PMC155330" href="/pmc/articles/PMC155330/"><img src="/pmc/articles/instance/155330/bin/44493-2f7_L1TT.gif" src-large="/pmc/articles/instance/155330/bin/44493-2f7_L1TT.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC155330/" ref="ordinalpos=2495&amp;ncbi_uid=310148&amp;link_uid=PMC155330" image-link="/pmc/articles/PMC155330/figure/F7/" class="imagepopup">Figure 7.  From: Activation of the JNK <span class="highlight" style="background-color:">pathway</span> during dorsal closure in Drosophila requires the mixed lineage kinase, slipper. </a></p><div class="supp"><p class="details"><div><i>slpr</i>, encoding <i>Drosophila</i> MLK, is the MKKK for the Jun kinase <span class="highlight" style="background-color:">pathway</span> during dorsal closure in the fly. A canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is indicated at <i>left</i>. Examples of mammalian proteins participating at each step are indicated at <i>right</i>. In <i>Drosophila</i>, dorsal closure involves <span class="highlight" style="background-color:">signaling</span> through the JNK <span class="highlight" style="background-color:">pathway</span> to control gene expression and cell sheet morphogenesis. Although the initiating signal is not known, it is likely that an early event is activation of dRac1, which by analogy to signal transduction with mammalian MKKKKs, participates with Msn to activate Slpr, the JNKKK. Sequential phosphorylation events through the kinase module, Slpr, Hep, and Bsk, lead to activation of dJun, regulation of gene expression, and cell sheet movement.</div></p></div><div class="aux"><div class="resc">Beth Stronach, et al. Genes Dev. 2002 February 1;16(3):377-387.</div><p class="links"><a class="dblinks" href="/pubmed/11825878" ref="ordinalpos=2495&amp;ncbi_uid=310148&amp;link_uid=11825878">Citation</a><a class="dblinks" href="/pmc/articles/PMC155330/" ref="ordinalpos=2495&amp;ncbi_uid=310148&amp;link_uid=PMC155330">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2496.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3212439/figure/F5/" ref="ordinalpos=2496&amp;ncbi_uid=3514002&amp;link_uid=PMC3212439" href="/pmc/articles/PMC3212439/"><img src="/pmc/articles/instance/3212439/bin/nihms-296577-f0007.gif" src-large="/pmc/articles/instance/3212439/bin/nihms-296577-f0007.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3212439/" ref="ordinalpos=2496&amp;ncbi_uid=3514002&amp;link_uid=PMC3212439" image-link="/pmc/articles/PMC3212439/figure/F5/" class="imagepopup">Figure 5. Two pathways for the LPS-induced cytokines in macrophages.  From: The Immunoproteasomes Regulate Lps-Induced Trif/Tram <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Murine Macrophages. </a></p><div class="supp"><p class="details"><div>LPS-induced cytokines are triggered by TLR4 and are dependent in part on the MyD88 <span class="highlight" style="background-color:">pathway</span> and/or the MyD88-independent TRIF <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Julia Reis, et al. Cell Biochem Biophys. ;60(1-2):119-126.</div><p class="links"><a class="dblinks" href="/pubmed/21455681" ref="ordinalpos=2496&amp;ncbi_uid=3514002&amp;link_uid=21455681">Citation</a><a class="dblinks" href="/pmc/articles/PMC3212439/" ref="ordinalpos=2496&amp;ncbi_uid=3514002&amp;link_uid=PMC3212439">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2497.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3013142/figure/pone-0015907-g003/" ref="ordinalpos=2497&amp;ncbi_uid=2700207&amp;link_uid=PMC3013142" href="/pmc/articles/PMC3013142/"><img src="/pmc/articles/instance/3013142/bin/pone.0015907.g003.gif" src-large="/pmc/articles/instance/3013142/bin/pone.0015907.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3013142/" ref="ordinalpos=2497&amp;ncbi_uid=2700207&amp;link_uid=PMC3013142" image-link="/pmc/articles/PMC3013142/figure/pone-0015907-g003/" class="imagepopup">Figure 3. Distinct <span class="highlight" style="background-color:">signaling</span> pathways regulate activation of pou3f4 HCNR 81675 and HCNR 82478 enhancers..  From: Characterization of New Otic Enhancers of the Pou3f4 Gene Reveal Distinct <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulation and Spatio-Temporal Patterns. </a></p><div class="supp"><p class="details"><div>(A–N) Transgenic embryos for both enhancers were treated with different pharmacological inhibitors from 5.5 hpf stage to 18–20 hpf and 7.5 hpf to 36–40 hpf respectively. Lateral view of HCNR 81675 (A–G) 18–20 hpf staged otic vesicles and HCNR 82478 (F–N) otic vesicles of 36–40 hpf embryos. HCNR 81675 activity was abrogated in the presence of RA <span class="highlight" style="background-color:">signaling</span> inhibitor DEAB (compare D to the control treatment with DMSO in A), whereas Fgf <span class="highlight" style="background-color:">signaling</span> inhibition by SU5402 completely disrupted <i>pou3f4</i> HCNR 82478 activity (compare I to control treatment in H). Orientation is anterior (left) to posterior (right). (O, P) Graphs representing the percentage of embryos displaying complete inhibition of GFP expression in <i>pou3f4</i> HCNR 81675 (O) and HCNR 82478 (P) transgenic embryos after specific <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blockade. The total number of embryos counted in three independent experiments is represented.</div></p></div><div class="aux"><div class="resc">Àlex Robert-Moreno, et al. PLoS One. 2010;5(12):e15907.</div><p class="links"><a class="dblinks" href="/pubmed/21209840" ref="ordinalpos=2497&amp;ncbi_uid=2700207&amp;link_uid=21209840">Citation</a><a class="dblinks" href="/pmc/articles/PMC3013142/" ref="ordinalpos=2497&amp;ncbi_uid=2700207&amp;link_uid=PMC3013142">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2498.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3382585/figure/pone-0039738-g008/" ref="ordinalpos=2498&amp;ncbi_uid=4103170&amp;link_uid=PMC3382585" href="/pmc/articles/PMC3382585/"><img src="/pmc/articles/instance/3382585/bin/pone.0039738.g008.gif" src-large="/pmc/articles/instance/3382585/bin/pone.0039738.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3382585/" ref="ordinalpos=2498&amp;ncbi_uid=4103170&amp;link_uid=PMC3382585" image-link="/pmc/articles/PMC3382585/figure/pone-0039738-g008/" class="imagepopup">Figure 8. NF-κB <span class="highlight" style="background-color:">signaling</span> is critical for mediating RANTES expression in tubular cells..  From: Uric Acid Induces Renal Inflammation via Activating Tubular NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>NRK-52E cells were treated with 200μmol/L of uric acid in the absence or presence of NF-κB inhibitor (NF-κB SN50). (A) Immunofluorescence staining demonstrated that incubation with NF-κB SN50 inhibited nuclear translocation of p65 NF-κB upon uric acid stimulation in NRK-52E cells. (b) Inhibition of NF-κB <span class="highlight" style="background-color:">signaling</span> abrogated RANTES expression stimulated by uric acid in NRK-52E cells.</div></p></div><div class="aux"><div class="resc">Yang Zhou, et al. PLoS One. 2012;7(6):e39738.</div><p class="links"><a class="dblinks" href="/pubmed/22761883" ref="ordinalpos=2498&amp;ncbi_uid=4103170&amp;link_uid=22761883">Citation</a><a class="dblinks" href="/pmc/articles/PMC3382585/" ref="ordinalpos=2498&amp;ncbi_uid=4103170&amp;link_uid=PMC3382585">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2499.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4091082/figure/F5/" ref="ordinalpos=2499&amp;ncbi_uid=6391217&amp;link_uid=PMC4091082" href="/pmc/articles/PMC4091082/"><img src="/pmc/articles/instance/4091082/bin/psb-8-e26752-g5.gif" src-large="/pmc/articles/instance/4091082/bin/psb-8-e26752-g5.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4091082/" ref="ordinalpos=2499&amp;ncbi_uid=6391217&amp;link_uid=PMC4091082" image-link="/pmc/articles/PMC4091082/figure/F5/" class="imagepopup">Figure. From: Salicylic acid induces vanillin synthesis through the phospholipid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in &lt;i&gt;Capsicum chinense &lt;/i&gt;cell cultures. </a></p><div class="supp"><p class="details"><div><b>Figure 5.</b> The conceptual model for salicylic acid role on vanillin synthesis involving phospholipid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(i)</b> Salicylic acid can be sensed on or near the plasma membrane by a receptor and activate a <span class="highlight" style="background-color:">signaling</span> cascade through phospholipases (PLC and/or PLD) function. This is followed by the regulation of PAL enzymatic activity and increased vanillin content. <b>(ii)</b> In the presence of U73122 or neomycin (inhibitors of PLC <span class="highlight" style="background-color:">signaling</span>) the levels of DAG and Ins(1,4,5)<i>P</i><sub>3</sub> (second messengers), reduced which lead to modifications of intracellular Ca<sup>2+</sup> levels that may affect the activity of PAL as well as the promoter of reduced vanillin production. <b>(iii)</b> The inhibitory function of 1-butanol on formation of PA could be affecting phosphorylation processes through the regulation of protein kinases activities that may be responding to PA levels. This event also could be affecting PAL activity following vanillin production. Therefore, response to SA resulting in the production of second messengers such as DAG, Ins(1,4,5)<i>P</i><sub>3</sub>, and PA produced in phospholipid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may be involved in regulating of PAL activity, and consequently vanillin production.</div></p></div><div class="aux"><div class="resc">Beatriz A Rodas-Junco, et al. Plant Signal Behav. 2013 October;8(10):e26752.</div><p class="links"><a class="dblinks" href="/pubmed/24169294" ref="ordinalpos=2499&amp;ncbi_uid=6391217&amp;link_uid=24169294">Citation</a><a class="dblinks" href="/pmc/articles/PMC4091082/" ref="ordinalpos=2499&amp;ncbi_uid=6391217&amp;link_uid=PMC4091082">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2500.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3086304/figure/f1-bcbcr-2009-047/" ref="ordinalpos=2500&amp;ncbi_uid=2938660&amp;link_uid=PMC3086304" href="/pmc/articles/PMC3086304/"><img src="/pmc/articles/instance/3086304/bin/bcbcr-2009-047f1.gif" src-large="/pmc/articles/instance/3086304/bin/bcbcr-2009-047f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3086304/" ref="ordinalpos=2500&amp;ncbi_uid=2938660&amp;link_uid=PMC3086304" image-link="/pmc/articles/PMC3086304/figure/f1-bcbcr-2009-047/" class="imagepopup">Figure 1. From: Growth Factor Receptors and Apoptosis Regulators: <span class="highlight" style="background-color:">Signaling</span> Pathways, Prognosis, Chemosensitivity and Treatment Outcomes of Breast Cancer. </a></p><div class="supp"><p class="details"><div>Homology of ERα and ERβ <b>a</b>) Estrogen receptor and cross-talk <span class="highlight" style="background-color:">signaling</span> with RTK in breast cancer. <b>b</b>) Estrogen leads to transcription of cell proliferation gene via classical <span class="highlight" style="background-color:">pathway</span> (black arrow). Cross talk and bidirectional (gray arrows) <span class="highlight" style="background-color:">signaling</span> between mER, RTK, mER and downstream phosphorylation of Ser118 and 167 by MAPK/ERK and PI3-K/Akt <span class="highlight" style="background-color:">pathway</span> respectively leads to activation of estrogen regulated genes. GSK-3β inhibits (blocking red arrows) phosphorylation of Ser118 which is further inactivated by PI3-K/AKT.<br /><b>Abbreviations:</b> DBD, DNA binding domain; NLS, Nuclear localization sequence.</div></p></div><div class="aux"><div class="resc">Siddik Sarkar, et al. Breast Cancer (Auckl). 2009;3:47-60.</div><p class="links"><a class="dblinks" href="/pubmed/21556249" ref="ordinalpos=2500&amp;ncbi_uid=2938660&amp;link_uid=21556249">Citation</a><a class="dblinks" href="/pmc/articles/PMC3086304/" ref="ordinalpos=2500&amp;ncbi_uid=2938660&amp;link_uid=PMC3086304">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="24" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="25" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="26" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:34:50-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC55A0543105E100000000001AD71C;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC55A0543105E100000000001AD71C" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>