
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="48" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB45F454313CD100000000006D2212" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="47" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="48" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="49" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="48" /></div>
            <div><h2 class="result_count">Results: 4701 to 4800 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4701.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3568388/figure/F7/" ref="ordinalpos=4701&amp;ncbi_uid=4616169&amp;link_uid=PMC3568388" href="/pmc/articles/PMC3568388/"><img src="/pmc/articles/instance/3568388/bin/nihms159497f7.gif" src-large="/pmc/articles/instance/3568388/bin/nihms159497f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3568388/" ref="ordinalpos=4701&amp;ncbi_uid=4616169&amp;link_uid=PMC3568388" image-link="/pmc/articles/PMC3568388/figure/F7/" class="imagepopup">Figure 7. Neurochemical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> organizing male rat sexual behavior and neuroanatomical correlates.  From: PGE2-induced masculinization of brain and behavior requires protein kinase A, AMPA/kainate and metabotropic glutamate receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Masculinization of male rat sexual behavior is initiated perinatally when testosterone is converted to estradiol by aromatase in the diencephalon. Estradiol binds to estrogen receptor-α, which increases COX-1 and COX-2 levels two-fold in the POA. A subsequent seven-fold increase in PGE<sub>2</sub> activates EP2 and EP4 receptors and recruits PKA <span class="highlight" style="background-color:">signaling</span>. PKA activity may induce AMPA/kainate and metabotropic glutamate receptor <span class="highlight" style="background-color:">signaling</span> which then concurrently initiates dendritic spine formation in the POA. A higher density of excitatory dendritic spines synapses on POA neurons is highly correlated with expression of adult male sexual behavior in individual animals (Wright et al., 2008).</div></p></div><div class="aux"><div class="resc">Christopher L. Wright, et al. J Neurosci. ;29(42):13274-13282.</div><p class="links"><a class="dblinks" href="/pubmed/19846715" ref="ordinalpos=4701&amp;ncbi_uid=4616169&amp;link_uid=19846715">Citation</a><a class="dblinks" href="/pmc/articles/PMC3568388/" ref="ordinalpos=4701&amp;ncbi_uid=4616169&amp;link_uid=PMC3568388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4702.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4057583/figure/F10/" ref="ordinalpos=4702&amp;ncbi_uid=6250621&amp;link_uid=PMC4057583" href="/pmc/articles/PMC4057583/"><img src="/pmc/articles/instance/4057583/bin/1755-8794-7-30-10.gif" src-large="/pmc/articles/instance/4057583/bin/1755-8794-7-30-10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4057583/" ref="ordinalpos=4702&amp;ncbi_uid=6250621&amp;link_uid=PMC4057583" image-link="/pmc/articles/PMC4057583/figure/F10/" class="imagepopup">Figure 10.  From: Imaging genomic mapping of an invasive MRI phenotype predicts patient outcome and metabolic dysfunction: a TCGA glioma phenotype research group project. </a></p><div class="supp"><p class="details"><div><b>Link between NFKBIA and EGFR <span class="highlight" style="background-color:">pathway</span> in Glioblastoma.</b> The relationship between EGFR and NF-kB <span class="highlight" style="background-color:">signaling</span> was performed using IPA Path Designer tool. Solid line indicates direct biological relationship. Dashed line indicates indirect biological relationship. The relationship between molecules were supported by IPA knowledge base. EGF <span class="highlight" style="background-color:">signaling</span> to NF-kB is affected by NFKBIA inhibition.</div></p></div><div class="aux"><div class="resc">Rivka R Colen, et al. BMC Med Genomics. 2014;7:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/24889866" ref="ordinalpos=4702&amp;ncbi_uid=6250621&amp;link_uid=24889866">Citation</a><a class="dblinks" href="/pmc/articles/PMC4057583/" ref="ordinalpos=4702&amp;ncbi_uid=6250621&amp;link_uid=PMC4057583">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4703.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3615581/figure/F2/" ref="ordinalpos=4703&amp;ncbi_uid=5916257&amp;link_uid=PMC3615581" href="/pmc/articles/PMC3615581/"><img src="/pmc/articles/instance/3615581/bin/zh70061338130002.gif" src-large="/pmc/articles/instance/3615581/bin/zh70061338130002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3615581/" ref="ordinalpos=4703&amp;ncbi_uid=5916257&amp;link_uid=PMC3615581" image-link="/pmc/articles/PMC3615581/figure/F2/" class="imagepopup">Fig. 2. From: Gene expression profiling of gastrocnemius of "minimuscle" mice. </a></p><div class="supp"><p class="details"><div>Differential expression of genes associated with calcium <span class="highlight" style="background-color:">signaling</span>. Prominent genes of the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis network “calcium <span class="highlight" style="background-color:">signaling</span>” are shown relative to their cellular location. The expression of genes colored red was significantly greater in HR<sub>mini</sub>, whereas the expression of genes colored green was greater in HR<sub>normal</sub> (<i>P</i> &lt; 0.05, after correction for multiple testing).</div></p></div><div class="aux"><div class="resc">Jatin G. Burniston, et al. Physiol Genomics. 2013 March 15;45(6):228-236.</div><p class="links"><a class="dblinks" href="/pubmed/23362141" ref="ordinalpos=4703&amp;ncbi_uid=5916257&amp;link_uid=23362141">Citation</a><a class="dblinks" href="/pmc/articles/PMC3615581/" ref="ordinalpos=4703&amp;ncbi_uid=5916257&amp;link_uid=PMC3615581">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4704.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3411131/figure/fig3-1947601911432334/" ref="ordinalpos=4704&amp;ncbi_uid=4158879&amp;link_uid=PMC3411131" href="/pmc/articles/PMC3411131/"><img src="/pmc/articles/instance/3411131/bin/10.1177_1947601911432334-fig3.gif" src-large="/pmc/articles/instance/3411131/bin/10.1177_1947601911432334-fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3411131/" ref="ordinalpos=4704&amp;ncbi_uid=4158879&amp;link_uid=PMC3411131" image-link="/pmc/articles/PMC3411131/figure/fig3-1947601911432334/" class="imagepopup">Figure 3. From: Molecular Mechanisms of Tumor Angiogenesis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways in angiogenesis. The key cellular events associated with tumor angiogenesis are very similar to those described in developmental angiogenesis. A vascular sprout is composed of stalk cells and tip cells. The distinction between stalk and tip is conveyed by Notch <span class="highlight" style="background-color:">signaling</span>, which is higher in the stalk than in the tip cell. Tip cells received signals from a gradient of VEGF and from the Slit-Robo <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Safiyyah Ziyad, et al. Genes Cancer. 2011 December;2(12):1085-1096.</div><p class="links"><a class="dblinks" href="/pubmed/22866200" ref="ordinalpos=4704&amp;ncbi_uid=4158879&amp;link_uid=22866200">Citation</a><a class="dblinks" href="/pmc/articles/PMC3411131/" ref="ordinalpos=4704&amp;ncbi_uid=4158879&amp;link_uid=PMC3411131">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4705.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2674225/figure/fig2/" ref="ordinalpos=4705&amp;ncbi_uid=1767274&amp;link_uid=PMC2674225" href="/pmc/articles/PMC2674225/"><img src="/pmc/articles/instance/2674225/bin/PROCATS62152f2.gif" src-large="/pmc/articles/instance/2674225/bin/PROCATS62152f2.jpg" alt="&lt;b&gt;Figure 2.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2674225/" ref="ordinalpos=4705&amp;ncbi_uid=1767274&amp;link_uid=PMC2674225" image-link="/pmc/articles/PMC2674225/figure/fig2/" class="imagepopup">Figure 2..  From: Genomics of Lung Cancer. </a></p><div class="supp"><p class="details"><div>Classification of lung adenocarcinoma reflects progression along a spectrum of invasiveness spanning epithelial atypia, bronchioloalveolar adenocarcinoma (BAC), BAC with foci of microinvasion, and invasive adenocarcinoma. In cells with repression of TGFβ-RII (<i>lower panel</i>), <span class="highlight" style="background-color:">signaling</span> through the Smad <span class="highlight" style="background-color:">pathway</span> is abrogated (<i>right</i>). Invasiveness is mediated through noncanonical <span class="highlight" style="background-color:">signaling</span> pathways that require RANTES (<i>left</i>).</div></p></div><div class="aux"><div class="resc">Alain C. Borczuk, et al. Proc Am Thorac Soc. 2009 April 15;6(2):152-158.</div><p class="links"><a class="dblinks" href="/pubmed/19349483" ref="ordinalpos=4705&amp;ncbi_uid=1767274&amp;link_uid=19349483">Citation</a><a class="dblinks" href="/pmc/articles/PMC2674225/" ref="ordinalpos=4705&amp;ncbi_uid=1767274&amp;link_uid=PMC2674225">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4706.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1386115/figure/F3/" ref="ordinalpos=4706&amp;ncbi_uid=668101&amp;link_uid=PMC1386115" href="/pmc/articles/PMC1386115/"><img src="/pmc/articles/instance/1386115/bin/JCI0627883.f3.gif" src-large="/pmc/articles/instance/1386115/bin/JCI0627883.f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1386115/" ref="ordinalpos=4706&amp;ncbi_uid=668101&amp;link_uid=PMC1386115" image-link="/pmc/articles/PMC1386115/figure/F3/" class="imagepopup">Figure 3.  From: Liver X receptors as integrators of metabolic and inflammatory <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Integration of lipid metabolic and inflammatory <span class="highlight" style="background-color:">signaling</span> in macrophages by LXRs. Recognition of cytokines, bacterial components, or intact pathogens by their corresponding receptors initiates expression of proinflammatory genes (e.g., iNOS). Activation of the TLR3/4 receptors by these signals blocks LXR-dependent gene transcription and cholesterol efflux from macrophages via an IFN regulatory factor 3–dependent (IRF3-dependent) <span class="highlight" style="background-color:">pathway</span>. On the other hand, ligand activation of LXRs inhibits NF-κB–dependent induction of inflammatory gene expression. Intracellular bacteria induce LXRα expression, possibly through a NOD2-dependent <span class="highlight" style="background-color:">pathway</span>, and promote macrophage survival, through induction of Api6 (also known as AIM and SPα) and other targets.</div></p></div><div class="aux"><div class="resc">Noam Zelcer, et al. J Clin Invest. 2006 March 1;116(3):607-614.</div><p class="links"><a class="dblinks" href="/pubmed/16511593" ref="ordinalpos=4706&amp;ncbi_uid=668101&amp;link_uid=16511593">Citation</a><a class="dblinks" href="/pmc/articles/PMC1386115/" ref="ordinalpos=4706&amp;ncbi_uid=668101&amp;link_uid=PMC1386115">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4707.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3039349/figure/F3/" ref="ordinalpos=4707&amp;ncbi_uid=3637909&amp;link_uid=PMC3039349" href="/pmc/articles/PMC3039349/"><img src="/pmc/articles/instance/3039349/bin/zbc0081148370003.gif" src-large="/pmc/articles/instance/3039349/bin/zbc0081148370003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3039349/" ref="ordinalpos=4707&amp;ncbi_uid=3637909&amp;link_uid=PMC3039349" image-link="/pmc/articles/PMC3039349/figure/F3/" class="imagepopup">FIGURE 3. From: RNA Polymerase 1-driven Transcription as a Mediator of BDNF-induced Neurite Outgrowth. </a></p><div class="supp"><p class="details"><div><b>BDNF regulates Pol1-driven transcription through the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, pharmacological inhibition of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> attenuated the BDNF-induced increase in 45 S pre-rRNA levels. DIV6 cortical neurons were treated with BDNF and the MKK1/2 inhibitor U0126 (50 μ<span style="font-variant: small-caps" class="small-caps">m</span>) or its vehicle control (0.2% Me<sub>2</sub>SO). After 8 h, cells were lysed, and 45 S pre-rRNA/18 S rRNA ratios were determined by quantitative RT-PCR. Our previously published studies validated the effectiveness and specificity of 50 μ<span style="font-variant: small-caps" class="small-caps">m</span> U0126 as an ERK1/2 <span class="highlight" style="background-color:">pathway</span> inhibitor in cultured cortical neurons (46). <i>B</i>, pharmacological inhibition of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> attenuated BDNF-induced activation of the rDNA promoter. DIV4 cortical neurons were cotransfected with the β-gal expression plasmid (pEF1αLacZ) and the rDNA promoter-luciferase construct (0.2 + 0.2 μg of plasmid DNAs/5 × 10<sup>5</sup> cells, respectively). Two days after transfection, neurons were stimulated as described for <i>A. C</i>, selective activation of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with MKK1ca was sufficient to stimulate the rDNA promoter. DIV4 cortical neurons were transfected as described for <i>B</i>. In addition, MKK1ca or MKK1wt was added to the transfection mixtures as indicated. At 48 h post-transfection, cells were washed twice with serum-free medium and then placed in serum-free medium containing the NMDA receptor antagonist MK801 (1 μ<span style="font-variant: small-caps" class="small-caps">m</span>). This treatment was used to reduce the basal levels of the ERK1/2 <span class="highlight" style="background-color:">pathway</span> activity. Four hours later, cells were lysed, and rDNA promoter activity was determined. In <i>A–C</i>, data from three independent experiments are presented; four sister cultures from each experiment were analyzed in <i>B</i> and <i>C. Error bars</i> are S.E. **, <i>p</i> &lt; 0.01; ***, <i>p</i> &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Cynthia Gomes, et al. J Biol Chem. 2011 February 11;286(6):4357-4363.</div><p class="links"><a class="dblinks" href="/pubmed/21098478" ref="ordinalpos=4707&amp;ncbi_uid=3637909&amp;link_uid=21098478">Citation</a><a class="dblinks" href="/pmc/articles/PMC3039349/" ref="ordinalpos=4707&amp;ncbi_uid=3637909&amp;link_uid=PMC3039349">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4708.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3866163/figure/pone-0082751-g011/" ref="ordinalpos=4708&amp;ncbi_uid=5616216&amp;link_uid=PMC3866163" href="/pmc/articles/PMC3866163/"><img src="/pmc/articles/instance/3866163/bin/pone.0082751.g011.gif" src-large="/pmc/articles/instance/3866163/bin/pone.0082751.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3866163/" ref="ordinalpos=4708&amp;ncbi_uid=5616216&amp;link_uid=PMC3866163" image-link="/pmc/articles/PMC3866163/figure/pone-0082751-g011/" class="imagepopup">Figure 11. AcFBE-induced downregulation of MAPK and PI3K/AKT pathways as demonstrated by B cell <span class="highlight" style="background-color:">signaling</span> network using KEGG diagram..  From: Global Assessment of Antrodia cinnamomea-Induced MicroRNA Alterations in Hepatocarcinoma Cells. </a></p><div class="supp"><p class="details"><div>Transcriptomes generated from messenger RNAs of SK-HEP-1 liver cancer cells with or without AcFBE treatment for 2 hours were analyzed and plugged into KEGG database for <span class="highlight" style="background-color:">pathway</span> analysis. Through the process, PI3K/AKT/IKKβ/NFκB and MAPK were identified among the most significantly affected pathways. Upregulated <span class="highlight" style="background-color:">pathway</span> genes are highlighted in red boxes, while downregulated <span class="highlight" style="background-color:">pathway</span> genes highlighted in blue boxes.</div></p></div><div class="aux"><div class="resc">Yen-Ju Chen, et al. PLoS One. 2013;8(12):e82751.</div><p class="links"><a class="dblinks" href="/pubmed/24358224" ref="ordinalpos=4708&amp;ncbi_uid=5616216&amp;link_uid=24358224">Citation</a><a class="dblinks" href="/pmc/articles/PMC3866163/" ref="ordinalpos=4708&amp;ncbi_uid=5616216&amp;link_uid=PMC3866163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4709.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2729372/figure/pone-0006792-g006/" ref="ordinalpos=4709&amp;ncbi_uid=1971087&amp;link_uid=PMC2729372" href="/pmc/articles/PMC2729372/"><img src="/pmc/articles/instance/2729372/bin/pone.0006792.g006.gif" src-large="/pmc/articles/instance/2729372/bin/pone.0006792.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2729372/" ref="ordinalpos=4709&amp;ncbi_uid=1971087&amp;link_uid=PMC2729372" image-link="/pmc/articles/PMC2729372/figure/pone-0006792-g006/" class="imagepopup">Figure 6. An informational network of the genes regulated by H3K9Ac and H3K9Me2 upon MSC osteogenic differentiation..  From: Genome-Wide Analysis of Histone H3 Lysine9 Modifications in Human Mesenchymal Stem Cell Osteogenic Differentiation. </a></p><div class="supp"><p class="details"><div>The informational <span class="highlight" style="background-color:">pathway</span> networks were generated by using BioRag (<i>http//:</i>www.biorag.org). The most affected pathways are the cell cycle <span class="highlight" style="background-color:">pathway</span> (cluster I), the cell communication (cluster II), the cytokine-cytokine receptor interaction (cluster III), the regulation of actin cytoskeleton (cluster IV), and the TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (cluster V)</div></p></div><div class="aux"><div class="resc">Jiang Tan, et al. PLoS One. 2009;4(8):e6792.</div><p class="links"><a class="dblinks" href="/pubmed/19710927" ref="ordinalpos=4709&amp;ncbi_uid=1971087&amp;link_uid=19710927">Citation</a><a class="dblinks" href="/pmc/articles/PMC2729372/" ref="ordinalpos=4709&amp;ncbi_uid=1971087&amp;link_uid=PMC2729372">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4710.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3050667/figure/F7/" ref="ordinalpos=4710&amp;ncbi_uid=3454764&amp;link_uid=PMC3050667" href="/pmc/articles/PMC3050667/"><img src="/pmc/articles/instance/3050667/bin/DEV060046F7.gif" src-large="/pmc/articles/instance/3050667/bin/DEV060046F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3050667/" ref="ordinalpos=4710&amp;ncbi_uid=3454764&amp;link_uid=PMC3050667" image-link="/pmc/articles/PMC3050667/figure/F7/" class="imagepopup">Fig. 7. From: A Slit/miR-218/Robo regulatory loop is required during heart tube formation in zebrafish. </a></p><div class="supp"><p class="details"><div><b>Model of a Slit/miR-218/Robo axis that regulates Vegf <span class="highlight" style="background-color:">signaling</span> and endocardial migration.</b> Slit2 inhibits the migration of the endocardium to the midline through a combination of negative regulation of the Vegf <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and Slit/Robo-mediated repulsion signals. During midline migration, the endocardium is likely to sense Slit2 secreted by the floor plate; however, autocrine <span class="highlight" style="background-color:">signaling</span> may also play a role in endocardial migration. The identity of the Slit2 receptor(s) that mediates the negative regulation of Vegf <span class="highlight" style="background-color:">signaling</span> is unclear as this effect appears to be at least partly Robo4 independent. The <i>Slit2/3</i>-encoded microRNA miR-218 targets <i>Robo1</i> (and <i>Robo2</i> and <i>Srgap2</i>, not shown) for repression. Robo1 positively regulates endocardial migration, perhaps by promoting the auto-phosphorylation of Vegfr2. However, increased Robo1 expression in <i>miR-218</i> morphants may mediate Slit/Robo-mediated repulsion, suggesting that miR-218 might titrate the levels of <i>Robo1</i> to facilitate heart field migration.</div></p></div><div class="aux"><div class="resc">Jason E. Fish, et al. Development. 2011 April 1;138(7):1409-1419.</div><p class="links"><a class="dblinks" href="/pubmed/21385766" ref="ordinalpos=4710&amp;ncbi_uid=3454764&amp;link_uid=21385766">Citation</a><a class="dblinks" href="/pmc/articles/PMC3050667/" ref="ordinalpos=4710&amp;ncbi_uid=3454764&amp;link_uid=PMC3050667">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4711.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3061884/figure/F9/" ref="ordinalpos=4711&amp;ncbi_uid=2881082&amp;link_uid=PMC3061884" href="/pmc/articles/PMC3061884/"><img src="/pmc/articles/instance/3061884/bin/1755-8794-4-21-9.gif" src-large="/pmc/articles/instance/3061884/bin/1755-8794-4-21-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3061884/" ref="ordinalpos=4711&amp;ncbi_uid=2881082&amp;link_uid=PMC3061884" image-link="/pmc/articles/PMC3061884/figure/F9/" class="imagepopup">Figure 9.  From: CDCOCA: A statistical method to define complexity dependence of co-occuring chromosomal aberrations. </a></p><div class="supp"><p class="details"><div><b>100 frequent associations obtained after p value cut off in MCL with CDCOCA</b>. <span class="highlight" style="background-color:">Pathway</span> associations for presumptive target genes from the 100 most frequent associations in MCL. TNF-alpha <span class="highlight" style="background-color:">signaling</span> genes (red triangles) and T-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes (blue triangles) present in these genetic locations are shown. Gains are represented with prefix "g" and green color circles whereas losses with prefix "l" and orange circles.</div></p></div><div class="aux"><div class="resc">Nitin Kumar, et al. BMC Med Genomics. 2011;4:21-21.</div><p class="links"><a class="dblinks" href="/pubmed/21371302" ref="ordinalpos=4711&amp;ncbi_uid=2881082&amp;link_uid=21371302">Citation</a><a class="dblinks" href="/pmc/articles/PMC3061884/" ref="ordinalpos=4711&amp;ncbi_uid=2881082&amp;link_uid=PMC3061884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4712.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3427289/figure/F6/" ref="ordinalpos=4712&amp;ncbi_uid=5144226&amp;link_uid=PMC3427289" href="/pmc/articles/PMC3427289/"><img src="/pmc/articles/instance/3427289/bin/epi-7-940-g6.gif" src-large="/pmc/articles/instance/3427289/bin/epi-7-940-g6.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3427289/" ref="ordinalpos=4712&amp;ncbi_uid=5144226&amp;link_uid=PMC3427289" image-link="/pmc/articles/PMC3427289/figure/F6/" class="imagepopup">Figure. From: Epigenetic deregulation of miR-29a and miR-1256 by isoflavone contributes to the inhibition of prostate cancer cell growth and invasion. </a></p><div class="supp"><p class="details"><div><b>Figure 6.</b> Inhibition of cell growth and invasion by miR-29a, miR-1256 and isoflavone through TRIM68/AR and PGK-1 <span class="highlight" style="background-color:">signaling</span>. Transfection of miR-29a and miR-1256 mimic into PCa cells, and isoflavone treatment significantly inhibited cell growth (A) and invasion (B) of PCa cells (*: p &lt; 0.05; n = 3). Microarray and Ingenuity <span class="highlight" style="background-color:">Pathway</span> analysis showed that isoflavone downregulated TRIM68/AR and PGK-1 <span class="highlight" style="background-color:">signaling</span> (C; Green: downregulated; Red: upregulated).</div></p></div><div class="aux"><div class="resc">Yiwei Li, et al. Epigenetics. 2012 August 1;7(8):940-949.</div><p class="links"><a class="dblinks" href="/pubmed/22805767" ref="ordinalpos=4712&amp;ncbi_uid=5144226&amp;link_uid=22805767">Citation</a><a class="dblinks" href="/pmc/articles/PMC3427289/" ref="ordinalpos=4712&amp;ncbi_uid=5144226&amp;link_uid=PMC3427289">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4713.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3812949/figure/F1/" ref="ordinalpos=4713&amp;ncbi_uid=5427210&amp;link_uid=PMC3812949" href="/pmc/articles/PMC3812949/"><img src="/pmc/articles/instance/3812949/bin/nihms461103f1.gif" src-large="/pmc/articles/instance/3812949/bin/nihms461103f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3812949/" ref="ordinalpos=4713&amp;ncbi_uid=5427210&amp;link_uid=PMC3812949" image-link="/pmc/articles/PMC3812949/figure/F1/" class="imagepopup">Fig. 1.  From: TGF-?: Titan of Lung Fibrogenesis. </a></p><div class="supp"><p class="details"><div>TGF-β <span class="highlight" style="background-color:">signaling</span> and inhibition in lung fibrogenesis. TGF-β plays pivotal roles in the key events of lung fibrogenesis: fibroblast to myofibroblast differentiation, AEC apoptosis and EMT. Each step along the <span class="highlight" style="background-color:">pathway</span> of activation and <span class="highlight" style="background-color:">signaling</span> of TGF-β represents a potential target for regulating the activity of TGF-β. Inh.: inhibition. Please see text for details.</div></p></div><div class="aux"><div class="resc">Xinping Yue, et al. Curr Enzym Inhib. 2010 July 1;6(2):10.2174/10067.</div><p class="links"><a class="dblinks" href="/pubmed/24187529" ref="ordinalpos=4713&amp;ncbi_uid=5427210&amp;link_uid=24187529">Citation</a><a class="dblinks" href="/pmc/articles/PMC3812949/" ref="ordinalpos=4713&amp;ncbi_uid=5427210&amp;link_uid=PMC3812949">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4714.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3532876/figure/fig2/" ref="ordinalpos=4714&amp;ncbi_uid=4517872&amp;link_uid=PMC3532876" href="/pmc/articles/PMC3532876/"><img src="/pmc/articles/instance/3532876/bin/OXIMED2012-932838.002.gif" src-large="/pmc/articles/instance/3532876/bin/OXIMED2012-932838.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3532876/" ref="ordinalpos=4714&amp;ncbi_uid=4517872&amp;link_uid=PMC3532876" image-link="/pmc/articles/PMC3532876/figure/fig2/" class="imagepopup">Figure 2.  From: Redox Homeostasis in Pancreatic β Cells. </a></p><div class="supp"><p class="details"><div>Vicious spirals of repeating self-accelerating oxidative stress and dysregulated redox and information <span class="highlight" style="background-color:">signaling</span> as possible causes of type 2 diabetes. Schema of cell events that occur at type 2 diabetes development, as related to oxidative stress and impaired redox homeostasis and <span class="highlight" style="background-color:">signaling</span>, dysfunctional insulin <span class="highlight" style="background-color:">signaling</span> in peripheral tissues, and autocrine insulin <span class="highlight" style="background-color:">signaling</span> in failing pancreatic <i>β</i> cells. Considered “vicious spirals” (depicted by black spirals) of progressive oxidative stress leading to oxidation of proteins, lipids, and DNA, notably mitochondrial DNA, all resulting in further turn of self-accelerating metabolic deterioration and specifically impairment of the glucose sensing. Progressive oxidative stress also impairs redox <span class="highlight" style="background-color:">signaling</span> and autocrine insulin <span class="highlight" style="background-color:">signaling</span> which further deteriorates fitness of <i>β</i> cells and their housekeeping mechanisms, specifically autophagy and mitochondria-specific autophagy, mitophagy, besides initiating an inappropriate apoptosis. Another component of oxidative stress comes from the intake of excessive fatty acids and lipid peroxidation products, generally termed as lipotoxicity. Yet another component results from elevated blood glucose as is known as glucotoxicity further accelerating cell oxidative stress, impairing cell maintenance, dysregulating information <span class="highlight" style="background-color:">signaling</span> and leading to advanced glycation end products (AGEs), (yet further accelerating oxidative stress and other cell stresses), activating polyol <span class="highlight" style="background-color:">pathway</span> and thus again contributing to <i>pro</i>-oxidation redox homeostasis, activating hexosamine <span class="highlight" style="background-color:">pathway</span> and dysregulating crucial survival pathways including insulin receptor (autocrine) <span class="highlight" style="background-color:">signaling</span>, and finally enhancing glycosylation and forming antiparallel crossed <i>β</i>-pleated sheet structure called amylin-derived islet amyloid, promoting <i>β</i> cell cytotoxicity.</div></p></div><div class="aux"><div class="resc">Petr Ježek, et al. Oxid Med Cell Longev. 2012;2012:932838.</div><p class="links"><a class="dblinks" href="/pubmed/23304259" ref="ordinalpos=4714&amp;ncbi_uid=4517872&amp;link_uid=23304259">Citation</a><a class="dblinks" href="/pmc/articles/PMC3532876/" ref="ordinalpos=4714&amp;ncbi_uid=4517872&amp;link_uid=PMC3532876">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4715.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1502564/figure/F3/" ref="ordinalpos=4715&amp;ncbi_uid=710241&amp;link_uid=PMC1502564" href="/pmc/articles/PMC1502564/"><img src="/pmc/articles/instance/1502564/bin/zpq0250625810003.gif" src-large="/pmc/articles/instance/1502564/bin/zpq0250625810003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1502564/" ref="ordinalpos=4715&amp;ncbi_uid=710241&amp;link_uid=PMC1502564" image-link="/pmc/articles/PMC1502564/figure/F3/" class="imagepopup">Fig. 3. From: Gene expression patterns define novel roles for E47 in cell cycle progression, cytokine-mediated <span class="highlight" style="background-color:">signaling</span>, and T lineage development. </a></p><div class="supp"><p class="details"><div>Model diagrams indicating regulation and function of E47 activity. (<i>A</i>) Model diagram depicting the effects of pre-TCR <span class="highlight" style="background-color:">signaling</span> on E2A function. Signals from the pre-TCR complex abrogate the inhibitory effect of E2A proteins on the proliferation and differentiation of DN3 thymocytes to the DP stage. However, pre-TCR <span class="highlight" style="background-color:">signaling</span> also induces a function of E2A proteins in the down-regulation of CD25 expression. The asterisk indicates novel E-protein activities in response to pre-TCR <span class="highlight" style="background-color:">signaling</span>. Genes regulated by E47 in E2A-deficient lymphomas and during β selection are indicated. (<i>B</i>) Model diagram showing a regulatory network involving cytokine receptor-mediated <span class="highlight" style="background-color:">signaling</span>, E47, and SOCS activities in a common <span class="highlight" style="background-color:">pathway</span>. (<i>C</i>) Similarities of regulatory networks in <i>D</i>. <i>melanogaster</i> sensory organ development and T lineage cells.</div></p></div><div class="aux"><div class="resc">Ruth Schwartz, et al. Proc Natl Acad Sci U S A. 2006 June 27;103(26):9976-9981.</div><p class="links"><a class="dblinks" href="/pubmed/16782810" ref="ordinalpos=4715&amp;ncbi_uid=710241&amp;link_uid=16782810">Citation</a><a class="dblinks" href="/pmc/articles/PMC1502564/" ref="ordinalpos=4715&amp;ncbi_uid=710241&amp;link_uid=PMC1502564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4716.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3532163/figure/pone-0053129-g004/" ref="ordinalpos=4716&amp;ncbi_uid=4495422&amp;link_uid=PMC3532163" href="/pmc/articles/PMC3532163/"><img src="/pmc/articles/instance/3532163/bin/pone.0053129.g004.gif" src-large="/pmc/articles/instance/3532163/bin/pone.0053129.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3532163/" ref="ordinalpos=4716&amp;ncbi_uid=4495422&amp;link_uid=PMC3532163" image-link="/pmc/articles/PMC3532163/figure/pone-0053129-g004/" class="imagepopup">Figure 4. The MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing differentially-expressed gene in SLE patients PBMCs versus healthy controls..  From: Characterization of the Phosphoproteome in SLE Patients. </a></p><div class="supp"><p class="details"><div>Red marks indicate the genes with differential phosphorylation profiles.</div></p></div><div class="aux"><div class="resc">Xinzhou Zhang, et al. PLoS One. 2012;7(12):e53129.</div><p class="links"><a class="dblinks" href="/pubmed/23285258" ref="ordinalpos=4716&amp;ncbi_uid=4495422&amp;link_uid=23285258">Citation</a><a class="dblinks" href="/pmc/articles/PMC3532163/" ref="ordinalpos=4716&amp;ncbi_uid=4495422&amp;link_uid=PMC3532163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4717.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3448567/figure/F3/" ref="ordinalpos=4717&amp;ncbi_uid=4257723&amp;link_uid=PMC3448567" href="/pmc/articles/PMC3448567/"><img src="/pmc/articles/instance/3448567/bin/zh89991292500003.gif" src-large="/pmc/articles/instance/3448567/bin/zh89991292500003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3448567/" ref="ordinalpos=4717&amp;ncbi_uid=4257723&amp;link_uid=PMC3448567" image-link="/pmc/articles/PMC3448567/figure/F3/" class="imagepopup">Figure 3.  From: Beyond Hox: the role of ParaHox genes in normal and malignant hematopoiesis. </a></p><div class="supp"><p class="details"><div><b>Hypothetical model of Cdx gene regulation in normal and malignant hematopoiesis.</b> (A) Normal hematopoiesis, based on results from studies in embryogenesis, the canonical Wnt <span class="highlight" style="background-color:">pathway</span> regulates the expression of <i>CDX4</i> by the Lef1/Tcf complex in hematopoiesis. The FGF-MAPK <span class="highlight" style="background-color:">pathway</span> is not constitutively active in stem and progenitors cells. (B) Malignant hematopoiesis, either deregulation of the WNT <span class="highlight" style="background-color:">pathway</span> or FGF <span class="highlight" style="background-color:">signaling</span> induces aberrant CDX2 expression in human leukemia.</div></p></div><div class="aux"><div class="resc">Vijay P. S. Rawat, et al. Blood. 2012 July 19;120(3):519-527.</div><p class="links"><a class="dblinks" href="/pubmed/22547580" ref="ordinalpos=4717&amp;ncbi_uid=4257723&amp;link_uid=22547580">Citation</a><a class="dblinks" href="/pmc/articles/PMC3448567/" ref="ordinalpos=4717&amp;ncbi_uid=4257723&amp;link_uid=PMC3448567">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4718.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3431263/figure/F1/" ref="ordinalpos=4718&amp;ncbi_uid=4215922&amp;link_uid=PMC3431263" href="/pmc/articles/PMC3431263/"><img src="/pmc/articles/instance/3431263/bin/1752-0509-6-50-1.gif" src-large="/pmc/articles/instance/3431263/bin/1752-0509-6-50-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3431263/" ref="ordinalpos=4718&amp;ncbi_uid=4215922&amp;link_uid=PMC3431263" image-link="/pmc/articles/PMC3431263/figure/F1/" class="imagepopup">Figure 1.  From: A systems biology approach reveals common metastatic pathways in osteosarcoma. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span> analysis of up-regulated genes from HOS/143B and SaOS-2/LM7 models.</b> Top 10 significant pathways identified by MetaCore using all the significantly up-regulated genes from <b>(A)</b> HOS/143B and <b>(b)</b> SaOS-2/LM7 models. Only one common <span class="highlight" style="background-color:">pathway</span>, the “Development_WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Part 2”, was identified in the top 10 significant pathways between the two models. Bars quantify the –log (p-value) indicating the enrichment of the individual pathways identified. False discovery rate (FDR) is calculated for all identified pathways and a cutoff of FDR &lt; 0.05 is set to define pathways as significant. Asterisks denote significant pathways shown.</div></p></div><div class="aux"><div class="resc">Ricardo J Flores, et al. BMC Syst Biol. 2012;6:50-50.</div><p class="links"><a class="dblinks" href="/pubmed/22640921" ref="ordinalpos=4718&amp;ncbi_uid=4215922&amp;link_uid=22640921">Citation</a><a class="dblinks" href="/pmc/articles/PMC3431263/" ref="ordinalpos=4718&amp;ncbi_uid=4215922&amp;link_uid=PMC3431263">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4719.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3361329/figure/fig2/" ref="ordinalpos=4719&amp;ncbi_uid=4045992&amp;link_uid=PMC3361329" href="/pmc/articles/PMC3361329/"><img src="/pmc/articles/instance/3361329/bin/PRI2012-172894.002.gif" src-large="/pmc/articles/instance/3361329/bin/PRI2012-172894.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3361329/" ref="ordinalpos=4719&amp;ncbi_uid=4045992&amp;link_uid=PMC3361329" image-link="/pmc/articles/PMC3361329/figure/fig2/" class="imagepopup">Figure 2.  From: Molecular Mechanisms of Liver Injury and Hepatocarcinogenesis: Focusing on the Role of Stress-Activated MAPK. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways of TNF<i>α</i>-induced hepatocyte death. After binding of TNF<i>α</i> to TNF-R1, complexes I and II are formed. As the amount of caspase-8 activation in complex II is not sufficient to induce cell death of hepatocytes, activation of the mitochondrial apoptotic <span class="highlight" style="background-color:">pathway</span> is essential. Activated caspase-8 initiates the mitochondrial apoptotic <span class="highlight" style="background-color:">pathway</span> by cleaving Bid to tBid. In addition, JNK can also activate the mitochondrial apoptotic <span class="highlight" style="background-color:">pathway</span> though modulation of Bcl-2 family members or degradation of c-FLIP. These pathways cooperate to induce cell death, while NF-<i>κ</i>B and p38 inhibit TNF<i>α</i>-induced hepatocyte death by suppressing JNK activation through various mechanisms.</div></p></div><div class="aux"><div class="resc">Hayato Nakagawa, et al. Patholog Res Int. 2012;2012:172894.</div><p class="links"><a class="dblinks" href="/pubmed/22666632" ref="ordinalpos=4719&amp;ncbi_uid=4045992&amp;link_uid=22666632">Citation</a><a class="dblinks" href="/pmc/articles/PMC3361329/" ref="ordinalpos=4719&amp;ncbi_uid=4045992&amp;link_uid=PMC3361329">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4720.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2168910/figure/f10/" ref="ordinalpos=4720&amp;ncbi_uid=1090402&amp;link_uid=PMC2168910" href="/pmc/articles/PMC2168910/"><img src="/pmc/articles/instance/2168910/bin/zmb0200770550010.gif" src-large="/pmc/articles/instance/2168910/bin/zmb0200770550010.jpg" alt="FIG. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2168910/" ref="ordinalpos=4720&amp;ncbi_uid=1090402&amp;link_uid=PMC2168910" image-link="/pmc/articles/PMC2168910/figure/f10/" class="imagepopup">FIG. 10. From: The Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Controls the Size of the Ocular Lens by Directly Suppressing p57Kip2 Expression  . </a></p><div class="supp"><p class="details"><div>A model describing the role of the Notch <span class="highlight" style="background-color:">signaling</span> in keeping the balance between proliferation and differentiation during lens development.</div></p></div><div class="aux"><div class="resc">Junling Jia, et al. Mol Cell Biol. 2007 October;27(20):7236-7247.</div><p class="links"><a class="dblinks" href="/pubmed/17709399" ref="ordinalpos=4720&amp;ncbi_uid=1090402&amp;link_uid=17709399">Citation</a><a class="dblinks" href="/pmc/articles/PMC2168910/" ref="ordinalpos=4720&amp;ncbi_uid=1090402&amp;link_uid=PMC2168910">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4721.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3160520/figure/F4/" ref="ordinalpos=4721&amp;ncbi_uid=3579052&amp;link_uid=PMC3160520" href="/pmc/articles/PMC3160520/"><img src="/pmc/articles/instance/3160520/bin/nihms292596f4.gif" src-large="/pmc/articles/instance/3160520/bin/nihms292596f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3160520/" ref="ordinalpos=4721&amp;ncbi_uid=3579052&amp;link_uid=PMC3160520" image-link="/pmc/articles/PMC3160520/figure/F4/" class="imagepopup">Fig. 4.  From: Apolipoprotein E reduces food intake via PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the hypothalamus. </a></p><div class="supp"><p class="details"><div>ApoE induced the activation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in rat hypothalamus. 4-h fasted rats received icv LY294002 (1 nmol) or vehicle prior to dark. Sixty min later, the rats were treated icv with apoE (4 μg) or saline. After 30 min, hypothalami were microdissected and the protein extracted. Proteins were analyzed using Western blot with phospho-specific anti-pAkt antibody or antibody that recognizes the total Akt protein as loading control. Pretreatment with LY294002 (LY) attenuated apoE-induced Akt phosphorylation in the hypothalamus. A, representative immunoblots of pAkt and total Akt; B, quantitative analysis. Data represent mean ± SEM; <i>n</i> = 6–7, <i>*P</i> &lt;0.05.</div></p></div><div class="aux"><div class="resc">Ling Shen, et al. Physiol Behav. ;105(1):124-128.</div><p class="links"><a class="dblinks" href="/pubmed/21536059" ref="ordinalpos=4721&amp;ncbi_uid=3579052&amp;link_uid=21536059">Citation</a><a class="dblinks" href="/pmc/articles/PMC3160520/" ref="ordinalpos=4721&amp;ncbi_uid=3579052&amp;link_uid=PMC3160520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4722.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2413058/figure/F7/" ref="ordinalpos=4722&amp;ncbi_uid=1463125&amp;link_uid=PMC2413058" href="/pmc/articles/PMC2413058/"><img src="/pmc/articles/instance/2413058/bin/nihms45866f7.gif" src-large="/pmc/articles/instance/2413058/bin/nihms45866f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2413058/" ref="ordinalpos=4722&amp;ncbi_uid=1463125&amp;link_uid=PMC2413058" image-link="/pmc/articles/PMC2413058/figure/F7/" class="imagepopup">Figure 7. Model for the effect of Mop on EGFR endocytosis.  From: Myopic acts in the endocytic <span class="highlight" style="background-color:">pathway</span> to enhance <span class="highlight" style="background-color:">signaling</span> by the Drosophila EGF receptor. </a></p><div class="supp"><p class="details"><div>Upon activation of the EGFR, ubiquitination by Cbl induces EGFR internalization through clathrin-coated vesicles. These vesicles fuse with early endosomes and the EGFR is passed from the Hrs complex to the ESCRT-I, ESCRT-II and ESCRT-III complexes as the endosomes are transformed into multivesicular bodies (MVBs). ESCRT-III promotes EGFR deubiquitination and entry into the internal vesicles of MVBs; fusion of MVBs with lysosomes results in EGFR degradation. Sprouty prevents the EGFR from progressing into late endosomes. We propose that Mop is required for EGFR progression through the endocytic <span class="highlight" style="background-color:">pathway</span>, perhaps through its effect on Hrs. This progression may allow the EGFR to encounter critical downstream components located on late endosomes (X), or to be recycled to the plasma membrane to prolong <span class="highlight" style="background-color:">signaling</span>. Cleavage of the receptor must occur at a stage after the requirement for <i>mop</i>.</div></p></div><div class="aux"><div class="resc">Grant I. Miura, et al. Development. ;135(11):1913-1922.</div><p class="links"><a class="dblinks" href="/pubmed/18434417" ref="ordinalpos=4722&amp;ncbi_uid=1463125&amp;link_uid=18434417">Citation</a><a class="dblinks" href="/pmc/articles/PMC2413058/" ref="ordinalpos=4722&amp;ncbi_uid=1463125&amp;link_uid=PMC2413058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4723.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3512134/figure/fig7/" ref="ordinalpos=4723&amp;ncbi_uid=5537240&amp;link_uid=PMC3512134" href="/pmc/articles/PMC3512134/"><img src="/pmc/articles/instance/3512134/bin/1165fig7.gif" src-large="/pmc/articles/instance/3512134/bin/1165fig7.jpg" alt="Figure 7 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3512134/" ref="ordinalpos=4723&amp;ncbi_uid=5537240&amp;link_uid=PMC3512134" image-link="/pmc/articles/PMC3512134/figure/fig7/" class="imagepopup">Figure 7 .  From: Mitotic Exit and Separation of Mother and Daughter Cells. </a></p><div class="supp"><p class="details"><div>Conserved structure of Mst/hippo <span class="highlight" style="background-color:">signaling</span> pathways. Pathways similar to the MEN and the RAM network are broadly conserved in eukaryotes. Activating phosphorylations are shown as green arrows, while inhibitory phosphorylation is shown as a red block arrow. The MEN is most similar to the <span class="highlight" style="background-color:">pathway</span> in the left box, although no <i>sav</i> ortholog is present. The RAM is most similar to the <span class="highlight" style="background-color:">pathway</span> in the right box.</div></p></div><div class="aux"><div class="resc">Eric L. Weiss. Genetics. 2012 December;192(4):1165-1202.</div><p class="links"><a class="dblinks" href="/pubmed/23212898" ref="ordinalpos=4723&amp;ncbi_uid=5537240&amp;link_uid=23212898">Citation</a><a class="dblinks" href="/pmc/articles/PMC3512134/" ref="ordinalpos=4723&amp;ncbi_uid=5537240&amp;link_uid=PMC3512134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4724.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3540840/figure/fig4/" ref="ordinalpos=4724&amp;ncbi_uid=5152796&amp;link_uid=PMC3540840" href="/pmc/articles/PMC3540840/"><img src="/pmc/articles/instance/3540840/bin/1472fig4.gif" src-large="/pmc/articles/instance/3540840/bin/1472fig4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3540840/" ref="ordinalpos=4724&amp;ncbi_uid=5152796&amp;link_uid=PMC3540840" image-link="/pmc/articles/PMC3540840/figure/fig4/" class="imagepopup">Fig. 4. From: Macrophage SR-BI regulates LPS-induced pro-inflammatory <span class="highlight" style="background-color:">signaling</span> in mice and isolated macrophages. </a></p><div class="supp"><p class="details"><div>Pro-inflammatory <span class="highlight" style="background-color:">signaling</span> in SR-BI<sup>+/+</sup> and SR-BI <sup>−/−</sup> BMMs. (A) Pro-inflammatory <span class="highlight" style="background-color:">signaling</span> in differentiated BMMs. BMMs were incubated with LPS (20 ng/ml) for 1 h. Cellular protein was harvested and proteins [total (T) or phosphorylated (P) protein] were analyzed by Western blotting using commercially available antibodies. (B) Inhibition of inflammatory <span class="highlight" style="background-color:">signaling</span> pathways using specific inhibitors. BMMs were pretreated with inhibitors as indicated for 1 h at 37°C. Cells were then treated with LPS (20 ng/ml) for 1 h at 37°C. (C) Pro-inflammatory cytokine production in macrophages. Macrophages were pretreated with <span class="highlight" style="background-color:">pathway</span> inhibitors for 1 h at 37°C followed by LPS treatment (20 ng/ml) for 1 h. Medium was collected for cytokine determination by ELISA. Values shown are the mean ± SD of triplicate determinations. *<i>P</i> &lt; 0.05. <i><sup>a</sup></i>Significantly different from SR-BI<sup>+/+</sup> LPS, <i>P</i> &lt; 0.05. <i><sup>b</sup></i>Significantly different from SR-BI<sup>−/−</sup> LPS, <i>P</i> &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Lei Cai, et al. J Lipid Res. 2012 August;53(8):1472-1481.</div><p class="links"><a class="dblinks" href="/pubmed/22589557" ref="ordinalpos=4724&amp;ncbi_uid=5152796&amp;link_uid=22589557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3540840/" ref="ordinalpos=4724&amp;ncbi_uid=5152796&amp;link_uid=PMC3540840">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4725.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3736205/figure/fig6/" ref="ordinalpos=4725&amp;ncbi_uid=5178559&amp;link_uid=PMC3736205" href="/pmc/articles/PMC3736205/"><img src="/pmc/articles/instance/3736205/bin/icb201327f6.gif" src-large="/pmc/articles/instance/3736205/bin/icb201327f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3736205/" ref="ordinalpos=4725&amp;ncbi_uid=5178559&amp;link_uid=PMC3736205" image-link="/pmc/articles/PMC3736205/figure/fig6/" class="imagepopup">Figure 6.  From: Host genetic background impacts modulation of the TLR4 <span class="highlight" style="background-color:">pathway</span> by RON in tissue-associated macrophages. </a></p><div class="supp"><p class="details"><div>Overview of the impact of the RON <span class="highlight" style="background-color:">pathway</span> on M1 versus M2 differentiation program in the context of TLR4 <span class="highlight" style="background-color:">signaling</span>. Transcript and protein levels of IFN-β and TNF-α were compiled from data presented in figures, as described in the text. The IFN-β transcript level was taken from Supplementary Figure S3A (FVB) and from Supplementary Figure S5A (C57Bl6). The TNF-α transcript level was taken from Supplementary Figure S1A (FVB) and Supplementary Figure S2A for C57Bl6 mice. The intermediate time points for TNF-α protein levels in both backgrounds were analyzed (data not shown). Protein or mRNA levels at each time point are expressed as percentage of maximal expression (100%). Optimal TNF-α expression in response to LPS in macrophages from FVB mice was highly dependent on early induction of IFN-β. In contrast, M1/Th1 predisposed macrophages from C57Bl6 mice were mostly refractory to the effects of RON on TNF-α production and IFN-β. We propose that RON <span class="highlight" style="background-color:">signaling</span> in macrophages from FVB mice preserves M2 differentiation in the presence of TLR4 <span class="highlight" style="background-color:">signaling</span>, whereas C57Bl6 macrophages maintain polarization toward M1 cells in the presence of RON <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Amitabha Chaudhuri, et al. Immunol Cell Biol. 2013 August;91(7):451-460.</div><p class="links"><a class="dblinks" href="/pubmed/23817579" ref="ordinalpos=4725&amp;ncbi_uid=5178559&amp;link_uid=23817579">Citation</a><a class="dblinks" href="/pmc/articles/PMC3736205/" ref="ordinalpos=4725&amp;ncbi_uid=5178559&amp;link_uid=PMC3736205">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4726.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3472294/figure/F3/" ref="ordinalpos=4726&amp;ncbi_uid=4300156&amp;link_uid=PMC3472294" href="/pmc/articles/PMC3472294/"><img src="/pmc/articles/instance/3472294/bin/1471-2164-13-348-3.gif" src-large="/pmc/articles/instance/3472294/bin/1471-2164-13-348-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3472294/" ref="ordinalpos=4726&amp;ncbi_uid=4300156&amp;link_uid=PMC3472294" image-link="/pmc/articles/PMC3472294/figure/F3/" class="imagepopup">Figure 3.  From: Identification of a radiosensitivity signature using integrative metaanalysis of published microarray data for NCI-60 cancer cells. </a></p><div class="supp"><p class="details"><div><b> Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its interaction as a radiosensitive target.</b><b> A</b>. Statistical ranking of pathways with the commonly selected 179 genes using SAM analysis. The x-axis displays the -log of the p-value calculated by Fisher's exact test, right-tailed. <b>B</b>. Gene plot showing the influence of individual genes of the integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> produced by a global test. The influence on the y-axis is represented as the p-value, the extent of correlation between SF2 (radiosensitivity) and gene expression in a gene set. A lower p-value means that the gene is well correlated between SF2 and the gene expression value. <b>C</b>. Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> interaction with identified adhesion molecules from the 31 radiosensitivity signature. (References from Ingenuity knowledge base, Additional file 6).</div></p></div><div class="aux"><div class="resc">Han Sang Kim, et al. BMC Genomics. 2012;13:348-348.</div><p class="links"><a class="dblinks" href="/pubmed/22846430" ref="ordinalpos=4726&amp;ncbi_uid=4300156&amp;link_uid=22846430">Citation</a><a class="dblinks" href="/pmc/articles/PMC3472294/" ref="ordinalpos=4726&amp;ncbi_uid=4300156&amp;link_uid=PMC3472294">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4727.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2829096/figure/fig2/" ref="ordinalpos=4727&amp;ncbi_uid=2790001&amp;link_uid=PMC2829096" href="/pmc/articles/PMC2829096/"><img src="/pmc/articles/instance/2829096/bin/intimmdxp128f02_4c.gif" src-large="/pmc/articles/instance/2829096/bin/intimmdxp128f02_4c.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2829096/" ref="ordinalpos=4727&amp;ncbi_uid=2790001&amp;link_uid=PMC2829096" image-link="/pmc/articles/PMC2829096/figure/fig2/" class="imagepopup">Fig. 2. From: Extracellular and intracellular pathogen recognition by Drosophila PGRP-LE and PGRP-LC. </a></p><div class="supp"><p class="details"><div>Recognition of microorganisms and induction of innate immune responses in <i>Drosophila</i>. (A) The primary humoral response, i.e. activation of the proPO cascade leading to melanization, is induced by the cleavage of proPO to PO by a serine protease, proPO-activating enzyme (PPAE). (B) A constitutive hemolymph protein, PGRP-LE, binds to monomeric and polymeric DAP-type peptidoglycans (PGNs), which are components of many Gram-negative and some Gram-positive bacteria, and activates the proPO cascade upstream of PPAE. PGRP-LE is also involved in activating the secondary humoral response, the imd <span class="highlight" style="background-color:">pathway</span>-dependent induction of antibacterial peptides. (C) Membrane PGRP-LC is also required for monomeric and polymeric DAP-type peptidoglycan-mediated activation of the imd <span class="highlight" style="background-color:">pathway</span>. (D) PGRP-SA in the hemolymph is involved in the recognition of Gram-positive bacteria with Lys-type peptidoglycans and is required for activation of the Toll <span class="highlight" style="background-color:">pathway</span> in cooperation with GNBP1. PGRP-SD has some redundant functions with PGRP-SA and GNBP1. (E) GNBP3 is involved in yeast-mediated activation of the Toll <span class="highlight" style="background-color:">pathway</span>. (F) The activation of the Toll <span class="highlight" style="background-color:">pathway</span> is mediated by its active ligand, Spz, cleaved from pro-Spz by the serine protease cascade, including the SPE, Spheroide, Spirit, Sphinx, Persephone, ModSP and Grass. (G) In addition to the extracellular functions, PGRP-LE induces anti-microbial peptides through the imd <span class="highlight" style="background-color:">pathway</span> and autophagy in the cytoplasm. <span class="highlight" style="background-color:">Signaling</span> via the Toll or imd <span class="highlight" style="background-color:">pathway</span> activates NF-κB and subsequent transcription of genes for anti-microbial peptides such as Drosomycin and Diptericin.</div></p></div><div class="aux"><div class="resc">Shoichiro Kurata. Int Immunol. 2010 March;22(3):143-148.</div><p class="links"><a class="dblinks" href="/pubmed/20089584" ref="ordinalpos=4727&amp;ncbi_uid=2790001&amp;link_uid=20089584">Citation</a><a class="dblinks" href="/pmc/articles/PMC2829096/" ref="ordinalpos=4727&amp;ncbi_uid=2790001&amp;link_uid=PMC2829096">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4728.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3948775/figure/pone-0091077-g008/" ref="ordinalpos=4728&amp;ncbi_uid=5912421&amp;link_uid=PMC3948775" href="/pmc/articles/PMC3948775/"><img src="/pmc/articles/instance/3948775/bin/pone.0091077.g008.gif" src-large="/pmc/articles/instance/3948775/bin/pone.0091077.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3948775/" ref="ordinalpos=4728&amp;ncbi_uid=5912421&amp;link_uid=PMC3948775" image-link="/pmc/articles/PMC3948775/figure/pone-0091077-g008/" class="imagepopup">Figure 8. Hypothetical network visualizing the proposed difference maintaining and regulating steroidogenesis and androstenone biosynthesis in high and low androstenone boars..  From: <span class="highlight" style="background-color:">Pathway</span> Based Analysis of Genes and Interactions Influencing Porcine Testis Samples from Boars with Divergent Androstenone Content in Back Fat. </a></p><div class="supp"><p class="details"><div>Legend: Circular nodes: genes, hexagonal nodes: enriched pathways, diamond nodes: pathways that might be involved in steroidogenesis, but not found in results, rectangular nodes: metabolites from pathways. In high androstenone samples, steroidogenesis was activated by cAMP-PKA <span class="highlight" style="background-color:">signaling</span>, lipid peroxidation activity of ROS was inhibited by metabolites from glutathione metabolism <span class="highlight" style="background-color:">pathway</span> and de novo synthesis of cholesterol as a result of an active fatty acid metabolism activity might have boosted steroidogenesis and androstenone synthesis. In low androstenone samples, weak cAMP-PKC <span class="highlight" style="background-color:">signaling</span> of steroidogenesis and inhibition of steroidogenesis by ceramides synthesized from sphnigolipid metabolim <span class="highlight" style="background-color:">pathway</span> might have lead to weak steroidogenesis and hence, low androstenone synthesis.</div></p></div><div class="aux"><div class="resc">Sudeep Sahadevan, et al. PLoS One. 2014;9(3):e91077.</div><p class="links"><a class="dblinks" href="/pubmed/24614349" ref="ordinalpos=4728&amp;ncbi_uid=5912421&amp;link_uid=24614349">Citation</a><a class="dblinks" href="/pmc/articles/PMC3948775/" ref="ordinalpos=4728&amp;ncbi_uid=5912421&amp;link_uid=PMC3948775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4729.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3209254/figure/F9/" ref="ordinalpos=4729&amp;ncbi_uid=3482797&amp;link_uid=PMC3209254" href="/pmc/articles/PMC3209254/"><img src="/pmc/articles/instance/3209254/bin/zmb9991092790009.gif" src-large="/pmc/articles/instance/3209254/bin/zmb9991092790009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3209254/" ref="ordinalpos=4729&amp;ncbi_uid=3482797&amp;link_uid=PMC3209254" image-link="/pmc/articles/PMC3209254/figure/F9/" class="imagepopup">Fig. 9. From: Distinct Functions of Sox2 Control Self-Renewal and Differentiation in the Osteoblast Lineage   . </a></p><div class="supp"><p class="details"><div>Model for the multiple mechanisms by which Sox2 regulates the osteoblast lineage. Sox2 maintains stemness and self-renewal by transcriptional mechanisms, fostering expression of genes such as <i>Bmi-1</i> that are necessary for the self-renewal of osteoprogenitor cells. Sox2 also regulates transcription of genes in the Wnt <span class="highlight" style="background-color:">pathway</span> (e.g., APC, GSK3β, and Fzd receptor genes) whose upregulation or downregulation would inhibit Wnt <span class="highlight" style="background-color:">signaling</span>. Additionally, Sox2 downregulates canonical Wnt <span class="highlight" style="background-color:">signaling</span> by interacting through its C-terminal activation domain (R1) with β-catenin, thus inhibiting the prodifferentiation Wnt <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Eunjeong Seo, et al. Mol Cell Biol. 2011 November;31(22):4593-4608.</div><p class="links"><a class="dblinks" href="/pubmed/21930787" ref="ordinalpos=4729&amp;ncbi_uid=3482797&amp;link_uid=21930787">Citation</a><a class="dblinks" href="/pmc/articles/PMC3209254/" ref="ordinalpos=4729&amp;ncbi_uid=3482797&amp;link_uid=PMC3209254">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4730.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2940738/figure/pgen-1001125-g007/" ref="ordinalpos=4730&amp;ncbi_uid=2474329&amp;link_uid=PMC2940738" href="/pmc/articles/PMC2940738/"><img src="/pmc/articles/instance/2940738/bin/pgen.1001125.g007.gif" src-large="/pmc/articles/instance/2940738/bin/pgen.1001125.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2940738/" ref="ordinalpos=4730&amp;ncbi_uid=2474329&amp;link_uid=PMC2940738" image-link="/pmc/articles/PMC2940738/figure/pgen-1001125-g007/" class="imagepopup">Figure 7. Model for ETS-4 function as a transcriptional regulator..  From: ETS-4 Is a Transcriptional Regulator of Life Span in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>ETS-4 functions in parallel to the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (blue) to modulate a set of downstream targets (red circle). <i>ets-4</i> may function upstream of (dotted line) or in parallel (solid line) to antagonize <i>daf-16</i> function in longevity regulation. ETS-4-regulated gene set overlaps with that of the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> acting through DAF-16 (blue circle) as well as that of ELT-3 (green circle), which directs the age-regulation of a majority of genes that change expression with age.</div></p></div><div class="aux"><div class="resc">Bargavi Thyagarajan, et al. PLoS Genet. 2010 September;6(9):e1001125.</div><p class="links"><a class="dblinks" href="/pubmed/20862312" ref="ordinalpos=4730&amp;ncbi_uid=2474329&amp;link_uid=20862312">Citation</a><a class="dblinks" href="/pmc/articles/PMC2940738/" ref="ordinalpos=4730&amp;ncbi_uid=2474329&amp;link_uid=PMC2940738">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4731.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2926953/figure/F4/" ref="ordinalpos=4731&amp;ncbi_uid=3213531&amp;link_uid=PMC2926953" href="/pmc/articles/PMC2926953/"><img src="/pmc/articles/instance/2926953/bin/DEV051565F4.gif" src-large="/pmc/articles/instance/2926953/bin/DEV051565F4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2926953/" ref="ordinalpos=4731&amp;ncbi_uid=3213531&amp;link_uid=PMC2926953" image-link="/pmc/articles/PMC2926953/figure/F4/" class="imagepopup">Fig. 4. From: Arterial pole progenitors interpret opposing FGF/BMP signals to proliferate or differentiate. </a></p><div class="supp"><p class="details"><div><b>Dissecting the role of the FGF intracellular <span class="highlight" style="background-color:">signaling</span> pathways in SHF explants grown for 24 hours.</b> (<b>A</b>,<b>B</b>) Inhibiting of the Ras/Erk <span class="highlight" style="background-color:">pathway</span> using the MEK inhibitor had no effect on proliferation (A) but significantly increased myocardial differentiation (B) compared with controls (B,*<i>P</i>=0.03). Inhibition of the AKT <span class="highlight" style="background-color:">pathway</span> with the PI3K inhibitor decreased proliferation (A,*<i>P</i>=0.001) but had no effect on differentiation. Inhibition of the PKC <span class="highlight" style="background-color:">pathway</span> with the PLCγ inhibitor decreased proliferation (A, *<i>P</i>=0.001) but had no effect on differentiation. Blocking the FGF <span class="highlight" style="background-color:">signaling</span> with SU5402 decreased proliferation (A, *<i>P</i>=0.003) in the same way as the PI3K and PLC-γ inhibitors, and increased differentiation (B; *<i>P</i>=0.02) in the same way as the MEK inhibitor. (<b>C</b>,<b>D</b>) Treatment with low dose BMP2 (25 ng/ml) plus the MEK inhibitor did not affect proliferation. Combining BMP2 and the Mek inhibitor significantly increased myocardial differentiation compared with controls (*<i>P</i>=0.001) but was not significantly increased compared with BMP alone.</div></p></div><div class="aux"><div class="resc">Mary Redmond Hutson, et al. Development. 2010 September 15;137(18):3001-3011.</div><p class="links"><a class="dblinks" href="/pubmed/20702561" ref="ordinalpos=4731&amp;ncbi_uid=3213531&amp;link_uid=20702561">Citation</a><a class="dblinks" href="/pmc/articles/PMC2926953/" ref="ordinalpos=4731&amp;ncbi_uid=3213531&amp;link_uid=PMC2926953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4732.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3128061/figure/pone-0021574-g010/" ref="ordinalpos=4732&amp;ncbi_uid=3078292&amp;link_uid=PMC3128061" href="/pmc/articles/PMC3128061/"><img src="/pmc/articles/instance/3128061/bin/pone.0021574.g010.gif" src-large="/pmc/articles/instance/3128061/bin/pone.0021574.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3128061/" ref="ordinalpos=4732&amp;ncbi_uid=3078292&amp;link_uid=PMC3128061" image-link="/pmc/articles/PMC3128061/figure/pone-0021574-g010/" class="imagepopup">Figure 10. Summary of pathways involved in LH-induced EGFR transactivation and downstream signals involved in oocyte maturation..  From: Genetic Dissection of Epidermal Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span> during Luteinizing Hormone-Induced Oocyte Maturation. </a></p><div class="supp"><p class="details"><div>The broken arrow indicates possible stimulatory pathways that signal maturation. The consequences of Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> are presently unknown. Although AKT is involved in cell survival, the function in granulosa cells is still unclear. An inhibitory <span class="highlight" style="background-color:">pathway</span> indicated by a question mark and leading to decreased cGMP, is inferred from the data shown in Figure 9.</div></p></div><div class="aux"><div class="resc">Minnie Hsieh, et al. PLoS One. 2011;6(6):e21574.</div><p class="links"><a class="dblinks" href="/pubmed/21738714" ref="ordinalpos=4732&amp;ncbi_uid=3078292&amp;link_uid=21738714">Citation</a><a class="dblinks" href="/pmc/articles/PMC3128061/" ref="ordinalpos=4732&amp;ncbi_uid=3078292&amp;link_uid=PMC3128061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4733.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2735973/figure/fig8/" ref="ordinalpos=4733&amp;ncbi_uid=2447860&amp;link_uid=PMC2735973" href="/pmc/articles/PMC2735973/"><img src="/pmc/articles/instance/2735973/bin/pp1510275f08.gif" src-large="/pmc/articles/instance/2735973/bin/pp1510275f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2735973/" ref="ordinalpos=4733&amp;ncbi_uid=2447860&amp;link_uid=PMC2735973" image-link="/pmc/articles/PMC2735973/figure/fig8/" class="imagepopup">Figure 8. From: The MYB96 Transcription Factor Mediates Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> during Drought Stress Response in Arabidopsis. </a></p><div class="supp"><p class="details"><div><i>MYB96</i> mediates ABA signals via <i>RD22</i> in regulating drought stress response. The <i>MYB96</i>-mediated ABA signals also induce <i>GH3</i> genes to regulate lateral root development under drought conditions. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is probably linked to the <i>GH3</i>-mediated negative feedback loop maintaining auxin homeostasis (Park et al., 2007). The solid arrows indicate ABA signal transduction, and the dotted arrows indicate auxin signal transduction.</div></p></div><div class="aux"><div class="resc">Pil Joon Seo, et al. Plant Physiol. 2009 September;151(1):275-289.</div><p class="links"><a class="dblinks" href="/pubmed/19625633" ref="ordinalpos=4733&amp;ncbi_uid=2447860&amp;link_uid=19625633">Citation</a><a class="dblinks" href="/pmc/articles/PMC2735973/" ref="ordinalpos=4733&amp;ncbi_uid=2447860&amp;link_uid=PMC2735973">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4734.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2835321/figure/F5/" ref="ordinalpos=4734&amp;ncbi_uid=2852782&amp;link_uid=PMC2835321" href="/pmc/articles/PMC2835321/"><img src="/pmc/articles/instance/2835321/bin/DEV038711F5.gif" src-large="/pmc/articles/instance/2835321/bin/DEV038711F5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2835321/" ref="ordinalpos=4734&amp;ncbi_uid=2852782&amp;link_uid=PMC2835321" image-link="/pmc/articles/PMC2835321/figure/F5/" class="imagepopup">Fig. 5. From: To build a synapse: <span class="highlight" style="background-color:">signaling</span> pathways in neuromuscular junction assembly. </a></p><div class="supp"><p class="details"><div><b>A working model of <span class="highlight" style="background-color:">signaling</span> pathways in postsynaptic assembly.</b> (<b>A</b>) Prior to innervation, Wnt interacts with Musk to activate the Wnt-Musk pathways necessary for forming aneural clusters and for guiding motor growth cones to the middle region of muscle fibers. At the same time, Wnt might also activate Wnt canonical and non-canonical pathways to regulate Musk-dependent pathways. (<b>B</b>) Upon innervation, neural agrin binds to Lrp4 and activates Musk to initiate the agrin-Lrp4-Musk <span class="highlight" style="background-color:">pathway</span> (see Fig. 3) for AChR clustering at the synapse. Wnt might regulate agrin <span class="highlight" style="background-color:">signaling</span> by directly binding to Musk or Lrp5/Lrp6-Frizzled. Dashed lines indicate pathways that remain to be determined.</div></p></div><div class="aux"><div class="resc">Haitao Wu, et al. Development. 2010 April 1;137(7):1017-1033.</div><p class="links"><a class="dblinks" href="/pubmed/20215342" ref="ordinalpos=4734&amp;ncbi_uid=2852782&amp;link_uid=20215342">Citation</a><a class="dblinks" href="/pmc/articles/PMC2835321/" ref="ordinalpos=4734&amp;ncbi_uid=2852782&amp;link_uid=PMC2835321">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4735.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4275227/figure/pone-0115383-g002/" ref="ordinalpos=4735&amp;ncbi_uid=7006588&amp;link_uid=PMC4275227" href="/pmc/articles/PMC4275227/"><img src="/pmc/articles/instance/4275227/bin/pone.0115383.g002.gif" src-large="/pmc/articles/instance/4275227/bin/pone.0115383.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4275227/" ref="ordinalpos=4735&amp;ncbi_uid=7006588&amp;link_uid=PMC4275227" image-link="/pmc/articles/PMC4275227/figure/pone-0115383-g002/" class="imagepopup">Figure 2. Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis indicates canonical <span class="highlight" style="background-color:">signaling</span> pathways involved in CCA..  From: Mutation Profiling in Cholangiocarcinoma: Prognostic and Therapeutic Implications. </a></p><div class="supp"><p class="details"><div>Homology noted with more commonly occurring solid tumors including melanoma and glioblastoma.</div></p></div><div class="aux"><div class="resc">Chaitanya R. Churi, et al. PLoS One. 2014;9(12):e115383.</div><p class="links"><a class="dblinks" href="/pubmed/25536104" ref="ordinalpos=4735&amp;ncbi_uid=7006588&amp;link_uid=25536104">Citation</a><a class="dblinks" href="/pmc/articles/PMC4275227/" ref="ordinalpos=4735&amp;ncbi_uid=7006588&amp;link_uid=PMC4275227">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4736.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4070412/figure/F5/" ref="ordinalpos=4736&amp;ncbi_uid=6357215&amp;link_uid=PMC4070412" href="/pmc/articles/PMC4070412/"><img src="/pmc/articles/instance/4070412/bin/fpls-05-00284-g005.gif" src-large="/pmc/articles/instance/4070412/bin/fpls-05-00284-g005.jpg" alt="FIGURE 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4070412/" ref="ordinalpos=4736&amp;ncbi_uid=6357215&amp;link_uid=PMC4070412" image-link="/pmc/articles/PMC4070412/figure/F5/" class="imagepopup">FIGURE 5.  From: Evolution of the fruit endocarp: molecular mechanisms underlying adaptations in seed protection and dispersal strategies. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating dehiscence in <i>Arabidopsis</i>.</b> Model depicting the hierarchy of transcription factors that control tissue patterning in the valve margin and replum.</div></p></div><div class="aux"><div class="resc">Chris Dardick, et al. Front Plant Sci. 2014;5:284.</div><p class="links"><a class="dblinks" href="/pubmed/25009543" ref="ordinalpos=4736&amp;ncbi_uid=6357215&amp;link_uid=25009543">Citation</a><a class="dblinks" href="/pmc/articles/PMC4070412/" ref="ordinalpos=4736&amp;ncbi_uid=6357215&amp;link_uid=PMC4070412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4737.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2869339/figure/F9/" ref="ordinalpos=4737&amp;ncbi_uid=2356980&amp;link_uid=PMC2869339" href="/pmc/articles/PMC2869339/"><img src="/pmc/articles/instance/2869339/bin/1710-1492-4-1-37-9.gif" src-large="/pmc/articles/instance/2869339/bin/1710-1492-4-1-37-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2869339/" ref="ordinalpos=4737&amp;ncbi_uid=2356980&amp;link_uid=PMC2869339" image-link="/pmc/articles/PMC2869339/figure/F9/" class="imagepopup">Figure 9.  From: Epigenetics, Behaviour, and Health. </a></p><div class="supp"><p class="details"><div><b>Behavioural gene programming</b>. Maternal care elicits a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hippocampal neurons, leading to epigenetic reprogramming of the glucocorticoid receptor exon 1<sub>7 </sub>promoter.</div></p></div><div class="aux"><div class="resc">Moshe Szyf, et al. Allergy Asthma Clin Immunol. 2008;4(1):37-49.</div><p class="links"><a class="dblinks" href="/pubmed/20525124" ref="ordinalpos=4737&amp;ncbi_uid=2356980&amp;link_uid=20525124">Citation</a><a class="dblinks" href="/pmc/articles/PMC2869339/" ref="ordinalpos=4737&amp;ncbi_uid=2356980&amp;link_uid=PMC2869339">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4738.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2835958/figure/F2/" ref="ordinalpos=4738&amp;ncbi_uid=2217063&amp;link_uid=PMC2835958" href="/pmc/articles/PMC2835958/"><img src="/pmc/articles/instance/2835958/bin/psb0501_0053_fig002.gif" src-large="/pmc/articles/instance/2835958/bin/psb0501_0053_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2835958/" ref="ordinalpos=4738&amp;ncbi_uid=2217063&amp;link_uid=PMC2835958" image-link="/pmc/articles/PMC2835958/figure/F2/" class="imagepopup">Figure 2.  From: Arabidopsis SUMO E3 ligase AtMMS21 regulates root meristem development. </a></p><div class="supp"><p class="details"><div>Proposed model of AtMM S21 function in root meristem development. AtMM S21 acts as a node element of cytokinin and auxin interaction. It plays role positively in the PLTs-mediated auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to promote the root meristem cell division on one hand.18 AtMM S21 is inhibited by cytokinin to modulate the root meristem cell differenciation on the other hand.17 AtMM S21 also may stimulate cytokinin <span class="highlight" style="background-color:">signaling</span> by inducing the expression of <i>CRE1</i> and <i>ARRs</i> genes to sustaining the homeostasis between cytokinin and auxin <span class="highlight" style="background-color:">signaling</span> in the progress of root meristem development.</div></p></div><div class="aux"><div class="resc">Shengchun Zhang, et al. Plant Signal Behav. 2010 January;5(1):53-55.</div><p class="links"><a class="dblinks" href="/pubmed/20592809" ref="ordinalpos=4738&amp;ncbi_uid=2217063&amp;link_uid=20592809">Citation</a><a class="dblinks" href="/pmc/articles/PMC2835958/" ref="ordinalpos=4738&amp;ncbi_uid=2217063&amp;link_uid=PMC2835958">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4739.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4152115/figure/fig04/" ref="ordinalpos=4739&amp;ncbi_uid=6561223&amp;link_uid=PMC4152115" href="/pmc/articles/PMC4152115/"><img src="/pmc/articles/instance/4152115/bin/ijcep0007-5369-f4.gif" src-large="/pmc/articles/instance/4152115/bin/ijcep0007-5369-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4152115/" ref="ordinalpos=4739&amp;ncbi_uid=6561223&amp;link_uid=PMC4152115" image-link="/pmc/articles/PMC4152115/figure/fig04/" class="imagepopup">Figure 4.  From: Cerium promotes bone marrow stromal cells migration and osteogenic differentiation via Smad1/5/8 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Ce promotes the expression of BMP2 and activates the Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A. qRT-PCR analysis indicated Ce increased the mRNA expression of BMP2 in BMSCs. B. Western bolt analysis showed Ce cause a time-dependent increase of Smad1/5/8 phosphorylation. C. BMSCs treated with Ce for 30 minutes led to the preferential nuclear localization of p-Smad1/5/8. D. Western blot analysis indicated that p-Smad1/5/8 was decreased in LDN-193189 pretreated BMSCs. E. qRT-PCR analysis indicated pre-treatment with LDN-193189 significantly blocked the Ce-mediated upregulation of SDF-1 and Runx2 mRNA expression, but not influence the expression of BMP2 mRNA. Data are presented as mean ± SD from a representative of three separate experiments. *<i>P</i>&lt; 0.05.</div></p></div><div class="aux"><div class="resc">Ying Hu, et al. Int J Clin Exp Pathol. 2014;7(8):5369-5378.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4739&amp;ncbi_uid=6561223&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4152115/" ref="ordinalpos=4739&amp;ncbi_uid=6561223&amp;link_uid=PMC4152115">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4740.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711585/figure/pone-0068987-g003/" ref="ordinalpos=4740&amp;ncbi_uid=5113406&amp;link_uid=PMC3711585" href="/pmc/articles/PMC3711585/"><img src="/pmc/articles/instance/3711585/bin/pone.0068987.g003.gif" src-large="/pmc/articles/instance/3711585/bin/pone.0068987.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711585/" ref="ordinalpos=4740&amp;ncbi_uid=5113406&amp;link_uid=PMC3711585" image-link="/pmc/articles/PMC3711585/figure/pone-0068987-g003/" class="imagepopup">Figure 3. Expression of Notch1 and RBP-Jκ in VSMCs..  From: Osteoprotegerin Inhibits Calcification of Vascular Smooth Muscle Cell via Down Regulation of the Notch1-RBP-J?/Msx2 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) mRNA level of Notch1 and RBP-Jκ were measured by real-time RT-PCR in VSMCs cultured in different media. (B) Protein level of Notch1 and RBP-Jκ were measured by Western blotting in VSMCs cultured in different media. Notch1 and RBP-Jκ significantly increased by 3 to 4 folds in VSMC cultured in osteogenic medium compared to control at both mRNA and protein levels, suggesting that the Notch1-RBP-Jκ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated in osteogenic conversion of VSMCs. DAPT reduced the expression of Notch1 by 50% and RBP-Jκ by 30% in VSMCs compared to osteogenic cells at bothe mRNA and protein levels. Similarly, OPG reduced the expression of Notch1 and RBP-Jκ compared to VSMCs in the osteogenic medium in a dose dependent manner. *, P&lt;0.05 vs control, #, P&lt;0.05 vs osteogenic medium, and &amp;, P&lt;0.05 vs OPG 0.1ng/ml. n=3.</div></p></div><div class="aux"><div class="resc">Shaoqiong Zhou, et al. PLoS One. 2013;8(7):e68987.</div><p class="links"><a class="dblinks" href="/pubmed/23874840" ref="ordinalpos=4740&amp;ncbi_uid=5113406&amp;link_uid=23874840">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711585/" ref="ordinalpos=4740&amp;ncbi_uid=5113406&amp;link_uid=PMC3711585">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4741.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3734838/figure/fig5/" ref="ordinalpos=4741&amp;ncbi_uid=5171606&amp;link_uid=PMC3734838" href="/pmc/articles/PMC3734838/"><img src="/pmc/articles/instance/3734838/bin/cddis20133f5.gif" src-large="/pmc/articles/instance/3734838/bin/cddis20133f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3734838/" ref="ordinalpos=4741&amp;ncbi_uid=5171606&amp;link_uid=PMC3734838" image-link="/pmc/articles/PMC3734838/figure/fig5/" class="imagepopup">Figure 5.  From: PTEN-regulated AKT/FoxO3a/Bim <span class="highlight" style="background-color:">signaling</span> contributes to reactive oxygen species-mediated apoptosis in selenite-treated colorectal cancer cells. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) ROS were involved in selenite-mediated regulation of the AKT/FoxO3a/Bim <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HCT116 and SW480 CRC cells. Cells samples were pretreated with MnTmPyP or Tiron solution and then were treated with selenite or PBS for 24 h. Cell lysates were analyzed by western blot for the molecules shown in the figures. (<b>b</b>) Western blot analysis of proteins extracted from tumor tissues using antibodies against critical molecules as indicated. (<b>c</b>) Selenite-mediated regulation of the PTEN/AKT/FoxO3a/Bim <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <i>in vivo</i>. Tumor tissues from a colon xenograft animal model were subjected to immunohistochemistry analysis using antibodies recognizing p-AKT (Thr308), AKT, p-FoxO3a (Ser253), FoxO3a, PTEN and Bim. Representative images from both control and selenite-treated samples are shown</div></p></div><div class="aux"><div class="resc">H Luo, et al. Cell Death Dis. 2013 February;4(2):e481.</div><p class="links"><a class="dblinks" href="/pubmed/23392169" ref="ordinalpos=4741&amp;ncbi_uid=5171606&amp;link_uid=23392169">Citation</a><a class="dblinks" href="/pmc/articles/PMC3734838/" ref="ordinalpos=4741&amp;ncbi_uid=5171606&amp;link_uid=PMC3734838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4742.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2861677/figure/F4/" ref="ordinalpos=4742&amp;ncbi_uid=2308437&amp;link_uid=PMC2861677" href="/pmc/articles/PMC2861677/"><img src="/pmc/articles/instance/2861677/bin/1471-2164-11-197-4.gif" src-large="/pmc/articles/instance/2861677/bin/1471-2164-11-197-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2861677/" ref="ordinalpos=4742&amp;ncbi_uid=2308437&amp;link_uid=PMC2861677" image-link="/pmc/articles/PMC2861677/figure/F4/" class="imagepopup">Figure 4.  From: Reciprocal regulation of metabolic and <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b>Patterns of <span class="highlight" style="background-color:">pathway</span> regulation</b>. <b>(A) </b>A schematic of reciprocal correlation of metabolic and <span class="highlight" style="background-color:">signaling</span> pathways in mammalian transcription. <b>(B) </b>Relation of major KEGG pathways to OXPHOS. The directional regulation of 14 major KEGG pathways (number of up- <i>minus </i>down-regulated genes in a given KEGG <span class="highlight" style="background-color:">pathway</span> normalized to the total number of regulated genes within a study) was color-coded with yellow and blue representing low and high expression of the pathways, respectively. The cellular pathways of "protein export", "cell cycle" and ubiquitin-mediated proteolysis" were positively correlated with OXPHOS, while "calcium-mediated <span class="highlight" style="background-color:">signaling</span>", and structural components important for cell-cell contact (e.g. "cell adhesion molecules", "tight junctions", "gap junctions", "adherens junctions") were negatively correlated with OXPHOS.</div></p></div><div class="aux"><div class="resc">Andreas S Barth, et al. BMC Genomics. 2010;11:197-197.</div><p class="links"><a class="dblinks" href="/pubmed/20334672" ref="ordinalpos=4742&amp;ncbi_uid=2308437&amp;link_uid=20334672">Citation</a><a class="dblinks" href="/pmc/articles/PMC2861677/" ref="ordinalpos=4742&amp;ncbi_uid=2308437&amp;link_uid=PMC2861677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4743.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3762816/figure/pone-0072699-g003/" ref="ordinalpos=4743&amp;ncbi_uid=5290434&amp;link_uid=PMC3762816" href="/pmc/articles/PMC3762816/"><img src="/pmc/articles/instance/3762816/bin/pone.0072699.g003.gif" src-large="/pmc/articles/instance/3762816/bin/pone.0072699.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3762816/" ref="ordinalpos=4743&amp;ncbi_uid=5290434&amp;link_uid=PMC3762816" image-link="/pmc/articles/PMC3762816/figure/pone-0072699-g003/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Pathway</span> analysis on deregulated miRNAs in melanoma showed possible combinatorial effects on predicted gene pathways..  From: In-Depth Characterization of microRNA Transcriptome in Melanoma. </a></p><div class="supp"><p class="details"><div>Interrogating the gene targets for down-regulated miRNAs in PCM vs. NS (A) and in PCM vs. CN (B) showed remarkable similarities in the pathways perturbed. The cutoff <i>P</i>-value was set at &lt;0.00095. Common KEGG pathways shared between the two analyses are shown in the same color. Mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showed that several down-regulated miRNAs could converge on the same putative gene targets (C). The predicted miRNA targets are highlighted in yellow.</div></p></div><div class="aux"><div class="resc">James Kozubek, et al. PLoS One. 2013;8(9):e72699.</div><p class="links"><a class="dblinks" href="/pubmed/24023765" ref="ordinalpos=4743&amp;ncbi_uid=5290434&amp;link_uid=24023765">Citation</a><a class="dblinks" href="/pmc/articles/PMC3762816/" ref="ordinalpos=4743&amp;ncbi_uid=5290434&amp;link_uid=PMC3762816">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4744.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3691323/figure/pone-0066799-g006/" ref="ordinalpos=4744&amp;ncbi_uid=5061321&amp;link_uid=PMC3691323" href="/pmc/articles/PMC3691323/"><img src="/pmc/articles/instance/3691323/bin/pone.0066799.g006.gif" src-large="/pmc/articles/instance/3691323/bin/pone.0066799.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3691323/" ref="ordinalpos=4744&amp;ncbi_uid=5061321&amp;link_uid=PMC3691323" image-link="/pmc/articles/PMC3691323/figure/pone-0066799-g006/" class="imagepopup">Figure 6. The G2/M transition is the major <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> altered by Hltf deletion..  From: Role of Helicase-Like Transcription Factor (Hltf) in the G2/M Transition and Apoptosis in Brain. </a></p><div class="supp"><p class="details"><div>Significant decreases (red thermometers = negative effects) in transcript expression of major components of the G2/M transition in Hltf null brain are superimposed on the proprietary multicomponent canonical <span class="highlight" style="background-color:">pathway</span> map from the MetaCore™ database [Straight lines = interactions; Symbols = events; +P = phosphorylation; B = binding; GR = group relation; CS = complex subunit. Colors for lines and symbols are green for positive, and gray for unspecified].</div></p></div><div class="aux"><div class="resc">Rebecca A. Helmer, et al. PLoS One. 2013;8(6):e66799.</div><p class="links"><a class="dblinks" href="/pubmed/23826137" ref="ordinalpos=4744&amp;ncbi_uid=5061321&amp;link_uid=23826137">Citation</a><a class="dblinks" href="/pmc/articles/PMC3691323/" ref="ordinalpos=4744&amp;ncbi_uid=5061321&amp;link_uid=PMC3691323">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4745.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4017039/figure/F1/" ref="ordinalpos=4745&amp;ncbi_uid=6135972&amp;link_uid=PMC4017039" href="/pmc/articles/PMC4017039/"><img src="/pmc/articles/instance/4017039/bin/WJG-20-5244-g001.gif" src-large="/pmc/articles/instance/4017039/bin/WJG-20-5244-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4017039/" ref="ordinalpos=4745&amp;ncbi_uid=6135972&amp;link_uid=PMC4017039" image-link="/pmc/articles/PMC4017039/figure/F1/" class="imagepopup">Figure 1.  From: Novel role of toll-like receptors in Helicobacter pylori - induced gastric malignancy. </a></p><div class="supp"><p class="details"><div>Intracellular signal propagation of human toll-like receptors and corresponding ligands in innate immunity. Toll-like receptors (TLRs) and ligands in human are revealed in the figure. Individual TLRs recognize specific (pathogen-associated molecular patterns, PAMPs) or damage-associated molecular patterns (DAMPs) of corresponding ligands. TLR <span class="highlight" style="background-color:">signaling</span> is propagated by activation of its cytoplasmic TIR (Toll/IL-1R) domain and cooperation with various adaptor molecules, such as myeloid differentiation factor 88 (MyD-88), toll-interleukin 1 receptor (TIR) domain-containing adapter protein (TIRAP), toll interacting protein (TOLLIP), (IRAK), (TRAF), TIR domain-containing adapter inducing interferon (IFN)-beta (TRIF), and TRIF-related adapter molecule (TRAM). TLR <span class="highlight" style="background-color:">signaling</span> consists of two distinct pathways, MyD-88-dependent and MyD-88-independent: (1) MyD-88-dependent <span class="highlight" style="background-color:">pathway</span>: The MyD88 dependent <span class="highlight" style="background-color:">pathway</span> is down-stream of TLR1, TLR2, TLR4, TLR5, TLR6, TLR7 and TLR9. This <span class="highlight" style="background-color:">pathway</span> leads to the production of proinflammatory cytokines and is triggered by the association of activated TIR domain and MyD-88, recruitment of IRAK1, IRAK4 and TRAF6 to the TLR-MyD-88 complex and consequent phosphorylation of IRAK1 and TRAF6. The signal is propagated by this phosphorylated adaptor molecular complex to the down-stream MAP kinases-AP1 and IKK complex-NF-κB; (2) MyD88-independent <span class="highlight" style="background-color:">pathway</span>: This is associated with the induction of INF-beta mediated by TLR3 or TLR4 activation. Intracellular <span class="highlight" style="background-color:">signaling</span> <i>via</i> the MyD-88 independent <span class="highlight" style="background-color:">pathway</span> is propagated by the action of TRIF and TRAM as adaptor molecules. The signal consequently activates the IKK <span class="highlight" style="background-color:">pathway</span> to produce INF-beta.</div></p></div><div class="aux"><div class="resc">Kaname Uno, et al. World J Gastroenterol. 2014 May 14;20(18):5244-5251.</div><p class="links"><a class="dblinks" href="/pubmed/24833854" ref="ordinalpos=4745&amp;ncbi_uid=6135972&amp;link_uid=24833854">Citation</a><a class="dblinks" href="/pmc/articles/PMC4017039/" ref="ordinalpos=4745&amp;ncbi_uid=6135972&amp;link_uid=PMC4017039">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4746.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3218933/figure/F3/" ref="ordinalpos=4746&amp;ncbi_uid=3517557&amp;link_uid=PMC3218933" href="/pmc/articles/PMC3218933/"><img src="/pmc/articles/instance/3218933/bin/bcr2876-3.gif" src-large="/pmc/articles/instance/3218933/bin/bcr2876-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3218933/" ref="ordinalpos=4746&amp;ncbi_uid=3517557&amp;link_uid=PMC3218933" image-link="/pmc/articles/PMC3218933/figure/F3/" class="imagepopup">Figure 3.  From: Breast cancer growth and metastasis: interplay between cancer stem cells, embryonic <span class="highlight" style="background-color:">signaling</span> pathways and epithelial-to-mesenchymal transition. </a></p><div class="supp"><p class="details"><div><b>Embryonic <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> leads to induction of epithelial-to-mesenchymal transition</b>. Hedgehog (Hh), Notch, Wnt, and transforming growth factor (TGF)-β <span class="highlight" style="background-color:">signaling</span> can activate epithelial-to-mesenchymal transition (EMT) regulators to induce phenotypic changes through a variety of <span class="highlight" style="background-color:">signaling</span> intermediates. EMT-inducing signals by Snail, Slug, KLF8, Twist, Goosecoid, Foxc1 and -2, or Zeb1 and -2 drive non-mobile epithelial-like cells to acquire more invasive phenotypes. By migrating into the circulatory system, cancer stem cells can translocate to new locations and initiate new metastatic growths. EGF, epidermal growth factor; Fzd, Frizzled; Hh, Hedgehog; HIF, hypoxia-inducible factor; NICD, Notch intracellular domain; Smo, Smoothened; TGF, transforming growth factor.</div></p></div><div class="aux"><div class="resc">Naoko Takebe, et al. Breast Cancer Res. 2011;13(3):211-211.</div><p class="links"><a class="dblinks" href="/pubmed/21672282" ref="ordinalpos=4746&amp;ncbi_uid=3517557&amp;link_uid=21672282">Citation</a><a class="dblinks" href="/pmc/articles/PMC3218933/" ref="ordinalpos=4746&amp;ncbi_uid=3517557&amp;link_uid=PMC3218933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4747.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3397862/figure/F7/" ref="ordinalpos=4747&amp;ncbi_uid=5087641&amp;link_uid=PMC3397862" href="/pmc/articles/PMC3397862/"><img src="/pmc/articles/instance/3397862/bin/zbc0311216140007.gif" src-large="/pmc/articles/instance/3397862/bin/zbc0311216140007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3397862/" ref="ordinalpos=4747&amp;ncbi_uid=5087641&amp;link_uid=PMC3397862" image-link="/pmc/articles/PMC3397862/figure/F7/" class="imagepopup">FIGURE 7. From: Lewis X-carrying N-Glycans Regulate the Proliferation of Mouse Embryonic Neural Stem Cells via the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic representation for the up-regulating mechanism of NSC proliferation by modulating Notch <span class="highlight" style="background-color:">signaling</span> with Lewis X-containing glycans in NSCs.</b></div></p></div><div class="aux"><div class="resc">Hirokazu Yagi, et al. J Biol Chem. 2012 July 13;287(29):24356-24364.</div><p class="links"><a class="dblinks" href="/pubmed/22645129" ref="ordinalpos=4747&amp;ncbi_uid=5087641&amp;link_uid=22645129">Citation</a><a class="dblinks" href="/pmc/articles/PMC3397862/" ref="ordinalpos=4747&amp;ncbi_uid=5087641&amp;link_uid=PMC3397862">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4748.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4258093/figure/fig05/" ref="ordinalpos=4748&amp;ncbi_uid=6955184&amp;link_uid=PMC4258093" href="/pmc/articles/PMC4258093/"><img src="/pmc/articles/instance/4258093/bin/jcp0229-1312-f5.gif" src-large="/pmc/articles/instance/4258093/bin/jcp0229-1312-f5.jpg" alt="Fig 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4258093/" ref="ordinalpos=4748&amp;ncbi_uid=6955184&amp;link_uid=PMC4258093" image-link="/pmc/articles/PMC4258093/figure/fig05/" class="imagepopup">Fig 5.  From: Signals Controlling Un-Differentiated States in Embryonic Stem and Cancer Cells: Role of the Phosphatidylinositol 3′ Kinase <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>TGF-β <span class="highlight" style="background-color:">signaling</span>. The <span class="highlight" style="background-color:">pathway</span> is activated when aTGF-β superfamily ligand binds to a TGF-β type II receptor dimer, which recruits and phosphorylates a type I receptor dimer forming a hetero-tetrameric complex with the ligand. The type I receptor goes on to phosphorylate R-Smads. Phosphorylated R-SMADs have a high affinity for the effector protein, coSMAD, and consequently forms a complex with coSMAD. R-SMAD/coSMAD complexes accumulate and enter the nucleus where they activate transcription.</div></p></div><div class="aux"><div class="resc">Daniel Voskas, et al. J Cell Physiol. 2014 October;229(10):1312-1322.</div><p class="links"><a class="dblinks" href="/pubmed/24604594" ref="ordinalpos=4748&amp;ncbi_uid=6955184&amp;link_uid=24604594">Citation</a><a class="dblinks" href="/pmc/articles/PMC4258093/" ref="ordinalpos=4748&amp;ncbi_uid=6955184&amp;link_uid=PMC4258093">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4749.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3499080/figure/F2/" ref="ordinalpos=4749&amp;ncbi_uid=5255577&amp;link_uid=PMC3499080" href="/pmc/articles/PMC3499080/"><img src="/pmc/articles/instance/3499080/bin/toxsci_kfs193_f0002.gif" src-large="/pmc/articles/instance/3499080/bin/toxsci_kfs193_f0002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3499080/" ref="ordinalpos=4749&amp;ncbi_uid=5255577&amp;link_uid=PMC3499080" image-link="/pmc/articles/PMC3499080/figure/F2/" class="imagepopup">Fig. 2. From: Elevated Hepatic Iron Activates NF-E2-Related Factor 2-Regulated <span class="highlight" style="background-color:">Pathway</span> in a Dietary Iron Overload Mouse Model. </a></p><div class="supp"><p class="details"><div>Alterations in the NRF2 canonical <span class="highlight" style="background-color:">pathway</span> in livers of mice treated with TMHF. Liver proteins from mice fed a control or TMHF diet mice for 4 weeks (four mice in each group) were labeled and analyzed by iTRAQ. The data from iTRAQ were then analyzed using IPA. A modified schematic of the NRF2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> obtained from IPA is shown. The diagram was reduced in magnitude to include only those categories containing proteins that were identified as upregulated (shades of red) or downregulated (green) by TMHF by more than ± 1.5-fold of the control in this iTRAQ profile. Proteins in white circles were not among the 74 identified proteins.</div></p></div><div class="aux"><div class="resc">Mi Sun Moon, et al. Toxicol Sci. 2012 September;129(1):74-85.</div><p class="links"><a class="dblinks" href="/pubmed/22649188" ref="ordinalpos=4749&amp;ncbi_uid=5255577&amp;link_uid=22649188">Citation</a><a class="dblinks" href="/pmc/articles/PMC3499080/" ref="ordinalpos=4749&amp;ncbi_uid=5255577&amp;link_uid=PMC3499080">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4750.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3790664/figure/F1/" ref="ordinalpos=4750&amp;ncbi_uid=5620228&amp;link_uid=PMC3790664" href="/pmc/articles/PMC3790664/"><img src="/pmc/articles/instance/3790664/bin/nihms430584f1.gif" src-large="/pmc/articles/instance/3790664/bin/nihms430584f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3790664/" ref="ordinalpos=4750&amp;ncbi_uid=5620228&amp;link_uid=PMC3790664" image-link="/pmc/articles/PMC3790664/figure/F1/" class="imagepopup">Figure 1.  From: Dynamics and precision in retinoic acid morphogen gradients. </a></p><div class="supp"><p class="details"><div>Comparing feedback mechanisms in morphogen <span class="highlight" style="background-color:">signaling</span>. (A) A cell responding to a typical polypeptide morphogen (purple ovals). Green dashed arrows point to extracellular inhibitory factors induced by the signal. Red lines denote intracellular mechanisms of repression. (B) An RA-producing cell showing the biosynthetic <span class="highlight" style="background-color:">pathway</span> for RA. Retinyl esters and beta-carotene are important stores of RA precursor. Green dashed arrows point to components of the <span class="highlight" style="background-color:">pathway</span> induced by RA <span class="highlight" style="background-color:">signaling</span>. Red lines denote repression of adhs and aldh1as by RA. (C) Polypeptide morphogens typically induce their own receptors, which are subsequently degraded through receptor-mediated endocytosis. (D) An RA-responding cell showing <span class="highlight" style="background-color:">signaling</span> and catabolism. RA (red circles) enters cells and binds Crabps (blue crescents), which transport it either to RARs (blue and green ovals) in the nucleus (orange) for <span class="highlight" style="background-color:">signaling</span> or to Cyp26s (black hexagons) for degradation. Adhs, alcohol dehydrogenases; aldh1as, aldehyde dehydrogenase 1as.</div></p></div><div class="aux"><div class="resc">Thomas F. Schilling, et al. Curr Opin Genet Dev. 2012 December;22(6):562-569.</div><p class="links"><a class="dblinks" href="/pubmed/23266215" ref="ordinalpos=4750&amp;ncbi_uid=5620228&amp;link_uid=23266215">Citation</a><a class="dblinks" href="/pmc/articles/PMC3790664/" ref="ordinalpos=4750&amp;ncbi_uid=5620228&amp;link_uid=PMC3790664">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4751.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4110025/figure/pone-0103440-g001/" ref="ordinalpos=4751&amp;ncbi_uid=6425535&amp;link_uid=PMC4110025" href="/pmc/articles/PMC4110025/"><img src="/pmc/articles/instance/4110025/bin/pone.0103440.g001.gif" src-large="/pmc/articles/instance/4110025/bin/pone.0103440.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4110025/" ref="ordinalpos=4751&amp;ncbi_uid=6425535&amp;link_uid=PMC4110025" image-link="/pmc/articles/PMC4110025/figure/pone-0103440-g001/" class="imagepopup">Figure 1. Effects of HPV-16 oncoproteins on HIF-1α expression and PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in NSCLC cells..  From: Roles of PI3K/Akt and c-Jun <span class="highlight" style="background-color:">Signaling</span> Pathways in Human Papillomavirus Type 16 Oncoprotein-Induced HIF-1?, VEGF, and IL-8 Expression and In Vitro Angiogenesis in Non-Small Cell Lung Cancer Cells. </a></p><div class="supp"><p class="details"><div>(A) Western blot analysis of HIF-1α protein levels in stable-transfected A549 cells. (B) Real-time PCR analysis of HIF-1α mRNA levels in stable-transfected A549 cells. (C and D) Western blot analysis of p-Akt, p-P70S6K, p-P85S6K, and p-mTOR protein levels in transfected A549 (C) and NCI-H460 (D) cells.</div></p></div><div class="aux"><div class="resc">Erying Zhang, et al. PLoS One. 2014;9(7):e103440.</div><p class="links"><a class="dblinks" href="/pubmed/25058399" ref="ordinalpos=4751&amp;ncbi_uid=6425535&amp;link_uid=25058399">Citation</a><a class="dblinks" href="/pmc/articles/PMC4110025/" ref="ordinalpos=4751&amp;ncbi_uid=6425535&amp;link_uid=PMC4110025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4752.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2953807/figure/F2/" ref="ordinalpos=4752&amp;ncbi_uid=2513299&amp;link_uid=PMC2953807" href="/pmc/articles/PMC2953807/"><img src="/pmc/articles/instance/3157716/bin/oncotarget-01-210-g002.gif" src-large="/pmc/articles/instance/3157716/bin/oncotarget-01-210-g002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2953807/" ref="ordinalpos=4752&amp;ncbi_uid=2513299&amp;link_uid=PMC2953807" image-link="/pmc/articles/PMC2953807/figure/F2/" class="imagepopup">Fig. 2. Downregulation of Jagged1 after knockdown of RBP-jκ, a mediator of the Notch3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, in OVCAR3 cells.  From: Jagged1 Expression Regulated by Notch3 and Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> Pathways in Ovarian Cancer. </a></p><div class="supp"><p class="details"><div>A. Both Jagged1 mRNA level (left panel) and protein level (right panel) decreased after <i>RBP-jκ</i> knockdown in OVCAR3 cells as detected by quantitative RT-PCR and Western blot analyses. B. Cell proliferation significantly decreases in OVCAR3 cells following <i>RBP-jκ</i> knockdown. RBP-jκ shRNA1 and shRNA was co-applied in this study. C. mRNA expression level of Notch3 is not altered by Jagged1 shRNA knockdown in OVCAR3 cells</div></p></div><div class="aux"><div class="resc">Xu Chen, et al. Oncotarget. 2010 July;1(3):210-218.</div><p class="links"><a class="dblinks" href="/pubmed/20953350" ref="ordinalpos=4752&amp;ncbi_uid=2513299&amp;link_uid=20953350">Citation</a><a class="dblinks" href="/pmc/articles/PMC2953807/" ref="ordinalpos=4752&amp;ncbi_uid=2513299&amp;link_uid=PMC2953807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4753.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2700095/figure/F10/" ref="ordinalpos=4753&amp;ncbi_uid=1874001&amp;link_uid=PMC2700095" href="/pmc/articles/PMC2700095/"><img src="/pmc/articles/instance/2700095/bin/1471-213X-9-32-10.gif" src-large="/pmc/articles/instance/2700095/bin/1471-213X-9-32-10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2700095/" ref="ordinalpos=4753&amp;ncbi_uid=1874001&amp;link_uid=PMC2700095" image-link="/pmc/articles/PMC2700095/figure/F10/" class="imagepopup">Figure 10.  From: Salivary gland branching morphogenesis: a quantitative systems analysis of the Eda/Edar/NF?B paradigm. </a></p><div class="supp"><p class="details"><div><b>Alternative kinetic model incorporating unknown transcription factor (TFx) activation by Eda/Edar <span class="highlight" style="background-color:">signaling</span></b>. This model is a diagram of the alternative hypothesis that is more consistent with the quantitative mRNA data for Eda/Edar <span class="highlight" style="background-color:">signaling</span> than the hypothesis that NFκB1 serves as the main <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediating Eda/Edar <span class="highlight" style="background-color:">signaling</span>. Black arrows represent processes (chemical reactions, transport, or binding). Rectangles represent states. A state is a molecule or complex in a physiologic place. The states are color coded as follows: Yellow is the unknown transcription factor TFx and its upstream activation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; purple are the mRNA's; green are the promoters that are upregulated by TFx; red are the promoters that are downregulated by TFx. Places are represented by the background ivory or mauve bands of color and are labeled at the bottom of the diagram. Green and red dashed lines represent, respectively, positive or negative regulation of process by states. Processes with only starts or ends cross the boundary of the modeled system. This diagram was produced by ProcessDB software .</div></p></div><div class="aux"><div class="resc">Michael Melnick, et al. BMC Dev Biol. 2009;9:32-32.</div><p class="links"><a class="dblinks" href="/pubmed/19500387" ref="ordinalpos=4753&amp;ncbi_uid=1874001&amp;link_uid=19500387">Citation</a><a class="dblinks" href="/pmc/articles/PMC2700095/" ref="ordinalpos=4753&amp;ncbi_uid=1874001&amp;link_uid=PMC2700095">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4754.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3335800/figure/pone-0035646-g007/" ref="ordinalpos=4754&amp;ncbi_uid=3970179&amp;link_uid=PMC3335800" href="/pmc/articles/PMC3335800/"><img src="/pmc/articles/instance/3335800/bin/pone.0035646.g007.gif" src-large="/pmc/articles/instance/3335800/bin/pone.0035646.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3335800/" ref="ordinalpos=4754&amp;ncbi_uid=3970179&amp;link_uid=PMC3335800" image-link="/pmc/articles/PMC3335800/figure/pone-0035646-g007/" class="imagepopup">Figure 7. JNK <span class="highlight" style="background-color:">pathway</span> interacts with NFκB and p38 MAPK pathways..  From: MEKK1-MKK4-JNK-AP1 <span class="highlight" style="background-color:">Pathway</span> Negatively Regulates Rgs4 Expression in Colonic Smooth Muscle Cells. </a></p><div class="supp"><p class="details"><div><b><i>A. SP600125 enhances IL-1β-induced activation of canonical IKK2/IκBα/NFκB <span class="highlight" style="background-color:">signaling</span>.</i></b> Cultured muscle cells after serum starvation for 24 h were pretreated with vehicle DMSO or JNK inhibitor SP600125 (10 µM) for 1 h before treatment with or without IL-1β (10 ng/ml) for 15 min. Activation of NFκB <span class="highlight" style="background-color:">signaling</span> was determined by Western blot analysis using indicated specific antibodies. <b><i>B. IL-1β-induced phosphorylation of JNK (Thr183/Tyr185) is blocked by SP600125, enhanced by p38 MAPK inhibitor but not affected by MEK1 inhibitor.</i></b> Cultured and serum-starved muscle cells were pretreated with p38 MAPK inhibitor SB203580 (1 µM) or MEK1 inhibitor PD98059 (20 µM) or JNK inhibitor SP600125 for 1 h before exposure to IL-1β (10 ng/ml) for 15 min. Activation of JNK <span class="highlight" style="background-color:">pathway</span> was determined by Western blot analysis using anti-phospho JNK antibody. The antibodies against GAPDH and β-actin were used for the loading control.</div></p></div><div class="aux"><div class="resc">Yonggang Zhang, et al. PLoS One. 2012;7(4):e35646.</div><p class="links"><a class="dblinks" href="/pubmed/22545125" ref="ordinalpos=4754&amp;ncbi_uid=3970179&amp;link_uid=22545125">Citation</a><a class="dblinks" href="/pmc/articles/PMC3335800/" ref="ordinalpos=4754&amp;ncbi_uid=3970179&amp;link_uid=PMC3335800">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4755.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3338759/figure/pone-0035902-g004/" ref="ordinalpos=4755&amp;ncbi_uid=3978413&amp;link_uid=PMC3338759" href="/pmc/articles/PMC3338759/"><img src="/pmc/articles/instance/3338759/bin/pone.0035902.g004.gif" src-large="/pmc/articles/instance/3338759/bin/pone.0035902.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3338759/" ref="ordinalpos=4755&amp;ncbi_uid=3978413&amp;link_uid=PMC3338759" image-link="/pmc/articles/PMC3338759/figure/pone-0035902-g004/" class="imagepopup">Figure 4. Effect of IFN-λ3 on the key elements in JAK-STAT <span class="highlight" style="background-color:">pathway</span> in HIV infected macrophages..  From: IFN-?3 Inhibits HIV Infection of Macrophages through the JAK-STAT <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Seven-day-cultured macrophages were infected with HIV Bal strain for 2 h. IFN-λ3 (100 ng/ml) was added to the cultures 4 days postinfection for 24 h. Total nucleotide were extracted from macrophages and subjected to the real-time reverse transcription polymerase chain reaction (RT-PCR) assay for mRNA expression of TLRs (<b>A</b>), IRFs (<b>B</b>) and the key elements in IFN-λ3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<b>C</b>). Seven-day-cultured macrophages neither infected with HIV Bal strain nor treated with IFN-λ3 were set as control for a fair comparison with that in Figure 3. The data shown were the mean ± standard deviation of triplicate culture (<i>*P&lt;0.05, **P&lt;0.01</i>).</div></p></div><div class="aux"><div class="resc">Man-Qing Liu, et al. PLoS One. 2012;7(4):e35902.</div><p class="links"><a class="dblinks" href="/pubmed/22558263" ref="ordinalpos=4755&amp;ncbi_uid=3978413&amp;link_uid=22558263">Citation</a><a class="dblinks" href="/pmc/articles/PMC3338759/" ref="ordinalpos=4755&amp;ncbi_uid=3978413&amp;link_uid=PMC3338759">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4756.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2884164/figure/F4/" ref="ordinalpos=4756&amp;ncbi_uid=2298066&amp;link_uid=PMC2884164" href="/pmc/articles/PMC2884164/"><img src="/pmc/articles/instance/2884164/bin/1755-8794-3-13-4.gif" src-large="/pmc/articles/instance/2884164/bin/1755-8794-3-13-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2884164/" ref="ordinalpos=4756&amp;ncbi_uid=2298066&amp;link_uid=PMC2884164" image-link="/pmc/articles/PMC2884164/figure/F4/" class="imagepopup">Figure 4.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> networks mined from human pituitary adenoma proteomics data. </a></p><div class="supp"><p class="details"><div><b>NRF2-mediated oxidative-stress response <span class="highlight" style="background-color:">pathway</span> that is involved with pituitary adenoma protein-mapping data</b>. A gray label denotes an identified protein. The various shapes of nodes denote the different functions. A duplicated shape means this node contains multiple components. An arrow denotes the <span class="highlight" style="background-color:">pathway</span> direction. A line with a small circle denotes a biological result.</div></p></div><div class="aux"><div class="resc">Xianquan Zhan, et al. BMC Med Genomics. 2010;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/20426862" ref="ordinalpos=4756&amp;ncbi_uid=2298066&amp;link_uid=20426862">Citation</a><a class="dblinks" href="/pmc/articles/PMC2884164/" ref="ordinalpos=4756&amp;ncbi_uid=2298066&amp;link_uid=PMC2884164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4757.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2683752/figure/F3/" ref="ordinalpos=4757&amp;ncbi_uid=1797470&amp;link_uid=PMC2683752" href="/pmc/articles/PMC2683752/"><img src="/pmc/articles/instance/2683752/bin/nihms99093f3.gif" src-large="/pmc/articles/instance/2683752/bin/nihms99093f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2683752/" ref="ordinalpos=4757&amp;ncbi_uid=1797470&amp;link_uid=PMC2683752" image-link="/pmc/articles/PMC2683752/figure/F3/" class="imagepopup">Fig. 3. A Two-<span class="highlight" style="background-color:">Pathway</span> model for TCR-coupled <span class="highlight" style="background-color:">signaling</span> leading to IFN-γ production in memory CD4 T cells.  From: Biochemical <span class="highlight" style="background-color:">Signaling</span> Pathways for Memory T cell Recall. </a></p><div class="supp"><p class="details"><div>A schematic diagram of a memory CD4 T cell is shown with surface TCR and CD28. TCR engagement triggers sequential signals through CD3, ZAP-70 and SLP-76 as shown. Two pathways then diverge distal to SLP-76 that couple to IFN-γ gene transcription in the nucleus. The Late <span class="highlight" style="background-color:">pathway</span> (right side) requires both TCR and CD28 engagement, occurs following sustained activation and is associated with an accumulation of MAP kinase and distal phosphorylation and upregulation of T-bet expression. The Rapid <span class="highlight" style="background-color:">pathway</span> (left side) does not require CD28 costimulation, and is not associated with increased distal phosphorylation or T-bet upregulation. This rapid <span class="highlight" style="background-color:">pathway</span> requires NF[kappa]B transcriptional activity.</div></p></div><div class="aux"><div class="resc">Donna L. Farber. Semin Immunol. ;21(2):84-91.</div><p class="links"><a class="dblinks" href="/pubmed/19298946" ref="ordinalpos=4757&amp;ncbi_uid=1797470&amp;link_uid=19298946">Citation</a><a class="dblinks" href="/pmc/articles/PMC2683752/" ref="ordinalpos=4757&amp;ncbi_uid=1797470&amp;link_uid=PMC2683752">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4758.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2172300/figure/fig1/" ref="ordinalpos=4758&amp;ncbi_uid=1121706&amp;link_uid=PMC2172300" href="/pmc/articles/PMC2172300/"><img src="/pmc/articles/instance/2172300/bin/200406151f1.gif" src-large="/pmc/articles/instance/2172300/bin/200406151f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2172300/" ref="ordinalpos=4758&amp;ncbi_uid=1121706&amp;link_uid=PMC2172300" image-link="/pmc/articles/PMC2172300/figure/fig1/" class="imagepopup">Figure 1. From: FAK and paxillin. </a></p><div class="supp"><p class="details"><div><b>Regulation of Rac activity by FAK paxillin.</b> (A) FAK/p130cas stimulates Rac activation, whereas FAK/paxillin may inhibit Rac activity. This may occur via direct targeting of Rac by FAK/paxillin-associated regulators of Rac (1). Alternatively, FAK/paxillin <span class="highlight" style="background-color:">signaling</span> may interfere via crosstalk with a distinct <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (X) that promotes Rac activation (2). (B) In this schematic of the edge of a cell, peripheral Rac activity is low (red), whereas nonperipheral Rac activity is high (green). At the periphery, FAK/paxillin may function to reduce Rac activity. This could be achieved by discrete relocalization of FAK/paxillin or spatially dependent modulation of the Rac-inhibiting activity of FAK/paxillin. (C) In this schematic of two cells in contact, FAK/paxillin <span class="highlight" style="background-color:">signaling</span> and N-cadherin <span class="highlight" style="background-color:">signaling</span> may both contribute to the reduced levels of active Rac near cell–cell junctions.</div></p></div><div class="aux"><div class="resc">Michael D. Schaller. J Cell Biol. 2004 July 19;166(2):157-159.</div><p class="links"><a class="dblinks" href="/pubmed/15263014" ref="ordinalpos=4758&amp;ncbi_uid=1121706&amp;link_uid=15263014">Citation</a><a class="dblinks" href="/pmc/articles/PMC2172300/" ref="ordinalpos=4758&amp;ncbi_uid=1121706&amp;link_uid=PMC2172300">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4759.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3680444/figure/pone-0063676-g008/" ref="ordinalpos=4759&amp;ncbi_uid=5002526&amp;link_uid=PMC3680444" href="/pmc/articles/PMC3680444/"><img src="/pmc/articles/instance/3680444/bin/pone.0063676.g008.gif" src-large="/pmc/articles/instance/3680444/bin/pone.0063676.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3680444/" ref="ordinalpos=4759&amp;ncbi_uid=5002526&amp;link_uid=PMC3680444" image-link="/pmc/articles/PMC3680444/figure/pone-0063676-g008/" class="imagepopup">Figure 8. ATDC up-regulates Cyclin D1 and c-Myc via activation of NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Ataxia-Telangiectasia Group D Complementing Gene (ATDC) Promotes Lung Cancer Cell Proliferation by Activating NF-?B <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A. ATDC overexpression up-regulated NF-κB reporter luciferase activity in HBE cells and ATDC depletion inhibited NF-κB reporter luciferase activity in both A549 and H1299 cells. B. ATDC transfection increased p-IκB expression in HBE cells and ATDC depletion decreased the level of p-IκB in A549 and H1299 cells. C. NF-κB inhibitor Bay 11-7082 completely blocked NF-κB reporter luciferase activity and reversed the effect of ATDC on cyclin D1, c-Myc and p-Rb up-regulation.</div></p></div><div class="aux"><div class="resc">Zhong-Ping Tang, et al. PLoS One. 2013;8(6):e63676.</div><p class="links"><a class="dblinks" href="/pubmed/23776433" ref="ordinalpos=4759&amp;ncbi_uid=5002526&amp;link_uid=23776433">Citation</a><a class="dblinks" href="/pmc/articles/PMC3680444/" ref="ordinalpos=4759&amp;ncbi_uid=5002526&amp;link_uid=PMC3680444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4760.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4371083/figure/f8/" ref="ordinalpos=4760&amp;ncbi_uid=7388031&amp;link_uid=PMC4371083" href="/pmc/articles/PMC4371083/"><img src="/pmc/articles/instance/4371083/bin/srep09401-f8.gif" src-large="/pmc/articles/instance/4371083/bin/srep09401-f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4371083/" ref="ordinalpos=4760&amp;ncbi_uid=7388031&amp;link_uid=PMC4371083" image-link="/pmc/articles/PMC4371083/figure/f8/" class="imagepopup">Figure 8. Schematic diagram of endothelial cell apoptotic <span class="highlight" style="background-color:">signaling</span> and protective effect of miR-26a..  From: MicroRNA-26a prevents endothelial cell apoptosis by directly targeting TRPC6 in the setting of atherosclerosis. </a></p><div class="supp"><p class="details"><div>Ox-LDL triggers a complex cell death <span class="highlight" style="background-color:">signaling</span> including suppression of miR-26a and upregulation of TRPC6, leading to massive calcium influx, cytochrome c release from mitochondrial and caspase-3 activation. The data reported in this study demonstrate that miR-26a inhibits endothelial apoptosis via suppressing TRPC6 expression, thereby preventing calcium influx and subsequent activation of the mitochondrial apoptotic <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yong Zhang, et al. Sci Rep. 2015;5:9401.</div><p class="links"><a class="dblinks" href="/pubmed/25801675" ref="ordinalpos=4760&amp;ncbi_uid=7388031&amp;link_uid=25801675">Citation</a><a class="dblinks" href="/pmc/articles/PMC4371083/" ref="ordinalpos=4760&amp;ncbi_uid=7388031&amp;link_uid=PMC4371083">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4761.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2905744/figure/F3/" ref="ordinalpos=4761&amp;ncbi_uid=2362134&amp;link_uid=PMC2905744" href="/pmc/articles/PMC2905744/"><img src="/pmc/articles/instance/2905744/bin/nihms220184f3.gif" src-large="/pmc/articles/instance/2905744/bin/nihms220184f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2905744/" ref="ordinalpos=4761&amp;ncbi_uid=2362134&amp;link_uid=PMC2905744" image-link="/pmc/articles/PMC2905744/figure/F3/" class="imagepopup">Figure 3. The Phosphatidylinsoitol-3 kinase (PI3K)-Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: Novel targets for the treatment and palliation of gastrointestinal neuroendocrine tumors. </a></p><div class="supp"><p class="details"><div>PI3K is activated by binding of growth factors (GF) to a receptor tyrosine kinase (RTK) through direct interaction or adapter molecules. PIP<sub>2</sub> is then phosphorylated by PI3K to form PIP<sub>3</sub>, which then engages Akt and PDK1 resulting in phosphorylation of Akt (either directly or by PDK1). Activated Akt then stimulates pathways involved in cell growth and protein synthesis (mTOR), activates pro-survival molecules (NF-κB), and inhibits pro-apoptotic molecules (FOXO, Bad, p53) thereby promoting cell survival and proliferation. PTEN negatively regulates PI3K-Akt <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Scott N. Pinchot, et al. Curr Opin Investig Drugs. ;9(6):576-582.</div><p class="links"><a class="dblinks" href="/pubmed/18516757" ref="ordinalpos=4761&amp;ncbi_uid=2362134&amp;link_uid=18516757">Citation</a><a class="dblinks" href="/pmc/articles/PMC2905744/" ref="ordinalpos=4761&amp;ncbi_uid=2362134&amp;link_uid=PMC2905744">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4762.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407375/figure/F3/" ref="ordinalpos=4762&amp;ncbi_uid=4841536&amp;link_uid=PMC3407375" href="/pmc/articles/PMC3407375/"><img src="/pmc/articles/instance/3407375/bin/nihms388184f3.gif" src-large="/pmc/articles/instance/3407375/bin/nihms388184f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407375/" ref="ordinalpos=4762&amp;ncbi_uid=4841536&amp;link_uid=PMC3407375" image-link="/pmc/articles/PMC3407375/figure/F3/" class="imagepopup">Figure 3. Acceleration of renal gene expression by <span class="highlight" style="background-color:">signaling</span> loops involving TGF-β and miRNAs.  From: MicroRNA Circuits in Transforming Growth Factor-? actions and Diabetic Nephropathy. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> loops initiated by TGF-β accelerate downstream action and effects via miRNAs. A) TGF-β increases miR-192 expression via Smad3 or p53. TGF-β1 gene expression is itself auto-upregulated by miR-192 through Zeb1/2. B) miR-192 enhances p53 levels via inhibition of Mdm2 (inhibitor of p53). p53 increases transcription of miR-192 through p53 responsive elements on the promoter. C) miR-200 family members can further augment their own expression through inhibition of Zeb1/2. Initial expression of miR-200 family can also be induced by miR-192 through Zeb1/2. D) The promoter of the non-coding RNA RP23 (the host gene of miR-216a and miR-217) also contains E-boxes <sup>6</sup> and increased miR-216a also activates its own promoter through Tsc-22 and Tfe3. miR-216a/Ybx1 <span class="highlight" style="background-color:">pathway</span> can amplify the <span class="highlight" style="background-color:">signaling</span> initiated by TGF-β and miR-192.</div></p></div><div class="aux"><div class="resc">Mitsuo Kato, et al. Semin Nephrol. ;32(3):253-260.</div><p class="links"><a class="dblinks" href="/pubmed/22835456" ref="ordinalpos=4762&amp;ncbi_uid=4841536&amp;link_uid=22835456">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407375/" ref="ordinalpos=4762&amp;ncbi_uid=4841536&amp;link_uid=PMC3407375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4763.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4151463/figure/F9/" ref="ordinalpos=4763&amp;ncbi_uid=6551054&amp;link_uid=PMC4151463" href="/pmc/articles/PMC4151463/"><img src="/pmc/articles/instance/4151463/bin/nihms319397f9.gif" src-large="/pmc/articles/instance/4151463/bin/nihms319397f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4151463/" ref="ordinalpos=4763&amp;ncbi_uid=6551054&amp;link_uid=PMC4151463" image-link="/pmc/articles/PMC4151463/figure/F9/" class="imagepopup">Figure 9. A working model for ILK <span class="highlight" style="background-color:">signaling</span> in quiescent and activated hepatic stellate cells.  From: The function of integrin linked-kinase (ILK) in normal and activated stellate cells-implications for fibrogenesis in wound healing. </a></p><div class="supp"><p class="details"><div>This figure highlights previously established ILK <span class="highlight" style="background-color:">signaling</span> partners and emphasizes data from the current study. In quiescent cells, integrins likely have low affinity for extracellular matrix (ECM) molecules (i.e. such as collagen I, fibronectin, etc... - which are in low abundance) and are therefore in a low affinity or inactive confirmation. Thus, ILK is relatively inactive. In addition, the Gα12/13 <span class="highlight" style="background-color:">pathway</span> is inactive (delineated by dotted lines). Upon ligand binding (i.e., collagen I, fibronectin, etc), the confirmation of the integrin changes and ILK binds to the cytoplasmic tail of β-integrin, resulting in formation of PINCH-ILK-parvin ternary complexes. ILK subsequently recruits other <span class="highlight" style="background-color:">signaling</span> molecules. In stellate cells, ILK is engaged in G-protein receptor <span class="highlight" style="background-color:">signaling</span> cross talk, where it appears to amplify Gα12/13 mediated signals. Activation of this ILK/Gα12/13 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has several functional consequences, including in actin reorganization (and actin synthesis), cell spreading, cell migration, and extracellular matrix synthesis. The pathways leading to synthesis of ECM are likely to be multiple, and likely include both ILK direct pathways, as well as those that cross talk through the Gα12/13 and the Rho <span class="highlight" style="background-color:">pathway</span>. The activated stellate cell cartoon is meant to depict these both.</div></p></div><div class="aux"><div class="resc">Mahnoush S. Shafiei, et al. Lab Invest. ;92(2):305-316.</div><p class="links"><a class="dblinks" href="/pubmed/22064318" ref="ordinalpos=4763&amp;ncbi_uid=6551054&amp;link_uid=22064318">Citation</a><a class="dblinks" href="/pmc/articles/PMC4151463/" ref="ordinalpos=4763&amp;ncbi_uid=6551054&amp;link_uid=PMC4151463">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4764.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2856263/figure/F6/" ref="ordinalpos=4764&amp;ncbi_uid=2871014&amp;link_uid=PMC2856263" href="/pmc/articles/PMC2856263/"><img src="/pmc/articles/instance/2856263/bin/zbc0171011300006.gif" src-large="/pmc/articles/instance/2856263/bin/zbc0171011300006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2856263/" ref="ordinalpos=4764&amp;ncbi_uid=2871014&amp;link_uid=PMC2856263" image-link="/pmc/articles/PMC2856263/figure/F6/" class="imagepopup">FIGURE 6. From: Activation of the Nucleotide Oligomerization Domain <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by the Non-bacterially Derived Xanthone Drug 5?6-Dimethylxanthenone-4-acetic Acid (Vadimezan). </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of <span class="highlight" style="background-color:">signaling</span> pathways activated by DMXAA.</b> Previous work (13) has shown that DMXAA can activate TANK-binding kinase 1 to initiate IRF3 <span class="highlight" style="background-color:">signaling</span>. Our data indicate that DMXAA also activates NOD-like receptors NOD2 and NOD1 that interact with RICK to activate NF-κB signal pathways that regulate production of inflammatory cytokines/chemokines.</div></p></div><div class="aux"><div class="resc">Guanjun Cheng, et al. J Biol Chem. 2010 April 2;285(14):10553-10562.</div><p class="links"><a class="dblinks" href="/pubmed/20118240" ref="ordinalpos=4764&amp;ncbi_uid=2871014&amp;link_uid=20118240">Citation</a><a class="dblinks" href="/pmc/articles/PMC2856263/" ref="ordinalpos=4764&amp;ncbi_uid=2871014&amp;link_uid=PMC2856263">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4765.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4284744/figure/ijms-15-22887-f001/" ref="ordinalpos=4765&amp;ncbi_uid=7087490&amp;link_uid=PMC4284744" href="/pmc/articles/PMC4284744/"><img src="/pmc/articles/instance/4284744/bin/ijms-15-22887-g001.gif" src-large="/pmc/articles/instance/4284744/bin/ijms-15-22887-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4284744/" ref="ordinalpos=4765&amp;ncbi_uid=7087490&amp;link_uid=PMC4284744" image-link="/pmc/articles/PMC4284744/figure/ijms-15-22887-f001/" class="imagepopup">Figure 1.  From: LRP-1: Functions, <span class="highlight" style="background-color:">Signaling</span> and Implications in Kidney and Other Diseases. </a></p><div class="supp"><p class="details"><div>Fibroblast Low-density lipoprotein (LDL)-related protein-1 (LRP-1) <span class="highlight" style="background-color:">signaling</span> in renal fibrogenesis. Interaction of LRP-1 and its ligands mediates fibroblast differentiation and transdifferentiation. Tissue-type plasminogen activator (tPA) binds to LRP-1 and induces its tyrosine phosphorylation, followed by activation of: (1) extracellular signal-regulated kinases (Erk)1/2 <span class="highlight" style="background-color:">pathway</span> to stimulate matrix metalloproteinase (MMP)-9 production and trigger the epithelial mesenchymal transition (EMT); (2) p90 ribosomal S6 kinase (p90RSK) and Bad <span class="highlight" style="background-color:">pathway</span> to promote fibroblast survival; (3) p90RSK and glycogen synthase kinase (GSK)3β <span class="highlight" style="background-color:">pathway</span> to induce proliferation; (4) TGF-β1-mediated β1 integrin and integrin-linked kinase (ILK) <span class="highlight" style="background-color:">pathway</span> to induce myofibroblast activation. Connective tissue growth factor (CTGF) binds and induces LRP-1 tyrosine phosphorylation, and promotes TGF-β1-mediated Erk1/2 activation, which leads to synergistic activation of myofibroblasts. Figure was modified with permission [43]. <b>*</b> stands for phosphorylation.</div></p></div><div class="aux"><div class="resc">Ling Lin, et al. Int J Mol Sci. 2014 December;15(12):22887-22901.</div><p class="links"><a class="dblinks" href="/pubmed/25514242" ref="ordinalpos=4765&amp;ncbi_uid=7087490&amp;link_uid=25514242">Citation</a><a class="dblinks" href="/pmc/articles/PMC4284744/" ref="ordinalpos=4765&amp;ncbi_uid=7087490&amp;link_uid=PMC4284744">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4766.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3098871/figure/pone-0020319-g004/" ref="ordinalpos=4766&amp;ncbi_uid=2970915&amp;link_uid=PMC3098871" href="/pmc/articles/PMC3098871/"><img src="/pmc/articles/instance/3098871/bin/pone.0020319.g004.gif" src-large="/pmc/articles/instance/3098871/bin/pone.0020319.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3098871/" ref="ordinalpos=4766&amp;ncbi_uid=2970915&amp;link_uid=PMC3098871" image-link="/pmc/articles/PMC3098871/figure/pone-0020319-g004/" class="imagepopup">Figure 4. Signal pathways of the SMAD3-bound target genes..  From: High Throughput Determination of TGF?1/SMAD3 Targets in A549 Lung Epithelial Cells. </a></p><div class="supp"><p class="details"><div>A: ChIP SMAD3-bound target genes grouped by <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and ranked in order of statistical significance. The ratio of genes (orange line) refers to number of genes involved in <span class="highlight" style="background-color:">pathway</span> divided by total genes; approximately 10% of bound genes are identified as belonging to the known TGFβ1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Other prominent <span class="highlight" style="background-color:">signaling</span> pathways include ERK/MAPK and integrin <span class="highlight" style="background-color:">signaling</span>, which is consistent with known interactions of TGFβ1. Data and image generated using Ingenuity Pathways Analysis. B: Combined gene expression microarray and ChIP-on-chip data grouped by biological process using MetaCore GeneGo systems biology analysis tools [46], [47]. The top 10 identified pathways are shown. The solid blue and orange bars represent the -log <i>(p</i>-value) for unique genes identified by the TGFβ1-induced gene expression and the ChIP SMAD3-bound genes, respectively. The stripped blue bars represent the -log (<i>p</i>-value) for common genes identified by both TGFβ1-induced gene expression and ChIP SMAD3-bound genes. The black bars represent the -log (<i>p</i>-value) for similar genes identified by both TGFβ1-induced gene expression and ChIP SMAD3-bound genes.</div></p></div><div class="aux"><div class="resc">Yingze Zhang, et al. PLoS One. 2011;6(5):e20319.</div><p class="links"><a class="dblinks" href="/pubmed/21625455" ref="ordinalpos=4766&amp;ncbi_uid=2970915&amp;link_uid=21625455">Citation</a><a class="dblinks" href="/pmc/articles/PMC3098871/" ref="ordinalpos=4766&amp;ncbi_uid=2970915&amp;link_uid=PMC3098871">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4767.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4198622/figure/Fig6/" ref="ordinalpos=4767&amp;ncbi_uid=6727233&amp;link_uid=PMC4198622" href="/pmc/articles/PMC4198622/"><img src="/pmc/articles/instance/4198622/bin/13046_2014_84_Fig6_HTML.gif" src-large="/pmc/articles/instance/4198622/bin/13046_2014_84_Fig6_HTML.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4198622/" ref="ordinalpos=4767&amp;ncbi_uid=6727233&amp;link_uid=PMC4198622" image-link="/pmc/articles/PMC4198622/figure/Fig6/" class="imagepopup">Figure 6.  From: A four-long non-coding RNA signature in predicting breast cancer survival. </a></p><div class="supp"><p class="details"><div> <b>Gene set enrichment analysis delineates biological pathways and processes associated with risk score in the training set(GSE 20685).</b> GSEA validated enhanced activity of <b>(A)</b> epithelial mesenchymal transition <span class="highlight" style="background-color:">pathway</span> <b>(B)</b> cell cycle <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(C)</b> DNA replication <span class="highlight" style="background-color:">pathway</span> in high risk score group.</div></p></div><div class="aux"><div class="resc">Jin Meng, et al. J Exp Clin Cancer Res. 2014;33(1):84.</div><p class="links"><a class="dblinks" href="/pubmed/25288503" ref="ordinalpos=4767&amp;ncbi_uid=6727233&amp;link_uid=25288503">Citation</a><a class="dblinks" href="/pmc/articles/PMC4198622/" ref="ordinalpos=4767&amp;ncbi_uid=6727233&amp;link_uid=PMC4198622">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4768.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3844829/figure/Fig1/" ref="ordinalpos=4768&amp;ncbi_uid=5577411&amp;link_uid=PMC3844829" href="/pmc/articles/PMC3844829/"><img src="/pmc/articles/instance/3844829/bin/13577_2011_29_Fig1_HTML.gif" src-large="/pmc/articles/instance/3844829/bin/13577_2011_29_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3844829/" ref="ordinalpos=4768&amp;ncbi_uid=5577411&amp;link_uid=PMC3844829" image-link="/pmc/articles/PMC3844829/figure/Fig1/" class="imagepopup">Fig. 1.  From: Apoptosis-related gene expression in glioblastoma (LN-18) and medulloblastoma (Daoy) cell lines. </a></p><div class="supp"><p class="details"><div>List of the genes for which expression in LN18 and Daoy cells was significantly different from the control HUVEC cells. The genes are divided into indicated apoptosis pathways: the TNF receptor <span class="highlight" style="background-color:">pathway</span>, NKkB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Bcl-2 family-regulated <span class="highlight" style="background-color:">pathway</span>, CARD family, caspases, IAP family, and others. Differences in gene expression are presented as fold changes (compared to reference HUVECs), accompanied by corresponding <i>p</i> values (according to [36])</div></p></div><div class="aux"><div class="resc">Iwona Wybranska, et al. Hum Cell. 2013;26(4):137-148.</div><p class="links"><a class="dblinks" href="/pubmed/24037645" ref="ordinalpos=4768&amp;ncbi_uid=5577411&amp;link_uid=24037645">Citation</a><a class="dblinks" href="/pmc/articles/PMC3844829/" ref="ordinalpos=4768&amp;ncbi_uid=5577411&amp;link_uid=PMC3844829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4769.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3391223/figure/pone-0040175-g001/" ref="ordinalpos=4769&amp;ncbi_uid=4117252&amp;link_uid=PMC3391223" href="/pmc/articles/PMC3391223/"><img src="/pmc/articles/instance/3391223/bin/pone.0040175.g001.gif" src-large="/pmc/articles/instance/3391223/bin/pone.0040175.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3391223/" ref="ordinalpos=4769&amp;ncbi_uid=4117252&amp;link_uid=PMC3391223" image-link="/pmc/articles/PMC3391223/figure/pone-0040175-g001/" class="imagepopup">Figure 1. Distribution of interaction p-values across genes mapping to pathways with weak interaction signals..  From: Different Genes Interact with Particulate Matter and Tobacco Smoke Exposure in Affecting Lung Function Decline in the General Population. </a></p><div class="supp"><p class="details"><div>P-values of interaction on the gene-level are given on a minus log<sub>10</sub> scale (y-axis), i.e. higher bars represent smaller interaction p-values. (A) Genes of the mitochondrial dysfunction <span class="highlight" style="background-color:">pathway</span> interacting with PM10 and packyears exposure between surveys on FEV<sub>1</sub>/FVC decline. (B) Genes of the methane metabolism <span class="highlight" style="background-color:">pathway</span> interacting with PM10 and packyears exposure between surveys on FEF<sub>25–75</sub> decline. (C) Genes of the apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> interacting with PM10 and packyears exposure between surveys on FEV<sub>1</sub> decline.</div></p></div><div class="aux"><div class="resc">Ivan Curjuric, et al. PLoS One. 2012;7(7):e40175.</div><p class="links"><a class="dblinks" href="/pubmed/22792237" ref="ordinalpos=4769&amp;ncbi_uid=4117252&amp;link_uid=22792237">Citation</a><a class="dblinks" href="/pmc/articles/PMC3391223/" ref="ordinalpos=4769&amp;ncbi_uid=4117252&amp;link_uid=PMC3391223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4770.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2709179/figure/F6/" ref="ordinalpos=4770&amp;ncbi_uid=1994474&amp;link_uid=PMC2709179" href="/pmc/articles/PMC2709179/"><img src="/pmc/articles/instance/2709179/bin/zjw0060933930006.gif" src-large="/pmc/articles/instance/2709179/bin/zjw0060933930006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2709179/" ref="ordinalpos=4770&amp;ncbi_uid=1994474&amp;link_uid=PMC2709179" image-link="/pmc/articles/PMC2709179/figure/F6/" class="imagepopup">Fig. 6. From: Enhanced Interferon <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Oral Cancer Revealed by Quantitative Proteome Analysis of Microdissected Specimens  Using 16O/18O Labeling and Integrated  Two-dimensional LC-ESI-MALDI Tandem MS. </a></p><div class="supp"><p class="details"><div><b>MetaCore analysis of altered <span class="highlight" style="background-color:">signaling</span> pathways in OSCC samples.</b> <i>A</i>, the cytoskeleton remodeling-keratin filaments <span class="highlight" style="background-color:">pathway</span>. <i>B</i>, the type I IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>Bars</i> labeled in <i>red</i> and <i>blue</i> denote up- and down-regulated target proteins, respectively. <i>Numbers</i> on the <i>bar</i> indicate the experiment in which the target was quantified. <i>1</i>, 16N1/18T1; <i>2</i>, 16N3/18T3; <i>3</i>, 16T1/18N1; <i>4</i>, 16T2/18N2; <i>5</i>, 16T3/18N3. <i>JNK</i>, c-Jun NH<sub>2</sub>-terminal kinase; <i>MAPK</i>, mitogen-activated protein kinase; <i>IFNAR</i>, IFN-alpha/beta receptor. Interaction mechanism is marked with a symbol in the <i>hexagon</i> in the <i>middle</i> of the interaction arrow. <i>CF</i>, complex formation; <i>Cm</i>, covalent modifications; <i>Tr</i>, transcription regulation; <i>B</i>, binding; <i>+P</i>, phosphorylation; <i>−P</i>, dephosphorylation; <i>Tn</i>, transport; <i>CS</i>, complex subunit.</div></p></div><div class="aux"><div class="resc">Lang-Ming Chi, et al. Mol Cell Proteomics. 2009 July;8(7):1453-1474.</div><p class="links"><a class="dblinks" href="/pubmed/19297561" ref="ordinalpos=4770&amp;ncbi_uid=1994474&amp;link_uid=19297561">Citation</a><a class="dblinks" href="/pmc/articles/PMC2709179/" ref="ordinalpos=4770&amp;ncbi_uid=1994474&amp;link_uid=PMC2709179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4771.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3492204/figure/F5/" ref="ordinalpos=4771&amp;ncbi_uid=4355601&amp;link_uid=PMC3492204" href="/pmc/articles/PMC3492204/"><img src="/pmc/articles/instance/3492204/bin/1750-1172-7-27-5.gif" src-large="/pmc/articles/instance/3492204/bin/1750-1172-7-27-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3492204/" ref="ordinalpos=4771&amp;ncbi_uid=4355601&amp;link_uid=PMC3492204" image-link="/pmc/articles/PMC3492204/figure/F5/" class="imagepopup">Figure 5.  From: A mutation in KIF7 is responsible for the autosomal recessive syndrome of macrocephaly, multiple epiphyseal dysplasia and distinctive facial appearance. </a></p><div class="supp"><p class="details"><div><b>A simplified scheme to illustrate the dual roles played by KIF7 in Indian Hedgehog (IHh) <span class="highlight" style="background-color:">signaling</span> and chondrocytes development.</b> In chondrocytes Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is initiated by the IHh ligand binding to PTC, which releases the inhibition of SMO. KIF7 positively modulates IHh <span class="highlight" style="background-color:">pathway</span> activity during development through downregulation of SUFU (indicated by thin straight line). SUFU is a known negative regulator of IHh <span class="highlight" style="background-color:">pathway</span> that prevents GLI-mediated transcription for the genes responsible for chondrocytes proliferation and differentiation during development. In the absence of KIF7, SUFU becomes uninhibited and exerts higher levels of suppression on GLI-mediated transcription (indicated by thick straight line). In the absence of SUFU, KIF7 negatively regulates IHh <span class="highlight" style="background-color:">signaling</span> to a lesser extent by inhibiting GLI-mediated transcription by undefined mechanism (indicated by dashed line). <i>ECM</i>: extracellular matrix. <i>Adapted from Hsu et al.</i>[19].</div></p></div><div class="aux"><div class="resc">Bassam R Ali, et al. Orphanet J Rare Dis. 2012;7:27-27.</div><p class="links"><a class="dblinks" href="/pubmed/22587682" ref="ordinalpos=4771&amp;ncbi_uid=4355601&amp;link_uid=22587682">Citation</a><a class="dblinks" href="/pmc/articles/PMC3492204/" ref="ordinalpos=4771&amp;ncbi_uid=4355601&amp;link_uid=PMC3492204">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4772.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3189824/figure/fig02/" ref="ordinalpos=4772&amp;ncbi_uid=3280536&amp;link_uid=PMC3189824" href="/pmc/articles/PMC3189824/"><img src="/pmc/articles/instance/3189824/bin/ajcr0001-0629-f2.gif" src-large="/pmc/articles/instance/3189824/bin/ajcr0001-0629-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3189824/" ref="ordinalpos=4772&amp;ncbi_uid=3280536&amp;link_uid=PMC3189824" image-link="/pmc/articles/PMC3189824/figure/fig02/" class="imagepopup">Figure 2.  From: The complex interplay between autophagy and NF-?B <span class="highlight" style="background-color:">signaling</span> pathways in cancer cells. </a></p><div class="supp"><p class="details"><div>Schematic model for the activation of NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. In the canonical NF-κB <span class="highlight" style="background-color:">pathway</span>, stimulation of pro-inflammatory receptors activates the IKK complex (comprised of IKKα, IKKβ and IKKγ) leading to the phos-phorylation of IKBCX and its proteasomal degradation. This induces the translocation of p65/p50 dimers into the nucleus where the dimer binds to the promoter regions of a subset of target genes. In the noncanonical NF-κB <span class="highlight" style="background-color:">pathway</span>, stimulation of BAFF-R, CD40, and lymphotoxina-β receptor (LTβR) leads to the stabilization of NIK and results in the activation of the IKKα homo-dimers. Active IKKα, in turn, induces the phosphorylation of the NF-κB family member NF-κB2/p100 leading to its processing by the proteasome to generate p52. This allows the translocation of the p52/Rel B dimer into nucleus where it regulates the transcription of target genes. For details, see text.</div></p></div><div class="aux"><div class="resc">Aurore Trocoli, et al. Am J Cancer Res. 2011;1(5):629-649.</div><p class="links"><a class="dblinks" href="/pubmed/21994903" ref="ordinalpos=4772&amp;ncbi_uid=3280536&amp;link_uid=21994903">Citation</a><a class="dblinks" href="/pmc/articles/PMC3189824/" ref="ordinalpos=4772&amp;ncbi_uid=3280536&amp;link_uid=PMC3189824">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4773.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3969038/figure/f1-molcell-37-3-183-1/" ref="ordinalpos=4773&amp;ncbi_uid=6012054&amp;link_uid=PMC3969038" href="/pmc/articles/PMC3969038/"><img src="/pmc/articles/instance/3969038/bin/molcell-37-3-189-1f1.gif" src-large="/pmc/articles/instance/3969038/bin/molcell-37-3-189-1f1.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3969038/" ref="ordinalpos=4773&amp;ncbi_uid=6012054&amp;link_uid=PMC3969038" image-link="/pmc/articles/PMC3969038/figure/f1-molcell-37-3-183-1/" class="imagepopup">Fig. 1. From: NF-?B in Cellular Senescence and Cancer Treatment. </a></p><div class="supp"><p class="details"><div>NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and cancer related mutations. Only canonical (classical) <span class="highlight" style="background-color:">pathway</span> is shown. Extracellular stimuli transmitted though receptor complexes activate IKK <i>via</i> phosphorylation. Consequently, IKK phosphorylates IκB, leading to its dissociation from inactive NF-κB complex and ubiquitin-mediated proteasomal degradation. Freed from IκB, active NF-κB dimer translocates into the nucleus and induces the transcription of various target genes involved in multiple cellular pathways. Cancer-relevant mutations, indicated in orange boxes, can occur in all steps of NF-κB <span class="highlight" style="background-color:">signaling</span> cascade.</div></p></div><div class="aux"><div class="resc">Hua Jing, et al. Mol Cells. 2014 March 31;37(3):189-195.</div><p class="links"><a class="dblinks" href="/pubmed/24608805" ref="ordinalpos=4773&amp;ncbi_uid=6012054&amp;link_uid=24608805">Citation</a><a class="dblinks" href="/pmc/articles/PMC3969038/" ref="ordinalpos=4773&amp;ncbi_uid=6012054&amp;link_uid=PMC3969038">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4774.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3201197/figure/fig1/" ref="ordinalpos=4774&amp;ncbi_uid=3486173&amp;link_uid=PMC3201197" href="/pmc/articles/PMC3201197/"><img src="/pmc/articles/instance/3201197/bin/JEM_20112087_RGB_Fig1.gif" src-large="/pmc/articles/instance/3201197/bin/JEM_20112087_RGB_Fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3201197/" ref="ordinalpos=4774&amp;ncbi_uid=3486173&amp;link_uid=PMC3201197" image-link="/pmc/articles/PMC3201197/figure/fig1/" class="imagepopup">Figure 1. From: Assembling defenses against therapy-resistant leukemic stem cells: Bcl6 joins the ranks. </a></p><div class="supp"><p class="details"><div><b>PI3K, AKT, FOXO, and BCL6 are key players in Ph<sup>+</sup> stem cell survival.</b> Ph<sup>+</sup> ALL and CML are dependent on signals emanating from BCR-ABL through the PI3K–AKT <span class="highlight" style="background-color:">pathway</span> that may be driven by TGF-β via inhibition of AKT. BCL6 acts downstream of FOXO TFs and appears to represent a critical missing piece of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that leads to cancer stem cell survival. This <span class="highlight" style="background-color:">signaling</span> cascade offers potential for therapeutic modulation at various levels, including BCR-ABL inhibition by kinase inhibitors, TGFβ inhibition by Ly364947, and Bcl6 inhibition by RI-BPI.</div></p></div><div class="aux"><div class="resc">Francesca Pellicano, et al. J Exp Med. 2011 October 24;208(11):2155-2158.</div><p class="links"><a class="dblinks" href="/pubmed/22025499" ref="ordinalpos=4774&amp;ncbi_uid=3486173&amp;link_uid=22025499">Citation</a><a class="dblinks" href="/pmc/articles/PMC3201197/" ref="ordinalpos=4774&amp;ncbi_uid=3486173&amp;link_uid=PMC3201197">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4775.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4107617/figure/F3/" ref="ordinalpos=4775&amp;ncbi_uid=6408550&amp;link_uid=PMC4107617" href="/pmc/articles/PMC4107617/"><img src="/pmc/articles/instance/4107617/bin/1742-2094-11-110-3.gif" src-large="/pmc/articles/instance/4107617/bin/1742-2094-11-110-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4107617/" ref="ordinalpos=4775&amp;ncbi_uid=6408550&amp;link_uid=PMC4107617" image-link="/pmc/articles/PMC4107617/figure/F3/" class="imagepopup">Figure 3.  From: Group I metabotropic glutamate receptor mediated dynamic immune dysfunction in children with fragile X syndrome. </a></p><div class="supp"><p class="details"><div><b>Group I mGluR-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in immune cells.</b> Activation of pathogen associated molecular pattern (PAMP) receptors leads to a <span class="highlight" style="background-color:">signaling</span> cascade which can be both inhibited and assisted by group I mGluR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Milo Careaga, et al. J Neuroinflammation. 2014;11:110-110.</div><p class="links"><a class="dblinks" href="/pubmed/24942544" ref="ordinalpos=4775&amp;ncbi_uid=6408550&amp;link_uid=24942544">Citation</a><a class="dblinks" href="/pmc/articles/PMC4107617/" ref="ordinalpos=4775&amp;ncbi_uid=6408550&amp;link_uid=PMC4107617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4776.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4047088/figure/pone-0099195-g007/" ref="ordinalpos=4776&amp;ncbi_uid=6234085&amp;link_uid=PMC4047088" href="/pmc/articles/PMC4047088/"><img src="/pmc/articles/instance/4047088/bin/pone.0099195.g007.gif" src-large="/pmc/articles/instance/4047088/bin/pone.0099195.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4047088/" ref="ordinalpos=4776&amp;ncbi_uid=6234085&amp;link_uid=PMC4047088" image-link="/pmc/articles/PMC4047088/figure/pone-0099195-g007/" class="imagepopup">Figure 7. Impact of Gef on βAR-mediated regulation of cardiac transcript expression..  From: ?-Adrenergic Receptor-Dependent Alterations in Murine Cardiac Transcript Expression Are Differentially Regulated by Gefitinib In Vivo. </a></p><div class="supp"><p class="details"><div>Stimulation of βAR leads to activation of G protein <span class="highlight" style="background-color:">signaling</span> to induce changes in transcript expression via classical cAMP-PKA-mediated <span class="highlight" style="background-color:">signaling</span>. βAR stimulation also induces the transactivation of EGFR, which alters transcript expression directly, or indirectly by antagonizing the classical βAR-cAMP-PKA <span class="highlight" style="background-color:">pathway</span>, effects that are blocked with Gef treatment.</div></p></div><div class="aux"><div class="resc">Jennifer A. Talarico, et al. PLoS One. 2014;9(6):e99195.</div><p class="links"><a class="dblinks" href="/pubmed/24901703" ref="ordinalpos=4776&amp;ncbi_uid=6234085&amp;link_uid=24901703">Citation</a><a class="dblinks" href="/pmc/articles/PMC4047088/" ref="ordinalpos=4776&amp;ncbi_uid=6234085&amp;link_uid=PMC4047088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4777.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3051174/figure/fig3/" ref="ordinalpos=4777&amp;ncbi_uid=2826719&amp;link_uid=PMC3051174" href="/pmc/articles/PMC3051174/"><img src="/pmc/articles/instance/3051174/bin/JBB2011-523481.003.gif" src-large="/pmc/articles/instance/3051174/bin/JBB2011-523481.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3051174/" ref="ordinalpos=4777&amp;ncbi_uid=2826719&amp;link_uid=PMC3051174" image-link="/pmc/articles/PMC3051174/figure/fig3/" class="imagepopup">Figure 3.  From: Modulation of Acetylation: Creating a Pro-survival and Anti-Inflammatory Phenotype in Lethal Hemorrhagic and Septic Shock. </a></p><div class="supp"><p class="details"><div>TLR4 <span class="highlight" style="background-color:">signaling</span> – a converged immune response <span class="highlight" style="background-color:">pathway</span> for hemorrhagic shock and septic shock. TLR4 not only serves as a key sensor of pathogen-associated molecular patterns (PAMPs) but also is proposed recently to act as a receptor for some endogenous molecules called “alarmins”. HDACIs block TLR4 <span class="highlight" style="background-color:">signaling</span> at multiple steps; therefore, they can inhibit immune response for both hemorrhagic shock and septic shock.</div></p></div><div class="aux"><div class="resc">Yongqing Li, et al. J Biomed Biotechnol. 2011;2011:523481.</div><p class="links"><a class="dblinks" href="/pubmed/21403879" ref="ordinalpos=4777&amp;ncbi_uid=2826719&amp;link_uid=21403879">Citation</a><a class="dblinks" href="/pmc/articles/PMC3051174/" ref="ordinalpos=4777&amp;ncbi_uid=2826719&amp;link_uid=PMC3051174">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4778.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3773347/figure/f1-mjms-20-4-005/" ref="ordinalpos=4778&amp;ncbi_uid=5310103&amp;link_uid=PMC3773347" href="/pmc/articles/PMC3773347/"><img src="/pmc/articles/instance/3773347/bin/mjms-20-4-005-g1.gif" src-large="/pmc/articles/instance/3773347/bin/mjms-20-4-005-g1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3773347/" ref="ordinalpos=4778&amp;ncbi_uid=5310103&amp;link_uid=PMC3773347" image-link="/pmc/articles/PMC3773347/figure/f1-mjms-20-4-005/" class="imagepopup">Figure 1.  From: Four Possible Itching Pathways Related to the TRPV1 Channel, Histamine, PAR-2 and Serotonin. </a></p><div class="supp"><p class="details"><div>A schematic diagram showing the four putative itch <span class="highlight" style="background-color:">signaling</span> pathways. 1) TRPV1-expressing small sensory neurons that mediate itching via histamine or ET-1, as well as inflammatory pain. 2) TRPV1-negative (capsaicin-insensitive) <span class="highlight" style="background-color:">pathway</span> activated by histamine; this <span class="highlight" style="background-color:">pathway</span> likely transmits acute noxious heat as well. Nonhistaminergic <span class="highlight" style="background-color:">pathway</span> activated by 3) PAR-2 or 4) 5-HT, which terminate in the lateral part of lamina I as separate from the histaminergic pathways.</div></p></div><div class="aux"><div class="resc">Hiroshi NAKAGAWA, et al. Malays J Med Sci. 2013 July;20(4):5-12.</div><p class="links"><a class="dblinks" href="/pubmed/24043991" ref="ordinalpos=4778&amp;ncbi_uid=5310103&amp;link_uid=24043991">Citation</a><a class="dblinks" href="/pmc/articles/PMC3773347/" ref="ordinalpos=4778&amp;ncbi_uid=5310103&amp;link_uid=PMC3773347">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4779.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3901342/figure/F4/" ref="ordinalpos=4779&amp;ncbi_uid=5738993&amp;link_uid=PMC3901342" href="/pmc/articles/PMC3901342/"><img src="/pmc/articles/instance/3901342/bin/1471-2164-15-47-4.gif" src-large="/pmc/articles/instance/3901342/bin/1471-2164-15-47-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3901342/" ref="ordinalpos=4779&amp;ncbi_uid=5738993&amp;link_uid=PMC3901342" image-link="/pmc/articles/PMC3901342/figure/F4/" class="imagepopup">Figure 4.  From: A comprehensive microRNA expression profile of the backfat tissue from castrated and intact full-sib pair male pigs. </a></p><div class="supp"><p class="details"><div><b>Putative target genes of miRNAs participating in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via DAVID KEGG analysis.</b> Red boxes represent the target genes of miRNAs.</div></p></div><div class="aux"><div class="resc">Ying Bai, et al. BMC Genomics. 2014;15:47-47.</div><p class="links"><a class="dblinks" href="/pubmed/24443800" ref="ordinalpos=4779&amp;ncbi_uid=5738993&amp;link_uid=24443800">Citation</a><a class="dblinks" href="/pmc/articles/PMC3901342/" ref="ordinalpos=4779&amp;ncbi_uid=5738993&amp;link_uid=PMC3901342">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4780.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3186365/figure/F9/" ref="ordinalpos=4780&amp;ncbi_uid=3555613&amp;link_uid=PMC3186365" href="/pmc/articles/PMC3186365/"><img src="/pmc/articles/instance/3186365/bin/zbc0461181590009.gif" src-large="/pmc/articles/instance/3186365/bin/zbc0461181590009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3186365/" ref="ordinalpos=4780&amp;ncbi_uid=3555613&amp;link_uid=PMC3186365" image-link="/pmc/articles/PMC3186365/figure/F9/" class="imagepopup">FIGURE 9. From: Fibroblast Growth Factor 21 Induces Glucose Transporter-1 Expression through Activation of the Serum Response Factor/Ets-Like Protein-1 in Adipocytes. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the molecular <span class="highlight" style="background-color:">pathway</span> by which FGF21 induces GLUT1 expression through the activation of SRE/ETS <span class="highlight" style="background-color:">signaling</span> cascade in adipocyte.</b> <i>FGFR</i>, FGF21 receptor.</div></p></div><div class="aux"><div class="resc">Xuan Ge, et al. J Biol Chem. 2011 October 7;286(40):34533-34541.</div><p class="links"><a class="dblinks" href="/pubmed/21846717" ref="ordinalpos=4780&amp;ncbi_uid=3555613&amp;link_uid=21846717">Citation</a><a class="dblinks" href="/pmc/articles/PMC3186365/" ref="ordinalpos=4780&amp;ncbi_uid=3555613&amp;link_uid=PMC3186365">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4781.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3438932/figure/F5/" ref="ordinalpos=4781&amp;ncbi_uid=5275301&amp;link_uid=PMC3438932" href="/pmc/articles/PMC3438932/"><img src="/pmc/articles/instance/3438932/bin/zbc0391223260005.gif" src-large="/pmc/articles/instance/3438932/bin/zbc0391223260005.jpg" alt="FIGURE 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3438932/" ref="ordinalpos=4781&amp;ncbi_uid=5275301&amp;link_uid=PMC3438932" image-link="/pmc/articles/PMC3438932/figure/F5/" class="imagepopup">FIGURE 5. From: Toll/Interleukin-1 Receptor Domain Dimers as the Platform for Activation and Enhanced Inhibition of Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Fusion of a designed coiled-coil with mTIR creates inhibitor with broad TLR and IL-1R inhibitory spectrum.</b> <i>A–C</i>, GCN-mTIR is a strong inhibitor of TLR5, TLR9 and TLR3 <span class="highlight" style="background-color:">signaling</span>. HEK293 cells were transfected with mTIR, cc-mTIR or GCN-mTIR plasmids (1, 5, 10 ng) and either TLR5 plasmid (20 ng) (<i>A</i>), TLR9 plasmid (10 ng) and Unc93B1 plasmid (1 ng) (<i>B</i>) or TLR3 plasmid (20 ng) (<i>C</i>). Cells were stimulated with flagellin (10 ng/ml), ODN (3 μ<span style="font-variant: small-caps" class="small-caps">m</span>) or poly(I:C) (10 μg/ml). <i>D</i>, GCN-mTIR inhibits <span class="highlight" style="background-color:">signaling</span> triggered by overexpression of TRIF. Cells were transfected with control or TRIF plasmid (2 ng) or cotransfected with TRIF (2 ng) and GCN-mTIR plasmid (10, 30, 50 ng). <i>E</i>, coiled-coils linked TIR proteins improve inhibition of IL-1R <span class="highlight" style="background-color:">signaling</span>. HEK293 cells were transfected with mTIR, cc-mTIR or GCN-mTIR plasmids (1, 5, 10 ng) and stimulated with IL-1β (50 ng/ml). <i>F</i>, TNFα-R <span class="highlight" style="background-color:">signaling</span> is unaffected by TIR fusion proteins. HEK293 cells were transfected with mTIR, cc-mTIR or GCN-mTIR plasmids (5, 35, 75 ng) and stimulated with TNF-α (100 ng/ml). Activation of TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was determined by the dual luciferase assay. Representative graphs are shown from three separate experiments. Data on graphs are represented as mean ± S.D.</div></p></div><div class="aux"><div class="resc">Ota Fekonja, et al. J Biol Chem. 2012 September 7;287(37):30993-31002.</div><p class="links"><a class="dblinks" href="/pubmed/22829600" ref="ordinalpos=4781&amp;ncbi_uid=5275301&amp;link_uid=22829600">Citation</a><a class="dblinks" href="/pmc/articles/PMC3438932/" ref="ordinalpos=4781&amp;ncbi_uid=5275301&amp;link_uid=PMC3438932">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4782.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2981059/figure/fig6/" ref="ordinalpos=4782&amp;ncbi_uid=3331033&amp;link_uid=PMC2981059" href="/pmc/articles/PMC2981059/"><img src="/pmc/articles/instance/2981059/bin/carcinbgq188f06_4c.gif" src-large="/pmc/articles/instance/2981059/bin/carcinbgq188f06_4c.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2981059/" ref="ordinalpos=4782&amp;ncbi_uid=3331033&amp;link_uid=PMC2981059" image-link="/pmc/articles/PMC2981059/figure/fig6/" class="imagepopup">Fig. 6. From: Secretory phospholipase A2-IIa is involved in prostate cancer progression and may potentially serve as a biomarker for prostate cancer. </a></p><div class="supp"><p class="details"><div>The HER/HER2-PI3K-Akt-NF-κB-elicited <span class="highlight" style="background-color:">pathway</span> in prostate cancer cells. PI3K-Akt <span class="highlight" style="background-color:">signaling</span> cross talks with the NF-κB-mediated <span class="highlight" style="background-color:">pathway</span> by Akt phosphorylation and activation of IKK, an upstream kinase of NF-κB. Small molecule inhibitors Erlotinib, Gefitinib, Lapatinib, CI-1033, LY294002, Bortezomib, cFLSYR and c(2NapA)LS(2NapA)R against EGFR, HER2, PI3K, NF-κB and sPLA2 are indicated.</div></p></div><div class="aux"><div class="resc">Zhongyun Dong, et al. Carcinogenesis. 2010 November;31(11):1948-1955.</div><p class="links"><a class="dblinks" href="/pubmed/20837598" ref="ordinalpos=4782&amp;ncbi_uid=3331033&amp;link_uid=20837598">Citation</a><a class="dblinks" href="/pmc/articles/PMC2981059/" ref="ordinalpos=4782&amp;ncbi_uid=3331033&amp;link_uid=PMC2981059">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4783.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2606916/figure/F1/" ref="ordinalpos=4783&amp;ncbi_uid=1612307&amp;link_uid=PMC2606916" href="/pmc/articles/PMC2606916/"><img src="/pmc/articles/instance/2606916/bin/nihms46332f1.gif" src-large="/pmc/articles/instance/2606916/bin/nihms46332f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2606916/" ref="ordinalpos=4783&amp;ncbi_uid=1612307&amp;link_uid=PMC2606916" image-link="/pmc/articles/PMC2606916/figure/F1/" class="imagepopup">Figure 1. NF-κB <span class="highlight" style="background-color:">signaling</span> pathways.  From: Focus on Molecules: Nuclear Factor-kappaB. </a></p><div class="supp"><p class="details"><div>In canonical <span class="highlight" style="background-color:">pathway</span> (left), the binding of specific ligand to a receptor leads to the phosphorylation of IκB by IKK complex comprising of IKKα, IKKβ and NEMO which releases active NF-κB dimer that translocates in to nucleus to transcribe target genes. In non-canonical <span class="highlight" style="background-color:">pathway</span> (right), the binding of ligand to receptor leads to processing of p100/RelB via NIK mediated activation of IKK complex comprising of two units of IKKα. The processed p52/RelB translocates in to the nucleus to transcribe target genes.         </div></p></div><div class="aux"><div class="resc">Satish K. Srivastava, et al. Exp Eye Res. ;88(1):2-3.</div><p class="links"><a class="dblinks" href="/pubmed/18472097" ref="ordinalpos=4783&amp;ncbi_uid=1612307&amp;link_uid=18472097">Citation</a><a class="dblinks" href="/pmc/articles/PMC2606916/" ref="ordinalpos=4783&amp;ncbi_uid=1612307&amp;link_uid=PMC2606916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4784.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3551737/figure/F8/" ref="ordinalpos=4784&amp;ncbi_uid=4556347&amp;link_uid=PMC3551737" href="/pmc/articles/PMC3551737/"><img src="/pmc/articles/instance/3551737/bin/1748-717X-7-205-8.gif" src-large="/pmc/articles/instance/3551737/bin/1748-717X-7-205-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3551737/" ref="ordinalpos=4784&amp;ncbi_uid=4556347&amp;link_uid=PMC3551737" image-link="/pmc/articles/PMC3551737/figure/F8/" class="imagepopup">Figure 8.  From: Exposure to ionizing radiation induced persistent gene expression changes in mouse mammary gland. </a></p><div class="supp"><p class="details"><div><b>Cell-to-cell <span class="highlight" style="background-color:">signaling</span> and molecular transport related to carcinogenesis were upregulated.</b><b>A</b>) With a score of 14 this <span class="highlight" style="background-color:">pathway</span> network is associated with cellular function and maintenance, molecular transport, cellular assembly and organization. <b>B</b>) Eleven genes from the microarray data set were associated with this <span class="highlight" style="background-color:">pathway</span> and identified cell death, cell-to-cell <span class="highlight" style="background-color:">signaling</span> and interaction, nervous system development and function as major functions.</div></p></div><div class="aux"><div class="resc">Kamal Datta, et al. Radiat Oncol. 2012;7:205-205.</div><p class="links"><a class="dblinks" href="/pubmed/23216862" ref="ordinalpos=4784&amp;ncbi_uid=4556347&amp;link_uid=23216862">Citation</a><a class="dblinks" href="/pmc/articles/PMC3551737/" ref="ordinalpos=4784&amp;ncbi_uid=4556347&amp;link_uid=PMC3551737">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4785.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3841102/figure/pbio-1001721-g006/" ref="ordinalpos=4785&amp;ncbi_uid=5560757&amp;link_uid=PMC3841102" href="/pmc/articles/PMC3841102/"><img src="/pmc/articles/instance/3841102/bin/pbio.1001721.g006.gif" src-large="/pmc/articles/instance/3841102/bin/pbio.1001721.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3841102/" ref="ordinalpos=4785&amp;ncbi_uid=5560757&amp;link_uid=PMC3841102" image-link="/pmc/articles/PMC3841102/figure/pbio-1001721-g006/" class="imagepopup">Figure 6. Bidirectional regulation and role of activated Smurf in the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Activation of Smurf E3 Ligase Promoted by Smoothened Regulates Hedgehog <span class="highlight" style="background-color:">Signaling</span> through Targeting Patched Turnover. </a></p><div class="supp"><p class="details"><div>(A) A simplified Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> network for mathematic modeling. (B) The effects of over expression of activated Smurf on [Hh], [Ptc], [Hh_Ptc], [activated Smo/Ci]. On the left, the levels of [Hh], [Ptc], [Hh_Ptc], and [activated Smo/Ci] correspond to the low level of [activated Smurf], and, similarly, on the right, the levels of [Hh], [Ptc], [Hh_Ptc], and [activated Smo/Ci] correspond to the high level of [activated Smurf] (i.e., over expression of activated Smurf). (For the value of each parameter, see Table S1.) (C) The effect of over expression of activated Smurf on the ratio [Hh_Ptc]/[Ptc]. For all three values of <i>β<sub>Hh_Ptc</sub></i>, the level of ratio [Hh_Ptc]/[Ptc] corresponding to the low activated Smurf is low (on the left). However, when the activated Smurf is over expressed, the level of ratio [Hh_Ptc]/[Ptc] corresponding to <i>β<sub>Hh_Ptc</sub></i> = 10<sup>−4</sup> is significantly high. (For the value of each parameter, see Table S1.) (D) The effects of the inhibition of activated Smurf on [Ptc] and [activated Smo/Ci]. When the expression of activated Smurf is inhibited, the levels of [Ptc] and [activated Smo/Ci] decrease immediately (on the right). (For the value of each parameter, see Table S1.) (E and F–F″) Wild-type control discs were immunostained with anti-Ptc antibody to show Ptc protein expression (E), and wing discs expressing Flag:Smurf driven by <i>ap-gal4</i> were immunostained to show the expression of Ptc (red) and GFP(green), which shows the area of Smurf expression (F–F″). (G and H–H″) Wild-type control discs were immunostained with anti-Ptc antibody to show Ptc protein expression (G), and wing discs expressing Src:Flag:Smurf driven by <i>ap-gal4</i> were immunostained to show the expression of Ptc (red) and Flag (green) (H–H″).</div></p></div><div class="aux"><div class="resc">Shoujun Huang, et al. PLoS Biol. 2013 November;11(11):e1001721.</div><p class="links"><a class="dblinks" href="/pubmed/24302888" ref="ordinalpos=4785&amp;ncbi_uid=5560757&amp;link_uid=24302888">Citation</a><a class="dblinks" href="/pmc/articles/PMC3841102/" ref="ordinalpos=4785&amp;ncbi_uid=5560757&amp;link_uid=PMC3841102">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4786.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4045758/figure/pone-0098653-g003/" ref="ordinalpos=4786&amp;ncbi_uid=6232473&amp;link_uid=PMC4045758" href="/pmc/articles/PMC4045758/"><img src="/pmc/articles/instance/4045758/bin/pone.0098653.g003.gif" src-large="/pmc/articles/instance/4045758/bin/pone.0098653.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4045758/" ref="ordinalpos=4786&amp;ncbi_uid=6232473&amp;link_uid=PMC4045758" image-link="/pmc/articles/PMC4045758/figure/pone-0098653-g003/" class="imagepopup">Figure 3. Biological functions tackled in modules of CNV-ICC-TRN..  From: Integrative Analysis of Transcriptional Regulatory Network and Copy Number Variation in Intrahepatic Cholangiocarcinoma. </a></p><div class="supp"><p class="details"><div>X-axis represents <span class="highlight" style="background-color:">signaling</span> pathways and IDs, y-axis represents the number of modules enriching to each <span class="highlight" style="background-color:">pathway</span>. Complete information is shown in Table S3.</div></p></div><div class="aux"><div class="resc">Ling Li, et al. PLoS One. 2014;9(6):e98653.</div><p class="links"><a class="dblinks" href="/pubmed/24897108" ref="ordinalpos=4786&amp;ncbi_uid=6232473&amp;link_uid=24897108">Citation</a><a class="dblinks" href="/pmc/articles/PMC4045758/" ref="ordinalpos=4786&amp;ncbi_uid=6232473&amp;link_uid=PMC4045758">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4787.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3199188/figure/fig1/" ref="ordinalpos=4787&amp;ncbi_uid=3319357&amp;link_uid=PMC3199188" href="/pmc/articles/PMC3199188/"><img src="/pmc/articles/instance/3199188/bin/CDI2011-947858.001.gif" src-large="/pmc/articles/instance/3199188/bin/CDI2011-947858.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3199188/" ref="ordinalpos=4787&amp;ncbi_uid=3319357&amp;link_uid=PMC3199188" image-link="/pmc/articles/PMC3199188/figure/fig1/" class="imagepopup">Figure 1.  From: PI3K Functions in Cancer Progression, Anticancer Immunity and Immune Evasion by Tumors. </a></p><div class="supp"><p class="details"><div>Schematic model of the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in the regulation of a broad range of cellular activities in both immune system and cancer.</div></p></div><div class="aux"><div class="resc">Francesco Dituri, et al. Clin Dev Immunol. 2011;2011:947858.</div><p class="links"><a class="dblinks" href="/pubmed/22046194" ref="ordinalpos=4787&amp;ncbi_uid=3319357&amp;link_uid=22046194">Citation</a><a class="dblinks" href="/pmc/articles/PMC3199188/" ref="ordinalpos=4787&amp;ncbi_uid=3319357&amp;link_uid=PMC3199188">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4788.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3855293/figure/F4/" ref="ordinalpos=4788&amp;ncbi_uid=5572796&amp;link_uid=PMC3855293" href="/pmc/articles/PMC3855293/"><img src="/pmc/articles/instance/3855293/bin/nihms525393f4a.gif" src-large="/pmc/articles/instance/3855293/bin/nihms525393f4a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3855293/figure/F4/" ref="ordinalpos=4788&amp;ncbi_uid=5572796&amp;link_uid=PMC3855293" href="/pmc/articles/PMC3855293/"><img src="/pmc/articles/instance/3855293/bin/nihms525393f4b.gif" src-large="/pmc/articles/instance/3855293/bin/nihms525393f4b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855293/" ref="ordinalpos=4788&amp;ncbi_uid=5572796&amp;link_uid=PMC3855293" image-link="/pmc/articles/PMC3855293/figure/F4/" class="imagepopup">Figure 4.  From: Gene expression profile in circulating mononuclear cells after exposure to ultrafine carbon particles. </a></p><div class="supp"><p class="details"><div>Top 6 <span class="highlight" style="background-color:">signaling</span> pathways (A) and top 6 toxicological pathways (B) based on the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis.</div></p></div><div class="aux"><div class="resc">Yuh-Chin T. Huang, et al. Inhal Toxicol. ;22(10):10.3109/08958378.2010.486419.</div><p class="links"><a class="dblinks" href="/pubmed/20507211" ref="ordinalpos=4788&amp;ncbi_uid=5572796&amp;link_uid=20507211">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855293/" ref="ordinalpos=4788&amp;ncbi_uid=5572796&amp;link_uid=PMC3855293">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4789.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807436/figure/F3/" ref="ordinalpos=4789&amp;ncbi_uid=2214330&amp;link_uid=PMC2807436" href="/pmc/articles/PMC2807436/"><img src="/pmc/articles/instance/2807436/bin/1752-0509-3-118-3.gif" src-large="/pmc/articles/instance/2807436/bin/1752-0509-3-118-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807436/" ref="ordinalpos=4789&amp;ncbi_uid=2214330&amp;link_uid=PMC2807436" image-link="/pmc/articles/PMC2807436/figure/F3/" class="imagepopup">Figure 3.  From: Computational modeling of the EGFR network elucidates control mechanisms regulating signal dynamics. </a></p><div class="supp"><p class="details"><div><b>Simplified graphical representation of the modules in the <span class="highlight" style="background-color:">pathway</span> with interface components</b>. Colored lines indicate the boundaries of the modules and the edge labels indicate reactions that are crucial to <span class="highlight" style="background-color:">signaling</span> between the molecular components.</div></p></div><div class="aux"><div class="resc">Dennis YQ Wang, et al. BMC Syst Biol. 2009;3:118-118.</div><p class="links"><a class="dblinks" href="/pubmed/20028552" ref="ordinalpos=4789&amp;ncbi_uid=2214330&amp;link_uid=20028552">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807436/" ref="ordinalpos=4789&amp;ncbi_uid=2214330&amp;link_uid=PMC2807436">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4790.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4021040/figure/F2/" ref="ordinalpos=4790&amp;ncbi_uid=6137181&amp;link_uid=PMC4021040" href="/pmc/articles/PMC4021040/"><img src="/pmc/articles/instance/4021040/bin/nihms-318070-f0002.gif" src-large="/pmc/articles/instance/4021040/bin/nihms-318070-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4021040/" ref="ordinalpos=4790&amp;ncbi_uid=6137181&amp;link_uid=PMC4021040" image-link="/pmc/articles/PMC4021040/figure/F2/" class="imagepopup">Figure 2. The Akt <span class="highlight" style="background-color:">pathway</span> is one of the effector mechanism of EGF/EGFR but warrants consideration as a stand-alone target.  From: Targeting glioma stem cells: a novel framework for brain tumors. </a></p><div class="supp"><p class="details"><div>Akt is a serine-threonine kinase that is part of the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. GSC are more dependent on Akt <span class="highlight" style="background-color:">signaling</span> than non-GSC, rendering Akt a potential target preferentially affecting GSC. Upon EGFR activation, PI3K stimulates the production of PIP3 from PIP2, that, in turn, stimulates the phosphorylation of Akt. Akt is a critical regulator of multiple cellular functions, such as proliferation, survival and metabolism. PTEN can directly counteract and reverse the effect of PI3K. Both EGFR and Akt may be inhibited by drugs used in the clinical setting. Erlotinib and gefitinib inhibit EGFR, while perifosine inhibits Akt. Abbreviations: PIP2= phosphatidylinositol-4-5-bisphosphate; PIP3= phosphatidylinositol-3,4,5-triphosphate; p=phosphorylated.</div></p></div><div class="aux"><div class="resc">Emanuela Binello, et al. Cancer Sci. 2011 November;102(11):1958-1966.</div><p class="links"><a class="dblinks" href="/pubmed/21848914" ref="ordinalpos=4790&amp;ncbi_uid=6137181&amp;link_uid=21848914">Citation</a><a class="dblinks" href="/pmc/articles/PMC4021040/" ref="ordinalpos=4790&amp;ncbi_uid=6137181&amp;link_uid=PMC4021040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4791.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3252742/figure/fig4/" ref="ordinalpos=4791&amp;ncbi_uid=3745113&amp;link_uid=PMC3252742" href="/pmc/articles/PMC3252742/"><img src="/pmc/articles/instance/3252742/bin/cddis2011130f4.gif" src-large="/pmc/articles/instance/3252742/bin/cddis2011130f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3252742/" ref="ordinalpos=4791&amp;ncbi_uid=3745113&amp;link_uid=PMC3252742" image-link="/pmc/articles/PMC3252742/figure/fig4/" class="imagepopup">Figure 4.  From: Cardiomyocyte death: mechanisms and translational implications. </a></p><div class="supp"><p class="details"><div>Apoptotic <span class="highlight" style="background-color:">pathway</span> in cardiac myocytes. In the extrinsic <span class="highlight" style="background-color:">pathway</span>, death receptor activation by a death ligand induces death-inducing <span class="highlight" style="background-color:">signaling</span> complex (DISC) formation and casp 8 activation, which in turn activates casp 3. This <span class="highlight" style="background-color:">pathway</span> can also activate the intrinsic <span class="highlight" style="background-color:">pathway</span> by the proteolysis of BID to t-BID by casp 8 and interaction of t-BID with BAX in the mitochondria. Pro-apoptotic BAX/BAK induces cyto <i>c</i>, Smac/DIABLO, AIF, and Endo G release from the mitochondria. Cyto <i>c</i> with Apaf1 and casp 9 form the apoptosome with activation of casp 9. Casp activity is regulated by the endogenous casp inhibitor XIAP. Cardiac myocytes are naturally resistant to apoptosis due to their low-level expression of Apaf1 and casps and high levels of XIAP</div></p></div><div class="aux"><div class="resc">M Chiong, et al. Cell Death Dis. 2011 December;2(12):e244.</div><p class="links"><a class="dblinks" href="/pubmed/22190003" ref="ordinalpos=4791&amp;ncbi_uid=3745113&amp;link_uid=22190003">Citation</a><a class="dblinks" href="/pmc/articles/PMC3252742/" ref="ordinalpos=4791&amp;ncbi_uid=3745113&amp;link_uid=PMC3252742">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4792.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3257109/figure/f3-ijms-12-08924/" ref="ordinalpos=4792&amp;ncbi_uid=3769425&amp;link_uid=PMC3257109" href="/pmc/articles/PMC3257109/"><img src="/pmc/articles/instance/3257109/bin/ijms-12-08924f3.gif" src-large="/pmc/articles/instance/3257109/bin/ijms-12-08924f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3257109/" ref="ordinalpos=4792&amp;ncbi_uid=3769425&amp;link_uid=PMC3257109" image-link="/pmc/articles/PMC3257109/figure/f3-ijms-12-08924/" class="imagepopup">Figure 3.  From: The Wnt and BMP Families of <span class="highlight" style="background-color:">Signaling</span> Morphogens at the Vertebrate Neuromuscular Junction. </a></p><div class="supp"><p class="details"><div>BMP <span class="highlight" style="background-color:">signaling</span> in motor neurons. (<b>A</b>) In <i>Drosophila</i>, the evidence suggests that a retrograde signal involving homologues of the BMP <span class="highlight" style="background-color:">pathway</span> regulates synaptic growth, assembly and maintenance by pathways related to either Smad or cytoskeleton regulators; (<b>B</b>) In vertebrates, spinal cord injury induces an increase of BMP ligands around the injury site. BMP receptors and Smad1 phosphorylation is also increased in spinal neurons, revealing the ability of BMP to transduce Smad-dependent <span class="highlight" style="background-color:">signaling</span>. However, the target genes induced by Smad pathways or the mechanisms involving local cytoskeletal rearrangement are unknown.</div></p></div><div class="aux"><div class="resc">Juan P. Henríquez, et al. Int J Mol Sci. 2011;12(12):8924-8946.</div><p class="links"><a class="dblinks" href="/pubmed/22272112" ref="ordinalpos=4792&amp;ncbi_uid=3769425&amp;link_uid=22272112">Citation</a><a class="dblinks" href="/pmc/articles/PMC3257109/" ref="ordinalpos=4792&amp;ncbi_uid=3769425&amp;link_uid=PMC3257109">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4793.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4133843/figure/F0002/" ref="ordinalpos=4793&amp;ncbi_uid=6615014&amp;link_uid=PMC4133843" href="/pmc/articles/PMC4133843/"><img src="/pmc/articles/instance/4133843/bin/BMB-46-550-g0002.gif" src-large="/pmc/articles/instance/4133843/bin/BMB-46-550-g0002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4133843/" ref="ordinalpos=4793&amp;ncbi_uid=6615014&amp;link_uid=PMC4133843" image-link="/pmc/articles/PMC4133843/figure/F0002/" class="imagepopup">Fig. 2.miR-15b is regulated by the PDGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Time-course expression of miR-15b and miR-221 normalized to U6 small nuclear RNA was examined by qRT-PCR in PASMC treated with PDGF-BB (20 ng/ml) for the time indicated. (B) Time-course analysis of the relative expression of miR-15b or miR-21 normalized to U6 small nuclear RNA in PASMC stimulated with BMP4 for the time indicated. The average of 3 experiments, each performed in triplicate, is presented with S.E..  From: miR-15b induced by platelet-derived growth factor <span class="highlight" style="background-color:">signaling</span> is required for vascular smooth muscle cell proliferation. </a></p><div class="aux"><div class="resc">Sunghwan Kim, et al. BMB Rep. 2013 November;46(11):550-554.</div><p class="links"><a class="dblinks" href="/pubmed/24152911" ref="ordinalpos=4793&amp;ncbi_uid=6615014&amp;link_uid=24152911">Citation</a><a class="dblinks" href="/pmc/articles/PMC4133843/" ref="ordinalpos=4793&amp;ncbi_uid=6615014&amp;link_uid=PMC4133843">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4794.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2610502/figure/fig9/" ref="ordinalpos=4794&amp;ncbi_uid=1630152&amp;link_uid=PMC2610502" href="/pmc/articles/PMC2610502/"><img src="/pmc/articles/instance/2610502/bin/zbc0060962900009.gif" src-large="/pmc/articles/instance/2610502/bin/zbc0060962900009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2610502/" ref="ordinalpos=4794&amp;ncbi_uid=1630152&amp;link_uid=PMC2610502" image-link="/pmc/articles/PMC2610502/figure/fig9/" class="imagepopup">FIGURE 9. From: How Does Arrestin Assemble MAPKs into a <span class="highlight" style="background-color:">Signaling</span>  Complex?. </a></p><div class="supp"><p class="details"><div><b>Model of the MAPK <span class="highlight" style="background-color:">signaling</span> module organized by arrestin.</b> Front  (<i>A</i>) and side (<i>B</i>) views of the three-dimensional model of  arrestin complex with the three MAP kinases are shown. Arrestin  (<i>green</i>) is shown as an elongated two-domain molecule. All the kinases  are shown with thick cylinders representing kinase domain, and thin ones  representing N- and C-terminal elements. MAPKKK (<i>pink</i>) is represented  as ASK1 dimer in the JNK3 <span class="highlight" style="background-color:">pathway</span> with its C and kinase domain interacting  with arrestin, and MAPK (<i>purple</i>) with its N terminus interacting with  arrestin. MAPKK (<i>yellow</i>) also interacts with both domains of  arrestin.</div></p></div><div class="aux"><div class="resc">Xiufeng Song, et al. J Biol Chem. 2009 January 2;284(1):685-695.</div><p class="links"><a class="dblinks" href="/pubmed/19001375" ref="ordinalpos=4794&amp;ncbi_uid=1630152&amp;link_uid=19001375">Citation</a><a class="dblinks" href="/pmc/articles/PMC2610502/" ref="ordinalpos=4794&amp;ncbi_uid=1630152&amp;link_uid=PMC2610502">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4795.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2847371/figure/F1/" ref="ordinalpos=4795&amp;ncbi_uid=2307008&amp;link_uid=PMC2847371" href="/pmc/articles/PMC2847371/"><img src="/pmc/articles/instance/2847371/bin/nihms168983f1.gif" src-large="/pmc/articles/instance/2847371/bin/nihms168983f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2847371/" ref="ordinalpos=4795&amp;ncbi_uid=2307008&amp;link_uid=PMC2847371" image-link="/pmc/articles/PMC2847371/figure/F1/" class="imagepopup">Figure 1. The NOTCH1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: The role of NOTCH1 <span class="highlight" style="background-color:">signaling</span> in T-ALL. </a></p><div class="supp"><p class="details"><div>Interaction of the NOTCH1 receptor with Delta-like and Jagged ligands expressed on the surface of a neighbor cells triggers the proteolytic cleavage of the receptor, first by an ADAM metalloprotease (S2 cleavage) and subsequently by the γ-secretase complex (S3 cleavage), which releases the intracellular domains of NOTCH1 (ICN1) from the membrane. ICN1 translocates to the nucleus and interacts with DNA via the RBPJ/CSL DNA binding protein and recruits the MAML1 coactivator to activate the expression of NOTCH1 target genes.</div></p></div><div class="aux"><div class="resc">Adolfo A. Ferrando. Hematology Am Soc Hematol Educ Program. :353-361.</div><p class="links"><a class="dblinks" href="/pubmed/20008221" ref="ordinalpos=4795&amp;ncbi_uid=2307008&amp;link_uid=20008221">Citation</a><a class="dblinks" href="/pmc/articles/PMC2847371/" ref="ordinalpos=4795&amp;ncbi_uid=2307008&amp;link_uid=PMC2847371">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4796.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC544021/figure/f6/" ref="ordinalpos=4796&amp;ncbi_uid=444881&amp;link_uid=PMC544021" href="/pmc/articles/PMC544021/"><img src="/pmc/articles/instance/544021/bin/zmb0030547160006.gif" src-large="/pmc/articles/instance/544021/bin/zmb0030547160006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC544021/" ref="ordinalpos=4796&amp;ncbi_uid=444881&amp;link_uid=PMC544021" image-link="/pmc/articles/PMC544021/figure/f6/" class="imagepopup">FIG. 6. From: Regulation of Rho and Rac <span class="highlight" style="background-color:">Signaling</span> to the Actin Cytoskeleton by Paxillin during Drosophila Development. </a></p><div class="supp"><p class="details"><div>Paxillin antagonizes the Rho-LIMK-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Paxillin suppresses LIMK1-induced SRF transcriptional activity in Cos7 cells. (B) Paxillin<sup>−/−</sup> cells exhibit approximately twofold-increased SRF transcriptional activity compared with rescued cells. (C) Paxillin<sup>−/−</sup> cells exhibit decreased Rac activity, as measured in pulldown assays with GST-PAK (PBD). The lower panel shows levels of rac in total cell lysate (TCL). (D) Paxillin<sup>−/−</sup> cells have increased Rho activity. The lower panel shows that levels of rho in different lysates are similar.</div></p></div><div class="aux"><div class="resc">Guang-Chao Chen, et al. Mol Cell Biol. 2005 February;25(3):979-987.</div><p class="links"><a class="dblinks" href="/pubmed/15657426" ref="ordinalpos=4796&amp;ncbi_uid=444881&amp;link_uid=15657426">Citation</a><a class="dblinks" href="/pmc/articles/PMC544021/" ref="ordinalpos=4796&amp;ncbi_uid=444881&amp;link_uid=PMC544021">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4797.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3128083/figure/pone-0021761-g009/" ref="ordinalpos=4797&amp;ncbi_uid=3078346&amp;link_uid=PMC3128083" href="/pmc/articles/PMC3128083/"><img src="/pmc/articles/instance/3128083/bin/pone.0021761.g009.gif" src-large="/pmc/articles/instance/3128083/bin/pone.0021761.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3128083/" ref="ordinalpos=4797&amp;ncbi_uid=3078346&amp;link_uid=PMC3128083" image-link="/pmc/articles/PMC3128083/figure/pone-0021761-g009/" class="imagepopup">Figure 9. Schematic diagram showing RIG-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to production of proinflammatory cyto/chemokines from neurons..  From: RIG-I Mediates Innate Immune Response in Mouse Neurons Following Japanese Encephalitis Virus Infection. </a></p><div class="supp"><p class="details"><div>Detection of viral ss/dsRNA by RIG-I activates a cascade of changes that leads to activation of P38MAPK and NFκB. As a result of such activation, proinflammatory cyto/chemokine gene transcription is enhanced in neurons that ultimately contribute to the inflammatory milieu in the brain, following JEV infection. Ablation of RIG-I using antisense morpholino oligonucleotide results in negative modulation of this <span class="highlight" style="background-color:">pathway</span>. ↑-upregulation; ↓-downregulation; ⊢ -ablation.</div></p></div><div class="aux"><div class="resc">Arshed Nazmi, et al. PLoS One. 2011;6(6):e21761.</div><p class="links"><a class="dblinks" href="/pubmed/21738791" ref="ordinalpos=4797&amp;ncbi_uid=3078346&amp;link_uid=21738791">Citation</a><a class="dblinks" href="/pmc/articles/PMC3128083/" ref="ordinalpos=4797&amp;ncbi_uid=3078346&amp;link_uid=PMC3128083">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4798.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3357286/figure/F7/" ref="ordinalpos=4798&amp;ncbi_uid=4948201&amp;link_uid=PMC3357286" href="/pmc/articles/PMC3357286/"><img src="/pmc/articles/instance/3357286/bin/1434fig7.gif" src-large="/pmc/articles/instance/3357286/bin/1434fig7.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3357286/" ref="ordinalpos=4798&amp;ncbi_uid=4948201&amp;link_uid=PMC3357286" image-link="/pmc/articles/PMC3357286/figure/F7/" class="imagepopup">FIG. 7. From: Amyloid-? Induces Hepatic Insulin Resistance by Activating JAK2/STAT3/SOCS-1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>JAK2/STAT3 <span class="highlight" style="background-color:">signaling</span> is required for the SOCS-1 upregulation induced by Aβ. <i>A</i>: Knockdown of STAT3 by siRNA blocked the upregulation of SOCS-1 by Aβ. <i>B</i>: Inhibition of JAK2 by AG490 blocked the phosphorylation of JAK2 and STAT3 and the upregulation of SOCS-1 induced by Aβ. <i>C</i>: Knockdown of JAK2 by siRNA blocked the phosphorylation of JAK2 and STAT3 induced by Aβ. <i>D</i>: Knockdown of JAK2 by siRNA blocked the upregulation of SOCS-1 induced by Aβ.</div></p></div><div class="aux"><div class="resc">Yi Zhang, et al. Diabetes. 2012 June;61(6):1434-1443.</div><p class="links"><a class="dblinks" href="/pubmed/22522613" ref="ordinalpos=4798&amp;ncbi_uid=4948201&amp;link_uid=22522613">Citation</a><a class="dblinks" href="/pmc/articles/PMC3357286/" ref="ordinalpos=4798&amp;ncbi_uid=4948201&amp;link_uid=PMC3357286">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4799.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766052/figure/ppat-1000650-g008/" ref="ordinalpos=4799&amp;ncbi_uid=2119357&amp;link_uid=PMC2766052" href="/pmc/articles/PMC2766052/"><img src="/pmc/articles/instance/2766052/bin/ppat.1000650.g008.gif" src-large="/pmc/articles/instance/2766052/bin/ppat.1000650.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766052/" ref="ordinalpos=4799&amp;ncbi_uid=2119357&amp;link_uid=PMC2766052" image-link="/pmc/articles/PMC2766052/figure/ppat-1000650-g008/" class="imagepopup">Figure 8. Model of TRAF3 dependent regulation of RIG-I <span class="highlight" style="background-color:">signaling</span> via sequential ubiquitination..  From: The E3 Ubiquitin Ligase Triad3A Negatively Regulates the RIG-I/MAVS <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by Targeting TRAF3 for Degradation. </a></p><div class="supp"><p class="details"><div>At early times after RNA virus infection (4–8h), MAVS interacts with TRAF3 via its TRAF interacting motif (TIM); TRAF3 is subject to Lys<sup>63</sup> polyubiquitination, leading to recruitment of the downstream kinases TBK1/IKKε and production of type I IFN. Subsequently, DUBA removes Lys<sup>63</sup> polyubiquitination from TRAF3 which dissociates the TRAF3-TBK1 <span class="highlight" style="background-color:">signaling</span> complex. At late times after infection (12–16h), Triad3A physically associates TRAF3 via its TIM which promotes Lys<sup>48</sup> polyubiquitination of TRAF3, and subsequent proteasomal degradation.</div></p></div><div class="aux"><div class="resc">Peyman Nakhaei, et al. PLoS Pathog. 2009 November;5(11):e1000650.</div><p class="links"><a class="dblinks" href="/pubmed/19893624" ref="ordinalpos=4799&amp;ncbi_uid=2119357&amp;link_uid=19893624">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766052/" ref="ordinalpos=4799&amp;ncbi_uid=2119357&amp;link_uid=PMC2766052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4800.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2096568/figure/Fig4/" ref="ordinalpos=4800&amp;ncbi_uid=1107658&amp;link_uid=PMC2096568" href="/pmc/articles/PMC2096568/"><img src="/pmc/articles/instance/2096568/bin/11302_2004_Article_4748_Fig4.gif" src-large="/pmc/articles/instance/2096568/bin/11302_2004_Article_4748_Fig4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2096568/" ref="ordinalpos=4800&amp;ncbi_uid=1107658&amp;link_uid=PMC2096568" image-link="/pmc/articles/PMC2096568/figure/Fig4/" class="imagepopup">Figure 4.  From: Delineation of ligand binding and receptor <span class="highlight" style="background-color:">signaling</span> activities of purified P2Y receptors reconstituted with heterotrimeric G proteins. </a></p><div class="supp"><p class="details"><div>Schematic representation of the P2Y<sub>1</sub>- or P2Y<sub>12</sub>-activated G protein cycle. Agonist activation of either the P2Y<sub>1</sub> or the P2Y<sub>12</sub> receptor causes an increase in the rate of guanine nucleotide exchange on Gα<sub>q</sub> or Gα<sub>i</sub>, respectively, whereby GTP replaces GDP, creating the active species Gα-GTP. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is turned off by hydrolysis of bound GTP to GDP by the G protein. GAPs, such as RGS proteins, or PLC-β in the case of Gα<sub>q</sub>, augment the rate of GTP hydrolysis. Hence, two important rate limiting steps exist in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: (1) the rate of guanine nucleotide exchange (regulated by the agonist-activated receptor acting as a GEF) and (2) the rate of GTP hydrolysis (regulated by GAPs).</div></p></div><div class="aux"><div class="resc">Erik T. Bodor, et al. Purinergic Signal. 2004 December;1(1):43-49.</div><p class="links"><a class="dblinks" href="/pubmed/18404399" ref="ordinalpos=4800&amp;ncbi_uid=1107658&amp;link_uid=18404399">Citation</a><a class="dblinks" href="/pmc/articles/PMC2096568/" ref="ordinalpos=4800&amp;ncbi_uid=1107658&amp;link_uid=PMC2096568">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="47" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="48" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="49" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:50:34-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB45F454313CD100000000006D2212;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB45F454313CD100000000006D2212" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>