
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="20" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC17C253FF8411000000000005303D" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="19" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="20" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="21" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="20" /></div>
            <div><h2 class="result_count">Results: 1901 to 2000 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1901.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3475297/figure/F3/" ref="ordinalpos=1901&amp;ncbi_uid=4306081&amp;link_uid=PMC3475297" href="/pmc/articles/PMC3475297/"><img src="/pmc/articles/instance/3475297/bin/fgene-03-00218-g0003.gif" src-large="/pmc/articles/instance/3475297/bin/fgene-03-00218-g0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3475297/" ref="ordinalpos=1901&amp;ncbi_uid=4306081&amp;link_uid=PMC3475297" image-link="/pmc/articles/PMC3475297/figure/F3/" class="imagepopup">Figure 3.  From: Regulation of lifespan by chemosensory and thermosensory systems: findings in invertebrates and their implications in mammalian aging. </a></p><div class="supp"><p class="details"><div><b>Implication of chemosensory regulation in human aging.</b> Food perception by sensory systems in humans appears to increase human insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span>. Because the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is known to regulate mammalian lifespan, perhaps human chemosensation also influences aging via the insulin/IGF-1 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Dae-Eun Jeong, et al. Front Genet. 2012;3:218.</div><p class="links"><a class="dblinks" href="/pubmed/23087711" ref="ordinalpos=1901&amp;ncbi_uid=4306081&amp;link_uid=23087711">Citation</a><a class="dblinks" href="/pmc/articles/PMC3475297/" ref="ordinalpos=1901&amp;ncbi_uid=4306081&amp;link_uid=PMC3475297">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1902.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1615341/figure/F9-4036/" ref="ordinalpos=1902&amp;ncbi_uid=890798&amp;link_uid=PMC1615341" href="/pmc/articles/PMC1615341/"><img src="/pmc/articles/instance/1615341/bin/zjh0040440360009.gif" src-large="/pmc/articles/instance/1615341/bin/zjh0040440360009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1615341/" ref="ordinalpos=1902&amp;ncbi_uid=890798&amp;link_uid=PMC1615341" image-link="/pmc/articles/PMC1615341/figure/F9-4036/" class="imagepopup">Figure 9.  From: Advanced Glycation End Products Induce Tubular Epithelial-Myofibroblast Transition through the RAGE-ERK1/2 MAP Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Western blot analysis demonstrates that AGEs signal through RAGE to activate the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>a:</b> A representative Western blot shows that AGE-BSA (30 Î¼g/ml) induces ERK1/2 phosphorylation in NRK52E in a time-dependent manner, being significant at 15 minutes. <b>b:</b> AGEs, but not BSA, induce ERK1/2 phosphorylation (at 30 minutes) in a dose-dependent manner and this is blocked by both sRAGE and a specific ERK1/2 inhibitor, PD 98059. Each sample represents the results from three independent experiments.</div></p></div><div class="aux"><div class="resc">Jin H. Li, et al. Am J Pathol. 2004 April;164(4):1389-1397.</div><p class="links"><a class="dblinks" href="/pubmed/15039226" ref="ordinalpos=1902&amp;ncbi_uid=890798&amp;link_uid=15039226">Citation</a><a class="dblinks" href="/pmc/articles/PMC1615341/" ref="ordinalpos=1902&amp;ncbi_uid=890798&amp;link_uid=PMC1615341">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1903.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4300945/figure/F7/" ref="ordinalpos=1903&amp;ncbi_uid=7087791&amp;link_uid=PMC4300945" href="/pmc/articles/PMC4300945/"><img src="/pmc/articles/instance/4300945/bin/nihms648758f7.gif" src-large="/pmc/articles/instance/4300945/bin/nihms648758f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4300945/" ref="ordinalpos=1903&amp;ncbi_uid=7087791&amp;link_uid=PMC4300945" image-link="/pmc/articles/PMC4300945/figure/F7/" class="imagepopup">Fig. 7.  From: Cross-Talk Between STAT1 and PI3K/AKT <span class="highlight" style="background-color:">Signaling</span> in HIV-1-Induced BloodâBrain Barrier Dysfunction: Role of CCR5 and Implications for Viral Neuropathogenesis. </a></p><div class="supp"><p class="details"><div>Hypothetical model for cross-talk of PI3K/AKT with HIV-1-induced STAT1 <span class="highlight" style="background-color:">signaling</span>. The arrows with solid lines indicate activation. Dotted line represents the classical JAK/STAT or PI3K <span class="highlight" style="background-color:">pathway</span>. Open arrows lines represent cross-talk between the PI3K and JAK/STAT pathways. The â¥ symbol indicates an inhibition effect.</div></p></div><div class="aux"><div class="resc">Bo Yang, et al. J Neurosci Res. ;88(14):3090-3101.</div><p class="links"><a class="dblinks" href="/pubmed/20568281" ref="ordinalpos=1903&amp;ncbi_uid=7087791&amp;link_uid=20568281">Citation</a><a class="dblinks" href="/pmc/articles/PMC4300945/" ref="ordinalpos=1903&amp;ncbi_uid=7087791&amp;link_uid=PMC4300945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1904.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3890654/figure/F2/" ref="ordinalpos=1904&amp;ncbi_uid=5730018&amp;link_uid=PMC3890654" href="/pmc/articles/PMC3890654/"><img src="/pmc/articles/instance/3890654/bin/fimmu-04-00512-g002.gif" src-large="/pmc/articles/instance/3890654/bin/fimmu-04-00512-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3890654/" ref="ordinalpos=1904&amp;ncbi_uid=5730018&amp;link_uid=PMC3890654" image-link="/pmc/articles/PMC3890654/figure/F2/" class="imagepopup">Figure 2.  From: Modulation of Intestinal TLR4-Inflammatory <span class="highlight" style="background-color:">Signaling</span> Pathways by Probiotic Microorganisms: Lessons Learned from Lactobacillus jensenii TL2937. </a></p><div class="supp"><p class="details"><div><b>Modulation of negative regulators of toll-like receptor-4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by <i>Lactobacillus jensenii</i>âTL2937 in porcine intestinal epithelial cells and antigen presenting cells</b>.</div></p></div><div class="aux"><div class="resc">Julio Villena, et al. Front Immunol. 2013;4:512.</div><p class="links"><a class="dblinks" href="/pubmed/24459463" ref="ordinalpos=1904&amp;ncbi_uid=5730018&amp;link_uid=24459463">Citation</a><a class="dblinks" href="/pmc/articles/PMC3890654/" ref="ordinalpos=1904&amp;ncbi_uid=5730018&amp;link_uid=PMC3890654">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1905.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262490/figure/F2/" ref="ordinalpos=1905&amp;ncbi_uid=3767252&amp;link_uid=PMC3262490" href="/pmc/articles/PMC3262490/"><img src="/pmc/articles/instance/3262490/bin/1471-2407-11-502-2.gif" src-large="/pmc/articles/instance/3262490/bin/1471-2407-11-502-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262490/" ref="ordinalpos=1905&amp;ncbi_uid=3767252&amp;link_uid=PMC3262490" image-link="/pmc/articles/PMC3262490/figure/F2/" class="imagepopup">Figure 2.  From: Current views on the role of Notch <span class="highlight" style="background-color:">signaling</span> and the pathogenesis of human leukemia. </a></p><div class="supp"><p class="details"><div><b>The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The initiation of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> begins when the Notch ligand binds to the Notch receptor. This action triggers two proteolytic cleavages by ADAM-type protease (S2) and Î³-secretase (S3), respectively. Following cleavages, the activated form of Notch is released (NICD) and is translocated to the nucleus, where NICD forms complexes with specific DNA-binding proteins (CBF1/Suppressor of Hairless/LAG-1 and Mastermind/SEL-8). Afterward the transcriptional process of target genes is initiated. MAML1: Mastermind-like 1 protein; CBF1: DNA-binding transcription factor.</div></p></div><div class="aux"><div class="resc">Joanna Pancewicz, et al. BMC Cancer. 2011;11:502-502.</div><p class="links"><a class="dblinks" href="/pubmed/22128846" ref="ordinalpos=1905&amp;ncbi_uid=3767252&amp;link_uid=22128846">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262490/" ref="ordinalpos=1905&amp;ncbi_uid=3767252&amp;link_uid=PMC3262490">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1906.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4346788/figure/f4/" ref="ordinalpos=1906&amp;ncbi_uid=7385694&amp;link_uid=PMC4346788" href="/pmc/articles/PMC4346788/"><img src="/pmc/articles/instance/4346788/bin/srep08689-f4.gif" src-large="/pmc/articles/instance/4346788/bin/srep08689-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4346788/" ref="ordinalpos=1906&amp;ncbi_uid=7385694&amp;link_uid=PMC4346788" image-link="/pmc/articles/PMC4346788/figure/f4/" class="imagepopup">Figure 4. Significant and insignificant expression of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes in oocytes injected with either miR-320 inhibitor or NC inhibitor..  From: MiRNA-320 in the human follicular fluid is associated with embryo quality in vivo and affects mouse embryonic development in vitro. </a></p><div class="supp"><p class="details"><div>Histograms depicting six significantly different gene expression (A) and seven insignificant gene expression (B) within or regulate Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> between miR-320 inhibitor-injected (n = 80) and NC inhibitor-injected (n = 50) groups. Injection experiments were performed in triplicate. Unpaired t-test, * represents p &lt; 0.05; ** represents p &lt; 0.01. NC, negative control.</div></p></div><div class="aux"><div class="resc">Ruizhi Feng, et al. Sci Rep. 2015;5:8689.</div><p class="links"><a class="dblinks" href="/pubmed/25732513" ref="ordinalpos=1906&amp;ncbi_uid=7385694&amp;link_uid=25732513">Citation</a><a class="dblinks" href="/pmc/articles/PMC4346788/" ref="ordinalpos=1906&amp;ncbi_uid=7385694&amp;link_uid=PMC4346788">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1907.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2134904/figure/F7/" ref="ordinalpos=1907&amp;ncbi_uid=1188398&amp;link_uid=PMC2134904" href="/pmc/articles/PMC2134904/"><img src="/pmc/articles/instance/2134904/bin/neo0912_1078_fig007.gif" src-large="/pmc/articles/instance/2134904/bin/neo0912_1078_fig007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2134904/" ref="ordinalpos=1907&amp;ncbi_uid=1188398&amp;link_uid=PMC2134904" image-link="/pmc/articles/PMC2134904/figure/F7/" class="imagepopup">Figure 7.  From: Statins Stimulate In Vitro Membrane FasL Expression and Lymphocyte Apoptosis through RhoA/ROCK <span class="highlight" style="background-color:">Pathway</span> in Murine Melanoma Cells. </a></p><div class="supp"><p class="details"><div>Schematic representation of the effects of RhoA/ROCK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibition on membrane FasL expression and lymphocytes apoptosis. <i>In vitro</i> treatments of B16F10 melanoma cells with inhibitors of the RhoA/ROCK <span class="highlight" style="background-color:">pathway</span> increase the membrane FasL expression on these treated tumor cells. This (over)expression promoted the apoptosis of Fas-sensitive lymphocytes.</div></p></div><div class="aux"><div class="resc">Guillaume Sarrabayrouse, et al. Neoplasia. 2007 December;9(12):1078-1090.</div><p class="links"><a class="dblinks" href="/pubmed/18084615" ref="ordinalpos=1907&amp;ncbi_uid=1188398&amp;link_uid=18084615">Citation</a><a class="dblinks" href="/pmc/articles/PMC2134904/" ref="ordinalpos=1907&amp;ncbi_uid=1188398&amp;link_uid=PMC2134904">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1908.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2883292/figure/F2/" ref="ordinalpos=1908&amp;ncbi_uid=2980591&amp;link_uid=PMC2883292" href="/pmc/articles/PMC2883292/"><img src="/pmc/articles/instance/2883292/bin/nihms205988f2.gif" src-large="/pmc/articles/instance/2883292/bin/nihms205988f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2883292/" ref="ordinalpos=1908&amp;ncbi_uid=2980591&amp;link_uid=PMC2883292" image-link="/pmc/articles/PMC2883292/figure/F2/" class="imagepopup">Fig. 2.  From: Biology and Pathology of Rho GTPase, PI-3 Kinase-Akt and MAP Kinase <span class="highlight" style="background-color:">Signaling</span> Pathways in Chondrocytes. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> through the PI-3 Kinase-Akt <span class="highlight" style="background-color:">pathway</span>. Several different types of receptors can activate PI-3 kinase-Akt <span class="highlight" style="background-color:">signaling</span> through upstream mediators including ILK, Src, and IRS1/2. PI-3 kinase can directly activate Akt or indirectly through activation of PDK1 via the conversion of PIP2 to PIP3 which is mediated by PI-3 kinase. Akt phosphorylates numerous downstream <span class="highlight" style="background-color:">signaling</span> proteins resulting in positive regulation of mTOR and negative regulation of GSK-3, FKHR, and Bad among other targets. Inhibition of PI-3 kinase-Akt <span class="highlight" style="background-color:">signaling</span> includes ERK1/2 and JNK1/2 inhibition of IRS-1 as well as Akt inhibition by TRB3. Not shown is the regulation of these pathways by phosphatases which include PTEN, SHP-2, and PP2A, among others.</div></p></div><div class="aux"><div class="resc">Frank Beier, et al. J Cell Biochem. ;110(3):573-580.</div><p class="links"><a class="dblinks" href="/pubmed/20512918" ref="ordinalpos=1908&amp;ncbi_uid=2980591&amp;link_uid=20512918">Citation</a><a class="dblinks" href="/pmc/articles/PMC2883292/" ref="ordinalpos=1908&amp;ncbi_uid=2980591&amp;link_uid=PMC2883292">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1909.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3188430/figure/F7/" ref="ordinalpos=1909&amp;ncbi_uid=3273400&amp;link_uid=PMC3188430" href="/pmc/articles/PMC3188430/"><img src="/pmc/articles/instance/3188430/bin/2044-5040-1-27-7.gif" src-large="/pmc/articles/instance/3188430/bin/2044-5040-1-27-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3188430/" ref="ordinalpos=1909&amp;ncbi_uid=3273400&amp;link_uid=PMC3188430" image-link="/pmc/articles/PMC3188430/figure/F7/" class="imagepopup">Figure 7.  From: Regulation of DMD pathology by an ankyrin-encoded miRNA. </a></p><div class="supp"><p class="details"><div><b>miR-486 is an important regulator of PTEN/AKT <span class="highlight" style="background-color:">signaling</span> in normal and dystrophic muscle</b>. Schematic of miR-486 regulation of PTEN/AKT <span class="highlight" style="background-color:">pathway</span> upstream and downstream <span class="highlight" style="background-color:">signaling</span> components in skeletal muscle is shown. Regulation of the PTEN/AKT <span class="highlight" style="background-color:">pathway</span> in turn regulates FOXO1, which is a well-characterized regulator of p21 and p27 in skeletal muscle.</div></p></div><div class="aux"><div class="resc">Matthew S Alexander, et al. Skelet Muscle. 2011;1:27-27.</div><p class="links"><a class="dblinks" href="/pubmed/21824387" ref="ordinalpos=1909&amp;ncbi_uid=3273400&amp;link_uid=21824387">Citation</a><a class="dblinks" href="/pmc/articles/PMC3188430/" ref="ordinalpos=1909&amp;ncbi_uid=3273400&amp;link_uid=PMC3188430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1910.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC126236/figure/F4/" ref="ordinalpos=1910&amp;ncbi_uid=204189&amp;link_uid=PMC126236" href="/pmc/articles/PMC126236/"><img src="/pmc/articles/instance/126236/bin/gb-2002-3-8-research0042-4.gif" src-large="/pmc/articles/instance/126236/bin/gb-2002-3-8-research0042-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC126236/" ref="ordinalpos=1910&amp;ncbi_uid=204189&amp;link_uid=PMC126236" image-link="/pmc/articles/PMC126236/figure/F4/" class="imagepopup">Figure 4.  From: Identification of genes involved in ceramide-dependent neuronal apoptosis using cDNA arrays. </a></p><div class="supp"><p class="details"><div>Differentially expressed genes that encode proteins with functions involved in ceramide-dependent apoptosis. Black boxes, Genes involved in the ceramide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; gray boxes, genes transcriptionally stimulated by TNF-Î±; white boxes, genes involved in the TNF-Î± <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Charles Decraene, et al. Genome Biol. 2002;3(8):research0042.1-research0042.22.</div><p class="links"><a class="dblinks" href="/pubmed/12186649" ref="ordinalpos=1910&amp;ncbi_uid=204189&amp;link_uid=12186649">Citation</a><a class="dblinks" href="/pmc/articles/PMC126236/" ref="ordinalpos=1910&amp;ncbi_uid=204189&amp;link_uid=PMC126236">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1911.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2238958/figure/F2/" ref="ordinalpos=1911&amp;ncbi_uid=1272602&amp;link_uid=PMC2238958" href="/pmc/articles/PMC2238958/"><img src="/pmc/articles/instance/2238958/bin/gkm659f2.gif" src-large="/pmc/articles/instance/2238958/bin/gkm659f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2238958/" ref="ordinalpos=1911&amp;ncbi_uid=1272602&amp;link_uid=PMC2238958" image-link="/pmc/articles/PMC2238958/figure/F2/" class="imagepopup">Figure 2. From: MutDB: update on development of tools for the biochemical analysis of genetic variation. </a></p><div class="supp"><p class="details"><div>MutDB-KEGG integration example of the VEGF <span class="highlight" style="background-color:">Pathway</span>. This <span class="highlight" style="background-color:">pathway</span> shows all proteins with SNPs or Swiss-Prot mutations and all unique diseases and comments provided by Swiss-Prot (top). The VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing proteins with mutations in yellow (bottom).</div></p></div><div class="aux"><div class="resc">Arti Singh, et al. Nucleic Acids Res. 2008 January;36(Database issue):D815-D819.</div><p class="links"><a class="dblinks" href="/pubmed/17827212" ref="ordinalpos=1911&amp;ncbi_uid=1272602&amp;link_uid=17827212">Citation</a><a class="dblinks" href="/pmc/articles/PMC2238958/" ref="ordinalpos=1911&amp;ncbi_uid=1272602&amp;link_uid=PMC2238958">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1912.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3910974/figure/F4/" ref="ordinalpos=1912&amp;ncbi_uid=6449241&amp;link_uid=PMC3910974" href="/pmc/articles/PMC3910974/"><img src="/pmc/articles/instance/3910974/bin/zek9990942190004.gif" src-large="/pmc/articles/instance/3910974/bin/zek9990942190004.jpg" alt="FIG 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3910974/" ref="ordinalpos=1912&amp;ncbi_uid=6449241&amp;link_uid=PMC3910974" image-link="/pmc/articles/PMC3910974/figure/F4/" class="imagepopup">FIG 4.  From: Sck1 Negatively Regulates Gpa2-Mediated Glucose <span class="highlight" style="background-color:">Signaling</span> in Schizosaccharomyces pombe. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> model showing two distinct roles for Sck1 in nutrient <span class="highlight" style="background-color:">signaling</span>. Work described here and elsewhere suggests that Sck1 is both a negative regulator of cAMP <span class="highlight" style="background-color:">signaling</span> via Gpa2 regulation and a kinase that acts in parallel to PKA, possibly as a sensor of nitrogen levels. Gpa2 activation of PKA is via the activation of adenylyl cyclase (Git2/Cyr1; not shown).</div></p></div><div class="aux"><div class="resc">Dayna K. Mudge, et al. Eukaryot Cell. 2014 February;13(2):202-208.</div><p class="links"><a class="dblinks" href="/pubmed/24297439" ref="ordinalpos=1912&amp;ncbi_uid=6449241&amp;link_uid=24297439">Citation</a><a class="dblinks" href="/pmc/articles/PMC3910974/" ref="ordinalpos=1912&amp;ncbi_uid=6449241&amp;link_uid=PMC3910974">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1913.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4103166/figure/F3/" ref="ordinalpos=1913&amp;ncbi_uid=6391754&amp;link_uid=PMC4103166" href="/pmc/articles/PMC4103166/"><img src="/pmc/articles/instance/4103166/bin/nihms588324f3.gif" src-large="/pmc/articles/instance/4103166/bin/nihms588324f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4103166/" ref="ordinalpos=1913&amp;ncbi_uid=6391754&amp;link_uid=PMC4103166" image-link="/pmc/articles/PMC4103166/figure/F3/" class="imagepopup">Figure 3. Circadian control of intracellular <span class="highlight" style="background-color:">signaling</span>.  From: The Circadian Clock in Cancer Development and Therapy. </a></p><div class="supp"><p class="details"><div>The central pacemaker controlled autonomic nervous and neuroendocrine systems (ANS and NES) rhythmically signal to all of their target tissues. The resulting circadian rhythm in peripheral tissue function also generates local and/or circulating <span class="highlight" style="background-color:">signaling</span> molecules that rhythmically act on their targets. Together, these extracellular signals including neurotransmitters, steroid hormones, peptide hormones, chemokines, growth factors and cytokines activate intracellular <span class="highlight" style="background-color:">signaling</span> mediated by G-protein coupled receptors (GPCRs), tyrosine kinase receptors, integrins (not shown), and nuclear receptors in a tissue and cell-type specific manner. These same intracellular <span class="highlight" style="background-color:">signaling</span> pathways also activate the peripheral clock. The coordinated activities of the central and peripheral clocks orchestrate the complicated extracellular and intracellular <span class="highlight" style="background-color:">signaling</span> to maintain tissue homeostasis by controlling a network of gene expression. Disruption of the central clock-controlled extracellular <span class="highlight" style="background-color:">signaling</span> or mutations in core circadian genes both abolish peripheral clock activity leading to loss of circadian homeostasis in peripheral tissues. The representative intracellular <span class="highlight" style="background-color:">signaling</span> pathways directly or indirectly controlled by the central clock shown in the figure include the c-AMP/PKA/CREB/AP1, Ras/MARK/JNK/ERK and PI3K/AKT/Î²-Catenin/TCF/LEF pathways essential for <i>c-Myc</i> activation and cell cycle progression<sup>478â480</sup>, the PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> controlling biosynthesis and drug resistance<sup>451,481</sup>, the GPCR/ATM <span class="highlight" style="background-color:">signaling</span> for p53 activation<sup>29</sup>, the GPCR/PKC/NF-ÎºB <span class="highlight" style="background-color:">pathway</span> that regulates stress and immune response<sup>482</sup>, the JAK/STAT <span class="highlight" style="background-color:">pathway</span> controlling apoptotic response<sup>483</sup>, and the GR and ERÎ± <span class="highlight" style="background-color:">signaling</span> pathways cross-talking with the AP1 <span class="highlight" style="background-color:">signaling</span><sup>484,485</sup>. These <span class="highlight" style="background-color:">signaling</span> pathways also control the expression and function of circadian genes leading to a coupled activation of the molecular clock with tissue-specific function <i>in vivo</i> including cell proliferation, metabolism, apoptosis, DNA repair, biosynthesis, secretion and senescence<sup>371â374</sup>.</div></p></div><div class="aux"><div class="resc">Loning Fu, et al. Prog Mol Biol Transl Sci. ;119:221-282.</div><p class="links"><a class="dblinks" href="/pubmed/23899600" ref="ordinalpos=1913&amp;ncbi_uid=6391754&amp;link_uid=23899600">Citation</a><a class="dblinks" href="/pmc/articles/PMC4103166/" ref="ordinalpos=1913&amp;ncbi_uid=6391754&amp;link_uid=PMC4103166">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1914.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3445042/figure/F7/" ref="ordinalpos=1914&amp;ncbi_uid=5280733&amp;link_uid=PMC3445042" href="/pmc/articles/PMC3445042/"><img src="/pmc/articles/instance/3445042/bin/nihms402300f7.gif" src-large="/pmc/articles/instance/3445042/bin/nihms402300f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3445042/" ref="ordinalpos=1914&amp;ncbi_uid=5280733&amp;link_uid=PMC3445042" image-link="/pmc/articles/PMC3445042/figure/F7/" class="imagepopup">Figure 7.  From: Wnt <span class="highlight" style="background-color:">signaling</span> regulates post-embryonic hypothalamic progenitor differentiation. </a></p><div class="supp"><p class="details"><div>Model for the role of Wnt <span class="highlight" style="background-color:">signaling</span> in hypothalamic progenitor differentiation. In zebrafish (above), Wnt <span class="highlight" style="background-color:">signaling</span> is active in unspecified embryonic progenitors and Sox3<sup>+</sup> post-embryonic neural progenitors, but is lost as these cells undergo neurogenesis. Wnt <span class="highlight" style="background-color:">signaling</span> promotes mitotic activity in progenitors, and is also required for their ability to undergo neuronal differentiation, which underlies the transition from an âuncommittedâ to a âcommittedâ progenitor state. Finally, Wnt <span class="highlight" style="background-color:">signaling</span> must be inhibited for differentiation to proceed. In contrast the formation of radial glia is inhibited by Wnt activity. In the adult mouse (below), Wnt <span class="highlight" style="background-color:">signaling</span> is active in ventricular/subventricular zone cells (VZ/SVZ) that express neural progenitor markers, and in parenchymal zone cells that express neuronal and neuronal precursor markers. Ventricular <i>Hes1<sup>+</sup></i> progenitors do not require Wnt activity to make radial glial tanycytes, and ectopic <span class="highlight" style="background-color:">pathway</span> activity inhibits radial glial formation at the expense of other fates.</div></p></div><div class="aux"><div class="resc">Xu Wang, et al. Dev Cell. 2012 September 11;23(3):624-636.</div><p class="links"><a class="dblinks" href="/pubmed/22975330" ref="ordinalpos=1914&amp;ncbi_uid=5280733&amp;link_uid=22975330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3445042/" ref="ordinalpos=1914&amp;ncbi_uid=5280733&amp;link_uid=PMC3445042">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1915.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4255183/figure/f4/" ref="ordinalpos=1915&amp;ncbi_uid=6942847&amp;link_uid=PMC4255183" href="/pmc/articles/PMC4255183/"><img src="/pmc/articles/instance/4255183/bin/srep07310-f4.gif" src-large="/pmc/articles/instance/4255183/bin/srep07310-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4255183/" ref="ordinalpos=1915&amp;ncbi_uid=6942847&amp;link_uid=PMC4255183" image-link="/pmc/articles/PMC4255183/figure/f4/" class="imagepopup">Figure 4. Identification of multiple dysregulated <span class="highlight" style="background-color:">signaling</span> transduction pathways by GEM and FCM..  From: Rigosertib as a selective anti-tumor agent can ameliorate multiple dysregulated <span class="highlight" style="background-color:">signaling</span> transduction pathways in high-grade myelodysplastic syndrome. </a></p><div class="supp"><p class="details"><div>The Akt-PI3K (A), MAPK (B), Jak-STAT (C), Wnt (D), Adipocytokine (E), Notch (F) and P53 (G) <span class="highlight" style="background-color:">signaling</span> pathways were identified with hierarchical clustering as potentially significant pathways that may be dysregulated in MDS (n = 12) compared with normal controls (NC; n = 6); (H) Further <span class="highlight" style="background-color:">pathway</span> analysis showed that the Akt-PI3K, MAPK, Jak-STAT, Wnt and Notch <span class="highlight" style="background-color:">signaling</span> pathways were considered remarkably significant; (I) <span class="highlight" style="background-color:">pathway</span>-net analysis revealed that the MAPK, Jak-STAT and apoptosis <span class="highlight" style="background-color:">signaling</span> pathways were described as key downstream pathways; (J) and (K) Akt1 and Akt phosphorylation in Akt-PI3K <span class="highlight" style="background-color:">pathway</span> were evidently elevated in MDS (n = 35) patients compared to the normal controls (n = 15). High-grade MDS (n = 18) exhibited higher expression levels of Akt1 and Akt phosphorylation; (L) the p38 phosphorylation in the p38 MAPK <span class="highlight" style="background-color:">pathway</span> was higher in MDS patients, especially in low-grade MDS, than in normal controls; (M) and (N) no significant difference in the phosphorylated ERK1/2 (ERK MAPK <span class="highlight" style="background-color:">pathway</span>) and SAPK/JNK (JNK MAPK <span class="highlight" style="background-color:">pathway</span>) levels was observed between the MDS and normal group; (O) no difference in phosphorylated STAT1 was observed in the Jak-STAT <span class="highlight" style="background-color:">pathway</span> between the MDS and normal group; (P) MDS patients showed increased STAT3 phosphorylation in the Jak-STAT <span class="highlight" style="background-color:">pathway</span> in the MDS group, especially high-grade patients. (Q) and (R) the levels of phosphorylated GSK-3Î±/Î² (Ser9) and Î²-catenin (Ser45) were significantly increased in MDS patients compared to the normal controls. In addition, the P53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was also investigated due to its critical role in regulating cell apoptosis in MDS; (S) P53 phosphorylation was significantly increased in low-grade MDS compared to normal controls and high-grade MDS; (T) the level of PLK1 phosphorylation did not differ between MDS and normal controls. Error bars throughout represent the SEM. The comparison analysis among normal controls, low-grade and high-grade MDS groups were performed using ANOVA test.</div></p></div><div class="aux"><div class="resc">Feng Xu, et al. Sci Rep. 2014;4:7310.</div><p class="links"><a class="dblinks" href="/pubmed/25472472" ref="ordinalpos=1915&amp;ncbi_uid=6942847&amp;link_uid=25472472">Citation</a><a class="dblinks" href="/pmc/articles/PMC4255183/" ref="ordinalpos=1915&amp;ncbi_uid=6942847&amp;link_uid=PMC4255183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1916.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2627788/figure/F5/" ref="ordinalpos=1916&amp;ncbi_uid=1629864&amp;link_uid=PMC2627788" href="/pmc/articles/PMC2627788/"><img src="/pmc/articles/instance/2627788/bin/nihms-79758-f0005.gif" src-large="/pmc/articles/instance/2627788/bin/nihms-79758-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2627788/" ref="ordinalpos=1916&amp;ncbi_uid=1629864&amp;link_uid=PMC2627788" image-link="/pmc/articles/PMC2627788/figure/F5/" class="imagepopup">Figure 5.  From: Targeting the leukemic stem cell: the Holy Grail of leukemia therapy. </a></p><div class="supp"><p class="details"><div>Overview of Wnt/Î²-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in leukemia. The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is a central <span class="highlight" style="background-color:">pathway</span>, which results in the transmission of growth-promoting signals that control the activity of Î²-catenin. Stabilization of Î²-catenin results in its translocation to the nucleus and promotion of gene transcription by TCF/LEF transcription complex. This can result in the transcription of genes such as Cyclin D, c-Myc, SALL-4, PPARÎ´ and other genes. In the absence of Wnt binding of Frizzled, the multiprotein destruction complex (MDC) is activated and glycogen synthase kinase 3Î² (GSK-3Î²) phosphorylates Î²-catenin and it is targeted for proteasomal degradation. In the presence of Wnt binding, this complex is inactivated. Interactions with other <span class="highlight" style="background-color:">signaling</span> pathways such as PI3K/PTEN/Akt exist, which influence GSK-3Î² activity (inactivation). This <span class="highlight" style="background-color:">pathway</span> can be activated by Flt-3 and other receptors. Thus, there are interactions between the Wnt and PI3K/PTEN/Akt pathways. Wnt is also postulated to interact with the Notch and other <span class="highlight" style="background-color:">signaling</span> pathways. The Wnt <span class="highlight" style="background-color:">pathway</span> is frequently dysregulated in leukemia. This can occur by mutations at Flt-3, chromosomal translocations, methylation of the promoter regions inhibitory genes such as WIF-1 and many other mechanisms (e.g., mutations at either PI3K or PTEN, which result in the activation of Akt expression).</div></p></div><div class="aux"><div class="resc">N Misaghian, et al. Leukemia. ;23(1):25-42.</div><p class="links"><a class="dblinks" href="/pubmed/18800146" ref="ordinalpos=1916&amp;ncbi_uid=1629864&amp;link_uid=18800146">Citation</a><a class="dblinks" href="/pmc/articles/PMC2627788/" ref="ordinalpos=1916&amp;ncbi_uid=1629864&amp;link_uid=PMC2627788">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1917.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2847468/figure/F6/" ref="ordinalpos=1917&amp;ncbi_uid=2891797&amp;link_uid=PMC2847468" href="/pmc/articles/PMC2847468/"><img src="/pmc/articles/instance/2847468/bin/DEV045971F6.gif" src-large="/pmc/articles/instance/2847468/bin/DEV045971F6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2847468/" ref="ordinalpos=1917&amp;ncbi_uid=2891797&amp;link_uid=PMC2847468" image-link="/pmc/articles/PMC2847468/figure/F6/" class="imagepopup">Fig. 6. From: Prostaglandin G?? <span class="highlight" style="background-color:">signaling</span> stimulates gastrulation movements by limiting cell adhesion through Snai1a stabilization. </a></p><div class="supp"><p class="details"><div><b>Model of the mechanism by which zebrafish PGE<sub>2</sub> negatively regulates E-cadherin via a Snai1a-dependent mechanism.</b> Following EP4 receptor engagement by PGE<sub>2</sub>, the GÎ²Î³ subunits bind Gsk3Î², preventing it from inhibiting Snai1a by phosphorylation. Ubiquitylation of Snail is decreased, stabilizing Snai1a protein. Snai1a can then transcriptionally repress <i>E-cadherin</i>, altering the cell adhesion of <span class="highlight" style="background-color:">signaling</span> cells. Blue arrows, <span class="highlight" style="background-color:">pathway</span> mechanism following PGE<sub>2</sub> <span class="highlight" style="background-color:">signaling</span> activation.</div></p></div><div class="aux"><div class="resc">Christina K. Speirs, et al. Development. 2010 April 15;137(8):1327-1337.</div><p class="links"><a class="dblinks" href="/pubmed/20332150" ref="ordinalpos=1917&amp;ncbi_uid=2891797&amp;link_uid=20332150">Citation</a><a class="dblinks" href="/pmc/articles/PMC2847468/" ref="ordinalpos=1917&amp;ncbi_uid=2891797&amp;link_uid=PMC2847468">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1918.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3798879/figure/toxins-05-01664-f002/" ref="ordinalpos=1918&amp;ncbi_uid=5402446&amp;link_uid=PMC3798879" href="/pmc/articles/PMC3798879/"><img src="/pmc/articles/instance/3798879/bin/toxins-05-01664-g002.gif" src-large="/pmc/articles/instance/3798879/bin/toxins-05-01664-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3798879/" ref="ordinalpos=1918&amp;ncbi_uid=5402446&amp;link_uid=PMC3798879" image-link="/pmc/articles/PMC3798879/figure/toxins-05-01664-f002/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Signaling</span> Cascades of Pasteurella multocida Toxin in Immune Evasion. </a></p><div class="supp"><p class="details"><div>Pim-dependent inactivation of the negative regulator SOCS. (<b>a</b>) In v-Abl as well as in (<b>b</b>) PMT-mediated <span class="highlight" style="background-color:">signaling</span>, activation of the JAK-STAT <span class="highlight" style="background-color:">pathway</span> leads to the constitutive expression of the otherwise short-lived serine/threonine kinase Pim. Phosphorylation of its downstream target, the suppressor of cytokine <span class="highlight" style="background-color:">signaling</span> (SOCS) inhibits the ability of SOCS to act as an ubiquitin ligase. Thus JAK molecules accumulate in the cell leading to the hyperactivation of STAT3 <span class="highlight" style="background-color:">signaling</span>, increased cell survival and anchorage-independent growth.</div></p></div><div class="aux"><div class="resc">Katharina F. Kubatzky, et al. Toxins (Basel). 2013 September;5(9):1664-1681.</div><p class="links"><a class="dblinks" href="/pubmed/24064721" ref="ordinalpos=1918&amp;ncbi_uid=5402446&amp;link_uid=24064721">Citation</a><a class="dblinks" href="/pmc/articles/PMC3798879/" ref="ordinalpos=1918&amp;ncbi_uid=5402446&amp;link_uid=PMC3798879">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1919.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3774928/figure/F3/" ref="ordinalpos=1919&amp;ncbi_uid=5321085&amp;link_uid=PMC3774928" href="/pmc/articles/PMC3774928/"><img src="/pmc/articles/instance/3774928/bin/jcbn13-28f03.gif" src-large="/pmc/articles/instance/3774928/bin/jcbn13-28f03.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3774928/" ref="ordinalpos=1919&amp;ncbi_uid=5321085&amp;link_uid=PMC3774928" image-link="/pmc/articles/PMC3774928/figure/F3/" class="imagepopup">Fig. 3.  From: Mithramycin A induces apoptosis by regulating the mTOR/Mcl-1/tBid <span class="highlight" style="background-color:">pathway</span> in androgen-independent prostate cancer cells. </a></p><div class="supp"><p class="details"><div>Mithramycin A regulates the mTOR/Mcl-1/truncated Bid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Levels of phospho-mTOR (p-mTOR), total mTOR, Mcl-1 and truncated Bid (tBid) extracted from athymic nude mice xenografts bearing DU145 cells were determined by Western blot analysis. (B) Working model by which the treatment of Mithramycin A (Mith) induces apoptosis in human prostate cancer through the mTOR/Mcl-1/tBid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Eun-Sun Choi, et al. J Clin Biochem Nutr. 2013 September;53(2):89-93.</div><p class="links"><a class="dblinks" href="/pubmed/24062605" ref="ordinalpos=1919&amp;ncbi_uid=5321085&amp;link_uid=24062605">Citation</a><a class="dblinks" href="/pmc/articles/PMC3774928/" ref="ordinalpos=1919&amp;ncbi_uid=5321085&amp;link_uid=PMC3774928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1920.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g012/" ref="ordinalpos=1920&amp;ncbi_uid=2242906&amp;link_uid=PMC2854141" href="/pmc/articles/PMC2854141/"><img src="/pmc/articles/instance/2854141/bin/pone.0010153.g012.gif" src-large="/pmc/articles/instance/2854141/bin/pone.0010153.g012.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2854141/" ref="ordinalpos=1920&amp;ncbi_uid=2242906&amp;link_uid=PMC2854141" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g012/" class="imagepopup">Figure 12. WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Uncovering Molecular Biomarkers That Correlate Cognitive Decline with the Changes of Hippocampus' Gene Expression Profiles in Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>The upper graph presents the stacked normalized expression values of all the probes involved in the <span class="highlight" style="background-color:">pathway</span> with an upregulation trend. The lower graph analyses the genes involved in the <span class="highlight" style="background-color:">pathway</span> with a downregulation tendency. In the supplementary material (File S3 sheet âWnt Signallingâ), the reader will find all the individual gene expression values, normalised and not normalised.</div></p></div><div class="aux"><div class="resc">MartÃ­n GÃ³mez Ravetti, et al. PLoS One. 2010;5(4):e10153.</div><p class="links"><a class="dblinks" href="/pubmed/20405009" ref="ordinalpos=1920&amp;ncbi_uid=2242906&amp;link_uid=20405009">Citation</a><a class="dblinks" href="/pmc/articles/PMC2854141/" ref="ordinalpos=1920&amp;ncbi_uid=2242906&amp;link_uid=PMC2854141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1921.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193402/figure/fig2/" ref="ordinalpos=1921&amp;ncbi_uid=3770379&amp;link_uid=PMC3193402" href="/pmc/articles/PMC3193402/"><img src="/pmc/articles/instance/3193402/bin/cr2010159f2.gif" src-large="/pmc/articles/instance/3193402/bin/cr2010159f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193402/" ref="ordinalpos=1921&amp;ncbi_uid=3770379&amp;link_uid=PMC3193402" image-link="/pmc/articles/PMC3193402/figure/fig2/" class="imagepopup">Figure 2.  From: Role of NF-?B in the skeleton. </a></p><div class="supp"><p class="details"><div>Crosstalk between RANKL- and TNF-induced NF-ÎºB <span class="highlight" style="background-color:">signaling</span> in OCs. RANKL activates the alternative NF-ÎºB <span class="highlight" style="background-color:">pathway</span> by blocking TRAF3-mediated NIK degradation, leading to nuclear translocation of RelB/p52 dimers. Transcriptional targets of this <span class="highlight" style="background-color:">pathway</span> provide signals for OC differentiation and function. RANKL also activates the classical <span class="highlight" style="background-color:">pathway</span> downstream of the IKK complex, supporting OC lineage cell survival downstream of p65/p50. TNF also activates the classical NF-ÎºB <span class="highlight" style="background-color:">pathway</span>, which blocks apoptosis, upregulates p100/RelB expression, and may support OC differentiation. TNF also stabilizes TRAF3, which can inhibit NIK.</div></p></div><div class="aux"><div class="resc">Deborah Veis Novack. Cell Res. 2011 January;21(1):169-182.</div><p class="links"><a class="dblinks" href="/pubmed/21079651" ref="ordinalpos=1921&amp;ncbi_uid=3770379&amp;link_uid=21079651">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193402/" ref="ordinalpos=1921&amp;ncbi_uid=3770379&amp;link_uid=PMC3193402">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1922.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC136456/figure/f7/" ref="ordinalpos=1922&amp;ncbi_uid=316514&amp;link_uid=PMC136456" href="/pmc/articles/PMC136456/"><img src="/pmc/articles/instance/136456/bin/jv1820687007.gif" src-large="/pmc/articles/instance/136456/bin/jv1820687007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC136456/" ref="ordinalpos=1922&amp;ncbi_uid=316514&amp;link_uid=PMC136456" image-link="/pmc/articles/PMC136456/figure/f7/" class="imagepopup">FIG. 7. From: Subversion of Cell <span class="highlight" style="background-color:">Signaling</span> Pathways by Hepatitis C Virus Nonstructural 5A Protein via Interaction with Grb2 and P85 Phosphatidylinositol 3-Kinase. </a></p><div class="supp"><p class="details"><div>Model of the modulation of cellular <span class="highlight" style="background-color:">signaling</span> pathways by NS5A. NS5A, through its C-terminal proline-rich SH3-binding motif, directly interacts with Grb2 to perturb the downstream MAPK <span class="highlight" style="background-color:">pathway</span>, which may contribute to regulation of translation (25) and perturbation of IFN <span class="highlight" style="background-color:">signaling</span> (24). On the other hand, the N-terminal region of NS5A directly interacts with the p85 subunit of PI3K, which enhances the PI3K-AKT <span class="highlight" style="background-color:">pathway</span> and regulates the downstream apoptosis machinery. The modulation of the PI3K-AKT <span class="highlight" style="background-color:">pathway</span> by NS5A may contribute to cell survival and inhibition of apoptosis in virus-infected cells, resulting in viral persistence and contributing to pathogenesis.</div></p></div><div class="aux"><div class="resc">Yupeng He, et al. J Virol. 2002 September;76(18):9207-9217.</div><p class="links"><a class="dblinks" href="/pubmed/12186904" ref="ordinalpos=1922&amp;ncbi_uid=316514&amp;link_uid=12186904">Citation</a><a class="dblinks" href="/pmc/articles/PMC136456/" ref="ordinalpos=1922&amp;ncbi_uid=316514&amp;link_uid=PMC136456">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1923.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3390638/figure/F2/" ref="ordinalpos=1923&amp;ncbi_uid=5063812&amp;link_uid=PMC3390638" href="/pmc/articles/PMC3390638/"><img src="/pmc/articles/instance/3390638/bin/zbc0311215790002.gif" src-large="/pmc/articles/instance/3390638/bin/zbc0311215790002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3390638/" ref="ordinalpos=1923&amp;ncbi_uid=5063812&amp;link_uid=PMC3390638" image-link="/pmc/articles/PMC3390638/figure/F2/" class="imagepopup">FIGURE 2. From: ATP Depletion Triggers Acute Myeloid Leukemia Differentiation through an ATR/Chk1 Protein-dependent and p53 Protein-independent <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>6BT induces DNA damage <span class="highlight" style="background-color:">signaling</span>.</b> Western analysis was performed for DNA damage <span class="highlight" style="background-color:">pathway</span> proteins in OCI cells treated with 6BT for the indicated time points. <i>UT</i>, untreated.</div></p></div><div class="aux"><div class="resc">Amitabha Chakrabarti, et al. J Biol Chem. 2012 July 6;287(28):23635-23643.</div><p class="links"><a class="dblinks" href="/pubmed/22621920" ref="ordinalpos=1923&amp;ncbi_uid=5063812&amp;link_uid=22621920">Citation</a><a class="dblinks" href="/pmc/articles/PMC3390638/" ref="ordinalpos=1923&amp;ncbi_uid=5063812&amp;link_uid=PMC3390638">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1924.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3973216/figure/fig4/" ref="ordinalpos=1924&amp;ncbi_uid=5967933&amp;link_uid=PMC3973216" href="/pmc/articles/PMC3973216/"><img src="/pmc/articles/instance/3973216/bin/cddis201498f4.gif" src-large="/pmc/articles/instance/3973216/bin/cddis201498f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3973216/" ref="ordinalpos=1924&amp;ncbi_uid=5967933&amp;link_uid=PMC3973216" image-link="/pmc/articles/PMC3973216/figure/fig4/" class="imagepopup">Figure 4.  From: Overcoming inherent resistance to histone deacetylase inhibitors in multiple myeloma cells by targeting pathways integral to the actin cytoskeleton. </a></p><div class="supp"><p class="details"><div>Simplified representation of the regulation of actin cytoskeleton <span class="highlight" style="background-color:">pathway</span> and differentially regulated genes. Regulation of actin cytoskeleton encompasses <span class="highlight" style="background-color:">signaling</span> to the cytoskeleton through GPCRs, ITGs and growth factor (GF) â RTKs, leading to diverse cell functions including changes in cell motility, proliferation and survival.<sup>24, 25, 26, 27</sup> GPCR <span class="highlight" style="background-color:">signaling</span> is activated by a number of external stimuli and signals through heterotrimeric G-proteins, which in turn stimulates a variety of downstream <span class="highlight" style="background-color:">signaling</span> pathways including MAPK, PI3K, FAK and Rho family of GTPases (Rho, Rac and cell division cycle 42 â CDC42) and downstream protein kinase effectors including p21-activated kinase (PAK).<sup>26, 27</sup> Within the 35-gene signature, F2R and OPN3, both GPCR proteins, were dysregulated. Regulators of G-protein <span class="highlight" style="background-color:">signaling</span> (RGS) are multifunctional <span class="highlight" style="background-color:">signaling</span> proteins that directly bind to activated G-proteins and are integral for modulation of the GPCR <span class="highlight" style="background-color:">signaling</span> process. RGS12, a RGS protein, is upregulated in the resistant cell lines. RTKs are high-affinity cell surface receptors for GF and are key regulators of normal cellular processes and progression of many types of cancers. Predominant <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that mediates signals from RTK-GF is the MAPK <span class="highlight" style="background-color:">pathway</span>. FAK and PI3K pathways are also known to have important roles in signal transduction. FGF9, a GF, is upregulated in the resistant cell lines. ITGs are a family of cell-surface-adhesion receptors that transmit signals to the cell to determine migration, survival, differentiation and motility in context with the GPCR and RTK-GF <span class="highlight" style="background-color:">signaling</span>. One of the first integrin <span class="highlight" style="background-color:">signaling</span> molecules to be activated is FAK, which in turn activates other <span class="highlight" style="background-color:">signaling</span> pathways. <span class="highlight" style="background-color:">Signaling</span> initiated by GPCR, RTK-GF or ITGs results in remodeling of the actin cytoskeleton and KIF4A, upregulated in resistant cell lines, is a known actin cytoskeleton protein. Genes upregulated in the resistant cell lines are shown in red, whereas downregulated genes are shown in blue</div></p></div><div class="aux"><div class="resc">S Mithraprabhu, et al. Cell Death Dis. 2014 March;5(3):e1134.</div><p class="links"><a class="dblinks" href="/pubmed/24651437" ref="ordinalpos=1924&amp;ncbi_uid=5967933&amp;link_uid=24651437">Citation</a><a class="dblinks" href="/pmc/articles/PMC3973216/" ref="ordinalpos=1924&amp;ncbi_uid=5967933&amp;link_uid=PMC3973216">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1925.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g013/" ref="ordinalpos=1925&amp;ncbi_uid=2242903&amp;link_uid=PMC2854141" href="/pmc/articles/PMC2854141/"><img src="/pmc/articles/instance/2854141/bin/pone.0010153.g013.gif" src-large="/pmc/articles/instance/2854141/bin/pone.0010153.g013.jpg" alt="Figure 13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2854141/" ref="ordinalpos=1925&amp;ncbi_uid=2242903&amp;link_uid=PMC2854141" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g013/" class="imagepopup">Figure 13. Phosphatidylinositol <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Uncovering Molecular Biomarkers That Correlate Cognitive Decline with the Changes of Hippocampus' Gene Expression Profiles in Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>The upper graph presents the stacked normalized expression values of all the probes involved in the <span class="highlight" style="background-color:">pathway</span> with an upregulation trend. The lower graph analyses the genes involved in the <span class="highlight" style="background-color:">pathway</span> with a downregulation tendency. In the supplementary material (File S3 sheet âPhosphatidylinositol signallingâ), the reader will find all the individual gene expression values, normalised and not normalised.</div></p></div><div class="aux"><div class="resc">MartÃ­n GÃ³mez Ravetti, et al. PLoS One. 2010;5(4):e10153.</div><p class="links"><a class="dblinks" href="/pubmed/20405009" ref="ordinalpos=1925&amp;ncbi_uid=2242903&amp;link_uid=20405009">Citation</a><a class="dblinks" href="/pmc/articles/PMC2854141/" ref="ordinalpos=1925&amp;ncbi_uid=2242903&amp;link_uid=PMC2854141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1926.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4050444/figure/F5/" ref="ordinalpos=1926&amp;ncbi_uid=6235475&amp;link_uid=PMC4050444" href="/pmc/articles/PMC4050444/"><img src="/pmc/articles/instance/4050444/bin/2045-3701-4-24-5.gif" src-large="/pmc/articles/instance/4050444/bin/2045-3701-4-24-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4050444/" ref="ordinalpos=1926&amp;ncbi_uid=6235475&amp;link_uid=PMC4050444" image-link="/pmc/articles/PMC4050444/figure/F5/" class="imagepopup">Figure 5.  From: Adipose-derived stem cells cooperate with fractional carbon dioxide laser in antagonizing photoaging: a potential role of Wnt and ?-catenin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Differential expression of wnt/Î²-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein and TGF- Î²2 during photoaging treatment.</b> Expression of wnt3a and Î²-catenin were upregulated by ADSC transplantation and CO<sub>2</sub> fractional laser treatment.Expression of TGF-Î²2 was upregulated during photoaging treatment.</div></p></div><div class="aux"><div class="resc">Xiao Xu, et al. Cell Biosci. 2014;4:24-24.</div><p class="links"><a class="dblinks" href="/pubmed/24917925" ref="ordinalpos=1926&amp;ncbi_uid=6235475&amp;link_uid=24917925">Citation</a><a class="dblinks" href="/pmc/articles/PMC4050444/" ref="ordinalpos=1926&amp;ncbi_uid=6235475&amp;link_uid=PMC4050444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1927.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3340364/figure/pone-0036248-g006/" ref="ordinalpos=1927&amp;ncbi_uid=3979248&amp;link_uid=PMC3340364" href="/pmc/articles/PMC3340364/"><img src="/pmc/articles/instance/3340364/bin/pone.0036248.g006.gif" src-large="/pmc/articles/instance/3340364/bin/pone.0036248.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3340364/" ref="ordinalpos=1927&amp;ncbi_uid=3979248&amp;link_uid=PMC3340364" image-link="/pmc/articles/PMC3340364/figure/pone-0036248-g006/" class="imagepopup">Figure 6. Pathways targeted by deregulated miRNAs in glioblastoma stem cells..  From: Genome-Wide Profiling Identified a Set of miRNAs that Are Differentially Expressed in Glioblastoma Stem Cells and Normal Neural Stem Cells. </a></p><div class="supp"><p class="details"><div>Common miRNA targets were subjected to DAVID functional annotation with KEGG <span class="highlight" style="background-color:">pathway</span> analysis. <b>A.</b> The up-regulated miRNAs in glioblastoma stem cells were predicted to target the p53 <span class="highlight" style="background-color:">pathway</span>. The p53-centered <span class="highlight" style="background-color:">pathway</span> has been shown to regulate cell cycle, apoptosis, angiogenesis, metastasis, and genome stability. <b>B.</b> The down-regulated miRNAs were predicted to target components of the IGF <span class="highlight" style="background-color:">pathway</span>. Various components of the IGF <span class="highlight" style="background-color:">signaling</span> pathways were targeted by down-regulated miRNAs. The IGF <span class="highlight" style="background-color:">pathway</span> has been shown to enhance cell growth, survival, and migration.</div></p></div><div class="aux"><div class="resc">Ming-Fei Lang, et al. PLoS One. 2012;7(4):e36248.</div><p class="links"><a class="dblinks" href="/pubmed/22558405" ref="ordinalpos=1927&amp;ncbi_uid=3979248&amp;link_uid=22558405">Citation</a><a class="dblinks" href="/pmc/articles/PMC3340364/" ref="ordinalpos=1927&amp;ncbi_uid=3979248&amp;link_uid=PMC3340364">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1928.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3874981/figure/fig5/" ref="ordinalpos=1928&amp;ncbi_uid=5699708&amp;link_uid=PMC3874981" href="/pmc/articles/PMC3874981/"><img src="/pmc/articles/instance/3874981/bin/SCIENTIFICA2013-684736.005.gif" src-large="/pmc/articles/instance/3874981/bin/SCIENTIFICA2013-684736.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3874981/" ref="ordinalpos=1928&amp;ncbi_uid=5699708&amp;link_uid=PMC3874981" image-link="/pmc/articles/PMC3874981/figure/fig5/" class="imagepopup">Figure 5.  From: Review of <span class="highlight" style="background-color:">Signaling</span> Pathways Governing MSC Osteogenic and Adipogenic Differentiation. </a></p><div class="supp"><p class="details"><div>Schematic of NELL-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. NELL-1 is a secreted osteoinductive protein that binds to the cell surface receptor Integrin<i>Î²</i>1. Binding to Integrin<i>Î±</i>3 has also been reported. Multiple intracellular <span class="highlight" style="background-color:">signaling</span> pathways have been shown to increase after NELL-1 stimulation, including MAPK, Hedgehog, and <i>Î²</i>-catenin dependent Wnt <span class="highlight" style="background-color:">signaling</span>. Although the relative importance of these pathways is still undefined, NELL-1 treatment results in increased <i>Runx2</i> transcription, Runx2 phosphorylation, and induction of osteogenic programming.</div></p></div><div class="aux"><div class="resc">Aaron W. James. Scientifica (Cairo). 2013;2013:684736.</div><p class="links"><a class="dblinks" href="/pubmed/24416618" ref="ordinalpos=1928&amp;ncbi_uid=5699708&amp;link_uid=24416618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3874981/" ref="ordinalpos=1928&amp;ncbi_uid=5699708&amp;link_uid=PMC3874981">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1929.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2630181/figure/F4/" ref="ordinalpos=1929&amp;ncbi_uid=1657727&amp;link_uid=PMC2630181" href="/pmc/articles/PMC2630181/"><img src="/pmc/articles/instance/2630181/bin/nihms-82845-f0004.gif" src-large="/pmc/articles/instance/2630181/bin/nihms-82845-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2630181/" ref="ordinalpos=1929&amp;ncbi_uid=1657727&amp;link_uid=PMC2630181" image-link="/pmc/articles/PMC2630181/figure/F4/" class="imagepopup">Figure 4.  From: Acute doxorubicin cardiotoxicity is associated with p53-induced inhibition of the mTOR <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Western blot analysis of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hearts from NON-TXG and MHC-CB7 mice treated with saline or DOX (see Supplemental Data for densitometric signal quantitation).</div></p></div><div class="aux"><div class="resc">Wuqiang Zhu, et al. Circulation. ;119(1):99-106.</div><p class="links"><a class="dblinks" href="/pubmed/19103993" ref="ordinalpos=1929&amp;ncbi_uid=1657727&amp;link_uid=19103993">Citation</a><a class="dblinks" href="/pmc/articles/PMC2630181/" ref="ordinalpos=1929&amp;ncbi_uid=1657727&amp;link_uid=PMC2630181">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1930.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2647628/figure/fig1/" ref="ordinalpos=1930&amp;ncbi_uid=1700396&amp;link_uid=PMC2647628" href="/pmc/articles/PMC2647628/"><img src="/pmc/articles/instance/2647628/bin/PROCATS43263fig1.gif" src-large="/pmc/articles/instance/2647628/bin/PROCATS43263fig1.jpg" alt="&lt;b&gt;Figure 1.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2647628/" ref="ordinalpos=1930&amp;ncbi_uid=1700396&amp;link_uid=PMC2647628" image-link="/pmc/articles/PMC2647628/figure/fig1/" class="imagepopup">Figure 1..  From: CATERPILLER (NLR) Family Members as Positive and Negative Regulators of Inflammatory Responses. </a></p><div class="supp"><p class="details"><div>Scheme depicting how Monarch-1 suppresses inflammatory <span class="highlight" style="background-color:">signaling</span>. Toll-like receptors (TLRs) and tumor necrosis factor family receptors (TNFRs) play a critical in the inflammatory response. TLRs recruit the adaptor molecules, myeloid differentiation factor 88 (MyD88) and Tollip, which, in turn, recruit the kinase, IL-1 receptorâassociated kinase (IRAK-1), to induce the activation of canonical nuclear factor (NF)âÎºB. IRAK-1 becomes hyperphosphorylated, facilitating its dissociation from the receptor complex to mediate additional <span class="highlight" style="background-color:">signaling</span> pathways. One of the genes induced by the canonical <span class="highlight" style="background-color:">pathway</span> is NF-ÎºB2/p100, which functions in the noncanonical <span class="highlight" style="background-color:">pathway</span>. The noncanonical <span class="highlight" style="background-color:">pathway</span> is triggered after the activation of NF-ÎºBâinducing kinase (NIK). This results in processing of p100 to its active form, p52. This alternative <span class="highlight" style="background-color:">pathway</span> induces a distinct set of genes and also regulates the expression of genes induced by the canonical <span class="highlight" style="background-color:">pathway</span>. Monarch-1 does not affect initial activation of the canonical <span class="highlight" style="background-color:">pathway</span>. However, Monarch-1 suppresses the noncanonical <span class="highlight" style="background-color:">pathway</span> by associating with NIK and inducing its degradation via the proteasome. Monarch-1 also associates with IRAK-1 and suppresses the accumulation of hyperphosphorylated forms of the kinase; however, it is unclear whether or not this also involves proteasome degradation.</div></p></div><div class="aux"><div class="resc">John D. Lich, et al. Proc Am Thorac Soc. 2007 July;4(3):263-266.</div><p class="links"><a class="dblinks" href="/pubmed/17607010" ref="ordinalpos=1930&amp;ncbi_uid=1700396&amp;link_uid=17607010">Citation</a><a class="dblinks" href="/pmc/articles/PMC2647628/" ref="ordinalpos=1930&amp;ncbi_uid=1700396&amp;link_uid=PMC2647628">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1931.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3041271/figure/F4/" ref="ordinalpos=1931&amp;ncbi_uid=2776509&amp;link_uid=PMC3041271" href="/pmc/articles/PMC3041271/"><img src="/pmc/articles/instance/3041271/bin/nihms270504f4.gif" src-large="/pmc/articles/instance/3041271/bin/nihms270504f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3041271/" ref="ordinalpos=1931&amp;ncbi_uid=2776509&amp;link_uid=PMC3041271" image-link="/pmc/articles/PMC3041271/figure/F4/" class="imagepopup">Fig. 4.  From: Cell <span class="highlight" style="background-color:">Signaling</span> in Space and Time: Where Proteins Come Together and When They're Apart. </a></p><div class="supp"><p class="details"><div>Temporal regulation of <span class="highlight" style="background-color:">signaling</span> events. (<b>A</b>) Schematic diagram of the NF-ÎºB <span class="highlight" style="background-color:">pathway</span>; individual protein components are indicated. (<b>B</b>) The concerted actions of oxygen-sensing enzymes and ubiquitin E3 ligases control the stability of the hypoxia-inducible factor protein HIF-1Î±. Schematic diagram of the HIF-1Î± <span class="highlight" style="background-color:">pathway</span> under normoxic (left) and hypoxic (right) conditions; individual protein components are indicated.</div></p></div><div class="aux"><div class="resc">John D. Scott, et al. Science. ;326(5957):1220-1224.</div><p class="links"><a class="dblinks" href="/pubmed/19965465" ref="ordinalpos=1931&amp;ncbi_uid=2776509&amp;link_uid=19965465">Citation</a><a class="dblinks" href="/pmc/articles/PMC3041271/" ref="ordinalpos=1931&amp;ncbi_uid=2776509&amp;link_uid=PMC3041271">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1932.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3102077/figure/pone-0020085-g003/" ref="ordinalpos=1932&amp;ncbi_uid=3009653&amp;link_uid=PMC3102077" href="/pmc/articles/PMC3102077/"><img src="/pmc/articles/instance/3102077/bin/pone.0020085.g003.gif" src-large="/pmc/articles/instance/3102077/bin/pone.0020085.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3102077/" ref="ordinalpos=1932&amp;ncbi_uid=3009653&amp;link_uid=PMC3102077" image-link="/pmc/articles/PMC3102077/figure/pone-0020085-g003/" class="imagepopup">Figure 3. OrthoList coverage of conserved <span class="highlight" style="background-color:">signaling</span> pathways..  From: OrthoList: A Compendium of C. elegans Genes with Human Orthologs. </a></p><div class="supp"><p class="details"><div>Genes in bold are found by at least one orthology-predicting program, and thus included in OrthoList. The source data for this figure can be found in Table S5. A) RTK/Ras/MAPK <span class="highlight" style="background-color:">pathway</span> (reviewed in [34]). Note that <i>ras-1</i> and <i>ras-2</i> have not been defined functionally, although they are highly conserved. B) Notch <span class="highlight" style="background-color:">pathway</span> (reviewed in reviewed in [32]). C) TGF-Ã <span class="highlight" style="background-color:">pathway</span> (reviewed in [33]). We note that <i>tag-68</i>, was only defined by conservation and no phenotype has been associated with its loss. D) Wnt <span class="highlight" style="background-color:">pathway</span> (reviewed in [30]). Note that our analysis was restricted to the conserved, canonical Wnt <span class="highlight" style="background-color:">pathway</span>. E) Insulin <span class="highlight" style="background-color:">pathway</span>. We specifically highlight the six insulins (<i>daf-28</i>, <i>ins-1</i>, <i>ins-4</i>, <i>ins-6</i>, <i>ins-</i>7 and <i>ins-</i>8), out of forty, that have been found (by overexpression, biochemical methods, RNAi or by existence of a semi-dominant allele) to be functional (reviewed in [31]).</div></p></div><div class="aux"><div class="resc">Daniel D. Shaye, et al. PLoS One. 2011;6(5):e20085.</div><p class="links"><a class="dblinks" href="/pubmed/21647448" ref="ordinalpos=1932&amp;ncbi_uid=3009653&amp;link_uid=21647448">Citation</a><a class="dblinks" href="/pmc/articles/PMC3102077/" ref="ordinalpos=1932&amp;ncbi_uid=3009653&amp;link_uid=PMC3102077">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1933.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4038461/figure/pcbi-1003578-g006/" ref="ordinalpos=1933&amp;ncbi_uid=6200963&amp;link_uid=PMC4038461" href="/pmc/articles/PMC4038461/"><img src="/pmc/articles/instance/4038461/bin/pcbi.1003578.g006.gif" src-large="/pmc/articles/instance/4038461/bin/pcbi.1003578.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4038461/" ref="ordinalpos=1933&amp;ncbi_uid=6200963&amp;link_uid=PMC4038461" image-link="/pmc/articles/PMC4038461/figure/pcbi-1003578-g006/" class="imagepopup">Figure 6. The VEGF <span class="highlight" style="background-color:">pathway</span> and its relevance to both BPD hypotheses..  From: Finding Novel Molecular Connections between Developmental Processes and Disease. </a></p><div class="supp"><p class="details"><div>The relationships shown here are derived from the VEGF, PI3K-AKT, mTOR, and HIF-1 <span class="highlight" style="background-color:">signaling</span> pathways and the âPathways in Cancerâ map in the KEGG <span class="highlight" style="background-color:">Pathway</span> database. Dashed lines represent indirect regulation. Genes highlighted in orange are the five lung development genes implicated in ROP.</div></p></div><div class="aux"><div class="resc">Jisoo Park, et al. PLoS Comput Biol. 2014 May;10(5):e1003578.</div><p class="links"><a class="dblinks" href="/pubmed/24874013" ref="ordinalpos=1933&amp;ncbi_uid=6200963&amp;link_uid=24874013">Citation</a><a class="dblinks" href="/pmc/articles/PMC4038461/" ref="ordinalpos=1933&amp;ncbi_uid=6200963&amp;link_uid=PMC4038461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1934.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4111578/figure/pone-0103290-g006/" ref="ordinalpos=1934&amp;ncbi_uid=6427222&amp;link_uid=PMC4111578" href="/pmc/articles/PMC4111578/"><img src="/pmc/articles/instance/4111578/bin/pone.0103290.g006.gif" src-large="/pmc/articles/instance/4111578/bin/pone.0103290.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4111578/" ref="ordinalpos=1934&amp;ncbi_uid=6427222&amp;link_uid=PMC4111578" image-link="/pmc/articles/PMC4111578/figure/pone-0103290-g006/" class="imagepopup">Figure 6. Putative mechanisms of WCESP-mediated protection against V. cholerae and other possible bacterial pathogens in C. elegans model..  From: Cranberry Extract Standardized for Proanthocyanidins Promotes the Immune Response of Caenorhabditis elegans to Vibrio cholerae through the p38 MAPK <span class="highlight" style="background-color:">Pathway</span> and HSF-1. </a></p><div class="supp"><p class="details"><div>WCESP can stimulate the innate immunity by activating the p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and by suppressing the IIS <span class="highlight" style="background-color:">pathway</span> which leads to an increase in HSF-1 activity.</div></p></div><div class="aux"><div class="resc">Jessica Dinh, et al. PLoS One. 2014;9(7):e103290.</div><p class="links"><a class="dblinks" href="/pubmed/25062095" ref="ordinalpos=1934&amp;ncbi_uid=6427222&amp;link_uid=25062095">Citation</a><a class="dblinks" href="/pmc/articles/PMC4111578/" ref="ordinalpos=1934&amp;ncbi_uid=6427222&amp;link_uid=PMC4111578">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1935.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2709717/figure/fig1/" ref="ordinalpos=1935&amp;ncbi_uid=1880305&amp;link_uid=PMC2709717" href="/pmc/articles/PMC2709717/"><img src="/pmc/articles/instance/2709717/bin/PPAR2009-830501.001.gif" src-large="/pmc/articles/instance/2709717/bin/PPAR2009-830501.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2709717/" ref="ordinalpos=1935&amp;ncbi_uid=1880305&amp;link_uid=PMC2709717" image-link="/pmc/articles/PMC2709717/figure/fig1/" class="imagepopup">Figure 1.  From: PPAR-Î³ Agonists and Their Effects on IGF-I Receptor <span class="highlight" style="background-color:">Signaling</span>: Implications for Cancer. </a></p><div class="supp"><p class="details"><div>      <i>The IGF system: receptors, ligands, and <span class="highlight" style="background-color:">signaling</span> pathways. </i>   (a) <i> Schematic representation of the major receptors and ligands involved in the IGF system. </i>  Insulin receptor isoforms (IR-A or IR-B) binds insulin with high affinity, while IGF-I receptor (IGF-IR) binds IGF-I and IGF-II (left). In cells expressing both IR and IGF-IR, IR hemireceptors may heterodimerize with IGF-IR hemireceptors, leading to the formation of hybrid IR/IGF-IRs (HRs), which bind IGF-I and IGF-II with high affinity and insulin with a much lower affinity (right). (b)  <i>The IGF system <span class="highlight" style="background-color:">signaling</span> pathways</i>. Schematic representation of the two major <span class="highlight" style="background-color:">signaling</span> pathways involved in IGF system. IGF-I, IGF-II, and insulin bind to their cognate receptors, leading to the activation of the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> and of the Ras/Raf-1/MEK/ERK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">A. Belfiore, et al. PPAR Res. 2009;2009:830501.</div><p class="links"><a class="dblinks" href="/pubmed/19609453" ref="ordinalpos=1935&amp;ncbi_uid=1880305&amp;link_uid=19609453">Citation</a><a class="dblinks" href="/pmc/articles/PMC2709717/" ref="ordinalpos=1935&amp;ncbi_uid=1880305&amp;link_uid=PMC2709717">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1936.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3351453/figure/pone-0037153-g004/" ref="ordinalpos=1936&amp;ncbi_uid=4009069&amp;link_uid=PMC3351453" href="/pmc/articles/PMC3351453/"><img src="/pmc/articles/instance/3351453/bin/pone.0037153.g004.gif" src-large="/pmc/articles/instance/3351453/bin/pone.0037153.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3351453/" ref="ordinalpos=1936&amp;ncbi_uid=4009069&amp;link_uid=PMC3351453" image-link="/pmc/articles/PMC3351453/figure/pone-0037153-g004/" class="imagepopup">Figure 4. Effect of edin RNAi on Drosophila immune <span class="highlight" style="background-color:">signaling</span> in vitro..  From: Functional Characterization of the Infection-Inducible Peptide Edin in Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div>A) <i>Edin</i> RNAi is effective in S2 cells. S2 cells were treated with <i>GFP</i> and <i>edin</i> dsRNA and the cells were induced by adding heat-killed <i>E. coli</i>. Relative expression levels of <i>edin</i> were analyzed from total RNAs with qRT-PCR. nâ=â4 for each sample. (B) <i>Edin</i> expression is not required for the Imd <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> <i>in vitro</i>. S2 cells were transfected with an <i>Attacin A-</i>luciferase reporter together with <i>GFP</i> (negative control), <i>Relish</i> (positive control) and <i>edin</i> dsRNAs. The Imd <span class="highlight" style="background-color:">pathway</span> was activated by adding heat-killed <i>E. coli</i> to the cell culture medium and samples were collected at indicated time points. <i>Edin</i> RNAi causes a 30% decrease in the Imd <span class="highlight" style="background-color:">pathway</span> activity at the 24 h time point. The data for the 0 h and 24 h time points are pooled from 5 indepent experiments (nâ=â17 per sample). For 1 h, 4 h and 8 h time points nâ=â4 per sample. (C) <i>Edin</i> RNAi does not decrease the Imd <span class="highlight" style="background-color:">pathway</span> activity when the <span class="highlight" style="background-color:">pathway</span> is induced with <i>S. marcescens</i>, <i>E. cloacae</i>, peptidoglycan or PGRP-LC. S2 cells were transfected with an <i>AttA</i>-luciferase reporter and <i>edin</i> dsRNA and the Imd <span class="highlight" style="background-color:">pathway</span> was activated with <i>S. marcescens</i> (<i>S.m.</i>), <i>E. cloacae</i> (<i>E.cl.</i>), peptidoglycan (PGN) or a <i>pMT[PGRP-LC]</i> construct. CuSO<sub>4</sub> was used to induce the expression of <i>PGRP-LC</i>. <i>GFP</i> and <i>Relish</i> dsRNAs were used as negative and positive controls, respectively. Unind.â=âno induction. The data for <i>S.m.</i>, <i>E.cl.</i> and PGN are pooled from 3 independent experiments (nâ=â12 per sample). For PGRP-LC, nâ=â3 per sample. (D) <i>Edin</i> RNAi does not affect the Toll <span class="highlight" style="background-color:">pathway</span> activity. S2 cells were transfected with a <i>Drosomycin</i>-luciferase reporter together with <i>GFP</i>, <i>edin</i> and <i>MyD88</i> (positive control) dsRNAs. A constitutively active form of the Toll receptor, Toll<sup>10B</sup>, was used to activate the <span class="highlight" style="background-color:">pathway</span>. The data are pooled from 3 independent experiments, nâ=â10 for each sample. (E) <i>Edin</i> has no effect on the SpÃ¤tzle-induced Toll-<span class="highlight" style="background-color:">pathway</span> activity. S2 cells were transfected with a <i>Drosomycin</i>-luciferase reporter together with <i>GFP</i>, <i>edin</i>, <i>MyD88</i> (control) and <i>Cactus</i> (control) dsRNAs. The Toll <span class="highlight" style="background-color:">pathway</span> was activated with the cleaved, active SpÃ¤tzle ligand (<i>Spz<sup>C106</sup></i>). nâ=â4 for each sample. (F) <i>Edin</i> RNAi has no effect on the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. S2 cells were transfected with a <i>Turandot M</i>-reporter and <i>GFP</i>, <i>STAT</i> (positive control) and <i>edin</i> dsRNAs. The JAK/STAT <span class="highlight" style="background-color:">pathway</span> was activated by overexpressing <i>Hop<sup>Tum-l</sup></i>. nâ=â4 for each sample.</div></p></div><div class="aux"><div class="resc">Leena-Maija Vanha-aho, et al. PLoS One. 2012;7(5):e37153.</div><p class="links"><a class="dblinks" href="/pubmed/22606343" ref="ordinalpos=1936&amp;ncbi_uid=4009069&amp;link_uid=22606343">Citation</a><a class="dblinks" href="/pmc/articles/PMC3351453/" ref="ordinalpos=1936&amp;ncbi_uid=4009069&amp;link_uid=PMC3351453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1937.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC149075/figure/fig9/" ref="ordinalpos=1937&amp;ncbi_uid=210191&amp;link_uid=PMC149075" href="/pmc/articles/PMC149075/"><img src="/pmc/articles/instance/149075/bin/00-0173f9.gif" src-large="/pmc/articles/instance/149075/bin/00-0173f9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC149075/" ref="ordinalpos=1937&amp;ncbi_uid=210191&amp;link_uid=PMC149075" image-link="/pmc/articles/PMC149075/figure/fig9/" class="imagepopup">Figure 9. From: Jasmonic Acid <span class="highlight" style="background-color:">Signaling</span> Modulates Ozone-Induced Hypersensitive Cell Death. </a></p><div class="supp"><p class="details"><div>Schematic Model Illustrating the Potential Role of JA <span class="highlight" style="background-color:">Signaling</span> Pathways in Modulating the Magnitude of O<sub>3</sub>-Induced, SA-Mediated Responses.<br />Both SA and JA <span class="highlight" style="background-color:">signaling</span> pathways are activated in response to O<sub>3</sub> exposure. JA <span class="highlight" style="background-color:">signaling</span> can attenuate the magnitude of the O<sub>3</sub>-induced oxidative burst (upstream of SA accumulation; step 1) and SA-mediated HR cell death <span class="highlight" style="background-color:">pathway</span> (downstream of SA accumulation; step 2). Therefore, plant resistance to oxidative stress stimuli is dependent on SA and JA <span class="highlight" style="background-color:">signaling</span> pathways that then act antagonistically; this allows plants to modulate the relative amount of SA- and JA-inducible defenses, including cell death, as a function of time in response to a specific stress.</div></p></div><div class="aux"><div class="resc">Mulpuri V. Rao, et al. Plant Cell. 2000 September;12(9):1633-1647.</div><p class="links"><a class="dblinks" href="/pubmed/11006337" ref="ordinalpos=1937&amp;ncbi_uid=210191&amp;link_uid=11006337">Citation</a><a class="dblinks" href="/pmc/articles/PMC149075/" ref="ordinalpos=1937&amp;ncbi_uid=210191&amp;link_uid=PMC149075">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1938.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3091413/figure/f6-cin-2011-093/" ref="ordinalpos=1938&amp;ncbi_uid=2952273&amp;link_uid=PMC3091413" href="/pmc/articles/PMC3091413/"><img src="/pmc/articles/instance/3091413/bin/cin-2011-093f6.gif" src-large="/pmc/articles/instance/3091413/bin/cin-2011-093f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3091413/" ref="ordinalpos=1938&amp;ncbi_uid=2952273&amp;link_uid=PMC3091413" image-link="/pmc/articles/PMC3091413/figure/f6-cin-2011-093/" class="imagepopup">Figure 6.  From: Associating GWAS Information with the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Using Transcription Profiling. </a></p><div class="supp"><p class="details"><div>Results of <span class="highlight" style="background-color:">pathway</span> prediction and network modeling showing interactions between genes containing SNPs associated with risk for breast cancer and genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> and other biological pathways. The results are based on genes containing SNPs replicated in multiple independent GWAS. Nodes represent genes and vertices represent interactions. The color code denotes the biological process in which the genes are involved as defined in Figure 3. The color codes in the vertices indicate the experimental techniques or a combination thereof used to confirm the relationship between the genes as determined by the experiments reported in the literature. Candidate genes containing SNPs associated with risk for breast cancer are shown in red, genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown in blue and novel genes are shown in black.</div></p></div><div class="aux"><div class="resc">Chindo Hicks, et al. Cancer Inform. 2011;10:93-108.</div><p class="links"><a class="dblinks" href="/pubmed/21584266" ref="ordinalpos=1938&amp;ncbi_uid=2952273&amp;link_uid=21584266">Citation</a><a class="dblinks" href="/pmc/articles/PMC3091413/" ref="ordinalpos=1938&amp;ncbi_uid=2952273&amp;link_uid=PMC3091413">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1939.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3733232/figure/F3/" ref="ordinalpos=1939&amp;ncbi_uid=5165997&amp;link_uid=PMC3733232" href="/pmc/articles/PMC3733232/"><img src="/pmc/articles/instance/3733232/bin/nihms237672f3.gif" src-large="/pmc/articles/instance/3733232/bin/nihms237672f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3733232/" ref="ordinalpos=1939&amp;ncbi_uid=5165997&amp;link_uid=PMC3733232" image-link="/pmc/articles/PMC3733232/figure/F3/" class="imagepopup">Figure 3. Crosstalk between the epidermal growth factor (EGF)-Ras- extracellular regulated kinase (Erk)-mitogen-activated protein (MAP) kinase <span class="highlight" style="background-color:">pathway</span> and both the transforming growth factor-Î²-inducible early response gene TIEG and Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancer.  From: TGF?-mediated <span class="highlight" style="background-color:">signaling</span> and transcriptional regulation in pancreatic development and cancer. </a></p><div class="supp"><p class="details"><div>Hypersensitive Ras and its downstream mediator kinase Erk block transforming growth factor-Î² (TGFÎ²) <span class="highlight" style="background-color:">signaling</span> through phosphorylation of the receptor Smads S2/S3 and thereby inhibit the complex formation with Smad4. In addition, EGF-activated Erk MAPkinase phosphorylates TIEG2 and thereby antagonizes its transcriptional repression activity through interfering with the binding of the Sin3A corepressor complex.</div></p></div><div class="aux"><div class="resc">Volker Ellenrieder, et al. Curr Opin Gastroenterol. ;17(5):434-440.</div><p class="links"><a class="dblinks" href="/pubmed/17031197" ref="ordinalpos=1939&amp;ncbi_uid=5165997&amp;link_uid=17031197">Citation</a><a class="dblinks" href="/pmc/articles/PMC3733232/" ref="ordinalpos=1939&amp;ncbi_uid=5165997&amp;link_uid=PMC3733232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1940.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2757291/figure/F6/" ref="ordinalpos=1940&amp;ncbi_uid=2026601&amp;link_uid=PMC2757291" href="/pmc/articles/PMC2757291/"><img src="/pmc/articles/instance/2757291/bin/nihms35067f6.gif" src-large="/pmc/articles/instance/2757291/bin/nihms35067f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2757291/" ref="ordinalpos=1940&amp;ncbi_uid=2026601&amp;link_uid=PMC2757291" image-link="/pmc/articles/PMC2757291/figure/F6/" class="imagepopup">Figure 6. cTEC- and thymocyte-mediated T cell selection pathways.  From: The SLAM-Associated Protein (SAP)/Fyn/PKC? <span class="highlight" style="background-color:">Pathway</span> is Required for Thymocyte-mediated CD4 T Cell Development. </a></p><div class="supp"><p class="details"><div>E-CD4 T cells are positively selected through heterotypic interactions between DP thymocytes and MHC class II-expressing cTEC (<span class="highlight" style="background-color:">pathway</span> 1). Thymocyte-mediated T cell selection <span class="highlight" style="background-color:">pathway</span> select T-CD4 T cells and NKT cells through homotypic interactions between TCR on developing DP thymocytes with MHC class II or CD1d on DP thymocytes, respectively (<span class="highlight" style="background-color:">pathway</span> 2). Our studies indicate that the SAP/Fyn/PKCÎ¸ <span class="highlight" style="background-color:">signaling</span> cascade is a common <span class="highlight" style="background-color:">pathway</span> required for the generation of both T-CD4 T cells and immature NKT cells, while T-bet and IL-15R are selectively involved in the maturation of NKT cells. Whether E-CD4 T cells and T-CD4 T cells share similar molecular requirements for their maturation is currently unknown.</div></p></div><div class="aux"><div class="resc">Wei Li, et al. Immunity. ;27(5):763-774.</div><p class="links"><a class="dblinks" href="/pubmed/18031696" ref="ordinalpos=1940&amp;ncbi_uid=2026601&amp;link_uid=18031696">Citation</a><a class="dblinks" href="/pmc/articles/PMC2757291/" ref="ordinalpos=1940&amp;ncbi_uid=2026601&amp;link_uid=PMC2757291">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1941.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2673763/figure/fig6/" ref="ordinalpos=1941&amp;ncbi_uid=1759872&amp;link_uid=PMC2673763" href="/pmc/articles/PMC2673763/"><img src="/pmc/articles/instance/2673763/bin/DEV029249F6.gif" src-large="/pmc/articles/instance/2673763/bin/DEV029249F6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2673763/" ref="ordinalpos=1941&amp;ncbi_uid=1759872&amp;link_uid=PMC2673763" image-link="/pmc/articles/PMC2673763/figure/fig6/" class="imagepopup">Fig. 6. From: Notch <span class="highlight" style="background-color:">signaling</span> promotes airway mucous metaplasia and inhibits alveolar  development. </a></p><div class="supp"><p class="details"><div><b>Stat6 is not necessary for DLL4-induced mucous metaplasia.</b>  (<b>A-C</b>) Immunohistochemistry of control tracheas that were harvested at  E14.5 and cultured for 7 days. Tracheas displayed normal, low levels of mucous  differentiation, identified by Muc5AC staining (green) (A). Addition of IL13  (B) or DLL4 (C) induced mucous metaplasia. (<b>Aâ²-C</b>â²)  Immunohistochemistry of <i>Stat6</i><sup>â/â</sup> tracheas that  were harvested at E14.5 and cultured for 7 days. Tracheas displayed normal,  low levels of mucous differentiation (Aâ²). Addition of IL13 (Bâ²)  failed to induce mucous metaplasia as predicted. Addition of DLL4 (Câ²)  induced mucous metaplasia. (<b>D</b>) Possible models for the interaction of  Notch and the IL13/Stat6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which both result in mucous  metaplasia. Notch <span class="highlight" style="background-color:">signaling</span> cannot be upstream of Stat6 activation as  Notch-induced metaplasia occurs in <i>Stat6</i>-null trachea. Notch  <span class="highlight" style="background-color:">signaling</span> might be downstream of Stat6 <span class="highlight" style="background-color:">signaling</span>, although this is unlikely  because Stat6 activation is not associated with Notch target induction.  Therefore, Notch and Stat6 <span class="highlight" style="background-color:">signaling</span> may represent two parallel pathways for  inducing mucous metaplasia. Scale bars: 100 Î¼m.</div></p></div><div class="aux"><div class="resc">J. Sawalla Guseh, et al. Development. 2009 May 15;136(10):1751-1759.</div><p class="links"><a class="dblinks" href="/pubmed/19369400" ref="ordinalpos=1941&amp;ncbi_uid=1759872&amp;link_uid=19369400">Citation</a><a class="dblinks" href="/pmc/articles/PMC2673763/" ref="ordinalpos=1941&amp;ncbi_uid=1759872&amp;link_uid=PMC2673763">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1942.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1592845/figure/f8/" ref="ordinalpos=1942&amp;ncbi_uid=849048&amp;link_uid=PMC1592845" href="/pmc/articles/PMC1592845/"><img src="/pmc/articles/instance/1592845/bin/zmb0170661960008.gif" src-large="/pmc/articles/instance/1592845/bin/zmb0170661960008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1592845/" ref="ordinalpos=1942&amp;ncbi_uid=849048&amp;link_uid=PMC1592845" image-link="/pmc/articles/PMC1592845/figure/f8/" class="imagepopup">FIG. 8. From: Actin-Induced Hyperactivation of the Ras <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Leads to Apoptosis in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>Actin stabilization triggers constitutive Ras/cAMP/PKA <span class="highlight" style="background-color:">signaling</span> and apoptosis in stationary-phase <i>S. cerevisiae</i> cells. In wild-type cells, components of the Ras <span class="highlight" style="background-color:">pathway</span> are not found in association with F-actin during stationary phase. A reduction in Ras activity leads to a fall in cAMP, which results in PKA inactivation and the downregulation of <span class="highlight" style="background-color:">signaling</span> (A). (B) In actin-stabilized cells, components of the cAMP generation machinery, adenylyl cyclase (Cyr1p) and Srv2p, are found in association with large aggregates of actin during stationary phase and Ras2p is found to be constitutively active. This hyperactive Ras <span class="highlight" style="background-color:">signaling</span> state gives rise to the elevation of cAMP and the activation of PKA. This cascade of events triggers the elevation of ROS derived from the mitochondria and apoptosis. The PKA subunit Tpk3p is primarily responsible for the induction of apoptosis in response to actin-mediated hyperactivation of the Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. It is likely that Tpk3p acts directly on both F-actin regulatory proteins and alters the enzymatic content of mitochondria to elicit this response. The combination of aggregated actin and constitutive Ras <span class="highlight" style="background-color:">signaling</span> may act together to establish conditions which lead to apoptotic cell death.</div></p></div><div class="aux"><div class="resc">C. W. Gourlay, et al. Mol Cell Biol. 2006 September;26(17):6487-6501.</div><p class="links"><a class="dblinks" href="/pubmed/16914733" ref="ordinalpos=1942&amp;ncbi_uid=849048&amp;link_uid=16914733">Citation</a><a class="dblinks" href="/pmc/articles/PMC1592845/" ref="ordinalpos=1942&amp;ncbi_uid=849048&amp;link_uid=PMC1592845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1943.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2778196/figure/F7/" ref="ordinalpos=1943&amp;ncbi_uid=2521270&amp;link_uid=PMC2778196" href="/pmc/articles/PMC2778196/"><img src="/pmc/articles/instance/2778196/bin/nihms150535f7.gif" src-large="/pmc/articles/instance/2778196/bin/nihms150535f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2778196/" ref="ordinalpos=1943&amp;ncbi_uid=2521270&amp;link_uid=PMC2778196" image-link="/pmc/articles/PMC2778196/figure/F7/" class="imagepopup">Figure 7. Downstream Ca2+-dependent <span class="highlight" style="background-color:">signaling</span> influences the balance between the long and the short paths of granule delivery.  From: Kinetics of early TCR <span class="highlight" style="background-color:">signaling</span> regulate the <span class="highlight" style="background-color:">pathway</span> of lytic granule delivery to the secretory domain. </a></p><div class="supp"><p class="details"><div>The stronger early TCR <span class="highlight" style="background-color:">signaling</span> is initiated by a greater number of activated <span class="highlight" style="background-color:">signaling</span> proteins per microcluster and leads to rapid kinetics of intracellular Ca<sup>2+</sup> accumulation, while a weaker early <span class="highlight" style="background-color:">signaling</span> does not. Rising the concentration of intracellular Ca<sup>2+</sup> results in the activation of downstream <span class="highlight" style="background-color:">signaling</span> that regulates granule movement to the MTOC. In contrast, MTOC translocation is Ca<sup>2+</sup>-independent and is primarily mediated by DAG-dependent <span class="highlight" style="background-color:">signaling</span>. A faster Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> in more effective CTL stimulates swift granule movement towards the MTOC and the granules are concentrated near the MTOC prior to MTOC polarization. The MTOC then directly delivers granules to the secretory domain â the short path. A slower Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> in less effective CTL results in a delay in granule recruitment and granules move along the microtubules to the periphery of the synapse looping through the pSMAC â a longer path. MTOC polarization to the mature synapse also requires F-actin remodeling and segregation as well as the recruitment of ADAP-associated dynein to the periphery of the actin ring. Vav-dependent downstream <span class="highlight" style="background-color:">signaling</span> plays a critical role in regulating these molecular events. Cartoons at the upper and bottom left are courtesy of Christoph Schmutte.</div></p></div><div class="aux"><div class="resc">Allison M. Beal, et al. Immunity. ;31(4):632-642.</div><p class="links"><a class="dblinks" href="/pubmed/19833088" ref="ordinalpos=1943&amp;ncbi_uid=2521270&amp;link_uid=19833088">Citation</a><a class="dblinks" href="/pmc/articles/PMC2778196/" ref="ordinalpos=1943&amp;ncbi_uid=2521270&amp;link_uid=PMC2778196">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1944.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4218794/figure/pone-0111640-g002/" ref="ordinalpos=1944&amp;ncbi_uid=6804058&amp;link_uid=PMC4218794" href="/pmc/articles/PMC4218794/"><img src="/pmc/articles/instance/4218794/bin/pone.0111640.g002.gif" src-large="/pmc/articles/instance/4218794/bin/pone.0111640.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4218794/" ref="ordinalpos=1944&amp;ncbi_uid=6804058&amp;link_uid=PMC4218794" image-link="/pmc/articles/PMC4218794/figure/pone-0111640-g002/" class="imagepopup">Figure 2. Genes that were involved in Toll-like receptor <span class="highlight" style="background-color:">signaling</span>, IL-10 <span class="highlight" style="background-color:">signaling</span>, Role of PKR in Interferon Induction and Antiviral Response and NFkB <span class="highlight" style="background-color:">signaling</span> pathways..  From: Patient-Based Transcriptome-Wide Analysis Identify Interferon and Ubiquination Pathways as Potential Predictors of Influenza A Disease Severity. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">pathway</span> names were shared between different groups but the activated genes in each <span class="highlight" style="background-color:">pathway</span> were different. Differentially expressed genes (FDR &lt;0.05, fold change &gt;2) were highlighted in grey.</div></p></div><div class="aux"><div class="resc">Long Truong Hoang, et al. PLoS One. 2014;9(11):e111640.</div><p class="links"><a class="dblinks" href="/pubmed/25365328" ref="ordinalpos=1944&amp;ncbi_uid=6804058&amp;link_uid=25365328">Citation</a><a class="dblinks" href="/pmc/articles/PMC4218794/" ref="ordinalpos=1944&amp;ncbi_uid=6804058&amp;link_uid=PMC4218794">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1945.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2951409/figure/fig05/" ref="ordinalpos=1945&amp;ncbi_uid=2518568&amp;link_uid=PMC2951409" href="/pmc/articles/PMC2951409/"><img src="/pmc/articles/instance/2951409/bin/pnas.1001794107fig05.gif" src-large="/pmc/articles/instance/2951409/bin/pnas.1001794107fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2951409/" ref="ordinalpos=1945&amp;ncbi_uid=2518568&amp;link_uid=PMC2951409" image-link="/pmc/articles/PMC2951409/figure/fig05/" class="imagepopup">Fig. 5. From: WASP is activated by phosphatidylinositol-4,5-bisphosphate to restrict synapse growth in a <span class="highlight" style="background-color:">pathway</span> parallel to bone morphogenetic protein <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Tweek acts in WSP and NWK <span class="highlight" style="background-color:">signaling</span>. (<i>A</i>â<i>C</i>) NMJs labeled with anti-HRP in controls (<i>A</i>), <i>tweek</i> mutants that neuronally express PLC<sub>Î´1</sub>-PH<sup>S39R</sup>-GFP (<i>B</i>), and <i>tweek</i> mutants that express PLC<sub>Î´1</sub>- PH-GFP (<i>C</i>). (<i>D</i>â<i>F</i>) NMJ labeling with anti-HRP in <i>tweek;wsp</i> double mutants (<i>D</i>), in <i>tweek;nwk</i> double mutants (<i>E</i>), and in <i>nwk wsp</i> double mutants neuronally expressing PLC<sub>Î´1</sub>-PH-GFP (<i>F</i>). (<i>G</i>) Quantification of data shown in <i>A</i>â<i>F</i>. Error bars indicate SEM; ***<i>P</i> &lt; 0.001; <i>n</i> is indicated in the bars; ns, not significant. (<i>H</i>) Model of the genetic <span class="highlight" style="background-color:">pathway</span> encompassing Tweek, NWK, and WSP (Wasp).</div></p></div><div class="aux"><div class="resc">Thang Manh Khuong, et al. Proc Natl Acad Sci U S A. 2010 October 5;107(40):17379-17384.</div><p class="links"><a class="dblinks" href="/pubmed/20844206" ref="ordinalpos=1945&amp;ncbi_uid=2518568&amp;link_uid=20844206">Citation</a><a class="dblinks" href="/pmc/articles/PMC2951409/" ref="ordinalpos=1945&amp;ncbi_uid=2518568&amp;link_uid=PMC2951409">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1946.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2574814/figure/F7/" ref="ordinalpos=1946&amp;ncbi_uid=1512371&amp;link_uid=PMC2574814" href="/pmc/articles/PMC2574814/"><img src="/pmc/articles/instance/2574814/bin/nihms-65799-f0027.gif" src-large="/pmc/articles/instance/2574814/bin/nihms-65799-f0027.jpg" alt="FIGURE 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2574814/" ref="ordinalpos=1946&amp;ncbi_uid=1512371&amp;link_uid=PMC2574814" image-link="/pmc/articles/PMC2574814/figure/F7/" class="imagepopup">FIGURE 7. Molecular mechanisms by which osteogenic oxysterols induce osteogenic differentiation of marrow stromal cells.  From: Oxysterol-Induced Osteogenic Differentiation of Marrow Stromal Cells is Regulated by Dkk-1 Inhibitable and PI3-Kinase Mediated <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Oxysterols activate two distinct <span class="highlight" style="background-color:">signaling</span> pathways: 1) the hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and 2) a Dkk-1-inhibitable &amp; Î²-catenin/TCF/Lef independent <span class="highlight" style="background-color:">pathway</span>. Activation of these pathways mediate osteogenic differentiation of MSC and cooperatively or distinctly regulate the markers and genes associated with osteogenic differentiation and Wnt <span class="highlight" style="background-color:">signaling</span> including alkaline phosphatase (ALP) activity, osteocalcin (OCN) mRNA expression, Wif-1 and Nkd2 expression, and matrix mineralization.</div></p></div><div class="aux"><div class="resc">Christopher M. Amantea, et al. J Cell Biochem. ;105(2):424-436.</div><p class="links"><a class="dblinks" href="/pubmed/18613030" ref="ordinalpos=1946&amp;ncbi_uid=1512371&amp;link_uid=18613030">Citation</a><a class="dblinks" href="/pmc/articles/PMC2574814/" ref="ordinalpos=1946&amp;ncbi_uid=1512371&amp;link_uid=PMC2574814">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1947.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3958333/figure/pbio-1001810-g013/" ref="ordinalpos=1947&amp;ncbi_uid=5949294&amp;link_uid=PMC3958333" href="/pmc/articles/PMC3958333/"><img src="/pmc/articles/instance/3958333/bin/pbio.1001810.g013.gif" src-large="/pmc/articles/instance/3958333/bin/pbio.1001810.g013.jpg" alt="Figure 13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3958333/" ref="ordinalpos=1947&amp;ncbi_uid=5949294&amp;link_uid=PMC3958333" image-link="/pmc/articles/PMC3958333/figure/pbio-1001810-g013/" class="imagepopup">Figure 13. Model for a bifurcation in the PDFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> controlling the molecular clock and neuronal excitability..  From: Dual PDF <span class="highlight" style="background-color:">Signaling</span> Pathways Reset Clocks Via TIMELESS and Acutely Excite Target Neurons to Control Circadian Behavior. </a></p><div class="supp"><p class="details"><div>PDFR acts through Gs and adenylate cyclase (AC) to increase levels of cAMP which may directly activate a cyclic-nucleotide-gated (CNG) channel (green <span class="highlight" style="background-color:">pathway</span>) to acutely depolarize the cell and increase the action potential firing rate. cAMP also activates PKA, promoting TIM stability and progression of the molecular clock (blue <span class="highlight" style="background-color:">pathway</span>). Light activates CRY, which promotes TIM degradation. The molecular clock also controls PKA transcripts, thereby controlling signal transduction to the clock through a feedback mechanism.</div></p></div><div class="aux"><div class="resc">Adam Seluzicki, et al. PLoS Biol. 2014 March;12(3):e1001810.</div><p class="links"><a class="dblinks" href="/pubmed/24643294" ref="ordinalpos=1947&amp;ncbi_uid=5949294&amp;link_uid=24643294">Citation</a><a class="dblinks" href="/pmc/articles/PMC3958333/" ref="ordinalpos=1947&amp;ncbi_uid=5949294&amp;link_uid=PMC3958333">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1948.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204831/figure/F8/" ref="ordinalpos=1948&amp;ncbi_uid=3321853&amp;link_uid=PMC3204831" href="/pmc/articles/PMC3204831/"><img src="/pmc/articles/instance/3204831/bin/JCI46122.f8.gif" src-large="/pmc/articles/instance/3204831/bin/JCI46122.f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204831/" ref="ordinalpos=1948&amp;ncbi_uid=3321853&amp;link_uid=PMC3204831" image-link="/pmc/articles/PMC3204831/figure/F8/" class="imagepopup">Figure 8. Schematic representation of FGF23 <span class="highlight" style="background-color:">signaling</span> in classic target cells and cardiomyocytes..  From: FGF23 induces left ventricular hypertrophy. </a></p><div class="supp"><p class="details"><div>In the kidney and parathyroid glands, FGF23 <span class="highlight" style="background-color:">signaling</span> requires FGFR and the coreceptor klotho. FGF23-klotho binding to FGFR stimulates autophosphorylation of the receptor tyrosine kinase and induces <span class="highlight" style="background-color:">signaling</span> through 3 major pathways: Ras-MAPK, PI3K-Akt, and PLCÎ³-PKC. FGF23 regulates phosphorus balance by altering expression of genes involved in parathyroid, vitamin D, and phosphorus metabolism. In cardiomyocytes, FGF2 <span class="highlight" style="background-color:">signaling</span> requires FGFR and heparan sulfate proteoglycans (HSP) as coreceptor and signals primarily through the Ras-MAPK <span class="highlight" style="background-color:">pathway</span>. Binding of FGF23 to FGFR on cardiomyocytes stimulates autophosphorylation of the receptor tyrosine kinase independent of klotho, which is not expressed in cardiomyocytes, and signals primarily through the PLCÎ³-calcineurin <span class="highlight" style="background-color:">pathway</span>. Whether HSP acts as coreceptor remains to be determined.</div></p></div><div class="aux"><div class="resc">Christian Faul, et al. J Clin Invest. 2011 November 1;121(11):4393-4408.</div><p class="links"><a class="dblinks" href="/pubmed/21985788" ref="ordinalpos=1948&amp;ncbi_uid=3321853&amp;link_uid=21985788">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204831/" ref="ordinalpos=1948&amp;ncbi_uid=3321853&amp;link_uid=PMC3204831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1949.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2242658/figure/F8/" ref="ordinalpos=1949&amp;ncbi_uid=1285048&amp;link_uid=PMC2242658" href="/pmc/articles/PMC2242658/"><img src="/pmc/articles/instance/2242658/bin/bcr1769-8.gif" src-large="/pmc/articles/instance/2242658/bin/bcr1769-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2242658/" ref="ordinalpos=1949&amp;ncbi_uid=1285048&amp;link_uid=PMC2242658" image-link="/pmc/articles/PMC2242658/figure/F8/" class="imagepopup">Figure 8.  From: Autocrine WNT <span class="highlight" style="background-color:">signaling</span> contributes to breast cancer cell proliferation via the canonical WNT <span class="highlight" style="background-color:">pathway</span> and EGFR transactivation. </a></p><div class="supp"><p class="details"><div>Schematic representation of wingless and integration site growth factor (WNT)-induced epidermal growth factor receptor (EGFR) transactivation. Our results show that Wnt1 induces a <span class="highlight" style="background-color:">signaling</span> cascade that links the activation of EGFR in a manner dependent on Dishevelled, SRC, metalloprotease, and EGF-like ligand to the sFRP1-sensitive activation of Frizzled receptors. The activation of EGFR, which may occur via phosphorylation of Y845, an SRC phosphorylation site, triggers activation of the extracellular signal-regulated kinase 1/2 (ERK1/2) <span class="highlight" style="background-color:">pathway</span>. Together with the canonical WNT/Î²-catenin <span class="highlight" style="background-color:">pathway</span> and its target genes, including <i>c-Myc</i>, the ERK1/2 <span class="highlight" style="background-color:">pathway</span> promotes proliferation and survival of breast cancer cells. Furthermore, activation of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> by Wnt1 may contribute to the development of anti-estrogen resistance. DVL, Dishevelled; EGF, epidermal growth factor; ER, estrogen receptor; sFRP1, secreted Frizzled-related protein 1.</div></p></div><div class="aux"><div class="resc">Thomas Schlange, et al. Breast Cancer Res. 2007;9(5):R63-R63.</div><p class="links"><a class="dblinks" href="/pubmed/17897439" ref="ordinalpos=1949&amp;ncbi_uid=1285048&amp;link_uid=17897439">Citation</a><a class="dblinks" href="/pmc/articles/PMC2242658/" ref="ordinalpos=1949&amp;ncbi_uid=1285048&amp;link_uid=PMC2242658">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1950.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3745214/figure/F1/" ref="ordinalpos=1950&amp;ncbi_uid=5204119&amp;link_uid=PMC3745214" href="/pmc/articles/PMC3745214/"><img src="/pmc/articles/instance/3745214/bin/nihms478633f1.gif" src-large="/pmc/articles/instance/3745214/bin/nihms478633f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3745214/" ref="ordinalpos=1950&amp;ncbi_uid=5204119&amp;link_uid=PMC3745214" image-link="/pmc/articles/PMC3745214/figure/F1/" class="imagepopup">Figure 1. TGF-Î² <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Targeting TGF-? <span class="highlight" style="background-color:">signaling</span> in cancer. </a></p><div class="supp"><p class="details"><div><b>A.</b> TGF-Î² ligands signal through distinct receptors and Smads that are modulated by adaptor proteins and ubiquitinators. TGF-Î² binds to serine threonine kinase receptor complexes that phosphorylate R-Smads as well as adaptor proteins such as Î²2-spectrin. R-Smads, Î²2-spectrin and Smad-4 form a heteromeric complex, translocate to the nucleus and regulate target genes expression. At all levels, Smad modulation occurs through adaptor proteins as well as E3 ligases such as PRAJA and Smurfs, generating diverse and complex signals. <b>B.</b> Smad-independent <span class="highlight" style="background-color:">signaling</span>. TGF-Î² can promote the activity of several <span class="highlight" style="background-color:">signaling</span> pathways other than Smad, including mitogen activated protein kinases (MAPKs), phosphoinositide 3â² kinase (PI3K), TRAF6-TAK1-p38/JNK, Rho-Rock, among others. Such alternative signal transducers often regulate the Smad <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Lior H Katz, et al. Expert Opin Ther Targets. ;17(7):743-760.</div><p class="links"><a class="dblinks" href="/pubmed/23651053" ref="ordinalpos=1950&amp;ncbi_uid=5204119&amp;link_uid=23651053">Citation</a><a class="dblinks" href="/pmc/articles/PMC3745214/" ref="ordinalpos=1950&amp;ncbi_uid=5204119&amp;link_uid=PMC3745214">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1951.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2644150/figure/F7/" ref="ordinalpos=1951&amp;ncbi_uid=1658281&amp;link_uid=PMC2644150" href="/pmc/articles/PMC2644150/"><img src="/pmc/articles/instance/2644150/bin/zpq9990965110007.gif" src-large="/pmc/articles/instance/2644150/bin/zpq9990965110007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2644150/" ref="ordinalpos=1951&amp;ncbi_uid=1658281&amp;link_uid=PMC2644150" image-link="/pmc/articles/PMC2644150/figure/F7/" class="imagepopup">Fig. 7. From: Tissue plasminogen activator modulates the cellular and behavioral response to cocaine. </a></p><div class="supp"><p class="details"><div>Schematic representation of tPA's role in acute cocaine-induced neuronal <span class="highlight" style="background-color:">signaling</span>. <span class="highlight" style="background-color:">Signaling</span> events modulated by tPA in the amygdala are indicated by black arrows, and those that occur in the NAc are indicated by red arrows. Components of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> hypothesized to be altered in tPAâ/â mice are shaded in gray. In the amygdala, acute cocaine exposure increases extracellular tPA activity in a CRF-dependent manner. tPA released into the extracellular space is hypothesized to modulate DA D1 receptor-mediated <span class="highlight" style="background-color:">signaling</span> to activate IEGs. In the NAc, tPA may modulate <span class="highlight" style="background-color:">signaling</span> through both D1 and NMDAR to modulate IEG expression. In the absence of tPA, cocaine-induced IEG expression is attenuated, thereby suggesting that tPA is an important component of the <span class="highlight" style="background-color:">signaling</span> cascade modulating cocaine-induced neuroadaptation.</div></p></div><div class="aux"><div class="resc">Rajani Maiya, et al. Proc Natl Acad Sci U S A. 2009 February 10;106(6):1983-1988.</div><p class="links"><a class="dblinks" href="/pubmed/19181855" ref="ordinalpos=1951&amp;ncbi_uid=1658281&amp;link_uid=19181855">Citation</a><a class="dblinks" href="/pmc/articles/PMC2644150/" ref="ordinalpos=1951&amp;ncbi_uid=1658281&amp;link_uid=PMC2644150">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1952.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3358533/figure/F5/" ref="ordinalpos=1952&amp;ncbi_uid=4949809&amp;link_uid=PMC3358533" href="/pmc/articles/PMC3358533/"><img src="/pmc/articles/instance/3358533/bin/nihms370101f5.gif" src-large="/pmc/articles/instance/3358533/bin/nihms370101f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3358533/" ref="ordinalpos=1952&amp;ncbi_uid=4949809&amp;link_uid=PMC3358533" image-link="/pmc/articles/PMC3358533/figure/F5/" class="imagepopup">Fig. 5.  From: Chronic occupational exposure to arsenic induces carcinogenic gene <span class="highlight" style="background-color:">signaling</span> networks and neoplastic transformation in human lung epithelial cells. </a></p><div class="supp"><p class="details"><div><i>PPARÎ±</i> <span class="highlight" style="background-color:">signaling</span> canonical <span class="highlight" style="background-color:">pathway</span> in chronic arsenic-exposed lung epithelial cells. Src/MAPK, NF-ÎºB and <i>NCOR1</i> <span class="highlight" style="background-color:">signaling</span> appeared to disrupt <i>PPARÎ±/Î´</i> lipid homeostatic control. Yellow and blue represent up- and down-regulation, respectively, compared to unexposed controls. Color intensity signifies fold change.</div></p></div><div class="aux"><div class="resc">Todd A. Stueckle, et al. Toxicol Appl Pharmacol. ;261(2):204-216.</div><p class="links"><a class="dblinks" href="/pubmed/22521957" ref="ordinalpos=1952&amp;ncbi_uid=4949809&amp;link_uid=22521957">Citation</a><a class="dblinks" href="/pmc/articles/PMC3358533/" ref="ordinalpos=1952&amp;ncbi_uid=4949809&amp;link_uid=PMC3358533">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1953.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4019493/figure/F8/" ref="ordinalpos=1953&amp;ncbi_uid=6454247&amp;link_uid=PMC4019493" href="/pmc/articles/PMC4019493/"><img src="/pmc/articles/instance/4019493/bin/1620fig8.gif" src-large="/pmc/articles/instance/4019493/bin/1620fig8.jpg" alt="FIGURE 8:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4019493/" ref="ordinalpos=1953&amp;ncbi_uid=6454247&amp;link_uid=PMC4019493" image-link="/pmc/articles/PMC4019493/figure/F8/" class="imagepopup">FIGURE 8:.  From: Dab2 inhibits the cholesterol-dependent activation of JNK by TGF-?. </a></p><div class="supp"><p class="details"><div>A model for regulation of TGF-Î² <span class="highlight" style="background-color:">signaling</span> by Dab2 and cholesterol. TÎ²RI is distributed in multiple plasma membrane domains, including CCPs and cholesterol-rich domains/caveolae. Although Smad <span class="highlight" style="background-color:">signaling</span> in response to TGF-Î² is not affected by TÎ²RI localization to either domain, TGF-Î² stimulation of the JNK <span class="highlight" style="background-color:">pathway</span> is initiated mainly in the cholesterol-rich domains. Dab2, which binds both TÎ²RI and clathrin, targets TÎ²RI preferentially to CCPs, shifting the balance of TGF-Î² <span class="highlight" style="background-color:">signaling</span> such that Smad <span class="highlight" style="background-color:">signaling</span> is retained while JNK activation is diminished.</div></p></div><div class="aux"><div class="resc">Keren E. Shapira, et al. Mol Biol Cell. 2014 May 15;25(10):1620-1628.</div><p class="links"><a class="dblinks" href="/pubmed/24648493" ref="ordinalpos=1953&amp;ncbi_uid=6454247&amp;link_uid=24648493">Citation</a><a class="dblinks" href="/pmc/articles/PMC4019493/" ref="ordinalpos=1953&amp;ncbi_uid=6454247&amp;link_uid=PMC4019493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1954.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3410582/figure/fig01/" ref="ordinalpos=1954&amp;ncbi_uid=4152063&amp;link_uid=PMC3410582" href="/pmc/articles/PMC3410582/"><img src="/pmc/articles/instance/3410582/bin/ajcr0002-0434-f1.gif" src-large="/pmc/articles/instance/3410582/bin/ajcr0002-0434-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3410582/" ref="ordinalpos=1954&amp;ncbi_uid=4152063&amp;link_uid=PMC3410582" image-link="/pmc/articles/PMC3410582/figure/fig01/" class="imagepopup">Figure 1.  From: Androgen receptor as a targeted therapy for breast cancer. </a></p><div class="supp"><p class="details"><div>Schematic of <span class="highlight" style="background-color:">signaling</span> crosstalk between androgen receptor and epidermal growth factor receptor pathways. Concurrent <span class="highlight" style="background-color:">signaling</span> via androgen receptor with the epidermal growth factor receptor (left) leads to cell cycle arrest. <span class="highlight" style="background-color:">Signaling</span> via one or the other <span class="highlight" style="background-color:">pathway</span> (right) leads to growth proliferation. p21 is required for AR <span class="highlight" style="background-color:">signaling</span> in either situation.</div></p></div><div class="aux"><div class="resc">Joseph P Garay, et al. Am J Cancer Res. 2012;2(4):434-445.</div><p class="links"><a class="dblinks" href="/pubmed/22860233" ref="ordinalpos=1954&amp;ncbi_uid=4152063&amp;link_uid=22860233">Citation</a><a class="dblinks" href="/pmc/articles/PMC3410582/" ref="ordinalpos=1954&amp;ncbi_uid=4152063&amp;link_uid=PMC3410582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1955.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2637482/figure/F2/" ref="ordinalpos=1955&amp;ncbi_uid=1746853&amp;link_uid=PMC2637482" href="/pmc/articles/PMC2637482/"><img src="/pmc/articles/instance/2637482/bin/cam0204_0220_fig002.gif" src-large="/pmc/articles/instance/2637482/bin/cam0204_0220_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2637482/" ref="ordinalpos=1955&amp;ncbi_uid=1746853&amp;link_uid=PMC2637482" image-link="/pmc/articles/PMC2637482/figure/F2/" class="imagepopup">Figure 2.  From: Robo4. </a></p><div class="supp"><p class="details"><div>Crosstalk between Robo4/Slit2 and VEGF <span class="highlight" style="background-color:">signaling</span> pathways. Jones et al. suggests that Robo4/Slit2 <span class="highlight" style="background-color:">signaling</span> can negatively regulate angiogenesis. Activation of VEGFR2 by VEGF-165 leads to the phosphorylation of SFK family member Src, which in turn stimulates downstream effector, Rac1 leading to angiogenesis and increased vascular permeability. Binding of Slit2 to Robo4 leads to <span class="highlight" style="background-color:">signaling</span> that blocks the VEGFR2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Matthew Weitzman, et al. Cell Adh Migr. 2008 Oct-Dec;2(4):220-222.</div><p class="links"><a class="dblinks" href="/pubmed/19270536" ref="ordinalpos=1955&amp;ncbi_uid=1746853&amp;link_uid=19270536">Citation</a><a class="dblinks" href="/pmc/articles/PMC2637482/" ref="ordinalpos=1955&amp;ncbi_uid=1746853&amp;link_uid=PMC2637482">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1956.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2833214/figure/F2/" ref="ordinalpos=1956&amp;ncbi_uid=2737391&amp;link_uid=PMC2833214" href="/pmc/articles/PMC2833214/"><img src="/pmc/articles/instance/2833214/bin/nihms157116f2.gif" src-large="/pmc/articles/instance/2833214/bin/nihms157116f2.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2833214/" ref="ordinalpos=1956&amp;ncbi_uid=2737391&amp;link_uid=PMC2833214" image-link="/pmc/articles/PMC2833214/figure/F2/" class="imagepopup">FIGURE 2. Myogenic <span class="highlight" style="background-color:">signaling</span> in pathogenesis of muscular dystrophy.  From: Therapeutic Targeting of <span class="highlight" style="background-color:">Signaling</span> Pathways in Muscular Dystrophy. </a></p><div class="supp"><p class="details"><div>A number of intrinsic and/or extrinsic stimuli lead to the activation of specific cell <span class="highlight" style="background-color:">signaling</span> pathways in skeletal muscle in muscular dystrophy. Depending on the type of muscular dystrophy, activation of a specific <span class="highlight" style="background-color:">pathway</span> may ameliorate or exacerbate disease progression. Red: exaggeration in pathogenesis; green: amelioration of pathogenesis; yellow: role not yet known. Cn, Calcineurin; DMD, Duchenne muscular dystrophy; EMD, Emery-Dreifuss muscular dystrophy; LGMD, Limb girdle muscular dystrophy.</div></p></div><div class="aux"><div class="resc">Shephali Bhatnagar, et al. J Mol Med (Berl). ;88(2):155-166.</div><p class="links"><a class="dblinks" href="/pubmed/19816663" ref="ordinalpos=1956&amp;ncbi_uid=2737391&amp;link_uid=19816663">Citation</a><a class="dblinks" href="/pmc/articles/PMC2833214/" ref="ordinalpos=1956&amp;ncbi_uid=2737391&amp;link_uid=PMC2833214">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1957.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2947224/figure/F1/" ref="ordinalpos=1957&amp;ncbi_uid=2501279&amp;link_uid=PMC2947224" href="/pmc/articles/PMC2947224/"><img src="/pmc/articles/instance/2947224/bin/JCI42442.f1.gif" src-large="/pmc/articles/instance/2947224/bin/JCI42442.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2947224/" ref="ordinalpos=1957&amp;ncbi_uid=2501279&amp;link_uid=PMC2947224" image-link="/pmc/articles/PMC2947224/figure/F1/" class="imagepopup">Figure 1. Effects on viability and <span class="highlight" style="background-color:">signaling</span> in MPN mutant cell lines treated with                   PU-H71..  From: HSP90 is a therapeutic target in JAK2-dependent myeloproliferative                neoplasms in mice and humans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Cells bearing mutations that result in constitutive activation of                   the JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (JAK2V617F [V617F] and MPLW515L [W515L]) have a                   lower IC<sub>50</sub> compared with that of Ba/F3 cells bearing BCR-Abl.                   Similarly, UKE-1 cells bearing JAK2V617F are more growth inhibited by PU-H71 than                   either KU812 (BCR-ABL) or THP-1 (JAK2 wild-type [JAK2WT]). (<b>B</b>)                   Western blots reveal a dose-dependent downmodulation of JAK2 and <span class="highlight" style="background-color:">signaling</span>                   intermediates in the JAK-STAT <span class="highlight" style="background-color:">pathway</span> after treatment with PU-H71 for 16 hours in                   Ba/F3 isogenic cell lines. </div></p></div><div class="aux"><div class="resc">Sachie Marubayashi, et al. J Clin Invest. 2010 October 1;120(10):3578-3593.</div><p class="links"><a class="dblinks" href="/pubmed/20852385" ref="ordinalpos=1957&amp;ncbi_uid=2501279&amp;link_uid=20852385">Citation</a><a class="dblinks" href="/pmc/articles/PMC2947224/" ref="ordinalpos=1957&amp;ncbi_uid=2501279&amp;link_uid=PMC2947224">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1958.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3861040/figure/pgen-1003975-g005/" ref="ordinalpos=1958&amp;ncbi_uid=5610226&amp;link_uid=PMC3861040" href="/pmc/articles/PMC3861040/"><img src="/pmc/articles/instance/3861040/bin/pgen.1003975.g005.gif" src-large="/pmc/articles/instance/3861040/bin/pgen.1003975.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3861040/" ref="ordinalpos=1958&amp;ncbi_uid=5610226&amp;link_uid=PMC3861040" image-link="/pmc/articles/PMC3861040/figure/pgen-1003975-g005/" class="imagepopup">Figure 5. Proposed model of FGF23-Klotho function in parathyroid glands..  From: Parathyroid-Specific Deletion of Klotho Unravels a Novel Calcineurin-Dependent FGF23 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> That Regulates PTH Secretion. </a></p><div class="supp"><p class="details"><div>FGF23 is an endocrine ligand that in the parathyroid glands either binds to an FGF-receptor and membrane-bound Klotho, to elicit activation of the MAPK-<span class="highlight" style="background-color:">pathway</span>, or acts via a calcineurin-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Both pathways induce a rapid inhibition of PTH secretion within 15 minutes. The MAPK-<span class="highlight" style="background-color:">pathway</span> is likely the dominant <span class="highlight" style="background-color:">pathway</span> in physiology, although their relative contributions are unknown. In addition to mediating MAPK-dependent FGF23 <span class="highlight" style="background-color:">signaling</span>, Klotho may play an intrinsic role in modulating parathyroid transcriptional activity for sustained demands and long-term parathyroid function. Speculatively, the clinical use of calcineurin inhibitors (CNi) that block calcineurin <span class="highlight" style="background-color:">signaling</span> may increase the susceptibility to develop, or aggravate pre-existing, hyperparathyroidism in patients with reduced Klotho level such as in chronic kidney disease and primary hyperparathyroidism.</div></p></div><div class="aux"><div class="resc">Hannes Olauson, et al. PLoS Genet. 2013 December;9(12):e1003975.</div><p class="links"><a class="dblinks" href="/pubmed/24348262" ref="ordinalpos=1958&amp;ncbi_uid=5610226&amp;link_uid=24348262">Citation</a><a class="dblinks" href="/pmc/articles/PMC3861040/" ref="ordinalpos=1958&amp;ncbi_uid=5610226&amp;link_uid=PMC3861040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1959.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137255/figure/cjc-30-01-013-g002/" ref="ordinalpos=1959&amp;ncbi_uid=6137743&amp;link_uid=PMC3137255" href="/pmc/articles/PMC3137255/"><img src="/pmc/articles/instance/4012260/bin/cjc-30-01-013-g002.gif" src-large="/pmc/articles/instance/4012260/bin/cjc-30-01-013-g002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137255/" ref="ordinalpos=1959&amp;ncbi_uid=6137743&amp;link_uid=PMC3137255" image-link="/pmc/articles/PMC3137255/figure/cjc-30-01-013-g002/" class="imagepopup">Figure 2.A simplified model for Hh <span class="highlight" style="background-color:">signaling</span> in mammalian cells. SMO is the key signal transducer of the Hh <span class="highlight" style="background-color:">pathway</span>. A, in the absence of the Hh ligands, the Hh receptor PTC is thought to be localized in the cilium to inhibit SMO <span class="highlight" style="background-color:">signaling</span> (via an unknown mechanism). Gli molecules are processed with the help of Su(Fu)/KIF7 molecules into repressor forms, which disable the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. B, in the presence of Hh, PTC is thought to be shuttled out of cilium and is unable to inhibit SMO. Co-receptors of Hh ligands include CDO, BOC, and GAS1. Hh reception promotes SMO conformational change, facilitating Gli activation (GMA), stimulating Hh target gene expression. This process can be inhibited by KIF7 and Su (Fu). (Positive regulators are in red, negative regulators are in blue and target genes are in orange.).  From: Clinical implications of hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors. </a></p><div class="aux"><div class="resc">Hailan Liu, et al. Chin J Cancer. 2011 January;30(1):13-26.</div><p class="links"><a class="dblinks" href="/pubmed/21192841" ref="ordinalpos=1959&amp;ncbi_uid=6137743&amp;link_uid=21192841">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137255/" ref="ordinalpos=1959&amp;ncbi_uid=6137743&amp;link_uid=PMC3137255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1960.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g014/" ref="ordinalpos=1960&amp;ncbi_uid=2242899&amp;link_uid=PMC2854141" href="/pmc/articles/PMC2854141/"><img src="/pmc/articles/instance/2854141/bin/pone.0010153.g014.gif" src-large="/pmc/articles/instance/2854141/bin/pone.0010153.g014.jpg" alt="Figure 14" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2854141/" ref="ordinalpos=1960&amp;ncbi_uid=2242899&amp;link_uid=PMC2854141" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g014/" class="imagepopup">Figure 14. Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Uncovering Molecular Biomarkers That Correlate Cognitive Decline with the Changes of Hippocampus' Gene Expression Profiles in Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>The upper graph presents the stacked normalized expression values of all the probes involved in the <span class="highlight" style="background-color:">pathway</span> with an upregulation trend. The lower graph analyses the genes involved in the <span class="highlight" style="background-color:">pathway</span> with a downregulation tendency. In the supplementary material (File S3 sheet âInsulin signallingâ), the reader will find all the individual gene expression values, normalised and not normalised.</div></p></div><div class="aux"><div class="resc">MartÃ­n GÃ³mez Ravetti, et al. PLoS One. 2010;5(4):e10153.</div><p class="links"><a class="dblinks" href="/pubmed/20405009" ref="ordinalpos=1960&amp;ncbi_uid=2242899&amp;link_uid=20405009">Citation</a><a class="dblinks" href="/pmc/articles/PMC2854141/" ref="ordinalpos=1960&amp;ncbi_uid=2242899&amp;link_uid=PMC2854141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1961.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2800199/figure/F1/" ref="ordinalpos=1961&amp;ncbi_uid=2146992&amp;link_uid=PMC2800199" href="/pmc/articles/PMC2800199/"><img src="/pmc/articles/instance/2800199/bin/nihms165903f1.gif" src-large="/pmc/articles/instance/2800199/bin/nihms165903f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2800199/" ref="ordinalpos=1961&amp;ncbi_uid=2146992&amp;link_uid=PMC2800199" image-link="/pmc/articles/PMC2800199/figure/F1/" class="imagepopup">Figure 1. Schematic depicting RRD-251 inhibition of Raf-1-Rb <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Disrupting the Rb-Raf-1 Interaction: A potential therapeutic target for cancer. </a></p><div class="supp"><p class="details"><div>Raf activation signals through MEK/ERK <span class="highlight" style="background-color:">pathway</span> and Rb <span class="highlight" style="background-color:">pathway</span>. Activation of ERK leads to induction of CDK4/cyclin D activity and Rb phosphorylation. Raf-1 also binds and phosphorylates Rb leading to dissociation of E2F, and expression of genes involved in cell cycle progression, angiogenesis and tumor growth. Treatment with RRD-251 disrupts RB-Raf-1 interaction and prevents Rb phosphorylation and inhibits E2F-mediated transcription of S-phase genes.</div></p></div><div class="aux"><div class="resc">Rebecca K. Davis, et al. Drug News Perspect. ;21(6):331-335.</div><p class="links"><a class="dblinks" href="/pubmed/18836591" ref="ordinalpos=1961&amp;ncbi_uid=2146992&amp;link_uid=18836591">Citation</a><a class="dblinks" href="/pmc/articles/PMC2800199/" ref="ordinalpos=1961&amp;ncbi_uid=2146992&amp;link_uid=PMC2800199">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1962.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3997007/figure/F1/" ref="ordinalpos=1962&amp;ncbi_uid=6100240&amp;link_uid=PMC3997007" href="/pmc/articles/PMC3997007/"><img src="/pmc/articles/instance/3997007/bin/fphys-05-00149-g0001.gif" src-large="/pmc/articles/instance/3997007/bin/fphys-05-00149-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3997007/" ref="ordinalpos=1962&amp;ncbi_uid=6100240&amp;link_uid=PMC3997007" image-link="/pmc/articles/PMC3997007/figure/F1/" class="imagepopup">Figure 1.  From: New roles for old enzymes: killer caspases as the engine of cell behavior changes. </a></p><div class="supp"><p class="details"><div><b>Cell-autonomous vs. non-cell-autonomous models of caspase <span class="highlight" style="background-color:">signaling</span>. (A)</b> Cell-autonomous model. In the proliferating cell, a non-apoptotic caspase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to a change in cell behavior through modulation of gene expression, cytoskeletal restructuring, or other means. This <span class="highlight" style="background-color:">signaling</span> is endogenous to the proliferating cell itself, and no apoptotic cell <span class="highlight" style="background-color:">signaling</span> is involved. <b>(B)</b> Non-cell-autonomous model. In this model, caspase activity within apoptotic cells lead to the generation of <span class="highlight" style="background-color:">signaling</span> factors which stimulate the cell behavior change of nearby cells in a paracrine fashion.</div></p></div><div class="aux"><div class="resc">Patrick F. Connolly, et al. Front Physiol. 2014;5:149.</div><p class="links"><a class="dblinks" href="/pubmed/24795644" ref="ordinalpos=1962&amp;ncbi_uid=6100240&amp;link_uid=24795644">Citation</a><a class="dblinks" href="/pmc/articles/PMC3997007/" ref="ordinalpos=1962&amp;ncbi_uid=6100240&amp;link_uid=PMC3997007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1963.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3296647/figure/F3/" ref="ordinalpos=1963&amp;ncbi_uid=3845283&amp;link_uid=PMC3296647" href="/pmc/articles/PMC3296647/"><img src="/pmc/articles/instance/3296647/bin/1471-2105-13-20-3.gif" src-large="/pmc/articles/instance/3296647/bin/1471-2105-13-20-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3296647/" ref="ordinalpos=1963&amp;ncbi_uid=3845283&amp;link_uid=PMC3296647" image-link="/pmc/articles/PMC3296647/figure/F3/" class="imagepopup">Figure 3.  From: graphite - a Bioconductor package to convert <span class="highlight" style="background-color:">pathway</span> topology to gene network. </a></p><div class="supp"><p class="details"><div><b>Differences in signal reconstruction of a selected portion of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of KEGG (hsa04910)</b>. Panel A. The original signal cascade. Panel B. <tt>graphite</tt> signal reconstruction through chemical compound propagation. Numbers represent EntrezGene IDs. Panel C. <tt>KEGGgraph</tt> signal reconstruction.</div></p></div><div class="aux"><div class="resc">Gabriele Sales, et al. BMC Bioinformatics. 2012;13:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/22292714" ref="ordinalpos=1963&amp;ncbi_uid=3845283&amp;link_uid=22292714">Citation</a><a class="dblinks" href="/pmc/articles/PMC3296647/" ref="ordinalpos=1963&amp;ncbi_uid=3845283&amp;link_uid=PMC3296647">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1964.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2908610/figure/F1/" ref="ordinalpos=1964&amp;ncbi_uid=2377080&amp;link_uid=PMC2908610" href="/pmc/articles/PMC2908610/"><img src="/pmc/articles/instance/2908610/bin/1475-2867-10-22-1.gif" src-large="/pmc/articles/instance/2908610/bin/1475-2867-10-22-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2908610/" ref="ordinalpos=1964&amp;ncbi_uid=2377080&amp;link_uid=PMC2908610" image-link="/pmc/articles/PMC2908610/figure/F1/" class="imagepopup">Figure 1.  From: Wnt signal transduction <span class="highlight" style="background-color:">pathway</span> and apoptosis: a review. </a></p><div class="supp"><p class="details"><div><b>Death receptor <span class="highlight" style="background-color:">pathway</span> <i>via </i>Fas ligand and Fas receptor</b>. DD - the specific "death domain"; DISC- death-inducing <span class="highlight" style="background-color:">signaling</span> complex (From: PeÄina-Å laus N. Acta Medica Croatica. Supplement., <b>63 </b>(Suppl. 2); 13-19, 2009.)</div></p></div><div class="aux"><div class="resc">Nives PeÄina-Å laus. Cancer Cell Int. 2010;10:22-22.</div><p class="links"><a class="dblinks" href="/pubmed/20591184" ref="ordinalpos=1964&amp;ncbi_uid=2377080&amp;link_uid=20591184">Citation</a><a class="dblinks" href="/pmc/articles/PMC2908610/" ref="ordinalpos=1964&amp;ncbi_uid=2377080&amp;link_uid=PMC2908610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1965.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4123883/figure/pone-0102689-g007/" ref="ordinalpos=1965&amp;ncbi_uid=6489984&amp;link_uid=PMC4123883" href="/pmc/articles/PMC4123883/"><img src="/pmc/articles/instance/4123883/bin/pone.0102689.g007.gif" src-large="/pmc/articles/instance/4123883/bin/pone.0102689.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4123883/" ref="ordinalpos=1965&amp;ncbi_uid=6489984&amp;link_uid=PMC4123883" image-link="/pmc/articles/PMC4123883/figure/pone-0102689-g007/" class="imagepopup">Figure 7. Modulation of the NF-kB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by MLB..  From: Anti-Wrinkle Effect of Magnesium Lithospermate B from Salvia miltiorrhiza BUNGE: Inhibition of MMPs via NF-kB <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) Western blotting was performed for p-ERK, p-p38, p-JNK, ERK, p-38, and JNK in the cytoplasmic extracts of aged rat skin. (B) Western blotting was performed for p-ERK, p-p38, p-JNK, ERK, p-38, and JNK in the cytoplasmic extracts of UVB-irradiated human skin fibroblasts.</div></p></div><div class="aux"><div class="resc">Yu Ri Jung, et al. PLoS One. 2014;9(8):e102689.</div><p class="links"><a class="dblinks" href="/pubmed/25099178" ref="ordinalpos=1965&amp;ncbi_uid=6489984&amp;link_uid=25099178">Citation</a><a class="dblinks" href="/pmc/articles/PMC4123883/" ref="ordinalpos=1965&amp;ncbi_uid=6489984&amp;link_uid=PMC4123883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1966.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g004/" ref="ordinalpos=1966&amp;ncbi_uid=4834033&amp;link_uid=PMC3633842" href="/pmc/articles/PMC3633842/"><img src="/pmc/articles/instance/3633842/bin/pone.0062845.g004.gif" src-large="/pmc/articles/instance/3633842/bin/pone.0062845.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633842/" ref="ordinalpos=1966&amp;ncbi_uid=4834033&amp;link_uid=PMC3633842" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g004/" class="imagepopup">Figure 4. Interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Decreased Pattern Recognition Receptor <span class="highlight" style="background-color:">Signaling</span>, Interferon-Signature, and Bactericidal/Permeability-Increasing Protein Gene Expression in Cord Blood of Term Low Birth Weight Human Newborns. </a></p><div class="supp"><p class="details"><div>The down-regulated ISGs in LBW newborns are shown in green color. The uncolored nodes are the genes inferred by IPA from its knowledge base.</div></p></div><div class="aux"><div class="resc">Vikas Vikram Singh, et al. PLoS One. 2013;8(4):e62845.</div><p class="links"><a class="dblinks" href="/pubmed/23626859" ref="ordinalpos=1966&amp;ncbi_uid=4834033&amp;link_uid=23626859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633842/" ref="ordinalpos=1966&amp;ncbi_uid=4834033&amp;link_uid=PMC3633842">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1967.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3615040/figure/pone-0060184-g002/" ref="ordinalpos=1967&amp;ncbi_uid=4782787&amp;link_uid=PMC3615040" href="/pmc/articles/PMC3615040/"><img src="/pmc/articles/instance/3615040/bin/pone.0060184.g002.gif" src-large="/pmc/articles/instance/3615040/bin/pone.0060184.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3615040/" ref="ordinalpos=1967&amp;ncbi_uid=4782787&amp;link_uid=PMC3615040" image-link="/pmc/articles/PMC3615040/figure/pone-0060184-g002/" class="imagepopup">Figure 2. Intracellular mTOR, AKT, and AMPK <span class="highlight" style="background-color:">signaling</span> and induction of autophagy are impacted by cytotoxic ATP. AâB).  From: P2X7 Integrates PI3K/AKT and AMPK-PRAS40-mTOR <span class="highlight" style="background-color:">Signaling</span> Pathways to Mediate Tumor Cell Death. </a></p><div class="supp"><p class="details"><div>Western blot analysis of <span class="highlight" style="background-color:">signaling</span> components of mTOR (A), AKT and AMPK (B), in MCA38 cells after ATP treatment at different times as indicated. Inhibition of mTOR and AKT with concurrent stimulation of AMPK <span class="highlight" style="background-color:">signaling</span> was noted. <b>CâD)</b> Delineation of ATP-elicited intracellular <span class="highlight" style="background-color:">signaling</span> cascades by blockade of certain component using specific inhibitors, determined by Western blotting. OA (okadaic acid): 100 nM; LY294 (LY294002): 50 ÂµM; RAPA (rapamycin): 100 nM; CC (compound C): 20 ÂµM. <b>E)</b> Impacts of <span class="highlight" style="background-color:">pathway</span> inhibitors on autophagy in MCA38 cells, as examined by Western blots of LC3-II. Î²-actin is the loading control. Data represent three to six experiments.</div></p></div><div class="aux"><div class="resc">Shu Bian, et al. PLoS One. 2013;8(4):e60184.</div><p class="links"><a class="dblinks" href="/pubmed/23565201" ref="ordinalpos=1967&amp;ncbi_uid=4782787&amp;link_uid=23565201">Citation</a><a class="dblinks" href="/pmc/articles/PMC3615040/" ref="ordinalpos=1967&amp;ncbi_uid=4782787&amp;link_uid=PMC3615040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1968.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3773501/figure/F1/" ref="ordinalpos=1968&amp;ncbi_uid=5307309&amp;link_uid=PMC3773501" href="/pmc/articles/PMC3773501/"><img src="/pmc/articles/instance/3773501/bin/nihms305731f1.gif" src-large="/pmc/articles/instance/3773501/bin/nihms305731f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3773501/" ref="ordinalpos=1968&amp;ncbi_uid=5307309&amp;link_uid=PMC3773501" image-link="/pmc/articles/PMC3773501/figure/F1/" class="imagepopup">Figure 1. NOD2-mediated <span class="highlight" style="background-color:">signaling</span> pathways.  From: NOD2, an Intracellular Innate Immune Sensor Involved in Host Defense and Crohn's Disease. </a></p><div class="supp"><p class="details"><div>This figure provides an overview of the major <span class="highlight" style="background-color:">signaling</span> pathways of NOD2. The central and perhaps dominant <span class="highlight" style="background-color:">pathway</span> is the RICK-dependent <span class="highlight" style="background-color:">pathway</span> that involves activation of RICK followed by downstream activation of TAK1; this then leads to activation of MAP kinases and NF-kB. As indicated in the text, interaction of NOD2 with CARD 9 or Bid may determine which downstream components are activated. On the right side of the figure a RICK-independent NOD2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> initiated by viral ssRNA is depicted; this involves binding of ssRNA to NOD2 followed by interaction with MAVS, an adaptor protein associated with mitochondria. This leads to activation of IRF3 and induction of type I interferon. On the left side of the figure interaction of NOD2 with the autophagy proteins in depicted. Such interaction can take the form of RICK-independent interaction with ATG16L1 and trafficking of the latter to the cell membrane, or RICK-dependent (and therefore NF-kB-dependent activation of autophagy proteins. Through this <span class="highlight" style="background-color:">pathway</span> NOD2 can affect bacterial handling and elimination or antigen presentation.</div></p></div><div class="aux"><div class="resc">Warren Strober, et al. Mucosal Immunol. 2011 September;4(5):484-495.</div><p class="links"><a class="dblinks" href="/pubmed/21750585" ref="ordinalpos=1968&amp;ncbi_uid=5307309&amp;link_uid=21750585">Citation</a><a class="dblinks" href="/pmc/articles/PMC3773501/" ref="ordinalpos=1968&amp;ncbi_uid=5307309&amp;link_uid=PMC3773501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1969.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2773007/figure/pone-0007841-g008/" ref="ordinalpos=1969&amp;ncbi_uid=2070185&amp;link_uid=PMC2773007" href="/pmc/articles/PMC2773007/"><img src="/pmc/articles/instance/2773007/bin/pone.0007841.g008.gif" src-large="/pmc/articles/instance/2773007/bin/pone.0007841.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2773007/" ref="ordinalpos=1969&amp;ncbi_uid=2070185&amp;link_uid=PMC2773007" image-link="/pmc/articles/PMC2773007/figure/pone-0007841-g008/" class="imagepopup">Figure 8. Grb2 mediates synergy between wnt, integrin and growth factor <span class="highlight" style="background-color:">signaling</span>..  From: Integration of the ?-Catenin-Dependent Wnt <span class="highlight" style="background-color:">Pathway</span> with Integrin <span class="highlight" style="background-color:">Signaling</span> through the Adaptor Molecule Grb2. </a></p><div class="supp"><p class="details"><div>Grb2 coordinates <span class="highlight" style="background-color:">signaling</span> downstream of integrin/FAK and growth factor receptors to activate rac and jnk. Grb2 also interacts directly with Dvl2 downstream of Wnt/Fz. Pathways converge at the level of transcriptional activation through LEF/TCF.</div></p></div><div class="aux"><div class="resc">Steve P. Crampton, et al. PLoS One. 2009;4(11):e7841.</div><p class="links"><a class="dblinks" href="/pubmed/19924227" ref="ordinalpos=1969&amp;ncbi_uid=2070185&amp;link_uid=19924227">Citation</a><a class="dblinks" href="/pmc/articles/PMC2773007/" ref="ordinalpos=1969&amp;ncbi_uid=2070185&amp;link_uid=PMC2773007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1970.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3965409/figure/pone-0092398-g006/" ref="ordinalpos=1970&amp;ncbi_uid=5949197&amp;link_uid=PMC3965409" href="/pmc/articles/PMC3965409/"><img src="/pmc/articles/instance/3965409/bin/pone.0092398.g006.gif" src-large="/pmc/articles/instance/3965409/bin/pone.0092398.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3965409/" ref="ordinalpos=1970&amp;ncbi_uid=5949197&amp;link_uid=PMC3965409" image-link="/pmc/articles/PMC3965409/figure/pone-0092398-g006/" class="imagepopup">Figure 6. LPS stimulates VSMCs proliferation via TLR4/Rac1/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and there is positive feedback..  From: Positive Feedback Regulation of Proliferation in Vascular Smooth Muscle Cells Stimulated by Lipopolysaccharide Is Mediated through the TLR 4/Rac1/Akt <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(The dotted line means TLR4 may activate other molecules such as IRAK, MYD88, and then activating PI3K).</div></p></div><div class="aux"><div class="resc">Dehua Jiang, et al. PLoS One. 2014;9(3):e92398.</div><p class="links"><a class="dblinks" href="/pubmed/24667766" ref="ordinalpos=1970&amp;ncbi_uid=5949197&amp;link_uid=24667766">Citation</a><a class="dblinks" href="/pmc/articles/PMC3965409/" ref="ordinalpos=1970&amp;ncbi_uid=5949197&amp;link_uid=PMC3965409">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1971.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3980473/figure/F6/" ref="ordinalpos=1971&amp;ncbi_uid=6008576&amp;link_uid=PMC3980473" href="/pmc/articles/PMC3980473/"><img src="/pmc/articles/instance/3980473/bin/nihms481367f6.gif" src-large="/pmc/articles/instance/3980473/bin/nihms481367f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3980473/" ref="ordinalpos=1971&amp;ncbi_uid=6008576&amp;link_uid=PMC3980473" image-link="/pmc/articles/PMC3980473/figure/F6/" class="imagepopup">Figure 6.  From: Type 5 Adenylyl Cyclase Increases Oxidative Stress by Transcriptional Regulation of MnSOD via the SIRT1/FoxO3a <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> diagram for AC5 regulation of MnSOD through the SIRT1/FoxO3a and the MEK/ERK <span class="highlight" style="background-color:">pathway</span>. Imbalance between reactive oxygen species production and the intracellular anti-oxidant system results in the intolerance of AC5 Tg to stress.</div></p></div><div class="aux"><div class="resc">Lo Lai, et al. Circulation. 2013 April 23;127(16):1692-1701.</div><p class="links"><a class="dblinks" href="/pubmed/23536361" ref="ordinalpos=1971&amp;ncbi_uid=6008576&amp;link_uid=23536361">Citation</a><a class="dblinks" href="/pmc/articles/PMC3980473/" ref="ordinalpos=1971&amp;ncbi_uid=6008576&amp;link_uid=PMC3980473">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1972.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3010279/figure/F8/" ref="ordinalpos=1972&amp;ncbi_uid=2867876&amp;link_uid=PMC3010279" href="/pmc/articles/PMC3010279/"><img src="/pmc/articles/instance/3010279/bin/nihms241833f8.gif" src-large="/pmc/articles/instance/3010279/bin/nihms241833f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3010279/" ref="ordinalpos=1972&amp;ncbi_uid=2867876&amp;link_uid=PMC3010279" image-link="/pmc/articles/PMC3010279/figure/F8/" class="imagepopup">Figure 8. Proposed <span class="highlight" style="background-color:">pathway</span> for morphine-induced enhancement of GIRK <span class="highlight" style="background-color:">signaling</span> in hippocampal neurons.  From: Morphine- and CaMKII dependent enhancement of GIRK channel <span class="highlight" style="background-color:">signaling</span> in hippocampal neurons. </a></p><div class="supp"><p class="details"><div><b>A</b>) Inhibition of intracellular Ca<sup>2+</sup> with 50 Âµm BAPTA-AM prevents the morphine-induced colocalization of GIRK2 and PSD95 in 20 DIV hippocampal neurons (left graph). Exposure to the metabotropic glutamate receptor agonist ACPD (100 Âµm) for 20 h increases the colocalization ratio for GIRK2 and PSD95 in 20 DIV hippocampal neurons (right graph). <b>B</b>) Schematic shows proposed <span class="highlight" style="background-color:">signaling</span> pathways for morphine and ACPD-dependent upregulation of GIRK2 in hippocampal neurons.</div></p></div><div class="aux"><div class="resc">Rounak Nassirpour, et al. J Neurosci. ;30(40):13419-13430.</div><p class="links"><a class="dblinks" href="/pubmed/20926668" ref="ordinalpos=1972&amp;ncbi_uid=2867876&amp;link_uid=20926668">Citation</a><a class="dblinks" href="/pmc/articles/PMC3010279/" ref="ordinalpos=1972&amp;ncbi_uid=2867876&amp;link_uid=PMC3010279">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1973.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355838/figure/F2/" ref="ordinalpos=1973&amp;ncbi_uid=4027622&amp;link_uid=PMC3355838" href="/pmc/articles/PMC3355838/"><img src="/pmc/articles/instance/3355838/bin/fendo-03-00031-g002.gif" src-large="/pmc/articles/instance/3355838/bin/fendo-03-00031-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355838/" ref="ordinalpos=1973&amp;ncbi_uid=4027622&amp;link_uid=PMC3355838" image-link="/pmc/articles/PMC3355838/figure/F2/" class="imagepopup">Figure 2.  From: Role of the Wnt <span class="highlight" style="background-color:">Pathway</span> in Thyroid Cancer. </a></p><div class="supp"><p class="details"><div><b>Genetic events involved in thyroid tumor initiation and progression</b>. Thyroid cell proliferation depends on the combined activation of cAMP/PKA, MAPK, and PI3K pathways, induced by TSH and other growth factors. Mutations of effectors along these <span class="highlight" style="background-color:">signaling</span> pathways play prominent roles in the pathogenesis of thyroid neoplasia. The figure represents a thyroid epithelial cell with the basal and apical membrane, and shows the tight and adherens junctions (TJ and AJ) between cells in order to form follicular structures. Activating mutations in <i>TSHR</i> or <i>G</i>Î±<i>s</i> lead to constitutive activation of the cAMP/PKA <span class="highlight" style="background-color:">pathway</span> and give rise to hyperfunctional adenomas. There is evidence that constitutive activation of the MAPK <span class="highlight" style="background-color:">pathway</span> is required for PTC initiation. Activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is required for FTC initiation. Finally, deregulation of the p53 <span class="highlight" style="background-color:">pathway</span> and the Wnt/Î²-catenin <span class="highlight" style="background-color:">pathway</span> is observed in ATC and has more to do with tumor progression.</div></p></div><div class="aux"><div class="resc">Ana Sastre-Perona, et al. Front Endocrinol (Lausanne). 2012;3:31.</div><p class="links"><a class="dblinks" href="/pubmed/22645520" ref="ordinalpos=1973&amp;ncbi_uid=4027622&amp;link_uid=22645520">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355838/" ref="ordinalpos=1973&amp;ncbi_uid=4027622&amp;link_uid=PMC3355838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1974.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1160519/figure/fig6/" ref="ordinalpos=1974&amp;ncbi_uid=531786&amp;link_uid=PMC1160519" href="/pmc/articles/PMC1160519/"><img src="/pmc/articles/instance/1172275/bin/zpq0270587990006.gif" src-large="/pmc/articles/instance/1172275/bin/zpq0270587990006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1160519/" ref="ordinalpos=1974&amp;ncbi_uid=531786&amp;link_uid=PMC1160519" image-link="/pmc/articles/PMC1160519/figure/fig6/" class="imagepopup">Fig. 6. From: Cardiac glycosides inhibit TNF-?/NF-?B <span class="highlight" style="background-color:">signaling</span> by blocking recruitment of TNF receptor-associated death domain to the TNF receptor. </a></p><div class="supp"><p class="details"><div>Cardiac glycosides inhibit TNF-Î±/NF-ÎºB <span class="highlight" style="background-color:">signaling</span> by blocking recruitment of TRADD to the TNFR. The binding of TNF-Î± to TNFR1 initiates the TNF signal transduction <span class="highlight" style="background-color:">pathway</span>. TNFR1 recruits TRADD to form a complex with TNFR1. A series of additional adapter molecules then assemble sequentially on this platform, including TRAF2, RIP, and Fas-associated death domain (FADD), where they initiate downstream events leading to NF-Îº B activation, JNK activation, and inflammation or apoptosis. Cardiac glycosides inhibit the TNF-Î± <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by blocking recruitment of TRADD to the TNFR.</div></p></div><div class="aux"><div class="resc">Qingfeng Yang, et al. Proc Natl Acad Sci U S A. 2005 July 5;102(27):9631-9636.</div><p class="links"><a class="dblinks" href="/pubmed/15983368" ref="ordinalpos=1974&amp;ncbi_uid=531786&amp;link_uid=15983368">Citation</a><a class="dblinks" href="/pmc/articles/PMC1160519/" ref="ordinalpos=1974&amp;ncbi_uid=531786&amp;link_uid=PMC1160519">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1975.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399217/figure/F3/" ref="ordinalpos=1975&amp;ncbi_uid=4125431&amp;link_uid=PMC3399217" href="/pmc/articles/PMC3399217/"><img src="/pmc/articles/instance/3399217/bin/ijmsv09p0370g03.gif" src-large="/pmc/articles/instance/3399217/bin/ijmsv09p0370g03.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399217/" ref="ordinalpos=1975&amp;ncbi_uid=4125431&amp;link_uid=PMC3399217" image-link="/pmc/articles/PMC3399217/figure/F3/" class="imagepopup">Figure 3.  From: 18?-Glycyrrhetinic Acid Down-Regulates Expression of Type I and III Collagen via TGF-?1/Smad <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human and Rat Hepatic Stellate Cells. </a></p><div class="supp"><p class="details"><div>Effect of 18Î±-GA on the expression of TGF-Î²1/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related proteins in human and rat HSCs. The protein expression of Smad3, type I and III collagen was significantly decreased, while that of Smad7 dramatically increased at 24 h (<i>P</i>&lt;0.05) and 48 h (<i>P</i>&lt;0.01) after 18Î±-GA treatment in both cell lines. *<i> P</i> &lt;0.05, **<i> P</i> &lt;0.01 vs control group.</div></p></div><div class="aux"><div class="resc">Lei Zong, et al. Int J Med Sci. 2012;9(5):370-379.</div><p class="links"><a class="dblinks" href="/pubmed/22811611" ref="ordinalpos=1975&amp;ncbi_uid=4125431&amp;link_uid=22811611">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399217/" ref="ordinalpos=1975&amp;ncbi_uid=4125431&amp;link_uid=PMC3399217">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1976.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2758485/figure/f1/" ref="ordinalpos=1976&amp;ncbi_uid=3141678&amp;link_uid=PMC2758485" href="/pmc/articles/PMC2758485/"><img src="/pmc/articles/instance/3140861/bin/fig-1.gif" src-large="/pmc/articles/instance/3140861/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758485/" ref="ordinalpos=1976&amp;ncbi_uid=3141678&amp;link_uid=PMC2758485" image-link="/pmc/articles/PMC2758485/figure/f1/" class="imagepopup">FIG. 1. From: The Emerging Role of Wnt/PCP <span class="highlight" style="background-color:">Signaling</span> in Organ Formation. </a></p><div class="supp"><p class="details"><div>Schematic of Wnt/PCP <span class="highlight" style="background-color:">signaling</span> and its effects on organogenesis. Diagramed are the genetic and physical interactions of the Wnt/PCP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components discussed in this review and shown to be essential for organ formation and proper function of the primary cilia. Based on the role of Wnt/PCP <span class="highlight" style="background-color:">signaling</span> in zebrafish and other animal models, we surmise that the actin cytoskeleton may be a main target of this <span class="highlight" style="background-color:">pathway</span> during organogenesis. Hair cell polarity within the lateral line neuromasts is dependent on the <i>vangl2.</i> However, loss of <i>glypican 4, wnt5b,</i> or <i>wnt11</i> does not perturb neuromast organization. In contrast, while <i>glypican 4</i> and <i>wnt5b</i> mutant embryos exhibit irregular chondrocyte stacking, the organization of <i>vangl2</i> mutant cartilages appears to be normal. Recently, Wnt/PCP <span class="highlight" style="background-color:">signaling</span> has been shown to interact with the components of the primary cilia. Proteins involved in the Wnt/PCP <span class="highlight" style="background-color:">pathway</span>, such as Dishevelled and Vangl2, can co-localize with the cilia and may regulate its localization and function. Primary cilia may be also involved with the down regulation of Wnt/<i>Î²</i>-catenin <span class="highlight" style="background-color:">signaling</span>,<sup>67</sup> similar to Wnt/PCP. Overexpression of the Wnt/PCP <span class="highlight" style="background-color:">pathway</span> ligands appears to inhibit Wnt/<i>Î²</i>-catenin <span class="highlight" style="background-color:">signaling</span> in fin regeneration,<sup>42</sup> but the precise mechanisms of this interaction were not explored. The neuromast hair cell schematic was adapted from Lopez-Schier <i>et al.</i>,<sup>28,29</sup> and the chondrocyte stacking schematic was adapted from Topczewski <i>et al.</i><sup>13</sup> Dub, Duboraya; Sea, Seahorse; Pk, Prickle; Vangl2, Van Goghâlike 2; Bbs4, BardetâBiedl syndrome 4; Ofd1, oral-facial-digital syndrome 1.</div></p></div><div class="aux"><div class="resc">Rodney M. Dale, et al. Zebrafish. 2009 March;6(1):9-14.</div><p class="links"><a class="dblinks" href="/pubmed/19250029" ref="ordinalpos=1976&amp;ncbi_uid=3141678&amp;link_uid=19250029">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758485/" ref="ordinalpos=1976&amp;ncbi_uid=3141678&amp;link_uid=PMC2758485">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1977.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2923715/figure/Fig5/" ref="ordinalpos=1977&amp;ncbi_uid=2459483&amp;link_uid=PMC2923715" href="/pmc/articles/PMC2923715/"><img src="/pmc/articles/instance/2923715/bin/467_2009_1416_Fig5_HTML.gif" src-large="/pmc/articles/instance/2923715/bin/467_2009_1416_Fig5_HTML.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2923715/" ref="ordinalpos=1977&amp;ncbi_uid=2459483&amp;link_uid=PMC2923715" image-link="/pmc/articles/PMC2923715/figure/Fig5/" class="imagepopup">Fig. 5.  From: Glomerulocystic kidney disease. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> pathways. The canonical <span class="highlight" style="background-color:">pathway</span> is mediated through the cystoplasmic protein disheveled (<i>Dvl</i>) and ultimately effected through Î²-catenin-mediated transcription. The non-canonical <span class="highlight" style="background-color:">pathway</span> is mediated through membrane-bound Dvl and ultimately controls planar cell polarity. Inversin, the <i>NPHP2</i> gene product, in concert with the NPHP3 protein, appears to facilitate development by controlling the Wnt <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. <i>RhoA</i> Ras homolog gene family, member A</div></p></div><div class="aux"><div class="resc">John J. Bissler, et al. Pediatr Nephrol. 2010 October;25(10):2049-2059.</div><p class="links"><a class="dblinks" href="/pubmed/20091054" ref="ordinalpos=1977&amp;ncbi_uid=2459483&amp;link_uid=20091054">Citation</a><a class="dblinks" href="/pmc/articles/PMC2923715/" ref="ordinalpos=1977&amp;ncbi_uid=2459483&amp;link_uid=PMC2923715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1978.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4122365/figure/pone-0103292-g001/" ref="ordinalpos=1978&amp;ncbi_uid=6488461&amp;link_uid=PMC4122365" href="/pmc/articles/PMC4122365/"><img src="/pmc/articles/instance/4122365/bin/pone.0103292.g001.gif" src-large="/pmc/articles/instance/4122365/bin/pone.0103292.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4122365/" ref="ordinalpos=1978&amp;ncbi_uid=6488461&amp;link_uid=PMC4122365" image-link="/pmc/articles/PMC4122365/figure/pone-0103292-g001/" class="imagepopup">Figure 1. Polymorphisms identified in flies adapted to 4% O2 hypoxia..  From: Wnt <span class="highlight" style="background-color:">Pathway</span> Activation Increases Hypoxia Tolerance during Development. </a></p><div class="supp"><p class="details"><div>(A,B) Chromosome and gene region distributions of SNPs (A) or indels (B) were normalized to the size of (euchromatin) X chromosome or number of total intron loci, respectively. The legends specify the actual number of polymorphisms detected and polymorphisms/kB (euchromatin) chromosome or gene region. Not included are 14 SNPs that map to XHet, YHet or U/Uextra and 2 indels that map to U and Chromosome 4. (C) DAVID [11], [12]-identified KEGG [14], [15] and Panther [16] <span class="highlight" style="background-color:">pathway</span> enrichment of polymorphisms. The number of <span class="highlight" style="background-color:">pathway</span> genes with â¥1 polymorphism is plotted and the total number of <span class="highlight" style="background-color:">pathway</span> genes is indicated below each <span class="highlight" style="background-color:">pathway</span> name (TCAâ=âCitrate, NKâ=âNK cell mediated cytotoxicity, Circadâ=âCircadian, D-V axisâ=âDorso-ventral axis formation, Cadhâ=âCadherin, Axonâ=âAxon guidance (semaphorins), Gi/Gsâ=âGi/Gs mediated <span class="highlight" style="background-color:">signaling</span>, Gq/Goâ=âGq/Go mediated <span class="highlight" style="background-color:">signaling</span>). (D) The Wnt <span class="highlight" style="background-color:">pathway</span> was adapted from KEGG [14], [15] with additional interactors added. Genes with one or more fixed polymorphisms are indicated with a red star. Genes differentially expressed in post-eclosion AF maintained in 4% O<sub>2</sub> are shaded in cyan (up-regulated) or yellow (down-regulated). Wnt <span class="highlight" style="background-color:">pathway</span> genes tested in genetic crosses are diamond-shaped; <span class="highlight" style="background-color:">pathway</span> activators are indicated by a red border and <span class="highlight" style="background-color:">pathway</span> inhibitors by a blue border.</div></p></div><div class="aux"><div class="resc">Merril Gersten, et al. PLoS One. 2014;9(8):e103292.</div><p class="links"><a class="dblinks" href="/pubmed/25093834" ref="ordinalpos=1978&amp;ncbi_uid=6488461&amp;link_uid=25093834">Citation</a><a class="dblinks" href="/pmc/articles/PMC4122365/" ref="ordinalpos=1978&amp;ncbi_uid=6488461&amp;link_uid=PMC4122365">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1979.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3021119/figure/F6/" ref="ordinalpos=1979&amp;ncbi_uid=3618290&amp;link_uid=PMC3021119" href="/pmc/articles/PMC3021119/"><img src="/pmc/articles/instance/3021119/bin/nihms-256143-f0006.gif" src-large="/pmc/articles/instance/3021119/bin/nihms-256143-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3021119/" ref="ordinalpos=1979&amp;ncbi_uid=3618290&amp;link_uid=PMC3021119" image-link="/pmc/articles/PMC3021119/figure/F6/" class="imagepopup">Figure 6. Jak/Stat-induced ISC proliferation requires EGFR <span class="highlight" style="background-color:">signaling</span>.  From: EGFR/Ras/MAPK <span class="highlight" style="background-color:">signaling</span> mediates adult midgut epithelial homeostasis and regeneration in Drosophila. </a></p><div class="supp"><p class="details"><div><b>A.</b> ISC proliferation induced by EGFR and Jak/Stat <span class="highlight" style="background-color:">signaling</span>. With the exception of co-expressing <i>sKrn</i> and <i>Upd</i> in the ECs (<i>MyoIA<sup>ts</sup></i>&gt;<i>Upd</i> + <i>sKrn</i>), all the other ectopic expression experiments were performed using the <i>esg<sup>ts</sup>F/O</i> driver. Midgut mitotic indices (PH3<sup>+</sup>) were quantified after activating the transgenes for 2 days. <b>B-J</b>. ISC clonal assay. GFP-marked ISC clones were induced using the MARCM system and analyzed 4 or 8 days later. The sizes of the ISC clones were indicated. <i>Vn</i>-induced ISC proliferation is dependent on Jak/Stat <span class="highlight" style="background-color:">signaling</span> (B-D). Activated Ras (<i>Ras<sup>V12</sup></i>)-induced ISC proliferation is independent of Jak/Stat <span class="highlight" style="background-color:">signaling</span> (F-G). Some EB clones overexpressing <i>Ras<sup>V12</sup></i> underwent extra round of endoreplication (E). <i>Upd</i>-induced ISC proliferation is dependent on EGFR <span class="highlight" style="background-color:">signaling</span> (H-J). <b>K.</b> Quantification of ISC clone sizes. The sizes of ISC clones were measured 4 or 8 days after clone induction (ACI) using the MARCM system. <b>L.</b> RT-qPCR analysis of the induction of Jak/Stat and EGFR signalings by <i>Pe</i> infection in the absence of either <span class="highlight" style="background-color:">pathway</span> (<i>esg<sup>ts</sup>F/O</i>&gt;<i>Stat</i> or <i>Egfr RNAi</i>).</div></p></div><div class="aux"><div class="resc">Huaqi Jiang, et al. Cell Stem Cell. ;8(1):84-95.</div><p class="links"><a class="dblinks" href="/pubmed/21167805" ref="ordinalpos=1979&amp;ncbi_uid=3618290&amp;link_uid=21167805">Citation</a><a class="dblinks" href="/pmc/articles/PMC3021119/" ref="ordinalpos=1979&amp;ncbi_uid=3618290&amp;link_uid=PMC3021119">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1980.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3789823/figure/pgen-1003845-g006/" ref="ordinalpos=1980&amp;ncbi_uid=5369776&amp;link_uid=PMC3789823" href="/pmc/articles/PMC3789823/"><img src="/pmc/articles/instance/3789823/bin/pgen.1003845.g006.gif" src-large="/pmc/articles/instance/3789823/bin/pgen.1003845.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3789823/" ref="ordinalpos=1980&amp;ncbi_uid=5369776&amp;link_uid=PMC3789823" image-link="/pmc/articles/PMC3789823/figure/pgen-1003845-g006/" class="imagepopup">Figure 6. Characterization of PI3K/mTOR pathways in TH-positive DA neurons of Atg7 and/or Pten cKO mice..  From: Coordinate Regulation of Mature Dopaminergic Axon Morphology by Macroautophagy and the PTEN <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>PI3K/mTOR <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>, in terms of accumulation of phospho-AKT (S473), phospho-TOR (S2448), and phospho-S6 (S235/236) as indicated (in red) were unchanged in the TH-positive (green) midbrain DA neurons of <i>Atg7</i> cKO mice (C), whereas these markers were increased in TH-positive DA neurons of <i>Pten</i> cKO and <i>Atg7/Pten</i> double cKO mice (arrows in B and D). (<b>A</b>) Control cWT mice, (<b>B</b>) <i>Pten</i> cKO mice, (<b>C</b>) <i>Atg7</i> cKO mice, and (<b>D</b>) <i>Atg7/Pten</i> double cKO mice. Scale bars, 10 Âµm.</div></p></div><div class="aux"><div class="resc">Keiichi Inoue, et al. PLoS Genet. 2013 October;9(10):e1003845.</div><p class="links"><a class="dblinks" href="/pubmed/24098148" ref="ordinalpos=1980&amp;ncbi_uid=5369776&amp;link_uid=24098148">Citation</a><a class="dblinks" href="/pmc/articles/PMC3789823/" ref="ordinalpos=1980&amp;ncbi_uid=5369776&amp;link_uid=PMC3789823">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1981.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3057579/figure/fig9/" ref="ordinalpos=1981&amp;ncbi_uid=2838868&amp;link_uid=PMC3057579" href="/pmc/articles/PMC3057579/"><img src="/pmc/articles/instance/3057579/bin/ECAM2011-750230.009.gif" src-large="/pmc/articles/instance/3057579/bin/ECAM2011-750230.009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3057579/" ref="ordinalpos=1981&amp;ncbi_uid=2838868&amp;link_uid=PMC3057579" image-link="/pmc/articles/PMC3057579/figure/fig9/" class="imagepopup">Figure 9.  From: Apoptotic Cell Death and Inhibition of Wnt/Î²-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human Colon Cancer Cells by an Active Fraction (HS7) from Taiwanofungus camphoratus. </a></p><div class="supp"><p class="details"><div>Hypothetical diaphragm of HS7 induced apoptosis and inhibition of Wnt/<i>Î²</i>-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HT-29 colon cancer cells. The arrow indicates promotion, and the T-shaped bar indicates repression. Dsh: dishevelled protein, GSK-3<i>Î²</i>: glycogen synthase kinase-3 beta, APC: adenomatous polyposis coli protein, Tcf/Lef: T-cell factor/lymphocyte enhancer factor family, PARP: poly (ADP-ribose) polymerase.</div></p></div><div class="aux"><div class="resc">Chi-Tai Yeh, et al. Evid Based Complement Alternat Med. 2011;2011:750230.</div><p class="links"><a class="dblinks" href="/pubmed/21423639" ref="ordinalpos=1981&amp;ncbi_uid=2838868&amp;link_uid=21423639">Citation</a><a class="dblinks" href="/pmc/articles/PMC3057579/" ref="ordinalpos=1981&amp;ncbi_uid=2838868&amp;link_uid=PMC3057579">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1982.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2267821/figure/pbio-0060061-g011/" ref="ordinalpos=1982&amp;ncbi_uid=1347633&amp;link_uid=PMC2267821" href="/pmc/articles/PMC2267821/"><img src="/pmc/articles/instance/2270317/bin/pbio.0060061.g011.gif" src-large="/pmc/articles/instance/2270317/bin/pbio.0060061.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2267821/" ref="ordinalpos=1982&amp;ncbi_uid=1347633&amp;link_uid=PMC2267821" image-link="/pmc/articles/PMC2267821/figure/pbio-0060061-g011/" class="imagepopup">Figure 11. <span class="highlight" style="background-color:">Signaling</span> Pathways That Control the Degradation of Apoptotic Cells.  From: Phagocytic Receptor CED-1 Initiates a <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> for Degrading Engulfed Apoptotic Cells. </a></p><div class="supp"><p class="details"><div>(A) Diagram summarizing the sequential molecular events that occur during the maturation of phagosomes containing apoptotic cells in C. elegans.<br />(B) Diagram of the <span class="highlight" style="background-color:">signaling</span> pathways that promote the formation of phagolysosome.<br />(C) A summary of the functions of C. elegans RAB-7.</div></p></div><div class="aux"><div class="resc">Xiaomeng Yu, et al. PLoS Biol. 2008 March;6(3):e61.</div><p class="links"><a class="dblinks" href="/pubmed/18351800" ref="ordinalpos=1982&amp;ncbi_uid=1347633&amp;link_uid=18351800">Citation</a><a class="dblinks" href="/pmc/articles/PMC2267821/" ref="ordinalpos=1982&amp;ncbi_uid=1347633&amp;link_uid=PMC2267821">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1983.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3343650/figure/fig5/" ref="ordinalpos=1983&amp;ncbi_uid=4852938&amp;link_uid=PMC3343650" href="/pmc/articles/PMC3343650/"><img src="/pmc/articles/instance/3343650/bin/cr201230f5.gif" src-large="/pmc/articles/instance/3343650/bin/cr201230f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3343650/" ref="ordinalpos=1983&amp;ncbi_uid=4852938&amp;link_uid=PMC3343650" image-link="/pmc/articles/PMC3343650/figure/fig5/" class="imagepopup">Figure 5.  From: RNA-seq analysis of prostate cancer in the Chinese population identifies recurrent gene fusions, cancer-associated long noncoding RNAs and aberrant alternative splicings. </a></p><div class="supp"><p class="details"><div>Three major <span class="highlight" style="background-color:">signaling</span> pathways are altered in prostate cancers. <b>(A)</b> Genes altered in the RAS-PI3K-AKT <span class="highlight" style="background-color:">pathway</span>. <b>(B)</b> Genes altered in the AR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(C)</b> Genes altered in the RB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The activated genes are colored in a red gradient, and the inactivated genes are colored in a blue gradient according to the percentage of alterations in the 14 prostate cancer samples. The darker the color, the greater the percentage.</div></p></div><div class="aux"><div class="resc">Shancheng Ren, et al. Cell Res. 2012 May;22(5):806-821.</div><p class="links"><a class="dblinks" href="/pubmed/22349460" ref="ordinalpos=1983&amp;ncbi_uid=4852938&amp;link_uid=22349460">Citation</a><a class="dblinks" href="/pmc/articles/PMC3343650/" ref="ordinalpos=1983&amp;ncbi_uid=4852938&amp;link_uid=PMC3343650">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1984.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3278757/figure/F7/" ref="ordinalpos=1984&amp;ncbi_uid=4610599&amp;link_uid=PMC3278757" href="/pmc/articles/PMC3278757/"><img src="/pmc/articles/instance/3278757/bin/btr696f7.gif" src-large="/pmc/articles/instance/3278757/bin/btr696f7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3278757/" ref="ordinalpos=1984&amp;ncbi_uid=4610599&amp;link_uid=PMC3278757" image-link="/pmc/articles/PMC3278757/figure/F7/" class="imagepopup">Fig. 7. From: Optimal structural inference of <span class="highlight" style="background-color:">signaling</span> pathways from unordered and overlapping gene sets. </a></p><div class="supp"><p class="details"><div>An example showcasing the performance of SA in recovering the true structure using the IFGS compendium derived from GnRH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in KEGG database. Structures represent true (<b>A</b>) and inferred <span class="highlight" style="background-color:">signaling</span> pathways (<b>B</b>), respectively. The black (solid) and blue (dashed) edges represent true positives and false positives, respectively. Figures were generated using Cytoscape (Shannon et al., 2003).</div></p></div><div class="aux"><div class="resc">Lipi R. Acharya, et al. Bioinformatics. 2012 February 15;28(4):546-556.</div><p class="links"><a class="dblinks" href="/pubmed/22199386" ref="ordinalpos=1984&amp;ncbi_uid=4610599&amp;link_uid=22199386">Citation</a><a class="dblinks" href="/pmc/articles/PMC3278757/" ref="ordinalpos=1984&amp;ncbi_uid=4610599&amp;link_uid=PMC3278757">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1985.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1855405/figure/F3/" ref="ordinalpos=1985&amp;ncbi_uid=963553&amp;link_uid=PMC1855405" href="/pmc/articles/PMC1855405/"><img src="/pmc/articles/instance/1855405/bin/zpq0150759240003.gif" src-large="/pmc/articles/instance/1855405/bin/zpq0150759240003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1855405/" ref="ordinalpos=1985&amp;ncbi_uid=963553&amp;link_uid=PMC1855405" image-link="/pmc/articles/PMC1855405/figure/F3/" class="imagepopup">Fig. 3. From: Chimaerin and Rac regulate cell number, adherens junctions, and ERK MAP kinase <span class="highlight" style="background-color:">signaling</span> in the Drosophila eye. </a></p><div class="supp"><p class="details"><div>RhoGAP5A interacts with the dEGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the fly eye. All flies depicted express the <i>GMR-GAL4</i> driver. (<i>A</i>) <i>dEGFR<sup>EL-B1</sup></i> causes a rough eye phenotype, which is suppressed by expression of <i>RhoGAP5A-dsRNAi</i> (<i>D</i>). (<i>B</i>) <i>hs-Argos</i> flies have a mildly rough eye because of Argos inhibition of dEGFR <span class="highlight" style="background-color:">signaling</span>. (<i>E</i>) Coexpression of <i>RhoGAP5A-dsRNAi</i> rescues the rough eye phenotype of <i>hs-Argos</i>. (<i>C</i>) A gain-of-function mutation in <i>Drosophila</i> ERK, <i>rl<sup>SEM</sup></i>), causes a rough eye, which is suppressed by expression of <i>RhoGAP5A-dsRNAi</i> (<i>F</i>).</div></p></div><div class="aux"><div class="resc">Stephen P. Bruinsma, et al. Proc Natl Acad Sci U S A. 2007 April 24;104(17):7098-7103.</div><p class="links"><a class="dblinks" href="/pubmed/17438281" ref="ordinalpos=1985&amp;ncbi_uid=963553&amp;link_uid=17438281">Citation</a><a class="dblinks" href="/pmc/articles/PMC1855405/" ref="ordinalpos=1985&amp;ncbi_uid=963553&amp;link_uid=PMC1855405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1986.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2689776/figure/f7/" ref="ordinalpos=1986&amp;ncbi_uid=1873422&amp;link_uid=PMC2689776" href="/pmc/articles/PMC2689776/"><img src="/pmc/articles/instance/2689776/bin/zjw0040933630007.gif" src-large="/pmc/articles/instance/2689776/bin/zjw0040933630007.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2689776/" ref="ordinalpos=1986&amp;ncbi_uid=1873422&amp;link_uid=PMC2689776" image-link="/pmc/articles/PMC2689776/figure/f7/" class="imagepopup">F ig . 7. From: Proteomics and <span class="highlight" style="background-color:">Pathway</span> Analysis Identifies JNK <span class="highlight" style="background-color:">Signaling</span> as Critical for High Linear Energy Transfer Radiation-induced Apoptosis in Non-small Lung Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>A simplified hypothetical illustration of <span class="highlight" style="background-color:">signaling</span> in response to low and high LET IR in U-1810 cells.</b> The NSCLC cell line U-1810 can resist apoptosis when treated with low LET IR. However, in response to high LET IR, these cells start a proapoptotic <span class="highlight" style="background-color:">signaling</span> through an early activation of JNK. <i>Left</i>, a hypothetical explanation is that high LET IR causes a decrease of 14-3-3 and possibly phosphorylation of c-Abl by JNK (<i>I</i>). As a consequence, c-Abl cannot form a complex with 14-3-3 (<i>II</i>) and is allowed to translocate to the nucleus (<i>III</i>) where it is activated by IR-induced DNA damage that in turn through activation of MKK4 mediates a late JNK activation (<i>IV</i>). This subsequently leads to activation of various proapoptotic <span class="highlight" style="background-color:">signaling</span> molecules and activation of mitochondria resulting in an apoptotic cell death. JNK may also phosphorylate the transcriptional factor c-Jun, which can increase expression of Fas ligand (<i>V</i>) and thereby activate non-mitochondria-mediated apoptotic <span class="highlight" style="background-color:">signaling</span> (<i>VI</i>). High LET IR also deactivates p38 that in turn inhibits the IGF-1R-mediated survival <span class="highlight" style="background-color:">signaling</span> in these NSCLC cells (<i>VII</i>). <i>Right</i>, in contrast, low LET IR can neither activate JNK-mediated apoptotic <span class="highlight" style="background-color:">signaling</span> (<i>VIII</i>) nor inhibit p38-mediated survival <span class="highlight" style="background-color:">signaling</span> via the IGF-1R/phosphatidylinositol 3-kinase (<i>PI3K</i>)/Akt <span class="highlight" style="background-color:">pathway</span> (<i>IX</i>). As a result, U-1810 cells can resist low LET IR-induced apoptotic cell death. <i>Fas R</i>, Fas receptor; <i>cyt. c</i>, cytochrome <i>c</i>. <i>FOX</i>, Forkhead box.</div></p></div><div class="aux"><div class="resc">Sara StÃ¥hl, et al. Mol Cell Proteomics. 2009 May;8(5):1117-1129.</div><p class="links"><a class="dblinks" href="/pubmed/19168796" ref="ordinalpos=1986&amp;ncbi_uid=1873422&amp;link_uid=19168796">Citation</a><a class="dblinks" href="/pmc/articles/PMC2689776/" ref="ordinalpos=1986&amp;ncbi_uid=1873422&amp;link_uid=PMC2689776">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1987.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3296313/figure/fig3/" ref="ordinalpos=1987&amp;ncbi_uid=3898829&amp;link_uid=PMC3296313" href="/pmc/articles/PMC3296313/"><img src="/pmc/articles/instance/3296313/bin/IJCB2012-501962.003.gif" src-large="/pmc/articles/instance/3296313/bin/IJCB2012-501962.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3296313/" ref="ordinalpos=1987&amp;ncbi_uid=3898829&amp;link_uid=PMC3296313" image-link="/pmc/articles/PMC3296313/figure/fig3/" class="imagepopup">Figure 3.  From: Macrophage Migration and Its Regulation by CSF-1. </a></p><div class="supp"><p class="details"><div>A schematic of <span class="highlight" style="background-color:">signaling</span> pathways activated by the Rho GTPases in macrophages. Generic GEFs and GAPs are designated as upstream regulators of Cdc42, Rac, and Rho activity. Downstream <span class="highlight" style="background-color:">signaling</span> molecules specifically labeled in the schematic are those that are described in the text. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> outcomes are depicted for actin cytoskeleton responses only. Regulation of adhesion by Rho GTPases in macrophages is not shown as the <span class="highlight" style="background-color:">signaling</span> pathways are not yet elucidated.</div></p></div><div class="aux"><div class="resc">Fiona J. Pixley. Int J Cell Biol. 2012;2012:501962.</div><p class="links"><a class="dblinks" href="/pubmed/22505929" ref="ordinalpos=1987&amp;ncbi_uid=3898829&amp;link_uid=22505929">Citation</a><a class="dblinks" href="/pmc/articles/PMC3296313/" ref="ordinalpos=1987&amp;ncbi_uid=3898829&amp;link_uid=PMC3296313">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1988.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3885983/figure/F1/" ref="ordinalpos=1988&amp;ncbi_uid=5692466&amp;link_uid=PMC3885983" href="/pmc/articles/PMC3885983/"><img src="/pmc/articles/instance/3885983/bin/nihms542217f1.gif" src-large="/pmc/articles/instance/3885983/bin/nihms542217f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3885983/" ref="ordinalpos=1988&amp;ncbi_uid=5692466&amp;link_uid=PMC3885983" image-link="/pmc/articles/PMC3885983/figure/F1/" class="imagepopup">Figure 1. The canonical RTK-Ras-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and alternative Ras-dependent pathways in C. elegans.  From: Canonical RTK-Ras-ERK <span class="highlight" style="background-color:">signaling</span> and related alternative pathways. </a></p><div class="supp"><p class="details"><div>A) The canonical <span class="highlight" style="background-color:">pathway</span>. Positively acting factors are shown in green; negatively acting factors are in red. Only the most widely required and conserved factors are shown. GF, growth factor. TX, transcription. B) LET-23/EGFR and C) EGL-15/FGFR both act in part through the canonical <span class="highlight" style="background-color:">pathway</span>, but they are activated by different ligands and control different sets of developmental processes. Included in B) are nuclear proteins, including two MPK-1/ERK substrates (LIN-1/ETS and EOR-1), that are jointly important for most LET-23-dependent somatic cell fates. Downstream targets of EGL-15-Ras-ERK <span class="highlight" style="background-color:">signaling</span> are not yet known. D) Ras-ERK functions to promote meiotic pachytene progression and various other aspects of germline development; the receptors that activate Ras-ERK for these functions are not currently known. MPK-1/ERK utilizes more than 30 different substrates in the germline. E) A G-protein coupled receptor (GPCR) may activate Ras-ERK <span class="highlight" style="background-color:">signaling</span> in some contexts. F) LET-60/Ras can signal through the alternative Ral <span class="highlight" style="background-color:">pathway</span>. See text for additional details and references.</div></p></div><div class="aux"><div class="resc">Meera V. Sundaram. WormBook. :10.1895/wormbook.1.80.2.</div><p class="links"><a class="dblinks" href="/pubmed/23908058" ref="ordinalpos=1988&amp;ncbi_uid=5692466&amp;link_uid=23908058">Citation</a><a class="dblinks" href="/pmc/articles/PMC3885983/" ref="ordinalpos=1988&amp;ncbi_uid=5692466&amp;link_uid=PMC3885983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1989.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634610/figure/F5/" ref="ordinalpos=1989&amp;ncbi_uid=4827894&amp;link_uid=PMC3634610" href="/pmc/articles/PMC3634610/"><img src="/pmc/articles/instance/3634610/bin/IRI-32-134-g005.gif" src-large="/pmc/articles/instance/3634610/bin/IRI-32-134-g005.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634610/" ref="ordinalpos=1989&amp;ncbi_uid=4827894&amp;link_uid=PMC3634610" image-link="/pmc/articles/PMC3634610/figure/F5/" class="imagepopup">Figure 5. From: C-Type Lectin-Like Receptors of the Dectin-1 Cluster: Ligands and <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div> <b>CTLRs within the Dectin-1 cluster that contain novel <span class="highlight" style="background-color:">signaling</span> motifs.</b> Many components downstream of LOX-1s novel DDL motif have been determined, although exact mechanisms are not entirely identified. Interestingly, LOX-1 has the ability to up-regulate transcription of other receptors including itself; a mechanism that has been shown to aid in plaque formation during atherosclerosis. Downstream <span class="highlight" style="background-color:">signaling</span> of CLEC-1 is largely unknown; however, it does contain a tri-acidic motif that may have <span class="highlight" style="background-color:">signaling</span> potential due to similar <span class="highlight" style="background-color:">signaling</span> sequences in Dectin-1 and CLEC-2. Dashed arrows represent a <span class="highlight" style="background-color:">pathway</span> that has yet to be fully defined, and the grey arrow represents potential pathways based on other related receptor functions.</div></p></div><div class="aux"><div class="resc">Anthony Plato, et al. Int Rev Immunol. 2013 April;32(2):134-156.</div><p class="links"><a class="dblinks" href="/pubmed/23570314" ref="ordinalpos=1989&amp;ncbi_uid=4827894&amp;link_uid=23570314">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634610/" ref="ordinalpos=1989&amp;ncbi_uid=4827894&amp;link_uid=PMC3634610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1990.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2970638/figure/F8/" ref="ordinalpos=1990&amp;ncbi_uid=3424635&amp;link_uid=PMC2970638" href="/pmc/articles/PMC2970638/"><img src="/pmc/articles/instance/2970638/bin/nihms219609f8.gif" src-large="/pmc/articles/instance/2970638/bin/nihms219609f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2970638/" ref="ordinalpos=1990&amp;ncbi_uid=3424635&amp;link_uid=PMC2970638" image-link="/pmc/articles/PMC2970638/figure/F8/" class="imagepopup">Figure 8. Non-linearities in paxillin vs tension and a <span class="highlight" style="background-color:">Signaling</span> Circuit for Anchorage-dependent Contractility and Viability.  From: Up-regulation of Paxillin and Focal Adhesion <span class="highlight" style="background-color:">Signaling</span> follows Dystroglycan Complex deletions and promotes a Hypertensive State of Differentiation. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Non-linear relationship between total paxillin and cell tension, summarizing the various findings.<br /><b>(B)</b> The underlying <span class="highlight" style="background-color:">Signaling</span> circuit for anchorage dependent contractility and viability. Dystroglycan-related defects in the muscular dystrophies leads to upregulation of integrins and associated proteins. Upregulation of FA proteins such as paxillin, vinculin and FAK helps to preserve muscle integrity by maintaining a balance between adhesion and contractility. A role for MAPK proteins in sustained stretch-mediated mechanosignaling emerges as a common <span class="highlight" style="background-color:">pathway</span> in both dystrophies studied.</div></p></div><div class="aux"><div class="resc">Shamik Sen, et al. Eur J Cell Biol. ;90(2-3):249-260.</div><p class="links"><a class="dblinks" href="/pubmed/20663583" ref="ordinalpos=1990&amp;ncbi_uid=3424635&amp;link_uid=20663583">Citation</a><a class="dblinks" href="/pmc/articles/PMC2970638/" ref="ordinalpos=1990&amp;ncbi_uid=3424635&amp;link_uid=PMC2970638">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1991.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3132106/figure/F1/" ref="ordinalpos=1991&amp;ncbi_uid=3081427&amp;link_uid=PMC3132106" href="/pmc/articles/PMC3132106/"><img src="/pmc/articles/instance/3132106/bin/nihms-304985-f0001.gif" src-large="/pmc/articles/instance/3132106/bin/nihms-304985-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3132106/" ref="ordinalpos=1991&amp;ncbi_uid=3081427&amp;link_uid=PMC3132106" image-link="/pmc/articles/PMC3132106/figure/F1/" class="imagepopup">Figure 1.  From: WNTS and WNT receptors as therapeutic tools and targets in human disease processes. </a></p><div class="supp"><p class="details"><div>A summary of the extracellular interactions between Wnts, Wnt receptors and Wnt regulatory proteins. This simplified representation of the known extracellular interactions governing Wnt <span class="highlight" style="background-color:">signaling</span> depicts how secreted Wnt ligands can interact with receptors that trigger either the Wnt/beta-catenin <span class="highlight" style="background-color:">pathway</span> or other pathways like the Wnt/Ca <span class="highlight" style="background-color:">pathway</span> that act via other <span class="highlight" style="background-color:">signaling</span> mediators. As indicated in the figure, activation of Wnt/Ca receptors can inhibit the Wnt/beta-catenin <span class="highlight" style="background-color:">pathway</span>. The portrayal of RYK reflects data suggesting that this receptor may be able to activate both the Wnt/beta-catenin as well as beta-catenin-independent Wnt <span class="highlight" style="background-color:">signaling</span> cascades (17). Extracellular levels of Wnt ligand are titrated by binding to sFRPs, which exhibit homology to the cysteine-rich Wnt-binding domains of the FZD receptor. The secreted protein DKK antagonizes Wnt <span class="highlight" style="background-color:">signaling</span> via binding to the canonical Wnt co-receptor LRP5/6.</div></p></div><div class="aux"><div class="resc">Andy J. Chien, et al. Front Biosci. ;12:448-457.</div><p class="links"><a class="dblinks" href="/pubmed/17127309" ref="ordinalpos=1991&amp;ncbi_uid=3081427&amp;link_uid=17127309">Citation</a><a class="dblinks" href="/pmc/articles/PMC3132106/" ref="ordinalpos=1991&amp;ncbi_uid=3081427&amp;link_uid=PMC3132106">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1992.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3870411/figure/F1/" ref="ordinalpos=1992&amp;ncbi_uid=5670568&amp;link_uid=PMC3870411" href="/pmc/articles/PMC3870411/"><img src="/pmc/articles/instance/3870411/bin/fimmu-04-00478-g001.gif" src-large="/pmc/articles/instance/3870411/bin/fimmu-04-00478-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3870411/" ref="ordinalpos=1992&amp;ncbi_uid=5670568&amp;link_uid=PMC3870411" image-link="/pmc/articles/PMC3870411/figure/F1/" class="imagepopup">Figure 1.  From: TNF Receptor 2 and Disease: Autoimmunity and Regenerative Medicine. </a></p><div class="supp"><p class="details"><div><b>TNF Signals throughâTNFR1 andâTNFR2 receptors (A) but abnormalities in this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in autoimmunity (B) can favor a <span class="highlight" style="background-color:">pathway</span> of selective apoptosis due to a variety of protein <span class="highlight" style="background-color:">signaling</span> defects</b>.</div></p></div><div class="aux"><div class="resc">Denise L. Faustman, et al. Front Immunol. 2013;4:478.</div><p class="links"><a class="dblinks" href="/pubmed/24391650" ref="ordinalpos=1992&amp;ncbi_uid=5670568&amp;link_uid=24391650">Citation</a><a class="dblinks" href="/pmc/articles/PMC3870411/" ref="ordinalpos=1992&amp;ncbi_uid=5670568&amp;link_uid=PMC3870411">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1993.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1237076/figure/fig7/" ref="ordinalpos=1993&amp;ncbi_uid=654519&amp;link_uid=PMC1237076" href="/pmc/articles/PMC1237076/"><img src="/pmc/articles/instance/1237076/bin/zmk0100573350007.gif" src-large="/pmc/articles/instance/1237076/bin/zmk0100573350007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1237076/" ref="ordinalpos=1993&amp;ncbi_uid=654519&amp;link_uid=PMC1237076" image-link="/pmc/articles/PMC1237076/figure/fig7/" class="imagepopup">Figure 7. From: The Wnt-NLK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Inhibits A-Myb Activity by Inhibiting the Association with Coactivator CBP and Methylating Histone H3. </a></p><div class="supp"><p class="details"><div>The Wnt-NLK <span class="highlight" style="background-color:">pathway</span> inhibits the activity of Myb family of proteins by two different mechanisms. The Wnt-NLK <span class="highlight" style="background-color:">pathway</span> inhibits A-Myb activity by blocking the interaction between A-Myb and CBP and inducing the methylation of histone H3-K9. In contrast, the Wnt-NLK <span class="highlight" style="background-color:">pathway</span> inhibits c-Myb activity by inducing proteasome-dependent c-Myb degradation.</div></p></div><div class="aux"><div class="resc">Toshihiro Kurahashi, et al. Mol Biol Cell. 2005 October;16(10):4705-4713.</div><p class="links"><a class="dblinks" href="/pubmed/16055500" ref="ordinalpos=1993&amp;ncbi_uid=654519&amp;link_uid=16055500">Citation</a><a class="dblinks" href="/pmc/articles/PMC1237076/" ref="ordinalpos=1993&amp;ncbi_uid=654519&amp;link_uid=PMC1237076">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1994.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3070197/figure/F1/" ref="ordinalpos=1994&amp;ncbi_uid=3494822&amp;link_uid=PMC3070197" href="/pmc/articles/PMC3070197/"><img src="/pmc/articles/instance/3070197/bin/nihms253368f1.gif" src-large="/pmc/articles/instance/3070197/bin/nihms253368f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3070197/" ref="ordinalpos=1994&amp;ncbi_uid=3494822&amp;link_uid=PMC3070197" image-link="/pmc/articles/PMC3070197/figure/F1/" class="imagepopup">Fig. 1. Model describing the effect of an enriched environment on the <span class="highlight" style="background-color:">signaling</span> pathways that promote LTP in the CA1 region of the hippocampus.  From: Long-Lasting and Transgenerational Effects of an Environmental Enrichment on Memory Formation. </a></p><div class="supp"><p class="details"><div><b>A</b>. After exposure of young animals (3â4 weeks of age) to an enriched environment for 2 weeks a new LTP inducing <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> appears that involves NMDA type glutamate receptor activation of p38 MAP kinase potentiated by cAMP. B. In conventionally housed animals this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is non-functional.</div></p></div><div class="aux"><div class="resc">Junko A. Arai, et al. Brain Res Bull. ;85(1-2):30-35.</div><p class="links"><a class="dblinks" href="/pubmed/21078373" ref="ordinalpos=1994&amp;ncbi_uid=3494822&amp;link_uid=21078373">Citation</a><a class="dblinks" href="/pmc/articles/PMC3070197/" ref="ordinalpos=1994&amp;ncbi_uid=3494822&amp;link_uid=PMC3070197">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1995.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3891361/figure/F8/" ref="ordinalpos=1995&amp;ncbi_uid=5742828&amp;link_uid=PMC3891361" href="/pmc/articles/PMC3891361/"><img src="/pmc/articles/instance/3891361/bin/an2013-0023i008.gif" src-large="/pmc/articles/instance/3891361/bin/an2013-0023i008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3891361/" ref="ordinalpos=1995&amp;ncbi_uid=5742828&amp;link_uid=PMC3891361" image-link="/pmc/articles/PMC3891361/figure/F8/" class="imagepopup">Figure 8. Current model.  From: Involvement of the Notch <span class="highlight" style="background-color:">pathway</span> in terminal astrocytic differentiation: role of PKA. </a></p><div class="supp"><p class="details"><div>cAMP/PKA/CREB and NICD/RBPJÎº <span class="highlight" style="background-color:">signaling</span> regulate the expression of the astrocytic marker GFAP and the Notch <span class="highlight" style="background-color:">pathway</span> components (Notch1 receptor and Hes5 transcription factor) during astrocytic differentiation of C6 cells.</div></p></div><div class="aux"><div class="resc">Carla Angulo-Rojo, et al. ASN Neuro. 2013;5(5):e00130.</div><p class="links"><a class="dblinks" href="/pubmed/24286475" ref="ordinalpos=1995&amp;ncbi_uid=5742828&amp;link_uid=24286475">Citation</a><a class="dblinks" href="/pmc/articles/PMC3891361/" ref="ordinalpos=1995&amp;ncbi_uid=5742828&amp;link_uid=PMC3891361">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1996.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3187934/figure/F3/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934" href="/pmc/articles/PMC3187934/"><img src="/pmc/articles/instance/3187934/bin/jcav02p0490g05.gif" src-large="/pmc/articles/instance/3187934/bin/jcav02p0490g05.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3187934/figure/F3/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934" href="/pmc/articles/PMC3187934/"><img src="/pmc/articles/instance/3187934/bin/jcav02p0490g06.gif" src-large="/pmc/articles/instance/3187934/bin/jcav02p0490g06.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3187934/figure/F3/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934" href="/pmc/articles/PMC3187934/"><img src="/pmc/articles/instance/3187934/bin/jcav02p0490g07.gif" src-large="/pmc/articles/instance/3187934/bin/jcav02p0490g07.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3187934/figure/F3/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934" href="/pmc/articles/PMC3187934/"><img src="/pmc/articles/instance/3187934/bin/jcav02p0490g08.gif" src-large="/pmc/articles/instance/3187934/bin/jcav02p0490g08.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3187934/figure/F3/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934" href="/pmc/articles/PMC3187934/"><img src="/pmc/articles/instance/3187934/bin/jcav02p0490g09.gif" src-large="/pmc/articles/instance/3187934/bin/jcav02p0490g09.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3187934/figure/F3/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934" href="/pmc/articles/PMC3187934/"><img src="/pmc/articles/instance/3187934/bin/jcav02p0490g10.gif" src-large="/pmc/articles/instance/3187934/bin/jcav02p0490g10.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3187934/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934" image-link="/pmc/articles/PMC3187934/figure/F3/" class="imagepopup">Fig 3.  From: miRNA Alterations Modify Kinase Activation In The IGF-1 <span class="highlight" style="background-color:">Pathway</span> And Correlate With Colorectal Cancer Stage And Progression In Patients. </a></p><div class="supp"><p class="details"><div>Phosphorylation events in the IGF-1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of CRC tumor samples. One Way ANOVA significance reported as P&lt;0.05 and unpaired t-tests to delineate significant alterations, p&lt; 0.05 between stages of CRC.</div></p></div><div class="aux"><div class="resc">David L. Knowlton, et al. J Cancer. 2011;2:490-502.</div><p class="links"><a class="dblinks" href="/pubmed/21980324" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=21980324">Citation</a><a class="dblinks" href="/pmc/articles/PMC3187934/" ref="ordinalpos=1996&amp;ncbi_uid=3273179&amp;link_uid=PMC3187934">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1997.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3691929/figure/fig4/" ref="ordinalpos=1997&amp;ncbi_uid=5081275&amp;link_uid=PMC3691929" href="/pmc/articles/PMC3691929/"><img src="/pmc/articles/instance/3691929/bin/BMRI2013-568671.004.gif" src-large="/pmc/articles/instance/3691929/bin/BMRI2013-568671.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3691929/" ref="ordinalpos=1997&amp;ncbi_uid=5081275&amp;link_uid=PMC3691929" image-link="/pmc/articles/PMC3691929/figure/fig4/" class="imagepopup">Figure 4.  From: The Omega-3 Polyunsaturated Fatty Acid DHA Induces Simultaneous Apoptosis and Autophagy via Mitochondrial ROS-Mediated Akt-mTOR <span class="highlight" style="background-color:">Signaling</span> in Prostate Cancer Cells Expressing Mutant p53. </a></p><div class="supp"><p class="details"><div>Inhibition of Akt-mTOR <span class="highlight" style="background-color:">signaling</span> contributes to the ROS-mediated apoptosis and autophagy. (a) Inhibition of Akt-mTOR <span class="highlight" style="background-color:">signaling</span> induced by DHA. PC3 (left) and DU145 (right) cells were incubated with 0â50â<i>Î¼</i>M DHA for 24âh, and then the expression levels of Akt-mTOR <span class="highlight" style="background-color:">pathway</span> molecules were detected by Western blot analysis. (b) Effect of NAC on the expression levels of Akt-mTOR <span class="highlight" style="background-color:">pathway</span> molecules in DHA-treated cells. PC3 (left) and DU145 (right) cells were exposed to 50â<i>Î¼</i>M DHA and/or 5âmM NAC for 12âh, and the expression levels of Akt-mTOR <span class="highlight" style="background-color:">pathway</span> molecules were examined by Western blot analysis.</div></p></div><div class="aux"><div class="resc">Soyeon Shin, et al. Biomed Res Int. 2013;2013:568671.</div><p class="links"><a class="dblinks" href="/pubmed/23841076" ref="ordinalpos=1997&amp;ncbi_uid=5081275&amp;link_uid=23841076">Citation</a><a class="dblinks" href="/pmc/articles/PMC3691929/" ref="ordinalpos=1997&amp;ncbi_uid=5081275&amp;link_uid=PMC3691929">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1998.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2813708/figure/F2/" ref="ordinalpos=1998&amp;ncbi_uid=2172237&amp;link_uid=PMC2813708" href="/pmc/articles/PMC2813708/"><img src="/pmc/articles/instance/2813708/bin/nihms169986f2.gif" src-large="/pmc/articles/instance/2813708/bin/nihms169986f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2813708/" ref="ordinalpos=1998&amp;ncbi_uid=2172237&amp;link_uid=PMC2813708" image-link="/pmc/articles/PMC2813708/figure/F2/" class="imagepopup">Figure 2.  From: T-Cell Infiltration and <span class="highlight" style="background-color:">Signaling</span> in the Adult Dorsal Spinal Cord Is a Major Contributor to Neuropathic Pain-Like Hypersensitivity. </a></p><div class="supp"><p class="details"><div>Canonical <span class="highlight" style="background-color:">pathway</span> analysis of <span class="highlight" style="background-color:">signaling</span> cascades associated genes regulated in the adult but not the neonate dorsal horn (Ad0.05/Ne0.9 gene list) using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. Bars represent the LOD [âlog(<i>p</i> value)] for that <span class="highlight" style="background-color:">pathway</span> being overrepresented within the gene list (right-tailed Fisher's exact test). LOD 1.3 (<i>p</i> value of 0.05) threshold used. Gene functional categories primarily representing microglial cell function and T-cell function are shown.</div></p></div><div class="aux"><div class="resc">Michael Costigan, et al. J Neurosci. ;29(46):14415.</div><p class="links"><a class="dblinks" href="/pubmed/19923276" ref="ordinalpos=1998&amp;ncbi_uid=2172237&amp;link_uid=19923276">Citation</a><a class="dblinks" href="/pmc/articles/PMC2813708/" ref="ordinalpos=1998&amp;ncbi_uid=2172237&amp;link_uid=PMC2813708">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1999.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3472468/figure/F4/" ref="ordinalpos=1999&amp;ncbi_uid=5385302&amp;link_uid=PMC3472468" href="/pmc/articles/PMC3472468/"><img src="/pmc/articles/instance/3472468/bin/zh70191237670004.gif" src-large="/pmc/articles/instance/3472468/bin/zh70191237670004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3472468/" ref="ordinalpos=1999&amp;ncbi_uid=5385302&amp;link_uid=PMC3472468" image-link="/pmc/articles/PMC3472468/figure/F4/" class="imagepopup">Fig. 4. From: Genomic analysis of sleep deprivation reveals translational regulation in the hippocampus. </a></p><div class="supp"><p class="details"><div>The effects of sleep deprivation on the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes regulated by sleep deprivation mapped to the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (adapted from KEGG and Wikipathways). Upregulated gene names are written in lower case, in bold and italics, and downregulated gene names are written in bold with underlining. Note that genes from several functional clusters and enriched pathways shown in Fig. 3 and Table 2 are contained within this <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Christopher G. Vecsey, et al. Physiol Genomics. 2012 October 15;44(20):981-991.</div><p class="links"><a class="dblinks" href="/pubmed/22930738" ref="ordinalpos=1999&amp;ncbi_uid=5385302&amp;link_uid=22930738">Citation</a><a class="dblinks" href="/pmc/articles/PMC3472468/" ref="ordinalpos=1999&amp;ncbi_uid=5385302&amp;link_uid=PMC3472468">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2000.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3206446/figure/F6/" ref="ordinalpos=2000&amp;ncbi_uid=3323280&amp;link_uid=PMC3206446" href="/pmc/articles/PMC3206446/"><img src="/pmc/articles/instance/3206446/bin/1471-213X-11-57-6.gif" src-large="/pmc/articles/instance/3206446/bin/1471-213X-11-57-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3206446/" ref="ordinalpos=2000&amp;ncbi_uid=3323280&amp;link_uid=PMC3206446" image-link="/pmc/articles/PMC3206446/figure/F6/" class="imagepopup">Figure 6.  From: Loss of the Drosophila cell polarity regulator Scribbled promotes epithelial tissue overgrowth and cooperation with oncogenic Ras-Raf through impaired Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>aPKC <span class="highlight" style="background-color:">signaling</span> is required for the impaired Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in <i>scrib </i>mutants</b>. Larval eye/antennal discs with <i>ey-FLP </i>induced mutant clones marked by GFP expression (green, or magenta in the merges). Grayscale is Î²-GAL. A white bar indicates the location of the MF. (A-D) Coexpression of <i>aPKC<sup>CAAXDN </sup></i>and <i>bsk<sup>DN </sup></i>in clones does not alter <i>ex-lacZ </i>(A) or <i>fj-lacZ </i>(C) expression, but when <i>aPKC<sup>CAAXDN </sup></i>and <i>bsk<sup>DN </sup></i>are coexpressed in <i>scrib<sup>1 </sup></i>clones, the normal pattern of <i>ex-lacZ </i>(B) and <i>fj-lacZ </i>(D) expression is restored (compared with <i>scrib<sup>1 </sup></i>clones expressing <i>bsk<sup>DN </sup></i>alone in Figure 1).</div></p></div><div class="aux"><div class="resc">Karen Doggett, et al. BMC Dev Biol. 2011;11:57-57.</div><p class="links"><a class="dblinks" href="/pubmed/21955824" ref="ordinalpos=2000&amp;ncbi_uid=3323280&amp;link_uid=21955824">Citation</a><a class="dblinks" href="/pmc/articles/PMC3206446/" ref="ordinalpos=2000&amp;ncbi_uid=3323280&amp;link_uid=PMC3206446">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="19" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="20" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="21" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553ff746437716036ad09555">Saving and Managing Searches - My NCBI Help</a>
          <div class="ralinkpop offscreen_noflow">Saving and Managing Searches - My NCBI Help<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553fef47cbe05d491bf71396">signaling pathway AND (free full text[sb]) <span class="number">(225335)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553fef249892033523880147">signaling pathway <span class="number">(491236)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553fedbe437716036ace4747">PMC Help</a>
          <div class="ralinkpop offscreen_noflow">PMC Help<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553fedb5cbe05d491bf6bb4a">NCBI Help Manual</a>
          <div class="ralinkpop offscreen_noflow">NCBI Help Manual<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T17:14:45-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC17C253FF8411000000000005303D;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC17C253FF8411000000000005303D" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>