
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="36" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A262C543122C100000000003F4E39" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="35" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="36" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="37" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="36" /></div>
            <div><h2 class="result_count">Results: 3501 to 3600 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3501.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2151969/figure/Fig3/" ref="ordinalpos=3501&amp;ncbi_uid=1248639&amp;link_uid=PMC2151969" href="/pmc/articles/PMC2151969/"><img src="/pmc/articles/instance/2151969/bin/10616_2007_9092_Fig3_HTML.gif" src-large="/pmc/articles/instance/2151969/bin/10616_2007_9092_Fig3_HTML.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2151969/" ref="ordinalpos=3501&amp;ncbi_uid=1248639&amp;link_uid=PMC2151969" image-link="/pmc/articles/PMC2151969/figure/Fig3/" class="imagepopup">Fig. 3.  From: FGF-2 increases osteogenic and chondrogenic differentiation potentials of human mesenchymal stem cells by inactivation of TGF-? <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of genes up-regulated and down-regulated by FGF-2 in hMSCs. <span class="highlight" style="background-color:">Pathway</span> analysis of up-regulated and down-regulated genes by FGF-2 (Fig. 2) was performed by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. The <i>y</i>-axis showed the ratio of genes mapped in Fig. 2 against all of genes belongs to each canonical <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Tomomi Ito, et al. Cytotechnology. 2008 January;56(1):1-7.</div><p class="links"><a class="dblinks" href="/pubmed/19002835" ref="ordinalpos=3501&amp;ncbi_uid=1248639&amp;link_uid=19002835">Citation</a><a class="dblinks" href="/pmc/articles/PMC2151969/" ref="ordinalpos=3501&amp;ncbi_uid=1248639&amp;link_uid=PMC2151969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3502.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3842013/figure/F3/" ref="ordinalpos=3502&amp;ncbi_uid=5525296&amp;link_uid=PMC3842013" href="/pmc/articles/PMC3842013/"><img src="/pmc/articles/instance/3842013/bin/nihms387699f3.gif" src-large="/pmc/articles/instance/3842013/bin/nihms387699f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3842013/" ref="ordinalpos=3502&amp;ncbi_uid=5525296&amp;link_uid=PMC3842013" image-link="/pmc/articles/PMC3842013/figure/F3/" class="imagepopup">Figure 3. Beneficial and detrimental effects of adenosine <span class="highlight" style="background-color:">signaling</span> in sickle cell disease.  From: Adenosine <span class="highlight" style="background-color:">signaling</span> in normal and sickle erythrocytes and beyond. </a></p><div class="supp"><p class="details"><div>Elevated adenosine associated with SCD has beneficial effects by activating ADORA2A receptors on invariant natural killer T (iNKT) cells, a process that prevents iNKT cells activation and pulmonary inflammation. However, elevated adenosine activates ADORA2B receptors on erythrocytes thereby activating a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to deoxy-HbS polymerization and erythrocyte sickling (see Fig. 2). The activation of ADORA2B receptors on penile endothelial cells and cavernosal smooth muscle cells results in priapism, a serious complication of SCD, leading to penile fibrosis and erectile dysfunction.</div></p></div><div class="aux"><div class="resc">Yujin Zhang, et al. Microbes Infect. ;14(10):10.1016/j.micinf.2012.05.005.</div><p class="links"><a class="dblinks" href="/pubmed/22634345" ref="ordinalpos=3502&amp;ncbi_uid=5525296&amp;link_uid=22634345">Citation</a><a class="dblinks" href="/pmc/articles/PMC3842013/" ref="ordinalpos=3502&amp;ncbi_uid=5525296&amp;link_uid=PMC3842013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3503.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3202302/figure/F2/" ref="ordinalpos=3503&amp;ncbi_uid=3512168&amp;link_uid=PMC3202302" href="/pmc/articles/PMC3202302/"><img src="/pmc/articles/instance/3202302/bin/nihms322868f2.gif" src-large="/pmc/articles/instance/3202302/bin/nihms322868f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3202302/" ref="ordinalpos=3503&amp;ncbi_uid=3512168&amp;link_uid=PMC3202302" image-link="/pmc/articles/PMC3202302/figure/F2/" class="imagepopup">Fig. 2. Model depicting the canonical and non-canonical NFB <span class="highlight" style="background-color:">pathway</span>.  From: <span class="highlight" style="background-color:">Signaling</span> by the TNFR superfamily in B-cell biology and disease. </a></p><div class="supp"><p class="details"><div>Activation of the IKK complex in the canonical <span class="highlight" style="background-color:">pathway</span> leads to the phosphorylation and ubiqutination of IB, which is subsequently cleaved in the proteasome. The freed NFB dimers are now able to enter the nucleus. NIK mediated IKK1 phosphorylation leads to the activation of the non-canonical <span class="highlight" style="background-color:">pathway</span> which is characterized by the cleavage of p100 and release of the NFB heterodimer p52/RelB.</div></p></div><div class="aux"><div class="resc">Robert C. Rickert, et al. Immunol Rev. ;244(1):115-133.</div><p class="links"><a class="dblinks" href="/pubmed/22017435" ref="ordinalpos=3503&amp;ncbi_uid=3512168&amp;link_uid=22017435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3202302/" ref="ordinalpos=3503&amp;ncbi_uid=3512168&amp;link_uid=PMC3202302">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3504.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4038485/figure/pgen-1004371-g008/" ref="ordinalpos=3504&amp;ncbi_uid=6201613&amp;link_uid=PMC4038485" href="/pmc/articles/PMC4038485/"><img src="/pmc/articles/instance/4038485/bin/pgen.1004371.g008.gif" src-large="/pmc/articles/instance/4038485/bin/pgen.1004371.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4038485/" ref="ordinalpos=3504&amp;ncbi_uid=6201613&amp;link_uid=PMC4038485" image-link="/pmc/articles/PMC4038485/figure/pgen-1004371-g008/" class="imagepopup">Figure 8. Graphic representation of how RP mutations result in autophagy and insulin <span class="highlight" style="background-color:">pathway</span> inhibition..  From: Ribosomal Protein Mutations Induce Autophagy through S6 Kinase Inhibition of the Insulin <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>In normal cells, insulin signals through the insulin receptor substrate (IRS) to the PI3-kinase/AKT <span class="highlight" style="background-color:">pathway</span> in a manner that inhibits autophagy and increases the expression of glucose transporters on the cell membrane. In cells with RP loss, an increase of ROS stabilizes p53 and induces phosphorylation of S6 kinase. This hyper-phosphorylation of S6 kinase results in degradation of IRS and a decrease of insulin <span class="highlight" style="background-color:">signaling</span> to PI3-kinase and AKT. This loss of AKT activation in turn releases the inhibition of autophagy and reduces glycolysis.</div></p></div><div class="aux"><div class="resc">Harry F. Heijnen, et al. PLoS Genet. 2014 May;10(5):e1004371.</div><p class="links"><a class="dblinks" href="/pubmed/24875531" ref="ordinalpos=3504&amp;ncbi_uid=6201613&amp;link_uid=24875531">Citation</a><a class="dblinks" href="/pmc/articles/PMC4038485/" ref="ordinalpos=3504&amp;ncbi_uid=6201613&amp;link_uid=PMC4038485">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3505.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3682754/figure/f1-ebo-9-2013-229/" ref="ordinalpos=3505&amp;ncbi_uid=5054258&amp;link_uid=PMC3682754" href="/pmc/articles/PMC3682754/"><img src="/pmc/articles/instance/3682754/bin/ebo-9-2013-229f1.gif" src-large="/pmc/articles/instance/3682754/bin/ebo-9-2013-229f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3682754/" ref="ordinalpos=3505&amp;ncbi_uid=5054258&amp;link_uid=PMC3682754" image-link="/pmc/articles/PMC3682754/figure/f1-ebo-9-2013-229/" class="imagepopup">Figure 1.  From: Comment on "The Molecular Evolutionary Patterns of the Insulin/FOXO <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>". </a></p><div class="supp"><p class="details"><div>Correlation between <i></i> estimates and <span class="highlight" style="background-color:">pathway</span> position in the <i>Drosophila</i> insulin/TOR <span class="highlight" style="background-color:">pathway</span>. The analysis is restricted to the gene families used both by Alvarez-Ponce et al2 and by Wang et al.1 The <i></i> estimates were obtained from Alvarez-Ponce et al.2 Positions 16 correspond, respectively, to genes <i>chico</i>, <i>Pi3K21B</i>, <i>Pi3K92E</i>, <i>Pk61C</i>, <i>Akt1</i> and <i>foxo</i>.</div></p></div><div class="aux"><div class="resc">David Alvarez-Ponce, et al. Evol Bioinform Online. 2013;9:229-234.</div><p class="links"><a class="dblinks" href="/pubmed/23818748" ref="ordinalpos=3505&amp;ncbi_uid=5054258&amp;link_uid=23818748">Citation</a><a class="dblinks" href="/pmc/articles/PMC3682754/" ref="ordinalpos=3505&amp;ncbi_uid=5054258&amp;link_uid=PMC3682754">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3506.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3367549/figure/A008151F2/" ref="ordinalpos=3506&amp;ncbi_uid=6186406&amp;link_uid=PMC3367549" href="/pmc/articles/PMC3367549/"><img src="/pmc/articles/instance/3367549/bin/cshperspect-MDV-008151_F2.gif" src-large="/pmc/articles/instance/3367549/bin/cshperspect-MDV-008151_F2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3367549/" ref="ordinalpos=3506&amp;ncbi_uid=6186406&amp;link_uid=PMC3367549" image-link="/pmc/articles/PMC3367549/figure/A008151F2/" class="imagepopup">Figure 2. From: <span class="highlight" style="background-color:">Signaling</span> in Cell Differentiation and Morphogenesis. </a></p><div class="supp"><p class="details"><div>Canonical (left) and noncanonical (right) WNT <span class="highlight" style="background-color:">signaling</span> pathways. In the absence of WNT <span class="highlight" style="background-color:">signaling</span>, the effector of canonical WNT <span class="highlight" style="background-color:">signaling</span>, -catenin is found complexed in the cytoplasm with a destruction complex containing Axin and GSK3. GSK3 phosphorylates -catenin, which results in its targeting to and degradation by the proteasome. The binding of canonical WNT ligands to Frizzled (Fz) and LRP5/6 receptors results in the phosphorylation of LRP5/6 through the action of Dishevelled (Dsh). Phosphorylated LRP5/6 recruits Axin away from the destruction complex, liberating -catenin that accumulates in the cytoplasm and translocates to the nucleus, where it interacts with transcriptional regulators of the TCF/LEF family to alter gene expression. AKT and PKC downstream from RTK <span class="highlight" style="background-color:">signaling</span> may inhibit the activity of the -catenin destruction complex to promote -catenin <span class="highlight" style="background-color:">signaling</span>. Noncanonical WNT/Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> is initiated by the activation of PLC, probably through trimeric G-proteins such as Gbg. The release of Ca<sup>2+</sup> results in the activation of NFAT (nuclear factor associated with T cells) transcription factors through calcineurin (Cn). Other downstream transcription factors (TFs) are activated by PKC and CaMKII. A Wnt/planar cell polarity <span class="highlight" style="background-color:">pathway</span> involves Diego (Dgo), Smurf, and Par6, which through cross-inhibitory interactions with Van Gogh-like (Vangl)-Prickle (Pk) complexes regulate planar cell polarity.</div></p></div><div class="aux"><div class="resc">M. Albert Basson. Cold Spring Harb Perspect Biol. 2012 June;4(6):a008151.</div><p class="links"><a class="dblinks" href="/pubmed/22570373" ref="ordinalpos=3506&amp;ncbi_uid=6186406&amp;link_uid=22570373">Citation</a><a class="dblinks" href="/pmc/articles/PMC3367549/" ref="ordinalpos=3506&amp;ncbi_uid=6186406&amp;link_uid=PMC3367549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3507.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3171751/figure/F3/" ref="ordinalpos=3507&amp;ncbi_uid=3480004&amp;link_uid=PMC3171751" href="/pmc/articles/PMC3171751/"><img src="/pmc/articles/instance/3171751/bin/nihms304572f3.gif" src-large="/pmc/articles/instance/3171751/bin/nihms304572f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3171751/" ref="ordinalpos=3507&amp;ncbi_uid=3480004&amp;link_uid=PMC3171751" image-link="/pmc/articles/PMC3171751/figure/F3/" class="imagepopup">Figure 3.  From: NEW DIRECTIONS IN BASIC RESEARCH IN HYPERTROPHY AND HEART FAILURE: RELEVANCE FOR PEDIATRIC CARDIOLOGY. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of altered gene expression during pressure overload hypertrophy. This type of analysis examines groups of genes which are altered during a particular cardiovascular stressor. If multiple components of a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are all up- or down-regulated in concert, the liklihood that <span class="highlight" style="background-color:">pathway</span> plays an important role in pathophysiology is enhanced. (Data courtesy of Dr. Sushma Reddy).</div></p></div><div class="aux"><div class="resc">Daniel Bernstein, et al. Prog Pediatr Cardiol. ;32(1):5-9.</div><p class="links"><a class="dblinks" href="/pubmed/21927547" ref="ordinalpos=3507&amp;ncbi_uid=3480004&amp;link_uid=21927547">Citation</a><a class="dblinks" href="/pmc/articles/PMC3171751/" ref="ordinalpos=3507&amp;ncbi_uid=3480004&amp;link_uid=PMC3171751">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3508.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2562016/figure/F4/" ref="ordinalpos=3508&amp;ncbi_uid=1769856&amp;link_uid=PMC2562016" href="/pmc/articles/PMC2562016/"><img src="/pmc/articles/instance/2639168/bin/btn451f4.gif" src-large="/pmc/articles/instance/2639168/bin/btn451f4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2562016/" ref="ordinalpos=3508&amp;ncbi_uid=1769856&amp;link_uid=PMC2562016" image-link="/pmc/articles/PMC2562016/figure/F4/" class="imagepopup">Fig. 4. From: Novel <span class="highlight" style="background-color:">pathway</span> compendium analysis elucidates mechanism of pro-angiogenic synthetic small molecule. </a></p><div class="supp"><p class="details"><div>Regulation of the TNF- <span class="highlight" style="background-color:">pathway</span> by PNF1. (<b>A</b>) The members of the TNF- <span class="highlight" style="background-color:">pathway</span> that were significantly regulated by PNF1 over at least one timepoint (spanning 1, 2, 4, 8, 16, 24 and 48 h post-supplementation). These renderings are based on available literature on these pathways, as taken from the BioCarta Proteomic <span class="highlight" style="background-color:">Pathway</span> Project (http://www.biocarta.com/genes/index.asp) and supplemented by the IPKB. Shaded gene products represent PNF1-stimulated genes; gene products not shaded (white) were not significantly regulated by the drug but are required for <span class="highlight" style="background-color:">signaling</span> between PNF1-regulated genes. (<b>B</b>) Plots of the fold changes over time for the individual gene products in the TNF- <span class="highlight" style="background-color:">pathway</span> that were significantly differentially expressed after PNF1 stimulation.</div></p></div><div class="aux"><div class="resc">Kristen A. Wieghaus, et al. Bioinformatics. 2008 October 15;24(20):2384-2390.</div><p class="links"><a class="dblinks" href="/pubmed/18718940" ref="ordinalpos=3508&amp;ncbi_uid=1769856&amp;link_uid=18718940">Citation</a><a class="dblinks" href="/pmc/articles/PMC2562016/" ref="ordinalpos=3508&amp;ncbi_uid=1769856&amp;link_uid=PMC2562016">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3509.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2881512/figure/F1/" ref="ordinalpos=3509&amp;ncbi_uid=2278569&amp;link_uid=PMC2881512" href="/pmc/articles/PMC2881512/"><img src="/pmc/articles/instance/2881512/bin/aging-02-238-g001.gif" src-large="/pmc/articles/instance/2881512/bin/aging-02-238-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2881512/" ref="ordinalpos=3509&amp;ncbi_uid=2278569&amp;link_uid=PMC2881512" image-link="/pmc/articles/PMC2881512/figure/F1/" class="imagepopup">Figure 1. From: Protein kinase A is a target for aging and the aging heart. </a></p><div class="supp"><p class="details"><div><b>The PKA                                                     <span class="highlight" style="background-color:">pathway</span>.</b>                                              The                                             PKA <span class="highlight" style="background-color:">pathway</span> is a nutrient sensing <span class="highlight" style="background-color:">pathway</span>.  In mammals, nutrients are                                             sensed by a G-protein (GEF) that activates an adenylyl cylase (AC).  AC                                             produces cAMP, which binds to the regulatory subunits (R) of the PKA                                             holoenzyme, releasing the catalytic subunits (C), which are then free to                                             enter the nucleus of the cell and activate gene transcription or to                                             interact with other <span class="highlight" style="background-color:">signaling</span> proteins in the cell.                                                              </div></p></div><div class="aux"><div class="resc">Linda C Enns, et al. Aging (Albany NY). 2010 April;2(4):238-243.</div><p class="links"><a class="dblinks" href="/pubmed/20448293" ref="ordinalpos=3509&amp;ncbi_uid=2278569&amp;link_uid=20448293">Citation</a><a class="dblinks" href="/pmc/articles/PMC2881512/" ref="ordinalpos=3509&amp;ncbi_uid=2278569&amp;link_uid=PMC2881512">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3510.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2680807/figure/F5/" ref="ordinalpos=3510&amp;ncbi_uid=1864536&amp;link_uid=PMC2680807" href="/pmc/articles/PMC2680807/"><img src="/pmc/articles/instance/2680807/bin/1752-0509-3-46-5.gif" src-large="/pmc/articles/instance/2680807/bin/1752-0509-3-46-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2680807/" ref="ordinalpos=3510&amp;ncbi_uid=1864536&amp;link_uid=PMC2680807" image-link="/pmc/articles/PMC2680807/figure/F5/" class="imagepopup">Figure 5.  From: Genetic and environmental pathways to complex diseases. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span> Interaction Network</b>. Nodes represent KEGG metabolic and <span class="highlight" style="background-color:">signaling</span> pathways and are connected based on the KEGG database [72]. Node size is reflective of the number of phenotypes associated with the particular <span class="highlight" style="background-color:">pathway</span> based on application of SEPEA (p  0.003) to gene lists annotated from The Genetic Association Database[1]. Intensity of node color is reflective of the number of environmental factors associated with the particular <span class="highlight" style="background-color:">pathway</span> based on enrichment of gene lists annotated from The Comparative Toxicogenomics Database[15].</div></p></div><div class="aux"><div class="resc">Julia M Gohlke, et al. BMC Syst Biol. 2009;3:46-46.</div><p class="links"><a class="dblinks" href="/pubmed/19416532" ref="ordinalpos=3510&amp;ncbi_uid=1864536&amp;link_uid=19416532">Citation</a><a class="dblinks" href="/pmc/articles/PMC2680807/" ref="ordinalpos=3510&amp;ncbi_uid=1864536&amp;link_uid=PMC2680807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3511.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2783549/figure/F2/" ref="ordinalpos=3511&amp;ncbi_uid=2546101&amp;link_uid=PMC2783549" href="/pmc/articles/PMC2783549/"><img src="/pmc/articles/instance/2783549/bin/nihms146079f2.gif" src-large="/pmc/articles/instance/2783549/bin/nihms146079f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783549/" ref="ordinalpos=3511&amp;ncbi_uid=2546101&amp;link_uid=PMC2783549" image-link="/pmc/articles/PMC2783549/figure/F2/" class="imagepopup">Figure 2. Similarities between the activated fibroblasts in Rheumatoid Arthritis and the Carcinoma-Associated Fibroblasts in cancer.  From: Matrix Metalloproteinase and G Protein Coupled Receptors: Co-conspirators in the pathogenesis of autoimmune disease and cancer. </a></p><div class="supp"><p class="details"><div>In both cell types, CXCR4, SDF-1, and MMP-1 are up-regulated compared to normal fibroblasts. <span class="highlight" style="background-color:">Signaling</span> through CXCR4 leads to activation of the ERK1/2 <span class="highlight" style="background-color:">pathway</span>, which is a component of the MAPK <span class="highlight" style="background-color:">pathway</span>, in CAFs and the MAPK <span class="highlight" style="background-color:">pathway</span> in activated fibroblasts. Both the MAPK and ERK1/2 pathways are known to stimulate MMP-1 and SDF-1 levels in other cell types [3,4,25].</div></p></div><div class="aux"><div class="resc">Sarah M. Eck, et al. J Autoimmun. ;33(3-4):214-221.</div><p class="links"><a class="dblinks" href="/pubmed/19800199" ref="ordinalpos=3511&amp;ncbi_uid=2546101&amp;link_uid=19800199">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783549/" ref="ordinalpos=3511&amp;ncbi_uid=2546101&amp;link_uid=PMC2783549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3512.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2727416/figure/F7/" ref="ordinalpos=3512&amp;ncbi_uid=2408166&amp;link_uid=PMC2727416" href="/pmc/articles/PMC2727416/"><img src="/pmc/articles/instance/2727416/bin/zh89990938960007.gif" src-large="/pmc/articles/instance/2727416/bin/zh89990938960007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2727416/" ref="ordinalpos=3512&amp;ncbi_uid=2408166&amp;link_uid=PMC2727416" image-link="/pmc/articles/PMC2727416/figure/F7/" class="imagepopup">Figure 7.  From: The granulocyte-macrophage colony-stimulating factor receptor: linking its structure to cell <span class="highlight" style="background-color:">signaling</span> and its role in disease. </a></p><div class="supp"><p class="details"><div><b>Hemopoietic cell survival is regulated by exogenous concentrations of GM-CSF.</b> The selective activation of the Ser<sup>585</sup>/14-3-3/PI-3 kinase survival-only <span class="highlight" style="background-color:">pathway</span> is triggered by low pM concentrations of GM-CSF (left). The alternative survival and proliferation <span class="highlight" style="background-color:">pathway</span> triggered by higher concentrations of GM-CSF is dependent on the dodecamer assembly and involves the Jak/STAT, Ras/mitogen-activated protein kinase, and PI-3 kinase pathways (right). The key receptor-proximal events that initiate these <span class="highlight" style="background-color:">signaling</span> pathways are the mutually exclusive phosphorylation of Ser<sup>585</sup> and of Tyr<sup>577</sup> of c.</div></p></div><div class="aux"><div class="resc">Timothy R. Hercus, et al. Blood. 2009 August 13;114(7):1289-1298.</div><p class="links"><a class="dblinks" href="/pubmed/19436055" ref="ordinalpos=3512&amp;ncbi_uid=2408166&amp;link_uid=19436055">Citation</a><a class="dblinks" href="/pmc/articles/PMC2727416/" ref="ordinalpos=3512&amp;ncbi_uid=2408166&amp;link_uid=PMC2727416">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3513.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2535674/figure/fig8/" ref="ordinalpos=3513&amp;ncbi_uid=1608036&amp;link_uid=PMC2535674" href="/pmc/articles/PMC2535674/"><img src="/pmc/articles/instance/2535674/bin/GEN1801191f8.gif" src-large="/pmc/articles/instance/2535674/bin/GEN1801191f8.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2535674/" ref="ordinalpos=3513&amp;ncbi_uid=1608036&amp;link_uid=PMC2535674" image-link="/pmc/articles/PMC2535674/figure/fig8/" class="imagepopup">F igure  8..  From: Three ?-Subunits of Heterotrimeric G Proteins and an Adenylyl Cyclase Have Distinct Roles in Fruiting Body Development in the Homothallic Fungus Sordaria macrospora. </a></p><div class="supp"><p class="details"><div>A model for the predicted G protein -subunit <span class="highlight" style="background-color:">signaling</span> in <i>S. macrospora</i>. GSA1 and GSA2 propagate signals within the pheromone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, in which GSA1 is the predominant regulator of fruiting body development upstream of the STE12 transcription factor. GSA3 and SAC1 act on sexual development in a less characterized, parallel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Putative myristoylation of G-subunits GSA1 and GSA3 is indicated by tails. Putative farnesylation of PPG2 is shown by a serrated tail.</div></p></div><div class="aux"><div class="resc">Jens Kamerewerd, et al. Genetics. 2008 September;180(1):191-206.</div><p class="links"><a class="dblinks" href="/pubmed/18723884" ref="ordinalpos=3513&amp;ncbi_uid=1608036&amp;link_uid=18723884">Citation</a><a class="dblinks" href="/pmc/articles/PMC2535674/" ref="ordinalpos=3513&amp;ncbi_uid=1608036&amp;link_uid=PMC2535674">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3514.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3122626/figure/F4/" ref="ordinalpos=3514&amp;ncbi_uid=3493347&amp;link_uid=PMC3122626" href="/pmc/articles/PMC3122626/"><img src="/pmc/articles/instance/3122626/bin/zmr9990922690004.gif" src-large="/pmc/articles/instance/3122626/bin/zmr9990922690004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3122626/" ref="ordinalpos=3514&amp;ncbi_uid=3493347&amp;link_uid=PMC3122626" image-link="/pmc/articles/PMC3122626/figure/F4/" class="imagepopup">Fig. 4. From: Regulatory Circuitry Governing Fungal Development, Drug Resistance, and Disease. </a></p><div class="supp"><p class="details"><div>Key cellular <span class="highlight" style="background-color:">signaling</span> cascades regulating morphogenesis in C. albicans. Numerous <span class="highlight" style="background-color:">signaling</span> pathways regulate C. albicans morphogenesis; the six most well-characterized pathways along with many of the key proteins involved in each <span class="highlight" style="background-color:">pathway</span> are depicted.</div></p></div><div class="aux"><div class="resc">Rebecca S. Shapiro, et al. Microbiol Mol Biol Rev. 2011 June;75(2):213-267.</div><p class="links"><a class="dblinks" href="/pubmed/21646428" ref="ordinalpos=3514&amp;ncbi_uid=3493347&amp;link_uid=21646428">Citation</a><a class="dblinks" href="/pmc/articles/PMC3122626/" ref="ordinalpos=3514&amp;ncbi_uid=3493347&amp;link_uid=PMC3122626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3515.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3470885/figure/F8/" ref="ordinalpos=3515&amp;ncbi_uid=5340190&amp;link_uid=PMC3470885" href="/pmc/articles/PMC3470885/"><img src="/pmc/articles/instance/3470885/bin/nihms-399227-f0008.gif" src-large="/pmc/articles/instance/3470885/bin/nihms-399227-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3470885/" ref="ordinalpos=3515&amp;ncbi_uid=5340190&amp;link_uid=PMC3470885" image-link="/pmc/articles/PMC3470885/figure/F8/" class="imagepopup">Figure 8. Working model.  From: Dectin-1 and IL-17A suppress murine asthma induced by Aspergillus versicolor but not Cladosporium cladosporioides due to differences in beta-glucan surface exposure. </a></p><div class="supp"><p class="details"><div>Exposure of beta-glucans (red) on the surface of <i>A. versicolor</i> promote <span class="highlight" style="background-color:">signaling</span> through Dectin-1 to induce a Th17 response. In the absence of Dectin-1, <i>A. versicolor</i> signals through an alternate <span class="highlight" style="background-color:">pathway</span> to induce Th2 cells and asthma. Beta-glucans (red) in <i>C. cladosporioides</i> are not readily available on the surface (black), thus preventing <span class="highlight" style="background-color:">signaling</span> through a Dectin-1, and promoting the use of an alternate <span class="highlight" style="background-color:">pathway</span> to induce a Th2 response and asthma.</div></p></div><div class="aux"><div class="resc">Rachael A. Mintz-Cole, et al. J Immunol. 2012 October 1;189(7):3609-3617.</div><p class="links"><a class="dblinks" href="/pubmed/22962686" ref="ordinalpos=3515&amp;ncbi_uid=5340190&amp;link_uid=22962686">Citation</a><a class="dblinks" href="/pmc/articles/PMC3470885/" ref="ordinalpos=3515&amp;ncbi_uid=5340190&amp;link_uid=PMC3470885">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3516.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3081620/figure/F2/" ref="ordinalpos=3516&amp;ncbi_uid=3386426&amp;link_uid=PMC3081620" href="/pmc/articles/PMC3081620/"><img src="/pmc/articles/instance/3081620/bin/nihms-275732-f0002.gif" src-large="/pmc/articles/instance/3081620/bin/nihms-275732-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3081620/" ref="ordinalpos=3516&amp;ncbi_uid=3386426&amp;link_uid=PMC3081620" image-link="/pmc/articles/PMC3081620/figure/F2/" class="imagepopup">Fig. 2.  From: Activation of The Transforming Growth Factor-Beta <span class="highlight" style="background-color:">Pathway</span> in Bacterial Otitis Media. </a></p><div class="supp"><p class="details"><div>Effects of bacterial OM and ETO on the expression of the TGF- and - genes (A). TGF- was mainly expressed in MEEC whereas TGF-1 was not. Unlike TGF-1, the expression of TGF-2 and TGF-3 was mainly in dMEEC. The TGF-1 gene expression was down-regulated by Pn6A at the early stage (day 3~7) compared with their controls. The TGF-2 gene was not responsive to bacterial OM and ETO. The TGF-3 gene expression was slightly responsive to Pn6A, NTHi, and ETO. Pn6A-3D, day 3 following pneumococcal infection in the middle ear cavity. The genes in the TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, regulated by bacterial OM and ETO, are presented in heat map (B). Note that many genes in the TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are up-regulated in bacterial OM, especially NTHi day 7, but only few genes are slightly up-regulated in ETO.</div></p></div><div class="aux"><div class="resc">Yun-Woo Lee, et al. Ann Otol Rhinol Laryngol. ;120(3):204-213.</div><p class="links"><a class="dblinks" href="/pubmed/21510147" ref="ordinalpos=3516&amp;ncbi_uid=3386426&amp;link_uid=21510147">Citation</a><a class="dblinks" href="/pmc/articles/PMC3081620/" ref="ordinalpos=3516&amp;ncbi_uid=3386426&amp;link_uid=PMC3081620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3517.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3963847/figure/pone-0091094-g004/" ref="ordinalpos=3517&amp;ncbi_uid=5951811&amp;link_uid=PMC3963847" href="/pmc/articles/PMC3963847/"><img src="/pmc/articles/instance/3963847/bin/pone.0091094.g004.gif" src-large="/pmc/articles/instance/3963847/bin/pone.0091094.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3963847/" ref="ordinalpos=3517&amp;ncbi_uid=5951811&amp;link_uid=PMC3963847" image-link="/pmc/articles/PMC3963847/figure/pone-0091094-g004/" class="imagepopup">Figure 4. Effect of CNT on the mTOR/p70S6K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Convallatoxin, a Dual Inducer of Autophagy and Apoptosis, Inhibits Angiogenesis In Vitro and In Vivo. </a></p><div class="supp"><p class="details"><div>Tubulin was used as an internal control.</div></p></div><div class="aux"><div class="resc">Seung Ya Yang, et al. PLoS One. 2014;9(3):e91094.</div><p class="links"><a class="dblinks" href="/pubmed/24663328" ref="ordinalpos=3517&amp;ncbi_uid=5951811&amp;link_uid=24663328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3963847/" ref="ordinalpos=3517&amp;ncbi_uid=5951811&amp;link_uid=PMC3963847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3518.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807333/figure/F12/" ref="ordinalpos=3518&amp;ncbi_uid=2721338&amp;link_uid=PMC2807333" href="/pmc/articles/PMC2807333/"><img src="/pmc/articles/instance/2807333/bin/zbc0051002020012.gif" src-large="/pmc/articles/instance/2807333/bin/zbc0051002020012.jpg" alt="FIGURE 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807333/" ref="ordinalpos=3518&amp;ncbi_uid=2721338&amp;link_uid=PMC2807333" image-link="/pmc/articles/PMC2807333/figure/F12/" class="imagepopup">FIGURE 12. From: Soluble Nogo Receptor Down-regulates Expression of Neuronal Nogo-A to Enhance Axonal Regeneration. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating Nogo-A expression in DRG neurons.</b></div></p></div><div class="aux"><div class="resc">Xiangmin Peng, et al. J Biol Chem. 2010 January 22;285(4):2783-2795.</div><p class="links"><a class="dblinks" href="/pubmed/19901030" ref="ordinalpos=3518&amp;ncbi_uid=2721338&amp;link_uid=19901030">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807333/" ref="ordinalpos=3518&amp;ncbi_uid=2721338&amp;link_uid=PMC2807333">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3519.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262193/figure/F3/" ref="ordinalpos=3519&amp;ncbi_uid=3897147&amp;link_uid=PMC3262193" href="/pmc/articles/PMC3262193/"><img src="/pmc/articles/instance/3262193/bin/bcr2921-3.gif" src-large="/pmc/articles/instance/3262193/bin/bcr2921-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262193/" ref="ordinalpos=3519&amp;ncbi_uid=3897147&amp;link_uid=PMC3262193" image-link="/pmc/articles/PMC3262193/figure/F3/" class="imagepopup">Figure 3.  From: Signal transducer and activator of transcription 5 as a key <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal mammary gland developmental biology and breast cancer. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> pathways and proteins that increase or decrease STAT5 <span class="highlight" style="background-color:">signaling</span> in normal mammary epithelial cells</b>. Research to date has identified several factors that contribute to higher levels of STAT5 activation or, conversely, reduce activation levels in normal mammary epithelial cells. E74-like factor 5 (ets domain transcription factor) (ELF5) acts in the PRL/JAK2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to increase levels of STAT5 activation in normal mammary epithelial cells. Other <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components that increase STAT5 activation are EGF, ErbB2, transforming growth factor-alpha (TGF-), estrogen acting with the progesterone <span class="highlight" style="background-color:">pathway</span> acting downstream, GH/JAK2, IGF, and c-Src. The TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can decrease STAT5 activation levels. Cellular proteins that contribute to increased STAT5 activation include PI 3-kinase enhancer A (PIKE-A), serine/threonine protein kinase Akt 1 (AKT-1), p21-activated kinase 1 (Pak1), the phosphotyrosine phosphatase Shp2, beta-integrin, and dystroglycan. Cellular proteins that have been shown to decrease STAT5 activation levels include caveolin-1 and suppressor of cytokine <span class="highlight" style="background-color:">signaling</span> 3 (SOCS-3). c-Src, human cellular-Src; EGF, epidermal growth factor; ErbB4, v-erb-b2 erythroblastic leukemia viral oncogene homolog 4, neuro/glioblastoma-derived oncogene homolog (avian); GH, growth hormone; IGF, insulin growth factor; JAK2, Janus kinase 2; PRL, prolactin; Shp2, Src homology region 2 domain-containing phosphatase-2; STAT5, signal transducer and activator of transcription 5.</div></p></div><div class="aux"><div class="resc">Priscilla A Furth, et al. Breast Cancer Res. 2011;13(5):220-220.</div><p class="links"><a class="dblinks" href="/pubmed/22018398" ref="ordinalpos=3519&amp;ncbi_uid=3897147&amp;link_uid=22018398">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262193/" ref="ordinalpos=3519&amp;ncbi_uid=3897147&amp;link_uid=PMC3262193">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3520.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3674457/figure/fig7/" ref="ordinalpos=3520&amp;ncbi_uid=4970550&amp;link_uid=PMC3674457" href="/pmc/articles/PMC3674457/"><img src="/pmc/articles/instance/3674457/bin/bcj201315f7.gif" src-large="/pmc/articles/instance/3674457/bin/bcj201315f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3674457/" ref="ordinalpos=3520&amp;ncbi_uid=4970550&amp;link_uid=PMC3674457" image-link="/pmc/articles/PMC3674457/figure/fig7/" class="imagepopup">Figure 7.  From: Lipid raft-mediated Akt <span class="highlight" style="background-color:">signaling</span> as a therapeutic target in mantle cell lymphoma. </a></p><div class="supp"><p class="details"><div>Schematic model of raft-mediated Akt <span class="highlight" style="background-color:">signaling</span> in the regulation of cell survival and death in MCL cells during edelfosine treatment. This scheme highlights the importance of lipid raft membrane domains for PI3K/Akt <span class="highlight" style="background-color:">pathway</span> activation in the regulation of MCL cell survival or death, based on the data reported here. For further details, see the text.</div></p></div><div class="aux"><div class="resc">M Reis-Sobreiro, et al. Blood Cancer J. 2013 May;3(5):e118.</div><p class="links"><a class="dblinks" href="/pubmed/23727661" ref="ordinalpos=3520&amp;ncbi_uid=4970550&amp;link_uid=23727661">Citation</a><a class="dblinks" href="/pmc/articles/PMC3674457/" ref="ordinalpos=3520&amp;ncbi_uid=4970550&amp;link_uid=PMC3674457">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3521.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3613938/figure/F1/" ref="ordinalpos=3521&amp;ncbi_uid=5670690&amp;link_uid=PMC3613938" href="/pmc/articles/PMC3613938/"><img src="/pmc/articles/instance/3613938/bin/JCI69324.f1.gif" src-large="/pmc/articles/instance/3613938/bin/JCI69324.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3613938/" ref="ordinalpos=3521&amp;ncbi_uid=5670690&amp;link_uid=PMC3613938" image-link="/pmc/articles/PMC3613938/figure/F1/" class="imagepopup">Figure 1. Canonical WNT <span class="highlight" style="background-color:">signaling</span> determines the proliferation and differentiation state of ESC-derived progenitors via full-length TCF7 (fTCF7)..  From: WNT <span class="highlight" style="background-color:">signaling</span> in stem cell differentiation and tumor formation. </a></p><div class="supp"><p class="details"><div>SOX2 and NESTIN act as direct targets of fTCF7 in vitro. Blocking the <span class="highlight" style="background-color:">pathway</span> by deleting TCF7 or treating cells with the WNT inhibitor DKK1 could reduce tumorigenicity and improve therapeutic efficiency. NTCF7, naturally truncated form of TCF7.</div></p></div><div class="aux"><div class="resc">Hong Ouyang, et al. J Clin Invest. 2013 April 1;123(4):1422-1424.</div><p class="links"><a class="dblinks" href="/pubmed/23524963" ref="ordinalpos=3521&amp;ncbi_uid=5670690&amp;link_uid=23524963">Citation</a><a class="dblinks" href="/pmc/articles/PMC3613938/" ref="ordinalpos=3521&amp;ncbi_uid=5670690&amp;link_uid=PMC3613938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3522.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2804351/figure/F4/" ref="ordinalpos=3522&amp;ncbi_uid=2184009&amp;link_uid=PMC2804351" href="/pmc/articles/PMC2804351/"><img src="/pmc/articles/instance/2804351/bin/zbc0021099310004.gif" src-large="/pmc/articles/instance/2804351/bin/zbc0021099310004.jpg" alt="FIGURE 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2804351/" ref="ordinalpos=3522&amp;ncbi_uid=2184009&amp;link_uid=PMC2804351" image-link="/pmc/articles/PMC2804351/figure/F4/" class="imagepopup">FIGURE 4. From: IRS1 Regulation by Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> and Varied Contribution of IRS1 to the Neoplastic Phenotype. </a></p><div class="supp"><p class="details"><div><b><i>IRS1</i> expression correlates with -catenin activation in ovarian endometrioid adenocarcinomas.</b> <i>A</i>, expression level of <i>IRS1</i> as assessed by Affymetrix GeneChip hybridization of ovarian endometroid adenocarcinomas (GSE6008) with or without nuclear -catenin staining. Significance was assessed using the two-sample <i>t</i> test of log-transformed values. <i>B</i>, immunohistochemical staining for IRS1 expression in ovarian endometrioid adenocarcinomas with or without the activated -catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Tumor samples were subdivided into samples showing inactivation of PTEN or not.</div></p></div><div class="aux"><div class="resc">Guido T. Bommer, et al. J Biol Chem. 2010 January 15;285(3):1928-1938.</div><p class="links"><a class="dblinks" href="/pubmed/19843521" ref="ordinalpos=3522&amp;ncbi_uid=2184009&amp;link_uid=19843521">Citation</a><a class="dblinks" href="/pmc/articles/PMC2804351/" ref="ordinalpos=3522&amp;ncbi_uid=2184009&amp;link_uid=PMC2804351">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3523.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3422314/figure/pone-0042127-g006/" ref="ordinalpos=3523&amp;ncbi_uid=4184988&amp;link_uid=PMC3422314" href="/pmc/articles/PMC3422314/"><img src="/pmc/articles/instance/3422314/bin/pone.0042127.g006.gif" src-large="/pmc/articles/instance/3422314/bin/pone.0042127.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3422314/" ref="ordinalpos=3523&amp;ncbi_uid=4184988&amp;link_uid=PMC3422314" image-link="/pmc/articles/PMC3422314/figure/pone-0042127-g006/" class="imagepopup">Figure 6. Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>..  From: Systems Biology Analysis of Gene Expression during In Vivo Mycobacterium avium paratuberculosis Enteric Colonization Reveals Role for Immune Tolerance. </a></p><div class="supp"><p class="details"><div>Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> Bayesian network representation at 30 min post-infection (left), and the DBGGA scores for the gene expression of MAP infected host Peyer's patch versus non-infected controls (right). Gene nodes with orange circles on the network are those defined as mechanistic genes that surpass a threshold |Bayesian z-score|&gt;2.24. The network shows gene nodes with gradient colors representing the level of expression (deeper red for higher up-regulated genes and deeper green for down-regulated). The heat map is colorized and corresponds to the gene node expression levels. Grey color represents little to no expression difference between MAP infected and controls. The heat map columns are by time post-infection in minutes.</div></p></div><div class="aux"><div class="resc">Sangeeta Khare, et al. PLoS One. 2012;7(8):e42127.</div><p class="links"><a class="dblinks" href="/pubmed/22912686" ref="ordinalpos=3523&amp;ncbi_uid=4184988&amp;link_uid=22912686">Citation</a><a class="dblinks" href="/pmc/articles/PMC3422314/" ref="ordinalpos=3523&amp;ncbi_uid=4184988&amp;link_uid=PMC3422314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3524.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4366388/figure/pone.0120133.g001/" ref="ordinalpos=3524&amp;ncbi_uid=7314472&amp;link_uid=PMC4366388" href="/pmc/articles/PMC4366388/"><img src="/pmc/articles/instance/4366388/bin/pone.0120133.g001.gif" src-large="/pmc/articles/instance/4366388/bin/pone.0120133.g001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4366388/" ref="ordinalpos=3524&amp;ncbi_uid=7314472&amp;link_uid=PMC4366388" image-link="/pmc/articles/PMC4366388/figure/pone.0120133.g001/" class="imagepopup">Fig 1. Functional diagrams of NIS and the PI3K/Akt/FoxO3a <span class="highlight" style="background-color:">pathway</span>..  From: Molecular Mechanisms of 2, 3, 4, 4, 5-Pentachlorobiphenyl-Induced Thyroid Dysfunction in FRTL-5 Cells. </a></p><div class="supp"><p class="details"><div>NIS is located on the outer membrane of the thyrocyte. Its function is to pump sodium out of the follicular cells while pumping iodide into the follicular cells (A); PCB118 stimulates cells, activates the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and increases Akt phosphorylation levels, after which p-Akt phosphorylates its downstream target gene FoxO3a (B).</div></p></div><div class="aux"><div class="resc">Hui Yang, et al. PLoS One. 2015;10(3):e0120133.</div><p class="links"><a class="dblinks" href="/pubmed/25789747" ref="ordinalpos=3524&amp;ncbi_uid=7314472&amp;link_uid=25789747">Citation</a><a class="dblinks" href="/pmc/articles/PMC4366388/" ref="ordinalpos=3524&amp;ncbi_uid=7314472&amp;link_uid=PMC4366388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3525.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC429373/figure/fig7/" ref="ordinalpos=3525&amp;ncbi_uid=415548&amp;link_uid=PMC429373" href="/pmc/articles/PMC429373/"><img src="/pmc/articles/instance/429373/bin/pp1350279f07.gif" src-large="/pmc/articles/instance/429373/bin/pp1350279f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC429373/" ref="ordinalpos=3525&amp;ncbi_uid=415548&amp;link_uid=PMC429373" image-link="/pmc/articles/PMC429373/figure/fig7/" class="imagepopup">Figure 7. From: Nitric Oxide Mediates the Indole Acetic Acid Induction Activation of a Mitogen-Activated Protein Kinase Cascade Involved in Adventitious Root Development. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the <span class="highlight" style="background-color:">signaling</span> networks involving IAA, NO, and cellular messengers that regulate ARF in cucumber. The auxin IAA triggers a transient NO accumulation (Pagnussat et al., 2002). In turn, NO regulates the activation of a MAPK <span class="highlight" style="background-color:">signaling</span> cascade. This <span class="highlight" style="background-color:">pathway</span> could be mediating both mitotic process and the expression of auxin-responsive genes (Mockaitis and Howell, 2000). In parallel, a cGMP-dependent <span class="highlight" style="background-color:">pathway</span> is also present leading to ARF (Pagnussat et al., 2003). NO-mediated <span class="highlight" style="background-color:">signaling</span> pathways that regulate ARF in cucumber: light gray, cGMP-independent; dark gray, cGMP-dependent.</div></p></div><div class="aux"><div class="resc">Gabriela Carolina Pagnussat, et al. Plant Physiol. 2004 May;135(1):279-286.</div><p class="links"><a class="dblinks" href="/pubmed/15122018" ref="ordinalpos=3525&amp;ncbi_uid=415548&amp;link_uid=15122018">Citation</a><a class="dblinks" href="/pmc/articles/PMC429373/" ref="ordinalpos=3525&amp;ncbi_uid=415548&amp;link_uid=PMC429373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3526.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3342359/figure/F2/" ref="ordinalpos=3526&amp;ncbi_uid=3988879&amp;link_uid=PMC3342359" href="/pmc/articles/PMC3342359/"><img src="/pmc/articles/instance/3342359/bin/fimmu-03-00044-g002.gif" src-large="/pmc/articles/instance/3342359/bin/fimmu-03-00044-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3342359/" ref="ordinalpos=3526&amp;ncbi_uid=3988879&amp;link_uid=PMC3342359" image-link="/pmc/articles/PMC3342359/figure/F2/" class="imagepopup">Figure 2.  From: Modulation of Pathogen Recognition by Autophagy. </a></p><div class="supp"><p class="details"><div><b>Autophagy induction by TLR stimulation</b>. Activation of TLR can induce autophagy for pathogen elimination. Stimulation of TLR4 with LPS in a TRIF-dependent, MyD88-independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and stimulation of endosomal TLR7 with two different TLR7 ligands via the MyD88-dependent <span class="highlight" style="background-color:">pathway</span> induces autophagosome formation and eliminates <i>Mycobacteria bacilli</i>, even though mycobacteria are normally not associated with TLR7 <span class="highlight" style="background-color:">signaling</span>. Atg proteins such as Atg5, Beclin-1, and PI3K are required for the formation of an autophagosome.</div></p></div><div class="aux"><div class="resc">Ji Eun Oh, et al. Front Immunol. 2012;3:44.</div><p class="links"><a class="dblinks" href="/pubmed/22566926" ref="ordinalpos=3526&amp;ncbi_uid=3988879&amp;link_uid=22566926">Citation</a><a class="dblinks" href="/pmc/articles/PMC3342359/" ref="ordinalpos=3526&amp;ncbi_uid=3988879&amp;link_uid=PMC3342359">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3527.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2951025/figure/F8/" ref="ordinalpos=3527&amp;ncbi_uid=2508148&amp;link_uid=PMC2951025" href="/pmc/articles/PMC2951025/"><img src="/pmc/articles/instance/2951025/bin/nihms85976f8.gif" src-large="/pmc/articles/instance/2951025/bin/nihms85976f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2951025/" ref="ordinalpos=3527&amp;ncbi_uid=2508148&amp;link_uid=PMC2951025" image-link="/pmc/articles/PMC2951025/figure/F8/" class="imagepopup">Figure 8.  From: Dynamic Regulation of the Kv2.1 Voltage-gated Potassium Channel during Brain Ischemia through Neuroglial Interaction. </a></p><div class="supp"><p class="details"><div>Schematic diagram of our model. During brain ischemia, the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> shown in red takes place immediately (&lt;5 min), and then later, the left-hand <span class="highlight" style="background-color:">pathway</span> will also be driven to cause Kv2.1 modulation. They both ultimately induce the increase in the intracellular Ca<sup>2+</sup> level [Ca<sup>2+</sup>]i, followed by Kv2.1 modulation (dephosphorylation, dispersion, and activation). The texts in the parentheses indicates the experimental supports of the <span class="highlight" style="background-color:">signaling</span> pathways. See Discussion for more detail.</div></p></div><div class="aux"><div class="resc">Hiroaki Misonou, et al. J Neurosci. ;28(34):8529-8538.</div><p class="links"><a class="dblinks" href="/pubmed/18716211" ref="ordinalpos=3527&amp;ncbi_uid=2508148&amp;link_uid=18716211">Citation</a><a class="dblinks" href="/pmc/articles/PMC2951025/" ref="ordinalpos=3527&amp;ncbi_uid=2508148&amp;link_uid=PMC2951025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3528.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC60280/figure/F8/" ref="ordinalpos=3528&amp;ncbi_uid=138638&amp;link_uid=PMC60280" href="/pmc/articles/PMC60280/"><img src="/pmc/articles/instance/60280/bin/mk1111644008.gif" src-large="/pmc/articles/instance/60280/bin/mk1111644008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC60280/" ref="ordinalpos=3528&amp;ncbi_uid=138638&amp;link_uid=PMC60280" image-link="/pmc/articles/PMC60280/figure/F8/" class="imagepopup">Figure 8.  From: Tumor Necrosis Factor-? Induces Stress Fiber Formation through Ceramide Production: Role of Sphingosine Kinase. </a></p><div class="supp"><p class="details"><div>Proposed scheme for the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of TNF- leading to stress fiber formation. The figure illustrates how ceramide formation can contribute to the <span class="highlight" style="background-color:">signaling</span> pathways whereby TNF- induces the formation of stress fibers. The ceramide-mediated <span class="highlight" style="background-color:">pathway</span> depends upon the formation of Ras-GTP and the activations of PI 3-K and SK. In addition, TNF- also induces the tyrosine phosphorylations of FAK and paxillin and stress fiber formation independently of the activation of Ras and PI 3-K.</div></p></div><div class="aux"><div class="resc">Atef N. Hanna, et al. Mol Biol Cell. 2001 November;12(11):3618-3630.</div><p class="links"><a class="dblinks" href="/pubmed/11694593" ref="ordinalpos=3528&amp;ncbi_uid=138638&amp;link_uid=11694593">Citation</a><a class="dblinks" href="/pmc/articles/PMC60280/" ref="ordinalpos=3528&amp;ncbi_uid=138638&amp;link_uid=PMC60280">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3529.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2528845/figure/F4/" ref="ordinalpos=3529&amp;ncbi_uid=1571032&amp;link_uid=PMC2528845" href="/pmc/articles/PMC2528845/"><img src="/pmc/articles/instance/2528845/bin/zef0020826790004.gif" src-large="/pmc/articles/instance/2528845/bin/zef0020826790004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2528845/" ref="ordinalpos=3529&amp;ncbi_uid=1571032&amp;link_uid=PMC2528845" image-link="/pmc/articles/PMC2528845/figure/F4/" class="imagepopup">Figure 4.  From: Cellular <span class="highlight" style="background-color:">Signaling</span> by Fibroblast Growth Factors (FGFs) and Their Receptors (FGFRs) in Male Reproduction. </a></p><div class="supp"><p class="details"><div>Opposing signals regulate sex determination in the bipotential gonad. In both XX and XY gonads at 10.511.5 d postcoitum (dpc), <i>Fgf9</i> transcripts (<i>blue</i>) are detected near the gonadal surface, whereas <i>Wnt4</i> transcripts (<i>pink</i>) are detected near the gonad mesonephric boundary (123). At this time, in the bipotential gonad, there is a balance between these two competing signals. A genetic or environmental switch initiates the male <span class="highlight" style="background-color:">pathway</span> by creating an imbalance between these signals. <i>Sry</i> is activated in the XY gonad, and its expression diverts the XY gonad toward a testis-specific fate. <i>Sry</i> up-regulates <i>Sox9</i>, which has been implicated in the early steps of the male-specific <span class="highlight" style="background-color:">pathway</span> (121,123). <i>Sox9</i> then up-regulates <i>Fgf9</i>, and <i>Fgf9</i> (through FGFR2) maintains <i>Sox9</i>, forming a positive feedback loop in XY gonads (125). In this circumstance, the balance between FGF9 and Wnt4 signals is shifted in favor of FGF9, silencing Wnt4 <span class="highlight" style="background-color:">signaling</span>, and the dominance of the male <span class="highlight" style="background-color:">pathway</span> is established. In the absence of a feed-forward loop between SOX9 and FGF9 (<i>e</i>.<i>g</i>., in XX gonads), WNT4 suppresses <i>Fgf9</i> transcription, initiating the female differentiation <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Leanne M. Cotton, et al. Endocr Rev. 2008 April;29(2):193-216.</div><p class="links"><a class="dblinks" href="/pubmed/18216218" ref="ordinalpos=3529&amp;ncbi_uid=1571032&amp;link_uid=18216218">Citation</a><a class="dblinks" href="/pmc/articles/PMC2528845/" ref="ordinalpos=3529&amp;ncbi_uid=1571032&amp;link_uid=PMC2528845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3530.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3924561/figure/F1/" ref="ordinalpos=3530&amp;ncbi_uid=5818612&amp;link_uid=PMC3924561" href="/pmc/articles/PMC3924561/"><img src="/pmc/articles/instance/3924561/bin/nihms-550516-f0001.gif" src-large="/pmc/articles/instance/3924561/bin/nihms-550516-f0001.jpg" alt="Fig. (1)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3924561/" ref="ordinalpos=3530&amp;ncbi_uid=5818612&amp;link_uid=PMC3924561" image-link="/pmc/articles/PMC3924561/figure/F1/" class="imagepopup">Fig. (1).  From: Functional Relevance of Biased <span class="highlight" style="background-color:">Signaling</span> at the Angiotensin II Type 1 Receptor. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Ang II-mediated stimulation of AT1R leads to the activation of both G<sub>q</sub> protein- and -arrestin-mediated <span class="highlight" style="background-color:">signaling</span> pathways, each of which can induce protein synthesis through the activation of the ERK1/2-Mnk1-eIF4E <span class="highlight" style="background-color:">pathway</span>, though it is not known if the effects of each <span class="highlight" style="background-color:">pathway</span> are equivalent. Sequestration of ERK1/2 in the cytosol by -arrestins prevents nuclear targeting and effects on gene expression and promotes anti-apoptotic <span class="highlight" style="background-color:">signaling</span> <i>via</i> p90RSK regulation of intracellular BAD targeting. -arrestins 1 and 2 each interact with a number of proteins involved in receptor internalization, including c-Src, AP-2 and ARNO, which promote AT1R endocytosis following stimulation. Interaction with AIP4 promotes ubiquitination of TRPV4, a nonselective cation channel, promoting its internalization in conjunction with AT1R. A possible -arrestin-Gaf-1-Rab11 complex may promote trafficking of internalized AT1R to the plasma membrane. (<b>B</b>) AT1R stimulation mediates hypertrophy <i>via</i> distinct mechanisms including transactivation of EGFR <i>via</i> either G<sub>q</sub> protein- or -arrestin-mediated activation of the c-Src-MMP-HB-EGF shedding <span class="highlight" style="background-color:">pathway</span>, as well as G<sub>q</sub> protein-mediated activation of PKC <span class="highlight" style="background-color:">signaling</span>. Each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> results in ERK1/2 activation and an increase in hypertrophy, though the relative contributions of each <span class="highlight" style="background-color:">pathway</span> to the hypertrophic response in different cell types and pathological states, and whether they are equivalent responses, is unknown. (<b>C</b>) Activation of AT1R leads to G<sub>q</sub> protein-mediated increases in contractility <i>via</i> modulation of intracellular Ca<sup>2+</sup> levels and PKC <span class="highlight" style="background-color:">signaling</span>. AT1R--arrestin-mediated <span class="highlight" style="background-color:">signaling</span> leads to activation of RhoA and ROCK, which subsequently contributes to stress fiber reorganization and membrane blebbing in HEK 293 cells. Whether these processes mediate contractility is not yet known. -arrestins 1 and 2 each interact with a number of other proteins involved in cytoskeletal regulation, including actin, cofilin, PP2A and slingshot, which though coordinated action can lead to actin depolymerization and possible effects on contractility following AT1R stimulation.</div></p></div><div class="aux"><div class="resc">Douglas G. Tilley. Endocr Metab Immune Disord Drug Targets. ;11(2):99-111.</div><p class="links"><a class="dblinks" href="/pubmed/21476968" ref="ordinalpos=3530&amp;ncbi_uid=5818612&amp;link_uid=21476968">Citation</a><a class="dblinks" href="/pmc/articles/PMC3924561/" ref="ordinalpos=3530&amp;ncbi_uid=5818612&amp;link_uid=PMC3924561">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3531.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4243459/figure/F2/" ref="ordinalpos=3531&amp;ncbi_uid=6879705&amp;link_uid=PMC4243459" href="/pmc/articles/PMC4243459/"><img src="/pmc/articles/instance/4243459/bin/nihms643248f2.gif" src-large="/pmc/articles/instance/4243459/bin/nihms643248f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4243459/" ref="ordinalpos=3531&amp;ncbi_uid=6879705&amp;link_uid=PMC4243459" image-link="/pmc/articles/PMC4243459/figure/F2/" class="imagepopup">Fig. 2.  From: Understanding Modularity in Molecular Networks Requires Dynamics. </a></p><div class="supp"><p class="details"><div>Dynamical constraints on a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in tissue differentiation in rat correlate with allelic variation at the population level in the same <span class="highlight" style="background-color:">pathway</span> in fruit flies, revealing a link between dynamics and evolution. The stiffest combination of parameters was associated with the Ras and Raf proteins at the top of the <span class="highlight" style="background-color:">pathway</span>, and in fruit flies Ras and Raf exhibit the least allelic variation of all of the components of the <span class="highlight" style="background-color:">pathway</span>, consistent with the concept that stiff parameter combinations are constrained during evolution. The dotted box indicates that some accessory components of the two cascades are excluded here for clarity of presentation.</div></p></div><div class="aux"><div class="resc">Roger P. Alexander, et al. Sci Signal. ;2(81):pe44-pe44.</div><p class="links"><a class="dblinks" href="/pubmed/19638611" ref="ordinalpos=3531&amp;ncbi_uid=6879705&amp;link_uid=19638611">Citation</a><a class="dblinks" href="/pmc/articles/PMC4243459/" ref="ordinalpos=3531&amp;ncbi_uid=6879705&amp;link_uid=PMC4243459">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3532.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3876277/figure/F1/" ref="ordinalpos=3532&amp;ncbi_uid=5641774&amp;link_uid=PMC3876277" href="/pmc/articles/PMC3876277/"><img src="/pmc/articles/instance/3876277/bin/nihms533252f1.gif" src-large="/pmc/articles/instance/3876277/bin/nihms533252f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3876277/" ref="ordinalpos=3532&amp;ncbi_uid=5641774&amp;link_uid=PMC3876277" image-link="/pmc/articles/PMC3876277/figure/F1/" class="imagepopup">Figure 1.  From: Preclinical Activity of the Rational Combination of Selumetinib (AZD6244) in Combination with Vorinostat in KRAS-Mutant Colorectal Cancer Models. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of CRC cell lines at baseline and after treatment with vorinostat, selumetinib, or the combination. A, top 5 pathways enriched in the vorinostat-resistant cell lines (HT29 and SW480) after vorinostat treatment. B, heatmap of the core genes in the HDAC and MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at baseline, after treatment with vorinostat, selumetinib, and the combination. <span class="highlight" style="background-color:">Pathway</span> diagram derived from BioCarta where the core genes are circled. Red and green in the heatmap represent gene overexpression and underexpression, respectively. C, <span class="highlight" style="background-color:">pathway</span> diagram derived from BioCarta where the core genes are colored in red.</div></p></div><div class="aux"><div class="resc">M. Pia Morelli, et al. Clin Cancer Res. ;18(4):10.1158/1078-0432.CCR-11-1507.</div><p class="links"><a class="dblinks" href="/pubmed/22173548" ref="ordinalpos=3532&amp;ncbi_uid=5641774&amp;link_uid=22173548">Citation</a><a class="dblinks" href="/pmc/articles/PMC3876277/" ref="ordinalpos=3532&amp;ncbi_uid=5641774&amp;link_uid=PMC3876277">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3533.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3337355/figure/F2/" ref="ordinalpos=3533&amp;ncbi_uid=4853467&amp;link_uid=PMC3337355" href="/pmc/articles/PMC3337355/"><img src="/pmc/articles/instance/3337355/bin/nihms363687f2.gif" src-large="/pmc/articles/instance/3337355/bin/nihms363687f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3337355/" ref="ordinalpos=3533&amp;ncbi_uid=4853467&amp;link_uid=PMC3337355" image-link="/pmc/articles/PMC3337355/figure/F2/" class="imagepopup">Fig. 2.  From: Tumor suppressor function of RUNX3 in breast cancer. </a></p><div class="supp"><p class="details"><div>RUNX3 integrates with various cellular <span class="highlight" style="background-color:">signaling</span> pathways for its tumor suppressor activity. (A) RUNX3 suppresses ER <span class="highlight" style="background-color:">signaling</span> in breast cancer. Binding of RUNX3 to ER induces the ubiquitination and degradation of ER by an unidentified E3 ligase (X). The low levels of ER maintain the normal growth of mammary epithelial cells. When RUNX3 is inactivated, the cellular levels of ER are enhanced, leading to the enhanced response to estradiol (E<sub>2</sub>), the hyperproliferation of breast cells, and eventually the formation of breast cancer. (B) RUNX3 could either act as a downstream target of a tumor suppressor <span class="highlight" style="background-color:">pathway</span> (e.g, TGF- <span class="highlight" style="background-color:">signaling</span>) or function as a suppressor in an oncogenic <span class="highlight" style="background-color:">pathway</span> (e.g., Wnt <span class="highlight" style="background-color:">signaling</span>) to regulate cell proliferation, apoptosis, and cell migration. By interacting with various <span class="highlight" style="background-color:">signaling</span> pathways, RUNX3 might also have a role in breast cancer carcinogenesis by regulating the EMT and breast cancer stem cell development.</div></p></div><div class="aux"><div class="resc">Lin-Feng Chen. J Cell Biochem. ;113(5):1470-1477.</div><p class="links"><a class="dblinks" href="/pubmed/22275124" ref="ordinalpos=3533&amp;ncbi_uid=4853467&amp;link_uid=22275124">Citation</a><a class="dblinks" href="/pmc/articles/PMC3337355/" ref="ordinalpos=3533&amp;ncbi_uid=4853467&amp;link_uid=PMC3337355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3534.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3498304/figure/pone-0048299-g009/" ref="ordinalpos=3534&amp;ncbi_uid=4371297&amp;link_uid=PMC3498304" href="/pmc/articles/PMC3498304/"><img src="/pmc/articles/instance/3498304/bin/pone.0048299.g009.gif" src-large="/pmc/articles/instance/3498304/bin/pone.0048299.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3498304/" ref="ordinalpos=3534&amp;ncbi_uid=4371297&amp;link_uid=PMC3498304" image-link="/pmc/articles/PMC3498304/figure/pone-0048299-g009/" class="imagepopup">Figure 9. Effects of <span class="highlight" style="background-color:">pathway</span> specific kinase inhibitors on <span class="highlight" style="background-color:">signaling</span> pathways involved in the TGF actions in KGN cells..  From: Transforming Growth Factor Alpha (TGF?) Regulates Granulosa Cell Tumor (GCT) Cell Proliferation and Migration through Activation of Multiple Pathways. </a></p><div class="supp"><p class="details"><div>KGN cells were pretreated with or without <span class="highlight" style="background-color:">pathway</span> inhibitors for 3 hours before treatment with TGF for 10 minutes. Phosphorylated and non-phosphorylated proteins were detected by Western blot. Results are representative of three separate experiments.</div></p></div><div class="aux"><div class="resc">Cheng Wang, et al. PLoS One. 2012;7(11):e48299.</div><p class="links"><a class="dblinks" href="/pubmed/23155381" ref="ordinalpos=3534&amp;ncbi_uid=4371297&amp;link_uid=23155381">Citation</a><a class="dblinks" href="/pmc/articles/PMC3498304/" ref="ordinalpos=3534&amp;ncbi_uid=4371297&amp;link_uid=PMC3498304">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3535.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3880276/figure/pone-0083908-g009/" ref="ordinalpos=3535&amp;ncbi_uid=5687090&amp;link_uid=PMC3880276" href="/pmc/articles/PMC3880276/"><img src="/pmc/articles/instance/3880276/bin/pone.0083908.g009.gif" src-large="/pmc/articles/instance/3880276/bin/pone.0083908.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3880276/" ref="ordinalpos=3535&amp;ncbi_uid=5687090&amp;link_uid=PMC3880276" image-link="/pmc/articles/PMC3880276/figure/pone-0083908-g009/" class="imagepopup">Figure 9. PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Rapamycin Ameliorates Inflammation and Fibrosis in the Early Phase of Cirrhotic Portal Hypertension in Rats through Inhibition of mTORC1 but Not mTORC2. </a></p><div class="supp"><p class="details"><div>Rapamycin bound to FKBP12 mainly blocked the activity of mTORC1.</div></p></div><div class="aux"><div class="resc">Weijie Wang, et al. PLoS One. 2014;9(1):e83908.</div><p class="links"><a class="dblinks" href="/pubmed/24404143" ref="ordinalpos=3535&amp;ncbi_uid=5687090&amp;link_uid=24404143">Citation</a><a class="dblinks" href="/pmc/articles/PMC3880276/" ref="ordinalpos=3535&amp;ncbi_uid=5687090&amp;link_uid=PMC3880276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3536.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3748845/figure/f1-drp-11-301/" ref="ordinalpos=3536&amp;ncbi_uid=5230956&amp;link_uid=PMC3748845" href="/pmc/articles/PMC3748845/"><img src="/pmc/articles/instance/3748845/bin/drp-11-301f1.gif" src-large="/pmc/articles/instance/3748845/bin/drp-11-301f1.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3748845/" ref="ordinalpos=3536&amp;ncbi_uid=5230956&amp;link_uid=PMC3748845" image-link="/pmc/articles/PMC3748845/figure/f1-drp-11-301/" class="imagepopup">FIGURE 1. From: Systems Cancer Biology and the Controlling Mechanisms for the J-Shaped Cancer Dose Response: Towards Relaxing the LNT Hypothesis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of checkpoint control and IR-induced perturbation</div></p></div><div class="aux"><div class="resc">In Chio Lou, et al. Dose Response. 2013;11(3):301-318.</div><p class="links"><a class="dblinks" href="/pubmed/23983661" ref="ordinalpos=3536&amp;ncbi_uid=5230956&amp;link_uid=23983661">Citation</a><a class="dblinks" href="/pmc/articles/PMC3748845/" ref="ordinalpos=3536&amp;ncbi_uid=5230956&amp;link_uid=PMC3748845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3537.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1603783/figure/F5-6649/" ref="ordinalpos=3537&amp;ncbi_uid=877846&amp;link_uid=PMC1603783" href="/pmc/articles/PMC1603783/"><img src="/pmc/articles/instance/1603783/bin/zjh0110566490005.gif" src-large="/pmc/articles/instance/1603783/bin/zjh0110566490005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1603783/" ref="ordinalpos=3537&amp;ncbi_uid=877846&amp;link_uid=PMC1603783" image-link="/pmc/articles/PMC1603783/figure/F5-6649/" class="imagepopup">Figure 5.  From: In Situ Analysis of Integrin and Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span> Pathways in Human Glioblastomas Suggests Overlapping Relationships with Focal Adhesion Kinase Activation. </a></p><div class="supp"><p class="details"><div>Schematic of the integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in glioblastoma. Overall FAK activation may be primarily accomplished by direct GFR <span class="highlight" style="background-color:">signaling</span>. However, in certain morphologically defined regions (perivascular, perinecrotic, tumor infiltration zone), in which extracellular matrix components (ECM) are present, integrins are up-regulated in a GFR-overexpressed background and might enhance FAK and ERK1/2 phosphorylation, potentially resulting in increased motility and transcription.</div></p></div><div class="aux"><div class="resc">Markus J. Riemenschneider, et al. Am J Pathol. 2005 November;167(5):1379-1387.</div><p class="links"><a class="dblinks" href="/pubmed/16251422" ref="ordinalpos=3537&amp;ncbi_uid=877846&amp;link_uid=16251422">Citation</a><a class="dblinks" href="/pmc/articles/PMC1603783/" ref="ordinalpos=3537&amp;ncbi_uid=877846&amp;link_uid=PMC1603783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3538.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3877975/figure/F5/" ref="ordinalpos=3538&amp;ncbi_uid=5648703&amp;link_uid=PMC3877975" href="/pmc/articles/PMC3877975/"><img src="/pmc/articles/instance/3877975/bin/1743-422X-10-361-5.gif" src-large="/pmc/articles/instance/3877975/bin/1743-422X-10-361-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3877975/" ref="ordinalpos=3538&amp;ncbi_uid=5648703&amp;link_uid=PMC3877975" image-link="/pmc/articles/PMC3877975/figure/F5/" class="imagepopup">Figure 5.  From: Transcriptome analysis of primary monocytes from HIV-positive patients with differential responses to antiretroviral therapy. </a></p><div class="supp"><p class="details"><div><b>Coordinately up</b>-<b>regulated genes of chemokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in monocytes from the VIR group (VIR versus BDL).</b> The <span class="highlight" style="background-color:">pathway</span> figure is adapted from Kyoto Encyclopedia of Genes and Genomes (KEGG; http://www.genome.jp/kegg/). The proteins encoded by the coordinately up-regulated genes in the VIR group are highlighted in red.</div></p></div><div class="aux"><div class="resc">Jing Qin Wu, et al. Virol J. 2013;10:361-361.</div><p class="links"><a class="dblinks" href="/pubmed/24370116" ref="ordinalpos=3538&amp;ncbi_uid=5648703&amp;link_uid=24370116">Citation</a><a class="dblinks" href="/pmc/articles/PMC3877975/" ref="ordinalpos=3538&amp;ncbi_uid=5648703&amp;link_uid=PMC3877975">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3539.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3428242/figure/f1-ott-5-153/" ref="ordinalpos=3539&amp;ncbi_uid=4210906&amp;link_uid=PMC3428242" href="/pmc/articles/PMC3428242/"><img src="/pmc/articles/instance/3428242/bin/ott-5-153f1.gif" src-large="/pmc/articles/instance/3428242/bin/ott-5-153f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3428242/" ref="ordinalpos=3539&amp;ncbi_uid=4210906&amp;link_uid=PMC3428242" image-link="/pmc/articles/PMC3428242/figure/f1-ott-5-153/" class="imagepopup">Figure 1.  From: Treating metastatic soft-tissue or bone sarcomas - potential role of ridaforolimus. </a></p><div class="supp"><p class="details"><div>Simplified schematic representation of the mammalian target of rapamycin (mTOR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.<br /><b>Notes:</b> Red, <span class="highlight" style="background-color:">pathway</span> inhibitor; Green, <span class="highlight" style="background-color:">pathway</span> activator; Blue, mTOR complex protein.<br /><b>Abbreviation:</b> NOS, not otherwise specified.</div></p></div><div class="aux"><div class="resc">Vicki L Keedy. Onco Targets Ther. 2012;5:153-160.</div><p class="links"><a class="dblinks" href="/pubmed/22942649" ref="ordinalpos=3539&amp;ncbi_uid=4210906&amp;link_uid=22942649">Citation</a><a class="dblinks" href="/pmc/articles/PMC3428242/" ref="ordinalpos=3539&amp;ncbi_uid=4210906&amp;link_uid=PMC3428242">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3540.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3629253/figure/f5-ol-05-04-1395/" ref="ordinalpos=3540&amp;ncbi_uid=4814110&amp;link_uid=PMC3629253" href="/pmc/articles/PMC3629253/"><img src="/pmc/articles/instance/3629253/bin/OL-05-04-1395-g04.gif" src-large="/pmc/articles/instance/3629253/bin/OL-05-04-1395-g04.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3629253/" ref="ordinalpos=3540&amp;ncbi_uid=4814110&amp;link_uid=PMC3629253" image-link="/pmc/articles/PMC3629253/figure/f5-ol-05-04-1395/" class="imagepopup">Figure 5.  From: Effects of oleic acid on cell proliferation through an integrin-linked kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 786-O renal cell carcinoma cells. </a></p><div class="supp"><p class="details"><div>Schematic representation of oleic acid <span class="highlight" style="background-color:">signaling</span> in human renal cell carcinoma (RCC). Oleic acid activates the Akt <span class="highlight" style="background-color:">pathway</span> through stimulation of ILK, identified as one of the kinases with PDK-2 activity in human RCC. The mechanisms involved in this action are not well understood but may be GPR40-dependent. FFA, free fatty acid; GPR40, G protein-coupled receptor 40; ILK, integrin-linked kinase; PTEN, phosphatase and tensin homolog deleted on chromosome 10; P(4,5)P<sub>2</sub>, phosphatidylinositol-4,5-bisphosphate; P(3,4,5)P<sub>3</sub>, phosphatidylinositol-3,4,5-trisphosphate.</div></p></div><div class="aux"><div class="resc">ZHENHUA LIU, et al. Oncol Lett. 2013 April;5(4):1395-1399.</div><p class="links"><a class="dblinks" href="/pubmed/23599801" ref="ordinalpos=3540&amp;ncbi_uid=4814110&amp;link_uid=23599801">Citation</a><a class="dblinks" href="/pmc/articles/PMC3629253/" ref="ordinalpos=3540&amp;ncbi_uid=4814110&amp;link_uid=PMC3629253">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3541.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2854204/figure/F2/" ref="ordinalpos=3541&amp;ncbi_uid=2858628&amp;link_uid=PMC2854204" href="/pmc/articles/PMC2854204/"><img src="/pmc/articles/instance/2854204/bin/nihms160856f2.gif" src-large="/pmc/articles/instance/2854204/bin/nihms160856f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2854204/" ref="ordinalpos=3541&amp;ncbi_uid=2858628&amp;link_uid=PMC2854204" image-link="/pmc/articles/PMC2854204/figure/F2/" class="imagepopup">Figure 2. Mechanisms of feedback <span class="highlight" style="background-color:">signaling</span> in the TOR <span class="highlight" style="background-color:">pathway</span>.  From: TOR-dependent control of autophagy: biting the hand that feeds. </a></p><div class="supp"><p class="details"><div>(a) Negative feedback mechanisms that limit activation of autophagy are shown in red. Protein degradation within the autolysosome raises the intracellular levels of amino acids, leading to activation of TOR through the RagA/RagC small GTPases and other nutrient <span class="highlight" style="background-color:">signaling</span> pathways. TOR activity is self-limited through an S6K-mediated inhibition of IRS1 (not shown), an essential component in the insulin/PI3K/Akt <span class="highlight" style="background-color:">pathway</span> upstream of Rheb. S6K is also involved in a self-limiting <span class="highlight" style="background-color:">pathway</span> in autophagy induction downstream of TOR, probably through its function in protein synthesis. Inhibition of TOR leads to both activation of Atg1 and inactivation of S6K; both of these kinases play positive roles in starvation-induced autophagy. (b) Feed-forward mechanisms in TOR <span class="highlight" style="background-color:">signaling</span>, indicated by green lines, may lead to amplification and stabilization of initially modest changes in TOR activity. DEPTOR, PRAS40 and Atg1 are each inhibited by TOR mediated phosphorylation and in turn inhibit TOR activity. In yeast, TOR mediates activation of Tap42, which leads to downregulation of the phosphatase Sit4 and decreased activity of the Tap42 inhibitor Tip41, resulting in further activation of Tap42.</div></p></div><div class="aux"><div class="resc">Thomas P. Neufeld. Curr Opin Cell Biol. ;22(2):157-168.</div><p class="links"><a class="dblinks" href="/pubmed/20006481" ref="ordinalpos=3541&amp;ncbi_uid=2858628&amp;link_uid=20006481">Citation</a><a class="dblinks" href="/pmc/articles/PMC2854204/" ref="ordinalpos=3541&amp;ncbi_uid=2858628&amp;link_uid=PMC2854204">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3542.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2875624/figure/F1/" ref="ordinalpos=3542&amp;ncbi_uid=2390260&amp;link_uid=PMC2875624" href="/pmc/articles/PMC2875624/"><img src="/pmc/articles/instance/2875624/bin/ar2896-1.gif" src-large="/pmc/articles/instance/2875624/bin/ar2896-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2875624/" ref="ordinalpos=3542&amp;ncbi_uid=2390260&amp;link_uid=PMC2875624" image-link="/pmc/articles/PMC2875624/figure/F1/" class="imagepopup">Figure 1.  From: A role for age-related changes in TGF? <span class="highlight" style="background-color:">signaling</span> in aberrant chondrocyte differentiation and osteoarthritis. </a></p><div class="supp"><p class="details"><div><b>Alterations in transforming growth factor beta <span class="highlight" style="background-color:">signaling</span> cause changes in chondrocyte differentiation and osteoarthritis development</b>. Transforming growth factor beta (TGF) can either signal by the Smad2/3 route (canonical) or the Smad1/5/8 route. Smad2/3 and Smad1/5/8 form a complex with Smad4 that enters the nucleus and modulates gene expression and Runx2 function. The <span class="highlight" style="background-color:">signaling</span> by Smad2/3 and Smad1/5/8 is differentially modified by a number of intracellular molecules. Both Smad routes are blocked by Smad7, while Smad6 blocks preferentially the Smad1/5/8 <span class="highlight" style="background-color:">pathway</span> [100,101]. wnt <span class="highlight" style="background-color:">signaling</span> modifies these pathways by stabilization of Smad1/5/8 [102]. Smurf1 and Smurf2 are E3 ubiquitin ligases that inhibit Smad <span class="highlight" style="background-color:">signaling</span>. Smurf1 triggers the degradation of Smad1/5/8 while Smurf2 stimulates mainly the degradation of Smad2/3 [103]. Mitogen-activated protein kinases (MAPKs) modulate the stability and degradation of the Smads by phosphorylation of these molecules [102].</div></p></div><div class="aux"><div class="resc">Peter M van der Kraan, et al. Arthritis Res Ther. 2010;12(1):201-201.</div><p class="links"><a class="dblinks" href="/pubmed/20156325" ref="ordinalpos=3542&amp;ncbi_uid=2390260&amp;link_uid=20156325">Citation</a><a class="dblinks" href="/pmc/articles/PMC2875624/" ref="ordinalpos=3542&amp;ncbi_uid=2390260&amp;link_uid=PMC2875624">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3543.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3819748/figure/F7/" ref="ordinalpos=3543&amp;ncbi_uid=5470356&amp;link_uid=PMC3819748" href="/pmc/articles/PMC3819748/"><img src="/pmc/articles/instance/3819748/bin/1471-2121-14-44-7.gif" src-large="/pmc/articles/instance/3819748/bin/1471-2121-14-44-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3819748/" ref="ordinalpos=3543&amp;ncbi_uid=5470356&amp;link_uid=PMC3819748" image-link="/pmc/articles/PMC3819748/figure/F7/" class="imagepopup">Figure 7.  From: Physiological ?-catenin <span class="highlight" style="background-color:">signaling</span> controls self-renewal networks and generation of stem-like cells from nasopharyngeal carcinoma. </a></p><div class="supp"><p class="details"><div><b>Analyses of spheres derived from HONE1 hybrid cells. A</b>. RT-PCR analyses show that the expression of genes that encode CSC markers, CD9, CD24, CD44, CD90, and CD133, in sphere-forming cells (spheres <b>A</b>, <b>B</b>, and <b>C</b>) are progressively up-regulated compared to HONE1 and HONE1 hybrid cells. <b>B</b>. and <b>C</b>. Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> qPCR Array assay lists the 34 up-regulated genes and 12 down-regulated genes in spheres, compared to HONE1 hybrid MCH4.4.</div></p></div><div class="aux"><div class="resc">Yue Cheng, et al. BMC Cell Biol. 2013;14:44-44.</div><p class="links"><a class="dblinks" href="/pubmed/24073846" ref="ordinalpos=3543&amp;ncbi_uid=5470356&amp;link_uid=24073846">Citation</a><a class="dblinks" href="/pmc/articles/PMC3819748/" ref="ordinalpos=3543&amp;ncbi_uid=5470356&amp;link_uid=PMC3819748">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3544.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3697618/figure/F7/" ref="ordinalpos=3544&amp;ncbi_uid=5670164&amp;link_uid=PMC3697618" href="/pmc/articles/PMC3697618/"><img src="/pmc/articles/instance/3697618/bin/zii9990901990007.gif" src-large="/pmc/articles/instance/3697618/bin/zii9990901990007.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3697618/" ref="ordinalpos=3544&amp;ncbi_uid=5670164&amp;link_uid=PMC3697618" image-link="/pmc/articles/PMC3697618/figure/F7/" class="imagepopup">Fig 7.  From: Toll-Like Receptor 4-Linked Janus Kinase 2 <span class="highlight" style="background-color:">Signaling</span> Contributes to Internalization of Brucella abortus by Macrophages. </a></p><div class="supp"><p class="details"><div>Diagram illustrating the phagocytic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> initiated by TLR4-linked JAK2 activation during the internalization of B. abortus into macrophage. The interaction of B. abortus with TLR4 induces the activation of Cdc42 GTPase and JAK2, and the subsequent activation of PI3K and MAPKs promotes actin polymerization. This event contributes to the phagocytosis of B. abortus by macrophage. Lines with arrows denote an activating reaction, and the dotted line denotes uncertainty of the reaction.</div></p></div><div class="aux"><div class="resc">Jin Ju Lee, et al. Infect Immun. 2013 July;81(7):2448-2458.</div><p class="links"><a class="dblinks" href="/pubmed/23630962" ref="ordinalpos=3544&amp;ncbi_uid=5670164&amp;link_uid=23630962">Citation</a><a class="dblinks" href="/pmc/articles/PMC3697618/" ref="ordinalpos=3544&amp;ncbi_uid=5670164&amp;link_uid=PMC3697618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3545.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3377643/figure/pone-0039276-g002/" ref="ordinalpos=3545&amp;ncbi_uid=4074746&amp;link_uid=PMC3377643" href="/pmc/articles/PMC3377643/"><img src="/pmc/articles/instance/3377643/bin/pone.0039276.g002.gif" src-large="/pmc/articles/instance/3377643/bin/pone.0039276.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3377643/" ref="ordinalpos=3545&amp;ncbi_uid=4074746&amp;link_uid=PMC3377643" image-link="/pmc/articles/PMC3377643/figure/pone-0039276-g002/" class="imagepopup">Figure 2. Mapping of changes in TGF-/BMP <span class="highlight" style="background-color:">signaling</span>..  From: TGF-? and Iron Differently Alter HBV Replication in Human Hepatocytes through TGF-?/BMP <span class="highlight" style="background-color:">Signaling</span> and Cellular MicroRNA Expression. </a></p><div class="supp"><p class="details"><div>Shows gene expression changes in 2.2.15 cells treated by 20 ng/ml TGF- for 48 h versus untreated controls. Changes were mapped with curated pathways in Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. Genes with increased expression are shown in red color and genes with decreased expression are shown in blue relative to untreated control cells.</div></p></div><div class="aux"><div class="resc">Sun O. Park, et al. PLoS One. 2012;7(6):e39276.</div><p class="links"><a class="dblinks" href="/pubmed/22723983" ref="ordinalpos=3545&amp;ncbi_uid=4074746&amp;link_uid=22723983">Citation</a><a class="dblinks" href="/pmc/articles/PMC3377643/" ref="ordinalpos=3545&amp;ncbi_uid=4074746&amp;link_uid=PMC3377643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3546.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2966790/figure/F10-8880/" ref="ordinalpos=3546&amp;ncbi_uid=3311168&amp;link_uid=PMC2966790" href="/pmc/articles/PMC2966790/"><img src="/pmc/articles/instance/2966790/bin/zjh0111088800010.gif" src-large="/pmc/articles/instance/2966790/bin/zjh0111088800010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2966790/" ref="ordinalpos=3546&amp;ncbi_uid=3311168&amp;link_uid=PMC2966790" image-link="/pmc/articles/PMC2966790/figure/F10-8880/" class="imagepopup">Figure 10.  From: IL-1?-Induced Increase in Intestinal Epithelial Tight Junction Permeability Is Mediated by MEKK-1 Activation of Canonical NF-?B <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed scheme of the intracellular pathways involved in IL-1induced increase in intestinal epithelial tight junction (TJ) permeability. IL-1 treatment resulted in activation of the MEKK-1 and NIK <span class="highlight" style="background-color:">signaling</span> cascades. MEKK-1 activation resulted in a step-wise activation of IKK and the canonical NF-B <span class="highlight" style="background-color:">pathway</span> and activation of the MLCK gene, culminating in the opening of the TJ barrier.</div></p></div><div class="aux"><div class="resc">Rana Al-Sadi, et al. Am J Pathol. 2010 November;177(5):2310-2322.</div><p class="links"><a class="dblinks" href="/pubmed/21048223" ref="ordinalpos=3546&amp;ncbi_uid=3311168&amp;link_uid=21048223">Citation</a><a class="dblinks" href="/pmc/articles/PMC2966790/" ref="ordinalpos=3546&amp;ncbi_uid=3311168&amp;link_uid=PMC2966790">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3547.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2935335/figure/f1/" ref="ordinalpos=3547&amp;ncbi_uid=3059577&amp;link_uid=PMC2935335" href="/pmc/articles/PMC2935335/"><img src="/pmc/articles/instance/2935335/bin/fig-1.gif" src-large="/pmc/articles/instance/2935335/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2935335/" ref="ordinalpos=3547&amp;ncbi_uid=3059577&amp;link_uid=PMC2935335" image-link="/pmc/articles/PMC2935335/figure/f1/" class="imagepopup">FIG. 1. From: Genetic Alterations in the Phosphatidylinositol-3 Kinase/Akt <span class="highlight" style="background-color:">Pathway</span> in Thyroid Cancer. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the highly simplified PI3K/Akt and MAP kinase pathways in thyroid cancer. Shown are the major elements in the two pathways. RTK, activated by growth factors, RET/PTC, and Ras can be coupled to both pathways. The remaining components regulate their respective pathways as described in the text. PTEN is a negative regulator of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. PI3K, phosphatidylinositol-3 kinase; RTK, receptor tyrosine kinase.</div></p></div><div class="aux"><div class="resc">Mingzhao Xing. Thyroid. 2010 July;20(7):697-706.</div><p class="links"><a class="dblinks" href="/pubmed/20578891" ref="ordinalpos=3547&amp;ncbi_uid=3059577&amp;link_uid=20578891">Citation</a><a class="dblinks" href="/pmc/articles/PMC2935335/" ref="ordinalpos=3547&amp;ncbi_uid=3059577&amp;link_uid=PMC2935335">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3548.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4121356/figure/f5-ijmm-34-03-0715/" ref="ordinalpos=3548&amp;ncbi_uid=6487213&amp;link_uid=PMC4121356" href="/pmc/articles/PMC4121356/"><img src="/pmc/articles/instance/4121356/bin/IJMM-34-03-0715-g04.gif" src-large="/pmc/articles/instance/4121356/bin/IJMM-34-03-0715-g04.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4121356/" ref="ordinalpos=3548&amp;ncbi_uid=6487213&amp;link_uid=PMC4121356" image-link="/pmc/articles/PMC4121356/figure/f5-ijmm-34-03-0715/" class="imagepopup">Figure 5.  From: Comprehensive identification of essential pathways and transcription factors related to epilepsy by gene set enrichment analysis on microarray datasets. </a></p><div class="supp"><p class="details"><div>Hierarchical clustering of genes involved in peroxisome proliferato-activated receptor (PPAR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related to epilepsy. In total, there were 68 genes involved in the PPAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related to epilepsy, which may be clustered into 4 gene set groups (groups AC) based on hierarchical clustering with Pearson correlation coefficient. Heatmap of 68 genes involved in the pentose phosphate <span class="highlight" style="background-color:">pathway</span> in 5 epilepsy samples (from EP-1 to EP-5) and 5 control samples (non-epilepsy, from CT-1 to CT-5) is shown. EP, epilepsy; CT, control.</div></p></div><div class="aux"><div class="resc">KAN HE, et al. Int J Mol Med. 2014 September;34(3):715-724.</div><p class="links"><a class="dblinks" href="/pubmed/25016997" ref="ordinalpos=3548&amp;ncbi_uid=6487213&amp;link_uid=25016997">Citation</a><a class="dblinks" href="/pmc/articles/PMC4121356/" ref="ordinalpos=3548&amp;ncbi_uid=6487213&amp;link_uid=PMC4121356">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3549.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2832169/figure/fig6/" ref="ordinalpos=3549&amp;ncbi_uid=3070509&amp;link_uid=PMC2832169" href="/pmc/articles/PMC2832169/"><img src="/pmc/articles/instance/2832169/bin/gr6.gif" src-large="/pmc/articles/instance/2832169/bin/gr6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2832169/" ref="ordinalpos=3549&amp;ncbi_uid=3070509&amp;link_uid=PMC2832169" image-link="/pmc/articles/PMC2832169/figure/fig6/" class="imagepopup">Figure 6.  From: Secreted Frizzled-Related Protein 4. </a></p><div class="supp"><p class="details"><div>Interaction between sFRP4 and the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. sFRP4 antagonizes the Wnt/-catenin and Wnt/PCP pathways while activating the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span>, the net result being a reduction in translocation of -catenin to the nucleus. As a consequence, there is diminished -cateninTcf interaction, resulting in a reduction in the transcription of angiogenesis genes.</div></p></div><div class="aux"><div class="resc">Ajit Muley, et al. Am J Pathol. 2010 March;176(3):1505-1516.</div><p class="links"><a class="dblinks" href="/pubmed/20056841" ref="ordinalpos=3549&amp;ncbi_uid=3070509&amp;link_uid=20056841">Citation</a><a class="dblinks" href="/pmc/articles/PMC2832169/" ref="ordinalpos=3549&amp;ncbi_uid=3070509&amp;link_uid=PMC2832169">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3550.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4154379/figure/F9/" ref="ordinalpos=3550&amp;ncbi_uid=6556724&amp;link_uid=PMC4154379" href="/pmc/articles/PMC4154379/"><img src="/pmc/articles/instance/4154379/bin/nihms157213f9.gif" src-large="/pmc/articles/instance/4154379/bin/nihms157213f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4154379/" ref="ordinalpos=3550&amp;ncbi_uid=6556724&amp;link_uid=PMC4154379" image-link="/pmc/articles/PMC4154379/figure/F9/" class="imagepopup">Fig. 9. A putative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in the TNF--induced production of MMP-9 in C2C12 myotubes.  From: Tumor Necrosis Factor-? Augments Matrix Metalloproteinase-9 Production in Skeletal Muscle Cells through the Activation of Transforming Growth Factor-?-activated Kinase 1 (TAK1)-dependent <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Binding of TNF- to its receptor causes the recruitment of TRADD, FADD, and TRAF-2 proteins and activates TAK1. Activation of TAK1 leads to the activation of downstream kinases, including ERK1/2, p38, and IKK, which in turn leads to the activation of AP-1 or NF-B transcription factors. TNF- also activates Akt kinase, which leads to the activation of NF-B transcription factor possibly through the activation of IKK. Activation and nuclear translocation of AP-1 and NF-B results in the increased expression of the Mmp-9 gene resulting in production of the MMP-9 protein in extracellular environment.</div></p></div><div class="aux"><div class="resc">Apurva K. Srivastava, et al. J Biol Chem. ;282(48):35113-35124.</div><p class="links"><a class="dblinks" href="/pubmed/17897957" ref="ordinalpos=3550&amp;ncbi_uid=6556724&amp;link_uid=17897957">Citation</a><a class="dblinks" href="/pmc/articles/PMC4154379/" ref="ordinalpos=3550&amp;ncbi_uid=6556724&amp;link_uid=PMC4154379">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3551.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC115912/figure/F5/" ref="ordinalpos=3551&amp;ncbi_uid=302790&amp;link_uid=PMC115912" href="/pmc/articles/PMC115912/"><img src="/pmc/articles/instance/115912/bin/jv0611901005.gif" src-large="/pmc/articles/instance/115912/bin/jv0611901005.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC115912/" ref="ordinalpos=3551&amp;ncbi_uid=302790&amp;link_uid=PMC115912" image-link="/pmc/articles/PMC115912/figure/F5/" class="imagepopup">FIG. 5.  From: Replicating Adenoviruses That Target Tumors with Constitutive Activation of the wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Activation of the wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> renders normal fibroblasts permissive for Tcf virus replication. WI38 cells were first infected with empty vector or -N -catenin-expressing lentiviruses and then infected with adenoviruses. (A) Western blot showing DBP expression 24 h after infection with wild-type Ad5 and vMB31. (B) PCR quantitation of adenoviral E1B and E2 mRNA 24 h after infection with wild-type Ad5 and vMB31 (upper two panels), and burst assay for virus production 48 h after infection with wild-type Ad5 and vMB31 (lower panel). The viral titer was measured by plaque assay on SW480.</div></p></div><div class="aux"><div class="resc">Michele Brunori, et al. J Virol. 2001 March;75(6):2857-2865.</div><p class="links"><a class="dblinks" href="/pubmed/11222711" ref="ordinalpos=3551&amp;ncbi_uid=302790&amp;link_uid=11222711">Citation</a><a class="dblinks" href="/pmc/articles/PMC115912/" ref="ordinalpos=3551&amp;ncbi_uid=302790&amp;link_uid=PMC115912">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3552.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3826850/figure/F2/" ref="ordinalpos=3552&amp;ncbi_uid=5482603&amp;link_uid=PMC3826850" href="/pmc/articles/PMC3826850/"><img src="/pmc/articles/instance/3826850/bin/1471-2148-13-245-2.gif" src-large="/pmc/articles/instance/3826850/bin/1471-2148-13-245-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3826850/" ref="ordinalpos=3552&amp;ncbi_uid=5482603&amp;link_uid=PMC3826850" image-link="/pmc/articles/PMC3826850/figure/F2/" class="imagepopup">Figure 2.  From: Evolutionary rate patterns of genes involved in the Drosophila Toll and Imd <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Nonsynonymous (</b><b><i>dN</i></b><b>) and synonymous (</b><b><i>dS</i></b><b>) substitution rates and their ratio (</b><b><i>=dN</i></b><b>/</b><b><i>dS</i></b><b>) as functions of <span class="highlight" style="background-color:">pathway</span> position.</b> The significant negative correlation between <i></i> and <span class="highlight" style="background-color:">pathway</span> position indicated that downstream genes were subjected to stronger purifying selection.</div></p></div><div class="aux"><div class="resc">Ming Han, et al. BMC Evol Biol. 2013;13:245-245.</div><p class="links"><a class="dblinks" href="/pubmed/24209511" ref="ordinalpos=3552&amp;ncbi_uid=5482603&amp;link_uid=24209511">Citation</a><a class="dblinks" href="/pmc/articles/PMC3826850/" ref="ordinalpos=3552&amp;ncbi_uid=5482603&amp;link_uid=PMC3826850">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3553.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4405166/figure/F2/" ref="ordinalpos=3553&amp;ncbi_uid=7444407&amp;link_uid=PMC4405166" href="/pmc/articles/PMC4405166/"><img src="/pmc/articles/instance/4405166/bin/nihms680041f2.gif" src-large="/pmc/articles/instance/4405166/bin/nihms680041f2.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4405166/" ref="ordinalpos=3553&amp;ncbi_uid=7444407&amp;link_uid=PMC4405166" image-link="/pmc/articles/PMC4405166/figure/F2/" class="imagepopup">FIGURE 2.  From: Insulin/IGF-I and Related <span class="highlight" style="background-color:">Signaling</span> Pathways Regulate Aging in Nondividing Cells: from Yeast to the Mammalian Brain. </a></p><div class="supp"><p class="details"><div>Interactions between insulin/IGF-I, TOR, and SIRT1 pathways in mammals. Nutrients, mainly amino acids, promote the activity of the TOR kinase. TOR, in its turn, activates S6K, a crucial AGC kinase involved cell growth, which can inhibit PI3K activation. Activated PKB activates TOR <span class="highlight" style="background-color:">signaling</span> via inhibition of TSC-1 and -2. The SIRT1 interactions represented in the figure have been observed in neurons[163,172]. SIRT1 promotes activation of the IRS-2 and Ras/ERK1/2 <span class="highlight" style="background-color:">pathway</span>, suggesting that SIRT1 may strengthen IGF-I <span class="highlight" style="background-color:">signaling</span>, in part, by deacetylating IRS-2. Inhibition of SIRT1 reverts the described mechanisms and enhances resistance to oxidative stress.</div></p></div><div class="aux"><div class="resc">Edoardo Parrella, et al. ScientificWorldJournal. ;10:161-177.</div><p class="links"><a class="dblinks" href="/pubmed/20098959" ref="ordinalpos=3553&amp;ncbi_uid=7444407&amp;link_uid=20098959">Citation</a><a class="dblinks" href="/pmc/articles/PMC4405166/" ref="ordinalpos=3553&amp;ncbi_uid=7444407&amp;link_uid=PMC4405166">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3554.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3719681/figure/F12/" ref="ordinalpos=3554&amp;ncbi_uid=5773120&amp;link_uid=PMC3719681" href="/pmc/articles/PMC3719681/"><img src="/pmc/articles/instance/3719681/bin/zmb9991000530012.gif" src-large="/pmc/articles/instance/3719681/bin/zmb9991000530012.jpg" alt="Fig 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3719681/" ref="ordinalpos=3554&amp;ncbi_uid=5773120&amp;link_uid=PMC3719681" image-link="/pmc/articles/PMC3719681/figure/F12/" class="imagepopup">Fig 12.  From: A-Kinase Anchoring Protein Lbc Coordinates a p38 Activating <span class="highlight" style="background-color:">Signaling</span> Complex Controlling Compensatory Cardiac Hypertrophy. </a></p><div class="supp"><p class="details"><div>Model illustrating the role of the AKAP-Lbc/p38 activation complex in the regulation of compensatory cardiac hypertrophy. AKAP-Lbc assembles a <span class="highlight" style="background-color:">signaling</span> complex, which includes the scaffolding protein PKN, as well as MLTK, MKK3, and p38. In response to pressure overload, AKAP-Lbc promotes the formation of RhoA-GTP, which in turn induces the sequential activation of PKN, MLTK, MKK3, and p38 within the AKAP-Lbc complex. This results in the activation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that controls protein synthesis and promotes compensatory cardiac hypertrophy, which temporarily preserves the function of the stressed heart.</div></p></div><div class="aux"><div class="resc">Irene Prez Lpez, et al. Mol Cell Biol. 2013 August;33(15):2903-2917.</div><p class="links"><a class="dblinks" href="/pubmed/23716597" ref="ordinalpos=3554&amp;ncbi_uid=5773120&amp;link_uid=23716597">Citation</a><a class="dblinks" href="/pmc/articles/PMC3719681/" ref="ordinalpos=3554&amp;ncbi_uid=5773120&amp;link_uid=PMC3719681">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3555.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2776832/figure/pbio-1000253-g006/" ref="ordinalpos=3555&amp;ncbi_uid=2111488&amp;link_uid=PMC2776832" href="/pmc/articles/PMC2776832/"><img src="/pmc/articles/instance/2776832/bin/pbio.1000253.g006.gif" src-large="/pmc/articles/instance/2776832/bin/pbio.1000253.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2776832/" ref="ordinalpos=3555&amp;ncbi_uid=2111488&amp;link_uid=PMC2776832" image-link="/pmc/articles/PMC2776832/figure/pbio-1000253-g006/" class="imagepopup">Figure 6. Implications of effects of the CD3 mutation on models of TCR signal transduction..  From: A Conserved CXXC Motif in CD3? Is Critical for T Cell Development and TCR <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) Schematic representation of proposed models for signal transduction across the T cell membrane to initiate downstream <span class="highlight" style="background-color:">signaling</span>. For simplicity the  chain and CD4/CD8 co-receptors are omitted from the diagrams. I denotes the critical protein required for initiation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. See text for discussion. (B) Predicted effect of the CD3 mutation on the various models. See text for discussion.</div></p></div><div class="aux"><div class="resc">Yibing Wang, et al. PLoS Biol. 2009 December;7(12):e1000253.</div><p class="links"><a class="dblinks" href="/pubmed/19956738" ref="ordinalpos=3555&amp;ncbi_uid=2111488&amp;link_uid=19956738">Citation</a><a class="dblinks" href="/pmc/articles/PMC2776832/" ref="ordinalpos=3555&amp;ncbi_uid=2111488&amp;link_uid=PMC2776832">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3556.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2831168/figure/F1/" ref="ordinalpos=3556&amp;ncbi_uid=2790870&amp;link_uid=PMC2831168" href="/pmc/articles/PMC2831168/"><img src="/pmc/articles/instance/2831168/bin/nihms163781f1.gif" src-large="/pmc/articles/instance/2831168/bin/nihms163781f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2831168/" ref="ordinalpos=3556&amp;ncbi_uid=2790870&amp;link_uid=PMC2831168" image-link="/pmc/articles/PMC2831168/figure/F1/" class="imagepopup">Figure 1. Systemic <span class="highlight" style="background-color:">signaling</span> regulates the shift in mitochondrial fuel utilization in response to CR.  From: Metabolic reprogramming, caloric restriction and aging. </a></p><div class="supp"><p class="details"><div>We propose a model where an AMPK and PGC-1 dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> permits a coordinate shift in fuel utilization in response to systemic input. Mitochondrial fatty acid oxidation is promoted through enhanced expression of genes encoding enzymes involved in -oxidation and molecules involved in fatty acid transport.</div></p></div><div class="aux"><div class="resc">Rozalyn M. Anderson, et al. Trends Endocrinol Metab. ;21(3):134-141.</div><p class="links"><a class="dblinks" href="/pubmed/20004110" ref="ordinalpos=3556&amp;ncbi_uid=2790870&amp;link_uid=20004110">Citation</a><a class="dblinks" href="/pmc/articles/PMC2831168/" ref="ordinalpos=3556&amp;ncbi_uid=2790870&amp;link_uid=PMC2831168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3557.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3367865/figure/F3/" ref="ordinalpos=3557&amp;ncbi_uid=4412188&amp;link_uid=PMC3367865" href="/pmc/articles/PMC3367865/"><img src="/pmc/articles/instance/3367865/bin/nihms330384f3.gif" src-large="/pmc/articles/instance/3367865/bin/nihms330384f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3367865/" ref="ordinalpos=3557&amp;ncbi_uid=4412188&amp;link_uid=PMC3367865" image-link="/pmc/articles/PMC3367865/figure/F3/" class="imagepopup">Figure 3. The RAS/RAF/MEK/MAPK <span class="highlight" style="background-color:">pathway</span>.  From: Molecular Biology of Lung Cancer: Clinical Implications. </a></p><div class="supp"><p class="details"><div>The RAS proto-oncogene family (<i>KRAS, HRAS, NRAS</i> and <i>RRAS</i>) encode four highly homologous 21kDa membrane-bound proteins involved in signal transduction. Proteins encoded by the RAS genes exist in two states: an active state, in which GTP is bound to the molecule and an inactive state, where the GTP has been cleaved to GDP<sup>383</sup>. Activating point mutations can confer oncogenic potential through a loss of intrinsic GTPase activity resulting in an inability to cleave GTP to GDP. This can initiate unchecked cell proliferation through the RAS/RAF/MEK/MAPK <span class="highlight" style="background-color:">pathway</span>, downstream of the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span><sup>384</sup>. Ras <span class="highlight" style="background-color:">signaling</span> also activates the PI3K/AKT <span class="highlight" style="background-color:">pathway</span> (leading to cell growth, proliferation, and survival), RalGDS and RASSF1. Adapted from<sup>12,385</sup>.</div></p></div><div class="aux"><div class="resc">Jill E. Larsen, et al. Clin Chest Med. ;32(4):703-740.</div><p class="links"><a class="dblinks" href="/pubmed/22054881" ref="ordinalpos=3557&amp;ncbi_uid=4412188&amp;link_uid=22054881">Citation</a><a class="dblinks" href="/pmc/articles/PMC3367865/" ref="ordinalpos=3557&amp;ncbi_uid=4412188&amp;link_uid=PMC3367865">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3558.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2890584/figure/pone-0011278-g002/" ref="ordinalpos=3558&amp;ncbi_uid=2347951&amp;link_uid=PMC2890584" href="/pmc/articles/PMC2890584/"><img src="/pmc/articles/instance/2890584/bin/pone.0011278.g002.gif" src-large="/pmc/articles/instance/2890584/bin/pone.0011278.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2890584/" ref="ordinalpos=3558&amp;ncbi_uid=2347951&amp;link_uid=PMC2890584" image-link="/pmc/articles/PMC2890584/figure/pone-0011278-g002/" class="imagepopup">Figure 2. HINT1 protein binds to a series of <span class="highlight" style="background-color:">signaling</span> proteins at the C terminus..  From: Mu-Opioid Receptors Transiently Activate the Akt-nNOS <span class="highlight" style="background-color:">Pathway</span> to Produce Sustained Potentiation of PKC-Mediated NMDAR-CaMKII <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>PAG synaptosomal membranes prepared from HINT1 (/) 129 SvJ mice were used to immunoprecipitate MORs and the association of the <span class="highlight" style="background-color:">signaling</span> proteins with MORs was studied. Signals corresponding to the MOR in HINT1 (/) and (+/+) mice were paired and co-precipitated proteins were directly compared. These assays were repeated twice on samples obtained from different mice. The results were consistent across replicates.</div></p></div><div class="aux"><div class="resc">Pilar Snchez-Blzquez, et al. PLoS One. 2010;5(6):e11278.</div><p class="links"><a class="dblinks" href="/pubmed/20585660" ref="ordinalpos=3558&amp;ncbi_uid=2347951&amp;link_uid=20585660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2890584/" ref="ordinalpos=3558&amp;ncbi_uid=2347951&amp;link_uid=PMC2890584">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3559.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2752567/figure/F3/" ref="ordinalpos=3559&amp;ncbi_uid=2109944&amp;link_uid=PMC2752567" href="/pmc/articles/PMC2752567/"><img src="/pmc/articles/instance/2752567/bin/zpq9990995430003.gif" src-large="/pmc/articles/instance/2752567/bin/zpq9990995430003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2752567/" ref="ordinalpos=3559&amp;ncbi_uid=2109944&amp;link_uid=PMC2752567" image-link="/pmc/articles/PMC2752567/figure/F3/" class="imagepopup">Fig. 3. From: Genetic heterogeneity of Myc-induced mammary tumors reflecting diverse phenotypes including metastatic potential. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> probabilities cluster tumor samples. Probability of <span class="highlight" style="background-color:">pathway</span> activation for MMTV-Myc, MMTV-Myc T58A, and MMTV-Neu tumors was calculated for the genes and conditions listed on the right axis of the heat map. Probability (from 0 to 1) is represented by a color scale with the probability of being like control represented in blue and probability of being like the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of interest being red. The probabilities were clustered for pathways and the samples were ordered in the same sequence shown for unsupervised clustering in Fig. 4. The histology, genotype, and <i>kras</i> mutation status are shown for each sample with the color codes being identified in the legend and kras mutations being illustrated with an  (<i>A</i>). The predicted probability of Ras <span class="highlight" style="background-color:">pathway</span> activation was graphed against Myc <span class="highlight" style="background-color:">pathway</span> activation with <i>r</i><sup>2</sup> = 0.5651 with <i>P</i> &lt; 0.0001 (<i>B</i>). The probability of Ras <span class="highlight" style="background-color:">pathway</span> activation was also examined in tumors with wild type and mutant <i>kras</i> sequence. Using a Mann-Whitney test, the significance was illustrated with <i>P</i> &lt; 0.0001 (<i>C</i>).</div></p></div><div class="aux"><div class="resc">Eran R. Andrechek, et al. Proc Natl Acad Sci U S A. 2009 September 22;106(38):16387-16392.</div><p class="links"><a class="dblinks" href="/pubmed/19805309" ref="ordinalpos=3559&amp;ncbi_uid=2109944&amp;link_uid=19805309">Citation</a><a class="dblinks" href="/pmc/articles/PMC2752567/" ref="ordinalpos=3559&amp;ncbi_uid=2109944&amp;link_uid=PMC2752567">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3560.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC369498/figure/f6/" ref="ordinalpos=3560&amp;ncbi_uid=476586&amp;link_uid=PMC369498" href="/pmc/articles/PMC369498/"><img src="/pmc/articles/instance/369498/bin/zjv0040416460006.gif" src-large="/pmc/articles/instance/369498/bin/zjv0040416460006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC369498/" ref="ordinalpos=3560&amp;ncbi_uid=476586&amp;link_uid=PMC369498" image-link="/pmc/articles/PMC369498/figure/f6/" class="imagepopup">FIG. 6. From: Selective Induction of Th2-Attracting Chemokines CCL17 and CCL22 in Human B Cells by Latent Membrane Protein 1 of Epstein-Barr Virus. </a></p><div class="supp"><p class="details"><div>Effects of various <span class="highlight" style="background-color:">signaling</span> inhibitors on the expression of chemokine genes in EBV-immortalized B cells and BJAB-LMP1. EBV-immortalized B cells (A) or BJAB-LMP1 cells (B) were treated for 12 h with or without PD98059 (the inhibitor of the MEK/ERK <span class="highlight" style="background-color:">pathway</span>) at 30 M, SB202190 (the inhibitor of the p38/ATF2 <span class="highlight" style="background-color:">pathway</span>) at 10 M, JNK inhibitor (the inhibitor of the JNK/AP-1 <span class="highlight" style="background-color:">pathway</span>) at 30 M, JAK3 inhibitor (the inhibitor of the JAK3/STAT <span class="highlight" style="background-color:">pathway</span>) at 30 M, AG490 (the inhibitor of the JAK2/STAT <span class="highlight" style="background-color:">pathway</span>) at 20 M, and BAY11-7082 (the inhibitor of the TRAF/NF-B <span class="highlight" style="background-color:">pathway</span>) at 5 M. The concentrations of the inhibitors used were carefully chosen as optimal based on preliminary experiments. Total RNA was prepared, and quantitative real-time RT-PCR was carried out for TARC/CCL17, MDC/CCL22, I-309/CCL1, MEC/CCL28, MIP-1/CCL3, MIP-1/CCL4, and RANTES/CCL5. For details, see Materials and Methods. The data are shown as means plus standard errors of the mean from three separate experiments.</div></p></div><div class="aux"><div class="resc">Takashi Nakayama, et al. J Virol. 2004 February;78(4):1665-1674.</div><p class="links"><a class="dblinks" href="/pubmed/14747532" ref="ordinalpos=3560&amp;ncbi_uid=476586&amp;link_uid=14747532">Citation</a><a class="dblinks" href="/pmc/articles/PMC369498/" ref="ordinalpos=3560&amp;ncbi_uid=476586&amp;link_uid=PMC369498">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3561.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3727838/figure/f1/" ref="ordinalpos=3561&amp;ncbi_uid=5199913&amp;link_uid=PMC3727838" href="/pmc/articles/PMC3727838/"><img src="/pmc/articles/instance/3727838/bin/bonekey2012111-f1.gif" src-large="/pmc/articles/instance/3727838/bin/bonekey2012111-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3727838/" ref="ordinalpos=3561&amp;ncbi_uid=5199913&amp;link_uid=PMC3727838" image-link="/pmc/articles/PMC3727838/figure/f1/" class="imagepopup">Figure 1. Reported methods to inhibit FGF23 <span class="highlight" style="background-color:">signaling</span>..  From: What's new in FGF23 research?. </a></p><div class="supp"><p class="details"><div>FGF23 binds to KlothoFGF receptor complex and activates mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> in kidney and parathyroid glands (left). Several methods have been reported as potential new therapy for FGF23-related hypophosphatemic diseases. C-terminal fragment of FGF23 competes with full-length FGF23 for binding to the KlothoFGF receptor complex (A). The inhibition of FGF receptor (B) and mitogen-activated protein kinase (C) can inhibit FGF23 <span class="highlight" style="background-color:">signaling</span>. Anti-FGF23 antibodies also antagonize actions of FGF23 (D).</div></p></div><div class="aux"><div class="resc">Seiji Fukumoto. Bonekey Rep. 2012;1:111.</div><p class="links"><a class="dblinks" href="/pubmed/23951494" ref="ordinalpos=3561&amp;ncbi_uid=5199913&amp;link_uid=23951494">Citation</a><a class="dblinks" href="/pmc/articles/PMC3727838/" ref="ordinalpos=3561&amp;ncbi_uid=5199913&amp;link_uid=PMC3727838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3562.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3124406/figure/F1/" ref="ordinalpos=3562&amp;ncbi_uid=3516170&amp;link_uid=PMC3124406" href="/pmc/articles/PMC3124406/"><img src="/pmc/articles/instance/3124406/bin/nihms-303040-f0001.gif" src-large="/pmc/articles/instance/3124406/bin/nihms-303040-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3124406/" ref="ordinalpos=3562&amp;ncbi_uid=3516170&amp;link_uid=PMC3124406" image-link="/pmc/articles/PMC3124406/figure/F1/" class="imagepopup">Figure 1. Summary of Bmp-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Bmp <span class="highlight" style="background-color:">Signaling</span> in Congenital Heart Disease: New Developments and Future Directions. </a></p><div class="supp"><p class="details"><div>In the canonical <span class="highlight" style="background-color:">pathway</span>, Bmp ligands bind to type II receptors then activate type I receptors. Activated type I receptors phosphorylate R-Smads (Smad 1,5,8) and following release from the type I receptors, phosphorylated R-Smads associate with the common Smad4 to form complex, which can then initiate transcription of target genes. Recent findings indicate that Bmp signals regulate gene expression through pathways mediated by microRNAs. Bmp <span class="highlight" style="background-color:">signaling</span> can regulate miRNA expression via the canonical <span class="highlight" style="background-color:">pathway</span>, and in addition, Smad1/5 can bind to a sequence specific site in the primary (pri)-miRNA to recruit the Drosha complex and promote the processing of the pri-miRNA to the precursor (pre)-miRNA (Davis et al., 2008; Davis et al., 2010; Li et al., 2010).</div></p></div><div class="aux"><div class="resc">Jun Wang, et al. Birth Defects Res A Clin Mol Teratol. ;91(6):441-448.</div><p class="links"><a class="dblinks" href="/pubmed/21384533" ref="ordinalpos=3562&amp;ncbi_uid=3516170&amp;link_uid=21384533">Citation</a><a class="dblinks" href="/pmc/articles/PMC3124406/" ref="ordinalpos=3562&amp;ncbi_uid=3516170&amp;link_uid=PMC3124406">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3563.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3520885/figure/F2/" ref="ordinalpos=3563&amp;ncbi_uid=4458090&amp;link_uid=PMC3520885" href="/pmc/articles/PMC3520885/"><img src="/pmc/articles/instance/3520885/bin/sgtp-3-219-g2.gif" src-large="/pmc/articles/instance/3520885/bin/sgtp-3-219-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3520885/" ref="ordinalpos=3563&amp;ncbi_uid=4458090&amp;link_uid=PMC3520885" image-link="/pmc/articles/PMC3520885/figure/F2/" class="imagepopup">Figure. From: R(h)oads to microvesicles. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> RhoA-dependent <span class="highlight" style="background-color:">signaling</span> induces MV biogenesis. Schematic showing the RhoA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that is responsible for promoting MV formation in human cancer cells. The <span class="highlight" style="background-color:">pathway</span> culminates with the phosphorylation of cofilin, which inhibits its actin-severing activity. This results in increased actin polymerization and the generation of MVs. The ROCK inhibitor, Y-27632, and LIMK siRNAs (for knocking-down LIMK expression) are particularly useful reagents for blocking RhoA-induced MV formation in cancer cells.</div></p></div><div class="aux"><div class="resc">Marc A. Antonyak, et al. Small GTPases. 2012 October 1;3(4):219-224.</div><p class="links"><a class="dblinks" href="/pubmed/22906997" ref="ordinalpos=3563&amp;ncbi_uid=4458090&amp;link_uid=22906997">Citation</a><a class="dblinks" href="/pmc/articles/PMC3520885/" ref="ordinalpos=3563&amp;ncbi_uid=4458090&amp;link_uid=PMC3520885">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3564.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938013/figure/F9/" ref="ordinalpos=3564&amp;ncbi_uid=2679743&amp;link_uid=PMC2938013" href="/pmc/articles/PMC2938013/"><img src="/pmc/articles/instance/2938013/bin/nihms-224788-f0009.gif" src-large="/pmc/articles/instance/2938013/bin/nihms-224788-f0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938013/" ref="ordinalpos=3564&amp;ncbi_uid=2679743&amp;link_uid=PMC2938013" image-link="/pmc/articles/PMC2938013/figure/F9/" class="imagepopup">Figure 9. Model for Dixdc1 function during cortical development.  From: Dixdc1 is a critical regulator of DISC1 and embryonic cortical development. </a></p><div class="supp"><p class="details"><div>Model for Dixdc1 function during cortical development. Top, During early cortical development, Dixdc1 binds DISC1 and together they synergistically regulate neural progenitor proliferation via modulation of Wnt-GSK3/-catenin <span class="highlight" style="background-color:">signaling</span>. Bottom, During later embryonic development, Dixdc1 integrates DISC1 into a <span class="highlight" style="background-color:">pathway</span> independent of Wnt-GSK3/-catenin <span class="highlight" style="background-color:">signaling</span>. In this neuronal migration <span class="highlight" style="background-color:">pathway</span>, Cdk5 phosphorylates Dixdc1, which regulate the formation of the DISC1/Dixdc1/Ndel1 complex that is essential for neuronal migration.</div></p></div><div class="aux"><div class="resc">Karun K. Singh, et al. Neuron. ;67(1):33-48.</div><p class="links"><a class="dblinks" href="/pubmed/20624590" ref="ordinalpos=3564&amp;ncbi_uid=2679743&amp;link_uid=20624590">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938013/" ref="ordinalpos=3564&amp;ncbi_uid=2679743&amp;link_uid=PMC2938013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3565.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4308786/figure/F6/" ref="ordinalpos=3565&amp;ncbi_uid=7127272&amp;link_uid=PMC4308786" href="/pmc/articles/PMC4308786/"><img src="/pmc/articles/instance/4308786/bin/NRR-7-1370-g008.gif" src-large="/pmc/articles/instance/4308786/bin/NRR-7-1370-g008.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4308786/" ref="ordinalpos=3565&amp;ncbi_uid=7127272&amp;link_uid=PMC4308786" image-link="/pmc/articles/PMC4308786/figure/F6/" class="imagepopup">Figure 6.  From: Mitogen activated protein kinase <span class="highlight" style="background-color:">signaling</span> pathways participate in the active principle region of Buyang Huanwu decoction-induced differentiation of bone marrow mesenchymal stem cells. </a></p><div class="supp"><p class="details"><div>Western blot electropherogram of neuron specific enolase expression after the mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> was blocked.<br />1: apr-BYHWD group; 2: apr-BYHWD + SB203580 group; 3: apr-BYHWD + PD98059 group; BYHWD: <i>Buyang Huanwu</i> decoction; apr-BYHWD: active principle region of BYHWD; PD98059 and SB203580: cellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocking agents.</div></p></div><div class="aux"><div class="resc">Jinghui Zheng, et al. Neural Regen Res. 2012 June 25;7(18):1370-1377.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3565&amp;ncbi_uid=7127272&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4308786/" ref="ordinalpos=3565&amp;ncbi_uid=7127272&amp;link_uid=PMC4308786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3566.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2705764/figure/fig1/" ref="ordinalpos=3566&amp;ncbi_uid=1951036&amp;link_uid=PMC2705764" href="/pmc/articles/PMC2705764/"><img src="/pmc/articles/instance/2705764/bin/PPAR2009-571365.001.gif" src-large="/pmc/articles/instance/2705764/bin/PPAR2009-571365.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2705764/" ref="ordinalpos=3566&amp;ncbi_uid=1951036&amp;link_uid=PMC2705764" image-link="/pmc/articles/PMC2705764/figure/fig1/" class="imagepopup">Figure 1.  From: New Target Genes for the Peroxisome Proliferator-Activated Receptor- (PPAR) Antitumour Activity: Perspectives from the Insulin Receptor. </a></p><div class="supp"><p class="details"><div>Upon binding of insulin, the IR undergoes autophosphorylation which enables the receptor to have a kinase activity and phosphorylates various cytoplasmic substrates, such as IRSs. From this point, <span class="highlight" style="background-color:">signaling</span> proceeds via a variety of <span class="highlight" style="background-color:">signaling</span> pathways (i.e., PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Ras and MAP kinase cascade) that are responsible for the metabolic, growth-promoting and mitogenic effects of insulin.</div></p></div><div class="aux"><div class="resc">Daniela P. Foti, et al. PPAR Res. 2009;2009:571365.</div><p class="links"><a class="dblinks" href="/pubmed/19587804" ref="ordinalpos=3566&amp;ncbi_uid=1951036&amp;link_uid=19587804">Citation</a><a class="dblinks" href="/pmc/articles/PMC2705764/" ref="ordinalpos=3566&amp;ncbi_uid=1951036&amp;link_uid=PMC2705764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3567.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3102652/figure/pone-0019664-g009/" ref="ordinalpos=3567&amp;ncbi_uid=3000494&amp;link_uid=PMC3102652" href="/pmc/articles/PMC3102652/"><img src="/pmc/articles/instance/3102652/bin/pone.0019664.g009.gif" src-large="/pmc/articles/instance/3102652/bin/pone.0019664.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3102652/" ref="ordinalpos=3567&amp;ncbi_uid=3000494&amp;link_uid=PMC3102652" image-link="/pmc/articles/PMC3102652/figure/pone-0019664-g009/" class="imagepopup">Figure 9. A Model for the HCV core/gC1qR-induced negative <span class="highlight" style="background-color:">signaling</span> (Tim-3/PD-1/SOCS-1) molecules in regulation of IL-12 and Th1/Tc1 responses during HCV infection..  From: Tim-3 Negatively Regulates IL-12 Expression by Monocytes in HCV Infection. </a></p><div class="supp"><p class="details"><div>We have previously shown that HCV core/gC1qR interaction up-regulates PD-1 and SOCS-1 negative <span class="highlight" style="background-color:">signaling</span> molecules, leading to suppression of TLR-mediated IL-12 production. In this study, we further demonstrated that the Tim-3 inhibitory <span class="highlight" style="background-color:">pathway</span> is involved in the HCV core/gC1qR-induced inhibition of IL-12 expression by M/M<sub></sub> during HCV infection. Specifically, we found that Tim-3 can be up-regulated by HCV core/gC1qR interaction, which in turn, inhibits TLR-mediated IL-12 production. We also found that Tim-3 can crosstalk with other inhibitory molecules such as PD-1 and SOCS-1 to coordinately inhibit TLR-mediated IL-12 <span class="highlight" style="background-color:">signaling</span> pathways during HCV infection. We conclude that HCV-mediated innate immune dysregulation (impaired M/M<sub></sub> IL-12 production) may ultimately lead to adaptive Th1/Tc1 dysfunction, and thus, HCV persistence.</div></p></div><div class="aux"><div class="resc">Ying Zhang, et al. PLoS One. 2011;6(5):e19664.</div><p class="links"><a class="dblinks" href="/pubmed/21637332" ref="ordinalpos=3567&amp;ncbi_uid=3000494&amp;link_uid=21637332">Citation</a><a class="dblinks" href="/pmc/articles/PMC3102652/" ref="ordinalpos=3567&amp;ncbi_uid=3000494&amp;link_uid=PMC3102652">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3568.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355158/figure/pone-0036560-g004/" ref="ordinalpos=3568&amp;ncbi_uid=4013514&amp;link_uid=PMC3355158" href="/pmc/articles/PMC3355158/"><img src="/pmc/articles/instance/3355158/bin/pone.0036560.g004.gif" src-large="/pmc/articles/instance/3355158/bin/pone.0036560.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355158/" ref="ordinalpos=3568&amp;ncbi_uid=4013514&amp;link_uid=PMC3355158" image-link="/pmc/articles/PMC3355158/figure/pone-0036560-g004/" class="imagepopup">Figure 4. Potential mechanism of photoreceptor death: TLR4 suppresses neuroprotective Wnt <span class="highlight" style="background-color:">signaling</span>..  From: Novel Role for the Innate Immune Receptor Toll-Like Receptor 4 (TLR4) in the Regulation of the Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Photoreceptor Apoptosis. </a></p><div class="supp"><p class="details"><div>The TLR4 activator LPS significantly decreased Wnt <span class="highlight" style="background-color:">signaling</span> in Muller glia-photoreceptor cultures (A) and the Muller glia cell line MIO-M1 (BC), as measured by luciferase reporter assay. Normalization to cotransfected Renilla was used to account for the effect of LPS reducing cell number (*p&lt;0.01, n=4). (C) Wnt <span class="highlight" style="background-color:">signaling</span> is suppressed by wild-type TLR4 (white bars) but not dominant-negative (grey bars) (*p&lt;0.05, n=5). Luciferase activity was normalized to cotransfected Renilla luciferase, and is shown as the ratio of Wnt <span class="highlight" style="background-color:">signaling</span> in Wnt3a-treated cells to control-treated cells. Mean + SD is shown.</div></p></div><div class="aux"><div class="resc">Hyun Yi, et al. PLoS One. 2012;7(5):e36560.</div><p class="links"><a class="dblinks" href="/pubmed/22615780" ref="ordinalpos=3568&amp;ncbi_uid=4013514&amp;link_uid=22615780">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355158/" ref="ordinalpos=3568&amp;ncbi_uid=4013514&amp;link_uid=PMC3355158">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3569.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3181349/figure/fig8/" ref="ordinalpos=3569&amp;ncbi_uid=3455228&amp;link_uid=PMC3181349" href="/pmc/articles/PMC3181349/"><img src="/pmc/articles/instance/3181349/bin/gr8.gif" src-large="/pmc/articles/instance/3181349/bin/gr8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3181349/" ref="ordinalpos=3569&amp;ncbi_uid=3455228&amp;link_uid=PMC3181349" image-link="/pmc/articles/PMC3181349/figure/fig8/" class="imagepopup">Figure 8.  From: Angiotensin II Contributes to Podocyte Injury by Increasing TRPC6 Expression via an NFAT-Mediated Positive Feedback <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that regulates the in AngII-dependent TRPC6 expression in podocytes. Results from our study together with previously published data 57  suggest the existence of the following <span class="highlight" style="background-color:">signaling</span> pathways in podocytes mediating AngII-induced TRPC6 expression and podocyte injury. AT1R stimulation by AngII results in Ca<sup>2+</sup> influx at least in part mediated by TRPC6. Ca<sup>2+</sup>-dependent calcineurin activation leads to activation and nuclear translocation of NFAT, which enhances transcription of NFAT-responsive genes such as TRPC6. A consequent increase in TRPC6 protein expression at the cell membrane could result in a positive feedback regulatory circuit, because Ca<sup>2+</sup> influx through TRPC6 itself appears to be involved in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to increased TRPC6 expression. Our studies show that NFAT activation per se is sufficient to induce TRPC6 transcription and proteinuria. Hypothetically, the proposed feedback mechanism could also result in a persistent calcineurin activation, which was previously demonstrated to lead to the dephosphorylation and degradation of the actin-binding protein synaptopodin (dashed box). 57  The latter was shown to result in cytoskeletal rearrangement and, eventually, proteinuria. Possibly, the latter <span class="highlight" style="background-color:">pathway</span> is involved in the generation of podocyte injury subsequent to AngII-induced TRPC6 expression.</div></p></div><div class="aux"><div class="resc">Tom Nijenhuis, et al. Am J Pathol. 2011 October;179(4):1719-1732.</div><p class="links"><a class="dblinks" href="/pubmed/21839714" ref="ordinalpos=3569&amp;ncbi_uid=3455228&amp;link_uid=21839714">Citation</a><a class="dblinks" href="/pmc/articles/PMC3181349/" ref="ordinalpos=3569&amp;ncbi_uid=3455228&amp;link_uid=PMC3181349">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3570.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4344557/figure/nutrients-07-00730-f002/" ref="ordinalpos=3570&amp;ncbi_uid=7294996&amp;link_uid=PMC4344557" href="/pmc/articles/PMC4344557/"><img src="/pmc/articles/instance/4344557/bin/nutrients-07-00730-g002.gif" src-large="/pmc/articles/instance/4344557/bin/nutrients-07-00730-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4344557/" ref="ordinalpos=3570&amp;ncbi_uid=7294996&amp;link_uid=PMC4344557" image-link="/pmc/articles/PMC4344557/figure/nutrients-07-00730-f002/" class="imagepopup">Figure 2.  From: The Positive Effects of Yerba Mat (Ilex paraguariensis) in Obesity. </a></p><div class="supp"><p class="details"><div>Proposed mechanism of action of yerba mat on the modulation of the PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Alessandra Gambero, et al. Nutrients. 2015 February;7(2):730-750.</div><p class="links"><a class="dblinks" href="/pubmed/25621503" ref="ordinalpos=3570&amp;ncbi_uid=7294996&amp;link_uid=25621503">Citation</a><a class="dblinks" href="/pmc/articles/PMC4344557/" ref="ordinalpos=3570&amp;ncbi_uid=7294996&amp;link_uid=PMC4344557">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3571.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3107442/figure/F5/" ref="ordinalpos=3571&amp;ncbi_uid=3006071&amp;link_uid=PMC3107442" href="/pmc/articles/PMC3107442/"><img src="/pmc/articles/instance/3107442/bin/fphys-02-00024-g005.gif" src-large="/pmc/articles/instance/3107442/bin/fphys-02-00024-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3107442/" ref="ordinalpos=3571&amp;ncbi_uid=3006071&amp;link_uid=PMC3107442" image-link="/pmc/articles/PMC3107442/figure/F5/" class="imagepopup">Figure 5.  From: Tracheal Occlusion Conditioning in Conscious Rats Modulates Gene Expression Profile of Medial Thalamus. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span> analysis of transcripts (<i>p</i>&lt;0.05) involved in cell <span class="highlight" style="background-color:">signaling</span></b>.</div></p></div><div class="aux"><div class="resc">Vipa Bernhardt, et al. Front Physiol. 2011;2:24.</div><p class="links"><a class="dblinks" href="/pubmed/21660287" ref="ordinalpos=3571&amp;ncbi_uid=3006071&amp;link_uid=21660287">Citation</a><a class="dblinks" href="/pmc/articles/PMC3107442/" ref="ordinalpos=3571&amp;ncbi_uid=3006071&amp;link_uid=PMC3107442">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3572.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2857646/figure/F1/" ref="ordinalpos=3572&amp;ncbi_uid=2270025&amp;link_uid=PMC2857646" href="/pmc/articles/PMC2857646/"><img src="/pmc/articles/instance/2857646/bin/nihms191420f1.gif" src-large="/pmc/articles/instance/2857646/bin/nihms191420f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2857646/" ref="ordinalpos=3572&amp;ncbi_uid=2270025&amp;link_uid=PMC2857646" image-link="/pmc/articles/PMC2857646/figure/F1/" class="imagepopup">Figure 1.  From: Epigenetic Targeting of Transforming Growth Factor ? Receptor II and Implications for Cancer Therapy. </a></p><div class="supp"><p class="details"><div>Schematic of the TGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (RI-TGFRI; RII-TGFRII).</div></p></div><div class="aux"><div class="resc">Sanjib Chowdhury, et al. Mol Cell Pharmacol. ;1(1):57-70.</div><p class="links"><a class="dblinks" href="/pubmed/20414468" ref="ordinalpos=3572&amp;ncbi_uid=2270025&amp;link_uid=20414468">Citation</a><a class="dblinks" href="/pmc/articles/PMC2857646/" ref="ordinalpos=3572&amp;ncbi_uid=2270025&amp;link_uid=PMC2857646">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3573.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4208607/figure/f2/" ref="ordinalpos=3573&amp;ncbi_uid=6820193&amp;link_uid=PMC4208607" href="/pmc/articles/PMC4208607/"><img src="/pmc/articles/instance/4208607/bin/fig-2.gif" src-large="/pmc/articles/instance/4208607/bin/fig-2.jpg" alt="&lt;b&gt;FIG. 2.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4208607/" ref="ordinalpos=3573&amp;ncbi_uid=6820193&amp;link_uid=PMC4208607" image-link="/pmc/articles/PMC4208607/figure/f2/" class="imagepopup">FIG. 2..  From: Mitophagy Is Required for Acute Cardioprotection by Simvastatin. </a></p><div class="supp"><p class="details"><div><b>Statins attenuate Akt/mTOR <span class="highlight" style="background-color:">signaling</span>. (A)</b> Representative western blots of Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins from HL-1 cells treated with vehicle (DMSO, solid bars) or simvastatin (statin, hashed bars) as indicated. <b>(B)</b> Quantitation of results shown in <b>(A)</b>. <i>n</i>=3 per group; experiments were performed at least three times. *<i>p</i>-value&lt;0.05. <b>(C)</b> Representative western blots of Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins in DMSO or statin-treated mouse hearts. <i>n</i>=4 mice per condition. <b>(D)</b> Quantitation of results shown in <b>(C)</b>. Akt, protein kinase B; mTOR, mammalian target of rapamycin.</div></p></div><div class="aux"><div class="resc">Allen M. Andres, et al. Antioxid Redox Signal. 2014 November 10;21(14):1960-1973.</div><p class="links"><a class="dblinks" href="/pubmed/23901824" ref="ordinalpos=3573&amp;ncbi_uid=6820193&amp;link_uid=23901824">Citation</a><a class="dblinks" href="/pmc/articles/PMC4208607/" ref="ordinalpos=3573&amp;ncbi_uid=6820193&amp;link_uid=PMC4208607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3574.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3369801/figure/fig1/" ref="ordinalpos=3574&amp;ncbi_uid=4729146&amp;link_uid=PMC3369801" href="/pmc/articles/PMC3369801/"><img src="/pmc/articles/instance/3369801/bin/cn-2011-00111m_0008.gif" src-large="/pmc/articles/instance/3369801/bin/cn-2011-00111m_0008.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3369801/" ref="ordinalpos=3574&amp;ncbi_uid=4729146&amp;link_uid=PMC3369801" image-link="/pmc/articles/PMC3369801/figure/fig1/" class="imagepopup">Figure 1.  From: A Simple Method for Quantifying Functional Selectivity and Agonist Bias. </a></p><div class="supp"><p class="details"><div>Schematic diagram showing the relevance of the BlackLeff operational model to quantification of agonist bias. The premise is based on the fact that the receptor conformation stabilized by the agonist will have a unique interaction with all <span class="highlight" style="background-color:">signaling</span> proteins that directly interact with it, thereby setting up the allosteric system(s) of modulator (agonist)/conduit (receptor)/guest (<span class="highlight" style="background-color:">signaling</span> protein). Under these circumstances, the affinity and the efficacy (the quality of the conformation) will be determined by the <span class="highlight" style="background-color:">signaling</span> protein, and this will be unique for each <span class="highlight" style="background-color:">pathway</span>. The magnitude of log(/<i>K</i><sub>A</sub>) will be characteristic of both the affinity and efficacy of the agonist for a particular <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Terry Kenakin, et al. ACS Chem Neurosci. 2012 March 21;3(3):193-203.</div><p class="links"><a class="dblinks" href="/pubmed/22860188" ref="ordinalpos=3574&amp;ncbi_uid=4729146&amp;link_uid=22860188">Citation</a><a class="dblinks" href="/pmc/articles/PMC3369801/" ref="ordinalpos=3574&amp;ncbi_uid=4729146&amp;link_uid=PMC3369801">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3575.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4067518/figure/F8/" ref="ordinalpos=3575&amp;ncbi_uid=6284651&amp;link_uid=PMC4067518" href="/pmc/articles/PMC4067518/"><img src="/pmc/articles/instance/4067518/bin/1477-7827-12-50-8.gif" src-large="/pmc/articles/instance/4067518/bin/1477-7827-12-50-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4067518/" ref="ordinalpos=3575&amp;ncbi_uid=6284651&amp;link_uid=PMC4067518" image-link="/pmc/articles/PMC4067518/figure/F8/" class="imagepopup">Figure 8.  From: Molecular aspects of development and regulation of endometriosis. </a></p><div class="supp"><p class="details"><div><b>TGF-/SMAD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The SMAD proteins are the only family of transcription factors known to propagate TGF- signals. Lerdelimumab (a recombinant human IgG4 targeting TGF- 2) and Metelimumab (a human monoclonal IgG4 against TGF- 1) can inhibit TGF- <span class="highlight" style="background-color:">signaling</span>. </div></p></div><div class="aux"><div class="resc">Yana B Aznaurova, et al. Reprod Biol Endocrinol. 2014;12:50-50.</div><p class="links"><a class="dblinks" href="/pubmed/24927773" ref="ordinalpos=3575&amp;ncbi_uid=6284651&amp;link_uid=24927773">Citation</a><a class="dblinks" href="/pmc/articles/PMC4067518/" ref="ordinalpos=3575&amp;ncbi_uid=6284651&amp;link_uid=PMC4067518">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3576.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2531163/figure/F3/" ref="ordinalpos=3576&amp;ncbi_uid=1582921&amp;link_uid=PMC2531163" href="/pmc/articles/PMC2531163/"><img src="/pmc/articles/instance/2531163/bin/nihms65923f3.gif" src-large="/pmc/articles/instance/2531163/bin/nihms65923f3.jpg" alt="Scheme 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2531163/" ref="ordinalpos=3576&amp;ncbi_uid=1582921&amp;link_uid=PMC2531163" image-link="/pmc/articles/PMC2531163/figure/F3/" class="imagepopup">Scheme 3.  From: Chemical Technologies for Probing Embryonic Development. </a></p><div class="supp"><p class="details"><div>Developmental <span class="highlight" style="background-color:">signaling</span> pathways. (A-E) Schematic depiction of five major pathways that mediate cell-cell communication during embryogenesis: Hh, Wnt, TGF, FGF, or Notch <span class="highlight" style="background-color:">signaling</span>. Ligands, receptors, downstream effectors or regulators, and transcription factors for each <span class="highlight" style="background-color:">pathway</span> are shown.</div></p></div><div class="aux"><div class="resc">Ilya A. Shestopalov, et al. Chem Soc Rev. ;37(7):1294-1307.</div><p class="links"><a class="dblinks" href="/pubmed/18568156" ref="ordinalpos=3576&amp;ncbi_uid=1582921&amp;link_uid=18568156">Citation</a><a class="dblinks" href="/pmc/articles/PMC2531163/" ref="ordinalpos=3576&amp;ncbi_uid=1582921&amp;link_uid=PMC2531163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3577.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4189604/figure/F2/" ref="ordinalpos=3577&amp;ncbi_uid=6819360&amp;link_uid=PMC4189604" href="/pmc/articles/PMC4189604/"><img src="/pmc/articles/instance/4189604/bin/s13588-014-0011-0-2.gif" src-large="/pmc/articles/instance/4189604/bin/s13588-014-0011-0-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4189604/" ref="ordinalpos=3577&amp;ncbi_uid=6819360&amp;link_uid=PMC4189604" image-link="/pmc/articles/PMC4189604/figure/F2/" class="imagepopup">Figure 2.  From: Effect of nano silver and silver nitrate on seed yield of (Ocimum basilicum L.). </a></p><div class="supp"><p class="details"><div><b>A model for the function of RAN1 in the ethylene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> RAN1 is presumed to be localized in the membrane of a post-Golgi compartment. Copper ions received from CCH, a putative copper chaperon, is transported by RAN1 into a post-Golgi compartment, delivering the metal to membrane-targeted ethylene receptor apoproteins that become able to coordinate ethylene after the incorporation of copper ions. In the absence of the hormone, the receptors are active and negatively regulate downstream <span class="highlight" style="background-color:">signaling</span> components, preventing hormone-response phenotypes. Ethylene is expected to inactivate the receptors upon binding, presumably by causing a reduction in histidine kinase/phosphatase activity. This, in turn, results in derepression of downstream <span class="highlight" style="background-color:">signaling</span> components (EIN2, EIN3) and activation of hormone-response phenotypes. The metal-deficient ethylene receptors are nonfunctional, resulting in a constitutively activated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Fatemeh Nejatzadeh-Barandozi, et al. Org Med Chem Lett. 2014;4:11-11.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3577&amp;ncbi_uid=6819360&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4189604/" ref="ordinalpos=3577&amp;ncbi_uid=6819360&amp;link_uid=PMC4189604">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3578.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4283144/figure/Fig8/" ref="ordinalpos=3578&amp;ncbi_uid=7038407&amp;link_uid=PMC4283144" href="/pmc/articles/PMC4283144/"><img src="/pmc/articles/instance/4283144/bin/12985_2014_2533_Fig8_HTML.gif" src-large="/pmc/articles/instance/4283144/bin/12985_2014_2533_Fig8_HTML.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4283144/" ref="ordinalpos=3578&amp;ncbi_uid=7038407&amp;link_uid=PMC4283144" image-link="/pmc/articles/PMC4283144/figure/Fig8/" class="imagepopup">Figure 8.  From: Development of kinomic analyses to identify dysregulated <span class="highlight" style="background-color:">signaling</span> pathways in cells expressing cytoplasmic PrP. </a></p><div class="supp"><p class="details"><div> <b>The levels of proteins in the Akt1/p70S6K branch of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> change coordinately.</b> Time-course analyses of the normalized expression levels of Akt1, mTOR, p70S6K, S6, eIF4B, and eEF2 in N2a cells expressing CyPrP<sup>EGFP</sup> for 12, 24, or 48 h. MeanSD; <i>n</i>=3.</div></p></div><div class="aux"><div class="resc">Rory H Shott, et al. Virol J. 2014;11(1):175.</div><p class="links"><a class="dblinks" href="/pubmed/25280966" ref="ordinalpos=3578&amp;ncbi_uid=7038407&amp;link_uid=25280966">Citation</a><a class="dblinks" href="/pmc/articles/PMC4283144/" ref="ordinalpos=3578&amp;ncbi_uid=7038407&amp;link_uid=PMC4283144">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3579.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4209606/figure/F2/" ref="ordinalpos=3579&amp;ncbi_uid=6767620&amp;link_uid=PMC4209606" href="/pmc/articles/PMC4209606/"><img src="/pmc/articles/instance/4209606/bin/ganc-05-348-g002.gif" src-large="/pmc/articles/instance/4209606/bin/ganc-05-348-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4209606/" ref="ordinalpos=3579&amp;ncbi_uid=6767620&amp;link_uid=PMC4209606" image-link="/pmc/articles/PMC4209606/figure/F2/" class="imagepopup">Figure 2. Alcohol- and LPS-induced lymphoma in TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutant mice is T-cell lymphoma.  From: Generation of a mouse model of T-cell lymphoma based on chronic LPS challenge and TGF-? <span class="highlight" style="background-color:">signaling</span> disruption. </a></p><div class="supp"><p class="details"><div>(A) Alcohol- and LPS-treated mice develop T-cell lymphomas as demonstrated by the expression of the T-cell marker CD3, and the absence of expression of the B-cell marker B220. (B) Bar graph indicates CD3 (T-cell lymphoma) or B220 (B-cell lymphoma) positive cell number.</div></p></div><div class="aux"><div class="resc">Nina M. Muoz, et al. Genes Cancer. 2014 September;5(9-10):348-352.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3579&amp;ncbi_uid=6767620&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4209606/" ref="ordinalpos=3579&amp;ncbi_uid=6767620&amp;link_uid=PMC4209606">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3580.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2176131/figure/fig7/" ref="ordinalpos=3580&amp;ncbi_uid=1174050&amp;link_uid=PMC2176131" href="/pmc/articles/PMC2176131/"><img src="/pmc/articles/instance/2176131/bin/AJRCMB38195f7.gif" src-large="/pmc/articles/instance/2176131/bin/AJRCMB38195f7.jpg" alt="&lt;b&gt;Figure 7.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2176131/" ref="ordinalpos=3580&amp;ncbi_uid=1174050&amp;link_uid=PMC2176131" image-link="/pmc/articles/PMC2176131/figure/fig7/" class="imagepopup">Figure 7..  From: Collagen I Promotes Epithelial-to-Mesenchymal Transition in Lung Cancer Cells via Transforming Growth Factor-? <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Model for EMT in A549 lung cancer cells. Extracellular stimuli such as collagen I or EGF activate PI3K/ERK <span class="highlight" style="background-color:">signaling</span>, which stimulates transcription of TGF-3 mRNA and autocrine secretion of TGF-3 protein, which binds to its receptor on the surface of the cell, activating the Smad <span class="highlight" style="background-color:">pathway</span> and up-regulating genes that promote EMT.</div></p></div><div class="aux"><div class="resc">Yasushi Shintani, et al. Am J Respir Cell Mol Biol. 2008 January;38(1):95-104.</div><p class="links"><a class="dblinks" href="/pubmed/17673689" ref="ordinalpos=3580&amp;ncbi_uid=1174050&amp;link_uid=17673689">Citation</a><a class="dblinks" href="/pmc/articles/PMC2176131/" ref="ordinalpos=3580&amp;ncbi_uid=1174050&amp;link_uid=PMC2176131">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3581.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1766317/figure/F5/" ref="ordinalpos=3581&amp;ncbi_uid=797694&amp;link_uid=PMC1766317" href="/pmc/articles/PMC1766317/"><img src="/pmc/articles/instance/1766317/bin/zpq0440639740005.gif" src-large="/pmc/articles/instance/1766317/bin/zpq0440639740005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1766317/" ref="ordinalpos=3581&amp;ncbi_uid=797694&amp;link_uid=PMC1766317" image-link="/pmc/articles/PMC1766317/figure/F5/" class="imagepopup">Fig. 5. From: CARMA3/Bcl10/MALT1-dependent NF-?B activation mediates angiotensin II-responsive inflammatory <span class="highlight" style="background-color:">signaling</span> in nonimmune cells. </a></p><div class="supp"><p class="details"><div>Model: A similar CARMA-Bcl10-MALT1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates NF-B activation in response to antigen receptor stimulation in lymphocytes and AT<sub>1</sub>R stimulation in liver cells. (<i>Left</i>) In lymphocytes, antigen-dependent NF-B activation leads to cytokine production and cellular proliferation. (<i>Right</i>) In liver, Ang II-dependent NF-B activation leads to a variety of proinflammatory effects that are implicated in the development of hepatic fibrosis/cirrhosis.</div></p></div><div class="aux"><div class="resc">Linda M. McAllister-Lucas, et al. Proc Natl Acad Sci U S A. 2007 January 2;104(1):139-144.</div><p class="links"><a class="dblinks" href="/pubmed/17101977" ref="ordinalpos=3581&amp;ncbi_uid=797694&amp;link_uid=17101977">Citation</a><a class="dblinks" href="/pmc/articles/PMC1766317/" ref="ordinalpos=3581&amp;ncbi_uid=797694&amp;link_uid=PMC1766317">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3582.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4031107/figure/f2-64_23/" ref="ordinalpos=3582&amp;ncbi_uid=6313515&amp;link_uid=PMC4031107" href="/pmc/articles/PMC4031107/"><img src="/pmc/articles/instance/4031107/bin/64_23_2.gif" src-large="/pmc/articles/instance/4031107/bin/64_23_2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4031107/" ref="ordinalpos=3582&amp;ncbi_uid=6313515&amp;link_uid=PMC4031107" image-link="/pmc/articles/PMC4031107/figure/f2-64_23/" class="imagepopup">Fig. 2.  From: Self-incompatibility in Brassicaceae crops: lessons for interspecific incompatibility. </a></p><div class="supp"><p class="details"><div>Model of <span class="highlight" style="background-color:">signaling</span> thorough a hypothesized universal <span class="highlight" style="background-color:">pathway</span> in <i>Brassica</i> self-incompatibility and interspecific incompatibility. X is a hypothesized receptor kinase.</div></p></div><div class="aux"><div class="resc">Hiroyasu Kitashiba, et al. Breed Sci. 2014 May;64(1):23-37.</div><p class="links"><a class="dblinks" href="/pubmed/24987288" ref="ordinalpos=3582&amp;ncbi_uid=6313515&amp;link_uid=24987288">Citation</a><a class="dblinks" href="/pmc/articles/PMC4031107/" ref="ordinalpos=3582&amp;ncbi_uid=6313515&amp;link_uid=PMC4031107">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3583.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3829355/figure/FU1/" ref="ordinalpos=3583&amp;ncbi_uid=6440400&amp;link_uid=PMC3829355" href="/pmc/articles/PMC3829355/"><img src="/pmc/articles/instance/3829355/bin/zbc0311359330001.gif" src-large="/pmc/articles/instance/3829355/bin/zbc0311359330001.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3829355/" ref="ordinalpos=3583&amp;ncbi_uid=6440400&amp;link_uid=PMC3829355" image-link="/pmc/articles/PMC3829355/figure/FU1/" class="imagepopup">Figure. From: Green Tea Compound Reduces Lipid Accumulation in Cells?. </a></p><div class="supp"><p class="details"><div>Schematic diagram of proposed EGCG-stimulated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to activate lipophagy.</div></p></div><div class="aux"><div class="resc">J Biol Chem. 2013 August 2;288(31):22706-22706.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3583&amp;ncbi_uid=6440400&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC3829355/" ref="ordinalpos=3583&amp;ncbi_uid=6440400&amp;link_uid=PMC3829355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3584.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3583082/figure/fig3/" ref="ordinalpos=3584&amp;ncbi_uid=4701966&amp;link_uid=PMC3583082" href="/pmc/articles/PMC3583082/"><img src="/pmc/articles/instance/3583082/bin/IJE2013-317487.003.gif" src-large="/pmc/articles/instance/3583082/bin/IJE2013-317487.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3583082/" ref="ordinalpos=3584&amp;ncbi_uid=4701966&amp;link_uid=PMC3583082" image-link="/pmc/articles/PMC3583082/figure/fig3/" class="imagepopup">Figure 3.  From: Comprehensive Literature Review: Recent Advances in Diagnosing and Managing Patients with Poorly Differentiated Thyroid Carcinoma. </a></p><div class="supp"><p class="details"><div>This diagram summarizes the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in thyroid carcinomas.</div></p></div><div class="aux"><div class="resc">Jack Hannallah, et al. Int J Endocrinol. 2013;2013:317487.</div><p class="links"><a class="dblinks" href="/pubmed/23476646" ref="ordinalpos=3584&amp;ncbi_uid=4701966&amp;link_uid=23476646">Citation</a><a class="dblinks" href="/pmc/articles/PMC3583082/" ref="ordinalpos=3584&amp;ncbi_uid=4701966&amp;link_uid=PMC3583082">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3585.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3395309/figure/fig4/" ref="ordinalpos=3585&amp;ncbi_uid=4126024&amp;link_uid=PMC3395309" href="/pmc/articles/PMC3395309/"><img src="/pmc/articles/instance/3395309/bin/CHRP2012-268681.004.gif" src-large="/pmc/articles/instance/3395309/bin/CHRP2012-268681.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3395309/" ref="ordinalpos=3585&amp;ncbi_uid=4126024&amp;link_uid=PMC3395309" image-link="/pmc/articles/PMC3395309/figure/fig4/" class="imagepopup">Figure 4.  From: Extracellular Matrix Proteins Modulate Antimigratory and Apoptotic Effects of Doxorubicin. </a></p><div class="supp"><p class="details"><div>Schematic representation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by doxorubicin in inhibition of TSP-1 expression and induction apoptosis.</div></p></div><div class="aux"><div class="resc">Georges Said, et al. Chemother Res Pract. 2012;2012:268681.</div><p class="links"><a class="dblinks" href="/pubmed/22811904" ref="ordinalpos=3585&amp;ncbi_uid=4126024&amp;link_uid=22811904">Citation</a><a class="dblinks" href="/pmc/articles/PMC3395309/" ref="ordinalpos=3585&amp;ncbi_uid=4126024&amp;link_uid=PMC3395309">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3586.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3296187/figure/fig1/" ref="ordinalpos=3586&amp;ncbi_uid=3892881&amp;link_uid=PMC3296187" href="/pmc/articles/PMC3296187/"><img src="/pmc/articles/instance/3296187/bin/IJBC2012-124704.001.gif" src-large="/pmc/articles/instance/3296187/bin/IJBC2012-124704.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3296187/" ref="ordinalpos=3586&amp;ncbi_uid=3892881&amp;link_uid=PMC3296187" image-link="/pmc/articles/PMC3296187/figure/fig1/" class="imagepopup">Figure 1.  From: RKIP Suppresses Breast Cancer Metastasis to the Bone by Regulating Stroma-Associated Genes. </a></p><div class="supp"><p class="details"><div>Schematic representation of the RKIP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its effects on metastatic progression to the bone.</div></p></div><div class="aux"><div class="resc">Elena Bevilacqua, et al. Int J Breast Cancer. 2012;2012:124704.</div><p class="links"><a class="dblinks" href="/pubmed/22482058" ref="ordinalpos=3586&amp;ncbi_uid=3892881&amp;link_uid=22482058">Citation</a><a class="dblinks" href="/pmc/articles/PMC3296187/" ref="ordinalpos=3586&amp;ncbi_uid=3892881&amp;link_uid=PMC3296187">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3587.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3293507/figure/F9/" ref="ordinalpos=3587&amp;ncbi_uid=4608512&amp;link_uid=PMC3293507" href="/pmc/articles/PMC3293507/"><img src="/pmc/articles/instance/3293507/bin/zh60031278470009.gif" src-large="/pmc/articles/instance/3293507/bin/zh60031278470009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3293507/" ref="ordinalpos=3587&amp;ncbi_uid=4608512&amp;link_uid=PMC3293507" image-link="/pmc/articles/PMC3293507/figure/F9/" class="imagepopup">Fig. 9. From: Role of ?-hydroxylase in adenosine-mediated aortic response through MAP kinase using A2A-receptor knockout mice. </a></p><div class="supp"><p class="details"><div>Schematic representation of hypothesis tested and proposed <span class="highlight" style="background-color:">pathway</span> for A<sub>1</sub>AR <span class="highlight" style="background-color:">signaling</span> in A<sub>2A</sub>KO mice.</div></p></div><div class="aux"><div class="resc">Dovenia S. Ponnoth, et al. Am J Physiol Regul Integr Comp Physiol. 2012 February 15;302(4):R400-R408.</div><p class="links"><a class="dblinks" href="/pubmed/22160543" ref="ordinalpos=3587&amp;ncbi_uid=4608512&amp;link_uid=22160543">Citation</a><a class="dblinks" href="/pmc/articles/PMC3293507/" ref="ordinalpos=3587&amp;ncbi_uid=4608512&amp;link_uid=PMC3293507">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3588.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3650458/figure/F3/" ref="ordinalpos=3588&amp;ncbi_uid=4885729&amp;link_uid=PMC3650458" href="/pmc/articles/PMC3650458/"><img src="/pmc/articles/instance/3650458/bin/emss-52503-f0003.gif" src-large="/pmc/articles/instance/3650458/bin/emss-52503-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3650458/" ref="ordinalpos=3588&amp;ncbi_uid=4885729&amp;link_uid=PMC3650458" image-link="/pmc/articles/PMC3650458/figure/F3/" class="imagepopup">Figure 3.  From: The bile acid membrane receptor TGR5 as an emerging target in metabolism and inflammation. </a></p><div class="supp"><p class="details"><div>Simplified overview of the TGR5-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to downstream <span class="highlight" style="background-color:">signaling</span> via cAMP induction.</div></p></div><div class="aux"><div class="resc">Thijs W.H. Pols, et al. J Hepatol. 2011 June;54(6):1263-1272.</div><p class="links"><a class="dblinks" href="/pubmed/21145931" ref="ordinalpos=3588&amp;ncbi_uid=4885729&amp;link_uid=21145931">Citation</a><a class="dblinks" href="/pmc/articles/PMC3650458/" ref="ordinalpos=3588&amp;ncbi_uid=4885729&amp;link_uid=PMC3650458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3589.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3031563/figure/pone-0016388-g006/" ref="ordinalpos=3589&amp;ncbi_uid=2757022&amp;link_uid=PMC3031563" href="/pmc/articles/PMC3031563/"><img src="/pmc/articles/instance/3031563/bin/pone.0016388.g006.gif" src-large="/pmc/articles/instance/3031563/bin/pone.0016388.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3031563/" ref="ordinalpos=3589&amp;ncbi_uid=2757022&amp;link_uid=PMC3031563" image-link="/pmc/articles/PMC3031563/figure/pone-0016388-g006/" class="imagepopup">Figure 6. Main proteins associated with insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Landscape Mapping of Functional Proteins in Insulin Signal Transduction and Insulin Resistance: A Network-Based Protein-Protein Interaction Analysis. </a></p><div class="supp"><p class="details"><div>Using WebLogo.</div></p></div><div class="aux"><div class="resc">Chiranjib Chakraborty, et al. PLoS One. 2011;6(1):e16388.</div><p class="links"><a class="dblinks" href="/pubmed/21305025" ref="ordinalpos=3589&amp;ncbi_uid=2757022&amp;link_uid=21305025">Citation</a><a class="dblinks" href="/pmc/articles/PMC3031563/" ref="ordinalpos=3589&amp;ncbi_uid=2757022&amp;link_uid=PMC3031563">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3590.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC239898/figure/F6/" ref="ordinalpos=3590&amp;ncbi_uid=289770&amp;link_uid=PMC239898" href="/pmc/articles/PMC239898/"><img src="/pmc/articles/instance/239898/bin/1477-7827-1-71-6.gif" src-large="/pmc/articles/instance/239898/bin/1477-7827-1-71-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC239898/" ref="ordinalpos=3590&amp;ncbi_uid=289770&amp;link_uid=PMC239898" image-link="/pmc/articles/PMC239898/figure/F6/" class="imagepopup">Figure 6.  From: Mitogen-activated protein kinases in normal and (pre)neoplastic ovarian surface epithelium. </a></p><div class="supp"><p class="details"><div>Activation of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by FSH, ATP and GnRH in ovarian cancer.</div></p></div><div class="aux"><div class="resc">Kyung-Chul Choi, et al. Reprod Biol Endocrinol. 2003;1:71-71.</div><p class="links"><a class="dblinks" href="/pubmed/14577832" ref="ordinalpos=3590&amp;ncbi_uid=289770&amp;link_uid=14577832">Citation</a><a class="dblinks" href="/pmc/articles/PMC239898/" ref="ordinalpos=3590&amp;ncbi_uid=289770&amp;link_uid=PMC239898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3591.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC100003/figure/F11/" ref="ordinalpos=3591&amp;ncbi_uid=67313&amp;link_uid=PMC100003" href="/pmc/articles/PMC100003/"><img src="/pmc/articles/instance/100003/bin/mb2410470011.gif" src-large="/pmc/articles/instance/100003/bin/mb2410470011.jpg" alt="FIG. 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC100003/" ref="ordinalpos=3591&amp;ncbi_uid=67313&amp;link_uid=PMC100003" image-link="/pmc/articles/PMC100003/figure/F11/" class="imagepopup">FIG. 11.  From: Neuropeptide-Induced Androgen Independence in Prostate Cancer Cells: Roles of Nonreceptor Tyrosine Kinases Etk/Bmx,  Src, and Focal Adhesion Kinase. </a></p><div class="supp"><p class="details"><div>Summary of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that connects neuropeptides to the AR.</div></p></div><div class="aux"><div class="resc">Li-Fen Lee, et al. Mol Cell Biol. 2001 December;21(24):8385-8397.</div><p class="links"><a class="dblinks" href="/pubmed/11713275" ref="ordinalpos=3591&amp;ncbi_uid=67313&amp;link_uid=11713275">Citation</a><a class="dblinks" href="/pmc/articles/PMC100003/" ref="ordinalpos=3591&amp;ncbi_uid=67313&amp;link_uid=PMC100003">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3592.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2040891/figure/F5/" ref="ordinalpos=3592&amp;ncbi_uid=1134675&amp;link_uid=PMC2040891" href="/pmc/articles/PMC2040891/"><img src="/pmc/articles/instance/2040891/bin/zpq0270768390005.gif" src-large="/pmc/articles/instance/2040891/bin/zpq0270768390005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2040891/" ref="ordinalpos=3592&amp;ncbi_uid=1134675&amp;link_uid=PMC2040891" image-link="/pmc/articles/PMC2040891/figure/F5/" class="imagepopup">Fig. 5. From: Bcl-2 is the target of a UV-inducible apoptosis switch and a node for UV <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>UVB acts at both the switch and the switched <span class="highlight" style="background-color:">pathway</span> to trigger UV-induced apoptosis. DNA photoproducts trigger phosphorylation of P53 and Mdm2, thereby reducing P53 degradation. In many systems, the elevated P53 triggers apoptosis by transcriptionally activating Bax, Bak, Puma, PIGs, and Noxa, as well as acting directly at mitochondria. For UVB-induced apoptosis, however, a larger signal proceeds through a <span class="highlight" style="background-color:">pathway</span> in which UV-induced P53 inactivates E2f1, as described in the text. Graded doses of UV cause the P53E2f1 regulatory apparatus to transition from a fully OFF <i>p53</i><sup>/</sup>-like state to a fully ON <i>E2f1</i><sup>/</sup>-like state. The apoptosis-inhibitory activity of E2f1 acts through Bcl-2, which suppresses the information flow in a direct <span class="highlight" style="background-color:">pathway</span> that is triggered by UV even when the P53E2f1 regulatory apparatus is removed. P53 disinhibits this <span class="highlight" style="background-color:">pathway</span>. Two secondary <span class="highlight" style="background-color:">signaling</span> pathways (dotted lines) are appreciable when basal Bcl-2 is low, as described in the text.</div></p></div><div class="aux"><div class="resc">Dejan Knezevic, et al. Proc Natl Acad Sci U S A. 2007 July 3;104(27):11286-11291.</div><p class="links"><a class="dblinks" href="/pubmed/17586682" ref="ordinalpos=3592&amp;ncbi_uid=1134675&amp;link_uid=17586682">Citation</a><a class="dblinks" href="/pmc/articles/PMC2040891/" ref="ordinalpos=3592&amp;ncbi_uid=1134675&amp;link_uid=PMC2040891">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3593.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3126716/figure/F3/" ref="ordinalpos=3593&amp;ncbi_uid=3046615&amp;link_uid=PMC3126716" href="/pmc/articles/PMC3126716/"><img src="/pmc/articles/instance/3126716/bin/1752-0509-5-82-3.gif" src-large="/pmc/articles/instance/3126716/bin/1752-0509-5-82-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3126716/" ref="ordinalpos=3593&amp;ncbi_uid=3046615&amp;link_uid=PMC3126716" image-link="/pmc/articles/PMC3126716/figure/F3/" class="imagepopup">Figure 3.  From: Time series gene expression profiling and temporal regulatory <span class="highlight" style="background-color:">pathway</span> analysis of BMP6 induced osteoblast differentiation and mineralization. </a></p><div class="supp"><p class="details"><div><b>Gene expression fold changes induced by BMP6 in three representative significant KEGG pathways</b>. Each <span class="highlight" style="background-color:">pathway</span> shown is at its most perturbed time point: (a) TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at 8h; (b) Focal adhesion at 10d; (c) MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at 24h. Gene expression level fold changes are standardized over the standard deviation of fold changes for all genes. The standardized fold changes are visualized by using KEGGanim web tool [56]. Note that one KEGG node may correspond to multiple closely related genes with the same function. Relevant other pathways are magnified locally for better view. Gene names are intentionally omitted by KEGGanim for clear view of the gene expression changes in pseudo-color.</div></p></div><div class="aux"><div class="resc">Weijun Luo, et al. BMC Syst Biol. 2011;5:82-82.</div><p class="links"><a class="dblinks" href="/pubmed/21605425" ref="ordinalpos=3593&amp;ncbi_uid=3046615&amp;link_uid=21605425">Citation</a><a class="dblinks" href="/pmc/articles/PMC3126716/" ref="ordinalpos=3593&amp;ncbi_uid=3046615&amp;link_uid=PMC3126716">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3594.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3227454/figure/F2/" ref="ordinalpos=3594&amp;ncbi_uid=3686678&amp;link_uid=PMC3227454" href="/pmc/articles/PMC3227454/"><img src="/pmc/articles/instance/3227454/bin/aging-03-896-g002.gif" src-large="/pmc/articles/instance/3227454/bin/aging-03-896-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3227454/" ref="ordinalpos=3594&amp;ncbi_uid=3686678&amp;link_uid=PMC3227454" image-link="/pmc/articles/PMC3227454/figure/F2/" class="imagepopup">Figure 2. Multiple pathways interact with the EGF <span class="highlight" style="background-color:">pathway</span> to regulate longevity.  From: Epidermal growth factor and aging: A <span class="highlight" style="background-color:">signaling</span> molecule reveals a new eye opening function. </a></p><div class="supp"><p class="details"><div>Different steps from multiple signal transduction pathways known to regulate longevity are shown on each row, including extrinsic inputs into the <span class="highlight" style="background-color:">pathway</span> (e.g., nutrient status, environmental stress, et cetera), extracellular ligands, transmembrane receptors, intracellular <span class="highlight" style="background-color:">signaling</span> cascades, transcription factors, and ultimate physiological outputs. Arrows indicate positive stimulatory genetic interactions, whereas T-bars indicate negative repressive genetic interactions. Dotted lines indicate feedbacks loops. Red letters indicate genes whose wild-type function ultimately acts to shorten lifespan, whereas green letters indicate genes whose wild-type function ultimately acts to enhance lifespan. Not all interactions should be considered of equal weight (e.g., different <span class="highlight" style="background-color:">signaling</span> pathways are activated depending on the timing and degree of nutrient deprivation). Additional reviews describe these other <span class="highlight" style="background-color:">signaling</span> pathways in more detail [30, 81-91].</div></p></div><div class="aux"><div class="resc">Christopher Rongo. Aging (Albany NY). 2011 September;3(9):896-905.</div><p class="links"><a class="dblinks" href="/pubmed/21931179" ref="ordinalpos=3594&amp;ncbi_uid=3686678&amp;link_uid=21931179">Citation</a><a class="dblinks" href="/pmc/articles/PMC3227454/" ref="ordinalpos=3594&amp;ncbi_uid=3686678&amp;link_uid=PMC3227454">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3595.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2775166/figure/F4/" ref="ordinalpos=3595&amp;ncbi_uid=2093424&amp;link_uid=PMC2775166" href="/pmc/articles/PMC2775166/"><img src="/pmc/articles/instance/2775166/bin/vetres-41-07-fig4.gif" src-large="/pmc/articles/instance/2775166/bin/vetres-41-07-fig4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2775166/" ref="ordinalpos=3595&amp;ncbi_uid=2093424&amp;link_uid=PMC2775166" image-link="/pmc/articles/PMC2775166/figure/F4/" class="imagepopup">Figure 4. From: Acute induction of cell death-related IFN stimulated genes (ISG) differentiates highly from moderately virulent CSFV strains. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways distribution of genes differentially modulated by Eystrup in relation to Paderborn. Genes revealed by two-class SAM analysis over the 3 days post-infection were organized with the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software according to the <span class="highlight" style="background-color:">pathway</span> in which they are involved in the current data base of this software, and classified according to the calculated Log(<i>p</i>-values). The significance of the association between the data set and the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was measured in two ways: ratio and <i>p</i>-value. Ratios refer to the proportion of variant genes from a <span class="highlight" style="background-color:">pathway</span> related to the number of genes included in the software database for this same <span class="highlight" style="background-color:">pathway</span>. The <i>p</i>-value, calculated using a right-tailed Fishers exact test, corresponds to the probability that the association between a set of focus genes and a given <span class="highlight" style="background-color:">pathway</span> is not due to random chance. The threshold corresponds to a limit of significance set by the software (<i>p</i> &lt; 0.05). The first nine networks are shown.</div></p></div><div class="aux"><div class="resc">Patricia Renson, et al. Vet Res. 2010 Jan-Feb;41(1):07.</div><p class="links"><a class="dblinks" href="/pubmed/19793538" ref="ordinalpos=3595&amp;ncbi_uid=2093424&amp;link_uid=19793538">Citation</a><a class="dblinks" href="/pmc/articles/PMC2775166/" ref="ordinalpos=3595&amp;ncbi_uid=2093424&amp;link_uid=PMC2775166">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3596.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3218018/figure/pone-0027710-g003/" ref="ordinalpos=3596&amp;ncbi_uid=3666588&amp;link_uid=PMC3218018" href="/pmc/articles/PMC3218018/"><img src="/pmc/articles/instance/3218018/bin/pone.0027710.g003.gif" src-large="/pmc/articles/instance/3218018/bin/pone.0027710.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3218018/" ref="ordinalpos=3596&amp;ncbi_uid=3666588&amp;link_uid=PMC3218018" image-link="/pmc/articles/PMC3218018/figure/pone-0027710-g003/" class="imagepopup">Figure 3. NDRG2 affects TGF-1/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the ratio of MMP2 to TIMP2 in LX-2 cells..  From: NDRG2 Ameliorates Hepatic Fibrosis by Inhibiting the TGF-?1/Smad <span class="highlight" style="background-color:">Pathway</span> and Altering the MMP2/TIMP2 Ratio in Rats. </a></p><div class="supp"><p class="details"><div>(A) Real-time PCR was used to detect the expression of Smad2, Smad3 and Smad7 in LX-2 cells with TGF-1 and/or adenovirus treatment. (B) Western blotting was used to detect the expression of Smad2, phospho-Smad2, Smad3, phospho-Smad3 and Smad7 in LX-2 cells with TGF-1 and/or adenovirus treatment. (C) Real-time PCR was used to detect the expression of MMP2 and TIMP2 in LX-2 cells following TGF-1 and/or adenovirus treatment. -actin was used as a loading control. All experiments were performed independently in triplicate. *P&lt;0.05 compared with the TGF-1 or TGF-1+AdLacZ group.</div></p></div><div class="aux"><div class="resc">Jiandong Yang, et al. PLoS One. 2011;6(11):e27710.</div><p class="links"><a class="dblinks" href="/pubmed/22110735" ref="ordinalpos=3596&amp;ncbi_uid=3666588&amp;link_uid=22110735">Citation</a><a class="dblinks" href="/pmc/articles/PMC3218018/" ref="ordinalpos=3596&amp;ncbi_uid=3666588&amp;link_uid=PMC3218018">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3597.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3759196/figure/f6-cancers-03-03242/" ref="ordinalpos=3597&amp;ncbi_uid=5270941&amp;link_uid=PMC3759196" href="/pmc/articles/PMC3759196/"><img src="/pmc/articles/instance/3759196/bin/cancers-03-03242f6.gif" src-large="/pmc/articles/instance/3759196/bin/cancers-03-03242f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759196/" ref="ordinalpos=3597&amp;ncbi_uid=5270941&amp;link_uid=PMC3759196" image-link="/pmc/articles/PMC3759196/figure/f6-cancers-03-03242/" class="imagepopup">Figure 6. From: Aberrant <span class="highlight" style="background-color:">Signaling</span> Pathways in Glioma. </a></p><div class="supp"><p class="details"><div>RTK/PI3K/Akt <span class="highlight" style="background-color:">signaling</span>. The binding of RTK, including EGFR and PDGFR, by the p85 subunit of PI3K results in activation of the catalytic subunit (p110), which then catalyzes phosphorylation of PI 3,4-bisphosphate (PiP2) into 3,4,5-triphosphate (PiP3). Inversely, PTEN turns PiP3 into PiP2. PiP3 in turn activates phosphoinositide-dependent kinase-1 (PDK1), which phosphorylates Thr308 of Akt, while Ser473 of Akt is phosphorylated by mTORC2. Activated Akt then inactivates TSC1/TSC2 suppressor complex, which in turn activates mTORC1 as a result. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> affects multiple cellular processes including cell survival, proliferation, and motility. Solid and dashed arrows indicate activation and suppression, respectively.</div></p></div><div class="aux"><div class="resc">Mitsutoshi Nakada, et al. Cancers (Basel). 2011 September;3(3):3242-3278.</div><p class="links"><a class="dblinks" href="/pubmed/24212955" ref="ordinalpos=3597&amp;ncbi_uid=5270941&amp;link_uid=24212955">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759196/" ref="ordinalpos=3597&amp;ncbi_uid=5270941&amp;link_uid=PMC3759196">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3598.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2688935/figure/F3/" ref="ordinalpos=3598&amp;ncbi_uid=1864093&amp;link_uid=PMC2688935" href="/pmc/articles/PMC2688935/"><img src="/pmc/articles/instance/2688935/bin/gb-2009-10-4-r44-3.gif" src-large="/pmc/articles/instance/2688935/bin/gb-2009-10-4-r44-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2688935/" ref="ordinalpos=3598&amp;ncbi_uid=1864093&amp;link_uid=PMC2688935" image-link="/pmc/articles/PMC2688935/figure/F3/" class="imagepopup">Figure 3.  From: Choosing the right path: enhancement of biologically relevant sets of genes or proteins using <span class="highlight" style="background-color:">pathway</span> structure. </a></p><div class="supp"><p class="details"><div>Schematic of networks used to generate simulated data. Illustrative schematic of the two pathways used to generate the simulated data. <b>(a) </b>The <i>Linear </i>network of 30 nodes/gene products, each of which is associated with one gene. The pair of squiggly lines across some arrows is used to indicate that there are more nodes that are not shown. <b>(b) </b>The Erbb <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the KEGG <span class="highlight" style="background-color:">pathway</span> database [44]. The expressions of the genes associated with the nodes circled in red are correlated with each other and are the genes that were affected by the treatment.</div></p></div><div class="aux"><div class="resc">Reuben Thomas, et al. Genome Biol. 2009;10(4):R44-R44.</div><p class="links"><a class="dblinks" href="/pubmed/19393085" ref="ordinalpos=3598&amp;ncbi_uid=1864093&amp;link_uid=19393085">Citation</a><a class="dblinks" href="/pmc/articles/PMC2688935/" ref="ordinalpos=3598&amp;ncbi_uid=1864093&amp;link_uid=PMC2688935">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3599.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3500394/figure/F1/" ref="ordinalpos=3599&amp;ncbi_uid=5484186&amp;link_uid=PMC3500394" href="/pmc/articles/PMC3500394/"><img src="/pmc/articles/instance/3500394/bin/nihms-394526-f0001.gif" src-large="/pmc/articles/instance/3500394/bin/nihms-394526-f0001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3500394/" ref="ordinalpos=3599&amp;ncbi_uid=5484186&amp;link_uid=PMC3500394" image-link="/pmc/articles/PMC3500394/figure/F1/" class="imagepopup">Fig 1.  From: FoxM1 and Wnt?-catenin <span class="highlight" style="background-color:">Signaling</span> in Glioma Stem Cells. </a></p><div class="supp"><p class="details"><div>Dysregulated Wnt-FoxM1/ -catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates self-renewal and tumorigenecity of GSCs<br />Wnt activates its receptor Frizzled and co-receptor LRP5/6 to disrupt or functionally inactivate the destruction complex containing Axin/APC/CK1/GSK3, leading to aberrant accumulation of -catenin in cytosol. FoxM1 can bind to free -catenin in cytosol and subsequently translocate into nucleus. In nucleus, FoxM1/-catenin complex is associated with TCF4 on Wnt target-gene promoters, such as c-myc and cyclinD1, to promote the expression of the Wnt target-genes hence self-renewal and tumorigenecity of GSCs.</div></p></div><div class="aux"><div class="resc">Aihua Gong, et al. Cancer Res. 2012 November 15;72(22):5658-5662.</div><p class="links"><a class="dblinks" href="/pubmed/23139209" ref="ordinalpos=3599&amp;ncbi_uid=5484186&amp;link_uid=23139209">Citation</a><a class="dblinks" href="/pmc/articles/PMC3500394/" ref="ordinalpos=3599&amp;ncbi_uid=5484186&amp;link_uid=PMC3500394">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3600.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3392816/figure/F5/" ref="ordinalpos=3600&amp;ncbi_uid=4111034&amp;link_uid=PMC3392816" href="/pmc/articles/PMC3392816/"><img src="/pmc/articles/instance/3392816/bin/ar3705-5.gif" src-large="/pmc/articles/instance/3392816/bin/ar3705-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3392816/" ref="ordinalpos=3600&amp;ncbi_uid=4111034&amp;link_uid=PMC3392816" image-link="/pmc/articles/PMC3392816/figure/F5/" class="imagepopup">Figure 5.  From: Extracellular nicotinamide phosphoribosyltransferase (NAMPT/visfatin) inhibits insulin-like growth factor-1 <span class="highlight" style="background-color:">signaling</span> and proteoglycan synthesis in human articular chondrocytes. </a></p><div class="supp"><p class="details"><div><b>Model for inhibitory effect of extracellular nicotinamide phosphoribosyltransferase on insulin-like growth factor-1 function in chondrocytes</b>. In this hypothetical model, sustained activation of extracellular signal-regulated kinases (ERK) by extracellular nicotinamide phosphoribosyltransferase (eNAMPT) promotes increased phosphorylation of insulin receptor substrate-1 (IRS-1) (serine-312), which negatively regulates the downstream insulin-like growth factor-1 (IGF-1) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and thereby matrix synthesis. IGF-1R, insulin-like growth factor-1 receptor.</div></p></div><div class="aux"><div class="resc">Raghunatha R Yammani, et al. Arthritis Res Ther. 2012;14(1):R23-R23.</div><p class="links"><a class="dblinks" href="/pubmed/22289259" ref="ordinalpos=3600&amp;ncbi_uid=4111034&amp;link_uid=22289259">Citation</a><a class="dblinks" href="/pmc/articles/PMC3392816/" ref="ordinalpos=3600&amp;ncbi_uid=4111034&amp;link_uid=PMC3392816">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="35" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="36" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="37" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:42:53-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A262C543122C100000000003F4E39;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A262C543122C100000000003F4E39" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>