
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="15" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB037453FEC8C10000000000497252" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="14" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="15" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="16" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="15" /></div>
            <div><h2 class="result_count">Results: 1401 to 1500 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1401.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2002608/figure/fig12/" ref="ordinalpos=1401&amp;ncbi_uid=1084861&amp;link_uid=PMC2002608" href="/pmc/articles/PMC2002608/"><img src="/pmc/articles/instance/2002608/bin/tpc1902484f12.gif" src-large="/pmc/articles/instance/2002608/bin/tpc1902484f12.jpg" alt="Figure 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2002608/" ref="ordinalpos=1401&amp;ncbi_uid=1084861&amp;link_uid=PMC2002608" image-link="/pmc/articles/PMC2002608/figure/fig12/" class="imagepopup">Figure 12. From: The SnRK1A Protein Kinase Plays a Key Role in Sugar <span class="highlight" style="background-color:">Signaling</span> during Germination and Seedling Growth of Rice. </a></p><div class="supp"><p class="details"><div>Proposed Roles of SnRK1 and MYBS1 in Sugar <span class="highlight" style="background-color:">Signaling</span> in Rice.<br />Details of the model are described in the text. The dashed line indicates the putative <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chung-An Lu, et al. Plant Cell. 2007 August;19(8):2484-2499.</div><p class="links"><a class="dblinks" href="/pubmed/17766403" ref="ordinalpos=1401&amp;ncbi_uid=1084861&amp;link_uid=17766403">Citation</a><a class="dblinks" href="/pmc/articles/PMC2002608/" ref="ordinalpos=1401&amp;ncbi_uid=1084861&amp;link_uid=PMC2002608">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1402.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3700945/figure/f5-bbi-7-2013-167/" ref="ordinalpos=1402&amp;ncbi_uid=5077015&amp;link_uid=PMC3700945" href="/pmc/articles/PMC3700945/"><img src="/pmc/articles/instance/3700945/bin/bbi-7-2013-167f5.gif" src-large="/pmc/articles/instance/3700945/bin/bbi-7-2013-167f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3700945/" ref="ordinalpos=1402&amp;ncbi_uid=5077015&amp;link_uid=PMC3700945" image-link="/pmc/articles/PMC3700945/figure/f5-bbi-7-2013-167/" class="imagepopup">Figure 5.  From: A Modular Cell-Type Focused Inflammatory Process Network Model for Non-Diseased Pulmonary Tissue. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> detail for the IPN: Mucus Hypersecretion model.<br /><b>Notes:</b> The Mucus Hypersecretion model contains <span class="highlight" style="background-color:">signaling</span> detail from four main pathways known to induce airway MUC5AC production. The TNF <span class="highlight" style="background-color:">pathway</span> (purple) includes the TNFR ligands TNF and TNFRSF1A and culminates in the transcriptional activation of NFkB. The IL13 <span class="highlight" style="background-color:">pathway</span> (red), describes the molecular events leading to epithelial cell differentiation into mucin-producing goblet cells through the cytokine IL13 and the transcription factor, SPDEF. The SP-1 <span class="highlight" style="background-color:">pathway</span> (green) describes the routes leading to transcriptional activation of MUC5AC by the transcription factor SP-1, and includes the upstream <span class="highlight" style="background-color:">signaling</span> kinases, PKA, PKC and MAPK14. The AP-1 <span class="highlight" style="background-color:">pathway</span> (blue) includes the primary pathways that activate MUC5AC expression via EGFR activation and the <span class="highlight" style="background-color:">signaling</span> intermediate, reactive oxygen species (ROS).</div></p></div><div class="aux"><div class="resc">Jurjen W. Westra, et al. Bioinform Biol Insights. 2013;7:167-192.</div><p class="links"><a class="dblinks" href="/pubmed/23843693" ref="ordinalpos=1402&amp;ncbi_uid=5077015&amp;link_uid=23843693">Citation</a><a class="dblinks" href="/pmc/articles/PMC3700945/" ref="ordinalpos=1402&amp;ncbi_uid=5077015&amp;link_uid=PMC3700945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1403.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4118962/figure/pone-0103794-g005/" ref="ordinalpos=1403&amp;ncbi_uid=6462704&amp;link_uid=PMC4118962" href="/pmc/articles/PMC4118962/"><img src="/pmc/articles/instance/4118962/bin/pone.0103794.g005.gif" src-large="/pmc/articles/instance/4118962/bin/pone.0103794.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4118962/" ref="ordinalpos=1403&amp;ncbi_uid=6462704&amp;link_uid=PMC4118962" image-link="/pmc/articles/PMC4118962/figure/pone-0103794-g005/" class="imagepopup">Figure 5. Agilent Whole Genome Oligo Microarray of common carotid arteries in Wistar and GK rats..  From: Interleukin-1/Toll-Like Receptor-Induced Nuclear Factor Kappa B <span class="highlight" style="background-color:">Signaling</span> Participates in Intima Hyperplasia after Carotid Artery Balloon Injury in Goto-Kakizaki Rats: A Potential Target Therapy <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Through three pairs of comparison (Wistar uninjured versus Wistar Day 7 post-injury (A), GK uninjured versus Day 7 post-injury (B), Wistar Day 7 post-injury versus GK Day 7 post-injury(C)), IL-1/TLR-induced NFκB activation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was filtered out.</div></p></div><div class="aux"><div class="resc">Xiaotian Zhang, et al. PLoS One. 2014;9(8):e103794.</div><p class="links"><a class="dblinks" href="/pubmed/25083789" ref="ordinalpos=1403&amp;ncbi_uid=6462704&amp;link_uid=25083789">Citation</a><a class="dblinks" href="/pmc/articles/PMC4118962/" ref="ordinalpos=1403&amp;ncbi_uid=6462704&amp;link_uid=PMC4118962">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1404.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4176998/figure/F9/" ref="ordinalpos=1404&amp;ncbi_uid=6654324&amp;link_uid=PMC4176998" href="/pmc/articles/PMC4176998/"><img src="/pmc/articles/instance/4176998/bin/1475-2840-12-158-9.gif" src-large="/pmc/articles/instance/4176998/bin/1475-2840-12-158-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4176998/" ref="ordinalpos=1404&amp;ncbi_uid=6654324&amp;link_uid=PMC4176998" image-link="/pmc/articles/PMC4176998/figure/F9/" class="imagepopup">Figure 9.  From: Protein kinase RNA- like endoplasmic reticulum kinase (PERK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> plays a major role in reactive oxygen species (ROS)- mediated endoplasmic reticulum stress- induced apoptosis in diabetic cardiomyopathy. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram demonstrating speculated roles of ER stress <span class="highlight" style="background-color:">signaling</span> pathways in diabetic cardiomyopathy.</b> Firstly, under pathological condition of hyperglycemia in myocytes, ROS is generated by glucose oxidation in mitochondria. Then ROS is released to ER and sensed by three independent ER stress sensors- ATF6, IER1 and PERK to induce ER stress. Under excessive ER stress, apoptotic signals are transducted by ATF6, IRE1 and PERK <span class="highlight" style="background-color:">signaling</span> pathways to induce myocytes apoptosis. Meanwhile, as a component of MAMs proteins, PERK also receives signals from ROS affecting MAMs. Thus, as PERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> received double stimulation of ROS in ER and MAMs, PERK is proposed the major <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> transducting apoptotic signals in ROS- mediated ER stress- induced cell apoptosis in DCM. (Non-specific or cross- talk <span class="highlight" style="background-color:">signaling</span> transduction by ER stress sensors is not shown in this figure).</div></p></div><div class="aux"><div class="resc">Zhong-Wei Liu, et al. Cardiovasc Diabetol. 2013;12:158-158.</div><p class="links"><a class="dblinks" href="/pubmed/24180212" ref="ordinalpos=1404&amp;ncbi_uid=6654324&amp;link_uid=24180212">Citation</a><a class="dblinks" href="/pmc/articles/PMC4176998/" ref="ordinalpos=1404&amp;ncbi_uid=6654324&amp;link_uid=PMC4176998">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1405.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4394890/figure/f5/" ref="ordinalpos=1405&amp;ncbi_uid=7438118&amp;link_uid=PMC4394890" href="/pmc/articles/PMC4394890/"><img src="/pmc/articles/instance/4394890/bin/srep09653-f5.gif" src-large="/pmc/articles/instance/4394890/bin/srep09653-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4394890/" ref="ordinalpos=1405&amp;ncbi_uid=7438118&amp;link_uid=PMC4394890" image-link="/pmc/articles/PMC4394890/figure/f5/" class="imagepopup">Figure 5. Cellular Model of essential components in MI progression..  From: MiRNA and TF co-regulatory network analysis for the pathology and recurrence of myocardial infarction. </a></p><div class="supp"><p class="details"><div>The graph illustrates the proposed model of MI progression co-regulated by miR-16-5p, miR-26b-5p, miR-29a-3p and the TFs; JUN, YY1, EGR1, SP1, GATA2 and MYC. These regulators control components of the VEGF <span class="highlight" style="background-color:">signaling</span> pathways, PI3K-Akt <span class="highlight" style="background-color:">signaling</span> pathways, MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Red rectangle represents miRNAs. Regulations by arrows in orange and green are predicted and verified regulations, respectively.</div></p></div><div class="aux"><div class="resc">Ying Lin, et al. Sci Rep. 2015;5:9653.</div><p class="links"><a class="dblinks" href="/pubmed/25867756" ref="ordinalpos=1405&amp;ncbi_uid=7438118&amp;link_uid=25867756">Citation</a><a class="dblinks" href="/pmc/articles/PMC4394890/" ref="ordinalpos=1405&amp;ncbi_uid=7438118&amp;link_uid=PMC4394890">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1406.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC321741/figure/fig1/" ref="ordinalpos=1406&amp;ncbi_uid=428601&amp;link_uid=PMC321741" href="/pmc/articles/PMC321741/"><img src="/pmc/articles/instance/321741/bin/zpq0050436800001.gif" src-large="/pmc/articles/instance/321741/bin/zpq0050436800001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC321741/" ref="ordinalpos=1406&amp;ncbi_uid=428601&amp;link_uid=PMC321741" image-link="/pmc/articles/PMC321741/figure/fig1/" class="imagepopup">Fig. 1. From: Unexpected similarities in cellular responses to bacterial and viral invasion. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways for TLR-3 and TLR-4. The interaction of LPS with TLR-4 that requires cooperation of two other proteins, CD14 and MD-2, initiates two major <span class="highlight" style="background-color:">signaling</span> pathways. The MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to the phosphorylation of IRAK, recruitment of TRAF-6, activation of NF-κB and JNK/p38, and release of proinflammatory cytokines. Another adapter, Mal/Tirap, participates in this <span class="highlight" style="background-color:">pathway</span>. The study of McWhirter <i>et al</i>. (5) focuses on the activation of the MyD88-independent <span class="highlight" style="background-color:">pathway</span> that results in the activation of IRF-3 and the induction of IFN-β and Rantes. TRIF and TRAM serve as adapter proteins in TLR-4 <span class="highlight" style="background-color:">signaling</span> that leads to activation of IRF-3. Association of TRIF with the noncanonic kinase TBK-1 results in a complex formation also containing IRF-3. Phosphorylated IRF-3 is translocated to the nucleus and stimulates expression of type I IFN and chemokine genes. The ligand for TLR-3 is dsRNA, viral RNA, or its surrogate poly(IC). TLR-3 does not associate with Mal TRAM and MyD88, and thus TRIF alone can mediate the dsRNA-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, leading to the phosphorylation of IRF-3 and the induction of IFN-β.</div></p></div><div class="aux"><div class="resc">Paula M. Pitha. Proc Natl Acad Sci U S A. 2004 January 20;101(3):695-696.</div><p class="links"><a class="dblinks" href="/pubmed/14718669" ref="ordinalpos=1406&amp;ncbi_uid=428601&amp;link_uid=14718669">Citation</a><a class="dblinks" href="/pmc/articles/PMC321741/" ref="ordinalpos=1406&amp;ncbi_uid=428601&amp;link_uid=PMC321741">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1407.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1950120/figure/F7/" ref="ordinalpos=1407&amp;ncbi_uid=1046231&amp;link_uid=PMC1950120" href="/pmc/articles/PMC1950120/"><img src="/pmc/articles/instance/1950120/bin/nihms26639f7.gif" src-large="/pmc/articles/instance/1950120/bin/nihms26639f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1950120/" ref="ordinalpos=1407&amp;ncbi_uid=1046231&amp;link_uid=PMC1950120" image-link="/pmc/articles/PMC1950120/figure/F7/" class="imagepopup">Figure 7. Diagrammatic depiction of the parallel <span class="highlight" style="background-color:">signaling</span> reactions occurring in BECs during infection by type 1 fimbriated enterobacteria.  From: TLR4 initiated and cAMP mediated abrogation of bacterial invasion of the bladder. </a></p><div class="supp"><p class="details"><div>Highlighted in the figure is the TLR4 initiated and AC-3, cAMP, and PKA dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that dissects with and abrogates the lipid raft mediated endocytic <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jeongmin Song, et al. Cell Host Microbe. ;1(4):287-298.</div><p class="links"><a class="dblinks" href="/pubmed/17710226" ref="ordinalpos=1407&amp;ncbi_uid=1046231&amp;link_uid=17710226">Citation</a><a class="dblinks" href="/pmc/articles/PMC1950120/" ref="ordinalpos=1407&amp;ncbi_uid=1046231&amp;link_uid=PMC1950120">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1408.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3142679/figure/F1/" ref="ordinalpos=1408&amp;ncbi_uid=3632290&amp;link_uid=PMC3142679" href="/pmc/articles/PMC3142679/"><img src="/pmc/articles/instance/3142679/bin/nihms304451f1.gif" src-large="/pmc/articles/instance/3142679/bin/nihms304451f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3142679/" ref="ordinalpos=1408&amp;ncbi_uid=3632290&amp;link_uid=PMC3142679" image-link="/pmc/articles/PMC3142679/figure/F1/" class="imagepopup">Figure 1. Major <span class="highlight" style="background-color:">signaling</span> pathways that are involved in virus reactivation.  From: Virus reactivation: a panoramic view in human infections. </a></p><div class="supp"><p class="details"><div>Alternative <span class="highlight" style="background-color:">signaling</span> pathways that involve Ras/Raf/NF-κB as key players in reactivation of different viruses are depicted. The Raf/MEK/ERK <span class="highlight" style="background-color:">pathway</span> of <span class="highlight" style="background-color:">signaling</span> interacts with other signal transduction pathways, leading to the activation of a variety of transcription factors that are critical for the transcription of specific cellular genes and in inducing viral reactivation. <span class="highlight" style="background-color:">Signaling</span> molecules that are boxed in blue are critical upstream components in the divergent MAPK and NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. These molecules can be activated by one or more of the ligands listed above.</div></p></div><div class="aux"><div class="resc">Christopher M Traylen, et al. Future Virol. ;6(4):451-463.</div><p class="links"><a class="dblinks" href="/pubmed/21799704" ref="ordinalpos=1408&amp;ncbi_uid=3632290&amp;link_uid=21799704">Citation</a><a class="dblinks" href="/pmc/articles/PMC3142679/" ref="ordinalpos=1408&amp;ncbi_uid=3632290&amp;link_uid=PMC3142679">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1409.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4027996/figure/F5/" ref="ordinalpos=1409&amp;ncbi_uid=6148426&amp;link_uid=PMC4027996" href="/pmc/articles/PMC4027996/"><img src="/pmc/articles/instance/4027996/bin/1471-2202-15-61-5.gif" src-large="/pmc/articles/instance/4027996/bin/1471-2202-15-61-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4027996/" ref="ordinalpos=1409&amp;ncbi_uid=6148426&amp;link_uid=PMC4027996" image-link="/pmc/articles/PMC4027996/figure/F5/" class="imagepopup">Figure 5.  From: Chronic alcohol consumption from adolescence-to-adulthood in mice - hypothalamic gene expression changes in the dilated cardiomyopathy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Hypothesis of adolescent chronic alcohol consumption effect on SPIF gene down regulation in the dilated cardiomyopathy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Diagram was adapted from KEGG <span class="highlight" style="background-color:">pathway</span> (http://www.genome.jp/kegg/) [92] for dilated cardiomyopathy (DCM), showing the β1AR-DHPR-Ca<sup>2+</sup> branch of the DCM <span class="highlight" style="background-color:">pathway</span>. Adolescent chronic alcohol consumption is hypothesized to result in the down regulation of SPIF genes (marked with grey color and solid down arrows), including β1-adrenergic receptor (Adrb1), Gs protein (Gnas), adenylyl cyclase 1 (Adcy1), and dihydropyridine receptor/L-type calcium channel (Cacna1d). The coordinated down regulation of these SPIF genes leads to impaired intracellular Ca<sup>2+</sup> homeostasis at the cellular level.</div></p></div><div class="aux"><div class="resc">Hong Zou, et al. BMC Neurosci. 2014;15:61-61.</div><p class="links"><a class="dblinks" href="/pubmed/24884436" ref="ordinalpos=1409&amp;ncbi_uid=6148426&amp;link_uid=24884436">Citation</a><a class="dblinks" href="/pmc/articles/PMC4027996/" ref="ordinalpos=1409&amp;ncbi_uid=6148426&amp;link_uid=PMC4027996">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1410.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2919057/figure/F9/" ref="ordinalpos=1410&amp;ncbi_uid=3212989&amp;link_uid=PMC2919057" href="/pmc/articles/PMC2919057/"><img src="/pmc/articles/instance/2919057/bin/nihms212787f9.gif" src-large="/pmc/articles/instance/2919057/bin/nihms212787f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919057/" ref="ordinalpos=1410&amp;ncbi_uid=3212989&amp;link_uid=PMC2919057" image-link="/pmc/articles/PMC2919057/figure/F9/" class="imagepopup">Figure 9. A model to implicate Tiam1/Rac1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PA-induced, CER-mediated effects on isolated β-cells.  From: Tiam1/Rac1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates palmitate-induced, ceramide-sensitive generation of superoxides and lipid peroxides and the loss of mitochondrial membrane potential in pancreatic ?-cells. </a></p><div class="supp"><p class="details"><div>Based on the data accrued in the current studies we propose that incubation of isolated β-cells to either PA [but not oleate] or C2-CER [but not DHC] leads to Tiam1-mediated activation of Rac1, which is a key member of the NOX assembly. In addition, PA or C2-CER promotes the expression of p47<sup>phox</sup> in these cells. Together, these <span class="highlight" style="background-color:">signaling</span> steps promote the assembly and activation of NOX holoenzyme leading to the production of superoxides and lipid peroxides. These, in turn, exert damaging effects on the mitochondria including reduction in MMP leading to the onset of mitochondrial dysfunction. Appropriate sites of inhibition of these <span class="highlight" style="background-color:">signaling</span> steps by FB-1, NSC23766 and DPI are also indicated in the figure.</div></p></div><div class="aux"><div class="resc">Ismail Syed, et al. Biochem Pharmacol. ;80(6):874-883.</div><p class="links"><a class="dblinks" href="/pubmed/20493824" ref="ordinalpos=1410&amp;ncbi_uid=3212989&amp;link_uid=20493824">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919057/" ref="ordinalpos=1410&amp;ncbi_uid=3212989&amp;link_uid=PMC2919057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1411.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4268240/figure/F3/" ref="ordinalpos=1411&amp;ncbi_uid=6969789&amp;link_uid=PMC4268240" href="/pmc/articles/PMC4268240/"><img src="/pmc/articles/instance/4268240/bin/nihms639712f3.gif" src-large="/pmc/articles/instance/4268240/bin/nihms639712f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4268240/" ref="ordinalpos=1411&amp;ncbi_uid=6969789&amp;link_uid=PMC4268240" image-link="/pmc/articles/PMC4268240/figure/F3/" class="imagepopup">Figure 3. KRAS Negatively Regulates TET1 Expression through the ERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: Suppression of TET1-Dependent DNA Demethylation is Essential for KRAS-Mediated Transformation. </a></p><div class="supp"><p class="details"><div>(A) In HBEC3 cell lines, mRNA levels of <i>DNMT1</i> and <i>TET1</i> were determined by RT-qPCR and normalized to V1 cells. Protein levels were determined by western blotting.<br />(B) After 30 μM ERK <span class="highlight" style="background-color:">pathway</span> inhibitor PD98059 or 2 μM AKT <span class="highlight" style="background-color:">pathway</span> inhibitor LY294002 treatment for 6 days, protein levels of DNMT1 and TET1 were determined by western blotting.<br />(C) After ERK <span class="highlight" style="background-color:">pathway</span> inhibition, mRNA levels were analyzed by RT-qPCR and normalized to DMSO control.<br />(D) Adherent and soft-agar colony formation after ERK <span class="highlight" style="background-color:">pathway</span> or AKT <span class="highlight" style="background-color:">pathway</span> inhibition indicate that cellular transformation is mediated by the ERK <span class="highlight" style="background-color:">pathway</span>. Cells were pre-treated with inhibitors for 6 days and then tested for colony formation. All data are presented as mean ± SD. ns, no significant difference; **, p &lt; 0.01; ***, p &lt; 0.001 in comparison to V1 cells or DMSO treated control. See also Figure S2.</div></p></div><div class="aux"><div class="resc">Bo-Kuan Wu, et al. Cell Rep. ;9(5):1827-1840.</div><p class="links"><a class="dblinks" href="/pubmed/25466250" ref="ordinalpos=1411&amp;ncbi_uid=6969789&amp;link_uid=25466250">Citation</a><a class="dblinks" href="/pmc/articles/PMC4268240/" ref="ordinalpos=1411&amp;ncbi_uid=6969789&amp;link_uid=PMC4268240">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1412.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3253522/figure/fig02/" ref="ordinalpos=1412&amp;ncbi_uid=3758387&amp;link_uid=PMC3253522" href="/pmc/articles/PMC3253522/"><img src="/pmc/articles/instance/3253522/bin/ajcd0001-0274-f2.gif" src-large="/pmc/articles/instance/3253522/bin/ajcd0001-0274-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3253522/" ref="ordinalpos=1412&amp;ncbi_uid=3758387&amp;link_uid=PMC3253522" image-link="/pmc/articles/PMC3253522/figure/fig02/" class="imagepopup">Figure 2.  From: Cardioproteomics: advancing the discovery of <span class="highlight" style="background-color:">signaling</span> mechanisms involved in cardiovascular diseases. </a></p><div class="supp"><p class="details"><div>Schematic diagram showing a specific phosphorylation related cardiovascular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving tropomyosin which was partly resolved by cardioproteomics. A mutation in tropomyosin (E54K) is associated with dilated cardiomyopathy and this mutation was investigated using transgenic mice expressing this mutant protein [44]. Proteomics revealed that decreased phosphorylation of tropomyosin may directly affect myofilament function and be part of the dilated cardiomyopathy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in E54K transgenic mice.</div></p></div><div class="aux"><div class="resc">Ziyou Cui, et al. Am J Cardiovasc Dis. 2011;1(3):274-292.</div><p class="links"><a class="dblinks" href="/pubmed/22254205" ref="ordinalpos=1412&amp;ncbi_uid=3758387&amp;link_uid=22254205">Citation</a><a class="dblinks" href="/pmc/articles/PMC3253522/" ref="ordinalpos=1412&amp;ncbi_uid=3758387&amp;link_uid=PMC3253522">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1413.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2861899/figure/F7/" ref="ordinalpos=1413&amp;ncbi_uid=2310223&amp;link_uid=PMC2861899" href="/pmc/articles/PMC2861899/"><img src="/pmc/articles/instance/2861899/bin/nihms183832f7.gif" src-large="/pmc/articles/instance/2861899/bin/nihms183832f7.jpg" alt="FIG. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2861899/" ref="ordinalpos=1413&amp;ncbi_uid=2310223&amp;link_uid=PMC2861899" image-link="/pmc/articles/PMC2861899/figure/F7/" class="imagepopup">FIG. 7. Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for the regulation of human MAO B gene expression.  From: Activation of Human Monoamine Oxidase B Gene Expression by a Protein Kinase C MAPK Signal Transduction <span class="highlight" style="background-color:">Pathway</span> Involves c-Jun and Egr-1. </a></p><div class="supp"><p class="details"><div>The kinases indicated with an <i>open box</i> appear not to be involved. <i>Solid lines</i> connect individual kinases in the signal transduction <span class="highlight" style="background-color:">pathway</span>, whereas <i>dotted lines</i> appear not to be important for the PMA-induced MAO B activity.</div></p></div><div class="aux"><div class="resc">Wai K. Wong, et al. J Biol Chem. ;277(25):22222-22230.</div><p class="links"><a class="dblinks" href="/pubmed/11956220" ref="ordinalpos=1413&amp;ncbi_uid=2310223&amp;link_uid=11956220">Citation</a><a class="dblinks" href="/pmc/articles/PMC2861899/" ref="ordinalpos=1413&amp;ncbi_uid=2310223&amp;link_uid=PMC2861899">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1414.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4002612/figure/fig3/" ref="ordinalpos=1414&amp;ncbi_uid=6626579&amp;link_uid=PMC4002612" href="/pmc/articles/PMC4002612/"><img src="/pmc/articles/instance/4002612/bin/aps20132f3.gif" src-large="/pmc/articles/instance/4002612/bin/aps20132f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4002612/" ref="ordinalpos=1414&amp;ncbi_uid=6626579&amp;link_uid=PMC4002612" image-link="/pmc/articles/PMC4002612/figure/fig3/" class="imagepopup">Figure 3.  From: Characterization of a novel curcumin analog P1 as potent inhibitor of the NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with distinct mechanisms. </a></p><div class="supp"><p class="details"><div>P1 exhibited stronger inhibition of the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> than curcumin. (A) The inhibition curve of curcumin on TNF-α (10 ng/mL)-induced NF-κB luciferase activity. Cell viability was examined in parallel with luciferase activity. (B) Dose dependence curve of P1 inhibition of TNF-α-induced NF-κB luciferase activity. Cell viability was examined in parallel with luciferase activity. (C) Time dependence curve of P1 inhibition of TNF-α-induced NF-κB luciferase activity. Cell viability was examined in parallel with luciferase activity. (D) Western blot analysis of the key proteins in the NF-κB <span class="highlight" style="background-color:">pathway</span> after curcumin or P1 treatment for 6 h. Mean±SD. <i>n</i>=3</div></p></div><div class="aux"><div class="resc">Yan-min Peng, et al. Acta Pharmacol Sin. 2013 July 5;34(7):939-950.</div><p class="links"><a class="dblinks" href="/pubmed/23603982" ref="ordinalpos=1414&amp;ncbi_uid=6626579&amp;link_uid=23603982">Citation</a><a class="dblinks" href="/pmc/articles/PMC4002612/" ref="ordinalpos=1414&amp;ncbi_uid=6626579&amp;link_uid=PMC4002612">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1415.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3638576/figure/fig6/" ref="ordinalpos=1415&amp;ncbi_uid=4883236&amp;link_uid=PMC3638576" href="/pmc/articles/PMC3638576/"><img src="/pmc/articles/instance/3638576/bin/ECAM2013-219472.006.gif" src-large="/pmc/articles/instance/3638576/bin/ECAM2013-219472.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3638576/" ref="ordinalpos=1415&amp;ncbi_uid=4883236&amp;link_uid=PMC3638576" image-link="/pmc/articles/PMC3638576/figure/fig6/" class="imagepopup">Figure 6.  From:  Ganoderma tsugae Extract Inhibits Growth of HER2-Overexpressing Cancer Cells via Modulation of HER2/PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A schematic model of the GTE-mediated antiproliferative effect on HER2-overexpressing cancer cells. Ligand stimulation induces the activation of the HER2 receptor, which in turn activates the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and then promotes cell growth and survival. After GTE treatment, the proliferation is inhibited because of an induction of cell cycle arrest. The GTE-mediated growth repression coincides with a reduction in the transcriptional activity of <i>HER2</i> gene and an induction in the degradation of HER2 protein, leading to a downregulation of the HER2/PI3K/Akt <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Han-Peng Kuo, et al. Evid Based Complement Alternat Med. 2013;2013:219472.</div><p class="links"><a class="dblinks" href="/pubmed/23662119" ref="ordinalpos=1415&amp;ncbi_uid=4883236&amp;link_uid=23662119">Citation</a><a class="dblinks" href="/pmc/articles/PMC3638576/" ref="ordinalpos=1415&amp;ncbi_uid=4883236&amp;link_uid=PMC3638576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1416.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3278757/figure/F3/" ref="ordinalpos=1416&amp;ncbi_uid=4610603&amp;link_uid=PMC3278757" href="/pmc/articles/PMC3278757/"><img src="/pmc/articles/instance/3278757/bin/btr696f3.gif" src-large="/pmc/articles/instance/3278757/bin/btr696f3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3278757/" ref="ordinalpos=1416&amp;ncbi_uid=4610603&amp;link_uid=PMC3278757" image-link="/pmc/articles/PMC3278757/figure/F3/" class="imagepopup">Fig. 3. From: Optimal structural inference of <span class="highlight" style="background-color:">signaling</span> pathways from unordered and overlapping gene sets. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) empirical <i>P</i>-values computed for true <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> structures (Left) and randomly selected feasible <span class="highlight" style="background-color:">pathway</span> structures (Right) corresponding to 83 IFGS compendiums derived from the KEGG pathways. (<b>B</b>) Energy values computed by varying the cooling schedule constant for a total of 2×10<sup>5</sup> jumps. The IFGS compendium was derived from the generic vascular smooth muscle contraction <span class="highlight" style="background-color:">pathway</span> in KEGG.</div></p></div><div class="aux"><div class="resc">Lipi R. Acharya, et al. Bioinformatics. 2012 February 15;28(4):546-556.</div><p class="links"><a class="dblinks" href="/pubmed/22199386" ref="ordinalpos=1416&amp;ncbi_uid=4610603&amp;link_uid=22199386">Citation</a><a class="dblinks" href="/pmc/articles/PMC3278757/" ref="ordinalpos=1416&amp;ncbi_uid=4610603&amp;link_uid=PMC3278757">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1417.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC124938/figure/f4/" ref="ordinalpos=1417&amp;ncbi_uid=378894&amp;link_uid=PMC124938" href="/pmc/articles/PMC124938/"><img src="/pmc/articles/instance/124938/bin/pq1222493004.gif" src-large="/pmc/articles/instance/124938/bin/pq1222493004.jpg" alt="Fig 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC124938/" ref="ordinalpos=1417&amp;ncbi_uid=378894&amp;link_uid=PMC124938" image-link="/pmc/articles/PMC124938/figure/f4/" class="imagepopup">Fig 4. From: Dissection of the c-Kit <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mouse primordial germ cells by retroviral-mediated gene transfer. </a></p><div class="supp"><p class="details"><div>Effect of AKT expression on PGC survival after 7 days in culture. c-AKT expressed from either an ecotropic MLV-derived vector (<i>A</i>) or an ALV-derived vector (<i>B</i>) improves PGC survival when c-Kit-mediated <span class="highlight" style="background-color:">signaling</span> is not activated (PGCs cultured on BM-Sl<sup>d</sup> cells that do not express membrane-bound KL). (<i>C</i>) ALV-mediated expression of c-AKT also improves PGC survival when c-Kit <span class="highlight" style="background-color:">signaling</span> is activated (PGCs cultured on STO cells that express membrane-bound KL). (<i>D</i>) ALV-mediated expression of a dominant-negative AKT gene (AKT-T308A/S473A) in PGCs with an activated Kit <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (PGCs cultured on STO cells that express membrane-bound KL) significantly impairs survival. Data represent the mean ± SEM of three pooled experiments each comprising triplicate cultures. *, Difference is significant at the 0.05 level.</div></p></div><div class="aux"><div class="resc">Maria P. De Miguel, et al. Proc Natl Acad Sci U S A. 2002 August 6;99(16):10458-10463.</div><p class="links"><a class="dblinks" href="/pubmed/12140361" ref="ordinalpos=1417&amp;ncbi_uid=378894&amp;link_uid=12140361">Citation</a><a class="dblinks" href="/pmc/articles/PMC124938/" ref="ordinalpos=1417&amp;ncbi_uid=378894&amp;link_uid=PMC124938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1418.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2843172/figure/F8/" ref="ordinalpos=1418&amp;ncbi_uid=2844546&amp;link_uid=PMC2843172" href="/pmc/articles/PMC2843172/"><img src="/pmc/articles/instance/2843172/bin/zbc0151009850008.gif" src-large="/pmc/articles/instance/2843172/bin/zbc0151009850008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2843172/" ref="ordinalpos=1418&amp;ncbi_uid=2844546&amp;link_uid=PMC2843172" image-link="/pmc/articles/PMC2843172/figure/F8/" class="imagepopup">FIGURE 8. From: CB1 Cannabinoid Receptors Increase Neuronal Precursor Proliferation through AKT/Glycogen Synthase Kinase-3?/?-Catenin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic drawing summarizing the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream from CB<sub>1</sub> receptors in cerebellar granule cell precursors.</b> Cannabinoids induce neuronal proliferation by CB<sub>1</sub> <span class="highlight" style="background-color:">signaling</span> via PI3K/AKT activation, GSK-3β inactivation, and β-catenin nuclear translocation.</div></p></div><div class="aux"><div class="resc">Stefania Trazzi, et al. J Biol Chem. 2010 March 26;285(13):10098-10109.</div><p class="links"><a class="dblinks" href="/pubmed/20083607" ref="ordinalpos=1418&amp;ncbi_uid=2844546&amp;link_uid=20083607">Citation</a><a class="dblinks" href="/pmc/articles/PMC2843172/" ref="ordinalpos=1418&amp;ncbi_uid=2844546&amp;link_uid=PMC2843172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1419.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1304483/figure/fig8/" ref="ordinalpos=1419&amp;ncbi_uid=664574&amp;link_uid=PMC1304483" href="/pmc/articles/PMC1304483/"><img src="/pmc/articles/instance/1304483/bin/biophysj00040469F08_HT.gif" src-large="/pmc/articles/instance/1304483/bin/biophysj00040469F08_HT.jpg" alt="FIGURE 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1304483/" ref="ordinalpos=1419&amp;ncbi_uid=664574&amp;link_uid=PMC1304483" image-link="/pmc/articles/PMC1304483/figure/fig8/" class="imagepopup">FIGURE 8.  From: <span class="highlight" style="background-color:">Signaling</span> in Small Subcellular Volumes. I. Stochastic and Diffusion Effects on Individual Pathways. </a></p><div class="supp"><p class="details"><div>Summary of stochastic and diffusive effects as a function of volume. “Severely degraded” indicates that the scatter or distortion in <span class="highlight" style="background-color:">signaling</span> response is so severe as to render the <span class="highlight" style="background-color:">pathway</span> output uninformative for downstream <span class="highlight" style="background-color:">signaling</span>. In the case of diffusive responses, “mixed” indicates that the response includes both washout and recruitment effects.</div></p></div><div class="aux"><div class="resc">Upinder S. Bhalla. Biophys J. 2004 August;87(2):733-744.</div><p class="links"><a class="dblinks" href="/pubmed/15298882" ref="ordinalpos=1419&amp;ncbi_uid=664574&amp;link_uid=15298882">Citation</a><a class="dblinks" href="/pmc/articles/PMC1304483/" ref="ordinalpos=1419&amp;ncbi_uid=664574&amp;link_uid=PMC1304483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1420.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3875765/figure/F7/" ref="ordinalpos=1420&amp;ncbi_uid=5685050&amp;link_uid=PMC3875765" href="/pmc/articles/PMC3875765/"><img src="/pmc/articles/instance/3875765/bin/oncotarget-04-1999-g007.gif" src-large="/pmc/articles/instance/3875765/bin/oncotarget-04-1999-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3875765/" ref="ordinalpos=1420&amp;ncbi_uid=5685050&amp;link_uid=PMC3875765" image-link="/pmc/articles/PMC3875765/figure/F7/" class="imagepopup">Figure 7. A proposed model for PDGF-D <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HCC GR EMT-type cells.  From: Chemoresistance to gemcitabine in hepatoma cells induces epithelial-mesenchymal transition and involves activation of PDGF-D <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>HCC GR cells acquired EMT phenotype where PDGF-D was up-regulated, leading to enhanced migration, invasion, and metastasis.</div></p></div><div class="aux"><div class="resc">Qiong Wu, et al. Oncotarget. 2013 November;4(11):1999-2009.</div><p class="links"><a class="dblinks" href="/pubmed/24158561" ref="ordinalpos=1420&amp;ncbi_uid=5685050&amp;link_uid=24158561">Citation</a><a class="dblinks" href="/pmc/articles/PMC3875765/" ref="ordinalpos=1420&amp;ncbi_uid=5685050&amp;link_uid=PMC3875765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1421.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2924916/figure/fig3/" ref="ordinalpos=1421&amp;ncbi_uid=3203044&amp;link_uid=PMC2924916" href="/pmc/articles/PMC2924916/"><img src="/pmc/articles/instance/2987418/bin/ejhg201062f3.gif" src-large="/pmc/articles/instance/2987418/bin/ejhg201062f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2924916/" ref="ordinalpos=1421&amp;ncbi_uid=3203044&amp;link_uid=PMC2924916" image-link="/pmc/articles/PMC2924916/figure/fig3/" class="imagepopup">Figure 3.  From: Genome-wide gene and <span class="highlight" style="background-color:">pathway</span> analysis. </a></p><div class="supp"><p class="details"><div><i>P</i>-values for testing association of the genes within the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with RA in NARAC studies. Blocks including significant genes are in red color, blocks including mild significant genes are in light red color and blocks including no significant genes are in green color.</div></p></div><div class="aux"><div class="resc">Li Luo, et al. Eur J Hum Genet. 2010 September;18(9):1045-1053.</div><p class="links"><a class="dblinks" href="/pubmed/20442747" ref="ordinalpos=1421&amp;ncbi_uid=3203044&amp;link_uid=20442747">Citation</a><a class="dblinks" href="/pmc/articles/PMC2924916/" ref="ordinalpos=1421&amp;ncbi_uid=3203044&amp;link_uid=PMC2924916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1422.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4202189/figure/F5/" ref="ordinalpos=1422&amp;ncbi_uid=6805683&amp;link_uid=PMC4202189" href="/pmc/articles/PMC4202189/"><img src="/pmc/articles/instance/4202189/bin/1753-6561-8-S6-S4-5.gif" src-large="/pmc/articles/instance/4202189/bin/1753-6561-8-S6-S4-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4202189/" ref="ordinalpos=1422&amp;ncbi_uid=6805683&amp;link_uid=PMC4202189" image-link="/pmc/articles/PMC4202189/figure/F5/" class="imagepopup">Figure 5.  From: Identifying common components across biological network graphs using a bipartite data model. </a></p><div class="supp"><p class="details"><div><b>Shared sub-graphs at the intersection of distinct <span class="highlight" style="background-color:">Pathway</span> Commons resources</b>. Maximal biclique analysis can rapidly isolate sub-pathways that are common between differing data sources, such as the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in this example. The Reactome BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown along with the sub-graph in common with the HumanCyc</div></p></div><div class="aux"><div class="resc">EJ Baker, et al. BMC Proc. 2014;8(Suppl 6):S4-S4.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1422&amp;ncbi_uid=6805683&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4202189/" ref="ordinalpos=1422&amp;ncbi_uid=6805683&amp;link_uid=PMC4202189">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1423.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2376838/figure/F8/" ref="ordinalpos=1423&amp;ncbi_uid=1355104&amp;link_uid=PMC2376838" href="/pmc/articles/PMC2376838/"><img src="/pmc/articles/instance/2376838/bin/nihms-45502-f0008.gif" src-large="/pmc/articles/instance/2376838/bin/nihms-45502-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2376838/" ref="ordinalpos=1423&amp;ncbi_uid=1355104&amp;link_uid=PMC2376838" image-link="/pmc/articles/PMC2376838/figure/F8/" class="imagepopup">Figure 8. Schematic diagram for LRG function in the unique <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by the full-length G-CSF receptor.  From: LRG-Accelerated Differentiation Defines Unique G-CSFR <span class="highlight" style="background-color:">Signaling</span> Pathways Downstream of PU.1 and C/EBP? That Modulate Neutrophil Activation. </a></p><div class="supp"><p class="details"><div>Refer to text for detailed description.</div></p></div><div class="aux"><div class="resc">Jing Ai, et al. J Leukoc Biol. ;83(5):1277-1285.</div><p class="links"><a class="dblinks" href="/pubmed/18272588" ref="ordinalpos=1423&amp;ncbi_uid=1355104&amp;link_uid=18272588">Citation</a><a class="dblinks" href="/pmc/articles/PMC2376838/" ref="ordinalpos=1423&amp;ncbi_uid=1355104&amp;link_uid=PMC2376838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1424.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2815088/figure/F4/" ref="ordinalpos=1424&amp;ncbi_uid=2732959&amp;link_uid=PMC2815088" href="/pmc/articles/PMC2815088/"><img src="/pmc/articles/instance/2815088/bin/nihms-136354-f0004.gif" src-large="/pmc/articles/instance/2815088/bin/nihms-136354-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2815088/" ref="ordinalpos=1424&amp;ncbi_uid=2732959&amp;link_uid=PMC2815088" image-link="/pmc/articles/PMC2815088/figure/F4/" class="imagepopup">Figure 4. The highly conserved insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is influenced by O-GlcNAc cycling.  From: The Hexosamine <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: O-GlcNAc cycling in feast or famine. </a></p><div class="supp"><p class="details"><div>The insulin <span class="highlight" style="background-color:">signaling</span> pathways of mammals, <i>D. melanogaster</i> and <i>C. elegans</i>, respectively are shown from top to bottom. All of the pathways are triggered by insulin-like peptides binding to an insulin-like receptor. The binding event triggers tyrosine phosphorylation of an insulin receptor substrate (IRS, termed <i>chico</i> in <i>Drosophila</i>). This activates the PI3K leading to recruitment of PDK1, activation of AKT and phosphorylation of a Foxo protein <i>(daf-16</i> in <i>C. elegans</i>). Phosphorylation of the Foxo protein leads to cytoplasmic ‘tethering’ of the transcription factor in the cytoplasm limiting its nuclear entry and transcriptional activation. Foxo may be O-GlcNAc-modified at the canonic AKT site leading to nuclear entry and transcriptional activation. In mammals, overexpression of OGT leads to insulin resistance by acting downstream of PI3K to modify key <span class="highlight" style="background-color:">signaling</span> molecules. Studies in both mammals, fly and <i>C. elegans</i> indicate that O-GlcNAc cycling impacts the insulin-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at several points downstream of PI3K <span class="highlight" style="background-color:">signaling</span> (indicated by the red shaded area). In <i>C. elegans</i> (bottom panel) knockouts of <i>ogt-1 and oga-1</i>, the enymes of O-GlcNAc cycling causes changes in macronutrient stores as indicated. Genetic analysis suggests that <i>ogt-1</i> is epistatic to <i>oga-1</i> as expected. Insulin <span class="highlight" style="background-color:">signaling</span> in <i>C. elegans</i> controls the dauer diapause, longevity, and stress reponses, all of which are impacted by O-GlcNAc cycling (See table at bottom right).</div></p></div><div class="aux"><div class="resc">John A. Hanover, et al. Biochim Biophys Acta. ;1800(2):80.</div><p class="links"><a class="dblinks" href="/pubmed/19647043" ref="ordinalpos=1424&amp;ncbi_uid=2732959&amp;link_uid=19647043">Citation</a><a class="dblinks" href="/pmc/articles/PMC2815088/" ref="ordinalpos=1424&amp;ncbi_uid=2732959&amp;link_uid=PMC2815088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1425.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3106143/figure/F2/" ref="ordinalpos=1425&amp;ncbi_uid=3358647&amp;link_uid=PMC3106143" href="/pmc/articles/PMC3106143/"><img src="/pmc/articles/instance/3106143/bin/nihms271494f2.gif" src-large="/pmc/articles/instance/3106143/bin/nihms271494f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3106143/" ref="ordinalpos=1425&amp;ncbi_uid=3358647&amp;link_uid=PMC3106143" image-link="/pmc/articles/PMC3106143/figure/F2/" class="imagepopup">Figure 2. The contribution of multiple pathways to a composite endpoint can result in oscillations.  From: Endocrine disruption via estrogen receptors that participate in nongenomic <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>This theoretical diagram of the contributions of <span class="highlight" style="background-color:">Pathway</span> A in conjunction with <span class="highlight" style="background-color:">Pathway</span> B can be applied to both temporal and dose considerations. An example of a common endpoint could be the phosphorylation level of a MAPK. Here <span class="highlight" style="background-color:">Pathway</span> A ends in endpoint activation early, or in response to low concentrations of a triggering estrogenic ligand. <span class="highlight" style="background-color:">Pathway</span> B has the same activation endpoint, but that <span class="highlight" style="background-color:">signaling</span> cascade arrives at the endpoint later, or in response to higher concentrations of the estrogenic ligand. However, since both pathways are activated by the same ligand-receptor interaction, the endpoint integrates these actions, the composite of which oscillates.</div></p></div><div class="aux"><div class="resc">Cheryl S. Watson, et al. J Steroid Biochem Mol Biol. ;127(1-2):44-50.</div><p class="links"><a class="dblinks" href="/pubmed/21300151" ref="ordinalpos=1425&amp;ncbi_uid=3358647&amp;link_uid=21300151">Citation</a><a class="dblinks" href="/pmc/articles/PMC3106143/" ref="ordinalpos=1425&amp;ncbi_uid=3358647&amp;link_uid=PMC3106143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1426.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2902905/figure/fig1/" ref="ordinalpos=1426&amp;ncbi_uid=2343156&amp;link_uid=PMC2902905" href="/pmc/articles/PMC2902905/"><img src="/pmc/articles/instance/2902905/bin/mplantssn069f01_4c.gif" src-large="/pmc/articles/instance/2902905/bin/mplantssn069f01_4c.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2902905/" ref="ordinalpos=1426&amp;ncbi_uid=2343156&amp;link_uid=PMC2902905" image-link="/pmc/articles/PMC2902905/figure/fig1/" class="imagepopup">Figure 1. From: Cell Polarity <span class="highlight" style="background-color:">Signaling</span>: Focus on Polar Auxin Transport. </a></p><div class="supp"><p class="details"><div>Sub-Cellular and <span class="highlight" style="background-color:">Signaling</span> Mechanisms Underlying Polar PIN Localization.<br />A model of regulators and pathways contributing to polar PIN localization in roots is proposed based on results discussed in this review. Endosome-mediated trafficking of PINs operates in regulating PIN polarity. At least two distinct pathways involving early endosomes are involved, which influence PIN1 and PIN2 trafficking through ES1-independent and ES1-dependent early endosomes, respectively. The SNX1/VPS29 retromer complex and the GNOM <span class="highlight" style="background-color:">pathway</span> appear to reside in late endosomes that may regulate PIN sorting, degradation in vacuoles, and recycling to the PM. Switching or re-polarization of PINs seems to involve distinct trafficking pathways as well. The GNOM-dependent <span class="highlight" style="background-color:">pathway</span> is involved in transcytosis-mediated polarity switching between the apical and basal sides, whereas the GNOM-independent and VSP29-dependent <span class="highlight" style="background-color:">pathway</span> may regulate PIN re-polarization to the lateral side. The precise relationship among early endosomes, TGN, late endosomes, recycling endosomes, and lytic endosomes remains unclear and is most likely quite complex. Speculative pathways are indicated by dotted lines. <span class="highlight" style="background-color:">Signaling</span> mechanisms regulating PIN trafficking are emerging. PID kinase directly phosphorylates PIN1 to direct the apical localization of PIN1. PP2A phosphatases have an opposite role by directing basipetal localization of PINs via the GNOM-mediated <span class="highlight" style="background-color:">pathway</span>. Calcium <span class="highlight" style="background-color:">signaling</span> and a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving PDK1 kinase are suggested to act upstream of those pathways. Sterols may also play some roles in generating polarity signal or regulate the formation of membrane microdomains necessary for the polar PIN localization. ROP <span class="highlight" style="background-color:">signaling</span>-mediated cell polarity may also contribute to PIN polarity <span class="highlight" style="background-color:">signaling</span> in an unknown manner. Auxin itself is shown to inhibit both ES1-insensitive PIN1 endocytosis and ES1-sensitive PIN2 endocytosis. There is also some evidence that auxin regulates PIN localization via the TIR1-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as well as having a more indirect effect by modulating PIN protein expression.</div></p></div><div class="aux"><div class="resc">Xiaowei Gao, et al. Mol Plant. 2008 November;1(6):899-909.</div><p class="links"><a class="dblinks" href="/pubmed/19825591" ref="ordinalpos=1426&amp;ncbi_uid=2343156&amp;link_uid=19825591">Citation</a><a class="dblinks" href="/pmc/articles/PMC2902905/" ref="ordinalpos=1426&amp;ncbi_uid=2343156&amp;link_uid=PMC2902905">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1427.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2920829/figure/pone-0012139-g003/" ref="ordinalpos=1427&amp;ncbi_uid=2416489&amp;link_uid=PMC2920829" href="/pmc/articles/PMC2920829/"><img src="/pmc/articles/instance/2920829/bin/pone.0012139.g003.gif" src-large="/pmc/articles/instance/2920829/bin/pone.0012139.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2920829/" ref="ordinalpos=1427&amp;ncbi_uid=2416489&amp;link_uid=PMC2920829" image-link="/pmc/articles/PMC2920829/figure/pone-0012139-g003/" class="imagepopup">Figure 3. Performance of the KEGG <span class="highlight" style="background-color:">pathway</span> prediction model..  From: A Genome-Wide Gene Function Prediction Resource for Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div>Receiver Operating Characteristic curves (<b>A</b>) and Precision-Recall curves (<b>B</b>) for the overall performance and contribution of each feature in the KEGG <span class="highlight" style="background-color:">pathway</span> prediction. Precision-Recall curves for the performance of the model in predicting metabolism only (<b>C</b>), <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> only (<b>D</b>), basic functions (<b>E</b>), and all non-metabolism functions (<b>F</b>).</div></p></div><div class="aux"><div class="resc">Han Yan, et al. PLoS One. 2010;5(8):e12139.</div><p class="links"><a class="dblinks" href="/pubmed/20711346" ref="ordinalpos=1427&amp;ncbi_uid=2416489&amp;link_uid=20711346">Citation</a><a class="dblinks" href="/pmc/articles/PMC2920829/" ref="ordinalpos=1427&amp;ncbi_uid=2416489&amp;link_uid=PMC2920829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1428.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3119055/figure/pone-0020754-g003/" ref="ordinalpos=1428&amp;ncbi_uid=3037670&amp;link_uid=PMC3119055" href="/pmc/articles/PMC3119055/"><img src="/pmc/articles/instance/3119055/bin/pone.0020754.g003.gif" src-large="/pmc/articles/instance/3119055/bin/pone.0020754.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3119055/" ref="ordinalpos=1428&amp;ncbi_uid=3037670&amp;link_uid=PMC3119055" image-link="/pmc/articles/PMC3119055/figure/pone-0020754-g003/" class="imagepopup">Figure 3. Classification of differentially expressed genes into various canonical pathways by Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis (IPA)..  From: Gene Expression Profiling of Preovulatory Follicle in the Buffalo Cow: Effects of Increased IGF-I Concentration on Periovulatory Events. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">pathway</span> analysis indicates that a large number of differentially expressed genes belonged to canonical pathways such as PI3K/AKT <span class="highlight" style="background-color:">signaling</span>; IGF-I <span class="highlight" style="background-color:">signaling</span> and coagulation system. The orange line represents score for the likelihood [−log (B–H <i>P</i>&lt;0.05)] that genes belonging to a specific <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> category affected at 1 and 22 h post peak LH surge. The stacked bars indicate the percentage of genes distributed according to regulation i.e. green (down), red (up) and open bars (No overlap with dataset) in each canonical <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jyotsna U. Rao, et al. PLoS One. 2011;6(6):e20754.</div><p class="links"><a class="dblinks" href="/pubmed/21701678" ref="ordinalpos=1428&amp;ncbi_uid=3037670&amp;link_uid=21701678">Citation</a><a class="dblinks" href="/pmc/articles/PMC3119055/" ref="ordinalpos=1428&amp;ncbi_uid=3037670&amp;link_uid=PMC3119055">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1429.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4261510/figure/F0002/" ref="ordinalpos=1429&amp;ncbi_uid=6955598&amp;link_uid=PMC4261510" href="/pmc/articles/PMC4261510/"><img src="/pmc/articles/instance/4261510/bin/BMB-47-540-g0002.gif" src-large="/pmc/articles/instance/4261510/bin/BMB-47-540-g0002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4261510/" ref="ordinalpos=1429&amp;ncbi_uid=6955598&amp;link_uid=PMC4261510" image-link="/pmc/articles/PMC4261510/figure/F0002/" class="imagepopup">Fig. 2.Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Schematic diagram for the core components and signal transduction of Hippo <span class="highlight" style="background-color:">pathway</span>. (A) When Hippo <span class="highlight" style="background-color:">signaling</span> is Off (in low cell density): The kinases MST1/2 and LATS are inactive, which results in inhibition of phosphorylation of YAP and TAZ. The stabilized YAP/TAZ in nuclei interacts with TEAD and enhances the transcription of target genes related to anti-apoptosis and proliferation. (B) When Hippo <span class="highlight" style="background-color:">signaling</span> is On (in high cell density): Activation of KIBRA and NF2 via unknown upstream <span class="highlight" style="background-color:">signaling</span> leads to activation of MST1/2. Activated MST1/2 phosphorylate SAV1 which in turn phosphorylate LATS and MOB1. The activated LATS/MOB phosphorylates YAP/TAZ which results in cytoplasmic retention by 14-3-3 protein and proteasomal degradation of YAP/TAZ. As a result, TEAD interacts with VGL4 and suppresses the expression of target genes..  From: Cross-talk between Wnt/β-catenin and Hippo <span class="highlight" style="background-color:">signaling</span> pathways: a brief review. </a></p><div class="aux"><div class="resc">Minseong Kim, et al. BMB Rep. 2014 October;47(10):540-545.</div><p class="links"><a class="dblinks" href="/pubmed/25154721" ref="ordinalpos=1429&amp;ncbi_uid=6955598&amp;link_uid=25154721">Citation</a><a class="dblinks" href="/pmc/articles/PMC4261510/" ref="ordinalpos=1429&amp;ncbi_uid=6955598&amp;link_uid=PMC4261510">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1430.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3071979/figure/F1/" ref="ordinalpos=1430&amp;ncbi_uid=3485237&amp;link_uid=PMC3071979" href="/pmc/articles/PMC3071979/"><img src="/pmc/articles/instance/3071979/bin/nihms280847f1.gif" src-large="/pmc/articles/instance/3071979/bin/nihms280847f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3071979/" ref="ordinalpos=1430&amp;ncbi_uid=3485237&amp;link_uid=PMC3071979" image-link="/pmc/articles/PMC3071979/figure/F1/" class="imagepopup">Figure 1. Top 10 canonical pathways.  From: Inhibition of NFKB <span class="highlight" style="background-color:">signaling</span> retards eosinophilic dermatitis in SHARPIN-deficient mice. </a></p><div class="supp"><p class="details"><div>NFKB <span class="highlight" style="background-color:">signaling</span> was highly associated pathways in the skin of <i>Sharpin<sup>cpdm</sup>/Sharpin<sup>cpdm</sup></i> mutant mice based on Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis<sup>®</sup> software.</div></p></div><div class="aux"><div class="resc">Yanhua Liang, et al. J Invest Dermatol. ;131(1):141-149.</div><p class="links"><a class="dblinks" href="/pubmed/20811394" ref="ordinalpos=1430&amp;ncbi_uid=3485237&amp;link_uid=20811394">Citation</a><a class="dblinks" href="/pmc/articles/PMC3071979/" ref="ordinalpos=1430&amp;ncbi_uid=3485237&amp;link_uid=PMC3071979">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1431.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2698598/figure/f6/" ref="ordinalpos=1431&amp;ncbi_uid=1851814&amp;link_uid=PMC2698598" href="/pmc/articles/PMC2698598/"><img src="/pmc/articles/instance/2698598/bin/zh60040967510006.gif" src-large="/pmc/articles/instance/2698598/bin/zh60040967510006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2698598/" ref="ordinalpos=1431&amp;ncbi_uid=1851814&amp;link_uid=PMC2698598" image-link="/pmc/articles/PMC2698598/figure/f6/" class="imagepopup">Fig. 6. From: Estrogen receptor ? mediates increased activation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> and improved myocardial function in female hearts following acute ischemia. </a></p><div class="supp"><p class="details"><div>Simplified schematic illustrates how the ERβ mediates activation of PI3K/Akt and apoptotic <span class="highlight" style="background-color:">pathway</span> and thus protects the female myocardium in response to I/R. ERβ-regulated JNK and ERK1/2 <span class="highlight" style="background-color:">signaling</span> is also shown.</div></p></div><div class="aux"><div class="resc">Meijing Wang, et al. Am J Physiol Regul Integr Comp Physiol. 2009 April;296(4):R972-R978.</div><p class="links"><a class="dblinks" href="/pubmed/19211725" ref="ordinalpos=1431&amp;ncbi_uid=1851814&amp;link_uid=19211725">Citation</a><a class="dblinks" href="/pmc/articles/PMC2698598/" ref="ordinalpos=1431&amp;ncbi_uid=1851814&amp;link_uid=PMC2698598">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1432.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1636849/figure/f8/" ref="ordinalpos=1432&amp;ncbi_uid=761807&amp;link_uid=PMC1636849" href="/pmc/articles/PMC1636849/"><img src="/pmc/articles/instance/1636849/bin/zmb0200662740008.gif" src-large="/pmc/articles/instance/1636849/bin/zmb0200662740008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1636849/" ref="ordinalpos=1432&amp;ncbi_uid=761807&amp;link_uid=PMC1636849" image-link="/pmc/articles/PMC1636849/figure/f8/" class="imagepopup">FIG. 8. From: The Grb2/Mek <span class="highlight" style="background-color:">Pathway</span> Represses Nanog in Murine Embryonic Stem Cells . </a></p><div class="supp"><p class="details"><div>Schematic summary of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to Nanog repression. Activation of the Grb2/Mek <span class="highlight" style="background-color:">pathway</span> induced by sodium vanadate or cell aggregation specifically represses the Nanog gene, resulting in primitive endoderm differentiation of ES cells. PTP, protein tyrosine phosphatase; Na<sub>3</sub>VO<sub>4</sub>, sodium vanadate.</div></p></div><div class="aux"><div class="resc">Takashi Hamazaki, et al. Mol Cell Biol. 2006 October;26(20):7539-7549.</div><p class="links"><a class="dblinks" href="/pubmed/16908534" ref="ordinalpos=1432&amp;ncbi_uid=761807&amp;link_uid=16908534">Citation</a><a class="dblinks" href="/pmc/articles/PMC1636849/" ref="ordinalpos=1432&amp;ncbi_uid=761807&amp;link_uid=PMC1636849">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1433.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3431819/figure/f4-ijms-13-09627/" ref="ordinalpos=1433&amp;ncbi_uid=4212616&amp;link_uid=PMC3431819" href="/pmc/articles/PMC3431819/"><img src="/pmc/articles/instance/3431819/bin/ijms-13-09627f4.gif" src-large="/pmc/articles/instance/3431819/bin/ijms-13-09627f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3431819/" ref="ordinalpos=1433&amp;ncbi_uid=4212616&amp;link_uid=PMC3431819" image-link="/pmc/articles/PMC3431819/figure/f4-ijms-13-09627/" class="imagepopup">Figure 4.  From: Arsenic Trioxide Inhibits Cell Growth and Induces Apoptosis through Inactivation of Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Breast Cancer. </a></p><div class="supp"><p class="details"><div>Inhibition of Notch-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by As<sub>2</sub>O<sub>3</sub> in breast cancer cells. (<b>A</b>) The Notch-1, Bcl-2 and NF-κB mRNA were detected by RT-PCR in breast cancer cells treated with varied concentrations of As<sub>2</sub>O<sub>3</sub> for 72 h; (<b>B</b>) The Notch-1, Bcl-2, and NF-κB proteins were measured by Western blotting analysis in breast cancer cells treated with varied concentrations of As<sub>2</sub>O<sub>3</sub> for 72 h; (C) The Notch-1 expression was detected by Western blotting analysis in breast cancer cells treated with 8 μM As<sub>2</sub>O<sub>3</sub> for different times.</div></p></div><div class="aux"><div class="resc">Jun Xia, et al. Int J Mol Sci. 2012;13(8):9627-9641.</div><p class="links"><a class="dblinks" href="/pubmed/22949821" ref="ordinalpos=1433&amp;ncbi_uid=4212616&amp;link_uid=22949821">Citation</a><a class="dblinks" href="/pmc/articles/PMC3431819/" ref="ordinalpos=1433&amp;ncbi_uid=4212616&amp;link_uid=PMC3431819">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1434.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3289920/figure/F2/" ref="ordinalpos=1434&amp;ncbi_uid=4661065&amp;link_uid=PMC3289920" href="/pmc/articles/PMC3289920/"><img src="/pmc/articles/instance/3289920/bin/bts018f2.gif" src-large="/pmc/articles/instance/3289920/bin/bts018f2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3289920/" ref="ordinalpos=1434&amp;ncbi_uid=4661065&amp;link_uid=PMC3289920" image-link="/pmc/articles/PMC3289920/figure/F2/" class="imagepopup">Fig. 2. From: Mining and integration of <span class="highlight" style="background-color:">pathway</span> diagrams from imaging data. </a></p><div class="supp"><p class="details"><div>Data associated with a <span class="highlight" style="background-color:">Pathway</span> in BiologicalNetworks system. (<b>A</b> and <b>A</b><sup>′</sup>) NF-<span style="font-variant: small-caps" class="small-caps">k</span>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> extracted from image (<b>B</b> and <b>B</b><sup>′</sup>). (<b>C</b>) <span class="highlight" style="background-color:">Pathway</span> Palette explaining <span class="highlight" style="background-color:">pathway</span> elements. (<b>D</b>) Pathways similar (in gene/protein members) to opened <span class="highlight" style="background-color:">pathway</span> are shown on the ‘Similar Pathways’ tabbed panel. (<b>E</b>) Multi-experiment viewer represents microarray experiments that are most relevant (by number of differentially expressed <span class="highlight" style="background-color:">pathway</span>'s gene members) searched over a wide variety of cell and tissue types. (<b>F</b>) Build <span class="highlight" style="background-color:">Pathway</span> Wizard (BPW) assists the user in finding regulatory paths, protein–protein, genetic and functional links, between selected objects, searches for common targets or regulators for the group of molecules, finds connection to canonical and other imaging pathways. (<b>G</b>) Node properties, (<b>H</b>) Link properties, including Abstracts from PudMed publications from which the link was extracted. (<b>I</b>) ‘Like/Don’t Like' buttons for <span class="highlight" style="background-color:">pathway</span> quality evaluation and users' comments.</div></p></div><div class="aux"><div class="resc">Sergey Kozhenkov, et al. Bioinformatics. 2012 March 1;28(5):739-742.</div><p class="links"><a class="dblinks" href="/pubmed/22267504" ref="ordinalpos=1434&amp;ncbi_uid=4661065&amp;link_uid=22267504">Citation</a><a class="dblinks" href="/pmc/articles/PMC3289920/" ref="ordinalpos=1434&amp;ncbi_uid=4661065&amp;link_uid=PMC3289920">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1435.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4122352/figure/pone-0101000-g007/" ref="ordinalpos=1435&amp;ncbi_uid=6487409&amp;link_uid=PMC4122352" href="/pmc/articles/PMC4122352/"><img src="/pmc/articles/instance/4122352/bin/pone.0101000.g007.gif" src-large="/pmc/articles/instance/4122352/bin/pone.0101000.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4122352/" ref="ordinalpos=1435&amp;ncbi_uid=6487409&amp;link_uid=PMC4122352" image-link="/pmc/articles/PMC4122352/figure/pone-0101000-g007/" class="imagepopup">Figure 7. Enhanced BMPRIA <span class="highlight" style="background-color:">signaling</span> activates Smad-dependent <span class="highlight" style="background-color:">pathway</span> but inhibits JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the condylar cartilage..  From: BMPRIA Mediated <span class="highlight" style="background-color:">Signaling</span> Is Essential for Temporomandibular Joint Development in Mice. </a></p><div class="supp"><p class="details"><div>Immunohistochemistry shows absent pSmad1/5, p-p38, and pERK, but the presence of pJNK in the E14.5 control condyle (A, C, E, G), and the presence of pSmad1/5, but absent pJNK as well as p-p38 and pERK in the transgenic condylar cartilage (B, D, F, H). Abbreviation: C, condylar cartilage; G, glenoid fossa; M, Meckel's cartilage. Scale bar = 100 µm.</div></p></div><div class="aux"><div class="resc">Shuping Gu, et al. PLoS One. 2014;9(8):e101000.</div><p class="links"><a class="dblinks" href="/pubmed/25093411" ref="ordinalpos=1435&amp;ncbi_uid=6487409&amp;link_uid=25093411">Citation</a><a class="dblinks" href="/pmc/articles/PMC4122352/" ref="ordinalpos=1435&amp;ncbi_uid=6487409&amp;link_uid=PMC4122352">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1436.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3894187/figure/pone-0085113-g004/" ref="ordinalpos=1436&amp;ncbi_uid=5721456&amp;link_uid=PMC3894187" href="/pmc/articles/PMC3894187/"><img src="/pmc/articles/instance/3894187/bin/pone.0085113.g004.gif" src-large="/pmc/articles/instance/3894187/bin/pone.0085113.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3894187/" ref="ordinalpos=1436&amp;ncbi_uid=5721456&amp;link_uid=PMC3894187" image-link="/pmc/articles/PMC3894187/figure/pone-0085113-g004/" class="imagepopup">Figure 4. Effect of ST6GAL1-activated PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on the expression of P-gp and MRP1..  From: Reversal Effect of ST6GAL 1 on Multidrug Resistance in Human Leukemia by Regulating the PI3K/Akt <span class="highlight" style="background-color:">Pathway</span> and the Expression of P-gp and MRP1. </a></p><div class="supp"><p class="details"><div>(A) The main molecules of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> were repressed at protein levels with ST6GAL1 shRNA transfection in K562/ADR cells. (B) The increased protein levels of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> molecules were determined by western blot in K562/ST6GAL1 cells. (C) Decreased expression of P-gp and MRP1 were examined by flow cytometry analysis in K562/ADR-ST6GAL1 shRNA cells. (D) Flow cytometry analysis revealed a higher co-expression of P-gp and MRP1 in K562 cells with ST6GAL1 transfection. The data are means ± SD of 3 independent assays (*P&lt;0.05).</div></p></div><div class="aux"><div class="resc">Hongye Ma, et al. PLoS One. 2014;9(1):e85113.</div><p class="links"><a class="dblinks" href="/pubmed/24454800" ref="ordinalpos=1436&amp;ncbi_uid=5721456&amp;link_uid=24454800">Citation</a><a class="dblinks" href="/pmc/articles/PMC3894187/" ref="ordinalpos=1436&amp;ncbi_uid=5721456&amp;link_uid=PMC3894187">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1437.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4002612/figure/fig4/" ref="ordinalpos=1437&amp;ncbi_uid=6626578&amp;link_uid=PMC4002612" href="/pmc/articles/PMC4002612/"><img src="/pmc/articles/instance/4002612/bin/aps20132f4.gif" src-large="/pmc/articles/instance/4002612/bin/aps20132f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4002612/" ref="ordinalpos=1437&amp;ncbi_uid=6626578&amp;link_uid=PMC4002612" image-link="/pmc/articles/PMC4002612/figure/fig4/" class="imagepopup">Figure 4.  From: Characterization of a novel curcumin analog P1 as potent inhibitor of the NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with distinct mechanisms. </a></p><div class="supp"><p class="details"><div>Effects of P1 on the inhibition of the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) EMSA analysis of NF-κB DNA-binding activity <i>in vitro</i>. (B) Immunofluorescence analysis of p65 nuclear translocation in HeLa cells. Cells were pretreated with DMSO or P1 for 6 h and then stimulated with or without TNF-α. (C) HeLa cells were pretreated with P1 for 6 h and then stimulated with TNF-α for 30 min. Nuclear proteins were prepared for Western blot analysis. (D) Western blot analysis of the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HeLa cells after treatment with P1 for 6 h. (E) The effect of P1 on IKK-β activity <i>in vitro</i>. (F) The effect of P1 on TNF-α-induced p38 activation after treatment for 6 h.</div></p></div><div class="aux"><div class="resc">Yan-min Peng, et al. Acta Pharmacol Sin. 2013 July 5;34(7):939-950.</div><p class="links"><a class="dblinks" href="/pubmed/23603982" ref="ordinalpos=1437&amp;ncbi_uid=6626578&amp;link_uid=23603982">Citation</a><a class="dblinks" href="/pmc/articles/PMC4002612/" ref="ordinalpos=1437&amp;ncbi_uid=6626578&amp;link_uid=PMC4002612">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1438.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3266971/figure/F1/" ref="ordinalpos=1438&amp;ncbi_uid=4154638&amp;link_uid=PMC3266971" href="/pmc/articles/PMC3266971/"><img src="/pmc/articles/instance/3266971/bin/nihms320016f1.gif" src-large="/pmc/articles/instance/3266971/bin/nihms320016f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3266971/" ref="ordinalpos=1438&amp;ncbi_uid=4154638&amp;link_uid=PMC3266971" image-link="/pmc/articles/PMC3266971/figure/F1/" class="imagepopup">Figure 1.  From: Involvement of the PI3K/Akt <span class="highlight" style="background-color:">Pathway</span> in Myxoid/Round Cell Liposarcoma. </a></p><div class="supp"><p class="details"><div>PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. Binding of growth factor to a receptor tyrosine kinase, such as IGF1R, activates PI3K, which phosphorylates a <span class="highlight" style="background-color:">signaling</span> molecule, phospatidyl inositol (PIP2), to its active form (PIP3). This in turn triggers a subsequent <span class="highlight" style="background-color:">signaling</span> cascade via Akt that results in phosphorylation of downstream targets such as S6 and 4EBP1. PI3K <span class="highlight" style="background-color:">signaling</span> is inhibited by PTEN, which dephosphorylates PIP3 to PIP2, thereby returning it to the inactive state.</div></p></div><div class="aux"><div class="resc">Elizabeth G. Demicco, et al. Mod Pathol. ;25(2):212-221.</div><p class="links"><a class="dblinks" href="/pubmed/22020193" ref="ordinalpos=1438&amp;ncbi_uid=4154638&amp;link_uid=22020193">Citation</a><a class="dblinks" href="/pmc/articles/PMC3266971/" ref="ordinalpos=1438&amp;ncbi_uid=4154638&amp;link_uid=PMC3266971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1439.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4260825/figure/pbio-1002012-g003/" ref="ordinalpos=1439&amp;ncbi_uid=6962503&amp;link_uid=PMC4260825" href="/pmc/articles/PMC4260825/"><img src="/pmc/articles/instance/4260825/bin/pbio.1002012.g003.gif" src-large="/pmc/articles/instance/4260825/bin/pbio.1002012.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4260825/" ref="ordinalpos=1439&amp;ncbi_uid=6962503&amp;link_uid=PMC4260825" image-link="/pmc/articles/PMC4260825/figure/pbio-1002012-g003/" class="imagepopup">Figure 3. Kinases are recruited to the <span class="highlight" style="background-color:">signaling</span> complex by alternative N-t localization domains..  From: The Robustness of a <span class="highlight" style="background-color:">Signaling</span> Complex to Domain Rearrangements Facilitates Network Evolution. </a></p><div class="supp"><p class="details"><div>(A) Comparison of mating <span class="highlight" style="background-color:">pathway</span> activation by kinase variants with or without N-t localization domains. In all cases, deletion of the full length kinase gene results in <span class="highlight" style="background-color:">pathway</span> inactivation. Expression of kinase variants lacking N-t localization domains only recovers partial <span class="highlight" style="background-color:">pathway</span> activation. In contrast, rearrangement events that fuse N-t domains known to interact with diverse partners in the mating <span class="highlight" style="background-color:">signaling</span> complex to the C-t kinase domains restore <span class="highlight" style="background-color:">pathway</span> activation to higher levels. (B) Schematic representation of the mutations introduced in Ste50 SAM domain, Ste5 RING domain, and Ste20 PBD domain. (C) <span class="highlight" style="background-color:">Pathway</span> activity for domain rearrangement variants carrying the mutations shown in (B) relative to the activity of the corresponding non-mutated variants. In most cases, mutations that disrupt specific recruitment interactions decrease <span class="highlight" style="background-color:">pathway</span> activation between 40%–50%. MUT Ste20[N]-Ste50[C] might still localize to the <span class="highlight" style="background-color:">signaling</span> complex, as Ste50's RA domain binds Cdc42 independently of Ste20's PBD domain [53]. (D) Fluorescence microscopy of GFP-tagged domain rearrangement variants shows that kinases can be recruited to the mating shmoo using alternative interaction domains. Statistically significant differences are marked with asterisks. Data shown in Data S1.</div></p></div><div class="aux"><div class="resc">Paloma M. Sato, et al. PLoS Biol. 2014 December;12(12):e1002012.</div><p class="links"><a class="dblinks" href="/pubmed/25490747" ref="ordinalpos=1439&amp;ncbi_uid=6962503&amp;link_uid=25490747">Citation</a><a class="dblinks" href="/pmc/articles/PMC4260825/" ref="ordinalpos=1439&amp;ncbi_uid=6962503&amp;link_uid=PMC4260825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1440.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3386560/figure/F8/" ref="ordinalpos=1440&amp;ncbi_uid=4711574&amp;link_uid=PMC3386560" href="/pmc/articles/PMC3386560/"><img src="/pmc/articles/instance/3386560/bin/nihms363895f8.gif" src-large="/pmc/articles/instance/3386560/bin/nihms363895f8.jpg" alt="FIGURE 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3386560/" ref="ordinalpos=1440&amp;ncbi_uid=4711574&amp;link_uid=PMC3386560" image-link="/pmc/articles/PMC3386560/figure/F8/" class="imagepopup">FIGURE 8.  From: A TRIF-INDEPENDENT BRANCH OF TLR3 <span class="highlight" style="background-color:">SIGNALING</span>. </a></p><div class="supp"><p class="details"><div>A new branch of TLR3 <span class="highlight" style="background-color:">signaling</span>. The Src-dependent branch of TLR3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is distinct from the gene induction branch. DsRNA-stimulated dimerization of TLR3 leads to Src recruitment, its auto-phosphorylation leading to transient functional activation, followed by sequestration in the lipid rafts as an inactive pool, causing inhibition of cell migration, proliferation and adhesion.</div></p></div><div class="aux"><div class="resc">Michifumi Yamashita, et al. J Immunol. ;188(6):2825-2833.</div><p class="links"><a class="dblinks" href="/pubmed/22323545" ref="ordinalpos=1440&amp;ncbi_uid=4711574&amp;link_uid=22323545">Citation</a><a class="dblinks" href="/pmc/articles/PMC3386560/" ref="ordinalpos=1440&amp;ncbi_uid=4711574&amp;link_uid=PMC3386560">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1441.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855473/figure/F2/" ref="ordinalpos=1441&amp;ncbi_uid=5569722&amp;link_uid=PMC3855473" href="/pmc/articles/PMC3855473/"><img src="/pmc/articles/instance/3855473/bin/nihms-492489-f0002.gif" src-large="/pmc/articles/instance/3855473/bin/nihms-492489-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855473/" ref="ordinalpos=1441&amp;ncbi_uid=5569722&amp;link_uid=PMC3855473" image-link="/pmc/articles/PMC3855473/figure/F2/" class="imagepopup">Fig. 2.  From: NF-?B as a target for oncogenic viruses. </a></p><div class="supp"><p class="details"><div>Tax-specific noncanonical NF-κB <span class="highlight" style="background-color:">pathway</span>. Noncanonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is stimulated in B cells by the BAFFR and CD40 signals. This <span class="highlight" style="background-color:">pathway</span> is negatively regulated by TRAF3, which recruits the E3 ubiquitin ligase c-IAP1 or c-IAP2 via TRAF2 and induces ubiquitin-dependent degradation of NIK. The receptor signals induce degradation of TRAF3 and TRAF2, leading to accumulation of NIK and NIK/IKKα-mediated p100 C-termimal phosphorylation. The phosphorylated p100 is then processed through the ubiquitin/proteasome <span class="highlight" style="background-color:">pathway</span> and produce the mature NF-κB2 p52 as a dimer with RelB. Under normal conditions, active processing of p100 does not occur in T cells. However, in HTLV1 infected T cells, Tax initiates an active noncanonical NF-κB <span class="highlight" style="background-color:">pathway</span> by bridging p100 and IKKα. In contrast to the cellular <span class="highlight" style="background-color:">pathway</span>, which is independent of NEMO, the viral <span class="highlight" style="background-color:">pathway</span> requires NEMO, which may function as an adaptor for Tax/IKKα association.</div></p></div><div class="aux"><div class="resc">Shao-Cong Sun, et al. Curr Top Microbiol Immunol. ;349:10.1007/82_2010_108.</div><p class="links"><a class="dblinks" href="/pubmed/20845110" ref="ordinalpos=1441&amp;ncbi_uid=5569722&amp;link_uid=20845110">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855473/" ref="ordinalpos=1441&amp;ncbi_uid=5569722&amp;link_uid=PMC3855473">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1442.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3464242/figure/pone-0046892-g001/" ref="ordinalpos=1442&amp;ncbi_uid=4287672&amp;link_uid=PMC3464242" href="/pmc/articles/PMC3464242/"><img src="/pmc/articles/instance/3464242/bin/pone.0046892.g001.gif" src-large="/pmc/articles/instance/3464242/bin/pone.0046892.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3464242/" ref="ordinalpos=1442&amp;ncbi_uid=4287672&amp;link_uid=PMC3464242" image-link="/pmc/articles/PMC3464242/figure/pone-0046892-g001/" class="imagepopup">Figure 1. Activation of Wnt <span class="highlight" style="background-color:">signaling</span> in the RPE cell line ARPE-19..  From: The Wnt/?-Catenin <span class="highlight" style="background-color:">Pathway</span> Cross-Talks with STAT3 <span class="highlight" style="background-color:">Signaling</span> to Regulate Survival of Retinal Pigment Epithelium Cells. </a></p><div class="supp"><p class="details"><div>(A) Cultured cells were co-transfected with the TOP-FLASH Wnt reporter plasmid and a plasmid containing β-galactosidase, and then incubated with Wnt3a or control conditioned media. Luciferase activity was normalized to β-galactosidase activity. Wnt3a induced a 32-fold increase in luciferase activity compared with control (n = 4, **p = 0.01). (B) Immunodetection of endogenous β-catenin was used to confirm Wnt <span class="highlight" style="background-color:">signaling</span> induction. Cells incubated in control conditioned media have predominantly cytoplasmic and membrane-associated β-catenin (red, arrowheads), indicating low basal levels of Wnt <span class="highlight" style="background-color:">signaling</span>. Stimulation with Wnt3a induced translocation of β-catenin into the nucleus (right, arrows), indicating activation of the Wnt <span class="highlight" style="background-color:">pathway</span>. Scale bar 50 µM.</div></p></div><div class="aux"><div class="resc">Miryam A. Fragoso, et al. PLoS One. 2012;7(10):e46892.</div><p class="links"><a class="dblinks" href="/pubmed/23056515" ref="ordinalpos=1442&amp;ncbi_uid=4287672&amp;link_uid=23056515">Citation</a><a class="dblinks" href="/pmc/articles/PMC3464242/" ref="ordinalpos=1442&amp;ncbi_uid=4287672&amp;link_uid=PMC3464242">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1443.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412720/figure/F4/" ref="ordinalpos=1443&amp;ncbi_uid=4162484&amp;link_uid=PMC3412720" href="/pmc/articles/PMC3412720/"><img src="/pmc/articles/instance/3412720/bin/1479-5876-10-116-4.gif" src-large="/pmc/articles/instance/3412720/bin/1479-5876-10-116-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412720/" ref="ordinalpos=1443&amp;ncbi_uid=4162484&amp;link_uid=PMC3412720" image-link="/pmc/articles/PMC3412720/figure/F4/" class="imagepopup">Figure 4.  From: JAK-STAT and AKT <span class="highlight" style="background-color:">pathway</span>-coupled genes in erythroid progenitor cells through ontogeny. </a></p><div class="supp"><p class="details"><div><b> AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The expression of genes linked to AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through ontogeny. (+p) phosphorylation, → stimulation, → translocation.</div></p></div><div class="aux"><div class="resc">Vladan P Cokic, et al. J Transl Med. 2012;10:116-116.</div><p class="links"><a class="dblinks" href="/pubmed/22676255" ref="ordinalpos=1443&amp;ncbi_uid=4162484&amp;link_uid=22676255">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412720/" ref="ordinalpos=1443&amp;ncbi_uid=4162484&amp;link_uid=PMC3412720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1444.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2924916/figure/fig2/" ref="ordinalpos=1444&amp;ncbi_uid=3203045&amp;link_uid=PMC2924916" href="/pmc/articles/PMC2924916/"><img src="/pmc/articles/instance/2987418/bin/ejhg201062f2.gif" src-large="/pmc/articles/instance/2987418/bin/ejhg201062f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2924916/" ref="ordinalpos=1444&amp;ncbi_uid=3203045&amp;link_uid=PMC2924916" image-link="/pmc/articles/PMC2924916/figure/fig2/" class="imagepopup">Figure 2.  From: Genome-wide gene and <span class="highlight" style="background-color:">pathway</span> analysis. </a></p><div class="supp"><p class="details"><div><i>P</i>-values for testing association of the genes within the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with RA in WTCCC studies. Blocks including significant genes are in red color, blocks including mild significant genes are in light red color and blocks including no significant genes are in green color.</div></p></div><div class="aux"><div class="resc">Li Luo, et al. Eur J Hum Genet. 2010 September;18(9):1045-1053.</div><p class="links"><a class="dblinks" href="/pubmed/20442747" ref="ordinalpos=1444&amp;ncbi_uid=3203045&amp;link_uid=20442747">Citation</a><a class="dblinks" href="/pmc/articles/PMC2924916/" ref="ordinalpos=1444&amp;ncbi_uid=3203045&amp;link_uid=PMC2924916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1445.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4107945/figure/F1/" ref="ordinalpos=1445&amp;ncbi_uid=6468613&amp;link_uid=PMC4107945" href="/pmc/articles/PMC4107945/"><img src="/pmc/articles/instance/4107945/bin/fimmu-05-00342-g001.gif" src-large="/pmc/articles/instance/4107945/bin/fimmu-05-00342-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4107945/" ref="ordinalpos=1445&amp;ncbi_uid=6468613&amp;link_uid=PMC4107945" image-link="/pmc/articles/PMC4107945/figure/F1/" class="imagepopup">Figure 1.  From: Pattern Recognition and <span class="highlight" style="background-color:">Signaling</span> Mechanisms of RIG-I and MDA5. </a></p><div class="supp"><p class="details"><div><b>The RLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown</b>. RIG-I and MDA5 recognize a complementary set of cytosolic viral dsRNA ligands. Their activation is tightly regulated by phosphorylation, ubiquitination, and host proteins such as LGP2. RIG-I and MDA5 signal to MAVS, which initiates the production of interferon <span class="highlight" style="background-color:">signaling</span>. Circled “P” indicates phosphorylation and slashed circled “P” indicates dephosphorylation.</div></p></div><div class="aux"><div class="resc">Stephanie Reikine, et al. Front Immunol. 2014;5:342.</div><p class="links"><a class="dblinks" href="/pubmed/25101084" ref="ordinalpos=1445&amp;ncbi_uid=6468613&amp;link_uid=25101084">Citation</a><a class="dblinks" href="/pmc/articles/PMC4107945/" ref="ordinalpos=1445&amp;ncbi_uid=6468613&amp;link_uid=PMC4107945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1446.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1939900/figure/F6/" ref="ordinalpos=1446&amp;ncbi_uid=1016421&amp;link_uid=PMC1939900" href="/pmc/articles/PMC1939900/"><img src="/pmc/articles/instance/1939900/bin/zh80140704970006.gif" src-large="/pmc/articles/instance/1939900/bin/zh80140704970006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1939900/" ref="ordinalpos=1446&amp;ncbi_uid=1016421&amp;link_uid=PMC1939900" image-link="/pmc/articles/PMC1939900/figure/F6/" class="imagepopup">Figure 6.  From: Regulation of COX-2-mediated <span class="highlight" style="background-color:">signaling</span> by ?3 type IV noncollagenous domain in tumor angiogenesis. </a></p><div class="supp"><p class="details"><div><b>Schematic illustration of different <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by α3(IV)NC1.</b> α3(IV)NC1 binds to αVβ3 and α3β1 integrins and inhibit phosphorylation of FAK. Inhibition of FAK activation leads to inhibition of FAK/phosphatidylinositol 3-kinase/eIF4E/4E-BP1. In addition, α3(IV)NC1 also inhibits NFκB-mediated <span class="highlight" style="background-color:">signaling</span> in hypoxic conditions leads to inhibition of COX-2/VEGF/bFGF expressions, resulting in inhibition of hypoxic tumor angiogenesis.</div></p></div><div class="aux"><div class="resc">Chandra Shekhar Boosani, et al. Blood. 2007 August 15;110(4):1168-1177.</div><p class="links"><a class="dblinks" href="/pubmed/17426256" ref="ordinalpos=1446&amp;ncbi_uid=1016421&amp;link_uid=17426256">Citation</a><a class="dblinks" href="/pmc/articles/PMC1939900/" ref="ordinalpos=1446&amp;ncbi_uid=1016421&amp;link_uid=PMC1939900">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1447.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1868540/figure/F1/" ref="ordinalpos=1447&amp;ncbi_uid=1055914&amp;link_uid=PMC1868540" href="/pmc/articles/PMC1868540/"><img src="/pmc/articles/instance/1868540/bin/nihms20764f1.gif" src-large="/pmc/articles/instance/1868540/bin/nihms20764f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1868540/" ref="ordinalpos=1447&amp;ncbi_uid=1055914&amp;link_uid=PMC1868540" image-link="/pmc/articles/PMC1868540/figure/F1/" class="imagepopup">Fig. 1.  From: 1,25-DIHYDROXYVITAMIN D3 INDUCES EXPRESSION OF THE WNT <span class="highlight" style="background-color:">SIGNALING</span> CO-REGULATOR LRP5 VIA REGULATORY ELEMENTS LOCATED SIGNIFICANTLY DOWNSTREAM OF THE GENE'S TRANSCRIPTIONAL START SITE. </a></p><div class="supp"><p class="details"><div>Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. A, Binding of Wnt to Frizzled and LRP5 results in stabilization of β-catenin and consequently activation of TCF/LEF-responsive gene transcription. B, Inhibition of Wnt <span class="highlight" style="background-color:">signaling</span> occurs when LRP5 is shunted away through interactions with antagonistic cytokines.</div></p></div><div class="aux"><div class="resc">Jackie A. Fretz, et al. J Steroid Biochem Mol Biol. ;103(3-5):440-445.</div><p class="links"><a class="dblinks" href="/pubmed/17229572" ref="ordinalpos=1447&amp;ncbi_uid=1055914&amp;link_uid=17229572">Citation</a><a class="dblinks" href="/pmc/articles/PMC1868540/" ref="ordinalpos=1447&amp;ncbi_uid=1055914&amp;link_uid=PMC1868540">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1448.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3039931/figure/A002352F2/" ref="ordinalpos=1448&amp;ncbi_uid=4654410&amp;link_uid=PMC3039931" href="/pmc/articles/PMC3039931/"><img src="/pmc/articles/instance/3039931/bin/cshperspect-IMS-002352_F2.gif" src-large="/pmc/articles/instance/3039931/bin/cshperspect-IMS-002352_F2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3039931/" ref="ordinalpos=1448&amp;ncbi_uid=4654410&amp;link_uid=PMC3039931" image-link="/pmc/articles/PMC3039931/figure/A002352F2/" class="imagepopup">Figure 2. From: Src-family and Syk Kinases in Activating and Inhibitory Pathways in Innate Immune Cells: <span class="highlight" style="background-color:">Signaling</span> Cross Talk. </a></p><div class="supp"><p class="details"><div>Tyrosine kinase pathways leading to CARD9 and TLR <span class="highlight" style="background-color:">signaling</span> responses. The central role of the CARD9/Malt1/Bcl10 complex in activating primarily the NF-κB <span class="highlight" style="background-color:">pathway</span> and to a lesser extent the MAPK <span class="highlight" style="background-color:">pathway</span> is shown (see Colonna 2007). Engagement of C-type lectin receptors, leading to Syk activation, is coupled to the CARD9 complex through an unclear <span class="highlight" style="background-color:">pathway</span>, but may involve PKCs. This in turn directly leads to activation of IKK, which in turn activates downstream NF-κB pathways (see Hara and Saito 2009). The importance of this <span class="highlight" style="background-color:">pathway</span> in innate immune cell recognition has been revealed by the similar defect in host defense in mice lacking either the C-type lectin receptors, Syk, or CARD9 (Gross et al. 2006). The CARD9 complex may also facilitate the ability of the TLRs to engage downstream MAPK <span class="highlight" style="background-color:">pathway</span> activation. Upstream of the TLRs, the major cytoplasmic tyrosine kinase involved in this innate immune receptor function is Tec-family members, as revealed by poor TLR <span class="highlight" style="background-color:">signaling</span> function in cells derived from either mice or humans lacking various Tec kinases. It remains unclear how the Tec kinases operate in the TLR <span class="highlight" style="background-color:">pathway</span>, but could involve phosphorylation of <span class="highlight" style="background-color:">signaling</span> adapters MyD88 or Mal.</div></p></div><div class="aux"><div class="resc">Clifford A. Lowell. Cold Spring Harb Perspect Biol. 2011 March;3(3):a002352.</div><p class="links"><a class="dblinks" href="/pubmed/21068150" ref="ordinalpos=1448&amp;ncbi_uid=4654410&amp;link_uid=21068150">Citation</a><a class="dblinks" href="/pmc/articles/PMC3039931/" ref="ordinalpos=1448&amp;ncbi_uid=4654410&amp;link_uid=PMC3039931">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1449.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC140053/figure/cdg009f9/" ref="ordinalpos=1449&amp;ncbi_uid=363780&amp;link_uid=PMC140053" href="/pmc/articles/PMC140053/"><img src="/pmc/articles/instance/140053/bin/cdg009f9.gif" src-large="/pmc/articles/instance/140053/bin/cdg009f9.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC140053/" ref="ordinalpos=1449&amp;ncbi_uid=363780&amp;link_uid=PMC140053" image-link="/pmc/articles/PMC140053/figure/cdg009f9/" class="imagepopup">Figure. From: Protein kinase D mediates a stress-induced NF-?B activation and survival <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Fig. 9.</b> Model for the NF-κB activation <span class="highlight" style="background-color:">pathway</span> induced by oxidative stress. Oxidative stress induces the transcriptional activity of NF-κB by activating the Src–Abl <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, leading to phosphorylation of PKD at Y463 in the PH domain. Oxidative stress also leads to activation of PKCs, which mediate phosphorylation of the PKD activation-loop residues S738/742 in the catalytic kinase domain. These two events result in a fully-active kinase, which mediates activation of IKKβ and degradation of IκBα, leading to the activation of NF-κB. This <span class="highlight" style="background-color:">pathway</span> contributes to the survival of cells exposed to oxidative stress.</div></p></div><div class="aux"><div class="resc">Peter Storz, et al. EMBO J. 2003 January 2;22(1):109-120.</div><p class="links"><a class="dblinks" href="/pubmed/12505989" ref="ordinalpos=1449&amp;ncbi_uid=363780&amp;link_uid=12505989">Citation</a><a class="dblinks" href="/pmc/articles/PMC140053/" ref="ordinalpos=1449&amp;ncbi_uid=363780&amp;link_uid=PMC140053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1450.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3184159/figure/pone-0025559-g002/" ref="ordinalpos=1450&amp;ncbi_uid=3272004&amp;link_uid=PMC3184159" href="/pmc/articles/PMC3184159/"><img src="/pmc/articles/instance/3184159/bin/pone.0025559.g002.gif" src-large="/pmc/articles/instance/3184159/bin/pone.0025559.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3184159/" ref="ordinalpos=1450&amp;ncbi_uid=3272004&amp;link_uid=PMC3184159" image-link="/pmc/articles/PMC3184159/figure/pone-0025559-g002/" class="imagepopup">Figure 2. CART analysis of genetic polymorphisms in the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and risk of ovarian cancer..  From: Genetic Variants in TGF-? <span class="highlight" style="background-color:">Pathway</span> Are Associated with Ovarian Cancer Risk. </a></p><div class="supp"><p class="details"><div>ORs and 95% CIs (in parenthesis) are presented underneath each terminal node.</div></p></div><div class="aux"><div class="resc">Jikai Yin, et al. PLoS One. 2011;6(9):e25559.</div><p class="links"><a class="dblinks" href="/pubmed/21984931" ref="ordinalpos=1450&amp;ncbi_uid=3272004&amp;link_uid=21984931">Citation</a><a class="dblinks" href="/pmc/articles/PMC3184159/" ref="ordinalpos=1450&amp;ncbi_uid=3272004&amp;link_uid=PMC3184159">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1451.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4150361/figure/F1/" ref="ordinalpos=1451&amp;ncbi_uid=6612694&amp;link_uid=PMC4150361" href="/pmc/articles/PMC4150361/"><img src="/pmc/articles/instance/4150361/bin/fnagi-06-00232-g0001.gif" src-large="/pmc/articles/instance/4150361/bin/fnagi-06-00232-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4150361/" ref="ordinalpos=1451&amp;ncbi_uid=6612694&amp;link_uid=PMC4150361" image-link="/pmc/articles/PMC4150361/figure/F1/" class="imagepopup">Figure 1.  From: Cyclin-dependent kinase 5, a node protein in diminished tauopathy: a systems biology approach. </a></p><div class="supp"><p class="details"><div><b>Scheme of CDK5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in Tau phosphorylation</b>. This draw shows the CDK5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> describing direct and indirect Tau phosphorylation by CDK5.</div></p></div><div class="aux"><div class="resc">John F. Castro-Alvarez, et al. Front Aging Neurosci. 2014;6:232.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1451&amp;ncbi_uid=6612694&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4150361/" ref="ordinalpos=1451&amp;ncbi_uid=6612694&amp;link_uid=PMC4150361">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1452.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2806696/figure/fig02/" ref="ordinalpos=1452&amp;ncbi_uid=2208656&amp;link_uid=PMC2806696" href="/pmc/articles/PMC2806696/"><img src="/pmc/articles/instance/2806696/bin/pnas.0909734107fig02.gif" src-large="/pmc/articles/instance/2806696/bin/pnas.0909734107fig02.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2806696/" ref="ordinalpos=1452&amp;ncbi_uid=2208656&amp;link_uid=PMC2806696" image-link="/pmc/articles/PMC2806696/figure/fig02/" class="imagepopup">Fig. 2. From: A genomewide study identifies the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a major target of p53 in murine embryonic stem cells. </a></p><div class="supp"><p class="details"><div>The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is significantly enriched in p53 target. (<i>A</i>) A Venn diagram of genes bound by p53 with adriamcyin treatment from ChIP-chip and genes that were significantly altered in mESCs treated with adriamycin detected in gene expression microarray (false discovery rate corrected <i>P</i> value &lt;0.05). Only genes that were represented by both ChIP-chip and gene expression microarray platforms were included in the analysis. (<i>B</i>) Heatmap of genes in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> enriched in the overlapped gene list shown in <i>A</i>. (<i>C</i>) Real-time PCR result of the fold induction of enriched genes in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in p53<sup>+/+</sup> and p53<sup>−/−</sup> mESCs treated with adriamycin for 8 h. Results are displayed as mean ± SEM; <i>n</i> = 3; *<i>P</i> &lt; 0.05; ns, not significant. (<i>D</i>) Binding of p53 on the <i>Wnt8a</i> and <i>Wnt3</i> genes from ChIP-chip.</div></p></div><div class="aux"><div class="resc">Kyoung-Hwa Lee, et al. Proc Natl Acad Sci U S A. 2010 January 5;107(1):69-74.</div><p class="links"><a class="dblinks" href="/pubmed/20018659" ref="ordinalpos=1452&amp;ncbi_uid=2208656&amp;link_uid=20018659">Citation</a><a class="dblinks" href="/pmc/articles/PMC2806696/" ref="ordinalpos=1452&amp;ncbi_uid=2208656&amp;link_uid=PMC2806696">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1453.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC153719/figure/fig9/" ref="ordinalpos=1453&amp;ncbi_uid=286423&amp;link_uid=PMC153719" href="/pmc/articles/PMC153719/"><img src="/pmc/articles/instance/153719/bin/010413f9.gif" src-large="/pmc/articles/instance/153719/bin/010413f9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC153719/" ref="ordinalpos=1453&amp;ncbi_uid=286423&amp;link_uid=PMC153719" image-link="/pmc/articles/PMC153719/figure/fig9/" class="imagepopup">Figure 9. From: Dominant-Negative Receptor Uncovers Redundancy in the Arabidopsis ERECTA Leucine-Rich Repeat Receptor-Like Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> That Regulates Organ Shape. </a></p><div class="supp"><p class="details"><div>Hypothetical Model for the ERECTA <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and ΔKinase Action.<br /><b>(A)</b> In wild-type plants, ERECTA associates with multiple partners (orange). Additional RLKs (blue) may share ligands (red) as well as receptor partners of ERECTA. The activation of these pathways promotes internode/organ elongation.<br /><b>(B)</b> <i>erecta-105</i> plants do not have any ERECT. However, the receptor complex with overlapping function partially promotes internode/organ elongation.<br /><b>(C)</b> In <i>ERECTA</i>::<i>ΔKinase/erecta-105</i> plants, highly stable ERECTA ΔKinase fragments shut down the entire <span class="highlight" style="background-color:">pathway</span>, presumably by forming inactive receptor heterodimers and/or depleting ligands for the RLK that has a function partially redundant with that of ERECTA.</div></p></div><div class="aux"><div class="resc">Elena D. Shpak, et al. Plant Cell. 2003 May;15(5):1095-1110.</div><p class="links"><a class="dblinks" href="/pubmed/12724536" ref="ordinalpos=1453&amp;ncbi_uid=286423&amp;link_uid=12724536">Citation</a><a class="dblinks" href="/pmc/articles/PMC153719/" ref="ordinalpos=1453&amp;ncbi_uid=286423&amp;link_uid=PMC153719">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1454.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3000242/figure/f1/" ref="ordinalpos=1454&amp;ncbi_uid=3596848&amp;link_uid=PMC3000242" href="/pmc/articles/PMC3000242/"><img src="/pmc/articles/instance/3000242/bin/fig-1.gif" src-large="/pmc/articles/instance/3000242/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3000242/" ref="ordinalpos=1454&amp;ncbi_uid=3596848&amp;link_uid=PMC3000242" image-link="/pmc/articles/PMC3000242/figure/f1/" class="imagepopup">FIG. 1. From: Nonhuman Primate Calorie Restriction. </a></p><div class="supp"><p class="details"><div><b>Calorie restriction (CR) is not mechanistically equivalent to lifespan extension by genetic manipulation leading to reduced insulin <span class="highlight" style="background-color:">signaling</span>.</b> The insulin/insulin-like growth factor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has been implicated in regulating longevity in numerous species. Transgenic, knockout, and mutant genetic variants have demonstrated that reduced <span class="highlight" style="background-color:">signaling</span> through this axis can increase lifespan in yeast (glucose <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>), worms, flies, and mice. Studies of CR in these species reveal that there is evidence to support reduced <span class="highlight" style="background-color:">signaling</span> through this axis as an outcome of CR, but that the pathways for lifespan extension are not identical and that other factors distinct from insulin/insulin-like growth factor play a role in the mechanism of CR.</div></p></div><div class="aux"><div class="resc">Ricki J. Colman, et al. Antioxid Redox Signal. 2011 January 15;14(2):229-239.</div><p class="links"><a class="dblinks" href="/pubmed/20698791" ref="ordinalpos=1454&amp;ncbi_uid=3596848&amp;link_uid=20698791">Citation</a><a class="dblinks" href="/pmc/articles/PMC3000242/" ref="ordinalpos=1454&amp;ncbi_uid=3596848&amp;link_uid=PMC3000242">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1455.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136910/figure/F2/" ref="ordinalpos=1455&amp;ncbi_uid=3090615&amp;link_uid=PMC3136910" href="/pmc/articles/PMC3136910/"><img src="/pmc/articles/instance/3136910/bin/sgtp0202_0089_fig002.gif" src-large="/pmc/articles/instance/3136910/bin/sgtp0202_0089_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136910/" ref="ordinalpos=1455&amp;ncbi_uid=3090615&amp;link_uid=PMC3136910" image-link="/pmc/articles/PMC3136910/figure/F2/" class="imagepopup">Figure 2.  From: Characterization of ATF2 in Rel/NF?B oncogenesis reveals its role in the regulation of Ras <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed model of ATF2 regulation of Ha-Ras <span class="highlight" style="background-color:">signaling</span>. (A) Activation of Ras-Raf-MEK-ERK <span class="highlight" style="background-color:">signaling</span> by v-Rel. The expression of v-Rel in fibroblast and lymphoid cells results in the induction of Ha-Ras activity. Activated Ha-Ras leads to the phosphorylation and activation of c-Raf (serine 338) and subsequent activation of MEK1/2 and ERK. The activation of ERK by this <span class="highlight" style="background-color:">signaling</span> cascade leads to the altered expression of a variety of genes. (B) ATF2 blocks Ha-Ras activation by v-Rel. The ectopic expression of ATF2 with v-Rel blocks v-Rel-mediated activation of Ha-Ras and the downstream Raf-MEK-ERK <span class="highlight" style="background-color:">pathway</span>. (C) The ectopic expression of ATF2 alone in DT40 cells has multiple effects on Ha-Ras <span class="highlight" style="background-color:">signaling</span>. Ectopically expressed ATF2 is phosphorylated and activated by p38MAPK resulting in the induction of Ha-Ras activity. However, the activation of the downstream Raf-MEK-ERK <span class="highlight" style="background-color:">pathway</span> is blocked by phosphorylation of c-Raf at serine 259, which likely allows for binding of the inhibitory protein 14-3-3.</div></p></div><div class="aux"><div class="resc">Andrew S Liss, et al. Small GTPases. 2011 Mar-Apr;2(2):89-94.</div><p class="links"><a class="dblinks" href="/pubmed/21776408" ref="ordinalpos=1455&amp;ncbi_uid=3090615&amp;link_uid=21776408">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136910/" ref="ordinalpos=1455&amp;ncbi_uid=3090615&amp;link_uid=PMC3136910">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1456.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3200079/figure/fig3/" ref="ordinalpos=1456&amp;ncbi_uid=3319639&amp;link_uid=PMC3200079" href="/pmc/articles/PMC3200079/"><img src="/pmc/articles/instance/3200079/bin/BMR2011-502751.003.gif" src-large="/pmc/articles/instance/3200079/bin/BMR2011-502751.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3200079/" ref="ordinalpos=1456&amp;ncbi_uid=3319639&amp;link_uid=PMC3200079" image-link="/pmc/articles/PMC3200079/figure/fig3/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Signaling</span> Proteins and Transcription Factors in Normal and Malignant Early B Cell Development. </a></p><div class="supp"><p class="details"><div>Homeostatic and leukemic expression of receptors, <span class="highlight" style="background-color:">signaling</span> proteins, and transcription factors along the B cell <span class="highlight" style="background-color:">pathway</span>. Developmental stages are indicated starting with the hematopoietic stem cell (HSC), the common lymphoid progenitor (CLP), and into the B cell <span class="highlight" style="background-color:">pathway</span>, stages proB, preB, and mature B cells. Normal gene expression along the developmental <span class="highlight" style="background-color:">pathway</span> is indicated with blue bars, and expression dependency between proteins is indicated with arrows. Most common modified forms of these receptors, <span class="highlight" style="background-color:">signaling</span> proteins, and transcription factors associated with acute lymphoblastic leukemia are also indicated.</div></p></div><div class="aux"><div class="resc">Patricia Pérez-Vera, et al. Bone Marrow Res. 2011;2011:502751.</div><p class="links"><a class="dblinks" href="/pubmed/22046564" ref="ordinalpos=1456&amp;ncbi_uid=3319639&amp;link_uid=22046564">Citation</a><a class="dblinks" href="/pmc/articles/PMC3200079/" ref="ordinalpos=1456&amp;ncbi_uid=3319639&amp;link_uid=PMC3200079">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1457.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2615499/figure/fig9/" ref="ordinalpos=1457&amp;ncbi_uid=1688225&amp;link_uid=PMC2615499" href="/pmc/articles/PMC2615499/"><img src="/pmc/articles/instance/2615499/bin/zbc0060962760009.gif" src-large="/pmc/articles/instance/2615499/bin/zbc0060962760009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2615499/" ref="ordinalpos=1457&amp;ncbi_uid=1688225&amp;link_uid=PMC2615499" image-link="/pmc/articles/PMC2615499/figure/fig9/" class="imagepopup">FIGURE 9. From: Helicobacter pylori VacA-induced Inhibition of GSK3 through the  PI3K/Akt <span class="highlight" style="background-color:">Signaling</span>  <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>VacA or an <i>H. pylori vacA</i>-positive strain induced activation of  PI3K p110α/Akt <span class="highlight" style="background-color:">pathway</span> leading to inactivation of GSK3β and  enhanced transcriptional activity of β-catenin.</b> VacA or an <i>H.  pylori vacA</i>-positive strain activated the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>,  resulting in phosphorylation and inhibition of GSK3β, and subsequent  translocation of β-catenin to the nucleus, consistent with effects of  VacA on β-catenin-regulated transcriptional activity.</div></p></div><div class="aux"><div class="resc">Masaaki Nakayama, et al. J Biol Chem. 2009 January 16;284(3):1612-1619.</div><p class="links"><a class="dblinks" href="/pubmed/18996844" ref="ordinalpos=1457&amp;ncbi_uid=1688225&amp;link_uid=18996844">Citation</a><a class="dblinks" href="/pmc/articles/PMC2615499/" ref="ordinalpos=1457&amp;ncbi_uid=1688225&amp;link_uid=PMC2615499">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1458.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2733081/figure/f4-grsb-2008-253/" ref="ordinalpos=1458&amp;ncbi_uid=2011416&amp;link_uid=PMC2733081" href="/pmc/articles/PMC2733081/"><img src="/pmc/articles/instance/2733081/bin/grsb-2008-253f4.gif" src-large="/pmc/articles/instance/2733081/bin/grsb-2008-253f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2733081/" ref="ordinalpos=1458&amp;ncbi_uid=2011416&amp;link_uid=PMC2733081" image-link="/pmc/articles/PMC2733081/figure/f4-grsb-2008-253/" class="imagepopup">Figure 4. Schematic representation of the <span class="highlight" style="background-color:">signaling</span> pathways affected by TPPII knockdown.  From: The MAPK <span class="highlight" style="background-color:">Signaling</span> Cascade is a Central Hub in the Regulation of Cell Cycle, Apoptosis and Cytoskeleton Remodeling by Tripeptidyl-Peptidase II. </a></p><div class="supp"><p class="details"><div>KEGG <span class="highlight" style="background-color:">pathway</span> analysis of the differentially regulated genes demonstrates enrichment of components of the MAPK, Focal adhesion and TGF-β <span class="highlight" style="background-color:">signaling</span>. The annotated relationship between the regulated gene products is illustrated as activation (↓) or inhibition (⊥). Genes that are up- and down-regulated in TPPII knockdown cells are indicated by thick and dash-line boarders, respectively.</div></p></div><div class="aux"><div class="resc">Ramakrishna Sompallae, et al. Gene Regul Syst Bio. 2008;2:253-265.</div><p class="links"><a class="dblinks" href="/pubmed/19787088" ref="ordinalpos=1458&amp;ncbi_uid=2011416&amp;link_uid=19787088">Citation</a><a class="dblinks" href="/pmc/articles/PMC2733081/" ref="ordinalpos=1458&amp;ncbi_uid=2011416&amp;link_uid=PMC2733081">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1459.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3971589/figure/evu038-F1/" ref="ordinalpos=1459&amp;ncbi_uid=5962967&amp;link_uid=PMC3971589" href="/pmc/articles/PMC3971589/"><img src="/pmc/articles/instance/3971589/bin/evu038f1.gif" src-large="/pmc/articles/instance/3971589/bin/evu038f1.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 1.—" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3971589/" ref="ordinalpos=1459&amp;ncbi_uid=5962967&amp;link_uid=PMC3971589" image-link="/pmc/articles/PMC3971589/figure/evu038-F1/" class="imagepopup">F ig . 1.—.  From: The Evolution of the GPCR <span class="highlight" style="background-color:">Signaling</span> System in Eukaryotes: Modularity, Conservation, and the Transition to Metazoan Multicellularity. </a></p><div class="supp"><p class="details"><div>Schematic representation of the GPCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Protein families belonging to similar functional categories are grouped as specified in the color legend.</div></p></div><div class="aux"><div class="resc">Alex de Mendoza, et al. Genome Biol Evol. 2014 March;6(3):606-619.</div><p class="links"><a class="dblinks" href="/pubmed/24567306" ref="ordinalpos=1459&amp;ncbi_uid=5962967&amp;link_uid=24567306">Citation</a><a class="dblinks" href="/pmc/articles/PMC3971589/" ref="ordinalpos=1459&amp;ncbi_uid=5962967&amp;link_uid=PMC3971589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1460.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3526732/figure/F1/" ref="ordinalpos=1460&amp;ncbi_uid=4463658&amp;link_uid=PMC3526732" href="/pmc/articles/PMC3526732/"><img src="/pmc/articles/instance/3526732/bin/fphys-03-00464-g001.gif" src-large="/pmc/articles/instance/3526732/bin/fphys-03-00464-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3526732/" ref="ordinalpos=1460&amp;ncbi_uid=4463658&amp;link_uid=PMC3526732" image-link="/pmc/articles/PMC3526732/figure/F1/" class="imagepopup">Figure 1.  From: Postsynaptic Receptors for Amyloid-? Oligomers as Mediators of Neuronal Damage in Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div><b>Postsynaptic receptors for Aβ oligomers affect different <span class="highlight" style="background-color:">signaling</span> pathways</b>. The scheme shows that the binding of Aβ oligomers to different postsynaptic receptors might trigger synaptotoxicity and consequent neuronal death by activating different <span class="highlight" style="background-color:">signaling</span> pathways, as indicated, only in the case of NL-1 (neuroligin-1) and the PrPc (cellular prion protein), the precise <span class="highlight" style="background-color:">pathway</span> is not known. p75 (p75 receptor of neurotrophin), α7-nAChR (α7-nicotinic acetylcholine receptor), NMDAR (<i>N</i>-Methyl-D-aspartate receptor), and mGluR (metabotropic glutamate receptors), the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is already known. In the case of the α7-nAChR, recent studies from our laboratory also implicated the <i>Wnt</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Margarita C. Dinamarca, et al. Front Physiol. 2012;3:464.</div><p class="links"><a class="dblinks" href="/pubmed/23267328" ref="ordinalpos=1460&amp;ncbi_uid=4463658&amp;link_uid=23267328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3526732/" ref="ordinalpos=1460&amp;ncbi_uid=4463658&amp;link_uid=PMC3526732">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1461.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4383884/figure/F5/" ref="ordinalpos=1461&amp;ncbi_uid=7397367&amp;link_uid=PMC4383884" href="/pmc/articles/PMC4383884/"><img src="/pmc/articles/instance/4383884/bin/gku1026fig5.gif" src-large="/pmc/articles/instance/4383884/bin/gku1026fig5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4383884/" ref="ordinalpos=1461&amp;ncbi_uid=7397367&amp;link_uid=PMC4383884" image-link="/pmc/articles/PMC4383884/figure/F5/" class="imagepopup">Figure 5. From: The Rat Genome Database 2015: genomic, phenotypic and environmental variations and disease. </a></p><div class="supp"><p class="details"><div>Cisplatin response <span class="highlight" style="background-color:">pathway</span>. RGD's interactive <span class="highlight" style="background-color:">pathway</span> diagrams give a detailed and intricate view of a growing list of regulatory, <span class="highlight" style="background-color:">signaling</span>, metabolic, disease and drug pathways. The page begins with an overview of what is known about that <span class="highlight" style="background-color:">pathway</span> (not shown in this figure), and a diagram showing the various players and their interactions and relationships in the functioning of the <span class="highlight" style="background-color:">pathway</span>. Disease pathways and altered pathways show the details of what can go wrong with a specific interaction to cause a breakdown in the function of the network. <span class="highlight" style="background-color:">Pathway</span> pages also give information about the other pathways, diseases and phenotypes with which the members of a particular <span class="highlight" style="background-color:">pathway</span> are associated.</div></p></div><div class="aux"><div class="resc">Mary Shimoyama, et al. Nucleic Acids Res. 2015 January 28;43(Database issue):D743-D750.</div><p class="links"><a class="dblinks" href="/pubmed/25355511" ref="ordinalpos=1461&amp;ncbi_uid=7397367&amp;link_uid=25355511">Citation</a><a class="dblinks" href="/pmc/articles/PMC4383884/" ref="ordinalpos=1461&amp;ncbi_uid=7397367&amp;link_uid=PMC4383884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1462.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2679751/figure/F7/" ref="ordinalpos=1462&amp;ncbi_uid=1846644&amp;link_uid=PMC2679751" href="/pmc/articles/PMC2679751/"><img src="/pmc/articles/instance/2679751/bin/1471-2202-10-27-7.gif" src-large="/pmc/articles/instance/2679751/bin/1471-2202-10-27-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2679751/" ref="ordinalpos=1462&amp;ncbi_uid=1846644&amp;link_uid=PMC2679751" image-link="/pmc/articles/PMC2679751/figure/F7/" class="imagepopup">Figure 7.  From: Proteomic and functional analysis of NCS-1 binding proteins reveals novel <span class="highlight" style="background-color:">signaling</span> pathways required for inner ear development in zebrafish. </a></p><div class="supp"><p class="details"><div><b>Proposed model for functional interaction between Ncs-1a, Pi4kβ, and Arf1 in trafficking and secretion</b>. Dan, Bmp4, and HA are secreted molecules that contribute to proper semicircular canal development and are predicted to require the action of an Ncs-1/Pi4kb/Arf1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for their secretion. Bold arrows indicate activation, whereas the bold T indicates inhibition, of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Thin arrows indicate directional flow of vesicles or proteins.</div></p></div><div class="aux"><div class="resc">Jessica A Petko, et al. BMC Neurosci. 2009;10:27-27.</div><p class="links"><a class="dblinks" href="/pubmed/19320994" ref="ordinalpos=1462&amp;ncbi_uid=1846644&amp;link_uid=19320994">Citation</a><a class="dblinks" href="/pmc/articles/PMC2679751/" ref="ordinalpos=1462&amp;ncbi_uid=1846644&amp;link_uid=PMC2679751">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1463.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC155783/figure/f7/" ref="ordinalpos=1463&amp;ncbi_uid=314664&amp;link_uid=PMC155783" href="/pmc/articles/PMC155783/"><img src="/pmc/articles/instance/155783/bin/ii0631887007.gif" src-large="/pmc/articles/instance/155783/bin/ii0631887007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC155783/" ref="ordinalpos=1463&amp;ncbi_uid=314664&amp;link_uid=PMC155783" image-link="/pmc/articles/PMC155783/figure/f7/" class="imagepopup">FIG. 7. From: c-Jun NH2-Terminal Kinase-Mediated <span class="highlight" style="background-color:">Signaling</span> Is Essential for Pseudomonas aeruginosa ExoS-Induced Apoptosis . </a></p><div class="supp"><p class="details"><div>Proposed pathways for the ExoS-mediated apoptosis. ExoS is delivered into host cells through type III machinery. Both proapoptotic and antiapoptotic (or proliferation) <span class="highlight" style="background-color:">signaling</span> pathways are implicated. In the proapoptotic <span class="highlight" style="background-color:">pathway</span>, ExoS targets unknown intracellular factor(s) to initiate the apoptotic signal, leading to the activation of JNK1, which results in cytochrome <i>c</i> release and caspase 9 activation. Caspase 3 is then activated and executes the apoptosis program. This <span class="highlight" style="background-color:">pathway</span> is also potentiated by inhibition of an antiapoptotic <span class="highlight" style="background-color:">pathway</span> by the ExoS, which likely occurs through the inhibition of Ras and Ras-associated Raf-MEK-ERK <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Jinghua Jia, et al. Infect Immun. 2003 June;71(6):3361-3370.</div><p class="links"><a class="dblinks" href="/pubmed/12761120" ref="ordinalpos=1463&amp;ncbi_uid=314664&amp;link_uid=12761120">Citation</a><a class="dblinks" href="/pmc/articles/PMC155783/" ref="ordinalpos=1463&amp;ncbi_uid=314664&amp;link_uid=PMC155783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1464.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633845/figure/pone-0062399-g001/" ref="ordinalpos=1464&amp;ncbi_uid=4833368&amp;link_uid=PMC3633845" href="/pmc/articles/PMC3633845/"><img src="/pmc/articles/instance/3633845/bin/pone.0062399.g001.gif" src-large="/pmc/articles/instance/3633845/bin/pone.0062399.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633845/" ref="ordinalpos=1464&amp;ncbi_uid=4833368&amp;link_uid=PMC3633845" image-link="/pmc/articles/PMC3633845/figure/pone-0062399-g001/" class="imagepopup">Figure 1. Expression of the PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by immunohistochemistry in NAGs, APA and IHA patients..  From: The PI3K/AKT/mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Is Overactivated in Primary Aldosteronism. </a></p><div class="supp"><p class="details"><div>NAGs served as a reference. In NAGs, faint staining of these proteins is visible in the cytoplasm of normal zona glomerulosa cells. However, in IHA and APA patients, the staining of these four proteins is stronger. In general, the immunohistochemical staining of the tissue sections indicated that p-AKT, p-mTOR, p-S6 and VEGF were highly expressed in APA and IHA sections compared with NAGs. (Magnification × 400).</div></p></div><div class="aux"><div class="resc">Hengchuan Su, et al. PLoS One. 2013;8(4):e62399.</div><p class="links"><a class="dblinks" href="/pubmed/23626817" ref="ordinalpos=1464&amp;ncbi_uid=4833368&amp;link_uid=23626817">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633845/" ref="ordinalpos=1464&amp;ncbi_uid=4833368&amp;link_uid=PMC3633845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1465.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3903585/figure/F2/" ref="ordinalpos=1465&amp;ncbi_uid=5742956&amp;link_uid=PMC3903585" href="/pmc/articles/PMC3903585/"><img src="/pmc/articles/instance/3903585/bin/nihms-546630-f0002.gif" src-large="/pmc/articles/instance/3903585/bin/nihms-546630-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3903585/" ref="ordinalpos=1465&amp;ncbi_uid=5742956&amp;link_uid=PMC3903585" image-link="/pmc/articles/PMC3903585/figure/F2/" class="imagepopup">Figure 2. TCF/LEF luciferase reporter system for Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> in human NPCs.  From: A High-Throughput Screen for Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Modulators in Human iPSC-derived Neural Progenitors. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Schematic representation of the reporter system. Validation of the reporter system with dose response curves of known <span class="highlight" style="background-color:">pathway</span> modulators, including <b>(B)</b> Wnt3a-CM, <b>(C)</b> CHIR-99021, and <b>(D)</b> lithium. Each data point represents the mean ± SEM of at least quadruplicate measurements in two different plates.</div></p></div><div class="aux"><div class="resc">Wen-Ning Zhao, et al. J Biomol Screen. ;17(9):1252-1263.</div><p class="links"><a class="dblinks" href="/pubmed/22923789" ref="ordinalpos=1465&amp;ncbi_uid=5742956&amp;link_uid=22923789">Citation</a><a class="dblinks" href="/pmc/articles/PMC3903585/" ref="ordinalpos=1465&amp;ncbi_uid=5742956&amp;link_uid=PMC3903585">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1466.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2669444/figure/F5/" ref="ordinalpos=1466&amp;ncbi_uid=1866887&amp;link_uid=PMC2669444" href="/pmc/articles/PMC2669444/"><img src="/pmc/articles/instance/2669444/bin/nihms98504f5.gif" src-large="/pmc/articles/instance/2669444/bin/nihms98504f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2669444/" ref="ordinalpos=1466&amp;ncbi_uid=1866887&amp;link_uid=PMC2669444" image-link="/pmc/articles/PMC2669444/figure/F5/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> changes in the mutant cerebral cortex.  From: Cortical deficiency of laminin ?1 impairs the AKT/GSK-3? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and leads to defects in neurite outgrowth and neuronal migration. </a></p><div class="supp"><p class="details"><div>Cerebral cortices from E16.5 control and mutant embryos were collected and proteins were extracted for western blots using antibodies as indicated. For quantitative analysis, seven mutant and control embryos were used. The intensity of the positive bands was normalized with actin and the differences between control and mutant were analyzed by Student's t-test.</div></p></div><div class="aux"><div class="resc">Zu-Lin Chen, et al. Dev Biol. ;327(1):158-168.</div><p class="links"><a class="dblinks" href="/pubmed/19118544" ref="ordinalpos=1466&amp;ncbi_uid=1866887&amp;link_uid=19118544">Citation</a><a class="dblinks" href="/pmc/articles/PMC2669444/" ref="ordinalpos=1466&amp;ncbi_uid=1866887&amp;link_uid=PMC2669444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1467.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3651307/figure/F1/" ref="ordinalpos=1467&amp;ncbi_uid=4887679&amp;link_uid=PMC3651307" href="/pmc/articles/PMC3651307/"><img src="/pmc/articles/instance/3651307/bin/1471-2407-13-219-1.gif" src-large="/pmc/articles/instance/3651307/bin/1471-2407-13-219-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3651307/" ref="ordinalpos=1467&amp;ncbi_uid=4887679&amp;link_uid=PMC3651307" image-link="/pmc/articles/PMC3651307/figure/F1/" class="imagepopup">Figure 1.  From: Genetic variation in TLR or NFkappaB pathways and the risk of breast cancer: a case-control study. </a></p><div class="supp"><p class="details"><div><b>Toll-like Receptor (TLR) <span class="highlight" style="background-color:">Signaling</span> Pathways.</b> Classical TLR <span class="highlight" style="background-color:">signaling</span> pathways that result in NFκB activation are either MyD88-dependent (A) or MyD88-independent (B). In MyD88-dependent pathways TLR <span class="highlight" style="background-color:">signaling</span> occurs through the IRAK4/ IRAK1 complex, while in MyD88-independent pathways TLRs signal through TICAM1. TRAF6 then signals to the IKK complex through MAP3K7, which finally leads to NFκB activation. MyD88-independent <span class="highlight" style="background-color:">signaling</span> pathways can also result in the activation of IRF3 or IRF7. Genes assayed by this study are bolded and italicized. This figure was adapted from the KEGG “Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” (http://www.genome.jp/kegg/<span class="highlight" style="background-color:">pathway</span>/hsa/hsa04620.html).</div></p></div><div class="aux"><div class="resc">Alexa J Resler, et al. BMC Cancer. 2013;13:219-219.</div><p class="links"><a class="dblinks" href="/pubmed/23634849" ref="ordinalpos=1467&amp;ncbi_uid=4887679&amp;link_uid=23634849">Citation</a><a class="dblinks" href="/pmc/articles/PMC3651307/" ref="ordinalpos=1467&amp;ncbi_uid=4887679&amp;link_uid=PMC3651307">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1468.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2715501/figure/F1/" ref="ordinalpos=1468&amp;ncbi_uid=1967345&amp;link_uid=PMC2715501" href="/pmc/articles/PMC2715501/"><img src="/pmc/articles/instance/2715501/bin/zpq9990986530001.gif" src-large="/pmc/articles/instance/2715501/bin/zpq9990986530001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2715501/" ref="ordinalpos=1468&amp;ncbi_uid=1967345&amp;link_uid=PMC2715501" image-link="/pmc/articles/PMC2715501/figure/F1/" class="imagepopup">Fig. 1. From: Independent elaboration of steroid hormone <span class="highlight" style="background-color:">signaling</span> pathways in metazoans. </a></p><div class="supp"><p class="details"><div>Study background. (<i>A</i>) The ligand exploitation hypothesis. The ancestral receptor, that is supposed to bind estradiol, should have been lost in ecdysozoans, have lost its ligand-binding ability in mollusks, and have undergone ligand diversification through gene duplications in vertebrates. (<i>B</i>) The human steroid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>C</i>) The steroid <span class="highlight" style="background-color:">signaling</span> pathways in ecdysozoans.</div></p></div><div class="aux"><div class="resc">Gabriel V. Markov, et al. Proc Natl Acad Sci U S A. 2009 July 21;106(29):11913-11918.</div><p class="links"><a class="dblinks" href="/pubmed/19571007" ref="ordinalpos=1468&amp;ncbi_uid=1967345&amp;link_uid=19571007">Citation</a><a class="dblinks" href="/pmc/articles/PMC2715501/" ref="ordinalpos=1468&amp;ncbi_uid=1967345&amp;link_uid=PMC2715501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1469.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3269784/figure/F1/" ref="ordinalpos=1469&amp;ncbi_uid=3784223&amp;link_uid=PMC3269784" href="/pmc/articles/PMC3269784/"><img src="/pmc/articles/instance/3269784/bin/nihms351758f1.gif" src-large="/pmc/articles/instance/3269784/bin/nihms351758f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3269784/" ref="ordinalpos=1469&amp;ncbi_uid=3784223&amp;link_uid=PMC3269784" image-link="/pmc/articles/PMC3269784/figure/F1/" class="imagepopup">Figure 1. Summary of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Interactions between SOX factors and Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> in development and disease. </a></p><div class="supp"><p class="details"><div>In the absence of a Wnt signal the destruction complex including Axin, APC and the kinase GSK3, phosphorylates (P) β-catenin. Phosphorylated β-catenin is recognized by the E3 ubiquitin (Ub) ligase β-TrCP and targeted for proteosome degradation. Wnt <span class="highlight" style="background-color:">signaling</span> inactivates the destruction complex, as a result β-catenin translocates into the nucleus, where it interacts with TCF/LEF transcription factors and co-activators such as CBP/p300, to stimulate transcription. In the absence of Wnt <span class="highlight" style="background-color:">signaling</span>, DNA-bound TCFs interact with transcriptional co-repressors such as, Tle/Groucho to keep target genes turned off.</div></p></div><div class="aux"><div class="resc">Jay D Kormish, et al. Dev Dyn. ;239(1):56-68.</div><p class="links"><a class="dblinks" href="/pubmed/19655378" ref="ordinalpos=1469&amp;ncbi_uid=3784223&amp;link_uid=19655378">Citation</a><a class="dblinks" href="/pmc/articles/PMC3269784/" ref="ordinalpos=1469&amp;ncbi_uid=3784223&amp;link_uid=PMC3269784">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1470.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4326428/figure/pone.0117911.g006/" ref="ordinalpos=1470&amp;ncbi_uid=7214166&amp;link_uid=PMC4326428" href="/pmc/articles/PMC4326428/"><img src="/pmc/articles/instance/4326428/bin/pone.0117911.g006.gif" src-large="/pmc/articles/instance/4326428/bin/pone.0117911.g006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4326428/" ref="ordinalpos=1470&amp;ncbi_uid=7214166&amp;link_uid=PMC4326428" image-link="/pmc/articles/PMC4326428/figure/pone.0117911.g006/" class="imagepopup">Fig 6. Schematic representation of the <span class="highlight" style="background-color:">signaling</span> pathways involved in the TNF-α-induced ICAM-1 expression in RPECs..  From: TNF-α Mediates PKCδ/JNK1/2/c-Jun-Dependent Monocyte Adhesion via ICAM-1 Induction in Human Retinal Pigment Epithelial Cells. </a></p><div class="supp"><p class="details"><div>TNF-α-induced ICAM-1 expression and monocyte adhesion are mediated through a TNFR1/TRAF2/PKCδ/JNK1/2/c-Jun <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human RPECs.</div></p></div><div class="aux"><div class="resc">I-Ta Lee, et al. PLoS One. 2015;10(2):e0117911.</div><p class="links"><a class="dblinks" href="/pubmed/25675437" ref="ordinalpos=1470&amp;ncbi_uid=7214166&amp;link_uid=25675437">Citation</a><a class="dblinks" href="/pmc/articles/PMC4326428/" ref="ordinalpos=1470&amp;ncbi_uid=7214166&amp;link_uid=PMC4326428">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1471.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3793171/figure/F4/" ref="ordinalpos=1471&amp;ncbi_uid=5387288&amp;link_uid=PMC3793171" href="/pmc/articles/PMC3793171/"><img src="/pmc/articles/instance/3793171/bin/fimmu-04-00318-g004.gif" src-large="/pmc/articles/instance/3793171/bin/fimmu-04-00318-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3793171/" ref="ordinalpos=1471&amp;ncbi_uid=5387288&amp;link_uid=PMC3793171" image-link="/pmc/articles/PMC3793171/figure/F4/" class="imagepopup">Figure 4.  From: Immunostimulation by Synthetic Lipopeptide-Based Vaccine Candidates: Structure-Activity Relationships. </a></p><div class="supp"><p class="details"><div><b>Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: recognition of pathogens through TLR ultimately triggers <span class="highlight" style="background-color:">signaling</span> through the NF-κB <span class="highlight" style="background-color:">pathway</span>, consequently leading to gene activation and production of pro-inflammatory cytokines/chemokines</b>.</div></p></div><div class="aux"><div class="resc">Mehfuz Zaman, et al. Front Immunol. 2013;4:318.</div><p class="links"><a class="dblinks" href="/pubmed/24130558" ref="ordinalpos=1471&amp;ncbi_uid=5387288&amp;link_uid=24130558">Citation</a><a class="dblinks" href="/pmc/articles/PMC3793171/" ref="ordinalpos=1471&amp;ncbi_uid=5387288&amp;link_uid=PMC3793171">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1472.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3023986/figure/F2/" ref="ordinalpos=1472&amp;ncbi_uid=2719843&amp;link_uid=PMC3023986" href="/pmc/articles/PMC3023986/"><img src="/pmc/articles/instance/3023986/bin/nihms-244092-f0002.gif" src-large="/pmc/articles/instance/3023986/bin/nihms-244092-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3023986/" ref="ordinalpos=1472&amp;ncbi_uid=2719843&amp;link_uid=PMC3023986" image-link="/pmc/articles/PMC3023986/figure/F2/" class="imagepopup">Figure 2. Activation and inhibition of Wnt <span class="highlight" style="background-color:">signaling</span> pathways.  From: Wnt <span class="highlight" style="background-color:">Signaling</span> as a Therapeutic Target for Bone Diseases. </a></p><div class="supp"><p class="details"><div><b>A)</b> The “canonical <span class="highlight" style="background-color:">pathway</span>” is stimulated when Wnts bind to Frizzled (Fzd) receptors and low-density lipoprotein receptor-related protein (Lrp)5/6 co-receptors (center). This activates Disheveled (Dsh), which inhibits a cytoplasmic complex composed of glycogen synthase kinase (GSK)3β, Axin, and adenomatous polyposis coli (APC). Cytoplasmic β-catenin levels rise and some β-catenin translocates to the nucleus where it associates with T-cell factor (Tcf)/lymphoid enhancer-binding factor (Lef) transcription factors to regulate gene expression. During non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> (right side of the figure), Wnts bind a Fzd receptor, but the downstream <span class="highlight" style="background-color:">signaling</span> events do not involve GSK3β or β-catenin. Two non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> cascades that have been identified are: 1) Wnt/calcium <span class="highlight" style="background-color:">signaling</span> increases intracellular Ca<sup>2+</sup> levels and activates protein kinase C and calcium/calmodulin-dependent kinase; and 2) The Wnt/planar cell polarity <span class="highlight" style="background-color:">pathway</span> that signals through Rho/Rac GTPases and c-Jun N-terminal kinase (JNK) to modulate cytoskeletal elements and gene expression. Wnts also bind to receptor tyrosine kinases receptor-like tyrosine (Ryk) and receptor tyrosine kinase-like orphan receptor 2 (Ror2) to activate oncogene (Src) and JNK <span class="highlight" style="background-color:">signaling</span>, respectively (left side of figure). <b>B</b>) Secreted frizzled-related proteins (Sfrps) antagonize canonical Wnt <span class="highlight" style="background-color:">signaling</span> by binding the ligands and preventing their association with Fzd receptors. In the absence of Wnt <span class="highlight" style="background-color:">signaling</span>, GSK3β phosphorylates (asterisks) β-catenin, which marks it for ubiquitination and degradation by the proteosome. <b>C</b>) Dickkoph (Dkk1) suppresses Wnt <span class="highlight" style="background-color:">signaling</span> by forming a ternary complex with Lrp5/6 and Kremen (Krm)1/2. Sclerostin (Scl) also binds to Lrp5/6, but not Krm1/2, to antagonize canonical Wnt <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Luke H. Hoeppner, et al. Expert Opin Ther Targets. ;13(4):485-496.</div><p class="links"><a class="dblinks" href="/pubmed/19335070" ref="ordinalpos=1472&amp;ncbi_uid=2719843&amp;link_uid=19335070">Citation</a><a class="dblinks" href="/pmc/articles/PMC3023986/" ref="ordinalpos=1472&amp;ncbi_uid=2719843&amp;link_uid=PMC3023986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1473.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3058986/figure/F8/" ref="ordinalpos=1473&amp;ncbi_uid=3532435&amp;link_uid=PMC3058986" href="/pmc/articles/PMC3058986/"><img src="/pmc/articles/instance/3058986/bin/zbc0151153940008.gif" src-large="/pmc/articles/instance/3058986/bin/zbc0151153940008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3058986/" ref="ordinalpos=1473&amp;ncbi_uid=3532435&amp;link_uid=PMC3058986" image-link="/pmc/articles/PMC3058986/figure/F8/" class="imagepopup">FIGURE 8. From: RACK1 Regulates VEGF/Flt1-mediated Cell Migration via Activation of a PI3K/Akt <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>The possible role of RACK1 in VEGF/Flt1-induced cell migration.</b> By interacting with Flt1, RACK1 plays an important role in VEGF/Flt1-induced cell migration by regulating the activation of the PI3K/Akt but not PLCγ-MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. RACK1 regulates the VEGF/Flt1-induced activation of Rac1 and cytoskeletal dynamics.</div></p></div><div class="aux"><div class="resc">Feng Wang, et al. J Biol Chem. 2011 March 18;286(11):9097-9106.</div><p class="links"><a class="dblinks" href="/pubmed/21212275" ref="ordinalpos=1473&amp;ncbi_uid=3532435&amp;link_uid=21212275">Citation</a><a class="dblinks" href="/pmc/articles/PMC3058986/" ref="ordinalpos=1473&amp;ncbi_uid=3532435&amp;link_uid=PMC3058986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1474.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2082143/figure/F10/" ref="ordinalpos=1474&amp;ncbi_uid=1105753&amp;link_uid=PMC2082143" href="/pmc/articles/PMC2082143/"><img src="/pmc/articles/instance/2096423/bin/JCI0732301.f10.gif" src-large="/pmc/articles/instance/2096423/bin/JCI0732301.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2082143/" ref="ordinalpos=1474&amp;ncbi_uid=1105753&amp;link_uid=PMC2082143" image-link="/pmc/articles/PMC2082143/figure/F10/" class="imagepopup">Figure 10. Schematic illustration of the <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span> leading to tPA promotion of myofibroblast activation..  From: Tissue-type plasminogen activator promotes murine myofibroblast activation through LDL receptor-related protein 1-mediated integrin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>tPA binds to membrane receptor LRP-1, induces its tyrosine phosphorylation, which leads to the increased recruitment of β1 integrin and activates downstream ILK. Blockade of each and every step within this <span class="highlight" style="background-color:">signaling</span> circuit by multiple strategies (numbers 1–5) abolishes the fibrogenic effect of tPA on myofibroblast activation, whereas over-expression of ILK mimics tPA and promotes the TGF-β1–mediated myofibroblast activation (number 6).</div></p></div><div class="aux"><div class="resc">Kebin Hu, et al. J Clin Invest. 2007 December 3;117(12):3821-3832.</div><p class="links"><a class="dblinks" href="/pubmed/18037995" ref="ordinalpos=1474&amp;ncbi_uid=1105753&amp;link_uid=18037995">Citation</a><a class="dblinks" href="/pmc/articles/PMC2082143/" ref="ordinalpos=1474&amp;ncbi_uid=1105753&amp;link_uid=PMC2082143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1475.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052696/figure/fig9/" ref="ordinalpos=1475&amp;ncbi_uid=6271080&amp;link_uid=PMC4052696" href="/pmc/articles/PMC4052696/"><img src="/pmc/articles/instance/4052696/bin/BMRI2014-351095.009.gif" src-large="/pmc/articles/instance/4052696/bin/BMRI2014-351095.009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052696/" ref="ordinalpos=1475&amp;ncbi_uid=6271080&amp;link_uid=PMC4052696" image-link="/pmc/articles/PMC4052696/figure/fig9/" class="imagepopup">Figure 9.  From: <span class="highlight" style="background-color:">Pathway</span> Bridge Based Multiobjective Optimization Approach for Lurking <span class="highlight" style="background-color:">Pathway</span> Prediction. </a></p><div class="supp"><p class="details"><div>Analyzing the observations comprehensively on modular study and referring to related literature, the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from TGF<i>β</i>1 targeting OCIAD2 was concluded as shown in Figure 9: the <span class="highlight" style="background-color:">signaling</span> transmits from TGF<i>β</i>1- TGF<i>β</i>R1- AR-OCIAD2 in liver cancer mesenchymal stem cell, then differentiates into Tumor-Associated-Fibroblasts (TAFs) in tumor stroma. As the only known mammalian coSMAD, SMAD4 transferred <span class="highlight" style="background-color:">signaling</span> from cytoplasm to TGFB signal. AR, the symbol of androgen receptor, mainly functioned as a DNA-binding transcription factor that regulates target gene expression from cytoplasm into nucleus.</div></p></div><div class="aux"><div class="resc">Rengjing Zhang, et al. Biomed Res Int. 2014;2014:351095.</div><p class="links"><a class="dblinks" href="/pubmed/24949437" ref="ordinalpos=1475&amp;ncbi_uid=6271080&amp;link_uid=24949437">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052696/" ref="ordinalpos=1475&amp;ncbi_uid=6271080&amp;link_uid=PMC4052696">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1476.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3505759/figure/F3/" ref="ordinalpos=1476&amp;ncbi_uid=4389853&amp;link_uid=PMC3505759" href="/pmc/articles/PMC3505759/"><img src="/pmc/articles/instance/3505759/bin/nihms12407f3.gif" src-large="/pmc/articles/instance/3505759/bin/nihms12407f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3505759/" ref="ordinalpos=1476&amp;ncbi_uid=4389853&amp;link_uid=PMC3505759" image-link="/pmc/articles/PMC3505759/figure/F3/" class="imagepopup">Fig. 3.  From: Proliferation of cardiomyocytes derived from human embryonic stem cells is mediated via the IGF/PI 3-kinase/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibition of human ES cell-derived cardiomyocyte proliferation. H7 human ES cells were differentiated for 19 days before Percoll separation, and fraction IV cells were cultured for an additional 4 days in serum-free media with various <span class="highlight" style="background-color:">signaling</span> inhibitors. The inhibitors were added for the final 48 hours of culture prior to fixation. Only the PI 3-kinase inhibitor (LY294002) significantly inhibited human ES cell-derived cardiomyocyte proliferation (* = <i>P</i> &lt; 0.05).</div></p></div><div class="aux"><div class="resc">Todd C. McDevitt, et al. J Mol Cell Cardiol. ;39(6):865-873.</div><p class="links"><a class="dblinks" href="/pubmed/16242146" ref="ordinalpos=1476&amp;ncbi_uid=4389853&amp;link_uid=16242146">Citation</a><a class="dblinks" href="/pmc/articles/PMC3505759/" ref="ordinalpos=1476&amp;ncbi_uid=4389853&amp;link_uid=PMC3505759">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1477.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432194/figure/fig2/" ref="ordinalpos=1477&amp;ncbi_uid=4217904&amp;link_uid=PMC3432194" href="/pmc/articles/PMC3432194/"><img src="/pmc/articles/instance/3432194/bin/tp201280f2.gif" src-large="/pmc/articles/instance/3432194/bin/tp201280f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432194/" ref="ordinalpos=1477&amp;ncbi_uid=4217904&amp;link_uid=PMC3432194" image-link="/pmc/articles/PMC3432194/figure/fig2/" class="imagepopup">Figure 2.  From: Epistasis network centrality analysis yields <span class="highlight" style="background-color:">pathway</span> replication across two GWAS cohorts for bipolar disorder. </a></p><div class="supp"><p class="details"><div>Epistasis network for WTCCC GWAS of bipolar disorder. Network inferred following ECML feature selection and regression-based genetic association interaction network (reGAIN) for the WTCCC GWAS of bipolar disorder, annotated by top enriched pathways. An edge threshold (0.575) was chosen as described in Materials and methods; interactions below this threshold are hidden. The 146 nodes are colored based on membership of the genes in the pathways with evidence of enrichment replication (Tables 1 and 2): red diamond (membership in both Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and cadherin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>), green square (Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> only) and magenta triangle (Neuroactive ligand-receptor interaction <span class="highlight" style="background-color:">pathway</span>). The weight of an edge is proportional to the gene–gene interaction strength. The 183 edges are colored based on connection of a gene node to a gene in the given <span class="highlight" style="background-color:">pathway</span> using the scheme above (red squiggle, green dashed, magenta solid). The size of a node is proportional to its degree (number of edges). Note, <i>ANK3</i> in the middle is the most connected.</div></p></div><div class="aux"><div class="resc">A Pandey, et al. Transl Psychiatry. 2012 August;2(8):e154.</div><p class="links"><a class="dblinks" href="/pubmed/22892719" ref="ordinalpos=1477&amp;ncbi_uid=4217904&amp;link_uid=22892719">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432194/" ref="ordinalpos=1477&amp;ncbi_uid=4217904&amp;link_uid=PMC3432194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1478.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3349255/figure/F7/" ref="ordinalpos=1478&amp;ncbi_uid=3996754&amp;link_uid=PMC3349255" href="/pmc/articles/PMC3349255/"><img src="/pmc/articles/instance/3349255/bin/neo1404_0283_fig007.gif" src-large="/pmc/articles/instance/3349255/bin/neo1404_0283_fig007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3349255/" ref="ordinalpos=1478&amp;ncbi_uid=3996754&amp;link_uid=PMC3349255" image-link="/pmc/articles/PMC3349255/figure/F7/" class="imagepopup">Figure 7.  From: Cross-regulation between FOXA1 and ErbB2 <span class="highlight" style="background-color:">Signaling</span> in Estrogen Receptor-Negative Breast Cancer. </a></p><div class="supp"><p class="details"><div>Schematic diagram of a cross-regulation network between FOXA1 and the ErbB2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ER-negative breast cancer. Red and black arrows denote stimulatory and inhibitory effects, respectively.</div></p></div><div class="aux"><div class="resc">Ali Naderi, et al. Neoplasia. 2012 April;14(4):283-296.</div><p class="links"><a class="dblinks" href="/pubmed/22577344" ref="ordinalpos=1478&amp;ncbi_uid=3996754&amp;link_uid=22577344">Citation</a><a class="dblinks" href="/pmc/articles/PMC3349255/" ref="ordinalpos=1478&amp;ncbi_uid=3996754&amp;link_uid=PMC3349255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1479.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3134994/figure/fig1/" ref="ordinalpos=1479&amp;ncbi_uid=3086873&amp;link_uid=PMC3134994" href="/pmc/articles/PMC3134994/"><img src="/pmc/articles/instance/3134994/bin/NRI2011-124256.001.gif" src-large="/pmc/articles/instance/3134994/bin/NRI2011-124256.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3134994/" ref="ordinalpos=1479&amp;ncbi_uid=3086873&amp;link_uid=PMC3134994" image-link="/pmc/articles/PMC3134994/figure/fig1/" class="imagepopup">Figure 1.  From: Transforming Growth Factor-Beta <span class="highlight" style="background-color:">Signaling</span> in the Neural Stem Cell Niche: A Therapeutic Target for Huntington's Disease. </a></p><div class="supp"><p class="details"><div>Schematic Illustration: TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TGF-beta ligand binds to TGF-betaRII that activates TGF betaRI and induces the downstream Smad-mediated signal transduction.</div></p></div><div class="aux"><div class="resc">Mahesh Kandasamy, et al. Neurol Res Int. 2011;2011:124256.</div><p class="links"><a class="dblinks" href="/pubmed/21766020" ref="ordinalpos=1479&amp;ncbi_uid=3086873&amp;link_uid=21766020">Citation</a><a class="dblinks" href="/pmc/articles/PMC3134994/" ref="ordinalpos=1479&amp;ncbi_uid=3086873&amp;link_uid=PMC3134994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1480.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3384423/figure/F4/" ref="ordinalpos=1480&amp;ncbi_uid=4083480&amp;link_uid=PMC3384423" href="/pmc/articles/PMC3384423/"><img src="/pmc/articles/instance/3384423/bin/neo1405_0360_fig004.gif" src-large="/pmc/articles/instance/3384423/bin/neo1405_0360_fig004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3384423/" ref="ordinalpos=1480&amp;ncbi_uid=4083480&amp;link_uid=PMC3384423" image-link="/pmc/articles/PMC3384423/figure/F4/" class="imagepopup">Figure 4.  From: Targeting the mTOR-DEPTOR <span class="highlight" style="background-color:">Pathway</span> by CRL E3 Ubiquitin Ligases: Therapeutic Application. </a></p><div class="supp"><p class="details"><div>CRLs/SCF E3 ubiquitin ligases regulate mTOR <span class="highlight" style="background-color:">pathway</span>. (A) Schematic summary of CRLs/SCF E3 substrates in mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) Schematic summary of cullin neddylation. (C) Schematic summary of ubiquitination reaction catalyzed by SCF E3 ubiquitin ligases and potential targeting points for inhibition. For details, see text.</div></p></div><div class="aux"><div class="resc">Yongchao Zhao, et al. Neoplasia. 2012 May;14(5):360-367.</div><p class="links"><a class="dblinks" href="/pubmed/22745582" ref="ordinalpos=1480&amp;ncbi_uid=4083480&amp;link_uid=22745582">Citation</a><a class="dblinks" href="/pmc/articles/PMC3384423/" ref="ordinalpos=1480&amp;ncbi_uid=4083480&amp;link_uid=PMC3384423">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1481.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2697172/figure/F8/" ref="ordinalpos=1481&amp;ncbi_uid=1831244&amp;link_uid=PMC2697172" href="/pmc/articles/PMC2697172/"><img src="/pmc/articles/instance/2697172/bin/1471-2164-10-220-8.gif" src-large="/pmc/articles/instance/2697172/bin/1471-2164-10-220-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2697172/" ref="ordinalpos=1481&amp;ncbi_uid=1831244&amp;link_uid=PMC2697172" image-link="/pmc/articles/PMC2697172/figure/F8/" class="imagepopup">Figure 8.  From: A network-based integrative approach to prioritize reliable hits from multiple genome-wide RNAi screens in Drosophila. </a></p><div class="supp"><p class="details"><div><b>A sub-network associated with the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Red: hits of RNAi screening. White: top-ranking nonhits by NePhe score. Green boundary: genes within KEGG Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Square: genes supported by literature for their association with the Wnt <span class="highlight" style="background-color:">pathway</span>. Module I: canonical participants-related. Module II: transcription factor TFIID complex-related. Module III: PcG protein complex-related. Module IV: other <span class="highlight" style="background-color:">signaling</span> pathways-related. The visualization was generated using Cytoscape [57].</div></p></div><div class="aux"><div class="resc">Li Wang, et al. BMC Genomics. 2009;10:220-220.</div><p class="links"><a class="dblinks" href="/pubmed/19435510" ref="ordinalpos=1481&amp;ncbi_uid=1831244&amp;link_uid=19435510">Citation</a><a class="dblinks" href="/pmc/articles/PMC2697172/" ref="ordinalpos=1481&amp;ncbi_uid=1831244&amp;link_uid=PMC2697172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1482.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3679036/figure/pone-0066029-g011/" ref="ordinalpos=1482&amp;ncbi_uid=5000186&amp;link_uid=PMC3679036" href="/pmc/articles/PMC3679036/"><img src="/pmc/articles/instance/3679036/bin/pone.0066029.g011.gif" src-large="/pmc/articles/instance/3679036/bin/pone.0066029.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3679036/" ref="ordinalpos=1482&amp;ncbi_uid=5000186&amp;link_uid=PMC3679036" image-link="/pmc/articles/PMC3679036/figure/pone-0066029-g011/" class="imagepopup">Figure 11. A schematic presentation describing the different <span class="highlight" style="background-color:">signaling</span> components involved in the effect of CCK on RhoA <span class="highlight" style="background-color:">signaling</span>..  From: Cholecystokinin-Mediated RhoGDI Phosphorylation via PKC? Promotes both RhoA and Rac1 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Inactive RhoA is localized in the cytosol complexed to RhoGDI1, which masks the geranylgeranylated group. Upon CCK stimulation, two pathways are activated: Gα13 <span class="highlight" style="background-color:">pathway</span> and PKCα <span class="highlight" style="background-color:">pathway</span>. RhoGDI1 is phosphorylated at Ser96 by PKCα, and thereby, releases inactive RhoA. Once inactive RhoA is dissociated, inactive RhoA is able to translocate to membranes and be activated by RhoGEF, in a mechanism which dependent on active Gα13. Finally, GAP inactivates RhoA, which is able to associate with cytosolic RhoGDI1.</div></p></div><div class="aux"><div class="resc">Maria Eugenia Sabbatini, et al. PLoS One. 2013;8(6):e66029.</div><p class="links"><a class="dblinks" href="/pubmed/23776598" ref="ordinalpos=1482&amp;ncbi_uid=5000186&amp;link_uid=23776598">Citation</a><a class="dblinks" href="/pmc/articles/PMC3679036/" ref="ordinalpos=1482&amp;ncbi_uid=5000186&amp;link_uid=PMC3679036">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1483.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262331/figure/f3-jbm-1-093/" ref="ordinalpos=1483&amp;ncbi_uid=3775456&amp;link_uid=PMC3262331" href="/pmc/articles/PMC3262331/"><img src="/pmc/articles/instance/3262331/bin/jbm-1-093f3.gif" src-large="/pmc/articles/instance/3262331/bin/jbm-1-093f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262331/" ref="ordinalpos=1483&amp;ncbi_uid=3775456&amp;link_uid=PMC3262331" image-link="/pmc/articles/PMC3262331/figure/f3-jbm-1-093/" class="imagepopup">Figure 3.  From: L-CBM <span class="highlight" style="background-color:">signaling</span> in lymphocyte development and function. </a></p><div class="supp"><p class="details"><div>Schematic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by the L-CBM complex in NK cells.<br /><b>Notes:</b> The L-CBM complex acts in the canonical NF-κB activation <span class="highlight" style="background-color:">pathway</span> through activating NK cell receptors (NKR) such as Ly49H, Ly49D, NK1.1, NKG2D, FcγRIII, etc. L-CBM might regulate MAPK activation through activating NKR, although this notion remains controversial. Similar to T cells, PKCθ activity is required for NF-κB activation through activating NKR. IL-18R-MyD88-mediated NF-κB activation does not require L-CBM in NK cells. While the PKCθ-L-CBM <span class="highlight" style="background-color:">signaling</span> is essential to cytokine/chemokine production, it is not necessary for cytotoxicity.</div></p></div><div class="aux"><div class="resc">Hiromitsu Hara, et al. J Blood Med. 2010;1:93-104.</div><p class="links"><a class="dblinks" href="/pubmed/22282688" ref="ordinalpos=1483&amp;ncbi_uid=3775456&amp;link_uid=22282688">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262331/" ref="ordinalpos=1483&amp;ncbi_uid=3775456&amp;link_uid=PMC3262331">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1484.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3178084/figure/F2/" ref="ordinalpos=1484&amp;ncbi_uid=4154420&amp;link_uid=PMC3178084" href="/pmc/articles/PMC3178084/"><img src="/pmc/articles/instance/3178084/bin/cto0194-0232-f02.gif" src-large="/pmc/articles/instance/3178084/bin/cto0194-0232-f02.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3178084/" ref="ordinalpos=1484&amp;ncbi_uid=4154420&amp;link_uid=PMC3178084" image-link="/pmc/articles/PMC3178084/figure/F2/" class="imagepopup">Fig. 2. From: Runx2 Regulates the Gene Network Associated with Insulin <span class="highlight" style="background-color:">Signaling</span> and Energy Homeostasis. </a></p><div class="supp"><p class="details"><div>Runx2-deficient cells exhibit suppression of genes by insulin at multiple stages of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Proteins at different stages of the <span class="highlight" style="background-color:">pathway</span> from the receptor, transducer, nuclear effecter, and target gene are shown. Expressions of respective genes are downregulated upon addition of insulin to Runx2-null cells (indicated by red arrows), while insulin robustly induces these genes in 3T3-L1 cells (indicated by green arrows).</div></p></div><div class="aux"><div class="resc">Mitra Adhami, et al. Cells Tissues Organs. 2011 August;194(2-4):232-237.</div><p class="links"><a class="dblinks" href="/pubmed/21597275" ref="ordinalpos=1484&amp;ncbi_uid=4154420&amp;link_uid=21597275">Citation</a><a class="dblinks" href="/pmc/articles/PMC3178084/" ref="ordinalpos=1484&amp;ncbi_uid=4154420&amp;link_uid=PMC3178084">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1485.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3742773/figure/pone-0071366-g001/" ref="ordinalpos=1485&amp;ncbi_uid=5215170&amp;link_uid=PMC3742773" href="/pmc/articles/PMC3742773/"><img src="/pmc/articles/instance/3742773/bin/pone.0071366.g001.gif" src-large="/pmc/articles/instance/3742773/bin/pone.0071366.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3742773/" ref="ordinalpos=1485&amp;ncbi_uid=5215170&amp;link_uid=PMC3742773" image-link="/pmc/articles/PMC3742773/figure/pone-0071366-g001/" class="imagepopup">Figure 1. Human JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by Heme..  From: STAT3 Regulates MMP3 in Heme-Induced Endothelial Cell Apoptosis. </a></p><div class="supp"><p class="details"><div>Total RNA was extracted and cDNA was synthesized and then used to screen the human JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> PCR array (SABiosciences, PAHS-039A). Fold changes and p values were calculated using Student’s t-test. A p value&lt;0.05 and a fold change greater than 2.0 were considered to be a significant dysregulation. Figure 1A is a list of up regulated and down regulated genes with fold-change greater than 2. The heat-map (Figure 1B) and scatter plot (Figure 1C) generated are also shown.</div></p></div><div class="aux"><div class="resc">Mingli Liu, et al. PLoS One. 2013;8(8):e71366.</div><p class="links"><a class="dblinks" href="/pubmed/23967200" ref="ordinalpos=1485&amp;ncbi_uid=5215170&amp;link_uid=23967200">Citation</a><a class="dblinks" href="/pmc/articles/PMC3742773/" ref="ordinalpos=1485&amp;ncbi_uid=5215170&amp;link_uid=PMC3742773">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1486.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4002015/figure/f01/" ref="ordinalpos=1486&amp;ncbi_uid=6191062&amp;link_uid=PMC4002015" href="/pmc/articles/PMC4002015/"><img src="/pmc/articles/instance/4002015/bin/1657-9534-cm-44-01-00057-gf01.gif" src-large="/pmc/articles/instance/4002015/bin/1657-9534-cm-44-01-00057-gf01.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4002015/" ref="ordinalpos=1486&amp;ncbi_uid=6191062&amp;link_uid=PMC4002015" image-link="/pmc/articles/PMC4002015/figure/f01/" class="imagepopup">Figure 1. Molecular pathways of insulin <span class="highlight" style="background-color:">signaling</span>. Grey background: phosphatydil-inositol-3 kinase (PI3K) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; black background: CAP/Cbl/TC10 <span class="highlight" style="background-color:">pathway</span>; White background: MAP-Kinase dependent.  From: An approach to the etiology of metabolic syndrome. </a></p><div class="aux"><div class="resc">Angélica M Muñoz Contreras, et al. Colomb Med (Cali). 2013 Jan-Mar;44(1):57-63.</div><p class="links"><a class="dblinks" href="/pubmed/24892324" ref="ordinalpos=1486&amp;ncbi_uid=6191062&amp;link_uid=24892324">Citation</a><a class="dblinks" href="/pmc/articles/PMC4002015/" ref="ordinalpos=1486&amp;ncbi_uid=6191062&amp;link_uid=PMC4002015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1487.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3600558/figure/F3/" ref="ordinalpos=1487&amp;ncbi_uid=4719788&amp;link_uid=PMC3600558" href="/pmc/articles/PMC3600558/"><img src="/pmc/articles/instance/3600558/bin/nihms95596f3.gif" src-large="/pmc/articles/instance/3600558/bin/nihms95596f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3600558/" ref="ordinalpos=1487&amp;ncbi_uid=4719788&amp;link_uid=PMC3600558" image-link="/pmc/articles/PMC3600558/figure/F3/" class="imagepopup">Figure 3. Immunoblotting for EGFR-related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins in the Tu212 cell line.  From: The Sequence Dependence of Cell Growth Inhibition by EGFR-Tyrosine Kinase Inhibitor ZD1839, Docetaxel and Cisplatin in Head and Neck Cancer. </a></p><div class="supp"><p class="details"><div>Z and any of its combinations efficiently inhibit the EGFR-mediated <span class="highlight" style="background-color:">signaling</span> pathways involving ERK and AKT.</div></p></div><div class="aux"><div class="resc">Carmen M. Klass, et al. Head Neck. 2009 October;31(10):1263-1273.</div><p class="links"><a class="dblinks" href="/pubmed/19399750" ref="ordinalpos=1487&amp;ncbi_uid=4719788&amp;link_uid=19399750">Citation</a><a class="dblinks" href="/pmc/articles/PMC3600558/" ref="ordinalpos=1487&amp;ncbi_uid=4719788&amp;link_uid=PMC3600558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1488.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3114428/figure/F1/" ref="ordinalpos=1488&amp;ncbi_uid=3018730&amp;link_uid=PMC3114428" href="/pmc/articles/PMC3114428/"><img src="/pmc/articles/instance/3114428/bin/nihms181259f1.gif" src-large="/pmc/articles/instance/3114428/bin/nihms181259f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3114428/" ref="ordinalpos=1488&amp;ncbi_uid=3018730&amp;link_uid=PMC3114428" image-link="/pmc/articles/PMC3114428/figure/F1/" class="imagepopup">Fig. 1. The role of CFTR in the inflammatory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Protein Processing and Inflammatory <span class="highlight" style="background-color:">Signaling</span> in Cystic Fibrosis: Challenges and Therapeutic Strategies. </a></p><div class="supp"><p class="details"><div>The lack of functional CFTR on the cell surface dysregulates pro-inflammatory <span class="highlight" style="background-color:">signaling</span> <i>via</i> innate immune response (TNFa-IL1b-TLR) pathways resulting in NFκB mediated hyper-inflammation, and the chronic CF lung disease. Therapeutic strategies that can control the chronic inflammatory pathophysiology of CF lung disease by not only restoring the chloride efflux function but also regulating the NFkB mediated chronic inflammatory <span class="highlight" style="background-color:">signaling</span> hold the promise as a potential cure of CF.</div></p></div><div class="aux"><div class="resc">C.N. Belcher, et al. Curr Mol Med. ;10(1):82-94.</div><p class="links"><a class="dblinks" href="/pubmed/20205681" ref="ordinalpos=1488&amp;ncbi_uid=3018730&amp;link_uid=20205681">Citation</a><a class="dblinks" href="/pmc/articles/PMC3114428/" ref="ordinalpos=1488&amp;ncbi_uid=3018730&amp;link_uid=PMC3114428">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1489.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3567686/figure/F9/" ref="ordinalpos=1489&amp;ncbi_uid=5795215&amp;link_uid=PMC3567686" href="/pmc/articles/PMC3567686/"><img src="/pmc/articles/instance/3567686/bin/zbc0091339080009.gif" src-large="/pmc/articles/instance/3567686/bin/zbc0091339080009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3567686/" ref="ordinalpos=1489&amp;ncbi_uid=5795215&amp;link_uid=PMC3567686" image-link="/pmc/articles/PMC3567686/figure/F9/" class="imagepopup">FIGURE 9. From: Wnt/?-Catenin <span class="highlight" style="background-color:">Pathway</span> Is Regulated by PITX2 Homeodomain Protein and Thus Contributes to the Proliferation of Human Ovarian Adenocarcinoma Cell, SKOV-3. </a></p><div class="supp"><p class="details"><div><b>Existence of both positive and negative feedback loop between PITX2 and Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is depicted schematically in this hypothetical model.</b></div></p></div><div class="aux"><div class="resc">Moitri Basu, et al. J Biol Chem. 2013 February 8;288(6):4355-4367.</div><p class="links"><a class="dblinks" href="/pubmed/23250740" ref="ordinalpos=1489&amp;ncbi_uid=5795215&amp;link_uid=23250740">Citation</a><a class="dblinks" href="/pmc/articles/PMC3567686/" ref="ordinalpos=1489&amp;ncbi_uid=5795215&amp;link_uid=PMC3567686">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1490.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3143906/figure/F3/" ref="ordinalpos=1490&amp;ncbi_uid=3110850&amp;link_uid=PMC3143906" href="/pmc/articles/PMC3143906/"><img src="/pmc/articles/instance/3143906/bin/2044-5040-1-4-3.gif" src-large="/pmc/articles/instance/3143906/bin/2044-5040-1-4-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3143906/" ref="ordinalpos=1490&amp;ncbi_uid=3110850&amp;link_uid=PMC3143906" image-link="/pmc/articles/PMC3143906/figure/F3/" class="imagepopup">Figure 3.  From: Regulation of skeletal muscle growth by the IGF1-Akt/PKB <span class="highlight" style="background-color:">pathway</span>: insights from genetic models. </a></p><div class="supp"><p class="details"><div><b>Multiple factors and pathways affect insulin-like growth factor 1 (IGF1)-Akt <span class="highlight" style="background-color:">signaling</span></b>. Various factors and pathways affecting the IGF1-Akt <span class="highlight" style="background-color:">pathway</span> are highlighted in red. See text for details.</div></p></div><div class="aux"><div class="resc">Stefano Schiaffino, et al. Skelet Muscle. 2011;1:4-4.</div><p class="links"><a class="dblinks" href="/pubmed/21798082" ref="ordinalpos=1490&amp;ncbi_uid=3110850&amp;link_uid=21798082">Citation</a><a class="dblinks" href="/pmc/articles/PMC3143906/" ref="ordinalpos=1490&amp;ncbi_uid=3110850&amp;link_uid=PMC3143906">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1491.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2891586/figure/pcbi-1000807-g003/" ref="ordinalpos=1491&amp;ncbi_uid=2353853&amp;link_uid=PMC2891586" href="/pmc/articles/PMC2891586/"><img src="/pmc/articles/instance/2891586/bin/pcbi.1000807.g003.gif" src-large="/pmc/articles/instance/2891586/bin/pcbi.1000807.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2891586/" ref="ordinalpos=1491&amp;ncbi_uid=2353853&amp;link_uid=PMC2891586" image-link="/pmc/articles/PMC2891586/figure/pcbi-1000807-g003/" class="imagepopup">Figure 3. A network derived from PPIs compared to the related canonical <span class="highlight" style="background-color:">pathway</span>..  From: Protein-Protein Interactions Essentials: Key Concepts to Building and Analyzing Interactome Networks. </a></p><div class="supp"><p class="details"><div>Comparison between a known <span class="highlight" style="background-color:">pathway</span> (NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, taken from the KEGG database, ID: hsa04330) and the corresponding interactome network build using the proteins that interact with human NOTCH proteins. The top panel (<b>A</b>) shows the <span class="highlight" style="background-color:">pathway</span> including nine proteins (green boxes) directly connected to NOTCH. In this <span class="highlight" style="background-color:">pathway</span>, the central element is the NOTCH receptor and the interaction of its intracellular domain (called NICD) with protein RBPJ. The bottom panel (<b>B</b>) shows the NOTCH PPI network (built with Cytoscape and APID2NET), including all interactors proven with at least two different experiments. The number of experiments is indicated next to each link (blue line). The PPI network provides complementry information to the KEGG <span class="highlight" style="background-color:">pathway</span>, revealing the particular links of each of the four NOTCH paralogous proteins (NOTCH1, 2, 3, and 4) present in the human proteome. The biomolecular elements included in both networks are quite similar and the information that can be deduced from them is complementary. This can be seen in the interaction between NOTCH and RBPJ that drives the central <span class="highlight" style="background-color:">signaling</span> of the <span class="highlight" style="background-color:">pathway</span> and it is present in both networks.</div></p></div><div class="aux"><div class="resc">Javier De Las Rivas, et al. PLoS Comput Biol. 2010 June;6(6):e1000807.</div><p class="links"><a class="dblinks" href="/pubmed/20589078" ref="ordinalpos=1491&amp;ncbi_uid=2353853&amp;link_uid=20589078">Citation</a><a class="dblinks" href="/pmc/articles/PMC2891586/" ref="ordinalpos=1491&amp;ncbi_uid=2353853&amp;link_uid=PMC2891586">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1492.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1360222/figure/F9/" ref="ordinalpos=1492&amp;ncbi_uid=596517&amp;link_uid=PMC1360222" href="/pmc/articles/PMC1360222/"><img src="/pmc/articles/instance/1360222/bin/nihms7040f9.gif" src-large="/pmc/articles/instance/1360222/bin/nihms7040f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1360222/" ref="ordinalpos=1492&amp;ncbi_uid=596517&amp;link_uid=PMC1360222" image-link="/pmc/articles/PMC1360222/figure/F9/" class="imagepopup">Fig. 9. Schematic representation of pro-survival <span class="highlight" style="background-color:">signaling</span> in mesenchymal cells by TGF-β1..  From: Activation of the Pro-survival Phosphatidylinositol 3-Kinase/AKT <span class="highlight" style="background-color:">Pathway</span> by Transforming Growth Factor-?1 in Mesenchymal Cells Is Mediated by p38 MAPK-dependent Induction of an Autocrine Growth Factor. </a></p><div class="supp"><p class="details"><div>In the current study, we have demonstrated that TGF-β1 induces rapid activation of p38 MAPK followed by a more delayed activation of the PKB/Akt <span class="highlight" style="background-color:">pathway</span> in lung mesenchymal cells/fibroblasts. Activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is mediated by p38 MAPK-dependent production of a putative growth factor(s) that functions in an autocrine/paracrine manner. Activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is sufficient to confer a pro-survival/apoptosis-resistant phenotype to human mesenchymal cells/fibroblasts.</div></p></div><div class="aux"><div class="resc">Jeffrey C. Horowitz, et al. J Biol Chem. ;279(2):1359-1367.</div><p class="links"><a class="dblinks" href="/pubmed/14576166" ref="ordinalpos=1492&amp;ncbi_uid=596517&amp;link_uid=14576166">Citation</a><a class="dblinks" href="/pmc/articles/PMC1360222/" ref="ordinalpos=1492&amp;ncbi_uid=596517&amp;link_uid=PMC1360222">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1493.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2621059/figure/F4/" ref="ordinalpos=1493&amp;ncbi_uid=1699088&amp;link_uid=PMC2621059" href="/pmc/articles/PMC2621059/"><img src="/pmc/articles/instance/2621059/bin/nihms85699f4.gif" src-large="/pmc/articles/instance/2621059/bin/nihms85699f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2621059/" ref="ordinalpos=1493&amp;ncbi_uid=1699088&amp;link_uid=PMC2621059" image-link="/pmc/articles/PMC2621059/figure/F4/" class="imagepopup">Figure 4. Network extension of the innate immune Toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its participation in necroptosis.  From: Identification of a molecular <span class="highlight" style="background-color:">signaling</span> network that regulates a cellular necrotic cell death <span class="highlight" style="background-color:">pathway</span> by a genome wide siRNA screen. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Network was constructed by anchoring on TLR <span class="highlight" style="background-color:">pathway</span> components (CD40, RIPK1, JUN and SPP1) that are also zVAD.fmk hits, using protein-protein interaction data curated from literature and high-throughput yeast-two-hybrid screens. (<b>B</b>) Autocrine production of TNFα is involved in zVAD-induced necroptosis. L929 cells were pretreated with indicated concentrations of anti-mouse TNFα antibody (μg/ml) for 1hr followed by treatment with or without 20μM zVAD.fmk or 30μM Nec-1 for 16hr. (<b>C</b>) Involvement of TLR <span class="highlight" style="background-color:">pathway</span> in necroptosis. L929 cells were treated with indicated chemicals for 19hr. Cellular viability was measured as described in Figure 1A. polyI:C; 25μg/ml, interferon gamma (IFNγ); 1000U/ml, Nec-1; 30μM.</div></p></div><div class="aux"><div class="resc">Junichi Hitomi, et al. Cell. ;135(7):1311-1323.</div><p class="links"><a class="dblinks" href="/pubmed/19109899" ref="ordinalpos=1493&amp;ncbi_uid=1699088&amp;link_uid=19109899">Citation</a><a class="dblinks" href="/pmc/articles/PMC2621059/" ref="ordinalpos=1493&amp;ncbi_uid=1699088&amp;link_uid=PMC2621059">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1494.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3380363/figure/F1/" ref="ordinalpos=1494&amp;ncbi_uid=4336011&amp;link_uid=PMC3380363" href="/pmc/articles/PMC3380363/"><img src="/pmc/articles/instance/3380363/bin/nihms322806f1.gif" src-large="/pmc/articles/instance/3380363/bin/nihms322806f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3380363/" ref="ordinalpos=1494&amp;ncbi_uid=4336011&amp;link_uid=PMC3380363" image-link="/pmc/articles/PMC3380363/figure/F1/" class="imagepopup">Figure 1.  From: Nitric oxide and protein phosphatase 2A provide novel therapeutic opportunities in ER-negative breast cancer. </a></p><div class="supp"><p class="details"><div>Oncogenic <span class="highlight" style="background-color:">signaling</span> pathways regulated by NO and PP2A. NO activates numerous <span class="highlight" style="background-color:">signaling</span> pathways associated with cancer cell proliferation, survival, chemoresistance and migration. Some of the molecular and <span class="highlight" style="background-color:">pathway</span> targets of NO <span class="highlight" style="background-color:">signaling</span> include sGC, HIF-1α, Src, PI3K/Akt, EGFR, Ras, c-Myc and ERK-1/2. Furthermore, NO activates oncogenic pathways by redundant mechanisms; for example, ERK-1/2 is activated from either Ras <span class="highlight" style="background-color:">signaling</span> or sGC activation and PI3K/Akt <span class="highlight" style="background-color:">signaling</span> is activated by either EGFR or Src kinases. These redundant mechanisms indicate that single agent therapies could fail to limit oncogenic NO <span class="highlight" style="background-color:">signaling</span>. The tumor suppressor PP2A is the endogenous negative regulator of many NO-activated <span class="highlight" style="background-color:">signaling</span> pathways. We propose that pharmacological activation of PP2A could represent an effective strategy to limit the myriad of pro-tumor NO effects.</div></p></div><div class="aux"><div class="resc">Christopher H. Switzer, et al. Trends Pharmacol Sci. ;32(11):644-651.</div><p class="links"><a class="dblinks" href="/pubmed/21893353" ref="ordinalpos=1494&amp;ncbi_uid=4336011&amp;link_uid=21893353">Citation</a><a class="dblinks" href="/pmc/articles/PMC3380363/" ref="ordinalpos=1494&amp;ncbi_uid=4336011&amp;link_uid=PMC3380363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1495.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4260825/figure/pbio-1002012-g005/" ref="ordinalpos=1495&amp;ncbi_uid=6962501&amp;link_uid=PMC4260825" href="/pmc/articles/PMC4260825/"><img src="/pmc/articles/instance/4260825/bin/pbio.1002012.g005.gif" src-large="/pmc/articles/instance/4260825/bin/pbio.1002012.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4260825/" ref="ordinalpos=1495&amp;ncbi_uid=6962501&amp;link_uid=PMC4260825" image-link="/pmc/articles/PMC4260825/figure/pbio-1002012-g005/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Signaling</span> complexes can tolerate multiple rearrangement-mediated gene replacements..  From: The Robustness of a <span class="highlight" style="background-color:">Signaling</span> Complex to Domain Rearrangements Facilitates Network Evolution. </a></p><div class="supp"><p class="details"><div>(A) Schematic representation of the reciprocally rearranged variants. (B) Co-expression of Ste20[N]-Ste11[C]+Ste11[N]-Ste20[C] restores <span class="highlight" style="background-color:">pathway</span> activation in the Ste20Δ Ste11Δ strain, co-expression of Ste7[N]-Ste11[C]+Ste11[N]-Ste7[C] restores <span class="highlight" style="background-color:">pathway</span> activation in the Ste7Δ Ste11Δ strain, and co-expression of Ste50[N]-Ste11[C]+Ste11[N]-Ste50[C] restores <span class="highlight" style="background-color:">pathway</span> activation in the Ste50Δ Ste11Δ strain. (C) Changes in network topology resulting from domain rearrangement events in our experiments, mimic changes in network topology that have occurred during evolution. (D) Expression of the domain rearranged variant Ste20[N]-Ste11[C] in the double deletion strain Ste50Δ Ste11Δ rescues <span class="highlight" style="background-color:">pathway</span> activation. Statistically significant differences are marked with asterisks. Data shown in Data S1.</div></p></div><div class="aux"><div class="resc">Paloma M. Sato, et al. PLoS Biol. 2014 December;12(12):e1002012.</div><p class="links"><a class="dblinks" href="/pubmed/25490747" ref="ordinalpos=1495&amp;ncbi_uid=6962501&amp;link_uid=25490747">Citation</a><a class="dblinks" href="/pmc/articles/PMC4260825/" ref="ordinalpos=1495&amp;ncbi_uid=6962501&amp;link_uid=PMC4260825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1496.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC152615/figure/f7/" ref="ordinalpos=1496&amp;ncbi_uid=270974&amp;link_uid=PMC152615" href="/pmc/articles/PMC152615/"><img src="/pmc/articles/instance/152615/bin/jb0831245007.gif" src-large="/pmc/articles/instance/152615/bin/jb0831245007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC152615/" ref="ordinalpos=1496&amp;ncbi_uid=270974&amp;link_uid=PMC152615" image-link="/pmc/articles/PMC152615/figure/f7/" class="imagepopup">FIG. 7. From: Signal Detection and Target Gene Induction by the CpxRA Two-Component System. </a></p><div class="supp"><p class="details"><div>CpxRA two-component signal transduction <span class="highlight" style="background-color:">pathway</span>. Components of the Cpx <span class="highlight" style="background-color:">pathway</span> reside in each compartment of the bacterial cell. The Cpx <span class="highlight" style="background-color:">pathway</span> can sense inducing cues that enter the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at three distinct points. The mechanism by which growth is sensed remains unclear. X represents the second, as yet unidentified, negative regulator of the Cpx <span class="highlight" style="background-color:">pathway</span>. Note that X could be the periplasmic domain of CpxA. OM, outer membrane; IM, inner membrane.</div></p></div><div class="aux"><div class="resc">Patricia A. DiGiuseppe, et al. J Bacteriol. 2003 April;185(8):2432-2440.</div><p class="links"><a class="dblinks" href="/pubmed/12670966" ref="ordinalpos=1496&amp;ncbi_uid=270974&amp;link_uid=12670966">Citation</a><a class="dblinks" href="/pmc/articles/PMC152615/" ref="ordinalpos=1496&amp;ncbi_uid=270974&amp;link_uid=PMC152615">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1497.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4001958/figure/f01/" ref="ordinalpos=1497&amp;ncbi_uid=6188065&amp;link_uid=PMC4001958" href="/pmc/articles/PMC4001958/"><img src="/pmc/articles/instance/4001958/bin/1657-9534-cm-43-03-00235-gf01.gif" src-large="/pmc/articles/instance/4001958/bin/1657-9534-cm-43-03-00235-gf01.jpg" alt="Figure 1. " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4001958/" ref="ordinalpos=1497&amp;ncbi_uid=6188065&amp;link_uid=PMC4001958" image-link="/pmc/articles/PMC4001958/figure/f01/" class="imagepopup">Figure 1. A. Schematic representation of different intracellular events involved in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> AKT/PKB. B. Proposed regulation model of the levels of cyclin D2 and D3 by the mTORC1 .  From: Role of AKT/mTORC1 <span class="highlight" style="background-color:">pathway</span> in pancreatic ?-cell proliferation.. </a></p><div class="aux"><div class="resc">Norman Balcazar Morales, et al. Colomb Med (Cali). 2012 Jul-Sep;43(3):235-243.</div><p class="links"><a class="dblinks" href="/pubmed/24893199" ref="ordinalpos=1497&amp;ncbi_uid=6188065&amp;link_uid=24893199">Citation</a><a class="dblinks" href="/pmc/articles/PMC4001958/" ref="ordinalpos=1497&amp;ncbi_uid=6188065&amp;link_uid=PMC4001958">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1498.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3710884/figure/fig03/" ref="ordinalpos=1498&amp;ncbi_uid=5132856&amp;link_uid=PMC3710884" href="/pmc/articles/PMC3710884/"><img src="/pmc/articles/instance/3710884/bin/pnas.1219852110fig03.gif" src-large="/pmc/articles/instance/3710884/bin/pnas.1219852110fig03.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3710884/" ref="ordinalpos=1498&amp;ncbi_uid=5132856&amp;link_uid=PMC3710884" image-link="/pmc/articles/PMC3710884/figure/fig03/" class="imagepopup">Fig. 3. From: Negative regulation of Toll-like receptor 4 <span class="highlight" style="background-color:">signaling</span> by IL-10-dependent microRNA-146b. </a></p><div class="supp"><p class="details"><div>miR-146b targets the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Canonical pathways significantly enriched for miR-146b predicted target genes as identified by the Ingenuity Pathways Analysis library.</div></p></div><div class="aux"><div class="resc">Graziella Curtale, et al. Proc Natl Acad Sci U S A. 2013 July 9;110(28):11499-11504.</div><p class="links"><a class="dblinks" href="/pubmed/23798430" ref="ordinalpos=1498&amp;ncbi_uid=5132856&amp;link_uid=23798430">Citation</a><a class="dblinks" href="/pmc/articles/PMC3710884/" ref="ordinalpos=1498&amp;ncbi_uid=5132856&amp;link_uid=PMC3710884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1499.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3343649/figure/fig7/" ref="ordinalpos=1499&amp;ncbi_uid=3989963&amp;link_uid=PMC3343649" href="/pmc/articles/PMC3343649/"><img src="/pmc/articles/instance/3343649/bin/cr20123f7.gif" src-large="/pmc/articles/instance/3343649/bin/cr20123f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3343649/" ref="ordinalpos=1499&amp;ncbi_uid=3989963&amp;link_uid=PMC3343649" image-link="/pmc/articles/PMC3343649/figure/fig7/" class="imagepopup">Figure 7.  From: A distinct response to endogenous DNA damage in the development of Nbs1-deficient cortical neurons. </a></p><div class="supp"><p class="details"><div>Working model for Nbs1 function and regulation in neocortex. Endogenous DNA damage during rapid development of neocortical neurons activates ATM-Chk2 and ATR-Chk1 <span class="highlight" style="background-color:">signaling</span>. However, defective Nbs1 impairs ATR-Chk1 <span class="highlight" style="background-color:">pathway</span> rather than ATM-Chk2 <span class="highlight" style="background-color:">signaling</span>. ATM-Chk2 <span class="highlight" style="background-color:">signaling</span> in Nbs1-deficient neocortex activates p53-Hzf <span class="highlight" style="background-color:">pathway</span> and induces defective proliferation rather than apoptosis of neuroprogenitors, leading to microcephaly.</div></p></div><div class="aux"><div class="resc">Rui Li, et al. Cell Res. 2012 May;22(5):859-872.</div><p class="links"><a class="dblinks" href="/pubmed/22212482" ref="ordinalpos=1499&amp;ncbi_uid=3989963&amp;link_uid=22212482">Citation</a><a class="dblinks" href="/pmc/articles/PMC3343649/" ref="ordinalpos=1499&amp;ncbi_uid=3989963&amp;link_uid=PMC3343649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1500.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2926987/figure/F1/" ref="ordinalpos=1500&amp;ncbi_uid=2409618&amp;link_uid=PMC2926987" href="/pmc/articles/PMC2926987/"><img src="/pmc/articles/instance/2926987/bin/nihms227060f1a.gif" src-large="/pmc/articles/instance/2926987/bin/nihms227060f1a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2926987/figure/F1/" ref="ordinalpos=1500&amp;ncbi_uid=2409618&amp;link_uid=PMC2926987" href="/pmc/articles/PMC2926987/"><img src="/pmc/articles/instance/2926987/bin/nihms227060f1b.gif" src-large="/pmc/articles/instance/2926987/bin/nihms227060f1b.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2926987/figure/F1/" ref="ordinalpos=1500&amp;ncbi_uid=2409618&amp;link_uid=PMC2926987" href="/pmc/articles/PMC2926987/"><img src="/pmc/articles/instance/2926987/bin/nihms227060f1c.gif" src-large="/pmc/articles/instance/2926987/bin/nihms227060f1c.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2926987/figure/F1/" ref="ordinalpos=1500&amp;ncbi_uid=2409618&amp;link_uid=PMC2926987" href="/pmc/articles/PMC2926987/"><img src="/pmc/articles/instance/2926987/bin/nihms227060f1d.gif" src-large="/pmc/articles/instance/2926987/bin/nihms227060f1d.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2926987/" ref="ordinalpos=1500&amp;ncbi_uid=2409618&amp;link_uid=PMC2926987" image-link="/pmc/articles/PMC2926987/figure/F1/" class="imagepopup">Figure 1.  From: Systematic investigation of lycopene effects in LNCaP cells by use of novel large-scale proteomic analysis software. </a></p><div class="supp"><p class="details"><div>Teranode<sup>®</sup> Design Suite biological protein <span class="highlight" style="background-color:">pathway</span> modeling process. (a) The proteomics data are used to search against entire <span class="highlight" style="background-color:">pathway</span> networks available via find tools. The search is complete with the list of pathways mapped by the input proteomics data; a selected example of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown. (b) The insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown on the right with nodes and edges, and the identified proteins are colored in turquoise. Property values (i.e. protein expression values) are imported via the menu system; and (c) property annotations are listed by the nodes or displayed in a separate window. Expression changes are color scaled representing reduced to induced by red to green, respectively. (d) Enlarged area view of identified proteins in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown.</div></p></div><div class="aux"><div class="resc">Young Ah Goo, et al. Proteomics Clin Appl. ;1(5):513-523.</div><p class="links"><a class="dblinks" href="/pubmed/20740054" ref="ordinalpos=1500&amp;ncbi_uid=2409618&amp;link_uid=20740054">Citation</a><a class="dblinks" href="/pmc/articles/PMC2926987/" ref="ordinalpos=1500&amp;ncbi_uid=2409618&amp;link_uid=PMC2926987">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="14" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="15" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="16" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:25:17-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB037453FEC8C10000000000497252;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB037453FEC8C10000000000497252" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>