
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="51" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB5960543142C1000000000072CFC6" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="50" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="51" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="52" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="51" /></div>
            <div><h2 class="result_count">Results: 5001 to 5100 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>5001.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2779086/figure/f8/" ref="ordinalpos=5001&amp;ncbi_uid=2134500&amp;link_uid=PMC2779086" href="/pmc/articles/PMC2779086/"><img src="/pmc/articles/instance/2779086/bin/msb200972-f8.gif" src-large="/pmc/articles/instance/2779086/bin/msb200972-f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2779086/" ref="ordinalpos=5001&amp;ncbi_uid=2134500&amp;link_uid=PMC2779086" image-link="/pmc/articles/PMC2779086/figure/f8/" class="imagepopup">Figure 8.  From: Application of an integrated physical and functional screening approach to identify inhibitors of the Wnt <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Nkd1 cooperates with Axin1 to inhibit the Wnt <span class="highlight" style="background-color:">pathway</span>. (<b>A</b>) Nkd1 and Axin1 cooperate to inhibit TOPflash. HEK293T cells were transfected with TOPflash, the indicated amount of 3XFlag-Nkd1 and 20 nM control or Axin1-targeting siRNA. Cells were treated overnight with or without Wnt3A and TOPflash activity was measured by luciferase assay. Data are shown as the mean of three samples (left panel) or percentage of control (right panel) ±standard deviation. (<b>B</b>) Model of inhibition of Wnt3A <span class="highlight" style="background-color:">signaling</span> by Nkd1 and Axin1 cooperation.</div></p></div><div class="aux"><div class="resc">Bryan W Miller, et al. Mol Syst Biol. 2009;5:315-315.</div><p class="links"><a class="dblinks" href="/pubmed/19888210" ref="ordinalpos=5001&amp;ncbi_uid=2134500&amp;link_uid=19888210">Citation</a><a class="dblinks" href="/pmc/articles/PMC2779086/" ref="ordinalpos=5001&amp;ncbi_uid=2134500&amp;link_uid=PMC2779086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5002.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3190892/figure/F5/" ref="ordinalpos=5002&amp;ncbi_uid=3456630&amp;link_uid=PMC3190892" href="/pmc/articles/PMC3190892/"><img src="/pmc/articles/instance/3190892/bin/zbc0421178240005.gif" src-large="/pmc/articles/instance/3190892/bin/zbc0421178240005.jpg" alt="FIGURE 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3190892/" ref="ordinalpos=5002&amp;ncbi_uid=3456630&amp;link_uid=PMC3190892" image-link="/pmc/articles/PMC3190892/figure/F5/" class="imagepopup">FIGURE 5. From: Tripartite Motif 24 (Trim24/Tif1?) Tumor Suppressor Protein Is a Novel Negative Regulator of Interferon (IFN)/Signal Transducers and Activators of Transcription (STAT) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Acting through Retinoic Acid Receptor ? (Rar?) Inhibition. </a></p><div class="supp"><p class="details"><div><b>Model of Trim24 limiting IFN/STAT <span class="highlight" style="background-color:">pathway</span> through inhibition of Rara activity.</b> <i>RXR</i>, retinoid X receptor.</div></p></div><div class="aux"><div class="resc">Johan Tisserand, et al. J Biol Chem. 2011 September 23;286(38):33369-33379.</div><p class="links"><a class="dblinks" href="/pubmed/21768647" ref="ordinalpos=5002&amp;ncbi_uid=3456630&amp;link_uid=21768647">Citation</a><a class="dblinks" href="/pmc/articles/PMC3190892/" ref="ordinalpos=5002&amp;ncbi_uid=3456630&amp;link_uid=PMC3190892">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5003.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2717289/figure/fig1/" ref="ordinalpos=5003&amp;ncbi_uid=1995812&amp;link_uid=PMC2717289" href="/pmc/articles/PMC2717289/"><img src="/pmc/articles/instance/2717289/bin/gr1.gif" src-large="/pmc/articles/instance/2717289/bin/gr1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2717289/" ref="ordinalpos=5003&amp;ncbi_uid=1995812&amp;link_uid=PMC2717289" image-link="/pmc/articles/PMC2717289/figure/fig1/" class="imagepopup">Figure 1.  From: Quantitative Modeling and Analysis of the Transforming Growth Factor β <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> . </a></p><div class="supp"><p class="details"><div>Schematic representation of the <span class="highlight" style="background-color:">pathway</span> components in the integrated model. Numbers in the cartoon refer to the chemical reaction indices in Table 1.</div></p></div><div class="aux"><div class="resc">Seung-Wook Chung, et al. Biophys J. 2009 March 4;96(5):1733-1750.</div><p class="links"><a class="dblinks" href="/pubmed/19254534" ref="ordinalpos=5003&amp;ncbi_uid=1995812&amp;link_uid=19254534">Citation</a><a class="dblinks" href="/pmc/articles/PMC2717289/" ref="ordinalpos=5003&amp;ncbi_uid=1995812&amp;link_uid=PMC2717289">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5004.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399319/figure/F3/" ref="ordinalpos=5004&amp;ncbi_uid=4126040&amp;link_uid=PMC3399319" href="/pmc/articles/PMC3399319/"><img src="/pmc/articles/instance/3399319/bin/ijbsv08p0964g03.gif" src-large="/pmc/articles/instance/3399319/bin/ijbsv08p0964g03.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399319/" ref="ordinalpos=5004&amp;ncbi_uid=4126040&amp;link_uid=PMC3399319" image-link="/pmc/articles/PMC3399319/figure/F3/" class="imagepopup">Figure 3.  From: Complexities of TGF-? Targeted Cancer Therapy. </a></p><div class="supp"><p class="details"><div><b> Inhibition of TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is often elevated in human tumors, and thus a clinical target. This has led to the development of a range of anti-TGF-β-<span class="highlight" style="background-color:">signaling</span> drugs for cancer therapy. The four major classes of TGF-β inhibitors include ligand traps (e.g. 1D11 or Fresolimumab), antisense oligonucleotides (ASO) like Trabedersen, small molecule receptor kinase inhibitors such as LY2109761 or LY2157299, and peptide aptamers (e.g. Trx-SARA).</div></p></div><div class="aux"><div class="resc">Erin C. Connolly, et al. Int J Biol Sci. 2012;8(7):964-978.</div><p class="links"><a class="dblinks" href="/pubmed/22811618" ref="ordinalpos=5004&amp;ncbi_uid=4126040&amp;link_uid=22811618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399319/" ref="ordinalpos=5004&amp;ncbi_uid=4126040&amp;link_uid=PMC3399319">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5005.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3531794/figure/F1/" ref="ordinalpos=5005&amp;ncbi_uid=5599662&amp;link_uid=PMC3531794" href="/pmc/articles/PMC3531794/"><img src="/pmc/articles/instance/3531794/bin/nihms427033f1.gif" src-large="/pmc/articles/instance/3531794/bin/nihms427033f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3531794/" ref="ordinalpos=5005&amp;ncbi_uid=5599662&amp;link_uid=PMC3531794" image-link="/pmc/articles/PMC3531794/figure/F1/" class="imagepopup">Figure 1.  From: Fifteen years of interleukin-27- discovery, advances and translation. </a></p><div class="supp"><p class="details"><div>Impact of IL-27 on lymphocyte <span class="highlight" style="background-color:">signaling</span> pathways. Dimerization of gp130 and IL-27Ra engages JAK1, 2 and Tyk2 that engage the MAPK <span class="highlight" style="background-color:">pathway</span> and activation of multiple STATs, most notably STAT1 and STAT3. The activation of STAT1 is linked to inhibition of GATA-3 and RoRγt but upregulation of PD-L1, T-bet and IL-10. The ability to engage STAT3 is linked to increased proliferation as well as IL-10 while the MAPK <span class="highlight" style="background-color:">pathway</span> intersects with AHR to promote IL-10 and IL-21.</div></p></div><div class="aux"><div class="resc">C.A. Hunter, et al. Immunity. ;37(6):960-969.</div><p class="links"><a class="dblinks" href="/pubmed/23244718" ref="ordinalpos=5005&amp;ncbi_uid=5599662&amp;link_uid=23244718">Citation</a><a class="dblinks" href="/pmc/articles/PMC3531794/" ref="ordinalpos=5005&amp;ncbi_uid=5599662&amp;link_uid=PMC3531794">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5006.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3213765/figure/F13/" ref="ordinalpos=5006&amp;ncbi_uid=3353818&amp;link_uid=PMC3213765" href="/pmc/articles/PMC3213765/"><img src="/pmc/articles/instance/3213765/bin/nihms-334040-f0013.gif" src-large="/pmc/articles/instance/3213765/bin/nihms-334040-f0013.jpg" alt="Figure 8.13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3213765/" ref="ordinalpos=5006&amp;ncbi_uid=3353818&amp;link_uid=PMC3213765" image-link="/pmc/articles/PMC3213765/figure/F13/" class="imagepopup">Figure 8.13.  From: Cerebellum: Development and Medulloblastoma. </a></p><div class="supp"><p class="details"><div>Schematic of the sonic hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the primary cilium. (A) In the absence of Shh, Ptch resides at the base of the cilium whereas Smo is in the cytoplasm and the <span class="highlight" style="background-color:">pathway</span> is off. (B) In the presence of Shh or in medulloblastoma, Smo becomes located to the cilium. In the absence of Ptch or when Smo or SuFu are mutated, Gli and the levels of targets increase. Asterisk (*) identifies proteins mutated in medulloblastoma.</div></p></div><div class="aux"><div class="resc">Martine F. Roussel, et al. Curr Top Dev Biol. ;94:235-282.</div><p class="links"><a class="dblinks" href="/pubmed/21295689" ref="ordinalpos=5006&amp;ncbi_uid=3353818&amp;link_uid=21295689">Citation</a><a class="dblinks" href="/pmc/articles/PMC3213765/" ref="ordinalpos=5006&amp;ncbi_uid=3353818&amp;link_uid=PMC3213765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5007.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2861642/figure/F3/" ref="ordinalpos=5007&amp;ncbi_uid=2307982&amp;link_uid=PMC2861642" href="/pmc/articles/PMC2861642/"><img src="/pmc/articles/instance/2861642/bin/1479-5876-8-30-3.gif" src-large="/pmc/articles/instance/2861642/bin/1479-5876-8-30-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2861642/" ref="ordinalpos=5007&amp;ncbi_uid=2307982&amp;link_uid=PMC2861642" image-link="/pmc/articles/PMC2861642/figure/F3/" class="imagepopup">Figure 3.  From: Epstein-Barr virus encoded latent membrane protein 1 regulates mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes which predict poor prognosis of nasopharyngeal carcinoma. </a></p><div class="supp"><p class="details"><div><b>LMP1-activates the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through the phosphorylated AKT/mTOR/P70S6K/4EBP1 cascade in NPC cell lines</b>. <b>A</b>. Western blot of stable transfected NPC cell line HONE1-LMP1 and LMP1 transiently transfected NPC cell line 6-10B. Increased expression of p-AKT, p-mTOR, p-P70S6K and p-4EBP1 was seen in both the HONE1-LMP1 and 6-10B-LMP1 cell lines. GAPDH acted as a loading control. <b>B</b>. Changes in gene expression measured by densitometry. LMP1 expression increased 2.9-fold in HONE1-LMP1 cells, and 2.8-fold in 6-10B-LMP1 cells compared to the vector control. Except for expression of mTOR in both cell lines, and AKT in 6-10B-LMP1 cell line, all genes increased in expression, with the ratio ranging from 1.2 to 1.9. Experiments were repeated three times independently, and mean and standard deviation are shown.</div></p></div><div class="aux"><div class="resc">Jing Chen, et al. J Transl Med. 2010;8:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/20338061" ref="ordinalpos=5007&amp;ncbi_uid=2307982&amp;link_uid=20338061">Citation</a><a class="dblinks" href="/pmc/articles/PMC2861642/" ref="ordinalpos=5007&amp;ncbi_uid=2307982&amp;link_uid=PMC2861642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5008.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2778586/figure/Fig1/" ref="ordinalpos=5008&amp;ncbi_uid=2129461&amp;link_uid=PMC2778586" href="/pmc/articles/PMC2778586/"><img src="/pmc/articles/instance/2778586/bin/12079_2009_66_Fig1_HTML.gif" src-large="/pmc/articles/instance/2778586/bin/12079_2009_66_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2778586/" ref="ordinalpos=5008&amp;ncbi_uid=2129461&amp;link_uid=PMC2778586" image-link="/pmc/articles/PMC2778586/figure/Fig1/" class="imagepopup">Fig. 1.  From: The matricellular functions of small leucine-rich proteoglycans (SLRPs). </a></p><div class="supp"><p class="details"><div>Cell type-specific functions mediated by decorin via modulation of IGF-IR <span class="highlight" style="background-color:">signaling</span>. Decorin binds and induces phosphorylation of the insulin-like growth factor-I (IGF-I) receptor, causing downstream activation of phosphoinositide-3 kinase (PI3K) and the Akt/protein kinase B (Akt/PKB) <span class="highlight" style="background-color:">pathway</span>. In endothelial and renal tubular epithelial cells this <span class="highlight" style="background-color:">signaling</span> cascade leads to an inhibition of apoptosis, whereas in renal fibroblasts <span class="highlight" style="background-color:">signaling</span> through mammalian target of rapamycin (mTOR) and p70S6 Kinase (p70S6K) leads to increased translation and synthesis of fibrillin-1. These pathways demonstrate the intricate regulatory mechanisms whereby decorin modulates IGF-IR <span class="highlight" style="background-color:">signaling</span> in a cell type-specific manner, thereby giving rise to different biological outcomes</div></p></div><div class="aux"><div class="resc">Rosetta Merline, et al. J Cell Commun Signal. 2009 December;3(3-4):323-335.</div><p class="links"><a class="dblinks" href="/pubmed/19809894" ref="ordinalpos=5008&amp;ncbi_uid=2129461&amp;link_uid=19809894">Citation</a><a class="dblinks" href="/pmc/articles/PMC2778586/" ref="ordinalpos=5008&amp;ncbi_uid=2129461&amp;link_uid=PMC2778586">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5009.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4014440/figure/pgen-1004297-g009/" ref="ordinalpos=5009&amp;ncbi_uid=6132529&amp;link_uid=PMC4014440" href="/pmc/articles/PMC4014440/"><img src="/pmc/articles/instance/4014440/bin/pgen.1004297.g009.gif" src-large="/pmc/articles/instance/4014440/bin/pgen.1004297.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4014440/" ref="ordinalpos=5009&amp;ncbi_uid=6132529&amp;link_uid=PMC4014440" image-link="/pmc/articles/PMC4014440/figure/pgen-1004297-g009/" class="imagepopup">Figure 9. Summary of RPM-1 <span class="highlight" style="background-color:">signaling</span>..  From: RPM-1 Uses Both Ubiquitin Ligase and Phosphatase-Based Mechanisms to Regulate DLK-1 during Neuronal Development. </a></p><div class="supp"><p class="details"><div>RPM-1 is a positive regulator of the GLO-4 <span class="highlight" style="background-color:">pathway</span>, and acts as part of a complex with FSN-1 that ubiquitinates and negatively regulates DLK-1. PPM-2 is also part of an RPM-1 protein complex and negatively regulates DLK-1 via dephosphorylation at S874.</div></p></div><div class="aux"><div class="resc">Scott T. Baker, et al. PLoS Genet. 2014 May;10(5):e1004297.</div><p class="links"><a class="dblinks" href="/pubmed/24810406" ref="ordinalpos=5009&amp;ncbi_uid=6132529&amp;link_uid=24810406">Citation</a><a class="dblinks" href="/pmc/articles/PMC4014440/" ref="ordinalpos=5009&amp;ncbi_uid=6132529&amp;link_uid=PMC4014440">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5010.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3087605/figure/F2/" ref="ordinalpos=5010&amp;ncbi_uid=2934201&amp;link_uid=PMC3087605" href="/pmc/articles/PMC3087605/"><img src="/pmc/articles/instance/3087605/bin/nihms-292696-f0002.gif" src-large="/pmc/articles/instance/3087605/bin/nihms-292696-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3087605/" ref="ordinalpos=5010&amp;ncbi_uid=2934201&amp;link_uid=PMC3087605" image-link="/pmc/articles/PMC3087605/figure/F2/" class="imagepopup">Figure 2.  From: Sepsis, Complement, and the Dysregulated Inflammatory Response. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> activation by C5a on neutrophils, monocytes/macrophages and endothelial cells, resulting in compromised innate immunity, resistance of neutrophils to apoptosis, a cytokine storm and disseminated intravascular coagulation.</div></p></div><div class="aux"><div class="resc">Peter A. Ward, et al. J Cell Mol Med. ;13(10):4154-4160.</div><p class="links"><a class="dblinks" href="/pubmed/19725914" ref="ordinalpos=5010&amp;ncbi_uid=2934201&amp;link_uid=19725914">Citation</a><a class="dblinks" href="/pmc/articles/PMC3087605/" ref="ordinalpos=5010&amp;ncbi_uid=2934201&amp;link_uid=PMC3087605">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5011.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2781983/figure/F3/" ref="ordinalpos=5011&amp;ncbi_uid=2452567&amp;link_uid=PMC2781983" href="/pmc/articles/PMC2781983/"><img src="/pmc/articles/instance/2781983/bin/zbc0350984580003.gif" src-large="/pmc/articles/instance/2781983/bin/zbc0350984580003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2781983/" ref="ordinalpos=5011&amp;ncbi_uid=2452567&amp;link_uid=PMC2781983" image-link="/pmc/articles/PMC2781983/figure/F3/" class="imagepopup">FIGURE 3. From: Keep Nibbling at the Edges. </a></p><div class="supp"><p class="details"><div>The discovery of PDK1 (47) enabled an outline of the insulin-stimulated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that leads to the activation of glycogen synthase to be formulated for the first time. <i>PI3K</i>, Class 1 phosphatidylinositol 3-kinase.</div></p></div><div class="aux"><div class="resc">Philip Cohen. J Biol Chem. 2009 September 4;284(36):23891-23901.</div><p class="links"><a class="dblinks" href="/pubmed/19515842" ref="ordinalpos=5011&amp;ncbi_uid=2452567&amp;link_uid=19515842">Citation</a><a class="dblinks" href="/pmc/articles/PMC2781983/" ref="ordinalpos=5011&amp;ncbi_uid=2452567&amp;link_uid=PMC2781983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5012.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657775/figure/F9/" ref="ordinalpos=5012&amp;ncbi_uid=1704956&amp;link_uid=PMC2657775" href="/pmc/articles/PMC2657775/"><img src="/pmc/articles/instance/2657775/bin/1752-0509-3-11-9.gif" src-large="/pmc/articles/instance/2657775/bin/1752-0509-3-11-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657775/" ref="ordinalpos=5012&amp;ncbi_uid=1704956&amp;link_uid=PMC2657775" image-link="/pmc/articles/PMC2657775/figure/F9/" class="imagepopup">Figure 9.  From: A Drosophila systems model of pentylenetetrazole induced locomotor plasticity responsive to antiepileptic drugs. </a></p><div class="supp"><p class="details"><div><b>Enrichment of Jak-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Pink boxes indicate CG7583 extended network genes that overrepresent one or more GO processes. For details, see text.</div></p></div><div class="aux"><div class="resc">Farhan Mohammad, et al. BMC Syst Biol. 2009;3:11-11.</div><p class="links"><a class="dblinks" href="/pubmed/19154620" ref="ordinalpos=5012&amp;ncbi_uid=1704956&amp;link_uid=19154620">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657775/" ref="ordinalpos=5012&amp;ncbi_uid=1704956&amp;link_uid=PMC2657775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5013.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3773788/figure/fig01/" ref="ordinalpos=5013&amp;ncbi_uid=5900800&amp;link_uid=PMC3773788" href="/pmc/articles/PMC3773788/"><img src="/pmc/articles/instance/3773788/bin/pnas.1303558110fig01.gif" src-large="/pmc/articles/instance/3773788/bin/pnas.1303558110fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3773788/" ref="ordinalpos=5013&amp;ncbi_uid=5900800&amp;link_uid=PMC3773788" image-link="/pmc/articles/PMC3773788/figure/fig01/" class="imagepopup">Fig. 1. From: ETV4 promotes metastasis in response to activation of PI3-kinase and Ras <span class="highlight" style="background-color:">signaling</span> in a mouse model of advanced prostate cancer. </a></p><div class="supp"><p class="details"><div>Coactivation of PI3-kinase and Ras <span class="highlight" style="background-color:">signaling</span> pathways is associated with adverse patient outcome. (<i>A</i>) Correlation of PI3-kinase and Ras <span class="highlight" style="background-color:">signaling</span> in human prostate cancer from the Taylor et al. (6) dataset. Shown is a heat map in which each clinical sample (i.e., primary tumors of the indicated Gleason scores and metastases) was evaluated to determine status for PI3-kinase and Ras <span class="highlight" style="background-color:">signaling</span> (<i>Materials and Methods</i>). Correlation coefficients and <i>P</i> values of the correlations are shown for both Spearman rho and Kendall z calculations. The color key indicates relative expression levels of the <span class="highlight" style="background-color:">pathway</span>. (<i>B</i>) Kaplan–Meier analyses showing the association of activation of both PI3-kinase and Ras <span class="highlight" style="background-color:">signaling</span> with adverse patient outcome in two independent cohorts using biochemical recurrence-free (BCR-free; <i>Left</i>) and prostate cancer-specific survival (<i>Right</i>) as clinical endpoints.</div></p></div><div class="aux"><div class="resc">Alvaro Aytes, et al. Proc Natl Acad Sci U S A. 2013 September 10;110(37):E3506-E3515.</div><p class="links"><a class="dblinks" href="/pubmed/23918374" ref="ordinalpos=5013&amp;ncbi_uid=5900800&amp;link_uid=23918374">Citation</a><a class="dblinks" href="/pmc/articles/PMC3773788/" ref="ordinalpos=5013&amp;ncbi_uid=5900800&amp;link_uid=PMC3773788">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5014.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3706185/figure/F3/" ref="ordinalpos=5014&amp;ncbi_uid=5100180&amp;link_uid=PMC3706185" href="/pmc/articles/PMC3706185/"><img src="/pmc/articles/instance/3706185/bin/JCI69187.f3.gif" src-large="/pmc/articles/instance/3706185/bin/JCI69187.f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3706185/" ref="ordinalpos=5014&amp;ncbi_uid=5100180&amp;link_uid=PMC3706185" image-link="/pmc/articles/PMC3706185/figure/F3/" class="imagepopup">Figure 3. A CD22-dependent tolerogenic program inhibits basal <span class="highlight" style="background-color:">signaling</span> in the Akt survival <span class="highlight" style="background-color:">pathway</span> and drives nuclear import of FoxO1..  From: Antigenic liposomes displaying CD22 ligands induce antigen-specific B cell apoptosis. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Western blot analysis of BCR <span class="highlight" style="background-color:">signaling</span> components in WT and <i>Cd22</i>-KO IgM<sup>HEL</sup> B cells 30 minutes after stimulation of cells with the indicated liposomes or PBS as a control. STALs inhibit phosphorylation of <span class="highlight" style="background-color:">signaling</span> components of all major BCR <span class="highlight" style="background-color:">signaling</span> pathways and induce hypophosphorylation of Akt and FoxO1 in WT B cells, but not <i>Cd22</i>-deficient IgM<sup>HEL</sup> B cells. Data are a subset of Supplemental Figure 4. (<b>B</b>) Analysis of FoxO1 staining in IgM<sup>HEL</sup> B cells by confocal microscopy. Cells were stimulated for 2 hours with the indicated liposomes and stained with anti-FoxO1, phalloidin, and DAPI. Inserts are a representative cell at 3 times the magnification. Original magnification, ×63. </div></p></div><div class="aux"><div class="resc">Matthew S. Macauley, et al. J Clin Invest. 2013 July 1;123(7):3074-3083.</div><p class="links"><a class="dblinks" href="/pubmed/23722906" ref="ordinalpos=5014&amp;ncbi_uid=5100180&amp;link_uid=23722906">Citation</a><a class="dblinks" href="/pmc/articles/PMC3706185/" ref="ordinalpos=5014&amp;ncbi_uid=5100180&amp;link_uid=PMC3706185">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5015.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3791549/figure/F4/" ref="ordinalpos=5015&amp;ncbi_uid=5380286&amp;link_uid=PMC3791549" href="/pmc/articles/PMC3791549/"><img src="/pmc/articles/instance/3791549/bin/fimmu-04-00319-g004.gif" src-large="/pmc/articles/instance/3791549/bin/fimmu-04-00319-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3791549/" ref="ordinalpos=5015&amp;ncbi_uid=5380286&amp;link_uid=PMC3791549" image-link="/pmc/articles/PMC3791549/figure/F4/" class="imagepopup">Figure 4.  From: Nucleic Acid Sensors and Type I Interferon Production in Systemic Lupus Erythematosus. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> through NLR receptors in lupus</b>. Intracellular DNA following microbial infection or phagocytosis of immune complexes can potentially trigger the assembly of NLRs. The nucleic acid-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> converges on the adaptor STING and the kinase TBK1, which phosphorylates IRF3 to mediate downstream <span class="highlight" style="background-color:">signaling</span> events leading to transcriptional induction of type 1 IFN and other inflammatory cytokines.</div></p></div><div class="aux"><div class="resc">Meena Shrivastav, et al. Front Immunol. 2013;4:319.</div><p class="links"><a class="dblinks" href="/pubmed/24109483" ref="ordinalpos=5015&amp;ncbi_uid=5380286&amp;link_uid=24109483">Citation</a><a class="dblinks" href="/pmc/articles/PMC3791549/" ref="ordinalpos=5015&amp;ncbi_uid=5380286&amp;link_uid=PMC3791549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5016.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1847584/figure/F4/" ref="ordinalpos=5016&amp;ncbi_uid=897065&amp;link_uid=PMC1847584" href="/pmc/articles/PMC1847584/"><img src="/pmc/articles/instance/1847584/bin/nihms-18015-0004.gif" src-large="/pmc/articles/instance/1847584/bin/nihms-18015-0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1847584/" ref="ordinalpos=5016&amp;ncbi_uid=897065&amp;link_uid=PMC1847584" image-link="/pmc/articles/PMC1847584/figure/F4/" class="imagepopup">Figure 4.  From: A Mechanism for Cell Cycle Regulation of MAP Kinase <span class="highlight" style="background-color:">Signaling</span> in a Yeast Differentiation <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Disruption of Ste5 membrane localization by G1 CDK activity or negative chargeLocalization of GFP-Ste5-Q59L ± 8A, expressed from the <i>GAL1</i> promoter in <i>ste4</i>Δ <i>ste7</i>Δ cells ± <i>P</i><sub><i>GAL1</i></sub><i>-CLN2</i>. Note that hyperpolarized bud growth is due to Cln2 overexpression, not mating <span class="highlight" style="background-color:">signaling</span>. Also see Figure S2B.The 16E mutations disrupt Ste5 membrane localization mediated by the PM domain (Q59L), but not that mediated by a foreign transmembrane domain (CTM). Top, membrane localization induces mating <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>, causing pear-shaped “shmoo” morphology. Bottom, localization results were similar in a non-<span class="highlight" style="background-color:">signaling</span> strain (<i>ste4</i>Δ <i>ste7</i>Δ).Negative charge disrupts membrane localization of Ste5 N-terminal fragments. Localization was compared (in <i>ste4</i>Δ <i>ste7</i>Δ cells) for WT and mutant derivatives of GFP-Ste5(1-214) and GST-GFP-Ste5(1-125), which can localize to the membrane in the absence of pheromone, Gβγ, and other Ste5 sequences (Winters et al., 2005).</div></p></div><div class="aux"><div class="resc">Shelly C. Strickfaden, et al. Cell. ;128(3):519-531.</div><p class="links"><a class="dblinks" href="/pubmed/17289571" ref="ordinalpos=5016&amp;ncbi_uid=897065&amp;link_uid=17289571">Citation</a><a class="dblinks" href="/pmc/articles/PMC1847584/" ref="ordinalpos=5016&amp;ncbi_uid=897065&amp;link_uid=PMC1847584">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5017.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2814915/figure/F1/" ref="ordinalpos=5017&amp;ncbi_uid=2764278&amp;link_uid=PMC2814915" href="/pmc/articles/PMC2814915/"><img src="/pmc/articles/instance/2814915/bin/nihms163452f1.gif" src-large="/pmc/articles/instance/2814915/bin/nihms163452f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2814915/" ref="ordinalpos=5017&amp;ncbi_uid=2764278&amp;link_uid=PMC2814915" image-link="/pmc/articles/PMC2814915/figure/F1/" class="imagepopup">Figure 1. Wnt <span class="highlight" style="background-color:">pathway</span> genes are differentially expressed in the developing AV valves at E12.5 and E17.5.  From: Wnt <span class="highlight" style="background-color:">signaling</span> in heart valve development and osteogenic gene induction. </a></p><div class="supp"><p class="details"><div>An Affymetrix microarray gene expression analysis was performed on duplicate samples of RNA isolated from E12.5 AV cushions or E17.5 AV valves. The presence of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes was determined among genes with &gt;2.0 fold statistically significant change at E12.5 versus E17.5 (Chakraborty et al., 2008). A heat map was generated of relative gene expression levels based on raw intensity values. Red indicates increased expression, blue indicates decreased expression, and yellow indicates no change in expression.</div></p></div><div class="aux"><div class="resc">Christina M. Alfieri, et al. Dev Biol. ;338(2):127.</div><p class="links"><a class="dblinks" href="/pubmed/19961844" ref="ordinalpos=5017&amp;ncbi_uid=2764278&amp;link_uid=19961844">Citation</a><a class="dblinks" href="/pmc/articles/PMC2814915/" ref="ordinalpos=5017&amp;ncbi_uid=2764278&amp;link_uid=PMC2814915">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5018.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1135298/figure/pbio-0030201-g008/" ref="ordinalpos=5018&amp;ncbi_uid=533546&amp;link_uid=PMC1135298" href="/pmc/articles/PMC1135298/"><img src="/pmc/articles/instance/1174793/bin/pbio.0030201.g008.gif" src-large="/pmc/articles/instance/1174793/bin/pbio.0030201.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1135298/" ref="ordinalpos=5018&amp;ncbi_uid=533546&amp;link_uid=PMC1135298" image-link="/pmc/articles/PMC1135298/figure/pbio-0030201-g008/" class="imagepopup">Figure 8. Genetic Interactions of Toll <span class="highlight" style="background-color:">Signaling</span> and ADGF-A.  From: A Role for Adenosine Deaminase in Drosophila Larval Development. </a></p><div class="supp"><p class="details"><div>Survival rate and melanotic tumor formation were compared in mutants in the Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and in similar mutants with overexpression of ADGF-A using the <i>HS-ADGF-A</i> construct.<br />(A) The bar graph shows the percentage of the pupae (blue bars) and adult flies (purple bars) demonstrating the larval and pupal survival of each genotype. The <i>x</i>-axis shows the genotypes and is shared with (B). Flies heterozygous for the <i>cact</i> mutation were used as a control.<br />(B) Percentage of late third instar larvae presenting melanotic tumor formation.</div></p></div><div class="aux"><div class="resc">Tomas Dolezal, et al. PLoS Biol. 2005 July;3(7):e201.</div><p class="links"><a class="dblinks" href="/pubmed/15907156" ref="ordinalpos=5018&amp;ncbi_uid=533546&amp;link_uid=15907156">Citation</a><a class="dblinks" href="/pmc/articles/PMC1135298/" ref="ordinalpos=5018&amp;ncbi_uid=533546&amp;link_uid=PMC1135298">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5019.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633414/figure/F3/" ref="ordinalpos=5019&amp;ncbi_uid=4821640&amp;link_uid=PMC3633414" href="/pmc/articles/PMC3633414/"><img src="/pmc/articles/instance/3633414/bin/nihms316126f3.gif" src-large="/pmc/articles/instance/3633414/bin/nihms316126f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633414/" ref="ordinalpos=5019&amp;ncbi_uid=4821640&amp;link_uid=PMC3633414" image-link="/pmc/articles/PMC3633414/figure/F3/" class="imagepopup">Figure 3.  From: Functional Genomic Analyses Identify Pathways Dysregulated by Progranulin Deficiency Implicating Wnt <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A. Gene ontology analysis was carried out using DAVID on the set of differentially expressed genes. B. qRT-PCR was performed on some of the important players in the Wnt <span class="highlight" style="background-color:">pathway</span> to verify these gene expression changes. The left two bars show expression changes on array, and the right bar shows expression changes as measured by qPCR. qPCR was performed by comparing the second GRN hairpin versus the scrambled hairpin. C. KEGG <span class="highlight" style="background-color:">Pathway</span> for Wnt <span class="highlight" style="background-color:">Signaling</span>, with differentially expressed genes during GRNi (p&lt;0.05) circled. Upregulated genes are colored red and downregulated genes are colored green. See also figures S4 and S5 and table S2.</div></p></div><div class="aux"><div class="resc">Ezra Y. Rosen, et al. Neuron. 2011 September 22;71(6):1030-1042.</div><p class="links"><a class="dblinks" href="/pubmed/21943601" ref="ordinalpos=5019&amp;ncbi_uid=4821640&amp;link_uid=21943601">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633414/" ref="ordinalpos=5019&amp;ncbi_uid=4821640&amp;link_uid=PMC3633414">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5020.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3608916/figure/F2/" ref="ordinalpos=5020&amp;ncbi_uid=4747285&amp;link_uid=PMC3608916" href="/pmc/articles/PMC3608916/"><img src="/pmc/articles/instance/3608916/bin/fendo-04-00042-g002.gif" src-large="/pmc/articles/instance/3608916/bin/fendo-04-00042-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3608916/" ref="ordinalpos=5020&amp;ncbi_uid=4747285&amp;link_uid=PMC3608916" image-link="/pmc/articles/PMC3608916/figure/F2/" class="imagepopup">Figure 2.  From: New Modulators for IGF-I Activity within IGF-I Processing Products. </a></p><div class="supp"><p class="details"><div><b>E-peptides affect IGF-IR <span class="highlight" style="background-color:">signaling</span></b>. <b>(A)</b> A Kinase Receptor Activation Assay (KIRA) to specifically measure IGF-IR phosphorylation was used. IGF-I at 2 nM was tested in addition to increasing amounts of synthetic E-peptides. Both EA and EB showed a dose-dependent augmentation of IGF-IR phosphorylation compared to IGF-I alone. Data taken from Brisson and Barton (2012). <b>(B)</b> Normal conditions when IGF-I is added to myoblasts, activating IGF-IR, and leading to increases in MAPK and PI3K/Akt <span class="highlight" style="background-color:">signaling</span>. <b>(C)</b> When IGF-I is present, synthetic E-peptides increase receptor activation and tune downstream <span class="highlight" style="background-color:">signaling</span> toward the MAPK <span class="highlight" style="background-color:">pathway</span>. <b>(D)</b> Pro-IGF-I leads to more IGF-IR phosphorylation than mature IGF-I alone, but the consequences on downstream <span class="highlight" style="background-color:">signaling</span> are unknown. <b>(E)</b> E-peptide MAPK activation is inhibited via NVP AEW541, an IGF-IR inhibitor. <b>(F)</b> E-peptide IGF-IR augmentation requires IGF-I.</div></p></div><div class="aux"><div class="resc">Becky K. Brisson, et al. Front Endocrinol (Lausanne). 2013;4:42.</div><p class="links"><a class="dblinks" href="/pubmed/23543904" ref="ordinalpos=5020&amp;ncbi_uid=4747285&amp;link_uid=23543904">Citation</a><a class="dblinks" href="/pmc/articles/PMC3608916/" ref="ordinalpos=5020&amp;ncbi_uid=4747285&amp;link_uid=PMC3608916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5021.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3771960/figure/F2/" ref="ordinalpos=5021&amp;ncbi_uid=5532924&amp;link_uid=PMC3771960" href="/pmc/articles/PMC3771960/"><img src="/pmc/articles/instance/3771960/bin/3000fig2.gif" src-large="/pmc/articles/instance/3771960/bin/3000fig2.jpg" alt="FIGURE 2:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3771960/" ref="ordinalpos=5021&amp;ncbi_uid=5532924&amp;link_uid=PMC3771960" image-link="/pmc/articles/PMC3771960/figure/F2/" class="imagepopup">FIGURE 2:.  From: Drosophila SOCS36E negatively regulates JAK/STAT <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> via two separable mechanisms. </a></p><div class="supp"><p class="details"><div>SOCS36E can suppress the JAK/STAT <span class="highlight" style="background-color:">pathway</span> in a manner independent of Elongin B/C and Cullin-5. (A and B) JAK/STAT <span class="highlight" style="background-color:">pathway</span> activity as indicated by the <i>6×2xDrafLuc</i> reporter in cells treated with dsRNAs targeting the indicated genes in different combinations. Individual genes were targeted by the same quantity of dsRNA, with the total amount of dsRNA kept constant by the addition of control dsRNA. Cells were treated with Upd2-GFP–conditioned (A) or mock-conditioned (B) media, and <span class="highlight" style="background-color:">pathway</span> activity was measured using the <i>6×2xDrafLuc</i> reporter. Significance is shown for selected combinations with ***, <i>p</i> &lt; 0.0001, **, <i>p</i> &lt; 0.001, *, <i>p</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Wojciech Stec, et al. Mol Biol Cell. 2013 September 15;24(18):3000-3009.</div><p class="links"><a class="dblinks" href="/pubmed/23885117" ref="ordinalpos=5021&amp;ncbi_uid=5532924&amp;link_uid=23885117">Citation</a><a class="dblinks" href="/pmc/articles/PMC3771960/" ref="ordinalpos=5021&amp;ncbi_uid=5532924&amp;link_uid=PMC3771960">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5022.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1563714/figure/f5/" ref="ordinalpos=5022&amp;ncbi_uid=812192&amp;link_uid=PMC1563714" href="/pmc/articles/PMC1563714/"><img src="/pmc/articles/instance/1563714/bin/zjv0150680050005.gif" src-large="/pmc/articles/instance/1563714/bin/zjv0150680050005.jpg" alt="FIG. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1563714/" ref="ordinalpos=5022&amp;ncbi_uid=812192&amp;link_uid=PMC1563714" image-link="/pmc/articles/PMC1563714/figure/f5/" class="imagepopup">FIG. 5. From: Induction of the Early Growth Response 1 Gene by Epstein-Barr Virus Lytic Transactivator Zta. </a></p><div class="supp"><p class="details"><div>The ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by Zta and is important for Zta-mediated induction of Egr-1. (A) Activation of the ERK <span class="highlight" style="background-color:">pathway</span> by Zta. HONE-tetonZ and control HONE-1 cells were treated with (+) or without (−) doxycycline (dox) for 24 h and then subjected to an immunoblotting assay. Antibodies used to detect phosphorylated and total forms of p44/p42 ERK, p38 MAPK, and p54/p46 JNK were purchased from Cell <span class="highlight" style="background-color:">Signaling</span> Technology. (B) Reduction of Zta-induced Egr-1 expression by inhibition of the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. HONE-tetonZ cells were left untreated or pretreated with solvent only (dimethyl sulfoxide) or inhibitors of ERK <span class="highlight" style="background-color:">signaling</span> (PD98059, 50 μM; U0126, 20 μM; Calbiochem) for 1 h and then subjected to treatment with doxycycline (dox) for 24 h. Protein expression was examined with an immunoblotting assay. (C) Inhibition of ERK activation impairs Elk-1 binding to the Egr-1 promoter. Nuclear extracts treated as indicated were subjected to EMSA with probe SE, which covers the most upstream SRE-Ets of the Egr-1 promoter (Fig. 4A and D). An immunoblotting assay was also performed to detect Elk-1 and phosphorylated and total p44/p42 ERKs in the nuclear extracts.</div></p></div><div class="aux"><div class="resc">Yao Chang, et al. J Virol. 2006 August;80(15):7748-7755.</div><p class="links"><a class="dblinks" href="/pubmed/16840354" ref="ordinalpos=5022&amp;ncbi_uid=812192&amp;link_uid=16840354">Citation</a><a class="dblinks" href="/pmc/articles/PMC1563714/" ref="ordinalpos=5022&amp;ncbi_uid=812192&amp;link_uid=PMC1563714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5023.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2647339/figure/F2/" ref="ordinalpos=5023&amp;ncbi_uid=1701414&amp;link_uid=PMC2647339" href="/pmc/articles/PMC2647339/"><img src="/pmc/articles/instance/2657459/bin/zpq9990967740002.gif" src-large="/pmc/articles/instance/2657459/bin/zpq9990967740002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2647339/" ref="ordinalpos=5023&amp;ncbi_uid=1701414&amp;link_uid=PMC2647339" image-link="/pmc/articles/PMC2647339/figure/F2/" class="imagepopup">Fig. 2. From: Angiotensin II <span class="highlight" style="background-color:">signaling</span> increases activity of the renal Na-Cl cotransporter through a WNK4-SPAK-dependent <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Effects of AngII <span class="highlight" style="background-color:">signaling</span> in the presence of PHAII-WNK4, WNK1 and WNK3. (<i>A</i>) PHAII mutant WNK4 phenocopys the effect of AngII. Thiazide-sensitive <sup>22</sup>Na<sup>+</sup> uptake was measured in oocytes injected with NCC and AT1R alone or together with wild-type WNK4 or WNK4 harboring either the Q562E or D561A PHAII mutation in the presence or absence of AngII. Unlike WT-WNK4, PHAII-WNK4s show no significant inhibition of NCC activity, and AngII <span class="highlight" style="background-color:">signaling</span> imparts no significant further increase in NCC activity. (<i>B</i> and <i>C</i>) AngII <span class="highlight" style="background-color:">signaling</span> does not alter NCC activity in the presence of WNK1 or WNK3. Thiazide-sensitive <sup>22</sup>Na<sup>+</sup> uptake was measured as in panel A except that WNK1 or WNK3 has been substituted for WNK4. WNK1 has no effect on NCC in the presence or absence of AngII; WNK3 markedly increases NCC activity as previously described, however, AngII <span class="highlight" style="background-color:">signaling</span> does not modulate this effect. *, significantly different from the uptake observed in NCC+AT1R group in the absence of AngII.</div></p></div><div class="aux"><div class="resc">Pedro San-Cristobal, et al. Proc Natl Acad Sci U S A. 2009 March 17;106(11):4384-4389.</div><p class="links"><a class="dblinks" href="/pubmed/19240212" ref="ordinalpos=5023&amp;ncbi_uid=1701414&amp;link_uid=19240212">Citation</a><a class="dblinks" href="/pmc/articles/PMC2647339/" ref="ordinalpos=5023&amp;ncbi_uid=1701414&amp;link_uid=PMC2647339">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5024.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4259928/figure/F1/" ref="ordinalpos=5024&amp;ncbi_uid=6945746&amp;link_uid=PMC4259928" href="/pmc/articles/PMC4259928/"><img src="/pmc/articles/instance/4259928/bin/WJCO-5-1020-g001.gif" src-large="/pmc/articles/instance/4259928/bin/WJCO-5-1020-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4259928/" ref="ordinalpos=5024&amp;ncbi_uid=6945746&amp;link_uid=PMC4259928" image-link="/pmc/articles/PMC4259928/figure/F1/" class="imagepopup">Figure 1.  From: Estrogen receptors as the novel therapeutic biomarker in non-small cell lung cancer. </a></p><div class="supp"><p class="details"><div>Intracellular estrogen receptor and epidermal growth factor receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in non-small cell lung cancer. A: Mechanisms of ER and EGFR <span class="highlight" style="background-color:">signaling</span> in NSCLC. E2 (17β-estradiol)-bound ER acts in part as a transcription factor in the nucleus. Once the ER binds to the DNA in the estrogen response element of a gene, it generally recruits co-activator complexes to modulate gene transcription (genomic signal). In addition, E2-bound ER also acts in the cytoplasm (non-genomic signal) by interacting with downstream mediators of EGFR <span class="highlight" style="background-color:">signaling</span>, such as MAPK and PI3K/AKT. EGFR is a cell membrane receptor tyrosine kinase that transmits a signal when it binds EGF; B: Mechanisms of combination therapy for NSCLC. The estrogen antagonist fulv binds to ERs and blocks estrogen <span class="highlight" style="background-color:">signaling</span>. EGFR-TKIs, such as gefitinib or erlotinib, bind to mtEGFR and inhibit EGFR <span class="highlight" style="background-color:">signaling</span>. Both treatments block the interaction between ER and EGFR <span class="highlight" style="background-color:">signaling</span>. EGFR: Epidermal growth factor receptor; ER: Estrogen receptor; TKI: Tyrosine kinase inhibitor.</div></p></div><div class="aux"><div class="resc">Hideki Kawai. World J Clin Oncol. 2014 December 10;5(5):1020-1027.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=5024&amp;ncbi_uid=6945746&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4259928/" ref="ordinalpos=5024&amp;ncbi_uid=6945746&amp;link_uid=PMC4259928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5025.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4132145/figure/fig01/" ref="ordinalpos=5025&amp;ncbi_uid=6497864&amp;link_uid=PMC4132145" href="/pmc/articles/PMC4132145/"><img src="/pmc/articles/instance/4132145/bin/ijcem0007-1786-f1.gif" src-large="/pmc/articles/instance/4132145/bin/ijcem0007-1786-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4132145/" ref="ordinalpos=5025&amp;ncbi_uid=6497864&amp;link_uid=PMC4132145" image-link="/pmc/articles/PMC4132145/figure/fig01/" class="imagepopup">Figure 1.  From: Aberrant sonic hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and STAT3 activation in papillary thyroid cancer. </a></p><div class="supp"><p class="details"><div>Immunohistochemical staining of SHH, PTCH, SMO, GLI1, and p-STAT3. The SHH <span class="highlight" style="background-color:">signaling</span> molecules and p-STAT3 expression was analyzed by IHC staining with their specifi c antibodies in PTC. Aberrant expression of the SHH <span class="highlight" style="background-color:">signaling</span> molecules and p-STAT3 was detected in PTC and graded as negative (-), positive (+), strongly positive (++), and very strongly positive (+++) (400 X).</div></p></div><div class="aux"><div class="resc">Wenwu Dong, et al. Int J Clin Exp Med. 2014;7(7):1786-1793.</div><p class="links"><a class="dblinks" href="/pubmed/25126181" ref="ordinalpos=5025&amp;ncbi_uid=6497864&amp;link_uid=25126181">Citation</a><a class="dblinks" href="/pmc/articles/PMC4132145/" ref="ordinalpos=5025&amp;ncbi_uid=6497864&amp;link_uid=PMC4132145">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5026.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3023131/figure/fig7/" ref="ordinalpos=5026&amp;ncbi_uid=3083714&amp;link_uid=PMC3023131" href="/pmc/articles/PMC3023131/"><img src="/pmc/articles/instance/3023131/bin/JEM_20092203_RGB_Fig7.gif" src-large="/pmc/articles/instance/3023131/bin/JEM_20092203_RGB_Fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3023131/" ref="ordinalpos=5026&amp;ncbi_uid=3083714&amp;link_uid=PMC3023131" image-link="/pmc/articles/PMC3023131/figure/fig7/" class="imagepopup">Figure 7. From: CD83 increases MHC II and CD86 on dendritic cells by opposing IL-10-driven MARCH1-mediated ubiquitination and degradation. </a></p><div class="supp"><p class="details"><div><b>Summary of CD83 <span class="highlight" style="background-color:">pathway</span> regulating display of MHC II and CD86 on DCs.</b> CD83 mRNA is induced by TLR <span class="highlight" style="background-color:">signaling</span>, whereas MARCH1 mRNA is induced by IL-10 and diminished by TLR <span class="highlight" style="background-color:">signaling</span>. The CD83 TM binds to MARCH1, decreasing MARCH1 association with the TM segments of MHC II or CD86 and preventing ubiquitination of lysines in their cytoplasmic tails by the MARCH1 RING-CH domain, thereby promoting MHC II or CD86 surface display.</div></p></div><div class="aux"><div class="resc">Lina E. Tze, et al. J Exp Med. 2011 January 17;208(1):149-165.</div><p class="links"><a class="dblinks" href="/pubmed/21220452" ref="ordinalpos=5026&amp;ncbi_uid=3083714&amp;link_uid=21220452">Citation</a><a class="dblinks" href="/pmc/articles/PMC3023131/" ref="ordinalpos=5026&amp;ncbi_uid=3083714&amp;link_uid=PMC3023131">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5027.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2785619/figure/F2/" ref="ordinalpos=5027&amp;ncbi_uid=2528259&amp;link_uid=PMC2785619" href="/pmc/articles/PMC2785619/"><img src="/pmc/articles/instance/2785619/bin/zbc0420989230002.gif" src-large="/pmc/articles/instance/2785619/bin/zbc0420989230002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2785619/" ref="ordinalpos=5027&amp;ncbi_uid=2528259&amp;link_uid=PMC2785619" image-link="/pmc/articles/PMC2785619/figure/F2/" class="imagepopup">FIGURE 2. From: Simultaneous Transforming Growth Factor ?-Tumor Necrosis Factor Activation and Cross-talk Cause Aberrant Remodeling Response and Myocardial Fibrosis in Timp3-deficient Heart. </a></p><div class="supp"><p class="details"><div><b>Up-regulation of TGFβ <span class="highlight" style="background-color:">signaling</span> in <i>Timp3</i><sup>−/−</sup>-AB hearts.</b> Gene set enrichment analysis indicated TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to be significantly up-regulated at 6 h after aortic banding. This was visualized by <i>GenMAPP</i>. Genes labeled in <i>red</i> were up-regulated, whereas those labeled in <i>green</i> were down-regulated in <i>Timp3</i><sup>−/−</sup> hearts.</div></p></div><div class="aux"><div class="resc">Zamaneh Kassiri, et al. J Biol Chem. 2009 October 23;284(43):29893-29904.</div><p class="links"><a class="dblinks" href="/pubmed/19625257" ref="ordinalpos=5027&amp;ncbi_uid=2528259&amp;link_uid=19625257">Citation</a><a class="dblinks" href="/pmc/articles/PMC2785619/" ref="ordinalpos=5027&amp;ncbi_uid=2528259&amp;link_uid=PMC2785619">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5028.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3209051/figure/F9/" ref="ordinalpos=5028&amp;ncbi_uid=3480742&amp;link_uid=PMC3209051" href="/pmc/articles/PMC3209051/"><img src="/pmc/articles/instance/3209051/bin/zek9990938130009.gif" src-large="/pmc/articles/instance/3209051/bin/zek9990938130009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3209051/" ref="ordinalpos=5028&amp;ncbi_uid=3480742&amp;link_uid=PMC3209051" image-link="/pmc/articles/PMC3209051/figure/F9/" class="imagepopup">Fig. 9. From: The Casein Kinase I Protein Cck1 Regulates Multiple <span class="highlight" style="background-color:">Signaling</span> Pathways and Is Essential for Cell Integrity and Fungal Virulence in Cryptococcus neoformans. </a></p><div class="supp"><p class="details"><div>The proposed model for Cck1 function. In our model, Cck1 regulates multiple downstream pathways and is essential for fungal virulence. Cck1 is regulated by glucose. It may participate in the phosphorylation of Fbp1 substrates to regulate cell integrity and fungal virulence. In addition, Cck1 may also play a more direct role in regulating Mpk1 <span class="highlight" style="background-color:">signaling</span>, or it regulates Mpk1 function via its target protein(s) in the Fbp1 <span class="highlight" style="background-color:">pathway</span>. Meanwhile, Cck1 also responds to stress signals and plays a role in the <i>HOG</i> <span class="highlight" style="background-color:">pathway</span> by regulating the phosphorylation of Hog1 protein and thus controls the cell adaptation to osmotic shock. Cck1 also regulates fungal mating, but its potential target(s) remains to be identified.</div></p></div><div class="aux"><div class="resc">Yina Wang, et al. Eukaryot Cell. 2011 November;10(11):1455-1464.</div><p class="links"><a class="dblinks" href="/pubmed/21926330" ref="ordinalpos=5028&amp;ncbi_uid=3480742&amp;link_uid=21926330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3209051/" ref="ordinalpos=5028&amp;ncbi_uid=3480742&amp;link_uid=PMC3209051">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5029.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2908618/figure/pone-0011716-g007/" ref="ordinalpos=5029&amp;ncbi_uid=2377137&amp;link_uid=PMC2908618" href="/pmc/articles/PMC2908618/"><img src="/pmc/articles/instance/2908618/bin/pone.0011716.g007.gif" src-large="/pmc/articles/instance/2908618/bin/pone.0011716.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2908618/" ref="ordinalpos=5029&amp;ncbi_uid=2377137&amp;link_uid=PMC2908618" image-link="/pmc/articles/PMC2908618/figure/pone-0011716-g007/" class="imagepopup">Figure 7. Summary of CXCL12-mediated <span class="highlight" style="background-color:">Signaling</span> in CLL..  From: Elucidating the CXCL12/CXCR4 <span class="highlight" style="background-color:">Signaling</span> Network in Chronic Lymphocytic Leukemia through Phosphoproteomics Analysis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> diagram depicting pathways activated downstream of CXLC12. Through direct or indirect mechanisms, arrows indicate factors that are activated, red lines ending with a bar indicate factors that are inhibited by the upstream factor, and lines (no arrowhead) indicate interactions. Proteins in hexagons were identified and validated herein or were previously known targets also detected in the LC-MS/MS. Proteins in rectangles are known key <span class="highlight" style="background-color:">signaling</span> molecules of these pathways that were not detected in this LC-MS/MS data set. Proteins in ovals with dashed lines are likely intermediates/targets of the pathways based on previous studies. Proteins in oval shape were also identified by LC-MS/MS but have yet to be validated. Much of our focus has been on the PI3K/Akt and Raf/MEK/ERK pathways due to known implications in CLL cell survival and resistance to apoptosis. Furthermore, the potential involvement of the p38-MAPK <span class="highlight" style="background-color:">pathway</span> in some CLL patients with activation of HSP27 and LSP1 is outlined.</div></p></div><div class="aux"><div class="resc">Morgan O'Hayre, et al. PLoS One. 2010;5(7):e11716.</div><p class="links"><a class="dblinks" href="/pubmed/20661426" ref="ordinalpos=5029&amp;ncbi_uid=2377137&amp;link_uid=20661426">Citation</a><a class="dblinks" href="/pmc/articles/PMC2908618/" ref="ordinalpos=5029&amp;ncbi_uid=2377137&amp;link_uid=PMC2908618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5030.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC197303/figure/fig1/" ref="ordinalpos=5030&amp;ncbi_uid=244753&amp;link_uid=PMC197303" href="/pmc/articles/PMC197303/"><img src="/pmc/articles/instance/197303/bin/015529f1.gif" src-large="/pmc/articles/instance/197303/bin/015529f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC197303/" ref="ordinalpos=5030&amp;ncbi_uid=244753&amp;link_uid=PMC197303" image-link="/pmc/articles/PMC197303/figure/fig1/" class="imagepopup">Figure 1. From: Arabidopsis sfd Mutants Affect Plastidic Lipid Composition and Suppress Dwarfing, Cell Death, and the Enhanced Disease Resistance Phenotypes Resulting from the Deficiency of a Fatty Acid Desaturase. </a></p><div class="supp"><p class="details"><div>Impact of the <i>ssi2</i> Mutant on Defense <span class="highlight" style="background-color:">Signaling</span> in Arabidopsis.<br />SA and JA are important <span class="highlight" style="background-color:">signaling</span> molecules in plant defense. JA <span class="highlight" style="background-color:">signaling</span> is required for the pathogen-activated expression of <i>PDF1.2</i> and resistance to the necrotrophic pathogen <i>B. cinerea</i>. SA <span class="highlight" style="background-color:">signaling</span> is required for the pathogen-activated expression of the <i>PR</i> genes and resistance to <i>Psm</i>. The <i>EDS5</i> and <i>SID2</i> genes are required for SA synthesis; the loss-of-function <i>eds5</i> and <i>sid2</i> mutations block SA synthesis. SA <span class="highlight" style="background-color:">signaling</span> is activated via both NPR1-dependent and -independent mechanisms. In addition to SA, an unknown pathogen-activated factor is required for <span class="highlight" style="background-color:">signaling</span> through the NPR1-independent <span class="highlight" style="background-color:">pathway</span>. The loss of the SSI2-encoded stearoyl-ACP desaturase activity in the <i>ssi2</i> mutant has pleiotropic effects on plant defense responses. The <i>ssi2</i> mutant allele promotes (+) the spontaneous development of lesions containing dead cells, the accumulation of increased SA levels, and the constitutive expression of NPR1-dependent and -independent defense mechanisms, which confer high-level expression of <i>PR</i> genes and enhanced resistance to <i>Psm</i>. By contrast, the <i>ssi2</i> mutant interferes with (−) the ability of JA/MeJA to activate <i>PDF1.2</i> expression and exhibits enhanced susceptibility to <i>B. cinerea</i>. 18:1 application restores the JA-inducible expression of <i>PDF1.2</i> in the <i>ssi2</i> mutant plant (Kachroo et al., 2001), suggesting a role for an 18:1-derived factor, which is limiting in the <i>ssi2</i> mutant plant, in promoting JA <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Ashis Nandi, et al. Plant Cell. 2003 October;15(10):2383-2398.</div><p class="links"><a class="dblinks" href="/pubmed/14507997" ref="ordinalpos=5030&amp;ncbi_uid=244753&amp;link_uid=14507997">Citation</a><a class="dblinks" href="/pmc/articles/PMC197303/" ref="ordinalpos=5030&amp;ncbi_uid=244753&amp;link_uid=PMC197303">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5031.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847485/figure/F5/" ref="ordinalpos=5031&amp;ncbi_uid=5574679&amp;link_uid=PMC3847485" href="/pmc/articles/PMC3847485/"><img src="/pmc/articles/instance/3847485/bin/1475-2867-13-90-5.gif" src-large="/pmc/articles/instance/3847485/bin/1475-2867-13-90-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847485/" ref="ordinalpos=5031&amp;ncbi_uid=5574679&amp;link_uid=PMC3847485" image-link="/pmc/articles/PMC3847485/figure/F5/" class="imagepopup">Figure 5.  From: TLR9 <span class="highlight" style="background-color:">signaling</span> repressed tumor suppressor miR-7 expression through up-regulation of HuR in human lung cancer cells. </a></p><div class="supp"><p class="details"><div><b>TLR9 <span class="highlight" style="background-color:">signaling</span> induced the expression of HuR through Akt <span class="highlight" style="background-color:">pathway</span>.</b> 95D cells were treated with 50 μM PI3K inhibitor (LY294002) or specific MEK (Mitogen-activated protein kinase) inhibitor (U0126), 2.5 μM Akt inhibitor IV respectively, and then cultured in 24-well plate in the presence of 10 ug/ml CpG ODNs. 72 hrs later, the expression of HuR protein was determined by western blot and calculated <b>(A)</b>. <b>(B)</b> Schematic diagram for TLR9 <span class="highlight" style="background-color:">signaling</span> induced expression of HuR.</div></p></div><div class="aux"><div class="resc">Yong-Ju Li, et al. Cancer Cell Int. 2013;13:90-90.</div><p class="links"><a class="dblinks" href="/pubmed/24004462" ref="ordinalpos=5031&amp;ncbi_uid=5574679&amp;link_uid=24004462">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847485/" ref="ordinalpos=5031&amp;ncbi_uid=5574679&amp;link_uid=PMC3847485">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5032.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2947752/figure/F7/" ref="ordinalpos=5032&amp;ncbi_uid=3284332&amp;link_uid=PMC2947752" href="/pmc/articles/PMC2947752/"><img src="/pmc/articles/instance/2947752/bin/DEV050864F7.gif" src-large="/pmc/articles/instance/2947752/bin/DEV050864F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2947752/" ref="ordinalpos=5032&amp;ncbi_uid=3284332&amp;link_uid=PMC2947752" image-link="/pmc/articles/PMC2947752/figure/F7/" class="imagepopup">Fig. 7. From: A role for PDGF <span class="highlight" style="background-color:">signaling</span> in expansion of the extra-embryonic endoderm lineage of the mouse blastocyst. </a></p><div class="supp"><p class="details"><div><b>Model for the role of PDGF <span class="highlight" style="background-color:">signaling</span> in the PrE lineage of the mouse blastocyst.</b> Early mouse development is characterized by progressive lineage restriction ensuring the segregation of the ICM cells (gray) into epiblast (EPI, red) and primitive endoderm (PrE, blue). XEN cells are derived from the PrE of the blastocyst. GATA6, which is considered a key regulator of PrE identity, may control the expression of genes, including <i>Gata4</i> and <i>Pdgfra</i>. The PDGF <span class="highlight" style="background-color:">pathway</span> exerts a mitogenic effect through MEK/PKC <span class="highlight" style="background-color:">signaling</span> and promotes in vivo and ex vivo PrE lineage expansion and XEN cell establishment.</div></p></div><div class="aux"><div class="resc">Jérôme Artus, et al. Development. 2010 October 15;137(20):3361-3372.</div><p class="links"><a class="dblinks" href="/pubmed/20826533" ref="ordinalpos=5032&amp;ncbi_uid=3284332&amp;link_uid=20826533">Citation</a><a class="dblinks" href="/pmc/articles/PMC2947752/" ref="ordinalpos=5032&amp;ncbi_uid=3284332&amp;link_uid=PMC2947752">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5033.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2997007/figure/F6/" ref="ordinalpos=5033&amp;ncbi_uid=2620896&amp;link_uid=PMC2997007" href="/pmc/articles/PMC2997007/"><img src="/pmc/articles/instance/2997007/bin/1471-2164-11-513-6.gif" src-large="/pmc/articles/instance/2997007/bin/1471-2164-11-513-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2997007/" ref="ordinalpos=5033&amp;ncbi_uid=2620896&amp;link_uid=PMC2997007" image-link="/pmc/articles/PMC2997007/figure/F6/" class="imagepopup">Figure 6.  From: Differential functional genomic effects of anti-inflammatory phytocompounds on immune <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Identification of putative <span class="highlight" style="background-color:">signaling</span> regulators for immune-modification</b>. Key node analysis in the TRANSPATH database for putative unique <span class="highlight" style="background-color:">signaling</span> master regulator/molecule at 0.5 h with significant inhibition of expression (≥3-fold decrease) in LPS-induced THP-1 cells co-treated with <b>(A) </b>shikonin, <b>(B) </b>emodin, or <b>(C) </b>BF/S+L/Ep at 12 h. Less than four hierarchical levels of regulation were accepted and selected. Pink circle: the putative master regulator/molecule; green box: the ratio of gene expression seen with the phytocompounds plus LPS compared to LPS only; gray circle: assigned molecule in the putative <span class="highlight" style="background-color:">pathway</span>; <b>↑</b>: activation; <b>⊥</b>: inhibition.</div></p></div><div class="aux"><div class="resc">Shao-Chih Chiu, et al. BMC Genomics. 2010;11:513-513.</div><p class="links"><a class="dblinks" href="/pubmed/20868472" ref="ordinalpos=5033&amp;ncbi_uid=2620896&amp;link_uid=20868472">Citation</a><a class="dblinks" href="/pmc/articles/PMC2997007/" ref="ordinalpos=5033&amp;ncbi_uid=2620896&amp;link_uid=PMC2997007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5034.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2901811/figure/F4/" ref="ordinalpos=5034&amp;ncbi_uid=2495784&amp;link_uid=PMC2901811" href="/pmc/articles/PMC2901811/"><img src="/pmc/articles/instance/2901811/bin/org0602_0076_fig004.gif" src-large="/pmc/articles/instance/2901811/bin/org0602_0076_fig004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2901811/" ref="ordinalpos=5034&amp;ncbi_uid=2495784&amp;link_uid=PMC2901811" image-link="/pmc/articles/PMC2901811/figure/F4/" class="imagepopup">Figure 4.  From: The regulation of organ size in Drosophila. </a></p><div class="supp"><p class="details"><div>The II S and TOR-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Drosophila. The II S <span class="highlight" style="background-color:">pathway</span> is shown in cyan, the TOR-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown in magenta, and the AMPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown in green. Variation in nutrition influences the release of dILPs, possibly by the action of sNPFs and AKH. dILPs bind to Inr which initiates a signal transduction cascade involving the phosphorylation of multiple intermediate proteins. Downstream growth affectors include dFOXO, which is deactivated by II S via phosphorylation by AKT, and S6K, which is activated by II S via PDK1. S6K is also a target of TOR, which also restricts the effects of dFOXO by inhibiting one of dFOXO’s transcriptional targets, 4EBP. TOR is also regulated indirectly by II S via the action of AKT on TSC1/2. TOR also responds to amino acids, by an unkown mehchanism, and glucose, via the AMPK <span class="highlight" style="background-color:">pathway</span>. Both FOXO and TOR regulate the activity of multiple growth inhibitors and promoters, respectively. Data from references 37, 43 and 50. Dotted lines are putative relationships.</div></p></div><div class="aux"><div class="resc">Alexander W Shingleton. Organogenesis. 2010 Apr-Jun;6(2):76-87.</div><p class="links"><a class="dblinks" href="/pubmed/20885854" ref="ordinalpos=5034&amp;ncbi_uid=2495784&amp;link_uid=20885854">Citation</a><a class="dblinks" href="/pmc/articles/PMC2901811/" ref="ordinalpos=5034&amp;ncbi_uid=2495784&amp;link_uid=PMC2901811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5035.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1868774/figure/pcbi-0030092-g003/" ref="ordinalpos=5035&amp;ncbi_uid=927859&amp;link_uid=PMC1868774" href="/pmc/articles/PMC1868774/"><img src="/pmc/articles/instance/1876485/bin/pcbi.0030092.g003.gif" src-large="/pmc/articles/instance/1876485/bin/pcbi.0030092.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1868774/" ref="ordinalpos=5035&amp;ncbi_uid=927859&amp;link_uid=PMC1868774" image-link="/pmc/articles/PMC1868774/figure/pcbi-0030092-g003/" class="imagepopup">Figure 3. Conceptual Model for the <span class="highlight" style="background-color:">Signaling</span> Events Underlying VPC Fate Specification.  From: Predictive Modeling of <span class="highlight" style="background-color:">Signaling</span> Crosstalk during C. elegans Vulval Development. </a></p><div class="supp"><p class="details"><div>The thickness of the inductive signal (IS) arrows indicates the relative levels received by the three VPCs shown. With a low level of IS (the rightmost cell), the EGFR/RAS/MAPK <span class="highlight" style="background-color:">pathway</span> is below the threshold needed for induction (indicated in grey), and the VPC adopts the 3° fate. A high level of IS (the leftmost cell) activates the EGFR/RAS/MAPK <span class="highlight" style="background-color:">pathway</span> and induces the 1° fate. High IS also results in the production of a strong lateral signal (LS) by this VPC, the downregulation of LIN-12, and, as predicted by our computational model (see below), in the inhibition of the <i>lst</i> genes as indicated by the red line. The middle VPC receives a medium IS; however, the EGFR/RAS/MAPK <span class="highlight" style="background-color:">pathway</span> is counteracted (indicated in grey) by the <i>lst</i> genes activated by the strong LS from the leftmost VPC. The middle VPC thus adopts the 2° fate.</div></p></div><div class="aux"><div class="resc">Jasmin Fisher, et al. PLoS Comput Biol. 2007 May;3(5):e92.</div><p class="links"><a class="dblinks" href="/pubmed/17511512" ref="ordinalpos=5035&amp;ncbi_uid=927859&amp;link_uid=17511512">Citation</a><a class="dblinks" href="/pmc/articles/PMC1868774/" ref="ordinalpos=5035&amp;ncbi_uid=927859&amp;link_uid=PMC1868774">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5036.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3702529/figure/F3/" ref="ordinalpos=5036&amp;ncbi_uid=5067514&amp;link_uid=PMC3702529" href="/pmc/articles/PMC3702529/"><img src="/pmc/articles/instance/3702529/bin/1471-2407-13-327-3.gif" src-large="/pmc/articles/instance/3702529/bin/1471-2407-13-327-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3702529/" ref="ordinalpos=5036&amp;ncbi_uid=5067514&amp;link_uid=PMC3702529" image-link="/pmc/articles/PMC3702529/figure/F3/" class="imagepopup">Figure 3.  From: Activation of AMPK inhibits cervical cancer cell growth through AKT/FOXO3a/FOXM1 <span class="highlight" style="background-color:">signaling</span> cascade. </a></p><div class="supp"><p class="details"><div><b>Activation of AMPK represses FOXM1 expression through inhibiting the AKT/FOXO3a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> Upon treatment of metformin (25 mM) in cervical cancer cells (C33A), the expression of FOXM1 diminished remarkably and the phosphorylation of AKT and AKT-specific phosphorylation of FOXO3a (Ser253) decreased concomitantly. <b>(B)</b> Co-treatment of PI3K/AKT inhibitor, LY294002 (10 μM), inhibited p-AKT and concomitantly reduced the levels of FOXO3a (Ser253) and FOXM1 in C33A cells. <b>(C)</b> Depletion of endogenous FOXO3a in C33A cells using siRNAs showed that FOXO3a expression is required for the suppression of FOXM1 by metformin (25 mM) treatment.</div></p></div><div class="aux"><div class="resc">Mingo Ming Ho Yung, et al. BMC Cancer. 2013;13:327-327.</div><p class="links"><a class="dblinks" href="/pubmed/23819460" ref="ordinalpos=5036&amp;ncbi_uid=5067514&amp;link_uid=23819460">Citation</a><a class="dblinks" href="/pmc/articles/PMC3702529/" ref="ordinalpos=5036&amp;ncbi_uid=5067514&amp;link_uid=PMC3702529">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5037.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634610/figure/F3/" ref="ordinalpos=5037&amp;ncbi_uid=4827896&amp;link_uid=PMC3634610" href="/pmc/articles/PMC3634610/"><img src="/pmc/articles/instance/3634610/bin/IRI-32-134-g003.gif" src-large="/pmc/articles/instance/3634610/bin/IRI-32-134-g003.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634610/" ref="ordinalpos=5037&amp;ncbi_uid=4827896&amp;link_uid=PMC3634610" image-link="/pmc/articles/PMC3634610/figure/F3/" class="imagepopup">Figure 3. From: C-Type Lectin-Like Receptors of the Dectin-1 Cluster: Ligands and <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div> <b>ITAM-like-bearing CTLRs within the Dectin-1 cluster.</b> As well as Dectin-1, there are two additional receptors that contain ITAM-like motifs. These receptors share Syk as a primary downstream <span class="highlight" style="background-color:">signaling</span> kinase; however, the lack of a tri-acidic in CLEC-9A and the presence of an isoleucine prevent the endocytic and activation functions of this receptor. CLEC-2 utilises its tri-acidic motif to complement activation through Syk resulting in adhesion and clot formation in platelets. Dashed arrows represent a <span class="highlight" style="background-color:">pathway</span> that has yet to be fully defined.</div></p></div><div class="aux"><div class="resc">Anthony Plato, et al. Int Rev Immunol. 2013 April;32(2):134-156.</div><p class="links"><a class="dblinks" href="/pubmed/23570314" ref="ordinalpos=5037&amp;ncbi_uid=4827896&amp;link_uid=23570314">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634610/" ref="ordinalpos=5037&amp;ncbi_uid=4827896&amp;link_uid=PMC3634610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5038.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3652399/figure/F1/" ref="ordinalpos=5038&amp;ncbi_uid=4892890&amp;link_uid=PMC3652399" href="/pmc/articles/PMC3652399/"><img src="/pmc/articles/instance/3652399/bin/nihms-456111-f0001.gif" src-large="/pmc/articles/instance/3652399/bin/nihms-456111-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3652399/" ref="ordinalpos=5038&amp;ncbi_uid=4892890&amp;link_uid=PMC3652399" image-link="/pmc/articles/PMC3652399/figure/F1/" class="imagepopup">Figure 1.  From: The transcriptome that mediates increased cyclic adenosine monophosphate <span class="highlight" style="background-color:">signaling</span> in PRKAR1A defects and other settings. </a></p><div class="supp"><p class="details"><div>The cyclic adenosine monophosphate (cAMP) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its genetic defects causing endocrine diseases. ATP = adeno-sine triphosphate; C = catalytic subunit of PKA; CNC = Carney complex; GDP = guanosine diphosphate; GTP = guanosine triphosphate; MAS = McCune-Albright syndrome; PDE = phosphodiesterase; PKA = protein kinase A; R = regulatory subunit of PKA; a, b, g = subunits of G protein.</div></p></div><div class="aux"><div class="resc">Monalisa F Azevedo, et al. Endocr Pract. 2011 Jul-Aug;17(0 3):2-7.</div><p class="links"><a class="dblinks" href="/pubmed/21454229" ref="ordinalpos=5038&amp;ncbi_uid=4892890&amp;link_uid=21454229">Citation</a><a class="dblinks" href="/pmc/articles/PMC3652399/" ref="ordinalpos=5038&amp;ncbi_uid=4892890&amp;link_uid=PMC3652399">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5039.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2834552/figure/F6/" ref="ordinalpos=5039&amp;ncbi_uid=2737563&amp;link_uid=PMC2834552" href="/pmc/articles/PMC2834552/"><img src="/pmc/articles/instance/2834552/bin/asn0021042730006.gif" src-large="/pmc/articles/instance/2834552/bin/asn0021042730006.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2834552/" ref="ordinalpos=5039&amp;ncbi_uid=2737563&amp;link_uid=PMC2834552" image-link="/pmc/articles/PMC2834552/figure/F6/" class="imagepopup">Figure 6. From: Advanced Glycation End-Products Induce Tubular CTGF via TGF-?-Independent Smad3 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>AGEs induce an early phosphorylation of ERK1/2 and p38 MAPK <i>via</i> the TGF-β–independent Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) ERK1/2 phosphorylation. (B) p38 phosphorylation. Western blots demonstrate that AGEs (33 μg/ml) but not control BSA (33 μg/ml) induce phosphorylation of ERK1/2 and p38 in TGF-β1 WT TECs with two peaks (30 min and 18 h). In contrast, in TGF-β1 KO TECs, AGEs induce only an early phosphorylation of ERK1/2 and p38. Data are means ± SEM for at least 3 independent experiments.</div></p></div><div class="aux"><div class="resc">Arthur C.K. Chung, et al. J Am Soc Nephrol. 2010 February;21(2):249-260.</div><p class="links"><a class="dblinks" href="/pubmed/19959709" ref="ordinalpos=5039&amp;ncbi_uid=2737563&amp;link_uid=19959709">Citation</a><a class="dblinks" href="/pmc/articles/PMC2834552/" ref="ordinalpos=5039&amp;ncbi_uid=2737563&amp;link_uid=PMC2834552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5040.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3858370/figure/pbio-1001733-g008/" ref="ordinalpos=5040&amp;ncbi_uid=5595123&amp;link_uid=PMC3858370" href="/pmc/articles/PMC3858370/"><img src="/pmc/articles/instance/3858370/bin/pbio.1001733.g008.gif" src-large="/pmc/articles/instance/3858370/bin/pbio.1001733.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3858370/" ref="ordinalpos=5040&amp;ncbi_uid=5595123&amp;link_uid=PMC3858370" image-link="/pmc/articles/PMC3858370/figure/pbio-1001733-g008/" class="imagepopup">Figure 8. Retrograde <span class="highlight" style="background-color:">signaling</span> impacts period determination..  From: Circadian Period Integrates Network Information Through Activation of the BMP <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The figure depicts a model in which different ligands released from sLNv postsynaptic targets (TIM+PDF− as well as the lLNvs) modulate the activity of the BMP <span class="highlight" style="background-color:">signaling</span> cascade in the sLNvs acting through the TKV/SAX/WIT receptors. In turn, this signal results in a transcriptionally active complex (including <i>schnurri</i>) that down-regulates <i>Clk</i> expression. Reduced <i>Clock</i> levels would then give rise to decreased <i>per</i> transcription and therefore reduced PER levels, thus slowing the pace of the molecular oscillator.</div></p></div><div class="aux"><div class="resc">Esteban J. Beckwith, et al. PLoS Biol. 2013 December;11(12):e1001733.</div><p class="links"><a class="dblinks" href="/pubmed/24339749" ref="ordinalpos=5040&amp;ncbi_uid=5595123&amp;link_uid=24339749">Citation</a><a class="dblinks" href="/pmc/articles/PMC3858370/" ref="ordinalpos=5040&amp;ncbi_uid=5595123&amp;link_uid=PMC3858370">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5041.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4009148/figure/fig2/" ref="ordinalpos=5041&amp;ncbi_uid=6130783&amp;link_uid=PMC4009148" href="/pmc/articles/PMC4009148/"><img src="/pmc/articles/instance/4009148/bin/JIR2014-602846.002.gif" src-large="/pmc/articles/instance/4009148/bin/JIR2014-602846.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4009148/" ref="ordinalpos=5041&amp;ncbi_uid=6130783&amp;link_uid=PMC4009148" image-link="/pmc/articles/PMC4009148/figure/fig2/" class="imagepopup">Figure 2.  From: Potential Involvement of IL-17F in Asthma. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways induced by IL-17F. IL-17F utilizes a heterodimer of IL-17RA and IL-17RC as its receptor, and then IL-17RA engages the SEFIR domain-containing Act-1. Act-1 is required for recruitment of TRAF6, leading to the activation of TAK1. The Raf1-MEK1/2-ERK1/2-p90RSK/MSK1-CREB is the pivotal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. On the other hand, IL-17F also activates transcriptional factors such as C/EBP<i>β</i>, C/EBP<i>γ</i>, and NF-<i>κ</i>B. Activation of these pathways leads to the expression of various inflammatory molecules.</div></p></div><div class="aux"><div class="resc">Kyoko Ota, et al. J Immunol Res. 2014;2014:602846.</div><p class="links"><a class="dblinks" href="/pubmed/24829928" ref="ordinalpos=5041&amp;ncbi_uid=6130783&amp;link_uid=24829928">Citation</a><a class="dblinks" href="/pmc/articles/PMC4009148/" ref="ordinalpos=5041&amp;ncbi_uid=6130783&amp;link_uid=PMC4009148">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5042.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1091762/figure/fig8/" ref="ordinalpos=5042&amp;ncbi_uid=425962&amp;link_uid=PMC1091762" href="/pmc/articles/PMC1091762/"><img src="/pmc/articles/instance/1091762/bin/tpc1701387f08.gif" src-large="/pmc/articles/instance/1091762/bin/tpc1701387f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1091762/" ref="ordinalpos=5042&amp;ncbi_uid=425962&amp;link_uid=PMC1091762" image-link="/pmc/articles/PMC1091762/figure/fig8/" class="imagepopup">Figure 8. From: Crown rootless1, Which Is Essential for Crown Root Formation in Rice, Is a Target of an AUXIN RESPONSE FACTOR in Auxin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Auxin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Leading to <i>Crl1</i> Expression in Root Formation.<br />AUX/IAAs function as negative regulators in auxin <span class="highlight" style="background-color:">signaling</span> to prevent the function of ARFs by direct interaction. Auxin treatment promotes the degradation of AUX/IAAs to release ARF. The released ARFs interact with the AuxRE in the <i>Crl1</i> promoter to trigger its transcription, resulting in root initiation.</div></p></div><div class="aux"><div class="resc">Yoshiaki Inukai, et al. Plant Cell. 2005 May;17(5):1387-1396.</div><p class="links"><a class="dblinks" href="/pubmed/15829602" ref="ordinalpos=5042&amp;ncbi_uid=425962&amp;link_uid=15829602">Citation</a><a class="dblinks" href="/pmc/articles/PMC1091762/" ref="ordinalpos=5042&amp;ncbi_uid=425962&amp;link_uid=PMC1091762">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5043.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3632418/figure/F1/" ref="ordinalpos=5043&amp;ncbi_uid=5049783&amp;link_uid=PMC3632418" href="/pmc/articles/PMC3632418/"><img src="/pmc/articles/instance/3632418/bin/nihms432411f1.gif" src-large="/pmc/articles/instance/3632418/bin/nihms432411f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3632418/" ref="ordinalpos=5043&amp;ncbi_uid=5049783&amp;link_uid=PMC3632418" image-link="/pmc/articles/PMC3632418/figure/F1/" class="imagepopup">Figure 1. Regulation of the mammalian target of rapamycin <span class="highlight" style="background-color:">pathway</span>.  From: mTOR as a potential treatment target for epilepsy. </a></p><div class="supp"><p class="details"><div>The mTOR is a serine–threonine protein kinase, which forms two complexes, rapamycin-sensitive mTORC1 and relatively rapamycin-insensitive mTORC2 (not shown). mTORC1 stimulates downstream <span class="highlight" style="background-color:">signaling</span> mechanisms involved in ribosomal biogenesis (S6K and ribosomal S6) and protein translation (4E-BP1 and eIF4E). Primarily through regulation of protein synthesis, the mTOR <span class="highlight" style="background-color:">pathway</span> modulates a variety of important physiological functions, such as cell growth and proliferation, metabolism and brain-specific processes. By similar mechanisms, the mTOR <span class="highlight" style="background-color:">pathway</span> may influence epileptogenesis under pathological conditions (e.g., synaptic reorganization, ion channel expression and neuronal death). In turn, the mTOR <span class="highlight" style="background-color:">pathway</span> may be activated or inhibited by various physiological or pathological stimuli via various upstream <span class="highlight" style="background-color:">signaling</span> pathways and intermediary proteins (TSC1, TSC2 and Rheb). Genetic mutations in different elements of these upstream pathways (<i>TSC1</i>, <i>TSC2</i>, <i>PTEN</i> and <i>STRAD</i>α) have been identified that cause hyperactivation of the mTOR <span class="highlight" style="background-color:">pathway</span> and may lead to various disease phenotypes, including epilepsy. Adapted with permission from [58].</div></p></div><div class="aux"><div class="resc">Michael Wong. Future Neurol. 2012 September 1;7(5):537-545.</div><p class="links"><a class="dblinks" href="/pubmed/23620711" ref="ordinalpos=5043&amp;ncbi_uid=5049783&amp;link_uid=23620711">Citation</a><a class="dblinks" href="/pmc/articles/PMC3632418/" ref="ordinalpos=5043&amp;ncbi_uid=5049783&amp;link_uid=PMC3632418">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5044.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2975372/figure/F5/" ref="ordinalpos=5044&amp;ncbi_uid=2567450&amp;link_uid=PMC2975372" href="/pmc/articles/PMC2975372/"><img src="/pmc/articles/instance/2975372/bin/nihms246517f5.gif" src-large="/pmc/articles/instance/2975372/bin/nihms246517f5.jpg" alt="FIGURE 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2975372/" ref="ordinalpos=5044&amp;ncbi_uid=2567450&amp;link_uid=PMC2975372" image-link="/pmc/articles/PMC2975372/figure/F5/" class="imagepopup">FIGURE 5. Engineering signal processing circuits.  From: Designing Customized Cell <span class="highlight" style="background-color:">Signaling</span> Circuits. </a></p><div class="supp"><p class="details"><div><b><i>a </i>|<i> engineered allosteric protein switches</i></b>. Dueber et al [49,51] showed that the allosterically regulation of the <span class="highlight" style="background-color:">signaling</span> protein N-WASP could be reprogrammed by recombining the catalytic domain from N-WASP with different combinations of interaction domains. Novel behaviors included multi-input (AND-gate) control and highly cooperative switch-like activation.<br /><b><i>b </i>|<i> using scaffold proteins as a molecular circuitboard for reshaping <span class="highlight" style="background-color:">signaling</span> output</i></b>. The input/output linkage of a MAP kinase <span class="highlight" style="background-color:">pathway</span> in yeast could be redirected via an engineered chimeric scaffold that assembled a novel combination of kinases [58]. Novel interaction sites can also be appended on to scaffolds to recruit additional modulatory factors. These additional factors can be build synthetic feedback loops that can be used to generate pathways that display diverse <span class="highlight" style="background-color:">signaling</span> dynamics [62].</div></p></div><div class="aux"><div class="resc">Wendell A. Lim. Nat Rev Mol Cell Biol. ;11(6):393-403.</div><p class="links"><a class="dblinks" href="/pubmed/20485291" ref="ordinalpos=5044&amp;ncbi_uid=2567450&amp;link_uid=20485291">Citation</a><a class="dblinks" href="/pmc/articles/PMC2975372/" ref="ordinalpos=5044&amp;ncbi_uid=2567450&amp;link_uid=PMC2975372">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5045.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889683/figure/Fig3/" ref="ordinalpos=5045&amp;ncbi_uid=5704360&amp;link_uid=PMC3889683" href="/pmc/articles/PMC3889683/"><img src="/pmc/articles/instance/3889683/bin/432_2013_1494_Fig3_HTML.gif" src-large="/pmc/articles/instance/3889683/bin/432_2013_1494_Fig3_HTML.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889683/" ref="ordinalpos=5045&amp;ncbi_uid=5704360&amp;link_uid=PMC3889683" image-link="/pmc/articles/PMC3889683/figure/Fig3/" class="imagepopup">Fig. 3.  From: Breast cancer adaptive resistance: HER2 and cancer stem cell repopulation in a heterogeneous tumor society. </a></p><div class="supp"><p class="details"><div>Schematic presentation of potential <span class="highlight" style="background-color:">signaling</span> network of HER2-NF-κB-PI3 K-Akt-STAT3. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is assumed based on the current literature, which, we believe, is activated by cellular adaptive response under anti-cancer chemo- and/or radio-therapy, and contributes to an adaptive resistance in cancer cells. Note that, the HER2-NF-κB-PI3 K-Akt-STAT3 <span class="highlight" style="background-color:">pathway</span> proposed here may be activated not only in the HER2-positive breast cancer but also activated in the HER2-negative cancer since <i>HER2</i> transcription is shown to be induced by radiation and DNA-damaging anti-cancer modalities without <i>HER2</i> gene copy enhancement</div></p></div><div class="aux"><div class="resc">Nadire Duru, et al. J Cancer Res Clin Oncol. 2014;140(1):1-14.</div><p class="links"><a class="dblinks" href="/pubmed/23990015" ref="ordinalpos=5045&amp;ncbi_uid=5704360&amp;link_uid=23990015">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889683/" ref="ordinalpos=5045&amp;ncbi_uid=5704360&amp;link_uid=PMC3889683">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5046.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3593803/figure/F1/" ref="ordinalpos=5046&amp;ncbi_uid=5856454&amp;link_uid=PMC3593803" href="/pmc/articles/PMC3593803/"><img src="/pmc/articles/instance/3593803/bin/nihms434906f1.gif" src-large="/pmc/articles/instance/3593803/bin/nihms434906f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3593803/" ref="ordinalpos=5046&amp;ncbi_uid=5856454&amp;link_uid=PMC3593803" image-link="/pmc/articles/PMC3593803/figure/F1/" class="imagepopup">Figure 1.  From: The Endothelium in Diabetes: its role in insulin access and diabetic complications. </a></p><div class="supp"><p class="details"><div>Insulin receptors in endothelial cells co-localize with cavaolae, and insulin <span class="highlight" style="background-color:">signaling</span> in endothelial cells leads to two downstream <span class="highlight" style="background-color:">signaling</span> pathways. Activation of the ERK <span class="highlight" style="background-color:">pathway</span> leads to ET-1 release, causing vasoconstriction in vascular smooth muscle cells, and the PI3K <span class="highlight" style="background-color:">pathway</span> leads to NO release and vasodilation. Studies have also shown that insulin binding to the insulin receptor is required for transcytosis of cavaeloe, possibly for translocation of insulin from the plasma to the interstitial space. (Image created in powerpoint)</div></p></div><div class="aux"><div class="resc">Cathryn M Kolka, et al. Rev Endocr Metab Disord. 2013 March;14(1):13-19.</div><p class="links"><a class="dblinks" href="/pubmed/23306780" ref="ordinalpos=5046&amp;ncbi_uid=5856454&amp;link_uid=23306780">Citation</a><a class="dblinks" href="/pmc/articles/PMC3593803/" ref="ordinalpos=5046&amp;ncbi_uid=5856454&amp;link_uid=PMC3593803">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5047.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2629187/figure/F6/" ref="ordinalpos=5047&amp;ncbi_uid=1646130&amp;link_uid=PMC2629187" href="/pmc/articles/PMC2629187/"><img src="/pmc/articles/instance/2629187/bin/zpq9990860690006.gif" src-large="/pmc/articles/instance/2629187/bin/zpq9990860690006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2629187/" ref="ordinalpos=5047&amp;ncbi_uid=1646130&amp;link_uid=PMC2629187" image-link="/pmc/articles/PMC2629187/figure/F6/" class="imagepopup">Fig. 6. From: Structurally diverse natural products that cause potassium leakage trigger multicellularity in Bacillus subtilis. </a></p><div class="supp"><p class="details"><div>Reconstitution of the signal transduction <span class="highlight" style="background-color:">pathway</span> in the heterologous host <i>L. monocytogenes</i>. (<i>A</i>) Activation of the <span class="highlight" style="background-color:">signaling</span> system is monitored by expression of the reporter gene P<i><sub>skf</sub>-cfp</i>. (<i>B</i>) Surfactin induces expression of P<i><sub>skf</sub>-cfp</i>, and this induction is diminished by adding potassium in the medium. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> activation requires the KinC PAS domain. Images are transmitted light (gray), fluorescence (white), and overlaid pictures, with fluorescence colored blue. (Scale bar: 3 μm.)</div></p></div><div class="aux"><div class="resc">Daniel López, et al. Proc Natl Acad Sci U S A. 2009 January 6;106(1):280-285.</div><p class="links"><a class="dblinks" href="/pubmed/19114652" ref="ordinalpos=5047&amp;ncbi_uid=1646130&amp;link_uid=19114652">Citation</a><a class="dblinks" href="/pmc/articles/PMC2629187/" ref="ordinalpos=5047&amp;ncbi_uid=1646130&amp;link_uid=PMC2629187">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5048.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3530847/figure/F7/" ref="ordinalpos=5048&amp;ncbi_uid=4467474&amp;link_uid=PMC3530847" href="/pmc/articles/PMC3530847/"><img src="/pmc/articles/instance/3530847/bin/nihms427843f7.gif" src-large="/pmc/articles/instance/3530847/bin/nihms427843f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3530847/" ref="ordinalpos=5048&amp;ncbi_uid=4467474&amp;link_uid=PMC3530847" image-link="/pmc/articles/PMC3530847/figure/F7/" class="imagepopup">Figure 7.  From: <span class="highlight" style="background-color:">PATHWAY</span> CROSS-TALK BETWEEN RAS/RAF AND PI3K IN PROMOTION OF M-CSF-INDUCED MEK/ERK-MEDIATED OSTEOCLAST SURVIVAL. </a></p><div class="supp"><p class="details"><div>Blocking Ras- and PI3K-Promoted MEK/ERK Activation. (A) Inhibition of downstream <span class="highlight" style="background-color:">signaling</span> components blocks Ras-mediated <span class="highlight" style="background-color:">pathway</span> activation. Osteoclasts were infected with the caRas adenovirus of vector control. Following infection, osteoclasts were serum starved and treated with M-CSF in the presence of the PI3K inhibitor LY294002, the Raf inhibitor GW5074 or DMSO vehicle control. Western blotting for phospho-MEK1/2Ser217/221, total-MEK1/2, phospho-ERK1/2Thr202/Tyr204, total-ERK1/2, phospho-RafSer338, phospho-RafSer259, and tubulin was performed. (B) Inhibition of Raf does not block PI3K-mediated <span class="highlight" style="background-color:">pathway</span> activation. Osteoclasts were infected with the caPI3K, dnRaf, dnRas, caPI3K and dnRaf, caPI3K and dnRas or vector control adenoviruses. Following infection, osteoclasts were serum starved and treated with M-CSF as indicated. Levels of phospho-MEK1/2Ser217/221, total-MEK1/2, phospho-ERK1/2Thr202/Tyr204, total-ERK1/2, phospho-RafSer338, phospho-RafSer259, and tubulin were then assessed via western blotting.</div></p></div><div class="aux"><div class="resc">Elizabeth W. Bradley, et al. J Cell Biochem. ;104(4):1439-1451.</div><p class="links"><a class="dblinks" href="/pubmed/18275061" ref="ordinalpos=5048&amp;ncbi_uid=4467474&amp;link_uid=18275061">Citation</a><a class="dblinks" href="/pmc/articles/PMC3530847/" ref="ordinalpos=5048&amp;ncbi_uid=4467474&amp;link_uid=PMC3530847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5049.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4145548/figure/fig5/" ref="ordinalpos=5049&amp;ncbi_uid=6551456&amp;link_uid=PMC4145548" href="/pmc/articles/PMC4145548/"><img src="/pmc/articles/instance/4145548/bin/BMRI2014-509149.005.gif" src-large="/pmc/articles/instance/4145548/bin/BMRI2014-509149.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4145548/" ref="ordinalpos=5049&amp;ncbi_uid=6551456&amp;link_uid=PMC4145548" image-link="/pmc/articles/PMC4145548/figure/fig5/" class="imagepopup">Figure 5.  From: FSim: A Novel Functional Similarity Search Algorithm and Tool for Discovering Functionally Related Gene Products. </a></p><div class="supp"><p class="details"><div>The ROC curve for the performance comparison of different algorithms using the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as an exemplary evaluation data set. The AUC value for FSim in the P53 <span class="highlight" style="background-color:">pathway</span> is 0.89, compared with 0.74, 0.83, 0.83, 0.85, and 0.62 for DAVID, funSimMax. Lin, funSimMax.relevance, dot.relevance, and hausdorff.relevance, respectively.</div></p></div><div class="aux"><div class="resc">Qiang Hu, et al. Biomed Res Int. 2014;2014:509149.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=5049&amp;ncbi_uid=6551456&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4145548/" ref="ordinalpos=5049&amp;ncbi_uid=6551456&amp;link_uid=PMC4145548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5050.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3833007/figure/fig7/" ref="ordinalpos=5050&amp;ncbi_uid=5527941&amp;link_uid=PMC3833007" href="/pmc/articles/PMC3833007/"><img src="/pmc/articles/instance/3833007/bin/TSWJ2013-536350.007.gif" src-large="/pmc/articles/instance/3833007/bin/TSWJ2013-536350.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3833007/" ref="ordinalpos=5050&amp;ncbi_uid=5527941&amp;link_uid=PMC3833007" image-link="/pmc/articles/PMC3833007/figure/fig7/" class="imagepopup">Figure 7.  From: Involvement of MAPKs and PLC Pathways in Modulation of Pacemaking Activity by So-Cheong-Ryong-Tang in Interstitial Cells of Cajal from Murine Small Intestine. </a></p><div class="supp"><p class="details"><div>Hypothetical schematic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of SCRT-induced membrane depolarizations on ICCs. SCRT-induced membrane depolarization seems to be mediated by 5-HT<sub>3</sub> and/or 5-HT<sub>4</sub> receptor, coupled to G protein, resulting in the activation of PLC <span class="highlight" style="background-color:">pathway</span>, which subsequently increases the intracellular Ca<sup>2+</sup> level from SR. The data also suggest that MAPKs are involved in the SCRT-induced membrane depolarizations.</div></p></div><div class="aux"><div class="resc">Min Woo Hwang, et al. ScientificWorldJournal. 2013;2013:536350.</div><p class="links"><a class="dblinks" href="/pubmed/24288491" ref="ordinalpos=5050&amp;ncbi_uid=5527941&amp;link_uid=24288491">Citation</a><a class="dblinks" href="/pmc/articles/PMC3833007/" ref="ordinalpos=5050&amp;ncbi_uid=5527941&amp;link_uid=PMC3833007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5051.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3670237/figure/F2/" ref="ordinalpos=5051&amp;ncbi_uid=5319477&amp;link_uid=PMC3670237" href="/pmc/articles/PMC3670237/"><img src="/pmc/articles/instance/3670237/bin/jkst-1-143-g2.gif" src-large="/pmc/articles/instance/3670237/bin/jkst-1-143-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3670237/" ref="ordinalpos=5051&amp;ncbi_uid=5319477&amp;link_uid=PMC3670237" image-link="/pmc/articles/PMC3670237/figure/F2/" class="imagepopup">Figure. From: STAT3 regulation of and by microRNAs in development and disease. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Scheme illustrating STAT3 mediated processing of individual miRNAs and contrariwise, the modulation of the STAT3 <span class="highlight" style="background-color:">pathway</span> by miRNAs at different levels. STAT3 mediates positive and negative regulation of various miRNAs at the transcriptional level. On the other hand, this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is controlled by numerous miRNAs at the receptor level, by modulating its activators and suppressors and by direct regulation of STAT3 mRNA.</div></p></div><div class="aux"><div class="resc">Arash Haghikia, et al. JAKSTAT. 2012 July 1;1(3):143-150.</div><p class="links"><a class="dblinks" href="/pubmed/24058763" ref="ordinalpos=5051&amp;ncbi_uid=5319477&amp;link_uid=24058763">Citation</a><a class="dblinks" href="/pmc/articles/PMC3670237/" ref="ordinalpos=5051&amp;ncbi_uid=5319477&amp;link_uid=PMC3670237">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5052.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4148370/figure/f3-mmr-10-04-1746/" ref="ordinalpos=5052&amp;ncbi_uid=6547197&amp;link_uid=PMC4148370" href="/pmc/articles/PMC4148370/"><img src="/pmc/articles/instance/4148370/bin/MMR-10-04-1746-g04.gif" src-large="/pmc/articles/instance/4148370/bin/MMR-10-04-1746-g04.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4148370/" ref="ordinalpos=5052&amp;ncbi_uid=6547197&amp;link_uid=PMC4148370" image-link="/pmc/articles/PMC4148370/figure/f3-mmr-10-04-1746/" class="imagepopup">Figure 3.  From: Differentially expressed genes and interacting pathways in bladder cancer revealed by bioinformatic analysis. </a></p><div class="supp"><p class="details"><div>Differentially expressed genes (DEGs) in the T-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Pink boxes, DEGs; green, non-differentially expressed genes. Source: Kyoto Encyclopedia of Genes and Genomes.</div></p></div><div class="aux"><div class="resc">YINZHOU SHEN, et al. Mol Med Rep. 2014 October;10(4):1746-1752.</div><p class="links"><a class="dblinks" href="/pubmed/25050631" ref="ordinalpos=5052&amp;ncbi_uid=6547197&amp;link_uid=25050631">Citation</a><a class="dblinks" href="/pmc/articles/PMC4148370/" ref="ordinalpos=5052&amp;ncbi_uid=6547197&amp;link_uid=PMC4148370">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5053.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3030591/figure/pone-0016516-g004/" ref="ordinalpos=5053&amp;ncbi_uid=2756763&amp;link_uid=PMC3030591" href="/pmc/articles/PMC3030591/"><img src="/pmc/articles/instance/3030591/bin/pone.0016516.g004.gif" src-large="/pmc/articles/instance/3030591/bin/pone.0016516.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3030591/" ref="ordinalpos=5053&amp;ncbi_uid=2756763&amp;link_uid=PMC3030591" image-link="/pmc/articles/PMC3030591/figure/pone-0016516-g004/" class="imagepopup">Figure 4. Different genes expression in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: A Comparative Transcriptomic Analysis of Uveal Melanoma and Normal Uveal Melanocyte. </a></p><div class="supp"><p class="details"><div>Red and green represent up-regulated and down-regulated genes, individually. The regulation extents are based on the color grade.</div></p></div><div class="aux"><div class="resc">Jianhong An, et al. PLoS One. 2011;6(1):e16516.</div><p class="links"><a class="dblinks" href="/pubmed/21305041" ref="ordinalpos=5053&amp;ncbi_uid=2756763&amp;link_uid=21305041">Citation</a><a class="dblinks" href="/pmc/articles/PMC3030591/" ref="ordinalpos=5053&amp;ncbi_uid=2756763&amp;link_uid=PMC3030591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5054.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3036618/figure/F4/" ref="ordinalpos=5054&amp;ncbi_uid=2781965&amp;link_uid=PMC3036618" href="/pmc/articles/PMC3036618/"><img src="/pmc/articles/instance/3036618/bin/1471-2164-12-50-4.gif" src-large="/pmc/articles/instance/3036618/bin/1471-2164-12-50-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3036618/" ref="ordinalpos=5054&amp;ncbi_uid=2781965&amp;link_uid=PMC3036618" image-link="/pmc/articles/PMC3036618/figure/F4/" class="imagepopup">Figure 4.  From: False negative rates in Drosophila cell-based RNAi screens: a case study. </a></p><div class="supp"><p class="details"><div><b>Transformed expression levels of core components of JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Genes expressed at low and high levels are displayed in gradations of black and red, correspondingly.</div></p></div><div class="aux"><div class="resc">Matthew Booker, et al. BMC Genomics. 2011;12:50-50.</div><p class="links"><a class="dblinks" href="/pubmed/21251254" ref="ordinalpos=5054&amp;ncbi_uid=2781965&amp;link_uid=21251254">Citation</a><a class="dblinks" href="/pmc/articles/PMC3036618/" ref="ordinalpos=5054&amp;ncbi_uid=2781965&amp;link_uid=PMC3036618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5055.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2949386/figure/pone-0013131-g006/" ref="ordinalpos=5055&amp;ncbi_uid=2521668&amp;link_uid=PMC2949386" href="/pmc/articles/PMC2949386/"><img src="/pmc/articles/instance/2949386/bin/pone.0013131.g006.gif" src-large="/pmc/articles/instance/2949386/bin/pone.0013131.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2949386/" ref="ordinalpos=5055&amp;ncbi_uid=2521668&amp;link_uid=PMC2949386" image-link="/pmc/articles/PMC2949386/figure/pone-0013131-g006/" class="imagepopup">Figure 6. Top ranked HGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Preclinical Evaluation of Novel Triphenylphosphonium Salts with Broad-Spectrum Activity. </a></p><div class="supp"><p class="details"><div>Molecules in green are down-regulated in our microarray results while those in red are up-regulated. Proteins whose phosphorylation level was affected by TP421 treatment are denoted by *.</div></p></div><div class="aux"><div class="resc">Melissa Millard, et al. PLoS One. 2010;5(10):e13131.</div><p class="links"><a class="dblinks" href="/pubmed/20957228" ref="ordinalpos=5055&amp;ncbi_uid=2521668&amp;link_uid=20957228">Citation</a><a class="dblinks" href="/pmc/articles/PMC2949386/" ref="ordinalpos=5055&amp;ncbi_uid=2521668&amp;link_uid=PMC2949386">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5056.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2174018/figure/fig1/" ref="ordinalpos=5056&amp;ncbi_uid=1147703&amp;link_uid=PMC2174018" href="/pmc/articles/PMC2174018/"><img src="/pmc/articles/instance/2174018/bin/200204089f1.gif" src-large="/pmc/articles/instance/2174018/bin/200204089f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2174018/" ref="ordinalpos=5056&amp;ncbi_uid=1147703&amp;link_uid=PMC2174018" image-link="/pmc/articles/PMC2174018/figure/fig1/" class="imagepopup">Figure 1. From: Remodeling of organelle-bound actin is required for yeast vacuole fusion. </a></p><div class="supp"><p class="details"><div><b>A <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> which regulates actin remodeling.</b> Arrows depict known protein interactions. Dashed lines and arrows depict pathways seen in mammalian cells. Lines show other interacting factors.</div></p></div><div class="aux"><div class="resc">Gary Eitzen, et al. J Cell Biol. 2002 August 19;158(4):669-679.</div><p class="links"><a class="dblinks" href="/pubmed/12177043" ref="ordinalpos=5056&amp;ncbi_uid=1147703&amp;link_uid=12177043">Citation</a><a class="dblinks" href="/pmc/articles/PMC2174018/" ref="ordinalpos=5056&amp;ncbi_uid=1147703&amp;link_uid=PMC2174018">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5057.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3250618/figure/Fig6/" ref="ordinalpos=5057&amp;ncbi_uid=3751890&amp;link_uid=PMC3250618" href="/pmc/articles/PMC3250618/"><img src="/pmc/articles/instance/3250618/bin/441_2011_1178_Fig6_HTML.gif" src-large="/pmc/articles/instance/3250618/bin/441_2011_1178_Fig6_HTML.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3250618/" ref="ordinalpos=5057&amp;ncbi_uid=3751890&amp;link_uid=PMC3250618" image-link="/pmc/articles/PMC3250618/figure/Fig6/" class="imagepopup">Fig. 6.  From: Smad phosphoisoform signals in acute and chronic liver injury: similarities and differences between epithelial and mesenchymal cells. </a></p><div class="supp"><p class="details"><div>Reversible shifting of hepatocytic Smad3-dependent <span class="highlight" style="background-color:">signaling</span> between tumor-suppression and carcinogenesis. During the carcinogenic process in human chronic liver diseases, chronic inflammation drives carcinogenesis via a shift from the pSmad3C to the pSmad3L <span class="highlight" style="background-color:">pathway</span>. Linker phosphorylation of Smad3 indirectly prevents Smad3C phosphorylation, pSmad3C-mediated transcription, and the cytostatic effect of TGF-β/activin upon hepatocytes. Inhibition of JNK activity abolishes pSmad3L-mediated oncogenic <span class="highlight" style="background-color:">signaling</span>, while restoring the lost tumor-suppressive pSmad3C <span class="highlight" style="background-color:">signaling</span> observed in normal hepatocytes</div></p></div><div class="aux"><div class="resc">Koichi Matsuzaki. Cell Tissue Res. 2012 January;347(1):225-243.</div><p class="links"><a class="dblinks" href="/pubmed/21626291" ref="ordinalpos=5057&amp;ncbi_uid=3751890&amp;link_uid=21626291">Citation</a><a class="dblinks" href="/pmc/articles/PMC3250618/" ref="ordinalpos=5057&amp;ncbi_uid=3751890&amp;link_uid=PMC3250618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5058.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3129269/figure/NOR036F4/" ref="ordinalpos=5058&amp;ncbi_uid=3460079&amp;link_uid=PMC3129269" href="/pmc/articles/PMC3129269/"><img src="/pmc/articles/instance/3129269/bin/nor03604.gif" src-large="/pmc/articles/instance/3129269/bin/nor03604.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3129269/" ref="ordinalpos=5058&amp;ncbi_uid=3460079&amp;link_uid=PMC3129269" image-link="/pmc/articles/PMC3129269/figure/NOR036F4/" class="imagepopup">Fig. 4. From: The Wnt inhibitory factor 1 (WIF1) is targeted in glioblastoma and has a tumor suppressing function potentially by induction of senescence. </a></p><div class="supp"><p class="details"><div>LN319 cell clones stably transfected with <i>WIF1</i> show reduced Wnt <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. Two stably transfected LN319 clones were analyzed for <i>WIF1</i> expression by qRT-PCR (normalized to the control cells transfected with the empty vector, pcDNA3.1) (A) and WIF1 secretion by enzyme-linked immunosorbent assay (ELISA) (B). Wnt <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> was measured both with the TCF luciferase reporter (TOP5/FOP5) and normalized to the control cells (C) and by measuring <i>AXIN2</i> mRNA expression (D). The specificity of the WIF1-induced effects in the 2 clones was controlled by transfection of specific siRNAs against <i>WIF1</i> or a respective scrambled control. Wnt <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> was measured using the TCF reporter (E) and WIF1 secretion using ELISA (F). Results are marked with 1 asterisk (*) if <i>P</i>&lt; .05 and 2 (**) if <i>P</i> &lt; .01.</div></p></div><div class="aux"><div class="resc">Wanyu L. Lambiv, et al. Neuro Oncol. 2011 July;13(7):736-747.</div><p class="links"><a class="dblinks" href="/pubmed/21642372" ref="ordinalpos=5058&amp;ncbi_uid=3460079&amp;link_uid=21642372">Citation</a><a class="dblinks" href="/pmc/articles/PMC3129269/" ref="ordinalpos=5058&amp;ncbi_uid=3460079&amp;link_uid=PMC3129269">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5059.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3974058/figure/F5/" ref="ordinalpos=5059&amp;ncbi_uid=5970034&amp;link_uid=PMC3974058" href="/pmc/articles/PMC3974058/"><img src="/pmc/articles/instance/3974058/bin/1471-2407-14-113-5.gif" src-large="/pmc/articles/instance/3974058/bin/1471-2407-14-113-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3974058/" ref="ordinalpos=5059&amp;ncbi_uid=5970034&amp;link_uid=PMC3974058" image-link="/pmc/articles/PMC3974058/figure/F5/" class="imagepopup">Figure 5.  From: TGF?-mediated suppression of CD248 in non-cancer cells via canonical Smad-dependent <span class="highlight" style="background-color:">signaling</span> pathways is uncoupled in cancer cells. </a></p><div class="supp"><p class="details"><div><b>TGFβ-induced suppression of CD248 is mediated via canonical <span class="highlight" style="background-color:">signaling</span> pathways. (A, B, C)</b> MEF were incubated for 48 hrs with TGFβ 3 ng/ml and the ALK-inhibitor SB431542 1 μM either singly or in combination. Controls included carriers for SB431542 (DMSO) or for TGFβ (0.1% BSA). <b>(A)</b> Western blots and <b>(B)</b> immunofluorescence were used to detect expression of CD248 (green). <b>(C)</b> CD248 mRNA levels were also quantified (n = 3 experiments, each in triplicate; *p &lt; 0.05). Results indicate that TGFβ-mediated suppression of CD248 protein and mRNA requires integrity of canonical ALK5-Smad2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Scale bar = 50 μm.</div></p></div><div class="aux"><div class="resc">Sahana Suresh Babu, et al. BMC Cancer. 2014;14:113-113.</div><p class="links"><a class="dblinks" href="/pubmed/24555435" ref="ordinalpos=5059&amp;ncbi_uid=5970034&amp;link_uid=24555435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3974058/" ref="ordinalpos=5059&amp;ncbi_uid=5970034&amp;link_uid=PMC3974058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5060.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4148637/figure/F1/" ref="ordinalpos=5060&amp;ncbi_uid=6609800&amp;link_uid=PMC4148637" href="/pmc/articles/PMC4148637/"><img src="/pmc/articles/instance/4148637/bin/fnagi-06-00230-g001.gif" src-large="/pmc/articles/instance/4148637/bin/fnagi-06-00230-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4148637/" ref="ordinalpos=5060&amp;ncbi_uid=6609800&amp;link_uid=PMC4148637" image-link="/pmc/articles/PMC4148637/figure/F1/" class="imagepopup">Figure 1.  From: The Intriguing Regulators of Muscle Mass in Sarcopenia and Muscular Dystrophy. </a></p><div class="supp"><p class="details"><div><b>The relationship between PI3-K–Akt–mTOR <span class="highlight" style="background-color:">signaling</span> and autophagy in muscular dystrophy</b>. The major anabolic <span class="highlight" style="background-color:">pathway</span> regulating protein synthesis in skeletal muscle is mTOR/TORC1 <span class="highlight" style="background-color:">signaling</span>. Upstream trigger (IGF-1, mechanical stress, etc.) activates mTOR <span class="highlight" style="background-color:">signaling</span> through a number of different intermediary proteins such as Akt and Rheb. Several anabolic stimulation increases the amount of activated Akt, which blocks the nuclear translocation of Foxo3 to enhance the expression of autophagy-related genes (Bnip, LC3, and Atg12) and atrogene (atrogin-1 and MuRF-1). In dystrophic muscle, higher Akt potently blocks the inhibition of Rheb by TSC-1/TSC-2, and hyperactivate mTORC1. Unnecessary activated mTORC1 would extremely enhance protein synthesis and blocks autophagy-dependent <span class="highlight" style="background-color:">signaling</span>. Therefore, muscular dystrophy exhibits apparent defect of autophagic process similar to sarcopenic muscle.</div></p></div><div class="aux"><div class="resc">Kunihiro Sakuma, et al. Front Aging Neurosci. 2014;6:230.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=5060&amp;ncbi_uid=6609800&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4148637/" ref="ordinalpos=5060&amp;ncbi_uid=6609800&amp;link_uid=PMC4148637">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5061.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2813409/figure/F4/" ref="ordinalpos=5061&amp;ncbi_uid=2692833&amp;link_uid=PMC2813409" href="/pmc/articles/PMC2813409/"><img src="/pmc/articles/instance/2813409/bin/nihms150379f4.gif" src-large="/pmc/articles/instance/2813409/bin/nihms150379f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2813409/" ref="ordinalpos=5061&amp;ncbi_uid=2692833&amp;link_uid=PMC2813409" image-link="/pmc/articles/PMC2813409/figure/F4/" class="imagepopup">Figure 4. LPS activates the AKT-GSK3β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in enterocytes.  From: Toll-like-receptor-4 Inhibits Enterocyte Proliferation via Impaired ?-catenin <span class="highlight" style="background-color:">Signaling</span> in Necrotizing Enterocolitis. </a></p><div class="supp"><p class="details"><div><b>A–E:</b> SDS-PAGE showing expression of the indicated protein in IEC-6 cells treated with LPS at the concentrations and durations indicated (<b>A–D</b>) or mucosal scrapings from the terminal ileum of mice injected with saline or LPS for the durations indicated (<b>E</b>). All blots are representative of 3 separate experiments; over 5 mice per group in E. <b>F:</b> Schematic summarizing panels A–D showing mechanism by which TLR4 inhibits enterocyte proliferation.</div></p></div><div class="aux"><div class="resc">Chhinder P. Sodhi, et al. Gastroenterology. ;138(1):185.</div><p class="links"><a class="dblinks" href="/pubmed/19786028" ref="ordinalpos=5061&amp;ncbi_uid=2692833&amp;link_uid=19786028">Citation</a><a class="dblinks" href="/pmc/articles/PMC2813409/" ref="ordinalpos=5061&amp;ncbi_uid=2692833&amp;link_uid=PMC2813409">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5062.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3522553/figure/F4/" ref="ordinalpos=5062&amp;ncbi_uid=4462243&amp;link_uid=PMC3522553" href="/pmc/articles/PMC3522553/"><img src="/pmc/articles/instance/3522553/bin/1471-2105-13-186-4.gif" src-large="/pmc/articles/instance/3522553/bin/1471-2105-13-186-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3522553/" ref="ordinalpos=5062&amp;ncbi_uid=4462243&amp;link_uid=PMC3522553" image-link="/pmc/articles/PMC3522553/figure/F4/" class="imagepopup">Figure 4.  From: Integrated QSAR study for inhibitors of hedgehog signal <span class="highlight" style="background-color:">pathway</span> against multiple cell lines:a collaborative filtering method. </a></p><div class="supp"><p class="details"><div><b>Comparison of CMF based QSAR modeling with ridge regression and SVR on the inhibitors of Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.</b> The compounds are represented with Drug-like index and General Descriptor respectively. The whole data was partitioned randomly with selecting 2/3 data as training dataset and 1/3 data as testing dataset. The whole procedure was repeated 10 times and the averaged performance was calculated. Cell line No. 1–4 corresponding to BxPC-3, NCI-H446, SW1990 and NCI-H157 respectively.</div></p></div><div class="aux"><div class="resc">Jun Gao, et al. BMC Bioinformatics. 2012;13:186-186.</div><p class="links"><a class="dblinks" href="/pubmed/22849868" ref="ordinalpos=5062&amp;ncbi_uid=4462243&amp;link_uid=22849868">Citation</a><a class="dblinks" href="/pmc/articles/PMC3522553/" ref="ordinalpos=5062&amp;ncbi_uid=4462243&amp;link_uid=PMC3522553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5063.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3763845/figure/F1/" ref="ordinalpos=5063&amp;ncbi_uid=5268958&amp;link_uid=PMC3763845" href="/pmc/articles/PMC3763845/"><img src="/pmc/articles/instance/3763845/bin/1742-4933-10-10-1.gif" src-large="/pmc/articles/instance/3763845/bin/1742-4933-10-10-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3763845/" ref="ordinalpos=5063&amp;ncbi_uid=5268958&amp;link_uid=PMC3763845" image-link="/pmc/articles/PMC3763845/figure/F1/" class="imagepopup">Figure 1.  From: Pinpointing beta adrenergic receptor in ageing pathophysiology: victim or executioner? Evidence from crime scenes. </a></p><div class="supp"><p class="details"><div><b>Classical <span class="highlight" style="background-color:">pathway</span> of β</b><sub><b>2</b></sub><b>adrenergic receptor (β</b><sub><b>2</b></sub><b>AR) activating Gα</b><sub><b>s</b></sub><b>protein, which in turn activates the conversion of ATP in cAMP by the adenylate cyclase (AC).</b> An alternative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves Gα<sub>i</sub> protein. G-protein-linked receptor kinase 2 (GRK2) and β-Arrestin participate in the desensitization process of the receptor.</div></p></div><div class="aux"><div class="resc">Gaetano Santulli, et al. Immun Ageing. 2013;10:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/23497413" ref="ordinalpos=5063&amp;ncbi_uid=5268958&amp;link_uid=23497413">Citation</a><a class="dblinks" href="/pmc/articles/PMC3763845/" ref="ordinalpos=5063&amp;ncbi_uid=5268958&amp;link_uid=PMC3763845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5064.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3979352/figure/F1/" ref="ordinalpos=5064&amp;ncbi_uid=6005930&amp;link_uid=PMC3979352" href="/pmc/articles/PMC3979352/"><img src="/pmc/articles/instance/3979352/bin/nihms566930f1.gif" src-large="/pmc/articles/instance/3979352/bin/nihms566930f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3979352/" ref="ordinalpos=5064&amp;ncbi_uid=6005930&amp;link_uid=PMC3979352" image-link="/pmc/articles/PMC3979352/figure/F1/" class="imagepopup">Figure 1.  From: Targeting leucine addiction and autophagy in melanoma. </a></p><div class="supp"><p class="details"><div>Melanoma cells with mutations in the RAS/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are sensitive to combine therapy of leucine deprivation and inhibition of autophagy. In normal cells, the mTORC1 complex is a sensor of nutrient levels, and nutrient starvation induces its inhibition-activating autophagy, which in turn replenishes nutrient levels. In melanoma cells, however, the hyperactivation of the RAS/MAPK <span class="highlight" style="background-color:">pathway</span> cells keeps mTORC1 active and therefore autophagy is not turned on, which induces apoptosis.</div></p></div><div class="aux"><div class="resc">Maria T. Diaz-Meco. Pigment Cell Melanoma Res. 2011 August;24(4):588-589.</div><p class="links"><a class="dblinks" href="/pubmed/22489330" ref="ordinalpos=5064&amp;ncbi_uid=6005930&amp;link_uid=22489330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3979352/" ref="ordinalpos=5064&amp;ncbi_uid=6005930&amp;link_uid=PMC3979352">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5065.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137750/figure/fig4/" ref="ordinalpos=5065&amp;ncbi_uid=3095559&amp;link_uid=PMC3137750" href="/pmc/articles/PMC3137750/"><img src="/pmc/articles/instance/3137750/bin/ECAM2011-870753.004.gif" src-large="/pmc/articles/instance/3137750/bin/ECAM2011-870753.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137750/" ref="ordinalpos=5065&amp;ncbi_uid=3095559&amp;link_uid=PMC3137750" image-link="/pmc/articles/PMC3137750/figure/fig4/" class="imagepopup">Figure 4.  From: Brazilian Propolis Suppresses Angiogenesis by Inducing Apoptosis in Tube-Forming Endothelial Cells through Inactivation of Survival Signal ERK1/2. </a></p><div class="supp"><p class="details"><div>Schematic diagram of angiogenesis suppression by EEBP. Proangiogenic factors such as bFGF and PMA stimulate ERK1/2 <span class="highlight" style="background-color:">signaling</span>. Subsequently, the survival signal inactivates caspase <span class="highlight" style="background-color:">pathway</span>, thereby maintaining cell survival and facilitating angiogenesis. In contrast, EEBP inhibits ERK1/2 activation, thereby activating caspase <span class="highlight" style="background-color:">pathway</span> and inducing apoptosis, which consequently leads to angiogenesis suppression.</div></p></div><div class="aux"><div class="resc">Kazuhiro Kunimasa, et al. Evid Based Complement Alternat Med. 2011;2011:870753.</div><p class="links"><a class="dblinks" href="/pubmed/19351710" ref="ordinalpos=5065&amp;ncbi_uid=3095559&amp;link_uid=19351710">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137750/" ref="ordinalpos=5065&amp;ncbi_uid=3095559&amp;link_uid=PMC3137750">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5066.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3304564/figure/fig01/" ref="ordinalpos=5066&amp;ncbi_uid=3893836&amp;link_uid=PMC3304564" href="/pmc/articles/PMC3304564/"><img src="/pmc/articles/instance/3304564/bin/ajcr0002-0178-f1.gif" src-large="/pmc/articles/instance/3304564/bin/ajcr0002-0178-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3304564/" ref="ordinalpos=5066&amp;ncbi_uid=3893836&amp;link_uid=PMC3304564" image-link="/pmc/articles/PMC3304564/figure/fig01/" class="imagepopup">Figure 1.  From: Therapeutic implications of targeting the PI3Kinase/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> module in melanoma therapy. </a></p><div class="supp"><p class="details"><div>The PI3K/AKT <span class="highlight" style="background-color:">signaling</span> module and cross-talk with other <span class="highlight" style="background-color:">pathway</span>. Upon binding of growth factors to receptor tyrosine kinases (RTK) a cascade of events are triggered resulting in the activation of PI3Kinase and subsequent AKT activation (module I). This cascade leads to inhibition of pro-apoptotic <span class="highlight" style="background-color:">signaling</span> molecules (BAD, BIM, BAX, Cas-pase 9) or induction of anti-apoptotic Bcl-2, Mcl-1, BCL-XL, Bcl-w conferring a growth advantage to the tumor cells. One of the major targets of AKT is the mTOR (module Ii) which is also implicated in cell growth and metabolism. The intricacy of the AKT <span class="highlight" style="background-color:">pathway</span> is further complicated by its cross-talk, via various positive and negative regulatory feedback loops, with multiple major cell survival pathways including: NF-κB (module II), RAS/RAF/MEK/ERK MAPK (module III), p38MAPK (module IV), JNK MAPK (module V). This elaborate network of <span class="highlight" style="background-color:">signaling</span> pathways will amplify the anti-apoptotic <span class="highlight" style="background-color:">signaling</span> and confer a resistant phenotype to the malignant melanoma cells.</div></p></div><div class="aux"><div class="resc">Ali R Jazirehi, et al. Am J Cancer Res. 2012;2(2):178-191.</div><p class="links"><a class="dblinks" href="/pubmed/22485197" ref="ordinalpos=5066&amp;ncbi_uid=3893836&amp;link_uid=22485197">Citation</a><a class="dblinks" href="/pmc/articles/PMC3304564/" ref="ordinalpos=5066&amp;ncbi_uid=3893836&amp;link_uid=PMC3304564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5067.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1405901/figure/f6/" ref="ordinalpos=5067&amp;ncbi_uid=698872&amp;link_uid=PMC1405901" href="/pmc/articles/PMC1405901/"><img src="/pmc/articles/instance/1405901/bin/zek0020626100006.gif" src-large="/pmc/articles/instance/1405901/bin/zek0020626100006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1405901/" ref="ordinalpos=5067&amp;ncbi_uid=698872&amp;link_uid=PMC1405901" image-link="/pmc/articles/PMC1405901/figure/f6/" class="imagepopup">FIG. 6. From: The Phosducin-Like Protein PhnA Is Required for G??-Mediated <span class="highlight" style="background-color:">Signaling</span> for Vegetative Growth, Developmental Control, and Toxin Biosynthesis in Aspergillus nidulans. </a></p><div class="supp"><p class="details"><div>Proposed differential roles of FadA, FlbA, PhnA, and Gβγ in controlling ST production. We propose that the <i>A. nidulans</i> PhLP PhnA is an essential component of SfaD-GpgA-mediated <span class="highlight" style="background-color:">signaling</span>, which is required for normal vegetative growth, sexual fruiting body formation, ST production, and proper down-regulation of asexual development. FadA-mediated vegetative growth <span class="highlight" style="background-color:">signaling</span> is transduced in part by the primary PKA PkaA (34). PkaB is the secondary (backup) PKA catalytic subunit, playing a role in hyphal growth and germination (25). The FadA→PkaA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is thought to be primarily responsible for the negative control of ST biosynthesis via modulating the activity of the AflR protein (35). The potential inhibitory role of the FadA→PkaA <span class="highlight" style="background-color:">pathway</span> on the transcription of <i>aflR</i> is indicated by a dotted line (34, 35). It is proposed that the results of SfaD-GpgA <span class="highlight" style="background-color:">signaling</span> include transcriptional activation of <i>aflR</i> and subsequent ST production. A dotted arrow presents the predicted posttranscriptional/posttranslational activation of AflR by FlbA. Potential autoactivation of <i>aflR</i> transcription by the AflR protein (10, 35) is also indicated.</div></p></div><div class="aux"><div class="resc">Jeong-Ah Seo, et al. Eukaryot Cell. 2006 February;5(2):400-410.</div><p class="links"><a class="dblinks" href="/pubmed/16467480" ref="ordinalpos=5067&amp;ncbi_uid=698872&amp;link_uid=16467480">Citation</a><a class="dblinks" href="/pmc/articles/PMC1405901/" ref="ordinalpos=5067&amp;ncbi_uid=698872&amp;link_uid=PMC1405901">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5068.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4229244/figure/pone-0112516-g006/" ref="ordinalpos=5068&amp;ncbi_uid=6849582&amp;link_uid=PMC4229244" href="/pmc/articles/PMC4229244/"><img src="/pmc/articles/instance/4229244/bin/pone.0112516.g006.gif" src-large="/pmc/articles/instance/4229244/bin/pone.0112516.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4229244/" ref="ordinalpos=5068&amp;ncbi_uid=6849582&amp;link_uid=PMC4229244" image-link="/pmc/articles/PMC4229244/figure/pone-0112516-g006/" class="imagepopup">Figure 6. Schematic model illustrating that CYLD negatively regulates NTHi-induced IL-8 expression via MKP-1-dependent inhibition of ERK..  From: CYLD Negatively Regulates Nontypeable Haemophilus influenzae-Induced IL-8 Expression via Phosphatase MKP-1-Dependent Inhibition of ERK. </a></p><div class="supp"><p class="details"><div>As indicated, CYLD negatively regulates NTHi-induced IL-8 expression through MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by upregulating phosphatase MKP-1.</div></p></div><div class="aux"><div class="resc">Wenzhuo Y. Wang, et al. PLoS One. 2014;9(11):e112516.</div><p class="links"><a class="dblinks" href="/pubmed/25389768" ref="ordinalpos=5068&amp;ncbi_uid=6849582&amp;link_uid=25389768">Citation</a><a class="dblinks" href="/pmc/articles/PMC4229244/" ref="ordinalpos=5068&amp;ncbi_uid=6849582&amp;link_uid=PMC4229244">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5069.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3792053/figure/pone-0076519-g006/" ref="ordinalpos=5069&amp;ncbi_uid=5377160&amp;link_uid=PMC3792053" href="/pmc/articles/PMC3792053/"><img src="/pmc/articles/instance/3792053/bin/pone.0076519.g006.gif" src-large="/pmc/articles/instance/3792053/bin/pone.0076519.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3792053/" ref="ordinalpos=5069&amp;ncbi_uid=5377160&amp;link_uid=PMC3792053" image-link="/pmc/articles/PMC3792053/figure/pone-0076519-g006/" class="imagepopup">Figure 6. Proposed Model for the study..  From: Curcumin Suppresses Gelatinase B Mediated Norepinephrine Induced Stress in H9c2 Cardiomyocytes. </a></p><div class="supp"><p class="details"><div>An illustration of the proposed mechanism of prevention of NE-induced cardiotoxicity by curcumin through downregulation of gelatinase B. It shows that curcumin targets different <span class="highlight" style="background-color:">signaling</span> players in the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Shrey Kohli, et al. PLoS One. 2013;8(10):e76519.</div><p class="links"><a class="dblinks" href="/pubmed/24116115" ref="ordinalpos=5069&amp;ncbi_uid=5377160&amp;link_uid=24116115">Citation</a><a class="dblinks" href="/pmc/articles/PMC3792053/" ref="ordinalpos=5069&amp;ncbi_uid=5377160&amp;link_uid=PMC3792053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5070.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405105/figure/pone-0040999-g007/" ref="ordinalpos=5070&amp;ncbi_uid=4146962&amp;link_uid=PMC3405105" href="/pmc/articles/PMC3405105/"><img src="/pmc/articles/instance/3405105/bin/pone.0040999.g007.gif" src-large="/pmc/articles/instance/3405105/bin/pone.0040999.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405105/" ref="ordinalpos=5070&amp;ncbi_uid=4146962&amp;link_uid=PMC3405105" image-link="/pmc/articles/PMC3405105/figure/pone-0040999-g007/" class="imagepopup">Figure 7. A bifurcated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in PMA-mediated adhesion of monocytes..  From: PKC-Dependent Human Monocyte Adhesion Requires AMPK and Syk Activation. </a></p><div class="supp"><p class="details"><div>PKC can activate LKB1/AMPK, leading to phosphorylation and activation of Syk, then Src and FAK subsequently. PKC-dependent ERK activation induces a coordinated signal for cytoskeleton rearrangement and cell adhesion.</div></p></div><div class="aux"><div class="resc">Mei-Ying Chang, et al. PLoS One. 2012;7(7):e40999.</div><p class="links"><a class="dblinks" href="/pubmed/22848421" ref="ordinalpos=5070&amp;ncbi_uid=4146962&amp;link_uid=22848421">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405105/" ref="ordinalpos=5070&amp;ncbi_uid=4146962&amp;link_uid=PMC3405105">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5071.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436276/figure/F7/" ref="ordinalpos=5071&amp;ncbi_uid=5240479&amp;link_uid=PMC3436276" href="/pmc/articles/PMC3436276/"><img src="/pmc/articles/instance/3436276/bin/zbc0351219890007.gif" src-large="/pmc/articles/instance/3436276/bin/zbc0351219890007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436276/" ref="ordinalpos=5071&amp;ncbi_uid=5240479&amp;link_uid=PMC3436276" image-link="/pmc/articles/PMC3436276/figure/F7/" class="imagepopup">FIGURE 7. From: The Oncoprotein HBXIP Uses Two Pathways to Up-regulate S100A4 in Promotion of Growth and Migration of Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>The model illustrates proposed two pathways of regulation of S100A4 mediated by HBXIP.</b> The model shows that HBXIP up-regulates S100A4 through not only binding to STAT4, but also activating PTEN/PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> independently of STAT4.</div></p></div><div class="aux"><div class="resc">Shuangping Liu, et al. J Biol Chem. 2012 August 31;287(36):30228-30239.</div><p class="links"><a class="dblinks" href="/pubmed/22740693" ref="ordinalpos=5071&amp;ncbi_uid=5240479&amp;link_uid=22740693">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436276/" ref="ordinalpos=5071&amp;ncbi_uid=5240479&amp;link_uid=PMC3436276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5072.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3030591/figure/pone-0016516-g007/" ref="ordinalpos=5072&amp;ncbi_uid=2756760&amp;link_uid=PMC3030591" href="/pmc/articles/PMC3030591/"><img src="/pmc/articles/instance/3030591/bin/pone.0016516.g007.gif" src-large="/pmc/articles/instance/3030591/bin/pone.0016516.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3030591/" ref="ordinalpos=5072&amp;ncbi_uid=2756760&amp;link_uid=PMC3030591" image-link="/pmc/articles/PMC3030591/figure/pone-0016516-g007/" class="imagepopup">Figure 7. Different genes expression in P53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: A Comparative Transcriptomic Analysis of Uveal Melanoma and Normal Uveal Melanocyte. </a></p><div class="supp"><p class="details"><div>Red and green represent up-regulated and down-regulated genes, individually. The regulation extents are based on the color grade.</div></p></div><div class="aux"><div class="resc">Jianhong An, et al. PLoS One. 2011;6(1):e16516.</div><p class="links"><a class="dblinks" href="/pubmed/21305041" ref="ordinalpos=5072&amp;ncbi_uid=2756760&amp;link_uid=21305041">Citation</a><a class="dblinks" href="/pmc/articles/PMC3030591/" ref="ordinalpos=5072&amp;ncbi_uid=2756760&amp;link_uid=PMC3030591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5073.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657775/figure/F7/" ref="ordinalpos=5073&amp;ncbi_uid=1704961&amp;link_uid=PMC2657775" href="/pmc/articles/PMC2657775/"><img src="/pmc/articles/instance/2657775/bin/1752-0509-3-11-7.gif" src-large="/pmc/articles/instance/2657775/bin/1752-0509-3-11-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657775/" ref="ordinalpos=5073&amp;ncbi_uid=1704961&amp;link_uid=PMC2657775" image-link="/pmc/articles/PMC2657775/figure/F7/" class="imagepopup">Figure 7.  From: A Drosophila systems model of pentylenetetrazole induced locomotor plasticity responsive to antiepileptic drugs. </a></p><div class="supp"><p class="details"><div><b>Enrichment of TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Pink boxes indicate CG7583 extended network genes that overrepresent one or more GO processes. For details, see text.</div></p></div><div class="aux"><div class="resc">Farhan Mohammad, et al. BMC Syst Biol. 2009;3:11-11.</div><p class="links"><a class="dblinks" href="/pubmed/19154620" ref="ordinalpos=5073&amp;ncbi_uid=1704961&amp;link_uid=19154620">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657775/" ref="ordinalpos=5073&amp;ncbi_uid=1704961&amp;link_uid=PMC2657775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5074.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2706661/figure/F1/" ref="ordinalpos=5074&amp;ncbi_uid=1963559&amp;link_uid=PMC2706661" href="/pmc/articles/PMC2706661/"><img src="/pmc/articles/instance/2706661/bin/nihms75822f1.gif" src-large="/pmc/articles/instance/2706661/bin/nihms75822f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2706661/" ref="ordinalpos=5074&amp;ncbi_uid=1963559&amp;link_uid=PMC2706661" image-link="/pmc/articles/PMC2706661/figure/F1/" class="imagepopup">Figure 1.  From: Back and Forth between Cell Fate Specification and Movement during Vertebrate Gastrulation. </a></p><div class="supp"><p class="details"><div>Relative timing of cell fate specification and morphogenetic events during early zebrafish morphogenesis. The roles of the major <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cell fate specification and morphogenetic events are indicated.</div></p></div><div class="aux"><div class="resc">Carl-Philipp Heisenberg, et al. Curr Opin Genet Dev. ;18(4):311-316.</div><p class="links"><a class="dblinks" href="/pubmed/18721878" ref="ordinalpos=5074&amp;ncbi_uid=1963559&amp;link_uid=18721878">Citation</a><a class="dblinks" href="/pmc/articles/PMC2706661/" ref="ordinalpos=5074&amp;ncbi_uid=1963559&amp;link_uid=PMC2706661">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5075.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3384410/figure/fig6/" ref="ordinalpos=5075&amp;ncbi_uid=4464181&amp;link_uid=PMC3384410" href="/pmc/articles/PMC3384410/"><img src="/pmc/articles/instance/3384410/bin/JCB_201109091_Fig6.gif" src-large="/pmc/articles/instance/3384410/bin/JCB_201109091_Fig6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3384410/" ref="ordinalpos=5075&amp;ncbi_uid=4464181&amp;link_uid=PMC3384410" image-link="/pmc/articles/PMC3384410/figure/fig6/" class="imagepopup">Figure 6. From: Follistatin-mediated skeletal muscle hypertrophy is regulated by Smad3 and mTOR independently of myostatin. </a></p><div class="supp"><p class="details"><div><b>Proposed model of <span class="highlight" style="background-color:">signaling</span> associated with Fst-mediated skeletal muscle hypertrophy.</b> Fst inhibits the activation of the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by binding to multiple extracellular ligands including myostatin and Activin A. Inhibiting the actions of myostatin and Activin A diminishes phosphorylation of Smad3, which alters the transcription of Smad target genes, and potentiates Akt/mTOR <span class="highlight" style="background-color:">signaling</span> that stimulates protein synthesis. Increased phosphorylation of Akt contributes to the inhibition of Smad3-dependent signals that otherwise repress muscle growth (arrow with broken line), thereby maximizing protein synthesis networks regulated by Fst.</div></p></div><div class="aux"><div class="resc">Catherine E. Winbanks, et al. J Cell Biol. 2012 June 25;197(7):997-1008.</div><p class="links"><a class="dblinks" href="/pubmed/22711699" ref="ordinalpos=5075&amp;ncbi_uid=4464181&amp;link_uid=22711699">Citation</a><a class="dblinks" href="/pmc/articles/PMC3384410/" ref="ordinalpos=5075&amp;ncbi_uid=4464181&amp;link_uid=PMC3384410">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5076.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2885137/figure/F4/" ref="ordinalpos=5076&amp;ncbi_uid=3118817&amp;link_uid=PMC2885137" href="/pmc/articles/PMC2885137/"><img src="/pmc/articles/instance/2885137/bin/nihms203801f4.gif" src-large="/pmc/articles/instance/2885137/bin/nihms203801f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2885137/" ref="ordinalpos=5076&amp;ncbi_uid=3118817&amp;link_uid=PMC2885137" image-link="/pmc/articles/PMC2885137/figure/F4/" class="imagepopup">Figure 4.  From: Down-regulation of the Rho GTPase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in the microRNA-138 mediated inhibition of cell migration and invasion in tongue squamous cell carcinoma. </a></p><div class="supp"><p class="details"><div>Potential roles of miR-138 on Rho GTPase <span class="highlight" style="background-color:">signaling</span> cascade and cancer cell metastasis.</div></p></div><div class="aux"><div class="resc">Lu Jiang, et al. Int J Cancer. ;127(3):505-512.</div><p class="links"><a class="dblinks" href="/pubmed/20232393" ref="ordinalpos=5076&amp;ncbi_uid=3118817&amp;link_uid=20232393">Citation</a><a class="dblinks" href="/pmc/articles/PMC2885137/" ref="ordinalpos=5076&amp;ncbi_uid=3118817&amp;link_uid=PMC2885137">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5077.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013670/figure/f5-ijms-15-06925/" ref="ordinalpos=5077&amp;ncbi_uid=6110925&amp;link_uid=PMC4013670" href="/pmc/articles/PMC4013670/"><img src="/pmc/articles/instance/4013670/bin/ijms-15-06925f5.gif" src-large="/pmc/articles/instance/4013670/bin/ijms-15-06925f5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013670/" ref="ordinalpos=5077&amp;ncbi_uid=6110925&amp;link_uid=PMC4013670" image-link="/pmc/articles/PMC4013670/figure/f5-ijms-15-06925/" class="imagepopup">Figure 5. From: Protective Effect of Resveratrol against IL-1?-Induced Inflammatory Response on Human Osteoarthritic Chondrocytes Partly via the TLR4/MyD88/NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: An "in Vitro Study". </a></p><div class="supp"><p class="details"><div>Effect of resveratrol on the TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human chondrocytes. The effect of resveratrol on (<b>a</b>) MyD88 and (<b>b</b>) TRAF6 mRNA and protein expression was assessed in serum-starved (0.5% FCS) articular chondrocytes treated with 10 ng/mL IL-1β for 1 h before being treated with either 12.5 or 100 μM resveratrol for 24 h. The relative expression levels of MyD88 and TRAF6 mRNA were determined by real-time RT-PCR. <i>p</i> &lt; 0.05, * <i>vs.</i> control, # <i>vs.</i> IL-1β. After incubation of chondrocytes in the presence or absence of resveratrol as described above, the relative amount of MyD88 and TRAF6 were determined by western blot analysis.</div></p></div><div class="aux"><div class="resc">Li Liu, et al. Int J Mol Sci. 2014 April;15(4):6925-6940.</div><p class="links"><a class="dblinks" href="/pubmed/24758933" ref="ordinalpos=5077&amp;ncbi_uid=6110925&amp;link_uid=24758933">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013670/" ref="ordinalpos=5077&amp;ncbi_uid=6110925&amp;link_uid=PMC4013670">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5078.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3486458/figure/F2/" ref="ordinalpos=5078&amp;ncbi_uid=4845505&amp;link_uid=PMC3486458" href="/pmc/articles/PMC3486458/"><img src="/pmc/articles/instance/3486458/bin/zjv9990967550002.gif" src-large="/pmc/articles/instance/3486458/bin/zjv9990967550002.jpg" alt="Fig 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3486458/" ref="ordinalpos=5078&amp;ncbi_uid=4845505&amp;link_uid=PMC3486458" image-link="/pmc/articles/PMC3486458/figure/F2/" class="imagepopup">Fig 2.  From: Porcine Circovirus Type 2 Induces Autophagy via the AMPK/ERK/TSC2/mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in PK-15 Cells. </a></p><div class="supp"><p class="details"><div>PCV2 induces autophagy by activating ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A and B) LC3-II, PCV2 Cap expression, phosphorylated ERK1/2 (p-ERK1/2), and total ERK1/2 in PK-15 cells infected with PCV2 or mock infected (Ctrl) for 36 h. Their ratios to β-actin were normalized to mock infection, set at 1.0. (C and D) Effect of UO126 treatment on endogenous LC3-II, p-ERK1/2, and total ERK1/2 of PK-15 cells infected with PCV2 for 36 h. Their ratios to β-actin were normalized to mock infection, set at 1.0. (E) Formation of autophagosomes, shown as green punctae in PK-15/EGFP-LC3 cells that were treated as described in panel C. (F) Average number of punctae in each cell from 60 to 80 cells in each treatment. (G and H) PK-15 cells were mock treated (NS) or treated with ERK1/2-specific siRNA for 24 h. The cells were then further infected with PCV2 or mock infected for 36 h, and p-ERK and LC3-II ratios to β-actin were normalized to mock-infected cells, set at 1.0. (I) Formation of autophagosomes, shown as green punctae in PK-15/EGFP-LC3 cells that were treated as described in panel G. (J) Average number of punctae in each cell from 60 to 80 cells in each treatment.</div></p></div><div class="aux"><div class="resc">Binglin Zhu, et al. J Virol. 2012 November;86(22):12003-12012.</div><p class="links"><a class="dblinks" href="/pubmed/22915817" ref="ordinalpos=5078&amp;ncbi_uid=4845505&amp;link_uid=22915817">Citation</a><a class="dblinks" href="/pmc/articles/PMC3486458/" ref="ordinalpos=5078&amp;ncbi_uid=4845505&amp;link_uid=PMC3486458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5079.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2874113/figure/F6/" ref="ordinalpos=5079&amp;ncbi_uid=2948448&amp;link_uid=PMC2874113" href="/pmc/articles/PMC2874113/"><img src="/pmc/articles/instance/2874113/bin/nihms-200801-f0006.gif" src-large="/pmc/articles/instance/2874113/bin/nihms-200801-f0006.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2874113/" ref="ordinalpos=5079&amp;ncbi_uid=2948448&amp;link_uid=PMC2874113" image-link="/pmc/articles/PMC2874113/figure/F6/" class="imagepopup">Fig. 6. The RIG-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and downstream components of IRF3 activation are not affected by SRT and TFP.  From: A high throughput screening for TLR3-IRF3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulators identifies several antipsychotic drugs as TLR inhibitors. </a></p><div class="supp"><p class="details"><div>HEK293 cells were co-transfected with ISG56-luciferase and β-actin Renilla luciferase, together with RIG-I (A) and IRF-3-5D mutant (B) for 24 h. Following transfection, cells were treated with SRT or TFP at the indicated concentrations for another 8 h. Cell lyasates were analyzed for firefly and Renila luciferese activities using the Dual Luciferase Reporter Assay System. The results were corrected for transfection efficiencies with Renilla luciferase and expressed as fold induction of ISG56 luciferase for RIG-I or IRF-3-5D transfected cells relative to those of vector transfected controls. The results were representative of two similar experiments.</div></p></div><div class="aux"><div class="resc">Jianzhong Zhu, et al. J Immunol. ;184(10):5768-5776.</div><p class="links"><a class="dblinks" href="/pubmed/20382888" ref="ordinalpos=5079&amp;ncbi_uid=2948448&amp;link_uid=20382888">Citation</a><a class="dblinks" href="/pmc/articles/PMC2874113/" ref="ordinalpos=5079&amp;ncbi_uid=2948448&amp;link_uid=PMC2874113">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5080.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC4005015/figure/F9/" ref="ordinalpos=5080&amp;ncbi_uid=6072629&amp;link_uid=PMC4005015" href="/pmc/articles/PMC4005015/"><img src="/pmc/articles/instance/4005015/bin/nihms239986f9a.gif" src-large="/pmc/articles/instance/4005015/bin/nihms239986f9a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC4005015/figure/F9/" ref="ordinalpos=5080&amp;ncbi_uid=6072629&amp;link_uid=PMC4005015" href="/pmc/articles/PMC4005015/"><img src="/pmc/articles/instance/4005015/bin/nihms239986f9b.gif" src-large="/pmc/articles/instance/4005015/bin/nihms239986f9b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4005015/" ref="ordinalpos=5080&amp;ncbi_uid=6072629&amp;link_uid=PMC4005015" image-link="/pmc/articles/PMC4005015/figure/F9/" class="imagepopup">Fig. 9. Hyperglycemic HIF <span class="highlight" style="background-color:">pathway</span> in both normoxic (9a) and hypoxic conditions (9b).  From: DIETARY HYPERGLYCEMIA, GLYCEMIC INDEX AND METABOLIC RETINAL DISEASES. </a></p><div class="supp"><p class="details"><div>Fig. 9a. Normaxic, hyperglycemic HIF <span class="highlight" style="background-color:">pathway</span>. In normoxia, hyperglycemic PKC activation, AGEs formation, mitochondrial ROS, and proinflammatory cytokines (e.g. IL-1β and TNF-α) decreases the degradation (through impairing proteasomal system) and/or increases the expression of HIF (through activating NF-<sub>κ</sub>B). The elevated cytoplasmic HIF proteins may switch glucose metabolism from aerobic respiration to fermentation giving rise to lactate accumulation (also see Fig. 4a). The hyperglycemia-induced excess cytosolic HIF proteins may also lead to increased autophagy, while the lysosomal proteases are impaired by hyperglycemia. The combination of the two effects may also results in the accumulation of lysosomal lipofuscin. In addition, the excess cytosolic HIF proteins, such as an ubiquitinated form of HIF-1α induced by TNF-α, can also transactivate HIF-inducible genes (also see Fig 9b). However, under hypoxia the hyperglycemia-induced HIF protein is more stable.<br />Fig. 9b. Hypoxic, hyperglycemic HIF <span class="highlight" style="background-color:">pathway</span>. In hypoxia, the transactivation activity of HIF is turned on because both oxygen sensors, PHD and FIH, become inactive. This eliminates proteasomal degradation of HIF proteins. The HIF proteins are further stabilized by hypoxia-induced HSP90 and bind to the HREs in the promoter or enhancer region of HIF-inducible genes to transactivate the transcriptions of the genes. The HIF <span class="highlight" style="background-color:">pathway</span> consists of many HIF-inducible genes, which encode a wide range of proteins, including RAGE, glycolytic enzymes, GLUTs, Epo, and VEGF. The hyperglycemic, hypoxic HIF <span class="highlight" style="background-color:">pathway</span> may be enhanced by hyperglycemic AGE formation and PKC activation, which can activate RAGE <span class="highlight" style="background-color:">signaling</span> cascades and increase HIF expression. Furthermore, in adaptation of lower efficiency of ATP generation from fermentation, the activation of GLUTs and glycolytic enzymes increase glucose uptake and up-regulate the glycolysis <span class="highlight" style="background-color:">pathway</span> (also see Fig. 4b). Therefore, in hyperglycemic, hypoxic conditions the HIF <span class="highlight" style="background-color:">pathway</span> may further deteriorate the four glycolysis-associated pathways and lactate accumulation in the cytoplasm. Furthermore, the hyperglycemia-induced excess cytosolic HIF proteins may also enhance autophage, while the lysosomal system is impaired by hyperglycemia. Together, they may also result in the accumulation of lysosomal lipofuscin. In addition, hyperglycemia-induced proinflammatory cytokines (e.g. IL-1β and TNF-α) can further enhance the HIF <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chung-Jung Chiu, et al. Prog Retin Eye Res. 2011 January;30(1):18-53.</div><p class="links"><a class="dblinks" href="/pubmed/20868767" ref="ordinalpos=5080&amp;ncbi_uid=6072629&amp;link_uid=20868767">Citation</a><a class="dblinks" href="/pmc/articles/PMC4005015/" ref="ordinalpos=5080&amp;ncbi_uid=6072629&amp;link_uid=PMC4005015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5081.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3807281/figure/F8/" ref="ordinalpos=5081&amp;ncbi_uid=5995063&amp;link_uid=PMC3807281" href="/pmc/articles/PMC3807281/"><img src="/pmc/articles/instance/3807281/bin/zjv9990981690008.gif" src-large="/pmc/articles/instance/3807281/bin/zjv9990981690008.jpg" alt="Fig 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3807281/" ref="ordinalpos=5081&amp;ncbi_uid=5995063&amp;link_uid=PMC3807281" image-link="/pmc/articles/PMC3807281/figure/F8/" class="imagepopup">Fig 8.  From: Soluble Interleukin-6 Receptor-Mediated Innate Immune Response to DNA and RNA Viruses. </a></p><div class="supp"><p class="details"><div>(A) The relative sIL6R and viral NP levels in throat swab samples were subjected to correlation analysis (<i>n</i> = 23). (B) Hypothetical model for COX-2-mediated sIL6R expression. Solid arrows represent <span class="highlight" style="background-color:">signaling</span> pathways identified in this study. Broken arrows indicate potential <span class="highlight" style="background-color:">signaling</span> pathways. Viruses (IAV, VSV, EV71, and HBV) induce sIL6R expression through the COX-2 <span class="highlight" style="background-color:">pathway</span>. The enhanced sIL6R activates type I IFN expression through the p28 <span class="highlight" style="background-color:">pathway</span>, leading to the activation of downstream effectors and the inhibition of viral replication. In addition, there are other potential <span class="highlight" style="background-color:">signaling</span> pathways that may regulate sIL6R-mediated antiviral function.</div></p></div><div class="aux"><div class="resc">Qing Wang, et al. J Virol. 2013 October;87(20):11244-11254.</div><p class="links"><a class="dblinks" href="/pubmed/23946454" ref="ordinalpos=5081&amp;ncbi_uid=5995063&amp;link_uid=23946454">Citation</a><a class="dblinks" href="/pmc/articles/PMC3807281/" ref="ordinalpos=5081&amp;ncbi_uid=5995063&amp;link_uid=PMC3807281">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5082.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3173392/figure/pone-0024361-g006/" ref="ordinalpos=5082&amp;ncbi_uid=3237436&amp;link_uid=PMC3173392" href="/pmc/articles/PMC3173392/"><img src="/pmc/articles/instance/3173392/bin/pone.0024361.g006.gif" src-large="/pmc/articles/instance/3173392/bin/pone.0024361.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3173392/" ref="ordinalpos=5082&amp;ncbi_uid=3237436&amp;link_uid=PMC3173392" image-link="/pmc/articles/PMC3173392/figure/pone-0024361-g006/" class="imagepopup">Figure 6. JNK <span class="highlight" style="background-color:">signaling</span> is responsible for cell death in GMR&gt;Aß42 background..  From: Activation of JNK <span class="highlight" style="background-color:">Signaling</span> Mediates Amyloid-ss-Dependent Cell Death. </a></p><div class="supp"><p class="details"><div>(A, A′) Misexpression of both P35 and <i>puc</i> along with Aß42 (GMR&gt;Aß42+P35<i>+puc</i>) results in strong rescue of cell death as evident from (A′) dramatically reduced TUNEL positive cells. However, the rescue of the phenotype was not significantly stronger than with blocking JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> alone (Figure 5G′). (B) Misexpression of Aß42 (GMR&gt;Aß42) in pupal retina showing cell death as evident from TUNEL positive cells (red channel). Blocking simultaneously both caspase-dependent cell death and caspase-independent JNK <span class="highlight" style="background-color:">signaling</span> mediated cell death in pupal retina (GMR&gt;Aß42+P35<i>+puc</i>) showed a strong rescue in (C, C′, C″) pupal retina and (D) adult eye as compared to (B) GMR&gt;Aß42 pupal retina, (Figure 3D) GMR&gt;Aß42 adult eye. The cell death is detected by TUNEL staining (red channel), which is (C′, C″) restricted to the periphery of the pupal retina. Note that dying cells on the periphery of the pupal retina corresponds to the programmed cell death as seen in the wild-type pupal retina too [25], [28]. (E) Quantification of the number of dying cells in eye imaginal discs based on TUNEL staining in different genetic combinations. The frequency of cell death in wild-type eye imaginal disc served as a control. Note that blocking JNK <span class="highlight" style="background-color:">signaling</span> (GMR&gt;Aß42+<i>puc</i>) or blocking JNK <span class="highlight" style="background-color:">signaling</span> along with caspase–dependent cell death (GMR&gt;Aß42+P35+<i>puc</i>) exhibit strong rescue of the neurodegenerative phenotype of GMR&gt;Aß42. This rescue is significant (**) as seen by calculation of P-values based on one-tailed <i>t</i>-test using Microsoft Excel 2007.</div></p></div><div class="aux"><div class="resc">Meghana Tare, et al. PLoS One. 2011;6(9):e24361.</div><p class="links"><a class="dblinks" href="/pubmed/21949710" ref="ordinalpos=5082&amp;ncbi_uid=3237436&amp;link_uid=21949710">Citation</a><a class="dblinks" href="/pmc/articles/PMC3173392/" ref="ordinalpos=5082&amp;ncbi_uid=3237436&amp;link_uid=PMC3173392">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5083.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4081080/figure/F3/" ref="ordinalpos=5083&amp;ncbi_uid=6359156&amp;link_uid=PMC4081080" href="/pmc/articles/PMC4081080/"><img src="/pmc/articles/instance/4081080/bin/gku461fig3.gif" src-large="/pmc/articles/instance/4081080/bin/gku461fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4081080/" ref="ordinalpos=5083&amp;ncbi_uid=6359156&amp;link_uid=PMC4081080" image-link="/pmc/articles/PMC4081080/figure/F3/" class="imagepopup">Figure 3. From: Lessons from miR-143/145: the importance of cell-type localization of miRNAs. </a></p><div class="supp"><p class="details"><div>miR-143/145 modulate <span class="highlight" style="background-color:">signaling</span> through the Ras-MAPK <span class="highlight" style="background-color:">pathway</span>. Extracellular mitogenic signals (triangles) leads to activation of Ras and the <span class="highlight" style="background-color:">signaling</span> through the Raf-MEK-ERK (MAPK) cascade to activate transcription factors such as Elk-1 and RREB1 resulting in proliferation and growth responses. Targets of miR-143 and miR-145 are shown in green and brown respectively. The miR-143/145 proximal promoter is negatively regulated by K-Ras-RREB1 feedback loop.</div></p></div><div class="aux"><div class="resc">Oliver A. Kent, et al. Nucleic Acids Res. 2014 August 1;42(12):7528-7538.</div><p class="links"><a class="dblinks" href="/pubmed/24875473" ref="ordinalpos=5083&amp;ncbi_uid=6359156&amp;link_uid=24875473">Citation</a><a class="dblinks" href="/pmc/articles/PMC4081080/" ref="ordinalpos=5083&amp;ncbi_uid=6359156&amp;link_uid=PMC4081080">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5084.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634480/figure/f7-ijms-14-05036/" ref="ordinalpos=5084&amp;ncbi_uid=4866055&amp;link_uid=PMC3634480" href="/pmc/articles/PMC3634480/"><img src="/pmc/articles/instance/3634480/bin/ijms-14-05036f7.gif" src-large="/pmc/articles/instance/3634480/bin/ijms-14-05036f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634480/" ref="ordinalpos=5084&amp;ncbi_uid=4866055&amp;link_uid=PMC3634480" image-link="/pmc/articles/PMC3634480/figure/f7-ijms-14-05036/" class="imagepopup">Figure 7.  From: Phospholipases of Mineralization Competent Cells and Matrix Vesicles: Roles in Physiological and Pathological Mineralizations. </a></p><div class="supp"><p class="details"><div>Calcium and strontium induce distinct intracelular calcium <span class="highlight" style="background-color:">signaling</span>. Upon stimulation by extracellular calcium, calcium-sensing receptor (CaR) activates PLC, which is responsible for the translocation of nuclear factor κB (NF-κB) from the cytoplasm to the nucleus in mature osteoclasts, in an IP3-dependent manner. Upon stimulation by extracellular strontium, CaR also activates PLC inducing the DAG-PKCβII <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and then promoting translocation of NF-κB from the cytoplasm to the nucleus in an IP3-independent manner. Taken from [465].</div></p></div><div class="aux"><div class="resc">Saida Mebarek, et al. Int J Mol Sci. 2013 March;14(3):5036-5129.</div><p class="links"><a class="dblinks" href="/pubmed/23455471" ref="ordinalpos=5084&amp;ncbi_uid=4866055&amp;link_uid=23455471">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634480/" ref="ordinalpos=5084&amp;ncbi_uid=4866055&amp;link_uid=PMC3634480">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5085.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2820199/figure/Fig4/" ref="ordinalpos=5085&amp;ncbi_uid=2224658&amp;link_uid=PMC2820199" href="/pmc/articles/PMC2820199/"><img src="/pmc/articles/instance/2820199/bin/12263_2009_152_Fig4_HTML.gif" src-large="/pmc/articles/instance/2820199/bin/12263_2009_152_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2820199/" ref="ordinalpos=5085&amp;ncbi_uid=2224658&amp;link_uid=PMC2820199" image-link="/pmc/articles/PMC2820199/figure/Fig4/" class="imagepopup">Fig. 4.  From: Chemopreventive potential of curcumin in prostate cancer. </a></p><div class="supp"><p class="details"><div>Curcumin’s targets intracellular components implicated in cell proliferation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in prostate cancer cells. Curcumin highly repressed the EGFR and VEGFR <span class="highlight" style="background-color:">signaling</span> pathways as well as cyclins implicated in cell cycle regulation in prostate cancer cells. <i>Arrow line</i> represented induction/activation and <i>blunt-ended line</i> represented repression/inactivation. This figure was generated by using ScienceSlides software with modifications</div></p></div><div class="aux"><div class="resc">Marie-Hélène Teiten, et al. Genes Nutr. 2010 March;5(1):61-74.</div><p class="links"><a class="dblinks" href="/pubmed/19806380" ref="ordinalpos=5085&amp;ncbi_uid=2224658&amp;link_uid=19806380">Citation</a><a class="dblinks" href="/pmc/articles/PMC2820199/" ref="ordinalpos=5085&amp;ncbi_uid=2224658&amp;link_uid=PMC2820199">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5086.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2873319/figure/F6/" ref="ordinalpos=5086&amp;ncbi_uid=2377546&amp;link_uid=PMC2873319" href="/pmc/articles/PMC2873319/"><img src="/pmc/articles/instance/2873319/bin/1752-0509-4-50-6.gif" src-large="/pmc/articles/instance/2873319/bin/1752-0509-4-50-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2873319/" ref="ordinalpos=5086&amp;ncbi_uid=2377546&amp;link_uid=PMC2873319" image-link="/pmc/articles/PMC2873319/figure/F6/" class="imagepopup">Figure 6.  From: A formal model for analyzing drug combination effects and its application in TNF-?-induced NF?B <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Case study of TNF-α-induced NFκB <span class="highlight" style="background-color:">pathway</span></b>. A. The kinetic model of TNF-α-induced NFκB <span class="highlight" style="background-color:">pathway</span>. The dashed box area indicates the simplified system. B. Biological schematic of TNF-α-induced NFκB <span class="highlight" style="background-color:">pathway</span>. Not all arrows represent direct physical interactions. Drugs used in this study and their key targets are highlighted. C. HUVECs were pretreated with three combinations at the same concentration 100 nM for 2 hours before being stimulated with 10 ng/ml TNF-α for 6 hours. Cells were lysed and output <span class="highlight" style="background-color:">signaling</span> protein was probed by western blotting. The level of β-actin was used as an internal control for normalization. The gray values are recorded at the bottom. D. Ratio of ICAM-1 to β-actin was determined with densitometry. C, control, T, 10 ng/ml TNF-α, G, Geldanamycin, A, Aldehyde, P, PS-1145.</div></p></div><div class="aux"><div class="resc">Han Yan, et al. BMC Syst Biol. 2010;4:50-50.</div><p class="links"><a class="dblinks" href="/pubmed/20416113" ref="ordinalpos=5086&amp;ncbi_uid=2377546&amp;link_uid=20416113">Citation</a><a class="dblinks" href="/pmc/articles/PMC2873319/" ref="ordinalpos=5086&amp;ncbi_uid=2377546&amp;link_uid=PMC2873319">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5087.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3177306/figure/F4/" ref="ordinalpos=5087&ampamp;ncbi_uid=3475476&amp;link_uid=PMC3177306" href="/pmc/articles/PMC3177306/"><img src="/pmc/articles/instance/3177306/bin/DEV066209F4.gif" src-large="/pmc/articles/instance/3177306/bin/DEV066209F4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3177306/" ref="ordinalpos=5087&amp;ncbi_uid=3475476&amp;link_uid=PMC3177306" image-link="/pmc/articles/PMC3177306/figure/F4/" class="imagepopup">Fig. 4. From: Maintaining embryonic stem cell pluripotency with Wnt <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Models for how Wnt <span class="highlight" style="background-color:">signaling</span> maintains pluripotency in ES cells.</b> (<b>A</b>) In the absence of Wnt <span class="highlight" style="background-color:">signaling</span>, β-catenin (βcat) is degraded, and TCF3 in complex with transcriptional co-repressors (CoR) constitutively represses Wnt target genes. (<b>B-E</b>) Upon Wnt <span class="highlight" style="background-color:">pathway</span> activation, several alternative models leading to pluripotency are possible: (B) stabilized β-catenin associates with OCT4 to activate OCT4-dependent transcription; (C) HIPK2 is activated by Wnt <span class="highlight" style="background-color:">signaling</span>, associates with β-catenin and phosphorylates TCF3; this phosphorylation results in the removal of TCF3 from target promoters, leading to transcriptional derepression; (D) stabilized β-catenin associates with TCF3, causing the removal of the co-repressors resulting in target derepression (this model predicts that TCF3 is still bound to the promoter but no longer represses its gene targets); and (E) the TCF switch model, in which TCF3 repressor is replaced by TCF1 activator, leading to target activation and pluripotency. OCT4, octamer-binding transcription factor 4.</div></p></div><div class="aux"><div class="resc">Sergei Y. Sokol. Development. 2011 October 15;138(20):4341-4350.</div><p class="links"><a class="dblinks" href="/pubmed/21903672" ref="ordinalpos=5087&amp;ncbi_uid=3475476&amp;link_uid=21903672">Citation</a><a class="dblinks" href="/pmc/articles/PMC3177306/" ref="ordinalpos=5087&amp;ncbi_uid=3475476&amp;link_uid=PMC3177306">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5088.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3018149/figure/F4/" ref="ordinalpos=5088&amp;ncbi_uid=3403251&amp;link_uid=PMC3018149" href="/pmc/articles/PMC3018149/"><img src="/pmc/articles/instance/3018149/bin/nihms-241911-f0004.gif" src-large="/pmc/articles/instance/3018149/bin/nihms-241911-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3018149/" ref="ordinalpos=5088&amp;ncbi_uid=3403251&amp;link_uid=PMC3018149" image-link="/pmc/articles/PMC3018149/figure/F4/" class="imagepopup">Figure 4.  From: Up Regulation of DLX5 Promotes Ovarian Cancer Cell Proliferation by Enhancing IRS-2-AKT <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><i>DLX5</i> knockdown attenuates AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by down regulating upstream effectors. (A) AKT <span class="highlight" style="background-color:">signaling</span> is compromised by <i>DLX5</i> knockdown in IGR-OV1 and OVCAR4 cells. Expression of p-PDK1/PDK1, p-AKT/AKT, p-GSK3, p-p70S6K and p-S6 were analyzed by Western blotting. (B) <i>DLX5</i> knockdown reduces MET and IRS-2 expression. Levels of membrane receptor kinases MET, EGFR, HER2 and IGFR1, and adaptor proteins IRS-1 and IRS-2 were determined by immunoblotting. (C) <i>DLX5</i> knockdown reduces MET/HGFR and IGFR <span class="highlight" style="background-color:">signaling</span> to AKT. IGR-OV1 and OVCAR4 cells with <i>DLX5</i> knockdown were starved and treated with 10 ng/ml HGF or 50 ng/ml IGFI for 10 min. p-AKT/AKT levels were analyzed by immunoblotting. (D) myr-AKT protects cells from <i>DLX5</i> knockdown-induced inhibition of cell proliferation. Ovarian cancer cells were co-transduced by viruses containing <i>DLX5</i> shRNA and myr-AKT, and cell viability/proliferation was analyzed by WST-1 assay 5 days after infection. (E) Cyclin levels and Akt <span class="highlight" style="background-color:">signaling</span> diminished by <i>DLX5</i> knockdown are restored by expression of myr-AKT.</div></p></div><div class="aux"><div class="resc">Yinfei Tan, et al. Cancer Res. ;70(22):9197-9206.</div><p class="links"><a class="dblinks" href="/pubmed/21045156" ref="ordinalpos=5088&amp;ncbi_uid=3403251&amp;link_uid=21045156">Citation</a><a class="dblinks" href="/pmc/articles/PMC3018149/" ref="ordinalpos=5088&amp;ncbi_uid=3403251&amp;link_uid=PMC3018149">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5089.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3553273/figure/F2/" ref="ordinalpos=5089&amp;ncbi_uid=5756469&amp;link_uid=PMC3553273" href="/pmc/articles/PMC3553273/"><img src="/pmc/articles/instance/3553273/bin/nihms405864f2.gif" src-large="/pmc/articles/instance/3553273/bin/nihms405864f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3553273/" ref="ordinalpos=5089&amp;ncbi_uid=5756469&amp;link_uid=PMC3553273" image-link="/pmc/articles/PMC3553273/figure/F2/" class="imagepopup">Figure 2. RAF→RAS <span class="highlight" style="background-color:">signaling</span>.  From: OVARIAN CANCER: MAKING ITS OWN RULES--AGAIN. </a></p><div class="supp"><p class="details"><div>Receptor tyrosine kinases (RTK) signal through Srchomology-2 domains to Src. It in turn activates the <span class="highlight" style="background-color:">pathway</span> to KRAS, RAF, to ultimately activate the mitogen-activated kinase, MAPK, <span class="highlight" style="background-color:">pathway</span> via MEK to ERK. An activating mutation in KRAS (shown in red), such as the codon 12 and 13 mutations described in low grade serous ovarian cancer, can drive downstream activation of the MAPK <span class="highlight" style="background-color:">pathway</span> without upstream the otherwise needed upstream stimulation by RTK. Similarly, an activating mutation in BRAF (shown in green), such as V600E</div></p></div><div class="aux"><div class="resc">Elise C. Kohn, et al. Cancer. ;119(3):474-476.</div><p class="links"><a class="dblinks" href="/pubmed/23233093" ref="ordinalpos=5089&amp;ncbi_uid=5756469&amp;link_uid=23233093">Citation</a><a class="dblinks" href="/pmc/articles/PMC3553273/" ref="ordinalpos=5089&amp;ncbi_uid=5756469&amp;link_uid=PMC3553273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5090.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3324360/figure/f5/" ref="ordinalpos=5090&amp;ncbi_uid=3904418&amp;link_uid=PMC3324360" href="/pmc/articles/PMC3324360/"><img src="/pmc/articles/instance/3324360/bin/mv-v18-730-f5.gif" src-large="/pmc/articles/instance/3324360/bin/mv-v18-730-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3324360/" ref="ordinalpos=5090&amp;ncbi_uid=3904418&amp;link_uid=PMC3324360" image-link="/pmc/articles/PMC3324360/figure/f5/" class="imagepopup">Figure 5.  From: Microarray analysis of gene expression in West Nile virus-infected human retinal pigment epithelium. </a></p><div class="supp"><p class="details"><div>Simplified TGF-β <span class="highlight" style="background-color:">pathway</span>. Several genes in this <span class="highlight" style="background-color:">pathway</span> are included in our analysis (<i>TGFβ 1/2</i>, <i>BAMBI</i>, <i>ATF3</i>, <i>ID1</i>), as well as associated genes that were not analyzed (TGF-β RI/II, SMAD3). TGF-β1 or 2 binds to the TGF-β RI and RII receptor complex, and results in signal transduction. However, if the TGFβ ligand binds to a BAMBI/TGF-β RI complex, this signal is not transduced. TGF-β <span class="highlight" style="background-color:">signaling</span> can lead to the binding of SMAD3 with ATF3. This complex can go on to repress the transcription of <i>ID1</i>. Independently of the TGF-β <span class="highlight" style="background-color:">pathway</span>, DDIT3 protein can inhibit the transcription of <i>ATF3</i>, while ATF3 can also suppress the transcription of <i>DDIT3</i> (insert).</div></p></div><div class="aux"><div class="resc">Luis Munoz-Erazo, et al. Mol Vis. 2012;18:730-743.</div><p class="links"><a class="dblinks" href="/pubmed/22509103" ref="ordinalpos=5090&amp;ncbi_uid=3904418&amp;link_uid=22509103">Citation</a><a class="dblinks" href="/pmc/articles/PMC3324360/" ref="ordinalpos=5090&amp;ncbi_uid=3904418&amp;link_uid=PMC3324360">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5091.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2724489/figure/F9/" ref="ordinalpos=5091&amp;ncbi_uid=1915425&amp;link_uid=PMC2724489" href="/pmc/articles/PMC2724489/"><img src="/pmc/articles/instance/2724489/bin/1752-0509-3-73-9.gif" src-large="/pmc/articles/instance/2724489/bin/1752-0509-3-73-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2724489/" ref="ordinalpos=5091&amp;ncbi_uid=1915425&amp;link_uid=PMC2724489" image-link="/pmc/articles/PMC2724489/figure/F9/" class="imagepopup">Figure 9.  From: An integrative ChIP-chip and gene expression profiling to model SMAD regulatory modules. </a></p><div class="supp"><p class="details"><div><b>Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis of SMAD targets: (A)</b> A graphical representation of overlapping <span class="highlight" style="background-color:">signaling</span> pathways detected in differentially expressed genes and SMAD-target genes. Wnt/β-catenin was found to be significant. (<b>B</b>) The result of Ingenuity analyses of Affymerix gene expression and SMAD-predicted datasets. There are total nine significant <span class="highlight" style="background-color:">signaling</span> pathways (shown in bar graphs) for gene expression data, and six significant <span class="highlight" style="background-color:">signaling</span> pathways for SMAD-predicted molecules (shown by asterisks). There is a major functional overlap (about 70% or 6/9) between gene expression and SMAD-predicted data. Note that the p values were calculated by Ingenuity algorithm  per each <span class="highlight" style="background-color:">pathway</span> within Affymerix gene expression dataset and minus log p values are shown. The minus log p values for SMAD-predicted molecules are not shown.</div></p></div><div class="aux"><div class="resc">Huaxia Qin, et al. BMC Syst Biol. 2009;3:73-73.</div><p class="links"><a class="dblinks" href="/pubmed/19615063" ref="ordinalpos=5091&amp;ncbi_uid=1915425&amp;link_uid=19615063">Citation</a><a class="dblinks" href="/pmc/articles/PMC2724489/" ref="ordinalpos=5091&amp;ncbi_uid=1915425&amp;link_uid=PMC2724489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5092.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4148086/figure/F1/" ref="ordinalpos=5092&amp;ncbi_uid=6543585&amp;link_uid=PMC4148086" href="/pmc/articles/PMC4148086/"><img src="/pmc/articles/instance/4148086/bin/oncotarget-05-4581-g001.gif" src-large="/pmc/articles/instance/4148086/bin/oncotarget-05-4581-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4148086/" ref="ordinalpos=5092&amp;ncbi_uid=6543585&amp;link_uid=PMC4148086" image-link="/pmc/articles/PMC4148086/figure/F1/" class="imagepopup">Figure 1. Schematic representation of the basic structure of CIP2A (A) and the mode of interaction of CIP2A with c-MYC and PP2A (B).  From: Oncogenic nexus of cancerous inhibitor of protein phosphatase 2A (CIP2A): An oncoprotein with many hands. </a></p><div class="supp"><p class="details"><div>The “oncogenic nexus” of CIP2A is primarily constituted of the interactive functional network between oncogenic transcription factors, tumor suppressors and different <span class="highlight" style="background-color:">signaling</span> components of the PI3K-mTOR <span class="highlight" style="background-color:">pathway</span>, the RAS-MEK-ERK <span class="highlight" style="background-color:">pathway</span> and the Wnt-beta-catenin <span class="highlight" style="background-color:">pathway</span>. Schematic representation of the major interacting partners of “oncogenic nexus” involving CIP2A protein has been compiled from <i>Puustinen, P., 2014; Perrotti, D. and Neviani, P., 2014; Li, Y., 2013; Khanna, A., 2013; Laine, A., 2013; Niemela, M., 2012; Westermarck, J. and Hahn, W. C. 2008; Junttila, M. R. and Westermarck, J. 2008; Junttila, M. R. et al., 2007.</i> Bi-directional blue arrows indicate interactions between two entities; uni-directional black arrows indicate a positive influence of one on the other entity; blocking red lines indicate a negative influence of one on the other entity. Transcription factors are color coded in pink. <span class="highlight" style="background-color:">Signaling</span> molecules of the RAS-MAPK-ERK <span class="highlight" style="background-color:">pathway</span> and the PI3K-AKT-mTOR <span class="highlight" style="background-color:">pathway</span> are coded in two different shades of blue.</div></p></div><div class="aux"><div class="resc">Pradip De, et al. Oncotarget. 2014 July;5(13):4581-4602.</div><p class="links"><a class="dblinks" href="/pubmed/25015035" ref="ordinalpos=5092&amp;ncbi_uid=6543585&amp;link_uid=25015035">Citation</a><a class="dblinks" href="/pmc/articles/PMC4148086/" ref="ordinalpos=5092&amp;ncbi_uid=6543585&amp;link_uid=PMC4148086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5093.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3666883/figure/F4/" ref="ordinalpos=5093&amp;ncbi_uid=5158000&amp;link_uid=PMC3666883" href="/pmc/articles/PMC3666883/"><img src="/pmc/articles/instance/3666883/bin/zmb9991098360004.gif" src-large="/pmc/articles/instance/3666883/bin/zmb9991098360004.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3666883/" ref="ordinalpos=5093&amp;ncbi_uid=5158000&amp;link_uid=PMC3666883" image-link="/pmc/articles/PMC3666883/figure/F4/" class="imagepopup">Fig 4.  From: Sonic hedgehog-Dependent Induction of MicroRNA 31 and MicroRNA 150 Regulates Mycobacterium bovis BCG-Driven Toll-Like Receptor 2 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>TNF-α is a critical mediator of M. bovis BCG-trigged SHH <span class="highlight" style="background-color:">signaling</span>. (A) Macrophages were treated <i>in vitro</i> with anti-TNF-α antibody (10 μg) 6 h prior to M. bovis BCG infection, and SHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components were monitored by quantitative real-time RT-PCR. Data represent means ± SEs (<i>n</i> = 3). *, <i>P</i> &lt; 0.05 versus control IgG-treated M. bovis BCG-infected macrophages. (B) Anti-TNF-α antibody (0.1 mg) was injected i.p. into each mouse for 12 h prior to i.p. infection with M. bovis BCG. Peritoneal macrophages isolated were analyzed for canonical SHH <span class="highlight" style="background-color:">signaling</span> molecules (mean ± SE, <i>n</i> = 3). *, <i>P</i> &lt; 0.05 versus control IgG-injected M. bovis BCG-infected mice. (C) The activation status of SHH <span class="highlight" style="background-color:">signaling</span> was ascertained by nuclear translocation of GLI1 with or without <i>in vitro</i> treatment of anti-TNF-α antibody (10 μg). Blots are representative of 3 independent experiments. (D and E) Macrophages were treated <i>in vitro</i> with 5 μg (D) or mice were injected with 0.5 mg (E) of TNF-α antagonist, and M. bovis BCG-induced SHH <span class="highlight" style="background-color:">signaling</span> induction in peritoneal macrophages was monitored using quantitative real-time RT-PCR for <i>Shh</i>, <i>Gli1</i>, and <i>Ptch1</i>. Data represent means ± SEs (<i>n</i> = 3). *, <i>P</i> &lt; 0.05 versus control IgG-treated M. bovis BCG-infected macrophages or mice. (F) GLI1 nuclear translocation was assayed in macrophages treated with TNF-α antagonist with immunoblotting. DMSO was utilized as the vehicle control. (G and H) M. bovis BCG-induced SHH <span class="highlight" style="background-color:">signaling</span> was analyzed in bone marrow-derived macrophages from TNF-α-null mice or WT mice using quantitative real-time RT-PCR (G) or immunoblotting (H). Data represent means ± SEs (<i>n</i> = 4). *, <i>P</i> &lt; 0.05 versus M. bovis BCG-infected WT bone marrow-derived macrophages. Blots are representative of 3 independent experiments.</div></p></div><div class="aux"><div class="resc">Devram Sampat Ghorpade, et al. Mol Cell Biol. 2013 February;33(3):543-556.</div><p class="links"><a class="dblinks" href="/pubmed/23166298" ref="ordinalpos=5093&amp;ncbi_uid=5158000&amp;link_uid=23166298">Citation</a><a class="dblinks" href="/pmc/articles/PMC3666883/" ref="ordinalpos=5093&amp;ncbi_uid=5158000&amp;link_uid=PMC3666883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5094.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3306320/figure/pone-0032455-g005/" ref="ordinalpos=5094&amp;ncbi_uid=3866796&amp;link_uid=PMC3306320" href="/pmc/articles/PMC3306320/"><img src="/pmc/articles/instance/3306320/bin/pone.0032455.g005.gif" src-large="/pmc/articles/instance/3306320/bin/pone.0032455.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3306320/" ref="ordinalpos=5094&amp;ncbi_uid=3866796&amp;link_uid=PMC3306320" image-link="/pmc/articles/PMC3306320/figure/pone-0032455-g005/" class="imagepopup">Figure 5. Upper panel: pattern of curve of lactation, milk lactose yield, and the direction of impact as calculated by the Dynamic Impact Approach of KEGG ‘Galactose metabolism’ <span class="highlight" style="background-color:">pathway</span> and GO Biological process ‘Lactose biosynthetic process’..  From: A Novel Dynamic Impact Approach (DIA) for Functional Analysis of Time-Course Omics Studies: Validation Using the Bovine Mammary Transcriptome. </a></p><div class="supp"><p class="details"><div>Lower panel: pattern of milk fat yield and direction of the impact of KEGG ‘PPAR <span class="highlight" style="background-color:">signaling</span>’ <span class="highlight" style="background-color:">pathway</span>. The correlation was positive between the curve of lactation and lactose yield with ‘Galactose metabolism’ (r = 0.87, P = 0.02 and r = 0.81, P = 0.09, respectively) and ‘Lactose biosynthetic process’ (r = 0.82, P = 0.05 and r = 0.88, P = 0.05, respectively). The correlation was positive (r = 0.56) but not significant (P = 0.32) between milk fat yield and ‘PPAR <span class="highlight" style="background-color:">signaling</span>’.</div></p></div><div class="aux"><div class="resc">Massimo Bionaz, et al. PLoS One. 2012;7(3):e32455.</div><p class="links"><a class="dblinks" href="/pubmed/22438877" ref="ordinalpos=5094&amp;ncbi_uid=3866796&amp;link_uid=22438877">Citation</a><a class="dblinks" href="/pmc/articles/PMC3306320/" ref="ordinalpos=5094&amp;ncbi_uid=3866796&amp;link_uid=PMC3306320">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5095.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3371699/figure/F4/" ref="ordinalpos=5095&amp;ncbi_uid=4416046&amp;link_uid=PMC3371699" href="/pmc/articles/PMC3371699/"><img src="/pmc/articles/instance/3371699/bin/1163fig4.gif" src-large="/pmc/articles/instance/3371699/bin/1163fig4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3371699/" ref="ordinalpos=5095&amp;ncbi_uid=4416046&amp;link_uid=PMC3371699" image-link="/pmc/articles/PMC3371699/figure/F4/" class="imagepopup">Figure 4. From: Identification of microRNA-regulated gene networks by expression analysis of target genes. </a></p><div class="supp"><p class="details"><div>miR-519d, miR-190, and miR-340 in the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Schematic of the network of interactions between genes and proteins involved in TGFβ <span class="highlight" style="background-color:">signaling</span>. Putative targets of miR-519d (purple), miR-190 (orange), and miR-340 (yellow) are indicated. (Double vertical lines) The cell membrane; (dotted vertical line) the nuclear membrane; (dotted horizontal lines) links to other cellular pathways.</div></p></div><div class="aux"><div class="resc">Vincenzo Alessandro Gennarino, et al. Genome Res. 2012 June;22(6):1163-1172.</div><p class="links"><a class="dblinks" href="/pubmed/22345618" ref="ordinalpos=5095&amp;ncbi_uid=4416046&amp;link_uid=22345618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3371699/" ref="ordinalpos=5095&amp;ncbi_uid=4416046&amp;link_uid=PMC3371699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5096.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052840/figure/F2/" ref="ordinalpos=5096&amp;ncbi_uid=6237442&amp;link_uid=PMC4052840" href="/pmc/articles/PMC4052840/"><img src="/pmc/articles/instance/4052840/bin/nihms-397790-f0002.gif" src-large="/pmc/articles/instance/4052840/bin/nihms-397790-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052840/" ref="ordinalpos=5096&amp;ncbi_uid=6237442&amp;link_uid=PMC4052840" image-link="/pmc/articles/PMC4052840/figure/F2/" class="imagepopup">Fig. 2.  From: Emerging therapeutic strategies for Epstein-Barr virus+ post-transplant lymphoproliferative disorder. </a></p><div class="supp"><p class="details"><div>Therapeutics in development that target survival pathways activated by LMP1 and LMP2a. Both LMP1 and LMP2a activate members of the SFK (Fyn and Lyn, respectively) upstream of Syk activation. Syk then activates the PI3K/Akt/mTor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Small molecule inhibitors of these <span class="highlight" style="background-color:">signaling</span> molecules that are currently in preclinical or clinical development in other malignancies are listed next to their described target.</div></p></div><div class="aux"><div class="resc">Olivia Hatton, et al. Pediatr Transplant. ;16(3):220-229.</div><p class="links"><a class="dblinks" href="/pubmed/22353174" ref="ordinalpos=5096&amp;ncbi_uid=6237442&amp;link_uid=22353174">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052840/" ref="ordinalpos=5096&amp;ncbi_uid=6237442&amp;link_uid=PMC4052840">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5097.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3130800/figure/F1/" ref="ordinalpos=5097&amp;ncbi_uid=3553541&amp;link_uid=PMC3130800" href="/pmc/articles/PMC3130800/"><img src="/pmc/articles/instance/3130800/bin/nihms290595f1.gif" src-large="/pmc/articles/instance/3130800/bin/nihms290595f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3130800/" ref="ordinalpos=5097&amp;ncbi_uid=3553541&amp;link_uid=PMC3130800" image-link="/pmc/articles/PMC3130800/figure/F1/" class="imagepopup">Figure 1. DNA damage-induced ATM <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is impaired in DKO cells.  From: Chfr and RNF8 synergistically regulate ATM activation. </a></p><div class="supp"><p class="details"><div>(a) Domain architecture of mouse RNF8 and Chfr. (b and c) RNF8 and Chfr synergistically regulate ATM auto-phosphorylation and the ATM-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> following DNA damage. MEFs (b) and Thymocytes (c) were treated with or without IR, lysed and subjected to Western blot using the indicated antibodies. (d) Chfr and RNF8 synergistically regulate ATM kinase activity. <i>In vitro</i> kinase assay was separated by SDS-PAGE and subjected to Western blot. Coomassie Brilliant Blue (CBB) staining is shown below for loading control. (e) RNF8 and Chfr double deficiency abrogates G1/S checkpoint activation. BrdU assay was performed and mean values were calculated from three independent experiments. Error bar represent s.e.m. (f) DNA damage-induced G2/M checkpoint is abrogated in DKO cells. G2/M checkpoint assay was performed and mean values were calculated from three independent experiments. Error bar represent s.e.m.</div></p></div><div class="aux"><div class="resc">Jiaxue Wu, et al. Nat Struct Mol Biol. ;18(7):761-768.</div><p class="links"><a class="dblinks" href="/pubmed/21706008" ref="ordinalpos=5097&amp;ncbi_uid=3553541&amp;link_uid=21706008">Citation</a><a class="dblinks" href="/pmc/articles/PMC3130800/" ref="ordinalpos=5097&amp;ncbi_uid=3553541&amp;link_uid=PMC3130800">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5098.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2842492/figure/bire-82-04-02-f03/" ref="ordinalpos=5098&amp;ncbi_uid=2854002&amp;link_uid=PMC2842492" href="/pmc/articles/PMC2842492/"><img src="/pmc/articles/instance/2842492/bin/bire-82-04-02-f03.gif" src-large="/pmc/articles/instance/2842492/bin/bire-82-04-02-f03.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2842492/" ref="ordinalpos=5098&amp;ncbi_uid=2854002&amp;link_uid=PMC2842492" image-link="/pmc/articles/PMC2842492/figure/bire-82-04-02-f03/" class="imagepopup">FIG. 3. From: Genomic Profiling of MicroRNAs and Messenger RNAs Reveals Hormonal Regulation in MicroRNA Expression in Human Endometrium. </a></p><div class="supp"><p class="details"><div>Transcripts mediating cell cycle regulation are differentially regulated between the late proliferative-phase vs. midsecretory-phase endometrial epithelium. Differentially expressed transcripts were subjected to bioinformatics analysis using IPA 7.6. <b>A</b>) Most significantly enriched groups relating to molecular and cellular functions. <b>B</b>) Most significantly enriched canonical <span class="highlight" style="background-color:">signaling</span> pathways. The total number of genes in a <span class="highlight" style="background-color:">pathway</span> is boldfaced. For each <span class="highlight" style="background-color:">pathway</span>, the percentage of genes that are of decreased (green) or increased (red) transcript abundance is indicated at the top. Portions of <span class="highlight" style="background-color:">pathway</span> genes that are not significantly changed are also shown (white). The yellow square for each <span class="highlight" style="background-color:">pathway</span> indicates –log(<i>P</i> value). The yellow arrow indicates <i>P</i> = 0.05.</div></p></div><div class="aux"><div class="resc">Satu Kuokkanen, et al. Biol Reprod. 2010 April;82(4):791-801.</div><p class="links"><a class="dblinks" href="/pubmed/19864316" ref="ordinalpos=5098&amp;ncbi_uid=2854002&amp;link_uid=19864316">Citation</a><a class="dblinks" href="/pmc/articles/PMC2842492/" ref="ordinalpos=5098&amp;ncbi_uid=2854002&amp;link_uid=PMC2842492">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5099.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3471537/figure/F4/" ref="ordinalpos=5099&amp;ncbi_uid=4296313&amp;link_uid=PMC3471537" href="/pmc/articles/PMC3471537/"><img src="/pmc/articles/instance/3471537/bin/nihms406899f4.gif" src-large="/pmc/articles/instance/3471537/bin/nihms406899f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3471537/" ref="ordinalpos=5099&amp;ncbi_uid=4296313&amp;link_uid=PMC3471537" image-link="/pmc/articles/PMC3471537/figure/F4/" class="imagepopup">Fig. 4.  From: Resistance to Trastuzumab in Breast Cancer. </a></p><div class="supp"><p class="details"><div>General mechanisms of resistance to trastuzumab: presence of upregulation of HER2 downstream <span class="highlight" style="background-color:">signaling</span> pathways. PTEN is a tumor suppressor. Trastuzumab binding stabilizes and activates PTEN and consequently down-regulates the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (39). When PTEN function is lost, PI3K remains constitutively active regardless of binding of trastuzumab to HER2. PTEN loss correlates with clinical unresponsiveness to trastuzumab treatment. Genomic aberrations in the PI3K <span class="highlight" style="background-color:">pathway</span> are a common event in a variety of cancer types (127). Multiple components of this <span class="highlight" style="background-color:">pathway</span> are affected by germline or somatic mutation, amplification, rearrangement, methylation, overexpression, and aberrant splicing, but only in a few instances <i>PIK3CA</i> and <i>PTEN</i> mutations are seen simultaneously (86). Genomic aberrations in the PI3K <span class="highlight" style="background-color:">pathway</span> produce constitutive activation of the <span class="highlight" style="background-color:">pathway</span>, which will signal downstream to the nucleus regardless of trastuzumab binding to HER2. This is the case with activating mutations of <i>PIK3R1</i> and <i>PIK3CA</i>, encoding genes for PI3K p85α and p110α, respectively. Increased Akt kinase activity and <i>PDPK1</i> overexpression have also been implicated with trastuzumab resistance.</div></p></div><div class="aux"><div class="resc">Paula R. Pohlmann, et al. Clin Cancer Res. 2009 December 15;15(24):7479-7491.</div><p class="links"><a class="dblinks" href="/pubmed/20008848" ref="ordinalpos=5099&amp;ncbi_uid=4296313&amp;link_uid=20008848">Citation</a><a class="dblinks" href="/pmc/articles/PMC3471537/" ref="ordinalpos=5099&amp;ncbi_uid=4296313&amp;link_uid=PMC3471537">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5100.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2886156/figure/F1/" ref="ordinalpos=5100&amp;ncbi_uid=3049041&amp;link_uid=PMC2886156" href="/pmc/articles/PMC2886156/"><img src="/pmc/articles/instance/2886156/bin/nihms195114f1.gif" src-large="/pmc/articles/instance/2886156/bin/nihms195114f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2886156/" ref="ordinalpos=5100&amp;ncbi_uid=3049041&amp;link_uid=PMC2886156" image-link="/pmc/articles/PMC2886156/figure/F1/" class="imagepopup">Figure 1. Microarray analysis reveals that WNT5A is strikingly upregulated, and WIF1 down-regulated in the vast majority of lesional skin samples.  From: Evidence for altered Wnt <span class="highlight" style="background-color:">signaling</span> in psoriatic skin. </a></p><div class="supp"><p class="details"><div>This gene expression heat map image used transcripts from 58 paired lesional and uninvolved psoriasis and 64 normal skin samples and displays Wnt <span class="highlight" style="background-color:">pathway</span> members, regulators of Wnt <span class="highlight" style="background-color:">signaling</span>, and regulators of growth, proliferation and transcription. In addition genes involved in protein kinase activity, protein phosphatases, protein ubiquitination and other genes related to Wnt <span class="highlight" style="background-color:">signaling</span> are shown. Color key: red, increased expression, blue, decreased expression, as indicated in the color key and histogram (inset).</div></p></div><div class="aux"><div class="resc">JE Gudjonsson, et al. J Invest Dermatol. ;130(7):1849-1859.</div><p class="links"><a class="dblinks" href="/pubmed/20376066" ref="ordinalpos=5100&amp;ncbi_uid=3049041&amp;link_uid=20376066">Citation</a><a class="dblinks" href="/pmc/articles/PMC2886156/" ref="ordinalpos=5100&amp;ncbi_uid=3049041&amp;link_uid=PMC2886156">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="50" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="51" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="52" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:52:09-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB5960543142C1000000000072CFC6;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB5960543142C1000000000072CFC6" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>