
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="18" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB2A2253FF7F01000000000037020A" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="17" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="18" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="19" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="18" /></div>
            <div><h2 class="result_count">Results: 1701 to 1800 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1701.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4068084/figure/fig6/" ref="ordinalpos=1701&amp;ncbi_uid=6349911&amp;link_uid=PMC4068084" href="/pmc/articles/PMC4068084/"><img src="/pmc/articles/instance/4068084/bin/ECAM2014-484269.006.gif" src-large="/pmc/articles/instance/4068084/bin/ECAM2014-484269.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4068084/" ref="ordinalpos=1701&amp;ncbi_uid=6349911&amp;link_uid=PMC4068084" image-link="/pmc/articles/PMC4068084/figure/fig6/" class="imagepopup">Figure 6.  From: Paeonol Inhibits Proliferation of Vascular Smooth Muscle Cells Stimulated by High Glucose via Ras-Raf-ERK1/2 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Coculture Model. </a></p><div class="supp"><p class="details"><div>Effect of paeonol on Ras-Raf-ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in VSMCs in the coculture model. VECs were pretreated with various concentrations of Pae for 24 h before being incubated with HG (final concentration of 35.5 mM) for another 48 h. Then VSMCs were inoculated into the top of the Transwell plate and cocultured with VECs for another 24 h. Protein concentrations were analyzed by Western blot. <sup>##</sup><i>P</i> &lt; 0.01 versus NG (5.5 mM) coculture group; *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01 versus HG (35.5 mM) coculture group.</div></p></div><div class="aux"><div class="resc">Junjun Chen, et al. Evid Based Complement Alternat Med. 2014;2014:484269.</div><p class="links"><a class="dblinks" href="/pubmed/25002903" ref="ordinalpos=1701&amp;ncbi_uid=6349911&amp;link_uid=25002903">Citation</a><a class="dblinks" href="/pmc/articles/PMC4068084/" ref="ordinalpos=1701&amp;ncbi_uid=6349911&amp;link_uid=PMC4068084">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1702.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3397898/figure/F6/" ref="ordinalpos=1702&amp;ncbi_uid=5087648&amp;link_uid=PMC3397898" href="/pmc/articles/PMC3397898/"><img src="/pmc/articles/instance/3397898/bin/zbc0301211300006.gif" src-large="/pmc/articles/instance/3397898/bin/zbc0301211300006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3397898/" ref="ordinalpos=1702&amp;ncbi_uid=5087648&amp;link_uid=PMC3397898" image-link="/pmc/articles/PMC3397898/figure/F6/" class="imagepopup">FIGURE 6. From: The Epac-Rap1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Controls cAMP-mediated Exocytosis of Weibel-Palade Bodies in Endothelial Cells. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> pathways that regulate cAMP-mediated WPB exocytosis.</b> Stimulation of the β<sub>2</sub>-adrenergic receptor (β<i>2AR</i>) by epinephrine stimulates cAMP production by G<sub>s</sub>-activated adenylate cyclase (<i>AC</i>). Emanating from cAMP, a PKA-dependent <span class="highlight" style="background-color:">pathway</span> induces the activation of the small GTPase RalA by RalGDS, while simultaneously Epac1 activates Rap1. Activated RalA promotes PLD1 activity as well as assembly of the exocyst complex. In parallel, activated Rap1 facilitates WPB exocytosis, possibly by inducing cytoskeletal rearrangements.</div></p></div><div class="aux"><div class="resc">Kathinka W. E. M. van Hooren, et al. J Biol Chem. 2012 July 13;287(29):24713-24720.</div><p class="links"><a class="dblinks" href="/pubmed/22511766" ref="ordinalpos=1702&amp;ncbi_uid=5087648&amp;link_uid=22511766">Citation</a><a class="dblinks" href="/pmc/articles/PMC3397898/" ref="ordinalpos=1702&amp;ncbi_uid=5087648&amp;link_uid=PMC3397898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1703.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4297339/figure/fig05/" ref="ordinalpos=1703&amp;ncbi_uid=7103949&amp;link_uid=PMC4297339" href="/pmc/articles/PMC4297339/"><img src="/pmc/articles/instance/4297339/bin/ajtr0006-0714-f5.gif" src-large="/pmc/articles/instance/4297339/bin/ajtr0006-0714-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4297339/" ref="ordinalpos=1703&amp;ncbi_uid=7103949&amp;link_uid=PMC4297339" image-link="/pmc/articles/PMC4297339/figure/fig05/" class="imagepopup">Figure 5.  From: Role of PDGFR-β/PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PDGF-BB induced myocardial fibrosis in rats. </a></p><div class="supp"><p class="details"><div>mRNA expression of different factors in PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Con group, P group, IMA group and LY294002 group. A: No significant difference was observed between four group of PDGFRα, and the mRNA expression of PDGFRβin P group was significantly higher than other three groups. B: The mRNA expression of PI3K and Akt in P group was significantly higher than other three groups. C: The mRNA expression of Col I and Col III in P group was significantly higher than other three groups. *P &lt; 0.05 vs. CON; #P &lt; 0.05 vs. P group.</div></p></div><div class="aux"><div class="resc">Hai Fan, et al. Am J Transl Res. 2014;6(6):714-723.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1703&amp;ncbi_uid=7103949&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4297339/" ref="ordinalpos=1703&amp;ncbi_uid=7103949&amp;link_uid=PMC4297339">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1704.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3743028/figure/F12/" ref="ordinalpos=1704&amp;ncbi_uid=5198204&amp;link_uid=PMC3743028" href="/pmc/articles/PMC3743028/"><img src="/pmc/articles/instance/3743028/bin/emss-53157-f0012.gif" src-large="/pmc/articles/instance/3743028/bin/emss-53157-f0012.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3743028/" ref="ordinalpos=1704&amp;ncbi_uid=5198204&amp;link_uid=PMC3743028" image-link="/pmc/articles/PMC3743028/figure/F12/" class="imagepopup">Figure 12. Proposed NCC-mediated mechanism of increased salt sensitivity in hyperinsulinemic conditions.  From: PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activates the WNK-OSR1/SPAK-NCC Phosphorylation Cascade in Hyperinsulinemic db/db Mice. </a></p><div class="supp"><p class="details"><div>In mice fed a high-salt diet, the WNK-OSR1/SPAK-NCC phosphorylation cascade is down-regulated through a low plasma aldosterone level (left panel). However, this newly-described insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is independent of aldosterone, increases phosphorylation of Akt through PI3K and activates the WNK-OSR1/SPAK-NCC phosphorylation cascade. As a result, phosphorylation of NCC in hyperinsulinemic mice fed a high-salt diet is not down-regulated effectively (right panel).</div></p></div><div class="aux"><div class="resc">Hidenori Nishida, et al. Hypertension. ;60(4):981-990.</div><p class="links"><a class="dblinks" href="/pubmed/22949526" ref="ordinalpos=1704&amp;ncbi_uid=5198204&amp;link_uid=22949526">Citation</a><a class="dblinks" href="/pmc/articles/PMC3743028/" ref="ordinalpos=1704&amp;ncbi_uid=5198204&amp;link_uid=PMC3743028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1705.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1462967/figure/f7/" ref="ordinalpos=1705&amp;ncbi_uid=657139&amp;link_uid=PMC1462967" href="/pmc/articles/PMC1462967/"><img src="/pmc/articles/instance/1462967/bin/7601115f7.gif" src-large="/pmc/articles/instance/1462967/bin/7601115f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1462967/" ref="ordinalpos=1705&amp;ncbi_uid=657139&amp;link_uid=PMC1462967" image-link="/pmc/articles/PMC1462967/figure/f7/" class="imagepopup">Figure 7.  From: ATM-Chk2-p53 activation prevents tumorigenesis at an expense of organ homeostasis upon Brca1 deficiency. </a></p><div class="supp"><p class="details"><div>Model of the ATM–Chk2–p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> upon Brca1 mutation-associated premature aging and tumorigenesis. ATM–Chk2–p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> senses DNA damage/genomic instability and acts as a gatekeeper to eliminate mutations, but, as a side effect, it may also lead to premature aging.</div></p></div><div class="aux"><div class="resc">Liu Cao, et al. EMBO J. 2006 May 17;25(10):2167-2177.</div><p class="links"><a class="dblinks" href="/pubmed/16675955" ref="ordinalpos=1705&amp;ncbi_uid=657139&amp;link_uid=16675955">Citation</a><a class="dblinks" href="/pmc/articles/PMC1462967/" ref="ordinalpos=1705&amp;ncbi_uid=657139&amp;link_uid=PMC1462967">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1706.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3845839/figure/F4/" ref="ordinalpos=1706&amp;ncbi_uid=5954061&amp;link_uid=PMC3845839" href="/pmc/articles/PMC3845839/"><img src="/pmc/articles/instance/3845839/bin/nihms-529254-f0004.gif" src-large="/pmc/articles/instance/3845839/bin/nihms-529254-f0004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3845839/" ref="ordinalpos=1706&amp;ncbi_uid=5954061&amp;link_uid=PMC3845839" image-link="/pmc/articles/PMC3845839/figure/F4/" class="imagepopup">Fig. 4. Response to heregulin and ErbB3 abundance are key predictors for response to drugs targeting the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>.  From: Profiles of Basal and Stimulated Receptor <span class="highlight" style="background-color:">Signaling</span> Networks Predict Drug Response in Breast Cancer Lines. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Quality of the predictions for the drugs targeting the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> using their basal and <span class="highlight" style="background-color:">signaling</span> (based on maximal response) profiles. Non-significant predictions (NS) are set to zero. <b>(B)</b> Projection of cell lines according to the pERK response to heregulin and cumulative amounts of ErbB2 and ErbB3 overlaid with the <i>GI<sub>50</sub></i> value to GSK2126458 (red, resistant; blue, sensitive). <b>(C)</b> Quality of the prediction for drugs targeting the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> using a linear model with a single variable or the variables ErbB3 and ErbB2.</div></p></div><div class="aux"><div class="resc">Mario Niepel, et al. Sci Signal. ;6(294):10.1126/scisignal.2004379.</div><p class="links"><a class="dblinks" href="/pubmed/24065145" ref="ordinalpos=1706&amp;ncbi_uid=5954061&amp;link_uid=24065145">Citation</a><a class="dblinks" href="/pmc/articles/PMC3845839/" ref="ordinalpos=1706&amp;ncbi_uid=5954061&amp;link_uid=PMC3845839">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1707.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2630889/figure/F8/" ref="ordinalpos=1707&amp;ncbi_uid=1668505&amp;link_uid=PMC2630889" href="/pmc/articles/PMC2630889/"><img src="/pmc/articles/instance/2630889/bin/zee0120844440008.gif" src-large="/pmc/articles/instance/2630889/bin/zee0120844440008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2630889/" ref="ordinalpos=1707&amp;ncbi_uid=1668505&amp;link_uid=PMC2630889" image-link="/pmc/articles/PMC2630889/figure/F8/" class="imagepopup">Figure 8.  From: Characterization of Thyrotropin Receptor Antibody-Induced <span class="highlight" style="background-color:">Signaling</span> Cascades. </a></p><div class="supp"><p class="details"><div>Increasing concentrations (0.1, 1.0, and 10 μg/ml) of TSHR blocking (A) and neutral (B) antibodies over a 60-min incubation showed variable changes in <span class="highlight" style="background-color:">signaling</span> molecules as shown in these representative immunoblots. The data (representing one set of experiments repeated three times) showed different <span class="highlight" style="background-color:">signaling</span> cascades induced by blocking and neutral antibodies. For example, the B2 blocking antibody increased several <span class="highlight" style="background-color:">signaling</span> molecules, the most important of which appeared to be the c-Raf/ERK <span class="highlight" style="background-color:">pathway</span>, whereas the 7G neutral TSHR antibody suppressed <span class="highlight" style="background-color:">signaling</span> molecules such as c-Raf/ERK. T16, also a neutral antibody, showed increasing activities of many <span class="highlight" style="background-color:">signaling</span> molecules including Akt, PKC, CREB, and c-Raf/ERK/p90RSK. By contrast, an isotype control monoclonal antibody (IgG2) did not produce such dose-dependent activities.</div></p></div><div class="aux"><div class="resc">Syed A. Morshed, et al. Endocrinology. 2009 January;150(1):519-529.</div><p class="links"><a class="dblinks" href="/pubmed/18719020" ref="ordinalpos=1707&amp;ncbi_uid=1668505&amp;link_uid=18719020">Citation</a><a class="dblinks" href="/pmc/articles/PMC2630889/" ref="ordinalpos=1707&amp;ncbi_uid=1668505&amp;link_uid=PMC2630889">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1708.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2215764/figure/f7-mol14_5p0264/" ref="ordinalpos=1708&amp;ncbi_uid=1331735&amp;link_uid=PMC2215764" href="/pmc/articles/PMC2215764/"><img src="/pmc/articles/instance/2374735/bin/mol14_5p0264f7.gif" src-large="/pmc/articles/instance/2374735/bin/mol14_5p0264f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2215764/" ref="ordinalpos=1708&amp;ncbi_uid=1331735&amp;link_uid=PMC2215764" image-link="/pmc/articles/PMC2215764/figure/f7-mol14_5p0264/" class="imagepopup">Figure 7.  From: Differential Expression of Receptor Tyrosine Kinases (RTKs) and IGF-I <span class="highlight" style="background-color:">Pathway</span> Activation in Human Uterine Leiomyomas. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">pathway</span>.<br />In this model, IGF-I peptide binds to IGF-IR to induce tyrosine autophosphorylation and phosphorylation of its adaptor protein Shc. Phosphorylated Shc then is associated with the Grb2-mSOS complex to activate p21/Ras, which leads to proliferation by activation of the Ras/Raf/MAPK <span class="highlight" style="background-color:">pathway</span>. The IGF-I and IGF-IR complex also autophosphorylates its docking protein IRS-I, which, in turn, activates the survival PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. However, on the basis of our studies, this <span class="highlight" style="background-color:">pathway</span> appears not to be playing a major role in leiomyoma growth. IRS-I also may recruit Grb2, but it appears that the Shc-Grb2 <span class="highlight" style="background-color:">pathway</span> is the predominant activator of p21/Ras in IGF-IR <span class="highlight" style="background-color:">signaling</span> in UtLM cells and uterine leiomyoma tissue.</div></p></div><div class="aux"><div class="resc">Linda Yu, et al. Mol Med. 2008 May-Jun;14(5-6):264-275.</div><p class="links"><a class="dblinks" href="/pubmed/18231572" ref="ordinalpos=1708&amp;ncbi_uid=1331735&amp;link_uid=18231572">Citation</a><a class="dblinks" href="/pmc/articles/PMC2215764/" ref="ordinalpos=1708&amp;ncbi_uid=1331735&amp;link_uid=PMC2215764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1709.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3173343/figure/F7/" ref="ordinalpos=1709&amp;ncbi_uid=3221197&amp;link_uid=PMC3173343" href="/pmc/articles/PMC3173343/"><img src="/pmc/articles/instance/3173343/bin/1742-2094-8-104-7.gif" src-large="/pmc/articles/instance/3173343/bin/1742-2094-8-104-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3173343/" ref="ordinalpos=1709&amp;ncbi_uid=3221197&amp;link_uid=PMC3173343" image-link="/pmc/articles/PMC3173343/figure/F7/" class="imagepopup">Figure 7.  From: Cigarette smoke extract upregulates heme oxygenase-1 via PKC/NADPH oxidase/ROS/PDGFR/PI3K/Akt <span class="highlight" style="background-color:">pathway</span> in mouse brain endothelial cells. </a></p><div class="supp"><p class="details"><div><b>PPCSE induces HO-1 expression through PDGFR/PI3K/Akt cascade in a PC-PLC/PKCδ/NADPH oxidase/ROS-dependent manner in bEnd.3 cells</b>. Schematic diagram illustrating the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in PPCSE-induced HO-1 expression in bEnd.3 cells. PPCSE-induced HO-1 expression is mediated by PC-PLC/PKCδ/NADPH oxidase/ROS-dependent PDGFR/PI3K/Akt cascade.</div></p></div><div class="aux"><div class="resc">Ruey-Horng Shih, et al. J Neuroinflammation. 2011;8:104-104.</div><p class="links"><a class="dblinks" href="/pubmed/21861928" ref="ordinalpos=1709&amp;ncbi_uid=3221197&amp;link_uid=21861928">Citation</a><a class="dblinks" href="/pmc/articles/PMC3173343/" ref="ordinalpos=1709&amp;ncbi_uid=3221197&amp;link_uid=PMC3173343">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1710.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3210035/figure/F1/" ref="ordinalpos=1710&amp;ncbi_uid=3472635&amp;link_uid=PMC3210035" href="/pmc/articles/PMC3210035/"><img src="/pmc/articles/instance/3210035/bin/nihms301449f1.gif" src-large="/pmc/articles/instance/3210035/bin/nihms301449f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3210035/" ref="ordinalpos=1710&amp;ncbi_uid=3472635&amp;link_uid=PMC3210035" image-link="/pmc/articles/PMC3210035/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Pathway</span> discovery using microRNAs.  From: From microRNAs to targets: <span class="highlight" style="background-color:">pathway</span> discovery in cell fate transitions. </a></p><div class="supp"><p class="details"><div>Introduction of single miRNAs that regulate cell fate decisions followed by profiling can uncover numerous target genes that are key nodes of <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Deepa Subramanyam, et al. Curr Opin Genet Dev. ;21(4):498-503.</div><p class="links"><a class="dblinks" href="/pubmed/21636265" ref="ordinalpos=1710&amp;ncbi_uid=3472635&amp;link_uid=21636265">Citation</a><a class="dblinks" href="/pmc/articles/PMC3210035/" ref="ordinalpos=1710&amp;ncbi_uid=3472635&amp;link_uid=PMC3210035">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1711.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2575803/figure/F1/" ref="ordinalpos=1711&amp;ncbi_uid=1528226&amp;link_uid=PMC2575803" href="/pmc/articles/PMC2575803/"><img src="/pmc/articles/instance/2575803/bin/nihms69787f1.gif" src-large="/pmc/articles/instance/2575803/bin/nihms69787f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2575803/" ref="ordinalpos=1711&amp;ncbi_uid=1528226&amp;link_uid=PMC2575803" image-link="/pmc/articles/PMC2575803/figure/F1/" class="imagepopup">Figure 1.  From: Gene expression profiling identifies genes predictive of oral squamous cell carcinoma. </a></p><div class="supp"><p class="details"><div>Most prominently involved biological pathways in OSCC. Top: JAK/STAT <span class="highlight" style="background-color:">pathway</span>. Bottom: IFN-γ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, antigen-presenting <span class="highlight" style="background-color:">pathway</span>. Red denotes up-regulation and green denotes down-regulation of the gene. Ingenuity®Systems, version 4.0</div></p></div><div class="aux"><div class="resc">Chu Chen, et al. Cancer Epidemiol Biomarkers Prev. ;17(8):2152-2162.</div><p class="links"><a class="dblinks" href="/pubmed/18669583" ref="ordinalpos=1711&amp;ncbi_uid=1528226&amp;link_uid=18669583">Citation</a><a class="dblinks" href="/pmc/articles/PMC2575803/" ref="ordinalpos=1711&amp;ncbi_uid=1528226&amp;link_uid=PMC2575803">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1712.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3668835/figure/F4/" ref="ordinalpos=1712&amp;ncbi_uid=4966040&amp;link_uid=PMC3668835" href="/pmc/articles/PMC3668835/"><img src="/pmc/articles/instance/3668835/bin/JCI67963.f4.gif" src-large="/pmc/articles/instance/3668835/bin/JCI67963.f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3668835/" ref="ordinalpos=1712&amp;ncbi_uid=4966040&amp;link_uid=PMC3668835" image-link="/pmc/articles/PMC3668835/figure/F4/" class="imagepopup">Figure 4. <span class="highlight" style="background-color:">Signaling</span> connected with the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> and altered in SI-NETs..  From: The genomic landscape of small intestine neuroendocrine tumors. </a></p><div class="supp"><p class="details"><div>Cancer therapeutics target multiple proteins in connected <span class="highlight" style="background-color:">signaling</span> cascades. Candidate therapeutically actionable genome alterations were noted in 72% of SI-NET patients. FLT3, fms-like tyrosine kinase receptor 3.</div></p></div><div class="aux"><div class="resc">Michaela S. Banck, et al. J Clin Invest. 2013 June 3;123(6):2502-2508.</div><p class="links"><a class="dblinks" href="/pubmed/23676460" ref="ordinalpos=1712&amp;ncbi_uid=4966040&amp;link_uid=23676460">Citation</a><a class="dblinks" href="/pmc/articles/PMC3668835/" ref="ordinalpos=1712&amp;ncbi_uid=4966040&amp;link_uid=PMC3668835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1713.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3421063/figure/F2/" ref="ordinalpos=1713&amp;ncbi_uid=5248841&amp;link_uid=PMC3421063" href="/pmc/articles/PMC3421063/"><img src="/pmc/articles/instance/3421063/bin/nihms-398439-f0002.gif" src-large="/pmc/articles/instance/3421063/bin/nihms-398439-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3421063/" ref="ordinalpos=1713&amp;ncbi_uid=5248841&amp;link_uid=PMC3421063" image-link="/pmc/articles/PMC3421063/figure/F2/" class="imagepopup">Figure 2.  From: Management of Antiangiogenic Therapy-Induced Hypertension. </a></p><div class="supp"><p class="details"><div>Proposed mechanisms leading to VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitor-induced hypertension. a) Activation of VEGFR-2 by VEGF-A leads to subsequent activation of multiple pathways including phosphatidylinositol-3-kinase (PI3K)-AKT. PI3K-AKT phosphorylates and activates endothelial nitric oxide synthase (eNOS), increasing NO production. NO migrates to adjacent vascular smooth muscle cells, binds soluble guanylate cyclase (sGC) leading to cGMP generation and subsequent vasodilation mediated by cGMP-dependent kinases. When VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is inhibited, NO <span class="highlight" style="background-color:">pathway</span> is suppressed and ET-1 <span class="highlight" style="background-color:">pathway</span> is stimulated, promoting vasoconstriction and subsequent hypertension. The source of ET-1 is unknown. b) VEGF maintains capillary network integrity. When VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is inhibited, rarefaction, or reduction of the density of capillary beds, may occur.</div></p></div><div class="aux"><div class="resc">Nilka de Jesus-Gonzalez, et al. Hypertension. ;60(3):607-615.</div><p class="links"><a class="dblinks" href="/pubmed/22851729" ref="ordinalpos=1713&amp;ncbi_uid=5248841&amp;link_uid=22851729">Citation</a><a class="dblinks" href="/pmc/articles/PMC3421063/" ref="ordinalpos=1713&amp;ncbi_uid=5248841&amp;link_uid=PMC3421063">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1714.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2725405/figure/F6/" ref="ordinalpos=1714&amp;ncbi_uid=1923911&amp;link_uid=PMC2725405" href="/pmc/articles/PMC2725405/"><img src="/pmc/articles/instance/2725405/bin/nihms134183f6.gif" src-large="/pmc/articles/instance/2725405/bin/nihms134183f6.jpg" alt="FIGURE 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2725405/" ref="ordinalpos=1714&amp;ncbi_uid=1923911&amp;link_uid=PMC2725405" image-link="/pmc/articles/PMC2725405/figure/F6/" class="imagepopup">FIGURE 6.  From: Synoviocyte Innate Immune Responses: I. Differential Regulation of Interferon Responses and the JNK <span class="highlight" style="background-color:">Pathway</span> by MAPK Kinases. </a></p><div class="supp"><p class="details"><div>Regulation of IFN response genes by TLR3 <span class="highlight" style="background-color:">signaling</span>. MKK7 regulates TLR3-mediated gene expression by activating both IRF3 and c-Jun through a JNK-dependent <span class="highlight" style="background-color:">pathway</span>. In contrast, MKK4 only regulates c-Jun. Both AP-1 and ISRE are required for IFN-β gene expression.</div></p></div><div class="aux"><div class="resc">Toshio Yoshizawa, et al. J Immunol. ;181(5):3252-3258.</div><p class="links"><a class="dblinks" href="/pubmed/18713996" ref="ordinalpos=1714&amp;ncbi_uid=1923911&amp;link_uid=18713996">Citation</a><a class="dblinks" href="/pmc/articles/PMC2725405/" ref="ordinalpos=1714&amp;ncbi_uid=1923911&amp;link_uid=PMC2725405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1715.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2719801/figure/pone-0006575-g003/" ref="ordinalpos=1715&amp;ncbi_uid=2030410&amp;link_uid=PMC2719801" href="/pmc/articles/PMC2719801/"><img src="/pmc/articles/instance/2719801/bin/pone.0006575.g003.gif" src-large="/pmc/articles/instance/2719801/bin/pone.0006575.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2719801/" ref="ordinalpos=1715&amp;ncbi_uid=2030410&amp;link_uid=PMC2719801" image-link="/pmc/articles/PMC2719801/figure/pone-0006575-g003/" class="imagepopup">Figure 3. Regulation of insulin <span class="highlight" style="background-color:">signaling</span> in first degree relatives of people with type 2 diabetes..  From: Gene Expression in Skeletal Muscle Biopsies from People with Type 2 Diabetes and Relatives: Differential Regulation of Insulin <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>The insulin <span class="highlight" style="background-color:">signaling</span> pathways were found to be significantly upregulated on the gene expression level using the program GenMAPP/MAPPFinder. Analysis criteria were set as described in the Methods section. Underneath each section of the <span class="highlight" style="background-color:">pathway</span>, genes found to have an increased expression are depicted in red, and genes found to have a decreased expression are depicted in green. Figure adapted from GenMAPP.</div></p></div><div class="aux"><div class="resc">Jane Palsgaard, et al. PLoS One. 2009;4(8):e6575.</div><p class="links"><a class="dblinks" href="/pubmed/19668377" ref="ordinalpos=1715&amp;ncbi_uid=2030410&amp;link_uid=19668377">Citation</a><a class="dblinks" href="/pmc/articles/PMC2719801/" ref="ordinalpos=1715&amp;ncbi_uid=2030410&amp;link_uid=PMC2719801">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1716.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2653622/figure/F5/" ref="ordinalpos=1716&amp;ncbi_uid=1780471&amp;link_uid=PMC2653622" href="/pmc/articles/PMC2653622/"><img src="/pmc/articles/instance/2653622/bin/nihms-78867-f0005.gif" src-large="/pmc/articles/instance/2653622/bin/nihms-78867-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2653622/" ref="ordinalpos=1716&amp;ncbi_uid=1780471&amp;link_uid=PMC2653622" image-link="/pmc/articles/PMC2653622/figure/F5/" class="imagepopup">Figure 5. The primary cilium likely coordinates a variety of <span class="highlight" style="background-color:">signaling</span> pathways important for cancer.  From: THE PRIMARY CILIUM: AT THE CROSSROADS OF MAMMALIAN HEDGEHOG <span class="highlight" style="background-color:">SIGNALING</span>. </a></p><div class="supp"><p class="details"><div>In Hh-dependent cancers such as BCC and medulloblastoma, the cilium may modulate the activity of <span class="highlight" style="background-color:">signaling</span> pathways in addition to Hh. For instance, localization of PDGF receptor-αα to the cilium is important for transducing downstream signals mediated by Akt and MAP kinases. The cilium may also normally restrain Wnt <span class="highlight" style="background-color:">pathway</span> activity by preventing the accumulation of β-catenin. The balance of the downstream effects of these pathways likely governs cellular decisions for proliferation and apoptosis.</div></p></div><div class="aux"><div class="resc">Sunny Y. Wong, et al. Curr Top Dev Biol. ;85:225-260.</div><p class="links"><a class="dblinks" href="/pubmed/19147008" ref="ordinalpos=1716&amp;ncbi_uid=1780471&amp;link_uid=19147008">Citation</a><a class="dblinks" href="/pmc/articles/PMC2653622/" ref="ordinalpos=1716&amp;ncbi_uid=1780471&amp;link_uid=PMC2653622">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1717.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2826040/figure/f4/" ref="ordinalpos=1717&amp;ncbi_uid=2443982&amp;link_uid=PMC2826040" href="/pmc/articles/PMC2826040/"><img src="/pmc/articles/instance/2826040/bin/zjv9990930120004.gif" src-large="/pmc/articles/instance/2826040/bin/zjv9990930120004.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2826040/" ref="ordinalpos=1717&amp;ncbi_uid=2443982&amp;link_uid=PMC2826040" image-link="/pmc/articles/PMC2826040/figure/f4/" class="imagepopup">FIG. 4. From: MicroRNA Expression and Virulence in Pandemic Influenza Virus-Infected Mice  . </a></p><div class="supp"><p class="details"><div>miR-200a is implicated in the type I interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (Top) Type I interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> comprised of miR-200a targets; (bottom) expression changes of miR-200a and its targets associated with the type I interferon <span class="highlight" style="background-color:">pathway</span> in r1918-infected samples. Red represents microRNA or targets that have increased expression in r1918-infected samples, relative to Tx/91-infected samples. Green represents microRNA or targets that have decreased expression in r1918-infected samples, relative to Tx/91-infected samples.</div></p></div><div class="aux"><div class="resc">Yu Li, et al. J Virol. 2010 March;84(6):3023-3032.</div><p class="links"><a class="dblinks" href="/pubmed/20071585" ref="ordinalpos=1717&amp;ncbi_uid=2443982&amp;link_uid=20071585">Citation</a><a class="dblinks" href="/pmc/articles/PMC2826040/" ref="ordinalpos=1717&amp;ncbi_uid=2443982&amp;link_uid=PMC2826040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1718.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847326/figure/fig5/" ref="ordinalpos=1718&amp;ncbi_uid=5560299&amp;link_uid=PMC3847326" href="/pmc/articles/PMC3847326/"><img src="/pmc/articles/instance/3847326/bin/cddis2013450f5.gif" src-large="/pmc/articles/instance/3847326/bin/cddis2013450f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847326/" ref="ordinalpos=1718&amp;ncbi_uid=5560299&amp;link_uid=PMC3847326" image-link="/pmc/articles/PMC3847326/figure/fig5/" class="imagepopup">Figure 5.  From: FUT family mediates the multidrug resistance of human hepatocellular carcinoma via the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Effect of FUT4-, FUT6- or FUT8-activated the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on the expression of multidrug resistance-associated proteins. (<b>a</b>) Expression of PI3K/Akt/NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> molecules were repressed at protein levels with FUT4, FUT6 or FUT8 shRNA transfection in BEL/FU cells. (<b>b</b>) The increased protein levels of PI3K/Akt/NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> molecules were determined using western blot in BEL7402/FUT4, FUT6 or FUT8 cells. (<b>c</b>) Decreased relative expression of MRP1 to Na<sup>+</sup>/K<sup>+</sup>-ATPase was examined using flow cytometry analysis in FUT4-, FUT6- or FUT8-shRNA-treated BEL/FU cells. (<b>d</b>) Flow cytometry analysis revealed a higher relative expression of MRP1 to Na<sup>+</sup>/K<sup>+</sup>-ATPase in BEL7402 cells with FUT4, FUT6 or FUT8 transfection. The data are means±S.D. of three independent assays (*<i>P</i>&lt;0.05)</div></p></div><div class="aux"><div class="resc">L Cheng, et al. Cell Death Dis. 2013 November;4(11):e923.</div><p class="links"><a class="dblinks" href="/pubmed/24232099" ref="ordinalpos=1718&amp;ncbi_uid=5560299&amp;link_uid=24232099">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847326/" ref="ordinalpos=1718&amp;ncbi_uid=5560299&amp;link_uid=PMC3847326">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1719.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163666/figure/pone-0024291-g003/" ref="ordinalpos=1719&amp;ncbi_uid=3208612&amp;link_uid=PMC3163666" href="/pmc/articles/PMC3163666/"><img src="/pmc/articles/instance/3163666/bin/pone.0024291.g003.gif" src-large="/pmc/articles/instance/3163666/bin/pone.0024291.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163666/" ref="ordinalpos=1719&amp;ncbi_uid=3208612&amp;link_uid=PMC3163666" image-link="/pmc/articles/PMC3163666/figure/pone-0024291-g003/" class="imagepopup">Figure 3. Identification of the effectors for ISG15 induction in the Ras-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Interferon-? <span class="highlight" style="background-color:">Signaling</span> Contributes to Ras Transformation. </a></p><div class="supp"><p class="details"><div>A, B. siRNA knock-down of interferon-β and the type I IFN receptor reduces ISG15 induction in the Ras-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The oncogenic Ras-transformed MCF-10A cells (10A/H-Ras and 10A/K-Ras) were transiently transfected with control, IFN-β or IFNAR1 siRNA. Cell lysates were collected 72 hrs post-transfection and immunoblotted with anti-ISG15 and anti-IFNAR1 antibodies. C. The PI3K inhibitor reduces ISG15 expression in oncogenic Ras-transformed cells. 10A/H-Ras and 10A/K-Ras cells were treated with 20 µM LY294002, a PI3K inhibitor, for 3 days. Lysates with equal protein amount were immunoblotted with anti-ISG15 antibodies (L: LY294002). D. The NF-κB inhibitor reduces ISG15 expression in oncogenic Ras-transformed cells. 10A/H-Ras and 10A/K-Ras cells were seeded and cultured overnight. Cells were then treated with or without 40 µM Bay 11-7085, an NF-κB inhibitor, for 24 hrs. Cell lysates were analyzed by immunoblotting using anti-ISG15, phospho-IκBα, and IκBα antibodies.</div></p></div><div class="aux"><div class="resc">Yu-Chen Tsai, et al. PLoS One. 2011;6(8):e24291.</div><p class="links"><a class="dblinks" href="/pubmed/21897875" ref="ordinalpos=1719&amp;ncbi_uid=3208612&amp;link_uid=21897875">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163666/" ref="ordinalpos=1719&amp;ncbi_uid=3208612&amp;link_uid=PMC3163666">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1720.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3022188/figure/fig1/" ref="ordinalpos=1720&amp;ncbi_uid=2719317&amp;link_uid=PMC3022188" href="/pmc/articles/PMC3022188/"><img src="/pmc/articles/instance/3022188/bin/SRCM2011-756982.001.gif" src-large="/pmc/articles/instance/3022188/bin/SRCM2011-756982.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3022188/" ref="ordinalpos=1720&amp;ncbi_uid=2719317&amp;link_uid=PMC3022188" image-link="/pmc/articles/PMC3022188/figure/fig1/" class="imagepopup">Figure 1.  From: Receptor Tyrosine Kinases as Therapeutic Targets in Rhabdomyosarcoma. </a></p><div class="supp"><p class="details"><div>Rationale for dual treatment targeting the IGF-1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RMS.  Rapamycin inhibits mTOR <span class="highlight" style="background-color:">signaling</span>, preventing inhibitory feedback on IRS-1 which allows proliferative signals from IGF-1R to IRS-1, PI3K, and AKT.  Dual treatment using rapamycin in combination with IGF-1R inhibition, such as monoclonal blocking antibodies, prevents <span class="highlight" style="background-color:">signaling</span> to these critical progrowth <span class="highlight" style="background-color:">signaling</span> nodes.</div></p></div><div class="aux"><div class="resc">Lisa E. S. Crose, et al. Sarcoma. 2011;2011:756982.</div><p class="links"><a class="dblinks" href="/pubmed/21253475" ref="ordinalpos=1720&amp;ncbi_uid=2719317&amp;link_uid=21253475">Citation</a><a class="dblinks" href="/pmc/articles/PMC3022188/" ref="ordinalpos=1720&amp;ncbi_uid=2719317&amp;link_uid=PMC3022188">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1721.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1482580/figure/F4/" ref="ordinalpos=1721&amp;ncbi_uid=726351&amp;link_uid=PMC1482580" href="/pmc/articles/PMC1482580/"><img src="/pmc/articles/instance/1482580/bin/zpq0240625140004.gif" src-large="/pmc/articles/instance/1482580/bin/zpq0240625140004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1482580/" ref="ordinalpos=1721&amp;ncbi_uid=726351&amp;link_uid=PMC1482580" image-link="/pmc/articles/PMC1482580/figure/F4/" class="imagepopup">Fig. 4. From: Ectodysplasin regulates the lymphotoxin-? <span class="highlight" style="background-color:">pathway</span> for hair differentiation. </a></p><div class="supp"><p class="details"><div>Schematic representation of EDA <span class="highlight" style="background-color:">signaling</span> in hair follicle development. EDA-A1 or EDA-A5 (ref. 43) activate the canonical NF-κB <span class="highlight" style="background-color:">pathway</span> via EDAR and p65/p50, which then signal to Wnt and BMP pathways and their antagonists for hair follicle induction, to Shh and its inhibitors for hair follicle growth, and to the alternative NF-κB <span class="highlight" style="background-color:">pathway</span>, involving LTβ and RelB, for hair-type differentiation. +, positive up-regulation; −, negative regulation (suppression).</div></p></div><div class="aux"><div class="resc">Chang-Yi Cui, et al. Proc Natl Acad Sci U S A. 2006 June 13;103(24):9142-9147.</div><p class="links"><a class="dblinks" href="/pubmed/16738056" ref="ordinalpos=1721&amp;ncbi_uid=726351&amp;link_uid=16738056">Citation</a><a class="dblinks" href="/pmc/articles/PMC1482580/" ref="ordinalpos=1721&amp;ncbi_uid=726351&amp;link_uid=PMC1482580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1722.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3913901/figure/F5/" ref="ordinalpos=1722&amp;ncbi_uid=5825133&amp;link_uid=PMC3913901" href="/pmc/articles/PMC3913901/"><img src="/pmc/articles/instance/3913901/bin/fimmu-05-00034-g005.gif" src-large="/pmc/articles/instance/3913901/bin/fimmu-05-00034-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3913901/" ref="ordinalpos=1722&amp;ncbi_uid=5825133&amp;link_uid=PMC3913901" image-link="/pmc/articles/PMC3913901/figure/F5/" class="imagepopup">Figure 5.  From: Role of the TWEAK-Fn14-cIAP1-NF-?B <span class="highlight" style="background-color:">Signaling</span> Axis in the Regulation of Myogenesis and Muscle Homeostasis. </a></p><div class="supp"><p class="details"><div><b>Differential activation of NF-κB pathways by TWEAK based on strength of Fn14 <span class="highlight" style="background-color:">signaling</span></b>. TWEAK can signal either in a juxtacrine manner, as a membrane-bound ligand, mTWEAK <b>(A)</b>, or in a paracrine manner as a soluble ligand, sTWEAK <b>(B,C)</b>. Depending on the concentration of ligand or receptor, or the propensity of the Fn14 ligand to oligomerize the receptor and form <span class="highlight" style="background-color:">signaling</span> clusters, differential activation of the NF-κB pathways occurs. <b>(A)</b> During juxtacrine <span class="highlight" style="background-color:">signaling</span>, mTWEAK favors the clustering of Fn14 on the opposite cell, leading to hyperactive <span class="highlight" style="background-color:">signaling</span> for which canonical NF-κB <span class="highlight" style="background-color:">signaling</span> predominates over non-canonical <span class="highlight" style="background-color:">signaling</span>. MAPK activation also occurs. <b>(B)</b> Under physiological conditions, low endogenous levels of sTWEAK signal in a paracrine manner, leading predominantly to non-canonical NF-κB activation. <b>(C)</b> However, under certain pathological or experimental conditions, sTWEAK can also lead to hyperactive <span class="highlight" style="background-color:">signaling</span> leading to canonical NF-κB and MAPK <span class="highlight" style="background-color:">signaling</span>. For example, high concentrations of TWEAK, applied to myoblasts or myotubes <i>ex vivo</i>, hinder differentiation and cause atrophy, respectively. Transgenic over-expression of TWEAK in mice also leads to pathological consequences. In a variety of injury states (such as denervation-induced muscle atrophy), the receptor Fn14 is induced, which then is followed by and pathological responses to endogenous levels of TWEAK. Furthermore, recombinant TWEAK fusion proteins with the ability to multimerize, or Fn14 agonistic antibodies, can both promote receptor clustering and the activation of the canonical NF-κB <span class="highlight" style="background-color:">pathway</span>. The pathways leading to pathological activation of the canonical NF-κB <span class="highlight" style="background-color:">pathway</span> are still poorly defined, but are thought to arise either from <span class="highlight" style="background-color:">pathway</span> cross-talk, such as that seen with NIK or IKK activation of the canonical mediators, or through the adaptor and E3 ubiquitin ligase TRAF6. In addition, activation of MAPK pathways may also contribute to the pathology observed.</div></p></div><div class="aux"><div class="resc">Emeka K. Enwere, et al. Front Immunol. 2014;5:34.</div><p class="links"><a class="dblinks" href="/pubmed/24550918" ref="ordinalpos=1722&amp;ncbi_uid=5825133&amp;link_uid=24550918">Citation</a><a class="dblinks" href="/pmc/articles/PMC3913901/" ref="ordinalpos=1722&amp;ncbi_uid=5825133&amp;link_uid=PMC3913901">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1723.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012433/figure/marinedrugs-12-01939-f009/" ref="ordinalpos=1723&amp;ncbi_uid=6108374&amp;link_uid=PMC4012433" href="/pmc/articles/PMC4012433/"><img src="/pmc/articles/instance/4012433/bin/marinedrugs-12-01939-g009.gif" src-large="/pmc/articles/instance/4012433/bin/marinedrugs-12-01939-g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012433/" ref="ordinalpos=1723&amp;ncbi_uid=6108374&amp;link_uid=PMC4012433" image-link="/pmc/articles/PMC4012433/figure/marinedrugs-12-01939-f009/" class="imagepopup">Figure 9.  From: The Marine Fungal Metabolite, Dicitrinone B, Induces A375 Cell Apoptosis through the ROS-Related Caspase <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The proposed <span class="highlight" style="background-color:">signaling</span> pathways for dicitrinone B-induced apoptosis in A375 cells. The effect of apoptosis is most likely achieved through rapid ROS generation, which, at least in part, leads to mitochondrial membrane permeabilization and triggers the intrinsic apoptosis <span class="highlight" style="background-color:">pathway</span>, as well as the extrinsic apoptosis <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Li Chen, et al. Mar Drugs. 2014 April;12(4):1939-1958.</div><p class="links"><a class="dblinks" href="/pubmed/24699111" ref="ordinalpos=1723&amp;ncbi_uid=6108374&amp;link_uid=24699111">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012433/" ref="ordinalpos=1723&amp;ncbi_uid=6108374&amp;link_uid=PMC4012433">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1724.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3973006/figure/F8/" ref="ordinalpos=1724&amp;ncbi_uid=5969032&amp;link_uid=PMC3973006" href="/pmc/articles/PMC3973006/"><img src="/pmc/articles/instance/3973006/bin/1478-811X-12-22-8.gif" src-large="/pmc/articles/instance/3973006/bin/1478-811X-12-22-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3973006/" ref="ordinalpos=1724&amp;ncbi_uid=5969032&amp;link_uid=PMC3973006" image-link="/pmc/articles/PMC3973006/figure/F8/" class="imagepopup">Figure 8.  From: Divergent <span class="highlight" style="background-color:">signaling</span> pathways cooperatively regulate TGF? induction of cysteine-rich protein 2 in vascular smooth muscle cells. </a></p><div class="supp"><p class="details"><div><b>Schematic model of TGFβ <span class="highlight" style="background-color:">signaling</span> pathways for CRP2 induction in VSMCs.</b> Upon TGFβ stimulation, two <span class="highlight" style="background-color:">signaling</span> axes are activated: the canonical TβRI-dependent Smad2/3 <span class="highlight" style="background-color:">pathway</span> and the non-canonical ATF2 <span class="highlight" style="background-color:">pathway</span>. In the canonical <span class="highlight" style="background-color:">pathway</span>, activated Smad2/3 form a complex with Smad4, translocates into the nucleus, and binds to SBE site at −445 of the <i>Csrp2</i> promoter. In the non-canonical <span class="highlight" style="background-color:">pathway</span>, following TβRII binding of TGFβ, TβRII (without the requirement of TβRII’s cytoplasmic kinase domain) activates Src family kinase (SFK), which in turn activates RhoA. Enhanced RhoA activation leads to ROCK activation and increased JNK phosphorylation, resulting in enhanced ATF2 phosphorylation. Activated ATF2 dimer binds to CRE site at −461. The two <span class="highlight" style="background-color:">signaling</span> pathways (ATF2 and Smad) converge on the CRE and SBE sites of the <i>Csrp2</i> promoter to cooperatively control CRP2 induction in VSMCs, which represents a previously unrecognized mechanism of VSMC gene induction by TGFβ.</div></p></div><div class="aux"><div class="resc">Meng-Ling Wu, et al. Cell Commun Signal. 2014;12:22-22.</div><p class="links"><a class="dblinks" href="/pubmed/24674138" ref="ordinalpos=1724&amp;ncbi_uid=5969032&amp;link_uid=24674138">Citation</a><a class="dblinks" href="/pmc/articles/PMC3973006/" ref="ordinalpos=1724&amp;ncbi_uid=5969032&amp;link_uid=PMC3973006">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1725.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3091413/figure/f5-cin-2011-093/" ref="ordinalpos=1725&amp;ncbi_uid=2952274&amp;link_uid=PMC3091413" href="/pmc/articles/PMC3091413/"><img src="/pmc/articles/instance/3091413/bin/cin-2011-093f5.gif" src-large="/pmc/articles/instance/3091413/bin/cin-2011-093f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3091413/" ref="ordinalpos=1725&amp;ncbi_uid=2952274&amp;link_uid=PMC3091413" image-link="/pmc/articles/PMC3091413/figure/f5-cin-2011-093/" class="imagepopup">Figure 5.  From: Associating GWAS Information with the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Using Transcription Profiling. </a></p><div class="supp"><p class="details"><div>Results of <span class="highlight" style="background-color:">pathway</span> prediction and network modeling showing interactions between genes containing SNPs associated with risk for breast cancer and genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> and other biological pathways based on the Asian population. Nodes represent genes and vertices represent interactions. The color code denotes the biological process in which the genes are involved as defined in Figure 3. The color codes in the vertices indicate the experimental techniques or a combination thereof used to confirm the relationship between the genes as determined by the experiments reported in the literature. Candidate genes containing SNPs associated with risk for breast cancer are shown in red, genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown in blue and novel genes are shown in black.</div></p></div><div class="aux"><div class="resc">Chindo Hicks, et al. Cancer Inform. 2011;10:93-108.</div><p class="links"><a class="dblinks" href="/pubmed/21584266" ref="ordinalpos=1725&amp;ncbi_uid=2952274&amp;link_uid=21584266">Citation</a><a class="dblinks" href="/pmc/articles/PMC3091413/" ref="ordinalpos=1725&amp;ncbi_uid=2952274&amp;link_uid=PMC3091413">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1726.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3388125/figure/f5-12_093_patel/" ref="ordinalpos=1726&amp;ncbi_uid=4103468&amp;link_uid=PMC3388125" href="/pmc/articles/PMC3388125/"><img src="/pmc/articles/instance/3388125/bin/12_093_patelf5.gif" src-large="/pmc/articles/instance/3388125/bin/12_093_patelf5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3388125/" ref="ordinalpos=1726&amp;ncbi_uid=4103468&amp;link_uid=PMC3388125" image-link="/pmc/articles/PMC3388125/figure/f5-12_093_patel/" class="imagepopup">Figure 5.  From: Delayed Administration of Pyroglutamate Helix B Surface Peptide (pHBSP), a Novel Nonerythropoietic Analog of Erythropoietin, Attenuates Acute Kidney Injury. </a></p><div class="supp"><p class="details"><div>Antiinflammatory mechanisms of tissue-protective molecules. The antiinflammatory mechanism of EPOR-βcR activation is associated with (a) the activation of the phosphoinositide 3-kinase (PI3K)-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which leads to the inhibition of the activation of GSK-3β, which subsequently suppresses NF-κB activity leading to reduced expression of the NF-κB–driven gene transcription of proinflammatory mediators; (b) in parallel inhibition of the MAPK-p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which subsequently suppresses the production of proinflammatory mediators; and (c) the activation of the PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which leads to increased activation of eNOS, resulting in an enhanced formation of NO.</div></p></div><div class="aux"><div class="resc">Nimesh S A Patel, et al. Mol Med. 2012;18(1):719-727.</div><p class="links"><a class="dblinks" href="/pubmed/22415011" ref="ordinalpos=1726&amp;ncbi_uid=4103468&amp;link_uid=22415011">Citation</a><a class="dblinks" href="/pmc/articles/PMC3388125/" ref="ordinalpos=1726&amp;ncbi_uid=4103468&amp;link_uid=PMC3388125">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1727.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC519143/figure/fig4/" ref="ordinalpos=1727&amp;ncbi_uid=515455&amp;link_uid=PMC519143" href="/pmc/articles/PMC519143/"><img src="/pmc/articles/instance/519143/bin/zmk0100428380004.gif" src-large="/pmc/articles/instance/519143/bin/zmk0100428380004.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC519143/" ref="ordinalpos=1727&amp;ncbi_uid=515455&amp;link_uid=PMC519143" image-link="/pmc/articles/PMC519143/figure/fig4/" class="imagepopup">Figure 4. From: Transcription Profiling of Cyclic AMP <span class="highlight" style="background-color:">Signaling</span> in Candida albicans. </a></p><div class="supp"><p class="details"><div>Model of the <i>C. albicans</i> cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The schematic representation shows the role of three components of the cAMP <span class="highlight" style="background-color:">pathway</span>: Ras1p, Cdc35p, and Efg1p. The model proposes that Ras1p is a primary regulator of adenylyl cyclase but that there are other regulators of this <span class="highlight" style="background-color:">pathway</span> because the loss of Ras1p modulates only a subset of genes affected by the loss of Cdc35p, and Ras1p defective strains can still produce pseudohyphae. It appears that all three components are involved in the morphogenetic switch, but the cAMP <span class="highlight" style="background-color:">pathway</span> mediates regulation of other cellular processes independent of Efg1p, and these other targets of the cAMP <span class="highlight" style="background-color:">pathway</span> may or may not be regulated through PKA. In addition, we suggest that Efg1p also exerts effects on the transcription of genes outside the cAMP regulatory circuit. The asterisk (<sup>*</sup>) shows that the cell wall processes affected by cAMP and Efg1 are different.</div></p></div><div class="aux"><div class="resc">Doreen Harcus, et al. Mol Biol Cell. 2004 October;15(10):4490-4499.</div><p class="links"><a class="dblinks" href="/pubmed/15269278" ref="ordinalpos=1727&amp;ncbi_uid=515455&amp;link_uid=15269278">Citation</a><a class="dblinks" href="/pmc/articles/PMC519143/" ref="ordinalpos=1727&amp;ncbi_uid=515455&amp;link_uid=PMC519143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1728.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3059061/figure/F4/" ref="ordinalpos=1728&amp;ncbi_uid=3540624&amp;link_uid=PMC3059061" href="/pmc/articles/PMC3059061/"><img src="/pmc/articles/instance/3059061/bin/zbc0171155130004.gif" src-large="/pmc/articles/instance/3059061/bin/zbc0171155130004.jpg" alt="FIGURE 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3059061/" ref="ordinalpos=1728&amp;ncbi_uid=3540624&amp;link_uid=PMC3059061" image-link="/pmc/articles/PMC3059061/figure/F4/" class="imagepopup">FIGURE 4. From: Complex Regulation of the TRPM8 Cold Receptor Channel. </a></p><div class="supp"><p class="details"><div><b>Schematic of principal <span class="highlight" style="background-color:">signaling</span> pathways involved in plasmalemmal TRPM8 regulation.</b> Each <i>numbered box</i> depicts molecular interconnection related to the specific <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>Box 1</i>, PIP<sub>2</sub> depletion <span class="highlight" style="background-color:">pathway</span>. PIP<sub>2</sub> is a key lipid co-factors in TRPM8 activation, which depletion as a result of Gq-coupled receptor-mediated stimulation (PDGF receptor depicted) of PLC reduces TRPM8 functionality. <i>Box 2</i>, iPLA2/LPL <span class="highlight" style="background-color:">pathway</span>. The products of iPLA2 catalytic activity, LPLs, act as TRPM8 activators; stimulation of iPLA2 and consequently enhancement of LPL production can be achieved by endoplasmic reticulum (<i>ER</i>) Ca<sup>2+</sup> store depletion. <i>Box 3</i>, the cAMP/PKA <span class="highlight" style="background-color:">pathway</span>. Decrease of the basal level of cAMP/PKA-dependent phosphorylation of TRPM8 via recruitment of the α2-adrenoreceptor (α<i>2AR</i>)-coupled Gi/AC/cAMP/PKA <span class="highlight" style="background-color:">pathway</span> inhibits channel function; the level of cAMP/PKA-dependent phosphorylation, and consequently, TRPM8 activation can be restored via the Gs/AC/cAMP/PKA <span class="highlight" style="background-color:">pathway</span> coupled to β-adrenoreceptor (β<i>AR</i>). <i>Box 4</i>, PLC/PKC <span class="highlight" style="background-color:">pathway</span>. The PSA can activate TRPM8 via the bradykinin receptor (<i>B2R</i>) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving protein kinase C (PKC) Ca<sup>2+</sup>-dependent activation of PKC may also indirectly contribute to the Ca<sup>2+</sup>-mediated TRPM8 channel desensitization. <i>Box 5</i>, M3 muscarinic receptor-stimulated cPLA2/AA <span class="highlight" style="background-color:">pathway</span> identified in this study. AA derived as a result of nonclassical M3 muscarinic receptor-mediated stimulation of cPLA2 (marked by <i>?</i>) acts as a TRPM8 channel inhibitor. <i>DAG</i>, diacylglycerol; <i>ISO</i>, isoproterenol; <i>PDGFR</i>, PDGF receptor. <i>Green circles</i> with + and <i>green arrows</i> indicate stimulatory action, <i>red circles</i> with − and <i>red arrows</i> indicate inhibitory action; and a <i>downward red arrow</i> near PIP<sub>2</sub> indicates decreasing of its content.</div></p></div><div class="aux"><div class="resc">Alexis Bavencoffe, et al. J Biol Chem. 2011 March 18;286(11):9849-9855.</div><p class="links"><a class="dblinks" href="/pubmed/21245133" ref="ordinalpos=1728&amp;ncbi_uid=3540624&amp;link_uid=21245133">Citation</a><a class="dblinks" href="/pmc/articles/PMC3059061/" ref="ordinalpos=1728&amp;ncbi_uid=3540624&amp;link_uid=PMC3059061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1729.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3335094/figure/pone-0035709-g004/" ref="ordinalpos=1729&amp;ncbi_uid=3968656&amp;link_uid=PMC3335094" href="/pmc/articles/PMC3335094/"><img src="/pmc/articles/instance/3335094/bin/pone.0035709.g004.gif" src-large="/pmc/articles/instance/3335094/bin/pone.0035709.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3335094/" ref="ordinalpos=1729&amp;ncbi_uid=3968656&amp;link_uid=PMC3335094" image-link="/pmc/articles/PMC3335094/figure/pone-0035709-g004/" class="imagepopup">Figure 4. Effect of mechanical strain on the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Mechanical Strain Regulates Osteoblast Proliferation through Integrin-Mediated ERK Activation. </a></p><div class="supp"><p class="details"><div>The expression values of the selected 5 genes from the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was measured by cDNA microarray and quantitative real-time RT-PCR (qRT-PCR). MC3T3-E1 cells were stimulated under mechanical strains of 2500 µε, once a day at 0.5 Hz, and a periodicity of 1 h/day for 3 days. Data are represented as the mean ± SD of at least three biological replicates, * <i>P</i>&lt;0.05, ** <i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Yu-xian Yan, et al. PLoS One. 2012;7(4):e35709.</div><p class="links"><a class="dblinks" href="/pubmed/22539993" ref="ordinalpos=1729&amp;ncbi_uid=3968656&amp;link_uid=22539993">Citation</a><a class="dblinks" href="/pmc/articles/PMC3335094/" ref="ordinalpos=1729&amp;ncbi_uid=3968656&amp;link_uid=PMC3335094">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1730.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC316761/figure/F9/" ref="ordinalpos=1730&amp;ncbi_uid=404366&amp;link_uid=PMC316761" href="/pmc/articles/PMC316761/"><img src="/pmc/articles/instance/316761/bin/gad.12f9.gif" src-large="/pmc/articles/instance/316761/bin/gad.12f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC316761/" ref="ordinalpos=1730&amp;ncbi_uid=404366&amp;link_uid=PMC316761" image-link="/pmc/articles/PMC316761/figure/F9/" class="imagepopup">Figure 9.  From: Periodic repression of Notch <span class="highlight" style="background-color:">pathway</span> genes governs the segmentation of Xenopus embryos. </a></p><div class="supp"><p class="details"><div>Model for <i>Xenopus</i> vertebrate segmentation. Segmental regulation of Notch <span class="highlight" style="background-color:">pathway</span> genes in the paraxial mesoderm of <i>Xenopus</i> embryos. The presomitic mesoderm of <i>Xenopus</i> embryos can be divided into three regions, TBD (red shading), TZ (blue shading), and somitomeric region (green shading) in which the Notch <span class="highlight" style="background-color:">pathway</span> is regulated by different factors. The Notch <span class="highlight" style="background-color:">pathway</span> is active when paraxial cells are in the TBD but is repressed in posterior half- segments when cells enter the TZ. This repression is likely to be due to several factors including negative feedback via <i>ESR-5</i>, as well as segmental repressors such as the Hairy-oscillator. At this point in the process, Notch <span class="highlight" style="background-color:">signaling</span> acts to establish segment size by restricting the domain in which the Notch <span class="highlight" style="background-color:">pathway</span> genes are active. In the somitomeric region, Notch <span class="highlight" style="background-color:">signaling</span> appears to take on another role in which ESR-5 appears to promote the expression of <i>Thylacine1</i> and <i>X-Delta-2</i> in a positive feedback loop. Positive feedback would act to maintain segmental identity by preserving the domains in which Notch <span class="highlight" style="background-color:">signaling</span> occurs.</div></p></div><div class="aux"><div class="resc">Wui-Chuong Jen, et al. Genes Dev. 1999 June 1;13(11):1486-1499.</div><p class="links"><a class="dblinks" href="/pubmed/10364164" ref="ordinalpos=1730&amp;ncbi_uid=404366&amp;link_uid=10364164">Citation</a><a class="dblinks" href="/pmc/articles/PMC316761/" ref="ordinalpos=1730&amp;ncbi_uid=404366&amp;link_uid=PMC316761">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1731.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3525563/figure/pone-0052154-g006/" ref="ordinalpos=1731&amp;ncbi_uid=4466568&amp;link_uid=PMC3525563" href="/pmc/articles/PMC3525563/"><img src="/pmc/articles/instance/3525563/bin/pone.0052154.g006.gif" src-large="/pmc/articles/instance/3525563/bin/pone.0052154.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3525563/" ref="ordinalpos=1731&amp;ncbi_uid=4466568&amp;link_uid=PMC3525563" image-link="/pmc/articles/PMC3525563/figure/pone-0052154-g006/" class="imagepopup">Figure 6. Downregulation of REDD1 expression inhibits insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 3T3-L1 adipocytes..  From: Regulated in Development and DNA Damage Responses -1 (REDD1) Protein Contributes to Insulin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Adipocytes. </a></p><div class="supp"><p class="details"><div>3T3-L1 adipocytes were transfected with REDD1 siRNA 48 h after transfection, adipocytes were stimulated with insulin (10 nM) for 5 minutes. Proteins were analyzed by immunoblots using indicated antibodies. Quantification of phosphorylation of insulin receptor (n = 6) and PKB (n = 5) in response to insulin is shown (*p&lt;0.05).</div></p></div><div class="aux"><div class="resc">Claire Regazzetti, et al. PLoS One. 2012;7(12):e52154.</div><p class="links"><a class="dblinks" href="/pubmed/23272222" ref="ordinalpos=1731&amp;ncbi_uid=4466568&amp;link_uid=23272222">Citation</a><a class="dblinks" href="/pmc/articles/PMC3525563/" ref="ordinalpos=1731&amp;ncbi_uid=4466568&amp;link_uid=PMC3525563">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1732.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2724029/figure/F5/" ref="ordinalpos=1732&amp;ncbi_uid=1908165&amp;link_uid=PMC2724029" href="/pmc/articles/PMC2724029/"><img src="/pmc/articles/instance/2724029/bin/fnmol-02-007-g005.gif" src-large="/pmc/articles/instance/2724029/bin/fnmol-02-007-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2724029/" ref="ordinalpos=1732&amp;ncbi_uid=1908165&amp;link_uid=PMC2724029" image-link="/pmc/articles/PMC2724029/figure/F5/" class="imagepopup">Figure 5.  From: Major <span class="highlight" style="background-color:">Signaling</span> Pathways in Migrating Neuroblasts. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> networks upregulated in migrating neuroblasts</b>. <b>(A)</b> Growth factor <span class="highlight" style="background-color:">signaling</span> network. <b>(B)</b> and <b>(C)</b> MAPK <span class="highlight" style="background-color:">signaling</span> network. <b>(C)</b> is simplified version of <b>(B)</b>. Networks were identified by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis and subsequently modified using Bibliosphere, PathwayArchitect, GO and <span class="highlight" style="background-color:">pathway</span> databases. The intensity of the red color indicates the extent of gene upregulation. Connecting lines with arrowheads indicate activation of proteins, without arrowheads protein–protein interaction. Continuous and dashed lines are direct and indirect activation/interaction, respectively. For node description see Figure 4.</div></p></div><div class="aux"><div class="resc">Konstantin Khodosevich, et al. Front Mol Neurosci. 2009;2:7.</div><p class="links"><a class="dblinks" href="/pubmed/19668709" ref="ordinalpos=1732&amp;ncbi_uid=1908165&amp;link_uid=19668709">Citation</a><a class="dblinks" href="/pmc/articles/PMC2724029/" ref="ordinalpos=1732&amp;ncbi_uid=1908165&amp;link_uid=PMC2724029">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1733.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3076053/figure/F7/" ref="ordinalpos=1733&amp;ncbi_uid=3578149&amp;link_uid=PMC3076053" href="/pmc/articles/PMC3076053/"><img src="/pmc/articles/instance/3076053/bin/nihms254298f7.gif" src-large="/pmc/articles/instance/3076053/bin/nihms254298f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3076053/" ref="ordinalpos=1733&amp;ncbi_uid=3578149&amp;link_uid=PMC3076053" image-link="/pmc/articles/PMC3076053/figure/F7/" class="imagepopup">Figure 7. Model for the negative regulation of menin in AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: The tumor suppressor protein menin inhibits AKT activation by regulating its cellular localization. </a></p><div class="supp"><p class="details"><div>Menin inhibits AKT <span class="highlight" style="background-color:">signaling</span> by association with AKT. Upon growth factor stimulation, menin interacts with AKT in the cytoplasm and subsequently reduces the translocation of AKT to the plasma membrane, thus inactivating AKT.</div></p></div><div class="aux"><div class="resc">Yan Wang, et al. Cancer Res. ;71(2):371-382.</div><p class="links"><a class="dblinks" href="/pubmed/21127195" ref="ordinalpos=1733&amp;ncbi_uid=3578149&amp;link_uid=21127195">Citation</a><a class="dblinks" href="/pmc/articles/PMC3076053/" ref="ordinalpos=1733&amp;ncbi_uid=3578149&amp;link_uid=PMC3076053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1734.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3017908/figure/fig6/" ref="ordinalpos=1734&amp;ncbi_uid=2709101&amp;link_uid=PMC3017908" href="/pmc/articles/PMC3017908/"><img src="/pmc/articles/instance/3017908/bin/JSC2011-541405.006.gif" src-large="/pmc/articles/instance/3017908/bin/JSC2011-541405.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3017908/" ref="ordinalpos=1734&amp;ncbi_uid=2709101&amp;link_uid=PMC3017908" image-link="/pmc/articles/PMC3017908/figure/fig6/" class="imagepopup">Figure 6.  From: Deciphering Squamous Cell Carcinoma Using Multidimensional Genomic Approaches. </a></p><div class="supp"><p class="details"><div>Major pathways affected in NMSC. (a) SqCC undergoes frequent alterations to the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (b) BCC undergoes frequent alterations to the sonic hedgehog <span class="highlight" style="background-color:">pathway</span>. Various <span class="highlight" style="background-color:">pathway</span> components affected are highlighted in red.</div></p></div><div class="aux"><div class="resc">Ewan A. Gibb, et al. J Skin Cancer. 2011;2011:541405.</div><p class="links"><a class="dblinks" href="/pubmed/21234096" ref="ordinalpos=1734&amp;ncbi_uid=2709101&amp;link_uid=21234096">Citation</a><a class="dblinks" href="/pmc/articles/PMC3017908/" ref="ordinalpos=1734&amp;ncbi_uid=2709101&amp;link_uid=PMC3017908">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1735.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2000831/figure/F3/" ref="ordinalpos=1735&amp;ncbi_uid=1079089&amp;link_uid=PMC2000831" href="/pmc/articles/PMC2000831/"><img src="/pmc/articles/instance/2000831/bin/nihms28507f3.gif" src-large="/pmc/articles/instance/2000831/bin/nihms28507f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2000831/" ref="ordinalpos=1735&amp;ncbi_uid=1079089&amp;link_uid=PMC2000831" image-link="/pmc/articles/PMC2000831/figure/F3/" class="imagepopup">Figure 3.  From: PROSTATE CANCER: Re-focusing on Androgen Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proliferation induced by androgen independent pathways. (A) Hypersensitive <span class="highlight" style="background-color:">pathway</span>: Lower levels of androgen can activate <span class="highlight" style="background-color:">signaling</span> through the AR transcription factor. This state is usually achieved by amplification of the receptor itself, but it can also occur in the presence of activating mutations. (B) Promiscuous <span class="highlight" style="background-color:">pathway</span>: Mutations allow non-androgenic steroid hormones to bind and activate AR <span class="highlight" style="background-color:">signaling</span>. (C) Outlaw <span class="highlight" style="background-color:">pathway</span>: AR is activated independently of ligand binding by growth factors, cytokines, interleukins, cytoplasmic kinases or nuclear co-activators. Importantly A, B and C rely in the presence of an active AR complex. (D) Bypass <span class="highlight" style="background-color:">pathway</span>: Alternative survival pathways are activated and promote proliferation in the absence of androgens. The presence of an active AR is not required. Classification adapted from (Feldman and Feldman, 2001).</div></p></div><div class="aux"><div class="resc">Maria Nieto, et al. Int J Biochem Cell Biol. ;39(9):1562-1568.</div><p class="links"><a class="dblinks" href="/pubmed/17321194" ref="ordinalpos=1735&amp;ncbi_uid=1079089&amp;link_uid=17321194">Citation</a><a class="dblinks" href="/pmc/articles/PMC2000831/" ref="ordinalpos=1735&amp;ncbi_uid=1079089&amp;link_uid=PMC2000831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1736.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3931733/figure/F5/" ref="ordinalpos=1736&amp;ncbi_uid=6448182&amp;link_uid=PMC3931733" href="/pmc/articles/PMC3931733/"><img src="/pmc/articles/instance/3931733/bin/DEV092502F5.gif" src-large="/pmc/articles/instance/3931733/bin/DEV092502F5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3931733/" ref="ordinalpos=1736&amp;ncbi_uid=6448182&amp;link_uid=PMC3931733" image-link="/pmc/articles/PMC3931733/figure/F5/" class="imagepopup">Fig. 5. From: Tbx20 acts upstream of Wnt <span class="highlight" style="background-color:">signaling</span> to regulate endocardial cushion formation and valve remodeling during mouse cardiogenesis. </a></p><div class="supp"><p class="details"><div><b><i>Tbx20</i> loss of function in endocardium disrupts Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> in developing valves.</b> (<b>A</b>) Significant canonical pathways identified from genes differentially expressed in <i>Tbx20</i> CKO versus control by ingenuity <span class="highlight" style="background-color:">pathway</span> analysis. The <i>y</i>-axis is the -log<sub>10</sub> of the <i>P</i>-value for the enrichment of a <span class="highlight" style="background-color:">pathway</span> in the input gene list compared with genes in the whole genome. Red boxes highlight Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The zigzag orange line indicates the enrichment ratio of the percentage of genes in a particular <span class="highlight" style="background-color:">pathway</span> over the percentage of <span class="highlight" style="background-color:">pathway</span> genes in the genome. (<b>B</b>) Significant Gene Ontology (GO) terms identified in the differentially expressed genes. <i>x</i>-axis is the -log<sub>10</sub> of the <i>P</i>-value for the enrichment of a GO term in the input gene list compared with genes in the whole genome. (<b>C</b>) Heat map of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes. Red and green denotes upregulated and downregulated genes, respectively, in the <i>Tbx20</i> CKO. (<b>D</b>) Relative expression of Wnt <span class="highlight" style="background-color:">pathway</span> genes at E10.5-12.5 determined by RT-qPCR. (<b>E-L</b>) X-Gal staining of <i>TOPGAL</i> indicator mice to detect active Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> at E11.5 (E-H) and E12.5 (I-L). The mutant tricuspid and mitral valves display fewer X-Gal-stained cells and fainter staining in the positive cells in both endocardium and adjacent mesenchyme (arrows in F,H,J,L versus E,G,I,K). G,H and K,L are higher magnification images of the tricuspid valve region (boxed) from E,F and I,J. (<b>M</b>) Relative expression of <i>lacZ</i> at different stages as determined by RT-qPCR. <sup>*</sup><i>P</i>&lt;0.05, versus control. Error bars indicate s.d.</div></p></div><div class="aux"><div class="resc">Xiaoqiang Cai, et al. Development. 2013 August 1;140(15):3176-3187.</div><p class="links"><a class="dblinks" href="/pubmed/23824573" ref="ordinalpos=1736&amp;ncbi_uid=6448182&amp;link_uid=23824573">Citation</a><a class="dblinks" href="/pmc/articles/PMC3931733/" ref="ordinalpos=1736&amp;ncbi_uid=6448182&amp;link_uid=PMC3931733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1737.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3730321/figure/cancers-05-00639-f001/" ref="ordinalpos=1737&amp;ncbi_uid=5166705&amp;link_uid=PMC3730321" href="/pmc/articles/PMC3730321/"><img src="/pmc/articles/instance/3730321/bin/cancers-05-00639-g001.gif" src-large="/pmc/articles/instance/3730321/bin/cancers-05-00639-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3730321/" ref="ordinalpos=1737&amp;ncbi_uid=5166705&amp;link_uid=PMC3730321" image-link="/pmc/articles/PMC3730321/figure/cancers-05-00639-f001/" class="imagepopup">Figure 1.  From: Ras and Rheb <span class="highlight" style="background-color:">Signaling</span> in Survival and Cell Death. </a></p><div class="supp"><p class="details"><div>Pro- and antiapoptotic Ras and Rheb <span class="highlight" style="background-color:">signaling</span> based on different effectors and cellular context. Growth factors and insulin lead to the activation of Ras or Rheb GTPase, via their specific receptors. Downstream of activated Ras, a pleiotropy of downstream targets (<i>i.e.</i>, PI3K, Raf. Ral Gef, Rassf1/5) are activated that, cooperatively, contribute to the overall cellular <span class="highlight" style="background-color:">signaling</span> outcome. Activated Rheb, however, stimulates the mTor <span class="highlight" style="background-color:">pathway</span>, leading to either pro- or antiapoptotic <span class="highlight" style="background-color:">signaling</span>, which critically depends on the cellular context. For details see text.</div></p></div><div class="aux"><div class="resc">Anja Ehrkamp, et al. Cancers (Basel). 2013 June;5(2):639-661.</div><p class="links"><a class="dblinks" href="/pubmed/24216995" ref="ordinalpos=1737&amp;ncbi_uid=5166705&amp;link_uid=24216995">Citation</a><a class="dblinks" href="/pmc/articles/PMC3730321/" ref="ordinalpos=1737&amp;ncbi_uid=5166705&amp;link_uid=PMC3730321">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1738.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3587307/figure/F1/" ref="ordinalpos=1738&amp;ncbi_uid=5547234&amp;link_uid=PMC3587307" href="/pmc/articles/PMC3587307/"><img src="/pmc/articles/instance/3587307/bin/nihms404507f1.gif" src-large="/pmc/articles/instance/3587307/bin/nihms404507f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3587307/" ref="ordinalpos=1738&amp;ncbi_uid=5547234&amp;link_uid=PMC3587307" image-link="/pmc/articles/PMC3587307/figure/F1/" class="imagepopup">Figure 1.  From: Outfits for different occasions: tissue-specific roles of Nuclear Envelope proteins. </a></p><div class="supp"><p class="details"><div><b>(A)</b> LEM2 is a negative regulator of ERK <span class="highlight" style="background-color:">signaling</span> during muscle differentiation. Although the mechanism by which ERK1/2 is inhibited is unclear, one possibility could be through the sequestration of ERK1/2 to the nuclear periphery. Emerin has been shown to play a role in two different <span class="highlight" style="background-color:">signaling</span> cascades. First, it negatively regulates Wnt <span class="highlight" style="background-color:">signaling</span> by preventing accumulation of β-catenin in the nucleus, presumably by stimulating export of β-catenin. Second, emerin negatively regulates ERK <span class="highlight" style="background-color:">signaling</span> through unclear mechanisms. MAN1 negatively regulates TGF-β <span class="highlight" style="background-color:">signaling</span> by sequestering R-Smads away from gene targets. <b>(B)</b> During myogenesis, NET39 acts as a repressor of the mTOR-IGF-II <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. NET39 blocks IGF-II transcription by inhibiting mTOR activity at the NE. NET37, an INM protein with glycosidase activity promotes myogenesis by activating AKT <span class="highlight" style="background-color:">signaling</span> through the maturation and secretion of IGF-II in the PNS/ER.</div></p></div><div class="aux"><div class="resc">J Sebastian Gomez-Cavazos, et al. Curr Opin Cell Biol. 2012 December;24(6):775-783.</div><p class="links"><a class="dblinks" href="/pubmed/22995343" ref="ordinalpos=1738&amp;ncbi_uid=5547234&amp;link_uid=22995343">Citation</a><a class="dblinks" href="/pmc/articles/PMC3587307/" ref="ordinalpos=1738&amp;ncbi_uid=5547234&amp;link_uid=PMC3587307">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1739.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3666273/figure/fig4/" ref="ordinalpos=1739&amp;ncbi_uid=4992325&amp;link_uid=PMC3666273" href="/pmc/articles/PMC3666273/"><img src="/pmc/articles/instance/3666273/bin/JL2013-890343.004.gif" src-large="/pmc/articles/instance/3666273/bin/JL2013-890343.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3666273/" ref="ordinalpos=1739&amp;ncbi_uid=4992325&amp;link_uid=PMC3666273" image-link="/pmc/articles/PMC3666273/figure/fig4/" class="imagepopup">Figure 4.  From:  Trans-10, cis 12-Conjugated Linoleic Acid-Induced Milk Fat Depression Is Associated with Inhibition of PPARγ <span class="highlight" style="background-color:">Signaling</span> and Inflammation in Murine Mammary Tissue. </a></p><div class="supp"><p class="details"><div>Acute-phase response <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (IPA) expression in the mammary tissue of lactating mice fed <i>trans</i>-10, <i>cis</i>-12-CLA. Green indicates downregulation while red indicates upregulation.</div></p></div><div class="aux"><div class="resc">Anil K. G. Kadegowda, et al. J Lipids. 2013;2013:890343.</div><p class="links"><a class="dblinks" href="/pubmed/23762566" ref="ordinalpos=1739&amp;ncbi_uid=4992325&amp;link_uid=23762566">Citation</a><a class="dblinks" href="/pmc/articles/PMC3666273/" ref="ordinalpos=1739&amp;ncbi_uid=4992325&amp;link_uid=PMC3666273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1740.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436110/figure/F3/" ref="ordinalpos=1740&amp;ncbi_uid=5339380&amp;link_uid=PMC3436110" href="/pmc/articles/PMC3436110/"><img src="/pmc/articles/instance/3436110/bin/DEV082222F3.gif" src-large="/pmc/articles/instance/3436110/bin/DEV082222F3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436110/" ref="ordinalpos=1740&amp;ncbi_uid=5339380&amp;link_uid=PMC3436110" image-link="/pmc/articles/PMC3436110/figure/F3/" class="imagepopup">Fig. 3. From: Myocardin-like protein 2 regulates TGF? <span class="highlight" style="background-color:">signaling</span> in embryonic stem cells and the developing vasculature. </a></p><div class="supp"><p class="details"><div><b>Microarray analysis reveals reduced expression of TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes in <i>Mkl2</i><sup>–/–</sup> ES cells.</b> Microarray analysis was performed with mRNA harvested from undifferentiated wild-type and <i>Mkl2<sup>–/–</sup></i> ES cells. To screen for pathways affected by <i>Mkl2</i> deletion, Gene Set Enrichment Analysis (GSEA) was applied to identify KEGG-designated <span class="highlight" style="background-color:">signaling</span> pathways that are coordinately induced or repressed in response to <i>Mkl2</i> deletion. The most repressed <span class="highlight" style="background-color:">pathway</span> was TGFβ <span class="highlight" style="background-color:">signaling</span> (FDR q value&lt;0.0001). The heat map shown on the left displays the list of KEGG-designated genes in the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and their relative level of gene expression in biological replicate samples of <i>Mkl2<sup>–/–</sup></i> (gray) and wild-type (yellow) ES cells. Red or blue signal indicates increased or decreased expression, respectively. Brightness is proportional to the difference from the median expression level. The NCBI gene identification names are listed on the right (see also Dünker and Krieglstein, 2002).</div></p></div><div class="aux"><div class="resc">Jian Li, et al. Development. 2012 October 1;139(19):3531-3542.</div><p class="links"><a class="dblinks" href="/pubmed/22899851" ref="ordinalpos=1740&amp;ncbi_uid=5339380&amp;link_uid=22899851">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436110/" ref="ordinalpos=1740&amp;ncbi_uid=5339380&amp;link_uid=PMC3436110">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1741.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2673764/figure/fig10/" ref="ordinalpos=1741&amp;ncbi_uid=1759897&amp;link_uid=PMC2673764" href="/pmc/articles/PMC2673764/"><img src="/pmc/articles/instance/2673764/bin/DEV034082F10.gif" src-large="/pmc/articles/instance/2673764/bin/DEV034082F10.jpg" alt="Fig. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2673764/" ref="ordinalpos=1741&amp;ncbi_uid=1759897&amp;link_uid=PMC2673764" image-link="/pmc/articles/PMC2673764/figure/fig10/" class="imagepopup">Fig. 10. From: FGF-regulated BMP <span class="highlight" style="background-color:">signaling</span> is required for eyelid closure and to specify  conjunctival epithelial cell fate. </a></p><div class="supp"><p class="details"><div><b>Summary of the role of BMP <span class="highlight" style="background-color:">signaling</span> and its interaction with the other  <span class="highlight" style="background-color:">signaling</span> pathways known or proposed to function in mouse eyelid  development.</b> (<b>A</b>) In eyelids that are deficient in BMP <span class="highlight" style="background-color:">signaling</span>,  the conjunctival epithelial cells express epidermis-specific keratin 10 and  form hair follicles near that lid margin. (<b>B</b>) The network of known or  proposed <span class="highlight" style="background-color:">signaling</span> pathways controlling eyelid development. Observations made  in this study are indicated by blue arrows, previous findings by orange arrows  and findings confirmed in this study by green arrows. Our model suggests that  epidermal cell fate is the default <span class="highlight" style="background-color:">pathway</span> and that BMP <span class="highlight" style="background-color:">signaling</span> is required  for prospective conjunctival epithelial cells to suppress the epidermal  differentiation <span class="highlight" style="background-color:">pathway</span> and become conjunctival cells. BMP <span class="highlight" style="background-color:">signaling</span> is not  required to initiate the migration of periderm cells at the lid margins, but  is required for the expansion of these cells across the corneal surface.  During this process, BMP <span class="highlight" style="background-color:">signaling</span> is required for the expression of Foxc1 and  Foxc2, and for the full activation (phosphorylation) of c-Jun. BMP-dependent  formation of active R-Smad-Smad4 complexes is required for the translocation  of p-c-Jun into the nuclei of periderm cells, where it has been reported to  increase the expression of the epidermal growth factor receptor.</div></p></div><div class="aux"><div class="resc">Jie Huang, et al. Development. 2009 May 15;136(10):1741-1750.</div><p class="links"><a class="dblinks" href="/pubmed/19369394" ref="ordinalpos=1741&amp;ncbi_uid=1759897&amp;link_uid=19369394">Citation</a><a class="dblinks" href="/pmc/articles/PMC2673764/" ref="ordinalpos=1741&amp;ncbi_uid=1759897&amp;link_uid=PMC2673764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1742.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3754784/figure/F1/" ref="ordinalpos=1742&amp;ncbi_uid=5232304&amp;link_uid=PMC3754784" href="/pmc/articles/PMC3754784/"><img src="/pmc/articles/instance/3754784/bin/nihms319742f1.gif" src-large="/pmc/articles/instance/3754784/bin/nihms319742f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3754784/" ref="ordinalpos=1742&amp;ncbi_uid=5232304&amp;link_uid=PMC3754784" image-link="/pmc/articles/PMC3754784/figure/F1/" class="imagepopup">Figure 1. Schematic Overview of TLR <span class="highlight" style="background-color:">signaling</span> pathways.  From: Toll-like receptor <span class="highlight" style="background-color:">signaling</span> in liver regeneration, fibrosis and carcinogenesis. </a></p><div class="supp"><p class="details"><div>TLR1/2, TLR2/6, TLR4, and TLR5 are expressed on plasma membrane and recognize triacyl lipopeptides, diacyl lipopeptides, LPS and flagellin, respectively. TLR3, TLR7/8 and TLR9 are located in endosome and sense dsRNA, ssRNA and CpG-DNA, respectively. All TLRs expect for TLR3 activate NF-κB and p38/JNK through MyD88. TIRAP and MyD88 are required for TLR2 and TLR4 <span class="highlight" style="background-color:">signaling</span>. TLR3 activate TBK1/IKKε through TRAF, and TLR4 requires both TRAM and TLR4 internalization for activation of TRIF-dependent <span class="highlight" style="background-color:">pathway</span>. Activated TRIF dependent pathways activate IRF-3 leading to IFN-β production. TLR7/8 and TLR9 require the complex of MyD88/IRAK1/IRF7/IKKα for induction of IFN-α.</div></p></div><div class="aux"><div class="resc">Ekihiro Seki, et al. Hepatol Res. ;41(7):597-610.</div><p class="links"><a class="dblinks" href="/pubmed/21696522" ref="ordinalpos=1742&amp;ncbi_uid=5232304&amp;link_uid=21696522">Citation</a><a class="dblinks" href="/pmc/articles/PMC3754784/" ref="ordinalpos=1742&amp;ncbi_uid=5232304&amp;link_uid=PMC3754784">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1743.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2270408/figure/F6/" ref="ordinalpos=1743&amp;ncbi_uid=1348787&amp;link_uid=PMC2270408" href="/pmc/articles/PMC2270408/"><img src="/pmc/articles/instance/2270408/bin/RevDiabeticStud-04-209-007.gif" src-large="/pmc/articles/instance/2270408/bin/RevDiabeticStud-04-209-007.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2270408/" ref="ordinalpos=1743&amp;ncbi_uid=1348787&amp;link_uid=PMC2270408" image-link="/pmc/articles/PMC2270408/figure/F6/" class="imagepopup">Figure 6. Involvement of Foxo1 in the nucleo-cytoplasmic translocation of PDX-1 which is induced by oxidative stress and the subsequent activation of the JNK <span class="highlight" style="background-color:">pathway</span>.  From: Pleiotropic Roles of PDX-1 in the Pancreas. </a></p><div class="supp"><p class="details"><div>Oxidative stress and the subsequent activation of the JNK <span class="highlight" style="background-color:">pathway</span> induce nuclear translocation of Foxo1 through modification of insulin <span class="highlight" style="background-color:">signaling</span> in β-cells, which leads to the nucleo-cytoplasmic translocation of PDX-1 and reduction of its DNA binding activity.</div></p></div><div class="aux"><div class="resc">Hideaki Kaneto, et al. Rev Diabet Stud. 2007 Winter;4(4):209-225.</div><p class="links"><a class="dblinks" href="/pubmed/18338074" ref="ordinalpos=1743&amp;ncbi_uid=1348787&amp;link_uid=18338074">Citation</a><a class="dblinks" href="/pmc/articles/PMC2270408/" ref="ordinalpos=1743&amp;ncbi_uid=1348787&amp;link_uid=PMC2270408">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1744.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1481596/figure/F4/" ref="ordinalpos=1744&amp;ncbi_uid=723120&amp;link_uid=PMC1481596" href="/pmc/articles/PMC1481596/"><img src="/pmc/articles/instance/1481596/bin/1748-7188-1-7-4.gif" src-large="/pmc/articles/instance/1481596/bin/1748-7188-1-7-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1481596/" ref="ordinalpos=1744&amp;ncbi_uid=723120&amp;link_uid=PMC1481596" image-link="/pmc/articles/PMC1481596/figure/F4/" class="imagepopup">Figure 4.  From: Decomposition of overlapping protein complexes: A graph theoretical method for analyzing static and dynamic protein associations. </a></p><div class="supp"><p class="details"><div><b>Pheromone <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></b>. The pheromone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (a) The network. (b) The Tree of Complexes representation. For the description of the elements of the tree see the text.</div></p></div><div class="aux"><div class="resc">Elena Zotenko, et al. Algorithms Mol Biol. 2006;1:7-7.</div><p class="links"><a class="dblinks" href="/pubmed/16722537" ref="ordinalpos=1744&amp;ncbi_uid=723120&amp;link_uid=16722537">Citation</a><a class="dblinks" href="/pmc/articles/PMC1481596/" ref="ordinalpos=1744&amp;ncbi_uid=723120&amp;link_uid=PMC1481596">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1745.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584578/figure/f4-ijo-40-04-1105/" ref="ordinalpos=1745&amp;ncbi_uid=4685439&amp;link_uid=PMC3584578" href="/pmc/articles/PMC3584578/"><img src="/pmc/articles/instance/3584578/bin/IJO-40-04-1105-g03.gif" src-large="/pmc/articles/instance/3584578/bin/IJO-40-04-1105-g03.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584578/" ref="ordinalpos=1745&amp;ncbi_uid=4685439&amp;link_uid=PMC3584578" image-link="/pmc/articles/PMC3584578/figure/f4-ijo-40-04-1105/" class="imagepopup">Figure 4.  From: MicroRNA miR-451 downregulates the PI3K/AKT <span class="highlight" style="background-color:">pathway</span> through CAB39 in human glioma. </a></p><div class="supp"><p class="details"><div>Impact of the miR-451 mimic oligonucleotide on some of the components of the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) U251, LN229, A172 and U87 cells were treated with the miR-451 mimics and LKB1, AMPK, p-AMPK, PI3K, p-AKT and GAPDH expressions were determined by Western blotting 48 h after treatment. (B) Schematic diagram of how miR-451 downregulation could lead to activation of the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">YUAN TIAN, et al. Int J Oncol. 2012 April;40(4):1105-1112.</div><p class="links"><a class="dblinks" href="/pubmed/22179124" ref="ordinalpos=1745&amp;ncbi_uid=4685439&amp;link_uid=22179124">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584578/" ref="ordinalpos=1745&amp;ncbi_uid=4685439&amp;link_uid=PMC3584578">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1746.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3719824/figure/F1/" ref="ordinalpos=1746&amp;ncbi_uid=5771191&amp;link_uid=PMC3719824" href="/pmc/articles/PMC3719824/"><img src="/pmc/articles/instance/3719824/bin/zjv9990978860001.gif" src-large="/pmc/articles/instance/3719824/bin/zjv9990978860001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3719824/" ref="ordinalpos=1746&amp;ncbi_uid=5771191&amp;link_uid=PMC3719824" image-link="/pmc/articles/PMC3719824/figure/F1/" class="imagepopup">Fig 1.  From: Cell-Type-Specific Activation of the Oligoadenylate Synthetase-RNase L <span class="highlight" style="background-color:">Pathway</span> by a Murine Coronavirus. </a></p><div class="supp"><p class="details"><div>Canonical type I IFN induction and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (Gray) MHV RNA is recognized by MDA5, leading to expression of IFN-β. IFN-β mediates the transcription of antiviral ISGs. (Black) OAS-RNase L <span class="highlight" style="background-color:">pathway</span>. dsRNA activates OAS, which synthesizes 2-5A. The oligonucleotide activates RNase L to degrade viral and cellular RNA. MHV ns2 can degrade 2-5A.</div></p></div><div class="aux"><div class="resc">Ling Zhao, et al. J Virol. 2013 August;87(15):8408-8418.</div><p class="links"><a class="dblinks" href="/pubmed/23698313" ref="ordinalpos=1746&amp;ncbi_uid=5771191&amp;link_uid=23698313">Citation</a><a class="dblinks" href="/pmc/articles/PMC3719824/" ref="ordinalpos=1746&amp;ncbi_uid=5771191&amp;link_uid=PMC3719824">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1747.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4218690/figure/f1-cin-suppl.5-2014-001/" ref="ordinalpos=1747&amp;ncbi_uid=6824821&amp;link_uid=PMC4218690" href="/pmc/articles/PMC4218690/"><img src="/pmc/articles/instance/4218690/bin/cin-suppl.5-2014-001f1.gif" src-large="/pmc/articles/instance/4218690/bin/cin-suppl.5-2014-001f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4218690/" ref="ordinalpos=1747&amp;ncbi_uid=6824821&amp;link_uid=PMC4218690" image-link="/pmc/articles/PMC4218690/figure/f1-cin-suppl.5-2014-001/" class="imagepopup">Figure 1.  From: Inferring the Effects of Honokiol on the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in SW480 Colon Cancer Cells. </a></p><div class="supp"><p class="details"><div>Honokiol and ionizing radiation (IR) affect Notch <span class="highlight" style="background-color:">signaling</span>. (<b>A</b>) Upon binding to a ligand, the transmembrane Notch receptor is cleaved by γ-secretase, allowing the intracellular domain to translocate to the nucleus, where it can activate a variety of transcription programs, including that of its own ligand. Notch <span class="highlight" style="background-color:">signaling</span> can, therefore, play a role in synchronizing <span class="highlight" style="background-color:">signaling</span> across cells in a population. (<b>B</b>) Western blot readouts of proteins affected by honokiol and/or IR from Ponnurangam et al.5 are represented as discrete digital values of activate and inactive states. (<b>C</b>) The initial proposed interaction network describing the effect honokiol and IR on cellular <span class="highlight" style="background-color:">signaling</span> with cross talk to the Notch <span class="highlight" style="background-color:">pathway</span>. Black edges with arrows indicate activating regulations and red edges with blunt ends indicate inhibiting regulations.</div></p></div><div class="aux"><div class="resc">Michelle L Wynn, et al. Cancer Inform. 2014;13(Suppl 5):1-12.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1747&amp;ncbi_uid=6824821&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4218690/" ref="ordinalpos=1747&amp;ncbi_uid=6824821&amp;link_uid=PMC4218690">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1748.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3269610/figure/F1/" ref="ordinalpos=1748&amp;ncbi_uid=3784977&amp;link_uid=PMC3269610" href="/pmc/articles/PMC3269610/"><img src="/pmc/articles/instance/3269610/bin/ijbsv08p0272g01.gif" src-large="/pmc/articles/instance/3269610/bin/ijbsv08p0272g01.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3269610/" ref="ordinalpos=1748&amp;ncbi_uid=3784977&amp;link_uid=PMC3269610" image-link="/pmc/articles/PMC3269610/figure/F1/" class="imagepopup">Figure 1.  From: TGF-? and BMP <span class="highlight" style="background-color:">Signaling</span> in Osteoblast Differentiation and Bone Formation. </a></p><div class="supp"><p class="details"><div>TGF-β <span class="highlight" style="background-color:">signaling</span> and negative regulation in bone formation. Canonical Smad-dependent TGF-β <span class="highlight" style="background-color:">signaling</span> first binds to receptor type II (R-II) and receptor type I (R-I), and then <span class="highlight" style="background-color:">signaling</span> transduces to their Smads. Activated Smads form a complex with Smad4 and then translocate into the nucleus where they interact with other transcription factors to trigger target gene expression. Smad7 disrupts the activated Smad2/3 to form a complex with Smad4. The non-Smad-dependent TAK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> also regulates bone formation. PTH binding activates PTH1R to stimulate several downstream effectors. PTH binding also drives internalization of PTH1R-TGFβRII complex, which attenuates both TGF-β and PTH <span class="highlight" style="background-color:">signaling</span> on bone development. Transcriptional factor cAMP response element binding protein (CREB) mediates PTH <span class="highlight" style="background-color:">signaling</span> in osteoblasts. P: phosphorylation; Ub: ubiquitination.</div></p></div><div class="aux"><div class="resc">Guiqian Chen, et al. Int J Biol Sci. 2012;8(2):272-288.</div><p class="links"><a class="dblinks" href="/pubmed/22298955" ref="ordinalpos=1748&amp;ncbi_uid=3784977&amp;link_uid=22298955">Citation</a><a class="dblinks" href="/pmc/articles/PMC3269610/" ref="ordinalpos=1748&amp;ncbi_uid=3784977&amp;link_uid=PMC3269610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1749.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2759143/figure/f3-grsb-2007-217/" ref="ordinalpos=1749&amp;ncbi_uid=2044640&amp;link_uid=PMC2759143" href="/pmc/articles/PMC2759143/"><img src="/pmc/articles/instance/2759143/bin/grsb-2007-217f3.gif" src-large="/pmc/articles/instance/2759143/bin/grsb-2007-217f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2759143/" ref="ordinalpos=1749&amp;ncbi_uid=2044640&amp;link_uid=PMC2759143" image-link="/pmc/articles/PMC2759143/figure/f3-grsb-2007-217/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Signaling</span> Mechanisms of Endogenous Angiogenesis Inhibitors Derived from Type IV Collagen. </a></p><div class="supp"><p class="details"><div>Schematic illustration of different <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by α3(IV)NC1. α3(IV)NC1 binds to αVβ3 and α3β1 integrins, and inhibits phosphorylation of FAK. Inhibition of FAK activation leads to inhibition of FAK/PI-3K/eIF4E/4E-BP1 cap dependent translation. In addition α3(IV)NC1 inhibits NFκB mediated <span class="highlight" style="background-color:">signaling</span> in hypoxic conditions leading to inhibition of COX-2/VEGF/bFGF expression, resulting in inhibition of hypoxic tumor angiogenesis.</div></p></div><div class="aux"><div class="resc">Akulapalli Sudhakar, et al. Gene Regul Syst Bio. 2007;1:217-226.</div><p class="links"><a class="dblinks" href="/pubmed/19936090" ref="ordinalpos=1749&amp;ncbi_uid=2044640&amp;link_uid=19936090">Citation</a><a class="dblinks" href="/pmc/articles/PMC2759143/" ref="ordinalpos=1749&amp;ncbi_uid=2044640&amp;link_uid=PMC2759143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1750.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2650010/figure/F2/" ref="ordinalpos=1750&amp;ncbi_uid=1735983&amp;link_uid=PMC2650010" href="/pmc/articles/PMC2650010/"><img src="/pmc/articles/instance/2650010/bin/nihms86788f2.gif" src-large="/pmc/articles/instance/2650010/bin/nihms86788f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2650010/" ref="ordinalpos=1750&amp;ncbi_uid=1735983&amp;link_uid=PMC2650010" image-link="/pmc/articles/PMC2650010/figure/F2/" class="imagepopup">Figure 2. Akt-independent mechanisms of mTOR regulation.  From: Akt-dependent and independent mechanisms of mTOR regulation in cancer. </a></p><div class="supp"><p class="details"><div>The mTOR <span class="highlight" style="background-color:">pathway</span> is regulated by mitogen-responsive <span class="highlight" style="background-color:">signaling</span> pathways (shown in green), as well as by pathways that signal the availability of nutrients, such as amino acids (shown in yellow), and intracellular energy (shown in blue). Mitogens activate mTOR independently of Akt through the Ras/Mek/ERK <span class="highlight" style="background-color:">pathway</span>, the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and through activation of phospholipase D. Amino acids also activate mTOR through multiple mechanisms: activation of Vps34, MAP4K3, the Rag family of GTPases, and phospholipase D. However, the mechanisms by which some of these proteins activate mTOR are not well understood, and there may be crosstalk between these <span class="highlight" style="background-color:">pathway</span> components. Conversely, conditions that deplete intracellular energy, such as hypoxia, inhibit the mTOR <span class="highlight" style="background-color:">pathway</span>. One mechanism by which this occurs is through activation of AMPK. AMPK is activated by increases in intracellular AMP, as well as by the upstream kinases, LKB1 and CaMKKβ. AMPK inhibits the mTOR <span class="highlight" style="background-color:">pathway</span> directly through phosphorylation of Raptor and indirectly through phosphorylation and activation of TSC2. Although hypoxia can inhibit the mTOR <span class="highlight" style="background-color:">pathway</span> through activation of AMPK, it also inhibits mTOR through increased expression of the hypoxia-inducible-factor-1 target gene, REDD1. The tuberous sclerosis complex TSC1/TSC2 and Rheb GTPase (shown in grey) integrate signals from mitogen-responsive and energy and nutrient-sensing pathways to regulate mTORC1 (shown in red).</div></p></div><div class="aux"><div class="resc">Regan M. Memmott, et al. Cell Signal. ;21(5):656-664.</div><p class="links"><a class="dblinks" href="/pubmed/19166931" ref="ordinalpos=1750&amp;ncbi_uid=1735983&amp;link_uid=19166931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2650010/" ref="ordinalpos=1750&amp;ncbi_uid=1735983&amp;link_uid=PMC2650010">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1751.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3312675/figure/A007146F4/" ref="ordinalpos=1751&amp;ncbi_uid=5961558&amp;link_uid=PMC3312675" href="/pmc/articles/PMC3312675/"><img src="/pmc/articles/instance/3312675/bin/cshperspect-IMT-007146_F4.gif" src-large="/pmc/articles/instance/3312675/bin/cshperspect-IMT-007146_F4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3312675/" ref="ordinalpos=1751&amp;ncbi_uid=5961558&amp;link_uid=PMC3312675" image-link="/pmc/articles/PMC3312675/figure/A007146F4/" class="imagepopup">Figure 4. From: Central B-Cell Tolerance: Where Selection Begins. </a></p><div class="supp"><p class="details"><div>Positive selection of nonautoreactive immature B cells into the peripheral B-cell compartment requires a threshold level of tonic BCR <span class="highlight" style="background-color:">signaling</span>. Differentiation of nonautoreactive immature B cells into transitional and mature B cells and entry into the peripheral B-cell population depends on a certain threshold of BCR expression and tonic BCR <span class="highlight" style="background-color:">signaling</span>, which is translated by the Ras-Mek-Erk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. BAFFR expression correlates with BCR surface expression and tonic <span class="highlight" style="background-color:">signaling</span>, and BAFFR <span class="highlight" style="background-color:">signaling</span> contributes to the differentiation of immature into transitional B cells. The scheme is based on data from Rowland et al. 2010a,b.</div></p></div><div class="aux"><div class="resc">Roberta Pelanda, et al. Cold Spring Harb Perspect Biol. 2012 April;4(4):a007146.</div><p class="links"><a class="dblinks" href="/pubmed/22378602" ref="ordinalpos=1751&amp;ncbi_uid=5961558&amp;link_uid=22378602">Citation</a><a class="dblinks" href="/pmc/articles/PMC3312675/" ref="ordinalpos=1751&amp;ncbi_uid=5961558&amp;link_uid=PMC3312675">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1752.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2133182/figure/F10/" ref="ordinalpos=1752&amp;ncbi_uid=1184203&amp;link_uid=PMC2133182" href="/pmc/articles/PMC2133182/"><img src="/pmc/articles/instance/2133182/bin/JCB9811074.f10.gif" src-large="/pmc/articles/instance/2133182/bin/JCB9811074.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2133182/" ref="ordinalpos=1752&amp;ncbi_uid=1184203&amp;link_uid=PMC2133182" image-link="/pmc/articles/PMC2133182/figure/F10/" class="imagepopup">Figure 10.  From: Changes in the Balance of Phosphoinositide 3-Kinase/Protein Kinase B (Akt) and the Mitogen-activated Protein Kinases (ERK/p38MAPK) Determine a Phenotype of Visceral and Vascular  Smooth Muscle Cells . </a></p><div class="supp"><p class="details"><div>Distinct <span class="highlight" style="background-color:">signaling</span> pathways are directly involved in  the phenotypic determination of visceral and vascular SMCs.  Maintenance of a differentiated phenotype of SMCs depends on  the PI3-K/PKB(Akt) <span class="highlight" style="background-color:">pathway</span>. In contrast, the coordinate activation of the ERK and p38MAPK pathways induces SMC de-differentiation. IGF-I, which is a potent factor for maintaining the  differentiated SMC phenotype, activates the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>  mediated through PI3-K/PKB(Akt), but not MAPKs. Blocking  the PI3-K/PKB(Akt) <span class="highlight" style="background-color:">pathway</span> with specific inhibitors of PI3-K,  LY249002, or wortmannin, induces SMC de-differentiation. Potent SMC de-differentiation-inducing factors, PDGF-BB, bFGF,  and EGF, all activate the ERK and p38MAPK pathways. bFGF  and EGF do not enhance the PI3-K/PKB(Akt) <span class="highlight" style="background-color:">pathway</span>,  whereas, PDGF-BB does activate it. Thus, PDGF-BB triggers  the dual <span class="highlight" style="background-color:">signaling</span> pathways, PI3-K/PKB(Akt) and two MAPKs.  When the ERK and p38MAPK pathways were simultaneously  blocked by their specific inhibitors, PD98059 and SB203580,  PDGF-BB in turn initiates to induce maintaining SMC differentiation. Therefore, the SMC phenotype would be determined by  the balance between the strengths of the PI3-K/PKB(Akt) <span class="highlight" style="background-color:">pathway</span> and the ERK and p38MAPK pathways.</div></p></div><div class="aux"><div class="resc">Ken'ichiro Hayashi, et al. J Cell Biol. 1999 May 17;145(4):727-740.</div><p class="links"><a class="dblinks" href="/pubmed/10330402" ref="ordinalpos=1752&amp;ncbi_uid=1184203&amp;link_uid=10330402">Citation</a><a class="dblinks" href="/pmc/articles/PMC2133182/" ref="ordinalpos=1752&amp;ncbi_uid=1184203&amp;link_uid=PMC2133182">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1753.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3974572/figure/fig-002/" ref="ordinalpos=1753&amp;ncbi_uid=6054845&amp;link_uid=PMC3974572" href="/pmc/articles/PMC3974572/"><img src="/pmc/articles/instance/3974572/bin/medrep-06-23-g002.gif" src-large="/pmc/articles/instance/3974572/bin/medrep-06-23-g002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3974572/" ref="ordinalpos=1753&amp;ncbi_uid=6054845&amp;link_uid=PMC3974572" image-link="/pmc/articles/PMC3974572/figure/fig-002/" class="imagepopup">Figure 2.Fanconi anemia (FA) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> guards the genome throughout the cell cycle.  From: Fanconi anemia and the cell cycle: new perspectives on aneuploidy. </a></p><div class="supp"><p class="details"><div>FA <span class="highlight" style="background-color:">signaling</span> prevents accumulation of DNA damages in interphase and ensures high-fidelity chromosome segregation in mitosis. Abbreviations: SAC, spindle assembly checkpoint.</div></p></div><div class="aux"><div class="resc">Grzegorz Nalepa, et al. F1000Prime Rep. 2014;6:23.</div><p class="links"><a class="dblinks" href="/pubmed/24765528" ref="ordinalpos=1753&amp;ncbi_uid=6054845&amp;link_uid=24765528">Citation</a><a class="dblinks" href="/pmc/articles/PMC3974572/" ref="ordinalpos=1753&amp;ncbi_uid=6054845&amp;link_uid=PMC3974572">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1754.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4074825/figure/cancers-06-01208-f003/" ref="ordinalpos=1754&amp;ncbi_uid=6307005&amp;link_uid=PMC4074825" href="/pmc/articles/PMC4074825/"><img src="/pmc/articles/instance/4074825/bin/cancers-06-01208-g003.gif" src-large="/pmc/articles/instance/4074825/bin/cancers-06-01208-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4074825/" ref="ordinalpos=1754&amp;ncbi_uid=6307005&amp;link_uid=PMC4074825" image-link="/pmc/articles/PMC4074825/figure/cancers-06-01208-f003/" class="imagepopup">Figure 3.  From: Replicative Stress and the FHIT Gene: Roles in Tumor Suppression, Genome Stability and Prevention of Carcinogenesis. </a></p><div class="supp"><p class="details"><div>FHIT-substrate complex as a <span class="highlight" style="background-color:">signaling</span> molecule. Intracellular Ap<sub>3</sub>A substrates may increase following external stimuli such as peroxide and UV light. The FHIT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated when FHIT binds Ap<sub>3</sub>A substrate. Hydrolysis of Ap<sub>3</sub>A substrate into AMP and ADP, or Src phosphorylation of Tyrosine 114 and subsequent degradation may inactivate the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jenna R. Karras, et al. Cancers (Basel). 2014 June;6(2):1208-1219.</div><p class="links"><a class="dblinks" href="/pubmed/24901304" ref="ordinalpos=1754&amp;ncbi_uid=6307005&amp;link_uid=24901304">Citation</a><a class="dblinks" href="/pmc/articles/PMC4074825/" ref="ordinalpos=1754&amp;ncbi_uid=6307005&amp;link_uid=PMC4074825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1755.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2987799/figure/pone-0014033-g006/" ref="ordinalpos=1755&amp;ncbi_uid=2609791&amp;link_uid=PMC2987799" href="/pmc/articles/PMC2987799/"><img src="/pmc/articles/instance/2987799/bin/pone.0014033.g006.gif" src-large="/pmc/articles/instance/2987799/bin/pone.0014033.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2987799/" ref="ordinalpos=1755&amp;ncbi_uid=2609791&amp;link_uid=PMC2987799" image-link="/pmc/articles/PMC2987799/figure/pone-0014033-g006/" class="imagepopup">Figure 6. FGF-2 induction of the intracellular FGF <span class="highlight" style="background-color:">signaling</span> pathways on FOb and POb cells..  From: Activation of FGF <span class="highlight" style="background-color:">Signaling</span> Mediates Proliferative and Osteogenic Differences between Neural Crest Derived Frontal and Mesoderm Parietal Derived Bone. </a></p><div class="supp"><p class="details"><div>FOb and POb cells harvested from E17.5 mice were cultured in serum free α-MEM for 12 hours in presence of specific inhibitors of the FGF <span class="highlight" style="background-color:">signaling</span> pathways to pre-empty endogenous FGF-2 activity. Then cells were treated with 20 ng/ml of rhFGF-2. <b>A</b>, effect of exogenous FGF-2 and inhibitors of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. FGF-2 stimulation for 30 minutes increased phosphorylation of ERK protein equally in FOb and POb cells. Co-treatment with 10 µM U0126 inhibitor inhibited the FGF-2 induction as well as the endogenous phosphorylation of pERK protein in untreated FGF-2 FOb and POb cells. Histogram represents densitometric analysis of electrophoresis bands as above. <b>B</b>, effect of exogenous FGF-2 and inhibitors of PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. FGF-2 treatment resulted in increased pAkt phosphorylation which was two fold higher in POb cells compared to FOb cells. Treatment with 20 µM LY294002 inhibitor mostly abrogated the effect of exogenous FGF-2 and drastically reduced the endogenous level of pAkt protein in untreated FGF-2 FOb cells. Histogram represents densitometric analysis of electrophoresis bands. <b>C</b>, effect of exogenous FGF-2 and inhibitors of PKC α/β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. FGF-2 treatment also induced an increased phosphorylation of PKC α/β proteins which was inhibited by co-treatment with 2 µM GÖ6983 inhibitor. Treatment with the inhibitor also reduced the endogenous level of p PKC α/β proteins in untreated FGF-2 cells. Histogram represents densitometric analysis of electrophoresis bands. <b>D</b>, effect of exogenous FGF-2 and inhibitors of PKC δ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. FGF-2 and/or inhibitor treatments on PKC δ protein produced a similar effect to that observed on PKC α/β proteins. Asterisk * represents statistical significance (*p&lt;0.05).</div></p></div><div class="aux"><div class="resc">Shuli Li, et al. PLoS One. 2010;5(11):e14033.</div><p class="links"><a class="dblinks" href="/pubmed/21124973" ref="ordinalpos=1755&amp;ncbi_uid=2609791&amp;link_uid=21124973">Citation</a><a class="dblinks" href="/pmc/articles/PMC2987799/" ref="ordinalpos=1755&amp;ncbi_uid=2609791&amp;link_uid=PMC2987799">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1756.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3674189/figure/F5/" ref="ordinalpos=1756&amp;ncbi_uid=6434521&amp;link_uid=PMC3674189" href="/pmc/articles/PMC3674189/"><img src="/pmc/articles/instance/3674189/bin/nihms-469608-f0005.gif" src-large="/pmc/articles/instance/3674189/bin/nihms-469608-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3674189/" ref="ordinalpos=1756&amp;ncbi_uid=6434521&amp;link_uid=PMC3674189" image-link="/pmc/articles/PMC3674189/figure/F5/" class="imagepopup">Figure 5. DPN treatment during EAE activates the PI3K/Akt/mTOR cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the CC.  From: Estrogen receptor ? ligand therapy activates PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> in oligodendrocytes and promotes remyelination in a mouse model of multiple sclerosis. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Treatment with DPN during EAE has shown increased myelination. To address the potential involvement of PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in DPN-treated EAE animals, we performed immunohistochemistry of CC-containing brain slices andWestern analysis of dissected callosal protein lysates. Using immunohistochemistry, robust p-PI3K immunostaining was seen co-labeled with CC1<sup>+</sup> OLs <b>(i)</b>. Likewise, by Western analysis, DPN-treated EAE mice revealed an elevated level of p-PI3K (**p&lt;0.001) to support the role of DPN-mediated activation of PI3K <b>(ii-iii). (B)</b> Western analysis of Akt, p-Akt (Thr308), and GAPDH revealed no significant difference in the levels of Akt between, normal, EAE+vehicle, and EAE+DPN groups <b>(i).</b> However, a significant increase in p-Akt was detected in DPN-treated EAE mice to confirm the activation of p-Akt in response to upstream <span class="highlight" style="background-color:">signaling</span> events (*p&lt;0.05; **p&lt;0.001) (<b>i-ii</b>). <b>(C)</b> Activation of the mTOR <span class="highlight" style="background-color:">pathway</span> in DPN-treated EAE animals was assessed by immunohistochemistry, which revealed the presence of p-mTOR (Ser2448) in the CC and spinal cord dorsal column <b>(i)</b>. Western analysis of mTOR, p-mTOR, and GAPDH presents a three-fold increase in p-mTOR levels in DPN-treated EAE mice relative to normal mice <b>(ii-iii). (D)</b> DPN treatment during EAE enhances P70S6K activation downstream of mTOR <span class="highlight" style="background-color:">pathway</span> to facilitate protein synthesis. Western analysis of P70S6K, p-P70S6K (Thr389), and GAPDH revealed undetectable levels of p-P70S6K in EAE+vehicle mice as compared to normal controls. DPN-treated EAE protein lysates showed a significant increase in p-P70S6K to confirm the activation of DPN-mediated mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<b>i-ii</b>) (**p&lt;0.001;ANOVA; n=6).</div></p></div><div class="aux"><div class="resc">Shalini Kumar, et al. Neurobiol Dis. ;0:131-144.</div><p class="links"><a class="dblinks" href="/pubmed/23603111" ref="ordinalpos=1756&amp;ncbi_uid=6434521&amp;link_uid=23603111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3674189/" ref="ordinalpos=1756&amp;ncbi_uid=6434521&amp;link_uid=PMC3674189">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1757.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151067/figure/cdg131f6/" ref="ordinalpos=1757&amp;ncbi_uid=243559&amp;link_uid=PMC151067" href="/pmc/articles/PMC151067/"><img src="/pmc/articles/instance/151067/bin/cdg131f6.gif" src-large="/pmc/articles/instance/151067/bin/cdg131f6.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151067/" ref="ordinalpos=1757&amp;ncbi_uid=243559&amp;link_uid=PMC151067" image-link="/pmc/articles/PMC151067/figure/cdg131f6/" class="imagepopup">Figure. From: A MAPK <span class="highlight" style="background-color:">pathway</span> mediates ethylene <span class="highlight" style="background-color:">signaling</span> in plants. </a></p><div class="supp"><p class="details"><div><b>Fig. 6.</b> Proposed model of the MAPK <span class="highlight" style="background-color:">pathway</span> mediating ethylene <span class="highlight" style="background-color:">signaling</span> in plants. The histidine kinase ETR1 functions as an ethylene receptor and activates CTR1 in the absence of ethylene. CTR1 is a negative regulator of the MAPKK SIMKK and the MAPKs SIMK/MMK3 in <i>Medicago</i>, and MPK6/13 in <i>Arabidopsis</i>. In the presence of ethylene, ETR1 and CTR1 become inactivated, relieving SIMKK from inhibition. Subsequent activation of the MAPKs activates gene expression of ethylene-responsive genes via direct activation of EIN2 and EIN3 or through other factors.</div></p></div><div class="aux"><div class="resc">Fatma Ouaked, et al. EMBO J. 2003 March 17;22(6):1282-1288.</div><p class="links"><a class="dblinks" href="/pubmed/12628921" ref="ordinalpos=1757&amp;ncbi_uid=243559&amp;link_uid=12628921">Citation</a><a class="dblinks" href="/pmc/articles/PMC151067/" ref="ordinalpos=1757&amp;ncbi_uid=243559&amp;link_uid=PMC151067">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1758.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3668775/figure/F3/" ref="ordinalpos=1758&amp;ncbi_uid=6181920&amp;link_uid=PMC3668775" href="/pmc/articles/PMC3668775/"><img src="/pmc/articles/instance/3668775/bin/zbc0261351330003.gif" src-large="/pmc/articles/instance/3668775/bin/zbc0261351330003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3668775/" ref="ordinalpos=1758&amp;ncbi_uid=6181920&amp;link_uid=PMC3668775" image-link="/pmc/articles/PMC3668775/figure/F3/" class="imagepopup">FIGURE 3. From: Innate Immunity in Pluripotent Human Cells. </a></p><div class="supp"><p class="details"><div><b>Expression of IFN <span class="highlight" style="background-color:">signaling</span> molecules in human pluripotent cells.</b> <i>A</i>, lower expression of key type I IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components in iPS(IMR-90) cells, analyzed by RT-qPCR. <i>B</i>, lower expression of key type I IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components in H9 cells analyzed by RT-qPCR. The mRNA expression levels were normalized to <i>actin</i> and <i>gapdh</i>. **, <i>p</i> &lt; 0.01; ***, <i>p</i> &lt; 0.001. <i>Error bars</i>, S.E. <i>C</i>, Protein expression of IFN-β <span class="highlight" style="background-color:">signaling</span> molecules in pluripotent and differentiated human cells.</div></p></div><div class="aux"><div class="resc">Xiao-Xiao Hong, et al. J Biol Chem. 2013 May 31;288(22):16196-16205.</div><p class="links"><a class="dblinks" href="/pubmed/23599426" ref="ordinalpos=1758&amp;ncbi_uid=6181920&amp;link_uid=23599426">Citation</a><a class="dblinks" href="/pmc/articles/PMC3668775/" ref="ordinalpos=1758&amp;ncbi_uid=6181920&amp;link_uid=PMC3668775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1759.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2857326/figure/F7/" ref="ordinalpos=1759&amp;ncbi_uid=2266720&amp;link_uid=PMC2857326" href="/pmc/articles/PMC2857326/"><img src="/pmc/articles/instance/2857326/bin/ukmss-4266-f0007.gif" src-large="/pmc/articles/instance/2857326/bin/ukmss-4266-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2857326/" ref="ordinalpos=1759&amp;ncbi_uid=2266720&amp;link_uid=PMC2857326" image-link="/pmc/articles/PMC2857326/figure/F7/" class="imagepopup">Figure 7. Model of centroblastic cell apoptosis pathways.  From: TGF-? induces apoptosis in human B cells via transcriptional regulation of BIK and BCL-XL. </a></p><div class="supp"><p class="details"><div>The contribution of TGF-β <span class="highlight" style="background-color:">signaling</span> to regulation of BCL-2 family members is included alongside known apoptosis pathways functional in germinal centres. The three separate apoptosis pathways converge to regulate death of human centroblasts. FAS <span class="highlight" style="background-color:">signaling</span> activates the extrinsic caspase-8/caspase-3 cascade while TGF-β <span class="highlight" style="background-color:">signaling</span> via its receptor complex represses the prosurvival factor BCL-X<sub>L</sub> and induces BIK to activate the intrinsic <span class="highlight" style="background-color:">pathway</span>. Fas and TGF-β <span class="highlight" style="background-color:">signaling</span> contribute to the default apoptotic state of ‘death by neglect’ while B cell receptor <span class="highlight" style="background-color:">signaling</span> also activates the mitochondrial apoptosis <span class="highlight" style="background-color:">pathway</span> in cells lacking environmental survival cues. Loss of TGF-β <span class="highlight" style="background-color:">signaling</span> through inhibition of TGF-β receptor <span class="highlight" style="background-color:">signaling</span> by SB-431542 would reduce the amount of caspase-3 activation and raise the threshold for apoptosis induction. The model highlights the requirement for maintenance of cFLIP and BCL-X<sub>L</sub> levels for cells to survive during germinal centre reactions.</div></p></div><div class="aux"><div class="resc">LC Spender, et al. Cell Death Differ. ;16(4):593-602.</div><p class="links"><a class="dblinks" href="/pubmed/19136942" ref="ordinalpos=1759&amp;ncbi_uid=2266720&amp;link_uid=19136942">Citation</a><a class="dblinks" href="/pmc/articles/PMC2857326/" ref="ordinalpos=1759&amp;ncbi_uid=2266720&amp;link_uid=PMC2857326">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1760.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2758284/figure/f5-grsb-2009-021/" ref="ordinalpos=1760&amp;ncbi_uid=2036973&amp;link_uid=PMC2758284" href="/pmc/articles/PMC2758284/"><img src="/pmc/articles/instance/2758284/bin/grsb-2009-021f5.gif" src-large="/pmc/articles/instance/2758284/bin/grsb-2009-021f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758284/" ref="ordinalpos=1760&amp;ncbi_uid=2036973&amp;link_uid=PMC2758284" image-link="/pmc/articles/PMC2758284/figure/f5-grsb-2009-021/" class="imagepopup">Figure 5. Diagram showing the <span class="highlight" style="background-color:">signaling</span> pathways involved in zygote formation.  From: A Novel Function of Ethylene. </a></p><div class="supp"><p class="details"><div>By several pharmacological experiments, it has been suggested that protein kinase A (PKA) is involved in the downstream of cAMP as an inhibitory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> together with calmodulin activated by Ca<sup>2+</sup>. Lydan and O’Day41 have shown evidence about the role of calmodulin as both a negative (gamete formation) and a positive (cell fusion) regulator of sexual events in <i>Dictyostelium.</i> It is most likely that ZYG1 may be phosphorylated by PKC activated by Ca<sup>2+</sup> in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inducing zygote formation.<br />: induction →<br />: inhibition ⫞</div></p></div><div class="aux"><div class="resc">Aiko Amagai. Gene Regul Syst Bio. 2009;3:21-30.</div><p class="links"><a class="dblinks" href="/pubmed/19838332" ref="ordinalpos=1760&amp;ncbi_uid=2036973&amp;link_uid=19838332">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758284/" ref="ordinalpos=1760&amp;ncbi_uid=2036973&amp;link_uid=PMC2758284">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1761.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3982180/figure/F1/" ref="ordinalpos=1761&amp;ncbi_uid=6012726&amp;link_uid=PMC3982180" href="/pmc/articles/PMC3982180/"><img src="/pmc/articles/instance/3982180/bin/jcav05p0336g001.gif" src-large="/pmc/articles/instance/3982180/bin/jcav05p0336g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3982180/" ref="ordinalpos=1761&amp;ncbi_uid=6012726&amp;link_uid=PMC3982180" image-link="/pmc/articles/PMC3982180/figure/F1/" class="imagepopup">Figure 1.  From: Identification of a Novel Compound That Suppresses Breast Cancer Invasiveness by Inhibiting Transforming Growth Factor-? <span class="highlight" style="background-color:">Signaling</span> via Estrogen Receptor ?. </a></p><div class="supp"><p class="details"><div><b> Identification of a compound that does not enhance ERα <span class="highlight" style="background-color:">signaling</span> but inhibits TGF-β <span class="highlight" style="background-color:">signaling</span>.</b> (<b>A</b>) Structure of the candidate compound N-1. (<b>B</b>,<b> C</b>) N-1 does not enhance ERα <span class="highlight" style="background-color:">pathway</span> but suppresses TGF-β <span class="highlight" style="background-color:">signaling</span>. MCF-7 cells were cultured in the absence or presence of TGF-β (1 ng/mL), estrogen (E2, 10<sup>-8</sup> M), or N-1 (10<sup>-6</sup> M). The mRNA levels of pS2 (<b>B</b>) and PAI-1 (<b>C</b>) were quantified by real-time RT-PCR. ** indicates p &lt; 0.01 and n.s. indicates p &gt; 0.05.</div></p></div><div class="aux"><div class="resc">Natsuka Goto, et al. J Cancer. 2014;5(5):336-343.</div><p class="links"><a class="dblinks" href="/pubmed/24723976" ref="ordinalpos=1761&amp;ncbi_uid=6012726&amp;link_uid=24723976">Citation</a><a class="dblinks" href="/pmc/articles/PMC3982180/" ref="ordinalpos=1761&amp;ncbi_uid=6012726&amp;link_uid=PMC3982180">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1762.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4127235/figure/fig4/" ref="ordinalpos=1762&amp;ncbi_uid=6508283&amp;link_uid=PMC4127235" href="/pmc/articles/PMC4127235/"><img src="/pmc/articles/instance/4127235/bin/MI2014-310183.004.gif" src-large="/pmc/articles/instance/4127235/bin/MI2014-310183.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4127235/" ref="ordinalpos=1762&amp;ncbi_uid=6508283&amp;link_uid=PMC4127235" image-link="/pmc/articles/PMC4127235/figure/fig4/" class="imagepopup">Figure 4.  From: The Wnt/β-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Controls the Inflammatory Response in Infections Caused by Pathogenic Bacteria. </a></p><div class="supp"><p class="details"><div>Frizzled receptors reduce inflammation. In epithelial cells and macrophages,<i> Mycobacterium tuberculosis </i>induces NF-<i>κ</i>B-dependent expression of IL-8, TNF<i>α</i>, and IFN<i>γ</i> (left panel) through TLR <span class="highlight" style="background-color:">signaling</span>. In turn, TNF<i>α</i> and IFN<i>γ</i> stimulation increases Fzd1, Wnt6, and Wnt5a expression at the membrane cell surface (middle panel). Finally, the binding of Wnt3a, Wnt5a, or Wnt6 to Fzd1 (right panel) induces stabilization of <i>β</i>-catenin, inhibiting the NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span>. This interaction of <i>β</i>-catenin with NF-<i>κ</i>B decreases the expression of proinflammatory molecules such as TNF<i>α</i>.</div></p></div><div class="aux"><div class="resc">Octavio Silva-García, et al. Mediators Inflamm. 2014;2014:310183.</div><p class="links"><a class="dblinks" href="/pubmed/25136145" ref="ordinalpos=1762&amp;ncbi_uid=6508283&amp;link_uid=25136145">Citation</a><a class="dblinks" href="/pmc/articles/PMC4127235/" ref="ordinalpos=1762&amp;ncbi_uid=6508283&amp;link_uid=PMC4127235">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1763.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2797767/figure/F4/" ref="ordinalpos=1763&amp;ncbi_uid=2120903&amp;link_uid=PMC2797767" href="/pmc/articles/PMC2797767/"><img src="/pmc/articles/instance/2797767/bin/1476-4598-8-117-4.gif" src-large="/pmc/articles/instance/2797767/bin/1476-4598-8-117-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2797767/" ref="ordinalpos=1763&amp;ncbi_uid=2120903&amp;link_uid=PMC2797767" image-link="/pmc/articles/PMC2797767/figure/F4/" class="imagepopup">Figure 4.  From: Identification of small molecule compounds that inhibit the HIF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Effect of four novel inhibitors on HIF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Inhibition of HIF-1 mediated β-lactamase activity in HRE-bla ME-180 cells by various concentrations of NCGC00043898, NCGC00044926, NCGC00049606 and NCGC00056044 in the presence of 1% oxygen (A) and 60 μM CoCl<sub>2 </sub>(B). The data are displayed as the percentage of control. Representative values are from a single experiment performed in duplicate. Data are expressed as mean ± SD.</div></p></div><div class="aux"><div class="resc">Menghang Xia, et al. Mol Cancer. 2009;8:117-117.</div><p class="links"><a class="dblinks" href="/pubmed/20003191" ref="ordinalpos=1763&amp;ncbi_uid=2120903&amp;link_uid=20003191">Citation</a><a class="dblinks" href="/pmc/articles/PMC2797767/" ref="ordinalpos=1763&amp;ncbi_uid=2120903&amp;link_uid=PMC2797767">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1764.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3689941/figure/F9/" ref="ordinalpos=1764&amp;ncbi_uid=6274963&amp;link_uid=PMC3689941" href="/pmc/articles/PMC3689941/"><img src="/pmc/articles/instance/3689941/bin/zbc0291353420009.gif" src-large="/pmc/articles/instance/3689941/bin/zbc0291353420009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3689941/" ref="ordinalpos=1764&amp;ncbi_uid=6274963&amp;link_uid=PMC3689941" image-link="/pmc/articles/PMC3689941/figure/F9/" class="imagepopup">FIGURE 9. From: Epithelial to Mesenchymal Transition Promotes Breast Cancer Progression via a Fibronectin-dependent STAT3 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>STAT3 <span class="highlight" style="background-color:">pathway</span> switching is induced by EMT during breast cancer progression.</b> Early-stage BCs utilize SRC-dependent EGFR <span class="highlight" style="background-color:">signaling</span> to drive STAT3 activation and primary tumor growth. Although EGFR is capable of activating STAT3 downstream of FN engagement, the growth factor-stimulated <span class="highlight" style="background-color:">pathway</span> appears to be the prominent mode in pre-EMT cells. Late-stage BCs that have acquired EMT phenotypes utilize β1 integrin receptors to engage the ECM containing aberrant levels of FN thereby activating a FAK/PYK2:JAK2:STAT3 <span class="highlight" style="background-color:">signaling</span> cascade. This <span class="highlight" style="background-color:">pathway</span> may be clinically relevant at the metastatic tumor site and likely contributes to the resistance of BC to EGFR targeted therapies. Targets for pharmacological and immunological inhibitors used in this study are also highlighted.</div></p></div><div class="aux"><div class="resc">Nikolas Balanis, et al. J Biol Chem. 2013 June 21;288(25):17954-17967.</div><p class="links"><a class="dblinks" href="/pubmed/23653350" ref="ordinalpos=1764&amp;ncbi_uid=6274963&amp;link_uid=23653350">Citation</a><a class="dblinks" href="/pmc/articles/PMC3689941/" ref="ordinalpos=1764&amp;ncbi_uid=6274963&amp;link_uid=PMC3689941">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1765.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3246694/figure/fig2/" ref="ordinalpos=1765&amp;ncbi_uid=3739120&amp;link_uid=PMC3246694" href="/pmc/articles/PMC3246694/"><img src="/pmc/articles/instance/3246694/bin/DRP2012-259170.002.gif" src-large="/pmc/articles/instance/3246694/bin/DRP2012-259170.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3246694/" ref="ordinalpos=1765&amp;ncbi_uid=3739120&amp;link_uid=PMC3246694" image-link="/pmc/articles/PMC3246694/figure/fig2/" class="imagepopup">Figure 2.  From: Targeting the Cellular <span class="highlight" style="background-color:">Signaling</span>: BRAF Inhibition and Beyond for the Treatment of Metastatic Malignant Melanoma. </a></p><div class="supp"><p class="details"><div>Representation of the resistance mechanisms to the vemurafenib. (a) Vemurafenib causes cell death by inhibiting V600E BRAF kinase activity and <span class="highlight" style="background-color:">signaling</span> transduction through the MAPK <span class="highlight" style="background-color:">pathway</span>. (b) New mutations in the NRAS causes heterodimerization of BRAF/CRAF and reactivation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (c) and (d) Phosphorylation of ERK independent of RAF stimulation is the result of the overexpression of COT (c) and of the additional mutation in MEK kinase (d). (e) Overexpression of PDGFR<i>β</i> can lead to resistance to vemurafenib treatment independent from the continuous inhibition of the MAPK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Felipe Ades, et al. Dermatol Res Pract. 2012;2012:259170.</div><p class="links"><a class="dblinks" href="/pubmed/22216021" ref="ordinalpos=1765&amp;ncbi_uid=3739120&amp;link_uid=22216021">Citation</a><a class="dblinks" href="/pmc/articles/PMC3246694/" ref="ordinalpos=1765&amp;ncbi_uid=3739120&amp;link_uid=PMC3246694">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1766.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3229919/figure/F6/" ref="ordinalpos=1766&amp;ncbi_uid=3695780&amp;link_uid=PMC3229919" href="/pmc/articles/PMC3229919/"><img src="/pmc/articles/instance/3229919/bin/nihms332329f6.gif" src-large="/pmc/articles/instance/3229919/bin/nihms332329f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3229919/" ref="ordinalpos=1766&amp;ncbi_uid=3695780&amp;link_uid=PMC3229919" image-link="/pmc/articles/PMC3229919/figure/F6/" class="imagepopup">Figure 6.  From: Fibroblast Growth Factor (FGF) regulates human neuroectoderm specification through ERK1/2-PARP-1 <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Scheme of the functional role of FGF <span class="highlight" style="background-color:">signaling</span> in neural specification of hESCs. Along neuroectoderm specification of hESCs, FGFs and BMPs regulate PAX6 transcription through parallel pathways besides potential convergence on SMAD1. FGFs mainly activate the ERK1/2-PARP-1 <span class="highlight" style="background-color:">pathway</span> to activate the transcription of PAX6, hence neural specification.</div></p></div><div class="aux"><div class="resc">Youngdong Yoo, et al. Stem Cells. ;29(12):1975-1982.</div><p class="links"><a class="dblinks" href="/pubmed/21997878" ref="ordinalpos=1766&amp;ncbi_uid=3695780&amp;link_uid=21997878">Citation</a><a class="dblinks" href="/pmc/articles/PMC3229919/" ref="ordinalpos=1766&amp;ncbi_uid=3695780&amp;link_uid=PMC3229919">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1767.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4150363/figure/F5/" ref="ordinalpos=1767&amp;ncbi_uid=6613491&amp;link_uid=PMC4150363" href="/pmc/articles/PMC4150363/"><img src="/pmc/articles/instance/4150363/bin/fnagi-06-00225-g0005.gif" src-large="/pmc/articles/instance/4150363/bin/fnagi-06-00225-g0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4150363/" ref="ordinalpos=1767&amp;ncbi_uid=6613491&amp;link_uid=PMC4150363" image-link="/pmc/articles/PMC4150363/figure/F5/" class="imagepopup">Figure 5.  From: The effects of aging on the BTBR mouse model of autism spectrum disorder. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis of cortical and hippocampal proteins differentially regulated BTBR and WT</b>. Cortical and hippocampal proteins exhibiting differential regulation between BTBR and control mice were analyzed for potential functional <span class="highlight" style="background-color:">pathway</span> interactions using KEGG (<b>A</b>-cortex, <b>B</b>-hippocampus) and Ingenuity Canonical <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis (<b>C</b>-cortex, <b>D</b>-hippocampus). For <b>(A,B)</b>, the top 10 most significantly-populated (calculated with Hybrid scores: enrichment factor (R) <sup>*</sup> (−log<sub>10</sub> enrichment probability, P) neuronally-specific KEGG or Canonical <span class="highlight" style="background-color:">signaling</span> pathways are shown.</div></p></div><div class="aux"><div class="resc">Joan M. Jasien, et al. Front Aging Neurosci. 2014;6:225.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1767&amp;ncbi_uid=6613491&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4150363/" ref="ordinalpos=1767&amp;ncbi_uid=6613491&amp;link_uid=PMC4150363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1768.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2937009/figure/F2/" ref="ordinalpos=1768&amp;ncbi_uid=3189492&amp;link_uid=PMC2937009" href="/pmc/articles/PMC2937009/"><img src="/pmc/articles/instance/2937009/bin/nihms227825f2.gif" src-large="/pmc/articles/instance/2937009/bin/nihms227825f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2937009/" ref="ordinalpos=1768&amp;ncbi_uid=3189492&amp;link_uid=PMC2937009" image-link="/pmc/articles/PMC2937009/figure/F2/" class="imagepopup">Figure 2. Insulin <span class="highlight" style="background-color:">Signaling</span> Pathways.  From: Muscle Atrophy in Chronic Kidney Disease Results from Abnormalities in Insulin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Shown are the two major insulin <span class="highlight" style="background-color:">signaling</span> pathways. In muscle, the phosphatidylinositol 3-kinase (PI3K)/Akt <span class="highlight" style="background-color:">pathway</span> controls many metabolic responses including glucose transport and protein degradation. Activated insulin receptors phosphorylate the insulin receptor substrate (IRS) proteins. The PI3K p85 regulatory subunit binds to the phosphorylated IRS protein and recruits the PI3K p110 catalytic subunit. The PI3K complex forms phosphatidylinositol (3,4,5) triphosphate which then activates Akt. Akt serves as a nexus enzyme that phosphorylates a variety of downstream effectors. The activated insulin receptor also stimulates a parallel <span class="highlight" style="background-color:">signaling</span> cascade of kinases called the MEK/ERK <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Pathway</span> <span class="highlight" style="background-color:">signaling</span> is initiated by the IRS proteins. It ultimately controls many cellular processes involved in cell growth and survival.</div></p></div><div class="aux"><div class="resc">S. Russ Price, et al. J Ren Nutr. ;20(5 Suppl):S24-S28.</div><p class="links"><a class="dblinks" href="/pubmed/20797566" ref="ordinalpos=1768&amp;ncbi_uid=3189492&amp;link_uid=20797566">Citation</a><a class="dblinks" href="/pmc/articles/PMC2937009/" ref="ordinalpos=1768&amp;ncbi_uid=3189492&amp;link_uid=PMC2937009">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1769.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2958481/figure/fig6/" ref="ordinalpos=1769&amp;ncbi_uid=2895814&amp;link_uid=PMC2958481" href="/pmc/articles/PMC2958481/"><img src="/pmc/articles/instance/2958481/bin/JCB_201004108_GS_Fig6.gif" src-large="/pmc/articles/instance/2958481/bin/JCB_201004108_GS_Fig6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2958481/" ref="ordinalpos=1769&amp;ncbi_uid=2895814&amp;link_uid=PMC2958481" image-link="/pmc/articles/PMC2958481/figure/fig6/" class="imagepopup">Figure 6. From: A mechanism for vertebrate Hedgehog <span class="highlight" style="background-color:">signaling</span>: recruitment to cilia and dissociation of SuFu-Gli protein complexes. </a></p><div class="supp"><p class="details"><div><b>A model for activation of Gli proteins during vertebrate Hh <span class="highlight" style="background-color:">signaling</span>.</b> In the resting state of the Hh <span class="highlight" style="background-color:">pathway</span> (left), SuFu forms inactive complexes with Gli2 and Gli3-FL, which are sequestered in the cytoplasm. Without Hh stimulation, SuFu–Gli complexes traffic to the primary cilium at a low level independently of Smo; this basal ciliary trafficking is antagonized by PKA. Hh <span class="highlight" style="background-color:">pathway</span> stimulation (right) leads to the translocation of active Smo to the cilium, which, in turn, recruits SuFu–Gli complexes. Active Smo at cilia causes the dissociation of SuFu from Gli. Monomeric SuFu and Gli leave the cilium followed by Gli nuclear translocation and activation of the transcriptional program of the Hh <span class="highlight" style="background-color:">pathway</span>. PKA antagonizes Hh <span class="highlight" style="background-color:">signaling</span> by blocking ciliary localization of SuFu–Gli complexes, thus preventing coupling between active Smo and dissociation of SuFu–Gli complexes.</div></p></div><div class="aux"><div class="resc">Hanna Tukachinsky, et al. J Cell Biol. 2010 October 18;191(2):415-428.</div><p class="links"><a class="dblinks" href="/pubmed/20956384" ref="ordinalpos=1769&amp;ncbi_uid=2895814&amp;link_uid=20956384">Citation</a><a class="dblinks" href="/pmc/articles/PMC2958481/" ref="ordinalpos=1769&amp;ncbi_uid=2895814&amp;link_uid=PMC2958481">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1770.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4176851/figure/Fig4/" ref="ordinalpos=1770&amp;ncbi_uid=6655784&amp;link_uid=PMC4176851" href="/pmc/articles/PMC4176851/"><img src="/pmc/articles/instance/4176851/bin/12943_2014_1420_Fig4_HTML.gif" src-large="/pmc/articles/instance/4176851/bin/12943_2014_1420_Fig4_HTML.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4176851/" ref="ordinalpos=1770&amp;ncbi_uid=6655784&amp;link_uid=PMC4176851" image-link="/pmc/articles/PMC4176851/figure/Fig4/" class="imagepopup">Figure 4.  From: OncomiR-196 promotes an invasive phenotype in oral cancer through the NME4-JNK-TIMP1-MMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>Cellular effect of miR-196 is exerted via the NME4-JNK-TIMP1-MMP1/9 molecular <span class="highlight" style="background-color:">pathway</span> in OECM1 cells. (A)</b> Effects of miR-196 or NME4 plasmid transfection on MAPK, TIMP1 and MMP protein expression in OECM1 cells. Protein expression was determined by western blot analysis, and the density of each band was quantified. <b>(B)</b> Effect of miR-196 or NME4 plasmid transfection on p-JNK and MMP9 expression in OECM1 cells. Protein expression was determined by immunofluorescence staining and confocal microscopy. <b>(C)</b> A hypothetical model illustrating the miR-196-NME4-JNK-TIMP1-MMP1/9 molecular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oral cancer. (*<i>p</i> &lt; 0.05, **<i>p</i> &lt; 0.01, ***<i>p</i> &lt; 0.005, t-test).</div></p></div><div class="aux"><div class="resc">Ya-Ching Lu, et al. Mol Cancer. 2014;13(1):218.</div><p class="links"><a class="dblinks" href="/pubmed/25233933" ref="ordinalpos=1770&amp;ncbi_uid=6655784&amp;link_uid=25233933">Citation</a><a class="dblinks" href="/pmc/articles/PMC4176851/" ref="ordinalpos=1770&amp;ncbi_uid=6655784&amp;link_uid=PMC4176851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1771.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1862818/figure/F2/" ref="ordinalpos=1771&amp;ncbi_uid=1004310&amp;link_uid=PMC1862818" href="/pmc/articles/PMC1862818/"><img src="/pmc/articles/instance/1862818/bin/nihms20335f2.gif" src-large="/pmc/articles/instance/1862818/bin/nihms20335f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1862818/" ref="ordinalpos=1771&amp;ncbi_uid=1004310&amp;link_uid=PMC1862818" image-link="/pmc/articles/PMC1862818/figure/F2/" class="imagepopup">Figure 2.  From: Dysfunction of Amyloid Precursor Protein <span class="highlight" style="background-color:">Signaling</span> in Neurons Leads to DNA Synthesis and Apoptosis. </a></p><div class="supp"><p class="details"><div>Graphic representation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by the interaction of PAK3 with APP and G<sub>o</sub>. Steps in the cascade that are potential therapeutic targets include interactions among APP, G<sub>o</sub> and PAK3 or the activity of JNK3.</div></p></div><div class="aux"><div class="resc">Rachael L. Neve, et al. Biochim Biophys Acta. ;1772(4):430-437.</div><p class="links"><a class="dblinks" href="/pubmed/17113271" ref="ordinalpos=1771&amp;ncbi_uid=1004310&amp;link_uid=17113271">Citation</a><a class="dblinks" href="/pmc/articles/PMC1862818/" ref="ordinalpos=1771&amp;ncbi_uid=1004310&amp;link_uid=PMC1862818">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1772.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3898176/figure/F2/" ref="ordinalpos=1772&amp;ncbi_uid=5712780&amp;link_uid=PMC3898176" href="/pmc/articles/PMC3898176/"><img src="/pmc/articles/instance/3898176/bin/nihms546640f2a.gif" src-large="/pmc/articles/instance/3898176/bin/nihms546640f2a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3898176/figure/F2/" ref="ordinalpos=1772&amp;ncbi_uid=5712780&amp;link_uid=PMC3898176" href="/pmc/articles/PMC3898176/"><img src="/pmc/articles/instance/3898176/bin/nihms546640f2b.gif" src-large="/pmc/articles/instance/3898176/bin/nihms546640f2b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3898176/" ref="ordinalpos=1772&amp;ncbi_uid=5712780&amp;link_uid=PMC3898176" image-link="/pmc/articles/PMC3898176/figure/F2/" class="imagepopup">Figure 2. Conservation of the Toll, IMD, and JAK/STAT pathways in Drosophila melanogaster, Anopheles gambiae, Aedes aegypti and Ixodes scapularis.  From: THE "UBIQUITOUS" REALITY OF VECTOR IMMUNOLOGY. </a></p><div class="supp"><p class="details"><div>(A) In <i>Drosophila</i>, gram-negative bacteria binding proteins (GNBPs) and peptidoglycan recognition proteins (PGRPs) have been shown to activate the Toll <span class="highlight" style="background-color:">pathway</span> in the presence of stimulants, such as fungi, yeast, lysine-type peptidoglycan (K-type PGN) and viruses. These recognition proteins signal downstream to Persephone (psh) and Grass. CLIP domain serine proteases (clip-SP) modulate the <span class="highlight" style="background-color:">signaling</span> after recognition as well. Sphinx1/2, Spheroide, and Spirit initiate the activation of Spätzle through the Spätzle processing enzyme (SPE). Spätzle binds to Toll which recruits three Death domain-containing molecules: MyD88, Tube, and Pelle. Pellino/Pellino homolog, perhaps, acts as a positive regulator of immunity by ubiquitinating Pelle. After which, TNF-receptor-associated factor (TRAF) signals to Dorsal-immune related factor (DIF), followed by <span class="highlight" style="background-color:">signaling</span> to Dorsal. The activation is facilitated by the degradation of Cactus through K48 ubiquitination. The transcription factor translocates to the nucleus in order to upregulate immune genes. The Toll <span class="highlight" style="background-color:">pathway</span> is highly conserved in: (B) <i>Anopheles</i>, (C) <i>Aedes</i>, and (D) <i>Ixodes</i>. The Toll <span class="highlight" style="background-color:">pathway</span> has been demonstrated to recognize <i>Plasmodium berghei</i> in (B) <i>Anopheles</i> and the Dengue virus in (C) <i>Aedes</i>. TRAF <span class="highlight" style="background-color:">signaling</span> initiates REL1 and REL1A /1B activity in (B) <i>Anopheles</i> and (C) <i>Aedes</i>, respectively. (A) On the other hand, the <i>Drosophila</i> IMD <span class="highlight" style="background-color:">pathway</span> recognizes primarily mono-diaminopimelic acid-type peptidoglycans (DAP-type PGN). Fas-associated protein with death domain (FADD) is recruited to IMD. FADD binds to the Death related ced-3/Nedd2-like caspase (Dredd)/CASPL1. Dredd can cleave IMD. Inhibitor of apoptosis (IAP) can also associate with Dredd/CASPL1. Effete, Uev1a, and Bendless play a role in the regulation of this step and caspar may also inhibit the activity of IMD-dependent transcription factors. TGF-β activated kinase (TAK1), TAK1 binding protein 2 (TAB2) complex forms as <span class="highlight" style="background-color:">signaling</span> continues. Two avenues may result from the IMD <span class="highlight" style="background-color:">pathway</span>: JNK or NF-κB. For NF-κB activation, Relish translocates to the nucleus to activate effector genes. There are several potential sites of ubiquitination throughout the IMD <span class="highlight" style="background-color:">pathway</span>: IMD, Dredd/CASPL1, and the IKK complex. Like the Toll <span class="highlight" style="background-color:">pathway</span>, the IMD <span class="highlight" style="background-color:">pathway</span> is found in many species: (B) <i>Anopheles</i>, (C) <i>Aedes</i>, and (D) <i>Ixodes</i>. <i>P. falciparum</i> and the dengue virus can trigger the IMD <span class="highlight" style="background-color:">pathway</span> in (B) <i>Anopheles</i> and (C) <i>Aedes</i>, respectively. While Relish is regulated by the IMD <span class="highlight" style="background-color:">pathway</span> in (A) <i>Drosophila</i> and (D) <i>Ixodes</i>, REL2, the homolog of Relish, acts as the transcription factor in (B) <i>Anopheles</i> and (C) <i>Aedes</i>. The third <span class="highlight" style="background-color:">pathway</span> is the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. (A) A ligand derived from the unpaired (UPD) gene activates the <span class="highlight" style="background-color:">pathway</span> by binding to Domeless (Dome). Phosphorylated JAK promotes the dimerization of STAT. Dimerized STAT can proceed to the nucleus. Countering the activation, both SOCS and PIAS negatively regulate the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. Though the JAK/STAT <span class="highlight" style="background-color:">pathway</span> is evolutionarily conserved across the organisms discussed, various pathogens have demonstrated the ability to activate the JAK-STAT <span class="highlight" style="background-color:">pathway</span>, such as: (B) <i>Plasmodium vivax</i>, (C) dengue virus and (D) <i>A. phagocytophilum</i>. For the Toll, IMD and JAK/STAT pathways, <i>B. burgdorferi</i> recognition in <i>I. scapularis</i> remains mostly elusive. The information illustrated here was obtained from gene sequences available at vectorbase (www.vectorbase.org) and the following articles: (Waterhouse et al., 2007, Liu et al., 2012, Souza-Neto et al., 2009, Cirimotich et al., 2010, Xi et al., 2008, Hoffmann, 2003).</div></p></div><div class="aux"><div class="resc">Maiara S. Severo, et al. Cell Microbiol. ;15(7):1070-1078.</div><p class="links"><a class="dblinks" href="/pubmed/23433059" ref="ordinalpos=1772&amp;ncbi_uid=5712780&amp;link_uid=23433059">Citation</a><a class="dblinks" href="/pmc/articles/PMC3898176/" ref="ordinalpos=1772&amp;ncbi_uid=5712780&amp;link_uid=PMC3898176">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1773.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3402304/figure/F6/" ref="ordinalpos=1773&amp;ncbi_uid=5140357&amp;link_uid=PMC3402304" href="/pmc/articles/PMC3402304/"><img src="/pmc/articles/instance/3402304/bin/1978fig6.gif" src-large="/pmc/articles/instance/3402304/bin/1978fig6.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3402304/" ref="ordinalpos=1773&amp;ncbi_uid=5140357&amp;link_uid=PMC3402304" image-link="/pmc/articles/PMC3402304/figure/F6/" class="imagepopup">FIG. 6. From: G Protein-Coupled Receptor Kinase 2, With ?-Arrestin 2, Impairs Insulin-Induced Akt/Endothelial Nitric Oxide Synthase <span class="highlight" style="background-color:">Signaling</span> in ob/ob Mouse Aorta. </a></p><div class="supp"><p class="details"><div>Proposed model of GRK2-mediated inhibition of Akt/eNOS <span class="highlight" style="background-color:">pathway</span> in diabetes. In normal aortas, binding of insulin to IR activates Akt/eNOS <span class="highlight" style="background-color:">pathway</span> via β-arrestin 2 (βarr2) activation, and GRK2 is not translocated. In diabetes, however, GRK2, which is present in increased quantities, binds to Akt (and βarr2 is not activated). This leads to reductions of Akt/eNOS <span class="highlight" style="background-color:">signaling</span> and NO production in the membrane.</div></p></div><div class="aux"><div class="resc">Kumiko Taguchi, et al. Diabetes. 2012 August;61(8):1978-1985.</div><p class="links"><a class="dblinks" href="/pubmed/22688330" ref="ordinalpos=1773&amp;ncbi_uid=5140357&amp;link_uid=22688330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3402304/" ref="ordinalpos=1773&amp;ncbi_uid=5140357&amp;link_uid=PMC3402304">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1774.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4007928/figure/f1-0990007/" ref="ordinalpos=1774&amp;ncbi_uid=6104213&amp;link_uid=PMC4007928" href="/pmc/articles/PMC4007928/"><img src="/pmc/articles/instance/4007928/bin/997.fig1.gif" src-large="/pmc/articles/instance/4007928/bin/997.fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4007928/" ref="ordinalpos=1774&amp;ncbi_uid=6104213&amp;link_uid=PMC4007928" image-link="/pmc/articles/PMC4007928/figure/f1-0990007/" class="imagepopup">Figure 1. From: Targeting the phosphoinositide 3-kinase <span class="highlight" style="background-color:">pathway</span> in hematologic malignancies. </a></p><div class="supp"><p class="details"><div>Key features of the phosphoinositide 3-kinase (PI3K) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Upon activation, class IA and B PI3Ks initiate <span class="highlight" style="background-color:">signaling</span> by phosphorylating phosphatidylinositol-4,5-bisphosphate (PI-4,5-P<sub>2</sub>) to generate phosphatidylinositol-3,4,5-trisphosphate (PIP<sub>3</sub>), which recruits AKT and 3-phosphoinositide-dependent kinase 1 (PDK1) to the plasma membrane. AKT-mediated phosphorylation can activate (arrows) or inhibit (bars) downstream proteins. PI3K <span class="highlight" style="background-color:">signaling</span> affects cell growth, apoptosis, cell-cycle regulation, glucose metabolism, and DNA repair. Some molecular details have been omitted from this diagram for simplicity.</div></p></div><div class="aux"><div class="resc">Elias Jabbour, et al. Haematologica. 2014 January;99(1):7-18.</div><p class="links"><a class="dblinks" href="/pubmed/24425689" ref="ordinalpos=1774&amp;ncbi_uid=6104213&amp;link_uid=24425689">Citation</a><a class="dblinks" href="/pmc/articles/PMC4007928/" ref="ordinalpos=1774&amp;ncbi_uid=6104213&amp;link_uid=PMC4007928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1775.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3670000/figure/F3/" ref="ordinalpos=1775&amp;ncbi_uid=4983499&amp;link_uid=PMC3670000" href="/pmc/articles/PMC3670000/"><img src="/pmc/articles/instance/3670000/bin/fendo-04-00066-g003.gif" src-large="/pmc/articles/instance/3670000/bin/fendo-04-00066-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3670000/" ref="ordinalpos=1775&amp;ncbi_uid=4983499&amp;link_uid=PMC3670000" image-link="/pmc/articles/PMC3670000/figure/F3/" class="imagepopup">Figure 3.  From: Estrogens Regulate the Hepatic Effects of Growth Hormone, a Hormonal Interplay with Multiple Fates. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of <span class="highlight" style="background-color:">signaling</span> pathways activated by E2 and its crosstalk with GH</b>. Estradiol can regulate GH actions in liver by modulating GH responsiveness which includes changes in hepatic GHR expression and crosstalk with GH-activated JAK2-STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Estrogens can induce SOCS2 expression which in turn negatively inhibits GHR-JAK2-STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Leandro Fernández-Pérez, et al. Front Endocrinol (Lausanne). 2013;4:66.</div><p class="links"><a class="dblinks" href="/pubmed/23761784" ref="ordinalpos=1775&amp;ncbi_uid=4983499&amp;link_uid=23761784">Citation</a><a class="dblinks" href="/pmc/articles/PMC3670000/" ref="ordinalpos=1775&amp;ncbi_uid=4983499&amp;link_uid=PMC3670000">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1776.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2174972/figure/pbio-0060008-g007/" ref="ordinalpos=1776&amp;ncbi_uid=1151545&amp;link_uid=PMC2174972" href="/pmc/articles/PMC2174972/"><img src="/pmc/articles/instance/2214806/bin/pbio.0060008.g007.gif" src-large="/pmc/articles/instance/2214806/bin/pbio.0060008.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2174972/" ref="ordinalpos=1776&amp;ncbi_uid=1151545&amp;link_uid=PMC2174972" image-link="/pmc/articles/PMC2174972/figure/pbio-0060008-g007/" class="imagepopup">Figure 7. Schematic Model of TRADD's Role in Nonapoptotic <span class="highlight" style="background-color:">Signaling</span> Induced by the TRADD-Binding Domain of LMP1.  From: The Viral Oncoprotein LMP1 Exploits TRADD for <span class="highlight" style="background-color:">Signaling</span> by Masking Its Apoptotic Activity. </a></p><div class="supp"><p class="details"><div>The unique interaction of the 16 C-terminal amino acids of LMP1 (LTB) with TRADD prevents apoptosis induction through TRADD, which is an intrinsic and transferable function of LTB. Additional LMP1 sequences are not required for masking TRADD's apoptotic activity. TRADD mediates the recruitment of IKKβ to the LMP1 <span class="highlight" style="background-color:">signaling</span> complex through a yet unidentified factor. Activation of IKKβ requires a parallel <span class="highlight" style="background-color:">pathway</span> that is dependent on TRAF6. TRAF6 activation of IKKβ most likely involves TAK1 and TAB1 (not depicted). The JNK <span class="highlight" style="background-color:">pathway</span> is TRADD-independent and works through TRAF6 and, probably, BS69. To keep this model concise, only key molecules of CTAR2 <span class="highlight" style="background-color:">signaling</span> are shown. Refer to the text for more details.</div></p></div><div class="aux"><div class="resc">Frank Schneider, et al. PLoS Biol. 2008 January;6(1):e8.</div><p class="links"><a class="dblinks" href="/pubmed/18198944" ref="ordinalpos=1776&amp;ncbi_uid=1151545&amp;link_uid=18198944">Citation</a><a class="dblinks" href="/pmc/articles/PMC2174972/" ref="ordinalpos=1776&amp;ncbi_uid=1151545&amp;link_uid=PMC2174972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1777.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2780819/figure/F1/" ref="ordinalpos=1777&amp;ncbi_uid=2153242&amp;link_uid=PMC2780819" href="/pmc/articles/PMC2780819/"><img src="/pmc/articles/instance/2780819/bin/CCR-4-179_F1.gif" src-large="/pmc/articles/instance/2780819/bin/CCR-4-179_F1.jpg" alt="Fig. (1). " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2780819/" ref="ordinalpos=1777&amp;ncbi_uid=2153242&amp;link_uid=PMC2780819" image-link="/pmc/articles/PMC2780819/figure/F1/" class="imagepopup">Fig. (1).  From: Roles of IL-6-gp130 <span class="highlight" style="background-color:">Signaling</span> in Vascular Inflammation. </a></p><div class="supp"><p class="details"><div><b>IL-6 induced classical and trans-<span class="highlight" style="background-color:">signaling</span> pathways.</b> Shown is a schematic view of classical IL-6 <span class="highlight" style="background-color:">signaling</span> <i>via</i> the IL-6Rα receptor and gp130 for a representative hepatocyte. IL-6Rα bound to the IL-6 ligand results in complex formation with gp130, activating tyrosine kinase activity, including and culminating in tyrosine phosphorylation of STAT3. The IL-6 trans-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is diagrammed at top, using a representative endothelial cell. Circulating IL-6∙IL-6Rα engages with gp130 expressed on cells, enabling activation of the IL-6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cells lacking IL-6Rα. See text for further details.</div></p></div><div class="aux"><div class="resc">Tieying Hou, et al. Curr Cardiol Rev. 2008 August;4(3):179-192.</div><p class="links"><a class="dblinks" href="/pubmed/19936194" ref="ordinalpos=1777&amp;ncbi_uid=2153242&amp;link_uid=19936194">Citation</a><a class="dblinks" href="/pmc/articles/PMC2780819/" ref="ordinalpos=1777&amp;ncbi_uid=2153242&amp;link_uid=PMC2780819">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1778.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4410689/figure/F1/" ref="ordinalpos=1778&amp;ncbi_uid=7460665&amp;link_uid=PMC4410689" href="/pmc/articles/PMC4410689/"><img src="/pmc/articles/instance/4410689/bin/nihms682551f1.gif" src-large="/pmc/articles/instance/4410689/bin/nihms682551f1.jpg" alt="FIGURE 5.1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4410689/" ref="ordinalpos=1778&amp;ncbi_uid=7460665&amp;link_uid=PMC4410689" image-link="/pmc/articles/PMC4410689/figure/F1/" class="imagepopup">FIGURE 5.1.  From: Connective Tissue Disorders and Cardiovascular Complications: The indomitable role of Transforming Growth Factor-beta <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Canonical, Noncanonical, and Endoglin/ALK-1 <span class="highlight" style="background-color:">signaling</span> pathways. Both Canonical and Endoglin/ALK-1 TGF-Beta <span class="highlight" style="background-color:">signaling</span> is mediated by the phosphorylation of distinct receptor Smad proteins. Nuclear translocation requires Co-Smad binding in both pathways. Inside the nucleus, R- and Co-Smads form a complex with transcription factors to either repress or activate TGF-Beta related gene expression. The noncanonical <span class="highlight" style="background-color:">pathway</span> is mediated by TGFBRI, TRAF6, and TAK1 and results in the phosphorylation of MAPKs such as Erk1/2, JNK, and p38. These MAPKs can re-enter the Smad-dependent <span class="highlight" style="background-color:">pathway</span> through phosphorylation or mediate downstream <span class="highlight" style="background-color:">signaling</span> through other Smad-independent pathways. The lightning bolts represent mutations to the indicated proteins causative of the syndromes listed in red.</div></p></div><div class="aux"><div class="resc">Jason B. Wheeler, et al. Adv Exp Med Biol. ;802:107-127.</div><p class="links"><a class="dblinks" href="/pubmed/24443024" ref="ordinalpos=1778&amp;ncbi_uid=7460665&amp;link_uid=24443024">Citation</a><a class="dblinks" href="/pmc/articles/PMC4410689/" ref="ordinalpos=1778&amp;ncbi_uid=7460665&amp;link_uid=PMC4410689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1779.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3621568/figure/f1-pjab-85-314/" ref="ordinalpos=1779&amp;ncbi_uid=4878147&amp;link_uid=PMC3621568" href="/pmc/articles/PMC3621568/"><img src="/pmc/articles/instance/3621568/bin/pjab-85-314-g001.gif" src-large="/pmc/articles/instance/3621568/bin/pjab-85-314-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3621568/" ref="ordinalpos=1779&amp;ncbi_uid=4878147&amp;link_uid=PMC3621568" image-link="/pmc/articles/PMC3621568/figure/f1-pjab-85-314/" class="imagepopup">Fig. 1.  From: Transforming growth factor-? <span class="highlight" style="background-color:">signaling</span> in epithelial-mesenchymal transition and progression of cancer. </a></p><div class="supp"><p class="details"><div>TGF-β transduces <span class="highlight" style="background-color:">signaling</span> through Smad and non-Smad <span class="highlight" style="background-color:">signaling</span> pathways. <i>Smad <span class="highlight" style="background-color:">pathway</span> (right):</i> TGF-β binds to TβRII and TβRI. TβRII phosphorylates TβRI, which activates Smad2 and Smad3. Activated Smad2/3 form complexes with Smad4, and translocate into the nucleus. The Smad complexes interact with various transcription factors and transcriptional co-activators, and regulate the transcription of target genes. <i>Non-Smad pathways (left):</i> TGF-β signals through multiple intra-cellular <span class="highlight" style="background-color:">signaling</span> cascades other than the Smad <span class="highlight" style="background-color:">pathway</span>. TGF-β receptors activate Erk, JNK, and p38 MAP kinases, PI3 kinase, and small GTPases such as Cdc42 and Rac. TGF-β receptors also bind Par6, induce TβRII kinase to phosphorylate Par6, and recruit Smurf1. Smurf1 then induces ubiquitylation and degradation of substrates such as RhoA.</div></p></div><div class="aux"><div class="resc">Kohei Miyazono. Proc Jpn Acad Ser B Phys Biol Sci. 2009 October;85(8):314-323.</div><p class="links"><a class="dblinks" href="/pubmed/19838011" ref="ordinalpos=1779&amp;ncbi_uid=4878147&amp;link_uid=19838011">Citation</a><a class="dblinks" href="/pmc/articles/PMC3621568/" ref="ordinalpos=1779&amp;ncbi_uid=4878147&amp;link_uid=PMC3621568">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1780.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3807000/figure/F2/" ref="ordinalpos=1780&amp;ncbi_uid=5418746&amp;link_uid=PMC3807000" href="/pmc/articles/PMC3807000/"><img src="/pmc/articles/instance/3807000/bin/en-22-133-g002.gif" src-large="/pmc/articles/instance/3807000/bin/en-22-133-g002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3807000/" ref="ordinalpos=1780&amp;ncbi_uid=5418746&amp;link_uid=PMC3807000" image-link="/pmc/articles/PMC3807000/figure/F2/" class="imagepopup">Fig. 2.  From: Neuronal Autophagy and Neurodevelopmental Disorders. </a></p><div class="supp"><p class="details"><div>Autophagy-related common <span class="highlight" style="background-color:">pathway</span> in neurodevelopmental disorders. Activation of PI3K-Atk <span class="highlight" style="background-color:">signaling</span> suppresses TSC1 and TSC2 complex leading to inactivation of Rheb (isoform of Ras superfamily) and release of mTOR from mTORC1 (mTOR complex 1). Dysfunctions of FMRP as a PIKE (PI3K enhancer) repressor in fragile X syndrome and TSC1/TSC2 as an mTOR repressor in tuberous sclerosis produce defects of neuronal development and synaptic plasticity with autistic behavioral phenotypes. In autism, mutations of PTEN (phosphatase and tensin homolog deleted on chromosome ten) also induce abnormal hyperactivity of PI3K/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Besides mTOR-dependent <span class="highlight" style="background-color:">pathway</span>, mTOR-independent <span class="highlight" style="background-color:">pathway</span> such as Epac2-Rap <span class="highlight" style="background-color:">signaling</span> affects neurite formation and autism phenotypes. Epac2, autism candidate gene RAPGEF4, is activated by cAMP binding and then changes an inactivated form to activated form of Rap which can regulate calpain via PLCε <span class="highlight" style="background-color:">pathway</span>. Neurofibromin encoded by NF1 normally leads to inactivation of Ras. However, mutations of neurofibromin in Neurofibromatosis 1 causes overactivation of Ras/ERK and PI3K <span class="highlight" style="background-color:">signaling</span>, leading to inactivation of TSC1/TSC2 complex and release of mTOR from Rheb suppression.</div></p></div><div class="aux"><div class="resc">Kyung-Min Lee, et al. Exp Neurobiol. 2013 September;22(3):133-142.</div><p class="links"><a class="dblinks" href="/pubmed/24167408" ref="ordinalpos=1780&amp;ncbi_uid=5418746&amp;link_uid=24167408">Citation</a><a class="dblinks" href="/pmc/articles/PMC3807000/" ref="ordinalpos=1780&amp;ncbi_uid=5418746&amp;link_uid=PMC3807000">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1781.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4171368/figure/pone-0105814-g008/" ref="ordinalpos=1781&amp;ncbi_uid=6649770&amp;link_uid=PMC4171368" href="/pmc/articles/PMC4171368/"><img src="/pmc/articles/instance/4171368/bin/pone.0105814.g008.gif" src-large="/pmc/articles/instance/4171368/bin/pone.0105814.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4171368/" ref="ordinalpos=1781&amp;ncbi_uid=6649770&amp;link_uid=PMC4171368" image-link="/pmc/articles/PMC4171368/figure/pone-0105814-g008/" class="imagepopup">Figure 8. The proposed anti-apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induced by DP in normal and AR neutrophils..  From: Regulation of Constitutive Neutrophil Apoptosis Due to House Dust Mite Allergen in Normal and Allergic Rhinitis Subjects. </a></p><div class="supp"><p class="details"><div>Anti-apoptotic <span class="highlight" style="background-color:">signaling</span> due to DP involves the activation of PKCδ, ERK and NF-κB and is associated with suppression of procaspase 3 and procaspase 9 cleavage.</div></p></div><div class="aux"><div class="resc">Eun Hye Kim, et al. PLoS One. 2014;9(9):e105814.</div><p class="links"><a class="dblinks" href="/pubmed/25243400" ref="ordinalpos=1781&amp;ncbi_uid=6649770&amp;link_uid=25243400">Citation</a><a class="dblinks" href="/pmc/articles/PMC4171368/" ref="ordinalpos=1781&amp;ncbi_uid=6649770&amp;link_uid=PMC4171368">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1782.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC59496/figure/F4/" ref="ordinalpos=1782&amp;ncbi_uid=128313&amp;link_uid=PMC59496" href="/pmc/articles/PMC59496/"><img src="/pmc/articles/instance/59496/bin/pp1295804004.gif" src-large="/pmc/articles/instance/59496/bin/pp1295804004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC59496/" ref="ordinalpos=1782&amp;ncbi_uid=128313&amp;link_uid=PMC59496" image-link="/pmc/articles/PMC59496/figure/F4/" class="imagepopup">Figure 4.  From: Changes in Cytosolic pH within Arabidopsis Root Columella Cells Play a Key Role in the Early <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> for Root Gravitropism. </a></p><div class="supp"><p class="details"><div>A, Results of modifier experiments showing that alkalinization of the root tip causes disruption of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ●, Control; ○, 100 μ<span style="font-variant: small-caps" class="small-caps">m</span> benzoic acid; ▾, 1.25 μ<span style="font-variant: small-caps" class="small-caps">m</span> bafilomycin A1; ▿, 0.3 μ<span style="font-variant: small-caps" class="small-caps">m</span> bafilomycin A1. B, Results of modifier experiments showing that acidification of the root tip causes enhancement of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ●, Control; ○, 50 μ<span style="font-variant: small-caps" class="small-caps">m</span> procaine; ▾, 500 μ<span style="font-variant: small-caps" class="small-caps">m</span> methylamine; ▿, 100 μ<span style="font-variant: small-caps" class="small-caps">m</span> procaine. Diagrams of Arabidopsis seedlings demonstrating the direction of roots bending at various times during the experiment. Stimulus 2 was applied at time = 0. Experiments are averages of 20 separate roots; for clarity, <span style="font-variant: small-caps" class="small-caps">se</span>s are not shown, but are approximately ±2 degrees of curvature for each point.</div></p></div><div class="aux"><div class="resc">Amie Caroline Scott, et al. Plant Physiol. 1999 December;121(4):1291-1298.</div><p class="links"><a class="dblinks" href="/pubmed/10594116" ref="ordinalpos=1782&amp;ncbi_uid=128313&amp;link_uid=10594116">Citation</a><a class="dblinks" href="/pmc/articles/PMC59496/" ref="ordinalpos=1782&amp;ncbi_uid=128313&amp;link_uid=PMC59496">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1783.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2933269/figure/F1/" ref="ordinalpos=1783&amp;ncbi_uid=3186169&amp;link_uid=PMC2933269" href="/pmc/articles/PMC2933269/"><img src="/pmc/articles/instance/2933269/bin/nihms210942f1.gif" src-large="/pmc/articles/instance/2933269/bin/nihms210942f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2933269/" ref="ordinalpos=1783&amp;ncbi_uid=3186169&amp;link_uid=PMC2933269" image-link="/pmc/articles/PMC2933269/figure/F1/" class="imagepopup">Figure 1.  From: A FoxL in the Smad House: Activin Regulation of FSH. </a></p><div class="supp"><p class="details"><div>Activin <span class="highlight" style="background-color:">signaling</span> regulates transcription of the FSHβ gene. Activin <span class="highlight" style="background-color:">signaling</span>, via ActRIIA/B and ALK4/7 receptors, results in the phosphorylation of Smad2 and 3, which then dimerize with the common Smad4 and translocate into the nucleus of the cell. Activated Smad3, and potentially Smad2, regulate transcription of FSHβ and other specific target genes. The inhibitory Smad7 prevents the phosphorylation of Smad3. Activin <span class="highlight" style="background-color:">signaling</span> also results in activation of the p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, potentially via TAK1, however, it is still unclear how p38 relays activin <span class="highlight" style="background-color:">signaling</span> to the FSHβ promoter.</div></p></div><div class="aux"><div class="resc">Djurdjica Coss, et al. Trends Endocrinol Metab. ;21(9):562-568.</div><p class="links"><a class="dblinks" href="/pubmed/20598900" ref="ordinalpos=1783&amp;ncbi_uid=3186169&amp;link_uid=20598900">Citation</a><a class="dblinks" href="/pmc/articles/PMC2933269/" ref="ordinalpos=1783&amp;ncbi_uid=3186169&amp;link_uid=PMC2933269">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1784.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3864863/figure/pone-0081937-g007/" ref="ordinalpos=1784&amp;ncbi_uid=5614041&amp;link_uid=PMC3864863" href="/pmc/articles/PMC3864863/"><img src="/pmc/articles/instance/3864863/bin/pone.0081937.g007.gif" src-large="/pmc/articles/instance/3864863/bin/pone.0081937.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3864863/" ref="ordinalpos=1784&amp;ncbi_uid=5614041&amp;link_uid=PMC3864863" image-link="/pmc/articles/PMC3864863/figure/pone-0081937-g007/" class="imagepopup">Figure 7. Stress condition and its relation with Ras/Rac1/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and epigenetic mechanisms regulation..  From: Ras and Rac1, Frequently Mutated in Melanomas, Are Activated by Superoxide Anion, Modulate Dnmt1 Level and Are Causally Related to Melanocyte Malignant Transformation. </a></p><div class="supp"><p class="details"><div>Ras/Rac1/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> seems to be regulated by and regulates superoxide anion production during melan-a anchorage blockade and its activation could be responsible for the high Dnmt1 protein level and changes in global DNA methylation during the loss of cell adhesion. Anchorage blockade might promote an epigenetic reprogramming in order to adapt the cells to the new microenvironmental condition and contribute to the acquisition of a malignant phenotype.</div></p></div><div class="aux"><div class="resc">Fernanda Molognoni, et al. PLoS One. 2013;8(12):e81937.</div><p class="links"><a class="dblinks" href="/pubmed/24358134" ref="ordinalpos=1784&amp;ncbi_uid=5614041&amp;link_uid=24358134">Citation</a><a class="dblinks" href="/pmc/articles/PMC3864863/" ref="ordinalpos=1784&amp;ncbi_uid=5614041&amp;link_uid=PMC3864863">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1785.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013649/figure/f7-ijms-15-06592/" ref="ordinalpos=1785&amp;ncbi_uid=6111644&amp;link_uid=PMC4013649" href="/pmc/articles/PMC4013649/"><img src="/pmc/articles/instance/4013649/bin/ijms-15-06592f7.gif" src-large="/pmc/articles/instance/4013649/bin/ijms-15-06592f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013649/" ref="ordinalpos=1785&amp;ncbi_uid=6111644&amp;link_uid=PMC4013649" image-link="/pmc/articles/PMC4013649/figure/f7-ijms-15-06592/" class="imagepopup">Figure 7. From: A20 Overexpression Inhibits Lipopolysaccharide-Induced NF-?B Activation, TRAF6 and CD40 Expression in Rat Peritoneal Mesothelial Cells. </a></p><div class="supp"><p class="details"><div>Effects of A20 on LPS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A20 depress the inflammatory response induced by LPS through negatively regulated the relevant function of adaptors in LPS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Xun-Liang Zou, et al. Int J Mol Sci. 2014 April;15(4):6592-6608.</div><p class="links"><a class="dblinks" href="/pubmed/24747594" ref="ordinalpos=1785&amp;ncbi_uid=6111644&amp;link_uid=24747594">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013649/" ref="ordinalpos=1785&amp;ncbi_uid=6111644&amp;link_uid=PMC4013649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1786.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3492858/figure/F2/" ref="ordinalpos=1786&amp;ncbi_uid=5433564&amp;link_uid=PMC3492858" href="/pmc/articles/PMC3492858/"><img src="/pmc/articles/instance/3492858/bin/zh10221266850002.gif" src-large="/pmc/articles/instance/3492858/bin/zh10221266850002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3492858/" ref="ordinalpos=1786&amp;ncbi_uid=5433564&amp;link_uid=PMC3492858" image-link="/pmc/articles/PMC3492858/figure/F2/" class="imagepopup">Fig. 2. From: The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> effector TCF7L2 is upregulated by insulin and represses hepatic gluconeogenesis. </a></p><div class="supp"><p class="details"><div>Feeding stimulates TCF7L2 expression. <i>A</i>: effect of fasting and refeeding (4 h) on hepatic TCF7L2 expression in C57BL/6 mice by Western blotting. Cyclin D1, a known downstream target of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Representative blot (<i>n</i> = 6/group). <i>B</i> and <i>C</i>: densitometric analysis of <i>A</i>. Values are presented as means ± SE. *<i>P</i> &lt; 0.05 and **<i>P</i> &lt; 0.01 vs. fasted mouse group.</div></p></div><div class="aux"><div class="resc">Wilfred Ip, et al. Am J Physiol Endocrinol Metab. 2012 November 1;303(9):E1166-E1176.</div><p class="links"><a class="dblinks" href="/pubmed/22967502" ref="ordinalpos=1786&amp;ncbi_uid=5433564&amp;link_uid=22967502">Citation</a><a class="dblinks" href="/pmc/articles/PMC3492858/" ref="ordinalpos=1786&amp;ncbi_uid=5433564&amp;link_uid=PMC3492858">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1787.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4236746/figure/F3/" ref="ordinalpos=1787&amp;ncbi_uid=6854107&amp;link_uid=PMC4236746" href="/pmc/articles/PMC4236746/"><img src="/pmc/articles/instance/4236746/bin/s12918-014-0099-x-3.gif" src-large="/pmc/articles/instance/4236746/bin/s12918-014-0099-x-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4236746/" ref="ordinalpos=1787&amp;ncbi_uid=6854107&amp;link_uid=PMC4236746" image-link="/pmc/articles/PMC4236746/figure/F3/" class="imagepopup">Figure 3.  From: KeyPathwayMiner 4.0: condition-specific <span class="highlight" style="background-color:">pathway</span> analysis by combining multiple omics studies and networks with Cytoscape. </a></p><div class="supp"><p class="details"><div><b>Largest key <span class="highlight" style="background-color:">pathway</span> obtained for the colorectal cancer case study.</b> It consists of 35 proteins. The colorectal cancer associated genes are highlighted with red border. PI3K <span class="highlight" style="background-color:">signaling</span> associated genes are highlighted with blue border.</div></p></div><div class="aux"><div class="resc">Nicolas Alcaraz, et al. BMC Syst Biol. 2014;8:99-99.</div><p class="links"><a class="dblinks" href="/pubmed/25134827" ref="ordinalpos=1787&amp;ncbi_uid=6854107&amp;link_uid=25134827">Citation</a><a class="dblinks" href="/pmc/articles/PMC4236746/" ref="ordinalpos=1787&amp;ncbi_uid=6854107&amp;link_uid=PMC4236746">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1788.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151255/figure/fig5/" ref="ordinalpos=1788&amp;ncbi_uid=247098&amp;link_uid=PMC151255" href="/pmc/articles/PMC151255/"><img src="/pmc/articles/instance/151255/bin/01-0455f5.gif" src-large="/pmc/articles/instance/151255/bin/01-0455f5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151255/" ref="ordinalpos=1788&amp;ncbi_uid=247098&amp;link_uid=PMC151255" image-link="/pmc/articles/PMC151255/figure/fig5/" class="imagepopup">Figure 5. From: Sugar Sensing and <span class="highlight" style="background-color:">Signaling</span> in Plants. </a></p><div class="supp"><p class="details"><div>Genetic Model of Interactions between Sugar and Hormone <span class="highlight" style="background-color:">Signaling</span> in Arabidopsis.<br />The <i>gin</i> phenotype (shown in Figure 4) is mimicked by ethylene precursor treatment of wild-type plants and is displayed in constitutive ethylene biosynthesis (<i>eto1</i>) and constitutive ethylene <span class="highlight" style="background-color:">signaling</span> (<i>ctr1</i>) mutants, whereas the ethylene-insensitive mutants <i>etr1-1</i> and <i>ein2</i> exhibit the <i>glo</i> phenotype (shown in Figure 4). Epistasis analysis with the <i>gin1 etr1</i> and <i>gin1 ein2</i> double mutants puts GIN1 downstream of the ETR1 receptor and EIN2 (Zhou et al., 1998; W.-H. Cheng and J. Sheen, unpublished data). Thus, Glc and ethylene <span class="highlight" style="background-color:">signaling</span> pathways antagonize each other (Zhou et al., 1998; W.-H. Cheng and J. Sheen, unpublished data). However, the triple response is not affected by Glc. The <i>gin1</i>, <i>sis4</i>, and <i>isi4</i> mutants are allelic to <i>aba2</i> (Laby et al., 2000; Rook et al., 2001; W.-H. Cheng and J. Sheen, unpublished data). <i>ABA2</i> encodes a short-chain dehydrogenase/reductase (SDR1) that is involved in the second to last step of ABA biosynthesis (Rook et al., 2001; W.-H. Cheng and J. Sheen, unpublished data; P.L. Rodríguez, personal communication; Seo and Koshiba, 2002) and is controlled directly by Glc (W.-H. Cheng and J. Sheen, unpublished data). Other ABA-deficient mutants (<i>aba1-1</i>, <i>aba2-1</i>, and <i>aba3-2</i>) also are Glc insensitive (Arenas-Huertero et al., 2000; Huijser et al., 2000; Laby et al., 2000). <i>ABA1</i> and <i>ABA3</i> are important for ABA biosynthesis and are regulated directly by Glc (W.-H. Cheng and J. Sheen, unpublished data). Characterization of the <i>gin5</i> mutant shows the requirement of Glc-specific ABA accumulation for HXK-mediated Glc <span class="highlight" style="background-color:">signaling</span> (Arenas-Huertero et al., 2000). In addition, <i>gin6</i>, s<i>un6</i>, <i>sis5</i>, and <i>isi3</i> are allelic to <i>abi4</i>, an ABA-insensitive mutant (Arenas-Huertero et al., 2000; Huijser et al., 2000; Laby et al., 2000; Rook et al., 2001). Glc activation of <i>ABI4</i>, which encodes an AP2 domain transcription factor (Finkelstein et al., 1998), requires ABA, although <i>ABI4</i> is not induced by ABA directly (Arenas-Huertero et al., 2000; Soderman et al., 2000; W.-H. Cheng and J. Sheen, unpublished data). The <i>abi5</i> mutant also is Glc insensitive. Glc activates <i>ABI5</i> directly (W.-H. Cheng and J. Sheen, unpublished data), encoding a basic Leu zipper transcription factor (Finkelstein and Lynch, 2000b). However, other ABA-insensitive <span class="highlight" style="background-color:">signaling</span> mutants (<i>abi1-1</i>, <i>abi2-1</i>, and <i>abi3-1</i>) do not exhibit the <i>gin</i> phenotype, as do <i>abi4</i> and <i>abi5</i> mutants (Arenas-Huertero et al., 2000; Huijser et al., 2000; Laby et al., 2000), suggesting that a distinct ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in Glc <span class="highlight" style="background-color:">signaling</span>. In summary, Glc activates ABA biosynthesis and ABA <span class="highlight" style="background-color:">signaling</span>, and both antagonize ethylene <span class="highlight" style="background-color:">signaling</span> (W.-H. Cheng and J. Sheen, unpublished data). It remains possible that Glc also inhibits ethylene <span class="highlight" style="background-color:">signaling</span> directly. The <i>AtHXK1</i> mutant (<i>gin2</i>) affects ABA and ethylene <span class="highlight" style="background-color:">signaling</span> but also displays reduced sensitivity to auxin and increased sensitivity to cytokinin (L. Zhou and J. Sheen, unpublished data).</div></p></div><div class="aux"><div class="resc">Filip Rolland, et al. Plant Cell. 2002;14(Suppl):s185-s205.</div><p class="links"><a class="dblinks" href="/pubmed/12045277" ref="ordinalpos=1788&amp;ncbi_uid=247098&amp;link_uid=12045277">Citation</a><a class="dblinks" href="/pmc/articles/PMC151255/" ref="ordinalpos=1788&amp;ncbi_uid=247098&amp;link_uid=PMC151255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1789.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4122412/figure/pone-0104156-g008/" ref="ordinalpos=1789&amp;ncbi_uid=6484522&amp;link_uid=PMC4122412" href="/pmc/articles/PMC4122412/"><img src="/pmc/articles/instance/4122412/bin/pone.0104156.g008.gif" src-large="/pmc/articles/instance/4122412/bin/pone.0104156.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4122412/" ref="ordinalpos=1789&amp;ncbi_uid=6484522&amp;link_uid=PMC4122412" image-link="/pmc/articles/PMC4122412/figure/pone-0104156-g008/" class="imagepopup">Figure 8. Diagrammatic representation of proposed mechanisms of interaction between eNOS deficiency, metabolic stress and insulin-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in coronary endothelial cells..  From: Partial Deletion of eNOS Gene Causes Hyperinsulinemic State, Unbalance of Cardiac Insulin <span class="highlight" style="background-color:">Signaling</span> Pathways and Coronary Dysfunction Independently of High Fat Diet. </a></p><div class="aux"><div class="resc">Cecilia Vecoli, et al. PLoS One. 2014;9(8):e104156.</div><p class="links"><a class="dblinks" href="/pubmed/25093405" ref="ordinalpos=1789&amp;ncbi_uid=6484522&amp;link_uid=25093405">Citation</a><a class="dblinks" href="/pmc/articles/PMC4122412/" ref="ordinalpos=1789&amp;ncbi_uid=6484522&amp;link_uid=PMC4122412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1790.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2997678/figure/F7/" ref="ordinalpos=1790&amp;ncbi_uid=3585235&amp;link_uid=PMC2997678" href="/pmc/articles/PMC2997678/"><img src="/pmc/articles/instance/2997678/bin/nihms-247584-f0007.gif" src-large="/pmc/articles/instance/2997678/bin/nihms-247584-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2997678/" ref="ordinalpos=1790&amp;ncbi_uid=3585235&amp;link_uid=PMC2997678" image-link="/pmc/articles/PMC2997678/figure/F7/" class="imagepopup">Figure 7.  From: POSTTRANSLATIONAL MODIFICATION OF GLUTAMIC ACID DECARBOXYLASE 67 BY INTERMITTENT HYPOXIA: Evidence for the involvement of dopamine D1 receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic presentation of IH-induced activation of D1R-cAMP-PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> facilitating GAD67 phosphorylation mediated GAD67 inhibition and GABA down regulation. Key:▲– up-regulation; ▼– down-regulation; P-GAD67 – phospho-GAD67</div></p></div><div class="aux"><div class="resc">Gayatri Raghuraman, et al. J Neurochem. ;115(6):1568-1578.</div><p class="links"><a class="dblinks" href="/pubmed/20969567" ref="ordinalpos=1790&amp;ncbi_uid=3585235&amp;link_uid=20969567">Citation</a><a class="dblinks" href="/pmc/articles/PMC2997678/" ref="ordinalpos=1790&amp;ncbi_uid=3585235&amp;link_uid=PMC2997678">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1791.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3586073/figure/pone-0057541-g009/" ref="ordinalpos=1791&amp;ncbi_uid=4685322&amp;link_uid=PMC3586073" href="/pmc/articles/PMC3586073/"><img src="/pmc/articles/instance/3586073/bin/pone.0057541.g009.gif" src-large="/pmc/articles/instance/3586073/bin/pone.0057541.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3586073/" ref="ordinalpos=1791&amp;ncbi_uid=4685322&amp;link_uid=PMC3586073" image-link="/pmc/articles/PMC3586073/figure/pone-0057541-g009/" class="imagepopup">Figure 9. A putative model for the molecular <span class="highlight" style="background-color:">pathway</span> linking hypoxic stress to decreased AQP5 expression..  From: Hypoxia and Hypoxia Mimetics Decrease Aquaporin 5 (AQP5) Expression through Both Hypoxia Inducible Factor-1? and Proteasome-Mediated Pathways. </a></p><div class="supp"><p class="details"><div>Hypoxic stress activates HIF-1α via the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. HIF-1α activates a repressor of AQP5 transcription via a proteasome-mediated mechanism (which could act by degrading inhibitors of the repressor protein).</div></p></div><div class="aux"><div class="resc">Jitesh D. Kawedia, et al. PLoS One. 2013;8(3):e57541.</div><p class="links"><a class="dblinks" href="/pubmed/23469202" ref="ordinalpos=1791&amp;ncbi_uid=4685322&amp;link_uid=23469202">Citation</a><a class="dblinks" href="/pmc/articles/PMC3586073/" ref="ordinalpos=1791&amp;ncbi_uid=4685322&amp;link_uid=PMC3586073">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1792.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4083685/figure/F2/" ref="ordinalpos=1792&amp;ncbi_uid=6347575&amp;link_uid=PMC4083685" href="/pmc/articles/PMC4083685/"><img src="/pmc/articles/instance/4083685/bin/nihms597163f2.gif" src-large="/pmc/articles/instance/4083685/bin/nihms597163f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4083685/" ref="ordinalpos=1792&amp;ncbi_uid=6347575&amp;link_uid=PMC4083685" image-link="/pmc/articles/PMC4083685/figure/F2/" class="imagepopup">Figure 2. 4F treatment down-regulates LPS-mediated expression of genes of the MyD88-dependent and MyD88-independent TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Regulation of Pattern Recognition Receptors by the Apolipoprotein A-I Mimetic Peptide 4F. </a></p><div class="supp"><p class="details"><div>MDMs were pre-treated with 4F (white bars) or vehicle (black bars) for 7days and then stimulated with LPS (1µg/ml) for 18hrs. RNA was extracted and transcriptional profiling was carried out using the Affymetrix Human Gene ST1.0 array. Absolute mRNA levels obtained were normalized to unstimulated MDMs, pre-treated with vehicle. Data are means ± SEM (n=4; p&lt;0.05 across all treatments). <b>(A)</b> Effect of LPS on the expression of surface TLRs <b>(B)</b> Effect of LPS on adaptor molecules <b>(C)</b> Effect of LPS on enzymes of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and <b>(D)</b> Effect of LPS on expression of genes of the MyD88-independent <span class="highlight" style="background-color:">pathway</span>. (<b>E</b>) mRNA levels of key genes in the TLR <span class="highlight" style="background-color:">pathway</span> in MDMs pretreated with vehicle (black bars) or 4F (white bars) for 7days and stimulated with LPS (1µg/ml) for 18hrs. The data are normalized to β-actin. Values are means ± SEM (n=4).</div></p></div><div class="aux"><div class="resc">C. Roger White, et al. Arterioscler Thromb Vasc Biol. ;32(11):2631-2639.</div><p class="links"><a class="dblinks" href="/pubmed/22982462" ref="ordinalpos=1792&amp;ncbi_uid=6347575&amp;link_uid=22982462">Citation</a><a class="dblinks" href="/pmc/articles/PMC4083685/" ref="ordinalpos=1792&amp;ncbi_uid=6347575&amp;link_uid=PMC4083685">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1793.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4314211/figure/F8/" ref="ordinalpos=1793&amp;ncbi_uid=7119448&amp;link_uid=PMC4314211" href="/pmc/articles/PMC4314211/"><img src="/pmc/articles/instance/4314211/bin/nihms141014f8.gif" src-large="/pmc/articles/instance/4314211/bin/nihms141014f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4314211/" ref="ordinalpos=1793&amp;ncbi_uid=7119448&amp;link_uid=PMC4314211" image-link="/pmc/articles/PMC4314211/figure/F8/" class="imagepopup">Figure 8. The cGMP signal transduction <span class="highlight" style="background-color:">pathway</span> in the GG.  From: The Grueneberg Ganglion olfactory subsystem employs a cGMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Binding of atrial natriuretic peptide (ANP), B-type natriuretic peptide (BNP), or other as-of-yet unidentified ligands to pGC-A or unknown ligands to pGC-G induces the conversion of GTP to cGMP by the particulate guanylyl cyclases. The produced cGMP can then open CNGA3 channels to depolarize the GG neuron, induce additional <span class="highlight" style="background-color:">signaling</span> through cGKII, or be degraded to GMP by Pde2a.</div></p></div><div class="aux"><div class="resc">Cambrian Y. Liu, et al. J Comp Neurol. ;516(1):36-48.</div><p class="links"><a class="dblinks" href="/pubmed/19565523" ref="ordinalpos=1793&amp;ncbi_uid=7119448&amp;link_uid=19565523">Citation</a><a class="dblinks" href="/pmc/articles/PMC4314211/" ref="ordinalpos=1793&amp;ncbi_uid=7119448&amp;link_uid=PMC4314211">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1794.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4246620/figure/f1-ol-09-01-0235/" ref="ordinalpos=1794&amp;ncbi_uid=6891250&amp;link_uid=PMC4246620" href="/pmc/articles/PMC4246620/"><img src="/pmc/articles/instance/4246620/bin/OL-09-01-0235-g00.gif" src-large="/pmc/articles/instance/4246620/bin/OL-09-01-0235-g00.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4246620/" ref="ordinalpos=1794&amp;ncbi_uid=6891250&amp;link_uid=PMC4246620" image-link="/pmc/articles/PMC4246620/figure/f1-ol-09-01-0235/" class="imagepopup">Figure 1.  From: The expression of Wnt-1 inducible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein-2 in astrocytoma: Correlation between pathological grade and clinical outcome. </a></p><div class="supp"><p class="details"><div>Reverse transcription-polymerase chain reaction analysis of WISP-2 expression levels in normal brain tissues and human astrocytomas of different grades. GAPDH was used as an internal standard. Representative images of electrophoresis of WISP-2 mRNA. A one-way analysis of variance was used for tests of significance: P&lt;0.01, significant difference from the normal control. WISP-2, Wnt-1 inducible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein-2; N, normal brain tissue; I–IV, astrocytoma grades.</div></p></div><div class="aux"><div class="resc">GELEI XIAO, et al. Oncol Lett. 2015 January;9(1):235-240.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1794&amp;ncbi_uid=6891250&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4246620/" ref="ordinalpos=1794&amp;ncbi_uid=6891250&amp;link_uid=PMC4246620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1795.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3162414/figure/F8/" ref="ordinalpos=1795&amp;ncbi_uid=3546169&amp;link_uid=PMC3162414" href="/pmc/articles/PMC3162414/"><img src="/pmc/articles/instance/3162414/bin/zbc0401176450008.gif" src-large="/pmc/articles/instance/3162414/bin/zbc0401176450008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3162414/" ref="ordinalpos=1795&amp;ncbi_uid=3546169&amp;link_uid=PMC3162414" image-link="/pmc/articles/PMC3162414/figure/F8/" class="imagepopup">FIGURE 8. From: Involvement of Protein Kinase C?/Extracellular Signal-regulated Kinase/Poly(ADP-ribose) Polymerase-1 (PARP-1) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Histamine-induced Up-regulation of Histamine H1 Receptor Gene Expression in HeLa Cells. </a></p><div class="supp"><p class="details"><div><b>Schematic presentation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in histamine-induced up-regulation of H1R mRNA expression in HeLa cells.</b> Binding of histamine to H1R activates PKCδ and translocates PKCδ from the cytosol to the Golgi where PKCδ phosphorylates MEK and ERK. Phosphorylated ERK then activates PARP-1. Poly(ADP)-ribosylation of an unknown protein (or PARP-1 itself) by PARP-1 causes activation of a transcription factor, resulting in up-regulation of H1R gene transcription.</div></p></div><div class="aux"><div class="resc">Hiroyuki Mizuguchi, et al. J Biol Chem. 2011 September 2;286(35):30542-30551.</div><p class="links"><a class="dblinks" href="/pubmed/21730054" ref="ordinalpos=1795&amp;ncbi_uid=3546169&amp;link_uid=21730054">Citation</a><a class="dblinks" href="/pmc/articles/PMC3162414/" ref="ordinalpos=1795&amp;ncbi_uid=3546169&amp;link_uid=PMC3162414">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1796.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2660258/figure/f4/" ref="ordinalpos=1796&amp;ncbi_uid=1732019&amp;link_uid=PMC2660258" href="/pmc/articles/PMC2660258/"><img src="/pmc/articles/instance/2660258/bin/zh00030958620004.gif" src-large="/pmc/articles/instance/2660258/bin/zh00030958620004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2660258/" ref="ordinalpos=1796&amp;ncbi_uid=1732019&amp;link_uid=PMC2660258" image-link="/pmc/articles/PMC2660258/figure/f4/" class="imagepopup">Fig. 4. From: Simvastatin enhances bone marrow stromal cell differentiation into endothelial cells via notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Transcript levels of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components. Q-RT-PCR showed that DLL4, Notch1, and Notch4 increased significantly when compared with control group, whereas no significant difference was found in mRNA levels of Jagged1 and Notch2 between the two groups. Results shown are means ± SE from at least three independent experiments.</div></p></div><div class="aux"><div class="resc">Jian Xu, et al. Am J Physiol Cell Physiol. 2009 March;296(3):C535-C543.</div><p class="links"><a class="dblinks" href="/pubmed/19109527" ref="ordinalpos=1796&amp;ncbi_uid=1732019&amp;link_uid=19109527">Citation</a><a class="dblinks" href="/pmc/articles/PMC2660258/" ref="ordinalpos=1796&amp;ncbi_uid=1732019&amp;link_uid=PMC2660258">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1797.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2944186/figure/F1/" ref="ordinalpos=1797&amp;ncbi_uid=2477478&amp;link_uid=PMC2944186" href="/pmc/articles/PMC2944186/"><img src="/pmc/articles/instance/2944186/bin/1476-4598-9-236-1.gif" src-large="/pmc/articles/instance/2944186/bin/1476-4598-9-236-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2944186/" ref="ordinalpos=1797&amp;ncbi_uid=2477478&amp;link_uid=PMC2944186" image-link="/pmc/articles/PMC2944186/figure/F1/" class="imagepopup">Figure 1.  From: Convergence between Wnt-?-catenin and EGFR <span class="highlight" style="background-color:">signaling</span> in cancer. </a></p><div class="supp"><p class="details"><div><b>Convergence between Wnt and EGFR pathways</b>. Wnt binds to Frizzled. Frizzled transactivates EGFR <span class="highlight" style="background-color:">signaling</span> by matrix metalloproteinase-mediated release of soluble EGFR ligands. Upon activation, EGFR could transactivate β-catenin, possibly through receptor tyrosine kinase-PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, and β-catenin might also form heterodimer with EGFR and activate EGFR <span class="highlight" style="background-color:">pathway</span>. NKD2 binds to TGFα and escorts it to the plasma membrane, where TGFα gets released, and then NKD2 binds to Dvl-1 and targets it for mutual degradation, thus maintains the epithelial cell homeostasis.</div></p></div><div class="aux"><div class="resc">Tianhui Hu, et al. Mol Cancer. 2010;9:236-236.</div><p class="links"><a class="dblinks" href="/pubmed/20828404" ref="ordinalpos=1797&amp;ncbi_uid=2477478&amp;link_uid=20828404">Citation</a><a class="dblinks" href="/pmc/articles/PMC2944186/" ref="ordinalpos=1797&amp;ncbi_uid=2477478&amp;link_uid=PMC2944186">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1798.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3337193/figure/F1/" ref="ordinalpos=1798&amp;ncbi_uid=4030797&amp;link_uid=PMC3337193" href="/pmc/articles/PMC3337193/"><img src="/pmc/articles/instance/3337193/bin/psb-6-1993-g1.gif" src-large="/pmc/articles/instance/3337193/bin/psb-6-1993-g1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3337193/" ref="ordinalpos=1798&amp;ncbi_uid=4030797&amp;link_uid=PMC3337193" image-link="/pmc/articles/PMC3337193/figure/F1/" class="imagepopup">Figure 1. From: EARLY RESPONSIVE to DEHYDRATION 15, a new transcription factor that integrates stress <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>ERD15 is involved in multiple stress response <span class="highlight" style="background-color:">signaling</span> pathways. (A) The induction of <i>Gm</i>ERD15 gene expression by ER or osmotic stress leads to the activation of the NRP-mediated cell death <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) The induction of <i>At</i>ERD15 gene expression by abiotic or biotic stress attenuates ABA <span class="highlight" style="background-color:">signaling</span> and induces PR genes, leading to the enhanced resistance to pathogens.</div></p></div><div class="aux"><div class="resc">Murilo S. Alves, et al. Plant Signal Behav. 2011 December 1;6(12):1993-1996.</div><p class="links"><a class="dblinks" href="/pubmed/22105026" ref="ordinalpos=1798&amp;ncbi_uid=4030797&amp;link_uid=22105026">Citation</a><a class="dblinks" href="/pmc/articles/PMC3337193/" ref="ordinalpos=1798&amp;ncbi_uid=4030797&amp;link_uid=PMC3337193">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1799.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2084312/figure/F2/" ref="ordinalpos=1799&amp;ncbi_uid=1093056&amp;link_uid=PMC2084312" href="/pmc/articles/PMC2084312/"><img src="/pmc/articles/instance/2084312/bin/zpq0440781850002.gif" src-large="/pmc/articles/instance/2084312/bin/zpq0440781850002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2084312/" ref="ordinalpos=1799&amp;ncbi_uid=1093056&amp;link_uid=PMC2084312" image-link="/pmc/articles/PMC2084312/figure/F2/" class="imagepopup">Fig. 2. From: Functional screening identifies miR-315 as a potent activator of Wingless <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Placement and specificity of miR-315 activity relative to the Wg <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) Epistasis test. The effect of ectopic miR-315 on TCF-luc was determined in the presence of dsRNAs against various positive components of the canonical Wg <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In clone 8 cells, miR-315 acts downstream of the Wg ligand and its receptors (Fz and Arr) but upstream of the nuclear components legless, Arm, and dTCF. (<i>B</i>) Specificity test. miR-315 had no effect on a reporter for transcriptional output of the JAK/STAT <span class="highlight" style="background-color:">pathway</span> (10X-STAT92-luc) and no effect on a reporter for transcriptional output of the Hedgehog (Hh) <span class="highlight" style="background-color:">pathway</span> (ptcΔ136-luc) under either Hh-stimulated or unstimulated conditions. RLU, relative luciferase units for the various reporters.</div></p></div><div class="aux"><div class="resc">Serena J. Silver, et al. Proc Natl Acad Sci U S A. 2007 November 13;104(46):18151-18156.</div><p class="links"><a class="dblinks" href="/pubmed/17989227" ref="ordinalpos=1799&amp;ncbi_uid=1093056&amp;link_uid=17989227">Citation</a><a class="dblinks" href="/pmc/articles/PMC2084312/" ref="ordinalpos=1799&amp;ncbi_uid=1093056&amp;link_uid=PMC2084312">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1800.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3688948/figure/pone-0066743-g007/" ref="ordinalpos=1800&amp;ncbi_uid=5056027&amp;link_uid=PMC3688948" href="/pmc/articles/PMC3688948/"><img src="/pmc/articles/instance/3688948/bin/pone.0066743.g007.gif" src-large="/pmc/articles/instance/3688948/bin/pone.0066743.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3688948/" ref="ordinalpos=1800&amp;ncbi_uid=5056027&amp;link_uid=PMC3688948" image-link="/pmc/articles/PMC3688948/figure/pone-0066743-g007/" class="imagepopup">Figure 7. <span class="highlight" style="background-color:">Signaling</span> by PAR2 in smooth muscle..  From: Distinctive G Protein-Dependent <span class="highlight" style="background-color:">Signaling</span> by Protease-Activated Receptor 2 (PAR2) in Smooth Muscle: Feedback Inhibition of RhoA by cAMP-Independent PKA. </a></p><div class="supp"><p class="details"><div>In gastric smooth muscle PAR2 receptors are coupled to G<sub>q</sub>, G<sub>13</sub> and Gi<sub>1/2</sub>, and stimulation of PI hydrolysis via Ga<sub>q</sub> and Gbg<sub>i1/2</sub>, and RhoA/Rho kinase via Ga<sub>13</sub>. Initial contraction is mediated via PLC-b/Ca<sup>2+</sup>/MLCK <span class="highlight" style="background-color:">pathway</span> whereas sustained contraction is mediated via RhoA/Rho kinase <span class="highlight" style="background-color:">pathway</span>. Although PAR2 receptors are coupled to inhibition of cAMP via Ga<sub> i1/2</sub>, activation of NF-kB <span class="highlight" style="background-color:">pathway</span> via Ga<sub>13</sub>/RhoA leads to degradation of IkB and activation of PKA that result in attenuation of Rho kinase activity in a feedback mechanism via phosphorylation of RhoA at Ser<sup>188</sup> by PKA.</div></p></div><div class="aux"><div class="resc">Wimolpak Sriwai, et al. PLoS One. 2013;8(6):e66743.</div><p class="links"><a class="dblinks" href="/pubmed/23825105" ref="ordinalpos=1800&amp;ncbi_uid=5056027&amp;link_uid=23825105">Citation</a><a class="dblinks" href="/pmc/articles/PMC3688948/" ref="ordinalpos=1800&amp;ncbi_uid=5056027&amp;link_uid=PMC3688948">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="17" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="18" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="19" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553ff746437716036ad09555">Saving and Managing Searches - My NCBI Help</a>
          <div class="ralinkpop offscreen_noflow">Saving and Managing Searches - My NCBI Help<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553fef47cbe05d491bf71396">signaling pathway AND (free full text[sb]) <span class="number">(225335)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553fef249892033523880147">signaling pathway <span class="number">(491236)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553fedbe437716036ace4747">PMC Help</a>
          <div class="ralinkpop offscreen_noflow">PMC Help<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553fedb5cbe05d491bf6bb4a">NCBI Help Manual</a>
          <div class="ralinkpop offscreen_noflow">NCBI Help Manual<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T17:13:53-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB2A2253FF7F01000000000037020A;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB2A2253FF7F01000000000037020A" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>