
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="1" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="8A1A393946D25221_0091SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC6E0553EBF9E10000000000090247" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene">Gene</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=12" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><span title="Inactive first page of results" class="inactive page_link">&lt;&lt; First</span><span title="Inactive previous page of results" class="inactive page_link prev">&lt; Prev</span><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="1" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="2" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="1" /></div>
            <div><h2 class="result_count">Results: 1 to 100 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3487967/figure/F2/" ref="ordinalpos=1&amp;ncbi_uid=4353701&amp;link_uid=PMC3487967" href="/pmc/articles/PMC3487967/"><img src="/pmc/articles/instance/3487967/bin/1471-2105-13-218-2.gif" src-large="/pmc/articles/instance/3487967/bin/1471-2105-13-218-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3487967/" ref="ordinalpos=1&amp;ncbi_uid=4353701&amp;link_uid=PMC3487967" image-link="/pmc/articles/PMC3487967/figure/F2/" class="imagepopup">Figure 2.  From: Multi-scale agent-based brain cancer modeling and prediction of TKI treatment response: Incorporating EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and angiogenesis. </a></p><div class="supp"><p class="details"><div><b>EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> connected to cell cycle <span class="highlight" style="background-color:">pathway</span> with TKIs to block the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is connected to the cell cycle <span class="highlight" style="background-color:">pathway</span>, and TKIs block the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Xiaoqiang Sun, et al. BMC Bioinformatics. 2012;13:218-218.</div><p class="links"><a class="dblinks" href="/pubmed/22935054" ref="ordinalpos=1&amp;ncbi_uid=4353701&amp;link_uid=22935054">Citation</a><a class="dblinks" href="/pmc/articles/PMC3487967/" ref="ordinalpos=1&amp;ncbi_uid=4353701&amp;link_uid=PMC3487967">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3070611/figure/F4/" ref="ordinalpos=2&amp;ncbi_uid=2874423&amp;link_uid=PMC3070611" href="/pmc/articles/PMC3070611/"><img src="/pmc/articles/instance/3070611/bin/1755-8794-4-23-4.gif" src-large="/pmc/articles/instance/3070611/bin/1755-8794-4-23-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3070611/" ref="ordinalpos=2&amp;ncbi_uid=2874423&amp;link_uid=PMC3070611" image-link="/pmc/articles/PMC3070611/figure/F4/" class="imagepopup">Figure 4.  From: An integrative approach to identifying cancer chemoresistance-associated pathways. </a></p><div class="supp"><p class="details"><div><b>One of experimental result from pathways intersection</b>. This figure demonstrates the experimental result after performing <span class="highlight" style="background-color:">pathway</span> interaction. Several pathways contributed to this result: the colorectal cancer related <span class="highlight" style="background-color:">pathway</span>, the hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The connected gene DVL (disheveled, dsh homolog) connects two critical pathways: the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Gatcliffe <i>et al</i>. suggested that WNT <span class="highlight" style="background-color:">signaling</span> plays a role in ovarian tumorigenesis [45]. WNT <span class="highlight" style="background-color:">signaling</span> has a significant influence on the embryonic development of the ovary and is also involved in normal follicular development and ovarian function [46,47]. The WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in ovarian cancer development via multiple, diverse mechanisms, including gene mutations and changes in <span class="highlight" style="background-color:">pathway</span> components such as extracellular inhibitors and intranuclear transcription cofactors. According to Wang <i>et al</i>., the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> passes signals to the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [48]. The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is known to be responsible for maintaining a balance between cell proliferation and death and, as such, plays an important role in the formation of many types of human tumors. In our computational results, WNT <span class="highlight" style="background-color:">signaling</span> connects the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through DVL gene, which indicates DVL is a critical gene for passing signals through pathways. In addition, the computational evidence provided by the values of betweenness centrality, degree and <i>p</i>-value indicate that DVL may be involved in platinum-based chemoresistance.</div></p></div><div class="aux"><div class="resc">Shih-Yi Chao, et al. BMC Med Genomics. 2011;4:23-23.</div><p class="links"><a class="dblinks" href="/pubmed/21429228" ref="ordinalpos=2&amp;ncbi_uid=2874423&amp;link_uid=21429228">Citation</a><a class="dblinks" href="/pmc/articles/PMC3070611/" ref="ordinalpos=2&amp;ncbi_uid=2874423&amp;link_uid=PMC3070611">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657316/figure/fig1/" ref="ordinalpos=3&amp;ncbi_uid=2257579&amp;link_uid=PMC2657316" href="/pmc/articles/PMC2657316/"><img src="/pmc/articles/instance/2824233/bin/sgas44-462-f1.gif" src-large="/pmc/articles/instance/2824233/bin/sgas44-462-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657316/" ref="ordinalpos=3&amp;ncbi_uid=2257579&amp;link_uid=PMC2657316" image-link="/pmc/articles/PMC2657316/figure/fig1/" class="imagepopup">Figure 1.  From: Possible regulation of genes associated with intracellular <span class="highlight" style="background-color:">signaling</span> cascade in rat liver regeneration. </a></p><div class="supp"><p class="details"><div>Initial and total changes in expression of 566 intracellular <span class="highlight" style="background-color:">signaling</span> cascade-associated genes in rat liver regeneration. The solid line denotes the up-regulated genes; the dashed line represents the down-regulated genes. A. The initial expression patterns. B. The total expression patterns.  = intracellular receptor-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>;  = second messenger-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>;  = nitric oxide-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; + = hormone-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>;  = carbohydrate-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>;  = protein kinase cascade;  = small GTPase-mediated signal transduction; * = ER-nuclear <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>;  = target of rampamycin (TOR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Cun-Shuan Xu, et al. Scand J Gastroenterol. 2009 April;44(1):462-470.</div><p class="links"><a class="dblinks" href="/pubmed/18991167" ref="ordinalpos=3&amp;ncbi_uid=2257579&amp;link_uid=18991167">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657316/" ref="ordinalpos=3&amp;ncbi_uid=2257579&amp;link_uid=PMC2657316">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g005/" ref="ordinalpos=4&amp;ncbi_uid=6232466&amp;link_uid=PMC4041745" href="/pmc/articles/PMC4041745/"><img src="/pmc/articles/instance/4041745/bin/pone.0098229.g005.gif" src-large="/pmc/articles/instance/4041745/bin/pone.0098229.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041745/" ref="ordinalpos=4&amp;ncbi_uid=6232466&amp;link_uid=PMC4041745" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g005/" class="imagepopup">Figure 5. GeneGo <span class="highlight" style="background-color:">pathway</span> Cytoskeleton remodeling..  From: Systematic <span class="highlight" style="background-color:">Pathway</span> Enrichment Analysis of a Genome-Wide Association Study on Breast Cancer Survival Reveals an Influence of Genes Involved in Cell Adhesion and Calcium <span class="highlight" style="background-color:">Signaling</span> on the Patients' Clinical Outcome. </a></p><div class="supp"><p class="details"><div>Barometer: 1=0.01 gene list; 2=general metastasis signature; 3=brain metastasis signature. orange=ECM-receptor interaction, purple=Adherens junction <span class="highlight" style="background-color:">pathway</span>, red=Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, pink=Focal adhesion <span class="highlight" style="background-color:">pathway</span>, yellow=TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, green=Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, blue=Smooth muscle contraction/relaxation.</div></p></div><div class="aux"><div class="resc">Andrea Woltmann, et al. PLoS One. 2014;9(6):e98229.</div><p class="links"><a class="dblinks" href="/pubmed/24886783" ref="ordinalpos=4&amp;ncbi_uid=6232466&amp;link_uid=24886783">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041745/" ref="ordinalpos=4&amp;ncbi_uid=6232466&amp;link_uid=PMC4041745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3922094/figure/F1/" ref="ordinalpos=5&amp;ncbi_uid=5810045&amp;link_uid=PMC3922094" href="/pmc/articles/PMC3922094/"><img src="/pmc/articles/instance/3922094/bin/2041-1480-5-7-1.gif" src-large="/pmc/articles/instance/3922094/bin/2041-1480-5-7-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3922094/" ref="ordinalpos=5&amp;ncbi_uid=5810045&amp;link_uid=PMC3922094" image-link="/pmc/articles/PMC3922094/figure/F1/" class="imagepopup">Figure 1.  From: The <span class="highlight" style="background-color:">pathway</span> ontology - updates and applications. </a></p><div class="supp"><p class="details"><div><b>The <span class="highlight" style="background-color:">pathway</span> ontology main nodes and positions of selected terms. A.</b> The five nodes of the <span class="highlight" style="background-color:">Pathway</span> Ontology. <b>B.</b> The term lipid hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the ontology showing the parent, siblings and children terms. <b>C.</b> The term insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the ontology showing the position of the term within the tree. Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is in a part_of relationship to the glucose and energy homeostasis <span class="highlight" style="background-color:">pathway</span> terms within the regulatory node and in an is_a relationship to peptide and protein hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> term within the <span class="highlight" style="background-color:">signaling</span> node.</div></p></div><div class="aux"><div class="resc">Victoria Petri, et al. J Biomed Semantics. 2014;5:7-7.</div><p class="links"><a class="dblinks" href="/pubmed/24499703" ref="ordinalpos=5&amp;ncbi_uid=5810045&amp;link_uid=24499703">Citation</a><a class="dblinks" href="/pmc/articles/PMC3922094/" ref="ordinalpos=5&amp;ncbi_uid=5810045&amp;link_uid=PMC3922094">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>6.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2561291/figure/pone-0003430-g005/" ref="ordinalpos=6&amp;ncbi_uid=1546526&amp;link_uid=PMC2561291" href="/pmc/articles/PMC2561291/"><img src="/pmc/articles/instance/2561291/bin/pone.0003430.g005.gif" src-large="/pmc/articles/instance/2561291/bin/pone.0003430.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2561291/" ref="ordinalpos=6&amp;ncbi_uid=1546526&amp;link_uid=PMC2561291" image-link="/pmc/articles/PMC2561291/figure/pone-0003430-g005/" class="imagepopup">Figure 5. Schematic representation of <span class="highlight" style="background-color:">Signaling</span> Flux Redistribution (SFR)..  From: <span class="highlight" style="background-color:">Signaling</span> Flux Redistribution at Toll-Like Receptor <span class="highlight" style="background-color:">Pathway</span> Junctions. </a></p><div class="supp"><p class="details"><div>(A) removal of MyD88 results in enhancement of TRAM-dependant <span class="highlight" style="background-color:">pathway</span>, (B) removal of TRAF6 results in enhancement of TRAM-dependant <span class="highlight" style="background-color:">pathway</span> downstream of TRIF, (C) overexpression of MyD88 downregulates TRAM-dependant <span class="highlight" style="background-color:">pathway</span>, (D) removal of TRAM does not enhance the MyD88-dependant <span class="highlight" style="background-color:">pathway</span> due to upstream intermediates. * <span class="highlight" style="background-color:">signaling</span> molecules/events upstream of TRAM [9], [25], [26].</div></p></div><div class="aux"><div class="resc">Kumar Selvarajoo, et al. PLoS ONE. 2008;3(10):e3430.</div><p class="links"><a class="dblinks" href="/pubmed/18927610" ref="ordinalpos=6&amp;ncbi_uid=1546526&amp;link_uid=18927610">Citation</a><a class="dblinks" href="/pmc/articles/PMC2561291/" ref="ordinalpos=6&amp;ncbi_uid=1546526&amp;link_uid=PMC2561291">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>7.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694443/figure/F8/" ref="ordinalpos=7&amp;ncbi_uid=1934988&amp;link_uid=PMC2694443" href="/pmc/articles/PMC2694443/"><img src="/pmc/articles/instance/2694443/bin/nihms114616f8.gif" src-large="/pmc/articles/instance/2694443/bin/nihms114616f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694443/" ref="ordinalpos=7&amp;ncbi_uid=1934988&amp;link_uid=PMC2694443" image-link="/pmc/articles/PMC2694443/figure/F8/" class="imagepopup">Fig. 8. A model of the relationship between BMPR1A and canonical Wnt <span class="highlight" style="background-color:">signaling</span> in mouse bone.  From: BMP <span class="highlight" style="background-color:">signaling</span> negatively regulates bone mass through sclerostin by inhibiting the canonical Wnt <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>BMPR1A <span class="highlight" style="background-color:">signaling</span> upregulates sclerostin expression, leading to an inhibition of canonical Wnt <span class="highlight" style="background-color:">signaling</span> and a decrease in bone mass by upregulating osteoclastogenesis through the RANKL-OPG <span class="highlight" style="background-color:">pathway</span>. Sclerostin, the <i>SOST</i> gene product, acts as a downstream effector of BMPR1A <span class="highlight" style="background-color:">signaling</span>, an inhibitor of canonical Wnt <span class="highlight" style="background-color:">signaling</span> and a bone mass-determining factor. Broken line indicates another possibility: that BMP <span class="highlight" style="background-color:">signaling</span> directly upregulates osteoclastogenesis through the RANKL-OPG <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nobuhiro Kamiya, et al. Development. ;135(22):3801-3811.</div><p class="links"><a class="dblinks" href="/pubmed/18927151" ref="ordinalpos=7&amp;ncbi_uid=1934988&amp;link_uid=18927151">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694443/" ref="ordinalpos=7&amp;ncbi_uid=1934988&amp;link_uid=PMC2694443">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>8.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3579804/figure/pone-0057168-g001/" ref="ordinalpos=8&amp;ncbi_uid=4673563&amp;link_uid=PMC3579804" href="/pmc/articles/PMC3579804/"><img src="/pmc/articles/instance/3579804/bin/pone.0057168.g001.gif" src-large="/pmc/articles/instance/3579804/bin/pone.0057168.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3579804/" ref="ordinalpos=8&amp;ncbi_uid=4673563&amp;link_uid=PMC3579804" image-link="/pmc/articles/PMC3579804/figure/pone-0057168-g001/" class="imagepopup">Figure 1. A network of pathways related to LGG, HGG, and others..  From: Distinct Genomic Aberrations between Low-Grade and High-Grade Gliomas of Chinese Patients. </a></p><div class="supp"><p class="details"><div>The abbreviations for KEGG pathways used in this analysis are listed as follows. AAM: Arachidonic acid metabolism; LAM: Linoleic acid metabolism; ALAM: alpha-Linolenic acid metabolism; ELM: Ether lipid metabolism; GPM: Glycerophospholipid metabolism; PD: Prion diseases; GNRH: GnRH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; LTD: Long-term depression; VSMC: Vascular smooth muscle contraction; VEGF: VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; FCER: Fc epsilon RI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; FAM: Fatty acid metabolism; MP: Metabolic pathways; NLI: Neuroactive ligand-receptor interaction; CSP: Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; ME: Melanogenesis; FMM: Fructose and mannose metabolism; LD: Lysine degradation; AEM: Androgen and estrogen metabolism; GM: Glycerolipid metabolism; GD: Glycosaminoglycan degradation; VCI: Vibrio cholerae infection; AD: Alzheimer's disease; PAD: Parkinson's disease; ALS: Amyotrophic lateral sclerosis; HUD: Huntington's disease; PI3K: PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; HIF-1: HIF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; MAPK: MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; CC: Cell cycle; P53: p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; AP: Apoptosis; GL: Glioma; ERBB: ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; WNT: Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; and OP: Oxidative phosphorylation.</div></p></div><div class="aux"><div class="resc">Yunbo Li, et al. PLoS One. 2013;8(2):e57168.</div><p class="links"><a class="dblinks" href="/pubmed/23451178" ref="ordinalpos=8&amp;ncbi_uid=4673563&amp;link_uid=23451178">Citation</a><a class="dblinks" href="/pmc/articles/PMC3579804/" ref="ordinalpos=8&amp;ncbi_uid=4673563&amp;link_uid=PMC3579804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>9.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3733613/figure/F3/" ref="ordinalpos=9&amp;ncbi_uid=5168030&amp;link_uid=PMC3733613" href="/pmc/articles/PMC3733613/"><img src="/pmc/articles/instance/3733613/bin/1479-5876-11-177-3.gif" src-large="/pmc/articles/instance/3733613/bin/1479-5876-11-177-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3733613/" ref="ordinalpos=9&amp;ncbi_uid=5168030&amp;link_uid=PMC3733613" image-link="/pmc/articles/PMC3733613/figure/F3/" class="imagepopup">Figure 3.  From: A network model of genomic hormone interactions underlying dementia and its translational validation through serendipitous off-target effect. </a></p><div class="supp"><p class="details"><div><b>Elements of the seven hormonal <span class="highlight" style="background-color:">signaling</span> pathways form the core connected component of the brain interactome (normal state with all possible interactions).</b> <span class="highlight" style="background-color:">Pathway</span> memberships are indicated by color codings; Green: estrogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Red: insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Light blue: leptin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Dark blue: melatonin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Gray: thyroid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Brown: corticotropin-releasing hormone <span class="highlight" style="background-color:">pathway</span>. Yellow color indicates common membership to two or more pathways and also embeds the elements of the growth hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Erfan Younesi, et al. J Transl Med. 2013;11:177-177.</div><p class="links"><a class="dblinks" href="/pubmed/23885764" ref="ordinalpos=9&amp;ncbi_uid=5168030&amp;link_uid=23885764">Citation</a><a class="dblinks" href="/pmc/articles/PMC3733613/" ref="ordinalpos=9&amp;ncbi_uid=5168030&amp;link_uid=PMC3733613">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>10.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g001/" ref="ordinalpos=10&amp;ncbi_uid=4442592&amp;link_uid=PMC3517549" href="/pmc/articles/PMC3517549/"><img src="/pmc/articles/instance/3517549/bin/pone.0051657.g001.gif" src-large="/pmc/articles/instance/3517549/bin/pone.0051657.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517549/" ref="ordinalpos=10&amp;ncbi_uid=4442592&amp;link_uid=PMC3517549" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g001/" class="imagepopup">Figure 1. The TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes in vertebrate..  From: The Evolution and Origin of Animal Toll-Like Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Revealed by Network-Level Molecular Evolutionary Analyses. </a></p><div class="supp"><p class="details"><div>Arrows indicate the direction of signal transduction. The numbers on the below side represent the position of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes. This picture is adapted from KEGG database.</div></p></div><div class="aux"><div class="resc">Xiaojun Song, et al. PLoS One. 2012;7(12):e51657.</div><p class="links"><a class="dblinks" href="/pubmed/23236523" ref="ordinalpos=10&amp;ncbi_uid=4442592&amp;link_uid=23236523">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517549/" ref="ordinalpos=10&amp;ncbi_uid=4442592&amp;link_uid=PMC3517549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>11.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3923659/figure/pcbi-1003470-g001/" ref="ordinalpos=11&amp;ncbi_uid=5826776&amp;link_uid=PMC3923659" href="/pmc/articles/PMC3923659/"><img src="/pmc/articles/instance/3923659/bin/pcbi.1003470.g001.gif" src-large="/pmc/articles/instance/3923659/bin/pcbi.1003470.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3923659/" ref="ordinalpos=11&amp;ncbi_uid=5826776&amp;link_uid=PMC3923659" image-link="/pmc/articles/PMC3923659/figure/pcbi-1003470-g001/" class="imagepopup">Figure 1. IL-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> diagram..  From: The Structural <span class="highlight" style="background-color:">Pathway</span> of Interleukin 1 (IL-1) Initiated <span class="highlight" style="background-color:">Signaling</span> Reveals Mechanisms of Oncogenic Mutations and SNPs in Inflammation and Cancer. </a></p><div class="supp"><p class="details"><div>In this simplified diagram of the IL-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the signal initiates by the recognition of cytokines by IL-1 receptors and propagates via multiple sub-pathways involving family homologs or alternate pathways to activate transcription factors downstream.</div></p></div><div class="aux"><div class="resc">Saliha Ece Acuner Ozbabacan, et al. PLoS Comput Biol. 2014 February;10(2):e1003470.</div><p class="links"><a class="dblinks" href="/pubmed/24550720" ref="ordinalpos=11&amp;ncbi_uid=5826776&amp;link_uid=24550720">Citation</a><a class="dblinks" href="/pmc/articles/PMC3923659/" ref="ordinalpos=11&amp;ncbi_uid=5826776&amp;link_uid=PMC3923659">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>12.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432059/figure/pone-0043955-g006/" ref="ordinalpos=12&amp;ncbi_uid=4215357&amp;link_uid=PMC3432059" href="/pmc/articles/PMC3432059/"><img src="/pmc/articles/instance/3432059/bin/pone.0043955.g006.gif" src-large="/pmc/articles/instance/3432059/bin/pone.0043955.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432059/" ref="ordinalpos=12&amp;ncbi_uid=4215357&amp;link_uid=PMC3432059" image-link="/pmc/articles/PMC3432059/figure/pone-0043955-g006/" class="imagepopup">Figure 6. the map of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Establishment of Mouse Teratocarcinomas Stem Cells Line and Screening Genes Responsible for Malignancy. </a></p><div class="supp"><p class="details"><div>The map of wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> included at least three different Wnt pathways: the canonical <span class="highlight" style="background-color:">pathway</span>, the planar cell polarity (PCP) <span class="highlight" style="background-color:">pathway</span> and the Wnt/Ca2+ <span class="highlight" style="background-color:">pathway</span>. Most of the interested genes enriched in canonical <span class="highlight" style="background-color:">pathway</span>. The highly-expressed genes identified from dataset are represented with red color, and the down-regulated genes are represented with green color. The key genes of wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> such as wnt and beta-catenin were identified and marked in the map.</div></p></div><div class="aux"><div class="resc">Tao Liu, et al. PLoS One. 2012;7(8):e43955.</div><p class="links"><a class="dblinks" href="/pubmed/22952821" ref="ordinalpos=12&amp;ncbi_uid=4215357&amp;link_uid=22952821">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432059/" ref="ordinalpos=12&amp;ncbi_uid=4215357&amp;link_uid=PMC3432059">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>13.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4309023/figure/bau126-F3/" ref="ordinalpos=13&amp;ncbi_uid=7212787&amp;link_uid=PMC4309023" href="/pmc/articles/PMC4309023/"><img src="/pmc/articles/instance/4309023/bin/bau126f3p.gif" src-large="/pmc/articles/instance/4309023/bin/bau126f3p.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4309023/" ref="ordinalpos=13&amp;ncbi_uid=7212787&amp;link_uid=PMC4309023" image-link="/pmc/articles/PMC4309023/figure/bau126-F3/" class="imagepopup">Figure 3. From: Comparison of human cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> databasesevolution, drawbacks and challenges. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> ontology used by INOH and <span class="highlight" style="background-color:">Pathway</span> Ontology portal. (<b>A</b>) INOH-based ontology uses the ontology or hierarchical formats to annotate the curated signal transductions and metabolic pathways in BioPAX and INOH formats. MoleculeRole, Process, Location and Event are four types of ontological terminologies, which are used by this database. Under the Event category, all the pathways are annotated and each <span class="highlight" style="background-color:">pathway</span> is then further categorized into several sub-categories according to their molecular functions. For example, the Hedgehog <span class="highlight" style="background-color:">pathway</span> is divided into four sub-categories: Negative regulation of Hedgehog <span class="highlight" style="background-color:">pathway</span>, Secretory Hedgehog <span class="highlight" style="background-color:">pathway</span>, general Hedgehog <span class="highlight" style="background-color:">pathway</span> and Regulation of Hedgehog <span class="highlight" style="background-color:">pathway</span>. For each category, there is also a unique ID number present, which classifies a <span class="highlight" style="background-color:">pathway</span> according to its various molecular functions. (<b>B</b>) Bioportal: <span class="highlight" style="background-color:">Pathway</span> Ontology (http://bioportal.bioontology.org/ontologies/PW) forms the ontological tree by using the conceptual terminologies: classic metabolic <span class="highlight" style="background-color:">pathway</span>, disease <span class="highlight" style="background-color:">pathway</span>, drug <span class="highlight" style="background-color:">pathway</span>, regulatory <span class="highlight" style="background-color:">pathway</span> and <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is further divided into various categories, such as calcium mediated, adhesion based, G-protein mediated and Developmental process. Hedgehog and Notch pathways are kept under the <span class="highlight" style="background-color:">signaling</span> pathways pertinent to development category. Furthermore, a unique ID is provided for each <span class="highlight" style="background-color:">pathway</span> with its sub-categories in the ontology tree. Here, the altered pathways for each <span class="highlight" style="background-color:">pathway</span> is also considered and kept under different classification, which in turn helps to distinguish the normal and its altered counterpart. For example, in case of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, it is classified as the possible alteration of Notch <span class="highlight" style="background-color:">pathway</span> by several factors such as by promoters, target genes and macromolecules and each altered <span class="highlight" style="background-color:">pathway</span> are assigned with different ID numbers.</div></p></div><div class="aux"><div class="resc">Saikat Chowdhury, et al. Database (Oxford). 2015;2015:bau126.</div><p class="links"><a class="dblinks" href="/pubmed/25632107" ref="ordinalpos=13&amp;ncbi_uid=7212787&amp;link_uid=25632107">Citation</a><a class="dblinks" href="/pmc/articles/PMC4309023/" ref="ordinalpos=13&amp;ncbi_uid=7212787&amp;link_uid=PMC4309023">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>14.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3724785/figure/pone-0070077-g007/" ref="ordinalpos=14&amp;ncbi_uid=5170687&amp;link_uid=PMC3724785" href="/pmc/articles/PMC3724785/"><img src="/pmc/articles/instance/3724785/bin/pone.0070077.g007.gif" src-large="/pmc/articles/instance/3724785/bin/pone.0070077.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3724785/" ref="ordinalpos=14&amp;ncbi_uid=5170687&amp;link_uid=PMC3724785" image-link="/pmc/articles/PMC3724785/figure/pone-0070077-g007/" class="imagepopup">Figure 7. A diagram of the Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Lysine-Specific Demethylase 1 (LSD1/KDM1A) Contributes to Colorectal Tumorigenesis via Activation of the Wnt/?-Catenin <span class="highlight" style="background-color:">Pathway</span> by Down-Regulating Dickkopf-1 (DKK1). </a></p><div class="supp"><p class="details"><div>We propose that LSD1 down-regulates the Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> antagonist, DKK1, and that this down-regulation causes free -catenin to avoid degradation and accumulate in the cytoplasm. The free -catenin then translocated to the nucleus, activates the transcription of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> target gene c-Myc, and finally leads to the aberrant proliferation of cells and tumorigenesis.</div></p></div><div class="aux"><div class="resc">Zebin Huang, et al. PLoS One. 2013;8(7):e70077.</div><p class="links"><a class="dblinks" href="/pubmed/23922913" ref="ordinalpos=14&amp;ncbi_uid=5170687&amp;link_uid=23922913">Citation</a><a class="dblinks" href="/pmc/articles/PMC3724785/" ref="ordinalpos=14&amp;ncbi_uid=5170687&amp;link_uid=PMC3724785">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>15.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1978528/figure/pone-0000936-g001/" ref="ordinalpos=15&amp;ncbi_uid=1130661&amp;link_uid=PMC1978528" href="/pmc/articles/PMC1978528/"><img src="/pmc/articles/instance/1978528/bin/pone.0000936.g001.gif" src-large="/pmc/articles/instance/1978528/bin/pone.0000936.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1978528/" ref="ordinalpos=15&amp;ncbi_uid=1130661&amp;link_uid=PMC1978528" image-link="/pmc/articles/PMC1978528/figure/pone-0000936-g001/" class="imagepopup">Figure 1. Schematic representation of Smad dependent TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Constraint-Based Modeling and Kinetic Analysis of the Smad Dependent TGF- <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Detailed information about this <span class="highlight" style="background-color:">pathway</span> is described in the text.</div></p></div><div class="aux"><div class="resc">Zhike Zi, et al. PLoS ONE. 2007;2(9):e936.</div><p class="links"><a class="dblinks" href="/pubmed/17895977" ref="ordinalpos=15&amp;ncbi_uid=1130661&amp;link_uid=17895977">Citation</a><a class="dblinks" href="/pmc/articles/PMC1978528/" ref="ordinalpos=15&amp;ncbi_uid=1130661&amp;link_uid=PMC1978528">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>16.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3258345/figure/F9/" ref="ordinalpos=16&amp;ncbi_uid=4939194&amp;link_uid=PMC3258345" href="/pmc/articles/PMC3258345/"><img src="/pmc/articles/instance/3258345/bin/nihms322695f9.gif" src-large="/pmc/articles/instance/3258345/bin/nihms322695f9.jpg" alt="Fig 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3258345/" ref="ordinalpos=16&amp;ncbi_uid=4939194&amp;link_uid=PMC3258345" image-link="/pmc/articles/PMC3258345/figure/F9/" class="imagepopup">Fig 9. Proposed interaction between RAR/RXR and oxidative stress/JNK <span class="highlight" style="background-color:">signaling</span>.  From: High Glucose-Induced Repression of RAR/RXR In Cardiomyocytes is Mediated Through Oxidative Stress/JNK <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The relationship between high glucose, oxidative stress and impaired RAR/RXR <span class="highlight" style="background-color:">signaling</span> are proposed. Retinoic acid, through activation of RAR/RXR-mediated <span class="highlight" style="background-color:">signaling</span>, inhibits intracellular oxidative stress and the JNK <span class="highlight" style="background-color:">pathway</span>, protecting cardiomyocytes from HG-induced apoptosis. High glucose promotes oxidative stress, which leads to activation of the JNK <span class="highlight" style="background-color:">pathway</span> and degradation of RAR/RXR, resulting in repression of RAR/RXR <span class="highlight" style="background-color:">signaling</span>. Impaired RAR/RXR <span class="highlight" style="background-color:">signaling</span> further potentiates intracellular oxidative stress and activation of JNK-mediated apoptotic <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Amar Bahadur Singh, et al. J Cell Physiol. ;227(6):2632-2644.</div><p class="links"><a class="dblinks" href="/pubmed/21882190" ref="ordinalpos=16&amp;ncbi_uid=4939194&amp;link_uid=21882190">Citation</a><a class="dblinks" href="/pmc/articles/PMC3258345/" ref="ordinalpos=16&amp;ncbi_uid=4939194&amp;link_uid=PMC3258345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>17.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g004/" ref="ordinalpos=17&amp;ncbi_uid=4442589&amp;link_uid=PMC3517549" href="/pmc/articles/PMC3517549/"><img src="/pmc/articles/instance/3517549/bin/pone.0051657.g004.gif" src-large="/pmc/articles/instance/3517549/bin/pone.0051657.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517549/" ref="ordinalpos=17&amp;ncbi_uid=4442589&amp;link_uid=PMC3517549" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g004/" class="imagepopup">Figure 4. The number of the core components of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in representative metazoans..  From: The Evolution and Origin of Animal Toll-Like Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Revealed by Network-Level Molecular Evolutionary Analyses. </a></p><div class="supp"><p class="details"><div>The numbers represent the copy number of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes in all studied species. 0/1 means not found using blast research, but it present in lower animals.</div></p></div><div class="aux"><div class="resc">Xiaojun Song, et al. PLoS One. 2012;7(12):e51657.</div><p class="links"><a class="dblinks" href="/pubmed/23236523" ref="ordinalpos=17&amp;ncbi_uid=4442589&amp;link_uid=23236523">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517549/" ref="ordinalpos=17&amp;ncbi_uid=4442589&amp;link_uid=PMC3517549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>18.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4045758/figure/pone-0098653-g004/" ref="ordinalpos=18&amp;ncbi_uid=6232472&amp;link_uid=PMC4045758" href="/pmc/articles/PMC4045758/"><img src="/pmc/articles/instance/4045758/bin/pone.0098653.g004.gif" src-large="/pmc/articles/instance/4045758/bin/pone.0098653.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4045758/" ref="ordinalpos=18&amp;ncbi_uid=6232472&amp;link_uid=PMC4045758" image-link="/pmc/articles/PMC4045758/figure/pone-0098653-g004/" class="imagepopup">Figure 4. Integrative analysis of CNV-ICC-TRN and KEGG <span class="highlight" style="background-color:">signaling</span> pathways..  From: Integrative Analysis of Transcriptional Regulatory Network and Copy Number Variation in Intrahepatic Cholangiocarcinoma. </a></p><div class="supp"><p class="details"><div>(A). Integrated network of CNV-ICC-TRN and KEGG <span class="highlight" style="background-color:">signaling</span> pathways. Triangle represents TF, circle represents gene and rectangle represents <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; red color means gene inside CNV region, green color means gene outside CNV region. (B) Integrative analysis in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (C) Integrative analysis in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (D) Integrative analysis in TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In figure B, C, D, red edges are from CNV-ICC-TRN, and off-white edges are from <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Ling Li, et al. PLoS One. 2014;9(6):e98653.</div><p class="links"><a class="dblinks" href="/pubmed/24897108" ref="ordinalpos=18&amp;ncbi_uid=6232472&amp;link_uid=24897108">Citation</a><a class="dblinks" href="/pmc/articles/PMC4045758/" ref="ordinalpos=18&amp;ncbi_uid=6232472&amp;link_uid=PMC4045758">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>19.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3815721/figure/f1/" ref="ordinalpos=19&amp;ncbi_uid=5495696&amp;link_uid=PMC3815721" href="/pmc/articles/PMC3815721/"><img src="/pmc/articles/instance/3815721/bin/jtd-05-05-706-f1.gif" src-large="/pmc/articles/instance/3815721/bin/jtd-05-05-706-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3815721/" ref="ordinalpos=19&amp;ncbi_uid=5495696&amp;link_uid=PMC3815721" image-link="/pmc/articles/PMC3815721/figure/f1/" class="imagepopup">Figure 1.  From: A novel hypothesis: up-regulation of HO-1 by activation of PPAR? inhibits HMGB1-RAGE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and ameliorates the development of ALI/ARDS. </a></p><div class="supp"><p class="details"><div>Proposed scheme of the effects of up-regulation of HO-1 by PPAR activation on HMGB1-RAGE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ALI/ARDS. Interaction of HMGB1 with RAGE activates the NF-B and MAPK <span class="highlight" style="background-color:">pathway</span>, resulting in the up-regulation of HMGB1, RAGE and other proinflammatory mediators and promoting the development of ALI/ARDS. Activation of PPAR induces HO-1 expression and subsequent suppression of HMGB1-RAGE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Therefore, we hypothesize that up-regulation of HO-1 by activation of PPAR might inhibit HMGB1-RAGE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and ameliorate the development of ALI/ARDS.</div></p></div><div class="aux"><div class="resc">Guizuo Wang, et al. J Thorac Dis. 2013 October;5(5):706-710.</div><p class="links"><a class="dblinks" href="/pubmed/24255785" ref="ordinalpos=19&amp;ncbi_uid=5495696&amp;link_uid=24255785">Citation</a><a class="dblinks" href="/pmc/articles/PMC3815721/" ref="ordinalpos=19&amp;ncbi_uid=5495696&amp;link_uid=PMC3815721">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>20.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3940357/figure/F1/" ref="ordinalpos=20&amp;ncbi_uid=6038191&amp;link_uid=PMC3940357" href="/pmc/articles/PMC3940357/"><img src="/pmc/articles/instance/3940357/bin/nihms397088f1.gif" src-large="/pmc/articles/instance/3940357/bin/nihms397088f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3940357/" ref="ordinalpos=20&amp;ncbi_uid=6038191&amp;link_uid=PMC3940357" image-link="/pmc/articles/PMC3940357/figure/F1/" class="imagepopup">Figure 1. Schematics of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancer cells.  From: Frizzled homolog proteins, microRNAs and Wnt <span class="highlight" style="background-color:">Signaling</span> in Cancer. </a></p><div class="supp"><p class="details"><div>A. Regulation of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal cells. Beta-catenin is usually phosphorylated and degradated by a multiprotein complex composed of APC, GSK3, CK1 and AXIN.<br />B. Wnt/beta-catenin <span class="highlight" style="background-color:">pathway</span>. Wnt ligands bind to the complex compromised of FZD/LRP/ROR/RYK, resulting in the constitutive stabilization of beta-catenin. The beta-catenin associates with TCF to activate target genes.<br />C. Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> (non-canonical <span class="highlight" style="background-color:">pathway</span>). Wnt activates intracellular Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> as well as Ca<sup>2+</sup>-dependent protein kinases such as PKC and CaMK II. TAK1 and NLK can interfere with TCF/beta-catenin <span class="highlight" style="background-color:">signaling</span>.<br />D. Wnt/JNK <span class="highlight" style="background-color:">pathway</span> (non-canonical <span class="highlight" style="background-color:">pathway</span>). Receptor stimulation activates DVL, which in turn activates the Rho family of GTPases. Rho stimulates c-Jun expression through phosphorylation of c-Jun by ROCK.</div></p></div><div class="aux"><div class="resc">Koji Ueno, et al. Int J Cancer. 2013 April 15;132(8):1731-1740.</div><p class="links"><a class="dblinks" href="/pubmed/22833265" ref="ordinalpos=20&amp;ncbi_uid=6038191&amp;link_uid=22833265">Citation</a><a class="dblinks" href="/pmc/articles/PMC3940357/" ref="ordinalpos=20&amp;ncbi_uid=6038191&amp;link_uid=PMC3940357">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>21.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713831/figure/pone-0006410-g007/" ref="ordinalpos=21&amp;ncbi_uid=1944665&amp;link_uid=PMC2713831" href="/pmc/articles/PMC2713831/"><img src="/pmc/articles/instance/2713831/bin/pone.0006410.g007.gif" src-large="/pmc/articles/instance/2713831/bin/pone.0006410.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713831/" ref="ordinalpos=21&amp;ncbi_uid=1944665&amp;link_uid=PMC2713831" image-link="/pmc/articles/PMC2713831/figure/pone-0006410-g007/" class="imagepopup">Figure 7. Four subclusters that are presumably involved in six <span class="highlight" style="background-color:">signaling</span> pathways (indices above protein names are their coordinates in Fig. 7a)..  From: Local Network Topology in Human Protein Interaction Data Predicts Functional Association. </a></p><div class="supp"><p class="details"><div>(<i>a</i>) Cluster for the NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>P</i>&lt;3.210<sup>20</sup>): TRAF1, TRAF3IP2, TANK, TRPC4AP and RIPK2 are potential <span class="highlight" style="background-color:">pathway</span> members. (<i>b</i>) Cluster for death receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>P</i>&lt;2.410<sup>19</sup>): CASP8AP2, DEDD and DEDD2 are potential members in this <span class="highlight" style="background-color:">pathway</span>. (<i>c</i>) Cluster for insulin receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>P</i>&lt;2.410<sup>10</sup>): SOCS2, PKD1, PTPN14 are potential <span class="highlight" style="background-color:">pathway</span> members. (<i>d</i>) Cluster for the immune response [T-cell receptor <span class="highlight" style="background-color:">signaling</span> (<i>P</i>&lt;4.910<sup>22</sup>), B-cell receptor <span class="highlight" style="background-color:">signaling</span> (<i>P</i>&lt;1.110<sup>17</sup>) and natural killer cell <span class="highlight" style="background-color:">signaling</span> (P&lt;1.710<sup>21</sup>)]: SRC, PTK2, PTK2B, PTPN12, CRK, CRKL, SIT1, PXN, NEDD9, EVL, KIT and IRS4 are potential members of the above three pathways.</div></p></div><div class="aux"><div class="resc">Hua Li, et al. PLoS One. 2009;4(7):e6410.</div><p class="links"><a class="dblinks" href="/pubmed/19641626" ref="ordinalpos=21&amp;ncbi_uid=1944665&amp;link_uid=19641626">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713831/" ref="ordinalpos=21&amp;ncbi_uid=1944665&amp;link_uid=PMC2713831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>22.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012938/figure/F2/" ref="ordinalpos=22&amp;ncbi_uid=6117463&amp;link_uid=PMC4012938" href="/pmc/articles/PMC4012938/"><img src="/pmc/articles/instance/4012938/bin/aging-06-215-g002.gif" src-large="/pmc/articles/instance/4012938/bin/aging-06-215-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012938/" ref="ordinalpos=22&amp;ncbi_uid=6117463&amp;link_uid=PMC4012938" image-link="/pmc/articles/PMC4012938/figure/F2/" class="imagepopup">Figure 2. The heat map and hierarchical clustering in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during aging.  From: Dynamic regulation of genetic pathways and targets during aging in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) It showed the heat map and hierarchical clustering in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of L4 to D6. The samples (column) were clustered into two groups, three replicates in the stage of L4 (L4-1, L4-2 and L4-3) were clustered together and three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together. There were 64 involved genes in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from L4 to D6, which were clustered into 6 groups (the group from A to F).(<b>B</b>) It showed the heat map and hierarchical clustering in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of D6 to D15. The samples were clustered into two groups, three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together and three replicates at day 15 (D15-1, D15-2 and D15-3) were clustered together. There were also 64 involved genes in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from D6 to D15, which were clustered into 4 groups (the group from A to D). Red is for up-regulated and green is for down-regulated.</div></p></div><div class="aux"><div class="resc">Kan He, et al. Aging (Albany NY). 2014 March;6(3):215-230.</div><p class="links"><a class="dblinks" href="/pubmed/24739375" ref="ordinalpos=22&amp;ncbi_uid=6117463&amp;link_uid=24739375">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012938/" ref="ordinalpos=22&amp;ncbi_uid=6117463&amp;link_uid=PMC4012938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>23.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012938/figure/F3/" ref="ordinalpos=23&amp;ncbi_uid=6117462&amp;link_uid=PMC4012938" href="/pmc/articles/PMC4012938/"><img src="/pmc/articles/instance/4012938/bin/aging-06-215-g003.gif" src-large="/pmc/articles/instance/4012938/bin/aging-06-215-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012938/" ref="ordinalpos=23&amp;ncbi_uid=6117462&amp;link_uid=PMC4012938" image-link="/pmc/articles/PMC4012938/figure/F3/" class="imagepopup">Figure 3. The heat map and hierarchical clustering in mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during aging.  From: Dynamic regulation of genetic pathways and targets during aging in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) It showed the heat map and hierarchical clustering in mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of L4 to D6. The samples (column) were clustered into two groups, three replicates in the stage of L4 (L4-1, L4-2 and L4-3) were clustered together and three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together. There were 24 involved genes (row) in mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from L4 to D6, which were clustered into 2 groups (the group of A and B). (<b>B</b>) It showed the heat map and hierarchical clustering in mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of D6 to D15. The samples were clustered into two groups, three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together and three replicates at day 15 (D15-1, D15-2 and D15-3) were clustered together. There were also 24 involved genes in mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from D6 to D15, which were clustered into 2 groups (the group of A and B).Red is for up-regulated and green is for down-regulated.</div></p></div><div class="aux"><div class="resc">Kan He, et al. Aging (Albany NY). 2014 March;6(3):215-230.</div><p class="links"><a class="dblinks" href="/pubmed/24739375" ref="ordinalpos=23&amp;ncbi_uid=6117462&amp;link_uid=24739375">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012938/" ref="ordinalpos=23&amp;ncbi_uid=6117462&amp;link_uid=PMC4012938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>24.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4121133/figure/pone-0103551-g004/" ref="ordinalpos=24&amp;ncbi_uid=6465247&amp;link_uid=PMC4121133" href="/pmc/articles/PMC4121133/"><img src="/pmc/articles/instance/4121133/bin/pone.0103551.g004.gif" src-large="/pmc/articles/instance/4121133/bin/pone.0103551.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4121133/" ref="ordinalpos=24&amp;ncbi_uid=6465247&amp;link_uid=PMC4121133" image-link="/pmc/articles/PMC4121133/figure/pone-0103551-g004/" class="imagepopup">Figure 4. Summary of colorectal cancer (CRC) sub-classifications based on their <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> profiles..  From: Activated cMET and IGF1R-Driven PI3K <span class="highlight" style="background-color:">Signaling</span> Predicts Poor Survival in Colorectal Cancers Independent of KRAS Mutational Status. </a></p><div class="supp"><p class="details"><div>Schematic summarizing the sub-classification of CRCs based on their <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> profiles and <i>KRAS</i> mutational status. The possible therapeutic options for each sub-class based on their <span class="highlight" style="background-color:">pathway</span> profiles are indicated.</div></p></div><div class="aux"><div class="resc">Jeeyun Lee, et al. PLoS One. 2014;9(8):e103551.</div><p class="links"><a class="dblinks" href="/pubmed/25090459" ref="ordinalpos=24&amp;ncbi_uid=6465247&amp;link_uid=25090459">Citation</a><a class="dblinks" href="/pmc/articles/PMC4121133/" ref="ordinalpos=24&amp;ncbi_uid=6465247&amp;link_uid=PMC4121133">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>25.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694616/figure/F1/" ref="ordinalpos=25&amp;ncbi_uid=1936765&amp;link_uid=PMC2694616" href="/pmc/articles/PMC2694616/"><img src="/pmc/articles/instance/2694616/bin/nihms114667f1.gif" src-large="/pmc/articles/instance/2694616/bin/nihms114667f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694616/" ref="ordinalpos=25&amp;ncbi_uid=1936765&amp;link_uid=PMC2694616" image-link="/pmc/articles/PMC2694616/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Pathway</span> module gene expression signatures.  From: A Genomic Strategy to Elucidate Modules of Oncogenic <span class="highlight" style="background-color:">Pathway</span> <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>[Right] The schematic depicts an interconnected <span class="highlight" style="background-color:">signaling</span> network organized as collection of modules that define <span class="highlight" style="background-color:">pathway</span> structure. Perturbations of key genes result in changes to the transcriptional profile of the cell. Expression signatures developed from the <span class="highlight" style="background-color:">pathway</span> module analyses represent activity of units of this structure.<br />[Left] To elucidate <span class="highlight" style="background-color:">pathway</span> modules, we decompose a <span class="highlight" style="background-color:">pathway</span> based on common patterns in the expression of the genes in the <span class="highlight" style="background-color:">pathway</span>. First, we identify a set of core <span class="highlight" style="background-color:">pathway</span> genes and a set of samples with microarray gene expression data evidencing a range of variation in activity in the <span class="highlight" style="background-color:">pathway</span>. We next apply the BFRM software to decompose the <span class="highlight" style="background-color:">pathway</span> into a series of signatures. As part of the modeling process, BFRM expands the <span class="highlight" style="background-color:">pathway</span> genes into a set that more comprehensively describes <span class="highlight" style="background-color:">pathway</span> structure, ultimately yielding a collection of estimated signatures. Finally, we link the signatures with known modules of the <span class="highlight" style="background-color:">pathway</span>. We reason that if a signature is significantly related to activity of a particular module of the <span class="highlight" style="background-color:">pathway</span>, then that signature is a representative of that <span class="highlight" style="background-color:">pathway</span> module.</div></p></div><div class="aux"><div class="resc">Jeffrey T. Chang, et al. Mol Cell. ;34(1):104-114.</div><p class="links"><a class="dblinks" href="/pubmed/19362539" ref="ordinalpos=25&amp;ncbi_uid=1936765&amp;link_uid=19362539">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694616/" ref="ordinalpos=25&amp;ncbi_uid=1936765&amp;link_uid=PMC2694616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>26.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4126662/figure/pone-0103845-g007/" ref="ordinalpos=26&amp;ncbi_uid=6485113&amp;link_uid=PMC4126662" href="/pmc/articles/PMC4126662/"><img src="/pmc/articles/instance/4126662/bin/pone.0103845.g007.gif" src-large="/pmc/articles/instance/4126662/bin/pone.0103845.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4126662/" ref="ordinalpos=26&amp;ncbi_uid=6485113&amp;link_uid=PMC4126662" image-link="/pmc/articles/PMC4126662/figure/pone-0103845-g007/" class="imagepopup">Figure 7. Model prediction and experimental verification of the LIMK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> knockdown and the actin polymerization inhibitor Lat-A on the virus infectivity..  From: Quantitative Analysis of the Processes and <span class="highlight" style="background-color:">Signaling</span> Events Involved in Early HIV-1 Infection of T Cells. </a></p><div class="supp"><p class="details"><div><b>A.</b> Black line displays the original solution showed in Figure 1 while the red line represents the model prediction of the COFILINa variable after inhibition of the LIMK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by a 50%. <b>B.</b> Black line displays the original solution showed in Figure 1; red line represents the model prediction of the ACTIN variable after inhibition of the LIMK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by a 50%. <b>C.</b> The black lines (control condition where cofilin is active before infection) show the model's predicted dynamics of the actin capping. Pink lines show the solutions obtained when the initial state of cofilin, just before infection, was inactive. Dark blue lines represent the predicted dynamics of the actin capping after the activation of virus <span class="highlight" style="background-color:">signaling</span> on the cofilin. Light blue lines represent an increase of the intensity of the activation <span class="highlight" style="background-color:">signaling</span> of cofilin by the virus <b>D.</b> Experimental measurements of infectivity of the virus in increased initial concentrations of the actin-severing factor Lat-A (Yoder et al. 2008, [7]). <b>E.</b> Black line displays the original solution showed in Figure 1 while the red line represents the model prediction of the COFILINa variable after inhibition of the WAVE2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by a 50%. <b>F.</b> Black line displays the original solution showed in Figure 1; red line represents the model prediction of the ACTIN variable after inhibition of the WAVE2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by a 50%.</div></p></div><div class="aux"><div class="resc">Guido Santos, et al. PLoS One. 2014;9(8):e103845.</div><p class="links"><a class="dblinks" href="/pubmed/25105875" ref="ordinalpos=26&amp;ncbi_uid=6485113&amp;link_uid=25105875">Citation</a><a class="dblinks" href="/pmc/articles/PMC4126662/" ref="ordinalpos=26&amp;ncbi_uid=6485113&amp;link_uid=PMC4126662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>27.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1705971/figure/F7/" ref="ordinalpos=27&amp;ncbi_uid=848098&amp;link_uid=PMC1705971" href="/pmc/articles/PMC1705971/"><img src="/pmc/articles/instance/1705971/bin/nihms-14550-0007.gif" src-large="/pmc/articles/instance/1705971/bin/nihms-14550-0007.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1705971/" ref="ordinalpos=27&amp;ncbi_uid=848098&amp;link_uid=PMC1705971" image-link="/pmc/articles/PMC1705971/figure/F7/" class="imagepopup">Fig. 7.  From: The Ras/cAMP-dependent Protein Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulates an Early Step of the Autophagy Process in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div><b>Potential models to describe how Ras/PKA <span class="highlight" style="background-color:">signaling</span> activity influences autophagy and how this control might be coordinated with that of the Tor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, Ras/PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates an early event in autophagy, an event that precedes the formation of the autophagosome. The schematic shows a number of the steps involved in the autophagy process and the relative position of the event most likely to be controlled by Ras/PKA <span class="highlight" style="background-color:">signaling</span> activity. <i>B</i>, schematic depicting the relative order of action of the Ras/PKA and Tor <span class="highlight" style="background-color:">signaling</span> pathways on the autophagy process. The data presented here, and in other recent studies, are consistent with the Ras/PKA <span class="highlight" style="background-color:">pathway</span> acting either (i) independently or (ii) downstream of the Tor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. See the text for further details.</div></p></div><div class="aux"><div class="resc">Yelena V. Budovskaya, et al. J Biol Chem. ;279(20):20663-20671.</div><p class="links"><a class="dblinks" href="/pubmed/15016820" ref="ordinalpos=27&amp;ncbi_uid=848098&amp;link_uid=15016820">Citation</a><a class="dblinks" href="/pmc/articles/PMC1705971/" ref="ordinalpos=27&amp;ncbi_uid=848098&amp;link_uid=PMC1705971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>28.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012938/figure/F5/" ref="ordinalpos=28&amp;ncbi_uid=6117460&amp;link_uid=PMC4012938" href="/pmc/articles/PMC4012938/"><img src="/pmc/articles/instance/4012938/bin/aging-06-215-g005.gif" src-large="/pmc/articles/instance/4012938/bin/aging-06-215-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012938/" ref="ordinalpos=28&amp;ncbi_uid=6117460&amp;link_uid=PMC4012938" image-link="/pmc/articles/PMC4012938/figure/F5/" class="imagepopup">Figure 5. The heat map and hierarchical clustering in TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during aging.  From: Dynamic regulation of genetic pathways and targets during aging in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) It showed the heat map and hierarchical clustering in TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of L4 to D6. The samples (column) were clustered into two groups, three replicates in the stage of L4 (L4-1, L4-2 and L4-3) were clustered together and three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together. There were 30 involved genes (row) in TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from L4 to D6, which were mainly clustered into 3 groups (the group from A to C). (<b>B</b>) It showed the heat map and hierarchical clustering in TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of D6 to D15. The samples were clustered into two groups, three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together and three replicates at day 15 (D15-1, D15-2 and D15-3) were clustered together. There were also 30 involved genes in TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from D6 to D15, which were mainly clustered into 2 groups (the group of A and B).Red is for up-regulated and green is for down-regulated.</div></p></div><div class="aux"><div class="resc">Kan He, et al. Aging (Albany NY). 2014 March;6(3):215-230.</div><p class="links"><a class="dblinks" href="/pubmed/24739375" ref="ordinalpos=28&amp;ncbi_uid=6117460&amp;link_uid=24739375">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012938/" ref="ordinalpos=28&amp;ncbi_uid=6117460&amp;link_uid=PMC4012938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>29.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3629553/figure/F7/" ref="ordinalpos=29&amp;ncbi_uid=6011240&amp;link_uid=PMC3629553" href="/pmc/articles/PMC3629553/"><img src="/pmc/articles/instance/3629553/bin/nihms-442476-f0007.gif" src-large="/pmc/articles/instance/3629553/bin/nihms-442476-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3629553/" ref="ordinalpos=29&amp;ncbi_uid=6011240&amp;link_uid=PMC3629553" image-link="/pmc/articles/PMC3629553/figure/F7/" class="imagepopup">Figure 7. GW182 regulates PDFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through modulation of DNC.  From: GW182 controls Drosophila circadian behavior and PDF-Receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Gs (Choi et al., 2012, this study), PKA-C1 (this study) and adenylate cyclase AC3 (at least in PDF-positive sLNvs) (Duvall and Taghert, 2012) are positive components of PDFR <span class="highlight" style="background-color:">signaling</span>. Our work reveals that GW182 also promotes PDFR <span class="highlight" style="background-color:">signaling</span> by repressing DNC, which degrades cAMP and thus represses PDFR <span class="highlight" style="background-color:">signaling</span>. GW182-mediated DNC repression is controlled by light. Grey shapes are as yet unidentified components of the PDFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yong Zhang, et al. Neuron. 2013 April 10;78(1):152-165.</div><p class="links"><a class="dblinks" href="/pubmed/23583112" ref="ordinalpos=29&amp;ncbi_uid=6011240&amp;link_uid=23583112">Citation</a><a class="dblinks" href="/pmc/articles/PMC3629553/" ref="ordinalpos=29&amp;ncbi_uid=6011240&amp;link_uid=PMC3629553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>30.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3585227/figure/pone-0056810-g007/" ref="ordinalpos=30&amp;ncbi_uid=4685993&amp;link_uid=PMC3585227" href="/pmc/articles/PMC3585227/"><img src="/pmc/articles/instance/3585227/bin/pone.0056810.g007.gif" src-large="/pmc/articles/instance/3585227/bin/pone.0056810.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3585227/" ref="ordinalpos=30&amp;ncbi_uid=4685993&amp;link_uid=PMC3585227" image-link="/pmc/articles/PMC3585227/figure/pone-0056810-g007/" class="imagepopup">Figure 7. Comparing Subnetworks of WNT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Participants. A..  From: Cell-Type-Specific Predictive Network Yields Novel Insights into Mouse Embryonic Stem Cell Self-Renewal and Cell Fate. </a></p><div class="supp"><p class="details"><div>WNT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Subnetwork. A model of the WNT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> adapted from the curated KEGG <span class="highlight" style="background-color:">pathway</span> for <i>M. musculus</i> (Mmu) includes SRCs for Wnt, Frizzled, and Dishevelled <span class="highlight" style="background-color:">pathway</span> participants, illustrating that not all family members are equally supported by evidential data. Curated pathways, which are cell-type agnostic, cannot capture these differences in connectivity. A corresponding network of mESC posterior edges involved in this view of WNT <span class="highlight" style="background-color:">Signaling</span> (created in Cytoscape) demonstrates the variance in edge weights and SRCs in the <span class="highlight" style="background-color:">signaling</span> cascade. <b>B.</b> The same WNT <span class="highlight" style="background-color:">Signaling</span> subnetwork produced using Mmu superset and negative control posterior edge weights and SRCs captures a different picture of connectivity as compared to the mESC network. Far more WNT <span class="highlight" style="background-color:">signaling</span> activity between different WNT family member ligands and Frizzled receptors is evident in the test subnetworks. This may reflect WNT <span class="highlight" style="background-color:">signaling</span> activity observed in data from both mESCs and other cellular contexts in the Mmu superset of features. The influence of WNT <span class="highlight" style="background-color:">signaling</span> in other cellular contexts is even stronger in the negative control subnetwork.</div></p></div><div class="aux"><div class="resc">Karen G. Dowell, et al. PLoS One. 2013;8(2):e56810.</div><p class="links"><a class="dblinks" href="/pubmed/23468881" ref="ordinalpos=30&amp;ncbi_uid=4685993&amp;link_uid=23468881">Citation</a><a class="dblinks" href="/pmc/articles/PMC3585227/" ref="ordinalpos=30&amp;ncbi_uid=4685993&amp;link_uid=PMC3585227">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>31.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2888385/figure/F6/" ref="ordinalpos=31&amp;ncbi_uid=3036933&amp;link_uid=PMC2888385" href="/pmc/articles/PMC2888385/"><img src="/pmc/articles/instance/2888385/bin/zbc0291022120006.gif" src-large="/pmc/articles/instance/2888385/bin/zbc0291022120006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2888385/" ref="ordinalpos=31&amp;ncbi_uid=3036933&amp;link_uid=PMC2888385" image-link="/pmc/articles/PMC2888385/figure/F6/" class="imagepopup">FIGURE 6. From: Nodal <span class="highlight" style="background-color:">Signaling</span> Regulates the Bone Morphogenic Protein Pluripotency <span class="highlight" style="background-color:">Pathway</span> in Mouse Embryonic Stem Cells. </a></p><div class="supp"><p class="details"><div><b>Model of Nodal-Smad2 <span class="highlight" style="background-color:">signaling</span> regulation of the BMP pluripotency <span class="highlight" style="background-color:">pathway</span> in mouse ES cells.</b> Nodal-Smad2 <span class="highlight" style="background-color:">signaling</span> regulates a significant number of genes in undifferentiated ES cells. Positively regulated targets include genes characteristic of the mesendoderm developmental lineage. Genes negatively regulated by Nodal <span class="highlight" style="background-color:">signaling</span> include some trophoblast markers; the mechanism of this regulation remains to be determined. The regulation of <i>Id</i> gene expression by Nodal-Smad2 is indirect, via the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The transcriptional regulation of <i>Smad7</i> by Nodal <span class="highlight" style="background-color:">signaling</span> is the major contributor to the modulation of the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. BMP <span class="highlight" style="background-color:">pathway</span> activation and <i>Id</i> gene expression are essential components to maintain ES cell pluripotency.</div></p></div><div class="aux"><div class="resc">Katherine E. Galvin, et al. J Biol Chem. 2010 June 25;285(26):19747-19756.</div><p class="links"><a class="dblinks" href="/pubmed/20427282" ref="ordinalpos=31&amp;ncbi_uid=3036933&amp;link_uid=20427282">Citation</a><a class="dblinks" href="/pmc/articles/PMC2888385/" ref="ordinalpos=31&amp;ncbi_uid=3036933&amp;link_uid=PMC2888385">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>32.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2249701/figure/pone-0001710-g007/" ref="ordinalpos=32&amp;ncbi_uid=1312992&amp;link_uid=PMC2249701" href="/pmc/articles/PMC2249701/"><img src="/pmc/articles/instance/2249701/bin/pone.0001710.g007.gif" src-large="/pmc/articles/instance/2249701/bin/pone.0001710.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2249701/" ref="ordinalpos=32&amp;ncbi_uid=1312992&amp;link_uid=PMC2249701" image-link="/pmc/articles/PMC2249701/figure/pone-0001710-g007/" class="imagepopup">Figure 7. Schematic model for a mechanism of Sirt1 activating ERK/S6K1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in cell senescence..  From: SIRT1 Overexpression Antagonizes Cellular Senescence with Activated ERK/S6k1 <span class="highlight" style="background-color:">Signaling</span> in Human Diploid Fibroblasts. </a></p><div class="supp"><p class="details"><div>The activation of SIRT1 in human lung fibroblasts suppressed p16<sup>INK4A</sup>/Rb <span class="highlight" style="background-color:">pathway</span> and provoked ERK/S6K1 <span class="highlight" style="background-color:">signaling</span>, which, in turn, may inhibit p16<sup>INK4A</sup>/Rb <span class="highlight" style="background-color:">signaling</span> via Id1 induction.</div></p></div><div class="aux"><div class="resc">Jing Huang, et al. PLoS ONE. 2008;3(3):e1710.</div><p class="links"><a class="dblinks" href="/pubmed/18320031" ref="ordinalpos=32&amp;ncbi_uid=1312992&amp;link_uid=18320031">Citation</a><a class="dblinks" href="/pmc/articles/PMC2249701/" ref="ordinalpos=32&amp;ncbi_uid=1312992&amp;link_uid=PMC2249701">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>33.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4029079/figure/F4/" ref="ordinalpos=33&amp;ncbi_uid=6162571&amp;link_uid=PMC4029079" href="/pmc/articles/PMC4029079/"><img src="/pmc/articles/instance/4029079/bin/1471-2164-15-182-4.gif" src-large="/pmc/articles/instance/4029079/bin/1471-2164-15-182-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4029079/" ref="ordinalpos=33&amp;ncbi_uid=6162571&amp;link_uid=PMC4029079" image-link="/pmc/articles/PMC4029079/figure/F4/" class="imagepopup">Figure 4.  From: Transcriptomic analysis of differentially expressed genes in the Ras1CA-overexpressed and wildtype posterior silk glands. </a></p><div class="supp"><p class="details"><div><b>Verification of transcriptomic results of DEGs in pathways in cancer, insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by qPCR. (A)</b> Common DEGs in all the three pathways. <b>(B)</b> Common DEGs in pathways in cancer and insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(C)</b> Common DEGs in pathways in cancer and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(D)</b> Common DEG in insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The transcriptional results of DEGs (FPKM) are marked with black. The q-PCR data of DEGs upregulated and downregulated by Ras1 are marked with red and green. The mis-matched DEGs are marked with gray. WT: wildtype; Ras1: Fil-GAL4&gt;UAS-Ras1<sup>CA</sup>.</div></p></div><div class="aux"><div class="resc">Li Ma, et al. BMC Genomics. 2014;15:182-182.</div><p class="links"><a class="dblinks" href="/pubmed/24606580" ref="ordinalpos=33&amp;ncbi_uid=6162571&amp;link_uid=24606580">Citation</a><a class="dblinks" href="/pmc/articles/PMC4029079/" ref="ordinalpos=33&amp;ncbi_uid=6162571&amp;link_uid=PMC4029079">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>34.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012938/figure/F4/" ref="ordinalpos=34&amp;ncbi_uid=6117461&amp;link_uid=PMC4012938" href="/pmc/articles/PMC4012938/"><img src="/pmc/articles/instance/4012938/bin/aging-06-215-g004.gif" src-large="/pmc/articles/instance/4012938/bin/aging-06-215-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012938/" ref="ordinalpos=34&amp;ncbi_uid=6117461&amp;link_uid=PMC4012938" image-link="/pmc/articles/PMC4012938/figure/F4/" class="imagepopup">Figure 4. The heat map and hierarchical clustering in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during aging.  From: Dynamic regulation of genetic pathways and targets during aging in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) It showed the heat map and hierarchical clustering in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of L4 to D6. The samples (column) were clustered into two groups, three replicates in the stage of L4 (L4-1, L4-2 and L4-3) were clustered together and three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together. There were 62 involved genes (row) in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from L4 to D6, which were clustered into 4 groups (the group from A to D). (<b>B</b>) It showed the heat map and hierarchical clustering in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of D6 to D15. The samples were clustered into two groups, three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together and three replicates at day 15 (D15-1, D15-2 and D15-3) were clustered together. There were also 62 involved genes in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from D6 to D15, which were clustered into 3 groups (the group from A to C). Red is for up-regulated and green is for down-regulated.</div></p></div><div class="aux"><div class="resc">Kan He, et al. Aging (Albany NY). 2014 March;6(3):215-230.</div><p class="links"><a class="dblinks" href="/pubmed/24739375" ref="ordinalpos=34&amp;ncbi_uid=6117461&amp;link_uid=24739375">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012938/" ref="ordinalpos=34&amp;ncbi_uid=6117461&amp;link_uid=PMC4012938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>35.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3009741/figure/pone-0015613-g003/" ref="ordinalpos=35&amp;ncbi_uid=2695432&amp;link_uid=PMC3009741" href="/pmc/articles/PMC3009741/"><img src="/pmc/articles/instance/3009741/bin/pone.0015613.g003.gif" src-large="/pmc/articles/instance/3009741/bin/pone.0015613.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3009741/" ref="ordinalpos=35&amp;ncbi_uid=2695432&amp;link_uid=PMC3009741" image-link="/pmc/articles/PMC3009741/figure/pone-0015613-g003/" class="imagepopup">Figure 3. The overall framework of the innate immune response triggered by all three stimuli..  From: Bioinformatics Analysis of Macrophages Exposed to Porphyromonas gingivalis: Implications in Acute vs. Chronic Infections. </a></p><div class="supp"><p class="details"><div>Pathways in CG were used in constructing the general regulatory <span class="highlight" style="background-color:">pathway</span>. Not all the genes shown in this <span class="highlight" style="background-color:">pathway</span> were up- or down-regulated across three conditions. 7 pathways were involved: Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, NOD-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Jak-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, apoptosis, chemokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and cytokine-cytokine receptor interaction.</div></p></div><div class="aux"><div class="resc">Wen-Han Yu, et al. PLoS One. 2010;5(12):e15613.</div><p class="links"><a class="dblinks" href="/pubmed/21203416" ref="ordinalpos=35&amp;ncbi_uid=2695432&amp;link_uid=21203416">Citation</a><a class="dblinks" href="/pmc/articles/PMC3009741/" ref="ordinalpos=35&amp;ncbi_uid=2695432&amp;link_uid=PMC3009741">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>36.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012938/figure/F6/" ref="ordinalpos=36&amp;ncbi_uid=6117459&amp;link_uid=PMC4012938" href="/pmc/articles/PMC4012938/"><img src="/pmc/articles/instance/4012938/bin/aging-06-215-g006.gif" src-large="/pmc/articles/instance/4012938/bin/aging-06-215-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012938/" ref="ordinalpos=36&amp;ncbi_uid=6117459&amp;link_uid=PMC4012938" image-link="/pmc/articles/PMC4012938/figure/F6/" class="imagepopup">Figure 6. The heat map and hierarchical clustering in ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during aging.  From: Dynamic regulation of genetic pathways and targets during aging in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) It showed the heat map and hierarchical clustering in ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of L4 to D6. The samples (column) were clustered into two groups, three replicates in the stage of L4 (L4-1, L4-2 and L4-3) were clustered together and three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together. There were 29 involved genes (row) in ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from L4 to D6, which were clustered into 4 groups (the group from A to D). (<b>B</b>) It showed the heat map and hierarchical clustering in ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the stage of D6 to D15. The samples were clustered into two groups, three replicates at day 6 (D6-1, D6-2 and D6-3) were clustered together and three replicates at day 15 (D15-1, D15-2 and D15-3) were clustered together. There were also 29 involved genes in ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from D6 to D15, in which there was mainly one group (the group of A).Red is for up-regulated and green is for down-regulated.</div></p></div><div class="aux"><div class="resc">Kan He, et al. Aging (Albany NY). 2014 March;6(3):215-230.</div><p class="links"><a class="dblinks" href="/pubmed/24739375" ref="ordinalpos=36&amp;ncbi_uid=6117459&amp;link_uid=24739375">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012938/" ref="ordinalpos=36&amp;ncbi_uid=6117459&amp;link_uid=PMC4012938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>37.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2888472/figure/F2/" ref="ordinalpos=37&amp;ncbi_uid=2333470&amp;link_uid=PMC2888472" href="/pmc/articles/PMC2888472/"><img src="/pmc/articles/instance/2888472/bin/nihms207791f2.gif" src-large="/pmc/articles/instance/2888472/bin/nihms207791f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2888472/" ref="ordinalpos=37&amp;ncbi_uid=2333470&amp;link_uid=PMC2888472" image-link="/pmc/articles/PMC2888472/figure/F2/" class="imagepopup">Figure 2.  From: Key Players in the Gene Networks Guiding ESCs toward Mesoderm. </a></p><div class="supp"><p class="details"><div>Simplified scheme of <span class="highlight" style="background-color:">signaling</span> pathways required for maintenance of pluripotency of mESC (top panel) and hESC (bottom panel). While both mouse and human ESC require an optimal expression level of Oct4 and Nanog for maintaining pluripotency, the <span class="highlight" style="background-color:">signaling</span> pathways involved in obtaining these optimal expression levels vary significantly in mouse and man. In mESC concomitant LIF and BMP4 <span class="highlight" style="background-color:">signaling</span> are required: LIF causes activation of the JAK/Stat3, MEK/ERK1/2 and PI3K/Akt pathways. JAK/Stat3 <span class="highlight" style="background-color:">signaling</span> is necessary for suppression of differentiation into mesoderm and endoderm. Activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>  apart from being a survival <span class="highlight" style="background-color:">pathway</span>, leads to inhibition of GSK3, resulting in catenin translocation to the nucleus and protection of c-myc from degradation, a further mechanism for maintenance of pluripotency. Alternatively this <span class="highlight" style="background-color:">pathway</span> is also activated by canonical Wnt <span class="highlight" style="background-color:">signaling</span>. LIF also activates the MEK/ERK1/2 <span class="highlight" style="background-color:">pathway</span>, which is a differentiation <span class="highlight" style="background-color:">pathway</span> in mESC, and inhibition of various MAPK pathways including the MEK/ERK1/2 <span class="highlight" style="background-color:">pathway</span> is one of the mechanisms by which BMP4 contributes to the maintenance of pluripotency. Concomitantly BMP4 activates the Smad1/5/8 <span class="highlight" style="background-color:">pathway</span>, which inhibits mESC differentiation to neuroectoderm and in combination with Stat3 <span class="highlight" style="background-color:">signaling</span> maintains pluripotency. The Activin <span class="highlight" style="background-color:">pathway</span> described below as important for the pluripotency of hESC, is also necessary for that purpose in mESC, but is provided in an autocrine fashion by the ESC themselves. In contrast, in hESC, FGF2 in combination with Activin, a member of the TGF family maintains pluripotency. FGF2 uses at least 3 different <span class="highlight" style="background-color:">signaling</span> cascades for this purpose: 1) It activates the MEK/ERK1/2 <span class="highlight" style="background-color:">pathway</span>, - which is necessary for maintenance of pluripotency in hESC, in contrast to mESC, where this <span class="highlight" style="background-color:">pathway</span> causes differentiation. 2) It inhibits the Smad1/5/8 <span class="highlight" style="background-color:">pathway</span>. Smad1/5/8 activation in hESC causes differentiation, - again opposite effects are seen in mouse and human cells, and 3) FGF2 is known to activate the PI3K/Akt cascade, a <span class="highlight" style="background-color:">pathway</span> contributing to the maintenance of pluripotency in both hESC and mESC. Finally Activin and possibly other TGF family members, activate the Smad2/3 <span class="highlight" style="background-color:">pathway</span>, which causes upregulation of Nanog and Oct4 transcription. Nanog prevents neuroectoderm formation induced by FGF2 alone.</div></p></div><div class="aux"><div class="resc">Nadezda Omelyanchuk, et al. J Stem Cells. ;4(3):147-160.</div><p class="links"><a class="dblinks" href="/pubmed/20232600" ref="ordinalpos=37&amp;ncbi_uid=2333470&amp;link_uid=20232600">Citation</a><a class="dblinks" href="/pmc/articles/PMC2888472/" ref="ordinalpos=37&amp;ncbi_uid=2333470&amp;link_uid=PMC2888472">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>38.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912139/figure/pone-0087875-g006/" ref="ordinalpos=38&amp;ncbi_uid=5785572&amp;link_uid=PMC3912139" href="/pmc/articles/PMC3912139/"><img src="/pmc/articles/instance/3912139/bin/pone.0087875.g006.gif" src-large="/pmc/articles/instance/3912139/bin/pone.0087875.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912139/" ref="ordinalpos=38&amp;ncbi_uid=5785572&amp;link_uid=PMC3912139" image-link="/pmc/articles/PMC3912139/figure/pone-0087875-g006/" class="imagepopup">Figure 6. Diverse effects of <span class="highlight" style="background-color:">signaling</span> pathways on MUC2 and MUC5AC expression in PK-8 and PCI-35 cells..  From: A GNAS Mutation Found in Pancreatic Intraductal Papillary Mucinous Neoplasms Induces Drastic Alterations of Gene Expression Profiles with Upregulation of Mucin Genes. </a></p><div class="supp"><p class="details"><div>(A) The regulation of <i>MUC2</i> expression in PK-8 cells was predominantly mediated by the GPCR <span class="highlight" style="background-color:">pathway</span>, with synergistic effects of the MAPK <span class="highlight" style="background-color:">pathway</span> and additive effects of the PI3K <span class="highlight" style="background-color:">pathway</span>. Minimal interactions existed among the 3 <span class="highlight" style="background-color:">signaling</span> pathways themselves. (B) <i>MUC2</i> expression in PCI-35 cells was regulated predominantly by the MAPK <span class="highlight" style="background-color:">pathway</span> and additively by the PI3K <span class="highlight" style="background-color:">pathway</span>, whereas the GPCR <span class="highlight" style="background-color:">pathway</span> was antagonistic. Active interactions existed among the <span class="highlight" style="background-color:">signaling</span> pathways: cAMP was upregulated by active MAPK and downregulated by active PI3K. (C) <i>MUC5AC</i> expression in PK-8 cells was regulated predominantly by the GPCR <span class="highlight" style="background-color:">pathway</span>, the MAPK <span class="highlight" style="background-color:">pathway</span> was antagonistic, and the PI3K <span class="highlight" style="background-color:">pathway</span> played a weak role. (D) <i>MUC5AC</i> expression in PCI-35 cells was regulated predominantly by the MAPK <span class="highlight" style="background-color:">pathway</span>, whereas both the PI3K and GPCR pathways were antagonistic.</div></p></div><div class="aux"><div class="resc">Hirotake Komatsu, et al. PLoS One. 2014;9(2):e87875.</div><p class="links"><a class="dblinks" href="/pubmed/24498386" ref="ordinalpos=38&amp;ncbi_uid=5785572&amp;link_uid=24498386">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912139/" ref="ordinalpos=38&amp;ncbi_uid=5785572&amp;link_uid=PMC3912139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>39.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3353643/figure/F6/" ref="ordinalpos=39&amp;ncbi_uid=4659753&amp;link_uid=PMC3353643" href="/pmc/articles/PMC3353643/"><img src="/pmc/articles/instance/3353643/bin/zh20031264670006.gif" src-large="/pmc/articles/instance/3353643/bin/zh20031264670006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3353643/" ref="ordinalpos=39&amp;ncbi_uid=4659753&amp;link_uid=PMC3353643" image-link="/pmc/articles/PMC3353643/figure/F6/" class="imagepopup">Fig. 6. From: WNK4 inhibits NCC protein expression through MAPK ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed model of WNK4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediating NCC regulation in DCT. WNK1 and WNK4 were shown to phosphorylate SPAK/OSR1 and subsequently phosphorylate NCC to activate NCC, whereas our present data also showed that WNK4 also phosphorylates ERK1/2 and subsequently decreases NCC protein expression, suggesting that WNK4-ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is inhibitory in NCC regulation. Aldo, aldosterone; SGK1, serum glucocorticoid-regulated kinase 1; WNK4 PHA II mutant indicates either E562K or R1185C; ERK1/2 is one component of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Gray black lines indicate the known stimulatory <span class="highlight" style="background-color:">signaling</span> pathways. Thick black lines indicate the novel inhibitory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. + Sign indicates a stimulatory effect and  sign indicates an inhibitory effect.</div></p></div><div class="aux"><div class="resc">Bo Zhou, et al. Am J Physiol Renal Physiol. 2012 March 1;302(5):F533-F539.</div><p class="links"><a class="dblinks" href="/pubmed/22114204" ref="ordinalpos=39&amp;ncbi_uid=4659753&amp;link_uid=22114204">Citation</a><a class="dblinks" href="/pmc/articles/PMC3353643/" ref="ordinalpos=39&amp;ncbi_uid=4659753&amp;link_uid=PMC3353643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>40.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2895628/figure/F5/" ref="ordinalpos=40&amp;ncbi_uid=2383460&amp;link_uid=PMC2895628" href="/pmc/articles/PMC2895628/"><img src="/pmc/articles/instance/2895628/bin/1471-2164-11-320-5.gif" src-large="/pmc/articles/instance/2895628/bin/1471-2164-11-320-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2895628/" ref="ordinalpos=40&amp;ncbi_uid=2383460&amp;link_uid=PMC2895628" image-link="/pmc/articles/PMC2895628/figure/F5/" class="imagepopup">Figure 5.  From: A deep investigation into the adipogenesis mechanism: Profile of microRNAs regulating adipogenesis by modulating the canonical Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>The mode of miRNAs that regulate adipogenesis by modulating the canonical Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. (A) Predicted miRNAs targets in WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> such as Sfrp, Dkk, Frizzled, Gsk3, Apc and Dsh, resulting in the repression of adipogenesis; (B) Predicted miRNAs targets of WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> such as Wnt, Lrp, Axin and Tcf, resulting in activation of adipogenesis. This figure was modified from a previous report [39].</div></p></div><div class="aux"><div class="resc">Limei Qin, et al. BMC Genomics. 2010;11:320-320.</div><p class="links"><a class="dblinks" href="/pubmed/20492721" ref="ordinalpos=40&amp;ncbi_uid=2383460&amp;link_uid=20492721">Citation</a><a class="dblinks" href="/pmc/articles/PMC2895628/" ref="ordinalpos=40&amp;ncbi_uid=2383460&amp;link_uid=PMC2895628">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>41.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4309023/figure/bau126-F6/" ref="ordinalpos=41&amp;ncbi_uid=7212784&amp;link_uid=PMC4309023" href="/pmc/articles/PMC4309023/"><img src="/pmc/articles/instance/4309023/bin/bau126f6p.gif" src-large="/pmc/articles/instance/4309023/bin/bau126f6p.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4309023/" ref="ordinalpos=41&amp;ncbi_uid=7212784&amp;link_uid=PMC4309023" image-link="/pmc/articles/PMC4309023/figure/bau126-F6/" class="imagepopup">Figure 6. From: Comparison of human cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> databasesevolution, drawbacks and challenges. </a></p><div class="supp"><p class="details"><div>Core Hedgehog <span class="highlight" style="background-color:">pathway</span> and its cross talks. Schematic shows the cross connection of the core Hedgehog <span class="highlight" style="background-color:">pathway</span> (taken from KEGG) with the other pathways and molecules [e.g. Parathyroid hormone (PTH) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, proteasome <span class="highlight" style="background-color:">pathway</span>, Vitamin D3, Megalin and TGF2 molecules], which are considered in different databases, such as NCI-PID, CPDB, GENE GO and MILLIPORE. Hence, the number of <span class="highlight" style="background-color:">pathway</span> components of this <span class="highlight" style="background-color:">pathway</span> varies significantly within these databases.</div></p></div><div class="aux"><div class="resc">Saikat Chowdhury, et al. Database (Oxford). 2015;2015:bau126.</div><p class="links"><a class="dblinks" href="/pubmed/25632107" ref="ordinalpos=41&amp;ncbi_uid=7212784&amp;link_uid=25632107">Citation</a><a class="dblinks" href="/pmc/articles/PMC4309023/" ref="ordinalpos=41&amp;ncbi_uid=7212784&amp;link_uid=PMC4309023">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>42.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3372214/figure/F1/" ref="ordinalpos=42&amp;ncbi_uid=4416011&amp;link_uid=PMC3372214" href="/pmc/articles/PMC3372214/"><img src="/pmc/articles/instance/3372214/bin/zjv9990960690001.gif" src-large="/pmc/articles/instance/3372214/bin/zjv9990960690001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3372214/" ref="ordinalpos=42&amp;ncbi_uid=4416011&amp;link_uid=PMC3372214" image-link="/pmc/articles/PMC3372214/figure/F1/" class="imagepopup">Fig 1.  From: Hepatitis C Virus Activates the mTOR/S6K1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Inhibiting IRS-1 Function for Insulin Resistance. </a></p><div class="supp"><p class="details"><div>Simplified schematic presentation of HCV genotype 2a-mediated blocking of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. We suggested previously that HCV infection upregulates Ser-312 phosphorylation of IRS-1 and impairs the Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by modulation of Ser/Thr phosphorylation in the upstream insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (11). Analysis of the downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reveals that HCV further downregulates tumor suppressor TSC1/TSC2, resulting in an upregulation of Rheb. This results in downstream activation of mTOR and S6K1, which in turn acts as a negative regulator of IRS-1, causing insulin resistance. Directions of solid arrows on the side indicate up- or downregulation of specific <span class="highlight" style="background-color:">signaling</span> molecules.</div></p></div><div class="aux"><div class="resc">Sandip K. Bose, et al. J Virol. 2012 June;86(11):6315-6322.</div><p class="links"><a class="dblinks" href="/pubmed/22457523" ref="ordinalpos=42&amp;ncbi_uid=4416011&amp;link_uid=22457523">Citation</a><a class="dblinks" href="/pmc/articles/PMC3372214/" ref="ordinalpos=42&amp;ncbi_uid=4416011&amp;link_uid=PMC3372214">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>43.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407218/figure/pone-0038375-g005/" ref="ordinalpos=43&amp;ncbi_uid=4147513&amp;link_uid=PMC3407218" href="/pmc/articles/PMC3407218/"><img src="/pmc/articles/instance/3407218/bin/pone.0038375.g005.gif" src-large="/pmc/articles/instance/3407218/bin/pone.0038375.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407218/" ref="ordinalpos=43&amp;ncbi_uid=4147513&amp;link_uid=PMC3407218" image-link="/pmc/articles/PMC3407218/figure/pone-0038375-g005/" class="imagepopup">Figure 5. Schematic representation of intracellular <span class="highlight" style="background-color:">signaling</span> pathways in the modulation of CsA on the invasion and proliferation of human trophoblast cells..  From: CXCL12/CXCR4 Axis Triggers the Activation of EGF Receptor and ERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in CsA-Induced Proliferation of Human Trophoblast Cells. </a></p><div class="supp"><p class="details"><div>CsA promotes the invasion of human trophoblast cells via two independent <span class="highlight" style="background-color:">signaling</span> pathways, calcium/CaN/NFAT and EGFR/ERK. E-cadherin (labeled as E-cad), MMP-2, MMP-9, and titin may function as the downstream targets of EGFR/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the CsA-induced invasion of human trophoblast cells. CsA may promote the activation of EGFR via CXCL12/CXCR4 axis. EGFR and its downstream ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may also be involved in the CsA-induced proliferation of human trophoblast cells. Blue lines represent the <span class="highlight" style="background-color:">signaling</span> pathways proved to be involved in the regulation of CsA on the invasion of human trophoblast cells, and red lines represent the putative <span class="highlight" style="background-color:">signaling</span> pathways involved in the regulation of CsA on the proliferation of trophoblast cells.</div></p></div><div class="aux"><div class="resc">Hong-Bo Zhao, et al. PLoS One. 2012;7(7):e38375.</div><p class="links"><a class="dblinks" href="/pubmed/22848341" ref="ordinalpos=43&amp;ncbi_uid=4147513&amp;link_uid=22848341">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407218/" ref="ordinalpos=43&amp;ncbi_uid=4147513&amp;link_uid=PMC3407218">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>44.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3724870/figure/pone-0069754-g005/" ref="ordinalpos=44&amp;ncbi_uid=5171204&amp;link_uid=PMC3724870" href="/pmc/articles/PMC3724870/"><img src="/pmc/articles/instance/3724870/bin/pone.0069754.g005.gif" src-large="/pmc/articles/instance/3724870/bin/pone.0069754.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3724870/" ref="ordinalpos=44&amp;ncbi_uid=5171204&amp;link_uid=PMC3724870" image-link="/pmc/articles/PMC3724870/figure/pone-0069754-g005/" class="imagepopup">Figure 5. Analysis of adipogenic-specific <span class="highlight" style="background-color:">signaling</span> pathways..  From: Reverse Differentiation as a Gene Filtering Tool in Genome Expression Profiling of Adipogenesis for Fat Marker Gene Selection and Their Analysis. </a></p><div class="supp"><p class="details"><div>The 991 genes that were differentially expressed during adipogenesis were uploaded to the KEGG database to determine their involvement in adipogenic <span class="highlight" style="background-color:">signaling</span> pathways. We found different <span class="highlight" style="background-color:">signaling</span> cascades for adipogenesis like the (<b>A</b>) insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, (<b>B</b>) <i>PPARG</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, (<b>C</b>) fatty acid biosynthesis <span class="highlight" style="background-color:">pathway</span>, (<b>D</b>) adipocytokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, (<b>E</b>) fatty acid elongation <span class="highlight" style="background-color:">pathway</span>, (<b>F</b>) <span class="highlight" style="background-color:">pathway</span> for biosynthesis of unsaturated fatty acids and (<b>G</b>) <span class="highlight" style="background-color:">pathway</span> for fatty acid metabolism. Error bars, Means  S.E.M.; <i>*P</i>&lt;0.05; <i>**P</i>&lt;0.01; <i>***P</i>&lt;0.001, NS, not significant (One way ANOVA, performed for statistical analysis).</div></p></div><div class="aux"><div class="resc">Mujib Ullah, et al. PLoS One. 2013;8(7):e69754.</div><p class="links"><a class="dblinks" href="/pubmed/23922792" ref="ordinalpos=44&amp;ncbi_uid=5171204&amp;link_uid=23922792">Citation</a><a class="dblinks" href="/pmc/articles/PMC3724870/" ref="ordinalpos=44&amp;ncbi_uid=5171204&amp;link_uid=PMC3724870">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>45.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1839983/figure/F4/" ref="ordinalpos=45&amp;ncbi_uid=890850&amp;link_uid=PMC1839983" href="/pmc/articles/PMC1839983/"><img src="/pmc/articles/instance/1839983/bin/nihms16683f4.gif" src-large="/pmc/articles/instance/1839983/bin/nihms16683f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1839983/" ref="ordinalpos=45&amp;ncbi_uid=890850&amp;link_uid=PMC1839983" image-link="/pmc/articles/PMC1839983/figure/F4/" class="imagepopup">Figure 4. The Completed Tubulogenic <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> for Fibronectin.  From: Intracellular <span class="highlight" style="background-color:">Signaling</span> via ERK/MAPK Completes the <span class="highlight" style="background-color:">Pathway</span> for Tubulogenic Fibronectin in MDCK Cells. </a></p><div class="supp"><p class="details"><div>During tubulogenesis in the MDCK cells, HGF binds to the CMET receptor tyrosine kinase receptor and activates the ERK/MAPK <span class="highlight" style="background-color:">pathway</span>, which leads to transcription and secretion of tubulogenic fibronectin.</div></p></div><div class="aux"><div class="resc">Zhao Liu, et al. Biochem Biophys Res Commun. ;353(3):793-798.</div><p class="links"><a class="dblinks" href="/pubmed/17196167" ref="ordinalpos=45&amp;ncbi_uid=890850&amp;link_uid=17196167">Citation</a><a class="dblinks" href="/pmc/articles/PMC1839983/" ref="ordinalpos=45&amp;ncbi_uid=890850&amp;link_uid=PMC1839983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>46.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3343102/figure/pgen-1002686-g009/" ref="ordinalpos=46&amp;ncbi_uid=3990973&amp;link_uid=PMC3343102" href="/pmc/articles/PMC3343102/"><img src="/pmc/articles/instance/3343102/bin/pgen.1002686.g009.gif" src-large="/pmc/articles/instance/3343102/bin/pgen.1002686.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3343102/" ref="ordinalpos=46&amp;ncbi_uid=3990973&amp;link_uid=PMC3343102" image-link="/pmc/articles/PMC3343102/figure/pgen-1002686-g009/" class="imagepopup">Figure 9. A hypothetical working model for the role of LIC in the BR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Dynamics of Brassinosteroid Response Modulated by Negative Regulator LIC in Rice. </a></p><div class="supp"><p class="details"><div>BZR1 is a positive transcription factor and represents an activation <span class="highlight" style="background-color:">pathway</span>, whereas LIC functions antagonistically as a negative transcription factor and mediates a brake <span class="highlight" style="background-color:">pathway</span> in BR <span class="highlight" style="background-color:">signaling</span>. Both BZR1 and LIC are phosphorylated by BIN2/GSK1 and transported to the cytoplasm in the absence of BR (green represents positive members and the activation <span class="highlight" style="background-color:">pathway</span>, and red represents negative members and repression <span class="highlight" style="background-color:">pathway</span> in BR <span class="highlight" style="background-color:">signaling</span>).</div></p></div><div class="aux"><div class="resc">Cui Zhang, et al. PLoS Genet. 2012 April;8(4):e1002686.</div><p class="links"><a class="dblinks" href="/pubmed/22570626" ref="ordinalpos=46&amp;ncbi_uid=3990973&amp;link_uid=22570626">Citation</a><a class="dblinks" href="/pmc/articles/PMC3343102/" ref="ordinalpos=46&amp;ncbi_uid=3990973&amp;link_uid=PMC3343102">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>47.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3156703/figure/pone-0022619-g001/" ref="ordinalpos=47&amp;ncbi_uid=3166310&amp;link_uid=PMC3156703" href="/pmc/articles/PMC3156703/"><img src="/pmc/articles/instance/3156703/bin/pone.0022619.g001.gif" src-large="/pmc/articles/instance/3156703/bin/pone.0022619.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3156703/" ref="ordinalpos=47&amp;ncbi_uid=3166310&amp;link_uid=PMC3156703" image-link="/pmc/articles/PMC3156703/figure/pone-0022619-g001/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> triggered by speract in sea urchin sperm..  From: Discrete Dynamics Model for the Speract-Activated Ca2+ <span class="highlight" style="background-color:">Signaling</span> Network Relevant to Sperm Motility. </a></p><div class="supp"><p class="details"><div>A) Main components involved in the speract <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The binding of speract with its receptor in the flagellar membrane triggers the cascade that produces changes in <img src="/pmc/articles/instance/3156703/bin/pone.0022619.e021.jpg" class="inline-graphic" style="border-style: none;" /> in two different forms: a sustained (tonic) increment and superimposed (supratonic) fluctuations. B) Events produced by the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jess Espinal, et al. PLoS One. 2011;6(8):e22619.</div><p class="links"><a class="dblinks" href="/pubmed/21857937" ref="ordinalpos=47&amp;ncbi_uid=3166310&amp;link_uid=21857937">Citation</a><a class="dblinks" href="/pmc/articles/PMC3156703/" ref="ordinalpos=47&amp;ncbi_uid=3166310&amp;link_uid=PMC3156703">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>48.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3306981/figure/fig1/" ref="ordinalpos=48&amp;ncbi_uid=3891420&amp;link_uid=PMC3306981" href="/pmc/articles/PMC3306981/"><img src="/pmc/articles/instance/3306981/bin/JO2012-351089.001.gif" src-large="/pmc/articles/instance/3306981/bin/JO2012-351089.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3306981/" ref="ordinalpos=48&amp;ncbi_uid=3891420&amp;link_uid=PMC3306981" image-link="/pmc/articles/PMC3306981/figure/fig1/" class="imagepopup">Figure 1.  From: Matricellular Proteins: A Sticky Affair with Cancers. </a></p><div class="supp"><p class="details"><div>Summarized the <span class="highlight" style="background-color:">signaling</span> mechanisms of various matricellular proteins contributing to cancer progression. ANGPTL4 binds to both integrins and ECM to promote tumor survival, tumor invasion and modulate the availability of ECM. (a) ANGPTL4 interacting with integrin activates Rac1 and NADPH oxidase, which generate high level of O<sub>2</sub><sup></sup>. This will further activating the Src machinery and stimulates its downstream PI3K/PKB mediated survival <span class="highlight" style="background-color:">pathway</span>. (b) ANGPTL4 interacting with integrin also activates FAK-src-PAK1 <span class="highlight" style="background-color:">signaling</span> and PKC/14-3-3 mediated <span class="highlight" style="background-color:">pathway</span> which modulate cell migration via integrin internalization. (c) ANGPTL4 binds specific matrix proteins and delays their degradation by proteases. However, this association does not interfere with integrin-matrix protein recognition unlike TNC. (d) TNC can compete with fibronectin to bind integrin <i></i>5<i></i>1 coreceptor, syndecan-4, which blocks the activation of promigratory FAK/RhoA/ROCK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (e) TNC can activate Wnt <span class="highlight" style="background-color:">signaling</span> by downregulating the soluble inhibitor DKK-1, thus resulted in nuclear localization of <i></i>-catenin. Nuclear <i></i>-catenin interacts with TCF/LEF to promote the expression of genes contributing to tumor formation, survival, and metastasis. OPN can interact with several (f) integrins and also (g) CD44 family of receptors. These complexes are able to mediate tumor cell survival through PI3k/PKB <span class="highlight" style="background-color:">pathway</span> activation and motility for detachment or invasion of tumor cell through the activation of AP-1-dependent gene expression via the MEK/Erk <span class="highlight" style="background-color:">pathway</span>. (h) Certain domains of TSP1 (such as NoC1) can bind directly to integrins to activate <span class="highlight" style="background-color:">signaling</span> proteins such as Erk1/2 and paxillin which modulates tumor formation. (i) TSP1 binding to CD36 activates Fyn and p38 MAPK <span class="highlight" style="background-color:">pathway</span> which is essential for the suppression of tumor growth. (j) TSP1 can also bind CD47, to modulate sGC and cGMP-dependent protein kinase, thus inhibiting the NO <span class="highlight" style="background-color:">signaling</span> necessary for angiogenesis. (k) TSP-1 association with CD47 or direct competitive binding of TSP1 to VEGF can inhibit VEGFR2 <span class="highlight" style="background-color:">signaling</span>. VEGFR2 activates the PI3K/PKB <span class="highlight" style="background-color:">pathway</span> which leads to activation of eNOS/NO <span class="highlight" style="background-color:">signaling</span>. Simultaneously, VEGFR2 can also signal through PLC<i></i>, which further increases AMPK-mediated eNOS phosphorylation and NO production. eNOS/NO <span class="highlight" style="background-color:">signaling</span> regulate downstream targets that increase endothelial cell proliferation, migration, survival, and permeability. (l) TSP1 can activate TGF<i></i>/smad <span class="highlight" style="background-color:">pathway</span> to inhibit tumor cell proliferation and induce apoptosis. (m) SPARC binds integrin, inducing ILK/FAK/PKB activation to increase cell migration. (n) Cyr61 can promote tumor cell proliferation and survival through the activation of integrin mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> either by direct binding with integrin or integrin-syndecan4. The downstream intracellular events may be mediated through the FAK/PI3K/PKB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, resulting in either activation of the NF-<i></i>B survival <span class="highlight" style="background-color:">pathway</span> or phosphorylation of GSK3<i></i> and nuclear translocation of <i></i>-catenin for cell proliferation. (o) Cyr61 allows protein degradation of E-cadherin leading to <i></i>-catenin translocation.</div></p></div><div class="aux"><div class="resc">Han Chung Chong, et al. J Oncol. 2012;2012:351089.</div><p class="links"><a class="dblinks" href="/pubmed/22481923" ref="ordinalpos=48&amp;ncbi_uid=3891420&amp;link_uid=22481923">Citation</a><a class="dblinks" href="/pmc/articles/PMC3306981/" ref="ordinalpos=48&amp;ncbi_uid=3891420&amp;link_uid=PMC3306981">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>49.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2855089/figure/fig2/" ref="ordinalpos=49&amp;ncbi_uid=2249150&amp;link_uid=PMC2855089" href="/pmc/articles/PMC2855089/"><img src="/pmc/articles/instance/2855089/bin/MI2010-823821.002.gif" src-large="/pmc/articles/instance/2855089/bin/MI2010-823821.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2855089/" ref="ordinalpos=49&amp;ncbi_uid=2249150&amp;link_uid=PMC2855089" image-link="/pmc/articles/PMC2855089/figure/fig2/" class="imagepopup">Figure 2.  From: Regulation of IB Function and NF-B <span class="highlight" style="background-color:">Signaling</span>: AEBP1 Is a Novel Proinflammatory Mediator in Macrophages. </a></p><div class="supp"><p class="details"><div>NF-<i></i>B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. A cartoon representing the cascade of biochemical events that are initiated by various stimuli, eventually leading to NF-<i></i>B nuclear translocation and transcriptional activation. Recruitment of different adaptor molecules to different receptor complexes coupled with activation of different downstream kinases is shown. There are mainly two <span class="highlight" style="background-color:">signaling</span> pathways leading to NF-<i></i>B activation, classical (also known as canonical) and alternative. The formation and activation of the IKK complex, which consists of catalytically active kinases (e.g., IKK<i></i>, IKK<i></i>, and IKK<i></i>) and noncatalytic regulatory proteins (e.g., NEMO and ELKS), is a universal event in both <span class="highlight" style="background-color:">signaling</span> pathways. In the classical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, ligand binding to a cell surface receptor leads to the recruitment of adaptor proteins (e.g., TRAF6) to the receptor, leading to the recruitment of IKK complex and subsequent phosphorylation and degradation of the I<i></i>B proteins. Unlike the classical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the alternative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is normally triggered by non-proinflammatory cytokines (e.g., LT<i></i>, BAFF, and CD40L) as well as some viruses (e.g., HTLV and EBV), does not allow the recruitment of NEMO. Instead, ligand binding to a cell surface receptor leads to the recruitment of NIK, which in turn phosphorylates and activates IKK<i></i> dimers. Typically, the classical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to the activation of NF-<i></i>B dimers consisting of RelA, c-Rel, RelB, and p50, while the alternative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to the activation of NF-<i></i>B dimers consisting primarily of RelB and p52.</div></p></div><div class="aux"><div class="resc">Amin Majdalawieh, et al. Mediators Inflamm. 2010;2010:823821.</div><p class="links"><a class="dblinks" href="/pubmed/20396415" ref="ordinalpos=49&amp;ncbi_uid=2249150&amp;link_uid=20396415">Citation</a><a class="dblinks" href="/pmc/articles/PMC2855089/" ref="ordinalpos=49&amp;ncbi_uid=2249150&amp;link_uid=PMC2855089">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>50.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2834552/figure/F11/" ref="ordinalpos=50&amp;ncbi_uid=2737558&amp;link_uid=PMC2834552" href="/pmc/articles/PMC2834552/"><img src="/pmc/articles/instance/2834552/bin/asn0021042730011.gif" src-large="/pmc/articles/instance/2834552/bin/asn0021042730011.jpg" alt="Figure 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2834552/" ref="ordinalpos=50&amp;ncbi_uid=2737558&amp;link_uid=PMC2834552" image-link="/pmc/articles/PMC2834552/figure/F11/" class="imagepopup">Figure 11. From: Advanced Glycation End-Products Induce Tubular CTGF via TGF-?-Independent Smad3 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A schematic diagram of AGE-induced CTGF expression is shown. Engagement of RAGE by AGEs activates Smad <span class="highlight" style="background-color:">signaling</span> <i>via</i> both the ERK/p38 MAPK-Smad <span class="highlight" style="background-color:">signaling</span> cross-talk <span class="highlight" style="background-color:">pathway</span> (TGF-independent <span class="highlight" style="background-color:">pathway</span>) and the classic TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (TGF-dependent <span class="highlight" style="background-color:">pathway</span>) to induce CTGF expression.</div></p></div><div class="aux"><div class="resc">Arthur C.K. Chung, et al. J Am Soc Nephrol. 2010 February;21(2):249-260.</div><p class="links"><a class="dblinks" href="/pubmed/19959709" ref="ordinalpos=50&amp;ncbi_uid=2737558&amp;link_uid=19959709">Citation</a><a class="dblinks" href="/pmc/articles/PMC2834552/" ref="ordinalpos=50&amp;ncbi_uid=2737558&amp;link_uid=PMC2834552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>51.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3278757/figure/F1/" ref="ordinalpos=51&amp;ncbi_uid=4610605&amp;link_uid=PMC3278757" href="/pmc/articles/PMC3278757/"><img src="/pmc/articles/instance/3278757/bin/btr696f1.gif" src-large="/pmc/articles/instance/3278757/bin/btr696f1.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3278757/" ref="ordinalpos=51&amp;ncbi_uid=4610605&amp;link_uid=PMC3278757" image-link="/pmc/articles/PMC3278757/figure/F1/" class="imagepopup">Fig. 1. From: Optimal structural inference of <span class="highlight" style="background-color:">signaling</span> pathways from unordered and overlapping gene sets. </a></p><div class="supp"><p class="details"><div>SA begins with a randomly chosen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> structure in the feasible set. It explores the feasible set in order to locate the structure with the minimum energy (the true <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> structure). The feasible set is composed of of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> structures with the same degree distribution as the true <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Lipi R. Acharya, et al. Bioinformatics. 2012 February 15;28(4):546-556.</div><p class="links"><a class="dblinks" href="/pubmed/22199386" ref="ordinalpos=51&amp;ncbi_uid=4610605&amp;link_uid=22199386">Citation</a><a class="dblinks" href="/pmc/articles/PMC3278757/" ref="ordinalpos=51&amp;ncbi_uid=4610605&amp;link_uid=PMC3278757">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>52.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4063991/figure/Fig1/" ref="ordinalpos=52&amp;ncbi_uid=6293983&amp;link_uid=PMC4063991" href="/pmc/articles/PMC4063991/"><img src="/pmc/articles/instance/4063991/bin/12079_2014_224_Fig1_HTML.gif" src-large="/pmc/articles/instance/4063991/bin/12079_2014_224_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4063991/" ref="ordinalpos=52&amp;ncbi_uid=6293983&amp;link_uid=PMC4063991" image-link="/pmc/articles/PMC4063991/figure/Fig1/" class="imagepopup">Fig. 1.  From: A network map of the gastrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of the gastrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: this map represents the reactions of the gastrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> present in NetPath. The different types of reactions are color coded as described in the legend. The <span class="highlight" style="background-color:">signaling</span> pathways are also distinguished by <i>colored boxes</i> </div></p></div><div class="aux"><div class="resc">Yashwanth Subbannayya, et al. J Cell Commun Signal. 2014 June;8(2):165-170.</div><p class="links"><a class="dblinks" href="/pubmed/24584707" ref="ordinalpos=52&amp;ncbi_uid=6293983&amp;link_uid=24584707">Citation</a><a class="dblinks" href="/pmc/articles/PMC4063991/" ref="ordinalpos=52&amp;ncbi_uid=6293983&amp;link_uid=PMC4063991">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>53.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2886156/figure/F3/" ref="ordinalpos=53&amp;ncbi_uid=3049039&amp;link_uid=PMC2886156" href="/pmc/articles/PMC2886156/"><img src="/pmc/articles/instance/2886156/bin/nihms195114f3.gif" src-large="/pmc/articles/instance/2886156/bin/nihms195114f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2886156/" ref="ordinalpos=53&amp;ncbi_uid=3049039&amp;link_uid=PMC2886156" image-link="/pmc/articles/PMC2886156/figure/F3/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Pathway</span> analysis reveals global down-regulation of nearly all members of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in psoriasis.  From: Evidence for altered Wnt <span class="highlight" style="background-color:">signaling</span> in psoriatic skin. </a></p><div class="supp"><p class="details"><div>Global gene expression differences between normal and lesional psoriatic skin were overlaid onto a global molecular and <span class="highlight" style="background-color:">pathway</span> network within the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Knowledge Base (A). This revealed near global down-regulation (green) of nearly all members of the canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. However, several genes, associated with the non-canonical Wnt <span class="highlight" style="background-color:">pathway</span>, were upregulated (pink) including; <i>WNT5A</i> and <i>FZD5</i>. Decreased activity of the canonical Wnt-<span class="highlight" style="background-color:">pathway</span> was confirmed by QRT-PCR for <i>AXIN2</i>, a marker of canonical Wnt <span class="highlight" style="background-color:">signaling</span> (B). Bars indicate mean  S.D, * p &lt; 0.05. Immunofluorescent microscopy of normal, uninvolved and psoriatic skin (C) (20X, n=3).</div></p></div><div class="aux"><div class="resc">JE Gudjonsson, et al. J Invest Dermatol. ;130(7):1849-1859.</div><p class="links"><a class="dblinks" href="/pubmed/20376066" ref="ordinalpos=53&amp;ncbi_uid=3049039&amp;link_uid=20376066">Citation</a><a class="dblinks" href="/pmc/articles/PMC2886156/" ref="ordinalpos=53&amp;ncbi_uid=3049039&amp;link_uid=PMC2886156">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>54.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584589/figure/f4-ijo-40-04-1162/" ref="ordinalpos=54&amp;ncbi_uid=4686864&amp;link_uid=PMC3584589" href="/pmc/articles/PMC3584589/"><img src="/pmc/articles/instance/3584589/bin/IJO-40-04-1162-g03.gif" src-large="/pmc/articles/instance/3584589/bin/IJO-40-04-1162-g03.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584589/" ref="ordinalpos=54&amp;ncbi_uid=4686864&amp;link_uid=PMC3584589" image-link="/pmc/articles/PMC3584589/figure/f4-ijo-40-04-1162/" class="imagepopup">Figure 4.  From: MicroRNA-200a suppresses the Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by interacting with ?-catenin. </a></p><div class="supp"><p class="details"><div>Western blot analyses of proteins in the Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and of downstream Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> targets, such as Fra-1 and Cyclin D1.</div></p></div><div class="aux"><div class="resc">JUAN SU, et al. Int J Oncol. 2012 April;40(4):1162-1170.</div><p class="links"><a class="dblinks" href="/pubmed/22211245" ref="ordinalpos=54&amp;ncbi_uid=4686864&amp;link_uid=22211245">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584589/" ref="ordinalpos=54&amp;ncbi_uid=4686864&amp;link_uid=PMC3584589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>55.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3421022/figure/Fig1/" ref="ordinalpos=55&amp;ncbi_uid=4179855&amp;link_uid=PMC3421022" href="/pmc/articles/PMC3421022/"><img src="/pmc/articles/instance/3421022/bin/12079_2012_168_Fig1_HTML.gif" src-large="/pmc/articles/instance/3421022/bin/12079_2012_168_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3421022/" ref="ordinalpos=55&amp;ncbi_uid=4179855&amp;link_uid=PMC3421022" image-link="/pmc/articles/PMC3421022/figure/Fig1/" class="imagepopup">Fig. 1.  From: A <span class="highlight" style="background-color:">pathway</span> map of prolactin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of the prolactin <span class="highlight" style="background-color:">signaling</span> network in NetSlim. Prolactin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> represents a set of reactions induced by the binding of prolactin to the prolactin receptor. Multiple colors differentiate protein-protein interactions, enzyme-catalysis events and translocation reactions in the <span class="highlight" style="background-color:">pathway</span>. The major downstream <span class="highlight" style="background-color:">signaling</span> modules that are activated include JAK/STAT, RAS/RAF/MAPK, PI3-Kinase/AKT and RAC <span class="highlight" style="background-color:">signaling</span></div></p></div><div class="aux"><div class="resc">Aneesha Radhakrishnan, et al. J Cell Commun Signal. 2012 August;6(3):169-173.</div><p class="links"><a class="dblinks" href="/pubmed/22684822" ref="ordinalpos=55&amp;ncbi_uid=4179855&amp;link_uid=22684822">Citation</a><a class="dblinks" href="/pmc/articles/PMC3421022/" ref="ordinalpos=55&amp;ncbi_uid=4179855&amp;link_uid=PMC3421022">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>56.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4029079/figure/F7/" ref="ordinalpos=56&amp;ncbi_uid=6162568&amp;link_uid=PMC4029079" href="/pmc/articles/PMC4029079/"><img src="/pmc/articles/instance/4029079/bin/1471-2164-15-182-7.gif" src-large="/pmc/articles/instance/4029079/bin/1471-2164-15-182-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4029079/" ref="ordinalpos=56&amp;ncbi_uid=6162568&amp;link_uid=PMC4029079" image-link="/pmc/articles/PMC4029079/figure/F7/" class="imagepopup">Figure 7.  From: Transcriptomic analysis of differentially expressed genes in the Ras1CA-overexpressed and wildtype posterior silk glands. </a></p><div class="supp"><p class="details"><div><b>Small-molecule chemical inhibitor treatments to identify which Ras1 downstream <span class="highlight" style="background-color:">signaling</span> pathways are involved in regulating DEGs in pathways in cancer, insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> Common DEGs in all the three pathways. <b>(B)</b> Common DEGs in pathways in cancer and insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(C)</b> A common DEG in pathways in cancer and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(D)</b> DEGs only in pathways in cancer. <b>(E)</b> DEGs only in insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(F)</b> DEGs only in MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Small-molecule chemical inhibitors of the Ras downstream effectors (Rafi, Raf inhibitor; LY294002, PI3K inhibitor; Rapa, rapamycin, TORC1 inhibitor; 15 g/larva) are injected into <i>Ras1</i><sup>CA</sup>-overexpressed silkworm larvae. Inhibitors are marked with different color: black, DMSO as a control; red: Raf inhibitor; green, LY294002; blue: rapamycin.</div></p></div><div class="aux"><div class="resc">Li Ma, et al. BMC Genomics. 2014;15:182-182.</div><p class="links"><a class="dblinks" href="/pubmed/24606580" ref="ordinalpos=56&amp;ncbi_uid=6162568&amp;link_uid=24606580">Citation</a><a class="dblinks" href="/pmc/articles/PMC4029079/" ref="ordinalpos=56&amp;ncbi_uid=6162568&amp;link_uid=PMC4029079">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>57.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889255/figure/Fig1/" ref="ordinalpos=57&amp;ncbi_uid=5705568&amp;link_uid=PMC3889255" href="/pmc/articles/PMC3889255/"><img src="/pmc/articles/instance/3889255/bin/12079_2013_197_Fig1_HTML.gif" src-large="/pmc/articles/instance/3889255/bin/12079_2013_197_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889255/" ref="ordinalpos=57&amp;ncbi_uid=5705568&amp;link_uid=PMC3889255" image-link="/pmc/articles/PMC3889255/figure/Fig1/" class="imagepopup">Fig. 1.  From: An integrated map of corticotropin-releasing hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of CRH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> - This map represents the NetPath reactions of CRH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The different types of reactions are distinguished by <i>colors</i> as described in the legend</div></p></div><div class="aux"><div class="resc">Tejaswini Subbannayya, et al. J Cell Commun Signal. 2013 December;7(4):295-300.</div><p class="links"><a class="dblinks" href="/pubmed/23504413" ref="ordinalpos=57&amp;ncbi_uid=5705568&amp;link_uid=23504413">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889255/" ref="ordinalpos=57&amp;ncbi_uid=5705568&amp;link_uid=PMC3889255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>58.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355158/figure/pone-0036560-g008/" ref="ordinalpos=58&amp;ncbi_uid=4013510&amp;link_uid=PMC3355158" href="/pmc/articles/PMC3355158/"><img src="/pmc/articles/instance/3355158/bin/pone.0036560.g008.gif" src-large="/pmc/articles/instance/3355158/bin/pone.0036560.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355158/" ref="ordinalpos=58&amp;ncbi_uid=4013510&amp;link_uid=PMC3355158" image-link="/pmc/articles/PMC3355158/figure/pone-0036560-g008/" class="imagepopup">Figure 8. Preconditioning by LPS does not require canonical Wnt <span class="highlight" style="background-color:">signaling</span>..  From: Novel Role for the Innate Immune Receptor Toll-Like Receptor 4 (TLR4) in the Regulation of the Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Photoreceptor Apoptosis. </a></p><div class="supp"><p class="details"><div>(A) Wnt <span class="highlight" style="background-color:">signaling</span> was measured in preconditioned Muller glia-photoreceptor cultures. The cultures were treated with LPS with or without Wnt3a, 2 hr prior to injury by H<sub>2</sub>O<sub>2</sub> (0.4 mM). Wnt <span class="highlight" style="background-color:">signaling</span> was measured using a luciferase reporter assay. Wnt3a increased Wnt <span class="highlight" style="background-color:">signaling</span> (p&lt;0.05, n=5, compared with H<sub>2</sub>O<sub>2</sub> only) but preconditioning with LPS did not induce Wnt <span class="highlight" style="background-color:">signaling</span> or change the level of Wnt3a-dependent Wnt <span class="highlight" style="background-color:">signaling</span>. Therefore, LPS-induced preconditioning is independent of the canonical Wnt <span class="highlight" style="background-color:">pathway</span>. (B) The Muller glia-photoreceptor cultures were incubated with LPS with or without the Wnt <span class="highlight" style="background-color:">pathway</span> inhibitor Dkk1 (200 ng/ml), followed by exposure to H<sub>2</sub>O<sub>2</sub>. The protection by LPS was not altered by Dkk1, indicating that endogenous Wnt <span class="highlight" style="background-color:">pathway</span> activation is not required. Mean  SD, n=5, compared with PBS treated. Viability was measured by Cell Titer Blue. (C) Proposed model of regulation of photoreceptor neuroprotection by cross-talk between the TLR4 and Wnt <span class="highlight" style="background-color:">signaling</span> pathways. (Left) During oxidative stress injury, TLR4 <span class="highlight" style="background-color:">signaling</span> suppresses Wnt <span class="highlight" style="background-color:">signaling</span> at the level of LRP6 receptor activation, leading to reduced Wnt3a-mediated photoreceptor protection. (Right) In the presence of LPS preconditioning prior to oxidative stress, activation of TLR4 reduces TNF, leading to increased neuroprotection. TLR4 does not regulate the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during preconditioning.</div></p></div><div class="aux"><div class="resc">Hyun Yi, et al. PLoS One. 2012;7(5):e36560.</div><p class="links"><a class="dblinks" href="/pubmed/22615780" ref="ordinalpos=58&amp;ncbi_uid=4013510&amp;link_uid=22615780">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355158/" ref="ordinalpos=58&amp;ncbi_uid=4013510&amp;link_uid=PMC3355158">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>59.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3982803/figure/F1/" ref="ordinalpos=59&amp;ncbi_uid=6014272&amp;link_uid=PMC3982803" href="/pmc/articles/PMC3982803/"><img src="/pmc/articles/instance/3982803/bin/nihms-569808-f0001.gif" src-large="/pmc/articles/instance/3982803/bin/nihms-569808-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3982803/" ref="ordinalpos=59&amp;ncbi_uid=6014272&amp;link_uid=PMC3982803" image-link="/pmc/articles/PMC3982803/figure/F1/" class="imagepopup">Figure 1.  From: The AKTion in non-canonical insulin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Insulin <span class="highlight" style="background-color:">signaling</span> pathways that control gluconeogenesis. (<b>a</b>) The canonical insulin receptor (IR)-IRS-PI3K-PDPK1-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates gluconeogenesis by inhibiting the transcription factor FoxO1 and the transcriptional regulatory CREB-CBP-CRTC2 complex. After feeding, insulin receptorIRS-PI3K-PDPK1-AKT <span class="highlight" style="background-color:">signaling</span> is stimulated (green), which phosphorylates and inactivates FoxO1 and the CREB-CBP-CRTC2 complex, thus suppressing the expression of gluconeogenic genes (encoding PEPCK and G6Pase) (gray). During fasting, insulin receptorIRS-PI3K-PDPK1-AKT <span class="highlight" style="background-color:">signaling</span> is less active than in feeding (gray), thus turning on the activities of FoxO1, the CREB-CBP-CRTC2 complex and gluconeogenesis (green). The P in the yellow circles denotes activating phosphorylation, and the P in the red circles denotes inhibitory phosphorylation. (<b>b</b>) Lu <i>et al</i>.<sup>8</sup> suggest the existence of a hypothetical non-canonical <span class="highlight" style="background-color:">pathway</span> by which insulin controls gluconeogenesis. They suggest that the non-canonical <span class="highlight" style="background-color:">pathway</span>, which has yet to be identified, suppresses gluconeogenesis in parallel with the canonical <span class="highlight" style="background-color:">pathway</span> during feeding; however, activated FoxO1 during fasting or insulin resistance dampens the non-canonical <span class="highlight" style="background-color:">pathway</span> and stimulates the expression of gluconeogenic genes and glucose production. A comparison of TLKO mice and LTKO<sup>IRS</sup> mice implied that the divergent point of the non-canonical <span class="highlight" style="background-color:">pathway</span> from the canonical <span class="highlight" style="background-color:">pathway</span> could reside between IRS and AKT. (<b>c</b>) The PDPK1-aPKCl <span class="highlight" style="background-color:">signaling</span> cascade may be a potential candidate accounting for the non-canonical <span class="highlight" style="background-color:">pathway</span>. In response to insulin or stimuli after feeding, insulin receptorIRS-PI3K-PDPK1 <span class="highlight" style="background-color:">signaling</span> activates aPKC, which, in turn, phosphorylates CBP and disrupts the CREB-CBP-CRTC2 complex and suppresses gluconeogenesis. However, in DLKO or TLKO mice, this <span class="highlight" style="background-color:">pathway</span> could remain active and contribute to the postprandial response.</div></p></div><div class="aux"><div class="resc">Zhiyong Cheng, et al. Nat Med. 2012 March 6;18(3):351-353.</div><p class="links"><a class="dblinks" href="/pubmed/22395698" ref="ordinalpos=59&amp;ncbi_uid=6014272&amp;link_uid=22395698">Citation</a><a class="dblinks" href="/pmc/articles/PMC3982803/" ref="ordinalpos=59&amp;ncbi_uid=6014272&amp;link_uid=PMC3982803">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>60.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3206446/figure/F10/" ref="ordinalpos=60&amp;ncbi_uid=3323276&amp;link_uid=PMC3206446" href="/pmc/articles/PMC3206446/"><img src="/pmc/articles/instance/3206446/bin/1471-213X-11-57-10.gif" src-large="/pmc/articles/instance/3206446/bin/1471-213X-11-57-10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3206446/" ref="ordinalpos=60&amp;ncbi_uid=3323276&amp;link_uid=PMC3206446" image-link="/pmc/articles/PMC3206446/figure/F10/" class="imagepopup">Figure 10.  From: Loss of the Drosophila cell polarity regulator Scribbled promotes epithelial tissue overgrowth and cooperation with oncogenic Ras-Raf through impaired Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model for how the loss of <i>scrib </i>affects Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span></b>. In an epithelial cell, the core Hippo <span class="highlight" style="background-color:">pathway</span> components Hpo (H), Wts (W), Sav (S) and Mats (M) inhibit the activity of Yki, which acts to regulate transcription through its DNA binding partner, Sd. Extracellular regulation of the <span class="highlight" style="background-color:">pathway</span> occurs through the two transmembrane proteins, Crb acting through Ex, and Ft, acting through both Ex and D. Ex can also directly bind to and inhibit Yki. Loss of <i>scrib </i>in the eye disc is associated with aPKC-dependent inhibition of the Hippo <span class="highlight" style="background-color:">pathway</span>. Although JNK <span class="highlight" style="background-color:">signaling</span> is activated by either the loss of <i>scrib</i>, or ectopic aPKC activity, JNK is not required for Hippo <span class="highlight" style="background-color:">pathway</span> inhibition. In the wing disc, loss of <i>scrib </i>is also associated with JNK activation, and although JNK <span class="highlight" style="background-color:">signaling</span> has been shown to be sufficient to downregulate the Hippo <span class="highlight" style="background-color:">pathway</span> in the wing disc [37], Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> remains impaired in <i>scrib </i>mutants even when JNK is blocked. Epistasis experiments suggest that <span class="highlight" style="background-color:">pathway</span> inhibition in <i>scrib </i>mutants occurs, at least in part, downstream or in parallel to Ex and Ft, and could impact upon either the core components (H/S/M/W) or upon Yki activity itself.</div></p></div><div class="aux"><div class="resc">Karen Doggett, et al. BMC Dev Biol. 2011;11:57-57.</div><p class="links"><a class="dblinks" href="/pubmed/21955824" ref="ordinalpos=60&amp;ncbi_uid=3323276&amp;link_uid=21955824">Citation</a><a class="dblinks" href="/pmc/articles/PMC3206446/" ref="ordinalpos=60&amp;ncbi_uid=3323276&amp;link_uid=PMC3206446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>61.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3496019/figure/F1/" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=PMC3496019" href="/pmc/articles/PMC3496019/"><img src="/pmc/articles/instance/3496019/bin/nihms-404973-f0001.gif" src-large="/pmc/articles/instance/3496019/bin/nihms-404973-f0001.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3496019/figure/F1/" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=PMC3496019" href="/pmc/articles/PMC3496019/"><img src="/pmc/articles/instance/3496019/bin/nihms-404973-f0002.gif" src-large="/pmc/articles/instance/3496019/bin/nihms-404973-f0002.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3496019/figure/F1/" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=PMC3496019" href="/pmc/articles/PMC3496019/"><img src="/pmc/articles/instance/3496019/bin/nihms-404973-f0003.gif" src-large="/pmc/articles/instance/3496019/bin/nihms-404973-f0003.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3496019/figure/F1/" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=PMC3496019" href="/pmc/articles/PMC3496019/"><img src="/pmc/articles/instance/3496019/bin/nihms-404973-f0004.gif" src-large="/pmc/articles/instance/3496019/bin/nihms-404973-f0004.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3496019/figure/F1/" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=PMC3496019" href="/pmc/articles/PMC3496019/"><img src="/pmc/articles/instance/3496019/bin/nihms-404973-f0005.gif" src-large="/pmc/articles/instance/3496019/bin/nihms-404973-f0005.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3496019/" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=PMC3496019" image-link="/pmc/articles/PMC3496019/figure/F1/" class="imagepopup">Figure 1.  From: Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways in regulation of gene expression, cell proliferation and differentiation. (A) Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (B) Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (C) Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (D) <span class="highlight" style="background-color:">Signaling</span> networks with PTEN involved; (E) Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Zuoren Yu, et al. Int J Biochem Cell Biol. 2012 December;44(12):2144-2151.</div><p class="links"><a class="dblinks" href="/pubmed/22981632" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=22981632">Citation</a><a class="dblinks" href="/pmc/articles/PMC3496019/" ref="ordinalpos=61&amp;ncbi_uid=5539014&amp;link_uid=PMC3496019">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>62.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4279405/figure/F8/" ref="ordinalpos=62&amp;ncbi_uid=7041271&amp;link_uid=PMC4279405" href="/pmc/articles/PMC4279405/"><img src="/pmc/articles/instance/4279405/bin/oncotarget-05-10718-g008.gif" src-large="/pmc/articles/instance/4279405/bin/oncotarget-05-10718-g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4279405/" ref="ordinalpos=62&amp;ncbi_uid=7041271&amp;link_uid=PMC4279405" image-link="/pmc/articles/PMC4279405/figure/F8/" class="imagepopup">Figure 8. Schema of <span class="highlight" style="background-color:">signaling</span> pathways involved in CCL5-induced VEGF expression and angiogenesis in chondrosarcoma cells.  From: CCL5 promotes VEGF-dependent angiogenesis by down-regulating miR-200b through PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human chondrosarcoma cells. </a></p><div class="supp"><p class="details"><div>CCL5 enhances VEGF-dependent angiogenesis in human chondrosarcoma cells by down-regulating miR-200b through PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Guan-Ting Liu, et al. Oncotarget. 2014 November;5(21):10718-10731.</div><p class="links"><a class="dblinks" href="/pubmed/25301739" ref="ordinalpos=62&amp;ncbi_uid=7041271&amp;link_uid=25301739">Citation</a><a class="dblinks" href="/pmc/articles/PMC4279405/" ref="ordinalpos=62&amp;ncbi_uid=7041271&amp;link_uid=PMC4279405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>63.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584057/figure/pone-0056667-g007/" ref="ordinalpos=63&amp;ncbi_uid=4690672&amp;link_uid=PMC3584057" href="/pmc/articles/PMC3584057/"><img src="/pmc/articles/instance/3584057/bin/pone.0056667.g007.gif" src-large="/pmc/articles/instance/3584057/bin/pone.0056667.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584057/" ref="ordinalpos=63&amp;ncbi_uid=4690672&amp;link_uid=PMC3584057" image-link="/pmc/articles/PMC3584057/figure/pone-0056667-g007/" class="imagepopup">Figure 7. Expression of EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream proteins in RCC cells after ER changed..  From: Estrogen Inhibits Renal Cell Carcinoma Cell Progression through Estrogen Receptor-? Activation. </a></p><div class="supp"><p class="details"><div>(A) Expression of EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream proteins in 786-O after siER transfection. (B) Expression of EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream proteins in A498 after ER overexpression.</div></p></div><div class="aux"><div class="resc">Cheng-Ping Yu, et al. PLoS One. 2013;8(2):e56667.</div><p class="links"><a class="dblinks" href="/pubmed/23460808" ref="ordinalpos=63&amp;ncbi_uid=4690672&amp;link_uid=23460808">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584057/" ref="ordinalpos=63&amp;ncbi_uid=4690672&amp;link_uid=PMC3584057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>64.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3595330/figure/F4/" ref="ordinalpos=64&amp;ncbi_uid=4699509&amp;link_uid=PMC3595330" href="/pmc/articles/PMC3595330/"><img src="/pmc/articles/instance/3595330/bin/nihms-425860-f0005.gif" src-large="/pmc/articles/instance/3595330/bin/nihms-425860-f0005.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3595330/" ref="ordinalpos=64&amp;ncbi_uid=4699509&amp;link_uid=PMC3595330" image-link="/pmc/articles/PMC3595330/figure/F4/" class="imagepopup">Figure 4. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by E2 and ERs.  From: Estrogen synthesis and <span class="highlight" style="background-color:">signaling</span> pathways during ageing: from periphery to brain. </a></p><div class="supp"><p class="details"><div>There are 4 estrogen and estrogen receptor <span class="highlight" style="background-color:">signaling</span> pathways. <span class="highlight" style="background-color:">Pathway</span> 1: The nuclear-initiated estrogen <span class="highlight" style="background-color:">signaling</span> mediated through classical ERs leads to the transcriptional changes in estrogen-responsive genes with or without EREs. <span class="highlight" style="background-color:">Pathway</span> 2: The membrane initiated estrogen <span class="highlight" style="background-color:">signaling</span> leads to diverse cytoplasmic effects, including the regulation of membrane-based ion channels, regulation of second messenger systems, and modification of transcription factors or other membrane receptors. <span class="highlight" style="background-color:">Pathway</span> 3: Estrogen can also exert antioxidant effects in an ER-independent manner. <span class="highlight" style="background-color:">Pathway</span> 4: Ligand-independent genomic actions. Growth factors (GF) activate protein-kinase cascades, leading to phosphorylation (P) and activation of nuclear ERs at EREs.</div></p></div><div class="aux"><div class="resc">Jie Cui, et al. Trends Mol Med. 2013 March;19(3):197-209.</div><p class="links"><a class="dblinks" href="/pubmed/23348042" ref="ordinalpos=64&amp;ncbi_uid=4699509&amp;link_uid=23348042">Citation</a><a class="dblinks" href="/pmc/articles/PMC3595330/" ref="ordinalpos=64&amp;ncbi_uid=4699509&amp;link_uid=PMC3595330">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>65.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3648526/figure/pone-0063799-g002/" ref="ordinalpos=65&amp;ncbi_uid=4888675&amp;link_uid=PMC3648526" href="/pmc/articles/PMC3648526/"><img src="/pmc/articles/instance/3648526/bin/pone.0063799.g002.gif" src-large="/pmc/articles/instance/3648526/bin/pone.0063799.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3648526/" ref="ordinalpos=65&amp;ncbi_uid=4888675&amp;link_uid=PMC3648526" image-link="/pmc/articles/PMC3648526/figure/pone-0063799-g002/" class="imagepopup">Figure 2. Activation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the glomeruli of PHN rats..  From: Rapamycin Upregulates Autophagy by Inhibiting the mTOR-ULK1 <span class="highlight" style="background-color:">Pathway</span>, Resulting in Reduced Podocyte Injury. </a></p><div class="supp"><p class="details"><div>Compared to control rats, the phosphorylation of mTOR, p70S6K, and 4EBP1 was enhanced in PHN rats on day 7 and continued to day 14, indicating that the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was activated, *<i>P</i>&lt;0.05 vs Control.</div></p></div><div class="aux"><div class="resc">Lingling Wu, et al. PLoS One. 2013;8(5):e63799.</div><p class="links"><a class="dblinks" href="/pubmed/23667674" ref="ordinalpos=65&amp;ncbi_uid=4888675&amp;link_uid=23667674">Citation</a><a class="dblinks" href="/pmc/articles/PMC3648526/" ref="ordinalpos=65&amp;ncbi_uid=4888675&amp;link_uid=PMC3648526">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>66.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2597734/figure/pone-0003966-g003/" ref="ordinalpos=66&amp;ncbi_uid=1550447&amp;link_uid=PMC2597734" href="/pmc/articles/PMC2597734/"><img src="/pmc/articles/instance/2597734/bin/pone.0003966.g003.gif" src-large="/pmc/articles/instance/2597734/bin/pone.0003966.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2597734/" ref="ordinalpos=66&amp;ncbi_uid=1550447&amp;link_uid=PMC2597734" image-link="/pmc/articles/PMC2597734/figure/pone-0003966-g003/" class="imagepopup">Figure 3. Components of the Toll-NFB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulate A42 polypetide induced neurodegeneration..  From: The Toll-&gt;NF?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Mediates the Neuropathological Effects of the Human Alzheimer's A?42 Polypeptide in Drosophila. </a></p><div class="supp"><p class="details"><div>Panels AF: In the experiments shown in this figure all flies are siblings that are hemizygous for the pGMR-A42 transgene. The fly on the left side of each panel is a wild type sib, while the fly on the right side of each panel is a sib that is heterozygous for the indicated mutation in the <i>Toll-NF</i><i>B</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutant. Panel A: <i>dl<sup>1</sup></i>. Panel B: <i>dl<sup>4</sup></i>. Panel C: <i>dl<sup>8</sup></i>. Panel D: <i>pll<sup>2</sup></i>. Panel E: <i>pll<sup>7</sup></i>. Panel F: <i>tub</i>.</div></p></div><div class="aux"><div class="resc">Lihua Tan, et al. PLoS ONE. 2008;3(12):e3966.</div><p class="links"><a class="dblinks" href="/pubmed/19088848" ref="ordinalpos=66&amp;ncbi_uid=1550447&amp;link_uid=19088848">Citation</a><a class="dblinks" href="/pmc/articles/PMC2597734/" ref="ordinalpos=66&amp;ncbi_uid=1550447&amp;link_uid=PMC2597734">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>67.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2698936/figure/F2/" ref="ordinalpos=67&amp;ncbi_uid=1856895&amp;link_uid=PMC2698936" href="/pmc/articles/PMC2698936/"><img src="/pmc/articles/instance/2698936/bin/nihms105796f2.gif" src-large="/pmc/articles/instance/2698936/bin/nihms105796f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2698936/" ref="ordinalpos=67&amp;ncbi_uid=1856895&amp;link_uid=PMC2698936" image-link="/pmc/articles/PMC2698936/figure/F2/" class="imagepopup">Figure 2. A model of 6 integrin/IGF-1R/ERK coordinated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated as lens cells initiate differentiation.  From: Integrins in Lens Development and Disease. </a></p><div class="supp"><p class="details"><div>The great versatility of integrin receptors relies on their ability to form <span class="highlight" style="background-color:">signaling</span> complexes with many distinct <span class="highlight" style="background-color:">signaling</span> effectors. Possibly most important among these is the coordinate <span class="highlight" style="background-color:">signaling</span> between integrins and growth factor receptors. These <span class="highlight" style="background-color:">signaling</span> complexes are identified by co-immunoprecipitation of their component molecules. Modeled in this diagram is the 6 integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated as lens cells initiate their differentiation in the equatorial zone. This <span class="highlight" style="background-color:">pathway</span> involves transactivation of IGF-1R and ERK by 6 integrin. The activation of the IGF-1R leads to an ERK-dependent induction of Bcl-2 and IAP3.</div></p></div><div class="aux"><div class="resc">Janice Walker, et al. Exp Eye Res. ;88(2):216-225.</div><p class="links"><a class="dblinks" href="/pubmed/18671967" ref="ordinalpos=67&amp;ncbi_uid=1856895&amp;link_uid=18671967">Citation</a><a class="dblinks" href="/pmc/articles/PMC2698936/" ref="ordinalpos=67&amp;ncbi_uid=1856895&amp;link_uid=PMC2698936">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>68.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3684622/figure/F4/" ref="ordinalpos=68&amp;ncbi_uid=5443757&amp;link_uid=PMC3684622" href="/pmc/articles/PMC3684622/"><img src="/pmc/articles/instance/3684622/bin/nihms473465f4.gif" src-large="/pmc/articles/instance/3684622/bin/nihms473465f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3684622/" ref="ordinalpos=68&amp;ncbi_uid=5443757&amp;link_uid=PMC3684622" image-link="/pmc/articles/PMC3684622/figure/F4/" class="imagepopup">Fig. 4. Models of miRNA-controlled <span class="highlight" style="background-color:">signaling</span> networks and the potential impact of miRNA regulation on the strength and threshold of <span class="highlight" style="background-color:">signaling</span>.  From: Regulation of immune responses and tolerance: the microRNA perspective. </a></p><div class="supp"><p class="details"><div>(A) miRNAs as negative feedback to dampen the expression of multiple positive <span class="highlight" style="background-color:">signaling</span> molecules. (B) miRNAs as a positive feedback to dampen a single negative <span class="highlight" style="background-color:">signaling</span> molecule. (C) miRNAs as a negative tuner of immune <span class="highlight" style="background-color:">signaling</span> by controlling the expression of multiple negative <span class="highlight" style="background-color:">signaling</span> molecules. (D) Effects of miRNA control on the strength and threshold of immune <span class="highlight" style="background-color:">signaling</span>. A1, A2, and An represent positive <span class="highlight" style="background-color:">signaling</span> molecules in the <span class="highlight" style="background-color:">pathway</span> (i.e. the positive kinases). B1, B2, and Bn represent negative <span class="highlight" style="background-color:">signaling</span> molecules in the <span class="highlight" style="background-color:">pathway</span> (i.e. the phosphatases that inactive positive kinases).</div></p></div><div class="aux"><div class="resc">Chang-Zheng Chen, et al. Immunol Rev. ;253(1):112-128.</div><p class="links"><a class="dblinks" href="/pubmed/23550642" ref="ordinalpos=68&amp;ncbi_uid=5443757&amp;link_uid=23550642">Citation</a><a class="dblinks" href="/pmc/articles/PMC3684622/" ref="ordinalpos=68&amp;ncbi_uid=5443757&amp;link_uid=PMC3684622">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>69.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2804830/figure/bire-80-04-16-f09/" ref="ordinalpos=69&amp;ncbi_uid=2188222&amp;link_uid=PMC2804830" href="/pmc/articles/PMC2804830/"><img src="/pmc/articles/instance/2804830/bin/bire-80-04-16-f09.gif" src-large="/pmc/articles/instance/2804830/bin/bire-80-04-16-f09.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2804830/" ref="ordinalpos=69&amp;ncbi_uid=2188222&amp;link_uid=PMC2804830" image-link="/pmc/articles/PMC2804830/figure/bire-80-04-16-f09/" class="imagepopup">FIG. 9. From: Mitogen-Activated Protein Kinase <span class="highlight" style="background-color:">Signaling</span> in Male Germ Cell Apoptosis in the Rat. </a></p><div class="supp"><p class="details"><div>Upstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in germ cell apoptosis triggered by mild testicular hyperthermia or by hormonal deprivation. In both models the induction of apoptosis is triggered by a MAPK14-mediated <span class="highlight" style="background-color:">pathway</span>. Activation of MAPK14, through changes in the ratio of BAX and BCL2 in the mitochondria, triggers intrinsic <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> and promotes germ cell apoptosis in response to a lack of hormonal stimulation [17]. The same <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is also the key <span class="highlight" style="background-color:">pathway</span> for heat-induced testicular germ cell apoptosis. However, unlike the hormone deprivation model, this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promotes germ cell apoptosis by provoking BCL2 phosphorylation, leading to its inactivation, thereby resulting in the perturbation of the BAX/BCL2 rheostat and the subsequent activation of the mitochondria-dependent death <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yue Jia, et al. Biol Reprod. 2009 April;80(4):771-780.</div><p class="links"><a class="dblinks" href="/pubmed/19109224" ref="ordinalpos=69&amp;ncbi_uid=2188222&amp;link_uid=19109224">Citation</a><a class="dblinks" href="/pmc/articles/PMC2804830/" ref="ordinalpos=69&amp;ncbi_uid=2188222&amp;link_uid=PMC2804830">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>70.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1413561/figure/F3/" ref="ordinalpos=70&amp;ncbi_uid=707089&amp;link_uid=PMC1413561" href="/pmc/articles/PMC1413561/"><img src="/pmc/articles/instance/1413561/bin/1471-2105-7-99-3.gif" src-large="/pmc/articles/instance/1413561/bin/1471-2105-7-99-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1413561/" ref="ordinalpos=70&amp;ncbi_uid=707089&amp;link_uid=PMC1413561" image-link="/pmc/articles/PMC1413561/figure/F3/" class="imagepopup">Figure 3.  From: Determination of strongly overlapping <span class="highlight" style="background-color:">signaling</span> activity from microarray data. </a></p><div class="supp"><p class="details"><div><b>Yeast MAPK <span class="highlight" style="background-color:">signaling</span> for mating and filamentation</b>. The strongly linked MAPK <span class="highlight" style="background-color:">signaling</span> pathways for mating and filamentation are shown schematically with black arrows indicating mating <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> and gray arrows showing filamentation <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. The mating <span class="highlight" style="background-color:">pathway</span> is initiated by binding to Ste2p or Ste3p receptors, while the causative molecular trigger for filamentation is unclear. The pathways share many components.</div></p></div><div class="aux"><div class="resc">Ghislain Bidaut, et al. BMC Bioinformatics. 2006;7:99-99.</div><p class="links"><a class="dblinks" href="/pubmed/16507110" ref="ordinalpos=70&amp;ncbi_uid=707089&amp;link_uid=16507110">Citation</a><a class="dblinks" href="/pmc/articles/PMC1413561/" ref="ordinalpos=70&amp;ncbi_uid=707089&amp;link_uid=PMC1413561">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>71.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4002811/figure/fig2/" ref="ordinalpos=71&amp;ncbi_uid=6509209&amp;link_uid=PMC4002811" href="/pmc/articles/PMC4002811/"><img src="/pmc/articles/instance/4002811/bin/cmi201160f2.gif" src-large="/pmc/articles/instance/4002811/bin/cmi201160f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4002811/" ref="ordinalpos=71&amp;ncbi_uid=6509209&amp;link_uid=PMC4002811" image-link="/pmc/articles/PMC4002811/figure/fig2/" class="imagepopup">Figure 2.  From: Peli: a family of signal-responsive E3 ubiquitin ligases mediating TLR <span class="highlight" style="background-color:">signaling</span> and T-cell tolerance. </a></p><div class="supp"><p class="details"><div>Peli1 mediates IKK activation by the TRIF-dependent TLR <span class="highlight" style="background-color:">pathway</span>. Based on the <span class="highlight" style="background-color:">signaling</span> adaptors, TLR <span class="highlight" style="background-color:">signaling</span> can be classified into two major pathways: the MyD88- and TRIF-dependent pathways. TLR3 is the only TLR that exclusively signals through TRIF, whereas TLR4 signals through both TRIF and MyD88. The rest of the TLRs, as well as IL-1R, signal through MyD88. An important mechanism of TLR <span class="highlight" style="background-color:">signaling</span> is K63 type ubiquitination of <span class="highlight" style="background-color:">signaling</span> adaptors, including TRAF6, RIP1 and TRAF3, which mediate activation of downstream <span class="highlight" style="background-color:">signaling</span> events. Activation of IKK/NF-B by the TRIF-dependent TLR <span class="highlight" style="background-color:">pathway</span>, but not that of the MyD88 <span class="highlight" style="background-color:">pathway</span>, depends on Peli1. Peli1 functions by mediating ubiquitination of RIP1 and, thereby, activating IKK and NF-B. The TRAF3-regulated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may crosstalk with the RIP1 <span class="highlight" style="background-color:">pathway</span> through induction of Peli1 gene expression and TBK1-/IKK-mediated Peli1 activation. IKK, IB kinase; Peli1, Pelle-interacting protein 1; RIP1, receptor-interacting protein 1; TCR, T-cell receptor; TLR, Toll-like receptor.</div></p></div><div class="aux"><div class="resc">Wei Jin, et al. Cell Mol Immunol. 2012 March;9(2):113-122.</div><p class="links"><a class="dblinks" href="/pubmed/22307041" ref="ordinalpos=71&amp;ncbi_uid=6509209&amp;link_uid=22307041">Citation</a><a class="dblinks" href="/pmc/articles/PMC4002811/" ref="ordinalpos=71&amp;ncbi_uid=6509209&amp;link_uid=PMC4002811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>72.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2697273/figure/F3/" ref="ordinalpos=72&amp;ncbi_uid=1831667&amp;link_uid=PMC2697273" href="/pmc/articles/PMC2697273/"><img src="/pmc/articles/instance/2697273/bin/nihms97740f3.gif" src-large="/pmc/articles/instance/2697273/bin/nihms97740f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2697273/" ref="ordinalpos=72&amp;ncbi_uid=1831667&amp;link_uid=PMC2697273" image-link="/pmc/articles/PMC2697273/figure/F3/" class="imagepopup">Figure 3.  From: A New Bioinformatics Approach to Analyze Gene Expressions and <span class="highlight" style="background-color:">Signaling</span> Pathways Reveals Unique Purine Gene Dysregulation Profiles that Distinguish between CD and UC. </a></p><div class="supp"><p class="details"><div>Canonical <span class="highlight" style="background-color:">pathway</span> analysis reveals novel <span class="highlight" style="background-color:">signaling</span> pathways associated with CD73 / NT5E or ADORA3 mRNA gene dysregulation that distinguish between CD and UC. (A, B) NT5E <span class="highlight" style="background-color:">pathway</span> analysis reveals that NT5E gene dysregulation is associated with acute phase response <span class="highlight" style="background-color:">signaling</span>, cAMP-mediated <span class="highlight" style="background-color:">signaling</span>, NRF2-mediated oxidative stress response and G-protein coupled receptor <span class="highlight" style="background-color:">signaling</span> (i.e. bars above threshold line for significant association). A distinct profile is observed in UC (B). ADORA3 <span class="highlight" style="background-color:">pathway</span> analysis reveals that ADORA3 gene dysregulation is associated with acute phase response <span class="highlight" style="background-color:">signaling</span>, cAMP-mediated <span class="highlight" style="background-color:">signaling</span>, G-protein coupled receptor <span class="highlight" style="background-color:">signaling</span>, IGF-1 <span class="highlight" style="background-color:">signaling</span> and IL-10 <span class="highlight" style="background-color:">signaling</span>. A distinct profile is observed in UC (D). Bars  The significance calculated for each function/Canonical <span class="highlight" style="background-color:">Pathway</span> is a measurement of the likelihood that the function is associated with the dataset by random chance. On the y-axis of the diagram, the significance is expressed as the minus log (10-based) of the p-values calculated from Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis and Fischer's exact test. Threshold set at 0.05 for p-value, in the chart it corresponds to log (0.05)). Note: the higher the bar, the lower the p-value and hence the more significant for the enrichment of the group/<span class="highlight" style="background-color:">pathway</span>/network. Ratio - The ratio between the number of genes in our list and the total number of genes in the defined functional group or <span class="highlight" style="background-color:">pathway</span> or network; curve just links all the data points for the ratio.</div></p></div><div class="aux"><div class="resc">Leszek Rybaczyk, et al. Inflamm Bowel Dis. ;15(7):971-984.</div><p class="links"><a class="dblinks" href="/pubmed/19253308" ref="ordinalpos=72&amp;ncbi_uid=1831667&amp;link_uid=19253308">Citation</a><a class="dblinks" href="/pmc/articles/PMC2697273/" ref="ordinalpos=72&amp;ncbi_uid=1831667&amp;link_uid=PMC2697273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>73.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2888582/figure/pone-0011240-g007/" ref="ordinalpos=73&amp;ncbi_uid=2334491&amp;link_uid=PMC2888582" href="/pmc/articles/PMC2888582/"><img src="/pmc/articles/instance/2888582/bin/pone.0011240.g007.gif" src-large="/pmc/articles/instance/2888582/bin/pone.0011240.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2888582/" ref="ordinalpos=73&amp;ncbi_uid=2334491&amp;link_uid=PMC2888582" image-link="/pmc/articles/PMC2888582/figure/pone-0011240-g007/" class="imagepopup">Figure 7. Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> generated by the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) software..  From: Proteomics-Based Systems Biology Modeling of Bovine Germinal Vesicle Stage Oocyte and Cumulus Cell Interaction. </a></p><div class="supp"><p class="details"><div>Integrin and actin cytoskeleton <span class="highlight" style="background-color:">signaling</span> pathways were the top two pathways associated with cell-to-cell <span class="highlight" style="background-color:">signaling</span>. Each node represents a protein; the proteins in shaded nodes in the <span class="highlight" style="background-color:">pathway</span> are identified/relate to an identified protein in the proteomic analysis. While proteins in clear nodes are part of the <span class="highlight" style="background-color:">pathway</span> but have not been identified in the GV oocyte or cumulus datasets. Proteins in red nodes were shown higher expression in cumulus cells compared to GV oocyte.</div></p></div><div class="aux"><div class="resc">Divyaswetha Peddinti, et al. PLoS One. 2010;5(6):e11240.</div><p class="links"><a class="dblinks" href="/pubmed/20574525" ref="ordinalpos=73&amp;ncbi_uid=2334491&amp;link_uid=20574525">Citation</a><a class="dblinks" href="/pmc/articles/PMC2888582/" ref="ordinalpos=73&amp;ncbi_uid=2334491&amp;link_uid=PMC2888582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>74.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2377155/figure/pcbi-1000086-g004/" ref="ordinalpos=74&amp;ncbi_uid=1358560&amp;link_uid=PMC2377155" href="/pmc/articles/PMC2377155/"><img src="/pmc/articles/instance/2377155/bin/pcbi.1000086.g004.gif" src-large="/pmc/articles/instance/2377155/bin/pcbi.1000086.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2377155/" ref="ordinalpos=74&amp;ncbi_uid=1358560&amp;link_uid=PMC2377155" image-link="/pmc/articles/PMC2377155/figure/pcbi-1000086-g004/" class="imagepopup">Figure 4. A representative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the prototypic integrated system..  From: Dynamic Analysis of Integrated <span class="highlight" style="background-color:">Signaling</span>, Metabolic, and Regulatory Networks. </a></p><div class="supp"><p class="details"><div>One of the pathways of the prototypic <span class="highlight" style="background-color:">signaling</span> network is illustrated. Here, solid black lines represent reactions that have non-zero fluxes, while dotted red lines represent reactions that have zero fluxes when the production of the activated transcription factor T<sub>1</sub>p is maximized as the <span class="highlight" style="background-color:">pathway</span> objective.</div></p></div><div class="aux"><div class="resc">Jong Min Lee, et al. PLoS Comput Biol. 2008 May;4(5):e1000086.</div><p class="links"><a class="dblinks" href="/pubmed/18483615" ref="ordinalpos=74&amp;ncbi_uid=1358560&amp;link_uid=18483615">Citation</a><a class="dblinks" href="/pmc/articles/PMC2377155/" ref="ordinalpos=74&amp;ncbi_uid=1358560&amp;link_uid=PMC2377155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>75.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3169577/figure/pone-0024152-g001/" ref="ordinalpos=75&amp;ncbi_uid=3224233&amp;link_uid=PMC3169577" href="/pmc/articles/PMC3169577/"><img src="/pmc/articles/instance/3169577/bin/pone.0024152.g001.gif" src-large="/pmc/articles/instance/3169577/bin/pone.0024152.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3169577/" ref="ordinalpos=75&amp;ncbi_uid=3224233&amp;link_uid=PMC3169577" image-link="/pmc/articles/PMC3169577/figure/pone-0024152-g001/" class="imagepopup">Figure 1. Basic overview of TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Evolution of the TGF-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Its Potential Role in the Ctenophore, Mnemiopsis leidyi. </a></p><div class="supp"><p class="details"><div>Binding of a ligand to a Type II receptor initiates <span class="highlight" style="background-color:">signaling</span>. The sequestering of a Type I receptor results in the activation of a Receptor-Smad (Smad1/5, Smad2/3). Together with the Co-Smad (Smad4), this complex enters the nucleus and activates the transcription of target genes. The <span class="highlight" style="background-color:">pathway</span> can be inhibited by extracelluar antagonists, or intracellularly via Inhibitor-Smad (Smad6/7) or the ubiquitin ligase SMURF.</div></p></div><div class="aux"><div class="resc">Kevin Pang, et al. PLoS One. 2011;6(9):e24152.</div><p class="links"><a class="dblinks" href="/pubmed/21931657" ref="ordinalpos=75&amp;ncbi_uid=3224233&amp;link_uid=21931657">Citation</a><a class="dblinks" href="/pmc/articles/PMC3169577/" ref="ordinalpos=75&amp;ncbi_uid=3224233&amp;link_uid=PMC3169577">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>76.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4309023/figure/bau126-F2/" ref="ordinalpos=76&amp;ncbi_uid=7212788&amp;link_uid=PMC4309023" href="/pmc/articles/PMC4309023/"><img src="/pmc/articles/instance/4309023/bin/bau126f2p.gif" src-large="/pmc/articles/instance/4309023/bin/bau126f2p.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4309023/" ref="ordinalpos=76&amp;ncbi_uid=7212788&amp;link_uid=PMC4309023" image-link="/pmc/articles/PMC4309023/figure/bau126-F2/" class="imagepopup">Figure 2. From: Comparison of human cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> databasesevolution, drawbacks and challenges. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> databases and the available <span class="highlight" style="background-color:">pathway</span> data types. This figure illustrates a matrix, whose Rows and Columns represent the number of different types of <span class="highlight" style="background-color:">signaling</span> pathways (<i>Y</i> axis) available in different databases (<i>X</i> axis), respectively. Color legends are used to represent the presence or absence of different types of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> data (total 19) available in the selected 24 databases. The numbers in first bracket indicate the number of the databases, which contain a particular type of <span class="highlight" style="background-color:">pathway</span> data (represented in row wise), whereas the numbers in square bracket indicate the total number of different types of <span class="highlight" style="background-color:">pathway</span> data present in a particular database (represented in column wise).</div></p></div><div class="aux"><div class="resc">Saikat Chowdhury, et al. Database (Oxford). 2015;2015:bau126.</div><p class="links"><a class="dblinks" href="/pubmed/25632107" ref="ordinalpos=76&amp;ncbi_uid=7212788&amp;link_uid=25632107">Citation</a><a class="dblinks" href="/pmc/articles/PMC4309023/" ref="ordinalpos=76&amp;ncbi_uid=7212788&amp;link_uid=PMC4309023">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>77.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g006/" ref="ordinalpos=77&amp;ncbi_uid=3758043&amp;link_uid=PMC3256223" href="/pmc/articles/PMC3256223/"><img src="/pmc/articles/instance/3256223/bin/pone.0030140.g006.gif" src-large="/pmc/articles/instance/3256223/bin/pone.0030140.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3256223/" ref="ordinalpos=77&amp;ncbi_uid=3758043&amp;link_uid=PMC3256223" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g006/" class="imagepopup">Figure 6. Histogram of miRNA/target relationships..  From: Modeling of miRNA and Drug Action in the EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A</b>: The top ranked 15 miRNAs of Table 2 correlated with the amount of their corresponding targets in the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Table 1). <b>B</b>: The 15 miRNAs correlated with amount of their corresponding targets in EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The anti-miRNAs of these miRNAs have much less effect on this <span class="highlight" style="background-color:">pathway</span> (Fig. 5A and B).</div></p></div><div class="aux"><div class="resc">Jian Li, et al. PLoS One. 2012;7(1):e30140.</div><p class="links"><a class="dblinks" href="/pubmed/22253908" ref="ordinalpos=77&amp;ncbi_uid=3758043&amp;link_uid=22253908">Citation</a><a class="dblinks" href="/pmc/articles/PMC3256223/" ref="ordinalpos=77&amp;ncbi_uid=3758043&amp;link_uid=PMC3256223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>78.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3104248/figure/F1/" ref="ordinalpos=78&amp;ncbi_uid=3004999&amp;link_uid=PMC3104248" href="/pmc/articles/PMC3104248/"><img src="/pmc/articles/instance/3104248/bin/emm-43-231-g001.gif" src-large="/pmc/articles/instance/3104248/bin/emm-43-231-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3104248/" ref="ordinalpos=78&amp;ncbi_uid=3004999&amp;link_uid=PMC3104248" image-link="/pmc/articles/PMC3104248/figure/F1/" class="imagepopup">Figure 1.  From: Frontier of Epilepsy Research - mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Overview of mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activation and inhibition of <span class="highlight" style="background-color:">signaling</span> molecules by phosphorylation are shown in red and blue respectively.</div></p></div><div class="aux"><div class="resc">Chang-Hoon Cho. Exp Mol Med. 2011 May 31;43(5):231-274.</div><p class="links"><a class="dblinks" href="/pubmed/21467839" ref="ordinalpos=78&amp;ncbi_uid=3004999&amp;link_uid=21467839">Citation</a><a class="dblinks" href="/pmc/articles/PMC3104248/" ref="ordinalpos=78&amp;ncbi_uid=3004999&amp;link_uid=PMC3104248">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>79.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3384069/figure/fig1/" ref="ordinalpos=79&amp;ncbi_uid=4668480&amp;link_uid=PMC3384069" href="/pmc/articles/PMC3384069/"><img src="/pmc/articles/instance/3384069/bin/carcinbgr305f01_4c.gif" src-large="/pmc/articles/instance/3384069/bin/carcinbgr305f01_4c.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3384069/" ref="ordinalpos=79&amp;ncbi_uid=4668480&amp;link_uid=PMC3384069" image-link="/pmc/articles/PMC3384069/figure/fig1/" class="imagepopup">Fig. 1. From: Modulation of Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by bioactive food components. </a></p><div class="supp"><p class="details"><div>Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Rohinton S. Tarapore, et al. Carcinogenesis. 2012 March;33(3):483-491.</div><p class="links"><a class="dblinks" href="/pubmed/22198211" ref="ordinalpos=79&amp;ncbi_uid=4668480&amp;link_uid=22198211">Citation</a><a class="dblinks" href="/pmc/articles/PMC3384069/" ref="ordinalpos=79&amp;ncbi_uid=4668480&amp;link_uid=PMC3384069">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>80.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2930805/figure/fig1/" ref="ordinalpos=80&amp;ncbi_uid=3184466&amp;link_uid=PMC2930805" href="/pmc/articles/PMC2930805/"><img src="/pmc/articles/instance/2930805/bin/carcinbgq142f01_ht.gif" src-large="/pmc/articles/instance/2930805/bin/carcinbgq142f01_ht.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2930805/" ref="ordinalpos=80&amp;ncbi_uid=3184466&amp;link_uid=PMC2930805" image-link="/pmc/articles/PMC2930805/figure/fig1/" class="imagepopup">Fig. 1. From: Genetic variation in a metabolic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and colon and rectal cancer risk: mTOR, PTEN, STK11, RPKAA1, PRKAG2, TSC1, TSC2, PI3K and Akt1. </a></p><div class="supp"><p class="details"><div>Metabolic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of candidate <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Martha L. Slattery, et al. Carcinogenesis. 2010 September;31(9):1604-1611.</div><p class="links"><a class="dblinks" href="/pubmed/20622004" ref="ordinalpos=80&amp;ncbi_uid=3184466&amp;link_uid=20622004">Citation</a><a class="dblinks" href="/pmc/articles/PMC2930805/" ref="ordinalpos=80&amp;ncbi_uid=3184466&amp;link_uid=PMC2930805">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>81.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2822851/figure/pone-0009256-g002/" ref="ordinalpos=81&amp;ncbi_uid=2246448&amp;link_uid=PMC2822851" href="/pmc/articles/PMC2822851/"><img src="/pmc/articles/instance/2822851/bin/pone.0009256.g002.gif" src-large="/pmc/articles/instance/2822851/bin/pone.0009256.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2822851/" ref="ordinalpos=81&amp;ncbi_uid=2246448&amp;link_uid=PMC2822851" image-link="/pmc/articles/PMC2822851/figure/pone-0009256-g002/" class="imagepopup">Figure 2. Atlas of <span class="highlight" style="background-color:">Signaling</span>..  From: Atlas of <span class="highlight" style="background-color:">Signaling</span> for Interpretation of Microarray Experiments. </a></p><div class="supp"><p class="details"><div>The major known <span class="highlight" style="background-color:">signaling</span> cascades were combined in one <span class="highlight" style="background-color:">pathway</span> diagram to provide birds-eye view of cellular <span class="highlight" style="background-color:">signaling</span>. <span class="highlight" style="background-color:">Signaling</span> functional groups are shown as orange hexagons; secondary messengers/small moleculesas green circles, and cellular processesas yellow rectangles.</div></p></div><div class="aux"><div class="resc">Ekaterina Kotelnikova, et al. PLoS One. 2010;5(2):e9256.</div><p class="links"><a class="dblinks" href="/pubmed/20174649" ref="ordinalpos=81&amp;ncbi_uid=2246448&amp;link_uid=20174649">Citation</a><a class="dblinks" href="/pmc/articles/PMC2822851/" ref="ordinalpos=81&amp;ncbi_uid=2246448&amp;link_uid=PMC2822851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>82.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g006/" ref="ordinalpos=82&amp;ncbi_uid=6232465&amp;link_uid=PMC4041745" href="/pmc/articles/PMC4041745/"><img src="/pmc/articles/instance/4041745/bin/pone.0098229.g006.gif" src-large="/pmc/articles/instance/4041745/bin/pone.0098229.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041745/" ref="ordinalpos=82&amp;ncbi_uid=6232465&amp;link_uid=PMC4041745" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g006/" class="imagepopup">Figure 6. GeneGo <span class="highlight" style="background-color:">pathway</span> Neurophysiological process_Receptor-mediated axon growth repulsion..  From: Systematic <span class="highlight" style="background-color:">Pathway</span> Enrichment Analysis of a Genome-Wide Association Study on Breast Cancer Survival Reveals an Influence of Genes Involved in Cell Adhesion and Calcium <span class="highlight" style="background-color:">Signaling</span> on the Patients' Clinical Outcome. </a></p><div class="supp"><p class="details"><div>Barometers: 1=0.01 gene list; 2=Oncotype DX. red=Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Andrea Woltmann, et al. PLoS One. 2014;9(6):e98229.</div><p class="links"><a class="dblinks" href="/pubmed/24886783" ref="ordinalpos=82&amp;ncbi_uid=6232465&amp;link_uid=24886783">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041745/" ref="ordinalpos=82&amp;ncbi_uid=6232465&amp;link_uid=PMC4041745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>83.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4373796/figure/pone.0121266.g006/" ref="ordinalpos=83&amp;ncbi_uid=7335925&amp;link_uid=PMC4373796" href="/pmc/articles/PMC4373796/"><img src="/pmc/articles/instance/4373796/bin/pone.0121266.g006.gif" src-large="/pmc/articles/instance/4373796/bin/pone.0121266.g006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4373796/" ref="ordinalpos=83&amp;ncbi_uid=7335925&amp;link_uid=PMC4373796" image-link="/pmc/articles/PMC4373796/figure/pone.0121266.g006/" class="imagepopup">Fig 6. Diagram showing changes in the osteocytes after activation of the HIF <span class="highlight" style="background-color:">pathway</span> and modulation of WNT/-catenin <span class="highlight" style="background-color:">signaling</span>..  From: Activation of HIFa <span class="highlight" style="background-color:">Pathway</span> in Mature Osteoblasts Disrupts the Integrity of the Osteocyte/Canalicular Network. </a></p><div class="supp"><p class="details"><div>(A) WNT/-catenin <span class="highlight" style="background-color:">signaling</span> induces the proliferation of mesenchymal/osteoblastic progenitors and enhances their differentiation into the osteoblastic lineage. This process is inhibited by sclerostin secreted through the canaliculi under normal conditions. (B) After activation, the HIF <span class="highlight" style="background-color:">pathway</span> reduces the sclerostin from calcified bone to the soft bone marrow, further activating WNT/-catenin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Gui-lai Zuo, et al. PLoS One. 2015;10(3):e0121266.</div><p class="links"><a class="dblinks" href="/pubmed/25806791" ref="ordinalpos=83&amp;ncbi_uid=7335925&amp;link_uid=25806791">Citation</a><a class="dblinks" href="/pmc/articles/PMC4373796/" ref="ordinalpos=83&amp;ncbi_uid=7335925&amp;link_uid=PMC4373796">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>84.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4098619/figure/fig1/" ref="ordinalpos=84&amp;ncbi_uid=6408598&amp;link_uid=PMC4098619" href="/pmc/articles/PMC4098619/"><img src="/pmc/articles/instance/4098619/bin/DM2014-291419.001.gif" src-large="/pmc/articles/instance/4098619/bin/DM2014-291419.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4098619/" ref="ordinalpos=84&amp;ncbi_uid=6408598&amp;link_uid=PMC4098619" image-link="/pmc/articles/PMC4098619/figure/fig1/" class="imagepopup">Figure 1.  From: A New Susceptibility Locus for Myocardial Infarction, Hypertension, Type 2 Diabetes Mellitus, and Dyslipidemia on Chromosome 12q24. </a></p><div class="supp"><p class="details"><div>Schematic diagram representation of some of the pathways regulated by HNF1a. AES: androgen receptor nuclear <span class="highlight" style="background-color:">signaling</span>, Wnt signalling <span class="highlight" style="background-color:">pathway</span>. CREBBP: Notch <span class="highlight" style="background-color:">signaling</span> pathways, <span class="highlight" style="background-color:">signaling</span> by Notch1. CTNNB1: androgen receptor nuclear <span class="highlight" style="background-color:">signaling</span>, Wnt signalling <span class="highlight" style="background-color:">pathway</span>. DYRK1B: glucose/energy metabolism. HDAC1: Notch <span class="highlight" style="background-color:">signaling</span> pathways, <span class="highlight" style="background-color:">signaling</span> by Notch1. HNF1a: maturity onset diabetes of the young (MODY), type II diabetes mellitus, presenilin action in Notch and Wnt <span class="highlight" style="background-color:">signaling</span>, and metabolism. PCBD1: metabolism. PCBD2: metabolism. PDX1: maturity onset diabetes of the young, type II diabetes mellitus. SLC2A2: maturity onset diabetes of the young, type II diabetes mellitus. TLE1: Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, <span class="highlight" style="background-color:">signaling</span> by Notch1, and presenilin action in Notch and Wnt <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Salma M. Wakil, et al. Dis Markers. 2014;2014:291419.</div><p class="links"><a class="dblinks" href="/pubmed/25057215" ref="ordinalpos=84&amp;ncbi_uid=6408598&amp;link_uid=25057215">Citation</a><a class="dblinks" href="/pmc/articles/PMC4098619/" ref="ordinalpos=84&amp;ncbi_uid=6408598&amp;link_uid=PMC4098619">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>85.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4066200/figure/F2/" ref="ordinalpos=85&amp;ncbi_uid=6349044&amp;link_uid=PMC4066200" href="/pmc/articles/PMC4066200/"><img src="/pmc/articles/instance/4066200/bin/fimmu-05-00277-g002.gif" src-large="/pmc/articles/instance/4066200/bin/fimmu-05-00277-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4066200/" ref="ordinalpos=85&amp;ncbi_uid=6349044&amp;link_uid=PMC4066200" image-link="/pmc/articles/PMC4066200/figure/F2/" class="imagepopup">Figure 2.  From: Biased and G Protein-Independent <span class="highlight" style="background-color:">Signaling</span> of Chemokine Receptors. </a></p><div class="supp"><p class="details"><div><b>Overview of different variations of biased <span class="highlight" style="background-color:">signaling</span></b>. Biased <span class="highlight" style="background-color:">signaling</span> describes a situation in which a receptor preferentially activates one <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> over another. (Left) Ligand bias is used to differentiate between two ligands acting on the same receptor, where ligand A favorably activates <span class="highlight" style="background-color:">pathway</span> 1, whereas ligand B activates <span class="highlight" style="background-color:">pathway</span> 2. (Center) In receptor bias, the same ligand binds to two different receptors, and activates <span class="highlight" style="background-color:">pathway</span> 1 via receptor A, but <span class="highlight" style="background-color:">pathway</span> 2 through receptor B. (Right) In tissue (or cell) bias, the same ligand: receptor-complex is activated in two different tissues or cell types (or different species), and in tissue A <span class="highlight" style="background-color:">pathway</span> 1 is preferentially activated, whereas <span class="highlight" style="background-color:">pathway</span> 2 is more likely to be activated in tissue B.</div></p></div><div class="aux"><div class="resc">Anne Steen, et al. Front Immunol. 2014;5:277.</div><p class="links"><a class="dblinks" href="/pubmed/25002861" ref="ordinalpos=85&amp;ncbi_uid=6349044&amp;link_uid=25002861">Citation</a><a class="dblinks" href="/pmc/articles/PMC4066200/" ref="ordinalpos=85&amp;ncbi_uid=6349044&amp;link_uid=PMC4066200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>86.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3857640/figure/F5/" ref="ordinalpos=86&amp;ncbi_uid=5584532&amp;link_uid=PMC3857640" href="/pmc/articles/PMC3857640/"><img src="/pmc/articles/instance/3857640/bin/nihms460775f5.gif" src-large="/pmc/articles/instance/3857640/bin/nihms460775f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3857640/" ref="ordinalpos=86&amp;ncbi_uid=5584532&amp;link_uid=PMC3857640" image-link="/pmc/articles/PMC3857640/figure/F5/" class="imagepopup">Figure 5. Schematic presentation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in pre-osteoblast proliferation induced by MOS-J/POS-1 CMs.  From: Conditioned media from mouse osteosarcoma cells promote MC3T3-E1 cell proliferation using JAKs and PI3-K/Akt signal crosstalk. </a></p><div class="supp"><p class="details"><div>MOS-J/POS-1 conditioned media (CMs) induced novel JAKs (JAK1 and JAK2) and PI3-K/Akt signal crosstalk in osteoblastic cells. The pre-osteoblast pro-proliferating signal induced by MOS-J/POS-1 CMs was transmitted <i>via</i> JAKs/PI3-K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span>. Soluble factor(s): Pro-proliferating factor(s) of pre-osteoblast cell produced by osteosarcoma cells. Solid line arrow: The responsible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for pre-osteoblast proliferation induced by MOS-J/POS-1 CMs. Dotted line arrow: MOS-J/POS-1 CMs-induced but non-responsible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for pre-osteoblast proliferation.</div></p></div><div class="aux"><div class="resc">Kanji MORI, et al. Cancer Sci. ;99(11):10.1111/j.1349-7006.2008.00919.x.</div><p class="links"><a class="dblinks" href="/pubmed/18957057" ref="ordinalpos=86&amp;ncbi_uid=5584532&amp;link_uid=18957057">Citation</a><a class="dblinks" href="/pmc/articles/PMC3857640/" ref="ordinalpos=86&amp;ncbi_uid=5584532&amp;link_uid=PMC3857640">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>87.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3325001/figure/f2-ott-5-047/" ref="ordinalpos=87&amp;ncbi_uid=3898412&amp;link_uid=PMC3325001" href="/pmc/articles/PMC3325001/"><img src="/pmc/articles/instance/3325001/bin/ott-5-047f2.gif" src-large="/pmc/articles/instance/3325001/bin/ott-5-047f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3325001/" ref="ordinalpos=87&amp;ncbi_uid=3898412&amp;link_uid=PMC3325001" image-link="/pmc/articles/PMC3325001/figure/f2-ott-5-047/" class="imagepopup">Figure 2.  From: Hedgehog <span class="highlight" style="background-color:">pathway</span> as a drug target: Smoothened inhibitors in development. </a></p><div class="supp"><p class="details"><div>Modes of Hh <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. (<b>A</b>) Type I Hh <span class="highlight" style="background-color:">signaling</span> is activated by specific mutations within <span class="highlight" style="background-color:">pathway</span> genes within tumor cells, resulting in ligand-independent constitutive activation. (<b>B</b>) Type II Hh <span class="highlight" style="background-color:">signaling</span> results from autocrine <span class="highlight" style="background-color:">signaling</span> from tumor cell to tumor cell. (<b>C</b>) Type IIIa activation results from secretion of Hh ligand by tumor cells, resulting in <span class="highlight" style="background-color:">pathway</span> activation in surrounding tumor stroma. (<b>D</b>) Type IIIb Hh <span class="highlight" style="background-color:">signaling</span> results from Hh ligand secretion by tumor stroma, resulting in activation of the <span class="highlight" style="background-color:">pathway</span> within tumor cells themselves.<br /><b>Abbreviation:</b> Hh, Hedgehog.</div></p></div><div class="aux"><div class="resc">Tara L Lin, et al. Onco Targets Ther. 2012;5:47-58.</div><p class="links"><a class="dblinks" href="/pubmed/22500124" ref="ordinalpos=87&amp;ncbi_uid=3898412&amp;link_uid=22500124">Citation</a><a class="dblinks" href="/pmc/articles/PMC3325001/" ref="ordinalpos=87&amp;ncbi_uid=3898412&amp;link_uid=PMC3325001">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>88.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3082570/figure/pone-0019309-g001/" ref="ordinalpos=88&amp;ncbi_uid=2930596&amp;link_uid=PMC3082570" href="/pmc/articles/PMC3082570/"><img src="/pmc/articles/instance/3082570/bin/pone.0019309.g001.gif" src-large="/pmc/articles/instance/3082570/bin/pone.0019309.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3082570/" ref="ordinalpos=88&amp;ncbi_uid=2930596&amp;link_uid=PMC3082570" image-link="/pmc/articles/PMC3082570/figure/pone-0019309-g001/" class="imagepopup">Figure 1. Src and some of the <span class="highlight" style="background-color:">signaling</span> pathways downstream of Src..  From: Simultaneous siRNA Targeting of Src and Downstream <span class="highlight" style="background-color:">Signaling</span> Molecules Inhibit Tumor Formation and Metastasis of a Human Model Breast Cancer Cell Line. </a></p><div class="supp"><p class="details"><div>Src regulates several downstream <span class="highlight" style="background-color:">signaling</span> pathways including the STAT3 <span class="highlight" style="background-color:">pathway</span>, the PI3K <span class="highlight" style="background-color:">pathway</span>, and the MAPK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jeffrey D. Bjorge, et al. PLoS One. 2011;6(4):e19309.</div><p class="links"><a class="dblinks" href="/pubmed/21541295" ref="ordinalpos=88&amp;ncbi_uid=2930596&amp;link_uid=21541295">Citation</a><a class="dblinks" href="/pmc/articles/PMC3082570/" ref="ordinalpos=88&amp;ncbi_uid=2930596&amp;link_uid=PMC3082570">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>89.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1091709/figure/f8/" ref="ordinalpos=89&amp;ncbi_uid=424619&amp;link_uid=PMC1091709" href="/pmc/articles/PMC1091709/"><img src="/pmc/articles/instance/1091709/bin/zjv0100562260008.gif" src-large="/pmc/articles/instance/1091709/bin/zjv0100562260008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1091709/" ref="ordinalpos=89&amp;ncbi_uid=424619&amp;link_uid=PMC1091709" image-link="/pmc/articles/PMC1091709/figure/f8/" class="imagepopup">FIG. 8. From: Nuclear Localization of the Nipah Virus W Protein Allows for Inhibition of both Virus- and Toll-Like Receptor 3-Triggered <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Illustration of the virus and TLR3 activated <span class="highlight" style="background-color:">signaling</span> pathways, and the points at which the Nipah virus V and W proteins are predicted to exert their inhibitory activity. Stimulation of TLR3 by dsRNA activates a <span class="highlight" style="background-color:">signaling</span> cascade that is mediated by the adapter, TRIF, and results in activation of the kinase TBK-1. TBK-1 phosphorylates the transcription factor, IRF3, leading to activation of IRF3-responsive promoters. The Nipah virus W protein targets a nuclear component of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that results in the loss of phosphorylated IRF3. Virus infection also results in the activation of IRF3-responsive promoters, and the nuclear portion of this <span class="highlight" style="background-color:">pathway</span> is probably shared with that of the TLR3 <span class="highlight" style="background-color:">pathway</span>, so the W protein is also able to inhibit virus-activated <span class="highlight" style="background-color:">signaling</span>. The upstream <span class="highlight" style="background-color:">signaling</span> events in the virus <span class="highlight" style="background-color:">pathway</span> are distinct from those in the TLR3 <span class="highlight" style="background-color:">pathway</span> in that <span class="highlight" style="background-color:">signaling</span> is mediated predominantly by IKK (thick arrow), although a minor contribution of TBK-1 cannot be excluded (broken arrow). The Nipah virus V protein is only able to prevent gene activation in response to IKK and therefore is restricted to blocking the virus <span class="highlight" style="background-color:">pathway</span>, presumably downstream of IKK.</div></p></div><div class="aux"><div class="resc">Megan L. Shaw, et al. J Virol. 2005 May;79(10):6078-6088.</div><p class="links"><a class="dblinks" href="/pubmed/15857993" ref="ordinalpos=89&amp;ncbi_uid=424619&amp;link_uid=15857993">Citation</a><a class="dblinks" href="/pmc/articles/PMC1091709/" ref="ordinalpos=89&amp;ncbi_uid=424619&amp;link_uid=PMC1091709">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>90.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g002/" ref="ordinalpos=90&amp;ncbi_uid=2941431&amp;link_uid=PMC3086880" href="/pmc/articles/PMC3086880/"><img src="/pmc/articles/instance/3086880/bin/pone.0019240.g002.gif" src-large="/pmc/articles/instance/3086880/bin/pone.0019240.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3086880/" ref="ordinalpos=90&amp;ncbi_uid=2941431&amp;link_uid=PMC3086880" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g002/" class="imagepopup">Figure 2. Statistical analysis of signalogs..  From: Signalogs: Orthology-Based Identification of Novel <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Components in Three Metazoans. </a></p><div class="supp"><p class="details"><div>The number of predicted novel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components (signalogs) in each <span class="highlight" style="background-color:">pathway</span> for the three species examined. The total number of signalogs in each species is shown in parentheses. Although we predicted a total of 253 signalogs, this statistical comparison contains 301 novel <span class="highlight" style="background-color:">pathway</span> components, because 39 signalog proteins were assigned to more than one <span class="highlight" style="background-color:">pathway</span> (these 39 had a total of 87 <span class="highlight" style="background-color:">pathway</span> annotations).</div></p></div><div class="aux"><div class="resc">Tams Korcsmros, et al. PLoS One. 2011;6(5):e19240.</div><p class="links"><a class="dblinks" href="/pubmed/21559328" ref="ordinalpos=90&amp;ncbi_uid=2941431&amp;link_uid=21559328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3086880/" ref="ordinalpos=90&amp;ncbi_uid=2941431&amp;link_uid=PMC3086880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>91.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4215754/figure/f5/" ref="ordinalpos=91&amp;ncbi_uid=6812503&amp;link_uid=PMC4215754" href="/pmc/articles/PMC4215754/"><img src="/pmc/articles/instance/4215819/bin/f1000research-3-5389-g0004.gif" src-large="/pmc/articles/instance/4215819/bin/f1000research-3-5389-g0004.jpg" alt="Figure 5. " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4215754/" ref="ordinalpos=91&amp;ncbi_uid=6812503&amp;link_uid=PMC4215754" image-link="/pmc/articles/PMC4215754/figure/f5/" class="imagepopup">Figure 5. PSA score ratios of Calcium <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> computed with simulated data..  From: CyKEGGParser: tailoring KEGG pathways to fit into systems biology analysis workflows. </a></p><div class="supp"><p class="details"><div>Target nodes are: M-MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, P-Phosphatidylinositol <span class="highlight" style="background-color:">signaling</span> system, A-Apoptosis. Tissues are: CD14_mo-CD14 monocytes, Adip-Adipocytes, Card-Cardiac myocytes.</div></p></div><div class="aux"><div class="resc">Lilit Nersisyan, et al. F1000Res. 2014;3:145.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=91&amp;ncbi_uid=6812503&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4215754/" ref="ordinalpos=91&amp;ncbi_uid=6812503&amp;link_uid=PMC4215754">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>92.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2844640/figure/F1/" ref="ordinalpos=92&amp;ncbi_uid=2282239&amp;link_uid=PMC2844640" href="/pmc/articles/PMC2844640/"><img src="/pmc/articles/instance/2844640/bin/nihms-183419-f0001.gif" src-large="/pmc/articles/instance/2844640/bin/nihms-183419-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2844640/" ref="ordinalpos=92&amp;ncbi_uid=2282239&amp;link_uid=PMC2844640" image-link="/pmc/articles/PMC2844640/figure/F1/" class="imagepopup">Figure 1. Dual effects of TGF- <span class="highlight" style="background-color:">signaling</span> on the growth of cells and impact of radiation.  From: Role of Radiation-induced TGF-beta <span class="highlight" style="background-color:">Signaling</span> in Cancer Therapy. </a></p><div class="supp"><p class="details"><div>Classical canonical TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing the dual effects of this <span class="highlight" style="background-color:">signaling</span> on the cell growth. Normally this <span class="highlight" style="background-color:">pathway</span> induces cell growth suppressive effects, however, during the process of tumorigenesis due to several factors TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induces proliferation, invasiveness, angiogenesis, metastasis and immune suppression. In this figure, based on the findings from our group, radiation can potently induce TGF- and this TGF- can be activated to exert negative growth effects by inducing the expression of cdk inhibitors such as p21 and p16. It is also shown here that the inhibitory Smad7 can be induced by Smad4 that can in turn inhibit the TGF- <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Horatiu C. Dancea, et al. Mol Cell Pharmacol. ;1(1):44-56.</div><p class="links"><a class="dblinks" href="/pubmed/20336170" ref="ordinalpos=92&amp;ncbi_uid=2282239&amp;link_uid=20336170">Citation</a><a class="dblinks" href="/pmc/articles/PMC2844640/" ref="ordinalpos=92&amp;ncbi_uid=2282239&amp;link_uid=PMC2844640">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>93.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694555/figure/F2/" ref="ordinalpos=93&amp;ncbi_uid=1936005&amp;link_uid=PMC2694555" href="/pmc/articles/PMC2694555/"><img src="/pmc/articles/instance/2694555/bin/CG-9-349_F2.gif" src-large="/pmc/articles/instance/2694555/bin/CG-9-349_F2.jpg" alt="Fig. (2)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694555/" ref="ordinalpos=93&amp;ncbi_uid=1936005&amp;link_uid=PMC2694555" image-link="/pmc/articles/PMC2694555/figure/F2/" class="imagepopup">Fig. (2).  From: Can Systems Biology Understand <span class="highlight" style="background-color:">Pathway</span> Activation? Gene Expression Signatures as Surrogate Markers for Understanding the Complexity of <span class="highlight" style="background-color:">Pathway</span> Activation. </a></p><div class="supp"><p class="details"><div>Expected application of <span class="highlight" style="background-color:">pathway</span> signatures to personalized medicine. In many diseases including cancer, the disorder is caused by deregulation of cellular <span class="highlight" style="background-color:">signaling</span>. <span class="highlight" style="background-color:">Pathway</span> signatures can be a novel tool to assess <span class="highlight" style="background-color:">signaling</span> statuses. The <span class="highlight" style="background-color:">pathway</span> activation/inactivation status in each patient will be monitored by the expression profile and a set of <span class="highlight" style="background-color:">pathway</span> signatures. Identification of the causative <span class="highlight" style="background-color:">pathway</span> allows us to select the "right" drug to inhibit the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hiraku Itadani, et al. Curr Genomics. 2008 August;9(5):349-360.</div><p class="links"><a class="dblinks" href="/pubmed/19517027" ref="ordinalpos=93&amp;ncbi_uid=1936005&amp;link_uid=19517027">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694555/" ref="ordinalpos=93&amp;ncbi_uid=1936005&amp;link_uid=PMC2694555">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>94.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4384220/figure/f1/" ref="ordinalpos=94&amp;ncbi_uid=7414006&amp;link_uid=PMC4384220" href="/pmc/articles/PMC4384220/"><img src="/pmc/articles/instance/4384220/bin/tlcr-04-02-165-f1.gif" src-large="/pmc/articles/instance/4384220/bin/tlcr-04-02-165-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4384220/" ref="ordinalpos=94&amp;ncbi_uid=7414006&amp;link_uid=PMC4384220" image-link="/pmc/articles/PMC4384220/figure/f1/" class="imagepopup">Figure 1.  From: Phosphatidylinositol 3-kinase-AKT-mammalian target of rapamycin (PI3K-Akt-mTOR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in non-small cell lung cancer. </a></p><div class="supp"><p class="details"><div>PI3K-Akt-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Po Yee Yip. Transl Lung Cancer Res. 2015 April;4(2):165-176.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=94&amp;ncbi_uid=7414006&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4384220/" ref="ordinalpos=94&amp;ncbi_uid=7414006&amp;link_uid=PMC4384220">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>95.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3349963/figure/F1/" ref="ordinalpos=95&amp;ncbi_uid=4003392&amp;link_uid=PMC3349963" href="/pmc/articles/PMC3349963/"><img src="/pmc/articles/instance/3349963/bin/fimmu-03-00116-g001.gif" src-large="/pmc/articles/instance/3349963/bin/fimmu-03-00116-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3349963/" ref="ordinalpos=95&amp;ncbi_uid=4003392&amp;link_uid=PMC3349963" image-link="/pmc/articles/PMC3349963/figure/F1/" class="imagepopup">Figure 1.  From: IL-13 <span class="highlight" style="background-color:">Signaling</span> in Liver Fibrogenesis. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Canonical and non-canonical IL-13 <span class="highlight" style="background-color:">signaling</span> in cells. IL-13 uses JAK/STAT6 <span class="highlight" style="background-color:">pathway</span> to transduce its canonical <span class="highlight" style="background-color:">signaling</span> in cells. In addition, IL-13 activates non-canonical STAT3, Erk1/2, etc., pathways in some settings. <b>(B)</b> Non-canonical <span class="highlight" style="background-color:">signaling</span> pathways of IL-13 play a critical role in liver fibrosis. IL-13 interacts with TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to induce CTGF expression in HSCs via Erk1/2 <span class="highlight" style="background-color:">pathway</span> (Liu et al., 2011).</div></p></div><div class="aux"><div class="resc">Yan Liu, et al. Front Immunol. 2012;3:116.</div><p class="links"><a class="dblinks" href="/pubmed/22593760" ref="ordinalpos=95&amp;ncbi_uid=4003392&amp;link_uid=22593760">Citation</a><a class="dblinks" href="/pmc/articles/PMC3349963/" ref="ordinalpos=95&amp;ncbi_uid=4003392&amp;link_uid=PMC3349963">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>96.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3837566/figure/F2/" ref="ordinalpos=96&amp;ncbi_uid=5511217&amp;link_uid=PMC3837566" href="/pmc/articles/PMC3837566/"><img src="/pmc/articles/instance/3837566/bin/nihms377464f2.gif" src-large="/pmc/articles/instance/3837566/bin/nihms377464f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3837566/" ref="ordinalpos=96&amp;ncbi_uid=5511217&amp;link_uid=PMC3837566" image-link="/pmc/articles/PMC3837566/figure/F2/" class="imagepopup">Figure 2. Targeting the BRAF <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> in melanoma.  From: Targeted inhibition of BRAF kinase: opportunities and challenges for therapeutics in melanoma. </a></p><div class="supp"><p class="details"><div>BRAF-specific inhibitors block the RAS-RAF-MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in BRAF<sup>V600E</sup> mutant cells. However, tumor cells acquire resistance to this therapy during the course of treatment. Several mechanisms that involve reactivation of ERK <span class="highlight" style="background-color:">signaling</span> through amplification or mutation of proteins in the RAS-RAF-MEK-ERK <span class="highlight" style="background-color:">pathway</span> (i.e. NRAS, BRAF, CRAF and COT1) have been recently described for the development of acquired resistance. In addition, hyper-activation of the PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (i.e. through upregulation of IGF-1R or PDGFR) can also be involved in resistance by promoting survival or proliferation. BRAF<sup>V600E</sup> suppresses the LKB1 tumor suppressor and AMPK through the BRAF-MEK-ERK <span class="highlight" style="background-color:">signaling</span> cascade, and this regulation is critical for melanoma cell proliferation.</div></p></div><div class="aux"><div class="resc">Rolando Prez-Lorenzo, et al. Biosci Rep. ;32(1):10.1042/BSR20110068.</div><p class="links"><a class="dblinks" href="/pubmed/21981139" ref="ordinalpos=96&amp;ncbi_uid=5511217&amp;link_uid=21981139">Citation</a><a class="dblinks" href="/pmc/articles/PMC3837566/" ref="ordinalpos=96&amp;ncbi_uid=5511217&amp;link_uid=PMC3837566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>97.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4309023/figure/bau126-F4/" ref="ordinalpos=97&amp;ncbi_uid=7212786&amp;link_uid=PMC4309023" href="/pmc/articles/PMC4309023/"><img src="/pmc/articles/instance/4309023/bin/bau126f4p.gif" src-large="/pmc/articles/instance/4309023/bin/bau126f4p.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4309023/" ref="ordinalpos=97&amp;ncbi_uid=7212786&amp;link_uid=PMC4309023" image-link="/pmc/articles/PMC4309023/figure/bau126-F4/" class="imagepopup">Figure 4. From: Comparison of human cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> databasesevolution, drawbacks and challenges. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> data heterogeneity across different databases. (<b>A</b>) and (<b>B</b>) show the comparison of the number of molecular species and interactions of Hedgehog and Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> across different databases, respectively. The names of the databases are presented along the <i>X</i>-axis with their corresponding version number (if available), mentioned in first parenthesis. <span class="highlight" style="background-color:">Pathway</span> statistics presented in these graphs (<i>Y</i>-axis) are taken from those corresponding database versions, i.e. on or before 21 July 2014. The blue arrows indicating the names of the <span class="highlight" style="background-color:">pathway</span> databases signify that these databases do not possess Hedgehog or Notch <span class="highlight" style="background-color:">pathway</span> information. Moreover, the databases indicated by red arrows show the databases that do not possess their own curated Hedgehog or Notch <span class="highlight" style="background-color:">pathway</span> data. These types of databases (e.g. WikiPathways, hiPathDB and GOLD.db) are mostly dependent on the curated data of other databases and therefore the data from these databases are not included in the comparisons. Although these databases possess the same data, which is available in other databases, but the importance of such databases are found in other aspects of <span class="highlight" style="background-color:">pathway</span> data analysis. (<b>C</b>) The total number of <span class="highlight" style="background-color:">signaling</span> pathways of each database is plotted and compared. Here, the black arrows indicate the names of the databases whose total number of available <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> data could not be counted. It shows hiPathDB has highest number of available <span class="highlight" style="background-color:">pathway</span> data present in its database, though the data it contains are all gathered from KEGG, REACTOME and NCI-PID.</div></p></div><div class="aux"><div class="resc">Saikat Chowdhury, et al. Database (Oxford). 2015;2015:bau126.</div><p class="links"><a class="dblinks" href="/pubmed/25632107" ref="ordinalpos=97&amp;ncbi_uid=7212786&amp;link_uid=25632107">Citation</a><a class="dblinks" href="/pmc/articles/PMC4309023/" ref="ordinalpos=97&amp;ncbi_uid=7212786&amp;link_uid=PMC4309023">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>98.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3681479/figure/F1/" ref="ordinalpos=98&amp;ncbi_uid=4999980&amp;link_uid=PMC3681479" href="/pmc/articles/PMC3681479/"><img src="/pmc/articles/instance/3681479/bin/oncotarget-03-1483-g001.gif" src-large="/pmc/articles/instance/3681479/bin/oncotarget-03-1483-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3681479/" ref="ordinalpos=98&amp;ncbi_uid=4999980&amp;link_uid=PMC3681479" image-link="/pmc/articles/PMC3681479/figure/F1/" class="imagepopup">Figure 1. Cross talk between Akt/mTOR and MEK <span class="highlight" style="background-color:">signaling</span> pathways.  From: The promise of dual targeting Akt/mTOR <span class="highlight" style="background-color:">signaling</span> in lethal prostate cancer. </a></p><div class="supp"><p class="details"><div>Loss of function of <i>PTEN</i> leads to activation of the mTOR <span class="highlight" style="background-color:">pathway</span> mainly via the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and secondarily via the MEK <span class="highlight" style="background-color:">pathway</span>. Inhibition of mTOR using MK-8669 leads to a feedback loop activation of Akt (1) Our findings suggest that inhibition of Akt using MK-2206 is not effective for inhibition of mTOR because of MEK <span class="highlight" style="background-color:">pathway</span> activation (2). Dual inhibition of both with MK-8669 and MK-2206 leads to effective inhibition of Akt/mTOR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Nicolas Floc'h, et al. Oncotarget. 2012 December;3(12):1483-1484.</div><p class="links"><a class="dblinks" href="/pubmed/23242005" ref="ordinalpos=98&amp;ncbi_uid=4999980&amp;link_uid=23242005">Citation</a><a class="dblinks" href="/pmc/articles/PMC3681479/" ref="ordinalpos=98&amp;ncbi_uid=4999980&amp;link_uid=PMC3681479">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>99.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4092141/figure/ppat-1004235-g001/" ref="ordinalpos=99&amp;ncbi_uid=6393321&amp;link_uid=PMC4092141" href="/pmc/articles/PMC4092141/"><img src="/pmc/articles/instance/4092141/bin/ppat.1004235.g001.gif" src-large="/pmc/articles/instance/4092141/bin/ppat.1004235.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4092141/" ref="ordinalpos=99&amp;ncbi_uid=6393321&amp;link_uid=PMC4092141" image-link="/pmc/articles/PMC4092141/figure/ppat-1004235-g001/" class="imagepopup">Figure 1. Regulation of C. elegans dauer development by cellular <span class="highlight" style="background-color:">signaling</span> pathways..  From: cGMP and NHR <span class="highlight" style="background-color:">Signaling</span> Co-regulate Expression of Insulin-Like Peptides and Developmental Activation of Infective Larvae in Strongyloides stercoralis. </a></p><div class="supp"><p class="details"><div>Dauer entry in <i>C. elegans</i> is regulated by four <span class="highlight" style="background-color:">signaling</span> pathways: a cyclic guanosine monophosphate (cGMP) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, an insulin/IGF-1-like <span class="highlight" style="background-color:">signaling</span> (IIS) <span class="highlight" style="background-color:">pathway</span>, a dauer transforming growth factor  (TGF) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and a DAF-12 nuclear hormone receptor (NHR) regulated by dafachronic acid (DA) steroid ligands. Unfavorable conditions stimulate dauer entry (left panel) by down-regulating cGMP production, increasing expression of antagonistic insulin-like peptides that down-regulate IIS, decreasing expression of the dauer TGF ligand, and inhibiting production of DAs. When dauer larvae encounter favorable conditions, these pathways are hypothesized to act in reverse. Dotted lines signify down-regulated <span class="highlight" style="background-color:">signaling</span> through the <span class="highlight" style="background-color:">pathway</span>, while black lines signify up-regulated <span class="highlight" style="background-color:">signaling</span> through the <span class="highlight" style="background-color:">pathway</span>. Colored proteins are active; grayed out proteins are inactive. Grayed out ligands are absent. Adapted from [12], [32], [43], [91].</div></p></div><div class="aux"><div class="resc">Jonathan D. Stoltzfus, et al. PLoS Pathog. 2014 July;10(7):e1004235.</div><p class="links"><a class="dblinks" href="/pubmed/25010340" ref="ordinalpos=99&amp;ncbi_uid=6393321&amp;link_uid=25010340">Citation</a><a class="dblinks" href="/pmc/articles/PMC4092141/" ref="ordinalpos=99&amp;ncbi_uid=6393321&amp;link_uid=PMC4092141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>100.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2869356/figure/pone-0010624-g003/" ref="ordinalpos=100&amp;ncbi_uid=2357148&amp;link_uid=PMC2869356" href="/pmc/articles/PMC2869356/"><img src="/pmc/articles/instance/2869356/bin/pone.0010624.g003.gif" src-large="/pmc/articles/instance/2869356/bin/pone.0010624.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2869356/" ref="ordinalpos=100&amp;ncbi_uid=2357148&amp;link_uid=PMC2869356" image-link="/pmc/articles/PMC2869356/figure/pone-0010624-g003/" class="imagepopup">Figure 3. Assessment of the biological relevance of the predicted genetic interaction network with <span class="highlight" style="background-color:">pathway</span> annotations..  From: Searching for <span class="highlight" style="background-color:">Signaling</span> Balance through the Identification of Genetic Interactors of the Rab Guanine-Nucleotide Dissociation Inhibitor gdi-1. </a></p><div class="supp"><p class="details"><div>Here we show scenarios where a pair of genes annotated to the same <span class="highlight" style="background-color:">pathway</span> (A) is directly connected or (B) shares 1 neighbor in a genetic interaction network, where the gene pair of interest is highlighted with thick grey rings. In (A), the genes exhibit a within-<span class="highlight" style="background-color:">pathway</span> genetic interaction based on the given set of <span class="highlight" style="background-color:">pathway</span> annotations. In (B), the genes belonging to the same <span class="highlight" style="background-color:">pathway</span> (e.g. <span class="highlight" style="background-color:">pathway</span> A) both interact with a gene that may either be an unknown member of the same <span class="highlight" style="background-color:">pathway</span> (within <span class="highlight" style="background-color:">pathway</span> interaction), or may belong to a different <span class="highlight" style="background-color:">pathway</span> (e.g. <span class="highlight" style="background-color:">pathway</span> B, between-<span class="highlight" style="background-color:">pathway</span> interactions). Below, the frequencies at which each scenario occurs in the predicted network and in randomized networks are shown with respect to all pathways and to <span class="highlight" style="background-color:">signaling</span> pathways only (see Methods). The all pathways and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> annotations were derived from human and <i>C. elegans</i> experimental data respectively. For each set of <span class="highlight" style="background-color:">pathway</span> annotations, the median, first and third quartile frequencies of each scenario were computed across <i>N</i>=100K randomized networks; the bar length depicts the median and the error bars depict the first and third quartiles. Both scenarios occur more frequently than what is expected by chance, for both sets of <span class="highlight" style="background-color:">pathway</span> annotations.</div></p></div><div class="aux"><div class="resc">Anna Y. Lee, et al. PLoS One. 2010;5(5):e10624.</div><p class="links"><a class="dblinks" href="/pubmed/20498707" ref="ordinalpos=100&amp;ncbi_uid=2357148&amp;link_uid=20498707">Citation</a><a class="dblinks" href="/pmc/articles/PMC2869356/" ref="ordinalpos=100&amp;ncbi_uid=2357148&amp;link_uid=PMC2869356">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><span title="Inactive first page of results" class="inactive page_link">&lt;&lt; First</span><span title="Inactive previous page of results" class="inactive page_link prev">&lt; Prev</span><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="1" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="2" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553ebca2cbe05d491bc324d4">signaling pathway <span class="number">(471856)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebc89437716036a8a124a">disease pathway <span class="number">(392428)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553ebc72437716036a8a0f36">differentiation pathway <span class="number">(234089)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553ebc53cbe05d491bc316a6">metabolic pathway <span class="number">(214028)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553ebc19cbe05d491bc30c66">cancer pathways <span class="number">(277189)</span></a>
          <div class="tertiary">PMC</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-27T19:00:54-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC6E0553EBF9E10000000000090247;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC6E0553EBF9E10000000000090247" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="12" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="8A1A393946D25221_0091SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- 8A1A393946D25221_0091SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r451781 Wed, Nov 12 2014 02:48:02 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er451781+Wed%2C+Nov+12+2014+02%3A48%3A02'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>