
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="12" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC70BF53FEC191000000000095762F" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="11" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="12" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="13" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="12" /></div>
            <div><h2 class="result_count">Results: 1101 to 1200 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1101.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2335355/figure/fig8/" ref="ordinalpos=1101&amp;ncbi_uid=1379071&amp;link_uid=PMC2335355" href="/pmc/articles/PMC2335355/"><img src="/pmc/articles/instance/2335355/bin/zbc0180833530008.gif" src-large="/pmc/articles/instance/2335355/bin/zbc0180833530008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2335355/" ref="ordinalpos=1101&amp;ncbi_uid=1379071&amp;link_uid=PMC2335355" image-link="/pmc/articles/PMC2335355/figure/fig8/" class="imagepopup">FIGURE 8. From: A Distinctive Physiological Role for I?B? in the Propagation  of Mitochondrial Respiratory Stress  <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>I</b>κ<b>B</b>β <b>as a critical landmark factor in  respiratory stress <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> A model showing interruption of  mitochondrial stress <span class="highlight" style="background-color:">signaling</span> that regulates different aspects of cellular  processes in IκBβ mRNA-silenced cells.</div></p></div><div class="aux"><div class="resc">Gopa Biswas, et al. J Biol Chem. 2008 May 2;283(18):12586-12594.</div><p class="links"><a class="dblinks" href="/pubmed/18272519" ref="ordinalpos=1101&amp;ncbi_uid=1379071&amp;link_uid=18272519">Citation</a><a class="dblinks" href="/pmc/articles/PMC2335355/" ref="ordinalpos=1101&amp;ncbi_uid=1379071&amp;link_uid=PMC2335355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1102.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355780/figure/F4/" ref="ordinalpos=1102&amp;ncbi_uid=4027582&amp;link_uid=PMC3355780" href="/pmc/articles/PMC3355780/"><img src="/pmc/articles/instance/3355780/bin/fpls-02-00047-g004.gif" src-large="/pmc/articles/instance/3355780/bin/fpls-02-00047-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355780/" ref="ordinalpos=1102&amp;ncbi_uid=4027582&amp;link_uid=PMC3355780" image-link="/pmc/articles/PMC3355780/figure/F4/" class="imagepopup">Figure 4.  From: Rewiring of the Jasmonate <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Arabidopsis during Insect Herbivory. </a></p><div class="supp"><p class="details"><div><b>Effect of the MYC2- and ERF-branch of the JA <span class="highlight" style="background-color:">pathway</span> on the preference of <i>P. rapae</i> in two-choice tests</b>. In the two-choice arena, two first-instar larvae of <i>P. rapae</i> were placed on each plant <b>(A)</b>. Each two-choice arena (<i>n</i> = 20) consisted of two 6-week-old plants of each genotype (total eight larvae per arena). After 4 days the number of caterpillars on each genotype was determined. The preference of <i>P. rapae</i> larvae was tested in combinations with wild-type Col-0 plants and genotypes that are affected in the MYC2-branch of the JA <span class="highlight" style="background-color:">pathway</span> <b>(B1–3)</b>, genotypes that are affected in the ERF-branch and/or the MYC2-branch of the JA <span class="highlight" style="background-color:">pathway</span> <b>(C1–5)</b>, and mutant <i>coi1-1</i> plants that are completely blocked in JA <span class="highlight" style="background-color:">signaling</span> <b>(D1)</b>. The right panel displays which branch of the JA <span class="highlight" style="background-color:">pathway</span> is predominantly activated in the corresponding genotypes that are displayed in the left panel. The “0” indicates that neither of the branches of the JA <span class="highlight" style="background-color:">pathway</span> was activated. Displayed are the average percentages (±SE) of the distribution of the <i>P. rapae</i> larvae over the two genotypes (<i>x</i>-axis). Statistically significant differences from the 50% percentile were analyzed using a Student’s <i>t</i>-test. In cases of statistically significant differences (Student’s <i>t-</i>test; <i>p</i> &lt; 0.05), the preferred branch of the JA <span class="highlight" style="background-color:">pathway</span> is marked with a red circle. Experiments were repeated with similar results.</div></p></div><div class="aux"><div class="resc">Adriaan Verhage, et al. Front Plant Sci. 2011;2:47.</div><p class="links"><a class="dblinks" href="/pubmed/22645537" ref="ordinalpos=1102&amp;ncbi_uid=4027582&amp;link_uid=22645537">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355780/" ref="ordinalpos=1102&amp;ncbi_uid=4027582&amp;link_uid=PMC3355780">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1103.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3624693/figure/f2/" ref="ordinalpos=1103&amp;ncbi_uid=5983352&amp;link_uid=PMC3624693" href="/pmc/articles/PMC3624693/"><img src="/pmc/articles/instance/3624693/bin/fig-2.gif" src-large="/pmc/articles/instance/3624693/bin/fig-2.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3624693/" ref="ordinalpos=1103&amp;ncbi_uid=5983352&amp;link_uid=PMC3624693" image-link="/pmc/articles/PMC3624693/figure/f2/" class="imagepopup">FIG. 2. From: Hope and Fear for Interferon: The Receptor-Centric Outlook on the Future of Interferon Therapy. </a></p><div class="supp"><p class="details"><div>Eliminative <span class="highlight" style="background-color:">signaling</span> by IFN1. The concept of eliminative <span class="highlight" style="background-color:">signaling</span> (Huangfu and Fuchs 2010) is illustrated specifically on the example of IFN1 responses. IFN1 binds to the receptor (IFNAR) to activate JAK (stem <span class="highlight" style="background-color:">signaling</span>). JAK serves as a central <span class="highlight" style="background-color:">signaling</span> dispatcher that further distributes the cascades between forward <span class="highlight" style="background-color:">signaling</span> (ie, activation of STAT resulting in the transcriptional activation of ISGs) and eliminative <span class="highlight" style="background-color:">signaling</span> [ie, activation of protein kinase D2 (PKD2) and subsequent downregulation of the receptor]. Unrelated ligands such as vascular endothelial growth factor (VEGF) or tumor necrosis factor-α (TNF-α) could utilize PKD2 or basal <span class="highlight" style="background-color:">pathway</span> to eliminate the receptor without benefits of forward <span class="highlight" style="background-color:">signaling</span> and ISG induction.</div></p></div><div class="aux"><div class="resc">Serge Y. Fuchs. J Interferon Cytokine Res. 2013 April;33(4):211-225.</div><p class="links"><a class="dblinks" href="/pubmed/23570388" ref="ordinalpos=1103&amp;ncbi_uid=5983352&amp;link_uid=23570388">Citation</a><a class="dblinks" href="/pmc/articles/PMC3624693/" ref="ordinalpos=1103&amp;ncbi_uid=5983352&amp;link_uid=PMC3624693">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1104.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4243705/figure/F4/" ref="ordinalpos=1104&amp;ncbi_uid=6879378&amp;link_uid=PMC4243705" href="/pmc/articles/PMC4243705/"><img src="/pmc/articles/instance/4243705/bin/nihms642954f4.gif" src-large="/pmc/articles/instance/4243705/bin/nihms642954f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4243705/" ref="ordinalpos=1104&amp;ncbi_uid=6879378&amp;link_uid=PMC4243705" image-link="/pmc/articles/PMC4243705/figure/F4/" class="imagepopup">Figure 4. The RAS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: THE CHROMOSOMAL INSTABILITY <span class="highlight" style="background-color:">PATHWAY</span> IN COLON CANCER. </a></p><div class="supp"><p class="details"><div>Growth factors binding to their cell surface receptors activate guanine exchange factors (GEF) such as SOS (son of sevenless) that are attached by the adaptor protein GRB2 (growth-factor-receptor bound protein 2). SOS stimulates the release of bound GDP from RAS, and it is exchanged for GTP, leading to the active RAS-GTP conformation. The GTPase-activating proteins (GAP) can bind to RAS-GTP and accelerate the conversion of RAS-GTP to RAS-GDP (guanosine diphosphate), which terminates <span class="highlight" style="background-color:">signaling</span>. Mutated RAS is constitutively active in the RAS-GTP conformation. Activated RAS regulates multiple cellular functions through effectors including the Raf-MEK-ERK <span class="highlight" style="background-color:">pathway</span>, PI3K, RALGDS, RALGDS-like gene (RLG) and RGL2.</div></p></div><div class="aux"><div class="resc">Maria S. Pino, et al. Gastroenterology. ;138(6):2059-2072.</div><p class="links"><a class="dblinks" href="/pubmed/20420946" ref="ordinalpos=1104&amp;ncbi_uid=6879378&amp;link_uid=20420946">Citation</a><a class="dblinks" href="/pmc/articles/PMC4243705/" ref="ordinalpos=1104&amp;ncbi_uid=6879378&amp;link_uid=PMC4243705">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1105.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3243355/figure/i1543-8120-67-1-1-f03/" ref="ordinalpos=1105&amp;ncbi_uid=3758727&amp;link_uid=PMC3243355" href="/pmc/articles/PMC3243355/"><img src="/pmc/articles/instance/3243355/bin/i1543-8120-67-1-1-f03.gif" src-large="/pmc/articles/instance/3243355/bin/i1543-8120-67-1-1-f03.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3243355/" ref="ordinalpos=1105&amp;ncbi_uid=3758727&amp;link_uid=PMC3243355" image-link="/pmc/articles/PMC3243355/figure/i1543-8120-67-1-1-f03/" class="imagepopup">Figure 3.  From: Sugar Sensing and <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Overview of the elaborate genetic interactions between glucose and hormone <span class="highlight" style="background-color:">signaling</span> in Arabidopsis thaliana.<br />Glucose <span class="highlight" style="background-color:">signaling</span> mediated by HXK1 leads to increased ABA levels and induces both ABA synthesis (ABA1-3 and AAO3 gene expression) and ABA (ABI3-5, ABI8, ABF2-mediated) <span class="highlight" style="background-color:">signaling</span> to control seedling development. Other ABA-insensitive <span class="highlight" style="background-color:">signaling</span> mutants (<i>abi1-1</i>, <i>abi2-1</i>) do not exhibit the gin phenotype, suggesting distinct ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for Glc <span class="highlight" style="background-color:">signaling</span> and the vegetative stress response. The glucose insensitive (<i>gin</i>) phenotype is mimicked by ethylene precursor treatment of wild-type plants and is also displayed in constitutive ethylene biosynthesis (<i>eto1</i>) and constitutive ethylene <span class="highlight" style="background-color:">signaling</span> (<i>ctr1</i>) mutants, whereas the ethylene-insensitive mutants <i>etr1-1</i>, <i>ein2</i>, <i>ein3</i> as well as <i>mkk9</i> exhibit glucose hypersensitivity (<i>glo</i> phenotypes). Glucose and ethylene <span class="highlight" style="background-color:">signaling</span> converge on the regulation of the ETHYLENE INSENSITIVE3 (EIN3) TF to differentially modulate its protein stability. The bifurcate and antagonistic CTR1 and MKK9 pathways are both critical in determining ethylene-<span class="highlight" style="background-color:">signaling</span> specificity through two MAPK phosphorylation sites with opposite effects on EIN3 stability. In addition, HXK-<span class="highlight" style="background-color:">signaling</span> positively and negatively interacts with auxin and cytokinin <span class="highlight" style="background-color:">signaling</span>, respectively. More details on hormone and sugar interactions are presented in the text.</div></p></div><div class="aux"><div class="resc">Matthew Ramon, et al. Arabidopsis Book. 2008;6:e0117.</div><p class="links"><a class="dblinks" href="/pubmed/22303242" ref="ordinalpos=1105&amp;ncbi_uid=3758727&amp;link_uid=22303242">Citation</a><a class="dblinks" href="/pmc/articles/PMC3243355/" ref="ordinalpos=1105&amp;ncbi_uid=3758727&amp;link_uid=PMC3243355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1106.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3488128/figure/F7/" ref="ordinalpos=1106&amp;ncbi_uid=5439719&amp;link_uid=PMC3488128" href="/pmc/articles/PMC3488128/"><img src="/pmc/articles/instance/3488128/bin/nihms399517f7.gif" src-large="/pmc/articles/instance/3488128/bin/nihms399517f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3488128/" ref="ordinalpos=1106&amp;ncbi_uid=5439719&amp;link_uid=PMC3488128" image-link="/pmc/articles/PMC3488128/figure/F7/" class="imagepopup">Figure 7. Differential expression of notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes and CoupTF1&amp;2.  From: Differential Responses to Retinoic Acid and Endocrine Disruptor Compounds of Subpopulations within Human Embryonic Stem Cell Lines. </a></p><div class="supp"><p class="details"><div>A) Heat map depicting expression of Notch <span class="highlight" style="background-color:">signaling</span> components and orphan receptors, CoupTF1 and CoupTF2 in H1 hES small and large cell population. B) mRNA expression of CoupTF1&amp;2 and notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes. mRNA expression is normalized to small cell population. C) Relative mRNA expression of CoupTF1 and Notch1 after treating the small and large cell population with RA for 6 days.</div></p></div><div class="aux"><div class="resc">Lois A. Annab, et al. Differentiation. 2012 November;84(4):330-343.</div><p class="links"><a class="dblinks" href="/pubmed/22906706" ref="ordinalpos=1106&amp;ncbi_uid=5439719&amp;link_uid=22906706">Citation</a><a class="dblinks" href="/pmc/articles/PMC3488128/" ref="ordinalpos=1106&amp;ncbi_uid=5439719&amp;link_uid=PMC3488128">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1107.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2783925/figure/F4/" ref="ordinalpos=1107&amp;ncbi_uid=2595301&amp;link_uid=PMC2783925" href="/pmc/articles/PMC2783925/"><img src="/pmc/articles/instance/2783925/bin/nihms-151040-f0004.gif" src-large="/pmc/articles/instance/2783925/bin/nihms-151040-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783925/" ref="ordinalpos=1107&amp;ncbi_uid=2595301&amp;link_uid=PMC2783925" image-link="/pmc/articles/PMC2783925/figure/F4/" class="imagepopup">Figure 4. Analysis of RAS and PI3-Kinase <span class="highlight" style="background-color:">pathway</span> activation signatures.  From: Polycythemia vera erythroid precursors exhibit increased proliferation and apoptosis resistance associated with abnormal RAS and PI3-Kinase <span class="highlight" style="background-color:">pathway</span> activation. </a></p><div class="supp"><p class="details"><div>Gene expression signatures of oncogenic <span class="highlight" style="background-color:">signaling</span> pathways to characterize the activation status of RAS and PI3-kinase pathways were applied to data from patients with PV and healthy controls. The probability of PI3-kinase (panels on left) and RAS <span class="highlight" style="background-color:">pathway</span> (panels on right) deregulation in PV erythroblasts are significantly increased compared to normal control erythroblasts.</div></p></div><div class="aux"><div class="resc">Jacob P. Laubach, et al. Exp Hematol. ;37(12):1411-1422.</div><p class="links"><a class="dblinks" href="/pubmed/19815050" ref="ordinalpos=1107&amp;ncbi_uid=2595301&amp;link_uid=19815050">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783925/" ref="ordinalpos=1107&amp;ncbi_uid=2595301&amp;link_uid=PMC2783925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1108.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4227245/figure/ijms-15-18762-f006/" ref="ordinalpos=1108&amp;ncbi_uid=6826279&amp;link_uid=PMC4227245" href="/pmc/articles/PMC4227245/"><img src="/pmc/articles/instance/4227245/bin/ijms-15-18762-g006.gif" src-large="/pmc/articles/instance/4227245/bin/ijms-15-18762-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4227245/" ref="ordinalpos=1108&amp;ncbi_uid=6826279&amp;link_uid=PMC4227245" image-link="/pmc/articles/PMC4227245/figure/ijms-15-18762-f006/" class="imagepopup">Figure 6.  From: Squamosamide Derivative FLZ Protects Retinal Pigment Epithelium Cells from Oxidative Stress through Activation of Epidermal Growth Factor Receptor (EGFR)-AKT <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of this study.</div></p></div><div class="aux"><div class="resc">Li-Bo Cheng, et al. Int J Mol Sci. 2014 October;15(10):18762-18775.</div><p class="links"><a class="dblinks" href="/pubmed/25329617" ref="ordinalpos=1108&amp;ncbi_uid=6826279&amp;link_uid=25329617">Citation</a><a class="dblinks" href="/pmc/articles/PMC4227245/" ref="ordinalpos=1108&amp;ncbi_uid=6826279&amp;link_uid=PMC4227245">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1109.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404876/figure/F1/" ref="ordinalpos=1109&amp;ncbi_uid=5035667&amp;link_uid=PMC3404876" href="/pmc/articles/PMC3404876/"><img src="/pmc/articles/instance/3404876/bin/cc-11-2452-g1.gif" src-large="/pmc/articles/instance/3404876/bin/cc-11-2452-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404876/" ref="ordinalpos=1109&amp;ncbi_uid=5035667&amp;link_uid=PMC3404876" image-link="/pmc/articles/PMC3404876/figure/F1/" class="imagepopup">Figure. From: Identification of novel metastasis suppressor <span class="highlight" style="background-color:">signaling</span> pathways for breast cancer. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Schematic representation of our strategy to identify the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components downstream of RKIP that lead to BMS genes.</div></p></div><div class="aux"><div class="resc">Andy J. Minn, et al. Cell Cycle. 2012 July 1;11(13):2452-2457.</div><p class="links"><a class="dblinks" href="/pubmed/22659842" ref="ordinalpos=1109&amp;ncbi_uid=5035667&amp;link_uid=22659842">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404876/" ref="ordinalpos=1109&amp;ncbi_uid=5035667&amp;link_uid=PMC3404876">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1110.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4255921/figure/Fig1/" ref="ordinalpos=1110&amp;ncbi_uid=6908226&amp;link_uid=PMC4255921" href="/pmc/articles/PMC4255921/"><img src="/pmc/articles/instance/4255921/bin/40104_2014_130_Fig1_HTML.gif" src-large="/pmc/articles/instance/4255921/bin/40104_2014_130_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4255921/" ref="ordinalpos=1110&amp;ncbi_uid=6908226&amp;link_uid=PMC4255921" image-link="/pmc/articles/PMC4255921/figure/Fig1/" class="imagepopup">Figure 1.  From: Transforming growth factor β <span class="highlight" style="background-color:">signaling</span> in uterine development and function. </a></p><div class="supp"><p class="details"><div> <b>Canonical and non-canonical TGFβ <span class="highlight" style="background-color:">signaling</span>.</b> In the canonical <span class="highlight" style="background-color:">pathway</span>, TGFβ ligands bind to serine/threonine kinase type II and type I receptors and phosphorylate R-SMADs, which form heteromeric complexes with SMAD4 and translocate into the nucleus to regulate gene transcription. The non-canonical <span class="highlight" style="background-color:">pathway</span> generally refers to the SMAD-independent <span class="highlight" style="background-color:">pathway</span> such as PI3K-AKT, ERK1/2, p38, and JNK pathways. Recent studies have identified an “R-SMAD-dependent but SMAD4-independent” non-canonical <span class="highlight" style="background-color:">pathway</span> that regulates miRNA maturation.</div></p></div><div class="aux"><div class="resc">Qinglei Li. J Anim Sci Biotechnol. 2014;5(1):52.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1110&amp;ncbi_uid=6908226&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4255921/" ref="ordinalpos=1110&amp;ncbi_uid=6908226&amp;link_uid=PMC4255921">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1111.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3359868/figure/F2/" ref="ordinalpos=1111&amp;ncbi_uid=4851500&amp;link_uid=PMC3359868" href="/pmc/articles/PMC3359868/"><img src="/pmc/articles/instance/3359868/bin/nihms362865f2.gif" src-large="/pmc/articles/instance/3359868/bin/nihms362865f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3359868/" ref="ordinalpos=1111&amp;ncbi_uid=4851500&amp;link_uid=PMC3359868" image-link="/pmc/articles/PMC3359868/figure/F2/" class="imagepopup">Figure 2.  From: Insights into the complex influence of 5-HT <span class="highlight" style="background-color:">signaling</span> on thalamocortical axonal system development. </a></p><div class="supp"><p class="details"><div>Timeline of the genetic and pharmacological alterations of 5-HT <span class="highlight" style="background-color:">signaling</span> and their effects on thalamocortical axons (TCA) <span class="highlight" style="background-color:">pathway</span> and cortical barrel field (BF) formation. Manipulations leading to increased 5-HT <span class="highlight" style="background-color:">signaling</span> during development are illustrated in the top panel. Overexpression of 5-HT1B/1D receptors (IUE: <i>in utero</i> electroporation) alters TCA <span class="highlight" style="background-color:">pathway</span> prenatally; the effect on postnatal BF formation is unknown (dotted box). In the SERT-KO and MAOA-KO mice, extracellular brain 5-HT concentration is increased and BF formation is altered postnatally. The blue arrows represent the duration of the effect on 5-HT <span class="highlight" style="background-color:">signaling</span>. The bottom panel illustrates manipulations leading to a decrease of 5-HT <span class="highlight" style="background-color:">signaling</span> prenatally (IUE) and/or pre- and postnatally (knockout mice and p-chlorophenylalanine (PCPA) treatment). Knocking down 5-HT1B/1D receptors expression by <i>in utero</i> electroporation alters TCA <span class="highlight" style="background-color:">pathway</span> prenatally, with unknown effect on postnatal BF formation. In most of the genetic mouse models, prenatal effects on TCA <span class="highlight" style="background-color:">pathway</span> formation were not analyzed in detail. The potential prenatal effects on the placental source of 5-HT, inferred from the targeted genes being expressed in the placenta, are indicated (yellow line).</div></p></div><div class="aux"><div class="resc">Esmee SB van Kleef, et al. Eur J Neurosci. ;35(10):1563-1572.</div><p class="links"><a class="dblinks" href="/pubmed/22607002" ref="ordinalpos=1111&amp;ncbi_uid=4851500&amp;link_uid=22607002">Citation</a><a class="dblinks" href="/pmc/articles/PMC3359868/" ref="ordinalpos=1111&amp;ncbi_uid=4851500&amp;link_uid=PMC3359868">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1112.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4196276/figure/F8/" ref="ordinalpos=1112&amp;ncbi_uid=6719031&amp;link_uid=PMC4196276" href="/pmc/articles/PMC4196276/"><img src="/pmc/articles/instance/4196276/bin/emss-60634-f0008.gif" src-large="/pmc/articles/instance/4196276/bin/emss-60634-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4196276/" ref="ordinalpos=1112&amp;ncbi_uid=6719031&amp;link_uid=PMC4196276" image-link="/pmc/articles/PMC4196276/figure/F8/" class="imagepopup">Fig. 8. TNF-alpha (the focus node) was originally explored as a target for the Graft-Versus-Host Disease (GVHD, top-right <span class="highlight" style="background-color:">pathway</span>)..  From: Entourage: Visualizing Relationships between Biological Pathways using Contextual Subsets. </a></p><div class="supp"><p class="details"><div>However, when tested in clinical trials, <i>TNF-alpha</i> inhibiting compounds were not effective against GVHD but could later be repositioned for the treatment of <i>Rheumatoid Arthritis</i> (focus <span class="highlight" style="background-color:">pathway</span>). Entourage shows Rheumatoid Athritis as closely related to the GVHD <span class="highlight" style="background-color:">pathway</span> (see <span class="highlight" style="background-color:">pathway</span> list on the left). Entourage also reveals seemingly contradictory roles of <i>TNF-alpha</i>. It is involved in cell death (Apoptosis) and also in cancer (i.e., uncontrolled cell growth) through the <i>MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></i>.</div></p></div><div class="aux"><div class="resc">Alexander Lex, et al. IEEE Trans Vis Comput Graph. ;19(12):2536-2545.</div><p class="links"><a class="dblinks" href="/pubmed/24051820" ref="ordinalpos=1112&amp;ncbi_uid=6719031&amp;link_uid=24051820">Citation</a><a class="dblinks" href="/pmc/articles/PMC4196276/" ref="ordinalpos=1112&amp;ncbi_uid=6719031&amp;link_uid=PMC4196276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1113.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3601458/figure/fig03/" ref="ordinalpos=1113&amp;ncbi_uid=4731865&amp;link_uid=PMC3601458" href="/pmc/articles/PMC3601458/"><img src="/pmc/articles/instance/3601458/bin/ijppp0005-0011-f3.gif" src-large="/pmc/articles/instance/3601458/bin/ijppp0005-0011-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3601458/" ref="ordinalpos=1113&amp;ncbi_uid=4731865&amp;link_uid=PMC3601458" image-link="/pmc/articles/PMC3601458/figure/fig03/" class="imagepopup">Figure 3.  From: Inhibition of the Rho <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> improves neurite outgrowth and neuronal differentiation of mouse neural stem cells. </a></p><div class="supp"><p class="details"><div> <i>Regulation of MAPK <span class="highlight" style="background-color:">pathway</span> molecules in C3-treated NCSs.</i> Western blot was applied to measure the protein levels of <span class="highlight" style="background-color:">signaling</span> molecules in the MAPK <span class="highlight" style="background-color:">pathway</span>. A. Representative immunoblots with antibodies against MAPK, pMAPK, Akt, pAkt, ROCK II and RhoA in cells exposed to 4-hr C3 (1.0 μg/ml) treatment. B. Quantification of Western blot bands for MAPK/pMAPK , Akt/pAkt , ROCK II and RhoA in C3 (1.0 μg/ml) treated cells. *. p&lt;0.05 compared with control group. N=4 assays.</div></p></div><div class="aux"><div class="resc">Haigang Gu, et al. Int J Physiol Pathophysiol Pharmacol. 2013;5(1):11-20.</div><p class="links"><a class="dblinks" href="/pubmed/23525456" ref="ordinalpos=1113&amp;ncbi_uid=4731865&amp;link_uid=23525456">Citation</a><a class="dblinks" href="/pmc/articles/PMC3601458/" ref="ordinalpos=1113&amp;ncbi_uid=4731865&amp;link_uid=PMC3601458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1114.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3345255/figure/fig1/" ref="ordinalpos=1114&amp;ncbi_uid=3996932&amp;link_uid=PMC3345255" href="/pmc/articles/PMC3345255/"><img src="/pmc/articles/instance/3345255/bin/SRCM2012-404810.001.gif" src-large="/pmc/articles/instance/3345255/bin/SRCM2012-404810.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3345255/" ref="ordinalpos=1114&amp;ncbi_uid=3996932&amp;link_uid=PMC3345255" image-link="/pmc/articles/PMC3345255/figure/fig1/" class="imagepopup">Figure 1.  From: MAPK/ERK <span class="highlight" style="background-color:">Signaling</span> in Osteosarcomas, Ewing Sarcomas and Chondrosarcomas: Therapeutic Implications and Future Directions. </a></p><div class="supp"><p class="details"><div>MAPK/ERK <span class="highlight" style="background-color:">signaling</span> and the hallmarks of cancers. The MAPK/ERK <span class="highlight" style="background-color:">pathway</span> mediates several upstream signals from well-known oncogenic growth factors and proinflammatory stimulants. Activation of the MAPK/ERK <span class="highlight" style="background-color:">pathway</span> by growth factors, proinflammatory stimulants and gain-of-function mutations of Ras/Raf promotes phenotypic changes characteristic of cancer cells [9–14].</div></p></div><div class="aux"><div class="resc">Chandhanarat Chandhanayingyong, et al. Sarcoma. 2012;2012:404810.</div><p class="links"><a class="dblinks" href="/pubmed/22577336" ref="ordinalpos=1114&amp;ncbi_uid=3996932&amp;link_uid=22577336">Citation</a><a class="dblinks" href="/pmc/articles/PMC3345255/" ref="ordinalpos=1114&amp;ncbi_uid=3996932&amp;link_uid=PMC3345255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1115.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3213922/figure/F8/" ref="ordinalpos=1115&amp;ncbi_uid=3406691&amp;link_uid=PMC3213922" href="/pmc/articles/PMC3213922/"><img src="/pmc/articles/instance/3213922/bin/zh00111167540008.gif" src-large="/pmc/articles/instance/3213922/bin/zh00111167540008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3213922/" ref="ordinalpos=1115&amp;ncbi_uid=3406691&amp;link_uid=PMC3213922" image-link="/pmc/articles/PMC3213922/figure/F8/" class="imagepopup">Fig. 8. From: Aldosterone stimulates vacuolar H+-ATPase activity in renal acid-secretory intercalated cells mainly via a protein kinase C-dependent <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Model of nongenomic <span class="highlight" style="background-color:">signaling</span> of aldosterone in mouse OMCD intercalated cells to stimulate H<sup>+</sup>-ATPase activity. Aldosterone stimulates H<sup>+</sup>-ATPase activity through a major <span class="highlight" style="background-color:">pathway</span> requiring G<sub>αq</sub> proteins, phospholipase C, intracellular Ca<sup>2+</sup>, PKC, and ERK1/2. A second interlinked <span class="highlight" style="background-color:">pathway</span> involves cAMP and PKA. The PKA-dependent <span class="highlight" style="background-color:">pathway</span> is likely upstream of PKC and ERK1/2 because inhibition of this <span class="highlight" style="background-color:">pathway</span> blocks the cAMP/PKA-dependent stimulation of H<sup>+</sup>-ATPase activity. Chelation of Ca<sup>2+</sup> may interfere at several stages because Ca<sup>2+</sup> may be involved in stimulation of PKC isoforms as well as in the exocytic insertion of H<sup>+</sup>-ATPases into the membrane, which has been shown earlier to occur during stimulation of H<sup>+</sup>-ATPase activity (58, 75). Inhibitors and activators of <span class="highlight" style="background-color:">signaling</span> molecules used in this study are shown in italics.</div></p></div><div class="aux"><div class="resc">Christian Winter, et al. Am J Physiol Cell Physiol. 2011 November;301(5):C1251-C1261.</div><p class="links"><a class="dblinks" href="/pubmed/21832245" ref="ordinalpos=1115&amp;ncbi_uid=3406691&amp;link_uid=21832245">Citation</a><a class="dblinks" href="/pmc/articles/PMC3213922/" ref="ordinalpos=1115&amp;ncbi_uid=3406691&amp;link_uid=PMC3213922">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1116.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3165102/figure/F1/" ref="ordinalpos=1116&amp;ncbi_uid=3451051&amp;link_uid=PMC3165102" href="/pmc/articles/PMC3165102/"><img src="/pmc/articles/instance/3165102/bin/nihms298879f1.gif" src-large="/pmc/articles/instance/3165102/bin/nihms298879f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3165102/" ref="ordinalpos=1116&amp;ncbi_uid=3451051&amp;link_uid=PMC3165102" image-link="/pmc/articles/PMC3165102/figure/F1/" class="imagepopup">Figure 1. Schematic representation of TGF-β and Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> in cancer.  From: The TGF-?/SMAD/GLI2 <span class="highlight" style="background-color:">signaling</span> axis in cancer progression and metastasis. </a></p><div class="supp"><p class="details"><div>TGF-β ligand activation of specific cell surface serine-threonine kinase receptors activates the SMAD cascade, resulting in transcriptional activation of the GLI2 gene. GLI2 protein may either regulate target gene expression and exert pro-oncogenic activities downstream of TGF-β <span class="highlight" style="background-color:">signaling</span> or may contribute to Hedgehog (Hh)-driven <span class="highlight" style="background-color:">signaling</span> which occurs via Smoothened (SMO) activation in the primary cilium following Hh binding to the <span class="highlight" style="background-color:">pathway</span> inhibitory receptor Patched-1 (PTCH1).</div></p></div><div class="aux"><div class="resc">Delphine Javelaud, et al. Cancer Res. ;71(17):5606-5610.</div><p class="links"><a class="dblinks" href="/pubmed/21862631" ref="ordinalpos=1116&amp;ncbi_uid=3451051&amp;link_uid=21862631">Citation</a><a class="dblinks" href="/pmc/articles/PMC3165102/" ref="ordinalpos=1116&amp;ncbi_uid=3451051&amp;link_uid=PMC3165102">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1117.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2710824/figure/F9/" ref="ordinalpos=1117&amp;ncbi_uid=1897494&amp;link_uid=PMC2710824" href="/pmc/articles/PMC2710824/"><img src="/pmc/articles/instance/2710824/bin/zmk0140991270009.gif" src-large="/pmc/articles/instance/2710824/bin/zmk0140991270009.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2710824/" ref="ordinalpos=1117&amp;ncbi_uid=1897494&amp;link_uid=PMC2710824" image-link="/pmc/articles/PMC2710824/figure/F9/" class="imagepopup">Figure 9. From: Estrogen Inhibits ATR <span class="highlight" style="background-color:">Signaling</span> to Cell Cycle Checkpoints and DNA Repair. </a></p><div class="supp"><p class="details"><div>Schematic of estrogen and membrane ERα-inhibiting ATR <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> after DNA damage. E<sub>2</sub> acting at membrane-localized ERα rapidly activates PI3K/AKT to phosphorylate and dissociate TopBP1 from ATR in the setting of DNA damage. This leads to inhibition of ATR activity and downstream <span class="highlight" style="background-color:">signaling</span> to the G2/M checkpoint and p53-induced p21 transcription. <span class="highlight" style="background-color:">Signaling</span> through PI3K and AKT also inhibits Chk1 activity by phosphorylating claspin.</div></p></div><div class="aux"><div class="resc">Ali Pedram, et al. Mol Biol Cell. 2009 July 15;20(14):3374-3389.</div><p class="links"><a class="dblinks" href="/pubmed/19477925" ref="ordinalpos=1117&amp;ncbi_uid=1897494&amp;link_uid=19477925">Citation</a><a class="dblinks" href="/pmc/articles/PMC2710824/" ref="ordinalpos=1117&amp;ncbi_uid=1897494&amp;link_uid=PMC2710824">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1118.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634403/figure/F1/" ref="ordinalpos=1118&amp;ncbi_uid=1711044&amp;link_uid=PMC2634403" href="/pmc/articles/PMC2634403/"><img src="/pmc/articles/instance/2634403/bin/psb0310_0877_fig001.gif" src-large="/pmc/articles/instance/2634403/bin/psb0310_0877_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634403/" ref="ordinalpos=1118&amp;ncbi_uid=1711044&amp;link_uid=PMC2634403" image-link="/pmc/articles/PMC2634403/figure/F1/" class="imagepopup">Figure 1.  From: Gibberellic acid-mediated salt <span class="highlight" style="background-color:">signaling</span> in seed germination. </a></p><div class="supp"><p class="details"><div>NTL8 regulation of GA-mediated salt <span class="highlight" style="background-color:">signaling</span> in seed germination. High salinity reduces GA biosynthesis. GA represses both <i>NTL8</i> transcription and NTL8 processing. However, it is likely that the <i>NTL8</i>-mediated <span class="highlight" style="background-color:">signaling</span> is not required under favorable growth conditions. It inhibits seed germination only when the surrounding conditions are unfavorable for plant propagation. <i>NTL8</i> may function as a molecular link that incorporates environmental signals into the GA-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in seed germination.</div></p></div><div class="aux"><div class="resc">Sang-Gyu Kim, et al. Plant Signal Behav. 2008 October;3(10):877-879.</div><p class="links"><a class="dblinks" href="/pubmed/19704528" ref="ordinalpos=1118&amp;ncbi_uid=1711044&amp;link_uid=19704528">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634403/" ref="ordinalpos=1118&amp;ncbi_uid=1711044&amp;link_uid=PMC2634403">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1119.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2504275/figure/Fig4/" ref="ordinalpos=1119&amp;ncbi_uid=1455848&amp;link_uid=PMC2504275" href="/pmc/articles/PMC2504275/"><img src="/pmc/articles/instance/2504275/bin/11420_2007_9072_Fig4_HTML.gif" src-large="/pmc/articles/instance/2504275/bin/11420_2007_9072_Fig4_HTML.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2504275/" ref="ordinalpos=1119&amp;ncbi_uid=1455848&amp;link_uid=PMC2504275" image-link="/pmc/articles/PMC2504275/figure/Fig4/" class="imagepopup">Fig. 4. From: Role of Wnt <span class="highlight" style="background-color:">Signaling</span> in Bone Remodeling and Repair. </a></p><div class="supp"><p class="details"><div>Effects of Wnt <span class="highlight" style="background-color:">signaling</span> on osteoblast and osteoclast differentiation. Wnt-regulated bone formation and remodeling result from an enhancement of mesenchymal precursor cell differentiation into osteoblasts. Wnt <span class="highlight" style="background-color:">signaling</span> blocks the alternative chondrocyte differentiation <span class="highlight" style="background-color:">pathway</span>. Differentiating osteoblasts synthesize and secrete osteoprotegerin in response to Wnt <span class="highlight" style="background-color:">signaling</span>; osteoprotegerin blocks osteoclast differentiation. Thus, Wnt signals increase bone formation by enhancing osteoblast-mediated bone formation and inhibiting osteoclast-mediated bone resorption and chondrocyte differentiation</div></p></div><div class="aux"><div class="resc">Paul S. Issack, et al. HSS J. 2008 February;4(1):66-70.</div><p class="links"><a class="dblinks" href="/pubmed/18751865" ref="ordinalpos=1119&amp;ncbi_uid=1455848&amp;link_uid=18751865">Citation</a><a class="dblinks" href="/pmc/articles/PMC2504275/" ref="ordinalpos=1119&amp;ncbi_uid=1455848&amp;link_uid=PMC2504275">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1120.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3104724/figure/F2/" ref="ordinalpos=1120&amp;ncbi_uid=2996243&amp;link_uid=PMC3104724" href="/pmc/articles/PMC3104724/"><img src="/pmc/articles/instance/3104724/bin/nihms294484f2.gif" src-large="/pmc/articles/instance/3104724/bin/nihms294484f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3104724/" ref="ordinalpos=1120&amp;ncbi_uid=2996243&amp;link_uid=PMC3104724" image-link="/pmc/articles/PMC3104724/figure/F2/" class="imagepopup">Fig. 2.  From: Functional role of KLF10 in multiple disease processes. </a></p><div class="supp"><p class="details"><div>This figure outlines the role of KLF10 protein as a target and as a regulator of the TGFβ-Smad 2,3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TGFβ binds to its membrane receptors (R<sub>2</sub> R<sub>1</sub>) to activate the Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. KLF10 (TIEG) is one of the rapid, early response genes whose expression is activated. KLF10 mRNA is rapidly translated into the KLF10 protein, which as a gene transcription repressor inhibits Smad 7 gene expression and, as a gene activator, induces Smad 2 gene expression. The ultimate outcome is a continued activation of the TGFβ-Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, that is, an extension of the <span class="highlight" style="background-color:">signaling</span> period. The KLF10 mRNA and protein are rapidly turned over, thus the <span class="highlight" style="background-color:">signaling</span> is fine tuned to the KLF10 levels and activity. The steroid, estrogen (E), also induces the expression of KLF10 gene transcription, which is one mechanism of the well documented crosstalk between TGFβ and estrogen <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Malayannan Subramaniam, et al. Biofactors. ;36(1):8-18.</div><p class="links"><a class="dblinks" href="/pubmed/20087894" ref="ordinalpos=1120&amp;ncbi_uid=2996243&amp;link_uid=20087894">Citation</a><a class="dblinks" href="/pmc/articles/PMC3104724/" ref="ordinalpos=1120&amp;ncbi_uid=2996243&amp;link_uid=PMC3104724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1121.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3461903/figure/F2/" ref="ordinalpos=1121&amp;ncbi_uid=4494678&amp;link_uid=PMC3461903" href="/pmc/articles/PMC3461903/"><img src="/pmc/articles/instance/3461903/bin/JCI61205.f2.gif" src-large="/pmc/articles/instance/3461903/bin/JCI61205.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3461903/" ref="ordinalpos=1121&amp;ncbi_uid=4494678&amp;link_uid=PMC3461903" image-link="/pmc/articles/PMC3461903/figure/F2/" class="imagepopup">Figure 2. Schematic representation of the PDGF/PDGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Generated using MetaCore software; GeneGo Inc.)..  From: New molecular insights into peripheral T cell lymphomas. </a></p><div class="supp"><p class="details"><div>Notably, different key points of PDGFR <span class="highlight" style="background-color:">signaling</span> can be the object of pharmacological inhibition. First, PDGFRs can be directly blocked by TKIs. Second, the immediate downstream JAK/STAT <span class="highlight" style="background-color:">pathway</span>, SRC kinase and PI3K can be selectively disrupted. Finally, in cases retaining NF-κB activity (around 30% of PTCL-NOSs), different molecules might be targeted. Adapted with permission from <i>Expert Review of Hematology</i> (59). </div></p></div><div class="aux"><div class="resc">Stefano A. Pileri, et al. J Clin Invest. 2012 October 1;122(10):3448-3455.</div><p class="links"><a class="dblinks" href="/pubmed/23023716" ref="ordinalpos=1121&amp;ncbi_uid=4494678&amp;link_uid=23023716">Citation</a><a class="dblinks" href="/pmc/articles/PMC3461903/" ref="ordinalpos=1121&amp;ncbi_uid=4494678&amp;link_uid=PMC3461903">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1122.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3265471/figure/pbio-1001249-g003/" ref="ordinalpos=1122&amp;ncbi_uid=3780292&amp;link_uid=PMC3265471" href="/pmc/articles/PMC3265471/"><img src="/pmc/articles/instance/3265471/bin/pbio.1001249.g003.gif" src-large="/pmc/articles/instance/3265471/bin/pbio.1001249.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3265471/" ref="ordinalpos=1122&amp;ncbi_uid=3780292&amp;link_uid=PMC3265471" image-link="/pmc/articles/PMC3265471/figure/pbio-1001249-g003/" class="imagepopup">Figure 3. The RIG-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated in IFNβ-producing cells..  From: Stochastic Expression of the Interferon-? Gene. </a></p><div class="supp"><p class="details"><div>(A and B) Western blots showing cytoplasmic (C) versus nuclear (N) distribution of different factors present in FACS-sorted cells 8 h.p.i. (A) and 12 h.p.i (B). (C) Western blots showing cytoplasmic distribution of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins present in FACS-sorted cells 8 h.p.i. Arrows indicate MAVS protein. (D) qPCR analysis illustrating the expression levels of RIG-I, Trim25, and MDA5 genes in sorted IFNβ/YFP homozygous MEF cells 8 h.p.i. IB, immunoblot.</div></p></div><div class="aux"><div class="resc">Mingwei Zhao, et al. PLoS Biol. 2012 January;10(1):e1001249.</div><p class="links"><a class="dblinks" href="/pubmed/22291574" ref="ordinalpos=1122&amp;ncbi_uid=3780292&amp;link_uid=22291574">Citation</a><a class="dblinks" href="/pmc/articles/PMC3265471/" ref="ordinalpos=1122&amp;ncbi_uid=3780292&amp;link_uid=PMC3265471">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1123.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3740053/figure/F5/" ref="ordinalpos=1123&amp;ncbi_uid=6434931&amp;link_uid=PMC3740053" href="/pmc/articles/PMC3740053/"><img src="/pmc/articles/instance/3740053/bin/nihms466905f5.gif" src-large="/pmc/articles/instance/3740053/bin/nihms466905f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3740053/" ref="ordinalpos=1123&amp;ncbi_uid=6434931&amp;link_uid=PMC3740053" image-link="/pmc/articles/PMC3740053/figure/F5/" class="imagepopup">Figure 5.  From: GENETIC ANALYSIS OF INTERFERON INDUCED THYROIDITIS (IIT): EVIDENCE FOR A KEY ROLE FOR MHC AND APOPTOSIS RELATED GENES AND PATHWAYS. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis (IPA) of PBMC RNAseq expression data. Shown are 7 pathways that belonged to groups of pathways identified by genomic convergence to be both genetically associated with IIT and significantly upregulated by IFNα in PBMCs. The percentage of genes that were upregulated are shown in light gray and genes that were downregulated are shown in dark gray, with the −log(p-value) shown at the right side of each bar. Interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was found to be the most significantly upregulated. Other groups of pathways found to be associated with IIT and upregulated by IFNα in PBMCs included pattern recognition receptor related, apoptosis, complement, NFκB <span class="highlight" style="background-color:">signaling</span>, IL-10 <span class="highlight" style="background-color:">signaling</span>, and IL-6 <span class="highlight" style="background-color:">signaling</span> pathways. Four of these groups of pathways were found to be strongly associated with IIT in both datasets (all IIT cases vs. HCV controls and Caucasian IIT cases vs. healthy, Caucasian controls). These include interferon <span class="highlight" style="background-color:">signaling</span>, apoptosis related pathways, NFκB <span class="highlight" style="background-color:">signaling</span>, and IL-10 <span class="highlight" style="background-color:">signaling</span> (underlined).</div></p></div><div class="aux"><div class="resc">Alia Hasham, et al. J Autoimmun. ;44:61-70.</div><p class="links"><a class="dblinks" href="/pubmed/23683877" ref="ordinalpos=1123&amp;ncbi_uid=6434931&amp;link_uid=23683877">Citation</a><a class="dblinks" href="/pmc/articles/PMC3740053/" ref="ordinalpos=1123&amp;ncbi_uid=6434931&amp;link_uid=PMC3740053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1124.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285553/figure/F4/" ref="ordinalpos=1124&amp;ncbi_uid=4578120&amp;link_uid=PMC3285553" href="/pmc/articles/PMC3285553/"><img src="/pmc/articles/instance/3285553/bin/nihms350220f4.gif" src-large="/pmc/articles/instance/3285553/bin/nihms350220f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285553/" ref="ordinalpos=1124&amp;ncbi_uid=4578120&amp;link_uid=PMC3285553" image-link="/pmc/articles/PMC3285553/figure/F4/" class="imagepopup">Figure 4.  From: Dysregulation of Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> in Gastrointestinal Cancers. </a></p><div class="supp"><p class="details"><div>Schematic illustrating numerous small molecules and biological compounds identified in the literature that inhibit Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> at various points in the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Bryan D. White, et al. Gastroenterology. ;142(2):219-232.</div><p class="links"><a class="dblinks" href="/pubmed/22155636" ref="ordinalpos=1124&amp;ncbi_uid=4578120&amp;link_uid=22155636">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285553/" ref="ordinalpos=1124&amp;ncbi_uid=4578120&amp;link_uid=PMC3285553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1125.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3069415/figure/F8/" ref="ordinalpos=1125&amp;ncbi_uid=3475202&amp;link_uid=PMC3069415" href="/pmc/articles/PMC3069415/"><img src="/pmc/articles/instance/3069415/bin/zbc0191157260008.gif" src-large="/pmc/articles/instance/3069415/bin/zbc0191157260008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3069415/" ref="ordinalpos=1125&amp;ncbi_uid=3475202&amp;link_uid=PMC3069415" image-link="/pmc/articles/PMC3069415/figure/F8/" class="imagepopup">FIGURE 8. From: <span class="highlight" style="background-color:">Signaling</span> via Macrophage G2A Enhances Efferocytosis of Dying Neutrophils by Augmentation of Rac Activity. </a></p><div class="supp"><p class="details"><div><b>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for G2A-mediated enhancement of apoptotic cell engulfment.</b> <i>PGES</i>, prostaglandin-E synthase; <i>AC</i>, adenylyl cyclase.</div></p></div><div class="aux"><div class="resc">S. Courtney Frasch, et al. J Biol Chem. 2011 April 8;286(14):12108-12122.</div><p class="links"><a class="dblinks" href="/pubmed/21297111" ref="ordinalpos=1125&amp;ncbi_uid=3475202&amp;link_uid=21297111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3069415/" ref="ordinalpos=1125&amp;ncbi_uid=3475202&amp;link_uid=PMC3069415">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1126.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3975838/figure/F2/" ref="ordinalpos=1126&amp;ncbi_uid=5992103&amp;link_uid=PMC3975838" href="/pmc/articles/PMC3975838/"><img src="/pmc/articles/instance/3975838/bin/1756-8722-7-18-2.gif" src-large="/pmc/articles/instance/3975838/bin/1756-8722-7-18-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3975838/" ref="ordinalpos=1126&amp;ncbi_uid=5992103&amp;link_uid=PMC3975838" image-link="/pmc/articles/PMC3975838/figure/F2/" class="imagepopup">Figure 2.  From: Targeting hedgehog <span class="highlight" style="background-color:">signaling</span> in myelofibrosis and other hematologic malignancies. </a></p><div class="supp"><p class="details"><div><b>Role of hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> in hematopoiesis.</b> Preclinical studies suggest that the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may be involved in numerous stages and processes of hematopoiesis, including primitive hematopoiesis, definitive hematopoiesis—establishment, proliferation, and differentiation (lymphoid and myeloid lineages)—and maintenance of HSCs. The exact role for Hh <span class="highlight" style="background-color:">signaling</span> at each of these stages/processes is not clear. HSC, hematopoietic stem cell.</div></p></div><div class="aux"><div class="resc">Raoul Tibes, et al. J Hematol Oncol. 2014;7:18-18.</div><p class="links"><a class="dblinks" href="/pubmed/24598114" ref="ordinalpos=1126&amp;ncbi_uid=5992103&amp;link_uid=24598114">Citation</a><a class="dblinks" href="/pmc/articles/PMC3975838/" ref="ordinalpos=1126&amp;ncbi_uid=5992103&amp;link_uid=PMC3975838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1127.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2646135/figure/pgen-1000408-g007/" ref="ordinalpos=1127&amp;ncbi_uid=1684253&amp;link_uid=PMC2646135" href="/pmc/articles/PMC2646135/"><img src="/pmc/articles/instance/2646135/bin/pgen.1000408.g007.gif" src-large="/pmc/articles/instance/2646135/bin/pgen.1000408.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2646135/" ref="ordinalpos=1127&amp;ncbi_uid=1684253&amp;link_uid=PMC2646135" image-link="/pmc/articles/PMC2646135/figure/pgen-1000408-g007/" class="imagepopup">Figure 7. A glucose-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in fission yeast longevity..  From: Pro-Aging Effects of Glucose <span class="highlight" style="background-color:">Signaling</span> through a G Protein-Coupled Glucose Receptor in Fission Yeast. </a></p><div class="supp"><p class="details"><div>Glucose availability is sensed through two major pathways in fission yeast, the Git3/PKA <span class="highlight" style="background-color:">pathway</span> and the glucose repression <span class="highlight" style="background-color:">pathway</span>. Each of these pathways regulates mitochondrial functions. In the present study, we underlined the importance of Git3/PKA dependent <span class="highlight" style="background-color:">signaling</span> in longevity versus the glucose repression <span class="highlight" style="background-color:">pathway</span> and the metabolic effect of glycolysis. Git3/PKA signal inhibits mitochondrial respiration, oxidative stress response and promotes ROS formation and pro-aging phenotypes. Discontinuous line represents functions inferred by homology with <i>S. cerevisiae</i>. Question mark represents the possible role of mitochondria in aging besides ROS production.</div></p></div><div class="aux"><div class="resc">Antoine E. Roux, et al. PLoS Genet. 2009 March;5(3):e1000408.</div><p class="links"><a class="dblinks" href="/pubmed/19266076" ref="ordinalpos=1127&amp;ncbi_uid=1684253&amp;link_uid=19266076">Citation</a><a class="dblinks" href="/pmc/articles/PMC2646135/" ref="ordinalpos=1127&amp;ncbi_uid=1684253&amp;link_uid=PMC2646135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1128.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1779520/figure/F3/" ref="ordinalpos=1128&amp;ncbi_uid=868459&amp;link_uid=PMC1779520" href="/pmc/articles/PMC1779520/"><img src="/pmc/articles/instance/1779520/bin/gb-2006-7-5-r42-3.gif" src-large="/pmc/articles/instance/1779520/bin/gb-2006-7-5-r42-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1779520/" ref="ordinalpos=1128&amp;ncbi_uid=868459&amp;link_uid=PMC1779520" image-link="/pmc/articles/PMC1779520/figure/F3/" class="imagepopup">Figure 3.  From: PI3K <span class="highlight" style="background-color:">signaling</span> and miRNA expression during the response of quiescent human fibroblasts to distinct proliferative stimuli. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways activated by serum and GF treatments. Major components from three pathways, the PI3K <span class="highlight" style="background-color:">pathway</span>, the JAK/STAT <span class="highlight" style="background-color:">pathway</span> and the MAP kinase <span class="highlight" style="background-color:">pathway</span> are indicated. Class I genes with consistent expression profiles across all treatments in our experiments are indicated by red (up-regulated) or green (down-regulated) at their appropriate position in each of the pathways.</div></p></div><div class="aux"><div class="resc">Jian Gu, et al. Genome Biol. 2006;7(5):R42-R42.</div><p class="links"><a class="dblinks" href="/pubmed/16737555" ref="ordinalpos=1128&amp;ncbi_uid=868459&amp;link_uid=16737555">Citation</a><a class="dblinks" href="/pmc/articles/PMC1779520/" ref="ordinalpos=1128&amp;ncbi_uid=868459&amp;link_uid=PMC1779520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1129.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4107957/figure/F1/" ref="ordinalpos=1129&amp;ncbi_uid=6468703&amp;link_uid=PMC4107957" href="/pmc/articles/PMC4107957/"><img src="/pmc/articles/instance/4107957/bin/fimmu-05-00352-g001.gif" src-large="/pmc/articles/instance/4107957/bin/fimmu-05-00352-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4107957/" ref="ordinalpos=1129&amp;ncbi_uid=6468703&amp;link_uid=PMC4107957" image-link="/pmc/articles/PMC4107957/figure/F1/" class="imagepopup">Figure 1.  From: Toll-Like Receptors and Prostate Cancer. </a></p><div class="supp"><p class="details"><div><b>Toll-like receptors and TLR-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. TLR1 and TLR6 recognize their ligands as heterodimers with TLR2. For TLR4, MD2, and CD14 are required for LPS recognition and <span class="highlight" style="background-color:">signaling</span>. TLR3, TLR4, TLR5, TLR7, and TLR9 are currently thought to deliver their signal by forming homodimers after interacting with their ligands. TLR3, TLR7/8, and TLR9 are intracellular TLRs and are involved in the recognition of nucleic acids. Most TLRs, except for TLR3, signal through MyD88 <span class="highlight" style="background-color:">pathway</span> to activate NF-κB and AP1. TLR3 and TLR4 can signal through MyD88-independent <span class="highlight" style="background-color:">pathway</span> (TRIF <span class="highlight" style="background-color:">pathway</span>) to activate INF-β.</div></p></div><div class="aux"><div class="resc">Shu Zhao, et al. Front Immunol. 2014;5:352.</div><p class="links"><a class="dblinks" href="/pubmed/25101092" ref="ordinalpos=1129&amp;ncbi_uid=6468703&amp;link_uid=25101092">Citation</a><a class="dblinks" href="/pmc/articles/PMC4107957/" ref="ordinalpos=1129&amp;ncbi_uid=6468703&amp;link_uid=PMC4107957">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1130.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285217/figure/F0001/" ref="ordinalpos=1130&amp;ncbi_uid=3823733&amp;link_uid=PMC3285217" href="/pmc/articles/PMC3285217/"><img src="/pmc/articles/instance/3285217/bin/JOM-4-10368-g001.gif" src-large="/pmc/articles/instance/3285217/bin/JOM-4-10368-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285217/" ref="ordinalpos=1130&amp;ncbi_uid=3823733&amp;link_uid=PMC3285217" image-link="/pmc/articles/PMC3285217/figure/F0001/" class="imagepopup">Fig. 1.  From: The Drosophila melanogaster host model. </a></p><div class="supp"><p class="details"><div><i>Drosophila</i> <span class="highlight" style="background-color:">signaling</span> pathways that regulate humoral responses. The Toll <span class="highlight" style="background-color:">pathway</span> regulates the expression of genes in response to the detection of Gram-positive type PPG or fungal β-1-3-glucans or activation by some fungal and bacterial proteases. The Imd <span class="highlight" style="background-color:">pathway</span> regulates the expression of genes in response to the detection of Gram-negative type PPG. The JAK/STAT <span class="highlight" style="background-color:">pathway</span> regulates the expression of genes in response to Upd3 <span class="highlight" style="background-color:">signaling</span> by plasmatocytes. Mammalian homologs of <span class="highlight" style="background-color:">pathway</span> components are shown in blue. Negative regulators of the pathways are not shown.</div></p></div><div class="aux"><div class="resc">Christina O. Igboin, et al. J Oral Microbiol. 2012;4:10.3402/jom.v4i0.10368.</div><p class="links"><a class="dblinks" href="/pubmed/22368770" ref="ordinalpos=1130&amp;ncbi_uid=3823733&amp;link_uid=22368770">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285217/" ref="ordinalpos=1130&amp;ncbi_uid=3823733&amp;link_uid=PMC3285217">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1131.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3236823/figure/Fig3/" ref="ordinalpos=1131&amp;ncbi_uid=3730501&amp;link_uid=PMC3236823" href="/pmc/articles/PMC3236823/"><img src="/pmc/articles/instance/3236823/bin/239_2011_9469_Fig3_HTML.gif" src-large="/pmc/articles/instance/3236823/bin/239_2011_9469_Fig3_HTML.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3236823/" ref="ordinalpos=1131&amp;ncbi_uid=3730501&amp;link_uid=PMC3236823" image-link="/pmc/articles/PMC3236823/figure/Fig3/" class="imagepopup">Fig. 3.  From: Evolution of the TOR <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Evolutionary reconstruction of the ancestral TOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> based on the evolutionary reconstruction of the AGC kinase ancestral genes of S6K, RSK1, PKB and SGK1. <b>a</b> Simplified representation of the phylogenetic tree of the AGC kinases (see Figure S3). Species indication: Hs <i>Homo sapiens</i>, Sc <i>S. cerevisiae</i>, Sp <i>S. pombe</i>, At <i>A. thaliana</i>. <b>b</b> Reconstruction of TOR <span class="highlight" style="background-color:">signaling</span> in the animal ancestor. An AGC kinase duplicated in the ancestor of animals and choanoflagellates to give rise to PKB and SGK. SGK lost the ability to inhibit TSC2. New <span class="highlight" style="background-color:">signaling</span> inputs were invented in the animal ancestor, among which is the insulin <span class="highlight" style="background-color:">signaling</span>. <b>c</b> TOR <span class="highlight" style="background-color:">signaling</span> in the ancestor of animals and fungi. An AGC kinase duplicated in the ancestor of animals and fungi (Opisthokont ancestor) that give rise to S6K and RSK. The duplication was followed by subsequent sub-functionalization of ancestral functions between the two paralogs. <b>d</b> TOR <span class="highlight" style="background-color:">signaling</span> in LECA. The duplication of the ancestral AGC kinase that give rise to the RSK-S6K and PKB-SGK ancestral precursor genes corresponds to the differential activation by either TORC1 or TORC2, respectively. <b>e</b> Reconstruction of pre-LECA TOR <span class="highlight" style="background-color:">signaling</span>. The shared subunits TOR and LST8 of both TOR complexes suggest that before LECA there was at one point only one proto-TOR complex. As the AGC kinases also share single ancestry we can reduce the complexity of the TOR <span class="highlight" style="background-color:">pathway</span> in pre-LECA even further</div></p></div><div class="aux"><div class="resc">Teunis J. P. van Dam, et al. J Mol Evol. 2011 October;73(3-4):209-220.</div><p class="links"><a class="dblinks" href="/pubmed/22057117" ref="ordinalpos=1131&amp;ncbi_uid=3730501&amp;link_uid=22057117">Citation</a><a class="dblinks" href="/pmc/articles/PMC3236823/" ref="ordinalpos=1131&amp;ncbi_uid=3730501&amp;link_uid=PMC3236823">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1132.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3117218/figure/F2/" ref="ordinalpos=1132&amp;ncbi_uid=3026953&amp;link_uid=PMC3117218" href="/pmc/articles/PMC3117218/"><img src="/pmc/articles/instance/3117218/bin/nihms301891f2.gif" src-large="/pmc/articles/instance/3117218/bin/nihms301891f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3117218/" ref="ordinalpos=1132&amp;ncbi_uid=3026953&amp;link_uid=PMC3117218" image-link="/pmc/articles/PMC3117218/figure/F2/" class="imagepopup">Fig. 2. Scaffold proteins can mediate <span class="highlight" style="background-color:">pathway</span> regulation and feedback to shape complex <span class="highlight" style="background-color:">signaling</span> responses.  From: Scaffold Proteins: Hubs for Controlling the Flow of Cellular Information. </a></p><div class="supp"><p class="details"><div>A) Scaffold proteins are analogous to circuit boards - modular platforms that wire together components and direct the flow of information - and can program complex <span class="highlight" style="background-color:">signaling</span> behaviors. B) Scaffold proteins function to wire <span class="highlight" style="background-color:">pathway</span> input and output through alternative possible routes. C) Scaffold proteins can mediate branching of pathways to multiple outputs. D) Scaffold proteins are themselves the targets of regulation. In T-cell <span class="highlight" style="background-color:">signaling</span>, activation of the T-cell receptor causes phosphorylation of the LAT and Slp76 scaffolds by Zap70, and phosphorylation-dependent recruitment of substrates leads to PLCγ activation and PIP2 hydrolysis. E) Scaffold proteins can be the target of feedback phosphorylation that tunes <span class="highlight" style="background-color:">pathway</span> responses. Feedback phosphorylation of the KSR scaffold by activated ERK blocks Raf (MAPKKK) binding and attenuates MEK activation, thereby decreasing <span class="highlight" style="background-color:">pathway</span> output. Plot adapted from (24).</div></p></div><div class="aux"><div class="resc">Matthew C. Good, et al. Science. ;332(6030):680-686.</div><p class="links"><a class="dblinks" href="/pubmed/21551057" ref="ordinalpos=1132&amp;ncbi_uid=3026953&amp;link_uid=21551057">Citation</a><a class="dblinks" href="/pmc/articles/PMC3117218/" ref="ordinalpos=1132&amp;ncbi_uid=3026953&amp;link_uid=PMC3117218">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1133.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3708163/figure/F1/" ref="ordinalpos=1133&amp;ncbi_uid=6328472&amp;link_uid=PMC3708163" href="/pmc/articles/PMC3708163/"><img src="/pmc/articles/instance/3708163/bin/zjw0071344780001.gif" src-large="/pmc/articles/instance/3708163/bin/zjw0071344780001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3708163/" ref="ordinalpos=1133&amp;ncbi_uid=6328472&amp;link_uid=PMC3708163" image-link="/pmc/articles/PMC3708163/figure/F1/" class="imagepopup">Fig. 1. From: Identification of Protein Interactions Involved in Cellular <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Protein interactions mediate cellular responses.</b> A hypothetical signal transduction <span class="highlight" style="background-color:">pathway</span> where sequential and transient interactions between upstream and downstream components of this <span class="highlight" style="background-color:">pathway</span> are active. All depicted interactions both between <span class="highlight" style="background-color:">signaling</span> proteins and with other cellular constituents are critical for proper <span class="highlight" style="background-color:">signaling</span>. They also are potential targets for disease causing alterations as well as for therapeutic targeting.</div></p></div><div class="aux"><div class="resc">Jukka Westermarck, et al. Mol Cell Proteomics. 2013 July;12(7):1752-1763.</div><p class="links"><a class="dblinks" href="/pubmed/23481661" ref="ordinalpos=1133&amp;ncbi_uid=6328472&amp;link_uid=23481661">Citation</a><a class="dblinks" href="/pmc/articles/PMC3708163/" ref="ordinalpos=1133&amp;ncbi_uid=6328472&amp;link_uid=PMC3708163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1134.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3050595/figure/F5/" ref="ordinalpos=1134&amp;ncbi_uid=3241642&amp;link_uid=PMC3050595" href="/pmc/articles/PMC3050595/"><img src="/pmc/articles/instance/3050595/bin/nihms275389f5a.gif" src-large="/pmc/articles/instance/3050595/bin/nihms275389f5a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3050595/figure/F5/" ref="ordinalpos=1134&amp;ncbi_uid=3241642&amp;link_uid=PMC3050595" href="/pmc/articles/PMC3050595/"><img src="/pmc/articles/instance/3050595/bin/nihms275389f5b.gif" src-large="/pmc/articles/instance/3050595/bin/nihms275389f5b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3050595/" ref="ordinalpos=1134&amp;ncbi_uid=3241642&amp;link_uid=PMC3050595" image-link="/pmc/articles/PMC3050595/figure/F5/" class="imagepopup">Figure 5. Wnt <span class="highlight" style="background-color:">pathway</span> in renal FGF23 <span class="highlight" style="background-color:">signaling</span>.  From: Altered Renal FGF23-Mediated Activity Involving MAPK and Wnt: Effects of the Hyp Mutation. </a></p><div class="supp"><p class="details"><div><i>a)</i> Strong p-β-catenin <span class="highlight" style="background-color:">signaling</span> was detected in the glomeruli of all animals (green), separate from the DCT, which was demarcated using anti-NCC IF (red). <i>b)</i> p-β-catenin <span class="highlight" style="background-color:">signaling</span> was tested by IF in vehicle and FGF23-injected mice at 10 min following injection. p-β-catenin <span class="highlight" style="background-color:">signaling</span> was not detected in the vehicle injected animals. In the FGF23-injected animals, p-β-catenin (red) was detected in the distal tubule, co-localized with Klotho (green). This activity did not co-localize with Npt2a in the renal PT (right column).</div></p></div><div class="aux"><div class="resc">Emily G. Farrow, et al. J Endocrinol. ;207(1):67-75.</div><p class="links"><a class="dblinks" href="/pubmed/20675303" ref="ordinalpos=1134&amp;ncbi_uid=3241642&amp;link_uid=20675303">Citation</a><a class="dblinks" href="/pmc/articles/PMC3050595/" ref="ordinalpos=1134&amp;ncbi_uid=3241642&amp;link_uid=PMC3050595">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1135.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3472464/figure/F6/" ref="ordinalpos=1135&amp;ncbi_uid=5339967&amp;link_uid=PMC3472464" href="/pmc/articles/PMC3472464/"><img src="/pmc/articles/instance/3472464/bin/zh70191237630006.gif" src-large="/pmc/articles/instance/3472464/bin/zh70191237630006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3472464/" ref="ordinalpos=1135&amp;ncbi_uid=5339967&amp;link_uid=PMC3472464" image-link="/pmc/articles/PMC3472464/figure/F6/" class="imagepopup">Fig. 6. From: Constructing the angiome: a global angiogenesis protein interaction network. </a></p><div class="supp"><p class="details"><div>Angiogenesis regulators and associated proteins. Based on Supplementary Tables S8 and S9, we divided these 171 angiogenesis regulator proteins into 11 categories: (<i>A</i>) vascular endothelial growth factor <span class="highlight" style="background-color:">signaling</span>, (<i>B</i>) fibroblast growth factor <span class="highlight" style="background-color:">signaling</span>, (<i>C</i>) epidermal growth factor <span class="highlight" style="background-color:">signaling</span>, (<i>D</i>) transforming growth factor <span class="highlight" style="background-color:">signaling</span>, (<i>E</i>) insulin-like growth factor <span class="highlight" style="background-color:">signaling</span>, (<i>F</i>) platelet-derived growth factor <span class="highlight" style="background-color:">signaling</span>, (<i>G</i>) inflammation and immune response, (<i>H</i>) other cytokines, (<i>I</i>) focal adhesion, (<i>J</i>) Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, (<i>K</i>) others.</div></p></div><div class="aux"><div class="resc">Liang-Hui Chu, et al. Physiol Genomics. 2012 October 1;44(19):915-924.</div><p class="links"><a class="dblinks" href="/pubmed/22911453" ref="ordinalpos=1135&amp;ncbi_uid=5339967&amp;link_uid=22911453">Citation</a><a class="dblinks" href="/pmc/articles/PMC3472464/" ref="ordinalpos=1135&amp;ncbi_uid=5339967&amp;link_uid=PMC3472464">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1136.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3824689/figure/fig2/" ref="ordinalpos=1136&amp;ncbi_uid=5464495&amp;link_uid=PMC3824689" href="/pmc/articles/PMC3824689/"><img src="/pmc/articles/instance/3824689/bin/cddis2013380f2.gif" src-large="/pmc/articles/instance/3824689/bin/cddis2013380f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3824689/" ref="ordinalpos=1136&amp;ncbi_uid=5464495&amp;link_uid=PMC3824689" image-link="/pmc/articles/PMC3824689/figure/fig2/" class="imagepopup">Figure 2.  From: IBP-mediated suppression of autophagy promotes growth and metastasis of breast cancer cells via activating mTORC2/Akt/FOXO3a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>IBP activated the mTORC1 and mTORC2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in breast cancer cells. (<b>a</b> and <b>b</b>) qRT-PCR analysis of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>-associated gene expression levels in MDA-MB-231 and MDA-MB-468 cells with different levels of IBP. (<b>c</b>) Those cells were analyzed for mTORC2 activity by immunoblotting for levels of p-mTOR Ser2481, p-Akt Ser473 and p-FOXO3a Thr32. (<b>d</b>) Cells were analyzed for mTORC1 activity by immunoblotting for levels of p-mTOR Ser2448, p-P70S6K Thr389 and p-4EBP1 Thr37/46</div></p></div><div class="aux"><div class="resc">S Chen, et al. Cell Death Dis. 2013 October;4(10):e842.</div><p class="links"><a class="dblinks" href="/pubmed/24113176" ref="ordinalpos=1136&amp;ncbi_uid=5464495&amp;link_uid=24113176">Citation</a><a class="dblinks" href="/pmc/articles/PMC3824689/" ref="ordinalpos=1136&amp;ncbi_uid=5464495&amp;link_uid=PMC3824689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1137.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2529314/figure/F8/" ref="ordinalpos=1137&amp;ncbi_uid=1578183&amp;link_uid=PMC2529314" href="/pmc/articles/PMC2529314/"><img src="/pmc/articles/instance/2529314/bin/1471-2164-9-369-8.gif" src-large="/pmc/articles/instance/2529314/bin/1471-2164-9-369-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2529314/" ref="ordinalpos=1137&amp;ncbi_uid=1578183&amp;link_uid=PMC2529314" image-link="/pmc/articles/PMC2529314/figure/F8/" class="imagepopup">Figure 8.  From: Peripheral arterial occlusive disease: Global gene expression analyses suggest a major role for immune and inflammatory responses. </a></p><div class="supp"><p class="details"><div><b>Over-representation of Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes</b>. Analysis of over-representation of differentially expressed genes in <span class="highlight" style="background-color:">pathway</span> from KEGG. The Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is illustrated with significantly regulated genes highlighted.</div></p></div><div class="aux"><div class="resc">Shijun Fu, et al. BMC Genomics. 2008;9:369-369.</div><p class="links"><a class="dblinks" href="/pubmed/18673543" ref="ordinalpos=1137&amp;ncbi_uid=1578183&amp;link_uid=18673543">Citation</a><a class="dblinks" href="/pmc/articles/PMC2529314/" ref="ordinalpos=1137&amp;ncbi_uid=1578183&amp;link_uid=PMC2529314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1138.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847091/figure/pone-0082022-g005/" ref="ordinalpos=1138&amp;ncbi_uid=5565745&amp;link_uid=PMC3847091" href="/pmc/articles/PMC3847091/"><img src="/pmc/articles/instance/3847091/bin/pone.0082022.g005.gif" src-large="/pmc/articles/instance/3847091/bin/pone.0082022.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847091/" ref="ordinalpos=1138&amp;ncbi_uid=5565745&amp;link_uid=PMC3847091" image-link="/pmc/articles/PMC3847091/figure/pone-0082022-g005/" class="imagepopup">Figure 5. Diagram summarizes the inter-regulation between ASPP2 and RAS..  From: Phosphorylation of ASPP2 by RAS/MAPK <span class="highlight" style="background-color:">Pathway</span> Is Critical for Its Full Pro-Apoptotic Function. </a></p><div class="supp"><p class="details"><div>ASPP2 binds active RAS at the plasma membrane, thereby increasing RAS <span class="highlight" style="background-color:">signaling</span> to its downstream <span class="highlight" style="background-color:">pathway</span> effectors Raf/MAPK. Activated MAPK phosphorylates ASPP2 which can then relocate to the nucleus and activate p53 pro-apoptotic <span class="highlight" style="background-color:">signaling</span>. </div></p></div><div class="aux"><div class="resc">Nadia Godin-Heymann, et al. PLoS One. 2013;8(12):e82022.</div><p class="links"><a class="dblinks" href="/pubmed/24312625" ref="ordinalpos=1138&amp;ncbi_uid=5565745&amp;link_uid=24312625">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847091/" ref="ordinalpos=1138&amp;ncbi_uid=5565745&amp;link_uid=PMC3847091">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1139.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1456930/figure/f5/" ref="ordinalpos=1139&amp;ncbi_uid=633623&amp;link_uid=PMC1456930" href="/pmc/articles/PMC1456930/"><img src="/pmc/articles/instance/1456930/bin/7601062f5.gif" src-large="/pmc/articles/instance/1456930/bin/7601062f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1456930/" ref="ordinalpos=1139&amp;ncbi_uid=633623&amp;link_uid=PMC1456930" image-link="/pmc/articles/PMC1456930/figure/f5/" class="imagepopup">Figure 5.  From: An aging <span class="highlight" style="background-color:">pathway</span> controls the TrkA to p75NTR receptor switch and amyloid ?-peptide generation. </a></p><div class="supp"><p class="details"><div>A model of the molecular pathways controlling Aβ generation during aging. Genetic and pharmacological manipulations described in the text are indicated in red or blue next to the steps in the <span class="highlight" style="background-color:">pathway</span> that they affect. Manipulations indicated in red inhibit <span class="highlight" style="background-color:">signaling</span>, while the blue manipulation amplifies <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Claudio Costantini, et al. EMBO J. 2006 May 3;25(9):1997-2006.</div><p class="links"><a class="dblinks" href="/pubmed/16619032" ref="ordinalpos=1139&amp;ncbi_uid=633623&amp;link_uid=16619032">Citation</a><a class="dblinks" href="/pmc/articles/PMC1456930/" ref="ordinalpos=1139&amp;ncbi_uid=633623&amp;link_uid=PMC1456930">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1140.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3556819/figure/F17/" ref="ordinalpos=1140&amp;ncbi_uid=4560794&amp;link_uid=PMC3556819" href="/pmc/articles/PMC3556819/"><img src="/pmc/articles/instance/3556819/bin/nihms-435125-f0017.gif" src-large="/pmc/articles/instance/3556819/bin/nihms-435125-f0017.jpg" alt="Figure 17" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3556819/" ref="ordinalpos=1140&amp;ncbi_uid=4560794&amp;link_uid=PMC3556819" image-link="/pmc/articles/PMC3556819/figure/F17/" class="imagepopup">Figure 17. Potential mechanisms for regulation of cadherin biology by tyrosine kinase <span class="highlight" style="background-color:">signaling</span>.  From: Tissue organization by cadherin adhesion molecules: dynamic molecular and cellular mechanisms of morphogenetic regulation. </a></p><div class="supp"><p class="details"><div>A) Uncoupling: tyrosine phosphorylation of the cadherin cytoplasmic tail and/or associated proteins dissociates catenins leading to disassembly of the cadherin molecular complex.<br />B) Scaffolding <span class="highlight" style="background-color:">signaling</span> complexes: tyrosine phosphorylation generates binding sites for association of <span class="highlight" style="background-color:">signaling</span> molecules such as C-terminal kinase (CSK, to VE-cadherin cytoplasmic tail) or PTP1B (to β-catenin).<br />C) Cadherin-based <span class="highlight" style="background-color:">signaling</span>: recruitment of tyrosine kinases as part of a cadherin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates control of downstream effectors, such as the cytoskeletal regulator, cortactin.</div></p></div><div class="aux"><div class="resc">Carien M. Niessen, et al. Physiol Rev. ;91(2):691-731.</div><p class="links"><a class="dblinks" href="/pubmed/21527735" ref="ordinalpos=1140&amp;ncbi_uid=4560794&amp;link_uid=21527735">Citation</a><a class="dblinks" href="/pmc/articles/PMC3556819/" ref="ordinalpos=1140&amp;ncbi_uid=4560794&amp;link_uid=PMC3556819">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1141.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3353658/figure/F3/" ref="ordinalpos=1141&amp;ncbi_uid=4006090&amp;link_uid=PMC3353658" href="/pmc/articles/PMC3353658/"><img src="/pmc/articles/instance/3353658/bin/nihms373948f3.gif" src-large="/pmc/articles/instance/3353658/bin/nihms373948f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3353658/" ref="ordinalpos=1141&amp;ncbi_uid=4006090&amp;link_uid=PMC3353658" image-link="/pmc/articles/PMC3353658/figure/F3/" class="imagepopup">Figure 3. Role of GRP78 and its isoform GRP78va in regulating UPR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and cell survival.  From: Beyond the endoplasmic reticulum: atypical GRP78 in cell viability, <span class="highlight" style="background-color:">signaling</span> and therapeutic targeting. </a></p><div class="supp"><p class="details"><div>ER stress induces <i>Grp78</i> transcription, resulting in upregulation of GRP78 and GRP78va, the latter is generated by alternative splicing. In the ER lumen, GRP78, in association with P58<sup>IPK</sup> and other co-chaperones, enhances protein folding and degradation of misfolded proteins (ERAD). Release of GRP78 from PERK, IRE1 and ATF6 induces the UPR <span class="highlight" style="background-color:">signaling</span>. PERK activation leads to eIF2α phosphorylation and translation attenuation, contributing to cell survival. In the cytosol, GRP78va enforces PERK <span class="highlight" style="background-color:">signaling</span> by inactivation of the cytosolic P58<sup>IPK</sup>.</div></p></div><div class="aux"><div class="resc">Min Ni, et al. Biochem J. ;434(2):181-188.</div><p class="links"><a class="dblinks" href="/pubmed/21309747" ref="ordinalpos=1141&amp;ncbi_uid=4006090&amp;link_uid=21309747">Citation</a><a class="dblinks" href="/pmc/articles/PMC3353658/" ref="ordinalpos=1141&amp;ncbi_uid=4006090&amp;link_uid=PMC3353658">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1142.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4261972/figure/f06/" ref="ordinalpos=1142&amp;ncbi_uid=6951342&amp;link_uid=PMC4261972" href="/pmc/articles/PMC4261972/"><img src="/pmc/articles/instance/4261972/bin/gmb-37-716-g006.gif" src-large="/pmc/articles/instance/4261972/bin/gmb-37-716-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4261972/" ref="ordinalpos=1142&amp;ncbi_uid=6951342&amp;link_uid=PMC4261972" image-link="/pmc/articles/PMC4261972/figure/f06/" class="imagepopup">Figure 6.  From: PCI-24781 down-regulates EZH2 expression and then promotes glioma apoptosis by suppressing the PIK3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Treatment with PCI-24781 attenuated the expression of proteins involved in the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The blots are representative of 3 experiments.</div></p></div><div class="aux"><div class="resc">Wei Zhang, et al. Genet Mol Biol. 2014 October;37(4):716-724.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1142&amp;ncbi_uid=6951342&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4261972/" ref="ordinalpos=1142&amp;ncbi_uid=6951342&amp;link_uid=PMC4261972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1143.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657487/figure/f1-nbtm-2-3-0185/" ref="ordinalpos=1143&amp;ncbi_uid=1701851&amp;link_uid=PMC2657487" href="/pmc/articles/PMC2657487/"><img src="/pmc/articles/instance/2657487/bin/nbtm-2-3-0185f1.gif" src-large="/pmc/articles/instance/2657487/bin/nbtm-2-3-0185f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657487/" ref="ordinalpos=1143&amp;ncbi_uid=1701851&amp;link_uid=PMC2657487" image-link="/pmc/articles/PMC2657487/figure/f1-nbtm-2-3-0185/" class="imagepopup">FIGURE 1.  From: Low-Dose Radiation and Genotoxic Chemicals Can Protect Against Stochastic Biological Effects. </a></p><div class="supp"><p class="details"><div>Schematic representation of intercellular <span class="highlight" style="background-color:">signaling</span> associated with the PAM process as specified by researchers at the University of Freiburg (Bauer, 2000; Engelmann and Bauer, 2000). The <span class="highlight" style="background-color:">signaling</span> indicated here is for the HOCl/hydroxyl radical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is one of two <span class="highlight" style="background-color:">signaling</span> pathways. The other NO/peroxynitrite <span class="highlight" style="background-color:">pathway</span> is less complicated (Bauer, 2000) and is not presented. The figure is explained in the text.</div></p></div><div class="aux"><div class="resc">Bobby R. Scott, et al. Nonlinearity Biol Toxicol Med. 2004 July;2(3):185-211.</div><p class="links"><a class="dblinks" href="/pubmed/19330143" ref="ordinalpos=1143&amp;ncbi_uid=1701851&amp;link_uid=19330143">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657487/" ref="ordinalpos=1143&amp;ncbi_uid=1701851&amp;link_uid=PMC2657487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1144.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3207648/figure/fig06/" ref="ordinalpos=1144&amp;ncbi_uid=3468545&amp;link_uid=PMC3207648" href="/pmc/articles/PMC3207648/"><img src="/pmc/articles/instance/3207648/bin/pnas.1108180108fig06.gif" src-large="/pmc/articles/instance/3207648/bin/pnas.1108180108fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3207648/" ref="ordinalpos=1144&amp;ncbi_uid=3468545&amp;link_uid=PMC3207648" image-link="/pmc/articles/PMC3207648/figure/fig06/" class="imagepopup">Fig. 6. From: Heightened uterine mammalian target of rapamycin complex 1 (mTORC1) <span class="highlight" style="background-color:">signaling</span> provokes preterm birth in mice. </a></p><div class="supp"><p class="details"><div>The proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to preterm birth. Uterine deletion of p53 in mice results in increased PI3K-Akt <span class="highlight" style="background-color:">signaling</span>, which suppresses TSC activity by phosphorylating TSC2. Thus, TSC's inhibition on mTORC1 <span class="highlight" style="background-color:">signaling</span> is released, reflecting higher levels of pS6, as well as p21 levels. These changes give rise to heightened cellular senescence associated with increased PGF<sub>2α</sub> generation via the COX2-PGFS <span class="highlight" style="background-color:">pathway</span>, driving preterm birth.</div></p></div><div class="aux"><div class="resc">Yasushi Hirota, et al. Proc Natl Acad Sci U S A. 2011 November 1;108(44):18073-18078.</div><p class="links"><a class="dblinks" href="/pubmed/22025690" ref="ordinalpos=1144&amp;ncbi_uid=3468545&amp;link_uid=22025690">Citation</a><a class="dblinks" href="/pmc/articles/PMC3207648/" ref="ordinalpos=1144&amp;ncbi_uid=3468545&amp;link_uid=PMC3207648">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1145.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2775047/figure/F4/" ref="ordinalpos=1145&amp;ncbi_uid=2091797&amp;link_uid=PMC2775047" href="/pmc/articles/PMC2775047/"><img src="/pmc/articles/instance/2775047/bin/nihms44974f4.gif" src-large="/pmc/articles/instance/2775047/bin/nihms44974f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2775047/" ref="ordinalpos=1145&amp;ncbi_uid=2091797&amp;link_uid=PMC2775047" image-link="/pmc/articles/PMC2775047/figure/F4/" class="imagepopup">Figure 4. A model depicting multiple immune outcomes of TLR4/cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in BECs.  From: TLR Mediated Immune Responses in the Urinary Tract. </a></p><div class="supp"><p class="details"><div>TLR4/cAMP <span class="highlight" style="background-color:">signaling</span> triggers rapid cytokine/chemokine secretion, inhibition of UPEC invasion, and UPEC expulsion from infected BECs.</div></p></div><div class="aux"><div class="resc">Jeongmin Song, et al. Curr Opin Microbiol. ;11(1):66-73.</div><p class="links"><a class="dblinks" href="/pubmed/18243043" ref="ordinalpos=1145&amp;ncbi_uid=2091797&amp;link_uid=18243043">Citation</a><a class="dblinks" href="/pmc/articles/PMC2775047/" ref="ordinalpos=1145&amp;ncbi_uid=2091797&amp;link_uid=PMC2775047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1146.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3374768/figure/pone-0039066-g011/" ref="ordinalpos=1146&amp;ncbi_uid=4072311&amp;link_uid=PMC3374768" href="/pmc/articles/PMC3374768/"><img src="/pmc/articles/instance/3374768/bin/pone.0039066.g011.gif" src-large="/pmc/articles/instance/3374768/bin/pone.0039066.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3374768/" ref="ordinalpos=1146&amp;ncbi_uid=4072311&amp;link_uid=PMC3374768" image-link="/pmc/articles/PMC3374768/figure/pone-0039066-g011/" class="imagepopup">Figure 11. Schematic model summarising our findings on the role of EGFR/erbB2 <span class="highlight" style="background-color:">signaling</span> in diabetes-induced cardiac dysfunction..  From: Activation of EGFR/ERBB2 via Pathways Involving ERK1/2, P38 MAPK, AKT and FOXO Enhances Recovery of Diabetic Hearts from Ischemia-Reperfusion Injury. </a></p><div class="supp"><p class="details"><div>Diabetes and/or hyperglycemia via attenuation of the EGFR/erbB2 <span class="highlight" style="background-color:">signaling</span> and through subsequent modulation of downstream effectors such as ERK1/2, p38 MAPK or AKT/FOXO can lead to cardiac dysfunction. The effects of diabetes on EGFR/erbB2 <span class="highlight" style="background-color:">pathway</span> are exacerbated by blockade of this <span class="highlight" style="background-color:">pathway</span> by AG1478 or AG825 which leads to worsening cardiac recovery from I/R. However, the inhibitory effects of diabetes on EGFR/ErbB2 <span class="highlight" style="background-color:">pathway</span> may be opposed by administering EGF that also leads to improved cardiac function. The Angiotensin II (Ang II)/AT<sub>1</sub> receptor <span class="highlight" style="background-color:">pathway</span> can also activate EGFR/erbB2 <span class="highlight" style="background-color:">pathway</span> that can be blocked by Losartan. Co-administration of EGF with Losartan attenuates losartan-mediated EGFR blockade and improves cardiac function in diabetes beyond that attained by either drug alone.</div></p></div><div class="aux"><div class="resc">Saghir Akhtar, et al. PLoS One. 2012;7(6):e39066.</div><p class="links"><a class="dblinks" href="/pubmed/22720029" ref="ordinalpos=1146&amp;ncbi_uid=4072311&amp;link_uid=22720029">Citation</a><a class="dblinks" href="/pmc/articles/PMC3374768/" ref="ordinalpos=1146&amp;ncbi_uid=4072311&amp;link_uid=PMC3374768">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1147.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2688657/figure/fig1/" ref="ordinalpos=1147&amp;ncbi_uid=1859189&amp;link_uid=PMC2688657" href="/pmc/articles/PMC2688657/"><img src="/pmc/articles/instance/2688657/bin/JBB2009-707580.001.gif" src-large="/pmc/articles/instance/2688657/bin/JBB2009-707580.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2688657/" ref="ordinalpos=1147&amp;ncbi_uid=1859189&amp;link_uid=PMC2688657" image-link="/pmc/articles/PMC2688657/figure/fig1/" class="imagepopup">Figure 1.  From: Integrating Multiple Microarray Data for Cancer <span class="highlight" style="background-color:">Pathway</span> Analysis Using Bootstrapping K-S Test. </a></p><div class="supp"><p class="details"><div>Antigrowth <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and cluster around SMAD proteins. (a) Antigrowth <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Nodes and edges represent human proteins and protein-protein interactions, respectively. Edges with direction represent a regulatory relation. →  means an activating relation and,  <i>⊣</i> means an inhibitory relation. (b) Cluster around smads. Red edges represent differentially changed relations. Blue edges represent unchanged relations. Red nodes represent tumor suppressor genes, and green nodes represent oncogenes.</div></p></div><div class="aux"><div class="resc">Bing Han, et al. J Biomed Biotechnol. 2009;2009:707580.</div><p class="links"><a class="dblinks" href="/pubmed/19704919" ref="ordinalpos=1147&amp;ncbi_uid=1859189&amp;link_uid=19704919">Citation</a><a class="dblinks" href="/pmc/articles/PMC2688657/" ref="ordinalpos=1147&amp;ncbi_uid=1859189&amp;link_uid=PMC2688657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1148.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3842152/figure/F2/" ref="ordinalpos=1148&amp;ncbi_uid=6344968&amp;link_uid=PMC3842152" href="/pmc/articles/PMC3842152/"><img src="/pmc/articles/instance/3842152/bin/mpphys_sst070_f0002.gif" src-large="/pmc/articles/instance/3842152/bin/mpphys_sst070_f0002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3842152/" ref="ordinalpos=1148&amp;ncbi_uid=6344968&amp;link_uid=PMC3842152" image-link="/pmc/articles/PMC3842152/figure/F2/" class="imagepopup">Figure 2. From: <span class="highlight" style="background-color:">Signaling</span> in Pollen Tube Growth: Crosstalk, Feedback, and Missing Links. </a></p><div class="supp"><p class="details"><div>Simplified Model of the Discussed Mechanical <span class="highlight" style="background-color:">Signaling</span> Networks in Self-Organizing Pollen Tube Growth.<br />For better-elucidated <span class="highlight" style="background-color:">signaling</span> modules, feedback regulations and <span class="highlight" style="background-color:">pathway</span> crosstalks are common. Their regulations of structural networks are partially revealed, whilst still needing further characterization. The role of emerging new pathways, such as hormone and small peptide <span class="highlight" style="background-color:">signaling</span>, remains elusive. For details, see the text.</div></p></div><div class="aux"><div class="resc">Yuefeng Guan, et al. Mol Plant. 2013 July;6(4):1053-1064.</div><p class="links"><a class="dblinks" href="/pubmed/23873928" ref="ordinalpos=1148&amp;ncbi_uid=6344968&amp;link_uid=23873928">Citation</a><a class="dblinks" href="/pmc/articles/PMC3842152/" ref="ordinalpos=1148&amp;ncbi_uid=6344968&amp;link_uid=PMC3842152">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1149.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267860/figure/F8/" ref="ordinalpos=1149&amp;ncbi_uid=4064473&amp;link_uid=PMC3267860" href="/pmc/articles/PMC3267860/"><img src="/pmc/articles/instance/3267860/bin/nihms337420f8.gif" src-large="/pmc/articles/instance/3267860/bin/nihms337420f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267860/" ref="ordinalpos=1149&amp;ncbi_uid=4064473&amp;link_uid=PMC3267860" image-link="/pmc/articles/PMC3267860/figure/F8/" class="imagepopup">Figure 8. Loss of plexin-D1 in direct <span class="highlight" style="background-color:">pathway</span> MSNs increases the number of vGluT2 and GluR1 positive puncta on direct <span class="highlight" style="background-color:">pathway</span> MSNs.  From: Semaphorin 3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> controls <span class="highlight" style="background-color:">pathway</span>-specific synapse formation in the striatum. </a></p><div class="supp"><p class="details"><div><b>(a–b)</b> Single-plane array tomography images of a 100 nm striatal section showing GFP (green) expressed in a sparse subset of direct <span class="highlight" style="background-color:">pathway</span> MSNs, vGluT2 (red), and GluR1 (blue) distribution in the striatum from control <b>(a)</b> and direct <span class="highlight" style="background-color:">pathway</span> <i>plexin-D1</i> mutant mice <b>(b). (c–d)</b> High-magnification images from control <b>(c)</b> and direct <span class="highlight" style="background-color:">pathway</span> <i>plexin-D1</i> mutant mice <b>(d)</b> the puncta distribution on soma (<i>left</i>) and dendrite (<i>right</i>) of direct <span class="highlight" style="background-color:">pathway</span> MSNs (green). Thalamostriatal synapses are defined by the close apposition of presynaptic (vGluT2) and postsynaptic (GluR1) markers (examples highlighted in white circles). In a–c, green indicates the segmented areas of GFP fluorescence that were used to identify sections of direct <span class="highlight" style="background-color:">pathway</span> MSNs. <b>(e)</b> Quantification of the density of putative thalamic synapses formed on direct <span class="highlight" style="background-color:">pathway</span> MSNs in control and direct <span class="highlight" style="background-color:">pathway</span> <i>plexin-D1</i> mutant mice. (*<i>P</i>&lt;0.0001; Mann-Whitney). Error bars denote s.e.m.</div></p></div><div class="aux"><div class="resc">Jun B. Ding, et al. Nat Neurosci. ;15(2):215-223.</div><p class="links"><a class="dblinks" href="/pubmed/22179111" ref="ordinalpos=1149&amp;ncbi_uid=4064473&amp;link_uid=22179111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267860/" ref="ordinalpos=1149&amp;ncbi_uid=4064473&amp;link_uid=PMC3267860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1150.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3461933/figure/F1/" ref="ordinalpos=1150&amp;ncbi_uid=4494293&amp;link_uid=PMC3461933" href="/pmc/articles/PMC3461933/"><img src="/pmc/articles/instance/3461933/bin/JCI66058.f1.gif" src-large="/pmc/articles/instance/3461933/bin/JCI66058.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3461933/" ref="ordinalpos=1150&amp;ncbi_uid=4494293&amp;link_uid=PMC3461933" image-link="/pmc/articles/PMC3461933/figure/F1/" class="imagepopup">Figure 1. Effects of TGF-β <span class="highlight" style="background-color:">signaling</span> in glioblastoma..  From: Looking in the miR-ror: TGF-?-mediated activation of NF-?B in glioma . </a></p><div class="supp"><p class="details"><div>TGF-β <span class="highlight" style="background-color:">signaling</span> is dysregulated on several levels in cancers, including glioblastoma, and mediates many systemic (immunosuppressive and angiogenic) and cellular effects. In contrast to other cancers, the canonical TGF-β <span class="highlight" style="background-color:">pathway</span> (ligand receptor/Smad mediators) is not commonly mutated in glioblastomas. However, modifiers of TGF-β <span class="highlight" style="background-color:">signaling</span> (e.g., FOXO, FOXG1, BF1, and USP15) and interaction with other pathways lead to a shift from primarily tumor-suppressive effects (e.g., antiproliferation) to oncogenesis. Targeting the interactions between TGF-β and NF-κB <span class="highlight" style="background-color:">signaling</span> may offer therapeutic options. Amplification of EGFR, an NF-κB activator, and mutation of <i>NFKBIA</i> (a NF-κB inhibitor) are mutually exclusive and may inform the effects of two miRs that regulate NF-κB activity, miR-182 and miR-30e*. The role of TGF-β and its relationship to other <span class="highlight" style="background-color:">signaling</span> pathways in gliomas contradicts the growth-suppressive effects and alternative <span class="highlight" style="background-color:">signaling</span> effects of TGF-β in noncancerous cells, such as astrocytes. </div></p></div><div class="aux"><div class="resc">Christine E. Eyler, et al. J Clin Invest. 2012 October 1;122(10):3473-3475.</div><p class="links"><a class="dblinks" href="/pubmed/23006324" ref="ordinalpos=1150&amp;ncbi_uid=4494293&amp;link_uid=23006324">Citation</a><a class="dblinks" href="/pmc/articles/PMC3461933/" ref="ordinalpos=1150&amp;ncbi_uid=4494293&amp;link_uid=PMC3461933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1151.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3539548/figure/f1-medscimonit-17-6-ra148/" ref="ordinalpos=1151&amp;ncbi_uid=4824327&amp;link_uid=PMC3539548" href="/pmc/articles/PMC3539548/"><img src="/pmc/articles/instance/3539548/bin/medscimonit-17-6-RA148-g001.gif" src-large="/pmc/articles/instance/3539548/bin/medscimonit-17-6-RA148-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3539548/" ref="ordinalpos=1151&amp;ncbi_uid=4824327&amp;link_uid=PMC3539548" image-link="/pmc/articles/PMC3539548/figure/f1-medscimonit-17-6-ra148/" class="imagepopup">Figure 1.  From: Influenza virus and cell <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Host cell machinery gets activated during influenza infection and leads to trigger <span class="highlight" style="background-color:">Signaling</span> pathways. Influenza is know to produce early and late viral proteins post infections. NS1, NP, PB1, PB2 are early protein and synthesize in 0–5 h.p.i. HA, NA, M1, NEP, M2, PB1-F2 are late proteins and synthesize in 5–10 h.p.i. This process leads to the virus production in between 5–12 h.p.i. and followed apoptosis of the cells in 20 h.p.i. During this whole processes some of the pathways are know to induce: 1, PI3-Akt <span class="highlight" style="background-color:">pathway</span>; 2, NFkB/lkB <span class="highlight" style="background-color:">pathway</span>; 3, MAPK <span class="highlight" style="background-color:">pathway</span> (A, JNK <span class="highlight" style="background-color:">pathway</span>; B, p38 <span class="highlight" style="background-color:">pathway</span>; C, ERK1/2 patyhway; D, ERK5 <span class="highlight" style="background-color:">pathway</span>); 4, protein kinase C (PKC)/PKR <span class="highlight" style="background-color:">signaling</span>; 5, TLR/RIG-I <span class="highlight" style="background-color:">signaling</span>.<br />Symbols used in the Figure: ⊣ inhibition; ⇢ unknown mechanism; → direct activation; ⇆ reversible process.</div></p></div><div class="aux"><div class="resc">Pratibha Gaur, et al. Med Sci Monit. 2011;17(6):RA148-RA154.</div><p class="links"><a class="dblinks" href="/pubmed/21629204" ref="ordinalpos=1151&amp;ncbi_uid=4824327&amp;link_uid=21629204">Citation</a><a class="dblinks" href="/pmc/articles/PMC3539548/" ref="ordinalpos=1151&amp;ncbi_uid=4824327&amp;link_uid=PMC3539548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1152.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1853162/figure/fig4/" ref="ordinalpos=1152&amp;ncbi_uid=950592&amp;link_uid=PMC1853162" href="/pmc/articles/PMC1853162/"><img src="/pmc/articles/instance/1853162/bin/biophysj00090084F04_RGB.gif" src-large="/pmc/articles/instance/1853162/bin/biophysj00090084F04_RGB.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1853162/" ref="ordinalpos=1152&amp;ncbi_uid=950592&amp;link_uid=PMC1853162" image-link="/pmc/articles/PMC1853162/figure/fig4/" class="imagepopup">FIGURE 4.  From: Mathematical Models of Specificity in Cell <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> network with shared components. (<i>A</i>) The simplest such network, herein referred to as the “basic architecture”, shown with activation and decay rate constants. Component <i>x</i><sub>1</sub> is common to pathways <i>X</i> and <i>Y</i>. This network cannot achieve specificity and fidelity. (<i>B</i>–<i>F</i>) Elaborations to the basic architecture that are found in nature and have been proposed to promote specificity. (<i>B</i>) Combinatorial <span class="highlight" style="background-color:">signaling</span> with an independent, parallel input provided by <span class="highlight" style="background-color:">pathway</span> <i>Z</i>. (<i>C</i>) Combinatorial <span class="highlight" style="background-color:">signaling</span> via the branching and reintegration of <span class="highlight" style="background-color:">pathway</span> <i>X</i>. (<i>D</i>) Cross-<span class="highlight" style="background-color:">pathway</span> inhibition. (<i>E</i>) Compartmentalization. (<i>F</i>) The action of a scaffold protein.</div></p></div><div class="aux"><div class="resc">Lee Bardwell, et al. Biophys J. 2007 May 15;92(10):3425-3441.</div><p class="links"><a class="dblinks" href="/pubmed/17325015" ref="ordinalpos=1152&amp;ncbi_uid=950592&amp;link_uid=17325015">Citation</a><a class="dblinks" href="/pmc/articles/PMC1853162/" ref="ordinalpos=1152&amp;ncbi_uid=950592&amp;link_uid=PMC1853162">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1153.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2776985/figure/pcbi-1000591-g002/" ref="ordinalpos=1153&amp;ncbi_uid=2113331&amp;link_uid=PMC2776985" href="/pmc/articles/PMC2776985/"><img src="/pmc/articles/instance/2776985/bin/pcbi.1000591.g002.gif" src-large="/pmc/articles/instance/2776985/bin/pcbi.1000591.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2776985/" ref="ordinalpos=1153&amp;ncbi_uid=2113331&amp;link_uid=PMC2776985" image-link="/pmc/articles/PMC2776985/figure/pcbi-1000591-g002/" class="imagepopup">Figure 2. Cell type specific topology using Integer Linear Programming..  From: Identifying Drug Effects via <span class="highlight" style="background-color:">Pathway</span> Alterations using an Integer Linear Programming Optimization Formulation on Phosphoproteomic Data. </a></p><div class="supp"><p class="details"><div>The ILP algorithm is using a subset of postulated reactions denoted with black and gray arrows in a generic <span class="highlight" style="background-color:">pathway</span> to construct a HepG2 <span class="highlight" style="background-color:">pathway</span> map (black arrows in <span class="highlight" style="background-color:">pathway</span> diagram). Gray triangles show phosphoprotein activation level upon stimuli (columns in top and bottom panels) and inhibitors (subcolumns in top and bottom panels). Red background denotes an error between experimental and <span class="highlight" style="background-color:">pathway</span>-inferred responses. Generic topology can hardly represent the HepG2 <span class="highlight" style="background-color:">signaling</span> responses (red background in top panel) and <span class="highlight" style="background-color:">pathway</span> optimization is critical to obtain a <span class="highlight" style="background-color:">pathway</span> topology that captures HepG2 function (limited red background in bottom panel). Pathways are visualized using Cytoscape [54].</div></p></div><div class="aux"><div class="resc">Alexander Mitsos, et al. PLoS Comput Biol. 2009 December;5(12):e1000591.</div><p class="links"><a class="dblinks" href="/pubmed/19997482" ref="ordinalpos=1153&amp;ncbi_uid=2113331&amp;link_uid=19997482">Citation</a><a class="dblinks" href="/pmc/articles/PMC2776985/" ref="ordinalpos=1153&amp;ncbi_uid=2113331&amp;link_uid=PMC2776985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1154.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4267164/figure/fig1/" ref="ordinalpos=1154&amp;ncbi_uid=6971795&amp;link_uid=PMC4267164" href="/pmc/articles/PMC4267164/"><img src="/pmc/articles/instance/4267164/bin/MI2014-432785.001.gif" src-large="/pmc/articles/instance/4267164/bin/MI2014-432785.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4267164/" ref="ordinalpos=1154&amp;ncbi_uid=6971795&amp;link_uid=PMC4267164" image-link="/pmc/articles/PMC4267164/figure/fig1/" class="imagepopup">Figure 1.  From: Collaborative Action of Toll-Like and Nod-Like Receptors as Modulators of the Inflammatory Response to Pathogenic Bacteria. </a></p><div class="supp"><p class="details"><div> <i>TLRs <span class="highlight" style="background-color:">signaling</span></i>. TLR <span class="highlight" style="background-color:">signaling</span> is activated in a MyD88-dependent (<i>black arrows</i>) and TRIF-dependent (<i>blue arrows</i>) manner. MyD88 <span class="highlight" style="background-color:">signaling</span> leads to NF-<i>κ</i>B activation, while TRIF <span class="highlight" style="background-color:">signaling</span> leads to both IRF3 and NF-<i>κ</i>B activation. In the MyD88-dependent ubiquitination of TRAF6 is important to activate MAPKs (JNK, p38 and ERK) or IKK complex to induce the translocation of NF-<i>κ</i>B to the nucleus. The TRIF-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can activate NF-<i>κ</i>B, IRF3, and IRF7.</div></p></div><div class="aux"><div class="resc">Javier Oviedo-Boyso, et al. Mediators Inflamm. 2014;2014:432785.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1154&amp;ncbi_uid=6971795&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4267164/" ref="ordinalpos=1154&amp;ncbi_uid=6971795&amp;link_uid=PMC4267164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1155.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3790638/figure/F6/" ref="ordinalpos=1155&amp;ncbi_uid=6543901&amp;link_uid=PMC3790638" href="/pmc/articles/PMC3790638/"><img src="/pmc/articles/instance/3790638/bin/nihms508140f6.gif" src-large="/pmc/articles/instance/3790638/bin/nihms508140f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3790638/" ref="ordinalpos=1155&amp;ncbi_uid=6543901&amp;link_uid=PMC3790638" image-link="/pmc/articles/PMC3790638/figure/F6/" class="imagepopup">Fig. 6. Working Model.  From: IGF-1 activates a cilium-localized non-canonical G?? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that regulates cell cycle progression. </a></p><div class="supp"><p class="details"><div>(A) Sequential diagram of our proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activated IGF-1R binds to the Gα subunit of the heterotrimeric G protein complex. AGS3 binds and stabilizes GDP-bound Gα. Free Gβγ triggers the release of Tctex-1 from the dynein complex, enabling subsequent phosphorylation of Tctex-1. (B) Spatial diagram of our working model. BB: basal body.</div></p></div><div class="aux"><div class="resc">Celine Yeh, et al. Dev Cell. 2013 August 26;26(4):358-368.</div><p class="links"><a class="dblinks" href="/pubmed/23954591" ref="ordinalpos=1155&amp;ncbi_uid=6543901&amp;link_uid=23954591">Citation</a><a class="dblinks" href="/pmc/articles/PMC3790638/" ref="ordinalpos=1155&amp;ncbi_uid=6543901&amp;link_uid=PMC3790638">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1156.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3098469/figure/F1/" ref="ordinalpos=1156&amp;ncbi_uid=3471918&amp;link_uid=PMC3098469" href="/pmc/articles/PMC3098469/"><img src="/pmc/articles/instance/3098469/bin/nihms280087f1.gif" src-large="/pmc/articles/instance/3098469/bin/nihms280087f1.jpg" alt="Figure" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3098469/" ref="ordinalpos=1156&amp;ncbi_uid=3471918&amp;link_uid=PMC3098469" image-link="/pmc/articles/PMC3098469/figure/F1/" class="imagepopup">Figure. Chemotactic <span class="highlight" style="background-color:">signaling</span> pathways.  From: <span class="highlight" style="background-color:">Signaling</span> Mechanisms for Chemotaxis. </a></p><div class="supp"><p class="details"><div>During chemotaxis, cAMP stimulates intracellular <span class="highlight" style="background-color:">signaling</span> through the cAR1 receptor and coupled heterotrimeric G-proteins (G). These <span class="highlight" style="background-color:">signaling</span> pathways control actin polymerization as well as the assembly and activity of myosin II. In the PtdIns(3,4,5)P3 (PIP3) <span class="highlight" style="background-color:">pathway</span>, the amount and localization of PIP3 are regulated by PI3Ks and PTEN. This <span class="highlight" style="background-color:">signaling</span> cascade participates in crosstalk with the Ras GTPase-TORC2 <span class="highlight" style="background-color:">pathway</span> by interacting with PI3K-Ras GTPase and activating PKBs. In the Ras GTPase-TORC2 <span class="highlight" style="background-color:">pathway</span>, PKBs generate a negative feedback loop by phosphorylating and inactivating the Sca1 complex (Sca1C). Soluble guanlyl cyclase (sGC) also modulates actin polymerization and myosin II. PLA2A generates a yet-to-be-determined arachidonic acid-related lipid (AARL) as a second messenger. Tsu: Tsunami, PDK: phosphoinositide dependent kinase, GbpC: cGMP binding prrotein C, MHCK: myosin heavy chain kinase, MLCK: myosin light chain kinase.</div></p></div><div class="aux"><div class="resc">Yu Wang, et al. Dev Growth Differ. ;53(4):495-502.</div><p class="links"><a class="dblinks" href="/pubmed/21585354" ref="ordinalpos=1156&amp;ncbi_uid=3471918&amp;link_uid=21585354">Citation</a><a class="dblinks" href="/pmc/articles/PMC3098469/" ref="ordinalpos=1156&amp;ncbi_uid=3471918&amp;link_uid=PMC3098469">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1157.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2744356/figure/F1/" ref="ordinalpos=1157&amp;ncbi_uid=2024433&amp;link_uid=PMC2744356" href="/pmc/articles/PMC2744356/"><img src="/pmc/articles/instance/2744356/bin/nihms92593f1.gif" src-large="/pmc/articles/instance/2744356/bin/nihms92593f1.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2744356/" ref="ordinalpos=1157&amp;ncbi_uid=2024433&amp;link_uid=PMC2744356" image-link="/pmc/articles/PMC2744356/figure/F1/" class="imagepopup">Fig 1. TLR <span class="highlight" style="background-color:">signaling</span>.  From: Toll-like receptors in skin infectious and inflammatory diseases. </a></p><div class="supp"><p class="details"><div>Upon the stimulation by cognate ligands, all TLRs, except for TLR3, recruit MyD88, IRAKs and TRAF6 to activate TLR <span class="highlight" style="background-color:">signaling</span>. Subsequently, TLR <span class="highlight" style="background-color:">signaling</span> diverges at TRAF6 to two different pathways to produce inflammatory cytokines. However, TLR3 utilizes TRIF for the activation of the TRIF-dependent <span class="highlight" style="background-color:">pathway</span> to produce type I interferons and inflammatory cytokines.</div></p></div><div class="aux"><div class="resc">Yuping Lai, et al. Infect Disord Drug Targets. ;8(3):144-155.</div><p class="links"><a class="dblinks" href="/pubmed/18782031" ref="ordinalpos=1157&amp;ncbi_uid=2024433&amp;link_uid=18782031">Citation</a><a class="dblinks" href="/pmc/articles/PMC2744356/" ref="ordinalpos=1157&amp;ncbi_uid=2024433&amp;link_uid=PMC2744356">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1158.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3475691/figure/pone-0047240-g008/" ref="ordinalpos=1158&amp;ncbi_uid=4311777&amp;link_uid=PMC3475691" href="/pmc/articles/PMC3475691/"><img src="/pmc/articles/instance/3475691/bin/pone.0047240.g008.gif" src-large="/pmc/articles/instance/3475691/bin/pone.0047240.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3475691/" ref="ordinalpos=1158&amp;ncbi_uid=4311777&amp;link_uid=PMC3475691" image-link="/pmc/articles/PMC3475691/figure/pone-0047240-g008/" class="imagepopup">Figure 8. KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis of significantly regulated hypothalamic proteins..  From: Altered Hypothalamic Protein Expression in a Rat Model of Huntington's Disease. </a></p><div class="supp"><p class="details"><div>Proteins significantly regulated in tgHD rats compared to WT controls were used as input data for KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> population analysis. The significantly populated pathways for tgHD rats were rationally clustered into subgroups focused upon energy metabolism (A), cell communication (B), and neuronal function (C). For each significantly-populated KEGG <span class="highlight" style="background-color:">pathway</span> a ‘hybrid’ score was generated which represents the −log<sub>10</sub> of the enrichment probability multiplied with the relative enrichment factor compared to the background proteomic expression.</div></p></div><div class="aux"><div class="resc">Wei-na Cong, et al. PLoS One. 2012; 2012;7(10):e47240.</div><p class="links"><a class="dblinks" href="/pubmed/23094041" ref="ordinalpos=1158&amp;ncbi_uid=4311777&amp;link_uid=23094041">Citation</a><a class="dblinks" href="/pmc/articles/PMC3475691/" ref="ordinalpos=1158&amp;ncbi_uid=4311777&amp;link_uid=PMC3475691">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1159.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3033390/figure/pone-0016556-g007/" ref="ordinalpos=1159&amp;ncbi_uid=2757592&amp;link_uid=PMC3033390" href="/pmc/articles/PMC3033390/"><img src="/pmc/articles/instance/3033390/bin/pone.0016556.g007.gif" src-large="/pmc/articles/instance/3033390/bin/pone.0016556.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3033390/" ref="ordinalpos=1159&amp;ncbi_uid=2757592&amp;link_uid=PMC3033390" image-link="/pmc/articles/PMC3033390/figure/pone-0016556-g007/" class="imagepopup">Figure 7. Schematic model of BBR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Berberine Improves Glucose Metabolism in Diabetic Rats by Inhibition of Hepatic Gluconeogenesis. </a></p><div class="supp"><p class="details"><div>BBR inhibits mitochondria function and decrease intracellular ATP. This leads to a reduction in gluconeogenic and lipogenic transcription factors (FoxO1, SREBP1, and ChREBP). As a result, expression of gluconeogenic genes (PEPCK and G6Pase) and lipogenic gene (FAS) are decreased. These molecular changes represent a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for improvement of fasting glucose and liver steatosis in the BBR-treated diabetic rats.</div></p></div><div class="aux"><div class="resc">Xuan Xia, et al. PLoS One. 2011;6(2):e16556.</div><p class="links"><a class="dblinks" href="/pubmed/21304897" ref="ordinalpos=1159&amp;ncbi_uid=2757592&amp;link_uid=21304897">Citation</a><a class="dblinks" href="/pmc/articles/PMC3033390/" ref="ordinalpos=1159&amp;ncbi_uid=2757592&amp;link_uid=PMC3033390">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1160.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3502344/figure/pone-0050185-g007/" ref="ordinalpos=1160&amp;ncbi_uid=4390827&amp;link_uid=PMC3502344" href="/pmc/articles/PMC3502344/"><img src="/pmc/articles/instance/3502344/bin/pone.0050185.g007.gif" src-large="/pmc/articles/instance/3502344/bin/pone.0050185.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3502344/" ref="ordinalpos=1160&amp;ncbi_uid=4390827&amp;link_uid=PMC3502344" image-link="/pmc/articles/PMC3502344/figure/pone-0050185-g007/" class="imagepopup">Figure 7. Proposed working model..  From: Fibronectin-Integrin <span class="highlight" style="background-color:">Signaling</span> Is Required for L-Glutamine's Protection against Gut Injury. </a></p><div class="supp"><p class="details"><div>GLN is protective in the intestine by preventing FN degradation after thermal injury, as well as by activating the protective FN-Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. GLN phosphorylates ERK1/2 via the FN-Integrin <span class="highlight" style="background-color:">pathway</span> leading to HSF-1 activation, which enhances HSP expression to prevent apoptosis.</div></p></div><div class="aux"><div class="resc">Stefanie Niederlechner, et al. PLoS One. 2012;7(11):e50185.</div><p class="links"><a class="dblinks" href="/pubmed/23185570" ref="ordinalpos=1160&amp;ncbi_uid=4390827&amp;link_uid=23185570">Citation</a><a class="dblinks" href="/pmc/articles/PMC3502344/" ref="ordinalpos=1160&amp;ncbi_uid=4390827&amp;link_uid=PMC3502344">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1161.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2689776/figure/f4/" ref="ordinalpos=1161&amp;ncbi_uid=1873429&amp;link_uid=PMC2689776" href="/pmc/articles/PMC2689776/"><img src="/pmc/articles/instance/2689776/bin/zjw0040933630004.gif" src-large="/pmc/articles/instance/2689776/bin/zjw0040933630004.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2689776/" ref="ordinalpos=1161&amp;ncbi_uid=1873429&amp;link_uid=PMC2689776" image-link="/pmc/articles/PMC2689776/figure/f4/" class="imagepopup">F ig . 4. From: Proteomics and <span class="highlight" style="background-color:">Pathway</span> Analysis Identifies JNK <span class="highlight" style="background-color:">Signaling</span> as Critical for High Linear Energy Transfer Radiation-induced Apoptosis in Non-small Lung Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>Summary of the four analyses for each pairwise comparison, showing significant outliers and top three ranked deactivated/activated pathways.</b> Using PSE, pathways could be identified as significantly (*) (de)activated by fulfilling one of two requirements or both of them. The most apparent identification of a (de)activated <span class="highlight" style="background-color:">pathway</span> is when the <span class="highlight" style="background-color:">pathway</span> is classified as an outlier in at least three of the four analyses (<i>upper panel</i>). A <span class="highlight" style="background-color:">pathway</span> that was not classified as an outlier could still be identified as significantly (de)activated by a more sensitive method, <i>i.e.</i> if it was ranked as top 3 according to Δ<i>S</i> in at least three of the four analyses (<i>lower panel</i>). The most significant (de)activated pathways are found in both <i>upper</i> and <i>lower panels</i>. The requirements for a significant (de)activated <span class="highlight" style="background-color:">pathway</span> were fulfilled for the JNK <span class="highlight" style="background-color:">pathway</span> (<i>p</i> = 2·10<sup>−6</sup>), the Fas <span class="highlight" style="background-color:">pathway</span> (<i>p</i> = 3·10<sup>−5</sup>), and the p38 <span class="highlight" style="background-color:">pathway</span> (<i>p</i> = 0.001) when comparing high LET-irradiated <i>versus</i> control cells (<i>A</i>) and for the JNK <span class="highlight" style="background-color:">pathway</span> (<i>p</i> = 6·10<sup>−6</sup>) and the Fas <span class="highlight" style="background-color:">pathway</span> (<i>p</i> = 3·10<sup>−5</sup>) when comparing high LET- <i>versus</i> low LET-irradiated cells (<i>C</i>). When comparing low LET-irradiated <i>versus</i> control cells no significantly (de)activated pathways were identified (<i>B</i>). <i>Nr</i>., number; <i>LPS</i>, lipopolysaccharide; <i>IFN</i>, interferon; <i>EGF</i>, epidermal growth factor; <i>PRL</i>, prolactin.</div></p></div><div class="aux"><div class="resc">Sara Ståhl, et al. Mol Cell Proteomics. 2009 May;8(5):1117-1129.</div><p class="links"><a class="dblinks" href="/pubmed/19168796" ref="ordinalpos=1161&amp;ncbi_uid=1873429&amp;link_uid=19168796">Citation</a><a class="dblinks" href="/pmc/articles/PMC2689776/" ref="ordinalpos=1161&amp;ncbi_uid=1873429&amp;link_uid=PMC2689776">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1162.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3095455/figure/F1/" ref="ordinalpos=1162&amp;ncbi_uid=2965341&amp;link_uid=PMC3095455" href="/pmc/articles/PMC3095455/"><img src="/pmc/articles/instance/3095455/bin/WJCO-1-24-g001.gif" src-large="/pmc/articles/instance/3095455/bin/WJCO-1-24-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3095455/" ref="ordinalpos=1162&amp;ncbi_uid=2965341&amp;link_uid=PMC3095455" image-link="/pmc/articles/PMC3095455/figure/F1/" class="imagepopup">Figure 1.  From: How can we kill cancer cells: Insights from the computational models of apoptosis. </a></p><div class="supp"><p class="details"><div>Schematic representation of the apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that indicates existence of three distinct <span class="highlight" style="background-color:">signaling</span> modules in the apoptotic intrinsic <span class="highlight" style="background-color:">pathway</span>. Apoptosis can be activated at various locations in the intrinsic <span class="highlight" style="background-color:">pathway</span>. We also show some of the targets of cancer drugs.</div></p></div><div class="aux"><div class="resc">Subhadip Raychaudhuri. World J Clin Oncol. 2010 November 10;1(1):24-28.</div><p class="links"><a class="dblinks" href="/pubmed/21603307" ref="ordinalpos=1162&amp;ncbi_uid=2965341&amp;link_uid=21603307">Citation</a><a class="dblinks" href="/pmc/articles/PMC3095455/" ref="ordinalpos=1162&amp;ncbi_uid=2965341&amp;link_uid=PMC3095455">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1163.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3611019/figure/F6/" ref="ordinalpos=1163&amp;ncbi_uid=5954352&amp;link_uid=PMC3611019" href="/pmc/articles/PMC3611019/"><img src="/pmc/articles/instance/3611019/bin/zbc0131341920006.gif" src-large="/pmc/articles/instance/3611019/bin/zbc0131341920006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3611019/" ref="ordinalpos=1163&amp;ncbi_uid=5954352&amp;link_uid=PMC3611019" image-link="/pmc/articles/PMC3611019/figure/F6/" class="imagepopup">FIGURE 6. From: MicroRNA-31 Activates the RAS <span class="highlight" style="background-color:">Pathway</span> and Functions as an Oncogenic MicroRNA in Human Colorectal Cancer by Repressing RAS p21 GTPase Activating Protein 1 (RASA1). </a></p><div class="supp"><p class="details"><div><b>Schematic of the entire <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> described, including the interaction between miR-31 and RASA1, along with a brief description of the portion of the RAS-MAPK <span class="highlight" style="background-color:">pathway</span> mentioned in the article.</b></div></p></div><div class="aux"><div class="resc">Defang Sun, et al. J Biol Chem. 2013 March 29;288(13):9508-9518.</div><p class="links"><a class="dblinks" href="/pubmed/23322774" ref="ordinalpos=1163&amp;ncbi_uid=5954352&amp;link_uid=23322774">Citation</a><a class="dblinks" href="/pmc/articles/PMC3611019/" ref="ordinalpos=1163&amp;ncbi_uid=5954352&amp;link_uid=PMC3611019">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1164.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3148240/figure/pone-0022933-g001/" ref="ordinalpos=1164&amp;ncbi_uid=3148234&amp;link_uid=PMC3148240" href="/pmc/articles/PMC3148240/"><img src="/pmc/articles/instance/3148240/bin/pone.0022933.g001.gif" src-large="/pmc/articles/instance/3148240/bin/pone.0022933.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3148240/" ref="ordinalpos=1164&amp;ncbi_uid=3148234&amp;link_uid=PMC3148240" image-link="/pmc/articles/PMC3148240/figure/pone-0022933-g001/" class="imagepopup">Figure 1. A model for Ras/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulated by scaffolds and modulators..  From: Simulating EGFR-ERK <span class="highlight" style="background-color:">Signaling</span> Control by Scaffold Proteins KSR and MP1 Reveals Differential Ligand-Sensitivity Co-Regulated by Cbl-CIN85 and Endophilin. </a></p><div class="supp"><p class="details"><div>More detailed biochemical model of scaffolding actions of KSR and MP1 in the dashed rectangular boxes are shown in Figures 2 and 3, respectively. Molecules highlighted in <i>blue</i>, <i>green</i> and <i>orange boxes</i> represent the separate modules of Ras/ERK <span class="highlight" style="background-color:">signaling</span> operating from the conventional mode (no-scaffolds; membrane), KSR-supported mode (membrane) and MP1-supported mode (late endosome), respectively.</div></p></div><div class="aux"><div class="resc">Lu Huang, et al. PLoS One. 2011;6(8):e22933.</div><p class="links"><a class="dblinks" href="/pubmed/21829671" ref="ordinalpos=1164&amp;ncbi_uid=3148234&amp;link_uid=21829671">Citation</a><a class="dblinks" href="/pmc/articles/PMC3148240/" ref="ordinalpos=1164&amp;ncbi_uid=3148234&amp;link_uid=PMC3148240">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1165.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3103888/figure/F1/" ref="ordinalpos=1165&amp;ncbi_uid=3318409&amp;link_uid=PMC3103888" href="/pmc/articles/PMC3103888/"><img src="/pmc/articles/instance/3103888/bin/nihms292347f1.gif" src-large="/pmc/articles/instance/3103888/bin/nihms292347f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3103888/" ref="ordinalpos=1165&amp;ncbi_uid=3318409&amp;link_uid=PMC3103888" image-link="/pmc/articles/PMC3103888/figure/F1/" class="imagepopup">Figure 1.  From: Clinical Features and Outcome of Patients With IRAK-4 and MyD88 Deficiency. </a></p><div class="supp"><p class="details"><div>Schematic representation of TIRs <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MyD88 interacts with TLRs and IL-1Rs through a shared TIR domain. MyD88 is a key cytosolic adapter molecule, providing a bridge from TLRs and IL-1Rs to the 2 active kinases IRAK-4 and IRAK-1. IRAK-4 and IRAK-1 then activate at least the 2 <span class="highlight" style="background-color:">signaling</span> NF-kB and MAPK pathways. The MyD88- and IRAK-4-dependent TIR <span class="highlight" style="background-color:">pathway</span> leads among others to the synthesis of inflammatory cytokines, such as IL-1β, IL-6, IL-8, TNF-α, and to IFN-α/β and IFN-λ, at least for TLR7, TLR8 and TLR9. The MyD88- and IRAK-4-independent TIR <span class="highlight" style="background-color:">pathway</span> uses TRIF <span class="highlight" style="background-color:">pathway</span> after stimulation of TLR3 and TLR4. This <span class="highlight" style="background-color:">pathway</span> is important for IFN-α and IFN-β production.</div></p></div><div class="aux"><div class="resc">Capucine Picard, et al. Medicine (Baltimore). ;89(6):403-425.</div><p class="links"><a class="dblinks" href="/pubmed/21057262" ref="ordinalpos=1165&amp;ncbi_uid=3318409&amp;link_uid=21057262">Citation</a><a class="dblinks" href="/pmc/articles/PMC3103888/" ref="ordinalpos=1165&amp;ncbi_uid=3318409&amp;link_uid=PMC3103888">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1166.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2955614/figure/F5/" ref="ordinalpos=1166&amp;ncbi_uid=2521538&amp;link_uid=PMC2955614" href="/pmc/articles/PMC2955614/"><img src="/pmc/articles/instance/2955614/bin/1476-4598-9-252-5.gif" src-large="/pmc/articles/instance/2955614/bin/1476-4598-9-252-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2955614/" ref="ordinalpos=1166&amp;ncbi_uid=2521538&amp;link_uid=PMC2955614" image-link="/pmc/articles/PMC2955614/figure/F5/" class="imagepopup">Figure 5.  From: MicroRNA roles in beta-catenin <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Novel molecular regulation of Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span></b>. Particular Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> pathways affected by upstream regulators, coregulators and downstream targets are described in detail in the review.</div></p></div><div class="aux"><div class="resc">Kai Huang, et al. Mol Cancer. 2010;9:252-252.</div><p class="links"><a class="dblinks" href="/pubmed/20858269" ref="ordinalpos=1166&amp;ncbi_uid=2521538&amp;link_uid=20858269">Citation</a><a class="dblinks" href="/pmc/articles/PMC2955614/" ref="ordinalpos=1166&amp;ncbi_uid=2521538&amp;link_uid=PMC2955614">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1167.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432564/figure/F2/" ref="ordinalpos=1167&amp;ncbi_uid=4612615&amp;link_uid=PMC3432564" href="/pmc/articles/PMC3432564/"><img src="/pmc/articles/instance/3432564/bin/ukmss-49846-f0002.gif" src-large="/pmc/articles/instance/3432564/bin/ukmss-49846-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432564/" ref="ordinalpos=1167&amp;ncbi_uid=4612615&amp;link_uid=PMC3432564" image-link="/pmc/articles/PMC3432564/figure/F2/" class="imagepopup">Figure 2.  From: C-type Lectin Receptors Orchestrate Anti-Fungal Immunity. </a></p><div class="supp"><p class="details"><div>Transmembrane CLRs involved in antifungal immunity and their intracellular <span class="highlight" style="background-color:">signaling</span> pathways. Dectin-1, Dectin-2 and Mincle induce intracellular <span class="highlight" style="background-color:">signaling</span> via tyrosine (Y)-based activation motifs (immunoreceptor tyrosine-based activation motifs or ITAMs) which recruit and activate Syk-kinase either directly, or indirectly through the FcγR adaptor chain. <span class="highlight" style="background-color:">Signaling</span> through protein-kinase C (PKC)δ, this <span class="highlight" style="background-color:">pathway</span> activates the Card9-Bcl10-Malt1 complex inducing gene transcription and the production of various inflammatory mediators. DC-SIGN and Dectin-1 can signal via the Raf-1 kinase <span class="highlight" style="background-color:">pathway</span> which modulates (dotted line) other <span class="highlight" style="background-color:">signaling</span> pathways, including those induced by the Toll-like receptors (TLR) and the Dectin-1/ Syk <span class="highlight" style="background-color:">pathway</span>. The mannose receptor (MR) can also induce intracellular <span class="highlight" style="background-color:">signaling</span>, but the mechanisms involved are unknown. CLR <span class="highlight" style="background-color:">signaling</span> can collaborate with that of the TLR (red bi-arrow), to synergistically induce or repress the induction of various cytokines and chemokines. CLR can also mediate fungal phagocytosis and induction of anti-microbial effector mechanisms (not shown).</div></p></div><div class="aux"><div class="resc">Sarah E. Hardison, et al. Nat Immunol. 2012 September;13(9):817-822.</div><p class="links"><a class="dblinks" href="/pubmed/22910394" ref="ordinalpos=1167&amp;ncbi_uid=4612615&amp;link_uid=22910394">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432564/" ref="ordinalpos=1167&amp;ncbi_uid=4612615&amp;link_uid=PMC3432564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1168.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3206446/figure/F7/" ref="ordinalpos=1168&amp;ncbi_uid=3323279&amp;link_uid=PMC3206446" href="/pmc/articles/PMC3206446/"><img src="/pmc/articles/instance/3206446/bin/1471-213X-11-57-7.gif" src-large="/pmc/articles/instance/3206446/bin/1471-213X-11-57-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3206446/" ref="ordinalpos=1168&amp;ncbi_uid=3323279&amp;link_uid=PMC3206446" image-link="/pmc/articles/PMC3206446/figure/F7/" class="imagepopup">Figure 7.  From: Loss of the Drosophila cell polarity regulator Scribbled promotes epithelial tissue overgrowth and cooperation with oncogenic Ras-Raf through impaired Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Ectopic aPKC <span class="highlight" style="background-color:">signaling</span> impairs Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> even when JNK <span class="highlight" style="background-color:">signaling</span> is blocked</b>. Larval eye/antennal discs with <i>ey-FLP </i>induced mutant clones marked by GFP expression (green, or magenta in the merges). Grayscale is β-GAL and F-actin. A white bar indicates the location of the MF. Apical (A, C) and basal (B, D) confocal sections are shown. (A-D) Coexpression of <i>aPKC<sup>ΔN </sup></i>with <i>bsk<sup>DN </sup></i>in clones results in large masses of tissue extruded basally from the epithelium which ectopically express <i>ex-lacZ </i>(A, B; arrows) and <i>fj-lacZ </i>(C, D; arrows).</div></p></div><div class="aux"><div class="resc">Karen Doggett, et al. BMC Dev Biol. 2011;11:57-57.</div><p class="links"><a class="dblinks" href="/pubmed/21955824" ref="ordinalpos=1168&amp;ncbi_uid=3323279&amp;link_uid=21955824">Citation</a><a class="dblinks" href="/pmc/articles/PMC3206446/" ref="ordinalpos=1168&amp;ncbi_uid=3323279&amp;link_uid=PMC3206446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1169.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3110895/figure/F4/" ref="ordinalpos=1169&amp;ncbi_uid=3359362&amp;link_uid=PMC3110895" href="/pmc/articles/PMC3110895/"><img src="/pmc/articles/instance/3110895/bin/zh70061135890004.gif" src-large="/pmc/articles/instance/3110895/bin/zh70061135890004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3110895/" ref="ordinalpos=1169&amp;ncbi_uid=3359362&amp;link_uid=PMC3110895" image-link="/pmc/articles/PMC3110895/figure/F4/" class="imagepopup">Fig. 4. From: Comprehensive microRNA analysis in bleomycin-induced pulmonary fibrosis identifies multiple sites of molecular regulation. </a></p><div class="supp"><p class="details"><div>Modeled miRNA mediated pathways in the pathogenesis of lung fibrosis. <i>A</i>: the mode of miRNAs that regulate apoptosis by modulating the apoptosis-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown. <i>B</i>: an overview of miRNAs regulating the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>C</i>: miRNAs regulate the inflammation by mediating the Toll-like receptor <span class="highlight" style="background-color:">pathway</span>. <i>D</i>: miRNAs regulate the fibroblast activation and collagen deposition by modulating the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Downregulated miRNAs are denoted in blue and upregulated in red.</div></p></div><div class="aux"><div class="resc">Ting Xie, et al. Physiol Genomics. 2011 May;43(9):479-487.</div><p class="links"><a class="dblinks" href="/pubmed/21266501" ref="ordinalpos=1169&amp;ncbi_uid=3359362&amp;link_uid=21266501">Citation</a><a class="dblinks" href="/pmc/articles/PMC3110895/" ref="ordinalpos=1169&amp;ncbi_uid=3359362&amp;link_uid=PMC3110895">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1170.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4214761/figure/pone-0111819-g005/" ref="ordinalpos=1170&amp;ncbi_uid=6808281&amp;link_uid=PMC4214761" href="/pmc/articles/PMC4214761/"><img src="/pmc/articles/instance/4214761/bin/pone.0111819.g005.gif" src-large="/pmc/articles/instance/4214761/bin/pone.0111819.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4214761/" ref="ordinalpos=1170&amp;ncbi_uid=6808281&amp;link_uid=PMC4214761" image-link="/pmc/articles/PMC4214761/figure/pone-0111819-g005/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Signaling</span> pathways in which the candidate genes are involved..  From: Expression Analysis of All Protease Genes Reveals Cathepsin K to Be Overexpressed in Glioblastoma. </a></p><div class="supp"><p class="details"><div>An extensive literature search revealed that the candidate genes are cross-linked to 3 <span class="highlight" style="background-color:">signaling</span> pathways: NF-κB, Akt and MAPK, which all play a role in cancer. NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has a crucial role in regulating immune responses, whereas Akt <span class="highlight" style="background-color:">signaling</span> has been shown to inhibit the growth of GBM cells and GBM stem-like cells that may also be impaired by MAPK <span class="highlight" style="background-color:">signaling</span> disruption. Because of the RT-qPCR results, <i>CTSK</i>'s role has been examined and it was found via cross linking to other candidate genes obtained via osteopontin (<i>OPN</i>) gene functions.</div></p></div><div class="aux"><div class="resc">Urška Verbovšek, et al. PLoS One. 2014;9(10):e111819.</div><p class="links"><a class="dblinks" href="/pubmed/25356585" ref="ordinalpos=1170&amp;ncbi_uid=6808281&amp;link_uid=25356585">Citation</a><a class="dblinks" href="/pmc/articles/PMC4214761/" ref="ordinalpos=1170&amp;ncbi_uid=6808281&amp;link_uid=PMC4214761">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1171.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3208682/figure/f1-0961644/" ref="ordinalpos=1171&amp;ncbi_uid=3403552&amp;link_uid=PMC3208682" href="/pmc/articles/PMC3208682/"><img src="/pmc/articles/instance/3208682/bin/961644.fig1.gif" src-large="/pmc/articles/instance/3208682/bin/961644.fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3208682/" ref="ordinalpos=1171&amp;ncbi_uid=3403552&amp;link_uid=PMC3208682" image-link="/pmc/articles/PMC3208682/figure/f1-0961644/" class="imagepopup">Figure 1. From: Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in chronic lymphocytic leukemia: distinct gene expression profiles of potential pathogenic significance in specific subsets of patients. </a></p><div class="supp"><p class="details"><div>Expression patterns of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in CLL. For reasons of clarity only the major molecules involved in TLR <span class="highlight" style="background-color:">signaling</span> are shown in the figure. Detailed results about the mRNA expression levels of all the molecules evaluated in this study are given in <i>Online Supplementary Table S6</i>. Gene names are those approved by the HUGO Gene Nomenclature Committee.</div></p></div><div class="aux"><div class="resc">Eleni Arvaniti, et al. Haematologica. 2011 November;96(11):1644-1652.</div><p class="links"><a class="dblinks" href="/pubmed/21750087" ref="ordinalpos=1171&amp;ncbi_uid=3403552&amp;link_uid=21750087">Citation</a><a class="dblinks" href="/pmc/articles/PMC3208682/" ref="ordinalpos=1171&amp;ncbi_uid=3403552&amp;link_uid=PMC3208682">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1172.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2808458/figure/F1/" ref="ordinalpos=1172&amp;ncbi_uid=2690339&amp;link_uid=PMC2808458" href="/pmc/articles/PMC2808458/"><img src="/pmc/articles/instance/2808458/bin/nihms-147724-f0001.gif" src-large="/pmc/articles/instance/2808458/bin/nihms-147724-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2808458/" ref="ordinalpos=1172&amp;ncbi_uid=2690339&amp;link_uid=PMC2808458" image-link="/pmc/articles/PMC2808458/figure/F1/" class="imagepopup">Figure 1. MyD88 dependent and independent TLR <span class="highlight" style="background-color:">signaling</span> pathways.  From: Toll-like receptor <span class="highlight" style="background-color:">signaling</span> in cell proliferation and survival. </a></p><div class="supp"><p class="details"><div>Activation of TLRs though ligand binding leads to the recruitment of the adaptor protein MyD88 or TRIF. In the MyD88 dependent <span class="highlight" style="background-color:">pathway</span>, activation of IRAK1 by IRAK4 leads to IRAK1 autophosphorylation which facilitates TRAF6 recruitment. TRAF6 then recruits MAP3Ks such as TAK1, MEKK2/3 and ASK1. TAK1 activation leads to the activation of IKK complex and the subsequent IkB degradation and NF-kB nucleus translocation. TAK1 also activates the p38-JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Tpl2, MEKK2/3 and ASK1 activate ERK1/2, p38-JNK and p38 respectively. In the MyD88-independent <span class="highlight" style="background-color:">pathway</span> TRIF activates TBK1 as well as IKKi which turns on the IRF-3 transcription factor. TRIF also activates NFkB <span class="highlight" style="background-color:">signaling</span> through RIP1. One of the most important consequences of TLR <span class="highlight" style="background-color:">signaling</span> is the transcriptional regulation of inflammatory genes as well as genes involved in cell proliferation and survival carried out by four groups of transcription factors.</div></p></div><div class="aux"><div class="resc">Xinyan Li, et al. Cytokine. ;49(1):1-9.</div><p class="links"><a class="dblinks" href="/pubmed/19775907" ref="ordinalpos=1172&amp;ncbi_uid=2690339&amp;link_uid=19775907">Citation</a><a class="dblinks" href="/pmc/articles/PMC2808458/" ref="ordinalpos=1172&amp;ncbi_uid=2690339&amp;link_uid=PMC2808458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1173.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3071676/figure/pone-0014686-g005/" ref="ordinalpos=1173&amp;ncbi_uid=2884203&amp;link_uid=PMC3071676" href="/pmc/articles/PMC3071676/"><img src="/pmc/articles/instance/3071676/bin/pone.0014686.g005.gif" src-large="/pmc/articles/instance/3071676/bin/pone.0014686.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3071676/" ref="ordinalpos=1173&amp;ncbi_uid=2884203&amp;link_uid=PMC3071676" image-link="/pmc/articles/PMC3071676/figure/pone-0014686-g005/" class="imagepopup">Figure 5. Examples of Ubx bound genes encoding <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>                             components..  From: Genome-Wide Tissue-Specific Occupancy of the Hox Protein                     Ultrabithorax and Hox Cofactor Homothorax in                     Drosophila. </a></p><div class="supp"><p class="details"><div>Ubx and Hth binding profiles at the Dpp <span class="highlight" style="background-color:">pathway</span> components                                 <i>thickveins (tkv)</i> (A), <i>dally</i> (B),                             and <i>notum</i> (C); at the Wg <span class="highlight" style="background-color:">pathway</span> components                                 <i>frizzled 2 (fz2)</i> (D), and <i>wingless                                 (wg)</i> (E); at the Hippo <span class="highlight" style="background-color:">pathway</span> components <i>bantam                                 (ban)</i> (F) and <i>expanded (ex)</i> (G); and at                             the Notch <span class="highlight" style="background-color:">pathway</span> components <i>Notch (N)</i> (H),                                 <i>mind bomb 1</i> (<i>mib1)</i> (I), and                                 <i>fringe (fng)</i> (J). Color scheme and tracks are as                             described in Figure                                 4.</div></p></div><div class="aux"><div class="resc">Matthew Slattery, et al. PLoS One. 2011;6(4):e14686.</div><p class="links"><a class="dblinks" href="/pubmed/21483663" ref="ordinalpos=1173&amp;ncbi_uid=2884203&amp;link_uid=21483663">Citation</a><a class="dblinks" href="/pmc/articles/PMC3071676/" ref="ordinalpos=1173&amp;ncbi_uid=2884203&amp;link_uid=PMC3071676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1174.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3376061/figure/F1/" ref="ordinalpos=1174&amp;ncbi_uid=4124358&amp;link_uid=PMC3376061" href="/pmc/articles/PMC3376061/"><img src="/pmc/articles/instance/3376061/bin/cib-5-190-g1.gif" src-large="/pmc/articles/instance/3376061/bin/cib-5-190-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3376061/" ref="ordinalpos=1174&amp;ncbi_uid=4124358&amp;link_uid=PMC3376061" image-link="/pmc/articles/PMC3376061/figure/F1/" class="imagepopup">Figure. From: The CD95 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Role of the Orai1-driven Ca<sup>2+</sup> entry in T lymphocytes challenged with the different forms of CD95L. When exposed to the two forms of CD95L, cleaved and membrane-bound, activated T lymphocytes, target cells (infected or transformed cells) or bystander cells undergo an Orai1-driven Ca<sup>2+</sup> entry that modulates differently the CD95-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Ca<sup>2+</sup> traces obtained with cells exposed to cleaved-CD95L (1) or membrane-bound CD95L (2) are depicted. MHC: Major Histocompatibility Complex.</div></p></div><div class="aux"><div class="resc">Aubin Penna, et al. Commun Integr Biol. 2012 March 1;5(2):190-192.</div><p class="links"><a class="dblinks" href="/pubmed/22808330" ref="ordinalpos=1174&amp;ncbi_uid=4124358&amp;link_uid=22808330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3376061/" ref="ordinalpos=1174&amp;ncbi_uid=4124358&amp;link_uid=PMC3376061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1175.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3534787/figure/F5/" ref="ordinalpos=1175&amp;ncbi_uid=5724663&amp;link_uid=PMC3534787" href="/pmc/articles/PMC3534787/"><img src="/pmc/articles/instance/3534787/bin/nihms421898f5.gif" src-large="/pmc/articles/instance/3534787/bin/nihms421898f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3534787/" ref="ordinalpos=1175&amp;ncbi_uid=5724663&amp;link_uid=PMC3534787" image-link="/pmc/articles/PMC3534787/figure/F5/" class="imagepopup">Figure 5. Vorinostat dampens cell survival <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Vorinostat, an HDAC Inhibitor Attenuates Epidermoid Squamous Cell Carcinoma Growth by Dampening mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in a Human Xenograft Murine Model. </a></p><div class="supp"><p class="details"><div>(A &amp; B) WB and relative densitometry analysis for AKT/mTOR <span class="highlight" style="background-color:">signaling</span> markers. p-AKT1/2/3 Thr308 (*p=0.004), p-AKT1/2/3 ser473 (*p=0.001), AKT1/2/3 (*p=0.19) significantly reduced in vorinostat treatment group of mice. (C&amp;D) WB and densitometry analysis of p-mTOR (*P=0.04) and total m-TOR (P=0.15) in vorinostat treated group represent significant reduction of p-mTOR level. (E&amp;F) WB and densitometry analysis of p-mTOR (*P=0.027) and total mTOR (*P=0.009) levels in vorinostat (2μM) treated A431 cells. (G&amp;H) WB and relative densitometry analysis mTOR <span class="highlight" style="background-color:">signaling</span> proteins, p-p70S6 kinase (**P=0.001), p70S6 kinase (P=0.976), p-S6 ribosomal protein (**P=0.004), S6 ribosomal protein (**P=0.005), p-4E-BP1 (**p=0.003) and 4E-BP1 (*p=0.033).</div></p></div><div class="aux"><div class="resc">Deepali Kurundkar, et al. Toxicol Appl Pharmacol. ;266(2):233-244.</div><p class="links"><a class="dblinks" href="/pubmed/23147569" ref="ordinalpos=1175&amp;ncbi_uid=5724663&amp;link_uid=23147569">Citation</a><a class="dblinks" href="/pmc/articles/PMC3534787/" ref="ordinalpos=1175&amp;ncbi_uid=5724663&amp;link_uid=PMC3534787">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1176.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4349708/figure/pcbi.1004088.g003/" ref="ordinalpos=1176&amp;ncbi_uid=7293553&amp;link_uid=PMC4349708" href="/pmc/articles/PMC4349708/"><img src="/pmc/articles/instance/4349708/bin/pcbi.1004088.g003.gif" src-large="/pmc/articles/instance/4349708/bin/pcbi.1004088.g003.jpg" alt="Fig 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4349708/" ref="ordinalpos=1176&amp;ncbi_uid=7293553&amp;link_uid=PMC4349708" image-link="/pmc/articles/PMC4349708/figure/pcbi.1004088.g003/" class="imagepopup">Fig 3. Focal adhesion <span class="highlight" style="background-color:">pathway</span> as a virulence factor target..  From: Mining Host-Pathogen Protein Interactions to Characterize Burkholderia mallei Infectivity Mechanisms. </a></p><div class="supp"><p class="details"><div>We identified the Kyoto Encyclopedia of Genes and Genomes (KEGG) focal adhesion <span class="highlight" style="background-color:">pathway</span> as enriched with multiple virulence factor targets. The majority (17 of 20) of host proteins interacting with <i>B. mallei</i> virulence factors in this <span class="highlight" style="background-color:">pathway</span> belong to a connected sub-<span class="highlight" style="background-color:">pathway</span> of proteins (yellow boxes and red lines), mainly grouped at the beginning of the <span class="highlight" style="background-color:">pathway</span>. This observation implied that receptors and <span class="highlight" style="background-color:">signaling</span> molecules were likely <i>B. mallei</i> targets, corroborating previous observations that pathogens tend to interfere with host processes related to cell communication and actin cytoskeleton organization. The <span class="highlight" style="background-color:">pathway</span> diagram for the focal adhesion <span class="highlight" style="background-color:">pathway</span> was adapted from the KEGG <span class="highlight" style="background-color:">pathway</span> map [32] with permission from the KEGG database administrators.</div></p></div><div class="aux"><div class="resc">Vesna Memišević, et al. PLoS Comput Biol. 2015 March;11(3):e1004088.</div><p class="links"><a class="dblinks" href="/pubmed/25738731" ref="ordinalpos=1176&amp;ncbi_uid=7293553&amp;link_uid=25738731">Citation</a><a class="dblinks" href="/pmc/articles/PMC4349708/" ref="ordinalpos=1176&amp;ncbi_uid=7293553&amp;link_uid=PMC4349708">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1177.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2883312/figure/F5/" ref="ordinalpos=1177&amp;ncbi_uid=2291707&amp;link_uid=PMC2883312" href="/pmc/articles/PMC2883312/"><img src="/pmc/articles/instance/2883312/bin/nihms-191662-f0005.gif" src-large="/pmc/articles/instance/2883312/bin/nihms-191662-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2883312/" ref="ordinalpos=1177&amp;ncbi_uid=2291707&amp;link_uid=PMC2883312" image-link="/pmc/articles/PMC2883312/figure/F5/" class="imagepopup">Figure 5. Regulation of CRF <span class="highlight" style="background-color:">signaling</span> by CRF1 isoforms.  From: Emerging role of alternative splicing of CRF1 receptor in CRF <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><i>CRF1</i> gene contains 14 exons and only one isoform of receptor–CRF1β (also called pro-CRF1) is coded by all exons. CRF1 transcript is also subjected to alternative splicing resulting in at least 8 isoforms. Recent studies showed that expression and/or co-expression of CRF1 isoforms is responsible for modulation of CRF1 <span class="highlight" style="background-color:">signaling</span>. Plus indicates stimulation of downsteam <span class="highlight" style="background-color:">signaling</span> by the classic <span class="highlight" style="background-color:">pathway</span> (CRF1α) or alternative <span class="highlight" style="background-color:">pathway</span> (CRF1β). Soluble isoforms (CRF1e and h) were also found to stimulate CRF <span class="highlight" style="background-color:">signaling</span> when co-expressed with CRF1α. Minus indicates inhibition of CRF <span class="highlight" style="background-color:">signaling</span> on different levels including: fast mRNA decay (CRF1e), dimerization and subsequent intercellular retention resulting in most probable premature receptor degradation (CRF1α with CRF1d, CRFf or CRFg), decoy receptor mechanism (CRF1h and e when secreted), agonist binding impairment (CRF1c) or G-protein wbinding inhibition (CRF1d). See text for details and citations.</div></p></div><div class="aux"><div class="resc">Michał A. Ẓmijewski, et al. Acta Biochim Pol. ;57(1):1-13.</div><p class="links"><a class="dblinks" href="/pubmed/20234885" ref="ordinalpos=1177&amp;ncbi_uid=2291707&amp;link_uid=20234885">Citation</a><a class="dblinks" href="/pmc/articles/PMC2883312/" ref="ordinalpos=1177&amp;ncbi_uid=2291707&amp;link_uid=PMC2883312">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1178.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3266773/figure/F10/" ref="ordinalpos=1178&amp;ncbi_uid=3784777&amp;link_uid=PMC3266773" href="/pmc/articles/PMC3266773/"><img src="/pmc/articles/instance/3266773/bin/JCI45977.f10.gif" src-large="/pmc/articles/instance/3266773/bin/JCI45977.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3266773/" ref="ordinalpos=1178&amp;ncbi_uid=3784777&amp;link_uid=PMC3266773" image-link="/pmc/articles/PMC3266773/figure/F10/" class="imagepopup">Figure 10. Model of CD27-induced Wnt <span class="highlight" style="background-color:">pathway</span> activation in LSCs..  From: CD27 <span class="highlight" style="background-color:">signaling</span> on chronic myelogenous leukemia stem cells activates Wnt target genes and promotes disease progression. </a></p><div class="supp"><p class="details"><div>Binding of Wnt proteins to FZD leads to activation of β-catenin in the cytoplasm. Subsequently, TNIK binds to active β-catenin and this complex is recruited to the nucleus, where TNIK directly phosphorylates Tcf, converting the Tcf/Lef-complex from a transcriptional repressor into a transcriptional activator of Wnt target genes. In CML, the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is constantly activated by BCR/ABL-mediated tyrosine phosphorylation of active β-catenin, resulting in enhanced stability and leading to pathological LSC proliferation/differentiation. CD27 <span class="highlight" style="background-color:">signaling</span> leads to recruitment of TRAF2 and subsequent activation of TNIK, promoting nuclear translocation of the active β-catenin/TNIK complex. Thereby, CD27 <span class="highlight" style="background-color:">signaling</span> contributes to increased transcription of Wnt target genes.</div></p></div><div class="aux"><div class="resc">Christian Schürch, et al. J Clin Invest. 2012 February 1;122(2):624-638.</div><p class="links"><a class="dblinks" href="/pubmed/22232214" ref="ordinalpos=1178&amp;ncbi_uid=3784777&amp;link_uid=22232214">Citation</a><a class="dblinks" href="/pmc/articles/PMC3266773/" ref="ordinalpos=1178&amp;ncbi_uid=3784777&amp;link_uid=PMC3266773">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1179.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4061568/figure/F4/" ref="ordinalpos=1179&amp;ncbi_uid=6264273&amp;link_uid=PMC4061568" href="/pmc/articles/PMC4061568/"><img src="/pmc/articles/instance/4061568/bin/nihms580685f4.gif" src-large="/pmc/articles/instance/4061568/bin/nihms580685f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4061568/" ref="ordinalpos=1179&amp;ncbi_uid=6264273&amp;link_uid=PMC4061568" image-link="/pmc/articles/PMC4061568/figure/F4/" class="imagepopup">Figure 4.  From: SOX17 Methylation Inhibits Its Antagonism of Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Lung Cancer. </a></p><div class="supp"><p class="details"><div>SOX17 influenced the transcriptional activation of β-catenin and its downstream genes in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A. H23 cells were transfected with β-catenin and empty vectors, respectively, to detect relative luciferase activities. B. The architecture of SOX17 with an HMG box is depicted in schematic form. Different deletion mutants (SOX17 constructs 50–414, 135–414, and 1–353) were generated by PCR. C. H23 cells were co-transfected with β-catenin and different SOX17 expression vectors (wild-type or mutant). Transfection with wild-type and mutant SOX17 showed greater or less suppressive effects on β-catenin/TCF-LEF transcriptional activity. D. Transfection with wild-type SOX17 inhibited cyclin D1, a downstream gene of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Dongtao Yin, et al. Discov Med. ;14(74):33-40.</div><p class="links"><a class="dblinks" href="/pubmed/22846201" ref="ordinalpos=1179&amp;ncbi_uid=6264273&amp;link_uid=22846201">Citation</a><a class="dblinks" href="/pmc/articles/PMC4061568/" ref="ordinalpos=1179&amp;ncbi_uid=6264273&amp;link_uid=PMC4061568">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1180.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3058300/figure/F5/" ref="ordinalpos=1180&amp;ncbi_uid=3452997&amp;link_uid=PMC3058300" href="/pmc/articles/PMC3058300/"><img src="/pmc/articles/instance/3058300/bin/nihms-249563-f0005.gif" src-large="/pmc/articles/instance/3058300/bin/nihms-249563-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3058300/" ref="ordinalpos=1180&amp;ncbi_uid=3452997&amp;link_uid=PMC3058300" image-link="/pmc/articles/PMC3058300/figure/F5/" class="imagepopup">Figure 5. Model of Effect of Increased Inhibition in Midline Thalamus on SB-PFC <span class="highlight" style="background-color:">signaling</span>..  From: Increased GABA-ergic Inhibition in the Midline Thalamus Affects <span class="highlight" style="background-color:">Signaling</span> and Seizure Spread in the Hippocampus-Prefrontal Cortex <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>In the normal circuit, there is a direct monosynaptic connection between the hippocampus and the PFC. There is also a disynaptic connection that terminates in the thalamus before reconverging in the PFC. When the thalamic response is reduced, the thalamic output is weakened. This attenuates the functional role of the accessory excitatory thalamic output to the PFC, and also reduces the capacity for seizures to be generated in the SB-PFC <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">David M. Sloan, et al. Epilepsia. ;52(3):523-530.</div><p class="links"><a class="dblinks" href="/pubmed/21204829" ref="ordinalpos=1180&amp;ncbi_uid=3452997&amp;link_uid=21204829">Citation</a><a class="dblinks" href="/pmc/articles/PMC3058300/" ref="ordinalpos=1180&amp;ncbi_uid=3452997&amp;link_uid=PMC3058300">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1181.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3072770/figure/F2/" ref="ordinalpos=1181&amp;ncbi_uid=2882099&amp;link_uid=PMC3072770" href="/pmc/articles/PMC3072770/"><img src="/pmc/articles/instance/3072770/bin/bar010f2.gif" src-large="/pmc/articles/instance/3072770/bin/bar010f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3072770/" ref="ordinalpos=1181&amp;ncbi_uid=2882099&amp;link_uid=PMC3072770" image-link="/pmc/articles/PMC3072770/figure/F2/" class="imagepopup">Figure 2. From: The Rat Genome Database <span class="highlight" style="background-color:">Pathway</span> Portal. </a></p><div class="supp"><p class="details"><div>Shows a partial view of the interactive diagram page for the ‘androgen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>’ (<b>A</b>) and for the ‘prostate cancer <span class="highlight" style="background-color:">pathway</span>’ with the inset showing the list of miRNAs of potential interest in prostate cancer which can be accessed by clicking on the functional class icon (<b>B</b>).</div></p></div><div class="aux"><div class="resc">Victoria Petri, et al. Database (Oxford). 2011;2011:bar010.</div><p class="links"><a class="dblinks" href="/pubmed/21478484" ref="ordinalpos=1181&amp;ncbi_uid=2882099&amp;link_uid=21478484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3072770/" ref="ordinalpos=1181&amp;ncbi_uid=2882099&amp;link_uid=PMC3072770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1182.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2652205/figure/F5/" ref="ordinalpos=1182&amp;ncbi_uid=1765455&amp;link_uid=PMC2652205" href="/pmc/articles/PMC2652205/"><img src="/pmc/articles/instance/2652205/bin/234fig5.gif" src-large="/pmc/articles/instance/2652205/bin/234fig5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2652205/" ref="ordinalpos=1182&amp;ncbi_uid=1765455&amp;link_uid=PMC2652205" image-link="/pmc/articles/PMC2652205/figure/F5/" class="imagepopup">Figure 5. From: Network-level molecular evolutionary analysis of the insulin/TOR signal transduction <span class="highlight" style="background-color:">pathway</span> across 12 Drosophila genomes. </a></p><div class="supp"><p class="details"><div>Schematic representation of the selective constraint levels expected along two hypothetical <span class="highlight" style="background-color:">signaling</span> pathways with different connection patterns. (<i>A</i>) <span class="highlight" style="background-color:">Pathway</span> receiving multiple <span class="highlight" style="background-color:">signaling</span> inputs along the <span class="highlight" style="background-color:">pathway</span> and with a single output. In this scenario, selective constraint levels will be higher at the downstream part, since the elements are progressively involved in a greater number of pathways. (<i>B</i>) <span class="highlight" style="background-color:">Pathway</span> with multiple outputs along the <span class="highlight" style="background-color:">pathway</span> (i.e., with multiple branching points able to transmit information to other pathways). In this scenario, the selective constraint levels will be higher for the upstream elements. The more constrained elements (nodes) are darker. The numbers in the nodes represent the number of pathways in which each element is involved.</div></p></div><div class="aux"><div class="resc">David Alvarez-Ponce, et al. Genome Res. 2009 February;19(2):234-242.</div><p class="links"><a class="dblinks" href="/pubmed/19141596" ref="ordinalpos=1182&amp;ncbi_uid=1765455&amp;link_uid=19141596">Citation</a><a class="dblinks" href="/pmc/articles/PMC2652205/" ref="ordinalpos=1182&amp;ncbi_uid=1765455&amp;link_uid=PMC2652205">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1183.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193412/figure/fig7/" ref="ordinalpos=1183&amp;ncbi_uid=3770415&amp;link_uid=PMC3193412" href="/pmc/articles/PMC3193412/"><img src="/pmc/articles/instance/3193412/bin/cr2010161f7.gif" src-large="/pmc/articles/instance/3193412/bin/cr2010161f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193412/" ref="ordinalpos=1183&amp;ncbi_uid=3770415&amp;link_uid=PMC3193412" image-link="/pmc/articles/PMC3193412/figure/fig7/" class="imagepopup">Figure 7.  From: A single NF?B system for both canonical and non-canonical <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Wiring diagrams of the NFκB <span class="highlight" style="background-color:">signaling</span> system to chart crosstalk between canonical and non-canonical <span class="highlight" style="background-color:">signaling</span> pathways. <b>(A)</b> Non-canonical control of RelA:p50. RelA:p50 is inhibited by not only IκBα, -β, -ɛ, but also IκBδ. Whereas inflammatory canonical signals lead to degradation of IκBα, -β, -ɛ, developmental signals engage the non-canonical <span class="highlight" style="background-color:">pathway</span> to disrupt IκBδ activity; however, both result in the nuclear translocation of RelA:p50. Interestingly, prior canonical <span class="highlight" style="background-color:">signaling</span> history results in an enhancement of the non-canonical-RelA:p50 axis, due to inducible expression of p100. <b>(B)</b> Canonical control of RelB:p52 activation. Expression of p100 and RelB are dependent on RelA:p50 activity and therefore canonical signals. The amount of basal RelB expression, controlled by constitutive canonical <span class="highlight" style="background-color:">pathway</span> activity, rather than the inducible p100 expression, is the main determinant of the strength of non-canonical <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Vincent Feng-Sheng Shih, et al. Cell Res. 2011 January;21(1):86-102.</div><p class="links"><a class="dblinks" href="/pubmed/21102550" ref="ordinalpos=1183&amp;ncbi_uid=3770415&amp;link_uid=21102550">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193412/" ref="ordinalpos=1183&amp;ncbi_uid=3770415&amp;link_uid=PMC3193412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1184.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2933925/figure/fig3-1947601910373795/" ref="ordinalpos=1184&amp;ncbi_uid=3024508&amp;link_uid=PMC2933925" href="/pmc/articles/PMC2933925/"><img src="/pmc/articles/instance/3092205/bin/10.1177_1947601910373795-fig3.gif" src-large="/pmc/articles/instance/3092205/bin/10.1177_1947601910373795-fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2933925/" ref="ordinalpos=1184&amp;ncbi_uid=3024508&amp;link_uid=PMC2933925" image-link="/pmc/articles/PMC2933925/figure/fig3-1947601910373795/" class="imagepopup">Figure 3. From: The Role of B-RAF Mutations in Melanoma and the Induction of EMT via Dysregulation of the NF-?B/Snail/RKIP/PTEN Circuit. </a></p><div class="supp"><p class="details"><div>Regulation of survival pathways by mutant B-RAF. (<b>A</b>) Mutant B-RAF activation of the mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">pathway</span>. B-RAF mutants, with elevated kinase activity, interact with the microenvironment to hyperactively phosphorylate MAP, which results in the phosphorylation downstream of the kinase MEK and the translocation of ERK into the nucleus. B-RAF mutants with lowered kinase activity regulate the MAPK <span class="highlight" style="background-color:">pathway</span> indirectly through C-RAF. Translocation of ERK to the nucleus via B-RAF/C-RAF <span class="highlight" style="background-color:">signaling</span> results in the expression of metalloproteinases and EMT. (<b>B</b>) Mutant B-RAF activation of the NF-κB <span class="highlight" style="background-color:">pathway</span>. The constitutively active mutant B-RAF activates the NF-κB <span class="highlight" style="background-color:">pathway</span> via IKKβ activation. Activation of NF-κB results in EMT via up-regulation of the metastasis inducer, Snail, as well as other metastatic gene products. NF-κB is inhibited directly through the metastasis suppressor, RKIP. RKIP also indirectly inhibits NF-κB through inhibition of B-RAF activity. (<b>C</b>) Up-regulation of mutant B-RAF via PTEN deletion, PI3K <span class="highlight" style="background-color:">signaling</span>, and AKT overexpression. RAS <span class="highlight" style="background-color:">signaling</span> results in PI3K activation and AKT expression. Normally, PI3K-induced expression of AKT is antagonized by PTEN. However, deletion of PTEN as well as signals from the microenvironment can result in uncontrolled AKT production. Higher levels of AKT expression inhibit mutant B-RAF <span class="highlight" style="background-color:">signaling</span> levels. Inhibition of B-RAF <span class="highlight" style="background-color:">signaling</span> by AKT counterintuitively promotes EMT. When mutant B-RAF <span class="highlight" style="background-color:">signaling</span> is extremely high, cellular defense mechanisms will induce the cells into senescence. AKT inhibition of B-RAF lowers B-RAF <span class="highlight" style="background-color:">signaling</span> to levels high enough to promote EMT and low enough to evade senescence.</div></p></div><div class="aux"><div class="resc">Kimberly Lin, et al. Genes Cancer. 2010 May;1(5):409-420.</div><p class="links"><a class="dblinks" href="/pubmed/20827424" ref="ordinalpos=1184&amp;ncbi_uid=3024508&amp;link_uid=20827424">Citation</a><a class="dblinks" href="/pmc/articles/PMC2933925/" ref="ordinalpos=1184&amp;ncbi_uid=3024508&amp;link_uid=PMC2933925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1185.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3252370/figure/ppat-1002461-g010/" ref="ordinalpos=1185&aamp;ncbi_uid=3750021&amp;link_uid=PMC3252370" href="/pmc/articles/PMC3252370/"><img src="/pmc/articles/instance/3252370/bin/ppat.1002461.g010.gif" src-large="/pmc/articles/instance/3252370/bin/ppat.1002461.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3252370/" ref="ordinalpos=1185&amp;ncbi_uid=3750021&amp;link_uid=PMC3252370" image-link="/pmc/articles/PMC3252370/figure/ppat-1002461-g010/" class="imagepopup">Figure 10. Hypothetical model for the interactions of HIV-1 gp120, FN, α5β1 integrin, Robo4 and Slit2, and their effect on the permeability of lymphatic endothelium..  From: Slit2/Robo4 <span class="highlight" style="background-color:">Signaling</span> Modulates HIV-1 gp120-Induced Lymphatic Hyperpermeability. </a></p><div class="supp"><p class="details"><div>Robo4 and α<sub>5</sub>β<sub>1</sub> integrin are expressed on the surface of lymphatic endothelial cells. Under physiologic conditions, endogenous Slit2 interacts with Robo4 via its Ig domains, and inhibits c-Src <span class="highlight" style="background-color:">signaling</span>. FN also interacts with Robo4 (via its FN type III domains), and binds to α<sub>5</sub>β<sub>1</sub> integrin on the endothelial cell surface. This contributes further to the dynamic regulation of Robo4 <span class="highlight" style="background-color:">signaling</span> by Slit2 and FN, to maintain the integrity of the lymphatic endothelial barrier. Upon HIV infection, gp120 significantly elevates FN levels and complexes with FN. FN then activates α<sub>5</sub>β<sub>1</sub> integrin, which results in enhanced intracellular <span class="highlight" style="background-color:">signaling</span> through α<sub>5</sub>β<sub>1</sub> integrin, a stronger interaction between FN and Robo4, and disruption of Robo4 <span class="highlight" style="background-color:">signaling</span>. These changes activate the c-Src <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and induce hyperpermeability of the lymphatic endothelial barrier. Exogenous Slit2 may protect the lymphatic channels from HIV-induced vasculopathy by interacting with Robo4 to restore Robo4 <span class="highlight" style="background-color:">signaling</span>, block the c-Src <span class="highlight" style="background-color:">pathway</span>, and inhibit HIV-induced lymphatic hyperpermeability.</div></p></div><div class="aux"><div class="resc">Xuefeng Zhang, et al. PLoS Pathog. 2012 January;8(1):e1002461.</div><p class="links"><a class="dblinks" href="/pubmed/22241990" ref="ordinalpos=1185&amp;ncbi_uid=3750021&amp;link_uid=22241990">Citation</a><a class="dblinks" href="/pmc/articles/PMC3252370/" ref="ordinalpos=1185&amp;ncbi_uid=3750021&amp;link_uid=PMC3252370">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1186.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1914214/figure/F7/" ref="ordinalpos=1186&amp;ncbi_uid=964293&amp;link_uid=PMC1914214" href="/pmc/articles/PMC1914214/"><img src="/pmc/articles/instance/1914214/bin/nihms22441f7.gif" src-large="/pmc/articles/instance/1914214/bin/nihms22441f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1914214/" ref="ordinalpos=1186&amp;ncbi_uid=964293&amp;link_uid=PMC1914214" image-link="/pmc/articles/PMC1914214/figure/F7/" class="imagepopup">Fig. 7. Schematic representation of the effects of ectopic Shh <span class="highlight" style="background-color:">signaling</span> on dorsoventral patterning in the hindbrain of ShhP1 embryos.  From: Activation of Class I transcription factors by low level Sonic hedgehog <span class="highlight" style="background-color:">signaling</span> is mediated by Gli2-dependent and independent mechanisms. </a></p><div class="supp"><p class="details"><div>(A) Low level Shh <span class="highlight" style="background-color:">signaling</span> from the ectopic source in the otic vesicle of <i>ShhP1</i> embryos (blue arrow) results in an expansion of v0 and v1 interneurons in the adjacent hindbrain. (B) A revised classification scheme for Shh target genes. Class I genes (light blue) are activated by low level Shh <span class="highlight" style="background-color:">signaling</span> and oppose the expression of corresponding Class II genes. Class II genes (navy blue) are activated by high level Shh <span class="highlight" style="background-color:">signaling</span> and oppose the expression of corresponding Class I genes. Dorsal genes (red) are repressed by the lowest level of Shh <span class="highlight" style="background-color:">signaling</span> and are not necessarily opposed by Class I and/or Class II genes. Class I genes are also activated by a Retinoic Acid (RA) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> operating in parallel to Shh (Novitch et al., 2003).</div></p></div><div class="aux"><div class="resc">Abraham Pachikara, et al. Dev Biol. ;305(1):52-62.</div><p class="links"><a class="dblinks" href="/pubmed/17321515" ref="ordinalpos=1186&amp;ncbi_uid=964293&amp;link_uid=17321515">Citation</a><a class="dblinks" href="/pmc/articles/PMC1914214/" ref="ordinalpos=1186&amp;ncbi_uid=964293&amp;link_uid=PMC1914214">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1187.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3558493/figure/pone-0055211-g008/" ref="ordinalpos=1187&amp;ncbi_uid=4596369&amp;link_uid=PMC3558493" href="/pmc/articles/PMC3558493/"><img src="/pmc/articles/instance/3558493/bin/pone.0055211.g008.gif" src-large="/pmc/articles/instance/3558493/bin/pone.0055211.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3558493/" ref="ordinalpos=1187&amp;ncbi_uid=4596369&amp;link_uid=PMC3558493" image-link="/pmc/articles/PMC3558493/figure/pone-0055211-g008/" class="imagepopup">Figure 8. Model of the ErbB2 <span class="highlight" style="background-color:">signaling</span> network that controls microtubule capture, microtubules stability and chemotaxis..  From: ErbB2-Dependent Chemotaxis Requires Microtubule Capture and Stabilization Coordinated by Distinct <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Our data show that ErbB2 controls microtubules via distinct, but complementary, <span class="highlight" style="background-color:">signaling</span> pathways. The Memo/RhoA and the PLCγ/cPKC pathways converge to control ACF7 localization at the leading edge, allowing microtubule capture. While PKCζ is also required downstream of PLCγ, its mode of action is still unknown. The PI3K/Akt <span class="highlight" style="background-color:">pathway</span> contributes to microtubule stability, at later stages, and involves the combined effect of the PAK/Stathmin <span class="highlight" style="background-color:">pathway</span> and repression of GSK3 activity. Microtubule capture and stabilization allow the chemotactic response to HRG.</div></p></div><div class="aux"><div class="resc">Khedidja Benseddik, et al. PLoS One. 2013;8(1):e55211.</div><p class="links"><a class="dblinks" href="/pubmed/23383112" ref="ordinalpos=1187&amp;ncbi_uid=4596369&amp;link_uid=23383112">Citation</a><a class="dblinks" href="/pmc/articles/PMC3558493/" ref="ordinalpos=1187&amp;ncbi_uid=4596369&amp;link_uid=PMC3558493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1188.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3268412/figure/F8/" ref="ordinalpos=1188&amp;ncbi_uid=4543368&amp;link_uid=PMC3268412" href="/pmc/articles/PMC3268412/"><img src="/pmc/articles/instance/3268412/bin/zbc0051293490008.gif" src-large="/pmc/articles/instance/3268412/bin/zbc0051293490008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3268412/" ref="ordinalpos=1188&amp;ncbi_uid=4543368&amp;link_uid=PMC3268412" image-link="/pmc/articles/PMC3268412/figure/F8/" class="imagepopup">FIGURE 8. From: Integrins Regulate Microtubule Nucleating Activity of Centrosome through Mitogen-activated Protein Kinase/Extracellular Signal-regulated Kinase Kinase/Extracellular Signal-regulated Kinase (MEK/ERK) <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model.</b> Our data are consistent with published reports that the androgen receptor and Src form a <span class="highlight" style="background-color:">signaling</span> module that activates MEK/ERK and PI3K/AKT <span class="highlight" style="background-color:">signaling</span> in response to stimulation with androgen. We show that integrins regulate the formation of the androgen receptor-Src complex and the ability of androgen to trigger the activation of AKT and ERK. Additionally, our results demonstrate that MEK/ERK <span class="highlight" style="background-color:">signaling</span> promotes the centrosomal accumulation of γ-tubulin and microtubule nucleation independent of cell type and culture conditions. Inhibiting MEK/ERK <span class="highlight" style="background-color:">signaling</span> with U0126 inhibits microtubule nucleation when cells are adhered by wild-type integrins, and expression of ca-RAF to activate MEK/ERK <span class="highlight" style="background-color:">signaling</span> is sufficient to rescue microtubule nucleation when integrin function is inhibited. Although inhibiting PI3K/AKT <span class="highlight" style="background-color:">signaling</span> does not effect microtubule nucleation, this <span class="highlight" style="background-color:">pathway</span> may regulate other processes required for the assembly of the microtubule cytoskeleton.</div></p></div><div class="aux"><div class="resc">Diane Colello, et al. J Biol Chem. 2012 January 20;287(4):2520-2530.</div><p class="links"><a class="dblinks" href="/pubmed/22117069" ref="ordinalpos=1188&amp;ncbi_uid=4543368&amp;link_uid=22117069">Citation</a><a class="dblinks" href="/pmc/articles/PMC3268412/" ref="ordinalpos=1188&amp;ncbi_uid=4543368&amp;link_uid=PMC3268412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1189.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3821079/figure/F3/" ref="ordinalpos=1189&amp;ncbi_uid=5472687&amp;link_uid=PMC3821079" href="/pmc/articles/PMC3821079/"><img src="/pmc/articles/instance/3821079/bin/TOMCJ-7-23_F3.gif" src-large="/pmc/articles/instance/3821079/bin/TOMCJ-7-23_F3.jpg" alt="Fig. (3)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3821079/" ref="ordinalpos=1189&amp;ncbi_uid=5472687&amp;link_uid=PMC3821079" image-link="/pmc/articles/PMC3821079/figure/F3/" class="imagepopup">Fig. (3).  From: Roles of PI3K/AKT/PTEN <span class="highlight" style="background-color:">Pathway</span> as a Target for Pharmaceutical Therapy. </a></p><div class="supp"><p class="details"><div>Several inhibitors linked to the PI3K/AKT/mTOR/PTEN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown. Arrowhead means stimulation whereas hammerhead represents inhibition, suggesting implication of PI3K/AKT/mTOR/PTEN modulators for pharmaceutical therapy of various diseases.</div></p></div><div class="aux"><div class="resc">Satoru Matsuda, et al. Open Med Chem J. 2013;7:23-29.</div><p class="links"><a class="dblinks" href="/pubmed/24222802" ref="ordinalpos=1189&amp;ncbi_uid=5472687&amp;link_uid=24222802">Citation</a><a class="dblinks" href="/pmc/articles/PMC3821079/" ref="ordinalpos=1189&amp;ncbi_uid=5472687&amp;link_uid=PMC3821079">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1190.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2778955/figure/pone-0008079-g010/" ref="ordinalpos=1190&amp;ncbi_uid=2132745&amp;link_uid=PMC2778955" href="/pmc/articles/PMC2778955/"><img src="/pmc/articles/instance/2778955/bin/pone.0008079.g010.gif" src-large="/pmc/articles/instance/2778955/bin/pone.0008079.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2778955/" ref="ordinalpos=1190&amp;ncbi_uid=2132745&amp;link_uid=PMC2778955" image-link="/pmc/articles/PMC2778955/figure/pone-0008079-g010/" class="imagepopup">Figure 10. IKKγ-WT-IKKγΔ affects IRF3-NF-κB <span class="highlight" style="background-color:">pathway</span> utilization in ssRNA viral infection..  From: Expression of an IKK? Splice Variant Determines IRF3 and Canonical NF-?B <span class="highlight" style="background-color:">Pathway</span> Utilization in ssRNA Virus Infection. </a></p><div class="supp"><p class="details"><div>Schematic diagram of two distinct <span class="highlight" style="background-color:">signaling</span> complexes produced by IKKγ splice variants. RIG-I recognition of ssRNA intermediates induces the activated RIG-IMAVS <span class="highlight" style="background-color:">signaling</span> complex. Depending on the relative abundance of IKKγ-WT and IKKγΔ isoforms, distinct <span class="highlight" style="background-color:">signaling</span> pathways are activated downstream. Mito, mitochondrion.</div></p></div><div class="aux"><div class="resc">Ping Liu, et al. PLoS One. 2009;4(11):e8079.</div><p class="links"><a class="dblinks" href="/pubmed/19956647" ref="ordinalpos=1190&amp;ncbi_uid=2132745&amp;link_uid=19956647">Citation</a><a class="dblinks" href="/pmc/articles/PMC2778955/" ref="ordinalpos=1190&amp;ncbi_uid=2132745&amp;link_uid=PMC2778955">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1191.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2705893/figure/F1/" ref="ordinalpos=1191&amp;ncbi_uid=1952946&amp;link_uid=PMC2705893" href="/pmc/articles/PMC2705893/"><img src="/pmc/articles/instance/2705893/bin/nihms26841f1.gif" src-large="/pmc/articles/instance/2705893/bin/nihms26841f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2705893/" ref="ordinalpos=1191&amp;ncbi_uid=1952946&amp;link_uid=PMC2705893" image-link="/pmc/articles/PMC2705893/figure/F1/" class="imagepopup">Figure 1. Schematic of major oncogenic PKCι <span class="highlight" style="background-color:">signaling</span> pathways.  From: Protein kinase C?: human oncogene, prognostic marker and therapeutic target. </a></p><div class="supp"><p class="details"><div>PKCι resides in several major <span class="highlight" style="background-color:">signaling</span> pathways implicated in human cancer. Many components of these <span class="highlight" style="background-color:">pathway</span> are mutated, often by multiple mechanisms (ie. gene amplification and somatic mutation), in human tumors (indicated by yellow boxes). Arrows indicate flow through <span class="highlight" style="background-color:">signaling</span> pathways; touching boxes indicate directly binding of <span class="highlight" style="background-color:">signaling</span> components. Phosphorylation events are indicated by circled Ps.</div></p></div><div class="aux"><div class="resc">Alan P. Fields, et al. Pharmacol Res. ;55(6):487-497.</div><p class="links"><a class="dblinks" href="/pubmed/17570678" ref="ordinalpos=1191&amp;ncbi_uid=1952946&amp;link_uid=17570678">Citation</a><a class="dblinks" href="/pmc/articles/PMC2705893/" ref="ordinalpos=1191&amp;ncbi_uid=1952946&amp;link_uid=PMC2705893">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1192.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4170618/figure/F6/" ref="ordinalpos=1192&amp;ncbi_uid=6639593&amp;link_uid=PMC4170618" href="/pmc/articles/PMC4170618/"><img src="/pmc/articles/instance/4170618/bin/oncotarget-05-5483-g006.gif" src-large="/pmc/articles/instance/4170618/bin/oncotarget-05-5483-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4170618/" ref="ordinalpos=1192&amp;ncbi_uid=6639593&amp;link_uid=PMC4170618" image-link="/pmc/articles/PMC4170618/figure/F6/" class="imagepopup">Figure 6. A two-step killing strategy of glioblastoma multiforme stem cells.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> sensitizes Glioma stem cells to endogenous nano-irradiation. </a></p><div class="supp"><p class="details"><div>Glioma stem cells are activated by HH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to enter mitosis. The proliferating glioma stem cells incorporate the radiolabelled thymidine analogue [I-125]ITdU into the DNA via the salvage <span class="highlight" style="background-color:">pathway</span>. This effect is further enhanced by simultaneous FdUrd mediated inhibition of the <i>de novo</i> <span class="highlight" style="background-color:">pathway</span> of thymidine synthesis.</div></p></div><div class="aux"><div class="resc">Agnieszka Morgenroth, et al. Oncotarget. 2014 July;5(14):5483-5493.</div><p class="links"><a class="dblinks" href="/pubmed/24978848" ref="ordinalpos=1192&amp;ncbi_uid=6639593&amp;link_uid=24978848">Citation</a><a class="dblinks" href="/pmc/articles/PMC4170618/" ref="ordinalpos=1192&amp;ncbi_uid=6639593&amp;link_uid=PMC4170618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1193.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3177754/figure/F3/" ref="ordinalpos=1193&amp;ncbi_uid=3475831&amp;link_uid=PMC3177754" href="/pmc/articles/PMC3177754/"><img src="/pmc/articles/instance/3177754/bin/nihms-299486-f0003.gif" src-large="/pmc/articles/instance/3177754/bin/nihms-299486-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3177754/" ref="ordinalpos=1193&amp;ncbi_uid=3475831&amp;link_uid=PMC3177754" image-link="/pmc/articles/PMC3177754/figure/F3/" class="imagepopup">Figure 3. Viral regulation of RLR <span class="highlight" style="background-color:">signaling</span>.  From: RIG-I Like Receptors and Their <span class="highlight" style="background-color:">Signaling</span> Crosstalk in the Regulation of Antiviral Immunity. </a></p><div class="supp"><p class="details"><div>The RLR signalsome is shown including the RNA Polymerase/RIG-I <span class="highlight" style="background-color:">pathway</span> for indirect recognition of cytosolic foreign DNA. Viral proteins known to block RLR <span class="highlight" style="background-color:">signaling</span> are shown with lines associating the specific factors they target. Viruses or viral Genus are denoted in parenthesis next to their specific proteins.<br />Abbreviations: CCHFV (Crimean Congo Hemorrhagic Fever virus), BDV(Borna Disease virus), HTNV (Haantan Virus).</div></p></div><div class="aux"><div class="resc">Hilario J Ramos, et al. Curr Opin Virol. ;1(3):167-176.</div><p class="links"><a class="dblinks" href="/pubmed/21949557" ref="ordinalpos=1193&amp;ncbi_uid=3475831&amp;link_uid=21949557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3177754/" ref="ordinalpos=1193&amp;ncbi_uid=3475831&amp;link_uid=PMC3177754">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1194.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3531865/figure/fig7/" ref="ordinalpos=1194&amp;ncbi_uid=4493217&amp;link_uid=PMC3531865" href="/pmc/articles/PMC3531865/"><img src="/pmc/articles/instance/3531865/bin/PC_105312_lw_f7.gif" src-large="/pmc/articles/instance/3531865/bin/PC_105312_lw_f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3531865/" ref="ordinalpos=1194&amp;ncbi_uid=4493217&amp;link_uid=PMC3531865" image-link="/pmc/articles/PMC3531865/figure/fig7/" class="imagepopup">Figure 7. From: The Coronatine Toxin of Pseudomonas syringae Is a Multifunctional Suppressor of Arabidopsis Defense[W][OA]. </a></p><div class="supp"><p class="details"><div>Model: Suppression of the <i>Arabidopsis</i> Immune Response by COR.<br />Unknown elicitors from Pto (PAMPs and/or T3Es) activate SA <span class="highlight" style="background-color:">signaling</span> and indole glucosinolate metabolism. SA <span class="highlight" style="background-color:">signaling</span> and JA <span class="highlight" style="background-color:">signaling</span> are antagonistic, with COR activating COI1 to suppress SA accumulation. COR also suppresses indole glucosinolate metabolism upstream of 4MI3G accumulation. SA <span class="highlight" style="background-color:">signaling</span> promotes indole glucosinolate metabolism, perhaps by suppressing JA accumulation, and also promotes callose deposition independent of COI1 through an unknown <span class="highlight" style="background-color:">pathway</span>. COR suppresses defense in a COI1-independent manner through an unknown <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Xueqing Geng, et al. Plant Cell. 2012 November;24(11):4763-4774.</div><p class="links"><a class="dblinks" href="/pubmed/23204405" ref="ordinalpos=1194&amp;ncbi_uid=4493217&amp;link_uid=23204405">Citation</a><a class="dblinks" href="/pmc/articles/PMC3531865/" ref="ordinalpos=1194&amp;ncbi_uid=4493217&amp;link_uid=PMC3531865">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1195.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2100073/figure/F6/" ref="ordinalpos=1195&amp;ncbi_uid=1122436&amp;link_uid=PMC2100073" href="/pmc/articles/PMC2100073/"><img src="/pmc/articles/instance/2100073/bin/1471-2105-8-335-6.gif" src-large="/pmc/articles/instance/2100073/bin/1471-2105-8-335-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2100073/" ref="ordinalpos=1195&amp;ncbi_uid=1122436&amp;link_uid=PMC2100073" image-link="/pmc/articles/PMC2100073/figure/F6/" class="imagepopup">Figure 6.  From: PathFinder: mining signal transduction <span class="highlight" style="background-color:">pathway</span> segments from protein-protein interaction networks. </a></p><div class="supp"><p class="details"><div><b>The filamentation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> recovered completely</b>. (A) The main chain of the filamentation <span class="highlight" style="background-color:">pathway</span> (KEGG Database), (B) PathFinder output for Sho1-Tec1 pair (100% recall, 12% precision). For (A), the dashed links indicate interactions that do not exists in the database. For (B), the interactions that were predicted as false negatives were shown with dashed lines. The proteins that were not on the main chain of any <span class="highlight" style="background-color:">pathway</span> were not colored, whereas proteins that are part of other pathways were colored.</div></p></div><div class="aux"><div class="resc">Gurkan Bebek, et al. BMC Bioinformatics. 2007;8:335-335.</div><p class="links"><a class="dblinks" href="/pubmed/17854489" ref="ordinalpos=1195&amp;ncbi_uid=1122436&amp;link_uid=17854489">Citation</a><a class="dblinks" href="/pmc/articles/PMC2100073/" ref="ordinalpos=1195&amp;ncbi_uid=1122436&amp;link_uid=PMC2100073">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1196.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4367559/figure/f1/" ref="ordinalpos=1196&amp;ncbi_uid=7320907&amp;link_uid=PMC4367559" href="/pmc/articles/PMC4367559/"><img src="/pmc/articles/instance/4367559/bin/tlcr-01-03-179-f1.gif" src-large="/pmc/articles/instance/4367559/bin/tlcr-01-03-179-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4367559/" ref="ordinalpos=1196&amp;ncbi_uid=7320907&amp;link_uid=PMC4367559" image-link="/pmc/articles/PMC4367559/figure/f1/" class="imagepopup">Figure 1.  From: Role of HGF/MET axis in resistance of lung cancer to contemporary management. </a></p><div class="supp"><p class="details"><div>Schematic representation of MET/HGF <span class="highlight" style="background-color:">signaling</span> and functional cross-talk with other pathways. Hepatocyte growth factor (HGF) binds to MET receptor and activates kinase activity which then phosphorylates the docking site and recruits effector molecules [Growth factor receptor-bound protein 2 (GRB2), GRB2-associated-binding protein 1 (GAB1), SRC homology 2 domain-containing phosphatase 2 (SHP2), Son of sevenless-1 (SOS-1) and Sarcoma non-receptor tyrosine kinase (SRC)]. Downstream <span class="highlight" style="background-color:">signaling</span> pathways such as the Mitogen-activated protein kinase (ERK/MAPK) <span class="highlight" style="background-color:">pathway</span>, the Phosphatidylinositol 3 kinase (PI3K-AKT) <span class="highlight" style="background-color:">pathway</span> and the Phospholipase-Cγ (PLC-γ) <span class="highlight" style="background-color:">pathway</span> are activated. These effector pathways are shared by the epidermal growth factor receptor (EGFR/EGF) <span class="highlight" style="background-color:">pathway</span>, the vascular epidermal growth factor (VEGFR-2/VEGF-A) <span class="highlight" style="background-color:">pathway</span>, the HER2 <span class="highlight" style="background-color:">pathway</span> and the EML4-ALK <span class="highlight" style="background-color:">pathway</span>. The signals affect gene expression and promote cell proliferation and survival, angiogenesis and cytoskeletal alterations resulting in cancer growth and progression. EGFR and HER2 can Trans-phosphorylate and activate MET. MET activation increases transcriptional expression of VEGF-A and WNT ligand and activates VEGF <span class="highlight" style="background-color:">pathway</span> and WNT-β-catenin <span class="highlight" style="background-color:">pathway</span>, respectively. β-catenin increases MET expression.</div></p></div><div class="aux"><div class="resc">Kanwal Pratap Singh Raghav, et al. Transl Lung Cancer Res. 2012 September;1(3):179-193.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1196&amp;ncbi_uid=7320907&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4367559/" ref="ordinalpos=1196&amp;ncbi_uid=7320907&amp;link_uid=PMC4367559">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1197.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g002/" ref="ordinalpos=1197&amp;ncbi_uid=4989295&amp;link_uid=PMC3677896" href="/pmc/articles/PMC3677896/"><img src="/pmc/articles/instance/3677896/bin/pone.0065032.g002.gif" src-large="/pmc/articles/instance/3677896/bin/pone.0065032.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3677896/" ref="ordinalpos=1197&amp;ncbi_uid=4989295&amp;link_uid=PMC3677896" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g002/" class="imagepopup">Figure 2. Evidence for SHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in irradiated Panc1 and HT29 cells..  From: Sonic Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Supports Cancer Cell Growth during Cancer Radiotherapy. </a></p><div class="supp"><p class="details"><div>A. Expression-profile changes of Shh and Gli1 proteins in Panc1 cells irradiated at various doses and detected by Western blot. B. Expression-profile changes of Shh and Gli1 proteins in HT29 cells irradiated at various doses and detected by Western blot. C. Relative intensity of Shh and Gli1 protein bands on Western blot in Panc1 cells irradiated at various doses. D. Relative intensity of Shh and Gli1 protein bands on Western blot in HT29 cells irradiated at various doses. E. Luciferase activity of Gli1 reporter in irradiated and non-irradiated Panc1 cells. **represents <i>P</i>&lt;0.01. F. Luciferase activity of Gli1 reporter in irradiated and non-irradiated HT29 cells. **represents <i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Jingjing Ma, et al. PLoS One. 2013;8(6):e65032.</div><p class="links"><a class="dblinks" href="/pubmed/23762282" ref="ordinalpos=1197&amp;ncbi_uid=4989295&amp;link_uid=23762282">Citation</a><a class="dblinks" href="/pmc/articles/PMC3677896/" ref="ordinalpos=1197&amp;ncbi_uid=4989295&amp;link_uid=PMC3677896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1198.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3672788/figure/F6/" ref="ordinalpos=1198&amp;ncbi_uid=4979811&amp;link_uid=PMC3672788" href="/pmc/articles/PMC3672788/"><img src="/pmc/articles/instance/3672788/bin/ar4162-6.gif" src-large="/pmc/articles/instance/3672788/bin/ar4162-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3672788/" ref="ordinalpos=1198&amp;ncbi_uid=4979811&amp;link_uid=PMC3672788" image-link="/pmc/articles/PMC3672788/figure/F6/" class="imagepopup">Figure 6.  From: Regulatory effect of calcineurin inhibitor, tacrolimus, on IL-6/sIL-6R-mediated RANKL expression through JAK2-STAT3-SOCS3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in fibroblast-like synoviocytes. </a></p><div class="supp"><p class="details"><div><b>Summary for the effect of calcineurin inhibitor, tacrolimus, on the regulation of RANKL expression through IL-6/sIL-6R/JAK2/STAT3/SOCS3 <span class="highlight" style="background-color:">pathway</span></b>. IL-6, interleukin-6; JAK2, Janus activated kinase; RANKL, receptor activator of NF-κB ligand; sIL-6R, soluble interleukin-6 receptor; SOCS3, suppressor of cytokine <span class="highlight" style="background-color:">signaling</span> 3; STAT3, signal transducer and activator of transcription 3.</div></p></div><div class="aux"><div class="resc">Jung-Yoon Choe, et al. Arthritis Res Ther. 2013;15(1):R26-R26.</div><p class="links"><a class="dblinks" href="/pubmed/23406906" ref="ordinalpos=1198&amp;ncbi_uid=4979811&amp;link_uid=23406906">Citation</a><a class="dblinks" href="/pmc/articles/PMC3672788/" ref="ordinalpos=1198&amp;ncbi_uid=4979811&amp;link_uid=PMC3672788">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1199.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2956057/figure/Fig1/" ref="ordinalpos=1199&amp;ncbi_uid=2570233&amp;link_uid=PMC2956057" href="/pmc/articles/PMC2956057/"><img src="/pmc/articles/instance/2956057/bin/109_2010_644_Fig1_HTML.gif" src-large="/pmc/articles/instance/2956057/bin/109_2010_644_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2956057/" ref="ordinalpos=1199&amp;ncbi_uid=2570233&amp;link_uid=PMC2956057" image-link="/pmc/articles/PMC2956057/figure/Fig1/" class="imagepopup">Fig. 1.  From: Mouse models of cancers: opportunities to address heterogeneity of human cancer and evaluate therapeutic strategies. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> probability in human tumors. Human breast cancer gene expression data from previous studies were downloaded (GSE4922 and GSE15852), normalized and the two datasets were merged together using the Affymetrix housekeeping genes to standardize between batches. The combined dataset was then examined for patterns of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. Upon generating <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation probabilities, the data was clustered and the resulting heat map is shown for the pathways indicated at the <i>right</i>. For a given <span class="highlight" style="background-color:">pathway</span>, <i>blue</i> represents a low probability of <span class="highlight" style="background-color:">pathway</span> activation while <i>red</i> represents a high probability of activation. These datasets also included clinical status for the <i>grade</i> of the tumor (<i>1</i>, <i>2</i>, <i>3</i>) and in one dataset normal breast samples were included (GSE15852). This additional clinical data is shown in the legend above the heat map. Using only <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> probabilities, the heterogeneity within human tumors is readily apparent</div></p></div><div class="aux"><div class="resc">Eran R. Andrechek, et al. J Mol Med. 2010 November;88(11):1095-1100.</div><p class="links"><a class="dblinks" href="/pubmed/20574808" ref="ordinalpos=1199&amp;ncbi_uid=2570233&amp;link_uid=20574808">Citation</a><a class="dblinks" href="/pmc/articles/PMC2956057/" ref="ordinalpos=1199&amp;ncbi_uid=2570233&amp;link_uid=PMC2956057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1200.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4131049/figure/Fig7/" ref="ordinalpos=1200&amp;ncbi_uid=6494327&amp;link_uid=PMC4131049" href="/pmc/articles/PMC4131049/"><img src="/pmc/articles/instance/4131049/bin/12885_2013_4746_Fig7_HTML.gif" src-large="/pmc/articles/instance/4131049/bin/12885_2013_4746_Fig7_HTML.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4131049/" ref="ordinalpos=1200&amp;ncbi_uid=6494327&amp;link_uid=PMC4131049" image-link="/pmc/articles/PMC4131049/figure/Fig7/" class="imagepopup">Figure 7.  From: Benzyl butyl phthalate induces migration, invasion, and angiogenesis of Huh7 hepatocellular carcinoma cells through nongenomic AhR/G-protein <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div> <b>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of Huh7 cells treated with BBP (see the text for details).</b> </div></p></div><div class="aux"><div class="resc">Cheng-Fang Tsai, et al. BMC Cancer. 2014;14(1):556.</div><p class="links"><a class="dblinks" href="/pubmed/25081364" ref="ordinalpos=1200&amp;ncbi_uid=6494327&amp;link_uid=25081364">Citation</a><a class="dblinks" href="/pmc/articles/PMC4131049/" ref="ordinalpos=1200&amp;ncbi_uid=6494327&amp;link_uid=PMC4131049">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="11" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="12" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="13" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:23:59-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC70BF53FEC191000000000095762F;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC70BF53FEC191000000000095762F" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>