
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="45" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB1FD954312FD10000000000808A0F" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="44" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="45" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="46" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="45" /></div>
            <div><h2 class="result_count">Results: 4401 to 4500 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4401.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3701595/figure/pone-0067880-g008/" ref="ordinalpos=4401&amp;ncbi_uid=5098053&amp;link_uid=PMC3701595" href="/pmc/articles/PMC3701595/"><img src="/pmc/articles/instance/3701595/bin/pone.0067880.g008.gif" src-large="/pmc/articles/instance/3701595/bin/pone.0067880.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3701595/" ref="ordinalpos=4401&amp;ncbi_uid=5098053&amp;link_uid=PMC3701595" image-link="/pmc/articles/PMC3701595/figure/pone-0067880-g008/" class="imagepopup">Figure 8. Acetic acid activates the AMP-activated protein kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to regulate lipid metabolism in bovine hepatocytes..  From: Acetic Acid Activates the AMP-Activated Protein Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> to Regulate Lipid Metabolism in Bovine Hepatocytes. </a></p><div class="supp"><p class="details"><div>Acetic acid is metabolized to acetyl-CoA with consumption of ATP in bovine hepatocytes, resulting in a significant elevation of the AMP/ATP ratio and the phosphorylation and activation of AMPKα. Activated AMPKα increases the expression and transcriptional activity of PPARα, thereby increasing the expression of lipolytic genes, including ACO, CPT1, CPT2, and L-FABP. AMPKα activation inhibits the expression and transcriptional activity of SREBP-1c and ChREBP, thereby reducing the expression of lipogenic genes, including ACC1, FAS, and SCD-1. In addition, activated AMPKα directly phosphorylates and inhibits ACC1. Consequently, acetic acid increases lipolysis and reduces lipid synthesis in bovine hepatocytes.</div></p></div><div class="aux"><div class="resc">Xinwei Li, et al. PLoS One. 2013;8(7):e67880.</div><p class="links"><a class="dblinks" href="/pubmed/23861826" ref="ordinalpos=4401&amp;ncbi_uid=5098053&amp;link_uid=23861826">Citation</a><a class="dblinks" href="/pmc/articles/PMC3701595/" ref="ordinalpos=4401&amp;ncbi_uid=5098053&amp;link_uid=PMC3701595">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4402.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2919048/figure/pone-0012016-g007/" ref="ordinalpos=4402&amp;ncbi_uid=2417988&amp;link_uid=PMC2919048" href="/pmc/articles/PMC2919048/"><img src="/pmc/articles/instance/2919048/bin/pone.0012016.g007.gif" src-large="/pmc/articles/instance/2919048/bin/pone.0012016.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919048/" ref="ordinalpos=4402&amp;ncbi_uid=2417988&amp;link_uid=PMC2919048" image-link="/pmc/articles/PMC2919048/figure/pone-0012016-g007/" class="imagepopup">Figure 7. TGFbeta <span class="highlight" style="background-color:">signaling</span> pathways are active in heart inducing cell lines..  From: A Comparative Analysis of Extra-Embryonic Endoderm Cell Lines. </a></p><div class="supp"><p class="details"><div>Diagram of TGFbeta <span class="highlight" style="background-color:">signaling</span> pathways from the Kyoto Encyclopedia of                             Genes and Genomes [75], [76], [77] showing <span class="highlight" style="background-color:">pathway</span> components that are                             considered to be present based on a p-value of detection &lt;0.01.                             Factors indicated in blue are present in all of the ExEn cell lines.                             Factors present in green are present in a subset of the cell lines (the                             particular cell lines are indicated by letters adjacent to the box                             indicating the factor).</div></p></div><div class="aux"><div class="resc">Kemar Brown, et al. PLoS One. 2010;5(8):e12016.</div><p class="links"><a class="dblinks" href="/pubmed/20711519" ref="ordinalpos=4402&amp;ncbi_uid=2417988&amp;link_uid=20711519">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919048/" ref="ordinalpos=4402&amp;ncbi_uid=2417988&amp;link_uid=PMC2919048">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4403.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2043540/figure/F7/" ref="ordinalpos=4403&amp;ncbi_uid=1151995&amp;link_uid=PMC2043540" href="/pmc/articles/PMC2043540/"><img src="/pmc/articles/instance/2043540/bin/zmk0110782710007.gif" src-large="/pmc/articles/instance/2043540/bin/zmk0110782710007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2043540/" ref="ordinalpos=4403&amp;ncbi_uid=1151995&amp;link_uid=PMC2043540" image-link="/pmc/articles/PMC2043540/figure/F7/" class="imagepopup">Figure 7. From: Parallels between Global Transcriptional Programs of Polarizing Caco-2 Intestinal Epithelial Cells In Vitro and Gene Expression Programs in Normal Colon and Colon Cancer. </a></p><div class="supp"><p class="details"><div>Regulation of the noncanonical Wnt <span class="highlight" style="background-color:">pathway</span>. (A) Schematic of noncanonical Wnt <span class="highlight" style="background-color:">pathway</span> proteins and their possible effect on canonical Wnt <span class="highlight" style="background-color:">signaling</span>. (B) Transcriptional levels of noncanonical Wnt <span class="highlight" style="background-color:">pathway</span> components increased during Caco-2 cell polarization in vitro. Transcript levels determined by microarray analysis are shown relative to a reference pool of human mRNAs.</div></p></div><div class="aux"><div class="resc">Annika M. Sääf, et al. Mol Biol Cell. 2007 November;18(11):4245-4260.</div><p class="links"><a class="dblinks" href="/pubmed/17699589" ref="ordinalpos=4403&amp;ncbi_uid=1151995&amp;link_uid=17699589">Citation</a><a class="dblinks" href="/pmc/articles/PMC2043540/" ref="ordinalpos=4403&amp;ncbi_uid=1151995&amp;link_uid=PMC2043540">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4404.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4110715/figure/F5/" ref="ordinalpos=4404&amp;ncbi_uid=6464517&amp;link_uid=PMC4110715" href="/pmc/articles/PMC4110715/"><img src="/pmc/articles/instance/4110715/bin/1755-8794-7-S1-S3-5.gif" src-large="/pmc/articles/instance/4110715/bin/1755-8794-7-S1-S3-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4110715/" ref="ordinalpos=4404&amp;ncbi_uid=6464517&amp;link_uid=PMC4110715" image-link="/pmc/articles/PMC4110715/figure/F5/" class="imagepopup">Figure 5.  From: Integrative analysis reveals disease-associated genes and biomarkers for prostate cancer progression. </a></p><div class="supp"><p class="details"><div><b>GeneGo graphic representation illustrates Transcription Androgen Receptor (AR) nuclear <span class="highlight" style="background-color:">signaling</span></b>. <span class="highlight" style="background-color:">Pathway</span> regarded as the enriched significant <span class="highlight" style="background-color:">pathway</span> associated in prostate cancer progression. Transcription factor AR was identified as a hub protein to play a critical role in prostate tumorigenesis and prostate cancer progression.</div></p></div><div class="aux"><div class="resc">Yin Li, et al. BMC Med Genomics. 2014;7(Suppl 1):S3-S3.</div><p class="links"><a class="dblinks" href="/pubmed/25080090" ref="ordinalpos=4404&amp;ncbi_uid=6464517&amp;link_uid=25080090">Citation</a><a class="dblinks" href="/pmc/articles/PMC4110715/" ref="ordinalpos=4404&amp;ncbi_uid=6464517&amp;link_uid=PMC4110715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4405.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3614511/figure/pbio-1001527-g008/" ref="ordinalpos=4405&amp;ncbi_uid=4783495&amp;link_uid=PMC3614511" href="/pmc/articles/PMC3614511/"><img src="/pmc/articles/instance/3614511/bin/pbio.1001527.g008.gif" src-large="/pmc/articles/instance/3614511/bin/pbio.1001527.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3614511/" ref="ordinalpos=4405&amp;ncbi_uid=4783495&amp;link_uid=PMC3614511" image-link="/pmc/articles/PMC3614511/figure/pbio-1001527-g008/" class="imagepopup">Figure 8. A working model for HIPK proteins in the transcriptional control of angiogenic gene expression in the downstream of TGF-β-TAK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Transcriptional Corepressors HIPK1 and HIPK2 Control Angiogenesis Via TGF-?-TAK1-Dependent Mechanism. </a></p><div class="aux"><div class="resc">Yulei Shang, et al. PLoS Biol. 2013 April;11(4):e1001527.</div><p class="links"><a class="dblinks" href="/pubmed/23565059" ref="ordinalpos=4405&amp;ncbi_uid=4783495&amp;link_uid=23565059">Citation</a><a class="dblinks" href="/pmc/articles/PMC3614511/" ref="ordinalpos=4405&amp;ncbi_uid=4783495&amp;link_uid=PMC3614511">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4406.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3712052/figure/F3/" ref="ordinalpos=4406&amp;ncbi_uid=6335317&amp;link_uid=PMC3712052" href="/pmc/articles/PMC3712052/"><img src="/pmc/articles/instance/3712052/bin/2605fig3.gif" src-large="/pmc/articles/instance/3712052/bin/2605fig3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3712052/" ref="ordinalpos=4406&amp;ncbi_uid=6335317&amp;link_uid=PMC3712052" image-link="/pmc/articles/PMC3712052/figure/F3/" class="imagepopup">FIG. 3. From: From Single Nucleotide Polymorphism to Transcriptional Mechanism. </a></p><div class="supp"><p class="details"><div>Functional association of <i>FRMD3</i>-correlated genes. Top 10 pathways (Ingenuity Pathways Analysis; Ingenuity Systems) of 581 <i>FRMD3</i>-correlated genes sorted by the ratio of members of the <span class="highlight" style="background-color:">pathway</span> among <i>FRMD3</i>-correlated genes vs. total number of members of that <span class="highlight" style="background-color:">pathway</span>. <i>P</i> ≤ 0.001 for all pathways. Junct., junction; med., mediated; Ox., oxidative; Pot., potentiation; Sign., <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Sebastian Martini, et al. Diabetes. 2013 July;62(7):2605-2612.</div><p class="links"><a class="dblinks" href="/pubmed/23434934" ref="ordinalpos=4406&amp;ncbi_uid=6335317&amp;link_uid=23434934">Citation</a><a class="dblinks" href="/pmc/articles/PMC3712052/" ref="ordinalpos=4406&amp;ncbi_uid=6335317&amp;link_uid=PMC3712052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4407.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2706146/figure/F3/" ref="ordinalpos=4407&amp;ncbi_uid=1956954&amp;link_uid=PMC2706146" href="/pmc/articles/PMC2706146/"><img src="/pmc/articles/instance/2706146/bin/nihms42936f3.gif" src-large="/pmc/articles/instance/2706146/bin/nihms42936f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2706146/" ref="ordinalpos=4407&amp;ncbi_uid=1956954&amp;link_uid=PMC2706146" image-link="/pmc/articles/PMC2706146/figure/F3/" class="imagepopup">Fig. 3.  From: The growth hormone-insulin-like growth factor-I axis in chronic kidney disease. </a></p><div class="supp"><p class="details"><div>Molecular mechanism of myostatin in muscle mass regulation in CKD. Myostatin appears to induce muscle wasting independent of NF-κB <span class="highlight" style="background-color:">pathway</span>. Increased levels of myostatin blocks myogenesis by downregulating <i>pax3</i> and <i>myoD</i> expression. In addition, myostatin upregulates proteolysis by phosphorylating FOXO1 through the inhibition of the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Blunt-ended line represents inhibition.</div></p></div><div class="aux"><div class="resc">Robert H. Mak, et al. Growth Horm IGF Res. ;18(1):17-25.</div><p class="links"><a class="dblinks" href="/pubmed/17826224" ref="ordinalpos=4407&amp;ncbi_uid=1956954&amp;link_uid=17826224">Citation</a><a class="dblinks" href="/pmc/articles/PMC2706146/" ref="ordinalpos=4407&amp;ncbi_uid=1956954&amp;link_uid=PMC2706146">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4408.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2556613/figure/F1/" ref="ordinalpos=4408&amp;ncbi_uid=1530988&amp;link_uid=PMC2556613" href="/pmc/articles/PMC2556613/"><img src="/pmc/articles/instance/2556613/bin/zh80200824920001.gif" src-large="/pmc/articles/instance/2556613/bin/zh80200824920001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2556613/" ref="ordinalpos=4408&amp;ncbi_uid=1530988&amp;link_uid=PMC2556613" image-link="/pmc/articles/PMC2556613/figure/F1/" class="imagepopup">Figure 1.  From: Identification of Fc?RIIa as the ITAM-bearing receptor mediating ?IIb?3 outside-in integrin <span class="highlight" style="background-color:">signaling</span> in human platelets. </a></p><div class="supp"><p class="details"><div><b>Outside-in <span class="highlight" style="background-color:">signaling</span> initiated by platelet interaction with immobilized fibrinogen activates the FcγRIIa signal transduction <span class="highlight" style="background-color:">pathway</span></b>. (A) Schematic showing use of the FcγRIIa <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream of αIIbβ3 engagement. Note the importance of physical approximation of the integrin with FcγRIIa, allowing integrin-associated Src family members to function as ITAM kinases, thus initiating the outside-in <span class="highlight" style="background-color:">signaling</span> cascade. (B,C) Washed platelets were plated onto 8-chamber glass tissue-culture slides coated with either BSA or fibrinogen (Fg), and allowed to spread for 45 minutes in the presence or absence of 10 μg/mL IV.3 Fab Immunoprecipitation and Western blot analysis (B) reveals strong activation of FcγRIIa, Syk, and PLCγ2 after platelet binding to immobilized fibrinogen, and inhibition of this outside-in <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Fab fragments of mAb IV.3. (C) Effect of IV.3 on platelet spreading. Preincubation of platelets with IV.3 Fab markedly inhibits platelet spreading on immobilized fibrinogen, shown in panels A through C, with quantitative analysis of the pixel area of spread platelets in panel. Data shown are the mean plus or minus SEM from one of 4 representative experiments using 3 different platelet donors.</div></p></div><div class="aux"><div class="resc">Brian Boylan, et al. Blood. 2008 October 1;112(7):2780-2786.</div><p class="links"><a class="dblinks" href="/pubmed/18641368" ref="ordinalpos=4408&amp;ncbi_uid=1530988&amp;link_uid=18641368">Citation</a><a class="dblinks" href="/pmc/articles/PMC2556613/" ref="ordinalpos=4408&amp;ncbi_uid=1530988&amp;link_uid=PMC2556613">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4409.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4394589/figure/Fig5/" ref="ordinalpos=4409&amp;ncbi_uid=7413379&amp;link_uid=PMC4394589" href="/pmc/articles/PMC4394589/"><img src="/pmc/articles/instance/4394589/bin/12864_2015_1475_Fig5_HTML.gif" src-large="/pmc/articles/instance/4394589/bin/12864_2015_1475_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4394589/" ref="ordinalpos=4409&amp;ncbi_uid=7413379&amp;link_uid=PMC4394589" image-link="/pmc/articles/PMC4394589/figure/Fig5/" class="imagepopup">Figure 5.  From: The transcriptome of a complete episode of acute otitis media. </a></p><div class="supp"><p class="details"><div> <b>Regulation of genes in the JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Diagrammatic representation of responses for genes involved in JAK/STAT <span class="highlight" style="background-color:">signaling</span>. For each gene, expression has been represented relative to 0 h across the course of AOM, from 3 h to 7d; with white representing no change, red: up-regulation and green: down-regulation. The degree of regulation is indicated by color intensity. Both proinflammatory and anti-inflammatory <span class="highlight" style="background-color:">signaling</span> genes are regulated with similar kinetics. An exception is the STAT inhibitor CAV1, which is down-regulated early in AOM.</div></p></div><div class="aux"><div class="resc">Michelle Hernandez, et al. BMC Genomics. 2015;16(1):259.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4409&amp;ncbi_uid=7413379&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4394589/" ref="ordinalpos=4409&amp;ncbi_uid=7413379&amp;link_uid=PMC4394589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4410.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2759143/figure/f1-grsb-2007-217/" ref="ordinalpos=4410&amp;ncbi_uid=2044648&amp;link_uid=PMC2759143" href="/pmc/articles/PMC2759143/"><img src="/pmc/articles/instance/2759143/bin/grsb-2007-217f1.gif" src-large="/pmc/articles/instance/2759143/bin/grsb-2007-217f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2759143/" ref="ordinalpos=4410&amp;ncbi_uid=2044648&amp;link_uid=PMC2759143" image-link="/pmc/articles/PMC2759143/figure/f1-grsb-2007-217/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Signaling</span> Mechanisms of Endogenous Angiogenesis Inhibitors Derived from Type IV Collagen. </a></p><div class="supp"><p class="details"><div>Schematic illustration of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by α1(IV)NC1. α1(IV)NC1 binds to α1β1 integrin and cross talk with growth factor receptors, and inhibit phosphorylation of FAK. Inhibition of FAK activation leads to inhibition of Raf/MEK/ERK1/2/p38 MAP kinase pathways that leads to inhibition of HIF-1α and VEGF expression which in turn results in inhibition of endothelial cell migration, proliferation and tube formation in proliferating endothelial cells.</div></p></div><div class="aux"><div class="resc">Akulapalli Sudhakar, et al. Gene Regul Syst Bio. 2007;1:217-226.</div><p class="links"><a class="dblinks" href="/pubmed/19936090" ref="ordinalpos=4410&amp;ncbi_uid=2044648&amp;link_uid=19936090">Citation</a><a class="dblinks" href="/pmc/articles/PMC2759143/" ref="ordinalpos=4410&amp;ncbi_uid=2044648&amp;link_uid=PMC2759143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4411.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2542903/figure/F6/" ref="ordinalpos=4411&amp;ncbi_uid=1476568&amp;link_uid=PMC2542903" href="/pmc/articles/PMC2542903/"><img src="/pmc/articles/instance/2542903/bin/nihms62958f6.gif" src-large="/pmc/articles/instance/2542903/bin/nihms62958f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2542903/" ref="ordinalpos=4411&amp;ncbi_uid=1476568&amp;link_uid=PMC2542903" image-link="/pmc/articles/PMC2542903/figure/F6/" class="imagepopup">Figure 6.  From: Quantitative Phosphotyrosine Proteomics of EphB2 <span class="highlight" style="background-color:">Signaling</span> by Stable Isotope Labeling with Amino Acids in Cell Culture (SILAC). </a></p><div class="supp"><p class="details"><div>Proposed EphB2 <span class="highlight" style="background-color:">signaling</span> network. The proposed <span class="highlight" style="background-color:">pathway</span> reflects SILAC data, <i>PathwayAssist</i>™ data, and interactions and protein functions found in the primary literature. Proteins in white ovals indicate previously established EphB2 interactions. Arrows indicate positive regulation or activation and blunt ended lines indicate inhibition. Proteins in boxes (Rac, Ras, and Erk) indicate known effectors not found in our experiments.</div></p></div><div class="aux"><div class="resc">Guoan Zhang, et al. J Proteome Res. ;5(3):581-588.</div><p class="links"><a class="dblinks" href="/pubmed/16512673" ref="ordinalpos=4411&amp;ncbi_uid=1476568&amp;link_uid=16512673">Citation</a><a class="dblinks" href="/pmc/articles/PMC2542903/" ref="ordinalpos=4411&amp;ncbi_uid=1476568&amp;link_uid=PMC2542903">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4412.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3544400/figure/F7/" ref="ordinalpos=4412&amp;ncbi_uid=4893191&amp;link_uid=PMC3544400" href="/pmc/articles/PMC3544400/"><img src="/pmc/articles/instance/3544400/bin/nihms421766f7.gif" src-large="/pmc/articles/instance/3544400/bin/nihms421766f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3544400/" ref="ordinalpos=4412&amp;ncbi_uid=4893191&amp;link_uid=PMC3544400" image-link="/pmc/articles/PMC3544400/figure/F7/" class="imagepopup">Figure 7. Go <span class="highlight" style="background-color:">signaling</span> downstream of the neuropeptide receptor EGL-6 is especially sensitive to irk-1 mutation.  From: IRK-1 potassium channels mediate peptidergic inhibition of C. elegans serotonin neurons via a Go <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) The egg-laying defect of <i>egl-6(gf)</i> mutants as measured by the abnormally late developmental stages of newly laid eggs was strongly suppressed by <i>irk-1</i> deletion.<br />(B) The egg-laying defect of <i>egl-10</i> mutants, which have increased G<sub>o</sub> <span class="highlight" style="background-color:">signaling</span> caused by loss of an RGS protein that inhibits G<sub>o</sub>α, was only partly suppressed by <i>irk-1</i> deletion.<br />(C) The egg-laying defect of animals carrying a gain-of-function mutation in EGL-47, another G<sub>o</sub>-coupled receptor that inhibits HSNs, was not suppressed by <i>irk-1</i> deletion.<br />(D) A model for the function of IRK-1 channels in the regulation of HSNs by G<sub>o</sub> <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Lesley Emtage, et al. J Neurosci. ;32(46):16285-16295.</div><p class="links"><a class="dblinks" href="/pubmed/23152612" ref="ordinalpos=4412&amp;ncbi_uid=4893191&amp;link_uid=23152612">Citation</a><a class="dblinks" href="/pmc/articles/PMC3544400/" ref="ordinalpos=4412&amp;ncbi_uid=4893191&amp;link_uid=PMC3544400">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4413.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3390346/figure/pone-0040255-g008/" ref="ordinalpos=4413&amp;ncbi_uid=4116420&amp;link_uid=PMC3390346" href="/pmc/articles/PMC3390346/"><img src="/pmc/articles/instance/3390346/bin/pone.0040255.g008.gif" src-large="/pmc/articles/instance/3390346/bin/pone.0040255.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3390346/" ref="ordinalpos=4413&amp;ncbi_uid=4116420&amp;link_uid=PMC3390346" image-link="/pmc/articles/PMC3390346/figure/pone-0040255-g008/" class="imagepopup">Figure 8. NMJ dismantling model..  From: Drosophila Motor Neuron Retraction during Metamorphosis Is Mediated by Inputs from TGF-?/BMP <span class="highlight" style="background-color:">Signaling</span> and Orphan Nuclear Receptors. </a></p><div class="supp"><p class="details"><div>The nuclear receptor <span class="highlight" style="background-color:">pathway</span> involving FTZ-F1 and HR39 regulates the <i>EcR-B1</i> expression in muscle and initiate dismantling. FTZ-F1 and HR39 positively and negatively respectively control the <i>EcR-B1</i> expression at the postsynapse. FTZ-F1-mediated <i>EcR-B1</i> expression leads to transcriptional regulation of target genes involved in postsynaptic degradation. Then, TGF-β/BMP <span class="highlight" style="background-color:">signaling</span> is supposed to be induced in response to postsynaptic degradation. We propose that this signal produced by the dismantling muscle is received by the motor neuron where activation of <i>EcR-B1</i> expression (by TGF-β <span class="highlight" style="background-color:">signaling</span> and nuclear receptor) triggers the axon retraction. In addition, glial cells and macrophages participate to the NMJ dismantling process.</div></p></div><div class="aux"><div class="resc">Ana Boulanger, et al. PLoS One. 2012;7(7):e40255.</div><p class="links"><a class="dblinks" href="/pubmed/22792255" ref="ordinalpos=4413&amp;ncbi_uid=4116420&amp;link_uid=22792255">Citation</a><a class="dblinks" href="/pmc/articles/PMC3390346/" ref="ordinalpos=4413&amp;ncbi_uid=4116420&amp;link_uid=PMC3390346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4414.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4003301/figure/fig5/" ref="ordinalpos=4414&amp;ncbi_uid=6627048&amp;link_uid=PMC4003301" href="/pmc/articles/PMC4003301/"><img src="/pmc/articles/instance/4003301/bin/aps201175f5.gif" src-large="/pmc/articles/instance/4003301/bin/aps201175f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4003301/" ref="ordinalpos=4414&amp;ncbi_uid=6627048&amp;link_uid=PMC4003301" image-link="/pmc/articles/PMC4003301/figure/fig5/" class="imagepopup">Figure 5.  From: Activating transcription factor 6 protects insulin receptor from ER stress-stimulated desensitization via p42/44 ERK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>ATF6 regulation of IR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activity is associated with ERK <span class="highlight" style="background-color:">pathway</span>. (A and B) After transfection, HEK293 cells were treated with tunicamycin (2 μg/mL) together with U0126 (10 μmol/L) (A) or EGF (200 ng/mL) (B) for the indicated time and insulin for 5 min (167 nmol/L). Then, the phosphorylated IR was analyzed. (C) HEK293 cells were transiently transfected with pcDNA3.1/pcDNA3.1-hATF6α. After 5 h, cells were refreshed and incubated with U0126 (10 μmol/L) or EGF (200 ng/mL) in serum-free DMEM for 4, 8, and 12 h without tunicamycin stimulation before 5-min insulin (167 nmol/L) stimulation. <sup>c</sup><i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Xuan Tang, et al. Acta Pharmacol Sin. 2011 September;32(9):1138-1147.</div><p class="links"><a class="dblinks" href="/pubmed/21841811" ref="ordinalpos=4414&amp;ncbi_uid=6627048&amp;link_uid=21841811">Citation</a><a class="dblinks" href="/pmc/articles/PMC4003301/" ref="ordinalpos=4414&amp;ncbi_uid=6627048&amp;link_uid=PMC4003301">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4415.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3423861/figure/F7/" ref="ordinalpos=4415&amp;ncbi_uid=5144740&amp;link_uid=PMC3423861" href="/pmc/articles/PMC3423861/"><img src="/pmc/articles/instance/3423861/bin/zh50151261570007.gif" src-large="/pmc/articles/instance/3423861/bin/zh50151261570007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3423861/" ref="ordinalpos=4415&amp;ncbi_uid=5144740&amp;link_uid=PMC3423861" image-link="/pmc/articles/PMC3423861/figure/F7/" class="imagepopup">Fig. 7. From: Angiotensin II regulates activation of Bim via Rb/E2F1 during apoptosis: involvement of interaction between AMPK?1/2 and Cdk4. </a></p><div class="supp"><p class="details"><div>Proposed models of Ang II-induced apoptosis <span class="highlight" style="background-color:">signaling</span> in PAEC. Schematic of the physiological AMPKβ1/2-Cdk4 complex for apoptotic <span class="highlight" style="background-color:">signaling</span> upon Ang II stimulation. Ang II binds to the G protein-coupled receptor (GPCR) AT<sub>2</sub> receptor to transduce downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for cell growth/differentiation, inflammation, and apoptosis (44). In untreated cells, AMPKβ1/2 is associated with Cdk4. Upon treatment with Ang II treatment, Cdk4 is dissociated from AMPKβ1/2, coincidentally with AMPKβ1 phosphorylation. Cdk4-mediated hyperphosphorylation of Rb renders release of E2F1 from Rb. Free E2F1 binds to the promoter region of the proapoptotic Bim protein for transcriptional activation. At the same time, Ang II treatment suppresses antiapoptotic <span class="highlight" style="background-color:">signaling</span> by the Akt <span class="highlight" style="background-color:">pathway</span> by suppressing basal Akt phosphorylation. MOMP, mitochondrial outer membrane permeabilization.</div></p></div><div class="aux"><div class="resc">Yong-Chul Kim, et al. Am J Physiol Lung Cell Mol Physiol. 2012 August 1;303(3):L228-L238.</div><p class="links"><a class="dblinks" href="/pubmed/22659879" ref="ordinalpos=4415&amp;ncbi_uid=5144740&amp;link_uid=22659879">Citation</a><a class="dblinks" href="/pmc/articles/PMC3423861/" ref="ordinalpos=4415&amp;ncbi_uid=5144740&amp;link_uid=PMC3423861">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4416.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3869759/figure/pone-0083312-g004/" ref="ordinalpos=4416&amp;ncbi_uid=5633684&amp;link_uid=PMC3869759" href="/pmc/articles/PMC3869759/"><img src="/pmc/articles/instance/3869759/bin/pone.0083312.g004.gif" src-large="/pmc/articles/instance/3869759/bin/pone.0083312.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3869759/" ref="ordinalpos=4416&amp;ncbi_uid=5633684&amp;link_uid=PMC3869759" image-link="/pmc/articles/PMC3869759/figure/pone-0083312-g004/" class="imagepopup">Figure 4. CHIP modulated PTEN, and knockdown of PTEN inhibited the AKT/FoxO3a/Bim <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: CHIP Regulates AKT/FoxO/Bim <span class="highlight" style="background-color:">Signaling</span> in MCF7 and MCF10A Cells. </a></p><div class="supp"><p class="details"><div>(A) CHIP overexpression reduced the synthesis of PTEN mRNA, and knockdown of CHIP increased PTEN mRNA level. MCF7 and MCF10A cells were transfected with Myc-CHIP, Myc empty vector, CHIP siRNA or negative control siRNA respectively. Total cellular mRNA was then extracted and subjected to reverse transcription PCR. The PTEN mRNA level was determined and calculated from three independent experiments (<i>P</i>&lt;0.05). (B) CHIP regulated PTEN protein level in both transcriptional and post-translational steps. MCF7 and MCF10A cells were transfected with or without Myc-CHIP. After 48 h, the cells were treated for western blot, ChIP assay (* <i>P</i>&lt;0.05) or <i>in vivo</i> ubiquitylation assay respectively. (C) Knockdown of PTEN promoted AKT activation and influenced the AKT/FoxO3/Bim <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MCF7 and MCF10A cells were transfected with PTEN siRNA or negative control siRNA. The altered expression patterns of p-AKT (T308), AKT, p-FoxO3, FoxO3, Bim, cleaved caspase 9, and cleaved PARP were determined by western blot.</div></p></div><div class="aux"><div class="resc">Yanrong Lv, et al. PLoS One. 2013;8(12):e83312.</div><p class="links"><a class="dblinks" href="/pubmed/24376685" ref="ordinalpos=4416&amp;ncbi_uid=5633684&amp;link_uid=24376685">Citation</a><a class="dblinks" href="/pmc/articles/PMC3869759/" ref="ordinalpos=4416&amp;ncbi_uid=5633684&amp;link_uid=PMC3869759">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4417.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3578363/figure/A008904F1/" ref="ordinalpos=4417&amp;ncbi_uid=7217032&amp;link_uid=PMC3578363" href="/pmc/articles/PMC3578363/"><img src="/pmc/articles/instance/3578363/bin/cshperspect-SIG-008904_F1.gif" src-large="/pmc/articles/instance/3578363/bin/cshperspect-SIG-008904_F1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3578363/" ref="ordinalpos=4417&amp;ncbi_uid=7217032&amp;link_uid=PMC3578363" image-link="/pmc/articles/PMC3578363/figure/A008904F1/" class="imagepopup">Figure 1. From: <span class="highlight" style="background-color:">Signaling</span> Pathways that Control Cell Proliferation. </a></p><div class="supp"><p class="details"><div>G<sub>1</sub> cell cycle control by the pRB <span class="highlight" style="background-color:">pathway</span>. Many cellular <span class="highlight" style="background-color:">signaling</span> events are intrinsically linked to G<sub>1</sub> phase of the cell cycle, which is controlled by the RB <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Signaling</span> to the RB <span class="highlight" style="background-color:">pathway</span> and thus G<sub>1</sub> control by different cellular processes is achieved mainly through the regulation of cyclins and CDK inhibitors (CKIs). In mammalian cells, mitogenic signals first induce the synthesis of D-type cyclins, leading to activation of cyclin-D-dependent CDK4 and CDK6, and then induce E-type cyclins to activate CDK2. Cyclin-D–CDK4/6 and cyclin-E–CDK2 cooperatively phosphorylate RB-family proteins, derepressing E2F to allow transcription of E2F-target genes, thereby promoting the G<sub>1</sub>/S transition. The INK4 proteins specifically inhibit CDK4 and CDK6, whereas the p21 (CIP/KIP) family of CKIs inhibits multiple CDKs. Although the schematic illustration is based on mammalian cells, the regulation of both G<sub>1</sub> cyclins and CDK inhibitors is evolutionarily conserved.</div></p></div><div class="aux"><div class="resc">Robert J. Duronio, et al. Cold Spring Harb Perspect Biol. 2013 March;5(3):a008904.</div><p class="links"><a class="dblinks" href="/pubmed/23457258" ref="ordinalpos=4417&amp;ncbi_uid=7217032&amp;link_uid=23457258">Citation</a><a class="dblinks" href="/pmc/articles/PMC3578363/" ref="ordinalpos=4417&amp;ncbi_uid=7217032&amp;link_uid=PMC3578363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4418.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3698152/figure/F1/" ref="ordinalpos=4418&amp;ncbi_uid=5057267&amp;link_uid=PMC3698152" href="/pmc/articles/PMC3698152/"><img src="/pmc/articles/instance/3698152/bin/1478-811X-11-43-1.gif" src-large="/pmc/articles/instance/3698152/bin/1478-811X-11-43-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3698152/" ref="ordinalpos=4418&amp;ncbi_uid=5057267&amp;link_uid=PMC3698152" image-link="/pmc/articles/PMC3698152/figure/F1/" class="imagepopup">Figure 1.  From: Modeling approaches for qualitative and semi-quantitative analysis of cellular <span class="highlight" style="background-color:">signaling</span> networks. </a></p><div class="supp"><p class="details"><div><b>Modeling pipeline: from qualitative information to quantitative models.</b> A black arrow in the interaction graph representation denotes a positive (activating) edge, a red blunt-ended line a negative (inhibiting) edge. In the hypergraph representation of the logical model, a red branch of a hyperedge means that the logical value of the input node is negated, a black branch (or black edge) that the input value is not negated. Illustration of the <span class="highlight" style="background-color:">pathway</span> scheme reproduced courtesy of Cell <span class="highlight" style="background-color:">Signaling</span> Technology [19].</div></p></div><div class="aux"><div class="resc">Regina Samaga, et al. Cell Commun Signal. 2013;11:43-43.</div><p class="links"><a class="dblinks" href="/pubmed/23803171" ref="ordinalpos=4418&amp;ncbi_uid=5057267&amp;link_uid=23803171">Citation</a><a class="dblinks" href="/pmc/articles/PMC3698152/" ref="ordinalpos=4418&amp;ncbi_uid=5057267&amp;link_uid=PMC3698152">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4419.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508522/figure/F3/" ref="ordinalpos=4419&amp;ncbi_uid=4847066&amp;link_uid=PMC3508522" href="/pmc/articles/PMC3508522/"><img src="/pmc/articles/instance/3508522/bin/nihms396062f3.gif" src-large="/pmc/articles/instance/3508522/bin/nihms396062f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508522/" ref="ordinalpos=4419&amp;ncbi_uid=4847066&amp;link_uid=PMC3508522" image-link="/pmc/articles/PMC3508522/figure/F3/" class="imagepopup">Figure 3. Non-invasively measuring AR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> output with a radiotracer targeting PSMA.  From: Measuring Oncogenic <span class="highlight" style="background-color:">Signaling</span> Pathways in Cancer with PET: An Emerging Paradigm from Studies in Castration Resistant Prostate Cancer. </a></p><div class="supp"><p class="details"><div>A schematic representation of the relationship between AR activity and PSMA expression, and the strategy to exploit this relationship for PET imaging. Several reports have shown that PSMA is an androgen repressed gene, and that AR inhibition elevates PSMA expression. ChIP-Seq data has shown AR to bind the PSMA gene, advancing the putative mechanism outlined in this figure.</div></p></div><div class="aux"><div class="resc">Michael J. Evans. Cancer Discov. ;2(11):985-994.</div><p class="links"><a class="dblinks" href="/pubmed/23043150" ref="ordinalpos=4419&amp;ncbi_uid=4847066&amp;link_uid=23043150">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508522/" ref="ordinalpos=4419&amp;ncbi_uid=4847066&amp;link_uid=PMC3508522">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4420.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281050/figure/pone-0031195-g007/" ref="ordinalpos=4420&amp;ncbi_uid=3816924&amp;link_uid=PMC3281050" href="/pmc/articles/PMC3281050/"><img src="/pmc/articles/instance/3281050/bin/pone.0031195.g007.gif" src-large="/pmc/articles/instance/3281050/bin/pone.0031195.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281050/" ref="ordinalpos=4420&amp;ncbi_uid=3816924&amp;link_uid=PMC3281050" image-link="/pmc/articles/PMC3281050/figure/pone-0031195-g007/" class="imagepopup">Figure 7. Effect of DHC on mTOR <span class="highlight" style="background-color:">signaling</span> downstream <span class="highlight" style="background-color:">pathway</span>..  From: Dehydrocostuslactone Suppresses Angiogenesis In Vitro and In Vivo through Inhibition of Akt/GSK-3? and mTOR <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>A, Western blot analysis of phosphorylation of mTOR, p70S6K, eIF4E and 4EBP in HUVECs treated with DHC for the indicated times and concentrations. B, after transfected with the indicated plasmids, HUVECs were starved for 24 hr and then pretreated with DHC followed by 10 min of EGM-2 incubation. Phosphorylation of mTOR and 4EBP were analyzed by western blot. Data represent from three independent experiments.</div></p></div><div class="aux"><div class="resc">Chih-Ya Wang, et al. PLoS One. 2012;7(2):e31195.</div><p class="links"><a class="dblinks" href="/pubmed/22359572" ref="ordinalpos=4420&amp;ncbi_uid=3816924&amp;link_uid=22359572">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281050/" ref="ordinalpos=4420&amp;ncbi_uid=3816924&amp;link_uid=PMC3281050">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4421.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2811493/figure/F5/" ref="ordinalpos=4421&amp;ncbi_uid=2491436&amp;link_uid=PMC2811493" href="/pmc/articles/PMC2811493/"><img src="/pmc/articles/instance/2811493/bin/nihms152036f5.gif" src-large="/pmc/articles/instance/2811493/bin/nihms152036f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2811493/" ref="ordinalpos=4421&amp;ncbi_uid=2491436&amp;link_uid=PMC2811493" image-link="/pmc/articles/PMC2811493/figure/F5/" class="imagepopup">Figure 5. Models for regulation of bone turnover by leptin and IL-17A.  From: A Bone-Protective Role for IL-17 Receptor <span class="highlight" style="background-color:">Signaling</span> in Ovariectomy-Induced Bone Loss. </a></p><div class="supp"><p class="details"><div>A. <span class="highlight" style="background-color:">Signaling</span> via IL-17A and IL-17F promote mesenchymal stem cells (MSC) to undergo osteoblastogenesis and simulataneously inhibit adipogenesis Thus, the absence of IL-17RA may limit osteoblast development and drive accelerated bone turnover during OVX. B. Leptin can limit osteogenesis through a central <span class="highlight" style="background-color:">pathway</span> from the hypothalamus. By inhibiting leptin expression, IL-17 may aid in limiting bone turnover during OVX. In IL-17RA<sup>KO</sup> mice, bone loss is thus enhanced.</div></p></div><div class="aux"><div class="resc">Jaya Goswami, et al. Eur J Immunol. ;39(10):2831-2839.</div><p class="links"><a class="dblinks" href="/pubmed/19731364" ref="ordinalpos=4421&amp;ncbi_uid=2491436&amp;link_uid=19731364">Citation</a><a class="dblinks" href="/pmc/articles/PMC2811493/" ref="ordinalpos=4421&amp;ncbi_uid=2491436&amp;link_uid=PMC2811493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4422.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3624252/figure/F9/" ref="ordinalpos=4422&amp;ncbi_uid=5348439&amp;link_uid=PMC3624252" href="/pmc/articles/PMC3624252/"><img src="/pmc/articles/instance/3624252/bin/zmb9991099310009.gif" src-large="/pmc/articles/instance/3624252/bin/zmb9991099310009.jpg" alt="Fig 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3624252/" ref="ordinalpos=4422&amp;ncbi_uid=5348439&amp;link_uid=PMC3624252" image-link="/pmc/articles/PMC3624252/figure/F9/" class="imagepopup">Fig 9.  From: Gab2 Phosphorylation by RSK Inhibits Shp2 Recruitment and Cell Motility. </a></p><div class="supp"><p class="details"><div>Schematic representation of Gab2 <span class="highlight" style="background-color:">signaling</span> and its negative regulation by RSK. While Gab2 phosphorylation at tyrosine residues promotes Shp2 recruitment and activation of the Ras/MAPK <span class="highlight" style="background-color:">pathway</span>, RSK-mediated phosphorylation of Gab2 on Ser/Thr residues inhibits Shp2 binding and Shp2-dependent <span class="highlight" style="background-color:">signaling</span>. RSK-mediated phosphorylation of Gab2 likely plays a role in tightly regulating Gab2-dependent biological functions, such as the regulation of cell motility.</div></p></div><div class="aux"><div class="resc">Xiaocui Zhang, et al. Mol Cell Biol. 2013 April;33(8):1657-1670.</div><p class="links"><a class="dblinks" href="/pubmed/23401857" ref="ordinalpos=4422&amp;ncbi_uid=5348439&amp;link_uid=23401857">Citation</a><a class="dblinks" href="/pmc/articles/PMC3624252/" ref="ordinalpos=4422&amp;ncbi_uid=5348439&amp;link_uid=PMC3624252">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4423.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3426490/figure/F2/" ref="ordinalpos=4423&amp;ncbi_uid=4191525&amp;link_uid=PMC3426490" href="/pmc/articles/PMC3426490/"><img src="/pmc/articles/instance/3426490/bin/1471-2334-12-125-2.gif" src-large="/pmc/articles/instance/3426490/bin/1471-2334-12-125-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3426490/" ref="ordinalpos=4423&amp;ncbi_uid=4191525&amp;link_uid=PMC3426490" image-link="/pmc/articles/PMC3426490/figure/F2/" class="imagepopup">Figure 2.  From: Chandipura Virus infection in mice: the role of toll like receptor 4 in pathogenesis. </a></p><div class="supp"><p class="details"><div><b> Flow chart showing the genes involved in TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> In the TLRs <span class="highlight" style="background-color:">signaling</span> path way chart, genes highlighted in red boxes represent TLR4-related genes up-regulated in RAW cells during Chandipura viral infection. The other TLRs are highlighted in green boxes.</div></p></div><div class="aux"><div class="resc">Balakrishnan Anukumar, et al. BMC Infect Dis. 2012;12:125-125.</div><p class="links"><a class="dblinks" href="/pubmed/22642811" ref="ordinalpos=4423&amp;ncbi_uid=4191525&amp;link_uid=22642811">Citation</a><a class="dblinks" href="/pmc/articles/PMC3426490/" ref="ordinalpos=4423&amp;ncbi_uid=4191525&amp;link_uid=PMC3426490">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4424.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405845/figure/F6/" ref="ordinalpos=4424&amp;ncbi_uid=4766281&amp;link_uid=PMC3405845" href="/pmc/articles/PMC3405845/"><img src="/pmc/articles/instance/3405845/bin/auto-8-577-g6.gif" src-large="/pmc/articles/instance/3405845/bin/auto-8-577-g6.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405845/" ref="ordinalpos=4424&amp;ncbi_uid=4766281&amp;link_uid=PMC3405845" image-link="/pmc/articles/PMC3405845/figure/F6/" class="imagepopup">Figure. From: Suppression of autophagy is protective in high glucose-induced cardiomyocyte injury. </a></p><div class="supp"><p class="details"><div><b>Figure 6.</b> High glucose activated the mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cardiomyocytes were treated with different doses of glucose for 72 h. The protein expression levels and phosphorylation states of the components of several <span class="highlight" style="background-color:">signaling</span> pathways were determined by western blotting analysis.</div></p></div><div class="aux"><div class="resc">Satoru Kobayashi, et al. Autophagy. 2012 April 1;8(4):577-592.</div><p class="links"><a class="dblinks" href="/pubmed/22498478" ref="ordinalpos=4424&amp;ncbi_uid=4766281&amp;link_uid=22498478">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405845/" ref="ordinalpos=4424&amp;ncbi_uid=4766281&amp;link_uid=PMC3405845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4425.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3687803/figure/F5/" ref="ordinalpos=4425&amp;ncbi_uid=5014728&amp;link_uid=PMC3687803" href="/pmc/articles/PMC3687803/"><img src="/pmc/articles/instance/3687803/bin/nihms475212f5.gif" src-large="/pmc/articles/instance/3687803/bin/nihms475212f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3687803/" ref="ordinalpos=4425&amp;ncbi_uid=5014728&amp;link_uid=PMC3687803" image-link="/pmc/articles/PMC3687803/figure/F5/" class="imagepopup">Fig. 5.  From: Vitamin D and the skin: Physiology and pathophysiology. </a></p><div class="supp"><p class="details"><div>The Sonic hedgehog (Shh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the absence of Shh, Ptch 1 suppresses <span class="highlight" style="background-color:">signaling</span> by smoothened (Smoh). Binding of Shh to Ptch 1 relieves this inhibition. Activation of Smoh leads to the activation and translocation of transcription factors of the Gli family into the nucleus, with subsequent changes in gene expression</div></p></div><div class="aux"><div class="resc">Daniel D. Bikle. Rev Endocr Metab Disord. ;13(1):3-19.</div><p class="links"><a class="dblinks" href="/pubmed/21845365" ref="ordinalpos=4425&amp;ncbi_uid=5014728&amp;link_uid=21845365">Citation</a><a class="dblinks" href="/pmc/articles/PMC3687803/" ref="ordinalpos=4425&amp;ncbi_uid=5014728&amp;link_uid=PMC3687803">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4426.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2773873/figure/F5/" ref="ordinalpos=4426&amp;ncbi_uid=2079973&amp;link_uid=PMC2773873" href="/pmc/articles/PMC2773873/"><img src="/pmc/articles/instance/2773873/bin/WJG-14-6786-g005.gif" src-large="/pmc/articles/instance/2773873/bin/WJG-14-6786-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2773873/" ref="ordinalpos=4426&amp;ncbi_uid=2079973&amp;link_uid=PMC2773873" image-link="/pmc/articles/PMC2773873/figure/F5/" class="imagepopup">Figure 5.  From: Dynamic localization of hepatocellular transporters in health and disease. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways involved in the co-stimulation of the canalicular targeting of AE2 and AQP8 by cAMP. AE2 and AQP8 are co-localized in the same population of pericanalicular vesicles, thus explaining common <span class="highlight" style="background-color:">signaling</span> modulation. cAMP stimulates AE2 and AQP8 targeting <i>via</i> activation of PKA. The PI3K <span class="highlight" style="background-color:">pathway</span> mediates the cAMP-stimulated, PKA-dependent targeting of AQP8, and probably that of AE2. cAMP effect on both transporters is counteracted by activation of PKC.</div></p></div><div class="aux"><div class="resc">Marcelo G Roma, et al. World J Gastroenterol. 2008 November 28;14(44):6786-6801.</div><p class="links"><a class="dblinks" href="/pubmed/19058304" ref="ordinalpos=4426&amp;ncbi_uid=2079973&amp;link_uid=19058304">Citation</a><a class="dblinks" href="/pmc/articles/PMC2773873/" ref="ordinalpos=4426&amp;ncbi_uid=2079973&amp;link_uid=PMC2773873">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4427.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4077886/figure/F9/" ref="ordinalpos=4427&amp;ncbi_uid=6405327&amp;link_uid=PMC4077886" href="/pmc/articles/PMC4077886/"><img src="/pmc/articles/instance/4077886/bin/auto-10-480-g9.gif" src-large="/pmc/articles/instance/4077886/bin/auto-10-480-g9.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4077886/" ref="ordinalpos=4427&amp;ncbi_uid=6405327&amp;link_uid=PMC4077886" image-link="/pmc/articles/PMC4077886/figure/F9/" class="imagepopup">Figure. From: Lysosomal-mediated waste clearance in retinal pigment epithelial cells is regulated by CRYBA1/?A3/A1-crystallin via V-ATPase-MTORC1 <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 9.</b><i>Cryba1</i> regulates the MTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>A</b>) Following induction of autophagy in vivo by starvation, RPE/choroid was isolated and levels of <span class="highlight" style="background-color:">signaling</span> intermediates determined. In the cKO samples p-AKT was decreased compared with <i>Cryba1</i><sup>fl/fl</sup>, while p-MTOR was increased. Total AKT and total MTOR were not changed. (<b>B</b>) In starved RPE cells cultured from cKO mice, overexpression of CRYBA1 increased p-AKT and p-RPTOR expression, while p-MTOR decreased. (<b>C</b>) Conversely, downregulation of βA3/A1- crystallin expression in <i>Cryba1</i><sup>fl/fl</sup> cells using siRNA, decreased p-AKT, and p-RPTOR while p-MTOR was upregulated. (<b>D</b>) Rapamycin, an inhibitor of MTOR, was effective at decreasing p-MTOR and increasing p-RPTOR levels in the cKO cells.</div></p></div><div class="aux"><div class="resc">Mallika Valapala, et al. Autophagy. 2014 March 1;10(3):480-496.</div><p class="links"><a class="dblinks" href="/pubmed/24468901" ref="ordinalpos=4427&amp;ncbi_uid=6405327&amp;link_uid=24468901">Citation</a><a class="dblinks" href="/pmc/articles/PMC4077886/" ref="ordinalpos=4427&amp;ncbi_uid=6405327&amp;link_uid=PMC4077886">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4428.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3329340/figure/F3/" ref="ordinalpos=4428&amp;ncbi_uid=3972121&amp;link_uid=PMC3329340" href="/pmc/articles/PMC3329340/"><img src="/pmc/articles/instance/3329340/bin/psb-6-1700-g3.gif" src-large="/pmc/articles/instance/3329340/bin/psb-6-1700-g3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3329340/" ref="ordinalpos=4428&amp;ncbi_uid=3972121&amp;link_uid=PMC3329340" image-link="/pmc/articles/PMC3329340/figure/F3/" class="imagepopup">Figure 3. From: Plant TOR <span class="highlight" style="background-color:">signaling</span> components. </a></p><div class="supp"><p class="details"><div>Simplified TOR <span class="highlight" style="background-color:">signaling</span> network with known Arabidopsis homologs in Italic writing. Activity of TOR is influenced by abundance of growth factors and nutrients as well as by the cellular energy level. Growth factors act through the PI3K <span class="highlight" style="background-color:">pathway</span> which finally leads to inactivation of the TOR inhibitory complex TSC1/TSC2. The cellular energy status is mostly represented by the abundance of amino acids which lead to the activation of AMPK kinase and to subsequent inactivation of TSC1/TSC2. How nutrient perception and intracellular <span class="highlight" style="background-color:">signaling</span> works in detail still remains to be elucidated. Gene identifiers represent possible Arabidopsis LST8 homologs.</div></p></div><div class="aux"><div class="resc">Florian John, et al. Plant Signal Behav. 2011 November 1;6(11):1700-1705.</div><p class="links"><a class="dblinks" href="/pubmed/22057328" ref="ordinalpos=4428&amp;ncbi_uid=3972121&amp;link_uid=22057328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3329340/" ref="ordinalpos=4428&amp;ncbi_uid=3972121&amp;link_uid=PMC3329340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4429.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2233618/figure/F4/" ref="ordinalpos=4429&amp;ncbi_uid=1246336&amp;link_uid=PMC2233618" href="/pmc/articles/PMC2233618/"><img src="/pmc/articles/instance/2233618/bin/1471-2121-8-52-4.gif" src-large="/pmc/articles/instance/2233618/bin/1471-2121-8-52-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2233618/" ref="ordinalpos=4429&amp;ncbi_uid=1246336&amp;link_uid=PMC2233618" image-link="/pmc/articles/PMC2233618/figure/F4/" class="imagepopup">Figure 4.  From: Identification of two novel activities of the Wnt <span class="highlight" style="background-color:">signaling</span> regulator Dickkopf 3 and characterization of its expression in the mouse retina. </a></p><div class="supp"><p class="details"><div><b>Dkk1 blocks Dkk3 activity</b>. Dkk3 (white bars) and control (black bars) stable cell lines (HEK293 cells) were transfected with the control vector ("no Dkk1") and TOP-FLASH and LacZ plasmids then treated with Wnt3a. The Dkk3 cell line had more Wnt <span class="highlight" style="background-color:">signaling</span> than the vector control line (Luciferase Activity units shown). Transfection of Dkk1 completely abolished Wnt <span class="highlight" style="background-color:">signaling</span> in both the Dkk3 stable cell line and the control cell line. This result suggests that Dkk3 acts in a Dkk1-dependent <span class="highlight" style="background-color:">pathway</span>. * = p &lt; 0.05. Mean ± standard deviation is shown.</div></p></div><div class="aux"><div class="resc">Rei EI Nakamura, et al. BMC Cell Biol. 2007;8:52-52.</div><p class="links"><a class="dblinks" href="/pubmed/18093317" ref="ordinalpos=4429&amp;ncbi_uid=1246336&amp;link_uid=18093317">Citation</a><a class="dblinks" href="/pmc/articles/PMC2233618/" ref="ordinalpos=4429&amp;ncbi_uid=1246336&amp;link_uid=PMC2233618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4430.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3292258/figure/F2/" ref="ordinalpos=4430&amp;ncbi_uid=4179283&amp;link_uid=PMC3292258" href="/pmc/articles/PMC3292258/"><img src="/pmc/articles/instance/3292258/bin/nihms-343991-f0002.gif" src-large="/pmc/articles/instance/3292258/bin/nihms-343991-f0002.jpg" alt="Figure. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3292258/" ref="ordinalpos=4430&amp;ncbi_uid=4179283&amp;link_uid=PMC3292258" image-link="/pmc/articles/PMC3292258/figure/F2/" class="imagepopup">Figure. 2. Arabidopsis GSK3s act downstream of ERf and TMM but upstream of YDA in the stomatal development <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Brassinosteroid regulates stomatal development by GSK3-mediated inhibition of a MAPK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>a–d</b>, CA-YDA expression eliminates stomata in <i>bri1-116, bsu-q</i> and <i>bin2-1</i>. <b>e–f</b>, Loss of <i>SPCH</i> eliminates stomata in <i>bsu-q</i> mutants. <b>g–k</b>, Representative stomatal phenotypes of leaf epidermis of <i>er;erl1;erl2</i> (<b>g</b>), <i>tmm</i> (<b>h</b>), <i>yda</i> (<b>i</b>), <i>HOPAI1</i> (<b>j</b>), and <i>scrm-D</i> (<b>k</b>) plants grown in the absence (−bk) or presence (+bk) of 30 μM bikinin. Scale bars = 50 μm.</div></p></div><div class="aux"><div class="resc">Tae-Wuk Kim, et al. Nature. ;482(7385):419-422.</div><p class="links"><a class="dblinks" href="/pubmed/22307275" ref="ordinalpos=4430&amp;ncbi_uid=4179283&amp;link_uid=22307275">Citation</a><a class="dblinks" href="/pmc/articles/PMC3292258/" ref="ordinalpos=4430&amp;ncbi_uid=4179283&amp;link_uid=PMC3292258">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4431.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2048934/figure/pone-0001155-g002/" ref="ordinalpos=4431&amp;ncbi_uid=1172761&amp;link_uid=PMC2048934" href="/pmc/articles/PMC2048934/"><img src="/pmc/articles/instance/2048934/bin/pone.0001155.g002.gif" src-large="/pmc/articles/instance/2048934/bin/pone.0001155.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2048934/" ref="ordinalpos=4431&amp;ncbi_uid=1172761&amp;link_uid=PMC2048934" image-link="/pmc/articles/PMC2048934/figure/pone-0001155-g002/" class="imagepopup">Figure 2. Wnt <span class="highlight" style="background-color:">signaling</span> is active in human pancreatic adenocarcinoma cell lines..  From: Common Activation of Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> in Pancreatic Adenocarcinoma. </a></p><div class="supp"><p class="details"><div>A. RT-PCR analysis for components of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 26 human pancreatic adenocarcinoma cell lines: 1-MiaPaca2; 2-Panc1; 3-CFPAC1; 4-HPAFII; 5-Capan-2; 6-AsPC1; 7-Hs766T; 8-BxPC3; 9-COLO357; 10-L3.3; 11-L3.6sl; 12-L3.6pl; 13-SW1990; 14-SU86.86; 15-PL45; 16-HPAC; 17-MPanc96; 18-Panc1.28; 19-Panc2.03; 20-Panc2.13; 21-Panc3.27; 22-Panc4.21; 23-Panc5.04; 24-Panc6.03; 25-Panc8.13; 26-Panc10.05. B. Wnt <span class="highlight" style="background-color:">signaling</span> is active in all nine pancreatic adenocarcinoma cell lines tested, as indicated by the activation of the Top-Flash reporter (black) compared to the basal activity of the Fop-Flash reporter (white). Renilla luciferase was used to normalize for transfection efficiency. Error bars are shown as St. Dev. P-values were calculated in comparison to control Fop-Flash activity (white bars). *, p&lt;0.05; **, p&lt;0.01.</div></p></div><div class="aux"><div class="resc">Marina Pasca di Magliano, et al. PLoS ONE. 2007;2(11):e1155.</div><p class="links"><a class="dblinks" href="/pubmed/17982507" ref="ordinalpos=4431&amp;ncbi_uid=1172761&amp;link_uid=17982507">Citation</a><a class="dblinks" href="/pmc/articles/PMC2048934/" ref="ordinalpos=4431&amp;ncbi_uid=1172761&amp;link_uid=PMC2048934">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4432.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3164150/figure/F1/" ref="ordinalpos=4432&amp;ncbi_uid=3215730&amp;link_uid=PMC3164150" href="/pmc/articles/PMC3164150/"><img src="/pmc/articles/instance/3164150/bin/ijbsv07p1003g02.gif" src-large="/pmc/articles/instance/3164150/bin/ijbsv07p1003g02.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3164150/" ref="ordinalpos=4432&amp;ncbi_uid=3215730&amp;link_uid=PMC3164150" image-link="/pmc/articles/PMC3164150/figure/F1/" class="imagepopup">Figure 1.  From: The Link between the Metabolic Syndrome and Cancer. </a></p><div class="supp"><p class="details"><div>The insulin receptor (IR) with its two subtypes IR-A and IR-B, the insulin growth factor 1 receptor (IGF-IR) and the hybrid receptors (IGF-1R/IR-A and IGF-1R/IR-B). <i>Structurally,</i> IR and the IGF-1R have two extracellular α-subunits and two transmembrane β-subunits that are jointed to each other by disulfide bonds. <i>Affinity,</i> insulin binds with high affinity to IR-A or IR-B but has low affinity for IGF-1R, while insulin has no binding to the hybrid receptors. IGF-1 binds to the IGF-1R and to the hybrid receptor IGF-1R/IR-A or IGF-1R/IR-B. IGF-2 binds to IR-A, IGF-1R or to IGF-1R/IR-A hybrid receptor. <i><span class="highlight" style="background-color:">Signaling</span>, </i>ligand binding to insulin receptor-A or to IGF-1 receptor mediates the mitogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, while ligand binding to insulin receptor-B activates metabolic <span class="highlight" style="background-color:">signaling</span>. Binding to the hybrid receptors, leading to mitogenic or metabolic <span class="highlight" style="background-color:">signaling</span>, is determined by the IR isoform that formed the hybrid receptors. Reproduced by permission of the RMMJ 54.</div></p></div><div class="aux"><div class="resc">Sandra Braun, et al. Int J Biol Sci. 2011;7(7):1003-1015.</div><p class="links"><a class="dblinks" href="/pubmed/21912508" ref="ordinalpos=4432&amp;ncbi_uid=3215730&amp;link_uid=21912508">Citation</a><a class="dblinks" href="/pmc/articles/PMC3164150/" ref="ordinalpos=4432&amp;ncbi_uid=3215730&amp;link_uid=PMC3164150">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4433.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3488210/figure/gks699-F5/" ref="ordinalpos=4433&amp;ncbi_uid=4349958&amp;link_uid=PMC3488210" href="/pmc/articles/PMC3488210/"><img src="/pmc/articles/instance/3488210/bin/gks699f5.gif" src-large="/pmc/articles/instance/3488210/bin/gks699f5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3488210/" ref="ordinalpos=4433&amp;ncbi_uid=4349958&amp;link_uid=PMC3488210" image-link="/pmc/articles/PMC3488210/figure/gks699-F5/" class="imagepopup">Figure 5. From: Discovering the hidden sub-network component in a ranked list of genes or proteins derived from genomic experiments. </a></p><div class="supp"><p class="details"><div>Significant sub-networks (see also Table 1) found among the lists of most essential genes, ranked by the RIGER parameter, obtained for both cell lines: (<b>A</b>) Jurkat and (<b>B</b>) K562. Genes are colored according to their main functions (KEGG terms): pale blue: connecting protein; blue: MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; green: ribosome; yellow: proteasome; pink: cell cycle; magenta: apoptosis; orange: spliceosome and pale green: RIG-I-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Luz García-Alonso, et al. Nucleic Acids Res. 2012 November;40(20):e158-e158.</div><p class="links"><a class="dblinks" href="/pubmed/22844098" ref="ordinalpos=4433&amp;ncbi_uid=4349958&amp;link_uid=22844098">Citation</a><a class="dblinks" href="/pmc/articles/PMC3488210/" ref="ordinalpos=4433&amp;ncbi_uid=4349958&amp;link_uid=PMC3488210">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4434.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3462743/figure/pone-0046216-g006/" ref="ordinalpos=4434&amp;ncbi_uid=4286128&amp;link_uid=PMC3462743" href="/pmc/articles/PMC3462743/"><img src="/pmc/articles/instance/3462743/bin/pone.0046216.g006.gif" src-large="/pmc/articles/instance/3462743/bin/pone.0046216.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3462743/" ref="ordinalpos=4434&amp;ncbi_uid=4286128&amp;link_uid=PMC3462743" image-link="/pmc/articles/PMC3462743/figure/pone-0046216-g006/" class="imagepopup">Figure 6. The MAPK and the alternative cAMP/Epac <span class="highlight" style="background-color:">signaling</span> pathways participate in leptin stimulation by hCG in placenta..  From: The Alternative Epac/cAMP <span class="highlight" style="background-color:">Pathway</span> and the MAPK <span class="highlight" style="background-color:">Pathway</span> Mediate hCG Induction of Leptin in Placental Cells. </a></p><div class="supp"><p class="details"><div>Proposed model of the <span class="highlight" style="background-color:">signaling</span> pathways involved in hCG stimulation based on current data and its relation to leptin expression. Pointed arrow: Stimulation; Flat arrow: Inhibition. Dash arrow: possible pathways involved.</div></p></div><div class="aux"><div class="resc">Julieta Lorena Maymó, et al. PLoS One. 2012;7(10):e46216.</div><p class="links"><a class="dblinks" href="/pubmed/23056265" ref="ordinalpos=4434&amp;ncbi_uid=4286128&amp;link_uid=23056265">Citation</a><a class="dblinks" href="/pmc/articles/PMC3462743/" ref="ordinalpos=4434&amp;ncbi_uid=4286128&amp;link_uid=PMC3462743">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4435.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3006290/figure/F1/" ref="ordinalpos=4435&amp;ncbi_uid=3592110&amp;link_uid=PMC3006290" href="/pmc/articles/PMC3006290/"><img src="/pmc/articles/instance/3006290/bin/zh40121096060001.gif" src-large="/pmc/articles/instance/3006290/bin/zh40121096060001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3006290/" ref="ordinalpos=4435&amp;ncbi_uid=3592110&amp;link_uid=PMC3006290" image-link="/pmc/articles/PMC3006290/figure/F1/" class="imagepopup">Fig. 1. From: Cardioprotective PKG-independent NO <span class="highlight" style="background-color:">signaling</span> at reperfusion. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> scheme for ischemic preconditioning. Note that in the schematic, nitric oxide synthase (NOS) signals through protein kinase G (PKG) upstream in the trigger <span class="highlight" style="background-color:">pathway</span> but acts in a PKG-independent manner downstream in the mediator <span class="highlight" style="background-color:">pathway</span> at reperfusion. The role of glycogen synthase kinase-3β (GSK-3β) is controversial, but if it is in the <span class="highlight" style="background-color:">pathway</span>, then it must be downstream of the newly described NOS site. AR, adenosine receptor; PI3-K, phosphatidylinositol 3-kinase; PLC, phospholipase C; PLD, phospholipase D; ERK, extracellular signal-regulated kinase; GC, guanylyl cyclase; PKC, protein kinase C; mK<sub>ATP</sub>, mitochondrial ATP-dependent K<sup>+</sup> channel; mPTP, mitochondrial permeability transition pore; RISK, reperfusion injury survival kinases.</div></p></div><div class="aux"><div class="resc">Michael V. Cohen, et al. Am J Physiol Heart Circ Physiol. 2010 December;299(6):H2028-H2036.</div><p class="links"><a class="dblinks" href="/pubmed/20852051" ref="ordinalpos=4435&amp;ncbi_uid=3592110&amp;link_uid=20852051">Citation</a><a class="dblinks" href="/pmc/articles/PMC3006290/" ref="ordinalpos=4435&amp;ncbi_uid=3592110&amp;link_uid=PMC3006290">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4436.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4060523/figure/F2/" ref="ordinalpos=4436&amp;ncbi_uid=6256602&amp;link_uid=PMC4060523" href="/pmc/articles/PMC4060523/"><img src="/pmc/articles/instance/4060523/bin/nihms588161f2.gif" src-large="/pmc/articles/instance/4060523/bin/nihms588161f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4060523/" ref="ordinalpos=4436&amp;ncbi_uid=6256602&amp;link_uid=PMC4060523" image-link="/pmc/articles/PMC4060523/figure/F2/" class="imagepopup">Figure 2.  From: Ecdysone Control of Developmental Transitions: Lessons from Drosophila Research. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways that positively regulate ecdysteroidogenesis in the PG in <i>Drosophila melanogaster</i>. Only the key components of each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are depicted. Abbreviations: PG, prothoracic gland; PTTH, prothoracicotropic hormone; MAPK, mitogen-activated protein kinase; NO, nitric oxide; InR, insulin receptor; TOR, target of rapamycin.</div></p></div><div class="aux"><div class="resc">Naoki Yamanaka, et al. Annu Rev Entomol. ;58:497-516.</div><p class="links"><a class="dblinks" href="/pubmed/23072462" ref="ordinalpos=4436&amp;ncbi_uid=6256602&amp;link_uid=23072462">Citation</a><a class="dblinks" href="/pmc/articles/PMC4060523/" ref="ordinalpos=4436&amp;ncbi_uid=6256602&amp;link_uid=PMC4060523">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4437.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4385831/figure/F6/" ref="ordinalpos=4437&amp;ncbi_uid=7416133&amp;link_uid=PMC4385831" href="/pmc/articles/PMC4385831/"><img src="/pmc/articles/instance/4385831/bin/oncotarget-06-1995-g006.gif" src-large="/pmc/articles/instance/4385831/bin/oncotarget-06-1995-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4385831/" ref="ordinalpos=4437&amp;ncbi_uid=7416133&amp;link_uid=PMC4385831" image-link="/pmc/articles/PMC4385831/figure/F6/" class="imagepopup">Figure 6. Proposed model of the mechanisms of NA-induced cell death.  From: Neoalbaconol induces cell death through necroptosis by regulating RIPK-dependent autocrine TNFα and ROS production. </a></p><div class="supp"><p class="details"><div>Production of TNFα by the RIPK/NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and ROS from mitochondria are required for necroptosis induced by NA.</div></p></div><div class="aux"><div class="resc">Xinfang Yu, et al. Oncotarget. 2015 February;6(4):1995-2008.</div><p class="links"><a class="dblinks" href="/pubmed/25575821" ref="ordinalpos=4437&amp;ncbi_uid=7416133&amp;link_uid=25575821">Citation</a><a class="dblinks" href="/pmc/articles/PMC4385831/" ref="ordinalpos=4437&amp;ncbi_uid=7416133&amp;link_uid=PMC4385831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4438.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4135228/figure/F2/" ref="ordinalpos=4438&amp;ncbi_uid=6510169&amp;link_uid=PMC4135228" href="/pmc/articles/PMC4135228/"><img src="/pmc/articles/instance/4135228/bin/ijmsv11p1065g002.gif" src-large="/pmc/articles/instance/4135228/bin/ijmsv11p1065g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4135228/" ref="ordinalpos=4438&amp;ncbi_uid=6510169&amp;link_uid=PMC4135228" image-link="/pmc/articles/PMC4135228/figure/F2/" class="imagepopup">Figure 2.  From: Bone Morphogenetic Protein-9 Induces PDLSCs Osteogenic Differentiation through the ERK and p38 Signal Pathways. </a></p><div class="supp"><p class="details"><div>Western blot assay showed that BMP-9 activated the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and increased the phosphorylation of p38 and ERK1/2.</div></p></div><div class="aux"><div class="resc">Guo Ye, et al. Int J Med Sci. 2014;11(10):1065-1072.</div><p class="links"><a class="dblinks" href="/pubmed/25136261" ref="ordinalpos=4438&amp;ncbi_uid=6510169&amp;link_uid=25136261">Citation</a><a class="dblinks" href="/pmc/articles/PMC4135228/" ref="ordinalpos=4438&amp;ncbi_uid=6510169&amp;link_uid=PMC4135228">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4439.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3625980/figure/F7/" ref="ordinalpos=4439&amp;ncbi_uid=5376529&amp;link_uid=PMC3625980" href="/pmc/articles/PMC3625980/"><img src="/pmc/articles/instance/3625980/bin/nihms451081f7.gif" src-large="/pmc/articles/instance/3625980/bin/nihms451081f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3625980/" ref="ordinalpos=4439&amp;ncbi_uid=5376529&amp;link_uid=PMC3625980" image-link="/pmc/articles/PMC3625980/figure/F7/" class="imagepopup">Figure 7. Schematic describing the feedback loops between TLR4, C/EBPδ and FBXW7α that control LPS <span class="highlight" style="background-color:">signaling</span>.  From: FBXW7? attenuates inflammatory signalling by downregulating C/EBP? and its target gene Tlr4. </a></p><div class="supp"><p class="details"><div>The shaded boxes indicate the elements of this <span class="highlight" style="background-color:">pathway</span> that were identified in this study.</div></p></div><div class="aux"><div class="resc">Kuppusamy Balamurugan, et al. Nat Commun. 2013 April 9;4:1662-1662.</div><p class="links"><a class="dblinks" href="/pubmed/23575666" ref="ordinalpos=4439&amp;ncbi_uid=5376529&amp;link_uid=23575666">Citation</a><a class="dblinks" href="/pmc/articles/PMC3625980/" ref="ordinalpos=4439&amp;ncbi_uid=5376529&amp;link_uid=PMC3625980">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4440.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2661541/figure/f1-gcr2_4ap0010/" ref="ordinalpos=4440&amp;ncbi_uid=1747464&amp;link_uid=PMC2661541" href="/pmc/articles/PMC2661541/"><img src="/pmc/articles/instance/2661541/bin/gcr2_4ap0010f1.gif" src-large="/pmc/articles/instance/2661541/bin/gcr2_4ap0010f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2661541/" ref="ordinalpos=4440&amp;ncbi_uid=1747464&amp;link_uid=PMC2661541" image-link="/pmc/articles/PMC2661541/figure/f1-gcr2_4ap0010/" class="imagepopup">Figure 1. From: Emerging Molecular Biology of Pancreatic Cancer. </a></p><div class="supp"><p class="details"><div>The transforming growth factor-beta (TGF-β) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Abbreviations: SMAD4 = homologue of Mothers Against Decapentaplegic, Drosophila 4, (MADH4); SARA = SMAD anchor for receptor activation.</div></p></div><div class="aux"><div class="resc">Ralph H. Hruban, et al. Gastrointest Cancer Res. 2008 Jul-Aug;2(4 Suppl 2):S10-S15.</div><p class="links"><a class="dblinks" href="/pubmed/19343140" ref="ordinalpos=4440&amp;ncbi_uid=1747464&amp;link_uid=19343140">Citation</a><a class="dblinks" href="/pmc/articles/PMC2661541/" ref="ordinalpos=4440&amp;ncbi_uid=1747464&amp;link_uid=PMC2661541">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4441.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1305894/figure/i1536-7509-4-4-323-f04/" ref="ordinalpos=4441&amp;ncbi_uid=681993&amp;link_uid=PMC1305894" href="/pmc/articles/PMC1305894/"><img src="/pmc/articles/instance/1305894/bin/i1536-7509-4-4-323-f04.gif" src-large="/pmc/articles/instance/1305894/bin/i1536-7509-4-4-323-f04.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1305894/" ref="ordinalpos=4441&amp;ncbi_uid=681993&amp;link_uid=PMC1305894" image-link="/pmc/articles/PMC1305894/figure/i1536-7509-4-4-323-f04/" class="imagepopup">Figure 4. Complete <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as determined by the integration of each individual genomic data set. Solid bars indicate protein–protein interactions; arrows indicate regulatory interactions..  From: Teaching Systems Biology: An Active-learning Approach. </a></p><div class="aux"><div class="resc">Anuj Kumar. Cell Biol Educ. 2005 Winter;4(4):323-329.</div><p class="links"><a class="dblinks" href="/pubmed/16341259" ref="ordinalpos=4441&amp;ncbi_uid=681993&amp;link_uid=16341259">Citation</a><a class="dblinks" href="/pmc/articles/PMC1305894/" ref="ordinalpos=4441&amp;ncbi_uid=681993&amp;link_uid=PMC1305894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4442.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3233205/figure/F1/" ref="ordinalpos=4442&amp;ncbi_uid=3705930&amp;link_uid=PMC3233205" href="/pmc/articles/PMC3233205/"><img src="/pmc/articles/instance/3233205/bin/nihms323230f1.gif" src-large="/pmc/articles/instance/3233205/bin/nihms323230f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3233205/" ref="ordinalpos=4442&amp;ncbi_uid=3705930&amp;link_uid=PMC3233205" image-link="/pmc/articles/PMC3233205/figure/F1/" class="imagepopup">Fig. 1.  From: BAYESIAN HIERARCHICAL MODELING FOR <span class="highlight" style="background-color:">SIGNALING</span> <span class="highlight" style="background-color:">PATHWAY</span> INFERENCE FROM SINGLE CELL INTERVENTIONAL DATA. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> adapted from Sachs et al. (2005) by including three missed edges and correcting one reversed edge. Nodes represent proteins, and directed edges represent signal transduction.</div></p></div><div class="aux"><div class="resc">Ruiyan Luo, et al. Ann Appl Stat. ;5(2A):725-745.</div><p class="links"><a class="dblinks" href="/pubmed/22162986" ref="ordinalpos=4442&amp;ncbi_uid=3705930&amp;link_uid=22162986">Citation</a><a class="dblinks" href="/pmc/articles/PMC3233205/" ref="ordinalpos=4442&amp;ncbi_uid=3705930&amp;link_uid=PMC3233205">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4443.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1893022/figure/fig3/" ref="ordinalpos=4443&amp;ncbi_uid=999142&amp;link_uid=PMC1893022" href="/pmc/articles/PMC1893022/"><img src="/pmc/articles/instance/1893022/bin/GEN1761441f3.gif" src-large="/pmc/articles/instance/1893022/bin/GEN1761441f3.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  3.—" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1893022/" ref="ordinalpos=4443&amp;ncbi_uid=999142&amp;link_uid=PMC1893022" image-link="/pmc/articles/PMC1893022/figure/fig3/" class="imagepopup">F igure  3.—.  From: Adaptive Divergence in Experimental Populations of Pseudomonas fluorescens. III. Mutational Origins of Wrinkly Spreader Diversity. </a></p><div class="supp"><p class="details"><div>A model to account for the effect of <i>wspF</i> mutations on the <span class="highlight" style="background-color:">signaling</span> status of the Wsp <span class="highlight" style="background-color:">pathway</span>. The Wsp <span class="highlight" style="background-color:">signaling</span> machinery (WspA, WspB, WspD, WspE) is shown as a box with the letters C and F indicating the proteins WspC (methyltransferase) and WspF (methylesterase), respectively. The links indicate activations (arrows) or repressions (bar ends). In the absence of a stimulatory ligand, the <span class="highlight" style="background-color:">signaling</span> machinery oscillates between an active (solid) and inactive (open) state, depending on the combined (and opposing) effects of WspC and WspF, which cause the cell to achieve a particular balance between phosphorylated and nonphosphorylated WspR. (Left) The <span class="highlight" style="background-color:">signaling</span> state of the wild-type Wsp <span class="highlight" style="background-color:">pathway</span> that spends time in both active (upward diagonal lines) and inactive (downward diagonal lines) states. Mutations in <i>wspF</i> that reduce protein function, even just slightly, destroy the capacity of the <span class="highlight" style="background-color:">pathway</span> to fluctuate between activity states, producing instead a steady-state output (a constant level of phosphorylated WspR). The precise strength of the output signal depends on the degree to which WspF function is reduced and is maximal when WspF function is completely eliminated (right). Different mutations are predicted to tune the <span class="highlight" style="background-color:">pathway</span> to different output levels (middle panels).</div></p></div><div class="aux"><div class="resc">Eleni Bantinaki, et al. Genetics. 2007 May;176(1):441-453.</div><p class="links"><a class="dblinks" href="/pubmed/17339222" ref="ordinalpos=4443&amp;ncbi_uid=999142&amp;link_uid=17339222">Citation</a><a class="dblinks" href="/pmc/articles/PMC1893022/" ref="ordinalpos=4443&amp;ncbi_uid=999142&amp;link_uid=PMC1893022">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4444.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2878296/figure/F2/" ref="ordinalpos=4444&amp;ncbi_uid=2253842&amp;link_uid=PMC2878296" href="/pmc/articles/PMC2878296/"><img src="/pmc/articles/instance/2878296/bin/1747-1028-5-13-2.gif" src-large="/pmc/articles/instance/2878296/bin/1747-1028-5-13-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2878296/" ref="ordinalpos=4444&amp;ncbi_uid=2253842&amp;link_uid=PMC2878296" image-link="/pmc/articles/PMC2878296/figure/F2/" class="imagepopup">Figure 2.  From: Merotelic attachments and non-homologous end joining are the basis of chromosomal instability. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> by spindle attachment defects</b>. Two independent pathways act to delay anaphase. The spindle attachment <span class="highlight" style="background-color:">pathway</span> senses kinetochore tension and is especially efficient for detecting monotelic and syntelic attachments, and the DNA damage <span class="highlight" style="background-color:">pathway</span> acts as an additional mechanism that responds to DSB generated by merotelic attachments. When kinetochore attachment defects are undetected, for example in tumors with a CIN phenotype (grey), merotelic attachments and DSB increase, leading to activation of the DNA damage <span class="highlight" style="background-color:">pathway</span> and ultimately to mitotic slippage.</div></p></div><div class="aux"><div class="resc">Astrid Alonso Guerrero, et al. Cell Div. 2010;5:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/20478024" ref="ordinalpos=4444&amp;ncbi_uid=2253842&amp;link_uid=20478024">Citation</a><a class="dblinks" href="/pmc/articles/PMC2878296/" ref="ordinalpos=4444&amp;ncbi_uid=2253842&amp;link_uid=PMC2878296">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4445.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2553141/figure/fig1/" ref="ordinalpos=4445&amp;ncbi_uid=1509448&amp;link_uid=PMC2553141" href="/pmc/articles/PMC2553141/"><img src="/pmc/articles/instance/2553141/bin/BIO.135483.lw.f1.gif" src-large="/pmc/articles/instance/2553141/bin/BIO.135483.lw.f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2553141/" ref="ordinalpos=4445&amp;ncbi_uid=1509448&amp;link_uid=PMC2553141" image-link="/pmc/articles/PMC2553141/figure/fig1/" class="imagepopup">FIGURE 1.  From: Monte Carlo Simulation of Cell Death <span class="highlight" style="background-color:">Signaling</span> Predicts Large Cell-to-Cell Stochastic Fluctuations through the Type 2 <span class="highlight" style="background-color:">Pathway</span> of Apoptosis. </a></p><div class="supp"><p class="details"><div>Schematics of the apoptotic cell death <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Subhadip Raychaudhuri, et al. Biophys J. 2008 October 15;95(8):3559-3562.</div><p class="links"><a class="dblinks" href="/pubmed/18641073" ref="ordinalpos=4445&amp;ncbi_uid=1509448&amp;link_uid=18641073">Citation</a><a class="dblinks" href="/pmc/articles/PMC2553141/" ref="ordinalpos=4445&amp;ncbi_uid=1509448&amp;link_uid=PMC2553141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4446.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4136933/figure/f3-medscimonit-20-1345/" ref="ordinalpos=4446&amp;ncbi_uid=6513890&amp;link_uid=PMC4136933" href="/pmc/articles/PMC4136933/"><img src="/pmc/articles/instance/4136933/bin/medscimonit-20-1345-g003.gif" src-large="/pmc/articles/instance/4136933/bin/medscimonit-20-1345-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4136933/" ref="ordinalpos=4446&amp;ncbi_uid=6513890&amp;link_uid=PMC4136933" image-link="/pmc/articles/PMC4136933/figure/f3-medscimonit-20-1345/" class="imagepopup">Figure 3.  From: Apelin-13 Decreases Lipid Storage in Hypertrophic Adipocytes In Vitro Through the Upregulation of AQP7 Expression by the PI3K <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Apelin-13 upregulated AQP7 protein expression by the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hypertrophic 3T3-L1 cells. Hypertrophic 3T3-L1 cells were treated with apelin-13 (100 nM) for 24 h and pretreated with a PI3K inhibitor, LY294002 (10 μM), for 30 min. (<b>A</b>) AQP7 protein levels were determined by Western blot. β-actin was used as an internal control. (<b>B</b>) Cytoplasmic TG content. Results are expressed as means ±SEM from 3 or 4 independent experiments, each performed in triplicate. * <i>P</i>&lt;0.05 <i>vs.</i> the control group (both apelin-13 + LY294002 non-treated); <sup>#</sup> <i>P</i>&lt;0.05 apelin-13 + LY294002 <i>vs.</i> apelin-13.</div></p></div><div class="aux"><div class="resc">Mei Guo, et al. Med Sci Monit. 2014;20:1345-1352.</div><p class="links"><a class="dblinks" href="/pubmed/25080850" ref="ordinalpos=4446&amp;ncbi_uid=6513890&amp;link_uid=25080850">Citation</a><a class="dblinks" href="/pmc/articles/PMC4136933/" ref="ordinalpos=4446&amp;ncbi_uid=6513890&amp;link_uid=PMC4136933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4447.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3243808/figure/F4/" ref="ordinalpos=4447&amp;ncbi_uid=4412146&amp;link_uid=PMC3243808" href="/pmc/articles/PMC3243808/"><img src="/pmc/articles/instance/3243808/bin/nihms330358f4.gif" src-large="/pmc/articles/instance/3243808/bin/nihms330358f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3243808/" ref="ordinalpos=4447&amp;ncbi_uid=4412146&amp;link_uid=PMC3243808" image-link="/pmc/articles/PMC3243808/figure/F4/" class="imagepopup">Figure 4. IL-6 mediates EMT changes via the JAK/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: IL-6 promotes head and neck tumor metastasis by inducing epithelial-mesenchymal transition via the JAK-STAT3-SNAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Akt, STAT3, or ERK1/2 proteins were knocked down in CAL-IL6 and IOE-IL-6 cells by respective siRNAs. <b>A-B:</b> Expression level of different proteins (Akt, STAT3, ERK1/2, E-cadherin, vimentin, and snail) was examined in CAL-IL-6 (A) or IOE-IL-6 (B) cells by Western blotting. Equal protein loading was verified by stripping the blots and reprobing with tubulin antibody. Band density of each protein was normalized with tubulin and expressed as % increase or decrease ± SE as compared to controls. <b>C-D:</b> Expression levels of E-cadherin (red) and vimentin (green) was examined in CAL-IL-6 cells (C) or IOE-IL-6 cells (D) by fluorescent microscope (600×).</div></p></div><div class="aux"><div class="resc">Arti Yadav, et al. Mol Cancer Res. ;9(12):1658-1667.</div><p class="links"><a class="dblinks" href="/pubmed/21976712" ref="ordinalpos=4447&amp;ncbi_uid=4412146&amp;link_uid=21976712">Citation</a><a class="dblinks" href="/pmc/articles/PMC3243808/" ref="ordinalpos=4447&amp;ncbi_uid=4412146&amp;link_uid=PMC3243808">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4448.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3950334/figure/fig3/" ref="ordinalpos=4448&amp;ncbi_uid=6499984&amp;link_uid=PMC3950334" href="/pmc/articles/PMC3950334/"><img src="/pmc/articles/instance/4120106/bin/c3mb70496c-f3.gif" src-large="/pmc/articles/instance/4120106/bin/c3mb70496c-f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3950334/" ref="ordinalpos=4448&amp;ncbi_uid=6499984&amp;link_uid=PMC3950334" image-link="/pmc/articles/PMC3950334/figure/fig3/" class="imagepopup">Fig. 3. An interface for browsing data from the PAHKB database. (A) Browsing PH-related genes by chromosome location, disease type, and genic region (protein-coding or non-coding region). (B) An example of browsing the data by <span class="highlight" style="background-color:">pathway</span>: KEGG TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mapped with PAH-related genes (color-marked) in the PAHKB database..  From: An evidence-based knowledgebase of pulmonary arterial hypertension to identify genes and pathways relevant to pathogenesis †Electronic supplementary information (ESI) available. See DOI: 10.1039/c3mb70496c Click here for additional data file. . </a></p><div class="aux"><div class="resc">Min Zhao, et al. Mol Biosyst. 2014 April 4;10(4):732-740.</div><p class="links"><a class="dblinks" href="/pubmed/24448676" ref="ordinalpos=4448&amp;ncbi_uid=6499984&amp;link_uid=24448676">Citation</a><a class="dblinks" href="/pmc/articles/PMC3950334/" ref="ordinalpos=4448&amp;ncbi_uid=6499984&amp;link_uid=PMC3950334">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4449.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1847322/figure/F5/" ref="ordinalpos=4449&amp;ncbi_uid=892780&amp;link_uid=PMC1847322" href="/pmc/articles/PMC1847322/"><img src="/pmc/articles/instance/1847322/bin/nihms17490f5.gif" src-large="/pmc/articles/instance/1847322/bin/nihms17490f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1847322/" ref="ordinalpos=4449&amp;ncbi_uid=892780&amp;link_uid=PMC1847322" image-link="/pmc/articles/PMC1847322/figure/F5/" class="imagepopup">Fig. 5.  From: Selective degradation of transcripts during meiotic maturation of mouse oocytes. </a></p><div class="supp"><p class="details"><div>Canonical pathways associated with the stable transcripts during oocyte maturation. (A): Pathways identified by IPA that are associated with the stable transcripts. The number of stable transcripts involved in each <span class="highlight" style="background-color:">pathway</span> is shown above the bars. There were 30 canonical pathways in total being identified to be associated with the degraded transcripts, and only the top 15 is shown here. (B): GenMAPP display of transcripts participating in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">You-Qiang Su, et al. Dev Biol. ;302(1):104-117.</div><p class="links"><a class="dblinks" href="/pubmed/17022963" ref="ordinalpos=4449&amp;ncbi_uid=892780&amp;link_uid=17022963">Citation</a><a class="dblinks" href="/pmc/articles/PMC1847322/" ref="ordinalpos=4449&amp;ncbi_uid=892780&amp;link_uid=PMC1847322">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4450.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267738/figure/pone-0030814-g006/" ref="ordinalpos=4450&amp;ncbi_uid=3785156&amp;link_uid=PMC3267738" href="/pmc/articles/PMC3267738/"><img src="/pmc/articles/instance/3267738/bin/pone.0030814.g006.gif" src-large="/pmc/articles/instance/3267738/bin/pone.0030814.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267738/" ref="ordinalpos=4450&amp;ncbi_uid=3785156&amp;link_uid=PMC3267738" image-link="/pmc/articles/PMC3267738/figure/pone-0030814-g006/" class="imagepopup">Figure 6. A novel region of LRP5 (1516MFYSSNIPATVRPYRPY1531) is required for Wnt <span class="highlight" style="background-color:">signaling</span>..  From: Activation of Wnt <span class="highlight" style="background-color:">Signaling</span> by Chemically Induced Dimerization of LRP5 Disrupts Cellular Homeostasis. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Cross-species analysis of the novel LRP5 (<sub>1516</sub>MFYSSNIPATVRPYRPY<sub>1531</sub>) sequence. (<b>B</b>) Membrane targeting of the cytoplasmic moiety of LRP5 (M<sub>F</sub>-LRP5) and three variants containing mutations of possible phosphorylation sites: (1) YSS→FAA, (2) YRPY→FLPF and (3) YSS→FAA and YRPY→FLPF. The mutants have reduced Wnt <span class="highlight" style="background-color:">pathway</span> inducibility. Western blot indicates comparable protein expression from respective TCF-SEAP assay. (<b>C</b>) Comparison of iLRP5 and iLRP5 mutants. YSS→FAA and YRPY→FLPF mutants have lower TCF-SEAP activity. The double mutant fails to activate Wnt <span class="highlight" style="background-color:">signaling</span>. Western blot indicates protein expression from respective TCF-SEAP assay. (B and C), Data representative of two separate experiments performed in triplicates.</div></p></div><div class="aux"><div class="resc">Payam Shahi, et al. PLoS One. 2012;7(1):e30814.</div><p class="links"><a class="dblinks" href="/pubmed/22303459" ref="ordinalpos=4450&amp;ncbi_uid=3785156&amp;link_uid=22303459">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267738/" ref="ordinalpos=4450&amp;ncbi_uid=3785156&amp;link_uid=PMC3267738">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4451.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2556008/figure/fig10/" ref="ordinalpos=4451&amp;ncbi_uid=1523044&amp;link_uid=PMC2556008" href="/pmc/articles/PMC2556008/"><img src="/pmc/articles/instance/2556008/bin/zbc0420851670010.gif" src-large="/pmc/articles/instance/2556008/bin/zbc0420851670010.jpg" alt="FIGURE 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2556008/" ref="ordinalpos=4451&amp;ncbi_uid=1523044&amp;link_uid=PMC2556008" image-link="/pmc/articles/PMC2556008/figure/fig10/" class="imagepopup">FIGURE 10. From: Focal Adhesion Kinase (FAK)-related Non-kinase Inhibits Myofibroblast  Differentiation through Differential MAPK Activation in a FAK-dependent  Manner. </a></p><div class="supp"><p class="details"><div><b>Hypothetical mechanism whereby FRNK inhibits TGF-β-induced  α-SMA expression in the presence and absence of FAK.</b>  TGF-β-induced α-SMA expression and myofibroblast differentiation  requires the involvement of multiple <span class="highlight" style="background-color:">signaling</span> initiated through the  integrins, extracellular matrix (<i>ECM</i>) environment, and TGF-β  receptors (<i>TGF-</i>β<i>R</i>). Recent evidence demonstrates that  certain integrins, in cooperation with the cytoskeleton, are capable of  activating latent TGF-β through binding to latency-associated protein  (21,  26,  68). TGF-β induces  α-SMA expression through two pathways; the major <span class="highlight" style="background-color:">pathway</span> through ERK  MAPK is FAK-dependent, and the minor <span class="highlight" style="background-color:">pathway</span> through p38 MAPK is  FAK-independent. FRNK can inhibit TGF-β-induced α-SMA expression  through inhibition of FAK activation and inhibition of the FAK/ERK-mediated  <span class="highlight" style="background-color:">pathway</span>. FRNK also can inhibit the <span class="highlight" style="background-color:">pathway</span> of TGF-β-induced α-SMA  expression mediated by p38, potentially through inhibition of critical  <span class="highlight" style="background-color:">signaling</span> proteins upstream of p38 (<i>i.e.</i> Src kinase).</div></p></div><div class="aux"><div class="resc">Qiang Ding, et al. J Biol Chem. 2008 October 3;283(40):26839-26849.</div><p class="links"><a class="dblinks" href="/pubmed/18669633" ref="ordinalpos=4451&amp;ncbi_uid=1523044&amp;link_uid=18669633">Citation</a><a class="dblinks" href="/pmc/articles/PMC2556008/" ref="ordinalpos=4451&amp;ncbi_uid=1523044&amp;link_uid=PMC2556008">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4452.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2857450/figure/f1/" ref="ordinalpos=4452&amp;ncbi_uid=2268476&amp;link_uid=PMC2857450" href="/pmc/articles/PMC2857450/"><img src="/pmc/articles/instance/2857450/bin/fig-1.gif" src-large="/pmc/articles/instance/2857450/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2857450/" ref="ordinalpos=4452&amp;ncbi_uid=2268476&amp;link_uid=PMC2857450" image-link="/pmc/articles/PMC2857450/figure/f1/" class="imagepopup">FIG. 1. From: Potent Inhibition of Thyroid Cancer Cells by the MEK Inhibitor PD0325901 and Its Potentiation by Suppression of the PI3K and NF-?B Pathways. </a></p><div class="supp"><p class="details"><div>PD0325901 inhibited the MAPK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in thyroid tumor cells. Cells were lysed for Western blotting after PD0325901 treatment. The MAPK <span class="highlight" style="background-color:">pathway</span> activity was reflected by the level of phosphorylated ERK detected with specific antiphosphorylated ERK (p-ERK) antibody. Total ERK (t-ERK) was used for quantity control of proteins. (<b>A</b>) Concentration response of the inhibitory effects of PD0325901 on MAPK <span class="highlight" style="background-color:">pathway</span> in two <i>B-type RAF</i> (<i>BRAF</i>) mutation–harboring thyroid cancer cell lines KAT5 and DRO. Cells were treated with the indicated concentrations of PD0325901 for 2 hours. (<b>B</b>) Time course of the inhibitory effects of PD0325901 on MAPK <span class="highlight" style="background-color:">pathway</span> in KAT5 and DRO cells. Cells were treated with 0.1 μM PD0325901 for the indicated times. (<b>C</b>) Inhibitory effects of PD0325901 on MAPK <span class="highlight" style="background-color:">pathway</span> in various thyroid tumor cells. Cells were treated with 0.1 μM PD0325901 (+) or vehicle DMSO (−) for 24 hours.</div></p></div><div class="aux"><div class="resc">Dingxie Liu, et al. Thyroid. 2008 August;18(8):853-864.</div><p class="links"><a class="dblinks" href="/pubmed/18651802" ref="ordinalpos=4452&amp;ncbi_uid=2268476&amp;link_uid=18651802">Citation</a><a class="dblinks" href="/pmc/articles/PMC2857450/" ref="ordinalpos=4452&amp;ncbi_uid=2268476&amp;link_uid=PMC2857450">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4453.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3134581/figure/F7/" ref="ordinalpos=4453&amp;ncbi_uid=3550322&amp;link_uid=PMC3134581" href="/pmc/articles/PMC3134581/"><img src="/pmc/articles/instance/3134581/bin/nihms283238f7.gif" src-large="/pmc/articles/instance/3134581/bin/nihms283238f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3134581/" ref="ordinalpos=4453&amp;ncbi_uid=3550322&amp;link_uid=PMC3134581" image-link="/pmc/articles/PMC3134581/figure/F7/" class="imagepopup">Fig. 7.  From: Rapid Communications (2nd Revision) Modulation of the tumor cell death <span class="highlight" style="background-color:">pathway</span> by androgen receptor in response to cytotoxic stimuli. </a></p><div class="supp"><p class="details"><div>Schematic representation of the proposed mechanism for Casodex plus proteasome inhibitor combinational therapy for prostate cancer treatment. The combination of Casodex and Velcade should cause dual inhibition of the androgen-AR-mediated survival <span class="highlight" style="background-color:">pathway</span> in prostate cancer cells since Casodex-mediated inhibition is <i>via</i> its binding to AR and preventing the activation of AR <span class="highlight" style="background-color:">signaling</span> while Velcade’s action involves activation of calpain that consequently cleaves AR protein. Such combinational strategy could significantly enhance AR <span class="highlight" style="background-color:">signaling</span> inhibition and ultimately prostate cancer cell death, which may provide a compelling strategy for the treatment of HRPC.</div></p></div><div class="aux"><div class="resc">Michael Frezza, et al. J Cell Physiol. ;226(11):2731-2739.</div><p class="links"><a class="dblinks" href="/pubmed/21448923" ref="ordinalpos=4453&amp;ncbi_uid=3550322&amp;link_uid=21448923">Citation</a><a class="dblinks" href="/pmc/articles/PMC3134581/" ref="ordinalpos=4453&amp;ncbi_uid=3550322&amp;link_uid=PMC3134581">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4454.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2453232/figure/pone-0002750-g010/" ref="ordinalpos=4454&amp;ncbi_uid=1442976&amp;link_uid=PMC2453232" href="/pmc/articles/PMC2453232/"><img src="/pmc/articles/instance/2453232/bin/pone.0002750.g010.gif" src-large="/pmc/articles/instance/2453232/bin/pone.0002750.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2453232/" ref="ordinalpos=4454&amp;ncbi_uid=1442976&amp;link_uid=PMC2453232" image-link="/pmc/articles/PMC2453232/figure/pone-0002750-g010/" class="imagepopup">Figure 10. KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis of up- or down-regulated protein sets from cortex and hippocampus from 3xTgAD animals..  From: iTRAQ Analysis of Complex Proteome Alterations in 3xTgAD Alzheimer's Mice: Understanding the Interface between Physiology and Disease. </a></p><div class="supp"><p class="details"><div>Panels A (cortex) and B (hippocampus) depict the significantly represented (p&lt;0.05) KEGG <span class="highlight" style="background-color:">signaling</span> pathways and their relative enrichment factor compared to background murine sets created by the up-regulated proteins (in 3xTgAD compared to control) identified in each tissue. Panels C (cortex) and D (hippocampus) depict analogous data to Panels A and B but for input protein sets down-regulated in 3xTgAD compared to control.</div></p></div><div class="aux"><div class="resc">Bronwen Martin, et al. PLoS ONE. 2008;3(7):e2750.</div><p class="links"><a class="dblinks" href="/pubmed/18648646" ref="ordinalpos=4454&amp;ncbi_uid=1442976&amp;link_uid=18648646">Citation</a><a class="dblinks" href="/pmc/articles/PMC2453232/" ref="ordinalpos=4454&amp;ncbi_uid=1442976&amp;link_uid=PMC2453232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4455.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4151266/figure/F5/" ref="ordinalpos=4455&amp;ncbi_uid=6551373&amp;link_uid=PMC4151266" href="/pmc/articles/PMC4151266/"><img src="/pmc/articles/instance/4151266/bin/nihms-624103-f0005.gif" src-large="/pmc/articles/instance/4151266/bin/nihms-624103-f0005.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4151266/" ref="ordinalpos=4455&amp;ncbi_uid=6551373&amp;link_uid=PMC4151266" image-link="/pmc/articles/PMC4151266/figure/F5/" class="imagepopup">Fig. 5.  From: Transcriptome dynamics during human erythroid differentiation and development. </a></p><div class="supp"><p class="details"><div>Pathways activated in each comparison. Pathways activated in the three stage transitions containing central nodes were identified using IPA. (A) Activation of the FLT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ESER cells compared with that in HESCs. (B) Activation of cyclins and cell cycle regulation <span class="highlight" style="background-color:">pathway</span> in FLER cells compared with that in ESER cells. (C) Activation of ILK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PBER cells compared with that in FLER cells. Red and green represent up-regulated and down-regulated genes, respectively. The extent of regulation is based on the color grade. Symbols with gradient colors indicate a gene family whose members change in different degree.</div></p></div><div class="aux"><div class="resc">Yadong Yang, et al. Genomics. ;102(0):431-441.</div><p class="links"><a class="dblinks" href="/pubmed/24121002" ref="ordinalpos=4455&amp;ncbi_uid=6551373&amp;link_uid=24121002">Citation</a><a class="dblinks" href="/pmc/articles/PMC4151266/" ref="ordinalpos=4455&amp;ncbi_uid=6551373&amp;link_uid=PMC4151266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4456.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4405777/figure/F2/" ref="ordinalpos=4456&amp;ncbi_uid=7448074&amp;link_uid=PMC4405777" href="/pmc/articles/PMC4405777/"><img src="/pmc/articles/instance/4405777/bin/nihms428482f2.gif" src-large="/pmc/articles/instance/4405777/bin/nihms428482f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4405777/" ref="ordinalpos=4456&amp;ncbi_uid=7448074&amp;link_uid=PMC4405777" image-link="/pmc/articles/PMC4405777/figure/F2/" class="imagepopup">Figure 2.  From: A potent combination of the novel PI3K inhibitor, GDC-0941, with imatinib in gastrointestinal stromal tumor xenografts: long-lasting responses after treatment withdrawal. </a></p><div class="supp"><p class="details"><div>Densitometric assessment of active (phospho-) protein forms in KIT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Giuseppe Floris, et al. Clin Cancer Res. ;19(3):620-630.</div><p class="links"><a class="dblinks" href="/pubmed/23231951" ref="ordinalpos=4456&amp;ncbi_uid=7448074&amp;link_uid=23231951">Citation</a><a class="dblinks" href="/pmc/articles/PMC4405777/" ref="ordinalpos=4456&amp;ncbi_uid=7448074&amp;link_uid=PMC4405777">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4457.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4196580/figure/F6/" ref="ordinalpos=4457&amp;ncbi_uid=6767319&amp;link_uid=PMC4196580" href="/pmc/articles/PMC4196580/"><img src="/pmc/articles/instance/4196580/bin/fgene-05-00348-g0006.gif" src-large="/pmc/articles/instance/4196580/bin/fgene-05-00348-g0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4196580/" ref="ordinalpos=4457&amp;ncbi_uid=6767319&amp;link_uid=PMC4196580" image-link="/pmc/articles/PMC4196580/figure/F6/" class="imagepopup">Figure 6.  From: Characterization of the rainbow trout spleen transcriptome and identification of immune-related genes. </a></p><div class="supp"><p class="details"><div><b>Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing the annotated and non-annotated proteins</b>. Proteins appearing in our transcriptome are represented in green color and absent proteins in white color.</div></p></div><div class="aux"><div class="resc">Ali Ali, et al. Front Genet. 2014;5:348.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4457&amp;ncbi_uid=6767319&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4196580/" ref="ordinalpos=4457&amp;ncbi_uid=6767319&amp;link_uid=PMC4196580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4458.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3857070/figure/f2-pharmaceutics-03-00338/" ref="ordinalpos=4458&amp;ncbi_uid=5606155&amp;link_uid=PMC3857070" href="/pmc/articles/PMC3857070/"><img src="/pmc/articles/instance/3857070/bin/pharmaceutics-03-00338f2.gif" src-large="/pmc/articles/instance/3857070/bin/pharmaceutics-03-00338f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3857070/" ref="ordinalpos=4458&amp;ncbi_uid=5606155&amp;link_uid=PMC3857070" image-link="/pmc/articles/PMC3857070/figure/f2-pharmaceutics-03-00338/" class="imagepopup">Figure 2. From: Adenovirus Vector-Derived VA-RNA-Mediated Innate Immune Responses. </a></p><div class="supp"><p class="details"><div>VA-RNA-induced production of type I IFNs through an IPS-1-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. VA-RNA: virus-associated RNA; RIG-I: retinoic acid-inducible gene I; IPS-1: IFN-β promoter stimulator-1; IFN: interferon.</div></p></div><div class="aux"><div class="resc">Mitsuhiro Machitani, et al. Pharmaceutics. 2011 September;3(3):338-353.</div><p class="links"><a class="dblinks" href="/pubmed/24310584" ref="ordinalpos=4458&amp;ncbi_uid=5606155&amp;link_uid=24310584">Citation</a><a class="dblinks" href="/pmc/articles/PMC3857070/" ref="ordinalpos=4458&amp;ncbi_uid=5606155&amp;link_uid=PMC3857070">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4459.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109859/figure/f5-0110124/" ref="ordinalpos=4459&amp;ncbi_uid=3157743&amp;link_uid=PMC3109859" href="/pmc/articles/PMC3109859/"><img src="/pmc/articles/instance/3109859/bin/Fig5_REVRettie.gif" src-large="/pmc/articles/instance/3109859/bin/Fig5_REVRettie.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109859/" ref="ordinalpos=4459&amp;ncbi_uid=3157743&amp;link_uid=PMC3109859" image-link="/pmc/articles/PMC3109859/figure/f5-0110124/" class="imagepopup">Figure 5. Anabolism of protectins and resolvins from DHA.  From: Finding Homes for Orphan Cytochrome P450s: CYP4V2 and CYP4F22 in Disease States. </a></p><div class="supp"><p class="details"><div>ω-Hydroxylation catalyzed by CYP4V2 may act at one or more levels of either <span class="highlight" style="background-color:">pathway</span> to modulate protectin/resolvin <span class="highlight" style="background-color:">signaling</span>. (See text for details.)</div></p></div><div class="aux"><div class="resc">Edward J. Kelly, et al. Mol Interv. 2011 April;11(2):124-132.</div><p class="links"><a class="dblinks" href="/pubmed/21540472" ref="ordinalpos=4459&amp;ncbi_uid=3157743&amp;link_uid=21540472">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109859/" ref="ordinalpos=4459&amp;ncbi_uid=3157743&amp;link_uid=PMC3109859">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4460.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC28451/figure/F7/" ref="ordinalpos=4460&amp;ncbi_uid=31039&amp;link_uid=PMC28451" href="/pmc/articles/PMC28451/"><img src="/pmc/articles/instance/28451/bin/pq2572499007.gif" src-large="/pmc/articles/instance/28451/bin/pq2572499007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC28451/" ref="ordinalpos=4460&amp;ncbi_uid=31039&amp;link_uid=PMC28451" image-link="/pmc/articles/PMC28451/figure/F7/" class="imagepopup">Figure 7.  From: Chlorophyll precursors are signals of chloroplast origin involved in light induction of nuclear heat-shock genes. </a></p><div class="supp"><p class="details"><div>Model for the light <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from chloroplast to nucleus. The model is explained in the text. ALA, δ-aminolevulinic acid; CHL, chlorophyll.</div></p></div><div class="aux"><div class="resc">Janette Kropat, et al. Proc Natl Acad Sci U S A. 1997 December 9;94(25):14168-14172.</div><p class="links"><a class="dblinks" href="/pubmed/9391171" ref="ordinalpos=4460&amp;ncbi_uid=31039&amp;link_uid=9391171">Citation</a><a class="dblinks" href="/pmc/articles/PMC28451/" ref="ordinalpos=4460&amp;ncbi_uid=31039&amp;link_uid=PMC28451">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4461.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3824880/figure/Fig1/" ref="ordinalpos=4461&amp;ncbi_uid=5514829&amp;link_uid=PMC3824880" href="/pmc/articles/PMC3824880/"><img src="/pmc/articles/instance/3824880/bin/294_2013_413_Fig1_HTML.gif" src-large="/pmc/articles/instance/3824880/bin/294_2013_413_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3824880/" ref="ordinalpos=4461&amp;ncbi_uid=5514829&amp;link_uid=PMC3824880" image-link="/pmc/articles/PMC3824880/figure/Fig1/" class="imagepopup">Fig. 1.  From: Yeast nutrient transceptors provide novel insight in the functionality of membrane transporters. </a></p><div class="supp"><p class="details"><div>Model of the Pho84 phosphate transceptor with two putative proton-binding residues allowing uncoupling of transport and <span class="highlight" style="background-color:">signaling</span>. The protein has 12 predicted transmembrane domains and functions as a phosphate/proton symporter. D178 and D358 are two conserved putative proton-binding residues, predicted to be located adjacent to the phosphate translocation <span class="highlight" style="background-color:">pathway</span>. Mutagenesis of D178 to E or N reduces <i>V</i> <sub>max</sub> with about 50 %, but does not lower <span class="highlight" style="background-color:">signaling</span> as measured by phosphate-induced activation of the PKA target trehalase. Mutagenesis of D358 to N strongly reduces transport to ≤20 % but does not affect <span class="highlight" style="background-color:">signaling</span>. Mutagenesis of D358 to E completely abolishes transport but only causes a 50 % drop in <span class="highlight" style="background-color:">signaling</span></div></p></div><div class="aux"><div class="resc">Joep Schothorst, et al. Curr Genet. 2013;59(4):197-206.</div><p class="links"><a class="dblinks" href="/pubmed/24114446" ref="ordinalpos=4461&amp;ncbi_uid=5514829&amp;link_uid=24114446">Citation</a><a class="dblinks" href="/pmc/articles/PMC3824880/" ref="ordinalpos=4461&amp;ncbi_uid=5514829&amp;link_uid=PMC3824880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4462.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3295195/figure/F9/" ref="ordinalpos=4462&amp;ncbi_uid=4209492&amp;link_uid=PMC3295195" href="/pmc/articles/PMC3295195/"><img src="/pmc/articles/instance/3295195/bin/zmb9991094060009.gif" src-large="/pmc/articles/instance/3295195/bin/zmb9991094060009.jpg" alt="Fig 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3295195/" ref="ordinalpos=4462&amp;ncbi_uid=4209492&amp;link_uid=PMC3295195" image-link="/pmc/articles/PMC3295195/figure/F9/" class="imagepopup">Fig 9.  From: Angiotensin II Induces Epithelial-to-Mesenchymal Transition in Renal Epithelial Cells through Reactive Oxygen Species/Src/Caveolin-Mediated Activation of an Epidermal Growth Factor Receptor-Extracellular Signal-Regulated Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Ang II induced EGFR-ERK activation and EMT by a Cav-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in AT1R/Cl4 cells. (A) At 48 h after transfection with different Cav siRNAs or scrambled control siRNA, quiescent cells lysates were subjected to immunoblotting with indicated antibodies following exposure to indicated stimuli for 10 min. (B to D) At 48 h after siRNA transfection as indicated, quiescent <i>AT1R/Cl4</i> cell lysates were subjected to immunoblotting analysis with indicated antibodies (B) or the cells were subjected to induction of EMT with Ang II as indicated in Fig. 1, cell photographs were taken at ×400 magnification following FITC-conjugated phalloidin staining (C), and cell lysates were subjected to immunoblotting analysis with the indicated antibodies (D).</div></p></div><div class="aux"><div class="resc">Jianchun Chen, et al. Mol Cell Biol. 2012 March;32(5):981-991.</div><p class="links"><a class="dblinks" href="/pubmed/22215616" ref="ordinalpos=4462&amp;ncbi_uid=4209492&amp;link_uid=22215616">Citation</a><a class="dblinks" href="/pmc/articles/PMC3295195/" ref="ordinalpos=4462&amp;ncbi_uid=4209492&amp;link_uid=PMC3295195">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4463.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3996160/figure/Fig6/" ref="ordinalpos=4463&amp;ncbi_uid=6051520&amp;link_uid=PMC3996160" href="/pmc/articles/PMC3996160/"><img src="/pmc/articles/instance/3996160/bin/13238_2014_26_Fig6_HTML.gif" src-large="/pmc/articles/instance/3996160/bin/13238_2014_26_Fig6_HTML.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3996160/" ref="ordinalpos=4463&amp;ncbi_uid=6051520&amp;link_uid=PMC3996160" image-link="/pmc/articles/PMC3996160/figure/Fig6/" class="imagepopup">Figure 6.  From: SARS coronavirus papain-like protease inhibits the type I interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through interaction with the STING-TRAF3-TBK1 complex. </a></p><div class="supp"><p class="details"><div> <b>A hypothetical model depicting the mechanisms used by SARS-CoV PLpro-TM to disrupt the STING-TRAF3-TBK1 complex required for activation of the IFN-β production <span class="highlight" style="background-color:">pathway</span></b>. (A) Viral infections are sensed by pattern-recognition receptors (PRRs), which in turn recognize pathogen-associated molecular patterns (PAMPs) and trigger antiviral responses. RIG-I identifies the 5′-3p RNA, and then interacts with MAVS to activate the IFN responses factor 3 (IRF3). In MAMs (mitochondrial-associated membranes), MAVS also interacts with STING that locates at the ER (endoplasmic reticulum), and induces the ubiquitination and dimerization of STING. The activated STING recruits TBK1 and IRF3 and contributes to the phosphorylation of IRF3 mediated by TBK1. STING recruits TBK1 and IKKε and forms the TBK1-IKKε complex via the association with TRAF3. The TBK1 complex induces the phosphorylation, dimerization, and nuclear translocation of IRF3. (B) When SARS-CoV invades the host, these processes are disrupted by SARS-CoV PLpro-TM. PLpro-TM disrupts activation of the IRF3 <span class="highlight" style="background-color:">pathway</span> induced by STING-TRAF3-TBK1 by blocking the assembly of the complex, and it deubiquitinates the <span class="highlight" style="background-color:">signaling</span> molecules RIG-I, TRAF3, STING, TBK1, and IRF3</div></p></div><div class="aux"><div class="resc">Xiaojuan Chen, et al. Protein Cell. 2014 May;5(5):369-381.</div><p class="links"><a class="dblinks" href="/pubmed/24622840" ref="ordinalpos=4463&amp;ncbi_uid=6051520&amp;link_uid=24622840">Citation</a><a class="dblinks" href="/pmc/articles/PMC3996160/" ref="ordinalpos=4463&amp;ncbi_uid=6051520&amp;link_uid=PMC3996160">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4464.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2770888/figure/F2/" ref="ordinalpos=4464&amp;ncbi_uid=2163181&amp;link_uid=PMC2770888" href="/pmc/articles/PMC2770888/"><img src="/pmc/articles/instance/2770888/bin/nihms99057f2.gif" src-large="/pmc/articles/instance/2770888/bin/nihms99057f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2770888/" ref="ordinalpos=4464&amp;ncbi_uid=2163181&amp;link_uid=PMC2770888" image-link="/pmc/articles/PMC2770888/figure/F2/" class="imagepopup">Figure 2. Resistance mechanisms originating from EGFR downstream effectors.  From: Targeting EGFR resistance networks in Head and Neck Cancer. </a></p><div class="supp"><p class="details"><div>Cell <span class="highlight" style="background-color:">signaling</span> effectors downstream of EGFR may mitigate the efficacy of EGFR inhibitors by maintaining <span class="highlight" style="background-color:">signaling</span> activity to drive cell cycle progression and inhibit apoptosis. Increased <span class="highlight" style="background-color:">signaling</span> of the PI3K-Akt <span class="highlight" style="background-color:">pathway</span>, STAT proteins and cell cycle progression mediators such as Cyclin D1 have been association with resistance to EGFR inhibitors in SCCHN. See text for details.</div></p></div><div class="aux"><div class="resc">Vladimir Ratushny, et al. Cell Signal. ;21(8):1255-1268.</div><p class="links"><a class="dblinks" href="/pubmed/19258037" ref="ordinalpos=4464&amp;ncbi_uid=2163181&amp;link_uid=19258037">Citation</a><a class="dblinks" href="/pmc/articles/PMC2770888/" ref="ordinalpos=4464&amp;ncbi_uid=2163181&amp;link_uid=PMC2770888">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4465.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3126665/figure/F3/" ref="ordinalpos=4465&amp;ncbi_uid=3203778&amp;link_uid=PMC3126665" href="/pmc/articles/PMC3126665/"><img src="/pmc/articles/instance/3126665/bin/nihms-299238-f0003.gif" src-large="/pmc/articles/instance/3126665/bin/nihms-299238-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3126665/" ref="ordinalpos=4465&amp;ncbi_uid=3203778&amp;link_uid=PMC3126665" image-link="/pmc/articles/PMC3126665/figure/F3/" class="imagepopup">Figure 3.  From: SULF-2: AN EXTRACELLULAR MODULATOR OF CELL <span class="highlight" style="background-color:">SIGNALING</span> AND A CANCER TARGET CANDIDATE. </a></p><div class="supp"><p class="details"><div>Model for Sulf regulation of Wnt <span class="highlight" style="background-color:">signaling</span>. The model of Ai et al. [35] proposes that Wnt ligands are sequestered away from signal transduction receptors (Frizzled’s) because of an association with HSPGs on the cell surface or ECM. The removal of 6-O-S (denoted in the solid color with the other S modifications denoted in white) from heparan sulfate chains by the action of the Sulf weaken the association of the Wnt ligand with the HSPG, which permits engagement of the Frizzled receptor. This initiates a chain of cytoplasmic events which leads to the accumulation of β-catenin in the nucleus of the cell and culminates in the activation of a set of “Wnt target genes” (“canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>”). The Sulfs, on the hand, can negatively regulate other <span class="highlight" style="background-color:">signaling</span> pathways. For example, the Sulfs inhibit FGF-2 mediated <span class="highlight" style="background-color:">signaling</span> by apparently disrupting ternary <span class="highlight" style="background-color:">signaling</span> complexes, consisting of FGF-2, an HSPG, and the FGF-2 receptor, FGFR1 [51]. A diagram depicting the negative the effect of a Sulf on FGF-2 <span class="highlight" style="background-color:">signaling</span> is provided in [4]. The Sulf is shown in forms: bound to the plasma membrane and as a soluble secreted protein.</div></p></div><div class="aux"><div class="resc">Steven D. Rosen, et al. Expert Opin Ther Targets. ;14(9):935-949.</div><p class="links"><a class="dblinks" href="/pubmed/20629619" ref="ordinalpos=4465&amp;ncbi_uid=3203778&amp;link_uid=20629619">Citation</a><a class="dblinks" href="/pmc/articles/PMC3126665/" ref="ordinalpos=4465&amp;ncbi_uid=3203778&amp;link_uid=PMC3126665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4466.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4394565/figure/Fig3/" ref="ordinalpos=4466&amp;ncbi_uid=7414343&amp;link_uid=PMC4394565" href="/pmc/articles/PMC4394565/"><img src="/pmc/articles/instance/4394565/bin/12865_2015_80_Fig3_HTML.gif" src-large="/pmc/articles/instance/4394565/bin/12865_2015_80_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4394565/" ref="ordinalpos=4466&amp;ncbi_uid=7414343&amp;link_uid=PMC4394565" image-link="/pmc/articles/PMC4394565/figure/Fig3/" class="imagepopup">Figure 3.  From: Wnt/β-Catenin <span class="highlight" style="background-color:">signaling</span> reduces Bacillus Calmette-Guerin-induced macrophage necrosis through a ROS -mediated PARP/AIF-dependent <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>Impacts of BCG and/or Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> on ROS production in RAW264.7 cells. (A)</b> RAW264.7 cells were infected with BCG at MOI of 10 for indicated time, then they were used for intracellular ROS measurement by a flow cytometry assay. A time-dependent ROS production was observed within 6 h. <b>(B)</b> RAW264.7 cells were infected with indicated doses of BCG for 6 h prior to be used for examination of intracellular ROS by a flow cytometry assay. A BCG dose-dependent ROS production was observed. <b>(C)</b> Impact of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> on BCG-induced ROS production in RAW264.7 cells. RAW264.7 cells were infected with BCG at MOI of 10 for 6 h prior to be used for measuring intracellular ROS by a flow cytometry assay. An activation of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> exhibited an ability to reduce BCG-induced ROS production. <b>(D)</b> Impact of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> on oxidative stress in RAW264.7 cells. RAW264.7 cells were exposed to 500 μmol/L of H<sub>2</sub>O<sub>2</sub> for 6 h before they were harvested for intracellular ROS measurement. The addition of Wnt3a showed a capacity to scavenge oxidative stressed-ROS accumulation.</div></p></div><div class="aux"><div class="resc">Xiaoling Wu, et al. BMC Immunol. 2015;16:16.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4466&amp;ncbi_uid=7414343&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4394565/" ref="ordinalpos=4466&amp;ncbi_uid=7414343&amp;link_uid=PMC4394565">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4467.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3047097/figure/F6/" ref="ordinalpos=4467&amp;ncbi_uid=3406153&amp;link_uid=PMC3047097" href="/pmc/articles/PMC3047097/"><img src="/pmc/articles/instance/3047097/bin/cbt1010_1056_fig006.gif" src-large="/pmc/articles/instance/3047097/bin/cbt1010_1056_fig006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3047097/" ref="ordinalpos=4467&amp;ncbi_uid=3406153&amp;link_uid=PMC3047097" image-link="/pmc/articles/PMC3047097/figure/F6/" class="imagepopup">Figure 6.  From: The induction of S100p expression by the Prostaglandin E2 (PGE2)/EP4 receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in colon cancer cells. </a></p><div class="supp"><p class="details"><div>Model showing PGE<sub>2</sub>/EP4/CREB/S100P <span class="highlight" style="background-color:">signaling</span> in a feedback regulatory loop.</div></p></div><div class="aux"><div class="resc">Anupama Chandramouli, et al. Cancer Biol Ther. 2010 November 15;10(10):1056-1066.</div><p class="links"><a class="dblinks" href="/pubmed/20890108" ref="ordinalpos=4467&amp;ncbi_uid=3406153&amp;link_uid=20890108">Citation</a><a class="dblinks" href="/pmc/articles/PMC3047097/" ref="ordinalpos=4467&amp;ncbi_uid=3406153&amp;link_uid=PMC3047097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4468.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012993/figure/pone-0091376-g007/" ref="ordinalpos=4468&amp;ncbi_uid=6121814&amp;link_uid=PMC4012993" href="/pmc/articles/PMC4012993/"><img src="/pmc/articles/instance/4012993/bin/pone.0091376.g007.gif" src-large="/pmc/articles/instance/4012993/bin/pone.0091376.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012993/" ref="ordinalpos=4468&amp;ncbi_uid=6121814&amp;link_uid=PMC4012993" image-link="/pmc/articles/PMC4012993/figure/pone-0091376-g007/" class="imagepopup">Figure 7. Scap modulates expression of Wnt <span class="highlight" style="background-color:">pathway</span> components..  From: Epithelial SCAP/INSIG/SREBP <span class="highlight" style="background-color:">Signaling</span> Regulates Multiple Biological Processes during Perinatal Lung Maturation. </a></p><div class="supp"><p class="details"><div>A and B) qPCR analysis demonstrates altered expression of Wnt <span class="highlight" style="background-color:">signaling</span> genes in <i>Scap<sup>Δ/Δ</sup></i> E17.5 (A) and PN1 (B) whole lung tissue compared to littermate controls. Results were normalized to 18 s RNA, n = 3–4/group, *p&lt;0.05 vs. control at the respective time point. C and D) qPCR analysis demonstrates decreased expression of <i>Scap</i> (C) and of Wnt <span class="highlight" style="background-color:">pathway</span> (D) genes in MLE15 cells following transfection with negative control (neg con) and two distinct <i>Scap</i> siRNAs (si1 and si2). Results were normalized to 18 s, n = 6 samples/group representing 3 independent experiments, *p&lt;0.05 vs negative control. ND = non-detectable.</div></p></div><div class="aux"><div class="resc">James P. Bridges, et al. PLoS One. 2014;9(5):e91376.</div><p class="links"><a class="dblinks" href="/pubmed/24806461" ref="ordinalpos=4468&amp;ncbi_uid=6121814&amp;link_uid=24806461">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012993/" ref="ordinalpos=4468&amp;ncbi_uid=6121814&amp;link_uid=PMC4012993">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4469.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3657461/figure/fig4/" ref="ordinalpos=4469&amp;ncbi_uid=4964524&amp;link_uid=PMC3657461" href="/pmc/articles/PMC3657461/"><img src="/pmc/articles/instance/3657461/bin/PC2013-920612.004.gif" src-large="/pmc/articles/instance/3657461/bin/PC2013-920612.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3657461/" ref="ordinalpos=4469&amp;ncbi_uid=4964524&amp;link_uid=PMC3657461" image-link="/pmc/articles/PMC3657461/figure/fig4/" class="imagepopup">Figure 4.  From: Dissecting Major <span class="highlight" style="background-color:">Signaling</span> Pathways throughout the Development of Prostate Cancer. </a></p><div class="supp"><p class="details"><div>The JAK/STAT <span class="highlight" style="background-color:">signaling</span> in prostate cancer. (1) The JAK/STAT <span class="highlight" style="background-color:">pathway</span> has been found constitutively activated in PCa cells, leading to induction of tumor cell proliferation and apoptosis inhibition mediated by STAT3 activation. (2) BRCA1/2 is required for DNA repair in normal cells. However, in PCa, BRCA1 can bind STAT3 to promote JAK/STAT3 activation. (3) AR is a well-characterized cross-talk <span class="highlight" style="background-color:">pathway</span> in PCa. When activated, AR can bind to STAT3 leading to the activation of JAK/STAT cascade, being important in the induction of cell proliferation and apoptosis inhibition. (4) Under stress conditions, ATF3 is activated and plays a crucial role in the maintenance of cell integrity and homeostasis. ATF3 does so by interacting with AR, leading to inhibition of androgen <span class="highlight" style="background-color:">signaling</span> and, consequently, the inhibition of cell proliferation. However, ATF3 is downregulated in PCa cells, suggesting that this <span class="highlight" style="background-color:">pathway</span> provides an important mechanism of defense against cancer. (5) Similarly, C/EBP<i>δ</i> is required to inhibit cell proliferation by binding to STAT3. Nevertheless, C/EBP<i>δ</i> is typically downregulated in PCa, and, therefore, it could be used as an strategy in the development of therapeutic drugs against PCa growth.</div></p></div><div class="aux"><div class="resc">Henrique B. da Silva, et al. Prostate Cancer. 2013;2013:920612.</div><p class="links"><a class="dblinks" href="/pubmed/23738079" ref="ordinalpos=4469&amp;ncbi_uid=4964524&amp;link_uid=23738079">Citation</a><a class="dblinks" href="/pmc/articles/PMC3657461/" ref="ordinalpos=4469&amp;ncbi_uid=4964524&amp;link_uid=PMC3657461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4470.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3410857/figure/pgen-1002864-g007/" ref="ordinalpos=4470&amp;ncbi_uid=4167712&amp;link_uid=PMC3410857" href="/pmc/articles/PMC3410857/"><img src="/pmc/articles/instance/3410857/bin/pgen.1002864.g007.gif" src-large="/pmc/articles/instance/3410857/bin/pgen.1002864.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3410857/" ref="ordinalpos=4470&amp;ncbi_uid=4167712&amp;link_uid=PMC3410857" image-link="/pmc/articles/PMC3410857/figure/pgen-1002864-g007/" class="imagepopup">Figure 7. Model for mir-71 function in AWC asymmetry..  From: The MicroRNA mir-71 Inhibits Calcium <span class="highlight" style="background-color:">Signaling</span> by Targeting the TIR-1/Sarm1 Adaptor Protein to Control Stochastic L/R Neuronal Asymmetry in C. elegans. </a></p><div class="supp"><p class="details"><div>In the default AWC<sup>OFF</sup> cell, <i>tir-1</i> acts in a calcium-regulated kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to represses the expression of the AWC<sup>ON</sup> marker <i>str-2</i>. Both <i>nsy-4</i> and <i>nsy-5</i> act to increase the level of mature <i>mir-71</i>, which results in downregulation of <i>tir-1</i> expression and subsequent de-repression of <i>str-2</i> gene expression in the cell that becomes AWC<sup>ON</sup>. Gray is used to indicate the gene product is less active or inactive.</div></p></div><div class="aux"><div class="resc">Yi-Wen Hsieh, et al. PLoS Genet. 2012 August;8(8):e1002864.</div><p class="links"><a class="dblinks" href="/pubmed/22876200" ref="ordinalpos=4470&amp;ncbi_uid=4167712&amp;link_uid=22876200">Citation</a><a class="dblinks" href="/pmc/articles/PMC3410857/" ref="ordinalpos=4470&amp;ncbi_uid=4167712&amp;link_uid=PMC3410857">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4471.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1150820/figure/fig3/" ref="ordinalpos=4471&amp;ncbi_uid=502590&amp;link_uid=PMC1150820" href="/pmc/articles/PMC1150820/"><img src="/pmc/articles/instance/1150820/bin/zpq0210583550003.gif" src-large="/pmc/articles/instance/1150820/bin/zpq0210583550003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1150820/" ref="ordinalpos=4471&amp;ncbi_uid=502590&amp;link_uid=PMC1150820" image-link="/pmc/articles/PMC1150820/figure/fig3/" class="imagepopup">Fig. 3. From: Uterine Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> is required for implantation. </a></p><div class="supp"><p class="details"><div>Wnt proteins known to activate the canonical Wnt <span class="highlight" style="background-color:">pathway</span> can selectively activate Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> in the uterus. (<i>A</i>) Whole-mount β-galactosidase staining of a TCF/Lef-lacZ transgenic uterus injected with Wnt5a-conditioned media in both uterine horns. (<i>B</i>) Whole-mount β-galactosidase staining of a TCF/Lef-lacZ transgenic uterus injected with Wnt7a-conditioned media in both uterine horns. (<i>C</i>) Whole-mount β-galactosidase staining of a TCF/Lef-lacZ transgenic uterus injected with Wnt7a-conditioned media only in the right uterine horn. (<i>D</i>) A Cytodex-3 bead (bd) is approximately the same size as a wild-type blastocyst (bl). (<i>E</i>) A Cytodex-3 bead coated with Wnt7a-expressing cells. (<i>F</i>) Whole-mount β-galactosidase staining of a TCF/Lef-lacZ transgenic uterus injected with Wnt5a-coated Cytodex-3 beads. (<i>G</i>) Whole-mount β-galactosidase staining of a TCF/Lef-lacZ transgenic uterus injected with Wnt7a-coated Cytodex-3 beads.</div></p></div><div class="aux"><div class="resc">Othman A. Mohamed, et al. Proc Natl Acad Sci U S A. 2005 June 14;102(24):8579-8584.</div><p class="links"><a class="dblinks" href="/pubmed/15930138" ref="ordinalpos=4471&amp;ncbi_uid=502590&amp;link_uid=15930138">Citation</a><a class="dblinks" href="/pmc/articles/PMC1150820/" ref="ordinalpos=4471&amp;ncbi_uid=502590&amp;link_uid=PMC1150820">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4472.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193400/figure/fig3/" ref="ordinalpos=4472&amp;ncbi_uid=3770388&amp;link_uid=PMC3193400" href="/pmc/articles/PMC3193400/"><img src="/pmc/articles/instance/3193400/bin/cr2010178f3.gif" src-large="/pmc/articles/instance/3193400/bin/cr2010178f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193400/" ref="ordinalpos=4472&amp;ncbi_uid=3770388&amp;link_uid=PMC3193400" image-link="/pmc/articles/PMC3193400/figure/fig3/" class="imagepopup">Figure 3.  From: Crosstalk of reactive oxygen species and NF-?B <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Crosstalk of ROS with NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. ROS interacts with NF-κB at various places within the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Many of these interactions occur in a cell type-specific manner. ROS has been proposed to both activate and inactivate the IKK complex leading to an effect on the downstream targets. Often ROS has been shown to activate NF-κB through alternative IκBα phosphorylation, which may or may not result in the degradation of IκBα. Lastly, ROS may influence the DNA binding properties of the NF-κB proteins themselves. Oxidation of p50 on its DNA binding domain has been shown to prevent its DNA binding, and must be reversed in the nucleus by a Trx1-dependent process involving Ref-1. On the other hand, the phosphorylation of RelA that is influenced by ROS-dependent processes leads to greater NF-κB activation.</div></p></div><div class="aux"><div class="resc">Michael J Morgan, et al. Cell Res. 2011 January;21(1):103-115.</div><p class="links"><a class="dblinks" href="/pubmed/21187859" ref="ordinalpos=4472&amp;ncbi_uid=3770388&amp;link_uid=21187859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193400/" ref="ordinalpos=4472&amp;ncbi_uid=3770388&amp;link_uid=PMC3193400">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4473.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3833016/figure/F1/" ref="ordinalpos=4473&amp;ncbi_uid=5563837&amp;link_uid=PMC3833016" href="/pmc/articles/PMC3833016/"><img src="/pmc/articles/instance/3833016/bin/fpls-04-00451-g001.gif" src-large="/pmc/articles/instance/3833016/bin/fpls-04-00451-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3833016/" ref="ordinalpos=4473&amp;ncbi_uid=5563837&amp;link_uid=PMC3833016" image-link="/pmc/articles/PMC3833016/figure/F1/" class="imagepopup">FIGURE 1.  From: Cytokinin cross-talking during biotic and abiotic stress responses. </a></p><div class="supp"><p class="details"><div><b>CK and crosstalks during abiotic stress responses.</b> Under non-stress conditions, CK activates <span class="highlight" style="background-color:">signaling</span> mediated through AHK receptors, AHPs, and type-B response regulators ARRs. Type-B ARRs stimulate the expression of the early CK response genes, including type-A <i>ARR</i> genes that provide a negative feedback loop of the CK <span class="highlight" style="background-color:">signaling</span>. Besides this negative feedback loop, type-A ARRs also repress the expression of <i>ABI5</i> and interfere with the ABA <span class="highlight" style="background-color:">signaling</span>, through the physical interaction with ABI5. In response to stress, ABA levels increase and, simultaneously, CK levels decrease. The recognition of ABA by the receptors PYR/PYL/RCAR promotes the interaction with PP2C proteins that will activate downstream responses through <span class="highlight" style="background-color:">signaling</span> components including ABI5 and ABI4. At the same time, ABA interferes with the activity of CK and auxin and via ABI4 attenuates the expression of the <i>PIN1</i> auxin efflux carrier and enhances the transcription of the CK <span class="highlight" style="background-color:">signaling</span> repressor <i>ARR5</i>. Interestingly, type-A ARRs, such as ARR5, are upregulated, despite the low CK levels, probably because of the indirect activation of the CK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by alternative receptors of the histidine kinase family, such as AHK1.</div></p></div><div class="aux"><div class="resc">José A. O’Brien, et al. Front Plant Sci. 2013;4:451.</div><p class="links"><a class="dblinks" href="/pubmed/24312105" ref="ordinalpos=4473&amp;ncbi_uid=5563837&amp;link_uid=24312105">Citation</a><a class="dblinks" href="/pmc/articles/PMC3833016/" ref="ordinalpos=4473&amp;ncbi_uid=5563837&amp;link_uid=PMC3833016">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4474.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4234565/figure/fig03/" ref="ordinalpos=4474&amp;ncbi_uid=6889055&amp;link_uid=PMC4234565" href="/pmc/articles/PMC4234565/"><img src="/pmc/articles/instance/4234565/bin/pnas.1408886111fig03.gif" src-large="/pmc/articles/instance/4234565/bin/pnas.1408886111fig03.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4234565/" ref="ordinalpos=4474&amp;ncbi_uid=6889055&amp;link_uid=PMC4234565" image-link="/pmc/articles/PMC4234565/figure/fig03/" class="imagepopup">Fig. 3. From: A circadian gene expression atlas in mammals: Implications for biology and medicine. </a></p><div class="supp"><p class="details"><div>Exploring pathways across biological space and time. (<i>A</i>) Expression of the deltex gene <i>Dtx4</i> in all organs superimposed. (<i>B</i>) Example of <span class="highlight" style="background-color:">pathway</span> components’ timing reflecting function: expression profiles from the heart, for <i>Vegfa</i> and its two receptors <i>Kdr</i> and <i>Flt1</i>. Black arrows highlight times at which <i>Flt1</i> and <i>Kdr</i> are anti-phased. (<i>C</i>) Example of systemic <span class="highlight" style="background-color:">pathway</span> orchestration segregating in time and space: expression profile of <i>Igf1</i> in the liver, compared with its downstream target <i>Pik3</i> in several organs. (<i>D</i>) Example of widespread <span class="highlight" style="background-color:">pathway</span> component synchronization within the same space (organ): expression profiles from the kidney for multiple <span class="highlight" style="background-color:">signaling</span> receptors that activate the PIK3-AKT-MTOR <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ray Zhang, et al. Proc Natl Acad Sci U S A. 2014 November 11;111(45):16219-16224.</div><p class="links"><a class="dblinks" href="/pubmed/25349387" ref="ordinalpos=4474&amp;ncbi_uid=6889055&amp;link_uid=25349387">Citation</a><a class="dblinks" href="/pmc/articles/PMC4234565/" ref="ordinalpos=4474&amp;ncbi_uid=6889055&amp;link_uid=PMC4234565">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4475.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2727076/figure/fig6/" ref="ordinalpos=4475&amp;ncbi_uid=1944030&amp;link_uid=PMC2727076" href="/pmc/articles/PMC2727076/"><img src="/pmc/articles/instance/2727076/bin/JEB024646F6.gif" src-large="/pmc/articles/instance/2727076/bin/JEB024646F6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2727076/" ref="ordinalpos=4475&amp;ncbi_uid=1944030&amp;link_uid=PMC2727076" image-link="/pmc/articles/PMC2727076/figure/fig6/" class="imagepopup">Fig. 6. From: <span class="highlight" style="background-color:">Signaling</span> to the apical membrane and to the paracellular <span class="highlight" style="background-color:">pathway</span>: changes  in the cytosolic proteome of Aedes Malpighian tubules. </a></p><div class="supp"><p class="details"><div>The paracellular <span class="highlight" style="background-color:">pathway</span> in Malpighian tubules of <i>Aedes aegypti</i>.  (A) The paracellular <span class="highlight" style="background-color:">pathway</span> between two principal cells and part of a  stellate cell (near the apical brush border). Mitochondria in microvilli of  the brush border are unique to principal cells of the tubule. Microvilli of  the brush border of stellate cells are devoid of mitochondria. (B,C) The  paracellular <span class="highlight" style="background-color:">pathway</span> between a stellate cell and principal cell is occupied  largely by a septate junction. Septa give rise to the ladder-like structure of  the junction. Note that the septate junction can extend from apical to basal  poles of epithelial cells. Abbreviations: bb, brush border; bmi, basolateral  membrane infoldings; mt, microtubule; pc, principal cell; sc, stellate cell.  White arrows point to the paracellular <span class="highlight" style="background-color:">pathway</span> in B and to septa in C.</div></p></div><div class="aux"><div class="resc">Klaus W. Beyenbach, et al. J Exp Biol. 2009 February 1;212(3):329-340.</div><p class="links"><a class="dblinks" href="/pubmed/19151207" ref="ordinalpos=4475&amp;ncbi_uid=1944030&amp;link_uid=19151207">Citation</a><a class="dblinks" href="/pmc/articles/PMC2727076/" ref="ordinalpos=4475&amp;ncbi_uid=1944030&amp;link_uid=PMC2727076">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4476.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2613047/figure/F5/" ref="ordinalpos=4476&amp;ncbi_uid=1649381&amp;link_uid=PMC2613047" href="/pmc/articles/PMC2613047/"><img src="/pmc/articles/instance/2613047/bin/zee0120844530005.gif" src-large="/pmc/articles/instance/2613047/bin/zee0120844530005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2613047/" ref="ordinalpos=4476&amp;ncbi_uid=1649381&amp;link_uid=PMC2613047" image-link="/pmc/articles/PMC2613047/figure/F5/" class="imagepopup">Figure 5.  From: Genes Associated with Membrane-Initiated <span class="highlight" style="background-color:">Signaling</span> of Estrogen and Energy Homeostasis. </a></p><div class="supp"><p class="details"><div>A, Classical ER membrane-associated and nuclear-associated <span class="highlight" style="background-color:">signaling</span>. B, <span class="highlight" style="background-color:">Signaling</span> pathways for the novel membrane ER (mER). 1) Estrogen can activate the classical ER-mediated (ERα/β) pathways leading to changes in mRNA transcription of relevant genes via ERE. 2) Both ERs are regulated by STX, and may be a mechanism for the interaction of classical ER and mER <span class="highlight" style="background-color:">signaling</span>. 3) Estrogen can bind to membrane-associated classical ER leading to the activation of signal transduction pathways to control gene transcription and cellular functions. The putative mER is a G protein-coupled receptor (GPCR) linked to G<sub>q/11</sub> protein that activates PLC-PKC-PKA <span class="highlight" style="background-color:">pathway</span> may control gene transcription via pCREB-CRE. The activation of PKC can also lead to changes in gene transcription via the phosphorylation of other <span class="highlight" style="background-color:">signaling</span> proteins (MAPK/ERK). 4) The hydrolysis of phosphatidylinositol (4,5)-bisphosphate (PIP<sub>2</sub>) into 1,4,5-trisphosphate (IP3) may also initiate calcium-dependent <span class="highlight" style="background-color:">signaling</span> molecules (CaM/CaMK) to control transcription. Several of these genes are associated with cation and anion channels (Ca<sub>v</sub>3.1, GLR) or proteins that impinge on channel functions (AKAP). 5) Changes in neuronal excitability either by channel expression or channel modulation may also control gene expression. A few of the genes analyzed by qPCR are involved in calcium <span class="highlight" style="background-color:">signaling</span> (CaMK II) and protein trafficking (GEC1).</div></p></div><div class="aux"><div class="resc">T. A. Roepke, et al. Endocrinology. 2008 December;149(12):6113-6124.</div><p class="links"><a class="dblinks" href="/pubmed/18755790" ref="ordinalpos=4476&amp;ncbi_uid=1649381&amp;link_uid=18755790">Citation</a><a class="dblinks" href="/pmc/articles/PMC2613047/" ref="ordinalpos=4476&amp;ncbi_uid=1649381&amp;link_uid=PMC2613047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4477.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4211408/figure/F6/" ref="ordinalpos=4477&amp;ncbi_uid=6822517&amp;link_uid=PMC4211408" href="/pmc/articles/PMC4211408/"><img src="/pmc/articles/instance/4211408/bin/fncel-08-00343-g006.gif" src-large="/pmc/articles/instance/4211408/bin/fncel-08-00343-g006.jpg" alt="FIGURE 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4211408/" ref="ordinalpos=4477&amp;ncbi_uid=6822517&amp;link_uid=PMC4211408" image-link="/pmc/articles/PMC4211408/figure/F6/" class="imagepopup">FIGURE 6.  From: Nitric oxide from inflammatory origin impairs neural stem cell proliferation by inhibiting epidermal growth factor receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>High levels of NO decrease the activation of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> High levels of NOC-18 (100 μM) decreased the phosphorylation of ERK1/2 in SVZ cell lysates. MnTBAP prevented the decrease of ERK1/2 phosphorylation. One-way ANOVA (Bonferroni’s post-test). *<i>p</i> &lt; 0.05, significantly different from control. <b>(B)</b> MnTBAP (100 μM) prevented the antiproliferative effect of NOC-18 (100 μM) in SVZ cell cultures, as determined by BrdU incorporation. One-way ANOVA (Bonferroni’s post-test). *<i>p</i> &lt; 0.05, significantly different from control. <sup>##</sup><i>p</i> &lt; 0.01, significantly different from NOC-18.</div></p></div><div class="aux"><div class="resc">Bruno P. Carreira, et al. Front Cell Neurosci. 2014;8:343.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4477&amp;ncbi_uid=6822517&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4211408/" ref="ordinalpos=4477&amp;ncbi_uid=6822517&amp;link_uid=PMC4211408">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4478.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1189324/figure/fig3/" ref="ordinalpos=4478&amp;ncbi_uid=586439&amp;link_uid=PMC1189324" href="/pmc/articles/PMC1189324/"><img src="/pmc/articles/instance/1189324/bin/zpq0310590400003.gif" src-large="/pmc/articles/instance/1189324/bin/zpq0310590400003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1189324/" ref="ordinalpos=4478&amp;ncbi_uid=586439&amp;link_uid=PMC1189324" image-link="/pmc/articles/PMC1189324/figure/fig3/" class="imagepopup">Fig. 3. From: Synergistic induction of tissue factor by coagulation factor Xa and TNF: Evidence for involvement of negative regulatory <span class="highlight" style="background-color:">signaling</span> cascades. </a></p><div class="supp"><p class="details"><div>Effect of fXa on TNF-induced JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Time course of JNK and c-Jun phosphorylation as detected by immunoblotting with anti-<i>p</i>-JNK and anti-<i>p</i>-c-Jun, respectively (<i>Top</i>) and the control antibodies anti-JNK and anti-c-Jun, respectively (<i>Bottom</i>). Cells were incubated with fXa (10 nM), TNF (5 ng/ml), or both for the indicated times. Representative data from three independent experiments are reported.</div></p></div><div class="aux"><div class="resc">Ayala Hezi-Yamit, et al. Proc Natl Acad Sci U S A. 2005 August 23;102(34):12077-12082.</div><p class="links"><a class="dblinks" href="/pubmed/16105945" ref="ordinalpos=4478&amp;ncbi_uid=586439&amp;link_uid=16105945">Citation</a><a class="dblinks" href="/pmc/articles/PMC1189324/" ref="ordinalpos=4478&amp;ncbi_uid=586439&amp;link_uid=PMC1189324">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4479.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151875/figure/F1/" ref="ordinalpos=4479&amp;ncbi_uid=259096&amp;link_uid=PMC151875" href="/pmc/articles/PMC151875/"><img src="/pmc/articles/instance/151875/bin/JCI0317745.f1.gif" src-large="/pmc/articles/instance/151875/bin/JCI0317745.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151875/" ref="ordinalpos=4479&amp;ncbi_uid=259096&amp;link_uid=PMC151875" image-link="/pmc/articles/PMC151875/figure/F1/" class="imagepopup">Figure 1.  From: DAP12 <span class="highlight" style="background-color:">signaling</span>: from immune cells to bone modeling and brain myelination. </a></p><div class="supp"><p class="details"><div>Hypothetical role of DAP12 in myeloid cell and oligodendrocyte development. In homeostatic conditions, the TREM-2 receptor may trigger the DAP12 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in immature oligodendrocytes and osteoclasts, promoting their differentiation into mature cells capable of myelination and bone resorption, respectively. During inflammatory conditions, DAP12 may be solicited by different receptors, leading to differentiation and/or activation of microglial cells, macrophages, and DCs. While oligodendrocytes are thought to derive from a neuroepithelial progenitor, it is possible that DAP12-expressing oligodendrocytes may be of myeloid origin.</div></p></div><div class="aux"><div class="resc">Marco Colonna. J Clin Invest. 2003 February 1;111(3):313-314.</div><p class="links"><a class="dblinks" href="/pubmed/12569153" ref="ordinalpos=4479&amp;ncbi_uid=259096&amp;link_uid=12569153">Citation</a><a class="dblinks" href="/pmc/articles/PMC151875/" ref="ordinalpos=4479&amp;ncbi_uid=259096&amp;link_uid=PMC151875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4480.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2156132/figure/F6/" ref="ordinalpos=4480&amp;ncbi_uid=1253222&amp;link_uid=PMC2156132" href="/pmc/articles/PMC2156132/"><img src="/pmc/articles/instance/2156132/bin/JCB9812039.f6.gif" src-large="/pmc/articles/instance/2156132/bin/JCB9812039.f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2156132/" ref="ordinalpos=4480&amp;ncbi_uid=1253222&amp;link_uid=PMC2156132" image-link="/pmc/articles/PMC2156132/figure/F6/" class="imagepopup">Figure 6.  From: Insulin-like Growth Factor II <span class="highlight" style="background-color:">Signaling</span> in Neoplastic Proliferation Is Blocked by Transgenic Expression of the Metalloproteinase Inhibitor Timp-1. </a></p><div class="supp"><p class="details"><div>The molecular mechanism underlying TIMP-1–mediated suppression of hepatocarcinogenesis. The schematic highlights the molecular events that were examined here to uncover the mechanism of TIMP-1–mediated inhibition of hepatocyte proliferation. The primary event in the <span class="highlight" style="background-color:">pathway</span> between TAg-induced IGF-II reactivation and hepatocyte proliferation affected by TIMP-1 overexpression was an increase in IGFBP-3 levels, due to the inhibition of MMP-mediated proteolysis of IGFBP-3 by TIMP-1. The direct result of increased IGFBP-3 levels was decreased dissociable IGF-II levels concomitant with decreased <span class="highlight" style="background-color:">signaling</span> through the IGF-IR and decreased hepatocyte proliferation.</div></p></div><div class="aux"><div class="resc">David C. Martin, et al. J Cell Biol. 1999 August 23;146(4):881-892.</div><p class="links"><a class="dblinks" href="/pubmed/10459021" ref="ordinalpos=4480&amp;ncbi_uid=1253222&amp;link_uid=10459021">Citation</a><a class="dblinks" href="/pmc/articles/PMC2156132/" ref="ordinalpos=4480&amp;ncbi_uid=1253222&amp;link_uid=PMC2156132">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4481.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3727952/figure/F1/" ref="ordinalpos=4481&amp;ncbi_uid=5157210&amp;link_uid=PMC3727952" href="/pmc/articles/PMC3727952/"><img src="/pmc/articles/instance/3727952/bin/1471-2164-14-503-1.gif" src-large="/pmc/articles/instance/3727952/bin/1471-2164-14-503-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3727952/" ref="ordinalpos=4481&amp;ncbi_uid=5157210&amp;link_uid=PMC3727952" image-link="/pmc/articles/PMC3727952/figure/F1/" class="imagepopup">Figure 1.  From: Identification of G protein-coupled receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins in marine diatoms using comparative genomics. </a></p><div class="supp"><p class="details"><div><b>Data analysis framework for investigating the G protein-coupled receptor (GPCR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in diatoms.</b> The diatom genomes were first TBLASTN searched against the human gpDB database to identify potential GPCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins (black). Selected conserved protein domains for those GPCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins that had no BLAST similarity to the diatoms were extracted from the Pfam v.26.0 database and HMMER searched against the translated diatom genomes (purple). Diatom genomic sequences with matches to the human gpDB or GPCR <span class="highlight" style="background-color:">signaling</span> protein Pfam were then searched against the diatom EST libraries. The GPCRDB was TBLASTN searched against the diatom genomes to identify putative diatom GPCRs (red). GPCRs were also identified by downloading sequence alignments for the GPCR families from the GPCRDB (classes A, B and C) to use as seed alignments for HMM searches against a custom microeukaryote database (red). Identified GPCRs were then further characterized using transmembrane domain (TMD) region and conserved domain analyses. Diatom GPCRs were also searched against the respective diatom EST libraries. A seed alignment was generated using the TMD regions of the putative diatom GPCRs and converted to an HMM profile to recruit related sequences from the custom microeukaryote database and GenBank. Phylogenetic analysis was then performed. BLAST, basic local alignment search tool; EST, expressed sequence tag; GPCRDB, G protein-coupled receptor database; HMM, hidden markov model; HMMTOP, Hidden Markov Model for Topology Prediction; TBLASTN, protein query versus translated nucleotide BLAST; TMHMM, transmembrane hidden markov model.</div></p></div><div class="aux"><div class="resc">Jesse A Port, et al. BMC Genomics. 2013;14:503-503.</div><p class="links"><a class="dblinks" href="/pubmed/23883327" ref="ordinalpos=4481&amp;ncbi_uid=5157210&amp;link_uid=23883327">Citation</a><a class="dblinks" href="/pmc/articles/PMC3727952/" ref="ordinalpos=4481&amp;ncbi_uid=5157210&amp;link_uid=PMC3727952">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4482.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3626246/figure/fig3/" ref="ordinalpos=4482&amp;ncbi_uid=4816839&amp;link_uid=PMC3626246" href="/pmc/articles/PMC3626246/"><img src="/pmc/articles/instance/3626246/bin/IJE2013-368970.003.gif" src-large="/pmc/articles/instance/3626246/bin/IJE2013-368970.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3626246/" ref="ordinalpos=4482&amp;ncbi_uid=4816839&amp;link_uid=PMC3626246" image-link="/pmc/articles/PMC3626246/figure/fig3/" class="imagepopup">Figure 3.  From: Omentin-1 Stimulates Human Osteoblast Proliferation through PI3K/Akt Signal <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Omentin-1 regulated proliferation of hOB through the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The hOBs were pretreated with vehicle, PI3K inhibitor LY294002 (10 <i>μ</i>M), or Akt inhibitor HIMO (10 <i>μ</i>M) for 3 h prior to treatment with omentin-1 (200 ng/mL) for 48 h. The cells were also transfected with control siRNA or Akt siRNA before treatment with omentin-1 (200 ng/mL) for 48 h. Cell proliferation was determined by measuring [<sup>3</sup>H]thymidine incorporation. Results are expressed as counts per minute. *<i>P</i> &lt; 0.05 versus control.</div></p></div><div class="aux"><div class="resc">Shan-Shan Wu, et al. Int J Endocrinol. 2013;2013:368970.</div><p class="links"><a class="dblinks" href="/pubmed/23606838" ref="ordinalpos=4482&amp;ncbi_uid=4816839&amp;link_uid=23606838">Citation</a><a class="dblinks" href="/pmc/articles/PMC3626246/" ref="ordinalpos=4482&amp;ncbi_uid=4816839&amp;link_uid=PMC3626246">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4483.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC25225/figure/F10/" ref="ordinalpos=4483&amp;ncbi_uid=176062&amp;link_uid=PMC25225" href="/pmc/articles/PMC25225/"><img src="/pmc/articles/instance/25225/bin/mk0490824010.gif" src-large="/pmc/articles/instance/25225/bin/mk0490824010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC25225/" ref="ordinalpos=4483&amp;ncbi_uid=176062&amp;link_uid=PMC25225" image-link="/pmc/articles/PMC25225/figure/F10/" class="imagepopup">Figure 10.  From: Regulation of Ribosome Biogenesis by the Rapamycin-sensitive TOR-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>Model for involvement of the TOR <span class="highlight" style="background-color:">pathway</span> in control of protein synthesis as well as transcription of genes encoding ribosomal components (adapted from Thomas and Hall, 1997). Broken arrows indicate that direct interactions between components have not been demonstrated. See text for details. Not depicted in the figure are Bmh1 and Bmh2, the yeast homologues of 14-3-3 proteins, which have been shown recently to be involved in TOR <span class="highlight" style="background-color:">signaling</span> (Bertram et al., 1998).</div></p></div><div class="aux"><div class="resc">Ted Powers, et al. Mol Biol Cell. 1999 April;10(4):987-1000.</div><p class="links"><a class="dblinks" href="/pubmed/10198052" ref="ordinalpos=4483&amp;ncbi_uid=176062&amp;link_uid=10198052">Citation</a><a class="dblinks" href="/pmc/articles/PMC25225/" ref="ordinalpos=4483&amp;ncbi_uid=176062&amp;link_uid=PMC25225">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4484.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4168223/figure/Fig1/" ref="ordinalpos=4484&amp;ncbi_uid=6648190&amp;link_uid=PMC4168223" href="/pmc/articles/PMC4168223/"><img src="/pmc/articles/instance/4168223/bin/726_2014_1790_Fig1_HTML.gif" src-large="/pmc/articles/instance/4168223/bin/726_2014_1790_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4168223/" ref="ordinalpos=4484&amp;ncbi_uid=6648190&amp;link_uid=PMC4168223" image-link="/pmc/articles/PMC4168223/figure/Fig1/" class="imagepopup">Fig. 1.  From: Kynurenic acid inhibits colon cancer proliferation in vitro: effects on <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Effect of KYNA on activation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HT-29 cells. Western blot analysis of phosphorylation status of selected proteins of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HT-29 cells after treatment with KYNA 1 mM for 5 min–48 h (<b>a</b>) and treatment with KYNA in the range of concentrations 0.01–5 mM for 4, 24 and 48 h or 24 and 48 h (<b>b</b>); (<i>C</i> control; not treated). Western blots shown in the figure were selected as the most representative of the series of repetitions <i>n</i> ≥ 3. The data were normalized relative to β-actin. The results of densitometric analysis are shown as % of control (the changes ≥20 % were considered as significant)</div></p></div><div class="aux"><div class="resc">Katarzyna Walczak, et al. Amino Acids. 2014;46(10):2393-2401.</div><p class="links"><a class="dblinks" href="/pubmed/25012123" ref="ordinalpos=4484&amp;ncbi_uid=6648190&amp;link_uid=25012123">Citation</a><a class="dblinks" href="/pmc/articles/PMC4168223/" ref="ordinalpos=4484&amp;ncbi_uid=6648190&amp;link_uid=PMC4168223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4485.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3425953/figure/F2/" ref="ordinalpos=4485&amp;ncbi_uid=4190612&amp;link_uid=PMC3425953" href="/pmc/articles/PMC3425953/"><img src="/pmc/articles/instance/3425953/bin/nihms398364f2.gif" src-large="/pmc/articles/instance/3425953/bin/nihms398364f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3425953/" ref="ordinalpos=4485&amp;ncbi_uid=4190612&amp;link_uid=PMC3425953" image-link="/pmc/articles/PMC3425953/figure/F2/" class="imagepopup">Figure 2.  From: High-Throughput Screening Strategies for Targeted Identification Small-Molecule Compounds Towards Activated Pathways in Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>The activity of the EGFR/KRAS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal and tumor cells. Left panel represents the activity of the <span class="highlight" style="background-color:">pathway</span> in the normal intestinal epithelial cells. Right panel shows three models of hyperactivation of EGRF/KRAS <span class="highlight" style="background-color:">pathway</span> in the colorectal cancer cells caused by EGFR copy number increased, KRAS and BRAF mutations, respectively. Red star activation; dashed lines – irrelevant signal.</div></p></div><div class="aux"><div class="resc">Agnieszka B. Bialkowska, et al. Future Oncol. ;8(3):259-272.</div><p class="links"><a class="dblinks" href="/pubmed/22409463" ref="ordinalpos=4485&amp;ncbi_uid=4190612&amp;link_uid=22409463">Citation</a><a class="dblinks" href="/pmc/articles/PMC3425953/" ref="ordinalpos=4485&amp;ncbi_uid=4190612&amp;link_uid=PMC3425953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4486.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3156782/figure/f2/" ref="ordinalpos=4486&amp;ncbi_uid=3162645&amp;link_uid=PMC3156782" href="/pmc/articles/PMC3156782/"><img src="/pmc/articles/instance/3156782/bin/mv-v17-2102-f2.gif" src-large="/pmc/articles/instance/3156782/bin/mv-v17-2102-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3156782/" ref="ordinalpos=4486&amp;ncbi_uid=3162645&amp;link_uid=PMC3156782" image-link="/pmc/articles/PMC3156782/figure/f2/" class="imagepopup">Figure 2.  From: A comparison of stem cell-related gene expression in the progenitor-rich limbal epithelium and the differentiating central corneal epithelium. </a></p><div class="supp"><p class="details"><div>Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> generated by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA). The IPA depicted the genes involved, their interactions, and the cellular and metabolic reactions that constituted the <span class="highlight" style="background-color:">pathway</span>. Colored molecules represented genes that appeared in the data set studied. Red and green molecules were up- and down-regulated, respectively, in limbal epithelial cells. Gray molecules did not meet the user defined cutoff of 2.</div></p></div><div class="aux"><div class="resc">Teresa Nieto-Miguel, et al. Mol Vis. 2011;17:2102-2117.</div><p class="links"><a class="dblinks" href="/pubmed/21850186" ref="ordinalpos=4486&amp;ncbi_uid=3162645&amp;link_uid=21850186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3156782/" ref="ordinalpos=4486&amp;ncbi_uid=3162645&amp;link_uid=PMC3156782">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4487.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2762706/figure/F3/" ref="ordinalpos=4487&amp;ncbi_uid=2085272&amp;link_uid=PMC2762706" href="/pmc/articles/PMC2762706/"><img src="/pmc/articles/instance/2762706/bin/nihms129225f3.gif" src-large="/pmc/articles/instance/2762706/bin/nihms129225f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2762706/" ref="ordinalpos=4487&amp;ncbi_uid=2085272&amp;link_uid=PMC2762706" image-link="/pmc/articles/PMC2762706/figure/F3/" class="imagepopup">Figure 3.  From: Increased vascular permeability after CABG in diabetic patients is associated with increased expression of VEGF and HGF. </a></p><div class="supp"><p class="details"><div>Hypoxia <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in DM patients 4d post-CPB/C. Genes colored in dark gray indicate up-regulated genes and in light gray represent down- regulated genes in DM patients. None of the genes in this <span class="highlight" style="background-color:">pathway</span> were up- or down- regulated in ND patients at 4d post CPB/C. Data was analyzed through the use of Ingenuity <span class="highlight" style="background-color:">Pathway</span> analysis (Ingenuity Systems, www.ingenuity.com). DM: patients with diabetes mellitus; ND: patients without diabetes mellitus.</div></p></div><div class="aux"><div class="resc">Sirisha Emani, et al. J Thorac Cardiovasc Surg. ;138(1):185-191.</div><p class="links"><a class="dblinks" href="/pubmed/19577077" ref="ordinalpos=4487&amp;ncbi_uid=2085272&amp;link_uid=19577077">Citation</a><a class="dblinks" href="/pmc/articles/PMC2762706/" ref="ordinalpos=4487&amp;ncbi_uid=2085272&amp;link_uid=PMC2762706">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4488.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1356334/figure/f7/" ref="ordinalpos=4488&amp;ncbi_uid=585790&amp;link_uid=PMC1356334" href="/pmc/articles/PMC1356334/"><img src="/pmc/articles/instance/1356334/bin/7600893f7.gif" src-large="/pmc/articles/instance/1356334/bin/7600893f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1356334/" ref="ordinalpos=4488&amp;ncbi_uid=585790&amp;link_uid=PMC1356334" image-link="/pmc/articles/PMC1356334/figure/f7/" class="imagepopup">Figure 7.  From: NGF rapidly increases membrane expression of TRPV1 heat-gated ion channels. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Alignment of sequences adjacent to Y200 for human TRPV1–4. The bottom line shows conserved residues (<sup>*</sup>), which include the TRPV1 Y200 site (yellow bar). (<b>B</b>) Schematic diagram of the <span class="highlight" style="background-color:">signaling</span> pathways important in sensitization of TRPV1 by TrkA. The functionally most significant <span class="highlight" style="background-color:">pathway</span> is shown at the left (yellow, solid arrows). A smaller component of sensitization following exposure to NGF is mediated by phosphorylation of TRPV1 at residues S502 and S801, probably by the PLCγ/PKCɛ <span class="highlight" style="background-color:">pathway</span> (green, dashed arrows). PKCɛ is a crucial intermediate in sensitization of TRPV1 by bradykinin (<span class="highlight" style="background-color:">pathway</span> shown at the lower right of the diagram).</div></p></div><div class="aux"><div class="resc">Xuming Zhang, et al. EMBO J. 2005 December 21;24(24):4211-4223.</div><p class="links"><a class="dblinks" href="/pubmed/16319926" ref="ordinalpos=4488&amp;ncbi_uid=585790&amp;link_uid=16319926">Citation</a><a class="dblinks" href="/pmc/articles/PMC1356334/" ref="ordinalpos=4488&amp;ncbi_uid=585790&amp;link_uid=PMC1356334">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4489.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3176436/figure/F1/" ref="ordinalpos=4489&amp;ncbi_uid=3519022&amp;link_uid=PMC3176436" href="/pmc/articles/PMC3176436/"><img src="/pmc/articles/instance/3176436/bin/nihms318427f1.gif" src-large="/pmc/articles/instance/3176436/bin/nihms318427f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3176436/" ref="ordinalpos=4489&amamp;ncbi_uid=3519022&amp;link_uid=PMC3176436" image-link="/pmc/articles/PMC3176436/figure/F1/" class="imagepopup">Figure 1.  From: The cAMP-dependent protein kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is a key regulator of P-body foci formation. </a></p><div class="supp"><p class="details"><div>P-body formation was regulated by Ras/PKA <span class="highlight" style="background-color:">signaling</span> activity.<br />(A) Elevated <span class="highlight" style="background-color:">signaling</span> through the Ras/PKA <span class="highlight" style="background-color:">pathway</span> inhibited P-body formation in response to an acute glucose starvation. Strains expressing GFP-tagged versions of Dcp2, Dhh1, Pat1 and Xrn1 were transformed with either a vector control or a <i>MET3-RAS2<sup>val19</sup></i> plasmid. Cells were grown to mid-log phase in SC-glucose medium and expression from the <i>RAS2<sup>val19</sup></i> locus was induced by transferring cells to a medium lacking methionine for 2 hr. The cells were then starved for glucose for 15 min and visualized by fluorescence microscopy.<br />(B) The formation of P-bodies in early stationary phase was inhibited by elevated Ras/PKA <span class="highlight" style="background-color:">signaling</span>. Wild-type yeast with either a control vector or a <i>MET3-RAS2<sup>val19</sup></i> plasmid were grown for 24 hr in an SC-glucose medium lacking methionine.<br />(C, D) The down-regulation of PKA activity was sufficient to induce P-body formation in glucose-replete conditions. In <i>C</i>, cells containing an analog-sensitive allele of <i>TPK1, tpk1-as</i>, were treated with 5 μM 1NM-PP1 for 15 min. In <i>D</i>, the cells were transformed with either a control vector or a high-copy <i>PDE2</i> plasmid.</div></p></div><div class="aux"><div class="resc">Vidhya Ramachandran, et al. Mol Cell. ;43(6):973-981.</div><p class="links"><a class="dblinks" href="/pubmed/21925385" ref="ordinalpos=4489&amp;ncbi_uid=3519022&amp;link_uid=21925385">Citation</a><a class="dblinks" href="/pmc/articles/PMC3176436/" ref="ordinalpos=4489&amp;ncbi_uid=3519022&amp;link_uid=PMC3176436">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4490.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3799783/figure/pone-0076166-g002/" ref="ordinalpos=4490&amp;ncbi_uid=5469836&amp;link_uid=PMC3799783" href="/pmc/articles/PMC3799783/"><img src="/pmc/articles/instance/3799783/bin/pone.0076166.g002.gif" src-large="/pmc/articles/instance/3799783/bin/pone.0076166.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3799783/" ref="ordinalpos=4490&amp;ncbi_uid=5469836&amp;link_uid=PMC3799783" image-link="/pmc/articles/PMC3799783/figure/pone-0076166-g002/" class="imagepopup">Figure 2. The connection relationships among 5 DEGs in the PPAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Analysis on Differential Gene Expression Data for Prediction of New Biological Features in Permanent Atrial Fibrillation. </a></p><div class="supp"><p class="details"><div>A. The connection relationships in pmAF. B. The connection relationships in controls. The threshold of CC is 0.9.</div></p></div><div class="aux"><div class="resc">Feng Ou, et al. PLoS One. 2013;8(10):e76166.</div><p class="links"><a class="dblinks" href="/pubmed/24204599" ref="ordinalpos=4490&amp;ncbi_uid=5469836&amp;link_uid=24204599">Citation</a><a class="dblinks" href="/pmc/articles/PMC3799783/" ref="ordinalpos=4490&amp;ncbi_uid=5469836&amp;link_uid=PMC3799783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4491.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3493915/figure/F7/" ref="ordinalpos=4491&amp;ncbi_uid=5463001&amp;link_uid=PMC3493915" href="/pmc/articles/PMC3493915/"><img src="/pmc/articles/instance/3493915/bin/zbc0471229380007.gif" src-large="/pmc/articles/instance/3493915/bin/zbc0471229380007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3493915/" ref="ordinalpos=4491&amp;ncbi_uid=5463001&amp;link_uid=PMC3493915" image-link="/pmc/articles/PMC3493915/figure/F7/" class="imagepopup">FIGURE 7. From: Matrix Metalloproteinase-13 (MMP-13) Directly and Indirectly Promotes Tumor Angiogenesis. </a></p><div class="supp"><p class="details"><div><b>A model of MMP-13-promoted angiogenesis.</b> MMP-13 is secreted from cancer cells. MMP-13 promotes angiogenesis through FAK and ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Moreover, MMP-13 enhances the secretion of VEGF-A from endothelial cells, fibroblasts or myofibroblasts. Secreted VEGF-A promotes angiogenesis.</div></p></div><div class="aux"><div class="resc">Yasusei Kudo, et al. J Biol Chem. 2012 November 9;287(46):38716-38728.</div><p class="links"><a class="dblinks" href="/pubmed/22992737" ref="ordinalpos=4491&amp;ncbi_uid=5463001&amp;link_uid=22992737">Citation</a><a class="dblinks" href="/pmc/articles/PMC3493915/" ref="ordinalpos=4491&amp;ncbi_uid=5463001&amp;link_uid=PMC3493915">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4492.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2876247/figure/fig2/" ref="ordinalpos=4492&amp;ncbi_uid=2241376&amp;link_uid=PMC2876247" href="/pmc/articles/PMC2876247/"><img src="/pmc/articles/instance/2876247/bin/MI2010-367838.002.gif" src-large="/pmc/articles/instance/2876247/bin/MI2010-367838.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2876247/" ref="ordinalpos=4492&amp;ncbi_uid=2241376&amp;link_uid=PMC2876247" image-link="/pmc/articles/PMC2876247/figure/fig2/" class="imagepopup">Figure 2.  From: Functional Food Targeting the Regulation of Obesity-Induced Inflammatory Responses and Pathologies. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of inflammatory gene expressions in obese adipose tissue and the strategy to prevent the obese-related pathologies by food components.</div></p></div><div class="aux"><div class="resc">Shizuka Hirai, et al. Mediators Inflamm. 2010;2010:367838.</div><p class="links"><a class="dblinks" href="/pubmed/20508825" ref="ordinalpos=4492&amp;ncbi_uid=2241376&amp;link_uid=20508825">Citation</a><a class="dblinks" href="/pmc/articles/PMC2876247/" ref="ordinalpos=4492&amp;ncbi_uid=2241376&amp;link_uid=PMC2876247">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4493.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2697172/figure/F4/" ref="ordinalpos=4493&amp;ncbi_uid=1831259&amp;link_uid=PMC2697172" href="/pmc/articles/PMC2697172/"><img src="/pmc/articles/instance/2697172/bin/1471-2164-10-220-4.gif" src-large="/pmc/articles/instance/2697172/bin/1471-2164-10-220-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2697172/" ref="ordinalpos=4493&amp;ncbi_uid=1831259&amp;link_uid=PMC2697172" image-link="/pmc/articles/PMC2697172/figure/F4/" class="imagepopup">Figure 4.  From: A network-based integrative approach to prioritize reliable hits from multiple genome-wide RNAi screens in Drosophila. </a></p><div class="supp"><p class="details"><div><b>The reproducibility rate of hits in the validation screen within each interval of the RR by NePhe score for Hh (a) and Wnt (b) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Li Wang, et al. BMC Genomics. 2009;10:220-220.</div><p class="links"><a class="dblinks" href="/pubmed/19435510" ref="ordinalpos=4493&amp;ncbi_uid=1831259&amp;link_uid=19435510">Citation</a><a class="dblinks" href="/pmc/articles/PMC2697172/" ref="ordinalpos=4493&amp;ncbi_uid=1831259&amp;link_uid=PMC2697172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4494.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2375051/figure/pone-0002215-g007/" ref="ordinalpos=4494&amp;ncbi_uid=1336148&amp;link_uid=PMC2375051" href="/pmc/articles/PMC2375051/"><img src="/pmc/articles/instance/2375051/bin/pone.0002215.g007.gif" src-large="/pmc/articles/instance/2375051/bin/pone.0002215.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2375051/" ref="ordinalpos=4494&amp;ncbi_uid=1336148&amp;link_uid=PMC2375051" image-link="/pmc/articles/PMC2375051/figure/pone-0002215-g007/" class="imagepopup">Figure 7. Involvement of MIF in HIF-1 regulation in MCF-7 cells..  From: Macrophage Migration Inhibitory Factor Activates Hypoxia-Inducible Factor in a p53-Dependent Manner. </a></p><div class="supp"><p class="details"><div>HIF-1α protein stabilization and transactivation activity induced by MIF is dependent on p53- and MAPK-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Seiko Oda, et al. PLoS ONE. 2008;3(5):e2215.</div><p class="links"><a class="dblinks" href="/pubmed/18493321" ref="ordinalpos=4494&amp;ncbi_uid=1336148&amp;link_uid=18493321">Citation</a><a class="dblinks" href="/pmc/articles/PMC2375051/" ref="ordinalpos=4494&amp;ncbi_uid=1336148&amp;link_uid=PMC2375051">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4495.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3129395/figure/F4/" ref="ordinalpos=4495&amp;ncbi_uid=3513566&amp;link_uid=PMC3129395" href="/pmc/articles/PMC3129395/"><img src="/pmc/articles/instance/3129395/bin/nihms286618f4.gif" src-large="/pmc/articles/instance/3129395/bin/nihms286618f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3129395/" ref="ordinalpos=4495&amp;ncbi_uid=3513566&amp;link_uid=PMC3129395" image-link="/pmc/articles/PMC3129395/figure/F4/" class="imagepopup">Figure 4.  From: Extranuclear <span class="highlight" style="background-color:">signaling</span> of mutated thyroid hormone receptors in promoting metastatic spread in thyroid carcinogenesis. </a></p><div class="supp"><p class="details"><div>Molecular model of extranuclear functions of PV in promoting thyroid carcinogenesis via the integrin-FAK-cSrc <span class="highlight" style="background-color:">signaling</span> cascade. PV complexes with integrin membrane receptors (e.g., α5β1, see Figure 1B) and actin cytoskeleton to activate <span class="highlight" style="background-color:">signaling</span> via intracellular key regulators, including cSrc and FAK. FAK undergoes autophosphorylation to provide binding motif for c-Src kinase. FAK is further phosphorylated by c-Src to enhance its activity and to recruit downstream adaptors, such as Grb2, for signal transduction. Activated c-Src via increasing phosphorylation subsequently activates the p38 MAPK <span class="highlight" style="background-color:">pathway</span>. Upon activation of both FAK-c-Src and p38 MAPK-ATF2 <span class="highlight" style="background-color:">signaling</span> pathways, the downstream affected genes, such as MMP-9, are therefore transactivated to increase expression and activity. The shown signal transduction <span class="highlight" style="background-color:">pathway</span>, initiated by PV via protein-protein interaction, promotes invasiveness and migration of thyroid cancer cells to undergo distant metastatic spread.</div></p></div><div class="aux"><div class="resc">Changxue Lu, et al. Steroids. ;76(9):885-891.</div><p class="links"><a class="dblinks" href="/pubmed/21473875" ref="ordinalpos=4495&amp;ncbi_uid=3513566&amp;link_uid=21473875">Citation</a><a class="dblinks" href="/pmc/articles/PMC3129395/" ref="ordinalpos=4495&amp;ncbi_uid=3513566&amp;link_uid=PMC3129395">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4496.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4142215/figure/F1/" ref="ordinalpos=4496&amp;ncbi_uid=6530059&amp;link_uid=PMC4142215" href="/pmc/articles/PMC4142215/"><img src="/pmc/articles/instance/4142215/bin/nihms484555f1.gif" src-large="/pmc/articles/instance/4142215/bin/nihms484555f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4142215/" ref="ordinalpos=4496&amp;ncbi_uid=6530059&amp;link_uid=PMC4142215" image-link="/pmc/articles/PMC4142215/figure/F1/" class="imagepopup">Figure 1. Annotation of KEGG ERBB2 <span class="highlight" style="background-color:">signaling</span> pathways with transcriptomic data, Part A SUM190, Part B SUM149. The <span class="highlight" style="background-color:">pathway</span> was derived from http://www.genome.jp/kegg/<span class="highlight" style="background-color:">pathway</span>/hsa/hsa04012.html in January 2012.  From: Genome wide proteomics of ERBB2 and EGFR and other oncogenic pathways in inflammatory breast cancer. </a></p><div class="supp"><p class="details"><div>The RPKM values are shown as follows. Green circle: RPKM value is larger than 15; yellow circle: RPKM value is between 3 and 15; red circle: RPKM value is between 1 and 3; blue cross: transcription value is under detection limit; blue line: potential preferred <span class="highlight" style="background-color:">signaling</span> based on transcript levels.</div></p></div><div class="aux"><div class="resc">Emma Yue Zhang, et al. J Proteome Res. ;12(6):2805-2817.</div><p class="links"><a class="dblinks" href="/pubmed/23647160" ref="ordinalpos=4496&amp;ncbi_uid=6530059&amp;link_uid=23647160">Citation</a><a class="dblinks" href="/pmc/articles/PMC4142215/" ref="ordinalpos=4496&amp;ncbi_uid=6530059&amp;link_uid=PMC4142215">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4497.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3570922/figure/fig1/" ref="ordinalpos=4497&amp;ncbi_uid=4650636&amp;link_uid=PMC3570922" href="/pmc/articles/PMC3570922/"><img src="/pmc/articles/instance/3570922/bin/ISRN.ENDOCRINOLOGY2013-472432.001.gif" src-large="/pmc/articles/instance/3570922/bin/ISRN.ENDOCRINOLOGY2013-472432.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3570922/" ref="ordinalpos=4497&amp;ncbi_uid=4650636&amp;link_uid=PMC3570922" image-link="/pmc/articles/PMC3570922/figure/fig1/" class="imagepopup">Figure 1.  From: Roles for PI3K/AKT/PTEN <span class="highlight" style="background-color:">Pathway</span> in Cell <span class="highlight" style="background-color:">Signaling</span> of Nonalcoholic Fatty Liver Disease. </a></p><div class="supp"><p class="details"><div>Schematic representation of PI3K/AKT/GSK3/mTOR <span class="highlight" style="background-color:">signaling</span>. Examples of molecules known to act on the regulatory pathways are shown. Note that some critical pathways have been omitted for clarity.</div></p></div><div class="aux"><div class="resc">Satoru Matsuda, et al. ISRN Endocrinol. 2013;2013:472432.</div><p class="links"><a class="dblinks" href="/pubmed/23431468" ref="ordinalpos=4497&amp;ncbi_uid=4650636&amp;link_uid=23431468">Citation</a><a class="dblinks" href="/pmc/articles/PMC3570922/" ref="ordinalpos=4497&amp;ncbi_uid=4650636&amp;link_uid=PMC3570922">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4498.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3290617/figure/pone-0032794-g001/" ref="ordinalpos=4498&amp;ncbi_uid=3838297&amp;link_uid=PMC3290617" href="/pmc/articles/PMC3290617/"><img src="/pmc/articles/instance/3290617/bin/pone.0032794.g001.gif" src-large="/pmc/articles/instance/3290617/bin/pone.0032794.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3290617/" ref="ordinalpos=4498&amp;ncbi_uid=3838297&amp;link_uid=PMC3290617" image-link="/pmc/articles/PMC3290617/figure/pone-0032794-g001/" class="imagepopup">Figure 1. Theoretical model showing the interaction between miR-146b-5p, NFκB-mediated inflammation, ROS production and the insulin <span class="highlight" style="background-color:">signaling</span>..  From: Decrease of miR-146b-5p in Monocytes during Obesity Is Associated with Loss of the Anti-Inflammatory but Not Insulin <span class="highlight" style="background-color:">Signaling</span> Action of Adiponectin. </a></p><div class="supp"><p class="details"><div>Activation of monocytes in the circulation is characterized by increased activation of the inflammatory toll-like receptor-2 (TLR2)/NFκB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activation of this <span class="highlight" style="background-color:">pathway</span> will elicit pro-inflammatory cytokine release (e.g. tumor necrosis factor α (TNFα)), mitochondrial and intracellular ROS production and impairment of the insulin sensitivity at the molecular level. More mitochondrial ROS production is associated with an increased expression of the mitochondrial antioxidant <i>superoxide dismutase-2</i> (<i>SOD2</i>), suggesting that <i>SOD2</i> only acts as mitochondrial stress marker (indicated as dotted line). The formed ROS will not only induce the IKK complex phosphorylation and thereby the translocation of NFκB but will also lead to inhibition of the insulin receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Interestingly, interleukin-1 receptor-associated kinase-3 (IRAK3) and miR-146b-5p are located upstream of the transcription factor NFκB and negatively regulates TLR <span class="highlight" style="background-color:">signaling</span>. IRAK3 prevents the dissociation of IRAK1 and IRAK4 from MyD88 and formation of IRAK-TRAF6 complexes. miR-146b-5p, on the other hand, inhibits the translation of IRAK1 and TRAF6. Flow of the <span class="highlight" style="background-color:">pathway</span> at the protein interaction level is indicated by black arrows. Blunted arrows indicate inhibition. Phosphorylation is indicated by a circled P.</div></p></div><div class="aux"><div class="resc">Maarten Hulsmans, et al. PLoS One. 2012;7(2):e32794.</div><p class="links"><a class="dblinks" href="/pubmed/22393448" ref="ordinalpos=4498&amp;ncbi_uid=3838297&amp;link_uid=22393448">Citation</a><a class="dblinks" href="/pmc/articles/PMC3290617/" ref="ordinalpos=4498&amp;ncbi_uid=3838297&amp;link_uid=PMC3290617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4499.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1567929/figure/F7/" ref="ordinalpos=4499&amp;ncbi_uid=834243&amp;link_uid=PMC1567929" href="/pmc/articles/PMC1567929/"><img src="/pmc/articles/instance/1567929/bin/zpq0310630370007.gif" src-large="/pmc/articles/instance/1567929/bin/zpq0310630370007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1567929/" ref="ordinalpos=4499&amp;ncbi_uid=834243&amp;link_uid=PMC1567929" image-link="/pmc/articles/PMC1567929/figure/F7/" class="imagepopup">Fig. 7. From: A Ca2+ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates a K+ channel for low-K response in Arabidopsis. </a></p><div class="supp"><p class="details"><div>A schematic model of the Ca<sup>2+</sup>-dependent <span class="highlight" style="background-color:">pathway</span> for low-K response in <i>Arabidopsis</i> (see details in <i>Concluding Remarks</i>). ROS, reactive oxygen species.</div></p></div><div class="aux"><div class="resc">Legong Li, et al. Proc Natl Acad Sci U S A. 2006 August 15;103(33):12625-12630.</div><p class="links"><a class="dblinks" href="/pubmed/16895985" ref="ordinalpos=4499&amp;ncbi_uid=834243&amp;link_uid=16895985">Citation</a><a class="dblinks" href="/pmc/articles/PMC1567929/" ref="ordinalpos=4499&amp;ncbi_uid=834243&amp;link_uid=PMC1567929">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4500.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4388007/figure/F4/" ref="ordinalpos=4500&amp;ncbi_uid=7446711&amp;link_uid=PMC4388007" href="/pmc/articles/PMC4388007/"><img src="/pmc/articles/instance/4388007/bin/fcimb-05-00030-g0004.gif" src-large="/pmc/articles/instance/4388007/bin/fcimb-05-00030-g0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4388007/" ref="ordinalpos=4500&amp;ncbi_uid=7446711&amp;link_uid=PMC4388007" image-link="/pmc/articles/PMC4388007/figure/F4/" class="imagepopup">Figure 4.  From: Comparative proteomic analyses demonstrate enhanced interferon and STAT-1 activation in reovirus T3D-infected HeLa cells. </a></p><div class="supp"><p class="details"><div><b>Significantly affected canonical <span class="highlight" style="background-color:">pathway</span> “interferon <span class="highlight" style="background-color:">signaling</span>”, as determined by IPA® analysis</b>. Red indicates highly up-regulated proteins, light green indicates down-regulated, gray represents not significantly affected, and white represents molecules known to be part of the <span class="highlight" style="background-color:">pathway</span> but not identified by iTRAQ. Examples of additional affected canonical pathways are depicted in Supplementary Figure S2.</div></p></div><div class="aux"><div class="resc">Peyman Ezzati, et al. Front Cell Infect Microbiol. 2015;5:30.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4500&amp;ncbi_uid=7446711&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4388007/" ref="ordinalpos=4500&amp;ncbi_uid=7446711&amp;link_uid=PMC4388007">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="44" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="45" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="46" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:47:22-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB1FD954312FD10000000000808A0F;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB1FD954312FD10000000000808A0F" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>