
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="14" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A4E7753FEC2D10000000000E83C91" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="13" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="14" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="15" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="14" /></div>
            <div><h2 class="result_count">Results: 1301 to 1400 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1301.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2825653/figure/fig5/" ref="ordinalpos=1301&amp;ncbi_uid=2273064&amp;link_uid=PMC2825653" href="/pmc/articles/PMC2825653/"><img src="/pmc/articles/instance/2825653/bin/JBB2010-795385.005.gif" src-large="/pmc/articles/instance/2825653/bin/JBB2010-795385.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2825653/" ref="ordinalpos=1301&amp;ncbi_uid=2273064&amp;link_uid=PMC2825653" image-link="/pmc/articles/PMC2825653/figure/fig5/" class="imagepopup">Figure 5.  From: Epidermal Growth Factor (EGF) Treatment on Multipotential Stromal Cells (MSCs). Possible Enhancement of Therapeutic Potential of MSC. </a></p><div class="supp"><p class="details"><div>Simplified diagram of EGFR/ErbB1 <span class="highlight" style="background-color:">signaling</span> pathways in MSC physiology. EGFR/ErbB1 ligands activate PLC<i>γ</i> <span class="highlight" style="background-color:">pathway</span>, p42/44 MAPK <span class="highlight" style="background-color:">pathway</span>, and PI3K/Akt pathways in MSCs [3]. PLC<i>γ</i> <span class="highlight" style="background-color:">pathway</span> plays a pivotal role in motogenic activity, whereas p42/44 MAPK <span class="highlight" style="background-color:">pathway</span> plays a key role in mitogenic activity and paracrine activities of certain factors such as VEGF    [42, 43, 66, 67].  Sustained and strong activation of p42/44 MAPK <span class="highlight" style="background-color:">pathway</span> exerts cytoprotective and pro-osteogenic effects    [44, 47], whereas PI3K/Akt <span class="highlight" style="background-color:">pathway</span> might exert anti-osteogenic effects [46].</div></p></div><div class="aux"><div class="resc">Kenichi Tamama, et al. J Biomed Biotechnol. 2010;2010:795385.</div><p class="links"><a class="dblinks" href="/pubmed/20182548" ref="ordinalpos=1301&amp;ncbi_uid=2273064&amp;link_uid=20182548">Citation</a><a class="dblinks" href="/pmc/articles/PMC2825653/" ref="ordinalpos=1301&amp;ncbi_uid=2273064&amp;link_uid=PMC2825653">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1302.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634381/figure/F1/" ref="ordinalpos=1302&amp;ncbi_uid=1710907&amp;link_uid=PMC2634381" href="/pmc/articles/PMC2634381/"><img src="/pmc/articles/instance/2634381/bin/psb0310_0813_fig001.gif" src-large="/pmc/articles/instance/2634381/bin/psb0310_0813_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634381/" ref="ordinalpos=1302&amp;ncbi_uid=1710907&amp;link_uid=PMC2634381" image-link="/pmc/articles/PMC2634381/figure/F1/" class="imagepopup">Figure 1.  From: Receptor-like protein kinases, BAK1 and BKK1, regulate a light-dependent cell-death control <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>BAK1 and BKK1 regulate a light-dependent cell-death control <span class="highlight" style="background-color:">pathway</span>. (A) A representative Trypan blue stained wild-type cotyledon from a dark grown seedling on a ½ MS plate; (B) A representative Trypan blue stained <i>bak1–4 bkk1-1</i> cotyledon from a dark grown seedling on a ½ MS plate; (C) A representative Trypan blue stained wild-type cotyledon from a long-day light-grown seedling on a ½ MS plate; (D) A representative Trypan blue stained <i>bak1–4 bkk1-1</i> cotyledon from a long-day light-grown seedling on a ½ MS plate; (E) A hypothetical model of BAK1 and BKK1 in regulating both the BR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to promote cell growth, and a novel light-dependent cell-death control <span class="highlight" style="background-color:">pathway</span> to prevent plants from unnecessary cell-death. Under a light condition, plants naturally produce unknown toxins (phototoxins), whose accumulation can lead to the cell-death. BAK1 and BKK1 likely mediate a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to constantly check and limit the levels of these toxins.</div></p></div><div class="aux"><div class="resc">Kai He, et al. Plant Signal Behav. 2008 October;3(10):813-815.</div><p class="links"><a class="dblinks" href="/pubmed/19704566" ref="ordinalpos=1302&amp;ncbi_uid=1710907&amp;link_uid=19704566">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634381/" ref="ordinalpos=1302&amp;ncbi_uid=1710907&amp;link_uid=PMC2634381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1303.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775773/figure/pone-0074322-g003/" ref="ordinalpos=1303&amp;ncbi_uid=5330274&amp;link_uid=PMC3775773" href="/pmc/articles/PMC3775773/"><img src="/pmc/articles/instance/3775773/bin/pone.0074322.g003.gif" src-large="/pmc/articles/instance/3775773/bin/pone.0074322.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775773/" ref="ordinalpos=1303&amp;ncbi_uid=5330274&amp;link_uid=PMC3775773" image-link="/pmc/articles/PMC3775773/figure/pone-0074322-g003/" class="imagepopup">Figure 3. NsPEF inhibits cell proliferation through repressing NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to reduce expressions of Cyclin proteins in vitro..  From: Nanosecond Pulsed Electric Field Inhibits Cancer Growth Followed by Alteration in Expressions of NF-?B and Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> Molecules. </a></p><div class="supp"><p class="details"><div>(A) Cell cycle of cancer cells exposed to nsPEF with different intensities was examined with PI staining assay by flow cytometry, and results were analyzed by FlowJo 7.6.1 Software. *p&lt;0.05, **p&lt;0.01. (B) Protein expressions of NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> including IKK-α, IKK-β IκB–α, p65 and p-p65 in cancer cells after exposure to nsPEF with different intensities were detected by Western-blot assay. (C) Protein expressions of Cyclin proteins including Cyclin D1 and Cyclin A in cancer cells after exposure to nsPEF with different intensities were detected by Western-blot assay. Relative intensity of each protein was analyzed by Photoshop CS4 software. *p&lt;0.05, **p&lt;0.01, ***p&lt;0.001.</div></p></div><div class="aux"><div class="resc">Zhigang Ren, et al. PLoS One. 2013;8(9):e74322.</div><p class="links"><a class="dblinks" href="/pubmed/24069295" ref="ordinalpos=1303&amp;ncbi_uid=5330274&amp;link_uid=24069295">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775773/" ref="ordinalpos=1303&amp;ncbi_uid=5330274&amp;link_uid=PMC3775773">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1304.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3894232/figure/F2/" ref="ordinalpos=1304&amp;ncbi_uid=5742753&amp;link_uid=PMC3894232" href="/pmc/articles/PMC3894232/"><img src="/pmc/articles/instance/3894232/bin/jkst-2-e24931-g2.gif" src-large="/pmc/articles/instance/3894232/bin/jkst-2-e24931-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3894232/" ref="ordinalpos=1304&amp;ncbi_uid=5742753&amp;link_uid=PMC3894232" image-link="/pmc/articles/PMC3894232/figure/F2/" class="imagepopup">Figure. From: JAK2-STAT5B <span class="highlight" style="background-color:">pathway</span> and osteoblast differentiation. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> GH transmits the signal in two phases. In the first phase (represented as dark arrow), GH binds to the GHR and triggers the JAK2-STAT5B <span class="highlight" style="background-color:">signaling</span> axis. This leads to the transcription of IGF-1 and IGF-1R. These then mediates the second phase (represented by dotted arrows) of signal transduction. Here IGF-1 binds to IGF-1R and triggers the MAPK <span class="highlight" style="background-color:">pathway</span> or PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and induces osteoblast proliferation and differentiation.</div></p></div><div class="aux"><div class="resc">Pramod Darvin, et al. JAKSTAT. 2013 October 1;2(4):e24931.</div><p class="links"><a class="dblinks" href="/pubmed/24470975" ref="ordinalpos=1304&amp;ncbi_uid=5742753&amp;link_uid=24470975">Citation</a><a class="dblinks" href="/pmc/articles/PMC3894232/" ref="ordinalpos=1304&amp;ncbi_uid=5742753&amp;link_uid=PMC3894232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1305.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3589553/figure/F3/" ref="ordinalpos=1305&amp;ncbi_uid=5197567&amp;link_uid=PMC3589553" href="/pmc/articles/PMC3589553/"><img src="/pmc/articles/instance/3589553/bin/355fig3.gif" src-large="/pmc/articles/instance/3589553/bin/355fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3589553/" ref="ordinalpos=1305&amp;ncbi_uid=5197567&amp;link_uid=PMC3589553" image-link="/pmc/articles/PMC3589553/figure/F3/" class="imagepopup">Figure 3. From: The Hippo <span class="highlight" style="background-color:">pathway</span>: regulators and regulations. </a></p><div class="supp"><p class="details"><div>Regulatory mechanisms for the Hippo <span class="highlight" style="background-color:">pathway</span>. Regulation of the Hippo <span class="highlight" style="background-color:">pathway</span> by apical–basal polarity (<i>A</i>), PCP (<i>B</i>), mechanical cues and GPCR <span class="highlight" style="background-color:">signaling</span> (<i>C</i>), and actin cytoskeleton (<i>D</i>). Arrowed or blunted ends indicate activation or inhibition, respectively. Dashed lines indicate indirect or unknown mechanisms. Red lines in <i>D</i> represent actin filaments.</div></p></div><div class="aux"><div class="resc">Fa-Xing Yu, et al. Genes Dev. 2013 February 15;27(4):355-371.</div><p class="links"><a class="dblinks" href="/pubmed/23431053" ref="ordinalpos=1305&amp;ncbi_uid=5197567&amp;link_uid=23431053">Citation</a><a class="dblinks" href="/pmc/articles/PMC3589553/" ref="ordinalpos=1305&amp;ncbi_uid=5197567&amp;link_uid=PMC3589553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1306.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2782488/figure/F3/" ref="ordinalpos=1306&amp;ncbi_uid=2161864&amp;link_uid=PMC2782488" href="/pmc/articles/PMC2782488/"><img src="/pmc/articles/instance/2782488/bin/nihms-107421-f0003.gif" src-large="/pmc/articles/instance/2782488/bin/nihms-107421-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2782488/" ref="ordinalpos=1306&amp;ncbi_uid=2161864&amp;link_uid=PMC2782488" image-link="/pmc/articles/PMC2782488/figure/F3/" class="imagepopup">Figure 3. Mechanisms associated with Socs3-mediated suppression of anti-inflammatory <span class="highlight" style="background-color:">signaling</span> by the IL-6R.  From: Cytokine <span class="highlight" style="background-color:">signaling</span> modules in inflammatory responses. </a></p><div class="supp"><p class="details"><div>The left side depicts IL-10 <span class="highlight" style="background-color:">signaling</span> in a macrophage activated by the TLR <span class="highlight" style="background-color:">pathway</span> (or other similar inflammatory stimuli). Socs3 expression is strongly induced by IL-10, along with the Stat3-dependent genes whose products regulate the anti-inflammatory <span class="highlight" style="background-color:">signaling</span> system (‘anti-inflammatory response’ AIR gene) illustrated as inhibiting the expression at the transcriptional level of classic pro-inflammatory genes. On the right side is shown IL-6 <span class="highlight" style="background-color:">signaling</span> via gp130, which also activates Socs3 and other genes. Unlike the IL-10R, the IL-6R cannot activate the expression of the AIR gene(s) unless Socs3 is absent. Thus, IL-6 and IL-10 (and any other receptors that activate Socs3 expression in macrophages) enforce the inability of the IL-6R to produce the anti-inflammatory response. Note that Socs3 (or any other Socs protein) does not inhibit the IL-10R.</div></p></div><div class="aux"><div class="resc">John J. O’Shea, et al. Immunity. ;28(4):477-487.</div><p class="links"><a class="dblinks" href="/pubmed/18400190" ref="ordinalpos=1306&amp;ncbi_uid=2161864&amp;link_uid=18400190">Citation</a><a class="dblinks" href="/pmc/articles/PMC2782488/" ref="ordinalpos=1306&amp;ncbi_uid=2161864&amp;link_uid=PMC2782488">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1307.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2661616/figure/F2/" ref="ordinalpos=1307&amp;ncbi_uid=1748524&amp;link_uid=PMC2661616" href="/pmc/articles/PMC2661616/"><img src="/pmc/articles/instance/2661616/bin/nihms93405f2.gif" src-large="/pmc/articles/instance/2661616/bin/nihms93405f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2661616/" ref="ordinalpos=1307&amp;ncbi_uid=1748524&amp;link_uid=PMC2661616" image-link="/pmc/articles/PMC2661616/figure/F2/" class="imagepopup">Figure 2.  From: RAGE <span class="highlight" style="background-color:">signaling</span> in inflammation and arterial aging. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> via RAGE. RAGE is activated by its ligand, and signals are relayed to three potential pathways. Among the identified <span class="highlight" style="background-color:">signaling</span> routes, NF-κB <span class="highlight" style="background-color:">pathway</span> is best studied and is likely to play major role in RAGE <span class="highlight" style="background-color:">signaling</span>-mediated inflammation. Depending upon ligand type, these pathways are either selectively activated, or, crosstalk, to regulate cellular activities.</div></p></div><div class="aux"><div class="resc">Li Lin, et al. Front Biosci. ;14:1403-1413.</div><p class="links"><a class="dblinks" href="/pubmed/19273137" ref="ordinalpos=1307&amp;ncbi_uid=1748524&amp;link_uid=19273137">Citation</a><a class="dblinks" href="/pmc/articles/PMC2661616/" ref="ordinalpos=1307&amp;ncbi_uid=1748524&amp;link_uid=PMC2661616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1308.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2719801/figure/pone-0006575-g002/" ref="ordinalpos=1308&amp;ncbi_uid=2030413&amp;link_uid=PMC2719801" href="/pmc/articles/PMC2719801/"><img src="/pmc/articles/instance/2719801/bin/pone.0006575.g002.gif" src-large="/pmc/articles/instance/2719801/bin/pone.0006575.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2719801/" ref="ordinalpos=1308&amp;ncbi_uid=2030413&amp;link_uid=PMC2719801" image-link="/pmc/articles/PMC2719801/figure/pone-0006575-g002/" class="imagepopup">Figure 2. Regulation of insulin <span class="highlight" style="background-color:">signaling</span> in people with type 2 diabetes..  From: Gene Expression in Skeletal Muscle Biopsies from People with Type 2 Diabetes and Relatives: Differential Regulation of Insulin <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>The insulin <span class="highlight" style="background-color:">signaling</span> pathways were found to be significantly downregulated on the gene expression level using the program GenMAPP/MAPPFinder. Analysis criteria were set as described in the Methods section. Underneath each section of the <span class="highlight" style="background-color:">pathway</span>, genes found to have an increased expression are depicted in red, and genes found to have a decreased expression are depicted in green. Figure adapted from GenMAPP.</div></p></div><div class="aux"><div class="resc">Jane Palsgaard, et al. PLoS One. 2009;4(8):e6575.</div><p class="links"><a class="dblinks" href="/pubmed/19668377" ref="ordinalpos=1308&amp;ncbi_uid=2030413&amp;link_uid=19668377">Citation</a><a class="dblinks" href="/pmc/articles/PMC2719801/" ref="ordinalpos=1308&amp;ncbi_uid=2030413&amp;link_uid=PMC2719801">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1309.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2683183/figure/F1/" ref="ordinalpos=1309&amp;ncbi_uid=1791095&amp;link_uid=PMC2683183" href="/pmc/articles/PMC2683183/"><img src="/pmc/articles/instance/2683183/bin/nihms84103f1.gif" src-large="/pmc/articles/instance/2683183/bin/nihms84103f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2683183/" ref="ordinalpos=1309&amp;ncbi_uid=1791095&amp;link_uid=PMC2683183" image-link="/pmc/articles/PMC2683183/figure/F1/" class="imagepopup">Figure 1. GPCR-mediated <span class="highlight" style="background-color:">signaling</span> to mitochondria.  From: Cardioprotective <span class="highlight" style="background-color:">Signaling</span> to Mitochondria. </a></p><div class="supp"><p class="details"><div>Occupation of the GPCR leads to activation of PI3-kinase, phosphorylation of phosphatidylinositol bisphosphate, and activation of the phosphatidylinositol-dependent kinases (PDKs) [128]. PDKs then phosphorylate Akt, which initiates the remainder of the cytosolic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: endothelial nitric oxide synthase (eNOS) is phosphorylated, leading to production of NO. NO stimulates guanylyl cyclase, and the cGMP produced activates protein kinase G (PKG) [129], which causes mitoK<sub>ATP</sub> opening [62, 130–132].</div></p></div><div class="aux"><div class="resc">Keith D. Garlid, et al. J Mol Cell Cardiol. ;46(6):858-866.</div><p class="links"><a class="dblinks" href="/pubmed/19118560" ref="ordinalpos=1309&amp;ncbi_uid=1791095&amp;link_uid=19118560">Citation</a><a class="dblinks" href="/pmc/articles/PMC2683183/" ref="ordinalpos=1309&amp;ncbi_uid=1791095&amp;link_uid=PMC2683183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1310.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3969039/figure/f2-molcell-37-3-196-2/" ref="ordinalpos=1310&amp;ncbi_uid=6012688&amp;link_uid=PMC3969039" href="/pmc/articles/PMC3969039/"><img src="/pmc/articles/instance/3969039/bin/molcell-37-3-196-2f2.gif" src-large="/pmc/articles/instance/3969039/bin/molcell-37-3-196-2f2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3969039/" ref="ordinalpos=1310&amp;ncbi_uid=6012688&amp;link_uid=PMC3969039" image-link="/pmc/articles/PMC3969039/figure/f2-molcell-37-3-196-2/" class="imagepopup">Fig. 2. From: Apelin-APJ <span class="highlight" style="background-color:">Signaling</span>: a Potential Therapeutic Target for Pulmonary Arterial Hypertension. </a></p><div class="supp"><p class="details"><div>Putative model of the mechanism of vascular homeostasis mediated by apelin-APJ <span class="highlight" style="background-color:">signaling</span>. (A) Basal apelin-APJ <span class="highlight" style="background-color:">signaling</span> maintains expression of miR-424/503, KLF2, and eNOS, as well as activity of AMPK. miR-424/503 in turn inhibits FGF2/FGFR1 expression, and eNOS induces the production of NO. (B) In PAH, there is decreased apelin, miR-424/503 and eNOS expression, leading to abnormal activation of FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and reducing NO level that target both the PAECs and PASMCs.</div></p></div><div class="aux"><div class="resc">Jongmin Kim. Mol Cells. 2014 March 31;37(3):196-201.</div><p class="links"><a class="dblinks" href="/pubmed/24608803" ref="ordinalpos=1310&amp;ncbi_uid=6012688&amp;link_uid=24608803">Citation</a><a class="dblinks" href="/pmc/articles/PMC3969039/" ref="ordinalpos=1310&amp;ncbi_uid=6012688&amp;link_uid=PMC3969039">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1311.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2797767/figure/F3/" ref="ordinalpos=1311&amp;ncbi_uid=2120906&amp;link_uid=PMC2797767" href="/pmc/articles/PMC2797767/"><img src="/pmc/articles/instance/2797767/bin/1476-4598-8-117-3.gif" src-large="/pmc/articles/instance/2797767/bin/1476-4598-8-117-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2797767/" ref="ordinalpos=1311&amp;ncbi_uid=2120906&amp;link_uid=PMC2797767" image-link="/pmc/articles/PMC2797767/figure/F3/" class="imagepopup">Figure 3.  From: Identification of small molecule compounds that inhibit the HIF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Chemical structures of inhibitors for HIF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. A. Four representative inhibitors from the qHTS. NCGC00044926 is a representative from series 31, NCGC00043898 and NCGC00049606 from series 39 and NCGC00056044 from series 71. B. Number of active and inactive compounds containing the 5-thiophen-1,2,4-oxadiazole core versus the 1,2,4-oxadiazole core. The thiophen-oxadiazole core appears to be important for compound activity. <i>Right</i>, Leadscope Maximal Common Substructures (MCS) used to populate series 31 and 39.</div></p></div><div class="aux"><div class="resc">Menghang Xia, et al. Mol Cancer. 2009;8:117-117.</div><p class="links"><a class="dblinks" href="/pubmed/20003191" ref="ordinalpos=1311&amp;ncbi_uid=2120906&amp;link_uid=20003191">Citation</a><a class="dblinks" href="/pmc/articles/PMC2797767/" ref="ordinalpos=1311&amp;ncbi_uid=2120906&amp;link_uid=PMC2797767">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1312.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3202302/figure/F5/" ref="ordinalpos=1312&amp;ncbi_uid=3512089&amp;link_uid=PMC3202302" href="/pmc/articles/PMC3202302/"><img src="/pmc/articles/instance/3202302/bin/nihms322868f5.gif" src-large="/pmc/articles/instance/3202302/bin/nihms322868f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3202302/" ref="ordinalpos=1312&amp;ncbi_uid=3512089&amp;link_uid=PMC3202302" image-link="/pmc/articles/PMC3202302/figure/F5/" class="imagepopup">Fig. 5. <span class="highlight" style="background-color:">Signaling</span> differences and similarities between CD40 and BAFF-R.  From: <span class="highlight" style="background-color:">Signaling</span> by the TNFR superfamily in B-cell biology and disease. </a></p><div class="supp"><p class="details"><div>Both BAFF-R and CD40 <span class="highlight" style="background-color:">signaling</span> utilize TRAF molecules to promote activation of NFκB. In the case of BAFF-R, direct binding of TRAF3 initiates a cascade resulting in release of NIK and downstream activation of non-canonical NFκB. In this regard, recruitment of a complex containing TRAF3- TRAF2-cIAP1/2-NIK leads to TRAF3 degradation via TRAF2-activated cIAP1/2. Alternatively, NIK can be released from the TRAF3-TRAF2-cIAP1/2-NIK complex via displacement of cAP1/2 and NIK from TRAF2 and TRAF3 which are then degraded. NIK then functions to activate IKK2 resulting in non-canonical NFκB <span class="highlight" style="background-color:">signaling</span>. Upon CD40L engagement of CD40, TRAF2 activation of cIAP can not only lead to activation of NIK but also of MEKK, promoting downstream MAPK <span class="highlight" style="background-color:">signaling</span>. In addition, TRAF6 recruitment results in the activation of the TAB1-TAK complex, which is also responsible for activation of the canonical NFκB <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Robert C. Rickert, et al. Immunol Rev. ;244(1):115-133.</div><p class="links"><a class="dblinks" href="/pubmed/22017435" ref="ordinalpos=1312&amp;ncbi_uid=3512089&amp;link_uid=22017435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3202302/" ref="ordinalpos=1312&amp;ncbi_uid=3512089&amp;link_uid=PMC3202302">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1313.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3746742/figure/F7/" ref="ordinalpos=1313&amp;ncbi_uid=5212002&amp;link_uid=PMC3746742" href="/pmc/articles/PMC3746742/"><img src="/pmc/articles/instance/3746742/bin/nihms142951f7.gif" src-large="/pmc/articles/instance/3746742/bin/nihms142951f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3746742/" ref="ordinalpos=1313&amp;ncbi_uid=5212002&amp;link_uid=PMC3746742" image-link="/pmc/articles/PMC3746742/figure/F7/" class="imagepopup">Figure 7.  From: Distinct growth factor-induced dynamic mass redistribution (DMR) profiles for monitoring oncogenic <span class="highlight" style="background-color:">signaling</span> pathways in various cancer cells. </a></p><div class="supp"><p class="details"><div>Model of the mechanism by which the EGF-induced DMR signal is mediated in UPCI-37B SCCHN cells and its application for HTS of EGFR <span class="highlight" style="background-color:">pathway</span> inhibitors. The PI3K/Akt (solid arrow), but not Ras/Raf/MEK/ERK (dotted arrow) <span class="highlight" style="background-color:">pathway</span>, mediates the EGF-induced DMR signal in UPCI-37B SCCHN cells. Our established cell-based optical signature assay is a novel approach for the discovery of inhibitors targeting the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through HTS.</div></p></div><div class="aux"><div class="resc">Yuhong Du, et al. J Recept Signal Transduct Res. ;29(0):182-194.</div><p class="links"><a class="dblinks" href="/pubmed/19604037" ref="ordinalpos=1313&amp;ncbi_uid=5212002&amp;link_uid=19604037">Citation</a><a class="dblinks" href="/pmc/articles/PMC3746742/" ref="ordinalpos=1313&amp;ncbi_uid=5212002&amp;link_uid=PMC3746742">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1314.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262490/figure/F3/" ref="ordinalpos=1314&amp;ncbi_uid=3767251&amp;link_uid=PMC3262490" href="/pmc/articles/PMC3262490/"><img src="/pmc/articles/instance/3262490/bin/1471-2407-11-502-3.gif" src-large="/pmc/articles/instance/3262490/bin/1471-2407-11-502-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262490/" ref="ordinalpos=1314&amp;ncbi_uid=3767251&amp;link_uid=PMC3262490" image-link="/pmc/articles/PMC3262490/figure/F3/" class="imagepopup">Figure 3.  From: Current views on the role of Notch <span class="highlight" style="background-color:">signaling</span> and the pathogenesis of human leukemia. </a></p><div class="supp"><p class="details"><div><b>Noncanonical Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. <b>(A) </b>NICD-dependent, CBF1-independent transcriptional activation by NICD, coactivators and other undefined factors <b>(B) </b>Interaction of NICD with components of other <span class="highlight" style="background-color:">signaling</span> pathways to activate Notch targets or tissue-specific factors.</div></p></div><div class="aux"><div class="resc">Joanna Pancewicz, et al. BMC Cancer. 2011;11:502-502.</div><p class="links"><a class="dblinks" href="/pubmed/22128846" ref="ordinalpos=1314&amp;ncbi_uid=3767251&amp;link_uid=22128846">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262490/" ref="ordinalpos=1314&amp;ncbi_uid=3767251&amp;link_uid=PMC3262490">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1315.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3196095/figure/F6/" ref="ordinalpos=1315&amp;ncbi_uid=3437668&amp;link_uid=PMC3196095" href="/pmc/articles/PMC3196095/"><img src="/pmc/articles/instance/3196095/bin/zbc0471182980006.gif" src-large="/pmc/articles/instance/3196095/bin/zbc0471182980006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3196095/" ref="ordinalpos=1315&amp;ncbi_uid=3437668&amp;link_uid=PMC3196095" image-link="/pmc/articles/PMC3196095/figure/F6/" class="imagepopup">FIGURE 6. From: Inhibition of c-Jun-N-terminal Kinase Increases Cardiac Peroxisome Proliferator-activated Receptor ? Expression and Fatty Acid Oxidation and Prevents Lipopolysaccharide-induced Heart Dysfunction. </a></p><div class="supp"><p class="details"><div><b>Proposed model.</b> <i>A</i>, schematic model that explains the role of the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the inhibition of fatty acid oxidation and relevant gene expression that leads to cardiac dysfunction during sepsis. Binding of the complex that consists of LPS and lipopolysaccharide binding protein (<i>LBP</i>) on the TLR4 and CD14 receptors activates the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In addition, inflammatory pathways are activated via stimulation of the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to down-regulation of the expression of fatty acid oxidation-associated genes and eventually causes cardiac dysfunction. <i>B</i>, inhibition of JNK prevents sepsis-mediated down-regulation of cardiac fatty acid oxidation, thus it protects cardiac function in sepsis despite increased levels of inflammatory cytokines.</div></p></div><div class="aux"><div class="resc">Konstantinos Drosatos, et al. J Biol Chem. 2011 October 21;286(42):36331-36339.</div><p class="links"><a class="dblinks" href="/pubmed/21873422" ref="ordinalpos=1315&amp;ncbi_uid=3437668&amp;link_uid=21873422">Citation</a><a class="dblinks" href="/pmc/articles/PMC3196095/" ref="ordinalpos=1315&amp;ncbi_uid=3437668&amp;link_uid=PMC3196095">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1316.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3657461/figure/fig1/" ref="ordinalpos=1316&amp;ncbi_uid=4964527&amp;link_uid=PMC3657461" href="/pmc/articles/PMC3657461/"><img src="/pmc/articles/instance/3657461/bin/PC2013-920612.001.gif" src-large="/pmc/articles/instance/3657461/bin/PC2013-920612.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3657461/" ref="ordinalpos=1316&amp;ncbi_uid=4964527&amp;link_uid=PMC3657461" image-link="/pmc/articles/PMC3657461/figure/fig1/" class="imagepopup">Figure 1.  From: Dissecting Major <span class="highlight" style="background-color:">Signaling</span> Pathways throughout the Development of Prostate Cancer. </a></p><div class="supp"><p class="details"><div>Androgen receptor (AR) <span class="highlight" style="background-color:">signaling</span> in prostate cancer. (a) Schematic representation of the AR gene, highlighting some major AR mutations and their exon localization. (b) Schematic representation of AR protein structure with indication of its functional domains. (c) AR-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The androgen-receptor (AR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> begins with the translocation of the testosterone to the cytoplasm, where it can be converted to dihydrotestosterone (DHT) and then promote the receptor dimerization and its further migration to the nucleus. A variety of signals, including PTEN-dependent downregulation, can also merge to AR stabilization and further activation (as indicated). </div></p></div><div class="aux"><div class="resc">Henrique B. da Silva, et al. Prostate Cancer. 2013;2013:920612.</div><p class="links"><a class="dblinks" href="/pubmed/23738079" ref="ordinalpos=1316&amp;ncbi_uid=4964527&amp;link_uid=23738079">Citation</a><a class="dblinks" href="/pmc/articles/PMC3657461/" ref="ordinalpos=1316&amp;ncbi_uid=4964527&amp;link_uid=PMC3657461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1317.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711797/figure/F4/" ref="ordinalpos=1317&amp;ncbi_uid=5104979&amp;link_uid=PMC3711797" href="/pmc/articles/PMC3711797/"><img src="/pmc/articles/instance/3711797/bin/1752-0509-7-59-4.gif" src-large="/pmc/articles/instance/3711797/bin/1752-0509-7-59-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711797/" ref="ordinalpos=1317&amp;ncbi_uid=5104979&amp;link_uid=PMC3711797" image-link="/pmc/articles/PMC3711797/figure/F4/" class="imagepopup">Figure 4.  From: Transient oscillatory dynamics of interferon beta <span class="highlight" style="background-color:">signaling</span> in macrophages. </a></p><div class="supp"><p class="details"><div><b>Mathematical model of type I IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Graphical representation of the biological processes modeled.</div></p></div><div class="aux"><div class="resc">Inna Pertsovskaya, et al. BMC Syst Biol. 2013;7:59-59.</div><p class="links"><a class="dblinks" href="/pubmed/23837526" ref="ordinalpos=1317&amp;ncbi_uid=5104979&amp;link_uid=23837526">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711797/" ref="ordinalpos=1317&amp;ncbi_uid=5104979&amp;link_uid=PMC3711797">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1318.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3586350/figure/F8/" ref="ordinalpos=1318&amp;ncbi_uid=4689455&amp;link_uid=PMC3586350" href="/pmc/articles/PMC3586350/"><img src="/pmc/articles/instance/3586350/bin/1479-5876-11-39-8.gif" src-large="/pmc/articles/instance/3586350/bin/1479-5876-11-39-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3586350/" ref="ordinalpos=1318&amp;ncbi_uid=4689455&amp;link_uid=PMC3586350" image-link="/pmc/articles/PMC3586350/figure/F8/" class="imagepopup">Figure 8.  From: Adipose stromal cells primed with hypoxia and inflammation enhance cardiomyocyte proliferation rate in vitro through STAT3 and Erk1/2. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of proposed Adipose Stromal Cell-derived IL-6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The ADSC-secreted cytokine IL-6 and growth factors, which enhance the proliferation of cardiomyocytes through canonical <span class="highlight" style="background-color:">signaling</span> <i>i</i>.<i>e</i>. STAT3 activation and through non-canonical <span class="highlight" style="background-color:">signaling</span> <i>i</i>.<i>e</i>. Erk1/2 activation. The balance between STAT3 and Erk1/2 <span class="highlight" style="background-color:">signaling</span> is governed by STAT3, which can suppress Erk1/2 activation, through an unknown intermediate. In the absence of STAT3 <span class="highlight" style="background-color:">signaling</span>, IL-6-RAS-activated Erk1/2 can compensate for the potential loss of stimulatory effect on the proliferation on cardiomyocytes.</div></p></div><div class="aux"><div class="resc">Ewa Przybyt, et al. J Transl Med. 2013;11:39-39.</div><p class="links"><a class="dblinks" href="/pubmed/23406316" ref="ordinalpos=1318&amp;ncbi_uid=4689455&amp;link_uid=23406316">Citation</a><a class="dblinks" href="/pmc/articles/PMC3586350/" ref="ordinalpos=1318&amp;ncbi_uid=4689455&amp;link_uid=PMC3586350">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1319.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3690219/figure/fig8/" ref="ordinalpos=1319&amp;ncbi_uid=5076504&amp;link_uid=PMC3690219" href="/pmc/articles/PMC3690219/"><img src="/pmc/articles/instance/3690219/bin/BMRI2013-182965.008.gif" src-large="/pmc/articles/instance/3690219/bin/BMRI2013-182965.008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3690219/" ref="ordinalpos=1319&amp;ncbi_uid=5076504&amp;link_uid=PMC3690219" image-link="/pmc/articles/PMC3690219/figure/fig8/" class="imagepopup">Figure 8.  From: Expression and Function of NUMB in Odontogenesis. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> PCR array. Genes from the Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that were affected by NUMB overexpression. </div></p></div><div class="aux"><div class="resc">Haitao Li, et al. Biomed Res Int. 2013;2013:182965.</div><p class="links"><a class="dblinks" href="/pubmed/23841055" ref="ordinalpos=1319&amp;ncbi_uid=5076504&amp;link_uid=23841055">Citation</a><a class="dblinks" href="/pmc/articles/PMC3690219/" ref="ordinalpos=1319&amp;ncbi_uid=5076504&amp;link_uid=PMC3690219">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1320.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3343698/figure/F8/" ref="ordinalpos=1320&amp;ncbi_uid=4659488&amp;link_uid=PMC3343698" href="/pmc/articles/PMC3343698/"><img src="/pmc/articles/instance/3343698/bin/nihms373059f8.gif" src-large="/pmc/articles/instance/3343698/bin/nihms373059f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3343698/" ref="ordinalpos=1320&amp;ncbi_uid=4659488&amp;link_uid=PMC3343698" image-link="/pmc/articles/PMC3343698/figure/F8/" class="imagepopup">Fig. 8. Perspective: the NF-κB <span class="highlight" style="background-color:">signaling</span> system within the context of upstream, downstream, and coordinated regulatory networks.  From: Lessons from mathematically modeling the NF-?B <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The NF-κB <span class="highlight" style="background-color:">signaling</span> system receives inputs from <span class="highlight" style="background-color:">signaling</span> modules associated with a variety of receptors that respond to the presence of distinct classes of molecules. These same <span class="highlight" style="background-color:">signaling</span> modules also activate a subset of other <span class="highlight" style="background-color:">signaling</span> systems. The coordinated activities of the effectors of theses <span class="highlight" style="background-color:">signaling</span> systems together determine the cellular response via gene regulatory networks. Understanding how the coordinated activation of multiple <span class="highlight" style="background-color:">signaling</span> systems produce stimulus-appropriate cellular responses remains an important future goal.</div></p></div><div class="aux"><div class="resc">Soumen Basak, et al. Immunol Rev. ;246(1):221-238.</div><p class="links"><a class="dblinks" href="/pubmed/22435558" ref="ordinalpos=1320&amp;ncbi_uid=4659488&amp;link_uid=22435558">Citation</a><a class="dblinks" href="/pmc/articles/PMC3343698/" ref="ordinalpos=1320&amp;ncbi_uid=4659488&amp;link_uid=PMC3343698">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1321.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3767783/figure/pone-0072999-g006/" ref="ordinalpos=1321&amp;ncbi_uid=5295103&amp;link_uid=PMC3767783" href="/pmc/articles/PMC3767783/"><img src="/pmc/articles/instance/3767783/bin/pone.0072999.g006.gif" src-large="/pmc/articles/instance/3767783/bin/pone.0072999.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3767783/" ref="ordinalpos=1321&amp;ncbi_uid=5295103&amp;link_uid=PMC3767783" image-link="/pmc/articles/PMC3767783/figure/pone-0072999-g006/" class="imagepopup">Figure 6. E2, G1 and OHT promoted cell migration through the GPR30/EGFR/PI3K/ERK/FAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Estradiol and Tamoxifen Induce Cell Migration through GPR30 and Activation of Focal Adhesion Kinase (FAK) in Endometrial Cancers with Low or without Nuclear Estrogen Receptor ? (ER?). </a></p><div class="supp"><p class="details"><div>Our current model of activation of the GPR30-EGFR-PI3K-ERK-FAK <span class="highlight" style="background-color:">pathway</span> by E2, G1 and OHT.</div></p></div><div class="aux"><div class="resc">Chia-Lung Tsai, et al. PLoS One. 2013;8(9):e72999.</div><p class="links"><a class="dblinks" href="/pubmed/24039841" ref="ordinalpos=1321&amp;ncbi_uid=5295103&amp;link_uid=24039841">Citation</a><a class="dblinks" href="/pmc/articles/PMC3767783/" ref="ordinalpos=1321&amp;ncbi_uid=5295103&amp;link_uid=PMC3767783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1322.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3026934/figure/Fig2/" ref="ordinalpos=1322&amp;ncbi_uid=2778128&amp;link_uid=PMC3026934" href="/pmc/articles/PMC3026934/"><img src="/pmc/articles/instance/3026934/bin/335_2010_9277_Fig2_HTML.gif" src-large="/pmc/articles/instance/3026934/bin/335_2010_9277_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3026934/" ref="ordinalpos=1322&amp;ncbi_uid=2778128&amp;link_uid=PMC3026934" image-link="/pmc/articles/PMC3026934/figure/Fig2/" class="imagepopup">Fig. 2.  From: Dectin-1: a role in antifungal defense and consequences of genetic polymorphisms in humans. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> induced by Dectin-1. Upon ligand binding, Dectin-1 becomes tyrosine-phosphorylated by Src kinases, thereby providing a docking site for Syk which initiates downstream <span class="highlight" style="background-color:">signaling</span>. The downstream <span class="highlight" style="background-color:">signaling</span> is effected by molecules such as CARD9, Bcl10, and MALT1, which lead to NF-κB activation and cytokine production. Dectin-1 can also activate NFAT and noncanonical NF-κB in a CARD9-Bcl10-MALT1-independent manner. Stimulation of Dectin-1 with β-glucans can also induce a second Syk-independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by the serine-threonine kinase Raf-1</div></p></div><div class="aux"><div class="resc">Mohlopheni J. Marakalala, et al. Mamm Genome. 2011 February;22(1-2):55-65.</div><p class="links"><a class="dblinks" href="/pubmed/20700596" ref="ordinalpos=1322&amp;ncbi_uid=2778128&amp;link_uid=20700596">Citation</a><a class="dblinks" href="/pmc/articles/PMC3026934/" ref="ordinalpos=1322&amp;ncbi_uid=2778128&amp;link_uid=PMC3026934">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1323.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3095967/figure/F6/" ref="ordinalpos=1323&amp;ncbi_uid=2994469&amp;link_uid=PMC3095967" href="/pmc/articles/PMC3095967/"><img src="/pmc/articles/instance/3095967/bin/nihms-294747-f0006.gif" src-large="/pmc/articles/instance/3095967/bin/nihms-294747-f0006.jpg" alt="Fig. (6)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3095967/" ref="ordinalpos=1323&amp;ncbi_uid=2994469&amp;link_uid=PMC3095967" image-link="/pmc/articles/PMC3095967/figure/F6/" class="imagepopup">Fig. (6). Inhibition of ErbB3 <span class="highlight" style="background-color:">signaling</span> using a multi-receptor targeting approach.  From: Targeting ErbB3: the New RTK(id) on the Prostate Cancer Block. </a></p><div class="supp"><p class="details"><div>The simultaneous inhibition of ErbB1 and ErbB2 in PCa will leave no dimerization partner for ErbB3 and halt its oncogenic <span class="highlight" style="background-color:">signaling</span>. The only possible ErbB dimers in PCa are EGFR homodimers and ErbB1-ErbB2, ErbB2-ErbB3, and ErbB1-ErbB3 heterodimers (see text). All these dimers would stimulate cell survival, for example, though the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> (shown) as well as by other pathways (not shown). ErbB1 inhibitors would disrupt ErbB homodimers and ErbB1-ErbB2 and ErbB1-ErbB3 homodimers, but <span class="highlight" style="background-color:">signaling</span> would still continue through the ErbB2-ErbB3 heterodimers. Similarly, ErbB2 inhibitors would prevent <span class="highlight" style="background-color:">signaling</span> downstream of ErbB1-ErbB2 and ErbB2-ErbB3 heterodimers but allow <span class="highlight" style="background-color:">signaling</span> downstream of EGFR homodimers and ErbB1-ErbB3 heterodimers. However, dual inhibition of both EGFR and ErbB2 would inhibit all 4 dimers, thereby eliminating cell survival downstream of the ErbB receptors.</div></p></div><div class="aux"><div class="resc">Maitreyee K. Jathal, et al. Immunol Endocr Metab Agents Med Chem. ;11(2):131-149.</div><p class="links"><a class="dblinks" href="/pubmed/21603064" ref="ordinalpos=1323&amp;ncbi_uid=2994469&amp;link_uid=21603064">Citation</a><a class="dblinks" href="/pmc/articles/PMC3095967/" ref="ordinalpos=1323&amp;ncbi_uid=2994469&amp;link_uid=PMC3095967">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1324.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204144/figure/F2/" ref="ordinalpos=1324&amp;ncbi_uid=3321736&amp;link_uid=PMC3204144" href="/pmc/articles/PMC3204144/"><img src="/pmc/articles/instance/3204144/bin/cib0405_0612_fig002.gif" src-large="/pmc/articles/instance/3204144/bin/cib0405_0612_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204144/" ref="ordinalpos=1324&amp;ncbi_uid=3321736&amp;link_uid=PMC3204144" image-link="/pmc/articles/PMC3204144/figure/F2/" class="imagepopup">Figure 2.  From: Homeostasis of the Drosophila adult retina by actin-capping protein and the Hippo <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Model for the role of F-actin in Hippo <span class="highlight" style="background-color:">signaling</span> activity. The core kinase module of the Hpo <span class="highlight" style="background-color:">pathway</span> regulates multiple outputs, including tissue growth, neuronal homeostasis and autophagy. Hpo-dependent tissue growth is mediated through inhibition of Yki by a phosphorylation-dependent and independent mechanisms. The neuroprotective effect of Hpo <span class="highlight" style="background-color:">signaling</span> also required Yki inhibition, independent of its phosphorylation by Wts. In contrast, Yki is not involved in Hpo-dependent autophagy. The core kinase complex is regulated by multiple inputs, including Fat and Ex. The interplay between Hpo <span class="highlight" style="background-color:">signaling</span> activity and F-actin dynamics may be a general requirement in Hpo <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Catarina Brás-Pereira, et al. Commun Integr Biol. 2011 Sep-Oct;4(5):612-615.</div><p class="links"><a class="dblinks" href="/pubmed/22046478" ref="ordinalpos=1324&amp;ncbi_uid=3321736&amp;link_uid=22046478">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204144/" ref="ordinalpos=1324&amp;ncbi_uid=3321736&amp;link_uid=PMC3204144">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1325.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3683728/figure/fig06/" ref="ordinalpos=1325&amp;ncbi_uid=5581599&amp;link_uid=PMC3683728" href="/pmc/articles/PMC3683728/"><img src="/pmc/articles/instance/3683728/bin/pnas.1305172110fig06.gif" src-large="/pmc/articles/instance/3683728/bin/pnas.1305172110fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3683728/" ref="ordinalpos=1325&amp;ncbi_uid=5581599&amp;link_uid=PMC3683728" image-link="/pmc/articles/PMC3683728/figure/fig06/" class="imagepopup">Fig. 6. From: HMGA2/TET1/HOXA9 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates breast cancer growth and metastasis. </a></p><div class="supp"><p class="details"><div>The HMGA2/TET1/HOXA <span class="highlight" style="background-color:">pathway</span> regulates breast tumorigenesis. (<i>A</i>) Comparison of the genes regulated by HMGA2, TET1, or HOXA9 in 1833 cells (human breast cancer cells; hBrCa). (<i>B</i>) Scheme illustrating HMGA2/TET1/HOXA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in breast tumorigenesis. (<i>C</i>) Kaplan–Meier analysis of gene expression data from 101 breast tumor patients (see Table S3 for patient information). Patients were stratified for survival using <i>HMGA2</i>, <i>TET1</i>, <i>HOXA9</i>, <i>HOXA7</i>, or the complete <span class="highlight" style="background-color:">pathway</span> as indicated. (<i>Right</i>) Red line, high <i>HMGA2</i> and low <i>TET1/HOXAs</i> (<i>n</i> = 34); blue line, low <i>HMGA2</i> and high <i>TET1/HOXAs</i> (<i>n</i> = 35); <i>P</i>, χ<sup>2</sup> <i>P</i> value.</div></p></div><div class="aux"><div class="resc">Miao Sun, et al. Proc Natl Acad Sci U S A. 2013 June 11;110(24):9920-9925.</div><p class="links"><a class="dblinks" href="/pubmed/23716660" ref="ordinalpos=1325&amp;ncbi_uid=5581599&amp;link_uid=23716660">Citation</a><a class="dblinks" href="/pmc/articles/PMC3683728/" ref="ordinalpos=1325&amp;ncbi_uid=5581599&amp;link_uid=PMC3683728">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1326.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3028836/figure/f8/" ref="ordinalpos=1326&amp;ncbi_uid=3136137&amp;link_uid=PMC3028836" href="/pmc/articles/PMC3028836/"><img src="/pmc/articles/instance/3028836/bin/zii9990989920008.gif" src-large="/pmc/articles/instance/3028836/bin/zii9990989920008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3028836/" ref="ordinalpos=1326&amp;ncbi_uid=3136137&amp;link_uid=PMC3028836" image-link="/pmc/articles/PMC3028836/figure/f8/" class="imagepopup">FIG. 8. From: Identification of Subtilase Cytotoxin (SubAB) Receptors Whose <span class="highlight" style="background-color:">Signaling</span>, in Association with SubAB-Induced BiP Cleavage, Is Responsible for Apoptosis in HeLa Cells  . </a></p><div class="supp"><p class="details"><div>Proposed model of SubAB-induced apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HeLa cells. See text for additional details.</div></p></div><div class="aux"><div class="resc">Kinnosuke Yahiro, et al. Infect Immun. 2011 February;79(2):617-627.</div><p class="links"><a class="dblinks" href="/pubmed/21098100" ref="ordinalpos=1326&amp;ncbi_uid=3136137&amp;link_uid=21098100">Citation</a><a class="dblinks" href="/pmc/articles/PMC3028836/" ref="ordinalpos=1326&amp;ncbi_uid=3136137&amp;link_uid=PMC3028836">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1327.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3827358/figure/pone-0079350-g008/" ref="ordinalpos=1327&amp;ncbi_uid=5486255&amp;link_uid=PMC3827358" href="/pmc/articles/PMC3827358/"><img src="/pmc/articles/instance/3827358/bin/pone.0079350.g008.gif" src-large="/pmc/articles/instance/3827358/bin/pone.0079350.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3827358/" ref="ordinalpos=1327&amp;ncbi_uid=5486255&amp;link_uid=PMC3827358" image-link="/pmc/articles/PMC3827358/figure/pone-0079350-g008/" class="imagepopup">Figure 8. Schematic illustration of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in the inhibitory effect of LPS on Cx43 gap junctions in rat astrocytes..  From: Lipopolysaccharide Induces Degradation of Connexin43 in Rat Astrocytes via the Ubiquitin-Proteasome Proteolytic <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>LPS treatment down-regulates caveolin-3 levels via TLR4-mediated iNOS activation, which in turn reduces the expression of Cx43. LPS also induces JNK phosphorylation, resulting in GJIC inhibition, gap junction disassembly, and Cx43 degradation by the ubiquitin-proteasome proteolytic <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chih-Kai Liao, et al. PLoS One. 2013;8(11):e79350.</div><p class="links"><a class="dblinks" href="/pubmed/24236122" ref="ordinalpos=1327&amp;ncbi_uid=5486255&amp;link_uid=24236122">Citation</a><a class="dblinks" href="/pmc/articles/PMC3827358/" ref="ordinalpos=1327&amp;ncbi_uid=5486255&amp;link_uid=PMC3827358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1328.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2975607/figure/F1/" ref="ordinalpos=1328&amp;ncbi_uid=2567991&amp;link_uid=PMC2975607" href="/pmc/articles/PMC2975607/"><img src="/pmc/articles/instance/2975607/bin/nihms247753f1.gif" src-large="/pmc/articles/instance/2975607/bin/nihms247753f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2975607/" ref="ordinalpos=1328&amp;ncbi_uid=2567991&amp;link_uid=PMC2975607" image-link="/pmc/articles/PMC2975607/figure/F1/" class="imagepopup">Fig. 1.  From: Cellular targets of estrogen <span class="highlight" style="background-color:">signaling</span> in regeneration of inner ear sensory epithelia. </a></p><div class="supp"><p class="details"><div>Schematic representation of the molecular mechanisms of estrogen receptors. The classical <span class="highlight" style="background-color:">pathway</span> of estrogen <span class="highlight" style="background-color:">signaling</span> is ligand-dependent activation of estrogen receptors (ERs) and direct genomic effects on gene regulation by binding estrogen response elements (ERE). The ligand-dependent indirect genomic regulation of gene transcription includes interaction with other transcription factors (TF). In ligand-dependent <span class="highlight" style="background-color:">signaling</span> the ligand activates either membrane-associated classical ERs and/or the G-protein-coupled receptor, GPR30, to initiate <span class="highlight" style="background-color:">signaling</span> cascades via second messengers. This leads to rapid physiological response without involving gene regulation. The ligand-independent <span class="highlight" style="background-color:">pathway</span> can be stimulated by other <span class="highlight" style="background-color:">signaling</span> pathways, such as growth factor <span class="highlight" style="background-color:">signaling</span>, and elicis both genomic and non-genomic effects. The activated kinases phosphorylate estrogen receptors, initiating dimerization and regulation of gene transcription or activation of second messengers. (Adapted from Heldring et al., 2007).</div></p></div><div class="aux"><div class="resc">Jennifer S. McCullar, et al. Hear Res. ;252(1-2):61-70.</div><p class="links"><a class="dblinks" href="/pubmed/19450430" ref="ordinalpos=1328&amp;ncbi_uid=2567991&amp;link_uid=19450430">Citation</a><a class="dblinks" href="/pmc/articles/PMC2975607/" ref="ordinalpos=1328&amp;ncbi_uid=2567991&amp;link_uid=PMC2975607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1329.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151252/figure/fig3/" ref="ordinalpos=1329&amp;ncbi_uid=247041&amp;link_uid=PMC151252" href="/pmc/articles/PMC151252/"><img src="/pmc/articles/instance/151252/bin/00-1768f3.gif" src-large="/pmc/articles/instance/151252/bin/00-1768f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151252/" ref="ordinalpos=1329&amp;ncbi_uid=247041&amp;link_uid=PMC151252" image-link="/pmc/articles/PMC151252/figure/fig3/" class="imagepopup">Figure 3. From: Ethylene Biosynthesis and <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>Interactions between the Ethylene Signal Transduction <span class="highlight" style="background-color:">Pathway</span> and Plant Disease Resistance.<br />The ethylene signal transduction <span class="highlight" style="background-color:">pathway</span> can interact with the JA <span class="highlight" style="background-color:">pathway</span> to co-regulate expression of a subset of defense-related <i>PR</i> genes, for example, <i>PDF1.2</i>, involved in plant disease resistance. Meanwhile, there are considerable interactions between JA/ethylene- and SA-dependent pathways in systemic acquired resistance. In <i>edr1</i> mutant, ethylene potentiates SA-mediated <i>PR-1</i> gene expression. In the absence of CPR5 and CPR6, the ethylene <span class="highlight" style="background-color:">pathway</span> can also activate SA-dependent <i>PR-1</i> gene expression independent of NPR1 to promote systemic acquired resistance. In the <i>ssi1</i> mutant, the JA/ethylene-dependent <i>PDF1.2</i> gene is constitutively expressed. Moreover, the ethylene <span class="highlight" style="background-color:">pathway</span> is also required for the rhizobacteria-mediated induced systemic resistance, which is independent of SA and pathogenesis-related gene activation. Ethylene <span class="highlight" style="background-color:">signaling</span> acts downstream of the JA <span class="highlight" style="background-color:">pathway</span> but upstream of NPR1 in ISR activation. Plants that lack ISR1 fail to develop ISR and display ethylene insensitivity. Arrows indicate positive regulation, and open blocks indicate negative regulation.</div></p></div><div class="aux"><div class="resc">Kevin L.-C. Wang, et al. Plant Cell. 2002;14(Suppl):s131-s151.</div><p class="links"><a class="dblinks" href="/pubmed/12045274" ref="ordinalpos=1329&amp;ncbi_uid=247041&amp;link_uid=12045274">Citation</a><a class="dblinks" href="/pmc/articles/PMC151252/" ref="ordinalpos=1329&amp;ncbi_uid=247041&amp;link_uid=PMC151252">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1330.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3593087/figure/F3/" ref="ordinalpos=1330&amp;ncbi_uid=4705646&amp;link_uid=PMC3593087" href="/pmc/articles/PMC3593087/"><img src="/pmc/articles/instance/3593087/bin/nihms341773f3.gif" src-large="/pmc/articles/instance/3593087/bin/nihms341773f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3593087/" ref="ordinalpos=1330&amp;ncbi_uid=4705646&amp;link_uid=PMC3593087" image-link="/pmc/articles/PMC3593087/figure/F3/" class="imagepopup">Figure 3.  From: Contributions of the Insulin/Insulin-Like Growth Factor-1 Axis to Diabetic Osteopathy. </a></p><div class="supp"><p class="details"><div>Potential cross-talk between insulin/IGF-1, BMP2, and Wnt <span class="highlight" style="background-color:">signaling</span> pathways. BMP2 receptor <span class="highlight" style="background-color:">signaling</span> and the MEK/ERK branch of the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> converge on ERK1/2 kinase, which can modulate the activity of the master osteoblastogenic transcription factor, Runx2. Wnt and the PI3-K/Akt branch of the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> converge on GSK-3b, inhibiting GSK-3b kinase activity and reducing its constitutive phosphorylation of β-catenin. Unphosphorylated β-catenin is not readily degraded and can accumulate in the nucleus, where it acts as a pro-osteoblastogenic transcription factor by regulating transcription of Runx2 and other genes.</div></p></div><div class="aux"><div class="resc">John L. Fowlkes, et al. J Diabetes Metab. 2011 November 25;1(3):S1-003.</div><p class="links"><a class="dblinks" href="/pubmed/23484069" ref="ordinalpos=1330&amp;ncbi_uid=4705646&amp;link_uid=23484069">Citation</a><a class="dblinks" href="/pmc/articles/PMC3593087/" ref="ordinalpos=1330&amp;ncbi_uid=4705646&amp;link_uid=PMC3593087">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1331.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4204332/figure/F1/" ref="ordinalpos=1331&amp;ncbi_uid=6739646&amp;link_uid=PMC4204332" href="/pmc/articles/PMC4204332/"><img src="/pmc/articles/instance/4204332/bin/nihms-583651-f0001.gif" src-large="/pmc/articles/instance/4204332/bin/nihms-583651-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4204332/" ref="ordinalpos=1331&amp;ncbi_uid=6739646&amp;link_uid=PMC4204332" image-link="/pmc/articles/PMC4204332/figure/F1/" class="imagepopup">Fig. 1.  From: The Parkinson's Disease-Associated Protein ?-Synuclein Disrupts Stress <span class="highlight" style="background-color:">Signaling</span> - A Possible Implication for Methamphetamine Use?. </a></p><div class="supp"><p class="details"><div>α-Syn disrupts cell <span class="highlight" style="background-color:">signaling</span>. (A) Model for how α-syn disrupts the cell wall integrity <span class="highlight" style="background-color:">pathway</span> in yeast. This <span class="highlight" style="background-color:">pathway</span> is a mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> that is similar to the p38 stress-activated <span class="highlight" style="background-color:">pathway</span> in humans. (B) Model for how α-syn disrupts the p38 stress-activated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in humans. In (A) and (B), α-syn inhibits the upstream polo-like kinase if and only if one of the following two conditions is met: (i) α-syn is expressed at high levels or (ii) cells are subjected to high temperatures (&gt; 40 °C). If conditions (i) and (ii) are met, then inhibition of stress <span class="highlight" style="background-color:">signaling</span> could be very robust. αS, α-synuclein; GEF, guanine exchange factor; GAP, GTPase-activating protein; MAPK, mitogen-activated protein kinase; Mpk1, yeast MAP kinase; Pkc1, yeast protein kinase C; Plks, human polo-like kinase; ↑, indicates increased expression and/or increased activity; ↓, indicates decreased expression and/or decreased activity.</div></p></div><div class="aux"><div class="resc">Shaoxiao Wang, et al. Microb Cell. ;1(4):131-132.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1331&amp;ncbi_uid=6739646&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4204332/" ref="ordinalpos=1331&amp;ncbi_uid=6739646&amp;link_uid=PMC4204332">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1332.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3502657/figure/F2/" ref="ordinalpos=1332&amp;ncbi_uid=5542805&amp;link_uid=PMC3502657" href="/pmc/articles/PMC3502657/"><img src="/pmc/articles/instance/3502657/bin/nihms404774f2.gif" src-large="/pmc/articles/instance/3502657/bin/nihms404774f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3502657/" ref="ordinalpos=1332&amp;ncbi_uid=5542805&amp;link_uid=PMC3502657" image-link="/pmc/articles/PMC3502657/figure/F2/" class="imagepopup">Figure 2. Overview of germline <span class="highlight" style="background-color:">signaling</span> in C. elegans aging.  From: Lessons from C. elegans: <span class="highlight" style="background-color:">Signaling</span> pathways for longevity. </a></p><div class="supp"><p class="details"><div>Lifespan extension via germline removal, for example by mutation of <i>glp-1</i>/Notch, depends on at least four <span class="highlight" style="background-color:">signaling</span> mechanisms: reduced TOR <span class="highlight" style="background-color:">signaling</span> (likely to regulate autophagy), DAF-16/FOXO regulation, increased steroid <span class="highlight" style="background-color:">signaling</span> via the DAF-36/DAF-9/DAF-12 <span class="highlight" style="background-color:">pathway</span> (which regulates various genes), and increased NHR-80/HNF-4 <span class="highlight" style="background-color:">signaling</span> (which enhances fatty-acid desaturation). Of these mechanisms, the most-studied mechanism is DAF-16/FOXO regulation. Germline-less animals require specific cofactors (not required in <i>daf-2/</i>InR mutants) for activation of DAF-16/FOXO in the nucleus of intestinal cells. DAF-12/FXR, TCER-1/TCERG1, KRI-1/KRIT-1 and PHI-62 all function specifically in germline-less animals, whereas DAF-18/PTEN, SMK-1/SMEK and HSF-1 are common between the insulin/IGF-1 and germline <span class="highlight" style="background-color:">signaling</span> pathways. Cell-nonautonomous regulation of DAF-16/FOXO is also mediated through a microRNA, <i>mir-71,</i> which is produced in neurons.<br />Factors with longevity-promoting effects are in green and those with lifespan-limiting effects are in red. TOR effects are listed in grey to indicate that these links are presently inferred. Transcription factors are in boxes. See text for details.</div></p></div><div class="aux"><div class="resc">Louis R. Lapierre, et al. Trends Endocrinol Metab. ;23(12):637-644.</div><p class="links"><a class="dblinks" href="/pubmed/22939742" ref="ordinalpos=1332&amp;ncbi_uid=5542805&amp;link_uid=22939742">Citation</a><a class="dblinks" href="/pmc/articles/PMC3502657/" ref="ordinalpos=1332&amp;ncbi_uid=5542805&amp;link_uid=PMC3502657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1333.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151022/figure/F1/" ref="ordinalpos=1333&amp;ncbi_uid=242652&amp;link_uid=PMC151022" href="/pmc/articles/PMC151022/"><img src="/pmc/articles/instance/151022/bin/JCI0215970.f1.gif" src-large="/pmc/articles/instance/151022/bin/JCI0215970.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151022/" ref="ordinalpos=1333&amp;ncbi_uid=242652&amp;link_uid=PMC151022" image-link="/pmc/articles/PMC151022/figure/F1/" class="imagepopup">Figure 1.  From: TGF-? antagonists: Why suppress a tumor suppressor?. </a></p><div class="supp"><p class="details"><div>The TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TGF-βs bind and activate the TGF-β receptor complex, which transmits signal predominantly via activation and nuclear translocation of Smad proteins. However, several Smad-independent <span class="highlight" style="background-color:">signaling</span> pathways are also activated by this receptor complex, and the outcome of Smad <span class="highlight" style="background-color:">signaling</span> can be modified by interaction with other <span class="highlight" style="background-color:">signaling</span> pathways (1).</div></p></div><div class="aux"><div class="resc">Rosemary J. Akhurst. J Clin Invest. 2002 June 15;109(12):1533-1536.</div><p class="links"><a class="dblinks" href="/pubmed/12070299" ref="ordinalpos=1333&amp;ncbi_uid=242652&amp;link_uid=12070299">Citation</a><a class="dblinks" href="/pmc/articles/PMC151022/" ref="ordinalpos=1333&amp;ncbi_uid=242652&amp;link_uid=PMC151022">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1334.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3952157/figure/fig3/" ref="ordinalpos=1334&amp;ncbi_uid=5913345&amp;link_uid=PMC3952157" href="/pmc/articles/PMC3952157/"><img src="/pmc/articles/instance/3952157/bin/mbo0011417670003.gif" src-large="/pmc/articles/instance/3952157/bin/mbo0011417670003.jpg" alt="FIG 3 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3952157/" ref="ordinalpos=1334&amp;ncbi_uid=5913345&amp;link_uid=PMC3952157" image-link="/pmc/articles/PMC3952157/figure/fig3/" class="imagepopup">FIG 3 .  From: Virus-Induced Transcriptional Changes in the Brain Include the Differential Expression of Genes Associated with Interferon, Apoptosis, Interleukin 17 Receptor A, and Glutamate <span class="highlight" style="background-color:">Signaling</span> as Well as Flavivirus-Specific Upregulation of tRNA Synthetases. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis identifies cellular <span class="highlight" style="background-color:">signaling</span> pathways that are associated with genes that are up-regulated in infections with JEV, WNV, and reovirus. IPA was performed on genes that were differentially up-regulated following virus infection (change, &gt;2-fold; <i>P</i> &lt; 0.001). IPA identified genes involved in IFN <span class="highlight" style="background-color:">signaling</span> (A), death receptor <span class="highlight" style="background-color:">signaling</span> (B), and IL-17RA <span class="highlight" style="background-color:">signaling</span> (C) as being common to infections with JEV, WNV, and reovirus. The charts are adapted from IPA results. Shaded boxes represent genes that were differentially up-regulated following virus infection.</div></p></div><div class="aux"><div class="resc">Penny Clarke, et al. mBio. 2014 Mar-Apr;5(2):e00902-14.</div><p class="links"><a class="dblinks" href="/pubmed/24618253" ref="ordinalpos=1334&amp;ncbi_uid=5913345&amp;link_uid=24618253">Citation</a><a class="dblinks" href="/pmc/articles/PMC3952157/" ref="ordinalpos=1334&amp;ncbi_uid=5913345&amp;link_uid=PMC3952157">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1335.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2516985/figure/fig8/" ref="ordinalpos=1335&amp;ncbi_uid=1489164&amp;link_uid=PMC2516985" href="/pmc/articles/PMC2516985/"><img src="/pmc/articles/instance/2516985/bin/zbc0350846720008.gif" src-large="/pmc/articles/instance/2516985/bin/zbc0350846720008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2516985/" ref="ordinalpos=1335&amp;ncbi_uid=1489164&amp;link_uid=PMC2516985" image-link="/pmc/articles/PMC2516985/figure/fig8/" class="imagepopup">FIGURE 8. From: Respiratory Syncytial Virus Induces RelA Release from Cytoplasmic 100-kDa  NF-?B2 Complexes via a Novel Retinoic Acid-inducible  Gene-I?NF-?B-inducing Kinase <span class="highlight" style="background-color:">Signaling</span>  <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Cross-talk <span class="highlight" style="background-color:">pathway</span> in RSV infection.</b> Shown is a schematic model that  shows the <span class="highlight" style="background-color:">signaling</span> cascade downstream of RIG-I that diverges to activate the  canonical, noncanonical, and cross-talk pathways. The presence of RSV dsRNA is  sensed by the cytosolic RIG-I helicase, which associates with the  mitochondrial bound MAVS via heterotypic CARD domain interactions. This  complex mediates activation of the typical IKK resulting in RelA release from  IκBα-sequestered complexes. In parallel, the NIK·IKKα  complex is activated, resulting in activation of the noncanonical and  cross-talk <span class="highlight" style="background-color:">pathway</span> by stimulating the 100-kDa NF-κB2 proteolysis.  <i>Mito</i>, mitochondria; <i>p65</i>, RelA.</div></p></div><div class="aux"><div class="resc">Ping Liu, et al. J Biol Chem. 2008 August 22;283(34):23169-23178.</div><p class="links"><a class="dblinks" href="/pubmed/18550535" ref="ordinalpos=1335&amp;ncbi_uid=1489164&amp;link_uid=18550535">Citation</a><a class="dblinks" href="/pmc/articles/PMC2516985/" ref="ordinalpos=1335&amp;ncbi_uid=1489164&amp;link_uid=PMC2516985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1336.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3859635/figure/pone-0082513-g002/" ref="ordinalpos=1336&amp;ncbi_uid=5600038&amp;link_uid=PMC3859635" href="/pmc/articles/PMC3859635/"><img src="/pmc/articles/instance/3859635/bin/pone.0082513.g002.gif" src-large="/pmc/articles/instance/3859635/bin/pone.0082513.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3859635/" ref="ordinalpos=1336&amp;ncbi_uid=5600038&amp;link_uid=PMC3859635" image-link="/pmc/articles/PMC3859635/figure/pone-0082513-g002/" class="imagepopup">Figure 2. Biological function and canonical <span class="highlight" style="background-color:">pathway</span> analysis of <span class="highlight" style="background-color:">signaling</span> pathways regulated by identified phosphopeptides in NB10 cells..  From: Quantitative Phosphoproteomic Analysis Identifies Activation of the RET and IGF-1R/IR <span class="highlight" style="background-color:">Signaling</span> Pathways in Neuroblastoma. </a></p><div class="supp"><p class="details"><div>(A) Ingenuity Systems <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) software was used to determine the highest biologically active functions in the NB10 cell phosphoproteome. (B) IPA software was also used to determine canonical pathways highly represented in the NB10 phosphoproteome. Phosphopeptides found to be relatively overexpressed in the NB10 cell line compared to the NPC cell line were analyzed.</div></p></div><div class="aux"><div class="resc">Bradley D. DeNardo, et al. PLoS One. 2013;8(12):e82513.</div><p class="links"><a class="dblinks" href="/pubmed/24349301" ref="ordinalpos=1336&amp;ncbi_uid=5600038&amp;link_uid=24349301">Citation</a><a class="dblinks" href="/pmc/articles/PMC3859635/" ref="ordinalpos=1336&amp;ncbi_uid=5600038&amp;link_uid=PMC3859635">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1337.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4032892/figure/F1/" ref="ordinalpos=1337&amp;ncbi_uid=6202169&amp;link_uid=PMC4032892" href="/pmc/articles/PMC4032892/"><img src="/pmc/articles/instance/4032892/bin/fonc-04-00108-g001.gif" src-large="/pmc/articles/instance/4032892/bin/fonc-04-00108-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4032892/" ref="ordinalpos=1337&amp;ncbi_uid=6202169&amp;link_uid=PMC4032892" image-link="/pmc/articles/PMC4032892/figure/F1/" class="imagepopup">Figure 1.  From: Targeting the PI3K/mTOR <span class="highlight" style="background-color:">Pathway</span> in Pediatric Hematologic Malignancies. </a></p><div class="supp"><p class="details"><div><b>Schema of PI3K/mTOR and other oncogenic <span class="highlight" style="background-color:">signaling</span> pathways in hematologic malignancies</b>. Inhibitors (i) of these <span class="highlight" style="background-color:">signaling</span> proteins under current preclinical or clinical study with potential therapeutic relevance for pediatric hematologic malignancies are delineated. Red = inhibitors of PI3K/mTOR <span class="highlight" style="background-color:">pathway</span> signal transduction protein(s).</div></p></div><div class="aux"><div class="resc">Sarah K. Tasian, et al. Front Oncol. 2014;4:108.</div><p class="links"><a class="dblinks" href="/pubmed/24904824" ref="ordinalpos=1337&amp;ncbi_uid=6202169&amp;link_uid=24904824">Citation</a><a class="dblinks" href="/pmc/articles/PMC4032892/" ref="ordinalpos=1337&amp;ncbi_uid=6202169&amp;link_uid=PMC4032892">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1338.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204078/figure/F5/" ref="ordinalpos=1338&amp;ncbi_uid=3551324&amp;link_uid=PMC3204078" href="/pmc/articles/PMC3204078/"><img src="/pmc/articles/instance/3204078/bin/4182fig5.gif" src-large="/pmc/articles/instance/3204078/bin/4182fig5.jpg" alt="FIGURE 5:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204078/" ref="ordinalpos=1338&amp;ncbi_uid=3551324&amp;link_uid=PMC3204078" image-link="/pmc/articles/PMC3204078/figure/F5/" class="imagepopup">FIGURE 5:.  From: The SHP-1 protein tyrosine phosphatase negatively modulates Akt <span class="highlight" style="background-color:">signaling</span> in the ghrelin/GHSR1a system. </a></p><div class="supp"><p class="details"><div>Model of SHP-1 activation and modulation on ghrelin <span class="highlight" style="background-color:">signaling</span> to Akt. Ghrelin binding to GHS-R1a activates the generation of the second messenger PtdIns(3,4,5)P<sub>3</sub> (PIP3) by PI3K activation through G<sub>i/o</sub> protein–dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Akt translocates to the plasma membrane by binding to PIP3, where it is phosphorylated at Y by the membrane-bound cSrc. Activation of cSrc induced phosphorylation of SHP-1, attenuation of p85 subunit from PI3K and PDK1, and phosphorylation of Akt. SHP-1 activation regulates cSrc activity by attenuation of Y416 phosphorylation. Once the receptor is activated, a second <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is mediated by β-arrestins 1 and 2, involving the recruitment of GHS-R1a, cSrc, SHP-1, and Akt into a β-arrestin–scaffolded complex.</div></p></div><div class="aux"><div class="resc">Maria Lodeiro, et al. Mol Biol Cell. 2011 November 1;22(21):4182-4191.</div><p class="links"><a class="dblinks" href="/pubmed/21900501" ref="ordinalpos=1338&amp;ncbi_uid=3551324&amp;link_uid=21900501">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204078/" ref="ordinalpos=1338&amp;ncbi_uid=3551324&amp;link_uid=PMC3204078">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1339.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3920940/figure/fig7/" ref="ordinalpos=1339&amp;ncbi_uid=5814134&amp;link_uid=PMC3920940" href="/pmc/articles/PMC3920940/"><img src="/pmc/articles/instance/3920940/bin/cddis2013407f7.gif" src-large="/pmc/articles/instance/3920940/bin/cddis2013407f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3920940/" ref="ordinalpos=1339&amp;ncbi_uid=5814134&amp;link_uid=PMC3920940" image-link="/pmc/articles/PMC3920940/figure/fig7/" class="imagepopup">Figure 7.  From: Acquisition of epithelial-mesenchymal transition and cancer stem cell phenotypes is associated with activation of the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> in prostate cancer radioresistance. </a></p><div class="supp"><p class="details"><div>Diagram showing the model proposed for the association of EMT, CSCs and the PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in CaP radioresistance and the effect of possible action of BEZ235 on radiosensitivity. (<b>a</b>) RT can induce CaP-RR cells with EMT and enhanced CSC phenotypes, and activation of the PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, resulting in CaP metastasis and recurrence after RT. (<b>b</b>) A dual PI3K/mTOR inhibitor BEZ235 combined with RT can inactivate the PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, reducing EMT and CSC phenotypes and leading to increased radiosensitivity. All data used in RR cell lines were based on cells between 5 and 6 weeks post radiation treatment</div></p></div><div class="aux"><div class="resc">L Chang, et al. Cell Death Dis. 2013 October;4(10):e875.</div><p class="links"><a class="dblinks" href="/pubmed/24157869" ref="ordinalpos=1339&amp;ncbi_uid=5814134&amp;link_uid=24157869">Citation</a><a class="dblinks" href="/pmc/articles/PMC3920940/" ref="ordinalpos=1339&amp;ncbi_uid=5814134&amp;link_uid=PMC3920940">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1340.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3543749/figure/Fig1/" ref="ordinalpos=1340&amp;ncbi_uid=4531748&amp;link_uid=PMC3543749" href="/pmc/articles/PMC3543749/"><img src="/pmc/articles/instance/3543749/bin/11357_2011_9340_Fig1_HTML.gif" src-large="/pmc/articles/instance/3543749/bin/11357_2011_9340_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3543749/" ref="ordinalpos=1340&amp;ncbi_uid=4531748&amp;link_uid=PMC3543749" image-link="/pmc/articles/PMC3543749/figure/Fig1/" class="imagepopup">Fig. 1.  From: Gene set analysis of GWAS data for human longevity highlights the relevance of the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> and telomere maintenance pathways. </a></p><div class="supp"><p class="details"><div>Insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> consists of the core components IGF1R/IR/IRR, IRS, PI3K, AKT/SGK, FOXO and SIRT, and proteins that have a direct activating or inhibiting effect on these proteins. The <i>small closed circles</i> (containing Ac, P, or Ub) indicate an activating effect of the posttranslational modification on the protein, while the <i>small dashed circles</i> indicate an inhibiting effect. The <i>straight arrows</i> pointing to these <i>small circles</i> indicate an activating effect on the posttranslational modification, while the <i>dashed arrows</i> indicate an inhibiting effect. <i>Ac</i> acetylation, <i>P</i> phosphorylation, <i>Ub</i> ubiquitylation</div></p></div><div class="aux"><div class="resc">Joris Deelen, et al. Age (Dordr). 2013 February;35(1):235-249.</div><p class="links"><a class="dblinks" href="/pubmed/22113349" ref="ordinalpos=1340&amp;ncbi_uid=4531748&amp;link_uid=22113349">Citation</a><a class="dblinks" href="/pmc/articles/PMC3543749/" ref="ordinalpos=1340&amp;ncbi_uid=4531748&amp;link_uid=PMC3543749">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1341.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3783342/figure/F3/" ref="ordinalpos=1341&amp;ncbi_uid=5335100&amp;link_uid=PMC3783342" href="/pmc/articles/PMC3783342/"><img src="/pmc/articles/instance/3783342/bin/nihms493687f3.gif" src-large="/pmc/articles/instance/3783342/bin/nihms493687f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3783342/" ref="ordinalpos=1341&amp;ncbi_uid=5335100&amp;link_uid=PMC3783342" image-link="/pmc/articles/PMC3783342/figure/F3/" class="imagepopup">Fig. 3.  From: Novel strategies targeting cancer stem cells through phytochemicals and their analogs. </a></p><div class="supp"><p class="details"><div>Several key factors/<span class="highlight" style="background-color:">signaling</span> pathways in CSCs that are targets of phytochemicals. Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation results in cleavage of Notch by γ-secretase into Notch intracellular domain (<i>NICD</i>) that translocates to nucleus, a process that is known to be inhibited by multiple phytochemicals. Receptor tyrosine kinases (<i>RTKs</i>), such as PDGFR, IGFR, EGFR etc., activate PI3K–Akt–mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and this <span class="highlight" style="background-color:">pathway</span> is also known to be regulated by phytochemicals at different steps. Binding of wnt ligand to its receptor Frizzled activates Dishevelled (<i>DSH</i>) leading to attenuation of GSK-3β-mediated repression of β-catenin. β-Catenin also translocates to nucleus resulting in induction of downstream <span class="highlight" style="background-color:">signaling</span> molecules, and this <span class="highlight" style="background-color:">pathway</span> is modulated by phytochemicals as well. Inhibition of these multiple <span class="highlight" style="background-color:">signaling</span> pathways represents a mechanism by which phytochemicals influence the phenotype of CSCs</div></p></div><div class="aux"><div class="resc">Prasad Dandawate, et al. Drug Deliv Transl Res. 2013 April 1;3(2):165-182.</div><p class="links"><a class="dblinks" href="/pubmed/24076568" ref="ordinalpos=1341&amp;ncbi_uid=5335100&amp;link_uid=24076568">Citation</a><a class="dblinks" href="/pmc/articles/PMC3783342/" ref="ordinalpos=1341&amp;ncbi_uid=5335100&amp;link_uid=PMC3783342">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1342.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3967167/figure/viruses-05-03192-f006/" ref="ordinalpos=1342&amp;ncbi_uid=5948543&amp;link_uid=PMC3967167" href="/pmc/articles/PMC3967167/"><img src="/pmc/articles/instance/3967167/bin/viruses-05-03192-g006.gif" src-large="/pmc/articles/instance/3967167/bin/viruses-05-03192-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3967167/" ref="ordinalpos=1342&amp;ncbi_uid=5948543&amp;link_uid=PMC3967167" image-link="/pmc/articles/PMC3967167/figure/viruses-05-03192-f006/" class="imagepopup">Figure 6.  From: Make Yourself at Home: Viral Hijacking of the PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Time schedule of viral-induced interference with PI3K/Akt <span class="highlight" style="background-color:">signaling</span> based on the time point of activation and benefit during viral life cycles. Early activation of the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during the first contact with the cell or a few minutes after entry can lead to both short-term cellular answers to facilitate viral entry and long-term reactions (lasting for hours or even days) to support viral and cellular splicing, viral translation and cell survival. Later activation, hours or days post-infection, can likewise promote such long-term effects.</div></p></div><div class="aux"><div class="resc">Nora Diehl, et al. Viruses. 2013 December;5(12):3192-3212.</div><p class="links"><a class="dblinks" href="/pubmed/24351799" ref="ordinalpos=1342&amp;ncbi_uid=5948543&amp;link_uid=24351799">Citation</a><a class="dblinks" href="/pmc/articles/PMC3967167/" ref="ordinalpos=1342&amp;ncbi_uid=5948543&amp;link_uid=PMC3967167">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1343.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3895693/figure/F6/" ref="ordinalpos=1343&amp;ncbi_uid=5721485&amp;link_uid=PMC3895693" href="/pmc/articles/PMC3895693/"><img src="/pmc/articles/instance/3895693/bin/1476-4598-13-6-6.gif" src-large="/pmc/articles/instance/3895693/bin/1476-4598-13-6-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3895693/" ref="ordinalpos=1343&amp;ncbi_uid=5721485&amp;link_uid=PMC3895693" image-link="/pmc/articles/PMC3895693/figure/F6/" class="imagepopup">Figure 6.  From: Reciprocal regulation of MicroRNA-99a and insulin-like growth factor I receptor <span class="highlight" style="background-color:">signaling</span> in oral squamous cell carcinoma cells. </a></p><div class="supp"><p class="details"><div><b>Negative regulation of miR-99a expression by the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> Downregulation of miR-99a in response to IGF1 treatment. After serum starvation, cells were treated with vehicle, 10% FBS or 10 nM IGF1. <b>(B)</b> Immunoblot analysis revealed increased levels of IGF1R protein following IGF1-induced downregulation of miR-99a. After serum starvation, cells were treated with vehicle, 10% FBS or 10 nM IGF1. The protein levels were normalized against an internal control α-tubulin. Ratios were determined by dividing the normalized protein levels in OEC-M1 cells with different conditions to that in complete medium (CM). The means in the lower panel were measured by averaging the ratios from independent blots. <b>(C)</b> The PI3K inhibitor LY294002 and <b>(D)</b> MAPK kinase inhibitor PD98059 suppressed the IGF1-induced downregulation of miR-99a. After serum starvation, cells were treated with vehicle, 10 nM IGF1, or combination of LY294002/PD98059 and IGF1. Expression of miR-99a was normalized against an endogenous control U6. The relative expression of miR-99a was determined by normalizing the expression of miR-99a in OEC-M1 cells in different conditions to that in cells in complete medium (CM). Bar, SE; *p &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.001. <b>(E)</b> Proposed model of the relationship between the activation of the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and miR-99a expression. The activation of the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> initiated the inhibitory signal leading to decreased miR-99a expression. Following the suppression of the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by inhibitors, or its silencing by the removal of stimulation, we observed increased miR-99a expression because of the removal of the inhibitory signal. This reciprocal regulation of miR-99a and IGF1R <span class="highlight" style="background-color:">signaling</span> augmented the activation of the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in response to IGF1 stimulation and accelerated its inactivation following the removal of stimulation.</div></p></div><div class="aux"><div class="resc">Yi-Chen Yen, et al. Mol Cancer. 2014;13:6-6.</div><p class="links"><a class="dblinks" href="/pubmed/24410957" ref="ordinalpos=1343&amp;ncbi_uid=5721485&amp;link_uid=24410957">Citation</a><a class="dblinks" href="/pmc/articles/PMC3895693/" ref="ordinalpos=1343&amp;ncbi_uid=5721485&amp;link_uid=PMC3895693">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1344.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355085/figure/pgen-1002725-g004/" ref="ordinalpos=1344&amp;ncbi_uid=4013464&amp;link_uid=PMC3355085" href="/pmc/articles/PMC3355085/"><img src="/pmc/articles/instance/3355085/bin/pgen.1002725.g004.gif" src-large="/pmc/articles/instance/3355085/bin/pgen.1002725.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355085/" ref="ordinalpos=1344&amp;ncbi_uid=4013464&amp;link_uid=PMC3355085" image-link="/pmc/articles/PMC3355085/figure/pgen-1002725-g004/" class="imagepopup">Figure 4. Ack anti-apoptotic function is stimulated by EGF receptor/Ras <span class="highlight" style="background-color:">signaling</span>..  From: Drosophila Activated Cdc42 Kinase Has an Anti-Apoptotic Function. </a></p><div class="supp"><p class="details"><div>The <i>Drosophila</i> eye size assay is used to determine how Ack functions with respect to EGF receptor/MAPK <span class="highlight" style="background-color:">signaling</span>. (A) Hid<sup>Ala5</sup> control eye. (B) Overexpression of Ack results in suppression of PCD induced by hid<sup>Ala5</sup> indicating that Ack suppression of apoptosis is independent of hid phosphorylation by MAPK. (C) Kinase inactive Ack fails to suppress PCD, showing that kinase activity is required for Ack suppression of hid<sup>Ala5</sup> induced PCD. (D and G) Increased <span class="highlight" style="background-color:">signaling</span> through the MAPK <span class="highlight" style="background-color:">pathway</span> by single copy loss of <i>argos</i> (D) or <i>Gap1</i> (G) fails to suppress apoptosis due to the inability of MAPK to phosphorylate and inhibit hid<sup>Ala5</sup>. (E and H) Ack overexpression combined with single copy loss of <i>argos</i> (E) or <i>Gap1</i> (H) further suppresses hid<sup>Ala5</sup> compared to panel B, showing that Ack can be activated by EGF receptor or Ras <span class="highlight" style="background-color:">signaling</span>. (F and I) Ack<sup>K156A</sup> combined with single copy loss of <i>argos</i> (F) or <i>Gap1</i> (I) does not suppress hid<sup>Ala5</sup> PCD. (J) A schematic of the EGF initiated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through MAPK is shown. The MAPK <span class="highlight" style="background-color:">pathway</span> regulates hid induced PCD by phosphorylating and inhibiting hid. A hid mutant having five serine/threonine phosphorylation sites mutated to alanine (hid<sup>Ala5</sup>) is unable to be inhibited by the MAPK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jessica A. Schoenherr, et al. PLoS Genet. 2012 May;8(5):e1002725.</div><p class="links"><a class="dblinks" href="/pubmed/22615583" ref="ordinalpos=1344&amp;ncbi_uid=4013464&amp;link_uid=22615583">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355085/" ref="ordinalpos=1344&amp;ncbi_uid=4013464&amp;link_uid=PMC3355085">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1345.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3672007/figure/F2/" ref="ordinalpos=1345&amp;ncbi_uid=4964462&amp;link_uid=PMC3672007" href="/pmc/articles/PMC3672007/"><img src="/pmc/articles/instance/3672007/bin/1471-2407-13-245-2.gif" src-large="/pmc/articles/instance/3672007/bin/1471-2407-13-245-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3672007/" ref="ordinalpos=1345&amp;ncbi_uid=4964462&amp;link_uid=PMC3672007" image-link="/pmc/articles/PMC3672007/figure/F2/" class="imagepopup">Figure 2.  From: IR/IGF1R <span class="highlight" style="background-color:">signaling</span> as potential target for treatment of high-grade osteosarcoma. </a></p><div class="supp"><p class="details"><div><b>Ingenuity pathways analysis canonical <span class="highlight" style="background-color:">pathway</span> IGF1 <span class="highlight" style="background-color:">signaling</span></b> This figure shows the IGF1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, with significantly upregulated genes in red, downregulated genes in green, and genes that did not meet our criteria for significance in gray. The left part of the symbols shows the analysis of osteosarcoma cell lines as compared with mesenchymal stem cells, the right part as compared with osteoblasts. Most consensus in gene expression is found upstream IGF1R <span class="highlight" style="background-color:">signaling</span>, in the expression of the IGF binding proteins.</div></p></div><div class="aux"><div class="resc">Marieke L Kuijjer, et al. BMC Cancer. 2013;13:245-245.</div><p class="links"><a class="dblinks" href="/pubmed/23688189" ref="ordinalpos=1345&amp;ncbi_uid=4964462&amp;link_uid=23688189">Citation</a><a class="dblinks" href="/pmc/articles/PMC3672007/" ref="ordinalpos=1345&amp;ncbi_uid=4964462&amp;link_uid=PMC3672007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1346.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4280060/figure/f3-medscimonit-20-2746/" ref="ordinalpos=1346&amp;ncbi_uid=7043654&amp;link_uid=PMC4280060" href="/pmc/articles/PMC4280060/"><img src="/pmc/articles/instance/4280060/bin/medscimonit-20-2746-g003.gif" src-large="/pmc/articles/instance/4280060/bin/medscimonit-20-2746-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4280060/" ref="ordinalpos=1346&amp;ncbi_uid=7043654&amp;link_uid=PMC4280060" image-link="/pmc/articles/PMC4280060/figure/f3-medscimonit-20-2746/" class="imagepopup">Figure 3.  From: NF-κB <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Confers Neuroblastoma Cells Migration and Invasion Ability via the Regulation of CXCR4. </a></p><div class="supp"><p class="details"><div>The macrophages-increased upregulation of CXCR4 expression was dependent on the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the co-culture system (<b>A1, A2</b>), (<b>B1, B2</b>). The SK-N-BE(2) or SH-SY5Y cells expressed more CXCR4 after co-culture with THP-1-derived macrophage cells. The upregulation of CXCR4 expression was inhibited after treatment with PDTC. NF-κB(p65) and CXCR4 expression was measured by Western Blot. β-actin was used as the loading control. CO: co-culture. *P&lt;0.05 <i>vs.</i> co-culture group. Data are expressed as mean ±SD.</div></p></div><div class="aux"><div class="resc">Yunlai Zhi, et al. Med Sci Monit. 2014;20:2746-2752.</div><p class="links"><a class="dblinks" href="/pubmed/25527973" ref="ordinalpos=1346&amp;ncbi_uid=7043654&amp;link_uid=25527973">Citation</a><a class="dblinks" href="/pmc/articles/PMC4280060/" ref="ordinalpos=1346&amp;ncbi_uid=7043654&amp;link_uid=PMC4280060">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1347.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4060082/figure/f6-bt-22-3-200/" ref="ordinalpos=1347&amp;ncbi_uid=6358623&amp;link_uid=PMC4060082" href="/pmc/articles/PMC4060082/"><img src="/pmc/articles/instance/4060082/bin/bt-22-3-200f6.gif" src-large="/pmc/articles/instance/4060082/bin/bt-22-3-200f6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4060082/" ref="ordinalpos=1347&amp;ncbi_uid=6358623&amp;link_uid=PMC4060082" image-link="/pmc/articles/PMC4060082/figure/f6-bt-22-3-200/" class="imagepopup">Fig. 6. From: N-(p-Coumaryol)-Tryptamine Suppresses the Activation of JNK/c-Jun <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in LPS-Challenged RAW264.7 Cells. </a></p><div class="supp"><p class="details"><div>Effect of N-(<i>p</i>-coumaryol) tryptamine on LPS-induced activation of Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RAW264.7 macrophage cells. (A) representative immunoblots, (B) quantitative analysis of immunoblots. <i>N</i>-(<i>p</i>-coumaryol) tryptamine appeared to significantly attenuate Akt phosphorylation in RAW264.7 macrophage cells. The images shown are representative of three independent experiments and the values are expressed as mean ± SD for three experiments. <sup>*</sup><i>p</i>&lt;0.05 and <sup>**</sup><i>p</i>&lt;0.01 indicate statistically significant differences from treatments with control alone. CT stands for <i>N</i>-(<i>p</i>coumaryol) tryptamine.</div></p></div><div class="aux"><div class="resc">Van Anh Vo, et al. Biomol Ther (Seoul). 2014 May;22(3):200-206.</div><p class="links"><a class="dblinks" href="/pubmed/25009700" ref="ordinalpos=1347&amp;ncbi_uid=6358623&amp;link_uid=25009700">Citation</a><a class="dblinks" href="/pmc/articles/PMC4060082/" ref="ordinalpos=1347&amp;ncbi_uid=6358623&amp;link_uid=PMC4060082">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1348.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3062326/figure/F5/" ref="ordinalpos=1348&amp;ncbi_uid=3413057&amp;link_uid=PMC3062326" href="/pmc/articles/PMC3062326/"><img src="/pmc/articles/instance/3062326/bin/zh89991066200005.gif" src-large="/pmc/articles/instance/3062326/bin/zh89991066200005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3062326/" ref="ordinalpos=1348&amp;ncbi_uid=3413057&amp;link_uid=PMC3062326" image-link="/pmc/articles/PMC3062326/figure/F5/" class="imagepopup">Figure 5.  From: GABP controls a critical transcription regulatory module that is essential for maintenance and differentiation of hematopoietic stem/progenitor cells. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span> analysis of differentially expressed genes because of GABPα ablation in HSCs</b>. (A-D) Flt3<sup>−</sup>LSKs were sorted from bone marrow cells of 2 pairs of Mx1Cre-GABPα<sup>FL/−</sup> and control mice and subjected to microarray analysis using GeneChip Mouse GENE 1.0 ST arrays. Genes exhibited significant expression changes (&gt; 1.5 fold and <i>P</i> &lt; .05) were analyzed using <span class="highlight" style="background-color:">pathway</span> analysis tools in DAVID bioinformatics resources. Heatmaps of genes in each <span class="highlight" style="background-color:">pathway</span> are shown. Color-coded scale bar indicates Z-score values, denoting number of standard deviations from the mean in each row. Asterisks denote genes containing GABP binding within 2 kb of their TISs as identified by ChIP-Seq. (A) KEGG apoptosis <span class="highlight" style="background-color:">pathway</span>. (B) KEGG Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (C) BioCarta adipocytokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (D) Rac1 cell mobility <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (E) Venn diagram showing the overlap of GABP-bound genes and down-regulated or up-regulated genes because of GABPα deficiency.</div></p></div><div class="aux"><div class="resc">Shuyang Yu, et al. Blood. 2011 February 17;117(7):2166-2178.</div><p class="links"><a class="dblinks" href="/pubmed/21139080" ref="ordinalpos=1348&amp;ncbi_uid=3413057&amp;link_uid=21139080">Citation</a><a class="dblinks" href="/pmc/articles/PMC3062326/" ref="ordinalpos=1348&amp;ncbi_uid=3413057&amp;link_uid=PMC3062326">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1349.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3582147/figure/F1/" ref="ordinalpos=1349&amp;ncbi_uid=5670633&amp;link_uid=PMC3582147" href="/pmc/articles/PMC3582147/"><img src="/pmc/articles/instance/3582147/bin/JCI67166.f1.gif" src-large="/pmc/articles/instance/3582147/bin/JCI67166.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3582147/" ref="ordinalpos=1349&amp;ncbi_uid=5670633&amp;link_uid=PMC3582147" image-link="/pmc/articles/PMC3582147/figure/F1/" class="imagepopup">Figure 1. Distinct <span class="highlight" style="background-color:">signaling</span> pathways mediate insulin effects on nitric oxide and endothelin in vascular endothelial cells..  From: Insulin resistance in the vasculature. </a></p><div class="supp"><p class="details"><div>Endothelial insulin resistance can preferentially impair <span class="highlight" style="background-color:">signaling</span> via the IRS/PI3K <span class="highlight" style="background-color:">pathway</span>, adversely affecting the balance between prothrombotic and antithrombotic <span class="highlight" style="background-color:">signaling</span>. This imbalance could potentially contribute to the increased cardiovascular mortality associated with insulin resistance.</div></p></div><div class="aux"><div class="resc">Kieren J. Mather, et al. J Clin Invest. 2013 March 1;123(3):1003-1004.</div><p class="links"><a class="dblinks" href="/pubmed/23454764" ref="ordinalpos=1349&amp;ncbi_uid=5670633&amp;link_uid=23454764">Citation</a><a class="dblinks" href="/pmc/articles/PMC3582147/" ref="ordinalpos=1349&amp;ncbi_uid=5670633&amp;link_uid=PMC3582147">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1350.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2728515/figure/F6/" ref="ordinalpos=1350&amp;ncbi_uid=1960302&amp;link_uid=PMC2728515" href="/pmc/articles/PMC2728515/"><img src="/pmc/articles/instance/2728515/bin/1471-2121-10-57-6.gif" src-large="/pmc/articles/instance/2728515/bin/1471-2121-10-57-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2728515/" ref="ordinalpos=1350&amp;ncbi_uid=1960302&amp;link_uid=PMC2728515" image-link="/pmc/articles/PMC2728515/figure/F6/" class="imagepopup">Figure 6.  From: All-trans retinoic acid promotes neural lineage entry by pluripotent embryonic stem cells via multiple pathways. </a></p><div class="supp"><p class="details"><div><b>A. Interaction between RA and the ERK <span class="highlight" style="background-color:">pathway</span> during promoted neural specification</b>. FGF receptor tyrosine kinase inhibitor SU5402 and MEK1/2 inhibitors PD184352 and PD0325901, respectively suppresses neural differentiation of ESCs. In RA-treated cultures, PD184352, PD0325901 and RARα-selective antagonist Ro 41-5253 could inhibit the neural specification of ESCs, while SU5402 shows weak neural inhibition effect. RA probably enhances ERK phosphorylation via MEK1/2 or its upstream but not in a manner that is dependent on FGF <span class="highlight" style="background-color:">signaling</span>, and activated ERK1/2 mediates phosphorylation of the MAPK sites of Smad1 (pSmad1<sup>MAPK</sup>), which inhibits the inhibitory function of the bone morphogenetic proteins (BMPs)-Smad1 <span class="highlight" style="background-color:">pathway</span> on neural differentiation. Blockade of RA <span class="highlight" style="background-color:">signaling</span> might affect the activation of the upstream of ERK1/2, which leads to failure of neural specification of ESCs even with the stimulation of FGF <span class="highlight" style="background-color:">signaling</span>. <b>B. Possible roles of Wnt <span class="highlight" style="background-color:">signaling</span> in RA-promoted neural differentiation</b>. RA promotes neural specification probably by up-regulating Sfrp2, which inhibits the Wnt-β-catenin anti-neural <span class="highlight" style="background-color:">pathway</span>. Upon adding LiCl or CHIR99021, which stimulates the β-catenin <span class="highlight" style="background-color:">signaling</span> and/or BMP <span class="highlight" style="background-color:">signaling</span> through inhibition of GSK3, the suppression effects on neural specification of β-catenin <span class="highlight" style="background-color:">pathway</span> can be passed down even with the stimuli of RA. RA also may promote neural differentiation by up-regulating RAR, which may decrease β-catenin/LEF/TCF mediated transactivation. There exists a possibility that RA may suppress the inhibition effects on neural differentiation of Wnt-β-catenin <span class="highlight" style="background-color:">pathway</span> by sequestration of β-catenin. pSmad1<sup>GSK3 </sup>indicates phosphorylation of the GSK3 sites of Smad1.</div></p></div><div class="aux"><div class="resc">Jianfeng Lu, et al. BMC Cell Biol. 2009;10:57-57.</div><p class="links"><a class="dblinks" href="/pubmed/19642999" ref="ordinalpos=1350&amp;ncbi_uid=1960302&amp;link_uid=19642999">Citation</a><a class="dblinks" href="/pmc/articles/PMC2728515/" ref="ordinalpos=1350&amp;ncbi_uid=1960302&amp;link_uid=PMC2728515">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1351.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3197531/figure/pone-0026581-g012/" ref="ordinalpos=1351&amp;ncbi_uid=3301256&amp;link_uid=PMC3197531" href="/pmc/articles/PMC3197531/"><img src="/pmc/articles/instance/3197531/bin/pone.0026581.g012.gif" src-large="/pmc/articles/instance/3197531/bin/pone.0026581.g012.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3197531/" ref="ordinalpos=1351&amp;ncbi_uid=3301256&amp;link_uid=PMC3197531" image-link="/pmc/articles/PMC3197531/figure/pone-0026581-g012/" class="imagepopup">Figure 12. A hypothetical <span class="highlight" style="background-color:">signaling</span> cascades for the crosstalk between β2AR and PI3K/Akt <span class="highlight" style="background-color:">pathway</span> in vivo..  From: ?-Adrenergic Receptor-PI3K <span class="highlight" style="background-color:">Signaling</span> Crosstalk in Mouse Heart: Elucidation of Immediate Downstream <span class="highlight" style="background-color:">Signaling</span> Cascades. </a></p><div class="supp"><p class="details"><div>Activation of ERK1/2-related <span class="highlight" style="background-color:">signaling</span> and the mTOR/P70S6K/S6 axis favors the progression of cardiac hypertrophy. Akt-induced phosphyrylation of GSK-3α/β inhibits its activity, thereby nullifying its anti-hypertrophic effect. Akt-induced phosphorylation of FOXOs also inhibits ist activity, negating its effects on activation of atrogin-1 and MuRF1, which results in decreased protein breakdown (atrophy). These multiple <span class="highlight" style="background-color:">signaling</span> scenarios likely contribute to the pathogenesis of cardiac hypertrophy and/or the anti-atrophic effect following sustained βAR stimulation. “→” represents activation, “⊣” represents inhibition. Solid lines represent data supplied by our study, and dotted lines represent hyperthetical schematics.</div></p></div><div class="aux"><div class="resc">Weizhi Zhang, et al. PLoS One. 2011;6(10):e26581.</div><p class="links"><a class="dblinks" href="/pubmed/22028912" ref="ordinalpos=1351&amp;ncbi_uid=3301256&amp;link_uid=22028912">Citation</a><a class="dblinks" href="/pmc/articles/PMC3197531/" ref="ordinalpos=1351&amp;ncbi_uid=3301256&amp;link_uid=PMC3197531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1352.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3131061/figure/F1/" ref="ordinalpos=1352&amp;ncbi_uid=3562470&amp;link_uid=PMC3131061" href="/pmc/articles/PMC3131061/"><img src="/pmc/articles/instance/3131061/bin/zcm9990923540001.gif" src-large="/pmc/articles/instance/3131061/bin/zcm9990923540001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3131061/" ref="ordinalpos=1352&amp;ncbi_uid=3562470&amp;link_uid=PMC3131061" image-link="/pmc/articles/PMC3131061/figure/F1/" class="imagepopup">Fig. 1. From: Infectious Diseases in Patients with IRAK-4, MyD88, NEMO, or I?B? Deficiency. </a></p><div class="supp"><p class="details"><div>TIR and NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. Immune receptor <span class="highlight" style="background-color:">signaling</span> pathways leading to NF-κB activation can be grouped into four categories on the basis of the surface receptors involved: members of the TIR superfamily (IL-1Rs/TLRs), antigen receptors (TCR and BCR), members of the TNF-R superfamily (TNF-Rs), and RANK, VEGFR3, and EDAR. The two proteins of the TIR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (MyD88 and IRAK-4) and the two proteins of the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (NEMO and IκBα) responsible for primary immunodeficiencies are shown.</div></p></div><div class="aux"><div class="resc">Capucine Picard, et al. Clin Microbiol Rev. 2011 July;24(3):490-497.</div><p class="links"><a class="dblinks" href="/pubmed/21734245" ref="ordinalpos=1352&amp;ncbi_uid=3562470&amp;link_uid=21734245">Citation</a><a class="dblinks" href="/pmc/articles/PMC3131061/" ref="ordinalpos=1352&amp;ncbi_uid=3562470&amp;link_uid=PMC3131061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1353.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2964935/figure/F4/" ref="ordinalpos=1353&amp;ncbi_uid=2535212&amp;link_uid=PMC2964935" href="/pmc/articles/PMC2964935/"><img src="/pmc/articles/instance/2964935/bin/nihms34081f4.gif" src-large="/pmc/articles/instance/2964935/bin/nihms34081f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2964935/" ref="ordinalpos=1353&amp;ncbi_uid=2535212&amp;link_uid=PMC2964935" image-link="/pmc/articles/PMC2964935/figure/F4/" class="imagepopup">Figure 4. The PAR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: Phosphorylation and Binding.  From: The PAR Proteins: Fundamental Players in Animal Cell Polarization. </a></p><div class="supp"><p class="details"><div>The PAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; phosphorylation and binding of PAR-5 provides a mechanism for mutual exclusion from different PAR domains. On the left, PAR-1 that has diffused onto the apical domain is phosphorylated by aPKC, which inhibits the PAR-1 kinase activity and induces binding of PAR-5, which in turn triggers release into the cytoplasm. A similar mechanism probably controls the cortical attachment of LGL, Numb, and perhaps XGAP. In a reciprocal fashion, PAR-3 that diffuses into the basolateral domain is phosphorylated by PAR-1, which induces binding of PAR-5 and release into the cytoplasm.</div></p></div><div class="aux"><div class="resc">Bob Goldstein, et al. Dev Cell. ;13(5):609-622.</div><p class="links"><a class="dblinks" href="/pubmed/17981131" ref="ordinalpos=1353&amp;ncbi_uid=2535212&amp;link_uid=17981131">Citation</a><a class="dblinks" href="/pmc/articles/PMC2964935/" ref="ordinalpos=1353&amp;ncbi_uid=2535212&amp;link_uid=PMC2964935">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1354.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2940771/figure/F1/" ref="ordinalpos=1354&amp;ncbi_uid=2504171&amp;link_uid=PMC2940771" href="/pmc/articles/PMC2940771/"><img src="/pmc/articles/instance/2940771/bin/1752-0509-4-119-1.gif" src-large="/pmc/articles/instance/2940771/bin/1752-0509-4-119-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2940771/" ref="ordinalpos=1354&amp;ncbi_uid=2504171&amp;link_uid=PMC2940771" image-link="/pmc/articles/PMC2940771/figure/F1/" class="imagepopup">Figure 1.  From: Ultrasensitive Responses and Specificity in Cell <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Networks and insulating mechanisms</b>. <b>(A) </b>Schematic of a network with crosstalk. This network has no insulating mechanisms. However the connections between <i>x</i><sub>1 </sub>and <i>x</i><sub>2 </sub>and between <i>x</i><sub>1 </sub>and <i>y</i><sub>2 </sub>are allowed to be hyperbolic or ultrasensitive, denoted by <img src="/pmc/articles/instance/2940771/bin/1752-0509-4-119-i1.gif" class="inline-graphic" style="border-style: none;" />. <b>(B-E) </b>Networks embellished with various insulating mechanisms <b>(B) </b>Cross <span class="highlight" style="background-color:">Pathway</span> Inhibition (CPI) from <i>x</i><sub>2 </sub>to <i>y</i><sub>2</sub>. <b>(C) </b>Combinatorial <span class="highlight" style="background-color:">Signaling</span> (CS) in the <i>X </i><span class="highlight" style="background-color:">pathway</span>. <b>(D) </b>Scaffolding. <b>(E) </b>Compartmentalization.</div></p></div><div class="aux"><div class="resc">Seth Haney, et al. BMC Syst Biol. 2010;4:119-119.</div><p class="links"><a class="dblinks" href="/pubmed/20735856" ref="ordinalpos=1354&amp;ncbi_uid=2504171&amp;link_uid=20735856">Citation</a><a class="dblinks" href="/pmc/articles/PMC2940771/" ref="ordinalpos=1354&amp;ncbi_uid=2504171&amp;link_uid=PMC2940771">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1355.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109866/figure/F3/" ref="ordinalpos=1355&amp;ncbi_uid=3006876&amp;link_uid=PMC3109866" href="/pmc/articles/PMC3109866/"><img src="/pmc/articles/instance/3109866/bin/nihms296160f3.gif" src-large="/pmc/articles/instance/3109866/bin/nihms296160f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109866/" ref="ordinalpos=1355&amp;ncbi_uid=3006876&amp;link_uid=PMC3109866" image-link="/pmc/articles/PMC3109866/figure/F3/" class="imagepopup">Figure 3. Mitochondrial <span class="highlight" style="background-color:">signaling</span> may mediate the cross-talk between the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> and TOR pathways to regulate longevity.  From: Converging Pathways in Lifespan Regulation. </a></p><div class="supp"><p class="details"><div>Mitochondria are known to signal to the nucleus by at least two mechanisms: one is the retrograde <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (RTG) that involves LST8 and possibly TOR; the other uses reactive oxygen species and signals through the JNK–FOXO <span class="highlight" style="background-color:">pathway</span>. AMPK also receives input from mitochondria and is yet another protein involved in the mitochondrial control of lifespan.</div></p></div><div class="aux"><div class="resc">Sri Devi Narasimhan, et al. Curr Biol. ;19(15):R657-R666.</div><p class="links"><a class="dblinks" href="/pubmed/19674551" ref="ordinalpos=1355&amp;ncbi_uid=3006876&amp;link_uid=19674551">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109866/" ref="ordinalpos=1355&amp;ncbi_uid=3006876&amp;link_uid=PMC3109866">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1356.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g003/" ref="ordinalpos=1356&amp;ncbi_uid=3758046&amp;link_uid=PMC3256223" href="/pmc/articles/PMC3256223/"><img src="/pmc/articles/instance/3256223/bin/pone.0030140.g003.gif" src-large="/pmc/articles/instance/3256223/bin/pone.0030140.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3256223/" ref="ordinalpos=1356&amp;ncbi_uid=3758046&amp;link_uid=PMC3256223" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g003/" class="imagepopup">Figure 3. Modeling of mir-192 and mir-181c effects on the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Modeling of miRNA and Drug Action in the EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A</b>: Increased expression of mir-192 gene corresponds with a reduced level of targets' protein expression; <b>B</b>: Concentrations of EGFR downstream activated proteins are inversely correlated with the mir-192 gene expression level; <b>C</b>: simulation result with fixed mir-181c gene expression level (1 nM), whereas all other 240 miRNA genes are not expressed. All 13 AKT-dependent proteins can only be activated after the concentration of EGFR protein is getting higher than 10 pM. This activation threshold is due to the presence of mir-181c. <b>D</b>: represents another simulation result without the presence of mir-181c, additionally all other 240 miRNAs were not expressed, as well. At this condition, the activation threshold of these proteins is at 0.001 pM. By comparing these two results (<b>C</b>, <b>D</b>), one can understand that mir-181c raises the activation threshold of the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> significantly.</div></p></div><div class="aux"><div class="resc">Jian Li, et al. PLoS One. 2012;7(1):e30140.</div><p class="links"><a class="dblinks" href="/pubmed/22253908" ref="ordinalpos=1356&amp;ncbi_uid=3758046&amp;link_uid=22253908">Citation</a><a class="dblinks" href="/pmc/articles/PMC3256223/" ref="ordinalpos=1356&amp;ncbi_uid=3758046&amp;link_uid=PMC3256223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1357.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436998/figure/F1/" ref="ordinalpos=1357&amp;ncbi_uid=5432842&amp;link_uid=PMC3436998" href="/pmc/articles/PMC3436998/"><img src="/pmc/articles/instance/3436998/bin/nihms376498f1.gif" src-large="/pmc/articles/instance/3436998/bin/nihms376498f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436998/" ref="ordinalpos=1357&amp;ncbi_uid=5432842&amp;link_uid=PMC3436998" image-link="/pmc/articles/PMC3436998/figure/F1/" class="imagepopup">FIGURE 1.  From: Multiscale Models of Cell <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Multiple subcellular scales of signal transduction. <span class="highlight" style="background-color:">Signaling</span> molecules undergo posttranslational modifications to modulate activity within a <span class="highlight" style="background-color:">pathway</span>. Multiple pathways interconnect to form a <span class="highlight" style="background-color:">signaling</span> network, which collectively integrates external cues to change cellular behavior. The multiscale properties of signal transduction can give rise to surprising cellular responses when <span class="highlight" style="background-color:">signaling</span> is perturbed, as with the B-Raf inhibitor PLX4032.<sup>34,70</sup> Models seek to gain additional biological insight by bridging adjacent scales.</div></p></div><div class="aux"><div class="resc">Sameer S. Bajikar, et al. Ann Biomed Eng. 2012 November;40(11):2319-2327.</div><p class="links"><a class="dblinks" href="/pubmed/22476894" ref="ordinalpos=1357&amp;ncbi_uid=5432842&amp;link_uid=22476894">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436998/" ref="ordinalpos=1357&amp;ncbi_uid=5432842&amp;link_uid=PMC3436998">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1358.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3871684/figure/pone-0083906-g001/" ref="ordinalpos=1358&amp;ncbi_uid=5633845&amp;link_uid=PMC3871684" href="/pmc/articles/PMC3871684/"><img src="/pmc/articles/instance/3871684/bin/pone.0083906.g001.gif" src-large="/pmc/articles/instance/3871684/bin/pone.0083906.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3871684/" ref="ordinalpos=1358&amp;ncbi_uid=5633845&amp;link_uid=PMC3871684" image-link="/pmc/articles/PMC3871684/figure/pone-0083906-g001/" class="imagepopup">Figure 1. EGF <span class="highlight" style="background-color:">Signaling</span> Pathways driving migration of RPCs..  From: Microfluidic Generated EGF-Gradients Induce Chemokinesis of Transplantable Retinal Progenitor Cells via the JAK/STAT and PI3Kinase <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Predicted network of direct molecular interactions influencing RPC migration in the mammalian retina. The network was generated using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) bioinformatics tool. Molecular components of the EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are localized to extracellular space, plasma membrane, cytoplasm and nucleus. Arrows indicate direct interactions between upstream and downstream <span class="highlight" style="background-color:">pathway</span> molecules. Molecule shapes outlined or filled in red denote significant association with chemotaxis as determined by a right-tailed Fischer test. Molecules displayed on the left represent key chemical groups predicted to participate indirectly in the network. </div></p></div><div class="aux"><div class="resc">Uchenna J. Unachukwu, et al. PLoS One. 2013;8(12):e83906.</div><p class="links"><a class="dblinks" href="/pubmed/24376770" ref="ordinalpos=1358&amp;ncbi_uid=5633845&amp;link_uid=24376770">Citation</a><a class="dblinks" href="/pmc/articles/PMC3871684/" ref="ordinalpos=1358&amp;ncbi_uid=5633845&amp;link_uid=PMC3871684">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1359.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3096649/figure/pone-0019846-g002/" ref="ordinalpos=1359&amp;ncbi_uid=2965921&amp;link_uid=PMC3096649" href="/pmc/articles/PMC3096649/"><img src="/pmc/articles/instance/3096649/bin/pone.0019846.g002.gif" src-large="/pmc/articles/instance/3096649/bin/pone.0019846.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3096649/" ref="ordinalpos=1359&amp;ncbi_uid=2965921&amp;link_uid=PMC3096649" image-link="/pmc/articles/PMC3096649/figure/pone-0019846-g002/" class="imagepopup">Figure 2. IGF1R internalization in TC71 cells by clathrin- and Cav1-dependent mechanisms and the effects of CAV1 or clathrin inhibition/interference in the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: IGF1R <span class="highlight" style="background-color:">Signaling</span> in Ewing Sarcoma Is Shaped by Clathrin-/Caveolin-Dependent Endocytosis. </a></p><div class="supp"><p class="details"><div><b>A</b>) Studies of RCM/EM microscopy show IGF1R in the ES cell cytoplasm (top panel, white arrows) and in the vesicles budding off the cell membrane (bottom picture, black arrows) after ligand binding. Scale bar represents 1 (RCM) or 0.1 µM (EM). <b>B</b>) IP/WB studies show that after stimulation with ligand (3rd, 5th and 7th lane), IGF1R co-precipitates with both clathrin (1<sup>st</sup> row 5th lane; 2<sup>nd</sup> row 3<sup>rd</sup> lane) and with Cav1, although to a lesser extent (1<sup>st</sup> row 7th lane, 3<sup>rd</sup> row 3<sup>rd</sup> lane). TE: total extract. <b>C</b>) WB detection of the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after CAV1 or D) clathrin interference/inhibition. Both approaches inhibited the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, inhibiting both AKT and MAPK42/44 phosphorylation. Data presented is representative of 3–5 independent experiments for the RCM/EM studies and 3–6 independent experiments for the IP/WB studies.</div></p></div><div class="aux"><div class="resc">Ana Sofia Martins, et al. PLoS One. 2011;6(5):e19846.</div><p class="links"><a class="dblinks" href="/pubmed/21611203" ref="ordinalpos=1359&amp;ncbi_uid=2965921&amp;link_uid=21611203">Citation</a><a class="dblinks" href="/pmc/articles/PMC3096649/" ref="ordinalpos=1359&amp;ncbi_uid=2965921&amp;link_uid=PMC3096649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1360.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3541667/figure/fig5/" ref="ordinalpos=1360&amp;ncbi_uid=5655541&amp;link_uid=PMC3541667" href="/pmc/articles/PMC3541667/"><img src="/pmc/articles/instance/3541667/bin/cr2012119f5.gif" src-large="/pmc/articles/instance/3541667/bin/cr2012119f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3541667/" ref="ordinalpos=1360&amp;ncbi_uid=5655541&amp;link_uid=PMC3541667" image-link="/pmc/articles/PMC3541667/figure/fig5/" class="imagepopup">Figure 5.  From: Histone H3K27me3 demethylases KDM6A and KDM6B modulate definitive endoderm differentiation from human ESCs by regulating WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Hypothetic model explaining how KDM6 may contribute to definitive endoderm differentiation. Treatment of hESCs with activin A causes activation of KDM6A and KDM6B, which can target and remove the H3K27me3 repressive mark leading to upregulation of WNT3. Increased WNT3 causes activation of the WNT <span class="highlight" style="background-color:">pathway</span> and promotes mesendoderm differentiation. At the late stage of endoderm differentiation, WNT antagonist gene <i>DKK1</i> is activated by KDM6A and KDM6B in a similar fashion to suppress WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for mesendoderm to definitive endoderm differentiation. ME, mesendoderm; DE, definitive endoderm.</div></p></div><div class="aux"><div class="resc">Wei Jiang, et al. Cell Res. 2013 January;23(1):122-130.</div><p class="links"><a class="dblinks" href="/pubmed/22907667" ref="ordinalpos=1360&amp;ncbi_uid=5655541&amp;link_uid=22907667">Citation</a><a class="dblinks" href="/pmc/articles/PMC3541667/" ref="ordinalpos=1360&amp;ncbi_uid=5655541&amp;link_uid=PMC3541667">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1361.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3916982/figure/f1-br-02-01-0041/" ref="ordinalpos=1361&amp;ncbi_uid=5926571&amp;link_uid=PMC3916982" href="/pmc/articles/PMC3916982/"><img src="/pmc/articles/instance/3916982/bin/BR-02-01-0041-g00.gif" src-large="/pmc/articles/instance/3916982/bin/BR-02-01-0041-g00.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3916982/" ref="ordinalpos=1361&amp;ncbi_uid=5926571&amp;link_uid=PMC3916982" image-link="/pmc/articles/PMC3916982/figure/f1-br-02-01-0041/" class="imagepopup">Figure 1.  From: Estrogen receptor-positive breast cancer molecular signatures and therapeutic potentials (Review). </a></p><div class="supp"><p class="details"><div>The estrogen receptor (ER) <span class="highlight" style="background-color:">pathway</span> includes the canonical and the non-canonical ER pathways. In the canonical <span class="highlight" style="background-color:">pathway</span>, the cytoplasmic estrogens bind directly to the nuclear membrane estrogen receptor and activate <span class="highlight" style="background-color:">signaling</span>. In the non-canonical <span class="highlight" style="background-color:">pathway</span>, the extracellular estrogens bind to the plasma membrane receptors and activate the phosphatidylinositol-3-kinase (PI3K) or Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Alternatively, the estrogens first penetrate through the cell membrane and then bind to the estrogen receptor monomer, which is then dimerized and transported into the nucleus to activate <span class="highlight" style="background-color:">signaling</span>. Treatment with tamoxifen blocks the binding of estrogen and, thus, inhibits the estrogen-activated <span class="highlight" style="background-color:">signaling</span>. Treatment with fulvestrant inhibits the cytoplasmic dimerization of estrogen. Treatment with aromatase inhibitors, such as letrozole, anastrozole and exemestane blocks estrogen production. Another characteristic correlated to the ER <span class="highlight" style="background-color:">pathway</span> is the progesterone receptor (PR) <span class="highlight" style="background-color:">signaling</span>, which is also involved in the regulation of luminal type breast cancer development. CoR, coregulators; RE, response element; TFs, transcription factors; mER, membrane ER; HER, human epidermal growth factor receptor; MAPK, mitogen-activated protein kinase; ERE, estrogen response element; PRE, progesterone response element.</div></p></div><div class="aux"><div class="resc">MEI HONG ZHANG, et al. Biomed Rep. 2014 January;2(1):41-52.</div><p class="links"><a class="dblinks" href="/pubmed/24649067" ref="ordinalpos=1361&amp;ncbi_uid=5926571&amp;link_uid=24649067">Citation</a><a class="dblinks" href="/pmc/articles/PMC3916982/" ref="ordinalpos=1361&amp;ncbi_uid=5926571&amp;link_uid=PMC3916982">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1362.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3026445/figure/F1/" ref="ordinalpos=1362&amp;ncbi_uid=3615815&amp;link_uid=PMC3026445" href="/pmc/articles/PMC3026445/"><img src="/pmc/articles/instance/3026445/bin/nihms254006f1.gif" src-large="/pmc/articles/instance/3026445/bin/nihms254006f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3026445/" ref="ordinalpos=1362&amp;ncbi_uid=3615815&amp;link_uid=PMC3026445" image-link="/pmc/articles/PMC3026445/figure/F1/" class="imagepopup">Figure 1. Insulin <span class="highlight" style="background-color:">signaling</span> is required cell nonautonomously for sperm guidance.  From: Insulin/FOXO <span class="highlight" style="background-color:">Signaling</span> Regulates Ovarian Prostaglandins Critical for Reproduction. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Canonical insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i>.<br /><b>(B)</b> Representative sperm distribution images 1 hour after mating wild-type or mutant hermaphrodites to wild-type MitoTracker-labelled (MT) males. Quantification of selected genotypes is shown in Fig. 2. Yellow outline marks the spermatheca. Vulva is to the right (see Fig. 2A for gonad diagram). Bar, 20 μm.</div></p></div><div class="aux"><div class="resc">Johnathan W. Edmonds, et al. Dev Cell. ;19(6):858-871.</div><p class="links"><a class="dblinks" href="/pubmed/21145501" ref="ordinalpos=1362&amp;ncbi_uid=3615815&amp;link_uid=21145501">Citation</a><a class="dblinks" href="/pmc/articles/PMC3026445/" ref="ordinalpos=1362&amp;ncbi_uid=3615815&amp;link_uid=PMC3026445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1363.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432052/figure/pone-0043898-g007/" ref="ordinalpos=1363&amp;ncbi_uid=4215612&amp;link_uid=PMC3432052" href="/pmc/articles/PMC3432052/"><img src="/pmc/articles/instance/3432052/bin/pone.0043898.g007.gif" src-large="/pmc/articles/instance/3432052/bin/pone.0043898.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432052/" ref="ordinalpos=1363&amp;ncbi_uid=4215612&amp;link_uid=PMC3432052" image-link="/pmc/articles/PMC3432052/figure/pone-0043898-g007/" class="imagepopup">Figure 7. The proposed mechanisms by which nicotine promotes NPC cell proliferation..  From: Nicotine Promotes Proliferation of Human Nasopharyngeal Carcinoma Cells by Regulating ?7AChR, ERK, HIF-1? and VEGF/PEDF <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Nicotine promotes NPC cell proliferation by simultaneously promoting the α7AChR, ERK, HIF-1α and VEGF/PEDF <span class="highlight" style="background-color:">signaling</span>. Nicotine interacts with α7AChR on the surface of NPC cells, activates ERK/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, upregulates HIF-1α expression, elevates VEGF/PEDF ratio, thereby promotes NPC cells proliferation.</div></p></div><div class="aux"><div class="resc">Dingbo Shi, et al. PLoS One. 2012;7(8):e43898.</div><p class="links"><a class="dblinks" href="/pubmed/22952803" ref="ordinalpos=1363&amp;ncbi_uid=4215612&amp;link_uid=22952803">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432052/" ref="ordinalpos=1363&amp;ncbi_uid=4215612&amp;link_uid=PMC3432052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1364.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3012164/figure/F2/" ref="ordinalpos=1364&amp;ncbi_uid=3444785&amp;link_uid=PMC3012164" href="/pmc/articles/PMC3012164/"><img src="/pmc/articles/instance/3012164/bin/zdb0011164200002.gif" src-large="/pmc/articles/instance/3012164/bin/zdb0011164200002.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3012164/" ref="ordinalpos=1364&amp;ncbi_uid=3444785&amp;link_uid=PMC3012164" image-link="/pmc/articles/PMC3012164/figure/F2/" class="imagepopup">FIG. 2. From: Direct Recruitment of Insulin Receptor and ERK <span class="highlight" style="background-color:">Signaling</span> Cascade to Insulin-Inducible Gene Loci. </a></p><div class="supp"><p class="details"><div>Insulin receptor <span class="highlight" style="background-color:">signaling</span> complex at the <i>egr-1</i> locus in HTC-IR cells. <i>A</i>: Schematic of one arm of the IR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> including some components of the IR <span class="highlight" style="background-color:">signaling</span> complex (highlighted) and MAPK <span class="highlight" style="background-color:">pathway</span> components. <i>B</i>: Schematic of the <i>egr-1</i> locus. <i>C</i>: Cells were treated as in Fig. 1<i>A</i>, cross-linked, and used in ChIP with antibodies to the β subunit of IR (IR-β), IR-β phosphorylated at Y1146 (pIR), Grb2, and SOS. ChIPed DNA was used in PCR as in Fig. 1<i>C</i>. Data represent mean ± SEM for three independent experiments.</div></p></div><div class="aux"><div class="resc">Joel D. Nelson, et al. Diabetes. 2011 January;60(1):127-137.</div><p class="links"><a class="dblinks" href="/pubmed/20929976" ref="ordinalpos=1364&amp;ncbi_uid=3444785&amp;link_uid=20929976">Citation</a><a class="dblinks" href="/pmc/articles/PMC3012164/" ref="ordinalpos=1364&amp;ncbi_uid=3444785&amp;link_uid=PMC3012164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1365.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2841699/figure/F8/" ref="ordinalpos=1365&amp;ncbi_uid=2794490&amp;link_uid=PMC2841699" href="/pmc/articles/PMC2841699/"><img src="/pmc/articles/instance/2841699/bin/nihms176202f8.gif" src-large="/pmc/articles/instance/2841699/bin/nihms176202f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2841699/" ref="ordinalpos=1365&amp;ncbi_uid=2794490&amp;link_uid=PMC2841699" image-link="/pmc/articles/PMC2841699/figure/F8/" class="imagepopup">Figure 8. Proposed <span class="highlight" style="background-color:">pathway</span> in regulation of ATX expression, LPA production and <span class="highlight" style="background-color:">signaling</span> by VEGF in HUVEC.  From: Autotaxin <span class="highlight" style="background-color:">signaling</span> via lysophosphatidic acid receptors contributes to vascular endothelial growth factor-induced endothelial cell migration. </a></p><div class="supp"><p class="details"><div>Cancer cells and stromal cells secrete VEGF, which <i>via</i> VEGFR2 stimulates expression of ATX and extracellular LPA production, as well as mRNA levels for LPA1 receptor. The elevated extracellular ATX would increase localized levels of LPA and stimulate LPA <span class="highlight" style="background-color:">signaling</span>, particularly LPA1. Elevated LPA1 <span class="highlight" style="background-color:">signaling</span>, in turn, increases Akt2 mRNA levels. Enhanced expression of Akt2 has been associated with cancer progression, cellular migration and promotion of epithelial-mesenchymal transition.</div></p></div><div class="aux"><div class="resc">Malgorzata M. Ptaszynska, et al. Mol Cancer Res. ;8(3):309-321.</div><p class="links"><a class="dblinks" href="/pubmed/20197381" ref="ordinalpos=1365&amp;ncbi_uid=2794490&amp;link_uid=20197381">Citation</a><a class="dblinks" href="/pmc/articles/PMC2841699/" ref="ordinalpos=1365&amp;ncbi_uid=2794490&amp;link_uid=PMC2841699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1366.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1682017/figure/f4/" ref="ordinalpos=1366&amp;ncbi_uid=811962&amp;link_uid=PMC1682017" href="/pmc/articles/PMC1682017/"><img src="/pmc/articles/instance/1682017/bin/msb4100094-f4.gif" src-large="/pmc/articles/instance/1682017/bin/msb4100094-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1682017/" ref="ordinalpos=1366&amp;ncbi_uid=811962&amp;link_uid=PMC1682017" image-link="/pmc/articles/PMC1682017/figure/f4/" class="imagepopup">Figure 4.  From: Effects of HER2 overexpression on cell <span class="highlight" style="background-color:">signaling</span> networks governing proliferation and migration. </a></p><div class="supp"><p class="details"><div>Effect of increased HER2 expression on phosphorylation sites within the EGFR <span class="highlight" style="background-color:">signaling</span> network. Visualization of the fold change in phosphorylation level between 24H and parental cells stimulated with EGF provides a network view of the mechanistic effects of HER2 overexpression on (<b>A</b>) the canonical EGFR <span class="highlight" style="background-color:">signaling</span> cascades and (<b>B</b>) the cell migration-associated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Alejandro Wolf-Yadlin, et al. Mol Syst Biol. 2006;2:54-54.</div><p class="links"><a class="dblinks" href="/pubmed/17016520" ref="ordinalpos=1366&amp;ncbi_uid=811962&amp;link_uid=17016520">Citation</a><a class="dblinks" href="/pmc/articles/PMC1682017/" ref="ordinalpos=1366&amp;ncbi_uid=811962&amp;link_uid=PMC1682017">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1367.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3145778/figure/pone-0022896-g009/" ref="ordinalpos=1367&amp;ncbi_uid=3152907&amp;link_uid=PMC3145778" href="/pmc/articles/PMC3145778/"><img src="/pmc/articles/instance/3145778/bin/pone.0022896.g009.gif" src-large="/pmc/articles/instance/3145778/bin/pone.0022896.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3145778/" ref="ordinalpos=1367&amp;ncbi_uid=3152907&amp;link_uid=PMC3145778" image-link="/pmc/articles/PMC3145778/figure/pone-0022896-g009/" class="imagepopup">Figure 9. A model for TGF-β1 stimulated maintenance of SMAD3 phosphorylation and PAI-1 induction via Src/FAK/Caveolin-1 <span class="highlight" style="background-color:">signaling</span>..  From: Redox-Induced Src Kinase and Caveolin-1 <span class="highlight" style="background-color:">Signaling</span> in TGF-?1-Initiated SMAD2/3 Activation and PAI-1 Expression. </a></p><div class="supp"><p class="details"><div>TGF-β1 stimulates caveolin-1<sup>Y14</sup> phosphorylation in a reactive oxygen species-FAK/<i>c-Src</i> dependent manner removing repressive influences on EGFR <span class="highlight" style="background-color:">signaling</span> (in red) leading to EGFR transactivation (also by <i>c-Src</i>), thereby, initiating <span class="highlight" style="background-color:">signaling</span> events leading to the MEK-ERK <span class="highlight" style="background-color:">pathway</span> activation necessary for PAI-1 induction. <i>Src</i> kinase phosphorylation of caveolin-1<sup>Y14</sup> also stimulates Rho-GTP loading and ROCK (an established downstream target of Rho) activation is necessary for PAI-1 induction. pCaveolin-1<sup>Y14</sup>-Rho-ROCK mediated <span class="highlight" style="background-color:">signaling</span> leads to inhibition of PTEN-PPM1A interactions resulting in a reduction of nuclear PPM1A phosphatase (black <span class="highlight" style="background-color:">pathway</span>), thereby, maintaining the pSMAD2/3 levels (highlighted in blue) required for PAI-1 induction by TGF-β1 (see text). PAI-1 is elevated in atherosclerotic plaques frequently colocalizing with α-smooth muscle actin-expressing cells, presumably VSMCs (insert).</div></p></div><div class="aux"><div class="resc">Rohan Samarakoon, et al. PLoS One. 2011;6(7):e22896.</div><p class="links"><a class="dblinks" href="/pubmed/21829547" ref="ordinalpos=1367&amp;ncbi_uid=3152907&amp;link_uid=21829547">Citation</a><a class="dblinks" href="/pmc/articles/PMC3145778/" ref="ordinalpos=1367&amp;ncbi_uid=3152907&amp;link_uid=PMC3145778">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1368.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1479092/figure/F11/" ref="ordinalpos=1368&amp;ncbi_uid=710114&amp;link_uid=PMC1479092" href="/pmc/articles/PMC1479092/"><img src="/pmc/articles/instance/1479092/bin/nihms-10426-0011.gif" src-large="/pmc/articles/instance/1479092/bin/nihms-10426-0011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1479092/" ref="ordinalpos=1368&amp;ncbi_uid=710114&amp;link_uid=PMC1479092" image-link="/pmc/articles/PMC1479092/figure/F11/" class="imagepopup">Figure 11.  From: Thioredoxin-ASK1 complex levels regulate ROS-mediated p38 MAPK <span class="highlight" style="background-color:">pathway</span> activity in livers of aged and long-lived Snell dwarf mice. </a></p><div class="supp"><p class="details"><div>Regulation of ROS-mediated activation of the p38 MAPK stress response <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> shows the mechanism that links the rotenone-mediated production of ROS by mitochondrial ETC Complex 1 to the activation of the p38 MAPK and SAPK/JNK stress response pathways. The model suggests that the Trx-ASK1 complex plays a key role in regulation of the p38 MAPK and SAPK/JNK stress response <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Ching-Chyuan Hsieh, et al. FASEB J. ;20(2):259-268.</div><p class="links"><a class="dblinks" href="/pubmed/16449798" ref="ordinalpos=1368&amp;ncbi_uid=710114&amp;link_uid=16449798">Citation</a><a class="dblinks" href="/pmc/articles/PMC1479092/" ref="ordinalpos=1368&amp;ncbi_uid=710114&amp;link_uid=PMC1479092">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1369.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3248125/figure/F1/" ref="ordinalpos=1369&amp;ncbi_uid=3765322&amp;link_uid=PMC3248125" href="/pmc/articles/PMC3248125/"><img src="/pmc/articles/instance/3248125/bin/oncotarget-01-530-g001.gif" src-large="/pmc/articles/instance/3248125/bin/oncotarget-01-530-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3248125/" ref="ordinalpos=1369&amp;ncbi_uid=3765322&amp;link_uid=PMC3248125" image-link="/pmc/articles/PMC3248125/figure/F1/" class="imagepopup">Figure 1. The PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and associated inhibitors.  From: Targeting the PI3K/Akt/mTOR <span class="highlight" style="background-color:">Pathway</span> - Beyond Rapalogs. </a></p><div class="supp"><p class="details"><div>A ligand engaged RTK binds PI3K, either directly or indirectly via adaptor molecules such as IRS1, removing the inhibitory action of the p85 regulatory subunit on the catalytic p110 subunit. The active kinase generates PIP3 at the lipid membrane. PIP3 facilitates the phosphorylation of Akt by PDK1, while the mTOR-rictor complex contributes a second phosphate residue to Akt. As the central effector of the PI3K <span class="highlight" style="background-color:">pathway</span>, Akt transmits signal to a host of downstream substrates, thus influencing a variety of key cellular functions. <span class="highlight" style="background-color:">Pathway</span> activity is negatively regulated by PTEN and the S6K-IRS1 feedback loop. Pharmacological inhibition of the <span class="highlight" style="background-color:">pathway</span> is achieved through a variety of compounds in clinical use at various points along the <span class="highlight" style="background-color:">pathway</span> that are indicated by the red ⊣.</div></p></div><div class="aux"><div class="resc">Ben Markman, et al. Oncotarget. 2010 November;1(7):530-543.</div><p class="links"><a class="dblinks" href="/pubmed/21317449" ref="ordinalpos=1369&amp;ncbi_uid=3765322&amp;link_uid=21317449">Citation</a><a class="dblinks" href="/pmc/articles/PMC3248125/" ref="ordinalpos=1369&amp;ncbi_uid=3765322&amp;link_uid=PMC3248125">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1370.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2323406/figure/pcbi-0040030-g004/" ref="ordinalpos=1370&amp;ncbi_uid=1346740&amp;link_uid=PMC2323406" href="/pmc/articles/PMC2323406/"><img src="/pmc/articles/instance/2323627/bin/pcbi.0040030.g004.gif" src-large="/pmc/articles/instance/2323627/bin/pcbi.0040030.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2323406/" ref="ordinalpos=1370&amp;ncbi_uid=1346740&amp;link_uid=PMC2323406" image-link="/pmc/articles/PMC2323406/figure/pcbi-0040030-g004/" class="imagepopup">Figure 4. Schematic of EGF-Induced <span class="highlight" style="background-color:">Signaling</span>.  From: Stimulus Design for Model Selection and Validation in Cell <span class="highlight" style="background-color:">Signaling</span> . </a></p><div class="supp"><p class="details"><div>This schematic shows the major steps in EGFR <span class="highlight" style="background-color:">signaling</span>. At the top of the <span class="highlight" style="background-color:">pathway</span> ligand binds to the receptor and induces receptor dimerization and activation. The signal is then transduced through a series of adaptor proteins SHC, GRB2, and SOS, which in turn activates the MAPK cascade RAF, MEK and ERK. There are two negative feedback loops: internalization and degradation of the receptor complex, and ERK deactivation of SOS.</div></p></div><div class="aux"><div class="resc">Joshua F Apgar, et al. PLoS Comput Biol. 2008 February;4(2):e30.</div><p class="links"><a class="dblinks" href="/pubmed/18282085" ref="ordinalpos=1370&amp;ncbi_uid=1346740&amp;link_uid=18282085">Citation</a><a class="dblinks" href="/pmc/articles/PMC2323406/" ref="ordinalpos=1370&amp;ncbi_uid=1346740&amp;link_uid=PMC2323406">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1371.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2652039/figure/F8/" ref="ordinalpos=1371&amp;ncbi_uid=1761995&amp;link_uid=PMC2652039" href="/pmc/articles/PMC2652039/"><img src="/pmc/articles/instance/2652039/bin/nihms-81179-f0009.gif" src-large="/pmc/articles/instance/2652039/bin/nihms-81179-f0009.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2652039/" ref="ordinalpos=1371&amp;ncbi_uid=1761995&amp;link_uid=PMC2652039" image-link="/pmc/articles/PMC2652039/figure/F8/" class="imagepopup">Figure 8. Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving TAK1 and Akt in TWEAK-induced activation of NF-κB.  From: Tumor Necrosis Factor-like Weak Inducer of Apoptosis (TWEAK) activates proinflammatory <span class="highlight" style="background-color:">signaling</span> pathways and gene expression through the activation of TGF-beta activated kinase 1. </a></p><div class="supp"><p class="details"><div>Binding of TWEAK to Fn14 receptors leads to the activation of TAK1 and Akt through recruitment of TRAF-1, 2, 3, and/or 5 proteins. Activated TAK1 and Akt then phosphorylate and activate IKKβ in IKK complex leading to the activation NF-κB through canonical <span class="highlight" style="background-color:">pathway</span>. Activated TAK1 can also activate JNK1 and p38 MAPK which in their turn activates AP-1. Increased activation of NF-κB and AP-1 leads to the increased expression of proinflammatory genes.</div></p></div><div class="aux"><div class="resc">Mukesh Kumar, et al. J Immunol. ;182(4):2439-2448.</div><p class="links"><a class="dblinks" href="/pubmed/19201899" ref="ordinalpos=1371&amp;ncbi_uid=1761995&amp;link_uid=19201899">Citation</a><a class="dblinks" href="/pmc/articles/PMC2652039/" ref="ordinalpos=1371&amp;ncbi_uid=1761995&amp;link_uid=PMC2652039">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1372.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC20092/figure/F5/" ref="ordinalpos=1372&amp;ncbi_uid=85049&amp;link_uid=PMC20092" href="/pmc/articles/PMC20092/"><img src="/pmc/articles/instance/20092/bin/pq0673858005.gif" src-large="/pmc/articles/instance/20092/bin/pq0673858005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC20092/" ref="ordinalpos=1372&amp;ncbi_uid=85049&amp;link_uid=PMC20092" image-link="/pmc/articles/PMC20092/figure/F5/" class="imagepopup">Figure 5.  From: Downstream of Crk adaptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: Activation of Jun kinase by v-Crk through the guanine nucleotide exchange protein C3G. </a></p><div class="supp"><p class="details"><div>A model for the v-Crk-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. v-Crk binds through its SH2 domain to multiple tyrosine-phosphorylated target molecules such as p130<sup>Cas</sup> or paxillin that supposedly localize to the focal adhesion. A guanine nucleotide exchange protein C3G, which associates with SH3 domain of v-Crk, can presumably activate Ras-like small G protein and generate signal for the activation of Jun kinase. The well established Grb2-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from tyrosine-phosphorylated receptor localized in the plasma membrane through Sos to the activation of MAPK is also described as a comparison of different <span class="highlight" style="background-color:">signaling</span> pathways mediated by two adaptor molecules.</div></p></div><div class="aux"><div class="resc">Shinya Tanaka, et al. Proc Natl Acad Sci U S A. 1997 March 18;94(6):2356-2361.</div><p class="links"><a class="dblinks" href="/pubmed/9122199" ref="ordinalpos=1372&amp;ncbi_uid=85049&amp;link_uid=9122199">Citation</a><a class="dblinks" href="/pmc/articles/PMC20092/" ref="ordinalpos=1372&amp;ncbi_uid=85049&amp;link_uid=PMC20092">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1373.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137643/figure/fig9/" ref="ordinalpos=1373&amp;ncbi_uid=3112689&amp;link_uid=PMC3137643" href="/pmc/articles/PMC3137643/"><img src="/pmc/articles/instance/3137643/bin/ECAM2011-807451.009.gif" src-large="/pmc/articles/instance/3137643/bin/ECAM2011-807451.009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137643/" ref="ordinalpos=1373&amp;ncbi_uid=3112689&amp;link_uid=PMC3137643" image-link="/pmc/articles/PMC3137643/figure/fig9/" class="imagepopup">Figure 9.  From: The Ayurvedic Medicine Salacia oblonga Attenuates Diabetic Renal Fibrosis in Rats: Suppression of Angiotensin II/AT1 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed model showing SO- and MA-dependent effects on the angiotensin II/AT1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and fibrogenic mediators in diabetic rat kidneys.</div></p></div><div class="aux"><div class="resc">Lan He, et al. Evid Based Complement Alternat Med. 2011;2011:807451.</div><p class="links"><a class="dblinks" href="/pubmed/19706694" ref="ordinalpos=1373&amp;ncbi_uid=3112689&amp;link_uid=19706694">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137643/" ref="ordinalpos=1373&amp;ncbi_uid=3112689&amp;link_uid=PMC3137643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1374.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3774688/figure/pone-0073230-g002/" ref="ordinalpos=1374&amp;ncbi_uid=5330956&amp;link_uid=PMC3774688" href="/pmc/articles/PMC3774688/"><img src="/pmc/articles/instance/3774688/bin/pone.0073230.g002.gif" src-large="/pmc/articles/instance/3774688/bin/pone.0073230.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3774688/" ref="ordinalpos=1374&amp;ncbi_uid=5330956&amp;link_uid=PMC3774688" image-link="/pmc/articles/PMC3774688/figure/pone-0073230-g002/" class="imagepopup">Figure 2. Functional classification of individual genes identified by gene expression analysis on TB patients..  From: A Helicopter Perspective on TB Biomarkers: <span class="highlight" style="background-color:">Pathway</span> and Process Based Analysis of Gene Expression Data Provides New Insight into TB Pathogenesis. </a></p><div class="supp"><p class="details"><div>Categories have been based on combined output from Ingenuity and GSEA software modules and may include multiple canonical pathways or cell processes. Myeloid cells includes the following canonical pathways: role of macrophages, fibroblasts and endothelial cells in rheumatoid arthritis; Fcg Receptor mediated phagocytosis in macrophages and monocytes; role of pattern recognition receptors in recognition of bacteria and viruses; IL12 <span class="highlight" style="background-color:">signaling</span> and production in macrophages; Dendritic cell maturation; production of Nitrox Oxide and Reactive Oxygen Species in Macrophages; Toll like receptor <span class="highlight" style="background-color:">signaling</span>. T cells includes: T cell receptor <span class="highlight" style="background-color:">signaling</span>; CD28 <span class="highlight" style="background-color:">signaling</span> in T helper cells; iCOS-iCOSL <span class="highlight" style="background-color:">signaling</span> in T helper cells. B cells includes: B cell receptor <span class="highlight" style="background-color:">signaling</span>; PI3K <span class="highlight" style="background-color:">signaling</span> in B lymphocytes. Interferon related pathways include: Interferon <span class="highlight" style="background-color:">signaling</span>, role of jak1, jak2 and tyk2 in interferon <span class="highlight" style="background-color:">signaling</span>, role of PKR in interferon induction and antiviral response. Inflammation includes: IL-8 <span class="highlight" style="background-color:">signaling</span>; NF-kB <span class="highlight" style="background-color:">signaling</span>; altered T cell and B cell <span class="highlight" style="background-color:">signaling</span> in Rheumatoid Arthritis; systemic lupus erythematosus <span class="highlight" style="background-color:">signaling</span>; chemokine <span class="highlight" style="background-color:">signaling</span>; IL-6 <span class="highlight" style="background-color:">signaling</span>. TREM1 includes specifically TREM1 <span class="highlight" style="background-color:">signaling</span> and mitochondrial dysfunction also only contains mitochondrial dysfunction. Finally, hematopoiesis includes: erythropoietin <span class="highlight" style="background-color:">signaling</span>; IL-3 <span class="highlight" style="background-color:">signaling</span>; FLT3 <span class="highlight" style="background-color:">signaling</span> in hematopoietic progenitor cells; prolactin <span class="highlight" style="background-color:">signaling</span>; HGF <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Simone A. Joosten, et al. PLoS One. 2013;8(9):e73230.</div><p class="links"><a class="dblinks" href="/pubmed/24066041" ref="ordinalpos=1374&amp;ncbi_uid=5330956&amp;link_uid=24066041">Citation</a><a class="dblinks" href="/pmc/articles/PMC3774688/" ref="ordinalpos=1374&amp;ncbi_uid=5330956&amp;link_uid=PMC3774688">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1375.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3414724/figure/F2/" ref="ordinalpos=1375&amp;ncbi_uid=4183601&amp;link_uid=PMC3414724" href="/pmc/articles/PMC3414724/"><img src="/pmc/articles/instance/3414724/bin/fimmu-03-00224-g002.gif" src-large="/pmc/articles/instance/3414724/bin/fimmu-03-00224-g002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3414724/" ref="ordinalpos=1375&amp;ncbi_uid=4183601&amp;link_uid=PMC3414724" image-link="/pmc/articles/PMC3414724/figure/F2/" class="imagepopup">FIGURE 2.  From: The phosphoinositide 3-kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal and malignant B cells: activation mechanisms, regulation and impact on cellular functions. </a></p><div class="supp"><p class="details"><div><b>Role of the PI3K <span class="highlight" style="background-color:">pathway</span> in B cell antigen presentation functions</b>. The Figure outlines the D3 phosphoinositides and PI-dependant <span class="highlight" style="background-color:">signaling</span> complexes implicated in cytoskeletal reorganization associated with B cell Ag uptake and presentation. The involvement of different components in different steps of Ag presentation is discussed in the text.</div></p></div><div class="aux"><div class="resc">Samantha D. Pauls, et al. Front Immunol. 2012;3:224.</div><p class="links"><a class="dblinks" href="/pubmed/22908014" ref="ordinalpos=1375&amp;ncbi_uid=4183601&amp;link_uid=22908014">Citation</a><a class="dblinks" href="/pmc/articles/PMC3414724/" ref="ordinalpos=1375&amp;ncbi_uid=4183601&amp;link_uid=PMC3414724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1376.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3314694/figure/pone-0033976-g001/" ref="ordinalpos=1376&amp;ncbi_uid=3882308&amp;link_uid=PMC3314694" href="/pmc/articles/PMC3314694/"><img src="/pmc/articles/instance/3314694/bin/pone.0033976.g001.gif" src-large="/pmc/articles/instance/3314694/bin/pone.0033976.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3314694/" ref="ordinalpos=1376&amp;ncbi_uid=3882308&amp;link_uid=PMC3314694" image-link="/pmc/articles/PMC3314694/figure/pone-0033976-g001/" class="imagepopup">Figure 1. Expression of Wnt <span class="highlight" style="background-color:">pathway</span> genes in tumorspheres (TMS), mammospheres (MMS) and mammospheres induced to differentiate (diffMMS)..  From: Small Molecule Antagonists of the Wnt/Beta-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Target Breast Tumor-Initiating Cells in a Her2/Neu Mouse Model of Breast Cancer. </a></p><div class="supp"><p class="details"><div>A) Heat maps of 3 independent TMS (A, n = 3), MMS (B, n = 3) and diffMMS (C, n = 3) RNA preparations profiled on MOE430A Gene Chips. B–E) qRT-PCR of transcripts encoding components and target genes of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (* p&lt;0.05, t-test) for all genes shown. F and G) Survival, (*p = 0.005, Log-rank test) and metastasis (*p = 0.0068, Log-rank test), curves for human breast cancer patients classified based on their expression of TMS-related and TMS-unrelated (diffMMS-related) specific Wnt <span class="highlight" style="background-color:">pathway</span> components.</div></p></div><div class="aux"><div class="resc">Robin M. Hallett, et al. PLoS One. 2012;7(3):e33976.</div><p class="links"><a class="dblinks" href="/pubmed/22470504" ref="ordinalpos=1376&amp;ncbi_uid=3882308&amp;link_uid=22470504">Citation</a><a class="dblinks" href="/pmc/articles/PMC3314694/" ref="ordinalpos=1376&amp;ncbi_uid=3882308&amp;link_uid=PMC3314694">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1377.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3188487/figure/pcbi-1002154-g003/" ref="ordinalpos=1377&amp;ncbi_uid=3282280&amp;link_uid=PMC3188487" href="/pmc/articles/PMC3188487/"><img src="/pmc/articles/instance/3188487/bin/pcbi.1002154.g003.gif" src-large="/pmc/articles/instance/3188487/bin/pcbi.1002154.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3188487/" ref="ordinalpos=1377&amp;ncbi_uid=3282280&amp;link_uid=PMC3188487" image-link="/pmc/articles/PMC3188487/figure/pcbi-1002154-g003/" class="imagepopup">Figure 3. Intramolecular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for PSD-95 with truncated third alpha helix predicted by the PRS method..  From: Change in Allosteric Network Affects Binding Affinities of PDZ Domains: Analysis through Perturbation Response Scanning. </a></p><div class="supp"><p class="details"><div>We obtain the allosteric <span class="highlight" style="background-color:">pathway</span> for the truncated PSD-95 through connections Ile314 → Ile 327 → Glu334 → His372 → Ile380 → Ile388. However, the predicted allosteric <span class="highlight" style="background-color:">pathway</span> of full PSD-95 is different. The interactions specifically located in the α<sub>1</sub> helix predicted for the PSD-95 were lost after removal of the α3 helix.</div></p></div><div class="aux"><div class="resc">Z. Nevin Gerek, et al. PLoS Comput Biol. 2011 October;7(10):e1002154.</div><p class="links"><a class="dblinks" href="/pubmed/21998559" ref="ordinalpos=1377&amp;ncbi_uid=3282280&amp;link_uid=21998559">Citation</a><a class="dblinks" href="/pmc/articles/PMC3188487/" ref="ordinalpos=1377&amp;ncbi_uid=3282280&amp;link_uid=PMC3188487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1378.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3599951/figure/F5/" ref="ordinalpos=1378&amp;ncbi_uid=4716726&amp;link_uid=PMC3599951" href="/pmc/articles/PMC3599951/"><img src="/pmc/articles/instance/3599951/bin/1748-717X-8-46-5.gif" src-large="/pmc/articles/instance/3599951/bin/1748-717X-8-46-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3599951/" ref="ordinalpos=1378&amp;ncbi_uid=4716726&amp;link_uid=PMC3599951" image-link="/pmc/articles/PMC3599951/figure/F5/" class="imagepopup">Figure 5.  From: Molecular basis of 'hypoxic' breast cancer cell radio-sensitization: phytochemicals converge on radiation induced Rel <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic illustration of phytochemicals targeting NFkB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Curcumin, EF24, Resveratrol and Genistein targets more than one signal transduction molecules in NFkB <span class="highlight" style="background-color:">pathway</span> and comprehensively affect NFkB dependent tumor progression.</div></p></div><div class="aux"><div class="resc">Sheeja Aravindan, et al. Radiat Oncol. 2013;8:46-46.</div><p class="links"><a class="dblinks" href="/pubmed/23452621" ref="ordinalpos=1378&amp;ncbi_uid=4716726&amp;link_uid=23452621">Citation</a><a class="dblinks" href="/pmc/articles/PMC3599951/" ref="ordinalpos=1378&amp;ncbi_uid=4716726&amp;link_uid=PMC3599951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1379.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634213/figure/F2/" ref="ordinalpos=1379&amp;ncbi_uid=1709362&amp;link_uid=PMC2634213" href="/pmc/articles/PMC2634213/"><img src="/pmc/articles/instance/2634213/bin/psb0205_0365_fig002.gif" src-large="/pmc/articles/instance/2634213/bin/psb0205_0365_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634213/" ref="ordinalpos=1379&amp;ncbi_uid=1709362&amp;link_uid=PMC2634213" image-link="/pmc/articles/PMC2634213/figure/F2/" class="imagepopup">Figure 2.  From: ROS <span class="highlight" style="background-color:">Signaling</span> Pathways in Chilling Stress. </a></p><div class="supp"><p class="details"><div>Schematic representation of a model for chilling stress as ROS <span class="highlight" style="background-color:">signaling</span> and gene expression. A competing <span class="highlight" style="background-color:">pathway</span> based on GB action blocks ROS accumulation and alleviates chilling stress.</div></p></div><div class="aux"><div class="resc">John Einset, et al. Plant Signal Behav. 2007 Sep-Oct;2(5):365-367.</div><p class="links"><a class="dblinks" href="/pubmed/19704600" ref="ordinalpos=1379&amp;ncbi_uid=1709362&amp;link_uid=19704600">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634213/" ref="ordinalpos=1379&amp;ncbi_uid=1709362&amp;link_uid=PMC2634213">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1380.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC98123/figure/F8/" ref="ordinalpos=1380&amp;ncbi_uid=32964&amp;link_uid=PMC98123" href="/pmc/articles/PMC98123/"><img src="/pmc/articles/instance/98123/bin/ii0411244008.gif" src-large="/pmc/articles/instance/98123/bin/ii0411244008.jpg" alt="FIG. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC98123/" ref="ordinalpos=1380&amp;ncbi_uid=32964&amp;link_uid=PMC98123" image-link="/pmc/articles/PMC98123/figure/F8/" class="imagepopup">FIG. 8.  From: Induction of Inducible Nitric Oxide Synthase-NO? by Lipoarabinomannan of Mycobacterium tuberculosis Is Mediated  by MEK1-ERK, MKK7-JNK, and NF-?B <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Diagram of the proposed <span class="highlight" style="background-color:">signaling</span> pathways that modulate ManLAM-induced iNOS expression. The required IFN-γ costimulus <span class="highlight" style="background-color:">pathway</span> utilizing transcription factors IRF-1 and Stat1α is also shown.</div></p></div><div class="aux"><div class="resc">Edward D. Chan, et al. Infect Immun. 2001 April;69(4):2001-2010.</div><p class="links"><a class="dblinks" href="/pubmed/11254551" ref="ordinalpos=1380&amp;ncbi_uid=32964&amp;link_uid=11254551">Citation</a><a class="dblinks" href="/pmc/articles/PMC98123/" ref="ordinalpos=1380&amp;ncbi_uid=32964&amp;link_uid=PMC98123">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1381.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2667153/figure/F6/" ref="ordinalpos=1381&amp;ncbi_uid=1832289&amp;link_uid=PMC2667153" href="/pmc/articles/PMC2667153/"><img src="/pmc/articles/instance/2667153/bin/nihms84743f6.gif" src-large="/pmc/articles/instance/2667153/bin/nihms84743f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2667153/" ref="ordinalpos=1381&amp;ncbi_uid=1832289&amp;link_uid=PMC2667153" image-link="/pmc/articles/PMC2667153/figure/F6/" class="imagepopup">Figure 6. Bmp <span class="highlight" style="background-color:">signaling</span> is downstream of Wnt <span class="highlight" style="background-color:">signaling</span> in the maintenance of dorsal retinal markers.  From: Canonical Wnt <span class="highlight" style="background-color:">signaling</span> is required for maintenance of dorsal retinal identity. </a></p><div class="supp"><p class="details"><div>A–F: Bmp4 can rescue dorsal retinal markers in the absence of Wnt <span class="highlight" style="background-color:">signaling</span>. hs:Dkk1 and control WT embryos at the one-cell stage were injected with a construct that expresses Bmp4 upon heat-shock <i>(pDestTol2pA2;hsp70l:bmp4-IRES-GFP)</i>, heat-shocked at 12 hpf, and fixed at 24 hpf. A, B: To illustrate transgene expression following heat-shock, <i>in situ</i> hybridization was performed for <i>gfp</i>. Widespread clonal expression was observed in the retinas in 85% of embryos. C–F: Expression of Bmp4 caused a clear expansion of <i>tbx5</i> in embryos not expressing Dkk1 (D) and rescue of <i>tbx5</i> in embryos expressing Dkk1 (F). G–L: Activation of Wnt <span class="highlight" style="background-color:">signaling</span> does not rescue dorsal markers in the absence of Bmp <span class="highlight" style="background-color:">signaling</span>. Embryos heterozygous for the <i>Tg(hsp70l:nog3)<sup>fr14</sup></i> transgene, which express the Bmp <span class="highlight" style="background-color:">pathway</span> inhibitor Noggin upon heat-shock, were outcrossed to TL strain fish and placed in 200 mM LiCl at 18 hpf. A 2 hour heat-shock was performed at 18 hpf, and embryos were fixed at 24 hpf. To illustrate Wnt <span class="highlight" style="background-color:">pathway</span> activation, TOP:dGFP embryos were similarly treated with LiCl from 18–24 hpf and <i>gfp</i> detected by <i>in situ</i> hybridization (G, H). For hs:Noggin embryos untreated with LiCl, 55% of 49 embryos lost expression of <i>tbx5</i> (J), and for embryos treated with LiCl, 57% of 122 embryos lost expression of <i>tbx5</i> (L), showing that activation of Wnt <span class="highlight" style="background-color:">signaling</span> cannot rescue <i>tbx5</i> in the absence of Bmp <span class="highlight" style="background-color:">signaling</span>. Lateral views, dorsal up, anterior left.</div></p></div><div class="aux"><div class="resc">Eric S. Veien, et al. Development. ;135(24):4101-4111.</div><p class="links"><a class="dblinks" href="/pubmed/19004855" ref="ordinalpos=1381&amp;ncbi_uid=1832289&amp;link_uid=19004855">Citation</a><a class="dblinks" href="/pmc/articles/PMC2667153/" ref="ordinalpos=1381&amp;ncbi_uid=1832289&amp;link_uid=PMC2667153">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1382.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2907396/figure/pone-0011671-g003/" ref="ordinalpos=1382&amp;ncbi_uid=2369927&amp;link_uid=PMC2907396" href="/pmc/articles/PMC2907396/"><img src="/pmc/articles/instance/2907396/bin/pone.0011671.g003.gif" src-large="/pmc/articles/instance/2907396/bin/pone.0011671.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2907396/" ref="ordinalpos=1382&amp;ncbi_uid=2369927&amp;link_uid=PMC2907396" image-link="/pmc/articles/PMC2907396/figure/pone-0011671-g003/" class="imagepopup">Figure 3. T Cell Receptor <span class="highlight" style="background-color:">Signaling</span> canonical <span class="highlight" style="background-color:">pathway</span> from Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis..  From: Genome-Wide Expression Profiling Deciphers Host Responses Altered during Dengue Shock Syndrome and Reveals the Role of Innate Immunity in Severe Dengue. </a></p><div class="supp"><p class="details"><div>Genes in green and red are respectively under- and over-expressed in the DSS-gene signature. Genes in white are other genes present in the canonical <span class="highlight" style="background-color:">pathway</span> but absent from the DSS-gene signature. DSS: Dengue Shock Syndrome.</div></p></div><div class="aux"><div class="resc">Stéphanie Devignot, et al. PLoS One. 2010;5(7):e11671.</div><p class="links"><a class="dblinks" href="/pubmed/20652028" ref="ordinalpos=1382&amp;ncbi_uid=2369927&amp;link_uid=20652028">Citation</a><a class="dblinks" href="/pmc/articles/PMC2907396/" ref="ordinalpos=1382&amp;ncbi_uid=2369927&amp;link_uid=PMC2907396">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1383.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1851985/figure/pcbi-0030064-g006/" ref="ordinalpos=1383&amp;ncbi_uid=985535&amp;link_uid=PMC1851985" href="/pmc/articles/PMC1851985/"><img src="/pmc/articles/instance/1857804/bin/pcbi.0030064.g006.gif" src-large="/pmc/articles/instance/1857804/bin/pcbi.0030064.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1851985/" ref="ordinalpos=1383&amp;ncbi_uid=985535&amp;link_uid=PMC1851985" image-link="/pmc/articles/PMC1851985/figure/pcbi-0030064-g006/" class="imagepopup">Figure 6. Effect of <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activation on Cell Cycle Progression.  From: Cell Size at S Phase Initiation: An Emergent Property of the G1/S Network. </a></p><div class="supp"><p class="details"><div>The effects of the pheromone <span class="highlight" style="background-color:">pathway</span> (A,B) and the stress-response Hog1-dependent <span class="highlight" style="background-color:">pathway</span> (C,D) on Cdk1-Cln1,2<sub>cyt</sub> (left panels) and Cdk1-Clb5,6<sub>nuc</sub> (right panels) are reported. In each panel, basal wild-type is shown in black and “activated pathways” as a dotted line.</div></p></div><div class="aux"><div class="resc">Matteo Barberis, et al. PLoS Comput Biol. 2007 April;3(4):e64.</div><p class="links"><a class="dblinks" href="/pubmed/17432928" ref="ordinalpos=1383&amp;ncbi_uid=985535&amp;link_uid=17432928">Citation</a><a class="dblinks" href="/pmc/articles/PMC1851985/" ref="ordinalpos=1383&amp;ncbi_uid=985535&amp;link_uid=PMC1851985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1384.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2578836/figure/F5/" ref="ordinalpos=1384&amp;ncbi_uid=1556907&amp;link_uid=PMC2578836" href="/pmc/articles/PMC2578836/"><img src="/pmc/articles/instance/2578836/bin/nihms63415f5.gif" src-large="/pmc/articles/instance/2578836/bin/nihms63415f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2578836/" ref="ordinalpos=1384&amp;ncbi_uid=1556907&amp;link_uid=PMC2578836" image-link="/pmc/articles/PMC2578836/figure/F5/" class="imagepopup">Fig. 5.  From: Smad2 functions as a co-activator of canonical Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> independent of Smad4 through histone acetyltransferase activity of p300. </a></p><div class="supp"><p class="details"><div>Smad4-independent activity of Smad2 to enhance canonical Wnt/β-catenin/Tcf4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) OT reporter assays were performed in MDA-MB468 cells transiently transfected with the indicated expression vectors. Different amount of Smad2 expression vector (100, 500 ng/well) were used. (B) OT reporter assay was performed in 293T cells transiently transfected with the indicated expression vectors. Error bars indicate standard deviations of triplicate experiments.</div></p></div><div class="aux"><div class="resc">Morihisa Hirota, et al. Cell Signal. ;20(9):1632-1641.</div><p class="links"><a class="dblinks" href="/pubmed/18595660" ref="ordinalpos=1384&amp;ncbi_uid=1556907&amp;link_uid=18595660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2578836/" ref="ordinalpos=1384&amp;ncbi_uid=1556907&amp;link_uid=PMC2578836">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1385.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3092371/figure/F7/" ref="ordinalpos=1385&amp;ncbi_uid=3137248&amp;link_uid=PMC3092371" href="/pmc/articles/PMC3092371/"><img src="/pmc/articles/instance/3092371/bin/nihms270173f7.gif" src-large="/pmc/articles/instance/3092371/bin/nihms270173f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3092371/" ref="ordinalpos=1385&amp;ncbi_uid=3137248&amp;link_uid=PMC3092371" image-link="/pmc/articles/PMC3092371/figure/F7/" class="imagepopup">Figure 7.  From: Disrupting Function of FK506-Binding Protein 1b/12.6 Induces the Ca2+-Dysregulation Aging Phenotype in Hippocampal Neurons. </a></p><div class="supp"><p class="details"><div>Working model of the role of FKBP1b in aging-related Ca<sup>2+</sup> dyshomeostasis. <i><b>A</b></i>, In young subjects, neuronal FKBP1b exerts strong tonic inhibitory effects on two pathways: the Ca<sup>2+</sup>regulatory <span class="highlight" style="background-color:">pathway</span> and the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the Ca<sup>2+</sup>regulatory <span class="highlight" style="background-color:">pathway</span>, FKBP1b inhibits cytosolic Ca<sup>2+</sup> rises generated by extracellular influx via L-VGCCs (<i><b>1</b></i>) and intracellular release from RYRs (<i><b>2</b></i>), by inhibiting both channel types directly (<i><b>3</b></i>, <i><b>4</b></i>). In the mTOR <span class="highlight" style="background-color:">pathway</span>, FKBP inhibition of mTOR(<i><b>5</b></i>) helps maintain the balance between growth-stimulating and autophagy-suppressing effects of mTOR.<i><b>B</b></i>, With aging (mimicked by FKBP1b knockdown), a decline of FKBP1b expression/function leads to weakened inhibition of both pathways and concomitant increases in both Ca<sup>2+</sup> and mTOR <span class="highlight" style="background-color:">signaling</span>. Increased Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> results in dampened neuronal excitability and function, whereas mTOR disinhibition leads to aberrant increases in growth <span class="highlight" style="background-color:">signaling</span> as well as decreased autophagy. <i><b>C</b></i>, Rapamycin in young subjects exerts opposite effects on FKBP inhibition in the two pathways, mimicking the effect of aging of weakened FKBP inhibition in the Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> (<i><b>B</b></i>) but paradoxically augmenting FKBP inhibition of the mTOR <span class="highlight" style="background-color:">pathway</span> (<i><b>C</b></i>).</div></p></div><div class="aux"><div class="resc">John C. Gant, et al. J Neurosci. ;31(5):1693-1703.</div><p class="links"><a class="dblinks" href="/pubmed/21289178" ref="ordinalpos=1385&amp;ncbi_uid=3137248&amp;link_uid=21289178">Citation</a><a class="dblinks" href="/pmc/articles/PMC3092371/" ref="ordinalpos=1385&amp;ncbi_uid=3137248&amp;link_uid=PMC3092371">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1386.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4185420/figure/fig7/" ref="ordinalpos=1386&amp;ncbi_uid=6735069&amp;link_uid=PMC4185420" href="/pmc/articles/PMC4185420/"><img src="/pmc/articles/instance/4185420/bin/elife03626f007.gif" src-large="/pmc/articles/instance/4185420/bin/elife03626f007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4185420/" ref="ordinalpos=1386&amp;ncbi_uid=6735069&amp;link_uid=PMC4185420" image-link="/pmc/articles/PMC4185420/figure/fig7/" class="imagepopup">Figure 7.Schematic of the equilibrium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> demonstrating the proposed roles of Bip1, RpS8, and Nup98 in controlling Pvr..  From: Pvr expression regulators in equilibrium signal control and maintenance of Drosophila blood progenitors. </a></p><div class="supp"><p class="details"><div>Bip1, RpS8, and Nup98 are independently required for the expression of Pvr (direct arrows). Rescue of endogenous Pvr expression by misexpression of <i>bip1</i> in the <i>Nup98</i> RNAi background indicates that <i>bip1</i> functions genetically downstream of <i>Nup98</i> (dashed arrow) in the control of Pvr expression. Bip1 and RpS8 may work together in a complex (dashed line) to control Pvr expression in vivo. These components collectively comprise the known equilibrium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> working within the lymph gland to promote progenitor cell maintenance, along with the previously known Hh niche <span class="highlight" style="background-color:">signaling</span> mechanism.<br /><b>DOI:</b> http://dx.doi.org/10.7554/eLife.03626.017</div></p></div><div class="aux"><div class="resc">Bama Charan Mondal, et al. eLife. 2014;3:e03626.</div><p class="links"><a class="dblinks" href="/pubmed/25201876" ref="ordinalpos=1386&amp;ncbi_uid=6735069&amp;link_uid=25201876">Citation</a><a class="dblinks" href="/pmc/articles/PMC4185420/" ref="ordinalpos=1386&amp;ncbi_uid=6735069&amp;link_uid=PMC4185420">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1387.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC317062/figure/F2/" ref="ordinalpos=1387&amp;ncbi_uid=412650&amp;link_uid=PMC317062" href="/pmc/articles/PMC317062/"><img src="/pmc/articles/instance/317062/bin/gad.1f2.gif" src-large="/pmc/articles/instance/317062/bin/gad.1f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC317062/" ref="ordinalpos=1387&amp;ncbi_uid=412650&amp;link_uid=PMC317062" image-link="/pmc/articles/PMC317062/figure/F2/" class="imagepopup">Figure 2.  From: A PP2A regulatory subunit positively regulates Ras-mediated <span class="highlight" style="background-color:">signaling</span> during Caenorhabditis elegans vulval induction. </a></p><div class="supp"><p class="details"><div>Models for SUR-6 PP2A-B function in the vulval induction <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) SUR-6 PP2A-B is likely to regulate the activity of a PP2A core complex composed of a C and A subunit to influence Ras <span class="highlight" style="background-color:">signaling</span>. SUR-6 PP2A-B is a positive regulator of Ras <span class="highlight" style="background-color:">signaling</span>, so SUR-6 could function either to activate the PP2A core, which in turn activates Ras <span class="highlight" style="background-color:">signaling</span>, or to inhibit the PP2A core that normally inhibits Ras <span class="highlight" style="background-color:">signaling</span>. Potential targets for SUR-6 PP2A-B-regulated PP2A activity include LIN-45 Raf and KSR-1. (<i>B</i>) Model for the signal transduction <span class="highlight" style="background-color:">pathway</span> regulated by SUR-6. The receptor tyrosine kinase LET-23 is activated by an inductive signal that leads to the activation of LET-60 Ras. Ras–GTP recruits Raf kinase to the membrane and Raf phosphorylates and activates the MEK-2 MEK/MPK-1 MAPK kinase cascade. We showed that SUR-6 PP2A-B positively regulates <span class="highlight" style="background-color:">signaling</span> at a branch point at the level of Ras or Raf. KSR-1 and SUR-6 may work together to strengthen Ras <span class="highlight" style="background-color:">pathway</span> output during vulval cell fate specification.</div></p></div><div class="aux"><div class="resc">Derek S. Sieburth, et al. Genes Dev. 1999 October 1;13(19):2562-2569.</div><p class="links"><a class="dblinks" href="/pubmed/10521400" ref="ordinalpos=1387&amp;ncbi_uid=412650&amp;link_uid=10521400">Citation</a><a class="dblinks" href="/pmc/articles/PMC317062/" ref="ordinalpos=1387&amp;ncbi_uid=412650&amp;link_uid=PMC317062">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1388.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3266027/figure/F1/" ref="ordinalpos=1388&amp;ncbi_uid=4155223&amp;link_uid=PMC3266027" href="/pmc/articles/PMC3266027/"><img src="/pmc/articles/instance/3266027/bin/188fig1.gif" src-large="/pmc/articles/instance/3266027/bin/188fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3266027/" ref="ordinalpos=1388&amp;ncbi_uid=4155223&amp;link_uid=PMC3266027" image-link="/pmc/articles/PMC3266027/figure/F1/" class="imagepopup">Figure 1. From: Translating cancer 'omics' to improved outcomes. </a></p><div class="supp"><p class="details"><div>Activation of EGFR <span class="highlight" style="background-color:">signaling</span> in non-small-cell lung cancer (NSCLC) can occur via disruption of several different components at multiple levels of the <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) Different proteins in the EGFR <span class="highlight" style="background-color:">pathway</span> can be activated (red) or inactivated (blue) by underlying genetic or epigenetic changes at the DNA level, leading to aberrant <span class="highlight" style="background-color:">pathway</span> activity and oncogenic <span class="highlight" style="background-color:">signaling</span> in NSCLC. Examples of key oncogenes affected include <i>EGFR</i>, <i>RAS</i>, <i>PIK3CA</i>, and <i>AKT</i>. Conversely, examples of tumor suppressors that are inactivated include <i>PTEN</i> and <i>RASSF1</i>. (<i>B</i>) Genetic and epigenetic mechanisms responsible for the disruption of genes in the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in NSCLC include DNA copy number alterations (amplification or deletion), point mutations, and DNA methylation changes. Thus, it is important to consider multiple aspects of the genome and epigenome simultaneously to elucidate the mechanisms driving <span class="highlight" style="background-color:">pathway</span> deregulation. This illustration was generated using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software.</div></p></div><div class="aux"><div class="resc">Emily A. Vucic, et al. Genome Res. 2012 February;22(2):188-195.</div><p class="links"><a class="dblinks" href="/pubmed/22301133" ref="ordinalpos=1388&amp;ncbi_uid=4155223&amp;link_uid=22301133">Citation</a><a class="dblinks" href="/pmc/articles/PMC3266027/" ref="ordinalpos=1388&amp;ncbi_uid=4155223&amp;link_uid=PMC3266027">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1389.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2993470/figure/F1/" ref="ordinalpos=1389&amp;ncbi_uid=3596336&amp;link_uid=PMC2993470" href="/pmc/articles/PMC2993470/"><img src="/pmc/articles/instance/2993470/bin/zmo0121062480001.gif" src-large="/pmc/articles/instance/2993470/bin/zmo0121062480001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2993470/" ref="ordinalpos=1389&amp;ncbi_uid=3596336&amp;link_uid=PMC2993470" image-link="/pmc/articles/PMC2993470/figure/F1/" class="imagepopup">Fig. 1. From: Functional Selectivity in Adrenergic and Angiotensin <span class="highlight" style="background-color:">Signaling</span> Systems. </a></p><div class="supp"><p class="details"><div>Collateral efficacy in GPCR <span class="highlight" style="background-color:">signaling</span>. In the traditional paradigm of linear efficacy, a given agonist or antagonist for a GPCR initiates or inhibits a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> before the process is terminated by regulatory proteins such as GRKs and β-arrestins. In collateral or pluridimensional efficacy, a given ligand may be an antagonist for one <span class="highlight" style="background-color:">pathway</span> whereas simultaneously permitting activation of other pathways through G protein-dependent and -independent <span class="highlight" style="background-color:">signaling</span>. Therefore, multiple functions may be seen via a single receptor-ligand interaction and a subsequent unique receptor conformation change.</div></p></div><div class="aux"><div class="resc">Chetan B. Patel, et al. Mol Pharmacol. 2010 December;78(6):983-992.</div><p class="links"><a class="dblinks" href="/pubmed/20855464" ref="ordinalpos=1389&amp;ncbi_uid=3596336&amp;link_uid=20855464">Citation</a><a class="dblinks" href="/pmc/articles/PMC2993470/" ref="ordinalpos=1389&amp;ncbi_uid=3596336&amp;link_uid=PMC2993470">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1390.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2794996/figure/F5/" ref="ordinalpos=1390&amp;ncbi_uid=2098714&amp;link_uid=PMC2794996" href="/pmc/articles/PMC2794996/"><img src="/pmc/articles/instance/2794996/bin/nihms139263f5.gif" src-large="/pmc/articles/instance/2794996/bin/nihms139263f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2794996/" ref="ordinalpos=1390&amp;ncbi_uid=2098714&amp;link_uid=PMC2794996" image-link="/pmc/articles/PMC2794996/figure/F5/" class="imagepopup">Figure 5. Alteration in TLR4 <span class="highlight" style="background-color:">signaling</span> in ovarian carcinoma cell lines treated with LPS or PTX.  From: TLR4 <span class="highlight" style="background-color:">signaling</span> induced by lipopolysacharide or paclitaxel regulates tumor survival and chemoresistance in ovarian cancer. </a></p><div class="supp"><p class="details"><div>Western blot of the cells which were either untreated or treated with LPS or PTX for 30 and 60 min. Strong up-regulation of p-cJun was observed only in SKOV3 cells treated with LPS or PTX but not in A2780 cells, where only the TRIF <span class="highlight" style="background-color:">pathway</span> is activated in response to LPS or PTX <span class="highlight" style="background-color:">signaling</span> via the TLR4.</div></p></div><div class="aux"><div class="resc">Marta Szajnik, et al. Oncogene. ;28(49):4353-4363.</div><p class="links"><a class="dblinks" href="/pubmed/19826413" ref="ordinalpos=1390&amp;ncbi_uid=2098714&amp;link_uid=19826413">Citation</a><a class="dblinks" href="/pmc/articles/PMC2794996/" ref="ordinalpos=1390&amp;ncbi_uid=2098714&amp;link_uid=PMC2794996">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1391.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3653798/figure/pbio-1001562-g007/" ref="ordinalpos=1391&amp;ncbi_uid=4916161&amp;link_uid=PMC3653798" href="/pmc/articles/PMC3653798/"><img src="/pmc/articles/instance/3653798/bin/pbio.1001562.g007.gif" src-large="/pmc/articles/instance/3653798/bin/pbio.1001562.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3653798/" ref="ordinalpos=1391&amp;ncbi_uid=4916161&amp;link_uid=PMC3653798" image-link="/pmc/articles/PMC3653798/figure/pbio-1001562-g007/" class="imagepopup">Figure 7. APPL is a novel modulator of Wnt-PCP <span class="highlight" style="background-color:">signaling</span> required for axon guidance..  From: The Drosophila Homologue of the Amyloid Precursor Protein Is a Conserved Modulator of Wnt PCP <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of the proposed model. APPL is a novel regulator of Wnt-PCP <span class="highlight" style="background-color:">pathway</span>. In the presence of Wnt <span class="highlight" style="background-color:">signaling</span>, Fz binds Dsh, which needs to be phosphorylated by Abelson kinase to correctly relocalize to the membrane. We propose that Appl is part of the membrane complex formed by the core PCP proteins Fz1 and Vang. In turn, Appl recruits Abelson kinase to the complex and positively modulates Dsh phosphorylation. The subsequent activation of the <span class="highlight" style="background-color:">signaling</span> is required for MB β-axon growth.</div></p></div><div class="aux"><div class="resc">Alessia Soldano, et al. PLoS Biol. 2013 May;11(5):e1001562.</div><p class="links"><a class="dblinks" href="/pubmed/23690751" ref="ordinalpos=1391&amp;ncbi_uid=4916161&amp;link_uid=23690751">Citation</a><a class="dblinks" href="/pmc/articles/PMC3653798/" ref="ordinalpos=1391&amp;ncbi_uid=4916161&amp;link_uid=PMC3653798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1392.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2703902/figure/F2/" ref="ordinalpos=1392&amp;ncbi_uid=1926060&amp;link_uid=PMC2703902" href="/pmc/articles/PMC2703902/"><img src="/pmc/articles/instance/2703902/bin/gkp303f2.gif" src-large="/pmc/articles/instance/2703902/bin/gkp303f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2703902/" ref="ordinalpos=1392&amp;ncbi_uid=1926060&amp;link_uid=PMC2703902" image-link="/pmc/articles/PMC2703902/figure/F2/" class="imagepopup">Figure 2. From: LitInspector: literature and signal transduction <span class="highlight" style="background-color:">pathway</span> mining in PubMed abstracts. </a></p><div class="supp"><p class="details"><div>Screenshot of the LitInspector signal transduction <span class="highlight" style="background-color:">pathway</span> mining output for the gene ACP1 (acid phosphatase 1). (<b>A</b>) List of the found <span class="highlight" style="background-color:">signaling</span> pathways sorted by the number of references. The user has full access to the color-coded PubMed sentences from which the <span class="highlight" style="background-color:">pathway</span> associations have been derived from. (<b>B</b>) In case of ACP1 most references were found for TCR (T-cell receptor) <span class="highlight" style="background-color:">signaling</span> (eight references).</div></p></div><div class="aux"><div class="resc">Matthias Frisch, et al. Nucleic Acids Res. 2009 July 1;37(Web Server issue):W135-W140.</div><p class="links"><a class="dblinks" href="/pubmed/19417065" ref="ordinalpos=1392&amp;ncbi_uid=1926060&amp;link_uid=19417065">Citation</a><a class="dblinks" href="/pmc/articles/PMC2703902/" ref="ordinalpos=1392&amp;ncbi_uid=1926060&amp;link_uid=PMC2703902">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1393.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4253415/figure/F1/" ref="ordinalpos=1393&amp;ncbi_uid=6903267&amp;link_uid=PMC4253415" href="/pmc/articles/PMC4253415/"><img src="/pmc/articles/instance/4253415/bin/oncotarget-05-9022-g001.gif" src-large="/pmc/articles/instance/4253415/bin/oncotarget-05-9022-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4253415/" ref="ordinalpos=1393&amp;ncbi_uid=6903267&amp;link_uid=PMC4253415" image-link="/pmc/articles/PMC4253415/figure/F1/" class="imagepopup">Figure 1. Representation of the Correlation coefficients for <span class="highlight" style="background-color:">pathway</span> activation states depending on the normal/cancerous nature of BC tissue specimens, calculated for the PAS values of the different intracellular <span class="highlight" style="background-color:">signaling</span> pathways for the Comparisons (i) and (ii).  From: Novel robust biomarkers for human bladder cancer based on activation of intracellular <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Negative correlation coefficients mean downregulation of a <span class="highlight" style="background-color:">pathway</span> in BC, positive – upregulation. Correlation Moscow/Moscow means correlation coefficients calculated for the Comparison (i); correlation Leth/Leth means correlation coefficients calculated for the Comparison (ii).</div></p></div><div class="aux"><div class="resc">Ksenia Lezhnina, et al. Oncotarget. 2014 October;5(19):9022-9032.</div><p class="links"><a class="dblinks" href="/pubmed/25296972" ref="ordinalpos=1393&amp;ncbi_uid=6903267&amp;link_uid=25296972">Citation</a><a class="dblinks" href="/pmc/articles/PMC4253415/" ref="ordinalpos=1393&amp;ncbi_uid=6903267&amp;link_uid=PMC4253415">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1394.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4308835/figure/Fig2/" ref="ordinalpos=1394&amp;ncbi_uid=7106961&amp;link_uid=PMC4308835" href="/pmc/articles/PMC4308835/"><img src="/pmc/articles/instance/4308835/bin/13075_2015_527_Fig2_HTML.gif" src-large="/pmc/articles/instance/4308835/bin/13075_2015_527_Fig2_HTML.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4308835/" ref="ordinalpos=1394&amp;ncbi_uid=7106961&amp;link_uid=PMC4308835" image-link="/pmc/articles/PMC4308835/figure/Fig2/" class="imagepopup">Figure 2.  From: Non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> in rheumatoid arthritis: Dr Jekyll and Mr Hyde?. </a></p><div class="supp"><p class="details"><div> <b>Non-canonical nuclear factor-</b> <b>κ</b> <b>B <span class="highlight" style="background-color:">signaling</span> in different cell types in rheumatoid arthritis: Dr Jekyll or Mr Hyde?</b> Schematic overview of the role of the non-canonical nuclear factor (NF)-κB <span class="highlight" style="background-color:">pathway</span> in cell types involved in the pathogenesis of rheumatoid arthritis. On the left side (Dr Jekyll), cells in which non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> plays an anti-inflammatory role. In the middle (on the dashed line), cells in which this <span class="highlight" style="background-color:">pathway</span> plays a dual role, which is largely dependent on the microenvironment. On the right side (Mr Hyde), cells in which non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> is pro-inflammatory. Treg, regulatory T cell.</div></p></div><div class="aux"><div class="resc">Ae R Noort, et al. Arthritis Res Ther. 2015;17(1):15.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1394&amp;ncbi_uid=7106961&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4308835/" ref="ordinalpos=1394&amp;ncbi_uid=7106961&amp;link_uid=PMC4308835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1395.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2727867/figure/F1/" ref="ordinalpos=1395&amp;ncbi_uid=1951741&amp;link_uid=PMC2727867" href="/pmc/articles/PMC2727867/"><img src="/pmc/articles/instance/2727867/bin/nihms136752f1.gif" src-large="/pmc/articles/instance/2727867/bin/nihms136752f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2727867/" ref="ordinalpos=1395&amp;ncbi_uid=1951741&amp;link_uid=PMC2727867" image-link="/pmc/articles/PMC2727867/figure/F1/" class="imagepopup">Figure 1.  From: The Important Role of the Apoptotic Effects of Zinc in the Development of Cancers. </a></p><div class="supp"><p class="details"><div>Representation of some zinc-induced apoptotic pathways. A. Direct effects on nucleus. B. Direct effects on mitochondria. C. Direct effect on apoptotic <span class="highlight" style="background-color:">signaling</span> pathways f1,f2,f3. C1/C2. Apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> effects on nucleus or mitochondria. D. Effects of mitochondrial factors on nucleus. E1/E2. Effects of mitochondrial factors mediated through the apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Renty B. Franklin, et al. J Cell Biochem. ;106(5):750-757.</div><p class="links"><a class="dblinks" href="/pubmed/19160419" ref="ordinalpos=1395&amp;ncbi_uid=1951741&amp;link_uid=19160419">Citation</a><a class="dblinks" href="/pmc/articles/PMC2727867/" ref="ordinalpos=1395&amp;ncbi_uid=1951741&amp;link_uid=PMC2727867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1396.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4299277/figure/DEV112797F7/" ref="ordinalpos=1396&amp;ncbi_uid=7111258&amp;link_uid=PMC4299277" href="/pmc/articles/PMC4299277/"><img src="/pmc/articles/instance/4299277/bin/develop-141-112797-g7.gif" src-large="/pmc/articles/instance/4299277/bin/develop-141-112797-g7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4299277/" ref="ordinalpos=1396&amp;ncbi_uid=7111258&amp;link_uid=PMC4299277" image-link="/pmc/articles/PMC4299277/figure/DEV112797F7/" class="imagepopup">Fig. 7. From: Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> integrates patterning and metabolism of the insect growth zone. </a></p><div class="supp"><p class="details"><div><b>Summary of early Wnt and Hh <span class="highlight" style="background-color:">signaling</span> functions in head and GZ.</b> The Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> has an early function in axis formation (left). During blastoderm stages, posterior Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> is required for GZ establishment and posterior patterning. Anterior repression of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> is required for anterior development, with the extraembryonic serosa and the anterior embryo being most sensitive to elevated Wnt activity. Hh <span class="highlight" style="background-color:">signaling</span> does not appear to play a role at this stage. At the germ rudiment stage (right), posterior Wnt <span class="highlight" style="background-color:">signaling</span> maintains its own activity and initiates Hh <span class="highlight" style="background-color:">signaling</span> in the GZ. At the same time, adjacent stripes of <i>Tc-wg</i> and <i>Tc-hh</i> expression arise in the head anlagen, with Hh <span class="highlight" style="background-color:">signaling</span> being required for the initiation of <i>Tc-wg</i> expression. Later, mutual activation maintains this boundary. At this stage, we detected only a few target genes for both pathways in the head. The posterior Hh target gene set is largely non-overlapping with the Wnt set but its function remains elusive. Posterior Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> plays a central role in posterior development. First, it activates genes required for pattern formation in the GZ. This includes segmentation of the ectoderm, the formation of the mesoderm and the hindgut. Second, the expression of genes required for protein metabolism is enhanced by Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. Finally, genes required for cell division are regulated by Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. Hence, Wnt/β-catenin integrates patterning, metabolism and growth. Note that later functions of these pathways (such as parasegment boundary formation) are not depicted in this scheme and that the expression of further Wnt ligands has been omitted for simplicity. See text for further details. ant, anterior; post, posterior.</div></p></div><div class="aux"><div class="resc">Georg Oberhofer, et al. Development. 2014 December 15;141(24):4740-4750.</div><p class="links"><a class="dblinks" href="/pubmed/25395458" ref="ordinalpos=1396&amp;ncbi_uid=7111258&amp;link_uid=25395458">Citation</a><a class="dblinks" href="/pmc/articles/PMC4299277/" ref="ordinalpos=1396&amp;ncbi_uid=7111258&amp;link_uid=PMC4299277">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1397.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2776706/figure/F1/" ref="ordinalpos=1397&amp;ncbi_uid=2110102&amp;link_uid=PMC2776706" href="/pmc/articles/PMC2776706/"><img src="/pmc/articles/instance/2776706/bin/nihms-146568-f0001.gif" src-large="/pmc/articles/instance/2776706/bin/nihms-146568-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2776706/" ref="ordinalpos=1397&amp;ncbi_uid=2110102&amp;link_uid=PMC2776706" image-link="/pmc/articles/PMC2776706/figure/F1/" class="imagepopup">Figure 1.  From: Navigating the network: <span class="highlight" style="background-color:">signaling</span> cross-talk in hematopoietic cells. </a></p><div class="supp"><p class="details"><div>Topology of <span class="highlight" style="background-color:">signaling</span> networks and interpretation of experimental readouts. (<b>a</b>) Representation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> schema from simple linear pathways (R<sup>x</sup>), to branched pathways downstream of multicomponent receptor complexes (R<sup>y</sup>), to <span class="highlight" style="background-color:">signaling</span> networks (R<sup>z</sup>). Multiple pathways can either induce distinct transcriptional signatures or converge on common genes. Feedback from induced transcripts can occur at the level of autocrine and/or paracrine actions of secreted proteins or at the level of direct effects on cytoplasmic <span class="highlight" style="background-color:">signaling</span> components. Several aspects of network architecture include the importance of scaffold proteins for interpathway insulation and how multiple pathways may ‘share’ common <span class="highlight" style="background-color:">signaling</span> components. <b>(b</b>) Continuous gene-versus-phenotype relationship from genome-wide RNAi screen for <span class="highlight" style="background-color:">signaling</span> regulators in <i>Drosophila melanogaster</i> cells (reproduced from ref. 41). The effect of dsRNA targeting 24,000 genes is represented as a <i>Z</i>-score relative to control Erk activation. Canonical <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> components and regulators are identified at the extremes of the distribution. The physiological influence of genes in the central section is discussed in the text. (<b>c</b>) Effects of various degrees of stimulus on a hypothetical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> ‘canonical’ target and an ‘off-<span class="highlight" style="background-color:">pathway</span>’ target. Whereas the off-<span class="highlight" style="background-color:">pathway</span> target can induce a measurable response under high-dose conditions, only the canonical target induces a response in the physiological dose range.</div></p></div><div class="aux"><div class="resc">Iain D C Fraser, et al. Nat Immunol. ;10(4):327-331.</div><p class="links"><a class="dblinks" href="/pubmed/19295628" ref="ordinalpos=1397&amp;ncbi_uid=2110102&amp;link_uid=19295628">Citation</a><a class="dblinks" href="/pmc/articles/PMC2776706/" ref="ordinalpos=1397&amp;ncbi_uid=2110102&amp;link_uid=PMC2776706">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1398.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3692501/figure/pone-0066268-g007/" ref="ordinalpos=1398&amp;ncbi_uid=5061716&amp;link_uid=PMC3692501" href="/pmc/articles/PMC3692501/"><img src="/pmc/articles/instance/3692501/bin/pone.0066268.g007.gif" src-large="/pmc/articles/instance/3692501/bin/pone.0066268.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3692501/" ref="ordinalpos=1398&amp;ncbi_uid=5061716&amp;link_uid=PMC3692501" image-link="/pmc/articles/PMC3692501/figure/pone-0066268-g007/" class="imagepopup">Figure 7. Schematic of the proposed signal transduction <span class="highlight" style="background-color:">pathway</span> of Mcl-1L induction in hepatocytes during liver regeneration..  From: IL-6 Regulates Mcl-1L Expression through the JAK/PI3K/Akt/CREB <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Hepatocytes: Implication of an Anti-Apoptotic Role during Liver Regeneration. </a></p><div class="supp"><p class="details"><div>IL-6 regulates Mcl-1L expression through the IL-6 dependent JAK/PI3K/Akt/CREB activation <span class="highlight" style="background-color:">pathway</span> in rat hepatocytes.</div></p></div><div class="aux"><div class="resc">Chia-Hung Chou, et al. PLoS One. 2013;8(6):e66268.</div><p class="links"><a class="dblinks" href="/pubmed/23825534" ref="ordinalpos=1398&amp;ncbi_uid=5061716&amp;link_uid=23825534">Citation</a><a class="dblinks" href="/pmc/articles/PMC3692501/" ref="ordinalpos=1398&amp;ncbi_uid=5061716&amp;link_uid=PMC3692501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1399.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3370224/figure/F9/" ref="ordinalpos=1399&amp;ncbi_uid=4976455&amp;link_uid=PMC3370224" href="/pmc/articles/PMC3370224/"><img src="/pmc/articles/instance/3370224/bin/zbc0251211860009.gif" src-large="/pmc/articles/instance/3370224/bin/zbc0251211860009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3370224/" ref="ordinalpos=1399&amp;ncbi_uid=4976455&amp;link_uid=PMC3370224" image-link="/pmc/articles/PMC3370224/figure/F9/" class="imagepopup">FIGURE 9. From: ARF6 Activated by the LHCG Receptor through the Cytohesin Family of Guanine Nucleotide Exchange Factors Mediates the Receptor Internalization and <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the ARF6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in HLHCGR internalization as deduced from the present study.</b></div></p></div><div class="aux"><div class="resc">Venkateswarlu Kanamarlapudi, et al. J Biol Chem. 2012 June 8;287(24):20443-20455.</div><p class="links"><a class="dblinks" href="/pubmed/22523074" ref="ordinalpos=1399&amp;ncbi_uid=4976455&amp;link_uid=22523074">Citation</a><a class="dblinks" href="/pmc/articles/PMC3370224/" ref="ordinalpos=1399&amp;ncbi_uid=4976455&amp;link_uid=PMC3370224">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1400.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2897923/figure/F2/" ref="ordinalpos=1400&amp;ncbi_uid=3186731&amp;link_uid=PMC2897923" href="/pmc/articles/PMC2897923/"><img src="/pmc/articles/instance/2897923/bin/nihms188775f2.gif" src-large="/pmc/articles/instance/2897923/bin/nihms188775f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2897923/" ref="ordinalpos=1400&amp;ncbi_uid=3186731&amp;link_uid=PMC2897923" image-link="/pmc/articles/PMC2897923/figure/F2/" class="imagepopup">Figure 2. The Wnt/PCP <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Points of Intervention.  From: Striking the Target in Wnt-y Conditions: Intervening in Wnt <span class="highlight" style="background-color:">Signaling</span> During Cancer Progression. </a></p><div class="supp"><p class="details"><div>Receptor activation by Wnt leads to the recruitment of Dsv, resulting in the activation of Rac1 and RhoA, both of which are involved in cytoskeletton remodeling and establishment of an EMT phenotype.</div></p></div><div class="aux"><div class="resc">Tura C. Camilli, et al. Biochem Pharmacol. ;80(5):702-711.</div><p class="links"><a class="dblinks" href="/pubmed/20211149" ref="ordinalpos=1400&amp;ncbi_uid=3186731&amp;link_uid=20211149">Citation</a><a class="dblinks" href="/pmc/articles/PMC2897923/" ref="ordinalpos=1400&amp;ncbi_uid=3186731&amp;link_uid=PMC2897923">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="13" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="14" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="15" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:24:50-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A4E7753FEC2D10000000000E83C91;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A4E7753FEC2D10000000000E83C91" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>