
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="9" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC40E053FEBB3100000000004B3DC5" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="8" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="9" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="10" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="9" /></div>
            <div><h2 class="result_count">Results: 801 to 900 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>801.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3056141/figure/Fig1/" ref="ordinalpos=801&amp;ncbi_uid=2875696&amp;link_uid=PMC3056141" href="/pmc/articles/PMC3056141/"><img src="/pmc/articles/instance/3056141/bin/18_2010_605_Fig1_HTML.gif" src-large="/pmc/articles/instance/3056141/bin/18_2010_605_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3056141/" ref="ordinalpos=801&amp;ncbi_uid=2875696&amp;link_uid=PMC3056141" image-link="/pmc/articles/PMC3056141/figure/Fig1/" class="imagepopup">Fig. 1.  From: The STING <span class="highlight" style="background-color:">pathway</span> and regulation of innate immune <span class="highlight" style="background-color:">signaling</span> in response to DNA pathogens. </a></p><div class="supp"><p class="details"><div>DNA-mediated activation of innate immune <span class="highlight" style="background-color:">signaling</span>. <b>a</b> TLR9-dependent innate immune <span class="highlight" style="background-color:">signaling</span>. TLR9 localizes in the ER and interacts with UNC-93B, which mediates TLR9 translocation. Upon stimulation of DNA-containing CpG motifs (CpG DNA) including viral genome, TLR9 traffics from ER to endosome to contact CpG DNA, and recruits a <span class="highlight" style="background-color:">signaling</span> complex consisting of MyD88, IRAK4, and TRAF6. TRAF6 in turn activates TAK1, and TAK1 subsequently activates MAPK and the IKK complex (IKKα, IKKβ, and IKKγ) to activate NF-κB. NF-κB and MAPK regulates inflammatory cytokines expression. IRAK4 activates TRAF3 and IRAK1, which catalyze IRF7 phosphorylation and induce type I IFNs expression. <b>b</b> Cytosolic DNA-mediated innate immune <span class="highlight" style="background-color:">signaling</span>. Cytosolic AT-rich dsDNA is recognized by RNA PolIII and transcribed into 5′ triphosphate RNA, which activates RIG-I/IPS-1 <span class="highlight" style="background-color:">signaling</span>. IPS1 interacts with FADD/RIP1, which activate NF-κB. IPS-1 also activates TBK1 which phosphorylates IRF-3 and induces type I IFNs expression. AT-rich dsDNA or non-AT-rich dsDNA is recognized by unknown receptor, and activate STING-dependent <span class="highlight" style="background-color:">signaling</span>. STING localizes in the ER with translocon complex (TRAPβ, Sec61), and activate Sec5/TBK1 to induce type I IFNs. STING also activates NF-κB to induce pro-inflammatory cytokines. dsDNA also activates AIM2/ASC/Caspase-1 inflammasome <span class="highlight" style="background-color:">pathway</span> to activate IL-1β</div></p></div><div class="aux"><div class="resc">Hiroki Ishikawa, et al. Cell Mol Life Sci. 2011 April;68(7):1157-1165.</div><p class="links"><a class="dblinks" href="/pubmed/21161320" ref="ordinalpos=801&amp;ncbi_uid=2875696&amp;link_uid=21161320">Citation</a><a class="dblinks" href="/pmc/articles/PMC3056141/" ref="ordinalpos=801&amp;ncbi_uid=2875696&amp;link_uid=PMC3056141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>802.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3696550/figure/F6/" ref="ordinalpos=802&amp;ncbi_uid=5055774&amp;link_uid=PMC3696550" href="/pmc/articles/PMC3696550/"><img src="/pmc/articles/instance/3696550/bin/JCI66203.f6.gif" src-large="/pmc/articles/instance/3696550/bin/JCI66203.f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3696550/" ref="ordinalpos=802&amp;ncbi_uid=5055774&amp;link_uid=PMC3696550" image-link="/pmc/articles/PMC3696550/figure/F6/" class="imagepopup">Figure 6. Representation of the BCR and TCR <span class="highlight" style="background-color:">signaling</span> pathways and their regulation through DNA methylation..  From: Integrated genetic and epigenetic analysis of childhood acute lymphoblastic leukemia. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Schematic representation of the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which was significantly differentially methylated in B-ALL versus T-ALL (DAVID <i>P</i> = 0.00064). Genes that were hypomethylated or overexpressed in B-ALL are highlighted red; genes that were hypermethylated or silenced in B-ALL are highlighted green. (<b>B</b>) Schematic representation of the TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (DAVID <i>P</i> = 0.029). Genes that were hypomethylated or overexpressed in T-ALL are highlighted red; genes that were hypermethylated or silenced in T-ALL are highlighted green. </div></p></div><div class="aux"><div class="resc">Maria E. Figueroa, et al. J Clin Invest. 2013 July 1;123(7):3099-3111.</div><p class="links"><a class="dblinks" href="/pubmed/23921123" ref="ordinalpos=802&amp;ncbi_uid=5055774&amp;link_uid=23921123">Citation</a><a class="dblinks" href="/pmc/articles/PMC3696550/" ref="ordinalpos=802&amp;ncbi_uid=5055774&amp;link_uid=PMC3696550">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>803.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151254/figure/fig6/" ref="ordinalpos=803&amp;ncbi_uid=247072&amp;link_uid=PMC151254" href="/pmc/articles/PMC151254/"><img src="/pmc/articles/instance/151254/bin/00-0596f6.gif" src-large="/pmc/articles/instance/151254/bin/00-0596f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151254/" ref="ordinalpos=803&amp;ncbi_uid=247072&amp;link_uid=PMC151254" image-link="/pmc/articles/PMC151254/figure/fig6/" class="imagepopup">Figure 6. From: Cell <span class="highlight" style="background-color:">Signaling</span> during Cold, Drought, and Salt Stress. </a></p><div class="supp"><p class="details"><div>Major Types of <span class="highlight" style="background-color:">Signaling</span> for Plants during Cold, Drought, and Salt Stress.<br />Representative cascades, outputs, biological functions, and examples of mutants with phenotypes indicative of defects in the respective biological functions are shown. Type I <span class="highlight" style="background-color:">signaling</span> involves the generation of ROS scavenging enzymes and antioxidant compounds as well as osmolytes. The involvement of a MAPK <span class="highlight" style="background-color:">pathway</span> in the production of osmolytes in plants has not been demonstrated experimentally. Under osmotic stress, altered MAPK <span class="highlight" style="background-color:">signaling</span> may contribute to changed cell cycle regulation and growth retardation. Type II <span class="highlight" style="background-color:">signaling</span> involves the production of stress-responsive proteins mostly of undefined functions. Pathways within Type II <span class="highlight" style="background-color:">signaling</span> are shown in Figure 4. Type III <span class="highlight" style="background-color:">signaling</span> involves the SOS <span class="highlight" style="background-color:">pathway</span> which is specific to ionic stress. <span class="highlight" style="background-color:">Signaling</span> events for homologs of SOS3 (SCaBP) and SO2 (PKS) are tentatively grouped with SOS3 and SOS2, yet these SCaBP-PKS pathways are not necessarily related to ion homeostasis. Connections between different types of <span class="highlight" style="background-color:">signaling</span> events are indicated with dashed lines. Arrows indicate the direction of signal flux. Primary sensors are shown to be localized in the membrane. Receptors for secondary <span class="highlight" style="background-color:">signaling</span> molecules (2ndSM) are not shown.</div></p></div><div class="aux"><div class="resc">Liming Xiong, et al. Plant Cell. 2002;14(Suppl):s165-s183.</div><p class="links"><a class="dblinks" href="/pubmed/12045276" ref="ordinalpos=803&amp;ncbi_uid=247072&amp;link_uid=12045276">Citation</a><a class="dblinks" href="/pmc/articles/PMC151254/" ref="ordinalpos=803&amp;ncbi_uid=247072&amp;link_uid=PMC151254">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>804.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3402511/figure/pone-0040028-g004/" ref="ordinalpos=804&amp;ncbi_uid=4141844&amp;link_uid=PMC3402511" href="/pmc/articles/PMC3402511/"><img src="/pmc/articles/instance/3402511/bin/pone.0040028.g004.gif" src-large="/pmc/articles/instance/3402511/bin/pone.0040028.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3402511/" ref="ordinalpos=804&amp;ncbi_uid=4141844&amp;link_uid=PMC3402511" image-link="/pmc/articles/PMC3402511/figure/pone-0040028-g004/" class="imagepopup">Figure 4. The insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is predicted to be targeted by many aging-regulated novel miRNA candidates..  From: Novel MicroRNAs Differentially Expressed during Aging in the Mouse Brain. </a></p><div class="supp"><p class="details"><div>Multiple novel miRNA candidates are predicted to target each of the genes implicated in the <span class="highlight" style="background-color:">pathway</span>, and each novel miRNA candidate is predicted to target multiple genes in the <span class="highlight" style="background-color:">pathway</span>. In red: upregulated (&gt;2.0-fold) novel miRNAs; in black: downregulated (&gt;2.0-fold) novel miRNAs.</div></p></div><div class="aux"><div class="resc">Sachi Inukai, et al. PLoS One. 2012;7(7):e40028.</div><p class="links"><a class="dblinks" href="/pubmed/22844398" ref="ordinalpos=804&amp;ncbi_uid=4141844&amp;link_uid=22844398">Citation</a><a class="dblinks" href="/pmc/articles/PMC3402511/" ref="ordinalpos=804&amp;ncbi_uid=4141844&amp;link_uid=PMC3402511">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>805.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2656612/figure/pone-0004979-g008/" ref="ordinalpos=805&amp;ncbi_uid=1689088&amp;link_uid=PMC2656612" href="/pmc/articles/PMC2656612/"><img src="/pmc/articles/instance/2656612/bin/pone.0004979.g008.gif" src-large="/pmc/articles/instance/2656612/bin/pone.0004979.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2656612/" ref="ordinalpos=805&amp;ncbi_uid=1689088&amp;link_uid=PMC2656612" image-link="/pmc/articles/PMC2656612/figure/pone-0004979-g008/" class="imagepopup">Figure 8. A hypothesized signal <span class="highlight" style="background-color:">pathway</span> that Mtmr8 regulates zebrafish muscle development..  From: Cooperation of Mtmr8 with PI3K Regulates Actin Filament Modeling and Muscle Development in Zebrafish. </a></p><div class="supp"><p class="details"><div>Knockdown of <i>Mtmr8</i> promotes PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and prevents Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The two pathways may interact with each other in regulating F-actin modeling and muscle development.</div></p></div><div class="aux"><div class="resc">Jie Mei, et al. PLoS ONE. 2009;4(3):e4979.</div><p class="links"><a class="dblinks" href="/pubmed/19325702" ref="ordinalpos=805&amp;ncbi_uid=1689088&amp;link_uid=19325702">Citation</a><a class="dblinks" href="/pmc/articles/PMC2656612/" ref="ordinalpos=805&amp;ncbi_uid=1689088&amp;link_uid=PMC2656612">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>806.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3104248/figure/F2/" ref="ordinalpos=806&amp;ncbi_uid=3004998&amp;link_uid=PMC3104248" href="/pmc/articles/PMC3104248/"><img src="/pmc/articles/instance/3104248/bin/emm-43-231-g002.gif" src-large="/pmc/articles/instance/3104248/bin/emm-43-231-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3104248/" ref="ordinalpos=806&amp;ncbi_uid=3004998&amp;link_uid=PMC3104248" image-link="/pmc/articles/PMC3104248/figure/F2/" class="imagepopup">Figure 2.  From: Frontier of Epilepsy Research - mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Genetic mutations of <span class="highlight" style="background-color:">signaling</span> molecules implicated in mTOR <span class="highlight" style="background-color:">pathway</span> (red Xs). Red arrows (up- or downward) indicate the changes in activity of particular molecules in epileptic conditions. Causatives for acquired epilepsy are described in gray. Therapeutic intervening possibilities are shown in boxes. NT - neurotransmitter receptor.</div></p></div><div class="aux"><div class="resc">Chang-Hoon Cho. Exp Mol Med. 2011 May 31;43(5):231-274.</div><p class="links"><a class="dblinks" href="/pubmed/21467839" ref="ordinalpos=806&amp;ncbi_uid=3004998&amp;link_uid=21467839">Citation</a><a class="dblinks" href="/pmc/articles/PMC3104248/" ref="ordinalpos=806&amp;ncbi_uid=3004998&amp;link_uid=PMC3104248">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>807.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2862420/figure/fig2/" ref="ordinalpos=807&amp;ncbi_uid=2916302&amp;link_uid=PMC2862420" href="/pmc/articles/PMC2862420/"><img src="/pmc/articles/instance/2862420/bin/pp153601F02_RGB.gif" src-large="/pmc/articles/instance/2862420/bin/pp153601F02_RGB.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2862420/" ref="ordinalpos=807&amp;ncbi_uid=2916302&amp;link_uid=PMC2862420" image-link="/pmc/articles/PMC2862420/figure/fig2/" class="imagepopup">Figure 2. From: Detoxification without Intoxication: Herbicide Safeners Activate Plant Defense Gene Expression1[C]. </a></p><div class="supp"><p class="details"><div>Possible safener-mediated <span class="highlight" style="background-color:">signaling</span> pathways for the regulation of defense genes and activation of detoxification pathways in plants. Safeners may tap into a RES oxylipin-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (e.g. OPDA and PPA<sub>1</sub>) and up-regulate TGA transcription factors (for review, see Mueller and Berger, 2009), an Nrf2-Keap1-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Nguyen et al., 2009), and/or a JA-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Schaller and Stintzi, 2009; Zander et al., 2010) to induce the expression of proteins involved in herbicide detoxification in plants. It is also possible that safeners may utilize an unknown (?) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to trigger plant defense gene expression. Dashed lines indicate possible but unproven <span class="highlight" style="background-color:">signaling</span> pathways, and solid lines indicate known <span class="highlight" style="background-color:">signaling</span> pathways for the regulation of plant gene expression. Note that the three distinct “Gene Expression” boxes indicate that only a subset of GST enzymes can metabolize herbicides or safeners as substrates, leading to detoxification and herbicide tolerance in cereal crops. CoI1, Coronatine-insensitive protein1; JAZ, transcriptional repressor protein; TF, transcription factor/activator. [See online article for color version of this figure.]</div></p></div><div class="aux"><div class="resc">Dean E. Riechers, et al. Plant Physiol. 2010 May;153(1):3-13.</div><p class="links"><a class="dblinks" href="/pubmed/20237021" ref="ordinalpos=807&amp;ncbi_uid=2916302&amp;link_uid=20237021">Citation</a><a class="dblinks" href="/pmc/articles/PMC2862420/" ref="ordinalpos=807&amp;ncbi_uid=2916302&amp;link_uid=PMC2862420">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>808.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2679403/figure/F4/" ref="ordinalpos=808&amp;ncbi_uid=1841565&amp;link_uid=PMC2679403" href="/pmc/articles/PMC2679403/"><img src="/pmc/articles/instance/2679403/bin/1471-2105-10-S5-S3-4.gif" src-large="/pmc/articles/instance/2679403/bin/1471-2105-10-S5-S3-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2679403/" ref="ordinalpos=808&amp;ncbi_uid=1841565&amp;link_uid=PMC2679403" image-link="/pmc/articles/PMC2679403/figure/F4/" class="imagepopup">Figure 4.  From: TGF-beta <span class="highlight" style="background-color:">signaling</span> proteins and the Protein Ontology. </a></p><div class="supp"><p class="details"><div><b>PRO mapping to the Reactome TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The <span class="highlight" style="background-color:">pathway</span> described herein corresponds to REACT_6844. Each step in the <span class="highlight" style="background-color:">pathway</span> is described by a Reactome event ID. Bold PRO IDs indicate objects that undergo some modification that is relevant for function (the modified form is underlined). The names in the boxes represent the names of the complexes or molecules involved in the processes, not of the PRO terms.</div></p></div><div class="aux"><div class="resc">Cecilia N Arighi, et al. BMC Bioinformatics. 2009;10(Suppl 5):S3-S3.</div><p class="links"><a class="dblinks" href="/pubmed/19426460" ref="ordinalpos=808&amp;ncbi_uid=1841565&amp;link_uid=19426460">Citation</a><a class="dblinks" href="/pmc/articles/PMC2679403/" ref="ordinalpos=808&amp;ncbi_uid=1841565&amp;link_uid=PMC2679403">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>809.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4065045/figure/pone-0100613-g006/" ref="ordinalpos=809&amp;ncbi_uid=6289658&amp;link_uid=PMC4065045" href="/pmc/articles/PMC4065045/"><img src="/pmc/articles/instance/4065045/bin/pone.0100613.g006.gif" src-large="/pmc/articles/instance/4065045/bin/pone.0100613.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4065045/" ref="ordinalpos=809&amp;ncbi_uid=6289658&amp;link_uid=PMC4065045" image-link="/pmc/articles/PMC4065045/figure/pone-0100613-g006/" class="imagepopup">Figure 6. The <span class="highlight" style="background-color:">pathway</span> modulation model..  From: A Systems Biology Approach to the Analysis of Subset-Specific Responses to Lipopolysaccharide in Dendritic Cells. </a></p><div class="supp"><p class="details"><div>Schematic representation of our proposed model. LPS-induced <span class="highlight" style="background-color:">signaling</span> occurs through a common set of <span class="highlight" style="background-color:">signaling</span> molecules. LPS induces the expression of subset-specific TLR4 <span class="highlight" style="background-color:">pathway</span> modulators, which ‘fine tune’ <span class="highlight" style="background-color:">signaling</span> and allow for distinct immune outcomes in closely related cell subsets. These <span class="highlight" style="background-color:">pathway</span> modulators likely integrate both signals derived directly from TLR4 and exogenous signals from the microenviroment that contribute to the subset-specificity of the response.</div></p></div><div class="aux"><div class="resc">David G. Hancock, et al. PLoS One. 2014;9(6):e100613.</div><p class="links"><a class="dblinks" href="/pubmed/24949855" ref="ordinalpos=809&amp;ncbi_uid=6289658&amp;link_uid=24949855">Citation</a><a class="dblinks" href="/pmc/articles/PMC4065045/" ref="ordinalpos=809&amp;ncbi_uid=6289658&amp;link_uid=PMC4065045">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>810.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4001041/figure/F1/" ref="ordinalpos=810&amp;ncbi_uid=6098651&amp;link_uid=PMC4001041" href="/pmc/articles/PMC4001041/"><img src="/pmc/articles/instance/4001041/bin/fimmu-05-00169-g001.gif" src-large="/pmc/articles/instance/4001041/bin/fimmu-05-00169-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4001041/" ref="ordinalpos=810&amp;ncbi_uid=6098651&amp;link_uid=PMC4001041" image-link="/pmc/articles/PMC4001041/figure/F1/" class="imagepopup">Figure 1.  From: Non-Inflammasome Forming NLRs in Inflammation and Tumorigenesis. </a></p><div class="supp"><p class="details"><div><b>Schematic illustrating NLR attenuation of canonical and non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span></b>. NF-κB is a master regulator of gene transcription and contributes to several hallmarks of cancer. NLRX1, NLRP12, and NLRC3 negatively regulate NF-κB <span class="highlight" style="background-color:">signaling</span> at multiple levels. NLRX1 interacts with and inhibits TRAF6 and the IKK complex resulting in the attenuation of NF-κB <span class="highlight" style="background-color:">signaling</span> following TLR stimulation. Likewise, NLRC3 was also shown to interact with TRAF6 and attenuate NF-κB <span class="highlight" style="background-color:">signaling</span> through a similar mechanism. NLRP12, has been shown to attenuate both the canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through modulating the phosphorylation of IRAK-1 and the non-canonical NF-κB <span class="highlight" style="background-color:">pathway</span> through interactions with TRAF3 and NIK.</div></p></div><div class="aux"><div class="resc">Irving Coy Allen. Front Immunol. 2014;5:169.</div><p class="links"><a class="dblinks" href="/pubmed/24795716" ref="ordinalpos=810&amp;ncbi_uid=6098651&amp;link_uid=24795716">Citation</a><a class="dblinks" href="/pmc/articles/PMC4001041/" ref="ordinalpos=810&amp;ncbi_uid=6098651&amp;link_uid=PMC4001041">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>811.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3772092/figure/pone-0075489-g007/" ref="ordinalpos=811&amp;ncbi_uid=5336264&amp;link_uid=PMC3772092" href="/pmc/articles/PMC3772092/"><img src="/pmc/articles/instance/3772092/bin/pone.0075489.g007.gif" src-large="/pmc/articles/instance/3772092/bin/pone.0075489.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3772092/" ref="ordinalpos=811&amp;ncbi_uid=5336264&amp;link_uid=PMC3772092" image-link="/pmc/articles/PMC3772092/figure/pone-0075489-g007/" class="imagepopup">Figure 7. TGF-β <span class="highlight" style="background-color:">signaling</span> in EMT program in thyroid cells..  From: Cadherin 6 Is a New RUNX2 Target in TGF-? Signalling <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of the TGF-β dependent EMT program in thyroid cells. Malignant transformation of thyroid cells is accompanied by a constitutive activation of the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TGF-β cascade controls the expression of a number of transcription factors (EMT-TF) including RUNX2 that alter the gene expression profile of the cells to activate the EMT program.</div></p></div><div class="aux"><div class="resc">Valentina Sancisi, et al. PLoS One. 2013;8(9):e75489.</div><p class="links"><a class="dblinks" href="/pubmed/24069422" ref="ordinalpos=811&amp;ncbi_uid=5336264&amp;link_uid=24069422">Citation</a><a class="dblinks" href="/pmc/articles/PMC3772092/" ref="ordinalpos=811&amp;ncbi_uid=5336264&amp;link_uid=PMC3772092">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>812.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3048679/figure/F9/" ref="ordinalpos=812&amp;ncbi_uid=3642909&amp;link_uid=PMC3048679" href="/pmc/articles/PMC3048679/"><img src="/pmc/articles/instance/3048679/bin/zbc0151154480009.gif" src-large="/pmc/articles/instance/3048679/bin/zbc0151154480009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3048679/" ref="ordinalpos=812&amp;ncbi_uid=3642909&amp;link_uid=PMC3048679" image-link="/pmc/articles/PMC3048679/figure/F9/" class="imagepopup">FIGURE 9. From: A-Kinase Anchoring Protein (AKAP)-Lbc Anchors a PKN-based <span class="highlight" style="background-color:">Signaling</span> Complex Involved in ?1-Adrenergic Receptor-induced p38 Activation. </a></p><div class="supp"><p class="details"><div><b>Model for the AKAP-Lbc·p38α activation complex.</b> AKAP-Lbc assembles a <span class="highlight" style="background-color:">signaling</span> complex that includes the scaffolding protein PKNα as well as MLTK, MKK3, and p38α. The AKAP-Lbc <span class="highlight" style="background-color:">signaling</span> complex is activated in response to α<sub>1</sub>-AR stimulation through a Gα<sub>12</sub>-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (17). Activated AKAP-Lbc promotes the formation of RhoA-GTP, which, in turn, induces the activation of a <span class="highlight" style="background-color:">signaling</span> cascade that includes PKNα, MLTK, MKK3, and p38α. The recruitment of 14-3-3 inhibits AKAP-Lbc Rho-GEF activity (19), impairs the interaction between PKNα and AKAP-Lbc, and reduces p38 activation.</div></p></div><div class="aux"><div class="resc">Luca Cariolato, et al. J Biol Chem. 2011 March 11;286(10):7925-7937.</div><p class="links"><a class="dblinks" href="/pubmed/21224381" ref="ordinalpos=812&amp;ncbi_uid=3642909&amp;link_uid=21224381">Citation</a><a class="dblinks" href="/pmc/articles/PMC3048679/" ref="ordinalpos=812&amp;ncbi_uid=3642909&amp;link_uid=PMC3048679">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>813.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3282588/figure/F6/" ref="ordinalpos=813&amp;ncbi_uid=3903277&amp;link_uid=PMC3282588" href="/pmc/articles/PMC3282588/"><img src="/pmc/articles/instance/3282588/bin/nihms269293f6.gif" src-large="/pmc/articles/instance/3282588/bin/nihms269293f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3282588/" ref="ordinalpos=813&amp;ncbi_uid=3903277&amp;link_uid=PMC3282588" image-link="/pmc/articles/PMC3282588/figure/F6/" class="imagepopup">Figure 6. Shh and Notch <span class="highlight" style="background-color:">signaling</span> act in parallel to specify floor plate.  From: A revised model of Xenopus dorsal midline development: Differential and separable requirements for Notch and Shh <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Epistasis (A-L) and enhancement (M-P) experiments to determine the contribution of Notch and Shh <span class="highlight" style="background-color:">signaling</span> to floor plate formation. Embryos were injected and/or treated with reagents listed, fixed at stage 20-22, bisected and stained for co-injected lineage tracer (pink), then processed for <i>Netrin</i> ISH. Activated Notch <span class="highlight" style="background-color:">signaling</span> upregulates <i>Netrin</i> (B, F), while blockade of Shh <span class="highlight" style="background-color:">signaling</span> slightly reduces <i>Netrin</i> (C, G). Activation of Notch and blockade of Shh in the same embryo results in a Notch phenotype (D, H). Shh overexpression leads to a slight increase in FP (I), and blockade of Notch <span class="highlight" style="background-color:">signaling</span> leads to a decrease (J). Activation of Shh and blockade of Notch in the same embryo results in a Notch phenotype (K). Blockade of Notch <span class="highlight" style="background-color:">signaling</span> downstream of NICD with SDBM is able to rescue FP and notochord development to normal (L). Blockade (M, O) or activation (P) of both Notch and Shh <span class="highlight" style="background-color:">signaling</span> in the same embryo gives more severe phenotypes than perturbing either <span class="highlight" style="background-color:">pathway</span> alone.</div></p></div><div class="aux"><div class="resc">Sara M. Peyrot, et al. Dev Biol. ;352(2):254-266.</div><p class="links"><a class="dblinks" href="/pubmed/21276789" ref="ordinalpos=813&amp;ncbi_uid=3903277&amp;link_uid=21276789">Citation</a><a class="dblinks" href="/pmc/articles/PMC3282588/" ref="ordinalpos=813&amp;ncbi_uid=3903277&amp;link_uid=PMC3282588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>814.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4047762/figure/F6/" ref="ordinalpos=814&amp;ncbi_uid=6206667&amp;link_uid=PMC4047762" href="/pmc/articles/PMC4047762/"><img src="/pmc/articles/instance/4047762/bin/nihms-580605-f0006.gif" src-large="/pmc/articles/instance/4047762/bin/nihms-580605-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4047762/" ref="ordinalpos=814&amp;ncbi_uid=6206667&amp;link_uid=PMC4047762" image-link="/pmc/articles/PMC4047762/figure/F6/" class="imagepopup">Figure 6. Working Model for ALK1 <span class="highlight" style="background-color:">Signaling</span> in ECs.  From: ALK1 <span class="highlight" style="background-color:">Signaling</span> Inhibits Angiogenesis by Cooperating with the Notch <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The data suggest that ALK1 <span class="highlight" style="background-color:">signaling</span>, triggered upon BMP9 binding, activates SMAD1,5,8 <span class="highlight" style="background-color:">signaling</span>, which induces expression of <i>HES1</i>, <i>HEY1</i>, and <i>HEY2</i> together with Notch <span class="highlight" style="background-color:">signaling</span>. ALK1 also induces expression of UNC5B and VEGFR1, therefore reducing VEGF response and tip cell specification. This modulation of gene expression by Alk1 contributes to the stability and quiescence of the endothelium.</div></p></div><div class="aux"><div class="resc">Bruno Larrivée, et al. Dev Cell. ;22(3):489-500.</div><p class="links"><a class="dblinks" href="/pubmed/22421041" ref="ordinalpos=814&amp;ncbi_uid=6206667&amp;link_uid=22421041">Citation</a><a class="dblinks" href="/pmc/articles/PMC4047762/" ref="ordinalpos=814&amp;ncbi_uid=6206667&amp;link_uid=PMC4047762">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>815.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4011321/figure/fig8/" ref="ordinalpos=815&amp;ncbi_uid=6625146&amp;link_uid=PMC4011321" href="/pmc/articles/PMC4011321/"><img src="/pmc/articles/instance/4011321/bin/aps201252f8.gif" src-large="/pmc/articles/instance/4011321/bin/aps201252f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4011321/" ref="ordinalpos=815&amp;ncbi_uid=6625146&amp;link_uid=PMC4011321" image-link="/pmc/articles/PMC4011321/figure/fig8/" class="imagepopup">Figure 8.  From: 8-(Tosylamino)quinoline inhibits macrophage-mediated inflammation by suppressing NF-?B <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Putative <span class="highlight" style="background-color:">pathway</span> of the inhibition of LPS-activated inflammatory <span class="highlight" style="background-color:">signaling</span> by compound 7.</div></p></div><div class="aux"><div class="resc">Yongwoo Jung, et al. Acta Pharmacol Sin. 2012 August;33(8):1037-1046.</div><p class="links"><a class="dblinks" href="/pubmed/22796759" ref="ordinalpos=815&amp;ncbi_uid=6625146&amp;link_uid=22796759">Citation</a><a class="dblinks" href="/pmc/articles/PMC4011321/" ref="ordinalpos=815&amp;ncbi_uid=6625146&amp;link_uid=PMC4011321">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>816.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3248195/figure/F1/" ref="ordinalpos=816&amp;ncbi_uid=3760593&amp;link_uid=PMC3248195" href="/pmc/articles/PMC3248195/"><img src="/pmc/articles/instance/3248195/bin/oncotarget-02-435-g001.gif" src-large="/pmc/articles/instance/3248195/bin/oncotarget-02-435-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3248195/" ref="ordinalpos=816&amp;ncbi_uid=3760593&amp;link_uid=PMC3248195" image-link="/pmc/articles/PMC3248195/figure/F1/" class="imagepopup">Figure 1. Schematic diagram of the PI3K <span class="highlight" style="background-color:">pathway</span> and its activation in human breast cancer.  From: Elevated PI3K <span class="highlight" style="background-color:">signaling</span> drives multiple Breast Cancer subtypes. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">pathway</span> can be activated at multiple levels including at the level of InsR/Igf-1R, c-MET or HEH2/HER3 receptors, at the level of adaptor proteins, PI3K regulatory or catalytic subunits, at the level of downstream Akt kinases or through deletion/inactivation of inhibitors of the <span class="highlight" style="background-color:">pathway</span>. Note, for simplicity, alternative receptor-activated <span class="highlight" style="background-color:">signaling</span> pathways as well as the full spectrum of PIP3 targets and Akt and TORC targets are not shown.</div></p></div><div class="aux"><div class="resc">Jessica R. Adams, et al. Oncotarget. 2011 June;2(6):435-447.</div><p class="links"><a class="dblinks" href="/pubmed/21646685" ref="ordinalpos=816&amp;ncbi_uid=3760593&amp;link_uid=21646685">Citation</a><a class="dblinks" href="/pmc/articles/PMC3248195/" ref="ordinalpos=816&amp;ncbi_uid=3760593&amp;link_uid=PMC3248195">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>817.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508164/figure/F10/" ref="ordinalpos=817&amp;ncbi_uid=5657096&amp;link_uid=PMC3508164" href="/pmc/articles/PMC3508164/"><img src="/pmc/articles/instance/3508164/bin/nihms411023f10.gif" src-large="/pmc/articles/instance/3508164/bin/nihms411023f10.jpg" alt="Fig. 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508164/" ref="ordinalpos=817&amp;ncbi_uid=5657096&amp;link_uid=PMC3508164" image-link="/pmc/articles/PMC3508164/figure/F10/" class="imagepopup">Fig. 10.  From: Cripto-1 enhances the canonical Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by binding to LRP5 and LRP6 co-receptors. </a></p><div class="supp"><p class="details"><div>F9 cells are more resistant to Dkk1 inhibition of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> than F9 Cripto-1<sup>-/</sup>- cells. (A) SuperTOPFLASH assay in F9 and F9 Cripto-1<sup>-/-</sup> cells treated with rhWnt3a (100 ng/ml) alone or in combination with different concentration of rmDkk1. (B) Dkk1 mRNA expression in F9 and F9 Cripto-1<sup>-/-</sup> cells as assessed by RT-PCR.</div></p></div><div class="aux"><div class="resc">Tadahiro Nagaoka, et al. Cell Signal. ;25(1):178-189.</div><p class="links"><a class="dblinks" href="/pubmed/23022962" ref="ordinalpos=817&amp;ncbi_uid=5657096&amp;link_uid=23022962">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508164/" ref="ordinalpos=817&amp;ncbi_uid=5657096&amp;link_uid=PMC3508164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>818.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2276699/figure/F7/" ref="ordinalpos=818&amp;ncbi_uid=1378894&amp;link_uid=PMC2276699" href="/pmc/articles/PMC2276699/"><img src="/pmc/articles/instance/2276699/bin/nihms39855f7.gif" src-large="/pmc/articles/instance/2276699/bin/nihms39855f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2276699/" ref="ordinalpos=818&amp;ncbi_uid=1378894&amp;link_uid=PMC2276699" image-link="/pmc/articles/PMC2276699/figure/F7/" class="imagepopup">Figure 7. Role of SCD1 and Its Related Lipid Species in DSS-Induced Proinflammatory <span class="highlight" style="background-color:">Signaling</span>.  From: Metabolomics Reveals that Hepatic Stearoyl-CoA Desaturase 1 Downregulation Exacerbates Inflammation and Acute Colitis. </a></p><div class="supp"><p class="details"><div>Red represents proinflammatory <span class="highlight" style="background-color:">signaling</span>; green represents anti-inflammatory <span class="highlight" style="background-color:">signaling</span>. Solid lines indicate known pathways; the dashed line denotes a putative <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chi Chen, et al. Cell Metab. ;7(2):135-147.</div><p class="links"><a class="dblinks" href="/pubmed/18249173" ref="ordinalpos=818&amp;ncbi_uid=1378894&amp;link_uid=18249173">Citation</a><a class="dblinks" href="/pmc/articles/PMC2276699/" ref="ordinalpos=818&amp;ncbi_uid=1378894&amp;link_uid=PMC2276699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>819.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872116/figure/F4/" ref="ordinalpos=819&amp;ncbi_uid=2370040&amp;link_uid=PMC2872116" href="/pmc/articles/PMC2872116/"><img src="/pmc/articles/instance/2872116/bin/nihms193429f4.gif" src-large="/pmc/articles/instance/2872116/bin/nihms193429f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872116/" ref="ordinalpos=819&amp;ncbi_uid=2370040&amp;link_uid=PMC2872116" image-link="/pmc/articles/PMC2872116/figure/F4/" class="imagepopup">Fig. 4.  From: Visual integration of quantitative proteomic data, pathways and protein interactions. </a></p><div class="supp"><p class="details"><div>Proteins and interactions from HPRD (small fonts) that are connected to the canonical <span class="highlight" style="background-color:">pathway</span> model are: (left) integrated directly into the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> image with one protein selected and its interactions highlighted; (right) structured around a user-constructed model; different classes of proteins have different appearances: experimental proteins are colored yellow, kinases are drawn as hexagons and receptors as irregular stars; several experimental proteins are not known to be connected to the <span class="highlight" style="background-color:">pathway</span> and are therefore located in the lower right corner. HPRD proteins are placed in a structured manner between the <span class="highlight" style="background-color:">pathway</span> proteins based on their separation from the <span class="highlight" style="background-color:">pathway</span> proteins. (cutout) Disadvantage of simply drawing the network on top of the <span class="highlight" style="background-color:">pathway</span> image: HPRD interactions obscure elements of the canonical <span class="highlight" style="background-color:">pathway</span>; compare to the improved method (left) in which important <span class="highlight" style="background-color:">pathway</span> elements remain in the foreground.</div></p></div><div class="aux"><div class="resc">Radu Jianu, et al. IEEE Trans Vis Comput Graph. ;16(4):609-620.</div><p class="links"><a class="dblinks" href="/pubmed/20467059" ref="ordinalpos=819&amp;ncbi_uid=2370040&amp;link_uid=20467059">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872116/" ref="ordinalpos=819&amp;ncbi_uid=2370040&amp;link_uid=PMC2872116">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>820.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3480380/figure/pone-0047067-g008/" ref="ordinalpos=820&amp;ncbi_uid=4326820&amp;link_uid=PMC3480380" href="/pmc/articles/PMC3480380/"><img src="/pmc/articles/instance/3480380/bin/pone.0047067.g008.gif" src-large="/pmc/articles/instance/3480380/bin/pone.0047067.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3480380/" ref="ordinalpos=820&amp;ncbi_uid=4326820&amp;link_uid=PMC3480380" image-link="/pmc/articles/PMC3480380/figure/pone-0047067-g008/" class="imagepopup">Figure 8. Coordinate regulation of EGFR/Akt <span class="highlight" style="background-color:">signaling</span> by miR-7 in HNC cells..  From: Regulation of Epidermal Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span> and Erlotinib Sensitivity in Head and Neck Cancer Cells by miR-7. </a></p><div class="supp"><p class="details"><div>Schematic representation of molecules in the EGFR/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that are inhibited by miR-7 in HNC. IPA software was used to map common miR-7-downregulated genes (shown in green) in FaDu and HN5 cells onto the canonical PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. The density of shading represents the fold-change downregulation of a gene by miR-7. Blue circles indicate that a gene is a predicted or validated target of miR-7 by IPA analysis. Several genes belonging to the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> that were downregulated by miR-7 in HN5 cells only are shaded in light blue.</div></p></div><div class="aux"><div class="resc">Felicity C. Kalinowski, et al. PLoS One. 2012;7(10):e47067.</div><p class="links"><a class="dblinks" href="/pubmed/23115635" ref="ordinalpos=820&amp;ncbi_uid=4326820&amp;link_uid=23115635">Citation</a><a class="dblinks" href="/pmc/articles/PMC3480380/" ref="ordinalpos=820&amp;ncbi_uid=4326820&amp;link_uid=PMC3480380">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>821.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3849657/figure/F1/" ref="ordinalpos=821&amp;ncbi_uid=5569011&amp;link_uid=PMC3849657" href="/pmc/articles/PMC3849657/"><img src="/pmc/articles/instance/3849657/bin/2045-3701-3-33-1.gif" src-large="/pmc/articles/instance/3849657/bin/2045-3701-3-33-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3849657/" ref="ordinalpos=821&amp;ncbi_uid=5569011&amp;link_uid=PMC3849657" image-link="/pmc/articles/PMC3849657/figure/F1/" class="imagepopup">Figure 1.  From: Integration of mechanical and chemical signals by YAP and TAZ transcription coactivators. </a></p><div class="supp"><p class="details"><div><b>YAP/TAZ are effectors of mechanical stress, GPCR <span class="highlight" style="background-color:">signaling</span>, and the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Mechanisms of YAP and TAZ regulation by mechanical stress, GPCR <span class="highlight" style="background-color:">signaling</span>, and the Wnt <span class="highlight" style="background-color:">pathway</span> as well as YAP/TAZ as modulators of the Wnt <span class="highlight" style="background-color:">pathway</span> are shown. Arrowed or blunted ends indicate activation or inhibition, respectively. Dashed lines indicate unknown mechanisms.</div></p></div><div class="aux"><div class="resc">Xiaocan Guo, et al. Cell Biosci. 2013;3:33-33.</div><p class="links"><a class="dblinks" href="/pubmed/23985334" ref="ordinalpos=821&amp;ncbi_uid=5569011&amp;link_uid=23985334">Citation</a><a class="dblinks" href="/pmc/articles/PMC3849657/" ref="ordinalpos=821&amp;ncbi_uid=5569011&amp;link_uid=PMC3849657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>822.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3157546/figure/F7/" ref="ordinalpos=822&amp;ncbi_uid=3509078&amp;link_uid=PMC3157546" href="/pmc/articles/PMC3157546/"><img src="/pmc/articles/instance/3157546/bin/nihms305602f7.gif" src-large="/pmc/articles/instance/3157546/bin/nihms305602f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3157546/" ref="ordinalpos=822&amp;ncbi_uid=3509078&amp;link_uid=PMC3157546" image-link="/pmc/articles/PMC3157546/figure/F7/" class="imagepopup">Figure 7.  From: TAK1-TAB2 <span class="highlight" style="background-color:">signaling</span> contributes to bone destruction by breast carcinoma cells. </a></p><div class="supp"><p class="details"><div>Model for the role of TAK1 <span class="highlight" style="background-color:">signaling</span> in bone metastasis. (<b>A</b>) TGF-β and pro-inflammatory cytokines can activate TAK1 in tumor cells. TAK1 is likely to interact with the TAB2/TAB3-TRAF complex leading to activation of NFkB and up-regulation of MMP-9, COX2, IL8 and PTHrP. Expression of dnTAK1 (TAK1-K63W) hinders this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in tumor cells. (<b>B</b>) In bone environment, the TAK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in tumor cells promotes the “vicious cycle” leading to the disproportionate activity of osteoclasts and bone destruction.</div></p></div><div class="aux"><div class="resc">Alfiya Safina, et al. Mol Cancer Res. ;9(8):1042-1053.</div><p class="links"><a class="dblinks" href="/pubmed/21700681" ref="ordinalpos=822&amp;ncbi_uid=3509078&amp;link_uid=21700681">Citation</a><a class="dblinks" href="/pmc/articles/PMC3157546/" ref="ordinalpos=822&amp;ncbi_uid=3509078&amp;link_uid=PMC3157546">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>823.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3591351/figure/pone-0058355-g006/" ref="ordinalpos=823&amp;ncbi_uid=4713950&amp;link_uid=PMC3591351" href="/pmc/articles/PMC3591351/"><img src="/pmc/articles/instance/3591351/bin/pone.0058355.g006.gif" src-large="/pmc/articles/instance/3591351/bin/pone.0058355.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3591351/" ref="ordinalpos=823&amp;ncbi_uid=4713950&amp;link_uid=PMC3591351" image-link="/pmc/articles/PMC3591351/figure/pone-0058355-g006/" class="imagepopup">Figure 6. ON01210.Na treatment in combination with radiation alters the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: ON01210.Na (Ex-RAD®) Mitigates Radiation Damage through Activation of the AKT <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Human Phospho-MAPK antibody arrays were treated with lysates from HFL-1 cells that were pre-treated with Vehicle (A), 20<b></b>uM ON01210.Na (B), Vehicle plus 10 Gy radiation (C), 20<b></b>uM ON01210.Na plus 10 Gy radiation (D). ON01210.Na treatment in combination with radiation increased the phosphorylation status of proteins associated with the AKT <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Anthony D. Kang, et al. PLoS One. 2013;8(3):e58355.</div><p class="links"><a class="dblinks" href="/pubmed/23505494" ref="ordinalpos=823&amp;ncbi_uid=4713950&amp;link_uid=23505494">Citation</a><a class="dblinks" href="/pmc/articles/PMC3591351/" ref="ordinalpos=823&amp;ncbi_uid=4713950&amp;link_uid=PMC3591351">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>824.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4017825/figure/fig2/" ref="ordinalpos=824&amp;ncbi_uid=6167500&amp;link_uid=PMC4017825" href="/pmc/articles/PMC4017825/"><img src="/pmc/articles/instance/4017825/bin/BMRI2014-319651.002.gif" src-large="/pmc/articles/instance/4017825/bin/BMRI2014-319651.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4017825/" ref="ordinalpos=824&amp;ncbi_uid=6167500&amp;link_uid=PMC4017825" image-link="/pmc/articles/PMC4017825/figure/fig2/" class="imagepopup">Figure 2.  From: Effects of Canonical NF-κB <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> on the Proliferation and Odonto/Osteogenic Differentiation of Human Stem Cells from Apical Papilla. </a></p><div class="supp"><p class="details"><div>Activation and inhibition of canonical NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in SCAPs. (a) The expression of NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span> proteins in TNF-<i>α</i>-treated SCAPs at different time points. (b) Semiquantitative analysis confirmed the upregulation of P-I<i>κ</i>B<i>α</i> and P-P65 after the activation of NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span>. (c) The ratio of phosphorylated to unphosphorylated form of P65. (d) The ratio of phosphorylated I<i>κ</i>B<i>α</i> to unphosphorylated I<i>κ</i>B<i>α</i>. (e) The protein levels of I<i>κ</i>B<i>α</i>, P-I<i>κ</i>B<i>α</i>, P65, and P-P65 in the cytoplasm of BMS-345541-treated SCAPs at indicated time points. (f) Semiquantitative analysis confirmed the declined expression of P-I<i>κ</i>B<i>α</i> and P-P65 after the inhibition of NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span>. (g) The ratio of phosphorylated to unphosphorylated form of P65. (h) The ratio of P-I<i>κ</i>B<i>α</i> to I<i>κ</i>B<i>α</i>. Values are the means ± SD; <i>n</i> = 3; **<i>P</i> &lt; 0.01; ***<i>P</i> &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Junjun Li, et al. Biomed Res Int. 2014;2014:319651.</div><p class="links"><a class="dblinks" href="/pubmed/24864235" ref="ordinalpos=824&amp;ncbi_uid=6167500&amp;link_uid=24864235">Citation</a><a class="dblinks" href="/pmc/articles/PMC4017825/" ref="ordinalpos=824&amp;ncbi_uid=6167500&amp;link_uid=PMC4017825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>825.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2780666/figure/Fig3/" ref="ordinalpos=825&amp;ncbi_uid=2151161&amp;link_uid=PMC2780666" href="/pmc/articles/PMC2780666/"><img src="/pmc/articles/instance/2780666/bin/10637_2005_Article_5199_Fig3.gif" src-large="/pmc/articles/instance/2780666/bin/10637_2005_Article_5199_Fig3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2780666/" ref="ordinalpos=825&amp;ncbi_uid=2151161&amp;link_uid=PMC2780666" image-link="/pmc/articles/PMC2780666/figure/Fig3/" class="imagepopup">Figure 3.  From: The Wnt-dependent <span class="highlight" style="background-color:">signaling</span> pathways as target in oncology drug discovery. </a></p><div class="supp"><p class="details"><div>Lead structures for Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors.</div></p></div><div class="aux"><div class="resc">Nico Janssens, et al. Invest New Drugs. 2006 July;24(4):263-280.</div><p class="links"><a class="dblinks" href="/pubmed/16683072" ref="ordinalpos=825&amp;ncbi_uid=2151161&amp;link_uid=16683072">Citation</a><a class="dblinks" href="/pmc/articles/PMC2780666/" ref="ordinalpos=825&amp;ncbi_uid=2151161&amp;link_uid=PMC2780666">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>826.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3064008/figure/F4/" ref="ordinalpos=826&amp;ncbi_uid=3497722&amp;link_uid=PMC3064008" href="/pmc/articles/PMC3064008/"><img src="/pmc/articles/instance/3064008/bin/zh10021161740004.gif" src-large="/pmc/articles/instance/3064008/bin/zh10021161740004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3064008/" ref="ordinalpos=826&amp;ncbi_uid=3497722&amp;link_uid=PMC3064008" image-link="/pmc/articles/PMC3064008/figure/F4/" class="imagepopup">Fig. 4. From: ?-Arrestin mediates oxytocin receptor <span class="highlight" style="background-color:">signaling</span>, which regulates uterine contractility and cellular migration. </a></p><div class="supp"><p class="details"><div>The OXTR signals through the MAPK <span class="highlight" style="background-color:">pathway</span> through the G<sub>q</sub>- and G<sub>i</sub>-mediated pathways. To determine the relative component of MAPK <span class="highlight" style="background-color:">signaling</span> from G protein-mediated <span class="highlight" style="background-color:">signaling</span>, inhibitors of G<sub>q</sub>- and G<sub>i</sub>-mediated <span class="highlight" style="background-color:">signaling</span> were utilized. <i>A</i>: HEK-293 cells expressing the OXTR were pretreated with the PKC inhibitor Ro-31, which inhibits G<sub>q</sub>-mediated <span class="highlight" style="background-color:">signaling</span>. Cells were then treated with 100 nM oxytocin at 2, 5, 10, 30, or 60 min and the cells lysed and processed on a Western blot and probed for p-ERK. Maximal p-ERK <span class="highlight" style="background-color:">signaling</span> in control cells occurred at 5 min and was diminished by ∼40–50% with G<sub>q</sub> inhibition. Mean densitometry data from 6 independent experiments is illustrated. There is a significant decrease in maximal p-ERK <span class="highlight" style="background-color:">signaling</span> at 5 min (*<i>P</i> &lt; 0.5) compared with control (CTL), and repeated-measures 2-way ANOVA demonstrates the 2 curves to be different (†<i>P</i> &lt; 0.05). <i>B</i>: to determine the component of G<sub>i</sub>-mediated <span class="highlight" style="background-color:">signaling</span> from the OXTR to the MAPK <span class="highlight" style="background-color:">pathway</span>, HEK-293 cells expressing the OXTR were pretreated with pertussis toxin (PTX) and then stimulated with an oxytocin time course. Mean densitometry data from 6 independent experiments illustrates an ∼30% decrease in maximal p-ERK <span class="highlight" style="background-color:">signaling</span> at 5 min (*<i>P</i> &lt; 0.05) compared with CTL, and repeated-measures 2-way ANOVA demonstrates that the 2 curves are different (†<i>P</i> &lt; 0.05). IB, immunoblotting.</div></p></div><div class="aux"><div class="resc">Chad A. Grotegut, et al. Am J Physiol Endocrinol Metab. 2011 March;300(3):E468-E477.</div><p class="links"><a class="dblinks" href="/pubmed/21139074" ref="ordinalpos=826&amp;ncbi_uid=3497722&amp;link_uid=21139074">Citation</a><a class="dblinks" href="/pmc/articles/PMC3064008/" ref="ordinalpos=826&amp;ncbi_uid=3497722&amp;link_uid=PMC3064008">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>827.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4132015/figure/pone-0104885-g005/" ref="ordinalpos=827&amp;ncbi_uid=6515907&amp;link_uid=PMC4132015" href="/pmc/articles/PMC4132015/"><img src="/pmc/articles/instance/4132015/bin/pone.0104885.g005.gif" src-large="/pmc/articles/instance/4132015/bin/pone.0104885.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4132015/" ref="ordinalpos=827&amp;ncbi_uid=6515907&amp;link_uid=PMC4132015" image-link="/pmc/articles/PMC4132015/figure/pone-0104885-g005/" class="imagepopup">Figure 5. Analysis of conserved developmental <span class="highlight" style="background-color:">signaling</span> pathways..  From: A Comprehensive Reference Transcriptome Resource for the Common House Spider Parasteatoda tepidariorum. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> components were extracted from the KEGG database (http://www.genome.jp/kegg/kegg1.html). Sequences mainly from <i>D. melanogaster</i> were compared to the final <i>P. tepidariorum</i> assembly by BLAST. In cases where no <i>D. melanogaster</i> ortholog was present (grey boxes with grey or white typeface) the respective mouse or human sequence was used as query (see Table S3 for details). Genes identified in the spider transcriptome are marked in blue. <span class="highlight" style="background-color:">Pathway</span> schematics modified from KEGG <span class="highlight" style="background-color:">pathway</span> model images http://www.genome.jp/kegg/kegg1.html. A Notch <span class="highlight" style="background-color:">pathway</span>; B TGFβ <span class="highlight" style="background-color:">pathway</span>; C Wnt <span class="highlight" style="background-color:">pathway</span>; D Hedgehog <span class="highlight" style="background-color:">pathway</span>; E Janus Kinase (JAK)-signal transducer and activator of transcription (STAT) <span class="highlight" style="background-color:">pathway</span>; F Mitogen-activated protein Kinase (MAPK) <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nico Posnien, et al. PLoS One. 2014;9(8):e104885.</div><p class="links"><a class="dblinks" href="/pubmed/25118601" ref="ordinalpos=827&amp;ncbi_uid=6515907&amp;link_uid=25118601">Citation</a><a class="dblinks" href="/pmc/articles/PMC4132015/" ref="ordinalpos=827&amp;ncbi_uid=6515907&amp;link_uid=PMC4132015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>828.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1867573/figure/F1-6371/" ref="ordinalpos=828&amp;ncbi_uid=1044603&amp;link_uid=PMC1867573" href="/pmc/articles/PMC1867573/"><img src="/pmc/articles/instance/1867573/bin/zjx0010663710001.gif" src-large="/pmc/articles/instance/1867573/bin/zjx0010663710001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1867573/" ref="ordinalpos=828&amp;ncbi_uid=1044603&amp;link_uid=PMC1867573" image-link="/pmc/articles/PMC1867573/figure/F1-6371/" class="imagepopup">Figure 1.  From: Molecular Classification of Crohn's Disease and Ulcerative Colitis Patients Using Transcriptional Profiles in Peripheral Blood Mononuclear Cells. </a></p><div class="supp"><p class="details"><div>Functional annotation and categories of transcripts identified as CD-associated, UC-associated, and differentially expressed between UC and CD. <b>A:</b> Canonical pathways overrepresented in the CD versus normal analysis of covariance comparison (light blue); the UC versus normal analysis of covariance comparison (dark blue), and the UC versus CD analysis of covariance comparison (light purple). In this panel the negative log10 of the <i>P</i> value is plotted to highlight more significant associations. The pathways interrogated (in order) were as follows: amyloid processing, apoptosis <span class="highlight" style="background-color:">signaling</span>, arginine and proline metabolism, B-cell receptor <span class="highlight" style="background-color:">signaling</span> (nonimmunoglobulin), cardiac β-adrenergic <span class="highlight" style="background-color:">signaling</span>, chemokine <span class="highlight" style="background-color:">signaling</span>, death receptor <span class="highlight" style="background-color:">signaling</span>, ERK/MAPK <span class="highlight" style="background-color:">signaling</span>, fatty acid metabolism, cell cycle regulation (G<sub>1</sub>/S), cell cycle regulation (G<sub>2</sub>/M), GPCR <span class="highlight" style="background-color:">signaling</span>, glutamate metabolism, histidine metabolism, IGF-1 <span class="highlight" style="background-color:">signaling</span>, IL-2 <span class="highlight" style="background-color:">signaling</span>, IL-4 <span class="highlight" style="background-color:">signaling</span>, inositol phosphate metabolism, insulin receptor <span class="highlight" style="background-color:">signaling</span>, integrin <span class="highlight" style="background-color:">signaling</span>, interferon <span class="highlight" style="background-color:">signaling</span>, JAK/STAT <span class="highlight" style="background-color:">signaling</span>, NF-κB <span class="highlight" style="background-color:">signaling</span>, nitrogen metabolism, p38 MAPK <span class="highlight" style="background-color:">signaling</span>, PI3K/AKT <span class="highlight" style="background-color:">signaling</span>, PPAR <span class="highlight" style="background-color:">signaling</span>, prostaglandin and leukotriene metabolism, purine metabolism, pyrimidine metabolism, starch and sucrose metabolism, T-cell receptor <span class="highlight" style="background-color:">signaling</span>, tryptophan metabolism, tyrosine metabolism, and VEGF <span class="highlight" style="background-color:">signaling</span>. <b>B:</b> CD-specific transcripts in PBMCs were functionally annotated using the Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis system (Ingenuity), and the relative distribution of transcripts in each of the chosen functional categories for the CD-associated genes are presented in the pie chart. <b>C:</b> UC-specific transcripts in PBMCs were functionally annotated manually, and the relative distribution of transcripts in immunoglobulin versus nonimmunoglobulin categories is presented.</div></p></div><div class="aux"><div class="resc">Michael E. Burczynski, et al. J Mol Diagn. 2006 February;8(1):51-61.</div><p class="links"><a class="dblinks" href="/pubmed/16436634" ref="ordinalpos=828&amp;ncbi_uid=1044603&amp;link_uid=16436634">Citation</a><a class="dblinks" href="/pmc/articles/PMC1867573/" ref="ordinalpos=828&amp;ncbi_uid=1044603&amp;link_uid=PMC1867573">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>829.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3492858/figure/F8/" ref="ordinalpos=829&amp;ncbi_uid=5433558&amp;link_uid=PMC3492858" href="/pmc/articles/PMC3492858/"><img src="/pmc/articles/instance/3492858/bin/zh10221266850008.gif" src-large="/pmc/articles/instance/3492858/bin/zh10221266850008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3492858/" ref="ordinalpos=829&amp;ncbi_uid=5433558&amp;link_uid=PMC3492858" image-link="/pmc/articles/PMC3492858/figure/F8/" class="imagepopup">Fig. 8. From: The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> effector TCF7L2 is upregulated by insulin and represses hepatic gluconeogenesis. </a></p><div class="supp"><p class="details"><div>A schematic presentation of the role of Wnt <span class="highlight" style="background-color:">signaling</span> and TCF7L2 in hepatic gluconeogenesis. Feeding upregulates plasma insulin levels, which leads to β-cat Ser<sup>675</sup> phosphorylation and TCF7L2 expression. <i>A</i>: increased levels of TCF7L2 may repress gluconeogenesis via competing with forkhead box O (FoxO) for β-cat. <i>B</i>: in addition, cat/TCF7L2, as the effector of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, may possess an intrinsic repressive effect on gluconeogenesis. P, Ser<sup>675</sup> phosphorylation.</div></p></div><div class="aux"><div class="resc">Wilfred Ip, et al. Am J Physiol Endocrinol Metab. 2012 November 1;303(9):E1166-E1176.</div><p class="links"><a class="dblinks" href="/pubmed/22967502" ref="ordinalpos=829&amp;ncbi_uid=5433558&amp;link_uid=22967502">Citation</a><a class="dblinks" href="/pmc/articles/PMC3492858/" ref="ordinalpos=829&amp;ncbi_uid=5433558&amp;link_uid=PMC3492858">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>830.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3784244/figure/F10/" ref="ordinalpos=830&amp;ncbi_uid=5335281&amp;link_uid=PMC3784244" href="/pmc/articles/PMC3784244/"><img src="/pmc/articles/instance/3784244/bin/nihms512881f10.gif" src-large="/pmc/articles/instance/3784244/bin/nihms512881f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3784244/" ref="ordinalpos=830&amp;ncbi_uid=5335281&amp;link_uid=PMC3784244" image-link="/pmc/articles/PMC3784244/figure/F10/" class="imagepopup">Figure 10.  From: Prostaglandin E2-Mediated Attenuation of Mesocortical Dopaminergic <span class="highlight" style="background-color:">Pathway</span> Is Critical for Susceptibility to Repeated Social Defeat Stress in Mice. </a></p><div class="supp"><p class="details"><div>The proposed role of EP1-mediated attenuation of the mesocortical dopaminergic <span class="highlight" style="background-color:">pathway</span> in susceptibility to repeated social defeat. Whereas acute social defeat activates the mesocortical dopaminergic <span class="highlight" style="background-color:">pathway</span> to the level enough to counteract induction of social avoidance by social defeat (left), repeated exposure to social defeat activates PGE<sub>2</sub>-EP1 <span class="highlight" style="background-color:">signaling</span>, which attenuates the mesocortical dopaminergic <span class="highlight" style="background-color:">pathway</span>, leading to social avoidance (right).</div></p></div><div class="aux"><div class="resc">Kohei Tanaka, et al. J Neurosci. 2012 March 21;32(12):4319-4329.</div><p class="links"><a class="dblinks" href="/pubmed/22442093" ref="ordinalpos=830&amp;ncbi_uid=5335281&amp;link_uid=22442093">Citation</a><a class="dblinks" href="/pmc/articles/PMC3784244/" ref="ordinalpos=830&amp;ncbi_uid=5335281&amp;link_uid=PMC3784244">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>831.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2668232/figure/f5/" ref="ordinalpos=831&amp;ncbi_uid=1848713&amp;link_uid=PMC2668232" href="/pmc/articles/PMC2668232/"><img src="/pmc/articles/instance/2668232/bin/zcm0020922780005.gif" src-large="/pmc/articles/instance/2668232/bin/zcm0020922780005.jpg" alt="FIG. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2668232/" ref="ordinalpos=831&amp;ncbi_uid=1848713&amp;link_uid=PMC2668232" image-link="/pmc/articles/PMC2668232/figure/f5/" class="imagepopup">FIG. 5. From: Pathogen Recognition and Inflammatory <span class="highlight" style="background-color:">Signaling</span> in Innate Immune Defenses. </a></p><div class="supp"><p class="details"><div>Principles in TLR <span class="highlight" style="background-color:">signaling</span>. TLR4 activates both the MyD88-dependent and MyD88-independent, TRIF-dependent pathways. The MyD88-dependent <span class="highlight" style="background-color:">pathway</span> is responsible for early-phase NF-κB and MAPK activation, which control the induction of proinflammatory cytokines. The MyD88-independent, TRIF-dependent <span class="highlight" style="background-color:">pathway</span> activates IRF3, which is required for the induction of IFN-β- and IFN-inducible genes. In addition, this <span class="highlight" style="background-color:">pathway</span> mediates late-phase NF-κB as well as MAPK activation, also contributing to inflammatory responses.</div></p></div><div class="aux"><div class="resc">Trine H. Mogensen. Clin Microbiol Rev. 2009 April;22(2):240-273.</div><p class="links"><a class="dblinks" href="/pubmed/19366914" ref="ordinalpos=831&amp;ncbi_uid=1848713&amp;link_uid=19366914">Citation</a><a class="dblinks" href="/pmc/articles/PMC2668232/" ref="ordinalpos=831&amp;ncbi_uid=1848713&amp;link_uid=PMC2668232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>832.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136946/figure/F2/" ref="ordinalpos=832&amp;ncbi_uid=3090761&amp;link_uid=PMC3136946" href="/pmc/articles/PMC3136946/"><img src="/pmc/articles/instance/3136946/bin/sgtp0203_0158_fig002.gif" src-large="/pmc/articles/instance/3136946/bin/sgtp0203_0158_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136946/" ref="ordinalpos=832&amp;ncbi_uid=3090761&amp;link_uid=PMC3136946" image-link="/pmc/articles/PMC3136946/figure/F2/" class="imagepopup">Figure 2.  From: Loco <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in longevity. </a></p><div class="supp"><p class="details"><div>Models of Loco <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with small GTPases. A decrease in Loco expression, an RGS protein in flies, enhances stress resistance and extends lifespan. Vertical arrow: our data;25 line and arrow: hypothesized pathways according to the previous published reports in references 6, 7, 14–16, 26 and 27.</div></p></div><div class="aux"><div class="resc">Yuh-Ru Lin, et al. Small GTPases. 2011 May-Jun;2(3):158-161.</div><p class="links"><a class="dblinks" href="/pubmed/21776417" ref="ordinalpos=832&amp;ncbi_uid=3090761&amp;link_uid=21776417">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136946/" ref="ordinalpos=832&amp;ncbi_uid=3090761&amp;link_uid=PMC3136946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>833.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4209606/figure/F3/" ref="ordinalpos=833&amp;ncbi_uid=6767619&amp;link_uid=PMC4209606" href="/pmc/articles/PMC4209606/"><img src="/pmc/articles/instance/4209606/bin/ganc-05-348-g003.gif" src-large="/pmc/articles/instance/4209606/bin/ganc-05-348-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4209606/" ref="ordinalpos=833&amp;ncbi_uid=6767619&amp;link_uid=PMC4209606" image-link="/pmc/articles/PMC4209606/figure/F3/" class="imagepopup">Figure 3. TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> impaired mice survive shorter than WT mice under alcohol- and LPS-treatment.  From: Generation of a mouse model of T-cell lymphoma based on chronic LPS challenge and TGF-? <span class="highlight" style="background-color:">signaling</span> disruption. </a></p><div class="supp"><p class="details"><div>Mice with constitutive disruption of the TGF-β <span class="highlight" style="background-color:">signaling</span> via heterozygous deletion of β2-Spectrin (<i>Sptbn1<sup>+/−</sup></i>) (n=14), Smad3 (<i>Smad3<sup>+/−</sup></i>) (n=9) orboth (<i>Sptbn1<sup>+/−</sup></i>; <i>Smad3<sup>+/−</sup></i>) (n=11) treated with alcohol and LPS have significantly decreased cancer-free survival compared to wild type mice (n=15). *p&lt;0.05/ ***p&lt;0.005.</div></p></div><div class="aux"><div class="resc">Nina M. Muñoz, et al. Genes Cancer. 2014 September;5(9-10):348-352.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=833&amp;ncbi_uid=6767619&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4209606/" ref="ordinalpos=833&amp;ncbi_uid=6767619&amp;link_uid=PMC4209606">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>834.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3351456/figure/pone-0036911-g007/" ref="ordinalpos=834&amp;ncbi_uid=4008512&amp;link_uid=PMC3351456" href="/pmc/articles/PMC3351456/"><img src="/pmc/articles/instance/3351456/bin/pone.0036911.g007.gif" src-large="/pmc/articles/instance/3351456/bin/pone.0036911.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3351456/" ref="ordinalpos=834&amp;ncbi_uid=4008512&amp;link_uid=PMC3351456" image-link="/pmc/articles/PMC3351456/figure/pone-0036911-g007/" class="imagepopup">Figure 7. Cellular context modification of downstream receptor <span class="highlight" style="background-color:">signaling</span> activity in SH-SY5Y cells..  From: VENNTURE-A Novel Venn Diagram Investigational Tool for Multiple Pharmacological Dataset Analysis. </a></p><div class="supp"><p class="details"><div>(A) Comparison of phosphoprotein/GO term group/canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in the non-stimulated (blue circle) and multiple MeCh-stimulated conditions (red circles) in either control-state (solid line) or CMP-state (dashed lines) SH-SY5Y cells. (B) Summary of potential non-stimulated and MeCh-induced <span class="highlight" style="background-color:">signaling</span> functionality (aggregate of highest-scoring GO-term group and canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> enrichments) in a dose-dependent manner compared between control-state or CMP-state cells.</div></p></div><div class="aux"><div class="resc">Bronwen Martin, et al. PLoS One. 2012;7(5):e36911.</div><p class="links"><a class="dblinks" href="/pubmed/22606307" ref="ordinalpos=834&amp;ncbi_uid=4008512&amp;link_uid=22606307">Citation</a><a class="dblinks" href="/pmc/articles/PMC3351456/" ref="ordinalpos=834&amp;ncbi_uid=4008512&amp;link_uid=PMC3351456">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>835.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3858175/figure/F5/" ref="ordinalpos=835&amp;ncbi_uid=5584108&amp;link_uid=PMC3858175" href="/pmc/articles/PMC3858175/"><img src="/pmc/articles/instance/3858175/bin/nihms513659f5.gif" src-large="/pmc/articles/instance/3858175/bin/nihms513659f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3858175/" ref="ordinalpos=835&amp;ncbi_uid=5584108&amp;link_uid=PMC3858175" image-link="/pmc/articles/PMC3858175/figure/F5/" class="imagepopup">Figure 5.  From: HER-2 <span class="highlight" style="background-color:">Signaling</span>, Acquisition of Growth Factor Independence, and Regulation of Biological Networks Associated with Cell Transformation. </a></p><div class="supp"><p class="details"><div>The progressive acquisition of HER-2–mediated transforming activity results in dramatic changes in downstream gene regulation affecting many different biological processes. Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis was performed on the genes regulated by EGFR/HER-2 <span class="highlight" style="background-color:">signaling</span> in the isogenic cell line series and grouped into the four biological categories based on the canonical pathways indicated in the figure. The set of bars on the far left indicates the number of common genes in each canonical <span class="highlight" style="background-color:">pathway</span> set that are regulated by EGFR/HER-2 <span class="highlight" style="background-color:">signaling</span> in all three cell lines. The middle set of bars indicates the number of common genes plus the number of genes in each canonical <span class="highlight" style="background-color:">pathway</span> set regulated by EGFR/HER-2 <span class="highlight" style="background-color:">signaling</span> in both HER-2–overexpressing lines, but not in MCF-10A cells. The set of bars on the right indicates the number of genes regulated in all three lines, plus the genes regulated in both HER-2–overexpressing lines, plus the genes regulated by HER-2/EGFR <span class="highlight" style="background-color:">signaling</span> in MCF-10HER2/E7 cells, but not the other two cell lines. Red bars show the number of genes positively regulated by <span class="highlight" style="background-color:">signaling</span>, and green bars indicate the number of genes negatively regulated by <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Aliccia Bollig-Fischer, et al. Cancer Res. ;70(20):10.1158/0008-5472.CAN-10-1529.</div><p class="links"><a class="dblinks" href="/pubmed/20736364" ref="ordinalpos=835&amp;ncbi_uid=5584108&amp;link_uid=20736364">Citation</a><a class="dblinks" href="/pmc/articles/PMC3858175/" ref="ordinalpos=835&amp;ncbi_uid=5584108&amp;link_uid=PMC3858175">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>836.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3555267/figure/f2/" ref="ordinalpos=836&amp;ncbi_uid=4560327&amp;link_uid=PMC3555267" href="/pmc/articles/PMC3555267/"><img src="/pmc/articles/instance/3555267/bin/cjcr-24-02-083-f2.gif" src-large="/pmc/articles/instance/3555267/bin/cjcr-24-02-083-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3555267/" ref="ordinalpos=836&amp;ncbi_uid=4560327&amp;link_uid=PMC3555267" image-link="/pmc/articles/PMC3555267/figure/f2/" class="imagepopup">Figure 2.  From: Typical Cell <span class="highlight" style="background-color:">Signaling</span> Response to Ionizing Radiation: DNA Damage and Extranuclear Damage. </a></p><div class="supp"><p class="details"><div>Typical MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hui Yu. Chin J Cancer Res. 2012 June;24(2):83-89.</div><p class="links"><a class="dblinks" href="/pubmed/23357898" ref="ordinalpos=836&amp;ncbi_uid=4560327&amp;link_uid=23357898">Citation</a><a class="dblinks" href="/pmc/articles/PMC3555267/" ref="ordinalpos=836&amp;ncbi_uid=4560327&amp;link_uid=PMC3555267">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>837.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3606804/figure/fig2/" ref="ordinalpos=837&amp;ncbi_uid=4740306&amp;link_uid=PMC3606804" href="/pmc/articles/PMC3606804/"><img src="/pmc/articles/instance/3606804/bin/OXIMED2013-306073.002.gif" src-large="/pmc/articles/instance/3606804/bin/OXIMED2013-306073.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3606804/" ref="ordinalpos=837&amp;ncbi_uid=4740306&amp;link_uid=PMC3606804" image-link="/pmc/articles/PMC3606804/figure/fig2/" class="imagepopup">Figure 2.  From: Preventive and Therapeutic Effects of MG132 by Activating Nrf2-ARE <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> on Oxidative Stress-Induced Cardiovascular and Renal Injury. </a></p><div class="supp"><p class="details"><div>The process of target protein degeneration by UPS in eukaryotic cells (a) and mechanism of MG132 activate Nrf2/Keap1-ARE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (b). Abbreviations: ARE: antioxidant-responsive element; Cys: cysteine; E1: ubiquitin-activating; E2: ubiquitin-conjugating; E3: ubiquitin-ligase; Keap1: Kelch-like ECH-associated protein 1; Nrf2: E2-related factor 2; Ub: ubiquitin.</div></p></div><div class="aux"><div class="resc">Wenpeng Cui, et al. Oxid Med Cell Longev. 2013;2013:306073.</div><p class="links"><a class="dblinks" href="/pubmed/23533688" ref="ordinalpos=837&amp;ncbi_uid=4740306&amp;link_uid=23533688">Citation</a><a class="dblinks" href="/pmc/articles/PMC3606804/" ref="ordinalpos=837&amp;ncbi_uid=4740306&amp;link_uid=PMC3606804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>838.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2711543/figure/F5/" ref="ordinalpos=838&amp;ncbi_uid=1909179&amp;link_uid=PMC2711543" href="/pmc/articles/PMC2711543/"><img src="/pmc/articles/instance/2711543/bin/nihms110404f5.gif" src-large="/pmc/articles/instance/2711543/bin/nihms110404f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2711543/" ref="ordinalpos=838&amp;ncbi_uid=1909179&amp;link_uid=PMC2711543" image-link="/pmc/articles/PMC2711543/figure/F5/" class="imagepopup">Figure 5. Caspase-8 tyrosine 380 may be involved in negative regulation of EGF <span class="highlight" style="background-color:">signaling</span>.  From: Critical Role for Caspase-8 in EGF <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A) Immunoblot analysis of EGF-induced Erk and Akt <span class="highlight" style="background-color:">pathway</span> activation in NB7 cells lacking caspase-8 re-constituted with empty vector (E.V.), the DEDs alone (DEDs) or the “caspase domain” of caspase-8 (Casp. Dom.) (lanes 1–3, respectively). B) Immunoblot analysis of EGF-induced Erk and Akt <span class="highlight" style="background-color:">pathway</span> activation in NB7 cells lacking caspase-8 re-constituted with empty vector (E.V.), the catalytically inactive point mutant (C360A) or the C360A/Y380F dual mutant of caspase-8 (C360A/Y380F) (lanes 1–3, respectively). C) Immunoblot analysis of EGF-induced Erk <span class="highlight" style="background-color:">pathway</span> activation in NB7 cells lacking caspase-8 re-constituted with empty vector, the catalytically inactive point mutant (C360A) or the C360A/Y380F dual mutant of caspase-8 (as in Fig 5B) with or without sodium orthovanadate (200 µM) as described (lanes 1–4, respectively). D) Simplified schematic depiction of the proposed model of the critical role for caspase-8 in EGF induction of Erk <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> events.</div></p></div><div class="aux"><div class="resc">Darren Finlay, et al. Cancer Res. ;69(12):5023-5029.</div><p class="links"><a class="dblinks" href="/pubmed/19470771" ref="ordinalpos=838&amp;ncbi_uid=1909179&amp;link_uid=19470771">Citation</a><a class="dblinks" href="/pmc/articles/PMC2711543/" ref="ordinalpos=838&amp;ncbi_uid=1909179&amp;link_uid=PMC2711543">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>839.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3960487/figure/F2/" ref="ordinalpos=839&amp;ncbi_uid=5962686&amp;link_uid=PMC3960487" href="/pmc/articles/PMC3960487/"><img src="/pmc/articles/instance/3960487/bin/fphys-05-00103-g0002.gif" src-large="/pmc/articles/instance/3960487/bin/fphys-05-00103-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3960487/" ref="ordinalpos=839&amp;ncbi_uid=5962686&amp;link_uid=PMC3960487" image-link="/pmc/articles/PMC3960487/figure/F2/" class="imagepopup">Figure 2.  From: Four-way regulation of mosquito yolk protein precursor genes by juvenile hormone-, ecdysone-, nutrient-, and insulin-like peptide <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> pathways involved in YPP regulation</b>. <b>(A)</b> Juvenile hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. JH, juvenile hormone; Met, methoprene tolerant. <b>(B)</b> Ecdysone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. 20E, 20 hydroxyecdysone; EcR, ecdysone receptor; USP, ultraspiracle. <b>(C)</b> Nutrient <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. 4E-BP, 4E-binding protein; TSC, tuberous sclerosis complex; RHEB, RAS homologue enriched in brain; S6K, S6 kinase; TOR, target of rapamycin. <b>(D)</b> Insulin-like peptide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. AKT, protein kinase B; FOXO, forkhead box protein O; ILP, insulin-like peptide; IRS, insulin receptor substrate; PI3K, phosphati dylinositide 3-kinase; PIP2, phosphatidylinositol 4,5-bisphosphate; PIP3, phosphatidylinositol (3,4,5)-triphosphate; PTEN, phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase. <b>(E)</b> Schematic of the vg-A promoter with transcription factor binding sites.</div></p></div><div class="aux"><div class="resc">Immo A. Hansen, et al. Front Physiol. 2014;5:103.</div><p class="links"><a class="dblinks" href="/pubmed/24688471" ref="ordinalpos=839&amp;ncbi_uid=5962686&amp;link_uid=24688471">Citation</a><a class="dblinks" href="/pmc/articles/PMC3960487/" ref="ordinalpos=839&amp;ncbi_uid=5962686&amp;link_uid=PMC3960487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>840.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2172611/figure/fig1/" ref="ordinalpos=840&amp;ncbi_uid=1126153&amp;link_uid=PMC2172611" href="/pmc/articles/PMC2172611/"><img src="/pmc/articles/instance/2172611/bin/200407100f1.gif" src-large="/pmc/articles/instance/2172611/bin/200407100f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2172611/" ref="ordinalpos=840&amp;ncbi_uid=1126153&amp;link_uid=PMC2172611" image-link="/pmc/articles/PMC2172611/figure/fig1/" class="imagepopup">Figure 1. From: The adhesion force of Notch with Delta and the rate of Notch <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Notch <span class="highlight" style="background-color:">signaling</span> and the various Notch receptors used in this paper.</b> (A) The SuH/N<sup>intra</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) The structures of Notch receptors used in this paper; relative activities shown on an arbitrary scale. TM, transmembrane domain.</div></p></div><div class="aux"><div class="resc">Francois Ahimou, et al. J Cell Biol. 2004 December 20;167(6):1217-1229.</div><p class="links"><a class="dblinks" href="/pubmed/15611340" ref="ordinalpos=840&amp;ncbi_uid=1126153&amp;link_uid=15611340">Citation</a><a class="dblinks" href="/pmc/articles/PMC2172611/" ref="ordinalpos=840&amp;ncbi_uid=1126153&amp;link_uid=PMC2172611">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>841.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3607076/figure/F4/" ref="ordinalpos=841&amp;ncbi_uid=4737900&amp;link_uid=PMC3607076" href="/pmc/articles/PMC3607076/"><img src="/pmc/articles/instance/3607076/bin/fgene-04-00037-g004.gif" src-large="/pmc/articles/instance/3607076/bin/fgene-04-00037-g004.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3607076/" ref="ordinalpos=841&amp;ncbi_uid=4737900&amp;link_uid=PMC3607076" image-link="/pmc/articles/PMC3607076/figure/F4/" class="imagepopup">FIGURE 4.  From: The ATM <span class="highlight" style="background-color:">signaling</span> network in development and disease. </a></p><div class="supp"><p class="details"><div><b>Selected summary of the ATM <span class="highlight" style="background-color:">signaling</span> network</b>. A non-comprehensive schematic of ATM <span class="highlight" style="background-color:">signaling</span> pathways described in the text. ATM substrates are indicated in bold black, other <span class="highlight" style="background-color:">pathway</span> effectors in black, affected cellular processes in blue, and cell cycle checkpoints in green. See text for details regarding the <span class="highlight" style="background-color:">signaling</span> pathways depicted.</div></p></div><div class="aux"><div class="resc">Travis H. Stracker, et al. Front Genet. 2013;4:37.</div><p class="links"><a class="dblinks" href="/pubmed/23532176" ref="ordinalpos=841&amp;ncbi_uid=4737900&amp;link_uid=23532176">Citation</a><a class="dblinks" href="/pmc/articles/PMC3607076/" ref="ordinalpos=841&amp;ncbi_uid=4737900&amp;link_uid=PMC3607076">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>842.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC316921/figure/F6/" ref="ordinalpos=842&amp;ncbi_uid=408393&amp;link_uid=PMC316921" href="/pmc/articles/PMC316921/"><img src="/pmc/articles/instance/316921/bin/gad.2f6.gif" src-large="/pmc/articles/instance/316921/bin/gad.2f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC316921/" ref="ordinalpos=842&amp;ncbi_uid=408393&amp;link_uid=PMC316921" image-link="/pmc/articles/PMC316921/figure/F6/" class="imagepopup">Figure 6.  From: Wnt <span class="highlight" style="background-color:">pathway</span> components orient a mitotic spindle in the early Caenorhabditis elegans embryo without requiring gene transcription in the responding cell. </a></p><div class="supp"><p class="details"><div>Model for Wnt <span class="highlight" style="background-color:">signaling</span> in the early <i>C. elegans</i> embryo. The Wnt <span class="highlight" style="background-color:">pathway</span> in the early <i>C. elegans</i> embryo (<i>B</i>) is contrasted to the ‘canonical’ <span class="highlight" style="background-color:">pathway</span> derived from experimental data from other organisms (<i>A</i>; see Cadigan and Nusse 1997). In this diagram, the <span class="highlight" style="background-color:">signaling</span> cell is drawn above the responding cell. As shown in <i>B</i>, Wnt <span class="highlight" style="background-color:">signaling</span> is required for both EMS mitotic spindle orientation (<i>left</i>) and endoderm specification (<i>right</i>), although there may be separate <span class="highlight" style="background-color:">signaling</span> ligands for these two processes (Rocheleau et al. 1997; Thorpe et al. 1997; A. Schlesinger et al., unpubl.). <i>gsk-3</i> may represent a branchpoint in the EMS Wnt <span class="highlight" style="background-color:">pathway</span>, regulating both endoderm induction and mitotic spindle orientation. Not depicted but also possible are arrows from <i>gsk-3</i> directly to the EMS spindle, and directly to the cytoskeleton during endoderm induction. In contrast to the roles of <i>pan</i> and LEF-1, which are converted into activators by Wnt <span class="highlight" style="background-color:">signaling</span>, <i>pop-1</i> acts purely as a repressor in the <i>C. elegans</i> <span class="highlight" style="background-color:">pathway</span> (Cadigan and Nusse 1997). Release from POP-1-mediated repression of endoderm occurs not in EMS but in its posterior daughter, E.</div></p></div><div class="aux"><div class="resc">Ann Schlesinger, et al. Genes Dev. 1999 August 1;13(15):2028-2038.</div><p class="links"><a class="dblinks" href="/pubmed/10444600" ref="ordinalpos=842&amp;ncbi_uid=408393&amp;link_uid=10444600">Citation</a><a class="dblinks" href="/pmc/articles/PMC316921/" ref="ordinalpos=842&amp;ncbi_uid=408393&amp;link_uid=PMC316921">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>843.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4014502/figure/pone-0096351-g005/" ref="ordinalpos=843&amp;ncbi_uid=6132602&amp;link_uid=PMC4014502" href="/pmc/articles/PMC4014502/"><img src="/pmc/articles/instance/4014502/bin/pone.0096351.g005.gif" src-large="/pmc/articles/instance/4014502/bin/pone.0096351.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4014502/" ref="ordinalpos=843&amp;ncbi_uid=6132602&amp;link_uid=PMC4014502" image-link="/pmc/articles/PMC4014502/figure/pone-0096351-g005/" class="imagepopup">Figure 5. Expression of NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecules in rat kidney tissues in different age groups..  From: Changes in the Expression of the Toll-Like Receptor System in the Aging Rat Kidneys. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Western blot detection of the expression of NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins in various rat groups. (<b>B</b>) Quantitative analysis of gray scale. Compared with the 3-month-old group, * indicates P&lt;0.05; compared with the 12-month-old group, # indicates P&lt;0.05.</div></p></div><div class="aux"><div class="resc">Yue Xi, et al. PLoS One. 2014;9(5):e96351.</div><p class="links"><a class="dblinks" href="/pubmed/24810370" ref="ordinalpos=843&amp;ncbi_uid=6132602&amp;link_uid=24810370">Citation</a><a class="dblinks" href="/pmc/articles/PMC4014502/" ref="ordinalpos=843&amp;ncbi_uid=6132602&amp;link_uid=PMC4014502">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>844.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3433278/figure/F7/" ref="ordinalpos=844&amp;ncbi_uid=4213135&amp;link_uid=PMC3433278" href="/pmc/articles/PMC3433278/"><img src="/pmc/articles/instance/3433278/bin/nihms-355255-f0007.gif" src-large="/pmc/articles/instance/3433278/bin/nihms-355255-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3433278/" ref="ordinalpos=844&amp;ncbi_uid=4213135&amp;link_uid=PMC3433278" image-link="/pmc/articles/PMC3433278/figure/F7/" class="imagepopup">Figure 7.  From: Starvation activates MAP kinase through the muscarinic acetylcholine <span class="highlight" style="background-color:">pathway</span> in Caenorhabditis elegans pharynx. </a></p><div class="supp"><p class="details"><div>Starvation induces pumping in a muscarinic <span class="highlight" style="background-color:">signaling</span>-dependent manner<br /><b>A)</b> Starvation induced pumping in a muscarinic <span class="highlight" style="background-color:">signaling</span>-dependent manner. Pumping rates increased for first 2 hr in the absence of food. Two mutants defective in muscarinic <span class="highlight" style="background-color:">signaling</span> showed altered pumping rates compared to wild-type. <i>gpb-2</i> mutants, which have upregulated muscarinic <span class="highlight" style="background-color:">signaling</span>, showed a higher pumping rate, and <i>gar-3</i> mutants, which have downregulated muscarinic <span class="highlight" style="background-color:">signaling</span>, showed a lower pumping rate than wild-type.<br /><b>B)</b> A suggested <span class="highlight" style="background-color:">pathway</span> for function of the muscarinic receptor → MAPK <span class="highlight" style="background-color:">pathway</span> in starvation.</div></p></div><div class="aux"><div class="resc">Young-jai You, et al. Cell Metab. 2006 April;3(4):237-245.</div><p class="links"><a class="dblinks" href="/pubmed/16581001" ref="ordinalpos=844&amp;ncbi_uid=4213135&amp;link_uid=16581001">Citation</a><a class="dblinks" href="/pmc/articles/PMC3433278/" ref="ordinalpos=844&amp;ncbi_uid=4213135&amp;link_uid=PMC3433278">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>845.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3841149/figure/pone-0081769-g009/" ref="ordinalpos=845&amp;ncbi_uid=5559644&amp;link_uid=PMC3841149" href="/pmc/articles/PMC3841149/"><img src="/pmc/articles/instance/3841149/bin/pone.0081769.g009.gif" src-large="/pmc/articles/instance/3841149/bin/pone.0081769.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3841149/" ref="ordinalpos=845&amp;ncbi_uid=5559644&amp;link_uid=PMC3841149" image-link="/pmc/articles/PMC3841149/figure/pone-0081769-g009/" class="imagepopup">Figure 9. Interactions between WNT3 and SHH <span class="highlight" style="background-color:">signaling</span> pathways in GCPs..  From: WNT3 Inhibits Cerebellar Granule Neuron Progenitor Proliferation and Medulloblastoma Formation via MAPK Activation. </a></p><div class="supp"><p class="details"><div>In Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span>, Sonic Hedgehog (SHH) binds Patched (PTCH) receptor, relieving its inhibition of Smoothened (SMO), and stimulating GCP proliferation. In WNT3 <span class="highlight" style="background-color:">signaling</span>, WNT3 binds an undefined receptor to activate MAPK <span class="highlight" style="background-color:">signaling</span> via a non canonical WNT <span class="highlight" style="background-color:">pathway</span> to promote GCP differentiation, whereas other WNTs bind Frizzled (FRZ) receptor to activate the canonical β-catenin <span class="highlight" style="background-color:">pathway</span> and stimulate proliferation. We propose a model whereby regulation of MAPK activity by WNT3 <span class="highlight" style="background-color:">signaling</span> alters the balance between proliferation and differentiation in GCPs, decreasing proliferation and increasing differentiation.</div></p></div><div class="aux"><div class="resc">Sandrine L. Anne, et al. PLoS One. 2013;8(11):e81769.</div><p class="links"><a class="dblinks" href="/pubmed/24303070" ref="ordinalpos=845&amp;ncbi_uid=5559644&amp;link_uid=24303070">Citation</a><a class="dblinks" href="/pmc/articles/PMC3841149/" ref="ordinalpos=845&amp;ncbi_uid=5559644&amp;link_uid=PMC3841149">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>846.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2768917/figure/fig1/" ref="ordinalpos=846&amp;ncbi_uid=2446632&amp;link_uid=PMC2768917" href="/pmc/articles/PMC2768917/"><img src="/pmc/articles/instance/2768917/bin/tpc2102542f01.gif" src-large="/pmc/articles/instance/2768917/bin/tpc2102542f01.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2768917/" ref="ordinalpos=846&amp;ncbi_uid=2446632&amp;link_uid=PMC2768917" image-link="/pmc/articles/PMC2768917/figure/fig1/" class="imagepopup">Figure 1.  From: Integration of <span class="highlight" style="background-color:">Signaling</span> Pathways in Stomatal Development. </a></p><div class="supp"><p class="details"><div>A genetic model proposing how light-stimulated stomatal development is regulated by interactions among photoreceptors, components of the light <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and members of the endogenous developmental <span class="highlight" style="background-color:">pathway</span>. Arrow, positive regulation; T-bar, negative regulation.</div></p></div><div class="aux"><div class="resc">Gregory Bertoni. Plant Cell. 2009 September;21(9):2542-2542.</div><p class="links"><a class="dblinks" href="/pubmed/19794117" ref="ordinalpos=846&amp;ncbi_uid=2446632&amp;link_uid=19794117">Citation</a><a class="dblinks" href="/pmc/articles/PMC2768917/" ref="ordinalpos=846&amp;ncbi_uid=2446632&amp;link_uid=PMC2768917">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>847.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3789883/figure/pgen-1003770-g007/" ref="ordinalpos=847&amp;ncbi_uid=5370218&amp;link_uid=PMC3789883" href="/pmc/articles/PMC3789883/"><img src="/pmc/articles/instance/3789883/bin/pgen.1003770.g007.gif" src-large="/pmc/articles/instance/3789883/bin/pgen.1003770.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3789883/" ref="ordinalpos=847&amp;ncbi_uid=5370218&amp;link_uid=PMC3789883" image-link="/pmc/articles/PMC3789883/figure/pgen-1003770-g007/" class="imagepopup">Figure 7. Top four BFs in CD for each setting of ..  From: Integrated Enrichment Analysis of Variants and Pathways in Genome-Wide Association Studies Indicates Central Role for IL-2 <span class="highlight" style="background-color:">Signaling</span> Genes in Type 1 Diabetes, and Cytokine <span class="highlight" style="background-color:">Signaling</span> Genes in Crohn's Disease. </a></p><div class="supp"><p class="details"><div>In each case, the 3 largest BFs correspond, in order, to Cytokine <span class="highlight" style="background-color:">signaling</span> in immune system, IL23-mediated <span class="highlight" style="background-color:">signaling</span> events, and IL12-mediated <span class="highlight" style="background-color:">signaling</span> events (these are the top 3 pathways for CD in Figure 2). <span class="highlight" style="background-color:">Pathway</span> with fourth largest BF differs across settings of <img src="/pmc/articles/instance/3789883/bin/pgen.1003770.e163.jpg" class="inline-graphic" style="border-style: none;" />.</div></p></div><div class="aux"><div class="resc">Peter Carbonetto, et al. PLoS Genet. 2013 October;9(10):e1003770.</div><p class="links"><a class="dblinks" href="/pubmed/24098138" ref="ordinalpos=847&amp;ncbi_uid=5370218&amp;link_uid=24098138">Citation</a><a class="dblinks" href="/pmc/articles/PMC3789883/" ref="ordinalpos=847&amp;ncbi_uid=5370218&amp;link_uid=PMC3789883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>848.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3070849/figure/F1/" ref="ordinalpos=848&amp;ncbi_uid=3504184&amp;link_uid=PMC3070849" href="/pmc/articles/PMC3070849/"><img src="/pmc/articles/instance/3070849/bin/nihms-271828-f0001.gif" src-large="/pmc/articles/instance/3070849/bin/nihms-271828-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3070849/" ref="ordinalpos=848&amp;ncbi_uid=3504184&amp;link_uid=PMC3070849" image-link="/pmc/articles/PMC3070849/figure/F1/" class="imagepopup">Figure 1. PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in B cell development.  From: Emergence of the PI3-kinase <span class="highlight" style="background-color:">pathway</span> as a central modulator of normal and aberrant B cell differentiation. </a></p><div class="supp"><p class="details"><div>Pro-B cell development is supported by IL-7 stimulation via the PI3K and STAT5 pathways. Weak PI3K <span class="highlight" style="background-color:">signaling</span> promotes retention of Foxo1 in the nucleus to drive <i>Il7ra</i> expression. PI3K <span class="highlight" style="background-color:">signaling</span> via the pre-BCR can be augmented by CD19 and countered by BLNK(SLP-65). Expression of pre-BCR components is repressed in an IRF4/IRF8-dependent manner to permit Foxo1-dependent expression of <i>Rag</i> in resting pre-B cells. Peripheral B cells require PI3K <span class="highlight" style="background-color:">signaling</span> to promote survival while Foxo1 activity is crucial for expression of <i>Sell</i> and <i>Aicda</i>, suggesting a dynamic balance for Foxo1 activity in peripheral B cell homeostasis and differentiation.</div></p></div><div class="aux"><div class="resc">G.V. Baracho, et al. Curr Opin Immunol. ;23(2):178-183.</div><p class="links"><a class="dblinks" href="/pubmed/21277760" ref="ordinalpos=848&amp;ncbi_uid=3504184&amp;link_uid=21277760">Citation</a><a class="dblinks" href="/pmc/articles/PMC3070849/" ref="ordinalpos=848&amp;ncbi_uid=3504184&amp;link_uid=PMC3070849">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>849.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3122626/figure/F8/" ref="ordinalpos=849&amp;ncbi_uid=3493281&amp;link_uid=PMC3122626" href="/pmc/articles/PMC3122626/"><img src="/pmc/articles/instance/3122626/bin/zmr9990922690008.gif" src-large="/pmc/articles/instance/3122626/bin/zmr9990922690008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3122626/" ref="ordinalpos=849&amp;ncbi_uid=3493281&amp;link_uid=PMC3122626" image-link="/pmc/articles/PMC3122626/figure/F8/" class="imagepopup">Fig. 8. From: Regulatory Circuitry Governing Fungal Development, Drug Resistance, and Disease. </a></p><div class="supp"><p class="details"><div>Comparison of cAMP-PKA and MAPK <span class="highlight" style="background-color:">signaling</span> cascades in C. albicans, C. neoformans, and A. fumigatus. The cAMP-PKA <span class="highlight" style="background-color:">signaling</span> cascade and MAPK cascade represent two key morphogenetic <span class="highlight" style="background-color:">signaling</span> pathways conserved in these species. The MAPK <span class="highlight" style="background-color:">signaling</span> cascades modulating morphogenesis in A. fumigatus (the HOG-MAPK <span class="highlight" style="background-color:">pathway</span> and cell wall integrity MAPK <span class="highlight" style="background-color:">pathway</span>) are distinct from the MAPK module depicted in this figure. (Top) The cAMP-PKA <span class="highlight" style="background-color:">pathway</span>. (Bottom) The MAPK cascade. Corresponding colors between pathways indicate orthologous proteins between the species. Gray arrows indicate that links between certain factors in this <span class="highlight" style="background-color:">pathway</span> have not yet been established.</div></p></div><div class="aux"><div class="resc">Rebecca S. Shapiro, et al. Microbiol Mol Biol Rev. 2011 June;75(2):213-267.</div><p class="links"><a class="dblinks" href="/pubmed/21646428" ref="ordinalpos=849&amp;ncbi_uid=3493281&amp;link_uid=21646428">Citation</a><a class="dblinks" href="/pmc/articles/PMC3122626/" ref="ordinalpos=849&amp;ncbi_uid=3493281&amp;link_uid=PMC3122626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>850.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2441985/figure/F3/" ref="ordinalpos=850&amp;ncbi_uid=1396287&amp;link_uid=PMC2441985" href="/pmc/articles/PMC2441985/"><img src="/pmc/articles/instance/2441985/bin/1276fig3.gif" src-large="/pmc/articles/instance/2441985/bin/1276fig3.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2441985/" ref="ordinalpos=850&amp;ncbi_uid=1396287&amp;link_uid=PMC2441985" image-link="/pmc/articles/PMC2441985/figure/F3/" class="imagepopup">FIGURE 3. From: A computational screen for mouse <span class="highlight" style="background-color:">signaling</span> pathways targeted by microRNA clusters. </a></p><div class="supp"><p class="details"><div>Model of the regulation of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by miRNA cluster <i>mmu–mir-183–96–182. Mir-96</i> and <i>mir-183</i> are mainly responsible for regulating <i>Irs1</i>, which is the key mediator of the metabolic aspect of the insulin <span class="highlight" style="background-color:">pathway</span>, whereas <i>mir-182</i> targets <i>Rasa1</i> and <i>Grb2</i>, both of which play important roles in linking insulin <span class="highlight" style="background-color:">signaling</span> with the <i>Ras</i> <span class="highlight" style="background-color:">pathway</span>. A dotted arrow indicates the experimentally verified miRNA and target gene interactions via luciferase activity assay.</div></p></div><div class="aux"><div class="resc">Jianzhen Xu, et al. RNA. 2008 July;14(7):1276-1283.</div><p class="links"><a class="dblinks" href="/pubmed/18511500" ref="ordinalpos=850&amp;ncbi_uid=1396287&amp;link_uid=18511500">Citation</a><a class="dblinks" href="/pmc/articles/PMC2441985/" ref="ordinalpos=850&amp;ncbi_uid=1396287&amp;link_uid=PMC2441985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>851.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2955764/figure/F1/" ref="ordinalpos=851&amp;ncbi_uid=3260571&amp;link_uid=PMC2955764" href="/pmc/articles/PMC2955764/"><img src="/pmc/articles/instance/2955764/bin/nihms-230789-f0001.gif" src-large="/pmc/articles/instance/2955764/bin/nihms-230789-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2955764/" ref="ordinalpos=851&amp;ncbi_uid=3260571&amp;link_uid=PMC2955764" image-link="/pmc/articles/PMC2955764/figure/F1/" class="imagepopup">Figure 1.  From: Genetic Variations in the Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span> Affect Clinical Outcomes in Non-muscle-invasive Bladder Cancer. </a></p><div class="supp"><p class="details"><div>Sonic hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Meng Chen, et al. Cancer Prev Res (Phila). ;3(10):1235-1245.</div><p class="links"><a class="dblinks" href="/pubmed/20858759" ref="ordinalpos=851&amp;ncbi_uid=3260571&amp;link_uid=20858759">Citation</a><a class="dblinks" href="/pmc/articles/PMC2955764/" ref="ordinalpos=851&amp;ncbi_uid=3260571&amp;link_uid=PMC2955764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>852.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4294543/figure/F3/" ref="ordinalpos=852&amp;ncbi_uid=7062544&amp;link_uid=PMC4294543" href="/pmc/articles/PMC4294543/"><img src="/pmc/articles/instance/4294543/bin/nihms-653802-f0003.gif" src-large="/pmc/articles/instance/4294543/bin/nihms-653802-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4294543/" ref="ordinalpos=852&amp;ncbi_uid=7062544&amp;link_uid=PMC4294543" image-link="/pmc/articles/PMC4294543/figure/F3/" class="imagepopup">Fig. 3.  From: Cholangiocarcinoma: Molecular Pathways and Therapeutic Opportunities. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and inhibitors. The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has been implicated in cholangiocarcinoma (CCA) carcino-genesis. Enhanced expression of NICD occurs in CCA. Several inhibitors of the Notch <span class="highlight" style="background-color:">pathway</span> are currently in development. These include inhibitors of γ-secretase, a key enzyme in the Notch <span class="highlight" style="background-color:">pathway</span>, as well as monoclonal antibodies (mABs) targeting the Notch receptors and ligands.</div></p></div><div class="aux"><div class="resc">Sumera Rizvi, et al. Semin Liver Dis. ;34(4):456-464.</div><p class="links"><a class="dblinks" href="/pubmed/25369307" ref="ordinalpos=852&amp;ncbi_uid=7062544&amp;link_uid=25369307">Citation</a><a class="dblinks" href="/pmc/articles/PMC4294543/" ref="ordinalpos=852&amp;ncbi_uid=7062544&amp;link_uid=PMC4294543">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>853.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC304121/figure/fig1/" ref="ordinalpos=853&amp;ncbi_uid=344246&amp;link_uid=PMC304121" href="/pmc/articles/PMC304121/"><img src="/pmc/articles/instance/304121/bin/pq1934788001.gif" src-large="/pmc/articles/instance/304121/bin/pq1934788001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC304121/" ref="ordinalpos=853&amp;ncbi_uid=344246&amp;link_uid=PMC304121" image-link="/pmc/articles/PMC304121/figure/fig1/" class="imagepopup">Fig. 1. From: Systemins: A functionally defined family of peptide signals that regulate defensive genes in Solanaceae species. </a></p><div class="supp"><p class="details"><div>A simplified diagram of the systemin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> shows several key steps of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and in particular the steps leading to the blockage of a proton ATPase that leads to the alkalinization of the extracellular medium, which is the basis of the assay developed to identify <span class="highlight" style="background-color:">signaling</span> peptides.</div></p></div><div class="aux"><div class="resc">Clarence A. Ryan, et al. Proc Natl Acad Sci U S A. 2003 November 25;100(Suppl 2):14577-14580.</div><p class="links"><a class="dblinks" href="/pubmed/12949264" ref="ordinalpos=853&amp;ncbi_uid=344246&amp;link_uid=12949264">Citation</a><a class="dblinks" href="/pmc/articles/PMC304121/" ref="ordinalpos=853&amp;ncbi_uid=344246&amp;link_uid=PMC304121">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>854.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4093200/figure/f2-ajas-27-5-733-15/" ref="ordinalpos=854&amp;ncbi_uid=6399621&amp;link_uid=PMC4093200" href="/pmc/articles/PMC4093200/"><img src="/pmc/articles/instance/4093200/bin/ajas-27-5-733-15f2.gif" src-large="/pmc/articles/instance/4093200/bin/ajas-27-5-733-15f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4093200/" ref="ordinalpos=854&amp;ncbi_uid=6399621&amp;link_uid=PMC4093200" image-link="/pmc/articles/PMC4093200/figure/f2-ajas-27-5-733-15/" class="imagepopup">Figure 2.  From: Effect of Glucagon-like Peptide 2 on Tight Junction in Jejunal Epithelium of Weaned Pigs though MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effect of GLP-2 on tight junction proteins mRNA expression with MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocked by U0126. GLP-2, glucagon-like peptide 2; MARK, mitogen-activated protein kinase; ZO-1, zonula occludens-1. <sup>a,b,c</sup> Treatments with different letters are different at p&lt;0.05.</div></p></div><div class="aux"><div class="resc">Changsong Yu, et al. Asian-Australas J Anim Sci. 2014 May;27(5):733-742.</div><p class="links"><a class="dblinks" href="/pubmed/25050009" ref="ordinalpos=854&amp;ncbi_uid=6399621&amp;link_uid=25050009">Citation</a><a class="dblinks" href="/pmc/articles/PMC4093200/" ref="ordinalpos=854&amp;ncbi_uid=6399621&amp;link_uid=PMC4093200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>855.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2716976/figure/F1-8108/" ref="ordinalpos=855&amp;ncbi_uid=1990583&amp;link_uid=PMC2716976" href="/pmc/articles/PMC2716976/"><img src="/pmc/articles/instance/2716976/bin/zjh0080981080001.gif" src-large="/pmc/articles/instance/2716976/bin/zjh0080981080001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2716976/" ref="ordinalpos=855&amp;ncbi_uid=1990583&amp;link_uid=PMC2716976" image-link="/pmc/articles/PMC2716976/figure/F1-8108/" class="imagepopup">Figure 1.  From: Activation of mTORC1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in AIDS-Related Lymphomas. </a></p><div class="supp"><p class="details"><div>Activation of the mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ARL, DLBCL type. A section of a lymph node reveals high-grade histology with large atypical cells, atypical mitoses, and brisk apoptotic rate. Immunohistochemical stains for a B-cell marker CD20, mTORC1-dependent phosphorylation of S6rp (at Ser235/236) and 4E-BP1 (at Ser65). All images are at ×400 magnification.</div></p></div><div class="aux"><div class="resc">Mouna El-Salem, et al. Am J Pathol. 2009 August;175(2):817-824.</div><p class="links"><a class="dblinks" href="/pubmed/19608873" ref="ordinalpos=855&amp;ncbi_uid=1990583&amp;link_uid=19608873">Citation</a><a class="dblinks" href="/pmc/articles/PMC2716976/" ref="ordinalpos=855&amp;ncbi_uid=1990583&amp;link_uid=PMC2716976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>856.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3174986/figure/pone-0024963-g004/" ref="ordinalpos=856&amp;ncbi_uid=3235409&amp;link_uid=PMC3174986" href="/pmc/articles/PMC3174986/"><img src="/pmc/articles/instance/3174986/bin/pone.0024963.g004.gif" src-large="/pmc/articles/instance/3174986/bin/pone.0024963.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3174986/" ref="ordinalpos=856&amp;ncbi_uid=3235409&amp;link_uid=PMC3174986" image-link="/pmc/articles/PMC3174986/figure/pone-0024963-g004/" class="imagepopup">Figure 4. Comparison of the gene expression patterns between donors and treatments..  From: Plasmodium falciparum-Infected Erythrocytes and IL-12/IL-18 Induce Diverse Transcriptomes in Human NK Cells: IFN-?/? <span class="highlight" style="background-color:">Pathway</span> versus TREM <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Members of the <i>Interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></i>, the <i>TREM-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></i>, and other genes that play a role in the immune response were selected. The image depicts the log of the fold change of NK genes regulated due to co-culture with 3D7-infected erythrocytes (iRBCs) or with the cytokines IL-12 andIL-18. The expression profile is shown for the three different donors (E, K, and V).</div></p></div><div class="aux"><div class="resc">Elisandra Grangeiro de Carvalho, et al. PLoS One. 2011;6(9):e24963.</div><p class="links"><a class="dblinks" href="/pubmed/21949811" ref="ordinalpos=856&amp;ncbi_uid=3235409&amp;link_uid=21949811">Citation</a><a class="dblinks" href="/pmc/articles/PMC3174986/" ref="ordinalpos=856&amp;ncbi_uid=3235409&amp;link_uid=PMC3174986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>857.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3270028/figure/pone-0030802-g006/" ref="ordinalpos=857&amp;ncbi_uid=3788932&amp;link_uid=PMC3270028" href="/pmc/articles/PMC3270028/"><img src="/pmc/articles/instance/3270028/bin/pone.0030802.g006.gif" src-large="/pmc/articles/instance/3270028/bin/pone.0030802.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3270028/" ref="ordinalpos=857&amp;ncbi_uid=3788932&amp;link_uid=PMC3270028" image-link="/pmc/articles/PMC3270028/figure/pone-0030802-g006/" class="imagepopup">Figure 6. Model depicting the mechanisms used by CoV PLPs to block STING from <span class="highlight" style="background-color:">signaling</span> the activation of the IFN-β induction <span class="highlight" style="background-color:">pathway</span>..  From: Coronavirus Papain-like Proteases Negatively Regulate Antiviral Innate Immune Response through Disruption of STING-Mediated <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Activation of sensors such as RIG-I induces interaction with the <span class="highlight" style="background-color:">signaling</span> complex including MAVS, STING, IRF-3 and TBK-1. Activated MAVS interacts with STING, which dimerizes, leading to the activation of IKK complex, TBK1 and IKKε [8]. The activation of this complex leads to the ubiquitination of RIG-I, STING, IRF-3 and TBK1 and the phosphorylatin of STING and IRF-3. Activated the transcription factor IRF-3 translocates to the nucleus inducing production of IFN. (<b>B</b>) Coronavirus papain-like protease domains (depicted here as PLP) interact with STING to block <span class="highlight" style="background-color:">signaling</span> by blocking assembly or stability of STING dimers and preventing the ubiquitination of <span class="highlight" style="background-color:">signaling</span> proteins, such as RIG-I, TBK1, and IRF-3.</div></p></div><div class="aux"><div class="resc">Li Sun, et al. PLoS One. 2012;7(2):e30802.</div><p class="links"><a class="dblinks" href="/pubmed/22312431" ref="ordinalpos=857&amp;ncbi_uid=3788932&amp;link_uid=22312431">Citation</a><a class="dblinks" href="/pmc/articles/PMC3270028/" ref="ordinalpos=857&amp;ncbi_uid=3788932&amp;link_uid=PMC3270028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>858.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3693466/figure/F1/" ref="ordinalpos=858&amp;ncbi_uid=5029456&amp;link_uid=PMC3693466" href="/pmc/articles/PMC3693466/"><img src="/pmc/articles/instance/3693466/bin/nihms256433f1.gif" src-large="/pmc/articles/instance/3693466/bin/nihms256433f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3693466/" ref="ordinalpos=858&amp;ncbi_uid=5029456&amp;link_uid=PMC3693466" image-link="/pmc/articles/PMC3693466/figure/F1/" class="imagepopup">Figure 1. The insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and inositol phosphates.  From: Insulin <span class="highlight" style="background-color:">Signaling</span>: Inositol Phosphates Get Into the Akt. </a></p><div class="supp"><p class="details"><div>(A) The figure shows the canonical insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to activation of the serine/threonine kinase Akt. Chakraborty et al. (2010) show that insulin also stimulates the inositol phosphate kinase IP6K1 to produce IP7 (5-diphosphoinositolpentakisphosphate) , which in turn inhibits Akt. The authors' results suggest a model for the inhibition of Akt by IP7. In this model, IP7 binding to the PH domain of Akt prevents the tranlsocation of Akt to the membrane, and also prevents the binding of PIP3 (phosphatidylinositol-3,4,5-trisphosphate) to the same domain, thus blocking insulin <span class="highlight" style="background-color:">signaling</span> to Akt. (B) Inositol-derivatives serve as <span class="highlight" style="background-color:">signaling</span> molecules when phosphorylated on distinct hydroxyl groups on the inositol ring. The figure shows the reactions catalyzed by phosphatidylinositol 3-kinase (PI3K) and IP6K1. PI3K phosphorylates the 3 position of PIP2 (phosphatidylinositol-4,5-bisphosphate) to make PIP3. IP6K1 phosphorylates the phosphate group at the 5 position of IP6 (inositol hexakisphosphate) to generate IP7.</div></p></div><div class="aux"><div class="resc">Brendan D. Manning. Cell. ;143(6):861-863.</div><p class="links"><a class="dblinks" href="/pubmed/21145450" ref="ordinalpos=858&amp;ncbi_uid=5029456&amp;link_uid=21145450">Citation</a><a class="dblinks" href="/pmc/articles/PMC3693466/" ref="ordinalpos=858&amp;ncbi_uid=5029456&amp;link_uid=PMC3693466">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>859.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3045689/figure/F6/" ref="ordinalpos=859&amp;ncbi_uid=2811751&amp;link_uid=PMC3045689" href="/pmc/articles/PMC3045689/"><img src="/pmc/articles/instance/3045689/bin/jcbn10-69f06.gif" src-large="/pmc/articles/instance/3045689/bin/jcbn10-69f06.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3045689/" ref="ordinalpos=859&amp;ncbi_uid=2811751&amp;link_uid=PMC3045689" image-link="/pmc/articles/PMC3045689/figure/F6/" class="imagepopup">Fig. 6.  From: Rebamipide suppresses TLR-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in regulating IRF3/7 and IFN-?/? reduction. </a></p><div class="supp"><p class="details"><div>Rebamipide suppressed TLR4-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human colonic epithelial cells. Lipopolysaccharide (TLR4 ligand) was added to CaCo2 colonic epithelial cells with or without rebamipide. LPS alone increased, whereas LPS plus rebamipide suppressed mRNA expression of all of these genes to control levels in CaCo2 cells.</div></p></div><div class="aux"><div class="resc">Naotaka Ogasawara, et al. J Clin Biochem Nutr. 2011 March;48(2):154-160.</div><p class="links"><a class="dblinks" href="/pubmed/21373269" ref="ordinalpos=859&amp;ncbi_uid=2811751&amp;link_uid=21373269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3045689/" ref="ordinalpos=859&amp;ncbi_uid=2811751&amp;link_uid=PMC3045689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>860.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3234905/figure/F7/" ref="ordinalpos=860&amp;ncbi_uid=4437256&amp;link_uid=PMC3234905" href="/pmc/articles/PMC3234905/"><img src="/pmc/articles/instance/3234905/bin/zbc0521189180007.gif" src-large="/pmc/articles/instance/3234905/bin/zbc0521189180007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3234905/" ref="ordinalpos=860&amp;ncbi_uid=4437256&amp;link_uid=PMC3234905" image-link="/pmc/articles/PMC3234905/figure/F7/" class="imagepopup">FIGURE 7. From: Juvenile Hormone Regulates Vitellogenin Gene Expression through Insulin-like Peptide <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in the Red Flour Beetle, Tribolium castaneum. </a></p><div class="supp"><p class="details"><div><b>A model for JH and ILP regulation of Vg synthesis.</b> After adult emergence, JH III titers increase and induce expression of genes coding for ILP2 and ILP3 in the fat body and the brain. The ILPs then work through the insulin-like peptide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to phosphorylate FOXO, resulting in expression of Vg genes. Feeding increases amino acid concentration and turn on production of ILP3 through most likely the TOR <span class="highlight" style="background-color:">pathway</span>. The ILP3 then induces expression of Vg genes through insulin-like peptide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and FOXO. Thus, both JH and feeding function through the insulin-like peptide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and regulate expression of genes coding for Vg. <i>JHRE</i>, JH response element; <i>CA</i>, corpora allata; <i>FHRE</i>, Forkhead response element; <i>SRC</i>, steroid receptor co-activator; <i>TORC1</i>, target of rapamycin complex 1; <i>IPC</i>, insulin-like peptide producing cell.</div></p></div><div class="aux"><div class="resc">Zhentao Sheng, et al. J Biol Chem. 2011 December 9;286(49):41924-41936.</div><p class="links"><a class="dblinks" href="/pubmed/22002054" ref="ordinalpos=860&amp;ncbi_uid=4437256&amp;link_uid=22002054">Citation</a><a class="dblinks" href="/pmc/articles/PMC3234905/" ref="ordinalpos=860&amp;ncbi_uid=4437256&amp;link_uid=PMC3234905">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>861.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3821994/figure/F1/" ref="ordinalpos=861&amp;ncbi_uid=5666414&amp;link_uid=PMC3821994" href="/pmc/articles/PMC3821994/"><img src="/pmc/articles/instance/3821994/bin/nihms510755f1.gif" src-large="/pmc/articles/instance/3821994/bin/nihms510755f1.jpg" alt="FIG 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3821994/" ref="ordinalpos=861&amp;ncbi_uid=5666414&amp;link_uid=PMC3821994" image-link="/pmc/articles/PMC3821994/figure/F1/" class="imagepopup">FIG 1.  From: Targeting the PI3K/AKT/mTOR <span class="highlight" style="background-color:">Pathway</span>: Biomarkers of Success and Tribulation. </a></p><div class="supp"><p class="details"><div>PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing interacting activation and regulatory feedback loops. Adapted from Hoeffer and Klann.<sup>35</sup></div></p></div><div class="aux"><div class="resc">Taofeek K. Owonikoko, et al. Am Soc Clin Oncol Educ Book. 2013: :10.1200/EdBook_AM.2013.33.e395.</div><p class="links"><a class="dblinks" href="/pubmed/23714559" ref="ordinalpos=861&amp;ncbi_uid=5666414&amp;link_uid=23714559">Citation</a><a class="dblinks" href="/pmc/articles/PMC3821994/" ref="ordinalpos=861&amp;ncbi_uid=5666414&amp;link_uid=PMC3821994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>862.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432093/figure/pone-0044107-g011/" ref="ordinalpos=862&amp;ncbi_uid=4216953&amp;link_uid=PMC3432093" href="/pmc/articles/PMC3432093/"><img src="/pmc/articles/instance/3432093/bin/pone.0044107.g011.gif" src-large="/pmc/articles/instance/3432093/bin/pone.0044107.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432093/" ref="ordinalpos=862&amp;ncbi_uid=4216953&amp;link_uid=PMC3432093" image-link="/pmc/articles/PMC3432093/figure/pone-0044107-g011/" class="imagepopup">Figure 11. A schematic diagram showing the interlinks of PI3-K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Resveratrol Inhibits LPS-Induced MAPKs Activation via Activation of the Phosphatidylinositol 3-Kinase <span class="highlight" style="background-color:">Pathway</span> in Murine RAW 264.7 Macrophage Cells. </a></p><div class="aux"><div class="resc">Yi Zong, et al. PLoS One. 2012;7(8):e44107.</div><p class="links"><a class="dblinks" href="/pubmed/22952890" ref="ordinalpos=862&amp;ncbi_uid=4216953&amp;link_uid=22952890">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432093/" ref="ordinalpos=862&amp;ncbi_uid=4216953&amp;link_uid=PMC3432093">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>863.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3467008/figure/F5/" ref="ordinalpos=863&amp;ncbi_uid=5343580&amp;link_uid=PMC3467008" href="/pmc/articles/PMC3467008/"><img src="/pmc/articles/instance/3467008/bin/nihms401232f5.gif" src-large="/pmc/articles/instance/3467008/bin/nihms401232f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3467008/" ref="ordinalpos=863&amp;ncbi_uid=5343580&amp;link_uid=PMC3467008" image-link="/pmc/articles/PMC3467008/figure/F5/" class="imagepopup">Figure 5. A schematic model for SHH expression as an autocrine survival factor during lipotoxic stress.  From: A Hedgehog Survival <span class="highlight" style="background-color:">Pathway</span> in 'Undead' Lipotoxic Hepatocytes. </a></p><div class="supp"><p class="details"><div>Lipotoxic insult-induced AP-1 mediates SHH expression. Blocking JNK <span class="highlight" style="background-color:">signaling</span> with SP600125 represses SHH generation. Inhibition of SHH <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> by either GANT61 or GDC-0449 blocks this autocrine SHH survival <span class="highlight" style="background-color:">pathway</span> in “undead cells” resulting in their cellular demise.</div></p></div><div class="aux"><div class="resc">Keisuke Kakisaka, et al. J Hepatol. 2012 October;57(4):844-851.</div><p class="links"><a class="dblinks" href="/pubmed/22641094" ref="ordinalpos=863&amp;ncbi_uid=5343580&amp;link_uid=22641094">Citation</a><a class="dblinks" href="/pmc/articles/PMC3467008/" ref="ordinalpos=863&amp;ncbi_uid=5343580&amp;link_uid=PMC3467008">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>864.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3594740/figure/F1/" ref="ordinalpos=864&amp;ncbi_uid=4707366&amp;link_uid=PMC3594740" href="/pmc/articles/PMC3594740/"><img src="/pmc/articles/instance/3594740/bin/CPPS-13-776_F1.gif" src-large="/pmc/articles/instance/3594740/bin/CPPS-13-776_F1.jpg" alt="Fig. (1)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3594740/" ref="ordinalpos=864&amp;ncbi_uid=4707366&amp;link_uid=PMC3594740" image-link="/pmc/articles/PMC3594740/figure/F1/" class="imagepopup">Fig. (1).  From: Suppression of TLR <span class="highlight" style="background-color:">Signaling</span> by Targeting TIR domain-Containing Proteins. </a></p><div class="supp"><p class="details"><div>TLR <span class="highlight" style="background-color:">signaling</span>. Schematic representation of MyD88 dependent and TRIF dependent TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> culminating in the production of inflammatory cytokines and type I interferons.</div></p></div><div class="aux"><div class="resc">Ota Fekonja, et al. Curr Protein Pept Sci. 2012 December;13(8):776-788.</div><p class="links"><a class="dblinks" href="/pubmed/23305364" ref="ordinalpos=864&amp;ncbi_uid=4707366&amp;link_uid=23305364">Citation</a><a class="dblinks" href="/pmc/articles/PMC3594740/" ref="ordinalpos=864&amp;ncbi_uid=4707366&amp;link_uid=PMC3594740">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>865.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3562371/figure/F3/" ref="ordinalpos=865&amp;ncbi_uid=5765106&amp;link_uid=PMC3562371" href="/pmc/articles/PMC3562371/"><img src="/pmc/articles/instance/3562371/bin/nihms430260f3.gif" src-large="/pmc/articles/instance/3562371/bin/nihms430260f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3562371/" ref="ordinalpos=865&amp;ncbi_uid=5765106&amp;link_uid=PMC3562371" image-link="/pmc/articles/PMC3562371/figure/F3/" class="imagepopup">Fig. 3.  From: SAMe Treatment Prevents the Ethanol-Induced Epigenetic Alterations of Genes in the Toll-Like Receptor <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>SAMe treatment prevents the ethanol-induced increase in total methylation levels of the genes in the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The average methylation levels of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for each of the four experimental groups was calculated. Error bars represent the Mean ± SEM (n=3).</div></p></div><div class="aux"><div class="resc">Ronik Khachatoorian, et al. Exp Mol Pathol. ;94(1):243-246.</div><p class="links"><a class="dblinks" href="/pubmed/23047067" ref="ordinalpos=865&amp;ncbi_uid=5765106&amp;link_uid=23047067">Citation</a><a class="dblinks" href="/pmc/articles/PMC3562371/" ref="ordinalpos=865&amp;ncbi_uid=5765106&amp;link_uid=PMC3562371">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>866.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2674015/figure/F2/" ref="ordinalpos=866&amp;ncbi_uid=1764046&amp;link_uid=PMC2674015" href="/pmc/articles/PMC2674015/"><img src="/pmc/articles/instance/2674015/bin/nihms40511f2a.gif" src-large="/pmc/articles/instance/2674015/bin/nihms40511f2a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2674015/figure/F2/" ref="ordinalpos=866&amp;ncbi_uid=1764046&amp;link_uid=PMC2674015" href="/pmc/articles/PMC2674015/"><img src="/pmc/articles/instance/2674015/bin/nihms40511f2b.gif" src-large="/pmc/articles/instance/2674015/bin/nihms40511f2b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2674015/" ref="ordinalpos=866&amp;ncbi_uid=1764046&amp;link_uid=PMC2674015" image-link="/pmc/articles/PMC2674015/figure/F2/" class="imagepopup">Figure 2.  From: Toll-like Receptors: Novel Pharmacological Targets for the Treatment of Neurological Diseases. </a></p><div class="supp"><p class="details"><div>Tolerance to the downstream inflammatory effects of TLR4 activation. A. Intracellular <span class="highlight" style="background-color:">signaling</span> events downstream of initial TLR4 activation. Both the MyD88 dependent and MyD88 independent pathways are active. B. <span class="highlight" style="background-color:">Signaling</span> events upon subsequent TLR4 activation. <span class="highlight" style="background-color:">Signaling</span> through the MyD88 dependent <span class="highlight" style="background-color:">pathway</span> is inhibited and <span class="highlight" style="background-color:">signaling</span> through the MyD88 independent <span class="highlight" style="background-color:">pathway</span> is enhanced.</div></p></div><div class="aux"><div class="resc">Brenda J. Marsh, et al. Curr Opin Pharmacol. ;8(1):8-13.</div><p class="links"><a class="dblinks" href="/pubmed/17974478" ref="ordinalpos=866&amp;ncbi_uid=1764046&amp;link_uid=17974478">Citation</a><a class="dblinks" href="/pmc/articles/PMC2674015/" ref="ordinalpos=866&amp;ncbi_uid=1764046&amp;link_uid=PMC2674015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>867.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2963142/figure/fig1/" ref="ordinalpos=867&amp;ncbi_uid=2534455&amp;link_uid=PMC2963142" href="/pmc/articles/PMC2963142/"><img src="/pmc/articles/instance/2963142/bin/MI2010-781235.001.gif" src-large="/pmc/articles/instance/2963142/bin/MI2010-781235.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2963142/" ref="ordinalpos=867&amp;ncbi_uid=2534455&amp;link_uid=PMC2963142" image-link="/pmc/articles/PMC2963142/figure/fig1/" class="imagepopup">Figure 1.  From: Toll-Like Receptor <span class="highlight" style="background-color:">Signaling</span> Pathways--Therapeutic Opportunities. </a></p><div class="supp"><p class="details"><div>The TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and downstream effector molecules. Depicted are key TLR molecules, their <span class="highlight" style="background-color:">signaling</span> adaptors and downstream mediators that are essential for TLR <span class="highlight" style="background-color:">signaling</span> and function. The specific molecules in this network that are presently being interrogated as potential therapeutic targets include TLR7/9, MyD88 and IRAK1/4, as discussed in the text.</div></p></div><div class="aux"><div class="resc">Jiankun Zhu, et al. Mediators Inflamm. 2010;2010:781235.</div><p class="links"><a class="dblinks" href="/pubmed/20981241" ref="ordinalpos=867&amp;ncbi_uid=2534455&amp;link_uid=20981241">Citation</a><a class="dblinks" href="/pmc/articles/PMC2963142/" ref="ordinalpos=867&amp;ncbi_uid=2534455&amp;link_uid=PMC2963142">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>868.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3160317/figure/pone-0023856-g006/" ref="ordinalpos=868&amp;ncbi_uid=3186411&amp;link_uid=PMC3160317" href="/pmc/articles/PMC3160317/"><img src="/pmc/articles/instance/3160317/bin/pone.0023856.g006.gif" src-large="/pmc/articles/instance/3160317/bin/pone.0023856.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3160317/" ref="ordinalpos=868&amp;ncbi_uid=3186411&amp;link_uid=PMC3160317" image-link="/pmc/articles/PMC3160317/figure/pone-0023856-g006/" class="imagepopup">Figure 6. Changes in expression of genes in the IFN and JAK-STAT <span class="highlight" style="background-color:">signaling</span> pathways under HCV infection and/or IFN administration..  From: Hepatitis C Virus Infection Suppresses the Interferon Response in the Liver of the Human Hepatocyte Chimeric Mouse. </a></p><div class="supp"><p class="details"><div>A) An overview of the IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> consisting of 26 representative genes is shown. Genes illustrated as gray shapes were not included in this study. B) Relative expression levels of genes with/without HCV infection and/or IFN administration were plotted (closed dots: with HCV infection; open dots: without HCV infection) using microarray data. The slopes of the dashed and solid lines represent IFN responsiveness with and without HCV infection, respectively. In 21 of the 22 examined genes in the IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, signal intensities increased and IFN responsiveness was repressed following HCV infection. Student's <i>t</i>-test was used for statistical analysis. C) An overview of the JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> consisting of 22 representative gene products is shown. Genes illustrated as gray shapes were not included in this study. D) Relative expression levels of genes with/without HCV infection and/or IFN administration were plotted using microarray data (closed dots: with HCV infection; open dots: without HCV infection). Signal intensities increased following HCV infection in 16 of 22 genes in the JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and IFN response was suppressed in 9 genes. Statistical analysis was performed using Student's <i>t</i>-test.</div></p></div><div class="aux"><div class="resc">Masataka Tsuge, et al. PLoS One. 2011;6(8):e23856.</div><p class="links"><a class="dblinks" href="/pubmed/21886832" ref="ordinalpos=868&amp;ncbi_uid=3186411&amp;link_uid=21886832">Citation</a><a class="dblinks" href="/pmc/articles/PMC3160317/" ref="ordinalpos=868&amp;ncbi_uid=3186411&amp;link_uid=PMC3160317">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>869.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3576034/figure/F6/" ref="ordinalpos=869&amp;ncbi_uid=5756742&amp;link_uid=PMC3576034" href="/pmc/articles/PMC3576034/"><img src="/pmc/articles/instance/3576034/bin/nihms425095f6.gif" src-large="/pmc/articles/instance/3576034/bin/nihms425095f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3576034/" ref="ordinalpos=869&amp;ncbi_uid=5756742&amp;link_uid=PMC3576034" image-link="/pmc/articles/PMC3576034/figure/F6/" class="imagepopup">Fig. 6.  From: Deciphering the Mechanisms of Tumorigenesis in Human Pancreatic Ductal Epithelial Cells. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of microarray data and the proposed model for transformation of the HPDE cell line. <i>A</i>, IPA canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis for transformation. Bar indicates significance. The length of the bar indicates only that the differentially expressed proteins are related to this <span class="highlight" style="background-color:">pathway</span> but does not indicate upregulation or downregulation of the <span class="highlight" style="background-color:">pathway</span>. <i>B</i>, IPA biological function analysis for transformation. <i>C</i>, Proposed model for transformation of the HPDE cell line.</div></p></div><div class="aux"><div class="resc">Zhe Chang, et al. Clin Cancer Res. ;19(3):549-559.</div><p class="links"><a class="dblinks" href="/pubmed/23340292" ref="ordinalpos=869&amp;ncbi_uid=5756742&amp;link_uid=23340292">Citation</a><a class="dblinks" href="/pmc/articles/PMC3576034/" ref="ordinalpos=869&amp;ncbi_uid=5756742&amp;link_uid=PMC3576034">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>870.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3606047/figure/F1/" ref="ordinalpos=870&amp;ncbi_uid=5764185&amp;link_uid=PMC3606047" href="/pmc/articles/PMC3606047/"><img src="/pmc/articles/instance/3606047/bin/nihms438583f1.gif" src-large="/pmc/articles/instance/3606047/bin/nihms438583f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3606047/" ref="ordinalpos=870&amp;ncbi_uid=5764185&amp;link_uid=PMC3606047" image-link="/pmc/articles/PMC3606047/figure/F1/" class="imagepopup">Figure 1. Adenosine <span class="highlight" style="background-color:">Signaling</span> in Acute and Chronic Disease.  From: Adenosine <span class="highlight" style="background-color:">Signaling</span> During Acute and Chronic Disease States. </a></p><div class="supp"><p class="details"><div>Cellular injury associated with hypoxia and inflammation promote the release of ATP that is subsequently dephosphorylated by membrane bound CD39 and CD73. Extracellular adenosine then stimulates cell surface adenosine receptors (ADORA1, ADORA2A, ADORA2B, ADORA3) to influence tissue responses to injury. In acute disease states, adenosine largely contributes to anti-inflammatory and tissue protective responses such as the promotion of vascular barrier function. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> also promotes wound healing. These adenosine responses are largely regulated by ADORA2A and ADORA2B <span class="highlight" style="background-color:">signaling</span> pathways. In chronic disease states, adenosine <span class="highlight" style="background-color:">signaling</span> can promote cellular processes such as fibrosis that contribute to disease progression. Depicted is fibrosis in the liver, lung and heart; however, findings suggest excessive adenosine <span class="highlight" style="background-color:">signaling</span> also contributes to fibrosis in the skin, kidney and penis. These responses are regulated by ADORA2A and ADORA2B <span class="highlight" style="background-color:">signaling</span> pathways. In addition, ADORA2B <span class="highlight" style="background-color:">signaling</span> promotes sickling of erythrocytes that can contribute to the progression of tissue injury, and promotes insulin resistance that can impact the development of diabetes.</div></p></div><div class="aux"><div class="resc">Harry Karmouty-Quintana, et al. J Mol Med (Berl). 2013 February;91(2):173-181.</div><p class="links"><a class="dblinks" href="/pubmed/23340998" ref="ordinalpos=870&amp;ncbi_uid=5764185&amp;link_uid=23340998">Citation</a><a class="dblinks" href="/pmc/articles/PMC3606047/" ref="ordinalpos=870&amp;ncbi_uid=5764185&amp;link_uid=PMC3606047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>871.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3712008/figure/F2/" ref="ordinalpos=871&amp;ncbi_uid=5100884&amp;link_uid=PMC3712008" href="/pmc/articles/PMC3712008/"><img src="/pmc/articles/instance/3712008/bin/1475-2867-13-66-2.gif" src-large="/pmc/articles/instance/3712008/bin/1475-2867-13-66-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3712008/" ref="ordinalpos=871&amp;ncbi_uid=5100884&amp;link_uid=PMC3712008" image-link="/pmc/articles/PMC3712008/figure/F2/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> switch in breast cancer. </a></p><div class="supp"><p class="details"><div><b>Subnetwork representation of protein interactions based on 78 mutated genes in 602 breast cancers </b>[1]<b>-</b>[3]<b>.</b> Gene networks were inferred using the Reactome FI Cytoscape Plugin [7]. A total of 182 genes with more than 7 mutations in 602 breast cancer samples were used. Among them, 78 (43%) were mapped in the subnetwork and subsequently clustered into 12 modules, six of which were greater than 4. Finally, modules functions were assessed with <span class="highlight" style="background-color:">pathway</span> enrichment analysis (FDR &lt; 0.05). Three main pathways can be recognized: the P53 <span class="highlight" style="background-color:">pathway</span> and DNA repair (14 genes/18%) in blue, the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (13 genes/18%) in pink and the MAPK <span class="highlight" style="background-color:">pathway</span> (8 genes/10%) in green.</div></p></div><div class="aux"><div class="resc">Arnaud Guille, et al. Cancer Cell Int. 2013;13:66-66.</div><p class="links"><a class="dblinks" href="/pubmed/23805779" ref="ordinalpos=871&amp;ncbi_uid=5100884&amp;link_uid=23805779">Citation</a><a class="dblinks" href="/pmc/articles/PMC3712008/" ref="ordinalpos=871&amp;ncbi_uid=5100884&amp;link_uid=PMC3712008">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>872.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4405775/figure/F1/" ref="ordinalpos=872&amp;ncbi_uid=7448428&amp;link_uid=PMC4405775" href="/pmc/articles/PMC4405775/"><img src="/pmc/articles/instance/4405775/bin/nihms381324f1.gif" src-large="/pmc/articles/instance/4405775/bin/nihms381324f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4405775/" ref="ordinalpos=872&amp;ncbi_uid=7448428&amp;link_uid=PMC4405775" image-link="/pmc/articles/PMC4405775/figure/F1/" class="imagepopup">Figure 1.  From: Targeting NADPH Oxidases for the Treatment of Cancer and Inflammation. </a></p><div class="supp"><p class="details"><div>Dependence on <span class="highlight" style="background-color:">signaling</span> pathways based upon tumor suppressor status. Tumors that are deficient in p16ink4a are more likely to use a reactive oxygen/rac/NFkB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, while tumors that have mutant p53 are more likely to use a noncanonical NFkB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and phosphoinositol-3 kinase dependent but Akt independent <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Michael Y. Bonner, et al. Cell Mol Life Sci. ;69(14):2435-2442.</div><p class="links"><a class="dblinks" href="/pubmed/22581366" ref="ordinalpos=872&amp;ncbi_uid=7448428&amp;link_uid=22581366">Citation</a><a class="dblinks" href="/pmc/articles/PMC4405775/" ref="ordinalpos=872&amp;ncbi_uid=7448428&amp;link_uid=PMC4405775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>873.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3938851/figure/Fig2/" ref="ordinalpos=873&amp;ncbi_uid=5893177&amp;link_uid=PMC3938851" href="/pmc/articles/PMC3938851/"><img src="/pmc/articles/instance/3973063/bin/13238_2013_2_Fig2_HTML.gif" src-large="/pmc/articles/instance/3973063/bin/13238_2013_2_Fig2_HTML.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3938851/" ref="ordinalpos=873&amp;ncbi_uid=5893177&amp;link_uid=PMC3938851" image-link="/pmc/articles/PMC3938851/figure/Fig2/" class="imagepopup">Figure 2.  From: Targeting tissue-specific metabolic <span class="highlight" style="background-color:">signaling</span> pathways in aging: the promise and limitations. </a></p><div class="supp"><p class="details"><div>Manipulation of the mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can promote lifespan extension and healthy aging. As a central energy sensor, the mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is regulated by various upstream signals such as nutrients, growth factor <span class="highlight" style="background-color:">signaling</span>, and caloric restriction (CR), leading to altered Endoplasmic reticulum (ER) and mitochondrial function and autophagic activity, which play critical roles in aging</div></p></div><div class="aux"><div class="resc">Fang Hu, et al. Protein Cell. 2014;5(1):21-35.</div><p class="links"><a class="dblinks" href="/pubmed/24474199" ref="ordinalpos=873&amp;ncbi_uid=5893177&amp;link_uid=24474199">Citation</a><a class="dblinks" href="/pmc/articles/PMC3938851/" ref="ordinalpos=873&amp;ncbi_uid=5893177&amp;link_uid=PMC3938851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>874.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4141640/figure/f3/" ref="ordinalpos=874&amp;ncbi_uid=6540552&amp;link_uid=PMC4141640" href="/pmc/articles/PMC4141640/"><img src="/pmc/articles/instance/4141640/bin/f1000research-3-4838-g0002.gif" src-large="/pmc/articles/instance/4141640/bin/f1000research-3-4838-g0002.jpg" alt="Figure 3. " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4141640/" ref="ordinalpos=874&amp;ncbi_uid=6540552&amp;link_uid=PMC4141640" image-link="/pmc/articles/PMC4141640/figure/f3/" class="imagepopup">Figure 3. Comparison of the original diagram and Cytoscape visualization for the human MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (KEGG hsa04010)..  From: KEGGscape: a Cytoscape app for <span class="highlight" style="background-color:">pathway</span> data integration. </a></p><div class="aux"><div class="resc">Kozo Nishida, et al. F1000Res. 2014;3:144.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=874&amp;ncbi_uid=6540552&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4141640/" ref="ordinalpos=874&amp;ncbi_uid=6540552&amp;link_uid=PMC4141640">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>875.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3199525/figure/F6/" ref="ordinalpos=875&amp;ncbi_uid=3433538&amp;link_uid=PMC3199525" href="/pmc/articles/PMC3199525/"><img src="/pmc/articles/instance/3199525/bin/zbc0491184050006.gif" src-large="/pmc/articles/instance/3199525/bin/zbc0491184050006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3199525/" ref="ordinalpos=875&amp;ncbi_uid=3433538&amp;link_uid=PMC3199525" image-link="/pmc/articles/PMC3199525/figure/F6/" class="imagepopup">FIGURE 6. From: The Regulator of Calcineurin 1 (RCAN1/DSCR1) Activates the cAMP Response Element-binding Protein (CREB) <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A diagram depicting the participation of RCAN1 in the CREB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown.</b></div></p></div><div class="aux"><div class="resc">Seon Sook Kim, et al. J Biol Chem. 2011 October 28;286(43):37841-37848.</div><p class="links"><a class="dblinks" href="/pubmed/21890628" ref="ordinalpos=875&amp;ncbi_uid=3433538&amp;link_uid=21890628">Citation</a><a class="dblinks" href="/pmc/articles/PMC3199525/" ref="ordinalpos=875&amp;ncbi_uid=3433538&amp;link_uid=PMC3199525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>876.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC14940/figure/F10/" ref="ordinalpos=876&amp;ncbi_uid=9619&amp;link_uid=PMC14940" href="/pmc/articles/PMC14940/"><img src="/pmc/articles/instance/14940/bin/mk0801266010.gif" src-large="/pmc/articles/instance/14940/bin/mk0801266010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC14940/" ref="ordinalpos=876&amp;ncbi_uid=9619&amp;link_uid=PMC14940" image-link="/pmc/articles/PMC14940/figure/F10/" class="imagepopup">Figure 10.  From: Involvement of an SHP-2-Rho Small G Protein <span class="highlight" style="background-color:">Pathway</span> in Hepatocyte  Growth Factor/Scatter  Factor-induced Cell Scattering. </a></p><div class="supp"><p class="details"><div>A <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of c-Met for cell  scattering.</div></p></div><div class="aux"><div class="resc">Atsuko Kodama, et al. Mol Biol Cell. 2000 August;11(8):2565-2575.</div><p class="links"><a class="dblinks" href="/pubmed/10930454" ref="ordinalpos=876&amp;ncbi_uid=9619&amp;link_uid=10930454">Citation</a><a class="dblinks" href="/pmc/articles/PMC14940/" ref="ordinalpos=876&amp;ncbi_uid=9619&amp;link_uid=PMC14940">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>877.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711356/figure/f04_01/" ref="ordinalpos=877&amp;ncbi_uid=5105612&amp;link_uid=PMC3711356" href="/pmc/articles/PMC3711356/"><img src="/pmc/articles/instance/3711356/bin/f04_01.gif" src-large="/pmc/articles/instance/3711356/bin/f04_01.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711356/" ref="ordinalpos=877&amp;ncbi_uid=5105612&amp;link_uid=PMC3711356" image-link="/pmc/articles/PMC3711356/figure/f04_01/" class="imagepopup">Figure 4. From: The UVR8 UV-B Photoreceptor: Perception, <span class="highlight" style="background-color:">Signaling</span> and Response. </a></p><div class="supp"><p class="details"><div> The UVR8 photocycle.<br />The UVR8 homodimer is monomerised by UV-B, with UV-B absorption proceeding via a tryptophan-based chromophore. The UVR8 monomer interacts directly with COP1 to initiate UV-B <span class="highlight" style="background-color:">signaling</span>. UVR8 monomer is redimerized through the action of RUP1 and RUP2, which disrupts the UVR8-COP1 interaction, inactivates the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and regenerates the UVR8 homodimer again ready for UV-B perception.</div></p></div><div class="aux"><div class="resc">Kimberley Tilbrook, et al. Arabidopsis Book. 2013;11:e0164.</div><p class="links"><a class="dblinks" href="/pubmed/23864838" ref="ordinalpos=877&amp;ncbi_uid=5105612&amp;link_uid=23864838">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711356/" ref="ordinalpos=877&amp;ncbi_uid=5105612&amp;link_uid=PMC3711356">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>878.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4032327/figure/pone-0098208-g005/" ref="ordinalpos=878&amp;ncbi_uid=6177363&amp;link_uid=PMC4032327" href="/pmc/articles/PMC4032327/"><img src="/pmc/articles/instance/4032327/bin/pone.0098208.g005.gif" src-large="/pmc/articles/instance/4032327/bin/pone.0098208.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4032327/" ref="ordinalpos=878&amp;ncbi_uid=6177363&amp;link_uid=PMC4032327" image-link="/pmc/articles/PMC4032327/figure/pone-0098208-g005/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Pathway</span> analysis of differentially expressed proteins..  From: Integrated Proteomics Identified Up-Regulated Focal Adhesion-Mediated Proteins in Human Squamous Cell Carcinoma in an Orthotopic Murine Model. </a></p><div class="supp"><p class="details"><div>(a) The highest score network generated by IPA comprised 23 differentially expressed proteins (up-regulated proteins are displayed in red, whereas the down-regulated <b>proteins</b> are in green) plus additional interacting molecules that were not identified in this study (white). The network revealed 15 proteins in the context of cancer. Actin cytoskeleton <span class="highlight" style="background-color:">signaling</span>, integrin <span class="highlight" style="background-color:">signaling</span> and FAK <span class="highlight" style="background-color:">signaling</span> represent canonical pathways with 13, 10 and 6 identified proteins of the network, respectively. The proteins were grouped according to the canonical <span class="highlight" style="background-color:">pathway</span> or function. Fx: function and Cp: canonical <span class="highlight" style="background-color:">pathway</span>. (b) Representation of the top canonical <span class="highlight" style="background-color:">pathway</span> of actin cytoskeleton <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Daniela C. Granato, et al. PLoS One. 2014;9(5):e98208.</div><p class="links"><a class="dblinks" href="/pubmed/24858105" ref="ordinalpos=878&amp;ncbi_uid=6177363&amp;link_uid=24858105">Citation</a><a class="dblinks" href="/pmc/articles/PMC4032327/" ref="ordinalpos=878&amp;ncbi_uid=6177363&amp;link_uid=PMC4032327">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>879.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2685460/figure/F7/" ref="ordinalpos=879&amp;ncbi_uid=1821717&amp;link_uid=PMC2685460" href="/pmc/articles/PMC2685460/"><img src="/pmc/articles/instance/2685460/bin/nihms103814f7.gif" src-large="/pmc/articles/instance/2685460/bin/nihms103814f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2685460/" ref="ordinalpos=879&amp;ncbi_uid=1821717&amp;link_uid=PMC2685460" image-link="/pmc/articles/PMC2685460/figure/F7/" class="imagepopup">Figure 7. P. gingivalis-activated TLR2 induces distinct proadhesive and proinflammatory <span class="highlight" style="background-color:">signaling</span> pathways.  From: Induction of Distinct TLR2-Mediated Proinflammatory and Proadhesive <span class="highlight" style="background-color:">Signaling</span> Pathways in Response to Porphyromonas gingivalis Fimbriae . </a></p><div class="supp"><p class="details"><div>The model is based on the current findings and our previously published studies (6, 19–21). <i>P. gingivalis</i> fimbriae interact with the CD14/TLR2 complex leading to TLR2 activation. Activated TLR2 induces TIRAP/MyD88-dependent <span class="highlight" style="background-color:">signaling</span> for NF-κB activation and induction of proinflammatory cytokines (proinflammatory <span class="highlight" style="background-color:">pathway</span>), and TIRAP/MyD88-independent <span class="highlight" style="background-color:">signaling</span> for transactivation of the CR3 adhesive activity (proadhesive <span class="highlight" style="background-color:">pathway</span>). The latter <span class="highlight" style="background-color:">pathway</span> proceeds via Rac1, PI3K and cytohesin-1 (Cyt-1). Cytohesin-1 is likely to function immediately downstream of PI3K, since it was previously shown to use PI3K-generated PIP3 for membrane docking and interaction with the CD18 cytoplasmic tail (48).</div></p></div><div class="aux"><div class="resc">George Hajishengallis, et al. J Immunol. ;182(11):6690-6696.</div><p class="links"><a class="dblinks" href="/pubmed/19454663" ref="ordinalpos=879&amp;ncbi_uid=1821717&amp;link_uid=19454663">Citation</a><a class="dblinks" href="/pmc/articles/PMC2685460/" ref="ordinalpos=879&amp;ncbi_uid=1821717&amp;link_uid=PMC2685460">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>880.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3538329/figure/F1/" ref="ordinalpos=880&amp;ncbi_uid=5661529&amp;link_uid=PMC3538329" href="/pmc/articles/PMC3538329/"><img src="/pmc/articles/instance/3538329/bin/zh89991200900001.gif" src-large="/pmc/articles/instance/3538329/bin/zh89991200900001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3538329/" ref="ordinalpos=880&amp;ncbi_uid=5661529&amp;link_uid=PMC3538329" image-link="/pmc/articles/PMC3538329/figure/F1/" class="imagepopup">Figure 1.  From: Canonical Wnt <span class="highlight" style="background-color:">signaling</span> in megakaryocytes regulates proplatelet formation. </a></p><div class="supp"><p class="details"><div><b>Expression of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components in the megakaryocyte.</b> (A) Overview of the canonical and noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> pathways. All components shown in green were detected at the transcript level in our previous microarray studies of in vitro differentiated MKs.<sup>21</sup> (B) Expression of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components in in vitro differentiated MKs. Data are presented as the mean and range of the expression relative to the <i>HPRT</i> gene from 2 biologic replicates. (C) Heatmap displaying MK-specific components of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The intensity of the shading reflects the level of expression of each gene in each cell type. Numbers shown indicate the normalized expression value of each gene as determined in Watkins et al.<sup>21</sup></div></p></div><div class="aux"><div class="resc">Iain C. Macaulay, et al. Blood. 2013 January 3;121(1):188-196.</div><p class="links"><a class="dblinks" href="/pubmed/23160460" ref="ordinalpos=880&amp;ncbi_uid=5661529&amp;link_uid=23160460">Citation</a><a class="dblinks" href="/pmc/articles/PMC3538329/" ref="ordinalpos=880&amp;ncbi_uid=5661529&amp;link_uid=PMC3538329">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>881.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4314211/figure/F4/" ref="ordinalpos=881&amp;ncbi_uid=7119452&amp;link_uid=PMC4314211" href="/pmc/articles/PMC4314211/"><img src="/pmc/articles/instance/4314211/bin/nihms141014f4.gif" src-large="/pmc/articles/instance/4314211/bin/nihms141014f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4314211/" ref="ordinalpos=881&amp;ncbi_uid=7119452&amp;link_uid=PMC4314211" image-link="/pmc/articles/PMC4314211/figure/F4/" class="imagepopup">Figure 4. The GG does not express pGC-D.  From: The Grueneberg Ganglion olfactory subsystem employs a cGMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>A) Pde2a immunolabeling in thick sections from the olfactory neuroepithelium of a transgenic GCD-iTG mouse. GFP-positive GC-D neurons (green) express Pde2a (magenta), consistent with their use of a cGMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. B) Pde2a-positive GG cells (magenta) in thick sections from the nasal vestibule are devoid of GFP (green) and, hence, pGC-D. Scale bars: 60 μm.</div></p></div><div class="aux"><div class="resc">Cambrian Y. Liu, et al. J Comp Neurol. ;516(1):36-48.</div><p class="links"><a class="dblinks" href="/pubmed/19565523" ref="ordinalpos=881&amp;ncbi_uid=7119452&amp;link_uid=19565523">Citation</a><a class="dblinks" href="/pmc/articles/PMC4314211/" ref="ordinalpos=881&amp;ncbi_uid=7119452&amp;link_uid=PMC4314211">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>882.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4247569/figure/fig10/" ref="ordinalpos=882&amp;ncbi_uid=6895462&amp;link_uid=PMC4247569" href="/pmc/articles/PMC4247569/"><img src="/pmc/articles/instance/4247569/bin/PC_129296_f10.gif" src-large="/pmc/articles/instance/4247569/bin/PC_129296_f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4247569/" ref="ordinalpos=882&amp;ncbi_uid=6895462&amp;link_uid=PMC4247569" image-link="/pmc/articles/PMC4247569/figure/fig10/" class="imagepopup">Figure 10. From: The Arabidopsis Ethylene/Jasmonic Acid-NRT <span class="highlight" style="background-color:">Signaling</span> Module Coordinates Nitrate Reallocation and the Trade-Off between Growth and Environmental Adaptation. </a></p><div class="supp"><p class="details"><div>A Simplified Model for the Interaction between ET/JA <span class="highlight" style="background-color:">Signaling</span> and Nitrate Reallocation to Mediate Plant Adaptation to the Environment.<br />Stresses such as cadmium or salt induce the production of ET and JA, which further activates the ET and JA <span class="highlight" style="background-color:">signaling</span> pathways, respectively. EIN3/EIL1 acts to converge signals transduced by ET and JA cascades, and further transduces the signal down to ERFs, which induce <i>NRT1.8</i> expression by binding to its promoter, or EIN3/EIL1 decodes signals via the ET <span class="highlight" style="background-color:">pathway</span> by binding to the promoter of <i>NRT1.5</i> and suppresses its expression. The JA <span class="highlight" style="background-color:">pathway</span> also participates in the downregulation of <i>NRT1.5</i> via COI1. The coordinated upregulation of <i>NRT1.8</i> and downregulation of <i>NRT1.5</i> fine-tunes nitrate reallocation to roots, which modulates the balance between stress tolerance and plant growth. Blue lines display the route for signals going through the ET <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, while red lines indicate those through the JA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Dashed lines indicate steps not shown or possible unidentified components. Gray dotted lines and question marks indicate alternative pathways/components that contribute, but to a much lesser extent, in the regulation of <i>NRT1.5</i> and <i>NRT1.8</i>.</div></p></div><div class="aux"><div class="resc">Guo-Bin Zhang, et al. Plant Cell. 2014 October;26(10):3984-3998.</div><p class="links"><a class="dblinks" href="/pubmed/25326291" ref="ordinalpos=882&amp;ncbi_uid=6895462&amp;link_uid=25326291">Citation</a><a class="dblinks" href="/pmc/articles/PMC4247569/" ref="ordinalpos=882&amp;ncbi_uid=6895462&amp;link_uid=PMC4247569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>883.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3418847/figure/F2/" ref="ordinalpos=883&amp;ncbi_uid=4845664&amp;link_uid=PMC3418847" href="/pmc/articles/PMC3418847/"><img src="/pmc/articles/instance/3418847/bin/zjw0061241440002.gif" src-large="/pmc/articles/instance/3418847/bin/zjw0061241440002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3418847/" ref="ordinalpos=883&amp;ncbi_uid=4845664&amp;link_uid=PMC3418847" image-link="/pmc/articles/PMC3418847/figure/F2/" class="imagepopup">Fig. 2. From: PTMScan Direct: Identification and Quantification of Peptides from Critical <span class="highlight" style="background-color:">Signaling</span> Proteins by Immunoaffinity Enrichment Coupled with LC-MS/MS. </a></p><div class="supp"><p class="details"><div><b>PTMScan Direct: Multipathway targets.</b> Examples of <span class="highlight" style="background-color:">signaling</span> pathways targeted by the PTMScan Direct: Multipathway reagent are shown at the <i>top</i>. Two pathways, MAP kinase <span class="highlight" style="background-color:">signaling</span> and AKT/PI3K <span class="highlight" style="background-color:">pathway</span>, are shown in detail as examples of <span class="highlight" style="background-color:">pathway</span> coverage by the reagent. The reagent targets proteins highlighted in <i>purple</i>. A list of phosphorylation sites from the targeted proteins is provided in supplemental Table S1.</div></p></div><div class="aux"><div class="resc">Matthew P. Stokes, et al. Mol Cell Proteomics. 2012 May;11(5):187-201.</div><p class="links"><a class="dblinks" href="/pubmed/22322096" ref="ordinalpos=883&amp;ncbi_uid=4845664&amp;link_uid=22322096">Citation</a><a class="dblinks" href="/pmc/articles/PMC3418847/" ref="ordinalpos=883&amp;ncbi_uid=4845664&amp;link_uid=PMC3418847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>884.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2804301/figure/F2/" ref="ordinalpos=884&amp;ncbi_uid=2710851&amp;link_uid=PMC2804301" href="/pmc/articles/PMC2804301/"><img src="/pmc/articles/instance/2804301/bin/btp640f2.gif" src-large="/pmc/articles/instance/2804301/bin/btp640f2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2804301/" ref="ordinalpos=884&amp;ncbi_uid=2710851&amp;link_uid=PMC2804301" image-link="/pmc/articles/PMC2804301/figure/F2/" class="imagepopup">Fig. 2. From: <span class="highlight" style="background-color:">Pathway</span> analysis using random forests with bivariate node-split for survival outcomes. </a></p><div class="supp"><p class="details"><div>A multidimensional scaling plot for TGF beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Herbert Pang, et al. Bioinformatics. 2010 January 15;26(2):250-258.</div><p class="links"><a class="dblinks" href="/pubmed/19933158" ref="ordinalpos=884&amp;ncbi_uid=2710851&amp;link_uid=19933158">Citation</a><a class="dblinks" href="/pmc/articles/PMC2804301/" ref="ordinalpos=884&amp;ncbi_uid=2710851&amp;link_uid=PMC2804301">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>885.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041713/figure/F4/" ref="ordinalpos=885&amp;ncbi_uid=6186547&amp;link_uid=PMC4041713" href="/pmc/articles/PMC4041713/"><img src="/pmc/articles/instance/4041713/bin/nihms589041f4.gif" src-large="/pmc/articles/instance/4041713/bin/nihms589041f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041713/" ref="ordinalpos=885&amp;ncbi_uid=6186547&amp;link_uid=PMC4041713" image-link="/pmc/articles/PMC4041713/figure/F4/" class="imagepopup">Figure 4. DE-miRNAs identified in PBMCs from sarcoidosis patients are predicted to target the TGFβ/WNT molecular <span class="highlight" style="background-color:">pathway</span>.  From: Differential Expression of MicroRNA and Predicted Targets in Pulmonary Sarcoidosis. </a></p><div class="supp"><p class="details"><div>DIANA-mirPath was used to predict the most likely molecular pathways targeted by PCR-validated DE-miRNAs in sarcoidosis PBMCs and identified the TGFβ/WNT <span class="highlight" style="background-color:">pathway</span> as the most likely candidate. The arrows represent activation, whereas the hammerheads represent inhibition. The molecular components of the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and their putative interactions were derived from the KEGG <span class="highlight" style="background-color:">pathway</span> database.</div></p></div><div class="aux"><div class="resc">Elliott D. Crouser, et al. Biochem Biophys Res Commun. ;417(2):886-891.</div><p class="links"><a class="dblinks" href="/pubmed/22209793" ref="ordinalpos=885&amp;ncbi_uid=6186547&amp;link_uid=22209793">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041713/" ref="ordinalpos=885&amp;ncbi_uid=6186547&amp;link_uid=PMC4041713">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>886.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3278804/figure/F4/" ref="ordinalpos=886&amp;ncbi_uid=4049119&amp;link_uid=PMC3278804" href="/pmc/articles/PMC3278804/"><img src="/pmc/articles/instance/3278804/bin/nihms347947f4.gif" src-large="/pmc/articles/instance/3278804/bin/nihms347947f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3278804/" ref="ordinalpos=886&amp;ncbi_uid=4049119&amp;link_uid=PMC3278804" image-link="/pmc/articles/PMC3278804/figure/F4/" class="imagepopup">Figure 4. Hypoxia alters JAK/STAT <span class="highlight" style="background-color:">signaling</span> in the white matter.  From: Chronic perinatal hypoxia reduces GLAST function in astrocytes through the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b><i>A.</i></b> Western blot analysis of components of the JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the white matter of hypoxic and normoxic mice at P5, P11, P18 and P45. <b><i>B.</i></b> Protein levels are expressed relative to actin loading control. Densitometric analysis is shown ± st. dev. (n=4–6 animals, *p&lt;0.05, unpaired Student’s t-test).</div></p></div><div class="aux"><div class="resc">Matthew Raymond, et al. J Neurosci. ;31(49):17864-17871.</div><p class="links"><a class="dblinks" href="/pubmed/22159101" ref="ordinalpos=886&amp;ncbi_uid=4049119&amp;link_uid=22159101">Citation</a><a class="dblinks" href="/pmc/articles/PMC3278804/" ref="ordinalpos=886&amp;ncbi_uid=4049119&amp;link_uid=PMC3278804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>887.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3852805/figure/fig2/" ref="ordinalpos=887&amp;ncbi_uid=5602092&amp;link_uid=PMC3852805" href="/pmc/articles/PMC3852805/"><img src="/pmc/articles/instance/3852805/bin/BMRI2013-740573.002.gif" src-large="/pmc/articles/instance/3852805/bin/BMRI2013-740573.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3852805/" ref="ordinalpos=887&amp;ncbi_uid=5602092&amp;link_uid=PMC3852805" image-link="/pmc/articles/PMC3852805/figure/fig2/" class="imagepopup">Figure 2.  From: Prooxidant Mechanisms in Iron Overload Cardiomyopathy. </a></p><div class="supp"><p class="details"><div>Adiponectin ameliorates cardiac inflammation in IOC through the PPAR<i>α</i>-PGC1-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The schematic diagram showing that adiponectin exerts its beneficial effects in cardiomyocytes through the PPAR<i>α</i>-dependent HO-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and requires PPAR<i>α</i>-PGC-1 interaction. The adiponectin-AdipoR2-PPAR<i>α</i> <span class="highlight" style="background-color:">signaling</span> may be the major <span class="highlight" style="background-color:">pathway</span> in exerting anti-inflammatory and antioxidative stress effects that ameliorate iron-overload cardiac dysfunction.</div></p></div><div class="aux"><div class="resc">Ching-Feng Cheng, et al. Biomed Res Int. 2013;2013:740573.</div><p class="links"><a class="dblinks" href="/pubmed/24350287" ref="ordinalpos=887&amp;ncbi_uid=5602092&amp;link_uid=24350287">Citation</a><a class="dblinks" href="/pmc/articles/PMC3852805/" ref="ordinalpos=887&amp;ncbi_uid=5602092&amp;link_uid=PMC3852805">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>888.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2787898/figure/F1/" ref="ordinalpos=888&amp;ncbi_uid=2637194&amp;link_uid=PMC2787898" href="/pmc/articles/PMC2787898/"><img src="/pmc/articles/instance/2787898/bin/nihms151957f1.gif" src-large="/pmc/articles/instance/2787898/bin/nihms151957f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2787898/" ref="ordinalpos=888&amp;ncbi_uid=2637194&amp;link_uid=PMC2787898" image-link="/pmc/articles/PMC2787898/figure/F1/" class="imagepopup">Figure 1. Sketch diagram of TLR4 mediated <span class="highlight" style="background-color:">signaling</span> pathways.  From: The TRIF-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not required for acute cerebral ischemia/reperfusion injury in mice. </a></p><div class="supp"><p class="details"><div>MyD88-dependent <span class="highlight" style="background-color:">pathway</span>: TLR4 recruits MyD88, which recruits IRAKs interacting with TRAF6, thereby activating IKKα/β and phosphorylating IκB, resulting in NFκB nuclear translocation and initiation of gene transcription associated with innate immune responses and inflammation (left in Fig. 1). TRIF-dependent <span class="highlight" style="background-color:">pathway</span>: TLR4 recruits TRIF, which in turn activates TBK1/IKKε that interacts with TRAF6, resulting in NFκB nuclear translocation and activating IRF-3-inducing interferon-β (IFN-β) (right in Fig. 1).</div></p></div><div class="aux"><div class="resc">Fang Hua, et al. Biochem Biophys Res Commun. ;390(3):678-683.</div><p class="links"><a class="dblinks" href="/pubmed/19825364" ref="ordinalpos=888&amp;ncbi_uid=2637194&amp;link_uid=19825364">Citation</a><a class="dblinks" href="/pmc/articles/PMC2787898/" ref="ordinalpos=888&amp;ncbi_uid=2637194&amp;link_uid=PMC2787898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>889.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1781473/figure/ppat-0030003-g003/" ref="ordinalpos=889&amp;ncbi_uid=884296&amp;link_uid=PMC1781473" href="/pmc/articles/PMC1781473/"><img src="/pmc/articles/instance/1781473/bin/ppat.0030003.g003.gif" src-large="/pmc/articles/instance/1781473/bin/ppat.0030003.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1781473/" ref="ordinalpos=889&amp;ncbi_uid=884296&amp;link_uid=PMC1781473" image-link="/pmc/articles/PMC1781473/figure/ppat-0030003-g003/" class="imagepopup">Figure 3. T3SS Effectors Interfering with Ub <span class="highlight" style="background-color:">Signaling</span> in the Mammalian Innate Immune Response.  From: Exploitation of Eukaryotic Ubiquitin <span class="highlight" style="background-color:">Signaling</span> Pathways by Effectors Translocated by Bacterial Type III and Type IV Secretion Systems. </a></p><div class="supp"><p class="details"><div>Ubiquitination plays an important regulatory role in different steps of the innate immune <span class="highlight" style="background-color:">signaling</span> cascades. This simplified representation shows the different levels at which T3SS effectors (black-boxes) thus far have been shown to interfere with the pro-inflammatory host immune response; this interference can be at the <span class="highlight" style="background-color:">signaling</span> node represented by the TAK1 complex (affecting both NF-κB and the MAPK <span class="highlight" style="background-color:">signaling</span> pathways), or downstream of TAK1, only affecting the NF-κB <span class="highlight" style="background-color:">pathway</span>. Numbers in brackets indicate the specific references. Details of the immune <span class="highlight" style="background-color:">signaling</span> are given in the text.</div></p></div><div class="aux"><div class="resc">Aurélie Angot, et al. PLoS Pathog. 2007 January;3(1):e3.</div><p class="links"><a class="dblinks" href="/pubmed/17257058" ref="ordinalpos=889&amp;ncbi_uid=884296&amp;link_uid=17257058">Citation</a><a class="dblinks" href="/pmc/articles/PMC1781473/" ref="ordinalpos=889&amp;ncbi_uid=884296&amp;link_uid=PMC1781473">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>890.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2649824/figure/F1/" ref="ordinalpos=890&amp;ncbi_uid=1733594&amp;link_uid=PMC2649824" href="/pmc/articles/PMC2649824/"><img src="/pmc/articles/instance/2649824/bin/nihms95369f1.gif" src-large="/pmc/articles/instance/2649824/bin/nihms95369f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2649824/" ref="ordinalpos=890&amp;ncbi_uid=1733594&amp;link_uid=PMC2649824" image-link="/pmc/articles/PMC2649824/figure/F1/" class="imagepopup">Fig. 1.  From: Neuro2A Differentiation by G?i/o <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> image captured from the dynamic graphical display of the information in the Database of Cell <span class="highlight" style="background-color:">Signaling</span>, available 23 December 2008. For a key to the colors and symbols and to access the underlying data, please visit the <span class="highlight" style="background-color:">pathway</span> (http://stke.sciencemag.org/cgi/cm/stkecm;CMP_20329).</div></p></div><div class="aux"><div class="resc">Avi Ma’ayan, et al. Sci Signal. ;2(54):cm1-cm1.</div><p class="links"><a class="dblinks" href="/pubmed/19155528" ref="ordinalpos=890&amp;ncbi_uid=1733594&amp;link_uid=19155528">Citation</a><a class="dblinks" href="/pmc/articles/PMC2649824/" ref="ordinalpos=890&amp;ncbi_uid=1733594&amp;link_uid=PMC2649824">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>891.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3314723/figure/F3/" ref="ordinalpos=891&amp;ncbi_uid=4751967&amp;link_uid=PMC3314723" href="/pmc/articles/PMC3314723/"><img src="/pmc/articles/instance/3314723/bin/nihms350318f3.gif" src-large="/pmc/articles/instance/3314723/bin/nihms350318f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3314723/" ref="ordinalpos=891&amp;ncbi_uid=4751967&amp;link_uid=PMC3314723" image-link="/pmc/articles/PMC3314723/figure/F3/" class="imagepopup">Figure 3. Effects of HIF-1α silencing and suppression of EPO <span class="highlight" style="background-color:">signaling</span> on cell viability.  From: Pyruvate Protects Brain Against Ischemia-Reperfusion Injury by Activating Erythropoietin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A)</b> HIF-1α siRNA blunted HIF-1α expression in HT22 neuronal cells. <b>B)</b> HIF-1α silencing abrogated pyruvate protection of hypoxic, glucose-deprived cells (*P&lt;0.05 <i>vs.</i> Normoxia). <b>C, D)</b> Interruption of EPO <span class="highlight" style="background-color:">pathway</span> with soluble EPOR dampened pyruvate-induced protection in HT22 cells. *P&lt;0.05 <i>vs.</i> Normoxia; <sup>†</sup>P&lt; 0.05 <i>vs.</i> Control; <sup>‡</sup>P&lt; 0.05 <i>vs.</i> Pyruvate.</div></p></div><div class="aux"><div class="resc">Myoung-Gwi Ryou, et al. Stroke. ;43(4):1101-1107.</div><p class="links"><a class="dblinks" href="/pubmed/22282883" ref="ordinalpos=891&amp;ncbi_uid=4751967&amp;link_uid=22282883">Citation</a><a class="dblinks" href="/pmc/articles/PMC3314723/" ref="ordinalpos=891&amp;ncbi_uid=4751967&amp;link_uid=PMC3314723">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>892.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3291810/figure/F8/" ref="ordinalpos=892&amp;ncbi_uid=4540593&amp;link_uid=PMC3291810" href="/pmc/articles/PMC3291810/"><img src="/pmc/articles/instance/3291810/bin/nihms324032f8.gif" src-large="/pmc/articles/instance/3291810/bin/nihms324032f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3291810/" ref="ordinalpos=892&amp;ncbi_uid=4540593&amp;link_uid=PMC3291810" image-link="/pmc/articles/PMC3291810/figure/F8/" class="imagepopup">Figure 8. PAK1 simultaneously activates two distinct <span class="highlight" style="background-color:">signaling</span> pathways to promote transformation.  From: PAK1 is a breast cancer oncogene that coordinately activates MAPK and MET <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The model depicts PAK1 mechanism of action to activate the RAS-MAPK (Orange) <span class="highlight" style="background-color:">pathway</span> through direct activation of RAF1 and MEK1. The second arm (Purple) shows that PAK1 and Merlin are in an inhibitory loop, which can be pushed by <i>PAK1</i> amplification and overexpression to inhibit Merlin that leads to MET <span class="highlight" style="background-color:">signaling</span> activation through GAB1.</div></p></div><div class="aux"><div class="resc">Yashaswi Shrestha, et al. Oncogene. ;31(29):3397-3408.</div><p class="links"><a class="dblinks" href="/pubmed/22105362" ref="ordinalpos=892&amp;ncbi_uid=4540593&amp;link_uid=22105362">Citation</a><a class="dblinks" href="/pmc/articles/PMC3291810/" ref="ordinalpos=892&amp;ncbi_uid=4540593&amp;link_uid=PMC3291810">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>893.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2806014/figure/F1/" ref="ordinalpos=893&amp;ncbi_uid=2200461&amp;link_uid=PMC2806014" href="/pmc/articles/PMC2806014/"><img src="/pmc/articles/instance/2806014/bin/aging-01-281-g001.gif" src-large="/pmc/articles/instance/2806014/bin/aging-01-281-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2806014/" ref="ordinalpos=893&amp;ncbi_uid=2200461&amp;link_uid=PMC2806014" image-link="/pmc/articles/PMC2806014/figure/F1/" class="imagepopup">Figure 1.The TOR intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Validation of anti-aging drugs by treating age-related diseases. </a></p><div class="supp"><p class="details"><div> Nutrients, GF (growth factors)                                             and insulin activate the TOR <span class="highlight" style="background-color:">pathway</span>, which is involved in aging and                                             age-related diseases. Other genetic factors and environmental factors                                             (e.g., smoking) contribute to specific age-related diseases. Three                                             potential anti-aging modalities (metformin, calorie restriction and                                             rapamycin) all inhibit the TOR <span class="highlight" style="background-color:">pathway</span>.                                                              </div></p></div><div class="aux"><div class="resc">Mikhail V. Blagosklonny. Aging (Albany NY). 2009 March;1(3):281-288.</div><p class="links"><a class="dblinks" href="/pubmed/20157517" ref="ordinalpos=893&amp;ncbi_uid=2200461&amp;link_uid=20157517">Citation</a><a class="dblinks" href="/pmc/articles/PMC2806014/" ref="ordinalpos=893&amp;ncbi_uid=2200461&amp;link_uid=PMC2806014">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>894.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3296647/figure/F6/" ref="ordinalpos=894&amp;ncbi_uid=3845280&amp;link_uid=PMC3296647" href="/pmc/articles/PMC3296647/"><img src="/pmc/articles/instance/3296647/bin/1471-2105-13-20-6.gif" src-large="/pmc/articles/instance/3296647/bin/1471-2105-13-20-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3296647/" ref="ordinalpos=894&amp;ncbi_uid=3845280&amp;link_uid=PMC3296647" image-link="/pmc/articles/PMC3296647/figure/F6/" class="imagepopup">Figure 6.  From: graphite - a Bioconductor package to convert <span class="highlight" style="background-color:">pathway</span> topology to gene network. </a></p><div class="supp"><p class="details"><div><b>Results of the simulation study on the Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> compound mediated signal propagation</b>. Panel A. Signal paths selected to be differentially expressed. Panel B. p-value distribution of the topological analysis <tt>SPIA</tt> (pPERT) with and without propagation. Panel C. <tt>graphite</tt> network obtained from insulin <span class="highlight" style="background-color:">pathway</span> with propagation. Panel D. network obtained from insulin <span class="highlight" style="background-color:">pathway</span> without propagation.</div></p></div><div class="aux"><div class="resc">Gabriele Sales, et al. BMC Bioinformatics. 2012;13:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/22292714" ref="ordinalpos=894&amp;ncbi_uid=3845280&amp;link_uid=22292714">Citation</a><a class="dblinks" href="/pmc/articles/PMC3296647/" ref="ordinalpos=894&amp;ncbi_uid=3845280&amp;link_uid=PMC3296647">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>895.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3462323/figure/F3/" ref="ordinalpos=895&amp;ncbi_uid=4286023&amp;link_uid=PMC3462323" href="/pmc/articles/PMC3462323/"><img src="/pmc/articles/instance/3462323/bin/fpls-03-00219-g003.gif" src-large="/pmc/articles/instance/3462323/bin/fpls-03-00219-g003.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3462323/" ref="ordinalpos=895&amp;ncbi_uid=4286023&amp;link_uid=PMC3462323" image-link="/pmc/articles/PMC3462323/figure/F3/" class="imagepopup">FIGURE 3.  From: Endocytic <span class="highlight" style="background-color:">signaling</span> in leaves and roots: same rules different players. </a></p><div class="supp"><p class="details"><div><b> A model in roots for PIN2 polar distribution via a ROP <span class="highlight" style="background-color:">signaling</span> mechanism.</b> Data indicate that the auxin-mediated inhibition of polar PIN2 internalization is regulated by the SPK1–ROP6–RIC1 <span class="highlight" style="background-color:">pathway</span>. Auxin is proposed to activate the SPK1–ROP6–RIC1 <span class="highlight" style="background-color:">pathway</span> and inhibit PIN2 internalization. The localized inhibition of PIN2 internalization via ROP6 <span class="highlight" style="background-color:">signaling</span> causes PIN2 to be retained in the PM, which generates a positive feedback mechanism for maintaining polar PIN2 distribution to the PM.</div></p></div><div class="aux"><div class="resc">Christian Craddock, et al. Front Plant Sci. 2012;3:219.</div><p class="links"><a class="dblinks" href="/pubmed/23060890" ref="ordinalpos=895&amp;ncbi_uid=4286023&amp;link_uid=23060890">Citation</a><a class="dblinks" href="/pmc/articles/PMC3462323/" ref="ordinalpos=895&amp;ncbi_uid=4286023&amp;link_uid=PMC3462323">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>896.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4355557/figure/fig4/" ref="ordinalpos=896&amp;ncbi_uid=7323059&amp;link_uid=PMC4355557" href="/pmc/articles/PMC4355557/"><img src="/pmc/articles/instance/4355557/bin/ECAM2015-393569.004.gif" src-large="/pmc/articles/instance/4355557/bin/ECAM2015-393569.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4355557/" ref="ordinalpos=896&amp;ncbi_uid=7323059&amp;link_uid=PMC4355557" image-link="/pmc/articles/PMC4355557/figure/fig4/" class="imagepopup">Figure 4.  From: Electroacupuncture Suppressed Neuronal Apoptosis and Improved Cognitive Impairment in the AD Model Rats Possibly via Downregulation of Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Downregulation of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in rat hippocampus after EA treatment. Quantitative real-time PCR of mRNA expression of Notch1 (a), Jag1 (b), and Hes1 (c) in each group. <sup>*</sup><i>P</i> &lt; 0.05 and <sup>**</sup><i>P</i> &lt; 0.01 versus control, <sup>#</sup><i>P</i> &lt; 0.05 and <sup>##</sup><i>P</i> &lt; 0.01 versus model, and <sup>$$</sup><i>P</i> &lt; 0.01 versus sham-EA.</div></p></div><div class="aux"><div class="resc">Hai-dong Guo, et al. Evid Based Complement Alternat Med. 2015;2015:393569.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=896&amp;ncbi_uid=7323059&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4355557/" ref="ordinalpos=896&amp;ncbi_uid=7323059&amp;link_uid=PMC4355557">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>897.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3578810/figure/pone-0057231-g005/" ref="ordinalpos=897&amp;ncbi_uid=4649216&amp;link_uid=PMC3578810" href="/pmc/articles/PMC3578810/"><img src="/pmc/articles/instance/3578810/bin/pone.0057231.g005.gif" src-large="/pmc/articles/instance/3578810/bin/pone.0057231.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3578810/" ref="ordinalpos=897&amp;ncbi_uid=4649216&amp;link_uid=PMC3578810" image-link="/pmc/articles/PMC3578810/figure/pone-0057231-g005/" class="imagepopup">Figure 5. Injection of F13A suppresses the effects of apelin on glycogen synthesis and insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in TNF-α-treated mice..  From: Apelin Ameliorates TNF-?-Induced Reduction of Glycogen Synthesis in the Hepatocytes through G Protein-Coupled Receptor APJ. </a></p><div class="supp"><p class="details"><div>F13A (20 µg/mouse) or/and apelin 13 (20 nmol/kg) was injected in TNF-α-treated C57BL/6J mice. Injection of F13A suppressed the effects of apelin on glycogen synthesis (A) and insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (B) in TNF-α-treated mice. Data represent the means ± S.E.M., n = 5 mice per group. * p&lt;0.05; ** p&lt;0.01 and *** p&lt;0.001 by ANOVA test.</div></p></div><div class="aux"><div class="resc">Jiaojiao Chu, et al. PLoS One. 2013;8(2):e57231.</div><p class="links"><a class="dblinks" href="/pubmed/23437347" ref="ordinalpos=897&amp;ncbi_uid=4649216&amp;link_uid=23437347">Citation</a><a class="dblinks" href="/pmc/articles/PMC3578810/" ref="ordinalpos=897&amp;ncbi_uid=4649216&amp;link_uid=PMC3578810">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>898.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2640107/figure/F5/" ref="ordinalpos=898&amp;ncbi_uid=1777426&amp;link_uid=PMC2640107" href="/pmc/articles/PMC2640107/"><img src="/pmc/articles/instance/2640107/bin/nihms91988f5.gif" src-large="/pmc/articles/instance/2640107/bin/nihms91988f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2640107/" ref="ordinalpos=898&amp;ncbi_uid=1777426&amp;link_uid=PMC2640107" image-link="/pmc/articles/PMC2640107/figure/F5/" class="imagepopup">Figure 5. Radial glial <span class="highlight" style="background-color:">signaling</span> pathways.  From: Regulation of Neural Progenitor Cell Development in the Nervous System. </a></p><div class="supp"><p class="details"><div>These pathways putatitvely influence the morphology of radial glia, as well as the ability of radial glia to provide a scaffold for migrating neurons and generate new neurons. The classic reelin <span class="highlight" style="background-color:">pathway</span> involves VLDL and ApoER2 receptors and Dab1 as shown on the left in shades of pink; PI3K is also an integral part of reelin <span class="highlight" style="background-color:">signaling</span>. Neuregulin (NRG1) <span class="highlight" style="background-color:">signaling</span> progresses primarily through erbB receptors and pathways leading to PI3K activation (shades of green) or involvement of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (indicated in shades of purple). Activation of Notch results in upregulation of BLBP production. In certain radial glial cells, blockade of PI3K can be compensated by NRG1 <span class="highlight" style="background-color:">signaling</span> through erbB receptors that coordinate with Notch.</div></p></div><div class="aux"><div class="resc">Joshua G. Corbin, et al. J Neurochem. ;106(6):2272-2287.</div><p class="links"><a class="dblinks" href="/pubmed/18819190" ref="ordinalpos=898&amp;ncbi_uid=1777426&amp;link_uid=18819190">Citation</a><a class="dblinks" href="/pmc/articles/PMC2640107/" ref="ordinalpos=898&amp;ncbi_uid=1777426&amp;link_uid=PMC2640107">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>899.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3881972/figure/F1/" ref="ordinalpos=899&amp;ncbi_uid=5680019&amp;link_uid=PMC3881972" href="/pmc/articles/PMC3881972/"><img src="/pmc/articles/instance/3881972/bin/nihms-278354-f0001.gif" src-large="/pmc/articles/instance/3881972/bin/nihms-278354-f0001.jpg" alt="Fig.1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3881972/" ref="ordinalpos=899&amp;ncbi_uid=5680019&amp;link_uid=PMC3881972" image-link="/pmc/articles/PMC3881972/figure/F1/" class="imagepopup">Fig.1. Identified molecular targets of phytochemicals on PRR-mediated inflammatory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Inhibition of pattern recognition receptor-mediated inflammation by bioactive phytochemicals: a review of recent research. </a></p><div class="supp"><p class="details"><div>TLRs and NODs are pathogen recognition receptors (PRRs) that detect conserved molecules of pathogens. Stimulation of TLRs by ligands leads to the recruitment of adaptor molecules such as MyD88 and TRIF through the interaction of TIR domains, leading to MyD88 dependent and MyD88-independent (TRIF dependent) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The activation of NODs leads to the recruitment of RICK/RIP2 through CARD–CARD interactions, leading to activation of inflammatory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Curcumin, helenalin, cinnamaldehyde, and sulforaphane inhibit TLR4-mediated NF-κB activation by inhibiting TLR4 dimerization. Resveratrol, EGCG, luteolin, quercetin, chrysin, and eriodictyol directly inhibit TBK1 kinase activity. Similarly, curcumin, helenalin and parthenolide also inhibit NOD2 mediated NF-κB activation, presumably through inhibition of NOD2 dimerization.</div></p></div><div class="aux"><div class="resc">Ling Zhao, et al. Nutr Rev. ;69(6):10.1111/j.1753-4887.2011.00394.x.</div><p class="links"><a class="dblinks" href="/pubmed/21631512" ref="ordinalpos=899&amp;ncbi_uid=5680019&amp;link_uid=21631512">Citation</a><a class="dblinks" href="/pmc/articles/PMC3881972/" ref="ordinalpos=899&amp;ncbi_uid=5680019&amp;link_uid=PMC3881972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>900.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109867/figure/F2/" ref="ordinalpos=900&amp;ncbi_uid=3006879&amp;link_uid=PMC3109867" href="/pmc/articles/PMC3109867/"><img src="/pmc/articles/instance/3109867/bin/nihms296159f2.gif" src-large="/pmc/articles/instance/3109867/bin/nihms296159f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109867/" ref="ordinalpos=900&amp;ncbi_uid=3006879&amp;link_uid=PMC3109867" image-link="/pmc/articles/PMC3109867/figure/F2/" class="imagepopup">Figure 2.  From: InAKTivation of insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> by dephosphorylation. </a></p><div class="supp"><p class="details"><div>Overlapping and distinct expression patterns of insulin/iGF-1 <span class="highlight" style="background-color:">signaling</span> components suggest a possible neuroendocrine regulation of the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Sri Devi Narasimhan, et al. Cell Cycle. ;8(23):3878-3884.</div><p class="links"><a class="dblinks" href="/pubmed/19901535" ref="ordinalpos=900&amp;ncbi_uid=3006879&amp;link_uid=19901535">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109867/" ref="ordinalpos=900&amp;ncbi_uid=3006879&amp;link_uid=PMC3109867">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="8" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="9" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="10" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:21:45-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC40E053FEBB3100000000004B3DC5;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC40E053FEBB3100000000004B3DC5" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>