
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="8" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A2E1F53FE8D81000000000023A919" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="7" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="8" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="9" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="8" /></div>
            <div><h2 class="result_count">Results: 701 to 800 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>701.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2559933/figure/f3/" ref="ordinalpos=701&amp;ncbi_uid=1540639&amp;link_uid=PMC2559933" href="/pmc/articles/PMC2559933/"><img src="/pmc/articles/instance/2559933/bin/zjw0100832530003.gif" src-large="/pmc/articles/instance/2559933/bin/zjw0100832530003.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2559933/" ref="ordinalpos=701&amp;ncbi_uid=1540639&amp;link_uid=PMC2559933" image-link="/pmc/articles/PMC2559933/figure/f3/" class="imagepopup">F ig . 3. From: Targeting the Human Cancer <span class="highlight" style="background-color:">Pathway</span> Protein Interaction Network by Structural Genomics. </a></p><div class="supp"><p class="details"><div><b>Cross-talk between pathways.</b> <i>A</i>, frequency of observing one protein in one or more of the seven KEGG <span class="highlight" style="background-color:">signaling</span> pathways. ∼20% of HCPIN <span class="highlight" style="background-color:">pathway</span> proteins are associated with two or more pathways. <i>B</i>, frequency of observing one HCPIN protein in one or more of seven <span class="highlight" style="background-color:">pathway</span> interaction subnets. &gt;50% of HCPIN proteins are associated with two or more interaction subnets. <i>C</i>, frequency of observing one <span class="highlight" style="background-color:">pathway</span> protein in one or more <span class="highlight" style="background-color:">pathway</span> interaction subnets. The frequencies (<i>1</i>–<i>7</i>) are also labeled on the <i>side</i> of these pie charts.</div></p></div><div class="aux"><div class="resc">Yuanpeng Janet Huang, et al. Mol Cell Proteomics. 2008 October;7(10):2048-2060.</div><p class="links"><a class="dblinks" href="/pubmed/18487680" ref="ordinalpos=701&amp;ncbi_uid=1540639&amp;link_uid=18487680">Citation</a><a class="dblinks" href="/pmc/articles/PMC2559933/" ref="ordinalpos=701&amp;ncbi_uid=1540639&amp;link_uid=PMC2559933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>702.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3283716/figure/pone-0031906-g001/" ref="ordinalpos=702&amp;ncbi_uid=3820138&amp;link_uid=PMC3283716" href="/pmc/articles/PMC3283716/"><img src="/pmc/articles/instance/3283716/bin/pone.0031906.g001.gif" src-large="/pmc/articles/instance/3283716/bin/pone.0031906.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3283716/" ref="ordinalpos=702&amp;ncbi_uid=3820138&amp;link_uid=PMC3283716" image-link="/pmc/articles/PMC3283716/figure/pone-0031906-g001/" class="imagepopup">Figure 1. A major <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> implicated in lung cancer is the EGFR <span class="highlight" style="background-color:">pathway</span> which signals to both the AKT/PI3K <span class="highlight" style="background-color:">pathway</span> (green) and the MAPK <span class="highlight" style="background-color:">pathway</span> (red) which regulate cell growth, proliferation and cell death..  From: A Novel Classification of Lung Cancer into Molecular Subtypes. </a></p><div class="supp"><p class="details"><div>There is significant cross-talk between these pathways and their downstream effectors, which we have classified into 6 pathways for simplicity to account for differences in treatment modalities. The additional 4 pathways are: EGFR (blue), KRAS (yellow), EML4-ALK (orange), and P53/BCL (purple). It is thought that the RAS/RAF/MEK/MAPK <span class="highlight" style="background-color:">pathway</span> may be constitutively activated by the EML4-ALK fusion oncogene [100]. The complex relationship between this <span class="highlight" style="background-color:">pathway</span> and EML4-ALK is indicated with a dashed line.</div></p></div><div class="aux"><div class="resc">Lisandra West, et al. PLoS One. 2012;7(2):e31906.</div><p class="links"><a class="dblinks" href="/pubmed/22363766" ref="ordinalpos=702&amp;ncbi_uid=3820138&amp;link_uid=22363766">Citation</a><a class="dblinks" href="/pmc/articles/PMC3283716/" ref="ordinalpos=702&amp;ncbi_uid=3820138&amp;link_uid=PMC3283716">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>703.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2496917/figure/F1/" ref="ordinalpos=703&amp;ncbi_uid=1448044&amp;link_uid=PMC2496917" href="/pmc/articles/PMC2496917/"><img src="/pmc/articles/instance/2496917/bin/1471-2164-9-348-1.gif" src-large="/pmc/articles/instance/2496917/bin/1471-2164-9-348-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2496917/" ref="ordinalpos=703&amp;ncbi_uid=1448044&amp;link_uid=PMC2496917" image-link="/pmc/articles/PMC2496917/figure/F1/" class="imagepopup">Figure 1.  From: Deciphering downstream gene targets of PI3K/mTOR/p70S6K <span class="highlight" style="background-color:">pathway</span> in breast cancer. </a></p><div class="supp"><p class="details"><div><b>Interfering the PI3K/mTOR/p70S6K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Schematic representation of the PI3K/mTOR/p70S6K <span class="highlight" style="background-color:">pathway</span> illustrates that p70S6K is located downstream of PI3K and mTOR. In this study, PI3K was inhibited with Ly294002, mTOR with rapamycin and p70S6K with siRNAs targeting <i>RPS6KB1</i>.</div></p></div><div class="aux"><div class="resc">Henna Heinonen, et al. BMC Genomics. 2008;9:348-348.</div><p class="links"><a class="dblinks" href="/pubmed/18652687" ref="ordinalpos=703&amp;ncbi_uid=1448044&amp;link_uid=18652687">Citation</a><a class="dblinks" href="/pmc/articles/PMC2496917/" ref="ordinalpos=703&amp;ncbi_uid=1448044&amp;link_uid=PMC2496917">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>704.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3308137/figure/F1/" ref="ordinalpos=704&amp;ncbi_uid=4715090&amp;link_uid=PMC3308137" href="/pmc/articles/PMC3308137/"><img src="/pmc/articles/instance/3308137/bin/nihms358197f1.gif" src-large="/pmc/articles/instance/3308137/bin/nihms358197f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3308137/" ref="ordinalpos=704&amp;ncbi_uid=4715090&amp;link_uid=PMC3308137" image-link="/pmc/articles/PMC3308137/figure/F1/" class="imagepopup">Figure 1.  From: MicroRNAs in stress <span class="highlight" style="background-color:">signaling</span> and human disease. </a></p><div class="supp"><p class="details"><div>miRNAs can regulate stress <span class="highlight" style="background-color:">signaling</span> pathways and thereby modify stress-related phenotypes through a number of distinct mechanisms, some of which are schematized here. A miRNA can perform a stress signal mediation function (A) in which it acts as a critical intermediate in a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> or it may act as a stress signal modulator (B) in which it titrates a <span class="highlight" style="background-color:">signaling</span> intermediate. A miRNA may participate in a negative (C) or positive (D) feedback loop which serves to dampen or amplify a signal, respectively. Lastly, a miRNA may target both positive and negative regulators of a <span class="highlight" style="background-color:">pathway</span> (E), thereby buffering <span class="highlight" style="background-color:">pathway</span> activity from stochastic fluctuation. For each mode of regulation, the consequences of deletion of the miRNA on <span class="highlight" style="background-color:">pathway</span> activity are schematized on the right.</div></p></div><div class="aux"><div class="resc">Joshua T. Mendell, et al. Cell. ;148(6):1172-1187.</div><p class="links"><a class="dblinks" href="/pubmed/22424228" ref="ordinalpos=704&amp;ncbi_uid=4715090&amp;link_uid=22424228">Citation</a><a class="dblinks" href="/pmc/articles/PMC3308137/" ref="ordinalpos=704&amp;ncbi_uid=4715090&amp;link_uid=PMC3308137">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>705.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4403904/figure/Fig1/" ref="ordinalpos=705&amp;ncbi_uid=7435223&amp;link_uid=PMC4403904" href="/pmc/articles/PMC4403904/"><img src="/pmc/articles/instance/4403904/bin/13020_2015_36_Fig1_HTML.gif" src-large="/pmc/articles/instance/4403904/bin/13020_2015_36_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4403904/" ref="ordinalpos=705&amp;ncbi_uid=7435223&amp;link_uid=PMC4403904" image-link="/pmc/articles/PMC4403904/figure/Fig1/" class="imagepopup">Figure 1.  From: Potent natural products and herbal medicines for treating liver fibrosis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates hepatic fibrogenesis: <b>(A)</b> the TGF-β1/SMAD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and <b>(B)</b> TLR4 activated-MyD88/TGFβ1/NFκB <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Shao-Ru Chen, et al. Chin Med. 2015;10:7.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=705&amp;ncbi_uid=7435223&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4403904/" ref="ordinalpos=705&amp;ncbi_uid=7435223&amp;link_uid=PMC4403904">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>706.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2863215/figure/F2/" ref="ordinalpos=706&amp;ncbi_uid=2936822&amp;link_uid=PMC2863215" href="/pmc/articles/PMC2863215/"><img src="/pmc/articles/instance/2863215/bin/zbc0231016410002.gif" src-large="/pmc/articles/instance/2863215/bin/zbc0231016410002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2863215/" ref="ordinalpos=706&amp;ncbi_uid=2936822&amp;link_uid=PMC2863215" image-link="/pmc/articles/PMC2863215/figure/F2/" class="imagepopup">FIGURE 2. From: Mammalian Target of Rapamycin (mTOR): Conducting the Cellular <span class="highlight" style="background-color:">Signaling</span> Symphony. </a></p><div class="supp"><p class="details"><div><b>Regulation of mTORC <span class="highlight" style="background-color:">signaling</span> networks.</b> Growth factors/mitogens (insulin, EGF) and nutrients (<i>e.g.</i> amino acids, energy) promote mTORC1 <span class="highlight" style="background-color:">signaling</span> via phosphorylation cascades that converge on TSC and the mTORCs themselves. Insulin signals via its receptor (<i>Insulin-R</i>) to activate the PI3K/Akt/TSC/Rheb <span class="highlight" style="background-color:">pathway</span>; EGF signals via its receptor (<i>EGF-R</i>) to activate the Ras/MEK/MAPK/RSK <span class="highlight" style="background-color:">pathway</span>; amino acid sufficiency signals via hVps34 and the RAG and RalA GTPases; and energy sufficiency suppresses AMPK. Insulin/PI3K <span class="highlight" style="background-color:">signaling</span> likely promotes mTORC2 <span class="highlight" style="background-color:">signaling</span> via an unknown <span class="highlight" style="background-color:">pathway</span>. An mTORC1/S6K1-mediated negative feedback loop signals via two pathways to suppress PI3K/mTORC2/Akt <span class="highlight" style="background-color:">signaling</span>. <i>Arrows versus blocked lines</i> indicate activation or inhibition of protein function, respectively, by an upstream regulator. Phosphorylation events (denoted by <i>circled yellow P</i>) known to modulate protein function are shown. The kinases responsible for phosphorylation events are also indicated, with (+) or (−) denoting activation or inhibition of protein function, respectively.</div></p></div><div class="aux"><div class="resc">Kathryn G. Foster, et al. J Biol Chem. 2010 May 7;285(19):14071-14077.</div><p class="links"><a class="dblinks" href="/pubmed/20231296" ref="ordinalpos=706&amp;ncbi_uid=2936822&amp;link_uid=20231296">Citation</a><a class="dblinks" href="/pmc/articles/PMC2863215/" ref="ordinalpos=706&amp;ncbi_uid=2936822&amp;link_uid=PMC2863215">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>707.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2518074/figure/F8/" ref="ordinalpos=707&amp;ncbi_uid=1531246&amp;link_uid=PMC2518074" href="/pmc/articles/PMC2518074/"><img src="/pmc/articles/instance/2522347/bin/JCI0834764.f8.gif" src-large="/pmc/articles/instance/2522347/bin/JCI0834764.f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2518074/" ref="ordinalpos=707&amp;ncbi_uid=1531246&amp;link_uid=PMC2518074" image-link="/pmc/articles/PMC2518074/figure/F8/" class="imagepopup">Figure 8. AKT/mTOR <span class="highlight" style="background-color:">pathway</span> activation is associated with human prostate cancer progression and correlated with activation of ERK MAPK..  From: Targeting AKT/mTOR and ERK MAPK <span class="highlight" style="background-color:">signaling</span> inhibits hormone-refractory prostate cancer in a preclinical mouse model. </a></p><div class="supp"><p class="details"><div>(<b>A</b>–<b>L</b>) Expression of PTEN and components on the AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human normal and primary prostate cancer samples. Representative adjacent sections from the specimens used for the TMAs show staining for the indicated proteins. Shown are examples of benign prostatic tissue (Normal), a tumor without activation of the PTEN/AKT/mTOR <span class="highlight" style="background-color:">pathway</span> (Tumor 1), and a tumor with activation of this <span class="highlight" style="background-color:">pathway</span> (Tumor 2). Note that the expression of PTEN is inversely correlated with expression of p-AKT, p-mTOR, and p-S6. (<b>M</b>–<b>O</b>) Expression of p-ERK activation on semiadjacent sections of the same specimens. Note that, in these samples, p-ERK activation is well correlated with activation of components of the AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Scale bar: 100 μm.    </div></p></div><div class="aux"><div class="resc">Carolyn Waugh Kinkade, et al. J Clin Invest. 2008 September 2;118(9):3051-3064.</div><p class="links"><a class="dblinks" href="/pubmed/18725989" ref="ordinalpos=707&amp;ncbi_uid=1531246&amp;link_uid=18725989">Citation</a><a class="dblinks" href="/pmc/articles/PMC2518074/" ref="ordinalpos=707&amp;ncbi_uid=1531246&amp;link_uid=PMC2518074">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>708.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4027372/figure/cht1/" ref="ordinalpos=708&amp;ncbi_uid=6365561&amp;link_uid=PMC4027372" href="/pmc/articles/PMC4027372/"><img src="/pmc/articles/instance/4027372/bin/ml-2013-00014t_0006.gif" src-large="/pmc/articles/instance/4027372/bin/ml-2013-00014t_0006.jpg" alt="Chart 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4027372/" ref="ordinalpos=708&amp;ncbi_uid=6365561&amp;link_uid=PMC4027372" image-link="/pmc/articles/PMC4027372/figure/cht1/" class="imagepopup">Chart 1. Hh <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Antagonists.  From: Identification of Vitamin D3-Based Hedgehog <span class="highlight" style="background-color:">Pathway</span> Inhibitors That Incorporate an Aromatic A-Ring Isostere. </a></p><div class="aux"><div class="resc">Albert M. DeBerardinis, et al. ACS Med Chem Lett. 2013 July 11;4(7):590-595.</div><p class="links"><a class="dblinks" href="/pubmed/24900716" ref="ordinalpos=708&amp;ncbi_uid=6365561&amp;link_uid=24900716">Citation</a><a class="dblinks" href="/pmc/articles/PMC4027372/" ref="ordinalpos=708&amp;ncbi_uid=6365561&amp;link_uid=PMC4027372">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>709.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2742821/figure/F10/" ref="ordinalpos=709&amp;ncbi_uid=2002296&amp;link_uid=PMC2742821" href="/pmc/articles/PMC2742821/"><img src="/pmc/articles/instance/2742821/bin/zbc0370985170010.gif" src-large="/pmc/articles/instance/2742821/bin/zbc0370985170010.jpg" alt="FIGURE 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2742821/" ref="ordinalpos=709&amp;ncbi_uid=2002296&amp;link_uid=PMC2742821" image-link="/pmc/articles/PMC2742821/figure/F10/" class="imagepopup">FIGURE 10. From: Peptidoglycan Induces Cyclooxygenase-2 Expression in Macrophages by Activating the Neutral Sphingomyelinase-Ceramide <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram illustrating the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in PGN-induced COX-2 expression in macrophages through two separate pathways: the Ras/Raf-1/ERK <span class="highlight" style="background-color:">pathway</span> to induce IKKα/β activation and the nSMase/ceramide <span class="highlight" style="background-color:">pathway</span> to induce MKK3/6/p38 MAPK activation cascades resulting in an increase in NF-κB transactivation, ultimately causing COX-2 expression in macrophages.</b></div></p></div><div class="aux"><div class="resc">Bing-Chang Chen, et al. J Biol Chem. 2009 July 31;284(31):20562-20573.</div><p class="links"><a class="dblinks" href="/pubmed/19531467" ref="ordinalpos=709&amp;ncbi_uid=2002296&amp;link_uid=19531467">Citation</a><a class="dblinks" href="/pmc/articles/PMC2742821/" ref="ordinalpos=709&amp;ncbi_uid=2002296&amp;link_uid=PMC2742821">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>710.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2483369/figure/fig6/" ref="ordinalpos=710&amp;ncbi_uid=1489203&amp;link_uid=PMC2483369" href="/pmc/articles/PMC2483369/"><img src="/pmc/articles/instance/2483369/bin/tpc2001678f06.gif" src-large="/pmc/articles/instance/2483369/bin/tpc2001678f06.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2483369/" ref="ordinalpos=710&amp;ncbi_uid=1489203&amp;link_uid=PMC2483369" image-link="/pmc/articles/PMC2483369/figure/fig6/" class="imagepopup">Figure 6. From: Antagonistic Interaction between Systemic Acquired Resistance and the Abscisic Acid-Mediated Abiotic Stress Response in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Proposed Model for the Crosstalk between Biotic and Abiotic Stress Responses.<br />SAR is induced through the SA-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is activated by the HR to avirulent biotrophic pathogens. The ABA-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by environmental stressors, such as salinity, drought, and, to some extent, by cold. Crosstalk between these <span class="highlight" style="background-color:">signaling</span> pathways, shown as broken lines, was shown to antagonistically regulate each other. Arrows indicate positive regulation, and blunt ends denote negative regulation. White arrows indicate the activation site of SAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by the chemicals BIT and BTH. Some genes functioning in the <span class="highlight" style="background-color:">signaling</span> pathways are also shown.</div></p></div><div class="aux"><div class="resc">Michiko Yasuda, et al. Plant Cell. 2008 June;20(6):1678-1692.</div><p class="links"><a class="dblinks" href="/pubmed/18586869" ref="ordinalpos=710&amp;ncbi_uid=1489203&amp;link_uid=18586869">Citation</a><a class="dblinks" href="/pmc/articles/PMC2483369/" ref="ordinalpos=710&amp;ncbi_uid=1489203&amp;link_uid=PMC2483369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>711.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2996291/figure/pone-0014180-g009/" ref="ordinalpos=711&amp;ncbi_uid=2626209&amp;link_uid=PMC2996291" href="/pmc/articles/PMC2996291/"><img src="/pmc/articles/instance/2996291/bin/pone.0014180.g009.gif" src-large="/pmc/articles/instance/2996291/bin/pone.0014180.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2996291/" ref="ordinalpos=711&amp;ncbi_uid=2626209&amp;link_uid=PMC2996291" image-link="/pmc/articles/PMC2996291/figure/pone-0014180-g009/" class="imagepopup">Figure 9. A model for <span class="highlight" style="background-color:">signaling</span> of SWAP-70-374-dependent transformation of MEFs..  From: A Mutant of SWAP-70, a Phosphatidylinositoltrisphosphate Binding Protein, Transforms Mouse Embryo Fibroblasts, Which Is Inhibited by Sanguinarine. </a></p><div class="supp"><p class="details"><div>The putative <span class="highlight" style="background-color:">signaling</span> model of SWAP-70-374-dependent transformation is shown. So far, only the role of the ERK1/2 <span class="highlight" style="background-color:">pathway</span> in regulation of contact inhibition is known: there should be at least one more <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> responsible for fast growth of the cells.</div></p></div><div class="aux"><div class="resc">Yasuhisa Fukui, et al. PLoS One. 2010;5(12):e14180.</div><p class="links"><a class="dblinks" href="/pubmed/21152038" ref="ordinalpos=711&amp;ncbi_uid=2626209&amp;link_uid=21152038">Citation</a><a class="dblinks" href="/pmc/articles/PMC2996291/" ref="ordinalpos=711&amp;ncbi_uid=2626209&amp;link_uid=PMC2996291">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>712.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC307657/figure/fig2/" ref="ordinalpos=712&amp;ncbi_uid=357453&amp;link_uid=PMC307657" href="/pmc/articles/PMC307657/"><img src="/pmc/articles/instance/307657/bin/zpq0250332250002.gif" src-large="/pmc/articles/instance/307657/bin/zpq0250332250002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC307657/" ref="ordinalpos=712&amp;ncbi_uid=357453&amp;link_uid=PMC307657" image-link="/pmc/articles/PMC307657/figure/fig2/" class="imagepopup">Fig. 2. From: Evidence that transgenes encoding components of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> preferentially induce mammary cancers from progenitor cells. </a></p><div class="supp"><p class="details"><div>Keratin 6-positive cells in mammary tumors arising in TG mice expressing components of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Keratin 6 was detected by immunohistochemical staining of tumors from mice carrying the indicated MMTV TG. The scale is shown in <i>F</i>.</div></p></div><div class="aux"><div class="resc">Yi Li, et al. Proc Natl Acad Sci U S A. 2003 December 23;100(26):15853-15858.</div><p class="links"><a class="dblinks" href="/pubmed/14668450" ref="ordinalpos=712&amp;ncbi_uid=357453&amp;link_uid=14668450">Citation</a><a class="dblinks" href="/pmc/articles/PMC307657/" ref="ordinalpos=712&amp;ncbi_uid=357453&amp;link_uid=PMC307657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>713.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2709100/figure/F1/" ref="ordinalpos=713&amp;ncbi_uid=1993970&amp;link_uid=PMC2709100" href="/pmc/articles/PMC2709100/"><img src="/pmc/articles/instance/2709100/bin/nihms113342f1.gif" src-large="/pmc/articles/instance/2709100/bin/nihms113342f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2709100/" ref="ordinalpos=713&amp;ncbi_uid=1993970&amp;link_uid=PMC2709100" image-link="/pmc/articles/PMC2709100/figure/F1/" class="imagepopup">Figure 1. Schematics of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Notch <span class="highlight" style="background-color:">signaling</span> in differentiation and function of dendritic cells. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> is initiated through binding of Notch ligands expressed an adjacent cell, followed by enzymatic cleavage and release of intercellular of Notch (ICN). The ICN then translocates to the nucleus, where it binds to CSL transcription repressor and converts it to activator by recruiting MAML (mastermind-like protein) to the complex including p300 and co-activator. Notch <span class="highlight" style="background-color:">signaling</span> is also regulated by other cellular proteins including Deltex, Nrarp (Notch-regulated ankyrin-repeat protein) and MINT (Msx2-interacting nuclear target protein).</div></p></div><div class="aux"><div class="resc">Pingyan Cheng, et al. Immunol Res. ;41(1):1-14.</div><p class="links"><a class="dblinks" href="/pubmed/18008056" ref="ordinalpos=713&amp;ncbi_uid=1993970&amp;link_uid=18008056">Citation</a><a class="dblinks" href="/pmc/articles/PMC2709100/" ref="ordinalpos=713&amp;ncbi_uid=1993970&amp;link_uid=PMC2709100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>714.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013419/figure/cjc-30-07-442-g001/" ref="ordinalpos=714&amp;ncbi_uid=6136980&amp;link_uid=PMC4013419" href="/pmc/articles/PMC4013419/"><img src="/pmc/articles/instance/4013419/bin/cjc-30-07-442-g001.gif" src-large="/pmc/articles/instance/4013419/bin/cjc-30-07-442-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013419/" ref="ordinalpos=714&amp;ncbi_uid=6136980&amp;link_uid=PMC4013419" image-link="/pmc/articles/PMC4013419/figure/cjc-30-07-442-g001/" class="imagepopup">Figure 1.Major components of Sonic hedgehog (Shh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Mouse models of medulloblastoma. </a></p><div class="supp"><p class="details"><div>In the absence of Shh, the negative regulator patched 1 (PTC1), localized at the plasma membrane, inhibits <span class="highlight" style="background-color:">signaling</span> through the activating mediator smoothened (SMO). In the presence of Shh ligand, Shh binds to PTC and derepresses its negative regulation on SMO, activating <span class="highlight" style="background-color:">signaling</span> by inducing the expression of transcription factors including MycN and GLI in the nucleus. The Shh <span class="highlight" style="background-color:">pathway</span> can be negatively regulated by suppressor of fused (SUFU) in the cytosol.</div></p></div><div class="aux"><div class="resc">Xiaochong Wu, et al. Chin J Cancer. 2011 July;30(7):442-449.</div><p class="links"><a class="dblinks" href="/pubmed/21718590" ref="ordinalpos=714&amp;ncbi_uid=6136980&amp;link_uid=21718590">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013419/" ref="ordinalpos=714&amp;ncbi_uid=6136980&amp;link_uid=PMC4013419">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>715.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4405131/figure/F8/" ref="ordinalpos=715&amp;ncbi_uid=7441420&amp;link_uid=PMC4405131" href="/pmc/articles/PMC4405131/"><img src="/pmc/articles/instance/4405131/bin/nihms-672469-f0008.gif" src-large="/pmc/articles/instance/4405131/bin/nihms-672469-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4405131/" ref="ordinalpos=715&amp;ncbi_uid=7441420&amp;link_uid=PMC4405131" image-link="/pmc/articles/PMC4405131/figure/F8/" class="imagepopup">Fig. 8. Model for the regulation of mTOR <span class="highlight" style="background-color:">signaling</span> by Carma1 and MALT1 in response to TCR <span class="highlight" style="background-color:">signaling</span> in T cells.  From: A Carma1/MALT1-dependent, Bcl10-independent, <span class="highlight" style="background-color:">pathway</span> regulates antigen receptor-mediated mTOR <span class="highlight" style="background-color:">signaling</span> in T cells. </a></p><div class="supp"><p class="details"><div>Our data suggest that Carma1 and MALT1 participate in the TCR-dependent activation of mTOR downstream of PKC. The other CBM component, Bcl10, seems to be dispensable for this <span class="highlight" style="background-color:">pathway</span>, although it is possible that another protein substitutes for Bcl10 in mediating an association between Carma1 and MALT1 (indicated by the oval with a question mark). See the Discussion for more details.</div></p></div><div class="aux"><div class="resc">Kristia S. Hamilton, et al. Sci Signal. ;7(329):ra55-ra55.</div><p class="links"><a class="dblinks" href="/pubmed/24917592" ref="ordinalpos=715&amp;ncbi_uid=7441420&amp;link_uid=24917592">Citation</a><a class="dblinks" href="/pmc/articles/PMC4405131/" ref="ordinalpos=715&amp;ncbi_uid=7441420&amp;link_uid=PMC4405131">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>716.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2265565/figure/fig3/" ref="ordinalpos=716&amp;ncbi_uid=1271320&amp;link_uid=PMC2265565" href="/pmc/articles/PMC2265565/"><img src="/pmc/articles/instance/2265565/bin/jcb1800721f03.gif" src-large="/pmc/articles/instance/2265565/bin/jcb1800721f03.jpg" alt="Figure 3. " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2265565/" ref="ordinalpos=716&amp;ncbi_uid=1271320&amp;link_uid=PMC2265565" image-link="/pmc/articles/PMC2265565/figure/fig3/" class="imagepopup">Figure 3.  From: The JAK/STAT <span class="highlight" style="background-color:">pathway</span> positively regulates DPP <span class="highlight" style="background-color:">signaling</span> in the Drosophila germline stem cell niche. </a></p><div class="supp"><p class="details"><div><b>JAK/STAT <span class="highlight" style="background-color:">signaling</span> functions downstream or in parallel of N <span class="highlight" style="background-color:">pathway</span>.</b> Similar number of lamin C–positive cap cells were present in (A) wild-type, (B) <i>c587-GAL4; uas-upd</i>, and (C) <i>stat92e<sup>TS</sup></i> germaria. Note that there are more lamin C–positive cap cells when N <span class="highlight" style="background-color:">signaling</span> was ectopically activated in wild-type (D), <i>dpp<sup>+/−</sup></i> (E), and <i>stat92e<sup>TS</sup></i> (F) background. Anterior toward left. Bars, 10 μm.</div></p></div><div class="aux"><div class="resc">Liwei Wang, et al. J Cell Biol. 2008 February 25;180(4):721-728.</div><p class="links"><a class="dblinks" href="/pubmed/18283115" ref="ordinalpos=716&amp;ncbi_uid=1271320&amp;link_uid=18283115">Citation</a><a class="dblinks" href="/pmc/articles/PMC2265565/" ref="ordinalpos=716&amp;ncbi_uid=1271320&amp;link_uid=PMC2265565">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>717.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2784807/figure/F2/" ref="ordinalpos=717&amp;ncbi_uid=2177815&amp;link_uid=PMC2784807" href="/pmc/articles/PMC2784807/"><img src="/pmc/articles/instance/2784807/bin/1471-2164-10-542-2.gif" src-large="/pmc/articles/instance/2784807/bin/1471-2164-10-542-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2784807/" ref="ordinalpos=717&amp;ncbi_uid=2177815&amp;link_uid=PMC2784807" image-link="/pmc/articles/PMC2784807/figure/F2/" class="imagepopup">Figure 2.  From: Gene network and <span class="highlight" style="background-color:">pathway</span> analysis of bovine mammary tissue challenged with Streptococcus uberis reveals induction of cell proliferation and inhibition of PPAR? <span class="highlight" style="background-color:">signaling</span> as potential mechanism for the negative relationships between immune response and lipid metabolism. </a></p><div class="supp"><p class="details"><div><b>Acute phase response <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> among 2,102 DEG due to IMI with <i>Streptococcus uberis</i></b>. Red denotes up-regulation and green down-regulation of the gene.</div></p></div><div class="aux"><div class="resc">Kasey M Moyes, et al. BMC Genomics. 2009;10:542-542.</div><p class="links"><a class="dblinks" href="/pubmed/19925655" ref="ordinalpos=717&amp;ncbi_uid=2177815&amp;link_uid=19925655">Citation</a><a class="dblinks" href="/pmc/articles/PMC2784807/" ref="ordinalpos=717&amp;ncbi_uid=2177815&amp;link_uid=PMC2784807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>718.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2958340/figure/fig4/" ref="ordinalpos=718&amp;ncbi_uid=2534176&amp;link_uid=PMC2958340" href="/pmc/articles/PMC2958340/"><img src="/pmc/articles/instance/2958340/bin/JO2011-941876.004.gif" src-large="/pmc/articles/instance/2958340/bin/JO2011-941876.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2958340/" ref="ordinalpos=718&amp;ncbi_uid=2534176&amp;link_uid=PMC2958340" image-link="/pmc/articles/PMC2958340/figure/fig4/" class="imagepopup">Figure 4.  From: Targeting the Mechanisms of Resistance to Chemotherapy and  Radiotherapy with the Cancer Stem Cell Hypothesis. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Notch/<i>γ</i>-secretase/Jagged <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is an important regulator of differentiation and helps specify cell fate. Notch <span class="highlight" style="background-color:">signaling</span> pathways have been shown to be activated in both breast CSCs and in endothelial cells in response to radiation. MK-0752 and R4733 are drugs under development targeting <i>γ</i>-secretase in this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ryan Morrison, et al. J Oncol. 2011;2011:941876.</div><p class="links"><a class="dblinks" href="/pubmed/20981352" ref="ordinalpos=718&amp;ncbi_uid=2534176&amp;link_uid=20981352">Citation</a><a class="dblinks" href="/pmc/articles/PMC2958340/" ref="ordinalpos=718&amp;ncbi_uid=2534176&amp;link_uid=PMC2958340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>719.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g005/" ref="ordinalpos=719&amp;ncbi_uid=6143990&amp;link_uid=PMC4018296" href="/pmc/articles/PMC4018296/"><img src="/pmc/articles/instance/4018296/bin/pone.0096776.g005.gif" src-large="/pmc/articles/instance/4018296/bin/pone.0096776.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4018296/" ref="ordinalpos=719&amp;ncbi_uid=6143990&amp;link_uid=PMC4018296" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g005/" class="imagepopup">Figure 5. Comparison of <span class="highlight" style="background-color:">signaling</span> network among three cancer cell lines..  From: Chemical Genomic-Based <span class="highlight" style="background-color:">Pathway</span> Analyses for Epidermal Growth Factor-Mediated <span class="highlight" style="background-color:">Signaling</span> in Migrating Cancer Cells. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) Common EGF-induced <span class="highlight" style="background-color:">signaling</span> network among three cancer cell lines. (<b>b∼d</b>) Specific EGF-induced <span class="highlight" style="background-color:">signaling</span> network in (<b>b</b>) A431 cells, (<b>c</b>) EC109 cells, and (<b>d</b>) TT cells. The edge color is decided based on the sign of edges; red indicates positive <span class="highlight" style="background-color:">signaling</span>, and blue indicates negative <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Shigeyuki Magi, et al. PLoS One. 2014;9(5):e96776.</div><p class="links"><a class="dblinks" href="/pubmed/24820097" ref="ordinalpos=719&amp;ncbi_uid=6143990&amp;link_uid=24820097">Citation</a><a class="dblinks" href="/pmc/articles/PMC4018296/" ref="ordinalpos=719&amp;ncbi_uid=6143990&amp;link_uid=PMC4018296">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>720.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2712453/figure/F6/" ref="ordinalpos=720&amp;ncbi_uid=1925236&amp;link_uid=PMC2712453" href="/pmc/articles/PMC2712453/"><img src="/pmc/articles/instance/2712453/bin/1752-0509-3-67-6.gif" src-large="/pmc/articles/instance/2712453/bin/1752-0509-3-67-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2712453/" ref="ordinalpos=720&amp;ncbi_uid=1925236&amp;link_uid=PMC2712453" image-link="/pmc/articles/PMC2712453/figure/F6/" class="imagepopup">Figure 6.  From: BowTieBuilder: modeling signal transduction pathways. </a></p><div class="supp"><p class="details"><div><b>TLR-mediated innate immune <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. TLR-mediated innate immune-associated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inferred by BowTieBuilder. Membrane-bound TLR-proteins are depicted at the top, TFs are depicted at the bottom and the inferred proteins are depicted in-between. The proteins MyD88 and FADD are both in the core module and considered to be essential to this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jochen Supper, et al. BMC Syst Biol. 2009;3:67-67.</div><p class="links"><a class="dblinks" href="/pubmed/19566957" ref="ordinalpos=720&amp;ncbi_uid=1925236&amp;link_uid=19566957">Citation</a><a class="dblinks" href="/pmc/articles/PMC2712453/" ref="ordinalpos=720&amp;ncbi_uid=1925236&amp;link_uid=PMC2712453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>721.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163944/figure/F7/" ref="ordinalpos=721&amp;ncbi_uid=3193364&amp;link_uid=PMC3163944" href="/pmc/articles/PMC3163944/"><img src="/pmc/articles/instance/3163944/bin/JCI43428.f7.gif" src-large="/pmc/articles/instance/3163944/bin/JCI43428.f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163944/" ref="ordinalpos=721&amp;ncbi_uid=3193364&amp;link_uid=PMC3163944" image-link="/pmc/articles/PMC3163944/figure/F7/" class="imagepopup">Figure 7. The MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Disruption of the histone acetyltransferase MYST4 leads to a Noonan syndrome-like phenotype and hyperactivated MAPK <span class="highlight" style="background-color:">signaling</span> in humans and mice. </a></p><div class="supp"><p class="details"><div>Overview of genes with ChIP-CHIP binding sites and differential expression. Three out of the four major branches of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown: ERK1/2 (MAP2K1/2), c-Jun, and p38 (modified from KEGG <span class="highlight" style="background-color:">pathway</span> map 04010;  http://www.genome.jp/kegg/). Mutations affecting the ERK1/2 (MAP2K1/2) branch are identified in approximately 70% of patients with Noonan syndrome (genes marked with orange stars). Note that genes with significant ChIP-on-CHIP bindings sites (Supplemental Table 23) are marked in yellow; additional significantly differentially expressed genes in the human cell lines are marked with red boxes.    </div></p></div><div class="aux"><div class="resc">Michael Kraft, et al. J Clin Invest. 2011 September 1;121(9):3479-3491.</div><p class="links"><a class="dblinks" href="/pubmed/21804188" ref="ordinalpos=721&amp;ncbi_uid=3193364&amp;link_uid=21804188">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163944/" ref="ordinalpos=721&amp;ncbi_uid=3193364&amp;link_uid=PMC3163944">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>722.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3213171/figure/pcbi-1002262-g001/" ref="ordinalpos=722&amp;ncbi_uid=3420622&amp;link_uid=PMC3213171" href="/pmc/articles/PMC3213171/"><img src="/pmc/articles/instance/3213171/bin/pcbi.1002262.g001.gif" src-large="/pmc/articles/instance/3213171/bin/pcbi.1002262.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3213171/" ref="ordinalpos=722&amp;ncbi_uid=3420622&amp;link_uid=PMC3213171" image-link="/pmc/articles/PMC3213171/figure/pcbi-1002262-g001/" class="imagepopup">Figure 1. Graph image of the <span class="highlight" style="background-color:">signaling</span> network..  From: Identification of Biochemical Network Modules Based on Shortest Retroactive Distances. </a></p><div class="supp"><p class="details"><div>Each reaction in the network was <i>a priori</i> assigned to one of 11 canonical <span class="highlight" style="background-color:">signaling</span> pathways as described in Methods. The <span class="highlight" style="background-color:">pathway</span> assignments are indicated by the color of the reaction vertex in the network. (NA: not assigned; SGTP: small guanosine triphosphatase mediated <span class="highlight" style="background-color:">signaling</span>; PIP: phosphatidylinositol polyphosphate <span class="highlight" style="background-color:">signaling</span>; REC: recycling; ENDO: endocytosis; DEG: degradation; CELLC: cell cycle; MAPK: mitogen-activated protein kinase cascade; TRANS: transcription; CAS: Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span>; GPCR: G-protein coupled receptor mediated <span class="highlight" style="background-color:">signaling</span>; ERBB: erythroblastic leukemia viral oncogene homolog receptor <span class="highlight" style="background-color:">signaling</span>).</div></p></div><div class="aux"><div class="resc">Gautham Vivek Sridharan, et al. PLoS Comput Biol. 2011 November;7(11):e1002262.</div><p class="links"><a class="dblinks" href="/pubmed/22102800" ref="ordinalpos=722&amp;ncbi_uid=3420622&amp;link_uid=22102800">Citation</a><a class="dblinks" href="/pmc/articles/PMC3213171/" ref="ordinalpos=722&amp;ncbi_uid=3420622&amp;link_uid=PMC3213171">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>723.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3675894/figure/F1/" ref="ordinalpos=723&amp;ncbi_uid=5545709&amp;link_uid=PMC3675894" href="/pmc/articles/PMC3675894/"><img src="/pmc/articles/instance/3675894/bin/nihms393899f1.gif" src-large="/pmc/articles/instance/3675894/bin/nihms393899f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3675894/" ref="ordinalpos=723&amp;ncbi_uid=5545709&amp;link_uid=PMC3675894" image-link="/pmc/articles/PMC3675894/figure/F1/" class="imagepopup">Figure 1.  From: Cholesterol and its derivatives in Sonic Hedgehog <span class="highlight" style="background-color:">signaling</span> and Cancer. </a></p><div class="supp"><p class="details"><div>Overview of the HH <span class="highlight" style="background-color:">signaling</span> network. HH <span class="highlight" style="background-color:">signaling</span> is initiated upon binding of a HH protein to its receptor PTC, which de-represses SMO. In the canonical <span class="highlight" style="background-color:">pathway</span>, SMO activation leads to increased transcriptional activity of the GLI family of transcription factors. GLI-independent actions of PTC and SMO are collectively known as “non-canonical” <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Natalia A. Riobo. Curr Opin Pharmacol. ;12(6):736-741.</div><p class="links"><a class="dblinks" href="/pubmed/22832232" ref="ordinalpos=723&amp;ncbi_uid=5545709&amp;link_uid=22832232">Citation</a><a class="dblinks" href="/pmc/articles/PMC3675894/" ref="ordinalpos=723&amp;ncbi_uid=5545709&amp;link_uid=PMC3675894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>724.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3440553/figure/F1/" ref="ordinalpos=724&amp;ncbi_uid=5131764&amp;link_uid=PMC3440553" href="/pmc/articles/PMC3440553/"><img src="/pmc/articles/instance/3440553/bin/nihms387341f1.gif" src-large="/pmc/articles/instance/3440553/bin/nihms387341f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3440553/" ref="ordinalpos=724&amp;ncbi_uid=5131764&amp;link_uid=PMC3440553" image-link="/pmc/articles/PMC3440553/figure/F1/" class="imagepopup">Figure 1. TOR <span class="highlight" style="background-color:">signaling</span> and cardiomyopathy.  From: TOR-based therapy for cardiomyopathy: evidence from zebrafish and human studies. </a></p><div class="supp"><p class="details"><div><b>A)</b> Schematic of TOR <span class="highlight" style="background-color:">signaling</span>. TOR responds to growth factors via the PI3K-TSC-RheB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, to nutrients via RAG GTPase, and to energy via AMPK. TOR regulates downstream cellular events including protein synthesis and autophagy. <b>B)</b> Partial inhibition of TOR <span class="highlight" style="background-color:">signaling</span> via rapamycin attenuates cardiomyopathy, which can be pursued as a new therapeutic strategy.</div></p></div><div class="aux"><div class="resc">Sudhir Kushwaha, et al. Trends Cardiovasc Med. 2012 February;22(2):39-43.</div><p class="links"><a class="dblinks" href="/pubmed/22841839" ref="ordinalpos=724&amp;ncbi_uid=5131764&amp;link_uid=22841839">Citation</a><a class="dblinks" href="/pmc/articles/PMC3440553/" ref="ordinalpos=724&amp;ncbi_uid=5131764&amp;link_uid=PMC3440553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>725.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2764313/figure/F1/" ref="ordinalpos=725&amp;ncbi_uid=2100298&amp;link_uid=PMC2764313" href="/pmc/articles/PMC2764313/"><img src="/pmc/articles/instance/2764313/bin/nihms140690f1.gif" src-large="/pmc/articles/instance/2764313/bin/nihms140690f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2764313/" ref="ordinalpos=725&amp;ncbi_uid=2100298&amp;link_uid=PMC2764313" image-link="/pmc/articles/PMC2764313/figure/F1/" class="imagepopup">Fig. 1. IGF-1 <span class="highlight" style="background-color:">signaling</span> cascade.  From: Running on empty: how p53 controls INS/IGF <span class="highlight" style="background-color:">signaling</span> and affects lifespan. </a></p><div class="supp"><p class="details"><div><b>A</b>. The paradox of INS/IGF activity: high activity promotes growth but limits longevity.<br /><b>B</b>. <span class="highlight" style="background-color:">Signaling</span> is regulated by phosphorylation (yellow symbols) through a highly conserved <span class="highlight" style="background-color:">pathway</span>, which is shown here for mammalian IGF-1. Receptor activation by ligand binding generates PIP3 and can be blocked by PTEN. AKT phosphorylation turns on a growth <span class="highlight" style="background-color:">pathway</span> that acts through mTOR and is blocked by 4E-BP (not shown) and turns off FOXO activity in the nucleus, which otherwise causes cell cycle arrest.</div></p></div><div class="aux"><div class="resc">Heidi Scrable, et al. Exp Gerontol. ;44(1-2):93-100.</div><p class="links"><a class="dblinks" href="/pubmed/18598747" ref="ordinalpos=725&amp;ncbi_uid=2100298&amp;link_uid=18598747">Citation</a><a class="dblinks" href="/pmc/articles/PMC2764313/" ref="ordinalpos=725&amp;ncbi_uid=2100298&amp;link_uid=PMC2764313">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>726.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3127207/figure/F2/" ref="ordinalpos=726&amp;ncbi_uid=3069275&amp;link_uid=PMC3127207" href="/pmc/articles/PMC3127207/"><img src="/pmc/articles/instance/3614825/bin/IJBS-7-101_F2.gif" src-large="/pmc/articles/instance/3614825/bin/IJBS-7-101_F2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3127207/" ref="ordinalpos=726&amp;ncbi_uid=3069275&amp;link_uid=PMC3127207" image-link="/pmc/articles/PMC3127207/figure/F2/" class="imagepopup">Figure 2.  From: Luteolin Regulation of Estrogen <span class="highlight" style="background-color:">Signaling</span> and Cell Cycle <span class="highlight" style="background-color:">Pathway</span> Genes in MCF-7 Human Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>Luteolin Effects on Estrogen <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (ESP) Genes.</b> Ingenuity Analysis of microarray data from MCF-7 cells treated with estradiol (controls) or estradiol + luteolin for 5 hours. Genes in this <span class="highlight" style="background-color:">pathway</span> were significantly (<i>p</i>&lt;0.01) upregulated by luteolin (light blue symbols) or estradiol (shaded symbols) or significantly (<i>p</i>&lt;0.01) downregulated by luteolin (gray symbols) or estradiol (dotted symbols). These data are numerically summarized in Table 2.</div></p></div><div class="aux"><div class="resc">Barry M. Markaverich, et al. Int J Biomed Sci. 2011 June;7(2):101-111.</div><p class="links"><a class="dblinks" href="/pubmed/21731475" ref="ordinalpos=726&amp;ncbi_uid=3069275&amp;link_uid=21731475">Citation</a><a class="dblinks" href="/pmc/articles/PMC3127207/" ref="ordinalpos=726&amp;ncbi_uid=3069275&amp;link_uid=PMC3127207">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>727.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2758619/figure/F4/" ref="ordinalpos=727&amp;ncbi_uid=2463562&amp;link_uid=PMC2758619" href="/pmc/articles/PMC2758619/"><img src="/pmc/articles/instance/2758619/bin/nihms-135664-f0004.gif" src-large="/pmc/articles/instance/2758619/bin/nihms-135664-f0004.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2758619/figure/F4/" ref="ordinalpos=727&amp;ncbi_uid=2463562&amp;link_uid=PMC2758619" href="/pmc/articles/PMC2758619/"><img src="/pmc/articles/instance/2758619/bin/nihms-135664-f0005.gif" src-large="/pmc/articles/instance/2758619/bin/nihms-135664-f0005.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758619/" ref="ordinalpos=727&amp;ncbi_uid=2463562&amp;link_uid=PMC2758619" image-link="/pmc/articles/PMC2758619/figure/F4/" class="imagepopup">Figure 4. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in mammary tumors.  From: NEDD9 promotes oncogenic <span class="highlight" style="background-color:">signaling</span> in mammary tumor development. </a></p><div class="supp"><p class="details"><div><b>A</b>. <span class="highlight" style="background-color:">Signaling</span> interactions of <i>Nedd9</i> relevant to mammary tumor development. <b>B, C</b>. Western analysis demonstrates expression and activation levels of the indicated proteins in primary tumor lysates, and quantification of relative difference in activation between <i>Nedd9<sup>-/-</sup></i> and <i>Nedd9<sup>+/+</sup></i> mice. <b>D</b>. Degree of <span class="highlight" style="background-color:">pathway</span> activation as a correlate of initial tumor appearance (see data in Figures <b>1A, 1C</b>). The statistical trend is highly significant for time of initial detection of tumors (p&lt;0.001 for all proteins evaluated).</div></p></div><div class="aux"><div class="resc">Eugene Izumchenko, et al. Cancer Res. ;69(18):7198-7206.</div><p class="links"><a class="dblinks" href="/pubmed/19738060" ref="ordinalpos=727&amp;ncbi_uid=2463562&amp;link_uid=19738060">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758619/" ref="ordinalpos=727&amp;ncbi_uid=2463562&amp;link_uid=PMC2758619">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>728.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4158873/figure/F3/" ref="ordinalpos=728&amp;ncbi_uid=6644607&amp;link_uid=PMC4158873" href="/pmc/articles/PMC4158873/"><img src="/pmc/articles/instance/4158873/bin/fendo-05-00145-g003.gif" src-large="/pmc/articles/instance/4158873/bin/fendo-05-00145-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4158873/" ref="ordinalpos=728&amp;ncbi_uid=6644607&amp;link_uid=PMC4158873" image-link="/pmc/articles/PMC4158873/figure/F3/" class="imagepopup">Figure 3.  From: O-GlcNAcylation and Metabolic Reprograming in Cancer. </a></p><div class="supp"><p class="details"><div><b>PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. mTORC1 is activated by receptor <span class="highlight" style="background-color:">signaling</span> through the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. mTORC2 is crucial factor in PI3K/Akt <span class="highlight" style="background-color:">signaling</span>, phosphorylating Akt on Ser473 to promote its maximal activation. mTOR regulates protein <i>O</i>-GlcNAcylation through affecting OGT stability. Several proteins of PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> are modified by OGT, i.e., PI3K, PDK1, and Akt.</div></p></div><div class="aux"><div class="resc">Paweł Jóźwiak, et al. Front Endocrinol (Lausanne). 2014;5:145.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=728&amp;ncbi_uid=6644607&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4158873/" ref="ordinalpos=728&amp;ncbi_uid=6644607&amp;link_uid=PMC4158873">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>729.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3876736/figure/F6/" ref="ordinalpos=729&amp;ncbi_uid=5642793&amp;link_uid=PMC3876736" href="/pmc/articles/PMC3876736/"><img src="/pmc/articles/instance/3876736/bin/nihms536072f6.gif" src-large="/pmc/articles/instance/3876736/bin/nihms536072f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3876736/" ref="ordinalpos=729&amp;ncbi_uid=5642793&amp;link_uid=PMC3876736" image-link="/pmc/articles/PMC3876736/figure/F6/" class="imagepopup">Figure 6. Model.  From: The TSC-mTOR <span class="highlight" style="background-color:">pathway</span> regulates macrophage polarization. </a></p><div class="supp"><p class="details"><div>Proposed model for how mTORC1 activity controls macrophage polarization. <b>a.</b> Physiological induction of the Akt-mTORC1 <span class="highlight" style="background-color:">signaling</span> loop by IL-4 stimulation (left) allows for transient, inducible activation of the <span class="highlight" style="background-color:">pathway</span>, and enables Akt to synergize with the JAK/STAT <span class="highlight" style="background-color:">pathway</span> for M2 polarization. mTORC1 activity is also regulated by nutrient availability (not shown here), so such wiring of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may allow calibration of M2 activation to metabolic status (left). In contrast, constitutive or aberrant mTORC1 activation corrupts this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and modulation of macrophage activation by metabolic inputs (right). (Green = activation; Red = inhibition; Black = attenuated) <b>b.</b> Constitutive or aberrant activation of mTORC1 impairs the ability of macrophages to respond appropriately to polarizing stimuli. A critical mediator of this process is Akt, whose activity is downregulated by increased mTORC1 activity.</div></p></div><div class="aux"><div class="resc">Vanessa Byles, et al. Nat Commun. ;4:10.1038/ncomms3834.</div><p class="links"><a class="dblinks" href="/pubmed/24280772" ref="ordinalpos=729&amp;ncbi_uid=5642793&amp;link_uid=24280772">Citation</a><a class="dblinks" href="/pmc/articles/PMC3876736/" ref="ordinalpos=729&amp;ncbi_uid=5642793&amp;link_uid=PMC3876736">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>730.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3824689/figure/fig8/" ref="ordinalpos=730&amp;ncbi_uid=5464480&amp;link_uid=PMC3824689" href="/pmc/articles/PMC3824689/"><img src="/pmc/articles/instance/3824689/bin/cddis2013380f8.gif" src-large="/pmc/articles/instance/3824689/bin/cddis2013380f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3824689/" ref="ordinalpos=730&amp;ncbi_uid=5464480&amp;link_uid=PMC3824689" image-link="/pmc/articles/PMC3824689/figure/fig8/" class="imagepopup">Figure 8.  From: IBP-mediated suppression of autophagy promotes growth and metastasis of breast cancer cells via activating mTORC2/Akt/FOXO3a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>A putative modal for the mechanisms of how suppression of autophagy by IBP promotes breast cancer cell growth and metastasis. IBP could promote breast cancer growth and metastasis by suppressing autophagy through activating the mTORC2/Akt/FOXO3a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (measured as the phosphorylation of their substrates: mTOR at Ser2481, Akt at Ser473 and FOXO3a at Thr32)</div></p></div><div class="aux"><div class="resc">S Chen, et al. Cell Death Dis. 2013 October;4(10):e842.</div><p class="links"><a class="dblinks" href="/pubmed/24113176" ref="ordinalpos=730&amp;ncbi_uid=5464480&amp;link_uid=24113176">Citation</a><a class="dblinks" href="/pmc/articles/PMC3824689/" ref="ordinalpos=730&amp;ncbi_uid=5464480&amp;link_uid=PMC3824689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>731.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204345/figure/fig4/" ref="ordinalpos=731&amp;ncbi_uid=3314085&amp;link_uid=PMC3204345" href="/pmc/articles/PMC3204345/"><img src="/pmc/articles/instance/3204345/bin/carcinbgr172f04_4c.gif" src-large="/pmc/articles/instance/3204345/bin/carcinbgr172f04_4c.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204345/" ref="ordinalpos=731&amp;ncbi_uid=3314085&amp;link_uid=PMC3204345" image-link="/pmc/articles/PMC3204345/figure/fig4/" class="imagepopup">Fig. 4. From: Smad phosphoisoform <span class="highlight" style="background-color:">signaling</span> specificity: the right place at the right time. </a></p><div class="supp"><p class="details"><div>Invasive pSmad2L/C together with oncogenic pSmad3L pathways characterize TGF-β <span class="highlight" style="background-color:">signaling</span> shared between Ras-transformed cells and human advanced carcinomas. (<b>A</b>) Tumor cells must evade pSmad3C-imposed growth inhibition if they are to thrive. More specifically, such tumor cells depend on highly activated Ras/pSmad3L <span class="highlight" style="background-color:">signaling</span> to drive their proliferation. Selective blockade of linker phosphorylation can suppress the Ras/pSmad3L <span class="highlight" style="background-color:">pathway</span>, while restoring the lost cytostatic pSmad3C <span class="highlight" style="background-color:">signaling</span> as present in mature epithelial cells. Later, in tumor development, TGF-β transmits an invasive signal via the pSmad2L/C <span class="highlight" style="background-color:">pathway</span> together with oncogenic pSmad3L <span class="highlight" style="background-color:">signaling</span>. (<b>B</b>) During human carcinogenic process, benign tumors affected by somatic mutations including <i>Ras</i> undergo transition from the tumor-suppressive pSmad3C <span class="highlight" style="background-color:">pathway</span> to the oncogenic pSmad3L <span class="highlight" style="background-color:">pathway</span>, becoming carcinoma <i>in situ</i>. Advanced carcinomas acquire more invasive and proliferative properties via the pSmad2L/C and pSmad3L pathways when carcinoma <i>in situ</i> receives TGF-β signals from the reactive stroma.</div></p></div><div class="aux"><div class="resc">Koichi Matsuzaki. Carcinogenesis. 2011 November;32(11):1578-1588.</div><p class="links"><a class="dblinks" href="/pubmed/21798854" ref="ordinalpos=731&amp;ncbi_uid=3314085&amp;link_uid=21798854">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204345/" ref="ordinalpos=731&amp;ncbi_uid=3314085&amp;link_uid=PMC3204345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>732.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2963150/figure/F5/" ref="ordinalpos=732&amp;ncbi_uid=2531413&amp;link_uid=PMC2963150" href="/pmc/articles/PMC2963150/"><img src="/pmc/articles/instance/2963150/bin/nihms239332f5.gif" src-large="/pmc/articles/instance/2963150/bin/nihms239332f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2963150/" ref="ordinalpos=732&amp;ncbi_uid=2531413&amp;link_uid=PMC2963150" image-link="/pmc/articles/PMC2963150/figure/F5/" class="imagepopup">Figure 5.  From: Role of Mitochondrial Dysfunction in Insulin Resistance. </a></p><div class="supp"><p class="details"><div>Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The metabolic PI3K branch of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and tissue-specific actions of insulin are shown. PI3K branch of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> plays a major role in gluconeogenesis in the liver, enhances NO production in the endothelium and heart, and glucose uptake in skeletal muscle, adipose tissue, and heart.</div></p></div><div class="aux"><div class="resc">Jeong-a Kim, et al. Circ Res. ;102(4):401-414.</div><p class="links"><a class="dblinks" href="/pubmed/18309108" ref="ordinalpos=732&amp;ncbi_uid=2531413&amp;link_uid=18309108">Citation</a><a class="dblinks" href="/pmc/articles/PMC2963150/" ref="ordinalpos=732&amp;ncbi_uid=2531413&amp;link_uid=PMC2963150">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>733.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3890654/figure/F1/" ref="ordinalpos=733&amp;ncbi_uid=5730019&amp;link_uid=PMC3890654" href="/pmc/articles/PMC3890654/"><img src="/pmc/articles/instance/3890654/bin/fimmu-04-00512-g001.gif" src-large="/pmc/articles/instance/3890654/bin/fimmu-04-00512-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3890654/" ref="ordinalpos=733&amp;ncbi_uid=5730019&amp;link_uid=PMC3890654" image-link="/pmc/articles/PMC3890654/figure/F1/" class="imagepopup">Figure 1.  From: Modulation of Intestinal TLR4-Inflammatory <span class="highlight" style="background-color:">Signaling</span> Pathways by Probiotic Microorganisms: Lessons Learned from Lactobacillus jensenii TL2937. </a></p><div class="supp"><p class="details"><div><b>Toll-like receptor-4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Julio Villena, et al. Front Immunol. 2013;4:512.</div><p class="links"><a class="dblinks" href="/pubmed/24459463" ref="ordinalpos=733&amp;ncbi_uid=5730019&amp;link_uid=24459463">Citation</a><a class="dblinks" href="/pmc/articles/PMC3890654/" ref="ordinalpos=733&amp;ncbi_uid=5730019&amp;link_uid=PMC3890654">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>734.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3895447/figure/F3/" ref="ordinalpos=734&amp;ncbi_uid=5718503&amp;link_uid=PMC3895447" href="/pmc/articles/PMC3895447/"><img src="/pmc/articles/instance/3895447/bin/nihms543951f3a.gif" src-large="/pmc/articles/instance/3895447/bin/nihms543951f3a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3895447/figure/F3/" ref="ordinalpos=734&amp;ncbi_uid=5718503&amp;link_uid=PMC3895447" href="/pmc/articles/PMC3895447/"><img src="/pmc/articles/instance/3895447/bin/nihms543951f3b.gif" src-large="/pmc/articles/instance/3895447/bin/nihms543951f3b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3895447/" ref="ordinalpos=734&amp;ncbi_uid=5718503&amp;link_uid=PMC3895447" image-link="/pmc/articles/PMC3895447/figure/F3/" class="imagepopup">Figure 3.  From: Wires in the soup: quantitative models of cell <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The role of scaffolds in cell <span class="highlight" style="background-color:">signaling</span>. (A) In many types of <span class="highlight" style="background-color:">signaling</span> pathways, scaffolds serve to bring together most of the members of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and localize them close to the receptor. This cross-linking property of scaffold molecules ensures that there is an optimal scaffold concentration for the given concentrations of other <span class="highlight" style="background-color:">signaling</span> molecules, potentially allowing for controlling where in the cell the <span class="highlight" style="background-color:">signaling</span> will be most efficient; (B) Preferential localization of scaffolds to particular membrane compartments can ensure that the <span class="highlight" style="background-color:">signaling</span> is relatively high only around those compartments but not elsewhere, the property that might be crucial for cell polarization and directed cell extension or migration. This image shows a <span class="highlight" style="background-color:">signaling</span> scaffold at the leading edge of a migrating cell and the subsequent compartmentalization to the leading edge of its signal (depicted as green gradient).</div></p></div><div class="aux"><div class="resc">Raymond Cheong, et al. Trends Cell Biol. ;18(3):112-118.</div><p class="links"><a class="dblinks" href="/pubmed/18291655" ref="ordinalpos=734&amp;ncbi_uid=5718503&amp;link_uid=18291655">Citation</a><a class="dblinks" href="/pmc/articles/PMC3895447/" ref="ordinalpos=734&amp;ncbi_uid=5718503&amp;link_uid=PMC3895447">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>735.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2491722/figure/F6/" ref="ordinalpos=735&amp;ncbi_uid=1512329&amp;link_uid=PMC2491722" href="/pmc/articles/PMC2491722/"><img src="/pmc/articles/instance/2491722/bin/nihms57217f6.gif" src-large="/pmc/articles/instance/2491722/bin/nihms57217f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2491722/" ref="ordinalpos=735&amp;ncbi_uid=1512329&amp;link_uid=PMC2491722" image-link="/pmc/articles/PMC2491722/figure/F6/" class="imagepopup">Figure 6. GDNF/Ras <span class="highlight" style="background-color:">signaling</span> in spermatogonial stem cells.  From: Gdnf <span class="highlight" style="background-color:">signaling</span> pathways within the mammalian spermatogonial stem cell niche. </a></p><div class="supp"><p class="details"><div>The schematic diagram demonstrates intracellular <span class="highlight" style="background-color:">signaling</span> events in the Ras/ERK1/2 <span class="highlight" style="background-color:">pathway</span> as well as the downstream cascades activated by GDNF in spermatogonial stem cells. “P” indicates “phosphorylate”, and “A” denotes “activate”. From He et al, Stem Cells, 2008 (He et al., 2008).</div></p></div><div class="aux"><div class="resc">Marie-Claude Hofmann. Mol Cell Endocrinol. ;288(1-2):95-103.</div><p class="links"><a class="dblinks" href="/pubmed/18485583" ref="ordinalpos=735&amp;ncbi_uid=1512329&amp;link_uid=18485583">Citation</a><a class="dblinks" href="/pmc/articles/PMC2491722/" ref="ordinalpos=735&amp;ncbi_uid=1512329&amp;link_uid=PMC2491722">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>736.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3518872/figure/F4/" ref="ordinalpos=736&amp;ncbi_uid=4810201&amp;link_uid=PMC3518872" href="/pmc/articles/PMC3518872/"><img src="/pmc/articles/instance/3518872/bin/nihms-416085-f0004.gif" src-large="/pmc/articles/instance/3518872/bin/nihms-416085-f0004.jpg" alt="Figure4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3518872/" ref="ordinalpos=736&amp;ncbi_uid=4810201&amp;link_uid=PMC3518872" image-link="/pmc/articles/PMC3518872/figure/F4/" class="imagepopup">Figure4.  From: Inflammatory Mediators Alter the Astrocyte Transcriptome and Calcium <span class="highlight" style="background-color:">Signaling</span> Elicited by Multiple G-Protein-Coupled Receptors. </a></p><div class="supp"><p class="details"><div>Three IPA Canonical <span class="highlight" style="background-color:">Signaling</span> Pathways: CD40, IL-1, and Chemokine <span class="highlight" style="background-color:">Signaling</span> in TGF-<i>β</i>1+LPS+IFN<i>γ</i>-treated astrocytes. <i><b>A–C</b></i>, Schematic diagrams of three of the top significantly altered predesignated Canonical <span class="highlight" style="background-color:">Signaling</span> identified using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software (IPA; Ingenuity) as being affected by TGF-<i>β</i>1+LPS+IFN<i>γ</i>-treatment of astrocytes. As selected from Table 5, these pathways are CD40 <span class="highlight" style="background-color:">Signaling</span> (<i><b>A</b></i>), IL-1 <span class="highlight" style="background-color:">Signaling</span> (<i><b>B</b></i>), and Chemokine <span class="highlight" style="background-color:">Signaling</span> (<i><b>C</b></i>). Molecules are indicated by standard abbreviations. Relative changes in gene expression are depicted by gradated shades of color coding: red, up; green, down; white, no change or not applicable. Direct and indirect interactions between molecules are depicted by solid and dotted lines, respectively.</div></p></div><div class="aux"><div class="resc">Mary E. Hamby, et al. J Neurosci. ;32(42):14489-14510.</div><p class="links"><a class="dblinks" href="/pubmed/23077035" ref="ordinalpos=736&amp;ncbi_uid=4810201&amp;link_uid=23077035">Citation</a><a class="dblinks" href="/pmc/articles/PMC3518872/" ref="ordinalpos=736&amp;ncbi_uid=4810201&amp;link_uid=PMC3518872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>737.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3278757/figure/F5/" ref="ordinalpos=737&amp;ncbi_uid=4610601&amp;link_uid=PMC3278757" href="/pmc/articles/PMC3278757/"><img src="/pmc/articles/instance/3278757/bin/btr696f5.gif" src-large="/pmc/articles/instance/3278757/bin/btr696f5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3278757/" ref="ordinalpos=737&amp;ncbi_uid=4610601&amp;link_uid=PMC3278757" image-link="/pmc/articles/PMC3278757/figure/F5/" class="imagepopup">Fig. 5. From: Optimal structural inference of <span class="highlight" style="background-color:">signaling</span> pathways from unordered and overlapping gene sets. </a></p><div class="supp"><p class="details"><div>Performance of SA in reconstructing the true <span class="highlight" style="background-color:">signaling</span> cascades and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> structures corresponding to 83 IFGS compendiums derived from the KEGG database.</div></p></div><div class="aux"><div class="resc">Lipi R. Acharya, et al. Bioinformatics. 2012 February 15;28(4):546-556.</div><p class="links"><a class="dblinks" href="/pubmed/22199386" ref="ordinalpos=737&amp;ncbi_uid=4610601&amp;link_uid=22199386">Citation</a><a class="dblinks" href="/pmc/articles/PMC3278757/" ref="ordinalpos=737&amp;ncbi_uid=4610601&amp;link_uid=PMC3278757">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>738.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3342643/figure/F3/" ref="ordinalpos=738&amp;ncbi_uid=3977191&amp;link_uid=PMC3342643" href="/pmc/articles/PMC3342643/"><img src="/pmc/articles/instance/3342643/bin/nihms-373396-f0003.gif" src-large="/pmc/articles/instance/3342643/bin/nihms-373396-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3342643/" ref="ordinalpos=738&amp;ncbi_uid=3977191&amp;link_uid=PMC3342643" image-link="/pmc/articles/PMC3342643/figure/F3/" class="imagepopup">Figure 3. Schematic illustration of TCR <span class="highlight" style="background-color:">signaling</span>.  From: Diacylglycerol kinases in immune cell function and self-tolerance. </a></p><div class="supp"><p class="details"><div>Following TCR engagement, a critical event is the activation of PLCγ following the orchestrated action of proximal tyrosine kinases and adaptor molecules. Activated PLCγ1 generates IP<sub>3</sub> and DAG, leading to the activation of the Ca<sup>++</sup>-calcineurin-NFAT <span class="highlight" style="background-color:">pathway</span>, the RasGPR1-Ras-Erk1/2-AP1 <span class="highlight" style="background-color:">pathway</span>, the PKCθ-Carma1/Bcl10-NFκB <span class="highlight" style="background-color:">pathway</span>, and PKD. NFAT alone can induce T cell anergy by increasing expression of anergy promoting molecules including DGKα. This <span class="highlight" style="background-color:">pathway</span>, in concert with DAG-mediated pathways, leads to T cell activation. CD28 strengthens TCR <span class="highlight" style="background-color:">signaling</span> to promote PI3K activation and downregulates DGKα and ζ expression. DGKs convert DAG to PA and may inhibit DAG-mediated <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Xiao-Ping Zhong, et al. Immunol Rev. ;224:249-264.</div><p class="links"><a class="dblinks" href="/pubmed/18759932" ref="ordinalpos=738&amp;ncbi_uid=3977191&amp;link_uid=18759932">Citation</a><a class="dblinks" href="/pmc/articles/PMC3342643/" ref="ordinalpos=738&amp;ncbi_uid=3977191&amp;link_uid=PMC3342643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>739.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2435278/figure/ppat-1000098-g002/" ref="ordinalpos=739&amp;ncbi_uid=1362781&amp;link_uid=PMC2435278" href="/pmc/articles/PMC2435278/"><img src="/pmc/articles/instance/2435278/bin/ppat.1000098.g002.gif" src-large="/pmc/articles/instance/2435278/bin/ppat.1000098.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2435278/" ref="ordinalpos=739&amp;ncbi_uid=1362781&amp;link_uid=PMC2435278" image-link="/pmc/articles/PMC2435278/figure/ppat-1000098-g002/" class="imagepopup">Figure 2. Regulation of putative Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes by dengue virus infection..  From: The Aedes aegypti Toll <span class="highlight" style="background-color:">Pathway</span> Controls Dengue Virus Infection. </a></p><div class="supp"><p class="details"><div>Red color indicates infection responsive up-regulation and green color indicate infection responsive down-regulation. Non-colored gene boxes indicate lack of infection responsive regulation. The <span class="highlight" style="background-color:">pathway</span> was built with GenMapp software based on the immunogenomics prediction by Waterhouse <i>et al</i> 2007.</div></p></div><div class="aux"><div class="resc">Zhiyong Xi, et al. PLoS Pathog. 2008 July;4(7):e1000098.</div><p class="links"><a class="dblinks" href="/pubmed/18604274" ref="ordinalpos=739&amp;ncbi_uid=1362781&amp;link_uid=18604274">Citation</a><a class="dblinks" href="/pmc/articles/PMC2435278/" ref="ordinalpos=739&amp;ncbi_uid=1362781&amp;link_uid=PMC2435278">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>740.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2764141/figure/pone-0007622-g006/" ref="ordinalpos=740&amp;ncbi_uid=2098344&amp;link_uid=PMC2764141" href="/pmc/articles/PMC2764141/"><img src="/pmc/articles/instance/2764141/bin/pone.0007622.g006.gif" src-large="/pmc/articles/instance/2764141/bin/pone.0007622.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2764141/" ref="ordinalpos=740&amp;ncbi_uid=2098344&amp;link_uid=PMC2764141" image-link="/pmc/articles/PMC2764141/figure/pone-0007622-g006/" class="imagepopup">Figure 6. ROC (Receiver Operating Characteristic) analysis of threshold assignment to the AIs-Deletions matrix of the mating <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Reverse Engineering a <span class="highlight" style="background-color:">Signaling</span> Network Using Alternative Inputs. </a></p><div class="supp"><p class="details"><div>(A) Histogram of the AIs-Deletions matrix values for the mating <span class="highlight" style="background-color:">pathway</span> example. Values were placed in bins of size 10. The proportion of the total number of on and off values are indicated for each bin. (B) The ROC curve for the experimental data of the AIs-Deletions matrix of the mating <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This curve represents the false positive rates (FPR) and true positive rates (TPR) for a range of threshold values. FPR is defined as FP (False positive)/(FP + TN (true negative)), and TPR is defined as TP (true positive)/(TP + FN (false negative)). Selected threshold values are shown on the curve.</div></p></div><div class="aux"><div class="resc">Hiromasa Tanaka, et al. PLoS One. 2009;4(10):e7622.</div><p class="links"><a class="dblinks" href="/pubmed/19898612" ref="ordinalpos=740&amp;ncbi_uid=2098344&amp;link_uid=19898612">Citation</a><a class="dblinks" href="/pmc/articles/PMC2764141/" ref="ordinalpos=740&amp;ncbi_uid=2098344&amp;link_uid=PMC2764141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>741.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3026562/figure/F4/" ref="ordinalpos=741&amp;ncbi_uid=3626084&amp;link_uid=PMC3026562" href="/pmc/articles/PMC3026562/"><img src="/pmc/articles/instance/3026562/bin/zh70011135550004.gif" src-large="/pmc/articles/instance/3026562/bin/zh70011135550004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3026562/" ref="ordinalpos=741&amp;ncbi_uid=3626084&amp;link_uid=PMC3026562" image-link="/pmc/articles/PMC3026562/figure/F4/" class="imagepopup">Fig. 4. From: Whole transcriptome analysis of the fasting and fed Burmese python heart: insights into extreme physiological cardiac adaptation. </a></p><div class="supp"><p class="details"><div>Canonical <span class="highlight" style="background-color:">signaling</span> pathways shown to be activated through Ingenuity Pathways Analysis (IPA). IPA highlighted 8 <span class="highlight" style="background-color:">signaling</span> pathways involved in 2 types of hypertrophic responses. The data are plotted as the −log<sub>2</sub> of the <i>P</i> value given during the analysis, meaning larger values correspond to more significant <span class="highlight" style="background-color:">pathway</span> activation. The pathological pathways are: nuclear factor of activated T-cell <span class="highlight" style="background-color:">signaling</span> (NFAT), nitric oxide <span class="highlight" style="background-color:">signaling</span> (NO), cardiac hypoxia <span class="highlight" style="background-color:">signaling</span> (Hypoxia), and mitochondrial dysfunction <span class="highlight" style="background-color:">signaling</span> (Mito. Dys.). The physiological pathways are: mammalian target of rapamycin <span class="highlight" style="background-color:">signaling</span> (mTOR), focal adhesion kinase-1 <span class="highlight" style="background-color:">signaling</span> (FAK), NF-E2 related factor 2 <span class="highlight" style="background-color:">signaling</span> (NFR2), and ras homolog member A <span class="highlight" style="background-color:">signaling</span> (RhoA).</div></p></div><div class="aux"><div class="resc">Christopher E. Wall, et al. Physiol Genomics. 2011 January;43(2):69-76.</div><p class="links"><a class="dblinks" href="/pubmed/21045117" ref="ordinalpos=741&amp;ncbi_uid=3626084&amp;link_uid=21045117">Citation</a><a class="dblinks" href="/pmc/articles/PMC3026562/" ref="ordinalpos=741&amp;ncbi_uid=3626084&amp;link_uid=PMC3026562">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>742.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399428/figure/fig2/" ref="ordinalpos=742&amp;ncbi_uid=4136445&amp;link_uid=PMC3399428" href="/pmc/articles/PMC3399428/"><img src="/pmc/articles/instance/3399428/bin/MBI2012-307628.002.gif" src-large="/pmc/articles/instance/3399428/bin/MBI2012-307628.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399428/" ref="ordinalpos=742&amp;ncbi_uid=4136445&amp;link_uid=PMC3399428" image-link="/pmc/articles/PMC3399428/figure/fig2/" class="imagepopup">Figure 2.  From: Hippo and rassf1a Pathways: A Growing Affair. </a></p><div class="supp"><p class="details"><div>“Hpo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Mammals and the cross-talk with RASFF1A <span class="highlight" style="background-color:">signaling</span>.” Schematic representation of mammalian Hippo kinases cascade and interconnections between Hippo <span class="highlight" style="background-color:">pathway</span> and <i>rassf1a</i> protein signal. Red lines indicate the impact of <i>rassf1a</i> <span class="highlight" style="background-color:">signaling</span> in modulating activity of Hpo <span class="highlight" style="background-color:">pathway</span> components.</div></p></div><div class="aux"><div class="resc">Francesca Fausti, et al. Mol Biol Int. 2012;2012:307628.</div><p class="links"><a class="dblinks" href="/pubmed/22830020" ref="ordinalpos=742&amp;ncbi_uid=4136445&amp;link_uid=22830020">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399428/" ref="ordinalpos=742&amp;ncbi_uid=4136445&amp;link_uid=PMC3399428">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>743.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3079023/figure/F1/" ref="ordinalpos=743&amp;ncbi_uid=3393976&amp;link_uid=PMC3079023" href="/pmc/articles/PMC3079023/"><img src="/pmc/articles/instance/3079023/bin/nihms268047f1.gif" src-large="/pmc/articles/instance/3079023/bin/nihms268047f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3079023/" ref="ordinalpos=743&amp;ncbi_uid=3393976&amp;link_uid=PMC3079023" image-link="/pmc/articles/PMC3079023/figure/F1/" class="imagepopup">Figure 1. CXCL12 <span class="highlight" style="background-color:">pathway</span>.  From: CXCL12 (SDF1?) - CXCR4/CXCR7 <span class="highlight" style="background-color:">Pathway</span> Inhibition: An Emerging Sensitizer for Anti-Cancer Therapies?. </a></p><div class="supp"><p class="details"><div>CXCL12 binds to CXCR4 and CXCR7, which are G protein-coupled receptors (GPCR) that can form homodimers or heterodimers. In the latter case, CXCR7 changes the conformation of the CXCR4/G-protein complexes and abrogates <span class="highlight" style="background-color:">signaling</span>. Activation of CXCR4 by CXCL12 leads to G-protein coupled <span class="highlight" style="background-color:">signaling</span> through PI3K/Akt, IP3, and MAPK pathways, which promote cell survival, proliferation and chemotaxis. In addition, the β-arrestin <span class="highlight" style="background-color:">pathway</span> can be activated through GRK to internalize CXCR4. When CXCR7 binds CXCL12, the classical GPCR mobilization of Ca<sup>2+</sup> does not occur, and activation of the β-arrestin <span class="highlight" style="background-color:">pathway</span> may lead to scavenging of CXCL12. In certain cancer cells (e.g., glioma) CXCR7 can also signal through PLC/MAPK to increase cell survival.</div></p></div><div class="aux"><div class="resc">Dan G. Duda, et al. Clin Cancer Res. ;17(8):2074-2080.</div><p class="links"><a class="dblinks" href="/pubmed/21349998" ref="ordinalpos=743&amp;ncbi_uid=3393976&amp;link_uid=21349998">Citation</a><a class="dblinks" href="/pmc/articles/PMC3079023/" ref="ordinalpos=743&amp;ncbi_uid=3393976&amp;link_uid=PMC3079023">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>744.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3251651/figure/F4/" ref="ordinalpos=744&amp;ncbi_uid=4479909&amp;link_uid=PMC3251651" href="/pmc/articles/PMC3251651/"><img src="/pmc/articles/instance/3251651/bin/nihms339102f4.gif" src-large="/pmc/articles/instance/3251651/bin/nihms339102f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3251651/" ref="ordinalpos=744&amp;ncbi_uid=4479909&amp;link_uid=PMC3251651" image-link="/pmc/articles/PMC3251651/figure/F4/" class="imagepopup">Fig. 4. OTEs and off-targets of raloxifene, tamoxifen, paclitaxel, and fulvestrant on the cell cycle G1/S checkpoint and P53 <span class="highlight" style="background-color:">signaling</span> pathways.  From: A novel method of transcriptional response analysis to facilitate drug repositioning for cancer therapy. </a></p><div class="supp"><p class="details"><div>The right side is for the signal cascade in the cell cycle G1/S checkpoint <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> while the left side is for the P53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The drug-dose pairs are listed in the middle. The drug-targeted <span class="highlight" style="background-color:">pathway</span> was generated using IPA software (Ingenuity Systems, Inc. Redwood City, CA).</div></p></div><div class="aux"><div class="resc">Guangxu Jin, et al. Cancer Res. ;72(1):33-44.</div><p class="links"><a class="dblinks" href="/pubmed/22108825" ref="ordinalpos=744&amp;ncbi_uid=4479909&amp;link_uid=22108825">Citation</a><a class="dblinks" href="/pmc/articles/PMC3251651/" ref="ordinalpos=744&amp;ncbi_uid=4479909&amp;link_uid=PMC3251651">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>745.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3481433/figure/F7/" ref="ordinalpos=745&amp;ncbi_uid=4318600&amp;link_uid=PMC3481433" href="/pmc/articles/PMC3481433/"><img src="/pmc/articles/instance/3481433/bin/1471-2164-13-472-7.gif" src-large="/pmc/articles/instance/3481433/bin/1471-2164-13-472-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3481433/" ref="ordinalpos=745&amp;ncbi_uid=4318600&amp;link_uid=PMC3481433" image-link="/pmc/articles/PMC3481433/figure/F7/" class="imagepopup">Figure 7.  From: Transcriptome classification reveals molecular subtypes in psoriasis. </a></p><div class="supp"><p class="details"><div><b>Text mining results for validation process according to the literature.</b> Co-occurrence of gene names with disease-related terms, such as”psoriasis”, “NK cells”, “T cells”, “immune response”, “Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>”, “Notch signalling <span class="highlight" style="background-color:">pathway</span>”, “TGF – beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” and “ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” was searched in Pubmed abstracts through PubMatrix.</div></p></div><div class="aux"><div class="resc">Chrysanthi Ainali, et al. BMC Genomics. 2012;13:472-472.</div><p class="links"><a class="dblinks" href="/pubmed/22971201" ref="ordinalpos=745&amp;ncbi_uid=4318600&amp;link_uid=22971201">Citation</a><a class="dblinks" href="/pmc/articles/PMC3481433/" ref="ordinalpos=745&amp;ncbi_uid=4318600&amp;link_uid=PMC3481433">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>746.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC469033/figure/F2/" ref="ordinalpos=746&amp;ncbi_uid=443164&amp;link_uid=PMC469033" href="/pmc/articles/PMC469033/"><img src="/pmc/articles/instance/469033/bin/jbiol9-2.gif" src-large="/pmc/articles/instance/469033/bin/jbiol9-2.jpg" alt="The Background" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC469033/" ref="ordinalpos=746&amp;ncbi_uid=443164&amp;link_uid=PMC469033" image-link="/pmc/articles/PMC469033/figure/F2/" class="imagepopup">The Background.  From: Wnt <span class="highlight" style="background-color:">signaling</span> and the developmental fate of lung cells. </a></p><div class="supp"><p class="details"><div>Tadashi Okubo and Brigid Hogan [1] report in this issue of <i>Journal of Biology </i>a surprising result from their studies of the <b>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>, a central cell-cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during development (see the 'Background' box). If a key component of this <span class="highlight" style="background-color:">pathway</span> is expressed in active form in the lungs of developing transgenic mouse embryos, cells appear within the lung that are more like cells of the gut than they are like their lung neighbors.</div></p></div><div class="aux"><div class="resc">Julie Clayton. J Biol. 2004;3(3):9-9.</div><p class="links"><a class="dblinks" href="/pubmed/15225354" ref="ordinalpos=746&amp;ncbi_uid=443164&amp;link_uid=15225354">Citation</a><a class="dblinks" href="/pmc/articles/PMC469033/" ref="ordinalpos=746&amp;ncbi_uid=443164&amp;link_uid=PMC469033">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>747.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3754916/figure/pone-0073004-g003/" ref="ordinalpos=747&amp;ncbi_uid=5279552&amp;link_uid=PMC3754916" href="/pmc/articles/PMC3754916/"><img src="/pmc/articles/instance/3754916/bin/pone.0073004.g003.gif" src-large="/pmc/articles/instance/3754916/bin/pone.0073004.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3754916/" ref="ordinalpos=747&amp;ncbi_uid=5279552&amp;link_uid=PMC3754916" image-link="/pmc/articles/PMC3754916/figure/pone-0073004-g003/" class="imagepopup">Figure 3. miR-33a decreases the post-transcriptional activity of β-catenin and related genes in the β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Aflatoxin B1 Negatively Regulates Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> through Activating miR-33a. </a></p><div class="supp"><p class="details"><div>After transfection of pSilence 4.1-miR-33a or pSilence 4.1 vector (NC) into Chang liver and HepG2 cells for 48 h, cells were collected for the extraction of total RNA for real-time PCR. mRNA levels of all genes were normalized to β2-MG (control). (A) Relative expression level of miR-33a was conﬁrmed by way of real-time PCR. (B) Overexpression of miR-33a down-regulates mRNA levels of β-catenin in Chang liver and HepG2 cells. mRNA expression patterns of the related genes in β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were analyzed in Chang liver (C) and HepG2 (D) cells with miR-33a overexpression. Data is presented as mean ± SD. *P &lt; 0.05, **P &lt; 0.01, ***P &lt;0.001.</div></p></div><div class="aux"><div class="resc">Yi Fang, et al. PLoS One. 2013;8(8):e73004.</div><p class="links"><a class="dblinks" href="/pubmed/24015284" ref="ordinalpos=747&amp;ncbi_uid=5279552&amp;link_uid=24015284">Citation</a><a class="dblinks" href="/pmc/articles/PMC3754916/" ref="ordinalpos=747&amp;ncbi_uid=5279552&amp;link_uid=PMC3754916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>748.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2967656/figure/F1/" ref="ordinalpos=748&amp;ncbi_uid=2560781&amp;link_uid=PMC2967656" href="/pmc/articles/PMC2967656/"><img src="/pmc/articles/instance/2967656/bin/nihms231921f1.gif" src-large="/pmc/articles/instance/2967656/bin/nihms231921f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2967656/" ref="ordinalpos=748&amp;ncbi_uid=2560781&amp;link_uid=PMC2967656" image-link="/pmc/articles/PMC2967656/figure/F1/" class="imagepopup">Figure 1. An overview of some ghrelin and GHS-R1a down-stream <span class="highlight" style="background-color:">signaling</span> pathways.  From: Ghrelin receptor <span class="highlight" style="background-color:">signaling</span>: a promising therapeutic target for metabolic syndrome and cognitive dysfunction. </a></p><div class="supp"><p class="details"><div>These <span class="highlight" style="background-color:">signaling</span> pathways include the dominant growth hormone secretagogue-related PLC/PKC/IP3 <span class="highlight" style="background-color:">pathway</span>, hypothalamic ghrelin activation of NPY-containing neurons and inhibition of POMC neurons, and ghrelin augmentation of dopamine <span class="highlight" style="background-color:">signaling</span>, involving GHS-R1a/D1R heterodimerization, leading to a synergistic activation of AC2 via Gαs (D1Rcoupling) and βγ-subunits derived from Gαi (GHS-R1a coupling). DAG = diacylglycerol, IP3 =Inositol(1,4,5)triphosphate, POMC = pro-opiomelanocortin, D1R = dopamine receptor 1, AC2 = adenylyl cyclase-2</div></p></div><div class="aux"><div class="resc">Wei-na Cong, et al. CNS Neurol Disord Drug Targets. ;9(5):557-563.</div><p class="links"><a class="dblinks" href="/pubmed/20632971" ref="ordinalpos=748&amp;ncbi_uid=2560781&amp;link_uid=20632971">Citation</a><a class="dblinks" href="/pmc/articles/PMC2967656/" ref="ordinalpos=748&amp;ncbi_uid=2560781&amp;link_uid=PMC2967656">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>749.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3400665/figure/pone-0041426-g008/" ref="ordinalpos=749&amp;ncbi_uid=4136712&amp;link_uid=PMC3400665" href="/pmc/articles/PMC3400665/"><img src="/pmc/articles/instance/3400665/bin/pone.0041426.g008.gif" src-large="/pmc/articles/instance/3400665/bin/pone.0041426.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3400665/" ref="ordinalpos=749&amp;ncbi_uid=4136712&amp;link_uid=PMC3400665" image-link="/pmc/articles/PMC3400665/figure/pone-0041426-g008/" class="imagepopup">Figure 8. The major results and conclusion of this study were summarized..  From: Elevated CO2 Reduces the Resistance and Tolerance of Tomato Plants to Helicoverpa armigera by Suppressing the JA <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Elevated CO<sub>2</sub> decreased resistance and tolerance of WT to <i>H. armigera</i>. In contrast, the resistance and tolerance of <i>spr2</i> were not changed by elevated CO<sub>2</sub>. Elevated CO<sub>2</sub> reduces the resistance and tolerance of WT plants by suppressing the JA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Huijuan Guo, et al. PLoS One. 2012;7(7):e41426.</div><p class="links"><a class="dblinks" href="/pubmed/22829948" ref="ordinalpos=749&amp;ncbi_uid=4136712&amp;link_uid=22829948">Citation</a><a class="dblinks" href="/pmc/articles/PMC3400665/" ref="ordinalpos=749&amp;ncbi_uid=4136712&amp;link_uid=PMC3400665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>750.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2962609/figure/F10/" ref="ordinalpos=750&amp;ncbi_uid=2528960&amp;link_uid=PMC2962609" href="/pmc/articles/PMC2962609/"><img src="/pmc/articles/instance/2962609/bin/nihms-242212-f0010.gif" src-large="/pmc/articles/instance/2962609/bin/nihms-242212-f0010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2962609/" ref="ordinalpos=750&amp;ncbi_uid=2528960&amp;link_uid=PMC2962609" image-link="/pmc/articles/PMC2962609/figure/F10/" class="imagepopup">Figure 10.  From: Rhodopsin-Regulated Insulin Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Rod Photoreceptor Neurons. </a></p><div class="supp"><p class="details"><div>Working model of IR activation in rod photoreceptor cells. The IR activation in photoreceptor cells requires the photobleaching of rhodopsin but not transducin <span class="highlight" style="background-color:">signaling</span>. Light-activated IR is subsequently associated with PI3K, a cell survival factor and thus regulates the downstream survival <span class="highlight" style="background-color:">pathway</span>. R*, photoactivated rhodopsin.</div></p></div><div class="aux"><div class="resc">Raju V.S. Rajala, et al. Mol Neurobiol. ;42(1):39-47.</div><p class="links"><a class="dblinks" href="/pubmed/20407846" ref="ordinalpos=750&amp;ncbi_uid=2528960&amp;link_uid=20407846">Citation</a><a class="dblinks" href="/pmc/articles/PMC2962609/" ref="ordinalpos=750&amp;ncbi_uid=2528960&amp;link_uid=PMC2962609">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>751.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2642928/figure/F1/" ref="ordinalpos=751&amp;ncbi_uid=1643158&amp;link_uid=PMC2642928" href="/pmc/articles/PMC2642928/"><img src="/pmc/articles/instance/2642928/bin/nihms92340f1.gif" src-large="/pmc/articles/instance/2642928/bin/nihms92340f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2642928/" ref="ordinalpos=751&amp;ncbi_uid=1643158&amp;link_uid=PMC2642928" image-link="/pmc/articles/PMC2642928/figure/F1/" class="imagepopup">Figure 1.  From: Aberrant T cell ERK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> and chromatin structure in lupus. </a></p><div class="supp"><p class="details"><div>Proposed mechanism by which epigenetic modifications in T cells may induce idiopathic and drug-induced lupus. Decreased ERK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> due to impaired PKCδ decreases DNA methylation what modifies gene expression making T cells autoreactive.</div></p></div><div class="aux"><div class="resc">Gabriela Gorelik, et al. Autoimmun Rev. ;8(3):196-198.</div><p class="links"><a class="dblinks" href="/pubmed/18723128" ref="ordinalpos=751&amp;ncbi_uid=1643158&amp;link_uid=18723128">Citation</a><a class="dblinks" href="/pmc/articles/PMC2642928/" ref="ordinalpos=751&amp;ncbi_uid=1643158&amp;link_uid=PMC2642928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>752.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3356297/figure/pone-0037654-g009/" ref="ordinalpos=752&amp;ncbi_uid=4016982&amp;link_uid=PMC3356297" href="/pmc/articles/PMC3356297/"><img src="/pmc/articles/instance/3356297/bin/pone.0037654.g009.gif" src-large="/pmc/articles/instance/3356297/bin/pone.0037654.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3356297/" ref="ordinalpos=752&amp;ncbi_uid=4016982&amp;link_uid=PMC3356297" image-link="/pmc/articles/PMC3356297/figure/pone-0037654-g009/" class="imagepopup">Figure 9. Certain marker genes of the defense <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are involved in the effects of MoHrip1 on rice..  From: Purification and Characterization of a Novel Hypersensitive Response-Inducing Elicitor from Magnaporthe oryzae that Triggers Defense Response in Rice. </a></p><div class="supp"><p class="details"><div><b>A</b>. <i>OsEDS1</i>, <b>B</b>. <i>OsPAL1</i>, <b>C</b>. <i>OsNH1</i>, <b>D</b>. <i>OsLOX2</i>, <b>E</b>. <i>OsAOS2</i>. Marker genes for the SA-dependent defense <span class="highlight" style="background-color:">pathway</span> (<i>OsEDS1</i>, <i>OsPAL1</i> and <i>OsNH1</i>) were all significantly induced by MoHrip1 on the second day after treatment, and the same expression pattern was found for the JA/Et <span class="highlight" style="background-color:">pathway</span> gene <i>OsLOX2</i>, but strong induction was not observed for <i>OsAOS2</i> (another JA/Et <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> marker gene). Dpi: days post-inoculation. Error bars represent ± SD of the mean. Essentially identical results were obtained in three independent experiments.</div></p></div><div class="aux"><div class="resc">Mingjia Chen, et al. PLoS One. 2012;7(5):e37654.</div><p class="links"><a class="dblinks" href="/pubmed/22624059" ref="ordinalpos=752&amp;ncbi_uid=4016982&amp;link_uid=22624059">Citation</a><a class="dblinks" href="/pmc/articles/PMC3356297/" ref="ordinalpos=752&amp;ncbi_uid=4016982&amp;link_uid=PMC3356297">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>753.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4166412/figure/F8/" ref="ordinalpos=753&amp;ncbi_uid=6621303&amp;link_uid=PMC4166412" href="/pmc/articles/PMC4166412/"><img src="/pmc/articles/instance/4166412/bin/1750-2187-9-8-8.gif" src-large="/pmc/articles/instance/4166412/bin/1750-2187-9-8-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4166412/" ref="ordinalpos=753&amp;ncbi_uid=6621303&amp;link_uid=PMC4166412" image-link="/pmc/articles/PMC4166412/figure/F8/" class="imagepopup">Figure 8.  From: Seminal plasma induces the expression of IL-1? in normal and neoplastic cervical cells via EP2/EGFR/PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic summary highlighting the role of SP and its constituents (PGE</b><sub><b>2 </b></sub><b>and EGF) in the regulation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediating IL-1α (IL-1 alpha) expression in cervical epithelial cells.</b> Expressed IL-1α can then bind to its cognate receptor, interleukin-1 receptor type I (IL-1RI) to recruit interleukin-1 receptor accessory protein (IL-1RAP) on same cell leading to the activation of IL-1α <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (IL-1a <span class="highlight" style="background-color:">pathway</span>).</div></p></div><div class="aux"><div class="resc">Anthonio O Adefuye, et al. J Mol Signal. 2014;9:8-8.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=753&amp;ncbi_uid=6621303&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4166412/" ref="ordinalpos=753&amp;ncbi_uid=6621303&amp;link_uid=PMC4166412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>754.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3418756/figure/F6/" ref="ordinalpos=754&amp;ncbi_uid=4670176&amp;link_uid=PMC3418756" href="/pmc/articles/PMC3418756/"><img src="/pmc/articles/instance/3418756/bin/zii9990997840006.gif" src-large="/pmc/articles/instance/3418756/bin/zii9990997840006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3418756/" ref="ordinalpos=754&amp;ncbi_uid=4670176&amp;link_uid=PMC3418756" image-link="/pmc/articles/PMC3418756/figure/F6/" class="imagepopup">Fig 6.  From: Genome-Wide Mapping of Cystitis Due to Streptococcus agalactiae and Escherichia coli in Mice Identifies a Unique Bladder Transcriptome That Signifies Pathogen-Specific Antimicrobial Defense against Urinary Tract Infection. </a></p><div class="supp"><p class="details"><div>Immune activation in the GBS UTI transcriptome comprises prominent cytokine-cytokine receptor <span class="highlight" style="background-color:">signaling</span> (innateDB <span class="highlight" style="background-color:">pathway</span> number 604) (A) in which 12 genes were highly active (red circles; expression and significance cutoffs, 2.0 and 0.1). The <span class="highlight" style="background-color:">pathway</span> is illustrated according to subcellular localization and includes the total of 255 genes annotated in the <span class="highlight" style="background-color:">pathway</span>. The bladder transcriptome also includes chemokine <span class="highlight" style="background-color:">signaling</span> (innateDB <span class="highlight" style="background-color:">pathway</span> 4351) (B), with activation of the <span class="highlight" style="background-color:">pathway</span> occurring via 8 genes (red circles; Cxcl10, Cxcl9, and Stat2 are shown twice) among a total of 175 genes in the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chee K. Tan, et al. Infect Immun. 2012 September;80(9):3145-3160.</div><p class="links"><a class="dblinks" href="/pubmed/22733575" ref="ordinalpos=754&amp;ncbi_uid=4670176&amp;link_uid=22733575">Citation</a><a class="dblinks" href="/pmc/articles/PMC3418756/" ref="ordinalpos=754&amp;ncbi_uid=4670176&amp;link_uid=PMC3418756">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>755.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4121416/figure/f5-ijo-45-03-1011/" ref="ordinalpos=755&amp;ncbi_uid=6485863&amp;link_uid=PMC4121416" href="/pmc/articles/PMC4121416/"><img src="/pmc/articles/instance/4121416/bin/IJO-45-03-1011-g04.gif" src-large="/pmc/articles/instance/4121416/bin/IJO-45-03-1011-g04.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4121416/" ref="ordinalpos=755&amp;ncbi_uid=6485863&amp;link_uid=PMC4121416" image-link="/pmc/articles/PMC4121416/figure/f5-ijo-45-03-1011/" class="imagepopup">Figure 5.  From: Induction of apoptosis by a peptide from Porphyra yezoensis: Regulation of the insulin-like growth factor I receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in MCF-7 cells. </a></p><div class="supp"><p class="details"><div>Effect of PPY on the expression of levels of the IGF-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cells were treated with various concentrations of PPY (0–500 ng/ml) for 24 h. IGF-IR, IRS-I, SHC, ERK and p-ERK protein levels are shown.</div></p></div><div class="aux"><div class="resc">SU-JIN PARK, et al. Int J Oncol. 2014 September;45(3):1011-1016.</div><p class="links"><a class="dblinks" href="/pubmed/24970277" ref="ordinalpos=755&amp;ncbi_uid=6485863&amp;link_uid=24970277">Citation</a><a class="dblinks" href="/pmc/articles/PMC4121416/" ref="ordinalpos=755&amp;ncbi_uid=6485863&amp;link_uid=PMC4121416">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>756.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3999155/figure/ppat-1004041-g006/" ref="ordinalpos=756&amp;ncbi_uid=6068514&amp;link_uid=PMC3999155" href="/pmc/articles/PMC3999155/"><img src="/pmc/articles/instance/3999155/bin/ppat.1004041.g006.gif" src-large="/pmc/articles/instance/3999155/bin/ppat.1004041.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3999155/" ref="ordinalpos=756&amp;ncbi_uid=6068514&amp;link_uid=PMC3999155" image-link="/pmc/articles/PMC3999155/figure/ppat-1004041-g006/" class="imagepopup">Figure 6. Working model of TRAF3-dependent regulation of RLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by HSCARG..  From: HSCARG Negatively Regulates the Cellular Antiviral RIG-I Like Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by Inhibiting TRAF3 Ubiquitination via Recruiting OTUB1. </a></p><div class="supp"><p class="details"><div>Followed viral infection, HSCARG interacts with TRAF3, and cooperates with OTUB1 to remove Lys63-linked polyubiquitin chain from TRAF3. This further decreases the recruitment of IKKε and results in decreased phosphorylation of IKKε and IRF3, and finally reduces the production of IFN-β.</div></p></div><div class="aux"><div class="resc">Yanyan Peng, et al. PLoS Pathog. 2014 April;10(4):e1004041.</div><p class="links"><a class="dblinks" href="/pubmed/24763515" ref="ordinalpos=756&amp;ncbi_uid=6068514&amp;link_uid=24763515">Citation</a><a class="dblinks" href="/pmc/articles/PMC3999155/" ref="ordinalpos=756&amp;ncbi_uid=6068514&amp;link_uid=PMC3999155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>757.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3203685/figure/F1/" ref="ordinalpos=757&amp;ncbi_uid=3512229&amp;link_uid=PMC3203685" href="/pmc/articles/PMC3203685/"><img src="/pmc/articles/instance/3203685/bin/nihms324149f1.gif" src-large="/pmc/articles/instance/3203685/bin/nihms324149f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3203685/" ref="ordinalpos=757&amp;ncbi_uid=3512229&amp;link_uid=PMC3203685" image-link="/pmc/articles/PMC3203685/figure/F1/" class="imagepopup">Figure 1. PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> cascade.  From: Summary of the Proceedings from the 10th Annual Meeting of Molecularly Targeted Therapy in NSCLC. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> through a transmembrane receptor activates the PI3K <span class="highlight" style="background-color:">signaling</span> network to phosphorylate Akt and promote cell proliferation and invasion through mTOR. Multiple feedback loops exist within this <span class="highlight" style="background-color:">signaling</span> cascade, and a number of inhibitors are in development to target this <span class="highlight" style="background-color:">pathway</span> in cancer.</div></p></div><div class="aux"><div class="resc">Sumanta K. Pal, et al. J Thorac Oncol. ;5(12 Suppl 6):S433-S433.</div><p class="links"><a class="dblinks" href="/pubmed/21102234" ref="ordinalpos=757&amp;ncbi_uid=3512229&amp;link_uid=21102234">Citation</a><a class="dblinks" href="/pmc/articles/PMC3203685/" ref="ordinalpos=757&amp;ncbi_uid=3512229&amp;link_uid=PMC3203685">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>758.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2763551/figure/F6/" ref="ordinalpos=758&amp;ncbi_uid=2094089&amp;link_uid=PMC2763551" href="/pmc/articles/PMC2763551/"><img src="/pmc/articles/instance/2763551/bin/nihms-110166-f0006.gif" src-large="/pmc/articles/instance/2763551/bin/nihms-110166-f0006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2763551/" ref="ordinalpos=758&amp;ncbi_uid=2094089&amp;link_uid=PMC2763551" image-link="/pmc/articles/PMC2763551/figure/F6/" class="imagepopup">Fig 6. Influence of Integrated Gene Regulatory Control on <span class="highlight" style="background-color:">Signaling</span>.  From: Perspectives on the quantitative immunobiology of the IL-7 <span class="highlight" style="background-color:">signaling</span> network. </a></p><div class="supp"><p class="details"><div><b>(a)</b> Schematic of co-regulation of TCR and IL-7R induced <span class="highlight" style="background-color:">signaling</span> according to the CD8 coreceptor tuning model. IL-7 transcriptionally increases CD8α expression to promote TCR-self peptide-MHC engagement, but TCR and IL-7R signals both impair IL-7R <span class="highlight" style="background-color:">signaling</span>. <b>(b)</b> The predicted effects of coreceptor tuning are shown by plotting how TCR and IL-7 signals would be expected to vary as a function of TCR input signal strength (‘affinity’ for spMHC) with constant IL-7 input signal, either with or without coreceptor tuning. In the absence of tuning, total signal strength increases with the TCR input signal, whereas coreceptor tuning dampens the TCR <span class="highlight" style="background-color:">signaling</span> induced by high ‘affinity’ ligands, which in turn increases the sensitivity of the IL-7 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Megan J. Palmer, et al. Cell Mol Immunol. ;5(2):79-89.</div><p class="links"><a class="dblinks" href="/pubmed/18445337" ref="ordinalpos=758&amp;ncbi_uid=2094089&amp;link_uid=18445337">Citation</a><a class="dblinks" href="/pmc/articles/PMC2763551/" ref="ordinalpos=758&amp;ncbi_uid=2094089&amp;link_uid=PMC2763551">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>759.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g003/" ref="ordinalpos=759&amp;ncbi_uid=4442590&amp;link_uid=PMC3517549" href="/pmc/articles/PMC3517549/"><img src="/pmc/articles/instance/3517549/bin/pone.0051657.g003.gif" src-large="/pmc/articles/instance/3517549/bin/pone.0051657.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517549/" ref="ordinalpos=759&amp;ncbi_uid=4442590&amp;link_uid=PMC3517549" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g003/" class="imagepopup">Figure 3. The relationships among element positions in the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, protein length, connectivity, nonsynonymous divergence.  From: The Evolution and Origin of Animal Toll-Like Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Revealed by Network-Level Molecular Evolutionary Analyses. </a></p><div class="supp"><p class="details"><div>(<b><i>dn</i></b>)<b>, </b><b><i>dn/ds</i></b><b> ratio</b> (<b><i>ω</i></b>)<b>, and codon bias</b> (<b>measured by the ENC</b>)<b>.</b> <span class="highlight" style="background-color:">Pathway</span> position, protein length, and connectivity were treated as exogenous variables, while the rest were treated as endogenous variables. The causal dependencies between variables assumed in the model are represented by single-headed arrows. Correlations between exogenous variables are represented by double-headed arrows. The numbers on the arrows represent the standardized path coefficients (β). Solid and broken lines represent significant and nonsignificant relationships, respectively.</div></p></div><div class="aux"><div class="resc">Xiaojun Song, et al. PLoS One. 2012;7(12):e51657.</div><p class="links"><a class="dblinks" href="/pubmed/23236523" ref="ordinalpos=759&amp;ncbi_uid=4442590&amp;link_uid=23236523">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517549/" ref="ordinalpos=759&amp;ncbi_uid=4442590&amp;link_uid=PMC3517549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>760.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2817626/figure/F6/" ref="ordinalpos=760&amp;ncbi_uid=2733615&amp;link_uid=PMC2817626" href="/pmc/articles/PMC2817626/"><img src="/pmc/articles/instance/2817626/bin/zee0021051760006.gif" src-large="/pmc/articles/instance/2817626/bin/zee0021051760006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2817626/" ref="ordinalpos=760&amp;ncbi_uid=2733615&amp;link_uid=PMC2817626" image-link="/pmc/articles/PMC2817626/figure/F6/" class="imagepopup">Figure 6.  From: Testosterone Supplementation Reverses Sarcopenia in Aging through Regulation of Myostatin, c-Jun NH2-Terminal Kinase, Notch, and Akt <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Key <span class="highlight" style="background-color:">signaling</span> pathways involved in testosterone-mediated mitigation of sarcopenia in aging. Testosterone, though suppression myostatin, inhibits JNK but stimulates Akt <span class="highlight" style="background-color:">signaling</span>. Suppression of JNK promotes muscle growth by not only inhibiting muscle cell apoptosis but also stimulating cellular proliferation by attenuating myostatin-induced up-regulation of p21. Akt can promote muscle growth through direct activation of Notch <span class="highlight" style="background-color:">signaling</span> (67) as well as through modulation of multiple <span class="highlight" style="background-color:">signaling</span> molecules involved in both apoptotic and survival pathways in muscle remodeling (24). Akt can also restrain caspase-2-mediated death <span class="highlight" style="background-color:">pathway</span> and promote muscle growth by stimulation of cellular metabolism (47). Thus, testosterone through stimulation of survival <span class="highlight" style="background-color:">pathway</span> together with the inhibition of death <span class="highlight" style="background-color:">pathway</span> possibly restores the microenvironment and promotes muscle growth in aging.</div></p></div><div class="aux"><div class="resc">Ekaterina L. Kovacheva, et al. Endocrinology. 2010 February;151(2):628-638.</div><p class="links"><a class="dblinks" href="/pubmed/20022929" ref="ordinalpos=760&amp;ncbi_uid=2733615&amp;link_uid=20022929">Citation</a><a class="dblinks" href="/pmc/articles/PMC2817626/" ref="ordinalpos=760&amp;ncbi_uid=2733615&amp;link_uid=PMC2817626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>761.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4024417/figure/fig1/" ref="ordinalpos=761&amp;ncbi_uid=6179836&amp;link_uid=PMC4024417" href="/pmc/articles/PMC4024417/"><img src="/pmc/articles/instance/4024417/bin/BMRI2014-965631.001.gif" src-large="/pmc/articles/instance/4024417/bin/BMRI2014-965631.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4024417/" ref="ordinalpos=761&amp;ncbi_uid=6179836&amp;link_uid=PMC4024417" image-link="/pmc/articles/PMC4024417/figure/fig1/" class="imagepopup">Figure 1.  From: Understanding the Process of Fibrosis in Duchenne Muscular Dystrophy. </a></p><div class="supp"><p class="details"><div>Crosstalk between TGF<i>β</i> <span class="highlight" style="background-color:">signaling</span> and the renin-angiotensin system in fibrosis. TGF<i>β</i> can signal via its canonical <span class="highlight" style="background-color:">pathway</span>, involving Smad proteins, or through several alternative pathways such as the p38 MAPK <span class="highlight" style="background-color:">signaling</span> or the RAS/ERK MAPK <span class="highlight" style="background-color:">signaling</span> pathways. Both canonical and alternative pathways lead to expression of molecules implicated in fibrosis such as CTGF or PAI-1. Similarly, Ang II signals through AT1 or AT2 and can also activate Smad proteins and the p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, leading to increased expression of profibrotic genes. Ang 1–7 has an opposite effect, inhibiting the canonical TGF<i>β</i> <span class="highlight" style="background-color:">pathway</span>. Antifibrotic molecules inhibiting RAS or the TGF<i>β</i> <span class="highlight" style="background-color:">signaling</span> are indicated in red.</div></p></div><div class="aux"><div class="resc">Yacine Kharraz, et al. Biomed Res Int. 2014;2014:965631.</div><p class="links"><a class="dblinks" href="/pubmed/24877152" ref="ordinalpos=761&amp;ncbi_uid=6179836&amp;link_uid=24877152">Citation</a><a class="dblinks" href="/pmc/articles/PMC4024417/" ref="ordinalpos=761&amp;ncbi_uid=6179836&amp;link_uid=PMC4024417">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>762.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4385945/figure/fig7/" ref="ordinalpos=762&amp;ncbi_uid=7392660&amp;link_uid=PMC4385945" href="/pmc/articles/PMC4385945/"><img src="/pmc/articles/instance/4385945/bin/cddis201569f7.gif" src-large="/pmc/articles/instance/4385945/bin/cddis201569f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4385945/" ref="ordinalpos=762&amp;ncbi_uid=7392660&amp;link_uid=PMC4385945" image-link="/pmc/articles/PMC4385945/figure/fig7/" class="imagepopup">Figure 7.  From: Induction of COX-2-PGE2 synthesis by activation of the MAPK/ERK <span class="highlight" style="background-color:">pathway</span> contributes to neuronal death triggered by TDP-43-depleted microglia. </a></p><div class="supp"><p class="details"><div>Neuronal death is mediated by the classical MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and upregulated PGE2 in TDP-43-depleted microglia. The schematic illustrates the mechanism by which TDP-43-depleted microglia, but not TDP-43-depleted astrocytes, induce neuronal death. The MAPK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated in TDP-43-depleted microglia, resulting in increased COX-2 and PGE2, which then trigger neuronal death. Inhibition of COX-2 and PGE2 production using celecoxib alleviates neuronal death mediated by TDP-43-depleted microglia</div></p></div><div class="aux"><div class="resc">Q Xia, et al. Cell Death Dis. 2015 March;6(3):e1702.</div><p class="links"><a class="dblinks" href="/pubmed/25811799" ref="ordinalpos=762&amp;ncbi_uid=7392660&amp;link_uid=25811799">Citation</a><a class="dblinks" href="/pmc/articles/PMC4385945/" ref="ordinalpos=762&amp;ncbi_uid=7392660&amp;link_uid=PMC4385945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>763.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3666582/figure/F7/" ref="ordinalpos=763&amp;ncbi_uid=5988438&amp;link_uid=PMC3666582" href="/pmc/articles/PMC3666582/"><img src="/pmc/articles/instance/3666582/bin/nihms463723f7.gif" src-large="/pmc/articles/instance/3666582/bin/nihms463723f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3666582/" ref="ordinalpos=763&amp;ncbi_uid=5988438&amp;link_uid=PMC3666582" image-link="/pmc/articles/PMC3666582/figure/F7/" class="imagepopup">Figure 7. Example of a differentially expressed canonical <span class="highlight" style="background-color:">pathway</span>.  From: IDENTIFICATION AND DESCRIPTION OF A NOVEL MURINE MODEL FOR POLYTRAUMA AND SHOCK. </a></p><div class="supp"><p class="details"><div>IL-10 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at two hours is differentially expressed between the PT and TH murine models.</div></p></div><div class="aux"><div class="resc">Lori F Gentile, et al. Crit Care Med. ;41(4):1075-1085.</div><p class="links"><a class="dblinks" href="/pubmed/23399937" ref="ordinalpos=763&amp;ncbi_uid=5988438&amp;link_uid=23399937">Citation</a><a class="dblinks" href="/pmc/articles/PMC3666582/" ref="ordinalpos=763&amp;ncbi_uid=5988438&amp;link_uid=PMC3666582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>764.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2806100/figure/F7/" ref="ordinalpos=764&amp;ncbi_uid=2685576&amp;link_uid=PMC2806100" href="/pmc/articles/PMC2806100/"><img src="/pmc/articles/instance/2806100/bin/zh30011055600007.gif" src-large="/pmc/articles/instance/2806100/bin/zh30011055600007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2806100/" ref="ordinalpos=764&amp;ncbi_uid=2685576&amp;link_uid=PMC2806100" image-link="/pmc/articles/PMC2806100/figure/F7/" class="imagepopup">Fig. 7. From: Inflammatory cells regulate p53 and caspases in acute pancreatitis. </a></p><div class="supp"><p class="details"><div>Scheme of the <span class="highlight" style="background-color:">signaling</span> mechanism-mediated apoptosis in pancreatic parenchymal cells during pancreatitis. <i>A</i>: 2 pathways mediate apoptosis in pancreatitis. The mitochondrial <span class="highlight" style="background-color:">pathway</span> is initiated by cytochrome <i>c</i> release followed by caspase activation (3). The p53-mediated <span class="highlight" style="background-color:">pathway</span> also stimulates activation of caspases. Inflammatory cells inhibit the p53-mediated <span class="highlight" style="background-color:">pathway</span> but do not affect the mitochondrial <span class="highlight" style="background-color:">pathway</span>. <i>B</i>: detailed scheme of the p53-mediated <span class="highlight" style="background-color:">pathway</span>. Inflammatory cells inhibit this <span class="highlight" style="background-color:">pathway</span> through upregulating Mdm2, a negative regulator of p53.</div></p></div><div class="aux"><div class="resc">Yuji Nakamura, et al. Am J Physiol Gastrointest Liver Physiol. 2010 January;298(1):G92-G100.</div><p class="links"><a class="dblinks" href="/pubmed/19850968" ref="ordinalpos=764&amp;ncbi_uid=2685576&amp;link_uid=19850968">Citation</a><a class="dblinks" href="/pmc/articles/PMC2806100/" ref="ordinalpos=764&amp;ncbi_uid=2685576&amp;link_uid=PMC2806100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>765.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4127235/figure/fig3/" ref="ordinalpos=765&amp;ncbi_uid=6508284&amp;link_uid=PMC4127235" href="/pmc/articles/PMC4127235/"><img src="/pmc/articles/instance/4127235/bin/MI2014-310183.003.gif" src-large="/pmc/articles/instance/4127235/bin/MI2014-310183.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4127235/" ref="ordinalpos=765&amp;ncbi_uid=6508284&amp;link_uid=PMC4127235" image-link="/pmc/articles/PMC4127235/figure/fig3/" class="imagepopup">Figure 3.  From: The Wnt/β-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Controls the Inflammatory Response in Infections Caused by Pathogenic Bacteria. </a></p><div class="supp"><p class="details"><div>The PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> enhances <i>β</i>-catenin activation. In colon epithelial cells infected with<i> Citrobacter rodentium</i>, Akt phosphorylates <i>β</i>-catenin at Ser552. The activation of <i>β</i>-catenin increases c-myc and reduces TNF<i>α</i> and IFN<i>γ</i> expression. An opposite effect is observed in cells incubated with the PI3K inhibitor LY294002.</div></p></div><div class="aux"><div class="resc">Octavio Silva-García, et al. Mediators Inflamm. 2014;2014:310183.</div><p class="links"><a class="dblinks" href="/pubmed/25136145" ref="ordinalpos=765&amp;ncbi_uid=6508284&amp;link_uid=25136145">Citation</a><a class="dblinks" href="/pmc/articles/PMC4127235/" ref="ordinalpos=765&amp;ncbi_uid=6508284&amp;link_uid=PMC4127235">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>766.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3941151/figure/f2-cgm-6-2013-053/" ref="ordinalpos=766&amp;ncbi_uid=5938932&amp;link_uid=PMC3941151" href="/pmc/articles/PMC3941151/"><img src="/pmc/articles/instance/3941151/bin/cgm-6-2013-053f2.gif" src-large="/pmc/articles/instance/3941151/bin/cgm-6-2013-053f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3941151/" ref="ordinalpos=766&amp;ncbi_uid=5938932&amp;link_uid=PMC3941151" image-link="/pmc/articles/PMC3941151/figure/f2-cgm-6-2013-053/" class="imagepopup">Figure 2. Automated high throughput assay robotic system for <span class="highlight" style="background-color:">pathway</span> readouts.  From: Enhancing Cancer Drug Discovery through Novel Cell <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Panel Strategy. </a></p><div class="supp"><p class="details"><div>Automation processes for reagent addition, compound dispense and assay detection enable <span class="highlight" style="background-color:">signaling</span> panel approaches for compound evaluation in multiple cell lines.</div></p></div><div class="aux"><div class="resc">Liang Schweizer, et al. Cancer Growth Metastasis. 2013;6:53-59.</div><p class="links"><a class="dblinks" href="/pubmed/24665207" ref="ordinalpos=766&amp;ncbi_uid=5938932&amp;link_uid=24665207">Citation</a><a class="dblinks" href="/pmc/articles/PMC3941151/" ref="ordinalpos=766&amp;ncbi_uid=5938932&amp;link_uid=PMC3941151">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>767.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2998167/figure/F7/" ref="ordinalpos=767&amp;ncbi_uid=3591021&amp;link_uid=PMC2998167" href="/pmc/articles/PMC2998167/"><img src="/pmc/articles/instance/2998167/bin/DEV055350F7.gif" src-large="/pmc/articles/instance/2998167/bin/DEV055350F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2998167/" ref="ordinalpos=767&amp;ncbi_uid=3591021&amp;link_uid=PMC2998167" image-link="/pmc/articles/PMC2998167/figure/F7/" class="imagepopup">Fig. 7. From: Regulation of C. elegans presynaptic differentiation and neurite branching via a novel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> initiated by SAM-10. </a></p><div class="supp"><p class="details"><div><b>Model of SAM-10 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> SAM-10 is translocated into the nucleus in an LDB-1-dependent manner. In the nucleus, SAM-10 regulates PLM synaptic differentiation and neurite branch positioning, presumably by regulating gene expression. SAM-10 regulation of synaptic differentiation is mediated at least partially by suppressing <i>prk-2</i> transcription.</div></p></div><div class="aux"><div class="resc">Qun Zheng, et al. Development. 2011 January 1;138(1):87-96.</div><p class="links"><a class="dblinks" href="/pubmed/21115607" ref="ordinalpos=767&amp;ncbi_uid=3591021&amp;link_uid=21115607">Citation</a><a class="dblinks" href="/pmc/articles/PMC2998167/" ref="ordinalpos=767&amp;ncbi_uid=3591021&amp;link_uid=PMC2998167">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>768.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1635163/figure/F9/" ref="ordinalpos=768&amp;ncbi_uid=806347&amp;link_uid=PMC1635163" href="/pmc/articles/PMC1635163/"><img src="/pmc/articles/instance/1679705/bin/JCI0627702.f9.gif" src-large="/pmc/articles/instance/1679705/bin/JCI0627702.f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1635163/" ref="ordinalpos=768&amp;ncbi_uid=806347&amp;link_uid=PMC1635163" image-link="/pmc/articles/PMC1635163/figure/F9/" class="imagepopup">Figure 9. Proposed model of the role of TRPC6 in the calcineurin-NFAT <span class="highlight" style="background-color:">pathway</span> in vivo..  From: TRPC6 fulfills a calcineurin <span class="highlight" style="background-color:">signaling</span> circuit during pathologic cardiac remodeling. </a></p><div class="supp"><p class="details"><div>TRPC6 gene transcription is activated by the calcineurin-NFAT <span class="highlight" style="background-color:">pathway</span>, and an increase in the expression of TRPC6 further activates the calcineurin-NFAT <span class="highlight" style="background-color:">pathway</span>. The β<i>-MHC </i>gene is one of the downstream targets of this <span class="highlight" style="background-color:">pathway</span>, and its expression may alter cardiac contractility. Activation of BNP expression also provides a negative feedback loop to curtail hypertrophic <span class="highlight" style="background-color:">signaling</span>. TRPC6 forms a positive regulatory circuit in calcineurin-NFAT pathways leading to pathologic remodeling. DAG, diacylglycerol; GATA, GATA-binding protein; IP3, inositol-1,4,5-triphosphate; PLC, phospholipase C.                </div></p></div><div class="aux"><div class="resc">Koichiro Kuwahara, et al. J Clin Invest. 2006 December 1;116(12):3114-3126.</div><p class="links"><a class="dblinks" href="/pubmed/17099778" ref="ordinalpos=768&amp;ncbi_uid=806347&amp;link_uid=17099778">Citation</a><a class="dblinks" href="/pmc/articles/PMC1635163/" ref="ordinalpos=768&amp;ncbi_uid=806347&amp;link_uid=PMC1635163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>769.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3080858/figure/pgen-1001377-g006/" ref="ordinalpos=769&amp;ncbi_uid=2911197&amp;link_uid=PMC3080858" href="/pmc/articles/PMC3080858/"><img src="/pmc/articles/instance/3080858/bin/pgen.1001377.g006.gif" src-large="/pmc/articles/instance/3080858/bin/pgen.1001377.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3080858/" ref="ordinalpos=769&amp;ncbi_uid=2911197&amp;link_uid=PMC3080858" image-link="/pmc/articles/PMC3080858/figure/pgen-1001377-g006/" class="imagepopup">Figure 6. PDP-1 links TGF-β <span class="highlight" style="background-color:">signaling</span> to the IIS <span class="highlight" style="background-color:">pathway</span> and DAF-16..  From: PDP-1 Links the TGF-? and IIS Pathways to Regulate Longevity, Development, and Metabolism. </a></p><div class="supp"><p class="details"><div>Top panel: Under favorable environmental conditions, <span class="highlight" style="background-color:">signaling</span> through the TGF-β <span class="highlight" style="background-color:">pathway</span> activates the R-SMAD proteins DAF-8 and DAF-14, which regulate insulin gene expression while antagonizing DAF-3 and DAF-5 function. These insulins may act as agonists and activate IIS, thereby promoting phosphorylation and suppression of DAF-16 activity. In this feed-forward model, the worm undergoes reproductive growth and has a normal life span. Middle panel: PDP-1 negatively regulates TGF-β <span class="highlight" style="background-color:">signaling</span> through dephosphorylation of DAF-8 and DAF-14. Under these conditions, DAF-3 and DAF-5 repress the transcription of agonistic insulins as well as expression of the <i>daf-7</i> TGF-β ligand and <i>daf-8</i>, leading to further downregulation of the TGF-β <span class="highlight" style="background-color:">pathway</span>. Alternatively, DAF-3 and DAF-5 may promote transcription of potential antagonistic insulins. This results in reduced <span class="highlight" style="background-color:">signaling</span> through the IIS <span class="highlight" style="background-color:">pathway</span>, enhancing DAF-16 nuclear localization. Lower panel: Under low IIS conditions, DAF-16 localization is predominantly nuclear, where it regulates the transcription of hundreds of target genes that act in combination to regulate longevity, stress resistance, dauer formation and the response to stress. Paradoxically, under low IIS conditions, DAF-3 and DAF-5 play opposite roles. DAF-5 is likely to synergize with DAF-16 and modulate the activity of its target genes. DAF-3 acts to antagonize DAF-16, either directly or through suppression of DAF-16 target genes. Therefore, the role of DAF-3 in modulating IIS depends upon the level of <span class="highlight" style="background-color:">signaling</span> through the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Sri Devi Narasimhan, et al. PLoS Genet. 2011 April;7(4):e1001377.</div><p class="links"><a class="dblinks" href="/pubmed/21533078" ref="ordinalpos=769&amp;ncbi_uid=2911197&amp;link_uid=21533078">Citation</a><a class="dblinks" href="/pmc/articles/PMC3080858/" ref="ordinalpos=769&amp;ncbi_uid=2911197&amp;link_uid=PMC3080858">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>770.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405272/figure/F9/" ref="ordinalpos=770&amp;ncbi_uid=5140250&amp;link_uid=PMC3405272" href="/pmc/articles/PMC3405272/"><img src="/pmc/articles/instance/3405272/bin/z380081288400009.gif" src-large="/pmc/articles/instance/3405272/bin/z380081288400009.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405272/" ref="ordinalpos=770&amp;ncbi_uid=5140250&amp;link_uid=PMC3405272" image-link="/pmc/articles/PMC3405272/figure/F9/" class="imagepopup">Figure 9. From: <span class="highlight" style="background-color:">Signaling</span> through the neuropeptide GPCR PAC1 induces neuritogenesis via a single linear cAMP- and ERK-dependent <span class="highlight" style="background-color:">pathway</span> using a novel cAMP sensor. </a></p><div class="supp"><p class="details"><div>Schematic diagram summarizing cAMP-mediated <span class="highlight" style="background-color:">signaling</span> in NS-1 cells. Black arrows represent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> connections; green arrows represent pharmacological agents that serve as agonists or activators; red lines represent small molecule inhibitors. In light blue are 3 possible effectors of cAMP <span class="highlight" style="background-color:">signaling</span> in NS-1 cells, including the regulatory subunit of PKA (Reg), Epac, and the neuritogenic cAMP sensor. Diagram is not to scale, and components depicted are not necessarily placed in their actual cellular compartments.</div></p></div><div class="aux"><div class="resc">Andrew C. Emery, et al. FASEB J. 2012 August;26(8):3199-3211.</div><p class="links"><a class="dblinks" href="/pubmed/22532442" ref="ordinalpos=770&amp;ncbi_uid=5140250&amp;link_uid=22532442">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405272/" ref="ordinalpos=770&amp;ncbi_uid=5140250&amp;link_uid=PMC3405272">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>771.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3541279/figure/pone-0053230-g005/" ref="ordinalpos=771&amp;ncbi_uid=4535039&amp;link_uid=PMC3541279" href="/pmc/articles/PMC3541279/"><img src="/pmc/articles/instance/3541279/bin/pone.0053230.g005.gif" src-large="/pmc/articles/instance/3541279/bin/pone.0053230.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3541279/" ref="ordinalpos=771&amp;ncbi_uid=4535039&amp;link_uid=PMC3541279" image-link="/pmc/articles/PMC3541279/figure/pone-0053230-g005/" class="imagepopup">Figure 5. Metyrapone and carbenoxolone attenuate gene expression in both pro-survival and pro-death pathways..  From: <span class="highlight" style="background-color:">Pathway</span> Analysis Reveals Common Pro-Survival Mechanisms of Metyrapone and Carbenoxolone after Traumatic Brain Injury. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Axon guidance <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is associated with regeneration and growth. (<b>B</b>) TNFR1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is associated with inflammation and cell death. (See Fig. S15 for symbol key).</div></p></div><div class="aux"><div class="resc">Helen L. Hellmich, et al. PLoS One. 2013;8(1):e53230.</div><p class="links"><a class="dblinks" href="/pubmed/23326402" ref="ordinalpos=771&amp;ncbi_uid=4535039&amp;link_uid=23326402">Citation</a><a class="dblinks" href="/pmc/articles/PMC3541279/" ref="ordinalpos=771&amp;ncbi_uid=4535039&amp;link_uid=PMC3541279">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>772.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1775040/figure/F2/" ref="ordinalpos=772&amp;ncbi_uid=865159&amp;link_uid=PMC1775040" href="/pmc/articles/PMC1775040/"><img src="/pmc/articles/instance/1775040/bin/1471-2202-7-S1-S10-2.gif" src-large="/pmc/articles/instance/1775040/bin/1471-2202-7-S1-S10-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1775040/" ref="ordinalpos=772&amp;ncbi_uid=865159&amp;link_uid=PMC1775040" image-link="/pmc/articles/PMC1775040/figure/F2/" class="imagepopup">Figure 2.  From: Mathematical modeling of intracellular <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> Pathways in Yeast</b>. Overview of <span class="highlight" style="background-color:">signaling</span> pathways in the baker's yeast <i>S. cerevisiae</i>. From left to right: HOG <span class="highlight" style="background-color:">pathway</span> activated by osmotic shock, pheromone <span class="highlight" style="background-color:">pathway</span> activated by pheromones secreted by cells of the opposite mating type, pseudohyphal growth <span class="highlight" style="background-color:">pathway</span> stimulated by starvation conditions, and the glucose sensing <span class="highlight" style="background-color:">pathway</span>. In each <span class="highlight" style="background-color:">pathway</span>, the activated receptor activates a cascade of intracellular processes including complex formation, phosphorylation and dephosphorylation, and transport steps. Most of these pathways comprise a MAP kinase cascade and transcription factors that regulate the expression of target genes. Besides this, the <span class="highlight" style="background-color:">signaling</span> pathways can directly interact with cell cycle progression and adaptation of metabolism.</div></p></div><div class="aux"><div class="resc">Edda Klipp, et al. BMC Neurosci. 2006;7(Suppl 1):S10-S10.</div><p class="links"><a class="dblinks" href="/pubmed/17118154" ref="ordinalpos=772&amp;ncbi_uid=865159&amp;link_uid=17118154">Citation</a><a class="dblinks" href="/pmc/articles/PMC1775040/" ref="ordinalpos=772&amp;ncbi_uid=865159&amp;link_uid=PMC1775040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>773.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3506961/figure/F5/" ref="ordinalpos=773&amp;ncbi_uid=4414913&amp;link_uid=PMC3506961" href="/pmc/articles/PMC3506961/"><img src="/pmc/articles/instance/3506961/bin/fnbeh-06-00079-g0005.gif" src-large="/pmc/articles/instance/3506961/bin/fnbeh-06-00079-g0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3506961/" ref="ordinalpos=773&amp;ncbi_uid=4414913&amp;link_uid=PMC3506961" image-link="/pmc/articles/PMC3506961/figure/F5/" class="imagepopup">Figure 5.  From: Integrating cortico-limbic-basal ganglia architectures for learning model-based and model-free navigation strategies. </a></p><div class="supp"><p class="details"><div><b>Dual pathways from shell to ventral tegmental area (VTA) potentially support prediction error computation. (A)</b> The medial and lateral shell both support a dual <span class="highlight" style="background-color:">pathway</span> circuit that converges on dopaminergic neurons in the VTA: a direct <span class="highlight" style="background-color:">pathway</span> originating from a population of D1 receptor expressing striatal projection neurons, and an indirect <span class="highlight" style="background-color:">pathway</span> originating from a mixed population of D1 and D2 receptor expressing striatal projection neurons [see (Humphries and Prescott, 2010) for review]. This arrangement is consistent with the shell's role as a “critic”: the pathways support the computation of a prediction error between the prediction transmitted by the direct <span class="highlight" style="background-color:">pathway</span> and the actual outcome transmitted by the indirect <span class="highlight" style="background-color:">pathway</span> (PPn, pedunculopontine nucleus; VP, ventral pallidum). <b>(B)</b> Simulation of neural population activity showing how a greater outcome (indirect <span class="highlight" style="background-color:">pathway</span>) than predicted (direct <span class="highlight" style="background-color:">pathway</span>) drives a phasic increase in VTA activity, <span class="highlight" style="background-color:">signaling</span> a positive prediction error. <b>(C)</b> Simulation of neural population activity showing how a lower outcome (indirect <span class="highlight" style="background-color:">pathway</span>) than predicted (direct <span class="highlight" style="background-color:">pathway</span>) drives a phasic dip in VTA activity, <span class="highlight" style="background-color:">signaling</span> a negative prediction error. Simulation details given in Humphries and Prescott (2010).</div></p></div><div class="aux"><div class="resc">Mehdi Khamassi, et al. Front Behav Neurosci. 2012;6:79.</div><p class="links"><a class="dblinks" href="/pubmed/23205006" ref="ordinalpos=773&amp;ncbi_uid=4414913&amp;link_uid=23205006">Citation</a><a class="dblinks" href="/pmc/articles/PMC3506961/" ref="ordinalpos=773&amp;ncbi_uid=4414913&amp;link_uid=PMC3506961">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>774.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2773638/figure/A001644F4/" ref="ordinalpos=774&amp;ncbi_uid=2077491&amp;link_uid=PMC2773638" href="/pmc/articles/PMC2773638/"><img src="/pmc/articles/instance/2773638/bin/cshperspect-SYM-001644_F4.gif" src-large="/pmc/articles/instance/2773638/bin/cshperspect-SYM-001644_F4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2773638/" ref="ordinalpos=774&amp;ncbi_uid=2077491&amp;link_uid=PMC2773638" image-link="/pmc/articles/PMC2773638/figure/A001644F4/" class="imagepopup">Figure 4. From: Neuronal Polarity. </a></p><div class="supp"><p class="details"><div>Coordination of extracellular and intracellular <span class="highlight" style="background-color:">signaling</span> regulates cytoskeletal dynamics and axon formation. Overview of selected <span class="highlight" style="background-color:">signaling</span> pathways that may initiate neuronal polarization and axon specification. BDNF <span class="highlight" style="background-color:">signaling</span> (blue) leads to activation of PKA/LKB1/SAD A/B <span class="highlight" style="background-color:">signaling</span> that regulates the stability of microtubules and induces axon formation. The PI3K/PIP<sub>3</sub>/Akt/GSK-3β <span class="highlight" style="background-color:">pathway</span> (purple) regulates MAPs and the stability of microtubules. Another <span class="highlight" style="background-color:">signaling</span> branch activates Rap1B and the Rho GTPases Cdc42 and Rac1. Signals from both of these GTPases regulate actin dynamics via WASP/WAVE (red) or PAK-cofilin pathways (green). Rac1 may also regulate microtubules by regulating the microtubule destabilizer Op18/Stathmin. Apart from being downstream of PI3K <span class="highlight" style="background-color:">signaling</span>, Cdc42 and Rac1 may be activated by other signals (potential green and red plasma membrane [PM] receptors). Another molecule implicated in axon development is RhoA, which might regulate actin via the LIMK/cofilin <span class="highlight" style="background-color:">pathway</span> (green) or microtubule dynamics via ROCK <span class="highlight" style="background-color:">signaling</span> to MAPs (orange).</div></p></div><div class="aux"><div class="resc">Sabina Tahirovic, et al. Cold Spring Harb Perspect Biol. 2009 September;1(3):a001644.</div><p class="links"><a class="dblinks" href="/pubmed/20066106" ref="ordinalpos=774&amp;ncbi_uid=2077491&amp;link_uid=20066106">Citation</a><a class="dblinks" href="/pmc/articles/PMC2773638/" ref="ordinalpos=774&amp;ncbi_uid=2077491&amp;link_uid=PMC2773638">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>775.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3112527/figure/fig3/" ref="ordinalpos=775&amp;ncbi_uid=3025526&amp;link_uid=PMC3112527" href="/pmc/articles/PMC3112527/"><img src="/pmc/articles/instance/3112527/bin/JTR2011-815826.003.gif" src-large="/pmc/articles/instance/3112527/bin/JTR2011-815826.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3112527/" ref="ordinalpos=775&amp;ncbi_uid=3025526&amp;link_uid=PMC3112527" image-link="/pmc/articles/PMC3112527/figure/fig3/" class="imagepopup">Figure 3.  From: Medullary Thyroid Carcinoma: Molecular <span class="highlight" style="background-color:">Signaling</span> Pathways and Emerging Therapies. </a></p><div class="supp"><p class="details"><div>A summary of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by RET.</div></p></div><div class="aux"><div class="resc">Karen Gómez, et al. J Thyroid Res. 2011;2011:815826.</div><p class="links"><a class="dblinks" href="/pubmed/21687607" ref="ordinalpos=775&amp;ncbi_uid=3025526&amp;link_uid=21687607">Citation</a><a class="dblinks" href="/pmc/articles/PMC3112527/" ref="ordinalpos=775&amp;ncbi_uid=3025526&amp;link_uid=PMC3112527">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>776.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1174965/figure/F4/" ref="ordinalpos=776&amp;ncbi_uid=535332&amp;link_uid=PMC1174965" href="/pmc/articles/PMC1174965/"><img src="/pmc/articles/instance/1174965/bin/ar1741-4.gif" src-large="/pmc/articles/instance/1174965/bin/ar1741-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1174965/" ref="ordinalpos=776&amp;ncbi_uid=535332&amp;link_uid=PMC1174965" image-link="/pmc/articles/PMC1174965/figure/F4/" class="imagepopup">Figure 4.  From: Negative regulation of cytokine <span class="highlight" style="background-color:">signaling</span> and immune responses by SOCS proteins. </a></p><div class="supp"><p class="details"><div>Regulation of lipopolysaccharide (LPS) <span class="highlight" style="background-color:">signaling</span> by suppressor of cytokine <span class="highlight" style="background-color:">signaling</span> 1 (SOCS1). LPS stimulates NF-κB and the JNK/p38 <span class="highlight" style="background-color:">pathway</span> through Toll-like receptor (TLR)4/MD2 receptor. IFN-β is rapidly induced through the TRIF/IRF-3 <span class="highlight" style="background-color:">pathway</span> and activates the JAK/STAT1 (Janus kinase/signal transduction and activators of transcription 1) <span class="highlight" style="background-color:">pathway</span>. SOCS1 is probably induced by STAT1 and NF-κB, and then suppresses both STAT1 and NF-κB. One possible mechanism of NF-κB suppression is the induction of degradation of the p65 NF-κB subunit.</div></p></div><div class="aux"><div class="resc">Akihiko Yoshimura, et al. Arthritis Res Ther. 2005;7(3):100-110.</div><p class="links"><a class="dblinks" href="/pubmed/15899058" ref="ordinalpos=776&amp;ncbi_uid=535332&amp;link_uid=15899058">Citation</a><a class="dblinks" href="/pmc/articles/PMC1174965/" ref="ordinalpos=776&amp;ncbi_uid=535332&amp;link_uid=PMC1174965">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>777.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2953615/figure/F3/" ref="ordinalpos=777&amp;ncbi_uid=2513627&amp;link_uid=PMC2953615" href="/pmc/articles/PMC2953615/"><img src="/pmc/articles/instance/2953615/bin/nihms237582f3.gif" src-large="/pmc/articles/instance/2953615/bin/nihms237582f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2953615/" ref="ordinalpos=777&amp;ncbi_uid=2513627&amp;link_uid=PMC2953615" image-link="/pmc/articles/PMC2953615/figure/F3/" class="imagepopup">Figure 3.  From: The primary cilium as a cellular <span class="highlight" style="background-color:">signaling</span> center: lessons from disease. </a></p><div class="supp"><p class="details"><div>The primary cilium as a <span class="highlight" style="background-color:">signaling</span> hub. <b>(a)</b> Cilia projecting into fluid-filled luminal spaces are positioned to respond to signals and fluid flow within the tubule lumen in order to activate <span class="highlight" style="background-color:">signaling</span> cascades such as calcium <span class="highlight" style="background-color:">signaling</span>, as shown in the inset. <b>(b)</b> The neuronal cilium however appears buried among the dendrites and axons of surrounding neurons implicating an alternative rationale for its presence. One model is that the cilium is positioned next to the nucleus such that components of <span class="highlight" style="background-color:">signaling</span> cascades like the Shh <span class="highlight" style="background-color:">pathway</span> (inset) may be concentrated within the cilium in order to faithfully relay messages. Thus, the cilium acts as a concentrator of <span class="highlight" style="background-color:">signaling</span> components, or a <span class="highlight" style="background-color:">signaling</span> hub. Background images are immunostaining within renal tubules or cultured hippocampal neurons.</div></p></div><div class="aux"><div class="resc">Madeline A Lancaster, et al. Curr Opin Genet Dev. ;19(3):220-229.</div><p class="links"><a class="dblinks" href="/pubmed/19477114" ref="ordinalpos=777&amp;ncbi_uid=2513627&amp;link_uid=19477114">Citation</a><a class="dblinks" href="/pmc/articles/PMC2953615/" ref="ordinalpos=777&amp;ncbi_uid=2513627&amp;link_uid=PMC2953615">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>778.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3960188/figure/F1/" ref="ordinalpos=778&amp;ncbi_uid=5992686&amp;link_uid=PMC3960188" href="/pmc/articles/PMC3960188/"><img src="/pmc/articles/instance/3960188/bin/oncotarget-05-0049-f001.gif" src-large="/pmc/articles/instance/3960188/bin/oncotarget-05-0049-f001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3960188/" ref="ordinalpos=778&amp;ncbi_uid=5992686&amp;link_uid=PMC3960188" image-link="/pmc/articles/PMC3960188/figure/F1/" class="imagepopup">Figure 1. Simplified scheme of mTOR <span class="highlight" style="background-color:">pathway</span> activation.  From: mTOR <span class="highlight" style="background-color:">pathway</span> in colorectal cancer: an update. </a></p><div class="supp"><p class="details"><div>Akt activation releases Rheb from the inhibitory effects of TSC1/2 thus allowing mTOR activation. <span class="highlight" style="background-color:">Signaling</span> defects upstream of mTOR in the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> (mutations in PTEN, PI3K, Akt and TSCs) lead to mTOR deregulation. Akt can also be a downstream effector of mTOR, due to mTOR association with different protein partners to form two functionally distinct <span class="highlight" style="background-color:">signaling</span> complexes, mTORC1 and mTORC2.</div></p></div><div class="aux"><div class="resc">Maria Giovanna Francipane, et al. Oncotarget. 2014 January;5(1):49-66.</div><p class="links"><a class="dblinks" href="/pubmed/24393708" ref="ordinalpos=778&amp;ncbi_uid=5992686&amp;link_uid=24393708">Citation</a><a class="dblinks" href="/pmc/articles/PMC3960188/" ref="ordinalpos=778&amp;ncbi_uid=5992686&amp;link_uid=PMC3960188">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>779.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3629228/figure/pone-0061622-g002/" ref="ordinalpos=779&amp;ncbi_uid=4822030&amp;link_uid=PMC3629228" href="/pmc/articles/PMC3629228/"><img src="/pmc/articles/instance/3629228/bin/pone.0061622.g002.gif" src-large="/pmc/articles/instance/3629228/bin/pone.0061622.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3629228/" ref="ordinalpos=779&amp;ncbi_uid=4822030&amp;link_uid=PMC3629228" image-link="/pmc/articles/PMC3629228/figure/pone-0061622-g002/" class="imagepopup">Figure 2. p53 signal <span class="highlight" style="background-color:">pathway</span> of KEGG..  From: De-Regulated MicroRNAs in Pediatric Cancer Stem Cells Target Pathways Involved in Cell Proliferation, Cell Cycle and Development. </a></p><div class="supp"><p class="details"><div>(A). The p53 <span class="highlight" style="background-color:">pathway</span> that includes 8 <span class="highlight" style="background-color:">signaling</span> components as defined by KEGG was highly regulated by the CSC associated DEmiRs. Five of the 26 DEmiRs were implicated in this <span class="highlight" style="background-color:">pathway</span>, <i>hsa-miR-21-5p, hsa-miR-135b, hsa-miR-29, hsa-miR-655, hsa-miR-494</i>. TGF-beta <span class="highlight" style="background-color:">pathway</span> of KEGG (B). The TGF-beta <span class="highlight" style="background-color:">pathway</span> was also regulated by the CSC associated DEmiRs. Receptor <i>TGFBR1</i> was regulated by <i>hsa-miR-101-3p</i>, <i>and</i> receptor <i>TGFBR2</i> was regulated by <i>hsa-miR-21-5p</i>, and <i>hsa-miR-519a-3p</i>. The receptor <i>BMPR2</i> was regulated by <i>hsa-miR-101-3p, hsa-miR-21-5p, hsa-miR-181c-5p, and hsa-miR-494</i>. Nodal type II receptor kinase and receptor <i>ACVR2B</i> were also regulated by <i>hsa-miR-101-3p</i>.</div></p></div><div class="aux"><div class="resc">Patricia C. Sanchez-Diaz, et al. PLoS One. 2013;8(4):e61622.</div><p class="links"><a class="dblinks" href="/pubmed/23613887" ref="ordinalpos=779&amp;ncbi_uid=4822030&amp;link_uid=23613887">Citation</a><a class="dblinks" href="/pmc/articles/PMC3629228/" ref="ordinalpos=779&amp;ncbi_uid=4822030&amp;link_uid=PMC3629228">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>780.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4314041/figure/fig05/" ref="ordinalpos=780&amp;ncbi_uid=7165870&amp;link_uid=PMC4314041" href="/pmc/articles/PMC4314041/"><img src="/pmc/articles/instance/4314041/bin/ijcep0007-8506-f5.gif" src-large="/pmc/articles/instance/4314041/bin/ijcep0007-8506-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4314041/" ref="ordinalpos=780&amp;ncbi_uid=7165870&amp;link_uid=PMC4314041" image-link="/pmc/articles/PMC4314041/figure/fig05/" class="imagepopup">Figure 5.  From: Placenta growth factor promotes migration through regulating epithelial-mesenchymal transition-related protein expression in cervical cancer. </a></p><div class="supp"><p class="details"><div>PLGF regulated the EMT-related protein expression through ERK/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A. Application of a specific ERK/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitor, PD98059 (10 μmol/l), or application of a specific PI3k/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitor, LY294002 (20 μmol/l), or a specific JNK <span class="highlight" style="background-color:">pathway</span> inhibitor, SP600125 (10 μmol/l) to siha cells before treated with PLGF. PD98059 substantially abolished the decrease of E-cadherin expression by Western blot. B. Application of only PD98059 significantly decreased the migration of siha cells in a transwell assay. *<i>P</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Wei Huang, et al. Int J Clin Exp Pathol. 2014;7(12):8506-8519.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=780&amp;ncbi_uid=7165870&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4314041/" ref="ordinalpos=780&amp;ncbi_uid=7165870&amp;link_uid=PMC4314041">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>781.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4266699/figure/fig02/" ref="ordinalpos=781&amp;ncbi_uid=6969046&amp;link_uid=PMC4266699" href="/pmc/articles/PMC4266699/"><img src="/pmc/articles/instance/4266699/bin/ajcr0004-0608-f2.gif" src-large="/pmc/articles/instance/4266699/bin/ajcr0004-0608-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4266699/" ref="ordinalpos=781&amp;ncbi_uid=6969046&amp;link_uid=PMC4266699" image-link="/pmc/articles/PMC4266699/figure/fig02/" class="imagepopup">Figure 2.  From: Redundant kinase activation and resistance of EGFR-tyrosine kinase inhibitors. </a></p><div class="supp"><p class="details"><div>Redundant kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The activation of redundant kinase leads to downstream <span class="highlight" style="background-color:">pathway</span> such as PI3K/AKT, RAS/RAF/MEK and JAK/STAT <span class="highlight" style="background-color:">signaling</span> actived, which offsets the blockade of EGFR <span class="highlight" style="background-color:">pathway</span> by TKIs.</div></p></div><div class="aux"><div class="resc">Min Luo, et al. Am J Cancer Res. 2014;4(6):608-628.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=781&amp;ncbi_uid=6969046&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4266699/" ref="ordinalpos=781&amp;ncbi_uid=6969046&amp;link_uid=PMC4266699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>782.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3763401/figure/f1-cancers-03-03894/" ref="ordinalpos=782&amp;ncbi_uid=5271163&amp;link_uid=PMC3763401" href="/pmc/articles/PMC3763401/"><img src="/pmc/articles/instance/3763401/bin/cancers-03-03894f1.gif" src-large="/pmc/articles/instance/3763401/bin/cancers-03-03894f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3763401/" ref="ordinalpos=782&amp;ncbi_uid=5271163&amp;link_uid=PMC3763401" image-link="/pmc/articles/PMC3763401/figure/f1-cancers-03-03894/" class="imagepopup">Figure 1. From: The Crosstalk of PTGS2 and EGF <span class="highlight" style="background-color:">Signaling</span> Pathways in Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>The crosstalk between COX-2 and EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in CRC.</div></p></div><div class="aux"><div class="resc">Dingzhi Wang, et al. Cancers (Basel). 2011 December;3(4):3894-3908.</div><p class="links"><a class="dblinks" href="/pubmed/24213116" ref="ordinalpos=782&amp;ncbi_uid=5271163&amp;link_uid=24213116">Citation</a><a class="dblinks" href="/pmc/articles/PMC3763401/" ref="ordinalpos=782&amp;ncbi_uid=5271163&amp;link_uid=PMC3763401">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>783.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2878172/figure/F2/" ref="ordinalpos=783&amp;ncbi_uid=2253152&amp;link_uid=PMC2878172" href="/pmc/articles/PMC2878172/"><img src="/pmc/articles/instance/2878172/bin/ijbsv06p0268g02.gif" src-large="/pmc/articles/instance/2878172/bin/ijbsv06p0268g02.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2878172/" ref="ordinalpos=783&amp;ncbi_uid=2253152&amp;link_uid=PMC2878172" image-link="/pmc/articles/PMC2878172/figure/F2/" class="imagepopup">Figure 2.  From: Notch and Wnt <span class="highlight" style="background-color:">Signaling</span>, Physiological Stimuli and Postnatal Myogenesis. </a></p><div class="supp"><p class="details"><div>Canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Susan Tsivitse. Int J Biol Sci. 2010;6(3):268-281.</div><p class="links"><a class="dblinks" href="/pubmed/20567496" ref="ordinalpos=783&amp;ncbi_uid=2253152&amp;link_uid=20567496">Citation</a><a class="dblinks" href="/pmc/articles/PMC2878172/" ref="ordinalpos=783&amp;ncbi_uid=2253152&amp;link_uid=PMC2878172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>784.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3885983/figure/F3/" ref="ordinalpos=784&amp;ncbi_uid=5692464&amp;link_uid=PMC3885983" href="/pmc/articles/PMC3885983/"><img src="/pmc/articles/instance/3885983/bin/nihms542217f3.gif" src-large="/pmc/articles/instance/3885983/bin/nihms542217f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3885983/" ref="ordinalpos=784&amp;ncbi_uid=5692464&amp;link_uid=PMC3885983" image-link="/pmc/articles/PMC3885983/figure/F3/" class="imagepopup">Figure 3. Biological roles of canonical RTK-Ras-ERK <span class="highlight" style="background-color:">signaling</span> and alternative pathways.  From: Canonical RTK-Ras-ERK <span class="highlight" style="background-color:">signaling</span> and related alternative pathways. </a></p><div class="supp"><p class="details"><div>A) Cell types in the hermaphrodite that require LET-23/EGFR, EGL-15/FGFR and/or LET-60/Ras-dependent <span class="highlight" style="background-color:">signaling</span> are colored according to the type of <span class="highlight" style="background-color:">pathway</span> involved. Not all biological roles are included here due to limited information about the relevant <span class="highlight" style="background-color:">pathway</span> or site of action. Note that many <span class="highlight" style="background-color:">signaling</span> events actually occur during embryogenesis or during larval development, but the locations of relevant cells (or their descendants) in the adult are shown here. B) Photomicrographs of selected mutant phenotypes observed in Ras <span class="highlight" style="background-color:">pathway</span> mutants. Left column, wild type; Right column, mutants. See text for details and references. Scale bar, 10 microns.</div></p></div><div class="aux"><div class="resc">Meera V. Sundaram. WormBook. :10.1895/wormbook.1.80.2.</div><p class="links"><a class="dblinks" href="/pubmed/23908058" ref="ordinalpos=784&amp;ncbi_uid=5692464&amp;link_uid=23908058">Citation</a><a class="dblinks" href="/pmc/articles/PMC3885983/" ref="ordinalpos=784&amp;ncbi_uid=5692464&amp;link_uid=PMC3885983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>785.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4091530/figure/F7/" ref="ordinalpos=785&amp;ncbi_uid=6403825&amp;link_uid=PMC4091530" href="/pmc/articles/PMC4091530/"><img src="/pmc/articles/instance/4091530/bin/ganc-05-113-g007.gif" src-large="/pmc/articles/instance/4091530/bin/ganc-05-113-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4091530/" ref="ordinalpos=785&amp;ncbi_uid=6403825&amp;link_uid=PMC4091530" image-link="/pmc/articles/PMC4091530/figure/F7/" class="imagepopup">Figure 7. <span class="highlight" style="background-color:">Signaling</span> changes that are observed with GDC-0941 resistance in SW48 cells..  From: Amphiregulin and PTEN evoke a multimodal mechanism of acquired resistance to PI3K inhibition. </a></p><div class="supp"><p class="details"><div>(A) SW48 cells are initially sensitive to GDC-0941. GDC-0941 resistance develops with autocrine (AREG/EGFR) <span class="highlight" style="background-color:">signaling</span> and activation of the MAPK <span class="highlight" style="background-color:">pathway</span>. Cells eventually lose PTEN protein as resistance develops. (B) Resistant cells must be maintained on GDC-0941 to prevent <span class="highlight" style="background-color:">signaling</span> flare. In the absence of GDC-0941, resistant cells signal strongly through the PI3K <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Kyle A. Edgar, et al. Genes Cancer. 2014 March;5(3-4):113-126.</div><p class="links"><a class="dblinks" href="/pubmed/25053989" ref="ordinalpos=785&amp;ncbi_uid=6403825&amp;link_uid=25053989">Citation</a><a class="dblinks" href="/pmc/articles/PMC4091530/" ref="ordinalpos=785&amp;ncbi_uid=6403825&amp;link_uid=PMC4091530">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>786.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3506267/figure/F2/" ref="ordinalpos=786&amp;ncbi_uid=4401107&amp;link_uid=PMC3506267" href="/pmc/articles/PMC3506267/"><img src="/pmc/articles/instance/3506267/bin/2050-6511-13-3-2.gif" src-large="/pmc/articles/instance/3506267/bin/2050-6511-13-3-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3506267/" ref="ordinalpos=786&amp;ncbi_uid=4401107&amp;link_uid=PMC3506267" image-link="/pmc/articles/PMC3506267/figure/F2/" class="imagepopup">Figure 2.  From: The potential for selective pharmacological therapies through biased receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Molecular mechanism for agonist-directed biased <span class="highlight" style="background-color:">signaling</span>.</b> Agonist A stabilizes a conformation of the receptor that preferably activates <span class="highlight" style="background-color:">pathway</span> 1 while agonist B stabilizes a receptor conformation that preferably activates <span class="highlight" style="background-color:">pathway</span> 2.</div></p></div><div class="aux"><div class="resc">Terry Kenakin. BMC Pharmacol Toxicol. 2012;13:3-3.</div><p class="links"><a class="dblinks" href="/pubmed/22947056" ref="ordinalpos=786&amp;ncbi_uid=4401107&amp;link_uid=22947056">Citation</a><a class="dblinks" href="/pmc/articles/PMC3506267/" ref="ordinalpos=786&amp;ncbi_uid=4401107&amp;link_uid=PMC3506267">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>787.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2258989/figure/f6/" ref="ordinalpos=787&amp;ncbi_uid=1248263&amp;link_uid=PMC2258989" href="/pmc/articles/PMC2258989/"><img src="/pmc/articles/instance/2258989/bin/zjv0060803350006.gif" src-large="/pmc/articles/instance/2258989/bin/zjv0060803350006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2258989/" ref="ordinalpos=787&amp;ncbi_uid=1248263&amp;link_uid=PMC2258989" image-link="/pmc/articles/PMC2258989/figure/f6/" class="imagepopup">FIG. 6. From: Hepatitis C Virus Core Protein Upregulates Serine Phosphorylation of Insulin Receptor Substrate-1 and Impairs the Downstream Akt/Protein Kinase B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> for Insulin Resistance . </a></p><div class="supp"><p class="details"><div>Schematic presentation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induced by HCV core protein for insulin resistance in human hepatocytes. The results from the present study suggest that HCV core protein increases Ser<sup>312</sup> phosphorylation of IRS-1, which in turn decreases Tyr phosphorylation of IRS-1. This results in impairment of the PI-3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and decreases insulin-induced glucose uptake by human hepatocytes. An arrowhead pointing upward (↑) represents activation, an arrowhead pointing downward (↓) represents the repression of activities, and blunt arrowheads (⊣) represent blockade of <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Sutapa Banerjee, et al. J Virol. 2008 March;82(6):2606-2612.</div><p class="links"><a class="dblinks" href="/pubmed/18160431" ref="ordinalpos=787&amp;ncbi_uid=1248263&amp;link_uid=18160431">Citation</a><a class="dblinks" href="/pmc/articles/PMC2258989/" ref="ordinalpos=787&amp;ncbi_uid=1248263&amp;link_uid=PMC2258989">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>788.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4113576/figure/f3-etm-08-03-1005/" ref="ordinalpos=788&amp;ncbi_uid=6484422&amp;link_uid=PMC4113576" href="/pmc/articles/PMC4113576/"><img src="/pmc/articles/instance/4113576/bin/ETM-08-03-1005-g02.gif" src-large="/pmc/articles/instance/4113576/bin/ETM-08-03-1005-g02.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4113576/" ref="ordinalpos=788&amp;ncbi_uid=6484422&amp;link_uid=PMC4113576" image-link="/pmc/articles/PMC4113576/figure/f3-etm-08-03-1005/" class="imagepopup">Figure 3.  From: Inhibition of Aurora-B suppresses HepG2 cell invasion and migration via the PI3K/Akt/NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in vitro. </a></p><div class="supp"><p class="details"><div>Inhibiting Aurora-B expression decreases the activity of the PI3K/Akt/NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. With the downregulation of Aurora-B, the p-Akt, Akt, MMP-2, MMP-9 and NF-κB protein expression levels decreased significantly in the cells transfected with MiR-Aurora-B when compared with the cells transfected with MiR-Neg. NF-κB, nuclear factor-κB, MMP, matrix metalloproteinase; PI3K, phosphoinositide 3-kinase,</div></p></div><div class="aux"><div class="resc">REN FENG SHAN, et al. Exp Ther Med. 2014 September;8(3):1005-1009.</div><p class="links"><a class="dblinks" href="/pubmed/25120638" ref="ordinalpos=788&amp;ncbi_uid=6484422&amp;link_uid=25120638">Citation</a><a class="dblinks" href="/pmc/articles/PMC4113576/" ref="ordinalpos=788&amp;ncbi_uid=6484422&amp;link_uid=PMC4113576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>789.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1857715/figure/ppat-0030060-g008/" ref="ordinalpos=789&amp;ncbi_uid=985014&amp;link_uid=PMC1857715" href="/pmc/articles/PMC1857715/"><img src="/pmc/articles/instance/1857715/bin/ppat.0030060.g008.gif" src-large="/pmc/articles/instance/1857715/bin/ppat.0030060.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1857715/" ref="ordinalpos=789&amp;ncbi_uid=985014&amp;link_uid=PMC1857715" image-link="/pmc/articles/PMC1857715/figure/ppat-0030060-g008/" class="imagepopup">Figure 8. Proposed Model for TLR4 <span class="highlight" style="background-color:">Signaling</span> in BECs.  From: A Novel TLR4-Mediated <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Leading to IL-6 Responses in Human Bladder Epithelial Cells. </a></p><div class="supp"><p class="details"><div>The proposed rapidly induced second messenger– and CREB-mediated <span class="highlight" style="background-color:">pathway</span> (dark line) as well as the classical NF-κB (grey line) are shown. Both pathways are triggered by TLR4. P, phosphorylation; R and C, regulatory and catalytic subunits of PKA.</div></p></div><div class="aux"><div class="resc">Jeongmin Song, et al. PLoS Pathog. 2007 April;3(4):e60.</div><p class="links"><a class="dblinks" href="/pubmed/17465679" ref="ordinalpos=789&amp;ncbi_uid=985014&amp;link_uid=17465679">Citation</a><a class="dblinks" href="/pmc/articles/PMC1857715/" ref="ordinalpos=789&amp;ncbi_uid=985014&amp;link_uid=PMC1857715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>790.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3029747/figure/fig06/" ref="ordinalpos=790&amp;ncbi_uid=2752365&amp;link_uid=PMC3029747" href="/pmc/articles/PMC3029747/"><img src="/pmc/articles/instance/3029747/bin/pnas.1018376108fig06.gif" src-large="/pmc/articles/instance/3029747/bin/pnas.1018376108fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3029747/" ref="ordinalpos=790&amp;ncbi_uid=2752365&amp;link_uid=PMC3029747" image-link="/pmc/articles/PMC3029747/figure/fig06/" class="imagepopup">Fig. 6. From: Bile acid stimulates hepatocyte polarization through a cAMP-Epac-MEK-LKB1-AMPK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> model for taurocholate-accelerated canalicular network formation.</div></p></div><div class="aux"><div class="resc">Dong Fu, et al. Proc Natl Acad Sci U S A. 2011 January 25;108(4):1403-1408.</div><p class="links"><a class="dblinks" href="/pubmed/21220320" ref="ordinalpos=790&amp;ncbi_uid=2752365&amp;link_uid=21220320">Citation</a><a class="dblinks" href="/pmc/articles/PMC3029747/" ref="ordinalpos=790&amp;ncbi_uid=2752365&amp;link_uid=PMC3029747">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>791.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4055412/figure/pcbi-1003659-g003/" ref="ordinalpos=791&amp;ncbi_uid=6265321&amp;link_uid=PMC4055412" href="/pmc/articles/PMC4055412/"><img src="/pmc/articles/instance/4055412/bin/pcbi.1003659.g003.gif" src-large="/pmc/articles/instance/4055412/bin/pcbi.1003659.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4055412/" ref="ordinalpos=791&amp;ncbi_uid=6265321&amp;link_uid=PMC4055412" image-link="/pmc/articles/PMC4055412/figure/pcbi-1003659-g003/" class="imagepopup">Figure 3. Conservation and abundance variability of components of human <span class="highlight" style="background-color:">signaling</span> pathways..  From: Protein Conservation and Variation Suggest Mechanisms of Cell Type-Specific Modulation of <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Different components of <span class="highlight" style="background-color:">signaling</span> pathways show significant differences with respect to conservation (<b>A</b>) and protein abundance variability (<b>B</b>). Layers are generated by pooling different functional <span class="highlight" style="background-color:">pathway</span> classes. Here, we show the values for <span class="highlight" style="background-color:">signaling</span> proteins and their annotations from the Signalink database. We assigned receptors to the input layer, transcription factors to the output and mediators and cofactors (that are neither receptors nor transcription factors) to the transmission layer. Trends are schematized in (<b>C</b>).</div></p></div><div class="aux"><div class="resc">Martin H. Schaefer, et al. PLoS Comput Biol. 2014 June;10(6):e1003659.</div><p class="links"><a class="dblinks" href="/pubmed/24922536" ref="ordinalpos=791&amp;ncbi_uid=6265321&amp;link_uid=24922536">Citation</a><a class="dblinks" href="/pmc/articles/PMC4055412/" ref="ordinalpos=791&amp;ncbi_uid=6265321&amp;link_uid=PMC4055412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>792.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2943975/figure/F1/" ref="ordinalpos=792&amp;ncbi_uid=3579962&amp;link_uid=PMC2943975" href="/pmc/articles/PMC2943975/"><img src="/pmc/articles/instance/2943975/bin/nihms204193f1.gif" src-large="/pmc/articles/instance/2943975/bin/nihms204193f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2943975/" ref="ordinalpos=792&amp;ncbi_uid=3579962&amp;link_uid=PMC2943975" image-link="/pmc/articles/PMC2943975/figure/F1/" class="imagepopup">Figure 1. The Tor <span class="highlight" style="background-color:">signaling</span> network.  From: TOR <span class="highlight" style="background-color:">signaling</span> never gets old: Aging, longevity and TORC1 activity. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> core of the Tor <span class="highlight" style="background-color:">pathway</span> integrates the activity of growth factor <span class="highlight" style="background-color:">signaling</span> like insulin and wnt, hypoxia, energy stress and amino acid sufficiency to regulate translation initiation. The individual <span class="highlight" style="background-color:">signaling</span> modules survey the intra- and extracellular environment for their particular stimuli, and relay their signal to key components of the Tor <span class="highlight" style="background-color:">signaling</span> core, Tsc2 and the Tor complex 1 (TORC1). Among other processes, Tor regulates the biosynthetic capacity of the cell by controlling translational initiation. For species-dependent differences, please see main text.</div></p></div><div class="aux"><div class="resc">Daniel S. Evans, et al. Ageing Res Rev. ;10(2):225-237.</div><p class="links"><a class="dblinks" href="/pubmed/20385253" ref="ordinalpos=792&amp;ncbi_uid=3579962&amp;link_uid=20385253">Citation</a><a class="dblinks" href="/pmc/articles/PMC2943975/" ref="ordinalpos=792&amp;ncbi_uid=3579962&amp;link_uid=PMC2943975">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>793.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3858220/figure/pone-0077922-g007/" ref="ordinalpos=793&amp;ncbi_uid=5590787&amp;link_uid=PMC3858220" href="/pmc/articles/PMC3858220/"><img src="/pmc/articles/instance/3858220/bin/pone.0077922.g007.gif" src-large="/pmc/articles/instance/3858220/bin/pone.0077922.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3858220/" ref="ordinalpos=793&amp;ncbi_uid=5590787&amp;link_uid=PMC3858220" image-link="/pmc/articles/PMC3858220/figure/pone-0077922-g007/" class="imagepopup">Figure 7. The putative PTEN/FOXO3a/PLZF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the development of prostate cancer..  From: PLZF Mediates the PTEN/AKT/FOXO3a <span class="highlight" style="background-color:">Signaling</span> in Suppression of Prostate Tumorigenesis. </a></p><div class="aux"><div class="resc">JingPing Cao, et al. PLoS One. 2013;8(12):e77922.</div><p class="links"><a class="dblinks" href="/pubmed/24339862" ref="ordinalpos=793&amp;ncbi_uid=5590787&amp;link_uid=24339862">Citation</a><a class="dblinks" href="/pmc/articles/PMC3858220/" ref="ordinalpos=793&amp;ncbi_uid=5590787&amp;link_uid=PMC3858220">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>794.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1603434/figure/F6-6549/" ref="ordinalpos=794&amp;ncbi_uid=875565&amp;link_uid=PMC1603434" href="/pmc/articles/PMC1603434/"><img src="/pmc/articles/instance/1603434/bin/zjh0070565490006.gif" src-large="/pmc/articles/instance/1603434/bin/zjh0070565490006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1603434/" ref="ordinalpos=794&amp;ncbi_uid=875565&amp;link_uid=PMC1603434" image-link="/pmc/articles/PMC1603434/figure/F6-6549/" class="imagepopup">Figure 6.  From: Aberrant ?-Catenin <span class="highlight" style="background-color:">Signaling</span> in Tuberous Sclerosis. </a></p><div class="supp"><p class="details"><div>Model of TSC1/TSC2 in the canonical β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. See text for details.</div></p></div><div class="aux"><div class="resc">Baldwin C. Mak, et al. Am J Pathol. 2005 July;167(1):107-116.</div><p class="links"><a class="dblinks" href="/pubmed/15972957" ref="ordinalpos=794&amp;ncbi_uid=875565&amp;link_uid=15972957">Citation</a><a class="dblinks" href="/pmc/articles/PMC1603434/" ref="ordinalpos=794&amp;ncbi_uid=875565&amp;link_uid=PMC1603434">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>795.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2585009/figure/pone-0003664-g007/" ref="ordinalpos=795&amp;ncbi_uid=1614741&amp;link_uid=PMC2585009" href="/pmc/articles/PMC2585009/"><img src="/pmc/articles/instance/2585009/bin/pone.0003664.g007.gif" src-large="/pmc/articles/instance/2585009/bin/pone.0003664.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2585009/" ref="ordinalpos=795&amp;ncbi_uid=1614741&amp;link_uid=PMC2585009" image-link="/pmc/articles/PMC2585009/figure/pone-0003664-g007/" class="imagepopup">Figure 7. TLR <span class="highlight" style="background-color:">pathway</span> is unaltered by HIV-1..  From: HIV-1 Activates Macrophages Independent of Toll-Like Receptors. </a></p><div class="supp"><p class="details"><div>TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Factors increased in expression by HIV-1 treatment are red and factors expressed at lower levels by virus are green. Ovals represent factors detected at the RNA level. <span class="highlight" style="background-color:">Signaling</span> hubs are depicted with a yellow border.</div></p></div><div class="aux"><div class="resc">Joseph N. Brown, et al. PLoS ONE. 2008;3(12):e3664.</div><p class="links"><a class="dblinks" href="/pubmed/19048100" ref="ordinalpos=795&amp;ncbi_uid=1614741&amp;link_uid=19048100">Citation</a><a class="dblinks" href="/pmc/articles/PMC2585009/" ref="ordinalpos=795&amp;ncbi_uid=1614741&amp;link_uid=PMC2585009">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>796.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3520888/figure/F1/" ref="ordinalpos=796&amp;ncbi_uid=4452849&amp;link_uid=PMC3520888" href="/pmc/articles/PMC3520888/"><img src="/pmc/articles/instance/3520888/bin/sgtp-3-236-g1.gif" src-large="/pmc/articles/instance/3520888/bin/sgtp-3-236-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3520888/" ref="ordinalpos=796&amp;ncbi_uid=4452849&amp;link_uid=PMC3520888" image-link="/pmc/articles/PMC3520888/figure/F1/" class="imagepopup">Figure. From: Escorting Ras. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> A model of the shuttling of H-Ras or N-Ras between the plasma membrane and endosomes. Activated Ras proteins stimulate the RIN1 effector, which then activates and recruits Rabex-5 to ubiquitylate Ras to facilitate retention in endosomes. 14  One consequence of endosomal internalization is suppression of the <span class="highlight" style="background-color:">signaling</span> output from the Ras <span class="highlight" style="background-color:">pathway</span> (I). 13 <sup>–</sup> 15  A fraction of the GTP-bound and ubiquitylated Ras can also stimulate ESCRT-III components CHMP6 and VPS4A to enable sorting and recycling of components from the Ras <span class="highlight" style="background-color:">pathway</span>, such as EGFR (and possibly Ras itself) back to the plasma membrane. This <span class="highlight" style="background-color:">pathway</span> apparently enhances Ras <span class="highlight" style="background-color:">signaling</span> (II). 28  Ubiquitylation of Ras in <span class="highlight" style="background-color:">Pathway</span> I forms a negative feedback loop to more efficiently attenuate Ras <span class="highlight" style="background-color:">signaling</span>, while <span class="highlight" style="background-color:">Pathway</span> II acts in a positive feedback loop for sustained and prolonged Ras <span class="highlight" style="background-color:">signaling</span>. Proper balancing of these two pathways can serve to fine tune Ras <span class="highlight" style="background-color:">signaling</span> outputs. Question marks denote many important questions that hopefully will be addressed in the future (see text).</div></p></div><div class="aux"><div class="resc">Ze-Yi Zheng, et al. Small GTPases. 2012 October 1;3(4):236-239.</div><p class="links"><a class="dblinks" href="/pubmed/22735486" ref="ordinalpos=796&amp;ncbi_uid=4452849&amp;link_uid=22735486">Citation</a><a class="dblinks" href="/pmc/articles/PMC3520888/" ref="ordinalpos=796&amp;ncbi_uid=4452849&amp;link_uid=PMC3520888">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>797.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4072777/figure/ppat-1004165-g002/" ref="ordinalpos=797&amp;ncbi_uid=6317528&amp;link_uid=PMC4072777" href="/pmc/articles/PMC4072777/"><img src="/pmc/articles/instance/4072777/bin/ppat.1004165.g002.gif" src-large="/pmc/articles/instance/4072777/bin/ppat.1004165.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4072777/" ref="ordinalpos=797&amp;ncbi_uid=6317528&amp;link_uid=PMC4072777" image-link="/pmc/articles/PMC4072777/figure/ppat-1004165-g002/" class="imagepopup">Figure 2. Activation of <span class="highlight" style="background-color:">signaling</span> proteins in the FGFR canonical <span class="highlight" style="background-color:">pathway</span> in splenic macrophages from hamsters with VL..  From: Growth Factor and Th2 Cytokine <span class="highlight" style="background-color:">Signaling</span> Pathways Converge at STAT6 to Promote Arginase Expression in Progressive Experimental Visceral Leishmaniasis. </a></p><div class="supp"><p class="details"><div>(<b>A–L</b>) Splenic macrophages were isolated by adherence from the spleens of uninfected hamsters (time 0) or hamsters infected for 7, 14, and 28 days and whole cell lysates probed with antibodies directed against arg1 (panel A, representative blot A) or members of the FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (panels and representative blots B–L). Bars represent the fold change with reference to control cells of uninfected hamsters calculated by densitometry analysis of immunoblot bands from samples pooled from 1–4 hamsters per determination from 2–3 independent experiments. <b>M</b>) Simplified schematic of the canonical FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for reference.</div></p></div><div class="aux"><div class="resc">E. Yaneth Osorio, et al. PLoS Pathog. 2014 June;10(6):e1004165.</div><p class="links"><a class="dblinks" href="/pubmed/24967908" ref="ordinalpos=797&amp;ncbi_uid=6317528&amp;link_uid=24967908">Citation</a><a class="dblinks" href="/pmc/articles/PMC4072777/" ref="ordinalpos=797&amp;ncbi_uid=6317528&amp;link_uid=PMC4072777">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>798.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3392199/figure/F1/" ref="ordinalpos=798&amp;ncbi_uid=4106535&amp;link_uid=PMC3392199" href="/pmc/articles/PMC3392199/"><img src="/pmc/articles/instance/3392199/bin/nihms284621f1.gif" src-large="/pmc/articles/instance/3392199/bin/nihms284621f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3392199/" ref="ordinalpos=798&amp;ncbi_uid=4106535&amp;link_uid=PMC3392199" image-link="/pmc/articles/PMC3392199/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Pathway</span> analysis identified that the STAT <span class="highlight" style="background-color:">pathway</span> was associated with resistance to the MEK inhibitor AZD6244.  From: STAT3 mediates resistance to MEK inhibitor through microRNA miR-17. </a></p><div class="supp"><p class="details"><div>A, Heat map shows the correlation of gene expression with sensitivity to AZD6244. B, The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> associated with sensitivity or resistance to AZD6244 obtained by using Ingenuity software analysis. STAT <span class="highlight" style="background-color:">pathway</span> is showed in the map to be associated with MEK inhibitor resistance.</div></p></div><div class="aux"><div class="resc">Bingbing Dai, et al. Cancer Res. ;71(10):3658-3668.</div><p class="links"><a class="dblinks" href="/pubmed/21444672" ref="ordinalpos=798&amp;ncbi_uid=4106535&amp;link_uid=21444672">Citation</a><a class="dblinks" href="/pmc/articles/PMC3392199/" ref="ordinalpos=798&amp;ncbi_uid=4106535&amp;link_uid=PMC3392199">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>799.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2982674/figure/F1/" ref="ordinalpos=799&amp;ncbi_uid=2578114&amp;link_uid=PMC2982674" href="/pmc/articles/PMC2982674/"><img src="/pmc/articles/instance/2982674/bin/nihms250318f1.gif" src-large="/pmc/articles/instance/2982674/bin/nihms250318f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2982674/" ref="ordinalpos=799&amp;ncbi_uid=2578114&amp;link_uid=PMC2982674" image-link="/pmc/articles/PMC2982674/figure/F1/" class="imagepopup">Fig. 1.  From: Evaluation of the physiological significance of botrocetin/von Willebrand factor in vitro <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mobilized by GPIbα in response to botrocetin (bt)/ von Willebrand factor (VWF) treatment of suspension washed mouse platelets. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not comprehensive, but provides background information for this <i>in vivo</i> study [4,7,9].</div></p></div><div class="aux"><div class="resc">J. Liu, et al. J Thromb Haemost. ;6(11):1915-1922.</div><p class="links"><a class="dblinks" href="/pubmed/18752568" ref="ordinalpos=799&amp;ncbi_uid=2578114&amp;link_uid=18752568">Citation</a><a class="dblinks" href="/pmc/articles/PMC2982674/" ref="ordinalpos=799&amp;ncbi_uid=2578114&amp;link_uid=PMC2982674">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>800.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3342643/figure/F5/" ref="ordinalpos=800&amp;ncbi_uid=3977189&amp;link_uid=PMC3342643" href="/pmc/articles/PMC3342643/"><img src="/pmc/articles/instance/3342643/bin/nihms-373396-f0005.gif" src-large="/pmc/articles/instance/3342643/bin/nihms-373396-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3342643/" ref="ordinalpos=800&amp;ncbi_uid=3977189&amp;link_uid=PMC3342643" image-link="/pmc/articles/PMC3342643/figure/F5/" class="imagepopup">Figure 5. Schematic illustration of TLR <span class="highlight" style="background-color:">signaling</span>.  From: Diacylglycerol kinases in immune cell function and self-tolerance. </a></p><div class="supp"><p class="details"><div>TLR <span class="highlight" style="background-color:">signaling</span> is mediated by the MyD88-dependent and TRIF-dependent <span class="highlight" style="background-color:">pathway</span>. MyD88-dependent <span class="highlight" style="background-color:">pathway</span> is utilized by all TLRs except TLR3 and is initiated by association of MyD88 with the TLRs, followed by subsequent recruitment and activation of IRAKs, TRAF6 and other <span class="highlight" style="background-color:">signaling</span> molecules to the cytoplasm membrane. The <span class="highlight" style="background-color:">pathway</span> leads to activation of IKK and nuclear translocation of NFκB to induce expression of its target genes such as IL-12 and TNFα. The TRIF-dependent <span class="highlight" style="background-color:">pathway</span> is utilized by TLR3 and TLR4 and activates IRF3 (IFN-regulatory factor 3) and transcription of IFNα and co-stimulatory molecules. GSK3α is critical for proinflammatory cytokine production induced by TLRs by promoting NFκB-CBP association. The PI3K-Akt <span class="highlight" style="background-color:">pathway</span> can inhibit TLR-induced proinflammatory cytokine production by phosphorylating and deactivating GSK3α. TAK1, transforming growth factor-β-activated kinase 1; TBK1, TRAF family member associated NFκB activator-binding kinase 1.</div></p></div><div class="aux"><div class="resc">Xiao-Ping Zhong, et al. Immunol Rev. ;224:249-264.</div><p class="links"><a class="dblinks" href="/pubmed/18759932" ref="ordinalpos=800&amp;ncbi_uid=3977189&amp;link_uid=18759932">Citation</a><a class="dblinks" href="/pmc/articles/PMC3342643/" ref="ordinalpos=800&amp;ncbi_uid=3977189&amp;link_uid=PMC3342643">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="7" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="8" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="9" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:09:13-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A2E1F53FE8D81000000000023A919;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A2E1F53FE8D81000000000023A919" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>