
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="31" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB4412543113E10000000000617FFB" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="30" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="31" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="32" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="31" /></div>
            <div><h2 class="result_count">Results: 3001 to 3100 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3001.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3914973/figure/pone-0088301-g006/" ref="ordinalpos=3001&amp;ncbi_uid=5792707&amp;link_uid=PMC3914973" href="/pmc/articles/PMC3914973/"><img src="/pmc/articles/instance/3914973/bin/pone.0088301.g006.gif" src-large="/pmc/articles/instance/3914973/bin/pone.0088301.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3914973/" ref="ordinalpos=3001&amp;ncbi_uid=5792707&amp;link_uid=PMC3914973" image-link="/pmc/articles/PMC3914973/figure/pone-0088301-g006/" class="imagepopup">Figure 6. Schematic diagram of the proposed hypoxia-induced PIM2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: A Regulatory Feedback Loop between HIF-1? and PIM2 in HepG2 Cells. </a></p><div class="aux"><div class="resc">Zhenhai Yu, et al. PLoS One. 2014;9(2):e88301.</div><p class="links"><a class="dblinks" href="/pubmed/24505470" ref="ordinalpos=3001&amp;ncbi_uid=5792707&amp;link_uid=24505470">Citation</a><a class="dblinks" href="/pmc/articles/PMC3914973/" ref="ordinalpos=3001&amp;ncbi_uid=5792707&amp;link_uid=PMC3914973">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3002.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3863464/figure/fig6/" ref="ordinalpos=3002&amp;ncbi_uid=5637604&amp;link_uid=PMC3863464" href="/pmc/articles/PMC3863464/"><img src="/pmc/articles/instance/3863464/bin/MI2013-518183.006.gif" src-large="/pmc/articles/instance/3863464/bin/MI2013-518183.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3863464/" ref="ordinalpos=3002&amp;ncbi_uid=5637604&amp;link_uid=PMC3863464" image-link="/pmc/articles/PMC3863464/figure/fig6/" class="imagepopup">Figure 6.  From: IRAK1/4-Targeted Anti-Inflammatory Action of Caffeic Acid. </a></p><div class="supp"><p class="details"><div>Putative CA-mediated inhibitory <span class="highlight" style="background-color:">pathway</span> of inflammatory <span class="highlight" style="background-color:">signaling</span> events.</div></p></div><div class="aux"><div class="resc">Woo Seok Yang, et al. Mediators Inflamm. 2013;2013:518183.</div><p class="links"><a class="dblinks" href="/pubmed/24379523" ref="ordinalpos=3002&amp;ncbi_uid=5637604&amp;link_uid=24379523">Citation</a><a class="dblinks" href="/pmc/articles/PMC3863464/" ref="ordinalpos=3002&amp;ncbi_uid=5637604&amp;link_uid=PMC3863464">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3003.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3829308/figure/F0001/" ref="ordinalpos=3003&amp;ncbi_uid=5496100&amp;link_uid=PMC3829308" href="/pmc/articles/PMC3829308/"><img src="/pmc/articles/instance/3829308/bin/IJNL-12-149-g001.gif" src-large="/pmc/articles/instance/3829308/bin/IJNL-12-149-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3829308/" ref="ordinalpos=3003&amp;ncbi_uid=5496100&amp;link_uid=PMC3829308" image-link="/pmc/articles/PMC3829308/figure/F0001/" class="imagepopup">Figure 1.  From: The molecular study of IFN? pleiotropic roles in MS treatment. </a></p><div class="supp"><p class="details"><div>Interferon beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Maryam Kay, et al. Iran J Neurol. 2013;12(4):149-156.</div><p class="links"><a class="dblinks" href="/pubmed/24250925" ref="ordinalpos=3003&amp;ncbi_uid=5496100&amp;link_uid=24250925">Citation</a><a class="dblinks" href="/pmc/articles/PMC3829308/" ref="ordinalpos=3003&amp;ncbi_uid=5496100&amp;link_uid=PMC3829308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3004.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC153558/figure/F1/" ref="ordinalpos=3004&amp;ncbi_uid=284624&amp;link_uid=PMC153558" href="/pmc/articles/PMC153558/"><img src="/pmc/articles/instance/153558/bin/pq0931010001.gif" src-large="/pmc/articles/instance/153558/bin/pq0931010001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC153558/" ref="ordinalpos=3004&amp;ncbi_uid=284624&amp;link_uid=PMC153558" image-link="/pmc/articles/PMC153558/figure/F1/" class="imagepopup">Figure 1.  From: Radar breaks the fog: Insights into dorsoventral patterning in zebrafish. </a></p><div class="supp"><p class="details"><div>BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (see text for details).</div></p></div><div class="aux"><div class="resc">Thomas P. Wilm, et al. Proc Natl Acad Sci U S A. 2003 April 15;100(8):4363-4365.</div><p class="links"><a class="dblinks" href="/pubmed/12682283" ref="ordinalpos=3004&amp;ncbi_uid=284624&amp;link_uid=12682283">Citation</a><a class="dblinks" href="/pmc/articles/PMC153558/" ref="ordinalpos=3004&amp;ncbi_uid=284624&amp;link_uid=PMC153558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3005.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517193/figure/F1/" ref="ordinalpos=3005&amp;ncbi_uid=4434015&amp;link_uid=PMC3517193" href="/pmc/articles/PMC3517193/"><img src="/pmc/articles/instance/3517193/bin/nihms414800f1.gif" src-large="/pmc/articles/instance/3517193/bin/nihms414800f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517193/" ref="ordinalpos=3005&amp;ncbi_uid=4434015&amp;link_uid=PMC3517193" image-link="/pmc/articles/PMC3517193/figure/F1/" class="imagepopup">Figure 1.  From: Sensing the Insulin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> with an Antibody Array. </a></p><div class="supp"><p class="details"><div>Graphic illustration of the perturbation of the insulin signal transduction pathways by cellular stressors. The binding of insulin to its receptor triggers the activation of downstream <span class="highlight" style="background-color:">signaling</span> cascades important for the metabolic and mitogenic actions of insulin. The phosphorylation of insulin receptor substrates on select tyrosine residues leads to the activation of PI-3 kinase with subsequent increase in the activity of serine/threonine kinase Akt. Insulin also activates the Ras/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and gene transcription. Under stressful conditions, the inflammatory pathways are activated, leading to the phosphorylation of IRS-1 at serine 307 by the kinases IKKβ and JNK1 and subsequent attenuation in insulin signal transduction. Activation of the lipid phosphatase PTEN decreases PI-3 kinase activity and, therefore, reduces Akt <span class="highlight" style="background-color:">signaling</span> and activation of downstream target proteins involved in multiple aspects of cell physiology.</div></p></div><div class="aux"><div class="resc">Hua-Jun He, et al. Proteomics Clin Appl. ;3(12):1440-1450.</div><p class="links"><a class="dblinks" href="/pubmed/21136963" ref="ordinalpos=3005&amp;ncbi_uid=4434015&amp;link_uid=21136963">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517193/" ref="ordinalpos=3005&amp;ncbi_uid=4434015&amp;link_uid=PMC3517193">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3006.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3398461/figure/F2/" ref="ordinalpos=3006&amp;ncbi_uid=4121544&amp;link_uid=PMC3398461" href="/pmc/articles/PMC3398461/"><img src="/pmc/articles/instance/3398461/bin/nihms386791f2.gif" src-large="/pmc/articles/instance/3398461/bin/nihms386791f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3398461/" ref="ordinalpos=3006&amp;ncbi_uid=4121544&amp;link_uid=PMC3398461" image-link="/pmc/articles/PMC3398461/figure/F2/" class="imagepopup">Figure 2.  From: PI3K <span class="highlight" style="background-color:">pathway</span> alterations in cancer: variations on a theme. </a></p><div class="supp"><p class="details"><div>Possible models of oncogenicity. In model 1, genetic alterations in any of the PI3K <span class="highlight" style="background-color:">pathway</span> members lead to amplification of PI3K <span class="highlight" style="background-color:">signaling</span>. In model 2, alterations of RTKs, RAS or PTEN in addition to mutant PIK3CA lead to activation of non-overlapping pathways or disrupt negative feedback loops to enhance PI3K <span class="highlight" style="background-color:">signaling</span>. In model 3, oncogenic RAS leads to tumorigenesis via a mechanism independent of PI3K. In model 4, loss of PTEN leads to p53-dependent cellular senescence, which may be overcome by acquisition of an additional mutation in the <span class="highlight" style="background-color:">pathway</span>. In model 5, activation of PI3K is enhanced through interactions with oncogenic RAS. PI3K, phosphoinositide 3-kinase; RTKs, receptor tyrosine kinases.</div></p></div><div class="aux"><div class="resc">TL Yuan, et al. Oncogene. ;27(41):5497-5510.</div><p class="links"><a class="dblinks" href="/pubmed/18794884" ref="ordinalpos=3006&amp;ncbi_uid=4121544&amp;link_uid=18794884">Citation</a><a class="dblinks" href="/pmc/articles/PMC3398461/" ref="ordinalpos=3006&amp;ncbi_uid=4121544&amp;link_uid=PMC3398461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3007.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2559911/figure/F7/" ref="ordinalpos=3007&amp;ncbi_uid=1540565&amp;link_uid=PMC2559911" href="/pmc/articles/PMC2559911/"><img src="/pmc/articles/instance/2559911/bin/2721fig7.gif" src-large="/pmc/articles/instance/2559911/bin/2721fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2559911/" ref="ordinalpos=3007&amp;ncbi_uid=1540565&amp;link_uid=PMC2559911" image-link="/pmc/articles/PMC2559911/figure/F7/" class="imagepopup">Figure 7. From: SHC-1/p52Shc targets the insulin/IGF-1 and JNK <span class="highlight" style="background-color:">signaling</span> pathways to modulate life span and stress response in C. elegans. </a></p><div class="supp"><p class="details"><div>Model for the function of SHC-1 in IIS and JNK <span class="highlight" style="background-color:">signaling</span>. Upon DAF-2 activation, AGE-1 and PIP3 activate PDK-1 and the downstream kinase complex AKT-1/AKT-2/SGK-1. The transcription factor DAF-16 is directly inhibited and phosphorylated via activity of the IIS <span class="highlight" style="background-color:">pathway</span>, a function of which is induction of stress response and longevity genes. In addition, DAF-16 integrates positive signals via JNK <span class="highlight" style="background-color:">signaling</span> stimulating nuclear accumulation. Genetic and biochemical analyses presented here suggest that SHC-1 opposes IIS upstream of AGE-1 and DAF-2 and directly interacts with DAF-2. Furthermore, SHC-1 activates JNK <span class="highlight" style="background-color:">signaling</span> and interacts with MEK-1 to control life span and response against heat, oxidation, and heavy metals.</div></p></div><div class="aux"><div class="resc">Elke Neumann-Haefelin, et al. Genes Dev. 2008 October 1;22(19):2721-2735.</div><p class="links"><a class="dblinks" href="/pubmed/18832074" ref="ordinalpos=3007&amp;ncbi_uid=1540565&amp;link_uid=18832074">Citation</a><a class="dblinks" href="/pmc/articles/PMC2559911/" ref="ordinalpos=3007&amp;ncbi_uid=1540565&amp;link_uid=PMC2559911">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3008.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4287412/figure/F1/" ref="ordinalpos=3008&amp;ncbi_uid=7048712&amp;link_uid=PMC4287412" href="/pmc/articles/PMC4287412/"><img src="/pmc/articles/instance/4287412/bin/nihms640187f1.gif" src-large="/pmc/articles/instance/4287412/bin/nihms640187f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4287412/" ref="ordinalpos=3008&amp;ncbi_uid=7048712&amp;link_uid=PMC4287412" image-link="/pmc/articles/PMC4287412/figure/F1/" class="imagepopup">Fig. 1.  From: Role of non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> in osteoblast maturation on microstructured titanium surfaces. </a></p><div class="supp"><p class="details"><div>Regulation of Wnt <span class="highlight" style="background-color:">pathway</span> activators and canonical molecules in MG63 cells grown on microstructured Ti surfaces. Expression of Wnt <span class="highlight" style="background-color:">pathway</span> activators WNT1 (A), WNT3A (B), WNT5A (C), WNT7B (D), WNT10B (E), and WNT11 (F) were measured by real-time PCR. Expression of activation (CTNNB (G)) and inhibition (AXIN2 (H)) of canonical Wnt <span class="highlight" style="background-color:">signaling</span> were also measured. Osteoblast maturation was confirmed by OCN expression (I). *<i>p</i> &lt; 0.05, vs. TCPS; <sup>#</sup><i>p</i> &lt; 0.05, vs. PT; <sup>$</sup><i>p</i> &lt; 0.05, vs. SLA.</div></p></div><div class="aux"><div class="resc">Rene Olivares-Navarrete, et al. Acta Biomater. ;7(6):2740-2750.</div><p class="links"><a class="dblinks" href="/pubmed/21352958" ref="ordinalpos=3008&amp;ncbi_uid=7048712&amp;link_uid=21352958">Citation</a><a class="dblinks" href="/pmc/articles/PMC4287412/" ref="ordinalpos=3008&amp;ncbi_uid=7048712&amp;link_uid=PMC4287412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3009.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3670292/figure/F1/" ref="ordinalpos=3009&amp;ncbi_uid=5316819&amp;link_uid=PMC3670292" href="/pmc/articles/PMC3670292/"><img src="/pmc/articles/instance/3670292/bin/jkst-1-125-g1.gif" src-large="/pmc/articles/instance/3670292/bin/jkst-1-125-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3670292/" ref="ordinalpos=3009&amp;ncbi_uid=5316819&amp;link_uid=PMC3670292" image-link="/pmc/articles/PMC3670292/figure/F1/" class="imagepopup">Figure. From: JAK-STAT <span class="highlight" style="background-color:">signaling</span> in cardiomyogenesis of cardiac stem cells. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> The proposed mechanism of STAT3-mediated vasculogenesis by cardiac stem/progenitor cells. (A) In the vasculogenesis by bone marrow-derived cells, endothelial progenitor cells migrate from the bone marrow, home to peripheral organs, and participate in the postnatal neovascularization. (B) JAK-STAT <span class="highlight" style="background-color:">pathway</span> in cardiac resident stem/progenitor cells may contribute to neovascularlization by inducing the endothelial differentiation. IL-6-type cytokines, secreted from myocardium, stimulate gp130-JAK-STAT-Pim-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cardiac resident stem/progenitor cells, and induce the endothelial differentiation. The newly differentiated endothelial cells may participate in vessel formation without homing process, designated as “in situ vasculogenesis.”</div></p></div><div class="aux"><div class="resc">Tomomi Mohri, et al. JAKSTAT. 2012 April 1;1(2):125-130.</div><p class="links"><a class="dblinks" href="/pubmed/24058761" ref="ordinalpos=3009&amp;ncbi_uid=5316819&amp;link_uid=24058761">Citation</a><a class="dblinks" href="/pmc/articles/PMC3670292/" ref="ordinalpos=3009&amp;ncbi_uid=5316819&amp;link_uid=PMC3670292">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3010.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1924805/figure/F8/" ref="ordinalpos=3010&amp;ncbi_uid=978288&amp;link_uid=PMC1924805" href="/pmc/articles/PMC1924805/"><img src="/pmc/articles/instance/1924805/bin/zmk0070781270008.gif" src-large="/pmc/articles/instance/1924805/bin/zmk0070781270008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1924805/" ref="ordinalpos=3010&amp;ncbi_uid=978288&amp;link_uid=PMC1924805" image-link="/pmc/articles/PMC1924805/figure/F8/" class="imagepopup">Figure 8. From: Inhibition of Integrin-mediated Crosstalk with Epidermal Growth Factor Receptor/Erk or Src <span class="highlight" style="background-color:">Signaling</span> Pathways in Autophagic Prostate Epithelial Cells Induces Caspase-independent Death. </a></p><div class="supp"><p class="details"><div>Model for LM5-mediated survival. Adhesion of growth factor-deprived PECs to LM5 via α3β1 and α6β4 integrin mediates cell survival by maintaining starvation-induced autophagy. <span class="highlight" style="background-color:">Signaling</span> via α3β1 to Erk through EGFR or through Src is also required for cell survival. The PI-3K/Akt <span class="highlight" style="background-color:">pathway</span> is not activated on LM5 and not required for survival. Disruption of either the EGFR/Erk or Src <span class="highlight" style="background-color:">pathway</span> leads to caspase-independent cell death, due to the generation of reactive oxygen species (ROS), whereas disruption of autophagy leads to caspase activation and death.</div></p></div><div class="aux"><div class="resc">Mathew J. Edick, et al. Mol Biol Cell. 2007 July;18(7):2481-2490.</div><p class="links"><a class="dblinks" href="/pubmed/17475774" ref="ordinalpos=3010&amp;ncbi_uid=978288&amp;link_uid=17475774">Citation</a><a class="dblinks" href="/pmc/articles/PMC1924805/" ref="ordinalpos=3010&amp;ncbi_uid=978288&amp;link_uid=PMC1924805">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3011.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4053448/figure/pone-0099137-g005/" ref="ordinalpos=3011&amp;ncbi_uid=6259755&amp;link_uid=PMC4053448" href="/pmc/articles/PMC4053448/"><img src="/pmc/articles/instance/4053448/bin/pone.0099137.g005.gif" src-large="/pmc/articles/instance/4053448/bin/pone.0099137.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4053448/" ref="ordinalpos=3011&amp;ncbi_uid=6259755&amp;link_uid=PMC4053448" image-link="/pmc/articles/PMC4053448/figure/pone-0099137-g005/" class="imagepopup">Figure 5. Dose-dependent estrogen inhibits osteo-adipogenic transdifferentiation of MC3T3-E1 cells likely via modulating canonical Wnt singling <span class="highlight" style="background-color:">pathway</span>..  From: Dose-Dependent Effect of Estrogen Suppresses the Osteo-Adipogenic Transdifferentiation of Osteoblasts via Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>After 14 days’ osteogenesis, two groups of MC3T3-E1 cells were divided. The first group was cultured in adipogenic cocktail medium accompanied with 10<sup>−7</sup> M of 17beta-estradiol for 14 days and 50 ng/ml DKK-1 was added to block canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> simultaneously. The second group was cultured in adipogenic cocktail medium accompanied with 10<sup>−7</sup> M of 17beta-estradiol and ICI together for 14 days and 50 ng/ml Wnt-3a was added to activate canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> simultaneously. O14+A14∶14 days’ adipogenesis after 14 days’ osteogenesis; O14+A14+E<sub>2</sub>∶14 days’ adipogenesis accompanied with 10<sup>−7</sup> M of 17beta-estradiol after 14 days’ osteogenesis; O14+A14+E<sub>2</sub>+ICI: 14 days’ adipogenesis accompanied with 10<sup>−7</sup> M of 17beta-estradiol and ICI after 14 days’ osteogenesis. O14+A14+E<sub>2</sub>+ICI+Wnt-3a: 14 days’ adipogenesis accompanied with 10<sup>−7</sup> M of 17beta-estradiol, ICI and 50 ng/ml Wnt-3a after 14 days’ osteogenesis; O14+A14+E<sub>2</sub>+DKK-1∶14 days adipogenesis accompanied with 10<sup>−7</sup> M 17beta-estradiol and 50 ng/ml DKK1 after 14 days’ osteogenesis. A: ALP staining of osteo-adipogenic transdifferentiation of MC3T3-E1 cells; B: Alizarin red staining of osteo-adipogenic transdifferentiation of MC3T3-E1 cells; C: Oil red staining of osteo-adipogenic transdifferentiation of MC3T3-E1 cells; D: ALP activity of osteo-adipogenic transdifferentiation of MC3T3-E1 cells. The control value for ALP activity was 0.687±0.014 unit/mg protein; E: The mineralization of osteo-adipogenic transdifferentiation of MC3T3-E1 cells. The control value for mineralization was 0.795±0.020 OD; F: Quantification of lipid; G: Effect of Wnts and 17beta-estradiol on the osteogenic mRNA expression; H: Effect of Wnts and 17beta-estradiol on the adipogenic mRNA expression; I: Effect of Wnts and 17beta-estradiol on the canonical Wnt <span class="highlight" style="background-color:">signaling</span> mRNA expression. Expression of each target gene was calculated as a relative expression to beta-actin and represented as normalized fold expression. Data are represented as mean±SD of 3 independent experiments. *P&lt;0.05 and **P&lt;0.01.</div></p></div><div class="aux"><div class="resc">Bo Gao, et al. PLoS One. 2014;9(6):e99137.</div><p class="links"><a class="dblinks" href="/pubmed/24918446" ref="ordinalpos=3011&amp;ncbi_uid=6259755&amp;link_uid=24918446">Citation</a><a class="dblinks" href="/pmc/articles/PMC4053448/" ref="ordinalpos=3011&amp;ncbi_uid=6259755&amp;link_uid=PMC4053448">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3012.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2848195/figure/F1/" ref="ordinalpos=3012&amp;ncbi_uid=2314608&amp;link_uid=PMC2848195" href="/pmc/articles/PMC2848195/"><img src="/pmc/articles/instance/2848195/bin/1752-0509-4-20-1.gif" src-large="/pmc/articles/instance/2848195/bin/1752-0509-4-20-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2848195/" ref="ordinalpos=3012&amp;ncbi_uid=2314608&amp;link_uid=PMC2848195" image-link="/pmc/articles/PMC2848195/figure/F1/" class="imagepopup">Figure 1.  From: Integrated cellular network of transcription regulations and protein-protein interactions. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the integrated cellular network</b>. The integrated cellular network consists of two subnetworks. One is the <span class="highlight" style="background-color:">signaling</span> regulatory <span class="highlight" style="background-color:">pathway</span> (the green part in the figure) in which protein-protein interactions are presented and the other is the gene regulatory network (the yellow part in the figure) in which transcription regulations exist. The transcription factors serve as the interface between the <span class="highlight" style="background-color:">signaling</span> regulatory <span class="highlight" style="background-color:">pathway</span> and the gene regulatory network.</div></p></div><div class="aux"><div class="resc">Yu-Chao Wang, et al. BMC Syst Biol. 2010;4:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/20211003" ref="ordinalpos=3012&amp;ncbi_uid=2314608&amp;link_uid=20211003">Citation</a><a class="dblinks" href="/pmc/articles/PMC2848195/" ref="ordinalpos=3012&amp;ncbi_uid=2314608&amp;link_uid=PMC2848195">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3013.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2099484/figure/F9/" ref="ordinalpos=3013&amp;ncbi_uid=1119857&amp;link_uid=PMC2099484" href="/pmc/articles/PMC2099484/"><img src="/pmc/articles/instance/2099484/bin/1471-2105-8-S7-S16-9.gif" src-large="/pmc/articles/instance/2099484/bin/1471-2105-8-S7-S16-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2099484/" ref="ordinalpos=3013&amp;ncbi_uid=1119857&amp;link_uid=PMC2099484" image-link="/pmc/articles/PMC2099484/figure/F9/" class="imagepopup">Figure 9.  From: Computational analysis of biological functions and pathways collectively targeted by co-expressed microRNAs in cancer. </a></p><div class="supp"><p class="details"><div><b>Diagram of p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. A. Genes known to be affected in Colon Cancer are shown in gray. B. Genes from significantly enriched GO categories that are targeted by at least 50% of over-expressed miRNAs from Colon Cancer dataset are shown in gray. Drug that are known to target this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are outlined in dark blue. Drug targets are outlined in light blue.</div></p></div><div class="aux"><div class="resc">Yuriy Gusev, et al. BMC Bioinformatics. 2007;8(Suppl 7):S16-S16.</div><p class="links"><a class="dblinks" href="/pubmed/18047715" ref="ordinalpos=3013&amp;ncbi_uid=1119857&amp;link_uid=18047715">Citation</a><a class="dblinks" href="/pmc/articles/PMC2099484/" ref="ordinalpos=3013&amp;ncbi_uid=1119857&amp;link_uid=PMC2099484">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3014.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3396691/figure/F1/" ref="ordinalpos=3014&amp;ncbi_uid=4119646&amp;link_uid=PMC3396691" href="/pmc/articles/PMC3396691/"><img src="/pmc/articles/instance/3396691/bin/viru-3-109-g1.gif" src-large="/pmc/articles/instance/3396691/bin/viru-3-109-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3396691/" ref="ordinalpos=3014&amp;ncbi_uid=4119646&amp;link_uid=PMC3396691" image-link="/pmc/articles/PMC3396691/figure/F1/" class="imagepopup">Figure. From: Protein kinase A and fungal virulence. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> (A) Schematic of various <span class="highlight" style="background-color:">signaling</span> inputs and regulatory pathways that govern PKA activity in <i>S. cerevisiae</i>. The cAMP-independent activation of PKA by ammonium (NH<sub>4</sub><sup>+</sup>), amino acids and phosphate (P<sub>i</sub>) make up the ‘fermentable-growth medium (FGM)’ <span class="highlight" style="background-color:">pathway</span>. (B) Left: <span class="highlight" style="background-color:">Signaling</span> inputs that activate PKA in the human fungal pathogen, <i>C. albicans</i> (top). Following activation, PKA induces the yeast-to-hypha transition that promotes invasion of the gut epithelium (bottom). MDPs, muramyl dipeptides. Right: Novel <span class="highlight" style="background-color:">signaling</span> inputs and regulatory mechanisms in the plant pathogenic species <i>F. solani</i> (blue <span class="highlight" style="background-color:">pathway</span>) and <i>M. oryzae</i> (red <span class="highlight" style="background-color:">pathway</span>) (top). In <i>M. oryzae</i>, the activation of PKA leads to the formation of the appressorium (AP), which promotes penetration through the outer plant cuticle into the underlying tissue (bottom).</div></p></div><div class="aux"><div class="resc">Kevin K. Fuller, et al. Virulence. 2012 March 1;3(2):109-121.</div><p class="links"><a class="dblinks" href="/pubmed/22460637" ref="ordinalpos=3014&amp;ncbi_uid=4119646&amp;link_uid=22460637">Citation</a><a class="dblinks" href="/pmc/articles/PMC3396691/" ref="ordinalpos=3014&amp;ncbi_uid=4119646&amp;link_uid=PMC3396691">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3015.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3690520/figure/F3/" ref="ordinalpos=3015&amp;ncbi_uid=5653857&amp;link_uid=PMC3690520" href="/pmc/articles/PMC3690520/"><img src="/pmc/articles/instance/3690520/bin/nihms481925f3.gif" src-large="/pmc/articles/instance/3690520/bin/nihms481925f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3690520/" ref="ordinalpos=3015&amp;ncbi_uid=5653857&amp;link_uid=PMC3690520" image-link="/pmc/articles/PMC3690520/figure/F3/" class="imagepopup">Figure 3.  From: The Role of CARMA1 in T Cells. </a></p><div class="supp"><p class="details"><div>Schematic of TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to NF-κB and JNK activation.</div></p></div><div class="aux"><div class="resc">Marly I. Roche, et al. Crit Rev Immunol. ;33(3):219-243.</div><p class="links"><a class="dblinks" href="/pubmed/23756245" ref="ordinalpos=3015&amp;ncbi_uid=5653857&amp;link_uid=23756245">Citation</a><a class="dblinks" href="/pmc/articles/PMC3690520/" ref="ordinalpos=3015&amp;ncbi_uid=5653857&amp;link_uid=PMC3690520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3016.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2954724/figure/F1/" ref="ordinalpos=3016&amp;ncbi_uid=3310583&amp;link_uid=PMC2954724" href="/pmc/articles/PMC2954724/"><img src="/pmc/articles/instance/2954724/bin/zee0111056530001.gif" src-large="/pmc/articles/instance/2954724/bin/zee0111056530001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2954724/" ref="ordinalpos=3016&amp;ncbi_uid=3310583&amp;link_uid=PMC2954724" image-link="/pmc/articles/PMC2954724/figure/F1/" class="imagepopup">Figure 1.  From: Antiestrogenic Effects of the Novel Sphingosine Kinase-2 Inhibitor ABC294640. </a></p><div class="supp"><p class="details"><div>A, Ceramide-S1P <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; B, structures of ABC294640 and 17β-estradiol (E2).</div></p></div><div class="aux"><div class="resc">James W. Antoon, et al. Endocrinology. 2010 November;151(11):5124-5135.</div><p class="links"><a class="dblinks" href="/pubmed/20861237" ref="ordinalpos=3016&amp;ncbi_uid=3310583&amp;link_uid=20861237">Citation</a><a class="dblinks" href="/pmc/articles/PMC2954724/" ref="ordinalpos=3016&amp;ncbi_uid=3310583&amp;link_uid=PMC2954724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3017.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4020542/figure/fig5/" ref="ordinalpos=3017&amp;ncbi_uid=6169408&amp;link_uid=PMC4020542" href="/pmc/articles/PMC4020542/"><img src="/pmc/articles/instance/4020542/bin/BMRI2014-728019.005.gif" src-large="/pmc/articles/instance/4020542/bin/BMRI2014-728019.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4020542/" ref="ordinalpos=3017&amp;ncbi_uid=6169408&amp;link_uid=PMC4020542" image-link="/pmc/articles/PMC4020542/figure/fig5/" class="imagepopup">Figure 5.  From: Amarogentin, a Secoiridoid Glycoside, Abrogates Platelet Activation through PLCγ2-PKC and MAPK Pathways . </a></p><div class="supp"><p class="details"><div>Hypothesis regarding the inhibitory <span class="highlight" style="background-color:">signaling</span> of amarogentin in platelet activation. Amarogentin may inhibit both the PLC<i>γ</i>2-PKC-p47 cascades and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, ultimately inhibiting platelet activation. DAG: diacylglycerol; IP<sub>3</sub>: inositol 1,4,5-trisphosphate.</div></p></div><div class="aux"><div class="resc">Ting-Lin Yen, et al. Biomed Res Int. 2014;2014:728019.</div><p class="links"><a class="dblinks" href="/pubmed/24868545" ref="ordinalpos=3017&amp;ncbi_uid=6169408&amp;link_uid=24868545">Citation</a><a class="dblinks" href="/pmc/articles/PMC4020542/" ref="ordinalpos=3017&amp;ncbi_uid=6169408&amp;link_uid=PMC4020542">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3018.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2794709/figure/fig2/" ref="ordinalpos=3018&amp;ncbi_uid=2683446&amp;link_uid=PMC2794709" href="/pmc/articles/PMC2794709/"><img src="/pmc/articles/instance/2794709/bin/JCS053678F2.gif" src-large="/pmc/articles/instance/2794709/bin/JCS053678F2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2794709/" ref="ordinalpos=3018&amp;ncbi_uid=2683446&amp;link_uid=PMC2794709" image-link="/pmc/articles/PMC2794709/figure/fig2/" class="imagepopup">Fig. 2. From: The intersections between O-GlcNAcylation and phosphorylation:  implications for multiple <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b><i>O</i>-GlcNAc and phosphate regulate the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>  at the plasma membrane.</b> (<b>A</b>) In the basal state, OGT is concentrated  in the nucleus of the insulin-responsive cell. (<b>B</b>) Upon insulin (I)  binding to its receptor (IR), active PI3K catalyzes the phosphorylation of the  plasma-membrane phospholipid phosphatidylinositol (4,5)-bisphosphate  [PtdIns(4,5)<i>P</i><sub>2</sub>; PIP2] to  PtdIns(3,4,5)<i>P</i><sub>3</sub> (PIP3), driving the translocation of OGT  from the nucleus to the plasma membrane by its binding to  PtdIns(3,4,5)<i>P</i><sub>3</sub>. OGT is then tyrosine phosphorylated by  active IR. This occurs simultaneously with the activation of other molecules  of the <span class="highlight" style="background-color:">pathway</span> such as phosphoinositide-dependent kinase-1 (PDK1), IRS-1 and  Akt, which is mediated in part by phosphorylation events. (<b>C</b>) Active  OGT can now <i>O</i>-GlcNAcylate substrates of the <span class="highlight" style="background-color:">pathway</span>, including IR,  IRS-1, Akt and OGT itself. <i>O</i>-GlcNAcylation of IRS-1 occurs in  parallel with its phosphorylation at several serine residues that are known to  inactivate IRS-1. Comparatively, <i>O</i>-GlcNAcylation of Akt inhibits its  (activating) phosphorylation at a threonine residue, thereby inhibiting its  kinase activity. Thus, the overall effect of <i>O</i>-GlcNAcylation of  insulin <span class="highlight" style="background-color:">signaling</span> molecules leads to attenuation of the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Quira Zeidan, et al. J Cell Sci. 2010 January 1;123(1):13-22.</div><p class="links"><a class="dblinks" href="/pubmed/20016062" ref="ordinalpos=3018&amp;ncbi_uid=2683446&amp;link_uid=20016062">Citation</a><a class="dblinks" href="/pmc/articles/PMC2794709/" ref="ordinalpos=3018&amp;ncbi_uid=2683446&amp;link_uid=PMC2794709">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3019.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2703689/figure/F6/" ref="ordinalpos=3019&amp;ncbi_uid=1922586&amp;link_uid=PMC2703689" href="/pmc/articles/PMC2703689/"><img src="/pmc/articles/instance/2703689/bin/nihms106964f6.gif" src-large="/pmc/articles/instance/2703689/bin/nihms106964f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2703689/" ref="ordinalpos=3019&amp;ncbi_uid=1922586&amp;link_uid=PMC2703689" image-link="/pmc/articles/PMC2703689/figure/F6/" class="imagepopup">Figure 6.  From: Phosphodiesterase 5 Restricts NOS3/Soluble Guanylate Cyclase <span class="highlight" style="background-color:">signaling</span> to L-type Ca2+ current in Cardiac Myocytes. </a></p><div class="supp"><p class="details"><div>PKG is the primary protein targart for the NOS3/sGC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A) Summary data (mean±SEM) of the effect of KT5823 (0.05 μmol/L)  on the YC-1 response on <i>I<sub>Ca</sub></i> in WT and NOS3<sup>−/−</sup> myocytes B) Summary data (mean±SEM) of the effects of KT5823  on the YC-1 response on shortening and Ca<sup>2+</sup> transient amplitudes in NOS3<sup>−/−</sup> myocytes. *p&lt;0.05 vs –KT5823. n= 7–16 cells/2–4 hearts.</div></p></div><div class="aux"><div class="resc">Honglan Wang, et al. J Mol Cell Cardiol. ;47(2):304-314.</div><p class="links"><a class="dblinks" href="/pubmed/19345227" ref="ordinalpos=3019&amp;ncbi_uid=1922586&amp;link_uid=19345227">Citation</a><a class="dblinks" href="/pmc/articles/PMC2703689/" ref="ordinalpos=3019&amp;ncbi_uid=1922586&amp;link_uid=PMC2703689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3020.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3958908/figure/f1-ijms-15-03234/" ref="ordinalpos=3020&amp;ncbi_uid=5929738&amp;link_uid=PMC3958908" href="/pmc/articles/PMC3958908/"><img src="/pmc/articles/instance/3958908/bin/ijms-15-03234f1.gif" src-large="/pmc/articles/instance/3958908/bin/ijms-15-03234f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3958908/" ref="ordinalpos=3020&amp;ncbi_uid=5929738&amp;link_uid=PMC3958908" image-link="/pmc/articles/PMC3958908/figure/f1-ijms-15-03234/" class="imagepopup">Figure 1. From: The Neuroprotective Role of Acupuncture and Activation of the BDNF <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Acupuncture can increase the expression of BDNF (Brain-derived neurotrophic factor) via PI3K/Akt (Phosphatidylinositol-4,5-bisphosphate 3-kinase) and MEK/ERK1/2 (extracellular-signal-regulated kinases) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by activating the tropomyosin receptor kinase B (TrkB), a high affinity catalytic receptor for several neurotrophins that induce the activation of survival <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This ultimately leads to the phosphorylation and activation of the transcription factor CREB (cAMP response element-binding protein) that mediates transcription of BDNF gene expression leading to neuroprotection. In this article, we hypothesize that cyclophilin A (CypA) plays an important role in the activation of BDNF. On the contrary, BDNF may also be an important upstream factor in regulating CypA expression during acupuncture treatment. Thus, the exact mechanism between the effect of acupuncture and the expression of CypA is still unknown.</div></p></div><div class="aux"><div class="resc">Dong Lin, et al. Int J Mol Sci. 2014 February;15(2):3234-3252.</div><p class="links"><a class="dblinks" href="/pubmed/24566146" ref="ordinalpos=3020&amp;ncbi_uid=5929738&amp;link_uid=24566146">Citation</a><a class="dblinks" href="/pmc/articles/PMC3958908/" ref="ordinalpos=3020&amp;ncbi_uid=5929738&amp;link_uid=PMC3958908">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3021.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3361329/figure/fig1/" ref="ordinalpos=3021&amp;ncbi_uid=4045993&amp;link_uid=PMC3361329" href="/pmc/articles/PMC3361329/"><img src="/pmc/articles/instance/3361329/bin/PRI2012-172894.001.gif" src-large="/pmc/articles/instance/3361329/bin/PRI2012-172894.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3361329/" ref="ordinalpos=3021&amp;ncbi_uid=4045993&amp;link_uid=PMC3361329" image-link="/pmc/articles/PMC3361329/figure/fig1/" class="imagepopup">Figure 1.  From: Molecular Mechanisms of Liver Injury and Hepatocarcinogenesis: Focusing on the Role of Stress-Activated MAPK. </a></p><div class="supp"><p class="details"><div>Stress-activated MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> consists of three classes of protein kinase—MAP3K, MAPKK, and MAPK—and converges on JNK and p38 through a kinase cascade. Only representative molecules are shown in MAP3K.</div></p></div><div class="aux"><div class="resc">Hayato Nakagawa, et al. Patholog Res Int. 2012;2012:172894.</div><p class="links"><a class="dblinks" href="/pubmed/22666632" ref="ordinalpos=3021&amp;ncbi_uid=4045993&amp;link_uid=22666632">Citation</a><a class="dblinks" href="/pmc/articles/PMC3361329/" ref="ordinalpos=3021&amp;ncbi_uid=4045993&amp;link_uid=PMC3361329">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3022.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2991480/figure/F5/" ref="ordinalpos=3022&amp;ncbi_uid=3587897&amp;link_uid=PMC2991480" href="/pmc/articles/PMC2991480/"><img src="/pmc/articles/instance/2991480/bin/nihms226012f5.gif" src-large="/pmc/articles/instance/2991480/bin/nihms226012f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2991480/" ref="ordinalpos=3022&amp;ncbi_uid=3587897&amp;link_uid=PMC2991480" image-link="/pmc/articles/PMC2991480/figure/F5/" class="imagepopup">Figure 5.  From: Independent component analysis: mining microarray data for fundamental human gene expression modules. </a></p><div class="supp"><p class="details"><div>FC-66 expression of the Ingenuity NF-κB <span class="highlight" style="background-color:">Signaling</span> canonical <span class="highlight" style="background-color:">pathway</span>. Ingenuity software overlaid gene expression loadings from FC-66 onto a pre-designed <span class="highlight" style="background-color:">pathway</span>. Red = over-expressed in PTL-treated versus untreated, green = under-expressed.</div></p></div><div class="aux"><div class="resc">Jesse M. Engreitz, et al. J Biomed Inform. ;43(6):932-944.</div><p class="links"><a class="dblinks" href="/pubmed/20619355" ref="ordinalpos=3022&amp;ncbi_uid=3587897&amp;link_uid=20619355">Citation</a><a class="dblinks" href="/pmc/articles/PMC2991480/" ref="ordinalpos=3022&amp;ncbi_uid=3587897&amp;link_uid=PMC2991480">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3023.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2712056/figure/fig2/" ref="ordinalpos=3023&amp;ncbi_uid=1918361&amp;link_uid=PMC2712056" href="/pmc/articles/PMC2712056/"><img src="/pmc/articles/instance/2712056/bin/gr2.gif" src-large="/pmc/articles/instance/2712056/bin/gr2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2712056/" ref="ordinalpos=3023&amp;ncbi_uid=1918361&amp;link_uid=PMC2712056" image-link="/pmc/articles/PMC2712056/figure/fig2/" class="imagepopup">Figure 2.  From: Mechanisms and Constraints on Yeast MAPK <span class="highlight" style="background-color:">Signaling</span> Specificity. </a></p><div class="supp"><p class="details"><div>A typical numerical result using our stimulation protocol. In the first time window, the input to the mating <span class="highlight" style="background-color:">pathway</span> (<i>x</i>) is high, whereas the input to the filamentation <span class="highlight" style="background-color:">pathway</span> (<i>y</i>) is low. During the second window, the input is reversed. The durations of both windows are set to 200 min. (<i>A</i>) Time traces of our model output (activated Ste12<sup>∗</sup>-Ste12<sup>∗</sup> dimer, <i>S</i><sub>2</sub>, and the activated Ste12<sup>∗</sup>-Tec1<sup>∗</sup> dimer, <i>T</i><sub>2</sub>). (<i>B</i>) Corresponding <span class="highlight" style="background-color:">signaling</span> specificity <i>S</i><sub>X</sub> and <i>S</i><sub>Y</sub> computed from panel <i>A</i>.</div></p></div><div class="aux"><div class="resc">Bo Hu, et al. Biophys J. 2009 June 17;96(12):4755-4763.</div><p class="links"><a class="dblinks" href="/pubmed/19527636" ref="ordinalpos=3023&amp;ncbi_uid=1918361&amp;link_uid=19527636">Citation</a><a class="dblinks" href="/pmc/articles/PMC2712056/" ref="ordinalpos=3023&amp;ncbi_uid=1918361&amp;link_uid=PMC2712056">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3024.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3092777/figure/pone-0019787-g004/" ref="ordinalpos=3024&amp;ncbi_uid=2952927&amp;link_uid=PMC3092777" href="/pmc/articles/PMC3092777/"><img src="/pmc/articles/instance/3092777/bin/pone.0019787.g004.gif" src-large="/pmc/articles/instance/3092777/bin/pone.0019787.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3092777/" ref="ordinalpos=3024&amp;ncbi_uid=2952927&amp;link_uid=PMC3092777" image-link="/pmc/articles/PMC3092777/figure/pone-0019787-g004/" class="imagepopup">Figure 4. Inhibition of the BMP4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> abolished the pro-cardiogenic effects of apoA-I gene transfer..  From: Exogenous Expression of Human apoA-I Enhances Cardiac Differentiation of Pluripotent Stem Cells. </a></p><div class="supp"><p class="details"><div>The BMP4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was inhibited by the application of noggin (1 µ/ml) to the apoA-I- transduced ESCs upon plating. (A) The phosphorylation status of the differentiating cells was evaluated 24 hr after noggin treatment. (B) The appearance of beating clusters during the 8-day period of differentiation. (C) The percentage of troponin-T positive cells as determined by flow cytometry analysis. Data shown as mean ± SEM from 3 independent experiments, <sup>#</sup> p&lt;0.05 comparing to the LV-apoA-I group; *p&lt;0.05 and ** p&lt;0.005 comparing to the LV-GFP group.</div></p></div><div class="aux"><div class="resc">Kwong-Man Ng, et al. PLoS One. 2011;6(5):e19787.</div><p class="links"><a class="dblinks" href="/pubmed/21589943" ref="ordinalpos=3024&amp;ncbi_uid=2952927&amp;link_uid=21589943">Citation</a><a class="dblinks" href="/pmc/articles/PMC3092777/" ref="ordinalpos=3024&amp;ncbi_uid=2952927&amp;link_uid=PMC3092777">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3025.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3563200/figure/fig04/" ref="ordinalpos=3025&amp;ncbi_uid=4616856&amp;link_uid=PMC3563200" href="/pmc/articles/PMC3563200/"><img src="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f4.gif" src-large="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3563200/" ref="ordinalpos=3025&amp;ncbi_uid=4616856&amp;link_uid=PMC3563200" image-link="/pmc/articles/PMC3563200/figure/fig04/" class="imagepopup">Figure 4.  From: Indoleamine 2,3-dioxygenase-1 (IDO1) enhances survival and invasiveness of endometrial stromal cells via the activation of JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>IDO1 inhibits p53 expression through JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The transfected ESCs were treated with JNK inhibitor (SP600125, 20 μM) or vehicle control (1‰ DMSO) for 24 h. Thereafter, the expressions of proliferation associated proteins survivin (A), p53 (B) were determined by in-cell Western. The columns represent the mean ± SD calculus of 8 different experiments. *P&lt;.05 vs. vector control transfected ESCs without SP600125; **P&lt;.01 vs. vector control transfected ESCs without SP600125; #P&lt;.05, IDO1-overexpressing ESCs with SP600125 vs. IDO1-overexpressing ESCs without SP600125.</div></p></div><div class="aux"><div class="resc">Jie Mei, et al. Int J Clin Exp Pathol. 2013;6(3):431-444.</div><p class="links"><a class="dblinks" href="/pubmed/23411497" ref="ordinalpos=3025&amp;ncbi_uid=4616856&amp;link_uid=23411497">Citation</a><a class="dblinks" href="/pmc/articles/PMC3563200/" ref="ordinalpos=3025&amp;ncbi_uid=4616856&amp;link_uid=PMC3563200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3026.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3513920/figure/F12/" ref="ordinalpos=3026&amp;ncbi_uid=4424824&amp;link_uid=PMC3513920" href="/pmc/articles/PMC3513920/"><img src="/pmc/articles/instance/3513920/bin/nihms406195f12.gif" src-large="/pmc/articles/instance/3513920/bin/nihms406195f12.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3513920/" ref="ordinalpos=3026&amp;ncbi_uid=4424824&amp;link_uid=PMC3513920" image-link="/pmc/articles/PMC3513920/figure/F12/" class="imagepopup">Figure 12.  From: Hierarchical Self Assembly of Patterns from the Robinson Tilings: DNA Tile Design in an Enhanced Tile Assembly Model. </a></p><div class="supp"><p class="details"><div>Decision Tile <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>. Some labeled sequences are divided into smaller lengths as indicated by subscripts. a) Activation through a neutral signal <span class="highlight" style="background-color:">pathway</span>, (cyan strands), at the beginning prevents the activation strand, (gold), from causing tile binding. b) The last step of the activation <span class="highlight" style="background-color:">pathway</span> opens the first transducer loop (magenta). c) The central downhill transducer cascade, (magenta strands), is made energetically favorable by the gain of four base pairs at each step. Another neutral <span class="highlight" style="background-color:">pathway</span>, (blue strands), transmits the signal to the binding site, (blue strands at the edge of the tile), to activate it by removing a protecting strand. d) After the last transducer loop has opened, the signal may be transmitted to a neighboring tile, allowing the transmission of signals to the edge of an assembly. e) The last transducer strand can instead activate the binding site through a neutral <span class="highlight" style="background-color:">pathway</span>. If the tile is at the edge of an assembly, this makes the edge active and ready for the next level of assembly. Otherwise, the neutral <span class="highlight" style="background-color:">pathway</span> will be able to reverse and release the transducer strand, giving it another chance to pass the signal to a neighboring tile.</div></p></div><div class="aux"><div class="resc">Jennifer E. Padilla, et al. Nat Comput. ;11(2):323-338.</div><p class="links"><a class="dblinks" href="/pubmed/23226722" ref="ordinalpos=3026&amp;ncbi_uid=4424824&amp;link_uid=23226722">Citation</a><a class="dblinks" href="/pmc/articles/PMC3513920/" ref="ordinalpos=3026&amp;ncbi_uid=4424824&amp;link_uid=PMC3513920">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3027.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3384380/figure/DDS132F2/" ref="ordinalpos=3027&amp;ncbi_uid=5092930&amp;link_uid=PMC3384380" href="/pmc/articles/PMC3384380/"><img src="/pmc/articles/instance/3384380/bin/dds13202.gif" src-large="/pmc/articles/instance/3384380/bin/dds13202.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3384380/" ref="ordinalpos=3027&amp;ncbi_uid=5092930&amp;link_uid=PMC3384380" image-link="/pmc/articles/PMC3384380/figure/DDS132F2/" class="imagepopup">Figure 2. From: Forkhead box protein p1 is a transcriptional repressor of immune <span class="highlight" style="background-color:">signaling</span> in the CNS: implications for transcriptional dysregulation in Huntington disease. </a></p><div class="supp"><p class="details"><div>Schematic depiction of NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> highlighting <i>FOXP1</i> target genes. Canonical pathways analysis was performed using Ingenuity Pathways Analysis. Foxp1 target genes were significantly associated with the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Fisher's exact test; <i>P</i> = 1.9E−16). Green indicates downregulated genes and red indicates upregulated genes in response to <i>FOXP1</i> overexpression in the striatum. Dark green indicates a greater magnitude of expression change compared to light green. With permission from Ingenuity Systems Inc. © 2000–2012. All rights reserved.</div></p></div><div class="aux"><div class="resc">Bin Tang, et al. Hum Mol Genet. 2012 July 15;21(14):3097-3111.</div><p class="links"><a class="dblinks" href="/pubmed/22492998" ref="ordinalpos=3027&amp;ncbi_uid=5092930&amp;link_uid=22492998">Citation</a><a class="dblinks" href="/pmc/articles/PMC3384380/" ref="ordinalpos=3027&amp;ncbi_uid=5092930&amp;link_uid=PMC3384380">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3028.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3196825/figure/F7/" ref="ordinalpos=3028&amp;ncbi_uid=3487051&amp;link_uid=PMC3196825" href="/pmc/articles/PMC3196825/"><img src="/pmc/articles/instance/3196825/bin/nihms307766f7.gif" src-large="/pmc/articles/instance/3196825/bin/nihms307766f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3196825/" ref="ordinalpos=3028&amp;ncbi_uid=3487051&amp;link_uid=PMC3196825" image-link="/pmc/articles/PMC3196825/figure/F7/" class="imagepopup">Figure 7. Schematic overview showing the effects of radiation and BV6 on the apoptotic <span class="highlight" style="background-color:">signaling</span> cascade in HCC193 and H460 cell lines.  From: BV6, an IAP antagonist, activates apoptosis and enhances radiosensitization of non-small cell lung carcinoma in vitro. </a></p><div class="supp"><p class="details"><div>(A) In HCC193 cells, apoptosis is primarily activated through the extrinsic <span class="highlight" style="background-color:">pathway</span>. Cell surface death receptors are activated by ligands such as TNFα and TRAIL, leading to the activation of the extrinsic apoptotic <span class="highlight" style="background-color:">signaling</span> cascade through initiator caspase-8. IAPs exert their anti-apoptotic activity by inhibiting both this cascade as well as TNFα-mediated NF-κB <span class="highlight" style="background-color:">signaling</span>. BV6 blocks these effects by antagonizing IAPs, inducing their degradation, and enhancing TNFα secretion. (B) In H460 cells, apoptosis is primarily mediated through the intrinsic <span class="highlight" style="background-color:">pathway</span>. Here, cell stress triggers mitochondria to release the Apaf-1/cytochrome c complex, leading to the activation of the intrinsic <span class="highlight" style="background-color:">pathway</span> through caspase-9. BV6 prevents IAPs from inhibiting this cascade, and also induces their proteosomal degradation.</div></p></div><div class="aux"><div class="resc">Wenyan Li, et al. J Thorac Oncol. ;6(11):1801-1809.</div><p class="links"><a class="dblinks" href="/pubmed/21760551" ref="ordinalpos=3028&amp;ncbi_uid=3487051&amp;link_uid=21760551">Citation</a><a class="dblinks" href="/pmc/articles/PMC3196825/" ref="ordinalpos=3028&amp;ncbi_uid=3487051&amp;link_uid=PMC3196825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3029.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2772978/figure/F6/" ref="ordinalpos=3029&amp;ncbi_uid=2069742&amp;link_uid=PMC2772978" href="/pmc/articles/PMC2772978/"><img src="/pmc/articles/instance/2772978/bin/emm-41-757-g006.gif" src-large="/pmc/articles/instance/2772978/bin/emm-41-757-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2772978/" ref="ordinalpos=3029&amp;ncbi_uid=2069742&amp;link_uid=PMC2772978" image-link="/pmc/articles/PMC2772978/figure/F6/" class="imagepopup">Figure 6.  From: Interleukin-1? promotes the expression of monocyte chemoattractant protein-1 in human aorta smooth muscle cells via multiple <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Schematic representation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of IL-1β-induced MCP1 expression in HASMCs. IL-1β activates PC-PLC to induce PKC activation and translocation of NF-κB to the nucleus. This activation cascade results in increased expression of MCP1. IL-1β additionally activates PLD and tyrosine kinase, which are involved in MCP1 expression independently of NF-κB translocation.</div></p></div><div class="aux"><div class="resc">Jun Hee Lim, et al. Exp Mol Med. 2009 October 31;41(10):757-764.</div><p class="links"><a class="dblinks" href="/pubmed/19561397" ref="ordinalpos=3029&amp;ncbi_uid=2069742&amp;link_uid=19561397">Citation</a><a class="dblinks" href="/pmc/articles/PMC2772978/" ref="ordinalpos=3029&amp;ncbi_uid=2069742&amp;link_uid=PMC2772978">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3030.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2438462/figure/fig8/" ref="ordinalpos=3030&amp;ncbi_uid=1373395&amp;link_uid=PMC2438462" href="/pmc/articles/PMC2438462/"><img src="/pmc/articles/instance/2438462/bin/tpc2001390f08.gif" src-large="/pmc/articles/instance/2438462/bin/tpc2001390f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2438462/" ref="ordinalpos=3030&amp;ncbi_uid=1373395&amp;link_uid=PMC2438462" image-link="/pmc/articles/PMC2438462/figure/fig8/" class="imagepopup">Figure 8. From: MAPK <span class="highlight" style="background-color:">Signaling</span> Regulates Nitric Oxide and NADPH Oxidase-Dependent Oxidative Bursts in Nicotiana benthamiana. </a></p><div class="supp"><p class="details"><div>Scheme of Proposed Model for MAPK Cascade <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Leading to NO and Oxidative Bursts.<br />INF1 induces NO burst by means of MEK2-SIPK/NTF4 cascade and oxidative burst by means of MEK2-SIPK/NTF4 and (NPK1)-MEK1-NTF6 cascades. Based on loss-of-function and gain-of-function analyses, SIPK and NTF4 play important roles in NOA1-mediated NO burst and Nb RBOHB–dependent oxidative burst. NTF6 is also responsible for Nb RBOHB–dependent oxidative burst. The question mark indicates unidentified MAPKKK(s).</div></p></div><div class="aux"><div class="resc">Shuta Asai, et al. Plant Cell. 2008 May;20(5):1390-1406.</div><p class="links"><a class="dblinks" href="/pubmed/18515503" ref="ordinalpos=3030&amp;ncbi_uid=1373395&amp;link_uid=18515503">Citation</a><a class="dblinks" href="/pmc/articles/PMC2438462/" ref="ordinalpos=3030&amp;ncbi_uid=1373395&amp;link_uid=PMC2438462">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3031.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC304101/figure/fig1/" ref="ordinalpos=3031&amp;ncbi_uid=343905&amp;link_uid=PMC304101" href="/pmc/articles/PMC304101/"><img src="/pmc/articles/instance/304101/bin/pq1834202001.gif" src-large="/pmc/articles/instance/304101/bin/pq1834202001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC304101/" ref="ordinalpos=3031&amp;ncbi_uid=343905&amp;link_uid=PMC304101" image-link="/pmc/articles/PMC304101/figure/fig1/" class="imagepopup">Fig. 1. From: Indian hedgehog and ?-catenin <span class="highlight" style="background-color:">signaling</span>: Role in the sebaceous lineage of normal and neoplastic mammalian epidermis. </a></p><div class="supp"><p class="details"><div>Expression of proteins in the PTCH-SHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in K14ΔNLef1 transgenic mouse sebaceous tumors. Immunostaining was performed with antibodies to the proteins indicated. (<i>a</i>) Ptch1 C terminus. (<i>b</i>) Ptch1 N terminus. (<i>c</i>) SHH. (<i>d</i>) IHH. (<i>e</i>) DHH. (<i>f</i>) pan-HH. (<i>g</i>) GLI1 (<i>h</i>) GLI2. (Scale bar: 200 μm.)</div></p></div><div class="aux"><div class="resc">C. Niemann, et al. Proc Natl Acad Sci U S A. 2003 September 30;100(Suppl 1):11873-11880.</div><p class="links"><a class="dblinks" href="/pubmed/12917489" ref="ordinalpos=3031&amp;ncbi_uid=343905&amp;link_uid=12917489">Citation</a><a class="dblinks" href="/pmc/articles/PMC304101/" ref="ordinalpos=3031&amp;ncbi_uid=343905&amp;link_uid=PMC304101">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3032.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC124530/figure/F5/" ref="ordinalpos=3032&amp;ncbi_uid=373616&amp;link_uid=PMC124530" href="/pmc/articles/PMC124530/"><img src="/pmc/articles/instance/124530/bin/pq0920169005.gif" src-large="/pmc/articles/instance/124530/bin/pq0920169005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC124530/" ref="ordinalpos=3032&amp;ncbi_uid=373616&amp;link_uid=PMC124530" image-link="/pmc/articles/PMC124530/figure/F5/" class="imagepopup">Figure 5.  From: Siderophore-mediated <span class="highlight" style="background-color:">signaling</span> regulates virulence factor production in Pseudomonas aeruginosa. </a></p><div class="supp"><p class="details"><div>Model of the FpvA/FpvR/PvdS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (Ferri)pyoverdine complexes bind the FpvA receptor protein, transmitting a signal to the FpvR protein that otherwise suppresses the activity of PvdS. PvdS then binds to RNA polymerase, causing expression of genes required for synthesis of pyoverdine and PrpL protease (<i>prpL</i>) and of the <i>ptxR</i> gene that encodes a transcriptional activator of the <i>toxA</i> gene.</div></p></div><div class="aux"><div class="resc">Iain L. Lamont, et al. Proc Natl Acad Sci U S A. 2002 May 14;99(10):7072-7077.</div><p class="links"><a class="dblinks" href="/pubmed/11997446" ref="ordinalpos=3032&amp;ncbi_uid=373616&amp;link_uid=11997446">Citation</a><a class="dblinks" href="/pmc/articles/PMC124530/" ref="ordinalpos=3032&amp;ncbi_uid=373616&amp;link_uid=PMC124530">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3033.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC137069/figure/F1/" ref="ordinalpos=3033&amp;ncbi_uid=330697&amp;link_uid=PMC137069" href="/pmc/articles/PMC137069/"><img src="/pmc/articles/instance/137069/bin/1475-4924-1-9-1.gif" src-large="/pmc/articles/instance/137069/bin/1475-4924-1-9-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC137069/" ref="ordinalpos=3033&amp;ncbi_uid=330697&amp;link_uid=PMC137069" image-link="/pmc/articles/PMC137069/figure/F1/" class="imagepopup">Figure 1.  From: The therapeutic potential of modulators of the Hedgehog-Gli <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The Hh-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(a)</b> A diagram of the Hh-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, showing the site of action of the agonists (green) and antagonists (red) discussed in the text, as well as many additional factors that affect the <span class="highlight" style="background-color:">pathway</span>. Abbreviations: CK1, Casein kinase 1; Cos2, Costal2 ; Dyrk1, dual-specificity Yak1-related kinase 1; GSK3, Glycogen synthase kinase 3; Fu, Fused; Gas1, growth arrest specific 1; Hh, Hedgehog; Hip, Hedgehog-interacting protein 1; Rab23, a Rab-family Ras-like GTPase associated with vesicle traffic; Ptc, Patched1; PKA, Protein kinase A; Smo, Smoothened; SuFu, Suppressor of Fused. <b>(b)</b> A schematic generalized view of the regulation of Gli activator (Gli-act) and Gli repressor (Gli-rep) forms by Hh <span class="highlight" style="background-color:">signaling</span>. See [2-4] for further details.</div></p></div><div class="aux"><div class="resc">Barbara Stecca, et al. J Biol. 2002;1(2):9-9.</div><p class="links"><a class="dblinks" href="/pubmed/12437768" ref="ordinalpos=3033&amp;ncbi_uid=330697&amp;link_uid=12437768">Citation</a><a class="dblinks" href="/pmc/articles/PMC137069/" ref="ordinalpos=3033&amp;ncbi_uid=330697&amp;link_uid=PMC137069">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3034.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3995778/figure/pone-0095413-g004/" ref="ordinalpos=3034&amp;ncbi_uid=6062707&amp;link_uid=PMC3995778" href="/pmc/articles/PMC3995778/"><img src="/pmc/articles/instance/3995778/bin/pone.0095413.g004.gif" src-large="/pmc/articles/instance/3995778/bin/pone.0095413.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3995778/" ref="ordinalpos=3034&amp;ncbi_uid=6062707&amp;link_uid=PMC3995778" image-link="/pmc/articles/PMC3995778/figure/pone-0095413-g004/" class="imagepopup">Figure 4. Structure and relationships between glutamate <span class="highlight" style="background-color:">signaling</span> SNs overrepresented in AD..  From: Overrepresentation of Glutamate <span class="highlight" style="background-color:">Signaling</span> in Alzheimer's Disease: Network-Based <span class="highlight" style="background-color:">Pathway</span> Enrichment Using Meta-Analysis of Genome-Wide Association Studies. </a></p><div class="supp"><p class="details"><div>SN gene composition (nodes) and interactions (edges) are shown for: Meta-SN1 in the upper left corner with green edges (which includes GW and GATES modules, GNAZ* gene only present in GW); ADNI-GW SN3 in the bottom left corner with dark blue edges; ADNI-GW SN4 in the bottom right corner with blue edges and ADNI-GW SN7 in the upper right corner with light blue edges. Genes shared by at least 2 SNs are located at the center in bold font and cross interactions between genes inside each module are denoted by light grey edges. Node color represents the OR behavior in a gradient from green to red values (i.e. green: OR&lt;1; red OR&gt;1; white: OR = 1), denoting protection and risk, respectively. Similarly, node size is proportional to the –log10 p-value obtained from the meta-analysis (if absent, node size is the minimum). Triangle shaped nodes marks genes belonging to the glutamate <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> GO term (GO:0007215); Diamond shaped nodes denotes genes belonging to KEGG Glutamatergic synapse <span class="highlight" style="background-color:">pathway</span> (hsa04724); Squares Square shaped nodes denotes genes belonging to both gene ontology term GO:0007215 and KEGG hsa04724 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Eduardo Pérez-Palma, et al. PLoS One. 2014;9(4):e95413.</div><p class="links"><a class="dblinks" href="/pubmed/24755620" ref="ordinalpos=3034&amp;ncbi_uid=6062707&amp;link_uid=24755620">Citation</a><a class="dblinks" href="/pmc/articles/PMC3995778/" ref="ordinalpos=3034&amp;ncbi_uid=6062707&amp;link_uid=PMC3995778">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3035.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC37511/figure/F6/" ref="ordinalpos=3035&amp;ncbi_uid=93232&amp;link_uid=PMC37511" href="/pmc/articles/PMC37511/"><img src="/pmc/articles/instance/37511/bin/pq1512360006.gif" src-large="/pmc/articles/instance/37511/bin/pq1512360006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC37511/" ref="ordinalpos=3035&amp;ncbi_uid=93232&amp;link_uid=PMC37511" image-link="/pmc/articles/PMC37511/figure/F6/" class="imagepopup">Figure 6.  From: Activation of NF-?B by nontypeable Hemophilus influenzae is mediated by toll-like receptor 2-TAK1-dependent NIK-IKK?/?-I?B? and MKK3/6-p38 MAP kinase <span class="highlight" style="background-color:">signaling</span> pathways in epithelial cells. </a></p><div class="supp"><p class="details"><div>P6 also induces NF-κB activation via the same TLR2–TAK1-dependent NIK–IKKα/β–IκBα and MKK3/6–p38 MAP kinase <span class="highlight" style="background-color:">signaling</span> pathways. (<i>A</i>) P6, but not LOS and HMW, induces NF-κB activation. HeLa cells were stimulated with either NTHi or polymyxin B-pretreated NTHi, or HMW (1.0 μg/ml) or P6 (0.005–0.5 μg/ml). PB, polymyxin B. (<i>B</i>) Inhibition of NIK–IKKα/β–IκBα <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocks P6-induced NF-κB activation. (<i>C</i>) Inhibition of MKK3/6–p38 MAP kinase <span class="highlight" style="background-color:">pathway</span> abrogates P6-induced NF-κB activation. (<i>D</i>) Overexpression of a dominant-negative mutant of either TAK1 or TLR2 inhibits P6-induced NF-κB activation, whereas overexpression of a wild type of TLR2 enhances P6-induced NF-κB activation. Values are the means ± SD; <i>n</i> = 3.</div></p></div><div class="aux"><div class="resc">Tsuyoshi Shuto, et al. Proc Natl Acad Sci U S A. 2001 July 17;98(15):8774-8779.</div><p class="links"><a class="dblinks" href="/pubmed/11438700" ref="ordinalpos=3035&amp;ncbi_uid=93232&amp;link_uid=11438700">Citation</a><a class="dblinks" href="/pmc/articles/PMC37511/" ref="ordinalpos=3035&amp;ncbi_uid=93232&amp;link_uid=PMC37511">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3036.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4014746/figure/F1/" ref="ordinalpos=3036&amp;ncbi_uid=6116299&amp;link_uid=PMC4014746" href="/pmc/articles/PMC4014746/"><img src="/pmc/articles/instance/4014746/bin/1897-4287-12-13-1.gif" src-large="/pmc/articles/instance/4014746/bin/1897-4287-12-13-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4014746/" ref="ordinalpos=3036&amp;ncbi_uid=6116299&amp;link_uid=PMC4014746" image-link="/pmc/articles/PMC4014746/figure/F1/" class="imagepopup">Figure 1.  From: The role of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in carcinogenesis and implications for anticancer therapeutics. </a></p><div class="supp"><p class="details"><div><b>An overview of the canonical Wnt <span class="highlight" style="background-color:">pathway</span> [</b>[11]<b>].</b></div></p></div><div class="aux"><div class="resc">Asfandyar Sheikh, et al. Hered Cancer Clin Pract. 2014;12(1):13-13.</div><p class="links"><a class="dblinks" href="/pubmed/24817919" ref="ordinalpos=3036&amp;ncbi_uid=6116299&amp;link_uid=24817919">Citation</a><a class="dblinks" href="/pmc/articles/PMC4014746/" ref="ordinalpos=3036&amp;ncbi_uid=6116299&amp;link_uid=PMC4014746">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3037.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3903278/figure/fig8/" ref="ordinalpos=3037&amp;ncbi_uid=5751697&amp;link_uid=PMC3903278" href="/pmc/articles/PMC3903278/"><img src="/pmc/articles/instance/3903278/bin/mbo0011417160008.gif" src-large="/pmc/articles/instance/3903278/bin/mbo0011417160008.jpg" alt="FIG 8 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3903278/" ref="ordinalpos=3037&amp;ncbi_uid=5751697&amp;link_uid=PMC3903278" image-link="/pmc/articles/PMC3903278/figure/fig8/" class="imagepopup">FIG 8 .  From: Epidermal Growth Factor Receptor-PI3K <span class="highlight" style="background-color:">Signaling</span> Controls Cofilin Activity To Facilitate Herpes Simplex Virus 1 Entry into Neuronal Cells. </a></p><div class="supp"><p class="details"><div>Model of cofilin regulating actin dynamics and initiating HSV-1 entry. During early infection, the virus induces cofilin inactivation and F-actin polymerization via the EGFR-PI3K-Erk1/2-ROCK-LIMK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which may promote receptor clustering and initiate the entry process. Lipid rafts are clustered and serve as <span class="highlight" style="background-color:">signaling</span> platforms for EGFR-mediated cofilin inactivation. Subsequent viral penetration requires F-actin depolymerization through cofilin activation. Thus, the biphasic activation of cofilin represents specific cellular machinery usurped by HSV-1.</div></p></div><div class="aux"><div class="resc">Kai Zheng, et al. mBio. 2014 Jan-Feb;5(1):e00958-13.</div><p class="links"><a class="dblinks" href="/pubmed/24425731" ref="ordinalpos=3037&amp;ncbi_uid=5751697&amp;link_uid=24425731">Citation</a><a class="dblinks" href="/pmc/articles/PMC3903278/" ref="ordinalpos=3037&amp;ncbi_uid=5751697&amp;link_uid=PMC3903278">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3038.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4039177/figure/F2/" ref="ordinalpos=3038&amp;ncbi_uid=6181520&amp;link_uid=PMC4039177" href="/pmc/articles/PMC4039177/"><img src="/pmc/articles/instance/4039177/bin/nihms199097f2.gif" src-large="/pmc/articles/instance/4039177/bin/nihms199097f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4039177/" ref="ordinalpos=3038&amp;ncbi_uid=6181520&amp;link_uid=PMC4039177" image-link="/pmc/articles/PMC4039177/figure/F2/" class="imagepopup">Figure 2. Inhibition of Hh <span class="highlight" style="background-color:">signaling</span> by itraconazole is not mediated by effects on cholesterol biosynthesis.  From: Itraconazole, a commonly used anti-fungal that inhibits Hedgehog <span class="highlight" style="background-color:">pathway</span> activity and cancer growth. </a></p><div class="supp"><p class="details"><div>(A) Increasing serum concentrations attenuated the inhibitory activity of itraconazole but not KAAD-cyclopamine. (B) The attenuating effect of serum was abolished by lipid depletion (LD). (C) Shh-Light2 cells were pretreated with methyl-β-cyclodextrin (MβCD) 8 mM in DMEM for 45 minutes to remove sterols from the cell surface (Cooper et al., 2003). Lovastatin, an HMG-CoA reductase inhibitor, inhibited Hh <span class="highlight" style="background-color:">pathway</span> activity induced by ShhN. Cholesterol, in ethanol solution, reversed the <span class="highlight" style="background-color:">pathway</span> inhibition due to lovastatin (D, E) Lathosterol, desmosterol, and cholesterol failed to rescue Hh <span class="highlight" style="background-color:">pathway</span> inhibition by itraconazole under low (D) or lipid depleted (E) serum conditions. All <span class="highlight" style="background-color:">signaling</span> assays were performed with Shh-Light2 cells and data are mean of triplicates ± s.d.<br />See also Figure S2.</div></p></div><div class="aux"><div class="resc">James Kim, et al. Cancer Cell. ;17(4):388-399.</div><p class="links"><a class="dblinks" href="/pubmed/20385363" ref="ordinalpos=3038&amp;ncbi_uid=6181520&amp;link_uid=20385363">Citation</a><a class="dblinks" href="/pmc/articles/PMC4039177/" ref="ordinalpos=3038&amp;ncbi_uid=6181520&amp;link_uid=PMC4039177">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3039.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2922544/figure/fig05/" ref="ordinalpos=3039&amp;ncbi_uid=2759346&amp;link_uid=PMC2922544" href="/pmc/articles/PMC2922544/"><img src="/pmc/articles/instance/2922544/bin/pnas.1009314107fig05.gif" src-large="/pmc/articles/instance/2922544/bin/pnas.1009314107fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2922544/" ref="ordinalpos=3039&amp;ncbi_uid=2759346&amp;link_uid=PMC2922544" image-link="/pmc/articles/PMC2922544/figure/fig05/" class="imagepopup">Fig. 5. From: Receptor tyrosine kinase-like orphan receptor 2 (ROR2) and Indian hedgehog regulate digit outgrowth mediated by the phalanx-forming region. </a></p><div class="supp"><p class="details"><div>Dysregulation of IHH and canonical WNT pathways in <i>Ror2<sup>W749X/W749X</sup></i> mutants. (<i>A</i>) Whole-mount ISH showing a down-regulated distal expression of <i>Ihh</i> in <i>Ror2<sup>W749X/W749X</sup></i> mutant hand plates at E13.5 (arrows) that recovered concomitant with the differentiation of the digit tip at E14.5 (arrows). (<i>B</i>) Down-regulation of the IHH <span class="highlight" style="background-color:">pathway</span> in distal mesenchyme shown by whole-mount and section ISH for the IHH targets <i>Gli1</i>, <i>Ptc1</i>, and <i>Runx2</i>. (<i>C</i>) <i>Ror2<sup>W749X</sup></i> line crossed to the canonical WNT reporter line <i>Axin2<sup>LacZ</sup></i>, demonstrating ectopic activation of the canonical WNT <span class="highlight" style="background-color:">pathway</span> in <i>Ror2<sup>W749X/W749X</sup></i> mice. LacZ staining on cryosections shows elevated/ectopic signal in the mutant in distal mesenchyme (arrows) and also in the distal-most condensation (arrowheads). Boxed areas are shown as magnifications. (<i>D</i>) Micromass cultures of E12.5 hand plates derived from <i>Ror2<sup>W749X/W749X</sup></i>/<i>Axin2<sup>LacZ</sup></i> embryos showing increased LacZ activity in cultures derived from mutant embryos. <i>Right:</i> Histomorphometrical quantification; error bars represent SEs from three independent experiments. (<i>E</i>) Schematic representation of <span class="highlight" style="background-color:">pathway</span> network regulating digit outgrowth in WT and its perturbation in the <i>Ihh<sup>E95K/E95K</sup></i> and the <i>Ror2<sup>W749X/W749X</sup></i> mutants leading to impaired distal elongation of the digit condensations in the BDA1 and BDB1 phenotypes. AER–FGF <span class="highlight" style="background-color:">signaling</span> maintains undifferentiated mesenchymal cells (yellow). BMP/pSMAD1/5/8 <span class="highlight" style="background-color:">signaling</span> in the PFR (green) mediates distal outgrowth of the phalangeal condensation by controlling commitment of mesenchymal cells (arrow) to the condensation. IHH promotes distal outgrowth by enhancing chondrogenic cell commitment, potentially via induction of mesenchymal BMPs. Canonical WNT factors emanating from the ectoderm (red) induce β-catenin <span class="highlight" style="background-color:">signaling</span> in the mesenchyme, which inhibits BMP/pSMAD1/5/8 <span class="highlight" style="background-color:">signaling</span> in the PFR, thus limiting distal growth. The β-catenin <span class="highlight" style="background-color:">signaling</span> in the mesenchyme is negatively regulated by ROR2 (strong expression domain of ROR2 in condensing distal mesenchyme is highlighted in blue). In BDA1, the IHH<sup>E95K</sup> protein interferes with normal IHH <span class="highlight" style="background-color:">signaling</span> in distal mesenchyme, subsequently leading to a down-regulation of the BMP <span class="highlight" style="background-color:">pathway</span> in the PFR. In BDB1, the expression of a truncated ROR2 molecule leads to an up-regulation of WNT/β-catenin <span class="highlight" style="background-color:">signaling</span> in the mesenchyme concomitant with a down-regulation of <i>Ihh</i> expression and <span class="highlight" style="background-color:">pathway</span> activation, both converging on a drastic down-regulation of BMP/pSMAD1/5/8 <span class="highlight" style="background-color:">signaling</span> in the PFR. m, metacarpal; p1, condensation of phalanx 1; p2/3, unseparated primordium of phalanges 2 and 3.</div></p></div><div class="aux"><div class="resc">Florian Witte, et al. Proc Natl Acad Sci U S A. 2010 August 10;107(32):14211-14216.</div><p class="links"><a class="dblinks" href="/pubmed/20660756" ref="ordinalpos=3039&amp;ncbi_uid=2759346&amp;link_uid=20660756">Citation</a><a class="dblinks" href="/pmc/articles/PMC2922544/" ref="ordinalpos=3039&amp;ncbi_uid=2759346&amp;link_uid=PMC2922544">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3040.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3248170/figure/F1/" ref="ordinalpos=3040&amp;ncbi_uid=3763748&amp;link_uid=PMC3248170" href="/pmc/articles/PMC3248170/"><img src="/pmc/articles/instance/3248170/bin/oncotarget-02-336-g001.gif" src-large="/pmc/articles/instance/3248170/bin/oncotarget-02-336-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3248170/" ref="ordinalpos=3040&amp;ncbi_uid=3763748&amp;link_uid=PMC3248170" image-link="/pmc/articles/PMC3248170/figure/F1/" class="imagepopup">Figure 1. Mechanisms of acquired resistance to BRAF and MEK inhibitors in BRAF mutant cancers.  From: Potential Therapeutic Strategies to Overcome Acquired Resistance to BRAF or MEK Inhibitors in BRAF Mutant Cancers. </a></p><div class="supp"><p class="details"><div>A schematic of the RAF-MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown with BRAF in red. Alterations of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components leading to resistance to BRAF or MEK inhibitors are indicated by number. Resistance mechanisms classified as ERK-dependent are shown in the left panel, and mechanisms classified as ERK-independent are shown in the right panel.</div></p></div><div class="aux"><div class="resc">Ryan B. Corcoran, et al. Oncotarget. 2011 April;2(4):336-346.</div><p class="links"><a class="dblinks" href="/pubmed/21505228" ref="ordinalpos=3040&amp;ncbi_uid=3763748&amp;link_uid=21505228">Citation</a><a class="dblinks" href="/pmc/articles/PMC3248170/" ref="ordinalpos=3040&amp;ncbi_uid=3763748&amp;link_uid=PMC3248170">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3041.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3791732/figure/fig01/" ref="ordinalpos=3041&amp;ncbi_uid=5992783&amp;link_uid=PMC3791732" href="/pmc/articles/PMC3791732/"><img src="/pmc/articles/instance/3791732/bin/pnas.1305937110fig01.gif" src-large="/pmc/articles/instance/3791732/bin/pnas.1305937110fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3791732/" ref="ordinalpos=3041&amp;ncbi_uid=5992783&amp;link_uid=PMC3791732" image-link="/pmc/articles/PMC3791732/figure/fig01/" class="imagepopup">Fig. 1. From: Regulation of human hematopoietic stem cell self-renewal by the microenvironment's control of retinoic acid <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Heatmap of relative expression of RARA <span class="highlight" style="background-color:">pathway</span> genes in CD34<sup>+</sup>CD38<sup>−</sup> and CD34<sup>+</sup>CD38<sup>+</sup> cells. The normalized transcriptional level of each of the known canonical RARA target genes is represented by its row-standardized Z score according to the red-blue color scale below. Unsupervised hierarchical clustering of the samples shows that the expression patterns of RARα <span class="highlight" style="background-color:">pathway</span> genes distinguishes CD34<sup>+</sup>CD38<sup>−</sup> cells (purple sample labels) from CD34<sup>+</sup>CD38<sup>+</sup> cells (orange sample labels). Genes are ordered according to the unsupervised hierarchical clustering of their expression patterns across samples. The colored boxes above each column indicate whether that gene is canonically predicted to be activated by RARα (yellow boxes), inhibited by RARα (green boxes), or unknown (gray boxes). Arrows indicate genes known to be part of RA <span class="highlight" style="background-color:">signaling</span> machinery (ALDH1A1 and RARα), whereas arrowheads indicate classical targets of RA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (RARβ, CARBP, CD38).</div></p></div><div class="aux"><div class="resc">Gabriel Ghiaur, et al. Proc Natl Acad Sci U S A. 2013 October 1;110(40):16121-16126.</div><p class="links"><a class="dblinks" href="/pubmed/24043786" ref="ordinalpos=3041&amp;ncbi_uid=5992783&amp;link_uid=24043786">Citation</a><a class="dblinks" href="/pmc/articles/PMC3791732/" ref="ordinalpos=3041&amp;ncbi_uid=5992783&amp;link_uid=PMC3791732">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3042.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3575576/figure/F6/" ref="ordinalpos=3042&amp;ncbi_uid=4622201&amp;link_uid=PMC3575576" href="/pmc/articles/PMC3575576/"><img src="/pmc/articles/instance/3575576/bin/fphys-04-00012-g0006.gif" src-large="/pmc/articles/instance/3575576/bin/fphys-04-00012-g0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3575576/" ref="ordinalpos=3042&amp;ncbi_uid=4622201&amp;link_uid=PMC3575576" image-link="/pmc/articles/PMC3575576/figure/F6/" class="imagepopup">Figure 6.  From: Analysis of Snail1 function and regulation by Twist1 in palatal fusion. </a></p><div class="supp"><p class="details"><div><b>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Twist1 is upregulated by Tgfβ3 through <span class="highlight" style="background-color:">signaling</span> molecules including PI3-kinase and it dimerizes with E2A proteins to upregulate Snail1 to repress E-cadherin to promote the degradation of the medial edge seam (MES).</div></p></div><div class="aux"><div class="resc">Wenli Yu, et al. Front Physiol. 2013;4:12.</div><p class="links"><a class="dblinks" href="/pubmed/23424071" ref="ordinalpos=3042&amp;ncbi_uid=4622201&amp;link_uid=23424071">Citation</a><a class="dblinks" href="/pmc/articles/PMC3575576/" ref="ordinalpos=3042&amp;ncbi_uid=4622201&amp;link_uid=PMC3575576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3043.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3841209/figure/F9/" ref="ordinalpos=3043&amp;ncbi_uid=5523343&amp;link_uid=PMC3841209" href="/pmc/articles/PMC3841209/"><img src="/pmc/articles/instance/3841209/bin/cbt-14-703-g9.gif" src-large="/pmc/articles/instance/3841209/bin/cbt-14-703-g9.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3841209/" ref="ordinalpos=3043&amp;ncbi_uid=5523343&amp;link_uid=PMC3841209" image-link="/pmc/articles/PMC3841209/figure/F9/" class="imagepopup">Figure. From: Off-label use of cetuximab plus sorafenib and panitumumab plus regorafenib to personalize therapy for a patient with V600E BRAF-mutant metastatic colon cancer. </a></p><div class="supp"><p class="details"><div><b>Figure 9.</b> EGFR and BRAF/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mhd Yaser Al-Marrawi, et al. Cancer Biol Ther. 2013 August 1;14(8):703-710.</div><p class="links"><a class="dblinks" href="/pubmed/23792568" ref="ordinalpos=3043&amp;ncbi_uid=5523343&amp;link_uid=23792568">Citation</a><a class="dblinks" href="/pmc/articles/PMC3841209/" ref="ordinalpos=3043&amp;ncbi_uid=5523343&amp;link_uid=PMC3841209">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3044.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3698421/figure/Fig10/" ref="ordinalpos=3044&amp;ncbi_uid=5076547&amp;link_uid=PMC3698421" href="/pmc/articles/PMC3698421/"><img src="/pmc/articles/instance/3698421/bin/535_2012_736_Fig10_HTML.gif" src-large="/pmc/articles/instance/3698421/bin/535_2012_736_Fig10_HTML.jpg" alt="Fig. 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3698421/" ref="ordinalpos=3044&amp;ncbi_uid=5076547&amp;link_uid=PMC3698421" image-link="/pmc/articles/PMC3698421/figure/Fig10/" class="imagepopup">Fig. 10.  From: Multicolor lineage tracing methods and intestinal tumors. </a></p><div class="supp"><p class="details"><div>Schematic presentation of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Hirotsugu Yanai, et al. J Gastroenterol. 2013 April;48(4):423-433.</div><p class="links"><a class="dblinks" href="/pubmed/23307044" ref="ordinalpos=3044&amp;ncbi_uid=5076547&amp;link_uid=23307044">Citation</a><a class="dblinks" href="/pmc/articles/PMC3698421/" ref="ordinalpos=3044&amp;ncbi_uid=5076547&amp;link_uid=PMC3698421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3045.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3320915/figure/F7/" ref="ordinalpos=3045&amp;ncbi_uid=4781971&amp;link_uid=PMC3320915" href="/pmc/articles/PMC3320915/"><img src="/pmc/articles/instance/3320915/bin/zbc0161203470007.gif" src-large="/pmc/articles/instance/3320915/bin/zbc0161203470007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3320915/" ref="ordinalpos=3045&amp;ncbi_uid=4781971&amp;link_uid=PMC3320915" image-link="/pmc/articles/PMC3320915/figure/F7/" class="imagepopup">FIGURE 7. From: CCR7/CCL19 Controls Expression of EDG-1 in T Cells. </a></p><div class="supp"><p class="details"><div><b>Model of proposed CCR7/CCL19 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in T cells.</b></div></p></div><div class="aux"><div class="resc">Laura A. Shannon, et al. J Biol Chem. 2012 April 6;287(15):11656-11664.</div><p class="links"><a class="dblinks" href="/pubmed/22334704" ref="ordinalpos=3045&amp;ncbi_uid=4781971&amp;link_uid=22334704">Citation</a><a class="dblinks" href="/pmc/articles/PMC3320915/" ref="ordinalpos=3045&amp;ncbi_uid=4781971&amp;link_uid=PMC3320915">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3046.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3241611/figure/F2/" ref="ordinalpos=3046&amp;ncbi_uid=3715378&amp;link_uid=PMC3241611" href="/pmc/articles/PMC3241611/"><img src="/pmc/articles/instance/3241611/bin/nihms341856f2.gif" src-large="/pmc/articles/instance/3241611/bin/nihms341856f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3241611/" ref="ordinalpos=3046&amp;ncbi_uid=3715378&amp;link_uid=PMC3241611" image-link="/pmc/articles/PMC3241611/figure/F2/" class="imagepopup">Fig. 2.  From: Muramyl dipeptide and its derivatives: peptide adjuvant in immunological disorders and cancer therapy. </a></p><div class="supp"><p class="details"><div>A simplified schematic of the NOD2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Chikako Ogawa, et al. Curr Bioact Compd. ;7(3):180-197.</div><p class="links"><a class="dblinks" href="/pubmed/22180736" ref="ordinalpos=3046&amp;ncbi_uid=3715378&amp;link_uid=22180736">Citation</a><a class="dblinks" href="/pmc/articles/PMC3241611/" ref="ordinalpos=3046&amp;ncbi_uid=3715378&amp;link_uid=PMC3241611">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3047.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2856826/figure/F5/" ref="ordinalpos=3047&amp;ncbi_uid=2264229&amp;link_uid=PMC2856826" href="/pmc/articles/PMC2856826/"><img src="/pmc/articles/instance/2856826/bin/WJG-15-1859-g005.gif" src-large="/pmc/articles/instance/2856826/bin/WJG-15-1859-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2856826/" ref="ordinalpos=3047&amp;ncbi_uid=2264229&amp;link_uid=PMC2856826" image-link="/pmc/articles/PMC2856826/figure/F5/" class="imagepopup">Figure 5.  From: Effect of S1P5 on proliferation and migration of human esophageal cancer cells. </a></p><div class="supp"><p class="details"><div>Putative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of S1P5.</div></p></div><div class="aux"><div class="resc">Wei-Min Hu, et al. World J Gastroenterol. 2010 April 21;16(15):1859-1866.</div><p class="links"><a class="dblinks" href="/pubmed/20397263" ref="ordinalpos=3047&amp;ncbi_uid=2264229&amp;link_uid=20397263">Citation</a><a class="dblinks" href="/pmc/articles/PMC2856826/" ref="ordinalpos=3047&amp;ncbi_uid=2264229&amp;link_uid=PMC2856826">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3048.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2999296/figure/F2/" ref="ordinalpos=3048&amp;ncbi_uid=2634245&amp;link_uid=PMC2999296" href="/pmc/articles/PMC2999296/"><img src="/pmc/articles/instance/2999296/bin/WJH-2-295-g002.gif" src-large="/pmc/articles/instance/2999296/bin/WJH-2-295-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2999296/" ref="ordinalpos=3048&amp;ncbi_uid=2634245&amp;link_uid=PMC2999296" image-link="/pmc/articles/PMC2999296/figure/F2/" class="imagepopup">Figure 2.  From: Mallory-Denk body pathogenesis revisited. </a></p><div class="supp"><p class="details"><div>Diagram of the Toll-like recepter 4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modified from Wikipedia Commons.</div></p></div><div class="aux"><div class="resc">Samuel W French, et al. World J Hepatol. 2010 August 27;2(8):295-301.</div><p class="links"><a class="dblinks" href="/pubmed/21161012" ref="ordinalpos=3048&amp;ncbi_uid=2634245&amp;link_uid=21161012">Citation</a><a class="dblinks" href="/pmc/articles/PMC2999296/" ref="ordinalpos=3048&amp;ncbi_uid=2634245&amp;link_uid=PMC2999296">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3049.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2417189/figure/fig8/" ref="ordinalpos=3049&amp;ncbi_uid=1471012&amp;link_uid=PMC2417189" href="/pmc/articles/PMC2417189/"><img src="/pmc/articles/instance/2417189/bin/zbc0140830440008.gif" src-large="/pmc/articles/instance/2417189/bin/zbc0140830440008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2417189/" ref="ordinalpos=3049&amp;ncbi_uid=1471012&amp;link_uid=PMC2417189" image-link="/pmc/articles/PMC2417189/figure/fig8/" class="imagepopup">FIGURE 8. From: Makorin-2 Is a Neurogenesis Inhibitor Downstream of Phosphatidylinositol  3-Kinase/Akt (PI3K/Akt)  Signal. </a></p><div class="supp"><p class="details"><div>Inhibition of neurogenesis and axis duplication by mkrn2 via PI3K/Akt  <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Pai-Hao Yang, et al. J Biol Chem. 2008 March 28;283(13):8486-8495.</div><p class="links"><a class="dblinks" href="/pubmed/18198183" ref="ordinalpos=3049&amp;ncbi_uid=1471012&amp;link_uid=18198183">Citation</a><a class="dblinks" href="/pmc/articles/PMC2417189/" ref="ordinalpos=3049&amp;ncbi_uid=1471012&amp;link_uid=PMC2417189">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3050.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3176137/figure/fig7/" ref="ordinalpos=3050&amp;ncbi_uid=3537004&amp;link_uid=PMC3176137" href="/pmc/articles/PMC3176137/"><img src="/pmc/articles/instance/3176137/bin/137fig7.gif" src-large="/pmc/articles/instance/3176137/bin/137fig7.jpg" alt="Figure 7 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3176137/" ref="ordinalpos=3050&amp;ncbi_uid=3537004&amp;link_uid=PMC3176137" image-link="/pmc/articles/PMC3176137/figure/fig7/" class="imagepopup">Figure 7 .  From: UNC-73/Trio RhoGEF-2 Activity Modulates Caenorhabditis elegans Motility Through Changes in Neurotransmitter <span class="highlight" style="background-color:">Signaling</span> Upstream of the GSA-1/G?s <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Rho GTPase and heterotrimeric G-protein <span class="highlight" style="background-color:">pathway</span> interactions regulating neurotransmission and locomotion. G-protein pathways (blue boxes) interact with Rho GTPase pathways (red boxes) directly (solid arrows) or through indirect or unknown mechanisms (dashed arrows). Rho exists in two separate populations, only one of which interacts with DGK-1 (Hiley et al. 2006; McMullan et al. 2006). The Gα<sub>o</sub> (GOA-1) <span class="highlight" style="background-color:">pathway</span> negatively regulates Gα<sub>q</sub> (EGL-30) and synaptic vesicle <span class="highlight" style="background-color:">signaling</span>, but is omitted for clarity (Hajdu-Cronin et al. 1999; Miller et al. 1999; Nurrish et al. 1999). UNC-73 RhoGEF-2 activity (UNC-73) functions downstream of EGL-30 and likely another factor(s) to modulate locomotion through dense core vesicle (DCV) and/or synaptic vesicle (SV) <span class="highlight" style="background-color:">signaling</span>. Constitutive Gα<sub>s</sub> (GSA-1) <span class="highlight" style="background-color:">pathway</span> activation can compensate for <i>unc-73</i> RhoGEF-2 mutant locomotion defects, but the mechanism is not known. See the <i>Discussion</i> and references for details (Brundage et al. 1996; Kozasa et al. 1998; Lackner et al. 1999; Reynolds et al. 2005; Schade et al. 2005; Charlie et al. 2006; Hiley et al. 2006; McMullan et al. 2006; Williams et al. 2007; Zhou et al. 2007).</div></p></div><div class="aux"><div class="resc">Shuang Hu, et al. Genetics. 2011 September;189(1):137-151.</div><p class="links"><a class="dblinks" href="/pubmed/21750262" ref="ordinalpos=3050&amp;ncbi_uid=3537004&amp;link_uid=21750262">Citation</a><a class="dblinks" href="/pmc/articles/PMC3176137/" ref="ordinalpos=3050&amp;ncbi_uid=3537004&amp;link_uid=PMC3176137">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3051.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3959166/figure/F2/" ref="ordinalpos=3051&amp;ncbi_uid=5925338&amp;link_uid=PMC3959166" href="/pmc/articles/PMC3959166/"><img src="/pmc/articles/instance/3959166/bin/nihms559321f2.gif" src-large="/pmc/articles/instance/3959166/bin/nihms559321f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3959166/" ref="ordinalpos=3051&amp;ncbi_uid=5925338&amp;link_uid=PMC3959166" image-link="/pmc/articles/PMC3959166/figure/F2/" class="imagepopup">Figure 2.  From: MicroRNA-382 expression is elevated in the olfactory neuroepithelium of schizophrenia patients. </a></p><div class="supp"><p class="details"><div><i>FGFR1</i> and <i>SPRY4</i> are relevant putative miR-382 targets. (A) KEGG <span class="highlight" style="background-color:">pathway</span> terms enrichment among miR-382 targets. Data obtained from DAVID Bioinformatics Resources (Huang da et al., 2009). In bold: MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that was used for further research. * Number of genes that are targets of miR-382 and are also associated with the <span class="highlight" style="background-color:">pathway</span>. (B) A Venn diagram for prioritizing relevant miR-382 targets. In blue: all miR-382 putative targets according to TargetScan 6.0; In yellow: MAPK <span class="highlight" style="background-color:">signaling</span> related genes; In green: olfactory cells’ down-regulated genes with fold-change &lt;−1.2 and p-value&lt;0.03.</div></p></div><div class="aux"><div class="resc">Eyal Mor, et al. Neurobiol Dis. 2013 July;55:1-10.</div><p class="links"><a class="dblinks" href="/pubmed/23542694" ref="ordinalpos=3051&amp;ncbi_uid=5925338&amp;link_uid=23542694">Citation</a><a class="dblinks" href="/pmc/articles/PMC3959166/" ref="ordinalpos=3051&amp;ncbi_uid=5925338&amp;link_uid=PMC3959166">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3052.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3126020/figure/fig2-1758834010366430/" ref="ordinalpos=3052&amp;ncbi_uid=3108605&amp;link_uid=PMC3126020" href="/pmc/articles/PMC3126020/"><img src="/pmc/articles/instance/3126020/bin/10.1177_1758834010366430-fig2.gif" src-large="/pmc/articles/instance/3126020/bin/10.1177_1758834010366430-fig2.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3126020/figure/fig2-1758834010366430/" ref="ordinalpos=3052&amp;ncbi_uid=3108605&amp;link_uid=PMC3126020" href="/pmc/articles/PMC3126020/"><img src="/pmc/articles/instance/3126020/bin/10.1177_1758834010366430-fig2a.gif" src-large="/pmc/articles/instance/3126020/bin/10.1177_1758834010366430-fig2a.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3126020/" ref="ordinalpos=3052&amp;ncbi_uid=3108605&amp;link_uid=PMC3126020" image-link="/pmc/articles/PMC3126020/figure/fig2-1758834010366430/" class="imagepopup">Figure 2. From: Targeting the Hedgehog <span class="highlight" style="background-color:">pathway</span> in cancer. </a></p><div class="supp"><p class="details"><div>Different models of Hedgehog <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. (A) Type I ligand-independent cancers harbor inactivating mutations in <i>Patched 1</i> protein (PTCH) or activating mutations in Smoothened (SMO) leading to constitutive activation of the Hedgehog (Hh) <span class="highlight" style="background-color:">pathway</span> even in the absence of the Hh ligand. (B) Type II ligand-dependent autocrine cancers both produce and respond to the Hh ligand leading to support tumor growth and survival. (C) Type III ligand-dependent paracrine cancers secrete the Hh ligand which is received by the stromal cells leading to <span class="highlight" style="background-color:">pathway</span> activation in the stroma. The stroma in turn feeds back various signals such as IGF, Wnt, VEGF to the tumor tissue leading to its growth or survival. (D) Type IIIb reverse paracrine tumors receive Hh secreted from the stroma leading to <span class="highlight" style="background-color:">pathway</span> activation in the tumor cells and upregulation of survival signals. (E) Cancer stem cells (CSCs): Hh <span class="highlight" style="background-color:">signaling</span> occurs only in the self-renewing CSCs, from the Hh ligand produced either by the CSCs or by the stroma. CSC will give rise to more Hh <span class="highlight" style="background-color:">pathway</span> dependent CSCs or possibly may differentiate into Hh-<span class="highlight" style="background-color:">pathway</span> negative tumor cells comprising the bulk of the tumor. Reprinted from: Scales, S.J. and de Sauvage, F.J. (2009) Mechanisms of Hedgehog <span class="highlight" style="background-color:">pathway</span> activation in cancer and implications for therapy. <i>Trends Pharmacol Sci</i> 30: 303–312, with permission from Elsevier.</div></p></div><div class="aux"><div class="resc">Sachin Gupta, et al. Ther Adv Med Oncol. 2010 July;2(4):237-250.</div><p class="links"><a class="dblinks" href="/pubmed/21789137" ref="ordinalpos=3052&amp;ncbi_uid=3108605&amp;link_uid=21789137">Citation</a><a class="dblinks" href="/pmc/articles/PMC3126020/" ref="ordinalpos=3052&amp;ncbi_uid=3108605&amp;link_uid=PMC3126020">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3053.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3581421/figure/F8/" ref="ordinalpos=3053&amp;ncbi_uid=5834693&amp;link_uid=PMC3581421" href="/pmc/articles/PMC3581421/"><img src="/pmc/articles/instance/3581421/bin/zbc0131341090008.gif" src-large="/pmc/articles/instance/3581421/bin/zbc0131341090008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3581421/" ref="ordinalpos=3053&amp;ncbi_uid=5834693&amp;link_uid=PMC3581421" image-link="/pmc/articles/PMC3581421/figure/F8/" class="imagepopup">FIGURE 8. From: Sphingosine 1-Phosphate (S1P) Receptors 1 and 2 Coordinately Induce Mesenchymal Cell Migration through S1P Activation of Complementary Kinase Pathways. </a></p><div class="supp"><p class="details"><div><b>Schematic of coupling and S1P <span class="highlight" style="background-color:">signaling</span> in mesenchymal cells.</b> Osteoclast SPHK generates S1P, which activates receptors S1PR1 and S1PR2 on mesenchymal cells. S1PR1 activated the JAK/STAT <span class="highlight" style="background-color:">pathway</span>, and S1PR2 activates the FAK/PI3K/AKT <span class="highlight" style="background-color:">pathway</span> to stimulate MSC migration.</div></p></div><div class="aux"><div class="resc">Patrick Quint, et al. J Biol Chem. 2013 February 22;288(8):5398-5406.</div><p class="links"><a class="dblinks" href="/pubmed/23300082" ref="ordinalpos=3053&amp;ncbi_uid=5834693&amp;link_uid=23300082">Citation</a><a class="dblinks" href="/pmc/articles/PMC3581421/" ref="ordinalpos=3053&amp;ncbi_uid=5834693&amp;link_uid=PMC3581421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3054.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3299304/figure/fig3/" ref="ordinalpos=3054&amp;ncbi_uid=3964848&amp;link_uid=PMC3299304" href="/pmc/articles/PMC3299304/"><img src="/pmc/articles/instance/3299304/bin/GRP2012-909350.003.gif" src-large="/pmc/articles/instance/3299304/bin/GRP2012-909350.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3299304/" ref="ordinalpos=3054&amp;ncbi_uid=3964848&amp;link_uid=PMC3299304" image-link="/pmc/articles/PMC3299304/figure/fig3/" class="imagepopup">Figure 3.  From: Improvement of Sepsis by Hepatocyte Growth Factor, an Anti-Inflammatory Regulator: Emerging Insights and Therapeutic Potential. </a></p><div class="supp"><p class="details"><div>Molecular mechanisms of HGF-c-Met-mediated <span class="highlight" style="background-color:">signaling</span> for inhibiting TLR4-mediated NF-<i>κ</i>B activation in macrophages. (a) HO-1-dependent <span class="highlight" style="background-color:">pathway</span>. HO-1 is upregulated at a downstream of TLR4 through the activation of transcriptional factors, such as Nrf2 or HIF-1. During heme metabolism by HO-1, carbon monoxide (CO), free iron and biliverdin are generated, and these products attenuate NF-<i>κ</i>B activation (i.e., negative feedback system). In this process, c-Met signal enhances the HO-1 transcriptional <span class="highlight" style="background-color:">pathway</span> (1a) or stabilizes HO-1 from degradation (1b). Overall, HGF-mediated increases in HO-1 partially contribute to NF-<i>κ</i>B inactivation (1c). (b) GSK3<i>β</i>-inactivated <span class="highlight" style="background-color:">pathway</span>. LPS-TLR4-mediated <span class="highlight" style="background-color:">signaling</span> leads to nuclear localization and activation of NF-<i>κ</i>B via GSK3<i>β</i> activation <span class="highlight" style="background-color:">pathway</span>. In contrast, HGF-c-Met <span class="highlight" style="background-color:">signaling</span> elicits PI3K and AKT phosphorylation, and then GSK3<i>β</i> is inactivated. As a result, CBP, a coactivator of NF-<i>κ</i>B, is sequestered from the p65 subunit of NF-<i>κ</i>B, and transcription of inflammatory molecules, such as TNF-<i>α</i>, IL-6 and ICAM-1, is arrested.</div></p></div><div class="aux"><div class="resc">Shinya Mizuno, et al. Gastroenterol Res Pract. 2012;2012:909350.</div><p class="links"><a class="dblinks" href="/pubmed/22536224" ref="ordinalpos=3054&amp;ncbi_uid=3964848&amp;link_uid=22536224">Citation</a><a class="dblinks" href="/pmc/articles/PMC3299304/" ref="ordinalpos=3054&amp;ncbi_uid=3964848&amp;link_uid=PMC3299304">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3055.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3735423/figure/F9/" ref="ordinalpos=3055&amp;ncbi_uid=5175123&amp;link_uid=PMC3735423" href="/pmc/articles/PMC3735423/"><img src="/pmc/articles/instance/3735423/bin/1478-811X-11-51-9.gif" src-large="/pmc/articles/instance/3735423/bin/1478-811X-11-51-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3735423/" ref="ordinalpos=3055&amp;ncbi_uid=5175123&amp;link_uid=PMC3735423" image-link="/pmc/articles/PMC3735423/figure/F9/" class="imagepopup">Figure 9.  From: Metastasis of aggressive amoeboid sarcoma cells is dependent on Rho/ROCK/MLC <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Rho <span class="highlight" style="background-color:">signaling</span> in the maintenance of amoeboid morphology and invasion/metastasis of sarcoma cells.</b> The activation of Rho, ROCK and the phosphorylation of MLC are all crucial for the effective invasiveness/metastasis of amoeboid sarcoma cells. In contrast, the rounded morphology of amoeboid sarcoma cells is not dependent on MLC phosphorylation or non-muscle myosin II ATPase activity, suggesting another downstream effectors of Rho/ROCK <span class="highlight" style="background-color:">pathway</span> are involved in the maintenance of rounded morphology of cancer cells.</div></p></div><div class="aux"><div class="resc">Jan Kosla, et al. Cell Commun Signal. 2013;11:51-51.</div><p class="links"><a class="dblinks" href="/pubmed/23899007" ref="ordinalpos=3055&amp;ncbi_uid=5175123&amp;link_uid=23899007">Citation</a><a class="dblinks" href="/pmc/articles/PMC3735423/" ref="ordinalpos=3055&amp;ncbi_uid=5175123&amp;link_uid=PMC3735423">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3056.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3613369/figure/pone-0060299-g003/" ref="ordinalpos=3056&amp;ncbi_uid=4778700&amp;link_uid=PMC3613369" href="/pmc/articles/PMC3613369/"><img src="/pmc/articles/instance/3613369/bin/pone.0060299.g003.gif" src-large="/pmc/articles/instance/3613369/bin/pone.0060299.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3613369/" ref="ordinalpos=3056&amp;ncbi_uid=4778700&amp;link_uid=PMC3613369" image-link="/pmc/articles/PMC3613369/figure/pone-0060299-g003/" class="imagepopup">Figure 3. Comparison of genes signatures associated with metastasis <span class="highlight" style="background-color:">signaling</span> between HCT116 and HCT116b microarray datasets by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) tool..  From: Intra-Tumoral Heterogeneity in Metastatic Potential and Survival <span class="highlight" style="background-color:">Signaling</span> between Iso-Clonal HCT116 and HCT116b Human Colon Carcinoma Cell Lines. </a></p><div class="supp"><p class="details"><div>The molecules highlighted in green are upregulated in the highly metastatic HCT116 cells and downregulated in poorly metastatic iso-clonal HCT116b cells. The molecules marked in red are upregulated in HCT116b cells compared to HCT116 cells.</div></p></div><div class="aux"><div class="resc">Sanjib Chowdhury, et al. PLoS One. 2013;8(4):e60299.</div><p class="links"><a class="dblinks" href="/pubmed/23560089" ref="ordinalpos=3056&amp;ncbi_uid=4778700&amp;link_uid=23560089">Citation</a><a class="dblinks" href="/pmc/articles/PMC3613369/" ref="ordinalpos=3056&amp;ncbi_uid=4778700&amp;link_uid=PMC3613369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3057.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3343662/figure/fig1/" ref="ordinalpos=3057&amp;ncbi_uid=4848746&amp;link_uid=PMC3343662" href="/pmc/articles/PMC3343662/"><img src="/pmc/articles/instance/3343662/bin/cr201253f1.gif" src-large="/pmc/articles/instance/3343662/bin/cr201253f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3343662/" ref="ordinalpos=3057&amp;ncbi_uid=4848746&amp;link_uid=PMC3343662" image-link="/pmc/articles/PMC3343662/figure/fig1/" class="imagepopup">Figure 1.  From: Enhanced TLR-induced NF-?B <span class="highlight" style="background-color:">signaling</span> and type I interferon responses in NLRC5 deficient mice. </a></p><div class="supp"><p class="details"><div>Stat1-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is critical for induction of <i>NLRC5</i> expression. <b>(A, B)</b> Induction of <i>NLRC5</i> expression by LPS (100 ng/ml), IFN-β (1 000 U/ml) or IFN-γ (10 ng/ml) treatment in peritoneal macrophages from WT, <i>MyD88</i><sup>−/−</sup>, <i>IRAK4</i><sup>−/−</sup> <b>(A)</b> or <i>Stat1</i><sup>−/−</sup> mice <b>(B)</b>. Results are representative of three independent experiments.</div></p></div><div class="aux"><div class="resc">Yanzheng Tong, et al. Cell Res. 2012 May;22(5):822-835.</div><p class="links"><a class="dblinks" href="/pubmed/22473004" ref="ordinalpos=3057&amp;ncbi_uid=4848746&amp;link_uid=22473004">Citation</a><a class="dblinks" href="/pmc/articles/PMC3343662/" ref="ordinalpos=3057&amp;ncbi_uid=4848746&amp;link_uid=PMC3343662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3058.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267734/figure/pone-0030601-g008/" ref="ordinalpos=3058&amp;ncbi_uid=3786757&amp;link_uid=PMC3267734" href="/pmc/articles/PMC3267734/"><img src="/pmc/articles/instance/3267734/bin/pone.0030601.g008.gif" src-large="/pmc/articles/instance/3267734/bin/pone.0030601.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267734/" ref="ordinalpos=3058&amp;ncbi_uid=3786757&amp;link_uid=PMC3267734" image-link="/pmc/articles/PMC3267734/figure/pone-0030601-g008/" class="imagepopup">Figure 8. Hypothetical model for the mode of action of FZC18..  From: Inhibition of Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> by a Soluble Collagen-Derived Frizzled Domain Interacting with Wnt3a and the Receptors Frizzled 1 and 8. </a></p><div class="supp"><p class="details"><div><i>High:</i> in the absence of FZC18, Wnt3a increases β-catenin <span class="highlight" style="background-color:">signaling</span>. <i>Low:</i> FZC18 binds Wnt3a and FZD receptors, blocking Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> activation in an SFRP-like mode of action. <i>Rescued:</i> FZC18 inhibitory effects can be partially rescued by increasing the number of FZD receptors at the cell surface.</div></p></div><div class="aux"><div class="resc">Ismaïl Hendaoui, et al. PLoS One. 2012;7(1):e30601.</div><p class="links"><a class="dblinks" href="/pubmed/22303445" ref="ordinalpos=3058&amp;ncbi_uid=3786757&amp;link_uid=22303445">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267734/" ref="ordinalpos=3058&amp;ncbi_uid=3786757&amp;link_uid=PMC3267734">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3059.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3478617/figure/AS-fig01/" ref="ordinalpos=3059&amp;ncbi_uid=4789655&amp;link_uid=PMC3478617" href="/pmc/articles/PMC3478617/"><img src="/pmc/articles/instance/3478617/bin/pnas.1204311S109fig01.gif" src-large="/pmc/articles/instance/3478617/bin/pnas.1204311S109fig01.jpg" alt="Fig. P1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3478617/" ref="ordinalpos=3059&amp;ncbi_uid=4789655&amp;link_uid=PMC3478617" image-link="/pmc/articles/PMC3478617/figure/AS-fig01/" class="imagepopup">Fig. P1. From: Selective requirement for Mediator MED23 in Ras-active lung cancer. </a></p><div class="supp"><p class="details"><div>Schematic model illustrating the oncogenic <span class="highlight" style="background-color:">pathway</span> addiction. In normal cells, multiple <span class="highlight" style="background-color:">signaling</span> pathways may cooperate to control the cell growth. After the progressive accumulation of mutations, such as oncogenic <i>Ras</i> mutations, the entire Ras/MAPK <span class="highlight" style="background-color:">pathway</span> could become hyperactive, resulting in activation of ELK1 and up-regulated MED23. Therefore, cancer cells with <i>Ras</i> mutations become heavily dependent on the Ras/MAPK/ELK1/MED23 <span class="highlight" style="background-color:">pathway</span> for the maintenance of the malignant phenotype. Targeting MED23, ELK1, or their interaction may counteract the oncogenic <span class="highlight" style="background-color:">signaling</span> transduction in the Ras-driven cancer cells.</div></p></div><div class="aux"><div class="resc">Xu Yang, et al. Proc Natl Acad Sci U S A. 2012 October 9;109(41):E2813-E2822.</div><p class="links"><a class="dblinks" href="/pubmed/22988093" ref="ordinalpos=3059&amp;ncbi_uid=4789655&amp;link_uid=22988093">Citation</a><a class="dblinks" href="/pmc/articles/PMC3478617/" ref="ordinalpos=3059&amp;ncbi_uid=4789655&amp;link_uid=PMC3478617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3060.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399452/figure/fig1/" ref="ordinalpos=3060&amp;ncbi_uid=4136163&amp;link_uid=PMC3399452" href="/pmc/articles/PMC3399452/"><img src="/pmc/articles/instance/3399452/bin/IJCB2012-595976.001.gif" src-large="/pmc/articles/instance/3399452/bin/IJCB2012-595976.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399452/" ref="ordinalpos=3060&amp;ncbi_uid=4136163&amp;link_uid=PMC3399452" image-link="/pmc/articles/PMC3399452/figure/fig1/" class="imagepopup">Figure 1.  From: Targeting Metabolism and Autophagy in the Context of Haematologic Malignancies. </a></p><div class="supp"><p class="details"><div>Metabolic <span class="highlight" style="background-color:">signaling</span> regulation. Glucose uptake in cells is regulated by Glut1 transporters that increase glycolysis and oxidative phosphorylation. This is enhanced by cMyc regulation of glutamine uptake in cells. Growth factors influence metabolism through activation of the PI3K/AKT/mTOR signalign <span class="highlight" style="background-color:">pathway</span> contribute to increased glycolysis and oxidative phosphorylation. This is inhibited through PTEN. Under glucose limiting conditions, AMPK is activated inhibiting the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In addition, p53 activaiton inhibits glycolysis, and the mTOR <span class="highlight" style="background-color:">pathway</span> but increases oxidative phosphorylation. ROS increases through inefficient oxidative phosphorylation at the mitochondria.</div></p></div><div class="aux"><div class="resc">Versha Banerji, et al. Int J Cell Biol. 2012;2012:595976.</div><p class="links"><a class="dblinks" href="/pubmed/22829831" ref="ordinalpos=3060&amp;ncbi_uid=4136163&amp;link_uid=22829831">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399452/" ref="ordinalpos=3060&amp;ncbi_uid=4136163&amp;link_uid=PMC3399452">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3061.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3469587/figure/F7/" ref="ordinalpos=3061&amp;ncbi_uid=5348711&amp;link_uid=PMC3469587" href="/pmc/articles/PMC3469587/"><img src="/pmc/articles/instance/3469587/bin/zh50201262040007.gif" src-large="/pmc/articles/instance/3469587/bin/zh50201262040007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3469587/" ref="ordinalpos=3061&amp;ncbi_uid=5348711&amp;link_uid=PMC3469587" image-link="/pmc/articles/PMC3469587/figure/F7/" class="imagepopup">Fig. 7. From: Pulmonary surfactant protein A and surfactant lipids upregulate IRAK-M, a negative regulator of TLR-mediated inflammation in human macrophages. </a></p><div class="supp"><p class="details"><div>Model of SP-A-regulated <span class="highlight" style="background-color:">signaling</span> pathways in human macrophages. Schematic diagram is based on our previous report (26) and current results of how SP-A regulates Toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> through multiple mechanisms: early, by dephosphorylating key biochemical <span class="highlight" style="background-color:">signaling</span> molecules such as Akt and MAPKs and, later, by upregulating negative regulators (i.e., IRAK-M) of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TRAF6, TNF receptor-associated factor 6.</div></p></div><div class="aux"><div class="resc">Huy A. Nguyen, et al. Am J Physiol Lung Cell Mol Physiol. 2012 October 1;303(7):L608-L616.</div><p class="links"><a class="dblinks" href="/pubmed/22886503" ref="ordinalpos=3061&amp;ncbi_uid=5348711&amp;link_uid=22886503">Citation</a><a class="dblinks" href="/pmc/articles/PMC3469587/" ref="ordinalpos=3061&amp;ncbi_uid=5348711&amp;link_uid=PMC3469587">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3062.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4252143/figure/pone-0114981-g003/" ref="ordinalpos=3062&amp;ncbi_uid=6937289&amp;link_uid=PMC4252143" href="/pmc/articles/PMC4252143/"><img src="/pmc/articles/instance/4252143/bin/pone.0114981.g003.gif" src-large="/pmc/articles/instance/4252143/bin/pone.0114981.g003.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4252143/" ref="ordinalpos=3062&amp;ncbi_uid=6937289&amp;link_uid=PMC4252143" image-link="/pmc/articles/PMC4252143/figure/pone-0114981-g003/" class="imagepopup">Figure 5. Dose-dependent estrogen inhibits osteo-adipogenic transdifferentiation of MC3T3-E1 cells likely via modulating canonical Wnt singling <span class="highlight" style="background-color:">pathway</span>..  From: Correction: Dose-Dependent Effect of Estrogen Suppresses the Osteo-Adipogenic Transdifferentiation of Osteoblasts via Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>After 14 days’ osteogenesis, two groups of MC3T3-E1 cells were divided. The first group was cultured in adipogenic cocktail medium accompanied with 10−7 M of 17beta-estradiol for 14 days and 50 ng/ml DKK-1 was added to block canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> simultaneously. The second group was cultured in adipogenic cocktail medium accompanied with 10−7 M of 17beta-estradiol and ICI together for 14 days and 50 ng/ml Wnt-3a was added to activate canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> simultaneously. O14+A14:14 days’ adipogenesis after 14 days’ osteogenesis; O14+A14+E2:14 days’ adipogenesis accompanied with 10−7 M of 17beta-estradiol after 14 days’ osteogenesis; O14+A14+E2+ICI: 14 days’ adipogenesis accompanied with 10−7 M of 17beta-estradiol and ICI after 14 days’ osteogenesis. O14+A14+E2+ICI+Wnt-3a: 14 days’ adipogenesis accompanied with 10−7 M of 17beta-estradiol, ICI and 50 ng/ml Wnt-3a after 14 days’ osteogenesis; O14+A14+E2+DKK-1:14 days adipogenesis accompanied with 10−7 M 17beta-estradiol and 50 ng/ml DKK1 after 14 days’ osteogenesis. A: ALP staining of osteo-adipogenic transdifferentiation of MC3T3-E1 cells; B: Alizarin red staining of osteo-adipogenic transdifferentiation of MC3T3-E1 cells; C: Oil red staining of osteo-adipogenic transdifferentiation of MC3T3-E1 cells; D: ALP activity of osteo-adipogenic transdifferentiation of MC3T3-E1 cells. The control value for ALP activity was 0.687±0.014 unit/mg protein; E: The mineralization of osteo-adipogenic transdifferentiation of MC3T3-E1 cells. The control value for mineralization was 0.795±0.020 OD; F: Quantification of lipid; G: Effect of Wnts and 17beta-estradiol on the osteogenic mRNA expression; H: Effect of Wnts and 17beta-estradiol on the adipogenic mRNA expression; I: Effect of Wnts and 17beta-estradiol on the canonical Wnt <span class="highlight" style="background-color:">signaling</span> mRNA expression. Expression of each target gene was calculated as a relative expression to beta-actin and represented as normalized fold expression. Data are represented as mean±SD of 3 independent experiments. *P&lt;0.05 and **P&lt;0.01.</div></p></div><div class="aux"><div class="resc">. PLoS One. 2014;9(12):e114981.</div><p class="links"><a class="dblinks" href="/pubmed/25462163" ref="ordinalpos=3062&amp;ncbi_uid=6937289&amp;link_uid=25462163">Citation</a><a class="dblinks" href="/pmc/articles/PMC4252143/" ref="ordinalpos=3062&amp;ncbi_uid=6937289&amp;link_uid=PMC4252143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3063.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4150760/figure/F3/" ref="ordinalpos=3063&amp;ncbi_uid=6895173&amp;link_uid=PMC4150760" href="/pmc/articles/PMC4150760/"><img src="/pmc/articles/instance/4150760/bin/gku577fig3.gif" src-large="/pmc/articles/instance/4150760/bin/gku577fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4150760/" ref="ordinalpos=3063&amp;ncbi_uid=6895173&amp;link_uid=PMC4150760" image-link="/pmc/articles/PMC4150760/figure/F3/" class="imagepopup">Figure 3. From: SpliceNet: recovering splicing isoform-specific differential gene networks from RNA-Seq data of normal and diseased samples. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) Inferred non-small cell lung cancer <span class="highlight" style="background-color:">pathway</span> using the SpliceNet and RNASeqNet. (<b>b</b>) Re-inferred ERBB2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, but with a reduced sample size.</div></p></div><div class="aux"><div class="resc">Hari Krishna Yalamanchili, et al. Nucleic Acids Res. 2014 September 2;42(15):e121-e121.</div><p class="links"><a class="dblinks" href="/pubmed/25034693" ref="ordinalpos=3063&amp;ncbi_uid=6895173&amp;link_uid=25034693">Citation</a><a class="dblinks" href="/pmc/articles/PMC4150760/" ref="ordinalpos=3063&amp;ncbi_uid=6895173&amp;link_uid=PMC4150760">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3064.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4222074/figure/F4/" ref="ordinalpos=3064&amp;ncbi_uid=6812127&amp;link_uid=PMC4222074" href="/pmc/articles/PMC4222074/"><img src="/pmc/articles/instance/4222074/bin/1755-8794-6-53-4.gif" src-large="/pmc/articles/instance/4222074/bin/1755-8794-6-53-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4222074/" ref="ordinalpos=3064&amp;ncbi_uid=6812127&amp;link_uid=PMC4222074" image-link="/pmc/articles/PMC4222074/figure/F4/" class="imagepopup">Figure 4.  From: Integrated molecular portrait of non-small cell lung cancers. </a></p><div class="supp"><p class="details"><div><b>Network links between 23 genes in <span class="highlight" style="background-color:">pathway</span> of mismatch repair and three driver genes.</b> Network links between 23 genes in <span class="highlight" style="background-color:">pathway</span> of mismatch repair and driver genes MRPS22, NDRG1, RNF7. Links shown include physical interactions, metabolic and <span class="highlight" style="background-color:">signaling</span> links from the functional coupling network (http://FunCoup.sbc.su.se).</div></p></div><div class="aux"><div class="resc">Vladimir Lazar, et al. BMC Med Genomics. 2013;6:53-53.</div><p class="links"><a class="dblinks" href="/pubmed/24299561" ref="ordinalpos=3064&amp;ncbi_uid=6812127&amp;link_uid=24299561">Citation</a><a class="dblinks" href="/pmc/articles/PMC4222074/" ref="ordinalpos=3064&amp;ncbi_uid=6812127&amp;link_uid=PMC4222074">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3065.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508721/figure/F4/" ref="ordinalpos=3065&amp;ncbi_uid=5449483&amp;link_uid=PMC3508721" href="/pmc/articles/PMC3508721/"><img src="/pmc/articles/instance/3508721/bin/nihms415567f4.gif" src-large="/pmc/articles/instance/3508721/bin/nihms415567f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508721/" ref="ordinalpos=3065&amp;ncbi_uid=5449483&amp;link_uid=PMC3508721" image-link="/pmc/articles/PMC3508721/figure/F4/" class="imagepopup">Figure 4.  From: Proteomic and Bioinformatic Profile of Primary Human Oral Epithelial Cells. </a></p><div class="supp"><p class="details"><div>Canonical <span class="highlight" style="background-color:">pathway</span> analysis: The number of proteins associated with canonical <span class="highlight" style="background-color:">signaling</span> pathways as defined by Ingenuity <span class="highlight" style="background-color:">Pathway</span> analysis is shown. Seventeen pathways with ≥ 25 NHOEC identified proteins are shown.</div></p></div><div class="aux"><div class="resc">Santosh K. Ghosh, et al. J Proteome Res. ;11(11):5492-5502.</div><p class="links"><a class="dblinks" href="/pubmed/23035736" ref="ordinalpos=3065&amp;ncbi_uid=5449483&amp;link_uid=23035736">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508721/" ref="ordinalpos=3065&amp;ncbi_uid=5449483&amp;link_uid=PMC3508721">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3066.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4310092/figure/f7/" ref="ordinalpos=3066&amp;ncbi_uid=7161195&amp;link_uid=PMC4310092" href="/pmc/articles/PMC4310092/"><img src="/pmc/articles/instance/4310092/bin/srep08087-f7.gif" src-large="/pmc/articles/instance/4310092/bin/srep08087-f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4310092/" ref="ordinalpos=3066&amp;ncbi_uid=7161195&amp;link_uid=PMC4310092" image-link="/pmc/articles/PMC4310092/figure/f7/" class="imagepopup">Figure 7. The effects of miR-148b on the downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of WNT1..  From: MiR-148b suppresses cell proliferation and invasion in hepatocellular carcinoma by targeting WNT1/β-catenin <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>HepG2 cells were transfected with miR-148b or miR-NC for 48 h, the expression levels of β-catenin, C-myc and E-cadherin proteins in HepG2 cells were detected by western blotting. The results were standardized against the levels of GAPDH and repeated three times, *P &lt; 0.05, **P &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Jun-gang Zhang, et al. Sci Rep. 2015;5:8087.</div><p class="links"><a class="dblinks" href="/pubmed/25627001" ref="ordinalpos=3066&amp;ncbi_uid=7161195&amp;link_uid=25627001">Citation</a><a class="dblinks" href="/pmc/articles/PMC4310092/" ref="ordinalpos=3066&amp;ncbi_uid=7161195&amp;link_uid=PMC4310092">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3067.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3236571/figure/fig03/" ref="ordinalpos=3067&amp;ncbi_uid=3731668&amp;link_uid=PMC3236571" href="/pmc/articles/PMC3236571/"><img src="/pmc/articles/instance/3236571/bin/ajcr0002-0045-f3.gif" src-large="/pmc/articles/instance/3236571/bin/ajcr0002-0045-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3236571/" ref="ordinalpos=3067&amp;ncbi_uid=3731668&amp;link_uid=PMC3236571" image-link="/pmc/articles/PMC3236571/figure/fig03/" class="imagepopup">Figure 3.  From: TRAIL receptor <span class="highlight" style="background-color:">signaling</span> and therapeutic option in bone tumors: the trap of the bone microenvironment. </a></p><div class="supp"><p class="details"><div>TRAIL-induced apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via the death receptors DR4 and DR5. The incoming trimeric ligand TRAIL recruits three receptors into a complex. The induced juxtaposition of the intracellular domains triggers recruitment of the intracellular <span class="highlight" style="background-color:">signaling</span> components forming the Death-Inducing <span class="highlight" style="background-color:">Signaling</span> Complex (DISC) and leading to the caspase cascade and cell death. DD: Death Domain; DED: Death Effector Domains; FADD: Fas-Associated protein with Death Domain; c-FLIP: cellular FLICE-inhibitory protein; BAK: B cell-CELL/Lymphoma 2 (Bcl-2) homologous Antagonist Killer; BAX: Bcl-2-Associated X; Bid: BH3-interacting domain death agonist.</div></p></div><div class="aux"><div class="resc">Gaëlle Picarda, et al. Am J Cancer Res. 2012;2(1):45-64.</div><p class="links"><a class="dblinks" href="/pubmed/22206045" ref="ordinalpos=3067&amp;ncbi_uid=3731668&amp;link_uid=22206045">Citation</a><a class="dblinks" href="/pmc/articles/PMC3236571/" ref="ordinalpos=3067&amp;ncbi_uid=3731668&amp;link_uid=PMC3236571">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3068.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4381582/figure/F6/" ref="ordinalpos=3068&amp;ncbi_uid=7400237&amp;link_uid=PMC4381582" href="/pmc/articles/PMC4381582/"><img src="/pmc/articles/instance/4381582/bin/oncotarget-06-116-g006.gif" src-large="/pmc/articles/instance/4381582/bin/oncotarget-06-116-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4381582/" ref="ordinalpos=3068&amp;ncbi_uid=7400237&amp;link_uid=PMC4381582" image-link="/pmc/articles/PMC4381582/figure/F6/" class="imagepopup">Figure 6. PCTK1 inhibition alters Akt/mTOR <span class="highlight" style="background-color:">pathway</span> in c-Myc hallmarked MB cell lines.  From: RNA interference screening identifies a novel role for PCTK1/CDK16 in medulloblastoma with c-Myc amplification. </a></p><div class="supp"><p class="details"><div>A. Analysis of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulators protein level upon treatment with PCTK1 inhibitor in DAOY V11 and M2.1 and B. in D458 cells C. PCTK1 inhibition induces cell death in dose dependent manner as shown by trypan blue staining and D. sub-G1 cell accumulation.</div></p></div><div class="aux"><div class="resc">Paulina Ćwiek, et al. Oncotarget. 2015 January;6(1):116-129.</div><p class="links"><a class="dblinks" href="/pubmed/25402633" ref="ordinalpos=3068&amp;ncbi_uid=7400237&amp;link_uid=25402633">Citation</a><a class="dblinks" href="/pmc/articles/PMC4381582/" ref="ordinalpos=3068&amp;ncbi_uid=7400237&amp;link_uid=PMC4381582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3069.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2802896/figure/F9/" ref="ordinalpos=3069&amp;ncbi_uid=2678439&amp;link_uid=PMC2802896" href="/pmc/articles/PMC2802896/"><img src="/pmc/articles/instance/2802896/bin/zmg0011045730009.gif" src-large="/pmc/articles/instance/2802896/bin/zmg0011045730009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2802896/" ref="ordinalpos=3069&amp;ncbi_uid=2678439&amp;link_uid=PMC2802896" image-link="/pmc/articles/PMC2802896/figure/F9/" class="imagepopup">Figure 9.  From: Agonist-Biased <span class="highlight" style="background-color:">Signaling</span> at the sst2A Receptor: The Multi-Somatostatin Analogs KE108 and SOM230 Activate and Antagonize Distinct <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> by the sst2A receptor upon activation by different somatostatin analogs. <i>Top</i>, In HEK293 cells, SS-14 and octreotide (SS/Oct) inhibit adenylyl cyclase (AC) and stimulate cytosolic calcium via Gi/Go, whereas they increase ERK phosphorylation by two distinct mechanisms: a Gi/Go-mediated <span class="highlight" style="background-color:">pathway</span>, which is inhibited by pertussis toxin, as well as a pertussis toxin-insensitive process, shown by X. Although SOM230 and KE108 (Som/KE) behave as full agonists to inhibit adenylyl cyclase, they antagonize the pertussis toxin-sensitive process that leads to an increase in intracellular calcium. Moreover, because the effect of SOM230 and KE108 to stimulate ERK phosphorylation is completely blocked by pertussis toxin, this action occurs solely via Gi/Go, and these analogs antagonize SS-14 stimulation of ERK phosphorylation by this pertussis toxin-insensitive <span class="highlight" style="background-color:">pathway</span>. <i>Bottom</i>, In AR42J cells, SS-14 and octreotide (SS/Oct) inhibit adenylyl cyclase (AC) via Gi/Go but act by the pertussis toxin-insensitive G protein G<sub>14</sub> to stimulate ERK phosphorylation and increase cytosolic calcium. SOM230 and KE108 (Som/KE) are only able to activate Gi/Go <span class="highlight" style="background-color:">signaling</span> and act as antagonists for sst2A receptor <span class="highlight" style="background-color:">signaling</span> via G<sub>14</sub>.</div></p></div><div class="aux"><div class="resc">Renzo Cescato, et al. Mol Endocrinol. 2010 January;24(1):240-249.</div><p class="links"><a class="dblinks" href="/pubmed/19910453" ref="ordinalpos=3069&amp;ncbi_uid=2678439&amp;link_uid=19910453">Citation</a><a class="dblinks" href="/pmc/articles/PMC2802896/" ref="ordinalpos=3069&amp;ncbi_uid=2678439&amp;link_uid=PMC2802896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3070.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3552917/figure/F2/" ref="ordinalpos=3070&amp;ncbi_uid=5538700&amp;link_uid=PMC3552917" href="/pmc/articles/PMC3552917/"><img src="/pmc/articles/instance/3552917/bin/cc-11-4344-g2.gif" src-large="/pmc/articles/instance/3552917/bin/cc-11-4344-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3552917/" ref="ordinalpos=3070&amp;ncbi_uid=5538700&amp;link_uid=PMC3552917" image-link="/pmc/articles/PMC3552917/figure/F2/" class="imagepopup">Figure. From: Developmental pathways in colon cancer. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Summary of Notch, WNT HH and BMP <span class="highlight" style="background-color:">signaling</span> pathways. Illustrated are the major components of each of the Notch, WNT, HH and BMP pathways, starting with ligands expressed by the <span class="highlight" style="background-color:">signaling</span> cell, receptors expressed on the receiving cell and cytoplasmic <span class="highlight" style="background-color:">signaling</span> intermediates and transcriptional effectors. In this illustration, signal transduction for each <span class="highlight" style="background-color:">pathway</span> travels from the top of the figure to the bottom.</div></p></div><div class="aux"><div class="resc">Fred E. Bertrand, et al. Cell Cycle. 2012 December 1;11(23):4344-4351.</div><p class="links"><a class="dblinks" href="/pubmed/23032367" ref="ordinalpos=3070&amp;ncbi_uid=5538700&amp;link_uid=23032367">Citation</a><a class="dblinks" href="/pmc/articles/PMC3552917/" ref="ordinalpos=3070&amp;ncbi_uid=5538700&amp;link_uid=PMC3552917">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3071.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3522553/figure/F5/" ref="ordinalpos=3071&amp;ncbi_uid=4462242&amp;link_uid=PMC3522553" href="/pmc/articles/PMC3522553/"><img src="/pmc/articles/instance/3522553/bin/1471-2105-13-186-5.gif" src-large="/pmc/articles/instance/3522553/bin/1471-2105-13-186-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3522553/" ref="ordinalpos=3071&amp;ncbi_uid=4462242&amp;link_uid=PMC3522553" image-link="/pmc/articles/PMC3522553/figure/F5/" class="imagepopup">Figure 5.  From: Integrated QSAR study for inhibitors of hedgehog signal <span class="highlight" style="background-color:">pathway</span> against multiple cell lines:a collaborative filtering method. </a></p><div class="supp"><p class="details"><div><b>Comparison of CMF based QSAR modeling with ridge regression and SVR on the inhibitors of Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.</b> The compounds are represented with Drug-like index and General Descriptor respectively. The whole data was partitioned with <i>Diverse subset</i> method. Cell line No. 1–4 corresponding to BxPC-3, NCI-H446, SW1990 and NCI-H157 respectively.</div></p></div><div class="aux"><div class="resc">Jun Gao, et al. BMC Bioinformatics. 2012;13:186-186.</div><p class="links"><a class="dblinks" href="/pubmed/22849868" ref="ordinalpos=3071&amp;ncbi_uid=4462242&amp;link_uid=22849868">Citation</a><a class="dblinks" href="/pmc/articles/PMC3522553/" ref="ordinalpos=3071&amp;ncbi_uid=4462242&amp;link_uid=PMC3522553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3072.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC316922/figure/F7/" ref="ordinalpos=3072&amp;ncbi_uid=408415&amp;link_uid=PMC316922" href="/pmc/articles/PMC316922/"><img src="/pmc/articles/instance/316922/bin/gad.3f7.gif" src-large="/pmc/articles/instance/316922/bin/gad.3f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC316922/" ref="ordinalpos=3072&amp;ncbi_uid=408415&amp;link_uid=PMC316922" image-link="/pmc/articles/PMC316922/figure/F7/" class="imagepopup">Figure 7.  From: The FAR1 locus encodes a novel nuclear protein specific to phytochrome A <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Models for the possible action of FAR1. (<i>A</i>) Genetic models of the <span class="highlight" style="background-color:">pathway</span> of phytochrome signal transduction. In the multiple-<span class="highlight" style="background-color:">pathway</span> model (<i>i</i>) specific and shared <span class="highlight" style="background-color:">signaling</span> components can transduce signals independently of one another. In the alternative single-<span class="highlight" style="background-color:">pathway</span> model shown (<i>ii</i>), loci specific to a single phytochrome-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (e.g., FAR1) are shown upstream of those components shared by multiple pathways (e.g., PIF3). (<i>B</i>) Speculative molecular modes of action for FAR1. Although we speculate in <i>A</i> that FAR1 is upstream of PIF3 on the basis of genetic data, its biochemical action may lie before that of PIF3 (<i>i</i>) or after it (<i>ii</i>). Either could produce a specific effect on phyA <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Matthew Hudson, et al. Genes Dev. 1999 August 1;13(15):2017-2027.</div><p class="links"><a class="dblinks" href="/pubmed/10444599" ref="ordinalpos=3072&amp;ncbi_uid=408415&amp;link_uid=10444599">Citation</a><a class="dblinks" href="/pmc/articles/PMC316922/" ref="ordinalpos=3072&amp;ncbi_uid=408415&amp;link_uid=PMC316922">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3073.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4275273/figure/pone-0115985-g005/" ref="ordinalpos=3073&amp;ncbi_uid=7007974&amp;link_uid=PMC4275273" href="/pmc/articles/PMC4275273/"><img src="/pmc/articles/instance/4275273/bin/pone.0115985.g005.gif" src-large="/pmc/articles/instance/4275273/bin/pone.0115985.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4275273/" ref="ordinalpos=3073&amp;ncbi_uid=7007974&amp;link_uid=PMC4275273" image-link="/pmc/articles/PMC4275273/figure/pone-0115985-g005/" class="imagepopup">Figure 5. Roles of Gαo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in regulating behavioral choice between conflicting alternatives..  From: Dopamine Receptors Antagonistically Regulate Behavioral Choice between Conflicting Alternatives in C. elegans. </a></p><div class="supp"><p class="details"><div>(A) Phenotypes of Gα<sub>o</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutants in the interaction assay under the well-fed or starved condition. In the assay system, the Cu<sup>2+</sup> ion concentration was 100 mM, and the diacetyl concentration was 10<sup>−2</sup>. (B) Dose-response curves of wild-type N2 and mutants to diacetyl with (+) or without (-) 100 mM of Cu<sup>2+</sup> ion. Differences between groups were determined using two-way ANOVA. (C) Dose-response curves of wild-type N2 and mutants to Cu<sup>2+</sup> ion with (+) or without (-) 10<sup>−2</sup> of diacetyl. Differences between groups were determined using two-way ANOVA. (D) Locomotion behavior of wild-type N2 and mutants in the absence (-) of food under the well-fed or starved condition. Locomotion behavior was assessed by the body bend. Bars represent mean ± S.E.M. **<i>P</i>&lt;0.01 <i>vs</i> N2 (if not specially indicated).</div></p></div><div class="aux"><div class="resc">Daoyong Wang, et al. PLoS One. 2014;9(12):e115985.</div><p class="links"><a class="dblinks" href="/pubmed/25536037" ref="ordinalpos=3073&amp;ncbi_uid=7007974&amp;link_uid=25536037">Citation</a><a class="dblinks" href="/pmc/articles/PMC4275273/" ref="ordinalpos=3073&amp;ncbi_uid=7007974&amp;link_uid=PMC4275273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3074.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2814867/figure/pone-0009024-g007/" ref="ordinalpos=3074&amp;ncbi_uid=2182334&amp;link_uid=PMC2814867" href="/pmc/articles/PMC2814867/"><img src="/pmc/articles/instance/2814867/bin/pone.0009024.g007.gif" src-large="/pmc/articles/instance/2814867/bin/pone.0009024.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2814867/" ref="ordinalpos=3074&amp;ncbi_uid=2182334&amp;link_uid=PMC2814867" image-link="/pmc/articles/PMC2814867/figure/pone-0009024-g007/" class="imagepopup">Figure 7. A feedback loop mediated by the PI3K-Akt axis controls Grb7 expression..  From: Grb7 Upregulation Is a Molecular Adaptation to HER2 <span class="highlight" style="background-color:">Signaling</span> Inhibition Due to Removal of Akt-Mediated Gene Repression. </a></p><div class="supp"><p class="details"><div>Grb7 interacts with HER2, participates in HER2 <span class="highlight" style="background-color:">signaling</span>, and promotes cell survival and cell migration. HER2 exerts a repressive control on Grb7 via the PI3K-Akt <span class="highlight" style="background-color:">pathway</span>. Inhibition of HER2 <span class="highlight" style="background-color:">signaling</span> (e.g. by lapatinib) de-represses Grb7 causing its rapid upregulation. Reducing Grb7 with RNAi or preventing its interaction with HER2 using protein-protein interaction inhibitors may help increase the efficacy of anti-HER2 therapeutics and avoid the adverse consequences of Grb7 oncogenic activity.</div></p></div><div class="aux"><div class="resc">Alessio Nencioni, et al. PLoS One. 2010;5(2):e9024.</div><p class="links"><a class="dblinks" href="/pubmed/20126311" ref="ordinalpos=3074&amp;ncbi_uid=2182334&amp;link_uid=20126311">Citation</a><a class="dblinks" href="/pmc/articles/PMC2814867/" ref="ordinalpos=3074&amp;ncbi_uid=2182334&amp;link_uid=PMC2814867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3075.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2936618/figure/fig05/" ref="ordinalpos=3075&amp;ncbi_uid=2814391&amp;link_uid=PMC2936618" href="/pmc/articles/PMC2936618/"><img src="/pmc/articles/instance/2936618/bin/pnas.1007909107fig05.gif" src-large="/pmc/articles/instance/2936618/bin/pnas.1007909107fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2936618/" ref="ordinalpos=3075&amp;ncbi_uid=2814391&amp;link_uid=PMC2936618" image-link="/pmc/articles/PMC2936618/figure/fig05/" class="imagepopup">Fig. 5. From: 12/15-lipoxygenase-derived lipid peroxides control receptor tyrosine kinase <span class="highlight" style="background-color:">signaling</span> through oxidation of protein tyrosine phosphatases. </a></p><div class="supp"><p class="details"><div>Accumulation of peroxidized lipids and increased PDGF receptor phosphorylation in Gpx4-deleted cells is dependent on 12/15-LOX–derived peroxides. (<i>A</i>) Addition of AA861 prevented PDGF β-receptor down-regulation and hyperphosphorylation of <i>Gpx4<sup>−/−</sup></i> cells. (<i>B</i>) A schematic summary of a unique <span class="highlight" style="background-color:">pathway</span> for control of RTK <span class="highlight" style="background-color:">signaling</span> including peroxidized lipid-mediated oxidation of PTPs.</div></p></div><div class="aux"><div class="resc">Marcus Conrad, et al. Proc Natl Acad Sci U S A. 2010 September 7;107(36):15774-15779.</div><p class="links"><a class="dblinks" href="/pubmed/20798033" ref="ordinalpos=3075&amp;ncbi_uid=2814391&amp;link_uid=20798033">Citation</a><a class="dblinks" href="/pmc/articles/PMC2936618/" ref="ordinalpos=3075&amp;ncbi_uid=2814391&amp;link_uid=PMC2936618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3076.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2500174/figure/pone-0003001-g007/" ref="ordinalpos=3076&amp;ncbi_uid=1451634&amp;link_uid=PMC2500174" href="/pmc/articles/PMC2500174/"><img src="/pmc/articles/instance/2500174/bin/pone.0003001.g007.gif" src-large="/pmc/articles/instance/2500174/bin/pone.0003001.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2500174/" ref="ordinalpos=3076&amp;ncbi_uid=1451634&amp;link_uid=PMC2500174" image-link="/pmc/articles/PMC2500174/figure/pone-0003001-g007/" class="imagepopup">Figure 7. A model summarizing the Rho-Rock-Myosin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that regulates basic cell-cell interactions in ES cells..  From: The Rho-Rock-Myosin <span class="highlight" style="background-color:">Signaling</span> Axis Determines Cell-Cell Integrity of Self-Renewing Pluripotent Stem Cells. </a></p><div class="supp"><p class="details"><div>Chemicals and siRNAs used in the study are highlighted in red and asterisks, respectively. Dotted lines indicate potential mechanistic interactions within or between the cell-integrity and self-renewal pathways, although further investigations are required to address these possibilities. Arrows denote activation and bars indicate inhibition.</div></p></div><div class="aux"><div class="resc">Nicole Harb, et al. PLoS ONE. 2008;3(8):e3001.</div><p class="links"><a class="dblinks" href="/pubmed/18714354" ref="ordinalpos=3076&amp;ncbi_uid=1451634&amp;link_uid=18714354">Citation</a><a class="dblinks" href="/pmc/articles/PMC2500174/" ref="ordinalpos=3076&amp;ncbi_uid=1451634&amp;link_uid=PMC2500174">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3077.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3940872/figure/pone-0090275-g007/" ref="ordinalpos=3077&amp;ncbi_uid=5895739&amp;link_uid=PMC3940872" href="/pmc/articles/PMC3940872/"><img src="/pmc/articles/instance/3940872/bin/pone.0090275.g007.gif" src-large="/pmc/articles/instance/3940872/bin/pone.0090275.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3940872/" ref="ordinalpos=3077&amp;ncbi_uid=5895739&amp;link_uid=PMC3940872" image-link="/pmc/articles/PMC3940872/figure/pone-0090275-g007/" class="imagepopup">Figure 7. Model of inhibition of the JAK2/STAT5 <span class="highlight" style="background-color:">pathway</span> by α-Br-TMC..  From: The Synthetic ?-Bromo-2?,3,4,4?-Tetramethoxychalcone (?-Br-TMC) Inhibits the JAK/STAT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>IL-3 binding to the α/βc chains of the IL-3 receptor leads to activation of the receptor-associated JAK2 tyrosine kinase by trans-phosphorylation. In turn, JAK2-mediated activation of the STAT5, MAPK and AKT pathways via phosphorylation (broad arrows) results in induced transcription of downstream target genes (thin arrows). We showed that α-Br-TMC inhibits JAK2 phosphorylation, hence impairing JAK2-regulated <span class="highlight" style="background-color:">signaling</span> pathways. In addition and independently of JAK2, α-Br-TMC inhibits STAT5 activity. Concomitantly, STAT5A and STAT5B protein mobility in SDS-PAGE is altered, indicating a change in their post-translational modification state induced by α-Br-TMC, which might be associated to STAT5 altered transcriptional activity.</div></p></div><div class="aux"><div class="resc">Sophia Pinz, et al. PLoS One. 2014;9(3):e90275.</div><p class="links"><a class="dblinks" href="/pubmed/24595334" ref="ordinalpos=3077&amp;ncbi_uid=5895739&amp;link_uid=24595334">Citation</a><a class="dblinks" href="/pmc/articles/PMC3940872/" ref="ordinalpos=3077&amp;ncbi_uid=5895739&amp;link_uid=PMC3940872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3078.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4146087/figure/F3/" ref="ordinalpos=3078&amp;ncbi_uid=6600602&amp;link_uid=PMC4146087" href="/pmc/articles/PMC4146087/"><img src="/pmc/articles/instance/4146087/bin/NRR-8-843-g003.gif" src-large="/pmc/articles/instance/4146087/bin/NRR-8-843-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4146087/" ref="ordinalpos=3078&amp;ncbi_uid=6600602&amp;link_uid=PMC4146087" image-link="/pmc/articles/PMC4146087/figure/F3/" class="imagepopup">Figure 3.  From: Mechanisms of extracellular signal-regulated kinase/cAMP response element-binding protein/brain-derived neurotrophic factor signal transduction <span class="highlight" style="background-color:">pathway</span> in depressive disorder☆. </a></p><div class="supp"><p class="details"><div>Regulation by zinc of the extracellular signal-regulated kinase (ERK) 1/2 <span class="highlight" style="background-color:">signaling</span> cascade.<br />Zinc can activate the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at various levels1: (1) by inhibiting phosphatases that are involved in ERK1/2 dephosphorylation, and inhibits kinases that indirectly regulate the phosphorylation levels of receptors that subsequently activate the ERK cascade; (2) by directly binding to and activating receptors (<i>e.g</i>., G protein-coupled receptor 39) which have ERK1/2 as a downstream <span class="highlight" style="background-color:">signaling</span> cascade; (3) by activating MMPs, which subsequently cleave and activate substrates (<i>e.g</i>., pro-BDNF) releasing ligands (<i>e.g</i>., BDNF) that activate receptors upstream from the ERK1/2 cascade.<br />MEK: Mitogen-activated protein kinase/ERK kinase; MMP: matrix metalloproteinases; BDNF: brain-derived neurotrophic factor.</div></p></div><div class="aux"><div class="resc">Hongyan Wang, et al. Neural Regen Res. 2013 March 25;8(9):843-852.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3078&amp;ncbi_uid=6600602&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4146087/" ref="ordinalpos=3078&amp;ncbi_uid=6600602&amp;link_uid=PMC4146087">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3079.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2928401/figure/F8/" ref="ordinalpos=3079&amp;ncbi_uid=3249033&amp;link_uid=PMC2928401" href="/pmc/articles/PMC2928401/"><img src="/pmc/articles/instance/2928401/bin/nihms223339f8.gif" src-large="/pmc/articles/instance/2928401/bin/nihms223339f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2928401/" ref="ordinalpos=3079&amp;ncbi_uid=3249033&amp;link_uid=PMC2928401" image-link="/pmc/articles/PMC2928401/figure/F8/" class="imagepopup">Figure 8. <span class="highlight" style="background-color:">Pathway</span> Associations of TFAP2C Target Genes.  From: Identification of Primary Gene Targets of TFAP2C in Hormone Responsive Breast Carcinoma Cells. </a></p><div class="supp"><p class="details"><div>The 447 TFAP2C target genes were analyzed using Ingenuity Systems to determine associations between genes for the purpose of defining physiologic pathways regulated by TFAP2C. The main <span class="highlight" style="background-color:">pathway</span> (with the greatest number of genes) is shown and was characterized as involved in cell growth and proliferation. Prominent genes noted in this network include <i>MYC</i>, <i>ESR1</i> and <i>ERK</i>. Red tone indicates genes repressed by TFAP2C and green indicates genes induced by TFAP2C. <b>Inset:</b> Retinoic acid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was also identified as a prominent network involved with TFAP2C target genes, <i>RXR</i>, <i>RAR</i> and <i>CRABP2</i>.</div></p></div><div class="aux"><div class="resc">George W. Woodfield, et al. Genes Chromosomes Cancer. ;49(10):948-962.</div><p class="links"><a class="dblinks" href="/pubmed/20629094" ref="ordinalpos=3079&amp;ncbi_uid=3249033&amp;link_uid=20629094">Citation</a><a class="dblinks" href="/pmc/articles/PMC2928401/" ref="ordinalpos=3079&amp;ncbi_uid=3249033&amp;link_uid=PMC2928401">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3080.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2923990/figure/F7/" ref="ordinalpos=3080&amp;ncbi_uid=3165098&amp;link_uid=PMC2923990" href="/pmc/articles/PMC2923990/"><img src="/pmc/articles/instance/2923990/bin/zbc0351027670007.gif" src-large="/pmc/articles/instance/2923990/bin/zbc0351027670007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2923990/" ref="ordinalpos=3080&amp;ncbi_uid=3165098&amp;link_uid=PMC2923990" image-link="/pmc/articles/PMC2923990/figure/F7/" class="imagepopup">FIGURE 7. From: CARD9 Mediates Dectin-2-induced I?B? Kinase Ubiquitination Leading to Activation of NF-?B in Response to Stimulation by the Hyphal Form of Candida albicans. </a></p><div class="supp"><p class="details"><div><b>The model for <i>C. albicans</i>-induced <span class="highlight" style="background-color:">signaling</span> pathways leading to NF-κB activation.</b> The hyphal form of <i>C. albicans</i> recognizes Dectin-2, whereas yeast and β-glucans, such as zymosan, activate Dectin-1. Although both Dectin-1 and Dectin-2 pathways are required for Syk that regulates IKK phosphorylation, CARD9 and Bcl10 are selectively involved in Dectin-2-induced <span class="highlight" style="background-color:">signaling</span> cascades leading to regulation of NEMO ubiquitination. The phosphorylation and ubiquitination of IKK in the Dectin-2-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are required for activation of the IKK complex for NF-κB activation. LPS and other TLR ligands activate NF-κB through a MyD88-dependent but CARD9-independent <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Liangkuan Bi, et al. J Biol Chem. 2010 August 20;285(34):25969-25977.</div><p class="links"><a class="dblinks" href="/pubmed/20538615" ref="ordinalpos=3080&amp;ncbi_uid=3165098&amp;link_uid=20538615">Citation</a><a class="dblinks" href="/pmc/articles/PMC2923990/" ref="ordinalpos=3080&amp;ncbi_uid=3165098&amp;link_uid=PMC2923990">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3081.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4123363/figure/F2/" ref="ordinalpos=3081&amp;ncbi_uid=6477351&amp;link_uid=PMC4123363" href="/pmc/articles/PMC4123363/"><img src="/pmc/articles/instance/4123363/bin/WJG-20-9744-g002.gif" src-large="/pmc/articles/instance/4123363/bin/WJG-20-9744-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4123363/" ref="ordinalpos=3081&amp;ncbi_uid=6477351&amp;link_uid=PMC4123363" image-link="/pmc/articles/PMC4123363/figure/F2/" class="imagepopup">Figure 2.  From: p38? MAPK <span class="highlight" style="background-color:">pathway</span>: A key factor in colorectal cancer therapy and chemoresistance. </a></p><div class="supp"><p class="details"><div>Schematic representation of the p38α/ERK cross-talk in colorectal cancer. A: Inhibition of MEK1 triggers phospho-activation of the p38 MAPK <span class="highlight" style="background-color:">pathway</span>, indicating the existence of a p38α/ERK crosstalk in CRC cells; B: p38α inhibition induces increased expression of HER3, one of the receptor tyrosine kinases (RTK) of the EGF <span class="highlight" style="background-color:">pathway</span>, and this effect is dependent upon the activity of FoxO3A and its cofactor SIRT1. In turn, HER3 up-regulation leads to over-activation of the MEK/ERK <span class="highlight" style="background-color:">pathway</span>. Moreover, p38α inhibition leads to the rescue of a pro-apoptotic program driven by the extrinsic <span class="highlight" style="background-color:">pathway</span> through transcriptional up-regulation of TRAIL and activation of caspase-8; C: Concomitant MEK inhibition triggers Bax-dependent apoptosis by enabling <span class="highlight" style="background-color:">signaling</span> propagation through t-Bid and caspase 3. Bid phosphorylation at MAPK phosphorylation sites is induced by p38α inhibition and abrogated by concomitant inhibition of MEK1.</div></p></div><div class="aux"><div class="resc">Valentina Grossi, et al. World J Gastroenterol. 2014 August 7;20(29):9744-9758.</div><p class="links"><a class="dblinks" href="/pubmed/25110412" ref="ordinalpos=3081&amp;ncbi_uid=6477351&amp;link_uid=25110412">Citation</a><a class="dblinks" href="/pmc/articles/PMC4123363/" ref="ordinalpos=3081&amp;ncbi_uid=6477351&amp;link_uid=PMC4123363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3082.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3599046/figure/F1/" ref="ordinalpos=3082&amp;ncbi_uid=4745759&amp;link_uid=PMC3599046" href="/pmc/articles/PMC3599046/"><img src="/pmc/articles/instance/3599046/bin/1471-2164-14-64-1.gif" src-large="/pmc/articles/instance/3599046/bin/1471-2164-14-64-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3599046/" ref="ordinalpos=3082&amp;ncbi_uid=4745759&amp;link_uid=PMC3599046" image-link="/pmc/articles/PMC3599046/figure/F1/" class="imagepopup">Figure 1.  From: Comparison and contrast of genes and biological pathways responding to Marek's disease virus infection using allele-specific expression and differential expression in broiler and layer chickens. </a></p><div class="supp"><p class="details"><div><b>The JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in response to MDV infection.</b> The JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is displayed and genes that exhibit either differential expression (DE) or allele-specific expression (ASE) following Marek’s disease virus (MDV) infection in both broilers and layers are shown.</div></p></div><div class="aux"><div class="resc">Sudeep Perumbakkam, et al. BMC Genomics. 2013;14:64-64.</div><p class="links"><a class="dblinks" href="/pubmed/23363372" ref="ordinalpos=3082&amp;ncbi_uid=4745759&amp;link_uid=23363372">Citation</a><a class="dblinks" href="/pmc/articles/PMC3599046/" ref="ordinalpos=3082&amp;ncbi_uid=4745759&amp;link_uid=PMC3599046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3083.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2974943/figure/F2/" ref="ordinalpos=3083&amp;ncbi_uid=3590123&amp;link_uid=PMC2974943" href="/pmc/articles/PMC2974943/"><img src="/pmc/articles/instance/2974943/bin/nihms213988f2.gif" src-large="/pmc/articles/instance/2974943/bin/nihms213988f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2974943/" ref="ordinalpos=3083&amp;ncbi_uid=3590123&amp;link_uid=PMC2974943" image-link="/pmc/articles/PMC2974943/figure/F2/" class="imagepopup">Figure 2.  From: Regulation of Facial Morphogenesis by Endothelin <span class="highlight" style="background-color:">Signaling</span>: Insights from Mice and Fish. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> within cranial neural crest cells mediated by endothelin-1 (Edn1). Preproendothelin-1 coming from cells within the pharyngeal arch environment undergoes two processing events (Furin A/B and endothelin converting enzyme 1/2 (Ece1, 2) before binding to the endothelin-A receptor (Ednra). This initiates <span class="highlight" style="background-color:">signaling</span> through Gαq/Gα11, resulting in phospholipase-β3 activity (Plcβ3) and thus changes in gene expression. Genetic evidence that individual Edn1/Ednra <span class="highlight" style="background-color:">pathway</span> members regulate craniofacial development has come from either mouse (highlighted in orange), zebrafish (highlighted in blue) or both (highlighted in orange/blue).</div></p></div><div class="aux"><div class="resc">David E. Clouthier, et al. Am J Med Genet A. ;152A(12):2962-2973.</div><p class="links"><a class="dblinks" href="/pubmed/20684004" ref="ordinalpos=3083&amp;ncbi_uid=3590123&amp;link_uid=20684004">Citation</a><a class="dblinks" href="/pmc/articles/PMC2974943/" ref="ordinalpos=3083&amp;ncbi_uid=3590123&amp;link_uid=PMC2974943">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3084.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3602389/figure/F1/" ref="ordinalpos=3084&amp;ncbi_uid=5859190&amp;link_uid=PMC3602389" href="/pmc/articles/PMC3602389/"><img src="/pmc/articles/instance/3602389/bin/nihms-440010-f0001.gif" src-large="/pmc/articles/instance/3602389/bin/nihms-440010-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3602389/" ref="ordinalpos=3084&amp;ncbi_uid=5859190&amp;link_uid=PMC3602389" image-link="/pmc/articles/PMC3602389/figure/F1/" class="imagepopup">Figure 1.  From: The effects of ingested mammalian blood factors on vector arthropod immunity and physiology. </a></p><div class="supp"><p class="details"><div>Model of insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in vector arthropods. Solid lines indicate proven direct interactions between proteins, and dashed lines indicate indirect interactions that may involve other <span class="highlight" style="background-color:">signaling</span> proteins.</div></p></div><div class="aux"><div class="resc">Nazzy Pakpour, et al. Microbes Infect. 2013 March;15(3):243-254.</div><p class="links"><a class="dblinks" href="/pubmed/23370408" ref="ordinalpos=3084&amp;ncbi_uid=5859190&amp;link_uid=23370408">Citation</a><a class="dblinks" href="/pmc/articles/PMC3602389/" ref="ordinalpos=3084&amp;ncbi_uid=5859190&amp;link_uid=PMC3602389">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3085.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3859629/figure/pone-0082639-g007/" ref="ordinalpos=3085&amp;ncbi_uid=5597857&amp;link_uid=PMC3859629" href="/pmc/articles/PMC3859629/"><img src="/pmc/articles/instance/3859629/bin/pone.0082639.g007.gif" src-large="/pmc/articles/instance/3859629/bin/pone.0082639.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3859629/" ref="ordinalpos=3085&amp;ncbi_uid=5597857&amp;link_uid=PMC3859629" image-link="/pmc/articles/PMC3859629/figure/pone-0082639-g007/" class="imagepopup">Figure 7. A and B: VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is modulated by sitagliptin treatment..  From: Dipeptidyl Peptidase IV Inhibition Activates CREB and Improves Islet Vascularization through VEGF-A/VEGFR-2 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Sections were treated with Insulin, VEGF and VEGFR-2 antibodies at 4°C overnight and then treated with secondary antibodies. The image was captured using Leica fluorescent microscope and analyzed using image J software. VEGF and VEGFR-2 staining was more pronounced in the sitagliptin treated group compared to the control. The area showing VEGF and VEGFR-2 staining was significantly (p &lt; 0.05) increased in the sitagliptin group as represented by their respective bar graphs. n  =  4; Data represents mean ± SEM.</div></p></div><div class="aux"><div class="resc">Balaji Samikannu, et al. PLoS One. 2013;8(12):e82639.</div><p class="links"><a class="dblinks" href="/pubmed/24349326" ref="ordinalpos=3085&amp;ncbi_uid=5597857&amp;link_uid=24349326">Citation</a><a class="dblinks" href="/pmc/articles/PMC3859629/" ref="ordinalpos=3085&amp;ncbi_uid=5597857&amp;link_uid=PMC3859629">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3086.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3561831/figure/F3/" ref="ordinalpos=3086&amp;ncbi_uid=5653900&amp;link_uid=PMC3561831" href="/pmc/articles/PMC3561831/"><img src="/pmc/articles/instance/3561831/bin/JCI64595.f3.gif" src-large="/pmc/articles/instance/3561831/bin/JCI64595.f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3561831/" ref="ordinalpos=3086&amp;ncbi_uid=5653900&amp;link_uid=PMC3561831" image-link="/pmc/articles/PMC3561831/figure/F3/" class="imagepopup">Figure 3. Boosting brain insulin <span class="highlight" style="background-color:">signaling</span> to combat AD..  From: Alzheimer's disease and insulin resistance: translating basic science into clinical applications. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) In healthy aging, IRs are present at synapses, and proper insulin <span class="highlight" style="background-color:">signaling</span> favors synapse function and leads to memory formation. (<b>B</b>) In early AD, accumulation of AβOs stimulates TNF-α <span class="highlight" style="background-color:">signaling</span>, which activates the JNK <span class="highlight" style="background-color:">pathway</span> (11, 54) and, possibly, PKR and IKK pathways. Activation of these stress-sensitive kinases, which can also be triggered by ER stress in peripheral tissues (69), results in IRS-1pSer, decreasing downstream insulin <span class="highlight" style="background-color:">signaling</span> (11, 31, 54). This contributes to initial cognitive impairment in early AD. (<b>C</b>) At later AD stages, increased accumulation of AβOs and their binding to synapses lead to removal of IRs from the neuronal plasma membrane (14, 15, 47). Additionally, TNF-α/JNK activation ultimately blocks insulin actions (11), contributing to severe impairment in cognition and memory. (<b>D</b>) Stimulation of insulin and GLP-1Rs blocks early AβO-induced defects in insulin <span class="highlight" style="background-color:">signaling</span>. Insulin protects neurons by preventing AβO binding to neurons (15). In addition, activation of GLP-1Rs by exendin-4 or liraglutide and of IRs by insulin prevents JNK activation, allowing physiological tyrosine phosphorylation of IRS-1 and stimulating downstream insulin <span class="highlight" style="background-color:">signaling</span>. (<b>E</b>) At later AD stages, activation of GLP-1Rs prevents inhibitory IRS-1pSer, stimulating insulin-related downstream <span class="highlight" style="background-color:">signaling</span> pathways and ameliorating cognitive and memory impairment. Red arrows indicate inhibitory pathways and green arrows indicate stimulatory pathways of insulin <span class="highlight" style="background-color:">signaling</span>. </div></p></div><div class="aux"><div class="resc">Fernanda G. De Felice. J Clin Invest. 2013 February 1;123(2):531-539.</div><p class="links"><a class="dblinks" href="/pubmed/23485579" ref="ordinalpos=3086&amp;ncbi_uid=5653900&amp;link_uid=23485579">Citation</a><a class="dblinks" href="/pmc/articles/PMC3561831/" ref="ordinalpos=3086&amp;ncbi_uid=5653900&amp;link_uid=PMC3561831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3087.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3264680/figure/F3/" ref="ordinalpos=3087&amp;ncbi_uid=3769526&amp;link_uid=PMC3264680" href="/pmc/articles/PMC3264680/"><img src="/pmc/articles/instance/3264680/bin/nihms349582f3.gif" src-large="/pmc/articles/instance/3264680/bin/nihms349582f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3264680/" ref="ordinalpos=3087&amp;ncbi_uid=3769526&amp;link_uid=PMC3264680" image-link="/pmc/articles/PMC3264680/figure/F3/" class="imagepopup">Fig. 3.  From: IGF2 Overexpression in Solitary Fibrous Tumors is Independent of Anatomic Location and is Related to Loss of Imprinting. </a></p><div class="supp"><p class="details"><div>Diagram of IGF/INSR/IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mihai Hajdu, et al. J Pathol. ;221(3):300-307.</div><p class="links"><a class="dblinks" href="/pubmed/20527023" ref="ordinalpos=3087&amp;ncbi_uid=3769526&amp;link_uid=20527023">Citation</a><a class="dblinks" href="/pmc/articles/PMC3264680/" ref="ordinalpos=3087&amp;ncbi_uid=3769526&amp;link_uid=PMC3264680">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3088.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2783837/figure/F2/" ref="ordinalpos=3088&amp;ncbi_uid=2647588&amp;link_uid=PMC2783837" href="/pmc/articles/PMC2783837/"><img src="/pmc/articles/instance/2783837/bin/nihms-152920-f0002.gif" src-large="/pmc/articles/instance/2783837/bin/nihms-152920-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783837/" ref="ordinalpos=3088&amp;ncbi_uid=2647588&amp;link_uid=PMC2783837" image-link="/pmc/articles/PMC2783837/figure/F2/" class="imagepopup">Figure 2.  From: TNF-related apoptosis-inducing ligand (TRAIL): A new path to anti-cancer therapies. </a></p><div class="supp"><p class="details"><div>Schematic representation of TRAIL-R1/-R2 apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Peter A. Holoch, et al. Eur J Pharmacol. ;625(1-3):63-72.</div><p class="links"><a class="dblinks" href="/pubmed/19836385" ref="ordinalpos=3088&amp;ncbi_uid=2647588&amp;link_uid=19836385">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783837/" ref="ordinalpos=3088&amp;ncbi_uid=2647588&amp;link_uid=PMC2783837">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3089.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3049961/figure/F1/" ref="ordinalpos=3089&amp;ncbi_uid=3448634&amp;link_uid=PMC3049961" href="/pmc/articles/PMC3049961/"><img src="/pmc/articles/instance/3049961/bin/nihms-266451-f0001.gif" src-large="/pmc/articles/instance/3049961/bin/nihms-266451-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3049961/" ref="ordinalpos=3089&amp;ncbi_uid=3448634&amp;link_uid=PMC3049961" image-link="/pmc/articles/PMC3049961/figure/F1/" class="imagepopup">Figure 1. PhosphoScan Analysis of ΔEGFR <span class="highlight" style="background-color:">signaling</span> in two glioma cell lines.  From: Analysis of phosphotyrosine <span class="highlight" style="background-color:">signaling</span> in glioblastoma identifies STAT5 as a novel downstream target of ?EGFR. </a></p><div class="supp"><p class="details"><div>(A) Western blot of EGR and PI3K <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in the cells used for the PhosphoScan analysis. For this analysis, the same lysates were run in parallel on different gels for blotting with the antibodies indicated. (B) Schematic of the PhosphoScan experiment: for each cell line (LNZ308 and LN428) three independent sets of four isolates were prepared (serum starved cells expressing ΔEGFR, ΔEGFR-ki, EGFR and EGFR stimulated with EGF). These were processed for PhosphoScan and analyzed by mass spectrometry.</div></p></div><div class="aux"><div class="resc">Vaibhav Chumbalkar, et al. J Proteome Res. ;10(3):1343-1352.</div><p class="links"><a class="dblinks" href="/pubmed/21214269" ref="ordinalpos=3089&amp;ncbi_uid=3448634&amp;link_uid=21214269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3049961/" ref="ordinalpos=3089&amp;ncbi_uid=3448634&amp;link_uid=PMC3049961">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3090.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2956057/figure/Fig2/" ref="ordinalpos=3090&amp;ncbi_uid=2570232&amp;link_uid=PMC2956057" href="/pmc/articles/PMC2956057/"><img src="/pmc/articles/instance/2956057/bin/109_2010_644_Fig2_HTML.gif" src-large="/pmc/articles/instance/2956057/bin/109_2010_644_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2956057/" ref="ordinalpos=3090&amp;ncbi_uid=2570232&amp;link_uid=PMC2956057" image-link="/pmc/articles/PMC2956057/figure/Fig2/" class="imagepopup">Fig. 2.  From: Mouse models of cancers: opportunities to address heterogeneity of human cancer and evaluate therapeutic strategies. </a></p><div class="supp"><p class="details"><div>Theraputic strategy based on <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> profiles. The design of a proof of the principle experiment where <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> probabilities guide the use of different drug combinations is shown. A line of MMTV-based transgenic mice that develop tumors with demonstrated heterogeneity are used in this scheme. As spontaneous tumors develop, they are biopsied and their gene expression patterns are immediately examined. Based on the patterns of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation, the mice will be grouped into the appropriate therapeutic option or into a control group. Following treatment with the individualized predicted therapeutic combinations, the response of the tumor will be assessed</div></p></div><div class="aux"><div class="resc">Eran R. Andrechek, et al. J Mol Med. 2010 November;88(11):1095-1100.</div><p class="links"><a class="dblinks" href="/pubmed/20574808" ref="ordinalpos=3090&amp;ncbi_uid=2570232&amp;link_uid=20574808">Citation</a><a class="dblinks" href="/pmc/articles/PMC2956057/" ref="ordinalpos=3090&amp;ncbi_uid=2570232&amp;link_uid=PMC2956057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3091.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1693942/figure/fig5/" ref="ordinalpos=3091&amp;ncbi_uid=820026&amp;link_uid=PMC1693942" href="/pmc/articles/PMC1693942/"><img src="/pmc/articles/instance/1693942/bin/tpc1803047f05.gif" src-large="/pmc/articles/instance/1693942/bin/tpc1803047f05.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1693942/" ref="ordinalpos=3091&amp;ncbi_uid=820026&amp;link_uid=PMC1693942" image-link="/pmc/articles/PMC1693942/figure/fig5/" class="imagepopup">Figure 5. From: The Exoribonuclease XRN4 Is a Component of the Ethylene Response <span class="highlight" style="background-color:">Pathway</span> in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Representation of the Ethylene <span class="highlight" style="background-color:">Signaling</span> Cascade Including <i>XRN4/EIN5</i>.<br />Ethylene is perceived by the ETHYLENE RESPONSE (ETR/ERS) receptors located in the endoplasmic reticulum membrane. Binding of ethylene to the receptors results in the inactivation of both the receptors and CTR1, thereby causing the derepression of positive regulatory factors, such as EIN2 (data not shown) and XRN4/EIN5. How CTR1 regulates EIN2 and XRN4/EIN5 and the relationship of EIN2 with XRN4/EIN5 in the ethylene <span class="highlight" style="background-color:">signaling</span> cascade are unknown. Strikingly, our model proposes the existence of two degradation pathways in ethylene <span class="highlight" style="background-color:">signaling</span>: one RNA decay <span class="highlight" style="background-color:">pathway</span> into the cytosol that indirectly controls the steady state levels of EBF1/2 mRNAs, which are components of a protein degradation <span class="highlight" style="background-color:">pathway</span> that controls EIN3 stability in the nucleus.</div></p></div><div class="aux"><div class="resc">Thomas Potuschak, et al. Plant Cell. 2006 November;18(11):3047-3057.</div><p class="links"><a class="dblinks" href="/pubmed/17085683" ref="ordinalpos=3091&amp;ncbi_uid=820026&amp;link_uid=17085683">Citation</a><a class="dblinks" href="/pmc/articles/PMC1693942/" ref="ordinalpos=3091&amp;ncbi_uid=820026&amp;link_uid=PMC1693942">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3092.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1251585/figure/f6/" ref="ordinalpos=3092&amp;ncbi_uid=671272&amp;link_uid=PMC1251585" href="/pmc/articles/PMC1251585/"><img src="/pmc/articles/instance/1251585/bin/zjb0190551050006.gif" src-large="/pmc/articles/instance/1251585/bin/zjb0190551050006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1251585/" ref="ordinalpos=3092&amp;ncbi_uid=671272&amp;link_uid=PMC1251585" image-link="/pmc/articles/PMC1251585/figure/f6/" class="imagepopup">FIG. 6. From: Role of RcsF in <span class="highlight" style="background-color:">Signaling</span> to the Rcs Phosphorelay <span class="highlight" style="background-color:">Pathway</span> in Escherichia coli . </a></p><div class="supp"><p class="details"><div>Model of the Rcs <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The schematic diagram is not to scale but shows the <span class="highlight" style="background-color:">signaling</span> flow through the Rcs cascade. Question marks indicate experimentally unresolved issues. RcsF localization to the outer membrane is according to in silico predictions based on the amino acid sequence and A. Jacq (personal communication; see Discussion). Amino acids involved in the phosphorelay are indicated. The shaded areas of RcsA and RcsB indicate the DNA-binding domains of these proteins. A dashed arrow points to genes that are RcsB dependent but RcsA independent. A blocked line indicates a gene repressed by an RcsA-RcsB heterodimer, and a solid arrow points to genes that require RcsA and RcsB for activation (see the text for details).</div></p></div><div class="aux"><div class="resc">Nadim Majdalani, et al. J Bacteriol. 2005 October;187(19):6770-6778.</div><p class="links"><a class="dblinks" href="/pubmed/16166540" ref="ordinalpos=3092&amp;ncbi_uid=671272&amp;link_uid=16166540">Citation</a><a class="dblinks" href="/pmc/articles/PMC1251585/" ref="ordinalpos=3092&amp;ncbi_uid=671272&amp;link_uid=PMC1251585">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3093.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4222910/figure/pone-0111244-g006/" ref="ordinalpos=3093&amp;ncbi_uid=6833908&amp;link_uid=PMC4222910" href="/pmc/articles/PMC4222910/"><img src="/pmc/articles/instance/4222910/bin/pone.0111244.g006.gif" src-large="/pmc/articles/instance/4222910/bin/pone.0111244.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4222910/" ref="ordinalpos=3093&amp;ncbi_uid=6833908&amp;link_uid=PMC4222910" image-link="/pmc/articles/PMC4222910/figure/pone-0111244-g006/" class="imagepopup">Figure 6. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> map screened by Metadrug..  From: In Vitro, In Silico and In Vivo Studies of Ursolic Acid as an Anti-Filarial Agent. </a></p><div class="aux"><div class="resc">Komal Kalani, et al. PLoS One. 2014;9(11):e111244.</div><p class="links"><a class="dblinks" href="/pubmed/25375886" ref="ordinalpos=3093&amp;ncbi_uid=6833908&amp;link_uid=25375886">Citation</a><a class="dblinks" href="/pmc/articles/PMC4222910/" ref="ordinalpos=3093&amp;ncbi_uid=6833908&amp;link_uid=PMC4222910">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3094.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2848195/figure/F3/" ref="ordinalpos=3094&amp;ncbi_uid=2314601&amp;link_uid=PMC2848195" href="/pmc/articles/PMC2848195/"><img src="/pmc/articles/instance/2848195/bin/1752-0509-4-20-3.gif" src-large="/pmc/articles/instance/2848195/bin/1752-0509-4-20-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2848195/" ref="ordinalpos=3094&amp;ncbi_uid=2314601&amp;link_uid=PMC2848195" image-link="/pmc/articles/PMC2848195/figure/F3/" class="imagepopup">Figure 3.  From: Integrated cellular network of transcription regulations and protein-protein interactions. </a></p><div class="supp"><p class="details"><div><b>The constructed subnetwork of HOG MAPK <span class="highlight" style="background-color:">signaling</span> regulatory <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Yu-Chao Wang, et al. BMC Syst Biol. 2010;4:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/20211003" ref="ordinalpos=3094&amp;ncbi_uid=2314601&amp;link_uid=20211003">Citation</a><a class="dblinks" href="/pmc/articles/PMC2848195/" ref="ordinalpos=3094&amp;ncbi_uid=2314601&amp;link_uid=PMC2848195">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3095.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4389992/figure/Fig1/" ref="ordinalpos=3095&amp;ncbi_uid=7397468&amp;link_uid=PMC4389992" href="/pmc/articles/PMC4389992/"><img src="/pmc/articles/instance/4389992/bin/12916_2015_292_Fig1_HTML.gif" src-large="/pmc/articles/instance/4389992/bin/12916_2015_292_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4389992/" ref="ordinalpos=3095&amp;ncbi_uid=7397468&amp;link_uid=PMC4389992" image-link="/pmc/articles/PMC4389992/figure/Fig1/" class="imagepopup">Figure 1.  From: Improved survival among colon cancer patients with increased differentially expressed pathways. </a></p><div class="supp"><p class="details"><div> <b>Significant canonical pathways identified from IPA.</b> The pathways were statistically significant at the 0.05 level after adjustment for multiple comparisons. <b>(a)</b> Metabolic <span class="highlight" style="background-color:">pathway</span>. <b>(b)</b> <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Martha L Slattery, et al. BMC Med. 2015;13:75.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3095&amp;ncbi_uid=7397468&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4389992/" ref="ordinalpos=3095&amp;ncbi_uid=7397468&amp;link_uid=PMC4389992">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3096.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3579325/figure/f3/" ref="ordinalpos=3096&amp;ncbi_uid=5817151&amp;link_uid=PMC3579325" href="/pmc/articles/PMC3579325/"><img src="/pmc/articles/instance/3579325/bin/fig-3.gif" src-large="/pmc/articles/instance/3579325/bin/fig-3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3579325/" ref="ordinalpos=3096&amp;ncbi_uid=5817151&amp;link_uid=PMC3579325" image-link="/pmc/articles/PMC3579325/figure/f3/" class="imagepopup">FIG. 3. From: The Effect of Environmental Enrichment on Substantia Nigra Gene Expression after Traumatic Brain Injury in Rats. </a></p><div class="supp"><p class="details"><div>Selected pathways from enriched injured (EI) versus non-enriched (i.e., STD housed) injured (NI): demonstrating calcium <span class="highlight" style="background-color:">signaling</span> importance. Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(a)</b>. Upregulation of calcineurin (CaN), calcium/calmodulin dependent protein kinase II (CaMK), and protein kinase c (PKC) is shown. There are downstream contributions to multiple functions such as apoptosis, MAPK <span class="highlight" style="background-color:">signaling</span>, and phosphatidylinositol <span class="highlight" style="background-color:">signaling</span>. The same components are also affected in the long-term potentiation pathways <b>(b)</b>.</div></p></div><div class="aux"><div class="resc">Samuel S. Shin, et al. J Neurotrauma. 2013 February 15;30(4):259-270.</div><p class="links"><a class="dblinks" href="/pubmed/23094804" ref="ordinalpos=3096&amp;ncbi_uid=5817151&amp;link_uid=23094804">Citation</a><a class="dblinks" href="/pmc/articles/PMC3579325/" ref="ordinalpos=3096&amp;ncbi_uid=5817151&amp;link_uid=PMC3579325">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3097.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4308786/figure/F8/" ref="ordinalpos=3097&amp;ncbi_uid=7127270&amp;link_uid=PMC4308786" href="/pmc/articles/PMC4308786/"><img src="/pmc/articles/instance/4308786/bin/NRR-7-1370-g010.gif" src-large="/pmc/articles/instance/4308786/bin/NRR-7-1370-g010.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4308786/" ref="ordinalpos=3097&amp;ncbi_uid=7127270&amp;link_uid=PMC4308786" image-link="/pmc/articles/PMC4308786/figure/F8/" class="imagepopup">Figure 8.  From: Mitogen activated protein kinase <span class="highlight" style="background-color:">signaling</span> pathways participate in the active principle region of Buyang Huanwu decoction-induced differentiation of bone marrow mesenchymal stem cells☆. </a></p><div class="supp"><p class="details"><div>Western blot electropherogram of nestin protein expression after the mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> was blocked.<br />1: apr-BYHWD group; 2: apr-BYHWD + SB203580 group; 3: apr-BYHWD + PD98059 group. BYHWD: <i>Buyang Huanwu</i> decoction; apr-BYHWD: active principle region of BYHWD; PD98059 and SB203580: cellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocking agents.</div></p></div><div class="aux"><div class="resc">Jinghui Zheng, et al. Neural Regen Res. 2012 June 25;7(18):1370-1377.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3097&amp;ncbi_uid=7127270&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4308786/" ref="ordinalpos=3097&amp;ncbi_uid=7127270&amp;link_uid=PMC4308786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3098.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405713/figure/F1/" ref="ordinalpos=3098&amp;ncbi_uid=4158410&amp;link_uid=PMC3405713" href="/pmc/articles/PMC3405713/"><img src="/pmc/articles/instance/3405713/bin/psb-7-273-g1.gif" src-large="/pmc/articles/instance/3405713/bin/psb-7-273-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405713/" ref="ordinalpos=3098&amp;ncbi_uid=4158410&amp;link_uid=PMC3405713" image-link="/pmc/articles/PMC3405713/figure/F1/" class="imagepopup">Figure. From: BiP links TOR <span class="highlight" style="background-color:">signaling</span> to ER stress in Chlamydomonas. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Control of BiP phosphorylation by the TOR <span class="highlight" style="background-color:">pathway</span> and ER stress in Chlamydomonas. Phosphorylation occurs at a conserved region within the peptide-binding domain (PBD) of BiP 4  and is associated to an inactive state of the protein that can be efficiently converted into an active, dephosphorylated form in response to increased protein synthesis or specific ER stress. Regulation of BiP function by phosphorylation/dephosphorylation events is therefore coordinated with nutritional and environmental inputs via the TOR <span class="highlight" style="background-color:">pathway</span> and ER stress <span class="highlight" style="background-color:">signaling</span> components.</div></p></div><div class="aux"><div class="resc">José L. Crespo. Plant Signal Behav. 2012 February 1;7(2):273-275.</div><p class="links"><a class="dblinks" href="/pubmed/22353876" ref="ordinalpos=3098&amp;ncbi_uid=4158410&amp;link_uid=22353876">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405713/" ref="ordinalpos=3098&amp;ncbi_uid=4158410&amp;link_uid=PMC3405713">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3099.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1853162/figure/fig3/" ref="ordinalpos=3099&amp;ncbi_uid=950594&amp;link_uid=PMC1853162" href="/pmc/articles/PMC1853162/"><img src="/pmc/articles/instance/1853162/bin/biophysj00090084F03_RGB.gif" src-large="/pmc/articles/instance/1853162/bin/biophysj00090084F03_RGB.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1853162/" ref="ordinalpos=3099&amp;ncbi_uid=950594&amp;link_uid=PMC1853162" image-link="/pmc/articles/PMC1853162/figure/fig3/" class="imagepopup">FIGURE 3.  From: Mathematical Models of Specificity in Cell <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The output of four <span class="highlight" style="background-color:">signaling</span> networks exhibiting different degrees of specificity and fidelity. The bars on the graph represent the total output of <span class="highlight" style="background-color:">signaling</span> pathways <i>X</i> and <i>Y</i> (<i>X</i><sub>out</sub> and <i>Y</i><sub>out</sub>, respectively) under the condition where <span class="highlight" style="background-color:">pathway</span> <i>X</i> is receiving a signal and <span class="highlight" style="background-color:">pathway</span> <i>Y</i> is not (<i>X</i><sub>in</sub>), and visa versa (<i>Y</i><sub>in</sub>). The results correspond to solutions of the network described in Eqs. 5–13. All parameter values were set equal to 1 except as specified below. The thickness of the links connecting the component nodes is proportional to the rate coefficient for that reaction (see Fig. 2 <i>A</i>; <i>j</i> is the coefficient for the connection from <i>x</i><sub>1</sub> to <i>y</i><sub>2</sub>). (<i>A</i>) This network exhibits both mutual specificity and mutual fidelity. Networks such as this are presumably the most useful to the organism. Rate coefficients are <i>a</i><sub>1</sub> = 2, <i>a</i><sub>2</sub> = 3, <i>b</i><sub>1</sub> = 2, <i>b</i><sub>2</sub> = 2.5, <i>h</i> = 0.5, <i>j</i> = 1. (<i>B</i>) This network displays mutual specificity but not mutual fidelity. <span class="highlight" style="background-color:">Pathway</span> <i>Y</i> does not exhibit fidelity with respect to <span class="highlight" style="background-color:">pathway</span> <i>X</i>, because <i>Y</i> output when <i>Y</i> is on is less than <i>Y</i> output when <i>X</i> is on. Parameter values are <i>a</i><sub>1</sub> = 2, <i>a</i><sub>2</sub> = 3, <i>b</i><sub>1</sub> = 1, <i>b</i><sub>2</sub> = 1.5, <i>h</i> = 0.5, <i>j</i> = 1. (<i>C</i>) This network possesses mutual fidelity, but <span class="highlight" style="background-color:">pathway</span> <i>Y</i> does not exhibit specificity with respect to <i>X</i>. Parameter values are <i>a</i><sub>1</sub> = 2, <i>a</i><sub>2</sub> = 3, <i>b</i><sub>1</sub> = 2, <i>b</i><sub>2</sub> = 1.5, <i>h</i> = 2, <i>j</i> = 1. (<i>D</i>) This network possesses neither mutual specificity nor mutual fidelity. Parameter values are <i>a</i><sub>1</sub> = 2, <i>a</i><sub>2</sub> = 1, <i>b</i><sub>1</sub> = 2, <i>b</i><sub>2</sub> = 1.5, <i>h</i> = 2, <i>j</i> = 3.</div></p></div><div class="aux"><div class="resc">Lee Bardwell, et al. Biophys J. 2007 May 15;92(10):3425-3441.</div><p class="links"><a class="dblinks" href="/pubmed/17325015" ref="ordinalpos=3099&amp;ncbi_uid=950594&amp;link_uid=17325015">Citation</a><a class="dblinks" href="/pmc/articles/PMC1853162/" ref="ordinalpos=3099&amp;ncbi_uid=950594&amp;link_uid=PMC1853162">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3100.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4226164/figure/F1/" ref="ordinalpos=3100&amp;ncbi_uid=6874919&amp;link_uid=PMC4226164" href="/pmc/articles/PMC4226164/"><img src="/pmc/articles/instance/4226164/bin/fnagi-06-00311-g001.gif" src-large="/pmc/articles/instance/4226164/bin/fnagi-06-00311-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4226164/" ref="ordinalpos=3100&amp;ncbi_uid=6874919&amp;link_uid=PMC4226164" image-link="/pmc/articles/PMC4226164/figure/F1/" class="imagepopup">Figure 1.  From: Bioenergetic Dysfunction and Inflammation in Alzheimer’s Disease: A Possible Connection. </a></p><div class="supp"><p class="details"><div><b>Damage-associated molecular pattern <span class="highlight" style="background-color:">signaling</span> mechanisms</b>. Mitochondrial components can induce a DAMP response through the activation of TLR, FPR, RAGE, P2X7, and/or the NALP3 inflammasome. FPR is a GCPR. FPR <span class="highlight" style="background-color:">signaling</span> is mediated through PLC, PIP2, IP3, and DAG. Downstream activation of PKC leads to <span class="highlight" style="background-color:">signaling</span> through MAPK, PI3K, and activation of NFκB and NAPDH oxidase. TLR receptors dimerize with Myd88 and can activate NFκB through RIP1 kinase, MAPK, or PI3K pathways. The TLR <span class="highlight" style="background-color:">pathway</span> also activates NADPH oxidase. RAGE receptor <span class="highlight" style="background-color:">signaling</span> activates PKC and CdC42/Rac (or Ras). Downstream PI3K, MAPK or JNK, and Erk1/2 are activated. This leads to NFκB activation (in addition to SP1 and AP1). P2X7 activation leads to NRLP3 inflammasome <span class="highlight" style="background-color:">signaling</span>. NLRP3 (or IL-1R) activation mediates inflammasome <span class="highlight" style="background-color:">signaling</span> through NLRP3, ACS, and caspase-1 (or RIG-1). Pro-IL-1 is cleaved into IL-1β and IL-1α. NFκB activation initiates transcription of Pro-IL-1, IL-10, IL-18, IL-6, TNFα, along with many other cytokine and chemotactic factors.</div></p></div><div class="aux"><div class="resc">Heather M. Wilkins, et al. Front Aging Neurosci. 2014;6:311.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3100&amp;ncbi_uid=6874919&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4226164/" ref="ordinalpos=3100&amp;ncbi_uid=6874919&amp;link_uid=PMC4226164">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="30" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="31" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="32" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:39:43-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB4412543113E10000000000617FFB;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB4412543113E10000000000617FFB" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>