
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="33" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB4B71543116410000000000B9844B" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="32" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="33" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="34" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="33" /></div>
            <div><h2 class="result_count">Results: 3201 to 3300 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3201.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1482501/figure/F4/" ref="ordinalpos=3201&amp;ncbi_uid=725122&amp;link_uid=PMC1482501" href="/pmc/articles/PMC1482501/"><img src="/pmc/articles/instance/1482501/bin/zpq0200621890004.gif" src-large="/pmc/articles/instance/1482501/bin/zpq0200621890004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1482501/" ref="ordinalpos=3201&amp;ncbi_uid=725122&amp;link_uid=PMC1482501" image-link="/pmc/articles/PMC1482501/figure/F4/" class="imagepopup">Fig. 4. From: Stochastic models for regulatory networks of the genetic toggle switch. </a></p><div class="supp"><p class="details"><div>The genetic toggle switch interfaced by a quorum-sensing <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tianhai Tian, et al. Proc Natl Acad Sci U S A. 2006 May 30;103(22):8372-8377.</div><p class="links"><a class="dblinks" href="/pubmed/16714385" ref="ordinalpos=3201&amp;ncbi_uid=725122&amp;link_uid=16714385">Citation</a><a class="dblinks" href="/pmc/articles/PMC1482501/" ref="ordinalpos=3201&amp;ncbi_uid=725122&amp;link_uid=PMC1482501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3202.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2670969/figure/F4/" ref="ordinalpos=3202&amp;ncbi_uid=1716869&amp;link_uid=PMC2670969" href="/pmc/articles/PMC2670969/"><img src="/pmc/articles/instance/2670969/bin/nihms103385f4.gif" src-large="/pmc/articles/instance/2670969/bin/nihms103385f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2670969/" ref="ordinalpos=3202&amp;ncbi_uid=1716869&amp;link_uid=PMC2670969" image-link="/pmc/articles/PMC2670969/figure/F4/" class="imagepopup">Figure 4.  From: Phenanthrene-based tylophorine-1 (PBT-1) inhibits lung cancer cell growth through the Akt and NF-?B pathways. </a></p><div class="supp"><p class="details"><div>PBT-1 induced genes in different pathways according to the KEGG and BIOCARTA <span class="highlight" style="background-color:">pathway</span> databases. PBT-1–induced genes are colored red, and the red numbers close to the genes indicate the fold increase. PBT-1–suppressed genes are colored green. (A) Cell cycle <span class="highlight" style="background-color:">pathway</span>. (B) Apoptosis <span class="highlight" style="background-color:">pathway</span>. The hypothetical PBT-1–regulated <span class="highlight" style="background-color:">signaling</span> pathways were modified from the KEGG and BIOCARTA databases.</div></p></div><div class="aux"><div class="resc">Jau-Chen Lin, et al. J Med Chem. ;52(7):1903-1911.</div><p class="links"><a class="dblinks" href="/pubmed/19284764" ref="ordinalpos=3202&amp;ncbi_uid=1716869&amp;link_uid=19284764">Citation</a><a class="dblinks" href="/pmc/articles/PMC2670969/" ref="ordinalpos=3202&amp;ncbi_uid=1716869&amp;link_uid=PMC2670969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3203.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3579806/figure/pone-0057156-g004/" ref="ordinalpos=3203&amp;ncbi_uid=4674627&amp;link_uid=PMC3579806" href="/pmc/articles/PMC3579806/"><img src="/pmc/articles/instance/3579806/bin/pone.0057156.g004.gif" src-large="/pmc/articles/instance/3579806/bin/pone.0057156.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3579806/" ref="ordinalpos=3203&amp;ncbi_uid=4674627&amp;link_uid=PMC3579806" image-link="/pmc/articles/PMC3579806/figure/pone-0057156-g004/" class="imagepopup">Figure 4. Predicted targets of miRNAs in GnRH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Differentially Expressed miRNAs after GnRH Treatment and Their Potential Roles in FSH Regulation in Porcine Anterior Pituitary Cell. </a></p><div class="supp"><p class="details"><div>This figure was drawn according to KEGG <span class="highlight" style="background-color:">pathway</span> (ssc4912), red arrow near the box (general on the right) represents up-regulated miRNAs while blue arrow represents down-regulated miRNAs. The number near the arrow indicates number of miRNAs targeting this gene or its family. For example, 10 up-regulated miRNAs and 3 down-regulated miRNAs were predicted to target FSHβ mRNA 3′-UTR.TF indicates transcription factor involved in FSHβ transcription.</div></p></div><div class="aux"><div class="resc">Rui-Song Ye, et al. PLoS One. 2013;8(2):e57156.</div><p class="links"><a class="dblinks" href="/pubmed/23451171" ref="ordinalpos=3203&amp;ncbi_uid=4674627&amp;link_uid=23451171">Citation</a><a class="dblinks" href="/pmc/articles/PMC3579806/" ref="ordinalpos=3203&amp;ncbi_uid=4674627&amp;link_uid=PMC3579806">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3204.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2376240/figure/fig8/" ref="ordinalpos=3204&amp;ncbi_uid=1351748&amp;link_uid=PMC2376240" href="/pmc/articles/PMC2376240/"><img src="/pmc/articles/instance/2376240/bin/zbc0230837520008.gif" src-large="/pmc/articles/instance/2376240/bin/zbc0230837520008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2376240/" ref="ordinalpos=3204&amp;ncbi_uid=1351748&amp;link_uid=PMC2376240" image-link="/pmc/articles/PMC2376240/figure/fig8/" class="imagepopup">FIGURE 8. From: The ?1 Integrin Activates JNK Independent of CagA, and JNK Activation Is Required for Helicobacter pylori CagA+-induced Motility of Gastric Cancer Cells. </a></p><div class="supp"><p class="details"><div><b><i>H. pylori</i> stimulates cancer cell motility through the combined CagA-dependent and -independent <span class="highlight" style="background-color:">signaling</span>.</b> CagA is injected into the cell via the α<sub>5</sub>β<sub>1</sub> integrin and associates with known regulators of cell adhesion and motility. CagA-independent activation of the β<sub>1</sub> integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to Src-induced JNK stimulation, which in combination with FAK causes paxillin<sup>Tyr-118</sup> phosphorylation. Although both CagA-dependent and CagA-independent <span class="highlight" style="background-color:">signaling</span> pathways are necessary, neither <span class="highlight" style="background-color:">pathway</span> is sufficient by itself for <i>H. pylori</i>-induced cell motility.</div></p></div><div class="aux"><div class="resc">Jared L. Snider, et al. J Biol Chem. 2008 May 16;283(20):13952-13963.</div><p class="links"><a class="dblinks" href="/pubmed/18356158" ref="ordinalpos=3204&amp;ncbi_uid=1351748&amp;link_uid=18356158">Citation</a><a class="dblinks" href="/pmc/articles/PMC2376240/" ref="ordinalpos=3204&amp;ncbi_uid=1351748&amp;link_uid=PMC2376240">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3205.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4211408/figure/F4/" ref="ordinalpos=3205&amp;ncbi_uid=6822519&amp;link_uid=PMC4211408" href="/pmc/articles/PMC4211408/"><img src="/pmc/articles/instance/4211408/bin/fncel-08-00343-g004.gif" src-large="/pmc/articles/instance/4211408/bin/fncel-08-00343-g004.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4211408/" ref="ordinalpos=3205&amp;ncbi_uid=6822519&amp;link_uid=PMC4211408" image-link="/pmc/articles/PMC4211408/figure/F4/" class="imagepopup">FIGURE 4.  From: Nitric oxide from inflammatory origin impairs neural stem cell proliferation by inhibiting epidermal growth factor receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Nitric oxide from microglia origin decreases the activation of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Decreased phosphorylation of ERK1/2 in iNOS<sup>+/+</sup> mixed culture, but not in iNOS<sup>-/-</sup> mixed culture lysates, following exposure to LPS plus IFN-γ. One-way ANOVA (Bonferroni’s post-test). ***<i>p</i> &lt; 0.001, significantly different from control.</div></p></div><div class="aux"><div class="resc">Bruno P. Carreira, et al. Front Cell Neurosci. 2014;8:343.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3205&amp;ncbi_uid=6822519&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4211408/" ref="ordinalpos=3205&amp;ncbi_uid=6822519&amp;link_uid=PMC4211408">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3206.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3600166/figure/F1/" ref="ordinalpos=3206&amp;ncbi_uid=4717648&amp;link_uid=PMC3600166" href="/pmc/articles/PMC3600166/"><img src="/pmc/articles/instance/3600166/bin/nihms114743f1.gif" src-large="/pmc/articles/instance/3600166/bin/nihms114743f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3600166/" ref="ordinalpos=3206&amp;ncbi_uid=4717648&amp;link_uid=PMC3600166" image-link="/pmc/articles/PMC3600166/figure/F1/" class="imagepopup">Figure 1.  From: Growth Factors and Targeted Therapy for Malignant Glioma Patients: Lessons Learned and the Road Ahead. </a></p><div class="supp"><p class="details"><div>The EGFR/PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The complexity of the <span class="highlight" style="background-color:">signaling</span> cascade is demonstrated by mTOR. Depending on its binding partner, raptor or rictor, the complex can be both a downstream target of, and upstream activator of, Akt.</div></p></div><div class="aux"><div class="resc">Tiffany T. Huang, et al. Neurotherapeutics. 2009 July;6(3):500-512.</div><p class="links"><a class="dblinks" href="/pubmed/19560740" ref="ordinalpos=3206&amp;ncbi_uid=4717648&amp;link_uid=19560740">Citation</a><a class="dblinks" href="/pmc/articles/PMC3600166/" ref="ordinalpos=3206&amp;ncbi_uid=4717648&amp;link_uid=PMC3600166">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3207.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC104410/figure/F8/" ref="ordinalpos=3207&amp;ncbi_uid=123207&amp;link_uid=PMC104410" href="/pmc/articles/PMC104410/"><img src="/pmc/articles/instance/104410/bin/mb0691964008.gif" src-large="/pmc/articles/instance/104410/bin/mb0691964008.jpg" alt="FIG. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC104410/" ref="ordinalpos=3207&amp;ncbi_uid=123207&amp;link_uid=PMC104410" image-link="/pmc/articles/PMC104410/figure/F8/" class="imagepopup">FIG. 8.  From: Protein Kinase B Localization and Activation Differentially Affect S6 Kinase 1 Activity and Eukaryotic Translation Initiation Factor 4E-Binding Protein 1 Phosphorylation. </a></p><div class="supp"><p class="details"><div>Model for differential PKB <span class="highlight" style="background-color:">signaling</span> to S6K1 and 4E-BP1. PI3K <span class="highlight" style="background-color:">signaling</span> to S6K1 is PKB independent, whereas 4E-BP1 and GSK-3 phosphorylation are PKB dependent. However, when PKB is constitutively targeted to the membrane it artificially activates X<sub>m</sub>, a potential mediator of the PI3K-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to S6K1. Asterisk indicates active PKB. m, membrane; c, cytoplasm.</div></p></div><div class="aux"><div class="resc">Almut Dufner, et al. Mol Cell Biol. 1999 June;19(6):4525-4534.</div><p class="links"><a class="dblinks" href="/pubmed/10330191" ref="ordinalpos=3207&amp;ncbi_uid=123207&amp;link_uid=10330191">Citation</a><a class="dblinks" href="/pmc/articles/PMC104410/" ref="ordinalpos=3207&amp;ncbi_uid=123207&amp;link_uid=PMC104410">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3208.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2861735/figure/F2/" ref="ordinalpos=3208&amp;ncbi_uid=2309231&amp;link_uid=PMC2861735" href="/pmc/articles/PMC2861735/"><img src="/pmc/articles/instance/2861735/bin/nihms196043f2.gif" src-large="/pmc/articles/instance/2861735/bin/nihms196043f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2861735/" ref="ordinalpos=3208&amp;ncbi_uid=2309231&amp;link_uid=PMC2861735" image-link="/pmc/articles/PMC2861735/figure/F2/" class="imagepopup">Fig. 2.  From: Planarian Hh <span class="highlight" style="background-color:">signaling</span> regulates regeneration polarity and links Hh <span class="highlight" style="background-color:">pathway</span> evolution to cilia. </a></p><div class="supp"><p class="details"><div>Mis-regulated Hh <span class="highlight" style="background-color:">signaling</span> results in A/P patterning defects. <b>(A–F)</b> Live images of regenerating trunk fragments 14d after amputation. Decreased Hh <span class="highlight" style="background-color:">signaling</span> achieved by <i>hh(RNAi)</i> (A–B”), and increased <span class="highlight" style="background-color:">signaling</span> by <i>ptc(RNAi)</i> (D–F”). Phenotypes are arranged according to severity. (A’–F’). Expression of anterior marker (<i>Smed-sFRP-1;</i> (31, 33)). (A”–F”) Expression of posterior marker (<i>Smed</i>-<i>fz-4;</i> (31)).</div></p></div><div class="aux"><div class="resc">Jochen C. Rink, et al. Science. ;326(5958):1406-1410.</div><p class="links"><a class="dblinks" href="/pubmed/19933103" ref="ordinalpos=3208&amp;ncbi_uid=2309231&amp;link_uid=19933103">Citation</a><a class="dblinks" href="/pmc/articles/PMC2861735/" ref="ordinalpos=3208&amp;ncbi_uid=2309231&amp;link_uid=PMC2861735">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3209.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3486458/figure/F7/" ref="ordinalpos=3209&amp;ncbi_uid=4845493&amp;link_uid=PMC3486458" href="/pmc/articles/PMC3486458/"><img src="/pmc/articles/instance/3486458/bin/zjv9990967550007.gif" src-large="/pmc/articles/instance/3486458/bin/zjv9990967550007.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3486458/" ref="ordinalpos=3209&amp;ncbi_uid=4845493&amp;link_uid=PMC3486458" image-link="/pmc/articles/PMC3486458/figure/F7/" class="imagepopup">Fig 7.  From: Porcine Circovirus Type 2 Induces Autophagy via the AMPK/ERK/TSC2/mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in PK-15 Cells. </a></p><div class="supp"><p class="details"><div>Hypothetical model of AMPK/ERK/TSC2/mTOR <span class="highlight" style="background-color:">signaling</span> in PCV2-induced autophagy in PK-15 cells. PCV2 infection activated AMPK and then activated ERK1/2 and TSC2 and suppressed mTOR <span class="highlight" style="background-color:">signaling</span>, thereby inducing autophagy in PK-15 cells.</div></p></div><div class="aux"><div class="resc">Binglin Zhu, et al. J Virol. 2012 November;86(22):12003-12012.</div><p class="links"><a class="dblinks" href="/pubmed/22915817" ref="ordinalpos=3209&amp;ncbi_uid=4845493&amp;link_uid=22915817">Citation</a><a class="dblinks" href="/pmc/articles/PMC3486458/" ref="ordinalpos=3209&amp;ncbi_uid=4845493&amp;link_uid=PMC3486458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3210.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3943226/figure/fig_001/" ref="ordinalpos=3210&amp;ncbi_uid=5895943&amp;link_uid=PMC3943226" href="/pmc/articles/PMC3943226/"><img src="/pmc/articles/instance/3943226/bin/jrd-59-018-g001.gif" src-large="/pmc/articles/instance/3943226/bin/jrd-59-018-g001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3943226/" ref="ordinalpos=3210&amp;ncbi_uid=5895943&amp;link_uid=PMC3943226" image-link="/pmc/articles/PMC3943226/figure/fig_001/" class="imagepopup">Fig. 1. From: Interrelationships Between Apoptosis and Fertility in Bull Sperm. </a></p><div class="supp"><p class="details"><div> Apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The intrinsic and extrinsic pathways with the molecules that are involved in these pathways are represented. Molecular mechanisms of the BAX and BCL-2 including their interaction with the mitochondria can be also seen. The picture was generated by IPA <span class="highlight" style="background-color:">Pathway</span> Analysis software.</div></p></div><div class="aux"><div class="resc">Sule DOGAN, et al. J Reprod Dev. 2013 February;59(1):18-26.</div><p class="links"><a class="dblinks" href="/pubmed/22986927" ref="ordinalpos=3210&amp;ncbi_uid=5895943&amp;link_uid=22986927">Citation</a><a class="dblinks" href="/pmc/articles/PMC3943226/" ref="ordinalpos=3210&amp;ncbi_uid=5895943&amp;link_uid=PMC3943226">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3211.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3354698/figure/F5/" ref="ordinalpos=3211&amp;ncbi_uid=4010634&amp;link_uid=PMC3354698" href="/pmc/articles/PMC3354698/"><img src="/pmc/articles/instance/3354698/bin/nihms375612f5.gif" src-large="/pmc/articles/instance/3354698/bin/nihms375612f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3354698/" ref="ordinalpos=3211&amp;ncbi_uid=4010634&amp;link_uid=PMC3354698" image-link="/pmc/articles/PMC3354698/figure/F5/" class="imagepopup">Figure 5.  From: Conditionally Replicating Adenoviruses for Cancer Treatment. </a></p><div class="supp"><p class="details"><div>Adenovirus VA RNA inhibition of both the Interferon/PKR <span class="highlight" style="background-color:">pathway</span>. VA RNAs (with imperfect-stemloop secondary structure) inhibit IFN-induced PKR <span class="highlight" style="background-color:">signaling</span> leading to downstream eIF2α phosphorylation and ultimate protein translation inhibition. Ras <span class="highlight" style="background-color:">signaling</span> via MEK and its substrate ERK may also inhibit IFN-induced PKR activation by preventing PKR phosphorylation. Additionally, VA RNAs may also be processed into siRNAs and inhibit the functionality of the host cell’s Dicer/RISC RNAi <span class="highlight" style="background-color:">pathway</span>, thus blocking degradation of short viral dsRNAs as well as the cell’s endogenous miRNA/siRNA synthesis <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Youssef Jounaidi, et al. Curr Cancer Drug Targets. ;7(3):285-301.</div><p class="links"><a class="dblinks" href="/pubmed/17504125" ref="ordinalpos=3211&amp;ncbi_uid=4010634&amp;link_uid=17504125">Citation</a><a class="dblinks" href="/pmc/articles/PMC3354698/" ref="ordinalpos=3211&amp;ncbi_uid=4010634&amp;link_uid=PMC3354698">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3212.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3630829/figure/F7/" ref="ordinalpos=3212&amp;ncbi_uid=6037957&amp;link_uid=PMC3630829" href="/pmc/articles/PMC3630829/"><img src="/pmc/articles/instance/3630829/bin/3135f7.gif" src-large="/pmc/articles/instance/3630829/bin/3135f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3630829/" ref="ordinalpos=3212&amp;ncbi_uid=6037957&amp;link_uid=PMC3630829" image-link="/pmc/articles/PMC3630829/figure/F7/" class="imagepopup">Figure 7.  From: Murine natural killer immunoreceptors use distinct proximal <span class="highlight" style="background-color:">signaling</span> complexes to direct cell function. </a></p><div class="supp"><p class="details"><div><b>SLP-76–dependent <span class="highlight" style="background-color:">signaling</span> pathways in NK cells stimulated through ITAM-bearing receptors.</b> SLP-76 is recruited to the plasma membrane by LAT1/LAT2 indirectly through Gads in the canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In addition, the SH2 domain of SLP-76 is necessary for recruitment via ADAP to the plasma membrane through an unknown mechanism. The strength of downstream signals generated by the LAT family-dependent or ADAP-dependent SLP-76 <span class="highlight" style="background-color:">pathway</span> alone is sufficient for phosphorylation of AKT and ERK and for NK cell proliferation. However, both pathways are necessary for optimal IFN-γ production and cytotoxicity. Although depicted in the figure, note only LAT1, not LAT2, cooperatively binds PLC with SLP-76.</div></p></div><div class="aux"><div class="resc">Rebecca M. May, et al. Blood. 2013 April 18;121(16):3135-3146.</div><p class="links"><a class="dblinks" href="/pubmed/23407547" ref="ordinalpos=3212&amp;ncbi_uid=6037957&amp;link_uid=23407547">Citation</a><a class="dblinks" href="/pmc/articles/PMC3630829/" ref="ordinalpos=3212&amp;ncbi_uid=6037957&amp;link_uid=PMC3630829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3213.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2869518/figure/A002485F3/" ref="ordinalpos=3213&amp;ncbi_uid=2358185&amp;link_uid=PMC2869518" href="/pmc/articles/PMC2869518/"><img src="/pmc/articles/instance/2869518/bin/cshperspect-IMS-002485_F3.gif" src-large="/pmc/articles/instance/2869518/bin/cshperspect-IMS-002485_F3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2869518/" ref="ordinalpos=3213&amp;ncbi_uid=2358185&amp;link_uid=PMC2869518" image-link="/pmc/articles/PMC2869518/figure/A002485F3/" class="imagepopup">Figure 3. From: ITAM-mediated <span class="highlight" style="background-color:">Signaling</span> by the T-Cell Antigen Receptor. </a></p><div class="supp"><p class="details"><div>Proximal <span class="highlight" style="background-color:">signaling</span> events in the TCR-coupled <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Interactions between the TCR and peptide-MHC results in the activation of Lck, a member of the Src family of tyrosine kinases. Lck then phosphorylates the two-tyrosine residues within the ITAMs of the CD3 and ζ chains. A second tyrosine kinase, ZAP-70, is specifically recruited to biphosphorylated ITAMs. Phosphorylation of ZAP-70 by Lck results in its activation. ZAP-70 and Lck phosphorylate and activate several downstream target proteins eventually leading to Ras activation, calcium mobilization, and actin cytoskeleton rearrangements, and ultimately to the activation of transcription factors.</div></p></div><div class="aux"><div class="resc">Paul E. Love, et al. Cold Spring Harb Perspect Biol. 2010 June;2(6):a002485.</div><p class="links"><a class="dblinks" href="/pubmed/20516133" ref="ordinalpos=3213&amp;ncbi_uid=2358185&amp;link_uid=20516133">Citation</a><a class="dblinks" href="/pmc/articles/PMC2869518/" ref="ordinalpos=3213&amp;ncbi_uid=2358185&amp;link_uid=PMC2869518">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3214.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2858875/figure/F3/" ref="ordinalpos=3214&amp;ncbi_uid=2410339&amp;link_uid=PMC2858875" href="/pmc/articles/PMC2858875/"><img src="/pmc/articles/instance/2858875/bin/nihms185641f3.gif" src-large="/pmc/articles/instance/2858875/bin/nihms185641f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2858875/" ref="ordinalpos=3214&amp;ncbi_uid=2410339&amp;link_uid=PMC2858875" image-link="/pmc/articles/PMC2858875/figure/F3/" class="imagepopup">Fig. 3. Effect of selected compounds on THP-1 monocyte <span class="highlight" style="background-color:">signaling</span>.  From: Discovery of dual inhibitors of the immune cell PI3Ks p110? and p110?: a prototype for new anti-inflammatory drugs. </a></p><div class="supp"><p class="details"><div>A. THP-1 monocyte <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to phosphorylation of Akt through either RTK-linked Macrophage Colony Stimulating Factor (M-CSF) or GPCR-linked Monocyte chemoattractant protein 1 (MCP-1).<br />B. pAkt was measured by FACS using fluorescent pSer473 Akt antibodies. Quantitated fluorescence was used to determine an EC<sub>50</sub> for selected compounds. Biochemical γ/δ ratios are provided for comparison.</div></p></div><div class="aux"><div class="resc">Olusegun Williams, et al. Chem Biol. ;17(2):123-134.</div><p class="links"><a class="dblinks" href="/pubmed/20189103" ref="ordinalpos=3214&amp;ncbi_uid=2410339&amp;link_uid=20189103">Citation</a><a class="dblinks" href="/pmc/articles/PMC2858875/" ref="ordinalpos=3214&amp;ncbi_uid=2410339&amp;link_uid=PMC2858875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3215.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2660173/figure/f2/" ref="ordinalpos=3215&amp;ncbi_uid=1729998&amp;link_uid=PMC2660173" href="/pmc/articles/PMC2660173/"><img src="/pmc/articles/instance/2660173/bin/zh30030953170002.gif" src-large="/pmc/articles/instance/2660173/bin/zh30030953170002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2660173/" ref="ordinalpos=3215&amp;ncbi_uid=1729998&amp;link_uid=PMC2660173" image-link="/pmc/articles/PMC2660173/figure/f2/" class="imagepopup">Fig. 2. From: KLF4 gene expression is inhibited by the notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that controls goblet cell differentiation in mouse gastrointestinal tract. </a></p><div class="supp"><p class="details"><div>Inhibition of Notch converted cryptal cells into goblet cells. The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was inhibited by administration of a γ-secretase inhibitor [Compound E (CompE)] for 5 days by daily ip injection (10 μmol·kg<sup>−1</sup>·day<sup>−1</sup>) of 6- to 8-wk-old C57BL/6 mice. Frozen sections from mouse small intestine with (<i>B</i> and <i>D</i>) and without (<i>A</i> and <i>C</i>) treatment of CompE were stained with hematoxylin and eosin (HE; <i>A</i> and <i>B</i>) and Periodic acid-Schiff (PAS; <i>C</i> and <i>D</i>), a staining marker of goblet cells.</div></p></div><div class="aux"><div class="resc">Hai Zheng, et al. Am J Physiol Gastrointest Liver Physiol. 2009 March;296(3):G490-G498.</div><p class="links"><a class="dblinks" href="/pubmed/19109406" ref="ordinalpos=3215&amp;ncbi_uid=1729998&amp;link_uid=19109406">Citation</a><a class="dblinks" href="/pmc/articles/PMC2660173/" ref="ordinalpos=3215&amp;ncbi_uid=1729998&amp;link_uid=PMC2660173">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3216.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2613770/figure/F1/" ref="ordinalpos=3216&amp;ncbi_uid=1661851&amp;link_uid=PMC2613770" href="/pmc/articles/PMC2613770/"><img src="/pmc/articles/instance/2613770/bin/nihms76174f1.gif" src-large="/pmc/articles/instance/2613770/bin/nihms76174f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2613770/" ref="ordinalpos=3216&amp;ncbi_uid=1661851&amp;link_uid=PMC2613770" image-link="/pmc/articles/PMC2613770/figure/F1/" class="imagepopup">Fig. 1.  From: Advances in Targeting IKK and IKK-Related Kinases for Cancer Therapy. </a></p><div class="supp"><p class="details"><div>The canonical and noncanonical NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. The canonical <span class="highlight" style="background-color:">pathway</span> is critical for the activation of innate immunity and inflammation (<i>left</i>). It is activated by TNFα, IL-1, and lipopolysaccharide. Activation of this <span class="highlight" style="background-color:">pathway</span> depends on the IKKα/IKKβ/IKKγ complex, which phosphorylates IκB proteins to trigger their degradation through β-TrCP ubiquitin proteasome. The noncanonical <span class="highlight" style="background-color:">pathway</span> is crucial for the activation of adaptive immunity (<i>right</i>). It is activated by lipopolysaccharide, B cell – activating factor of the TNF family, and latent membrane protein 1. Activation of this <span class="highlight" style="background-color:">pathway</span> depends on IKKα homodimers which phosphorylate p100 to induce p100/RELB heterodimers processing intop52/RELB heterodimers.</div></p></div><div class="aux"><div class="resc">Dung-Fang Lee, et al. Clin Cancer Res. ;14(18):5656-5662.</div><p class="links"><a class="dblinks" href="/pubmed/18794072" ref="ordinalpos=3216&amp;ncbi_uid=1661851&amp;link_uid=18794072">Citation</a><a class="dblinks" href="/pmc/articles/PMC2613770/" ref="ordinalpos=3216&amp;ncbi_uid=1661851&amp;link_uid=PMC2613770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3217.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2787811/figure/F2/" ref="ordinalpos=3217&amp;ncbi_uid=2558910&amp;link_uid=PMC2787811" href="/pmc/articles/PMC2787811/"><img src="/pmc/articles/instance/2787811/bin/nihms154360f2.gif" src-large="/pmc/articles/instance/2787811/bin/nihms154360f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2787811/" ref="ordinalpos=3217&amp;ncbi_uid=2558910&amp;link_uid=PMC2787811" image-link="/pmc/articles/PMC2787811/figure/F2/" class="imagepopup">Fig. 2. Regulation of miRNA maturation by the TGFβ superfamily <span class="highlight" style="background-color:">signaling</span>.  From: Control of microRNA Biogenesis by TGF? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> ----a novel role of Smads in the nucleus. </a></p><div class="supp"><p class="details"><div>TGFβ and BMP <span class="highlight" style="background-color:">signaling</span> stimulates the production of pre-miR-21 by promoting the Drosha-mediated processing by controlling nuclear localization of R-Smad proteins. Thus, Smads regulate gene expression in two distinct manners; (i) transcriptional regulation by DNA binding and (ii) regulation of miRNA maturation by associating with the Drosha/DGCR8 complex.</div></p></div><div class="aux"><div class="resc">Akiko Hata, et al. Cytokine Growth Factor Rev. ;20(5-6):517-521.</div><p class="links"><a class="dblinks" href="/pubmed/19892582" ref="ordinalpos=3217&amp;ncbi_uid=2558910&amp;link_uid=19892582">Citation</a><a class="dblinks" href="/pmc/articles/PMC2787811/" ref="ordinalpos=3217&amp;ncbi_uid=2558910&amp;link_uid=PMC2787811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3218.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204355/figure/F6/" ref="ordinalpos=3218&amp;ncbi_uid=3314094&amp;link_uid=PMC3204355" href="/pmc/articles/PMC3204355/"><img src="/pmc/articles/instance/3204355/bin/nihms108478f6.gif" src-large="/pmc/articles/instance/3204355/bin/nihms108478f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204355/" ref="ordinalpos=3218&amp;ncbi_uid=3314094&amp;link_uid=PMC3204355" image-link="/pmc/articles/PMC3204355/figure/F6/" class="imagepopup">Figure 6.  From: Caspase-, cathepsin-, and PERK-dependent regulation of MDA-7/IL-24-induced cell killing in primary human glioma cells. </a></p><div class="supp"><p class="details"><div>GST-MDA-7 causes cell killing by promoting PERK-dependent vacuolization and JNK <span class="highlight" style="background-color:">pathway</span> activation in transformed cells. <b>A,</b> U251 cells were plated and 12 h after plating infected at a multiplicity of infection of 50 to express no gene or HSP70. Twenty-four hours after infection, cells were treated with GST or GST-MDA-7 (100 nmol/L). Forty-eight hours after GST-MDA-7 exposure, the viability of the U251 cells was determined by trypan blue exclusion assay and by Annexin V/PI staining (±SE; <i>n</i> = 2; #, <i>P</i> &lt; 0.05, less than value in CMV-infected cells). <b>B,</b> treatment of transformed cells with GST-MDA-7 causes PERK-dependent activation of the JNK <span class="highlight" style="background-color:">pathway</span> and PERK-dependent inactivation of the ERK1/2 <span class="highlight" style="background-color:">pathway</span>. JNK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> plays a key role in the activation of caspase-2 and caspase-4 and cathepsin B; cathepsin B promotes the activation of the BH3 domain protein BID. JNK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> also activates BAX. Inactivation of ERK1/2 promotes activation of BAD and BIM. Thus, GST-MDA-7, via PERK <span class="highlight" style="background-color:">signaling</span>, promotes activation of at least four BH3 domain proteins, which cause mitochondrial dysfunction and promote cell killing.</div></p></div><div class="aux"><div class="resc">Adly Yacoub, et al. Mol Cancer Ther. ;7(2):297-313.</div><p class="links"><a class="dblinks" href="/pubmed/18281515" ref="ordinalpos=3218&amp;ncbi_uid=3314094&amp;link_uid=18281515">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204355/" ref="ordinalpos=3218&amp;ncbi_uid=3314094&amp;link_uid=PMC3204355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3219.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3253704/figure/F1/" ref="ordinalpos=3219&amp;ncbi_uid=3746006&amp;link_uid=PMC3253704" href="/pmc/articles/PMC3253704/"><img src="/pmc/articles/instance/3253704/bin/nihms347168f1.gif" src-large="/pmc/articles/instance/3253704/bin/nihms347168f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3253704/" ref="ordinalpos=3219&amp;ncbi_uid=3746006&amp;link_uid=PMC3253704" image-link="/pmc/articles/PMC3253704/figure/F1/" class="imagepopup">Figure 1.  From: Aberrant <span class="highlight" style="background-color:">signaling</span> pathways in pancreatic cancer: A two compartment view. </a></p><div class="supp"><p class="details"><div>Schematic representation of prominent <span class="highlight" style="background-color:">signaling</span> pathways involved with PDAC tumoral compartment and stromal compartment crosstalk. Induction of some of these pathways by RAS activation is a contributing factor. The cells of the tumoral compartment can both respond to TGF-β <span class="highlight" style="background-color:">pathway</span> activation as well as produce ligand that can then act in a paracrine fashion to affect cells of the stromal compartment. Likewise, cells of the tumoral compartment can respond to Hh activation and also produced ligand to affect the surrounding stromal cells. Chemokines and growth factors produced by inflammatory cells, fibroblasts, and PSCs can stimulate NF-κB <span class="highlight" style="background-color:">signaling</span> in cells of both compartments. Likewise, these cells can produce chemokines and growth factors to further stimulate NF-κB <span class="highlight" style="background-color:">signaling</span> in a paracrine manner. This schematic greatly simplifies the complexity of this crosstalk but highlights its importance in function of both cellular compartments.</div></p></div><div class="aux"><div class="resc">Angela L. McCleary-Wheeler, et al. Mol Carcinog. ;51(1):25-39.</div><p class="links"><a class="dblinks" href="/pubmed/22162229" ref="ordinalpos=3219&amp;ncbi_uid=3746006&amp;link_uid=22162229">Citation</a><a class="dblinks" href="/pmc/articles/PMC3253704/" ref="ordinalpos=3219&amp;ncbi_uid=3746006&amp;link_uid=PMC3253704">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3220.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3366938/figure/pone-0038378-g006/" ref="ordinalpos=3220&amp;ncbi_uid=4046921&amp;link_uid=PMC3366938" href="/pmc/articles/PMC3366938/"><img src="/pmc/articles/instance/3366938/bin/pone.0038378.g006.gif" src-large="/pmc/articles/instance/3366938/bin/pone.0038378.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3366938/" ref="ordinalpos=3220&amp;ncbi_uid=4046921&amp;link_uid=PMC3366938" image-link="/pmc/articles/PMC3366938/figure/pone-0038378-g006/" class="imagepopup">Figure 6. The c-Met, PI3K, Akt, and c-Src <span class="highlight" style="background-color:">pathway</span> is involved in HGF-induced AP-1 activation..  From: Hepatocyte Growth Factor Increases Osteopontin Expression in Human Osteoblasts through PI3K, Akt, c-Src, and AP-1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A&amp;B) Primary osteoblasts were transfected with the AP-1-luciferase expression vector and then pretreated with c-Met inhibitor, Ly294002, Akt inhibitor, and PP2 or cotransfected with c-Met and c-Jun siRNA or p85, Akt and c-Src mutant before incubation with HGF for 24 h. Luciferase activity was then assayed (n = 4). *: p<i>&lt;</i>0.05 as compared with HGF-treated group. (C) Schematic diagram of the <span class="highlight" style="background-color:">signaling</span> pathways involved in HGF-induced OPN expression in osteoblasts. HGF increases OPN expression by binding to the c-Met receptor and activating PI3K, Akt, and c-Src, which enhances binding of c-Jun to the AP-1 site. This results in the transactivation of OPN expression.</div></p></div><div class="aux"><div class="resc">Hsien-Te Chen, et al. PLoS One. 2012;7(6):e38378.</div><p class="links"><a class="dblinks" href="/pubmed/22675553" ref="ordinalpos=3220&amp;ncbi_uid=4046921&amp;link_uid=22675553">Citation</a><a class="dblinks" href="/pmc/articles/PMC3366938/" ref="ordinalpos=3220&amp;ncbi_uid=4046921&amp;link_uid=PMC3366938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3221.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3696710/figure/f1/" ref="ordinalpos=3221&amp;ncbi_uid=5063128&amp;link_uid=PMC3696710" href="/pmc/articles/PMC3696710/"><img src="/pmc/articles/instance/3696710/bin/cjcr-25-03-346-f1.gif" src-large="/pmc/articles/instance/3696710/bin/cjcr-25-03-346-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3696710/" ref="ordinalpos=3221&amp;ncbi_uid=5063128&amp;link_uid=PMC3696710" image-link="/pmc/articles/PMC3696710/figure/f1/" class="imagepopup">Figure 1.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and ovarian cancer. </a></p><div class="supp"><p class="details"><div>The constitutive components and the transduced mechanisms of Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A. In the absence of Hh ligand, the latent Gli transcription factors Gli2/3 are bound to a multiprotein complex with the negative regulator Sufu. Phosphorylation of Gli2/3 by PKA, GSK3β and CK1 targets latent Gli proteins to proteasome-dependent repressor formation (Rep-Gli2/Gli3) that cannot activate target gene transcription upon binding to DNA in the nucleus; B. Upon binding of Hh ligand to its receptor Ptch, Ptch translocates out of the primary cilium, losing its ability to inhibit Smo, which moves into the cilium, thus stimulating the <span class="highlight" style="background-color:">pathway</span> and preventing Gli2 and Gli3 cleavage. The activated forms of Gli2 and Gli3 bind to Gli-promoters in the nucleus and stimulate transcription of the ubiquitous mammalian target genes</div></p></div><div class="aux"><div class="resc">Qi Chen, et al. Chin J Cancer Res. 2013 June;25(3):346-353.</div><p class="links"><a class="dblinks" href="/pubmed/23825912" ref="ordinalpos=3221&amp;ncbi_uid=5063128&amp;link_uid=23825912">Citation</a><a class="dblinks" href="/pmc/articles/PMC3696710/" ref="ordinalpos=3221&amp;ncbi_uid=5063128&amp;link_uid=PMC3696710">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3222.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3147112/figure/F4/" ref="ordinalpos=3222&amp;ncbi_uid=3526430&amp;link_uid=PMC3147112" href="/pmc/articles/PMC3147112/"><img src="/pmc/articles/instance/3147112/bin/nihms301266f4.gif" src-large="/pmc/articles/instance/3147112/bin/nihms301266f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3147112/" ref="ordinalpos=3222&amp;ncbi_uid=3526430&amp;link_uid=PMC3147112" image-link="/pmc/articles/PMC3147112/figure/F4/" class="imagepopup">Figure 4. Downregulation of Wnt/ƒβ-catenin <span class="highlight" style="background-color:">signaling</span> components in Wt1 loss of function.  From: WT1 regulates epicardial epithelial to mesenchymal transition through ?-catenin and retinoic acid <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b>A</b>. Wt1<sup>CreERT2/+</sup> Rosa26<sup>mTmG</sup> hearts were pulsed with Tam for 12 hours in vivo, then explanted and cultured in the presence of the indicated inhibitor of canonical Wnt <span class="highlight" style="background-color:">signaling</span>. DKK1 and sFRP2 consistently blocked formation of GFP<sup>+</sup> EPDCs. <b>B</b>. Differential expression of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> components in Wt1<sup>KO</sup> (Wt1<sup>GFPCre/GFPCre</sup>) compared to Ctrl (Wt1<sup>GFPCre/+</sup>) FACS-purified epicardial cells. <i>β-Catenin</i> (<i>Ctnnb1)</i> was measured with unamplified RNA, n=10–12. Other genes were measured with amplified RNA, n=4–5. <b>C</b>. Cryosections of E13.5 heart stained for CTNNB1, CTNNB1 phosphorylated at Ser675 (ph-CTNNB1), or CTNNB1 lacking phorphorylation at Ser37 or Thr41 (unph-CTNNB1). GFP immunostaining marked epicardium and derivatives. In controls, CTNNB1 was expressed in epicardium (arrows) and myocardium. In mutant heart, CTNNB1 was unchanged in myocardium but decreased in epicardium (arrowheads). <b>D</b>. Downstream targets of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> were downregulated in Wt1<sup>KO</sup>, consistent with decreased activity of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Amplified RNA, n=4–5. <b>E</b>. Activity of the Batgal transgene, which expresses LacZ in response to canonical Wnt <span class="highlight" style="background-color:">signaling</span>, in control and Wt1<sup>KO</sup> epicardium. *, P &lt; 0.05. Scale bars: 10 μm (white), 100 μm (yellow).</div></p></div><div class="aux"><div class="resc">Alexander von Gise, et al. Dev Biol. ;356(2):421-431.</div><p class="links"><a class="dblinks" href="/pubmed/21663736" ref="ordinalpos=3222&amp;ncbi_uid=3526430&amp;link_uid=21663736">Citation</a><a class="dblinks" href="/pmc/articles/PMC3147112/" ref="ordinalpos=3222&amp;ncbi_uid=3526430&amp;link_uid=PMC3147112">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3223.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3312242/figure/F1/" ref="ordinalpos=3223&amp;ncbi_uid=4260161&amp;link_uid=PMC3312242" href="/pmc/articles/PMC3312242/"><img src="/pmc/articles/instance/3312242/bin/nihms341854f1.gif" src-large="/pmc/articles/instance/3312242/bin/nihms341854f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3312242/" ref="ordinalpos=3223&amp;ncbi_uid=4260161&amp;link_uid=PMC3312242" image-link="/pmc/articles/PMC3312242/figure/F1/" class="imagepopup">Figure 1.  From: Regulation of Cancer Stem Cells by Cytokine Networks: Attacking Cancers Inflammatory Roots. </a></p><div class="supp"><p class="details"><div>Regulation of cancer stem cells by inflammatory cytokine networks. Binding of IL-6-IL-6R complex to gp130 and IL-8 to CXCR1/2 activates NF-κB <span class="highlight" style="background-color:">pathway</span> via Stat3 and Akt <span class="highlight" style="background-color:">signaling</span>. IL-6 and IL-8 production by NF-κB generates a positive feedback loop which maintains constitutive <span class="highlight" style="background-color:">pathway</span> activation. Inhibition of <span class="highlight" style="background-color:">pathway</span> components by pharmacologic agents is illustrated.</div></p></div><div class="aux"><div class="resc">Hasan Korkaya, et al. Clin Cancer Res. ;17(19):6125-6129.</div><p class="links"><a class="dblinks" href="/pubmed/21685479" ref="ordinalpos=3223&amp;ncbi_uid=4260161&amp;link_uid=21685479">Citation</a><a class="dblinks" href="/pmc/articles/PMC3312242/" ref="ordinalpos=3223&amp;ncbi_uid=4260161&amp;link_uid=PMC3312242">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3224.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2494517/figure/f9/" ref="ordinalpos=3224&amp;ncbi_uid=1437361&amp;link_uid=PMC2494517" href="/pmc/articles/PMC2494517/"><img src="/pmc/articles/instance/2494517/bin/zh20070851630009.gif" src-large="/pmc/articles/instance/2494517/bin/zh20070851630009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2494517/" ref="ordinalpos=3224&amp;ncbi_uid=1437361&amp;link_uid=PMC2494517" image-link="/pmc/articles/PMC2494517/figure/f9/" class="imagepopup">Fig. 9. From: Src regulates cell cycle protein expression and renal epithelial cell proliferation via PI3K/Akt <span class="highlight" style="background-color:">signaling</span>-dependent and -independent mechanisms. </a></p><div class="supp"><p class="details"><div>Schematic illustration of Src-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to RPTC proliferation. Src is activated in response of the bath serum containing a variety of growth factors (GF). Src-dependent activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is required for upregulation of cyclin D1, whereas PI3K/Akt-independent <span class="highlight" style="background-color:">pathway</span> is associated with downregulation of p27 and p57.</div></p></div><div class="aux"><div class="resc">Jingping Xing, et al. Am J Physiol Renal Physiol. 2008 July;295(1):F145-F152.</div><p class="links"><a class="dblinks" href="/pubmed/18434386" ref="ordinalpos=3224&amp;ncbi_uid=1437361&amp;link_uid=18434386">Citation</a><a class="dblinks" href="/pmc/articles/PMC2494517/" ref="ordinalpos=3224&amp;ncbi_uid=1437361&amp;link_uid=PMC2494517">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3225.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3077214/figure/Fig1/" ref="ordinalpos=3225&amp;ncbi_uid=3488746&amp;link_uid=PMC3077214" href="/pmc/articles/PMC3077214/"><img src="/pmc/articles/instance/3077214/bin/12154_2010_51_Fig1_HTML.gif" src-large="/pmc/articles/instance/3077214/bin/12154_2010_51_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3077214/" ref="ordinalpos=3225&amp;ncbi_uid=3488746&amp;link_uid=PMC3077214" image-link="/pmc/articles/PMC3077214/figure/Fig1/" class="imagepopup">Fig. 1.  From: In vivo analysis of compound activity and mechanism of action using epistasis in Drosophila. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> components used to test gene–compound interactions. Key components of Hh (<b>a</b>), insulin/PI3K (<b>b</b>), Ras/MAPK (<b>c</b>), JNK (<b>d</b>), Wnt (<b>e</b>), cell cycle (<b>f</b>), and apoptosis (<b>g</b>) are depicted. <i>Arrows</i> indicate activation and <i>T bars</i> represent inhibition. For simplicity, only the core components of each <span class="highlight" style="background-color:">pathway</span> are shown. <span class="highlight" style="background-color:">Pathway</span> components used for gene–compound interaction experiments are in <i>blue</i> and compounds are in <i>red</i></div></p></div><div class="aux"><div class="resc">Erdem Bangi, et al. J Chem Biol. 2011 April;4(2):55-68.</div><p class="links"><a class="dblinks" href="/pubmed/22190992" ref="ordinalpos=3225&amp;ncbi_uid=3488746&amp;link_uid=22190992">Citation</a><a class="dblinks" href="/pmc/articles/PMC3077214/" ref="ordinalpos=3225&amp;ncbi_uid=3488746&amp;link_uid=PMC3077214">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3226.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3741135/figure/pone-0071576-g009/" ref="ordinalpos=3226&amp;ncbi_uid=5199246&amp;link_uid=PMC3741135" href="/pmc/articles/PMC3741135/"><img src="/pmc/articles/instance/3741135/bin/pone.0071576.g009.gif" src-large="/pmc/articles/instance/3741135/bin/pone.0071576.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3741135/" ref="ordinalpos=3226&amp;ncbi_uid=5199246&amp;link_uid=PMC3741135" image-link="/pmc/articles/PMC3741135/figure/pone-0071576-g009/" class="imagepopup">Figure 9. Model for the adipogenic regulation of Akt2 and Sirt1 by C/EBPα and PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Sirt1 Inhibits Akt2-Mediated Porcine Adipogenesis Potentially by Direct Protein-Protein Interaction. </a></p><div class="aux"><div class="resc">Weijun Pang, et al. PLoS One. 2013;8(8):e71576.</div><p class="links"><a class="dblinks" href="/pubmed/23951196" ref="ordinalpos=3226&amp;ncbi_uid=5199246&amp;link_uid=23951196">Citation</a><a class="dblinks" href="/pmc/articles/PMC3741135/" ref="ordinalpos=3226&amp;ncbi_uid=5199246&amp;link_uid=PMC3741135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3227.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3643961/figure/pone-0062652-g004/" ref="ordinalpos=3227&amp;ncbi_uid=4891284&amp;link_uid=PMC3643961" href="/pmc/articles/PMC3643961/"><img src="/pmc/articles/instance/3643961/bin/pone.0062652.g004.gif" src-large="/pmc/articles/instance/3643961/bin/pone.0062652.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3643961/" ref="ordinalpos=3227&amp;ncbi_uid=4891284&amp;link_uid=PMC3643961" image-link="/pmc/articles/PMC3643961/figure/pone-0062652-g004/" class="imagepopup">Figure 4. The effects of PTE on proteins in the Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the mitochondrial apoptotic <span class="highlight" style="background-color:">pathway</span> in lung adenocarcinoma cells treated for 24 h..  From: Pterostilbene Exerts Antitumor Activity via the Notch1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human Lung Adenocarcinoma Cells. </a></p><div class="supp"><p class="details"><div>(A) Representative Western blot results are shown. The PTE treatment increased the expression levels of NICD, Hes1, Nicastrin, Presenilin-1, Bax and Cytochrome c. (B) Representative images of Notch1 immunofluorescence are shown (×400). (C) Representative images of Hes1 immunofluorescence are shown (×400). The results are expressed as the mean ± SEM, n = 6. **P&lt;0.01 compared with the control group, <sup>##</sup>P&lt;0.01 compared with the PTE 1.5 µM group, <sup>$$</sup>P&lt;0.01 compared with the PTE 3 µM group. PTE, pterostilbene.</div></p></div><div class="aux"><div class="resc">Yang Yang, et al. PLoS One. 2013;8(5):e62652.</div><p class="links"><a class="dblinks" href="/pubmed/23671619" ref="ordinalpos=3227&amp;ncbi_uid=4891284&amp;link_uid=23671619">Citation</a><a class="dblinks" href="/pmc/articles/PMC3643961/" ref="ordinalpos=3227&amp;ncbi_uid=4891284&amp;link_uid=PMC3643961">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3228.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4067306/figure/pone-0100295-g005/" ref="ordinalpos=3228&amp;ncbi_uid=6290207&amp;link_uid=PMC4067306" href="/pmc/articles/PMC4067306/"><img src="/pmc/articles/instance/4067306/bin/pone.0100295.g005.gif" src-large="/pmc/articles/instance/4067306/bin/pone.0100295.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4067306/" ref="ordinalpos=3228&amp;ncbi_uid=6290207&amp;link_uid=PMC4067306" image-link="/pmc/articles/PMC4067306/figure/pone-0100295-g005/" class="imagepopup">Figure 5. Most discriminative <span class="highlight" style="background-color:">signaling</span> pathways distinguishing different types of gliomas..  From: Autoregressive Higher-Order Hidden Markov Models: Exploiting Local Chromosomal Dependencies in the Analysis of Tumor Expression Profiles. </a></p><div class="supp"><p class="details"><div>Overview of known cancer <span class="highlight" style="background-color:">signaling</span> pathways identified to show largely distinct patterns of overexpressed genes in oligodendrogliomas, astrocytomas and glioblastomas based on predictions of the AR(<img src="/pmc/articles/instance/4067306/bin/pone.0100295.e234.jpg" class="inline-graphic" style="border-style: none;" />)-HMM(<img src="/pmc/articles/instance/4067306/bin/pone.0100295.e235.jpg" class="inline-graphic" style="border-style: none;" />). The overlap of the top-ranking overexpressed genes with the specific <span class="highlight" style="background-color:">signaling</span> pathways is quantified from top 100 to top 600. Grey curves show random expectations with respect to the number of genes in a specific <span class="highlight" style="background-color:">pathway</span>. Robust systematic differences between the different types of gliomas are clearly visible.</div></p></div><div class="aux"><div class="resc">Michael Seifert, et al. PLoS One. 2014;9(6):e100295.</div><p class="links"><a class="dblinks" href="/pubmed/24955771" ref="ordinalpos=3228&amp;ncbi_uid=6290207&amp;link_uid=24955771">Citation</a><a class="dblinks" href="/pmc/articles/PMC4067306/" ref="ordinalpos=3228&amp;ncbi_uid=6290207&amp;link_uid=PMC4067306">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3229.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1409734/figure/f7/" ref="ordinalpos=3229&amp;ncbi_uid=701868&amp;link_uid=PMC1409734" href="/pmc/articles/PMC1409734/"><img src="/pmc/articles/instance/1409734/bin/7601016f7.gif" src-large="/pmc/articles/instance/1409734/bin/7601016f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1409734/" ref="ordinalpos=3229&amp;ncbi_uid=701868&amp;link_uid=PMC1409734" image-link="/pmc/articles/PMC1409734/figure/f7/" class="imagepopup">Figure 7.  From: The role of receptor internalization in CD95 <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>CD95L-mediated internalization-independent <span class="highlight" style="background-color:">signaling</span>. (<b>A</b>) Activation of Erk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cells transfected with AP-2(α+μ<sub>2</sub>) or CHC siRNAs. BJAB cells, transfected with control, AP-2(α+μ<sub>2</sub>) or CHC siRNAs, were incubated with Flag-CD95L for the specified times or anti-IgM Fab<sub>2</sub>′ for 5 min. Cell lysates were analyzed for phospho-Erk1/2 (top) or total Erk1/2 (bottom) expression. Data shown are representative of two experiments. (<b>B</b>) Activation of NF-κB transcription in FynC-GFP-INP54p<sup>+</sup> cells. BJAB cells transfected with FynC-GFP-INP54p were analyzed for transcriptional activation of NF-κB following Flag-CD95L, PMA or BCR stimulation. These data are representative of two experiments. (<b>C</b>) Activation of Erk <span class="highlight" style="background-color:">signaling</span> pathways by CD95(Y291F). A20 cells, transfected with wt CD95 or CD95(Y291F), were incubated with anti-hCD95 mAb (CH11) for the specified times. Cell lysates were analyzed for phospho-Erk1/2 (top) or total Erk1/2 (bottom) expression. Cells were also treated with PMA as a positive control.</div></p></div><div class="aux"><div class="resc">Kyeong-Hee Lee, et al. EMBO J. 2006 March 8;25(5):1009-1023.</div><p class="links"><a class="dblinks" href="/pubmed/16498403" ref="ordinalpos=3229&amp;ncbi_uid=701868&amp;link_uid=16498403">Citation</a><a class="dblinks" href="/pmc/articles/PMC1409734/" ref="ordinalpos=3229&amp;ncbi_uid=701868&amp;link_uid=PMC1409734">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3230.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1253564/figure/fig8/" ref="ordinalpos=3230&amp;ncbi_uid=673307&amp;link_uid=PMC1253564" href="/pmc/articles/PMC1253564/"><img src="/pmc/articles/instance/1253564/bin/zpq0410597730008.gif" src-large="/pmc/articles/instance/1253564/bin/zpq0410597730008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1253564/" ref="ordinalpos=3230&amp;ncbi_uid=673307&amp;link_uid=PMC1253564" image-link="/pmc/articles/PMC1253564/figure/fig8/" class="imagepopup">Fig. 8. From: Mechanism of radiation-induced bystander effect: Role of the cyclooxygenase-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Working model of the <span class="highlight" style="background-color:">signaling</span> pathways involved in the radiation-induced bystander effect. Binding of TNF-α, TGFβ, IGF, IL-1, and IL-8 ligands to the corresponding receptors activate <span class="highlight" style="background-color:">signaling</span> pathways including MEK1/2; mitogen-activated protein kinase kinase (MKK3/6), and p38 kinase (all in blue lines), which lead to COX-2 gene expression. Specific inhibitors of the various <span class="highlight" style="background-color:">signaling</span> pathways, such as Wortmannin (wt) for phosphatidylinositol 3-kinase and <i>N</i>-(α)-tosyl-<span style="font-variant: small-caps" class="small-caps">l</span>-lysine chloromethylketone hydrochloride for the NF-κB can be used to delineate specific <span class="highlight" style="background-color:">signaling</span> routes involved in the bystander phenomenon.</div></p></div><div class="aux"><div class="resc">Hongning Zhou, et al. Proc Natl Acad Sci U S A. 2005 October 11;102(41):14641-14646.</div><p class="links"><a class="dblinks" href="/pubmed/16203985" ref="ordinalpos=3230&amp;ncbi_uid=673307&amp;link_uid=16203985">Citation</a><a class="dblinks" href="/pmc/articles/PMC1253564/" ref="ordinalpos=3230&amp;ncbi_uid=673307&amp;link_uid=PMC1253564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3231.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2881284/figure/F1/" ref="ordinalpos=3231&amp;ncbi_uid=2436121&amp;link_uid=PMC2881284" href="/pmc/articles/PMC2881284/"><img src="/pmc/articles/instance/2881284/bin/psb0503_0300_fig001.gif" src-large="/pmc/articles/instance/2881284/bin/psb0503_0300_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2881284/" ref="ordinalpos=3231&amp;ncbi_uid=2436121&amp;link_uid=PMC2881284" image-link="/pmc/articles/PMC2881284/figure/F1/" class="imagepopup">Figure 1.  From: Multiple receptor complexes assembled for transmitting CLV3 <span class="highlight" style="background-color:">signaling</span> in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Hypothetic <i>CLV3</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> model. As it is shown, CLV3 might be transmitted via at least three receptor complexes. The first one is that CLV1 monomer or CLV1 homodimers. The second one is that CLV2-CRN heterodimers forming a tetramer through CRN-CRN interaction. In addition, CLV1, CLV2 and CRN may form complexes as the third and functional receptor complex.</div></p></div><div class="aux"><div class="resc">Yingfang Zhu, et al. Plant Signal Behav. 2010 March;5(3):300-302.</div><p class="links"><a class="dblinks" href="/pubmed/20220313" ref="ordinalpos=3231&amp;ncbi_uid=2436121&amp;link_uid=20220313">Citation</a><a class="dblinks" href="/pmc/articles/PMC2881284/" ref="ordinalpos=3231&amp;ncbi_uid=2436121&amp;link_uid=PMC2881284">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3232.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC97273/figure/F6/" ref="ordinalpos=3232&amp;ncbi_uid=17197&amp;link_uid=PMC97273" href="/pmc/articles/PMC97273/"><img src="/pmc/articles/instance/97273/bin/ii0300898006.gif" src-large="/pmc/articles/instance/97273/bin/ii0300898006.jpg" alt="FIG. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC97273/" ref="ordinalpos=3232&amp;ncbi_uid=17197&amp;link_uid=PMC97273" image-link="/pmc/articles/PMC97273/figure/F6/" class="imagepopup">FIG. 6.  From: Synergistic Epithelial Responses to Endotoxin and a Naturally Occurring Muramyl Peptide. </a></p><div class="supp"><p class="details"><div>Model of TCT and endotoxin <span class="highlight" style="background-color:">signaling</span>. TCT and LPS each act through cell surface receptors to induce differential <span class="highlight" style="background-color:">signaling</span> events, which combine synergistically to activate a unique H7-sensitive kinase. Direct activation of PKC by PMA can produce a signal that mimics TCT, in that it can synergize with LPS in the same manner as does TCT. However, TCT and LPS <span class="highlight" style="background-color:">signaling</span> pathways do not proceed via PKC, since the PKC-specific inhibitor calphostin C does not block effects of TCT and LPS but does block effects of PMA and LPS. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induces IL-1α, causing induction of iNOS and the production of NO·, which leads to cytostasis. Note that while the PMA-stimulated <span class="highlight" style="background-color:">pathway</span> may not proceed through the same kinase as TCT and LPS, the depiction is consistent with our data.</div></p></div><div class="aux"><div class="resc">Tod A. Flak, et al. Infect Immun. 2000 March;68(3):1235-1242.</div><p class="links"><a class="dblinks" href="/pubmed/10678932" ref="ordinalpos=3232&amp;ncbi_uid=17197&amp;link_uid=10678932">Citation</a><a class="dblinks" href="/pmc/articles/PMC97273/" ref="ordinalpos=3232&amp;ncbi_uid=17197&amp;link_uid=PMC97273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3233.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1978100/figure/F1/" ref="ordinalpos=3233&amp;ncbi_uid=1125862&amp;link_uid=PMC1978100" href="/pmc/articles/PMC1978100/"><img src="/pmc/articles/instance/1978100/bin/nihms23071f1.gif" src-large="/pmc/articles/instance/1978100/bin/nihms23071f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1978100/" ref="ordinalpos=3233&amp;ncbi_uid=1125862&amp;link_uid=PMC1978100" image-link="/pmc/articles/PMC1978100/figure/F1/" class="imagepopup">Figure 1. ERK <span class="highlight" style="background-color:">signaling</span> affects c-Jun expression.  From: Re-wired ERK-JNK <span class="highlight" style="background-color:">signaling</span> pathways in melanoma. </a></p><div class="supp"><p class="details"><div><b>A.</b><i> Levels of P-ERK, P-JNK and proteins relevant to the PKC/JNK <span class="highlight" style="background-color:">pathway</span> in melanoma cell lines.</i> Protein extracts (40μg) from melanoma cell lines were analyzed by Western blots using the indicated antibodies. Human melanocytes (HuMel) were used as a control. α-Tubulin reveals equal loading. <br /><b>B.</b><i> Inhibition of B-Raf reduces c-Jun levels.</i> Lu1205 cells were treated with 300nM CHR-265 for 6h and 24h. Protein samples were analyzed by Western blots using the indicated antibodies. α-Tubulin reveals equal loading. <br /><b>C.</b><i> Inhibition of the MEK/ERK <span class="highlight" style="background-color:">pathway</span> reduces c-Jun levels.</i> A375 cells were treated with 50μM PD98059 and 20μM U0126 for 24h. Protein samples were analyzed by Western blots using the indicated antibodies.</div></p></div><div class="aux"><div class="resc">Pablo Lopez-Bergami, et al. Cancer Cell. ;11(5):447-460.</div><p class="links"><a class="dblinks" href="/pubmed/17482134" ref="ordinalpos=3233&amp;ncbi_uid=1125862&amp;link_uid=17482134">Citation</a><a class="dblinks" href="/pmc/articles/PMC1978100/" ref="ordinalpos=3233&amp;ncbi_uid=1125862&amp;link_uid=PMC1978100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3234.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3599046/figure/F2/" ref="ordinalpos=3234&amp;ncbi_uid=4745758&amp;link_uid=PMC3599046" href="/pmc/articles/PMC3599046/"><img src="/pmc/articles/instance/3599046/bin/1471-2164-14-64-2.gif" src-large="/pmc/articles/instance/3599046/bin/1471-2164-14-64-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3599046/" ref="ordinalpos=3234&amp;ncbi_uid=4745758&amp;link_uid=PMC3599046" image-link="/pmc/articles/PMC3599046/figure/F2/" class="imagepopup">Figure 2.  From: Comparison and contrast of genes and biological pathways responding to Marek's disease virus infection using allele-specific expression and differential expression in broiler and layer chickens. </a></p><div class="supp"><p class="details"><div><b>Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in response to MDV infection.</b> The Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is displayed and genes that exhibit either differential expression (DE) or allele-specific expression (ASE) following Marek’s disease virus (MDV) infection in both broilers and layers are shown.</div></p></div><div class="aux"><div class="resc">Sudeep Perumbakkam, et al. BMC Genomics. 2013;14:64-64.</div><p class="links"><a class="dblinks" href="/pubmed/23363372" ref="ordinalpos=3234&amp;ncbi_uid=4745758&amp;link_uid=23363372">Citation</a><a class="dblinks" href="/pmc/articles/PMC3599046/" ref="ordinalpos=3234&amp;ncbi_uid=4745758&amp;link_uid=PMC3599046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3235.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3875925/figure/nutrients-05-04978-f002/" ref="ordinalpos=3235&amp;ncbi_uid=5642737&amp;link_uid=PMC3875925" href="/pmc/articles/PMC3875925/"><img src="/pmc/articles/instance/3875925/bin/nutrients-05-04978-g002.gif" src-large="/pmc/articles/instance/3875925/bin/nutrients-05-04978-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3875925/" ref="ordinalpos=3235&amp;ncbi_uid=5642737&amp;link_uid=PMC3875925" image-link="/pmc/articles/PMC3875925/figure/nutrients-05-04978-f002/" class="imagepopup">Figure 2.  From: Cancer Preventive Efficacy of Marine Carotenoid Fucoxanthin: Cell Cycle Arrest and Apoptosis. </a></p><div class="supp"><p class="details"><div>Fucoxanthin and the apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Thamaraiselvan Rengarajan, et al. Nutrients. 2013 December;5(12):4978-4989.</div><p class="links"><a class="dblinks" href="/pubmed/24322524" ref="ordinalpos=3235&amp;ncbi_uid=5642737&amp;link_uid=24322524">Citation</a><a class="dblinks" href="/pmc/articles/PMC3875925/" ref="ordinalpos=3235&amp;ncbi_uid=5642737&amp;link_uid=PMC3875925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3236.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC117989/figure/F12/" ref="ordinalpos=3236&amp;ncbi_uid=325858&amp;link_uid=PMC117989" href="/pmc/articles/PMC117989/"><img src="/pmc/articles/instance/117989/bin/pp0916802012.gif" src-large="/pmc/articles/instance/117989/bin/pp0916802012.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC117989/" ref="ordinalpos=3236&amp;ncbi_uid=325858&amp;link_uid=PMC117989" image-link="/pmc/articles/PMC117989/figure/F12/" class="imagepopup">Figure 12.  From: Salicylic Acid and Ethylene Pathways Are Differentially Activated  in Melon Cotyledons by Active or Heat-Denatured Cellulase from  Trichoderma longibrachiatum. </a></p><div class="supp"><p class="details"><div>Cellulase-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in  melon cotyledons.</div></p></div><div class="aux"><div class="resc">Christelle Martinez, et al. Plant Physiol. 2001 September;127(1):334-344.</div><p class="links"><a class="dblinks" href="/pubmed/11553761" ref="ordinalpos=3236&amp;ncbi_uid=325858&amp;link_uid=11553761">Citation</a><a class="dblinks" href="/pmc/articles/PMC117989/" ref="ordinalpos=3236&amp;ncbi_uid=325858&amp;link_uid=PMC117989">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3237.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2873688/figure/F3/" ref="ordinalpos=3237&amp;ncbi_uid=2381390&amp;link_uid=PMC2873688" href="/pmc/articles/PMC2873688/"><img src="/pmc/articles/instance/2873688/bin/nihms201344f3.gif" src-large="/pmc/articles/instance/2873688/bin/nihms201344f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2873688/" ref="ordinalpos=3237&amp;ncbi_uid=2381390&amp;link_uid=PMC2873688" image-link="/pmc/articles/PMC2873688/figure/F3/" class="imagepopup">Fig. 3.  From: Insulin/IGF-1 and ROS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> cross-talk in aging and longevity determination. </a></p><div class="supp"><p class="details"><div>The regulation of insulin <span class="highlight" style="background-color:">signaling</span> by the ratio of the positive mediator, p85–p110 and the negative regulator, unbound p85. This model proposes that: (A) a given ratio between p85–p110 and monomeric p85 defines the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activity and the state of insulin sensitivity in the wild-type background. (B) In p85α<sup>(−/−)</sup> and p85β<sup>(−/−)</sup> mice, free p85 is preferentially decreased and the balance between p85–p110 and free p85 is shifted toward the positive mediator, p85–p110. This causes increased insulin sensitivity in the mutant mice. (C) In contrast, in p110α <sup>(+/−)</sup> and p110β<sup>(+/−)</sup> mice, the free p85 pool level is preferentially decreased (over free p85). This shifts the balance toward the higher levels of free p85, negative regulator, which inactivates IRS-1 and IRS-2 (free p85), resulting in decreased insulin <span class="highlight" style="background-color:">signaling</span> and insulin resistance. (D) Over expression of p85 causes an increase in the p85 pool which shifts the balance toward the negative regulator, resulting in decreased insulin sensitivity (This figure is a modification from Brachmann et al., 2005.)</div></p></div><div class="aux"><div class="resc">John Papaconstantinou. Mol Cell Endocrinol. ;299(1):89-100.</div><p class="links"><a class="dblinks" href="/pubmed/19103250" ref="ordinalpos=3237&amp;ncbi_uid=2381390&amp;link_uid=19103250">Citation</a><a class="dblinks" href="/pmc/articles/PMC2873688/" ref="ordinalpos=3237&amp;ncbi_uid=2381390&amp;link_uid=PMC2873688">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3238.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3404158/figure/F2/" ref="ordinalpos=3238&amp;ncbi_uid=5427781&amp;link_uid=PMC3404158" href="/pmc/articles/PMC3404158/"><img src="/pmc/articles/instance/3404158/bin/nihms369788f2a.gif" src-large="/pmc/articles/instance/3404158/bin/nihms369788f2a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3404158/figure/F2/" ref="ordinalpos=3238&amp;ncbi_uid=5427781&amp;link_uid=PMC3404158" href="/pmc/articles/PMC3404158/"><img src="/pmc/articles/instance/3404158/bin/nihms369788f2b.gif" src-large="/pmc/articles/instance/3404158/bin/nihms369788f2b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404158/" ref="ordinalpos=3238&amp;ncbi_uid=5427781&amp;link_uid=PMC3404158" image-link="/pmc/articles/PMC3404158/figure/F2/" class="imagepopup">Figure 2. Activation and downregulation of NOD2 <span class="highlight" style="background-color:">signaling</span>.  From: Ubiquitination and Phosphorylation in the Regulation of NOD2 <span class="highlight" style="background-color:">signaling</span> and NOD2-mediated Disease. </a></p><div class="supp"><p class="details"><div>A) Exposure to MDP liberates NOD2 from its autoinhibited state allowing it to bind and activate the kinase RIP2. RIP2 undergoes tyrosine autophosphorylation and is ubiquitinated by a number of E3 ligases. RIP2 also promotes ubiquitination of NEMO/IKK γand by doing so links together proximal and distal <span class="highlight" style="background-color:">signaling</span> components of the <span class="highlight" style="background-color:">pathway</span>. The TAB proteins of the TAK1/TAB2/TAB3 complex bind these ubiquitin chains allowing TAK1 to phosphorylate and activate the IKK complex. IKK activation leads to IκBα phosphorylation, marking IκB for degradation and liberating NF-κB for translocation to the nucleus. B) TRAF4 inhibits NOD2 <span class="highlight" style="background-color:">signaling</span> in a manner dependent on NOD2 binding and IKKα-mediated phosphorylation. The downstream events mediating such inhibition are unknown. A20 is phosphorylated and activated by IKKβ and together with ITCH, targets activated, tyrosine phosphorylated RIP2 for degradation. In a manner independent of its ubiquitin editing function, A20 also inhibits TAK1-mediated phosphorylation of IKK.</div></p></div><div class="aux"><div class="resc">Justine T. Tigno-Aranjuez, et al. Biochim Biophys Acta. ;1823(11):2022-2028.</div><p class="links"><a class="dblinks" href="/pubmed/22522061" ref="ordinalpos=3238&amp;ncbi_uid=5427781&amp;link_uid=22522061">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404158/" ref="ordinalpos=3238&amp;ncbi_uid=5427781&amp;link_uid=PMC3404158">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3239.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3213329/figure/F3/" ref="ordinalpos=3239&amp;ncbi_uid=3543273&amp;link_uid=PMC3213329" href="/pmc/articles/PMC3213329/"><img src="/pmc/articles/instance/3213329/bin/nihms-315343-f0003.gif" src-large="/pmc/articles/instance/3213329/bin/nihms-315343-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3213329/" ref="ordinalpos=3239&amp;ncbi_uid=3543273&amp;link_uid=PMC3213329" image-link="/pmc/articles/PMC3213329/figure/F3/" class="imagepopup">Figure 3.  From: Angiotensin II stimulates in vitro branching morphogenesis of the isolated ureteric bud. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis of phosphatidylinositol 3-kinase (PI3K) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> affected by angiotensin II. Pink color indicates upregulated genes; green color indicates downregulated genes.</div></p></div><div class="aux"><div class="resc">Renfang Song, et al. Mech Dev. ;128(7-10):359-367.</div><p class="links"><a class="dblinks" href="/pubmed/21820050" ref="ordinalpos=3239&amp;ncbi_uid=3543273&amp;link_uid=21820050">Citation</a><a class="dblinks" href="/pmc/articles/PMC3213329/" ref="ordinalpos=3239&amp;ncbi_uid=3543273&amp;link_uid=PMC3213329">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3240.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2873859/figure/F5/" ref="ordinalpos=3240&amp;ncbi_uid=2739457&amp;link_uid=PMC2873859" href="/pmc/articles/PMC2873859/"><img src="/pmc/articles/instance/2873859/bin/nihms165015f5.gif" src-large="/pmc/articles/instance/2873859/bin/nihms165015f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2873859/" ref="ordinalpos=3240&amp;ncbi_uid=2739457&amp;link_uid=PMC2873859" image-link="/pmc/articles/PMC2873859/figure/F5/" class="imagepopup">Fig. 5.  From: A cytoplasmic prolyl hydroxylation and glycosylation <span class="highlight" style="background-color:">pathway</span> modifies Skp1 and regulates O2-dependent development in Dictyostelium. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> model. A proposed scheme for O<sub>2</sub>-<span class="highlight" style="background-color:">signaling</span> via the Skp1 modification enzymes, Skp1 (as a subunit of E3(SCF)Ub-ligases), a hypothetical activator, and protein kinase A (PKA). See text for explanation. The dependent sequence of posttranslational modifications are shown vertically at the left, and the dependent sequence of developmental transitions is shown vertically on the right. Speculative aspects of the model, such as regulation of glycosyltransferases by Fuc and Gal (at the left), are denoted in pink. High O<sub>2</sub> can override the <span class="highlight" style="background-color:">pathway</span>, as shown at the right.</div></p></div><div class="aux"><div class="resc">Christopher M. West, et al. Biochim Biophys Acta. ;1800(2):160-171.</div><p class="links"><a class="dblinks" href="/pubmed/19914348" ref="ordinalpos=3240&amp;ncbi_uid=2739457&amp;link_uid=19914348">Citation</a><a class="dblinks" href="/pmc/articles/PMC2873859/" ref="ordinalpos=3240&amp;ncbi_uid=2739457&amp;link_uid=PMC2873859">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3241.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3715701/figure/f0005/" ref="ordinalpos=3241&amp;ncbi_uid=5111556&amp;link_uid=PMC3715701" href="/pmc/articles/PMC3715701/"><img src="/pmc/articles/instance/3715701/bin/gr1.gif" src-large="/pmc/articles/instance/3715701/bin/gr1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3715701/" ref="ordinalpos=3241&amp;ncbi_uid=5111556&amp;link_uid=PMC3715701" image-link="/pmc/articles/PMC3715701/figure/f0005/" class="imagepopup">Fig. 1.  From: Functional roles for myosin 1c in cellular <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Myosin 1c in cellular <span class="highlight" style="background-color:">signaling</span> pathways. This figure provides an overview of the cellular <span class="highlight" style="background-color:">signaling</span> pathways involving myosin 1c function. In particular, myosin 1c has been shown to play a role in (1) the integrin-mediated <span class="highlight" style="background-color:">signaling</span> pathways underlying cell migration, (2) the <span class="highlight" style="background-color:">signaling</span> events facilitating pathogen uptake by macropinocytosis, (3) the Neph1 translocation and <span class="highlight" style="background-color:">signaling</span> mediating podocyte foot formation, (4) the tumor necrosis factor-alpha (TNF-α)-induced insulin resistance pathways, (5) insulin-stimulated GLUT4 transport to the plasma membrane, and (6) mechanical signal transduction in the hair cells of the inner ear. The molecular role of myosin 1c in each <span class="highlight" style="background-color:">pathway</span> is outlined and additional key <span class="highlight" style="background-color:">signaling</span> players are displayed for reference. This presentation of the diverse assembly of <span class="highlight" style="background-color:">signaling</span> pathways involving myosin 1c function highlights the importance of this motor in cellular <span class="highlight" style="background-color:">signaling</span>. Abbreviations: FAK (focal adhesion kinase), PAK1 (p21-activated kinase 1), CtBP1 (C-terminal binding protein 1), Nck (non-catalytic region of tyrosine kinase adaptor protein 1), Grb2 (growth factor receptor-bound protein 2), IKK (IκB kinase), IRS-1 (insulin receptor substrate 1), CaMKII (Ca<sup>2 +</sup>/calmodulin-dependent protein kinase II), Ca<sup>2 +</sup> (Calcium), ATP (adenosine triphosphate), and PIP<sub>2</sub> (phosphatidylinositol 4,5-bisphosphate).</div></p></div><div class="aux"><div class="resc">Lisa M. Bond, et al. Cell Signal. 2013 January;25(1):229-235.</div><p class="links"><a class="dblinks" href="/pubmed/23022959" ref="ordinalpos=3241&amp;ncbi_uid=5111556&amp;link_uid=23022959">Citation</a><a class="dblinks" href="/pmc/articles/PMC3715701/" ref="ordinalpos=3241&amp;ncbi_uid=5111556&amp;link_uid=PMC3715701">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3242.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2603135/figure/F6/" ref="ordinalpos=3242&amp;ncbi_uid=1582208&amp;link_uid=PMC2603135" href="/pmc/articles/PMC2603135/"><img src="/pmc/articles/instance/2603135/bin/nihms57516f6.gif" src-large="/pmc/articles/instance/2603135/bin/nihms57516f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2603135/" ref="ordinalpos=3242&amp;ncbi_uid=1582208&amp;link_uid=PMC2603135" image-link="/pmc/articles/PMC2603135/figure/F6/" class="imagepopup">Figure 6. RRM2 is involved in radiation-induced ATM-p53-mediated DNA repair <span class="highlight" style="background-color:">pathway</span>.  From: Integrated Bioinformatics for Radiation-Induced <span class="highlight" style="background-color:">Pathway</span> Analysis from Proteomics and Microarray Data. </a></p><div class="supp"><p class="details"><div>The ATM <span class="highlight" style="background-color:">pathway</span> map on the left was based on http://www.cs.tau.ac.il/~spike/images/1.png. Shown in the map are two proteins, p53 (TP53) and BRCA1 (highlighted in red boxes), directly interact with ATM (green box) as its downstream <span class="highlight" style="background-color:">signaling</span> proteins. The right diagram depicts the ATM-p53 mediated and radiation-induced DNA repair <span class="highlight" style="background-color:">pathway</span> involving RRM2. The RRM2’s connections to p53 and HDAC1 were derived from the network analysis and were consistent with the expression data in this study. Both BRCA1 and HDAC1 are known to be involved in DNA repair. The interactions between p53 and RRM2 proteins and between HDAC1 and RRM2 gene promoter have been reported in literature. The red colored proteins are observed as differentially expressed in response to radiation from this study. Solid arrows indicate direct binding, and dotted ones indicate multi-step processes.</div></p></div><div class="aux"><div class="resc">Zhang-Zhi Hu, et al. J Proteomics Bioinform. ;1(2):47-60.</div><p class="links"><a class="dblinks" href="/pubmed/19088860" ref="ordinalpos=3242&amp;ncbi_uid=1582208&amp;link_uid=19088860">Citation</a><a class="dblinks" href="/pmc/articles/PMC2603135/" ref="ordinalpos=3242&amp;ncbi_uid=1582208&amp;link_uid=PMC2603135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3243.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193487/figure/fig1/" ref="ordinalpos=3243&amp;ncbi_uid=4089908&amp;link_uid=PMC3193487" href="/pmc/articles/PMC3193487/"><img src="/pmc/articles/instance/3193487/bin/cr201188f1.gif" src-large="/pmc/articles/instance/3193487/bin/cr201188f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193487/" ref="ordinalpos=3243&amp;ncbi_uid=4089908&amp;link_uid=PMC3193487" image-link="/pmc/articles/PMC3193487/figure/fig1/" class="imagepopup">Figure 1.  From: TLR-induced activation of Btk - Role for endosomal MHC class II molecules revealed. </a></p><div class="supp"><p class="details"><div>Schematic of crosstalk between TLR <span class="highlight" style="background-color:">pathway</span> and MHC molecules. <b>(A)</b> The traditional view of TLR <span class="highlight" style="background-color:">signaling</span> which holds that TLR ligation initiates several <span class="highlight" style="background-color:">signaling</span> cascades within the cell that drive the production of type I IFNs and pro-inflammatory cytokines that are essential for both innate and adaptive immune responses resulting in pathogen clearance. <b>(B)</b> Recent evidence has shown that a second <span class="highlight" style="background-color:">pathway</span> involving crosstalk between the TLR <span class="highlight" style="background-color:">pathway</span> and MHC <span class="highlight" style="background-color:">pathway</span> is initiated. This crosstalk results in the formation the Btk-CD40-MHC class II <span class="highlight" style="background-color:">signaling</span> complex at the endosome that enhances TLR-driven cytokine production. Ligation of TLRs also results in the inhibition of MARCH-1-mediated ubiquitination and degradation of MHC class II molecules, potentially increasing the pool of intracellular MHC class II molecules available to form the MHC-Btk signalosome at the endosome.</div></p></div><div class="aux"><div class="resc">Joan Ní Gabhann, et al. Cell Res. 2011 July;21(7):998-1001.</div><p class="links"><a class="dblinks" href="/pubmed/21606951" ref="ordinalpos=3243&amp;ncbi_uid=4089908&amp;link_uid=21606951">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193487/" ref="ordinalpos=3243&amp;ncbi_uid=4089908&amp;link_uid=PMC3193487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3244.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3368902/figure/pone-0038604-g003/" ref="ordinalpos=3244&amp;ncbi_uid=4062125&amp;link_uid=PMC3368902" href="/pmc/articles/PMC3368902/"><img src="/pmc/articles/instance/3368902/bin/pone.0038604.g003.gif" src-large="/pmc/articles/instance/3368902/bin/pone.0038604.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3368902/" ref="ordinalpos=3244&amp;ncbi_uid=4062125&amp;link_uid=PMC3368902" image-link="/pmc/articles/PMC3368902/figure/pone-0038604-g003/" class="imagepopup">Figure 3. Temporal profile of components of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in rainbow trout, as determined by Western Blot densitometry..  From: Postprandial Regulation of Hepatic MicroRNAs Predicted to Target the Insulin <span class="highlight" style="background-color:">Pathway</span> in Rainbow Trout. </a></p><div class="supp"><p class="details"><div>Means and S.E. of n = 6 samples per group are shown. Data were analysed using a one-way ANOVA, followed by the Newman-Keuls post-hoc test. Different letters indicate a significant difference at p&lt;0.05.</div></p></div><div class="aux"><div class="resc">Jan A. Mennigen, et al. PLoS One. 2012;7(6):e38604.</div><p class="links"><a class="dblinks" href="/pubmed/22701681" ref="ordinalpos=3244&amp;ncbi_uid=4062125&amp;link_uid=22701681">Citation</a><a class="dblinks" href="/pmc/articles/PMC3368902/" ref="ordinalpos=3244&amp;ncbi_uid=4062125&amp;link_uid=PMC3368902">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3245.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3063861/figure/F4/" ref="ordinalpos=3245&amp;ncbi_uid=3656581&amp;link_uid=PMC3063861" href="/pmc/articles/PMC3063861/"><img src="/pmc/articles/instance/3063861/bin/nihms252493f4.gif" src-large="/pmc/articles/instance/3063861/bin/nihms252493f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3063861/" ref="ordinalpos=3245&amp;ncbi_uid=3656581&amp;link_uid=PMC3063861" image-link="/pmc/articles/PMC3063861/figure/F4/" class="imagepopup">Figure 4. Biased ligands and biased receptors.  From: ?-arrestin: a <span class="highlight" style="background-color:">signaling</span> molecule and potential therapeutic target for heart failure. </a></p><div class="supp"><p class="details"><div>GPCR can preferentially activate various <span class="highlight" style="background-color:">signaling</span> mechanisms following ligand stimulation. A) Binding of a biased ligand (purple: β arrestin bias; blue: G-protein bias), to an unbiased receptor results in a biased response activating either G-protein mediated or β-arrestin <span class="highlight" style="background-color:">pathway</span>. B) Similarly, binding of an unbiased ligand to the biased receptor (β-arrestin-biased in purple or G protein-biased in blue) also results in a biased response (Reproduced with permission from Ref [31]).</div></p></div><div class="aux"><div class="resc">Nabila Noor, et al. J Mol Cell Cardiol. ;51(4):534-541.</div><p class="links"><a class="dblinks" href="/pubmed/21074538" ref="ordinalpos=3245&amp;ncbi_uid=3656581&amp;link_uid=21074538">Citation</a><a class="dblinks" href="/pmc/articles/PMC3063861/" ref="ordinalpos=3245&amp;ncbi_uid=3656581&amp;link_uid=PMC3063861">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3246.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3032813/figure/F1/" ref="ordinalpos=3246&amp;ncbi_uid=3650555&amp;link_uid=PMC3032813" href="/pmc/articles/PMC3032813/"><img src="/pmc/articles/instance/3032813/bin/nihms257155f1.gif" src-large="/pmc/articles/instance/3032813/bin/nihms257155f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3032813/" ref="ordinalpos=3246&amp;ncbi_uid=3650555&amp;link_uid=PMC3032813" image-link="/pmc/articles/PMC3032813/figure/F1/" class="imagepopup">Figure 1.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> drives cellular survival in human colon carcinoma cells. </a></p><div class="supp"><p class="details"><div>Components of the canonical HH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are expressed in human colon carcinoma cell lines. <i>A</i>: mRNA expression, with GAPDH employed as the endogenous control; <i>B</i>: Western blot analysis, with β-Actin for loading control.</div></p></div><div class="aux"><div class="resc">Tapati Mazumdar, et al. Cancer Res. ;71(3):1092-1102.</div><p class="links"><a class="dblinks" href="/pubmed/21135115" ref="ordinalpos=3246&amp;ncbi_uid=3650555&amp;link_uid=21135115">Citation</a><a class="dblinks" href="/pmc/articles/PMC3032813/" ref="ordinalpos=3246&amp;ncbi_uid=3650555&amp;link_uid=PMC3032813">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3247.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2799352/figure/fig9/" ref="ordinalpos=3247&amp;ncbi_uid=2137982&amp;link_uid=PMC2799352" href="/pmc/articles/PMC2799352/"><img src="/pmc/articles/instance/2799352/bin/pp1520133f09.gif" src-large="/pmc/articles/instance/2799352/bin/pp1520133f09.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2799352/" ref="ordinalpos=3247&amp;ncbi_uid=2137982&amp;link_uid=PMC2799352" image-link="/pmc/articles/PMC2799352/figure/fig9/" class="imagepopup">Figure 9. From: The Nuclear Interactor PYL8/RCAR3 of Fagus sylvatica FsPP2C1 Is a Positive Regulator of Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> in Seeds and Stress. </a></p><div class="supp"><p class="details"><div>Schematic model describing the function of the PP2C-PYL/RCAR interaction in the control of the ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the absence of ABA, PYR-PYL/RCAR proteins may interact with PP2Cs, but these PP2Cs remain active. Upon ABA perception, PYR-PYL/RCAR proteins interact with PP2Cs, maintaining these PP2Cs as inactive and subsequently activating the transcription of ABA-responsive genes (including PP2Cs). The specific colocalization of FsPP2C1-PYL8/RCAR3 in the nucleus is shown.</div></p></div><div class="aux"><div class="resc">Xandra Saavedra, et al. Plant Physiol. 2010 January;152(1):133-150.</div><p class="links"><a class="dblinks" href="/pubmed/19889877" ref="ordinalpos=3247&amp;ncbi_uid=2137982&amp;link_uid=19889877">Citation</a><a class="dblinks" href="/pmc/articles/PMC2799352/" ref="ordinalpos=3247&amp;ncbi_uid=2137982&amp;link_uid=PMC2799352">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3248.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC166246/figure/fig5/" ref="ordinalpos=3248&amp;ncbi_uid=367194&amp;link_uid=PMC166246" href="/pmc/articles/PMC166246/"><img src="/pmc/articles/instance/166246/bin/pq1332636005.gif" src-large="/pmc/articles/instance/166246/bin/pq1332636005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC166246/" ref="ordinalpos=3248&amp;ncbi_uid=367194&amp;link_uid=PMC166246" image-link="/pmc/articles/PMC166246/figure/fig5/" class="imagepopup">Fig. 5. From: Transforming growth factor ? <span class="highlight" style="background-color:">signaling</span> impairs Neu-induced mammary  tumorigenesis while promoting pulmonary metastasis. </a></p><div class="supp"><p class="details"><div>TGF-β enhances lung metastases despite suppressive effects on primary  tumor formation. The onset of Neu-induced mammary tumors is delayed by  expression of an activated TβRI and accelerated by a dominant negative  TβRII. However, activation of TGF-β <span class="highlight" style="background-color:">signaling</span> enhances extravasation  of Neu-induced mammary tumor cells into lung tissue, which can be impaired by  inhibition of the TGF-β <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Peter M. Siegel, et al. Proc Natl Acad Sci U S A. 2003 July 8;100(14):8430-8435.</div><p class="links"><a class="dblinks" href="/pubmed/12808151" ref="ordinalpos=3248&amp;ncbi_uid=367194&amp;link_uid=12808151">Citation</a><a class="dblinks" href="/pmc/articles/PMC166246/" ref="ordinalpos=3248&amp;ncbi_uid=367194&amp;link_uid=PMC166246">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3249.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3919687/figure/F6/" ref="ordinalpos=3249&amp;ncbi_uid=5799945&amp;link_uid=PMC3919687" href="/pmc/articles/PMC3919687/"><img src="/pmc/articles/instance/3919687/bin/nihms547222f6.gif" src-large="/pmc/articles/instance/3919687/bin/nihms547222f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3919687/" ref="ordinalpos=3249&amp;ncbi_uid=5799945&amp;link_uid=PMC3919687" image-link="/pmc/articles/PMC3919687/figure/F6/" class="imagepopup">Fig. 6.  From: Comparative kinetic analyses of gene profiles of na?ve CD4+ and CD8+ T cells from young and old animals reveal novel age-related alterations. </a></p><div class="supp"><p class="details"><div>Kinetic changes in gene expression of selected components of Th2 responses. Pathways were generated by Ingenuity <span class="highlight" style="background-color:">Pathway</span> analysis. (A) <span class="highlight" style="background-color:">Pathway</span> analysis of Th2 differentiation. (B) IL-10 <span class="highlight" style="background-color:">Signaling</span> and IL-4Rα expression, and (C) IL-4 <span class="highlight" style="background-color:">signaling</span> and production. Adjacent to each step of the <span class="highlight" style="background-color:">pathway</span> are the gene profile plots of old and young CD4<sup>+</sup> T cells. Axis labels the same as in Fig. 2.</div></p></div><div class="aux"><div class="resc">Noweeda Mirza, et al. Aging Cell. ;10(5):853-867.</div><p class="links"><a class="dblinks" href="/pubmed/21711441" ref="ordinalpos=3249&amp;ncbi_uid=5799945&amp;link_uid=21711441">Citation</a><a class="dblinks" href="/pmc/articles/PMC3919687/" ref="ordinalpos=3249&amp;ncbi_uid=5799945&amp;link_uid=PMC3919687">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3250.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3819741/figure/F2/" ref="ordinalpos=3250&amp;ncbi_uid=5479576&amp;link_uid=PMC3819741" href="/pmc/articles/PMC3819741/"><img src="/pmc/articles/instance/3819741/bin/1471-2156-14-94-2.gif" src-large="/pmc/articles/instance/3819741/bin/1471-2156-14-94-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3819741/" ref="ordinalpos=3250&amp;ncbi_uid=5479576&amp;link_uid=PMC3819741" image-link="/pmc/articles/PMC3819741/figure/F2/" class="imagepopup">Figure 2.  From: Single nucleotide polymorphisms and haplotypes associated with feed efficiency in beef cattle. </a></p><div class="supp"><p class="details"><div><b>Identification of the genes in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that harbor or are in the close proximity of SNPs associated with feed efficiency.</b> <span class="highlight" style="background-color:">Pathway</span> components marked with a star harbor or are in the proximity of SNPs associated with feed efficiency.</div></p></div><div class="aux"><div class="resc">Nick VL Serão, et al. BMC Genet. 2013;14:94-94.</div><p class="links"><a class="dblinks" href="/pubmed/24066663" ref="ordinalpos=3250&amp;ncbi_uid=5479576&amp;link_uid=24066663">Citation</a><a class="dblinks" href="/pmc/articles/PMC3819741/" ref="ordinalpos=3250&amp;ncbi_uid=5479576&amp;link_uid=PMC3819741">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3251.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3597812/figure/F7/" ref="ordinalpos=3251&amp;ncbi_uid=5917306&amp;link_uid=PMC3597812" href="/pmc/articles/PMC3597812/"><img src="/pmc/articles/instance/3597812/bin/zbc0151342630007.gif" src-large="/pmc/articles/instance/3597812/bin/zbc0151342630007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3597812/" ref="ordinalpos=3251&amp;ncbi_uid=5917306&amp;link_uid=PMC3597812" image-link="/pmc/articles/PMC3597812/figure/F7/" class="imagepopup">FIGURE 7. From: Fucoidan Is a Novel Platelet Agonist for the C-type Lectin-like Receptor 2 (CLEC-2). </a></p><div class="supp"><p class="details"><div><b>Model depicting tyrosine kinase-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in platelets activated with fucoidan.</b> Fucoidan activates platelets through an SFK-dependent <span class="highlight" style="background-color:">pathway</span> through the CLEC-2 receptor. YP represents tyrosine phosphorylation sites on the cytoplasmic domain of CLEC-2 receptor.</div></p></div><div class="aux"><div class="resc">Bhanu Kanth Manne, et al. J Biol Chem. 2013 March 15;288(11):7717-7726.</div><p class="links"><a class="dblinks" href="/pubmed/23341451" ref="ordinalpos=3251&amp;ncbi_uid=5917306&amp;link_uid=23341451">Citation</a><a class="dblinks" href="/pmc/articles/PMC3597812/" ref="ordinalpos=3251&amp;ncbi_uid=5917306&amp;link_uid=PMC3597812">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3252.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2894844/figure/F7/" ref="ordinalpos=3252&amp;ncbi_uid=2379439&amp;link_uid=PMC2894844" href="/pmc/articles/PMC2894844/"><img src="/pmc/articles/instance/2894844/bin/1471-2202-11-74-7.gif" src-large="/pmc/articles/instance/2894844/bin/1471-2202-11-74-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2894844/" ref="ordinalpos=3252&amp;ncbi_uid=2379439&amp;link_uid=PMC2894844" image-link="/pmc/articles/PMC2894844/figure/F7/" class="imagepopup">Figure 7.  From: GT1b-induced neurotoxicity is mediated by the Akt/GSK-3/tau <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> but not caspase-3 in mesencephalic dopaminergic neurons. </a></p><div class="supp"><p class="details"><div><b>Schematic drawing of Akt/GSK-3β/tau <span class="highlight" style="background-color:">signaling</span></b>. Akt is activated in response to ILK or PI3 Kinase <span class="highlight" style="background-color:">signaling</span>, and subsequent activation of Akt. Akt is activated in response to ILK or PI3 kinase <span class="highlight" style="background-color:">signaling</span>, and subsequent activation of Akt. Activated Akt phosphorylates GSK-3β at Ser 9, leading to GSK-3β inactivation. Akt inactivation by GT1b induces dephosphorylation of GSK-3β, leading to GSK-3β activation. GSK-3β activation results in aberrant tau phosphorylation, axon instability, and neuronal death. Although we did not provide direct evidence of whether GT1b neuroxocicity is mediated by ILK or PI3K <span class="highlight" style="background-color:">signaling</span> or both, participation of GT1b in inhibition of Akt upstream signals, such as ILK or PI3K, is predicted.</div></p></div><div class="aux"><div class="resc">Eun S Chung, et al. BMC Neurosci. 2010;11:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/20540782" ref="ordinalpos=3252&amp;ncbi_uid=2379439&amp;link_uid=20540782">Citation</a><a class="dblinks" href="/pmc/articles/PMC2894844/" ref="ordinalpos=3252&amp;ncbi_uid=2379439&amp;link_uid=PMC2894844">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3253.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1862519/figure/F1/" ref="ordinalpos=3253&amp;ncbi_uid=1004115&amp;link_uid=PMC1862519" href="/pmc/articles/PMC1862519/"><img src="/pmc/articles/instance/1862519/bin/nihms18021f1.gif" src-large="/pmc/articles/instance/1862519/bin/nihms18021f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1862519/" ref="ordinalpos=3253&amp;ncbi_uid=1004115&amp;link_uid=PMC1862519" image-link="/pmc/articles/PMC1862519/figure/F1/" class="imagepopup">Figure 1.  From: TRPC Channels as Novel Effectors of BDNF <span class="highlight" style="background-color:">Signaling</span>: Potential Implications for Rett Syndrome. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> cascades activated by BDNF-stimulated autophosphorylation of TrkB receptors may affect prominent pathways responsible for elevations of intracellular Ca<sup>2+</sup> levels in neurons. Activation of the PLC-γ/IP<sub>3</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to Ca<sup>2+</sup> mobilization from IP3-sensitive intracellular stores. In addition, NMDA receptors and voltage-gated Ca<sup>2+</sup> channels could be directly phosphorylated by activated TrkB receptors or indirectly through the Ras-MAPK/ERK or PI3-K pathways. Combined, these covalent modifications potentially enhance Ca<sup>2+</sup> signals within spines and dendrites.</div></p></div><div class="aux"><div class="resc">Michelle D. Amaral, et al. Pharmacol Ther. ;113(2):394-409.</div><p class="links"><a class="dblinks" href="/pubmed/17118456" ref="ordinalpos=3253&amp;ncbi_uid=1004115&amp;link_uid=17118456">Citation</a><a class="dblinks" href="/pmc/articles/PMC1862519/" ref="ordinalpos=3253&amp;ncbi_uid=1004115&amp;link_uid=PMC1862519">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3254.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3843001/figure/F9/" ref="ordinalpos=3254&amp;ncbi_uid=5834493&amp;link_uid=PMC3843001" href="/pmc/articles/PMC3843001/"><img src="/pmc/articles/instance/3843001/bin/3754fig9.gif" src-large="/pmc/articles/instance/3843001/bin/3754fig9.jpg" alt="FIGURE 9:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3843001/" ref="ordinalpos=3254&amp;ncbi_uid=5834493&amp;link_uid=PMC3843001" image-link="/pmc/articles/PMC3843001/figure/F9/" class="imagepopup">FIGURE 9:.  From: Distinct amino acid-sensing mTOR pathways regulate skeletal myogenesis. </a></p><div class="supp"><p class="details"><div>A proposed model of mTOR myogenic <span class="highlight" style="background-color:">signaling</span>. Amino acids activate two pathways of opposing functions in myogenesis. The Rag GTPases activate mTORC1 and subsequently suppress the PI3K-Akt <span class="highlight" style="background-color:">pathway</span> through IRS1 phosphorylation. The Vps34-PLD1-PA <span class="highlight" style="background-color:">pathway</span> activates a kinase-independent function of mTOR that regulates myogenic expression of IGF-II. A third mTOR <span class="highlight" style="background-color:">pathway</span>, mediated by MyoD and follistatin, is rapamycin sensitive and kinase dependent but is independent of amino acids.</div></p></div><div class="aux"><div class="resc">Mee-Sup Yoon, et al. Mol Biol Cell. 2013 December 1;24(23):3754-3763.</div><p class="links"><a class="dblinks" href="/pubmed/24068326" ref="ordinalpos=3254&amp;ncbi_uid=5834493&amp;link_uid=24068326">Citation</a><a class="dblinks" href="/pmc/articles/PMC3843001/" ref="ordinalpos=3254&amp;ncbi_uid=5834493&amp;link_uid=PMC3843001">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3255.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3125348/figure/F6/" ref="ordinalpos=3255&amp;ncbi_uid=3044642&amp;link_uid=PMC3125348" href="/pmc/articles/PMC3125348/"><img src="/pmc/articles/instance/3125348/bin/1471-2229-11-88-6.gif" src-large="/pmc/articles/instance/3125348/bin/1471-2229-11-88-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3125348/" ref="ordinalpos=3255&amp;ncbi_uid=3044642&amp;link_uid=PMC3125348" image-link="/pmc/articles/PMC3125348/figure/F6/" class="imagepopup">Figure 6.  From: Prospecting for Genes involved in transcriptional regulation of plant defenses, a bioinformatics approach. </a></p><div class="supp"><p class="details"><div><b>Co-expression network of the ET biosynthesis and <span class="highlight" style="background-color:">signaling</span> pathways</b>. In panel A, the genes in the blue boxes in the network correspond to the blue-colored components of the ET biosynthesis <span class="highlight" style="background-color:">pathway</span> indicated in Figure 2. In panel B, The genes in colored boxes correspond to genes in similarly colored boxes of the ET signal transduction <span class="highlight" style="background-color:">pathway</span> shown in Figure 2. The genes in the white boxes in both panels indicate co-expressed genes having at least one edge to the <span class="highlight" style="background-color:">pathway</span> genes.</div></p></div><div class="aux"><div class="resc">Marcel C van Verk, et al. BMC Plant Biol. 2011;11:88-88.</div><p class="links"><a class="dblinks" href="/pubmed/21595873" ref="ordinalpos=3255&amp;ncbi_uid=3044642&amp;link_uid=21595873">Citation</a><a class="dblinks" href="/pmc/articles/PMC3125348/" ref="ordinalpos=3255&amp;ncbi_uid=3044642&amp;link_uid=PMC3125348">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3256.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517183/figure/F2/" ref="ordinalpos=3256&amp;ncbi_uid=4435009&amp;link_uid=PMC3517183" href="/pmc/articles/PMC3517183/"><img src="/pmc/articles/instance/3517183/bin/nihms-421438-f0002.gif" src-large="/pmc/articles/instance/3517183/bin/nihms-421438-f0002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517183/" ref="ordinalpos=3256&amp;ncbi_uid=4435009&amp;link_uid=PMC3517183" image-link="/pmc/articles/PMC3517183/figure/F2/" class="imagepopup">FIGURE 2.  From: Single-Cell Network Profiling of Peripheral Blood Mononuclear Cells from Healthy Donors Reveals Age- and Race-Associated Differences in Immune <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activation. </a></p><div class="supp"><p class="details"><div>Range of <span class="highlight" style="background-color:">signaling</span> for each modulated <span class="highlight" style="background-color:">signaling</span> node within each of the seven distinct immune cell subpopulations.</div></p></div><div class="aux"><div class="resc">Diane M. Longo, et al. J Immunol. ;188(4):1717-1725.</div><p class="links"><a class="dblinks" href="/pubmed/22246624" ref="ordinalpos=3256&amp;ncbi_uid=4435009&amp;link_uid=22246624">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517183/" ref="ordinalpos=3256&amp;ncbi_uid=4435009&amp;link_uid=PMC3517183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3257.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3681934/figure/pone-0066254-g009/" ref="ordinalpos=3257&amp;ncbi_uid=5008537&amp;link_uid=PMC3681934" href="/pmc/articles/PMC3681934/"><img src="/pmc/articles/instance/3681934/bin/pone.0066254.g009.gif" src-large="/pmc/articles/instance/3681934/bin/pone.0066254.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3681934/" ref="ordinalpos=3257&amp;ncbi_uid=5008537&amp;link_uid=PMC3681934" image-link="/pmc/articles/PMC3681934/figure/pone-0066254-g009/" class="imagepopup">Figure 9. RhoA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates IL1RAPL1-mediated AMPA receptor newly insertion to surface in cultured cortical neurons..  From: IL1RAPL1 Associated with Mental Retardation and Autism Regulates the Formation and Stabilization of Glutamatergic Synapses of Cortical Neurons through RhoA <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A,</b> Effects of knock-down of endogenous Mcf2l on IL1RAPL1-mediated suppression of pH-GluA1 insertion (<i>n</i> = 9, <i>n</i> = 9, <i>n</i> = 10, <i>n</i> = 10, <i>F</i> = 3.54, <i>p</i>&lt;0.05, ANOVA). <b>B,</b> Effects of Y27632 on IL1RAPL1-mediated suppression of pH-GluA1 insertion (<i>n</i> = 10, respectively, <i>F</i> = 10.4, <i>p</i>&lt;0.0001, ANOVA). <b>C,</b> Effects of Y27632 on IL1RAPL1-induced increase of pH-GluA2 insertion (<i>n</i> = 10, respectively, <i>F</i> = 5.21, <i>p</i>&lt;0.005, ANOVA). <b>D,</b> Influences of NLGN1 overexpression on pH-GluA1 insertion with or without Y27632 (<i>n</i> = 8, respectively, <i>F</i> = 9.36, <i>p</i>&lt;0.0002, ANOVA). E, Influences of NLGN1 overexpression on pH-GluA2 insertion with or without Y27632 (<i>n</i> = 8, respectively, <i>F</i> = 13.1, <i>p</i>&lt;0.0001, ANOVA). Cultured cortical neurons were treated with 10 μM Y27632 for 3 days. *, <i>p</i>&lt;0.05; **, <i>p</i>&lt;0.01; ***, <i>p</i>&lt;0.001. Error bars represent s.e.m.</div></p></div><div class="aux"><div class="resc">Takashi Hayashi, et al. PLoS One. 2013;8(6):e66254.</div><p class="links"><a class="dblinks" href="/pubmed/23785489" ref="ordinalpos=3257&amp;ncbi_uid=5008537&amp;link_uid=23785489">Citation</a><a class="dblinks" href="/pmc/articles/PMC3681934/" ref="ordinalpos=3257&amp;ncbi_uid=5008537&amp;link_uid=PMC3681934">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3258.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3203172/figure/pone-0027053-g004/" ref="ordinalpos=3258&amp;ncbi_uid=3321084&amp;link_uid=PMC3203172" href="/pmc/articles/PMC3203172/"><img src="/pmc/articles/instance/3203172/bin/pone.0027053.g004.gif" src-large="/pmc/articles/instance/3203172/bin/pone.0027053.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3203172/" ref="ordinalpos=3258&amp;ncbi_uid=3321084&amp;link_uid=PMC3203172" image-link="/pmc/articles/PMC3203172/figure/pone-0027053-g004/" class="imagepopup">Figure 4. PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation protected cells from apoptosis induced by combination treatment of LY294002 and tamoxifen..  From: Sensitization of Glioma Cells to Tamoxifen-Induced Apoptosis by Pl3-Kinase Inhibitor through the GSK-3?/?-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>U251 cells were treated with 10 µmol/L LY294002 (LY), or/and 5 µmol/L tamoxifen (TAM), or combination of LY and TAM with additional IGF-1 (50 ng/ml) for 12 h, followed by Annexin V-FITC/propidium iodide staining. Columns representing the flow cytometry data presented in left. Data are means ±S.E.M. of values from three independent experiments; FL2-H, propidium iodide; FL1-H, Annexin V.</div></p></div><div class="aux"><div class="resc">Cuixian Li, et al. PLoS One. 2011;6(10):e27053.</div><p class="links"><a class="dblinks" href="/pubmed/22046442" ref="ordinalpos=3258&amp;ncbi_uid=3321084&amp;link_uid=22046442">Citation</a><a class="dblinks" href="/pmc/articles/PMC3203172/" ref="ordinalpos=3258&amp;ncbi_uid=3321084&amp;link_uid=PMC3203172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3259.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2821978/figure/F3/" ref="ordinalpos=3259&amp;ncbi_uid=2788041&amp;link_uid=PMC2821978" href="/pmc/articles/PMC2821978/"><img src="/pmc/articles/instance/2821978/bin/nihms150136f3.gif" src-large="/pmc/articles/instance/2821978/bin/nihms150136f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2821978/" ref="ordinalpos=3259&amp;ncbi_uid=2788041&amp;link_uid=PMC2821978" image-link="/pmc/articles/PMC2821978/figure/F3/" class="imagepopup">Figure 3.  From: Association of the C8orf13-BLK Region with Systemic Sclerosis in North-American and European Populations. </a></p><div class="supp"><p class="details"><div>Comparison of gene expression in peripheral blood cells stratified by the <i>C8orf13-BLK</i> variants rs2736340-rs13277113 heterozygote versus homozygote in SSc demonstrates alterations in B Cell Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (A) and NFκB <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (B). Red represents upregulated genes. Green represents downregulated genes.</div></p></div><div class="aux"><div class="resc">Pravitt Gourh, et al. J Autoimmun. ;34(2):155.</div><p class="links"><a class="dblinks" href="/pubmed/19796918" ref="ordinalpos=3259&amp;ncbi_uid=2788041&amp;link_uid=19796918">Citation</a><a class="dblinks" href="/pmc/articles/PMC2821978/" ref="ordinalpos=3259&amp;ncbi_uid=2788041&amp;link_uid=PMC2821978">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3260.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3398465/figure/F4/" ref="ordinalpos=3260&amp;ncbi_uid=4123853&amp;link_uid=PMC3398465" href="/pmc/articles/PMC3398465/"><img src="/pmc/articles/instance/3398465/bin/nihms384614f4.gif" src-large="/pmc/articles/instance/3398465/bin/nihms384614f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3398465/" ref="ordinalpos=3260&amp;ncbi_uid=4123853&amp;link_uid=PMC3398465" image-link="/pmc/articles/PMC3398465/figure/F4/" class="imagepopup">Figure 4. Cellular <span class="highlight" style="background-color:">signaling</span> map of embryonic salivary gland development.  From: Systems analysis of salivary gland development and disease. </a></p><div class="supp"><p class="details"><div>A simplified overview of major <span class="highlight" style="background-color:">signaling</span> pathways known to control salivary gland development based on experimental studies. Slash dot slash: known effect, but <span class="highlight" style="background-color:">pathway</span> not identified/described yet; orange lines/arrows: pathways that affect morphogenesis/differentiation; plus sign: positive effect (activation); minus: negative effect (inhibition); interrupted lines: intermediate steps omitted; blue dotted lines and arrows: affect expression; open arrows: protein is modified (proteolysis); blunt line (inverted T): inhibition; and circular edge rectangles: <span class="highlight" style="background-color:">signaling</span> modules. See [66] for a detailed computational model of <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Melinda Larsen, et al. Wiley Interdiscip Rev Syst Biol Med. ;2(6):670-682.</div><p class="links"><a class="dblinks" href="/pubmed/20890964" ref="ordinalpos=3260&amp;ncbi_uid=4123853&amp;link_uid=20890964">Citation</a><a class="dblinks" href="/pmc/articles/PMC3398465/" ref="ordinalpos=3260&amp;ncbi_uid=4123853&amp;link_uid=PMC3398465">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3261.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3891255/figure/F2/" ref="ordinalpos=3261&amp;ncbi_uid=6503138&amp;link_uid=PMC3891255" href="/pmc/articles/PMC3891255/"><img src="/pmc/articles/instance/3891255/bin/zh20161370120002.gif" src-large="/pmc/articles/instance/3891255/bin/zh20161370120002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3891255/" ref="ordinalpos=3261&amp;ncbi_uid=6503138&amp;link_uid=PMC3891255" image-link="/pmc/articles/PMC3891255/figure/F2/" class="imagepopup">Fig. 2. From: Renal and cardiovascular sensory receptors and blood pressure regulation. </a></p><div class="supp"><p class="details"><div>When WT mice are treated with antibiotics (ABX) to reduce gut flora and therefore the circulating SCFA, the Olfr78 and Gpr41 sides of the <span class="highlight" style="background-color:">pathway</span> are eliminated simultaneously, and therefore there is little change in blood pressure. However, in mice null for Olfr78, SCFA are <span class="highlight" style="background-color:">signaling</span> solely through the hypotensive side of the <span class="highlight" style="background-color:">pathway</span>. Therefore, if the ligand is reduced by antibiotic treatment in these mice, this hypotensive <span class="highlight" style="background-color:">pathway</span> is disrupted and there is a relative increase in blood pressure.</div></p></div><div class="aux"><div class="resc">Jennifer L. Pluznick. Am J Physiol Renal Physiol. 2013 August 15;305(4):F439-F444.</div><p class="links"><a class="dblinks" href="/pubmed/23761671" ref="ordinalpos=3261&amp;ncbi_uid=6503138&amp;link_uid=23761671">Citation</a><a class="dblinks" href="/pmc/articles/PMC3891255/" ref="ordinalpos=3261&amp;ncbi_uid=6503138&amp;link_uid=PMC3891255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3262.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3849893/figure/F1/" ref="ordinalpos=3262&amp;ncbi_uid=5575968&amp;link_uid=PMC3849893" href="/pmc/articles/PMC3849893/"><img src="/pmc/articles/instance/3849893/bin/1743-422X-10-299-1.gif" src-large="/pmc/articles/instance/3849893/bin/1743-422X-10-299-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3849893/" ref="ordinalpos=3262&amp;ncbi_uid=5575968&amp;link_uid=PMC3849893" image-link="/pmc/articles/PMC3849893/figure/F1/" class="imagepopup">Figure 1.  From: Role of viral and host factors in interferon based therapy of hepatitis C virus infection. </a></p><div class="supp"><p class="details"><div>Interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Muhammad Imran, et al. Virol J. 2013;10:299-299.</div><p class="links"><a class="dblinks" href="/pubmed/24079723" ref="ordinalpos=3262&amp;ncbi_uid=5575968&amp;link_uid=24079723">Citation</a><a class="dblinks" href="/pmc/articles/PMC3849893/" ref="ordinalpos=3262&amp;ncbi_uid=5575968&amp;link_uid=PMC3849893">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3263.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3547518/figure/fig3/" ref="ordinalpos=3263&amp;ncbi_uid=5724608&amp;link_uid=PMC3547518" href="/pmc/articles/PMC3547518/"><img src="/pmc/articles/instance/3547518/bin/cn-2012-00186b_0004.gif" src-large="/pmc/articles/instance/3547518/bin/cn-2012-00186b_0004.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3547518/" ref="ordinalpos=3263&amp;ncbi_uid=5724608&amp;link_uid=PMC3547518" image-link="/pmc/articles/PMC3547518/figure/fig3/" class="imagepopup">Figure 3.  From: A Dialogue between the Immune System and Brain, Spoken in the Language of Serotonin. </a></p><div class="supp"><p class="details"><div>Interleukin-1β (IL-1β) stimulates SERT activity through an interleukin-1 receptor (IL-1R)- and p38 MAPK-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nicole L. Baganz, et al. ACS Chem Neurosci. 2013 January 16;4(1):48-63.</div><p class="links"><a class="dblinks" href="/pubmed/23336044" ref="ordinalpos=3263&amp;ncbi_uid=5724608&amp;link_uid=23336044">Citation</a><a class="dblinks" href="/pmc/articles/PMC3547518/" ref="ordinalpos=3263&amp;ncbi_uid=5724608&amp;link_uid=PMC3547518">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3264.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3370249/figure/F7/" ref="ordinalpos=3264&amp;ncbi_uid=4976248&amp;link_uid=PMC3370249" href="/pmc/articles/PMC3370249/"><img src="/pmc/articles/instance/3370249/bin/zbc0261212310007.gif" src-large="/pmc/articles/instance/3370249/bin/zbc0261212310007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3370249/" ref="ordinalpos=3264&amp;ncbi_uid=4976248&amp;link_uid=PMC3370249" image-link="/pmc/articles/PMC3370249/figure/F7/" class="imagepopup">FIGURE 7. From: Fas Ligand Enhances Malignant Behavior of Tumor Cells through Interaction with Met, Hepatocyte Growth Factor Receptor, in Lipid Rafts. </a></p><div class="supp"><p class="details"><div><b>Proposed FasL-Met <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b></div></p></div><div class="aux"><div class="resc">Huan-Ching Lin, et al. J Biol Chem. 2012 June 8;287(24):20664-20673.</div><p class="links"><a class="dblinks" href="/pubmed/22535954" ref="ordinalpos=3264&amp;ncbi_uid=4976248&amp;link_uid=22535954">Citation</a><a class="dblinks" href="/pmc/articles/PMC3370249/" ref="ordinalpos=3264&amp;ncbi_uid=4976248&amp;link_uid=PMC3370249">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3265.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC127927/figure/f6/" ref="ordinalpos=3265&amp;ncbi_uid=224908&amp;link_uid=PMC127927" href="/pmc/articles/PMC127927/"><img src="/pmc/articles/instance/127927/bin/ii0521395006.gif" src-large="/pmc/articles/instance/127927/bin/ii0521395006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC127927/" ref="ordinalpos=3265&amp;ncbi_uid=224908&amp;link_uid=PMC127927" image-link="/pmc/articles/PMC127927/figure/f6/" class="imagepopup">FIG. 6. From: Chlamydia trachomatis Disrupts N-Cadherin-Dependent Cell-Cell Junctions and Sequesters ?-Catenin in Human Cervical Epithelial Cells  . </a></p><div class="supp"><p class="details"><div>Role of β-catenin in the wingless/Wnt nuclear <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Walter C. Prozialeck, et al. Infect Immun. 2002 May;70(5):2605-2613.</div><p class="links"><a class="dblinks" href="/pubmed/11953402" ref="ordinalpos=3265&amp;ncbi_uid=224908&amp;link_uid=11953402">Citation</a><a class="dblinks" href="/pmc/articles/PMC127927/" ref="ordinalpos=3265&amp;ncbi_uid=224908&amp;link_uid=PMC127927">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3266.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3619098/figure/F5/" ref="ordinalpos=3266&amp;ncbi_uid=4788006&amp;link_uid=PMC3619098" href="/pmc/articles/PMC3619098/"><img src="/pmc/articles/instance/3619098/bin/ijmsv10p0599g05.gif" src-large="/pmc/articles/instance/3619098/bin/ijmsv10p0599g05.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3619098/" ref="ordinalpos=3266&amp;ncbi_uid=4788006&amp;link_uid=PMC3619098" image-link="/pmc/articles/PMC3619098/figure/F5/" class="imagepopup">Figure 5.  From: Propofol-Induced Protection of SH-SY5Y Cells against Hydrogen Peroxide Is Associated with the HO-1 via the ERK <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> participate in the propofol reduces H<sub>2</sub>O<sub>2</sub>-induced SH-SY5Y cell death and ROS Production; results (presented as mean ± S.D.) are from three independent tests. (<b>A</b>) ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> participate in the propofol reduces H<sub>2</sub>O<sub>2</sub>-induced cell death. ** significant differences (P &lt; 0.01) compared with 1 mM H<sub>2</sub>O<sub>2</sub> group (the second bar). (<b>B</b>) The ROS production assay. (B-a) Blank control; (B-b) SH-SY5Y cell was treated with H<sub>2</sub>O<sub>2</sub>; (B-c) SH-SY5Y cell were treated with H<sub>2</sub>O<sub>2 </sub>and propofol; (B-d) The cells was treated with propofol for co-incubation, H<sub>2</sub>O<sub>2</sub> and the ERK inhibitor PD98059. *** significant differences (P &lt; 0.001) compared with the blank control group (the first bar).</div></p></div><div class="aux"><div class="resc">Jing Gu, et al. Int J Med Sci. 2013;10(5):599-606.</div><p class="links"><a class="dblinks" href="/pubmed/23569422" ref="ordinalpos=3266&amp;ncbi_uid=4788006&amp;link_uid=23569422">Citation</a><a class="dblinks" href="/pmc/articles/PMC3619098/" ref="ordinalpos=3266&amp;ncbi_uid=4788006&amp;link_uid=PMC3619098">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3267.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355894/figure/F1/" ref="ordinalpos=3267&amp;ncbi_uid=4025595&amp;link_uid=PMC3355894" href="/pmc/articles/PMC3355894/"><img src="/pmc/articles/instance/3355894/bin/fonc-02-00030-g001.gif" src-large="/pmc/articles/instance/3355894/bin/fonc-02-00030-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355894/" ref="ordinalpos=3267&amp;ncbi_uid=4025595&amp;link_uid=PMC3355894" image-link="/pmc/articles/PMC3355894/figure/F1/" class="imagepopup">Figure 1.  From: Role of STAT3 in Transformation and Drug Resistance in CML. </a></p><div class="supp"><p class="details"><div><b>Activation and regulation of the STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is turned on by the activation of cell surface receptor (growth factor or cytokine receptors) leading to recruitment and activation of JAK-family of proteins, which in turn recruits and phosphorylates STAT3. STAT3 can also be directly phosphorylated by non-tyrosine kinase receptors (Src or c-abl). Phosphorylated STAT3 homodimerizes and translocates to the nucleus where it regulates gene expression. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be switched off by the actions of a phosphatase which dephosphorylates STAT3 and prevents dimer formation. Also, PIAS and SOCS family of protein can directly compete with STAT3 for either binding opportunities with the activating receptor or for dimerization and translocation into the nucleus. STAT3, signal transducers and activators of transcription 3; JAK, Janus kinase; SOCS, suppressors of cytokine <span class="highlight" style="background-color:">signaling</span>; PTP, protein-tyrosine phosphatase; PIAS, protein inhibitor of activated STATs.</div></p></div><div class="aux"><div class="resc">Rajesh R. Nair, et al. Front Oncol. 2012;2:30.</div><p class="links"><a class="dblinks" href="/pubmed/22649784" ref="ordinalpos=3267&amp;ncbi_uid=4025595&amp;link_uid=22649784">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355894/" ref="ordinalpos=3267&amp;ncbi_uid=4025595&amp;link_uid=PMC3355894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3268.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4324947/figure/fig6/" ref="ordinalpos=3268&amp;ncbi_uid=7185922&amp;link_uid=PMC4324947" href="/pmc/articles/PMC4324947/"><img src="/pmc/articles/instance/4324947/bin/JDR2015-920280.006.gif" src-large="/pmc/articles/instance/4324947/bin/JDR2015-920280.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4324947/" ref="ordinalpos=3268&amp;ncbi_uid=7185922&amp;link_uid=PMC4324947" image-link="/pmc/articles/PMC4324947/figure/fig6/" class="imagepopup">Figure 6.  From: The PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Mediates the High Glucose-Induced Expression of Extracellular Matrix Molecules in Human Retinal Pigment Epithelial Cells. </a></p><div class="supp"><p class="details"><div>The protein level of fibronectin, collagen IV, and laminin was mediated by PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RPE cells under high glucose conditions. (a, b, c) The protein expression of fibronectin, collagen IV, and laminin was significantly decreased when exposed to high glucose conditions for 24 h in RPE cells by pretreatment with 20 <i>μ</i>M LY294002 for 12 h. <sup>*</sup><i>P</i> &lt; 0.01 versus NG or HG + LY. The data shown represent the mean ± SD of three independent experiments.</div></p></div><div class="aux"><div class="resc">Dong Qin, et al. J Diabetes Res. 2015;2015:920280.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3268&amp;ncbi_uid=7185922&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4324947/" ref="ordinalpos=3268&amp;ncbi_uid=7185922&amp;link_uid=PMC4324947">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3269.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3808354/figure/pone-0078284-g003/" ref="ordinalpos=3269&amp;ncbi_uid=5477769&amp;link_uid=PMC3808354" href="/pmc/articles/PMC3808354/"><img src="/pmc/articles/instance/3808354/bin/pone.0078284.g003.gif" src-large="/pmc/articles/instance/3808354/bin/pone.0078284.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3808354/" ref="ordinalpos=3269&amp;ncbi_uid=5477769&amp;link_uid=PMC3808354" image-link="/pmc/articles/PMC3808354/figure/pone-0078284-g003/" class="imagepopup">Figure 3. Uric acid causes excessive nitric oxide production in pancreatic β-cells via the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Hyperuricemia Causes Pancreatic ?-Cell Death and Dysfunction through NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) The NO content, determined by a Griess assay kit, of Min6 cells treated with different concentrations of uric acid. NO production increased in a concentration-dependent manner. (<b>B</b>) Uric acid treatment of Min6 cells induced an increase in iNOS mRNA levels, as detected by RT-PCR. This effect was reversed by 50 µmol/L benzbromarone. (<b>C</b>) 5 µmol/L BAY 11–7082 also reversed the uric acid-induced increase in iNOS mRNA levels. **<i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Lu Jia, et al. PLoS One. 2013;8(10):e78284.</div><p class="links"><a class="dblinks" href="/pubmed/24205181" ref="ordinalpos=3269&amp;ncbi_uid=5477769&amp;link_uid=24205181">Citation</a><a class="dblinks" href="/pmc/articles/PMC3808354/" ref="ordinalpos=3269&amp;ncbi_uid=5477769&amp;link_uid=PMC3808354">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3270.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3417373/figure/F4/" ref="ordinalpos=3270&amp;ncbi_uid=4192666&amp;link_uid=PMC3417373" href="/pmc/articles/PMC3417373/"><img src="/pmc/articles/instance/3417373/bin/fcimb-01-00021-g004.gif" src-large="/pmc/articles/instance/3417373/bin/fcimb-01-00021-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3417373/" ref="ordinalpos=3270&amp;ncbi_uid=4192666&amp;link_uid=PMC3417373" image-link="/pmc/articles/PMC3417373/figure/F4/" class="imagepopup">Figure 4.  From: The Role of microRNAs in Helicobacter pylori Pathogenesis and Gastric Carcinogenesis. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> cascades that regulate the intrinsic and extrinsic pathways of apoptosis</b>. TNFα <span class="highlight" style="background-color:">signaling</span> leads to activation of NF-κB and the anti-apoptotic protein XIAP. Other receptors that detect survival factors, such as growth factors and cytokines, induce ERK1/2 and PI3K/Akt <span class="highlight" style="background-color:">signaling</span> cascades that ultimately result in the inhibition of the pro-apoptotic protein, Bad. In contrast, upon removal of survival factors, these receptors can signal via JNK to induce the pro-apoptotic protein, Bax. Pro-apoptotic and anti-apoptotic proteins govern the intrinsic <span class="highlight" style="background-color:">pathway</span> of cell death, which results in the release of cytochrome c from the mitochondria and induction of the caspase cascade. <span class="highlight" style="background-color:">Signaling</span> through death receptors initiate the extrinsic <span class="highlight" style="background-color:">pathway</span> of apoptosis, leading to the induction of caspases and cell death. There are numerous miRNAs that regulate each of these pathways and dysregulation of these miRNAs can lead to anti-apoptotic and tumorigenic responses.</div></p></div><div class="aux"><div class="resc">Jennifer M. Noto, et al. Front Cell Infect Microbiol. 2011;1:21.</div><p class="links"><a class="dblinks" href="/pubmed/22919587" ref="ordinalpos=3270&amp;ncbi_uid=4192666&amp;link_uid=22919587">Citation</a><a class="dblinks" href="/pmc/articles/PMC3417373/" ref="ordinalpos=3270&amp;ncbi_uid=4192666&amp;link_uid=PMC3417373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3271.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2953964/figure/F1/" ref="ordinalpos=3271&amp;ncbi_uid=3260221&amp;link_uid=PMC2953964" href="/pmc/articles/PMC2953964/"><img src="/pmc/articles/instance/2953964/bin/nihms235590f1.gif" src-large="/pmc/articles/instance/2953964/bin/nihms235590f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2953964/" ref="ordinalpos=3271&amp;ncbi_uid=3260221&amp;link_uid=PMC2953964" image-link="/pmc/articles/PMC2953964/figure/F1/" class="imagepopup">Figure 1. Model of S1P Stimulated Fibroblast Contractility in Dupuytren’s Disease.  From: Dupuytren's Fibroblast Contractility by Sphingosine-1-Phosphate is Mediated through Non Muscle Myosin II. </a></p><div class="supp"><p class="details"><div>S1P is proposed to promote cell contractility through S1P<sub>2</sub> activation of calcium-independent and calcium-dependent <span class="highlight" style="background-color:">signaling</span> pathways. In the calcium-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, MLCK is activated to phosphorylate MRLC. In the calcium-independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, ROCK is activated to phosphorylate MRLC and inactivate myosin light chain phosphatase. Activated MRLC bind to the neck domain of NMMII and activate actomyosin contractility. Actomyosin contractility is believed to cause Dupuytren’s disease. S1P stimulates fibroblast contractility through S1P<sub>2</sub> receptor mediated calcium-dependent and -independent pathways.</div></p></div><div class="aux"><div class="resc">Issei Komatsu, et al. J Hand Surg Am. ;35(10):1580-1588.</div><p class="links"><a class="dblinks" href="/pubmed/20888494" ref="ordinalpos=3271&amp;ncbi_uid=3260221&amp;link_uid=20888494">Citation</a><a class="dblinks" href="/pmc/articles/PMC2953964/" ref="ordinalpos=3271&amp;ncbi_uid=3260221&amp;link_uid=PMC2953964">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3272.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3018616/figure/fig3/" ref="ordinalpos=3272&amp;ncbi_uid=2709317&amp;link_uid=PMC3018616" href="/pmc/articles/PMC3018616/"><img src="/pmc/articles/instance/3018616/bin/CRP2011-175381.003.gif" src-large="/pmc/articles/instance/3018616/bin/CRP2011-175381.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3018616/" ref="ordinalpos=3272&amp;ncbi_uid=2709317&amp;link_uid=PMC3018616" image-link="/pmc/articles/PMC3018616/figure/fig3/" class="imagepopup">Figure 3.  From: Transforming Growth Factor-β1 as a Common Target Molecule for Development of Cardiovascular Diseases, Renal Insufficiency and Metabolic Syndrome. </a></p><div class="supp"><p class="details"><div>Accelerated activation of the TGF-<i>β</i>1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> causes CKD, MetS, and CVD.  Since TGF-<i>β</i>1 is a common target molecule and interactive regulator in those pathological conditions, manipulation of the TGF-<i>β</i>1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may be a useful approach for amelioration of mortality and morbidity in individuals with cardiovascular risk factors.</div></p></div><div class="aux"><div class="resc">Ken-ichi Aihara, et al. Cardiol Res Pract. 2011;2011:175381.</div><p class="links"><a class="dblinks" href="/pubmed/21234356" ref="ordinalpos=3272&amp;ncbi_uid=2709317&amp;link_uid=21234356">Citation</a><a class="dblinks" href="/pmc/articles/PMC3018616/" ref="ordinalpos=3272&amp;ncbi_uid=2709317&amp;link_uid=PMC3018616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3273.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2963354/figure/F7/" ref="ordinalpos=3273&amp;ncbi_uid=3292340&amp;link_uid=PMC2963354" href="/pmc/articles/PMC2963354/"><img src="/pmc/articles/instance/2963354/bin/zbc0441035110007.gif" src-large="/pmc/articles/instance/2963354/bin/zbc0441035110007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2963354/" ref="ordinalpos=3273&amp;ncbi_uid=3292340&amp;link_uid=PMC2963354" image-link="/pmc/articles/PMC2963354/figure/F7/" class="imagepopup">FIGURE 7. From: MTA1 Coregulator Regulates LPS Response via MyD88-dependent <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>MTA1 regulates LPS response via TLR4 MyD88-dependent <span class="highlight" style="background-color:">signaling</span>.</b> <i>A</i> and <i>B</i>, qPCR analysis of TLR4, TLR2, TLR3, and INF-β mRNA expressions in LPS-stimulated MEFs and macrophages isolated from MTA1<sup>+/+</sup> and MTA1<sup>−/−</sup> mice, respectively. <i>C</i>, Western blot analysis of phospho-ERK (<i>P-ERK</i>), ERK, phospho-JNK (<i>P-JNK</i>), and JNK in MTA1<sup>+/+</sup> and MTA1<sup>−/−</sup> MEFs in the presence and absence of LPS. <i>D</i>, schematic representation of MTA1 regulating LPS response via MyD88-dependent <span class="highlight" style="background-color:">signaling</span>: LPS stimulation of macrophages is accompanied by transcriptional stimulation of MTA1 through NF-κB <span class="highlight" style="background-color:">pathway</span>. Once MTA1 is induced, it acts as a coactivator and gets recruited to the <i>MyD</i>88 promoter, leading to increased MyD88 expression, which in turn, further potentiates NF-κB <span class="highlight" style="background-color:">signaling</span> as well as MyD88 transcription.</div></p></div><div class="aux"><div class="resc">Suresh B. Pakala, et al. J Biol Chem. 2010 October 22;285(43):32787-32792.</div><p class="links"><a class="dblinks" href="/pubmed/20702415" ref="ordinalpos=3273&amp;ncbi_uid=3292340&amp;link_uid=20702415">Citation</a><a class="dblinks" href="/pmc/articles/PMC2963354/" ref="ordinalpos=3273&amp;ncbi_uid=3292340&amp;link_uid=PMC2963354">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3274.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3273665/figure/F2/" ref="ordinalpos=3274&amp;ncbi_uid=4570763&amp;link_uid=PMC3273665" href="/pmc/articles/PMC3273665/"><img src="/pmc/articles/instance/3273665/bin/nihms336568f2.gif" src-large="/pmc/articles/instance/3273665/bin/nihms336568f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3273665/" ref="ordinalpos=3274&amp;ncbi_uid=4570763&amp;link_uid=PMC3273665" image-link="/pmc/articles/PMC3273665/figure/F2/" class="imagepopup">Figure 2.  From: SREBPs: Metabolic Integrators in Physiology and Metabolism. </a></p><div class="supp"><p class="details"><div>SREBPs in the innate immune response. The <span class="highlight" style="background-color:">pathway</span> whereby SREBP-1a is regulated by TLR4 <span class="highlight" style="background-color:">signaling</span> and stimulates Caspase-1 activity and IL-1β secretion is shown on the left. The involvement of interferon <span class="highlight" style="background-color:">signaling</span> (IRF) in regulating SREBP-2 is shown on the right. Both pathways are discussed in the text.</div></p></div><div class="aux"><div class="resc">Tae-Il Jeon, et al. Trends Endocrinol Metab. ;23(2):65-72.</div><p class="links"><a class="dblinks" href="/pubmed/22154484" ref="ordinalpos=3274&amp;ncbi_uid=4570763&amp;link_uid=22154484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3273665/" ref="ordinalpos=3274&amp;ncbi_uid=4570763&amp;link_uid=PMC3273665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3275.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC333401/figure/F1/" ref="ordinalpos=3275&amp;ncbi_uid=437863&amp;link_uid=PMC333401" href="/pmc/articles/PMC333401/"><img src="/pmc/articles/instance/333401/bin/1475-4924-2-17-1.gif" src-large="/pmc/articles/instance/333401/bin/1475-4924-2-17-1.jpg" alt="The bottom line" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC333401/" ref="ordinalpos=3275&amp;ncbi_uid=437863&amp;link_uid=PMC333401" image-link="/pmc/articles/PMC333401/figure/F1/" class="imagepopup">The bottom line.  From: Identifying new T-cell regulators. </a></p><div class="supp"><p class="details"><div>But unraveling this complex <span class="highlight" style="background-color:">pathway</span> and identifying all the players is a long and painstaking task. In this issue of <i>Journal of Biology </i>[1], Peter Chu, Jorge Pardo, Donald Payan, Charlene Liao and colleagues describe a large-scale functional genetic screen that is in theory capable of picking up most, if not all, of the molecules of the TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Their work will benefit researchers in immunology and other fields who are eager to identify all the players in a particular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (see 'The bottom line' box for a summary of the work).</div></p></div><div class="aux"><div class="resc">Diane Martindale. J Biol. 2003;2(3):17-17.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3275&amp;ncbi_uid=437863&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC333401/" ref="ordinalpos=3275&amp;ncbi_uid=437863&amp;link_uid=PMC333401">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3276.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3858357/figure/pone-0083360-g005/" ref="ordinalpos=3276&amp;ncbi_uid=5595354&amp;link_uid=PMC3858357" href="/pmc/articles/PMC3858357/"><img src="/pmc/articles/instance/3858357/bin/pone.0083360.g005.gif" src-large="/pmc/articles/instance/3858357/bin/pone.0083360.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3858357/" ref="ordinalpos=3276&amp;ncbi_uid=5595354&amp;link_uid=PMC3858357" image-link="/pmc/articles/PMC3858357/figure/pone-0083360-g005/" class="imagepopup">Figure 5. Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for leptin action..  From: Leptin Inhibits Glucose Intestinal Absorption via PKC, p38MAPK, PI3K and MEK/ERK. </a></p><div class="supp"><p class="details"><div>(-): inhibitory effect; (+): stimulatory effect. </div></p></div><div class="aux"><div class="resc">Ola El-Zein, et al. PLoS One. 2013;8(12):e83360.</div><p class="links"><a class="dblinks" href="/pubmed/24340098" ref="ordinalpos=3276&amp;ncbi_uid=5595354&amp;link_uid=24340098">Citation</a><a class="dblinks" href="/pmc/articles/PMC3858357/" ref="ordinalpos=3276&amp;ncbi_uid=5595354&amp;link_uid=PMC3858357">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3277.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3834580/figure/F2/" ref="ordinalpos=3277&amp;ncbi_uid=5508094&amp;link_uid=PMC3834580" href="/pmc/articles/PMC3834580/"><img src="/pmc/articles/instance/3834580/bin/nihms503603f2.gif" src-large="/pmc/articles/instance/3834580/bin/nihms503603f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3834580/" ref="ordinalpos=3277&amp;ncbi_uid=5508094&amp;link_uid=PMC3834580" image-link="/pmc/articles/PMC3834580/figure/F2/" class="imagepopup">Figure 2. EGF receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Future of targeted agents in metastatic colorectal cancer. </a></p><div class="supp"><p class="details"><div>Adapted with permission from [69].</div></p></div><div class="aux"><div class="resc">Mauricio Burotto, et al. Colorectal Cancer. ;1(5):10.2217/crc.12.52.</div><p class="links"><a class="dblinks" href="/pubmed/24273599" ref="ordinalpos=3277&amp;ncbi_uid=5508094&amp;link_uid=24273599">Citation</a><a class="dblinks" href="/pmc/articles/PMC3834580/" ref="ordinalpos=3277&amp;ncbi_uid=5508094&amp;link_uid=PMC3834580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3278.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3151497/figure/fig3/" ref="ordinalpos=3278&amp;ncbi_uid=3154494&amp;link_uid=PMC3151497" href="/pmc/articles/PMC3151497/"><img src="/pmc/articles/instance/3151497/bin/CFG2011-245137.003.gif" src-large="/pmc/articles/instance/3151497/bin/CFG2011-245137.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3151497/" ref="ordinalpos=3278&amp;ncbi_uid=3154494&amp;link_uid=PMC3151497" image-link="/pmc/articles/PMC3151497/figure/fig3/" class="imagepopup">Figure 3.  From: Gene Expression Profiling in Human High-Grade Astrocytomas. </a></p><div class="supp"><p class="details"><div>MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Red stars marks indicate dysregulated genes.</div></p></div><div class="aux"><div class="resc">Zhongyu Liu, et al. Comp Funct Genomics. 2011;2011:245137.</div><p class="links"><a class="dblinks" href="/pubmed/21836821" ref="ordinalpos=3278&amp;ncbi_uid=3154494&amp;link_uid=21836821">Citation</a><a class="dblinks" href="/pmc/articles/PMC3151497/" ref="ordinalpos=3278&amp;ncbi_uid=3154494&amp;link_uid=PMC3151497">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3279.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2668775/figure/F1/" ref="ordinalpos=3279&amp;ncbi_uid=1857602&amp;link_uid=PMC2668775" href="/pmc/articles/PMC2668775/"><img src="/pmc/articles/instance/2668775/bin/WJG-15-1702-g001.gif" src-large="/pmc/articles/instance/2668775/bin/WJG-15-1702-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2668775/" ref="ordinalpos=3279&amp;ncbi_uid=1857602&amp;link_uid=PMC2668775" image-link="/pmc/articles/PMC2668775/figure/F1/" class="imagepopup">Figure 1.  From: Reactive oxygen species: A double-edged sword in oncogenesis. </a></p><div class="supp"><p class="details"><div>Roles of ROS in the Ras-Raf-MEK-ERK <span class="highlight" style="background-color:">signaling</span> and p38 <span class="highlight" style="background-color:">pathway</span> (Modified from Ref. 31, with permission).</div></p></div><div class="aux"><div class="resc">Jin-Shui Pan, et al. World J Gastroenterol. 2009 April 14;15(14):1702-1707.</div><p class="links"><a class="dblinks" href="/pubmed/19360913" ref="ordinalpos=3279&amp;ncbi_uid=1857602&amp;link_uid=19360913">Citation</a><a class="dblinks" href="/pmc/articles/PMC2668775/" ref="ordinalpos=3279&amp;ncbi_uid=1857602&amp;link_uid=PMC2668775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3280.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2771945/figure/fig1-1756285608100427/" ref="ordinalpos=3280&amp;ncbi_uid=2640579&amp;link_uid=PMC2771945" href="/pmc/articles/PMC2771945/"><img src="/pmc/articles/instance/3002618/bin/10.1177_1756285608100427-fig1.gif" src-large="/pmc/articles/instance/3002618/bin/10.1177_1756285608100427-fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2771945/" ref="ordinalpos=3280&amp;ncbi_uid=2640579&amp;link_uid=PMC2771945" image-link="/pmc/articles/PMC2771945/figure/fig1-1756285608100427/" class="imagepopup">Figure 1. From: The Potential for Estrogens in Preventing Alzheimer's                     Disease and Vascular Dementia. </a></p><div class="supp"><p class="details"><div>The p25/cdk5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after ischemia/reperfusion injury and its                             correlation with AD.</div></p></div><div class="aux"><div class="resc">James W. Simpkins, et al. Ther Adv Neurol Disord. 2009 January;2(1):31-49.</div><p class="links"><a class="dblinks" href="/pubmed/19890493" ref="ordinalpos=3280&amp;ncbi_uid=2640579&amp;link_uid=19890493">Citation</a><a class="dblinks" href="/pmc/articles/PMC2771945/" ref="ordinalpos=3280&amp;ncbi_uid=2640579&amp;link_uid=PMC2771945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3281.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2783415/figure/F4/" ref="ordinalpos=3281&amp;ncbi_uid=2557425&amp;link_uid=PMC2783415" href="/pmc/articles/PMC2783415/"><img src="/pmc/articles/instance/2783415/bin/nihms151949f4.gif" src-large="/pmc/articles/instance/2783415/bin/nihms151949f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783415/" ref="ordinalpos=3281&amp;ncbi_uid=2557425&amp;link_uid=PMC2783415" image-link="/pmc/articles/PMC2783415/figure/F4/" class="imagepopup">Figure 4.  From: Interplay of CCR2 <span class="highlight" style="background-color:">signaling</span> and local shear force determines vein graft neointimal hyperplasia in vivo. </a></p><div class="supp"><p class="details"><div>Flow regulated neointimal hyperplasia in CCR2 -/- and wild type vein grafts. While reduction in flow accelerated neointimal hyperplasia in wild type vein grafts (P=0.01), this augmentation was abolished when CCR2 was absent. Absence of the CCR2 receptor had an influence only under low flow conditions, where significant attenuation in neointimal hyperplasia was observed with loss of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (P=0.04).</div></p></div><div class="aux"><div class="resc">Zhihua Jiang, et al. FEBS Lett. ;583(21):3536-3540.</div><p class="links"><a class="dblinks" href="/pubmed/19822149" ref="ordinalpos=3281&amp;ncbi_uid=2557425&amp;link_uid=19822149">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783415/" ref="ordinalpos=3281&amp;ncbi_uid=2557425&amp;link_uid=PMC2783415">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3282.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC150933/figure/F2/" ref="ordinalpos=3282&amp;ncbi_uid=241234&amp;link_uid=PMC150933" href="/pmc/articles/PMC150933/"><img src="/pmc/articles/instance/150933/bin/JCI0214685.f2.gif" src-large="/pmc/articles/instance/150933/bin/JCI0214685.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC150933/" ref="ordinalpos=3282&amp;ncbi_uid=241234&amp;link_uid=PMC150933" image-link="/pmc/articles/PMC150933/figure/F2/" class="imagepopup">Figure 2.  From: IL-7 inhibits fibroblast TGF-? production and <span class="highlight" style="background-color:">signaling</span> in pulmonary fibrosis. </a></p><div class="supp"><p class="details"><div>IL-7R <span class="highlight" style="background-color:">signaling</span> requires an intact JAK1/STAT1 <span class="highlight" style="background-color:">pathway</span>. U4A, U3A, and U4A/JAK1 fibroblast cell lines were cultured in medium containing variable doses of recombinant IL-7 for a 24-hour incubation. Culture supernatants were assayed for TGF-β by ELISA. Whereas TGF-β production remained unchanged in the U4A and U3A cells, IL-7 decreased TGF-β production in a dose-dependent manner in U4A/JAK1 cells (<i>P</i> &lt; 0.01 at 50–400 ng/ml IL-7).</div></p></div><div class="aux"><div class="resc">Min Huang, et al. J Clin Invest. 2002 April 1;109(7):931-937.</div><p class="links"><a class="dblinks" href="/pubmed/11927620" ref="ordinalpos=3282&amp;ncbi_uid=241234&amp;link_uid=11927620">Citation</a><a class="dblinks" href="/pmc/articles/PMC150933/" ref="ordinalpos=3282&amp;ncbi_uid=241234&amp;link_uid=PMC150933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3283.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3344210/figure/f4-ijms-13-04202/" ref="ordinalpos=3283&amp;ncbi_uid=4008753&amp;link_uid=PMC3344210" href="/pmc/articles/PMC3344210/"><img src="/pmc/articles/instance/3344210/bin/ijms-13-04202f4.gif" src-large="/pmc/articles/instance/3344210/bin/ijms-13-04202f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3344210/" ref="ordinalpos=3283&amp;ncbi_uid=4008753&amp;link_uid=PMC3344210" image-link="/pmc/articles/PMC3344210/figure/f4-ijms-13-04202/" class="imagepopup">Figure 4.  From: Scientific Evidence and Rationale for the Development of Curcumin and Resveratrol as Nutraceutricals for Joint Health. </a></p><div class="supp"><p class="details"><div>Schematic of the effects of curcumin on the TNF-α receptor and its downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The biochemical <span class="highlight" style="background-color:">pathway</span> illustrated here was generated by text mining and makes use of a collection of canonical Ariadne pathways in addition to MedScan text mining.</div></p></div><div class="aux"><div class="resc">Ali Mobasheri, et al. Int J Mol Sci. 2012;13(4):4202-4232.</div><p class="links"><a class="dblinks" href="/pubmed/22605974" ref="ordinalpos=3283&amp;ncbi_uid=4008753&amp;link_uid=22605974">Citation</a><a class="dblinks" href="/pmc/articles/PMC3344210/" ref="ordinalpos=3283&amp;ncbi_uid=4008753&amp;link_uid=PMC3344210">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3284.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC524168/figure/F4/" ref="ordinalpos=3284&amp;ncbi_uid=545184&amp;link_uid=PMC524168" href="/pmc/articles/PMC524168/"><img src="/pmc/articles/instance/524168/bin/1471-213X-4-11-4.gif" src-large="/pmc/articles/instance/524168/bin/1471-213X-4-11-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC524168/" ref="ordinalpos=3284&amp;ncbi_uid=545184&amp;link_uid=PMC524168" image-link="/pmc/articles/PMC524168/figure/F4/" class="imagepopup">Figure 4.  From: Regulation of <span class="highlight" style="background-color:">signaling</span> genes by TGF? during entry into dauer diapause in C. elegans. </a></p><div class="supp"><p class="details"><div><b>Model for gene regulatory events under the control of TGFβ <span class="highlight" style="background-color:">signaling</span>. </b>Transcriptional regulatory events suggested by expression data in this paper are indicated in red. The TGFβ <span class="highlight" style="background-color:">pathway</span> is shown in green; DAF-3 and DAF-5 form a transcription factor complex that function in neurons to control dauer entry [9]. DAF-3 and DAF-5 are shown regulating DAF-9 and DAF-12 indirectly because these genes are likely to be regulated in non-neuronal tissues. Insulin genes are shown as directly regulated by DAF-3 and DAF-5, but it is equally likely that these genes are regulated by feedback from the insulin <span class="highlight" style="background-color:">pathway</span> or by DAF-12.</div></p></div><div class="aux"><div class="resc">Tao Liu, et al. BMC Dev Biol. 2004;4:11-11.</div><p class="links"><a class="dblinks" href="/pubmed/15380030" ref="ordinalpos=3284&amp;ncbi_uid=545184&amp;link_uid=15380030">Citation</a><a class="dblinks" href="/pmc/articles/PMC524168/" ref="ordinalpos=3284&amp;ncbi_uid=545184&amp;link_uid=PMC524168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3285.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2919348/figure/F1/" ref="ordinalpos=3285&amp;ncbi_uid=2684015&amp;link_uid=PMC2919348" href="/pmc/articles/PMC2919348/"><img src="/pmc/articles/instance/2919348/bin/nihms223413f1.gif" src-large="/pmc/articles/instance/2919348/bin/nihms223413f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919348/" ref="ordinalpos=3285&amp;ncbi_uid=2684015&amp;link_uid=PMC2919348" image-link="/pmc/articles/PMC2919348/figure/F1/" class="imagepopup">Figure 1. The Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Yorkie: the final destination of Hippo <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic depictions of the regulatory interactions among genes linked to Hippo <span class="highlight" style="background-color:">signaling</span>. Pointed arrows indicate a positive regulatory connection, blocks indicate an inhibitory regulatory connection. Genes depicted in red can function as tumor suppressors (loss of function mutations can cause overgrowth), genes depicted in green can function as oncogenes (over expression and/or activation can cause overgrowth), genes depicted in black have not been shown to function as tmor suppressors or oncogenes. At the botom, genes depicted in gray encode DNA binding proteins that can partner with Yki/Yap to influence transcription of downstream target genes. Hippo <span class="highlight" style="background-color:">signaling</span> influences growth by influencing the transcription of target genes, but transcriptional targets that are not directly related to growth control have also been identified. A) Depicts genes and regulatory links identified in <i>Drosophila</i>. B) Depicts some of the genes identified in mammals. The Fat regulatory branch is shaded by a grey box to indicate that it is not yet known whether these genes actually influence Hpo <span class="highlight" style="background-color:">signaling</span> in mammals.</div></p></div><div class="aux"><div class="resc">Hyangyee Oh, et al. Trends Cell Biol. ;20(7):410-417.</div><p class="links"><a class="dblinks" href="/pubmed/20452772" ref="ordinalpos=3285&amp;ncbi_uid=2684015&amp;link_uid=20452772">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919348/" ref="ordinalpos=3285&amp;ncbi_uid=2684015&amp;link_uid=PMC2919348">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3286.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3133834/figure/f3/" ref="ordinalpos=3286&amp;ncbi_uid=3118379&amp;link_uid=PMC3133834" href="/pmc/articles/PMC3133834/"><img src="/pmc/articles/instance/3133834/bin/fig-3.gif" src-large="/pmc/articles/instance/3133834/bin/fig-3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3133834/" ref="ordinalpos=3286&amp;ncbi_uid=3118379&amp;link_uid=PMC3133834" image-link="/pmc/articles/PMC3133834/figure/f3/" class="imagepopup">FIG. 3. From: Aerosol Therapy for the Treatment of Osteosarcoma Lung Metastases: Targeting the Fas/FasL <span class="highlight" style="background-color:">Pathway</span> and Rationale for the Use of Gemcitabine. </a></p><div class="supp"><p class="details"><div>Blocking the Fas <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> allows Fas+ cells to induce lung metastases following i.v. injection. Fas+ K7 cells were transfected with FDN, Fas-associated death domain dominant negative or a control (neo) vector and injected into mice. Mice were euthanized 4 weeks later, and lung metastases were quantified.</div></p></div><div class="aux"><div class="resc">Nancy Gordon, et al. J Aerosol Med Pulm Drug Deliv. 2010 August;23(4):189-196.</div><p class="links"><a class="dblinks" href="/pubmed/20528149" ref="ordinalpos=3286&amp;ncbi_uid=3118379&amp;link_uid=20528149">Citation</a><a class="dblinks" href="/pmc/articles/PMC3133834/" ref="ordinalpos=3286&amp;ncbi_uid=3118379&amp;link_uid=PMC3133834">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3287.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3217239/figure/F8/" ref="ordinalpos=3287&amp;ncbi_uid=3411505&amp;link_uid=PMC3217239" href="/pmc/articles/PMC3217239/"><img src="/pmc/articles/instance/3217239/bin/nihms-336084-f0008.gif" src-large="/pmc/articles/instance/3217239/bin/nihms-336084-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3217239/" ref="ordinalpos=3287&amp;ncbi_uid=3411505&amp;link_uid=PMC3217239" image-link="/pmc/articles/PMC3217239/figure/F8/" class="imagepopup">Fig. 8.  From: ANG II induces apoptosis of human vascular smooth muscle via extrinsic <span class="highlight" style="background-color:">pathway</span> involving inhibition of Akt phosphorylation and increased FasL expression. </a></p><div class="supp"><p class="details"><div>Model of ANG II-induced apoptosis. ANG II-induced apoptosis of VSMC is characterized by marked inhibition of pAkt and pFKHR, and stimulation of extrinsic cell death <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via an increase in total FasL expression and the activity of caspase-8 and a reduction in soluble FasL expression.</div></p></div><div class="aux"><div class="resc">Yangxin Li, et al. Am J Physiol Heart Circ Physiol. ;290(5):H2116-H2123.</div><p class="links"><a class="dblinks" href="/pubmed/16339840" ref="ordinalpos=3287&amp;ncbi_uid=3411505&amp;link_uid=16339840">Citation</a><a class="dblinks" href="/pmc/articles/PMC3217239/" ref="ordinalpos=3287&amp;ncbi_uid=3411505&amp;link_uid=PMC3217239">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3288.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3441127/figure/DDS265F1/" ref="ordinalpos=3288&amp;ncbi_uid=5342703&amp;link_uid=PMC3441127" href="/pmc/articles/PMC3441127/"><img src="/pmc/articles/instance/3441127/bin/dds26501.gif" src-large="/pmc/articles/instance/3441127/bin/dds26501.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3441127/" ref="ordinalpos=3288&amp;ncbi_uid=5342703&amp;link_uid=PMC3441127" image-link="/pmc/articles/PMC3441127/figure/DDS265F1/" class="imagepopup">Figure 1. From: Abnormal p38? mitogen-activated protein kinase <span class="highlight" style="background-color:">signaling</span> in dilated cardiomyopathy caused by lamin A/C gene mutation. </a></p><div class="supp"><p class="details"><div>Altered expression of genes in the p38α branch of the MAP kinase <span class="highlight" style="background-color:">signaling</span> cascade in hearts from <i>Lmna</i><sup>H222P/H222P</sup> compared with wild-type mice at 10 weeks of age. (<b>A</b>) Representation of genes in the MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) identified by the KEGG visual <span class="highlight" style="background-color:">pathway</span> resource, which includes the ERK1/2 (ERK <span class="highlight" style="background-color:">signaling</span>), JNK (JNK <span class="highlight" style="background-color:">signaling</span>) and p38α (p38 <span class="highlight" style="background-color:">signaling</span>) branches within this GO term. Genes with statistically significant differences (<i>q</i>&lt; 0.05) in expression detected on Affymetrix Mouse Genome 430 2.0 Arrays in hearts of <i>Lmna</i><sup>H222P/H222P</sup> mice compared with wild-type mice are indicated by asterisks; red asterisks indicate genes in the p38α branch. (<b>B</b>) Matrices showing array data of corresponding probe sets corresponding to 10 genes in the in the p38α branch with statistically significant differences (<i>q</i>&lt; 0.05) in expression between wild-type (<i>Lmna</i><sup>+/+</sup>) and <i>Lmna</i><sup>H222P/H222P</sup> mouse hearts. In the matrices, each probe set is visualized as a row of colored squares and each column is a biological replicate (sample form a different mouse heart). Darker color means lower expression and brighter color higher expression. The genes correspond to those indicated with red asterisks in (A) [<i>DDIT3</i>/CHOP10 is the same as GADD153 and <i>MAP2K6</i>/MEK6 the same as MKK6 in (A)]. (<b>C</b>) Validation of differential expression in hearts from <i>Lmna</i><sup>H222P/H222P</sup> mice of four selected genes in the p38α MAP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> using real-time quantitative RT–PCR. RNA was obtained from hearts of mice at different ages as indicated on the <i>x</i>-axis. White bars show relative RNA expression levels in hearts of <i>Lmna</i><sup>+/+</sup> mice and black bars in hearts of <i>Lmna</i><sup>H222P/H222P</sup> mice. Values are means ± SEM for <i>n</i>= 3 samples per group. The real-time quantitative RT–PCR was performed in triplicate with the different RNA samples. *<i>P</i>&lt; 0.05, **<i>P</i>&lt; 0.005, ***<i>P</i>&lt; 0.0005.</div></p></div><div class="aux"><div class="resc">Antoine Muchir, et al. Hum Mol Genet. 2012 October 1;21(19):4325-4333.</div><p class="links"><a class="dblinks" href="/pubmed/22773734" ref="ordinalpos=3288&amp;ncbi_uid=5342703&amp;link_uid=22773734">Citation</a><a class="dblinks" href="/pmc/articles/PMC3441127/" ref="ordinalpos=3288&amp;ncbi_uid=5342703&amp;link_uid=PMC3441127">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3289.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2773873/figure/F2/" ref="ordinalpos=3289&amp;ncbi_uid=2079985&amp;link_uid=PMC2773873" href="/pmc/articles/PMC2773873/"><img src="/pmc/articles/instance/2773873/bin/WJG-14-6786-g002.gif" src-large="/pmc/articles/instance/2773873/bin/WJG-14-6786-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2773873/" ref="ordinalpos=3289&amp;ncbi_uid=2079985&amp;link_uid=PMC2773873" image-link="/pmc/articles/PMC2773873/figure/F2/" class="imagepopup">Figure 2.  From: Dynamic localization of hepatocellular transporters in health and disease. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways that regulate the cAMP-induced exocytic insertion of Ntcp into the basolateral membrane. cAMP stimulatory effect involves elevations in cytosolic Ca<sup>2+</sup> and activation of PI3K-dependent <span class="highlight" style="background-color:">pathway</span>, probably <i>via</i> protein kinase A (PKA). CaM complex activates phosphatase 2B (PP2B), which promotes insertion of Ntcp by dephosphorylation. This <span class="highlight" style="background-color:">pathway</span> is counter-regulated by cPKC. cAMP also stimulates Ntcp targeting by PI3K-dependent activation of PDK1 and subsequent PKB activation. Alternatively, PKB is activated by the concerted action of the atypical PKCζ and PDK2. Finally, cAMP/PI3K <span class="highlight" style="background-color:">signaling</span> stimulatory <span class="highlight" style="background-color:">pathway</span> may involve PKCδ.</div></p></div><div class="aux"><div class="resc">Marcelo G Roma, et al. World J Gastroenterol. 2008 November 28;14(44):6786-6801.</div><p class="links"><a class="dblinks" href="/pubmed/19058304" ref="ordinalpos=3289&amp;ncbi_uid=2079985&amp;link_uid=19058304">Citation</a><a class="dblinks" href="/pmc/articles/PMC2773873/" ref="ordinalpos=3289&amp;ncbi_uid=2079985&amp;link_uid=PMC2773873">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3290.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2277371/figure/F6/" ref="ordinalpos=3290&amp;ncbi_uid=1388418&amp;link_uid=PMC2277371" href="/pmc/articles/PMC2277371/"><img src="/pmc/articles/instance/2277371/bin/nihms-35656-f0006.gif" src-large="/pmc/articles/instance/2277371/bin/nihms-35656-f0006.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2277371/" ref="ordinalpos=3290&amp;ncbi_uid=1388418&amp;link_uid=PMC2277371" image-link="/pmc/articles/PMC2277371/figure/F6/" class="imagepopup">Fig. 6.  From: Proteolytic cleavage of Chordin as a switch for the dual activities of Twisted gastrulation in BMP <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Model for a biochemical <span class="highlight" style="background-color:">pathway</span> that regulates BMP <span class="highlight" style="background-color:">signaling</span> in the extracellular space. Tsg-BMP complexes can be bound by full-length Chordin to form a ternary complex that is a potent BMP antagonist. Xolloid cleaves Chordin releasing Tsg-BMP binary complexes and Chordin fragments. In the presence of full-length Chordin, the binary complex will re-bind to Chordin, reforming the ternary complex. After all full-length Chordin is cleaved by Xolloid, however, Tsg is able to dislodge BMP from Chordin and to destabilize the Chordin proteolytic products, displacing the equilibrium. This model explains why Tsg has the dual ability to increase BMP antagonism by full-length Chordin and to promote BMP <span class="highlight" style="background-color:">signaling</span> after Xolloid cleavage.</div></p></div><div class="aux"><div class="resc">Juan Larraín, et al. Development. ;128(22):4439-4447.</div><p class="links"><a class="dblinks" href="/pubmed/11714670" ref="ordinalpos=3290&amp;ncbi_uid=1388418&amp;link_uid=11714670">Citation</a><a class="dblinks" href="/pmc/articles/PMC2277371/" ref="ordinalpos=3290&amp;ncbi_uid=1388418&amp;link_uid=PMC2277371">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3291.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4384750/figure/fig9/" ref="ordinalpos=3291&amp;ncbi_uid=7392853&amp;link_uid=PMC4384750" href="/pmc/articles/PMC4384750/"><img src="/pmc/articles/instance/4384750/bin/elife06132f009.gif" src-large="/pmc/articles/instance/4384750/bin/elife06132f009.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4384750/" ref="ordinalpos=3291&amp;ncbi_uid=7392853&amp;link_uid=PMC4384750" image-link="/pmc/articles/PMC4384750/figure/fig9/" class="imagepopup">Figure 9.Model of SOX2 feedback <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Inhibition of mutant EGFR in lung cancer cells triggers SOX2-FOXO6-dependent survival pathways. </a></p><div class="supp"><p class="details"><div>In untreated cells, mutant EGFR drives cell survival by activating downstream <span class="highlight" style="background-color:">signaling</span> pathways, including PI3K and MAPK, which inhibit apoptosis through transcriptional and post-transcriptional effects on BH3-domain proteins, including pro-apoptotic BIM and BMF. In most cells (red lines), erlotinib treatment results in EGFR inhibition, inhibition of downstream <span class="highlight" style="background-color:">signaling</span> and increased pro-apoptotic proteins, leading to apoptosis. The high SOX2 induced by erlotinib through activation of FOXO6 in some cells (blue lines) counteracts the pro-apoptotic effects of EGFR inhibition, sufficiently decreasing the levels of BIM and BMF to delay the apoptotic response.<br /><b>DOI:</b> http://dx.doi.org/10.7554/eLife.06132.045</div></p></div><div class="aux"><div class="resc">S Michael Rothenberg, et al. eLife. 2015;4:e06132.</div><p class="links"><a class="dblinks" href="/pubmed/25686219" ref="ordinalpos=3291&amp;ncbi_uid=7392853&amp;link_uid=25686219">Citation</a><a class="dblinks" href="/pmc/articles/PMC4384750/" ref="ordinalpos=3291&amp;ncbi_uid=7392853&amp;link_uid=PMC4384750">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3292.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC196055/figure/F7/" ref="ordinalpos=3292&amp;ncbi_uid=238149&amp;link_uid=PMC196055" href="/pmc/articles/PMC196055/"><img src="/pmc/articles/instance/196055/bin/64236-02f7_L1OT.gif" src-large="/pmc/articles/instance/196055/bin/64236-02f7_L1OT.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC196055/" ref="ordinalpos=3292&amp;ncbi_uid=238149&amp;link_uid=PMC196055" image-link="/pmc/articles/PMC196055/figure/F7/" class="imagepopup">Figure 7.  From: A complex of LIN-5 and GPR proteins regulates G protein <span class="highlight" style="background-color:">signaling</span> and spindle function in C. elegans. </a></p><div class="supp"><p class="details"><div>A model for GPR-mediated initiation of G protein <span class="highlight" style="background-color:">signaling</span>. (<i>A</i>) A <span class="highlight" style="background-color:">pathway</span> for spindle force regulation by LIN-5/GPR/Gα, which may be locally regulated by polarity genes. (<i>B</i>) Classical heterotrimeric G protein activation. Exchange of GDP for GTP associated with Gα is stimulated by the guanine nucleotide exchange activity of a GPCR. The heterotrimer is dissociated into two independent <span class="highlight" style="background-color:">signaling</span> modules, Gα·GTP and Gβγ, which are then competent to activate downstream effectors. (<i>C</i>) Activation of G protein <span class="highlight" style="background-color:">signaling</span> in the absence of a GPCR. A GoLoco/GPR motif protein, such as Pins, could compete with and dissociate Gβγ from the heterotrimer, thus initiating Gβγ <span class="highlight" style="background-color:">signaling</span>. It is not known if Gα·GDP/Pins is competent to signal. (<i>D</i>) Receptor-independent initiation of Gα·GTP <span class="highlight" style="background-color:">signaling</span>. The GPR-1/GPR-2 GoLoco proteins could dissociate Gβγ from the heterotrimer and allow the subsequent production of Gα·GTP through the activity of a GEF such as RIC-8. Additionally, free Gβγ could regulate its specific effectors. The signal could be attenuated by intrinsic or GAP-stimulated GTP hydrolysis, allowing for cycling activation of this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Dayalan G. Srinivasan, et al. Genes Dev. 2003 May 15;17(10):1225-1239.</div><p class="links"><a class="dblinks" href="/pubmed/12730122" ref="ordinalpos=3292&amp;ncbi_uid=238149&amp;link_uid=12730122">Citation</a><a class="dblinks" href="/pmc/articles/PMC196055/" ref="ordinalpos=3292&amp;ncbi_uid=238149&amp;link_uid=PMC196055">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3293.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3047191/figure/F6/" ref="ordinalpos=3293&amp;ncbi_uid=2813493&amp;link_uid=PMC3047191" href="/pmc/articles/PMC3047191/"><img src="/pmc/articles/instance/3047191/bin/emm-43-101-g006.gif" src-large="/pmc/articles/instance/3047191/bin/emm-43-101-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3047191/" ref="ordinalpos=3293&amp;ncbi_uid=2813493&amp;link_uid=PMC3047191" image-link="/pmc/articles/PMC3047191/figure/F6/" class="imagepopup">Figure 6.  From: Ligation of CD40 receptor in human B lymphocytes triggers the 5-lipoxygenase <span class="highlight" style="background-color:">pathway</span> to produce reactive oxygen species and activate p38 MAPK. </a></p><div class="supp"><p class="details"><div>A schematic diagram showing the <span class="highlight" style="background-color:">signaling</span> pathways in the CD40-induced ROS production. The solid lines represent direct protein: protein associations and the dashed line represent the presence of intermediary steps. As shown, the NADPH oxidase <span class="highlight" style="background-color:">pathway</span> is presented in gray color.</div></p></div><div class="aux"><div class="resc">Yun Jung Ha, et al. Exp Mol Med. 2011 February 28;43(2):101-110.</div><p class="links"><a class="dblinks" href="/pubmed/21200133" ref="ordinalpos=3293&amp;ncbi_uid=2813493&amp;link_uid=21200133">Citation</a><a class="dblinks" href="/pmc/articles/PMC3047191/" ref="ordinalpos=3293&amp;ncbi_uid=2813493&amp;link_uid=PMC3047191">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3294.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2246166/figure/F7/" ref="ordinalpos=3294&amp;ncbi_uid=1300936&amp;link_uid=PMC2246166" href="/pmc/articles/PMC2246166/"><img src="/pmc/articles/instance/2246166/bin/bcr1840-7.gif" src-large="/pmc/articles/instance/2246166/bin/bcr1840-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2246166/" ref="ordinalpos=3294&amp;ncbi_uid=1300936&amp;link_uid=PMC2246166" image-link="/pmc/articles/PMC2246166/figure/F7/" class="imagepopup">Figure 7.  From: Effects of high-fat diet and/or body weight on mammary tumor leptin and apoptosis <span class="highlight" style="background-color:">signaling</span> pathways in MMTV-TGF-? mice. </a></p><div class="supp"><p class="details"><div>Densitometric analysis of apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein expression in mammary tumor (MT) and mammary fat pad (MFP) from MMTV-TGF-α mice. <b>(a) </b>MT samples obtained from three Low-Fat and six High-Fat mice. <b>(b) </b>MFP samples obtained from five Low-Fat and four High-Fat mice. *<i>p </i>&lt; 0.05. pAkt, phosphorylated Akt.</div></p></div><div class="aux"><div class="resc">Soner Dogan, et al. Breast Cancer Res. 2007;9(6):R91-R91.</div><p class="links"><a class="dblinks" href="/pubmed/18162139" ref="ordinalpos=3294&amp;ncbi_uid=1300936&amp;link_uid=18162139">Citation</a><a class="dblinks" href="/pmc/articles/PMC2246166/" ref="ordinalpos=3294&amp;ncbi_uid=1300936&amp;link_uid=PMC2246166">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3295.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3070154/figure/F1/" ref="ordinalpos=3295&amp;ncbi_uid=3578131&amp;link_uid=PMC3070154" href="/pmc/articles/PMC3070154/"><img src="/pmc/articles/instance/3070154/bin/nihms240504f1.gif" src-large="/pmc/articles/instance/3070154/bin/nihms240504f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3070154/" ref="ordinalpos=3295&amp;ncbi_uid=3578131&amp;link_uid=PMC3070154" image-link="/pmc/articles/PMC3070154/figure/F1/" class="imagepopup">Figure 1. Cross-talk between estrogen and leptin <span class="highlight" style="background-color:">signaling</span> in POMC neurons.  From: Estrogen <span class="highlight" style="background-color:">Signaling</span> in Hypothalamic Circuits Controling Reproduction. </a></p><div class="supp"><p class="details"><div>Through a putative GPCR (mER), E2 (and STX) activates a PLCβ-PKCδ-AC-PKA <span class="highlight" style="background-color:">pathway</span> in POMC neurons that attenuates the coupling of μ-opioid and GABA<sub>B</sub> receptors to GIRK channels (heterologous desensitization) and triggers the release of Ca<sup>2+</sup> from intracellular stores. This novel <span class="highlight" style="background-color:">pathway</span> also activates PKA-pCREB mediated transcription and in tandem with ERE-mediated transcription will alter the mRNA expression of relevant genes (channels and <span class="highlight" style="background-color:">signaling</span> molecules, etc.). Leptin binds to LRb to activate PI3 kinase and PLCγ1 to augment TRPC-channel currents and POMC neuronal excitability. Leptin will also activate JAK2-STAT3 transcriptional pathways. Note: The guinea pig, similar to the primate, has all of the key cellular elements in the mediobasal hypothalamus for controlling reproduction and energy homeostasis.</div></p></div><div class="aux"><div class="resc">Martin J. Kelly, et al. Brain Res. ;1364:44-52.</div><p class="links"><a class="dblinks" href="/pubmed/20807512" ref="ordinalpos=3295&amp;ncbi_uid=3578131&amp;link_uid=20807512">Citation</a><a class="dblinks" href="/pmc/articles/PMC3070154/" ref="ordinalpos=3295&amp;ncbi_uid=3578131&amp;link_uid=PMC3070154">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3296.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC139542/figure/fig10/" ref="ordinalpos=3296&amp;ncbi_uid=354625&amp;link_uid=PMC139542" href="/pmc/articles/PMC139542/"><img src="/pmc/articles/instance/139542/bin/01-0008f10.gif" src-large="/pmc/articles/instance/139542/bin/01-0008f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC139542/" ref="ordinalpos=3296&amp;ncbi_uid=354625&amp;link_uid=PMC139542" image-link="/pmc/articles/PMC139542/figure/fig10/" class="imagepopup">Figure 10. From: Overexpression of the Heterotrimeric G-Protein ?-Subunit Enhances Phytochrome-Mediated Inhibition of Hypocotyl Elongation in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Schemes of Gα Interaction with phyA and phyB Signal Transduction.<br /><b>(A)</b> The FR light <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by phyA branches after photoperception. The heterotrimeric G-protein is involved in the <span class="highlight" style="background-color:">pathway</span> defined by FHY1 but not in the <span class="highlight" style="background-color:">pathway</span> that requires FHY3 and FIN219.<br /><b>(B)</b> phyB mediates R light–dependent hypocotyl regulation and the EODFR response. The heterotrimeric Gα protein is involved in the R light inhibition of hypocotyl elongation but not in the EODFR response.</div></p></div><div class="aux"><div class="resc">Haruko Okamoto, et al. Plant Cell. 2001 July;13(7):1639-1652.</div><p class="links"><a class="dblinks" href="/pubmed/11449056" ref="ordinalpos=3296&amp;ncbi_uid=354625&amp;link_uid=11449056">Citation</a><a class="dblinks" href="/pmc/articles/PMC139542/" ref="ordinalpos=3296&amp;ncbi_uid=354625&amp;link_uid=PMC139542">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3297.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4016833/figure/fig1/" ref="ordinalpos=3297&amp;ncbi_uid=6167532&amp;link_uid=PMC4016833" href="/pmc/articles/PMC4016833/"><img src="/pmc/articles/instance/4016833/bin/BMRI2014-679168.001.gif" src-large="/pmc/articles/instance/4016833/bin/BMRI2014-679168.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4016833/" ref="ordinalpos=3297&amp;ncbi_uid=6167532&amp;link_uid=PMC4016833" image-link="/pmc/articles/PMC4016833/figure/fig1/" class="imagepopup">Figure 1.  From: Building and Repairing the Heart: What Can We Learn from Embryonic Development?. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> events in murine heart development and cardiac ESC differentiation. In both systems, mesodermal induction from the epiblast is regulated by Wnt/<i>β</i>-catenin, Nodal/Activin, and BMP <span class="highlight" style="background-color:">signaling</span> pathways and correlates with Brachyury upregulation. Further commitment of mesodermal progenitors to cardiac mesoderm and consequent first heart field (FHF) formation require the inhibition of Wnt <span class="highlight" style="background-color:">signaling</span> and expression of BMPs. Similarly, in ESCs system, Notch <span class="highlight" style="background-color:">pathway</span> inhibits Wnt/<i>β</i>-catenin <span class="highlight" style="background-color:">signaling</span> and activates BMP to specify cardiac fates. Wnt/<i>β</i>-catenin <span class="highlight" style="background-color:">signaling</span> is then activated to allow proliferation and maintenance of the SHF, both in embryo and ESCs. Further differentiation from the cardiac crescent stage to the following morphogenic phases of embryonic heart development and, in parallel, the expression of cardiomyocyte differentiation genes in ESCs require inhibition of Wnt/<i>β</i>-catenin. In the embryo and ESCs, this is achieved by Notch and noncanonical Wnt <span class="highlight" style="background-color:">signaling</span>, which inhibit the effect of Wnt/<i>β</i>-catenin and instruct progenitor cells within the SHF to leave the proliferative state and start differentiating. ⊣ represents inhibitory effect; ⤾ represents maintenance of a proliferative state.</div></p></div><div class="aux"><div class="resc">Ana G. Freire, et al. Biomed Res Int. 2014;2014:679168.</div><p class="links"><a class="dblinks" href="/pubmed/24864252" ref="ordinalpos=3297&amp;ncbi_uid=6167532&amp;link_uid=24864252">Citation</a><a class="dblinks" href="/pmc/articles/PMC4016833/" ref="ordinalpos=3297&amp;ncbi_uid=6167532&amp;link_uid=PMC4016833">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3298.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2919351/figure/F1/" ref="ordinalpos=3298&amp;ncbi_uid=3057919&amp;link_uid=PMC2919351" href="/pmc/articles/PMC2919351/"><img src="/pmc/articles/instance/2919351/bin/nihms-221725-f0001.gif" src-large="/pmc/articles/instance/2919351/bin/nihms-221725-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919351/" ref="ordinalpos=3298&amp;ncbi_uid=3057919&amp;link_uid=PMC2919351" image-link="/pmc/articles/PMC2919351/figure/F1/" class="imagepopup">Figure 1. Hexosamine Biosynthetic <span class="highlight" style="background-color:">Pathway</span>.  From: O-GlcNAc <span class="highlight" style="background-color:">Signaling</span> in the Cardiovascular System. </a></p><div class="supp"><p class="details"><div>Phosphorylated glucose enters either the glycogen synthetic <span class="highlight" style="background-color:">pathway</span> or is further converted to fructose-6-phosphate by glucose-6-phosphate isomerase. The majority of fructose-6-phosphate is channeled to glycolysis. Less than 5% of glucose uptake is ultimately channeled to a unique accessory <span class="highlight" style="background-color:">pathway</span> for glucose metabolism, the hexosamine biosynthetic <span class="highlight" style="background-color:">pathway</span> (HBP). This <span class="highlight" style="background-color:">pathway</span> begins with the rate-limiting enzyme, L-glutamine:D-fructose-6-phosphate amidotransferase (GFAT), followed by acetylation of gluocsamine-6-phosphate by glucosamine-6-phosphate acetyltransferase (Emeg32) to N-acetylglucosamine-6-phosphate (GlcNAc-6-P). Next are two reversible reactions: the conversion of GlcNAc-6-P to GlcNAc-1-P by phosphate-acetylglucosamine mutase and then the formation uridine diphosphate-GlcNAc (UDP-GlcNAc) by UDP-GlcNAc pyrophosphorylase. This high-energy molecule serves as the monosaccharide donor for the post-translational modification by O-GlcNAc transferase (OGT). O-GlcNAcase (OGA) removes O-GlcNAc modification from proteins.</div></p></div><div class="aux"><div class="resc">Gladys A. Ngoh, et al. Circ Res. ;107(2):171-185.</div><p class="links"><a class="dblinks" href="/pubmed/20651294" ref="ordinalpos=3298&amp;ncbi_uid=3057919&amp;link_uid=20651294">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919351/" ref="ordinalpos=3298&amp;ncbi_uid=3057919&amp;link_uid=PMC2919351">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3299.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3349672/figure/pone-0035584-g004/" ref="ordinalpos=3299&amp;ncbi_uid=4003610&amp;link_uid=PMC3349672" href="/pmc/articles/PMC3349672/"><img src="/pmc/articles/instance/3349672/bin/pone.0035584.g004.gif" src-large="/pmc/articles/instance/3349672/bin/pone.0035584.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3349672/" ref="ordinalpos=3299&amp;ncbi_uid=4003610&amp;link_uid=PMC3349672" image-link="/pmc/articles/PMC3349672/figure/pone-0035584-g004/" class="imagepopup">Figure 4. Identification of <span class="highlight" style="background-color:">signaling</span> pathways and interaction of gene products associated with cellular response of cancer cells towards formulated arsenic trioxide..  From: Factors Determining Sensitivity and Resistance of Tumor Cells to Arsenic Trioxide. </a></p><div class="supp"><p class="details"><div>The genes identified by COMPARE analyses (<b>Table S1</b>) were subjected to Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis Software. (<b>A</b>) Top 10 categories of biological functions of the candidate genes. (<b>B</b>) Top 10 canonical <span class="highlight" style="background-color:">signaling</span> pathways, which the candidate genes were assigned to.</div></p></div><div class="aux"><div class="resc">Serkan Sertel, et al. PLoS One. 2012;7(5):e35584.</div><p class="links"><a class="dblinks" href="/pubmed/22590507" ref="ordinalpos=3299&amp;ncbi_uid=4003610&amp;link_uid=22590507">Citation</a><a class="dblinks" href="/pmc/articles/PMC3349672/" ref="ordinalpos=3299&amp;ncbi_uid=4003610&amp;link_uid=PMC3349672">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3300.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3106338/figure/F6/" ref="ordinalpos=3300&amp;ncbi_uid=3363626&amp;link_uid=PMC3106338" href="/pmc/articles/PMC3106338/"><img src="/pmc/articles/instance/3106338/bin/nihms291224f6.gif" src-large="/pmc/articles/instance/3106338/bin/nihms291224f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3106338/" ref="ordinalpos=3300&amp;ncbi_uid=3363626&amp;link_uid=PMC3106338" image-link="/pmc/articles/PMC3106338/figure/F6/" class="imagepopup">Figure 6.  From: INTERFERON-ALPHA MEDIATES THE DEVELOPMENT OF AUTOIMMUNITY BOTH BY DIRECT TISSUE TOXICITY AND THROUGH IMMUNE-CELL RECRUITMENT MECHANISMS. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) of RNAseq expression data. <b>(A)</b> IPA analysis of mRNA expression data from transgenic mouse thyroids. <span class="highlight" style="background-color:">Pathway</span> analysis was performed using the Ingenuity software as described in the methods. Genes that are upregulated are shown in dark gray and genes that are downregulated are shown in light gray. Significant upregulation of interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in TG mice is shown. In addition the antigen presentation, pattern recognition receptor, complement, and Granzyme B pathways are upregulated in TG mice. In contrast the fatty acid metabolism and arachidonic acid metabolism pathways are significantly downregulated. <b>(B)</b> IPA analysis of mRNA expression data from human thyrocytes exposed to IFNα for 24 hrs. As expected the interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is upregulated in cells exposed to IFNα. In addition, antigen presentation, complement, and pattern recognition receptor pathways are upregulated similar to the TG thyroids. Additional upregulated pathways include retinoic acid receptor <span class="highlight" style="background-color:">signaling</span> and retinoic acid mediated apoptosis pathways, while the cell cycle <span class="highlight" style="background-color:">pathway</span> is downregulated.</div></p></div><div class="aux"><div class="resc">Nagako Akeno, et al. J Immunol. ;186(8):4693-4706.</div><p class="links"><a class="dblinks" href="/pubmed/21402899" ref="ordinalpos=3300&amp;ncbi_uid=3363626&amp;link_uid=21402899">Citation</a><a class="dblinks" href="/pmc/articles/PMC3106338/" ref="ordinalpos=3300&amp;ncbi_uid=3363626&amp;link_uid=PMC3106338">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="32" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="33" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="34" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:41:33-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB4B71543116410000000000B9844B;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB4B71543116410000000000B9844B" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>