
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="26" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB17805431052100000000004AAEC3" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="25" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="26" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="27" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="26" /></div>
            <div><h2 class="result_count">Results: 2501 to 2600 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2501.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC385422/figure/F5/" ref="ordinalpos=2501&amp;ncbi_uid=512339&amp;link_uid=PMC385422" href="/pmc/articles/PMC385422/"><img src="/pmc/articles/instance/385422/bin/1471-2121-5-2-5.gif" src-large="/pmc/articles/instance/385422/bin/1471-2121-5-2-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC385422/" ref="ordinalpos=2501&amp;ncbi_uid=512339&amp;link_uid=PMC385422" image-link="/pmc/articles/PMC385422/figure/F5/" class="imagepopup">Figure 5.  From: Cloning of a novel <span class="highlight" style="background-color:">signaling</span> molecule, AMSH-2, that potentiates transforming growth factor ? <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>AMSH-2 RNA interference has a negative effect on TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </b>HepG2 cells were transfected with 3TP-lux reporter and scramble, laminA/C, AMSH-2 or Smad7 siRNA duplexes as indicated in the bottom of the graph. Forty-eight hours posttransfection cells were serum-starved and left treated (black bars) or not (white bars) with 5 ng/ml TGF-β1.</div></p></div><div class="aux"><div class="resc">Nieves Ibarrola, et al. BMC Cell Biol. 2004;5:2-2.</div><p class="links"><a class="dblinks" href="/pubmed/14728725" ref="ordinalpos=2501&amp;ncbi_uid=512339&amp;link_uid=14728725">Citation</a><a class="dblinks" href="/pmc/articles/PMC385422/" ref="ordinalpos=2501&amp;ncbi_uid=512339&amp;link_uid=PMC385422">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2502.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4229655/figure/F10/" ref="ordinalpos=2502&amp;ncbi_uid=6830543&amp;link_uid=PMC4229655" href="/pmc/articles/PMC4229655/"><img src="/pmc/articles/instance/4229655/bin/nihms573356f10.gif" src-large="/pmc/articles/instance/4229655/bin/nihms573356f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4229655/" ref="ordinalpos=2502&amp;ncbi_uid=6830543&amp;link_uid=PMC4229655" image-link="/pmc/articles/PMC4229655/figure/F10/" class="imagepopup">Figure 10.  From: Cell-to-cell <span class="highlight" style="background-color:">signaling</span> in E. coli and Salmonella. </a></p><div class="supp"><p class="details"><div>The AI-3/epi/NE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. AI-3, epinephrine, and norepinephrine bind the membrane receptor protein QseC, resulting in autophosphorylation. Subsequently, QseC phosphorylates its response regulator QseB, initiating a complex phosphorelay <span class="highlight" style="background-color:">signaling</span> cascade that results in expression of the flagellar biosynthesis and motility genes (<i>flhDC</i>), Shiga toxin (<i>stxAB</i>), the locus of enterocyte effacement (LEE), and a second two-component system, QseEF that also promotes expression of the LEE.</div></p></div><div class="aux"><div class="resc">Melissa M. Kendall, et al. Ecosal Plus. ;2014:10.1128/ecosalplus.ESP-0002-2013.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2502&amp;ncbi_uid=6830543&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4229655/" ref="ordinalpos=2502&amp;ncbi_uid=6830543&amp;link_uid=PMC4229655">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2503.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3497314/figure/f1-ijms-13-13104/" ref="ordinalpos=2503&amp;ncbi_uid=4398644&amp;link_uid=PMC3497314" href="/pmc/articles/PMC3497314/"><img src="/pmc/articles/instance/3497314/bin/ijms-13-13104f1.gif" src-large="/pmc/articles/instance/3497314/bin/ijms-13-13104f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3497314/" ref="ordinalpos=2503&amp;ncbi_uid=4398644&amp;link_uid=PMC3497314" image-link="/pmc/articles/PMC3497314/figure/f1-ijms-13-13104/" class="imagepopup">Figure 1.  From: Disease Progression Mediated by Egr-1 Associated <span class="highlight" style="background-color:">Signaling</span> in Response to Oxidative Stress. </a></p><div class="supp"><p class="details"><div>Overview of <span class="highlight" style="background-color:">signaling</span> partners involved in oxidative stress mediated Egr-1 <span class="highlight" style="background-color:">signaling</span>. Oxidative stress leads to Egr-1 activation (↑) and promotes atherosclerosis, diabetes, apoptosis and pulmonary hypertension. The MEK/ERK <span class="highlight" style="background-color:">pathway</span> is the main signal transduction cascade involved. Depending on the targeted cell type and <i>in vitro</i> or <i>in vivo</i> data, different elements are involved.</div></p></div><div class="aux"><div class="resc">Judith-Irina Pagel, et al. Int J Mol Sci. 2012;13(10):13104-13117.</div><p class="links"><a class="dblinks" href="/pubmed/23202940" ref="ordinalpos=2503&amp;ncbi_uid=4398644&amp;link_uid=23202940">Citation</a><a class="dblinks" href="/pmc/articles/PMC3497314/" ref="ordinalpos=2503&amp;ncbi_uid=4398644&amp;link_uid=PMC3497314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2504.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4049900/figure/F2/" ref="ordinalpos=2504&amp;ncbi_uid=6331768&amp;link_uid=PMC4049900" href="/pmc/articles/PMC4049900/"><img src="/pmc/articles/instance/4049900/bin/org-10-96-g2.gif" src-large="/pmc/articles/instance/4049900/bin/org-10-96-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4049900/" ref="ordinalpos=2504&amp;ncbi_uid=6331768&amp;link_uid=PMC4049900" image-link="/pmc/articles/PMC4049900/figure/F2/" class="imagepopup">Figure. From: Meckel-Gruber syndrome and the role of primary cilia in kidney, skeleton, and central nervous system development. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Hedgehog and Wnt <span class="highlight" style="background-color:">Signaling</span> in Kidney, Limb, and Neural Tube. A cartoon to show the effects of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on renal cystic disease, of the Hedgehog (Hh) <span class="highlight" style="background-color:">pathway</span> in the developing limb, and of both pathways in the neural tube. In the kidney, the balance between canonical (β-catenin dependent) and non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> must be maintained; dysregulation of either branch, whether up- or downregulation, leads to development of cysts. In the limb bud, a gradient of Hh <span class="highlight" style="background-color:">signaling</span> is required to pattern the digits; where upregulation of Hh occurs, the gradient is expanded and additional digits are specified whereas when Hh is downregulated, the gradient becomes restricted and a reduced number of digits is specified. In the neural tube, a Wnt (and, to perhaps a greater extent, BMP) gradient is involved in dorsal cell fate determination; the gradient is counteracted by a ventral Hh gradient. Up- or downregulation of either <span class="highlight" style="background-color:">pathway</span> perturbs this balance and affects axis specification in the developing neural tube. Wnt <span class="highlight" style="background-color:">signaling</span> also has important effects on neuronal migration in the developing brain, leading to numerous other CNS effects.</div></p></div><div class="aux"><div class="resc">Amy R Barker, et al. Organogenesis. 2014 January 1;10(1):96-107.</div><p class="links"><a class="dblinks" href="/pubmed/24322779" ref="ordinalpos=2504&amp;ncbi_uid=6331768&amp;link_uid=24322779">Citation</a><a class="dblinks" href="/pmc/articles/PMC4049900/" ref="ordinalpos=2504&amp;ncbi_uid=6331768&amp;link_uid=PMC4049900">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2505.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2806057/figure/F4/" ref="ordinalpos=2505&amp;ncbi_uid=2200983&amp;link_uid=PMC2806057" href="/pmc/articles/PMC2806057/"><img src="/pmc/articles/instance/2806057/bin/nihms111638f4.gif" src-large="/pmc/articles/instance/2806057/bin/nihms111638f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2806057/" ref="ordinalpos=2505&amp;ncbi_uid=2200983&amp;link_uid=PMC2806057" image-link="/pmc/articles/PMC2806057/figure/F4/" class="imagepopup">Figure 4. The migration-promoting activity of ASCs is controlled by the Cofilin-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Interleukin 6 Secreted from Adipose Stromal Cells Promotes Migration and Invasion of Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>(A)</b> A diagram illustrating the Cofilin-1-dependent <span class="highlight" style="background-color:">pathway</span>, adapted from Wang et al. (Wang et al., 2007). <b>(B)</b> siRNA knockdown of several players of the Cofilin-1 <span class="highlight" style="background-color:">pathway</span> reduced the migration-promoting activity of ASCs. <b>(C)</b> Pre-treatment of ASCs with the specific Rock inhibitor Y27632 reduced the migration-promoting activity of ASC in a dose-dependent manner. All experiments were carried out in triplicate and repeated at least 3 times.</div></p></div><div class="aux"><div class="resc">M Walter, et al. Oncogene. ;28(30):2745-2755.</div><p class="links"><a class="dblinks" href="/pubmed/19483720" ref="ordinalpos=2505&amp;ncbi_uid=2200983&amp;link_uid=19483720">Citation</a><a class="dblinks" href="/pmc/articles/PMC2806057/" ref="ordinalpos=2505&amp;ncbi_uid=2200983&amp;link_uid=PMC2806057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2506.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2957366/figure/F4/" ref="ordinalpos=2506&amp;ncbi_uid=2523981&amp;link_uid=PMC2957366" href="/pmc/articles/PMC2957366/"><img src="/pmc/articles/instance/2957366/bin/nihms188624f4a.gif" src-large="/pmc/articles/instance/2957366/bin/nihms188624f4a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2957366/figure/F4/" ref="ordinalpos=2506&amp;ncbi_uid=2523981&amp;link_uid=PMC2957366" href="/pmc/articles/PMC2957366/"><img src="/pmc/articles/instance/2957366/bin/nihms188624f4b.gif" src-large="/pmc/articles/instance/2957366/bin/nihms188624f4b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2957366/" ref="ordinalpos=2506&amp;ncbi_uid=2523981&amp;link_uid=PMC2957366" image-link="/pmc/articles/PMC2957366/figure/F4/" class="imagepopup">Fig. 4.  From: Gene expression profiling reveals signatures characterizing histologic subtypes of hepatoblastoma and global deregulation in cell growth and survival pathways. </a></p><div class="supp"><p class="details"><div>Analysis of genes differentially expressed between PF HB and epithelial HB with small cell component shows (A) significant perturbation of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> among others. MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes and their fold change are shown in (B).</div></p></div><div class="aux"><div class="resc">Adekunle M. Adesina, et al. Hum Pathol. ;40(6):843-853.</div><p class="links"><a class="dblinks" href="/pubmed/19200578" ref="ordinalpos=2506&amp;ncbi_uid=2523981&amp;link_uid=19200578">Citation</a><a class="dblinks" href="/pmc/articles/PMC2957366/" ref="ordinalpos=2506&amp;ncbi_uid=2523981&amp;link_uid=PMC2957366">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2507.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2544376/figure/F4/" ref="ordinalpos=2507&amp;ncbi_uid=1480163&amp;link_uid=PMC2544376" href="/pmc/articles/PMC2544376/"><img src="/pmc/articles/instance/2544376/bin/nihms59539f4.gif" src-large="/pmc/articles/instance/2544376/bin/nihms59539f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2544376/" ref="ordinalpos=2507&amp;ncbi_uid=1480163&amp;link_uid=PMC2544376" image-link="/pmc/articles/PMC2544376/figure/F4/" class="imagepopup">Figure 4. TGF-β and Ras <span class="highlight" style="background-color:">signaling</span> associated with vimentin expression.  From: Intestinal Adenomagenesis Involves Core Molecular Signatures of the Epithelial-Mesenchymal Transition. </a></p><div class="supp"><p class="details"><div><b>A</b>. No vimentin signal detected by IHC in a Smad-3-deficient 129 Min adenoma. <b>B</b>. Vimentin (brown) detected by IHC in a Smad-3 wildtype 129 Min adenoma. <i>C</i>omparison of the Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> indicated by (<b>C</b>) IHC for pMAPK and (<b>D</b>) vimentin expression by ISH in a B6 Min adenoma. <i>Black arrows</i>, regions with strong pMAPK staining and weak vimentin expression. <i>White-filled arrows</i>, regions with weak pMAPK staining and strong vimentin expression. Size bars, 200 µm.</div></p></div><div class="aux"><div class="resc">Xiaodi Chen, et al. J Mol Histol. ;39(3):283-294.</div><p class="links"><a class="dblinks" href="/pubmed/18327651" ref="ordinalpos=2507&amp;ncbi_uid=1480163&amp;link_uid=18327651">Citation</a><a class="dblinks" href="/pmc/articles/PMC2544376/" ref="ordinalpos=2507&amp;ncbi_uid=1480163&amp;link_uid=PMC2544376">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2508.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432438/figure/fig6/" ref="ordinalpos=2508&amp;ncbi_uid=4679902&amp;link_uid=PMC3432438" href="/pmc/articles/PMC3432438/"><img src="/pmc/articles/instance/3432438/bin/gr6.gif" src-large="/pmc/articles/instance/3432438/bin/gr6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432438/" ref="ordinalpos=2508&amp;ncbi_uid=4679902&amp;link_uid=PMC3432438" image-link="/pmc/articles/PMC3432438/figure/fig6/" class="imagepopup">Figure 6.  From: A Novel Function of p53. </a></p><div class="supp"><p class="details"><div>RV-engaged PDGFRα triggers two <span class="highlight" style="background-color:">signaling</span> pathways that drive cellular response intrinsic to PVR: <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> 1, leading to a reduction in the level of p53; this is sufficient for protection from apoptosis and senescence. Contraction and proliferation require an additional set of <span class="highlight" style="background-color:">signaling</span> events that constitute <span class="highlight" style="background-color:">pathway</span> 2. ROS, reactive oxygen species; SFK, src family kinases.</div></p></div><div class="aux"><div class="resc">Hetian Lei, et al. Am J Pathol. 2012 September;181(3):866-874.</div><p class="links"><a class="dblinks" href="/pubmed/22901751" ref="ordinalpos=2508&amp;ncbi_uid=4679902&amp;link_uid=22901751">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432438/" ref="ordinalpos=2508&amp;ncbi_uid=4679902&amp;link_uid=PMC3432438">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2509.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3862576/figure/pone-0082236-g004/" ref="ordinalpos=2509&amp;ncbi_uid=5611039&amp;link_uid=PMC3862576" href="/pmc/articles/PMC3862576/"><img src="/pmc/articles/instance/3862576/bin/pone.0082236.g004.gif" src-large="/pmc/articles/instance/3862576/bin/pone.0082236.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3862576/" ref="ordinalpos=2509&amp;ncbi_uid=5611039&amp;link_uid=PMC3862576" image-link="/pmc/articles/PMC3862576/figure/pone-0082236-g004/" class="imagepopup">Figure 4. Growth and survival <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation is refractory to ROS1 inhibition in HCC78-TR cells..  From: Resistance to ROS1 Inhibition Mediated by EGFR <span class="highlight" style="background-color:">Pathway</span> Activation in Non-Small Cell Lung Cancer. </a></p><div class="supp"><p class="details"><div>Cells were treated with TAE684 (A) or crizotinib (B) for 4 hours. Lysates of the cells were then analyzed by Western blot using the indicated antibodies.</div></p></div><div class="aux"><div class="resc">Kurtis D. Davies, et al. PLoS One. 2013;8(12):e82236.</div><p class="links"><a class="dblinks" href="/pubmed/24349229" ref="ordinalpos=2509&amp;ncbi_uid=5611039&amp;link_uid=24349229">Citation</a><a class="dblinks" href="/pmc/articles/PMC3862576/" ref="ordinalpos=2509&amp;ncbi_uid=5611039&amp;link_uid=PMC3862576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2510.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3360562/figure/F7/" ref="ordinalpos=2510&amp;ncbi_uid=4369221&amp;link_uid=PMC3360562" href="/pmc/articles/PMC3360562/"><img src="/pmc/articles/instance/3360562/bin/1070fig7.gif" src-large="/pmc/articles/instance/3360562/bin/1070fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3360562/" ref="ordinalpos=2510&amp;ncbi_uid=4369221&amp;link_uid=PMC3360562" image-link="/pmc/articles/PMC3360562/figure/F7/" class="imagepopup">Figure 7. From: Recruitment of sphingosine kinase to presynaptic terminals by a conserved muscarinic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promotes neurotransmitter release. </a></p><div class="supp"><p class="details"><div>Regulation of SPHK-1 synaptic abundance by the Gαq <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>,<i>B</i>) Representative images and quantification of SPHK-1-GFP fluorescence of DA motor axons in the indicated strains. (<i>C</i>,<i>D</i>) Rates of paralysis on aldicarb (1.0 mM) for the indicated strains. For <i>unc-73</i>, the <i>ce362</i> mutation was used. The indicated animals were pretreated with arecoline (15 mM in M9) or control (M9) for 2 h. (<i>E</i>) Model of the muscarinic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i> cholinergic motor neurons that regulates SPHK-1 synaptic abundance and neurotransmitter release. Established model of the G-protein <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i> motor neuron (Perez-Mansilla and Nurrish 2009), with novel components identified in this study shown in boxes. For quantification of SPHK-1-GFP, values are normalized to wild type (represented by dotted line); (*) <i>P</i> &lt; 0.05 (Student's <i>t</i>-test). Bars, 10 μm. Error bars are ±SEM for aldicarb curves, or ±propagated SEM for quantitative imaging.</div></p></div><div class="aux"><div class="resc">Jason P. Chan, et al. Genes Dev. 2012 May 15;26(10):1070-1085.</div><p class="links"><a class="dblinks" href="/pubmed/22588719" ref="ordinalpos=2510&amp;ncbi_uid=4369221&amp;link_uid=22588719">Citation</a><a class="dblinks" href="/pmc/articles/PMC3360562/" ref="ordinalpos=2510&amp;ncbi_uid=4369221&amp;link_uid=PMC3360562">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2511.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3327221/figure/fig7/" ref="ordinalpos=2511&amp;ncbi_uid=3963244&amp;link_uid=PMC3327221" href="/pmc/articles/PMC3327221/"><img src="/pmc/articles/instance/3327221/bin/PP_181883_gs_f7.gif" src-large="/pmc/articles/instance/3327221/bin/PP_181883_gs_f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3327221/" ref="ordinalpos=2511&amp;ncbi_uid=3963244&amp;link_uid=PMC3327221" image-link="/pmc/articles/PMC3327221/figure/fig7/" class="imagepopup">Figure 7. From: Apoplastic Reactive Oxygen Species Transiently Decrease Auxin <span class="highlight" style="background-color:">Signaling</span> and Cause Stress-Induced Morphogenic Response in Arabidopsis1[W][OA]. </a></p><div class="supp"><p class="details"><div>The auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is modulated by apoplastic ROS. The binding of auxin to TIR1/AFB receptors leads to the degradation of AUX/IAA repressors via the 26S proteasome, allowing the activation of ARF transcription factors and changes in gene expression. O<sub>3</sub> treatment leads to the production of apoplastic ROS, which could suppress the auxin <span class="highlight" style="background-color:">pathway</span> by decreasing the expression of <i>TIR1/AFB</i>s independently of miR393 and SA. O<sub>3</sub> may affect the stability of Aux/IAAs (e.g. IAA10 and IAA28), or ARF activity could be directly modulated independently of auxin F-box proteins. ARFs regulate auxin-dependent gene expression, which includes <i>Aux/IAA</i> transcripts. Decreased levels of <i>Aux/IAA</i> transcripts provide a feedback mechanism counteracting the increased Aux/IAA stability. In addition, apoplastic ROS modulated the expression of several genes involved in auxin <span class="highlight" style="background-color:">signaling</span>, polar transport, and biosynthesis. Chronic O<sub>3</sub> exposure leads to SIMR, which is under negative regulation of the auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tiina Blomster, et al. Plant Physiol. 2011 December;157(4):1866-1883.</div><p class="links"><a class="dblinks" href="/pubmed/22007024" ref="ordinalpos=2511&amp;ncbi_uid=3963244&amp;link_uid=22007024">Citation</a><a class="dblinks" href="/pmc/articles/PMC3327221/" ref="ordinalpos=2511&amp;ncbi_uid=3963244&amp;link_uid=PMC3327221">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2512.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3148591/figure/fig2/" ref="ordinalpos=2512&amp;ncbi_uid=3144500&amp;link_uid=PMC3148591" href="/pmc/articles/PMC3148591/"><img src="/pmc/articles/instance/3148591/bin/EDR2012-198048.002.gif" src-large="/pmc/articles/instance/3148591/bin/EDR2012-198048.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3148591/" ref="ordinalpos=2512&amp;ncbi_uid=3144500&amp;link_uid=PMC3148591" image-link="/pmc/articles/PMC3148591/figure/fig2/" class="imagepopup">Figure 2.  From: Diabetic Inhibition of Preconditioning- and Postconditioning-Mediated Myocardial Protection against Ischemia/Reperfusion Injury. </a></p><div class="supp"><p class="details"><div>Major <span class="highlight" style="background-color:">signaling</span> pathways of IPC- and Ipost-mediated protection against cardiac cell death. Myocardial protection of IPC and Ipost were proposed to be mediated by stimulation of the prosurvival <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>—PI3K/Akt <span class="highlight" style="background-color:">pathway</span> to inhibit the GSK-3<i>β</i> activation either via PI3K <span class="highlight" style="background-color:">pathway</span> or JAK2/STAT3 <span class="highlight" style="background-color:">pathway</span>. Diabetes (DM) can inhibit the activation of STAT3 or Akt to consequently activate GSK-3<i>β</i> that in turn induces mitochondrial cell death that is the critical cellular event for ischemia/reperfusion-induced myocardial infarction.</div></p></div><div class="aux"><div class="resc">Xia Yin, et al. Exp Diabetes Res. 2012;2012:198048.</div><p class="links"><a class="dblinks" href="/pubmed/21822424" ref="ordinalpos=2512&amp;ncbi_uid=3144500&amp;link_uid=21822424">Citation</a><a class="dblinks" href="/pmc/articles/PMC3148591/" ref="ordinalpos=2512&amp;ncbi_uid=3144500&amp;link_uid=PMC3148591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2513.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2613484/figure/F3/" ref="ordinalpos=2513&amp;ncbi_uid=1656627&amp;link_uid=PMC2613484" href="/pmc/articles/PMC2613484/"><img src="/pmc/articles/instance/2613484/bin/nihms-82795-f0003.gif" src-large="/pmc/articles/instance/2613484/bin/nihms-82795-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2613484/" ref="ordinalpos=2513&amp;ncbi_uid=1656627&amp;link_uid=PMC2613484" image-link="/pmc/articles/PMC2613484/figure/F3/" class="imagepopup">Fig. 3. TNFR and LTβR <span class="highlight" style="background-color:">signaling</span> pathways.  From: Targeting lymphocyte activation through the lymphotoxin and LIGHT pathways. </a></p><div class="supp"><p class="details"><div>Components in the TNFR1 and LTβR pathways for NF-κB activation. TNFR1 can access TRAF2 to activate the canonical NF-κB <span class="highlight" style="background-color:">pathway</span> RelA(p50/p65) via IκB degradation. This <span class="highlight" style="background-color:">pathway</span> controls many inflammatory genes and p100 synthesis. The LTβR induces both the canonical and the RelB NF-κB <span class="highlight" style="background-color:">pathway</span> via the NIK-IKKα-mediated processing of p100 and activation of p52/ RelB target genes.</div></p></div><div class="aux"><div class="resc">Carl F. Ware. Immunol Rev. ;223:186-201.</div><p class="links"><a class="dblinks" href="/pubmed/18613837" ref="ordinalpos=2513&amp;ncbi_uid=1656627&amp;link_uid=18613837">Citation</a><a class="dblinks" href="/pmc/articles/PMC2613484/" ref="ordinalpos=2513&amp;ncbi_uid=1656627&amp;link_uid=PMC2613484">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2514.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4006126/figure/Fig3/" ref="ordinalpos=2514&amp;ncbi_uid=6107042&amp;link_uid=PMC4006126" href="/pmc/articles/PMC4006126/"><img src="/pmc/articles/instance/4006126/bin/11010_2014_1987_Fig3_HTML.gif" src-large="/pmc/articles/instance/4006126/bin/11010_2014_1987_Fig3_HTML.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4006126/" ref="ordinalpos=2514&amp;ncbi_uid=6107042&amp;link_uid=PMC4006126" image-link="/pmc/articles/PMC4006126/figure/Fig3/" class="imagepopup">Fig. 3.  From: Loss of the TNF? function inhibits Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span>, exacerbates obesity development in adolescent spontaneous obese mice. </a></p><div class="supp"><p class="details"><div>mRNA levels of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and adipogenic genes in inguinal fat detected by qPCR. mRNA levels of the target gene are normalized to β-actin. <i>*</i>, <i>**P</i> &lt; 0.05 and <i>P</i> &lt; 0.01 significance relative to TNFα<sup>−/−</sup> mice, respectively. <sup>#</sup> <i>P</i> &lt; 0.05 significance relative to <i>db/db</i> mice. Values are expressed as the form of mean ± SEM</div></p></div><div class="aux"><div class="resc">Maolei Gong, et al. Mol Cell Biochem. 2014;391(1-2):59-66.</div><p class="links"><a class="dblinks" href="/pubmed/24522555" ref="ordinalpos=2514&amp;ncbi_uid=6107042&amp;link_uid=24522555">Citation</a><a class="dblinks" href="/pmc/articles/PMC4006126/" ref="ordinalpos=2514&amp;ncbi_uid=6107042&amp;link_uid=PMC4006126">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2515.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3433886/figure/F6/" ref="ordinalpos=2515&amp;ncbi_uid=4949412&amp;link_uid=PMC3433886" href="/pmc/articles/PMC3433886/"><img src="/pmc/articles/instance/3433886/bin/zjw0061241500006.gif" src-large="/pmc/articles/instance/3433886/bin/zjw0061241500006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3433886/" ref="ordinalpos=2515&amp;ncbi_uid=4949412&amp;link_uid=PMC3433886" image-link="/pmc/articles/PMC3433886/figure/F6/" class="imagepopup">Fig. 6. From: TSLP <span class="highlight" style="background-color:">Signaling</span> Network Revealed by SILAC-Based Phosphoproteomics. </a></p><div class="supp"><p class="details"><div><b>Novel kinases/phosphatases identified in TSLP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A–H</i>, phosphorylation status of tyrosine residues of nonreceptor tyrosine kinases—Btk, Tec, Lyn, Fes, and Hck/Src/Yes1/Fyn/Lck—and nonreceptor tyrosine phosphatases—Ptpn6 and Ptpn18—and PI-3 kinase catalytic subunit were up-regulated as evidenced by MS spectra showing the changes in the relative abundance of phosphopeptides.</div></p></div><div class="aux"><div class="resc">Jun Zhong, et al. Mol Cell Proteomics. 2012 June;11(6):M112.017764.</div><p class="links"><a class="dblinks" href="/pubmed/22345495" ref="ordinalpos=2515&amp;ncbi_uid=4949412&amp;link_uid=22345495">Citation</a><a class="dblinks" href="/pmc/articles/PMC3433886/" ref="ordinalpos=2515&amp;ncbi_uid=4949412&amp;link_uid=PMC3433886">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2516.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2277504/figure/F8/" ref="ordinalpos=2516&amp;ncbi_uid=1390348&amp;link_uid=PMC2277504" href="/pmc/articles/PMC2277504/"><img src="/pmc/articles/instance/2277504/bin/nihms-37907-f0008.gif" src-large="/pmc/articles/instance/2277504/bin/nihms-37907-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2277504/" ref="ordinalpos=2516&amp;ncbi_uid=1390348&amp;link_uid=PMC2277504" image-link="/pmc/articles/PMC2277504/figure/F8/" class="imagepopup">Figure 8.  From: Activity-dependent Synaptogenesis: Regulation by a CaM-kinase kinase/CaM-kinase I/?PIX <span class="highlight" style="background-color:">Signaling</span> Complex. </a></p><div class="supp"><p class="details"><div>Model of CaMK/βPIX <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activity-dependent Ca<sup>2+</sup> influx through the NMDA receptor of a spine activates the CaMKK/CaMKI/βPIX complex to promote CaMKI-mediated phosphorylation of S516 in βPIX with resultant activation of its GEF activity. The activated βPIX promotes GTP-loading of Rac (and/or Cdc42) to activate Pak and modulate actin dynamics and enhance spinogenesis and synapse formation.</div></p></div><div class="aux"><div class="resc">Takeo Saneyoshi, et al. Neuron. ;57(1):94-107.</div><p class="links"><a class="dblinks" href="/pubmed/18184567" ref="ordinalpos=2516&amp;ncbi_uid=1390348&amp;link_uid=18184567">Citation</a><a class="dblinks" href="/pmc/articles/PMC2277504/" ref="ordinalpos=2516&amp;ncbi_uid=1390348&amp;link_uid=PMC2277504">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2517.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3367861/figure/F1/" ref="ordinalpos=2517&amp;ncbi_uid=4946474&amp;link_uid=PMC3367861" href="/pmc/articles/PMC3367861/"><img src="/pmc/articles/instance/3367861/bin/nihms374230f1.gif" src-large="/pmc/articles/instance/3367861/bin/nihms374230f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3367861/" ref="ordinalpos=2517&amp;ncbi_uid=4946474&amp;link_uid=PMC3367861" image-link="/pmc/articles/PMC3367861/figure/F1/" class="imagepopup">Figure 1. Cross-talk between signals from self-pMHC and IL-7.  From: IL-7 <span class="highlight" style="background-color:">signaling</span> and CD127 receptor regulation in the control of T cell homeostasis. </a></p><div class="supp"><p class="details"><div>Many transcription factors and molecules downstream of TCR control CD127 expression, either by inducing or inhibiting its transcription. In green; transcription factors inducing CD127 transcription. In red; transcription factors repressing CD127 transcription. TCR signals can also induce formation of lipid rafts, which could recruit and concentrate IL-7R, thereby amplifying its downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. It is still unclear if IL-7 <span class="highlight" style="background-color:">signaling</span> activates the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in lymphoreplete hosts, but if it were to be true, Foxo1 and Foxp1 could finely regulate CD127 expression downstream of IL-7, allowing a limited IL-7 to support survival of a maximal number of T cells. Whether Gfi-1, GABPα or Ets-1 are are activated downstream of IL-7R or self-pMHC/TCR interactions is still not known, but Gfi-1 expression is induced downstream of strong TCR triggering [79, 80, 106].</div></p></div><div class="aux"><div class="resc">Florent Carrette, et al. Semin Immunol. ;24(3):209-217.</div><p class="links"><a class="dblinks" href="/pubmed/22551764" ref="ordinalpos=2517&amp;ncbi_uid=4946474&amp;link_uid=22551764">Citation</a><a class="dblinks" href="/pmc/articles/PMC3367861/" ref="ordinalpos=2517&amp;ncbi_uid=4946474&amp;link_uid=PMC3367861">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2518.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2217644/figure/fig10/" ref="ordinalpos=2518&amp;ncbi_uid=1166312&amp;link_uid=PMC2217644" href="/pmc/articles/PMC2217644/"><img src="/pmc/articles/instance/2217644/bin/tpc1903944f10.gif" src-large="/pmc/articles/instance/2217644/bin/tpc1903944f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2217644/" ref="ordinalpos=2518&amp;ncbi_uid=1166312&amp;link_uid=PMC2217644" image-link="/pmc/articles/PMC2217644/figure/fig10/" class="imagepopup">Figure 10. From: Plastid Signals Remodel Light <span class="highlight" style="background-color:">Signaling</span> Networks and Are Essential for Efficient Chloroplast Biogenesis in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Model for PhANG Regulation by a Network of Plastid and Light <span class="highlight" style="background-color:">Signaling</span> Pathways.<br />The current model for the GUN1-dependent plastid-to-nucleus <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was adapted from Koussevitzky et al. (2007). In this model, a second messenger (indicated with Y) that requires GUN1 for either its production or transduction (dotted arrows) triggers a plastid-to-nucleus <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that represses PhANGs. A plastid signal(s) that is independent of GUN1 (indicated with X) represses both <i>Lhcb</i> and <i>Rbcs</i> in the dark (data not shown) and also converts cry1 and one or more photoreceptors that perceive red light into negative regulators of <i>Lhcb</i> and <i>Rbcs</i>. cry1 becomes a negative regulator of <i>Lhcb</i> when X converts HY5 from a positive to a negative regulator of <i>Lhcb</i>. Under these same conditions, <i>Rbcs</i> is induced by cry1 and simultaneously repressed by a combination of blue and red light.</div></p></div><div class="aux"><div class="resc">Michael E. Ruckle, et al. Plant Cell. 2007 December;19(12):3944-3960.</div><p class="links"><a class="dblinks" href="/pubmed/18065688" ref="ordinalpos=2518&amp;ncbi_uid=1166312&amp;link_uid=18065688">Citation</a><a class="dblinks" href="/pmc/articles/PMC2217644/" ref="ordinalpos=2518&amp;ncbi_uid=1166312&amp;link_uid=PMC2217644">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2519.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3413998/figure/fig6/" ref="ordinalpos=2519&amp;ncbi_uid=4178342&amp;link_uid=PMC3413998" href="/pmc/articles/PMC3413998/"><img src="/pmc/articles/instance/3413998/bin/BMR2012-270425.006.gif" src-large="/pmc/articles/instance/3413998/bin/BMR2012-270425.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3413998/" ref="ordinalpos=2519&amp;ncbi_uid=4178342&amp;link_uid=PMC3413998" image-link="/pmc/articles/PMC3413998/figure/fig6/" class="imagepopup">Figure 6.  From: Hematopoietic Stem Cell Development, Niches, and <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>A canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the presence of Wnt, the signals are transduced through Frizzled family receptors and LRP5/LRP6 coreceptor to the <i>β</i>-catenin <span class="highlight" style="background-color:">signaling</span> cascade which then stabilize hypophosphorylated <i>β</i>-catenin and interact with TCF/LEF, Legless and PYGO for target gene activations. MARK and CKI<i>ε</i> are the positive regulators of canonical Wnt <span class="highlight" style="background-color:">pathway</span>, while APC, AXIN1, AXIN2, CKI<i>α</i>, NKD1, NKD2, <i>β</i>TRCP1, <i>β</i>TRCP2, and ANKRD6 are negative regulators. In the absence of Wnt, <i>β</i>-catenin-forming complex with AXIN and APC is phosphorylates, leading to be a target for ubiquitination and degradation by proteasome (adapted from [16]).</div></p></div><div class="aux"><div class="resc">Kamonnaree Chotinantakul, et al. Bone Marrow Res. 2012;2012:270425.</div><p class="links"><a class="dblinks" href="/pubmed/22900188" ref="ordinalpos=2519&amp;ncbi_uid=4178342&amp;link_uid=22900188">Citation</a><a class="dblinks" href="/pmc/articles/PMC3413998/" ref="ordinalpos=2519&amp;ncbi_uid=4178342&amp;link_uid=PMC3413998">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2520.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3691611/figure/F3/" ref="ordinalpos=2520&amp;ncbi_uid=5022885&amp;link_uid=PMC3691611" href="/pmc/articles/PMC3691611/"><img src="/pmc/articles/instance/3691611/bin/1478-811X-11-35-3.gif" src-large="/pmc/articles/instance/3691611/bin/1478-811X-11-35-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3691611/" ref="ordinalpos=2520&amp;ncbi_uid=5022885&amp;link_uid=PMC3691611" image-link="/pmc/articles/PMC3691611/figure/F3/" class="imagepopup">Figure 3.  From: Inhibition of a novel specific neuroglial integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> increases STAT3-mediated CNTF expression. </a></p><div class="supp"><p class="details"><div><b>A specific intracellular FAK-JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> represses CNTF. </b><b>A</b>) Incubation of C6 cells with inhibitors of JNK, but not p38 or ERK, for 4 hours, increases CNTF mRNA compared to vehicle controls (Ctrl). Data are means +/− SEM and are from 4 independent experiments. <b>B</b>) Incubation with FAK inhibitor reduced JNK activation as shown by reduced JNK phosphorylation in western blots, which are representative of 4 independent observations. Antibodies against total JNK were used as internal controls.</div></p></div><div class="aux"><div class="resc">Matthew P Keasey, et al. Cell Commun Signal. 2013;11:35-35.</div><p class="links"><a class="dblinks" href="/pubmed/23693126" ref="ordinalpos=2520&amp;ncbi_uid=5022885&amp;link_uid=23693126">Citation</a><a class="dblinks" href="/pmc/articles/PMC3691611/" ref="ordinalpos=2520&amp;ncbi_uid=5022885&amp;link_uid=PMC3691611">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2521.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3014294/figure/F1/" ref="ordinalpos=2521&amp;ncbi_uid=3315449&amp;link_uid=PMC3014294" href="/pmc/articles/PMC3014294/"><img src="/pmc/articles/instance/3014294/bin/nihms238244f1.gif" src-large="/pmc/articles/instance/3014294/bin/nihms238244f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3014294/" ref="ordinalpos=2521&amp;ncbi_uid=3315449&amp;link_uid=PMC3014294" image-link="/pmc/articles/PMC3014294/figure/F1/" class="imagepopup">Fig. 1. mAKAP <span class="highlight" style="background-color:">signaling</span> complexes.  From: Organizing signal transduction through A-kinase Anchoring Proteins (AKAPs). </a></p><div class="supp"><p class="details"><div><b>A)</b> mAKAP assembles a cAMP-responsive complex of <span class="highlight" style="background-color:">signaling</span> enzymes at the perinuclear membrane in the heart. PKA, PDE4D3 Epac1 and ERK5 are brought together with other associated enzymes to control different aspects of cardiomyocyte physiology. <b>B)</b> When intracellular cAMP levels are elevated the mAKAP-associated PKA phosphorylates the PDE4D3 in the complex at two sites, leading to increased metabolism of cAMP by the phosphodiesterase. Likewise, cAMP activation of Epac1 in the complex activates Rap1 to inhibit ERK5 <span class="highlight" style="background-color:">signaling</span>. C) As cAMP levels fall, the Epac1-mediated inhibition of ERK <span class="highlight" style="background-color:">signaling</span> is lost and mitogenic <span class="highlight" style="background-color:">signaling</span> favors cell growth. <b>D)</b> mAKAP assembles an oxygen sensitive <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that includes the ubiquitin E3 ligase Siah2, prolyl hydroxylase, von Hippel-Lindau protein and the transcription factor HIF-1α. Under normoxic conditions, HIF-1α is continually degraded however, when oxygen levels fall, the mAKAP associated PHD is degraded and HIF-1α accumulates and translocates into the nucleus.</div></p></div><div class="aux"><div class="resc">Jeremy S. Logue, et al. FEBS J. ;277(21):4370-4375.</div><p class="links"><a class="dblinks" href="/pubmed/20883492" ref="ordinalpos=2521&amp;ncbi_uid=3315449&amp;link_uid=20883492">Citation</a><a class="dblinks" href="/pmc/articles/PMC3014294/" ref="ordinalpos=2521&amp;ncbi_uid=3315449&amp;link_uid=PMC3014294">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2522.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1838942/figure/F1/" ref="ordinalpos=2522&amp;ncbi_uid=1029442&amp;link_uid=PMC1838942" href="/pmc/articles/PMC1838942/"><img src="/pmc/articles/instance/1838942/bin/JCI0730453.f1.gif" src-large="/pmc/articles/instance/1838942/bin/JCI0730453.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1838942/" ref="ordinalpos=2522&amp;ncbi_uid=1029442&amp;link_uid=PMC1838942" image-link="/pmc/articles/PMC1838942/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Signaling</span> via LRb and gp130R: similarities and differences..  From: gp130 receptor ligands as potential therapeutic targets for obesity. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Leptin binds its homodimeric receptor LRb, which results in autophosphorylation and activation of JAK, subsequently activating the insulin receptor substrate/PI3K (IRS/PI3K) signalling <span class="highlight" style="background-color:">pathway</span>. JAK activation also results in the phosphorylation of LRb at Tyr985 and Tyr1138. The phosphorylation of Tyr1138 mediates the recruitment, phosphorylation, and activation of the transcription factor STAT3, resulting in the transcription of <i>SOCS3</i> and other STAT3-dependent genes in the nucleus. SOCS3 inhibits leptin <span class="highlight" style="background-color:">signaling</span> via binding to SHP-2 bound to the LRb, recruiting ubiquitin transferases to the SOCS3 box domain at Tyr985; and binding JAK. (<b>B</b>) <span class="highlight" style="background-color:">Signaling</span> through gp130Rβ is similar to that through LRb. In the case of IL-6 <span class="highlight" style="background-color:">signaling</span>, IL-6 binds to the IL-6Rα/gp130Rβ homodimer, which results in JAK/STAT, insulin receptor substrate/PI3K, and ERK <span class="highlight" style="background-color:">signaling</span>. Importantly, however, there are 4 tyrosine phosphorylation sites (at residues 767, 814, 905, and 915) distal to the SHP-2 domain bound at Tyr759. As with the LRb, SOCS3 can inhibit JAK <span class="highlight" style="background-color:">signaling</span> on gp130Rβ. It is not clear why gp130R ligands may overcome SOCS3 inhibition, but it may be due to the 4 additional STAT3 binding sites, since truncation of gp130R to remove these sites does not allow ligand-mediated STAT3 phosphorylation in mice in vivo (18).                </div></p></div><div class="aux"><div class="resc">Mark A. Febbraio. J Clin Invest. 2007 April 2;117(4):841-849.</div><p class="links"><a class="dblinks" href="/pubmed/17404609" ref="ordinalpos=2522&amp;ncbi_uid=1029442&amp;link_uid=17404609">Citation</a><a class="dblinks" href="/pmc/articles/PMC1838942/" ref="ordinalpos=2522&amp;ncbi_uid=1029442&amp;link_uid=PMC1838942">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2523.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3173230/figure/F6/" ref="ordinalpos=2523&amp;ncbi_uid=3491764&amp;link_uid=PMC3173230" href="/pmc/articles/PMC3173230/"><img src="/pmc/articles/instance/3173230/bin/zbc0431178800006.gif" src-large="/pmc/articles/instance/3173230/bin/zbc0431178800006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3173230/" ref="ordinalpos=2523&amp;ncbi_uid=3491764&amp;link_uid=PMC3173230" image-link="/pmc/articles/PMC3173230/figure/F6/" class="imagepopup">FIGURE 6. From: Adiponectin Ameliorates Doxorubicin-induced Cardiotoxicity through Akt Protein-dependent Mechanism. </a></p><div class="supp"><p class="details"><div><b>Proposed scheme for protective actions of adiponectin on DOX-induced cardiomyopathy.</b> Adiponectin activates SphK-1/Akt <span class="highlight" style="background-color:">signaling</span> via an LRP1/CRT-dependent <span class="highlight" style="background-color:">pathway</span>, and this <span class="highlight" style="background-color:">pathway</span> protects against DOX-induced cardiac dysfunction by its ability to attenuate myocardial apoptosis.</div></p></div><div class="aux"><div class="resc">Sonomi Maruyama, et al. J Biol Chem. 2011 September 16;286(37):32790-32800.</div><p class="links"><a class="dblinks" href="/pubmed/21784858" ref="ordinalpos=2523&amp;ncbi_uid=3491764&amp;link_uid=21784858">Citation</a><a class="dblinks" href="/pmc/articles/PMC3173230/" ref="ordinalpos=2523&amp;ncbi_uid=3491764&amp;link_uid=PMC3173230">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2524.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3838012/figure/F5/" ref="ordinalpos=2524&amp;ncbi_uid=6212163&amp;link_uid=PMC3838012" href="/pmc/articles/PMC3838012/"><img src="/pmc/articles/instance/3838012/bin/zmb9991002140005.gif" src-large="/pmc/articles/instance/3838012/bin/zmb9991002140005.jpg" alt="Fig 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3838012/" ref="ordinalpos=2524&amp;ncbi_uid=6212163&amp;link_uid=PMC3838012" image-link="/pmc/articles/PMC3838012/figure/F5/" class="imagepopup">Fig 5.  From: Cessation of Epithelial Bmp <span class="highlight" style="background-color:">Signaling</span> Switches the Differentiation of Crown Epithelia to the Root Lineage in a ?-Catenin-Dependent Manner. </a></p><div class="supp"><p class="details"><div>Depletion of Alk3 in dental epithelial upregulated activity of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> but not that of TGF-β <span class="highlight" style="background-color:">signaling</span>. (A to C) AKO and WT DEpSCs were induced as described in the legend to Fig. 4 and analyzed by Western blotting of nuclear proteins (A), qRT-PCR (B), or IF (C) to examine the expression of genes related to the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> or the TGF-β <span class="highlight" style="background-color:">pathway</span> (<i>n</i> = 3 in each assay) (n.s., not significant; ∗, <i>P</i> &lt; 0.05). (D) P7 WT and AKO teeth were examined with double IF for β-catenin and Krt14.</div></p></div><div class="aux"><div class="resc">Zhenhua Yang, et al. Mol Cell Biol. 2013 December;33(23):4732-4744.</div><p class="links"><a class="dblinks" href="/pubmed/24081330" ref="ordinalpos=2524&amp;ncbi_uid=6212163&amp;link_uid=24081330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3838012/" ref="ordinalpos=2524&amp;ncbi_uid=6212163&amp;link_uid=PMC3838012">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2525.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4016200/figure/pone-0096946-g006/" ref="ordinalpos=2525&amp;ncbi_uid=6131076&amp;link_uid=PMC4016200" href="/pmc/articles/PMC4016200/"><img src="/pmc/articles/instance/4016200/bin/pone.0096946.g006.gif" src-large="/pmc/articles/instance/4016200/bin/pone.0096946.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4016200/" ref="ordinalpos=2525&amp;ncbi_uid=6131076&amp;link_uid=PMC4016200" image-link="/pmc/articles/PMC4016200/figure/pone-0096946-g006/" class="imagepopup">Figure 6. Overview of Cd-dependent <span class="highlight" style="background-color:">signaling</span> cascade that affects drought, high-salinity and low temperature stress <span class="highlight" style="background-color:">signaling</span> pathways..  From: Genome-Wide Transcriptome Analysis Reveals that Cadmium Stress <span class="highlight" style="background-color:">Signaling</span> Controls the Expression of Genes in Drought Stress Signal Pathways in Rice. </a></p><div class="supp"><p class="details"><div>Many abiotic stress-related transcription factors including DREB, bZIP and NAC may function in the signal transduction <span class="highlight" style="background-color:">pathway</span> under Cd exposure. These TFs may regulate the expression of many downstream genes for stress tolerance and responses.</div></p></div><div class="aux"><div class="resc">Youko Oono, et al. PLoS One. 2014;9(5):e96946.</div><p class="links"><a class="dblinks" href="/pubmed/24816929" ref="ordinalpos=2525&amp;ncbi_uid=6131076&amp;link_uid=24816929">Citation</a><a class="dblinks" href="/pmc/articles/PMC4016200/" ref="ordinalpos=2525&amp;ncbi_uid=6131076&amp;link_uid=PMC4016200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2526.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3392028/figure/F1/" ref="ordinalpos=2526&amp;ncbi_uid=4107698&amp;link_uid=PMC3392028" href="/pmc/articles/PMC3392028/"><img src="/pmc/articles/instance/3392028/bin/nihms219131f1.gif" src-large="/pmc/articles/instance/3392028/bin/nihms219131f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3392028/" ref="ordinalpos=2526&amp;ncbi_uid=4107698&amp;link_uid=PMC3392028" image-link="/pmc/articles/PMC3392028/figure/F1/" class="imagepopup">FIGURE 1.  From: <span class="highlight" style="background-color:">Signaling</span> Networks that Control the Lineage Commitment and Differentiation of Bone Cells. </a></p><div class="supp"><p class="details"><div>Transcription factors and <span class="highlight" style="background-color:">signaling</span> involved in the osteoblast differentiation <span class="highlight" style="background-color:">pathway</span>. Osteoblasts and chondrocytes are derived from common mesenchymal stem cell precursors. Runx2 stimulates terminal differentiation. A number of transcription factors are involved in Runx2 regulation and function, either upstream of Runx2 or as coactivators or corepressors. Runx2 functions upstream of Osx, which is required after Runx2 for osteoblast differentiation. Ihh and Wnt/β-catenin are key <span class="highlight" style="background-color:">signaling</span> molecules in osteoblastogenesis.</div></p></div><div class="aux"><div class="resc">Carrie S. Soltanoff, et al. Crit Rev Eukaryot Gene Expr. ;19(1):1-46.</div><p class="links"><a class="dblinks" href="/pubmed/19191755" ref="ordinalpos=2526&amp;ncbi_uid=4107698&amp;link_uid=19191755">Citation</a><a class="dblinks" href="/pmc/articles/PMC3392028/" ref="ordinalpos=2526&amp;ncbi_uid=4107698&amp;link_uid=PMC3392028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2527.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2902748/figure/fig1/" ref="ordinalpos=2527&amp;ncbi_uid=2342138&amp;link_uid=PMC2902748" href="/pmc/articles/PMC2902748/"><img src="/pmc/articles/instance/2902748/bin/JO2010-412985.001.gif" src-large="/pmc/articles/instance/2902748/bin/JO2010-412985.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2902748/" ref="ordinalpos=2527&amp;ncbi_uid=2342138&amp;link_uid=PMC2902748" image-link="/pmc/articles/PMC2902748/figure/fig1/" class="imagepopup">Figure 1.  From: Hsp90 as a Gatekeeper of Tumor Angiogenesis: Clinical Promise and Potential Pitfalls. </a></p><div class="supp"><p class="details"><div>      <i>Hsp90-dependent modulation of proangiogenic <span class="highlight" style="background-color:">signaling</span> pathways in cancer</i>. Hsp90 regulates multiple arms of angiogenic <span class="highlight" style="background-color:">signaling</span> in cancer. Key <span class="highlight" style="background-color:">signaling</span> molecules that are either direct clients or indirectly modulated by Hsp90 are shaded in red. One <span class="highlight" style="background-color:">pathway</span> that is commonly upregulated during tumorigenesis is the HIF/VEGF <span class="highlight" style="background-color:">signaling</span> axis. Tumor hypoxia and other stimuli induce HIF expression and subsequent activity, leading to a cascade of events that reinforce VEGF expression and angiogenesis. Importantly, several key mediators of this <span class="highlight" style="background-color:">pathway</span>, including HIF and VEGFR, are dependent upon Hsp90 for their function. As indicated, RTK activation also potently upregulates HIF via AKT/mTOR -mediated translation. RTKs additionally transactivate EphA2, a recently identified Hsp90 client protein known to participate in tumor vascularization. Providing another level of complexity, HIF also upregulates the expression of several RTK ligands (e.g., HGF and TGF-alpha), as well as RTK receptors (EGFR, IGFR), thereby reinforcing these <span class="highlight" style="background-color:">signaling</span> networks. Hsp90 additionally plays a role in NF<i>κ</i>B-dependent VEGF expression and regulates downstream effectors of VEGF <span class="highlight" style="background-color:">signaling</span>, including AKT-mediated eNOS phosphorylation. Given the intertwining levels among Hsp90 and angiogenic <span class="highlight" style="background-color:">signaling</span> cascades, Hsp90 intervention is predicted to impair <span class="highlight" style="background-color:">signaling</span> at many levels within these redundant pathways, with the overall effect of suppressing tumor angiogenesis.</div></p></div><div class="aux"><div class="resc">J. E. Bohonowych, et al. J Oncol. 2010;2010:412985.</div><p class="links"><a class="dblinks" href="/pubmed/20628489" ref="ordinalpos=2527&amp;ncbi_uid=2342138&amp;link_uid=20628489">Citation</a><a class="dblinks" href="/pmc/articles/PMC2902748/" ref="ordinalpos=2527&amp;ncbi_uid=2342138&amp;link_uid=PMC2902748">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2528.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436201/figure/F1/" ref="ordinalpos=2528&amp;ncbi_uid=5219695&amp;link_uid=PMC3436201" href="/pmc/articles/PMC3436201/"><img src="/pmc/articles/instance/3436201/bin/zbc0371221580001.gif" src-large="/pmc/articles/instance/3436201/bin/zbc0371221580001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436201/" ref="ordinalpos=2528&amp;ncbi_uid=5219695&amp;link_uid=PMC3436201" image-link="/pmc/articles/PMC3436201/figure/F1/" class="imagepopup">FIGURE 1. From: The Chemokine CCL5 Regulates Glucose Uptake and AMP Kinase <span class="highlight" style="background-color:">Signaling</span> in Activated T Cells to Facilitate Chemotaxis. </a></p><div class="supp"><p class="details"><div><b>CCL5 induces phosphorylation of proteins in the AMPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, the Full Moon BioSystems AMPK <span class="highlight" style="background-color:">signaling</span> phospho-specific antibody array includes six replicates (vertical columns) of phospho-specific antibodies and their non-phospho pairs, targeted against proteins in the AMPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Biotinylated protein lysates were added to microscope slide chambers and fluorescence from Cy3-streptavidin was measured with the Axon GenePix 400A microarray scanner. <i>B</i>, the extent of protein phosphorylation (mean fluorescence intensity, <i>MFI</i>) was normalized within each slide and compared between untreated control and cells treated with 10 n<span style="font-variant: small-caps" class="small-caps">m</span> CCL5 for 10 min. The data are represented as fold CCL5-induction relative to untreated controls. Phosphorylated <span class="highlight" style="background-color:">signaling</span> intermediates associated with metabolism are indicated (<i>red arrows</i>).</div></p></div><div class="aux"><div class="resc">Olivia Chan, et al. J Biol Chem. 2012 August 24;287(35):29406-29416.</div><p class="links"><a class="dblinks" href="/pubmed/22782897" ref="ordinalpos=2528&amp;ncbi_uid=5219695&amp;link_uid=22782897">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436201/" ref="ordinalpos=2528&amp;ncbi_uid=5219695&amp;link_uid=PMC3436201">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2529.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3895947/figure/F4/" ref="ordinalpos=2529&amp;ncbi_uid=5715649&amp;link_uid=PMC3895947" href="/pmc/articles/PMC3895947/"><img src="/pmc/articles/instance/3895947/bin/nihms474454f4.gif" src-large="/pmc/articles/instance/3895947/bin/nihms474454f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3895947/" ref="ordinalpos=2529&amp;ncbi_uid=5715649&amp;link_uid=PMC3895947" image-link="/pmc/articles/PMC3895947/figure/F4/" class="imagepopup">Figure 4.  From: Functional roles of gangliosides in neurodevelopment--An overview of recent advances. </a></p><div class="supp"><p class="details"><div>A wide variety of glycoconjugates such as heparin sulfate proteoglycan (HSPG), chondrotin sulfate proteoglycan (CSPG), N-glycosylated cystatin C, and galectins-1 are involved in <span class="highlight" style="background-color:">signaling</span> pathways inducing proliferation and self-renewal of NSCs. The Ras-MAPK <span class="highlight" style="background-color:">pathway</span> mediating the proliferation signal is modulated in the lipid rafts, cell-surface microdomains rich in GSLs (e.g., GD3). Modification of GSL expression represses the Ras-MAPK <span class="highlight" style="background-color:">pathway</span> but not the JAK-STAT <span class="highlight" style="background-color:">pathway</span> (inducing astrocyte differentiation) and the PI3K-Akt <span class="highlight" style="background-color:">pathway</span> (inducing survival), suggesting a specific role of GSLs in the Ras-MAPK <span class="highlight" style="background-color:">pathway</span>. The glycoconjugate antigens and related molecules are indicated in red. (Reproduced from [54])</div></p></div><div class="aux"><div class="resc">Robert K. Yu, et al. Neurochem Res. ;37(6):1230-1244.</div><p class="links"><a class="dblinks" href="/pubmed/22410735" ref="ordinalpos=2529&amp;ncbi_uid=5715649&amp;link_uid=22410735">Citation</a><a class="dblinks" href="/pmc/articles/PMC3895947/" ref="ordinalpos=2529&amp;ncbi_uid=5715649&amp;link_uid=PMC3895947">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2530.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3044050/figure/F6/" ref="ordinalpos=2530&amp;ncbi_uid=3638250&amp;link_uid=PMC3044050" href="/pmc/articles/PMC3044050/"><img src="/pmc/articles/instance/3044050/bin/zh40021196950006.gif" src-large="/pmc/articles/instance/3044050/bin/zh40021196950006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3044050/" ref="ordinalpos=2530&amp;ncbi_uid=3638250&amp;link_uid=PMC3044050" image-link="/pmc/articles/PMC3044050/figure/F6/" class="imagepopup">Fig. 6. From: Sprouty2 downregulates angiogenesis during mouse skin wound healing. </a></p><div class="supp"><p class="details"><div>Spry2 and Spry2<sup>Y55F</sup> function in the context of EC MAPK <span class="highlight" style="background-color:">signaling</span>. Upon growth factor (GF) binding to its respective receptor tyrosine kinase (RTK) and activation of the respective <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Spry2 is induced to translocate to the inner plasma membrane where it gets activated (in many cases via phosphorylation on the Y55 by a Src-like kinase) and functions by interacting with various MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>-associated proteins in a GF-specific manner. When RTK <span class="highlight" style="background-color:">signaling</span> is Ras dependent (<i>left</i>), Spry2 inhibition of this <span class="highlight" style="background-color:">pathway</span> is thought to occur at the level of GF receptor-bound protein-2 (Grb2) or Raf1. When RTK <span class="highlight" style="background-color:">signaling</span> is Ras independent (<i>right</i>), Spry2 inhibition of this <span class="highlight" style="background-color:">pathway</span> is thought to occur at the level of Raf1. In many cases, pY55 is required for Spry2 inhibition of the Raf/Mek/Erk <span class="highlight" style="background-color:">pathway</span>. The dominant-negative Y55F mutant of Spry2 inhibits endogenous Spry2 action, thereby promoting the Raf/Mek/Erk <span class="highlight" style="background-color:">pathway</span>. FGF-2, fibroblast GF-2; PDGF, platelet-derived GF; pY, phosphorylated tyrosine; Sos1, son of sevenless-1; PLC-γ, phospholipase C-γ; PIP<sub>2</sub>, phosphatidylinositol bisphosphate; DAG, diacylglycerol; IP<sub>3</sub>, inositol trisphosphate; PKC, protein kinase C. Information in this figure was compiled from Refs. 10, 16, 22, 23, 24, 27, 30, 34, 38.</div></p></div><div class="aux"><div class="resc">Mateusz S. Wietecha, et al. Am J Physiol Heart Circ Physiol. 2011 February;300(2):H459-H467.</div><p class="links"><a class="dblinks" href="/pubmed/21076020" ref="ordinalpos=2530&amp;ncbi_uid=3638250&amp;link_uid=21076020">Citation</a><a class="dblinks" href="/pmc/articles/PMC3044050/" ref="ordinalpos=2530&amp;ncbi_uid=3638250&amp;link_uid=PMC3044050">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2531.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4261839/figure/F7/" ref="ordinalpos=2531&amp;ncbi_uid=6953321&amp;link_uid=PMC4261839" href="/pmc/articles/PMC4261839/"><img src="/pmc/articles/instance/4261839/bin/mpphys_ssu106_f0007.gif" src-large="/pmc/articles/instance/4261839/bin/mpphys_ssu106_f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4261839/" ref="ordinalpos=2531&amp;ncbi_uid=6953321&amp;link_uid=PMC4261839" image-link="/pmc/articles/PMC4261839/figure/F7/" class="imagepopup">Figure 7.  From: SPINDLY, ERECTA, and Its Ligand STOMAGEN Have a Role in Redox-Mediated Cortex Proliferation in the Arabidopsis Root. </a></p><div class="supp"><p class="details"><div>Schematic of the Interplay between SPY and the STOMAGEN–ERECTA <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in ROS Sensing and Redox-Mediated Cortex Proliferation in the <i>Arabidopsis</i> Root.<br />Under stress, plants accumulate ROS, which oxidize and activate STOMAGEN. STOMAGEN in turn activates ERECTA, which exerts its effect on mitosis in the endodermis through intercellular <span class="highlight" style="background-color:">signaling</span>. ROS also oxidizes and inactivates SPY, which normally represses middle-cortex formation by maintaining the expression of peroxidases and thus cellular redox. SPY acts downstream of ERECTA, but how SPY is affected by the ERECTA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> remains unknown.</div></p></div><div class="aux"><div class="resc">Hongchang Cui, et al. Mol Plant. 2014 December;7(12):1727-1739.</div><p class="links"><a class="dblinks" href="/pubmed/25267734" ref="ordinalpos=2531&amp;ncbi_uid=6953321&amp;link_uid=25267734">Citation</a><a class="dblinks" href="/pmc/articles/PMC4261839/" ref="ordinalpos=2531&amp;ncbi_uid=6953321&amp;link_uid=PMC4261839">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2532.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2882367/figure/pone-0011014-g009/" ref="ordinalpos=2532&amp;ncbi_uid=2285150&amp;link_uid=PMC2882367" href="/pmc/articles/PMC2882367/"><img src="/pmc/articles/instance/2882367/bin/pone.0011014.g009.gif" src-large="/pmc/articles/instance/2882367/bin/pone.0011014.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2882367/" ref="ordinalpos=2532&amp;ncbi_uid=2285150&amp;link_uid=PMC2882367" image-link="/pmc/articles/PMC2882367/figure/pone-0011014-g009/" class="imagepopup">Figure 9. Model depicting the LRP6 turnover pathways..  From: Dkk1 Stabilizes Wnt Co-Receptor LRP6: Implication for Wnt Ligand-Induced LRP6 Down-Regulation. </a></p><div class="supp"><p class="details"><div>The first <span class="highlight" style="background-color:">pathway</span> is mediated by Kremen. In the presence of Kremen, LRP6 antagonist Dkk1 serves as a bridge between LRP6 and Kremen and induces LRP6 internalization and degradation, terminating Wnt <span class="highlight" style="background-color:">signaling</span>. Furthermore, Kremen might directly bind to LRP6 and induce LRP6 degradation. The second <span class="highlight" style="background-color:">pathway</span> is mediated by Wnt/Frizzled. While Wnt binding to its receptors LRP6 and Frizzled (Fz) induces LRP6-Wnt-Fz complex formation, the endocytosis and Wnt/LRP6 <span class="highlight" style="background-color:">signaling</span> decay of this complex occur after Wnt <span class="highlight" style="background-color:">signaling</span> activation. Dkk1 blocks Wnt binding to LRP6 and then inhibits LRP6 turnover mediated by Wnt/Fz.</div></p></div><div class="aux"><div class="resc">Yonghe Li, et al. PLoS One. 2010;5(6):e11014.</div><p class="links"><a class="dblinks" href="/pubmed/20543981" ref="ordinalpos=2532&amp;ncbi_uid=2285150&amp;link_uid=20543981">Citation</a><a class="dblinks" href="/pmc/articles/PMC2882367/" ref="ordinalpos=2532&amp;ncbi_uid=2285150&amp;link_uid=PMC2882367">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2533.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3138532/figure/F1/" ref="ordinalpos=2533&amp;ncbi_uid=3088190&amp;link_uid=PMC3138532" href="/pmc/articles/PMC3138532/"><img src="/pmc/articles/instance/3138532/bin/nihms251481f1.gif" src-large="/pmc/articles/instance/3138532/bin/nihms251481f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3138532/" ref="ordinalpos=2533&amp;ncbi_uid=3088190&amp;link_uid=PMC3138532" image-link="/pmc/articles/PMC3138532/figure/F1/" class="imagepopup">Figure 1.  From: The molecular pathogenesis of head and neck cancer. </a></p><div class="supp"><p class="details"><div>HNSCC tumor microenvironment. This model depicts the potential <span class="highlight" style="background-color:">signaling</span> pathways for targeting in HNSCC, including ligands, receptors and <span class="highlight" style="background-color:">signaling</span> intermediates of the tumor cells and the contributing vascular endothelial cells and fibroblasts. Also shown are the molecular inhibitors currently in clinical investigation for the treatment of HNSCC. This model shows <span class="highlight" style="background-color:">pathway</span> convergence, <span class="highlight" style="background-color:">pathway</span> divergence and microenvironment <span class="highlight" style="background-color:">signaling</span> in order to demonstrate the potential of both single-targeting and multi-targeting drug approaches.</div></p></div><div class="aux"><div class="resc">Jonah D. Klein, et al. Cancer Biol Ther. ;9(1):1-7.</div><p class="links"><a class="dblinks" href="/pubmed/20038820" ref="ordinalpos=2533&amp;ncbi_uid=3088190&amp;link_uid=20038820">Citation</a><a class="dblinks" href="/pmc/articles/PMC3138532/" ref="ordinalpos=2533&amp;ncbi_uid=3088190&amp;link_uid=PMC3138532">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2534.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3121451/figure/F7/" ref="ordinalpos=2534&amp;ncbi_uid=3544040&amp;link_uid=PMC3121451" href="/pmc/articles/PMC3121451/"><img src="/pmc/articles/instance/3121451/bin/zbc0291165730007.gif" src-large="/pmc/articles/instance/3121451/bin/zbc0291165730007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3121451/" ref="ordinalpos=2534&amp;ncbi_uid=3544040&amp;link_uid=PMC3121451" image-link="/pmc/articles/PMC3121451/figure/F7/" class="imagepopup">FIGURE 7. From: A Novel Specificity Protein 1 (SP1)-like Gene Regulating Protein Kinase C-1 (Pkc1)-dependent Cell Wall Integrity and Virulence Factors in Cryptococcus neoformans. </a></p><div class="supp"><p class="details"><div><b>Role of Sp1 in the Pkc1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of <i>C. neoformans</i>.</b> Pkc1 phosphorylates a member of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Bck1, with subsequent phosphorylation of Mkk2 and the downstream component Mpk1 to mediate cell wall integrity, growth at high temperature, and virulence. Pkc1 also regulates the transcription factor Sp1 by phosphorylation, leading to activation by nuclear translocation as well as by transcriptional mechanisms, leading to regulation of osmoresistance, cell wall integrity, and virulence in parallel with the MAPK <span class="highlight" style="background-color:">pathway</span> as well as a unique <span class="highlight" style="background-color:">pathway</span> leading to resistance to nitrosative stress, capsule production, and caspofungin resistance. Laccase (Lac1) is regulated by Pkc1 via an unknown <span class="highlight" style="background-color:">pathway</span>. DAG, diacylglycerol. Adapted from Gerik <i>et al.</i> (16).</div></p></div><div class="aux"><div class="resc">Amos Adler, et al. J Biol Chem. 2011 June 10;286(23):20977-20990.</div><p class="links"><a class="dblinks" href="/pubmed/21487010" ref="ordinalpos=2534&amp;ncbi_uid=3544040&amp;link_uid=21487010">Citation</a><a class="dblinks" href="/pmc/articles/PMC3121451/" ref="ordinalpos=2534&amp;ncbi_uid=3544040&amp;link_uid=PMC3121451">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2535.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2194020/figure/fig4/" ref="ordinalpos=2535&amp;ncbi_uid=1219287&amp;link_uid=PMC2194020" href="/pmc/articles/PMC2194020/"><img src="/pmc/articles/instance/2194020/bin/20020298f4.gif" src-large="/pmc/articles/instance/2194020/bin/20020298f4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2194020/" ref="ordinalpos=2535&amp;ncbi_uid=1219287&amp;link_uid=PMC2194020" image-link="/pmc/articles/PMC2194020/figure/fig4/" class="imagepopup">Figure 4. From: Constitutive Expression of the B7h Ligand for Inducible Costimulator on Naive B Cells Is Extinguished after Activation by Distinct B Cell Receptor and Interleukin 4 Receptor-mediated Pathways and Can Be Rescued by CD40 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>BCR-mediated B7h down-regulation occurs through Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span>, whereas IL-4–mediated down-regulation occurs through a distinct Stat6-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) B7h down-regulation induced by BCR <span class="highlight" style="background-color:">signaling</span> is Ca<sup>2+</sup> dependent. Purified B cells were treated for 24h with PMA alone, ionomycin alone, or a combination of both and stained for B7h expression. (B) B7h down-regulation by BCR <span class="highlight" style="background-color:">signaling</span> is CsA sensitive. Purified B cells were activated with PMA and ionomycin, anti-IgM F(ab′)<sub>2</sub>, IL-4, or untreated either in the presence (solid lines) or absence (solid filled) of CsA. After 24 h of stimulation, cells were stained for B7h expression compared with control staining (shaded histogram). (C) IL-4–mediated B7h down-regulation is Stat-6 dependent. Purified B cells from Stat6<sup>−/−</sup> and wild-type control mice were either untreated or activated with IL-4, anti-IgM F(ab′)<sub>2</sub> and IL-4, or PMA and ionomycin. Cells were stained for B7h expression after 24 h of culture.</div></p></div><div class="aux"><div class="resc">Linda Liang, et al. J Exp Med. 2002 July 1;196(1):97-108.</div><p class="links"><a class="dblinks" href="/pubmed/12093874" ref="ordinalpos=2535&amp;ncbi_uid=1219287&amp;link_uid=12093874">Citation</a><a class="dblinks" href="/pmc/articles/PMC2194020/" ref="ordinalpos=2535&amp;ncbi_uid=1219287&amp;link_uid=PMC2194020">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2536.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3462653/figure/F1/" ref="ordinalpos=2536&amp;ncbi_uid=4274250&amp;link_uid=PMC3462653" href="/pmc/articles/PMC3462653/"><img src="/pmc/articles/instance/3462653/bin/nihms169248f1.gif" src-large="/pmc/articles/instance/3462653/bin/nihms169248f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3462653/" ref="ordinalpos=2536&amp;ncbi_uid=4274250&amp;link_uid=PMC3462653" image-link="/pmc/articles/PMC3462653/figure/F1/" class="imagepopup">Figure 1. Toll and IMD immune <span class="highlight" style="background-color:">signaling</span> pathways involved in anti-Plasmodium defense.  From: Mosquito immune defenses against Plasmodium infection. </a></p><div class="supp"><p class="details"><div>Following recognition of a microbe, or unknown <i>Plasmodium</i> ligand, by soluble PGRP molecules, the Toll <span class="highlight" style="background-color:">pathway</span> is stimulated by binding of the ligand Spätzle with the Toll transmembrane receptor. This triggers a series of molecular events that culminate in the activation (*) and translocation of Rel1 into the nucleus, up-regulating transcription of immune genes that are responsible for microbial killing. The IMD <span class="highlight" style="background-color:">pathway</span> is stimulated when the transmembrane PGRPLC receptor binds peptidoglycan or an unknown <i>Plasmodium</i> ligand that leads to the cleavage of Rel2-F and translocation of active Rel2-S (*) into the nucleus. A different set of anti-<i>Plasmodium</i> genes are up-regulated when the IMD <span class="highlight" style="background-color:">pathway</span> is stimulated. Branching of the IMD <span class="highlight" style="background-color:">pathway</span> is indicated, but the JNK <span class="highlight" style="background-color:">pathway</span> has not been extensively characterized in <i>Anopheles</i> mosquitoes.</div></p></div><div class="aux"><div class="resc">Chris M. Cirimotich, et al. Dev Comp Immunol. 2010 April;34(4):387-395.</div><p class="links"><a class="dblinks" href="/pubmed/20026176" ref="ordinalpos=2536&amp;ncbi_uid=4274250&amp;link_uid=20026176">Citation</a><a class="dblinks" href="/pmc/articles/PMC3462653/" ref="ordinalpos=2536&amp;ncbi_uid=4274250&amp;link_uid=PMC3462653">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2537.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136090/figure/fig5/" ref="ordinalpos=2537&amp;ncbi_uid=3087429&amp;link_uid=PMC3136090" href="/pmc/articles/PMC3136090/"><img src="/pmc/articles/instance/3136090/bin/JTE2011-534603.005.gif" src-large="/pmc/articles/instance/3136090/bin/JTE2011-534603.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136090/" ref="ordinalpos=2537&amp;ncbi_uid=3087429&amp;link_uid=PMC3136090" image-link="/pmc/articles/PMC3136090/figure/fig5/" class="imagepopup">Figure 5.  From: Protein Expression of STRO-1 Cells in Response to Different Topographic Features. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> schematic representation: The postulated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in this study. Proteins with significant change in their expressions are highlighted yellow.</div></p></div><div class="aux"><div class="resc">Fahsai Kantawong, et al. J Tissue Eng. 2011;2011:534603.</div><p class="links"><a class="dblinks" href="/pubmed/21772957" ref="ordinalpos=2537&amp;ncbi_uid=3087429&amp;link_uid=21772957">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136090/" ref="ordinalpos=2537&amp;ncbi_uid=3087429&amp;link_uid=PMC3136090">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2538.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1345641/figure/fig11/" ref="ordinalpos=2538&amp;ncbi_uid=732414&amp;link_uid=PMC1345641" href="/pmc/articles/PMC1345641/"><img src="/pmc/articles/instance/1345641/bin/zmk0010674710011.gif" src-large="/pmc/articles/instance/1345641/bin/zmk0010674710011.jpg" alt="Figure 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1345641/" ref="ordinalpos=2538&amp;ncbi_uid=732414&amp;link_uid=PMC1345641" image-link="/pmc/articles/PMC1345641/figure/fig11/" class="imagepopup">Figure 11. From: Molecular Interaction Maps of Bioregulatory Networks: A General Rubric for Systems Biology. </a></p><div class="supp"><p class="details"><div>Interactions at membranes: <span class="highlight" style="background-color:">signaling</span> via G-proteins. This classic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is here shown in MIM notation. See text for detailed description and discussion.</div></p></div><div class="aux"><div class="resc">Kurt W. Kohn, et al. Mol Biol Cell. 2006 January;17(1):1-13.</div><p class="links"><a class="dblinks" href="/pubmed/16267266" ref="ordinalpos=2538&amp;ncbi_uid=732414&amp;link_uid=16267266">Citation</a><a class="dblinks" href="/pmc/articles/PMC1345641/" ref="ordinalpos=2538&amp;ncbi_uid=732414&amp;link_uid=PMC1345641">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2539.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3030928/figure/F4/" ref="ordinalpos=2539&amp;ncbi_uid=3612823&amp;link_uid=PMC3030928" href="/pmc/articles/PMC3030928/"><img src="/pmc/articles/instance/3030928/bin/nihms-245612-f0004.gif" src-large="/pmc/articles/instance/3030928/bin/nihms-245612-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3030928/" ref="ordinalpos=2539&amp;ncbi_uid=3612823&amp;link_uid=PMC3030928" image-link="/pmc/articles/PMC3030928/figure/F4/" class="imagepopup">Figure 4. Upregulation of a CNTF-mediated Neuroprotection in MS Motor Cortex.  From: Mechanisms of Neuronal Dysfunction and Degeneration in Multiple Sclerosis. </a></p><div class="supp"><p class="details"><div>mRNA encoding CNTF and multiple members of the CNTF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are increased in MS cortex (<b>a</b>). (<b>b</b>) Schematic representation of the CNTF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members which were increased in MS cortex. Red denotes increased in MS cortex. (Reproduced from (Dutta et al., 2007)</div></p></div><div class="aux"><div class="resc">Ranjan Dutta, et al. Prog Neurobiol. ;93(1):1-12.</div><p class="links"><a class="dblinks" href="/pubmed/20946934" ref="ordinalpos=2539&amp;ncbi_uid=3612823&amp;link_uid=20946934">Citation</a><a class="dblinks" href="/pmc/articles/PMC3030928/" ref="ordinalpos=2539&amp;ncbi_uid=3612823&amp;link_uid=PMC3030928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2540.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507227/figure/f03/" ref="ordinalpos=2540&amp;ncbi_uid=4424809&amp;link_uid=PMC3507227" href="/pmc/articles/PMC3507227/"><img src="/pmc/articles/instance/3507227/bin/bio-01-08-754-f03.gif" src-large="/pmc/articles/instance/3507227/bin/bio-01-08-754-f03.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507227/" ref="ordinalpos=2540&amp;ncbi_uid=4424809&amp;link_uid=PMC3507227" image-link="/pmc/articles/PMC3507227/figure/f03/" class="imagepopup">Fig. 3.Fluorescence imaging of the DAG <span class="highlight" style="background-color:">pathway</span> during macropinocytosis..  From: A growth factor <span class="highlight" style="background-color:">signaling</span> cascade confined to circular ruffles in macrophages. </a></p><div class="supp"><p class="details"><div>(<b>A–C</b>) Macrophages expressing mCerulean, mCherry-Rab5, and Citrine-tagged probes were imaged during macropinocytosis. For each panel: top row  =  phase-contrast, middle row  =  ratio of Citrine-tagged fluorophore/mCerulean, bottom row  =  ratio of mCherry-Rab5/mCerulean. Ruffle closure occurred at <i>t = 0</i>. Scale bars  =  3.0 µm. (A) Citrine-C1δ peaked at <i>t = 60 s.</i> (B) Citrine-PLCγ did not demonstrate differential localization during macropinocytosis. (C) Citrine-PKCα localization peaked at <i>t = 180 s</i>, resembling localization patterns of Rab5. (<b>D</b>) Relative timing of <span class="highlight" style="background-color:">signaling</span> in the DAG <span class="highlight" style="background-color:">pathway</span>. <i>n</i>≥9 for each curve. Error bars indicate standard deviation.</div></p></div><div class="aux"><div class="resc">Timothy P. Welliver, et al. Biol Open. 2012 August 15;1(8):754-760.</div><p class="links"><a class="dblinks" href="/pubmed/23213469" ref="ordinalpos=2540&amp;ncbi_uid=4424809&amp;link_uid=23213469">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507227/" ref="ordinalpos=2540&amp;ncbi_uid=4424809&amp;link_uid=PMC3507227">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2541.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3283704/figure/pone-0032013-g001/" ref="ordinalpos=2541&amp;ncbi_uid=3822339&amp;link_uid=PMC3283704" href="/pmc/articles/PMC3283704/"><img src="/pmc/articles/instance/3283704/bin/pone.0032013.g001.gif" src-large="/pmc/articles/instance/3283704/bin/pone.0032013.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3283704/" ref="ordinalpos=2541&amp;ncbi_uid=3822339&amp;link_uid=PMC3283704" image-link="/pmc/articles/PMC3283704/figure/pone-0032013-g001/" class="imagepopup">Figure 1. Components of the Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are expressed in hematopoietic stem and progenitor cells..  From: Normal Hematopoietic Stem Cell Function in Mice with Enforced Expression of the Hippo <span class="highlight" style="background-color:">Signaling</span> Effector YAP1. </a></p><div class="supp"><p class="details"><div>mRNA expression levels of upstreams components of the Hippo <span class="highlight" style="background-color:">pathway</span> (<b>A</b>) and the downstream effector Yap1 (<b>B</b>) in different stem and progenitor compartments. Values are normalized against Hprt1 for n = 2 (<b>A</b>) and n = 3 (<b>B</b>). Error bars represent SEM. Abbreviations: LT, long-term HSCs; ST, short-term HSCs; LMPPs, lymphoid myeloid primed progenitor; Lin+, lineage positive.</div></p></div><div class="aux"><div class="resc">Lina Jansson, et al. PLoS One. 2012;7(2):e32013.</div><p class="links"><a class="dblinks" href="/pubmed/22363786" ref="ordinalpos=2541&amp;ncbi_uid=3822339&amp;link_uid=22363786">Citation</a><a class="dblinks" href="/pmc/articles/PMC3283704/" ref="ordinalpos=2541&amp;ncbi_uid=3822339&amp;link_uid=PMC3283704">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2542.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3430985/figure/F1/" ref="ordinalpos=2542&amp;ncbi_uid=4201554&amp;link_uid=PMC3430985" href="/pmc/articles/PMC3430985/"><img src="/pmc/articles/instance/3430985/bin/nihms-401334-f0001.gif" src-large="/pmc/articles/instance/3430985/bin/nihms-401334-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3430985/" ref="ordinalpos=2542&amp;ncbi_uid=4201554&amp;link_uid=PMC3430985" image-link="/pmc/articles/PMC3430985/figure/F1/" class="imagepopup">Figure 1. Bacterial Effector Proteins Target Conserved Eukaryotic <span class="highlight" style="background-color:">Signaling</span> Pathways.  From: The Yeast Saccharomyces cerevisiae: A Versatile Model System for the Identification and Characterization of Bacterial Virulence Proteins. </a></p><div class="supp"><p class="details"><div>The intersection of a MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and Rho-GTPase cycling is illustrated along with the effector proteins from bacteria that target these pathways. Effector proteins with functions that mimic a mammalian protein are displayed in blue, while those that modify and deactivate mammalian proteins are displayed in red.</div></p></div><div class="aux"><div class="resc">Keri A. Siggers, et al. Cell Host Microbe. ;4(1):8-15.</div><p class="links"><a class="dblinks" href="/pubmed/18621006" ref="ordinalpos=2542&amp;ncbi_uid=4201554&amp;link_uid=18621006">Citation</a><a class="dblinks" href="/pmc/articles/PMC3430985/" ref="ordinalpos=2542&amp;ncbi_uid=4201554&amp;link_uid=PMC3430985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2543.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC26685/figure/F3/" ref="ordinalpos=2543&amp;ncbi_uid=8234&amp;link_uid=PMC26685" href="/pmc/articles/PMC26685/"><img src="/pmc/articles/instance/26685/bin/pq0103652003.gif" src-large="/pmc/articles/instance/26685/bin/pq0103652003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC26685/" ref="ordinalpos=2543&amp;ncbi_uid=8234&amp;link_uid=PMC26685" image-link="/pmc/articles/PMC26685/figure/F3/" class="imagepopup">Figure 3.  From: A common muscarinic <span class="highlight" style="background-color:">pathway</span> for diapause recovery in the distantly related nematode species Caenorhabditis elegans and Ancylostoma caninum. </a></p><div class="supp"><p class="details"><div>A model for cholinergic input induction of dauer recovery. In dauer pheromone or in a <i>daf-7</i> mutant, the DAF-7 TGF-β ligand is not produced by the ASI sensory/secretory neuron. Therefore, there is no activation of the DAF-1 and DAF-4 TGF-β receptors or downstream DAF-8 and DAF-14 Smad proteins, and this results in high DAF-3 Smad activity in <span class="highlight" style="background-color:">signaling</span> cells or target tissues. In pheromone without muscarinic agonists, no insulin-like signal is released, causing less insulin receptor signal transduction to the transcription factor DAF-16, which, in combination with unregulated DAF-3, induces dauer arrest. Under pheromone-induced or <i>daf-7</i> mutation-induced dauer induction conditions, muscarinic stimulation causes release of an insulin-like DAF-2 ligand, which stimulates the DAF-2/AGE-1-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to DAF-16 inactivation. Because <i>daf-7</i> mutants can recover in muscarinic agonists, the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not required for dauer recovery. Under normal conditions of dauer recovery upon release from pheromone and addition of food and low temperature, we suggest that acetylcholine released via temperature or food pathways binds to the muscarinic receptor on the insulin-like <span class="highlight" style="background-color:">signaling</span> cell to cause an increase in insulin release. We suggest that temperature is coupled via the interneurons AIY and AIZ to the DAF-2 insulin-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> rather than the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> because mutations in the thermoregulatory gene <i>ttx-3</i> can suppress mutations in <i>daf-7</i> and not mutations in <i>daf-2</i> (ref. 11; O. Hobert and G.R., unpublished observations).</div></p></div><div class="aux"><div class="resc">Heidi A. Tissenbaum, et al. Proc Natl Acad Sci U S A. 2000 January 4;97(1):460-465.</div><p class="links"><a class="dblinks" href="/pubmed/10618440" ref="ordinalpos=2543&amp;ncbi_uid=8234&amp;link_uid=10618440">Citation</a><a class="dblinks" href="/pmc/articles/PMC26685/" ref="ordinalpos=2543&amp;ncbi_uid=8234&amp;link_uid=PMC26685">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2544.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3906428/figure/F1/" ref="ordinalpos=2544&amp;ncbi_uid=5782893&amp;link_uid=PMC3906428" href="/pmc/articles/PMC3906428/"><img src="/pmc/articles/instance/3906428/bin/jkst-2-e27203-g1.gif" src-large="/pmc/articles/instance/3906428/bin/jkst-2-e27203-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3906428/" ref="ordinalpos=2544&amp;ncbi_uid=5782893&amp;link_uid=PMC3906428" image-link="/pmc/articles/PMC3906428/figure/F1/" class="imagepopup">Figure. From: JAK-STAT in lipid metabolism of adipocytes. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of JAK-STAT.</div></p></div><div class="aux"><div class="resc">Dong Xu, et al. JAKSTAT. 2013 October 1;2(4):e27203.</div><p class="links"><a class="dblinks" href="/pubmed/24498541" ref="ordinalpos=2544&amp;ncbi_uid=5782893&amp;link_uid=24498541">Citation</a><a class="dblinks" href="/pmc/articles/PMC3906428/" ref="ordinalpos=2544&amp;ncbi_uid=5782893&amp;link_uid=PMC3906428">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2545.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3617211/figure/pcbi-1003016-g002/" ref="ordinalpos=2545&amp;ncbi_uid=4808769&amp;link_uid=PMC3617211" href="/pmc/articles/PMC3617211/"><img src="/pmc/articles/instance/3617211/bin/pcbi.1003016.g002.gif" src-large="/pmc/articles/instance/3617211/bin/pcbi.1003016.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3617211/" ref="ordinalpos=2545&amp;ncbi_uid=4808769&amp;link_uid=PMC3617211" image-link="/pmc/articles/PMC3617211/figure/pcbi-1003016-g002/" class="imagepopup">Figure 2. Normalized proportion of negative genetic interactions within one KEGG <span class="highlight" style="background-color:">pathway</span>..  From: Synthetic Lethality between Gene Defects Affecting a Single Non-essential Molecular <span class="highlight" style="background-color:">Pathway</span> with Reversible Steps. </a></p><div class="supp"><p class="details"><div>The proportion is related to the number of all possible pairwise interactions <i>S</i>(<i>S</i>−1)/2, where <i>S</i> is the size of the KEGG <span class="highlight" style="background-color:">pathway</span>. Negative interactions between components of a molecular complex were excluded (compare the fourth and the fifth columns). We counted the normalized number of negative genetic interactions within one <span class="highlight" style="background-color:">pathway</span> using recent data on the genome-wide screening of genetic interactions in yeast from [5], using the most stringent filter on the epistasis measure ε. Definitions of yeast <span class="highlight" style="background-color:">signaling</span> and metabolic pathways were taken from KEGG database [30]. Only KEGG pathways with a normalized proportion of ≥1% are shown. DNA repair KEGG pathways are highlighted in bold.</div></p></div><div class="aux"><div class="resc">Andrei Zinovyev, et al. PLoS Comput Biol. 2013 April;9(4):e1003016.</div><p class="links"><a class="dblinks" href="/pubmed/23592964" ref="ordinalpos=2545&amp;ncbi_uid=4808769&amp;link_uid=23592964">Citation</a><a class="dblinks" href="/pmc/articles/PMC3617211/" ref="ordinalpos=2545&amp;ncbi_uid=4808769&amp;link_uid=PMC3617211">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2546.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g006/" ref="ordinalpos=2546&amp;ncbi_uid=4989291&amp;link_uid=PMC3677896" href="/pmc/articles/PMC3677896/"><img src="/pmc/articles/instance/3677896/bin/pone.0065032.g006.gif" src-large="/pmc/articles/instance/3677896/bin/pone.0065032.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3677896/" ref="ordinalpos=2546&amp;ncbi_uid=4989291&amp;link_uid=PMC3677896" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g006/" class="imagepopup">Figure 6. Effects of SHH <span class="highlight" style="background-color:">signaling</span> agonists on tumor cell growth without dead cells..  From: Sonic Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Supports Cancer Cell Growth during Cancer Radiotherapy. </a></p><div class="supp"><p class="details"><div>A. SHH <span class="highlight" style="background-color:">signaling</span> agonist SAG enhances Panc1Fluc cell growth in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. **represents <i>P</i>&lt;0.01. B. SHH <span class="highlight" style="background-color:">signaling</span> agonist SAG enhances HT29Fluc cell growth in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. *represents <i>P</i>&lt;0.05, **represents <i>P</i>&lt;0.01. C. Panc1Fluc cells treated with N-terminal fragment Shh demonstrate increased growth. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. *represents <i>P</i>&lt;0.05. D. HT29Fluc cells treated with N-terminal fragment Shh demonstrate increased growth. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. **represents <i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Jingjing Ma, et al. PLoS One. 2013;8(6):e65032.</div><p class="links"><a class="dblinks" href="/pubmed/23762282" ref="ordinalpos=2546&amp;ncbi_uid=4989291&amp;link_uid=23762282">Citation</a><a class="dblinks" href="/pmc/articles/PMC3677896/" ref="ordinalpos=2546&amp;ncbi_uid=4989291&amp;link_uid=PMC3677896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2547.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3110643/figure/F5/" ref="ordinalpos=2547&amp;ncbi_uid=3376170&amp;link_uid=PMC3110643" href="/pmc/articles/PMC3110643/"><img src="/pmc/articles/instance/3110643/bin/nihms-274721-f0005.gif" src-large="/pmc/articles/instance/3110643/bin/nihms-274721-f0005.jpg" alt="Fig 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3110643/" ref="ordinalpos=2547&amp;ncbi_uid=3376170&amp;link_uid=PMC3110643" image-link="/pmc/articles/PMC3110643/figure/F5/" class="imagepopup">Fig 5. CsA increases TGFβ <span class="highlight" style="background-color:">pathway</span>-dependent <span class="highlight" style="background-color:">signaling</span> in xenograft human squamous tumors.  From: Cyclosporine A mediates Pathogenesis of Aggressive Cutaneous Squamous Cell Carcinoma by Augmenting Epithelial-Mesenchymal Transition: Role of TGF-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) Immunofluorescence staining illustrates expression of TGFβ receptors and downstream <span class="highlight" style="background-color:">signaling</span> proteins Smads, Smad 2, 3, 4, and 7, in vehicle and CsA-treated tumors. (B) Western Blot analysis showing expression of TGFβ receptors and downstream <span class="highlight" style="background-color:">signaling</span> proteins Smads, Smad 2, 3, 4, and 7, in vehicle and CsA-treated tumors.</div></p></div><div class="aux"><div class="resc">Stephanie B. Walsh, et al. Mol Carcinog. ;50(7):516-527.</div><p class="links"><a class="dblinks" href="/pubmed/21308804" ref="ordinalpos=2547&amp;ncbi_uid=3376170&amp;link_uid=21308804">Citation</a><a class="dblinks" href="/pmc/articles/PMC3110643/" ref="ordinalpos=2547&amp;ncbi_uid=3376170&amp;link_uid=PMC3110643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2548.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204848/figure/F13/" ref="ordinalpos=2548&amp;ncbi_uid=3322007&amp;link_uid=PMC3204848" href="/pmc/articles/PMC3204848/"><img src="/pmc/articles/instance/3204848/bin/JCI59291.f13.gif" src-large="/pmc/articles/instance/3204848/bin/JCI59291.f13.jpg" alt="Figure 13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204848/" ref="ordinalpos=2548&amp;ncbi_uid=3322007&amp;link_uid=PMC3204848" image-link="/pmc/articles/PMC3204848/figure/F13/" class="imagepopup">Figure 13. Proposed mechanism for hyperglycemia-induced platelet hyperactivity through TX <span class="highlight" style="background-color:">signaling</span> mediated by the AR <span class="highlight" style="background-color:">pathway</span>..  From: Glucose and collagen regulate human platelet activity through aldose reductase induction of thromboxane. </a></p><div class="supp"><p class="details"><div>Hyperglycemia synergizes with collagen stimulation in platelets to activate the AR-dependent polyol <span class="highlight" style="background-color:">pathway</span>, leading to increased ROS production (via GSH depletion and NADH oxidase [Nox]). Excessive ROS can increase TXA<sub>2</sub> production by potentiating PLCγ2/PKC/p38α MAPK <span class="highlight" style="background-color:">signaling</span>, thus providing more arachidonic acid substrate for COX-1 and TXA<sub>2</sub> synthase (TS) to produce TXA<sub>2</sub>, leading to platelet hyperactivity. The oxidative stress can also lead to enhanced surface expression of TP, which increases the potency of circulating TXA2.    </div></p></div><div class="aux"><div class="resc">Wai Ho Tang, et al. J Clin Invest. 2011 November 1;121(11):4462-4476.</div><p class="links"><a class="dblinks" href="/pubmed/22005299" ref="ordinalpos=2548&amp;ncbi_uid=3322007&amp;link_uid=22005299">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204848/" ref="ordinalpos=2548&amp;ncbi_uid=3322007&amp;link_uid=PMC3204848">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2549.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4395498/figure/F1/" ref="ordinalpos=2549&amp;ncbi_uid=7414368&amp;link_uid=PMC4395498" href="/pmc/articles/PMC4395498/"><img src="/pmc/articles/instance/4395498/bin/nihms586541f1.gif" src-large="/pmc/articles/instance/4395498/bin/nihms586541f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4395498/" ref="ordinalpos=2549&amp;ncbi_uid=7414368&amp;link_uid=PMC4395498" image-link="/pmc/articles/PMC4395498/figure/F1/" class="imagepopup">Figure 1. Schematic of Akt <span class="highlight" style="background-color:">signaling</span> and regulation.  From: <span class="highlight" style="background-color:">Signaling</span> Mechanisms Regulating Myelination in the Central Nervous System. </a></p><div class="supp"><p class="details"><div>Extracellular ligand (purple) binding a transmembrane receptor and activating the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> (blue icons). Lipid second messengers PIP2 and PIP3 indicated in yellow. Negative regulators indicated in red. Blunted arrows indicate inhibition. Dashed line indicates indirect regulation.</div></p></div><div class="aux"><div class="resc">Jared T. AHRENDSEN, et al. Neurosci Bull. ;29(2):199-215.</div><p class="links"><a class="dblinks" href="/pubmed/23558589" ref="ordinalpos=2549&amp;ncbi_uid=7414368&amp;link_uid=23558589">Citation</a><a class="dblinks" href="/pmc/articles/PMC4395498/" ref="ordinalpos=2549&amp;ncbi_uid=7414368&amp;link_uid=PMC4395498">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2550.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3642497/figure/F3/" ref="ordinalpos=2550&amp;ncbi_uid=4883446&amp;link_uid=PMC3642497" href="/pmc/articles/PMC3642497/"><img src="/pmc/articles/instance/3642497/bin/fncel-07-00056-g003.gif" src-large="/pmc/articles/instance/3642497/bin/fncel-07-00056-g003.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3642497/" ref="ordinalpos=2550&amp;ncbi_uid=4883446&amp;link_uid=PMC3642497" image-link="/pmc/articles/PMC3642497/figure/F3/" class="imagepopup">FIGURE 3.  From: Methylene blue induces macroautophagy through 5? adenosine monophosphate-activated protein kinase <span class="highlight" style="background-color:">pathway</span> to protect neurons from serum deprivation. </a></p><div class="supp"><p class="details"><div><b>Induction of macroautophagy in HT22 cells by methylene blue is not through mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <b>(A–C)</b> Representative Western blots showed that methylene blue did not alter activation of Erk1/2, Akt, or mTOR. <b>(D)</b> Representative Western bots depict that Bcl-2 protein level was increased by methylene blue in a dose-dependent manner.</div></p></div><div class="aux"><div class="resc">Luokun Xie, et al. Front Cell Neurosci. 2013;7:56.</div><p class="links"><a class="dblinks" href="/pubmed/23653592" ref="ordinalpos=2550&amp;ncbi_uid=4883446&amp;link_uid=23653592">Citation</a><a class="dblinks" href="/pmc/articles/PMC3642497/" ref="ordinalpos=2550&amp;ncbi_uid=4883446&amp;link_uid=PMC3642497">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2551.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC149052/figure/fig10/" ref="ordinalpos=2551&amp;ncbi_uid=209593&amp;link_uid=PMC149052" href="/pmc/articles/PMC149052/"><img src="/pmc/articles/instance/149052/bin/00-0018f10.gif" src-large="/pmc/articles/instance/149052/bin/00-0018f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC149052/" ref="ordinalpos=2551&amp;ncbi_uid=209593&amp;link_uid=PMC149052" image-link="/pmc/articles/PMC149052/figure/fig10/" class="imagepopup">Figure 10. From: Interactions between Abscisic Acid and Ethylene <span class="highlight" style="background-color:">Signaling</span> Cascades. </a></p><div class="supp"><p class="details"><div>Schematic Representations of the Interactions between the ABA and Ethylene <span class="highlight" style="background-color:">Signaling</span> Cascades.<br /><b>(A)</b> Regulation of seed germination. Ethylene and GA <span class="highlight" style="background-color:">signaling</span> cascades promote seed germination by decreasing sensitivity to endogenous ABA in imbibed seeds. The <i>abi1-1</i> and <i>abi3</i> mutants are ABA insensitive, the <i>sly1</i> mutant is GA insensitive, and the <i>etr1-1</i> and <i>ein2</i> mutants are ethylene insensitive. The positions of the intersections between the ABA, ethylene, and GA pathways are speculative.<br /><b>(B)</b> Inhibition of seedling root growth. Inhibition of root growth in response to increased amounts of ethylene is largely mediated through internal accumulation of auxin. Cytokinin inhibits root growth by stimulating ethylene biosynthesis. ABA inhibition of root growth is apparently not mediated by an increase in ethylene biosynthesis but requires an active ethylene <span class="highlight" style="background-color:">signaling</span> cascade. Either the basal activity of the ethylene <span class="highlight" style="background-color:">pathway</span> is required to synergize ABA action or ABA stimulates the ethylene <span class="highlight" style="background-color:">signaling</span> cascade.</div></p></div><div class="aux"><div class="resc">Nathalie Beaudoin, et al. Plant Cell. 2000 July;12(7):1103-1116.</div><p class="links"><a class="dblinks" href="/pubmed/10899977" ref="ordinalpos=2551&amp;ncbi_uid=209593&amp;link_uid=10899977">Citation</a><a class="dblinks" href="/pmc/articles/PMC149052/" ref="ordinalpos=2551&amp;ncbi_uid=209593&amp;link_uid=PMC149052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2552.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4217307/figure/F9/" ref="ordinalpos=2552&amp;ncbi_uid=6845406&amp;link_uid=PMC4217307" href="/pmc/articles/PMC4217307/"><img src="/pmc/articles/instance/4217307/bin/fphys-05-00420-g0009.gif" src-large="/pmc/articles/instance/4217307/bin/fphys-05-00420-g0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4217307/" ref="ordinalpos=2552&amp;ncbi_uid=6845406&amp;link_uid=PMC4217307" image-link="/pmc/articles/PMC4217307/figure/F9/" class="imagepopup">Figure 9.  From: Release of GLP-1 and PYY in response to the activation of G protein-coupled bile acid receptor TGR5 is mediated by Epac/PLC-? <span class="highlight" style="background-color:">pathway</span> and modulated by endogenous H2S. </a></p><div class="supp"><p class="details"><div><b>TGR5-mediated <span class="highlight" style="background-color:">signaling</span> pathways to release GLP-1 and PYY and inhibition of release of GLP-1 and PYY by H<sub>2</sub>S</b>. In STC-1 cells, activation of G<sub>s</sub>-coupled TGR5 receptors by OA causes stimulation of PI hydrolysis, and release of GLP-1 and PYY via a PKA-independent, cAMP-dependent mechanism involving Epac/PLC-ε/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span>. H<sub>2</sub>S inhibits GLP-1 and PYY release by inhibiting PI hydrolysis and Ca<sup>2+</sup> release.</div></p></div><div class="aux"><div class="resc">Vanitha Bala, et al. Front Physiol. 2014;5:420.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2552&amp;ncbi_uid=6845406&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4217307/" ref="ordinalpos=2552&amp;ncbi_uid=6845406&amp;link_uid=PMC4217307">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2553.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3427312/figure/pone-0043874-g008/" ref="ordinalpos=2553&amp;ncbi_uid=4201555&amp;link_uid=PMC3427312" href="/pmc/articles/PMC3427312/"><img src="/pmc/articles/instance/3427312/bin/pone.0043874.g008.gif" src-large="/pmc/articles/instance/3427312/bin/pone.0043874.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3427312/" ref="ordinalpos=2553&amp;ncbi_uid=4201555&amp;link_uid=PMC3427312" image-link="/pmc/articles/PMC3427312/figure/pone-0043874-g008/" class="imagepopup">Figure 8. Schematic representation of the possible mechanism of BBR on the S1P2 receptor..  From: Berberine Reduces Fibronectin Expression by Suppressing the S1P-S1P2 Receptor <span class="highlight" style="background-color:">Pathway</span> in Experimental Diabetic Nephropathy Models. </a></p><div class="supp"><p class="details"><div>Subjecting GMCs to HG results in the activation of NF-<b>κ</b>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and thus up-regulates S1P2 receptor expression, which contributes to the increased FN levels. In contrast, pretreatment with BBR inhibits S1P2 receptor and FN levels possibly by down-regulating NF-<b>κ</b>B activation.</div></p></div><div class="aux"><div class="resc">Kaipeng Huang, et al. PLoS One. 2012;7(8):e43874.</div><p class="links"><a class="dblinks" href="/pubmed/22937115" ref="ordinalpos=2553&amp;ncbi_uid=4201555&amp;link_uid=22937115">Citation</a><a class="dblinks" href="/pmc/articles/PMC3427312/" ref="ordinalpos=2553&amp;ncbi_uid=4201555&amp;link_uid=PMC3427312">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2554.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052302/figure/marinedrugs-12-02485-f010/" ref="ordinalpos=2554&amp;ncbi_uid=6238969&amp;link_uid=PMC4052302" href="/pmc/articles/PMC4052302/"><img src="/pmc/articles/instance/4052302/bin/marinedrugs-12-02485-g010.gif" src-large="/pmc/articles/instance/4052302/bin/marinedrugs-12-02485-g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052302/" ref="ordinalpos=2554&amp;ncbi_uid=6238969&amp;link_uid=PMC4052302" image-link="/pmc/articles/PMC4052302/figure/marinedrugs-12-02485-f010/" class="imagepopup">Figure 10.  From: Gram-Negative Marine Bacteria: Structural Features of Lipopolysaccharides and Their Relevance for Economically Important Diseases. </a></p><div class="supp"><p class="details"><div>General overview of TLR4 <span class="highlight" style="background-color:">signaling</span> in humans. TLR4 is activated by LPS, which in turn triggers both MyD88-dependent and MyD88-independent TRIF-dependent pathways. <span class="highlight" style="background-color:">Signaling</span> through the MyD88-dependent <span class="highlight" style="background-color:">pathway</span> is responsible for early phase nuclear factor (NF)-κB and mitogen-activated protein kinase (MAPK) activation that facilitates the induction of pro-inflammatory cytokines. The TRIF-dependent <span class="highlight" style="background-color:">pathway</span> activates IRF3, which culminates in the induction of IFN-β- and IFN-inducible genes. This <span class="highlight" style="background-color:">pathway</span> also helps in secondary NF-κB and MAPK activation that reinforces the defense response.</div></p></div><div class="aux"><div class="resc">Muhammad Ayaz Anwar, et al. Mar Drugs. 2014 May;12(5):2485-2514.</div><p class="links"><a class="dblinks" href="/pubmed/24796306" ref="ordinalpos=2554&amp;ncbi_uid=6238969&amp;link_uid=24796306">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052302/" ref="ordinalpos=2554&amp;ncbi_uid=6238969&amp;link_uid=PMC4052302">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2555.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4215754/figure/f3/" ref="ordinalpos=2555&amp;ncbi_uid=6812505&amp;link_uid=PMC4215754" href="/pmc/articles/PMC4215754/"><img src="/pmc/articles/instance/4215819/bin/f1000research-3-5389-g0002.gif" src-large="/pmc/articles/instance/4215819/bin/f1000research-3-5389-g0002.jpg" alt="Figure 3. " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4215754/" ref="ordinalpos=2555&amp;ncbi_uid=6812505&amp;link_uid=PMC4215754" image-link="/pmc/articles/PMC4215754/figure/f3/" class="imagepopup">Figure 3. KEGG B cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> tuned in CD19 B cells and CD4 T cells..  From: CyKEGGParser: tailoring KEGG pathways to fit into systems biology analysis workflows. </a></p><div class="supp"><p class="details"><div>Tuning was performed with 25 percentile threshold of gene expression values for each tissue. Highlighted in yellow are the nodes (“LYN” and “CD19”) not present in the <span class="highlight" style="background-color:">pathway</span> tuned in CD4 T cells.</div></p></div><div class="aux"><div class="resc">Lilit Nersisyan, et al. F1000Res. 2014;3:145.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2555&amp;ncbi_uid=6812505&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4215754/" ref="ordinalpos=2555&amp;ncbi_uid=6812505&amp;link_uid=PMC4215754">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2556.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC30221/figure/F4/" ref="ordinalpos=2556&amp;ncbi_uid=41334&amp;link_uid=PMC30221" href="/pmc/articles/PMC30221/"><img src="/pmc/articles/instance/30221/bin/pq0516341004.gif" src-large="/pmc/articles/instance/30221/bin/pq0516341004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC30221/" ref="ordinalpos=2556&amp;ncbi_uid=41334&amp;link_uid=PMC30221" image-link="/pmc/articles/PMC30221/figure/F4/" class="imagepopup">Figure 4.  From: Activity-dependent CREB phosphorylation: Convergence of a fast,  sensitive calmodulin kinase <span class="highlight" style="background-color:">pathway</span> and a slow,  less sensitive mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Differential recruitment of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  (<i>A</i>) Dynamics of [Ca<sup>2+</sup>]<sub>i</sub>  generated by mild (20 mM K<sup>+</sup>) or strong (90 mM  K<sup>+</sup>) depolarization. Traces represent somatic  [Ca<sup>2+</sup>]<sub>i</sub> measured with fura-2 ratiometric  imaging. (<i>B</i>) Dynamics of pMAPK generated by mild (20 mM  K<sup>+</sup>) or strong (90 mM K<sup>+</sup>) depolarization,  determined with an antibody specific for pMAPK. (<i>C</i>)  Dynamics of CREB phosphorylation generated by mild (20 mM  K<sup>+</sup>) or strong (90 mM K<sup>+</sup>) depolarization.</div></p></div><div class="aux"><div class="resc">Gang-Yi Wu, et al. Proc Natl Acad Sci U S A. 2001 February 27;98(5):2808-2813.</div><p class="links"><a class="dblinks" href="/pubmed/11226322" ref="ordinalpos=2556&amp;ncbi_uid=41334&amp;link_uid=11226322">Citation</a><a class="dblinks" href="/pmc/articles/PMC30221/" ref="ordinalpos=2556&amp;ncbi_uid=41334&amp;link_uid=PMC30221">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2557.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3759173/figure/f3-cancers-03-02827/" ref="ordinalpos=2557&amp;ncbi_uid=5269502&amp;link_uid=PMC3759173" href="/pmc/articles/PMC3759173/"><img src="/pmc/articles/instance/3759173/bin/cancers-03-02827f3a.gif" src-large="/pmc/articles/instance/3759173/bin/cancers-03-02827f3a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3759173/figure/f3-cancers-03-02827/" ref="ordinalpos=2557&amp;ncbi_uid=5269502&amp;link_uid=PMC3759173" href="/pmc/articles/PMC3759173/"><img src="/pmc/articles/instance/3759173/bin/cancers-03-02827f3b.gif" src-large="/pmc/articles/instance/3759173/bin/cancers-03-02827f3b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759173/" ref="ordinalpos=2557&amp;ncbi_uid=5269502&amp;link_uid=PMC3759173" image-link="/pmc/articles/PMC3759173/figure/f3-cancers-03-02827/" class="imagepopup">Figure 3. From: Microarray Analysis in a Cell Death Resistant Glioma Cell Line to Identify <span class="highlight" style="background-color:">Signaling</span> Pathways and Novel Genes Controlling Resistance and Malignancy. </a></p><div class="supp"><p class="details"><div>The NF-κB <span class="highlight" style="background-color:">pathway</span> and subcellular localization of differentially regulated genes connected with NF-κB: Within the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> a total of 26 genes were differentially expressed in Group 1 (<b>a</b>); whereas 27 genes were shown to be regulated in Group 2 (<b>b</b>). (<i>Green marks</i> indicate upregulation, <i>red marks</i> downregulation of the respective mRNA in 229R cells.)</div></p></div><div class="aux"><div class="resc">Janina Seznec, et al. Cancers (Basel). 2011 September;3(3):2827-2843.</div><p class="links"><a class="dblinks" href="/pubmed/24212935" ref="ordinalpos=2557&amp;ncbi_uid=5269502&amp;link_uid=24212935">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759173/" ref="ordinalpos=2557&amp;ncbi_uid=5269502&amp;link_uid=PMC3759173">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2558.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4301577/figure/F3/" ref="ordinalpos=2558&amp;ncbi_uid=7083591&amp;link_uid=PMC4301577" href="/pmc/articles/PMC4301577/"><img src="/pmc/articles/instance/4301577/bin/nihms-656057-f0003.gif" src-large="/pmc/articles/instance/4301577/bin/nihms-656057-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4301577/" ref="ordinalpos=2558&amp;ncbi_uid=7083591&amp;link_uid=PMC4301577" image-link="/pmc/articles/PMC4301577/figure/F3/" class="imagepopup">Figure 3. Upregulated MSC-Derived Shh Corresponds to TGFβ <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activation.  From: Gastritis Promotes an Activated Bone Marrow-Derived Mesenchymal Stem Cell with a Phenotype Reminiscent of a Cancer-Promoting Cell. </a></p><div class="supp"><p class="details"><div>Luminex-based assay was performed for analysis of TGFβ cytokine levels in the (<b>A</b>) gastric tissue and (<b>B</b>) serum of 3 and 6 month BL/6 and GKO animals used for MSC isolation. An increased concentration of TGFβ is detected by 3 months in both the stomach and serum of GKO animals compared to BL/6 mice. (<b>C</b>) Whole bone marrow isolated from 6 month BL/6 and GKO mice shows significantly more phospho-Smad3 in the GKO versus BL/6 group, indicating TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. (<b>D</b>) Quantification of immunoblots shown in C. n=4 mice per group.</div></p></div><div class="aux"><div class="resc">Jessica M. Donnelly, et al. Dig Dis Sci. ;59(3):569-582.</div><p class="links"><a class="dblinks" href="/pubmed/24202649" ref="ordinalpos=2558&amp;ncbi_uid=7083591&amp;link_uid=24202649">Citation</a><a class="dblinks" href="/pmc/articles/PMC4301577/" ref="ordinalpos=2558&amp;ncbi_uid=7083591&amp;link_uid=PMC4301577">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2559.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3287101/figure/F3/" ref="ordinalpos=2559&amp;ncbi_uid=3828157&amp;link_uid=PMC3287101" href="/pmc/articles/PMC3287101/"><img src="/pmc/articles/instance/3287101/bin/1756-6606-5-1-3.gif" src-large="/pmc/articles/instance/3287101/bin/1756-6606-5-1-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3287101/" ref="ordinalpos=2559&amp;ncbi_uid=3828157&amp;link_uid=PMC3287101" image-link="/pmc/articles/PMC3287101/figure/F3/" class="imagepopup">Figure 3.  From: NMDA receptor activation stimulates transcription-independent rapid wnt5a protein synthesis via the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not required for the NMDAR-dependent Wnt5a protein synthesis</b>. Primary cortical neurons (10 DIV) were pre-treated with vehicle (Control) or 25nΜ Rapamycin for 30 min, followed by addition of 50 μΜ NMDA for 15 min. Western blotting analysis of Wnt5a and phosphor-P70S6K proteins were performed. Graphs (mean ± SEM) are from four independent experiments (*, p &lt; 0.05; **, p &lt; 0.01; #, p &gt; 0.05; One-way ANOVA).</div></p></div><div class="aux"><div class="resc">Yichen Li, et al. Mol Brain. 2012;5:1-1.</div><p class="links"><a class="dblinks" href="/pubmed/22217152" ref="ordinalpos=2559&amp;ncbi_uid=3828157&amp;link_uid=22217152">Citation</a><a class="dblinks" href="/pmc/articles/PMC3287101/" ref="ordinalpos=2559&amp;ncbi_uid=3828157&amp;link_uid=PMC3287101">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2560.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2582405/figure/F4/" ref="ordinalpos=2560&amp;ncbi_uid=1583754&amp;link_uid=PMC2582405" href="/pmc/articles/PMC2582405/"><img src="/pmc/articles/instance/2582405/bin/nihms76090f4.gif" src-large="/pmc/articles/instance/2582405/bin/nihms76090f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2582405/" ref="ordinalpos=2560&amp;ncbi_uid=1583754&amp;link_uid=PMC2582405" image-link="/pmc/articles/PMC2582405/figure/F4/" class="imagepopup">Fig. 4.  From: Effect of artemisinins and other endoperoxides on nitric oxide-related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RAW 264.7 mouse macrophage cells. </a></p><div class="supp"><p class="details"><div>Identification of (A) functional groups of genes and (B) <span class="highlight" style="background-color:">signaling</span> pathways regulated upon artesunate treatment in RAW 264.7 cells. Gene expression of cells treated with 1 × IC<sub>50</sub> or 10 × IC<sub>50</sub> of artesunate was compared to gene expression in untreated cells. The evaluation of differentially expressed genes was performed using the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software, version 5.5 (see Material and methods).</div></p></div><div class="aux"><div class="resc">V. Badireenath Konkimalla, et al. Nitric Oxide. ;19(2):184-191.</div><p class="links"><a class="dblinks" href="/pubmed/18472018" ref="ordinalpos=2560&amp;ncbi_uid=1583754&amp;link_uid=18472018">Citation</a><a class="dblinks" href="/pmc/articles/PMC2582405/" ref="ordinalpos=2560&amp;ncbi_uid=1583754&amp;link_uid=PMC2582405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2561.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3776247/figure/F1/" ref="ordinalpos=2561&amp;ncbi_uid=5502810&amp;link_uid=PMC3776247" href="/pmc/articles/PMC3776247/"><img src="/pmc/articles/instance/3776247/bin/2050-7771-1-5-1.gif" src-large="/pmc/articles/instance/3776247/bin/2050-7771-1-5-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3776247/" ref="ordinalpos=2561&amp;ncbi_uid=5502810&amp;link_uid=PMC3776247" image-link="/pmc/articles/PMC3776247/figure/F1/" class="imagepopup">Figure 1.  From: Dysregulation of JAK-STAT <span class="highlight" style="background-color:">pathway</span> in hematological malignancies and JAK inhibitors for clinical application. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> cascade of JAK-STAT <span class="highlight" style="background-color:">pathway</span>.</b> Binding of the ligand to cytokine receptor induces receptor dimerization and activation of receptor associated JAK kinase, which in turn phosphorylates STAT proteins. After forming a homodimer, STAT proteins translocate to the nucleus to control gene expression. Negative regulation of the JAK-STAT <span class="highlight" style="background-color:">pathway</span> is provided by phospho-tyrosine phosphatases (PTP), member for SOCS family proteins and PIAS proteins (shown in red).</div></p></div><div class="aux"><div class="resc">Muhammad Furqan, et al. Biomark Res. 2013;1:5-5.</div><p class="links"><a class="dblinks" href="/pubmed/24252238" ref="ordinalpos=2561&amp;ncbi_uid=5502810&amp;link_uid=24252238">Citation</a><a class="dblinks" href="/pmc/articles/PMC3776247/" ref="ordinalpos=2561&amp;ncbi_uid=5502810&amp;link_uid=PMC3776247">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2562.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2963700/figure/F3/" ref="ordinalpos=2562&amp;ncbi_uid=3325623&amp;link_uid=PMC2963700" href="/pmc/articles/PMC2963700/"><img src="/pmc/articles/instance/2963700/bin/nihms232446f3a.gif" src-large="/pmc/articles/instance/2963700/bin/nihms232446f3a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2963700/figure/F3/" ref="ordinalpos=2562&amp;ncbi_uid=3325623&amp;link_uid=PMC2963700" href="/pmc/articles/PMC2963700/"><img src="/pmc/articles/instance/2963700/bin/nihms232446f3b.gif" src-large="/pmc/articles/instance/2963700/bin/nihms232446f3b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2963700/" ref="ordinalpos=2562&amp;ncbi_uid=3325623&amp;link_uid=PMC2963700" image-link="/pmc/articles/PMC2963700/figure/F3/" class="imagepopup">Figure 3.  From: Differential expression of genes in the calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> underlies lesion development in the LDb mouse model of atherosclerosis. </a></p><div class="supp"><p class="details"><div>Fig. 3A. Western blot analysis of genes associated with calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in aortic tissues obtained from C57BL/6 and LDb mice<br />Selected proteins associated with calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were analyzed by Western blot in aortic tissues obtained from C57BL/6 and LDb mice at 2-months and 8-months of ages fed a chow diet. Aortic homogenates (15 μg) were separated by SDS/PAGE. Protein abundances of Gnaq, Calm1, Atp2a2 (SERCA), Ptger3 (EP3), and Mlck (Mylk) were analyzed and quantified using an Odyssey infrared imaging system (LI-COR). Gapdh was used as control to demonstrate equal sample loading to each lane. Each protein was normalized with the corresponding Gapdh value. Statistical analysis was performed as described in the Methods. The symbol of * represents significant difference of p&lt; 0.05. The ratios of LDb/C57BL6 of 2-month and 8-month of each protein are shown.<br />Fig. 3B. Western blot analysis of genes associated with calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in primary aortic endothelial cells obtained from C57BL/6 and LDb mice<br />Aortic endothelial cell protein (15 μg) of LP and LR from C57BL/6 and LDb mice were separated by SDS/PAGE. Protein abundances of Gnaq, Calm1, Atp2a2 (SERCA), Ptger3 (EP3), and Mlck (Mylk) were analyzed and quantified using an Odyssey infrared imaging system (LI-COR). Gapdh was used as control to demonstrate equal sample loading to each lane. . Each protein was normalized with the corresponding Gapdh value. Statistical analysis was performed as described in the Methods. The symbol of * represents significant difference of p&lt; 0.05. The ratios of LDb/C57BL6 of LP and LR of each protein are shown.</div></p></div><div class="aux"><div class="resc">Solida Mak, et al. Atherosclerosis. ;213(1):40-51.</div><p class="links"><a class="dblinks" href="/pubmed/20667539" ref="ordinalpos=2562&amp;ncbi_uid=3325623&amp;link_uid=20667539">Citation</a><a class="dblinks" href="/pmc/articles/PMC2963700/" ref="ordinalpos=2562&amp;ncbi_uid=3325623&amp;link_uid=PMC2963700">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2563.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3704018/figure/fig05/" ref="ordinalpos=2563&amp;ncbi_uid=5672224&amp;link_uid=PMC3704018" href="/pmc/articles/PMC3704018/"><img src="/pmc/articles/instance/3704018/bin/pnas.1221803110fig05.gif" src-large="/pmc/articles/instance/3704018/bin/pnas.1221803110fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3704018/" ref="ordinalpos=2563&amp;ncbi_uid=5672224&amp;link_uid=PMC3704018" image-link="/pmc/articles/PMC3704018/figure/fig05/" class="imagepopup">Fig. 5. From: Muscarinic acetylcholine receptor activation blocks long-term potentiation at cerebellar parallel fiber-Purkinje cell synapses via cannabinoid <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Suggested <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of M1/M3 muscarinic receptors. Activation of M1/M3 receptors on Purkinje cells linked to the activation of PLCβ triggers cannabinoid production and release that is sufficient to block presynaptically expressed LTP. eCBs, endocannabinoids; PIP<sub>2</sub>, phosphatidylinositol 4,5-bisphosphate; VGCC, voltage-gated calcium channel.</div></p></div><div class="aux"><div class="resc">Lorenzo Rinaldo, et al. Proc Natl Acad Sci U S A. 2013 July 2;110(27):11181-11186.</div><p class="links"><a class="dblinks" href="/pubmed/23776234" ref="ordinalpos=2563&amp;ncbi_uid=5672224&amp;link_uid=23776234">Citation</a><a class="dblinks" href="/pmc/articles/PMC3704018/" ref="ordinalpos=2563&amp;ncbi_uid=5672224&amp;link_uid=PMC3704018">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2564.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3886813/figure/F2/" ref="ordinalpos=2564&amp;ncbi_uid=5690746&amp;link_uid=PMC3886813" href="/pmc/articles/PMC3886813/"><img src="/pmc/articles/instance/3886813/bin/nihms199929f2.gif" src-large="/pmc/articles/instance/3886813/bin/nihms199929f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3886813/" ref="ordinalpos=2564&amp;ncbi_uid=5690746&amp;link_uid=PMC3886813" image-link="/pmc/articles/PMC3886813/figure/F2/" class="imagepopup">Figure 2. IL-6 family <span class="highlight" style="background-color:">signaling</span> through the JAK/STAT-3 <span class="highlight" style="background-color:">pathway</span>.  From: NF-?B and STAT3 <span class="highlight" style="background-color:">signaling</span> in glioma: targets for future therapies. </a></p><div class="supp"><p class="details"><div>OSM binding to the OSM receptor activates intracellular JAK kinases, predominantly JAK2, which phosphorylate (yellow circles) STAT-3. Once phosphorylated, STAT-3 molecules dimerize, translocate to the nucleus and bind to STAT response elements in the promoters of target genes. OSM: Oncostatin-M.</div></p></div><div class="aux"><div class="resc">George P Atkinson, et al. Expert Rev Neurother. ;10(4):10.1586/ern.10.21.</div><p class="links"><a class="dblinks" href="/pubmed/20367209" ref="ordinalpos=2564&amp;ncbi_uid=5690746&amp;link_uid=20367209">Citation</a><a class="dblinks" href="/pmc/articles/PMC3886813/" ref="ordinalpos=2564&amp;ncbi_uid=5690746&amp;link_uid=PMC3886813">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2565.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3530037/figure/A006569F3/" ref="ordinalpos=2565&amp;ncbi_uid=7009211&amp;link_uid=PMC3530037" href="/pmc/articles/PMC3530037/"><img src="/pmc/articles/instance/3530037/bin/cshperspectmed-ANG-006569_F3.gif" src-large="/pmc/articles/instance/3530037/bin/cshperspectmed-ANG-006569_F3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3530037/" ref="ordinalpos=2565&amp;ncbi_uid=7009211&amp;link_uid=PMC3530037" image-link="/pmc/articles/PMC3530037/figure/A006569F3/" class="imagepopup">Figure 3. From: VEGF and Notch in Tip and Stalk Cell Selection. </a></p><div class="supp"><p class="details"><div>Canonical Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Notch <span class="highlight" style="background-color:">pathway</span> is an evolutionarily conserved intercellular <span class="highlight" style="background-color:">signaling</span> mechanism implicated in cell fate determination and differentiation of endothelial cells. Notch <span class="highlight" style="background-color:">signaling</span> begins with receptor–ligand interaction between neighboring cells. This interaction triggers a series of proteolytic cleavages of the Notch receptor. The final one, catalyzed by the γ-secretase complex, releases the active Notch intercellular domain (NICD) from the cell membrane. NICD is translocated to the nucleus, where it interacts directly with the transcription factor CSL. Following binding of NICD to CSL, the repressor complex is converted into an activating complex by displacing corepressors and recruiting coactivators. After the coactivator complex is recruited, the transcription of promoters that contains CSL-binding elements is induced. Hairy/enhancer of split (HES) and HES-related proteins (HEY/HRT/HERP) family genes are included among the target genes of Notch <span class="highlight" style="background-color:">signaling</span>. Dll4 ligand is also a target gene regulated by the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Raquel Blanco, et al. Cold Spring Harb Perspect Med. 2013 January;3(1):a006569.</div><p class="links"><a class="dblinks" href="/pubmed/23085847" ref="ordinalpos=2565&amp;ncbi_uid=7009211&amp;link_uid=23085847">Citation</a><a class="dblinks" href="/pmc/articles/PMC3530037/" ref="ordinalpos=2565&amp;ncbi_uid=7009211&amp;link_uid=PMC3530037">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2566.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267482/figure/fig02/" ref="ordinalpos=2566&amp;ncbi_uid=3781550&amp;link_uid=PMC3267482" href="/pmc/articles/PMC3267482/"><img src="/pmc/articles/instance/3267482/bin/ijcep0005-0023-f2.gif" src-large="/pmc/articles/instance/3267482/bin/ijcep0005-0023-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267482/" ref="ordinalpos=2566&amp;ncbi_uid=3781550&amp;link_uid=PMC3267482" image-link="/pmc/articles/PMC3267482/figure/fig02/" class="imagepopup">Figure 2.  From: c-Ret-mediated hearing losses. </a></p><div class="supp"><p class="details"><div>Schematic summary of c-Ret-mediated <span class="highlight" style="background-color:">signaling</span> in SGNs. (A and B) Schematic illustration of a mechanistic model for c-Ret-mediated congenital deafness with neurodegeneration of SGNs in c-Ret-KI-mice. (A) Wild-type mice (WT) acquire normal hearing with phosphorylation of the c-Ret Y1062-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. P in A indicates phosphorylation. (B) Impairments of the c-Ret Y1062-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (blue double line and downward dashed arrows in B) cause decreased expression of calbindin D28k (blue arrow in B) via Akt/NF-κB <span class="highlight" style="background-color:">signaling</span>, resulting in auditory nerve degeneration in c-Ret-KI<sup>Y1062F/Y1062F</sup>-mice (c-Ret Y1062F). This figure was drawn on the basis of results of previous studies [1, 18].</div></p></div><div class="aux"><div class="resc">Nobutaka Ohgami, et al. Int J Clin Exp Pathol. 2012;5(1):23-28.</div><p class="links"><a class="dblinks" href="/pubmed/22295143" ref="ordinalpos=2566&amp;ncbi_uid=3781550&amp;link_uid=22295143">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267482/" ref="ordinalpos=2566&amp;ncbi_uid=3781550&amp;link_uid=PMC3267482">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2567.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1304358/figure/fig1/" ref="ordinalpos=2567&amp;ncbi_uid=663006&amp;link_uid=PMC1304358" href="/pmc/articles/PMC1304358/"><img src="/pmc/articles/instance/1304358/bin/biophysj00029884F01_4C.gif" src-large="/pmc/articles/instance/1304358/bin/biophysj00029884F01_4C.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1304358/" ref="ordinalpos=2567&amp;ncbi_uid=663006&amp;link_uid=PMC1304358" image-link="/pmc/articles/PMC1304358/figure/fig1/" class="imagepopup">FIGURE 1.  From: The JAK-STAT <span class="highlight" style="background-color:">Signaling</span> Network in the Human B-Cell: An Extreme <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Analysis. </a></p><div class="supp"><p class="details"><div>Schematic of generalized reactions for the JAK-STAT <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Jason A. Papin, et al. Biophys J. 2004 July;87(1):37-46.</div><p class="links"><a class="dblinks" href="/pubmed/15240442" ref="ordinalpos=2567&amp;ncbi_uid=663006&amp;link_uid=15240442">Citation</a><a class="dblinks" href="/pmc/articles/PMC1304358/" ref="ordinalpos=2567&amp;ncbi_uid=663006&amp;link_uid=PMC1304358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2568.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3148700/figure/F9/" ref="ordinalpos=2568&amp;ncbi_uid=3449390&amp;link_uid=PMC3148700" href="/pmc/articles/PMC3148700/"><img src="/pmc/articles/instance/3148700/bin/asn0081147830009.gif" src-large="/pmc/articles/instance/3148700/bin/asn0081147830009.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3148700/" ref="ordinalpos=2568&amp;ncbi_uid=3449390&amp;link_uid=PMC3148700" image-link="/pmc/articles/PMC3148700/figure/F9/" class="imagepopup">Figure 9. From: Calcium Mediates Glomerular Filtration through Calcineurin and mTORC2/Akt <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed mechanism of podocyte foot process effacement by receptor-operated calcium channel <span class="highlight" style="background-color:">signaling</span>. PS initiates the activation of receptor-operated calcium channels. Calcium <span class="highlight" style="background-color:">signaling</span> can in turn activate both the calcineurin and mTORC2 pathways. Calcineurin dephosphorylates Synpo, thereby making it available for degradation by CatL and changing podocyte morphology. Calcium also activates mTORC2, in turn inducing Akt activity and affecting the cytoskeleton. Inhibition of either <span class="highlight" style="background-color:">pathway</span> is sufficient to block PS-induced damage to isolated glomeruli.</div></p></div><div class="aux"><div class="resc">John Vassiliadis, et al. J Am Soc Nephrol. 2011 August;22(8):1453-1461.</div><p class="links"><a class="dblinks" href="/pubmed/21784900" ref="ordinalpos=2568&amp;ncbi_uid=3449390&amp;link_uid=21784900">Citation</a><a class="dblinks" href="/pmc/articles/PMC3148700/" ref="ordinalpos=2568&amp;ncbi_uid=3449390&amp;link_uid=PMC3148700">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2569.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889251/figure/Fig9/" ref="ordinalpos=2569&amp;ncbi_uid=5702969&amp;link_uid=PMC3889251" href="/pmc/articles/PMC3889251/"><img src="/pmc/articles/instance/3889251/bin/12079_2013_199_Fig9_HTML.gif" src-large="/pmc/articles/instance/3889251/bin/12079_2013_199_Fig9_HTML.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889251/" ref="ordinalpos=2569&amp;ncbi_uid=5702969&amp;link_uid=PMC3889251" image-link="/pmc/articles/PMC3889251/figure/Fig9/" class="imagepopup">Fig. 9.  From: Inhibition of CK2 binding to BMPRIa induces C2C12 differentiation into osteoblasts and adipocytes. </a></p><div class="supp"><p class="details"><div>New proposed model of BMP2-induced C2C12 cells differentiation. Release of CK2 from different CK2-BMPRIa interaction sited regulates differentiation into osteoblasts (release from CK2.2, aa 324–328) via activation of Smad4 and mTor <span class="highlight" style="background-color:">signaling</span> pathways or via ERK/MEK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (release from CK2.3, aa 213–217), and adipocytes (release from CK2.2, aa 324–328) by activation of Smad4, p38, and MEK <span class="highlight" style="background-color:">signaling</span> pathways</div></p></div><div class="aux"><div class="resc">Oleksandra Moseychuk, et al. J Cell Commun Signal. 2013 December;7(4):265-278.</div><p class="links"><a class="dblinks" href="/pubmed/23637019" ref="ordinalpos=2569&amp;ncbi_uid=5702969&amp;link_uid=23637019">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889251/" ref="ordinalpos=2569&amp;ncbi_uid=5702969&amp;link_uid=PMC3889251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2570.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3635912/figure/F4/" ref="ordinalpos=2570&amp;ncbi_uid=4829972&amp;link_uid=PMC3635912" href="/pmc/articles/PMC3635912/"><img src="/pmc/articles/instance/3635912/bin/1476-4598-12-22-4.gif" src-large="/pmc/articles/instance/3635912/bin/1476-4598-12-22-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3635912/" ref="ordinalpos=2570&amp;ncbi_uid=4829972&amp;link_uid=PMC3635912" image-link="/pmc/articles/PMC3635912/figure/F4/" class="imagepopup">Figure 4.  From: Timp1 interacts with beta-1 integrin and CD63 along melanoma genesis and confers anoikis resistance by activating PI3-K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> independently of Akt phosphorylation. </a></p><div class="supp"><p class="details"><div><b>PI3-K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in </b><b><i>anoikis </i></b><b>resistance phenotype conferred by Timp1.</b> The MaGFP and MaT1S cell lines were treated overnight with PI3-K inhibitors, Wortmannin (<b>A</b>) or LY294002 (<b>B</b>), and their clonogenic capability was evaluated. <b>C.</b> Melan-a melanocytes stably transfected with GFP (control transfection, MaGFP) and Timp1 (MaT1S) were maintained in suspension for 1, 3, 5 and 24 hours. The Akt activation was assessed by Western blotting. *p &lt; 0.05, **p &lt; 0.01, ****p &lt; 0.0001.</div></p></div><div class="aux"><div class="resc">Mariana Toricelli, et al. Mol Cancer. 2013;12:22-22.</div><p class="links"><a class="dblinks" href="/pubmed/23522389" ref="ordinalpos=2570&amp;ncbi_uid=4829972&amp;link_uid=23522389">Citation</a><a class="dblinks" href="/pmc/articles/PMC3635912/" ref="ordinalpos=2570&amp;ncbi_uid=4829972&amp;link_uid=PMC3635912">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2571.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4368808/figure/pone.0119872.g003/" ref="ordinalpos=2571&amp;ncbi_uid=7334692&amp;link_uid=PMC4368808" href="/pmc/articles/PMC4368808/"><img src="/pmc/articles/instance/4368808/bin/pone.0119872.g003.gif" src-large="/pmc/articles/instance/4368808/bin/pone.0119872.g003.jpg" alt="Fig 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4368808/" ref="ordinalpos=2571&amp;ncbi_uid=7334692&amp;link_uid=PMC4368808" image-link="/pmc/articles/PMC4368808/figure/pone.0119872.g003/" class="imagepopup">Fig 3. IL-22 activates the STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in GBM cell lines..  From: IL22/IL-22R <span class="highlight" style="background-color:">Pathway</span> Induces Cell Survival in Human Glioblastoma Cells. </a></p><div class="supp"><p class="details"><div>(A, B) The ability of IL-22 to activate <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in GBM cells was assessed using antibodies specific to STAT3 and Phospho-STAT3 (P-STAT3). U87MG (A) and U118MG (B) cells were stimulated with IL-22 and harvested at indicated times. Thirty mg of protein lysates was analyzed for P-STAT3 (Tyr705) and total STAT3 by western blot analysis. The density of each P-STAT3 band was corrected for variance in loading, using the density of the corresponding total STAT3. The expression level was evaluated as the ratio of phosphorylated STAT3 protein densities between control (0 min) and treated cells. Histograms are means ± SEM of three independent experiments. *, <i>p</i> &lt; 0.05; **, <i>p</i> &lt; 0.01; ***, <i>p</i> &lt; 0.001; when compared with control.</div></p></div><div class="aux"><div class="resc">Hussein Akil, et al. PLoS One. 2015;10(3):e0119872.</div><p class="links"><a class="dblinks" href="/pubmed/25793261" ref="ordinalpos=2571&amp;ncbi_uid=7334692&amp;link_uid=25793261">Citation</a><a class="dblinks" href="/pmc/articles/PMC4368808/" ref="ordinalpos=2571&amp;ncbi_uid=7334692&amp;link_uid=PMC4368808">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2572.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3601827/figure/F5/" ref="ordinalpos=2572&amp;ncbi_uid=4722655&amp;link_uid=PMC3601827" href="/pmc/articles/PMC3601827/"><img src="/pmc/articles/instance/3601827/bin/nihms442827f5.gif" src-large="/pmc/articles/instance/3601827/bin/nihms442827f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3601827/" ref="ordinalpos=2572&amp;ncbi_uid=4722655&amp;link_uid=PMC3601827" image-link="/pmc/articles/PMC3601827/figure/F5/" class="imagepopup">Figure 5.  From: Corticotropin-releasing factor and noradrenergic <span class="highlight" style="background-color:">signaling</span> exert reciprocal control over startle reactivity. </a></p><div class="supp"><p class="details"><div>Schematic proposing a circuit model of CRF/NE <span class="highlight" style="background-color:">signaling</span> effects on startle reactivity. In this model, CRF<sub>1</sub> receptor activation modulates startle by activating two pathways that converge at the nucleus reticularis pontis caudalis (PRC) of the startle circuit. One <span class="highlight" style="background-color:">pathway</span> is the LC-PRC <span class="highlight" style="background-color:">pathway</span> which requires NE release and α<sub>1</sub> receptor activation at the PRC. The second <span class="highlight" style="background-color:">pathway</span>, the BNST-PRC <span class="highlight" style="background-color:">pathway</span>, involves CRF<sub>1</sub> receptor activation of glutamatergic neurons projecting to the PRC. See text for additional details. Abbreviations: CRF – corticotropin releasing factor; NE – norepinephrine; LC - locus coeruleus; BNST – bed nucleus stria terminalis; GLU – glutamate; PRC: nucleus reticularis pontis caudalis</div></p></div><div class="aux"><div class="resc">Jodi E. Gresack, et al. Int J Neuropsychopharmacol. 2011 October;14(9):1179-1194.</div><p class="links"><a class="dblinks" href="/pubmed/21205416" ref="ordinalpos=2572&amp;ncbi_uid=4722655&amp;link_uid=21205416">Citation</a><a class="dblinks" href="/pmc/articles/PMC3601827/" ref="ordinalpos=2572&amp;ncbi_uid=4722655&amp;link_uid=PMC3601827">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2573.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3831500/figure/fig05/" ref="ordinalpos=2573&amp;ncbi_uid=5518020&amp;link_uid=PMC3831500" href="/pmc/articles/PMC3831500/"><img src="/pmc/articles/instance/3831500/bin/pnas.1208764110fig05.gif" src-large="/pmc/articles/instance/3831500/bin/pnas.1208764110fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3831500/" ref="ordinalpos=2573&amp;ncbi_uid=5518020&amp;link_uid=PMC3831500" image-link="/pmc/articles/PMC3831500/figure/fig05/" class="imagepopup">Fig. 5. From: Neuronal activity regulates astrocytic Nrf2 <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Model for regulation of Nrf2 <span class="highlight" style="background-color:">signaling</span> at tripartite neuron–astrocyte synapse. At a low activity synapse (<i>A</i>), glutamate secreted by presynaptic terminus (red arrow) mildly activates Nrf2 <span class="highlight" style="background-color:">signaling</span> in adjacent astrocytes through mGluR1/5 and Ca<sup>2+</sup>-dependent <span class="highlight" style="background-color:">pathway</span>. At a high activity synapse (<i>B</i>), increased secretion of glutamate and other (currently unidentified) soluble factors secreted by a presynaptic neuron (brown arrow), postsynaptic neuron (blue arrow), or astrocyte itself (green arrow) strongly activate astrocytic Nrf2 <span class="highlight" style="background-color:">signaling</span>. This regulatory mechanism matches Nrf2 activity in a perisynaptic astrocyte to the firing rate of the adjacent neuronal synapse, maximizing the neuroprotection while minimizing the associated physiologic costs.</div></p></div><div class="aux"><div class="resc">Agata Habas, et al. Proc Natl Acad Sci U S A. 2013 November 5;110(45):18291-18296.</div><p class="links"><a class="dblinks" href="/pubmed/24145448" ref="ordinalpos=2573&amp;ncbi_uid=5518020&amp;link_uid=24145448">Citation</a><a class="dblinks" href="/pmc/articles/PMC3831500/" ref="ordinalpos=2573&amp;ncbi_uid=5518020&amp;link_uid=PMC3831500">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2574.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4369723/figure/f5/" ref="ordinalpos=2574&amp;ncbi_uid=7386895&amp;link_uid=PMC4369723" href="/pmc/articles/PMC4369723/"><img src="/pmc/articles/instance/4369723/bin/srep09395-f5.gif" src-large="/pmc/articles/instance/4369723/bin/srep09395-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4369723/" ref="ordinalpos=2574&amp;ncbi_uid=7386895&amp;link_uid=PMC4369723" image-link="/pmc/articles/PMC4369723/figure/f5/" class="imagepopup">Figure 5. T4 potentiates the EGF induced cellular proliferation and keratinization..  From: Integrating Clinical Indexes into Four-Diagnostic Information Contributes to the Traditional Chinese Medicine (TCM) Syndrome Diagnosis of Chronic Hepatitis B. </a></p><div class="supp"><p class="details"><div>The proliferative signal of EGF is transmitted through two pathways: (1) JAK/STAT <span class="highlight" style="background-color:">pathway</span> and (2) Ras/MAPK <span class="highlight" style="background-color:">pathway</span>, ultimately causing activation of transcription factors such as c-Fos, STAT1 and STAT3, which all have important roles in proliferation and keratinization. T<sub>4</sub> was reported to potentiate the EGF <span class="highlight" style="background-color:">signaling</span> through both the STAT <span class="highlight" style="background-color:">pathway</span> and the MAPK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hong Kang, et al. Sci Rep. 2015;5:9395.</div><p class="links"><a class="dblinks" href="/pubmed/25797918" ref="ordinalpos=2574&amp;ncbi_uid=7386895&amp;link_uid=25797918">Citation</a><a class="dblinks" href="/pmc/articles/PMC4369723/" ref="ordinalpos=2574&amp;ncbi_uid=7386895&amp;link_uid=PMC4369723">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2575.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3836044/figure/F2/" ref="ordinalpos=2575&amp;ncbi_uid=5510992&amp;link_uid=PMC3836044" href="/pmc/articles/PMC3836044/"><img src="/pmc/articles/instance/3836044/bin/nihms528215f2.gif" src-large="/pmc/articles/instance/3836044/bin/nihms528215f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3836044/" ref="ordinalpos=2575&amp;ncbi_uid=5510992&amp;link_uid=PMC3836044" image-link="/pmc/articles/PMC3836044/figure/F2/" class="imagepopup">Fig. 2.  From: Role of ?-Catenin/TCF-4 <span class="highlight" style="background-color:">Signaling</span> in HIV Replication and Pathogenesis: Insights to Informing Novel Anti-HIV Molecular Therapeutics. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways interface with Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. Akt and p38α can both activate the canonical <span class="highlight" style="background-color:">pathway</span> by inhibiting GSK3β activity, preventing it from phosphorylating β-catenin. Akt phosphorylates GSK3β at the N terminus on Ser9, while P38α phosphorylates the C-terminal residue Ser839. Conversely, activation of the non-canonical Wnt/JNK <span class="highlight" style="background-color:">pathway</span> has been observed to inhibit nuclear accumulation of β-catenin under some circumstances. Release of calcium from intracellular stores, such as occurs when the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> is activated, can lead to activation of nemo-like kinase (<i>NLK</i>), which directly phosphorylates TCF to inhibit interaction with β-catenin. Induction of nuclear factor of activated T cells (<i>NFAT</i>) via the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> has also been observed to promote GSK3β-independent degradation of β-catenin, but the mechanism is poorly understood. <i>MKKK</i> mitogen-activated protein kinase (<i>MAPK</i>) kinase kinase, <i>MKK</i> MAPK kinase, <i>PI3K</i> phosphoinositide-3-kinase, <i>PLC</i> phospholipase C, <i>PKC</i> protein kinase C, <i>CaMKII</i> calcium/calmodulin-dependent kinase II, <i>JNK</i> c-Jun N-terminal kinase</div></p></div><div class="aux"><div class="resc">Lisa J. Henderson, et al. J Neuroimmune Pharmacol. ;6(2):10.1007/s11481-011-9266-7.</div><p class="links"><a class="dblinks" href="/pubmed/21384147" ref="ordinalpos=2575&amp;ncbi_uid=5510992&amp;link_uid=21384147">Citation</a><a class="dblinks" href="/pmc/articles/PMC3836044/" ref="ordinalpos=2575&amp;ncbi_uid=5510992&amp;link_uid=PMC3836044">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2576.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3648504/figure/pone-0063674-g006/" ref="ordinalpos=2576&amp;ncbi_uid=4888010&amp;link_uid=PMC3648504" href="/pmc/articles/PMC3648504/"><img src="/pmc/articles/instance/3648504/bin/pone.0063674.g006.gif" src-large="/pmc/articles/instance/3648504/bin/pone.0063674.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3648504/" ref="ordinalpos=2576&amp;ncbi_uid=4888010&amp;link_uid=PMC3648504" image-link="/pmc/articles/PMC3648504/figure/pone-0063674-g006/" class="imagepopup">Figure 6. Proposed mechanism by which Jam-A regulates corneal inflammation, neovascularization, and scarring..  From: Junctional Adhesion Molecule-A Regulates Vascular Endothelial Growth Factor Receptor-2 <span class="highlight" style="background-color:">Signaling</span>-Dependent Mouse Corneal Wound Healing. </a></p><div class="supp"><p class="details"><div>Jam-A negatively governs VEGF-A expression thereby regulating VEGFR-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The VEGF-A/VEGFR-2 <span class="highlight" style="background-color:">pathway</span> leads to angiogenesis as well as recruits inflammatory cells. The VEGF-A/VEGFR-2 <span class="highlight" style="background-color:">pathway</span> also has a positive feed back loop with TGFβ <span class="highlight" style="background-color:">pathway</span> that along with ECM remodeling leads to scarring. Introduction of silk suture results in inflammation leading to neovascularization and no scarring due to the presence of an intact Descemet's membrane. Full thickness wounds that cause a break in the Descemet's membrane lead to ECM remodeling and TGFβ <span class="highlight" style="background-color:">pathway</span> activation that together can lead to scarring. Jam-A deficient mice exhibit increased VEGF-A/VEGFR-2 <span class="highlight" style="background-color:">signaling</span> and demonstrate increased inflammation, angiogenesis as well as scarring compared to WT mice.</div></p></div><div class="aux"><div class="resc">Sharmila Chatterjee, et al. PLoS One. 2013;8(5):e63674.</div><p class="links"><a class="dblinks" href="/pubmed/23667656" ref="ordinalpos=2576&amp;ncbi_uid=4888010&amp;link_uid=23667656">Citation</a><a class="dblinks" href="/pmc/articles/PMC3648504/" ref="ordinalpos=2576&amp;ncbi_uid=4888010&amp;link_uid=PMC3648504">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2577.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3732476/figure/F2/" ref="ordinalpos=2577&amp;ncbi_uid=6437160&amp;link_uid=PMC3732476" href="/pmc/articles/PMC3732476/"><img src="/pmc/articles/instance/3732476/bin/nihms484782f2.gif" src-large="/pmc/articles/instance/3732476/bin/nihms484782f2.jpg" alt="Fig 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3732476/" ref="ordinalpos=2577&amp;ncbi_uid=6437160&amp;link_uid=PMC3732476" image-link="/pmc/articles/PMC3732476/figure/F2/" class="imagepopup">Fig 2.  From: Notch Inhibitors for Cancer Treatment. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Ingrid Espinoza, et al. Pharmacol Ther. ;139(2):95-110.</div><p class="links"><a class="dblinks" href="/pubmed/23458608" ref="ordinalpos=2577&amp;ncbi_uid=6437160&amp;link_uid=23458608">Citation</a><a class="dblinks" href="/pmc/articles/PMC3732476/" ref="ordinalpos=2577&amp;ncbi_uid=6437160&amp;link_uid=PMC3732476">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2578.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3797786/figure/pone-0076476-g003/" ref="ordinalpos=2578&amp;ncbi_uid=5402025&amp;link_uid=PMC3797786" href="/pmc/articles/PMC3797786/"><img src="/pmc/articles/instance/3797786/bin/pone.0076476.g003.gif" src-large="/pmc/articles/instance/3797786/bin/pone.0076476.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3797786/" ref="ordinalpos=2578&amp;ncbi_uid=5402025&amp;link_uid=PMC3797786" image-link="/pmc/articles/PMC3797786/figure/pone-0076476-g003/" class="imagepopup">Figure 3. Integrated <span class="highlight" style="background-color:">signaling</span> network..  From: Identification of Cisplatin-Regulated Metabolic Pathways in Pluripotent Stem Cells. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Schematic representation of metabolomics and transcriptomics data integration leading to identification of common <span class="highlight" style="background-color:">signaling</span> networks related to nucleotide metabolism, SAMe pathways, polyamine pathways and urea cycle, and arginine &amp; proline metabolism. (<b>B</b>) Integrated <span class="highlight" style="background-color:">signaling</span> network of metabolic enzymes and metabolites obtained with Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis. Clusters of significantly enriched canonical pathways and related enzymes and metabolites are highlighted. Upregulated enzymes in green, downregulated enzymes in red. Upregulated metabolites in dark red, downregulated metabolites in blue.</div></p></div><div class="aux"><div class="resc">Louise von Stechow, et al. PLoS One. 2013;8(10):e76476.</div><p class="links"><a class="dblinks" href="/pubmed/24146875" ref="ordinalpos=2578&amp;ncbi_uid=5402025&amp;link_uid=24146875">Citation</a><a class="dblinks" href="/pmc/articles/PMC3797786/" ref="ordinalpos=2578&amp;ncbi_uid=5402025&amp;link_uid=PMC3797786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2579.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3230665/figure/F2/" ref="ordinalpos=2579&amp;ncbi_uid=4405843&amp;link_uid=PMC3230665" href="/pmc/articles/PMC3230665/"><img src="/pmc/articles/instance/3230665/bin/nihms327830f2.gif" src-large="/pmc/articles/instance/3230665/bin/nihms327830f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3230665/" ref="ordinalpos=2579&amp;ncbi_uid=4405843&amp;link_uid=PMC3230665" image-link="/pmc/articles/PMC3230665/figure/F2/" class="imagepopup">Figure 2.  From: Cellular senescence: a link between cancer and age-related degenerative disease?. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> mechanisms used by senescent cells. An early event in the senescence response is an increase in the expression of IL-1α, a cytokine that is rarely secreted but rather is membrane-associated where it binds its juxtaposed receptor (juxtacrine <span class="highlight" style="background-color:">signaling</span>). IL-1αreceptor engagement triggers a <span class="highlight" style="background-color:">signaling</span> cascade that ultimately activates the NF-kB transcription factor that transcribes the genes for many of the pro-inflammatory components of the SASP. The senescence response also activates pathways, such as the p38MAPK <span class="highlight" style="background-color:">pathway</span>, which ultimately stimulates the transcription of genes that enforce the senescence growth arrest. SASP components also include secreted factors such as IL-6 and IL-8, which can reinforce the senescence growth arrest by autocrine <span class="highlight" style="background-color:">signaling</span>. Finally, SASP components can markedly affect the behavior of neighboring cells by paracrine <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Judith Campisi, et al. Semin Cancer Biol. ;21(6):354-359.</div><p class="links"><a class="dblinks" href="/pubmed/21925603" ref="ordinalpos=2579&amp;ncbi_uid=4405843&amp;link_uid=21925603">Citation</a><a class="dblinks" href="/pmc/articles/PMC3230665/" ref="ordinalpos=2579&amp;ncbi_uid=4405843&amp;link_uid=PMC3230665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2580.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2737473/figure/F5/" ref="ordinalpos=2580&amp;ncbi_uid=2059709&amp;link_uid=PMC2737473" href="/pmc/articles/PMC2737473/"><img src="/pmc/articles/instance/2737473/bin/nihms-126263-f0005.gif" src-large="/pmc/articles/instance/2737473/bin/nihms-126263-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2737473/" ref="ordinalpos=2580&amp;ncbi_uid=2059709&amp;link_uid=PMC2737473" image-link="/pmc/articles/PMC2737473/figure/F5/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Signaling</span> Pathways Linking Dopamine to the Circadian Clock.  From: Decoding the Epigenetic Language of Neuronal Plasticity. </a></p><div class="supp"><p class="details"><div>Dopamine controls many brain functions, including roles in behavior and cognition, motor activity, motivation, and reward. <span class="highlight" style="background-color:">Signaling</span> mediated by D2 receptor is envisaged to control chromatin remodeling via modulation of various enzymatic functions. By using either the ERK transduction <span class="highlight" style="background-color:">pathway</span> or the alternative AKT/GSK-3 <span class="highlight" style="background-color:">signaling</span> system, dopamine could control kinases that phosphorylate H3, such as Rsk2 and MSK1, or the enzymatic function of HATs, such as CBP or CLOCK. The GSK-3 kinase is inhibited by lithium and thereby is implicated in the treatment of mood disorders. These regulatory pathways may integrate circadian control, dopamine <span class="highlight" style="background-color:">signaling</span>, and chromatin remodeling.</div></p></div><div class="aux"><div class="resc">Emiliana Borrelli, et al. Neuron. ;60(6):961-974.</div><p class="links"><a class="dblinks" href="/pubmed/19109904" ref="ordinalpos=2580&amp;ncbi_uid=2059709&amp;link_uid=19109904">Citation</a><a class="dblinks" href="/pmc/articles/PMC2737473/" ref="ordinalpos=2580&amp;ncbi_uid=2059709&amp;link_uid=PMC2737473">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2581.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2823981/figure/F5/" ref="ordinalpos=2581&amp;ncbi_uid=2788433&amp;link_uid=PMC2823981" href="/pmc/articles/PMC2823981/"><img src="/pmc/articles/instance/2823981/bin/nihms167862f5.gif" src-large="/pmc/articles/instance/2823981/bin/nihms167862f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2823981/" ref="ordinalpos=2581&amp;ncbi_uid=2788433&amp;link_uid=PMC2823981" image-link="/pmc/articles/PMC2823981/figure/F5/" class="imagepopup">Figure 5. Major mechanisms proposed to regulate hormone-dependent transformation in some tumors.  From: RSK in Tumorigenesis: Connections to Steroid <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Estrogens stimulate ERα complex formation with cytoplasmic <span class="highlight" style="background-color:">signaling</span> proteins like MNAR, cas, and c-SRC. This complex activates the ERK1/2 signal transduction <span class="highlight" style="background-color:">pathway</span>, and phosphorylation of RSK. RSK1 <span class="highlight" style="background-color:">signaling</span> inhibits apoptosis via phosphorylation and inactivation of the pro-apoptotic protein Bad. RSK2 can translocate to the nucleus where it regulates nuclear targets that drive proliferation. In the nucleus RSK directly phosphorylates ERα stimulating its transcriptional activity.</div></p></div><div class="aux"><div class="resc">T.S. Karin Eisinger-Mathason, et al. Steroids. ;75(3):191-202.</div><p class="links"><a class="dblinks" href="/pubmed/20045011" ref="ordinalpos=2581&amp;ncbi_uid=2788433&amp;link_uid=20045011">Citation</a><a class="dblinks" href="/pmc/articles/PMC2823981/" ref="ordinalpos=2581&amp;ncbi_uid=2788433&amp;link_uid=PMC2823981">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2582.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3744053/figure/F5/" ref="ordinalpos=2582&amp;ncbi_uid=5213585&amp;link_uid=PMC3744053" href="/pmc/articles/PMC3744053/"><img src="/pmc/articles/instance/3744053/bin/fonc-03-00211-g005.gif" src-large="/pmc/articles/instance/3744053/bin/fonc-03-00211-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3744053/" ref="ordinalpos=2582&amp;ncbi_uid=5213585&amp;link_uid=PMC3744053" image-link="/pmc/articles/PMC3744053/figure/F5/" class="imagepopup">Figure 5.  From: Vascular Permeability and Drug Delivery in Cancers. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> pathways of anti-permeability factors</b>. The most relevant anti-permeability factors are angiopoietin-1 (Ang1), sphingosine-1-phosphate (S1P), and fibroblast growth factor (FGF). Ang1 activation of its cognate receptor, Tie2, elicits a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and promotes Src sequestration; thus hindering VEGF-A <span class="highlight" style="background-color:">signaling</span> and VE-cadherin internalization. S1P <span class="highlight" style="background-color:">signaling</span> maintains vascular integrity by modulating VE-cadherin internalization, cytoskeletal rearrangement, barrier enhancement and integrity, through its cognate receptor S1P1R. FGF maintains the integrity of the VE-cadherin/p120-catenin complex, thus stabilizing VE-cadherin at the membrane.</div></p></div><div class="aux"><div class="resc">Sandy Azzi, et al. Front Oncol. 2013;3:211.</div><p class="links"><a class="dblinks" href="/pubmed/23967403" ref="ordinalpos=2582&amp;ncbi_uid=5213585&amp;link_uid=23967403">Citation</a><a class="dblinks" href="/pmc/articles/PMC3744053/" ref="ordinalpos=2582&amp;ncbi_uid=5213585&amp;link_uid=PMC3744053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2583.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3679471/figure/FA6/" ref="ordinalpos=2583&amp;ncbi_uid=5003838&amp;link_uid=PMC3679471" href="/pmc/articles/PMC3679471/"><img src="/pmc/articles/instance/3679471/bin/fnins-07-00100-a0006.gif" src-large="/pmc/articles/instance/3679471/bin/fnins-07-00100-a0006.jpg" alt="Figure A6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3679471/" ref="ordinalpos=2583&amp;ncbi_uid=5003838&amp;link_uid=PMC3679471" image-link="/pmc/articles/PMC3679471/figure/FA6/" class="imagepopup">Figure A6.  From: Immune regulation of ovarian development: programming by neonatal immune challenge. </a></p><div class="supp"><p class="details"><div><b>MSP-RON <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Luba Sominsky, et al. Front Neurosci. 2013;7:100.</div><p class="links"><a class="dblinks" href="/pubmed/23781169" ref="ordinalpos=2583&amp;ncbi_uid=5003838&amp;link_uid=23781169">Citation</a><a class="dblinks" href="/pmc/articles/PMC3679471/" ref="ordinalpos=2583&amp;ncbi_uid=5003838&amp;link_uid=PMC3679471">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2584.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3049820/figure/F5/" ref="ordinalpos=2584&amp;ncbi_uid=3573518&amp;link_uid=PMC3049820" href="/pmc/articles/PMC3049820/"><img src="/pmc/articles/instance/3049820/bin/nihms265966f5.gif" src-large="/pmc/articles/instance/3049820/bin/nihms265966f5.jpg" alt="FIGURE 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3049820/" ref="ordinalpos=2584&amp;ncbi_uid=3573518&amp;link_uid=PMC3049820" image-link="/pmc/articles/PMC3049820/figure/F5/" class="imagepopup">FIGURE 5. Mechanistic model for P2X7 and P2Y6 <span class="highlight" style="background-color:">signaling</span> under FSS in osteoblastic cells.  From: PURINERGIC <span class="highlight" style="background-color:">SIGNALING</span> IS REQUIRED FOR FLUID SHEAR STRESS-INDUCED NF-?B TRANSLOCATION IN OSTEOBLASTS. </a></p><div class="supp"><p class="details"><div>FSS promotes the vesicular release of ATP, where it binds to metabotropic P2Y or ionotropic P2X receptors; activation of P2X<sub>7</sub> causes ERK1/2 phosphorylation, although this is not involved in IκBα degradation, and nuclear localization of NF-κB. Activation of P2X<sub>7</sub> promotes LPA formation through PLA<sub>2</sub> or PLD, but subsequent activation of LPA<sub>1</sub> or LPA<sub>3</sub> does not promote IκBα degradation. Activation of P2Y<sub>6</sub> promotes the release of intracellular calcium from the endoplasmic reticulum. Purinoceptor activation alone is not sufficient for NF-κB translocation, suggesting that another FSS-activated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (indicated by <b>?</b>) is involved in this process.</div></p></div><div class="aux"><div class="resc">Damian C. Genetos, et al. Exp Cell Res. ;317(6):737-744.</div><p class="links"><a class="dblinks" href="/pubmed/21237152" ref="ordinalpos=2584&amp;ncbi_uid=3573518&amp;link_uid=21237152">Citation</a><a class="dblinks" href="/pmc/articles/PMC3049820/" ref="ordinalpos=2584&amp;ncbi_uid=3573518&amp;link_uid=PMC3049820">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2585.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3183084/figure/pgen-1002299-g002/" ref="ordinalpos=2585&amp;ncbi_uid=3271414&amp;link_uid=PMC3183084" href="/pmc/articles/PMC3183084/"><img src="/pmc/articles/instance/3183084/bin/pgen.1002299.g002.gif" src-large="/pmc/articles/instance/3183084/bin/pgen.1002299.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3183084/" ref="ordinalpos=2585&amp;ncbi_uid=3271414&amp;link_uid=PMC3183084" image-link="/pmc/articles/PMC3183084/figure/pgen-1002299-g002/" class="imagepopup">Figure 2. Gene expression signature connects zfp-1, rde-4 and the insulin-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: A Conserved PHD Finger Protein and Endogenous RNAi Modulate Insulin <span class="highlight" style="background-color:">Signaling</span> in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(A) <i>pdk-1</i> mRNA levels and mRNA levels of downstream targets repressed by insulin <span class="highlight" style="background-color:">signaling</span> as measured by real time RT-qPCR in the indicated mutants (L4 stage animals) and normalized to wild type. Results of three biological replicas are shown; error bars represent Standard deviation. (B) <i>zfp-1</i> and <i>sod-3</i> mRNA levels measured by real time RT-qPCR in <i>daf-2; daf-16</i> double mutant L4 worms and normalized to <i>daf-2</i> mutant background, results of two biological replicas are shown. (C) Insulin-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i> modified according to results shown in (A, B) and Figure 1.</div></p></div><div class="aux"><div class="resc">Andres R. Mansisidor, et al. PLoS Genet. 2011 September;7(9):e1002299.</div><p class="links"><a class="dblinks" href="/pubmed/21980302" ref="ordinalpos=2585&amp;ncbi_uid=3271414&amp;link_uid=21980302">Citation</a><a class="dblinks" href="/pmc/articles/PMC3183084/" ref="ordinalpos=2585&amp;ncbi_uid=3271414&amp;link_uid=PMC3183084">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2586.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4345741/figure/F6/" ref="ordinalpos=2586&amp;ncbi_uid=7237023&amp;link_uid=PMC4345741" href="/pmc/articles/PMC4345741/"><img src="/pmc/articles/instance/4345741/bin/IJMR-140-112-g006.gif" src-large="/pmc/articles/instance/4345741/bin/IJMR-140-112-g006.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4345741/" ref="ordinalpos=2586&amp;ncbi_uid=7237023&amp;link_uid=PMC4345741" image-link="/pmc/articles/PMC4345741/figure/F6/" class="imagepopup">Fig. 6.  From: Regulation of luteinizing hormone receptor expression by an RNA binding protein: Role of ERK <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic model depicting the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in LH/hCG-induced LHR mRNA down regulation. Binding of ligand to LH receptor induces activation of ERK1/2 through the cAMP/PKA <span class="highlight" style="background-color:">pathway</span>. This leads to an increase in the expression of LRBP and thereby its LHR mRNA binding activity, which ultimately results in LHR mRNA degradation. (<i>Source</i>: Ref. 27, Reproduced with permission).</div></p></div><div class="aux"><div class="resc">K.M.J. Menon, et al. Indian J Med Res. 2014 November;140(Suppl 1):S112-S119.</div><p class="links"><a class="dblinks" href="/pubmed/25673531" ref="ordinalpos=2586&amp;ncbi_uid=7237023&amp;link_uid=25673531">Citation</a><a class="dblinks" href="/pmc/articles/PMC4345741/" ref="ordinalpos=2586&amp;ncbi_uid=7237023&amp;link_uid=PMC4345741">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2587.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2972639/figure/fig02/" ref="ordinalpos=2587&amp;ncbi_uid=2570858&amp;link_uid=PMC2972639" href="/pmc/articles/PMC2972639/"><img src="/pmc/articles/instance/2972639/bin/dvdy0239-2522-f2.gif" src-large="/pmc/articles/instance/2972639/bin/dvdy0239-2522-f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2972639/" ref="ordinalpos=2587&amp;ncbi_uid=2570858&amp;link_uid=PMC2972639" image-link="/pmc/articles/PMC2972639/figure/fig02/" class="imagepopup">Fig. 2.  From: Organogenesis and tumorigenesis: Insight from the JAK/STAT <span class="highlight" style="background-color:">pathway</span> in the Drosophila eye. </a></p><div class="supp"><p class="details"><div>Functions of the JAK/STAT <span class="highlight" style="background-color:">pathway</span> in <i>Drosophila</i> eye development. The JAK/STAT <span class="highlight" style="background-color:">pathway</span> plays various roles during eye development. This schematic diagram shows the eye disc at different larval stages and the corresponding functions of the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. Details are described in the text. The eye-antennal discs are drawn with anterior to the left and dorsal to the top. Light green, <i>Iro-C</i> expression; Pink, <i>upd</i> expression; yellow lines, dorsal–ventral midline; light blue, Notch/Eyg <span class="highlight" style="background-color:">signaling</span>; dark green, <i>wg</i> expression; gray lines, morphogenetic furrow (MF); gray arrow, MF progression; black and red arrow, ommatidial polarity.</div></p></div><div class="aux"><div class="resc">Ying-Hsuan Wang, et al. Dev Dyn. 2010 October;239(10):2522-2533.</div><p class="links"><a class="dblinks" href="/pubmed/20737505" ref="ordinalpos=2587&amp;ncbi_uid=2570858&amp;link_uid=20737505">Citation</a><a class="dblinks" href="/pmc/articles/PMC2972639/" ref="ordinalpos=2587&amp;ncbi_uid=2570858&amp;link_uid=PMC2972639">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2588.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3599410/figure/F2/" ref="ordinalpos=2588&amp;ncbi_uid=4716680&amp;link_uid=PMC3599410" href="/pmc/articles/PMC3599410/"><img src="/pmc/articles/instance/3599410/bin/1752-0509-7-7-2.gif" src-large="/pmc/articles/instance/3599410/bin/1752-0509-7-7-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3599410/" ref="ordinalpos=2588&amp;ncbi_uid=4716680&amp;link_uid=PMC3599410" image-link="/pmc/articles/PMC3599410/figure/F2/" class="imagepopup">Figure 2.  From: SignaLink 2 - a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resource with multi-layered regulatory networks. </a></p><div class="supp"><p class="details"><div><b>The construction and structure of SignaLink 2.</b> First, with manual curation we updated previous <span class="highlight" style="background-color:">pathway</span> data, extended the set of <span class="highlight" style="background-color:">pathway</span> proteins and included information on scaffold and endocytotic proteins. Then, we integrated the protein-protein interactions affecting proteins in the manually curated core of SignaLink 2. With this step we also imported first neighbor proteins of the <span class="highlight" style="background-color:">pathway</span> member and <span class="highlight" style="background-color:">pathway</span> regulator proteins. Then, we integrated the transcription factors (TF) that regulate all these proteins. Finally, we integrated miRNAs and their interactions with the mRNAs of all the already inserted proteins. We also added the TFs of the imported miRNAs.</div></p></div><div class="aux"><div class="resc">Dávid Fazekas, et al. BMC Syst Biol. 2013;7:7-7.</div><p class="links"><a class="dblinks" href="/pubmed/23331499" ref="ordinalpos=2588&amp;ncbi_uid=4716680&amp;link_uid=23331499">Citation</a><a class="dblinks" href="/pmc/articles/PMC3599410/" ref="ordinalpos=2588&amp;ncbi_uid=4716680&amp;link_uid=PMC3599410">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2589.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3210339/figure/fig3/" ref="ordinalpos=2589&amp;ncbi_uid=4330718&amp;link_uid=PMC3210339" href="/pmc/articles/PMC3210339/"><img src="/pmc/articles/instance/3210339/bin/jcbfm2011116f3.gif" src-large="/pmc/articles/instance/3210339/bin/jcbfm2011116f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3210339/" ref="ordinalpos=2589&amp;ncbi_uid=4330718&amp;link_uid=PMC3210339" image-link="/pmc/articles/PMC3210339/figure/fig3/" class="imagepopup">Figure 3.  From: Multipotent mesenchymal stromal cells increase tPA expression and concomitantly decrease PAI-1 expression in astrocytes through the sonic hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after stroke (in vitro study). </a></p><div class="supp"><p class="details"><div>Shh affected PAI-1 expression in astrocytes by regulating the TGF-<i>β</i>1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ELISA detection shows that MSCs and rm-Shh significantly decreased TGF-<i>β</i>1 expression in cultured astrocytes. (<b>A</b>) The Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitor cyclopamine increased TGF-<i>β</i>1 level in astrocytes cocultured with MSCs. (<b>B</b>) Exogenous addition of rh-TGF-<i>β</i>1 significantly increased PAI-1 expression in astrocytes, and the TGF-<i>β</i> activity neutralizer TGF-<i>β</i>1 LAP specifically downregulated the PAI-1 expression. <sup>*</sup><i>P</i>&lt;0.05, <sup>**</sup><i>P</i>&lt;0.01, compared with normal cultured astrocytes; <sup>#</sup><i>P</i>&lt;0.05, compared with MSC treatment; ^<i>P</i>&lt;0.05 compared with TGF-<i>β</i>1 treatment. C: control (normal cultured astrocytes); L: LAP treatment; T: TGF-<i>β</i>1 treatment; T+L: TGF-<i>β</i>1 and LAP treatment; Shh: rm-Shh treatment; MSCs: MSC treatment. MSC, mesenchymal stromal cell; PAI-1, plasminogen activator inhibitor 1; rm-Shh, recombinant mouse Shh; Shh, sonic hedgehog; TGF-<i>β</i>1, transforming growth factor-<i>β</i>1; ELISA, enzyme-linked immunosorbent assay.</div></p></div><div class="aux"><div class="resc">Hongqi Xin, et al. J Cereb Blood Flow Metab. 2011 November;31(11):2181-2188.</div><p class="links"><a class="dblinks" href="/pubmed/21829213" ref="ordinalpos=2589&amp;ncbi_uid=4330718&amp;link_uid=21829213">Citation</a><a class="dblinks" href="/pmc/articles/PMC3210339/" ref="ordinalpos=2589&amp;ncbi_uid=4330718&amp;link_uid=PMC3210339">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2590.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g006/" ref="ordinalpos=2590&amp;ncbi_uid=4834031&amp;link_uid=PMC3633842" href="/pmc/articles/PMC3633842/"><img src="/pmc/articles/instance/3633842/bin/pone.0062845.g006.gif" src-large="/pmc/articles/instance/3633842/bin/pone.0062845.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633842/" ref="ordinalpos=2590&amp;ncbi_uid=4834031&amp;link_uid=PMC3633842" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g006/" class="imagepopup">Figure 6. EIF2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Decreased Pattern Recognition Receptor <span class="highlight" style="background-color:">Signaling</span>, Interferon-Signature, and Bactericidal/Permeability-Increasing Protein Gene Expression in Cord Blood of Term Low Birth Weight Human Newborns. </a></p><div class="supp"><p class="details"><div>The down-regulated genes in LBW newborns are represented in green color. The intensity of node colour indicates the degree of down-regulation. The uncolored nodes are the genes inferred by IPA from its knowledge base.</div></p></div><div class="aux"><div class="resc">Vikas Vikram Singh, et al. PLoS One. 2013;8(4):e62845.</div><p class="links"><a class="dblinks" href="/pubmed/23626859" ref="ordinalpos=2590&amp;ncbi_uid=4834031&amp;link_uid=23626859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633842/" ref="ordinalpos=2590&amp;ncbi_uid=4834031&amp;link_uid=PMC3633842">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2591.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3175799/figure/F8/" ref="ordinalpos=2591&amp;ncbi_uid=3227842&amp;link_uid=PMC3175799" href="/pmc/articles/PMC3175799/"><img src="/pmc/articles/instance/3175799/bin/nihms309273f8.gif" src-large="/pmc/articles/instance/3175799/bin/nihms309273f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3175799/" ref="ordinalpos=2591&amp;ncbi_uid=3227842&amp;link_uid=PMC3175799" image-link="/pmc/articles/PMC3175799/figure/F8/" class="imagepopup">Figure 8. A schematic model of ZnR <span class="highlight" style="background-color:">signaling</span> in the CA3 region of the hippocampus.  From: Synaptically-Released Zinc Triggers Metabotropic <span class="highlight" style="background-color:">Signaling</span> via a Zinc Sensing Receptor in the Hippocampus. </a></p><div class="supp"><p class="details"><div>Synaptic Zn<sup>2+</sup>, released from the mossy fibers, activates a metabotropic ZnR. The ZnR activity induces metabotropic release of Ca<sup>2+</sup> mediated by a Gαq protein, GPR39, and the IP3 <span class="highlight" style="background-color:">pathway</span>, leading to phosphorylation of extracellular-regulated kinase (ERK1/2) and Ca<sup>2+</sup>/calmodulin kinase (CAMKII).</div></p></div><div class="aux"><div class="resc">Limor Besser, et al. J Neurosci. ;29(9):2890-2901.</div><p class="links"><a class="dblinks" href="/pubmed/19261885" ref="ordinalpos=2591&amp;ncbi_uid=3227842&amp;link_uid=19261885">Citation</a><a class="dblinks" href="/pmc/articles/PMC3175799/" ref="ordinalpos=2591&amp;ncbi_uid=3227842&amp;link_uid=PMC3175799">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2592.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2593196/figure/f1/" ref="ordinalpos=2592&amp;ncbi_uid=1679225&amp;link_uid=PMC2593196" href="/pmc/articles/PMC2593196/"><img src="/pmc/articles/instance/2593196/bin/zek0120832510001.gif" src-large="/pmc/articles/instance/2593196/bin/zek0120832510001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2593196/" ref="ordinalpos=2592&amp;ncbi_uid=1679225&amp;link_uid=PMC2593196" image-link="/pmc/articles/PMC2593196/figure/f1/" class="imagepopup">FIG. 1. From: Master and Commander in Fungal Pathogens: the Two-Component System and the HOG <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> . </a></p><div class="supp"><p class="details"><div>General regulatory mechanism of the HOG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in fungi. Abbreviations: HK, histidine KD; RR, response regulator receiver domain; S/T kinase, Ser/Thr protein KD; SBD, Ssk2/Ssk22 binding domain; HDS, Hog1 docking site; S/T-Y KD, Ser/Thr and Tyr dual-protein KD.</div></p></div><div class="aux"><div class="resc">Yong-Sun Bahn. Eukaryot Cell. 2008 December;7(12):2017-2036.</div><p class="links"><a class="dblinks" href="/pubmed/18952900" ref="ordinalpos=2592&amp;ncbi_uid=1679225&amp;link_uid=18952900">Citation</a><a class="dblinks" href="/pmc/articles/PMC2593196/" ref="ordinalpos=2592&amp;ncbi_uid=1679225&amp;link_uid=PMC2593196">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2593.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3496019/figure/F2/" ref="ordinalpos=2593&amp;ncbi_uid=5539012&amp;link_uid=PMC3496019" href="/pmc/articles/PMC3496019/"><img src="/pmc/articles/instance/3496019/bin/nihms-404973-f0006.gif" src-large="/pmc/articles/instance/3496019/bin/nihms-404973-f0006.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3496019/" ref="ordinalpos=2593&amp;ncbi_uid=5539012&amp;link_uid=PMC3496019" image-link="/pmc/articles/PMC3496019/figure/F2/" class="imagepopup">Figure 2.  From: Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div>Regulation of CSCs by inflammatory cytokine networks. Binding of the IL-6-IL-6R complex to gp130, and IL-8 to CXCR1/2 activates the NF-κB <span class="highlight" style="background-color:">pathway</span> via Stat3 and Akt <span class="highlight" style="background-color:">signaling</span>. IL-6 and IL-8 production of NF-κB generates a positive feedback loop that maintains constitutive <span class="highlight" style="background-color:">pathway</span> activation. Inhibition of <span class="highlight" style="background-color:">pathway</span> components by pharmacologic agents is illustrated.</div></p></div><div class="aux"><div class="resc">Zuoren Yu, et al. Int J Biochem Cell Biol. 2012 December;44(12):2144-2151.</div><p class="links"><a class="dblinks" href="/pubmed/22981632" ref="ordinalpos=2593&amp;ncbi_uid=5539012&amp;link_uid=22981632">Citation</a><a class="dblinks" href="/pmc/articles/PMC3496019/" ref="ordinalpos=2593&amp;ncbi_uid=5539012&amp;link_uid=PMC3496019">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2594.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4007780/figure/F1/" ref="ordinalpos=2594&amp;ncbi_uid=6150769&amp;link_uid=PMC4007780" href="/pmc/articles/PMC4007780/"><img src="/pmc/articles/instance/4007780/bin/1752-0509-8-30-1.gif" src-large="/pmc/articles/instance/4007780/bin/1752-0509-8-30-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4007780/" ref="ordinalpos=2594&amp;ncbi_uid=6150769&amp;link_uid=PMC4007780" image-link="/pmc/articles/PMC4007780/figure/F1/" class="imagepopup">Figure 1.  From: An integrative modeling framework reveals plasticity of TGF-? <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>CADBIOM model for the canonical TGF-β <span class="highlight" style="background-color:">pathway</span>. (A)</b> Schematic representation of the canonical <span class="highlight" style="background-color:">pathway</span>. Binding of TGF-β to the receptor induces recruitment and phosphorylation of Smad2/3. Activated Smad2/3 heterodimerizes with Smad4 and the complex migrates to nucleus where it regulates gene transcription. <b>(B)</b> Two examples of translations from the PID database (left) into the CADBIOM representation (right). Top: interaction of Smad2/3 with Smad4. The CADBIOM representation uses two transitions from a component toward the complex, conditioned by the presence of the other component and leading to the synchronization of the transition. The same event h1 is associated with the two transitions that occur simultaneously. Bottom: migration of the Smad complex to the nucleus. The CADBIOM representation uses a transition between the two places “SMAD3_SMAD4_active_cytoplasm” and “SMAD3_SMAD4_active_nucleus” conditioned by the presence of importins within the guard. The event h1 indicates the timing of the transition. <b>(C)</b> CADBIOM illustration of the canonical <span class="highlight" style="background-color:">pathway</span> after integration of the three PID pathways: “TGF-β receptor <span class="highlight" style="background-color:">signaling</span>” (PID Ref: tgfbrpathway), “regulation of cytoplasmic and nuclear Smad2/3 <span class="highlight" style="background-color:">signaling</span>” (PID Ref: Smad2-3pathway) and “regulation of nuclear Smad2/3 <span class="highlight" style="background-color:">signaling</span>” (PID Ref: Smad2-3nuclearpathway).</div></p></div><div class="aux"><div class="resc">Geoffroy Andrieux, et al. BMC Syst Biol. 2014;8:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/24618419" ref="ordinalpos=2594&amp;ncbi_uid=6150769&amp;link_uid=24618419">Citation</a><a class="dblinks" href="/pmc/articles/PMC4007780/" ref="ordinalpos=2594&amp;ncbi_uid=6150769&amp;link_uid=PMC4007780">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2595.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1913781/figure/fig7/" ref="ordinalpos=2595&amp;ncbi_uid=958473&amp;link_uid=PMC1913781" href="/pmc/articles/PMC1913781/"><img src="/pmc/articles/instance/1913781/bin/pp1440324f07.gif" src-large="/pmc/articles/instance/1913781/bin/pp1440324f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1913781/" ref="ordinalpos=2595&amp;ncbi_uid=958473&amp;link_uid=PMC1913781" image-link="/pmc/articles/PMC1913781/figure/fig7/" class="imagepopup">Figure 7. From: Medicago truncatula NIN Is Essential for Rhizobial-Independent Nodule Organogenesis Induced by Autoactive Calcium/Calmodulin-Dependent Protein Kinase. </a></p><div class="supp"><p class="details"><div>Model relating <i>NIN</i> to NF perception and downstream events. The autoactivation of nodule organogenesis by CCaMK (<i>DMI3</i>) is dependent on <i>NIN</i>. Calcium spiking and the induction of <i>ENOD11</i> expression are normal in <i>Mtnin</i>-<i>1</i>, demonstrating that <i>NIN</i> functions downstream of the early NF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (indicated by bracket). All of the components of the early NF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are also required for the induction of <i>NIN</i> expression. In addition, the expression of <i>ENOD11</i> outside of the root responsive zone is suppressed by <i>NIN</i>. Recently published data demonstrates that nodule organogenesis requires cytokinin <span class="highlight" style="background-color:">signaling</span> in the root cortex (shown in gray), which is dependent on both <i>NIN</i> and <i>NSP2</i>. The <i>NIN</i> gene is also required for bacterial infection (as is <i>HCL</i>), suggesting <i>NIN</i> is involved in both epidermal and cortical cell responses. The precise relationship between <i>HCL</i> and <i>NIN</i> cannot be resolved in the work presented here. However, given that <i>HCL</i> expression does not depend on the early NF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and autoactive CCaMK induces nodules to form in the <i>hcl</i> mutant, we conclude that <i>HCL</i> is not downstream of early NF <span class="highlight" style="background-color:">signaling</span> leading to nodule organogenesis. Arrows do not imply direct interactions.</div></p></div><div class="aux"><div class="resc">John F. Marsh, et al. Plant Physiol. 2007 May;144(1):324-335.</div><p class="links"><a class="dblinks" href="/pubmed/17369436" ref="ordinalpos=2595&amp;ncbi_uid=958473&amp;link_uid=17369436">Citation</a><a class="dblinks" href="/pmc/articles/PMC1913781/" ref="ordinalpos=2595&amp;ncbi_uid=958473&amp;link_uid=PMC1913781">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2596.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4007577/figure/F3/" ref="ordinalpos=2596&amp;ncbi_uid=6097499&amp;link_uid=PMC4007577" href="/pmc/articles/PMC4007577/"><img src="/pmc/articles/instance/4007577/bin/nihms574351f3.gif" src-large="/pmc/articles/instance/4007577/bin/nihms574351f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4007577/" ref="ordinalpos=2596&amp;ncbi_uid=6097499&amp;link_uid=PMC4007577" image-link="/pmc/articles/PMC4007577/figure/F3/" class="imagepopup">Figure 3. Activation of BMP/Smad1/5/8 <span class="highlight" style="background-color:">signaling</span> is downstream of PI3K/AKT <span class="highlight" style="background-color:">pathway</span>.  From: EGF Stimulates M?ller Glial Proliferation via a BMP Dependent Mechanism. </a></p><div class="supp"><p class="details"><div><b>A.</b> Smad1/5/8 phosphorylation (pSmad1/5/8; red) induced by EGF in Müller glia (Hes5-GFP; green) is blocked by inhibition of the PI3K/AKT (EGF+LY) or the BMP/Smad1/5/8 (EGF+DM) pathways, whereas inhibition of the ERK1/2 <span class="highlight" style="background-color:">pathway</span> (EGF+PD) did not affect Smad1/5/8 phosphorylation. <b>B.</b> Real-time PCR showing that EGF-induced expression of <i>Bmp7</i> is significantly attenuated by PI3K/AKT inhibition. Expression was normalized to NT value for each set of experiment (n = 4). *p &lt; 0.05 with t-test. <b>C.</b> A representative Western Blot for Id1, a BMP/Smad1/5/8 <span class="highlight" style="background-color:">signaling</span> target. <b>D.</b> Quantification of Western Blots showing that EGF-induced Id1 expression was blocked significantly by PI3K/AKT inhibition. The Id1/actin ratio of non-treated (NT) explants is normalized to 1 for each set of experiment (n = 3). *p &lt; 0.05 with t-test. Error bars: SEM.</div></p></div><div class="aux"><div class="resc">Yumi Ueki, et al. Glia. 2013 May;61(5):778-789.</div><p class="links"><a class="dblinks" href="/pubmed/23362023" ref="ordinalpos=2596&amp;ncbi_uid=6097499&amp;link_uid=23362023">Citation</a><a class="dblinks" href="/pmc/articles/PMC4007577/" ref="ordinalpos=2596&amp;ncbi_uid=6097499&amp;link_uid=PMC4007577">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2597.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3076855/figure/fig01/" ref="ordinalpos=2597&amp;ncbi_uid=3278658&amp;link_uid=PMC3076855" href="/pmc/articles/PMC3076855/"><img src="/pmc/articles/instance/3076855/bin/pnas.1103102108fig01.gif" src-large="/pmc/articles/instance/3076855/bin/pnas.1103102108fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3076855/" ref="ordinalpos=2597&amp;ncbi_uid=3278658&amp;link_uid=PMC3076855" image-link="/pmc/articles/PMC3076855/figure/fig01/" class="imagepopup">Fig. 1. From: Winning WNT: Race to Wnt <span class="highlight" style="background-color:">signaling</span> inhibitors. </a></p><div class="supp"><p class="details"><div>The Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its small molecule inhibitors. Binding of a Wnt ligand to membrane receptor/coreceptor triggers an intracellular <span class="highlight" style="background-color:">signaling</span> cascade leading to the stabilization and nuclear translocation of β-catenin. β-catenin in the nucleus binds to T cell factor (TCF)/lymphoid enhancer factor (LEF) and a number of cofactors to regulate gene expression. Red and green arrows indicate events that negatively and positively, respectively, affect <span class="highlight" style="background-color:">signaling</span> output. Examples of recently discovered small molecule inhibitors are shown, with their sites of action detailed in the table below the diagram.</div></p></div><div class="aux"><div class="resc">Kazuhide Watanabe, et al. Proc Natl Acad Sci U S A. 2011 April 12;108(15):5929-5930.</div><p class="links"><a class="dblinks" href="/pubmed/21451134" ref="ordinalpos=2597&amp;ncbi_uid=3278658&amp;link_uid=21451134">Citation</a><a class="dblinks" href="/pmc/articles/PMC3076855/" ref="ordinalpos=2597&amp;ncbi_uid=3278658&amp;link_uid=PMC3076855">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2598.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3411757/figure/pone-0042430-g002/" ref="ordinalpos=2598&amp;ncbi_uid=4170096&amp;link_uid=PMC3411757" href="/pmc/articles/PMC3411757/"><img src="/pmc/articles/instance/3411757/bin/pone.0042430.g002.gif" src-large="/pmc/articles/instance/3411757/bin/pone.0042430.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3411757/" ref="ordinalpos=2598&amp;ncbi_uid=4170096&amp;link_uid=PMC3411757" image-link="/pmc/articles/PMC3411757/figure/pone-0042430-g002/" class="imagepopup">Figure 2. DC-driven <span class="highlight" style="background-color:">signaling</span> response upon fungal receptor engagement..  From: The Modular Nature of Dendritic Cell Responses to Commensal and Pathogenic Fungi. </a></p><div class="supp"><p class="details"><div>Clustering of <span class="highlight" style="background-color:">pathway</span> Enrichment Factor (PEFs) obtained from the FET analysis on DC samples challenged with fungi. (A) <span class="highlight" style="background-color:">Pathway</span> signatures generated on DC-ATLAS. (B) Insert of the <span class="highlight" style="background-color:">pathway</span> signature generated using immune-related public <span class="highlight" style="background-color:">pathway</span> available from KEGG and Reactome. The detail highlights the available <span class="highlight" style="background-color:">pathway</span> involved in the initial recognition of an antigen, in particular the TLR signalings. The complete clustering is available as Figure S3. Colored spots indicate significant (p≤0.05) up- (red) or down- (green) regulation. The colors of the dendrogram indicate the percentages of the tree support (significance), from 50% (pink) to 100% (black).</div></p></div><div class="aux"><div class="resc">Lisa Rizzetto, et al. PLoS One. 2012;7(8):e42430.</div><p class="links"><a class="dblinks" href="/pubmed/22879980" ref="ordinalpos=2598&amp;ncbi_uid=4170096&amp;link_uid=22879980">Citation</a><a class="dblinks" href="/pmc/articles/PMC3411757/" ref="ordinalpos=2598&amp;ncbi_uid=4170096&amp;link_uid=PMC3411757">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2599.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4251064/figure/f4-geg-6-2014-001/" ref="ordinalpos=2599&amp;ncbi_uid=6964993&amp;link_uid=PMC4251064" href="/pmc/articles/PMC4251064/"><img src="/pmc/articles/instance/4251064/bin/geg-6-2014-001f4.gif" src-large="/pmc/articles/instance/4251064/bin/geg-6-2014-001f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4251064/" ref="ordinalpos=2599&amp;ncbi_uid=6964993&amp;link_uid=PMC4251064" image-link="/pmc/articles/PMC4251064/figure/f4-geg-6-2014-001/" class="imagepopup">Figure 4.  From: Bioinformatic Dissecting of TP53 Regulation <span class="highlight" style="background-color:">Pathway</span> Underlying Butyrate-induced Histone Modification in Epigenetic Regulation. </a></p><div class="supp"><p class="details"><div>Canonical <span class="highlight" style="background-color:">pathway</span> of TP53 <span class="highlight" style="background-color:">signaling</span>: dataset was analyzed with the IPA software (Ingenuity Systems®, www.ingeuity.com). The number under the gene’s name is the Log2 fold changes of gene expression induced by butyrate. The color indicates the expression level of the genes (red indicating upregulated and green indicating downregulated genes).</div></p></div><div class="aux"><div class="resc">Cong-Jun Li, et al. Genet Epigenet. 2014;6:1-7.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2599&amp;ncbi_uid=6964993&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4251064/" ref="ordinalpos=2599&amp;ncbi_uid=6964993&amp;link_uid=PMC4251064">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2600.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2562038/figure/F2/" ref="ordinalpos=2600&amp;ncbi_uid=1549917&amp;link_uid=PMC2562038" href="/pmc/articles/PMC2562038/"><img src="/pmc/articles/instance/2562038/bin/nihms68083f2.gif" src-large="/pmc/articles/instance/2562038/bin/nihms68083f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2562038/" ref="ordinalpos=2600&amp;ncbi_uid=1549917&amp;link_uid=PMC2562038" image-link="/pmc/articles/PMC2562038/figure/F2/" class="imagepopup">Figure 2.  From: Ischemic stroke and Neurogenesis in the Subventricular Zone. </a></p><div class="supp"><p class="details"><div>The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. When ligands including Delta-like and Jagged families from <span class="highlight" style="background-color:">signaling</span> cells bind to Notch receptor in <span class="highlight" style="background-color:">signaling</span> receiving cells, NICD is cleaved by the γ-secretase enzyme complex. The NICD translocates into the nucleus and associates with CBF1 and others to activate target genes including Hes family members.</div></p></div><div class="aux"><div class="resc">Rui Lan Zhang, et al. Neuropharmacology. ;55(3):345-352.</div><p class="links"><a class="dblinks" href="/pubmed/18632119" ref="ordinalpos=2600&amp;ncbi_uid=1549917&amp;link_uid=18632119">Citation</a><a class="dblinks" href="/pmc/articles/PMC2562038/" ref="ordinalpos=2600&amp;ncbi_uid=1549917&amp;link_uid=PMC2562038">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="25" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="26" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="27" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:35:16-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB17805431052100000000004AAEC3;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB17805431052100000000004AAEC3" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>