
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="30" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A721B5431133100000000005E0C16" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="29" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="30" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="31" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="30" /></div>
            <div><h2 class="result_count">Results: 2901 to 3000 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2901.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3697783/figure/fig8/" ref="ordinalpos=2901&amp;ncbi_uid=5072332&amp;link_uid=PMC3697783" href="/pmc/articles/PMC3697783/"><img src="/pmc/articles/instance/3697783/bin/JAMC2013-951319.008.gif" src-large="/pmc/articles/instance/3697783/bin/JAMC2013-951319.008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3697783/" ref="ordinalpos=2901&amp;ncbi_uid=5072332&amp;link_uid=PMC3697783" image-link="/pmc/articles/PMC3697783/figure/fig8/" class="imagepopup">Figure 8.  From: Antitumor Molecular Mechanism of Chlorogenic Acid on Inducting Genes GSK-3β and APC and Inhibiting Gene β-Catenin. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ruoshi Xu, et al. J Anal Methods Chem. 2013;2013:951319.</div><p class="links"><a class="dblinks" href="/pubmed/23844319" ref="ordinalpos=2901&amp;ncbi_uid=5072332&amp;link_uid=23844319">Citation</a><a class="dblinks" href="/pmc/articles/PMC3697783/" ref="ordinalpos=2901&amp;ncbi_uid=5072332&amp;link_uid=PMC3697783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2902.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2581573/figure/fig1/" ref="ordinalpos=2902&amp;ncbi_uid=1572325&amp;link_uid=PMC2581573" href="/pmc/articles/PMC2581573/"><img src="/pmc/articles/instance/2581573/bin/zbc0490856650001.gif" src-large="/pmc/articles/instance/2581573/bin/zbc0490856650001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2581573/" ref="ordinalpos=2902&amp;ncbi_uid=1572325&amp;link_uid=PMC2581573" image-link="/pmc/articles/PMC2581573/figure/fig1/" class="imagepopup">FIGURE 1. From: The Kinase Activity of IL-1 Receptor-associated Kinase 4 Is Required for  Interleukin-1 Receptor/Toll-like Receptor-induced TAK1-dependent NF?B  Activation. </a></p><div class="supp"><p class="details"><div>Model of the IL-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jerzy Frączek, et al. J Biol Chem. 2008 November 14;283(46):31697-31705.</div><p class="links"><a class="dblinks" href="/pubmed/18794297" ref="ordinalpos=2902&amp;ncbi_uid=1572325&amp;link_uid=18794297">Citation</a><a class="dblinks" href="/pmc/articles/PMC2581573/" ref="ordinalpos=2902&amp;ncbi_uid=1572325&amp;link_uid=PMC2581573">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2903.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1599906/figure/F3/" ref="ordinalpos=2903&amp;ncbi_uid=864578&amp;link_uid=PMC1599906" href="/pmc/articles/PMC1599906/"><img src="/pmc/articles/instance/1599906/bin/zpq0360634000003.gif" src-large="/pmc/articles/instance/1599906/bin/zpq0360634000003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1599906/" ref="ordinalpos=2903&amp;ncbi_uid=864578&amp;link_uid=PMC1599906" image-link="/pmc/articles/PMC1599906/figure/F3/" class="imagepopup">Fig. 3. From: The transcriptome of human oocytes. </a></p><div class="supp"><p class="details"><div>TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes shown in red are differentially up-regulated in human oocytes.</div></p></div><div class="aux"><div class="resc">Arif Murat Kocabas, et al. Proc Natl Acad Sci U S A. 2006 September 19;103(38):14027-14032.</div><p class="links"><a class="dblinks" href="/pubmed/16968779" ref="ordinalpos=2903&amp;ncbi_uid=864578&amp;link_uid=16968779">Citation</a><a class="dblinks" href="/pmc/articles/PMC1599906/" ref="ordinalpos=2903&amp;ncbi_uid=864578&amp;link_uid=PMC1599906">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2904.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC186427/figure/F7/" ref="ordinalpos=2904&amp;ncbi_uid=214342&amp;link_uid=PMC186427" href="/pmc/articles/PMC186427/"><img src="/pmc/articles/instance/186427/bin/51627-13f7_C4TT.gif" src-large="/pmc/articles/instance/186427/bin/51627-13f7_C4TT.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC186427/" ref="ordinalpos=2904&amp;ncbi_uid=214342&amp;link_uid=PMC186427" image-link="/pmc/articles/PMC186427/figure/F7/" class="imagepopup">Figure 7.  From: Smad3 allostery links TGF-? receptor kinase activation to transcriptional control. </a></p><div class="supp"><p class="details"><div>Trimerization-dependent Smad <span class="highlight" style="background-color:">signaling</span> in the TGF-β <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Bin Y. Qin, et al. Genes Dev. 2002 August 1;16(15):1950-1963.</div><p class="links"><a class="dblinks" href="/pubmed/12154125" ref="ordinalpos=2904&amp;ncbi_uid=214342&amp;link_uid=12154125">Citation</a><a class="dblinks" href="/pmc/articles/PMC186427/" ref="ordinalpos=2904&amp;ncbi_uid=214342&amp;link_uid=PMC186427">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2905.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3759196/figure/f9-cancers-03-03242/" ref="ordinalpos=2905&amp;ncbi_uid=5270938&amp;link_uid=PMC3759196" href="/pmc/articles/PMC3759196/"><img src="/pmc/articles/instance/3759196/bin/cancers-03-03242f9.gif" src-large="/pmc/articles/instance/3759196/bin/cancers-03-03242f9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759196/" ref="ordinalpos=2905&amp;ncbi_uid=5270938&amp;link_uid=PMC3759196" image-link="/pmc/articles/PMC3759196/figure/f9-cancers-03-03242/" class="imagepopup">Figure 9. From: Aberrant <span class="highlight" style="background-color:">Signaling</span> Pathways in Glioma. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> in glioma stem cell.<br />BMP: bone morphogenetic proteins; BMPR: BMP receptor; Ptch: patched gene; SHH: Sonic hedgehog homolog; STAT: signal transducers and activators of transcription; Smo: smoothened gene; TNFAIP: tumor necrosis factor alpha-induced protein; TRRAP: transformation/transcription domain-associated protein.</div></p></div><div class="aux"><div class="resc">Mitsutoshi Nakada, et al. Cancers (Basel). 2011 September;3(3):3242-3278.</div><p class="links"><a class="dblinks" href="/pubmed/24212955" ref="ordinalpos=2905&amp;ncbi_uid=5270938&amp;link_uid=24212955">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759196/" ref="ordinalpos=2905&amp;ncbi_uid=5270938&amp;link_uid=PMC3759196">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2906.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2859534/figure/F7/" ref="ordinalpos=2906&amp;ncbi_uid=2909818&amp;link_uid=PMC2859534" href="/pmc/articles/PMC2859534/"><img src="/pmc/articles/instance/2859534/bin/zbc0201014560007.gif" src-large="/pmc/articles/instance/2859534/bin/zbc0201014560007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2859534/" ref="ordinalpos=2906&amp;ncbi_uid=2909818&amp;link_uid=PMC2859534" image-link="/pmc/articles/PMC2859534/figure/F7/" class="imagepopup">FIGURE 7. From: Activation of cAMP-dependent <span class="highlight" style="background-color:">Signaling</span> Induces Oxidative Modification of the Cardiac Na+-K+ Pump and Inhibits Its Activity. </a></p><div class="supp"><p class="details"><div><b>Summary of molecular, fluorescence, and patch clamp studies on effects of forskolin.</b> Steps in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that are implicated are shown at the top with stimulation and inhibition indicated by + and −. Blocks of responses to forskolin are indicated below by ×. Glutathionylation of the β<sub>1</sub> subunit is indicated by GSS-β<sub>1</sub>, and Na<sup>+</sup>-K<sup>+</sup> pump current is indicated by I<sub>p</sub>. <i>co-IP</i>, co-immunoprecipitation.</div></p></div><div class="aux"><div class="resc">Caroline N. White, et al. J Biol Chem. 2010 April 30;285(18):13712-13720.</div><p class="links"><a class="dblinks" href="/pubmed/20194511" ref="ordinalpos=2906&amp;ncbi_uid=2909818&amp;link_uid=20194511">Citation</a><a class="dblinks" href="/pmc/articles/PMC2859534/" ref="ordinalpos=2906&amp;ncbi_uid=2909818&amp;link_uid=PMC2859534">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2907.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2072925/figure/Fig1/" ref="ordinalpos=2907&amp;ncbi_uid=1208955&amp;link_uid=PMC2072925" href="/pmc/articles/PMC2072925/"><img src="/pmc/articles/instance/2072925/bin/11302_2007_9074_Fig1_HTML.gif" src-large="/pmc/articles/instance/2072925/bin/11302_2007_9074_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2072925/" ref="ordinalpos=2907&amp;ncbi_uid=1208955&amp;link_uid=PMC2072925" image-link="/pmc/articles/PMC2072925/figure/Fig1/" class="imagepopup">Fig. 1.  From: New insights into purinergic receptor <span class="highlight" style="background-color:">signaling</span> in neuronal differentiation, neuroprotection, and brain disorders. </a></p><div class="supp"><p class="details"><div>Purine-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves the activation of P1 adenosine and P2 purinergic receptors and purine hydrolysis by ectonucleotidases. The scheme demonstrates purinergic receptor activity present in glia-glia, neuron-glia, and neuron-neuron interaction during neurogenesis as well as in the metabolism of the adult brain</div></p></div><div class="aux"><div class="resc">Paromita Majumder, et al. Purinergic Signal. 2007 September;3(4):317-331.</div><p class="links"><a class="dblinks" href="/pubmed/18404445" ref="ordinalpos=2907&amp;ncbi_uid=1208955&amp;link_uid=18404445">Citation</a><a class="dblinks" href="/pmc/articles/PMC2072925/" ref="ordinalpos=2907&amp;ncbi_uid=1208955&amp;link_uid=PMC2072925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2908.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2659592/figure/F7/" ref="ordinalpos=2908&amp;ncbi_uid=1723890&amp;link_uid=PMC2659592" href="/pmc/articles/PMC2659592/"><img src="/pmc/articles/instance/2659592/bin/nihms-96516-f0007.gif" src-large="/pmc/articles/instance/2659592/bin/nihms-96516-f0007.jpg" alt="FIGURE 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2659592/" ref="ordinalpos=2908&amp;ncbi_uid=1723890&amp;link_uid=PMC2659592" image-link="/pmc/articles/PMC2659592/figure/F7/" class="imagepopup">FIGURE 7. Mechanisms of LPA <span class="highlight" style="background-color:">signaling</span> in IL-13Rα2 expression and IL-13-induced STAT6 phosphorylation.  From: Lysophosphatidic Acid Induces Interleukin-13 (IL-13) Receptor α2 Expression and Inhibits IL-13 <span class="highlight" style="background-color:">Signaling</span> in Primary Human Bronchial Epithelial Cells. </a></p><div class="supp"><p class="details"><div>Ligation of LPA to G protein-coupled LPA receptors results in IL-13R<i>α</i>2 expression through activation of PLD and JNK/AP-1. Elevated IL-13R<i>α</i>2 expression attenuated the IL-13- but not the IL-4-mediated STAT6 <span class="highlight" style="background-color:">pathway</span>. <i>PA</i>, phosphatidic acid; <i>PC</i>, phosphatidylcholine.</div></p></div><div class="aux"><div class="resc">Yutong Zhao, et al. J Biol Chem. ;282(14):10172-10179.</div><p class="links"><a class="dblinks" href="/pubmed/17287216" ref="ordinalpos=2908&amp;ncbi_uid=1723890&amp;link_uid=17287216">Citation</a><a class="dblinks" href="/pmc/articles/PMC2659592/" ref="ordinalpos=2908&amp;ncbi_uid=1723890&amp;link_uid=PMC2659592">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2909.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1682017/figure/f5/" ref="ordinalpos=2909&amp;ncbi_uid=811958&amp;link_uid=PMC1682017" href="/pmc/articles/PMC1682017/"><img src="/pmc/articles/instance/1682017/bin/msb4100094-f5.gif" src-large="/pmc/articles/instance/1682017/bin/msb4100094-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1682017/" ref="ordinalpos=2909&amp;ncbi_uid=811958&amp;link_uid=PMC1682017" image-link="/pmc/articles/PMC1682017/figure/f5/" class="imagepopup">Figure 5.  From: Effects of HER2 overexpression on cell <span class="highlight" style="background-color:">signaling</span> networks governing proliferation and migration. </a></p><div class="supp"><p class="details"><div>Effect of HRG stimulation versus EGF stimulation in high HER2-expressing cells. Visualization of the fold change in phosphorylation level between HRG-stimulated and EGF-stimulated 24H cells provides a network view of the mechanistic effects underlying differential response to distinct growth factor stimuli on (<b>A</b>) the canonical EGFR <span class="highlight" style="background-color:">signaling</span> cascades and (<b>B</b>) the cell migration-associated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Alejandro Wolf-Yadlin, et al. Mol Syst Biol. 2006;2:54-54.</div><p class="links"><a class="dblinks" href="/pubmed/17016520" ref="ordinalpos=2909&amp;ncbi_uid=811958&amp;link_uid=17016520">Citation</a><a class="dblinks" href="/pmc/articles/PMC1682017/" ref="ordinalpos=2909&amp;ncbi_uid=811958&amp;link_uid=PMC1682017">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2910.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2043575/figure/F8/" ref="ordinalpos=2910&amp;ncbi_uid=1153112&amp;link_uid=PMC2043575" href="/pmc/articles/PMC2043575/"><img src="/pmc/articles/instance/2043575/bin/zmk0110782920008.gif" src-large="/pmc/articles/instance/2043575/bin/zmk0110782920008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2043575/" ref="ordinalpos=2910&amp;ncbi_uid=1153112&amp;link_uid=PMC2043575" image-link="/pmc/articles/PMC2043575/figure/F8/" class="imagepopup">Figure 8. From: A Single MAPKKK Regulates the Hog1 MAPK <span class="highlight" style="background-color:">Pathway</span> in the Pathogenic Fungus Candida albicans. </a></p><div class="supp"><p class="details"><div>Model depicting the relay of osmotic and oxidative stress signals to the Hog1 MAPK in <i>C. albicans</i>. Oxidative stress <span class="highlight" style="background-color:">signaling</span> to Hog1 requires the Ssk1 response regulator (Chauhan et al., 2003), the Ssk2 MAPKKK (this study), and the Pbs2 MAPKK (Arana et al., 2005). However, which histidine kinase(s) regulates Ssk1 in response to oxidative stress is unclear (Li et al., 2004; Roman et al., 2005). Although the relay of osmotic stress signals to Hog1 also requires Ssk2 (this study) and Pbs2 (Arana et al., 2005), Hog1 is activated in response to osmotic stress in <i>ssk1</i> mutant cells (Chauhan et al., 2003). This, together with the finding that the Sho1–Ste11 <span class="highlight" style="background-color:">pathway</span> does not transmit osmotic stress signals to the Pbs2 MAPKK (Roman et al., 2005; this study) indicates the presence of a novel osmotic-stress <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. albicans</i>. Such a <span class="highlight" style="background-color:">pathway</span> may function in parallel with the Ssk1 response regulator to relay osmotic stress signals to Hog1.</div></p></div><div class="aux"><div class="resc">Jill Cheetham, et al. Mol Biol Cell. 2007 November;18(11):4603-4614.</div><p class="links"><a class="dblinks" href="/pubmed/17804815" ref="ordinalpos=2910&amp;ncbi_uid=1153112&amp;link_uid=17804815">Citation</a><a class="dblinks" href="/pmc/articles/PMC2043575/" ref="ordinalpos=2910&amp;ncbi_uid=1153112&amp;link_uid=PMC2043575">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2911.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3427795/figure/F1/" ref="ordinalpos=2911&amp;ncbi_uid=4339663&amp;link_uid=PMC3427795" href="/pmc/articles/PMC3427795/"><img src="/pmc/articles/instance/3427795/bin/nihms-284990-f0001.gif" src-large="/pmc/articles/instance/3427795/bin/nihms-284990-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3427795/" ref="ordinalpos=2911&amp;ncbi_uid=4339663&amp;link_uid=PMC3427795" image-link="/pmc/articles/PMC3427795/figure/F1/" class="imagepopup">Figure 1. The interplay between FOXO, TOR, and Akt.  From: Interplay between FOXO, TOR, and Akt. </a></p><div class="supp"><p class="details"><div><b>A.</b> The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from Insulin/IGF1 receptor through IRS, PI3K, and Akt in mammalian cells. Akt inhibits FOXO through direct phosphorylation, and indirectly activates mTORC1, which in turn elevates protein synthesis. mTORC1 and its downstream effector, S6K, elicit negative feedback loops to inhibit Akt. When activated FOXO induces the expression of Sestrin 3, which activates AMPK to inhibit mTORC1. FOXO also induces Insulin/IGF1 receptors, IRS2, and Rictor expression to activate Akt (see text for details). <b>B.</b> The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from Insulin/IGF1 receptor through IRS, PI3K, and Akt in flies. The mechanisms of FOXO inhibition and mTORC1 activation by Akt are conserved in flies and mammals. In flies, the mechanism of TORC1 inhibition by FOXO through Sestrin and AMPK is conserved. In addition, FOXO elevates the expression of 4E-BP downstream of TORC1, and induces the expression of Insuin/IGF1 receptor to activate Akt (see text for details). <b>C.</b> The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from Insulin/IGF1 receptor through PI3K, and Akt in worms. The inhibition of FOXO by Akt is conserved in worms. However, the mechanism of TORC1 activation by Akt is not conserved. Akt can activate TORC1 through inhibition of FOXO because FOXO inhibits the expression of Raptor, which is required for TORC1 activity (see text for details).</div></p></div><div class="aux"><div class="resc">Nissim Hay. Biochim Biophys Acta. ;1813(11):1965-1970.</div><p class="links"><a class="dblinks" href="/pubmed/21440577" ref="ordinalpos=2911&amp;ncbi_uid=4339663&amp;link_uid=21440577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3427795/" ref="ordinalpos=2911&amp;ncbi_uid=4339663&amp;link_uid=PMC3427795">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2912.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3898101/figure/fig8/" ref="ordinalpos=2912&amp;ncbi_uid=5733336&amp;link_uid=PMC3898101" href="/pmc/articles/PMC3898101/"><img src="/pmc/articles/instance/3898101/bin/onc201344f8.gif" src-large="/pmc/articles/instance/3898101/bin/onc201344f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3898101/" ref="ordinalpos=2912&amp;ncbi_uid=5733336&amp;link_uid=PMC3898101" image-link="/pmc/articles/PMC3898101/figure/fig8/" class="imagepopup">Figure 8.  From: Activation of PDK-1 maintains mouse embryonic stem cell self-renewal in a PKB-dependent manner. </a></p><div class="supp"><p class="details"><div>Separate PI3K and Wnt <span class="highlight" style="background-color:">signaling</span> pathways. GSK-3 is a key regulator in both the PI3K and Wnt signal transduction pathways. In the context of the myr-PDK-1 and PKB-DD transgenic ES cell lines, the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> does not cross talk with the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through GSK-3. APC, adenomatous polyposis coli; LRP, lipoprotein-related protein; GF, growth factor.</div></p></div><div class="aux"><div class="resc">L S Ling, et al. Oncogene. 2013 November 21;32(47):5397-5408.</div><p class="links"><a class="dblinks" href="/pubmed/23455320" ref="ordinalpos=2912&amp;ncbi_uid=5733336&amp;link_uid=23455320">Citation</a><a class="dblinks" href="/pmc/articles/PMC3898101/" ref="ordinalpos=2912&amp;ncbi_uid=5733336&amp;link_uid=PMC3898101">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2913.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3149633/figure/pone-0023057-g002/" ref="ordinalpos=2913&amp;ncbi_uid=3149050&amp;link_uid=PMC3149633" href="/pmc/articles/PMC3149633/"><img src="/pmc/articles/instance/3149633/bin/pone.0023057.g002.gif" src-large="/pmc/articles/instance/3149633/bin/pone.0023057.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3149633/" ref="ordinalpos=2913&amp;ncbi_uid=3149050&amp;link_uid=PMC3149633" image-link="/pmc/articles/PMC3149633/figure/pone-0023057-g002/" class="imagepopup">Figure 2. High TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activity is associated with decreased patient survival..  From: The Notch and TGF-? <span class="highlight" style="background-color:">Signaling</span> Pathways Contribute to the Aggressiveness of Clear Cell Renal Cell Carcinoma. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Kaplan-Meier curves comparing disease-specific survival among 176 CCRCC patients divided into quartiles based on the TGF-β activity score. High TGF-β <span class="highlight" style="background-color:">pathway</span> activity score (76–100%) was significantly (log-rank, p = 0.006) associated with a worse disease-specific survival. (<b>B</b>) Immunohistochemical analyses using phospho-specific SMAD2 (pSMAD2) antibody of control (c) or SB431542 treated 786-O cells. Sections were counterstained with hematoxylin-eosin. Original magnification, ×40. (<b>C</b>) pSMAD2 is expressed <i>in vivo</i>. Immunohistochemistry of pSMAD2 in nine CCRCC tumors, that all showed positive nuclear staining. Sections were counterstained with hematoxylin-eosin. Original magnification, ×40.</div></p></div><div class="aux"><div class="resc">Jonas Sjölund, et al. PLoS One. 2011;6(8):e23057.</div><p class="links"><a class="dblinks" href="/pubmed/21826227" ref="ordinalpos=2913&amp;ncbi_uid=3149050&amp;link_uid=21826227">Citation</a><a class="dblinks" href="/pmc/articles/PMC3149633/" ref="ordinalpos=2913&amp;ncbi_uid=3149050&amp;link_uid=PMC3149633">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2914.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3522877/figure/f08/" ref="ordinalpos=2914&amp;ncbi_uid=4461147&amp;link_uid=PMC3522877" href="/pmc/articles/PMC3522877/"><img src="/pmc/articles/instance/3522877/bin/bio-01-12-1169-f08.gif" src-large="/pmc/articles/instance/3522877/bin/bio-01-12-1169-f08.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3522877/" ref="ordinalpos=2914&amp;ncbi_uid=4461147&amp;link_uid=PMC3522877" image-link="/pmc/articles/PMC3522877/figure/f08/" class="imagepopup">Fig. 8.A schematic representation of anti-motility versus anti-growth <span class="highlight" style="background-color:">signaling</span> by TGFβ3 in two human skin cell types critical for wound healing..  From: The anti-motility <span class="highlight" style="background-color:">signaling</span> mechanism of TGF?3 that controls cell traffic during skin wound healing. </a></p><div class="supp"><p class="details"><div>While TGFβ3 blocks proliferation of both HDFs and HKCs, it selectively inhibits migration of HDFs, but not HKCs, due to 7 to 10 fold higher expression of TβRII (Bandyopadhyay et al., 2006). TGFβ3 uses TβRII/TβRI/Smads<sub>2/3</sub>/Smad4/PKA <span class="highlight" style="background-color:">pathway</span> to transmit its anti-motility signal. The anti-proliferation <span class="highlight" style="background-color:">signaling</span> by TGFβ is beyond the scope of this study. The numbers in “n  = ” are arbitrary to reflect the differences in TβR expression in fold between HDFs and HKCs.</div></p></div><div class="aux"><div class="resc">Arum Han, et al. Biol Open. 2012 December 15;1(12):1169-1177.</div><p class="links"><a class="dblinks" href="/pubmed/23259050" ref="ordinalpos=2914&amp;ncbi_uid=4461147&amp;link_uid=23259050">Citation</a><a class="dblinks" href="/pmc/articles/PMC3522877/" ref="ordinalpos=2914&amp;ncbi_uid=4461147&amp;link_uid=PMC3522877">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2915.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3048767/figure/fig1/" ref="ordinalpos=2915&amp;ncbi_uid=2817156&amp;link_uid=PMC3048767" href="/pmc/articles/PMC3048767/"><img src="/pmc/articles/instance/3048767/bin/ja-2010-109979_0002.gif" src-large="/pmc/articles/instance/3048767/bin/ja-2010-109979_0002.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3048767/" ref="ordinalpos=2915&amp;ncbi_uid=2817156&amp;link_uid=PMC3048767" image-link="/pmc/articles/PMC3048767/figure/fig1/" class="imagepopup">Figure 1.  From: Light-Activated Kinases Enable Temporal Dissection of <span class="highlight" style="background-color:">Signaling</span> Networks in Living Cells. </a></p><div class="supp"><p class="details"><div>Isolating a subnetwork in MAP kinase <span class="highlight" style="background-color:">signaling</span> via genetically encoding a photocaged lysine in MEK1 active site. (a) Schematic of the MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the synthetic, photoactivable subnetwork. (b) Caging a near universally conserved lysine (K97) in the MEK1 active site inactivates the enzyme by sterically blocking ATP binding. Decaging with light rapidly removes the caging group and activates the kinase (figures created using Pymol and MEK1 structure PDB: 1S9J). (c) Photocaged lysine <b>1</b>.</div></p></div><div class="aux"><div class="resc">Arnaud Gautier, et al. J Am Chem Soc. 2011 February 23;133(7):2124-2127.</div><p class="links"><a class="dblinks" href="/pubmed/21271704" ref="ordinalpos=2915&amp;ncbi_uid=2817156&amp;link_uid=21271704">Citation</a><a class="dblinks" href="/pmc/articles/PMC3048767/" ref="ordinalpos=2915&amp;ncbi_uid=2817156&amp;link_uid=PMC3048767">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2916.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3028259/figure/fig8/" ref="ordinalpos=2916&amp;ncbi_uid=3615868&amp;link_uid=PMC3028259" href="/pmc/articles/PMC3028259/"><img src="/pmc/articles/instance/3028259/bin/AJRCMB44199f8.gif" src-large="/pmc/articles/instance/3028259/bin/AJRCMB44199f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3028259/" ref="ordinalpos=2916&amp;ncbi_uid=3615868&amp;link_uid=PMC3028259" image-link="/pmc/articles/PMC3028259/figure/fig8/" class="imagepopup">Figure 8. From: Prostaglandin E2 Inhibits Human Lung Fibroblast Chemotaxis through Disparate Actions on Different E-Prostanoid Receptors. </a></p><div class="supp"><p class="details"><div>The EP receptor intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its effects on HFL-1 cell migration. PGE<sub>2</sub> binds EP1, EP2, EP3, and EP4 receptors, and can either stimulate or inhibit cell migration. AC, adenylyl cyclase; cAMP, cyclic adenosine monophosphate; PKA, protein kinase A; PI3K, phosphatidylinositol-3–kinase; <i>solid line</i>, established signal <span class="highlight" style="background-color:">pathway</span>; <i>dotted line</i>, possible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; <i>↓</i>, inhibit; <i>↑</i>, stimulate.</div></p></div><div class="aux"><div class="resc">Ying-Ji Li, et al. Am J Respir Cell Mol Biol. 2011 January;44(1):99-107.</div><p class="links"><a class="dblinks" href="/pubmed/20203295" ref="ordinalpos=2916&amp;ncbi_uid=3615868&amp;link_uid=20203295">Citation</a><a class="dblinks" href="/pmc/articles/PMC3028259/" ref="ordinalpos=2916&amp;ncbi_uid=3615868&amp;link_uid=PMC3028259">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2917.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4000931/figure/fig3/" ref="ordinalpos=2917&amp;ncbi_uid=6111682&amp;link_uid=PMC4000931" href="/pmc/articles/PMC4000931/"><img src="/pmc/articles/instance/4000931/bin/BMRI2014-797898.003.gif" src-large="/pmc/articles/instance/4000931/bin/BMRI2014-797898.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4000931/" ref="ordinalpos=2917&amp;ncbi_uid=6111682&amp;link_uid=PMC4000931" image-link="/pmc/articles/PMC4000931/figure/fig3/" class="imagepopup">Figure 3.  From: Therapeutic Potential of MicroRNA: A New Target to Treat Intrahepatic Portal Hypertension?. </a></p><div class="supp"><p class="details"><div>Summary of the miRNAs and the VEGF <span class="highlight" style="background-color:">pathway</span> involved in HSCs. The particular VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> affected by miRNAs is described in detail in the review.</div></p></div><div class="aux"><div class="resc">Can-Jie Guo, et al. Biomed Res Int. 2014;2014:797898.</div><p class="links"><a class="dblinks" href="/pubmed/24812632" ref="ordinalpos=2917&amp;ncbi_uid=6111682&amp;link_uid=24812632">Citation</a><a class="dblinks" href="/pmc/articles/PMC4000931/" ref="ordinalpos=2917&amp;ncbi_uid=6111682&amp;link_uid=PMC4000931">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2918.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3684135/figure/f1/" ref="ordinalpos=2918&amp;ncbi_uid=6228150&amp;link_uid=PMC3684135" href="/pmc/articles/PMC3684135/"><img src="/pmc/articles/instance/3684135/bin/fig-1.gif" src-large="/pmc/articles/instance/3684135/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3684135/" ref="ordinalpos=2918&amp;ncbi_uid=6228150&amp;link_uid=PMC3684135" image-link="/pmc/articles/PMC3684135/figure/f1/" class="imagepopup">FIG. 1. From: DNA Microarray Analysis of the Mouse Adrenal Gland for the Detection of Hypothermia Biomarkers: Potential Usefulness for Forensic Investigation. </a></p><div class="supp"><p class="details"><div>Transforming growth factor β (TGF-β) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hypothermia. <i>Fos,</i> which is the most upregulated gene, is found in this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Masataka Takamiya, et al. Ther Hypothermia Temp Manag. 2013 June;3(2):63-73.</div><p class="links"><a class="dblinks" href="/pubmed/23781398" ref="ordinalpos=2918&amp;ncbi_uid=6228150&amp;link_uid=23781398">Citation</a><a class="dblinks" href="/pmc/articles/PMC3684135/" ref="ordinalpos=2918&amp;ncbi_uid=6228150&amp;link_uid=PMC3684135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2919.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3396054/figure/F3/" ref="ordinalpos=2919&amp;ncbi_uid=4119568&amp;link_uid=PMC3396054" href="/pmc/articles/PMC3396054/"><img src="/pmc/articles/instance/3396054/bin/TORJ-6-156_F3.gif" src-large="/pmc/articles/instance/3396054/bin/TORJ-6-156_F3.jpg" alt="Fig. (3)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3396054/" ref="ordinalpos=2919&amp;ncbi_uid=4119568&amp;link_uid=PMC3396054" image-link="/pmc/articles/PMC3396054/figure/F3/" class="imagepopup">Fig. (3).  From: The Role of TGF-? Receptors in Fibrosis. </a></p><div class="supp"><p class="details"><div>TGF-β activates Smad 2/3 and Smad1 <span class="highlight" style="background-color:">pathway</span>. Activation of classical Smad 2/3 <span class="highlight" style="background-color:">pathway</span> promotes wound healing. Elevated levels of Endoglin in fibroblasts promote Smad1 <span class="highlight" style="background-color:">signaling</span> leading to autocrine upregulation of CCN2 and promoting fibrosis.</div></p></div><div class="aux"><div class="resc">Sashidhar Nakerakanti, et al. Open Rheumatol J. 2012;6:156-162.</div><p class="links"><a class="dblinks" href="/pubmed/22802914" ref="ordinalpos=2919&amp;ncbi_uid=4119568&amp;link_uid=22802914">Citation</a><a class="dblinks" href="/pmc/articles/PMC3396054/" ref="ordinalpos=2919&amp;ncbi_uid=4119568&amp;link_uid=PMC3396054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2920.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726008/figure/F5/" ref="ordinalpos=2920&amp;ncbi_uid=5133475&amp;link_uid=PMC3726008" href="/pmc/articles/PMC3726008/"><img src="/pmc/articles/instance/3726008/bin/nihms484660f5.gif" src-large="/pmc/articles/instance/3726008/bin/nihms484660f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726008/" ref="ordinalpos=2920&amp;ncbi_uid=5133475&amp;link_uid=PMC3726008" image-link="/pmc/articles/PMC3726008/figure/F5/" class="imagepopup">Fig. 5.  From: A Systems-Based Approach to Investigate Dose- and Time-Dependent Methylmercury-Induced Gene Expression Response in C57BL/6 Mouse Embryos Undergoing Neurulation. </a></p><div class="supp"><p class="details"><div>Dose-dependent MeHg-induced gene expression alterations within the Wnt-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The degree of impact by MeHg is displayed in genes identified to be significantly altered by MeHg (ANOVA, <i>P</i>&lt;0.0005) and linked with the Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (Dennis et al., 2003). Each gene is color coded based on fold change of response (log 2 scale) for each gene with 1, 4, and 6 mg/kg BW MeHg, respectively. Only fold change values associated with 8 h exposure data are displayed. Fold change values are not present for genes identified to be not significantly altered by MeHg. Pathways were generated via the Wnt-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (http://www.genome.jp/) with modifications, including the addition of genes, Wnt5a and Celsr1 to the Non-Canonical Wnt-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> due to recent studies suggesting an important role in activation of this <span class="highlight" style="background-color:">pathway</span> (Montcouquiol, 2007; Montcouquiol et al., 2006, 2008; Qian et al., 2007; Tissir and Goffinet, 2006; Zhou et al., 2007). The Wnt-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> represented the most enriched KEGG <span class="highlight" style="background-color:">pathway</span> within genes significantly altered by MeHg.</div></p></div><div class="aux"><div class="resc">Joshua F. Robinson, et al. Birth Defects Res B Dev Reprod Toxicol. ;89(3):188-200.</div><p class="links"><a class="dblinks" href="/pubmed/20540155" ref="ordinalpos=2920&amp;ncbi_uid=5133475&amp;link_uid=20540155">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726008/" ref="ordinalpos=2920&amp;ncbi_uid=5133475&amp;link_uid=PMC3726008">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2921.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3030666/figure/F2/" ref="ordinalpos=2921&amp;ncbi_uid=3621692&amp;link_uid=PMC3030666" href="/pmc/articles/PMC3030666/"><img src="/pmc/articles/instance/3030666/bin/nihms244872f2.gif" src-large="/pmc/articles/instance/3030666/bin/nihms244872f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3030666/" ref="ordinalpos=2921&amp;ncbi_uid=3621692&amp;link_uid=PMC3030666" image-link="/pmc/articles/PMC3030666/figure/F2/" class="imagepopup">Figure 2. Age-associated alterations in <span class="highlight" style="background-color:">signaling</span> pathways in DCs impacting their functions.  From: Impact of Aging on Dendritic Cell Functions in humans. </a></p><div class="supp"><p class="details"><div><b>A. Pathogens:</b> Figure depicts age-associated changes in <span class="highlight" style="background-color:">signaling</span> mechanisms leading to increased cytokine secretion in response to pathogens and decreased phagocytosis and migration of DCs. Increased basal expression of PTEN results in inhibition of activation of AKT kinase of PI3kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Reduced AKT phosphorylation functions as a positive regulator for TLR <span class="highlight" style="background-color:">signaling</span> leading to increased activation of p38 MAP kinase and NFκB which in turn result in increased proinflammatory cytokine secretion and inflammation. Reduced activation of AKT however inhibits phagocytosis and migration of DCs. <b>B. Self Antigens:</b> Figure depicts age-associated changes in <span class="highlight" style="background-color:">signaling</span> mechanisms leading to increased reactivity of DCs to self- antigens such as DNA. Increased basal level activation of NFκB leads to semi-maturation of DCs which results in enhanced response to self antigens and subsequent inflammation.</div></p></div><div class="aux"><div class="resc">Anshu Agrawal, et al. Ageing Res Rev. ;10(3):336-345.</div><p class="links"><a class="dblinks" href="/pubmed/20619360" ref="ordinalpos=2921&amp;ncbi_uid=3621692&amp;link_uid=20619360">Citation</a><a class="dblinks" href="/pmc/articles/PMC3030666/" ref="ordinalpos=2921&amp;ncbi_uid=3621692&amp;link_uid=PMC3030666">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2922.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2667153/figure/F4/" ref="ordinalpos=2922&amp;ncbi_uid=1832293&amp;link_uid=PMC2667153" href="/pmc/articles/PMC2667153/"><img src="/pmc/articles/instance/2667153/bin/nihms84743f4.gif" src-large="/pmc/articles/instance/2667153/bin/nihms84743f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2667153/" ref="ordinalpos=2922&amp;ncbi_uid=1832293&amp;link_uid=PMC2667153" image-link="/pmc/articles/PMC2667153/figure/F4/" class="imagepopup">Figure 4. Wnt <span class="highlight" style="background-color:">signaling</span> is required for the maintenance of dorsal retinal identity.  From: Canonical Wnt <span class="highlight" style="background-color:">signaling</span> is required for maintenance of dorsal retinal identity. </a></p><div class="supp"><p class="details"><div>The <i>Tg(hsp70l:dkk1-GFP)<sup>w32</sup></i> transgenic zebrafish line, which expresses the secreted Wnt <span class="highlight" style="background-color:">pathway</span> inhibitor Dkk1 upon heat-shock, was used for these experiments. A–J: Dorsal views, anterior left. Embryos were heat-shocked and fixed at the indicated times, and sorted for GFP expression. A, B: Embryos fixed just before Wnt <span class="highlight" style="background-color:">signaling</span> becomes active in dorsal RPE express <i>tbx5</i> normally, showing that <i>tbx5</i> expression initiates properly in the absence of Wnt <span class="highlight" style="background-color:">signaling</span>. C–J: Inhibition of Wnt <span class="highlight" style="background-color:">signaling</span> caused a strong downregulation of <i>tbx5</i> at the early timepoints, with a weaker effect at the last timepoint. This demonstrates a requirement for Wnt <span class="highlight" style="background-color:">signaling</span> in the maintenance of <i>tbx5</i>. K–T: Lateral views, dorsal up, anterior left. K–P: Expression of the Bmp ligands <i>bmp4, gdf6a</i>, and <i>bmp2b</i> are lost from the dorsal retina following Wnt inhibition, suggesting loss of dorsal character. Q–T: <i>pax6b</i> is expressed normally and <i>vax2</i> expands dorsally, suggesting a ventralized retina. U–X: Whole eyes, dorsal up. Expression of <i>ephrinB2a</i> is downregulated in the dorsal retina, but maintained in the lens, and <i>ephB2</i> expands dorsally following Wnt inhibition.</div></p></div><div class="aux"><div class="resc">Eric S. Veien, et al. Development. ;135(24):4101-4111.</div><p class="links"><a class="dblinks" href="/pubmed/19004855" ref="ordinalpos=2922&amp;ncbi_uid=1832293&amp;link_uid=19004855">Citation</a><a class="dblinks" href="/pmc/articles/PMC2667153/" ref="ordinalpos=2922&amp;ncbi_uid=1832293&amp;link_uid=PMC2667153">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2923.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3699457/figure/pone-0067773-g003/" ref="ordinalpos=2923&amp;ncbi_uid=5080233&amp;link_uid=PMC3699457" href="/pmc/articles/PMC3699457/"><img src="/pmc/articles/instance/3699457/bin/pone.0067773.g003.gif" src-large="/pmc/articles/instance/3699457/bin/pone.0067773.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3699457/" ref="ordinalpos=2923&amp;ncbi_uid=5080233&amp;link_uid=PMC3699457" image-link="/pmc/articles/PMC3699457/figure/pone-0067773-g003/" class="imagepopup">Figure 3. Wnt/Frizzled <span class="highlight" style="background-color:">signaling</span> restores the normal mitochondrial phenotype in Alex3-overexpressing cells..  From: The Non-Canonical Wnt/PKC <span class="highlight" style="background-color:">Pathway</span> Regulates Mitochondrial Dynamics through Degradation of the Arm-Like Domain-Containing Protein Alex3. </a></p><div class="supp"><p class="details"><div>(<b>A–F</b>) Co-expression of Alex3 (green) and several members of the Wnt/Frizzled <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Wnt1, Fz2, Wnt5a and Wnt11 reverse the aggregated mitochondrial phenotypes induced by Alex3 overexpression in HEK293AD cells. (<b>G–L</b>) High magnifications of boxed areas shown in (<b>A–F</b>). Note that the aggregated phenotype induced by the expression of Alex3 (G,H) is reversed by the co-expression of different members of the Wnt <span class="highlight" style="background-color:">pathway</span> (I–L). (<b>M</b>) Quantification and graphical representation (mean ± standard deviation) of mitochondrial phenotypes resulting after transfection with distinct Wnts and Fz2, demonstrating different degrees of mitochondrial aggregation reversion by the constructs used. Alex3 protein was visualized in green, control mitochondrial distribution (<b>A</b>) in red (MitDsRed), β-catenin in red (<b>B–F</b>) and nuclei in blue (bisbenzimide). Scale bar: 10 µm.</div></p></div><div class="aux"><div class="resc">Román Serrat, et al. PLoS One. 2013;8(7):e67773.</div><p class="links"><a class="dblinks" href="/pubmed/23844091" ref="ordinalpos=2923&amp;ncbi_uid=5080233&amp;link_uid=23844091">Citation</a><a class="dblinks" href="/pmc/articles/PMC3699457/" ref="ordinalpos=2923&amp;ncbi_uid=5080233&amp;link_uid=PMC3699457">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2924.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3489642/figure/F2/" ref="ordinalpos=2924&amp;ncbi_uid=4670055&amp;link_uid=PMC3489642" href="/pmc/articles/PMC3489642/"><img src="/pmc/articles/instance/3489642/bin/psb-7-1121-g2.gif" src-large="/pmc/articles/instance/3489642/bin/psb-7-1121-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3489642/" ref="ordinalpos=2924&amp;ncbi_uid=4670055&amp;link_uid=PMC3489642" image-link="/pmc/articles/PMC3489642/figure/F2/" class="imagepopup">Figure. From: CML9, a multifunctional Arabidopsis thaliana calmodulin-like protein involved in stress responses and plant growth?. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Hypothetical model involving CML9 in flg22-induced plant growth inhibition through GA-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Model of plant growth control through GA-dependent <span class="highlight" style="background-color:">signaling</span>. Upon GA binding to its soluble receptor (GID1), <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated; DELLA proteins are degraded via the ubiquitin-proteasome <span class="highlight" style="background-color:">pathway</span>. The major players of this GA-<span class="highlight" style="background-color:">signaling</span> cascade are the GA receptors (GID1), the DELLA repressor proteins and the F-box protein part of the SCF E3 Ubiquitin ligase complex. DELLAs are nuclear transcriptional regulators, which interact with other transcription factors (<i>i.e.</i> GA-TF, GA-dependent transcription factor) to modulate expression of GA-responsive genes. (B) Working model of flg22-induced plant growth inhibition that involve GA <span class="highlight" style="background-color:">signaling</span> cascade components and CML9.</div></p></div><div class="aux"><div class="resc">Louis-Jérôme Leba, et al. Plant Signal Behav. 2012 September 1;7(9):1121-1124.</div><p class="links"><a class="dblinks" href="/pubmed/22899061" ref="ordinalpos=2924&amp;ncbi_uid=4670055&amp;link_uid=22899061">Citation</a><a class="dblinks" href="/pmc/articles/PMC3489642/" ref="ordinalpos=2924&amp;ncbi_uid=4670055&amp;link_uid=PMC3489642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2925.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3318714/figure/F8/" ref="ordinalpos=2925&amp;ncbi_uid=4693183&amp;link_uid=PMC3318714" href="/pmc/articles/PMC3318714/"><img src="/pmc/articles/instance/3318714/bin/zbc0131200990008.gif" src-large="/pmc/articles/instance/3318714/bin/zbc0131200990008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3318714/" ref="ordinalpos=2925&amp;ncbi_uid=4693183&amp;link_uid=PMC3318714" image-link="/pmc/articles/PMC3318714/figure/F8/" class="imagepopup">FIGURE 8. From: Connections between Sphingosine Kinase and Phospholipase D in the Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Arabidopsis. </a></p><div class="supp"><p class="details"><div><b>Proposed model for the role of SPHK/phyto-S1P and PLDα1/PA in the ABA-mediated stomatal closure <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> This model depicts the known targets of PLD/PA in the ABA-mediated stomatal closure and other ABA regulators are not included in this model. ABA activates SPHKs through unknown mechanisms and ABA receptors may be involved. The activation of SPHKs produces phyto-S1P that activates PLDα1 to produce PA. PA inhibits ABI1 function but promotes NADPH oxidase to promote ABA-mediated stomatal closure. Meanwhile, PLDα1-produced PA stimulates SPHK activity through a positive loop. <i>Arrows with solid lines</i> indicate established links and <i>arrows with dashed lines</i> denote putative links.</div></p></div><div class="aux"><div class="resc">Liang Guo, et al. J Biol Chem. 2012 March 9;287(11):8286-8296.</div><p class="links"><a class="dblinks" href="/pubmed/22275366" ref="ordinalpos=2925&amp;ncbi_uid=4693183&amp;link_uid=22275366">Citation</a><a class="dblinks" href="/pmc/articles/PMC3318714/" ref="ordinalpos=2925&amp;ncbi_uid=4693183&amp;link_uid=PMC3318714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2926.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3197365/figure/F7/" ref="ordinalpos=2926&amp;ncbi_uid=3529841&amp;link_uid=PMC3197365" href="/pmc/articles/PMC3197365/"><img src="/pmc/articles/instance/3197365/bin/zh40101100800007.gif" src-large="/pmc/articles/instance/3197365/bin/zh40101100800007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3197365/" ref="ordinalpos=2926&amp;ncbi_uid=3529841&amp;link_uid=PMC3197365" image-link="/pmc/articles/PMC3197365/figure/F7/" class="imagepopup">Fig. 7. From: SDF-1/CXCR4 mediates acute protection of cardiac function through myocardial STAT3 <span class="highlight" style="background-color:">signaling</span> following global ischemia/reperfusion injury. </a></p><div class="supp"><p class="details"><div>Disruption of STAT3 <span class="highlight" style="background-color:">pathway</span> on SDF-1-suppressed apoptotic-<span class="highlight" style="background-color:">signaling</span> following acute global I/R injury. SDF-1 treatment neither reduces proapoptotic proteins caspase-8 and caspase-3 nor upregulates antiapoptotic <span class="highlight" style="background-color:">signaling</span> Bcl-2-to-Bax and Bcl-XL in STAT3 inhibitor (Stattic)-treated hearts (<i>A</i>) or cardiac-restricted STAT3KO mouse hearts (<i>B</i>) compared with their counterparts after I/R. Results are means ± SE; <i>n</i> = 5–6/group.</div></p></div><div class="aux"><div class="resc">Chunyan Huang, et al. Am J Physiol Heart Circ Physiol. 2011 October;301(4):H1496-H1505.</div><p class="links"><a class="dblinks" href="/pubmed/21821779" ref="ordinalpos=2926&amp;ncbi_uid=3529841&amp;link_uid=21821779">Citation</a><a class="dblinks" href="/pmc/articles/PMC3197365/" ref="ordinalpos=2926&amp;ncbi_uid=3529841&amp;link_uid=PMC3197365">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2927.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1880897/figure/F4/" ref="ordinalpos=2927&amp;ncbi_uid=943064&amp;link_uid=PMC1880897" href="/pmc/articles/PMC1880897/"><img src="/pmc/articles/instance/1880897/bin/nihms14900f4.gif" src-large="/pmc/articles/instance/1880897/bin/nihms14900f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1880897/" ref="ordinalpos=2927&amp;ncbi_uid=943064&amp;link_uid=PMC1880897" image-link="/pmc/articles/PMC1880897/figure/F4/" class="imagepopup">Figure 4.  From: Genomics and expression profiles of the Hedgehog and Notch <span class="highlight" style="background-color:">signaling</span> pathways in sea urchin development. </a></p><div class="supp"><p class="details"><div>RNA in situ localization pattern for main components of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A–D, <i>Delta</i>; E–G, <i>Notch</i>. Stages are: HB, hatched blastula; MB, mesenchyme blastula; EG, early gastrula; LG, late gastrula; PR, prism. All views are lateral except C,F and J which are indicated as vegetal views.</div></p></div><div class="aux"><div class="resc">Katherine D. Walton, et al. Dev Biol. ;300(1):153-164.</div><p class="links"><a class="dblinks" href="/pubmed/17067570" ref="ordinalpos=2927&amp;ncbi_uid=943064&amp;link_uid=17067570">Citation</a><a class="dblinks" href="/pmc/articles/PMC1880897/" ref="ordinalpos=2927&amp;ncbi_uid=943064&amp;link_uid=PMC1880897">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2928.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2491602/figure/F6/" ref="ordinalpos=2928&amp;ncbi_uid=1510877&amp;link_uid=PMC2491602" href="/pmc/articles/PMC2491602/"><img src="/pmc/articles/instance/2491602/bin/1752-0509-2-64-6.gif" src-large="/pmc/articles/instance/2491602/bin/1752-0509-2-64-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2491602/" ref="ordinalpos=2928&amp;ncbi_uid=1510877&amp;link_uid=PMC2491602" image-link="/pmc/articles/PMC2491602/figure/F6/" class="imagepopup">Figure 6.  From: Integrated modeling and experimental approach for determining transcription factor profiles from fluorescent reporter data. </a></p><div class="supp"><p class="details"><div>TNF-α <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Zuyi Huang, et al. BMC Syst Biol. 2008;2:64-64.</div><p class="links"><a class="dblinks" href="/pubmed/18637177" ref="ordinalpos=2928&amp;ncbi_uid=1510877&amp;link_uid=18637177">Citation</a><a class="dblinks" href="/pmc/articles/PMC2491602/" ref="ordinalpos=2928&amp;ncbi_uid=1510877&amp;link_uid=PMC2491602">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2929.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3520483/figure/F2/" ref="ordinalpos=2929&amp;ncbi_uid=4457750&amp;link_uid=PMC3520483" href="/pmc/articles/PMC3520483/"><img src="/pmc/articles/instance/3520483/bin/fphys-03-00468-g0002.gif" src-large="/pmc/articles/instance/3520483/bin/fphys-03-00468-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3520483/" ref="ordinalpos=2929&amp;ncbi_uid=4457750&amp;link_uid=PMC3520483" image-link="/pmc/articles/PMC3520483/figure/F2/" class="imagepopup">Figure 2.  From: Modulation of myocardial contraction by peroxynitrite. </a></p><div class="supp"><p class="details"><div><b>Pathophysiological <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of high peroxynitrite.</b> This is a diagrammatic representation of high peroxynitrite <span class="highlight" style="background-color:">signaling</span> and various proposed end targets within the cardiomyocyte. The endogenous production of high levels of peroxynitrite can occur via NOS2/NADPH oxidase/xanthine oxidase.</div></p></div><div class="aux"><div class="resc">Mark J. Kohr, et al. Front Physiol. 2012;3:468.</div><p class="links"><a class="dblinks" href="/pubmed/23248603" ref="ordinalpos=2929&amp;ncbi_uid=4457750&amp;link_uid=23248603">Citation</a><a class="dblinks" href="/pmc/articles/PMC3520483/" ref="ordinalpos=2929&amp;ncbi_uid=4457750&amp;link_uid=PMC3520483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2930.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2901141/figure/DDQ208F7/" ref="ordinalpos=2930&amp;ncbi_uid=3121118&amp;link_uid=PMC2901141" href="/pmc/articles/PMC2901141/"><img src="/pmc/articles/instance/2901141/bin/ddq20807.gif" src-large="/pmc/articles/instance/2901141/bin/ddq20807.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2901141/" ref="ordinalpos=2930&amp;ncbi_uid=3121118&amp;link_uid=PMC2901141" image-link="/pmc/articles/PMC2901141/figure/DDQ208F7/" class="imagepopup">Figure 7. From: BMP antagonism protects Nodal <span class="highlight" style="background-color:">signaling</span> in the gastrula to promote the tissue interactions underlying mammalian forebrain and craniofacial patterning. </a></p><div class="supp"><p class="details"><div>Model for the interaction of Nodal and anti-BMP activities during the formation of the anterior primitive streak and its derivatives. (<b>A</b>) BMP <span class="highlight" style="background-color:">signaling</span> activation (blue) is restricted, and Nodal <span class="highlight" style="background-color:">signaling</span> enhanced, by the synergistic interaction of Chrd and Nodal in the APS (green/red) at the MS (A1) stage, and of Nog and Smad3 in the anterior AME (green) at the 0B stage (A2). These interactions promote the specification of the APS and/or its derivatives, the AME and ADE, and the BMP antagonism they convey to promote forebrain initiation in the ectoderm (green arrows). (<b>B</b>) In <i>Chrd</i><sup>−/−</sup>;<i>Nodal</i><sup>+/−</sup> embryos, the APS is defective due to excess BMP activation and reduced Nodal <span class="highlight" style="background-color:">signaling</span> (B1), which results in AME/ADE deficiency (B2). (<b>C</b>) In <i>Nog</i><sup>−/−</sup>;<i>Smad3</i><sup>+/−</sup> embryos, early APS appears normal (C1), but AME emerging from the compromised distal APS (early node) is defective due to ectopic BMP activation and decreased Nodal <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> (C2).</div></p></div><div class="aux"><div class="resc">Yu-Ping Yang, et al. Hum Mol Genet. 2010 August 1;19(15):3030-3042.</div><p class="links"><a class="dblinks" href="/pubmed/20508035" ref="ordinalpos=2930&amp;ncbi_uid=3121118&amp;link_uid=20508035">Citation</a><a class="dblinks" href="/pmc/articles/PMC2901141/" ref="ordinalpos=2930&amp;ncbi_uid=3121118&amp;link_uid=PMC2901141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2931.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3177269/figure/fig4/" ref="ordinalpos=2931&amp;ncbi_uid=3434563&amp;link_uid=PMC3177269" href="/pmc/articles/PMC3177269/"><img src="/pmc/articles/instance/3177269/bin/PP_174151_gs_f4.gif" src-large="/pmc/articles/instance/3177269/bin/PP_174151_gs_f4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3177269/" ref="ordinalpos=2931&amp;ncbi_uid=3434563&amp;link_uid=PMC3177269" image-link="/pmc/articles/PMC3177269/figure/fig4/" class="imagepopup">Figure 4. From: Transcriptomics of Actinorhizal Symbioses Reveals Homologs of the Whole Common Symbiotic <span class="highlight" style="background-color:">Signaling</span> Cascade1[W]. </a></p><div class="supp"><p class="details"><div>Proposed model for a common symbiotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> between actinorhizal plants and legumes. The common genes identified in <i>A. glutinosa</i> and/or <i>C. glauca</i> were assigned to infection and organogenesis pathways. Black boxes indicate legume genes not yet identified in <i>A. glutinosa</i> or <i>C. glauca</i>. The common <span class="highlight" style="background-color:">pathway</span> between RNS and <span style="font-variant: small-caps" class="small-caps">AM</span> is indicated by the gray box. Adapted from Madsen et al. (2010).</div></p></div><div class="aux"><div class="resc">Valérie Hocher, et al. Plant Physiol. 2011 June;156(2):700-711.</div><p class="links"><a class="dblinks" href="/pubmed/21464474" ref="ordinalpos=2931&amp;ncbi_uid=3434563&amp;link_uid=21464474">Citation</a><a class="dblinks" href="/pmc/articles/PMC3177269/" ref="ordinalpos=2931&amp;ncbi_uid=3434563&amp;link_uid=PMC3177269">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2932.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2710288/figure/f6-09_25_zuo/" ref="ordinalpos=2932&amp;ncbi_uid=1993675&amp;link_uid=PMC2710288" href="/pmc/articles/PMC2710288/"><img src="/pmc/articles/instance/2741699/bin/09_25_zuof6.gif" src-large="/pmc/articles/instance/2741699/bin/09_25_zuof6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2710288/" ref="ordinalpos=2932&amp;ncbi_uid=1993675&amp;link_uid=PMC2710288" image-link="/pmc/articles/PMC2710288/figure/f6-09_25_zuo/" class="imagepopup">Figure 6.  From: CD151 Gene Delivery after Myocardial Infarction Promotes Functional Neovascularization and Activates FAK <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed model for FAK-dependent mechanism for the effects of CD151. CD151, through formation of the CD151-integrin complex as a functional unit, activates <span class="highlight" style="background-color:">signaling</span> molecules. Overexpression of CD151 leads to the activation of FAK on Tyr397 and Tyr925. Subsequently, activated FAK397 activates the PI3K/Akt/eNOS <span class="highlight" style="background-color:">pathway</span>, and more NO is produced. Alternately, activated FAK397 and FAK925 contribute to the activation of the MAPK <span class="highlight" style="background-color:">pathway</span>, and ERK and JNK both are phosphorylated and activated. In this process, p38MAPK may not be involved. Whether CD151 promotes angiogenesis through activation of MAPKs remains to be tested.</div></p></div><div class="aux"><div class="resc">Houjuan Zuo, et al. Mol Med. 2009 Sep-Oct;15(9-10):307-315.</div><p class="links"><a class="dblinks" href="/pubmed/19603100" ref="ordinalpos=2932&amp;ncbi_uid=1993675&amp;link_uid=19603100">Citation</a><a class="dblinks" href="/pmc/articles/PMC2710288/" ref="ordinalpos=2932&amp;ncbi_uid=1993675&amp;link_uid=PMC2710288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2933.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4042299/figure/fig3/" ref="ordinalpos=2933&amp;ncbi_uid=6244129&amp;link_uid=PMC4042299" href="/pmc/articles/PMC4042299/"><img src="/pmc/articles/instance/4042299/bin/bcj201429f3.gif" src-large="/pmc/articles/instance/4042299/bin/bcj201429f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4042299/" ref="ordinalpos=2933&amp;ncbi_uid=6244129&amp;link_uid=PMC4042299" image-link="/pmc/articles/PMC4042299/figure/fig3/" class="imagepopup">Figure 3.  From: The role of epigenetics in the biology of multiple myeloma. </a></p><div class="supp"><p class="details"><div>Combination of promoter hypermethylation of tumor suppressor genes acting as inhibitors of cancer pathways as well as abnormal expression of miRNAs contribute to the activation of the cyclin D/CDK <span class="highlight" style="background-color:">pathway</span>, Jak/STAT3 <span class="highlight" style="background-color:">pathway</span>, Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and disruption of DAPk/p53 interaction. The dark circles above genes and miRNAs indicate DNA methylation and silencing of the respective gene/miRNA.</div></p></div><div class="aux"><div class="resc">K Dimopoulos, et al. Blood Cancer J. 2014 May;4(5):e207.</div><p class="links"><a class="dblinks" href="/pubmed/24786391" ref="ordinalpos=2933&amp;ncbi_uid=6244129&amp;link_uid=24786391">Citation</a><a class="dblinks" href="/pmc/articles/PMC4042299/" ref="ordinalpos=2933&amp;ncbi_uid=6244129&amp;link_uid=PMC4042299">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2934.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3669360/figure/pone-0065309-g003/" ref="ordinalpos=2934&amp;ncbi_uid=4971338&amp;link_uid=PMC3669360" href="/pmc/articles/PMC3669360/"><img src="/pmc/articles/instance/3669360/bin/pone.0065309.g003.gif" src-large="/pmc/articles/instance/3669360/bin/pone.0065309.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3669360/" ref="ordinalpos=2934&amp;ncbi_uid=4971338&amp;link_uid=PMC3669360" image-link="/pmc/articles/PMC3669360/figure/pone-0065309-g003/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> pathways of differentially expressed RNAs..  From: The Noncoding RNA Expression Profile and the Effect of lncRNA AK126698 on Cisplatin Resistance in Non-Small-Cell Lung Cancer Cell. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways of differentially expressed upregulated mRNAs (A) and downregulated mRNAs (B). Western blot analysis of MAPK <span class="highlight" style="background-color:">pathway</span> (C) and Cell cycle <span class="highlight" style="background-color:">pathway</span> (D) levels in A549 cells and A549/CDDP cells. P-p44/42 MAPK, phosphor-p44/42 MAPK; P-SAPK/JNK, phosphor-SAPK/JNK; P-cdc2, phosphor-cdc2; P-Rb, phosphor-Rb; P-chk2, phosphor-chk2; P-p53, phosphor-p53. <span class="highlight" style="background-color:">Signaling</span> pathways on the intersection between differently expressed mRNAs and predicted target genes from differently expressed upregulated mRNAs (E) and downregulated mRNAs (F). <span class="highlight" style="background-color:">Pathway</span> analysis was predominantly based on the KEGG database. P-values &lt;0.05 using the two-sided Fisher’s exact test were classed as being statistically significant. The vertical axis represents the <span class="highlight" style="background-color:">pathway</span> category and the horizontal axis represents the -log10 (p value) of these significant pathways.</div></p></div><div class="aux"><div class="resc">Yong Yang, et al. PLoS One. 2013;8(5):e65309.</div><p class="links"><a class="dblinks" href="/pubmed/23741487" ref="ordinalpos=2934&amp;ncbi_uid=4971338&amp;link_uid=23741487">Citation</a><a class="dblinks" href="/pmc/articles/PMC3669360/" ref="ordinalpos=2934&amp;ncbi_uid=4971338&amp;link_uid=PMC3669360">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2935.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633845/figure/pone-0062399-g002/" ref="ordinalpos=2935&amp;ncbi_uid=4833367&amp;link_uid=PMC3633845" href="/pmc/articles/PMC3633845/"><img src="/pmc/articles/instance/3633845/bin/pone.0062399.g002.gif" src-large="/pmc/articles/instance/3633845/bin/pone.0062399.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633845/" ref="ordinalpos=2935&amp;ncbi_uid=4833367&amp;link_uid=PMC3633845" image-link="/pmc/articles/PMC3633845/figure/pone-0062399-g002/" class="imagepopup">Figure 2. Different activities of PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in NAGs, APA and IHA patients..  From: The PI3K/AKT/mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Is Overactivated in Primary Aldosteronism. </a></p><div class="supp"><p class="details"><div>Proteins were analyzed by western blotting using monoclonal antibodies specific for the protein forms. NAGs served as a reference. Panel A shows that p-AKT, p-S6 and VEGF are highly elevated in APA and IHA tissues compared with NAGs. β-actin was used as the endogenous control protein. Panels B, C and D reveal significant elevations of the p-AKT/β-actin, p-S6/β-actin and VEGF/β-actin ratios in APA and IHA tissues by photodensitometric analysis. There were significant differences in the ratios of p-AKT/β-actin, p-S6/β-actin and VEGF/β-actin between NAGs and APA and IHA tissues. Mean and SEM shown; t test:* P&lt;0.05, in comparison with NAGs.</div></p></div><div class="aux"><div class="resc">Hengchuan Su, et al. PLoS One. 2013;8(4):e62399.</div><p class="links"><a class="dblinks" href="/pubmed/23626817" ref="ordinalpos=2935&amp;ncbi_uid=4833367&amp;link_uid=23626817">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633845/" ref="ordinalpos=2935&amp;ncbi_uid=4833367&amp;link_uid=PMC3633845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2936.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3169604/figure/pone-0024468-g006/" ref="ordinalpos=2936&amp;ncbi_uid=3224360&amp;link_uid=PMC3169604" href="/pmc/articles/PMC3169604/"><img src="/pmc/articles/instance/3169604/bin/pone.0024468.g006.gif" src-large="/pmc/articles/instance/3169604/bin/pone.0024468.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3169604/" ref="ordinalpos=2936&amp;ncbi_uid=3224360&amp;link_uid=PMC3169604" image-link="/pmc/articles/PMC3169604/figure/pone-0024468-g006/" class="imagepopup">Figure 6. Effect of vorinostat on the mTor and Ampk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in endometrial cancer cells..  From: The Mechanism of Action of the Histone Deacetylase Inhibitor Vorinostat Involves Interaction with the Insulin-Like Growth Factor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Ishikawa and USPC-2 cells were treated with vorinostat and/or IGF-I, and cell lysates (100 µg) were prepared after 24 h. Western blot analysis was performed with antibodies against pmTor, T-mTor, p-Ampk, T-Ampk, p21, cyclin D1, PI3K and actin. The figure shows the results of a typical experiment, repeated three times with similar results.</div></p></div><div class="aux"><div class="resc">Rive Sarfstein, et al. PLoS One. 2011;6(9):e24468.</div><p class="links"><a class="dblinks" href="/pubmed/21931726" ref="ordinalpos=2936&amp;ncbi_uid=3224360&amp;link_uid=21931726">Citation</a><a class="dblinks" href="/pmc/articles/PMC3169604/" ref="ordinalpos=2936&amp;ncbi_uid=3224360&amp;link_uid=PMC3169604">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2937.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3725164/figure/F1/" ref="ordinalpos=2937&amp;ncbi_uid=5131873&amp;link_uid=PMC3725164" href="/pmc/articles/PMC3725164/"><img src="/pmc/articles/instance/3725164/bin/1471-2172-14-30-1.gif" src-large="/pmc/articles/instance/3725164/bin/1471-2172-14-30-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3725164/" ref="ordinalpos=2937&amp;ncbi_uid=5131873&amp;link_uid=PMC3725164" image-link="/pmc/articles/PMC3725164/figure/F1/" class="imagepopup">Figure 1.  From: The timing of TGF-? inhibition affects the generation of antigen-specific CD8+ T Cells. </a></p><div class="supp"><p class="details"><div><b>TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Panel <b>A</b>: unimpeded TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: 1. Ligand binding induces TGF-β receptor type II (TGF-βRII) dimerization. 2. TGF-β type I (TGF-βRI) receptor recruitment. 3. TGF-βRI phosphorylation and activation. 4. SMAD phosphorylation by TGF-βRI. 5. Co-SMAD4 binding. 7. Translocation to the nucleus to activate or repress target genes. Panel <b>B</b>: sTGF-βR-mediated inhibition of TGF-β <span class="highlight" style="background-color:">signaling</span>. 1. Ligand does not bind TGF-βRII. 2. TGF-βRI is not recruited, phosphorylated, and activated. 3. Downstream effect is inhibition of TGF-βRII mediated phosphorylation of SMAD proteins.</div></p></div><div class="aux"><div class="resc">Jon G Quatromoni, et al. BMC Immunol. 2013;14:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/23865808" ref="ordinalpos=2937&amp;ncbi_uid=5131873&amp;link_uid=23865808">Citation</a><a class="dblinks" href="/pmc/articles/PMC3725164/" ref="ordinalpos=2937&amp;ncbi_uid=5131873&amp;link_uid=PMC3725164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2938.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2048693/figure/fig8/" ref="ordinalpos=2938&amp;ncbi_uid=1169798&amp;link_uid=PMC2048693" href="/pmc/articles/PMC2048693/"><img src="/pmc/articles/instance/2048693/bin/tpc1902940f08.gif" src-large="/pmc/articles/instance/2048693/bin/tpc1902940f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2048693/" ref="ordinalpos=2938&amp;ncbi_uid=1169798&amp;link_uid=PMC2048693" image-link="/pmc/articles/PMC2048693/figure/fig8/" class="imagepopup">Figure 8. From: Rice Pti1a Negatively Regulates RAR1-Dependent Defense Responses. </a></p><div class="supp"><p class="details"><div>Working Models of the Interplay of Pti1a and RAR1 in Defense <span class="highlight" style="background-color:">Signaling</span> in Rice.<br />PAMP-triggered basal resistance and R protein–mediated resistance mostly share common <span class="highlight" style="background-color:">signaling</span> pathways for induction of defense responses, including defense gene expression, accumulation of phytoalexin, and HR. We hypothesize that Os RAR1 positively regulates defense signal transduction required for both basal and R protein–mediated resistance through the unknown protein X. Os Pti1a negatively regulates the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> indirectly by repression of Os RAR1 <b>(A)</b> or by directly acting on the <span class="highlight" style="background-color:">pathway</span> upstream or downstream of X-Os RAR1 <b>(B)</b>.</div></p></div><div class="aux"><div class="resc">Akira Takahashi, et al. Plant Cell. 2007 September;19(9):2940-2951.</div><p class="links"><a class="dblinks" href="/pubmed/17890377" ref="ordinalpos=2938&amp;ncbi_uid=1169798&amp;link_uid=17890377">Citation</a><a class="dblinks" href="/pmc/articles/PMC2048693/" ref="ordinalpos=2938&amp;ncbi_uid=1169798&amp;link_uid=PMC2048693">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2939.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3042531/figure/F3/" ref="ordinalpos=2939&amp;ncbi_uid=3654502&amp;link_uid=PMC3042531" href="/pmc/articles/PMC3042531/"><img src="/pmc/articles/instance/3042531/bin/nihms249589f3.gif" src-large="/pmc/articles/instance/3042531/bin/nihms249589f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3042531/" ref="ordinalpos=2939&amp;ncbi_uid=3654502&amp;link_uid=PMC3042531" image-link="/pmc/articles/PMC3042531/figure/F3/" class="imagepopup">Figure 3. Regulatory loops connecting TLR and ER-<span class="highlight" style="background-color:">signaling</span>.  From: Regulation of Innate Immunity by <span class="highlight" style="background-color:">signaling</span> pathways emerging from the endoplasmic reticulum. </a></p><div class="supp"><p class="details"><div>TLR4 and TLR2 activate the transcription factor XBP1 by selectively activating the IRE1 kinase while suppressing the other branches of the ER-stress <span class="highlight" style="background-color:">pathway</span>. Because of the absence of the other ER-<span class="highlight" style="background-color:">signaling</span> pathways, the activation of IRE1 and XBP1 does not contribute to an ER-stress response. However, activated XBP1 amplifies TLR <span class="highlight" style="background-color:">signaling</span> by enhancing cytokine production.</div></p></div><div class="aux"><div class="resc">Fabio Martinon, et al. Curr Opin Immunol. ;23(1):35-40.</div><p class="links"><a class="dblinks" href="/pubmed/21094031" ref="ordinalpos=2939&amp;ncbi_uid=3654502&amp;link_uid=21094031">Citation</a><a class="dblinks" href="/pmc/articles/PMC3042531/" ref="ordinalpos=2939&amp;ncbi_uid=3654502&amp;link_uid=PMC3042531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2940.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1880897/figure/F3/" ref="ordinalpos=2940&amp;ncbi_uid=943066&amp;link_uid=PMC1880897" href="/pmc/articles/PMC1880897/"><img src="/pmc/articles/instance/1880897/bin/nihms14900f3.gif" src-large="/pmc/articles/instance/1880897/bin/nihms14900f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1880897/" ref="ordinalpos=2940&amp;ncbi_uid=943066&amp;link_uid=PMC1880897" image-link="/pmc/articles/PMC1880897/figure/F3/" class="imagepopup">Figure 3.  From: Genomics and expression profiles of the Hedgehog and Notch <span class="highlight" style="background-color:">signaling</span> pathways in sea urchin development. </a></p><div class="supp"><p class="details"><div>RNA in situ localization pattern for main components of the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A–E, <i>hh</i>; F–J, <i>ptc;</i> K–O, <i>smo</i>. Stages are: EG, early gastrula; MG, mid-gastrula; LG, late gastrula; PR, prism; EPl, early pluteus; PL, 2 day pluteus. All views are lateral.</div></p></div><div class="aux"><div class="resc">Katherine D. Walton, et al. Dev Biol. ;300(1):153-164.</div><p class="links"><a class="dblinks" href="/pubmed/17067570" ref="ordinalpos=2940&amp;ncbi_uid=943066&amp;link_uid=17067570">Citation</a><a class="dblinks" href="/pmc/articles/PMC1880897/" ref="ordinalpos=2940&amp;ncbi_uid=943066&amp;link_uid=PMC1880897">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2941.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3759892/figure/f4-ijms-14-15885/" ref="ordinalpos=2941&amp;ncbi_uid=5263516&amp;link_uid=PMC3759892" href="/pmc/articles/PMC3759892/"><img src="/pmc/articles/instance/3759892/bin/ijms-14-15885f4.gif" src-large="/pmc/articles/instance/3759892/bin/ijms-14-15885f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759892/" ref="ordinalpos=2941&amp;ncbi_uid=5263516&amp;link_uid=PMC3759892" image-link="/pmc/articles/PMC3759892/figure/f4-ijms-14-15885/" class="imagepopup">Figure 4.  From: Angiogenesis-Related Pathways in the Pathogenesis of Ovarian Cancer. </a></p><div class="supp"><p class="details"><div>An overview of the FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nikos G. Gavalas, et al. Int J Mol Sci. 2013 August;14(8):15885-15909.</div><p class="links"><a class="dblinks" href="/pubmed/23903048" ref="ordinalpos=2941&amp;ncbi_uid=5263516&amp;link_uid=23903048">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759892/" ref="ordinalpos=2941&amp;ncbi_uid=5263516&amp;link_uid=PMC3759892">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2942.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436227/figure/f2-0971281/" ref="ordinalpos=2942&amp;ncbi_uid=4230821&amp;link_uid=PMC3436227" href="/pmc/articles/PMC3436227/"><img src="/pmc/articles/instance/3436227/bin/971281.fig2.gif" src-large="/pmc/articles/instance/3436227/bin/971281.fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436227/" ref="ordinalpos=2942&amp;ncbi_uid=4230821&amp;link_uid=PMC3436227" image-link="/pmc/articles/PMC3436227/figure/f2-0971281/" class="imagepopup">Figure 2. From: Molecular pathogenesis of Waldenstr?m's macroglobulinemia. </a></p><div class="supp"><p class="details"><div>Schematic representation of MyD-88 dependent TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Esteban Braggio, et al. Haematologica. 2012 September;97(9):1281-1290.</div><p class="links"><a class="dblinks" href="/pubmed/22773606" ref="ordinalpos=2942&amp;ncbi_uid=4230821&amp;link_uid=22773606">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436227/" ref="ordinalpos=2942&amp;ncbi_uid=4230821&amp;link_uid=PMC3436227">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2943.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3704172/figure/F3/" ref="ordinalpos=2943&amp;ncbi_uid=5069199&amp;link_uid=PMC3704172" href="/pmc/articles/PMC3704172/"><img src="/pmc/articles/instance/3704172/bin/nihms483318f3.gif" src-large="/pmc/articles/instance/3704172/bin/nihms483318f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3704172/" ref="ordinalpos=2943&amp;ncbi_uid=5069199&amp;link_uid=PMC3704172" image-link="/pmc/articles/PMC3704172/figure/F3/" class="imagepopup">Figure 3.  From: MicroRNA regulation in cancer-associated fibroblasts. </a></p><div class="supp"><p class="details"><div>miR-181b and miR-146a induced in CAFs are involved in the regulation of the NF-kB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Olga Aprelikova, et al. Cancer Immunol Immunother. ;61(2):231-237.</div><p class="links"><a class="dblinks" href="/pubmed/22083346" ref="ordinalpos=2943&amp;ncbi_uid=5069199&amp;link_uid=22083346">Citation</a><a class="dblinks" href="/pmc/articles/PMC3704172/" ref="ordinalpos=2943&amp;ncbi_uid=5069199&amp;link_uid=PMC3704172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2944.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2246086/figure/F1/" ref="ordinalpos=2944&amp;ncbi_uid=1299642&amp;link_uid=PMC2246086" href="/pmc/articles/PMC2246086/"><img src="/pmc/articles/instance/2246086/bin/nihms37246f1.gif" src-large="/pmc/articles/instance/2246086/bin/nihms37246f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2246086/" ref="ordinalpos=2944&amp;ncbi_uid=1299642&amp;link_uid=PMC2246086" image-link="/pmc/articles/PMC2246086/figure/F1/" class="imagepopup">Figure 1. Direct interaction of insulin <span class="highlight" style="background-color:">signaling</span> and inflammatory pathways.  From: Inflammation and Insulin Resistance. </a></p><div class="supp"><p class="details"><div>The insulin <span class="highlight" style="background-color:">signaling</span> cascade branches into two main pathways. The PI3K/AKT <span class="highlight" style="background-color:">pathway</span> mediates insulin action on nutrient metabolism including glucose uptake. The Ras/MAPK <span class="highlight" style="background-color:">pathway</span> mediates insulin’s effect on gene expression, but also interacts with the PI3K-AKT <span class="highlight" style="background-color:">pathway</span> to control cell growth and differentiation. Activation of the insulin receptor leads to tyrosine phosphorylation of IRS1 thereby initiating signal transduction. Stimulation of the NFκB and AP-1 Fos/Jun inflammatory pathways results in the activation of the serine kinases, Ikkb and Jnk1, which reduce the <span class="highlight" style="background-color:">signaling</span> ability of IRS1. Additional inflammation-related negative regulators of IRS proteins include the Socs proteins and NO, which are induced in inflammation, and promote IRS degradation. NO also reduces PI3K/Akt activity by s-nitrosylation of Akt.</div></p></div><div class="aux"><div class="resc">Carl de Luca, et al. FEBS Lett. ;582(1):97-105.</div><p class="links"><a class="dblinks" href="/pubmed/18053812" ref="ordinalpos=2944&amp;ncbi_uid=1299642&amp;link_uid=18053812">Citation</a><a class="dblinks" href="/pmc/articles/PMC2246086/" ref="ordinalpos=2944&amp;ncbi_uid=1299642&amp;link_uid=PMC2246086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2945.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3666437/figure/fig2/" ref="ordinalpos=2945&amp;ncbi_uid=4994627&amp;link_uid=PMC3666437" href="/pmc/articles/PMC3666437/"><img src="/pmc/articles/instance/3666437/bin/ECAM2013-510208.002.gif" src-large="/pmc/articles/instance/3666437/bin/ECAM2013-510208.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3666437/" ref="ordinalpos=2945&amp;ncbi_uid=4994627&amp;link_uid=PMC3666437" image-link="/pmc/articles/PMC3666437/figure/fig2/" class="imagepopup">Figure 2.  From: A Systems Biology Approach to Characterize Biomarkers for Blood Stasis Syndrome of Unstable Angina Patients by Integrating MicroRNA and Messenger RNA Expression Profiling. </a></p><div class="supp"><p class="details"><div>NOD-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Upregulated mRNAs were depicted in red by Search &amp; Color <span class="highlight" style="background-color:">Pathway</span> in KEGG.</div></p></div><div class="aux"><div class="resc">Jie Wang, et al. Evid Based Complement Alternat Med. 2013;2013:510208.</div><p class="links"><a class="dblinks" href="/pubmed/23762142" ref="ordinalpos=2945&amp;ncbi_uid=4994627&amp;link_uid=23762142">Citation</a><a class="dblinks" href="/pmc/articles/PMC3666437/" ref="ordinalpos=2945&amp;ncbi_uid=4994627&amp;link_uid=PMC3666437">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2946.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3706831/figure/F5/" ref="ordinalpos=2946&amp;ncbi_uid=5099085&amp;link_uid=PMC3706831" href="/pmc/articles/PMC3706831/"><img src="/pmc/articles/instance/3706831/bin/gm444-5.gif" src-large="/pmc/articles/instance/3706831/bin/gm444-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3706831/" ref="ordinalpos=2946&amp;ncbi_uid=5099085&amp;link_uid=PMC3706831" image-link="/pmc/articles/PMC3706831/figure/F5/" class="imagepopup">Figure 5.  From: An imprinted rheumatoid arthritis methylome signature reflects pathogenic phenotype. </a></p><div class="supp"><p class="details"><div><b>DMGs in the KEGG 'Toll-like receptor <span class="highlight" style="background-color:">signaling</span>' <span class="highlight" style="background-color:">pathway</span></b>. The methylation status at the promoters of genes within the <span class="highlight" style="background-color:">pathway</span> is shown. The coloring scheme is as follows: yellow are hyper-methylated in RA, blue are hypo-methylated in RA and green contain no significantly DML in their promoters.</div></p></div><div class="aux"><div class="resc">John W Whitaker, et al. Genome Med. 2013;5(4):40-40.</div><p class="links"><a class="dblinks" href="/pubmed/23631487" ref="ordinalpos=2946&amp;ncbi_uid=5099085&amp;link_uid=23631487">Citation</a><a class="dblinks" href="/pmc/articles/PMC3706831/" ref="ordinalpos=2946&amp;ncbi_uid=5099085&amp;link_uid=PMC3706831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2947.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634307/figure/F1/" ref="ordinalpos=2947&amp;ncbi_uid=1710385&amp;link_uid=PMC2634307" href="/pmc/articles/PMC2634307/"><img src="/pmc/articles/instance/2634307/bin/psb0306_0381_fig001.gif" src-large="/pmc/articles/instance/2634307/bin/psb0306_0381_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634307/" ref="ordinalpos=2947&amp;ncbi_uid=1710385&amp;link_uid=PMC2634307" image-link="/pmc/articles/PMC2634307/figure/F1/" class="imagepopup">Figure 1.  From: Carbon monoxide. </a></p><div class="supp"><p class="details"><div>Schematic illustration of a proposed model for HO/CO <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of root elongation in wheat. IAA triggers a transient CO accumulation, which activates an NO or ROS-dependent <span class="highlight" style="background-color:">pathway</span>, thus leading to root elongation.</div></p></div><div class="aux"><div class="resc">Wei Xuan, et al. Plant Signal Behav. 2008 June;3(6):381-382.</div><p class="links"><a class="dblinks" href="/pubmed/19704571" ref="ordinalpos=2947&amp;ncbi_uid=1710385&amp;link_uid=19704571">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634307/" ref="ordinalpos=2947&amp;ncbi_uid=1710385&amp;link_uid=PMC2634307">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2948.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2676111/figure/F3/" ref="ordinalpos=2948&amp;ncbi_uid=1794226&amp;link_uid=PMC2676111" href="/pmc/articles/PMC2676111/"><img src="/pmc/articles/instance/2676111/bin/nihms87788f3.gif" src-large="/pmc/articles/instance/2676111/bin/nihms87788f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2676111/" ref="ordinalpos=2948&amp;ncbi_uid=1794226&amp;link_uid=PMC2676111" image-link="/pmc/articles/PMC2676111/figure/F3/" class="imagepopup">Figure 3. Activation of the JAK/STAT <span class="highlight" style="background-color:">pathway</span> by IL-7.  From: Interleukin-7 Stimulates Secretion of S100A4 by Activating the JAK-STAT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human Articular Chondrocytes. </a></p><div class="supp"><p class="details"><div><b>A</b>. Chondrocytes in serum-free confluent monolayer cultures were stimulated with or without 10ng/ml IL-7 for 0–120 minutes. The cell lysates were immunoblotted for the phosphorylated and total JAK-3 and STAT-3 <span class="highlight" style="background-color:">signaling</span> proteins. <b>B.</b> DNA binding activity of STAT-3 was measured in nuclear extracts by using the TransAM ELISA according to manufacturer’s protocol. <b>C.</b> Chondrocytes were pretreated for 45 minutes with a JAK-3 inhibitor (10 uM) before stimulation with IL-7. Cell lysates were immunoblotted for the phosphorylated and total JAK-3 and STAT-3 <span class="highlight" style="background-color:">signaling</span> proteins <b>D</b>. The DNA binding activity of STAT-3 was measured in the nuclear extracts following 30 minutes incubation with IL-7. Values in panel B and D are the mean and SEM of three independent experiments.</div></p></div><div class="aux"><div class="resc">Raghunatha R. Yammani, et al. Arthritis Rheum. ;60(3):792-800.</div><p class="links"><a class="dblinks" href="/pubmed/19248116" ref="ordinalpos=2948&amp;ncbi_uid=1794226&amp;link_uid=19248116">Citation</a><a class="dblinks" href="/pmc/articles/PMC2676111/" ref="ordinalpos=2948&amp;ncbi_uid=1794226&amp;link_uid=PMC2676111">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2949.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1951760/figure/F8/" ref="ordinalpos=2949&amp;ncbi_uid=1070903&amp;link_uid=PMC1951760" href="/pmc/articles/PMC1951760/"><img src="/pmc/articles/instance/1951760/bin/zmk0090781940008.gif" src-large="/pmc/articles/instance/1951760/bin/zmk0090781940008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1951760/" ref="ordinalpos=2949&amp;ncbi_uid=1070903&amp;link_uid=PMC1951760" image-link="/pmc/articles/PMC1951760/figure/F8/" class="imagepopup">Figure 8. From: G protein <span class="highlight" style="background-color:">signaling</span> governing cell fate decisions involves opposing G? subunits in Cryptococcus neoformans. </a></p><div class="supp"><p class="details"><div>Models on the dualism of G protein <span class="highlight" style="background-color:">signaling</span> in the pheromone response <span class="highlight" style="background-color:">pathway</span> in <i>C. neoformans</i>. Under nutrient limiting conditions, Gpa3 mainly serves to repress Gβγ <span class="highlight" style="background-color:">signaling</span> and prevent induction of pheromone response. On pheromone activation, the expression of Gpa2 is induced and in addition to functioning passively through releasing the Gβγ subunit, the active form of Gpa2 also plays a positive <span class="highlight" style="background-color:">signaling</span> role, contributing to the pheromone response that leads to mating, whereas the activated form of Gpa3 inhibits mating. The RGS proteins Crg1 (in the pheromone activated state) and Crg2 (in the basal and activated states) both function to constrain Gpa2 and Gpa3 <span class="highlight" style="background-color:">signaling</span> by stimulating GTPase activity.</div></p></div><div class="aux"><div class="resc">Yen-Ping Hsueh, et al. Mol Biol Cell. 2007 September;18(9):3237-3249.</div><p class="links"><a class="dblinks" href="/pubmed/17581859" ref="ordinalpos=2949&amp;ncbi_uid=1070903&amp;link_uid=17581859">Citation</a><a class="dblinks" href="/pmc/articles/PMC1951760/" ref="ordinalpos=2949&amp;ncbi_uid=1070903&amp;link_uid=PMC1951760">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2950.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2988335/figure/F6/" ref="ordinalpos=2950&amp;ncbi_uid=3597267&amp;link_uid=PMC2988335" href="/pmc/articles/PMC2988335/"><img src="/pmc/articles/instance/2988335/bin/zbc0501039830006.gif" src-large="/pmc/articles/instance/2988335/bin/zbc0501039830006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2988335/" ref="ordinalpos=2950&amp;ncbi_uid=3597267&amp;link_uid=PMC2988335" image-link="/pmc/articles/PMC2988335/figure/F6/" class="imagepopup">FIGURE 6. From: HNK-1 Epitope-carrying Tenascin-C Spliced Variant Regulates the Proliferation of Mouse Embryonic Neural Stem Cells. </a></p><div class="supp"><p class="details"><div><b>Cell death and cell survival in HNK-1 knockdown NSCs.</b> <i>a</i>, cell death of NSCs transfected with HNK-1ST or TNC siRNA was evaluated by TUNEL assay. Tunicamycin was used as a control inducer of cell death (1 μg/ml for 10 h). There is no difference in the numbers of TUNEL<sup>+</sup> cells between control siRNA, HNK-1ST siRNA, and TNC siRNA-transfected cells. <i>Scale bar</i>, 25 μm. <i>b</i>, activation of cell survival <span class="highlight" style="background-color:">signaling</span> and cell death <span class="highlight" style="background-color:">signaling</span> pathways (phosphoinositide 3-kinase-Akt <span class="highlight" style="background-color:">pathway</span> and caspase <span class="highlight" style="background-color:">pathway</span>, respectively) in NSCs transfected with HNK-1ST siRNA was evaluated by Western blotting with antibodies to Akt, phospho-Akt, caspase 3, and β-actin. The cell survival <span class="highlight" style="background-color:">signaling</span> and cell death <span class="highlight" style="background-color:">signaling</span> pathways in NSCs were not inhibited or activated by transfection with HNK-1ST siRNA.</div></p></div><div class="aux"><div class="resc">Hirokazu Yagi, et al. J Biol Chem. 2010 November 26;285(48):37293-37301.</div><p class="links"><a class="dblinks" href="/pubmed/20855890" ref="ordinalpos=2950&amp;ncbi_uid=3597267&amp;link_uid=20855890">Citation</a><a class="dblinks" href="/pmc/articles/PMC2988335/" ref="ordinalpos=2950&amp;ncbi_uid=3597267&amp;link_uid=PMC2988335">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2951.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889251/figure/Fig1/" ref="ordinalpos=2951&amp;ncbi_uid=5702977&amp;link_uid=PMC3889251" href="/pmc/articles/PMC3889251/"><img src="/pmc/articles/instance/3889251/bin/12079_2013_199_Fig1_HTML.gif" src-large="/pmc/articles/instance/3889251/bin/12079_2013_199_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889251/" ref="ordinalpos=2951&amp;ncbi_uid=5702977&amp;link_uid=PMC3889251" image-link="/pmc/articles/PMC3889251/figure/Fig1/" class="imagepopup">Fig. 1.  From: Inhibition of CK2 binding to BMPRIa induces C2C12 differentiation into osteoblasts and adipocytes. </a></p><div class="supp"><p class="details"><div>BMP2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in osteogenesis or adipogenesis. <b>a</b> Ligand binding to BMPRIa and BMPRII results in the release of CK2, followed by phosphorylation of BMPRIa by a constitutively active BMPRII, leading to <span class="highlight" style="background-color:">signaling</span>. <b>b</b> <span class="highlight" style="background-color:">Signaling</span> via Smad1, 5, and 8, ERK, PI3K, JNK, p38, and NF-kB results in osteogenesis. <b>c</b> Activation of Smad1, 5, and 8 and p38 pathways are implicated in adipogenesis</div></p></div><div class="aux"><div class="resc">Oleksandra Moseychuk, et al. J Cell Commun Signal. 2013 December;7(4):265-278.</div><p class="links"><a class="dblinks" href="/pubmed/23637019" ref="ordinalpos=2951&amp;ncbi_uid=5702977&amp;link_uid=23637019">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889251/" ref="ordinalpos=2951&amp;ncbi_uid=5702977&amp;link_uid=PMC3889251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2952.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3947888/figure/F1/" ref="ordinalpos=2952&amp;ncbi_uid=5900902&amp;link_uid=PMC3947888" href="/pmc/articles/PMC3947888/"><img src="/pmc/articles/instance/3947888/bin/nihms557264f1.gif" src-large="/pmc/articles/instance/3947888/bin/nihms557264f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3947888/" ref="ordinalpos=2952&amp;ncbi_uid=5900902&amp;link_uid=PMC3947888" image-link="/pmc/articles/PMC3947888/figure/F1/" class="imagepopup">FIGURE 1.  From: The Role of Astrocytes in Mediating Exogenous Cell-Based Restorative Therapy for Stroke. </a></p><div class="supp"><p class="details"><div>Schematic of proposed mechanisms underlying MSC-enhanced tPA. Our previous studies show that administration of MSCs significantly increased tPA in astrocytes in the ipsilateral IBZ and coculture of MCS with neural parenchymal cells upregulated mRNA and protein levels of Shh and tPA, which were abolished by the small interfering RNA (siRNA) that reduces the expression of the Shh gene (siRNA-Shh), and cyclopamine that specifically blocks Shh <span class="highlight" style="background-color:">signaling</span> (Chen et al., 2002a; Cooper et al., 1998). These data suggest that MSC stimulation of the Shh <span class="highlight" style="background-color:">pathway</span> mediates this increase of tPA. Our in vivo data support the observation that administration of MSCs not only increases tPA but also significantly reduces PAI-1 expression in astrocytes. The TGF-β <span class="highlight" style="background-color:">pathway</span> regulates PAI-1 expression (Docagne et al., 2002; Milei et al., 2004). Our in vitro data show that MSCs repress the TGF-β <span class="highlight" style="background-color:">pathway</span> in astrocytes, which may lead to reduced PAI-1 expression. Blockage of the TGF-β <span class="highlight" style="background-color:">pathway</span> in astrocytes by a TGF-β neutralizing antibody abolished PAI-1 induced by coculture of MSCs and astrocytes. These data suggest that MSCs interact with astrocytes to enhance tPA and concurrently reduce PAI-1 via the TGF-β <span class="highlight" style="background-color:">pathway</span>. The spatiotemporal pattern in the adult IBZ of how the Shh <span class="highlight" style="background-color:">pathway</span> upregulates tPA expression in parenchymal cells and the TGF-β <span class="highlight" style="background-color:">pathway</span> regulates PAI-1 expression in activated astrocytes, have not been fully investigated.</div></p></div><div class="aux"><div class="resc">Yi Li, et al. Glia. 2014 January;62(1):1-16.</div><p class="links"><a class="dblinks" href="/pubmed/24272702" ref="ordinalpos=2952&amp;ncbi_uid=5900902&amp;link_uid=24272702">Citation</a><a class="dblinks" href="/pmc/articles/PMC3947888/" ref="ordinalpos=2952&amp;ncbi_uid=5900902&amp;link_uid=PMC3947888">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2953.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3782004/figure/fig1-1947601913488598/" ref="ordinalpos=2953&amp;ncbi_uid=5329788&amp;link_uid=PMC3782004" href="/pmc/articles/PMC3782004/"><img src="/pmc/articles/instance/3782004/bin/10.1177_1947601913488598-fig1.gif" src-large="/pmc/articles/instance/3782004/bin/10.1177_1947601913488598-fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3782004/" ref="ordinalpos=2953&amp;ncbi_uid=5329788&amp;link_uid=PMC3782004" image-link="/pmc/articles/PMC3782004/figure/fig1-1947601913488598/" class="imagepopup">Figure 1. From: Survival of HER2-Positive Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div>Interface between the HER2 <span class="highlight" style="background-color:">signaling</span> network and apoptotic control centers. HER2 activates 2 major <span class="highlight" style="background-color:">signaling</span> pathways (ERK and AKT) that utilize transcriptional and posttranscriptional mechanisms to inhibit the mitochondrial <span class="highlight" style="background-color:">pathway</span> of apoptosis. The network dynamics are illustrated before the addition of lapatinib (<b>A</b>) and after the addition of the drug (<b>B</b>). These interactions are supported by the literature, but the true complexity of this interface is likely much greater.</div></p></div><div class="aux"><div class="resc">Marc Y. Fink, et al. Genes Cancer. 2013 May;4(5-6):187-195.</div><p class="links"><a class="dblinks" href="/pubmed/24069506" ref="ordinalpos=2953&amp;ncbi_uid=5329788&amp;link_uid=24069506">Citation</a><a class="dblinks" href="/pmc/articles/PMC3782004/" ref="ordinalpos=2953&amp;ncbi_uid=5329788&amp;link_uid=PMC3782004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2954.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3465961/figure/fig1/" ref="ordinalpos=2954&amp;ncbi_uid=4287754&amp;link_uid=PMC3465961" href="/pmc/articles/PMC3465961/"><img src="/pmc/articles/instance/3465961/bin/IJEB2012-548081.001.gif" src-large="/pmc/articles/instance/3465961/bin/IJEB2012-548081.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3465961/" ref="ordinalpos=2954&amp;ncbi_uid=4287754&amp;link_uid=PMC3465961" image-link="/pmc/articles/PMC3465961/figure/fig1/" class="imagepopup">Figure 1.  From: Investigating the Relationship between Topology and Evolution in a Dynamic Nematode Odor Genetic Network. </a></p><div class="supp"><p class="details"><div>Odor-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i> illustrated by <span class="highlight" style="background-color:">pathway</span> position and based on genetic interactions. (a) Summary of volatile odor <span class="highlight" style="background-color:">signaling</span> in <i>C. elegans</i>. (b) Schematic of an interaction-based odor network from <i>C. elegans</i>. </div></p></div><div class="aux"><div class="resc">David A. Fitzpatrick, et al. Int J Evol Biol. 2012;2012:548081.</div><p class="links"><a class="dblinks" href="/pubmed/23056995" ref="ordinalpos=2954&amp;ncbi_uid=4287754&amp;link_uid=23056995">Citation</a><a class="dblinks" href="/pmc/articles/PMC3465961/" ref="ordinalpos=2954&amp;ncbi_uid=4287754&amp;link_uid=PMC3465961">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2955.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3982715/figure/f5-ijo-41-05-1675/" ref="ordinalpos=2955&amp;ncbi_uid=6015179&amp;link_uid=PMC3982715" href="/pmc/articles/PMC3982715/"><img src="/pmc/articles/instance/3982715/bin/IJO-41-05-1675-g04.gif" src-large="/pmc/articles/instance/3982715/bin/IJO-41-05-1675-g04.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3982715/" ref="ordinalpos=2955&amp;ncbi_uid=6015179&amp;link_uid=PMC3982715" image-link="/pmc/articles/PMC3982715/figure/f5-ijo-41-05-1675/" class="imagepopup">Figure 5. From: Differential gene expression pattern in early gastric cancer by an integrative systematic approach. </a></p><div class="supp"><p class="details"><div>The upregulated genes belonging to the KEGG TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (A) and cell cycle <span class="highlight" style="background-color:">pathway</span> (B) in our EGC tissues.</div></p></div><div class="aux"><div class="resc">SEUNGYOON NAM, et al. Int J Oncol. 2012 November;41(5):1675-1682.</div><p class="links"><a class="dblinks" href="/pubmed/22961301" ref="ordinalpos=2955&amp;ncbi_uid=6015179&amp;link_uid=22961301">Citation</a><a class="dblinks" href="/pmc/articles/PMC3982715/" ref="ordinalpos=2955&amp;ncbi_uid=6015179&amp;link_uid=PMC3982715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2956.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4133232/figure/pone-0104718-g010/" ref="ordinalpos=2956&amp;ncbi_uid=6515129&amp;link_uid=PMC4133232" href="/pmc/articles/PMC4133232/"><img src="/pmc/articles/instance/4133232/bin/pone.0104718.g010.gif" src-large="/pmc/articles/instance/4133232/bin/pone.0104718.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4133232/" ref="ordinalpos=2956&amp;ncbi_uid=6515129&amp;link_uid=PMC4133232" image-link="/pmc/articles/PMC4133232/figure/pone-0104718-g010/" class="imagepopup">Figure 10. Summary of the proposed 5-HT3R-mediated downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> underlying 2-Me-5-HT-induced emesis in the least shrew..  From: Serotonin 5-HT3 Receptor-Mediated Vomiting Occurs via the Activation of Ca2+/CaMKII-Dependent ERK1/2 <span class="highlight" style="background-color:">Signaling</span> in the Least Shrew (Cryptotis parva). </a></p><div class="supp"><p class="details"><div>5-HT<sub>3</sub>R stimulation by the selective agonist 2-Me-5-HT causes an influx of extracellular Ca<sup>2+</sup> through 5-HT<sub>3</sub>Rs/L-type Ca<sup>2+</sup> ion channels which increases the free cytoplasmic concentration of Ca<sup>2+</sup>, thereby promoting Ca<sup>2+</sup> release via calcium-induced calcium release (CICR) from the endoplasmic reticulum stores through ryanodine receptors (RyRs). This elevation in cellular Ca<sup>2+</sup> level initiates attachment of calmodulin (CaM) with the 5-HT<sub>3</sub>R, and leads to CaMKIIα activation and subsequent ERK1/2 <span class="highlight" style="background-color:">signaling</span>. The 5-HT<sub>3</sub>R antagonist palonosetron<sup>(1)</sup>, the L-type Ca<sup>2+</sup> channel blocker amlodipine<sup>(2)</sup>, the RyR blocker dantrolene<sup>(3)</sup>, the CaMKII inhibitor KN93<sup>(4)</sup>, and the ERK inhibitor PD98059<sup>(5)</sup>, respectively exhibit anti-emetic efficacy against 2-Me-5-HT-induced vomiting. These findings demonstrate that the 2-Me-5-HT-induced emesis is regulated by 5-HT<sub>3</sub>R-mediated Ca<sup>2+</sup>/CaMKII-dependent ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Weixia Zhong, et al. PLoS One. 2014;9(8):e104718.</div><p class="links"><a class="dblinks" href="/pubmed/25121483" ref="ordinalpos=2956&amp;ncbi_uid=6515129&amp;link_uid=25121483">Citation</a><a class="dblinks" href="/pmc/articles/PMC4133232/" ref="ordinalpos=2956&amp;ncbi_uid=6515129&amp;link_uid=PMC4133232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2957.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634088/figure/F4/" ref="ordinalpos=2957&amp;ncbi_uid=1708458&amp;link_uid=PMC2634088" href="/pmc/articles/PMC2634088/"><img src="/pmc/articles/instance/2634088/bin/cam0203_0192_fig004.gif" src-large="/pmc/articles/instance/2634088/bin/cam0203_0192_fig004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634088/" ref="ordinalpos=2957&amp;ncbi_uid=1708458&amp;link_uid=PMC2634088" image-link="/pmc/articles/PMC2634088/figure/F4/" class="imagepopup">Figure 4.  From: A role for the NG2 proteoglycan in glioma progression. </a></p><div class="supp"><p class="details"><div>Functional Interactions of NG2. On the apical cell surface, the thr-2314 phosphorylated NG2 proteoglycan (green oval) activates α3β1 integrin (yellow dimer) <span class="highlight" style="background-color:">signaling</span> to promote enhanced proliferation. NG2 also promotes cell proliferation via potentiation of growth factor/growth factor receptor <span class="highlight" style="background-color:">signaling</span> (indicated here by red FGF and blue FGFR dimer). On leading edge lamellipodia, thr-2256 phosphorylated NG2 activates α3β1 integrin <span class="highlight" style="background-color:">signaling</span> to promote enhanced motility. NG2-mediated integrin <span class="highlight" style="background-color:">signaling</span> can also enhance cell survival via the PI3/AKT <span class="highlight" style="background-color:">pathway</span>. NG2 also provides a linkage between the cell surface and the extracellular matrix via its interaction with type VI collagen (triple turqouise rods). <span class="highlight" style="background-color:">Signaling</span> pathways are shown in abbreviated format due to space considerations.</div></p></div><div class="aux"><div class="resc">William B Stallcup, et al. Cell Adh Migr. 2008 Jul-Sep;2(3):192-201.</div><p class="links"><a class="dblinks" href="/pubmed/19262111" ref="ordinalpos=2957&amp;ncbi_uid=1708458&amp;link_uid=19262111">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634088/" ref="ordinalpos=2957&amp;ncbi_uid=1708458&amp;link_uid=PMC2634088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2958.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678724/figure/F1/" ref="ordinalpos=2958&amp;ncbi_uid=4982838&amp;link_uid=PMC3678724" href="/pmc/articles/PMC3678724/"><img src="/pmc/articles/instance/3678724/bin/nihms457387f1.gif" src-large="/pmc/articles/instance/3678724/bin/nihms457387f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678724/" ref="ordinalpos=2958&amp;ncbi_uid=4982838&amp;link_uid=PMC3678724" image-link="/pmc/articles/PMC3678724/figure/F1/" class="imagepopup">Figure 1.  From: Diverse mechanisms of AKT <span class="highlight" style="background-color:">pathway</span> activation in human malignancy. </a></p><div class="supp"><p class="details"><div>Alterations of the AKT <span class="highlight" style="background-color:">pathway</span> in human cancer. Activation of tyrosine kinase receptors through growth factor stimulation or constitutive activation of the receptor through mutation/amplification leads to activation of the AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Other mechanisms observed in cancer resulting in AKT <span class="highlight" style="background-color:">signaling</span> include activation of proteins encoded by oncogenes (shown in green) and/or inactivation of tumor suppressors (shown in red). Germline mutations of genes encoding some of these tumor suppressors have been found to be responsible for various hereditary cancer syndromes. AKT-mediated phosphorylation of different downstream proteins converges on <span class="highlight" style="background-color:">signaling</span> pathways important in tumorigenesis, such as cell survival and cell cycle progression (convergence of pathways indicated with a thin, black arrow at the lower part of the figure).</div></p></div><div class="aux"><div class="resc">Mitchell Cheung, et al. Curr Cancer Drug Targets. ;13(3):234-244.</div><p class="links"><a class="dblinks" href="/pubmed/23297823" ref="ordinalpos=2958&amp;ncbi_uid=4982838&amp;link_uid=23297823">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678724/" ref="ordinalpos=2958&amp;ncbi_uid=4982838&amp;link_uid=PMC3678724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2959.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2261378/figure/F9/" ref="ordinalpos=2959&amp;ncbi_uid=1256364&amp;link_uid=PMC2261378" href="/pmc/articles/PMC2261378/"><img src="/pmc/articles/instance/2261378/bin/nihms-39292-f0009.gif" src-large="/pmc/articles/instance/2261378/bin/nihms-39292-f0009.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2261378/" ref="ordinalpos=2959&amp;ncbi_uid=1256364&amp;link_uid=PMC2261378" image-link="/pmc/articles/PMC2261378/figure/F9/" class="imagepopup">Fig. 9.  From: A Golgi fragmentation <span class="highlight" style="background-color:">pathway</span> in neurodegeneration. </a></p><div class="supp"><p class="details"><div>Crosstalk between subcellular organelles during neuronal cell death. Organelles sense damage to neurons and integrate apoptotic <span class="highlight" style="background-color:">signaling</span>. Each organelle can activate specific signal transduction pathways to promote or prevent cell death. Data presented here show that inhibition of the Golgi-mediated cell death <span class="highlight" style="background-color:">pathway</span> only delays apoptosis after NMDA or SNOC insult because other death pathways (e.g., mitochondrial) can intervene. In contrast, interruption of the Golgi <span class="highlight" style="background-color:">pathway</span> produces a more long-lasting inhibition of thapsigargin-initiated damage. Note that the ER can also directly communicate with mitochondria in apoptotic <span class="highlight" style="background-color:">signaling</span> using Ca<sup>2+</sup> as a messenger (Breckenridge et al., 2003; Scorrano, 2003; Scorrano et al., 2003).</div></p></div><div class="aux"><div class="resc">Saya Nakagomi, et al. Neurobiol Dis. ;29(2):221-231.</div><p class="links"><a class="dblinks" href="/pubmed/17964175" ref="ordinalpos=2959&amp;ncbi_uid=1256364&amp;link_uid=17964175">Citation</a><a class="dblinks" href="/pmc/articles/PMC2261378/" ref="ordinalpos=2959&amp;ncbi_uid=1256364&amp;link_uid=PMC2261378">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2960.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2276716/figure/F8/" ref="ordinalpos=2960&amp;ncbi_uid=1379337&amp;link_uid=PMC2276716" href="/pmc/articles/PMC2276716/"><img src="/pmc/articles/instance/2276716/bin/zee0040839180008.gif" src-large="/pmc/articles/instance/2276716/bin/zee0040839180008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2276716/" ref="ordinalpos=2960&amp;ncbi_uid=1379337&amp;link_uid=PMC2276716" image-link="/pmc/articles/PMC2276716/figure/F8/" class="imagepopup">Figure 8.  From: Dopamine Inhibits Basal Prolactin Release in Pituitary Lactotrophs through Pertussis Toxin-Sensitive and -Insensitive <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Dopamine-induced inhibition of basal PRL release in cells with inhibited PI3-kinase and Akt/β-arrestin <span class="highlight" style="background-color:">signaling</span> pathways. A and B, The lack of effect of 3 m<span style="font-variant: small-caps" class="small-caps">m</span> Li<sup>+</sup>, an inhibitor of GSK-3, on bromocriptine (Bromo)-induced inhibition of basal PRL release in cells without (A) and with (B) blocked G<sub>i/o</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (250 ng/ml PTX, overnight). Traces shown are representative from three similar experiments. C, The lack of effects of 1 μ<span style="font-variant: small-caps" class="small-caps">m</span> wortmannin (WT), an inhibitor of PI3-kinase, and 3 m<span style="font-variant: small-caps" class="small-caps">m</span> Li<sup>+</sup>, on dopamine (DA)-induced inhibition of basal PRL release. Traces shown are pooled data from five independent experiments. D, Effects of acute treatment with 100 n<span style="font-variant: small-caps" class="small-caps">m</span> PMA on 1 μ<span style="font-variant: small-caps" class="small-caps">m</span> dopamine (DA)-induced inhibition of PRL release in cells with and without blocked G<sub>i/o</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. PMA was applied 30 min before and during application of dopamine. Traces shown are representative from three similar experiments.</div></p></div><div class="aux"><div class="resc">Arturo E. Gonzalez-Iglesias, et al. Endocrinology. 2008 April;149(4):1470-1479.</div><p class="links"><a class="dblinks" href="/pubmed/18096663" ref="ordinalpos=2960&amp;ncbi_uid=1379337&amp;link_uid=18096663">Citation</a><a class="dblinks" href="/pmc/articles/PMC2276716/" ref="ordinalpos=2960&amp;ncbi_uid=1379337&amp;link_uid=PMC2276716">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2961.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3543627/figure/F10/" ref="ordinalpos=2961&amp;ncbi_uid=5724598&amp;link_uid=PMC3543627" href="/pmc/articles/PMC3543627/"><img src="/pmc/articles/instance/3543627/bin/zh20021368080010.gif" src-large="/pmc/articles/instance/3543627/bin/zh20021368080010.jpg" alt="Fig. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3543627/" ref="ordinalpos=2961&amp;ncbi_uid=5724598&amp;link_uid=PMC3543627" image-link="/pmc/articles/PMC3543627/figure/F10/" class="imagepopup">Fig. 10. From: Vasopressin inhibits apoptosis in renal collecting duct cells. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> networks involved in regulation of apoptosis in collecting duct cells. Schematic shows regulation of caspase cleavage and apoptosis through the cAMP-PKA <span class="highlight" style="background-color:">pathway</span> downstream of the vasopressin V2 receptor as well as the PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Sites of action for the various inhibitors used in the study are indicated by red lettering.</div></p></div><div class="aux"><div class="resc">R. Lance Miller, et al. Am J Physiol Renal Physiol. 2013 January 15;304(2):F177-F188.</div><p class="links"><a class="dblinks" href="/pubmed/23136001" ref="ordinalpos=2961&amp;ncbi_uid=5724598&amp;link_uid=23136001">Citation</a><a class="dblinks" href="/pmc/articles/PMC3543627/" ref="ordinalpos=2961&amp;ncbi_uid=5724598&amp;link_uid=PMC3543627">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2962.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2996036/figure/fig8/" ref="ordinalpos=2962&amp;ncbi_uid=2889344&amp;link_uid=PMC2996036" href="/pmc/articles/PMC2996036/"><img src="/pmc/articles/instance/2996036/bin/PP_161661_wc_f8.gif" src-large="/pmc/articles/instance/2996036/bin/PP_161661_wc_f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2996036/" ref="ordinalpos=2962&amp;ncbi_uid=2889344&amp;link_uid=PMC2996036" image-link="/pmc/articles/PMC2996036/figure/fig8/" class="imagepopup">Figure 8. From: Global Gene Profiling of Laser-Captured Pollen Mother Cells Indicates Molecular Pathways and Gene Subfamilies Involved in Rice Meiosis1[W][OA]. </a></p><div class="supp"><p class="details"><div>Expression of genes in the GA biosynthetic and <span class="highlight" style="background-color:">signaling</span> pathways. The biosynthetic <span class="highlight" style="background-color:">pathway</span> is modified from Yamaguchi (2008). The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is modified from Ueguchi-Tanaka et al. (2007). The intensities are directly labeled besides the RAP identifiers in order as labeled for <i>GID2</i>. The color scale for intensities is shown at bottom left.</div></p></div><div class="aux"><div class="resc">Xiang Tang, et al. Plant Physiol. 2010 December;154(4):1855-1870.</div><p class="links"><a class="dblinks" href="/pubmed/20959420" ref="ordinalpos=2962&amp;ncbi_uid=2889344&amp;link_uid=20959420">Citation</a><a class="dblinks" href="/pmc/articles/PMC2996036/" ref="ordinalpos=2962&amp;ncbi_uid=2889344&amp;link_uid=PMC2996036">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2963.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4284788/figure/ijms-15-23705-f006/" ref="ordinalpos=2963&amp;ncbi_uid=7086391&amp;link_uid=PMC4284788" href="/pmc/articles/PMC4284788/"><img src="/pmc/articles/instance/4284788/bin/ijms-15-23705-g006.gif" src-large="/pmc/articles/instance/4284788/bin/ijms-15-23705-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4284788/" ref="ordinalpos=2963&amp;ncbi_uid=7086391&amp;link_uid=PMC4284788" image-link="/pmc/articles/PMC4284788/figure/ijms-15-23705-f006/" class="imagepopup">Figure 6.  From: Lunasin Sensitivity in Non-Small Cell Lung Cancer Cells Is Linked to Suppression of Integrin <span class="highlight" style="background-color:">Signaling</span> and Changes in Histone Acetylation. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Immunoblot analyses of integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components in lunasin-treated NSCLC cells. Analyses were repeated in three independent experiments and representative data from one experiment is shown; (<b>B</b>) Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing effects of lunasin treatment in H661 cells. RTKs, Receptor Tyrosine Kinases; SFKs, Src Family Kinases. Proteins shown in red indicate <span class="highlight" style="background-color:">signaling</span> steps negatively affected specifically in H661 cells by lunasin treatment; (<b>C</b>) Relative expression levels of integrin <span class="highlight" style="background-color:">signaling</span> proteins. Immunoblots shown in (<b>A</b>) were analyzed using ImageJ software v1.45 (National Institutes of Health, Bethesda, MD, USA). Data represent the mean ± SD for three independent experiments. The asterisk (*) indicates a significant (<i>p</i> &lt; 0.05) difference in expression levels relative to the vehicle control.</div></p></div><div class="aux"><div class="resc">Junichi Inaba, et al. Int J Mol Sci. 2014 December;15(12):23705-23724.</div><p class="links"><a class="dblinks" href="/pubmed/25530619" ref="ordinalpos=2963&amp;ncbi_uid=7086391&amp;link_uid=25530619">Citation</a><a class="dblinks" href="/pmc/articles/PMC4284788/" ref="ordinalpos=2963&amp;ncbi_uid=7086391&amp;link_uid=PMC4284788">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2964.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3367552/figure/A009654F6/" ref="ordinalpos=2964&amp;ncbi_uid=6185153&amp;link_uid=PMC3367552" href="/pmc/articles/PMC3367552/"><img src="/pmc/articles/instance/3367552/bin/cshperspect-SIG-009654_F6.gif" src-large="/pmc/articles/instance/3367552/bin/cshperspect-SIG-009654_F6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3367552/" ref="ordinalpos=2964&amp;ncbi_uid=6185153&amp;link_uid=PMC3367552" image-link="/pmc/articles/PMC3367552/figure/A009654F6/" class="imagepopup">Figure 6. From: <span class="highlight" style="background-color:">Signaling</span> Pathways in Cell Polarity. </a></p><div class="supp"><p class="details"><div>Interaction map showing potential links between the Par proteins and components of the Hippo <span class="highlight" style="background-color:">pathway</span>. A <span class="highlight" style="background-color:">signaling</span> cascade involving Salvador, Hippo, and Warts controls phosphorylation and nuclear localization of the transcription factors Yorkie (YKI) and TAZ to regulate epithelial growth. Epithelial integrity is monitored by cell-adhesion complexes (E-cadherin, α-catenin) and by the Par and Crumbs polarity complexes through the adaptor Kibra. The extracellular matrix (ECM) also has an impact on YKI nuclear localization through the Rho GTPase, independently of the Hippo <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Luke Martin McCaffrey, et al. Cold Spring Harb Perspect Biol. 2012 June;4(6):a009654.</div><p class="links"><a class="dblinks" href="/pubmed/22553378" ref="ordinalpos=2964&amp;ncbi_uid=6185153&amp;link_uid=22553378">Citation</a><a class="dblinks" href="/pmc/articles/PMC3367552/" ref="ordinalpos=2964&amp;ncbi_uid=6185153&amp;link_uid=PMC3367552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2965.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1413672/figure/F6/" ref="ordinalpos=2965&amp;ncbi_uid=708715&amp;link_uid=PMC1413672" href="/pmc/articles/PMC1413672/"><img src="/pmc/articles/instance/1413672/bin/zpq0050610600006.gif" src-large="/pmc/articles/instance/1413672/bin/zpq0050610600006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1413672/" ref="ordinalpos=2965&amp;ncbi_uid=708715&amp;link_uid=PMC1413672" image-link="/pmc/articles/PMC1413672/figure/F6/" class="imagepopup">Fig. 6. From: Activation of phosphatidylinositol 3-kinase <span class="highlight" style="background-color:">signaling</span> by a mutant thyroid hormone ? receptor. </a></p><div class="supp"><p class="details"><div>Activation of PI3K <span class="highlight" style="background-color:">signaling</span> by PV. The physical interaction of PV with p85α results in the activation of two PI3K downstream pathways: the AKT–ILK–MMP <span class="highlight" style="background-color:">pathway</span> and the AKT–mTOR–p70<sup>S6K</sup> <span class="highlight" style="background-color:">pathway</span>. The activation of the former leads to the degradation of the extracellular matrix involved in cell invasion and metastasis, and the activation of the latter results in increased cell proliferation and suppression of apoptosis. The bracket indicates that PV did not have an effect on the expression of PTEN, suggesting that PV-induced activation of PI3K is not mediated by the repression of PTEN.</div></p></div><div class="aux"><div class="resc">Fumihiko Furuya, et al. Proc Natl Acad Sci U S A. 2006 February 7;103(6):1780-1785.</div><p class="links"><a class="dblinks" href="/pubmed/16446424" ref="ordinalpos=2965&amp;ncbi_uid=708715&amp;link_uid=16446424">Citation</a><a class="dblinks" href="/pmc/articles/PMC1413672/" ref="ordinalpos=2965&amp;ncbi_uid=708715&amp;link_uid=PMC1413672">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2966.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4380406/figure/ppat.1004777.g001/" ref="ordinalpos=2966&amp;ncbi_uid=7401726&amp;link_uid=PMC4380406" href="/pmc/articles/PMC4380406/"><img src="/pmc/articles/instance/4380406/bin/ppat.1004777.g001.gif" src-large="/pmc/articles/instance/4380406/bin/ppat.1004777.g001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4380406/" ref="ordinalpos=2966&amp;ncbi_uid=7401726&amp;link_uid=PMC4380406" image-link="/pmc/articles/PMC4380406/figure/ppat.1004777.g001/" class="imagepopup">Fig 1. Overview of the nutrient-induced TORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: The Impact of Host Diet on Wolbachia Titer in Drosophila. </a></p><div class="aux"><div class="resc">Laura R. Serbus, et al. PLoS Pathog. 2015 March;11(3):e1004777.</div><p class="links"><a class="dblinks" href="/pubmed/25826386" ref="ordinalpos=2966&amp;ncbi_uid=7401726&amp;link_uid=25826386">Citation</a><a class="dblinks" href="/pmc/articles/PMC4380406/" ref="ordinalpos=2966&amp;ncbi_uid=7401726&amp;link_uid=PMC4380406">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2967.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3834580/figure/F1/" ref="ordinalpos=2967&amp;ncbi_uid=5508095&amp;link_uid=PMC3834580" href="/pmc/articles/PMC3834580/"><img src="/pmc/articles/instance/3834580/bin/nihms503603f1.gif" src-large="/pmc/articles/instance/3834580/bin/nihms503603f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3834580/" ref="ordinalpos=2967&amp;ncbi_uid=5508095&amp;link_uid=PMC3834580" image-link="/pmc/articles/PMC3834580/figure/F1/" class="imagepopup">Figure 1. VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Future of targeted agents in metastatic colorectal cancer. </a></p><div class="supp"><p class="details"><div>Adapted with permission from [69].</div></p></div><div class="aux"><div class="resc">Mauricio Burotto, et al. Colorectal Cancer. ;1(5):10.2217/crc.12.52.</div><p class="links"><a class="dblinks" href="/pubmed/24273599" ref="ordinalpos=2967&amp;ncbi_uid=5508095&amp;link_uid=24273599">Citation</a><a class="dblinks" href="/pmc/articles/PMC3834580/" ref="ordinalpos=2967&amp;ncbi_uid=5508095&amp;link_uid=PMC3834580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2968.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2635405/figure/f1-0940157/" ref="ordinalpos=2968&amp;ncbi_uid=1721852&amp;link_uid=PMC2635405" href="/pmc/articles/PMC2635405/"><img src="/pmc/articles/instance/2635405/bin/94157.fig1.gif" src-large="/pmc/articles/instance/2635405/bin/94157.fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2635405/" ref="ordinalpos=2968&amp;ncbi_uid=1721852&amp;link_uid=PMC2635405" image-link="/pmc/articles/PMC2635405/figure/f1-0940157/" class="imagepopup">Figure 1. From: Malarial anemia and STAT6. </a></p><div class="supp"><p class="details"><div>The classical STAT6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (adapted from a paper by Hebenstreit <i>et al.</i>18).</div></p></div><div class="aux"><div class="resc">Kathryn J.H. Robson, et al. Haematologica. 2009 February;94(2):157-159.</div><p class="links"><a class="dblinks" href="/pubmed/19181787" ref="ordinalpos=2968&amp;ncbi_uid=1721852&amp;link_uid=19181787">Citation</a><a class="dblinks" href="/pmc/articles/PMC2635405/" ref="ordinalpos=2968&amp;ncbi_uid=1721852&amp;link_uid=PMC2635405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2969.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3154935/figure/pone-0023111-g004/" ref="ordinalpos=2969&amp;ncbi_uid=3161834&amp;link_uid=PMC3154935" href="/pmc/articles/PMC3154935/"><img src="/pmc/articles/instance/3154935/bin/pone.0023111.g004.gif" src-large="/pmc/articles/instance/3154935/bin/pone.0023111.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3154935/" ref="ordinalpos=2969&amp;ncbi_uid=3161834&amp;link_uid=PMC3154935" image-link="/pmc/articles/PMC3154935/figure/pone-0023111-g004/" class="imagepopup">Figure 4. Genes in surviving neurons prominently represented in canonical pathways associated with development, survival and synaptic plasticity..  From: Influence of Stochastic Gene Expression on the Cell Survival Rheostat after Traumatic Brain Injury. </a></p><div class="supp"><p class="details"><div><b><i>A,</i></b> Canonical ephrin receptor <span class="highlight" style="background-color:">pathway</span>. <b><i>B,</i></b> Canonical neurotrophin//TRK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b><i>C,</i></b> Canonical synaptic long-term potentiation <span class="highlight" style="background-color:">pathway</span>. Red indicates genes expressed &gt;5-fold in surviving neurons relative to dying neurons. Blue indicates genes expressed &gt;5-fold in dying neurons relative to surviving neurons.</div></p></div><div class="aux"><div class="resc">Daniel R. Rojo, et al. PLoS One. 2011;6(8):e23111.</div><p class="links"><a class="dblinks" href="/pubmed/21853077" ref="ordinalpos=2969&amp;ncbi_uid=3161834&amp;link_uid=21853077">Citation</a><a class="dblinks" href="/pmc/articles/PMC3154935/" ref="ordinalpos=2969&amp;ncbi_uid=3161834&amp;link_uid=PMC3154935">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2970.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC135552/figure/fig9/" ref="ordinalpos=2970&amp;ncbi_uid=295380&amp;link_uid=PMC135552" href="/pmc/articles/PMC135552/"><img src="/pmc/articles/instance/135552/bin/00-0458f9.gif" src-large="/pmc/articles/instance/135552/bin/00-0458f9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC135552/" ref="ordinalpos=2970&amp;ncbi_uid=295380&amp;link_uid=PMC135552" image-link="/pmc/articles/PMC135552/figure/fig9/" class="imagepopup">Figure 9. From: slender Rice, a Constitutive Gibberellin Response Mutant, Is Caused by a Null Mutation of the SLR1 Gene, an Ortholog of the Height-Regulating Gene GAI/RGA/RHT/D8. </a></p><div class="supp"><p class="details"><div>Model of the GA Signal Transduction <span class="highlight" style="background-color:">Pathway</span> in Rice, Arabidopsis, and Wheat.<br />Arrows indicate the GA signal transduction <span class="highlight" style="background-color:">pathway</span> from GA reception (GA) to various actions of GA, such as shoot elongation, α-amylase induction, and so on. SLR1 in rice, GAI and RGA in Arabidopsis, and RHT-A1, -B1, and -D1 proteins in wheat negatively regulate the GA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Akira Ikeda, et al. Plant Cell. 2001 May;13(5):999-1010.</div><p class="links"><a class="dblinks" href="/pubmed/11340177" ref="ordinalpos=2970&amp;ncbi_uid=295380&amp;link_uid=11340177">Citation</a><a class="dblinks" href="/pmc/articles/PMC135552/" ref="ordinalpos=2970&amp;ncbi_uid=295380&amp;link_uid=PMC135552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2971.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1150820/figure/fig1/" ref="ordinalpos=2971&amp;ncbi_uid=502597&amp;link_uid=PMC1150820" href="/pmc/articles/PMC1150820/"><img src="/pmc/articles/instance/1150820/bin/zpq0210583550001.gif" src-large="/pmc/articles/instance/1150820/bin/zpq0210583550001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1150820/" ref="ordinalpos=2971&amp;ncbi_uid=502597&amp;link_uid=PMC1150820" image-link="/pmc/articles/PMC1150820/figure/fig1/" class="imagepopup">Fig. 1. From: Uterine Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> is required for implantation. </a></p><div class="supp"><p class="details"><div>The Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated in the circular smooth muscle cells of the myometrium. (<i>A</i>) Whole-mount β-galactosidase staining of a uterus from a nonpregnant transgenic female. (<i>B</i>) Whole-mount β-galactosidase staining of a uterus from a TCF/Lef-lacZ transgenic female at mid-day 4. (<i>C</i>) Whole-mount β-galactosidase staining of a uterus from a TCF/Lef-lacZ transgenic female on the evening of day 4. (<i>D</i> and <i>E</i>) Sagittal (<i>D</i>) and transverse (<i>E</i>) sections of the uterus in <i>C</i> showing activity in the circular smooth muscle cells (csm).</div></p></div><div class="aux"><div class="resc">Othman A. Mohamed, et al. Proc Natl Acad Sci U S A. 2005 June 14;102(24):8579-8584.</div><p class="links"><a class="dblinks" href="/pubmed/15930138" ref="ordinalpos=2971&amp;ncbi_uid=502597&amp;link_uid=15930138">Citation</a><a class="dblinks" href="/pmc/articles/PMC1150820/" ref="ordinalpos=2971&amp;ncbi_uid=502597&amp;link_uid=PMC1150820">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2972.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3989280/figure/pone-0095016-g001/" ref="ordinalpos=2972&amp;ncbi_uid=6044484&amp;link_uid=PMC3989280" href="/pmc/articles/PMC3989280/"><img src="/pmc/articles/instance/3989280/bin/pone.0095016.g001.gif" src-large="/pmc/articles/instance/3989280/bin/pone.0095016.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3989280/" ref="ordinalpos=2972&amp;ncbi_uid=6044484&amp;link_uid=PMC3989280" image-link="/pmc/articles/PMC3989280/figure/pone-0095016-g001/" class="imagepopup">Figure 1. Effect of diosgenin on human Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> gene expression after 24 and 48 h treatment..  From: Sonic Hedgehog Activation Is Implicated in Diosgenin-Induced Megakaryocytic Differentiation of Human Erythroleukemia Cells. </a></p><div class="supp"><p class="details"><div>Cells were treated or not (control) with 10 µM diosgenin for 24 and 48 h then total RNA was extracted. 1µg of total RNA were transcribed into cDNA and used for quantitative-PCR according to the RT<sup>2</sup> profiler PCR array « human hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> ». Relative levels of mRNA gene expression were calculated using the 2<sup>−ΔΔCt</sup> method versus untreated cells. Each value represents the mean ± SEM of three separate experiments.</div></p></div><div class="aux"><div class="resc">Lamia Ghezali, et al. PLoS One. 2014;9(4):e95016.</div><p class="links"><a class="dblinks" href="/pubmed/24740159" ref="ordinalpos=2972&amp;ncbi_uid=6044484&amp;link_uid=24740159">Citation</a><a class="dblinks" href="/pmc/articles/PMC3989280/" ref="ordinalpos=2972&amp;ncbi_uid=6044484&amp;link_uid=PMC3989280">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2973.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2919275/figure/F2/" ref="ordinalpos=2973&amp;ncbi_uid=3057870&amp;link_uid=PMC2919275" href="/pmc/articles/PMC2919275/"><img src="/pmc/articles/instance/2919275/bin/nihms212450f2.gif" src-large="/pmc/articles/instance/2919275/bin/nihms212450f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919275/" ref="ordinalpos=2973&amp;ncbi_uid=3057870&amp;link_uid=PMC2919275" image-link="/pmc/articles/PMC2919275/figure/F2/" class="imagepopup">Fig. 2. PD-1 <span class="highlight" style="background-color:">signaling</span>.  From: The PD-1 <span class="highlight" style="background-color:">Pathway</span> in Tolerance and Autoimmunity. </a></p><div class="supp"><p class="details"><div>Ligation of TCR and PD-1 leads to tyrosine phosphorylation (P) of the ITIM and ITSM of PD-1. Binding of the ITSM by SHP-1 or SHP-2 results in the dephosphorylation of proximal <span class="highlight" style="background-color:">signaling</span> molecules and augmentation of PTEN expression. This effectively attenuates the activation of the PI3K and Akt pathways. PD-1 <span class="highlight" style="background-color:">signaling</span> may result in decreased T-cell proliferation, survival, protein synthesis, and IL-2 production. (Red arrows and text indicate consequence of PD-1-mediated <span class="highlight" style="background-color:">signaling</span>)</div></p></div><div class="aux"><div class="resc">Loise M. Francisco, et al. Immunol Rev. ;236:219-242.</div><p class="links"><a class="dblinks" href="/pubmed/20636820" ref="ordinalpos=2973&amp;ncbi_uid=3057870&amp;link_uid=20636820">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919275/" ref="ordinalpos=2973&amp;ncbi_uid=3057870&amp;link_uid=PMC2919275">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2974.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3784588/figure/fig10/" ref="ordinalpos=2974&amp;ncbi_uid=6450188&amp;link_uid=PMC3784588" href="/pmc/articles/PMC3784588/"><img src="/pmc/articles/instance/3784588/bin/pc_112631_f10.gif" src-large="/pmc/articles/instance/3784588/bin/pc_112631_f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3784588/" ref="ordinalpos=2974&amp;ncbi_uid=6450188&amp;link_uid=PMC3784588" image-link="/pmc/articles/PMC3784588/figure/fig10/" class="imagepopup">Figure 10. From: Jasmonate Regulates the INDUCER OF CBF EXPRESSION-C-REPEAT BINDING FACTOR/DRE BINDING FACTOR1 Cascade and Freezing Tolerance in Arabidopsis[W]. </a></p><div class="supp"><p class="details"><div>Base Expression of Genes Responsive to Cold but Independent of the CBF/DREB1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Genes for the Synthesis of Secondary Metabolites in Soil-Grown <i>coi1-1</i>.<br /><b>(A)</b> Base expression of genes responsive to cold but independent of the CBF/DREB1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 18-d-old soil-grown <i>coi1-1</i>. WT, the wild type.<br /><b>(B)</b> Base expression of genes encoding enzymes for the synthesis of polyamine, anthocyanins, and glutathione in 18-d-old soil-grown <i>coi1-1.</i> Error bars show <span style="font-variant: small-caps" class="small-caps">sd</span> from three independent RNA extractions used for qRT-PCR.</div></p></div><div class="aux"><div class="resc">Yanru Hu, et al. Plant Cell. 2013 August;25(8):2907-2924.</div><p class="links"><a class="dblinks" href="/pubmed/23933884" ref="ordinalpos=2974&amp;ncbi_uid=6450188&amp;link_uid=23933884">Citation</a><a class="dblinks" href="/pmc/articles/PMC3784588/" ref="ordinalpos=2974&amp;ncbi_uid=6450188&amp;link_uid=PMC3784588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2975.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3079765/figure/F1/" ref="ordinalpos=2975&amp;ncbi_uid=3394245&amp;link_uid=PMC3079765" href="/pmc/articles/PMC3079765/"><img src="/pmc/articles/instance/3079765/bin/nihms-257507-f0001.gif" src-large="/pmc/articles/instance/3079765/bin/nihms-257507-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3079765/" ref="ordinalpos=2975&amp;ncbi_uid=3394245&amp;link_uid=PMC3079765" image-link="/pmc/articles/PMC3079765/figure/F1/" class="imagepopup">Figure 1.  From: Drosophila as a model to study cardiac aging. </a></p><div class="supp"><p class="details"><div>Model of the effects of insulin/TOR <span class="highlight" style="background-color:">signaling</span> on cardiac aging.<br />The insulin/TOR <span class="highlight" style="background-color:">signaling</span> is necessary for normal cardiac senescence to occur. A critical target of that <span class="highlight" style="background-color:">pathway</span> is d4eBP whose negative regulation in cardiac tissue is required for normal cardiac senescence to occur (red heart). In our model cardiac aging can be delayed when d4eBP function is maintained, either by increasing dFOXO expression or by reducing insulin-TOR <span class="highlight" style="background-color:">pathway</span> activation (green heart).</div></p></div><div class="aux"><div class="resc">Mayuko Nishimura, et al. Exp Gerontol. ;46(5):326-330.</div><p class="links"><a class="dblinks" href="/pubmed/21130861" ref="ordinalpos=2975&amp;ncbi_uid=3394245&amp;link_uid=21130861">Citation</a><a class="dblinks" href="/pmc/articles/PMC3079765/" ref="ordinalpos=2975&amp;ncbi_uid=3394245&amp;link_uid=PMC3079765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2976.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2099484/figure/F11/" ref="ordinalpos=2976&amp;ncbi_uid=1119849&amp;link_uid=PMC2099484" href="/pmc/articles/PMC2099484/"><img src="/pmc/articles/instance/2099484/bin/1471-2105-8-S7-S16-11.gif" src-large="/pmc/articles/instance/2099484/bin/1471-2105-8-S7-S16-11.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2099484/" ref="ordinalpos=2976&amp;ncbi_uid=1119849&amp;link_uid=PMC2099484" image-link="/pmc/articles/PMC2099484/figure/F11/" class="imagepopup">Figure 11.  From: Computational analysis of biological functions and pathways collectively targeted by co-expressed microRNAs in cancer. </a></p><div class="supp"><p class="details"><div><b>Diagram of Apoptosis <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></b>. A. Genes known to be affected in Pancreatic Cancer are shown in gray. B. Genes from significantly enriched GO categories that are targeted by at least 50% of over-expressed miRNAs from Pancreatic Cancer dataset are shown in gray. Drug that are known to target genes from this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are outlined in dark blue. Drug targets are outlined in light blue.</div></p></div><div class="aux"><div class="resc">Yuriy Gusev, et al. BMC Bioinformatics. 2007;8(Suppl 7):S16-S16.</div><p class="links"><a class="dblinks" href="/pubmed/18047715" ref="ordinalpos=2976&amp;ncbi_uid=1119849&amp;link_uid=18047715">Citation</a><a class="dblinks" href="/pmc/articles/PMC2099484/" ref="ordinalpos=2976&amp;ncbi_uid=1119849&amp;link_uid=PMC2099484">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2977.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2990951/figure/F5/" ref="ordinalpos=2977&amp;ncbi_uid=2588101&amp;link_uid=PMC2990951" href="/pmc/articles/PMC2990951/"><img src="/pmc/articles/instance/2990951/bin/nihms224994f5.gif" src-large="/pmc/articles/instance/2990951/bin/nihms224994f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2990951/" ref="ordinalpos=2977&amp;ncbi_uid=2588101&amp;link_uid=PMC2990951" image-link="/pmc/articles/PMC2990951/figure/F5/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Signaling</span> Pathways Mediating TGF-β and IL-1 stimulation of MMP-9 in CECs.  From: Cytokines and <span class="highlight" style="background-color:">Signaling</span> Pathways Regulating Matrix Metalloproteinase-9 (MMP-9) Expression in Corneal Epithelial Cells. </a></p><div class="supp"><p class="details"><div>MMP-9 promoter activity in was measured in rabbit CEC cultures. Cultures were co-transfected with a luciferase construct under the SV-40 promoter and Pr22. Transiently transfected cells were left untreated or were stimulated with 10 ng/ml of TGF-β2 or IL-1β alone or in combinations with or without various intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors for 24 hours. Promoter activities were analyzed by dividing β–Gal activity by luciferase activity. Intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors (A) SB202190(20μM), (B) JSH-23 (30μM), (C) SP600125(30μM), (D) SIS-3(10μM), and (E) PD98059(40μM) were used. Each value is the mean of results obtained from three independent experiments performed in triplicate. Error bars indicate standard deviation. * indicates a p-value of less than 0.05. ** indicates a p-value of less than 0.005.</div></p></div><div class="aux"><div class="resc">Gabriel M. Gordon, et al. J Cell Physiol. ;221(2):402-411.</div><p class="links"><a class="dblinks" href="/pubmed/19626678" ref="ordinalpos=2977&amp;ncbi_uid=2588101&amp;link_uid=19626678">Citation</a><a class="dblinks" href="/pmc/articles/PMC2990951/" ref="ordinalpos=2977&amp;ncbi_uid=2588101&amp;link_uid=PMC2990951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2978.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3980897/figure/F1/" ref="ordinalpos=2978&amp;ncbi_uid=6056357&amp;link_uid=PMC3980897" href="/pmc/articles/PMC3980897/"><img src="/pmc/articles/instance/3980897/bin/1471-2105-14-S19-S3-1.gif" src-large="/pmc/articles/instance/3980897/bin/1471-2105-14-S19-S3-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3980897/" ref="ordinalpos=2978&amp;ncbi_uid=6056357&amp;link_uid=PMC3980897" image-link="/pmc/articles/PMC3980897/figure/F1/" class="imagepopup">Figure 1.  From: enRoute: dynamic path extraction from biological <span class="highlight" style="background-color:">pathway</span> maps for exploring heterogeneous experimental datasets. </a></p><div class="supp"><p class="details"><div><b>The dual-view setup of the enRoute visualization technique</b>. (a) The ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the Wikipathways database, augmented to show abstract experimental data and a selected path (orange). (b) The selected path is extracted and displayed top-down along with associated experimental data from a TCGA glioblastoma multiforme dataset.</div></p></div><div class="aux"><div class="resc">Christian Partl, et al. BMC Bioinformatics. 2013;14(Suppl 19):S3-S3.</div><p class="links"><a class="dblinks" href="/pubmed/24564375" ref="ordinalpos=2978&amp;ncbi_uid=6056357&amp;link_uid=24564375">Citation</a><a class="dblinks" href="/pmc/articles/PMC3980897/" ref="ordinalpos=2978&amp;ncbi_uid=6056357&amp;link_uid=PMC3980897">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2979.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267860/figure/F7/" ref="ordinalpos=2979&amp;ncbi_uid=4064474&amp;link_uid=PMC3267860" href="/pmc/articles/PMC3267860/"><img src="/pmc/articles/instance/3267860/bin/nihms337420f7.gif" src-large="/pmc/articles/instance/3267860/bin/nihms337420f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267860/" ref="ordinalpos=2979&amp;ncbi_uid=4064474&amp;link_uid=PMC3267860" image-link="/pmc/articles/PMC3267860/figure/F7/" class="imagepopup">Figure 7. Loss of plexin-D1 in direct <span class="highlight" style="background-color:">pathway</span> MSNs selectively increases the strength of thalamostriatal glutamatergic inputs.  From: Semaphorin 3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> controls <span class="highlight" style="background-color:">pathway</span>-specific synapse formation in the striatum. </a></p><div class="supp"><p class="details"><div><b>(a)</b> Schematic of the recording configuration. <b>(b)</b> Image of a 50 μm thick coronal slice showing ChR2-mCherry expression in the parafascicular (PF) nucleus of the thalamus. <b>(c)</b> Confocal image of thalamostriatal axons expressing ChR2-mCherry (<i>red</i>) in the striatum of <i>Drd</i>2-GFP;<i>Drd1a</i>-Cre;<i>plexin-D1</i><sup>f/+</sup> (<i>green</i>) mouse. <b>(d)</b> 2PLSM image (<i>left</i>) of direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs under simultaneous whole-cell voltage-clamp. Examples (<i>right</i>) of blue-light evoked thalamostriatal EPSCs recorded from neighboring direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs in <i>Drd</i>2-GFP;<i>Drd1a</i>-Cre;<i>plexin-D1</i><sup>f/f</sup> conditional KO mice and their littermate control mice. <b>(e)</b> Summary of normalized thalamostriatal EPSCs amplitudes. <b>(f)</b> <i>left</i>, Recording configuration. <i>right</i>, Examples of evoked corticostriatal EPSCs recorded simultaneously in neighboring direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs from a <i>Drd2</i>-GFP;<i>Drd1a</i>-Cre;<i>plexin-D1</i><sup>f/f</sup> mouse and a littermate control. <b>(g)</b> Summary of normalized corticostriatal EPSC amplitudes in direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs. <b>(h)</b> <i>left</i>, Recording configuration. <i>right</i>, Examples of evoked corticostriatal EPSCs recorded simultaneously in neighboring direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs from a <i>Drd2</i>-GFP;<i>Drd1a</i>-Cre;<i>plexin-D1</i><sup>f/f</sup> mouse and a littermate control. <b>(i)</b> Summary of normalized corticostriatal EPSC amplitudes in direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs. L: lateral, A: anterior. (*<i>P</i>&lt;0.05; Wilcoxon). Error bars denote s.e.m.</div></p></div><div class="aux"><div class="resc">Jun B. Ding, et al. Nat Neurosci. ;15(2):215-223.</div><p class="links"><a class="dblinks" href="/pubmed/22179111" ref="ordinalpos=2979&amp;ncbi_uid=4064474&amp;link_uid=22179111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267860/" ref="ordinalpos=2979&amp;ncbi_uid=4064474&amp;link_uid=PMC3267860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2980.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3735704/figure/F8/" ref="ordinalpos=2980&amp;ncbi_uid=6277471&amp;link_uid=PMC3735704" href="/pmc/articles/PMC3735704/"><img src="/pmc/articles/instance/3735704/bin/cc-12-1901-g8.gif" src-large="/pmc/articles/instance/3735704/bin/cc-12-1901-g8.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3735704/" ref="ordinalpos=2980&amp;ncbi_uid=6277471&amp;link_uid=PMC3735704" image-link="/pmc/articles/PMC3735704/figure/F8/" class="imagepopup">Figure. From: Intermittent injections of osteocalcin reverse autophagic dysfunction and endoplasmic reticulum stress resulting from diet-induced obesity in the vascular tissue via the NF?B-p65-dependent mechanism. </a></p><div class="supp"><p class="details"><div><b>Figure 8.</b> Schematic representation of the putative mechanisms characterizing the functional interaction between autophagy, ER stress and insulin <span class="highlight" style="background-color:">signaling</span>. In tunicamycin-induced cells, osteocalcin may activate NFκB-dependent intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via inducing NFκB nuclear translocation, leading to decreased ER stress and autophagy. Simultaneously, inhibited ER stress can activate Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, thereby autophagic dysfunction is reversed by triggering mTOR activity. As a result, osteocalcin treatment may suppress autophagy and ER stress via the NFκB-p65-dependent mechanism.</div></p></div><div class="aux"><div class="resc">Bo Zhou, et al. Cell Cycle. 2013 June 15;12(12):1901-1913.</div><p class="links"><a class="dblinks" href="/pubmed/23708521" ref="ordinalpos=2980&amp;ncbi_uid=6277471&amp;link_uid=23708521">Citation</a><a class="dblinks" href="/pmc/articles/PMC3735704/" ref="ordinalpos=2980&amp;ncbi_uid=6277471&amp;link_uid=PMC3735704">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2981.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3379883/figure/F2/" ref="ordinalpos=2981&amp;ncbi_uid=4069131&amp;link_uid=PMC3379883" href="/pmc/articles/PMC3379883/"><img src="/pmc/articles/instance/3379883/bin/nihms-384772-f0002.gif" src-large="/pmc/articles/instance/3379883/bin/nihms-384772-f0002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3379883/" ref="ordinalpos=2981&amp;ncbi_uid=4069131&amp;link_uid=PMC3379883" image-link="/pmc/articles/PMC3379883/figure/F2/" class="imagepopup">FIGURE 2. Leptin receptor <span class="highlight" style="background-color:">signaling</span>.  From: Leptin and the CNS Control of Glucose Metabolism. </a></p><div class="supp"><p class="details"><div>Leptin binding to LepRb mediates activation of the tyrosine kinase Janus kinase-2 (Jak2), resulting in phosphorylation of three tyrosine residues (Tyr<sub>985</sub>, Tyr<sub>1077</sub>, Tyr<sub>1138</sub>). Phosphorylation of Tyr<sub>985</sub> recruits SHP-2, leading to the activation of the ERK <span class="highlight" style="background-color:">pathway</span>, and also binds to <span style="text-decoration: underline" class="underline">s</span>uppressor <span style="text-decoration: underline" class="underline">o</span>f <span style="text-decoration: underline" class="underline">c</span>ytokine <span style="text-decoration: underline" class="underline">s</span>ignaling-3 (SOCS-3), a negative regulator of LepRb <span class="highlight" style="background-color:">signaling</span>. Phosphorylation of Tyr<sub>1077</sub> mediates <span style="text-decoration: underline" class="underline">s</span>ignal <span style="text-decoration: underline" class="underline">t</span>ransduction and <span style="text-decoration: underline" class="underline">a</span>ctivator of <span style="text-decoration: underline" class="underline">t</span>ranscription-5 (STAT5). Phosphorylation of Tyr<sub>1138</sub> recruits and activates STAT3 and STAT5. Activation of STAT3 leads to increased SOCS-3 expression which acts as a negative feedback inhibitor of leptin <span class="highlight" style="background-color:">signaling</span>, in part, by binding to Tyr<sub>985</sub> of LepRb. In addition, hypothalamic inflammation is also induced during high-fat feeding and increases SOCS-3 expression. Similar to SOCS-3, protein tyrosine phosphatase (PTP)1B may also inhibit LepRb <span class="highlight" style="background-color:">signaling</span> via dephosphorylation of Jak2.</div></p></div><div class="aux"><div class="resc">Gregory J. Morton, et al. Physiol Rev. ;91(2):389-411.</div><p class="links"><a class="dblinks" href="/pubmed/21527729" ref="ordinalpos=2981&amp;ncbi_uid=4069131&amp;link_uid=21527729">Citation</a><a class="dblinks" href="/pmc/articles/PMC3379883/" ref="ordinalpos=2981&amp;ncbi_uid=4069131&amp;link_uid=PMC3379883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2982.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3460778/figure/F1/" ref="ordinalpos=2982&amp;ncbi_uid=4274672&amp;link_uid=PMC3460778" href="/pmc/articles/PMC3460778/"><img src="/pmc/articles/instance/3460778/bin/1756-0500-5-392-1.gif" src-large="/pmc/articles/instance/3460778/bin/1756-0500-5-392-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3460778/" ref="ordinalpos=2982&amp;ncbi_uid=4274672&amp;link_uid=PMC3460778" image-link="/pmc/articles/PMC3460778/figure/F1/" class="imagepopup">Figure 1.  From: Phytohormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis method for comparing hormone responses in plant-pest interactions. </a></p><div class="supp"><p class="details"><div><b>Participation of phytohormones in the response to Asian soybean rust.</b> Hormone <span class="highlight" style="background-color:">pathway</span> scores indicating induction and suppression of <span class="highlight" style="background-color:">signaling</span> for the hormones abscisic acid (ABA), ethylene (ET), jasmonic acid (JA), and salicylic acid (SA) are shown for datasets corresponding to the susceptible (Susc) and the resistance response (Res) at 24 and 168 hours post-infection. The genetic differences (Gen) between susceptible and resistant plants were also analyzed. A three-color scale was used to highlight the scores: blue for negative (suppression), white for no change, and yellow for positive (induction).</div></p></div><div class="aux"><div class="resc">Matthew E Studham, et al. BMC Res Notes. 2012;5:392-392.</div><p class="links"><a class="dblinks" href="/pubmed/22846705" ref="ordinalpos=2982&amp;ncbi_uid=4274672&amp;link_uid=22846705">Citation</a><a class="dblinks" href="/pmc/articles/PMC3460778/" ref="ordinalpos=2982&amp;ncbi_uid=4274672&amp;link_uid=PMC3460778">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2983.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4030835/figure/biomolecules-02-00331-f003/" ref="ordinalpos=2983&amp;ncbi_uid=6287063&amp;link_uid=PMC4030835" href="/pmc/articles/PMC4030835/"><img src="/pmc/articles/instance/4030835/bin/biomolecules-02-00331-g003.gif" src-large="/pmc/articles/instance/4030835/bin/biomolecules-02-00331-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4030835/" ref="ordinalpos=2983&amp;ncbi_uid=6287063&amp;link_uid=PMC4030835" image-link="/pmc/articles/PMC4030835/figure/biomolecules-02-00331-f003/" class="imagepopup">Figure 3.  From: SUMOylation in Drosophila Development. </a></p><div class="supp"><p class="details"><div>Homeodomain interacting protein kinase (Hipk) SUMOylation may down-regulate c-Jun N-terminal kinase (JNK) <span class="highlight" style="background-color:">signaling</span>. Phosphorylation of JNK in the cytoplasm by Hipk may stimulate JNK <span class="highlight" style="background-color:">pathway</span> activity. SUMOylation of Hipk may interfere with JNK <span class="highlight" style="background-color:">signaling</span> by sequestering Hipk in the nucleus. </div></p></div><div class="aux"><div class="resc">Matthew Smith, et al. Biomolecules. 2012 September;2(3):331-349.</div><p class="links"><a class="dblinks" href="/pubmed/24970141" ref="ordinalpos=2983&amp;ncbi_uid=6287063&amp;link_uid=24970141">Citation</a><a class="dblinks" href="/pmc/articles/PMC4030835/" ref="ordinalpos=2983&amp;ncbi_uid=6287063&amp;link_uid=PMC4030835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2984.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3423502/figure/F6/" ref="ordinalpos=2984&amp;ncbi_uid=5306097&amp;link_uid=PMC3423502" href="/pmc/articles/PMC3423502/"><img src="/pmc/articles/instance/3423502/bin/nihms394881f6.gif" src-large="/pmc/articles/instance/3423502/bin/nihms394881f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3423502/" ref="ordinalpos=2984&amp;ncbi_uid=5306097&amp;link_uid=PMC3423502" image-link="/pmc/articles/PMC3423502/figure/F6/" class="imagepopup">Figure 6. A simple model of the Fgf and Wnt gene network in the zebrafish tailbud.  From: Crosstalk between Fgf and Wnt <span class="highlight" style="background-color:">signaling</span> in the zebrafish tailbud. </a></p><div class="supp"><p class="details"><div>In addition to the previously described autoregulatory loops between Fgf, Wnt and T-box genes (grey arrows), we find evidence of more direct intertwining of the Fgf and Wnt pathways (colored arrows). Fgf <span class="highlight" style="background-color:">signaling</span> inhibits transcription of the Wnt inhibitors <i>dkk1</i> and <i>notum1a</i>. PI3k <span class="highlight" style="background-color:">signaling</span> activates the Wnt <span class="highlight" style="background-color:">pathway</span> by phosphorylating (inactivating) Gsk3β. Phosphorylation of Gsk3β also leads to an increase in active Erk levels, suggesting Wnt <span class="highlight" style="background-color:">signaling</span> promotes Mapk activity. Fgf and Wnt have opposing effects on transcription of co-regulated genes, which include Fgf (<i>spred3</i>) and Wnt (<i>dkk1</i>, <i>notum1a</i>) inhibitors.</div></p></div><div class="aux"><div class="resc">Michael J. Stulberg, et al. Dev Biol. ;369(2):298-307.</div><p class="links"><a class="dblinks" href="/pubmed/22796649" ref="ordinalpos=2984&amp;ncbi_uid=5306097&amp;link_uid=22796649">Citation</a><a class="dblinks" href="/pmc/articles/PMC3423502/" ref="ordinalpos=2984&amp;ncbi_uid=5306097&amp;link_uid=PMC3423502">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2985.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3307847/figure/F3/" ref="ordinalpos=2985&amp;ncbi_uid=4873416&amp;link_uid=PMC3307847" href="/pmc/articles/PMC3307847/"><img src="/pmc/articles/instance/3307847/bin/nihms354051f3.gif" src-large="/pmc/articles/instance/3307847/bin/nihms354051f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3307847/" ref="ordinalpos=2985&amp;ncbi_uid=4873416&amp;link_uid=PMC3307847" image-link="/pmc/articles/PMC3307847/figure/F3/" class="imagepopup">Figure 3.  From: Identifying a static nonlinear structure in a biological system using noisy, sparse data. </a></p><div class="supp"><p class="details"><div>(a) The Sre1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>S. pombe</i>. Symbols are defined in the text. (b) Ergosterol sensing in this <span class="highlight" style="background-color:">pathway</span> can be described in the form of Figure 1.</div></p></div><div class="aux"><div class="resc">Joshua R. Porter, et al. J Theor Biol. ;300:232-241.</div><p class="links"><a class="dblinks" href="/pubmed/22310068" ref="ordinalpos=2985&amp;ncbi_uid=4873416&amp;link_uid=22310068">Citation</a><a class="dblinks" href="/pmc/articles/PMC3307847/" ref="ordinalpos=2985&amp;ncbi_uid=4873416&amp;link_uid=PMC3307847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2986.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4065826/figure/F2/" ref="ordinalpos=2986&amp;ncbi_uid=6285490&amp;link_uid=PMC4065826" href="/pmc/articles/PMC4065826/"><img src="/pmc/articles/instance/4065826/bin/en-23-130-g002.gif" src-large="/pmc/articles/instance/4065826/bin/en-23-130-g002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4065826/" ref="ordinalpos=2986&amp;ncbi_uid=6285490&amp;link_uid=PMC4065826" image-link="/pmc/articles/PMC4065826/figure/F2/" class="imagepopup">Fig. 2.  From: Mitogen Activated Protein Kinase Family Proteins and c-jun <span class="highlight" style="background-color:">Signaling</span> in Injury-induced Schwann Cell Plasticity. </a></p><div class="supp"><p class="details"><div>MAP kinase <span class="highlight" style="background-color:">signaling</span> in SC plasticity. NRG1-ErbB2 <span class="highlight" style="background-color:">signaling</span> regulates SC plasticity via c-jun expression which is mediated by JNK and p38 kinase pathways. c-jun increases the expression of GDNF and LIF but suppresses the crucial transcription factor for myelination, Krox20, thereby inducing SC dedifferentiation. The Raf-ERK <span class="highlight" style="background-color:">pathway</span> is involved in the expression of cell cycle proteins and chemotactic factors such as Ccl2.</div></p></div><div class="aux"><div class="resc">Hye Jeong Lee, et al. Exp Neurobiol. 2014 June;23(2):130-137.</div><p class="links"><a class="dblinks" href="/pubmed/24963277" ref="ordinalpos=2986&amp;ncbi_uid=6285490&amp;link_uid=24963277">Citation</a><a class="dblinks" href="/pmc/articles/PMC4065826/" ref="ordinalpos=2986&amp;ncbi_uid=6285490&amp;link_uid=PMC4065826">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2987.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3488630/figure/fig2/" ref="ordinalpos=2987&amp;ncbi_uid=5344062&amp;link_uid=PMC3488630" href="/pmc/articles/PMC3488630/"><img src="/pmc/articles/instance/3488630/bin/AJRCMB474395fig2.gif" src-large="/pmc/articles/instance/3488630/bin/AJRCMB474395fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3488630/" ref="ordinalpos=2987&amp;ncbi_uid=5344062&amp;link_uid=PMC3488630" image-link="/pmc/articles/PMC3488630/figure/fig2/" class="imagepopup">Figure 2. From: Human Epidermal Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span> in Acute Lung Injury. </a></p><div class="supp"><p class="details"><div>Proposed model of HER2/3 <span class="highlight" style="background-color:">signaling</span> in epithelial cells. Injurious stimuli, including IL-1β, induce protease-mediated ligand shedding, which is followed by autocrine or paracrine epithelial HER receptor activation. This induces activation of intracellular <span class="highlight" style="background-color:">signaling</span> pathways, including TGF-β and Rac, with subsequent modulation of tight junction and adherens junction proteins and regulation of epithelial cell–cell adhesion. IL-1β activation of HER2/3 through shedding ADAM17-induced shedding of neuregulin (NRG)-1 serves as a prototypical <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">James H. Finigan, et al. Am J Respir Cell Mol Biol. 2012 October;47(4):395-404.</div><p class="links"><a class="dblinks" href="/pubmed/22652197" ref="ordinalpos=2987&amp;ncbi_uid=5344062&amp;link_uid=22652197">Citation</a><a class="dblinks" href="/pmc/articles/PMC3488630/" ref="ordinalpos=2987&amp;ncbi_uid=5344062&amp;link_uid=PMC3488630">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2988.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3845449/figure/fig1/" ref="ordinalpos=2988&amp;ncbi_uid=5581241&amp;link_uid=PMC3845449" href="/pmc/articles/PMC3845449/"><img src="/pmc/articles/instance/3845449/bin/CDI2013-570731.001.gif" src-large="/pmc/articles/instance/3845449/bin/CDI2013-570731.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3845449/" ref="ordinalpos=2988&amp;ncbi_uid=5581241&amp;link_uid=PMC3845449" image-link="/pmc/articles/PMC3845449/figure/fig1/" class="imagepopup">Figure 1.  From: Notch <span class="highlight" style="background-color:">Signaling</span> and T-Helper Cells in EAE/MS. </a></p><div class="supp"><p class="details"><div>Schematic illustration of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Binding of the extracellular part of Notch receptor to ligands of the Delta and Jagged families induces proteolytic cleavage of Notch, releasing the intracellular part of the protein (NICD). NICD is then translocated to the nucleus and binds to the nuclear transcription factor RBP-J<i>κ</i> inducing its conversion from a repressor into an activator to stimulate the transcription of Notch target genes.</div></p></div><div class="aux"><div class="resc">Ribal Bassil, et al. Clin Dev Immunol. 2013;2013:570731.</div><p class="links"><a class="dblinks" href="/pubmed/24324509" ref="ordinalpos=2988&amp;ncbi_uid=5581241&amp;link_uid=24324509">Citation</a><a class="dblinks" href="/pmc/articles/PMC3845449/" ref="ordinalpos=2988&amp;ncbi_uid=5581241&amp;link_uid=PMC3845449">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2989.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3777479/figure/cjc-31-04-178-g002/" ref="ordinalpos=2989&amp;ncbi_uid=5590735&amp;link_uid=PMC3777479" href="/pmc/articles/PMC3777479/"><img src="/pmc/articles/instance/3777479/bin/cjc-31-04-178-g002.gif" src-large="/pmc/articles/instance/3777479/bin/cjc-31-04-178-g002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3777479/" ref="ordinalpos=2989&amp;ncbi_uid=5590735&amp;link_uid=PMC3777479" image-link="/pmc/articles/PMC3777479/figure/cjc-31-04-178-g002/" class="imagepopup">Figure 2.Niclosamide targets the Wnt/β-catenin, Notch, and mTORC1 pathways..  From: Niclosamide, an old antihelminthic agent, demonstrates antitumor activity by blocking multiple <span class="highlight" style="background-color:">signaling</span> pathways of cancer stem cells. </a></p><div class="supp"><p class="details"><div>Niclosamide down-regulates the expression of Dishevelled-2 and inhibits the stabilization of β-catenin. Niclosamide also inhibits Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and mTORC1. LRP, low-density lipoprotein-related protein receptor; IRS1, insulin receptor substrate 1; RTK, receptor tyrosine kinase.</div></p></div><div class="aux"><div class="resc">Jing-Xuan Pan, et al. Chin J Cancer. 2012 April;31(4):178-184.</div><p class="links"><a class="dblinks" href="/pubmed/22237038" ref="ordinalpos=2989&amp;ncbi_uid=5590735&amp;link_uid=22237038">Citation</a><a class="dblinks" href="/pmc/articles/PMC3777479/" ref="ordinalpos=2989&amp;ncbi_uid=5590735&amp;link_uid=PMC3777479">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2990.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC490034/figure/fig7/" ref="ordinalpos=2990&amp;ncbi_uid=455882&amp;link_uid=PMC490034" href="/pmc/articles/PMC490034/"><img src="/pmc/articles/instance/490034/bin/tpc1601392f07.gif" src-large="/pmc/articles/instance/490034/bin/tpc1601392f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC490034/" ref="ordinalpos=2990&amp;ncbi_uid=455882&amp;link_uid=PMC490034" image-link="/pmc/articles/PMC490034/figure/fig7/" class="imagepopup">Figure 7. From: The Arabidopsis F-Box Protein SLEEPY1 Targets Gibberellin <span class="highlight" style="background-color:">Signaling</span> Repressors for Gibberellin-Induced Degradation. </a></p><div class="supp"><p class="details"><div>An Updated Model of GA <span class="highlight" style="background-color:">Signaling</span> in Plants.<br />GA induced phosphorylation (P) of DELLA proteins via an unidentified kinase and the SCF<sup>SLY1</sup> complex interacts with the GRAS domain of DELLA proteins and targets their polyubiquitination ([Ub]n) and degradation via the ubiquitin-26S proteasome <span class="highlight" style="background-color:">pathway</span>. The novel <i>sly1-d</i> mutation results in an elevated affinity of SCF<sup>sly1-d</sup> with both wild-type and GA-resistant mutant DELLA proteins.</div></p></div><div class="aux"><div class="resc">Alyssa Dill, et al. Plant Cell. 2004 June;16(6):1392-1405.</div><p class="links"><a class="dblinks" href="/pubmed/15155881" ref="ordinalpos=2990&amp;ncbi_uid=455882&amp;link_uid=15155881">Citation</a><a class="dblinks" href="/pmc/articles/PMC490034/" ref="ordinalpos=2990&amp;ncbi_uid=455882&amp;link_uid=PMC490034">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2991.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584589/figure/f2-ijo-40-04-1162/" ref="ordinalpos=2991&amp;ncbi_uid=4686866&amp;link_uid=PMC3584589" href="/pmc/articles/PMC3584589/"><img src="/pmc/articles/instance/3584589/bin/IJO-40-04-1162-g01.gif" src-large="/pmc/articles/instance/3584589/bin/IJO-40-04-1162-g01.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584589/" ref="ordinalpos=2991&amp;ncbi_uid=4686866&amp;link_uid=PMC3584589" image-link="/pmc/articles/PMC3584589/figure/f2-ijo-40-04-1162/" class="imagepopup">Figure 2.  From: MicroRNA-200a suppresses the Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by interacting with ?-catenin. </a></p><div class="supp"><p class="details"><div>Effect of miR-200a on the activity the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) The luciferase reporter assay using TopFlash and FopFlash vectors was utilized to study β-catenin TCF/LEF promoter activity. The TopFlash reporter vector has 6-TCF binding sites, while FopFlash has mutated TCF binding sites. SGC7901 and U251 cells were co-transfected with different expression vectors as indicated. MiR-200a mimic treatment decreased β-catenin TCF/LEF promoter activity. (B) Up-regulation of miR-200a decreased β-catenin and TCF-4 expression. (C) Immunofluorescence assay for β-catenin indicates that the location of β-catenin in cells shifts from nuclear to cytoplasmic when the expression of miR-200a was increased, and that TCF-4 expression decreased in the nucleus compared to control vector.</div></p></div><div class="aux"><div class="resc">JUAN SU, et al. Int J Oncol. 2012 April;40(4):1162-1170.</div><p class="links"><a class="dblinks" href="/pubmed/22211245" ref="ordinalpos=2991&amp;ncbi_uid=4686866&amp;link_uid=22211245">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584589/" ref="ordinalpos=2991&amp;ncbi_uid=4686866&amp;link_uid=PMC3584589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2992.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1409799/figure/F4/" ref="ordinalpos=2992&amp;ncbi_uid=702839&amp;link_uid=PMC1409799" href="/pmc/articles/PMC1409799/"><img src="/pmc/articles/instance/1409799/bin/1471-2105-7-55-4.gif" src-large="/pmc/articles/instance/1409799/bin/1471-2105-7-55-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1409799/" ref="ordinalpos=2992&amp;ncbi_uid=702839&amp;link_uid=PMC1409799" image-link="/pmc/articles/PMC1409799/figure/F4/" class="imagepopup">Figure 4.  From: PathSys: integrating molecular interaction graphs for systems biology. </a></p><div class="supp"><p class="details"><div><b>MAPK <span class="highlight" style="background-color:">signaling</span> pathways produced by PathSys in comparison with the canonical KEGG MAPK <span class="highlight" style="background-color:">signaling</span> pathways. </b>a). MAPK <span class="highlight" style="background-color:">signaling</span> pathways in Yeast reproduced by KEGG.b). Pathways generated by PathSys for (b1) Pheromone response, (b2) Cell Wall modeling and (b3) Filamentation of MAPK <span class="highlight" style="background-color:">signaling</span> pathways, with the proteins as red ovals, complexes as blue ovals; processes (binary and multiple) and interaction types as small colored circles, squares, diamonds, <i>etc</i>.; pathways (Cell Cycle <span class="highlight" style="background-color:">pathway</span>) as yellow triangles, compartments as grey boxes. Network graphs are produced by <b>BiologicalNetworks </b>client software.</div></p></div><div class="aux"><div class="resc">Michael Baitaluk, et al. BMC Bioinformatics. 2006;7:55-55.</div><p class="links"><a class="dblinks" href="/pubmed/16464251" ref="ordinalpos=2992&amp;ncbi_uid=702839&amp;link_uid=16464251">Citation</a><a class="dblinks" href="/pmc/articles/PMC1409799/" ref="ordinalpos=2992&amp;ncbi_uid=702839&amp;link_uid=PMC1409799">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2993.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3694812/figure/pcbi-1003077-g001/" ref="ordinalpos=2993&amp;ncbi_uid=5055765&amp;link_uid=PMC3694812" href="/pmc/articles/PMC3694812/"><img src="/pmc/articles/instance/3694812/bin/pcbi.1003077.g001.gif" src-large="/pmc/articles/instance/3694812/bin/pcbi.1003077.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3694812/" ref="ordinalpos=2993&amp;ncbi_uid=5055765&amp;link_uid=PMC3694812" image-link="/pmc/articles/PMC3694812/figure/pcbi-1003077-g001/" class="imagepopup">Figure 1. Graphical illustration of the filamentous growth <span class="highlight" style="background-color:">pathway</span> in budding yeast from previous studies..  From: Towards Systematic Discovery of <span class="highlight" style="background-color:">Signaling</span> Networks in Budding Yeast Filamentous Growth Stress Response Using Interventional Phosphorylation Data. </a></p><div class="supp"><p class="details"><div>The ellipses are proteins; the rectangles are genes; and the triangles are metabolites. The linkage between shapes: sharp-end arrows indicate stimulation, T-end arrows indicate inhibition, and wavy lines indicate association. The information were extracted from <i>Science <span class="highlight" style="background-color:">Signaling</span></i> Database of Cell <span class="highlight" style="background-color:">Signaling</span> [20] and KEGG database [27]. The white ellipses are five of the eight kinases selected to be mutated in our experiments.</div></p></div><div class="aux"><div class="resc">Yan Zhang, et al. PLoS Comput Biol. 2013 June;9(6):e1003077.</div><p class="links"><a class="dblinks" href="/pubmed/23825934" ref="ordinalpos=2993&amp;ncbi_uid=5055765&amp;link_uid=23825934">Citation</a><a class="dblinks" href="/pmc/articles/PMC3694812/" ref="ordinalpos=2993&amp;ncbi_uid=5055765&amp;link_uid=PMC3694812">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2994.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3464875/figure/F7/" ref="ordinalpos=2994&amp;ncbi_uid=4280465&amp;link_uid=PMC3464875" href="/pmc/articles/PMC3464875/"><img src="/pmc/articles/instance/3464875/bin/1476-4598-11-24-7.gif" src-large="/pmc/articles/instance/3464875/bin/1476-4598-11-24-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3464875/" ref="ordinalpos=2994&amp;ncbi_uid=4280465&amp;link_uid=PMC3464875" image-link="/pmc/articles/PMC3464875/figure/F7/" class="imagepopup">Figure 7.  From: Nicotine, IFN-? and retinoic acid mediated induction of MUC4 in pancreatic cancer requires E2F1 and STAT-1 transcription factors and utilize different <span class="highlight" style="background-color:">signaling</span> cascades. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of <span class="highlight" style="background-color:">signaling</span> involved in MUC4 expression upon Nicotine, IFN-γ and Retinoic acid stimulation.</b> Nicotine-mediated recruitment of E2F1 and STAT1 requires <span class="highlight" style="background-color:">signaling</span> through all the three pathways tested. In contrast, IFN-γ stimulation did not require PI3 kinase/Akt <span class="highlight" style="background-color:">pathway</span> to recruit E2F1 or STAT1, but ERK and Src seemed to contribute. In the case of RA stimulation, Akt and ERK pathways appeared to be important in upregulation of MUC4 expression.</div></p></div><div class="aux"><div class="resc">Sateesh Kunigal, et al. Mol Cancer. 2012;11:24-24.</div><p class="links"><a class="dblinks" href="/pubmed/22537161" ref="ordinalpos=2994&amp;ncbi_uid=4280465&amp;link_uid=22537161">Citation</a><a class="dblinks" href="/pmc/articles/PMC3464875/" ref="ordinalpos=2994&amp;ncbi_uid=4280465&amp;link_uid=PMC3464875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2995.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3539134/figure/F2/" ref="ordinalpos=2995&amp;ncbi_uid=4525739&amp;link_uid=PMC3539134" href="/pmc/articles/PMC3539134/"><img src="/pmc/articles/instance/3539134/bin/fendo-03-00181-g0002.gif" src-large="/pmc/articles/instance/3539134/bin/fendo-03-00181-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3539134/" ref="ordinalpos=2995&amp;ncbi_uid=4525739&amp;link_uid=PMC3539134" image-link="/pmc/articles/PMC3539134/figure/F2/" class="imagepopup">Figure 2.  From: Implication of inflammatory <span class="highlight" style="background-color:">signaling</span> pathways in obesity-induced insulin resistance. </a></p><div class="supp"><p class="details"><div><b>Inhibition of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by SOCS1 and SOCS3.</b> SOCS1 and SOCS3 are induced by cytokines and involved in a negative feed-back loop. SOCS1 and SOCS3 also inhibit insulin <span class="highlight" style="background-color:">signaling</span> by different mechanisms. They interfere with the binding between the insulin receptor and IRS1/2 proteins. SOCS1 also inhibits the tyrosine kinase activity of the insulin receptor. Both SOCS1 and SOCS3 can interact with the tyrosine-phosphorylated IRS proteins leading to their degradation by the proteasome. The resulting effect is a decrease in the insulin-induced activation of the IRS1/2-PI3K-PKB axis leading to a reduction in the metabolic effects of insulin.</div></p></div><div class="aux"><div class="resc">Jean-François Tanti, et al. Front Endocrinol (Lausanne). 2012;3:181.</div><p class="links"><a class="dblinks" href="/pubmed/23316186" ref="ordinalpos=2995&amp;ncbi_uid=4525739&amp;link_uid=23316186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3539134/" ref="ordinalpos=2995&amp;ncbi_uid=4525739&amp;link_uid=PMC3539134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2996.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2935318/figure/F6/" ref="ordinalpos=2996&amp;ncbi_uid=2678806&amp;link_uid=PMC2935318" href="/pmc/articles/PMC2935318/"><img src="/pmc/articles/instance/2935318/bin/nihms231869f6.gif" src-large="/pmc/articles/instance/2935318/bin/nihms231869f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2935318/" ref="ordinalpos=2996&amp;ncbi_uid=2678806&amp;link_uid=PMC2935318" image-link="/pmc/articles/PMC2935318/figure/F6/" class="imagepopup">Figure 6. <span class="highlight" style="background-color:">Pathway</span> Schematic.  From: Rho Kinase-Dependent Sox9 Activation in Chondrocytes. </a></p><div class="supp"><p class="details"><div>The established <span class="highlight" style="background-color:">pathway</span> of Rho <span class="highlight" style="background-color:">signaling</span> is through ROCK and LIM Kinase to inactivate cofilin, an actin-severing protein. This results in an overall increase in F-actin, which through a series of unknown events leads to changes in matrix gene expression. Based on the observations in this manuscript we present a revised model in which ROCK directly phosphorylates Sox9<sup>Ser181</sup>, causing increased nuclear localization and enhanced matrix gene expression. Thus, while actin remodeling also occurs upon stimulation of the revised <span class="highlight" style="background-color:">pathway</span>, this is a parallel event to the direct activation of Sox9 by ROCK.</div></p></div><div class="aux"><div class="resc">Dominik R. Haudenschild, et al. Arthritis Rheum. ;62(1):191-200.</div><p class="links"><a class="dblinks" href="/pubmed/20039424" ref="ordinalpos=2996&amp;ncbi_uid=2678806&amp;link_uid=20039424">Citation</a><a class="dblinks" href="/pmc/articles/PMC2935318/" ref="ordinalpos=2996&amp;ncbi_uid=2678806&amp;link_uid=PMC2935318">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2997.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC27208/figure/F3/" ref="ordinalpos=2997&amp;ncbi_uid=17718&amp;link_uid=PMC27208" href="/pmc/articles/PMC27208/"><img src="/pmc/articles/instance/27208/bin/pq2302842003.gif" src-large="/pmc/articles/instance/27208/bin/pq2302842003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC27208/" ref="ordinalpos=2997&amp;ncbi_uid=17718&amp;link_uid=PMC27208" image-link="/pmc/articles/PMC27208/figure/F3/" class="imagepopup">Figure 3.  From: ROP-1, an RNA quality-control <span class="highlight" style="background-color:">pathway</span> component, affects  Caenorhabditis elegans dauer formation. </a></p><div class="supp"><p class="details"><div>Proposed model for the interaction of ROP-1 with components of the  dauer formation <span class="highlight" style="background-color:">pathway</span>. ROP-1 modulates the activity of components in  both the <i>daf-2</i> and <i>daf-7</i> pathways to  influence dauer formation <span class="highlight" style="background-color:">signaling</span>. Favorable growth conditions allow  a signal from <i>daf-2</i> (and maybe also  <i>daf-7</i>) to activate an aspartic proteinase. This  proteinase in turn catalyses the processing of ROP-1, which relieves  its influence on both <span class="highlight" style="background-color:">signaling</span> pathways. This processing allows ROP-1  to perform another set of activities, possibly also related to RNA  degradation as part of its RNA quality control function(s).</div></p></div><div class="aux"><div class="resc">Jean-Claude Labbé, et al. Proc Natl Acad Sci U S A. 2000 November 21;97(24):13233-13238.</div><p class="links"><a class="dblinks" href="/pubmed/11069285" ref="ordinalpos=2997&amp;ncbi_uid=17718&amp;link_uid=11069285">Citation</a><a class="dblinks" href="/pmc/articles/PMC27208/" ref="ordinalpos=2997&amp;ncbi_uid=17718&amp;link_uid=PMC27208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2998.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3427262/figure/F1/" ref="ordinalpos=2998&amp;ncbi_uid=4941794&amp;link_uid=PMC3427262" href="/pmc/articles/PMC3427262/"><img src="/pmc/articles/instance/3427262/bin/auto-8-970-g1.gif" src-large="/pmc/articles/instance/3427262/bin/auto-8-970-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3427262/" ref="ordinalpos=2998&amp;ncbi_uid=4941794&amp;link_uid=PMC3427262" image-link="/pmc/articles/PMC3427262/figure/F1/" class="imagepopup">Figure. From: Crosstalk between the UPR and autophagy <span class="highlight" style="background-color:">pathway</span> contributes to handling cellular stress in neurodegenerative disease. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> <span class="highlight" style="background-color:">Signaling</span> crosstalk between the UPR and autophagy in the nervous system. mHTT pathogenesis triggers ER stress, engaging UPR stress sensors. Activated ERN1/IRE1 induces the TRAF2-MAPK8 <span class="highlight" style="background-color:">pathway</span>, inducing autophagy independent of XBP1. PERK activation and phosphorylation of EIF2S1/eIF2α, leads to upregulation of ATG12 possibly through the transcriptional activity of ATF4. XBP1 negatively regulates the levels of FOXO1, inducing its degradation by the proteasome. Then, inactivation of XBP1 protects against HD due to increased accumulation of FOXO1 and the upregulation of its target genes, which include many autophagy regulators. <span class="highlight" style="background-color:">Signaling</span> events related with aging (i.e., insulin and IGF1 <span class="highlight" style="background-color:">signaling</span>) may intersect in FOXO1 and XBP1 to modulate autophagy levels in HD.</div></p></div><div class="aux"><div class="resc">René L. Vidal, et al. Autophagy. 2012 June 1;8(6):970-972.</div><p class="links"><a class="dblinks" href="/pubmed/22617512" ref="ordinalpos=2998&amp;ncbi_uid=4941794&amp;link_uid=22617512">Citation</a><a class="dblinks" href="/pmc/articles/PMC3427262/" ref="ordinalpos=2998&amp;ncbi_uid=4941794&amp;link_uid=PMC3427262">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2999.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC120796/figure/f1/" ref="ordinalpos=2999&amp;ncbi_uid=333343&amp;link_uid=PMC120796" href="/pmc/articles/PMC120796/"><img src="/pmc/articles/instance/120796/bin/mr0320019001.gif" src-large="/pmc/articles/instance/120796/bin/mr0320019001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC120796/" ref="ordinalpos=2999&amp;ncbi_uid=333343&amp;link_uid=PMC120796" image-link="/pmc/articles/PMC120796/figure/f1/" class="imagepopup">FIG. 1. From: Regulation of Gene Expression by Ambient pH in Filamentous Fungi and Yeasts. </a></p><div class="supp"><p class="details"><div>Formal genetic model of pH regulation in <i>A. nidulans</i>. PacC is synthesized as an inactive form whose activation requires a signal transduced under alkaline pH conditions by the <i>pal</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The active form of PacC is a transcriptional repressor of acid-expressed genes, such as <i>pacA</i> and <i>gabA</i>, and a transcriptional activator of alkaline-expressed genes, such as <i>ipnA, prtA</i>, and <i>palD</i>. Mutations inactivating <i>pacC</i> (<i>pacC</i><sup>−</sup> or <i>pacC</i><sup>+/−</sup>) or the <i>pal</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> lead to absence of expression of alkaline genes and derepression of acidic genes, which results in acidity mimicry. Gain-of-function <i>pacC<sup>c</sup></i> mutations bypassing the <i>pal</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (i.e., leading to active PacC at any ambient pH) result in permanent activation of alkaline genes and superrepression of acidic genes, which leads to alkalinity mimicry.</div></p></div><div class="aux"><div class="resc">Miguel A. Peñalva, et al. Microbiol Mol Biol Rev. 2002 September;66(3):426-446.</div><p class="links"><a class="dblinks" href="/pubmed/12208998" ref="ordinalpos=2999&amp;ncbi_uid=333343&amp;link_uid=12208998">Citation</a><a class="dblinks" href="/pmc/articles/PMC120796/" ref="ordinalpos=2999&amp;ncbi_uid=333343&amp;link_uid=PMC120796">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3000.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC154144/figure/F2/" ref="ordinalpos=3000&amp;ncbi_uid=292927&amp;link_uid=PMC154144" href="/pmc/articles/PMC154144/"><img src="/pmc/articles/instance/154144/bin/bcr566-2.gif" src-large="/pmc/articles/instance/154144/bin/bcr566-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC154144/" ref="ordinalpos=3000&amp;ncbi_uid=292927&amp;link_uid=PMC154144" image-link="/pmc/articles/PMC154144/figure/F2/" class="imagepopup">Figure 2.  From: ?-Catenin: a transforming actor on many stages. </a></p><div class="supp"><p class="details"><div>Canonic Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecules can induce a similar phenotype. When each molecule in the canonic Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was activated, they could induce hyperplasias, adenocarcinomas and squamous metaplasias. GSK3, dominant negative glycogen synthase kinase 3β; APC, adenomatous polyposis coli; P, phosphorylated form; dsh, dishevelled.</div></p></div><div class="aux"><div class="resc">Keiko Miyoshi, et al. Breast Cancer Res. 2003;5(2):63-68.</div><p class="links"><a class="dblinks" href="/pubmed/12631383" ref="ordinalpos=3000&amp;ncbi_uid=292927&amp;link_uid=12631383">Citation</a><a class="dblinks" href="/pmc/articles/PMC154144/" ref="ordinalpos=3000&amp;ncbi_uid=292927&amp;link_uid=PMC154144">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="29" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="30" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="31" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:39:14-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A721B5431133100000000005E0C16;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A721B5431133100000000005E0C16" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>