
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="21" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC37065430F8E1000000000032AEBC" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="20" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="21" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="22" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="21" /></div>
            <div><h2 class="result_count">Results: 2001 to 2100 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2001.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3472468/figure/F4/" ref="ordinalpos=2001&amp;ncbi_uid=5385302&amp;link_uid=PMC3472468" href="/pmc/articles/PMC3472468/"><img src="/pmc/articles/instance/3472468/bin/zh70191237670004.gif" src-large="/pmc/articles/instance/3472468/bin/zh70191237670004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3472468/" ref="ordinalpos=2001&amp;ncbi_uid=5385302&amp;link_uid=PMC3472468" image-link="/pmc/articles/PMC3472468/figure/F4/" class="imagepopup">Fig. 4. From: Genomic analysis of sleep deprivation reveals translational regulation in the hippocampus. </a></p><div class="supp"><p class="details"><div>The effects of sleep deprivation on the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes regulated by sleep deprivation mapped to the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (adapted from KEGG and Wikipathways). Upregulated gene names are written in lower case, in bold and italics, and downregulated gene names are written in bold with underlining. Note that genes from several functional clusters and enriched pathways shown in Fig. 3 and Table 2 are contained within this <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Christopher G. Vecsey, et al. Physiol Genomics. 2012 October 15;44(20):981-991.</div><p class="links"><a class="dblinks" href="/pubmed/22930738" ref="ordinalpos=2001&amp;ncbi_uid=5385302&amp;link_uid=22930738">Citation</a><a class="dblinks" href="/pmc/articles/PMC3472468/" ref="ordinalpos=2001&amp;ncbi_uid=5385302&amp;link_uid=PMC3472468">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2002.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3206446/figure/F6/" ref="ordinalpos=2002&amp;ncbi_uid=3323280&amp;link_uid=PMC3206446" href="/pmc/articles/PMC3206446/"><img src="/pmc/articles/instance/3206446/bin/1471-213X-11-57-6.gif" src-large="/pmc/articles/instance/3206446/bin/1471-213X-11-57-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3206446/" ref="ordinalpos=2002&amp;ncbi_uid=3323280&amp;link_uid=PMC3206446" image-link="/pmc/articles/PMC3206446/figure/F6/" class="imagepopup">Figure 6.  From: Loss of the Drosophila cell polarity regulator Scribbled promotes epithelial tissue overgrowth and cooperation with oncogenic Ras-Raf through impaired Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>aPKC <span class="highlight" style="background-color:">signaling</span> is required for the impaired Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in <i>scrib </i>mutants</b>. Larval eye/antennal discs with <i>ey-FLP </i>induced mutant clones marked by GFP expression (green, or magenta in the merges). Grayscale is β-GAL. A white bar indicates the location of the MF. (A-D) Coexpression of <i>aPKC<sup>CAAXDN </sup></i>and <i>bsk<sup>DN </sup></i>in clones does not alter <i>ex-lacZ </i>(A) or <i>fj-lacZ </i>(C) expression, but when <i>aPKC<sup>CAAXDN </sup></i>and <i>bsk<sup>DN </sup></i>are coexpressed in <i>scrib<sup>1 </sup></i>clones, the normal pattern of <i>ex-lacZ </i>(B) and <i>fj-lacZ </i>(D) expression is restored (compared with <i>scrib<sup>1 </sup></i>clones expressing <i>bsk<sup>DN </sup></i>alone in Figure 1).</div></p></div><div class="aux"><div class="resc">Karen Doggett, et al. BMC Dev Biol. 2011;11:57-57.</div><p class="links"><a class="dblinks" href="/pubmed/21955824" ref="ordinalpos=2002&amp;ncbi_uid=3323280&amp;link_uid=21955824">Citation</a><a class="dblinks" href="/pmc/articles/PMC3206446/" ref="ordinalpos=2002&amp;ncbi_uid=3323280&amp;link_uid=PMC3206446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2003.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC85884/figure/F13/" ref="ordinalpos=2003&amp;ncbi_uid=19405&amp;link_uid=PMC85884" href="/pmc/articles/PMC85884/"><img src="/pmc/articles/instance/85884/bin/mb1301793013.gif" src-large="/pmc/articles/instance/85884/bin/mb1301793013.jpg" alt="FIG. 13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC85884/" ref="ordinalpos=2003&amp;ncbi_uid=19405&amp;link_uid=PMC85884" image-link="/pmc/articles/PMC85884/figure/F13/" class="imagepopup">FIG. 13.  From: Distinct Pathways of Cell Migration and Antiapoptotic Response  to Epithelial Injury: Structure-Function Analysis of  Human Intestinal Trefoil Factor. </a></p><div class="supp"><p class="details"><div>Hypothetical pathways involved in ITF <span class="highlight" style="background-color:">signaling</span>. According to this model, ITF <span class="highlight" style="background-color:">signaling</span> utilizes MEK-ERK <span class="highlight" style="background-color:">pathway</span> 1 for cell migration, which does not require intact trefoil peptide. In contrast, for antiapoptotic effect, ITF requires intact dimerized peptide in the activation of both PI3K (<span class="highlight" style="background-color:">pathway</span> 2) and EGF-R (<span class="highlight" style="background-color:">pathway</span> 3) signal pathways.</div></p></div><div class="aux"><div class="resc">Koichi Kinoshita, et al. Mol Cell Biol. 2000 July;20(13):4680-4690.</div><p class="links"><a class="dblinks" href="/pubmed/10848594" ref="ordinalpos=2003&amp;ncbi_uid=19405&amp;link_uid=10848594">Citation</a><a class="dblinks" href="/pmc/articles/PMC85884/" ref="ordinalpos=2003&amp;ncbi_uid=19405&amp;link_uid=PMC85884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2004.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4209605/figure/F1/" ref="ordinalpos=2004&amp;ncbi_uid=6764950&amp;link_uid=PMC4209605" href="/pmc/articles/PMC4209605/"><img src="/pmc/articles/instance/4209605/bin/ganc-05-303-g001.gif" src-large="/pmc/articles/instance/4209605/bin/ganc-05-303-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4209605/" ref="ordinalpos=2004&amp;ncbi_uid=6764950&amp;link_uid=PMC4209605" image-link="/pmc/articles/PMC4209605/figure/F1/" class="imagepopup">Figure 1. TMEPAI modulates TGF-β <span class="highlight" style="background-color:">signaling</span>.  From: Cutting the brakes and flooring the gas: how TMEPAI turns TGF-? into a tumor promoter. </a></p><div class="supp"><p class="details"><div>Exposure of cells to active TGF-β leads to assembly of the tetrameric receptor complex, composed of two type I and two type II TGF-β receptor subunits (TGFBRI and TGFBRII). TGFBRII then phosphorylates TGFBRI, enabling it to activate downstream <span class="highlight" style="background-color:">signaling</span> pathways. In the canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (left), TGFBRI phosphorylates receptor SMADs (SMAD2/3), which can then associate with SMAD4 and become translocated to the nucleus. The SMAD <span class="highlight" style="background-color:">signaling</span> complex can inhibit cell proliferation by inhibiting expression of <i>MYC</i> and increasing expression of <i>p21CIP1</i> and <i>p15INK4b</i>. In the noncanonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (right), the activated TGF-β receptor complex regulates SMAD-independent pathways, including activation of Rho family members, and can stimulate cellular proliferation through activation of MAPK and PI3K/Akt. In triple negative breast cancer cell lines, expression of TMEPAI blocks canonical <span class="highlight" style="background-color:">signaling</span> through sequestration of SMAD4 and potentiates the noncanonical activation of PI3K/Akt by downregulating the PI3K inhibitor PTEN.</div></p></div><div class="aux"><div class="resc">Magdalena A. Cichon, et al. Genes Cancer. 2014 September;5(9-10):303-305.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2004&amp;ncbi_uid=6764950&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4209605/" ref="ordinalpos=2004&amp;ncbi_uid=6764950&amp;link_uid=PMC4209605">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2005.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3047110/figure/F6/" ref="ordinalpos=2005&amp;ncbi_uid=2875168&amp;link_uid=PMC3047110" href="/pmc/articles/PMC3047110/"><img src="/pmc/articles/instance/3047110/bin/cbt1104_0410_fig006.gif" src-large="/pmc/articles/instance/3047110/bin/cbt1104_0410_fig006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3047110/" ref="ordinalpos=2005&amp;ncbi_uid=2875168&amp;link_uid=PMC3047110" image-link="/pmc/articles/PMC3047110/figure/F6/" class="imagepopup">Figure 6.  From: ERG oncogene modulates prostaglandin <span class="highlight" style="background-color:">signaling</span> in prostate cancer cells. </a></p><div class="supp"><p class="details"><div>Proposed model for ERG functions in prostaglandin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Inhibition of HPGD as result of TMPRSS2-ERG overexpression prevents PGE<sub>2</sub> catabolism, thus accumulation of PGE<sub>2</sub> will result in uPA activation and cell growth, contributing to the progression of CaP. ERG directly binds to the promoter of uPA.</div></p></div><div class="aux"><div class="resc">Ahmed A Mohamed, et al. Cancer Biol Ther. 2011 February 15;11(4):410-417.</div><p class="links"><a class="dblinks" href="/pubmed/21178489" ref="ordinalpos=2005&amp;ncbi_uid=2875168&amp;link_uid=21178489">Citation</a><a class="dblinks" href="/pmc/articles/PMC3047110/" ref="ordinalpos=2005&amp;ncbi_uid=2875168&amp;link_uid=PMC3047110">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2006.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2735864/figure/F1/" ref="ordinalpos=2006&amp;ncbi_uid=2037714&amp;link_uid=PMC2735864" href="/pmc/articles/PMC2735864/"><img src="/pmc/articles/instance/2735864/bin/nihms-116298-f0001.gif" src-large="/pmc/articles/instance/2735864/bin/nihms-116298-f0001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2735864/" ref="ordinalpos=2006&amp;ncbi_uid=2037714&amp;link_uid=PMC2735864" image-link="/pmc/articles/PMC2735864/figure/F1/" class="imagepopup">FIGURE 1. TNF <span class="highlight" style="background-color:">signaling</span> through the NF-kB canonical <span class="highlight" style="background-color:">pathway</span>.  From: Genetic Variation in TNF and the NF-?B Canonical <span class="highlight" style="background-color:">Pathway</span> and Risk of Non-Hodgkin Lymphoma. </a></p><div class="supp"><p class="details"><div>Binding of TNF-α (TNFSF1A) to TNF-R1 (TNFRSF1) at the cell surface results in receptor oligomerization and recruitment of <span class="highlight" style="background-color:">signaling</span> intermediates TRADD, TRAF2, TRAF5, and RIPK1, leading to the activation of the IκB kinase (IKK) complex. IKK is composed of two catalytic kinase subunits (IKKα and IKKβ) and one regulatory scaffold subunit (IKKγ or NEMO); IKKβ is the main kinase of the canonical <span class="highlight" style="background-color:">pathway</span>. IKK phosphorylates and promotes degradation of IκB, resulting in the release of NF-κB dimers, which can then translocate to the cell nucleus and bind to specific sequences in the promoter or enhancer regions of a large number of target inflammatory and other genes related to cellular growth, differentiation and apoptosis. Key canonical <span class="highlight" style="background-color:">pathway</span> NF-κB dimers include NF-κB1 (p50), RelA (p65), and c-Rel.</div></p></div><div class="aux"><div class="resc">James R Cerhan, et al. Cancer Epidemiol Biomarkers Prev. ;17(11):3161-3169.</div><p class="links"><a class="dblinks" href="/pubmed/18990758" ref="ordinalpos=2006&amp;ncbi_uid=2037714&amp;link_uid=18990758">Citation</a><a class="dblinks" href="/pmc/articles/PMC2735864/" ref="ordinalpos=2006&amp;ncbi_uid=2037714&amp;link_uid=PMC2735864">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2007.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2213494/figure/F8/" ref="ordinalpos=2007&amp;ncbi_uid=1144160&amp;link_uid=PMC2213494" href="/pmc/articles/PMC2213494/"><img src="/pmc/articles/instance/2213494/bin/JEM001349.f8.gif" src-large="/pmc/articles/instance/2213494/bin/JEM001349.f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2213494/" ref="ordinalpos=2007&amp;ncbi_uid=1144160&amp;link_uid=PMC2213494" image-link="/pmc/articles/PMC2213494/figure/F8/" class="imagepopup">Figure 8.  From: Hematopoietic Stem Cells Need Two Signals to Prevent Apoptosis; Bcl-2 Can Provide One of These, Kitl/C-KIT <span class="highlight" style="background-color:">Signaling</span> the Other. </a></p><div class="supp"><p class="details"><div>Model for apoptosis <span class="highlight" style="background-color:">signaling</span> in HSCs. The middle two arrows in each panel represent the two active apoptosis pathways that need to be blocked. It is currently unclear whether these represent two completely separate pathways, or two actions within one <span class="highlight" style="background-color:">pathway</span>. White arrows and text show the inactive <span class="highlight" style="background-color:">pathway</span>. Black arrows and text show the active <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Bcl-2 is meant to represent antiapoptosis Bcl-2 family members. The top panel shows Kitl <span class="highlight" style="background-color:">signaling</span> in the wild-type HSCs. Although the proliferative signal from Kitl is conveyed, apoptosis is blocked only partially at best. In the middle panel, transgene-derived BCL-2, in combination with Kitl/c-Kit–transmitted signals, blocks apoptosis, allowing the Kitl-conveyed proliferation, and possibly differentiation-inducing activities to be seen. In the bottom panel, multifactor stimulation also completely blocks apoptosis and results in specific proliferation/differentiation, depending on the factor combination used.</div></p></div><div class="aux"><div class="resc">Jos Domen, et al. J Exp Med. 2000 December 18;192(12):1707-1718.</div><p class="links"><a class="dblinks" href="/pubmed/11120768" ref="ordinalpos=2007&amp;ncbi_uid=1144160&amp;link_uid=11120768">Citation</a><a class="dblinks" href="/pmc/articles/PMC2213494/" ref="ordinalpos=2007&amp;ncbi_uid=1144160&amp;link_uid=PMC2213494">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2008.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4200133/figure/Fig2/" ref="ordinalpos=2008&amp;ncbi_uid=6731987&amp;link_uid=PMC4200133" href="/pmc/articles/PMC4200133/"><img src="/pmc/articles/instance/4200133/bin/12865_2014_46_Fig2_HTML.gif" src-large="/pmc/articles/instance/4200133/bin/12865_2014_46_Fig2_HTML.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4200133/" ref="ordinalpos=2008&amp;ncbi_uid=6731987&amp;link_uid=PMC4200133" image-link="/pmc/articles/PMC4200133/figure/Fig2/" class="imagepopup">Figure 2.  From: C-Jun N-terminal kinase (JNK) isoforms play differing roles in otitis media. </a></p><div class="supp"><p class="details"><div> <b>Regulation of JNK <span class="highlight" style="background-color:">signaling</span> genes during OM.</b> ME expression of genes related to the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in WT mice during OM, assessed by DNA microarray. Genes invoved in growth factor JNK <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">William Yao, et al. BMC Immunol. 2014;15(1):46.</div><p class="links"><a class="dblinks" href="/pubmed/25311344" ref="ordinalpos=2008&amp;ncbi_uid=6731987&amp;link_uid=25311344">Citation</a><a class="dblinks" href="/pmc/articles/PMC4200133/" ref="ordinalpos=2008&amp;ncbi_uid=6731987&amp;link_uid=PMC4200133">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2009.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3146006/figure/F4/" ref="ordinalpos=2009&amp;ncbi_uid=3147638&amp;link_uid=PMC3146006" href="/pmc/articles/PMC3146006/"><img src="/pmc/articles/instance/3146006/bin/lar-27-117-g004.gif" src-large="/pmc/articles/instance/3146006/bin/lar-27-117-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3146006/" ref="ordinalpos=2009&amp;ncbi_uid=3147638&amp;link_uid=PMC3146006" image-link="/pmc/articles/PMC3146006/figure/F4/" class="imagepopup">Figure 4.  From: Effects of Steaming Time and Frequency for Manufactured Red Liriope platyphylla on the Insulin Secretion Ability and Insulin Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effects of the different steaming frequencies of <i>Liriope platyphylla</i> on down-stream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of insulin receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via Western blotting. Total cell lysates were prepared from INS cells treated with vehicle, KWG, KRG and six different LPs prepared under different steaming conditions, as described in the Materials and Methods sections. Fifty micrograms of protein per sample were immunoblotted with antibodies for each protein. Three samples were assayed in triplicate via Western blotting. The values are expressed as the means±SD. <sup>*</sup><i>P</i>&lt;0.05 is the significance level compared to the vehicle-treated group.</div></p></div><div class="aux"><div class="resc">Sun Il Choi, et al. Lab Anim Res. 2011 June;27(2):117-126.</div><p class="links"><a class="dblinks" href="/pubmed/21826171" ref="ordinalpos=2009&amp;ncbi_uid=3147638&amp;link_uid=21826171">Citation</a><a class="dblinks" href="/pmc/articles/PMC3146006/" ref="ordinalpos=2009&amp;ncbi_uid=3147638&amp;link_uid=PMC3146006">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2010.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1855985/figure/pone-0000426-g001/" ref="ordinalpos=2010&amp;ncbi_uid=971899&amp;link_uid=PMC1855985" href="/pmc/articles/PMC1855985/"><img src="/pmc/articles/instance/1855985/bin/pone.0000426.g001.gif" src-large="/pmc/articles/instance/1855985/bin/pone.0000426.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1855985/" ref="ordinalpos=2010&amp;ncbi_uid=971899&amp;link_uid=PMC1855985" image-link="/pmc/articles/PMC1855985/figure/pone-0000426-g001/" class="imagepopup">Figure 1. Genes sequenced in this study..  From: Mutational Analysis of EGFR and Related <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Genes in Lung Adenocarcinomas Identifies a Novel Somatic Kinase Domain Mutation in FGFR4. </a></p><div class="supp"><p class="details"><div>The schematic diagram depicts the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes listed in red were sequenced only in genomic DNAs from 217 tumors (“Group 1”). Genes listed in blue were sequenced only in WGA-treated DNA tumor samples (“Group 2”). Genes in black were sequenced in both groups. Gene nomenclature is as reported in GenBank as of December 2006. See Supplemental Table S3 for clinical characteristics of all tumors sequenced.</div></p></div><div class="aux"><div class="resc">Jenifer L. Marks, et al. PLoS ONE. 2007;2(5):e426.</div><p class="links"><a class="dblinks" href="/pubmed/17487277" ref="ordinalpos=2010&amp;ncbi_uid=971899&amp;link_uid=17487277">Citation</a><a class="dblinks" href="/pmc/articles/PMC1855985/" ref="ordinalpos=2010&amp;ncbi_uid=971899&amp;link_uid=PMC1855985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2011.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3705595/figure/fig7/" ref="ordinalpos=2011&amp;ncbi_uid=6444256&amp;link_uid=PMC3705595" href="/pmc/articles/PMC3705595/"><img src="/pmc/articles/instance/3705595/bin/cdd201331f7.gif" src-large="/pmc/articles/instance/3705595/bin/cdd201331f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3705595/" ref="ordinalpos=2011&amp;ncbi_uid=6444256&amp;link_uid=PMC3705595" image-link="/pmc/articles/PMC3705595/figure/fig7/" class="imagepopup">Figure 7.  From: Rescue of platinum-damaged oocytes from programmed cell death through inactivation of the p53 family <span class="highlight" style="background-color:">signaling</span> network. </a></p><div class="supp"><p class="details"><div>Model for oocyte death regulated by p53 family members. All three p53 family members are involved in the apoptotic <span class="highlight" style="background-color:">pathway</span> in primordial follicle oocytes, with TAp63 playing a central role. DNA damage by <i>γ</i>-radiation activates <span class="highlight" style="background-color:">signaling</span> pathways through TAp63, resulting in the induction of <i>Puma</i> and <i>Noxa</i> and subsequent binding of PUMA and NOXA proteins to BAX/BAK as described in the recent report by Kerr <i>et al.</i> (red lines).<sup>23</sup> DNA damage by cisplatin also activates TAp63, which results in upregulation of c-Abl. The kinase activity of c-Abl (green), which can be inhibited by imatinib, induces TAp63-dependent expression of TAp73 and subsequent degradation of TAp63. c-Abl may also activate TAp73 via phosphorylation; phospho-TAp73 then activates the apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. p53 may have a minor or non-essential role in the activation of the apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in response to cisplatin (shown as dotted line). The sequence of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the consequences on primordial oocyte survival is profound and provides new insights into targets that may mitigate fertility loss in the cancer setting</div></p></div><div class="aux"><div class="resc">S-Y Kim, et al. Cell Death Differ. 2013 August;20(8):987-997.</div><p class="links"><a class="dblinks" href="/pubmed/23598363" ref="ordinalpos=2011&amp;ncbi_uid=6444256&amp;link_uid=23598363">Citation</a><a class="dblinks" href="/pmc/articles/PMC3705595/" ref="ordinalpos=2011&amp;ncbi_uid=6444256&amp;link_uid=PMC3705595">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2012.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2937064/figure/F3/" ref="ordinalpos=2012&amp;ncbi_uid=3131654&amp;link_uid=PMC2937064" href="/pmc/articles/PMC2937064/"><img src="/pmc/articles/instance/2937064/bin/nihms218132f3.gif" src-large="/pmc/articles/instance/2937064/bin/nihms218132f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2937064/" ref="ordinalpos=2012&amp;ncbi_uid=3131654&amp;link_uid=PMC2937064" image-link="/pmc/articles/PMC2937064/figure/F3/" class="imagepopup">Figure 3.  From: Metabolic Syndrome, Hormones, and Maintenance of T Cells during Aging. </a></p><div class="supp"><p class="details"><div>The CD28 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in T cells is equivalent to the IGF-1 <span class="highlight" style="background-color:">pathway</span> in the whole organism. Stimulation of the CD28/PI-3K/AKT <span class="highlight" style="background-color:">pathway</span> and thereby inhibition of the FOXO <span class="highlight" style="background-color:">signaling</span> axis is a unique mechanism developed to separate T cells from the major glucose metabolic system. This would confer a great evolutionary advantage on the organism, because the signal that triggers T-cell co-stimulation by CD28 also triggers the energy utilization of T cells. However, life-long over-utilization of this <span class="highlight" style="background-color:">pathway</span> is associated with acceleration of T-cell senescence, including loss of CD28 expression and development of AICD-resistant T cells.</div></p></div><div class="aux"><div class="resc">Hui-Chen Hsu, et al. Curr Opin Immunol. ;22(4):541-548.</div><p class="links"><a class="dblinks" href="/pubmed/20591642" ref="ordinalpos=2012&amp;ncbi_uid=3131654&amp;link_uid=20591642">Citation</a><a class="dblinks" href="/pmc/articles/PMC2937064/" ref="ordinalpos=2012&amp;ncbi_uid=3131654&amp;link_uid=PMC2937064">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2013.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3241422/figure/fig7/" ref="ordinalpos=2013&amp;ncbi_uid=4404080&amp;link_uid=PMC3241422" href="/pmc/articles/PMC3241422/"><img src="/pmc/articles/instance/3241422/bin/1145fig7.gif" src-large="/pmc/articles/instance/3241422/bin/1145fig7.jpg" alt="Figure 7 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3241422/" ref="ordinalpos=2013&amp;ncbi_uid=4404080&amp;link_uid=PMC3241422" image-link="/pmc/articles/PMC3241422/figure/fig7/" class="imagepopup">Figure 7 .  From: Regulation of Cell Wall Biogenesis in Saccharomyces cerevisiae: The Cell Wall Integrity <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The CWI <span class="highlight" style="background-color:">pathway</span> transcriptional program. The majority of genes regulated by CWI <span class="highlight" style="background-color:">signaling</span> are under the control of the Rlm1 transcription factor, which is phosphorylated and activated by Mpk1. Among these genes is <i>MLP1</i>, which encodes a pseudokinase paralog of Mpk1. Using a mechanism that is independent of protein kinase catalytic activity, Mpk1, together with Mlp1, drive expression of a subset of cell wall stress-induced genes through the Swi4/Swi6 transcription factor (including <i>FKS2</i>).</div></p></div><div class="aux"><div class="resc">David E. Levin. Genetics. 2011 December;189(4):1145-1175.</div><p class="links"><a class="dblinks" href="/pubmed/22174182" ref="ordinalpos=2013&amp;ncbi_uid=4404080&amp;link_uid=22174182">Citation</a><a class="dblinks" href="/pmc/articles/PMC3241422/" ref="ordinalpos=2013&amp;ncbi_uid=4404080&amp;link_uid=PMC3241422">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2014.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3068308/figure/fig5/" ref="ordinalpos=2014&amp;ncbi_uid=2899237&amp;link_uid=PMC3068308" href="/pmc/articles/PMC3068308/"><img src="/pmc/articles/instance/3068308/bin/PRI2011-620601.005.gif" src-large="/pmc/articles/instance/3068308/bin/PRI2011-620601.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3068308/" ref="ordinalpos=2014&amp;ncbi_uid=2899237&amp;link_uid=PMC3068308" image-link="/pmc/articles/PMC3068308/figure/fig5/" class="imagepopup">Figure 5.  From: Molecular Characteristics of Pancreatic Ductal Adenocarcinoma. </a></p><div class="supp"><p class="details"><div>The TGF-<i>β</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The TGF-<i>β</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by binding of TGF-<i>β</i> to a type II receptor, which facilitates the recruitment and phosphorylation of the type I receptor. This pTGF-<i>β</i>RI activates either SMAD2 or SMAD3 by phosphorylation. The phosphorylated SMAD2/3 forms a complex with SMAD4 and transports to the nucleus where it influences SMAD target gene transcription. SMAD7 can inhibit TGF-<i>β</i> <span class="highlight" style="background-color:">signaling</span> through inhibition of TGF-<i>β</i>RI phosphorylation.</div></p></div><div class="aux"><div class="resc">Niki A. Ottenhof, et al. Patholog Res Int. 2011;2011:620601.</div><p class="links"><a class="dblinks" href="/pubmed/21512581" ref="ordinalpos=2014&amp;ncbi_uid=2899237&amp;link_uid=21512581">Citation</a><a class="dblinks" href="/pmc/articles/PMC3068308/" ref="ordinalpos=2014&amp;ncbi_uid=2899237&amp;link_uid=PMC3068308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2015.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3351456/figure/pone-0036911-g006/" ref="ordinalpos=2015&amp;ncbi_uid=4008513&amp;link_uid=PMC3351456" href="/pmc/articles/PMC3351456/"><img src="/pmc/articles/instance/3351456/bin/pone.0036911.g006.gif" src-large="/pmc/articles/instance/3351456/bin/pone.0036911.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3351456/" ref="ordinalpos=2015&amp;ncbi_uid=4008513&amp;link_uid=PMC3351456" image-link="/pmc/articles/PMC3351456/figure/pone-0036911-g006/" class="imagepopup">Figure 6. Dose-dependent distribution of MeCh-stimulated <span class="highlight" style="background-color:">signaling</span> activity in diverse cellular contexts..  From: VENNTURE-A Novel Venn Diagram Investigational Tool for Multiple Pharmacological Dataset Analysis. </a></p><div class="supp"><p class="details"><div>Proportional diagrams representing the relative distribution of phosphoproteins (Proteins)/GO term groups (GO term)/<span class="highlight" style="background-color:">signaling</span> pathways (<span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>) unique to an individual stimulating MeCh dose (color coded) in either cellular context (control or CMP).</div></p></div><div class="aux"><div class="resc">Bronwen Martin, et al. PLoS One. 2012;7(5):e36911.</div><p class="links"><a class="dblinks" href="/pubmed/22606307" ref="ordinalpos=2015&amp;ncbi_uid=4008513&amp;link_uid=22606307">Citation</a><a class="dblinks" href="/pmc/articles/PMC3351456/" ref="ordinalpos=2015&amp;ncbi_uid=4008513&amp;link_uid=PMC3351456">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2016.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3338759/figure/pone-0035902-g003/" ref="ordinalpos=2016&amp;ncbi_uid=3978414&amp;link_uid=PMC3338759" href="/pmc/articles/PMC3338759/"><img src="/pmc/articles/instance/3338759/bin/pone.0035902.g003.gif" src-large="/pmc/articles/instance/3338759/bin/pone.0035902.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3338759/" ref="ordinalpos=2016&amp;ncbi_uid=3978414&amp;link_uid=PMC3338759" image-link="/pmc/articles/PMC3338759/figure/pone-0035902-g003/" class="imagepopup">Figure 3. Effect of IFN-λ3 on the key elements in JAK-STAT <span class="highlight" style="background-color:">pathway</span> in uninfected macrophages..  From: IFN-?3 Inhibits HIV Infection of Macrophages through the JAK-STAT <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Seven-day-cultured macrophages were treated with or without IFN-λ3 (100 ng/ml) for 24 h. Total nucleotide were extracted from macrophages and subjected to the real-time reverse transcription polymerase chain reaction (RT-PCR) assay for mRNA expression of TLRs (<b>A</b>), IRFs (<b>B</b>) and the key elements in IFN-λ3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<b>C</b>). The data shown were the mean ± standard deviation of triplicate culture (<i>*P&lt;0.05, **P&lt;0.01</i>).</div></p></div><div class="aux"><div class="resc">Man-Qing Liu, et al. PLoS One. 2012;7(4):e35902.</div><p class="links"><a class="dblinks" href="/pubmed/22558263" ref="ordinalpos=2016&amp;ncbi_uid=3978414&amp;link_uid=22558263">Citation</a><a class="dblinks" href="/pmc/articles/PMC3338759/" ref="ordinalpos=2016&amp;ncbi_uid=3978414&amp;link_uid=PMC3338759">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2017.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3496286/figure/F1/" ref="ordinalpos=2017&amp;ncbi_uid=5026415&amp;link_uid=PMC3496286" href="/pmc/articles/PMC3496286/"><img src="/pmc/articles/instance/3496286/bin/nihms390108f1.gif" src-large="/pmc/articles/instance/3496286/bin/nihms390108f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3496286/" ref="ordinalpos=2017&amp;ncbi_uid=5026415&amp;link_uid=PMC3496286" image-link="/pmc/articles/PMC3496286/figure/F1/" class="imagepopup">Figure 1. Generalized Schematic of Conventional and Biased Cardiac β1-AR <span class="highlight" style="background-color:">Signaling</span>.  From: Conducting the G-protein Coupled Receptor (GPCR) <span class="highlight" style="background-color:">Signaling</span> Symphony in Cardiovascular Diseases: New Therapeutic Approaches. </a></p><div class="supp"><p class="details"><div>Epinephrine and norepinephrine enhance myocardial contractility, heart rate, and apoptosis through Gα<sub>s</sub>-coupled <span class="highlight" style="background-color:">signaling</span> pathways in heart (black arrows). Furthermore, these catecholamines can activate an anti-apoptotic cascade independent of Gα<sub>s</sub> (green arrows). Putative biased ligands induce cardioprotection by exclusively promoting the G protein coupled receptor kinase (GRK) and β-arrestin-mediated <span class="highlight" style="background-color:">pathway</span> (green arrows).</div></p></div><div class="aux"><div class="resc">Stephen L. Belmonte, et al. Drug Discov Today Dis Models. 2012;9(3):e85-e90.</div><p class="links"><a class="dblinks" href="/pubmed/23162605" ref="ordinalpos=2017&amp;ncbi_uid=5026415&amp;link_uid=23162605">Citation</a><a class="dblinks" href="/pmc/articles/PMC3496286/" ref="ordinalpos=2017&amp;ncbi_uid=5026415&amp;link_uid=PMC3496286">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2018.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3181225/figure/pbio-1001160-g004/" ref="ordinalpos=2018&amp;ncbi_uid=3269975&amp;link_uid=PMC3181225" href="/pmc/articles/PMC3181225/"><img src="/pmc/articles/instance/3181225/bin/pbio.1001160.g004.gif" src-large="/pmc/articles/instance/3181225/bin/pbio.1001160.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3181225/" ref="ordinalpos=2018&amp;ncbi_uid=3269975&amp;link_uid=PMC3181225" image-link="/pmc/articles/PMC3181225/figure/pbio-1001160-g004/" class="imagepopup">Figure 4. Effects on DHR4 subcellular localization by manipulating PTTH <span class="highlight" style="background-color:">pathway</span> components..  From: Nuclear Receptor DHR4 Controls the Timing of Steroid Hormone Pulses During Drosophila Development. </a></p><div class="supp"><p class="details"><div>DHR4 antibody stains. <i>phm22&gt;w<sup>1118</sup></i> and <i>ptth&gt;w<sup>1118</sup></i> ring glands serve as controls. The <i>ptth&gt;grim</i> and <i>phm22&gt;torso-</i>RNAi lines disrupt PTTH <span class="highlight" style="background-color:">signaling</span>. <i>phm22&gt;Ras<sup>V12</sup></i> constitutively activates the PTTH <span class="highlight" style="background-color:">pathway</span>. Hours indicate time after the L2/L3 molt. 10–15 ring glands were tested per condition.</div></p></div><div class="aux"><div class="resc">Qiuxiang Ou, et al. PLoS Biol. 2011 September;9(9):e1001160.</div><p class="links"><a class="dblinks" href="/pubmed/21980261" ref="ordinalpos=2018&amp;ncbi_uid=3269975&amp;link_uid=21980261">Citation</a><a class="dblinks" href="/pmc/articles/PMC3181225/" ref="ordinalpos=2018&amp;ncbi_uid=3269975&amp;link_uid=PMC3181225">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2019.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2394659/figure/pone-0002339-g003/" ref="ordinalpos=2019&amp;ncbi_uid=1401650&amp;link_uid=PMC2394659" href="/pmc/articles/PMC2394659/"><img src="/pmc/articles/instance/2394659/bin/pone.0002339.g003.gif" src-large="/pmc/articles/instance/2394659/bin/pone.0002339.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2394659/" ref="ordinalpos=2019&amp;ncbi_uid=1401650&amp;link_uid=PMC2394659" image-link="/pmc/articles/PMC2394659/figure/pone-0002339-g003/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Pathway</span> diagram showing the molecules involved in leukocyte extravasation <span class="highlight" style="background-color:">signaling</span> and their interaction..  From: The Aged Retinal Pigment Epithelium/Choroid: A Potential Substratum for the Pathogenesis of Age-Related Macular Degeneration. </a></p><div class="supp"><p class="details"><div>Color nodes: genes that changed significantly in RPE/choroid from old animals with a fold change &gt;2. Red: upregulation; green: downregulation. The diagram was modified from Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (Ingenuity® Systems).</div></p></div><div class="aux"><div class="resc">Huiyi Chen, et al. PLoS ONE. 2008;3(6):e2339.</div><p class="links"><a class="dblinks" href="/pubmed/18523633" ref="ordinalpos=2019&amp;ncbi_uid=1401650&amp;link_uid=18523633">Citation</a><a class="dblinks" href="/pmc/articles/PMC2394659/" ref="ordinalpos=2019&amp;ncbi_uid=1401650&amp;link_uid=PMC2394659">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2020.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3606632/figure/F6/" ref="ordinalpos=2020&amp;ncbi_uid=4736363&amp;link_uid=PMC3606632" href="/pmc/articles/PMC3606632/"><img src="/pmc/articles/instance/3606632/bin/1471-2164-14-195-6.gif" src-large="/pmc/articles/instance/3606632/bin/1471-2164-14-195-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3606632/" ref="ordinalpos=2020&amp;ncbi_uid=4736363&amp;link_uid=PMC3606632" image-link="/pmc/articles/PMC3606632/figure/F6/" class="imagepopup">Figure 6.  From: 5'-Serial Analysis of Gene Expression studies reveal a transcriptomic switch during fruiting body development in Coprinopsis cinerea. </a></p><div class="supp"><p class="details"><div><b>The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in KEGG <span class="highlight" style="background-color:">pathway</span> analysis.</b> The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is among the S1-Pri-up-regulated pathways in KEGG analysis. Rheb and ribosomal protein small subunit s6e (red boxes) were identified through 5<sup>′</sup>-SAGE as DEGs in the S1-Pri stage. Expression of five other components (green boxes) were verified by real-time PCR.</div></p></div><div class="aux"><div class="resc">Chi Keung Cheng, et al. BMC Genomics. 2013;14:195-195.</div><p class="links"><a class="dblinks" href="/pubmed/23514374" ref="ordinalpos=2020&amp;ncbi_uid=4736363&amp;link_uid=23514374">Citation</a><a class="dblinks" href="/pmc/articles/PMC3606632/" ref="ordinalpos=2020&amp;ncbi_uid=4736363&amp;link_uid=PMC3606632">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2021.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3466241/figure/pone-0044686-g007/" ref="ordinalpos=2021&amp;ncbi_uid=4286398&amp;link_uid=PMC3466241" href="/pmc/articles/PMC3466241/"><img src="/pmc/articles/instance/3466241/bin/pone.0044686.g007.gif" src-large="/pmc/articles/instance/3466241/bin/pone.0044686.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3466241/" ref="ordinalpos=2021&amp;ncbi_uid=4286398&amp;link_uid=PMC3466241" image-link="/pmc/articles/PMC3466241/figure/pone-0044686-g007/" class="imagepopup">Figure 7. Hypothetical modes of action of actives compounds..  From: Implementation of a High-Throughput Screen for Identifying Small Molecules to Activate the Keap1-Nrf2-ARE <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Four potential mechanisms of action for hit compounds to activate Keap1-Nrf2-ARE <span class="highlight" style="background-color:">pathway</span>. Hit compounds can disrupt binding of Keap1 to Nrf2, facilitate Keap1 ubiquitination, inactivate Nrf2 export <span class="highlight" style="background-color:">signaling</span>, or activate protein kinase cascades for Nrf2 phosphorylation.</div></p></div><div class="aux"><div class="resc">Kai Connie Wu, et al. PLoS One. 2012;7(10):e44686.</div><p class="links"><a class="dblinks" href="/pubmed/23056183" ref="ordinalpos=2021&amp;ncbi_uid=4286398&amp;link_uid=23056183">Citation</a><a class="dblinks" href="/pmc/articles/PMC3466241/" ref="ordinalpos=2021&amp;ncbi_uid=4286398&amp;link_uid=PMC3466241">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2022.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3596690/figure/jbr-24-06-424-g004/" ref="ordinalpos=2022&amp;ncbi_uid=4769676&amp;link_uid=PMC3596690" href="/pmc/articles/PMC3596690/"><img src="/pmc/articles/instance/3596690/bin/jbr-24-06-424-g004.gif" src-large="/pmc/articles/instance/3596690/bin/jbr-24-06-424-g004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3596690/" ref="ordinalpos=2022&amp;ncbi_uid=4769676&amp;link_uid=PMC3596690" image-link="/pmc/articles/PMC3596690/figure/jbr-24-06-424-g004/" class="imagepopup">Fig. 4. The resistant GBM cells exhibit defective <span class="highlight" style="background-color:">signaling</span> through the ATM-Chk2 <span class="highlight" style="background-color:">pathway</span>..  From: Decoupling of DNA damage response <span class="highlight" style="background-color:">signaling</span> from DNA damages underlies temozolomide resistance in glioblastoma cells. </a></p><div class="supp"><p class="details"><div>A: U251 (OTR) fails to activate the ATM-Chk2 <span class="highlight" style="background-color:">pathway</span> in response to TMZ. The Western blotting were probed with phospho-specific antibodies against ATM (p-ATM), Chk1 (p-Chk1) and Chk2 (p-Chk2). B: TMZ induces focus formation by p-ATM in U251, but not in U251 (OTR). C and D, TMZ induces focus formation by p-Chk2 in D54 and U251, but not in their TMZ-resistant counterparts. The cells in B, C and D were analyzed at 72 h post TMZ treatment. CT: control; TMZ: temozolomide.</div></p></div><div class="aux"><div class="resc">Bo Cui, et al. J Biomed Res. 2010 November;24(6):424-435.</div><p class="links"><a class="dblinks" href="/pubmed/23554659" ref="ordinalpos=2022&amp;ncbi_uid=4769676&amp;link_uid=23554659">Citation</a><a class="dblinks" href="/pmc/articles/PMC3596690/" ref="ordinalpos=2022&amp;ncbi_uid=4769676&amp;link_uid=PMC3596690">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2023.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3101807/figure/fig7/" ref="ordinalpos=2023&amp;ncbi_uid=3073358&amp;link_uid=PMC3101807" href="/pmc/articles/PMC3101807/"><img src="/pmc/articles/instance/3101807/bin/cddis201110f7.gif" src-large="/pmc/articles/instance/3101807/bin/cddis201110f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3101807/" ref="ordinalpos=2023&amp;ncbi_uid=3073358&amp;link_uid=PMC3101807" image-link="/pmc/articles/PMC3101807/figure/fig7/" class="imagepopup">Figure 7.  From: NPR-A regulates self-renewal and pluripotency of embryonic stem cells. </a></p><div class="supp"><p class="details"><div>Proposed model of NPR-A <span class="highlight" style="background-color:">signaling</span>-mediated maintenance of ES cell self-renewal and pluripotency. Activation of NPR-A by ANP stimulates the cGMP/PKG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, leading to increased expressions of Oct4 and Nanog, and reduced expression of p21, which in turn suppresses ES cell differentiation and maintains ES cell pluripotency. In another <span class="highlight" style="background-color:">pathway</span>, binding of ANP to NPR-A activates the phosphoinositide 3-kinase (PI3K) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which enhances G1-to-S progression through induction of cyclin D1, and maintains pluripotency by activation of Nanog expression</div></p></div><div class="aux"><div class="resc">E M Abdelalim, et al. Cell Death Dis. 2011 March;2(3):e127.</div><p class="links"><a class="dblinks" href="/pubmed/21390061" ref="ordinalpos=2023&amp;ncbi_uid=3073358&amp;link_uid=21390061">Citation</a><a class="dblinks" href="/pmc/articles/PMC3101807/" ref="ordinalpos=2023&amp;ncbi_uid=3073358&amp;link_uid=PMC3101807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2024.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4320493/figure/Fig7/" ref="ordinalpos=2024&amp;ncbi_uid=7156931&amp;link_uid=PMC4320493" href="/pmc/articles/PMC4320493/"><img src="/pmc/articles/instance/4320493/bin/12861_2015_57_Fig7_HTML.gif" src-large="/pmc/articles/instance/4320493/bin/12861_2015_57_Fig7_HTML.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4320493/" ref="ordinalpos=2024&amp;ncbi_uid=7156931&amp;link_uid=PMC4320493" image-link="/pmc/articles/PMC4320493/figure/Fig7/" class="imagepopup">Figure 7.  From: The medaka dhc2 mutant reveals conserved and distinct mechanisms of Hedgehog <span class="highlight" style="background-color:">signaling</span> in teleosts. </a></p><div class="supp"><p class="details"><div> <b>Proposed model of the distinct features of Hh signal transduction in insect, fish and mammal.</b> <i>fu</i> is expressed in a Hedgehog-dependent fashion and is also one of the components of the Hedgehog <span class="highlight" style="background-color:">pathway</span> in fish. Fused negatively regulates Suppressor of Fused (SuFu), which is a negative regulator of Gli/Ci in Hh <span class="highlight" style="background-color:">signaling</span>. The transcription of <i>fused</i> in fish could lead to Hh activation. This positive-feedback loop amplifies Hedgehog <span class="highlight" style="background-color:">pathway</span> in fish downstream of cilia.</div></p></div><div class="aux"><div class="resc">Takayoshi Yamamoto, et al. BMC Dev Biol. 2015;15:9.</div><p class="links"><a class="dblinks" href="/pubmed/25645819" ref="ordinalpos=2024&amp;ncbi_uid=7156931&amp;link_uid=25645819">Citation</a><a class="dblinks" href="/pmc/articles/PMC4320493/" ref="ordinalpos=2024&amp;ncbi_uid=7156931&amp;link_uid=PMC4320493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2025.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713568/figure/F7/" ref="ordinalpos=2025&amp;ncbi_uid=1940875&amp;link_uid=PMC2713568" href="/pmc/articles/PMC2713568/"><img src="/pmc/articles/instance/2713568/bin/zbc0290979420007.gif" src-large="/pmc/articles/instance/2713568/bin/zbc0290979420007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713568/" ref="ordinalpos=2025&amp;ncbi_uid=1940875&amp;link_uid=PMC2713568" image-link="/pmc/articles/PMC2713568/figure/F7/" class="imagepopup">FIGURE 7. From: Notch1 <span class="highlight" style="background-color:">Signaling</span> Sensitizes Tumor Necrosis Factor-related Apoptosis-inducing Ligand-induced Apoptosis in Human Hepatocellular Carcinoma Cells by Inhibiting Akt/Hdm2-mediated p53 Degradation and Up-regulating p53-dependent DR5 Expression. </a></p><div class="supp"><p class="details"><div><b>A model for Notch1 <span class="highlight" style="background-color:">signaling</span>-enhanced apoptosis induction by TRAIL in HCC cells.</b> The model depicts the involvement of Notch1 <span class="highlight" style="background-color:">signaling</span> in extrinsic and intrinsic apoptotic <span class="highlight" style="background-color:">pathway</span>. Notch1 activation suppressed the Akt/Hdm2 <span class="highlight" style="background-color:">pathway</span>, leading to up-regulation of p53 via suppression of p53 proteasome degradation and p53-dependent up-regulation of DR5, resulting in cellular apoptosis.</div></p></div><div class="aux"><div class="resc">Chunmei Wang, et al. J Biol Chem. 2009 June 12;284(24):16183-16190.</div><p class="links"><a class="dblinks" href="/pubmed/19376776" ref="ordinalpos=2025&amp;ncbi_uid=1940875&amp;link_uid=19376776">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713568/" ref="ordinalpos=2025&amp;ncbi_uid=1940875&amp;link_uid=PMC2713568">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2026.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3620549/figure/F1/" ref="ordinalpos=2026&amp;ncbi_uid=4791950&amp;link_uid=PMC3620549" href="/pmc/articles/PMC3620549/"><img src="/pmc/articles/instance/3620549/bin/1471-2148-13-76-1.gif" src-large="/pmc/articles/instance/3620549/bin/1471-2148-13-76-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3620549/" ref="ordinalpos=2026&amp;ncbi_uid=4791950&amp;link_uid=PMC3620549" image-link="/pmc/articles/PMC3620549/figure/F1/" class="imagepopup">Figure 1.  From: Hippo <span class="highlight" style="background-color:">pathway</span> genes developed varied exon numbers and coevolved functional domains in metazoans for species specific growth control. </a></p><div class="supp"><p class="details"><div><b>Structure and function of the Hippo <span class="highlight" style="background-color:">pathway</span>.</b> (<b>A</b>) Hippo <span class="highlight" style="background-color:">pathway</span> in <i>D. melanogaster</i> (from [10] with permission). (<b>B</b>) A wildtype (left) and an overgrown (right) Drosophila eye caused by a <i>hippo</i> mutation (from [7] with permission). (<b>C</b>) Mouse livers from a control (left) and an individual with dysregulated Hippo <span class="highlight" style="background-color:">signaling</span> (right) (from [53] with permission).</div></p></div><div class="aux"><div class="resc">Henan Zhu, et al. BMC Evol Biol. 2013;13:76-76.</div><p class="links"><a class="dblinks" href="/pubmed/23547742" ref="ordinalpos=2026&amp;ncbi_uid=4791950&amp;link_uid=23547742">Citation</a><a class="dblinks" href="/pmc/articles/PMC3620549/" ref="ordinalpos=2026&amp;ncbi_uid=4791950&amp;link_uid=PMC3620549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2027.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC1828071/figure/F1/" ref="ordinalpos=2027&amp;ncbi_uid=996331&amp;link_uid=PMC1828071" href="/pmc/articles/PMC1828071/"><img src="/pmc/articles/instance/1828071/bin/nihms16392f1a.gif" src-large="/pmc/articles/instance/1828071/bin/nihms16392f1a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC1828071/figure/F1/" ref="ordinalpos=2027&amp;ncbi_uid=996331&amp;link_uid=PMC1828071" href="/pmc/articles/PMC1828071/"><img src="/pmc/articles/instance/1828071/bin/nihms16392f1b.gif" src-large="/pmc/articles/instance/1828071/bin/nihms16392f1b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1828071/" ref="ordinalpos=2027&amp;ncbi_uid=996331&amp;link_uid=PMC1828071" image-link="/pmc/articles/PMC1828071/figure/F1/" class="imagepopup">Fig. 1.  From: The Role of Intracellular <span class="highlight" style="background-color:">Signaling</span> in Insulin-mediated Regulation of Drug Metabolizing Enzyme Gene and Protein Expression. </a></p><div class="supp"><p class="details"><div>Fig. 1A Insulin-mediated <span class="highlight" style="background-color:">signaling</span> pathways. A diagrammatic representation of insulin and growth factor receptor mediated <span class="highlight" style="background-color:">signaling</span> with effects on gene transcription, through phosphorylation of the FOXO1, 3a and 4 transcription factors, and mRNA translation, through 4E-PB1, p70 S6 kinase, eIF4B, S6 ribosomal protein, and eIF4G.<br />Fig. 1B Glucagon-mediated <span class="highlight" style="background-color:">signaling</span> pathways. A diagrammatic representation of glucagon mediated <span class="highlight" style="background-color:">signaling</span> through the adenylate cyclase, cAMP, PKA, AMPK <span class="highlight" style="background-color:">pathway</span> with inhibitory effects on mTOR and the transcription factors HNF1alpha and HNF4alpha. The opposing effects of glucagon on insulin-mediated alterations in gene expression may occur through glucagon repression of insulin <span class="highlight" style="background-color:">signaling</span> through mTOR, which is associated with activation of PKA, the phosphorylation of LKB1, and activation of AMPK. Glucagon also represses activation of the phosphorylation of 4E-B P1 and the p70S6 kinase.</div></p></div><div class="aux"><div class="resc">Sang K. Kim, et al. Pharmacol Ther. ;113(1):88-120.</div><p class="links"><a class="dblinks" href="/pubmed/17097148" ref="ordinalpos=2027&amp;ncbi_uid=996331&amp;link_uid=17097148">Citation</a><a class="dblinks" href="/pmc/articles/PMC1828071/" ref="ordinalpos=2027&amp;ncbi_uid=996331&amp;link_uid=PMC1828071">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2028.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4196276/figure/F7/" ref="ordinalpos=2028&amp;ncbi_uid=6719032&amp;link_uid=PMC4196276" href="/pmc/articles/PMC4196276/"><img src="/pmc/articles/instance/4196276/bin/emss-60634-f0007.gif" src-large="/pmc/articles/instance/4196276/bin/emss-60634-f0007.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4196276/" ref="ordinalpos=2028&amp;ncbi_uid=6719032&amp;link_uid=PMC4196276" image-link="/pmc/articles/PMC4196276/figure/F7/" class="imagepopup">Fig. 7. The ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (the focus <span class="highlight" style="background-color:">pathway</span>) is a target of the drugs Lapatinib and Erlotinib that are used for cancer treatment..  From: Entourage: Visualizing Relationships between Biological Pathways using Contextual Subsets. </a></p><div class="supp"><p class="details"><div>As shown in the <span class="highlight" style="background-color:">pathway</span> list on the left that results from a query for similar pathways, the <i>ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></i> is related to many cancer pathways. A <span class="highlight" style="background-color:">signaling</span> cascade from <i>ErbB2</i> to <i>Ras</i> is selected. The integrated enRoute view shows copy number and mRNA expression data for breast cancer cell lines. The sensitivity of the different cell lines to <i>Lapatinib</i> and <i>Erlotinib</i> is reported at the top. For the shown cell lines, increased copy numbers of <i>ErB2</i> (high red bars in the <i>ErbB2</i> row) result in over-expression of this gene (high blue bars). Furthermore, there is a strong relation between <i>ErbB2</i> over-expression and sensitivity to <i>Lapatinib</i> (high blue bars for gene over-expression in the <i>ErbB2</i> row coincide with low bars in the <i>Lapatinib</i> row). This means that <i>Lapatinib</i> is effective if <i>ErbB2</i> is highly expressed. There are, however, two exceptions - the highlighted cell lines (gold and orange), for which an under-expression in <i>Ras</i> downstream in the <span class="highlight" style="background-color:">pathway</span> is observed, likely causing <i>Lapatinib</i> to be ineffective in these cases. While this observation was made for breast cancer tissue, exploring the related context pathways by setting the focus node to <i>Ras</i> reveals that the same <span class="highlight" style="background-color:">signaling</span> cascade (i.e., path) is also contained in the non-small cell lung cancer <span class="highlight" style="background-color:">pathway</span>. Thus, it would be interesting to explore the transferability of the observed resistance pattern to this tissue type.</div></p></div><div class="aux"><div class="resc">Alexander Lex, et al. IEEE Trans Vis Comput Graph. ;19(12):2536-2545.</div><p class="links"><a class="dblinks" href="/pubmed/24051820" ref="ordinalpos=2028&amp;ncbi_uid=6719032&amp;link_uid=24051820">Citation</a><a class="dblinks" href="/pmc/articles/PMC4196276/" ref="ordinalpos=2028&amp;ncbi_uid=6719032&amp;link_uid=PMC4196276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2029.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3670293/figure/F1/" ref="ordinalpos=2029&amp;ncbi_uid=5317606&amp;link_uid=PMC3670293" href="/pmc/articles/PMC3670293/"><img src="/pmc/articles/instance/3670293/bin/jkst-1-131-g1.gif" src-large="/pmc/articles/instance/3670293/bin/jkst-1-131-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3670293/" ref="ordinalpos=2029&amp;ncbi_uid=5317606&amp;link_uid=PMC3670293" image-link="/pmc/articles/PMC3670293/figure/F1/" class="imagepopup">Figure. From: The JAK-STAT <span class="highlight" style="background-color:">pathway</span> in hypertrophic stress <span class="highlight" style="background-color:">signaling</span> and genomic stress response. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Models of mechanical stretch activation of JAK kinases and STAT proteins. Conformational model. Mechanical stretch is postulated to induce conformational changes in the gp130 and AT1 receptors that expose binding sites for JAK2 kinases and Gαq proteins, respectively. It is unclear how or if gp130 dimerizes in order to allow JAK2 kinase cross-phosphorylation. The AT1 receptor acting as a stretch receptor will activate the RAS/MEK/ERK/p38 <span class="highlight" style="background-color:">pathway</span> leading to upregulation of the angiotensinogen gene. The transcription factor activated by p38 or ERK1/2 has not been identified. Paracrine model. Mechanical stretch initiates signal transducer association with receptors as in conformational model. Continued <span class="highlight" style="background-color:">signaling</span> after mechanical stretch exhaustion is performed by Ang II produced in response to stretch-activated AT1 receptors to give autocrine <span class="highlight" style="background-color:">signaling</span> as well as paracrine <span class="highlight" style="background-color:">signaling</span>. Ang II-activated cardiac fibroblasts produce IL-6 family cytokines that activate the canonical JAK-STAT <span class="highlight" style="background-color:">pathway</span> in cardiomyocytes via paracrine <span class="highlight" style="background-color:">signaling</span> thereby maintaining the post-mechanical stretch hypertrophic state.</div></p></div><div class="aux"><div class="resc">Michael A. Wagner, et al. JAKSTAT. 2012 April 1;1(2):131-141.</div><p class="links"><a class="dblinks" href="/pubmed/24058762" ref="ordinalpos=2029&amp;ncbi_uid=5317606&amp;link_uid=24058762">Citation</a><a class="dblinks" href="/pmc/articles/PMC3670293/" ref="ordinalpos=2029&amp;ncbi_uid=5317606&amp;link_uid=PMC3670293">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2030.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1289420/figure/fig7/" ref="ordinalpos=2030&amp;ncbi_uid=614906&amp;link_uid=PMC1289420" href="/pmc/articles/PMC1289420/"><img src="/pmc/articles/instance/1289420/bin/zmk0120574260007.gif" src-large="/pmc/articles/instance/1289420/bin/zmk0120574260007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1289420/" ref="ordinalpos=2030&amp;ncbi_uid=614906&amp;link_uid=PMC1289420" image-link="/pmc/articles/PMC1289420/figure/fig7/" class="imagepopup">Figure 7. From: Plasma Membrane Organization Is Essential for Balancing Competing Pseudopod- and Uropod-promoting Signals during Neutrophil Polarization and Migration. </a></p><div class="supp"><p class="details"><div>A model for the role of cholesterol in membrane organization during neutrophil polarization and migration. During polarization and migration (top diagram), activation of the fMLP-receptor at the front stimulates the G<sub>i</sub>/PI3-K <span class="highlight" style="background-color:">pathway</span> (red), which promotes actin polymerization and protrusion. Receptor activation at the rear stimulates the G<sub>12/13</sub>/RhoA <span class="highlight" style="background-color:">pathway</span> (green), which promotes actin-myosin contraction. These pathways segregate and are maintained at opposite poles of the cell in part via mutual negative feedback pathways (dotted red and green lines; Xu et al., 2003). Cholesterol is critical for the formation of membrane domains that enable the spatial segregation of <span class="highlight" style="background-color:">signaling</span> pathways to the front and rear of cells. Cholesterol may directly influence the activation of <span class="highlight" style="background-color:">signaling</span> proteins involved in the G<sub>i</sub>/PI3-K <span class="highlight" style="background-color:">pathway</span> and inhibit lateral diffusion of D3-PIs. Cholesterol may also down-regulate the G<sub>i</sub>/PI3-K <span class="highlight" style="background-color:">pathway</span> in the uropod, inhibiting negative feedback on the RhoA <span class="highlight" style="background-color:">pathway</span> and promoting uropod formation. In response to cholesterol depletion by MβCD (bottom diagram), activation of the G<sub>i</sub>/PI3-K <span class="highlight" style="background-color:">pathway</span> (red) is enhanced, leading to D3-PI accumulation over the entire plasma membrane, and to actin polymerization and the formation of multiple protrusions. This in turn enhances negative feedback on the RhoA <span class="highlight" style="background-color:">pathway</span> (green), inhibiting uropod function.</div></p></div><div class="aux"><div class="resc">Stéphane Bodin, et al. Mol Biol Cell. 2005 December;16(12):5773-5783.</div><p class="links"><a class="dblinks" href="/pubmed/16207810" ref="ordinalpos=2030&amp;ncbi_uid=614906&amp;link_uid=16207810">Citation</a><a class="dblinks" href="/pmc/articles/PMC1289420/" ref="ordinalpos=2030&amp;ncbi_uid=614906&amp;link_uid=PMC1289420">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2031.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3393662/figure/pone-0040566-g008/" ref="ordinalpos=2031&amp;ncbi_uid=4119661&amp;link_uid=PMC3393662" href="/pmc/articles/PMC3393662/"><img src="/pmc/articles/instance/3393662/bin/pone.0040566.g008.gif" src-large="/pmc/articles/instance/3393662/bin/pone.0040566.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3393662/" ref="ordinalpos=2031&amp;ncbi_uid=4119661&amp;link_uid=PMC3393662" image-link="/pmc/articles/PMC3393662/figure/pone-0040566-g008/" class="imagepopup">Figure 8. Model of Shp2 action in FcεRΙ <span class="highlight" style="background-color:">signaling</span>..  From: Shp2 Activates Fyn and Ras to Regulate RBL-2H3 Mast Cell Activation following Fc?RI Aggregation. </a></p><div class="supp"><p class="details"><div>Upon FcεRΙ stimulation, Fyn kinase and Ras become activated in a Shp2-dependent manner. Activated Fyn transmits positive signals to PI3K, leading to PI3K activation that accounts for calcium mobilization as well as for Akt phosphoryaltion. MAPKs also become phosphorylated following Ras and Fyn activation that are responsible for production of most cytokines. Paxillin is phosphoryated indirectly by Shp2 upon FcεRΙ stimulation. The direct substrates of Shp2 in FcεRΙ <span class="highlight" style="background-color:">signaling</span> remain to be defined. Another acknowledged <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> Lyn/Syk/LAT is also shown.</div></p></div><div class="aux"><div class="resc">Xiaoyun Fang, et al. PLoS One. 2012;7(7):e40566.</div><p class="links"><a class="dblinks" href="/pubmed/22802969" ref="ordinalpos=2031&amp;ncbi_uid=4119661&amp;link_uid=22802969">Citation</a><a class="dblinks" href="/pmc/articles/PMC3393662/" ref="ordinalpos=2031&amp;ncbi_uid=4119661&amp;link_uid=PMC3393662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2032.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3216625/figure/f6/" ref="ordinalpos=2032&amp;ncbi_uid=3724762&amp;link_uid=PMC3216625" href="/pmc/articles/PMC3216625/"><img src="/pmc/articles/instance/3216625/bin/srep00144-f6.gif" src-large="/pmc/articles/instance/3216625/bin/srep00144-f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3216625/" ref="ordinalpos=2032&amp;ncbi_uid=3724762&amp;link_uid=PMC3216625" image-link="/pmc/articles/PMC3216625/figure/f6/" class="imagepopup">Figure 6. Simulations of the proposed TRAIL <span class="highlight" style="background-color:">signaling</span> topology..  From: Enhancing apoptosis in TRAIL-resistant cancer cells using fundamental             response rules. </a></p><div class="supp"><p class="details"><div>(A) Comparison of M5 simulations (solid lines) with experimental data (black                         points) in wildtype, RIP1 KD, FADD KD, caspase-8 KD and TRAF2 KD. (B) Static                         topology of the proposed model for TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Modifications                         are indicated by blue arrows.</div></p></div><div class="aux"><div class="resc">Vincent Piras, et al. Sci Rep. 2011;1:144.</div><p class="links"><a class="dblinks" href="/pubmed/22355661" ref="ordinalpos=2032&amp;ncbi_uid=3724762&amp;link_uid=22355661">Citation</a><a class="dblinks" href="/pmc/articles/PMC3216625/" ref="ordinalpos=2032&amp;ncbi_uid=3724762&amp;link_uid=PMC3216625">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2033.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4246620/figure/f2-ol-09-01-0235/" ref="ordinalpos=2033&amp;ncbi_uid=6891249&amp;link_uid=PMC4246620" href="/pmc/articles/PMC4246620/"><img src="/pmc/articles/instance/4246620/bin/OL-09-01-0235-g01.gif" src-large="/pmc/articles/instance/4246620/bin/OL-09-01-0235-g01.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4246620/" ref="ordinalpos=2033&amp;ncbi_uid=6891249&amp;link_uid=PMC4246620" image-link="/pmc/articles/PMC4246620/figure/f2-ol-09-01-0235/" class="imagepopup">Figure 2.  From: The expression of Wnt-1 inducible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein-2 in astrocytoma: Correlation between pathological grade and clinical outcome. </a></p><div class="supp"><p class="details"><div>WISP-2 in normal brain and various grades of astrocytic gliomas. (A) Negative WISP-2 expression in normal human brain tissue. (B and C) Weak WISP-2 in stage I and II astrocytomas, respectively, with delicate staining of cytoplasm. (D and E) Markedly increased WISP-2 in malignant stage III and IV astrocytomas, respectively, with positive staining of cytoplasm (magnification, ×400). WISP-2, Wnt-1 inducible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein-2.</div></p></div><div class="aux"><div class="resc">GELEI XIAO, et al. Oncol Lett. 2015 January;9(1):235-240.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2033&amp;ncbi_uid=6891249&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4246620/" ref="ordinalpos=2033&amp;ncbi_uid=6891249&amp;link_uid=PMC4246620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2034.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2704880/figure/fig8/" ref="ordinalpos=2034&amp;ncbi_uid=1939436&amp;link_uid=PMC2704880" href="/pmc/articles/PMC2704880/"><img src="/pmc/articles/instance/2704880/bin/JCS041137F8.gif" src-large="/pmc/articles/instance/2704880/bin/JCS041137F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2704880/" ref="ordinalpos=2034&amp;ncbi_uid=1939436&amp;link_uid=PMC2704880" image-link="/pmc/articles/PMC2704880/figure/fig8/" class="imagepopup">Fig. 8. From: The yeast endocytic protein Epsin 2 functions in a cell-division  <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Model for yeast epsin function in cell polarity and division. Yeast epsins,  Ent1 and Ent2, bind Cdc42 GAPs Rga1 and Rga2 to activate Cdc42 <span class="highlight" style="background-color:">signaling</span>  towards cell polarity (Gic1/2-dependent) establishment pathways  (Aguilar et al., 2006). From  this study we propose that epsins (particularly Ent2) affect septin  organization pathways by interacting with another GAP, Bem3, which is also  involved in septin ring formation. This novel septin-regulatory circuit would  be redundant with another <span class="highlight" style="background-color:">pathway</span> (X), yet to be identified.</div></p></div><div class="aux"><div class="resc">Debarati Mukherjee, et al. J Cell Sci. 2009 July 15;122(14):2453-2463.</div><p class="links"><a class="dblinks" href="/pubmed/19531587" ref="ordinalpos=2034&amp;ncbi_uid=1939436&amp;link_uid=19531587">Citation</a><a class="dblinks" href="/pmc/articles/PMC2704880/" ref="ordinalpos=2034&amp;ncbi_uid=1939436&amp;link_uid=PMC2704880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2035.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3598224/figure/F3/" ref="ordinalpos=2035&amp;ncbi_uid=4718121&amp;link_uid=PMC3598224" href="/pmc/articles/PMC3598224/"><img src="/pmc/articles/instance/3598224/bin/1475-2891-12-24-3.gif" src-large="/pmc/articles/instance/3598224/bin/1475-2891-12-24-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3598224/" ref="ordinalpos=2035&amp;ncbi_uid=4718121&amp;link_uid=PMC3598224" image-link="/pmc/articles/PMC3598224/figure/F3/" class="imagepopup">Figure 3.  From: Associations between dietary patterns and gene expression profiles of healthy men and women: a cross-sectional study. </a></p><div class="supp"><p class="details"><div><b>The modified canonical pathways according to scores for the Prudent dietary pattern. Legend:</b> Gene expression differences (≥ ± 1.2 fold change) in canonical pathways comparing high and low scores for the Prudent dietary pattern <b>(A)</b> For the men <b>(B)</b> For the women. P-values for Functional Analysis of dataset by IPA (Fisher’s Exact Test) are presented. % of genes significantly up-regulated and down-regulated in each canonical <span class="highlight" style="background-color:">pathway</span> are presented (number of genes differently expressed/number of genes in the pathways*100). <b>(A)</b> PDGF <span class="highlight" style="background-color:">Signaling</span>: P = 0.004, 14 genes significantly different; ERK/MAPK <span class="highlight" style="background-color:">Signaling</span>: P = 0.004, 29 genes significantly different; FcyRIIB <span class="highlight" style="background-color:">Signaling</span> in B Lymphocytes: P = 0.003, 10 genes significantly different; B Cell Receptor <span class="highlight" style="background-color:">Signaling</span>: P = 0.002, 24 genes significantly different; CTLA4 <span class="highlight" style="background-color:">Signaling</span> in Cytotoxic T Lymphocytes: P = 0.003, 17 genes significantly different; PI3K <span class="highlight" style="background-color:">Signaling</span> in B Lymphocytes: P = 0.0009, 23 genes significantly different; Leucocyte Extravasation <span class="highlight" style="background-color:">Signaling</span>: P = 0.0002, 32 genes significantly different. <b>(B)</b> <span class="highlight" style="background-color:">Signaling</span> by Rho Family GTPase: P = 0.02, 20 genes significantly different; B Cell Receptor <span class="highlight" style="background-color:">Signaling</span>: P = 0.02, 14 genes significantly different; Role of NFAT in Regulation of the Immune Response: P = 0.01, 16 genes significantly different; RhoGDI <span class="highlight" style="background-color:">Signaling</span>: P = 0.009, 17 genes significantly different; Cyclins and Cell Cycle Regulation: P = 0.007, 10 genes significantly different; Breast Cancer Regulation by Stathmin1: P = 0.0005, 22 genes significantly different; B Cell Development: P = 0.0004, 7 genes significantly different.</div></p></div><div class="aux"><div class="resc">Annie Bouchard-Mercier, et al. Nutr J. 2013;12:24-24.</div><p class="links"><a class="dblinks" href="/pubmed/23398686" ref="ordinalpos=2035&amp;ncbi_uid=4718121&amp;link_uid=23398686">Citation</a><a class="dblinks" href="/pmc/articles/PMC3598224/" ref="ordinalpos=2035&amp;ncbi_uid=4718121&amp;link_uid=PMC3598224">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2036.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2933676/figure/F3/" ref="ordinalpos=2036&amp;ncbi_uid=2454797&amp;link_uid=PMC2933676" href="/pmc/articles/PMC2933676/"><img src="/pmc/articles/instance/2933676/bin/1423-0127-17-69-3.gif" src-large="/pmc/articles/instance/2933676/bin/1423-0127-17-69-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2933676/" ref="ordinalpos=2036&amp;ncbi_uid=2454797&amp;link_uid=PMC2933676" image-link="/pmc/articles/PMC2933676/figure/F3/" class="imagepopup">Figure 3.  From: Genetic susceptibility to advanced retinopathy of prematurity (ROP). </a></p><div class="supp"><p class="details"><div><b>A schematic representation of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Norrin and Wnt act as ligands to bind FZD4 that interact with LRP5. In the absence of Wnt <span class="highlight" style="background-color:">signaling</span>, beta-catenin is phosphorylated and subjected to proteosomal degradation. In the presence of Wnt <span class="highlight" style="background-color:">signaling</span>, beta-catenin accumulates in the cytoplasm and enters the nucleus. Its subsequent interactions with a member of Tcf/Lef family activate the transcription of Wnt target genes. It was also shown that TSPAN12 is a component of the norrin-LRP5-FZD4 <span class="highlight" style="background-color:">signaling</span> complex and enhances the levels of norrin-beta-catenin <span class="highlight" style="background-color:">signaling</span> but not Wnt-beta-catenin <span class="highlight" style="background-color:">signaling</span>. Because of mutations in NDP, FZD4, LRP5 and TSPAN12 genes, abnormal <span class="highlight" style="background-color:">signaling</span> may occur which may result in defective Wnt target genes activation that may give rise to FEVR and ROP pathology. AD = autosomal dominant; AR = autosomal recessive; XL = X-linked recessive; FEVR = familial exudative vitreoretinopathy; ROP = retinopathy of prematurity; NDP = Norrie disease pseudoglioma.</div></p></div><div class="aux"><div class="resc">Barkur S Shastry. J Biomed Sci. 2010;17(1):69-69.</div><p class="links"><a class="dblinks" href="/pubmed/20738858" ref="ordinalpos=2036&amp;ncbi_uid=2454797&amp;link_uid=20738858">Citation</a><a class="dblinks" href="/pmc/articles/PMC2933676/" ref="ordinalpos=2036&amp;ncbi_uid=2454797&amp;link_uid=PMC2933676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2037.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109514/figure/fig4/" ref="ordinalpos=2037&amp;ncbi_uid=3006627&amp;link_uid=PMC3109514" href="/pmc/articles/PMC3109514/"><img src="/pmc/articles/instance/3109514/bin/IJAD2011-189728.004.gif" src-large="/pmc/articles/instance/3109514/bin/IJAD2011-189728.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109514/" ref="ordinalpos=2037&amp;ncbi_uid=3006627&amp;link_uid=PMC3109514" image-link="/pmc/articles/PMC3109514/figure/fig4/" class="imagepopup">Figure 4.  From: GSK3 Function in the Brain during Development, Neuronal Plasticity, and Neurodegeneration. </a></p><div class="supp"><p class="details"><div>Proposed model of GSK3 activation by amyloid-<i>β</i> protein in AD. Amyloid-<i>β</i> oligomers bind to the insulin receptor and inhibit PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, and Akt is unable to phosphorylate and inactivate GSK3. A<i>β</i> also induces the expression of DKK1, which internalizes LRP6 receptor and inhibits Wnt <span class="highlight" style="background-color:">signaling</span> leading to GSK3 activation. A<i>β</i> can bind to Frizzled receptor (FzR) and inactivate Wnt <span class="highlight" style="background-color:">signaling</span> as well. ApoE also inhibits this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and activates GSK3. Tau hyperphosphorylation and NFT formation may result from GSK3 overactivation.</div></p></div><div class="aux"><div class="resc">Pamela Salcedo-Tello, et al. Int J Alzheimers Dis. 2011;2011:189728.</div><p class="links"><a class="dblinks" href="/pubmed/21660241" ref="ordinalpos=2037&amp;ncbi_uid=3006627&amp;link_uid=21660241">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109514/" ref="ordinalpos=2037&amp;ncbi_uid=3006627&amp;link_uid=PMC3109514">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2038.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3180046/figure/fig02/" ref="ordinalpos=2038&amp;ncbi_uid=3263762&amp;link_uid=PMC3180046" href="/pmc/articles/PMC3180046/"><img src="/pmc/articles/instance/3180046/bin/ajcr0001-0192-f2.gif" src-large="/pmc/articles/instance/3180046/bin/ajcr0001-0192-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3180046/" ref="ordinalpos=2038&amp;ncbi_uid=3263762&amp;link_uid=PMC3180046" image-link="/pmc/articles/PMC3180046/figure/fig02/" class="imagepopup">Figure 2.  From: NF-?B and cancer: a paradigm of Yin-Yang. </a></p><div class="supp"><p class="details"><div>NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. Although the canonical and non-canonical <span class="highlight" style="background-color:">signaling</span> pathways primarily activate the RelA/p50 and RelB/p52 dimers, respectively, all NF-κB members can be activated by either <span class="highlight" style="background-color:">pathway</span> or both. In fact, the RelA/p50 dimers may be sequestered in the cytoplasm by p100 and can be activated through p100 processing. On the other hand, NF-κB dimers containing p52 may be sequestered in the cytoplasm by IκBα and can be activated through IκBα degradation. Furthermore, activation of the canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be induced through inducible degradation of IκBβ, IκBε and p105, a process similar to the inducible IκBα degradation, although their degradation dynamics can be different.</div></p></div><div class="aux"><div class="resc">Gutian Xiao, et al. Am J Cancer Res. 2011;1(2):192-221.</div><p class="links"><a class="dblinks" href="/pubmed/21969033" ref="ordinalpos=2038&amp;ncbi_uid=3263762&amp;link_uid=21969033">Citation</a><a class="dblinks" href="/pmc/articles/PMC3180046/" ref="ordinalpos=2038&amp;ncbi_uid=3263762&amp;link_uid=PMC3180046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2039.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3205587/figure/F5/" ref="ordinalpos=2039&amp;ncbi_uid=3403293&amp;link_uid=PMC3205587" href="/pmc/articles/PMC3205587/"><img src="/pmc/articles/instance/3205587/bin/2173fig5.gif" src-large="/pmc/articles/instance/3205587/bin/2173fig5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3205587/" ref="ordinalpos=2039&amp;ncbi_uid=3403293&amp;link_uid=PMC3205587" image-link="/pmc/articles/PMC3205587/figure/F5/" class="imagepopup">Figure 5. From: Genome-wide identification of microRNA targets in human ES cells reveals a role for miR-302 in modulating BMP response. </a></p><div class="supp"><p class="details"><div>miR-302/367 targets modulate TGF-β and BMP <span class="highlight" style="background-color:">signaling</span>. In hESCs, neural induction is initiated by inhibition of TGF-β and BMP <span class="highlight" style="background-color:">signaling</span>. The rapid decline in <i>mir-302</i> levels upon neural induction suggests that miR-302/367 promotes self-renewal and pluripotency and represses neural differentiation, possibly by suppressing inhibitors of TGF-β and BMP pathways. In this study, we identified three novel miR-302/367 targets that inhibit BMP <span class="highlight" style="background-color:">signaling</span>. Thus, by promoting TGF-β and BMP <span class="highlight" style="background-color:">signaling</span> at the ESC stage through down-regulation of <span class="highlight" style="background-color:">pathway</span> inhibitors, miR-302/367 promotes the pluripotency stage and attenuates neural differentiation.</div></p></div><div class="aux"><div class="resc">Inna Lipchina, et al. Genes Dev. 2011 October 15;25(20):2173-2186.</div><p class="links"><a class="dblinks" href="/pubmed/22012620" ref="ordinalpos=2039&amp;ncbi_uid=3403293&amp;link_uid=22012620">Citation</a><a class="dblinks" href="/pmc/articles/PMC3205587/" ref="ordinalpos=2039&amp;ncbi_uid=3403293&amp;link_uid=PMC3205587">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2040.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3930980/figure/F3/" ref="ordinalpos=2040&amp;ncbi_uid=5872127&amp;link_uid=PMC3930980" href="/pmc/articles/PMC3930980/"><img src="/pmc/articles/instance/3930980/bin/WJG-20-1822-g003.gif" src-large="/pmc/articles/instance/3930980/bin/WJG-20-1822-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3930980/" ref="ordinalpos=2040&amp;ncbi_uid=5872127&amp;link_uid=PMC3930980" image-link="/pmc/articles/PMC3930980/figure/F3/" class="imagepopup">Figure 3.  From: Sophocarpine attenuates liver fibrosis by inhibiting the TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in rats. </a></p><div class="supp"><p class="details"><div>Expression of pro-fibrotic cytokines and toll-like receptor 4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related-proteins is suppressed in sophocarpine-treated rats. Immunochemical analysis of the protein expression of α-SMA, TGF-β1, IL-6, TLR4 and ERK1/2 in the liver tissue of each group was performed as described in Materials and Methods. The results show the protein expression of α-SMA (× 200), TGF-β1 (× 200), IL-6 (× 200), TLR4 (× 200) and ERK1/2 (× 200) in the fibrotic livers of each group. TLR4: Toll-like receptor 4; IL-6: Interleukin-6; TGF-β: Transforming growth factor-β; SMA: Smooth muscle actin.</div></p></div><div class="aux"><div class="resc">Hui Qian, et al. World J Gastroenterol. 2014 February 21;20(7):1822-1832.</div><p class="links"><a class="dblinks" href="/pubmed/24587659" ref="ordinalpos=2040&amp;ncbi_uid=5872127&amp;link_uid=24587659">Citation</a><a class="dblinks" href="/pmc/articles/PMC3930980/" ref="ordinalpos=2040&amp;ncbi_uid=5872127&amp;link_uid=PMC3930980">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2041.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3460778/figure/F2/" ref="ordinalpos=2041&amp;ncbi_uid=4274671&amp;link_uid=PMC3460778" href="/pmc/articles/PMC3460778/"><img src="/pmc/articles/instance/3460778/bin/1756-0500-5-392-2.gif" src-large="/pmc/articles/instance/3460778/bin/1756-0500-5-392-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3460778/" ref="ordinalpos=2041&amp;ncbi_uid=4274671&amp;link_uid=PMC3460778" image-link="/pmc/articles/PMC3460778/figure/F2/" class="imagepopup">Figure 2.  From: Phytohormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis method for comparing hormone responses in plant-pest interactions. </a></p><div class="supp"><p class="details"><div><b>Participation of phytohormones in the response to soybean cyst nematode.</b> Hormone <span class="highlight" style="background-color:">pathway</span> scores indicating induction and suppression of <span class="highlight" style="background-color:">signaling</span> for the hormones abscisic acid (ABA), ethylene (ET), jasmonic acid (JA), and salicylic acid (SA) are shown for datasets corresponding to the compatible interaction at 2, 5 and 10 days post-infection. The color scale is the same used in Figure1.</div></p></div><div class="aux"><div class="resc">Matthew E Studham, et al. BMC Res Notes. 2012;5:392-392.</div><p class="links"><a class="dblinks" href="/pubmed/22846705" ref="ordinalpos=2041&amp;ncbi_uid=4274671&amp;link_uid=22846705">Citation</a><a class="dblinks" href="/pmc/articles/PMC3460778/" ref="ordinalpos=2041&amp;ncbi_uid=4274671&amp;link_uid=PMC3460778">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2042.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938097/figure/F8/" ref="ordinalpos=2042&amp;ncbi_uid=3059918&amp;link_uid=PMC2938097" href="/pmc/articles/PMC2938097/"><img src="/pmc/articles/instance/2938097/bin/zjw0071036280008.gif" src-large="/pmc/articles/instance/2938097/bin/zjw0071036280008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938097/" ref="ordinalpos=2042&amp;ncbi_uid=3059918&amp;link_uid=PMC2938097" image-link="/pmc/articles/PMC2938097/figure/F8/" class="imagepopup">Fig. 8. From: Studies of Phosphoproteomic Changes Induced by Nucleophosmin-Anaplastic Lymphoma Kinase (ALK) Highlight Deregulation of Tumor Necrosis Factor (TNF)/Fas/TNF-related Apoptosis-induced Ligand <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in ALK-positive Anaplastic Large Cell Lymphoma. </a></p><div class="supp"><p class="details"><div><b>Validation of phosphoproteins involved in TNF/Fas/TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> using GP293 cells transfected with empty vector, <i>NPM-ALK<sup>FFF</sup></i> mutant, and <i>NPM-ALK</i>.</b> The immunoprecipitation (<i>IP</i>) experiments were performed using empty vector-, <i>NPM-ALK<sup>FFF</sup></i> mutant-, and <i>NPM-ALK</i>-transfected GP293 cell with anti-STAT3 antibody (<i>A</i>), anti-TRAP1 antibody (<i>B</i>), anti-RIP antibody (<i>C</i>), and anti-FAF1 antibody (<i>D</i>). The phosphorylation statuses of these four proteins were detected by using anti-phosphotyrosine antibody. <i>E</i>, the presence of NPM-ALK was detected in lysates from empty vector-, <i>NPM-ALK<sup>FFF</sup></i> mutant-, and <i>NPM-ALK</i>-transfected cells. <i>IB</i>, immunoblot; <i>phos</i>, phospho.</div></p></div><div class="aux"><div class="resc">Fang Wu, et al. Mol Cell Proteomics. 2010 July;9(7):1616-1632.</div><p class="links"><a class="dblinks" href="/pubmed/20393185" ref="ordinalpos=2042&amp;ncbi_uid=3059918&amp;link_uid=20393185">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938097/" ref="ordinalpos=2042&amp;ncbi_uid=3059918&amp;link_uid=PMC2938097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2043.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3526534/figure/pone-0051988-g006/" ref="ordinalpos=2043&amp;ncbi_uid=4494191&amp;link_uid=PMC3526534" href="/pmc/articles/PMC3526534/"><img src="/pmc/articles/instance/3526534/bin/pone.0051988.g006.gif" src-large="/pmc/articles/instance/3526534/bin/pone.0051988.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3526534/" ref="ordinalpos=2043&amp;ncbi_uid=4494191&amp;link_uid=PMC3526534" image-link="/pmc/articles/PMC3526534/figure/pone-0051988-g006/" class="imagepopup">Figure 6. Hierarchical clustering and Tree View analysis..  From: Different Transcriptional Profiles of RAW264.7 Infected with Mycobacterium tuberculosis H37Rv and BCG Identified via Deep Sequencing. </a></p><div class="supp"><p class="details"><div> Figure 6A. The metabolism pathways analysis. Figure 6B. The genetic information processing pathways analysis data. Figure 6C. The Environmental information processing analysis. Figure 6D. The cellular processes analysis Figure 6E. The Organismal systems analysis. <img src="/pmc/articles/instance/3526534/bin/pone.0051988.e001.jpg" class="inline-graphic" style="border-style: none;" /> The up regulated genes showed value over zero, while the down regulated genes showed value range from 0 to −2. Results showed that the most up regulated genes belongs to the following pathways: cellular process pathways, such as cell cycle and apoptosis; environmental information processing pathways, such as MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; genetic information processing pathways such as ribosom; global pathways such as microbial metabolism in diverse environment; organismal systems pathways, such as T cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, antigen processing and presentation molecules, while the most down regulated genes belongs to the following pathways: cellular process pathways, such as phagosome, endocytosis and autophagy; environmental information processing pathways, such as calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; global <span class="highlight" style="background-color:">pathway</span> such as metabolic pathways; organismal systems pathways, such as Fc epsilon RI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (as shown in Figure 6, Table S3, and Table S5). Results suggested that if we want to improve present tuberculosis therapy, we may pay more attention to the function of these DGEs to effectively control <i>Mycobacterium tuberculosis</i> infection.</div></p></div><div class="aux"><div class="resc">Fengguang Pan, et al. PLoS One. 2012;7(12):e51988.</div><p class="links"><a class="dblinks" href="/pubmed/23284841" ref="ordinalpos=2043&amp;ncbi_uid=4494191&amp;link_uid=23284841">Citation</a><a class="dblinks" href="/pmc/articles/PMC3526534/" ref="ordinalpos=2043&amp;ncbi_uid=4494191&amp;link_uid=PMC3526534">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2044.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2494844/figure/f2/" ref="ordinalpos=2044&amp;ncbi_uid=1443126&amp;link_uid=PMC2494844" href="/pmc/articles/PMC2494844/"><img src="/pmc/articles/instance/2494844/bin/zh70100832360002.gif" src-large="/pmc/articles/instance/2494844/bin/zh70100832360002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2494844/" ref="ordinalpos=2044&amp;ncbi_uid=1443126&amp;link_uid=PMC2494844" image-link="/pmc/articles/PMC2494844/figure/f2/" class="imagepopup">Fig. 2. From: Genomic dissection of the cytokine-controlled STAT5 <span class="highlight" style="background-color:">signaling</span> network in liver . </a></p><div class="supp"><p class="details"><div>JAK-STAT <span class="highlight" style="background-color:">signaling</span> in hepatocytes in the presence and absence of STAT5A/B. <i>A</i>: cytokine-induced <span class="highlight" style="background-color:">signaling</span> in the presence of STAT5A/B. The growth hormone receptor (GHR) specifically recruits and preferentially activates STAT5A/B, which in turn induce the expression of specific sets of genes and elicit a STAT5 response. Suppressors of cytokine <span class="highlight" style="background-color:">signaling</span> (SOCS)2 and SOCS3, 2 negative regulators of the JAK/STAT <span class="highlight" style="background-color:">pathway</span>, are encoded by genes that are controlled by STAT5. Members of the interleukin (IL)-6 family of cytokines activate STAT3 and elicit a STAT3 response in the cell, while interferons (IFNs) activate STAT1. <i>B</i>: cytokine-induced <span class="highlight" style="background-color:">signaling</span> in the absence of STAT5A/B. In the absence of STAT5, both STAT1 and STAT3 can be recruited to the GHR, although with lower affinity than STAT5. Thus GH can aberrantly activate a STAT1 and a STAT3 program in hepatocytes as demonstrated by gene expression profiling. Moreover, impaired expression of negative regulators from the SOCS family will exacerbate STAT3 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Atsushi Hosui, et al. Physiol Genomics. 2008 July;34(2):135-143.</div><p class="links"><a class="dblinks" href="/pubmed/18460640" ref="ordinalpos=2044&amp;ncbi_uid=1443126&amp;link_uid=18460640">Citation</a><a class="dblinks" href="/pmc/articles/PMC2494844/" ref="ordinalpos=2044&amp;ncbi_uid=1443126&amp;link_uid=PMC2494844">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2045.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4047536/figure/f1/" ref="ordinalpos=2045&amp;ncbi_uid=6241710&amp;link_uid=PMC4047536" href="/pmc/articles/PMC4047536/"><img src="/pmc/articles/instance/4047536/bin/srep05208-f1.gif" src-large="/pmc/articles/instance/4047536/bin/srep05208-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4047536/" ref="ordinalpos=2045&amp;ncbi_uid=6241710&amp;link_uid=PMC4047536" image-link="/pmc/articles/PMC4047536/figure/f1/" class="imagepopup">Figure 1. CART analysis of genetic variants in Ca2+/NFAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and KD risk..  From: The Roles of Ca2+/NFAT <span class="highlight" style="background-color:">Signaling</span> Genes in Kawasaki Disease: Single- and Multiple-Risk Genetic Variants. </a></p><div class="supp"><p class="details"><div>Cases and controls are denoted by white and black box respectively, and each node contains frequencies and percentages of cases and controls in each subgroup.</div></p></div><div class="aux"><div class="resc">Wei Wang, et al. Sci Rep. 2014;4:5208.</div><p class="links"><a class="dblinks" href="/pubmed/24903211" ref="ordinalpos=2045&amp;ncbi_uid=6241710&amp;link_uid=24903211">Citation</a><a class="dblinks" href="/pmc/articles/PMC4047536/" ref="ordinalpos=2045&amp;ncbi_uid=6241710&amp;link_uid=PMC4047536">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2046.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4298699/figure/fig8/" ref="ordinalpos=2046&amp;ncbi_uid=7111714&amp;link_uid=PMC4298699" href="/pmc/articles/PMC4298699/"><img src="/pmc/articles/instance/4298699/bin/elife04437f008.gif" src-large="/pmc/articles/instance/4298699/bin/elife04437f008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4298699/" ref="ordinalpos=2046&amp;ncbi_uid=7111714&amp;link_uid=PMC4298699" image-link="/pmc/articles/PMC4298699/figure/fig8/" class="imagepopup">Figure 8.A model for the role of EGFR in the establishment of epithelial polarity..  From: EGFR <span class="highlight" style="background-color:">signaling</span> promotes self-renewal through the establishment of cell polarity in Drosophila follicle stem cells. </a></p><div class="supp"><p class="details"><div>High levels of EGFR <span class="highlight" style="background-color:">signaling</span> in the FSC promote maintenance in the niche and the formation of basal and lateral domains while suppressing the formation of an apical domain. EGFR activates both the canonical Ras-mediated <span class="highlight" style="background-color:">pathway</span> leading to the phosphorylation of Erk, and the LKB1–AMPK <span class="highlight" style="background-color:">pathway</span>. Both Erk and AMPK are kinases that can regulate gene activity by activating transcription factors and phosphorylating proteins in the cytoplasm. AMPK directly promotes the lateral identity in polarized cells by activating lateral proteins. PKA is an upstream activator of LKB1 in follicle cells, and PKA can be activated by EGFR <span class="highlight" style="background-color:">signaling</span>, suggesting that EGFR <span class="highlight" style="background-color:">signaling</span> may activate LKB1 via PKA. EGFR <span class="highlight" style="background-color:">signaling</span> may suppress apical polarity either directly by regulating the transcription or activity of apical proteins, or indirectly by enhancing the activity of lateral proteins that suppress the localization of apical proteins. Low levels of EGFR <span class="highlight" style="background-color:">signaling</span> in prefollicle cells relieves this suppression, allowing apical domains to form and permitting differentiation away from the stem cell fate.<br /><b>DOI:</b> http://dx.doi.org/10.7554/eLife.04437.020</div></p></div><div class="aux"><div class="resc">Angela Castanieto, et al. eLife. 2014;3:e04437.</div><p class="links"><a class="dblinks" href="/pubmed/25437306" ref="ordinalpos=2046&amp;ncbi_uid=7111714&amp;link_uid=25437306">Citation</a><a class="dblinks" href="/pmc/articles/PMC4298699/" ref="ordinalpos=2046&amp;ncbi_uid=7111714&amp;link_uid=PMC4298699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2047.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3382591/figure/pone-0039666-g003/" ref="ordinalpos=2047&amp;ncbi_uid=4101703&amp;link_uid=PMC3382591" href="/pmc/articles/PMC3382591/"><img src="/pmc/articles/instance/3382591/bin/pone.0039666.g003.gif" src-large="/pmc/articles/instance/3382591/bin/pone.0039666.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3382591/" ref="ordinalpos=2047&amp;ncbi_uid=4101703&amp;link_uid=PMC3382591" image-link="/pmc/articles/PMC3382591/figure/pone-0039666-g003/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> enrichment in the MCN proteins..  From: A Minimal Connected Network of Transcription Factors Regulated in Human Tumors and Its Application to the Quest for Universal Cancer Biomarkers. </a></p><div class="supp"><p class="details"><div>All proteins (nodes) in the MCN were submitted to DAVID web tool for KEGG <span class="highlight" style="background-color:">pathway</span> enrichment analysis. Significant pathways are shown by categories according to the −log10(p-value) and the percentage of intersection between the submitted list and queried annotations.</div></p></div><div class="aux"><div class="resc">Ahmed Essaghir, et al. PLoS One. 2012;7(6):e39666.</div><p class="links"><a class="dblinks" href="/pubmed/22761861" ref="ordinalpos=2047&amp;ncbi_uid=4101703&amp;link_uid=22761861">Citation</a><a class="dblinks" href="/pmc/articles/PMC3382591/" ref="ordinalpos=2047&amp;ncbi_uid=4101703&amp;link_uid=PMC3382591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2048.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2077852/figure/F3/" ref="ordinalpos=2048&amp;ncbi_uid=1072301&amp;link_uid=PMC2077852" href="/pmc/articles/PMC2077852/"><img src="/pmc/articles/instance/2077852/bin/nihms32806f3.gif" src-large="/pmc/articles/instance/2077852/bin/nihms32806f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2077852/" ref="ordinalpos=2048&amp;ncbi_uid=1072301&amp;link_uid=PMC2077852" image-link="/pmc/articles/PMC2077852/figure/F3/" class="imagepopup">Figure 3.  From: Dmp1 and tumor suppression. </a></p><div class="supp"><p class="details"><div>Dmp1 links the Rb and p53 pathways. The novel ‘Jun-Dmp1’ <span class="highlight" style="background-color:">pathway</span> links oncogenic Ras-Raf <span class="highlight" style="background-color:">signaling</span> and the Arf-p53 <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> is independent of the classical cyclin D1/Cdk4-Rb-E2F <span class="highlight" style="background-color:">signaling</span>. E2Fs directly bind to the <i>Dmp1</i> promoter and causes repression; thus they have differential effects on the <i>Dmp1</i> and on the <i>Arf</i> promoter. Overexpression of D-cyclins inhibits the activity of Dmp1 in a Cdk-independent fashion; however, on the <i>Arf</i> promoter, Dmp1 and cyclin D1 act synergistically, and this activity is dependent on Cdks. Dmp1 might have other targets than <i>Arf</i>, especially in T lymphocytes.</div></p></div><div class="aux"><div class="resc">K Inoue, et al. Oncogene. ;26(30):4329-4335.</div><p class="links"><a class="dblinks" href="/pubmed/17237816" ref="ordinalpos=2048&amp;ncbi_uid=1072301&amp;link_uid=17237816">Citation</a><a class="dblinks" href="/pmc/articles/PMC2077852/" ref="ordinalpos=2048&amp;ncbi_uid=1072301&amp;link_uid=PMC2077852">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2049.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1949361/figure/F7/" ref="ordinalpos=2049&amp;ncbi_uid=1042593&amp;link_uid=PMC1949361" href="/pmc/articles/PMC1949361/"><img src="/pmc/articles/instance/1949361/bin/zmk0080781650007.gif" src-large="/pmc/articles/instance/1949361/bin/zmk0080781650007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1949361/" ref="ordinalpos=2049&amp;ncbi_uid=1042593&amp;link_uid=PMC1949361" image-link="/pmc/articles/PMC1949361/figure/F7/" class="imagepopup">Figure 7. From: The M-Ras-RA-GEF-2-Rap1 <span class="highlight" style="background-color:">Pathway</span> Mediates Tumor Necrosis Factor-?-dependent Regulation of Integrin Activation in Splenocytes. </a></p><div class="supp"><p class="details"><div>Presumptive <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. On binding to the receptor (1), TNF-α activates M-Ras (2). The activated M-Ras recruits RA-GEF-2 from the cytoplasm to the plasma membrane (3), and RA-GEF-2 activates Rap1 in the plasma membrane (4) to induce LFA-1 activation (5) and adherence to ICAM-1 (6).</div></p></div><div class="aux"><div class="resc">Yoko Yoshikawa, et al. Mol Biol Cell. 2007 August;18(8):2949-2959.</div><p class="links"><a class="dblinks" href="/pubmed/17538012" ref="ordinalpos=2049&amp;ncbi_uid=1042593&amp;link_uid=17538012">Citation</a><a class="dblinks" href="/pmc/articles/PMC1949361/" ref="ordinalpos=2049&amp;ncbi_uid=1042593&amp;link_uid=PMC1949361">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2050.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3217187/figure/F6/" ref="ordinalpos=2050&amp;ncbi_uid=3473348&amp;link_uid=PMC3217187" href="/pmc/articles/PMC3217187/"><img src="/pmc/articles/instance/3217187/bin/nihms322922f6.gif" src-large="/pmc/articles/instance/3217187/bin/nihms322922f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3217187/" ref="ordinalpos=2050&amp;ncbi_uid=3473348&amp;link_uid=PMC3217187" image-link="/pmc/articles/PMC3217187/figure/F6/" class="imagepopup">Figure 6. A model for Tao-1’s function in the HSW <span class="highlight" style="background-color:">pathway</span>.  From: Tao-1 Phosphorylates Hippo/MST kinases to Regulate the Hippo-Salvador-Warts Tumor Suppressor <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Tao-1 directly phosphorylates Hpo at T195 in the kinase activation loop, leading to the activation of Wts and inhibition of Yki, which remains cytoplasmic when phosphorylated. In the absence of HSW <span class="highlight" style="background-color:">pathway</span> activity, Yki translocates to the nucleus to promote transcription of target genes that promote growth and inhibit cell death. Genetic and biochemical experiments position Tao-1 upstream of Hpo and suggest that Mer and Ex function through Tao-1 to activate HSW <span class="highlight" style="background-color:">signaling</span>. Kibra, Mer and Ex can form a protein complex and promote HSW activation, but it remains unclear if Kibra functions upstream or in parallel to Tao-1.</div></p></div><div class="aux"><div class="resc">Julian C. Boggiano, et al. Dev Cell. ;21(5):888-895.</div><p class="links"><a class="dblinks" href="/pubmed/22075147" ref="ordinalpos=2050&amp;ncbi_uid=3473348&amp;link_uid=22075147">Citation</a><a class="dblinks" href="/pmc/articles/PMC3217187/" ref="ordinalpos=2050&amp;ncbi_uid=3473348&amp;link_uid=PMC3217187">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2051.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4085288/figure/Fig1/" ref="ordinalpos=2051&amp;ncbi_uid=6394387&amp;link_uid=PMC4085288" href="/pmc/articles/PMC4085288/"><img src="/pmc/articles/instance/4085288/bin/13238_2014_58_Fig1_HTML.gif" src-large="/pmc/articles/instance/4085288/bin/13238_2014_58_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4085288/" ref="ordinalpos=2051&amp;ncbi_uid=6394387&amp;link_uid=PMC4085288" image-link="/pmc/articles/PMC4085288/figure/Fig1/" class="imagepopup">Figure 1.  From: The regulation of TGF-?/SMAD <span class="highlight" style="background-color:">signaling</span> by protein deubiquitination. </a></p><div class="supp"><p class="details"><div> <b>A schematic representation of the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Upon ligand binding, the TGF-β type II receptor (TβRII) recruits and phosphorylates the type I receptor (TβRI) in the cell membrane, and then the activated type II-I heteromeric receptor complex can induce phosphorylation of R-SMADs. Activated R-SMADs can recruit Co-SMAD (SMAD4) for binding. The R-SMAD/Co-SMAD complexes translocate and accumulate in the nucleus and then initiate the expression of the target transcription factors with the help of other co-factors. TGF-β can in addition promote other intracellular (non-SMAD) <span class="highlight" style="background-color:">signaling</span> pathways, such as mitogen activated protein kinases (MAPKs) pathways, the phosphoinositide 3-kinase (PI3K)/AKT <span class="highlight" style="background-color:">pathway</span>, the nuclear factor κB (NF-κB) <span class="highlight" style="background-color:">pathway</span>, and the Rho-like GTPase <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Juan Zhang, et al. Protein Cell. 2014 July;5(7):503-517.</div><p class="links"><a class="dblinks" href="/pubmed/24756567" ref="ordinalpos=2051&amp;ncbi_uid=6394387&amp;link_uid=24756567">Citation</a><a class="dblinks" href="/pmc/articles/PMC4085288/" ref="ordinalpos=2051&amp;ncbi_uid=6394387&amp;link_uid=PMC4085288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2052.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4020352/figure/F3/" ref="ordinalpos=2052&amp;ncbi_uid=6132096&amp;link_uid=PMC4020352" href="/pmc/articles/PMC4020352/"><img src="/pmc/articles/instance/4020352/bin/nihms575521f3.gif" src-large="/pmc/articles/instance/4020352/bin/nihms575521f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4020352/" ref="ordinalpos=2052&amp;ncbi_uid=6132096&amp;link_uid=PMC4020352" image-link="/pmc/articles/PMC4020352/figure/F3/" class="imagepopup">Figure 3. Antagonistic effects of sphingosine-1 phosphate and dihydrosphingosine-1 phosphate on the transforming growth factor β/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Noncanonical transforming growth factor ? <span class="highlight" style="background-color:">signaling</span> in scleroderma fibrosis. </a></p><div class="supp"><p class="details"><div>Sphingosine-1 phosphate (S1P) treatment reproduces profibrotic effects of transforming growth factor β (TGF-β) through the enhancement of the endogenous TGF-β ligand/receptor <span class="highlight" style="background-color:">signaling</span> (Bu and Trojanowska, unpublished data), but the specific details of this process remain to be clarified. In contrast, dihydrosphingosine-1 phosphate (dhS1P) suppresses TGF-β <span class="highlight" style="background-color:">signaling</span> through the TGF-β receptor-independent, PTEN-dependent mechanism [41]. DhS1P treatment triggers rapid phosphorylation of PTEN and its subsequent translocation to the nucleus, in which PTEN facilitates Smad2/3 dephosphorylation by stabilizing Smad1/2 phosphatase, PPM1A. Interestingly, both sphingolipid mediators utilize the same receptor, S1P<sub>1</sub>, suggesting that additional, presently unknown components of these pathways contribute to the distinct postreceptor <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Maria Trojanowska. Curr Opin Rheumatol. 2009 November;21(6):623-629.</div><p class="links"><a class="dblinks" href="/pubmed/19713852" ref="ordinalpos=2052&amp;ncbi_uid=6132096&amp;link_uid=19713852">Citation</a><a class="dblinks" href="/pmc/articles/PMC4020352/" ref="ordinalpos=2052&amp;ncbi_uid=6132096&amp;link_uid=PMC4020352">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2053.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3638297/figure/F3/" ref="ordinalpos=2053&amp;ncbi_uid=4835507&amp;link_uid=PMC3638297" href="/pmc/articles/PMC3638297/"><img src="/pmc/articles/instance/3638297/bin/ijmsv10p0738g03.gif" src-large="/pmc/articles/instance/3638297/bin/ijmsv10p0738g03.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3638297/" ref="ordinalpos=2053&amp;ncbi_uid=4835507&amp;link_uid=PMC3638297" image-link="/pmc/articles/PMC3638297/figure/F3/" class="imagepopup">Figure 3.  From: Lef1 Contributes to the Differentiation of Bulge Stem Cells by Nuclear Translocation and Cross-Talk with the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Lef1 promoted cell differentiation via Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Lef1 mRNA expression level measured by realtime PCR (*<i>p</i>&lt;0.05). (B) The cytoplasmic and nuclear expression of Lef1 and β-catenin in the bulge stem cells treated with DP or not. (C) Immunofluorescence analysis on the location of Lef1 in those cells. Scale bars: 20μm. (D) The transcriptional activity of Wnt <span class="highlight" style="background-color:">signaling</span> was determined by luciferase reporter assay. (*<i>p</i>&lt;0.05). Data shown here are representative for three experiments.</div></p></div><div class="aux"><div class="resc">Yi Zhang, et al. Int J Med Sci. 2013;10(6):738-746.</div><p class="links"><a class="dblinks" href="/pubmed/23630438" ref="ordinalpos=2053&amp;ncbi_uid=4835507&amp;link_uid=23630438">Citation</a><a class="dblinks" href="/pmc/articles/PMC3638297/" ref="ordinalpos=2053&amp;ncbi_uid=4835507&amp;link_uid=PMC3638297">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2054.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3119514/figure/F4/" ref="ordinalpos=2054&amp;ncbi_uid=3462284&amp;link_uid=PMC3119514" href="/pmc/articles/PMC3119514/"><img src="/pmc/articles/instance/3119514/bin/nihms291316f4.gif" src-large="/pmc/articles/instance/3119514/bin/nihms291316f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3119514/" ref="ordinalpos=2054&amp;ncbi_uid=3462284&amp;link_uid=PMC3119514" image-link="/pmc/articles/PMC3119514/figure/F4/" class="imagepopup">Fig. 4.  From: Genomics of the NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: hypothesized role in ovarian cancer. </a></p><div class="supp"><p class="details"><div>Non-canonical activation pathways. Receptor binding of LTβ, BAFF, CD40, and RANK initiate the non-canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TRAF proteins and other intracellular target molecules are recruited, eventually leading to the activation of NIK through an unknown mechanism. NIK contains binding domains that interact with TRAF2, TRAF3, TRAF5, and TRAF6, but the specific roles of TRAF and other proteins in the non-canonical <span class="highlight" style="background-color:">pathway</span> is still controversial. After NIK phosphorylation, IKKα phosphorylates p100, marking it for poly-ubiquitination and partial degradation by the SCF<sup>β-TrCP</sup> complex and 26S proteasome. Partial degradation of p100 yields functional p52, which most commonly forms heterodimers with Rel-B and translocates to the nucleus to initiate gene transcription</div></p></div><div class="aux"><div class="resc">Kristin L. White, et al. Cancer Causes Control. ;22(5):785-801.</div><p class="links"><a class="dblinks" href="/pubmed/21359843" ref="ordinalpos=2054&amp;ncbi_uid=3462284&amp;link_uid=21359843">Citation</a><a class="dblinks" href="/pmc/articles/PMC3119514/" ref="ordinalpos=2054&amp;ncbi_uid=3462284&amp;link_uid=PMC3119514">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2055.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1303061/figure/fig9/" ref="ordinalpos=2055&amp;ncbi_uid=638255&amp;link_uid=PMC1303061" href="/pmc/articles/PMC1303061/"><img src="/pmc/articles/instance/1303061/bin/biophysj19414f09.gif" src-large="/pmc/articles/instance/1303061/bin/biophysj19414f09.jpg" alt="FIGURE 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1303061/" ref="ordinalpos=2055&amp;ncbi_uid=638255&amp;link_uid=PMC1303061" image-link="/pmc/articles/PMC1303061/figure/fig9/" class="imagepopup">FIGURE 9.  From: The Convex Basis of the Left Null Space of the Stoichiometric Matrix Leads to the Definition of Metabolically Meaningful Pools. </a></p><div class="supp"><p class="details"><div>Schematic representation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (e.g., a tyrosine kinase-type <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) and its corresponding conservation pools. (<i>A</i>) Reaction map of the sample tyrosine kinase <span class="highlight" style="background-color:">pathway</span>. (<i>B</i>) Compound map of the tyrosine kinase <span class="highlight" style="background-color:">pathway</span>. (<i>C</i>) Metabolite pool maps of conservation pools. (L, ligand; R, receptor; LR, ligand-receptor complex; LR-P, phosphorylated ligand-receptor complex; S, signal transducers and activators of transcription (STAT); SP, phosphorylated STAT; T, transcriptional factor; TP, phosphorylated transcriptional factor; DNA-TP, active DNA-transcriptional factor complex; P, inorganic phosphate; act, active; inact, inactive).</div></p></div><div class="aux"><div class="resc">Iman Famili, et al. Biophys J. 2003 July;85(1):16-26.</div><p class="links"><a class="dblinks" href="/pubmed/12829460" ref="ordinalpos=2055&amp;ncbi_uid=638255&amp;link_uid=12829460">Citation</a><a class="dblinks" href="/pmc/articles/PMC1303061/" ref="ordinalpos=2055&amp;ncbi_uid=638255&amp;link_uid=PMC1303061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2056.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193412/figure/fig2/" ref="ordinalpos=2056&amp;ncbi_uid=3770420&amp;link_uid=PMC3193412" href="/pmc/articles/PMC3193412/"><img src="/pmc/articles/instance/3193412/bin/cr2010161f2.gif" src-large="/pmc/articles/instance/3193412/bin/cr2010161f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193412/" ref="ordinalpos=2056&amp;ncbi_uid=3770420&amp;link_uid=PMC3193412" image-link="/pmc/articles/PMC3193412/figure/fig2/" class="imagepopup">Figure 2.  From: A single NF?B system for both canonical and non-canonical <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The NFκB <span class="highlight" style="background-color:">signaling</span> module consisting of canonical and non-canonical pathways. The non-canonical <span class="highlight" style="background-color:">pathway</span> is activated through developmental signals activating NIK/IKK1. This activation results in the degradation of IκBδ and processing of p100 and allows the nuclear translocation of RelA:p50, RelB:p50 and RelB:p52 dimers, which activate genes responsible for organ development. The canonical <span class="highlight" style="background-color:">pathway</span> is activated through pathogen and inflammatory signals activating NEMO/IKK. This activation results in the degradation of IκBα/IκBβ/IκBɛ, allowing for the nuclear translocation of RelA:p50, RelA:RelA and cRel:p50 dimers, which then activate genes responsible for inflammation and survival.</div></p></div><div class="aux"><div class="resc">Vincent Feng-Sheng Shih, et al. Cell Res. 2011 January;21(1):86-102.</div><p class="links"><a class="dblinks" href="/pubmed/21102550" ref="ordinalpos=2056&amp;ncbi_uid=3770420&amp;link_uid=21102550">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193412/" ref="ordinalpos=2056&amp;ncbi_uid=3770420&amp;link_uid=PMC3193412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2057.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2169177/figure/f10/" ref="ordinalpos=2057&amp;ncbi_uid=1095986&amp;link_uid=PMC2169177" href="/pmc/articles/PMC2169177/"><img src="/pmc/articles/instance/2169177/bin/zmb0230771550010.gif" src-large="/pmc/articles/instance/2169177/bin/zmb0230771550010.jpg" alt="FIG. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2169177/" ref="ordinalpos=2057&amp;ncbi_uid=1095986&amp;link_uid=PMC2169177" image-link="/pmc/articles/PMC2169177/figure/f10/" class="imagepopup">FIG. 10. From: RasGRF2, a Guanosine Nucleotide Exchange Factor for Ras GTPases, Participates in T-Cell <span class="highlight" style="background-color:">Signaling</span> Responses . </a></p><div class="supp"><p class="details"><div>A schematic view of Ras- and calcineurin-dependent pathways in T cells. The known components of the combined <span class="highlight" style="background-color:">pathway</span> are shown in black. The new <span class="highlight" style="background-color:">pathway</span> described in this work is shown in blue. The main second messengers are indicated in green. Negative regulators of T-cell <span class="highlight" style="background-color:">signaling</span> operating in this <span class="highlight" style="background-color:">pathway</span> are shown in red. Thicker arrows highlight the prevalence of the Vav1/RasGRP1 <span class="highlight" style="background-color:">pathway</span> over the rest of <span class="highlight" style="background-color:">signaling</span> connections presented in this figure. For the sake of simplicity, we have not included adaptor proteins and kinases involved in the <span class="highlight" style="background-color:">pathway</span>. Likewise, we have omitted other transcriptional factors that cooperate in the transcription of NF-AT-regulated genes. The direct connection between Rac1 and RasGRP1 is based on previous observations indicating that the polymerization of F-actin by Rac1 contributes to the translocation of RasGRP family members to the plasma membrane (11, 12). Other regulatory points are discussed in the main text.</div></p></div><div class="aux"><div class="resc">Sergio Ruiz, et al. Mol Cell Biol. 2007 December;27(23):8127-8142.</div><p class="links"><a class="dblinks" href="/pubmed/17923690" ref="ordinalpos=2057&amp;ncbi_uid=1095986&amp;link_uid=17923690">Citation</a><a class="dblinks" href="/pmc/articles/PMC2169177/" ref="ordinalpos=2057&amp;ncbi_uid=1095986&amp;link_uid=PMC2169177">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2058.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262399/figure/F6/" ref="ordinalpos=2058&amp;ncbi_uid=3873935&amp;link_uid=PMC3262399" href="/pmc/articles/PMC3262399/"><img src="/pmc/articles/instance/3262399/bin/nihms347394f6.gif" src-large="/pmc/articles/instance/3262399/bin/nihms347394f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262399/" ref="ordinalpos=2058&amp;ncbi_uid=3873935&amp;link_uid=PMC3262399" image-link="/pmc/articles/PMC3262399/figure/F6/" class="imagepopup">Figure 6. Model of Hedgehog-Induced Thermal Allodynia and Hyperalgesia.  From: Hedgehog <span class="highlight" style="background-color:">Signaling</span> Regulates Nociceptive Sensitization. </a></p><div class="supp"><p class="details"><div>Hh (from an as yet undetermined source tissue) acts through Patched (whose overexpression blocks sensitization), which subsequently inhibits Smoothened (required for sensitization) to activate signal transduction. The subsequent <span class="highlight" style="background-color:">signaling</span> steps likely involve the transcriptional activity of Cubitus interruptus (required for sensitization) and other components of the canonical Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. For thermal allodynia, this <span class="highlight" style="background-color:">pathway</span> acts through the transient receptor potential (TRP) channel Painless, and for thermal hyperalgesia it requires the TRP channel dTRPA1.</div></p></div><div class="aux"><div class="resc">Daniel T. Babcock, et al. Curr Biol. ;21(18):1525-1533.</div><p class="links"><a class="dblinks" href="/pubmed/21906949" ref="ordinalpos=2058&amp;ncbi_uid=3873935&amp;link_uid=21906949">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262399/" ref="ordinalpos=2058&amp;ncbi_uid=3873935&amp;link_uid=PMC3262399">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2059.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2820430/figure/F8/" ref="ordinalpos=2059&amp;ncbi_uid=2227559&amp;link_uid=PMC2820430" href="/pmc/articles/PMC2820430/"><img src="/pmc/articles/instance/2820430/bin/zmk0041093560008.gif" src-large="/pmc/articles/instance/2820430/bin/zmk0041093560008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2820430/" ref="ordinalpos=2059&amp;ncbi_uid=2227559&amp;link_uid=PMC2820430" image-link="/pmc/articles/PMC2820430/figure/F8/" class="imagepopup">Figure 8. From: The Cell Surface Protein Gene ecm33+ Is a Target of the Two Transcription Factors Atf1 and Mbx1 and Negatively Regulates Pmk1 MAPK Cell Integrity <span class="highlight" style="background-color:">Signaling</span> in Fission Yeast. </a></p><div class="supp"><p class="details"><div>A model for the dual regulation of <i>ecm33</i><sup>+</sup> gene expression and its putative involvement in the negative feedback regulation of the Pmk1 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Regulatory elements in the promoter are schematically represented. Dotted lines denote hitherto uncharacterized processes. Ecm33 may exert its effect on Ca<sup>2+</sup>/calcineurin <span class="highlight" style="background-color:">signaling</span> largely via Pmk1 <span class="highlight" style="background-color:">pathway</span> and partly via other Ca<sup>2+</sup>-influx machineries.</div></p></div><div class="aux"><div class="resc">Hirofumi Takada, et al. Mol Biol Cell. 2010 February 15;21(4):674-685.</div><p class="links"><a class="dblinks" href="/pubmed/20032302" ref="ordinalpos=2059&amp;ncbi_uid=2227559&amp;link_uid=20032302">Citation</a><a class="dblinks" href="/pmc/articles/PMC2820430/" ref="ordinalpos=2059&amp;ncbi_uid=2227559&amp;link_uid=PMC2820430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2060.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3248020/figure/F9/" ref="ordinalpos=2060&amp;ncbi_uid=4469786&amp;link_uid=PMC3248020" href="/pmc/articles/PMC3248020/"><img src="/pmc/articles/instance/3248020/bin/zbc0521189810009.gif" src-large="/pmc/articles/instance/3248020/bin/zbc0521189810009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3248020/" ref="ordinalpos=2060&amp;ncbi_uid=4469786&amp;link_uid=PMC3248020" image-link="/pmc/articles/PMC3248020/figure/F9/" class="imagepopup">FIGURE 9. From: Tumor-suppressive Effects of Psoriasin (S100A7) Are Mediated through the ?-Catenin/T Cell Factor 4 Protein <span class="highlight" style="background-color:">Pathway</span> in Estrogen Receptor-positive Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of S100A7-mediated <span class="highlight" style="background-color:">signaling</span> that inhibits proliferation and motility in ERα<sup>+</sup> breast cancer.</b> S100A7 overexpression in ERα<sup>+</sup> cells may down-regulate the β-catenin/TCF4 <span class="highlight" style="background-color:">pathway</span> by regulating the expression and phosphorylation of GSK3β. The β-catenin/TCF4 <span class="highlight" style="background-color:">pathway</span> may in turn down-regulate the expression of downstream targets such as cyclin D1 and c-myc in S100A7-overexpressing ERα<sup>+</sup> cells. Therefore, the β-catenin/TCF4 <span class="highlight" style="background-color:">pathway</span> may be responsible for decreased proliferation and tumor growth in S100A7-overexpressing cells. In addition, S100A7 may modulate migration through enhanced co-localization and interaction of β-catenin and E-cadherin in the membranes of ERα<sup>+</sup> cells.</div></p></div><div class="aux"><div class="resc">Yadwinder S. Deol, et al. J Biol Chem. 2011 December 30;286(52):44845-44854.</div><p class="links"><a class="dblinks" href="/pubmed/22016394" ref="ordinalpos=2060&amp;ncbi_uid=4469786&amp;link_uid=22016394">Citation</a><a class="dblinks" href="/pmc/articles/PMC3248020/" ref="ordinalpos=2060&amp;ncbi_uid=4469786&amp;link_uid=PMC3248020">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2061.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4213501/figure/Fig4/" ref="ordinalpos=2061&amp;ncbi_uid=6769863&amp;link_uid=PMC4213501" href="/pmc/articles/PMC4213501/"><img src="/pmc/articles/instance/4213501/bin/12864_2014_6609_Fig4_HTML.gif" src-large="/pmc/articles/instance/4213501/bin/12864_2014_6609_Fig4_HTML.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4213501/" ref="ordinalpos=2061&amp;ncbi_uid=6769863&amp;link_uid=PMC4213501" image-link="/pmc/articles/PMC4213501/figure/Fig4/" class="imagepopup">Figure 4.  From: Differential transcriptomic analyses revealed genes and <span class="highlight" style="background-color:">signaling</span> pathways involved in iono-osmoregulation and cellular remodeling in the gills of euryhaline Mozambique tilapia, Oreochromis mossambicus. </a></p><div class="supp"><p class="details"><div> <b>Canonical <span class="highlight" style="background-color:">signaling</span> pathways that were significantly (Benjamini-Hochberg adjusted P-value &lt; 0.05) enriched with putative human homolog proteins encoded by the assembled contiq sequences used in the analysis.</b> Proportion (%) of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> being represented by the human homolog encoded proteins is been indicated on the x-axis. Refer to Additional file for complete list.</div></p></div><div class="aux"><div class="resc">Siew Hong Lam, et al. BMC Genomics. 2014;15(1):921.</div><p class="links"><a class="dblinks" href="/pubmed/25342237" ref="ordinalpos=2061&amp;ncbi_uid=6769863&amp;link_uid=25342237">Citation</a><a class="dblinks" href="/pmc/articles/PMC4213501/" ref="ordinalpos=2061&amp;ncbi_uid=6769863&amp;link_uid=PMC4213501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2062.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3289338/figure/fig03/" ref="ordinalpos=2062&amp;ncbi_uid=4174318&amp;link_uid=PMC3289338" href="/pmc/articles/PMC3289338/"><img src="/pmc/articles/instance/3289338/bin/pnas.1116582109fig03.gif" src-large="/pmc/articles/instance/3289338/bin/pnas.1116582109fig03.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3289338/" ref="ordinalpos=2062&amp;ncbi_uid=4174318&amp;link_uid=PMC3289338" image-link="/pmc/articles/PMC3289338/figure/fig03/" class="imagepopup">Fig. 3. From: Structural reorganization of the interleukin-7 <span class="highlight" style="background-color:">signaling</span> complex. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> mechanism of the IL-7 <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Signaling</span> model of IL-7Rα–IL-7–γ<sub>c</sub> interactions indicating preassembly of the IL-7Rα homodimer and IL-7Rα–γ<sub>c</sub> heterodimer from cell biological studies (12, 13). There are no structures of the IL-7Rα–γ<sub>c</sub> heterodimer or the IL-7Rα–IL-7–γ<sub>c</sub> ternary complex. The distance between the D2 domains of IL-7Rα and γ<sub>c</sub> was an average of the γ<sub>c</sub> structures with IL-2 [PDB ID code 2bfi (20); PDB ID code 2erj (29)] and IL-4 [PDB ID code 3bpl (23)].</div></p></div><div class="aux"><div class="resc">Craig A. McElroy, et al. Proc Natl Acad Sci U S A. 2012 February 14;109(7):2503-2508.</div><p class="links"><a class="dblinks" href="/pubmed/22308406" ref="ordinalpos=2062&amp;ncbi_uid=4174318&amp;link_uid=22308406">Citation</a><a class="dblinks" href="/pmc/articles/PMC3289338/" ref="ordinalpos=2062&amp;ncbi_uid=4174318&amp;link_uid=PMC3289338">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2063.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2826040/figure/f5/" ref="ordinalpos=2063&amp;ncbi_uid=2443981&amp;link_uid=PMC2826040" href="/pmc/articles/PMC2826040/"><img src="/pmc/articles/instance/2826040/bin/zjv9990930120005.gif" src-large="/pmc/articles/instance/2826040/bin/zjv9990930120005.jpg" alt="FIG. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2826040/" ref="ordinalpos=2063&amp;ncbi_uid=2443981&amp;link_uid=PMC2826040" image-link="/pmc/articles/PMC2826040/figure/f5/" class="imagepopup">FIG. 5. From: MicroRNA Expression and Virulence in Pandemic Influenza Virus-Infected Mice  . </a></p><div class="supp"><p class="details"><div>miR-223 is implicated in the CREB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (Top) CREB <span class="highlight" style="background-color:">pathway</span> comprised of miR-223 targets; (bottom) expression changes of miR-223 and its targets associated with the CREB <span class="highlight" style="background-color:">pathway</span> in r1918-infected samples. Red represents microRNA or targets that have increased expression in r1918-infected samples, relative to Tx/91-infected samples. Green represents microRNA or targets that have decreased expression in r1918-infected samples, relative to Tx/91-infected samples.</div></p></div><div class="aux"><div class="resc">Yu Li, et al. J Virol. 2010 March;84(6):3023-3032.</div><p class="links"><a class="dblinks" href="/pubmed/20071585" ref="ordinalpos=2063&amp;ncbi_uid=2443981&amp;link_uid=20071585">Citation</a><a class="dblinks" href="/pmc/articles/PMC2826040/" ref="ordinalpos=2063&amp;ncbi_uid=2443981&amp;link_uid=PMC2826040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2064.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC99013/figure/F13/" ref="ordinalpos=2064&amp;ncbi_uid=52551&amp;link_uid=PMC99013" href="/pmc/articles/PMC99013/"><img src="/pmc/articles/instance/99013/bin/mr0400030013.gif" src-large="/pmc/articles/instance/99013/bin/mr0400030013.jpg" alt="FIG. 13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC99013/" ref="ordinalpos=2064&amp;ncbi_uid=52551&amp;link_uid=PMC99013" image-link="/pmc/articles/PMC99013/figure/F13/" class="imagepopup">FIG. 13.  From: Signal Transduction Cascades Regulating Fungal  Development and Virulence. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> networks in <i>U. maydis</i>. Mating, filamentation, and virulence in the smut fungus <i>U. maydis</i> are regulated by a pheromone response <span class="highlight" style="background-color:">pathway</span> (left) and the cAMP-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> acting through PKA (Ubc1, Adr1, and Uka1) (right).</div></p></div><div class="aux"><div class="resc">Klaus B. Lengeler, et al. Microbiol Mol Biol Rev. 2000 December;64(4):746-785.</div><p class="links"><a class="dblinks" href="/pubmed/11104818" ref="ordinalpos=2064&amp;ncbi_uid=52551&amp;link_uid=11104818">Citation</a><a class="dblinks" href="/pmc/articles/PMC99013/" ref="ordinalpos=2064&amp;ncbi_uid=52551&amp;link_uid=PMC99013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2065.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3897773/figure/pone-0087014-g007/" ref="ordinalpos=2065&amp;ncbi_uid=5736478&amp;link_uid=PMC3897773" href="/pmc/articles/PMC3897773/"><img src="/pmc/articles/instance/3897773/bin/pone.0087014.g007.gif" src-large="/pmc/articles/instance/3897773/bin/pone.0087014.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3897773/" ref="ordinalpos=2065&amp;ncbi_uid=5736478&amp;link_uid=PMC3897773" image-link="/pmc/articles/PMC3897773/figure/pone-0087014-g007/" class="imagepopup">Figure 7. Schematic representation of the proposed 5-HT4d receptor-stimulated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to increased sAPPα production..  From: Regulation of Amyloid Precursor Protein Processing by Serotonin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The proteins involved in 5-HT<sub>4d</sub> receptor-mediated non-amyloidogenic APP shedding are shown with green circles while orange circles and red characters indicate proteins or second messengers that were tested but were ineffective in modulating 5-HT<sub>4d</sub> receptor-stimulated sAPPα release. The dotted lines with the question marks indicate remaining areas of investigation for further elucidation of the molecular mechanism of α-secretase activation. cAMP-dependent <span class="highlight" style="background-color:">pathway</span> of α-secretase induction was previously reported and is depicted as a plausible way for 5-HT<sub>4d</sub> receptor-mediated sAPPα release [15].</div></p></div><div class="aux"><div class="resc">Anna A. Pimenova, et al. PLoS One. 2014;9(1):e87014.</div><p class="links"><a class="dblinks" href="/pubmed/24466315" ref="ordinalpos=2065&amp;ncbi_uid=5736478&amp;link_uid=24466315">Citation</a><a class="dblinks" href="/pmc/articles/PMC3897773/" ref="ordinalpos=2065&amp;ncbi_uid=5736478&amp;link_uid=PMC3897773">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2066.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3674400/figure/pct042-F5/" ref="ordinalpos=2066&amp;ncbi_uid=6207913&amp;link_uid=PMC3674400" href="/pmc/articles/PMC3674400/"><img src="/pmc/articles/instance/3674400/bin/pct042f5p.gif" src-large="/pmc/articles/instance/3674400/bin/pct042f5p.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3674400/" ref="ordinalpos=2066&amp;ncbi_uid=6207913&amp;link_uid=PMC3674400" image-link="/pmc/articles/PMC3674400/figure/pct042-F5/" class="imagepopup">Fig. 5.  From: Three Transcription Factors, HFR1, LAF1 and HY5, Regulate Largely Independent <span class="highlight" style="background-color:">Signaling</span> Pathways Downstream of Phytochrome A. </a></p><div class="supp"><p class="details"><div>Schematic diagram of proposed phyA <span class="highlight" style="background-color:">signaling</span>. (1) Upon FRc exposure, phyA localizes from the cytosol into the nucleus through direct interaction with FHY1/FHL (Genoud et al. 2008). (2) In the nucleus, the phyA signal is transmitted to two major transcription factors (HFR1 and LAF1) through FHY1/FHL to promote photomorphogenesis (Yang et al. 2009). However, FHY1/FHL does not directly transmit phyA signal to one of the major transcription factors, HY5. The unidentified factor X may be involved in the FHY1/FHL-dependent <span class="highlight" style="background-color:">pathway</span>. (3) Direct interaction between HY5 and HFR1 or LAF1 may modulate transcription factor abundance and hence <span class="highlight" style="background-color:">signaling</span> strength. Taken together, we proposed here that HY5 may share the FHY1/FHL signal through interactions with HFR1 and/or LAF1 (4). Alternatively, HY5 may transmit phyA signals through a FHY1/FHL-independent <span class="highlight" style="background-color:">pathway</span> (5). Gray arrows represent the direction of the phyA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">In-Cheol Jang, et al. Plant Cell Physiol. 2013 June;54(6):907-916.</div><p class="links"><a class="dblinks" href="/pubmed/23503597" ref="ordinalpos=2066&amp;ncbi_uid=6207913&amp;link_uid=23503597">Citation</a><a class="dblinks" href="/pmc/articles/PMC3674400/" ref="ordinalpos=2066&amp;ncbi_uid=6207913&amp;link_uid=PMC3674400">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2067.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404301/figure/F1/" ref="ordinalpos=2067&amp;ncbi_uid=5145806&amp;link_uid=PMC3404301" href="/pmc/articles/PMC3404301/"><img src="/pmc/articles/instance/3404301/bin/zmg0081250450001.gif" src-large="/pmc/articles/instance/3404301/bin/zmg0081250450001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404301/" ref="ordinalpos=2067&amp;ncbi_uid=5145806&amp;link_uid=PMC3404301" image-link="/pmc/articles/PMC3404301/figure/F1/" class="imagepopup">Fig. 1. From: COUP-TFII Is a Major Regulator of Cell Cycle and Notch <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Up-regulation of members of the cell cycle <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the COUP-TFII knockdown HUVEC cells. A, Heat map of microarray analysis of members of the cell cycle <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HUVEC cells treated with COUP-TFII-specific siRNA or scrambled control siRNA. B, qRT-PCR analysis of CyclinA2, Cyclin B1, Cyclin B2, Cyclin E1, Cyclin E2, CDC6, CDC25A, SRC-3, E2F1, and COUP-TFII expression in control and COUP-TFII-depleted HUVEC cells. <i>Error bars</i> indicate <span style="font-variant: small-caps" class="small-caps">sem</span>. *, <i>P</i> &lt; 0.05; **, <i>P</i> &lt; 0.005; ***, <i>P</i> &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Xinpu Chen, et al. Mol Endocrinol. 2012 August;26(8):1268-1277.</div><p class="links"><a class="dblinks" href="/pubmed/22734039" ref="ordinalpos=2067&amp;ncbi_uid=5145806&amp;link_uid=22734039">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404301/" ref="ordinalpos=2067&amp;ncbi_uid=5145806&amp;link_uid=PMC3404301">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2068.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3646677/figure/F1/" ref="ordinalpos=2068&amp;ncbi_uid=4884348&amp;link_uid=PMC3646677" href="/pmc/articles/PMC3646677/"><img src="/pmc/articles/instance/3646677/bin/1756-8722-6-30-1.gif" src-large="/pmc/articles/instance/3646677/bin/1756-8722-6-30-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3646677/" ref="ordinalpos=2068&amp;ncbi_uid=4884348&amp;link_uid=PMC3646677" image-link="/pmc/articles/PMC3646677/figure/F1/" class="imagepopup">Figure 1.  From: B-Raf and the inhibitors: from bench to bedside. </a></p><div class="supp"><p class="details"><div><b>B-Raf protein and <span class="highlight" style="background-color:">signaling</span> pathways.</b> The B-Raf protein and its related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown along with potential targets for treatment. <b>A</b>) The PI3K/AKT/mTOR and Ras/Raf/MAPK <span class="highlight" style="background-color:">signaling</span> pathways are shown along with potential targets. <b>B</b>) The structural domains of the B-Raf isoforms are shown. The position of the V600E mutation is indicated (arrow).</div></p></div><div class="aux"><div class="resc">Tiangui Huang, et al. J Hematol Oncol. 2013;6:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/23617957" ref="ordinalpos=2068&amp;ncbi_uid=4884348&amp;link_uid=23617957">Citation</a><a class="dblinks" href="/pmc/articles/PMC3646677/" ref="ordinalpos=2068&amp;ncbi_uid=4884348&amp;link_uid=PMC3646677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2069.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4064313/figure/F2/" ref="ordinalpos=2069&amp;ncbi_uid=6274577&amp;link_uid=PMC4064313" href="/pmc/articles/PMC4064313/"><img src="/pmc/articles/instance/4064313/bin/gm430-2.gif" src-large="/pmc/articles/instance/4064313/bin/gm430-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4064313/" ref="ordinalpos=2069&amp;ncbi_uid=6274577&amp;link_uid=PMC4064313" image-link="/pmc/articles/PMC4064313/figure/F2/" class="imagepopup">Figure 2.  From: Pancreatic cancer genomics: insights and opportunities for clinical translation. </a></p><div class="supp"><p class="details"><div><b>Core <span class="highlight" style="background-color:">signaling</span> pathways in pancreatic cancer</b>. The pathways and processes whose component genes are genetically altered in most pancreatic cancers based on whole-exome sequencing are shown. Although some genes may correspond to a single <span class="highlight" style="background-color:">pathway</span> (for example, <i>KRAS2 </i>mutations and the KRAS <span class="highlight" style="background-color:">signaling</span> core <span class="highlight" style="background-color:">pathway</span>) others may have a role in more than one <span class="highlight" style="background-color:">pathway</span> (for example, <i>TP53 </i>mutations and the apoptosis, DNA damage, and JNK core <span class="highlight" style="background-color:">signaling</span> pathways). Therapeutic targeting of one or more of these pathways, rather than specific gene alterations that occur within a <span class="highlight" style="background-color:">pathway</span>, provides a new way of treating pancreatic cancer. TGF-β, transforming growth factor β.</div></p></div><div class="aux"><div class="resc">Alvin Makohon-Moore, et al. Genome Med. 2013;5(3):26-26.</div><p class="links"><a class="dblinks" href="/pubmed/23673020" ref="ordinalpos=2069&amp;ncbi_uid=6274577&amp;link_uid=23673020">Citation</a><a class="dblinks" href="/pmc/articles/PMC4064313/" ref="ordinalpos=2069&amp;ncbi_uid=6274577&amp;link_uid=PMC4064313">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2070.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3497784/figure/fig01/" ref="ordinalpos=2070&amp;ncbi_uid=4836261&amp;link_uid=PMC3497784" href="/pmc/articles/PMC3497784/"><img src="/pmc/articles/instance/3497784/bin/pnas.1208396109fig01.gif" src-large="/pmc/articles/instance/3497784/bin/pnas.1208396109fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3497784/" ref="ordinalpos=2070&amp;ncbi_uid=4836261&amp;link_uid=PMC3497784" image-link="/pmc/articles/PMC3497784/figure/fig01/" class="imagepopup">Fig. 1. From: Inhibition of oncogenic Wnt <span class="highlight" style="background-color:">signaling</span> through direct targeting of ?-catenin. </a></p><div class="supp"><p class="details"><div>Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. (<i>A</i>) The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with its major components in the inactive (<i>Left</i>) and active (<i>Right</i>) state. BCL9 and CBP are transcriptional coactivators. (<i>B</i>) Superimposed crystal structures of the β-catenin armadillo domain bound to the CBDs of Axin (PDB ID 1QZ7) (18) and TCF4 (PDB ID 2GL7) (17).</div></p></div><div class="aux"><div class="resc">Tom N. Grossmann, et al. Proc Natl Acad Sci U S A. 2012 October 30;109(44):17942-17947.</div><p class="links"><a class="dblinks" href="/pubmed/23071338" ref="ordinalpos=2070&amp;ncbi_uid=4836261&amp;link_uid=23071338">Citation</a><a class="dblinks" href="/pmc/articles/PMC3497784/" ref="ordinalpos=2070&amp;ncbi_uid=4836261&amp;link_uid=PMC3497784">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2071.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3682211/figure/F7/" ref="ordinalpos=2071&amp;ncbi_uid=4989467&amp;link_uid=PMC3682211" href="/pmc/articles/PMC3682211/"><img src="/pmc/articles/instance/3682211/bin/emss-52952-f0007.gif" src-large="/pmc/articles/instance/3682211/bin/emss-52952-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3682211/" ref="ordinalpos=2071&amp;ncbi_uid=4989467&amp;link_uid=PMC3682211" image-link="/pmc/articles/PMC3682211/figure/F7/" class="imagepopup">Figure 7.  From: Aspirin Inhibits mTOR <span class="highlight" style="background-color:">Signaling</span>, Activates AMP-Activated Protein Kinase, and Induces Autophagy in Colorectal Cancer Cells. </a></p><div class="supp"><p class="details"><div>Aspirin targets multiple components of the AMPK/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in colorectal cancer. Aspirin increases phosphorylation of AMPK and ULK1. Aspirin inhibits mTOR <span class="highlight" style="background-color:">signaling</span> as evidenced by decreased phosphorylation of S6K1, S6, and 4E-BP1. Hence, aspirin is targeting multiple cellular pathways involved in mRNA stability, cell cycle, autophagy, protein translation, and ribosome biogenesis.</div></p></div><div class="aux"><div class="resc">FARHAT V. N. DIN, et al. Gastroenterology. ;142(7):1504-15.e3.</div><p class="links"><a class="dblinks" href="/pubmed/22406476" ref="ordinalpos=2071&amp;ncbi_uid=4989467&amp;link_uid=22406476">Citation</a><a class="dblinks" href="/pmc/articles/PMC3682211/" ref="ordinalpos=2071&amp;ncbi_uid=4989467&amp;link_uid=PMC3682211">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2072.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2997345/figure/fig5/" ref="ordinalpos=2072&amp;ncbi_uid=2628314&amp;link_uid=PMC2997345" href="/pmc/articles/PMC2997345/"><img src="/pmc/articles/instance/2997345/bin/IJAD2010-604792.005.gif" src-large="/pmc/articles/instance/2997345/bin/IJAD2010-604792.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2997345/" ref="ordinalpos=2072&amp;ncbi_uid=2628314&amp;link_uid=PMC2997345" image-link="/pmc/articles/PMC2997345/figure/fig5/" class="imagepopup">Figure 5.  From: Complex and Multidimensional Lipid Raft Alterations in a Murine Model of Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>Functional receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> informatic clustering of control and 3xTgAD lipid raft proteins.  Subtractive representation of hybrid score generation after clustering of raft proteins from control and 3xTgAD animal rafts into receptor <span class="highlight" style="background-color:">signaling</span> pathways.  The data is presented as a numerical value of the control <span class="highlight" style="background-color:">pathway</span> hybrid score subtracted from the 3xTgAD <span class="highlight" style="background-color:">pathway</span> hybrid score.  Pathways in which the score in 3xTgAD was greater than the control are denoted in red; pathways in which the control hybrid score was greater than the 3xTgAD hybrid score are denoted in blue.</div></p></div><div class="aux"><div class="resc">Wayne Chadwick, et al. Int J Alzheimers Dis. 2010;2010:604792.</div><p class="links"><a class="dblinks" href="/pubmed/21151659" ref="ordinalpos=2072&amp;ncbi_uid=2628314&amp;link_uid=21151659">Citation</a><a class="dblinks" href="/pmc/articles/PMC2997345/" ref="ordinalpos=2072&amp;ncbi_uid=2628314&amp;link_uid=PMC2997345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2073.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3325001/figure/f1-ott-5-047/" ref="ordinalpos=2073&amp;ncbi_uid=3898413&amp;link_uid=PMC3325001" href="/pmc/articles/PMC3325001/"><img src="/pmc/articles/instance/3325001/bin/ott-5-047f1.gif" src-large="/pmc/articles/instance/3325001/bin/ott-5-047f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3325001/" ref="ordinalpos=2073&amp;ncbi_uid=3898413&amp;link_uid=PMC3325001" image-link="/pmc/articles/PMC3325001/figure/f1-ott-5-047/" class="imagepopup">Figure 1.  From: Hedgehog <span class="highlight" style="background-color:">pathway</span> as a drug target: Smoothened inhibitors in development. </a></p><div class="supp"><p class="details"><div>Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the absence of Hh ligand, Ptch exerts an inhibitory effect on Smo, and no downstream <span class="highlight" style="background-color:">signaling</span> occurs. In the presence of Hh ligand binding to Ptch, the suppression of Smo is released. Smo interacts with Suppressor of fused (SUFU), which promotes the activation and nuclear translocation of Gli1. Gli1 translocation results in the transcription of Hh target genes.<br /><b>Abbreviations:</b> Hh, Hedgehog; Ptch, Patched; Smo, Smoothened, COS, Ccoastal-2; FU, Fused.</div></p></div><div class="aux"><div class="resc">Tara L Lin, et al. Onco Targets Ther. 2012;5:47-58.</div><p class="links"><a class="dblinks" href="/pubmed/22500124" ref="ordinalpos=2073&amp;ncbi_uid=3898413&amp;link_uid=22500124">Citation</a><a class="dblinks" href="/pmc/articles/PMC3325001/" ref="ordinalpos=2073&amp;ncbi_uid=3898413&amp;link_uid=PMC3325001">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2074.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3869757/figure/pone-0083102-g005/" ref="ordinalpos=2074&amp;ncbi_uid=5633100&amp;link_uid=PMC3869757" href="/pmc/articles/PMC3869757/"><img src="/pmc/articles/instance/3869757/bin/pone.0083102.g005.gif" src-large="/pmc/articles/instance/3869757/bin/pone.0083102.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3869757/" ref="ordinalpos=2074&amp;ncbi_uid=5633100&amp;link_uid=PMC3869757" image-link="/pmc/articles/PMC3869757/figure/pone-0083102-g005/" class="imagepopup">Figure 5. Nicotine deteriorates the osteogenic differentiation of hPDLSCs through α7 nAChR regulating Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span>..  From: Nicotine Deteriorates the Osteogenic Differentiation of Periodontal Ligament Stem Cells through ?7 Nicotinic Acetylcholine Receptor Regulating wnt <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A</b>: The expression of wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related proteins were determined by western blot, wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was activated by nicotine in the downstream of α7 nAChR; <b>B, C</b>: hPDLSCs treated with/without nicotine and dkk1 were cultured in osteogenic medium and osteogenic differentiation was determined by Alizarin red S staining after 21 days. <b>D</b>: The expression of osteogenic related genes Runx2, BSP, ALP and OCN were determined by qRT-PCR after 14 days of culturing with osteogenic supplement. <b>E</b>: The expression of osteogenic related proteins Runx2, BSP, ALP and OCN were determined by western blot after 14 days of culturing with osteogenic supplement. Data represent the means ± SD. *p&lt;0.05 (n = 3).</div></p></div><div class="aux"><div class="resc">Zhifei Zhou, et al. PLoS One. 2013;8(12):e83102.</div><p class="links"><a class="dblinks" href="/pubmed/24376645" ref="ordinalpos=2074&amp;ncbi_uid=5633100&amp;link_uid=24376645">Citation</a><a class="dblinks" href="/pmc/articles/PMC3869757/" ref="ordinalpos=2074&amp;ncbi_uid=5633100&amp;link_uid=PMC3869757">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2075.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4007724/figure/fig4/" ref="ordinalpos=2075&amp;ncbi_uid=6636039&amp;link_uid=PMC4007724" href="/pmc/articles/PMC4007724/"><img src="/pmc/articles/instance/4007724/bin/aps201065f4.gif" src-large="/pmc/articles/instance/4007724/bin/aps201065f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4007724/" ref="ordinalpos=2075&amp;ncbi_uid=6636039&amp;link_uid=PMC4007724" image-link="/pmc/articles/PMC4007724/figure/fig4/" class="imagepopup">Figure 4.  From: Activation of the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in CD151-induced angiogenic effects on the formation of CD151-integrin complexes. </a></p><div class="supp"><p class="details"><div>rAAV-CD151 transfection promotes the proliferation, migration, tube formation of HUVECs via ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, but not via p38. (A) MTT assays showing effects of inhibitors of ERK (PD98059, 20 μmol/L), and p38 inhibitor (SB203580, 20 μmol/L) on CD151-induced HUVECs proliferation. (B) Transwell Boyden chamber assays showing effects of inhibitors of ERK (PD98059, 20 μmol/L) and p38 inhibitor (SB203580, 20 μmol/L) on CD151 enhancement of HUVECs migration. (C) The effects of inhibitors of ERK (PD98059, 20 μmol/L), and p38 inhibitor (SB203580, 20 μmol/L) on CD151-induced HUVEC tube formation on matrigel. <sup>b</sup><i>P</i>&lt;0.05 <i>vs</i> the control and GFP groups. <sup>d</sup><i>P</i>&gt;0.05, <sup>e</sup><i>P</i>&lt;0.05 <i>vs</i> the CD151 group without inhibitors. <i>n</i>=3. Mean±SEM.</div></p></div><div class="aux"><div class="resc">Hou-juan Zuo, et al. Acta Pharmacol Sin. 2010 July;31(7):805-812.</div><p class="links"><a class="dblinks" href="/pubmed/20581856" ref="ordinalpos=2075&amp;ncbi_uid=6636039&amp;link_uid=20581856">Citation</a><a class="dblinks" href="/pmc/articles/PMC4007724/" ref="ordinalpos=2075&amp;ncbi_uid=6636039&amp;link_uid=PMC4007724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2076.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1435660/figure/f3/" ref="ordinalpos=2076&amp;ncbi_uid=575409&amp;link_uid=PMC1435660" href="/pmc/articles/PMC1435660/"><img src="/pmc/articles/instance/1435660/bin/69f3c.gif" src-large="/pmc/articles/instance/1435660/bin/69f3c.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1435660/" ref="ordinalpos=2076&amp;ncbi_uid=575409&amp;link_uid=PMC1435660" image-link="/pmc/articles/PMC1435660/figure/f3/" class="imagepopup">Figure 3. From: Hereditary Hemorrhagic Telangiectasia, a Vascular Dysplasia Affecting the TGF-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Balanced <span class="highlight" style="background-color:">signaling</span> through the TGF-β receptors type I, ALK1 and ALK5, in endothelial cells. When the TGF-β response is transmitted via ALK1, the proliferation and migration genes are stimulated giving rise to the activated phase of angiogenesis (activated endothelium). However, when the ALK5 <span class="highlight" style="background-color:">pathway</span> is predominant, the genes coding for extracellular matrix (ECM) components are upregulated and the proliferation and cell migration genes are repressed, leading to a quiescent endothelium. In addition, ALK1 and ALK5 functionally regulate each other. Adapted from Lebrin et al.44</div></p></div><div class="aux"><div class="resc">Africa Fernández-L, et al. Clin Med Res. 2006 March;4(1):66-78.</div><p class="links"><a class="dblinks" href="/pubmed/16595794" ref="ordinalpos=2076&amp;ncbi_uid=575409&amp;link_uid=16595794">Citation</a><a class="dblinks" href="/pmc/articles/PMC1435660/" ref="ordinalpos=2076&amp;ncbi_uid=575409&amp;link_uid=PMC1435660">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2077.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1088314/figure/fig1/" ref="ordinalpos=2077&amp;ncbi_uid=420628&amp;link_uid=PMC1088314" href="/pmc/articles/PMC1088314/"><img src="/pmc/articles/instance/1088314/bin/pp1371205f01.gif" src-large="/pmc/articles/instance/1088314/bin/pp1371205f01.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1088314/" ref="ordinalpos=2077&amp;ncbi_uid=420628&amp;link_uid=PMC1088314" image-link="/pmc/articles/PMC1088314/figure/fig1/" class="imagepopup">Figure 1. From: Peace Talks and Trade Deals. Keys to Long-Term Harmony in Legume-Microbe Symbioses. </a></p><div class="supp"><p class="details"><div>The Nod factor and mycorrhizal <span class="highlight" style="background-color:">signaling</span> pathways. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has been defined through genetics in the model legumes <i>M. truncatula</i> and <i>L. japonicus</i>, and <i>Medicago sativa</i>. The genes identified are defined in boxes. Mutations in all these genes have been characterized for calcium spiking except SYMRK. In addition, it has been shown that mutations in <i>NFR5</i> and <i>NFR1</i> lack the calcium flux, whereas mutations in <i>DMI1</i> and <i>DMI2</i> show the first phase of the flux response. Components of the Nod factor (NF) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are conserved with mycorrhizal <span class="highlight" style="background-color:">signaling</span>. We presume that genes specific to mycorrhizae must exist at equivalent positions to <i>NFR1</i> and <i>NFR5</i> and possibly <i>NSP1</i> and <i>NSP2</i>.</div></p></div><div class="aux"><div class="resc">Giles E.D. Oldroyd, et al. Plant Physiol. 2005 April;137(4):1205-1210.</div><p class="links"><a class="dblinks" href="/pubmed/15824283" ref="ordinalpos=2077&amp;ncbi_uid=420628&amp;link_uid=15824283">Citation</a><a class="dblinks" href="/pmc/articles/PMC1088314/" ref="ordinalpos=2077&amp;ncbi_uid=420628&amp;link_uid=PMC1088314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2078.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3043160/figure/F7/" ref="ordinalpos=2078&amp;ncbi_uid=3644487&amp;link_uid=PMC3043160" href="/pmc/articles/PMC3043160/"><img src="/pmc/articles/instance/3043160/bin/nihms244054f7.gif" src-large="/pmc/articles/instance/3043160/bin/nihms244054f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3043160/" ref="ordinalpos=2078&amp;ncbi_uid=3644487&amp;link_uid=PMC3043160" image-link="/pmc/articles/PMC3043160/figure/F7/" class="imagepopup">Figure 7. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of mTOR.  From: Calcium-mediated mechanisms of cystic expansion. </a></p><div class="supp"><p class="details"><div>In healthy kidney cells, the cysto-protein complex interacts with tuberin. Hamartin and tuberin form a complex and localize in the basal body. Tuberin is a GTPase activating protein (GAP) which controls the activity of Rheb. Tuberin inactivates Rheb, thereby inhibiting the mTOR <span class="highlight" style="background-color:">pathway</span>. In polycystic kidneys, however, tuberin is no longer protected in this complex and is phosphorylated by Akt, RSK (via ERK). As a result, tubulin is unable to form a heteradimer with hamartin. Hence, tuberin is no longer able inhibit Rheb, and the mTOR <span class="highlight" style="background-color:">pathway</span> is activated.</div></p></div><div class="aux"><div class="resc">Shakila Abdul-Majeed, et al. Biochim Biophys Acta. ;1812(10):1281-1290.</div><p class="links"><a class="dblinks" href="/pubmed/20932898" ref="ordinalpos=2078&amp;ncbi_uid=3644487&amp;link_uid=20932898">Citation</a><a class="dblinks" href="/pmc/articles/PMC3043160/" ref="ordinalpos=2078&amp;ncbi_uid=3644487&amp;link_uid=PMC3043160">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2079.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4163887/figure/F0001/" ref="ordinalpos=2079&amp;ncbi_uid=6614487&amp;link_uid=PMC4163887" href="/pmc/articles/PMC4163887/"><img src="/pmc/articles/instance/4163887/bin/BMB-47-184-g0001.gif" src-large="/pmc/articles/instance/4163887/bin/BMB-47-184-g0001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4163887/" ref="ordinalpos=2079&amp;ncbi_uid=6614487&amp;link_uid=PMC4163887" image-link="/pmc/articles/PMC4163887/figure/F0001/" class="imagepopup">Fig. 1.Role of Toll-like receptors during RSV infection. Stimulation of TLR4 with RSV F glycoprotein in a TRIF- or MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induces production of type I interferon or proinflammatory cytokines. TLR2/6, which are activated by RSV, enhance production of type I interferon or proinflammatory cytokines through a MyD88-mediated <span class="highlight" style="background-color:">pathway</span>. TLR7, localized in the endosome, recognizes the RSV ssRNA genome and activates expression of type I interferon and proinflammatory cytokine genes via a MyD88-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Endosomal TLR3 detects RSV dsRNA, an RSV replication intermediate, and then induces production of type I interferon or proinflammatory cytokines in a TRIF-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Abbreviations: TLR, Toll-like receptor; MyD88, myeloid differentiation primary response 88; TRIF, TIR-domain-containing adaptor-inducing interferon-beta; IRF, interferon regulatory factor; NF-κB, nuclear factor-κB..  From: Innate immune recognition of respiratory syncytial virus infection. </a></p><div class="aux"><div class="resc">Tae Hoon Kim, et al. BMB Rep. 2014 April;47(4):184-191.</div><p class="links"><a class="dblinks" href="/pubmed/24568879" ref="ordinalpos=2079&amp;ncbi_uid=6614487&amp;link_uid=24568879">Citation</a><a class="dblinks" href="/pmc/articles/PMC4163887/" ref="ordinalpos=2079&amp;ncbi_uid=6614487&amp;link_uid=PMC4163887">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2080.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3548460/figure/F3/" ref="ordinalpos=2080&amp;ncbi_uid=5724578&amp;link_uid=PMC3548460" href="/pmc/articles/PMC3548460/"><img src="/pmc/articles/instance/3548460/bin/zbc0061336990003.gif" src-large="/pmc/articles/instance/3548460/bin/zbc0061336990003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3548460/" ref="ordinalpos=2080&amp;ncbi_uid=5724578&amp;link_uid=PMC3548460" image-link="/pmc/articles/PMC3548460/figure/F3/" class="imagepopup">FIGURE 3. From: Epidermal Growth Factor Promotes Protein Degradation of Epithelial Protein Lost in Neoplasm (EPLIN), a Putative Metastasis Suppressor, during Epithelial-mesenchymal Transition. </a></p><div class="supp"><p class="details"><div><b>EGF induces serine phosphorylation and degradation of EPLIN through the ERK1/2 <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, EGF (50 ng/ml) induces EPLIN phosphorylation at serine residues in ARCaP<sub>E</sub> cells. <i>B</i>, EGF (50 ng/ml) activates ERK1/2 <span class="highlight" style="background-color:">signaling</span> in ARCaP<sub>E</sub> cells. <i>C</i>, blockade of ERK1/2 <span class="highlight" style="background-color:">signaling</span> with PD98059 or U0126 inhibits EGF-induced EPLIN phosphorylation in ARCaP<sub>E</sub> cells. <i>Input</i>: total cell lysates. <i>D</i>, blockade of ERK1/2 <span class="highlight" style="background-color:">signaling</span> inhibits EGF-induced EPLIN degradation in ARCaP<sub>E</sub> cells. <i>E</i>, blockade of ERK1/2 <span class="highlight" style="background-color:">signaling</span> with U0126 inhibits EGF-induced EPLIN degradation in C4-2 cells. <i>CHX</i>, cycloheximide; <i>DMSO</i>, dimethyl sulfoxide.</div></p></div><div class="aux"><div class="resc">Shumin Zhang, et al. J Biol Chem. 2013 January 18;288(3):1469-1479.</div><p class="links"><a class="dblinks" href="/pubmed/23188829" ref="ordinalpos=2080&amp;ncbi_uid=5724578&amp;link_uid=23188829">Citation</a><a class="dblinks" href="/pmc/articles/PMC3548460/" ref="ordinalpos=2080&amp;ncbi_uid=5724578&amp;link_uid=PMC3548460">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2081.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137012/figure/fig8/" ref="ordinalpos=2081&amp;ncbi_uid=3495289&amp;link_uid=PMC3137012" href="/pmc/articles/PMC3137012/"><img src="/pmc/articles/instance/3137012/bin/1450fig8.gif" src-large="/pmc/articles/instance/3137012/bin/1450fig8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137012/" ref="ordinalpos=2081&amp;ncbi_uid=3495289&amp;link_uid=PMC3137012" image-link="/pmc/articles/PMC3137012/figure/fig8/" class="imagepopup">Fig. 8. From: Differential regulation of CIDEA and CIDEC expression by insulin via Akt1/2- and JNK2-dependent pathways in human adipocytes. </a></p><div class="supp"><p class="details"><div>Hypothesis of insulin <span class="highlight" style="background-color:">signaling</span> pathways associated with the regulation of CIDEA and CIDEC expression in human adipocytes. The regulation of CIDEA and CIDEC expression by insulin is mediated by PI3K <span class="highlight" style="background-color:">signaling</span>. However, the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in the regulation of CIDEA and CIDEC diverges into different pathways downstream of PI3K. Akt1/2 mediates insulin-induced downregulation of CIDEA, whereas JNK2 mediates insulin-induced upregulation of CIDEC. The insulin regulation of CIDEA and CIDEC via different <span class="highlight" style="background-color:">signaling</span> pathways would contribute separately to insulin-induced anti-apoptosis and lipid droplet formation in human adipocytes.</div></p></div><div class="aux"><div class="resc">Minoru Ito, et al. J Lipid Res. 2011 August;52(8):1450-1460.</div><p class="links"><a class="dblinks" href="/pubmed/21636835" ref="ordinalpos=2081&amp;ncbi_uid=3495289&amp;link_uid=21636835">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137012/" ref="ordinalpos=2081&amp;ncbi_uid=3495289&amp;link_uid=PMC3137012">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2082.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2668926/figure/fig1/" ref="ordinalpos=2082&amp;ncbi_uid=1859616&amp;link_uid=PMC2668926" href="/pmc/articles/PMC2668926/"><img src="/pmc/articles/instance/2668926/bin/JO2009-526963.001.gif" src-large="/pmc/articles/instance/2668926/bin/JO2009-526963.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2668926/" ref="ordinalpos=2082&amp;ncbi_uid=1859616&amp;link_uid=PMC2668926" image-link="/pmc/articles/PMC2668926/figure/fig1/" class="imagepopup">Figure 1.  From: Anti-EGFR Therapy: Mechanism and Advances in  Clinical Efficacy in Breast Cancer. </a></p><div class="supp"><p class="details"><div> The EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. (a) Upon EGF-ligand binding to the EGFR there is subsequent dimerization (homo- or hetero-) and tyrosine kinase residue auto-/transphosphorylation of dimer partners, which in turn initiates the actual downstream <span class="highlight" style="background-color:">signaling</span> pathways. (b) Ras <span class="highlight" style="background-color:">signaling</span> cascade in tabulated form. (c) PI3K <span class="highlight" style="background-color:">signaling</span> cascade in tabulated form.</div></p></div><div class="aux"><div class="resc">John F. Flynn, et al. J Oncol. 2009;2009:526963.</div><p class="links"><a class="dblinks" href="/pubmed/19390622" ref="ordinalpos=2082&amp;ncbi_uid=1859616&amp;link_uid=19390622">Citation</a><a class="dblinks" href="/pmc/articles/PMC2668926/" ref="ordinalpos=2082&amp;ncbi_uid=1859616&amp;link_uid=PMC2668926">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2083.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1276168/figure/f3/" ref="ordinalpos=2083&amp;ncbi_uid=580584&amp;link_uid=PMC1276168" href="/pmc/articles/PMC1276168/"><img src="/pmc/articles/instance/1276168/bin/7600807f3.gif" src-large="/pmc/articles/instance/1276168/bin/7600807f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1276168/" ref="ordinalpos=2083&amp;ncbi_uid=580584&amp;link_uid=PMC1276168" image-link="/pmc/articles/PMC1276168/figure/f3/" class="imagepopup">Figure 3.  From: Inhibitor of apoptosis 2 and TAK1-binding protein are components of the Drosophila Imd <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Iap2 and TAB are located below Imd in the Imd <span class="highlight" style="background-color:">signaling</span> cascade. (<b>A</b>) Iap2 and TAB are located downstream of Imd. Overexpression of Relish constructs or Imd caused an activation of <i>Att</i> response in S2 cells. <i>Att</i> induction caused by expression of either Rel ΔS29–S45 or wild-type Relish could be blocked by RNAi targeting <i>Rel</i> but not RNAi targeting either <i>imd</i>, <i>TAB</i> or <i>Iap2</i>, indicating that both TAB and Iap2 are located upstream of Relish. <i>Att</i> induction caused by overexpression of Imd was blocked by RNAi targeting either <i>Rel</i>, <i>imd</i>, <i>TAB</i> or <i>Iap2</i>, indicating that both TAB and Iap2 are downstream of Imd. (<b>B</b>) Iap2 lies below TAK1 in the Imd <span class="highlight" style="background-color:">signaling</span> cascade. Overexpression of Iap2 caused a minimal but reproducible induction of <i>Att</i> expression, which was blocked by dsRNAs targeting the known components of the Imd <span class="highlight" style="background-color:">pathway</span> except dsRNA targeting either <i>imd</i> or <i>TAK1</i>. This indicates that Iap2 lies downstream of TAK1 in the Imd <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <sup>*</sup><i>P</i>&lt;0.05, <sup>**</sup><i>P</i>&lt;0.01 and <sup>***</sup><i>P</i>&lt;0.001, when the <i>Att</i> expression levels are compared to <i>GFP</i> RNAi controls. Data are shown as mean±s.d. of three to six independent dsRNA treatments.</div></p></div><div class="aux"><div class="resc">Anni Kleino, et al. EMBO J. 2005 October 5;24(19):3423-3434.</div><p class="links"><a class="dblinks" href="/pubmed/16163390" ref="ordinalpos=2083&amp;ncbi_uid=580584&amp;link_uid=16163390">Citation</a><a class="dblinks" href="/pmc/articles/PMC1276168/" ref="ordinalpos=2083&amp;ncbi_uid=580584&amp;link_uid=PMC1276168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2084.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3013293/figure/F8/" ref="ordinalpos=2084&amp;ncbi_uid=3437744&amp;link_uid=PMC3013293" href="/pmc/articles/PMC3013293/"><img src="/pmc/articles/instance/3013293/bin/nihms-245494-f0008.gif" src-large="/pmc/articles/instance/3013293/bin/nihms-245494-f0008.jpg" alt="FIG 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3013293/" ref="ordinalpos=2084&amp;ncbi_uid=3437744&amp;link_uid=PMC3013293" image-link="/pmc/articles/PMC3013293/figure/F8/" class="imagepopup">FIG 8.  From: Phospholipase D Controls Dictyostelium Development By Regulating G Protein <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed mechanism for CMF signal transduction. CMF binds to its receptor, and inhibits PldB. This could be through the known CMF <span class="highlight" style="background-color:">pathway</span> involving Gα1βγ, and PLC, or through an as yet unidentified <span class="highlight" style="background-color:">pathway</span>. Decreased PldB activity decreases phosphatidic acid production and increases the dissociation between Gα2 and G βγ, most likely through inhibiting the Gα2 GTPase activity. This allows cAMP <span class="highlight" style="background-color:">signaling</span> and thus aggregation to occur.</div></p></div><div class="aux"><div class="resc">Sibnath Ray, et al. Cell Signal. ;23(2):335-343.</div><p class="links"><a class="dblinks" href="/pubmed/20950684" ref="ordinalpos=2084&amp;ncbi_uid=3437744&amp;link_uid=20950684">Citation</a><a class="dblinks" href="/pmc/articles/PMC3013293/" ref="ordinalpos=2084&amp;ncbi_uid=3437744&amp;link_uid=PMC3013293">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2085.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3108236/figure/F2/" ref="ordinalpos=2085&amp;ncbi_uid=3010983&amp;link_uid=PMC3108236" href="/pmc/articles/PMC3108236/"><img src="/pmc/articles/instance/3108236/bin/1753-6561-5-S4-S6-2.gif" src-large="/pmc/articles/instance/3108236/bin/1753-6561-5-S4-S6-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3108236/" ref="ordinalpos=2085&amp;ncbi_uid=3010983&amp;link_uid=PMC3108236" image-link="/pmc/articles/PMC3108236/figure/F2/" class="imagepopup">Figure 2.  From: Multi-comparative systems biology analysis reveals time-course biosignatures of in vivo bovine <span class="highlight" style="background-color:">pathway</span> responses to B.melitensis, S.enterica Typhimurium and M.avium paratuberculosis. </a></p><div class="supp"><p class="details"><div><b>Biosignature heat maps of gene scores for the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for each host-pathogen condition</b><i>. Brucella melitensis</i> (BMEL), <i>Salmonella enterica</i> Typhimurium (STM) and <i>Mycobacterium avium paratuberculosis</i> (MAP).</div></p></div><div class="aux"><div class="resc">L Garry Adams, et al. BMC Proc. 2011;5(Suppl 4):S6-S6.</div><p class="links"><a class="dblinks" href="/pubmed/21645321" ref="ordinalpos=2085&amp;ncbi_uid=3010983&amp;link_uid=21645321">Citation</a><a class="dblinks" href="/pmc/articles/PMC3108236/" ref="ordinalpos=2085&amp;ncbi_uid=3010983&amp;link_uid=PMC3108236">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2086.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3699517/figure/pone-0068525-g002/" ref="ordinalpos=2086&amp;ncbi_uid=5073330&amp;link_uid=PMC3699517" href="/pmc/articles/PMC3699517/"><img src="/pmc/articles/instance/3699517/bin/pone.0068525.g002.gif" src-large="/pmc/articles/instance/3699517/bin/pone.0068525.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3699517/" ref="ordinalpos=2086&amp;ncbi_uid=5073330&amp;link_uid=PMC3699517" image-link="/pmc/articles/PMC3699517/figure/pone-0068525-g002/" class="imagepopup">Figure 2. Henryin inhibits Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>..  From: Henryin, an ent-kaurane Diterpenoid, Inhibits Wnt <span class="highlight" style="background-color:">Signaling</span> through Interference with ?-Catenin/TCF4 Interaction in Colorectal Cancer Cells. </a></p><div class="supp"><p class="details"><div>(A) (B) Microarray assay and data analysis of differential gene expression under henryin treatment. SW480 cells were treated for 12h with 4µM of henryin with 0.5% DMSO used as a control in the assay. GO annotation and <span class="highlight" style="background-color:">pathway</span> analysis were employed from the BioCarta and KEGG databases, respectively. The genes with fold alteration of at least &gt;2 (up-regulated) or &lt;0.5(down-regulated) were taken as differentially expressed genes. Data analysis and the statistics were generated only when the hits &gt; 5 in each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Representative pathways obtained after microarray data analysis are shown. (C) Henryin inhibits the Wnt reporter expression in a dose-dependent manner in wnt1 transfected HEK293T cells, and in colorectal cancer cells, SW480 and HCT116. Three hours after transfection of the Wnt1 and/or ST-Luc, DMSO or henryin with indicated dosage was added to the cells for additional 24h and luciferase activity was then measured. (D) Henryin (4µM) preferentially inhibits the Wnt <span class="highlight" style="background-color:">signaling</span> (ST-Luc) over the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (NF-κB-Luc) in HEK293T cells. Wnt <span class="highlight" style="background-color:">signaling</span> was stimulated by transfection of wnt1 and NF-κB <span class="highlight" style="background-color:">signaling</span> was stimulated with 25ng/mL TNFα. Each bar is the mean ± SD from three independent experiments. *P&lt;0.05, **P&lt;0.01, relative to vehicle control. NS, not significant.</div></p></div><div class="aux"><div class="resc">Xingyao Li, et al. PLoS One. 2013;8(7):e68525.</div><p class="links"><a class="dblinks" href="/pubmed/23844215" ref="ordinalpos=2086&amp;ncbi_uid=5073330&amp;link_uid=23844215">Citation</a><a class="dblinks" href="/pmc/articles/PMC3699517/" ref="ordinalpos=2086&amp;ncbi_uid=5073330&amp;link_uid=PMC3699517">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2087.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2913337/figure/F2-8704/" ref="ordinalpos=2087&amp;ncbi_uid=3124595&amp;link_uid=PMC2913337" href="/pmc/articles/PMC2913337/"><img src="/pmc/articles/instance/2913337/bin/zjh0081087040002.gif" src-large="/pmc/articles/instance/2913337/bin/zjh0081087040002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2913337/" ref="ordinalpos=2087&amp;ncbi_uid=3124595&amp;link_uid=PMC2913337" image-link="/pmc/articles/PMC2913337/figure/F2-8704/" class="imagepopup">Figure 2.  From: Role of Cripto-1 in Stem Cell Maintenance and Malignant Progression. </a></p><div class="supp"><p class="details"><div>Cross talk of the Nodal/Cripto-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with <i>Wnt</i>, hypoxia, Notch, Oct-4, and Nanog <span class="highlight" style="background-color:">signaling</span> pathways. Activated Smad-2/Smad-3/Smad-4 complex can bind to specific TGF-β binding elements within the Cripto-1 promoter and regulate <i>Cripto-1</i> gene expression. <i>Wnt</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: <i>Cripto-1</i> is a downstream target gene of the <i>Wnt</i>/β-catenin canonical <span class="highlight" style="background-color:">pathway</span>. Non-canonical <i>Wnt</i>11 can also bind to Cripto-1 and, in a complex with Frizzled 7 (Fzd7) and Glypican-4, induces β-catenin activation. Hypoxia: Hypoxia through HIF-1α directly regulates Cripto-1 expression in ES cells through binding to specific HREs within the <i>Cripto-1</i> promoter. ES cells specific transcription factors: Oct-4 and Nanog directly regulate Cripto-1 expression in ES cells by binding to the <i>Cripto-1</i> promoter. Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: Cripto-1 directly interacts with Notch in ER/Golgi membranes and enhances cleavage of the Notch extracellular domain, potentiating the Notch ligand-activated signal. Nodal–independent Cripto-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: Cripto-1, independently of Nodal, binds to Glypican-1 and induces activation of c-Src/MAPK/AKT, leading to cell motility and EMT. TBE: TCF/LEF binding elements; HRE: hypoxia binding elements; SBE: Smad binding elements; CBP: CREB binding protein; TCF/LEF: T-cell factor/lymphoid enhancer factor. EMT indicates epithelial to mesenchymal transition; Notch ICD, Notch intracellular domain; ER, endoplasmic reticulum; GDF, Growth and Differentiation Factor; LRP, low-density lipoprotein receptor-related protein; and MAPK, mitogen activated protein kinase.</div></p></div><div class="aux"><div class="resc">Caterina Bianco, et al. Am J Pathol. 2010 August;177(2):532-540.</div><p class="links"><a class="dblinks" href="/pubmed/20616345" ref="ordinalpos=2087&amp;ncbi_uid=3124595&amp;link_uid=20616345">Citation</a><a class="dblinks" href="/pmc/articles/PMC2913337/" ref="ordinalpos=2087&amp;ncbi_uid=3124595&amp;link_uid=PMC2913337">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2088.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2743241/figure/F1/" ref="ordinalpos=2088&amp;ncbi_uid=2009771&amp;link_uid=PMC2743241" href="/pmc/articles/PMC2743241/"><img src="/pmc/articles/instance/2743241/bin/nihms-104435-f0001.gif" src-large="/pmc/articles/instance/2743241/bin/nihms-104435-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2743241/" ref="ordinalpos=2088&amp;ncbi_uid=2009771&amp;link_uid=PMC2743241" image-link="/pmc/articles/PMC2743241/figure/F1/" class="imagepopup">Figure 1. Hedgehog <span class="highlight" style="background-color:">pathway</span> activators and inhibitors.  From: Hedgehog-Gli <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Inhibitors as Anticancer Agents. </a></p><div class="supp"><p class="details"><div>The Hh receptor Ptch is a transmembrane protein that inhibits the activity of Smo in the absence of Hh. Binding of Hh results in activation of Smo, which then modulates Gli transcription factors to initiate transcription of Hh target genes. Proteins involved in modulating signal relay between Smo and Gli are as indicated. <span class="highlight" style="background-color:">Pathway</span> suppressors are indicated in red and activators in green color. The molecules in this <span class="highlight" style="background-color:">pathway</span> that are targets for chemical inhibitors include Smo and Gli.</div></p></div><div class="aux"><div class="resc">Neeraj Mahindroo, et al. J Med Chem. ;52(13):3829-3845.</div><p class="links"><a class="dblinks" href="/pubmed/19309080" ref="ordinalpos=2088&amp;ncbi_uid=2009771&amp;link_uid=19309080">Citation</a><a class="dblinks" href="/pmc/articles/PMC2743241/" ref="ordinalpos=2088&amp;ncbi_uid=2009771&amp;link_uid=PMC2743241">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2089.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g005/" ref="ordinalpos=2089&amp;ncbi_uid=2941428&amp;link_uid=PMC3086880" href="/pmc/articles/PMC3086880/"><img src="/pmc/articles/instance/3086880/bin/pone.0019240.g005.gif" src-large="/pmc/articles/instance/3086880/bin/pone.0019240.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3086880/" ref="ordinalpos=2089&amp;ncbi_uid=2941428&amp;link_uid=PMC3086880" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g005/" class="imagepopup">Figure 5. Snapshot of the Notch <span class="highlight" style="background-color:">pathway</span> visualized with the ortholog network viewer..  From: Signalogs: Orthology-Based Identification of Novel <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Components in Three Metazoans. </a></p><div class="supp"><p class="details"><div>This figure displays a subset of the <i>C. elegans</i> Notch <span class="highlight" style="background-color:">pathway</span> with selected predicted <span class="highlight" style="background-color:">pathway</span> member proteins and their properties. This image includes multiple snapshots taken from the interactive ortholog viewer available at http://signalink.org. Links representing interactions with direct and indirect evidence are displayed as normal and dashed line styles, respectively. Activations (and inhibitions) are represented with normal (and blunted) arrows.</div></p></div><div class="aux"><div class="resc">Tamás Korcsmáros, et al. PLoS One. 2011;6(5):e19240.</div><p class="links"><a class="dblinks" href="/pubmed/21559328" ref="ordinalpos=2089&amp;ncbi_uid=2941428&amp;link_uid=21559328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3086880/" ref="ordinalpos=2089&amp;ncbi_uid=2941428&amp;link_uid=PMC3086880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2090.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2930606/figure/F1/" ref="ordinalpos=2090&amp;ncbi_uid=2676922&amp;link_uid=PMC2930606" href="/pmc/articles/PMC2930606/"><img src="/pmc/articles/instance/2930606/bin/nihms230262f1.gif" src-large="/pmc/articles/instance/2930606/bin/nihms230262f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2930606/" ref="ordinalpos=2090&amp;ncbi_uid=2676922&amp;link_uid=PMC2930606" image-link="/pmc/articles/PMC2930606/figure/F1/" class="imagepopup">Figure 1. Canonical and non-canonical <span class="highlight" style="background-color:">signaling</span> cascades downstream of D1 and D2 dopamine receptors.  From: Maladaptive striatal plasticity in L-DOPA-induced dyskinesia. </a></p><div class="supp"><p class="details"><div>Being coupled to stimulatory (Gs/olf) and inhibitory (Gi/o) GTP-binding proteins, D<sub>1</sub> and D<sub>2</sub> receptors have opposite effects on the adenylyl cyclase/cAMP/PKA/DARPP-32 cascade (‘canonical <span class="highlight" style="background-color:">pathway</span>’), which regulates the levels of phosphorylation of multiple cellular and nuclear targets. In particular, the cAMP/PKA/DARPP-32 <span class="highlight" style="background-color:">pathway</span> modulates the activity of MAPK-dependent <span class="highlight" style="background-color:">signaling</span> pathways downstream of glutamate receptors. It has been recently recognized that cAMP-independent pathways are also recruited following D<sub>2</sub> receptor stimulation (non-canonical pathways; <i>cf.</i> Section ‘DA receptor <span class="highlight" style="background-color:">signaling</span> in the neurons of the intact striatum: modulation of cAMP/PKA pathways and non-canonical <span class="highlight" style="background-color:">signaling</span>’). Full names of the molecules shown in this drawing are given in the list of abbreviations.</div></p></div><div class="aux"><div class="resc">M. Angela Cenci, et al. Prog Brain Res. ;183:209-233.</div><p class="links"><a class="dblinks" href="/pubmed/20696322" ref="ordinalpos=2090&amp;ncbi_uid=2676922&amp;link_uid=20696322">Citation</a><a class="dblinks" href="/pmc/articles/PMC2930606/" ref="ordinalpos=2090&amp;ncbi_uid=2676922&amp;link_uid=PMC2930606">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2091.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3280986/figure/ppat-1002530-g007/" ref="ordinalpos=2091&amp;ncbi_uid=3816978&amp;link_uid=PMC3280986" href="/pmc/articles/PMC3280986/"><img src="/pmc/articles/instance/3280986/bin/ppat.1002530.g007.gif" src-large="/pmc/articles/instance/3280986/bin/ppat.1002530.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3280986/" ref="ordinalpos=2091&amp;ncbi_uid=3816978&amp;link_uid=PMC3280986" image-link="/pmc/articles/PMC3280986/figure/ppat-1002530-g007/" class="imagepopup">Figure 7. Gαq-Rho GEF Trio-Rho <span class="highlight" style="background-color:">Signaling</span> and Ras converge on Raf to regulate morphology during the immune response to infection..  From: Behavioral and Immune Responses to Infection Require G?q- RhoA <span class="highlight" style="background-color:">Signaling</span> in C. elegans. </a></p><div class="supp"><p class="details"><div>The simplest explanation of our results is that following pathogen infection RHO-1 is activated in the rectal epithelial cells by multiple upstream regulators including EGL-30 (Gαq) and UNC-73 (Trio). Together with Ras, Rho <span class="highlight" style="background-color:">signaling</span> converges on Raf to activate the MAPK <span class="highlight" style="background-color:">pathway</span>. Activation of these pathways, together with at least one other, in the rectal epithelial cells leads to the changes in morphology that occur as part of the immune response.</div></p></div><div class="aux"><div class="resc">Rachel McMullan, et al. PLoS Pathog. 2012 February;8(2):e1002530.</div><p class="links"><a class="dblinks" href="/pubmed/22359503" ref="ordinalpos=2091&amp;ncbi_uid=3816978&amp;link_uid=22359503">Citation</a><a class="dblinks" href="/pmc/articles/PMC3280986/" ref="ordinalpos=2091&amp;ncbi_uid=3816978&amp;link_uid=PMC3280986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2092.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2897908/figure/F2/" ref="ordinalpos=2092&amp;ncbi_uid=3186667&amp;link_uid=PMC2897908" href="/pmc/articles/PMC2897908/"><img src="/pmc/articles/instance/2897908/bin/nihms-209662-f0002.gif" src-large="/pmc/articles/instance/2897908/bin/nihms-209662-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2897908/" ref="ordinalpos=2092&amp;ncbi_uid=3186667&amp;link_uid=PMC2897908" image-link="/pmc/articles/PMC2897908/figure/F2/" class="imagepopup">Figure 2. Inhibitors of the MAPK <span class="highlight" style="background-color:">pathway</span>.  From: Targeting the MAPK <span class="highlight" style="background-color:">Pathway</span> in Melanoma: Why some approaches succeed and other fail. </a></p><div class="supp"><p class="details"><div>Schematic representation of key targets and pharmacological agents that inhibit MAPK <span class="highlight" style="background-color:">signaling</span> proteins. Imitanib, dastinib and sorafenib inhibit receptor tyrosine kinases thereby preventing activation of mitogen dependent activation in the MAPK <span class="highlight" style="background-color:">pathway</span>. Tipifarnib, R115777, BMS-214662 and L-778123 are selective to RAS while PLX4032, RAF-265 and XL281 specifically target activated B-RAF and are at various stages of clinical development. AZD6244, U0126, PD0325901, CI-1040, XL518, AZD8330, ARRY-162 and ARRY-300 inhibit MEK 1/2 while curcumin, plumbagin and DHMEQ target IKKβand NFκB.</div></p></div><div class="aux"><div class="resc">Gajanan S Inamdar, et al. Biochem Pharmacol. ;80(5):624-637.</div><p class="links"><a class="dblinks" href="/pubmed/20450891" ref="ordinalpos=2092&amp;ncbi_uid=3186667&amp;link_uid=20450891">Citation</a><a class="dblinks" href="/pmc/articles/PMC2897908/" ref="ordinalpos=2092&amp;ncbi_uid=3186667&amp;link_uid=PMC2897908">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2093.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3035994/figure/F1/" ref="ordinalpos=2093&amp;ncbi_uid=3644225&amp;link_uid=PMC3035994" href="/pmc/articles/PMC3035994/"><img src="/pmc/articles/instance/3035994/bin/nihms-254214-f0002.gif" src-large="/pmc/articles/instance/3035994/bin/nihms-254214-f0002.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3035994/" ref="ordinalpos=2093&amp;ncbi_uid=3644225&amp;link_uid=PMC3035994" image-link="/pmc/articles/PMC3035994/figure/F1/" class="imagepopup">Figure 1.  From: High Fat Diet Induced Obesity and Nutrient Sensing TOR <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Simplified model of nutrient <span class="highlight" style="background-color:">signaling</span> pathways influencing metabolic responses. InR and TOR <span class="highlight" style="background-color:">pathway</span> components are virtually all unique orthologous genes in <i>Drosophila</i>. TOR <span class="highlight" style="background-color:">signaling</span> also receives inputs from the energy sensing AMPK. Future studies will elucidate how the components of the InR-TOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulate fat accumulation due to high levels of dietary fats, using the versatility and genetic simplicity of the <i>Drosophila</i> model. Unlike the complexity and interrelation between these pathways in mammals, <i>Drosophila</i> is ideal to study their role in controlling complex traits like HFD effects on disease.</div></p></div><div class="aux"><div class="resc">Sean Oldham. Trends Endocrinol Metab. ;22(2):45-52.</div><p class="links"><a class="dblinks" href="/pubmed/21216618" ref="ordinalpos=2093&amp;ncbi_uid=3644225&amp;link_uid=21216618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3035994/" ref="ordinalpos=2093&amp;ncbi_uid=3644225&amp;link_uid=PMC3035994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2094.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2944161/figure/F8/" ref="ordinalpos=2094&amp;ncbi_uid=2476903&amp;link_uid=PMC2944161" href="/pmc/articles/PMC2944161/"><img src="/pmc/articles/instance/2944161/bin/1471-213X-10-96-8.gif" src-large="/pmc/articles/instance/2944161/bin/1471-213X-10-96-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2944161/" ref="ordinalpos=2094&amp;ncbi_uid=2476903&amp;link_uid=PMC2944161" image-link="/pmc/articles/PMC2944161/figure/F8/" class="imagepopup">Figure 8.  From: Mtmr8 is essential for vasculature development in zebrafish embryos. </a></p><div class="supp"><p class="details"><div><b>A hypothesized signal <span class="highlight" style="background-color:">pathway</span> that Mtmr8 regulates artery specification and hematovascular progenitor specification in zebrafish embryo</b>. Mtmr8 inhibits PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, and activates Hh and Vegf <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which may interact with each other in regulating vasculature development. The pink arrows indicate the regulatory loop.</div></p></div><div class="aux"><div class="resc">Jie Mei, et al. BMC Dev Biol. 2010;10:96-96.</div><p class="links"><a class="dblinks" href="/pubmed/20815916" ref="ordinalpos=2094&amp;ncbi_uid=2476903&amp;link_uid=20815916">Citation</a><a class="dblinks" href="/pmc/articles/PMC2944161/" ref="ordinalpos=2094&amp;ncbi_uid=2476903&amp;link_uid=PMC2944161">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2095.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4171617/figure/F9/" ref="ordinalpos=2095&amp;ncbi_uid=6643461&amp;link_uid=PMC4171617" href="/pmc/articles/PMC4171617/"><img src="/pmc/articles/instance/4171617/bin/oncotarget-05-6113-g009.gif" src-large="/pmc/articles/instance/4171617/bin/oncotarget-05-6113-g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4171617/" ref="ordinalpos=2095&amp;ncbi_uid=6643461&amp;link_uid=PMC4171617" image-link="/pmc/articles/PMC4171617/figure/F9/" class="imagepopup">Figure 9. A proposed model for miR-451/c-Myc-rad-51/survivin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in chemo- and radiotherapy cross resistance in LAD cells.  From: Acquisition of radioresistance in docetaxel-resistant human lung adenocarcinoma cells is linked with dysregulation of miR-451/c-Myc-survivin/rad-51 <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Docetaxel-resistant LAD cells acquired radioresistant phenotype where miR-451 was downregulated, resulting in chemo- and radiotherapy cross resistance.</div></p></div><div class="aux"><div class="resc">Rui Wang, et al. Oncotarget. 2014 August;5(15):6113-6129.</div><p class="links"><a class="dblinks" href="/pubmed/25026294" ref="ordinalpos=2095&amp;ncbi_uid=6643461&amp;link_uid=25026294">Citation</a><a class="dblinks" href="/pmc/articles/PMC4171617/" ref="ordinalpos=2095&amp;ncbi_uid=6643461&amp;link_uid=PMC4171617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2096.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4115241/figure/F5/" ref="ordinalpos=2096&amp;ncbi_uid=6429989&amp;link_uid=PMC4115241" href="/pmc/articles/PMC4115241/"><img src="/pmc/articles/instance/4115241/bin/ijmsv11p1022g005.gif" src-large="/pmc/articles/instance/4115241/bin/ijmsv11p1022g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4115241/" ref="ordinalpos=2096&amp;ncbi_uid=6429989&amp;link_uid=PMC4115241" image-link="/pmc/articles/PMC4115241/figure/F5/" class="imagepopup">Figure 5.  From: Activated Hair Follicle Stem Cells and Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> Involve in Pathnogenesis of Sebaceous Neoplasms. </a></p><div class="supp"><p class="details"><div>Schematic drawing showed activated HFSCs contributed to SN development. DMBA combined with TPA treatment disturbed the homeostasis of SGs and induced SN development during which HFSCs were activated and possibly induced to migrate towards SNs. And Wnt10b/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was activated to promote proliferation and differentiation of HFSCs in SNs.</div></p></div><div class="aux"><div class="resc">Weiming Qiu, et al. Int J Med Sci. 2014;11(10):1022-1028.</div><p class="links"><a class="dblinks" href="/pubmed/25076848" ref="ordinalpos=2096&amp;ncbi_uid=6429989&amp;link_uid=25076848">Citation</a><a class="dblinks" href="/pmc/articles/PMC4115241/" ref="ordinalpos=2096&amp;ncbi_uid=6429989&amp;link_uid=PMC4115241">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2097.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2394752/figure/F1/" ref="ordinalpos=2097&amp;ncbi_uid=1403182&amp;link_uid=PMC2394752" href="/pmc/articles/PMC2394752/"><img src="/pmc/articles/instance/2394752/bin/gb-2007-8-6-r104-1.gif" src-large="/pmc/articles/instance/2394752/bin/gb-2007-8-6-r104-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2394752/" ref="ordinalpos=2097&amp;ncbi_uid=1403182&amp;link_uid=PMC2394752" image-link="/pmc/articles/PMC2394752/figure/F1/" class="imagepopup">Figure 1.  From: A functional map of NF?B <span class="highlight" style="background-color:">signaling</span> identifies novel modulators and multiple system controls. </a></p><div class="supp"><p class="details"><div>Flowchart outlining the steps in the gain-of-function and loss-of-function genomic screens. The gain-of-function screen was used to identify positive modifiers of the NFκB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the loss-of-function screen was used to identify negative modifiers. Approximately 14,500 full-length human and mouse genes were screened for activity.</div></p></div><div class="aux"><div class="resc">Thomas A Halsey, et al. Genome Biol. 2007;8(6):R104-R104.</div><p class="links"><a class="dblinks" href="/pubmed/17553156" ref="ordinalpos=2097&amp;ncbi_uid=1403182&amp;link_uid=17553156">Citation</a><a class="dblinks" href="/pmc/articles/PMC2394752/" ref="ordinalpos=2097&amp;ncbi_uid=1403182&amp;link_uid=PMC2394752">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2098.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4386408/figure/fig05/" ref="ordinalpos=2098&amp;ncbi_uid=7401474&amp;link_uid=PMC4386408" href="/pmc/articles/PMC4386408/"><img src="/pmc/articles/instance/4386408/bin/pnas.1419690112fig05.gif" src-large="/pmc/articles/instance/4386408/bin/pnas.1419690112fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4386408/" ref="ordinalpos=2098&amp;ncbi_uid=7401474&amp;link_uid=PMC4386408" image-link="/pmc/articles/PMC4386408/figure/fig05/" class="imagepopup">Fig. 5. From: Inversion of Sonic hedgehog action on its canonical <span class="highlight" style="background-color:">pathway</span> by electrical activity. </a></p><div class="supp"><p class="details"><div>Model for the mechanism of the switch off in Shh canonical <span class="highlight" style="background-color:">signaling</span> during spinal cord development. The transition from neural plate to spinal cord is accompanied by the appearance of Ca<sup>2+</sup> spikes, which intercalate in the Shh-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to down-regulate Gli activity. This switch off of the canonical Shh <span class="highlight" style="background-color:">pathway</span> is implemented at several levels including regulation of Gli subcellular localization, posttranslational processing, and transcriptional repression, all converging in inhibiting Gli activity.</div></p></div><div class="aux"><div class="resc">Yesser H. Belgacem, et al. Proc Natl Acad Sci U S A. 2015 March 31;112(13):4140-4145.</div><p class="links"><a class="dblinks" href="/pubmed/25829542" ref="ordinalpos=2098&amp;ncbi_uid=7401474&amp;link_uid=25829542">Citation</a><a class="dblinks" href="/pmc/articles/PMC4386408/" ref="ordinalpos=2098&amp;ncbi_uid=7401474&amp;link_uid=PMC4386408">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2099.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC198534/figure/F1/" ref="ordinalpos=2099&amp;ncbi_uid=245604&amp;link_uid=PMC198534" href="/pmc/articles/PMC198534/"><img src="/pmc/articles/instance/198534/bin/JCI0319974.f1.gif" src-large="/pmc/articles/instance/198534/bin/JCI0319974.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC198534/" ref="ordinalpos=2099&amp;ncbi_uid=245604&amp;link_uid=PMC198534" image-link="/pmc/articles/PMC198534/figure/F1/" class="imagepopup">Figure 1.  From: Hepatoprotection via the IL-6/Stat3 <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Model for IL-6/Stat3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IL-6 binds to its soluble receptor, sIL-6r, which binds to the gp130 receptor, resulting in the activation of Janus kinase (JAK). This leads to activation of the MAPK <span class="highlight" style="background-color:">pathway</span> and activation of Stat3 by tyrosine (Y) phosphorylation. Dimerized Stat3 is able to translocate into the nucleus and activate gene transcription. In the liver, this process promotes liver regeneration, the acute-phase response, and hepatoprotection against Fas and toxic damage. P, phosphate.</div></p></div><div class="aux"><div class="resc">Rebecca Taub. J Clin Invest. 2003 October 1;112(7):978-980.</div><p class="links"><a class="dblinks" href="/pubmed/14523032" ref="ordinalpos=2099&amp;ncbi_uid=245604&amp;link_uid=14523032">Citation</a><a class="dblinks" href="/pmc/articles/PMC198534/" ref="ordinalpos=2099&amp;ncbi_uid=245604&amp;link_uid=PMC198534">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2100.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412720/figure/F3/" ref="ordinalpos=2100&amp;ncbi_uid=4162485&amp;link_uid=PMC3412720" href="/pmc/articles/PMC3412720/"><img src="/pmc/articles/instance/3412720/bin/1479-5876-10-116-3.gif" src-large="/pmc/articles/instance/3412720/bin/1479-5876-10-116-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412720/" ref="ordinalpos=2100&amp;ncbi_uid=4162485&amp;link_uid=PMC3412720" image-link="/pmc/articles/PMC3412720/figure/F3/" class="imagepopup">Figure 3.  From: JAK-STAT and AKT <span class="highlight" style="background-color:">pathway</span>-coupled genes in erythroid progenitor cells through ontogeny. </a></p><div class="supp"><p class="details"><div><b> JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The expression of genes linked to JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through ontogeny. (+p) phosphorylation, (−p) dephosphorylation, → stimulation, ⟂ inhibition, → translocation, ↓ decreased gene expression, ↑ increased gene expression.</div></p></div><div class="aux"><div class="resc">Vladan P Cokic, et al. J Transl Med. 2012;10:116-116.</div><p class="links"><a class="dblinks" href="/pubmed/22676255" ref="ordinalpos=2100&amp;ncbi_uid=4162485&amp;link_uid=22676255">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412720/" ref="ordinalpos=2100&amp;ncbi_uid=4162485&amp;link_uid=PMC3412720">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="20" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="21" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="22" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:31:39-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC37065430F8E1000000000032AEBC;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC37065430F8E1000000000032AEBC" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>