
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="2" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="8A1A393946D25221_0091SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC3F26540048610000000000E045F1" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene">Gene</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="1" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="2" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="3" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="2" /></div>
            <div><h2 class="result_count">Results: 101 to 200 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>101.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g001/" ref="ordinalpos=101&amp;ncbi_uid=2941432&amp;link_uid=PMC3086880" href="/pmc/articles/PMC3086880/"><img src="/pmc/articles/instance/3086880/bin/pone.0019240.g001.gif" src-large="/pmc/articles/instance/3086880/bin/pone.0019240.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3086880/" ref="ordinalpos=101&amp;ncbi_uid=2941432&amp;link_uid=PMC3086880" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g001/" class="imagepopup">Figure 1. Workflow of <span class="highlight" style="background-color:">pathway</span> annotation transfer for predicting the <span class="highlight" style="background-color:">signaling</span> roles of proteins in C. elegans, D. melanogaster, and humans..  From: Signalogs: Orthology-Based Identification of Novel <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Components in Three Metazoans. </a></p><div class="supp"><p class="details"><div>The lists of <span class="highlight" style="background-color:">signaling</span> proteins and their interactions were obtained from the SignaLink database ([22], http://signalink.org). Orthology assignment was performed between each pair of the 3 species. Proteins were predicted to be members of the same <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>(s) where their orthologs belong.</div></p></div><div class="aux"><div class="resc">Tamás Korcsmáros, et al. PLoS One. 2011;6(5):e19240.</div><p class="links"><a class="dblinks" href="/pubmed/21559328" ref="ordinalpos=101&amp;ncbi_uid=2941432&amp;link_uid=21559328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3086880/" ref="ordinalpos=101&amp;ncbi_uid=2941432&amp;link_uid=PMC3086880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>102.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2266794/figure/pone-0001803-g004/" ref="ordinalpos=102&amp;ncbi_uid=1283796&amp;link_uid=PMC2266794" href="/pmc/articles/PMC2266794/"><img src="/pmc/articles/instance/2266794/bin/pone.0001803.g004.gif" src-large="/pmc/articles/instance/2266794/bin/pone.0001803.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2266794/" ref="ordinalpos=102&amp;ncbi_uid=1283796&amp;link_uid=PMC2266794" image-link="/pmc/articles/PMC2266794/figure/pone-0001803-g004/" class="imagepopup">Figure 4. HRG-regulated TFs involved in the KEGG MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells. </a></p><div class="supp"><p class="details"><div>Thirteen TFs included in the 117 HRG-regulated TF list involved in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and are highlighted in red. Note that since <i>Elk-1</i> appears twice in the figure (downstream of classical MAP kinase <span class="highlight" style="background-color:">pathway</span> and JNK and p38 MAP kinase <span class="highlight" style="background-color:">pathway</span>), 14 boxes are highlighted.</div></p></div><div class="aux"><div class="resc">Takeshi Nagashima, et al. PLoS ONE. 2008;3(3):e1803.</div><p class="links"><a class="dblinks" href="/pubmed/18350142" ref="ordinalpos=102&amp;ncbi_uid=1283796&amp;link_uid=18350142">Citation</a><a class="dblinks" href="/pmc/articles/PMC2266794/" ref="ordinalpos=102&amp;ncbi_uid=1283796&amp;link_uid=PMC2266794">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>103.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2585047/figure/F1/" ref="ordinalpos=103&amp;ncbi_uid=1615298&amp;link_uid=PMC2585047" href="/pmc/articles/PMC2585047/"><img src="/pmc/articles/instance/2585047/bin/nihms65573f1.gif" src-large="/pmc/articles/instance/2585047/bin/nihms65573f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2585047/" ref="ordinalpos=103&amp;ncbi_uid=1615298&amp;link_uid=PMC2585047" image-link="/pmc/articles/PMC2585047/figure/F1/" class="imagepopup">Figure 1.  From: Nrf2 <span class="highlight" style="background-color:">signaling</span>: An adaptive response <span class="highlight" style="background-color:">pathway</span> for protection against environmental toxic insults. </a></p><div class="supp"><p class="details"><div>The Nrf2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">William O Osburn, et al. Mutat Res. ;659(1-2):31-39.</div><p class="links"><a class="dblinks" href="/pubmed/18164232" ref="ordinalpos=103&amp;ncbi_uid=1615298&amp;link_uid=18164232">Citation</a><a class="dblinks" href="/pmc/articles/PMC2585047/" ref="ordinalpos=103&amp;ncbi_uid=1615298&amp;link_uid=PMC2585047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>104.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013640/figure/f2-ijms-15-06453/" ref="ordinalpos=104&amp;ncbi_uid=6110981&amp;link_uid=PMC4013640" href="/pmc/articles/PMC4013640/"><img src="/pmc/articles/instance/4013640/bin/ijms-15-06453f2.gif" src-large="/pmc/articles/instance/4013640/bin/ijms-15-06453f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013640/" ref="ordinalpos=104&amp;ncbi_uid=6110981&amp;link_uid=PMC4013640" image-link="/pmc/articles/PMC4013640/figure/f2-ijms-15-06453/" class="imagepopup">Figure 2. From: The Role of Pericytes in Neurovascular Unit Remodeling in Brain Disorders. </a></p><div class="supp"><p class="details"><div>Pericyte physical and biochemical interactions at the NVU. Pericytes are embedded within the basal lamina structure and project elongated processes that wrap endothelial cells, thus establishing specialized cell-cell contacts. These contacts are mainly from peg-socket types, where cell-cell contacts are established when pericyte cell membrane protrusion-like structures (pegs) are inserted into endothelial cell membrane invaginations (pockets) cells (outlined by dashed circles). The peg-socket contacts contain cell-to-cell junction proteins, such as N-cadherin and CX43 hemichannels. CX43 hemichannels form gap junctions that allow the biochemical exchange between pericytes and endothelial cells. In parallel, several autocrine and paracrine <span class="highlight" style="background-color:">signaling</span> pathways govern the interaction between pericytes and endothelial cells, such as the PDGFBB/PDGFRβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (paracrine <span class="highlight" style="background-color:">pathway</span>: PDGFBB secreted by endothelial cells binds to PDGFRβ expressed on pericytes), the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (paracrine and autocrine <span class="highlight" style="background-color:">pathway</span>: TGFβ secreted by endothelial cells and pericytes binds to TGFβR2 expressed on both cell types), the Ang1/Tie2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (paracrine <span class="highlight" style="background-color:">pathway</span>: Ang1 is secreted by pericytes binds to Tie2 expressed on endothelial cells), and the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (cleavage-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: the sequential proteolytic cleavage and release in the intracellular space of the Notch intracellular domain (NICD) that translocates to the nucleus and controls downstream gene expression).</div></p></div><div class="aux"><div class="resc">Ayman ElAli, et al. Int J Mol Sci. 2014 April;15(4):6453-6474.</div><p class="links"><a class="dblinks" href="/pubmed/24743889" ref="ordinalpos=104&amp;ncbi_uid=6110981&amp;link_uid=24743889">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013640/" ref="ordinalpos=104&amp;ncbi_uid=6110981&amp;link_uid=PMC4013640">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>105.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4087218/figure/F4/" ref="ordinalpos=105&amp;ncbi_uid=6361130&amp;link_uid=PMC4087218" href="/pmc/articles/PMC4087218/"><img src="/pmc/articles/instance/4087218/bin/jvs-15-179-g004.gif" src-large="/pmc/articles/instance/4087218/bin/jvs-15-179-g004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4087218/" ref="ordinalpos=105&amp;ncbi_uid=6361130&amp;link_uid=PMC4087218" image-link="/pmc/articles/PMC4087218/figure/F4/" class="imagepopup">Fig. 4.  From: Effects of resveratrol on the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of obese mice. </a></p><div class="supp"><p class="details"><div>Effects of resveratrol on protein phosphorylation in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the mTOR <span class="highlight" style="background-color:">pathway</span> of obese mice.</div></p></div><div class="aux"><div class="resc">Hyun Ju Hong, et al. J Vet Sci. 2014 June;15(2):179-185.</div><p class="links"><a class="dblinks" href="/pubmed/24675832" ref="ordinalpos=105&amp;ncbi_uid=6361130&amp;link_uid=24675832">Citation</a><a class="dblinks" href="/pmc/articles/PMC4087218/" ref="ordinalpos=105&amp;ncbi_uid=6361130&amp;link_uid=PMC4087218">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>106.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC149053/figure/fig7/" ref="ordinalpos=106&amp;ncbi_uid=209630&amp;link_uid=PMC149053" href="/pmc/articles/PMC149053/"><img src="/pmc/articles/instance/149053/bin/00-0019f7.gif" src-large="/pmc/articles/instance/149053/bin/00-0019f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC149053/" ref="ordinalpos=106&amp;ncbi_uid=209630&amp;link_uid=PMC149053" image-link="/pmc/articles/PMC149053/figure/fig7/" class="imagepopup">Figure 7. From: Regulation of Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> by the Ethylene Response <span class="highlight" style="background-color:">Pathway</span> in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Hypothetical Model for the Role of ABA and Ethylene in Regulating Root Growth in Arabidopsis.<br /><b>(A)</b> In the absence of ethylene, one way for ABA to inhibit root growth is by <span class="highlight" style="background-color:">signaling</span> through the ETR1 response <span class="highlight" style="background-color:">pathway</span>. Because this <span class="highlight" style="background-color:">pathway</span> is also regulated by ethylene, mutations in the <span class="highlight" style="background-color:">pathway</span> interfere with both ABA and ethylene <span class="highlight" style="background-color:">signaling</span>.<br /><b>(B)</b> In the presence of ethylene, ABA is unable to use this <span class="highlight" style="background-color:">pathway</span>; thus, mutations that increase ethylene synthesis confer an ABA-insensitive root phenotype.</div></p></div><div class="aux"><div class="resc">Majid Ghassemian, et al. Plant Cell. 2000 July;12(7):1117-1126.</div><p class="links"><a class="dblinks" href="/pubmed/10899978" ref="ordinalpos=106&amp;ncbi_uid=209630&amp;link_uid=10899978">Citation</a><a class="dblinks" href="/pmc/articles/PMC149053/" ref="ordinalpos=106&amp;ncbi_uid=209630&amp;link_uid=PMC149053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>107.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151254/figure/fig1/" ref="ordinalpos=107&amp;ncbi_uid=247092&amp;link_uid=PMC151254" href="/pmc/articles/PMC151254/"><img src="/pmc/articles/instance/151254/bin/00-0596f1.gif" src-large="/pmc/articles/instance/151254/bin/00-0596f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151254/" ref="ordinalpos=107&amp;ncbi_uid=247092&amp;link_uid=PMC151254" image-link="/pmc/articles/PMC151254/figure/fig1/" class="imagepopup">Figure 1. From: Cell <span class="highlight" style="background-color:">Signaling</span> during Cold, Drought, and Salt Stress. </a></p><div class="supp"><p class="details"><div>A Generic <span class="highlight" style="background-color:">Pathway</span> for the Transduction of Cold, Drought, and Salt Stress Signals in Plants.<br />Examples of <span class="highlight" style="background-color:">signaling</span> components in each of the steps are shown (for more detailed information, see Xiong and Zhu, 2001). Secondary <span class="highlight" style="background-color:">signaling</span> molecules can cause receptor-mediated Ca<sup>2+</sup> release (indicated with a feedback arrow). Examples of <span class="highlight" style="background-color:">signaling</span> partners that modulate the main <span class="highlight" style="background-color:">pathway</span> are also shown. These partners can be regulated by the main <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Signaling</span> can also bypass Ca<sup>2+</sup> or secondary <span class="highlight" style="background-color:">signaling</span> molecules in early <span class="highlight" style="background-color:">signaling</span> steps. GPCR, G-protein coupled receptor; InsP, inositol polyphosphates; RLK, receptor-like kinase. Other abbreviations are given in the text.</div></p></div><div class="aux"><div class="resc">Liming Xiong, et al. Plant Cell. 2002;14(Suppl):s165-s183.</div><p class="links"><a class="dblinks" href="/pubmed/12045276" ref="ordinalpos=107&amp;ncbi_uid=247092&amp;link_uid=12045276">Citation</a><a class="dblinks" href="/pmc/articles/PMC151254/" ref="ordinalpos=107&amp;ncbi_uid=247092&amp;link_uid=PMC151254">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>108.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3914444/figure/F2/" ref="ordinalpos=108&amp;ncbi_uid=5823752&amp;link_uid=PMC3914444" href="/pmc/articles/PMC3914444/"><img src="/pmc/articles/instance/3914444/bin/fonc-04-00013-g002.gif" src-large="/pmc/articles/instance/3914444/bin/fonc-04-00013-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3914444/" ref="ordinalpos=108&amp;ncbi_uid=5823752&amp;link_uid=PMC3914444" image-link="/pmc/articles/PMC3914444/figure/F2/" class="imagepopup">Figure 2.  From: Customizing the Therapeutic Response of <span class="highlight" style="background-color:">Signaling</span> Networks to Promote Antitumor Responses by Drug Combinations. </a></p><div class="supp"><p class="details"><div><b>Scheme of PI3K/PTEN/AKT <span class="highlight" style="background-color:">signaling</span> network activated by heterodimerization of HER2/HER3 receptors induced by heregulin (HRG) and inhibited by pertuzumab, 2C4</b>. Decomposition of the <span class="highlight" style="background-color:">signaling</span> network to upstream <span class="highlight" style="background-color:">pathway</span> (receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) and down-stream <span class="highlight" style="background-color:">pathway</span> (PI3K/PTEN/AKT network) is shown.</div></p></div><div class="aux"><div class="resc">Alexey Goltsov, et al. Front Oncol. 2014;4:13.</div><p class="links"><a class="dblinks" href="/pubmed/24551596" ref="ordinalpos=108&amp;ncbi_uid=5823752&amp;link_uid=24551596">Citation</a><a class="dblinks" href="/pmc/articles/PMC3914444/" ref="ordinalpos=108&amp;ncbi_uid=5823752&amp;link_uid=PMC3914444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>109.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3357548/figure/fig3/" ref="ordinalpos=109&amp;ncbi_uid=4028644&amp;link_uid=PMC3357548" href="/pmc/articles/PMC3357548/"><img src="/pmc/articles/instance/3357548/bin/JST2012-376470.003.gif" src-large="/pmc/articles/instance/3357548/bin/JST2012-376470.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3357548/" ref="ordinalpos=109&amp;ncbi_uid=4028644&amp;link_uid=PMC3357548" image-link="/pmc/articles/PMC3357548/figure/fig3/" class="imagepopup">Figure 3.  From: A Bioinformatics Resource for TWEAK-Fn14 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>TWEAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: illustration of the TWEAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as visualized in NetSlim web page. Each molecule is linked to its corresponding page in NetPath. Each reaction is linked to its respective PubMed citation. Dashed arrows represent the downstream reactions leading to the corresponding events while the solid arrows indicate the direct association between the indicated molecules. Gene symbol(s) has been used to denote proteins in the <span class="highlight" style="background-color:">pathway</span> map (refer to Synonymous for common names).</div></p></div><div class="aux"><div class="resc">Mitali Bhattacharjee, et al. J Signal Transduct. 2012;2012:376470.</div><p class="links"><a class="dblinks" href="/pubmed/22649723" ref="ordinalpos=109&amp;ncbi_uid=4028644&amp;link_uid=22649723">Citation</a><a class="dblinks" href="/pmc/articles/PMC3357548/" ref="ordinalpos=109&amp;ncbi_uid=4028644&amp;link_uid=PMC3357548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>110.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1304358/figure/fig2/" ref="ordinalpos=110&amp;ncbi_uid=663002&amp;link_uid=PMC1304358" href="/pmc/articles/PMC1304358/"><img src="/pmc/articles/instance/1304358/bin/biophysj00029884F02_HT.gif" src-large="/pmc/articles/instance/1304358/bin/biophysj00029884F02_HT.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1304358/" ref="ordinalpos=110&amp;ncbi_uid=663002&amp;link_uid=PMC1304358" image-link="/pmc/articles/PMC1304358/figure/fig2/" class="imagepopup">FIGURE 2.  From: The JAK-STAT <span class="highlight" style="background-color:">Signaling</span> Network in the Human B-Cell: An Extreme <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Analysis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> types in the JAK-STAT network. Other <span class="highlight" style="background-color:">signaling</span> types, not present in this network, include signal pleiotropy (one-input, multiple-outputs) and cross talk (multiple inputs, multiple outputs).</div></p></div><div class="aux"><div class="resc">Jason A. Papin, et al. Biophys J. 2004 July;87(1):37-46.</div><p class="links"><a class="dblinks" href="/pubmed/15240442" ref="ordinalpos=110&amp;ncbi_uid=663002&amp;link_uid=15240442">Citation</a><a class="dblinks" href="/pmc/articles/PMC1304358/" ref="ordinalpos=110&amp;ncbi_uid=663002&amp;link_uid=PMC1304358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>111.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2269037/figure/Fig3/" ref="ordinalpos=111&amp;ncbi_uid=1317748&amp;link_uid=PMC2269037" href="/pmc/articles/PMC2269037/"><img src="/pmc/articles/instance/2269037/bin/11568_2008_9018_Fig3_HTML.gif" src-large="/pmc/articles/instance/2269037/bin/11568_2008_9018_Fig3_HTML.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2269037/" ref="ordinalpos=111&amp;ncbi_uid=1317748&amp;link_uid=PMC2269037" image-link="/pmc/articles/PMC2269037/figure/Fig3/" class="imagepopup">Fig. 3.  From: 5?-androstane-3?,17?-diol selectively activates the canonical PI3K/AKT <span class="highlight" style="background-color:">pathway</span>: a bioinformatics-based evidence for androgen-activated cytoplasmic <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by 3α-diol. Networks of PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown. Genes that are up- or down-regulated by 3α-diol are marked by gradient fill, whereas genes that are not regulated by 3α-diol are marked as gray color. Genes that are not colored indicate the absence of these genes in the membrane array</div></p></div><div class="aux"><div class="resc">Mikhail G. Dozmorov, et al. Genomic Med. 2007 September;1(3-4):139-146.</div><p class="links"><a class="dblinks" href="/pubmed/18923939" ref="ordinalpos=111&amp;ncbi_uid=1317748&amp;link_uid=18923939">Citation</a><a class="dblinks" href="/pmc/articles/PMC2269037/" ref="ordinalpos=111&amp;ncbi_uid=1317748&amp;link_uid=PMC2269037">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>112.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3351446/figure/pone-0036975-g004/" ref="ordinalpos=112&amp;ncbi_uid=4008402&amp;link_uid=PMC3351446" href="/pmc/articles/PMC3351446/"><img src="/pmc/articles/instance/3351446/bin/pone.0036975.g004.gif" src-large="/pmc/articles/instance/3351446/bin/pone.0036975.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3351446/" ref="ordinalpos=112&amp;ncbi_uid=4008402&amp;link_uid=PMC3351446" image-link="/pmc/articles/PMC3351446/figure/pone-0036975-g004/" class="imagepopup">Figure 4. Latent semantic indexing correlations of KEGG <span class="highlight" style="background-color:">signaling</span> pathways terms with proteins..  From: GIT2 Acts as a Potential Keystone Protein in Functional Hypothalamic Networks Associated with Age-Related Phenotypic Changes in Rats. </a></p><div class="supp"><p class="details"><div>(A) Latent semantic indexing (LSI) interrogation matrix between input significantly-regulated KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> terms. Colored blocks represent the individual LSI implicit correlation of the specific protein (vertically organized on left of heatmap: 1–2524 – see Table S20) with the respective KEGG term (1-Regulation of actin cytoskeleton, 2-Chemokine <span class="highlight" style="background-color:">signaling</span>, 3-Alzheimer's disease, 4-Focal adhesion, 5-MAPK <span class="highlight" style="background-color:">signaling</span>, 6-Gap junction, 7-GnRH <span class="highlight" style="background-color:">signaling</span>, 8-Long term potentiation, 9-Notch <span class="highlight" style="background-color:">signaling</span>, 10-VEGF <span class="highlight" style="background-color:">signaling</span>, 11-p53 <span class="highlight" style="background-color:">signaling</span>, 12-Calcium <span class="highlight" style="background-color:">signaling</span>). The number of KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> correlations for each protein is indicated by the color of the respective heatmap blocks (9 correlations-red; 8 correlations-orange; 7 correlations-yellow; 6 correlations-green; 5 correlations-light blue; 4 correlations-dark blue; 3 correlations-purple; 2 correlations-grey). (B) Mean ± SEM for the total implicitly-correlating proteins for each of the 12 input KEGG <span class="highlight" style="background-color:">signaling</span> pathways. (C) Box and whisker plot with 1–99% statistical cut-offs (GraphPad Prism) of the number of specific correlations to KEGG pathways each protein possessed. Twelve proteins demonstrated a statistically-significantly greater number of KEGG <span class="highlight" style="background-color:">pathway</span> correlations compared to the total protein mean number of correlations (*** = p&lt;0.001). (D) Expanded heatmap identification of specific proteins possessing a significantly greater number of KEGG <span class="highlight" style="background-color:">pathway</span> correlations compared to the mean number of KEGG <span class="highlight" style="background-color:">pathway</span> correlations for all implicit proteins. (E) Mean ± SEM of LSI correlation scores (across all 9 correlations) for Grit and GIT2.</div></p></div><div class="aux"><div class="resc">Wayne Chadwick, et al. PLoS One. 2012;7(5):e36975.</div><p class="links"><a class="dblinks" href="/pubmed/22606319" ref="ordinalpos=112&amp;ncbi_uid=4008402&amp;link_uid=22606319">Citation</a><a class="dblinks" href="/pmc/articles/PMC3351446/" ref="ordinalpos=112&amp;ncbi_uid=4008402&amp;link_uid=PMC3351446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>113.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3180377/figure/pone-0025249-g009/" ref="ordinalpos=113&amp;ncbi_uid=3252235&amp;link_uid=PMC3180377" href="/pmc/articles/PMC3180377/"><img src="/pmc/articles/instance/3180377/bin/pone.0025249.g009.gif" src-large="/pmc/articles/instance/3180377/bin/pone.0025249.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3180377/" ref="ordinalpos=113&amp;ncbi_uid=3252235&amp;link_uid=PMC3180377" image-link="/pmc/articles/PMC3180377/figure/pone-0025249-g009/" class="imagepopup">Figure 9. Potential TLR <span class="highlight" style="background-color:">signaling</span> pathways for Ppargc1 metabolic co-activator gene activation after S. aureus infection..  From: A Toll-Like Receptor 2 <span class="highlight" style="background-color:">Pathway</span> Regulates the Ppargc1a/b Metabolic Co-Activators in Mice with Staphylococcal aureus Sepsis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> 1 shows the canonical TLR2 MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that activates NF-kB after <i>S. aureus</i>. <span class="highlight" style="background-color:">Pathway</span> 2 shows TLR4 MyD88-dependent <span class="highlight" style="background-color:">signaling</span> to NF-kB and MyD88–independent <span class="highlight" style="background-color:">signaling</span> to TRIF/TRAM. Both MyD88 pathways have been excluded as causes of the <i>Ppargc1a</i> gene expression. <span class="highlight" style="background-color:">Pathway</span> 3 shows a putative TLR2-TLR4 heterodimer interacting with TRIF/TRAM. <span class="highlight" style="background-color:">Pathway</span> 4 indicates TLR2 in the TLR4 null state, as a homodimer or a heterodimer involving a non-TLR4 partner such as TLR1 or 6, interacting with TRIF/TRAM and unmasking the innate immune regulation of <i>Ppargc1a</i> expression. <span class="highlight" style="background-color:">Pathway</span> 5 shows canonical TLR3 endosome <span class="highlight" style="background-color:">signaling</span> also excluded in <i>Ppargc1</i> gene regulation after <i>S. aureus</i>; however, independent TLR3 activation partially rescues the <i>Ppargc1</i> phenotype in mice. TIRAP is Toll/interleukin-1 receptor domain-containing adapter protein (MAL); IRAK4 is Interleukin-1 receptor-associated kinase 4; TRAF3 and TRAF6 are TNF receptor-associated factor 3 and 6; TAK1 is TGF-beta-activated kinase 1 and TBK1 is NF-kappa-B-activating kinase.</div></p></div><div class="aux"><div class="resc">Timothy E. Sweeney, et al. PLoS One. 2011;6(9):e25249.</div><p class="links"><a class="dblinks" href="/pubmed/21966468" ref="ordinalpos=113&amp;ncbi_uid=3252235&amp;link_uid=21966468">Citation</a><a class="dblinks" href="/pmc/articles/PMC3180377/" ref="ordinalpos=113&amp;ncbi_uid=3252235&amp;link_uid=PMC3180377">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>114.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3708595/figure/F4/" ref="ordinalpos=114&amp;ncbi_uid=5499198&amp;link_uid=PMC3708595" href="/pmc/articles/PMC3708595/"><img src="/pmc/articles/instance/3708595/bin/nihms474972f4.gif" src-large="/pmc/articles/instance/3708595/bin/nihms474972f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3708595/" ref="ordinalpos=114&amp;ncbi_uid=5499198&amp;link_uid=PMC3708595" image-link="/pmc/articles/PMC3708595/figure/F4/" class="imagepopup">Figure 4. Diversity and frequency of ACC genetic changes culminating in aberrant <span class="highlight" style="background-color:">signaling</span> pathways.  From: The Mutational Landscape of Adenoid Cystic Carcinoma. </a></p><div class="supp"><p class="details"><div>Alterations are defined by somatic mutations, homozygous deletions, high-level focal amplifications, and structural variants validated by FISH or PCR. Frequencies are expressed as a percentage of all cases. Red background denotes activating alteration, blue background denotes inactivating alteration, white background denotes numerous alterations, and purple background denotes alteration of unclear significance. Interactions based upon IPA or <span class="highlight" style="background-color:">Pathway</span> Commons.<sup>38</sup> (<b>a</b>). Epigenetic modification. (<b>b</b>). DNA damage/checkpoint <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>c</b>). MYB/MYC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, with fusion denoted by recurrent t(6;9) translocation between <i>MYB</i> and <i>NFIB</i>. (<b>d</b>). FGF/IGF/PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>e</b>). Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Allen S. Ho, et al. Nat Genet. ;45(7):10.1038/ng.2643.</div><p class="links"><a class="dblinks" href="/pubmed/23685749" ref="ordinalpos=114&amp;ncbi_uid=5499198&amp;link_uid=23685749">Citation</a><a class="dblinks" href="/pmc/articles/PMC3708595/" ref="ordinalpos=114&amp;ncbi_uid=5499198&amp;link_uid=PMC3708595">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>115.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2928279/figure/pone-0012405-g003/" ref="ordinalpos=115&amp;ncbi_uid=2449691&amp;link_uid=PMC2928279" href="/pmc/articles/PMC2928279/"><img src="/pmc/articles/instance/2928279/bin/pone.0012405.g003.gif" src-large="/pmc/articles/instance/2928279/bin/pone.0012405.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2928279/" ref="ordinalpos=115&amp;ncbi_uid=2449691&amp;link_uid=PMC2928279" image-link="/pmc/articles/PMC2928279/figure/pone-0012405-g003/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Pathway</span> analysis of induced Jak/Stat and PI3K activity reveals distinct profiles..  From: Distinct Patterns of DNA Damage Response and Apoptosis Correlate with Jak/Stat and PI3Kinase Response Profiles in Human Acute Myelogenous Leukemia. </a></p><div class="supp"><p class="details"><div>Principal Component Analysis (PCA) of NR samples (unfilled red squares) compared to CR samples (filled blue circles) for Jak/Stat and PI3K <span class="highlight" style="background-color:">signaling</span>. The metrics used in the analysis were “basal”, “fold” and “total”. (A) Basal <span class="highlight" style="background-color:">signaling</span>: basal metric values for p-Stat1, p-Stat3, p-Stat5 and p-Stat6 were chosen to represent Jak/Stat <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>, while the basal values for p-Erk, p-S6 and p-Akt were chosen to represent the PI3K <span class="highlight" style="background-color:">pathway</span>. The first principal component for the Jak/Stat <span class="highlight" style="background-color:">pathway</span> explained 85% of the variance while the one for the PI3K <span class="highlight" style="background-color:">pathway</span> explained 71% of the variance. (B) Induced <span class="highlight" style="background-color:">signaling</span>: the metrics used in the analysis were “fold” and “total” metric for the same intra-cellular readouts used for the PCA of basal <span class="highlight" style="background-color:">signaling</span>. The percentage of variance explained by the first principal component for induced Jak/Stat <span class="highlight" style="background-color:">signaling</span> was 66% and for induced PI3K <span class="highlight" style="background-color:">signaling</span> it was 46%. The PCA reveals a heterogeneity of profiles for NR samples when treated with modulators which is not seen in the PCA of untreated samples in their basal state.</div></p></div><div class="aux"><div class="resc">David B. Rosen, et al. PLoS One. 2010;5(8):e12405.</div><p class="links"><a class="dblinks" href="/pubmed/20811632" ref="ordinalpos=115&amp;ncbi_uid=2449691&amp;link_uid=20811632">Citation</a><a class="dblinks" href="/pmc/articles/PMC2928279/" ref="ordinalpos=115&amp;ncbi_uid=2449691&amp;link_uid=PMC2928279">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>116.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC185249/figure/F3/" ref="ordinalpos=116&amp;ncbi_uid=211878&amp;link_uid=PMC185249" href="/pmc/articles/PMC185249/"><img src="/pmc/articles/instance/185249/bin/1465-9921-4-5-3.gif" src-large="/pmc/articles/instance/185249/bin/1465-9921-4-5-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC185249/" ref="ordinalpos=116&amp;ncbi_uid=211878&amp;link_uid=PMC185249" image-link="/pmc/articles/PMC185249/figure/F3/" class="imagepopup">Figure 3.  From: Growth factor <span class="highlight" style="background-color:">signaling</span> in lung morphogenetic centers: automaticity, stereotypy and symmetry. </a></p><div class="supp"><p class="details"><div><b>Sprouty is a rapidly inducible negative regulator of fibroblast growth factor (FGF) <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. </b>The figure shows a model describing the interaction of murine Sprouty2 (mSPRY2) with other key <span class="highlight" style="background-color:">signaling</span> proteins in the FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the upper panel, the FGF <span class="highlight" style="background-color:">pathway</span> is shown <span class="highlight" style="background-color:">signaling</span> the activation of MAP kinase/ERK2 via the FGFR, FRS2, Shp2, Grb2, Sos, Ras and Raf <span class="highlight" style="background-color:">pathway</span>. In the lower panel Sprouty2 (SPRY2) is shown binding FRS2 and Grb2 and displacing Shp2 from FRS2 and Grb2, thereby preventing subsequent activation of the Sos, Ras-GAP, Raf <span class="highlight" style="background-color:">pathway</span>, resulting in net inhibition of MAP kinase/ERK2 activation.</div></p></div><div class="aux"><div class="resc">David Warburton, et al. Respir Res. 2003;4(1):5-5.</div><p class="links"><a class="dblinks" href="/pubmed/12818006" ref="ordinalpos=116&amp;ncbi_uid=211878&amp;link_uid=12818006">Citation</a><a class="dblinks" href="/pmc/articles/PMC185249/" ref="ordinalpos=116&amp;ncbi_uid=211878&amp;link_uid=PMC185249">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>117.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3063241/figure/pone-0018066-g004/" ref="ordinalpos=117&amp;ncbi_uid=2848593&amp;link_uid=PMC3063241" href="/pmc/articles/PMC3063241/"><img src="/pmc/articles/instance/3063241/bin/pone.0018066.g004.gif" src-large="/pmc/articles/instance/3063241/bin/pone.0018066.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3063241/" ref="ordinalpos=117&amp;ncbi_uid=2848593&amp;link_uid=PMC3063241" image-link="/pmc/articles/PMC3063241/figure/pone-0018066-g004/" class="imagepopup">Figure 4. Log2 fold change of microarray estimated expression for genes from an investigated <span class="highlight" style="background-color:">pathway</span> are plotted as a cumulative distribution function (in red color) and compared with the corresponding cumulative distribution function for the rest of genes (i.e., not belonging to the investigated <span class="highlight" style="background-color:">pathway</span>, in blue color)..  From: Gene Expression Profiles in Stage I Uterine Serous Carcinoma in Comparison to Grade 3 and Grade 1 Stage I Endometrioid Adenocarcinoma. </a></p><div class="supp"><p class="details"><div>A) P53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in USC vs. EAC-G3 comparison. The two cumulative distribution functions are significantly different (P = 4.5e-3) by the Kolmogorov-Smirnov test. B) P53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in USC vs. EAC-G1 comparison. C) PTEN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in USC vs. EAC-G3 comparison. D) PTEN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in USC vs. EAC-G1 comparison.</div></p></div><div class="aux"><div class="resc">Paulette Mhawech-Fauceglia, et al. PLoS One. 2011;6(3):e18066.</div><p class="links"><a class="dblinks" href="/pubmed/21448288" ref="ordinalpos=117&amp;ncbi_uid=2848593&amp;link_uid=21448288">Citation</a><a class="dblinks" href="/pmc/articles/PMC3063241/" ref="ordinalpos=117&amp;ncbi_uid=2848593&amp;link_uid=PMC3063241">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>118.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2741634/figure/F2/" ref="ordinalpos=118&amp;ncbi_uid=1992814&amp;link_uid=PMC2741634" href="/pmc/articles/PMC2741634/"><img src="/pmc/articles/instance/2741634/bin/nihms134316f2.gif" src-large="/pmc/articles/instance/2741634/bin/nihms134316f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2741634/" ref="ordinalpos=118&amp;ncbi_uid=1992814&amp;link_uid=PMC2741634" image-link="/pmc/articles/PMC2741634/figure/F2/" class="imagepopup">Figure 2. Compartmentalized <span class="highlight" style="background-color:">signaling</span> of TLR4.  From: How location governs Toll like receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>TLR4 binds to LPS-MD2 complexes and engages MyD88- TIRAP adaptors at the cell surface leading to activation of early MAP kinase <span class="highlight" style="background-color:">pathway</span>. TLR4-LPS complex is endocytosed through a dynamin-dependent <span class="highlight" style="background-color:">pathway</span> terminating the MyD88-TIRAP <span class="highlight" style="background-color:">signaling</span>. TLR4-LPS complex within the endosome initiates a second phase of <span class="highlight" style="background-color:">signaling</span> by engaging TRAM-TRIF adaptors that leads to the production of interferon-β.</div></p></div><div class="aux"><div class="resc">Akanksha Chaturvedi, et al. Traffic. ;10(6):621-628.</div><p class="links"><a class="dblinks" href="/pubmed/19302269" ref="ordinalpos=118&amp;ncbi_uid=1992814&amp;link_uid=19302269">Citation</a><a class="dblinks" href="/pmc/articles/PMC2741634/" ref="ordinalpos=118&amp;ncbi_uid=1992814&amp;link_uid=PMC2741634">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>119.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4000238/figure/F1/" ref="ordinalpos=119&amp;ncbi_uid=6061068&amp;link_uid=PMC4000238" href="/pmc/articles/PMC4000238/"><img src="/pmc/articles/instance/4000238/bin/nihms568219f1.gif" src-large="/pmc/articles/instance/4000238/bin/nihms568219f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4000238/" ref="ordinalpos=119&amp;ncbi_uid=6061068&amp;link_uid=PMC4000238" image-link="/pmc/articles/PMC4000238/figure/F1/" class="imagepopup">Figure 1.  From: Wnt and the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in bone development and disease. </a></p><div class="supp"><p class="details"><div>The Wnt <span class="highlight" style="background-color:">pathway</span> and its interactions with other pathways in bone. (a)Wnt <span class="highlight" style="background-color:">signaling</span> and the TGF-β/BMP-Smad <span class="highlight" style="background-color:">pathway</span> influence each other during bone development. The Wnt <span class="highlight" style="background-color:">pathway</span> participates in R-Smad degradation and so does the Bmp <span class="highlight" style="background-color:">pathway</span> in β-catenin degradation. (b. c)The Wnt canonical <span class="highlight" style="background-color:">pathway</span>: Wnt proteins, following their binding to a frizzled receptor and a Lrp co-receptor (most likely LRP6), activate the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. These receptors transduce a signal to several intracellular proteins that include Dishevelled (Dsh), glycogen synthase kinase-3β(GSK-3), Axin, Adenomatous Polyposis Coli(APC), and the transcriptional regulator, β-catenin. This results in the translocation to nucleus of β-catenin, β-catenin’s association with members of the Lef1/Tcf family of nuclear proteins, and activation of a specific program of gene expression. (d) Wnt interacts with PTH1R, decreases wnt antagonists, Sost, WIF1 and Dkk expression, and sustains β-catenin stabilization. (e, f) The Wnt noncanonical <span class="highlight" style="background-color:">pathway</span>. The Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> and PCP <span class="highlight" style="background-color:">pathway</span> affect osteoblastogenesis and osteoclastogenesis.</div></p></div><div class="aux"><div class="resc">Yiping Wang, et al. Front Biosci (Landmark Ed). 2014 January 1;19:379-407.</div><p class="links"><a class="dblinks" href="/pubmed/24389191" ref="ordinalpos=119&amp;ncbi_uid=6061068&amp;link_uid=24389191">Citation</a><a class="dblinks" href="/pmc/articles/PMC4000238/" ref="ordinalpos=119&amp;ncbi_uid=6061068&amp;link_uid=PMC4000238">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>120.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3263596/figure/F2/" ref="ordinalpos=120&amp;ncbi_uid=3769454&amp;link_uid=PMC3263596" href="/pmc/articles/PMC3263596/"><img src="/pmc/articles/instance/3263596/bin/bar032f2.gif" src-large="/pmc/articles/instance/3263596/bin/bar032f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3263596/" ref="ordinalpos=120&amp;ncbi_uid=3769454&amp;link_uid=PMC3263596" image-link="/pmc/articles/PMC3263596/figure/F2/" class="imagepopup">Figure 2. From: NetSlim: high-confidence curated <span class="highlight" style="background-color:">signaling</span> maps. </a></p><div class="supp"><p class="details"><div>An overview of NetSlim. The NetSlim home page has options to browse the <span class="highlight" style="background-color:">pathway</span> maps. A description of the criteria used for NetSlim is available as a separate tab. Choosing a <span class="highlight" style="background-color:">pathway</span> such as the TGF beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the ‘<span class="highlight" style="background-color:">Pathway</span> map’ tab provides access to the TGF-β NetSlim <span class="highlight" style="background-color:">pathway</span> map image and the ‘Description’ tab which has a brief description of the <span class="highlight" style="background-color:">pathway</span>. The ‘map with citation’ provided in the NetSlim <span class="highlight" style="background-color:">pathway</span> page contains molecules linked to specific NetPath molecule pages and their reactions linked to their corresponding citations. NetSlim maps can be downloaded in different file formats from specific <span class="highlight" style="background-color:">pathway</span> pages. The NetSlim data is also available to download in BioPAX, PSI-MI and SBML formats. A ‘Feedback’ page allows users to provide comments about the NetSlim resource. A link to NetSlim is provided on each <span class="highlight" style="background-color:">pathway</span> page in NetPath and <i>vice versa</i>.</div></p></div><div class="aux"><div class="resc">Rajesh Raju, et al. Database (Oxford). 2011;2011:bar032.</div><p class="links"><a class="dblinks" href="/pubmed/21959865" ref="ordinalpos=120&amp;ncbi_uid=3769454&amp;link_uid=21959865">Citation</a><a class="dblinks" href="/pmc/articles/PMC3263596/" ref="ordinalpos=120&amp;ncbi_uid=3769454&amp;link_uid=PMC3263596">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>121.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3414439/figure/pone-0042573-g004/" ref="ordinalpos=121&amp;ncbi_uid=4182476&amp;link_uid=PMC3414439" href="/pmc/articles/PMC3414439/"><img src="/pmc/articles/instance/3414439/bin/pone.0042573.g004.gif" src-large="/pmc/articles/instance/3414439/bin/pone.0042573.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3414439/" ref="ordinalpos=121&amp;ncbi_uid=4182476&amp;link_uid=PMC3414439" image-link="/pmc/articles/PMC3414439/figure/pone-0042573-g004/" class="imagepopup">Figure 4. Network map of halofantrine and vinblastine on triple-negative breast cancer using phExplorer..  From: CDA: Combinatorial Drug Discovery Using Transcriptional Response Modules. </a></p><div class="supp"><p class="details"><div>(A) It seems that halofantrine and vinblastine could affect on five different <span class="highlight" style="background-color:">signaling</span> pathways in TNBC. Group 5: Halofantrine- or vinblatine-related proteins which are also related with proteins of A6B1 and A6B4 Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Group 6: Proteins which are related with vinblasitne as well as proteins of EGFR family <span class="highlight" style="background-color:">signaling</span> pathways (such as ERBB1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, ERBB2/ERBB3 <span class="highlight" style="background-color:">signaling</span> events, ERBB4 <span class="highlight" style="background-color:">signaling</span> events, ERBB receptor <span class="highlight" style="background-color:">signaling</span> network). (B) We hypnotized that halofantrine and vinblastine are synergistic because they complementary regulate integrin and EGFR <span class="highlight" style="background-color:">signaling</span> pathways. Group 0: A part of EGFR family <span class="highlight" style="background-color:">signaling</span> pathways. Group 1: A part of A6B1 and A6B4 Integrin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ji-Hyun Lee, et al. PLoS One. 2012;7(8):e42573.</div><p class="links"><a class="dblinks" href="/pubmed/22905152" ref="ordinalpos=121&amp;ncbi_uid=4182476&amp;link_uid=22905152">Citation</a><a class="dblinks" href="/pmc/articles/PMC3414439/" ref="ordinalpos=121&amp;ncbi_uid=4182476&amp;link_uid=PMC3414439">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>122.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC86880/figure/F1/" ref="ordinalpos=122&amp;ncbi_uid=38373&amp;link_uid=PMC86880" href="/pmc/articles/PMC86880/"><img src="/pmc/articles/instance/86880/bin/mb0711517001.gif" src-large="/pmc/articles/instance/86880/bin/mb0711517001.jpg" alt="FIG. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC86880/" ref="ordinalpos=122&amp;ncbi_uid=38373&amp;link_uid=PMC86880" image-link="/pmc/articles/PMC86880/figure/F1/" class="imagepopup">FIG. 1.  From: Interleukin-1 (IL-1) Receptor-Associated Kinase Leads to Activation of TAK1 by Inducing TAB2 Translocation in the  IL-1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Receptor proximal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for IL-1.</div></p></div><div class="aux"><div class="resc">Giichi Takaesu, et al. Mol Cell Biol. 2001 April;21(7):2475-2484.</div><p class="links"><a class="dblinks" href="/pubmed/11259596" ref="ordinalpos=122&amp;ncbi_uid=38373&amp;link_uid=11259596">Citation</a><a class="dblinks" href="/pmc/articles/PMC86880/" ref="ordinalpos=122&amp;ncbi_uid=38373&amp;link_uid=PMC86880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>123.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2825570/figure/F1/" ref="ordinalpos=123&amp;ncbi_uid=2490545&amp;link_uid=PMC2825570" href="/pmc/articles/PMC2825570/"><img src="/pmc/articles/instance/2825570/bin/nihms160203f1.gif" src-large="/pmc/articles/instance/2825570/bin/nihms160203f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2825570/" ref="ordinalpos=123&amp;ncbi_uid=2490545&amp;link_uid=PMC2825570" image-link="/pmc/articles/PMC2825570/figure/F1/" class="imagepopup">Figure 1. The Smad and Non-Smad <span class="highlight" style="background-color:">pathway</span> of BMP/TGFβ <span class="highlight" style="background-color:">signaling</span>.  From: Smad <span class="highlight" style="background-color:">Signaling</span> in Skeletal Development and Regeneration. </a></p><div class="supp"><p class="details"><div>BMP/TGFβ <span class="highlight" style="background-color:">signaling</span> <i>in vivo</i> is mediated by either the Smad <span class="highlight" style="background-color:">pathway</span> (canonical) or the Non-Smad (non-canonical) pathways. Both pathways could mediate important functions of BMP/TGFβ in skeletogenesis. The non-Smad pathways also regulate Smad proteins by modulating linker region and C-terminal phosphorylation. Moreover, Smad proteins also modulate activities of molecules in Non-Smad pathways. The preference of intracellular pathways by different receptor complexes, the crosstalk between these two pathways, and the exact <span class="highlight" style="background-color:">signaling</span> mechanisms of the Non-Smad pathways in skeletal system remain to be investigated.</div></p></div><div class="aux"><div class="resc">Buer Song, et al. Cytokine Growth Factor Rev. ;20(5-6):379-388.</div><p class="links"><a class="dblinks" href="/pubmed/19926329" ref="ordinalpos=123&amp;ncbi_uid=2490545&amp;link_uid=19926329">Citation</a><a class="dblinks" href="/pmc/articles/PMC2825570/" ref="ordinalpos=123&amp;ncbi_uid=2490545&amp;link_uid=PMC2825570">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>124.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2689134/figure/F4/" ref="ordinalpos=124&amp;ncbi_uid=1866959&amp;link_uid=PMC2689134" href="/pmc/articles/PMC2689134/"><img src="/pmc/articles/instance/2689134/bin/JCI38937.f4.gif" src-large="/pmc/articles/instance/2689134/bin/JCI38937.f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2689134/" ref="ordinalpos=124&amp;ncbi_uid=1866959&amp;link_uid=PMC2689134" image-link="/pmc/articles/PMC2689134/figure/F4/" class="imagepopup">Figure 4. <span class="highlight" style="background-color:">Signaling</span> pathways involved in the pathogenesis of HPE..  From: Pathogenesis of holoprosencephaly. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Nodal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) Bmp <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.                      (<b>C</b>) Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ActR2A, activin A receptor, type 2A;                   BmpR2, Bmp receptor 2; Disp, dispatched; Ptch, patched; Smo, smoothened; Tld,                   tolloid. </div></p></div><div class="aux"><div class="resc">Xin Geng, et al. J Clin Invest. 2009 June 1;119(6):1403-1413.</div><p class="links"><a class="dblinks" href="/pubmed/19487816" ref="ordinalpos=124&amp;ncbi_uid=1866959&amp;link_uid=19487816">Citation</a><a class="dblinks" href="/pmc/articles/PMC2689134/" ref="ordinalpos=124&amp;ncbi_uid=1866959&amp;link_uid=PMC2689134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>125.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3063861/figure/F1/" ref="ordinalpos=125&amp;ncbi_uid=3656584&amp;link_uid=PMC3063861" href="/pmc/articles/PMC3063861/"><img src="/pmc/articles/instance/3063861/bin/nihms252493f1.gif" src-large="/pmc/articles/instance/3063861/bin/nihms252493f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3063861/" ref="ordinalpos=125&amp;ncbi_uid=3656584&amp;link_uid=PMC3063861" image-link="/pmc/articles/PMC3063861/figure/F1/" class="imagepopup">Figure 1. GPCR <span class="highlight" style="background-color:">signaling</span>.  From: ?-arrestin: a <span class="highlight" style="background-color:">signaling</span> molecule and potential therapeutic target for heart failure. </a></p><div class="supp"><p class="details"><div>In response to ligand stimulation, GPCR <span class="highlight" style="background-color:">signaling</span> can occur via both G-protein mediated <span class="highlight" style="background-color:">pathway</span> (which leads to activation of second messengers) or via β-arrestin mediated <span class="highlight" style="background-color:">pathway</span> which further stimulate a plethora of downstream <span class="highlight" style="background-color:">signaling</span> pathways (MAPK, PI3K, AKT, EGFR etc.) (Reproduced with permission from Ref [31]).</div></p></div><div class="aux"><div class="resc">Nabila Noor, et al. J Mol Cell Cardiol. ;51(4):534-541.</div><p class="links"><a class="dblinks" href="/pubmed/21074538" ref="ordinalpos=125&amp;ncbi_uid=3656584&amp;link_uid=21074538">Citation</a><a class="dblinks" href="/pmc/articles/PMC3063861/" ref="ordinalpos=125&amp;ncbi_uid=3656584&amp;link_uid=PMC3063861">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>126.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285584/figure/pgen-1002524-g009/" ref="ordinalpos=126&amp;ncbi_uid=3832068&amp;link_uid=PMC3285584" href="/pmc/articles/PMC3285584/"><img src="/pmc/articles/instance/3285584/bin/pgen.1002524.g009.gif" src-large="/pmc/articles/instance/3285584/bin/pgen.1002524.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285584/" ref="ordinalpos=126&amp;ncbi_uid=3832068&amp;link_uid=PMC3285584" image-link="/pmc/articles/PMC3285584/figure/pgen-1002524-g009/" class="imagepopup">Figure 9. Model for the role of Tgifs in <span class="highlight" style="background-color:">signaling</span> during forebrain development..  From: Loss of Tgif Function Causes Holoprosencephaly by Disrupting the Shh <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A tentative model is shown that describes the data presented here. Briefly, Tgifs limit Smad2 transcriptional activity, which is required for activation of <i>Fgf8</i> expression. Tgif regulation of the Nodal-Smad2 <span class="highlight" style="background-color:">pathway</span> is required for the correct balance between Gli3 and Shh activity in the Shh <span class="highlight" style="background-color:">pathway</span>. Dashed lines indicate that the links from the Nodal-Smad2 <span class="highlight" style="background-color:">pathway</span> to Shh <span class="highlight" style="background-color:">signaling</span> components may not be direct, and that the regulation may be of both <i>Shh</i> and <i>Gli3</i>, or may occur primarily via one of them.</div></p></div><div class="aux"><div class="resc">Kenichiro Taniguchi, et al. PLoS Genet. 2012 February;8(2):e1002524.</div><p class="links"><a class="dblinks" href="/pubmed/22383895" ref="ordinalpos=126&amp;ncbi_uid=3832068&amp;link_uid=22383895">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285584/" ref="ordinalpos=126&amp;ncbi_uid=3832068&amp;link_uid=PMC3285584">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>127.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3685346/figure/F0001/" ref="ordinalpos=127&amp;ncbi_uid=5010722&amp;link_uid=PMC3685346" href="/pmc/articles/PMC3685346/"><img src="/pmc/articles/instance/3685346/bin/WO-17-20396-g001.gif" src-large="/pmc/articles/instance/3685346/bin/WO-17-20396-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3685346/" ref="ordinalpos=127&amp;ncbi_uid=5010722&amp;link_uid=PMC3685346" image-link="/pmc/articles/PMC3685346/figure/F0001/" class="imagepopup">Fig. 1.  From: Role of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in gastric cancer pathogenesis. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> – details in the text</div></p></div><div class="aux"><div class="resc">Marlena Brzozowa, et al. Contemp Oncol (Pozn). 2013;17(1):1-5.</div><p class="links"><a class="dblinks" href="/pubmed/23788953" ref="ordinalpos=127&amp;ncbi_uid=5010722&amp;link_uid=23788953">Citation</a><a class="dblinks" href="/pmc/articles/PMC3685346/" ref="ordinalpos=127&amp;ncbi_uid=5010722&amp;link_uid=PMC3685346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>128.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2763551/figure/F3/" ref="ordinalpos=128&amp;ncbi_uid=2094097&amp;link_uid=PMC2763551" href="/pmc/articles/PMC2763551/"><img src="/pmc/articles/instance/2763551/bin/nihms-110166-f0003.gif" src-large="/pmc/articles/instance/2763551/bin/nihms-110166-f0003.jpg" alt="Fig 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2763551/" ref="ordinalpos=128&amp;ncbi_uid=2094097&amp;link_uid=PMC2763551" image-link="/pmc/articles/PMC2763551/figure/F3/" class="imagepopup">Fig 3. Influence of <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Crosstalk on <span class="highlight" style="background-color:">Signaling</span>.  From: Perspectives on the quantitative immunobiology of the IL-7 <span class="highlight" style="background-color:">signaling</span> network. </a></p><div class="supp"><p class="details"><div><b>(a)</b> Illustration of <span class="highlight" style="background-color:">signaling</span> through the common Jak-Stat <span class="highlight" style="background-color:">pathway</span> downstream of IL-7Rα and IL-15R. <b>(b)</b> Example of the anticipated kinetics of Stat5 phosphorylation after stimulation with either IL-7 or IL-15 or both assuming constitutive Stat5 dephosphorylation rates. In this case, the combined IL-7 and IL-15 signal would be expected to give more rapid Stat5 phosphorylation kinetics than each of the cytokines alone, but a less than additive level of phosphorylation at the peak of the response due to limiting amounts of Jak1 and Stat5.</div></p></div><div class="aux"><div class="resc">Megan J. Palmer, et al. Cell Mol Immunol. ;5(2):79-89.</div><p class="links"><a class="dblinks" href="/pubmed/18445337" ref="ordinalpos=128&amp;ncbi_uid=2094097&amp;link_uid=18445337">Citation</a><a class="dblinks" href="/pmc/articles/PMC2763551/" ref="ordinalpos=128&amp;ncbi_uid=2094097&amp;link_uid=PMC2763551">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>129.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2613546/figure/F5/" ref="ordinalpos=129&amp;ncbi_uid=1657805&amp;link_uid=PMC2613546" href="/pmc/articles/PMC2613546/"><img src="/pmc/articles/instance/2613546/bin/nihms79323f5.gif" src-large="/pmc/articles/instance/2613546/bin/nihms79323f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2613546/" ref="ordinalpos=129&amp;ncbi_uid=1657805&amp;link_uid=PMC2613546" image-link="/pmc/articles/PMC2613546/figure/F5/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Signaling</span> in resistant and sensitive lines.  From: Mutations in the phosphatidylinositol-3-kinase <span class="highlight" style="background-color:">pathway</span> predict for antitumor activity of the inhibitor PX-866 while oncogenic Ras is a dominant predictor for resistance. </a></p><div class="supp"><p class="details"><div>Diagram showing the interactions of the cell <span class="highlight" style="background-color:">signaling</span> studied in sensitive and resistant lines. <span class="highlight" style="background-color:">Signaling</span> in PX-866 sensitive tumors comes from an increased reliance on the PI-3-kinase <span class="highlight" style="background-color:">pathway</span>, arising from aberrant activation through growth factors (GF) or mutated components of the <span class="highlight" style="background-color:">pathway</span> itself (*). Tumors with an activated Ras protein show a minimal response to inhibition of the PI-3-kinase <span class="highlight" style="background-color:">pathway</span> due to a shared reliance on alternate <span class="highlight" style="background-color:">signaling</span> pathways including the Raf and RalGDS pathways.</div></p></div><div class="aux"><div class="resc">NathanT. Ihle, et al. Cancer Res. ;69(1):143-150.</div><p class="links"><a class="dblinks" href="/pubmed/19117997" ref="ordinalpos=129&amp;ncbi_uid=1657805&amp;link_uid=19117997">Citation</a><a class="dblinks" href="/pmc/articles/PMC2613546/" ref="ordinalpos=129&amp;ncbi_uid=1657805&amp;link_uid=PMC2613546">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>130.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3667388/figure/F1/" ref="ordinalpos=130&amp;ncbi_uid=5653041&amp;link_uid=PMC3667388" href="/pmc/articles/PMC3667388/"><img src="/pmc/articles/instance/3667388/bin/nihms465974f1.gif" src-large="/pmc/articles/instance/3667388/bin/nihms465974f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3667388/" ref="ordinalpos=130&amp;ncbi_uid=5653041&amp;link_uid=PMC3667388" image-link="/pmc/articles/PMC3667388/figure/F1/" class="imagepopup">Figure 1. Regulation of TLR7/9 <span class="highlight" style="background-color:">signaling</span> by pDC surface receptors.  From: Regulation of TLR7/9 <span class="highlight" style="background-color:">signaling</span> in plasmacytoid dendritic cells. </a></p><div class="supp"><p class="details"><div>PDCs express ITAM-associating or ITIM-containing receptors that play an important role in the TLR7/9-induced responses. Upon ligands engagement, TLR7/9 are activated and lead to IFNα secretion. Activation of the ITAM-associating receptors including FcεR1α, ILT7, BDCA2, NKp44 and Siglec-H negatively regulates TLR7/9 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, except pDC-TREM that is induced by IFNα and enhances TLR7/9 <span class="highlight" style="background-color:">signaling</span>. CD2AP and SHIP1 are essential components and are required for ITAM <span class="highlight" style="background-color:">signaling</span> pathways. In contrast, activation of ITIM-containing receptors like Ly49Q and CD300a positively regulate TLR7/9 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Musheng Bao, et al. Protein Cell. ;4(1):40-52.</div><p class="links"><a class="dblinks" href="/pubmed/23132256" ref="ordinalpos=130&amp;ncbi_uid=5653041&amp;link_uid=23132256">Citation</a><a class="dblinks" href="/pmc/articles/PMC3667388/" ref="ordinalpos=130&amp;ncbi_uid=5653041&amp;link_uid=PMC3667388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>131.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2449355/figure/F1/" ref="ordinalpos=131&amp;ncbi_uid=1438344&amp;link_uid=PMC2449355" href="/pmc/articles/PMC2449355/"><img src="/pmc/articles/instance/2449355/bin/zpq9990835010001.gif" src-large="/pmc/articles/instance/2449355/bin/zpq9990835010001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2449355/" ref="ordinalpos=131&amp;ncbi_uid=1438344&amp;link_uid=PMC2449355" image-link="/pmc/articles/PMC2449355/figure/F1/" class="imagepopup">Fig. 1. From: Identification of the TFII-I family target genes in the vertebrate genome. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> involvement of BEN- (<i>A</i>) and TFII-I- (<i>B</i>) modulated genes. <i>y</i> axis, <i>P</i> values for significance of enrichment; <i>x</i> axis, KEGG <span class="highlight" style="background-color:">pathway</span> terms, listed as follows: 1, arachidonic acid metabolism; 2, nitrogen metabolism; 3, hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; 4, type I diabetes mellitus; 5, calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; 6, maturity-onset diabetes of the young; 7, cell adhesion molecules (CAMS); 8, hematopoietic cell lineage; 9, cytokine–cytokine receptor interaction; 10, complement and coagulation cascades; 11, neuroactive ligand–receptor interaction; 12, cell cycle; 13, ribosome; 14, proteasome; 15, focal adhesion; 16, apoptosis; 17, insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; 18, antigen processing and presentation; 19, adherens junction; 20, MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; 21, ATP synthesis; 22, gap junction; 23, citrate cycle (tricarboxylic acid); 24, oxidative phosphorylation; 25, TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; 26, reductive carboxylate cycle (CO<sub>2</sub> fixation); 27, neurodegenerative disorders; 28, FcεRI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; 29, Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; 30, dorso–ventral axis formation; 31, leukocyte transendothelial migration; 32, cysteine metabolism.</div></p></div><div class="aux"><div class="resc">Nyam-Osor Chimge, et al. Proc Natl Acad Sci U S A. 2008 July 1;105(26):9006-9010.</div><p class="links"><a class="dblinks" href="/pubmed/18579769" ref="ordinalpos=131&amp;ncbi_uid=1438344&amp;link_uid=18579769">Citation</a><a class="dblinks" href="/pmc/articles/PMC2449355/" ref="ordinalpos=131&amp;ncbi_uid=1438344&amp;link_uid=PMC2449355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>132.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3922814/figure/pone-0088386-g006/" ref="ordinalpos=132&amp;ncbi_uid=5819251&amp;link_uid=PMC3922814" href="/pmc/articles/PMC3922814/"><img src="/pmc/articles/instance/3922814/bin/pone.0088386.g006.gif" src-large="/pmc/articles/instance/3922814/bin/pone.0088386.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3922814/" ref="ordinalpos=132&amp;ncbi_uid=5819251&amp;link_uid=PMC3922814" image-link="/pmc/articles/PMC3922814/figure/pone-0088386-g006/" class="imagepopup">Figure 6. Hh <span class="highlight" style="background-color:">signaling</span> regulates ovarian cancer cell migration and invasion via integrin β4-mediated FAK <span class="highlight" style="background-color:">signaling</span>..  From: Down-Regulation of Gli Transcription Factor Leads to the Inhibition of Migration and Invasion of Ovarian Cancer Cells via Integrin ?4-Mediated FAK <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed role of Hh <span class="highlight" style="background-color:">signaling</span> activation involving ITGB4/FAK dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Qi Chen, et al. PLoS One. 2014;9(2):e88386.</div><p class="links"><a class="dblinks" href="/pubmed/24533083" ref="ordinalpos=132&amp;ncbi_uid=5819251&amp;link_uid=24533083">Citation</a><a class="dblinks" href="/pmc/articles/PMC3922814/" ref="ordinalpos=132&amp;ncbi_uid=5819251&amp;link_uid=PMC3922814">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>133.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3493591/figure/pone-0048935-g005/" ref="ordinalpos=133&amp;ncbi_uid=4360311&amp;link_uid=PMC3493591" href="/pmc/articles/PMC3493591/"><img src="/pmc/articles/instance/3493591/bin/pone.0048935.g005.gif" src-large="/pmc/articles/instance/3493591/bin/pone.0048935.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3493591/" ref="ordinalpos=133&amp;ncbi_uid=4360311&amp;link_uid=PMC3493591" image-link="/pmc/articles/PMC3493591/figure/pone-0048935-g005/" class="imagepopup">Figure 5. Schematic representation of the classic Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and model of US28-mediated activation of ß-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Constitutive ss-Catenin <span class="highlight" style="background-color:">Signaling</span> by the Viral Chemokine Receptor US28. </a></p><div class="supp"><p class="details"><div>The left side of the model indicates components of the classic Wnt/Frizzled mediated activation of ß-catenin. In this <span class="highlight" style="background-color:">pathway</span> the disruption complex (Axin, APC) that enables GSK3ß- and Caseine kinase 1 (CK1)-mediated phosphorylation of ß-catenin leading to its degradation, is disrupted in a Dishevelled-mediated way upon Wnt binding to Frizzled/LRP. US28 activates ß-catenin <span class="highlight" style="background-color:">signaling</span> in a ligand-dependent and independent manner, involving respectively Gα<sub>12/13</sub> and Gα<sub>q</sub> proteins and respective RhoGEFs, converging at RhoA/ROCK, resulting in increased Tcf-Lef transcriptional activation.</div></p></div><div class="aux"><div class="resc">Ellen V. Langemeijer, et al. PLoS One. 2012;7(11):e48935.</div><p class="links"><a class="dblinks" href="/pubmed/23145028" ref="ordinalpos=133&amp;ncbi_uid=4360311&amp;link_uid=23145028">Citation</a><a class="dblinks" href="/pmc/articles/PMC3493591/" ref="ordinalpos=133&amp;ncbi_uid=4360311&amp;link_uid=PMC3493591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>134.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3501616/figure/F4/" ref="ordinalpos=134&amp;ncbi_uid=5434847&amp;link_uid=PMC3501616" href="/pmc/articles/PMC3501616/"><img src="/pmc/articles/instance/3501616/bin/nihms402023f4.gif" src-large="/pmc/articles/instance/3501616/bin/nihms402023f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3501616/" ref="ordinalpos=134&amp;ncbi_uid=5434847&amp;link_uid=PMC3501616" image-link="/pmc/articles/PMC3501616/figure/F4/" class="imagepopup">Figure 4. Model of hypothetical <span class="highlight" style="background-color:">pathway</span> involving IRF6, ARHGAP29, and the Rho <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Expression and mutation analyses implicate ARHGAP29 as the etiologic gene for the cleft lip with or without cleft palate locus identified by genome wide association on chromosome 1p22. </a></p><div class="supp"><p class="details"><div>The hypothetical <span class="highlight" style="background-color:">pathway</span> links the IRF6 gene regulatory network (IRF6, TP63, TFAP2A, and TGFB3, all previously implicated in craniofacial development) with the Rho <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through ARHGAP29. This <span class="highlight" style="background-color:">pathway</span> may also involve WNT and TGFB3, which can signal through the Rho <span class="highlight" style="background-color:">pathway</span>, and are required for craniofacial development. We propose that decreased ARHGAP29 would maintain Rho in an active state, which would negatively regulate processes critical for craniofacial development such as cellular migration. GEF, guanine nucleotide exchange factor.</div></p></div><div class="aux"><div class="resc">Elizabeth J Leslie, et al. Birth Defects Res A Clin Mol Teratol. ;94(11):934-942.</div><p class="links"><a class="dblinks" href="/pubmed/23008150" ref="ordinalpos=134&amp;ncbi_uid=5434847&amp;link_uid=23008150">Citation</a><a class="dblinks" href="/pmc/articles/PMC3501616/" ref="ordinalpos=134&amp;ncbi_uid=5434847&amp;link_uid=PMC3501616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>135.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC538777/figure/f7/" ref="ordinalpos=135&amp;ncbi_uid=427772&amp;link_uid=PMC538777" href="/pmc/articles/PMC538777/"><img src="/pmc/articles/instance/538777/bin/zmb0010546640007.gif" src-large="/pmc/articles/instance/538777/bin/zmb0010546640007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC538777/" ref="ordinalpos=135&amp;ncbi_uid=427772&amp;link_uid=PMC538777" image-link="/pmc/articles/PMC538777/figure/f7/" class="imagepopup">FIG. 7. From: Ras-Raf-Arf <span class="highlight" style="background-color:">Signaling</span> Critically Depends on the Dmp1 Transcription Factor. </a></p><div class="supp"><p class="details"><div>The novel Jun-Dmp1 <span class="highlight" style="background-color:">pathway</span>. Dmp1 is a key molecule linking Ras-Raf-MEK-ERK oncogenic <span class="highlight" style="background-color:">signaling</span> and the Arf-Mdm2-p53 tumor suppressor <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> mediated by Dmp1 directly links oncogenic Ras-Raf <span class="highlight" style="background-color:">signaling</span> and p19<sup><i>Arf</i></sup> through activation of c-Jun/JunB proteins. Oncogenic Ras activates both the Jun-Dmp1 <span class="highlight" style="background-color:">pathway</span> and the classical cyclin D1/Cdk4-Rb-E2F <span class="highlight" style="background-color:">pathway</span> to activate <i>Arf</i> gene expression to achieve premature senescence. Since the Ras-Raf <span class="highlight" style="background-color:">pathway</span> activates the <i>Mdm2</i> promoter as well, the levels of p53 are delicately regulated by the activity of the Dmp1-Arf and AP-1/Ets-Mdm2 pathways.</div></p></div><div class="aux"><div class="resc">Ramesh Sreeramaneni, et al. Mol Cell Biol. 2005 January;25(1):220-232.</div><p class="links"><a class="dblinks" href="/pubmed/15601844" ref="ordinalpos=135&amp;ncbi_uid=427772&amp;link_uid=15601844">Citation</a><a class="dblinks" href="/pmc/articles/PMC538777/" ref="ordinalpos=135&amp;ncbi_uid=427772&amp;link_uid=PMC538777">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>136.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3862800/figure/pone-0083580-g006/" ref="ordinalpos=136&amp;ncbi_uid=5608055&amp;link_uid=PMC3862800" href="/pmc/articles/PMC3862800/"><img src="/pmc/articles/instance/3862800/bin/pone.0083580.g006.gif" src-large="/pmc/articles/instance/3862800/bin/pone.0083580.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3862800/" ref="ordinalpos=136&amp;ncbi_uid=5608055&amp;link_uid=PMC3862800" image-link="/pmc/articles/PMC3862800/figure/pone-0083580-g006/" class="imagepopup">Figure 6. Proposed mechanism of Periostin-mediated accelerated epithelial healing through the mTOR <span class="highlight" style="background-color:">pathway</span>..  From: Periostin Responds to Mechanical Stress and Tension by Activating the MTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>During wound healing, activation of the Periostin <span class="highlight" style="background-color:">signaling</span> circuitry is initiated by the extracellular accumulation of Periostin secreted by fibroblasts, and by intracellular periostin originated after mechanical stress. Following, activation of the mTOR <span class="highlight" style="background-color:">pathway</span> occurs. Notably, the mTORC1 is required for Periostin-driven accelerated epithelial migration, while activation of mTORC1 and mTORC2 is required for epithelial proliferation. </div></p></div><div class="aux"><div class="resc">Luciana K. Rosselli-Murai, et al. PLoS One. 2013;8(12):e83580.</div><p class="links"><a class="dblinks" href="/pubmed/24349533" ref="ordinalpos=136&amp;ncbi_uid=5608055&amp;link_uid=24349533">Citation</a><a class="dblinks" href="/pmc/articles/PMC3862800/" ref="ordinalpos=136&amp;ncbi_uid=5608055&amp;link_uid=PMC3862800">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>137.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3572906/figure/F3/" ref="ordinalpos=137&amp;ncbi_uid=5549857&amp;link_uid=PMC3572906" href="/pmc/articles/PMC3572906/"><img src="/pmc/articles/instance/3572906/bin/nihms404901f3.gif" src-large="/pmc/articles/instance/3572906/bin/nihms404901f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3572906/" ref="ordinalpos=137&amp;ncbi_uid=5549857&amp;link_uid=PMC3572906" image-link="/pmc/articles/PMC3572906/figure/F3/" class="imagepopup">Figure 3. Toll-Like Receptors <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: Microbial Cell Components Induced Tolerance to Flagellin-Stimulated Inflammation through Toll-Like Receptor Pathways in Intestinal Epithelial Cells. </a></p><div class="supp"><p class="details"><div>The diagram of the <span class="highlight" style="background-color:">signaling</span> cascades in TLR <span class="highlight" style="background-color:">pathway</span>. Affected signals by treatments were shaded.</div></p></div><div class="aux"><div class="resc">Nan Li, et al. Cytokine. ;60(3):806-811.</div><p class="links"><a class="dblinks" href="/pubmed/22944462" ref="ordinalpos=137&amp;ncbi_uid=5549857&amp;link_uid=22944462">Citation</a><a class="dblinks" href="/pmc/articles/PMC3572906/" ref="ordinalpos=137&amp;ncbi_uid=5549857&amp;link_uid=PMC3572906">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>138.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2929951/figure/pone-0012385-g005/" ref="ordinalpos=138&amp;ncbi_uid=2449447&amp;link_uid=PMC2929951" href="/pmc/articles/PMC2929951/"><img src="/pmc/articles/instance/2929951/bin/pone.0012385.g005.gif" src-large="/pmc/articles/instance/2929951/bin/pone.0012385.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2929951/" ref="ordinalpos=138&amp;ncbi_uid=2449447&amp;link_uid=PMC2929951" image-link="/pmc/articles/PMC2929951/figure/pone-0012385-g005/" class="imagepopup">Figure 5. Example models linking PCFs with molecular <span class="highlight" style="background-color:">pathway</span> activity..  From: Mapping Drug Physico-Chemical Features to <span class="highlight" style="background-color:">Pathway</span> Activity Reveals Molecular Networks Linked to Toxicity Outcome. </a></p><div class="supp"><p class="details"><div>The figure shows the relationships between the observed (x axis) and predicted (y axis) indices of <span class="highlight" style="background-color:">pathway</span> activity for a number of exemplar KEGG pathways. Nephrotoxic samples are represented by red dots whereas non-nephrotoxic samples are represented by black dots. <i>Gap Junction</i> and <i>ErbB <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></i> contain features belonging to ET-State indices, Geometrical descriptors and RDF descriptors. The <img src="/pmc/articles/instance/2929951/bin/pone.0012385.e010.jpg" class="inline-graphic" style="border-style: none;" /> values are 0.55 and 0.57 respectively. Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Adipocyte <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> contain features belonging to GSFRAG, Information indices, Edge adjacency indices and 3D-MoRSE descriptors. The <img src="/pmc/articles/instance/2929951/bin/pone.0012385.e011.jpg" class="inline-graphic" style="border-style: none;" /> values are 0.52 and 0.51 respectively. Note that models containing a feature from E-State indices and RDF descriptors better separate nephrotoxic and non-nephrotoxic samples.</div></p></div><div class="aux"><div class="resc">Philipp Antczak, et al. PLoS One. 2010;5(8):e12385.</div><p class="links"><a class="dblinks" href="/pubmed/20811577" ref="ordinalpos=138&amp;ncbi_uid=2449447&amp;link_uid=20811577">Citation</a><a class="dblinks" href="/pmc/articles/PMC2929951/" ref="ordinalpos=138&amp;ncbi_uid=2449447&amp;link_uid=PMC2929951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>139.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3271860/figure/F7/" ref="ordinalpos=139&amp;ncbi_uid=4658859&amp;link_uid=PMC3271860" href="/pmc/articles/PMC3271860/"><img src="/pmc/articles/instance/3271860/bin/nihms346803f7.gif" src-large="/pmc/articles/instance/3271860/bin/nihms346803f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3271860/" ref="ordinalpos=139&amp;ncbi_uid=4658859&amp;link_uid=PMC3271860" image-link="/pmc/articles/PMC3271860/figure/F7/" class="imagepopup">Figure 7. CDDP-R cells have greater IGF <span class="highlight" style="background-color:">pathway</span> activation than the parental H460 cell line.  From: THE ROLE OF IGF-1 <span class="highlight" style="background-color:">SIGNALING</span> <span class="highlight" style="background-color:">PATHWAY</span> IN CISPLATIN-RESISTANT LUNG CANCER CELLS. </a></p><div class="supp"><p class="details"><div>Binding of IGFBP-3 to IGF-1 prevents receptor binding and downstream <span class="highlight" style="background-color:">pathway</span> activation. With decreased expression of IGFBP-3 in CDDP-R cells, IGF-1 <span class="highlight" style="background-color:">signaling</span> through IGF-1R is maximized, leading to increased metabolism, proliferation, and survival through the Ras-raf-MAPK and PI3K-PDK/AKT-TOR-S6K pathways.</div></p></div><div class="aux"><div class="resc">Yunguang Sun, et al. Int J Radiat Oncol Biol Phys. ;82(3):e563-e572.</div><p class="links"><a class="dblinks" href="/pubmed/22197230" ref="ordinalpos=139&amp;ncbi_uid=4658859&amp;link_uid=22197230">Citation</a><a class="dblinks" href="/pmc/articles/PMC3271860/" ref="ordinalpos=139&amp;ncbi_uid=4658859&amp;link_uid=PMC3271860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>140.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2567351/figure/F2/" ref="ordinalpos=140&amp;ncbi_uid=1595503&amp;link_uid=PMC2567351" href="/pmc/articles/PMC2567351/"><img src="/pmc/articles/instance/2567351/bin/1471-2105-9-412-2.gif" src-large="/pmc/articles/instance/2567351/bin/1471-2105-9-412-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2567351/" ref="ordinalpos=140&amp;ncbi_uid=1595503&amp;link_uid=PMC2567351" image-link="/pmc/articles/PMC2567351/figure/F2/" class="imagepopup">Figure 2.  From: Gene and <span class="highlight" style="background-color:">pathway</span> identification with Lp penalized Bayesian logistic regression. </a></p><div class="supp"><p class="details"><div><b>MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></b>. MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the associated genes. The over-expressed and down-expressed genes on the <span class="highlight" style="background-color:">pathway</span> are shown in red and blue respectively.</div></p></div><div class="aux"><div class="resc">Zhenqiu Liu, et al. BMC Bioinformatics. 2008;9:412-412.</div><p class="links"><a class="dblinks" href="/pubmed/18834526" ref="ordinalpos=140&amp;ncbi_uid=1595503&amp;link_uid=18834526">Citation</a><a class="dblinks" href="/pmc/articles/PMC2567351/" ref="ordinalpos=140&amp;ncbi_uid=1595503&amp;link_uid=PMC2567351">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>141.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4259407/figure/F7/" ref="ordinalpos=141&amp;ncbi_uid=6950286&amp;link_uid=PMC4259407" href="/pmc/articles/PMC4259407/"><img src="/pmc/articles/instance/4259407/bin/oncotarget-05-10084-g007.gif" src-large="/pmc/articles/instance/4259407/bin/oncotarget-05-10084-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4259407/" ref="ordinalpos=141&amp;ncbi_uid=6950286&amp;link_uid=PMC4259407" image-link="/pmc/articles/PMC4259407/figure/F7/" class="imagepopup">Figure 7. Model depicting the role of HAX1 in the non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: HAX1 regulates E3 ubiquitin ligase activity of cIAPs by promoting their dimerization. </a></p><div class="supp"><p class="details"><div>In the steady state, cIAPs lead to poly-ubiquitination and proteasomal degradation of NIK via their RING domain together with TRAF2 and TRAF3. The degradation of NIK results in the suppression of the non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Binding of HAX1 to the cIAPs results in the auto-ubiquitination and proteasomal degradation of cIAPs. Consequently, reduced cIAP levels leads to the stabilization of NIK, which induces activation of the non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jin Sun Choi, et al. Oncotarget. 2014 October;5(20):10084-10099.</div><p class="links"><a class="dblinks" href="/pubmed/25275296" ref="ordinalpos=141&amp;ncbi_uid=6950286&amp;link_uid=25275296">Citation</a><a class="dblinks" href="/pmc/articles/PMC4259407/" ref="ordinalpos=141&amp;ncbi_uid=6950286&amp;link_uid=PMC4259407">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>142.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2886346/figure/fig10/" ref="ordinalpos=142&amp;ncbi_uid=2630818&amp;link_uid=PMC2886346" href="/pmc/articles/PMC2886346/"><img src="/pmc/articles/instance/2886346/bin/JCB_200912082_RGB_Fig10.gif" src-large="/pmc/articles/instance/2886346/bin/JCB_200912082_RGB_Fig10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2886346/" ref="ordinalpos=142&amp;ncbi_uid=2630818&amp;link_uid=PMC2886346" image-link="/pmc/articles/PMC2886346/figure/fig10/" class="imagepopup">Figure 10. From: MAPK <span class="highlight" style="background-color:">signaling</span> to the early secretory <span class="highlight" style="background-color:">pathway</span> revealed by kinase/phosphatase functional screening. </a></p><div class="supp"><p class="details"><div><b>Model of the <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> to ERESs.</b> EGFR, EGF receptor.</div></p></div><div class="aux"><div class="resc">Hesso Farhan, et al. J Cell Biol. 2010 June 14;189(6):997-1011.</div><p class="links"><a class="dblinks" href="/pubmed/20548102" ref="ordinalpos=142&amp;ncbi_uid=2630818&amp;link_uid=20548102">Citation</a><a class="dblinks" href="/pmc/articles/PMC2886346/" ref="ordinalpos=142&amp;ncbi_uid=2630818&amp;link_uid=PMC2886346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>143.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3192015/figure/F1/" ref="ordinalpos=143&amp;ncbi_uid=3487446&amp;link_uid=PMC3192015" href="/pmc/articles/PMC3192015/"><img src="/pmc/articles/instance/3192015/bin/nihms325485f1.gif" src-large="/pmc/articles/instance/3192015/bin/nihms325485f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3192015/" ref="ordinalpos=143&amp;ncbi_uid=3487446&amp;link_uid=PMC3192015" image-link="/pmc/articles/PMC3192015/figure/F1/" class="imagepopup">Figure 1. PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> cascade.  From: Mammalian target of rapamycin, Akt, and phosphatidylinositol 3-kinase <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> through a transmembrane receptor activates the PI3K <span class="highlight" style="background-color:">signaling</span> network to phosphorylate Akt and promote cell proliferation and invasion through mTOR. Multiple feedback loops exist within this <span class="highlight" style="background-color:">signaling</span> cascade, and a number of inhibitors are in development to target this <span class="highlight" style="background-color:">pathway</span> in cancer.</div></p></div><div class="aux"><div class="resc">Sumanta K. Pal, et al. J Thorac Oncol. ;5(12 Suppl 6):S487-S489.</div><p class="links"><a class="dblinks" href="/pubmed/21102251" ref="ordinalpos=143&amp;ncbi_uid=3487446&amp;link_uid=21102251">Citation</a><a class="dblinks" href="/pmc/articles/PMC3192015/" ref="ordinalpos=143&amp;ncbi_uid=3487446&amp;link_uid=PMC3192015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>144.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938097/figure/F6/" ref="ordinalpos=144&amp;ncbi_uid=3059920&amp;link_uid=PMC2938097" href="/pmc/articles/PMC2938097/"><img src="/pmc/articles/instance/2938097/bin/zjw0071036280006.gif" src-large="/pmc/articles/instance/2938097/bin/zjw0071036280006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938097/" ref="ordinalpos=144&amp;ncbi_uid=3059920&amp;link_uid=PMC2938097" image-link="/pmc/articles/PMC2938097/figure/F6/" class="imagepopup">Fig. 6. From: Studies of Phosphoproteomic Changes Induced by Nucleophosmin-Anaplastic Lymphoma Kinase (ALK) Highlight Deregulation of Tumor Necrosis Factor (TNF)/Fas/TNF-related Apoptosis-induced Ligand <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in ALK-positive Anaplastic Large Cell Lymphoma. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of TNF/Fas/TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> By analyzing our proteomics data, antibody microarray data, and our previously published binding protein data (20), a number of <span class="highlight" style="background-color:">signaling</span> pathways were identified, and the TNF/Fas/TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is highlighted here. DEDD: Death effector domain-containing protein; DAXX: Death domain-associated protein 6.</div></p></div><div class="aux"><div class="resc">Fang Wu, et al. Mol Cell Proteomics. 2010 July;9(7):1616-1632.</div><p class="links"><a class="dblinks" href="/pubmed/20393185" ref="ordinalpos=144&amp;ncbi_uid=3059920&amp;link_uid=20393185">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938097/" ref="ordinalpos=144&amp;ncbi_uid=3059920&amp;link_uid=PMC2938097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>145.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3190874/figure/F2/" ref="ordinalpos=145&amp;ncbi_uid=3290766&amp;link_uid=PMC3190874" href="/pmc/articles/PMC3190874/"><img src="/pmc/articles/instance/3190874/bin/zbc0411179700002.gif" src-large="/pmc/articles/instance/3190874/bin/zbc0411179700002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3190874/" ref="ordinalpos=145&amp;ncbi_uid=3290766&amp;link_uid=PMC3190874" image-link="/pmc/articles/PMC3190874/figure/F2/" class="imagepopup">FIGURE 2. From: Immunoglobulin E Receptor <span class="highlight" style="background-color:">Signaling</span> and Asthma. </a></p><div class="supp"><p class="details"><div><b>The FcϵRI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is negatively regulated by a number of cell surface and intracellular proteins that act at various points in the FcϵRI <span class="highlight" style="background-color:">signaling</span> network.</b> Proximal intracellular FcϵRI <span class="highlight" style="background-color:">signaling</span> and adaptor proteins are negatively regulated by Csk, protein-tyrosine phosphatase ϵ (<i>PTP</i>ϵ), SHP-1, and SHP-2. PI3K <span class="highlight" style="background-color:">signaling</span> is negatively regulated by SHIP, PTEN, LAX, and RGS13. Ras <span class="highlight" style="background-color:">signaling</span> is negatively regulated by RasGAP. Cell surface proteins that negatively regulate FcϵRI <span class="highlight" style="background-color:">signaling</span> include TRPM4, which reduces calcium <span class="highlight" style="background-color:">signaling</span>, and TLR4, which, upon formation of a complex with the ES62 product of filarial nematodes, inhibits SphK and PLD <span class="highlight" style="background-color:">signaling</span> by sequestering and degrading SphK- and PLD-activating PKCα. Co-cross-linking of the ITIM-containing cell surface receptor FcγRIIb with FcϵRI triggers the inhibition of FcϵRI <span class="highlight" style="background-color:">signaling</span> at several points, including SHIP-mediated inactivation of PI3K <span class="highlight" style="background-color:">signaling</span> and DOK1/RasGAP-mediated inactivation of Ras <span class="highlight" style="background-color:">signaling</span>. Co-cross-linking of FcϵRI with other ITIM-containing cell surface receptors triggers the inhibition of FcϵRI <span class="highlight" style="background-color:">signaling</span> mediated by SHP-1 and SHP-2. <i>DAG</i>, diacylglycerol; <i>IP3</i>, inositol 1,4,5-trisphosphate.</div></p></div><div class="aux"><div class="resc">Lawren C. Wu. J Biol Chem. 2011 September 23;286(38):32891-32897.</div><p class="links"><a class="dblinks" href="/pubmed/21799019" ref="ordinalpos=145&amp;ncbi_uid=3290766&amp;link_uid=21799019">Citation</a><a class="dblinks" href="/pmc/articles/PMC3190874/" ref="ordinalpos=145&amp;ncbi_uid=3290766&amp;link_uid=PMC3190874">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>146.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3375273/figure/pone-0038048-g006/" ref="ordinalpos=146&amp;ncbi_uid=4069656&amp;link_uid=PMC3375273" href="/pmc/articles/PMC3375273/"><img src="/pmc/articles/instance/3375273/bin/pone.0038048.g006.gif" src-large="/pmc/articles/instance/3375273/bin/pone.0038048.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3375273/" ref="ordinalpos=146&amp;ncbi_uid=4069656&amp;link_uid=PMC3375273" image-link="/pmc/articles/PMC3375273/figure/pone-0038048-g006/" class="imagepopup">Figure 6. A model for TRAIL-induced activation of the TRAF6 associated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in osteolcast differentiation..  From: TRAF-6 Dependent <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Is Essential for TNF-Related Apoptosis-Inducing Ligand (TRAIL) Induces Osteoclast Differentiation. </a></p><div class="supp"><p class="details"><div>Upon receptor engagement, in addition to transduce apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, TRAIL induces activation of TRAF6, at least in part through the E3 ubiquitin ligase activity of the TRAF6 RING finger domain via a distinct <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In osteoclast precursors, TRAIL is not able to induce sufficient amount of signals to activate the downstream apoptosis <span class="highlight" style="background-color:">signaling</span> to induce apoptosis; instead, TRAIL induces <span class="highlight" style="background-color:">signaling</span> to activate TRAF6 after receptor engagement. TRAF6 is the key molecule linking cytoplasmic <span class="highlight" style="background-color:">signaling</span> to the nuclear transcriptional program in osteoclast differentiation and bone resorption. These signals include activation of NF-κB by a process of ubiquitination in which TRAF6 functions as an E3 ligase [36] and recruitment of the TAB1-TAB2-TAK1 complex, which lead to the osteoclast-specific event, that is, autoamplification of NFATc1, the master transcription factor for osteoclast differentiation [29], [30], [31].</div></p></div><div class="aux"><div class="resc">Men-Luh Yen, et al. PLoS One. 2012;7(6):e38048.</div><p class="links"><a class="dblinks" href="/pubmed/22719861" ref="ordinalpos=146&amp;ncbi_uid=4069656&amp;link_uid=22719861">Citation</a><a class="dblinks" href="/pmc/articles/PMC3375273/" ref="ordinalpos=146&amp;ncbi_uid=4069656&amp;link_uid=PMC3375273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>147.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3652804/figure/F1/" ref="ordinalpos=147&amp;ncbi_uid=4891622&amp;link_uid=PMC3652804" href="/pmc/articles/PMC3652804/"><img src="/pmc/articles/instance/3652804/bin/nihms134496f1.gif" src-large="/pmc/articles/instance/3652804/bin/nihms134496f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3652804/" ref="ordinalpos=147&amp;ncbi_uid=4891622&amp;link_uid=PMC3652804" image-link="/pmc/articles/PMC3652804/figure/F1/" class="imagepopup">Figure 1. Graphic of insulin/IGF1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing proteins encoded by scanned genes.  From: Association of common genetic variation in the insulin/IGF1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with human longevity. </a></p><div class="aux"><div class="resc">Ludmila Pawlikowska, et al. Aging Cell. 2009 August;8(4):460-472.</div><p class="links"><a class="dblinks" href="/pubmed/19489743" ref="ordinalpos=147&amp;ncbi_uid=4891622&amp;link_uid=19489743">Citation</a><a class="dblinks" href="/pmc/articles/PMC3652804/" ref="ordinalpos=147&amp;ncbi_uid=4891622&amp;link_uid=PMC3652804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>148.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4120774/figure/F9/" ref="ordinalpos=148&amp;ncbi_uid=6460823&amp;link_uid=PMC4120774" href="/pmc/articles/PMC4120774/"><img src="/pmc/articles/instance/4120774/bin/nihms-314392-f0009.gif" src-large="/pmc/articles/instance/4120774/bin/nihms-314392-f0009.jpg" alt="Figure - 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4120774/" ref="ordinalpos=148&amp;ncbi_uid=6460823&amp;link_uid=PMC4120774" image-link="/pmc/articles/PMC4120774/figure/F9/" class="imagepopup">Figure - 9. Model of the Canonical and non canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: ATTENUATION OF MULTI-TARGETED PROLIFERATION-LINKED <span class="highlight" style="background-color:">SIGNALING</span> BY 3,3?-DIINDOLYLMETHANE (DIM): FROM BENCH TO CLINIC. </a></p><div class="supp"><p class="details"><div>Lef-1/ TCF proteins are the downstream DNA binding effectors of canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. There is evidence that DIM can function downstream enhancing the phosphorylation of β-catenin and inhibit the nuclear translocation of β-catenin leading to inhibition of tumor cell proliferation and induction of apoptosis.</div></p></div><div class="aux"><div class="resc">Sanjeev Banerjee, et al. Mutat Res. ;728(0):47-66.</div><p class="links"><a class="dblinks" href="/pubmed/21703360" ref="ordinalpos=148&amp;ncbi_uid=6460823&amp;link_uid=21703360">Citation</a><a class="dblinks" href="/pmc/articles/PMC4120774/" ref="ordinalpos=148&amp;ncbi_uid=6460823&amp;link_uid=PMC4120774">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>149.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407377/figure/F1/" ref="ordinalpos=149&amp;ncbi_uid=4842848&amp;link_uid=PMC3407377" href="/pmc/articles/PMC3407377/"><img src="/pmc/articles/instance/3407377/bin/nihms388183f1.gif" src-large="/pmc/articles/instance/3407377/bin/nihms388183f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407377/" ref="ordinalpos=149&amp;ncbi_uid=4842848&amp;link_uid=PMC3407377" image-link="/pmc/articles/PMC3407377/figure/F1/" class="imagepopup">Fig. 1. Overview of TGF-β <span class="highlight" style="background-color:">Signaling</span>.  From: TGF-? <span class="highlight" style="background-color:">signaling</span> via TAK1 <span class="highlight" style="background-color:">pathway</span>: Role in kidney fibrosis. </a></p><div class="supp"><p class="details"><div>Initiation of the TGF-β <span class="highlight" style="background-color:">signaling</span> cascade occurs upon ligand binding to TβRII and subsequent TβRI-TβRII hetero-tetrameric complex formation, and TβRI is activated through phosphorylated in the cytoplasmic GS domain. This is followed by activation of intracellular <span class="highlight" style="background-color:">signaling</span> mediators of TGF-β1. The canonical Smad <span class="highlight" style="background-color:">pathway</span> involves activation of Smad2/3 through recruitment and phosphorylation by activated TβRI, and requires kinase activity of TβRI. Smad2/3 is then released from the receptor complex to interact with Smad4 to transmit TGF-β1 signals. TGF-β1 also induces various non-Smad <span class="highlight" style="background-color:">signaling</span> pathways, including the PI3K/AKT, Rho-like GTPases, and the MAPKs.</div></p></div><div class="aux"><div class="resc">Mary E. Choi, et al. Semin Nephrol. ;32(3):244-252.</div><p class="links"><a class="dblinks" href="/pubmed/22835455" ref="ordinalpos=149&amp;ncbi_uid=4842848&amp;link_uid=22835455">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407377/" ref="ordinalpos=149&amp;ncbi_uid=4842848&amp;link_uid=PMC3407377">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>150.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412645/figure/fig6/" ref="ordinalpos=150&amp;ncbi_uid=4173747&amp;link_uid=PMC3412645" href="/pmc/articles/PMC3412645/"><img src="/pmc/articles/instance/3412645/bin/oncsis201212f6.gif" src-large="/pmc/articles/instance/3412645/bin/oncsis201212f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412645/" ref="ordinalpos=150&amp;ncbi_uid=4173747&amp;link_uid=PMC3412645" image-link="/pmc/articles/PMC3412645/figure/fig6/" class="imagepopup">Figure 6.  From: Activation of the AKT/cyclin D1/Cdk4 survival <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in radioresistant cancer stem cells. </a></p><div class="supp"><p class="details"><div>DNA damage <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in parental cells and 82FR-31NR cells. Radiation activates the p53 <span class="highlight" style="background-color:">pathway</span> to induce apoptosis in irradiated parental cells. On the other hand, the AKT/cyclin D1/Cdk4 <span class="highlight" style="background-color:">pathway</span> is activated by irradiation in 82FR-31NR cells. The AKT <span class="highlight" style="background-color:">pathway</span> facilitates DNA repair of radiation-induced DNA damage and promotes cell survival in 82FR-31NR cells after irradiation.</div></p></div><div class="aux"><div class="resc">T Shimura, et al. Oncogenesis. 2012 June;1(6):e12.</div><p class="links"><a class="dblinks" href="/pubmed/23552696" ref="ordinalpos=150&amp;ncbi_uid=4173747&amp;link_uid=23552696">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412645/" ref="ordinalpos=150&amp;ncbi_uid=4173747&amp;link_uid=PMC3412645">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>151.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3494542/figure/F3/" ref="ordinalpos=151&amp;ncbi_uid=4363008&amp;link_uid=PMC3494542" href="/pmc/articles/PMC3494542/"><img src="/pmc/articles/instance/3494542/bin/1479-5876-10-183-3.gif" src-large="/pmc/articles/instance/3494542/bin/1479-5876-10-183-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3494542/" ref="ordinalpos=151&amp;ncbi_uid=4363008&amp;link_uid=PMC3494542" image-link="/pmc/articles/PMC3494542/figure/F3/" class="imagepopup">Figure 3.  From: TGF-? - an excellent servant but a bad master. </a></p><div class="supp"><p class="details"><div><b>TGF-β non-canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> After ligand binding, several different branching <span class="highlight" style="background-color:">signaling</span> pathways can be activated in malignant cells, such as Notch <span class="highlight" style="background-color:">signaling</span>, MAP kinases, AKT/PKB <span class="highlight" style="background-color:">pathway</span>, GTP-binding proteins <span class="highlight" style="background-color:">pathway</span>, PTK <span class="highlight" style="background-color:">pathway</span>, NF-κB and Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Lenka Kubiczkova, et al. J Transl Med. 2012;10:183-183.</div><p class="links"><a class="dblinks" href="/pubmed/22943793" ref="ordinalpos=151&amp;ncbi_uid=4363008&amp;link_uid=22943793">Citation</a><a class="dblinks" href="/pmc/articles/PMC3494542/" ref="ordinalpos=151&amp;ncbi_uid=4363008&amp;link_uid=PMC3494542">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>152.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC125566/figure/cde434f7/" ref="ordinalpos=152&amp;ncbi_uid=187373&amp;link_uid=PMC125566" href="/pmc/articles/PMC125566/"><img src="/pmc/articles/instance/125566/bin/cde434f7.gif" src-large="/pmc/articles/instance/125566/bin/cde434f7.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC125566/" ref="ordinalpos=152&amp;ncbi_uid=187373&amp;link_uid=PMC125566" image-link="/pmc/articles/PMC125566/figure/cde434f7/" class="imagepopup">Figure. From: Negative regulation of the Wnt-?-catenin <span class="highlight" style="background-color:">pathway</span> by the transcriptional repressor HBP1. </a></p><div class="supp"><p class="details"><div><b>Fig. 7.</b> The proposed role of HBP1 in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. As described in the background, the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is conserved in evolution and has an intricate role in cancer. When Wnt proteins bind to Frizzled receptors, a <span class="highlight" style="background-color:">signaling</span> cascade is triggered where the β-catenin protein becomes stabilized and can enter the nucleus to activate transcription of target genes through binding target HMG-box proteins. Lef/TCF proteins directly bind the DNA, and when bound to β-catenin activate transcription of target genes such as cyclin D1 and c-<i>myc</i> (reviewed in Bienz and Clevers, 2000; Polakis, 2000). In normal tissues, the tumor suppressor protein, APC, along with a complex containing axin and GSK-3β, triggers the degradation of β-catenin when Wnt <span class="highlight" style="background-color:">signaling</span> is turned off. All three components are important for preventing accumulation of β-catenin; loss-of-function mutations in APC and axin are associated with colorectal and hepatocellular cancers. In addition, stabilizing mutations in β-catenin have been identified in many human cancers. With the literature of the Wnt <span class="highlight" style="background-color:">pathway</span> as a backdrop, the work in this paper supports a model in which HBP1 is a suppressor of Wnt <span class="highlight" style="background-color:">signaling</span> and is a negative regulator of proliferation in normal tissues. HBP1 can block Wnt <span class="highlight" style="background-color:">signaling</span> from four points in the <span class="highlight" style="background-color:">pathway</span>: Wnt itself, GSK3β inhibition, β-catenin and LEF/TCF. The endogenous cyclin D1 and c-<i>myc</i> target genes are also inhibited by HBP1 expression in a cell with a constitutive Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Our data indicate that the probable mechanism is the inhibition of DNA binding by LEF and TCF through a physical interaction with HBP1. The same regions of HBP1 that are necessary for suppression of Wnt <span class="highlight" style="background-color:">signaling</span> are also required for growth inhibition in cells with a constitutive Wnt <span class="highlight" style="background-color:">pathway</span>. Thus, HBP1 may have a role in tumor suppression by inhibiting the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ellen M. Sampson, et al. EMBO J. 2001 August 15;20(16):4500-4511.</div><p class="links"><a class="dblinks" href="/pubmed/11500377" ref="ordinalpos=152&amp;ncbi_uid=187373&amp;link_uid=11500377">Citation</a><a class="dblinks" href="/pmc/articles/PMC125566/" ref="ordinalpos=152&amp;ncbi_uid=187373&amp;link_uid=PMC125566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>153.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2824373/figure/fig06/" ref="ordinalpos=153&amp;ncbi_uid=2259739&amp;link_uid=PMC2824373" href="/pmc/articles/PMC2824373/"><img src="/pmc/articles/instance/2824373/bin/pnas.0907907107fig06.gif" src-large="/pmc/articles/instance/2824373/bin/pnas.0907907107fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2824373/" ref="ordinalpos=153&amp;ncbi_uid=2259739&amp;link_uid=PMC2824373" image-link="/pmc/articles/PMC2824373/figure/fig06/" class="imagepopup">Fig. 6. From: EGF potentiated oncogenesis requires a tissue transglutaminase-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to Src activation. </a></p><div class="supp"><p class="details"><div>Diagram depicting how tTG participates in an EGFR/Src <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that leads to enhanced cancer-cell growth.</div></p></div><div class="aux"><div class="resc">Bo Li, et al. Proc Natl Acad Sci U S A. 2010 January 26;107(4):1408-1413.</div><p class="links"><a class="dblinks" href="/pubmed/20080707" ref="ordinalpos=153&amp;ncbi_uid=2259739&amp;link_uid=20080707">Citation</a><a class="dblinks" href="/pmc/articles/PMC2824373/" ref="ordinalpos=153&amp;ncbi_uid=2259739&amp;link_uid=PMC2824373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>154.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4029773/figure/F1/" ref="ordinalpos=154&amp;ncbi_uid=6153422&amp;link_uid=PMC4029773" href="/pmc/articles/PMC4029773/"><img src="/pmc/articles/instance/4029773/bin/nihms577662f1.gif" src-large="/pmc/articles/instance/4029773/bin/nihms577662f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4029773/" ref="ordinalpos=154&amp;ncbi_uid=6153422&amp;link_uid=PMC4029773" image-link="/pmc/articles/PMC4029773/figure/F1/" class="imagepopup">Figure 1. Roles of the JAK-STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in adipose tissue.  From: The role of JAK-STAT <span class="highlight" style="background-color:">signaling</span> in adipose tissue function. </a></p><div class="supp"><p class="details"><div>The JAK-STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by GH, PRL, and GM-CSF, and it regulates the development and/or function of preadipocytes, adipocytes, and macrophages in adipose tissue. Substantial evidence supports that STAT5 proteins are activated in preadipocytes and that they promote adipogenesis. In mature adipocytes, multiple STAT5 target genes have been identified, demonstrating a role for STAT5 in modulating key physiological properties of adipocytes. Studies also indicate that JAK-STAT5 <span class="highlight" style="background-color:">signaling</span> is important in the recruitment and development of adipose tissue macrophages.</div></p></div><div class="aux"><div class="resc">Allison J. Richard, et al. Biochim Biophys Acta. ;1842(3):431-439.</div><p class="links"><a class="dblinks" href="/pubmed/23735217" ref="ordinalpos=154&amp;ncbi_uid=6153422&amp;link_uid=23735217">Citation</a><a class="dblinks" href="/pmc/articles/PMC4029773/" ref="ordinalpos=154&amp;ncbi_uid=6153422&amp;link_uid=PMC4029773">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>155.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1986755/figure/F7/" ref="ordinalpos=155&amp;ncbi_uid=1037069&amp;link_uid=PMC1986755" href="/pmc/articles/PMC1986755/"><img src="/pmc/articles/instance/1986755/bin/nihms25854f7.gif" src-large="/pmc/articles/instance/1986755/bin/nihms25854f7.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1986755/" ref="ordinalpos=155&amp;ncbi_uid=1037069&amp;link_uid=PMC1986755" image-link="/pmc/articles/PMC1986755/figure/F7/" class="imagepopup">Fig 7. Model of the Rho <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating apical constriction during ventral furrow, posterior midgut, and salivary gland morphogenesis.  From: 18 wheeler Regulates Apical Constriction of Salivary Gland Cells via the Rho-GTPase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>While 18W has not been linked to the Rho <span class="highlight" style="background-color:">pathway</span> during the development of the ventral furrow and posterior midgut, it does appear to negatively regulate RhoGAPs during salivary gland invagination.</div></p></div><div class="aux"><div class="resc">Tereza Kolesnikov, et al. Dev Biol. ;307(1):53-61.</div><p class="links"><a class="dblinks" href="/pubmed/17512518" ref="ordinalpos=155&amp;ncbi_uid=1037069&amp;link_uid=17512518">Citation</a><a class="dblinks" href="/pmc/articles/PMC1986755/" ref="ordinalpos=155&amp;ncbi_uid=1037069&amp;link_uid=PMC1986755">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>156.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4057083/figure/pone-0097902-g008/" ref="ordinalpos=156&amp;ncbi_uid=6262931&amp;link_uid=PMC4057083" href="/pmc/articles/PMC4057083/"><img src="/pmc/articles/instance/4057083/bin/pone.0097902.g008.gif" src-large="/pmc/articles/instance/4057083/bin/pone.0097902.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4057083/" ref="ordinalpos=156&amp;ncbi_uid=6262931&amp;link_uid=PMC4057083" image-link="/pmc/articles/PMC4057083/figure/pone-0097902-g008/" class="imagepopup">Figure 8. Schematic diagram of wound healing-related cellular pathways and their cross-talks among inflammation, neurogenesis, and angiogenesis..  From: On the Crucial Cerebellar Wound Healing-Related Pathways and Their Cross-Talks after Traumatic Brain Injury in Danio rerio. </a></p><div class="supp"><p class="details"><div>(A) Pathways shown in blocks were enriched in the sub-networks (Fig. 7). Their cross-talks were observed in the cerebellar wound healing PPI network of zebrafish (solid arrows indicate cross-talks involving proteins in groups A, N, and P; dashed arrows indicate cross-talks involving proteins not in these groups). To establish cell communication and defense mechanisms (inflammation and immune response) after brain injury, cell communication was related to neurotransmitter-related pathways (through AP2 and AN2), the axon guidance <span class="highlight" style="background-color:">pathway</span> (through NP4 indirectly), and itself (through AN1). Defense mechanisms were related to the cell cycle <span class="highlight" style="background-color:">pathway</span> (through AP1), neurotransmitter-related pathways (through AN2), the cell communication <span class="highlight" style="background-color:">pathway</span> (through NP2), and neurodegenerative diseases (through NP2). Cell cycle pathways also interacted with gene regulation pathways through NP3 indirectly. (B) The cross-talks are found among pathways of inflammation, neurogenesis, and angiogenesis in cerebellar wound healing process. Inflammation is mediated by chemokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. New neural and vascular cells generation is mediated by PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and axon guidance <span class="highlight" style="background-color:">pathway</span>. PI3K in chemokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> also regulates the PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and axon guidance <span class="highlight" style="background-color:">pathway</span>. PAK2 in axon guidance <span class="highlight" style="background-color:">pathway</span> also maintains the stability of vascular.</div></p></div><div class="aux"><div class="resc">Chia-Chou Wu, et al. PLoS One. 2014;9(6):e97902.</div><p class="links"><a class="dblinks" href="/pubmed/24926785" ref="ordinalpos=156&amp;ncbi_uid=6262931&amp;link_uid=24926785">Citation</a><a class="dblinks" href="/pmc/articles/PMC4057083/" ref="ordinalpos=156&amp;ncbi_uid=6262931&amp;link_uid=PMC4057083">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>157.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4095003/figure/F3/" ref="ordinalpos=157&amp;ncbi_uid=6408560&amp;link_uid=PMC4095003" href="/pmc/articles/PMC4095003/"><img src="/pmc/articles/instance/4095003/bin/1471-2105-15-S5-S3-3.gif" src-large="/pmc/articles/instance/4095003/bin/1471-2105-15-S5-S3-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4095003/" ref="ordinalpos=157&amp;ncbi_uid=6408560&amp;link_uid=PMC4095003" image-link="/pmc/articles/PMC4095003/figure/F3/" class="imagepopup">Figure 3.  From: timeClip: <span class="highlight" style="background-color:">pathway</span> analysis for time course data without replicates. </a></p><div class="supp"><p class="details"><div><b>Activation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Panel A. Junction tree of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (using graphite R package and database KEGG). The top ranked time-dependent paths identified in <tt>timeClip</tt> step 2 are highlighted using the wheel of time visualization. Panel B. KEGG representation of mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes are colored according to the paths in panel A. Panel C. Enlargement of the wheels of time representative of the main block of mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: from <i>t</i><sub>0 </sub>to <i>t</i><sub>27 </sub>(clock-wise) every slice of the pie is colored according to the value of the clique first PC (green means no activation; red means activation).</div></p></div><div class="aux"><div class="resc">Paolo Martini, et al. BMC Bioinformatics. 2014;15(Suppl 5):S3-S3.</div><p class="links"><a class="dblinks" href="/pubmed/25077979" ref="ordinalpos=157&amp;ncbi_uid=6408560&amp;link_uid=25077979">Citation</a><a class="dblinks" href="/pmc/articles/PMC4095003/" ref="ordinalpos=157&amp;ncbi_uid=6408560&amp;link_uid=PMC4095003">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>158.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3741135/figure/pone-0071576-g008/" ref="ordinalpos=158&amp;ncbi_uid=5199248&amp;link_uid=PMC3741135" href="/pmc/articles/PMC3741135/"><img src="/pmc/articles/instance/3741135/bin/pone.0071576.g008.gif" src-large="/pmc/articles/instance/3741135/bin/pone.0071576.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3741135/" ref="ordinalpos=158&amp;ncbi_uid=5199248&amp;link_uid=PMC3741135" image-link="/pmc/articles/PMC3741135/figure/pone-0071576-g008/" class="imagepopup">Figure 8. Sirt1 interacts with upstream and downstream genes of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Sirt1 Inhibits Akt2-Mediated Porcine Adipogenesis Potentially by Direct Protein-Protein Interaction. </a></p><div class="supp"><p class="details"><div>At day 8 after induction, porcine fat cells were collected and protein was isolated for immunoprecipitated analysis. A. Sirt1 interacts with PI3K which is an upstream gene in PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Endogenous PI3K in fat cell lysate was immunoprecipitated with anti–PI3K antibody, and coprecipitation of Sirt1 was detected by Western blot assay. B. Sirt1 interacts with GSK3β which is a downstream gene in PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Endogenous GSK3β in fat cell lysate was immunoprecipitated with anti–GSK3β antibody, and coprecipitation of Sirt1 was detected by Western blot assay.</div></p></div><div class="aux"><div class="resc">Weijun Pang, et al. PLoS One. 2013;8(8):e71576.</div><p class="links"><a class="dblinks" href="/pubmed/23951196" ref="ordinalpos=158&amp;ncbi_uid=5199248&amp;link_uid=23951196">Citation</a><a class="dblinks" href="/pmc/articles/PMC3741135/" ref="ordinalpos=158&amp;ncbi_uid=5199248&amp;link_uid=PMC3741135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>159.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766659/figure/F4/" ref="ordinalpos=159&amp;ncbi_uid=2125739&amp;link_uid=PMC2766659" href="/pmc/articles/PMC2766659/"><img src="/pmc/articles/instance/2766659/bin/nihms-101752-f0004.gif" src-large="/pmc/articles/instance/2766659/bin/nihms-101752-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766659/" ref="ordinalpos=159&amp;ncbi_uid=2125739&amp;link_uid=PMC2766659" image-link="/pmc/articles/PMC2766659/figure/F4/" class="imagepopup">Figure 4. Crucial <span class="highlight" style="background-color:">signaling</span> pathways activated by various ligands and the regulatory networks regulating cell fate decisions of SSCs and differentiating spermatogonia.  From: <span class="highlight" style="background-color:">Signaling</span> molecules and pathways regulating the fate of spermatogonial stem cells. </a></p><div class="supp"><p class="details"><div>GDNF and SCF activate both the Ras/Erk and PI3K/Akt pathways, whereas SCF cannot signal via JAK/STAT <span class="highlight" style="background-color:">pathway</span>. GDNF can activate Bcl6b, Lhx1, and ERM, but not PLZF or TAF4b. BMP4 <span class="highlight" style="background-color:">signaling</span> via Smad4/5 <span class="highlight" style="background-color:">pathway</span> and Upd (Unpaired) <span class="highlight" style="background-color:">signaling</span> through JAK/STAT <span class="highlight" style="background-color:">pathway</span> are also shown. Arrows indicated “activation”, whereas the blocked arrows denote “inactivation”.</div></p></div><div class="aux"><div class="resc">Zuping He, et al. Microsc Res Tech. ;72(8):586-595.</div><p class="links"><a class="dblinks" href="/pubmed/19263492" ref="ordinalpos=159&amp;ncbi_uid=2125739&amp;link_uid=19263492">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766659/" ref="ordinalpos=159&amp;ncbi_uid=2125739&amp;link_uid=PMC2766659">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>160.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2772972/figure/F7/" ref="ordinalpos=160&amp;ncbi_uid=2069610&amp;link_uid=PMC2772972" href="/pmc/articles/PMC2772972/"><img src="/pmc/articles/instance/2772972/bin/emm-41-695-g007.gif" src-large="/pmc/articles/instance/2772972/bin/emm-41-695-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2772972/" ref="ordinalpos=160&amp;ncbi_uid=2069610&amp;link_uid=PMC2772972" image-link="/pmc/articles/PMC2772972/figure/F7/" class="imagepopup">Figure 7.  From: Negative feedback regulation of Wnt <span class="highlight" style="background-color:">signaling</span> by G??-mediated reduction of Dishevelled. </a></p><div class="supp"><p class="details"><div>Model for a negative feedback mechanism of Wnt signal transduction by Gβγ <span class="highlight" style="background-color:">signaling</span>. Left panel: In the presence of Wnt, Dishevelled (Dvl) and Axin (for canonical Wnt <span class="highlight" style="background-color:">signaling</span>) are translocated to membrane-bound Frizzled and LRP, respectively, and initiate downstream <span class="highlight" style="background-color:">signaling</span>. Released Gα subunits from Gβγ may then synergistically enhance canonical Wnt <span class="highlight" style="background-color:">signaling</span>. During this time if LRP-bound Axin binds the Gβγ subunit and blocks the downstream <span class="highlight" style="background-color:">signaling</span> of Gβγ then Dishevelled will not be degraded and <span class="highlight" style="background-color:">signaling</span> will continue. Middle panel: Upon further Wnt <span class="highlight" style="background-color:">signaling</span> LRP-bound Axin is degraded which leads to release of Gβγ subunits from Axin. Right panel: The released Gβγ subunits interact with Dishevelled, which lead to its destruction via a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> including PLC, Ca<sup>+2</sup> and PKC. The activation of the Gβγ <span class="highlight" style="background-color:">signaling</span> cascade leads to degradation of Dishevelled in lysosomes and thereby turns off Wnt <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Hwajin Jung, et al. Exp Mol Med. 2009 October 31;41(10):695-706.</div><p class="links"><a class="dblinks" href="/pubmed/19561403" ref="ordinalpos=160&amp;ncbi_uid=2069610&amp;link_uid=19561403">Citation</a><a class="dblinks" href="/pmc/articles/PMC2772972/" ref="ordinalpos=160&amp;ncbi_uid=2069610&amp;link_uid=PMC2772972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>161.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3829470/figure/F4/" ref="ordinalpos=161&amp;ncbi_uid=5492638&amp;link_uid=PMC3829470" href="/pmc/articles/PMC3829470/"><img src="/pmc/articles/instance/3829470/bin/nihms-525698-f0004.gif" src-large="/pmc/articles/instance/3829470/bin/nihms-525698-f0004.jpg" alt="Fig. (4)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3829470/" ref="ordinalpos=161&amp;ncbi_uid=5492638&amp;link_uid=PMC3829470" image-link="/pmc/articles/PMC3829470/figure/F4/" class="imagepopup">Fig. (4).  From: <span class="highlight" style="background-color:">Signaling</span> Through Rho GTPase <span class="highlight" style="background-color:">Pathway</span> as Viable Drug Target. </a></p><div class="supp"><p class="details"><div>Selection of statin drugs capable of modulating Rho <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Qun Lu, et al. Curr Med Chem. 2009;16(11):1355-1365.</div><p class="links"><a class="dblinks" href="/pubmed/19355891" ref="ordinalpos=161&amp;ncbi_uid=5492638&amp;link_uid=19355891">Citation</a><a class="dblinks" href="/pmc/articles/PMC3829470/" ref="ordinalpos=161&amp;ncbi_uid=5492638&amp;link_uid=PMC3829470">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>162.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3169907/figure/F7/" ref="ordinalpos=162&amp;ncbi_uid=3212193&amp;link_uid=PMC3169907" href="/pmc/articles/PMC3169907/"><img src="/pmc/articles/instance/3169907/bin/nihms77081f7.gif" src-large="/pmc/articles/instance/3169907/bin/nihms77081f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3169907/" ref="ordinalpos=162&amp;ncbi_uid=3212193&amp;link_uid=PMC3169907" image-link="/pmc/articles/PMC3169907/figure/F7/" class="imagepopup">Figure 7.  From: AKAP-Lbc mobilizes a cardiac hypertrophy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Model showing the role of AKAP-Lbc in the regulation of cardiac hypertrophy, through the PKD-HDAC5-MEF2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Graeme K. Carnegie, et al. Mol Cell. ;32(2):169-179.</div><p class="links"><a class="dblinks" href="/pubmed/18951085" ref="ordinalpos=162&amp;ncbi_uid=3212193&amp;link_uid=18951085">Citation</a><a class="dblinks" href="/pmc/articles/PMC3169907/" ref="ordinalpos=162&amp;ncbi_uid=3212193&amp;link_uid=PMC3169907">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>163.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3175751/figure/F2/" ref="ordinalpos=163&amp;ncbi_uid=3227758&amp;link_uid=PMC3175751" href="/pmc/articles/PMC3175751/"><img src="/pmc/articles/instance/3175751/bin/nihms323572f2.gif" src-large="/pmc/articles/instance/3175751/bin/nihms323572f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3175751/" ref="ordinalpos=163&amp;ncbi_uid=3227758&amp;link_uid=PMC3175751" image-link="/pmc/articles/PMC3175751/figure/F2/" class="imagepopup">Fig. 2.  From: Emerging Roles of Nodal and Cripto-1: From Embryogenesis to Breast Cancer Progression. </a></p><div class="supp"><p class="details"><div>Schematic Overview of Nodal and Cripto-1 <span class="highlight" style="background-color:">Signaling</span> Pathways. <b>A</b>) Canonical Nodal <span class="highlight" style="background-color:">signaling</span> using Cripto-1 as a co-receptor. Nodal can also activate this <span class="highlight" style="background-color:">pathway</span>, albeit less efficiently, independently of Cripto-1. <b>B</b>) Cripto-1 Nodal-independent <span class="highlight" style="background-color:">signaling</span> via binding with Glypican-1 and activating src/Ras/raf/PI3K downstream <span class="highlight" style="background-color:">signaling</span>. <b>C</b>) Possible <span class="highlight" style="background-color:">signaling</span> of Cripto-1 through binding to beta-integrins and <span class="highlight" style="background-color:">signaling</span> via FAK-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Luigi Strizzi, et al. Breast Dis. ;29:91-103.</div><p class="links"><a class="dblinks" href="/pubmed/19029628" ref="ordinalpos=163&amp;ncbi_uid=3227758&amp;link_uid=19029628">Citation</a><a class="dblinks" href="/pmc/articles/PMC3175751/" ref="ordinalpos=163&amp;ncbi_uid=3227758&amp;link_uid=PMC3175751">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>164.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3998676/figure/F1/" ref="ordinalpos=164&amp;ncbi_uid=6053331&amp;link_uid=PMC3998676" href="/pmc/articles/PMC3998676/"><img src="/pmc/articles/instance/3998676/bin/nihms395811f1.gif" src-large="/pmc/articles/instance/3998676/bin/nihms395811f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3998676/" ref="ordinalpos=164&amp;ncbi_uid=6053331&amp;link_uid=PMC3998676" image-link="/pmc/articles/PMC3998676/figure/F1/" class="imagepopup">Figure 1. Schematic Diagram of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: ADVANCING AGE IS ASSOCIATED WITH GENE EXPRESSION CHANGES RESEMBLING mTOR INHIBITION: EVIDENCE FROM TWO HUMAN POPULATIONS. </a></p><div class="supp"><p class="details"><div>The figure illustrates the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mammals. Adapted from (Dazert and Hall, 2011).</div></p></div><div class="aux"><div class="resc">Lorna W Harries, et al. Mech Ageing Dev. 2012 August;133(8):556-562.</div><p class="links"><a class="dblinks" href="/pubmed/22813852" ref="ordinalpos=164&amp;ncbi_uid=6053331&amp;link_uid=22813852">Citation</a><a class="dblinks" href="/pmc/articles/PMC3998676/" ref="ordinalpos=164&amp;ncbi_uid=6053331&amp;link_uid=PMC3998676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>165.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3529366/figure/F6/" ref="ordinalpos=165&amp;ncbi_uid=5658377&amp;link_uid=PMC3529366" href="/pmc/articles/PMC3529366/"><img src="/pmc/articles/instance/3529366/bin/zee0011368290006.gif" src-large="/pmc/articles/instance/3529366/bin/zee0011368290006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3529366/" ref="ordinalpos=165&amp;ncbi_uid=5658377&amp;link_uid=PMC3529366" image-link="/pmc/articles/PMC3529366/figure/F6/" class="imagepopup">Fig. 6. From: WNT4 Acts Downstream of BMP2 and Functions via ?-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> to Regulate Human Endometrial Stromal Cell Differentiation. </a></p><div class="supp"><p class="details"><div>BMP2-WNT4-FOXO1 <span class="highlight" style="background-color:">pathway</span> in HESC differentiation. BMP2 is induced in response to steroid hormones and cAMP in HESCs. BMP2 activates Smad1/5/8-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to promote WNT4 expression. WNT4 then functions via β-catenin-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to induce FOXO1 and its downstream genes to regulate differentiation of HESCs. FOXO1 also may be an essential maintenance factor for WNT4 expression. In addition, BMP2 controls expression of several WNT inhibitors including Dickkopf proteins and, secreted frizzled-related proteins to fine tune WNT <span class="highlight" style="background-color:">signaling</span> in HESCs.</div></p></div><div class="aux"><div class="resc">Quanxi Li, et al. Endocrinology. 2013 January;154(1):446-457.</div><p class="links"><a class="dblinks" href="/pubmed/23142810" ref="ordinalpos=165&amp;ncbi_uid=5658377&amp;link_uid=23142810">Citation</a><a class="dblinks" href="/pmc/articles/PMC3529366/" ref="ordinalpos=165&amp;ncbi_uid=5658377&amp;link_uid=PMC3529366">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>166.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g004/" ref="ordinalpos=166&amp;ncbi_uid=6232467&amp;link_uid=PMC4041745" href="/pmc/articles/PMC4041745/"><img src="/pmc/articles/instance/4041745/bin/pone.0098229.g004.gif" src-large="/pmc/articles/instance/4041745/bin/pone.0098229.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041745/" ref="ordinalpos=166&amp;ncbi_uid=6232467&amp;link_uid=PMC4041745" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g004/" class="imagepopup">Figure 4. GeneGo <span class="highlight" style="background-color:">pathway</span> “Airway smooth muscle contraction in asthma”..  From: Systematic <span class="highlight" style="background-color:">Pathway</span> Enrichment Analysis of a Genome-Wide Association Study on Breast Cancer Survival Reveals an Influence of Genes Involved in Cell Adhesion and Calcium <span class="highlight" style="background-color:">Signaling</span> on the Patients' Clinical Outcome. </a></p><div class="supp"><p class="details"><div>Barometers: 1 = 0.01 gene list; 2 = general metastasis signature. red = Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, blue = Smooth muscle contraction/relaxation.</div></p></div><div class="aux"><div class="resc">Andrea Woltmann, et al. PLoS One. 2014;9(6):e98229.</div><p class="links"><a class="dblinks" href="/pubmed/24886783" ref="ordinalpos=166&amp;ncbi_uid=6232467&amp;link_uid=24886783">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041745/" ref="ordinalpos=166&amp;ncbi_uid=6232467&amp;link_uid=PMC4041745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>167.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3413921/figure/F1/" ref="ordinalpos=167&amp;ncbi_uid=4669914&amp;link_uid=PMC3413921" href="/pmc/articles/PMC3413921/"><img src="/pmc/articles/instance/3413921/bin/nihms372812f1.gif" src-large="/pmc/articles/instance/3413921/bin/nihms372812f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3413921/" ref="ordinalpos=167&amp;ncbi_uid=4669914&amp;link_uid=PMC3413921" image-link="/pmc/articles/PMC3413921/figure/F1/" class="imagepopup">Figure 1. Hippo <span class="highlight" style="background-color:">signaling</span> and some of its connections with other pathways.  From: Integration of intercellular <span class="highlight" style="background-color:">signaling</span> through the Hippo <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The schematic depicts regulatory some of connections within the Hippo <span class="highlight" style="background-color:">pathway</span>, and between Hippo <span class="highlight" style="background-color:">signaling</span> and other pathways discussed in the text. The <i>Drosophila</i> names of <span class="highlight" style="background-color:">pathway</span> components are used, and the schematic is over-simplified in that many Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components are not shown. Interactions with other pathways depicted include 1) Within the nucleus, Yki/Yap/Taz promotes the expression of ligands for other <span class="highlight" style="background-color:">signaling</span> pathways. 2) Yki/Yap/Taz interacts with Smad proteins (Mad) and β-catenin, which are transcription factors of TGF-β and Wnt pathways. When Yki/Yap/Taz is cytoplasmic this interaction inhibits transcriptional activation by these pathways, but when its nuclear they cooperate to promote transcription of downstream genes. 3) Fj and Ds expression are regulated by the Notch, Dpp, and Wg signlaing pathways in the Drosophila wing, at least some of this regulation is mediated through Vg (not shown). 4) Wounding, apoptosis, or infection (“stress”) can activate the Jnk <span class="highlight" style="background-color:">pathway</span>, which can lead to Yki activation.</div></p></div><div class="aux"><div class="resc">Kenneth D. Irvine. Semin Cell Dev Biol. ;23(7):812-817.</div><p class="links"><a class="dblinks" href="/pubmed/22554983" ref="ordinalpos=167&amp;ncbi_uid=4669914&amp;link_uid=22554983">Citation</a><a class="dblinks" href="/pmc/articles/PMC3413921/" ref="ordinalpos=167&amp;ncbi_uid=4669914&amp;link_uid=PMC3413921">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>168.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3853583/figure/F4/" ref="ordinalpos=168&amp;ncbi_uid=5577893&amp;link_uid=PMC3853583" href="/pmc/articles/PMC3853583/"><img src="/pmc/articles/instance/3853583/bin/1746-1596-8-158-4.gif" src-large="/pmc/articles/instance/3853583/bin/1746-1596-8-158-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3853583/" ref="ordinalpos=168&amp;ncbi_uid=5577893&amp;link_uid=PMC3853583" image-link="/pmc/articles/PMC3853583/figure/F4/" class="imagepopup">Figure 4.  From: Identification of aberrant microRNA expression pattern in pediatric gliomas by microarray. </a></p><div class="supp"><p class="details"><div><b>KEGG <span class="highlight" style="background-color:">pathway</span> analysis for target genes of dysregulated microRNAs. (A)</b> Target genes of up-regulated microRNAs were involved in the ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, axon guidance, glioma, long-term potentiation, the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> et al. <b>(B)</b> Target genes of down-regulated microRNAs were related to the neurotrophin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, amino acids metabolism et al.</div></p></div><div class="aux"><div class="resc">Fatao Liu, et al. Diagn Pathol. 2013;8:158-158.</div><p class="links"><a class="dblinks" href="/pubmed/24053158" ref="ordinalpos=168&amp;ncbi_uid=5577893&amp;link_uid=24053158">Citation</a><a class="dblinks" href="/pmc/articles/PMC3853583/" ref="ordinalpos=168&amp;ncbi_uid=5577893&amp;link_uid=PMC3853583">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>169.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163686/figure/pone-0023989-g003/" ref="ordinalpos=169&amp;ncbi_uid=3208671&amp;link_uid=PMC3163686" href="/pmc/articles/PMC3163686/"><img src="/pmc/articles/instance/3163686/bin/pone.0023989.g003.gif" src-large="/pmc/articles/instance/3163686/bin/pone.0023989.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163686/" ref="ordinalpos=169&amp;ncbi_uid=3208671&amp;link_uid=PMC3163686" image-link="/pmc/articles/PMC3163686/figure/pone-0023989-g003/" class="imagepopup">Figure 3. Models of ST2L inhibiting the TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: In Silico Approach to Inhibition of <span class="highlight" style="background-color:">Signaling</span> Pathways of Toll-Like Receptors 2 and 4 by ST2L. </a></p><div class="supp"><p class="details"><div>Pictorial representation of ST2L inhibiting MyD88-dependent TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ST2L heterodimerizes with Mal and MyD88 by occupying their receptor-adapter and adapter-adapter interacting sites, thereby preventing the engagement of <span class="highlight" style="background-color:">signaling</span> adapters, Mal and MyD88, into the post-receptor <span class="highlight" style="background-color:">signaling</span> complex (TLR4-Mal-MyD88 hexamer) and thus exhibiting its inhibitory effect. Full-length structure of TLR4 is colored in orchid. The TIR domains of ST2L, Mal and MyD88 are colored in hot pink, coral and gold, respectively. The available TLR4 ECD structure coordinates along with its ligand was taken from the PDB (3FXI). The red color dotted line along with the bar represents inhibition symbol.</div></p></div><div class="aux"><div class="resc">Shaherin Basith, et al. PLoS One. 2011;6(8):e23989.</div><p class="links"><a class="dblinks" href="/pubmed/21897866" ref="ordinalpos=169&amp;ncbi_uid=3208671&amp;link_uid=21897866">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163686/" ref="ordinalpos=169&amp;ncbi_uid=3208671&amp;link_uid=PMC3163686">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>170.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1197292/figure/pbio-0030358-g001/" ref="ordinalpos=170&amp;ncbi_uid=669833&amp;link_uid=PMC1197292" href="/pmc/articles/PMC1197292/"><img src="/pmc/articles/instance/1250307/bin/pbio.0030358.g001.gif" src-large="/pmc/articles/instance/1250307/bin/pbio.0030358.g001.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1197292/" ref="ordinalpos=170&amp;ncbi_uid=669833&amp;link_uid=PMC1197292" image-link="/pmc/articles/PMC1197292/figure/pbio-0030358-g001/" class="imagepopup">Drosophila adult muscle founder cells are selected by the Heartless <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Overexpression of components of the Heartless <span class="highlight" style="background-color:">pathway</span> increases the number of founder cells (right) compared to normal expression (left).  From: A <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> at the Heart of Muscle Development. </a></p><div class="aux"><div class="resc">PLoS Biol. 2005 October;3(10):e358.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=170&amp;ncbi_uid=669833&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC1197292/" ref="ordinalpos=170&amp;ncbi_uid=669833&amp;link_uid=PMC1197292">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>171.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2749257/figure/F1/" ref="ordinalpos=171&amp;ncbi_uid=2428354&amp;link_uid=PMC2749257" href="/pmc/articles/PMC2749257/"><img src="/pmc/articles/instance/2749257/bin/nihms-139826-f0001.gif" src-large="/pmc/articles/instance/2749257/bin/nihms-139826-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2749257/" ref="ordinalpos=171&amp;ncbi_uid=2428354&amp;link_uid=PMC2749257" image-link="/pmc/articles/PMC2749257/figure/F1/" class="imagepopup">Figure 1. Crosstalk between FGF and BMP <span class="highlight" style="background-color:">Signaling</span> Pathways during Proepicardial Differentiation.  From: Look Who's Talking: FGFs and BMPs in the Proepicardium. </a></p><div class="supp"><p class="details"><div>Proepicardial cells, likely a heterogeneous population (as indicated by shading), differentiate down the cardiomyocyte lineage in response to BMP <span class="highlight" style="background-color:">signaling</span> or down a non-cardiomyocyte cardiac lineage in response to FGF <span class="highlight" style="background-color:">signaling</span>. Further, the FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is dominant over the BMP <span class="highlight" style="background-color:">pathway</span> as FGF <span class="highlight" style="background-color:">signaling</span> through ERK can block BMP-induced cardiomyocyte differentiation.</div></p></div><div class="aux"><div class="resc">Eric C. Svensson. Circ Res. ;105(5):406-407.</div><p class="links"><a class="dblinks" href="/pubmed/19713545" ref="ordinalpos=171&amp;ncbi_uid=2428354&amp;link_uid=19713545">Citation</a><a class="dblinks" href="/pmc/articles/PMC2749257/" ref="ordinalpos=171&amp;ncbi_uid=2428354&amp;link_uid=PMC2749257">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>172.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3538588/figure/pone-0053522-g008/" ref="ordinalpos=172&amp;ncbi_uid=4521122&amp;link_uid=PMC3538588" href="/pmc/articles/PMC3538588/"><img src="/pmc/articles/instance/3538588/bin/pone.0053522.g008.gif" src-large="/pmc/articles/instance/3538588/bin/pone.0053522.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3538588/" ref="ordinalpos=172&amp;ncbi_uid=4521122&amp;link_uid=PMC3538588" image-link="/pmc/articles/PMC3538588/figure/pone-0053522-g008/" class="imagepopup">Figure 8. Genome to phenome <span class="highlight" style="background-color:">pathway</span> of TGF-beta <span class="highlight" style="background-color:">signaling</span>..  From: Systems Biology Approach Reveals Genome to Phenome Correlation in Type 2 Diabetes. </a></p><div class="supp"><p class="details"><div>T2D genome, T2D interactome, T2D transcriptome and antidiabetic drug interacting genes are mapped on to KEGG <span class="highlight" style="background-color:">pathway</span> for TGF-beta <span class="highlight" style="background-color:">signaling</span>. Red: genome, interactome and transcriptome; brown: interactome, transcriptome and antidiabetic drug interacting genes; green: interactome and transcriptome; yellow: interactome; grey: antidiabetic drug interacting genes.</div></p></div><div class="aux"><div class="resc">Priyanka Jain, et al. PLoS One. 2013;8(1):e53522.</div><p class="links"><a class="dblinks" href="/pubmed/23308243" ref="ordinalpos=172&amp;ncbi_uid=4521122&amp;link_uid=23308243">Citation</a><a class="dblinks" href="/pmc/articles/PMC3538588/" ref="ordinalpos=172&amp;ncbi_uid=4521122&amp;link_uid=PMC3538588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>173.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3601818/figure/fig1s3/" ref="ordinalpos=173&amp;ncbi_uid=4745970&amp;link_uid=PMC3601818" href="/pmc/articles/PMC3601818/"><img src="/pmc/articles/instance/3601818/bin/elife00378fs003.gif" src-large="/pmc/articles/instance/3601818/bin/elife00378fs003.jpg" alt="Figure 1—figure supplement 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3601818/" ref="ordinalpos=173&amp;ncbi_uid=4745970&amp;link_uid=PMC3601818" image-link="/pmc/articles/PMC3601818/figure/fig1s3/" class="imagepopup">Figure 1—figure supplement 3.Schematic model of LIN-12/Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: LIN-12/Notch <span class="highlight" style="background-color:">signaling</span> instructs postsynaptic muscle arm development by regulating UNC-40/DCC and MADD-2 in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Schematic model showing the core LIN-12/Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The binding of the ligand APX-1/DSL to the receptor LIN-12/Notch triggers proteolytic cleavages. SEL-12/Presenilin releases the Notch intracellular domain (NICD), which enters the cell nucleus and interacts with LAG-1/CSL and other transcription factors to activate target gene transcription.<br /><b>DOI:</b> http://dx.doi.org/10.7554/eLife.00378.006</div></p></div><div class="aux"><div class="resc">Pengpeng Li, et al. eLife. 2013;2:e00378.</div><p class="links"><a class="dblinks" href="/pubmed/23539368" ref="ordinalpos=173&amp;ncbi_uid=4745970&amp;link_uid=23539368">Citation</a><a class="dblinks" href="/pmc/articles/PMC3601818/" ref="ordinalpos=173&amp;ncbi_uid=4745970&amp;link_uid=PMC3601818">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>174.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3516830/figure/F2/" ref="ordinalpos=174&amp;ncbi_uid=4441370&amp;link_uid=PMC3516830" href="/pmc/articles/PMC3516830/"><img src="/pmc/articles/instance/3516830/bin/fgene-03-00286-g0002.gif" src-large="/pmc/articles/instance/3516830/bin/fgene-03-00286-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3516830/" ref="ordinalpos=174&amp;ncbi_uid=4441370&amp;link_uid=PMC3516830" image-link="/pmc/articles/PMC3516830/figure/F2/" class="imagepopup">Figure 2.  From: EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and related-miRNAs in age-related diseases: the example of miR-221 and miR-222. </a></p><div class="supp"><p class="details"><div><b>Interaction loop between the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and miR-221/222 expression in the modulation of cellular processes</b>.</div></p></div><div class="aux"><div class="resc">Ana L. Teixeira, et al. Front Genet. 2012;3:286.</div><p class="links"><a class="dblinks" href="/pubmed/23233863" ref="ordinalpos=174&amp;ncbi_uid=4441370&amp;link_uid=23233863">Citation</a><a class="dblinks" href="/pmc/articles/PMC3516830/" ref="ordinalpos=174&amp;ncbi_uid=4441370&amp;link_uid=PMC3516830">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>175.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3131451/figure/F1/" ref="ordinalpos=175&amp;ncbi_uid=3558802&amp;link_uid=PMC3131451" href="/pmc/articles/PMC3131451/"><img src="/pmc/articles/instance/3131451/bin/nihms251700f1.gif" src-large="/pmc/articles/instance/3131451/bin/nihms251700f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3131451/" ref="ordinalpos=175&amp;ncbi_uid=3558802&amp;link_uid=PMC3131451" image-link="/pmc/articles/PMC3131451/figure/F1/" class="imagepopup">Figure 1.  From: Design, synthesis, and structure-activity relationships of 3-ethynyl-1H-indazoles as inhibitors of Phosphatidylinositol 3-kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of the PI3K/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> highlighting the phosphorylation sites tested in our cellular and biochemical assays.</div></p></div><div class="aux"><div class="resc">Elisa Barile, et al. J Med Chem. ;53(23):8368-8375.</div><p class="links"><a class="dblinks" href="/pubmed/21062009" ref="ordinalpos=175&amp;ncbi_uid=3558802&amp;link_uid=21062009">Citation</a><a class="dblinks" href="/pmc/articles/PMC3131451/" ref="ordinalpos=175&amp;ncbi_uid=3558802&amp;link_uid=PMC3131451">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>176.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3518120/figure/F5/" ref="ordinalpos=176&amp;ncbi_uid=5534859&amp;link_uid=PMC3518120" href="/pmc/articles/PMC3518120/"><img src="/pmc/articles/instance/3518120/bin/zjw0121243150005.gif" src-large="/pmc/articles/instance/3518120/bin/zjw0121243150005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3518120/" ref="ordinalpos=176&amp;ncbi_uid=5534859&amp;link_uid=PMC3518120" image-link="/pmc/articles/PMC3518120/figure/F5/" class="imagepopup">Fig. 5. From: Targeted Proteomics of the Secretory <span class="highlight" style="background-color:">Pathway</span> Reveals the Secretome of Mouse Embryonic Fibroblasts and Human Embryonic Stem Cells . </a></p><div class="supp"><p class="details"><div><b>Identification of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins in hESCs and MEFs.</b> Cytokines and growth factor receptors identified in hESCs and cytokines identified in MEFs that are associated with the TGF-β <span class="highlight" style="background-color:">pathway</span>, Wnt <span class="highlight" style="background-color:">pathway</span>, FGF <span class="highlight" style="background-color:">pathway</span>, and Insulin/IGF <span class="highlight" style="background-color:">pathway</span> are listed. Also shown are proteases and protease inhibitors identified in hESCs and MEFs.</div></p></div><div class="aux"><div class="resc">Prasenjit Sarkar, et al. Mol Cell Proteomics. 2012 December;11(12):1829-1839.</div><p class="links"><a class="dblinks" href="/pubmed/22984290" ref="ordinalpos=176&amp;ncbi_uid=5534859&amp;link_uid=22984290">Citation</a><a class="dblinks" href="/pmc/articles/PMC3518120/" ref="ordinalpos=176&amp;ncbi_uid=5534859&amp;link_uid=PMC3518120">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>177.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4000644/figure/fig1/" ref="ordinalpos=177&amp;ncbi_uid=6110915&amp;link_uid=PMC4000644" href="/pmc/articles/PMC4000644/"><img src="/pmc/articles/instance/4000644/bin/CMMM2014-340758.001.gif" src-large="/pmc/articles/instance/4000644/bin/CMMM2014-340758.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4000644/" ref="ordinalpos=177&amp;ncbi_uid=6110915&amp;link_uid=PMC4000644" image-link="/pmc/articles/PMC4000644/figure/fig1/" class="imagepopup">Figure 1.  From: Integrating Gene Expression and Protein Interaction Data for <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Prediction of Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>Protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> predicting result between NF-<i>κ</i>B and APP.</div></p></div><div class="aux"><div class="resc">Wei Kong, et al. Comput Math Methods Med. 2014;2014:340758.</div><p class="links"><a class="dblinks" href="/pubmed/24812571" ref="ordinalpos=177&amp;ncbi_uid=6110915&amp;link_uid=24812571">Citation</a><a class="dblinks" href="/pmc/articles/PMC4000644/" ref="ordinalpos=177&amp;ncbi_uid=6110915&amp;link_uid=PMC4000644">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>178.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3746099/figure/F6/" ref="ordinalpos=178&amp;ncbi_uid=5220556&amp;link_uid=PMC3746099" href="/pmc/articles/PMC3746099/"><img src="/pmc/articles/instance/3746099/bin/97320630009739F6.gif" src-large="/pmc/articles/instance/3746099/bin/97320630009739F6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3746099/" ref="ordinalpos=178&amp;ncbi_uid=5220556&amp;link_uid=PMC3746099" image-link="/pmc/articles/PMC3746099/figure/F6/" class="imagepopup">Figure 6.  From: Analysis of gene expression on ngn3 gene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in endocrine pancreatic cancer. </a></p><div class="supp"><p class="details"><div>Functional networks involved in many notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and other associated networks.<i>Network</i></div></p></div><div class="aux"><div class="resc">Prashantha Nagaraja, et al. Bioinformation. 2013;9(14):739-747.</div><p class="links"><a class="dblinks" href="/pubmed/23976832" ref="ordinalpos=178&amp;ncbi_uid=5220556&amp;link_uid=23976832">Citation</a><a class="dblinks" href="/pmc/articles/PMC3746099/" ref="ordinalpos=178&amp;ncbi_uid=5220556&amp;link_uid=PMC3746099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>179.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3186080/figure/F2/" ref="ordinalpos=179&amp;ncbi_uid=4402911&amp;link_uid=PMC3186080" href="/pmc/articles/PMC3186080/"><img src="/pmc/articles/instance/3186080/bin/zpg0041123340002.gif" src-large="/pmc/articles/instance/3186080/bin/zpg0041123340002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3186080/" ref="ordinalpos=179&amp;ncbi_uid=4402911&amp;link_uid=PMC3186080" image-link="/pmc/articles/PMC3186080/figure/F2/" class="imagepopup">Fig. 2. From: Functional Selectivity at the ?-Opioid Receptor: Implications for Understanding Opioid Analgesia and Tolerance. </a></p><div class="supp"><p class="details"><div>Schematic representation of functional selectivity at the level of GPCR and G protein or β-arrestin bias. In this diagram, agonists 1 and 2 both activate receptor G protein-coupling pathways, represented as “<span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> A.” However, agonist 1 leads to recruitment of β-arrestin but agonist 2 does not. Receptor-β-arrestin interactions can serve to disrupt receptor-G protein coupling and may also serve to facilitate GPCR <span class="highlight" style="background-color:">signaling</span> to alternate pathways such as <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> B (Schmid and Bohn, 2010). By not recruiting β-arrestins, agonist 2 activates the receptor without engaging <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> B while allowing <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> A to proceed. Agonist 2 would be considered biased against β-arrestin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Kirsten M. Raehal, et al. Pharmacol Rev. 2011 December;63(4):1001-1019.</div><p class="links"><a class="dblinks" href="/pubmed/21873412" ref="ordinalpos=179&amp;ncbi_uid=4402911&amp;link_uid=21873412">Citation</a><a class="dblinks" href="/pmc/articles/PMC3186080/" ref="ordinalpos=179&amp;ncbi_uid=4402911&amp;link_uid=PMC3186080">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>180.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3058120/figure/F1/" ref="ordinalpos=180&amp;ncbi_uid=3505361&amp;link_uid=PMC3058120" href="/pmc/articles/PMC3058120/"><img src="/pmc/articles/instance/3058120/bin/nihms257215f1.gif" src-large="/pmc/articles/instance/3058120/bin/nihms257215f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3058120/" ref="ordinalpos=180&amp;ncbi_uid=3505361&amp;link_uid=PMC3058120" image-link="/pmc/articles/PMC3058120/figure/F1/" class="imagepopup">Figure 1. Schematic of TOR <span class="highlight" style="background-color:">signaling</span> network.  From: Role of TOR <span class="highlight" style="background-color:">signaling</span> in aging and related biological processes in Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div>Various intra- and extracellular environmental cues activate the TOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to modulation of various biological processes that can modulate lifespan of flies. This figure also indicates the interaction with Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and other signalling pathways. See the main text for details.</div></p></div><div class="aux"><div class="resc">Subhash D. Katewa, et al. Exp Gerontol. ;46(5):382-390.</div><p class="links"><a class="dblinks" href="/pubmed/21130151" ref="ordinalpos=180&amp;ncbi_uid=3505361&amp;link_uid=21130151">Citation</a><a class="dblinks" href="/pmc/articles/PMC3058120/" ref="ordinalpos=180&amp;ncbi_uid=3505361&amp;link_uid=PMC3058120">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>181.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4033362/figure/F3/" ref="ordinalpos=181&amp;ncbi_uid=6198942&amp;link_uid=PMC4033362" href="/pmc/articles/PMC4033362/"><img src="/pmc/articles/instance/4033362/bin/fendo-05-00079-g003.gif" src-large="/pmc/articles/instance/4033362/bin/fendo-05-00079-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4033362/" ref="ordinalpos=181&amp;ncbi_uid=6198942&amp;link_uid=PMC4033362" image-link="/pmc/articles/PMC4033362/figure/F3/" class="imagepopup">Figure 3.  From: Insulin Analogs and Cancer: A Note of Caution. </a></p><div class="supp"><p class="details"><div><b>The IGF-I receptor <span class="highlight" style="background-color:">signaling</span> pathways during normoglycemia:</b> (A) stimulation of the IGF-I receptor leads to phosphorylation of insulin receptor substrate-1 (IRS-1) that couples IGF-I receptor stimulation to downstream activation of the PI-3-kinase (“metabolic”) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(B)</b> IGF-I receptor <span class="highlight" style="background-color:">signaling</span> pathways during hyperglycemia: stimulation of the IGF-I receptor leads not phosphorylation of IRS-1. Instead of this, IGF-IR linked <span class="highlight" style="background-color:">signaling</span> occurs via its ability to phosphorylate SHPS-1, which results in assembly of a SHPS-1 <span class="highlight" style="background-color:">signaling</span> complex which leads to stimulation of both the PI-3 kinase (“metabolic”) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and MAP kinase (“mitogenic”) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [Figure modified from Clemmons et al. (32); see also text].</div></p></div><div class="aux"><div class="resc">Joseph A. M. J. L. Janssen, et al. Front Endocrinol (Lausanne). 2014;5:79.</div><p class="links"><a class="dblinks" href="/pubmed/24904529" ref="ordinalpos=181&amp;ncbi_uid=6198942&amp;link_uid=24904529">Citation</a><a class="dblinks" href="/pmc/articles/PMC4033362/" ref="ordinalpos=181&amp;ncbi_uid=6198942&amp;link_uid=PMC4033362">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>182.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1150820/figure/fig5/" ref="ordinalpos=182&amp;ncbi_uid=502584&amp;link_uid=PMC1150820" href="/pmc/articles/PMC1150820/"><img src="/pmc/articles/instance/1150820/bin/zpq0210583550005.gif" src-large="/pmc/articles/instance/1150820/bin/zpq0210583550005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1150820/" ref="ordinalpos=182&amp;ncbi_uid=502584&amp;link_uid=PMC1150820" image-link="/pmc/articles/PMC1150820/figure/fig5/" class="imagepopup">Fig. 5. From: Uterine Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> is required for implantation. </a></p><div class="supp"><p class="details"><div>Model for blastocyst-induced Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> in the uterus. On the morning of day 4, the blastocyst emits a signal that results either directly or indirectly in the activation of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in discrete groups of cells within the circular smooth muscle, which form bands along the uterus. On late day 4 to early day 5, the blastocyst activates the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the luminal epithelium of the uterus. Activation of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> results in the modulation of target genes in the uterus, which facilitates implantation.</div></p></div><div class="aux"><div class="resc">Othman A. Mohamed, et al. Proc Natl Acad Sci U S A. 2005 June 14;102(24):8579-8584.</div><p class="links"><a class="dblinks" href="/pubmed/15930138" ref="ordinalpos=182&amp;ncbi_uid=502584&amp;link_uid=15930138">Citation</a><a class="dblinks" href="/pmc/articles/PMC1150820/" ref="ordinalpos=182&amp;ncbi_uid=502584&amp;link_uid=PMC1150820">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>183.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g008/" ref="ordinalpos=183&amp;ncbi_uid=5583240&amp;link_uid=PMC3855681" href="/pmc/articles/PMC3855681/"><img src="/pmc/articles/instance/3855681/bin/pone.0080918.g008.gif" src-large="/pmc/articles/instance/3855681/bin/pone.0080918.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855681/" ref="ordinalpos=183&amp;ncbi_uid=5583240&amp;link_uid=PMC3855681" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g008/" class="imagepopup">Figure 8. Comparing -values [0–1] of the molecules present in TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of H. sapiens in three cases: (1) in an unperturbed TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, (2) in a perturbed one and (3) after applying conflicting objective function optimization (case 4) on the perturbed TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: A Model of an Integrated Immune System <span class="highlight" style="background-color:">Pathway</span> in Homo sapiens and Its Interaction with Superantigen Producing Expression Regulatory <span class="highlight" style="background-color:">Pathway</span> in Staphylococcus aureus: Comparing Behavior of Pathogen Perturbed and Unperturbed <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="aux"><div class="resc">Namrata Tomar, et al. PLoS One. 2013;8(12):e80918.</div><p class="links"><a class="dblinks" href="/pubmed/24324645" ref="ordinalpos=183&amp;ncbi_uid=5583240&amp;link_uid=24324645">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855681/" ref="ordinalpos=183&amp;ncbi_uid=5583240&amp;link_uid=PMC3855681">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>184.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3096649/figure/pone-0019846-g007/" ref="ordinalpos=184&amp;ncbi_uid=2965916&amp;link_uid=PMC3096649" href="/pmc/articles/PMC3096649/"><img src="/pmc/articles/instance/3096649/bin/pone.0019846.g007.gif" src-large="/pmc/articles/instance/3096649/bin/pone.0019846.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3096649/" ref="ordinalpos=184&amp;ncbi_uid=2965916&amp;link_uid=PMC3096649" image-link="/pmc/articles/PMC3096649/figure/pone-0019846-g007/" class="imagepopup">Figure 7. Simplified model for IGF1R <span class="highlight" style="background-color:">signaling</span> in ES..  From: IGF1R <span class="highlight" style="background-color:">Signaling</span> in Ewing Sarcoma Is Shaped by Clathrin-/Caveolin-Dependent Endocytosis. </a></p><div class="supp"><p class="details"><div>Globally, after ligand binding IGF1R can signal directly from the membrane (IGF1R <span class="highlight" style="background-color:">signaling</span> internalization-independent), with a major degree of IRS1/PI3K/AKT <span class="highlight" style="background-color:">signaling</span> activation and a smaller enhancement of the Ras/Raf/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, or, when internalized either by clathrin and/or caveolin-dependent mechanisms, it can signal from vesicles. In the case of clathrin-dependent endocytosis, IGF1R is recruited to CCPs through interaction of its cytoplasmic tails with the protein adaptor AP2, which in turn recruits clathrin to the plasma membrane and promotes its assembly into a lattice network. Through protein-protein interaction, clathrin and AP2 also bind several proteins of the endocytic machinery, such as dynamin, which regulates the fission of the CCPs with the consequent formation of CCVs, and GRB2, which recruits and binds the SOS protein and the IGF1R-phosphorylated protein Shc, activating the RAS/RAF/MEK/ERK and p38MAPK <span class="highlight" style="background-color:">signaling</span> pathways. To a lesser extent, it also activates the IRS1/PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the case of caveolin-dependent endocytosis, IGF1R is recruited to caveolae, promoting caveolin phosphorylation and the subsequent recruitment of dynamin, which promotes caveolar budding and the formation of cavicles. Cav-1 phosphorylation phosphorylates IRS-1, enhancing the activation of thePI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The thickness of the arrows indicates the prevalence of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. CPMZ treatment inhibits the formation of CCP by inhibiting clathrin-heavy-chain conformations and MCB inhibits the formation of cavicles by depleting cholesterol, disrupting the caveolae.</div></p></div><div class="aux"><div class="resc">Ana Sofia Martins, et al. PLoS One. 2011;6(5):e19846.</div><p class="links"><a class="dblinks" href="/pubmed/21611203" ref="ordinalpos=184&amp;ncbi_uid=2965916&amp;link_uid=21611203">Citation</a><a class="dblinks" href="/pmc/articles/PMC3096649/" ref="ordinalpos=184&amp;ncbi_uid=2965916&amp;link_uid=PMC3096649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>185.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3481134/figure/fig7/" ref="ordinalpos=185&amp;ncbi_uid=4318294&amp;link_uid=PMC3481134" href="/pmc/articles/PMC3481134/"><img src="/pmc/articles/instance/3481134/bin/cddis2012146f7.gif" src-large="/pmc/articles/instance/3481134/bin/cddis2012146f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3481134/" ref="ordinalpos=185&amp;ncbi_uid=4318294&amp;link_uid=PMC3481134" image-link="/pmc/articles/PMC3481134/figure/fig7/" class="imagepopup">Figure 7.  From: A transcriptionally active pRb-E2F1-P/CAF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is central to TGFβ-mediated apoptosis. </a></p><div class="supp"><p class="details"><div>The pRb/E2F <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates three distinct arms of TGFβ tumor-suppressive effects. See text for details</div></p></div><div class="aux"><div class="resc">J Korah, et al. Cell Death Dis. 2012 October;3(10):e407.</div><p class="links"><a class="dblinks" href="/pubmed/23059826" ref="ordinalpos=185&amp;ncbi_uid=4318294&amp;link_uid=23059826">Citation</a><a class="dblinks" href="/pmc/articles/PMC3481134/" ref="ordinalpos=185&amp;ncbi_uid=4318294&amp;link_uid=PMC3481134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>186.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2779448/figure/pone-0008040-g007/" ref="ordinalpos=186&amp;ncbi_uid=2138026&amp;link_uid=PMC2779448" href="/pmc/articles/PMC2779448/"><img src="/pmc/articles/instance/2779448/bin/pone.0008040.g007.gif" src-large="/pmc/articles/instance/2779448/bin/pone.0008040.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2779448/" ref="ordinalpos=186&amp;ncbi_uid=2138026&amp;link_uid=PMC2779448" image-link="/pmc/articles/PMC2779448/figure/pone-0008040-g007/" class="imagepopup">Figure 7. Essential <span class="highlight" style="background-color:">signaling</span> network of insulin <span class="highlight" style="background-color:">signaling</span> and its feedbacks..  From: A Dynamic Analysis of IRS-PKR <span class="highlight" style="background-color:">Signaling</span> in Liver Cells: A Discrete Modeling Approach. </a></p><div class="supp"><p class="details"><div>The direct interaction between PP2A and AKT is removed. The ERK feedback is removed, together with the downstream factors in the ERK <span class="highlight" style="background-color:">pathway</span> that do not have an effect on the regulation of insulin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Ming Wu, et al. PLoS One. 2009;4(12):e8040.</div><p class="links"><a class="dblinks" href="/pubmed/19956598" ref="ordinalpos=186&amp;ncbi_uid=2138026&amp;link_uid=19956598">Citation</a><a class="dblinks" href="/pmc/articles/PMC2779448/" ref="ordinalpos=186&amp;ncbi_uid=2138026&amp;link_uid=PMC2779448">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>187.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3374043/figure/F3/" ref="ordinalpos=187&amp;ncbi_uid=5151234&amp;link_uid=PMC3374043" href="/pmc/articles/PMC3374043/"><img src="/pmc/articles/instance/3374043/bin/nihms357982f3.gif" src-large="/pmc/articles/instance/3374043/bin/nihms357982f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3374043/" ref="ordinalpos=187&amp;ncbi_uid=5151234&amp;link_uid=PMC3374043" image-link="/pmc/articles/PMC3374043/figure/F3/" class="imagepopup">Figure 3. Overview of Ras <span class="highlight" style="background-color:">Signaling</span>.  From: Ras Family of Small GTPases In Immunity And Inflammation. </a></p><div class="supp"><p class="details"><div>Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is highlighted in Red. The Ral <span class="highlight" style="background-color:">signaling</span> segment of the Ras <span class="highlight" style="background-color:">pathway</span> is additionally marked in Green.</div></p></div><div class="aux"><div class="resc">Derek S. Johnson, et al. Curr Opin Pharmacol. ;12(4):458-463.</div><p class="links"><a class="dblinks" href="/pubmed/22401931" ref="ordinalpos=187&amp;ncbi_uid=5151234&amp;link_uid=22401931">Citation</a><a class="dblinks" href="/pmc/articles/PMC3374043/" ref="ordinalpos=187&amp;ncbi_uid=5151234&amp;link_uid=PMC3374043">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>188.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2432088/figure/F1/" ref="ordinalpos=188&amp;ncbi_uid=1355202&amp;link_uid=PMC2432088" href="/pmc/articles/PMC2432088/"><img src="/pmc/articles/instance/2432088/bin/nihms51274f1.gif" src-large="/pmc/articles/instance/2432088/bin/nihms51274f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2432088/" ref="ordinalpos=188&amp;ncbi_uid=1355202&amp;link_uid=PMC2432088" image-link="/pmc/articles/PMC2432088/figure/F1/" class="imagepopup">Figure 1. Intracellular <span class="highlight" style="background-color:">signaling</span> for Wnt is mediated through the canonical and non-canonical pathways.  From: THE WNT <span class="highlight" style="background-color:">SIGNALING</span> <span class="highlight" style="background-color:">PATHWAY</span>: AGING GRACEFULLY AS A PROTECTIONIST?. </a></p><div class="supp"><p class="details"><div>A representative schematic for the canonical and non-canonical pathways is shown. The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is referred to as the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> since it can regulate β-catenin. All Wnt <span class="highlight" style="background-color:">signaling</span> pathways are initiated by interaction of Wnt proteins with Frizzled (FZD) receptors. The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> will only be activated if the binding of the Wnt protein to the FZD receptor takes place in the presence of the co-receptor LRP-5/6 resulting in the formation of a Wnt-FZD-LRP-5/6 tri-molecular complex. Once Wnt protein binds to the FZD receptor and the co-receptor LRP-5/6, this is followed by recruitment of Dishevelled (DVL). DVL is phosphorylated by casein kinase Iε to form a complex with Frat1 and inhibit glycogen synthase kinase (GSK-3β) activity. The non-canonical or atypical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has two intracellular <span class="highlight" style="background-color:">signaling</span> cascades that consist of the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> and the Wnt/PCP <span class="highlight" style="background-color:">pathway</span>. In the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span>, Wnt protein binds to FZD receptors on the cell surface resulting in several cellular processes that involve stimulation of heterotrimeric G proteins, increased intracellular Ca<sup>2+</sup> release, decreased cyclic guanosine monophosphate (cGMP) levels, and activation of the two kinases Ca<sup>2+</sup>-calmodulin-dependent protein kinase II (CamKII) or calcineurin (CaCN) and protein kinase C (PKC). Activation of these pathways can stimulate several transcription factors. In the Wnt/PCP <span class="highlight" style="background-color:">pathway</span>, Wnt proteins bind to FZD receptors on the cell surface followed by activating Rho/Rac small GTPase and Jun N-terminal kinase (JNK) to assist with cytoskeletal organization and gene expression.</div></p></div><div class="aux"><div class="resc">Kenneth Maiese, et al. Pharmacol Ther. ;118(1):58-81.</div><p class="links"><a class="dblinks" href="/pubmed/18313758" ref="ordinalpos=188&amp;ncbi_uid=1355202&amp;link_uid=18313758">Citation</a><a class="dblinks" href="/pmc/articles/PMC2432088/" ref="ordinalpos=188&amp;ncbi_uid=1355202&amp;link_uid=PMC2432088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>189.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F2/" ref="ordinalpos=189&amp;ncbi_uid=2525567&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-2.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=189&amp;ncbi_uid=2525567&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F2/" class="imagepopup">Figure 2.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>ERBB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. ERBB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human taken from KEGG [9].</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=189&amp;ncbi_uid=2525567&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=189&amp;ncbi_uid=2525567&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>190.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2940266/figure/F2/" ref="ordinalpos=190&amp;ncbi_uid=2467904&amp;link_uid=PMC2940266" href="/pmc/articles/PMC2940266/"><img src="/pmc/articles/instance/2940266/bin/nihms99348f2.gif" src-large="/pmc/articles/instance/2940266/bin/nihms99348f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2940266/" ref="ordinalpos=190&amp;ncbi_uid=2467904&amp;link_uid=PMC2940266" image-link="/pmc/articles/PMC2940266/figure/F2/" class="imagepopup">Figure 2. Crosstalks between PTEN and other <span class="highlight" style="background-color:">signaling</span> networks.  From: Phosphatase and Tensin Homologue deleted on Chromosome 10: Extending its PTENtacles. </a></p><div class="supp"><p class="details"><div>The PTEN regulated PI3K/AKT <span class="highlight" style="background-color:">signaling</span> cross talks with several other <span class="highlight" style="background-color:">signaling</span> pathways. These pathways include the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the energy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, as well as p53 <span class="highlight" style="background-color:">signaling</span>. Through inhibiting GSK3β, activation of AKT leads to the stabilization of β-catenin, a critical factor in Wnt <span class="highlight" style="background-color:">signaling</span>. PI3K/AKT signal also cross talks with energy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at TSC. Whereas AKT phophorylates and inhibit TSC, AMPK, a kinase activated in response to an increase in AMP:ATP ratio phosporylates and activates TSC. In addition to its function primarily through AKT in the cytoplasm, nuclear PTEN may have its unique functions. These include direct association with p53, CENP and regulation of rad 51 to control the later stages of the cell cycle and DNA repair mechanisms.</div></p></div><div class="aux"><div class="resc">Bangyan L. Stiles. Int J Biochem Cell Biol. ;41(4):757-761.</div><p class="links"><a class="dblinks" href="/pubmed/18950730" ref="ordinalpos=190&amp;ncbi_uid=2467904&amp;link_uid=18950730">Citation</a><a class="dblinks" href="/pmc/articles/PMC2940266/" ref="ordinalpos=190&amp;ncbi_uid=2467904&amp;link_uid=PMC2940266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>191.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4263532/figure/pone-0114269-g001/" ref="ordinalpos=191&amp;ncbi_uid=6982027&amp;link_uid=PMC4263532" href="/pmc/articles/PMC4263532/"><img src="/pmc/articles/instance/4263532/bin/pone.0114269.g001.gif" src-large="/pmc/articles/instance/4263532/bin/pone.0114269.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4263532/" ref="ordinalpos=191&amp;ncbi_uid=6982027&amp;link_uid=PMC4263532" image-link="/pmc/articles/PMC4263532/figure/pone-0114269-g001/" class="imagepopup">Figure 1. Model of MOPr <span class="highlight" style="background-color:">signaling</span> showing that distinct molecules contribute to the development and expression of opioid tolerance..  From: Functionally Selective <span class="highlight" style="background-color:">Signaling</span> for Morphine and Fentanyl Antinociception and Tolerance Mediated by the Rat Periaqueductal Gray. </a></p><div class="supp"><p class="details"><div>Tolerance could be caused by a change anywhere along the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. If this change occurs at point C in the model, then repeated co-administration of an opioid with a drug that blocks <span class="highlight" style="background-color:">signaling</span> at points A, B, or C will prevent the development of tolerance. Once tolerance has developed, blocking <span class="highlight" style="background-color:">signaling</span> at points A or B will have no effect on the expression of tolerance because <span class="highlight" style="background-color:">signaling</span> at point C is already altered. However, a drug that blocks the enhanced <span class="highlight" style="background-color:">signaling</span> from points C, D, or E will block the expression of tolerance.</div></p></div><div class="aux"><div class="resc">Michael M. Morgan, et al. PLoS One. 2014;9(12):e114269.</div><p class="links"><a class="dblinks" href="/pubmed/25503060" ref="ordinalpos=191&amp;ncbi_uid=6982027&amp;link_uid=25503060">Citation</a><a class="dblinks" href="/pmc/articles/PMC4263532/" ref="ordinalpos=191&amp;ncbi_uid=6982027&amp;link_uid=PMC4263532">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>192.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508164/figure/F12/" ref="ordinalpos=192&amp;ncbi_uid=5657094&amp;link_uid=PMC3508164" href="/pmc/articles/PMC3508164/"><img src="/pmc/articles/instance/3508164/bin/nihms411023f12.gif" src-large="/pmc/articles/instance/3508164/bin/nihms411023f12.jpg" alt="Fig. 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508164/" ref="ordinalpos=192&amp;ncbi_uid=5657094&amp;link_uid=PMC3508164" image-link="/pmc/articles/PMC3508164/figure/F12/" class="imagepopup">Fig. 12.  From: Cripto-1 enhances the canonical Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by binding to LRP5 and LRP6 co-receptors. </a></p><div class="supp"><p class="details"><div>Proposed model of Cripto-1 modulation of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cripto-1 can regulate the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through the binding of LRP5/6, which also bind to caveolin-1 in the lipid raft as well as Cripto-1. Cripto-1 facilitates the phosphorylation of LRP5/6 induced by Wnt3a. The phosphorylated LRP6 leads to recruitment of the Axin complex (Axin, APC and GSK3β) to the Wnt receptor complex. The receptor complex is then internalized via a caveolin-mediated route to induce the cytoplasmic accumulation of β-catenin. β-catenin can then form a complex with Tcf/Lef and activate the transcription of target genes after translocation to the nucleus. Cripto-1 also might interfere with the internalization of LRP5/6 that is facilitated by Dkk1, which is a potent inhibitor of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tadahiro Nagaoka, et al. Cell Signal. ;25(1):178-189.</div><p class="links"><a class="dblinks" href="/pubmed/23022962" ref="ordinalpos=192&amp;ncbi_uid=5657094&amp;link_uid=23022962">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508164/" ref="ordinalpos=192&amp;ncbi_uid=5657094&amp;link_uid=PMC3508164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>193.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3622756/figure/F8/" ref="ordinalpos=193&amp;ncbi_uid=6083110&amp;link_uid=PMC3622756" href="/pmc/articles/PMC3622756/"><img src="/pmc/articles/instance/3622756/bin/nihms446473f8.gif" src-large="/pmc/articles/instance/3622756/bin/nihms446473f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3622756/" ref="ordinalpos=193&amp;ncbi_uid=6083110&amp;link_uid=PMC3622756" image-link="/pmc/articles/PMC3622756/figure/F8/" class="imagepopup">Figure 8. Schematic depicting RKIP regulation of MAPK <span class="highlight" style="background-color:">pathway</span> in MEFs.  From: RKIP Regulates MAP Kinase <span class="highlight" style="background-color:">Signaling</span> in Cells with Defective B-Raf Activity. </a></p><div class="supp"><p class="details"><div><b>(A)</b> In normal conditions, activation of growth factor receptors (GFR) lead to activation of the MAPK (ERK) <span class="highlight" style="background-color:">pathway</span> mainly through B-Raf. In this condition Raf-1 is repressed by RKIP and only marginally participates in the <span class="highlight" style="background-color:">signaling</span>. <b>(B)</b> When B-Raf is depleted or deficient, <span class="highlight" style="background-color:">signaling</span> through MAPK is severely hindered since RKIP prevents full activation of Raf-1 and downstream phosphorylation of ERK. <b>(C)</b> When RKIP and B-Raf are deficient or depleted, <span class="highlight" style="background-color:">signaling</span> to ERK occurs through Raf-1. The thickness of the blue arrows is proportional to the intensity of the <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Lingchun Zeng, et al. Cell Signal. 2013 May;25(5):1156-1165.</div><p class="links"><a class="dblinks" href="/pubmed/23416466" ref="ordinalpos=193&amp;ncbi_uid=6083110&amp;link_uid=23416466">Citation</a><a class="dblinks" href="/pmc/articles/PMC3622756/" ref="ordinalpos=193&amp;ncbi_uid=6083110&amp;link_uid=PMC3622756">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>194.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912804/figure/F3/" ref="ordinalpos=194&amp;ncbi_uid=5822249&amp;link_uid=PMC3912804" href="/pmc/articles/PMC3912804/"><img src="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g003.gif" src-large="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g003.jpg" alt="Figure 2a" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912804/" ref="ordinalpos=194&amp;ncbi_uid=5822249&amp;link_uid=PMC3912804" image-link="/pmc/articles/PMC3912804/figure/F3/" class="imagepopup">Figure 2a.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: A novel target for cancer therapy: Vismodegib, a promising therapeutic option in treatment of basal cell carcinomas. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancers-Type I-Ligand-independent cancers</div></p></div><div class="aux"><div class="resc">Afroz Abidi. Indian J Pharmacol. 2014 Jan-Feb;46(1):3-12.</div><p class="links"><a class="dblinks" href="/pubmed/24550577" ref="ordinalpos=194&amp;ncbi_uid=5822249&amp;link_uid=24550577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912804/" ref="ordinalpos=194&amp;ncbi_uid=5822249&amp;link_uid=PMC3912804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>195.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4200751/figure/ijms-15-15778-f006/" ref="ordinalpos=195&amp;ncbi_uid=6735453&amp;link_uid=PMC4200751" href="/pmc/articles/PMC4200751/"><img src="/pmc/articles/instance/4200751/bin/ijms-15-15778-g006.gif" src-large="/pmc/articles/instance/4200751/bin/ijms-15-15778-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4200751/" ref="ordinalpos=195&amp;ncbi_uid=6735453&amp;link_uid=PMC4200751" image-link="/pmc/articles/PMC4200751/figure/ijms-15-15778-f006/" class="imagepopup">Figure 6.  From: Leptin Induces Oncostatin M Production in Osteoblasts by Downregulating miR-93 through the Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Schema of <span class="highlight" style="background-color:">signaling</span> pathways involved in leptin-induced OSM expression in osteoblasts.Leptin enhances OSM production in human osteoblasts by inhibition miR-93 expression through the Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Wei-Hung Yang, et al. Int J Mol Sci. 2014 September;15(9):15778-15790.</div><p class="links"><a class="dblinks" href="/pubmed/25198901" ref="ordinalpos=195&amp;ncbi_uid=6735453&amp;link_uid=25198901">Citation</a><a class="dblinks" href="/pmc/articles/PMC4200751/" ref="ordinalpos=195&amp;ncbi_uid=6735453&amp;link_uid=PMC4200751">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>196.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2888641/figure/F2/" ref="ordinalpos=196&amp;ncbi_uid=2982415&amp;link_uid=PMC2888641" href="/pmc/articles/PMC2888641/"><img src="/pmc/articles/instance/2888641/bin/nihms196795f2.gif" src-large="/pmc/articles/instance/2888641/bin/nihms196795f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2888641/" ref="ordinalpos=196&amp;ncbi_uid=2982415&amp;link_uid=PMC2888641" image-link="/pmc/articles/PMC2888641/figure/F2/" class="imagepopup">Figure 2. Models of Hh <span class="highlight" style="background-color:">signaling</span> in cancer.  From: Targeting Hedgehog - a Cancer Stem Cell <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) Non-ligand mediated <span class="highlight" style="background-color:">signaling</span> (mutational activation). Mutations in <i>PTCH1</i>, <i>SMO</i>, <i>SUFU</i> or amplification of <i>GLI1</i> have all been reported in human cacners. i. Loss of PTCH1 activity may increase Hh <span class="highlight" style="background-color:">pathway</span> activity. ii. Overexpression or activating mutations in SMO are also depicted. (B) Ligand mediated autocrine <span class="highlight" style="background-color:">signaling</span>. Tumor cells produce Hh ligand that stimulates Hh <span class="highlight" style="background-color:">pathway</span> activity in tumor cells. (C) Ligand mediated paracrine <span class="highlight" style="background-color:">signaling</span>. Non-malignant stromal cells produces Hh ligand required by tumor cells for growth and survival. (D) Ligand mediated paracrine <span class="highlight" style="background-color:">signaling</span>. Tumor cells produce Hh ligand that activates Hh <span class="highlight" style="background-color:">signaling</span> in non-malignant stromal and endothelial cells. This results in the production of unknown factors within the microenvironment that ultimately supports tumor cell growth and survival as well as angiogenesis.</div></p></div><div class="aux"><div class="resc">Akil Merchant, et al. Clin Cancer Res. ;16(12):3130-3140.</div><p class="links"><a class="dblinks" href="/pubmed/20530699" ref="ordinalpos=196&amp;ncbi_uid=2982415&amp;link_uid=20530699">Citation</a><a class="dblinks" href="/pmc/articles/PMC2888641/" ref="ordinalpos=196&amp;ncbi_uid=2982415&amp;link_uid=PMC2888641">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>197.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3049961/figure/F8/" ref="ordinalpos=197&amp;ncbi_uid=3448569&amp;link_uid=PMC3049961" href="/pmc/articles/PMC3049961/"><img src="/pmc/articles/instance/3049961/bin/nihms-266451-f0008.gif" src-large="/pmc/articles/instance/3049961/bin/nihms-266451-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3049961/" ref="ordinalpos=197&amp;ncbi_uid=3448569&amp;link_uid=PMC3049961" image-link="/pmc/articles/PMC3049961/figure/F8/" class="imagepopup">Figure 8. <span class="highlight" style="background-color:">Pathway</span> analysis of ΔEGFR <span class="highlight" style="background-color:">signaling</span> in glioma cells.  From: Analysis of phosphotyrosine <span class="highlight" style="background-color:">signaling</span> in glioblastoma identifies STAT5 as a novel downstream target of ?EGFR. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> Studio 6 was used to construct a <span class="highlight" style="background-color:">pathway</span> from all phosphoproteins that had a higher signal in ΔEGFR vs ΔEGFR-ki and a p&lt;0.05 in statistical analysis (red). The mammalian database was used with protein modification being the only parameter used to connect different protein nodes. <span class="highlight" style="background-color:">Pathway</span> Studio added several intermediate nodes (yellow) to complete the <span class="highlight" style="background-color:">pathway</span>. For more information on the proteins shown in this analysis and their relationship see Supplemental Data, Tables S1 and S2.</div></p></div><div class="aux"><div class="resc">Vaibhav Chumbalkar, et al. J Proteome Res. ;10(3):1343-1352.</div><p class="links"><a class="dblinks" href="/pubmed/21214269" ref="ordinalpos=197&amp;ncbi_uid=3448569&amp;link_uid=21214269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3049961/" ref="ordinalpos=197&amp;ncbi_uid=3448569&amp;link_uid=PMC3049961">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>198.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4076832/figure/pone-0100094-g001/" ref="ordinalpos=198&amp;ncbi_uid=6317570&amp;link_uid=PMC4076832" href="/pmc/articles/PMC4076832/"><img src="/pmc/articles/instance/4076832/bin/pone.0100094.g001.gif" src-large="/pmc/articles/instance/4076832/bin/pone.0100094.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4076832/" ref="ordinalpos=198&amp;ncbi_uid=6317570&amp;link_uid=PMC4076832" image-link="/pmc/articles/PMC4076832/figure/pone-0100094-g001/" class="imagepopup">Figure 1. A schematic picture demonstrating the <span class="highlight" style="background-color:">pathway</span> reconstruction workflow..  From: <span class="highlight" style="background-color:">Pathway</span> Reconstruction of Airway Remodeling in Chronic Lung Diseases: A Systems Biology Approach. </a></p><div class="supp"><p class="details"><div>The workflow shows that the master gene list is provided from two sources (microarray gene expression and literature mining). Protein-Protein interaction network (PPi) and Gene Regulatory Network (GRN) are compiled using Cytoscape software and its plugins. Also, <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reconstruction is accomplished in CellDesigner software based on biological knowledge accumulated in related literatures and public databases (e.g. Gene Ontology, Uniprot, and NCBI geneRIFs) and related <span class="highlight" style="background-color:">signaling</span> pathways deposited in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> databases such as Biocarta, Reactome, and NCI-PID. The final mustard <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was drawn manually using CellDesigner.</div></p></div><div class="aux"><div class="resc">Ali Najafi, et al. PLoS One. 2014;9(6):e100094.</div><p class="links"><a class="dblinks" href="/pubmed/24978043" ref="ordinalpos=198&amp;ncbi_uid=6317570&amp;link_uid=24978043">Citation</a><a class="dblinks" href="/pmc/articles/PMC4076832/" ref="ordinalpos=198&amp;ncbi_uid=6317570&amp;link_uid=PMC4076832">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>199.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3306546/figure/F6/" ref="ordinalpos=199&amp;ncbi_uid=4750178&amp;link_uid=PMC3306546" href="/pmc/articles/PMC3306546/"><img src="/pmc/articles/instance/3306546/bin/nihms330947f6.gif" src-large="/pmc/articles/instance/3306546/bin/nihms330947f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3306546/" ref="ordinalpos=199&amp;ncbi_uid=4750178&amp;link_uid=PMC3306546" image-link="/pmc/articles/PMC3306546/figure/F6/" class="imagepopup">Fig. 6. Schematic representation of the network of metabolic and <span class="highlight" style="background-color:">signaling</span> pathways in proliferating cells treated with CP-320626.  From: Inhibition of glycogen phosphorylation induces changes in cellular proteome and <span class="highlight" style="background-color:">signaling</span> pathways in MIA pancreatic cancer cells. </a></p><div class="supp"><p class="details"><div>This schema shows the interconnected network of the key metabolic and <span class="highlight" style="background-color:">signaling</span> pathways, including glycolysis, oxidative phosphorylation, the pentose phosphate <span class="highlight" style="background-color:">pathway</span>, and critical cell <span class="highlight" style="background-color:">signaling</span> pathways. Biological macromolecular synthesis can be influenced by these key metabolic and <span class="highlight" style="background-color:">signaling</span> pathways. The critical metabolic and <span class="highlight" style="background-color:">signaling</span> pathways are labeled in blue with boxes. The up-regulated proteins are shown in red and down-regulated proteins are in green</div></p></div><div class="aux"><div class="resc">Danjun Ma, et al. Pancreas. ;41(3):397-408.</div><p class="links"><a class="dblinks" href="/pubmed/22158071" ref="ordinalpos=199&amp;ncbi_uid=4750178&amp;link_uid=22158071">Citation</a><a class="dblinks" href="/pmc/articles/PMC3306546/" ref="ordinalpos=199&amp;ncbi_uid=4750178&amp;link_uid=PMC3306546">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>200.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912804/figure/F2/" ref="ordinalpos=200&amp;ncbi_uid=5822250&amp;link_uid=PMC3912804" href="/pmc/articles/PMC3912804/"><img src="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g002.gif" src-large="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g002.jpg" alt="Figure 1b" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912804/" ref="ordinalpos=200&amp;ncbi_uid=5822250&amp;link_uid=PMC3912804" image-link="/pmc/articles/PMC3912804/figure/F2/" class="imagepopup">Figure 1b.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: A novel target for cancer therapy: Vismodegib, a promising therapeutic option in treatment of basal cell carcinomas. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in vertebrates — with hedgehog ligand</div></p></div><div class="aux"><div class="resc">Afroz Abidi. Indian J Pharmacol. 2014 Jan-Feb;46(1):3-12.</div><p class="links"><a class="dblinks" href="/pubmed/24550577" ref="ordinalpos=200&amp;ncbi_uid=5822250&amp;link_uid=24550577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912804/" ref="ordinalpos=200&amp;ncbi_uid=5822250&amp;link_uid=PMC3912804">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="1" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="2" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="3" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fc656cbe05d491bef44d0">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553fc5f4989203352380d0dd">Multi-scale agent-based brain cancer modeling and prediction of TKI treatment re...</a>
          <div class="ralinkpop offscreen_noflow">Multi-scale agent-based brain cancer modeling and prediction of TKI treatment response: Incorporating EGFR signaling pathway and angiogenesis<div class="brieflinkpopdesc">BMC Bioinformatics. 2012; 13()218</div></div>
          <div class="tertiary"></div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553ebc89437716036a8a124a">disease pathway <span class="number">(392428)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553ebc72437716036a8a0f36">differentiation pathway <span class="number">(234089)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553ebc53cbe05d491bc316a6">metabolic pathway <span class="number">(214028)</span></a>
          <div class="tertiary">PMC</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T18:09:46-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC3F26540048610000000000E045F1;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC3F26540048610000000000E045F1" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="8A1A393946D25221_0091SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- 8A1A393946D25221_0091SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>