
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="13" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A65AA53FEC7010000000000128346" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="12" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="13" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="14" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="13" /></div>
            <div><h2 class="result_count">Results: 1201 to 1300 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1201.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3730813/figure/F2/" ref="ordinalpos=1201&amp;ncbi_uid=5446915&amp;link_uid=PMC3730813" href="/pmc/articles/PMC3730813/"><img src="/pmc/articles/instance/3730813/bin/nihms491257f2.gif" src-large="/pmc/articles/instance/3730813/bin/nihms491257f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3730813/" ref="ordinalpos=1201&amp;ncbi_uid=5446915&amp;link_uid=PMC3730813" image-link="/pmc/articles/PMC3730813/figure/F2/" class="imagepopup">Figure 2. PRRs share immune <span class="highlight" style="background-color:">signaling</span> pathways.  From: The Role of Pattern Recognition Receptors in Intestinal Inflammation. </a></p><div class="supp"><p class="details"><div>Most TLRs except for TLR3 induce NF-κB activation through the MyD88 <span class="highlight" style="background-color:">pathway</span>. TLR3 exclusively induces IRF3 activation through the TRIF <span class="highlight" style="background-color:">pathway</span>. The TRIF <span class="highlight" style="background-color:">pathway</span> is shared by TLR4 and TLR5. NOD1 and NOD2 also activate NF-κB via the RICK <span class="highlight" style="background-color:">pathway</span>. All TLRs and NODs also induce activate MAP-kinases.</div></p></div><div class="aux"><div class="resc">Masayuki Fukata, et al. Mucosal Immunol. ;6(3):451-463.</div><p class="links"><a class="dblinks" href="/pubmed/23515136" ref="ordinalpos=1201&amp;ncbi_uid=5446915&amp;link_uid=23515136">Citation</a><a class="dblinks" href="/pmc/articles/PMC3730813/" ref="ordinalpos=1201&amp;ncbi_uid=5446915&amp;link_uid=PMC3730813">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1202.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2826372/figure/pbio-1000313-g010/" ref="ordinalpos=1202&amp;ncbi_uid=2278142&amp;link_uid=PMC2826372" href="/pmc/articles/PMC2826372/"><img src="/pmc/articles/instance/2826372/bin/pbio.1000313.g010.gif" src-large="/pmc/articles/instance/2826372/bin/pbio.1000313.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2826372/" ref="ordinalpos=1202&amp;ncbi_uid=2278142&amp;link_uid=PMC2826372" image-link="/pmc/articles/PMC2826372/figure/pbio-1000313-g010/" class="imagepopup">Figure 10. The IMD immune <span class="highlight" style="background-color:">pathway</span> is missing in the pea aphid..  From: Genome Sequence of the Pea Aphid Acyrthosiphon pisum. </a></p><div class="supp"><p class="details"><div>Previously sequenced insect genomes (fly, mosquitoes, honeybee, red flour beetle) have indicated that the immune <span class="highlight" style="background-color:">signaling</span> pathways, including IMD and Toll pathways shown here, are conserved across insects. In <i>Drosophila</i>, response to many Gram-negative bacteria and some Gram-positive bacteria and fungi relies on the IMD <span class="highlight" style="background-color:">pathway</span>. In aphids, missing IMD <span class="highlight" style="background-color:">pathway</span> genes (dashed lines) include those involved in recognition (PGRPs) and <span class="highlight" style="background-color:">signaling</span> (IMD, dFADD, Dredd, REL). Genes encoding antimicrobial peptides common in other insects, including defensins and cecropins, are also missing. In contrast, we found putative homologs for all genes central to the Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is key to response to bacteria, fungi, and other microbes in <i>Drosophila.</i></div></p></div><div class="aux"><div class="resc">. PLoS Biol. 2010 February;8(2):e1000313.</div><p class="links"><a class="dblinks" href="/pubmed/20186266" ref="ordinalpos=1202&amp;ncbi_uid=2278142&amp;link_uid=20186266">Citation</a><a class="dblinks" href="/pmc/articles/PMC2826372/" ref="ordinalpos=1202&amp;ncbi_uid=2278142&amp;link_uid=PMC2826372">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1203.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2891218/figure/F7/" ref="ordinalpos=1203&amp;ncbi_uid=3182985&amp;link_uid=PMC2891218" href="/pmc/articles/PMC2891218/"><img src="/pmc/articles/instance/2891218/bin/nihms168087f7.gif" src-large="/pmc/articles/instance/2891218/bin/nihms168087f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2891218/" ref="ordinalpos=1203&amp;ncbi_uid=3182985&amp;link_uid=PMC2891218" image-link="/pmc/articles/PMC2891218/figure/F7/" class="imagepopup">Figure 7.  From: ENDOCANNABINOID <span class="highlight" style="background-color:">SIGNALING</span> IN NEUROTOXICITY AND NEUROPROTECTION. </a></p><div class="supp"><p class="details"><div>Possible interactions between cholinergic and eCB <span class="highlight" style="background-color:">signaling</span> in the toxicity of organophosphorus anticholinesterases. OPs may enhance eCB <span class="highlight" style="background-color:">signaling</span> either indirectly through acetylcholinesterase inhibition or directly by binding to components of the eCB <span class="highlight" style="background-color:">pathway</span>. Extensive acetylcholinesterase inhibition will increase synaptic acetylcholine levels and prolong cholinergic receptor activation to elicit cholinergic signs of toxicity. Depolarization of postsynaptic neurons can lead to release of eCBs and modulation of neurotransmitter release. Prolonged cholinergic receptor activation can stimulate downstream non-cholinergic neurons. Glutamatergic neuron activation through NMDA receptors can increase intracellular Ca<sup>++</sup> levels and cellular toxicity, or through metabotropic MGluR receptors can lead to eCB release that may attenuate further recruitment of downstream <span class="highlight" style="background-color:">signaling</span> and toxicity. Postsynaptic muscarinic M1/M3 receptor activation leads to receptor-mediated eCB release to activate CB1 receptors and inhibit ACh release at the presynaptic cholinergic terminal. Upon release, AEA can activate CB1 receptors (extracellularly) to inhibit neurotransmitter release and TRPV1 receptors (intracellularly) to increase Ca<sup>++</sup> influx, while 2-AG acts at the CB1 receptor but not at the TRPV1 site. In the direct <span class="highlight" style="background-color:">pathway</span>, some OPs may bind to molecular components of eCB <span class="highlight" style="background-color:">signaling</span> to alter CB1 receptor activation, eCB synthesis, eCB degradation or eCB uptake and modulate cholinergic and/or non-cholinergic neurotransmitter release. Direct FAAH inhibition by some OPs can increase intracellular AEA, and in some neurons, this may lead to increased TRPV1 activation, elevated intracellular Ca<sup>++</sup> and decreased 2-AG <span class="highlight" style="background-color:">signaling</span>. Reduced 2-AG <span class="highlight" style="background-color:">signaling</span> could in turn reduce eCB mediated inhibition of neurotransmitter release, enhancing cholinergic and non-cholinergic <span class="highlight" style="background-color:">signaling</span>. Direct inhibition of MAGL by some OPs may in turn selectively increase 2-AG <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">C. Pope, et al. Neurotoxicology. ;31(5):562-571.</div><p class="links"><a class="dblinks" href="/pubmed/19969019" ref="ordinalpos=1203&amp;ncbi_uid=3182985&amp;link_uid=19969019">Citation</a><a class="dblinks" href="/pmc/articles/PMC2891218/" ref="ordinalpos=1203&amp;ncbi_uid=3182985&amp;link_uid=PMC2891218">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1204.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193496/figure/fig4/" ref="ordinalpos=1204&amp;ncbi_uid=4090098&amp;link_uid=PMC3193496" href="/pmc/articles/PMC3193496/"><img src="/pmc/articles/instance/3193496/bin/cr201186f4.gif" src-large="/pmc/articles/instance/3193496/bin/cr201186f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193496/" ref="ordinalpos=1204&amp;ncbi_uid=4090098&amp;link_uid=PMC3193496" image-link="/pmc/articles/PMC3193496/figure/fig4/" class="imagepopup">Figure 4.  From: Wnt <span class="highlight" style="background-color:">signaling</span> through T-cell factor phosphorylation. </a></p><div class="supp"><p class="details"><div>Context-dependent function of HIPK2 in Wnt <span class="highlight" style="background-color:">signaling</span>. A positive or negative role of HIPK2 in Wnt <span class="highlight" style="background-color:">signaling</span> depends on the availability and type of TCF proteins that are present in the responding cell. HIPK2 would inhibit the <span class="highlight" style="background-color:">pathway</span> upon phosphorylation of an activator type TCF, such as LEF1, but would activate it upon phosphorylation of a repressor, such as TCF3.</div></p></div><div class="aux"><div class="resc">Sergei Y Sokol. Cell Res. 2011 July;21(7):1002-1012.</div><p class="links"><a class="dblinks" href="/pubmed/21606952" ref="ordinalpos=1204&amp;ncbi_uid=4090098&amp;link_uid=21606952">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193496/" ref="ordinalpos=1204&amp;ncbi_uid=4090098&amp;link_uid=PMC3193496">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1205.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262331/figure/f2-jbm-1-093/" ref="ordinalpos=1205&amp;ncbi_uid=3775457&amp;link_uid=PMC3262331" href="/pmc/articles/PMC3262331/"><img src="/pmc/articles/instance/3262331/bin/jbm-1-093f2.gif" src-large="/pmc/articles/instance/3262331/bin/jbm-1-093f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262331/" ref="ordinalpos=1205&amp;ncbi_uid=3775457&amp;link_uid=PMC3262331" image-link="/pmc/articles/PMC3262331/figure/f2-jbm-1-093/" class="imagepopup">Figure 2.  From: L-CBM <span class="highlight" style="background-color:">signaling</span> in lymphocyte development and function. </a></p><div class="supp"><p class="details"><div>Schematic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by the L-CBM complex in T and B lymphocytes.<br /><b>Notes:</b> The L-CBM complex acts in the canonical NF-κB (p65/p50, cRel/p50) activation <span class="highlight" style="background-color:">pathway</span> mediated by IKKβ downstream of TCRs and BCRs. L-CBM also controls the JNK, particularly JNK2, activation <span class="highlight" style="background-color:">pathway</span> through antigen receptors, leading to c-JUN activation. Upon antigen receptor stimulation, PKCθ in T cells and PKCβ in B cells phosphorylate CARMA1 in the PRD region (see Figure 4), enabling the formation and membrane recruitment of the L-CBM complex. IKKβ positively and negatively regulates L-CBM <span class="highlight" style="background-color:">signaling</span>. These <span class="highlight" style="background-color:">signaling</span> events are regulated by the recruitment of <span class="highlight" style="background-color:">signaling</span> components to specific membrane domains, such as lipid rafts, after receptor triggering. In MZ B cells, L-CBM mediates signals for canonical NF-κB activation through TLR4-Myd88. L-CBM also mediates signals for noncanonical NF-κB (RelB/p52) activation mediated by NIK and IKKα downstream of the BAFF receptor (BAFF-R), which regulates the survival of MZ B cells.</div></p></div><div class="aux"><div class="resc">Hiromitsu Hara, et al. J Blood Med. 2010;1:93-104.</div><p class="links"><a class="dblinks" href="/pubmed/22282688" ref="ordinalpos=1205&amp;ncbi_uid=3775457&amp;link_uid=22282688">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262331/" ref="ordinalpos=1205&amp;ncbi_uid=3775457&amp;link_uid=PMC3262331">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1206.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2278039/figure/F2/" ref="ordinalpos=1206&amp;ncbi_uid=1391521&amp;link_uid=PMC2278039" href="/pmc/articles/PMC2278039/"><img src="/pmc/articles/instance/2278039/bin/nihms43806f2.gif" src-large="/pmc/articles/instance/2278039/bin/nihms43806f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2278039/" ref="ordinalpos=1206&amp;ncbi_uid=1391521&amp;link_uid=PMC2278039" image-link="/pmc/articles/PMC2278039/figure/F2/" class="imagepopup">Figure 2. FIP200 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: FIP200, a key <span class="highlight" style="background-color:">signaling</span> node to coordinately regulate various cellular processes. </a></p><div class="supp"><p class="details"><div>FIP200 functions to regulate diverse cellular processes, including cell proliferation, cell spreading, cell migration, cell growth and cell survival. FIP200 suppresses cell proliferation by regulation of p53-p21, Cyclin D1, Rb1 and FAK pathways; FIP200 negatively regulates cell spreading and migration by its inhibition of FAK function; FIP200 promotes mTOR activation and cell growth through its interaction with TSC1-TSC2 complex; FIP200 positively regulates cell survival by its inhibition of Pyk2 activity and regulation of TNFα-JNK <span class="highlight" style="background-color:">signaling</span> cascade (see text for details).</div></p></div><div class="aux"><div class="resc">Boyi Gan, et al. Cell Signal. ;20(5):787-794.</div><p class="links"><a class="dblinks" href="/pubmed/18036779" ref="ordinalpos=1206&amp;ncbi_uid=1391521&amp;link_uid=18036779">Citation</a><a class="dblinks" href="/pmc/articles/PMC2278039/" ref="ordinalpos=1206&amp;ncbi_uid=1391521&amp;link_uid=PMC2278039">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1207.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2217664/figure/F1/" ref="ordinalpos=1207&amp;ncbi_uid=1166758&amp;link_uid=PMC2217664" href="/pmc/articles/PMC2217664/"><img src="/pmc/articles/instance/2217664/bin/1471-2105-8-S9-S6-1.gif" src-large="/pmc/articles/instance/2217664/bin/1471-2105-8-S9-S6-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2217664/" ref="ordinalpos=1207&amp;ncbi_uid=1166758&amp;link_uid=PMC2217664" image-link="/pmc/articles/PMC2217664/figure/F1/" class="imagepopup">Figure 1.  From: Extracting unrecognized gene relationships from the biomedical literature via matrix factorizations. </a></p><div class="supp"><p class="details"><div><b>Visualization of genes associated with the Reelin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Three-dimensional representations of 50 genes, which were obtained from NMF (<i>k </i>= 3) using an initial matrix <i>H </i>built from genes directly associated with Reelin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <i>j</i>-th gene is located at (<i>h</i><sub>1<i>j</i></sub>, <i>h</i><sub>2<i>j</i></sub>, <i>h</i><sub>3<i>j</i></sub>), where <i>H </i>∈ ℝ<sup>3×50 </sup>= [<i>h</i><sub><i>ij</i></sub>]. (A red circle: RELN; A red circle and blue diamonds: genes associated with the Reelin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Black dots: other genes)</div></p></div><div class="aux"><div class="resc">Hyunsoo Kim, et al. BMC Bioinformatics. 2007;8(Suppl 9):S6-S6.</div><p class="links"><a class="dblinks" href="/pubmed/18047707" ref="ordinalpos=1207&amp;ncbi_uid=1166758&amp;link_uid=18047707">Citation</a><a class="dblinks" href="/pmc/articles/PMC2217664/" ref="ordinalpos=1207&amp;ncbi_uid=1166758&amp;link_uid=PMC2217664">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1208.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4100154/figure/ijms-15-10334-f008/" ref="ordinalpos=1208&amp;ncbi_uid=6381509&amp;link_uid=PMC4100154" href="/pmc/articles/PMC4100154/"><img src="/pmc/articles/instance/4100154/bin/ijms-15-10334-g008.gif" src-large="/pmc/articles/instance/4100154/bin/ijms-15-10334-g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4100154/" ref="ordinalpos=1208&amp;ncbi_uid=6381509&amp;link_uid=PMC4100154" image-link="/pmc/articles/PMC4100154/figure/ijms-15-10334-f008/" class="imagepopup">Figure 8.  From: Eicosapentaenoic Acid Protects against Palmitic Acid-Induced Endothelial Dysfunction via Activation of the AMPK/eNOS <span class="highlight" style="background-color:">Pathway</span> . </a></p><div class="supp"><p class="details"><div>Proposed model of the protective effects of EPA against the PA-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation that leads to endothelial dysfunction. </div></p></div><div class="aux"><div class="resc">Che-Hsin Lee, et al. Int J Mol Sci. 2014 June;15(6):10334-10349.</div><p class="links"><a class="dblinks" href="/pubmed/24918290" ref="ordinalpos=1208&amp;ncbi_uid=6381509&amp;link_uid=24918290">Citation</a><a class="dblinks" href="/pmc/articles/PMC4100154/" ref="ordinalpos=1208&amp;ncbi_uid=6381509&amp;link_uid=PMC4100154">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1209.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3665261/figure/fig2/" ref="ordinalpos=1209&amp;ncbi_uid=4989193&amp;link_uid=PMC3665261" href="/pmc/articles/PMC3665261/"><img src="/pmc/articles/instance/3665261/bin/OXIMED2013-763257.002.gif" src-large="/pmc/articles/instance/3665261/bin/OXIMED2013-763257.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3665261/" ref="ordinalpos=1209&amp;ncbi_uid=4989193&amp;link_uid=PMC3665261" image-link="/pmc/articles/PMC3665261/figure/fig2/" class="imagepopup">Figure 2.  From: Role of the Nrf2-ARE <span class="highlight" style="background-color:">Pathway</span> in Liver Diseases. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> in Nrf2 activation by HCV infection. Oxidative stress and Ca<sup>2+</sup> mobilization induced by HCV infection activate various kinases, resulting in Nrf2 activation.</div></p></div><div class="aux"><div class="resc">Sang Mi Shin, et al. Oxid Med Cell Longev. 2013;2013:763257.</div><p class="links"><a class="dblinks" href="/pubmed/23766860" ref="ordinalpos=1209&amp;ncbi_uid=4989193&amp;link_uid=23766860">Citation</a><a class="dblinks" href="/pmc/articles/PMC3665261/" ref="ordinalpos=1209&amp;ncbi_uid=4989193&amp;link_uid=PMC3665261">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1210.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3398165/figure/F4/" ref="ordinalpos=1210&amp;ncbi_uid=5428403&amp;link_uid=PMC3398165" href="/pmc/articles/PMC3398165/"><img src="/pmc/articles/instance/3398165/bin/nihms-361363-f0004.gif" src-large="/pmc/articles/instance/3398165/bin/nihms-361363-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3398165/" ref="ordinalpos=1210&amp;ncbi_uid=5428403&amp;link_uid=PMC3398165" image-link="/pmc/articles/PMC3398165/figure/F4/" class="imagepopup">Figure 4.  From: Remarkable inhibition of mTOR <span class="highlight" style="background-color:">signaling</span> by the combination of rapamycin and 1,4-phenylenebis(methylene)selenocyanate in human prostate cancer cells. </a></p><div class="supp"><p class="details"><div>Proposed scheme for the inhibition of both arms of the mTOR <span class="highlight" style="background-color:">pathway</span> by <i>p</i>-XSC and rapamycin in C4-2 cells. Rapamycin inhibits mTORC1 <span class="highlight" style="background-color:">signaling</span> while <i>p</i>-XSC inhibits mTORC2 <span class="highlight" style="background-color:">signaling</span>. <i>p</i>-XSC treatment may offset feedback activation of Akt by rapamycin (represented by broken lines).</div></p></div><div class="aux"><div class="resc">Nicole D. Facompre, et al. Int J Cancer. ;131(9):2134-2142.</div><p class="links"><a class="dblinks" href="/pubmed/22307455" ref="ordinalpos=1210&amp;ncbi_uid=5428403&amp;link_uid=22307455">Citation</a><a class="dblinks" href="/pmc/articles/PMC3398165/" ref="ordinalpos=1210&amp;ncbi_uid=5428403&amp;link_uid=PMC3398165">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1211.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2727709/figure/F7/" ref="ordinalpos=1211&amp;ncbi_uid=1950302&amp;link_uid=PMC2727709" href="/pmc/articles/PMC2727709/"><img src="/pmc/articles/instance/2727709/bin/nihms90178f7.gif" src-large="/pmc/articles/instance/2727709/bin/nihms90178f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2727709/" ref="ordinalpos=1211&amp;ncbi_uid=1950302&amp;link_uid=PMC2727709" image-link="/pmc/articles/PMC2727709/figure/F7/" class="imagepopup">Figure 7.  From: Quantitative Time-Resolved Phosphoproteomic Analysis of Mast Cell <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Quantitative proteomic analysis of proteins not previously associated definitively with any <span class="highlight" style="background-color:">signaling</span> cascade but observed in MCP5 cells. This heatmap represents the temporal changes in tyrosine phosphorylation of proteins with undefined function in any <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> following FcεRI aggregation in MCP5 cells.</div></p></div><div class="aux"><div class="resc">Lulu Cao, et al. J Immunol. ;179(9):5864-5876.</div><p class="links"><a class="dblinks" href="/pubmed/17947660" ref="ordinalpos=1211&amp;ncbi_uid=1950302&amp;link_uid=17947660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2727709/" ref="ordinalpos=1211&amp;ncbi_uid=1950302&amp;link_uid=PMC2727709">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1212.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2515336/figure/F2/" ref="ordinalpos=1212&amp;ncbi_uid=1476194&amp;link_uid=PMC2515336" href="/pmc/articles/PMC2515336/"><img src="/pmc/articles/instance/2515336/bin/1476-4598-7-64-2.gif" src-large="/pmc/articles/instance/2515336/bin/1476-4598-7-64-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2515336/" ref="ordinalpos=1212&amp;ncbi_uid=1476194&amp;link_uid=PMC2515336" image-link="/pmc/articles/PMC2515336/figure/F2/" class="imagepopup">Figure 2.  From: Apoptotic pathways in pancreatic ductal adenocarcinoma. </a></p><div class="supp"><p class="details"><div><b>Representation of the two NF-κB <span class="highlight" style="background-color:">signaling</span> pathways</b>. Classical <span class="highlight" style="background-color:">signaling</span> (depicted left) is activated by TNFα, IL1β or LPS. Activation of canonical <span class="highlight" style="background-color:">signaling</span> depends on the IKK holocomplex (IKKα, IKKβ, IKKγ/NEMO). Activated IKK phosphorylates IκBs, leading to the proteasomal degradation and subsequent nuclear translocation of classical NF-κB. An alternative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (depicted right) is activated by lymphotoxin, CD40L or BAFF. <span class="highlight" style="background-color:">Signaling</span> depends on IKKα and leads to the nuclear translocation of a p52/RelB dimer.</div></p></div><div class="aux"><div class="resc">Rainer Hamacher, et al. Mol Cancer. 2008;7:64-64.</div><p class="links"><a class="dblinks" href="/pubmed/18652674" ref="ordinalpos=1212&amp;ncbi_uid=1476194&amp;link_uid=18652674">Citation</a><a class="dblinks" href="/pmc/articles/PMC2515336/" ref="ordinalpos=1212&amp;ncbi_uid=1476194&amp;link_uid=PMC2515336">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1213.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3656715/figure/F1/" ref="ordinalpos=1213&amp;ncbi_uid=5857631&amp;link_uid=PMC3656715" href="/pmc/articles/PMC3656715/"><img src="/pmc/articles/instance/3656715/bin/nihms-455681-f0001.gif" src-large="/pmc/articles/instance/3656715/bin/nihms-455681-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3656715/" ref="ordinalpos=1213&amp;ncbi_uid=5857631&amp;link_uid=PMC3656715" image-link="/pmc/articles/PMC3656715/figure/F1/" class="imagepopup">Figure 1.  From: The Role of Neurotrophins in Major Depressive Disorder. </a></p><div class="supp"><p class="details"><div>Hippocampal BDNF/TrkB <span class="highlight" style="background-color:">signaling</span> pathways that modulate depressive behavior and antidepressant responses. Binding of BDNF to TrkB activates different intracellular <span class="highlight" style="background-color:">signaling</span> cascades, including the PI3K/Akt, Ras/ERK (MAPK) and PLCγ pathways [12]. Phosphorylation of Y490 of TrkB leads to recruitment of adaptor proteins such as Shc, Grb2 and Gab1 and subsequent activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> [192] and Rho family of GTPases (RhoA, Rac, Cdc42), which positively regulate actin polymerization through downstream <span class="highlight" style="background-color:">signaling</span> including ROCK/PAK, LimK and Cofilin [151-153]. Another downstream effector of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is GSK-3, whose activity is negatively regulated by Akt-mediated phosphorylation [193]. One of the downstream targets of GSK-3 is β-catenin, which can be translocated into the nucleus to promote gene expression. The mTOR <span class="highlight" style="background-color:">pathway</span>, which facilitates local translation and the rapid antidepressant effects of ketamine, is also regulated by PI3K/Akt <span class="highlight" style="background-color:">signaling</span> [122]. Activation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> is believed to positively modulate spinogenesis, synaptogenesis and neurogenesis, thus inducing antidepressant effects. In addition to inductive effects on gene expression, the Ras/ERK (MAPK) <span class="highlight" style="background-color:">pathway</span> also acts through Rho family of GTPases, mTOR <span class="highlight" style="background-color:">signaling</span>, and synapsin to induce spinogenesis, synaptogenesis and neurogenesis (not shown) [122, 153, 154]. Phosphorylation of Y816 of TrkB recruits PLCγ [194], inducing the hydrolysis of PIP2 to IP3 and DAG. IP3 regulates calcium release and CAMK activation, while DAG activates PKC. Both IP3/CAMK and DAG/PKC <span class="highlight" style="background-color:">signaling</span> regulate synaptic plasticity [195], however their roles in depression remain largely unknown.</div></p></div><div class="aux"><div class="resc">Cheng Jiang, et al. Transl Neurosci. 2013 March 1;4(1):46-58.</div><p class="links"><a class="dblinks" href="/pubmed/23691270" ref="ordinalpos=1213&amp;ncbi_uid=5857631&amp;link_uid=23691270">Citation</a><a class="dblinks" href="/pmc/articles/PMC3656715/" ref="ordinalpos=1213&amp;ncbi_uid=5857631&amp;link_uid=PMC3656715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1214.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713831/figure/pone-0006410-g005/" ref="ordinalpos=1214&amp;ncbi_uid=1944673&amp;link_uid=PMC2713831" href="/pmc/articles/PMC2713831/"><img src="/pmc/articles/instance/2713831/bin/pone.0006410.g005.gif" src-large="/pmc/articles/instance/2713831/bin/pone.0006410.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713831/" ref="ordinalpos=1214&amp;ncbi_uid=1944673&amp;link_uid=PMC2713831" image-link="/pmc/articles/PMC2713831/figure/pone-0006410-g005/" class="imagepopup">Figure 5. TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>–related subcluster..  From: Local Network Topology in Human Protein Interaction Data Predicts Functional Association. </a></p><div class="supp"><p class="details"><div>(<i>a</i>) One subcluster identified by our method consists of proteins presumably involved in the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>b</i>) Detailed interpretation of the relationships between each protein from the subcluster. On the basis of the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis 5.0, the 35 blue-green proteins on the circle participate in the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and the 10 red proteins inside the circle are unrelated. The violet proteins outside the circle are common neighbors that do not belong to the subcluster in panel <i>a</i>. Red lines represent significant protein pairs, green lines represent direct protein–protein interactions, and yellow lines represent both.</div></p></div><div class="aux"><div class="resc">Hua Li, et al. PLoS One. 2009;4(7):e6410.</div><p class="links"><a class="dblinks" href="/pubmed/19641626" ref="ordinalpos=1214&amp;ncbi_uid=1944673&amp;link_uid=19641626">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713831/" ref="ordinalpos=1214&amp;ncbi_uid=1944673&amp;link_uid=PMC2713831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1215.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3657316/figure/F3/" ref="ordinalpos=1215&amp;ncbi_uid=5542827&amp;link_uid=PMC3657316" href="/pmc/articles/PMC3657316/"><img src="/pmc/articles/instance/3657316/bin/nihms444128f3.gif" src-large="/pmc/articles/instance/3657316/bin/nihms444128f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3657316/" ref="ordinalpos=1215&amp;ncbi_uid=5542827&amp;link_uid=PMC3657316" image-link="/pmc/articles/PMC3657316/figure/F3/" class="imagepopup">Fig. 3.  From: The role of the cilium in normal and abnormal cell cycles: emphasis on renal cystic pathologies. </a></p><div class="supp"><p class="details"><div>Ciliary <span class="highlight" style="background-color:">signaling</span> pathways implicated in control of cell proliferation. Mechanical sensation of cilia induced by fluid flow activates the LKb1-AMPK <span class="highlight" style="background-color:">pathway</span> in a calcium independent manner and inhibits mTOR1 <span class="highlight" style="background-color:">pathway</span>. Mechanical flow induces activation of PC1 and PC2 and results in elevation of internal calcium level, which suppresses cAMP <span class="highlight" style="background-color:">signaling</span>. In addition, calcium is required for PC1-CT targeting into the nucleus. PC1 also activates JAK2, which induces gene expression that promotes cell cycle progression. Ligand binding of PDGFRα in the cilia activates RAF-ERK <span class="highlight" style="background-color:">pathway</span>. Activation of cAMP <span class="highlight" style="background-color:">signaling</span> promotes PKA and AKAP activity, triggering cell cycle progression whereas calcium inhibits this <span class="highlight" style="background-color:">pathway</span>. In addition, aberrant activation of wnt or hedgehog <span class="highlight" style="background-color:">pathway</span> stimulates cell cycle progression.</div></p></div><div class="aux"><div class="resc">Junmin Pan, et al. Cell Mol Life Sci. 2013 June;70(11):1849-1874.</div><p class="links"><a class="dblinks" href="/pubmed/22782110" ref="ordinalpos=1215&amp;ncbi_uid=5542827&amp;link_uid=22782110">Citation</a><a class="dblinks" href="/pmc/articles/PMC3657316/" ref="ordinalpos=1215&amp;ncbi_uid=5542827&amp;link_uid=PMC3657316">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1216.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2547063/figure/f8/" ref="ordinalpos=1216&amp;ncbi_uid=1495965&amp;link_uid=PMC2547063" href="/pmc/articles/PMC2547063/"><img src="/pmc/articles/instance/2547063/bin/zek0090831870008.gif" src-large="/pmc/articles/instance/2547063/bin/zek0090831870008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2547063/" ref="ordinalpos=1216&amp;ncbi_uid=1495965&amp;link_uid=PMC2547063" image-link="/pmc/articles/PMC2547063/figure/f8/" class="imagepopup">FIG. 8. From: The RGS Protein Crg2 Regulates Pheromone and Cyclic AMP <span class="highlight" style="background-color:">Signaling</span> in Cryptococcus neoformans  . </a></p><div class="supp"><p class="details"><div>The RGS protein Crg2 links the mating and cAMP signal transduction pathways in <i>C</i>. <i>neoformans</i>. The pheromone-responsive mating <span class="highlight" style="background-color:">pathway</span> is regulated by the pheromone receptor Ste3α, heterotrimeric G-protein Gpa2/Gpb1/Gpg1/2, and the RGS protein Crg1. The G-protein-coupled receptor homolog Gpr4, Gpa1/Gib2/Gpg1/Gpg2, and Crg2 govern the conserved cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to regulate differentiation and the production of virulence factors. The Gα protein Gpa3 mediates a distinct <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mating that is negatively regulated by both Crg1 and Crg2. Both the mating and cAMP <span class="highlight" style="background-color:">signaling</span> pathways are important for fungal virulence. MF<b>a</b>/α, mating type <b>a</b> or α; MAPK, mitogen-activated protein kinase.</div></p></div><div class="aux"><div class="resc">Gui Shen, et al. Eukaryot Cell. 2008 September;7(9):1540-1548.</div><p class="links"><a class="dblinks" href="/pubmed/18658258" ref="ordinalpos=1216&amp;ncbi_uid=1495965&amp;link_uid=18658258">Citation</a><a class="dblinks" href="/pmc/articles/PMC2547063/" ref="ordinalpos=1216&amp;ncbi_uid=1495965&amp;link_uid=PMC2547063">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1217.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3153445/figure/F1/" ref="ordinalpos=1217&amp;ncbi_uid=3659758&amp;link_uid=PMC3153445" href="/pmc/articles/PMC3153445/"><img src="/pmc/articles/instance/3153445/bin/nihms-307602-f0001.gif" src-large="/pmc/articles/instance/3153445/bin/nihms-307602-f0001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3153445/" ref="ordinalpos=1217&amp;ncbi_uid=3659758&amp;link_uid=PMC3153445" image-link="/pmc/articles/PMC3153445/figure/F1/" class="imagepopup">Fig 1. Schematic of proposed pathways for β-AR <span class="highlight" style="background-color:">signaling</span> to CaMKII in cardiac myocytes.  From: Crossing Signals: relationships between ?-adrenergic stimulation and CaMKII activation. </a></p><div class="supp"><p class="details"><div>Pathways are color coded. In blue the canonical <span class="highlight" style="background-color:">pathway</span> in which cAMP activates PKA, phosphorylating the L-type Ca<sup>++</sup> channel (LTCC) and RyR2, is shown to activate Ca<sup>++</sup>/CaM and CaMKII. The <span class="highlight" style="background-color:">pathway</span> shown in orange utilizes the exchange protein directly activated by cAMP (Epac) rather than PKA, activating CaMKII through the small GTP’ase Rap, phospholipase C epsilon (PLCε), and protein kinase C epsilon (PKCε). The purple <span class="highlight" style="background-color:">pathway</span> utilizes reactive oxygen species (ROS) <span class="highlight" style="background-color:">signaling</span> to effect CaMKII activation at known sites; the green <span class="highlight" style="background-color:">pathway</span> utilizes NO synthase (NOS) <span class="highlight" style="background-color:">signaling</span> to effect CaMKII activation through more hypothetical mechanisms.</div></p></div><div class="aux"><div class="resc">Michael Grimm, et al. Heart Rhythm. ;8(8):1296-1298.</div><p class="links"><a class="dblinks" href="/pubmed/21354330" ref="ordinalpos=1217&amp;ncbi_uid=3659758&amp;link_uid=21354330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3153445/" ref="ordinalpos=1217&amp;ncbi_uid=3659758&amp;link_uid=PMC3153445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1218.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3478688/figure/F4/" ref="ordinalpos=1218&amp;ncbi_uid=5485201&amp;link_uid=PMC3478688" href="/pmc/articles/PMC3478688/"><img src="/pmc/articles/instance/3478688/bin/DEV083428F4.gif" src-large="/pmc/articles/instance/3478688/bin/DEV083428F4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3478688/" ref="ordinalpos=1218&amp;ncbi_uid=5485201&amp;link_uid=PMC3478688" image-link="/pmc/articles/PMC3478688/figure/F4/" class="imagepopup">Fig. 4. From: Intercellular calcium <span class="highlight" style="background-color:">signaling</span> in a gap junction-coupled cell network establishes asymmetric neuronal fates in C. elegans. </a></p><div class="supp"><p class="details"><div><b>Calcium buffers antagonize the <i>unc-2/unc-36</i> calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> (<b>A</b>) AWC phenotypes of animals expressing calcium buffers alone or in mutants defective in AWC asymmetry. cbn, calbindin D28K; parv, parvalbumin. (<b>B</b>) The AWC asymmetry determination <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jennifer A. Schumacher, et al. Development. 2012 November 15;139(22):4191-4201.</div><p class="links"><a class="dblinks" href="/pubmed/23093425" ref="ordinalpos=1218&amp;ncbi_uid=5485201&amp;link_uid=23093425">Citation</a><a class="dblinks" href="/pmc/articles/PMC3478688/" ref="ordinalpos=1218&amp;ncbi_uid=5485201&amp;link_uid=PMC3478688">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1219.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137710/figure/F2/" ref="ordinalpos=1219&amp;ncbi_uid=3663935&amp;link_uid=PMC3137710" href="/pmc/articles/PMC3137710/"><img src="/pmc/articles/instance/3137710/bin/nihms290606f2.gif" src-large="/pmc/articles/instance/3137710/bin/nihms290606f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137710/" ref="ordinalpos=1219&amp;ncbi_uid=3663935&amp;link_uid=PMC3137710" image-link="/pmc/articles/PMC3137710/figure/F2/" class="imagepopup">Figure 2. Pharmacological inhibition of Wnt <span class="highlight" style="background-color:">signaling</span>.  From: Chemical 'Jekyll and Hyde's: Small-molecule inhibitors of developmental <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>(A) Schematic representation of the Wnt <span class="highlight" style="background-color:">pathway</span>, with key phosphorylation events indicated by the black circles and the nucleus depicted by the dashed box. (B) Chemical structures of the Wnt <span class="highlight" style="background-color:">pathway</span> inhibitors IWR-1, IWR-3, and XAV939.</div></p></div><div class="aux"><div class="resc">Tomoyo Sakata, et al. Chem Soc Rev. ;40(8):4318-4331.</div><p class="links"><a class="dblinks" href="/pubmed/21505654" ref="ordinalpos=1219&amp;ncbi_uid=3663935&amp;link_uid=21505654">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137710/" ref="ordinalpos=1219&amp;ncbi_uid=3663935&amp;link_uid=PMC3137710">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1220.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3954580/figure/F2/" ref="ordinalpos=1220&amp;ncbi_uid=5918739&amp;link_uid=PMC3954580" href="/pmc/articles/PMC3954580/"><img src="/pmc/articles/instance/3954580/bin/nihms337812f2.gif" src-large="/pmc/articles/instance/3954580/bin/nihms337812f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3954580/" ref="ordinalpos=1220&amp;ncbi_uid=5918739&amp;link_uid=PMC3954580" image-link="/pmc/articles/PMC3954580/figure/F2/" class="imagepopup">Fig. 2.  From: MEK5/ERK5 <span class="highlight" style="background-color:">Pathway</span>: The first fifteen years. </a></p><div class="supp"><p class="details"><div>MEK5 regulation and downstream targets. MEK5/ERK5 <span class="highlight" style="background-color:">pathway</span> activation can occur through extracellular stressors, mitogens or cytokine activation. Activation of this <span class="highlight" style="background-color:">pathway</span> may lead to epithelial mesenchymal transitions (EMT), cell survival, anti-apoptotic <span class="highlight" style="background-color:">signaling</span> and an increase in cell proliferation.</div></p></div><div class="aux"><div class="resc">Barbara A. Drew, et al. Biochim Biophys Acta. 2012 January;1825(1):37-48.</div><p class="links"><a class="dblinks" href="/pubmed/22020294" ref="ordinalpos=1220&amp;ncbi_uid=5918739&amp;link_uid=22020294">Citation</a><a class="dblinks" href="/pmc/articles/PMC3954580/" ref="ordinalpos=1220&amp;ncbi_uid=5918739&amp;link_uid=PMC3954580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1221.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3496878/figure/F1/" ref="ordinalpos=1221&amp;ncbi_uid=4370682&amp;link_uid=PMC3496878" href="/pmc/articles/PMC3496878/"><img src="/pmc/articles/instance/3496878/bin/WJG-18-6005-g001.gif" src-large="/pmc/articles/instance/3496878/bin/WJG-18-6005-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3496878/" ref="ordinalpos=1221&amp;ncbi_uid=4370682&amp;link_uid=PMC3496878" image-link="/pmc/articles/PMC3496878/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>/molecular targets and new targeted agents under development in hepatocellular carcinoma. </a></p><div class="supp"><p class="details"><div>Signal transduction in solid cancer cells including hepatocellular carcinoma. Some of the genes known to be functionally altered are highlighted in red. These <span class="highlight" style="background-color:">signaling</span> pathways, including growth factor <span class="highlight" style="background-color:">pathway</span>, Wnt <span class="highlight" style="background-color:">pathway</span>, Hedgehog <span class="highlight" style="background-color:">pathway</span>, Akt/mammalian target of rapamycin (mTOR) <span class="highlight" style="background-color:">pathway</span>, and Jak/Stat <span class="highlight" style="background-color:">pathway</span>, can be a molecular targets for treatment of hepatocellular carcinoma. (Cited and modified from Hanahan et al[10] with permission.) EGFR: Epidermal growth factor receptor; TGF: Transforming growth factor; IGF: Insulin-like growth factor; MAPK: Mitogen-activated protein kinase; PI3K: Phosphoinositide 3-kinase; ERK: Extracellular <span class="highlight" style="background-color:">signaling</span>-regulated kinase; NF-κB: Nuclear factor-kappa B; IL: Interleukin.</div></p></div><div class="aux"><div class="resc">Masatoshi Kudo. World J Gastroenterol. 2012 November 14;18(42):6005-6017.</div><p class="links"><a class="dblinks" href="/pubmed/23155330" ref="ordinalpos=1221&amp;ncbi_uid=4370682&amp;link_uid=23155330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3496878/" ref="ordinalpos=1221&amp;ncbi_uid=4370682&amp;link_uid=PMC3496878">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1222.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3976145/figure/F2/" ref="ordinalpos=1222&amp;ncbi_uid=5995248&amp;link_uid=PMC3976145" href="/pmc/articles/PMC3976145/"><img src="/pmc/articles/instance/3976145/bin/nihms567403f2.gif" src-large="/pmc/articles/instance/3976145/bin/nihms567403f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3976145/" ref="ordinalpos=1222&amp;ncbi_uid=5995248&amp;link_uid=PMC3976145" image-link="/pmc/articles/PMC3976145/figure/F2/" class="imagepopup">Figure 2.  From: Epigenetic Regulation of WNT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Genes in Inflammatory Bowel Disease (IBD) Associated Neoplasia. </a></p><div class="supp"><p class="details"><div>Methylation profile of individual WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes in paired samples from one of the IBD-associated cancers showing the increase in methylation from colitis to cancer. <i>Black bars</i> represent methylation, whereas <i>white bars</i> represent unmethylated sample. Note that the patient has acquired methylation of <i>APC1A</i>, <i>SFRP1</i>, <i>SFRP2</i>, and <i>WIF1</i> during progression to cancer.</div></p></div><div class="aux"><div class="resc">Mashaal Dhir, et al. J Gastrointest Surg. 2008 October;12(10):1745-1753.</div><p class="links"><a class="dblinks" href="/pubmed/18716850" ref="ordinalpos=1222&amp;ncbi_uid=5995248&amp;link_uid=18716850">Citation</a><a class="dblinks" href="/pmc/articles/PMC3976145/" ref="ordinalpos=1222&amp;ncbi_uid=5995248&amp;link_uid=PMC3976145">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1223.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3908308/figure/fig4/" ref="ordinalpos=1223&amp;ncbi_uid=5963707&amp;link_uid=PMC3908308" href="/pmc/articles/PMC3908308/"><img src="/pmc/articles/instance/3908308/bin/jsrm-09-025-g009.gif" src-large="/pmc/articles/instance/3908308/bin/jsrm-09-025-g009.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3908308/" ref="ordinalpos=1223&amp;ncbi_uid=5963707&amp;link_uid=PMC3908308" image-link="/pmc/articles/PMC3908308/figure/fig4/" class="imagepopup">Figure 4.  From: Regulated expression of CXCR4 constitutive active mutants revealed the up-modulated chemotaxis and up-regulation of genes crucial for CXCR4 mediated homing and engraftment of hematopoietic stem/progenitor cells. </a></p><div class="supp"><p class="details"><div>A schematic figure representing the different groups of up-regulated genes based on their relation to the classical CXCR4 <span class="highlight" style="background-color:">signaling</span> at protein level, which altogether may contribute in likely up modulation of mutant downstream <span class="highlight" style="background-color:">signaling</span>. A. as downstream effectors of CXCR4 <span class="highlight" style="background-color:">signaling</span>, B. as positively associated members of CXCR4 activated downstream pathways (Phosphatidylinositol <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, MAPK <span class="highlight" style="background-color:">pathway</span>), C. as positive regulators of CXCR4 <span class="highlight" style="background-color:">signaling</span>, D. genes whose protein product have established role in cell migration and chemotaxis but so far not reported to play a role in CXCR4 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">M Sharma, et al. J Stem Cells Regen Med. 2013;9(1):19-27.</div><p class="links"><a class="dblinks" href="/pubmed/24693205" ref="ordinalpos=1223&amp;ncbi_uid=5963707&amp;link_uid=24693205">Citation</a><a class="dblinks" href="/pmc/articles/PMC3908308/" ref="ordinalpos=1223&amp;ncbi_uid=5963707&amp;link_uid=PMC3908308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1224.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3774329/figure/F6/" ref="ordinalpos=1224&amp;ncbi_uid=5329279&amp;link_uid=PMC3774329" href="/pmc/articles/PMC3774329/"><img src="/pmc/articles/instance/3774329/bin/zh50111159640006.gif" src-large="/pmc/articles/instance/3774329/bin/zh50111159640006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3774329/" ref="ordinalpos=1224&amp;ncbi_uid=5329279&amp;link_uid=PMC3774329" image-link="/pmc/articles/PMC3774329/figure/F6/" class="imagepopup">Fig. 6. From: Rho-kinase inhibition alleviates pulmonary hypertension in transgenic mice expressing a dominant-negative type II bone morphogenetic protein receptor gene. </a></p><div class="supp"><p class="details"><div>Schematic representation of the interactions between Rho/Rho-kinase <span class="highlight" style="background-color:">signaling</span> and bone morphogenetic protein (BMP)/transforming growth factor (TGF)-β <span class="highlight" style="background-color:">signaling</span>. In SM22-tet-BMPR2<sup>R899X</sup> mice, the levels of GTP-RhoA and MYPT1 phosphorylation were elevated, thus indicating that Rho/Rho-kinase <span class="highlight" style="background-color:">signaling</span> was activated. Two possible pathways, namely the Smad-dependent and Smad-independent pathways, in which BMP/TGF-β <span class="highlight" style="background-color:">signaling</span> activated Rho/Rho-kinase <span class="highlight" style="background-color:">signaling</span> are known. We confirmed that the Smad-dependent <span class="highlight" style="background-color:">pathway</span> was not altered. We therefore speculate that Rho/Rho-kinase <span class="highlight" style="background-color:">signaling</span> was activated via the Smad-independent <span class="highlight" style="background-color:">pathway</span>. Rho-GEF, Rho guanine nucleotide exchange factor; PP1c, catalytic subunit; M20, a 20-kDa subunit of unknown function; P, inorganic phosphate.</div></p></div><div class="aux"><div class="resc">Tadashi Yasuda, et al. Am J Physiol Lung Cell Mol Physiol. 2011 November;301(5):L667-L674.</div><p class="links"><a class="dblinks" href="/pubmed/21856816" ref="ordinalpos=1224&amp;ncbi_uid=5329279&amp;link_uid=21856816">Citation</a><a class="dblinks" href="/pmc/articles/PMC3774329/" ref="ordinalpos=1224&amp;ncbi_uid=5329279&amp;link_uid=PMC3774329">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1225.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4246235/figure/viruses-06-04479-f001/" ref="ordinalpos=1225&amp;ncbi_uid=6893996&amp;link_uid=PMC4246235" href="/pmc/articles/PMC4246235/"><img src="/pmc/articles/instance/4246235/bin/viruses-06-04479-g001.gif" src-large="/pmc/articles/instance/4246235/bin/viruses-06-04479-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4246235/" ref="ordinalpos=1225&amp;ncbi_uid=6893996&amp;link_uid=PMC4246235" image-link="/pmc/articles/PMC4246235/figure/viruses-06-04479-f001/" class="imagepopup">Figure 1.  From: Mosquito Immunity against Arboviruses. </a></p><div class="supp"><p class="details"><div>Mosquito immune <span class="highlight" style="background-color:">signaling</span> and RNAi pathways. In Toll <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>, detection of pathogen-derived ligands by pattern recognition receptors (PRRs) such as PGRP-SA and -SD triggers proteolytic cleavage of the cytokine Späetzle, which binds to and activates the Toll receptor. This triggers <span class="highlight" style="background-color:">signaling</span> through the adaptor proteins MyD88, Tube, and Pelle, resulting in the phosphorylation and degradation of Cactus, a negative regulator which binds to and sequesters the Rel1 transcription factor in the cytoplasm. Cactus degradation allows Rel1 translocation to the nucleus to activate transcription of Toll-<span class="highlight" style="background-color:">pathway</span> regulated genes. The IMD <span class="highlight" style="background-color:">pathway</span> is activated by ligand binding to PGRP-LCs and -LEs. This triggers <span class="highlight" style="background-color:">signaling</span> through IMD and various caspases and kinases, leading to a functional split in the <span class="highlight" style="background-color:">pathway</span>. One branch triggers JNK <span class="highlight" style="background-color:">signaling</span> to activate the transcription factor AP1, while the other results in the phosphorylation of the Rel2 transcription factor and its subsequent DREDD-mediated cleavage. Activated Rel2 translocates to the nucleus to activate IMD-regulated transcription. The JAK-STAT <span class="highlight" style="background-color:">pathway</span> is triggered by Unpaired (Upd) binding to the receptor Dome, activating the receptor-associated Hop Janus kinases, which phosphorylate each other and subsequently recruit and phosphorylate the STAT transcription factor. Phosphorylated STATs dimerize and translocate to the nucleus to activate JAK-STAT-regulated transcription. The exogenous siRNA <span class="highlight" style="background-color:">pathway</span> is activated when virus-derived long dsRNA is recognized and cleaved by Dcr2 into siRNAs, usually 21 bp in length. siRNAs are loaded onto the multi-protein RISC complex, which degrades one strand of the duplex and uses the other for targeted degradation of complementary single stranded viral RNA. Sensing of viral dsRNA by Dcr2 also activates TRAF, leading to Rel2 cleavage and activation via a distinct <span class="highlight" style="background-color:">pathway</span>. Rel2 activates transcription of Vago, a secreted peptide which subsequently triggers JAK-STAT <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. Please refer to the text for more details.</div></p></div><div class="aux"><div class="resc">Shuzhen Sim, et al. Viruses. 2014 November;6(11):4479-4504.</div><p class="links"><a class="dblinks" href="/pubmed/25415198" ref="ordinalpos=1225&amp;ncbi_uid=6893996&amp;link_uid=25415198">Citation</a><a class="dblinks" href="/pmc/articles/PMC4246235/" ref="ordinalpos=1225&amp;ncbi_uid=6893996&amp;link_uid=PMC4246235">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1226.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4290619/figure/F7/" ref="ordinalpos=1226&amp;ncbi_uid=7065966&amp;link_uid=PMC4290619" href="/pmc/articles/PMC4290619/"><img src="/pmc/articles/instance/4290619/bin/1471-2164-15-S9-S6-7.gif" src-large="/pmc/articles/instance/4290619/bin/1471-2164-15-S9-S6-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4290619/" ref="ordinalpos=1226&amp;ncbi_uid=7065966&amp;link_uid=PMC4290619" image-link="/pmc/articles/PMC4290619/figure/F7/" class="imagepopup">Figure 7.  From: Transcriptome alterations of mitochondrial and coagulation function in schizophrenia by cortical sequencing analysis. </a></p><div class="supp"><p class="details"><div><b>Under-expressed enrichment <span class="highlight" style="background-color:">pathway</span> network for schizophrenia</b>. These proteins involved in three <span class="highlight" style="background-color:">signaling</span> pathways including BARD1 <span class="highlight" style="background-color:">signaling</span> events, LKB1 <span class="highlight" style="background-color:">signaling</span> events and Aurora A <span class="highlight" style="background-color:">signaling</span> reveals the under-expressed enrichment pathways which may implicate key roles in pathogenesis of schizophrenia.</div></p></div><div class="aux"><div class="resc">Kuo-Chuan Huang, et al. BMC Genomics. 2014;15(Suppl 9):S6-S6.</div><p class="links"><a class="dblinks" href="/pubmed/25522158" ref="ordinalpos=1226&amp;ncbi_uid=7065966&amp;link_uid=25522158">Citation</a><a class="dblinks" href="/pmc/articles/PMC4290619/" ref="ordinalpos=1226&amp;ncbi_uid=7065966&amp;link_uid=PMC4290619">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1227.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3408231/figure/F7/" ref="ordinalpos=1227&amp;ncbi_uid=4748979&amp;link_uid=PMC3408231" href="/pmc/articles/PMC3408231/"><img src="/pmc/articles/instance/3408231/bin/nihms362522f7.gif" src-large="/pmc/articles/instance/3408231/bin/nihms362522f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3408231/" ref="ordinalpos=1227&amp;ncbi_uid=4748979&amp;link_uid=PMC3408231" image-link="/pmc/articles/PMC3408231/figure/F7/" class="imagepopup">Figure 7. The HIF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in osteoblasts directly modulates erythropoiesis through the production of EPO and the number of HSCs.  From: The HIF <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Osteoblasts Directly Modulates Erythropoiesis Through the Production of EPO. </a></p><div class="supp"><p class="details"><div>Osteoblasts produce EPO through a HIF dependent mechanism under physiologic and pathophysiologic conditions. Osteoblasts express HIF-1 and HIF-2 that transcriptionally activate EPO expression. In conditions of augmented HIF <span class="highlight" style="background-color:">signaling</span> (dashed line), osteoblasts produce both local and systemic levels of EPO that stimulate erythropoiesis in the bone marrow and spleen. Constitutive hyperactivation of HIF <span class="highlight" style="background-color:">signaling</span> leads to increased red blood cells (RBC) and oxygen in the blood that feeds back to suppress renal EPO production. Augmented HIF <span class="highlight" style="background-color:">signaling</span> in osteoblasts through VHL or PHD1/2/3 inhibition is sufficient to drive EPO expression in adult bone, elevate hematocrit, and protect mice from stress-induced anemia. Furthermore, pharmacologic inhibition of PHDs using prolyl hydroxylase inhibitors (PHI) is sufficient to activate EPO in adult bone. Additionally, augmented HIF activity in osteoblasts expands HSCs indicating that therapeutic manipulation of the PHD/VHL/HIF <span class="highlight" style="background-color:">pathway</span> in osteoblasts is a therapeutic strategy to elevate both EPO and HSCs in the local hematopoietic microenvironment.</div></p></div><div class="aux"><div class="resc">Erinn B. Rankin, et al. Cell. ;149(1):63-74.</div><p class="links"><a class="dblinks" href="/pubmed/22464323" ref="ordinalpos=1227&amp;ncbi_uid=4748979&amp;link_uid=22464323">Citation</a><a class="dblinks" href="/pmc/articles/PMC3408231/" ref="ordinalpos=1227&amp;ncbi_uid=4748979&amp;link_uid=PMC3408231">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1228.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2661875/figure/F4/" ref="ordinalpos=1228&amp;ncbi_uid=1751005&amp;link_uid=PMC2661875" href="/pmc/articles/PMC2661875/"><img src="/pmc/articles/instance/2661875/bin/nihms92737f4.gif" src-large="/pmc/articles/instance/2661875/bin/nihms92737f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2661875/" ref="ordinalpos=1228&amp;ncbi_uid=1751005&amp;link_uid=PMC2661875" image-link="/pmc/articles/PMC2661875/figure/F4/" class="imagepopup">Figure 4. Controlling circuit behavior.  From: Designing new cellular <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>A. Creation of an irreversible, self-perpetuating <span class="highlight" style="background-color:">signaling</span> circuit in a MAP kinase cascade by placing a constitutively-active form of one <span class="highlight" style="background-color:">pathway</span> component (KKK*) under transcriptional control of the <span class="highlight" style="background-color:">pathway</span> (Ingolia and Murray, 2007).<br />B. A modified scaffold alters response dynamics in a MAP kinase cascade. Using leucine zippers to recruit new <span class="highlight" style="background-color:">pathway</span> regulators to the scaffold, <span class="highlight" style="background-color:">signaling</span> could be enhanced or dampened. Then, by placing expression of the recruited regulator under transcriptional control of the <span class="highlight" style="background-color:">pathway</span> itself, positive or negative feedback loops were established (Bashor et al., 2008). Several types of feedback circuit were constructed. Here, a <span class="highlight" style="background-color:">pathway</span>-induced positive regulator displaces a pre-existing negative regulator, creating a response that is switch-like rather than graded.</div></p></div><div class="aux"><div class="resc">Peter M. Pryciak. Chem Biol. ;16(3):249-254.</div><p class="links"><a class="dblinks" href="/pubmed/19318206" ref="ordinalpos=1228&amp;ncbi_uid=1751005&amp;link_uid=19318206">Citation</a><a class="dblinks" href="/pmc/articles/PMC2661875/" ref="ordinalpos=1228&amp;ncbi_uid=1751005&amp;link_uid=PMC2661875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1229.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3506052/figure/i1552-5783-53-12-7746-f07/" ref="ordinalpos=1229&amp;ncbi_uid=4845701&amp;link_uid=PMC3506052" href="/pmc/articles/PMC3506052/"><img src="/pmc/articles/instance/3506052/bin/i1552-5783-53-12-7746-f07.gif" src-large="/pmc/articles/instance/3506052/bin/i1552-5783-53-12-7746-f07.jpg" alt=" Figure 7.  " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3506052/" ref="ordinalpos=1229&amp;ncbi_uid=4845701&amp;link_uid=PMC3506052" image-link="/pmc/articles/PMC3506052/figure/i1552-5783-53-12-7746-f07/" class="imagepopup"> Figure 7.  .  From: Interleukin-6 Production in CD40-Engaged Fibrocytes in Thyroid-Associated Ophthalmopathy: Involvement of Akt and NF-?B. </a></p><div class="supp"><p class="details"><div>Schematic view of CD40 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of IL-6 production in fibrocytes. CD40–CD40L interaction activates a downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, resulting in the production of proinflammatory cytokine, IL-6. CD40 <span class="highlight" style="background-color:">signaling</span> for IL-6 is mediated by Akt phosphorylation and subsequent NF-κB translocation.</div></p></div><div class="aux"><div class="resc">Erin F. Gillespie, et al. Invest Ophthalmol Vis Sci. 2012 November;53(12):7746-7753.</div><p class="links"><a class="dblinks" href="/pubmed/23092922" ref="ordinalpos=1229&amp;ncbi_uid=4845701&amp;link_uid=23092922">Citation</a><a class="dblinks" href="/pmc/articles/PMC3506052/" ref="ordinalpos=1229&amp;ncbi_uid=4845701&amp;link_uid=PMC3506052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1230.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3080193/figure/F2/" ref="ordinalpos=1230&amp;ncbi_uid=2903399&amp;link_uid=PMC3080193" href="/pmc/articles/PMC3080193/"><img src="/pmc/articles/instance/3080193/bin/viru0101_0045_fig002.gif" src-large="/pmc/articles/instance/3080193/bin/viru0101_0045_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3080193/" ref="ordinalpos=1230&amp;ncbi_uid=2903399&amp;link_uid=PMC3080193" image-link="/pmc/articles/PMC3080193/figure/F2/" class="imagepopup">Figure 2.  From: Coupling temperature sensing and development. </a></p><div class="supp"><p class="details"><div>Models for Hsp90's interaction with the PKA <span class="highlight" style="background-color:">pathway</span>. (A) Hsp90 could interact with a positive regulator of the cAMP-PKA <span class="highlight" style="background-color:">pathway</span> and maintain it in an inactive, but active competent state. This could involve Hsp90 interacting the positive catalytic subunits of PKA, Tpk1 or Tpk2. (B) Hsp90 could stabilize a negative regulator of the cAMP-PKA <span class="highlight" style="background-color:">pathway</span>, such that inhibiting Hsp90 would destabilize the negative regulator. This could involve Hsp90 interacting the negative regulatory subunit of PKA, Bcy1. (C) Hsp90 might interact indirectly with the cAMP-PKA <span class="highlight" style="background-color:">pathway</span>. This could occur via a positive regulator such as Tpk1 or Tpk2, or by a negative regulator. More complex models are possible, where Hsp90 could regulate cAMP-PKA <span class="highlight" style="background-color:">signaling</span> via multiple client proteins.</div></p></div><div class="aux"><div class="resc">Rebecca S Shapiro, et al. Virulence. 2010 Jan-Feb;1(1):45-48.</div><p class="links"><a class="dblinks" href="/pubmed/21178413" ref="ordinalpos=1230&amp;ncbi_uid=2903399&amp;link_uid=21178413">Citation</a><a class="dblinks" href="/pmc/articles/PMC3080193/" ref="ordinalpos=1230&amp;ncbi_uid=2903399&amp;link_uid=PMC3080193">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1231.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3320948/figure/F6/" ref="ordinalpos=1231&amp;ncbi_uid=4782110&amp;link_uid=PMC3320948" href="/pmc/articles/PMC3320948/"><img src="/pmc/articles/instance/3320948/bin/zbc0161203810006.gif" src-large="/pmc/articles/instance/3320948/bin/zbc0161203810006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3320948/" ref="ordinalpos=1231&amp;ncbi_uid=4782110&amp;link_uid=PMC3320948" image-link="/pmc/articles/PMC3320948/figure/F6/" class="imagepopup">FIGURE 6. From: Cross-talk between Insulin and Wnt <span class="highlight" style="background-color:">Signaling</span> in Preadipocytes. </a></p><div class="supp"><p class="details"><div><b>Insulin and IGF-1 <span class="highlight" style="background-color:">signaling</span> in LRP5 and LRP6 KD preadipocytes.</b> 3T3-L1 preadipocytes were stimulated with 100 n<span style="font-variant: small-caps" class="small-caps">m</span> insulin or IGF-1 at various time points for 60 min. Total cell extracts were made and used directly for Western blot analysis. <i>A</i>, the insulin phosphorylation response of proteins in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>n</i> = 3). <i>B</i>, the IGF-1 phosphorylation response of proteins in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>n</i> = 3). Total protein levels for the various insulin time points can be seen in supplemental Fig. S4. <i>Con</i>, control; <i>ut</i>, untreated.</div></p></div><div class="aux"><div class="resc">Jane Palsgaard, et al. J Biol Chem. 2012 April 6;287(15):12016-12026.</div><p class="links"><a class="dblinks" href="/pubmed/22337886" ref="ordinalpos=1231&amp;ncbi_uid=4782110&amp;link_uid=22337886">Citation</a><a class="dblinks" href="/pmc/articles/PMC3320948/" ref="ordinalpos=1231&amp;ncbi_uid=4782110&amp;link_uid=PMC3320948">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1232.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3347452/figure/F7/" ref="ordinalpos=1232&amp;ncbi_uid=4333767&amp;link_uid=PMC3347452" href="/pmc/articles/PMC3347452/"><img src="/pmc/articles/instance/3347452/bin/zii9990996330007.gif" src-large="/pmc/articles/instance/3347452/bin/zii9990996330007.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3347452/" ref="ordinalpos=1232&amp;ncbi_uid=4333767&amp;link_uid=PMC3347452" image-link="/pmc/articles/PMC3347452/figure/F7/" class="imagepopup">Fig 7.  From: Regulation of Subtilase Cytotoxin-Induced Cell Death by an RNA-Dependent Protein Kinase-Like Endoplasmic Reticulum Kinase-Dependent Proteasome <span class="highlight" style="background-color:">Pathway</span> in HeLa Cells. </a></p><div class="supp"><p class="details"><div>Proposed model of SubAB-induced PERK-mediated apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HeLa cells. See the text for additional details.</div></p></div><div class="aux"><div class="resc">Kinnosuke Yahiro, et al. Infect Immun. 2012 May;80(5):1803-1814.</div><p class="links"><a class="dblinks" href="/pubmed/22354021" ref="ordinalpos=1232&amp;ncbi_uid=4333767&amp;link_uid=22354021">Citation</a><a class="dblinks" href="/pmc/articles/PMC3347452/" ref="ordinalpos=1232&amp;ncbi_uid=4333767&amp;link_uid=PMC3347452">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1233.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3520613/figure/F5/" ref="ordinalpos=1233&amp;ncbi_uid=4443974&amp;link_uid=PMC3520613" href="/pmc/articles/PMC3520613/"><img src="/pmc/articles/instance/3520613/bin/nihms425617f5.gif" src-large="/pmc/articles/instance/3520613/bin/nihms425617f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3520613/" ref="ordinalpos=1233&amp;ncbi_uid=4443974&amp;link_uid=PMC3520613" image-link="/pmc/articles/PMC3520613/figure/F5/" class="imagepopup">Figure 5. Reln-VLDLR-Dab2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in macrophages regulates Pla2g7 expression.  From: Macrophage VLDL Receptor Promotes PAFAH Secretion in Mother's Milk and Suppresses Systemic Inflammation in Nursing Neonates. </a></p><div class="supp"><p class="details"><div>According to our working model, activation of macrophage VLDLR by ligands such as Reln triggers Dab2-mediated <span class="highlight" style="background-color:">signaling</span> transduction to promote <i>Pla2g7</i> expression.</div></p></div><div class="aux"><div class="resc">Yang Du, et al. Nat Commun. ;3:1008-1008.</div><p class="links"><a class="dblinks" href="/pubmed/22910354" ref="ordinalpos=1233&amp;ncbi_uid=4443974&amp;link_uid=22910354">Citation</a><a class="dblinks" href="/pmc/articles/PMC3520613/" ref="ordinalpos=1233&amp;ncbi_uid=4443974&amp;link_uid=PMC3520613">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1234.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3357464/figure/F3/" ref="ordinalpos=1234&amp;ncbi_uid=4335363&amp;link_uid=PMC3357464" href="/pmc/articles/PMC3357464/"><img src="/pmc/articles/instance/3357464/bin/nihms-373061-f0003.gif" src-large="/pmc/articles/instance/3357464/bin/nihms-373061-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3357464/" ref="ordinalpos=1234&amp;ncbi_uid=4335363&amp;link_uid=PMC3357464" image-link="/pmc/articles/PMC3357464/figure/F3/" class="imagepopup">Fig. 3. Scaffolding to achieve <span class="highlight" style="background-color:">signaling</span> specificity.  From: How do pleiotropic kinase hubs mediate specific <span class="highlight" style="background-color:">signaling</span> by TNFR superfamily members?. </a></p><div class="supp"><p class="details"><div>Scaffolds can organize pleiotropic kinases into stimulus-specific <span class="highlight" style="background-color:">signaling</span> modules to direct them to one specific <span class="highlight" style="background-color:">pathway</span>. (A) Filamin interacts with TRAF2, MEKK4, and JNK to allow for JNK activation upon TNFR1 engagement. ERK1/2 and p38 might similarly be organized into stimulus-specific modules through yet unidentified scaffolding proteins. (B) Binding of NEMO to IKK2 allows for its activation but is also required for proper <span class="highlight" style="background-color:">signaling</span> propagation to IκBα in response to inflammatory signals. (C) NIK is not only the activating kinase of IKK1 but also channels IKK1 activity to p100.</div></p></div><div class="aux"><div class="resc">Bärbel Schröfelbauer, et al. Immunol Rev. ;244(1):29-43.</div><p class="links"><a class="dblinks" href="/pubmed/22017429" ref="ordinalpos=1234&amp;ncbi_uid=4335363&amp;link_uid=22017429">Citation</a><a class="dblinks" href="/pmc/articles/PMC3357464/" ref="ordinalpos=1234&amp;ncbi_uid=4335363&amp;link_uid=PMC3357464">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1235.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3091628/figure/F3/" ref="ordinalpos=1235&amp;ncbi_uid=2943140&amp;link_uid=PMC3091628" href="/pmc/articles/PMC3091628/"><img src="/pmc/articles/instance/3091628/bin/1471-2164-11-380-3.gif" src-large="/pmc/articles/instance/3091628/bin/1471-2164-11-380-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3091628/" ref="ordinalpos=1235&amp;ncbi_uid=2943140&amp;link_uid=PMC3091628" image-link="/pmc/articles/PMC3091628/figure/F3/" class="imagepopup">Figure 3.  From: Response of the mosquito protein interaction network to dengue infection. </a></p><div class="supp"><p class="details"><div><b>Local views of mosquito pathways or biological processes</b>. Genes related to Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, JAK/STAT <span class="highlight" style="background-color:">pathway</span>, Insulin <span class="highlight" style="background-color:">pathway</span>, sex determination <span class="highlight" style="background-color:">pathway</span> and mosquito olfaction are selected to identify their interaction partners in the network.</div></p></div><div class="aux"><div class="resc">Xiang Guo, et al. BMC Genomics. 2010;11:380-380.</div><p class="links"><a class="dblinks" href="/pubmed/20553610" ref="ordinalpos=1235&amp;ncbi_uid=2943140&amp;link_uid=20553610">Citation</a><a class="dblinks" href="/pmc/articles/PMC3091628/" ref="ordinalpos=1235&amp;ncbi_uid=2943140&amp;link_uid=PMC3091628">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1236.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3099707/figure/F10/" ref="ordinalpos=1236&amp;ncbi_uid=3364177&amp;link_uid=PMC3099707" href="/pmc/articles/PMC3099707/"><img src="/pmc/articles/instance/3099707/bin/zbc0271164280010.gif" src-large="/pmc/articles/instance/3099707/bin/zbc0271164280010.jpg" alt="FIGURE 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3099707/" ref="ordinalpos=1236&amp;ncbi_uid=3364177&amp;link_uid=PMC3099707" image-link="/pmc/articles/PMC3099707/figure/F10/" class="imagepopup">FIGURE 10. From: Mahoganoid and Mahogany Mutations Rectify the Obesity of the Yellow Mouse by Effects on Endosomal Traffic of MC4R Protein. </a></p><div class="supp"><p class="details"><div><b>Dual MC4R <span class="highlight" style="background-color:">signaling</span> model to account for “rescue” of <i>A<sup>y</sup></i> phenotype by alterations in MC4R trafficking.</b> <i>A</i>, MC4R is shown with constitutive cAMP-dependent and hypothetical cAMP-independent activity (possibly the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by β-defensins and represented in the figure by pERK). <i>B</i>, α-MSH up-regulates both the cAMP-dependent and -independent <span class="highlight" style="background-color:">signaling</span> pathways (<i>step 1</i>) (<i>arrow/text size</i> indicates direction of effect). cAMP is generated only while the MC4R is on the plasma membrane; pERK remains up-regulated following receptor internalization (<i>step 2</i>) until ligand and receptor dissociate along the endocytic recycling <span class="highlight" style="background-color:">pathway</span> (<i>step 3</i>). Upon α-MSH withdrawal, cAMP and pERK return to basal levels (<i>step 4</i>). <i>C</i>, ASP antagonizes cAMP <span class="highlight" style="background-color:">signaling</span> through its C terminus (<i>step 1</i>) and pERK <span class="highlight" style="background-color:">signaling</span> through its N terminus (<i>step 2</i>) (requiring MGRN1 and ATRN). pERK is only completely attenuated following ASP-dependent MC4R degradation in the lysosome (<i>step 3</i>). Loss-of-function mutations in ATRN or MGRN1 impair the ASP-dependent lysosomal trafficking of MC4R, preventing the full reduction of pERK <span class="highlight" style="background-color:">signaling</span> and allowing basal cAMP-independent activity to be maintained. The hypothetical pERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is analogous to that of the activated EGF receptor; ERK <span class="highlight" style="background-color:">signaling</span> initiated at the cell surface continues from within the cell until the EGF receptor enters the lysosome.</div></p></div><div class="aux"><div class="resc">John D. Overton, et al. J Biol Chem. 2011 May 27;286(21):18914-18929.</div><p class="links"><a class="dblinks" href="/pubmed/21460229" ref="ordinalpos=1236&amp;ncbi_uid=3364177&amp;link_uid=21460229">Citation</a><a class="dblinks" href="/pmc/articles/PMC3099707/" ref="ordinalpos=1236&amp;ncbi_uid=3364177&amp;link_uid=PMC3099707">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1237.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3639415/figure/F8/" ref="ordinalpos=1237&amp;ncbi_uid=5353507&amp;link_uid=PMC3639415" href="/pmc/articles/PMC3639415/"><img src="/pmc/articles/instance/3639415/bin/734fig8.gif" src-large="/pmc/articles/instance/3639415/bin/734fig8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3639415/" ref="ordinalpos=1237&amp;ncbi_uid=5353507&amp;link_uid=PMC3639415" image-link="/pmc/articles/PMC3639415/figure/F8/" class="imagepopup">Figure 8. From: Amplitude modulation of androgen <span class="highlight" style="background-color:">signaling</span> by c-MYC. </a></p><div class="supp"><p class="details"><div>A model of the regulation and function of MYC in the androgen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in molecular apocrine breast cancers.</div></p></div><div class="aux"><div class="resc">Min Ni, et al. Genes Dev. 2013 April 1;27(7):734-748.</div><p class="links"><a class="dblinks" href="/pubmed/23530127" ref="ordinalpos=1237&amp;ncbi_uid=5353507&amp;link_uid=23530127">Citation</a><a class="dblinks" href="/pmc/articles/PMC3639415/" ref="ordinalpos=1237&amp;ncbi_uid=5353507&amp;link_uid=PMC3639415">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1238.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3444437/figure/F9/" ref="ordinalpos=1238&amp;ncbi_uid=4240500&amp;link_uid=PMC3444437" href="/pmc/articles/PMC3444437/"><img src="/pmc/articles/instance/3444437/bin/1477-7827-10-58-9.gif" src-large="/pmc/articles/instance/3444437/bin/1477-7827-10-58-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3444437/" ref="ordinalpos=1238&amp;ncbi_uid=4240500&amp;link_uid=PMC3444437" image-link="/pmc/articles/PMC3444437/figure/F9/" class="imagepopup">Figure 9.  From: The proto-oncogene c-src is involved in primordial follicle activation through the PI3K, PKC and MAPK <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b><i>C-src</i></b><b>regulating primordial follicle activation and growth through PI3K → PKC → MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b></div></p></div><div class="aux"><div class="resc">Xiao-Yu Du, et al. Reprod Biol Endocrinol. 2012;10:58-58.</div><p class="links"><a class="dblinks" href="/pubmed/22905678" ref="ordinalpos=1238&amp;ncbi_uid=4240500&amp;link_uid=22905678">Citation</a><a class="dblinks" href="/pmc/articles/PMC3444437/" ref="ordinalpos=1238&amp;ncbi_uid=4240500&amp;link_uid=PMC3444437">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1239.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3364461/figure/F4/" ref="ordinalpos=1239&amp;ncbi_uid=4043514&amp;link_uid=PMC3364461" href="/pmc/articles/PMC3364461/"><img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-g004.gif" src-large="/pmc/articles/instance/3364461/bin/fendo-02-00045-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3364461/" ref="ordinalpos=1239&amp;ncbi_uid=4043514&amp;link_uid=PMC3364461" image-link="/pmc/articles/PMC3364461/figure/F4/" class="imagepopup">Figure 4.  From: Mapping the Follicle-Stimulating Hormone-Induced <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div><b>PI3K/mTOR <span class="highlight" style="background-color:">signaling</span> at the FSHR</b>. The PI3K/mTOR <span class="highlight" style="background-color:">pathway</span> plays an important role in FSH-induced actions, including proliferation, regulation of gene transcription as well as of protein translation.</div></p></div><div class="aux"><div class="resc">Pauline Gloaguen, et al. Front Endocrinol (Lausanne). 2011;2:45.</div><p class="links"><a class="dblinks" href="/pubmed/22666216" ref="ordinalpos=1239&amp;ncbi_uid=4043514&amp;link_uid=22666216">Citation</a><a class="dblinks" href="/pmc/articles/PMC3364461/" ref="ordinalpos=1239&amp;ncbi_uid=4043514&amp;link_uid=PMC3364461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1240.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2776759/figure/F4/" ref="ordinalpos=1240&amp;ncbi_uid=2110704&amp;link_uid=PMC2776759" href="/pmc/articles/PMC2776759/"><img src="/pmc/articles/instance/2776759/bin/nihms-156819-f0004.gif" src-large="/pmc/articles/instance/2776759/bin/nihms-156819-f0004.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2776759/" ref="ordinalpos=1240&amp;ncbi_uid=2110704&amp;link_uid=PMC2776759" image-link="/pmc/articles/PMC2776759/figure/F4/" class="imagepopup">FIGURE 4.  From: Systemic Analysis of Tumor Cell-Induced Endothelial Calcium <span class="highlight" style="background-color:">Signaling</span> and Junction Disassembly. </a></p><div class="supp"><p class="details"><div>Schematic diagram of calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> initiated by melanoma contact to endothelium. Arrows indicate the direct signals and dashed arrows indicate tentative signals.</div></p></div><div class="aux"><div class="resc">Hsin-Hsin Peng, et al. Cell Mol Bioeng. ;2(3):375-385.</div><p class="links"><a class="dblinks" href="/pubmed/19915693" ref="ordinalpos=1240&amp;ncbi_uid=2110704&amp;link_uid=19915693">Citation</a><a class="dblinks" href="/pmc/articles/PMC2776759/" ref="ordinalpos=1240&amp;ncbi_uid=2110704&amp;link_uid=PMC2776759">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1241.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3650001/figure/pgen-1003510-g009/" ref="ordinalpos=1241&amp;ncbi_uid=4891184&amp;link_uid=PMC3650001" href="/pmc/articles/PMC3650001/"><img src="/pmc/articles/instance/3650001/bin/pgen.1003510.g009.gif" src-large="/pmc/articles/instance/3650001/bin/pgen.1003510.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3650001/" ref="ordinalpos=1241&amp;ncbi_uid=4891184&amp;link_uid=PMC3650001" image-link="/pmc/articles/PMC3650001/figure/pgen-1003510-g009/" class="imagepopup">Figure 9. Proposed genetic control of calcium <span class="highlight" style="background-color:">signaling</span> in the spermatheca..  From: Filamin and Phospholipase C-? Are Required for Calcium <span class="highlight" style="background-color:">Signaling</span> in the Caenorhabditis elegans Spermatheca. </a></p><div class="supp"><p class="details"><div>The diagram represents a working model of the spermathecal calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Green lines indicate interactions that promote cell contraction, and magenta lines indicate those which inhibit contraction. Dashed lines indicate possible regulatory interactions. We propose FLN-1 acts upstream or in parallel to PLC-1 to initiate calcium release via ITR-1. FLN-1 also plays a downstream role in contraction and maintenance of the F-actin cytoskeleton.</div></p></div><div class="aux"><div class="resc">Ismar Kovacevic, et al. PLoS Genet. 2013 May;9(5):e1003510.</div><p class="links"><a class="dblinks" href="/pubmed/23671426" ref="ordinalpos=1241&amp;ncbi_uid=4891184&amp;link_uid=23671426">Citation</a><a class="dblinks" href="/pmc/articles/PMC3650001/" ref="ordinalpos=1241&amp;ncbi_uid=4891184&amp;link_uid=PMC3650001">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1242.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1998879/figure/Fig2/" ref="ordinalpos=1242&amp;ncbi_uid=1071847&amp;link_uid=PMC1998879" href="/pmc/articles/PMC1998879/"><img src="/pmc/articles/instance/1998879/bin/335_2007_9034_Fig2_HTML.gif" src-large="/pmc/articles/instance/1998879/bin/335_2007_9034_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1998879/" ref="ordinalpos=1242&amp;ncbi_uid=1071847&amp;link_uid=PMC1998879" image-link="/pmc/articles/PMC1998879/figure/Fig2/" class="imagepopup">Fig. 2.  From: Neuronal migration: unraveling the molecular <span class="highlight" style="background-color:">pathway</span> with humans, mice, and a fungus. </a></p><div class="supp"><p class="details"><div>Molecular <span class="highlight" style="background-color:">pathway</span> associated with neuronal migration. This diagram shows an outline of the molecular <span class="highlight" style="background-color:">pathway</span> required for neuronal migration. Extracellular reelin binds to the membrane-bound receptors APOER2 and VLDLR, which stimulate the intracellular <span class="highlight" style="background-color:">signaling</span> molecule DAB1. Phosphorylated DAB1 interacts with LIS1. CDK5, thought to act via a parallel but intersecting <span class="highlight" style="background-color:">pathway</span>, is activated by P35 and phosphorylates the microtubule stabilizer DCX and NDEL1. NDEL1 complexes with LIS1 and dynein, which act to sustain microtubule bundles and facilitate nuclear migration</div></p></div><div class="aux"><div class="resc">David A. Keays. Mamm Genome. 2007 July;18(6-7):425-430.</div><p class="links"><a class="dblinks" href="/pubmed/17629745" ref="ordinalpos=1242&amp;ncbi_uid=1071847&amp;link_uid=17629745">Citation</a><a class="dblinks" href="/pmc/articles/PMC1998879/" ref="ordinalpos=1242&amp;ncbi_uid=1071847&amp;link_uid=PMC1998879">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1243.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3140648/figure/F3/" ref="ordinalpos=1243&amp;ncbi_uid=3141542&amp;link_uid=PMC3140648" href="/pmc/articles/PMC3140648/"><img src="/pmc/articles/instance/3140648/bin/fnana-05-00043-g003.gif" src-large="/pmc/articles/instance/3140648/bin/fnana-05-00043-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3140648/" ref="ordinalpos=1243&amp;ncbi_uid=3141542&amp;link_uid=PMC3140648" image-link="/pmc/articles/PMC3140648/figure/F3/" class="imagepopup">Figure 3.  From: Mechanisms for the Modulation of Dopamine D1 Receptor <span class="highlight" style="background-color:">Signaling</span> in Striatal Neurons. </a></p><div class="supp"><p class="details"><div><b>Roles of phosphodiesterases (PDEs) in the control of basal ganglia–thalamocortical circuitry</b>. Output neurons in the striatum are medium spiny neurons (MSNs), which consist of striatonigral/direct <span class="highlight" style="background-color:">pathway</span> and striatopallidal/indirect <span class="highlight" style="background-color:">pathway</span> neurons. Direct <span class="highlight" style="background-color:">pathway</span> neurons are GABAergic, and inhibit tonically active neurons in globus pallidus interna (GPi)/substantia nigra pars reticulata (SNpr). Indirect <span class="highlight" style="background-color:">pathway</span> neurons are also GABAergic, and activate neurons in GPi/SNpr via inhibition of globus pallidus externa (GPe) GABAergic neurons and activation of subthalamic nucleus (STN) glutamatergic neurons. Direct and indirect <span class="highlight" style="background-color:">pathway</span> neurons induce opposing effects on the output neurons in GPi/SNpr, resulting in dis-inhibition and pro-inhibition of output, respectively, to motor areas of the thalamus and cortex. The inhibition of PDEs increases cAMP/PKA <span class="highlight" style="background-color:">signaling</span> in both direct and indirect <span class="highlight" style="background-color:">pathway</span> neurons. PDE inhibitors that predominantly act in direct <span class="highlight" style="background-color:">pathway</span> neurons work like dopamine D<sub>1</sub> receptor agonists and activate motor function, whereas PDE inhibitors that predominantly act in indirect <span class="highlight" style="background-color:">pathway</span> neurons work like dopamine D<sub>2</sub> receptor antagonists and inhibit motor function. SNpc, substantia nigra pars compacta. Reproduced with permission from reference Nishi and Snyder (2010).</div></p></div><div class="aux"><div class="resc">Akinori Nishi, et al. Front Neuroanat. 2011;5:43.</div><p class="links"><a class="dblinks" href="/pubmed/21811441" ref="ordinalpos=1243&amp;ncbi_uid=3141542&amp;link_uid=21811441">Citation</a><a class="dblinks" href="/pmc/articles/PMC3140648/" ref="ordinalpos=1243&amp;ncbi_uid=3141542&amp;link_uid=PMC3140648">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1244.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3093464/figure/AS-fig01/" ref="ordinalpos=1244&amp;ncbi_uid=3349894&amp;link_uid=PMC3093464" href="/pmc/articles/PMC3093464/"><img src="/pmc/articles/instance/3093464/bin/pnas.1100426S108fig01.gif" src-large="/pmc/articles/instance/3093464/bin/pnas.1100426S108fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3093464/" ref="ordinalpos=1244&amp;ncbi_uid=3349894&amp;link_uid=PMC3093464" image-link="/pmc/articles/PMC3093464/figure/AS-fig01/" class="imagepopup">Fig. 1. From: Regeneration of axons in injured spinal cord by activation of bone morphogenetic protein/Smad1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in adult neurons. </a></p><div class="supp"><p class="details"><div>BMP/Smad1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates the axon growth potential. As DRG neurons mature, there is an age-dependent decline of the axon growth potential and a concurrent down-regulation of the BMP/Smad1 <span class="highlight" style="background-color:">signaling</span>. A peripheral lesion of adult DRG neurons rekindles the innate growth potential, whereas a central branch lesion in SCI does not lead to regeneration. Reactivating Smad1 selectively in adult DRG neurons either before or after SCI enhances the growth potential, thereby promoting axonal regeneration. <i>y</i> axis: relative neuronal regeneration capacity.</div></p></div><div class="aux"><div class="resc">Pranav Parikh, et al. Proc Natl Acad Sci U S A. 2011 May 10;108(19):E99-E107.</div><p class="links"><a class="dblinks" href="/pubmed/21518886" ref="ordinalpos=1244&amp;ncbi_uid=3349894&amp;link_uid=21518886">Citation</a><a class="dblinks" href="/pmc/articles/PMC3093464/" ref="ordinalpos=1244&amp;ncbi_uid=3349894&amp;link_uid=PMC3093464">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1245.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4092152/figure/ppat-1004253-g008/" ref="ordinalpos=1245&amp;ncbi_uid=6394764&amp;link_uid=PMC4092152" href="/pmc/articles/PMC4092152/"><img src="/pmc/articles/instance/4092152/bin/ppat.1004253.g008.gif" src-large="/pmc/articles/instance/4092152/bin/ppat.1004253.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4092152/" ref="ordinalpos=1245&amp;ncbi_uid=6394764&amp;link_uid=PMC4092152" image-link="/pmc/articles/PMC4092152/figure/ppat-1004253-g008/" class="imagepopup">Figure 8. A schematic working model..  From: Oncogenic Herpesvirus KSHV Hijacks BMP-Smad1-Id <span class="highlight" style="background-color:">Signaling</span> to Promote Tumorigenesis. </a></p><div class="supp"><p class="details"><div>Small inhibitors targeting BMP-Smad1-Id <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through different mechanisms may serve as potential therapeutics of KS by targeting Id expression.</div></p></div><div class="aux"><div class="resc">Deguang Liang, et al. PLoS Pathog. 2014 July;10(7):e1004253.</div><p class="links"><a class="dblinks" href="/pubmed/25010525" ref="ordinalpos=1245&amp;ncbi_uid=6394764&amp;link_uid=25010525">Citation</a><a class="dblinks" href="/pmc/articles/PMC4092152/" ref="ordinalpos=1245&amp;ncbi_uid=6394764&amp;link_uid=PMC4092152">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1246.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1592599/figure/F1/" ref="ordinalpos=1246&amp;ncbi_uid=844560&amp;link_uid=PMC1592599" href="/pmc/articles/PMC1592599/"><img src="/pmc/articles/instance/1592599/bin/MI2006-12682.001.gif" src-large="/pmc/articles/instance/1592599/bin/MI2006-12682.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1592599/" ref="ordinalpos=1246&amp;ncbi_uid=844560&amp;link_uid=PMC1592599" image-link="/pmc/articles/PMC1592599/figure/F1/" class="imagepopup">Figure 1.  From: Expression of TNF-? and Related <span class="highlight" style="background-color:">Signaling</span> Molecules in the Peripheral Blood Mononuclear Cells of Rheumatoid Arthritis Patients. </a></p><div class="supp"><p class="details"><div>An overview of the TNF-α <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to apoptosis or proliferation of the cells.</div></p></div><div class="aux"><div class="resc">Sunil Kumar Raghav, et al. Mediators Inflamm. 2006;2006(3):12682.</div><p class="links"><a class="dblinks" href="/pubmed/16951485" ref="ordinalpos=1246&amp;ncbi_uid=844560&amp;link_uid=16951485">Citation</a><a class="dblinks" href="/pmc/articles/PMC1592599/" ref="ordinalpos=1246&amp;ncbi_uid=844560&amp;link_uid=PMC1592599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1247.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3413009/figure/f5/" ref="ordinalpos=1247&amp;ncbi_uid=4165256&amp;link_uid=PMC3413009" href="/pmc/articles/PMC3413009/"><img src="/pmc/articles/instance/3413009/bin/srep00563-f5.gif" src-large="/pmc/articles/instance/3413009/bin/srep00563-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3413009/" ref="ordinalpos=1247&amp;ncbi_uid=4165256&amp;link_uid=PMC3413009" image-link="/pmc/articles/PMC3413009/figure/f5/" class="imagepopup">Figure 5. The effect of AuNPs on larva fat metabolism is dependent on Akt/mTOR <span class="highlight" style="background-color:">signaling</span>..  From: Akt <span class="highlight" style="background-color:">signaling</span>-associated metabolic effects of dietary gold nanoparticles in Drosophila. </a></p><div class="supp"><p class="details"><div>(A) (B) In standard diet (A) and CR (B), the increase in larva TAG levels by dietary AuNPs was abolished when the Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was blocked by rapamycin. *, P&lt;0.05. ns, not significant. (C) (D) Increased expression of <i>FAS</i> (C) and <i>ACC</i> (D) by dietary AuNPs was abolished when the Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was blocked. **, P&lt;0.01. ns, not significant. (E) A diagram for the effect of dietary AuNPs on larva fat metabolism.</div></p></div><div class="aux"><div class="resc">Bin Wang, et al. Sci Rep. 2012;2:563.</div><p class="links"><a class="dblinks" href="/pubmed/22872808" ref="ordinalpos=1247&amp;ncbi_uid=4165256&amp;link_uid=22872808">Citation</a><a class="dblinks" href="/pmc/articles/PMC3413009/" ref="ordinalpos=1247&amp;ncbi_uid=4165256&amp;link_uid=PMC3413009">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1248.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC33436/figure/F3/" ref="ordinalpos=1248&amp;ncbi_uid=62744&amp;link_uid=PMC33436" href="/pmc/articles/PMC33436/"><img src="/pmc/articles/instance/33436/bin/pq1015969003.gif" src-large="/pmc/articles/instance/33436/bin/pq1015969003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC33436/" ref="ordinalpos=1248&amp;ncbi_uid=62744&amp;link_uid=PMC33436" image-link="/pmc/articles/PMC33436/figure/F3/" class="imagepopup">Figure 3.  From: Drosophila phosphoinositide-dependent kinase-1  regulates apoptosis and growth via the phosphoinositide  3-kinase-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Genetic interactions between <i>dPDK-1</i> and  <i>Drosophila</i> PI3K in cell size control. Microscopic views  of the wings expressing <i>ptc</i>-GAL4/UAS-PI3KDN  (<i>A</i>), <i>ptc</i>-GAL4/UAS-PI3KDN; EP(3)3553/+  (<i>B</i>), <i>ptc</i>-GAL4/UAS-PI3K  (<i>C</i>), and <i>ptc</i>-GAL4/UAS-PI3K;  EP(3)3553/+ (<i>D</i>). The relative distances between two  arrows are <i>ptc</i>-GAL4/UAS-PI3KDN (58%),  <i>ptc</i>-GAL4/UAS-PI3KDN; EP(3)3553/+ (100%),  <i>ptc</i>-GAL4/UAS-PI3K (125%), and  <i>ptc</i>-GAL4/UAS-PI3K; EP(3)3553/+ (142%),  respectively. Increased activities of the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>  induced a loss of cross-veins (<i>B</i> and <i>D</i>),  as reported (45).</div></p></div><div class="aux"><div class="resc">Kyoung Sang Cho, et al. Proc Natl Acad Sci U S A. 2001 May 22;98(11):6144-6149.</div><p class="links"><a class="dblinks" href="/pubmed/11344272" ref="ordinalpos=1248&amp;ncbi_uid=62744&amp;link_uid=11344272">Citation</a><a class="dblinks" href="/pmc/articles/PMC33436/" ref="ordinalpos=1248&amp;ncbi_uid=62744&amp;link_uid=PMC33436">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1249.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4260340/figure/F3/" ref="ordinalpos=1249&amp;ncbi_uid=6946320&amp;link_uid=PMC4260340" href="/pmc/articles/PMC4260340/"><img src="/pmc/articles/instance/4260340/bin/nihms608244f3.gif" src-large="/pmc/articles/instance/4260340/bin/nihms608244f3.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4260340/" ref="ordinalpos=1249&amp;ncbi_uid=6946320&amp;link_uid=PMC4260340" image-link="/pmc/articles/PMC4260340/figure/F3/" class="imagepopup">FIGURE 3.  From: The Meteoric Rise of the <span class="highlight" style="background-color:">Signaling</span> Molecule Cyclic di-GMP. </a></p><div class="supp"><p class="details"><div>A major question in the field of c-di-GMP <span class="highlight" style="background-color:">signaling</span> is the degree of <span class="highlight" style="background-color:">signaling</span> specificity. C-di-GMP may exist as a high specificity <span class="highlight" style="background-color:">pathway</span> in which segregated “pools” linked to specific DGCs or PDEs regulate downstream phenotypes via parallel <span class="highlight" style="background-color:">signaling</span> pathways (left). Alternatively, in a low specificity system all DGCs and PDEs modulate a common c-di-GMP hub that impacts all regulated phenotypes (right).</div></p></div><div class="aux"><div class="resc">Christopher M. Waters. Microbe Wash DC. ;7(8):353-359.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1249&amp;ncbi_uid=6946320&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4260340/" ref="ordinalpos=1249&amp;ncbi_uid=6946320&amp;link_uid=PMC4260340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1250.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3897430/figure/pone-0085482-g006/" ref="ordinalpos=1250&amp;ncbi_uid=5737170&amp;link_uid=PMC3897430" href="/pmc/articles/PMC3897430/"><img src="/pmc/articles/instance/3897430/bin/pone.0085482.g006.gif" src-large="/pmc/articles/instance/3897430/bin/pone.0085482.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3897430/" ref="ordinalpos=1250&amp;ncbi_uid=5737170&amp;link_uid=PMC3897430" image-link="/pmc/articles/PMC3897430/figure/pone-0085482-g006/" class="imagepopup">Figure 6. Effects of clenbuterol or ractopamine exposure on expression patterns of genes required for aging control..  From: Adverse Effects from Clenbuterol and Ractopamine on Nematode Caenorhabditis elegans and the Underlying Mechanism. </a></p><div class="supp"><p class="details"><div>(A) Effects of clenbuterol or ractopamine exposure on expression patterns of genes in insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) Effects of clenbuterol or ractopamine exposure on expression patterns of genes in TOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (C) Effects of clenbuterol or ractopamine exposure on expression patterns of genes in germline <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Exposures were performed from L1-larvae to adult (prolonged exposure) at the concentration of 10 µg/L. Bars represent mean ± S.E.M. **<i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Ziheng Zhuang, et al. PLoS One. 2014;9(1):e85482.</div><p class="links"><a class="dblinks" href="/pubmed/24465573" ref="ordinalpos=1250&amp;ncbi_uid=5737170&amp;link_uid=24465573">Citation</a><a class="dblinks" href="/pmc/articles/PMC3897430/" ref="ordinalpos=1250&amp;ncbi_uid=5737170&amp;link_uid=PMC3897430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1251.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2823973/figure/F4/" ref="ordinalpos=1251&amp;ncbi_uid=2852169&amp;link_uid=PMC2823973" href="/pmc/articles/PMC2823973/"><img src="/pmc/articles/instance/2823973/bin/nihms-159692-f0004.gif" src-large="/pmc/articles/instance/2823973/bin/nihms-159692-f0004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2823973/" ref="ordinalpos=1251&amp;ncbi_uid=2852169&amp;link_uid=PMC2823973" image-link="/pmc/articles/PMC2823973/figure/F4/" class="imagepopup">Fig. 4.  From: Sonic Hedgehog (SHH) Promotes the Differentiation of Mouse Cochlear Neural Progenitors via the Math1-Brn3.1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> in vitro. </a></p><div class="supp"><p class="details"><div>LY294002 significantly inhibits the expression of <i>Math1</i> by SHH in CNPs. Specific inhibitors for the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (LY at 20 μM) and the Erk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (PD) at 24-hour significantly reduced the promoter activity of <i>Math1</i> induced by SHH (A). In contrast, CY and SB did not show any effect on the <i>Math1</i> promoter activity. LY at 48-hour incubation canceled out the expression of <i>Math1</i> in the nuclei by SHH (SHH+LY) in CNPs compared with control (B). Reduction of luciferase activity by LY and PD did not cause cellular apoptosis as judged by Annexin-V and 7-AAD (C). PD, PD98059; SB203580; LY, LY294002; CY, cyclopamine. Bar=10 μM applying to all panels in B.</div></p></div><div class="aux"><div class="resc">Xiaohua Hu, et al. J Neurosci Res. ;88(5):927-935.</div><p class="links"><a class="dblinks" href="/pubmed/19908278" ref="ordinalpos=1251&amp;ncbi_uid=2852169&amp;link_uid=19908278">Citation</a><a class="dblinks" href="/pmc/articles/PMC2823973/" ref="ordinalpos=1251&amp;ncbi_uid=2852169&amp;link_uid=PMC2823973">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1252.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3248194/figure/F2/" ref="ordinalpos=1252&amp;ncbi_uid=3761565&amp;link_uid=PMC3248194" href="/pmc/articles/PMC3248194/"><img src="/pmc/articles/instance/3248194/bin/oncotarget-02-497-g002.gif" src-large="/pmc/articles/instance/3248194/bin/oncotarget-02-497-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3248194/" ref="ordinalpos=1252&amp;ncbi_uid=3761565&amp;link_uid=PMC3248194" image-link="/pmc/articles/PMC3248194/figure/F2/" class="imagepopup">Figure 2. Schematic of caspase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: RIP Kinase-Mediated Necrosis as an Alternative Mechanism of Photoreceptor Death. </a></p><div class="supp"><p class="details"><div>The extrinsic <span class="highlight" style="background-color:">pathway</span> is initiated by biding of death ligands such as TNF-α and Fas ligand to their cell-surface death receptors such as TNFR and Fas. The death domains of these receptors recruit adaptor molecules like FADD and caspase-8, which leads to the activation of caspase-8. Activated caspase-8 cleaves the effector caspases such as caspase-3, thereby activating them and inducing apoptosis. The extrinsic <span class="highlight" style="background-color:">pathway</span> interacts with the intrinsic <span class="highlight" style="background-color:">pathway</span> via caspse-8-mediated cleavage of Bid. The intrinsic <span class="highlight" style="background-color:">pathway</span> is initiated by release of mitochondrial intermembrane proteins such as cytochrome <i>c</i> and Smac/Diablo into the cytosol. Released cytochrome <i>c</i> forms an apoptosome with Apaf-1 and caspase-9, which leads to caspase-9 activation. Smac/Diablo enhances caspase activation through the neutralization of IAP proteins.</div></p></div><div class="aux"><div class="resc">Yusuke Murakami, et al. Oncotarget. 2011 June;2(6):497-509.</div><p class="links"><a class="dblinks" href="/pubmed/21670490" ref="ordinalpos=1252&amp;ncbi_uid=3761565&amp;link_uid=21670490">Citation</a><a class="dblinks" href="/pmc/articles/PMC3248194/" ref="ordinalpos=1252&amp;ncbi_uid=3761565&amp;link_uid=PMC3248194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1253.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012911/figure/fig4/" ref="ordinalpos=1253&amp;ncbi_uid=6623876&amp;link_uid=PMC4012911" href="/pmc/articles/PMC4012911/"><img src="/pmc/articles/instance/4012911/bin/aps201089f4.gif" src-large="/pmc/articles/instance/4012911/bin/aps201089f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012911/" ref="ordinalpos=1253&amp;ncbi_uid=6623876&amp;link_uid=PMC4012911" image-link="/pmc/articles/PMC4012911/figure/fig4/" class="imagepopup">Figure 4.  From: Adrenomedullin up-regulates osteopontin and attenuates vascular calcification via the cAMP/PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>ADM upregulates OPN mRNA expression through the cyclic adenosine 3′,5′-monophosphate (cAMP)/protein kinase A (PKA) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Forskolin, an adenylyl cyclase agonist; H89, a PKA inhibitor; Fura-2, a Ca<sup>2+</sup> chelator; and PD98059, a mitogen-activated protein kinase kinase (MEK)/extracellular signal-related kinase (ERK) inhibitor. <sup>c</sup><i>P</i>&lt;0.01 <i>vs</i> Con; <sup>e</sup><i>P</i>&lt;0.05 <i>vs</i> ADM 10<sup>−7</sup> mol/L.</div></p></div><div class="aux"><div class="resc">Yan Cai, et al. Acta Pharmacol Sin. 2010 October;31(10):1359-1366.</div><p class="links"><a class="dblinks" href="/pubmed/20802507" ref="ordinalpos=1253&amp;ncbi_uid=6623876&amp;link_uid=20802507">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012911/" ref="ordinalpos=1253&amp;ncbi_uid=6623876&amp;link_uid=PMC4012911">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1254.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2739756/figure/fig3/" ref="ordinalpos=1254&amp;ncbi_uid=2482820&amp;link_uid=PMC2739756" href="/pmc/articles/PMC2739756/"><img src="/pmc/articles/instance/2739756/bin/1955fig3.gif" src-large="/pmc/articles/instance/2739756/bin/1955fig3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2739756/" ref="ordinalpos=1254&amp;ncbi_uid=2482820&amp;link_uid=PMC2739756" image-link="/pmc/articles/PMC2739756/figure/fig3/" class="imagepopup">Fig. 3. From: Bile acids: regulation of synthesis. </a></p><div class="supp"><p class="details"><div>Mechanisms of FXR regulation of enterohepatic circulation of bile acid. Bile acids synthesized in the liver are excreted into bile via BSEP and stored in the gallbladder. After each meal, bile acids are excreted into the intestinal tract. In the ileum, bile acids are reabsorbed by ASBT in the brush border membrane. Bile acids activate FXR to induce IBABP in enterocytes. OSTα/β transporter in the basolateral membrane effluxes bile acids to portal circulation to hepatocytes where they are taken up by NTCP. In the liver, bile acids activate FXR, which induces SHP expression. SHP then inhibits LRH-1 (or human FTF) and HNF4α transactivation of <i>CYP7A1</i> (FXR/SHP <span class="highlight" style="background-color:">pathway</span> 1). In the endocrine <span class="highlight" style="background-color:">pathway</span>, intestinal bile acids activate FXR, which induces FGF19 expression. FGF19 may be transported to the liver to activate a liver-specific receptor tyrosine kinase FGFR4 (FXR/FGF19/FGFR4 <span class="highlight" style="background-color:">pathway</span> 2). In the autocrine <span class="highlight" style="background-color:">pathway</span> (<span class="highlight" style="background-color:">pathway</span> 3), cholestatic bile acids may activate FXR and FGF19/FGFR4 <span class="highlight" style="background-color:">signaling</span>, which activates the MAPK/ERK1/2 <span class="highlight" style="background-color:">pathway</span> to inhibit <i>CYP7A1</i> transcription. It is not clear how the FGF19/ERK1/2 <span class="highlight" style="background-color:">pathway</span> downregulates <i>CYP7A1</i> transcription. The endocrine <span class="highlight" style="background-color:">pathway</span> may be a physiological <span class="highlight" style="background-color:">pathway</span> for bile acid inhibition of bile acid synthesis, while the autocrine <span class="highlight" style="background-color:">pathway</span> may be an adaptive response to protect liver from cholestatic injury. BARE-II contains 18 bp sequence of overlapping HNF4α and FTF (α-fetoprotein transcription factor, a human homolog of mouse LRH-1) binding site, which is completely conserved in all species.</div></p></div><div class="aux"><div class="resc">John Y. L. Chiang. J Lipid Res. 2009 October;50(10):1955-1966.</div><p class="links"><a class="dblinks" href="/pubmed/19346330" ref="ordinalpos=1254&amp;ncbi_uid=2482820&amp;link_uid=19346330">Citation</a><a class="dblinks" href="/pmc/articles/PMC2739756/" ref="ordinalpos=1254&amp;ncbi_uid=2482820&amp;link_uid=PMC2739756">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1255.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2841156/figure/F7/" ref="ordinalpos=1255&amp;ncbi_uid=2256965&amp;link_uid=PMC2841156" href="/pmc/articles/PMC2841156/"><img src="/pmc/articles/instance/2841156/bin/1475-2867-10-3-7.gif" src-large="/pmc/articles/instance/2841156/bin/1475-2867-10-3-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2841156/" ref="ordinalpos=1255&amp;ncbi_uid=2256965&amp;link_uid=PMC2841156" image-link="/pmc/articles/PMC2841156/figure/F7/" class="imagepopup">Figure 7.  From: Upstream molecular <span class="highlight" style="background-color:">signaling</span> pathways of p27(Kip1) expression: Effects of 4-hydroxytamoxifen, dexamethasone, and retinoic acids. </a></p><div class="supp"><p class="details"><div><b>Schematic drawing of the four upstream molecular <span class="highlight" style="background-color:">signaling</span> pathways of p27 expression that could lead to activation of the unusually long 5'-untranslated region (5'-UTR) (-575) of p27 mRNA by 4-hydroxytamoxifen, dexamethasone and retinoic acids</b>. (a) The two upstream molecular <span class="highlight" style="background-color:">signaling</span> pathways of p27 shown in Figure 5a are pathways #1 and #2. The <span class="highlight" style="background-color:">pathway</span> #1 consisted of receptor tyrosine kinases/phosphoinositide 3-kinase/Akt/tuberous sclerosis complex/mammalian target of rapamycin/eukaryotic translation initiation factor 4E (eIF4E) -binding protein 1 (RTKs/PI3K/Akt/TSC/mTOR/4E-BP1). The <span class="highlight" style="background-color:">pathway</span> #2 consisted of 5'-AMP-activated protein kinase (metabolic energy sensor or cellular fuel gauge)/tuberous sclerosis complex/mammalian target of rapamycin/eIF4E-binding protein 1 (AMPK/TSC/mTOR/4E-BP1). (b) In addition to these two pathways, two more upstream molecular <span class="highlight" style="background-color:">signaling</span> pathways of p27 expression were previously identified. They were pathways #3 and #4. The <span class="highlight" style="background-color:">pathway</span> #3 consisted of receptor tyrosine kinases/MAPKs/eIF4E (RTKs/MAPKs/eIF4E). The <span class="highlight" style="background-color:">pathway</span> #4 consisted of global hypomethylation of the 5'-7-methylguanosine (m<sup>7</sup>G) cap of mRNAs. The specific inhibitors and activators used previously to identify these four pathways are indicated next to each of the four pathways. The results of this study suggested that 4-hydroxytamoxifen used <span class="highlight" style="background-color:">pathway</span> #1 and dexamethasone primarily used <span class="highlight" style="background-color:">pathway</span> #2 to up-regulate the expression of p27. Dexamethasone could also use a portion of <span class="highlight" style="background-color:">pathway</span> #1 secondarily. We also believe, but could not conclude, that 4-hydroxytamoxifen up-regulated the expression of p27 using MAP kinase pathways (<span class="highlight" style="background-color:">Pathway</span> #3 in Figures 5b and 7b). Retinoic acids up-regulated p27 expression without using pathways #1, #2 and #3. We propose a hypothesis that retinoic acids are likely to have used <span class="highlight" style="background-color:">pathway</span> #4 to up-regulate the expression of p27. Abbreviations: see the legend of Figure 5.</div></p></div><div class="aux"><div class="resc">Isao Eto. Cancer Cell Int. 2010;10:3-3.</div><p class="links"><a class="dblinks" href="/pubmed/20170512" ref="ordinalpos=1255&amp;ncbi_uid=2256965&amp;link_uid=20170512">Citation</a><a class="dblinks" href="/pmc/articles/PMC2841156/" ref="ordinalpos=1255&amp;ncbi_uid=2256965&amp;link_uid=PMC2841156">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1256.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3587808/figure/fig1/" ref="ordinalpos=1256&amp;ncbi_uid=5854000&amp;link_uid=PMC3587808" href="/pmc/articles/PMC3587808/"><img src="/pmc/articles/instance/3587808/bin/jcbfm2012174f1.gif" src-large="/pmc/articles/instance/3587808/bin/jcbfm2012174f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3587808/" ref="ordinalpos=1256&amp;ncbi_uid=5854000&amp;link_uid=PMC3587808" image-link="/pmc/articles/PMC3587808/figure/fig1/" class="imagepopup">Figure 1.  From: Mrp1 is essential for sphingolipid <span class="highlight" style="background-color:">signaling</span> to p-glycoprotein in mouse blood-brain and blood-spinal cord barriers. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> that regulates basal P-glycoprotein activity in brain capillaries.<sup>12</sup> Agents that activate <span class="highlight" style="background-color:">signaling</span> at various points in the <span class="highlight" style="background-color:">pathway</span> are shown above each affected step. Identifying the molecular basis for sphingosine-1-phosphate (S1P) efflux from brain and spinal cord capillary endothelial cells (curved arrow) is the focus of the current study.</div></p></div><div class="aux"><div class="resc">Tara A Cartwright, et al. J Cereb Blood Flow Metab. 2013 March;33(3):381-388.</div><p class="links"><a class="dblinks" href="/pubmed/23168528" ref="ordinalpos=1256&amp;ncbi_uid=5854000&amp;link_uid=23168528">Citation</a><a class="dblinks" href="/pmc/articles/PMC3587808/" ref="ordinalpos=1256&amp;ncbi_uid=5854000&amp;link_uid=PMC3587808">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1257.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3694976/figure/pone-0067134-g010/" ref="ordinalpos=1257&amp;ncbi_uid=5057809&amp;link_uid=PMC3694976" href="/pmc/articles/PMC3694976/"><img src="/pmc/articles/instance/3694976/bin/pone.0067134.g010.gif" src-large="/pmc/articles/instance/3694976/bin/pone.0067134.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3694976/" ref="ordinalpos=1257&amp;ncbi_uid=5057809&amp;link_uid=PMC3694976" image-link="/pmc/articles/PMC3694976/figure/pone-0067134-g010/" class="imagepopup">Figure 10. <span class="highlight" style="background-color:">Pathway</span> diagram of the KEGG Jak-Stat <span class="highlight" style="background-color:">pathway</span> using DAVID analysis..  From: Global Identification of EVI1 Target Genes in Acute Myeloid Leukemia. </a></p><div class="supp"><p class="details"><div>EVI1 binds to the majority (78%) of major genes involved in the regulation of the Jak-Stat <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. These include <i>Il10, Il10ra, Il6, Il6ra, Cbl, Jak1, Pias2, Stat1, Stat4, Stat5, Stat6, Grb, Sos2, Akt1, Akt2, Socs1,Pim1, Ccnd1, Ccnd2, Ccnd3, Myc,</i> and <i>Spred1</i>. EVI1 binds to an AGGAAG ETS-like motif which is present in the promoter region of all of these Jak-Stat <span class="highlight" style="background-color:">pathway</span> genes. Illustration taken from DAVID website http://david.abcc.ncifcrf.gov/.</div></p></div><div class="aux"><div class="resc">Carolyn Glass, et al. PLoS One. 2013;8(6):e67134.</div><p class="links"><a class="dblinks" href="/pubmed/23826213" ref="ordinalpos=1257&amp;ncbi_uid=5057809&amp;link_uid=23826213">Citation</a><a class="dblinks" href="/pmc/articles/PMC3694976/" ref="ordinalpos=1257&amp;ncbi_uid=5057809&amp;link_uid=PMC3694976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1258.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3423373/figure/pone-0037046-g008/" ref="ordinalpos=1258&amp;ncbi_uid=4187636&amp;link_uid=PMC3423373" href="/pmc/articles/PMC3423373/"><img src="/pmc/articles/instance/3423373/bin/pone.0037046.g008.gif" src-large="/pmc/articles/instance/3423373/bin/pone.0037046.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3423373/" ref="ordinalpos=1258&amp;ncbi_uid=4187636&amp;link_uid=PMC3423373" image-link="/pmc/articles/PMC3423373/figure/pone-0037046-g008/" class="imagepopup">Figure 8. Changes of lung mRNAs related to the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> were influenced by strain..  From: Transcriptional Programs Controlling Perinatal Lung Maturation. </a></p><div class="supp"><p class="details"><div>(A) Representation of mRNAs changed during lung maturation in KEGG Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (map04330). Red nodes indicate mRNAs that were significantly increased, while dark green indicates decreased expression during lung maturation. Light green indicates no significant change in gene expression or data not available. (B) Genes encoding a number of key components in the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> were expressed at significantly higher levels in A/J than in B6 mice at E18.5 (*: P&lt;0.05). Results are expressed as the means ± SEM of 3 animals per time point.</div></p></div><div class="aux"><div class="resc">Yan Xu, et al. PLoS One. 2012;7(8):e37046.</div><p class="links"><a class="dblinks" href="/pubmed/22916088" ref="ordinalpos=1258&amp;ncbi_uid=4187636&amp;link_uid=22916088">Citation</a><a class="dblinks" href="/pmc/articles/PMC3423373/" ref="ordinalpos=1258&amp;ncbi_uid=4187636&amp;link_uid=PMC3423373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1259.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3663521/figure/F7/" ref="ordinalpos=1259&amp;ncbi_uid=6164079&amp;link_uid=PMC3663521" href="/pmc/articles/PMC3663521/"><img src="/pmc/articles/instance/3663521/bin/zbc0251350330007.gif" src-large="/pmc/articles/instance/3663521/bin/zbc0251350330007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3663521/" ref="ordinalpos=1259&amp;ncbi_uid=6164079&amp;link_uid=PMC3663521" image-link="/pmc/articles/PMC3663521/figure/F7/" class="imagepopup">FIGURE 7. From: Disruption of the Dapper3 Gene Aggravates Ureteral Obstruction-mediated Renal Fibrosis by Amplifying Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram illustrates the role of Dapper3 in UUO-induced renal fibrosis by regulating Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>.</b> The Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> is activated by UUO and contributes to renal fibrosis. Dapper3 induces Dvl degradation and thus attenuates Wnt <span class="highlight" style="background-color:">signaling</span>. Ablation of Dapper3 leads to increased protein levels of Dvl and Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>, further activating fibrotic gene expression and causing aggravated renal fibrosis. <i>TCF</i>, T cell factor; <i>LEF</i>, lymphoid enhancer-binding factor; <i>Fz</i>, Frizzled.</div></p></div><div class="aux"><div class="resc">Hua Xue, et al. J Biol Chem. 2013 May 24;288(21):15006-15014.</div><p class="links"><a class="dblinks" href="/pubmed/23580654" ref="ordinalpos=1259&amp;ncbi_uid=6164079&amp;link_uid=23580654">Citation</a><a class="dblinks" href="/pmc/articles/PMC3663521/" ref="ordinalpos=1259&amp;ncbi_uid=6164079&amp;link_uid=PMC3663521">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1260.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3671665/figure/fig1/" ref="ordinalpos=1260&amp;ncbi_uid=4994482&amp;link_uid=PMC3671665" href="/pmc/articles/PMC3671665/"><img src="/pmc/articles/instance/3671665/bin/ECAM2013-748587.001.gif" src-large="/pmc/articles/instance/3671665/bin/ECAM2013-748587.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3671665/" ref="ordinalpos=1260&amp;ncbi_uid=4994482&amp;link_uid=PMC3671665" image-link="/pmc/articles/PMC3671665/figure/fig1/" class="imagepopup">Figure 1.  From: Targeting Sonic Hedgehog <span class="highlight" style="background-color:">Signaling</span> by Compounds and Derivatives from Natural Products. </a></p><div class="supp"><p class="details"><div>Targeting the Sonic hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by compounds and derivatives from natural products. Circled numbers beside compound names indicate target Shh <span class="highlight" style="background-color:">signaling</span> proteins. Question marks indicate that the targeted protein is uncertain.</div></p></div><div class="aux"><div class="resc">Yu-Chuen Huang, et al. Evid Based Complement Alternat Med. 2013;2013:748587.</div><p class="links"><a class="dblinks" href="/pubmed/23762158" ref="ordinalpos=1260&amp;ncbi_uid=4994482&amp;link_uid=23762158">Citation</a><a class="dblinks" href="/pmc/articles/PMC3671665/" ref="ordinalpos=1260&amp;ncbi_uid=4994482&amp;link_uid=PMC3671665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1261.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3075605/figure/fig2/" ref="ordinalpos=1261&amp;ncbi_uid=3770448&amp;link_uid=PMC3075605" href="/pmc/articles/PMC3075605/"><img src="/pmc/articles/instance/3193403/bin/cr2010166f2.gif" src-large="/pmc/articles/instance/3193403/bin/cr2010166f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3075605/" ref="ordinalpos=1261&amp;ncbi_uid=3770448&amp;link_uid=PMC3075605" image-link="/pmc/articles/PMC3075605/figure/fig2/" class="imagepopup">Figure 2.  From: Deubiquitinases in the regulation of NF-?B <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>NF-κB <span class="highlight" style="background-color:">signaling</span> pathways downstream of TNFR1, TLR4 and TCR/CD28. TNFR1, TLR4 and TCR/CD28 engagement trigger the K63-linked ubiquitination of RIP1, TRAF6 and MALT1, respectively. Each of these ubiquitinated <span class="highlight" style="background-color:">signaling</span> molecules are targeted for deubiquitination by either A20, CYLD, Cezanne or USP21, as indicated. MALT1 cleaves A20 to inactivate its function. CYLD also removes polyubiquitin chains from TAK1 in the TCR <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Edward W Harhaj, et al. Cell Res. 2011 January;21(1):22-39.</div><p class="links"><a class="dblinks" href="/pubmed/21119682" ref="ordinalpos=1261&amp;ncbi_uid=3770448&amp;link_uid=21119682">Citation</a><a class="dblinks" href="/pmc/articles/PMC3075605/" ref="ordinalpos=1261&amp;ncbi_uid=3770448&amp;link_uid=PMC3075605">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1262.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2246119/figure/F7/" ref="ordinalpos=1262&amp;ncbi_uid=1300200&amp;link_uid=PMC2246119" href="/pmc/articles/PMC2246119/"><img src="/pmc/articles/instance/2246119/bin/1471-2121-9-4-7.gif" src-large="/pmc/articles/instance/2246119/bin/1471-2121-9-4-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2246119/" ref="ordinalpos=1262&amp;ncbi_uid=1300200&amp;link_uid=PMC2246119" image-link="/pmc/articles/PMC2246119/figure/F7/" class="imagepopup">Figure 7.  From: The androgen receptor can signal through Wnt/?-Catenin in prostate cancer cells as an adaptation mechanism to castration levels of androgens. </a></p><div class="supp"><p class="details"><div><b>Working model for the interaction of AR and Wnt/β-Catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in castration-resistant prostate cancer cells</b>. In normal conditions, Wnt and androgen <span class="highlight" style="background-color:">signaling</span> function at different cell populations (A&amp;B). In prostate cancer where both pathways may be deregulated in the same cells, androgens may inhibit the input of AR into the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> where β-Catenin can potentiate the AR <span class="highlight" style="background-color:">signaling</span> (C). In castration-resistant prostate cancer cells, Wnt can activate both <span class="highlight" style="background-color:">signaling</span> pathways turning the unliganded AR into a coactivator of the Wnt dependent transcriptional program (D).</div></p></div><div class="aux"><div class="resc">Liang Schweizer, et al. BMC Cell Biol. 2008;9:4-4.</div><p class="links"><a class="dblinks" href="/pubmed/18218096" ref="ordinalpos=1262&amp;ncbi_uid=1300200&amp;link_uid=18218096">Citation</a><a class="dblinks" href="/pmc/articles/PMC2246119/" ref="ordinalpos=1262&amp;ncbi_uid=1300200&amp;link_uid=PMC2246119">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1263.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3245902/figure/F1/" ref="ordinalpos=1263&amp;ncbi_uid=4369448&amp;link_uid=PMC3245902" href="/pmc/articles/PMC3245902/"><img src="/pmc/articles/instance/3245902/bin/nihms-335494-f0001.gif" src-large="/pmc/articles/instance/3245902/bin/nihms-335494-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3245902/" ref="ordinalpos=1263&amp;ncbi_uid=4369448&amp;link_uid=PMC3245902" image-link="/pmc/articles/PMC3245902/figure/F1/" class="imagepopup">Figure 1. Model for the Context-Dependent Opposing Roles of NF-κB <span class="highlight" style="background-color:">Signaling</span> in Cancer Development and Therapy.  From: The Two Faces of NF-?B <span class="highlight" style="background-color:">Signaling</span> in Cancer Development and Therapy. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) In tumors with genetic mutations in NF-κB <span class="highlight" style="background-color:">pathway</span> components leading to constitutive NF-κB <span class="highlight" style="background-color:">signaling</span>, the pro-survival <i>BCL2</i> oncogene is upregulated. In these tumors, such as ABC-DLBCL, NF-κB inhibitors are beneficial as they counteract the oncogenic function of NF-κB and would promote tumor cell death.<br />(<b>B</b>) In tumors where the pro-survival signal derives from NF-κB-independent <i>BCL2</i> overexpression, a functional NF-κB <span class="highlight" style="background-color:">pathway</span> is required to mediate the therapy-induced senescence response by activating the transcription of genes of SASP. In this scenario, such as in GCB DLBCL, NF-κB inhibitors are detrimental as they interfere with the senescence response that would normally promote tumor cell death.</div></p></div><div class="aux"><div class="resc">Ulf Klein, et al. Cancer Cell. ;20(5):556-558.</div><p class="links"><a class="dblinks" href="/pubmed/22094250" ref="ordinalpos=1263&amp;ncbi_uid=4369448&amp;link_uid=22094250">Citation</a><a class="dblinks" href="/pmc/articles/PMC3245902/" ref="ordinalpos=1263&amp;ncbi_uid=4369448&amp;link_uid=PMC3245902">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1264.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4275273/figure/pone-0115985-g006/" ref="ordinalpos=1264&amp;ncbi_uid=7007973&amp;link_uid=PMC4275273" href="/pmc/articles/PMC4275273/"><img src="/pmc/articles/instance/4275273/bin/pone.0115985.g006.gif" src-large="/pmc/articles/instance/4275273/bin/pone.0115985.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4275273/" ref="ordinalpos=1264&amp;ncbi_uid=7007973&amp;link_uid=PMC4275273" image-link="/pmc/articles/PMC4275273/figure/pone-0115985-g006/" class="imagepopup">Figure 6. Genetic interactions of dopamine receptors with G-protein <span class="highlight" style="background-color:">signaling</span> pathways in regulating behavioral choice between conflicting alternatives..  From: Dopamine Receptors Antagonistically Regulate Behavioral Choice between Conflicting Alternatives in C. elegans. </a></p><div class="supp"><p class="details"><div>(A) Genetic interactions of DOP-1 with Gα<sub>q</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in regulating behavioral choice between conflicting alternatives. (B) Genetic interactions of DOP-3 with Gα<sub>o</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in regulating behavioral choice between conflicting alternatives. The behavioral choice was examined under the well-fed condition. In the assay system, the Cu<sup>2+</sup> ion concentration was 100 mM, and the diacetyl concentration was 10<sup>−2</sup>. Bars represent mean ± S.E.M. **<i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Daoyong Wang, et al. PLoS One. 2014;9(12):e115985.</div><p class="links"><a class="dblinks" href="/pubmed/25536037" ref="ordinalpos=1264&amp;ncbi_uid=7007973&amp;link_uid=25536037">Citation</a><a class="dblinks" href="/pmc/articles/PMC4275273/" ref="ordinalpos=1264&amp;ncbi_uid=7007973&amp;link_uid=PMC4275273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1265.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4008425/figure/pone-0095765-g004/" ref="ordinalpos=1265&amp;ncbi_uid=6116407&amp;link_uid=PMC4008425" href="/pmc/articles/PMC4008425/"><img src="/pmc/articles/instance/4008425/bin/pone.0095765.g004.gif" src-large="/pmc/articles/instance/4008425/bin/pone.0095765.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4008425/" ref="ordinalpos=1265&amp;ncbi_uid=6116407&amp;link_uid=PMC4008425" image-link="/pmc/articles/PMC4008425/figure/pone-0095765-g004/" class="imagepopup">Figure 4. The differentially expressed genes in U266/CD9 and U266/Vector cells tested by genome-wide expression profile chip experiments and the analysis of related <span class="highlight" style="background-color:">signaling</span> pathways..  From: Down-Regulation of CD9 by Methylation Decreased Bortezomib Sensitivity in Multiple Myeloma. </a></p><div class="supp"><p class="details"><div>CD9 expression mainly affected three important <span class="highlight" style="background-color:">signaling</span> pathways in MM. They are the endoplasmic reticulum stress <span class="highlight" style="background-color:">pathway</span>, cell adhesion related <span class="highlight" style="background-color:">signaling</span> pathways and osteoclast differentiation related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, respectively (A). The degree of differentially expressed genes and p values were also analysed (B).</div></p></div><div class="aux"><div class="resc">Xiaotong Hu, et al. PLoS One. 2014;9(5):e95765.</div><p class="links"><a class="dblinks" href="/pubmed/24788635" ref="ordinalpos=1265&amp;ncbi_uid=6116407&amp;link_uid=24788635">Citation</a><a class="dblinks" href="/pmc/articles/PMC4008425/" ref="ordinalpos=1265&amp;ncbi_uid=6116407&amp;link_uid=PMC4008425">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1266.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3059306/figure/F1/" ref="ordinalpos=1266&amp;ncbi_uid=3523509&amp;link_uid=PMC3059306" href="/pmc/articles/PMC3059306/"><img src="/pmc/articles/instance/3059306/bin/nihms-278202-f0001.gif" src-large="/pmc/articles/instance/3059306/bin/nihms-278202-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3059306/" ref="ordinalpos=1266&amp;ncbi_uid=3523509&amp;link_uid=PMC3059306" image-link="/pmc/articles/PMC3059306/figure/F1/" class="imagepopup">Figure 1.  From: Mechanisms of Neurocognitive Dysfunction and Therapeutic Considerations in Tuberous Sclerosis Complex. </a></p><div class="supp"><p class="details"><div>TSC mediated <span class="highlight" style="background-color:">signaling</span> in the central nervous system. This cartoon of TSC mediated <span class="highlight" style="background-color:">signaling</span> has been simplified to highlight the demonstrated biologic roles for TSC mediated mTOR <span class="highlight" style="background-color:">signaling</span> in the nervous system. Among the upstream <span class="highlight" style="background-color:">signaling</span> pathways, only the growth factors and ephrins have been shown to modulate TSC-mTOR <span class="highlight" style="background-color:">pathway</span> in neurons (growth factors, ephrins) while others (e.g. Wnts) have been implicated, but not proven to regulate TSC <span class="highlight" style="background-color:">signaling</span> in the nervous system.</div></p></div><div class="aux"><div class="resc">Peter Tsai, et al. Curr Opin Neurol. ;24(2):106-113.</div><p class="links"><a class="dblinks" href="/pubmed/21301339" ref="ordinalpos=1266&amp;ncbi_uid=3523509&amp;link_uid=21301339">Citation</a><a class="dblinks" href="/pmc/articles/PMC3059306/" ref="ordinalpos=1266&amp;ncbi_uid=3523509&amp;link_uid=PMC3059306">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1267.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4260825/figure/pbio-1002012-g004/" ref="ordinalpos=1267&amp;ncbi_uid=6962502&amp;link_uid=PMC4260825" href="/pmc/articles/PMC4260825/"><img src="/pmc/articles/instance/4260825/bin/pbio.1002012.g004.gif" src-large="/pmc/articles/instance/4260825/bin/pbio.1002012.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4260825/" ref="ordinalpos=1267&amp;ncbi_uid=6962502&amp;link_uid=PMC4260825" image-link="/pmc/articles/PMC4260825/figure/pbio-1002012-g004/" class="imagepopup">Figure 4. Exploring the mechanisms that enable <span class="highlight" style="background-color:">signaling</span> complexes to tolerate rearrangement-mediated gene replacements..  From: The Robustness of a <span class="highlight" style="background-color:">Signaling</span> Complex to Domain Rearrangements Facilitates Network Evolution. </a></p><div class="supp"><p class="details"><div>(A) Differentiating between two alternative hypotheses: removal of IDRs should negatively impact <span class="highlight" style="background-color:">signaling</span> function if the <span class="highlight" style="background-color:">signaling</span> complex possesses well-defined spatial constraints and therefore a rather rigid structure (left). In contrast, removal of IDRs could be tolerated if the complex is flexible and can adopt a wide ensemble of conformations (right). (B) Schematic representation of the IDR deletion variants. (C) Mating <span class="highlight" style="background-color:">pathway</span> function in yeast strains with IDR-deleted Ste11, Ste20, or Ste50 variants (“Short”), relative to their corresponding full-length variants. (D) Mating <span class="highlight" style="background-color:">pathway</span> function in yeast strains with pairs of simultaneously IDR-deleted variants (either Ste11 and Ste20, or Ste11 and Ste50) in the respective double KO, relative to their corresponding full-length variants. (E) Mating <span class="highlight" style="background-color:">pathway</span> function in yeast strains with IDR-deleted domain rearrangement variants. Statistically significant differences are marked with asterisks. Data shown in Data S1.</div></p></div><div class="aux"><div class="resc">Paloma M. Sato, et al. PLoS Biol. 2014 December;12(12):e1002012.</div><p class="links"><a class="dblinks" href="/pubmed/25490747" ref="ordinalpos=1267&amp;ncbi_uid=6962502&amp;link_uid=25490747">Citation</a><a class="dblinks" href="/pmc/articles/PMC4260825/" ref="ordinalpos=1267&amp;ncbi_uid=6962502&amp;link_uid=PMC4260825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1268.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3625703/figure/F1/" ref="ordinalpos=1268&amp;ncbi_uid=4804879&amp;link_uid=PMC3625703" href="/pmc/articles/PMC3625703/"><img src="/pmc/articles/instance/3625703/bin/nihms-452614-f0001.gif" src-large="/pmc/articles/instance/3625703/bin/nihms-452614-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3625703/" ref="ordinalpos=1268&amp;ncbi_uid=4804879&amp;link_uid=PMC3625703" image-link="/pmc/articles/PMC3625703/figure/F1/" class="imagepopup">Figure 1. Major GPCR <span class="highlight" style="background-color:">signaling</span> responses and regulation by endocytosis.  From: Role of ubiquitination in endocytic trafficking of G protein-coupled receptors. </a></p><div class="supp"><p class="details"><div><b>A.</b> Simplified schematic depicting the range of <span class="highlight" style="background-color:">signaling</span> responses mediated by GPCRs in mammals. <b>B.</b> Major trafficking itineraries of GPCRs that have been linked to functional regulation of cellular <span class="highlight" style="background-color:">signaling</span>. Endocytosis of receptors can produce functional desensitization of cellular responsiveness by physically removing receptors from access to extracellular ligands, and preventing access to plasma membrane-delimited <span class="highlight" style="background-color:">signaling</span> mediators. Recycling of internalized receptors can function, conversely, to resensitize cellular responsiveness. Trafficking of internalized receptors via the MVB/lysosome <span class="highlight" style="background-color:">pathway</span> causes a prolonged attenuation of cellular responsiveness by promoting proteolytic downregulation of receptors.</div></p></div><div class="aux"><div class="resc">James N Hislop, et al. Traffic. 2011 February;12(2):137-148.</div><p class="links"><a class="dblinks" href="/pubmed/20854416" ref="ordinalpos=1268&amp;ncbi_uid=4804879&amp;link_uid=20854416">Citation</a><a class="dblinks" href="/pmc/articles/PMC3625703/" ref="ordinalpos=1268&amp;ncbi_uid=4804879&amp;link_uid=PMC3625703">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1269.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3587990/figure/f4-ijms-14-02334/" ref="ordinalpos=1269&amp;ncbi_uid=4699642&amp;link_uid=PMC3587990" href="/pmc/articles/PMC3587990/"><img src="/pmc/articles/instance/3587990/bin/ijms-14-02334f4.gif" src-large="/pmc/articles/instance/3587990/bin/ijms-14-02334f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3587990/" ref="ordinalpos=1269&amp;ncbi_uid=4699642&amp;link_uid=PMC3587990" image-link="/pmc/articles/PMC3587990/figure/f4-ijms-14-02334/" class="imagepopup">Figure 4.  From: Cell Survival and Apoptosis <span class="highlight" style="background-color:">Signaling</span> as Therapeutic Target for Cancer: Marine Bioactive Compounds. </a></p><div class="supp"><p class="details"><div>Interaction of marine compounds on survival and apoptotic <span class="highlight" style="background-color:">signaling</span> molecules. Regulation of this <span class="highlight" style="background-color:">pathway</span> with marine compounds; arrow indicates activation and blocked arrow indicates inhibition of the molecules.</div></p></div><div class="aux"><div class="resc">Senthilkumar Kalimuthu, et al. Int J Mol Sci. 2013 February;14(2):2334-2354.</div><p class="links"><a class="dblinks" href="/pubmed/23348928" ref="ordinalpos=1269&amp;ncbi_uid=4699642&amp;link_uid=23348928">Citation</a><a class="dblinks" href="/pmc/articles/PMC3587990/" ref="ordinalpos=1269&amp;ncbi_uid=4699642&amp;link_uid=PMC3587990">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1270.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC526379/figure/F1/" ref="ordinalpos=1270&amp;ncbi_uid=563864&amp;link_uid=PMC526379" href="/pmc/articles/PMC526379/"><img src="/pmc/articles/instance/526379/bin/1471-2105-5-158-1.gif" src-large="/pmc/articles/instance/526379/bin/1471-2105-5-158-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC526379/" ref="ordinalpos=1270&amp;ncbi_uid=563864&amp;link_uid=PMC526379" image-link="/pmc/articles/PMC526379/figure/F1/" class="imagepopup">Figure 1.  From: A computational approach for ordering signal transduction <span class="highlight" style="background-color:">pathway</span> components from genomics and proteomics Data. </a></p><div class="supp"><p class="details"><div><b>MAPK <span class="highlight" style="background-color:">signaling</span> pathways in <i>Saccharomyces cerevisiae</i>. </b>Membrane receptors are marked in blue, and transcription factors are marked in red. The figure is adapted from KEGG <span class="highlight" style="background-color:">pathway</span> database [12], and the scaffold proteins and proteins on the <span class="highlight" style="background-color:">pathway</span> branches are omitted for simplicity.</div></p></div><div class="aux"><div class="resc">Yin Liu, et al. BMC Bioinformatics. 2004;5:158-158.</div><p class="links"><a class="dblinks" href="/pubmed/15504238" ref="ordinalpos=1270&amp;ncbi_uid=563864&amp;link_uid=15504238">Citation</a><a class="dblinks" href="/pmc/articles/PMC526379/" ref="ordinalpos=1270&amp;ncbi_uid=563864&amp;link_uid=PMC526379">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1271.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2923360/figure/F8/" ref="ordinalpos=1271&amp;ncbi_uid=3178441&amp;link_uid=PMC2923360" href="/pmc/articles/PMC2923360/"><img src="/pmc/articles/instance/2923360/bin/z380091079210008.gif" src-large="/pmc/articles/instance/2923360/bin/z380091079210008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2923360/" ref="ordinalpos=1271&amp;ncbi_uid=3178441&amp;link_uid=PMC2923360" image-link="/pmc/articles/PMC2923360/figure/F8/" class="imagepopup">Figure 8. From: Anisotropic regulation of Ankrd2 gene expression in skeletal muscle by mechanical stretch. </a></p><div class="supp"><p class="details"><div>Schematic representation of anisotropic regulation of Ankrd2 in the diaphragm muscle. Potential mechanism of activation of two distinct <span class="highlight" style="background-color:">signaling</span> pathways and their involvement in the transcription of Ankrd2 gene. Longitudinal stretch of the diaphragm up-regulates Ankrd2 gene expression <i>via</i> the activation of Akt-dependent NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Despite activation of Raf-1-dependent AP-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <i>via</i> the activation of Erk1/2 by either form of stretch, only the transverse stretch-activated Raf-1-dependent AP-1 <span class="highlight" style="background-color:">pathway</span> regulates Ankrd2 gene expression by GTP-Ras, which is activated only by transverse stretch.</div></p></div><div class="aux"><div class="resc">Junaith S. Mohamed, et al. FASEB J. 2010 September;24(9):3330-3340.</div><p class="links"><a class="dblinks" href="/pubmed/20442316" ref="ordinalpos=1271&amp;ncbi_uid=3178441&amp;link_uid=20442316">Citation</a><a class="dblinks" href="/pmc/articles/PMC2923360/" ref="ordinalpos=1271&amp;ncbi_uid=3178441&amp;link_uid=PMC2923360">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1272.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3173392/figure/pone-0024361-g004/" ref="ordinalpos=1272&amp;ncbi_uid=3237438&amp;link_uid=PMC3173392" href="/pmc/articles/PMC3173392/"><img src="/pmc/articles/instance/3173392/bin/pone.0024361.g004.gif" src-large="/pmc/articles/instance/3173392/bin/pone.0024361.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3173392/" ref="ordinalpos=1272&amp;ncbi_uid=3237438&amp;link_uid=PMC3173392" image-link="/pmc/articles/PMC3173392/figure/pone-0024361-g004/" class="imagepopup">Figure 4. JNK <span class="highlight" style="background-color:">signaling</span> is activated upon misexpression of Aß42 in the eye..  From: Activation of JNK <span class="highlight" style="background-color:">Signaling</span> Mediates Amyloid-ss-Dependent Cell Death. </a></p><div class="supp"><p class="details"><div>(A) Schematic representation of hierarchy of Jun-kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members. (B, B′) Wild-type expression of <i>puc</i> in the developing third instar larval eye imaginal disc using lacZ reporter where reporter (red channel) is restricted only to the developing photoreceptors in the eye imaginal disc proper and in the peripodial membrane cells on the margin of the antennal disc [43]. (C, C′) GMR&gt;Aß42 eye imaginal disc exhibits ectopic upregulation of <i>puc</i>-lacZ reporter. (C′) Split channel showing ectopic <i>puc</i>-lacZ expression in the photoreceptor neurons of the eye imaginal disc. (D) Activation of JNK <span class="highlight" style="background-color:">signaling</span> in GMR&gt;Aß42 was detected by checking phospho-Jun levels. Levels of JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> increases three fold in GMR&gt;Aß42 as compared to the wild-type eye imaginal disc.</div></p></div><div class="aux"><div class="resc">Meghana Tare, et al. PLoS One. 2011;6(9):e24361.</div><p class="links"><a class="dblinks" href="/pubmed/21949710" ref="ordinalpos=1272&amp;ncbi_uid=3237438&amp;link_uid=21949710">Citation</a><a class="dblinks" href="/pmc/articles/PMC3173392/" ref="ordinalpos=1272&amp;ncbi_uid=3237438&amp;link_uid=PMC3173392">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1273.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3158608/figure/F7/" ref="ordinalpos=1273&amp;ncbi_uid=3445282&amp;link_uid=PMC3158608" href="/pmc/articles/PMC3158608/"><img src="/pmc/articles/instance/3158608/bin/nihms314773f7.gif" src-large="/pmc/articles/instance/3158608/bin/nihms314773f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3158608/" ref="ordinalpos=1273&amp;ncbi_uid=3445282&amp;link_uid=PMC3158608" image-link="/pmc/articles/PMC3158608/figure/F7/" class="imagepopup">Fig. 7. Schematic representation summarizing the cell-type dependent selectivity of CCK2 receptor <span class="highlight" style="background-color:">signaling</span>.  From: Cell-Type Specific CCK2 Receptor <span class="highlight" style="background-color:">Signaling</span> Underlies the Cholecystokinin-Mediated Selective Excitation of Hippocampal Parvalbumin-Positive Fast-Spiking Basket Cells. </a></p><div class="supp"><p class="details"><div>A, CCK signals through the same CCK2 receptor to utilize divergent <span class="highlight" style="background-color:">signaling</span> pathways and exert opposing effects on hippocampal basket cells, as described in panels B and C. B, In PV+ basket cells, CCK2 receptors couple to an unusual, pertussis-toxin-sensitive <span class="highlight" style="background-color:">pathway</span> and ryanodine receptors on calcium stores that ultimately activate a non-selective cationic conductance to depolarize the cells. C, CCK2 receptors signal through the more canonical G<sub>q</sub>-PLC <span class="highlight" style="background-color:">pathway</span> in pyramidal cells in conjunction with CB1 receptor <span class="highlight" style="background-color:">signaling</span> to regulate CCK+ basket cell transmission.</div></p></div><div class="aux"><div class="resc">Soo Yeun Lee, et al. J Neurosci. ;31(30):10993-11002.</div><p class="links"><a class="dblinks" href="/pubmed/21795548" ref="ordinalpos=1273&amp;ncbi_uid=3445282&amp;link_uid=21795548">Citation</a><a class="dblinks" href="/pmc/articles/PMC3158608/" ref="ordinalpos=1273&amp;ncbi_uid=3445282&amp;link_uid=PMC3158608">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1274.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2745827/figure/F2/" ref="ordinalpos=1274&amp;ncbi_uid=2042246&amp;link_uid=PMC2745827" href="/pmc/articles/PMC2745827/"><img src="/pmc/articles/instance/2745827/bin/nihms129629f2.gif" src-large="/pmc/articles/instance/2745827/bin/nihms129629f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2745827/" ref="ordinalpos=1274&amp;ncbi_uid=2042246&amp;link_uid=PMC2745827" image-link="/pmc/articles/PMC2745827/figure/F2/" class="imagepopup">Figure 2.  From: Genomic profiles of damage and protection in human intracerebral hemorrhage. </a></p><div class="supp"><p class="details"><div>Functional grouping of differentially expressed genes. The red line in the left graph indicates the threshold for a significant association, the −log(0.05). The right plot shows the percentage of genes within a canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that are differentially regulated by ICH versus the total number of genes in the database for that <span class="highlight" style="background-color:">pathway</span>. Green indicates the number that is downregulated; red is upregulated. The number at the right of the column indicates the total number of genes for that category in the database. All of the association in the canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are significant (Fisher’s exact <i>t</i>-test): the probability of finding the number of genes listed for each <span class="highlight" style="background-color:">pathway</span> as regulated in ICH out of all of the genes in the database for that <span class="highlight" style="background-color:">pathway</span> is greater than 0.05 for each class.</div></p></div><div class="aux"><div class="resc">S Thomas Carmichael, et al. J Cereb Blood Flow Metab. ;28(11):1860-1875.</div><p class="links"><a class="dblinks" href="/pubmed/18628781" ref="ordinalpos=1274&amp;ncbi_uid=2042246&amp;link_uid=18628781">Citation</a><a class="dblinks" href="/pmc/articles/PMC2745827/" ref="ordinalpos=1274&amp;ncbi_uid=2042246&amp;link_uid=PMC2745827">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1275.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3932036/figure/F5/" ref="ordinalpos=1275&amp;ncbi_uid=5837856&amp;link_uid=PMC3932036" href="/pmc/articles/PMC3932036/"><img src="/pmc/articles/instance/3932036/bin/1755-8794-7-4-5.gif" src-large="/pmc/articles/instance/3932036/bin/1755-8794-7-4-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3932036/" ref="ordinalpos=1275&amp;ncbi_uid=5837856&amp;link_uid=PMC3932036" image-link="/pmc/articles/PMC3932036/figure/F5/" class="imagepopup">Figure 5.  From: Kinome and mRNA expression profiling of high-grade osteosarcoma cell lines implies Akt <span class="highlight" style="background-color:">signaling</span> as possible target for therapy. </a></p><div class="supp"><p class="details"><div><b>Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in IPA. Blue: significantly lower, orange: significantly higher phosphorylation in osteosarcoma cell lines, gray, no significant difference in phosphorylation, white: no phosphorylation sites of the particular protein on the PamGene Ser/Thr chip. Blue lines indicate known downstream phosphorylation by the upstream kinase.</div></p></div><div class="aux"><div class="resc">Marieke L Kuijjer, et al. BMC Med Genomics. 2014;7:4-4.</div><p class="links"><a class="dblinks" href="/pubmed/24447333" ref="ordinalpos=1275&amp;ncbi_uid=5837856&amp;link_uid=24447333">Citation</a><a class="dblinks" href="/pmc/articles/PMC3932036/" ref="ordinalpos=1275&amp;ncbi_uid=5837856&amp;link_uid=PMC3932036">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1276.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2720598/figure/F4/" ref="ordinalpos=1276&amp;ncbi_uid=2040127&amp;link_uid=PMC2720598" href="/pmc/articles/PMC2720598/"><img src="/pmc/articles/instance/2720598/bin/nihms30595f4.gif" src-large="/pmc/articles/instance/2720598/bin/nihms30595f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2720598/" ref="ordinalpos=1276&amp;ncbi_uid=2040127&amp;link_uid=PMC2720598" image-link="/pmc/articles/PMC2720598/figure/F4/" class="imagepopup">Figure 4.  From: GRASP-1 is a neuronal scaffold protein for the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Caspase-3-cleavage is required for GRASP-1-mediated JNK <span class="highlight" style="background-color:">pathway</span> activation. (A–B) Full length GRASP-1 containing a mutated Caspase-3 cleavage site (F.L.-D/E) is a much less effective activator for JNK <span class="highlight" style="background-color:">signaling</span>. (A) Western blot and quantification show comparable expression levels of Myc-GRASP1-F.L.-D/E and Myc-GRASP1-C. n.s: not significant. N=3. (B) GRASP1-C but not GRASP1-F.L.-D/E enhances JNK phosphorylation. *: p&lt;0.05 (t-test), n=3. (C) A model that summarizes our findings that GRASP-1 serves as a scaffold protein for JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Bing Ye, et al. FEBS Lett. ;581(23):4403-4410.</div><p class="links"><a class="dblinks" href="/pubmed/17761173" ref="ordinalpos=1276&amp;ncbi_uid=2040127&amp;link_uid=17761173">Citation</a><a class="dblinks" href="/pmc/articles/PMC2720598/" ref="ordinalpos=1276&amp;ncbi_uid=2040127&amp;link_uid=PMC2720598">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1277.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3981814/figure/pone-0094496-g006/" ref="ordinalpos=1277&amp;ncbi_uid=6018333&amp;link_uid=PMC3981814" href="/pmc/articles/PMC3981814/"><img src="/pmc/articles/instance/3981814/bin/pone.0094496.g006.gif" src-large="/pmc/articles/instance/3981814/bin/pone.0094496.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3981814/" ref="ordinalpos=1277&amp;ncbi_uid=6018333&amp;link_uid=PMC3981814" image-link="/pmc/articles/PMC3981814/figure/pone-0094496-g006/" class="imagepopup">Figure 6. VEGF rescues vascular endothelial tube formation suppressed by PI3K <span class="highlight" style="background-color:">pathway</span> inhibitors..  From: Identification of a Role for the PI3K/AKT/mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Innate Immune Cells. </a></p><div class="supp"><p class="details"><div>(A) Photographs of the formation of tube-like structures. (B) Experiments were performed as in panel A, and the cumulative tube length was examined. a, b, c, and d indicate P&lt;0.05 versus Unstim, DMSO, LY, and AZD, respectively. (C) Diagram of the PI3K <span class="highlight" style="background-color:">signaling</span> in immune cells.</div></p></div><div class="aux"><div class="resc">Songbo Xie, et al. PLoS One. 2014;9(4):e94496.</div><p class="links"><a class="dblinks" href="/pubmed/24718556" ref="ordinalpos=1277&amp;ncbi_uid=6018333&amp;link_uid=24718556">Citation</a><a class="dblinks" href="/pmc/articles/PMC3981814/" ref="ordinalpos=1277&amp;ncbi_uid=6018333&amp;link_uid=PMC3981814">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1278.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4267749/figure/Fig1/" ref="ordinalpos=1278&amp;ncbi_uid=6967784&amp;link_uid=PMC4267749" href="/pmc/articles/PMC4267749/"><img src="/pmc/articles/instance/4267749/bin/13045_2014_87_Fig1_HTML.gif" src-large="/pmc/articles/instance/4267749/bin/13045_2014_87_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4267749/" ref="ordinalpos=1278&amp;ncbi_uid=6967784&amp;link_uid=PMC4267749" image-link="/pmc/articles/PMC4267749/figure/Fig1/" class="imagepopup">Figure 1.  From: Notch <span class="highlight" style="background-color:">signaling</span> and EMT in non-small cell lung cancer: biological significance and therapeutic application. </a></p><div class="supp"><p class="details"><div> <b>The <span class="highlight" style="background-color:">signaling</span> pathways in lung cancer.</b> Oncogene activation or loss of tumor suppressor initiates lung tumor formation. The Notch, Wnt and TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are involved in the induction of EMT, CSCs and metastasis of NSCLC.</div></p></div><div class="aux"><div class="resc">Xun Yuan, et al. J Hematol Oncol. 2014;7(1):87.</div><p class="links"><a class="dblinks" href="/pubmed/25477004" ref="ordinalpos=1278&amp;ncbi_uid=6967784&amp;link_uid=25477004">Citation</a><a class="dblinks" href="/pmc/articles/PMC4267749/" ref="ordinalpos=1278&amp;ncbi_uid=6967784&amp;link_uid=PMC4267749">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1279.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4216001/figure/pone-0110737-g008/" ref="ordinalpos=1279&amp;ncbi_uid=6807570&amp;link_uid=PMC4216001" href="/pmc/articles/PMC4216001/"><img src="/pmc/articles/instance/4216001/bin/pone.0110737.g008.gif" src-large="/pmc/articles/instance/4216001/bin/pone.0110737.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4216001/" ref="ordinalpos=1279&amp;ncbi_uid=6807570&amp;link_uid=PMC4216001" image-link="/pmc/articles/PMC4216001/figure/pone-0110737-g008/" class="imagepopup">Figure 8. Schematic illustration of a working model for Dab2 in modulation of TGF-beta <span class="highlight" style="background-color:">pathway</span>..  From: Hormonal Induction and Roles of Disabled-2 in Lactation and Involution. </a></p><div class="supp"><p class="details"><div>Canonical TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> consists of Smad phosphorylation and mediation of transcriptional regulation, leading to cell death and growth suppression, representing a tumor suppressor activity. The non-canonical route includes the activation of Ras/MAPK <span class="highlight" style="background-color:">pathway</span> as a result of phosphorylation and binding of TGF-beta receptor to Shc and consequently recruitment of Grb2 and Sos. TGF-beta-stimulated activation of Ras/MAPK <span class="highlight" style="background-color:">pathway</span> induces expression of genes involved in cell survival and growth. Dab2 modulates TGF-beta <span class="highlight" style="background-color:">signaling</span> by sequestering Grb2 from Sos, resulting in a reduction of Ras/MAPK activation yet allowing Smad-mediated gene transcription.</div></p></div><div class="aux"><div class="resc">Wensi Tao, et al. PLoS One. 2014;9(10):e110737.</div><p class="links"><a class="dblinks" href="/pubmed/25360623" ref="ordinalpos=1279&amp;ncbi_uid=6807570&amp;link_uid=25360623">Citation</a><a class="dblinks" href="/pmc/articles/PMC4216001/" ref="ordinalpos=1279&amp;ncbi_uid=6807570&amp;link_uid=PMC4216001">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1280.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC283607/figure/fig6/" ref="ordinalpos=1280&amp;ncbi_uid=319898&amp;link_uid=PMC283607" href="/pmc/articles/PMC283607/"><img src="/pmc/articles/instance/283607/bin/pq2336152006.gif" src-large="/pmc/articles/instance/283607/bin/pq2336152006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC283607/" ref="ordinalpos=1280&amp;ncbi_uid=319898&amp;link_uid=PMC283607" image-link="/pmc/articles/PMC283607/figure/fig6/" class="imagepopup">Fig. 6. From: Neurotrophin 3 activation of TrkC induces Schwann cell migration through the c-Jun N-terminal kinase <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic model for the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> coupling NT3/TrkC to Schwann cell migration. Details are described in <i>Discussion</i>.</div></p></div><div class="aux"><div class="resc">Junji Yamauchi, et al. Proc Natl Acad Sci U S A. 2003 November 25;100(24):14421-14426.</div><p class="links"><a class="dblinks" href="/pubmed/14614136" ref="ordinalpos=1280&amp;ncbi_uid=319898&amp;link_uid=14614136">Citation</a><a class="dblinks" href="/pmc/articles/PMC283607/" ref="ordinalpos=1280&amp;ncbi_uid=319898&amp;link_uid=PMC283607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1281.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3972149/figure/pone-0093547-g005/" ref="ordinalpos=1281&amp;ncbi_uid=5998340&amp;link_uid=PMC3972149" href="/pmc/articles/PMC3972149/"><img src="/pmc/articles/instance/3972149/bin/pone.0093547.g005.gif" src-large="/pmc/articles/instance/3972149/bin/pone.0093547.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3972149/" ref="ordinalpos=1281&amp;ncbi_uid=5998340&amp;link_uid=PMC3972149" image-link="/pmc/articles/PMC3972149/figure/pone-0093547-g005/" class="imagepopup">Figure 5. A proposed mechanism depicts CuB-induced actin aggregation and VASP activation via the Gα13/RhoA/PKA <span class="highlight" style="background-color:">pathway</span>..  From: VASP Activation via the G?13/RhoA/PKA <span class="highlight" style="background-color:">Pathway</span> Mediates Cucurbitacin-B-Induced Actin Aggregation and Cofilin-Actin Rod Formation. </a></p><div class="supp"><p class="details"><div>Dashed arrows and question marks indicate unidentifed/unconfirmed <span class="highlight" style="background-color:">signaling</span> components.</div></p></div><div class="aux"><div class="resc">Yan-Ting Zhang, et al. PLoS One. 2014;9(4):e93547.</div><p class="links"><a class="dblinks" href="/pubmed/24691407" ref="ordinalpos=1281&amp;ncbi_uid=5998340&amp;link_uid=24691407">Citation</a><a class="dblinks" href="/pmc/articles/PMC3972149/" ref="ordinalpos=1281&amp;ncbi_uid=5998340&amp;link_uid=PMC3972149">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1282.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3427288/figure/F5/" ref="ordinalpos=1282&amp;ncbi_uid=5149312&amp;link_uid=PMC3427288" href="/pmc/articles/PMC3427288/"><img src="/pmc/articles/instance/3427288/bin/epi-7-930-g5.gif" src-large="/pmc/articles/instance/3427288/bin/epi-7-930-g5.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3427288/" ref="ordinalpos=1282&amp;ncbi_uid=5149312&amp;link_uid=PMC3427288" image-link="/pmc/articles/PMC3427288/figure/F5/" class="imagepopup">Figure. From: The TCF4/?-catenin <span class="highlight" style="background-color:">pathway</span> and chromatin structure cooperate to regulate D-glucuronyl C5-epimerase expression in breast cancer. </a></p><div class="supp"><p class="details"><div><b>Figure 5.</b> Wnt <span class="highlight" style="background-color:">pathway</span> repression in TSA-treated MCF7 breast cancer cells. (A) Fold changes in the expression of 84 genes relevant to the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Fold change in normalized gene expression in the Test Sample (TSA-treated MCF7 cells) divided by the normalized gene expression in the Control Sample (MCF7 cells). The middle line shows similar expression in both groups with 2-fold change boundaries. (B) <span class="highlight" style="background-color:">Pathway</span> activity score for Wnt <span class="highlight" style="background-color:">signaling</span> in experimental samples relative to control samples.</div></p></div><div class="aux"><div class="resc">Luydmila A. Mostovich, et al. Epigenetics. 2012 August 1;7(8):930-939.</div><p class="links"><a class="dblinks" href="/pubmed/22805760" ref="ordinalpos=1282&amp;ncbi_uid=5149312&amp;link_uid=22805760">Citation</a><a class="dblinks" href="/pmc/articles/PMC3427288/" ref="ordinalpos=1282&amp;ncbi_uid=5149312&amp;link_uid=PMC3427288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1283.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285553/figure/F3/" ref="ordinalpos=1283&amp;ncbi_uid=4578123&amp;link_uid=PMC3285553" href="/pmc/articles/PMC3285553/"><img src="/pmc/articles/instance/3285553/bin/nihms350220f3.gif" src-large="/pmc/articles/instance/3285553/bin/nihms350220f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285553/" ref="ordinalpos=1283&amp;ncbi_uid=4578123&amp;link_uid=PMC3285553" image-link="/pmc/articles/PMC3285553/figure/F3/" class="imagepopup">Figure 3.  From: Dysregulation of Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> in Gastrointestinal Cancers. </a></p><div class="supp"><p class="details"><div>Additional mechanisms by which Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> can be modulated in cancer. (A) In hepatocytes, β-catenin can be released from an additional membrane bound pool associated with the receptor C-MET. When engaged by HGF (hepatocyte growth factor), C-MET releases β-catenin into the cytoplasm with its eventual translocation into the nucleus. (B) Membrane-bound NOTCH1 can bind activated β-catenin and cause its lysosomal destruction, thereby inhibiting Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. A decrease in NOTCH1 expression may therefore potentiate Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> in certain types of cancer. (C) Interactions with numerous proteins can modulate Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> at various levels in the <span class="highlight" style="background-color:">pathway</span>. SULF-1 (Sulfatase-1) can increase the efficiency of Wnt ligands by modulating their interactions with heparin sulfate proteoglycans in the extracellular environment. ATDC (Ataxia telangiectasia group D-complementing) protein can potentiate <span class="highlight" style="background-color:">pathway</span> activation through its interaction with DVL (Dishevelled). SMO (Smoothened) can increase Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> by an unknown mechanism. Undoubtedly, novel mechanisms will be uncovered to explain the complex context dependency of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Bryan D. White, et al. Gastroenterology. ;142(2):219-232.</div><p class="links"><a class="dblinks" href="/pubmed/22155636" ref="ordinalpos=1283&amp;ncbi_uid=4578123&amp;link_uid=22155636">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285553/" ref="ordinalpos=1283&amp;ncbi_uid=4578123&amp;link_uid=PMC3285553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1284.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2966483/figure/f2/" ref="ordinalpos=1284&amp;ncbi_uid=3582182&amp;link_uid=PMC2966483" href="/pmc/articles/PMC2966483/"><img src="/pmc/articles/instance/2966483/bin/fig-2.gif" src-large="/pmc/articles/instance/2966483/bin/fig-2.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2966483/" ref="ordinalpos=1284&amp;ncbi_uid=3582182&amp;link_uid=PMC2966483" image-link="/pmc/articles/PMC2966483/figure/f2/" class="imagepopup">FIG. 2. From: Regulation of NF-E2-Related Factor 2 <span class="highlight" style="background-color:">Signaling</span> for Cancer Chemoprevention: Antioxidant Coupled with Antiinflammatory. </a></p><div class="supp"><p class="details"><div>Chemical structures of selected chemopreventive agents with ability to activate the NF-E2-related factor 2–antioxidant responsive element <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Rong Hu, et al. Antioxid Redox Signal. 2010 December 1;13(11):1679-1698.</div><p class="links"><a class="dblinks" href="/pubmed/20486765" ref="ordinalpos=1284&amp;ncbi_uid=3582182&amp;link_uid=20486765">Citation</a><a class="dblinks" href="/pmc/articles/PMC2966483/" ref="ordinalpos=1284&amp;ncbi_uid=3582182&amp;link_uid=PMC2966483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1285.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3825194/figure/F001/" ref="ordinalpos=1285&amp;ncbi_uid=5493460&amp;link_uid=PMC3825194" href="/pmc/articles/PMC3825194/"><img src="/pmc/articles/instance/3825194/bin/ooomb4-21-323-g001.gif" src-large="/pmc/articles/instance/3825194/bin/ooomb4-21-323-g001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3825194/" ref="ordinalpos=1285&amp;ncbi_uid=5493460&amp;link_uid=PMC3825194" image-link="/pmc/articles/PMC3825194/figure/F001/" class="imagepopup">Fig. 1.TGF-β <span class="highlight" style="background-color:">signaling</span> through Smad-dependent and Smad-independent pathways. (A) Canonical <span class="highlight" style="background-color:">pathway</span>. TGF-β ligands activated TGF- βRI and TGF-βRII receptor complex recruits and phosphorylates the receptor specific Smad2/3. Hetero-oligomeric complex of Smad2/3- Smad4 translocates to the nucleus and binds to specific DNA sequence and interacts with transcription factors and cofactors to regulate transcription. The <span class="highlight" style="background-color:">pathway</span> is negatively regulated by the Smad6/7, which bind activated TGF-βRI, thereby preventing phosphorylation of Smad2/3, or recruit the E3 ubiquitine ligases to induce proteasomal degradation of the Smad2/3 (B) Noncanonical <span class="highlight" style="background-color:">pathway</span>. Smad-independent <span class="highlight" style="background-color:">signaling</span>. TGF-β can promote the activity of several <span class="highlight" style="background-color:">signaling</span> pathways other than Smad, including MAPKs, PI3K kinases, TRAF6- TAK1-p38/JNK, Rho-Rock, among others..  From: Targeting the Transforming Growth Factor-? <span class="highlight" style="background-color:">Signaling</span> in Cancer Therapy. </a></p><div class="aux"><div class="resc">Yhun Yhong Sheen, et al. Biomol Ther (Seoul). 2013 September 30;21(5):323-331.</div><p class="links"><a class="dblinks" href="/pubmed/24244818" ref="ordinalpos=1285&amp;ncbi_uid=5493460&amp;link_uid=24244818">Citation</a><a class="dblinks" href="/pmc/articles/PMC3825194/" ref="ordinalpos=1285&amp;ncbi_uid=5493460&amp;link_uid=PMC3825194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1286.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807379/figure/F6/" ref="ordinalpos=1286&amp;ncbi_uid=2213955&amp;link_uid=PMC2807379" href="/pmc/articles/PMC2807379/"><img src="/pmc/articles/instance/2807379/bin/nihms166563f6.gif" src-large="/pmc/articles/instance/2807379/bin/nihms166563f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807379/" ref="ordinalpos=1286&amp;ncbi_uid=2213955&amp;link_uid=PMC2807379" image-link="/pmc/articles/PMC2807379/figure/F6/" class="imagepopup">Figure 6. Inversin/NPHP2 mediates a switch from the canonical the non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which plays a role in planar cell polarity maintenance60.  From: Nephronophthisis: Disease Mechanisms of a Ciliopathy. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) This cartoon of arenal tubular epithelial cell shows how Wnt <span class="highlight" style="background-color:">signaling</span> occurs primarily through β-catenin–dependent pathways in the absence of urine flow. Ligand binding by the frizzled receptor results in inactivation of the β-catenin destruction complex through the presence of disheveled (Dvl), increased β-catenin levels, and upregulation of effector gene expression of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) Stimulation of the primary cilium, e.g. by urine flow, results in increased expression of inversin (Inv), which then reduces levels of cytoplasmic Dvl by increasing its proteasomal degradation. This allows reassembly and activation of the β-catenin destruction complex, thereby switching from the canonical to the non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The model is consistent with the finding that overexpression of β-catenin (equivalent to canonical Wnt <span class="highlight" style="background-color:">signaling</span>) leads to renal cysts in a mouse model140 (from: Germino <i>Nature Genet</i> 37:455, 2005)34.</div></p></div><div class="aux"><div class="resc">Friedhelm Hildebrandt, et al. J Am Soc Nephrol. ;20(1):23-35.</div><p class="links"><a class="dblinks" href="/pubmed/19118152" ref="ordinalpos=1286&amp;ncbi_uid=2213955&amp;link_uid=19118152">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807379/" ref="ordinalpos=1286&amp;ncbi_uid=2213955&amp;link_uid=PMC2807379">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1287.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3907022/figure/F7/" ref="ordinalpos=1287&amp;ncbi_uid=5752839&amp;link_uid=PMC3907022" href="/pmc/articles/PMC3907022/"><img src="/pmc/articles/instance/3907022/bin/1471-2164-14-831-7.gif" src-large="/pmc/articles/instance/3907022/bin/1471-2164-14-831-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3907022/" ref="ordinalpos=1287&amp;ncbi_uid=5752839&amp;link_uid=PMC3907022" image-link="/pmc/articles/PMC3907022/figure/F7/" class="imagepopup">Figure 7.  From: Clustering of transcriptional profiles identifies changes to insulin <span class="highlight" style="background-color:">signaling</span> as an early event in a mouse model of Alzheimer's disease. </a></p><div class="supp"><p class="details"><div><b>Transcript clustering of 5 months samples identified insulin <span class="highlight" style="background-color:">signaling</span> as a critical <span class="highlight" style="background-color:">pathway</span> activated in early stages of AD. (A)</b> Hierarchical clustering clustered three of the 5 months AD samples with the 4 months samples (now termed Stage 1), and the remaining four with the 6 months samples (now termed Stage 2). The 5 months AD samples are shown in red. <b>(B)</b> The 5 months samples in Stage 1 were compared with the 5 months samples in Stage 2 using Cuffdiff. A total of 187 genes were differentially expressed (47 upregulated, 187 downregulated, see Additional file 3). <b>(C)</b> <span class="highlight" style="background-color:">Pathway</span> analysis showed that both the ribosome (Rib.) and insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Ins. Sign.) were overrepresented in the DE genes (P &lt; 0.05). <b>(D)</b> Comparing Stage 1 with Stage 2, six genes in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were differentially expressed (3 upregulated, 3 downregulated).</div></p></div><div class="aux"><div class="resc">Harriet M Jackson, et al. BMC Genomics. 2013;14:831-831.</div><p class="links"><a class="dblinks" href="/pubmed/24274089" ref="ordinalpos=1287&amp;ncbi_uid=5752839&amp;link_uid=24274089">Citation</a><a class="dblinks" href="/pmc/articles/PMC3907022/" ref="ordinalpos=1287&amp;ncbi_uid=5752839&amp;link_uid=PMC3907022">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1288.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3480306/figure/fig6/" ref="ordinalpos=1288&amp;ncbi_uid=4318428&amp;link_uid=PMC3480306" href="/pmc/articles/PMC3480306/"><img src="/pmc/articles/instance/3480306/bin/PC_103267_wc_f6.gif" src-large="/pmc/articles/instance/3480306/bin/PC_103267_wc_f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3480306/" ref="ordinalpos=1288&amp;ncbi_uid=4318428&amp;link_uid=PMC3480306" image-link="/pmc/articles/PMC3480306/figure/fig6/" class="imagepopup">Figure 6. From: Two Direct Targets of Cytokinin <span class="highlight" style="background-color:">Signaling</span> Regulate Symbiotic Nodulation in Medicago truncatula[W][OA]. </a></p><div class="supp"><p class="details"><div>Downstream Cytokinin <span class="highlight" style="background-color:">Signaling</span> Events in Symbiotic Nodulation.<br />Schematic representation of a <i>M</i>. <i>truncatula</i> symbiotic nodule primordium, including selected known components of the <i>Rhizobium</i> Nod factor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Nod Factor Perception/Lys-M Kinase receptor [NFP/LYK], Does Not Make Infections/Calcium Calmodulin Protein Kinase [DMI3/CCaMK], Nod factor <span class="highlight" style="background-color:">Signaling</span> Pathway2 [NSP2]) and of the cytokinin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Cytokinin Response1 [CRE1], Response Regulators [RRs]). New genes identified in this study as acting downstream of the CRE1/RR cytokinin <span class="highlight" style="background-color:">pathway</span> are also indicated: the miR171 h isoform targeting NSP2 and the new direct cytokinin targets bHLH476 and CKX1. Proven interactions are indicated by full lines, and putative interactions by dotted lines.<br />Orange indicates epidermis, green indicates cortex, yellow indicates endodermis, red indicates pericycle, brown indicates stele, and blue indicates nodule primordium.</div></p></div><div class="aux"><div class="resc">Federico Ariel, et al. Plant Cell. 2012 September;24(9):3838-3852.</div><p class="links"><a class="dblinks" href="/pubmed/23023168" ref="ordinalpos=1288&amp;ncbi_uid=4318428&amp;link_uid=23023168">Citation</a><a class="dblinks" href="/pmc/articles/PMC3480306/" ref="ordinalpos=1288&amp;ncbi_uid=4318428&amp;link_uid=PMC3480306">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1289.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2827874/figure/F4/" ref="ordinalpos=1289&amp;ncbi_uid=2288067&amp;link_uid=PMC2827874" href="/pmc/articles/PMC2827874/"><img src="/pmc/articles/instance/2827874/bin/nihms-177952-f0004.gif" src-large="/pmc/articles/instance/2827874/bin/nihms-177952-f0004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2827874/" ref="ordinalpos=1289&amp;ncbi_uid=2288067&amp;link_uid=PMC2827874" image-link="/pmc/articles/PMC2827874/figure/F4/" class="imagepopup">Fig. 4.  From: From fate to function: the Drosophila trachea and salivary gland as models for tubulogenesis. </a></p><div class="supp"><p class="details"><div>Several <span class="highlight" style="background-color:">signaling</span> pathways have been implicated in tracheal migration, including endothelial growth factor (EGF), fibroblast growth factor (FGF), Dpp, and Wg. EGF <span class="highlight" style="background-color:">signaling</span> is localized by the expression of <i>rho</i>, which encodes a protease essential for the processing of the EGF ligand. EGF <span class="highlight" style="background-color:">signaling</span> is required for the complete internalization of the tracheal primordia. FGF <span class="highlight" style="background-color:">signaling</span> is limited by the expression of the ligand Bnl in the tissues toward which the tracheal branches migrate. FGF <span class="highlight" style="background-color:">signaling</span> is required for all migration. Dpp <span class="highlight" style="background-color:">signaling</span> is limited by the expression of the ligand Dpp in cells dorsal and ventral to the trachea. Dpp <span class="highlight" style="background-color:">signaling</span> is required for migration of the dorsal branch, lateral trunk, and ganglionic branch. Wg <span class="highlight" style="background-color:">signaling</span> is limited by the localization of the Wg signal in ectodermal cells near the tracheal primordia. Wg <span class="highlight" style="background-color:">signaling</span> is required to maintain <i>salm</i> expression in the dorsal trunk (DT) and to prevent the DT from migrating with the visceral branch. Green indicates the spatial position of the signal relative to the trachea. Blue indicates the tracheal primordia in the drawing along the top. Blue filled circles indicate that the cells have migrated to their correct positions in wild type and in the various <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutants. White circles indicate that cells are missing in particular branches in the mutants.</div></p></div><div class="aux"><div class="resc">Bilal E. Kerman, et al. Differentiation. ;74(7):326-348.</div><p class="links"><a class="dblinks" href="/pubmed/16916373" ref="ordinalpos=1289&amp;ncbi_uid=2288067&amp;link_uid=16916373">Citation</a><a class="dblinks" href="/pmc/articles/PMC2827874/" ref="ordinalpos=1289&amp;ncbi_uid=2288067&amp;link_uid=PMC2827874">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1290.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912096/figure/fig11/" ref="ordinalpos=1290&amp;ncbi_uid=5791595&amp;link_uid=PMC3912096" href="/pmc/articles/PMC3912096/"><img src="/pmc/articles/instance/3912096/bin/pp_231928_f11.gif" src-large="/pmc/articles/instance/3912096/bin/pp_231928_f11.jpg" alt="Figure 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912096/" ref="ordinalpos=1290&amp;ncbi_uid=5791595&amp;link_uid=PMC3912096" image-link="/pmc/articles/PMC3912096/figure/fig11/" class="imagepopup">Figure 11. From: The Endoplasmic Reticulum Binding Protein BiP Displays Dual Function in Modulating Cell Death Events1[W][OPEN]. </a></p><div class="supp"><p class="details"><div>BiP may modulate the NRP-mediated cell death <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by inhibiting UPR and stimulating SA <span class="highlight" style="background-color:">signaling</span>. The scheme illustrates the propagation of a cell death signal derived from prolonged ER, osmotic, and biotic stress through the NRP-mediated PCD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A broken arrow indicates an effect on gene expression, while a solid arrow indicates that the gene is an immediate downstream target. This investigation revealed that overexpression of BiP inhibits the UPR, the expression of NRPs, and GmNAC81 and as consequence attenuates cell death mediated by the NRP <span class="highlight" style="background-color:">signaling</span> as in leaf senescence. In response to a biotic stimulus, BiP positively modulates the SA <span class="highlight" style="background-color:">signaling</span> and activates the NRP-mediated cell death <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Humberto H. Carvalho, et al. Plant Physiol. 2014 February;164(2):654-670.</div><p class="links"><a class="dblinks" href="/pubmed/24319082" ref="ordinalpos=1290&amp;ncbi_uid=5791595&amp;link_uid=24319082">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912096/" ref="ordinalpos=1290&amp;ncbi_uid=5791595&amp;link_uid=PMC3912096">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1291.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3038240/figure/F8/" ref="ordinalpos=1291&amp;ncbi_uid=3116446&amp;link_uid=PMC3038240" href="/pmc/articles/PMC3038240/"><img src="/pmc/articles/instance/3038240/bin/nihms-267702-f0008.gif" src-large="/pmc/articles/instance/3038240/bin/nihms-267702-f0008.jpg" alt="Fig 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3038240/" ref="ordinalpos=1291&amp;ncbi_uid=3116446&amp;link_uid=PMC3038240" image-link="/pmc/articles/PMC3038240/figure/F8/" class="imagepopup">Fig 8. Diverse inputs and outputs of Jnk <span class="highlight" style="background-color:">signaling</span>.  From: Regulation of Hippo <span class="highlight" style="background-color:">signaling</span> by Jun kinase <span class="highlight" style="background-color:">signaling</span> during compensatory cell proliferation and regeneration, and in neoplastic tumors. </a></p><div class="supp"><p class="details"><div>Jnk, a stress activated protein kinase, is activated by diverse stimuli, including caspase activation, ligands for Jnk <span class="highlight" style="background-color:">signaling</span> pathways (Egr), cellular stresses (reactive oxygen species, infection, cytoskeletal changes), and loss of apical-basal polarity (Igaki, 2009; Karin and Gallagher, 2005). Jnk activates apoptosis, but also cell proliferation. Activation of proliferation is mediated through Hippo <span class="highlight" style="background-color:">signaling</span>. The ultimate target of Jnk <span class="highlight" style="background-color:">signaling</span> within the Hippo <span class="highlight" style="background-color:">pathway</span> is not known, it could act upstream of Wts and Hpo, but might act in parallel to them (dashed lines and question mark). Yki both promotes cell proliferation and inhibits apoptosis through transcriptional regulation of downstream target genes.</div></p></div><div class="aux"><div class="resc">Gongping Sun, et al. Dev Biol. ;350(1):139-151.</div><p class="links"><a class="dblinks" href="/pubmed/21145886" ref="ordinalpos=1291&amp;ncbi_uid=3116446&amp;link_uid=21145886">Citation</a><a class="dblinks" href="/pmc/articles/PMC3038240/" ref="ordinalpos=1291&amp;ncbi_uid=3116446&amp;link_uid=PMC3038240">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1292.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3215849/figure/F7/" ref="ordinalpos=1292&amp;ncbi_uid=3409019&amp;link_uid=PMC3215849" href="/pmc/articles/PMC3215849/"><img src="/pmc/articles/instance/3215849/bin/nihms332032f7.gif" src-large="/pmc/articles/instance/3215849/bin/nihms332032f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3215849/" ref="ordinalpos=1292&amp;ncbi_uid=3409019&amp;link_uid=PMC3215849" image-link="/pmc/articles/PMC3215849/figure/F7/" class="imagepopup">Figure 7. Model for regulation of Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in a post-mitotic fate decision.  From: Binary Regulation of Hippo <span class="highlight" style="background-color:">Pathway</span> by Merlin/NF2, Kibra, Lgl, and Melted Specifies and Maintains Post-mitotic Neuronal Fate. </a></p><div class="supp"><p class="details"><div>(A–C′) All panels show adult retinas stained for Rh5 (blue) and Rh6 (red).<br />(A) <i>PanR8&gt;melt. melt</i> expressed immediately after subtype-specific Rh expression can reset the bistable feedback loop and induce the pR8 fate.<br />(A′) close up of (A). Note increase in Rh5-only expressing pR8s and decrease in Rh6-expressing yR8 cells.<br />(B) ectopic <i>melt</i> expressed in adult flies (see methods).<br />(B′) close-up of (B). Note that Rh6 staining appears weaker in cells that co-express Rh5 and Rh6.<br />(C) Rh5 of (B).<br />(C′) Rh5 of (B′).<br />(D) Model of Hippo <span class="highlight" style="background-color:">pathway</span> regulation for binary fate specification of R8. Constitutive and continuous upstream <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> by Mer, Kib, and Lgl promotes yR8 fate, while repression of <i>warts</i> by Melt disrupts Hippo <span class="highlight" style="background-color:">signaling</span> to specify pR8 fate. Genes in black or colored font are expressed and active in that particular cell; genes in gray font are not expressed or not active.</div></p></div><div class="aux"><div class="resc">David Jukam, et al. Dev Cell. ;21(5):874-887.</div><p class="links"><a class="dblinks" href="/pubmed/22055343" ref="ordinalpos=1292&amp;ncbi_uid=3409019&amp;link_uid=22055343">Citation</a><a class="dblinks" href="/pmc/articles/PMC3215849/" ref="ordinalpos=1292&amp;ncbi_uid=3409019&amp;link_uid=PMC3215849">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1293.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3478512/figure/Fig3/" ref="ordinalpos=1293&amp;ncbi_uid=4326421&amp;link_uid=PMC3478512" href="/pmc/articles/PMC3478512/"><img src="/pmc/articles/instance/3478512/bin/18_2012_991_Fig3_HTML.gif" src-large="/pmc/articles/instance/3478512/bin/18_2012_991_Fig3_HTML.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3478512/" ref="ordinalpos=1293&amp;ncbi_uid=4326421&amp;link_uid=PMC3478512" image-link="/pmc/articles/PMC3478512/figure/Fig3/" class="imagepopup">Fig. 3.  From: Current perspectives of the <span class="highlight" style="background-color:">signaling</span> pathways directing neural crest induction. </a></p><div class="supp"><p class="details"><div>Temporal and spatial participation of <span class="highlight" style="background-color:">signaling</span> molecules involved in <i>Xenopus</i> neural crest induction. <b>a</b> Timeline of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation and requirement in early NC development.<i> Closed arrows</i>/<i>lines</i> indicate activation and requirement in NC tissues. <i>Dotted lines</i> indicate activation in NC tissues, but requirement is unknown. BMP/Smad <span class="highlight" style="background-color:">signaling</span> must be inhibited during gastrulation, but activated upon neurulation. The specific stage when Smad <span class="highlight" style="background-color:">signaling</span> first becomes activated in the NC has not been determined. FGF/Erk <span class="highlight" style="background-color:">signaling</span> is activated throughout early NC development, but has only been functionally implicated during early gastrula stages [49, 50]. Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> is thought to be required at all stages of early NC development, but becomes more strongly activated by neurulation [31]. The precise time when Notch is required is still debated, but may play a role in the initial induction during gastrulation. <b>b</b> Spatial activation of BMP, FGF, Wnt, and Notch <span class="highlight" style="background-color:">signaling</span> during <i>Xenopus</i> gastrulation and neurulation. BMP/Smad, FGF/Erk, and Wnt/β-catenin activation based on data from [199, 200]. Notch/Delta activation inferred from requirements in germ layer segregation and NC development [8, 12, 201]. Overall, spatiotemporal activation of these pathways is conserved between <i>Xenopus</i> and zebrafish. <b>c</b> Spatial expression and participation of <span class="highlight" style="background-color:">signaling</span> molecules in <i>Xenopus</i> neural crest induction at the gastrula stage. Diagram corresponds to <i>dotted box</i> of stage 10 gastrula in <b>b</b>. Molecules in bold have support from multiple studies. <i>Solid lines</i> indicate known relationships, <i>dotted lines</i> indicate potential relationships. NC induction results from the combined action of Wnt/β-catenin, FGF, Indian Hedgehog, and non-canonical Wnt <span class="highlight" style="background-color:">signaling</span>. Fgf8a is thought to regulate the expression of Wnt8 in the dorsolateral marginal zone (DLMZ), but may signal to the prospective neural crest itself. Wnt8 and Wnt3a <span class="highlight" style="background-color:">signaling</span> from the DLMZ activate canonical Wnt <span class="highlight" style="background-color:">signaling</span> in the prospective neural crest. Multiple agonists and antagonists of BMP and Wnt <span class="highlight" style="background-color:">signaling</span> are expressed in the Organizer, DMZ, and DLMZ and function in dorsal–ventral and anterior–posterior patterning, and these molecules likely also participate in NC induction (<i>dotted line</i>). Expression of other potential <span class="highlight" style="background-color:">signaling</span> molecules and regulators is presented. See main text for details on the participation of individual <span class="highlight" style="background-color:">signaling</span> molecules. <b>d</b> Participation of <span class="highlight" style="background-color:">signaling</span> molecules in the maintenance of NC progenitors in <i>Xenopus</i> neurulation. Diagram corresponds to section at <i>dotted line</i> in stage 14 neurula in <b>b</b>. NC maintenance requires activation of Smad1/5/8, Wnt/β-catenin, Notch/Delta, Indian Hedgehog, and Endothelin-A <span class="highlight" style="background-color:">signaling</span>. BMP and Wnt signals are likely mediated by Bmp4, Bmp7, Wnt1, and Wnt8, expressed in the neural folds upon neurulation. Additionally, Wnt8 is present in the paraxial mesoderm and Wnt3a in the neural plate. Notch <span class="highlight" style="background-color:">signaling</span> is thought to operate both by regulating Bmp ligand levels and leading to the expression of NC specifiers. See main text for details on the participation of individual <span class="highlight" style="background-color:">signaling</span> molecules. Expression data gathered from references in text and from http://www.xenbase.org</div></p></div><div class="aux"><div class="resc">Timothy J. Stuhlmiller, et al. Cell Mol Life Sci. 2012 November;69(22):3715-3737.</div><p class="links"><a class="dblinks" href="/pubmed/22547091" ref="ordinalpos=1293&amp;ncbi_uid=4326421&amp;link_uid=22547091">Citation</a><a class="dblinks" href="/pmc/articles/PMC3478512/" ref="ordinalpos=1293&amp;ncbi_uid=4326421&amp;link_uid=PMC3478512">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1294.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2676447/figure/F5/" ref="ordinalpos=1294&amp;ncbi_uid=1798553&amp;link_uid=PMC2676447" href="/pmc/articles/PMC2676447/"><img src="/pmc/articles/instance/2676447/bin/nihms87682f5.gif" src-large="/pmc/articles/instance/2676447/bin/nihms87682f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2676447/" ref="ordinalpos=1294&amp;ncbi_uid=1798553&amp;link_uid=PMC2676447" image-link="/pmc/articles/PMC2676447/figure/F5/" class="imagepopup">Fig. 5.  From: Growth compensatory role of sulindac sulfide-induced thrombospondin-1 linked with ERK1/2 and RhoA GTPase <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Proposed contribution of TSP-1 linked with ERK1/2 and RhoA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in NSAID-treated cells. Sulindac sulfide usually causes cellular growth suppression or cytotoxicity. As the compensatory mechanism, it also induces TSP-1 expression via ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Moreover, activated RhoA signals can enhance TSP-1 induction levels. All of the compensatory signals elevated cellular viability under the cytotoxic NSAID action.</div></p></div><div class="aux"><div class="resc">Yuseok Moon, et al. Life Sci. ;82(11-12):591-599.</div><p class="links"><a class="dblinks" href="/pubmed/18261746" ref="ordinalpos=1294&amp;ncbi_uid=1798553&amp;link_uid=18261746">Citation</a><a class="dblinks" href="/pmc/articles/PMC2676447/" ref="ordinalpos=1294&amp;ncbi_uid=1798553&amp;link_uid=PMC2676447">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1295.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4049898/figure/F2/" ref="ordinalpos=1295&amp;ncbi_uid=6344786&amp;link_uid=PMC4049898" href="/pmc/articles/PMC4049898/"><img src="/pmc/articles/instance/4049898/bin/org-10-86-g2.gif" src-large="/pmc/articles/instance/4049898/bin/org-10-86-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4049898/" ref="ordinalpos=1295&amp;ncbi_uid=6344786&amp;link_uid=PMC4049898" image-link="/pmc/articles/PMC4049898/figure/F2/" class="imagepopup">Figure. From: Wnt and planar cell polarity <span class="highlight" style="background-color:">signaling</span> in cystic renal disease. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Ciliogenesis and Wnt <span class="highlight" style="background-color:">signaling</span> in the kidney. Ciliogenesis depends on anterograde and retrograde transport involving kinesin II, dynein 2 and IFT proteins. The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (left side of figure) initiates when a Wnt ligand binds to a Frizzled receptor in the presence of LRP. This activates Dishevelled that inhibits the β-catenin destruction complex (GSK3β, APC, Axin). In the active <span class="highlight" style="background-color:">pathway</span>, β-catenin accumulates in the cytosol and causes the transcriptional activation of Wnt target genes. In the non-canonical Wnt/PCP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the kidney (right side of figure), the binding of a Wnt ligand will result in the activation of Frizzled and Dishevelled, leading to downstream cytoskeletal rearrangements or transcriptional activation through the activation of RhoA or Rac1. Core molecules (Vangl, Celsr, Scribble) are important for non-canonical Wnt/PCP <span class="highlight" style="background-color:">signaling</span>, while effector PCP molecules (Fuzzy, Fat4, Ptk7) are significant for kidney morphogenesis. A number of Wnt <span class="highlight" style="background-color:">signaling</span> proteins localize to the basal body (Dishevelled, β-catenin, Fat4) or base (Inversin, Dishevelled, Frizzled, Vangl) of kidney primary cilia.</div></p></div><div class="aux"><div class="resc">Paraskevi Goggolidou. Organogenesis. 2014 January 1;10(1):86-95.</div><p class="links"><a class="dblinks" href="/pubmed/24162855" ref="ordinalpos=1295&amp;ncbi_uid=6344786&amp;link_uid=24162855">Citation</a><a class="dblinks" href="/pmc/articles/PMC4049898/" ref="ordinalpos=1295&amp;ncbi_uid=6344786&amp;link_uid=PMC4049898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1296.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2779086/figure/f1/" ref="ordinalpos=1296&amp;ncbi_uid=2134519&amp;link_uid=PMC2779086" href="/pmc/articles/PMC2779086/"><img src="/pmc/articles/instance/2779086/bin/msb200972-f1.gif" src-large="/pmc/articles/instance/2779086/bin/msb200972-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2779086/" ref="ordinalpos=1296&amp;ncbi_uid=2134519&amp;link_uid=PMC2779086" image-link="/pmc/articles/PMC2779086/figure/f1/" class="imagepopup">Figure 1.  From: Application of an integrated physical and functional screening approach to identify inhibitors of the Wnt <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic of an integrated, multi-dimensional approach to identify <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components. Genes harboring cell <span class="highlight" style="background-color:">signaling</span>-associated domains were randomly selected and then used to generate libraries of corresponding Flag-tagged cDNAs and siRNAs. These were then used in three high-throughput screens to map protein–protein interactions by LUMIER and to examine the effect of altering protein expression levels on a transcriptional reporter. The data from the three screens were integrated to derive a combined <span class="highlight" style="background-color:">pathway</span> score (CPS), which predicts the likelihood that a protein is a component of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> being studied.</div></p></div><div class="aux"><div class="resc">Bryan W Miller, et al. Mol Syst Biol. 2009;5:315-315.</div><p class="links"><a class="dblinks" href="/pubmed/19888210" ref="ordinalpos=1296&amp;ncbi_uid=2134519&amp;link_uid=19888210">Citation</a><a class="dblinks" href="/pmc/articles/PMC2779086/" ref="ordinalpos=1296&amp;ncbi_uid=2134519&amp;link_uid=PMC2779086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1297.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1196350/figure/fig9/" ref="ordinalpos=1297&amp;ncbi_uid=603236&amp;link_uid=PMC1196350" href="/pmc/articles/PMC1196350/"><img src="/pmc/articles/instance/1196350/bin/zmk0090573040009.gif" src-large="/pmc/articles/instance/1196350/bin/zmk0090573040009.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1196350/" ref="ordinalpos=1297&amp;ncbi_uid=603236&amp;link_uid=PMC1196350" image-link="/pmc/articles/PMC1196350/figure/fig9/" class="imagepopup">Figure 9. From: Calcium-activated RAF/MEK/ERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Mediates p53-dependent Apoptosis and Is Abrogated by ?B-Crystallin through Inhibition of RAS Activation. </a></p><div class="supp"><p class="details"><div>Diagram to show a novel <span class="highlight" style="background-color:">pathway</span> for calcimycin-induced apoptosis in N/N1003A cells, also a novel protective mechanism of αB-crystallin against calcimycin-induced apoptosis and the partial explanation why the ocular lens lacks spontaneous tumor. Calcimycin induces an increase of the intracellular calcium, which induces RAS-GRF2 translocation from cytosol to plasma membrane to activate RAS. RAS then activates RAF1, leading to activation of the downstream MEK1/2, which activates ERK1/2. Activation of this <span class="highlight" style="background-color:">pathway</span> triggers p53 up-regulation and activation, which through regulation of Bax, signals the mitochondrial death <span class="highlight" style="background-color:">pathway</span> as reflected by the release of cytochrome <i>c</i> followed by activation of caspase-3. In lens epithelial cells (LECs), stress-activated RAS/RAF/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates apoptosis, leading to elimination of cells with genome lesion. In contrast, in most other cells, the same <span class="highlight" style="background-color:">pathway</span> activated by stress promotes survival, allowing accumulation of cells with damaged DNA, which likely develops into cancer. This partially explains why the ocular lens lacks spontaneous tumor. αB-crystallin suppresses calcimycin-induced RAS-GRF2 translocation to block RAS activation and thus suppress the activation of ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Through repression of ERK1/2 activation, it prevents downstream apoptotic events including calcimycin-induced up-regulation of p53 and Bax, release of cytochrome <i>c</i> and activation of caspase-3. Up-arrow stands for the increase of intracellular calcium (Ca<sup>2+</sup>), or up-regulation of enzyme activity (RAS, RAF1, MEK1/2, ERK1/2, and capase-3), or up-regulation of protein (p53, Bax, and cytochrome <i>c</i>).</div></p></div><div class="aux"><div class="resc">David Wan-Cheng Li, et al. Mol Biol Cell. 2005 September;16(9):4437-4453.</div><p class="links"><a class="dblinks" href="/pubmed/16000378" ref="ordinalpos=1297&amp;ncbi_uid=603236&amp;link_uid=16000378">Citation</a><a class="dblinks" href="/pmc/articles/PMC1196350/" ref="ordinalpos=1297&amp;ncbi_uid=603236&amp;link_uid=PMC1196350">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1298.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3924831/figure/genes-04-00306-f002/" ref="ordinalpos=1298&amp;ncbi_uid=5947941&amp;link_uid=PMC3924831" href="/pmc/articles/PMC3924831/"><img src="/pmc/articles/instance/3924831/bin/genes-04-00306-g002.gif" src-large="/pmc/articles/instance/3924831/bin/genes-04-00306-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3924831/" ref="ordinalpos=1298&amp;ncbi_uid=5947941&amp;link_uid=PMC3924831" image-link="/pmc/articles/PMC3924831/figure/genes-04-00306-f002/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Signaling</span> Pathways from the Endoplasmic Reticulum and Their Roles in Disease. </a></p><div class="supp"><p class="details"><div>Apoptosis <span class="highlight" style="background-color:">signaling</span> under ER stress conditions. Prolonged or severe ER stress, as well as dysfunction of the unfolded protein response (UPR), induces apoptosis <span class="highlight" style="background-color:">signaling</span>, primarily through the IRE1 and PERK pathways. In the IRE1 <span class="highlight" style="background-color:">pathway</span>, activated IRE1 recruits TRAF2 and ASK1 on the ER membrane and activates the ASK1-dependent apoptosis <span class="highlight" style="background-color:">pathway</span>. In addition, the IKK-NFκB <span class="highlight" style="background-color:">pathway</span> is also activated by IRE1-TRAF2 and induces an apoptotic response. Proapoptotic Bcl-2 family members, Bax and Bak, interact with IRE1 and promote its RNase/kinase activity. Moreover, IRE1 induces ER-localized mRNA degradation. In the PERK <span class="highlight" style="background-color:">pathway</span>, ATF4 induced by the PERK-eIF2α <span class="highlight" style="background-color:">pathway</span> upregulates the expression of CHOP, which in turn activates the transcription of GADD34, ER oxidoreductase 1 (ERO1), and many proapoptotic factors. GADD34 then promotes dephosphorylation of eIF2α with PP1, canceling translational attenuation, and leads to an increase of protein loads into the ER. Furthermore, the translational attenuation of global proteins by the PERK-eIF2α <span class="highlight" style="background-color:">pathway</span> also applies to IκB, which has been terminally linked to the activation of NFκB.</div></p></div><div class="aux"><div class="resc">Hisae Kadowaki, et al. Genes (Basel). 2013 September;4(3):306-333.</div><p class="links"><a class="dblinks" href="/pubmed/24705207" ref="ordinalpos=1298&amp;ncbi_uid=5947941&amp;link_uid=24705207">Citation</a><a class="dblinks" href="/pmc/articles/PMC3924831/" ref="ordinalpos=1298&amp;ncbi_uid=5947941&amp;link_uid=PMC3924831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1299.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3666883/figure/F11/" ref="ordinalpos=1299&amp;ncbi_uid=5157990&amp;link_uid=PMC3666883" href="/pmc/articles/PMC3666883/"><img src="/pmc/articles/instance/3666883/bin/zmb9991098360011.gif" src-large="/pmc/articles/instance/3666883/bin/zmb9991098360011.jpg" alt="Fig 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3666883/" ref="ordinalpos=1299&amp;ncbi_uid=5157990&amp;link_uid=PMC3666883" image-link="/pmc/articles/PMC3666883/figure/F11/" class="imagepopup">Fig 11.  From: Sonic hedgehog-Dependent Induction of MicroRNA 31 and MicroRNA 150 Regulates Mycobacterium bovis BCG-Driven Toll-Like Receptor 2 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Model. M. bovis BCG, among most of the common bacterial pathogens, predominantly activates SHH <span class="highlight" style="background-color:">signaling</span> through stimulation of the TLR2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The PI3K/PKC/MAPK <span class="highlight" style="background-color:">signaling</span> cohort holds the capacity to activate NF-κB, which leads to upregulation of SHH, GLI1/2, SMO, and PTCH1. The robust production of TNF-α by M. bovis BCG-infected macrophages amplifies SHH <span class="highlight" style="background-color:">signaling</span> activation to constitute a feedback loop which plays a central role in orchestrating TLR2 responses via miR-31- and miR-150-targeted MyD88. RISC,RNA-induced silencing complex; TNFR, TNF-α receptor.</div></p></div><div class="aux"><div class="resc">Devram Sampat Ghorpade, et al. Mol Cell Biol. 2013 February;33(3):543-556.</div><p class="links"><a class="dblinks" href="/pubmed/23166298" ref="ordinalpos=1299&amp;ncbi_uid=5157990&amp;link_uid=23166298">Citation</a><a class="dblinks" href="/pmc/articles/PMC3666883/" ref="ordinalpos=1299&amp;ncbi_uid=5157990&amp;link_uid=PMC3666883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1300.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634825/figure/pone-0061946-g002/" ref="ordinalpos=1300&amp;ncbi_uid=4866741&amp;link_uid=PMC3634825" href="/pmc/articles/PMC3634825/"><img src="/pmc/articles/instance/3634825/bin/pone.0061946.g002.gif" src-large="/pmc/articles/instance/3634825/bin/pone.0061946.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634825/" ref="ordinalpos=1300&amp;ncbi_uid=4866741&amp;link_uid=PMC3634825" image-link="/pmc/articles/PMC3634825/figure/pone-0061946-g002/" class="imagepopup">Figure 2. Modified “TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” from KEGG..  From: Transcriptome Analysis of Skin Photoaging in Chinese Females Reveals the Involvement of Skin Homeostasis and Metabolic Changes. </a></p><div class="supp"><p class="details"><div>Protein symbols were replaced by gene symbols to reflect gene-based data. For better legibility, gene symbols were not italicized. See the online information http://www.genome.jp/kegg/document/help_<span class="highlight" style="background-color:">pathway</span>.html for detailed KEGG <span class="highlight" style="background-color:">pathway</span> map notation.</div></p></div><div class="aux"><div class="resc">Wei Yan, et al. PLoS One. 2013;8(4):e61946.</div><p class="links"><a class="dblinks" href="/pubmed/23637934" ref="ordinalpos=1300&amp;ncbi_uid=4866741&amp;link_uid=23637934">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634825/" ref="ordinalpos=1300&amp;ncbi_uid=4866741&amp;link_uid=PMC3634825">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="12" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="13" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="14" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:24:25-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A65AA53FEC7010000000000128346;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A65AA53FEC7010000000000128346" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>