
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="43" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A4B6254312E810000000000669C4C" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="42" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="43" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="44" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="43" /></div>
            <div><h2 class="result_count">Results: 4201 to 4300 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4201.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2975164/figure/F7/" ref="ordinalpos=4201&amp;ncbi_uid=3353976&amp;link_uid=PMC2975164" href="/pmc/articles/PMC2975164/"><img src="/pmc/articles/instance/2975164/bin/zbc0481038360007.gif" src-large="/pmc/articles/instance/2975164/bin/zbc0481038360007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2975164/" ref="ordinalpos=4201&amp;ncbi_uid=3353976&amp;link_uid=PMC2975164" image-link="/pmc/articles/PMC2975164/figure/F7/" class="imagepopup">FIGURE 7. From: Modification of Cysteine 179 of I?B? Kinase by Nimbolide Leads to Down-regulation of NF-?B-regulated Cell Survival and Proliferative Proteins and Sensitization of Tumor Cells to Chemotherapeutic Agents. </a></p><div class="supp"><p class="details"><div><b>A schematic diagram showing nimbolide (<i>NL</i>) activity in the TNF-α-induced NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b></div></p></div><div class="aux"><div class="resc">Subash C. Gupta, et al. J Biol Chem. 2010 November 12;285(46):35406-35417.</div><p class="links"><a class="dblinks" href="/pubmed/20829362" ref="ordinalpos=4201&amp;ncbi_uid=3353976&amp;link_uid=20829362">Citation</a><a class="dblinks" href="/pmc/articles/PMC2975164/" ref="ordinalpos=4201&amp;ncbi_uid=3353976&amp;link_uid=PMC2975164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4202.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2697172/figure/F5/" ref="ordinalpos=4202&amp;ncbi_uid=1831256&amp;link_uid=PMC2697172" href="/pmc/articles/PMC2697172/"><img src="/pmc/articles/instance/2697172/bin/1471-2164-10-220-5.gif" src-large="/pmc/articles/instance/2697172/bin/1471-2164-10-220-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2697172/" ref="ordinalpos=4202&amp;ncbi_uid=1831256&amp;link_uid=PMC2697172" image-link="/pmc/articles/PMC2697172/figure/F5/" class="imagepopup">Figure 5.  From: A network-based integrative approach to prioritize reliable hits from multiple genome-wide RNAi screens in Drosophila. </a></p><div class="supp"><p class="details"><div><b>The proportion of OT-related hits within each interval of the RR by NePhe score for Hh (a) and Wnt (b) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Li Wang, et al. BMC Genomics. 2009;10:220-220.</div><p class="links"><a class="dblinks" href="/pubmed/19435510" ref="ordinalpos=4202&amp;ncbi_uid=1831256&amp;link_uid=19435510">Citation</a><a class="dblinks" href="/pmc/articles/PMC2697172/" ref="ordinalpos=4202&amp;ncbi_uid=1831256&amp;link_uid=PMC2697172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4203.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2708566/figure/f9/" ref="ordinalpos=4203&amp;ncbi_uid=1985245&amp;link_uid=PMC2708566" href="/pmc/articles/PMC2708566/"><img src="/pmc/articles/instance/2708566/bin/zii0070980230009.gif" src-large="/pmc/articles/instance/2708566/bin/zii0070980230009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2708566/" ref="ordinalpos=4203&amp;ncbi_uid=1985245&amp;link_uid=PMC2708566" image-link="/pmc/articles/PMC2708566/figure/f9/" class="imagepopup">FIG. 9. From: Novel Subtilase Cytotoxin Produced by Shiga-Toxigenic Escherichia coli Induces Apoptosis in Vero Cells via Mitochondrial Membrane Damage . </a></p><div class="supp"><p class="details"><div>Proposed model of the SubAB-induced apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. See text for additional details.</div></p></div><div class="aux"><div class="resc">Gen Matsuura, et al. Infect Immun. 2009 July;77(7):2919-2924.</div><p class="links"><a class="dblinks" href="/pubmed/19380466" ref="ordinalpos=4203&amp;ncbi_uid=1985245&amp;link_uid=19380466">Citation</a><a class="dblinks" href="/pmc/articles/PMC2708566/" ref="ordinalpos=4203&amp;ncbi_uid=1985245&amp;link_uid=PMC2708566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4204.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2526161/figure/fig4/" ref="ordinalpos=4204&amp;ncbi_uid=1543936&amp;link_uid=PMC2526161" href="/pmc/articles/PMC2526161/"><img src="/pmc/articles/instance/2526161/bin/PPAR2008-358052.004.gif" src-large="/pmc/articles/instance/2526161/bin/PPAR2008-358052.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2526161/" ref="ordinalpos=4204&amp;ncbi_uid=1543936&amp;link_uid=PMC2526161" image-link="/pmc/articles/PMC2526161/figure/fig4/" class="imagepopup">Figure 4.  From: Omega-3 Fatty Acids and PPARγ in Cancer. </a></p><div class="supp"><p class="details"><div>The syndecan-1 <span class="highlight" style="background-color:">pathway</span> for n-3 PUFA induction of apoptosis. Dashed lines indicate that effects may be indirect with involvement of other metabolites and <span class="highlight" style="background-color:">signaling</span> molecules.</div></p></div><div class="aux"><div class="resc">Iris J. Edwards, et al. PPAR Res. 2008;2008:358052.</div><p class="links"><a class="dblinks" href="/pubmed/18769551" ref="ordinalpos=4204&amp;ncbi_uid=1543936&amp;link_uid=18769551">Citation</a><a class="dblinks" href="/pmc/articles/PMC2526161/" ref="ordinalpos=4204&amp;ncbi_uid=1543936&amp;link_uid=PMC2526161">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4205.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2259313/figure/F1/" ref="ordinalpos=4205&amp;ncbi_uid=1253467&amp;link_uid=PMC2259313" href="/pmc/articles/PMC2259313/"><img src="/pmc/articles/instance/2259313/bin/1742-4682-4-50-1.gif" src-large="/pmc/articles/instance/2259313/bin/1742-4682-4-50-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2259313/" ref="ordinalpos=4205&amp;ncbi_uid=1253467&amp;link_uid=PMC2259313" image-link="/pmc/articles/PMC2259313/figure/F1/" class="imagepopup">Figure 1.  From: Simulating non-small cell lung cancer with a multiscale agent-based model. </a></p><div class="supp"><p class="details"><div><b>Kinetic model of the NSCLC-specific EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The <i>arrows </i>represent the reactions specified in Tables 1 and 2.</div></p></div><div class="aux"><div class="resc">Zhihui Wang, et al. Theor Biol Med Model. 2007;4:50-50.</div><p class="links"><a class="dblinks" href="/pubmed/18154660" ref="ordinalpos=4205&amp;ncbi_uid=1253467&amp;link_uid=18154660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2259313/" ref="ordinalpos=4205&amp;ncbi_uid=1253467&amp;link_uid=PMC2259313">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4206.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3075804/figure/F4/" ref="ordinalpos=4206&amp;ncbi_uid=3568547&amp;link_uid=PMC3075804" href="/pmc/articles/PMC3075804/"><img src="/pmc/articles/instance/3075804/bin/nihms253610f4.gif" src-large="/pmc/articles/instance/3075804/bin/nihms253610f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3075804/" ref="ordinalpos=4206&amp;ncbi_uid=3568547&amp;link_uid=PMC3075804" image-link="/pmc/articles/PMC3075804/figure/F4/" class="imagepopup">Figure 4. The cytokine production of DCs induced by E. bieneusi is MyD88-dependent but independent of TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: MyD88-dependent <span class="highlight" style="background-color:">pathway</span> is essential for the innate immunity to Enterocytozoon bieneusi. </a></p><div class="supp"><p class="details"><div>BMDCs were generated from MyD88<sup>-/-</sup> (A) or C3H/HeJ (B) mice and treated with medium, LPS, or <i>E. bieneusi</i> for 24 hours. Cells were added with Brefeldin A to block cytokine secretion in the last 12 hours of incubation. Cells were harvested and IL-12 and IL-6 production were measured using intracellular cytokine staining.</div></p></div><div class="aux"><div class="resc">Quanshun Zhang, et al. Parasite Immunol. ;33(4):217-225.</div><p class="links"><a class="dblinks" href="/pubmed/21204848" ref="ordinalpos=4206&amp;ncbi_uid=3568547&amp;link_uid=21204848">Citation</a><a class="dblinks" href="/pmc/articles/PMC3075804/" ref="ordinalpos=4206&amp;ncbi_uid=3568547&amp;link_uid=PMC3075804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4207.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4300715/figure/fig04/" ref="ordinalpos=4207&amp;ncbi_uid=7104650&amp;link_uid=PMC4300715" href="/pmc/articles/PMC4300715/"><img src="/pmc/articles/instance/4300715/bin/ajcr0005-0114-f4.gif" src-large="/pmc/articles/instance/4300715/bin/ajcr0005-0114-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4300715/" ref="ordinalpos=4207&amp;ncbi_uid=7104650&amp;link_uid=PMC4300715" image-link="/pmc/articles/PMC4300715/figure/fig04/" class="imagepopup">Figure 4.  From: Effect of luteinizing hormone-induced prohibitin and matrix metalloproteinases on ovarian epithelial tumor cell proliferation. </a></p><div class="supp"><p class="details"><div>LH regulated PHB, MMP-2 and MMP-9 protein expression through AKT and ERK1/2 signal <span class="highlight" style="background-color:">pathway</span>. The cells were treated with or without 50 mIU/ml LH for 48 h in serum-free cultures after pre-treatment with 10 μmol/L U0126 and LY294002 for 1 h, the protein expressions were investigated by western blot analysis. PBS treatment was used as a negative control. A. Shows ERK1/2 <span class="highlight" style="background-color:">signaling</span> inhibitor U0126 and AKT <span class="highlight" style="background-color:">signaling</span> inhibitor LY294002 block the up-regulation of prohibitin, MMP-2 and MMP-9 induced by LH in MCV152 cells. B. Shows ERK1/2 <span class="highlight" style="background-color:">signaling</span> inhibitor U0126 and AKT <span class="highlight" style="background-color:">signaling</span> inhibitor LY294002 block the up-regulation of prohibitin, MMP-2 and MMP-9 induced by LH in SKOV3 cells.</div></p></div><div class="aux"><div class="resc">Yue Wang, et al. Am J Cancer Res. 2015;5(1):114-124.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4207&amp;ncbi_uid=7104650&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4300715/" ref="ordinalpos=4207&amp;ncbi_uid=7104650&amp;link_uid=PMC4300715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4208.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4127075/figure/F2/" ref="ordinalpos=4208&amp;ncbi_uid=6479407&amp;link_uid=PMC4127075" href="/pmc/articles/PMC4127075/"><img src="/pmc/articles/instance/4127075/bin/nihms600767f2.gif" src-large="/pmc/articles/instance/4127075/bin/nihms600767f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4127075/" ref="ordinalpos=4208&amp;ncbi_uid=6479407&amp;link_uid=PMC4127075" image-link="/pmc/articles/PMC4127075/figure/F2/" class="imagepopup">Fig. 2. The functions of KLF4 involved in the mechanism of EMT.  From: Regulation of EMT by KLF4 in Gastrointestinal Cancer. </a></p><div class="supp"><p class="details"><div>EMT is a product of crosstalk between <span class="highlight" style="background-color:">signaling</span> pathways, including TGF-β, Wnt/β-catenin, Notch, FGF, STAT3 and EGF. By regulating the transcription factors (ZEBs, Snails and Twist), these <span class="highlight" style="background-color:">signaling</span> pathways switch the epithelial phenotype to mesenchymal phenotype. In cancer cells, KLF4 represses the EMT through interacting with TGF-β, Notch and Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> pathways. The degradation of KLF4 through UPS is necessary to TGF-β induced transcriptional activation. In Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, KLF4 negatively regulates the expression of Notch1 and functions as a negative modulator of Notch1 target genes’ transcription. KLF4 also interacts with Wnt/β-catenin by antagonizing the binding of β-catenin to TCF and represses the transcriptional activity of TCF.</div></p></div><div class="aux"><div class="resc">Jiujie Cui, et al. Curr Cancer Drug Targets. ;13(9):986-995.</div><p class="links"><a class="dblinks" href="/pubmed/24168184" ref="ordinalpos=4208&amp;ncbi_uid=6479407&amp;link_uid=24168184">Citation</a><a class="dblinks" href="/pmc/articles/PMC4127075/" ref="ordinalpos=4208&amp;ncbi_uid=6479407&amp;link_uid=PMC4127075">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4209.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889478/figure/F1/" ref="ordinalpos=4209&amp;ncbi_uid=5702368&amp;link_uid=PMC3889478" href="/pmc/articles/PMC3889478/"><img src="/pmc/articles/instance/3889478/bin/nihms-540501-f0001.gif" src-large="/pmc/articles/instance/3889478/bin/nihms-540501-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889478/" ref="ordinalpos=4209&amp;ncbi_uid=5702368&amp;link_uid=PMC3889478" image-link="/pmc/articles/PMC3889478/figure/F1/" class="imagepopup">Figure 1.  From: Epithelial and mesenchymal contribution to the niche: a safeguard for intestinal stem cell homeostasis. </a></p><div class="supp"><p class="details"><div>Comparison of <i>in vivo</i> (left) and <i>in vitro</i> (right) <span class="highlight" style="background-color:">signaling</span> support to the intestinal stem cell. <span class="highlight" style="background-color:">Signaling</span> networks between the Paneth cell and stem cell appear to be broadly conserved. However, disruption on one element of a single <span class="highlight" style="background-color:">pathway</span> highlights the important compensation wired into the system where <i>in vivo</i> environments can rely upon mesenchymal cells to maintain <span class="highlight" style="background-color:">signaling</span> homeostasis. Wnt, Notch and <span class="highlight" style="background-color:">signaling</span> pathways are key players in the balance between stem cell maintenance and differentiation.</div></p></div><div class="aux"><div class="resc">Nicholas R. Smith, et al. Gastroenterology. ;143(6):1426-1430.</div><p class="links"><a class="dblinks" href="/pubmed/23085353" ref="ordinalpos=4209&amp;ncbi_uid=5702368&amp;link_uid=23085353">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889478/" ref="ordinalpos=4209&amp;ncbi_uid=5702368&amp;link_uid=PMC3889478">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4210.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3483025/figure/F10/" ref="ordinalpos=4210&amp;ncbi_uid=4320448&amp;link_uid=PMC3483025" href="/pmc/articles/PMC3483025/"><img src="/pmc/articles/instance/3483025/bin/fpls-03-00236-g010.gif" src-large="/pmc/articles/instance/3483025/bin/fpls-03-00236-g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3483025/" ref="ordinalpos=4210&amp;ncbi_uid=4320448&amp;link_uid=PMC3483025" image-link="/pmc/articles/PMC3483025/figure/F10/" class="imagepopup">Figure 10.  From: Evidence for a Contribution of ALA Synthesis to Plastid-To-Nucleus <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model summarizing control mechanisms that act on and are emitted from ALA biosynthesis</b>. The contribution to plastid-derived retrograde <span class="highlight" style="background-color:">signaling</span> is highlighted. ALA formation is posttranslationally controlled by several known feedback mechanisms of the Chl and Mg branch of the <span class="highlight" style="background-color:">pathway</span> (e.g., heme, Mg chelatase assembly, GUN4) also reflecting the impact of the <i>gun2</i>-<i>gun5</i> mutations on ALA synthesis. Intermediates of the tetrapyrrole biosynthetic <span class="highlight" style="background-color:">pathway</span> (e.g., Mg Protoporphyrin IX) were discussed to be involved in <span class="highlight" style="background-color:">signaling</span> pathways and likely act via reactive oxygen species (ROS)-mediated changes in nuclear gene expression (NGE). Gabaculine (GAB) directly inhibits ALA formation resulting in changes of NGE at very early stages of chloroplast development. Norflurazon (NF) inhibits tetrapyrrole biosynthesis indirectly.</div></p></div><div class="aux"><div class="resc">Olaf Czarnecki, et al. Front Plant Sci. 2012;3:236.</div><p class="links"><a class="dblinks" href="/pubmed/23112801" ref="ordinalpos=4210&amp;ncbi_uid=4320448&amp;link_uid=23112801">Citation</a><a class="dblinks" href="/pmc/articles/PMC3483025/" ref="ordinalpos=4210&amp;ncbi_uid=4320448&amp;link_uid=PMC3483025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4211.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3279303/figure/F5/" ref="ordinalpos=4211&amp;ncbi_uid=3831274&amp;link_uid=PMC3279303" href="/pmc/articles/PMC3279303/"><img src="/pmc/articles/instance/3279303/bin/zh10031161840005.gif" src-large="/pmc/articles/instance/3279303/bin/zh10031161840005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3279303/" ref="ordinalpos=4211&amp;ncbi_uid=3831274&amp;link_uid=PMC3279303" image-link="/pmc/articles/PMC3279303/figure/F5/" class="imagepopup">Fig. 5. From: Glucagon-like peptide-2-stimulated protein synthesis through the PI 3-kinase-dependent Akt-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>GLP-2 activated mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in a PI3K-dependent manner. <i>A</i>: GLP-2 induced mTOR phosphorylation on Ser<sup>2448</sup> in a PI3K-dependent manner. PI3K inhibitor LY-294002 (50 μM) not only suppressed basal phosphorylation of mTOR Ser<sup>2448</sup> but also abolished GLP-2-induced stimulation (<i>P</i> &lt; 0.01). <i>B</i>: GLP-2 induced p70<sup>S6K</sup> phosphorylation on Thr<sup>389</sup> in a PI3K-dependent manner. LY-294002 (50 μM) not only suppressed basal phosphorylation of p70<sup>S6K</sup> Thr<sup>389</sup> but also abolished GLP-2-induced stimulation (<i>P</i> &lt; 0.01). <i>C</i>: GLP-2 induced 4E-BP1 phosphorylation on Thr<sup>37/46</sup> in a PI3K-dependent manner. LY-294002 (50 μM) not only suppressed basal phosphorylation of 4E-BP1 Thr<sup>37/46</sup> but also abolished GLP-2-induced stimulation (<i>P</i> &lt; 0.01). HEK 293 cells (transfected with 1 μg GLP-2R cDNA) were treated with 1 or 20 nM GLP-2 for 15 or 30 min in the presence of 0 or 50 μM LY-294002. * and **, Significance at <i>P</i> &lt; 0.05 and 0.01, respectively, vs. control with 0 nM GLP-2 for 0 min.</div></p></div><div class="aux"><div class="resc">Xuemei Shi, et al. Am J Physiol Endocrinol Metab. 2011 March;300(3):E554-E563.</div><p class="links"><a class="dblinks" href="/pubmed/21177288" ref="ordinalpos=4211&amp;ncbi_uid=3831274&amp;link_uid=21177288">Citation</a><a class="dblinks" href="/pmc/articles/PMC3279303/" ref="ordinalpos=4211&amp;ncbi_uid=3831274&amp;link_uid=PMC3279303">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4212.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2678983/figure/F4/" ref="ordinalpos=4212&amp;ncbi_uid=1835134&amp;link_uid=PMC2678983" href="/pmc/articles/PMC2678983/"><img src="/pmc/articles/instance/2678983/bin/1752-0509-3-36-4.gif" src-large="/pmc/articles/instance/2678983/bin/1752-0509-3-36-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2678983/" ref="ordinalpos=4212&amp;ncbi_uid=1835134&amp;link_uid=PMC2678983" image-link="/pmc/articles/PMC2678983/figure/F4/" class="imagepopup">Figure 4.  From: Identifying disease-specific genes based on their topological significance in protein networks. </a></p><div class="supp"><p class="details"><div><b>IFN-gamma <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Mapped on the <span class="highlight" style="background-color:">pathway</span> are the set of high-scored network nodes (set #1) and the set of differentially expressed genes (set #2). The datasets are visualized by thermometer-like symbols next to protein icons. Differentially expressed genes with a well-established relation to disease (e.g. STAT1, c-Myc) are cross-validated by high statistical scoring. Many additional important elements of the <span class="highlight" style="background-color:">pathway</span> are identified by topological scoring – IFN-gamma, IRF-1, PKC-delta, CaMK II, CBP.</div></p></div><div class="aux"><div class="resc">Zoltán Dezső, et al. BMC Syst Biol. 2009;3:36-36.</div><p class="links"><a class="dblinks" href="/pubmed/19309513" ref="ordinalpos=4212&amp;ncbi_uid=1835134&amp;link_uid=19309513">Citation</a><a class="dblinks" href="/pmc/articles/PMC2678983/" ref="ordinalpos=4212&amp;ncbi_uid=1835134&amp;link_uid=PMC2678983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4213.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3597511/figure/pgen-1003362-g006/" ref="ordinalpos=4213&amp;ncbi_uid=4725563&amp;link_uid=PMC3597511" href="/pmc/articles/PMC3597511/"><img src="/pmc/articles/instance/3597511/bin/pgen.1003362.g006.gif" src-large="/pmc/articles/instance/3597511/bin/pgen.1003362.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3597511/" ref="ordinalpos=4213&amp;ncbi_uid=4725563&amp;link_uid=PMC3597511" image-link="/pmc/articles/PMC3597511/figure/pgen-1003362-g006/" class="imagepopup">Figure 6. <span class="highlight" style="background-color:">Pathway</span> analysis of transcript abundance..  From: Blood-Informative Transcripts Define Nine Common Axes of Peripheral Blood Gene Expression. </a></p><div class="supp"><p class="details"><div>The relative transcript abundance of 51 transcripts in the KEGG TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (map04620) is shown for two representative divergent individuals (A). These genes are all co-regulated along Axis 5 (B), resulting in differential activity throughout the <span class="highlight" style="background-color:">pathway</span>. Two representative individuals at either extreme, indicated in red (individual 22) and blue (individual 69) in panel B, clearly differ with respect to which genes have high or low expression relative to regulation of apoptosis, MAPK <span class="highlight" style="background-color:">signaling</span>, and inflammatory cytokine production. This likely has consequences for the sensitivity of neutrophil and other immune cell function. Red, high expression; blue, low expression; gray intermediate, scaled as the eigenvector of PC1.</div></p></div><div class="aux"><div class="resc">Marcela Preininger, et al. PLoS Genet. 2013 March;9(3):e1003362.</div><p class="links"><a class="dblinks" href="/pubmed/23516379" ref="ordinalpos=4213&amp;ncbi_uid=4725563&amp;link_uid=23516379">Citation</a><a class="dblinks" href="/pmc/articles/PMC3597511/" ref="ordinalpos=4213&amp;ncbi_uid=4725563&amp;link_uid=PMC3597511">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4214.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412346/figure/F4/" ref="ordinalpos=4214&amp;ncbi_uid=5125449&amp;link_uid=PMC3412346" href="/pmc/articles/PMC3412346/"><img src="/pmc/articles/instance/3412346/bin/zh89991294300004.gif" src-large="/pmc/articles/instance/3412346/bin/zh89991294300004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412346/" ref="ordinalpos=4214&amp;ncbi_uid=5125449&amp;link_uid=PMC3412346" image-link="/pmc/articles/PMC3412346/figure/F4/" class="imagepopup">Figure 4.  From: Aberrant STAT5 and PI3K/mTOR <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> occurs in human CRLF2-rearranged B-precursor acute lymphoblastic leukemia. </a></p><div class="supp"><p class="details"><div><b>Expanded analysis of primary CRLF2<sub>r</sub> ALL samples demonstrates inhibition of JAK/STAT and PI3K <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>.</b> Analysis of JAK/STAT, PI3K/mTOR, and MAPK <span class="highlight" style="background-color:">pathway</span> STIs in 29 CRLF2<sub>wt</sub> (A) and CRLF2<sub>r</sub> (B) primary human leukemias. Data are displayed as whisker plots of 25th to 75th percentiles with means (central bars) and ranges (whiskers). The dashed horizontal line represents normalized basal phosphorylation for each phosphoprotein. Pink bars indicate TSLP; striped red, ruxolitinib, solid red, ruxolitinib + TSLP; striped green, rapamycin; solid green, rapamycin + TSLP; striped blue, PI103; solid blue, PI103 + TSLP; striped purple, PP242; solid purple, PP242 + TSLP; striped gray, PD901; solid gray, PD901 + TSLP. Minimal effects of TSLP stimulation or STIs were observed for the CRLF2<sub>wt</sub> samples. TSLP stimulation induced pSTAT5, pAkt, pS6, p4EBP1, peIF4E, and pERK in the CRLF2<sub>r</sub> samples (<i>P</i> &lt; .05), and incubation with ruxolitinib resulted in continued inhibition of all aforementioned phosphoproteins (<i>P</i> &lt; .05). PI103 inhibited TSLP-induced pAkt, pS6, and pERK (<i>P</i> &lt; .05). PP242 abrogated TSLP-induced pAkt, pS6, and peIF4E (<i>P</i> &lt; .05). PD901 inhibited TSLP-induced pS6, peIF4E, and pERK (<i>P</i> &lt; .05).</div></p></div><div class="aux"><div class="resc">Sarah K. Tasian, et al. Blood. 2012 July 26;120(4):833-842.</div><p class="links"><a class="dblinks" href="/pubmed/22685175" ref="ordinalpos=4214&amp;ncbi_uid=5125449&amp;link_uid=22685175">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412346/" ref="ordinalpos=4214&amp;ncbi_uid=5125449&amp;link_uid=PMC3412346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4215.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1867372/figure/fig7/" ref="ordinalpos=4215&amp;ncbi_uid=1042493&amp;link_uid=PMC1867372" href="/pmc/articles/PMC1867372/"><img src="/pmc/articles/instance/1867372/bin/tpc1900805f07.gif" src-large="/pmc/articles/instance/1867372/bin/tpc1900805f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1867372/" ref="ordinalpos=4215&amp;ncbi_uid=1042493&amp;link_uid=PMC1867372" image-link="/pmc/articles/PMC1867372/figure/fig7/" class="imagepopup">Figure 7. From: The Mitogen-Activated Protein Kinase Cascade MKK3-MPK6 Is an Important Part of the Jasmonate Signal Transduction <span class="highlight" style="background-color:">Pathway</span> in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Possible Roles of MPK6 <span class="highlight" style="background-color:">Signaling</span> in <i>Arabidopsis</i>.<br />At least three cascades, MKK2–MPK6, MKK4/MKK5–MPK6, and MKK3–MPK6, function in <i>Arabidopsis</i>. Each plays a specific role in response to different stimuli, and the cascades transduce their signals to adapt to various environmental changes by crosstalking with each other. The MKK2–MPK6 cascade regulates cold and salt stress-responsive genes, the MKK4/MKK5–MPK6 cascade regulates pathogen-responsive genes by activating ET biosynthesis, and the novel MKK3–MPK6 cascade regulates JA <span class="highlight" style="background-color:">signaling</span>. The JA-activated MKK3–MPK6 signal negatively regulates the JA <span class="highlight" style="background-color:">pathway</span> and affects JA-dependent gene expression and root growth sensitivity through <i>ATMYC2</i>.</div></p></div><div class="aux"><div class="resc">Fuminori Takahashi, et al. Plant Cell. 2007 March;19(3):805-818.</div><p class="links"><a class="dblinks" href="/pubmed/17369371" ref="ordinalpos=4215&amp;ncbi_uid=1042493&amp;link_uid=17369371">Citation</a><a class="dblinks" href="/pmc/articles/PMC1867372/" ref="ordinalpos=4215&amp;ncbi_uid=1042493&amp;link_uid=PMC1867372">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4216.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3694216/figure/F1/" ref="ordinalpos=4216&amp;ncbi_uid=5053844&amp;link_uid=PMC3694216" href="/pmc/articles/PMC3694216/"><img src="/pmc/articles/instance/3694216/bin/fpls-04-00215-g001.gif" src-large="/pmc/articles/instance/3694216/bin/fpls-04-00215-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3694216/" ref="ordinalpos=4216&amp;ncbi_uid=5053844&amp;link_uid=PMC3694216" image-link="/pmc/articles/PMC3694216/figure/F1/" class="imagepopup">FIGURE 1.  From: Integrating nitric oxide into salicylic acid and jasmonic acid/ ethylene plant defense pathways. </a></p><div class="supp"><p class="details"><div><b>The impact of nitric oxide on salicylic acid, jasmonate (and ethylene <span class="highlight" style="background-color:">signaling</span> cascades.</b> Schematic versions of salicylic acid (SA), jasmonate (JA), and ethylene (ET) <span class="highlight" style="background-color:">signaling</span> cascade. Biosynthetic enzymes are represented as gray ovals and <span class="highlight" style="background-color:">signaling</span> components are gray rectangles. Abbreviations in the jasmonate biosynthetic <span class="highlight" style="background-color:">pathway</span> are as follows: LOX, lipoxygenase; AOC, allene oxide cyclase; OPR, oxo-phytodienoate reductase; for the ethylene biosynthetic <span class="highlight" style="background-color:">pathway</span>: ACS, 1-aminocyclopropane-1-carboxylic acid synthase; ACO, 1-aminocyclopropane-1-carboxylic acid oxidase. Genes and their regulatory promoters are represented as open boxes. For details of <span class="highlight" style="background-color:">signaling</span> cascades, see main text. The known NO-regulated steps are indicated: green arrows indicating that NO has a promontory effects via induction of gene transcription; <i>S</i>-nitrosylative steps are indicated via a solid black bar and whether this promotes (green arrows) or inhibit a <span class="highlight" style="background-color:">signaling</span> (red bar) step. Note that for and ACO and ACS4, the effect of <i>S</i>-nitrosylation has yet to be determined.</div></p></div><div class="aux"><div class="resc">Luis A. J. Mur, et al. Front Plant Sci. 2013;4:215.</div><p class="links"><a class="dblinks" href="/pubmed/23818890" ref="ordinalpos=4216&amp;ncbi_uid=5053844&amp;link_uid=23818890">Citation</a><a class="dblinks" href="/pmc/articles/PMC3694216/" ref="ordinalpos=4216&amp;ncbi_uid=5053844&amp;link_uid=PMC3694216">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4217.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938097/figure/F4/" ref="ordinalpos=4217&amp;ncbi_uid=3059922&amp;link_uid=PMC2938097" href="/pmc/articles/PMC2938097/"><img src="/pmc/articles/instance/2938097/bin/zjw0071036280004.gif" src-large="/pmc/articles/instance/2938097/bin/zjw0071036280004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938097/" ref="ordinalpos=4217&amp;ncbi_uid=3059922&amp;link_uid=PMC2938097" image-link="/pmc/articles/PMC2938097/figure/F4/" class="imagepopup">Fig. 4. From: Studies of Phosphoproteomic Changes Induced by Nucleophosmin-Anaplastic Lymphoma Kinase (ALK) Highlight Deregulation of Tumor Necrosis Factor (TNF)/Fas/TNF-related Apoptosis-induced Ligand <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in ALK-positive Anaplastic Large Cell Lymphoma. </a></p><div class="supp"><p class="details"><div><b>Functional <span class="highlight" style="background-color:">pathway</span> analysis of the NPM-ALK-induced phosphoproteins.</b> The Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software was used to analyze the functional pathways of the NPM-ALK-induced phosphoproteins. The numbers in parentheses represent the number of proteins belonging to the corresponding pathways.</div></p></div><div class="aux"><div class="resc">Fang Wu, et al. Mol Cell Proteomics. 2010 July;9(7):1616-1632.</div><p class="links"><a class="dblinks" href="/pubmed/20393185" ref="ordinalpos=4217&amp;ncbi_uid=3059922&amp;link_uid=20393185">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938097/" ref="ordinalpos=4217&amp;ncbi_uid=3059922&amp;link_uid=PMC2938097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4218.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3376383/figure/F1/" ref="ordinalpos=4218&amp;ncbi_uid=4810523&amp;link_uid=PMC3376383" href="/pmc/articles/PMC3376383/"><img src="/pmc/articles/instance/3376383/bin/nihms368160f1.gif" src-large="/pmc/articles/instance/3376383/bin/nihms368160f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3376383/" ref="ordinalpos=4218&amp;ncbi_uid=4810523&amp;link_uid=PMC3376383" image-link="/pmc/articles/PMC3376383/figure/F1/" class="imagepopup">Figure 1. Functionally conserved components of the Hippo tumor suppressor <span class="highlight" style="background-color:">pathway</span>.  From: Growth Control by Committee - Intercellular Junctions, Cell Polarity, and the Cytoskeleton Regulate Hippo <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The FERM domain proteins Merlin and Expanded/FMD6 have been proposed to function upstream of the core kinase <span class="highlight" style="background-color:">signaling</span> cassette together with the WW domain protein Kibra. The sterile 20 kinase Tao-1 directly phosphorylates Hippo and MST kinases in vitro and promotes <span class="highlight" style="background-color:">pathway</span> activation in vivo. The scaffolding proteins Salvador/WW45 and Mats/Mob promote Hippo/MST and Warts/Lats kinase activity, respectively. Hippo/MST phosphorylates Warts/Lats, which in turn phosphorylates Yorkie/Yap to promote 14-3-3 binding. 14-3-3 sequesters Yorkie/Yap in the cytoplasm, preventing the transcription of target genes that promote tissue growth. In addition, Fat <span class="highlight" style="background-color:">signaling</span> (not shown) has been shown to regulate Warts stability and Yorkie activation in parallel to Hippo.</div></p></div><div class="aux"><div class="resc">Julian C. Boggiano, et al. Dev Cell. ;22(4):695-702.</div><p class="links"><a class="dblinks" href="/pubmed/22516196" ref="ordinalpos=4218&amp;ncbi_uid=4810523&amp;link_uid=22516196">Citation</a><a class="dblinks" href="/pmc/articles/PMC3376383/" ref="ordinalpos=4218&amp;ncbi_uid=4810523&amp;link_uid=PMC3376383">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4219.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2798675/figure/F9/" ref="ordinalpos=4219&amp;ncbi_uid=2132255&amp;link_uid=PMC2798675" href="/pmc/articles/PMC2798675/"><img src="/pmc/articles/instance/2798675/bin/JCI37964.f9.gif" src-large="/pmc/articles/instance/2798675/bin/JCI37964.f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2798675/" ref="ordinalpos=4219&amp;ncbi_uid=2132255&amp;link_uid=PMC2798675" image-link="/pmc/articles/PMC2798675/figure/F9/" class="imagepopup">Figure 9. STAT3/p63/Notch <span class="highlight" style="background-color:">signaling</span> is controlled under mTORC1..  From: Mammalian target of rapamycin regulates murine and human cell differentiation through STAT3/p63/Jagged/Notch cascade. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) <i>Tsc2<sup>–/–</sup></i> MEFs were transfected with siRNA to knock down <i>mTOR</i> expression for 48 hours and then subjected to immunoblotting for p-STAT3 (Ser705), p63, and Notch components. Non–target-directed random siRNA served as a control. (<b>B</b>) <i>Tsc2<sup>–/–</sup></i> MEFs were transfected with siRNA to knock down <i>rictor</i> expression for 48 hours and then subjected to immunoblotting for p-STAT3, p63, and Notch components. Non–target-directed random siRNA served as a control. (<b>C</b>) Schematic illustration of how the RTK/PI3K/AKT/mTOR <span class="highlight" style="background-color:">pathway</span> regulates cell differentiation through the STAT/p63/Jag/Notch cascade. Upon stimulation of RTKs, PI3K activates AKT, which phosphorylates TSC2 and reduces the GAP activity of the TSC1/TSC2 complex toward Rheb-GTP, increasing Rheb-GTP levels. Rheb-GTP activates mTORC1, which in turn enhances Notch <span class="highlight" style="background-color:">signaling</span> through upregulation of the STAT3/p63 axis. mTORC1 regulates cell differentiation through Notch <span class="highlight" style="background-color:">signaling</span> in a dose-dependent manner.    </div></p></div><div class="aux"><div class="resc">Jianhui Ma, et al. J Clin Invest. 2010 January 4;120(1):103-114.</div><p class="links"><a class="dblinks" href="/pubmed/20038814" ref="ordinalpos=4219&amp;ncbi_uid=2132255&amp;link_uid=20038814">Citation</a><a class="dblinks" href="/pmc/articles/PMC2798675/" ref="ordinalpos=4219&amp;ncbi_uid=2132255&amp;link_uid=PMC2798675">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4220.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3539134/figure/F3/" ref="ordinalpos=4220&amp;ncbi_uid=4525738&amp;link_uid=PMC3539134" href="/pmc/articles/PMC3539134/"><img src="/pmc/articles/instance/3539134/bin/fendo-03-00181-g0003.gif" src-large="/pmc/articles/instance/3539134/bin/fendo-03-00181-g0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3539134/" ref="ordinalpos=4220&amp;ncbi_uid=4525738&amp;link_uid=PMC3539134" image-link="/pmc/articles/PMC3539134/figure/F3/" class="imagepopup">Figure 3.  From: Implication of inflammatory <span class="highlight" style="background-color:">signaling</span> pathways in obesity-induced insulin resistance. </a></p><div class="supp"><p class="details"><div><b>Serine kinases involved in obesity-induced inflammation and insulin resistance.</b> In obesity, a network of serine kinases is activated including IKKβ, JNK, and ERK1/2. JNK and IKKβ are activated downstream of pattern-recognition receptors (PRRs) such as TLRs or by ER stress. IKKβ is also involved in the activation of the Tpl2/ERK <span class="highlight" style="background-color:">pathway</span>. These pathways participate in the production of inflammatory cytokines via the transcription factors AP-1 and NF-κB. Many of the produced inflammatory cytokines are able to activate these kinases leading to a feed-forward amplification loop. JNK and ERK1/2 are involved in the desensitization of insulin <span class="highlight" style="background-color:">signaling</span> through phosphorylation of IRS1/2 on inhibitory serine sites (SerP). IKKβ can directly phosphorylate IRS1/2 on serine sites but can also act indirectly through activation of mTORC1/S6 kinase. Over-activation of the mTORC1/S6K <span class="highlight" style="background-color:">pathway</span> could promote ER stress leading to an amplification loop.</div></p></div><div class="aux"><div class="resc">Jean-François Tanti, et al. Front Endocrinol (Lausanne). 2012;3:181.</div><p class="links"><a class="dblinks" href="/pubmed/23316186" ref="ordinalpos=4220&amp;ncbi_uid=4525738&amp;link_uid=23316186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3539134/" ref="ordinalpos=4220&amp;ncbi_uid=4525738&amp;link_uid=PMC3539134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4221.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3797697/figure/pone-0077806-g005/" ref="ordinalpos=4221&amp;ncbi_uid=5404474&amp;link_uid=PMC3797697" href="/pmc/articles/PMC3797697/"><img src="/pmc/articles/instance/3797697/bin/pone.0077806.g005.gif" src-large="/pmc/articles/instance/3797697/bin/pone.0077806.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3797697/" ref="ordinalpos=4221&amp;ncbi_uid=5404474&amp;link_uid=PMC3797697" image-link="/pmc/articles/PMC3797697/figure/pone-0077806-g005/" class="imagepopup">Figure 5. JNK <span class="highlight" style="background-color:">signaling</span> was involved in CD9-regulated MMP-9 production..  From: Downregulation of CD9 in Keratinocyte Contributes to Cell Migration via Upregulation of Matrix Metalloproteinase-9 . </a></p><div class="supp"><p class="details"><div>(A) Representative western blot results showing increased JNK phosphorylation in CD9-shRNA adenovirus-infected HaCaT cells, with no alteration to p38 <span class="highlight" style="background-color:">pathway</span> and ERK <span class="highlight" style="background-color:">pathway</span>. (B) Representative western blot results and data summary of p-JNK and JNK in HaCaT cells infected with CD9-shRNA and Ad-CD9 adenovirus (n = 5). (C) Gelatin zymograms were shown for activity of MMP-9 and MMP-2 in the culture supernatants (n = 3). Mock-infected and CD9-silencing HaCaT cells were treated with JNK <span class="highlight" style="background-color:">signaling</span> inhibitor, SP600125 (SP). (D) Western blots were shown for MMP-9 and MMP-2 (n = 3). *P&lt; 0.05 vs. Mock group. #P&lt; 0.05 vs. Vector group. **P&lt; 0.05 vs. CD9-shRNA group.</div></p></div><div class="aux"><div class="resc">Xu-pin Jiang, et al. PLoS One. 2013;8(10):e77806.</div><p class="links"><a class="dblinks" href="/pubmed/24147081" ref="ordinalpos=4221&amp;ncbi_uid=5404474&amp;link_uid=24147081">Citation</a><a class="dblinks" href="/pmc/articles/PMC3797697/" ref="ordinalpos=4221&amp;ncbi_uid=5404474&amp;link_uid=PMC3797697">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4222.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3974726/figure/pone-0092481-g008/" ref="ordinalpos=4222&amp;ncbi_uid=6006113&amp;link_uid=PMC3974726" href="/pmc/articles/PMC3974726/"><img src="/pmc/articles/instance/3974726/bin/pone.0092481.g008.gif" src-large="/pmc/articles/instance/3974726/bin/pone.0092481.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3974726/" ref="ordinalpos=4222&amp;ncbi_uid=6006113&amp;link_uid=PMC3974726" image-link="/pmc/articles/PMC3974726/figure/pone-0092481-g008/" class="imagepopup">Figure 8. Shortest positive and negative <span class="highlight" style="background-color:">signaling</span> path from ERK1/2 to IL-10..  From: Comprehensive Logic Based Analyses of Toll-Like Receptor 4 Signal Transduction <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Positive and negative <span class="highlight" style="background-color:">signaling</span> path from the ERK1/2 to IL-10 is shown. Red bar show inhibition and black shows activatory signal.</div></p></div><div class="aux"><div class="resc">Mahesh Kumar Padwal, et al. PLoS One. 2014;9(4):e92481.</div><p class="links"><a class="dblinks" href="/pubmed/24699232" ref="ordinalpos=4222&amp;ncbi_uid=6006113&amp;link_uid=24699232">Citation</a><a class="dblinks" href="/pmc/articles/PMC3974726/" ref="ordinalpos=4222&amp;ncbi_uid=6006113&amp;link_uid=PMC3974726">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4223.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3499189/figure/F3/" ref="ordinalpos=4223&amp;ncbi_uid=4375531&amp;link_uid=PMC3499189" href="/pmc/articles/PMC3499189/"><img src="/pmc/articles/instance/3499189/bin/1743-7075-9-74-3.gif" src-large="/pmc/articles/instance/3499189/bin/1743-7075-9-74-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3499189/" ref="ordinalpos=4223&amp;ncbi_uid=4375531&amp;link_uid=PMC3499189" image-link="/pmc/articles/PMC3499189/figure/F3/" class="imagepopup">Figure 3.  From: The impact of cow's milk-mediated mTORC1-<span class="highlight" style="background-color:">signaling</span> in the initiation and progression of prostate cancer. </a></p><div class="supp"><p class="details"><div><b> Aberrations of PI3K-Akt-mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in prostate cancer.</b> Mutation-induced hyperactivated PI3K, Akt and overexpression of Rheb stimulate mTORC1. Loss of PTEN results in hyperactive Akt <span class="highlight" style="background-color:">signaling</span>. Androgens activate Akt via mTORC2 and via upregulation of L-type amino acid transporters (LAT) promote leucine-stimulated mTORC1 activation.</div></p></div><div class="aux"><div class="resc">Bodo C Melnik, et al. Nutr Metab (Lond). 2012;9:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/22891897" ref="ordinalpos=4223&amp;ncbi_uid=4375531&amp;link_uid=22891897">Citation</a><a class="dblinks" href="/pmc/articles/PMC3499189/" ref="ordinalpos=4223&amp;ncbi_uid=4375531&amp;link_uid=PMC3499189">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4224.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3631470/figure/F2/" ref="ordinalpos=4224&amp;ncbi_uid=5987848&amp;link_uid=PMC3631470" href="/pmc/articles/PMC3631470/"><img src="/pmc/articles/instance/3631470/bin/nihms441884f2.gif" src-large="/pmc/articles/instance/3631470/bin/nihms441884f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3631470/" ref="ordinalpos=4224&amp;ncbi_uid=5987848&amp;link_uid=PMC3631470" image-link="/pmc/articles/PMC3631470/figure/F2/" class="imagepopup">Fig. 2.  From: Titanium dioxide nanoparticles increase inflammatory responses in vascular endothelial cells. </a></p><div class="supp"><p class="details"><div>TiO<sub>2</sub> NP-induced phosphorylation of MAPKs and PI3K/Akt pathways in endothelial cells. Cells were treated with TiO<sub>2</sub> NPs to observe the activation of three types of MAPK <span class="highlight" style="background-color:">signaling</span> pathways, p38, ERK and JNK, as well as the Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The images shown here are phosphorylated <span class="highlight" style="background-color:">signaling</span> molecules at the time of maximum induction. The Western blots shown here are representative images of three independent experiments.</div></p></div><div class="aux"><div class="resc">Sung Gu Han, et al. Toxicology. 2013 April 5;0:1-8.</div><p class="links"><a class="dblinks" href="/pubmed/23380242" ref="ordinalpos=4224&amp;ncbi_uid=5987848&amp;link_uid=23380242">Citation</a><a class="dblinks" href="/pmc/articles/PMC3631470/" ref="ordinalpos=4224&amp;ncbi_uid=5987848&amp;link_uid=PMC3631470">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4225.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2744676/figure/F2/" ref="ordinalpos=4225&amp;ncbi_uid=2028154&amp;link_uid=PMC2744676" href="/pmc/articles/PMC2744676/"><img src="/pmc/articles/instance/2744676/bin/1478-811X-7-20-2.gif" src-large="/pmc/articles/instance/2744676/bin/1478-811X-7-20-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2744676/" ref="ordinalpos=4225&amp;ncbi_uid=2028154&amp;link_uid=PMC2744676" image-link="/pmc/articles/PMC2744676/figure/F2/" class="imagepopup">Figure 2.  From: Cholinergic receptor pathways involved in apoptosis, cell proliferation and neuronal differentiation. </a></p><div class="supp"><p class="details"><div><b>Diagram depicting proliferative and survival <span class="highlight" style="background-color:">signaling</span> pathways in cells</b>. <i>In vitro</i>, the homomeric and heteromeric nAChRs jointly stimulate the indicated <span class="highlight" style="background-color:">signaling</span> cascades. Yellow arrows indicate proliferative pathways triggered by nAChRs. This activation triggers the MAP kinase <span class="highlight" style="background-color:">pathway</span>, leading to DNA synthesis. Sustained mitogenic <span class="highlight" style="background-color:">signaling</span> induces to S-phase entry. Black arrows indicate nAChR survival and proliferation pathways triggered by intracellular calcium increases involving indirect activation of β-adrenergic receptor <span class="highlight" style="background-color:">signaling</span>, which in turn, induces activation of epidermal growth factor (EGF) receptor leading to the cascade indicated by blue arrows. α7 nAChR and heteromeric α-βnAChRs are activated by their agonists. Influx of Ca<sup>2+ </sup>and other cations through the nAChRs and voltage-gated Ca<sup>2+ </sup>channels trigger the release of adrenaline and noradrenalin. Adenylyl cyclase activation downstream of β-adrenergic receptors induce the cyclic AMP-protein kinase A (PKA)-CREB (cAMP response element-binding protein) <span class="highlight" style="background-color:">pathway</span>, transactivates epidermal growth factor receptor (EGFR) and induces the release of EGF, and perhaps another growth factors. The responsiveness of this <span class="highlight" style="background-color:">pathway</span> is enhanced by α7 nAChR-mediated activation of Ras through β-arrestin-dependent SRC <span class="highlight" style="background-color:">signaling</span>. In turn, the EGFR activates the Akt <span class="highlight" style="background-color:">pathway</span> and its downstream effectors, X-linked inhibitor of apoptosis protein (XIAP)-survivin and nuclear factor-κB (NF-κB).</div></p></div><div class="aux"><div class="resc">Rodrigo R Resende, et al. Cell Commun Signal. 2009;7:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/19712465" ref="ordinalpos=4225&amp;ncbi_uid=2028154&amp;link_uid=19712465">Citation</a><a class="dblinks" href="/pmc/articles/PMC2744676/" ref="ordinalpos=4225&amp;ncbi_uid=2028154&amp;link_uid=PMC2744676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4226.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3520896/figure/pone-0051327-g003/" ref="ordinalpos=4226&amp;ncbi_uid=4453695&amp;link_uid=PMC3520896" href="/pmc/articles/PMC3520896/"><img src="/pmc/articles/instance/3520896/bin/pone.0051327.g003.gif" src-large="/pmc/articles/instance/3520896/bin/pone.0051327.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3520896/" ref="ordinalpos=4226&amp;ncbi_uid=4453695&amp;link_uid=PMC3520896" image-link="/pmc/articles/PMC3520896/figure/pone-0051327-g003/" class="imagepopup">Figure 3. Western blot analysis of the inflammation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RAW264.7 cells induced by LPS..  From: The Lipid-Bound Apolipoprotein A-I Cysteine Mutant (N74C) Inhibits the Activation of NF-?B, JNK and p38 in Endotoxemic Mice and RAW264.7 Cells. </a></p><div class="supp"><p class="details"><div>To investigate the effect of different rHDLs on inflammation, we examined the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of the RAW264.7 inflammation model treated with different rHDLs. Our results showed that rHDL74 and rHDLwt significantly decreased the activation of p38 and JNK (p&lt;0.05 vs. LPS), while rHDL228 aggravated the activation of ERK (p&lt;0.05 vs. LPS). Control group: RAW264.7 cells without any treatment; LPS group: RAW264.7 cells induced by LPS; rHDL74 group: RAW264.7 inflammation cells treated with rHDL74; rHDL228 group: RAW264.7 inflammation cells treated with rHDL228; rHDLwt group: RAW264.7 inflammation cells treated with rHDLwt. To confirm our experimental results, we performed three independent experiments. A. Western blot analysis at 12 h after rHDL treatment. B. Western blot analysis at 24 h after rHDL treatment. C. The gray value analysis of western blot bands with Image J at 12 h after rHDL treatment. D. The gray value analysis of western blot bands with Image J at 24 h after rHDL treatment. <sup>*</sup>P&lt;0.05 vs. control; <sup>#</sup>P&lt;0.05 vs. LPS; <sup>Ψ</sup>P&lt;0.05 vs. rHDLwt. Error bars indicate SD.</div></p></div><div class="aux"><div class="resc">Yunlong Wang, et al. PLoS One. 2012;7(12):e51327.</div><p class="links"><a class="dblinks" href="/pubmed/23251501" ref="ordinalpos=4226&amp;ncbi_uid=4453695&amp;link_uid=23251501">Citation</a><a class="dblinks" href="/pmc/articles/PMC3520896/" ref="ordinalpos=4226&amp;ncbi_uid=4453695&amp;link_uid=PMC3520896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4227.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g005/" ref="ordinalpos=4227&amp;ncbi_uid=1718062&amp;link_uid=PMC2634968" href="/pmc/articles/PMC2634968/"><img src="/pmc/articles/instance/2634968/bin/pcbi.1000292.g005.gif" src-large="/pmc/articles/instance/2634968/bin/pcbi.1000292.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634968/" ref="ordinalpos=4227&amp;ncbi_uid=1718062&amp;link_uid=PMC2634968" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g005/" class="imagepopup">Figure 5. A simplified illustration of the reactive oxygen species (ROS) production DIOS <span class="highlight" style="background-color:">pathway</span>..  From: Identification of Potential <span class="highlight" style="background-color:">Pathway</span> Mediation Targets in Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The three critical network reactions are highlighted in red. Although there were over forty reactions in the ROS production <span class="highlight" style="background-color:">pathway</span>, most were associated with the TLR-induced activation of various phox proteins by the protein kinases PDK1 and PKCz. However, because PDK1 and PKCz work in parallel, none of these reactions could control the flux through the entire <span class="highlight" style="background-color:">pathway</span>, and therefore were not critical network reactions. On the other hand, the three critical network reactions <i>Vav1-Rac1/GDP binding</i>, <i>Rac1 phosphorylation</i>, and <i>gp91-p22 binding</i>, produced the two other components that comprised the final phox protein complex, and were therefore critical to the overall output ROS production. Note also that these critical network reactions were localized to the ROS production <span class="highlight" style="background-color:">pathway</span> and did not interfere with other cellular processes. Thus, they represent ideal targets for mediation of TLR-induced ROS production.</div></p></div><div class="aux"><div class="resc">Fan Li, et al. PLoS Comput Biol. 2009 February;5(2):e1000292.</div><p class="links"><a class="dblinks" href="/pubmed/19229310" ref="ordinalpos=4227&amp;ncbi_uid=1718062&amp;link_uid=19229310">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634968/" ref="ordinalpos=4227&amp;ncbi_uid=1718062&amp;link_uid=PMC2634968">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4228.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694616/figure/F5/" ref="ordinalpos=4228&amp;ncbi_uid=1936752&amp;link_uid=PMC2694616" href="/pmc/articles/PMC2694616/"><img src="/pmc/articles/instance/2694616/bin/nihms114667f5.gif" src-large="/pmc/articles/instance/2694616/bin/nihms114667f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694616/" ref="ordinalpos=4228&amp;ncbi_uid=1936752&amp;link_uid=PMC2694616" image-link="/pmc/articles/PMC2694616/figure/F5/" class="imagepopup">Figure 5. Predicting clinical response with <span class="highlight" style="background-color:">pathway</span> module signatures.  From: A Genomic Strategy to Elucidate Modules of Oncogenic <span class="highlight" style="background-color:">Pathway</span> <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>A. Cetuximab, a monoclonal antibody, targets the EGF receptor. Using a signature for EGFR activation as a starting point, we apply BFRM to decompose the <span class="highlight" style="background-color:">pathway</span> into 20 modules, as represented by gene expression signatures.<br />B. Clinical response to cetuximab is assessed in 68 patients with advanced colorectal cancer (CRC), In these patients, we predict the activity of the 20 EGFR modules using the <span class="highlight" style="background-color:">pathway</span> module signatures. The plot shows that EGFR module 18 can significantly distinguish patients with response (R) and no response (NR) to the EGFR inhibitor cetuximab.<br />C. Assessment of each of the EGFR, Ras, and E2F <span class="highlight" style="background-color:">pathway</span> module signatures for prediction of cetuximab response. The table shows the ability of each module to identify patients that respond to the EGFR inhibitor cetuximab. The <i>p</i>-value is calculated using a non-parametric Wilcoxon test that compares the predicted modules scores between responders and non-responders.</div></p></div><div class="aux"><div class="resc">Jeffrey T. Chang, et al. Mol Cell. ;34(1):104-114.</div><p class="links"><a class="dblinks" href="/pubmed/19362539" ref="ordinalpos=4228&amp;ncbi_uid=1936752&amp;link_uid=19362539">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694616/" ref="ordinalpos=4228&amp;ncbi_uid=1936752&amp;link_uid=PMC2694616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4229.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3772117/figure/F2/" ref="ordinalpos=4229&amp;ncbi_uid=5329581&amp;link_uid=PMC3772117" href="/pmc/articles/PMC3772117/"><img src="/pmc/articles/instance/3772117/bin/jkst-2-e25408-g2.gif" src-large="/pmc/articles/instance/3772117/bin/jkst-2-e25408-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3772117/" ref="ordinalpos=4229&amp;ncbi_uid=5329581&amp;link_uid=PMC3772117" image-link="/pmc/articles/PMC3772117/figure/F2/" class="imagepopup">Figure. From: Regulation of proliferation, cell competition, and cellular growth by the Drosophila JAK-STAT <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> The role of the <i>Drosophila</i> JAK-STAT <span class="highlight" style="background-color:">pathway</span> in eye development. During the second larval instar, <i>Iro-C</i> genes are induced by Wg <span class="highlight" style="background-color:">signaling</span> in the dorsal domain of the eye disc. Iro-C proteins repress Fng expression such that only ventral cells express Fng. This creates a border of Fng-expressing and non-Fng-expressing cells at the midline of the disc, leading to Notch <span class="highlight" style="background-color:">signaling</span> there. Notch induces expression of <i>upd</i> at the posterior midline of the eye disc. JNK <span class="highlight" style="background-color:">signaling</span> can also induce <i>upd</i> during invasive tumor formation. <i>upd</i> is presumed to be repressed by PRC1 in cells outside of the posterior midline. Upd leads to activation of STAT92E (STAT<sup>P</sup>), which represses the Notch ligand Ser to the ventral domain and Wg to the anterior margin. Csk negatively regulates STAT<sup>P</sup>, either through Src or through Dome/Hop. Delta (Dl), another Notch ligand in <i>Drosophila</i>, is expressed only in the dorsal domain.</div></p></div><div class="aux"><div class="resc">Tamara Zoranovic, et al. JAKSTAT. 2013 July 1;2(3):e25408.</div><p class="links"><a class="dblinks" href="/pubmed/24069565" ref="ordinalpos=4229&amp;ncbi_uid=5329581&amp;link_uid=24069565">Citation</a><a class="dblinks" href="/pmc/articles/PMC3772117/" ref="ordinalpos=4229&amp;ncbi_uid=5329581&amp;link_uid=PMC3772117">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4230.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC4204446/figure/F9/" ref="ordinalpos=4230&amp;ncbi_uid=6744496&amp;link_uid=PMC4204446" href="/pmc/articles/PMC4204446/"><img src="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g009a.gif" src-large="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g009a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC4204446/figure/F9/" ref="ordinalpos=4230&amp;ncbi_uid=6744496&amp;link_uid=PMC4204446" href="/pmc/articles/PMC4204446/"><img src="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g009b.gif" src-large="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g009b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4204446/" ref="ordinalpos=4230&amp;ncbi_uid=6744496&amp;link_uid=PMC4204446" image-link="/pmc/articles/PMC4204446/figure/F9/" class="imagepopup">Figures 9A and 9B.  From: Updates on Psoriasis and Cutaneous Oncology. </a></p><div class="supp"><p class="details"><div>Inactivating mutations of PTCH prevent inhibition of SMO leading to constitutive activation of the hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This leads to tumor proliferation and survival.</div></p></div><div class="aux"><div class="resc">Neal Bhatia, et al. J Clin Aesthet Dermatol. 2014 July;7(7 Suppl SC2S5-S22):S5-S22.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4230&amp;ncbi_uid=6744496&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4204446/" ref="ordinalpos=4230&amp;ncbi_uid=6744496&amp;link_uid=PMC4204446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4231.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3712174/figure/F1/" ref="ordinalpos=4231&amp;ncbi_uid=5104814&amp;link_uid=PMC3712174" href="/pmc/articles/PMC3712174/"><img src="/pmc/articles/instance/3712174/bin/roj-31-57-g001.gif" src-large="/pmc/articles/instance/3712174/bin/roj-31-57-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3712174/" ref="ordinalpos=4231&amp;ncbi_uid=5104814&amp;link_uid=PMC3712174" image-link="/pmc/articles/PMC3712174/figure/F1/" class="imagepopup">Fig. 1.  From: Enhancement of radiation effect using beta-lapachone and underlying mechanism. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of beta-lapachone (β-Lap)-induced cell death. HQ, hydroquinone; SQ, semiquinone; NAD(P)H, nicotinamide adenine dinucleotide phosphate.</div></p></div><div class="aux"><div class="resc">Ki Jung Ahn, et al. Radiat Oncol J. 2013 June;31(2):57-65.</div><p class="links"><a class="dblinks" href="/pubmed/23865001" ref="ordinalpos=4231&amp;ncbi_uid=5104814&amp;link_uid=23865001">Citation</a><a class="dblinks" href="/pmc/articles/PMC3712174/" ref="ordinalpos=4231&amp;ncbi_uid=5104814&amp;link_uid=PMC3712174">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4232.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4093421/figure/F1/" ref="ordinalpos=4232&amp;ncbi_uid=6379775&amp;link_uid=PMC4093421" href="/pmc/articles/PMC4093421/"><img src="/pmc/articles/instance/4093421/bin/jadp-04-182-g01.gif" src-large="/pmc/articles/instance/4093421/bin/jadp-04-182-g01.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4093421/" ref="ordinalpos=4232&amp;ncbi_uid=6379775&amp;link_uid=PMC4093421" image-link="/pmc/articles/PMC4093421/figure/F1/" class="imagepopup">Figure 1.  From: Enzalutamide: A New Agent for the Prostate Cancer Treatment Armamentarium. </a></p><div class="supp"><p class="details"><div> Figure 1. Enzalutamide’s mechanism of action on three steps in the androgen receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </div></p></div><div class="aux"><div class="resc">Stephanie B. McCutcheon. J Adv Pract Oncol. 2013 May-Jun;4(3):182-185.</div><p class="links"><a class="dblinks" href="/pubmed/25031999" ref="ordinalpos=4232&amp;ncbi_uid=6379775&amp;link_uid=25031999">Citation</a><a class="dblinks" href="/pmc/articles/PMC4093421/" ref="ordinalpos=4232&amp;ncbi_uid=6379775&amp;link_uid=PMC4093421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4233.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3560972/figure/F5/" ref="ordinalpos=4233&amp;ncbi_uid=4599258&amp;link_uid=PMC3560972" href="/pmc/articles/PMC3560972/"><img src="/pmc/articles/instance/3560972/bin/2001-1326-1-24-5.gif" src-large="/pmc/articles/instance/3560972/bin/2001-1326-1-24-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3560972/" ref="ordinalpos=4233&amp;ncbi_uid=4599258&amp;link_uid=PMC3560972" image-link="/pmc/articles/PMC3560972/figure/F5/" class="imagepopup">Figure 5.  From: Versatile inhibitory effects of the flavonoid-derived PI3K/Akt inhibitor, LY294002, on ATP-binding cassette transporters that characterize stem cells. </a></p><div class="supp"><p class="details"><div>Summary of inhibitory effects of LY294002 on the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and ABC transporters that characterize stem cells.</div></p></div><div class="aux"><div class="resc">Yasuo Imai, et al. Clin Transl Med. 2012;1:24-24.</div><p class="links"><a class="dblinks" href="/pubmed/23369170" ref="ordinalpos=4233&amp;ncbi_uid=4599258&amp;link_uid=23369170">Citation</a><a class="dblinks" href="/pmc/articles/PMC3560972/" ref="ordinalpos=4233&amp;ncbi_uid=4599258&amp;link_uid=PMC3560972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4234.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3944789/figure/F1/" ref="ordinalpos=4234&amp;ncbi_uid=5920281&amp;link_uid=PMC3944789" href="/pmc/articles/PMC3944789/"><img src="/pmc/articles/instance/3944789/bin/fncel-08-00065-g001.gif" src-large="/pmc/articles/instance/3944789/bin/fncel-08-00065-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3944789/" ref="ordinalpos=4234&amp;ncbi_uid=5920281&amp;link_uid=PMC3944789" image-link="/pmc/articles/PMC3944789/figure/F1/" class="imagepopup">FIGURE 1.  From: CXCL12 chemokine and its receptors as major players in the interactions between immune and nervous systems. </a></p><div class="supp"><p class="details"><div><b>CXCL12 chemokine <span class="highlight" style="background-color:">signaling</span></b>. CXCL12 acts through its receptors CXCR4 and CXCR7. CXCR4 stimulation leads to the activation of numerous <span class="highlight" style="background-color:">signaling</span> pathways depending on the cell types while CXCR7 has mainly been shown to be involved in scavenging CXCL12, although it can activate a MAP kinase <span class="highlight" style="background-color:">pathway</span> through β-arrestin in several systems.</div></p></div><div class="aux"><div class="resc">Alice Guyon. Front Cell Neurosci. 2014;8:65.</div><p class="links"><a class="dblinks" href="/pubmed/24639628" ref="ordinalpos=4234&amp;ncbi_uid=5920281&amp;link_uid=24639628">Citation</a><a class="dblinks" href="/pmc/articles/PMC3944789/" ref="ordinalpos=4234&amp;ncbi_uid=5920281&amp;link_uid=PMC3944789">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4235.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3780793/figure/F1/" ref="ordinalpos=4235&amp;ncbi_uid=5357294&amp;link_uid=PMC3780793" href="/pmc/articles/PMC3780793/"><img src="/pmc/articles/instance/3780793/bin/nihms503327f1.gif" src-large="/pmc/articles/instance/3780793/bin/nihms503327f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3780793/" ref="ordinalpos=4235&amp;ncbi_uid=5357294&amp;link_uid=PMC3780793" image-link="/pmc/articles/PMC3780793/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Pathway</span>-based analysis of primary biliary cirrhosis genome-wide association studies. </a></p><div class="supp"><p class="details"><div>Genes identified in the phosphatidylinositol and hedgehog <span class="highlight" style="background-color:">signaling</span> pathways. Genes reaching statistical significance for the phosphatidylinositol <span class="highlight" style="background-color:">signaling</span> system (panel a) and the hedgehog <span class="highlight" style="background-color:">pathway</span> (panel b) are shown by color highlights (red, Italian dataset; blue, Canadian dataset)</div></p></div><div class="aux"><div class="resc">SP Kar, et al. Genes Immun. 2013 April;14(3):179-186.</div><p class="links"><a class="dblinks" href="/pubmed/23392275" ref="ordinalpos=4235&amp;ncbi_uid=5357294&amp;link_uid=23392275">Citation</a><a class="dblinks" href="/pmc/articles/PMC3780793/" ref="ordinalpos=4235&amp;ncbi_uid=5357294&amp;link_uid=PMC3780793">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4236.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3663770/figure/F1/" ref="ordinalpos=4236&amp;ncbi_uid=4923200&amp;link_uid=PMC3663770" href="/pmc/articles/PMC3663770/"><img src="/pmc/articles/instance/3663770/bin/1471-213X-13-15-1.gif" src-large="/pmc/articles/instance/3663770/bin/1471-213X-13-15-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3663770/" ref="ordinalpos=4236&amp;ncbi_uid=4923200&amp;link_uid=PMC3663770" image-link="/pmc/articles/PMC3663770/figure/F1/" class="imagepopup">Figure 1.  From: A bright single-cell resolution live imaging reporter of Notch <span class="highlight" style="background-color:">signaling</span> in the mouse. </a></p><div class="supp"><p class="details"><div><b><i>CBF:H2B-Venus </i></b><b>construct design and validation in MEF cells. </b><i>CBF:H2B-Venus </i>MEFs are derived from E13.5 transgenic embryos and transfected with a <i>pCX:NICD </i>construct. Ectopic expression of NICD activates the canonical Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and in turn leads to transgene expression, which can be assessed using flow cytometry. FACS analysis shows transgenic <i>CBF:H2B-Venus </i>MEFs comprise a Venus negative and Venus positive population and display an increase in fluorescence upon ectopic NICD expression.</div></p></div><div class="aux"><div class="resc">Sonja Nowotschin, et al. BMC Dev Biol. 2013;13:15-15.</div><p class="links"><a class="dblinks" href="/pubmed/23617465" ref="ordinalpos=4236&amp;ncbi_uid=4923200&amp;link_uid=23617465">Citation</a><a class="dblinks" href="/pmc/articles/PMC3663770/" ref="ordinalpos=4236&amp;ncbi_uid=4923200&amp;link_uid=PMC3663770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4237.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3302441/figure/F1/" ref="ordinalpos=4237&amp;ncbi_uid=4261630&amp;link_uid=PMC3302441" href="/pmc/articles/PMC3302441/"><img src="/pmc/articles/instance/3302441/bin/zmb9991094300001.gif" src-large="/pmc/articles/instance/3302441/bin/zmb9991094300001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3302441/" ref="ordinalpos=4237&amp;ncbi_uid=4261630&amp;link_uid=PMC3302441" image-link="/pmc/articles/PMC3302441/figure/F1/" class="imagepopup">Fig 1.  From: Hepatic Deletion of SIRT1 Decreases Hepatocyte Nuclear Factor 1?/Farnesoid X Receptor <span class="highlight" style="background-color:">Signaling</span> and Induces Formation of Cholesterol Gallstones in Mice. </a></p><div class="supp"><p class="details"><div>Loss of SIRT1 reduces the activity of FXR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in primary hepatocytes. (A) SIRT1 deficiency in primary hepatocytes reduces the induction of FXR target genes by GW4064. *, <i>P</i> &lt; 0.05. (B) Overexpression of SIRT1 in primary hepatocytes restores the expression of FXR targets. Primary hepatocytes from control and SIRT1 LKO mice were infected with lentiviruses expressing GFP or SIRT1. The mRNA levels of FXR target genes were analyzed by qPCR. *, <i>P</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Aparna Purushotham, et al. Mol Cell Biol. 2012 April;32(7):1226-1236.</div><p class="links"><a class="dblinks" href="/pubmed/22290433" ref="ordinalpos=4237&amp;ncbi_uid=4261630&amp;link_uid=22290433">Citation</a><a class="dblinks" href="/pmc/articles/PMC3302441/" ref="ordinalpos=4237&amp;ncbi_uid=4261630&amp;link_uid=PMC3302441">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4238.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2949153/figure/fig7/" ref="ordinalpos=4238&amp;ncbi_uid=2803567&amp;link_uid=PMC2949153" href="/pmc/articles/PMC2949153/"><img src="/pmc/articles/instance/2949153/bin/jcbfm2009214f7.gif" src-large="/pmc/articles/instance/2949153/bin/jcbfm2009214f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2949153/" ref="ordinalpos=4238&amp;ncbi_uid=2803567&amp;link_uid=PMC2949153" image-link="/pmc/articles/PMC2949153/figure/fig7/" class="imagepopup">Figure 7.  From: Intact lipid rafts regulate HIV-1 Tat protein-induced activation of the Rho <span class="highlight" style="background-color:">signaling</span> and upregulation of P-glycoprotein in brain endothelial cells. </a></p><div class="supp"><p class="details"><div>The proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of Tat-induced upregulation of P-gp expression in human brain endothelial cells. The HIV-1 protein Tat, acting via intact lipid rafts, activates RhoA and ROCK kinases, leading to inhibition of MLCP (via phosphorylation of MYPT-1) and increased phosphorylation of MLC. This study provides novel evidence that these events are involved in the regulation of P-gp expression.</div></p></div><div class="aux"><div class="resc">Yu Zhong, et al. J Cereb Blood Flow Metab. 2010 March;30(3):522-533.</div><p class="links"><a class="dblinks" href="/pubmed/19794400" ref="ordinalpos=4238&amp;ncbi_uid=2803567&amp;link_uid=19794400">Citation</a><a class="dblinks" href="/pmc/articles/PMC2949153/" ref="ordinalpos=4238&amp;ncbi_uid=2803567&amp;link_uid=PMC2949153">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4239.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3085983/figure/F3/" ref="ordinalpos=4239&amp;ncbi_uid=2931468&amp;link_uid=PMC3085983" href="/pmc/articles/PMC3085983/"><img src="/pmc/articles/instance/3085983/bin/nihms276215f3.gif" src-large="/pmc/articles/instance/3085983/bin/nihms276215f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3085983/" ref="ordinalpos=4239&amp;ncbi_uid=2931468&amp;link_uid=PMC3085983" image-link="/pmc/articles/PMC3085983/figure/F3/" class="imagepopup">Figure 3.  From: Effects of Germline Mutations in the Ras/MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> on Adaptive Behavior: Cardiofaciocutaneous Syndrome and Noonan Syndrome. </a></p><div class="supp"><p class="details"><div>Distribution of adaptive behavior scores in individuals with CFC and NS, grouped by the Ras/MAPK <span class="highlight" style="background-color:">pathway</span> gene in which a mutation was identified.</div></p></div><div class="aux"><div class="resc">Elizabeth I. Pierpont, et al. Am J Med Genet A. ;152A(3):591-600.</div><p class="links"><a class="dblinks" href="/pubmed/20186801" ref="ordinalpos=4239&amp;ncbi_uid=2931468&amp;link_uid=20186801">Citation</a><a class="dblinks" href="/pmc/articles/PMC3085983/" ref="ordinalpos=4239&amp;ncbi_uid=2931468&amp;link_uid=PMC3085983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4240.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2867239/figure/F2/" ref="ordinalpos=4240&amp;ncbi_uid=2642276&amp;link_uid=PMC2867239" href="/pmc/articles/PMC2867239/"><img src="/pmc/articles/instance/2867239/bin/nihms-199205-f0002.gif" src-large="/pmc/articles/instance/2867239/bin/nihms-199205-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2867239/" ref="ordinalpos=4240&amp;ncbi_uid=2642276&amp;link_uid=PMC2867239" image-link="/pmc/articles/PMC2867239/figure/F2/" class="imagepopup">Figure 2. The core Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: The Primary Cilium as a Hedgehog Signal Transduction Machine. </a></p><div class="supp"><p class="details"><div>The vertebrate Hedgehog (Hh) <span class="highlight" style="background-color:">pathway</span> in the absence (A) or presence (B) of Sonic hedgehog (Shh) ligand. (A) In the absence of ligand, the Shh receptor Ptch1 prevents the activation of another transmembrane domain protein, Smo. In this state, full length Gli2 and Gli3 are proteolytically processed into a smaller repressor form. (B) Upon Shh ligand binding to Ptch1, the inhibition on Smo is relieved and activates full-length Gli proteins and blocks production of Gli repressors.</div></p></div><div class="aux"><div class="resc">Sarah C. Goetz, et al. Methods Cell Biol. ;94:199-222.</div><p class="links"><a class="dblinks" href="/pubmed/20362092" ref="ordinalpos=4240&amp;ncbi_uid=2642276&amp;link_uid=20362092">Citation</a><a class="dblinks" href="/pmc/articles/PMC2867239/" ref="ordinalpos=4240&amp;ncbi_uid=2642276&amp;link_uid=PMC2867239">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4241.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3679475/figure/F1/" ref="ordinalpos=4241&amp;ncbi_uid=5004963&amp;link_uid=PMC3679475" href="/pmc/articles/PMC3679475/"><img src="/pmc/articles/instance/3679475/bin/fendo-04-00071-g001.gif" src-large="/pmc/articles/instance/3679475/bin/fendo-04-00071-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3679475/" ref="ordinalpos=4241&amp;ncbi_uid=5004963&amp;link_uid=PMC3679475" image-link="/pmc/articles/PMC3679475/figure/F1/" class="imagepopup">Figure 1.  From: Adipokines Mediate Inflammation and Insulin Resistance. </a></p><div class="supp"><p class="details"><div><b>Inflammatory adipokines suppress insulin <span class="highlight" style="background-color:">signaling</span> resulting in insulin resistance</b>. IRS1/2 phosphorylated on specific tyrosine residues activates the phosphatidylinositol 3-kinase (PI3K)-AKT/protein kinase B (PKB) <span class="highlight" style="background-color:">pathway</span> and Ras-mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">pathway</span>. PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates metabolic processes such as glucose uptake (muscle and adipocytes), glycogen synthesis (muscle and liver), protein synthesis (muscle and liver), and gluconeogenesis (liver). Inflammatory signals, TNF-α, IL-6, LPS, and saturated free fatty acid, activate inhibitory molecules such as SOCS and JNK to suppress insulin <span class="highlight" style="background-color:">signaling</span> resulting in insulin resistance. PI3K dependent PDK1 activation is negatively regulated by phospholipid phosphatases such as phosphatase and tensin homolog (PTEN) that degrade PIP<sub>3</sub>.</div></p></div><div class="aux"><div class="resc">Hyokjoon Kwon, et al. Front Endocrinol (Lausanne). 2013;4:71.</div><p class="links"><a class="dblinks" href="/pubmed/23781214" ref="ordinalpos=4241&amp;ncbi_uid=5004963&amp;link_uid=23781214">Citation</a><a class="dblinks" href="/pmc/articles/PMC3679475/" ref="ordinalpos=4241&amp;ncbi_uid=5004963&amp;link_uid=PMC3679475">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4242.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2593881/figure/F3/" ref="ordinalpos=4242&amp;ncbi_uid=1521812&amp;link_uid=PMC2593881" href="/pmc/articles/PMC2593881/"><img src="/pmc/articles/instance/2593881/bin/nihms14106f3.gif" src-large="/pmc/articles/instance/2593881/bin/nihms14106f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2593881/" ref="ordinalpos=4242&amp;ncbi_uid=1521812&amp;link_uid=PMC2593881" image-link="/pmc/articles/PMC2593881/figure/F3/" class="imagepopup">Figure 3.  From: Loss of Raf Kinase Inhibitor Protein Promotes Cell Proliferation and Migration of Human Hepatoma Cells. </a></p><div class="supp"><p class="details"><div>The role of RKIP in IGF-I-induced ERK/MAPK <span class="highlight" style="background-color:">pathway</span> activation in HCC. (<i>A</i>) Expression levels of RKIP protein and downstream components of the ERK/MAPK <span class="highlight" style="background-color:">pathway</span> in HCC cell lines. Cell lysates of FOCUS, Huh7, Hep3B and HepG2 cells containing 100 µg of total protein were immunoblotted for RKIP, MEK1, ERK and their phosphorylated forms; actin served as a loading control. (<i>B</i>) The bar charts represent the results of densitometric analysis and reveal the ratio of RKIP and actin, phospho-MEK1 and total MEK1, and phospho-ERK and total ERK. (<i>C</i>) Ectopic expression of RKIP blocked IGF-I-induced ERK/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in FOCUS cells. Cells were transiently transfected with either RKIP or vector plasmid. After 24 hours serum starvation, cells were treated with 100 nM of IGF-I (+) for 15 minutes or untreated (−). Cell lysates were prepared and immunoblotted for RKIP and downstream <span class="highlight" style="background-color:">signaling</span> components of the <span class="highlight" style="background-color:">pathway</span> (c-Raf, MEK1, ERK and phosphorylated form); actin served as a loading control. (<i>D</i>) The bar charts depict the results of densitometric analysis from (<i>C</i>), and shown as the ratio of RKIP and actin, phospho-c-Raf and total c-Raf, phospho-MEK1 and total MEK1, and phospho-ERK and total ERK. The results are presented as the mean ± SE of triplicate experiments. (E) Knock-down of RKIP expression with siRNA induced the activation of IGF-I-induced ERK/MAPK <span class="highlight" style="background-color:">signaling</span> in HepG2 cells. Cells were transfected with either control si-RNA (C-siRNA) or RKIP-siRNA using the DharmaFECT 4 transfection reagent. Cells were treated with 100 nM of IGF-I (+) for 15 minutes or untreated (−) 24 hours after serum starvation followed by immunoblot for RKIP and downstream <span class="highlight" style="background-color:">signaling</span> components of the <span class="highlight" style="background-color:">pathway</span> (c-Raf, MEK1, ERK and phosphorylated form); actin served as a loading control. (<i>F</i>) The bar charts depict the results of densitometric analysis from (<i>E</i>), and are expressed as the ratio of RKIP and actin, phospho-c-Raf and total c-Raf, phospho-MEK1 and total MEK1, and phospho-ERK and total ERK.</div></p></div><div class="aux"><div class="resc">Han Chu Lee, et al. Gastroenterology. ;131(4):1208-1217.</div><p class="links"><a class="dblinks" href="/pubmed/17030190" ref="ordinalpos=4242&amp;ncbi_uid=1521812&amp;link_uid=17030190">Citation</a><a class="dblinks" href="/pmc/articles/PMC2593881/" ref="ordinalpos=4242&amp;ncbi_uid=1521812&amp;link_uid=PMC2593881">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4243.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4138085/figure/pone-0104518-g002/" ref="ordinalpos=4243&amp;ncbi_uid=6520022&amp;link_uid=PMC4138085" href="/pmc/articles/PMC4138085/"><img src="/pmc/articles/instance/4138085/bin/pone.0104518.g002.gif" src-large="/pmc/articles/instance/4138085/bin/pone.0104518.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4138085/" ref="ordinalpos=4243&amp;ncbi_uid=6520022&amp;link_uid=PMC4138085" image-link="/pmc/articles/PMC4138085/figure/pone-0104518-g002/" class="imagepopup">Figure 2. Selected pathways enriched by GSEA analysis..  From: Transcriptional Profiling in Rat Hair Follicles following Simulated Blast Insult: A New Diagnostic Tool for Traumatic Brain Injury. </a></p><div class="supp"><p class="details"><div>Red indicates that the gene is increased and green indicates that the gene is decreased in mRNA levels. (A) Factors involved in TLR/MAPK <span class="highlight" style="background-color:">pathway</span>; (B) factors involved in JAK/STAT <span class="highlight" style="background-color:">pathway</span> pathways for GH, prolactin, and interleukin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Jing Zhang, et al. PLoS One. 2014;9(8):e104518.</div><p class="links"><a class="dblinks" href="/pubmed/25136963" ref="ordinalpos=4243&amp;ncbi_uid=6520022&amp;link_uid=25136963">Citation</a><a class="dblinks" href="/pmc/articles/PMC4138085/" ref="ordinalpos=4243&amp;ncbi_uid=6520022&amp;link_uid=PMC4138085">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4244.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3625555/figure/fig3/" ref="ordinalpos=4244&amp;ncbi_uid=4911817&amp;link_uid=PMC3625555" href="/pmc/articles/PMC3625555/"><img src="/pmc/articles/instance/3625555/bin/ECAM2013-548498.003.gif" src-large="/pmc/articles/instance/3625555/bin/ECAM2013-548498.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3625555/" ref="ordinalpos=4244&amp;ncbi_uid=4911817&amp;link_uid=PMC3625555" image-link="/pmc/articles/PMC3625555/figure/fig3/" class="imagepopup">Figure 3.  From: A Systems Biology-Based Investigation into the Pharmacological Mechanisms of Wu Tou Tang Acting on Rheumatoid Arthritis by Integrating Network Analysis. </a></p><div class="supp"><p class="details"><div>Effects of active compounds in Wu Tou Tang on neuroactive ligand-receptor interaction (a) and calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (b) by their candidate effector molecules marked with red panes. This plot is modified from KEGG <span class="highlight" style="background-color:">pathway</span> map.</div></p></div><div class="aux"><div class="resc">Yanqiong Zhang, et al. Evid Based Complement Alternat Med. 2013;2013:548498.</div><p class="links"><a class="dblinks" href="/pubmed/23690848" ref="ordinalpos=4244&amp;ncbi_uid=4911817&amp;link_uid=23690848">Citation</a><a class="dblinks" href="/pmc/articles/PMC3625555/" ref="ordinalpos=4244&amp;ncbi_uid=4911817&amp;link_uid=PMC3625555">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4245.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3393606/figure/F6/" ref="ordinalpos=4245&amp;ncbi_uid=4113101&amp;link_uid=PMC3393606" href="/pmc/articles/PMC3393606/"><img src="/pmc/articles/instance/3393606/bin/1756-0500-5-268-6.gif" src-large="/pmc/articles/instance/3393606/bin/1756-0500-5-268-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3393606/" ref="ordinalpos=4245&amp;ncbi_uid=4113101&amp;link_uid=PMC3393606" image-link="/pmc/articles/PMC3393606/figure/F6/" class="imagepopup">Figure 6.  From: Ribozyme-mediated gene knock down strategy to dissect the consequences of PDGF stimulation in vascular smooth muscle cells. </a></p><div class="supp"><p class="details"><div><b>Working model describing the different effects on PDI-A3 and HSP-60 phosphorylation due to PDGFR-β knock-down.</b> Ribozyme might impair the β-β and α-β dimer formations decreasing the opportunities of PDGF-BB binding. This inhibition might reduce (if not abolish) the PDGFR-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and conversely increase the PDGFR-α <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Caterina Lande, et al. BMC Res Notes. 2012;5:268-268.</div><p class="links"><a class="dblinks" href="/pubmed/22676333" ref="ordinalpos=4245&amp;ncbi_uid=4113101&amp;link_uid=22676333">Citation</a><a class="dblinks" href="/pmc/articles/PMC3393606/" ref="ordinalpos=4245&amp;ncbi_uid=4113101&amp;link_uid=PMC3393606">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4246.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2671254/figure/F11-8010/" ref="ordinalpos=4246&amp;ncbi_uid=1721562&amp;link_uid=PMC2671254" href="/pmc/articles/PMC2671254/"><img src="/pmc/articles/instance/2671254/bin/zjh0050980100011.gif" src-large="/pmc/articles/instance/2671254/bin/zjh0050980100011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2671254/" ref="ordinalpos=4246&amp;ncbi_uid=1721562&amp;link_uid=PMC2671254" image-link="/pmc/articles/PMC2671254/figure/F11-8010/" class="imagepopup">Figure 11.  From: Caveolin-1 (P132L), a Common Breast Cancer Mutation, Confers Mammary Cell Invasiveness and Defines a Novel Stem Cell/Metastasis-Associated Gene Signature. </a></p><div class="supp"><p class="details"><div>Results from <span class="highlight" style="background-color:">pathway</span> analysis with ASSESS. We also performed <span class="highlight" style="background-color:">pathway</span> analysis using ASSESS (analysis of sample set enrichment scores). Note that ER-α-, Wnt-, TGF-β-, and Rho GTPase-dependent <span class="highlight" style="background-color:">signaling</span> pathways all appear to be activated by Cav-1 (P132L), among others.</div></p></div><div class="aux"><div class="resc">Gloria Bonuccelli, et al. Am J Pathol. 2009 May;174(5):1650-1662.</div><p class="links"><a class="dblinks" href="/pubmed/19395651" ref="ordinalpos=4246&amp;ncbi_uid=1721562&amp;link_uid=19395651">Citation</a><a class="dblinks" href="/pmc/articles/PMC2671254/" ref="ordinalpos=4246&amp;ncbi_uid=1721562&amp;link_uid=PMC2671254">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4247.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1502453/figure/F5/" ref="ordinalpos=4247&amp;ncbi_uid=707874&amp;link_uid=PMC1502453" href="/pmc/articles/PMC1502453/"><img src="/pmc/articles/instance/1502453/bin/zpq0250625570005.gif" src-large="/pmc/articles/instance/1502453/bin/zpq0250625570005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1502453/" ref="ordinalpos=4247&amp;ncbi_uid=707874&amp;link_uid=PMC1502453" image-link="/pmc/articles/PMC1502453/figure/F5/" class="imagepopup">Fig. 5. From: Identification of a Hoxc8-regulated transcriptional network in mouse embryo fibroblast cells. </a></p><div class="supp"><p class="details"><div>The <i>Hoxc8</i>-regulated transcriptional network <i>in vivo</i>. <i>Hoxc8</i> can regulate the cross-talk between Wnt, BMP, FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by direct down-regulating <i>Ncam</i>, <i>opn</i>, <i>pedf</i>, and <i>Zac1</i> and up-regulating <i>Cdh11 in vivo</i>. Direct pathways are indicated by solid lines and arrows, and indirect consequences of <span class="highlight" style="background-color:">pathway</span> activation and inactivation are indicated by dotted lines.</div></p></div><div class="aux"><div class="resc">Haiyan Lei, et al. Proc Natl Acad Sci U S A. 2006 July 5;103(27):10305-10309.</div><p class="links"><a class="dblinks" href="/pubmed/16793922" ref="ordinalpos=4247&amp;ncbi_uid=707874&amp;link_uid=16793922">Citation</a><a class="dblinks" href="/pmc/articles/PMC1502453/" ref="ordinalpos=4247&amp;ncbi_uid=707874&amp;link_uid=PMC1502453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4248.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2825052/figure/F1/" ref="ordinalpos=4248&amp;ncbi_uid=2267428&amp;link_uid=PMC2825052" href="/pmc/articles/PMC2825052/"><img src="/pmc/articles/instance/2825052/bin/nihms-166476-f0003.gif" src-large="/pmc/articles/instance/2825052/bin/nihms-166476-f0003.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2825052/" ref="ordinalpos=4248&amp;ncbi_uid=2267428&amp;link_uid=PMC2825052" image-link="/pmc/articles/PMC2825052/figure/F1/" class="imagepopup">Figure 1.  From: Fine-tuning of GPCR activity by receptor-interacting proteins. </a></p><div class="supp"><p class="details"><div>GPCR <span class="highlight" style="background-color:">signaling</span> can be mediated by receptor-interacting proteins. A) Certain GPCR-interacting proteins can act as mediators of agonist-induced receptor <span class="highlight" style="background-color:">signaling</span>, independent of G protein- or arrestin-mediated <span class="highlight" style="background-color:">signaling</span> pathways. An example of this phenomenon is the interaction of the non-receptor tyrosine kinase Jak2 with the angiotensin AT1 receptor. Association of Jak2 with the phosphatase SHP-2 and stimulation of AT1 with angiotensin II together promote Jak2 association with the AT1 receptor and initiation of Jak2-dependent <span class="highlight" style="background-color:">signaling</span>. Activated Jak2 can phosphorylate members of the STAT family of transcription factors, leading to STAT dimerization, translocation into the nucleus, and regulation of genes controlling cell growth. B) In other cases, the agonist-dependent dissociation of an interacting protein from a GPCR can alter the activity of an intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. For example, the interaction between the p85 regulatory subunit of phosphatidylinositol 3-kinase (PI3K) and the somatostatin type II receptor (SSTR2) is disrupted by agonist stimulation, leading to reduced PI3K-mediated Akt <span class="highlight" style="background-color:">signaling</span> and suppression of cell survival pathways.</div></p></div><div class="aux"><div class="resc">Stefanie L. Ritter, et al. Nat Rev Mol Cell Biol. ;10(12):819-830.</div><p class="links"><a class="dblinks" href="/pubmed/19935667" ref="ordinalpos=4248&amp;ncbi_uid=2267428&amp;link_uid=19935667">Citation</a><a class="dblinks" href="/pmc/articles/PMC2825052/" ref="ordinalpos=4248&amp;ncbi_uid=2267428&amp;link_uid=PMC2825052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4249.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2330318/figure/fig7/" ref="ordinalpos=4249&amp;ncbi_uid=1375470&amp;link_uid=PMC2330318" href="/pmc/articles/PMC2330318/"><img src="/pmc/articles/instance/2330318/bin/pp1470188f07.gif" src-large="/pmc/articles/instance/2330318/bin/pp1470188f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2330318/" ref="ordinalpos=4249&amp;ncbi_uid=1375470&amp;link_uid=PMC2330318" image-link="/pmc/articles/PMC2330318/figure/fig7/" class="imagepopup">Figure 7. From: Nitric Oxide Triggers Phosphatidic Acid Accumulation via Phospholipase D during Auxin-Induced Adventitious Root Formation in Cucumber. </a></p><div class="supp"><p class="details"><div>Schematic model of the PL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induced by auxin during AR formation in cucumber. NO acts downstream of auxin (IAA) to trigger PIP, PIP<sub>2</sub>, and PA accumulation. PLD is involved in PA formation induced by NO. PLD could be activated by PIP and PIP<sub>2</sub> increases and/or by other NO-mediated pathways (see “Discussion” for details). PLD-derived PA induces AR formation. The solid arrow denotes the enzymatic <span class="highlight" style="background-color:">pathway</span>. Dashed arrows denote regulatory pathways. The question mark denotes a yet undescribed <span class="highlight" style="background-color:">pathway</span> for PLD activation in cucumber.</div></p></div><div class="aux"><div class="resc">María Luciana Lanteri, et al. Plant Physiol. 2008 May;147(1):188-198.</div><p class="links"><a class="dblinks" href="/pubmed/18375601" ref="ordinalpos=4249&amp;ncbi_uid=1375470&amp;link_uid=18375601">Citation</a><a class="dblinks" href="/pmc/articles/PMC2330318/" ref="ordinalpos=4249&amp;ncbi_uid=1375470&amp;link_uid=PMC2330318">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4250.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2871634/figure/F4/" ref="ordinalpos=4250&amp;ncbi_uid=2367253&amp;link_uid=PMC2871634" href="/pmc/articles/PMC2871634/"><img src="/pmc/articles/instance/2871634/bin/gnl-1-101-g004.gif" src-large="/pmc/articles/instance/2871634/bin/gnl-1-101-g004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2871634/" ref="ordinalpos=4250&amp;ncbi_uid=2367253&amp;link_uid=PMC2871634" image-link="/pmc/articles/PMC2871634/figure/F4/" class="imagepopup">Fig. 4.  From: Molecular Pathogenesis of Hepatitis-B-virus-associated Hepatocellular Carcinoma. </a></p><div class="supp"><p class="details"><div>WNT/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. WNT <span class="highlight" style="background-color:">signaling</span> leads to a stabilization of β-catenin, the chief downstream effector of the WNT <span class="highlight" style="background-color:">pathway</span>. In the activated state WNT binds to its receptor, Frizzled, which ultimately leads to inactivation of GSK3β (glycogen synthase kinase 3β) via activation of the DSH (Disheveled). This leads to hypophosphorylation of β-catenin and its release from a complex with APC (adenomatosis polyposis coli) and AXIN, with ensuing nuclear translocation of β-catenin where it binds to the various target genes.</div></p></div><div class="aux"><div class="resc">Neung Hwa Park, et al. Gut Liver. 2007 December;1(2):101-117.</div><p class="links"><a class="dblinks" href="/pubmed/20485626" ref="ordinalpos=4250&amp;ncbi_uid=2367253&amp;link_uid=20485626">Citation</a><a class="dblinks" href="/pmc/articles/PMC2871634/" ref="ordinalpos=4250&amp;ncbi_uid=2367253&amp;link_uid=PMC2871634">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4251.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3725503/figure/nutrients-05-02231-f001/" ref="ordinalpos=4251&amp;ncbi_uid=5135057&amp;link_uid=PMC3725503" href="/pmc/articles/PMC3725503/"><img src="/pmc/articles/instance/3725503/bin/nutrients-05-02231-g001.gif" src-large="/pmc/articles/instance/3725503/bin/nutrients-05-02231-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3725503/" ref="ordinalpos=4251&amp;ncbi_uid=5135057&amp;link_uid=PMC3725503" image-link="/pmc/articles/PMC3725503/figure/nutrients-05-02231-f001/" class="imagepopup">Figure 1.  From: The Role of Mechanistic Target of Rapamycin (mTOR) Complexes <span class="highlight" style="background-color:">Signaling</span> in the Immune Responses. </a></p><div class="supp"><p class="details"><div>Network of the mechanistic Target of Rapamycin (mTOR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Figure 1 shows diagram summarizing the inputs and outputs of mTOR <span class="highlight" style="background-color:">signaling</span> network. mTORC1 integrates multiple inputs from <span class="highlight" style="background-color:">signaling</span> pathways including insulin, growth factors, energy, stress, mitogens, and amino acids. Additionally PI3K-mTOR also receives signals from the immune microenvironment such as T Cell Receptors (TCR), and co-stimulatory molecules CD28 (Cluster of Differentiation 28), as well as interleukins. mTORC2 complex activates Akt by phosphorylating it on serine 473, and indirectly activates mTORC1, which in turns phosphorylates its effector S6K on threonine 389.</div></p></div><div class="aux"><div class="resc">Ghada A. Soliman. Nutrients. 2013 June;5(6):2231-2257.</div><p class="links"><a class="dblinks" href="/pubmed/23783557" ref="ordinalpos=4251&amp;ncbi_uid=5135057&amp;link_uid=23783557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3725503/" ref="ordinalpos=4251&amp;ncbi_uid=5135057&amp;link_uid=PMC3725503">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4252.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3597489/figure/pntd-0002117-g004/" ref="ordinalpos=4252&amp;ncbi_uid=4726225&amp;link_uid=PMC3597489" href="/pmc/articles/PMC3597489/"><img src="/pmc/articles/instance/3597489/bin/pntd.0002117.g004.gif" src-large="/pmc/articles/instance/3597489/bin/pntd.0002117.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3597489/" ref="ordinalpos=4252&amp;ncbi_uid=4726225&amp;link_uid=PMC3597489" image-link="/pmc/articles/PMC3597489/figure/pntd-0002117-g004/" class="imagepopup">Figure 4. TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Conjunctival MicroRNA Expression in Inflammatory Trachomatous Scarring. </a></p><div class="supp"><p class="details"><div>Genes highlighted in yellow are predicted targets of miR differentially regulated in N v TSI (35 miR) (FC&gt;3 p&lt;0.05). Genes highlighted in green are not predicted targets.</div></p></div><div class="aux"><div class="resc">Tamsyn Derrick, et al. PLoS Negl Trop Dis. 2013 March;7(3):e2117.</div><p class="links"><a class="dblinks" href="/pubmed/23516655" ref="ordinalpos=4252&amp;ncbi_uid=4726225&amp;link_uid=23516655">Citation</a><a class="dblinks" href="/pmc/articles/PMC3597489/" ref="ordinalpos=4252&amp;ncbi_uid=4726225&amp;link_uid=PMC3597489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4253.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2720065/figure/F8/" ref="ordinalpos=4253&amp;ncbi_uid=2034553&amp;link_uid=PMC2720065" href="/pmc/articles/PMC2720065/"><img src="/pmc/articles/instance/2720065/bin/nihms-92061-f0009.gif" src-large="/pmc/articles/instance/2720065/bin/nihms-92061-f0009.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2720065/" ref="ordinalpos=4253&amp;ncbi_uid=2034553&amp;link_uid=PMC2720065" image-link="/pmc/articles/PMC2720065/figure/F8/" class="imagepopup">Figure 8.  From: Macrophages generate reactive oxygen species in response to minimally oxidized LDL: TLR4- and Syk-dependent activation of Nox2. </a></p><div class="supp"><p class="details"><div>A proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of mmLDL-induced, TLR4/Syk-dependent ROS generation and proinflammatory cytokine expression.</div></p></div><div class="aux"><div class="resc">Yun Soo Bae, et al. Circ Res. ;104(2):210-218.</div><p class="links"><a class="dblinks" href="/pubmed/19096031" ref="ordinalpos=4253&amp;ncbi_uid=2034553&amp;link_uid=19096031">Citation</a><a class="dblinks" href="/pmc/articles/PMC2720065/" ref="ordinalpos=4253&amp;ncbi_uid=2034553&amp;link_uid=PMC2720065">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4254.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4191885/figure/pcbi-1003884-g004/" ref="ordinalpos=4254&amp;ncbi_uid=6721715&amp;link_uid=PMC4191885" href="/pmc/articles/PMC4191885/"><img src="/pmc/articles/instance/4191885/bin/pcbi.1003884.g004.gif" src-large="/pmc/articles/instance/4191885/bin/pcbi.1003884.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4191885/" ref="ordinalpos=4254&amp;ncbi_uid=6721715&amp;link_uid=PMC4191885" image-link="/pmc/articles/PMC4191885/figure/pcbi-1003884-g004/" class="imagepopup">Figure 4. Illustration of Gata3 ChIP-seq binding sites on the selected genes..  From: Reconstruction of the Gene Regulatory Network Involved in the Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span> with a Potential Role in Early Development of the Mouse Brain. </a></p><div class="supp"><p class="details"><div>Red boxes indicate the binding peak locations called by MACS program. (A) <i>Sufu</i> and <i>Gsk3b</i> in the Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) <i>Slit2/3</i> involved in axon guidance. (C) <i>Nfasc, Mapt</i> and <i>Limk1</i> regulating brain development. (D) <i>Qdpr, Th</i> and <i>Slc18a1</i> involved in neurotransmitter synthesis and release.</div></p></div><div class="aux"><div class="resc">Jinhua Liu, et al. PLoS Comput Biol. 2014 October;10(10):e1003884.</div><p class="links"><a class="dblinks" href="/pubmed/25299227" ref="ordinalpos=4254&amp;ncbi_uid=6721715&amp;link_uid=25299227">Citation</a><a class="dblinks" href="/pmc/articles/PMC4191885/" ref="ordinalpos=4254&amp;ncbi_uid=6721715&amp;link_uid=PMC4191885">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4255.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3359946/figure/fig7/" ref="ordinalpos=4255&amp;ncbi_uid=4764138&amp;link_uid=PMC3359946" href="/pmc/articles/PMC3359946/"><img src="/pmc/articles/instance/3359946/bin/AJRCMB464507fig7.gif" src-large="/pmc/articles/instance/3359946/bin/AJRCMB464507fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3359946/" ref="ordinalpos=4255&amp;ncbi_uid=4764138&amp;link_uid=PMC3359946" image-link="/pmc/articles/PMC3359946/figure/fig7/" class="imagepopup">Figure 7. From: Hyperoxia-Induced LC3B Interacts with the Fas Apoptotic <span class="highlight" style="background-color:">Pathway</span> in Epithelial Cell Death. </a></p><div class="supp"><p class="details"><div>Schemata of proposed pathways. Autophagy promotes cell survival after hyperoxia via interacting with Fas-mediated apoptosis pathways. Hyperoxia induces LC3B expression, a key initiator of autophagy. In this study, we describe dynamic interactions between LC3B and the apoptotic regulator Fas. Forced expression of LC3B was found to diminish apoptotic <span class="highlight" style="background-color:">pathway</span> initiation by the Fas-dependent death-inducing <span class="highlight" style="background-color:">signaling</span> complex. The interactions of LC3B and Fas are dependent on caveolin-1 and its phosphorylation state at Y-14. The balance between LC3B/cav-1/Fas complex determines the fate of the cell.</div></p></div><div class="aux"><div class="resc">Akihiko Tanaka, et al. Am J Respir Cell Mol Biol. 2012 April;46(4):507-514.</div><p class="links"><a class="dblinks" href="/pubmed/22095627" ref="ordinalpos=4255&amp;ncbi_uid=4764138&amp;link_uid=22095627">Citation</a><a class="dblinks" href="/pmc/articles/PMC3359946/" ref="ordinalpos=4255&amp;ncbi_uid=4764138&amp;link_uid=PMC3359946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4256.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3858975/figure/F2/" ref="ordinalpos=4256&amp;ncbi_uid=5591839&amp;link_uid=PMC3858975" href="/pmc/articles/PMC3858975/"><img src="/pmc/articles/instance/3858975/bin/nihms362673f2.gif" src-large="/pmc/articles/instance/3858975/bin/nihms362673f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3858975/" ref="ordinalpos=4256&amp;ncbi_uid=5591839&amp;link_uid=PMC3858975" image-link="/pmc/articles/PMC3858975/figure/F2/" class="imagepopup">Figure 2.  From: Ubiquitylation and the Fanconi Anemia <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Deactivation of the Fanconi anemia <span class="highlight" style="background-color:">pathway</span> by deubiquitylation<br />The <span class="highlight" style="background-color:">signaling</span> that takes place in the activation of DNA repair must be attenuated when DNA repair has concluded. The ID complex is deubiquitylated by USP1 whose activity is enhanced by interaction with UAF1. USP1 has a parallel role in the deubiquitylation of PCNA. USP1/UAF1 substrate affinity is regulated by auxiliary factors such as ELG1, which is required for the deubiquitylation of PCNA specifically. The DUBs are themselves subject to regulation on both a transcriptional and post-transcriptional level. Importantly, USP1 is polyubiquitylated as DNA replication concludes. Polyubiquitylated USP1 is targeted for proteasomal degradation.</div></p></div><div class="aux"><div class="resc">Elizabeth Garner, et al. FEBS Lett. ;585(18):10.1016/j.febslet.2011.04.078.</div><p class="links"><a class="dblinks" href="/pubmed/21605559" ref="ordinalpos=4256&amp;ncbi_uid=5591839&amp;link_uid=21605559">Citation</a><a class="dblinks" href="/pmc/articles/PMC3858975/" ref="ordinalpos=4256&amp;ncbi_uid=5591839&amp;link_uid=PMC3858975">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4257.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3911272/figure/F5/" ref="ordinalpos=4257&amp;ncbi_uid=6343533&amp;link_uid=PMC3911272" href="/pmc/articles/PMC3911272/"><img src="/pmc/articles/instance/3911272/bin/zmb9991002440005.gif" src-large="/pmc/articles/instance/3911272/bin/zmb9991002440005.jpg" alt="FIG 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3911272/" ref="ordinalpos=4257&amp;ncbi_uid=6343533&amp;link_uid=PMC3911272" image-link="/pmc/articles/PMC3911272/figure/F5/" class="imagepopup">FIG 5.  From: Crumbs3 Is Essential for Proper Epithelial Development and Viability. </a></p><div class="supp"><p class="details"><div>Investigation of <span class="highlight" style="background-color:">signaling</span> pathways in <i>Crumbs3</i> knockout lungs. (A) Western blot of p-ERM and ERM protein levels among +/+ and −/− lungs at E18.5. β-Actin and α-tubulin are shown as loading controls. Samples were run on nonadjacent lanes of the same gel. (B) Frozen sections from E18.5 +/+ and −/− mouse lungs were stained for Yap1 (transcription factor downstream of Hippo <span class="highlight" style="background-color:">pathway</span>) (×600; zoom, 3). (C) Western blot of p-Yap1 (Hippo <span class="highlight" style="background-color:">pathway</span> activation) and Hes1 (Notch <span class="highlight" style="background-color:">pathway</span> activation) among +/+ and −/− lungs at E18.5. α-Tubulin is shown as the loading control. Samples were run on nonadjacent lanes of the same gel.</div></p></div><div class="aux"><div class="resc">Eileen L. Whiteman, et al. Mol Cell Biol. 2014 January;34(1):43-56.</div><p class="links"><a class="dblinks" href="/pubmed/24164893" ref="ordinalpos=4257&amp;ncbi_uid=6343533&amp;link_uid=24164893">Citation</a><a class="dblinks" href="/pmc/articles/PMC3911272/" ref="ordinalpos=4257&amp;ncbi_uid=6343533&amp;link_uid=PMC3911272">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4258.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3226342/figure/F2/" ref="ordinalpos=4258&amp;ncbi_uid=3736539&amp;link_uid=PMC3226342" href="/pmc/articles/PMC3226342/"><img src="/pmc/articles/instance/3226342/bin/JCI61127.f2.gif" src-large="/pmc/articles/instance/3226342/bin/JCI61127.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3226342/" ref="ordinalpos=4258&amp;ncbi_uid=3736539&amp;link_uid=PMC3226342" image-link="/pmc/articles/PMC3226342/figure/F2/" class="imagepopup">Figure 2. Proposed function of the EGFR in pituitary corticotrophs..  From: A new medical therapy for Cushing disease?. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Relationship between <i>Pomc</i> expression (which encodes a precursor protein that can be cleaved to generate ACTH) and pituitary cell growth mediated by the MAPK <span class="highlight" style="background-color:">pathway</span>. Corticotroph adenomas have aberrant MAPK <span class="highlight" style="background-color:">pathway</span> activation, resulting in pituitary cell growth and increased <i>Pomc</i> expression (4). (<b>B</b>) Use of a TKI reduces corticotroph adenoma growth and <i>Pomc</i> expression, which is mediated by the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.    </div></p></div><div class="aux"><div class="resc">Fredric E. Wondisford. J Clin Invest. 2011 December 1;121(12):4621-4623.</div><p class="links"><a class="dblinks" href="/pubmed/22105164" ref="ordinalpos=4258&amp;ncbi_uid=3736539&amp;link_uid=22105164">Citation</a><a class="dblinks" href="/pmc/articles/PMC3226342/" ref="ordinalpos=4258&amp;ncbi_uid=3736539&amp;link_uid=PMC3226342">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4259.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3089989/figure/F2/" ref="ordinalpos=4259&amp;ncbi_uid=2939401&amp;link_uid=PMC3089989" href="/pmc/articles/PMC3089989/"><img src="/pmc/articles/instance/3089989/bin/fpsyt-02-00005-g002.gif" src-large="/pmc/articles/instance/3089989/bin/fpsyt-02-00005-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3089989/" ref="ordinalpos=4259&amp;ncbi_uid=2939401&amp;link_uid=PMC3089989" image-link="/pmc/articles/PMC3089989/figure/F2/" class="imagepopup">Figure 2.  From: Genome-Wide Expression Analysis Reveals Diverse Effects of Acute Nicotine Exposure on Neuronal Function-Related Genes and Pathways. </a></p><div class="supp"><p class="details"><div><b>Gene expression pattern of the P38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Genes shown in green are down-regulated, whereas those shown in red are induced by acute nicotine treatment. The genes significantly regulated by nicotine are those with closed blue edges. To demonstrate the expression pattern of the <span class="highlight" style="background-color:">pathway</span>, all the genes with expression ratios &gt; 1.1 or &lt;0.90 relative to the controls are also shown. The genes in white with light gray edges are either not regulated by nicotine (expression ratios between 0.9 and 1.1) or were not measured in our analysis. The <span class="highlight" style="background-color:">pathway</span> was drawn by the IPA program.</div></p></div><div class="aux"><div class="resc">Ju Wang, et al. Front Psychiatry. 2011;2:5.</div><p class="links"><a class="dblinks" href="/pubmed/21556275" ref="ordinalpos=4259&amp;ncbi_uid=2939401&amp;link_uid=21556275">Citation</a><a class="dblinks" href="/pmc/articles/PMC3089989/" ref="ordinalpos=4259&amp;ncbi_uid=2939401&amp;link_uid=PMC3089989">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4260.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3932224/figure/fig3/" ref="ordinalpos=4260&amp;ncbi_uid=5947832&amp;link_uid=PMC3932224" href="/pmc/articles/PMC3932224/"><img src="/pmc/articles/instance/3932224/bin/IJCB2014-715867.003.gif" src-large="/pmc/articles/instance/3932224/bin/IJCB2014-715867.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3932224/" ref="ordinalpos=4260&amp;ncbi_uid=5947832&amp;link_uid=PMC3932224" image-link="/pmc/articles/PMC3932224/figure/fig3/" class="imagepopup">Figure 3.  From: 2-Cys Peroxiredoxins: Emerging Hubs Determining Redox Dependency of Mammalian <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>Summary of regulatory networks centered by typical 2-Cys Prxs. Functional correlation and interaction among the <span class="highlight" style="background-color:">signaling</span> proteins and typical 2-Cys Prxs are reconstituted into a network model using the <i><span class="highlight" style="background-color:">Pathway</span> Studio</i> software (Ariadne Genomics Inc., USA). All molecules are shown as gene symbol. Direct and indirect regulations were indicated in gray lines and dotted gray lines, respectively. Green arrows indicate regulation by protein modifications. Purple lines indicate direct protein-protein binding.</div></p></div><div class="aux"><div class="resc">Jinah Park, et al. Int J Cell Biol. 2014;2014:715867.</div><p class="links"><a class="dblinks" href="/pubmed/24672551" ref="ordinalpos=4260&amp;ncbi_uid=5947832&amp;link_uid=24672551">Citation</a><a class="dblinks" href="/pmc/articles/PMC3932224/" ref="ordinalpos=4260&amp;ncbi_uid=5947832&amp;link_uid=PMC3932224">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4261.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3228139/figure/F4/" ref="ordinalpos=4261&amp;ncbi_uid=4096894&amp;link_uid=PMC3228139" href="/pmc/articles/PMC3228139/"><img src="/pmc/articles/instance/3228139/bin/onc9991108270004.gif" src-large="/pmc/articles/instance/3228139/bin/onc9991108270004.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3228139/" ref="ordinalpos=4261&amp;ncbi_uid=4096894&amp;link_uid=PMC3228139" image-link="/pmc/articles/PMC3228139/figure/F4/" class="imagepopup">Figure 4. From: Global Profiling of <span class="highlight" style="background-color:">Signaling</span> Networks: Study of Breast Cancer Stem Cells and Potential Regulation. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis. This chart shows the differences in the gene <span class="highlight" style="background-color:">signaling</span> pathways between SP and non-SP cells. The threshold of significance was defined by the <i>p</i>-value and FDR (<i>p</i> &lt; .01; FDR &lt; .01). All these proteins show increased enrichment, <i>p</i>-values, and FDRs.<br />Abbreviations: FDR, false discovery rate; GnRH, gonadotropin-releasing hormone; MAPK, mitogen-activated protein kinase; mTOR, mammalian target of rapamycin; SP, side population.</div></p></div><div class="aux"><div class="resc">Hongxia Wang, et al. Oncologist. 2011 July;16(7):966-979.</div><p class="links"><a class="dblinks" href="/pubmed/21665913" ref="ordinalpos=4261&amp;ncbi_uid=4096894&amp;link_uid=21665913">Citation</a><a class="dblinks" href="/pmc/articles/PMC3228139/" ref="ordinalpos=4261&amp;ncbi_uid=4096894&amp;link_uid=PMC3228139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4262.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3603002/figure/F6/" ref="ordinalpos=4262&amp;ncbi_uid=4727246&amp;link_uid=PMC3603002" href="/pmc/articles/PMC3603002/"><img src="/pmc/articles/instance/3603002/bin/nihms207342f6.gif" src-large="/pmc/articles/instance/3603002/bin/nihms207342f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3603002/" ref="ordinalpos=4262&amp;ncbi_uid=4727246&amp;link_uid=PMC3603002" image-link="/pmc/articles/PMC3603002/figure/F6/" class="imagepopup">Figure 6.  From: QUANTITATIVE PHOSPHOPROTEOMIC ANALYSIS OF T CELL RECEPTOR <span class="highlight" style="background-color:">SIGNALING</span> IN DIABETES PRONE AND RESISTANT MICE. </a></p><div class="supp"><p class="details"><div>T cell <span class="highlight" style="background-color:">signaling</span> pathways and protein phosphorylation sites identified to be differential between NOD and B6g7 mice in stimulated cells. In red: high tyrosine-phosphorylation upregulated in NOD; pink: low tyrosine-phosphorylation in NOD; blue: high tyrosine phosphorylation upregulated in B6g7; light blue: low tyrosine phosphorylation upregulated in B6g7; in white: tyrosine-phosphorylation equal in NOD and B6g7 (recompiled from KEGG <span class="highlight" style="background-color:">Pathway</span> database, Refs <sup>33</sup> and <sup>34</sup>).</div></p></div><div class="aux"><div class="resc">Leo K. Iwai, et al. J Proteome Res. 2010 June 4;9(6):3135-3145.</div><p class="links"><a class="dblinks" href="/pubmed/20438120" ref="ordinalpos=4262&amp;ncbi_uid=4727246&amp;link_uid=20438120">Citation</a><a class="dblinks" href="/pmc/articles/PMC3603002/" ref="ordinalpos=4262&amp;ncbi_uid=4727246&amp;link_uid=PMC3603002">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4263.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC124530/figure/F3/" ref="ordinalpos=4263&amp;ncbi_uid=373622&amp;link_uid=PMC124530" href="/pmc/articles/PMC124530/"><img src="/pmc/articles/instance/124530/bin/pq0920169003.gif" src-large="/pmc/articles/instance/124530/bin/pq0920169003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC124530/" ref="ordinalpos=4263&amp;ncbi_uid=373622&amp;link_uid=PMC124530" image-link="/pmc/articles/PMC124530/figure/F3/" class="imagepopup">Figure 3.  From: Siderophore-mediated <span class="highlight" style="background-color:">signaling</span> regulates virulence factor production in Pseudomonas aeruginosa. </a></p><div class="supp"><p class="details"><div>Production of exotoxin A in response to the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Culture supernatants for the strains shown, which had been grown with or without the addition of pyoverdine, were serially diluted. and the presence of exotoxin A was determined with an anti-exotoxin antibody. <i>toxA</i> promoter activity was determined by measuring the amounts of β-galactosidase produced by strains carrying the <i>toxA</i>-<i>lacZ</i> reporter fusion plasmid construct pPZ-<i>toxA</i>. All SDs were less than 12%. ND, not determined.</div></p></div><div class="aux"><div class="resc">Iain L. Lamont, et al. Proc Natl Acad Sci U S A. 2002 May 14;99(10):7072-7077.</div><p class="links"><a class="dblinks" href="/pubmed/11997446" ref="ordinalpos=4263&amp;ncbi_uid=373622&amp;link_uid=11997446">Citation</a><a class="dblinks" href="/pmc/articles/PMC124530/" ref="ordinalpos=4263&amp;ncbi_uid=373622&amp;link_uid=PMC124530">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4264.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4287242/figure/Fig5/" ref="ordinalpos=4264&amp;ncbi_uid=7048538&amp;link_uid=PMC4287242" href="/pmc/articles/PMC4287242/"><img src="/pmc/articles/instance/4287242/bin/12864_2014_6784_Fig5_HTML.gif" src-large="/pmc/articles/instance/4287242/bin/12864_2014_6784_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4287242/" ref="ordinalpos=4264&amp;ncbi_uid=7048538&amp;link_uid=PMC4287242" image-link="/pmc/articles/PMC4287242/figure/Fig5/" class="imagepopup">Figure 5.  From: A comprehensive proteomic approach to identifying capacitation related proteins in boar spermatozoa. </a></p><div class="supp"><p class="details"><div> <b><span class="highlight" style="background-color:">Signaling</span> pathways associated with capacitation-related proteins.</b> The <span class="highlight" style="background-color:">pathway</span> was drawn using <span class="highlight" style="background-color:">Pathway</span> Studio 9.0 after a database search in PubMed. Red denotes the proteins that were abundantly expressed after-capacitation, while blue denotes proteins that were abundantly expressed before-capacitation. At least 8 proteins were implicated in different sperm-specific process among the 10 proteins.</div></p></div><div class="aux"><div class="resc">Woo-Sung Kwon, et al. BMC Genomics. 2014;15(1):897.</div><p class="links"><a class="dblinks" href="/pubmed/25315394" ref="ordinalpos=4264&amp;ncbi_uid=7048538&amp;link_uid=25315394">Citation</a><a class="dblinks" href="/pmc/articles/PMC4287242/" ref="ordinalpos=4264&amp;ncbi_uid=7048538&amp;link_uid=PMC4287242">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4265.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3518257/figure/fig7/" ref="ordinalpos=4265&amp;ncbi_uid=4447861&amp;link_uid=PMC3518257" href="/pmc/articles/PMC3518257/"><img src="/pmc/articles/instance/3518257/bin/ECAM2012-278652.007.gif" src-large="/pmc/articles/instance/3518257/bin/ECAM2012-278652.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3518257/" ref="ordinalpos=4265&amp;ncbi_uid=4447861&amp;link_uid=PMC3518257" image-link="/pmc/articles/PMC3518257/figure/fig7/" class="imagepopup">Figure 7.  From: The Cytotoxicity Mechanism of 6-Shogaol-Treated HeLa Human Cervical Cancer Cells Revealed by Label-Free Shotgun Proteomics and Bioinformatics Analysis. </a></p><div class="supp"><p class="details"><div>14-3-3 <span class="highlight" style="background-color:">signaling</span> canonical <span class="highlight" style="background-color:">pathway</span> from IPA. The analysis data showed that the response proteins 14-3-3 (SFN, YWHAE, YWHAG, and YWHAQ), Tubulin (TUBA4A, TUBB3, TUBB, TUBB2B, and TUBB4B), PLC (PDIA3), and VIM were involved in this <span class="highlight" style="background-color:">pathway</span>. Complex are represented by a shape with a double-line.</div></p></div><div class="aux"><div class="resc">Qun Liu, et al. Evid Based Complement Alternat Med. 2012;2012:278652.</div><p class="links"><a class="dblinks" href="/pubmed/23243437" ref="ordinalpos=4265&amp;ncbi_uid=4447861&amp;link_uid=23243437">Citation</a><a class="dblinks" href="/pmc/articles/PMC3518257/" ref="ordinalpos=4265&amp;ncbi_uid=4447861&amp;link_uid=PMC3518257">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4266.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281649/figure/F2/" ref="ordinalpos=4266&amp;ncbi_uid=4610093&amp;link_uid=PMC3281649" href="/pmc/articles/PMC3281649/"><img src="/pmc/articles/instance/3281649/bin/zbc0091297680002.gif" src-large="/pmc/articles/instance/3281649/bin/zbc0091297680002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281649/" ref="ordinalpos=4266&amp;ncbi_uid=4610093&amp;link_uid=PMC3281649" image-link="/pmc/articles/PMC3281649/figure/F2/" class="imagepopup">FIGURE 2. From: Dual Regulation of Notch1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by Adaptor Protein Fe65. </a></p><div class="supp"><p class="details"><div><b>Tip60 does not affect Fe65-mediated inhibition of Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, HEK293 cells were transfected with expression vectors for 4XCSL-Luc and β-galactosidase, along with ΔEN1, Fe65, Tip60, and the Tip60* mutant as indicated. <i>B</i>, HEK293 cells were transfected with expression vectors for 4XCSL-Luc and β-galactosidase, along with Notch1-IC, Fe65, Tip60, and the Tip60* mutant as indicated. <i>A</i> and <i>B</i>, after 48 h of transfection, the cells were lysed, and the luciferase activity was determined. The data were normalized with β-galactosidase. These results represent the mean ± S.D. of three independent experiments. <i>R.L.U</i>., relative luciferase units. We assessed the data for significant differences via Student's <i>t</i> test compared with ΔEN1 or Notch1-IC. *, <i>p</i> &lt; 0.01 (analysis of variance).</div></p></div><div class="aux"><div class="resc">Mi-Yeon Kim, et al. J Biol Chem. 2012 February 10;287(7):4690-4701.</div><p class="links"><a class="dblinks" href="/pubmed/22199353" ref="ordinalpos=4266&amp;ncbi_uid=4610093&amp;link_uid=22199353">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281649/" ref="ordinalpos=4266&amp;ncbi_uid=4610093&amp;link_uid=PMC3281649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4267.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2881761/figure/F8/" ref="ordinalpos=4267&amp;ncbi_uid=3008083&amp;link_uid=PMC2881761" href="/pmc/articles/PMC2881761/"><img src="/pmc/articles/instance/2881761/bin/zbc0261018540008.gif" src-large="/pmc/articles/instance/2881761/bin/zbc0261018540008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2881761/" ref="ordinalpos=4267&amp;ncbi_uid=3008083&amp;link_uid=PMC2881761" image-link="/pmc/articles/PMC2881761/figure/F8/" class="imagepopup">FIGURE 8. From: Thrombin and Collagen Induce a Feedback Inhibitory <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Platelets Involving Dissociation of the Catalytic Subunit of Protein Kinase A from an NF?B-I?B Complex. </a></p><div class="supp"><p class="details"><div><b>Proposed mechanism of cAMP-independent PKAc activation by dissociation of an NFκB-IκB-PKAc complex in thrombin- and collagen- stimulated platelets.</b> In platelets, both collagen and thrombin activate the PI3K <span class="highlight" style="background-color:">pathway</span> that stimulates PKC and PKB which are involved in platelet activation. However, both pathways stimulate IKK and degradation of an NFκB-IκB-PKAc complex, setting free active PKAc that phosphorylates VASP and other substrates involved in platelet inhibitory pathways. Note that in order to simplify the illustration, it was necessary to limit it to the few pathways shown that were the focus of this study. <i>Dotted lines</i> indicate that intermediary agents which transmit thrombin/collagen effects on NFκB-IκB-PKA <span class="highlight" style="background-color:">signaling</span> require further elucidation.</div></p></div><div class="aux"><div class="resc">Stepan Gambaryan, et al. J Biol Chem. 2010 June 11;285(24):18352-18363.</div><p class="links"><a class="dblinks" href="/pubmed/20356841" ref="ordinalpos=4267&amp;ncbi_uid=3008083&amp;link_uid=20356841">Citation</a><a class="dblinks" href="/pmc/articles/PMC2881761/" ref="ordinalpos=4267&amp;ncbi_uid=3008083&amp;link_uid=PMC2881761">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4268.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2727076/figure/fig8/" ref="ordinalpos=4268&amp;ncbi_uid=1944026&amp;link_uid=PMC2727076" href="/pmc/articles/PMC2727076/"><img src="/pmc/articles/instance/2727076/bin/JEB024646F8.gif" src-large="/pmc/articles/instance/2727076/bin/JEB024646F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2727076/" ref="ordinalpos=4268&amp;ncbi_uid=1944026&amp;link_uid=PMC2727076" image-link="/pmc/articles/PMC2727076/figure/fig8/" class="imagepopup">Fig. 8. From: <span class="highlight" style="background-color:">Signaling</span> to the apical membrane and to the paracellular <span class="highlight" style="background-color:">pathway</span>: changes  in the cytosolic proteome of Aedes Malpighian tubules. </a></p><div class="supp"><p class="details"><div>Hypothetical model of aedeskinin-III <span class="highlight" style="background-color:">signaling</span> to the paracellular <span class="highlight" style="background-color:">pathway</span>  in Malpighian tubules of the yellow-fever mosquito. The paracellular protein  complex consists of cytoplasmic elements such as the cytoskeleton, scaffolding  and regulatory proteins, and integral membrane proteins reaching into the  paracellular space. The proteins defining the paracellular  barrier/permeability properties are unknown in insect epithelia. ADF, actin  depolymerizing factor; GPCR, G-protein-coupled receptor;  α,β,γ, subunits of G protein; GTP, guanosine triphosphate;  PLC, phospholipase C; PIP<sub>2</sub>, phosphatidylinositol  (4,5)-bisphosphate; DAG, diacylglycerol; PKC, protein kinase C; SERCA,  sarcoplasmic and endoplasmic reticulum calcium ATPase.</div></p></div><div class="aux"><div class="resc">Klaus W. Beyenbach, et al. J Exp Biol. 2009 February 1;212(3):329-340.</div><p class="links"><a class="dblinks" href="/pubmed/19151207" ref="ordinalpos=4268&amp;ncbi_uid=1944026&amp;link_uid=19151207">Citation</a><a class="dblinks" href="/pmc/articles/PMC2727076/" ref="ordinalpos=4268&amp;ncbi_uid=1944026&amp;link_uid=PMC2727076">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4269.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3704220/figure/F1/" ref="ordinalpos=4269&amp;ncbi_uid=6415891&amp;link_uid=PMC3704220" href="/pmc/articles/PMC3704220/"><img src="/pmc/articles/instance/3704220/bin/nihms488608f1.gif" src-large="/pmc/articles/instance/3704220/bin/nihms488608f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3704220/" ref="ordinalpos=4269&amp;ncbi_uid=6415891&amp;link_uid=PMC3704220" image-link="/pmc/articles/PMC3704220/figure/F1/" class="imagepopup">Fig. 1.  From: Microfluidic interrogation and mathematical modeling of multi-regime calcium <span class="highlight" style="background-color:">signaling</span> dynamics. </a></p><div class="supp"><p class="details"><div>Pulsatile stimulation in both the acute and oscillatory regime provide experimental calcium responses critical for mechanism elucidation, in addition to previously characterized responses under continuous stimulation in the two regimes. Left- subjecting cells to continuous ligand stimulation at different concentrations can reveal the presence of multiple <span class="highlight" style="background-color:">signaling</span> regimes: oscillatory and acute. Right- Exposing cells to periodic ligand stimulation can reveal the recovery properties of <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> regimes not observable with continuous stimulation alone. Recovery properties are represented by a reduction of the amplitude of responses in later stimulation pulses in the acute regime, and skipped signals in the oscillatory regime.</div></p></div><div class="aux"><div class="resc">Andreja Jovic, et al. Integr Biol (Camb). ;5(7):932-939.</div><p class="links"><a class="dblinks" href="/pubmed/23732791" ref="ordinalpos=4269&amp;ncbi_uid=6415891&amp;link_uid=23732791">Citation</a><a class="dblinks" href="/pmc/articles/PMC3704220/" ref="ordinalpos=4269&amp;ncbi_uid=6415891&amp;link_uid=PMC3704220">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4270.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3298530/figure/F7/" ref="ordinalpos=4270&amp;ncbi_uid=3850979&amp;link_uid=PMC3298530" href="/pmc/articles/PMC3298530/"><img src="/pmc/articles/instance/3298530/bin/1471-230X-12-9-7.gif" src-large="/pmc/articles/instance/3298530/bin/1471-230X-12-9-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3298530/" ref="ordinalpos=4270&amp;ncbi_uid=3850979&amp;link_uid=PMC3298530" image-link="/pmc/articles/PMC3298530/figure/F7/" class="imagepopup">Figure 7.  From: Luteolin decreases IGF-II production and downregulates insulin-like growth factor-I receptor <span class="highlight" style="background-color:">signaling</span> in HT-29 human colon cancer cells. </a></p><div class="supp"><p class="details"><div><b>A tentative scheme for luteolin regulation of the IGF-IR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HT-29 human colon cancer cells</b>. Luteolin reduces the secretion of IGF-II and levels of IGF-IR mRNA and protein, which leads to a reduction in IGF-IR phosphorylation and a subsequent inhibition of PI3K/Akt and ERK1/2/CDC25c activation. Additionally, luteolin directly inhibits PI3K activity. These changes in IGF-I <span class="highlight" style="background-color:">signaling</span> contribute to luteolin-induced apoptosis and cell cycle arrest in HT-29 cells. *From our previously published results [21].</div></p></div><div class="aux"><div class="resc">Do Young Lim, et al. BMC Gastroenterol. 2012;12:9-9.</div><p class="links"><a class="dblinks" href="/pubmed/22269172" ref="ordinalpos=4270&amp;ncbi_uid=3850979&amp;link_uid=22269172">Citation</a><a class="dblinks" href="/pmc/articles/PMC3298530/" ref="ordinalpos=4270&amp;ncbi_uid=3850979&amp;link_uid=PMC3298530">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4271.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3116140/figure/F2/" ref="ordinalpos=4271&amp;ncbi_uid=3043164&amp;link_uid=PMC3116140" href="/pmc/articles/PMC3116140/"><img src="/pmc/articles/instance/3116140/bin/fnins-05-00078-g002.gif" src-large="/pmc/articles/instance/3116140/bin/fnins-05-00078-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3116140/" ref="ordinalpos=4271&amp;ncbi_uid=3043164&amp;link_uid=PMC3116140" image-link="/pmc/articles/PMC3116140/figure/F2/" class="imagepopup">Figure 2.  From: Dynamic Expression of Notch <span class="highlight" style="background-color:">Signaling</span> Genes in Neural Stem/Progenitor Cells. </a></p><div class="supp"><p class="details"><div><b>The core <span class="highlight" style="background-color:">pathway</span> of Notch <span class="highlight" style="background-color:">signaling</span></b>. Proneural genes such as <i>Mash1</i> and <i>Ngn2</i> promote neuronal differentiation and induce the expression of Dll1, which in turn activates Notch in neighboring cells. Upon activation of Notch, the Notch intracellular domain (NICD) is released from the transmembrane portion and transferred to the nucleus, where it forms a complex with the DNA-binding protein Rbpj and the transcriptional co-activator Maml. The NICD–Rbpj–Maml complex induces the expression of transcriptional repressor genes such as <i>Hes1</i> and <i>Hes5</i>. Hes1 and Hes5 then repress the expression of proneural genes and Dll1, thereby leading to the maintenance of neural stem/progenitor cells. Numb inhibits Notch <span class="highlight" style="background-color:">signaling</span> and induces neuronal differentiation.</div></p></div><div class="aux"><div class="resc">Hiromi Shimojo, et al. Front Neurosci. 2011;5:78.</div><p class="links"><a class="dblinks" href="/pubmed/21716644" ref="ordinalpos=4271&amp;ncbi_uid=3043164&amp;link_uid=21716644">Citation</a><a class="dblinks" href="/pmc/articles/PMC3116140/" ref="ordinalpos=4271&amp;ncbi_uid=3043164&amp;link_uid=PMC3116140">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4272.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3942836/figure/F1/" ref="ordinalpos=4272&amp;ncbi_uid=5894737&amp;link_uid=PMC3942836" href="/pmc/articles/PMC3942836/"><img src="/pmc/articles/instance/3942836/bin/WJG-20-2335-g001.gif" src-large="/pmc/articles/instance/3942836/bin/WJG-20-2335-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3942836/" ref="ordinalpos=4272&amp;ncbi_uid=5894737&amp;link_uid=PMC3942836" image-link="/pmc/articles/PMC3942836/figure/F1/" class="imagepopup">Figure 1.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a new therapeutic target in pancreatic cancer. </a></p><div class="supp"><p class="details"><div>Schematic review. Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> is activated in both autocrine and paracrine pathways. Tumor associated stroma cells play a pivotal role in tumor progression related to activation of Hh <span class="highlight" style="background-color:">signaling</span> [paracrine pathways (1) and (2)]. Induction of sonic Hh (Shh) is closely associated with activation of the nuclear factor kappa-light-chain-enhancer of activated B cells transcription factor in pancreatic cancer. Shh is produced by NF-κB activation in pancreatic cancer cells and tumor associated stroma cells. Pathways contributing to Smo and Gli1 activation include SDF-1/CXCR4, hypoxia, KRAS and MAP3K. The effects of Hh inhibitors including GDC-0449 (vismodegib), LDE225 (erismodegib), IPI-926 (saridegib), BMS-663513 and GANT-61 in clinical trials are under investigation. Dotted arrows show components of the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in tumor cells focused on in this review.</div></p></div><div class="aux"><div class="resc">Hideya Onishi, et al. World J Gastroenterol. 2014 March 7;20(9):2335-2342.</div><p class="links"><a class="dblinks" href="/pubmed/24605030" ref="ordinalpos=4272&amp;ncbi_uid=5894737&amp;link_uid=24605030">Citation</a><a class="dblinks" href="/pmc/articles/PMC3942836/" ref="ordinalpos=4272&amp;ncbi_uid=5894737&amp;link_uid=PMC3942836">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4273.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4102070/figure/fig4/" ref="ordinalpos=4273&amp;ncbi_uid=6469745&amp;link_uid=PMC4102070" href="/pmc/articles/PMC4102070/"><img src="/pmc/articles/instance/4102070/bin/SCIENTIFICA2014-208928.004.gif" src-large="/pmc/articles/instance/4102070/bin/SCIENTIFICA2014-208928.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4102070/" ref="ordinalpos=4273&amp;ncbi_uid=6469745&amp;link_uid=PMC4102070" image-link="/pmc/articles/PMC4102070/figure/fig4/" class="imagepopup">Figure 4.  From: Does B Cell Receptor <span class="highlight" style="background-color:">Signaling</span> in Chronic Lymphocytic Leukaemia Cells Differ from That in Other B Cell Types?. </a></p><div class="supp"><p class="details"><div>Points of inhibition for ibrutinib, idelalisib, fostamatinib, and Lck-i in the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Illustration of proximal signals initiated during BCR engagement and points where well described inhibitors act. (a) Lck-i acts to inhibit the most proximal <span class="highlight" style="background-color:">signaling</span> event of SFK-mediated phosphorylation of tyrosine residues within CD79. Ibrutinib acts to inhibit the ability of Btk to phosphorylate and activate PLC<i>γ</i>2. (b) Fostamatinib acts to inhibit Syk kinase activity as well as that of Lyn. Idelalisib works to inhibit PI3K<i>δ</i> to limit the formation of PIP<sub>3</sub> and has the effect of blocking membrane interaction of <span class="highlight" style="background-color:">signaling</span> proteins containing PH domains such as PDK1, Akt, and Btk.</div></p></div><div class="aux"><div class="resc">Joseph R. Slupsky. Scientifica (Cairo). 2014;2014:208928.</div><p class="links"><a class="dblinks" href="/pubmed/25101192" ref="ordinalpos=4273&amp;ncbi_uid=6469745&amp;link_uid=25101192">Citation</a><a class="dblinks" href="/pmc/articles/PMC4102070/" ref="ordinalpos=4273&amp;ncbi_uid=6469745&amp;link_uid=PMC4102070">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4274.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4079089/figure/F2/" ref="ordinalpos=4274&amp;ncbi_uid=6315567&amp;link_uid=PMC4079089" href="/pmc/articles/PMC4079089/"><img src="/pmc/articles/instance/4079089/bin/nihms582282f2.gif" src-large="/pmc/articles/instance/4079089/bin/nihms582282f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4079089/" ref="ordinalpos=4274&amp;ncbi_uid=6315567&amp;link_uid=PMC4079089" image-link="/pmc/articles/PMC4079089/figure/F2/" class="imagepopup">Figure 2.  From: Growth arrest <span class="highlight" style="background-color:">signaling</span> of the Raf/MEK/ERK <span class="highlight" style="background-color:">pathway</span> in cancer. </a></p><div class="supp"><p class="details"><div>Two different thresholds of Raf/MEK/ERK activity determine cell fate to proliferation or growth arrest. In this model, not only too low but also too high Raf/MEK/ERK activity restricts cell proliferation. Different cell types may maintain different extent of <span class="highlight" style="background-color:">pathway</span> activity between these thresholds, displaying heterogeneous sensitivity to <span class="highlight" style="background-color:">pathway</span> activity.</div></p></div><div class="aux"><div class="resc">Jong-In Park. Front Biol (Beijing). ;9(2):95-103.</div><p class="links"><a class="dblinks" href="/pubmed/24999356" ref="ordinalpos=4274&amp;ncbi_uid=6315567&amp;link_uid=24999356">Citation</a><a class="dblinks" href="/pmc/articles/PMC4079089/" ref="ordinalpos=4274&amp;ncbi_uid=6315567&amp;link_uid=PMC4079089">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4275.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3909204/figure/pone-0087714-g001/" ref="ordinalpos=4275&amp;ncbi_uid=5783602&amp;link_uid=PMC3909204" href="/pmc/articles/PMC3909204/"><img src="/pmc/articles/instance/3909204/bin/pone.0087714.g001.gif" src-large="/pmc/articles/instance/3909204/bin/pone.0087714.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3909204/" ref="ordinalpos=4275&amp;ncbi_uid=5783602&amp;link_uid=PMC3909204" image-link="/pmc/articles/PMC3909204/figure/pone-0087714-g001/" class="imagepopup">Figure 1. Drosophila EGFR and FGFR <span class="highlight" style="background-color:">signaling</span> pathways..  From: EGFR and FGFR Pathways Have Distinct Roles in Drosophila Mushroom Body Development and Ethanol-Induced Behavior. </a></p><div class="supp"><p class="details"><div>(A) Schematic of EGFR <span class="highlight" style="background-color:">pathway</span> genes in this study. The EGFR ligands Vein (vn) and Spitz (spi) are synthesized as precursor peptides, which are cleaved to an active form by the protease Rhomboid (rho). These processed ligands are then secreted, and bind the receptor, Egfr. Bound receptor dimers initiate <span class="highlight" style="background-color:">signaling</span> that can activate a number of downstream pathways, most notably ERK and JNK MAP kinase cascades. (B) Schematic of FGFR <span class="highlight" style="background-color:">pathway</span> genes. Two FGFR receptors, Breathless (btl) and Heartless (htl) are activated by separate ligands – Heartless by Pyramus (pyr) or Thysbee (ths), and Breathless by Branchless (bnl). FGFR activation can also initiate multiple downstream <span class="highlight" style="background-color:">signaling</span> pathways, including MAPK cascades.</div></p></div><div class="aux"><div class="resc">Ian F. G. King, et al. PLoS One. 2014;9(1):e87714.</div><p class="links"><a class="dblinks" href="/pubmed/24498174" ref="ordinalpos=4275&amp;ncbi_uid=5783602&amp;link_uid=24498174">Citation</a><a class="dblinks" href="/pmc/articles/PMC3909204/" ref="ordinalpos=4275&amp;ncbi_uid=5783602&amp;link_uid=PMC3909204">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4276.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2469272/figure/Fig2/" ref="ordinalpos=4276&amp;ncbi_uid=1460077&amp;link_uid=PMC2469272" href="/pmc/articles/PMC2469272/"><img src="/pmc/articles/instance/2469272/bin/11095_2008_9571_Fig2_HTML.gif" src-large="/pmc/articles/instance/2469272/bin/11095_2008_9571_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2469272/" ref="ordinalpos=4276&amp;ncbi_uid=1460077&amp;link_uid=PMC2469272" image-link="/pmc/articles/PMC2469272/figure/Fig2/" class="imagepopup">Fig. 2.  From: Discovery and Development of Toll-Like Receptor 4 (TLR4) Antagonists: A New Paradigm for Treating Sepsis and Other Diseases. </a></p><div class="supp"><p class="details"><div>LPS <span class="highlight" style="background-color:">signaling</span> [modified from O’Neill and Bowie (10) with permission]. TLR4 requires four <span class="highlight" style="background-color:">signaling</span> adaptors to function upon activation by LPS. Similarly to TLR2 it uses MAL to recruit MyD88 and to activate the NFκB <span class="highlight" style="background-color:">pathway</span> and p38 and JNK MAPK pathways. A second <span class="highlight" style="background-color:">signaling</span> cascade triggered by the LPS-TLR4 interaction involves TRAM. TRAM recruits TRIF which activates pathways involving TBK1 to IRF3, TRAF6 to NFκB and RIP1 to apoptosis.</div></p></div><div class="aux"><div class="resc">Carlos G. Leon, et al. Pharm Res. 2008 August;25(8):1751-1761.</div><p class="links"><a class="dblinks" href="/pubmed/18493843" ref="ordinalpos=4276&amp;ncbi_uid=1460077&amp;link_uid=18493843">Citation</a><a class="dblinks" href="/pmc/articles/PMC2469272/" ref="ordinalpos=4276&amp;ncbi_uid=1460077&amp;link_uid=PMC2469272">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4277.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3605164/figure/F2/" ref="ordinalpos=4277&amp;ncbi_uid=5440326&amp;link_uid=PMC3605164" href="/pmc/articles/PMC3605164/"><img src="/pmc/articles/instance/3605164/bin/isl-4-359-g2.gif" src-large="/pmc/articles/instance/3605164/bin/isl-4-359-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3605164/" ref="ordinalpos=4277&amp;ncbi_uid=5440326&amp;link_uid=PMC3605164" image-link="/pmc/articles/PMC3605164/figure/F2/" class="imagepopup">Figure. From: New insight into the mechanisms underlying the function of the incretin hormone glucagon-like peptide-1 in pancreatic ?-cells. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> An illustration of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Without Wnt ligand stimulation, β-cat is trapped within the “destruction complex,” phosphorylated by the protein kinase GSK-3 and CK-1α at Ser33 and adjacent Ser positions, and subsequently degraded by proteasome (left). Following Wnt ligand stimulation and Dishvelled (Dvl) activation, β-cat escapes the trapping, enters the nucleus and forms the bipartite transcription factor cat/TCF, which leads to the stimulation of Wnt target gene expression (middle). GLP-1 was shown to activate cAMP-dependent protein kinase A (PKA), and stimulate β-cat Ser675 phosphorylation, which is positively associated with its nuclear translocation and Wnt target gene expression (right).</div></p></div><div class="aux"><div class="resc">Xiaoquan Xiong, et al. Islets. 2012 November 1;4(6):359-365.</div><p class="links"><a class="dblinks" href="/pubmed/23314611" ref="ordinalpos=4277&amp;ncbi_uid=5440326&amp;link_uid=23314611">Citation</a><a class="dblinks" href="/pmc/articles/PMC3605164/" ref="ordinalpos=4277&amp;ncbi_uid=5440326&amp;link_uid=PMC3605164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4278.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2453100/figure/F7/" ref="ordinalpos=4278&amp;ncbi_uid=1440681&amp;link_uid=PMC2453100" href="/pmc/articles/PMC2453100/"><img src="/pmc/articles/instance/2453100/bin/zee0070841090007.gif" src-large="/pmc/articles/instance/2453100/bin/zee0070841090007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2453100/" ref="ordinalpos=4278&amp;ncbi_uid=1440681&amp;link_uid=PMC2453100" image-link="/pmc/articles/PMC2453100/figure/F7/" class="imagepopup">Figure 7.  From: Activation of Phosphatidylinositol 3-Kinase <span class="highlight" style="background-color:">Signaling</span> Promotes Aberrant Pituitary Growth in a Mouse Model of Thyroid-Stimulating Hormone-Secreting Pituitary Tumors. </a></p><div class="supp"><p class="details"><div>Schematic representation of the activated PI3K-AKT <span class="highlight" style="background-color:">pathway</span> and its downstream effectors that contribute to the tumorigenesis of TSHomas in TRβ<sup>PV/PV</sup> mice. Aberrant activation of the PI3K-AKT-mTOR-p70<sup>S6K</sup> <span class="highlight" style="background-color:">pathway</span> in pituitaries of TRβ<sup>PV/PV</sup> mice increases protein synthesis and cell growth. AKT decreases cyclin D1 degradation by inhibiting GSK-3β activity through phosphorylation (17). AKT also inhibits cell apoptosis through phosphorylation of BAD and FoxO3a. Phosphorylated BAD is incapable of inhibiting the activity of antiapoptotic protein Bcl-2 (33,34). Similar to the phosphorylated BAD, phosphorylated FoxO3a is retained in cytoplasm and thus is incapable of activating the expression of the proapoptotic protein Bim (35). All these processes are reversed by LY294002 <i>in vivo</i> by deactivating the PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. By decreasing the subsequent phosphorylation events after the AKT activation, LY ultimately reduces tumor growth by inhibiting cell proliferation and promoting apoptosis in TSHomas of TRβ<sup>PV/PV</sup> mice.</div></p></div><div class="aux"><div class="resc">Changxue Lu, et al. Endocrinology. 2008 July;149(7):3339-3345.</div><p class="links"><a class="dblinks" href="/pubmed/18356276" ref="ordinalpos=4278&amp;ncbi_uid=1440681&amp;link_uid=18356276">Citation</a><a class="dblinks" href="/pmc/articles/PMC2453100/" ref="ordinalpos=4278&amp;ncbi_uid=1440681&amp;link_uid=PMC2453100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4279.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3574110/figure/pone-0056122-g004/" ref="ordinalpos=4279&amp;ncbi_uid=4672909&amp;link_uid=PMC3574110" href="/pmc/articles/PMC3574110/"><img src="/pmc/articles/instance/3574110/bin/pone.0056122.g004.gif" src-large="/pmc/articles/instance/3574110/bin/pone.0056122.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3574110/" ref="ordinalpos=4279&amp;ncbi_uid=4672909&amp;link_uid=PMC3574110" image-link="/pmc/articles/PMC3574110/figure/pone-0056122-g004/" class="imagepopup">Figure 4. Molecular network of genes related to atherosclerosis <span class="highlight" style="background-color:">signaling</span> and hepatic fibrosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Differential Effects of Drug Interventions and Dietary Lifestyle in Developing Type 2 Diabetes and Complications: A Systems Biology Analysis in LDLr-/- Mice. </a></p><div class="supp"><p class="details"><div>Genes differentially expressed in dietary lifestyle intervention (DLI), compared to HFD were subjected to network analysis (Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis). The network of genes associated with processes “Cellular Growth and Proliferation”, “Connective Tissue Development and Function” and “Hepatic System Development and Function” (network score 32) is represented in the figure. Genes or gene products are represented as nodes, and the biological relationships between two nodes are represented as edges (lines). The nodes of the network are colored according to log<sub>2</sub> gene expression changes in the DLI vs. HFD comparison (red: upregulation, green: downregulation). The bar graph associated with each node represents log<sub>2</sub> expression changes in chow (1st bar) and DLI groups (2nd bar) vs. HFD group, highlighting that all represented genes change in equivalent direction in chow and DLI conditions. The function “Overlay: Canonical <span class="highlight" style="background-color:">Pathway</span>” was used to highlight network genes associated with “atherosclerosis <span class="highlight" style="background-color:">signaling</span>” (11 genes, top enriched <span class="highlight" style="background-color:">pathway</span>) and “hepatic fibrosis/hepatic stellate cell activation” (9 genes, 3rd enriched <span class="highlight" style="background-color:">pathway</span>). All genes associated with these pathways, as well as majority of genes in the network are downregulated in DLI group, indicating withdrawal of pathogenic signals upon dietary lifestyle intervention.</div></p></div><div class="aux"><div class="resc">Marijana Radonjic, et al. PLoS One. 2013;8(2):e56122.</div><p class="links"><a class="dblinks" href="/pubmed/23457508" ref="ordinalpos=4279&amp;ncbi_uid=4672909&amp;link_uid=23457508">Citation</a><a class="dblinks" href="/pmc/articles/PMC3574110/" ref="ordinalpos=4279&amp;ncbi_uid=4672909&amp;link_uid=PMC3574110">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4280.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508147/figure/F2/" ref="ordinalpos=4280&amp;ncbi_uid=5674757&amp;link_uid=PMC3508147" href="/pmc/articles/PMC3508147/"><img src="/pmc/articles/instance/3508147/bin/nihms413894f2.gif" src-large="/pmc/articles/instance/3508147/bin/nihms413894f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508147/" ref="ordinalpos=4280&amp;ncbi_uid=5674757&amp;link_uid=PMC3508147" image-link="/pmc/articles/PMC3508147/figure/F2/" class="imagepopup">Figure 2. Activation of stress response pathways suppresses germ cell loss in response to DGLA feeding.  From: Stress response pathways protect germ cells from omega-6 polyunsaturated fatty acid-mediated toxicity in C. elegans. </a></p><div class="supp"><p class="details"><div>A) Schematic of the conserved insulin/IGF-1 and TGF-b <span class="highlight" style="background-color:">signaling</span> pathways in <i>C. elegans.</i> B, C, and D) Mutations in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the TGF-b <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulate the sterility responses. Graphs display the percentage of worms in a population that are sterile sterility fed on the various concentrations of DGLA. Error bars, S.E.M, *p≤0.05, **p≤0.01, determined by two tailed Student’s t-test.</div></p></div><div class="aux"><div class="resc">Christopher M. Webster, et al. Dev Biol. ;373(1):14-25.</div><p class="links"><a class="dblinks" href="/pubmed/23064027" ref="ordinalpos=4280&amp;ncbi_uid=5674757&amp;link_uid=23064027">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508147/" ref="ordinalpos=4280&amp;ncbi_uid=5674757&amp;link_uid=PMC3508147">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4281.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2718703/figure/pbio-1000172-g015/" ref="ordinalpos=4281&amp;ncbi_uid=2015720&amp;link_uid=PMC2718703" href="/pmc/articles/PMC2718703/"><img src="/pmc/articles/instance/2718703/bin/pbio.1000172.g015.gif" src-large="/pmc/articles/instance/2718703/bin/pbio.1000172.g015.jpg" alt="Figure 15" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2718703/" ref="ordinalpos=4281&amp;ncbi_uid=2015720&amp;link_uid=PMC2718703" image-link="/pmc/articles/PMC2718703/figure/pbio-1000172-g015/" class="imagepopup">Figure 15. Mathematical model of the GPCR-cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Persistent cAMP-Signals Triggered by Internalized G-Protein-Coupled Receptors. </a></p><div class="supp"><p class="details"><div>A model of spatial partial differential equations was generated to simulate the temporal and spatial dynamics of GPCR <span class="highlight" style="background-color:">signaling</span>. (A) Schematic illustration of the basic model. The receptor, G-proteins, and adenylyl cyclase are placed on the plasma membrane, whereas ATP, cAMP, and PDE4 are freely diffusing in the cytosol. PKA is cytosolic, but nondiffusing. (B) Model with the addition of an intracellular <span class="highlight" style="background-color:">signaling</span> compartment (ICSC). To simulate GPCR-cAMP <span class="highlight" style="background-color:">signaling</span> from an ICSC, we placed G-proteins and adenylyl cyclase also on an intracellular membrane and simulated the internalization of both GPCR and ligand to this compartment. (C) Results of simulations. A cell was transiently stimulated by application and removal of the ligand from the extracellular compartment. In a first simulation in which <span class="highlight" style="background-color:">signaling</span> from the ICSC was not implemented (no ICSC), the cAMP response was completely reversible. On the contrary, inclusion of the ICSC in the model lead to sustained cAMP production. Also note the different levels and spatial patterns of PKA activation predicted in the presence or absence of the ICSC.</div></p></div><div class="aux"><div class="resc">Davide Calebiro, et al. PLoS Biol. 2009 August;7(8):e1000172.</div><p class="links"><a class="dblinks" href="/pubmed/19688034" ref="ordinalpos=4281&amp;ncbi_uid=2015720&amp;link_uid=19688034">Citation</a><a class="dblinks" href="/pmc/articles/PMC2718703/" ref="ordinalpos=4281&amp;ncbi_uid=2015720&amp;link_uid=PMC2718703">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4282.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2981611/figure/F1/" ref="ordinalpos=4282&amp;ncbi_uid=3591092&amp;link_uid=PMC2981611" href="/pmc/articles/PMC2981611/"><img src="/pmc/articles/instance/2981611/bin/nihms-229692-f0001.gif" src-large="/pmc/articles/instance/2981611/bin/nihms-229692-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2981611/" ref="ordinalpos=4282&amp;ncbi_uid=3591092&amp;link_uid=PMC2981611" image-link="/pmc/articles/PMC2981611/figure/F1/" class="imagepopup">Figure 1. Wnt <span class="highlight" style="background-color:">Signaling</span>: A Basic View.  From: Wnt and Mammary Stem Cells: Hormones Can't Fly Wingless. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> is broadly divided into ß-catenin-dependent and ß-catenin-independent pathways. The ß-catenin-dependent <span class="highlight" style="background-color:">pathway</span> is involved in the stabilization of ß-catenin by the dissociation of a destruction complex that normally phosphorylates and targets ß-catenin for degradation. These signals require Fzd receptors to recruit the LRP co-receptor. ß-catenin-independent pathways encompass a broad range of outputs, dictated by the receptor context and downstream signals chosen. The newly discovered receptor tyrosine kinases Ror and Ryk provide another level of Wnt regulation in addition to Fzd. Within the Wnt family, there are 19 individual Wnt genes along with 10 different Frizzled receptors, illustrating the complexity governing this <span class="highlight" style="background-color:">pathway</span> and the multitude of Wnt-receptor combinations. The secreted Wnt inhibitors, Sfrps and Wifs, can regulate <span class="highlight" style="background-color:">signaling</span> in the extracellular space by sequestering Wnts away from their receptors. Wnt/ß-catenin-independent pathways can antagonize the ß-catenin-dependent <span class="highlight" style="background-color:">pathway</span>, revealing that these pathways are far more integrated than originally thought.</div></p></div><div class="aux"><div class="resc">Kevin Roarty, et al. Curr Opin Pharmacol. ;10(6):643-649.</div><p class="links"><a class="dblinks" href="/pubmed/20810315" ref="ordinalpos=4282&amp;ncbi_uid=3591092&amp;link_uid=20810315">Citation</a><a class="dblinks" href="/pmc/articles/PMC2981611/" ref="ordinalpos=4282&amp;ncbi_uid=3591092&amp;link_uid=PMC2981611">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4283.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4117313/figure/F6/" ref="ordinalpos=4283&amp;ncbi_uid=6431492&amp;link_uid=PMC4117313" href="/pmc/articles/PMC4117313/"><img src="/pmc/articles/instance/4117313/bin/nihms606096f6.gif" src-large="/pmc/articles/instance/4117313/bin/nihms606096f6.jpg" alt="FIGURE 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4117313/" ref="ordinalpos=4283&amp;ncbi_uid=6431492&amp;link_uid=PMC4117313" image-link="/pmc/articles/PMC4117313/figure/F6/" class="imagepopup">FIGURE 6.  From: The Tyrosine Kinase Pyk2 Mediates Lipopolysaccharide-Induced IL-8 Expression in Human Endothelial Cells. </a></p><div class="supp"><p class="details"><div>Proposed scheme of Pyk2 regulation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to IL-8 expression in endothelial cells upon LPS stimulation.</div></p></div><div class="aux"><div class="resc">Appakkudal R. Anand, et al. J Immunol. ;180(8):5636-5644.</div><p class="links"><a class="dblinks" href="/pubmed/18390748" ref="ordinalpos=4283&amp;ncbi_uid=6431492&amp;link_uid=18390748">Citation</a><a class="dblinks" href="/pmc/articles/PMC4117313/" ref="ordinalpos=4283&amp;ncbi_uid=6431492&amp;link_uid=PMC4117313">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4284.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3521658/figure/ppat-1003062-g008/" ref="ordinalpos=4284&amp;ncbi_uid=4466421&amp;link_uid=PMC3521658" href="/pmc/articles/PMC3521658/"><img src="/pmc/articles/instance/3521658/bin/ppat.1003062.g008.gif" src-large="/pmc/articles/instance/3521658/bin/ppat.1003062.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3521658/" ref="ordinalpos=4284&amp;ncbi_uid=4466421&amp;link_uid=PMC3521658" image-link="/pmc/articles/PMC3521658/figure/ppat-1003062-g008/" class="imagepopup">Figure 8. Diagram of the proposed mechanism of action..  From: Block of Death-Receptor Apoptosis Protects Mouse Cytomegalovirus from Macrophages and Is a Determinant of Virulence in Immunodeficient Hosts. </a></p><div class="supp"><p class="details"><div>Activated macrophages secrete TNFα (and possibly additional cytokines) which synergize with IFNγ in fibroblasts to block virus growth by a mechanism that is dependent on caspase <span class="highlight" style="background-color:">signaling</span>. M36 blocks the caspase-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and thus prevents apoptosis and rescues the virus growth.</div></p></div><div class="aux"><div class="resc">Linda Ebermann, et al. PLoS Pathog. 2012 December;8(12):e1003062.</div><p class="links"><a class="dblinks" href="/pubmed/23271968" ref="ordinalpos=4284&amp;ncbi_uid=4466421&amp;link_uid=23271968">Citation</a><a class="dblinks" href="/pmc/articles/PMC3521658/" ref="ordinalpos=4284&amp;ncbi_uid=4466421&amp;link_uid=PMC3521658">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4285.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4020729/figure/f1/" ref="ordinalpos=4285&amp;ncbi_uid=6145227&amp;link_uid=PMC4020729" href="/pmc/articles/PMC4020729/"><img src="/pmc/articles/instance/4020729/bin/jdi-3-130-g1.gif" src-large="/pmc/articles/instance/4020729/bin/jdi-3-130-g1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4020729/" ref="ordinalpos=4285&amp;ncbi_uid=6145227&amp;link_uid=PMC4020729" image-link="/pmc/articles/PMC4020729/figure/f1/" class="imagepopup">Figure 1.  From: GLUT12: a second insulin-responsive glucose transporters as an emerging target for type 2 diabetes. </a></p><div class="supp"><p class="details"><div> Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating GLUT12 translocation and glucose transport. The insulin receptor phosphorylates IRS1, which activates PI3K and downstream putative effectors that modulate steps in the GLUT12 trafficking pathways. Activation of PI3K <span class="highlight" style="background-color:">signaling</span> is required for insulin‐stimulated GLUT12 translocation to the plasma membrane.</div></p></div><div class="aux"><div class="resc">Takashi Matsuzaka, et al. J Diabetes Investig. 2012 March 28;3(2):130-131.</div><p class="links"><a class="dblinks" href="/pubmed/24843555" ref="ordinalpos=4285&amp;ncbi_uid=6145227&amp;link_uid=24843555">Citation</a><a class="dblinks" href="/pmc/articles/PMC4020729/" ref="ordinalpos=4285&amp;ncbi_uid=6145227&amp;link_uid=PMC4020729">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4286.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2752049/figure/F1/" ref="ordinalpos=4286&amp;ncbi_uid=2103361&amp;link_uid=PMC2752049" href="/pmc/articles/PMC2752049/"><img src="/pmc/articles/instance/2752049/bin/nihms-96112-f0001.gif" src-large="/pmc/articles/instance/2752049/bin/nihms-96112-f0001.jpg" alt="Fig. 2.1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2752049/" ref="ordinalpos=4286&amp;ncbi_uid=2103361&amp;link_uid=PMC2752049" image-link="/pmc/articles/PMC2752049/figure/F1/" class="imagepopup">Fig. 2.1.  From: MULTISCALE MODELS OF TAXIS-DRIVEN PATTERNING IN BACTERIAL POPULATIONS. </a></p><div class="supp"><p class="details"><div>The signal transduction <span class="highlight" style="background-color:">pathway</span> for E. coli chemotaxis. Chemoreceptors (MCPs) span the cytoplasmic membrane (hatched lines), with a ligand-binding domain on the periplasmic side and a <span class="highlight" style="background-color:">signaling</span> domain on the cytoplasmic side. The cytoplasmic <span class="highlight" style="background-color:">signaling</span> proteins are represented by single letters, e.g., A = CheA. (From [34] with permission.)</div></p></div><div class="aux"><div class="resc">CHUAN XUE, et al. SIAM J Appl Math. ;70(1):133-169.</div><p class="links"><a class="dblinks" href="/pubmed/19784399" ref="ordinalpos=4286&amp;ncbi_uid=2103361&amp;link_uid=19784399">Citation</a><a class="dblinks" href="/pmc/articles/PMC2752049/" ref="ordinalpos=4286&amp;ncbi_uid=2103361&amp;link_uid=PMC2752049">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4287.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3755508/figure/F3/" ref="ordinalpos=4287&amp;ncbi_uid=5234710&amp;link_uid=PMC3755508" href="/pmc/articles/PMC3755508/"><img src="/pmc/articles/instance/3755508/bin/nihms-504354-f0003.gif" src-large="/pmc/articles/instance/3755508/bin/nihms-504354-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3755508/" ref="ordinalpos=4287&amp;ncbi_uid=5234710&amp;link_uid=PMC3755508" image-link="/pmc/articles/PMC3755508/figure/F3/" class="imagepopup">Figure 3.  From: Alteration of peripheral blood monocyte gene expression in humans following diesel exhaust inhalation. </a></p><div class="supp"><p class="details"><div>Canonical pathways from the IPA program with the greatest ratio of gene expression change to total <span class="highlight" style="background-color:">pathway</span> gene expression. The canonical pathways for <span class="highlight" style="background-color:">signaling</span> and metabolic activity were created using differential expression data. This figure illustrates pathways affected by differentially regulated transcripts and significance of <span class="highlight" style="background-color:">pathway</span> assignment is a function of the total number of eligible molecules in a given reference <span class="highlight" style="background-color:">pathway</span>. IPA uses the Fisher's exact test to calculate statistical significance of <span class="highlight" style="background-color:">pathway</span> involvement.</div></p></div><div class="aux"><div class="resc">Ashley P. Pettit, et al. Inhal Toxicol. ;24(3):172-181.</div><p class="links"><a class="dblinks" href="/pubmed/22369193" ref="ordinalpos=4287&amp;ncbi_uid=5234710&amp;link_uid=22369193">Citation</a><a class="dblinks" href="/pmc/articles/PMC3755508/" ref="ordinalpos=4287&amp;ncbi_uid=5234710&amp;link_uid=PMC3755508">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4288.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4309523/figure/fig05/" ref="ordinalpos=4288&amp;ncbi_uid=7159045&amp;link_uid=PMC4309523" href="/pmc/articles/PMC4309523/"><img src="/pmc/articles/instance/4309523/bin/pmic0015-0608-f5.gif" src-large="/pmc/articles/instance/4309523/bin/pmic0015-0608-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4309523/" ref="ordinalpos=4288&amp;ncbi_uid=7159045&amp;link_uid=PMC4309523" image-link="/pmc/articles/PMC4309523/figure/fig05/" class="imagepopup">Figure 5.  From: Comparative network analysis via differential graphlet communities. </a></p><div class="supp"><p class="details"><div>An example from <i>dGCsp</i><sub><i>ALL</i></sub>2. Edges link coexpressed genes. Nodes are colored based on GO biological function. <i>IL7R</i> belongs to the Jak-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the hematopoietic cell lineage. <i>LCK</i> belongs to the canonical <i>NF-kappaB</i> <span class="highlight" style="background-color:">pathway</span> and the natural killer cell mediated cytotoxicity.</div></p></div><div class="aux"><div class="resc">Serene W H Wong, et al. Proteomics. 2015 January;15(2-3):608-617.</div><p class="links"><a class="dblinks" href="/pubmed/25283527" ref="ordinalpos=4288&amp;ncbi_uid=7159045&amp;link_uid=25283527">Citation</a><a class="dblinks" href="/pmc/articles/PMC4309523/" ref="ordinalpos=4288&amp;ncbi_uid=7159045&amp;link_uid=PMC4309523">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4289.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4238427/figure/F4/" ref="ordinalpos=4289&amp;ncbi_uid=6859362&amp;link_uid=PMC4238427" href="/pmc/articles/PMC4238427/"><img src="/pmc/articles/instance/4238427/bin/nihms596094f4.gif" src-large="/pmc/articles/instance/4238427/bin/nihms596094f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4238427/" ref="ordinalpos=4289&amp;ncbi_uid=6859362&amp;link_uid=PMC4238427" image-link="/pmc/articles/PMC4238427/figure/F4/" class="imagepopup">Fig. 4.  From: Placental ischemia induces changes in gene expression in chorionic tissue. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis of top differentially affected pathways from the microarray data. <span class="highlight" style="background-color:">Pathway</span> analysis was performed with a statistical cutoff of <i>p</i> &lt;0.02. The <i>top</i> two differentially affected pathways were <b>a</b> involved in NF-κB regulation, and <b>b</b> production of inflammatory cytokines associated with notch <span class="highlight" style="background-color:">signaling</span></div></p></div><div class="aux"><div class="resc">Eric M. George, et al. Mamm Genome. ;25(0):253-261.</div><p class="links"><a class="dblinks" href="/pubmed/24668059" ref="ordinalpos=4289&amp;ncbi_uid=6859362&amp;link_uid=24668059">Citation</a><a class="dblinks" href="/pmc/articles/PMC4238427/" ref="ordinalpos=4289&amp;ncbi_uid=6859362&amp;link_uid=PMC4238427">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4290.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4148386/figure/f3-mmr-10-04-1800/" ref="ordinalpos=4290&amp;ncbi_uid=6545450&amp;link_uid=PMC4148386" href="/pmc/articles/PMC4148386/"><img src="/pmc/articles/instance/4148386/bin/MMR-10-04-1800-g02.gif" src-large="/pmc/articles/instance/4148386/bin/MMR-10-04-1800-g02.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4148386/" ref="ordinalpos=4290&amp;ncbi_uid=6545450&amp;link_uid=PMC4148386" image-link="/pmc/articles/PMC4148386/figure/f3-mmr-10-04-1800/" class="imagepopup">Figure 3.  From: Functional analysis of the mRNA profile of neutrophil gelatinase-associated lipocalin overexpression in esophageal squamous cell carcinoma using multiple bioinformatic tools. </a></p><div class="supp"><p class="details"><div>Differentially expressed gene-enriched subpathway in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (path:04010). The six differentially expressed genes are indicated by a black star in the entire origin <span class="highlight" style="background-color:">pathway</span>. The subpathway structures of path:04010_2, path:04010_5 and path:04010_8 generated by Subpathway package are also shown. MAPK, mitogen-activated protein kinase.</div></p></div><div class="aux"><div class="resc">BING-LI WU, et al. Mol Med Rep. 2014 October;10(4):1800-1812.</div><p class="links"><a class="dblinks" href="/pubmed/25109818" ref="ordinalpos=4290&amp;ncbi_uid=6545450&amp;link_uid=25109818">Citation</a><a class="dblinks" href="/pmc/articles/PMC4148386/" ref="ordinalpos=4290&amp;ncbi_uid=6545450&amp;link_uid=PMC4148386">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4291.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713319/figure/fig3/" ref="ordinalpos=4291&amp;ncbi_uid=1936997&amp;link_uid=PMC2713319" href="/pmc/articles/PMC2713319/"><img src="/pmc/articles/instance/2713319/bin/PROCATS23214f3.gif" src-large="/pmc/articles/instance/2713319/bin/PROCATS23214f3.jpg" alt="&lt;b&gt;Figure 3.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713319/" ref="ordinalpos=4291&amp;ncbi_uid=1936997&amp;link_uid=PMC2713319" image-link="/pmc/articles/PMC2713319/figure/fig3/" class="imagepopup">Figure 3..  From: Apoptosis and Epithelial Injury in the Lungs. </a></p><div class="supp"><p class="details"><div>Interactions between the Fas-dependent apoptosis <span class="highlight" style="background-color:">pathway</span> and the TLR4 <span class="highlight" style="background-color:">pathway</span> that mediates cellular activation in response to bacterial lipopolysaccharide. FADD, Fas-associated death domain protein. The FADD protein shuttles back and forth between the Fas intracellular tail, where it promotes apoptosis, and the TLR4 adaptor protein, MyD88, where it antagonizes TLR4-dependent <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Thomas R. Martin, et al. Proc Am Thorac Soc. 2005 October;2(3):214-220.</div><p class="links"><a class="dblinks" href="/pubmed/16222040" ref="ordinalpos=4291&amp;ncbi_uid=1936997&amp;link_uid=16222040">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713319/" ref="ordinalpos=4291&amp;ncbi_uid=1936997&amp;link_uid=PMC2713319">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4292.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4396520/figure/f6-copd-10-715/" ref="ordinalpos=4292&amp;ncbi_uid=7433398&amp;link_uid=PMC4396520" href="/pmc/articles/PMC4396520/"><img src="/pmc/articles/instance/4396520/bin/copd-10-715Fig6.gif" src-large="/pmc/articles/instance/4396520/bin/copd-10-715Fig6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4396520/" ref="ordinalpos=4292&amp;ncbi_uid=7433398&amp;link_uid=PMC4396520" image-link="/pmc/articles/PMC4396520/figure/f6-copd-10-715/" class="imagepopup">Figure 6.  From: Rosiglitazone attenuates the metalloprotease/anti-metalloprotease imbalance in emphysema induced by cigarette smoke: involvement of extracellular signal-regulated kinase and NFκB <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Effects of rosiglitazone on the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of MAPK.<br /><b>Notes:</b> The expression of P38, phospho-P38, ERK1/2, phospho-ERK1/2, JNK, and phospho-JNK was measured by Western blotting (<b>A</b>), and the target protein bands were densitometrically analyzed normalized to β-actin (<b>B</b>). *<i>P</i>&lt;0.05, compared with the control group; <sup>▲</sup><i>P</i>&lt;0.05, compared with the CS group.<br /><b>Abbreviations:</b> MAPK, mitogen-activated protein kinase; ERK1/2, extracellular signal-regulated kinase-1 and -2; JNK, c-Jun N-terminal kinase; CS, cigarette smoke; RCS, rosiglitazone-cigarette smoke.</div></p></div><div class="aux"><div class="resc">Gang Hou, et al. Int J Chron Obstruct Pulmon Dis. 2015;10:715-724.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4292&amp;ncbi_uid=7433398&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4396520/" ref="ordinalpos=4292&amp;ncbi_uid=7433398&amp;link_uid=PMC4396520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4293.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4072718/figure/pone-0101102-g006/" ref="ordinalpos=4293&amp;ncbi_uid=6316793&amp;link_uid=PMC4072718" href="/pmc/articles/PMC4072718/"><img src="/pmc/articles/instance/4072718/bin/pone.0101102.g006.gif" src-large="/pmc/articles/instance/4072718/bin/pone.0101102.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4072718/" ref="ordinalpos=4293&amp;ncbi_uid=6316793&amp;link_uid=PMC4072718" image-link="/pmc/articles/PMC4072718/figure/pone-0101102-g006/" class="imagepopup">Figure 6. Canonical DAF-2/DAF-16 Stress <span class="highlight" style="background-color:">Pathway</span> Also Confers Anoxic Resistance..  From: Oxygen Sensing Neurons and Neuropeptides Regulate Survival after Anoxia in Developing C. elegans. </a></p><div class="supp"><p class="details"><div>Survival after anoxia in worms with null alleles in the DAF-2/DAF-16 insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>daf-2(e1370)</i> demonstrates significant survival advantage after anoxia compared to N2 (64.6% vs 38.9%, <i>p</i> = 0.0034), which is eliminated in <i>daf-16(mgDf50)</i> (<i>p</i> = 0.73) and <i>daf-2 (e1370); daf-16(mgDf50)</i> doubles (<i>p</i> = 0.64). Error bars represent standard deviation. All data were analyzed using a two-tailed paired t-test. The numbers of animals evaluated and trials conducted are listed in <b>Table 1</b>.</div></p></div><div class="aux"><div class="resc">John J. Flibotte, et al. PLoS One. 2014;9(6):e101102.</div><p class="links"><a class="dblinks" href="/pubmed/24967811" ref="ordinalpos=4293&amp;ncbi_uid=6316793&amp;link_uid=24967811">Citation</a><a class="dblinks" href="/pmc/articles/PMC4072718/" ref="ordinalpos=4293&amp;ncbi_uid=6316793&amp;link_uid=PMC4072718">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4294.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2997002/figure/F5/" ref="ordinalpos=4294&amp;ncbi_uid=2620879&amp;link_uid=PMC2997002" href="/pmc/articles/PMC2997002/"><img src="/pmc/articles/instance/2997002/bin/1471-2164-11-506-5.gif" src-large="/pmc/articles/instance/2997002/bin/1471-2164-11-506-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2997002/" ref="ordinalpos=4294&amp;ncbi_uid=2620879&amp;link_uid=PMC2997002" image-link="/pmc/articles/PMC2997002/figure/F5/" class="imagepopup">Figure 5.  From: Transcriptome and expression profiling analysis revealed changes of multiple <span class="highlight" style="background-color:">signaling</span> pathways involved in immunity in the large yellow croaker during Aeromonas hydrophila infection. </a></p><div class="supp"><p class="details"><div><b>The expression analysis of selected genes from the expression profile by relative quantitative real-time PCR</b>. Total RNA was extracted from spleens of fish infected with <i>A. hydrophila </i>or injected with 0.9% NaCl. Real-time PCR was used to validate gene expression changes in the TLR <span class="highlight" style="background-color:">pathway</span> <b>(A)</b>, cytokines <b>(B)</b>, the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(C)</b>, and the TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(D)</b>. Increases and decreases in relative levels of transcripts with respect to the control β-actin gene are shown. For each gene, the black bar indicates the gene expression ratio of fish injected with 0.9% NaCl and is defined as 1; the grey bar indicates the expression ratio of fish infected with <i>A. hydrophila</i>, with associated standard error bars. Statistical significance of the relative expression ratio is indicated (*, P &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Yinnan Mu, et al. BMC Genomics. 2010;11:506-506.</div><p class="links"><a class="dblinks" href="/pubmed/20858287" ref="ordinalpos=4294&amp;ncbi_uid=2620879&amp;link_uid=20858287">Citation</a><a class="dblinks" href="/pmc/articles/PMC2997002/" ref="ordinalpos=4294&amp;ncbi_uid=2620879&amp;link_uid=PMC2997002">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4295.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4338202/figure/pone.0118587.g005/" ref="ordinalpos=4295&amp;ncbi_uid=7233357&amp;link_uid=PMC4338202" href="/pmc/articles/PMC4338202/"><img src="/pmc/articles/instance/4338202/bin/pone.0118587.g005.gif" src-large="/pmc/articles/instance/4338202/bin/pone.0118587.g005.jpg" alt="Fig 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4338202/" ref="ordinalpos=4295&amp;ncbi_uid=7233357&amp;link_uid=PMC4338202" image-link="/pmc/articles/PMC4338202/figure/pone.0118587.g005/" class="imagepopup">Fig 5. TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Whole Transcriptome Analysis Using Next-Generation Sequencing of Sterile-Cultured Eisenia andrei for Immune System Research. </a></p><div class="supp"><p class="details"><div>The red boxes indicate BLAST hits against the NR protein database.</div></p></div><div class="aux"><div class="resc">Yoshikazu Mikami, et al. PLoS One. 2015;10(2):e0118587.</div><p class="links"><a class="dblinks" href="/pubmed/25706644" ref="ordinalpos=4295&amp;ncbi_uid=7233357&amp;link_uid=25706644">Citation</a><a class="dblinks" href="/pmc/articles/PMC4338202/" ref="ordinalpos=4295&amp;ncbi_uid=7233357&amp;link_uid=PMC4338202">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4296.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3983859/figure/F6/" ref="ordinalpos=4296&amp;ncbi_uid=6017710&amp;link_uid=PMC3983859" href="/pmc/articles/PMC3983859/"><img src="/pmc/articles/instance/3983859/bin/1742-2094-11-72-6.gif" src-large="/pmc/articles/instance/3983859/bin/1742-2094-11-72-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3983859/" ref="ordinalpos=4296&amp;ncbi_uid=6017710&amp;link_uid=PMC3983859" image-link="/pmc/articles/PMC3983859/figure/F6/" class="imagepopup">Figure 6.  From: Resolvin D1 promotes the interleukin-4-induced alternative activation in BV-2 microglial cells. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the RvD1 contribution to microglia alternative activation.</b> RvD1 promotes IL-4-induced microglia alternative activation by increasing Arg1 and Ym1 expression via STAT6/PPARγ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Longyan Li, et al. J Neuroinflammation. 2014;11:72-72.</div><p class="links"><a class="dblinks" href="/pubmed/24708771" ref="ordinalpos=4296&amp;ncbi_uid=6017710&amp;link_uid=24708771">Citation</a><a class="dblinks" href="/pmc/articles/PMC3983859/" ref="ordinalpos=4296&amp;ncbi_uid=6017710&amp;link_uid=PMC3983859">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4297.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507315/figure/fig2/" ref="ordinalpos=4297&amp;ncbi_uid=4461475&amp;link_uid=PMC3507315" href="/pmc/articles/PMC3507315/"><img src="/pmc/articles/instance/3507315/bin/MI2012-563709.002.gif" src-large="/pmc/articles/instance/3507315/bin/MI2012-563709.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507315/" ref="ordinalpos=4297&amp;ncbi_uid=4461475&amp;link_uid=PMC3507315" image-link="/pmc/articles/PMC3507315/figure/fig2/" class="imagepopup">Figure 2.  From: Clinical Application of Development of Nonantibiotic Macrolides That Correct Inflammation-Driven Immune Dysfunction in Inflammatory Skin Diseases. </a></p><div class="supp"><p class="details"><div>T cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> extracted from BioCarta database [9].</div></p></div><div class="aux"><div class="resc">Carmen Rodriguez-Cerdeira, et al. Mediators Inflamm. 2012;2012:563709.</div><p class="links"><a class="dblinks" href="/pubmed/23258954" ref="ordinalpos=4297&amp;ncbi_uid=4461475&amp;link_uid=23258954">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507315/" ref="ordinalpos=4297&amp;ncbi_uid=4461475&amp;link_uid=PMC3507315">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4298.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3443607/figure/fig4/" ref="ordinalpos=4298&amp;ncbi_uid=4242233&amp;link_uid=PMC3443607" href="/pmc/articles/PMC3443607/"><img src="/pmc/articles/instance/3443607/bin/IJI2012-678190.004.gif" src-large="/pmc/articles/instance/3443607/bin/IJI2012-678190.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3443607/" ref="ordinalpos=4298&amp;ncbi_uid=4242233&amp;link_uid=PMC3443607" image-link="/pmc/articles/PMC3443607/figure/fig4/" class="imagepopup">Figure 4.  From: Reactive Oxygen Species, SUMOylation, and Endothelial Inflammation. </a></p><div class="supp"><p class="details"><div>A <span class="highlight" style="background-color:">signaling</span> scheme describing the relationship between the laminar flow-mediated ERK5/MEF2/KLF2/eNOS <span class="highlight" style="background-color:">pathway</span> and H<sub>2</sub>O<sub>2</sub> or AGE-mediated ERK5-SUMOylation (License number: 2860201127159, date: Mar 01, 2012). </div></p></div><div class="aux"><div class="resc">Nhat-Tu Le, et al. Int J Inflam. 2012;2012:678190.</div><p class="links"><a class="dblinks" href="/pubmed/22991685" ref="ordinalpos=4298&amp;ncbi_uid=4242233&amp;link_uid=22991685">Citation</a><a class="dblinks" href="/pmc/articles/PMC3443607/" ref="ordinalpos=4298&amp;ncbi_uid=4242233&amp;link_uid=PMC3443607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4299.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3288169/figure/F4/" ref="ordinalpos=4299&amp;ncbi_uid=4756776&amp;link_uid=PMC3288169" href="/pmc/articles/PMC3288169/"><img src="/pmc/articles/instance/3288169/bin/nihms-338000-f0004.gif" src-large="/pmc/articles/instance/3288169/bin/nihms-338000-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3288169/" ref="ordinalpos=4299&amp;ncbi_uid=4756776&amp;link_uid=PMC3288169" image-link="/pmc/articles/PMC3288169/figure/F4/" class="imagepopup">Figure 4.  From: LBH-589 (Panobinostat) potentiates fludarabine anti-leukemic activity through a JNK- and XIAP-dependent mechanism. </a></p><div class="supp"><p class="details"><div>Overexpression of XIAP prevents JNK activation and cell death mediated by the LBH-589/fludarabine regimen, whereas interruption of the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> fails to reverse XIAP down-regulation.</div></p></div><div class="aux"><div class="resc">Roberto Rosato, et al. Leuk Res. ;36(4):491-498.</div><p class="links"><a class="dblinks" href="/pubmed/22074700" ref="ordinalpos=4299&amp;ncbi_uid=4756776&amp;link_uid=22074700">Citation</a><a class="dblinks" href="/pmc/articles/PMC3288169/" ref="ordinalpos=4299&amp;ncbi_uid=4756776&amp;link_uid=PMC3288169">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4300.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4321714/figure/Fig5/" ref="ordinalpos=4300&amp;ncbi_uid=7158350&amp;link_uid=PMC4321714" href="/pmc/articles/PMC4321714/"><img src="/pmc/articles/instance/4321714/bin/13046_2014_108_Fig5_HTML.gif" src-large="/pmc/articles/instance/4321714/bin/13046_2014_108_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4321714/" ref="ordinalpos=4300&amp;ncbi_uid=7158350&amp;link_uid=PMC4321714" image-link="/pmc/articles/PMC4321714/figure/Fig5/" class="imagepopup">Figure 5.  From: Histone acetyltransferase inhibitor II induces apoptosis in glioma cell lines via the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>Bioinformatic analysis of the differentially expressed genes in HATi II-</b> <b>treated U251 cells.</b> Gene ontology enrichment analysis provides a controlled vocabulary to describe the attributes of differentially expressed transcripts in all organisms (<i>p</i>-values ≤ 0.05 are recommended). <b>(A)</b> Upregulated genes and <b>(B)</b> downregulated genes in HATi II-treated U251 cells. <span class="highlight" style="background-color:">Pathway</span> analysis is a functional analysis that maps genes to KEGG pathways. The top five upregulated pathways <b>(C)</b> and top five downregulated pathways <b>(D)</b> in HATi II-treated U251 cells are listed. The “p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>”, which is associated with apoptosis, was activated in HATi II-treated U251 cells. <b>(E)</b> Confirmation of the microarray results. The expression of p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>-related genes in HATi II-treated U251, HS683, U87 and SHG44 cells was validated by quantitative real-time PCR. The columns represent the log-transformed median fold changes in expression relative to DMSO-treated cells.</div></p></div><div class="aux"><div class="resc">Li-Xiao Xu, et al. J Exp Clin Cancer Res. 2014;33(1):108.</div><p class="links"><a class="dblinks" href="/pubmed/25523932" ref="ordinalpos=4300&amp;ncbi_uid=7158350&amp;link_uid=25523932">Citation</a><a class="dblinks" href="/pmc/articles/PMC4321714/" ref="ordinalpos=4300&amp;ncbi_uid=7158350&amp;link_uid=PMC4321714">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="42" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="43" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="44" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:46:32-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A4B6254312E810000000000669C4C;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A4B6254312E810000000000669C4C" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>