
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="46" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB200E54312FF100000000009EB1E9" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="45" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="46" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="47" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="46" /></div>
            <div><h2 class="result_count">Results: 4501 to 4600 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4501.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3274818/figure/F1/" ref="ordinalpos=4501&amp;ncbi_uid=3793869&amp;link_uid=PMC3274818" href="/pmc/articles/PMC3274818/"><img src="/pmc/articles/instance/3274818/bin/nihms348296f1.gif" src-large="/pmc/articles/instance/3274818/bin/nihms348296f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3274818/" ref="ordinalpos=4501&amp;ncbi_uid=3793869&amp;link_uid=PMC3274818" image-link="/pmc/articles/PMC3274818/figure/F1/" class="imagepopup">Fig. 1.  From: Evolving Therapies and FAK Inhibitors for the Treatment of Cancer. </a></p><div class="supp"><p class="details"><div>Activation of an RTK induces dimerization and downstream phosphorylation of non-receptor tyrosine kinases. Major <span class="highlight" style="background-color:">signaling</span> cascades include the RAS/RAF/MAPK <span class="highlight" style="background-color:">pathway</span>, MEK <span class="highlight" style="background-color:">pathway</span>, and the PI3kinase/AKT <span class="highlight" style="background-color:">pathway</span>.<br />(Adapted by permission from Macmillan Publishers Ltd: <i>Nature Reviews Cancer</i>, Zhu &amp; Parada, <i>Nature Reviews Cancer</i>, 2002, 2: 616–626, copyright 2002.)</div></p></div><div class="aux"><div class="resc">Kelli Bullard Dunn, et al. Anticancer Agents Med Chem. ;10(10):722-734.</div><p class="links"><a class="dblinks" href="/pubmed/21291406" ref="ordinalpos=4501&amp;ncbi_uid=3793869&amp;link_uid=21291406">Citation</a><a class="dblinks" href="/pmc/articles/PMC3274818/" ref="ordinalpos=4501&amp;ncbi_uid=3793869&amp;link_uid=PMC3274818">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4502.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2901320/figure/F5/" ref="ordinalpos=4502&amp;ncbi_uid=2332500&amp;link_uid=PMC2901320" href="/pmc/articles/PMC2901320/"><img src="/pmc/articles/instance/2901320/bin/1471-2164-11-388-5.gif" src-large="/pmc/articles/instance/2901320/bin/1471-2164-11-388-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2901320/" ref="ordinalpos=4502&amp;ncbi_uid=2332500&amp;link_uid=PMC2901320" image-link="/pmc/articles/PMC2901320/figure/F5/" class="imagepopup">Figure 5.  From: Expression of a retinoic acid signature in circulating CD34 cells from coronary artery disease patients. </a></p><div class="supp"><p class="details"><div><b>The CXCR4 <span class="highlight" style="background-color:">pathway</span> is affected in CD34+ cells from CAD patients</b>. Genes with a lower expression in CD34+ cells from CAD patients represent the Chemotaxis CXCR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (P = 2.1 × 10<sup>-3</sup>). Genes expressed at significantly lower levels are indicated by a red arrow and are encircled for clarity.</div></p></div><div class="aux"><div class="resc">Tineke CTM van der Pouw Kraan, et al. BMC Genomics. 2010;11:388-388.</div><p class="links"><a class="dblinks" href="/pubmed/20565948" ref="ordinalpos=4502&amp;ncbi_uid=2332500&amp;link_uid=20565948">Citation</a><a class="dblinks" href="/pmc/articles/PMC2901320/" ref="ordinalpos=4502&amp;ncbi_uid=2332500&amp;link_uid=PMC2901320">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4503.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2963056/figure/fig2/" ref="ordinalpos=4503&amp;ncbi_uid=3325227&amp;link_uid=PMC2963056" href="/pmc/articles/PMC2963056/"><img src="/pmc/articles/instance/2963056/bin/schbulsbp033f02_ht.gif" src-large="/pmc/articles/instance/2963056/bin/schbulsbp033f02_ht.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2963056/" ref="ordinalpos=4503&amp;ncbi_uid=3325227&amp;link_uid=PMC2963056" image-link="/pmc/articles/PMC2963056/figure/fig2/" class="imagepopup">Fig. 2. From: Fibroblast Growth Factors in Schizophrenia. </a></p><div class="supp"><p class="details"><div>Fibroblast Growth Factor (FGF) Receptor Complex and Its Schematic <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. After ligand binding, a complex of 2 FGF molecules, bound to a receptor and linked by heparan sulphate proteoglycan, is formed.8 This can activate the Stat 1 or the ERK1/2 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Afke F. Terwisscha van Scheltinga, et al. Schizophr Bull. 2010 November;36(6):1157-1166.</div><p class="links"><a class="dblinks" href="/pubmed/19429845" ref="ordinalpos=4503&amp;ncbi_uid=3325227&amp;link_uid=19429845">Citation</a><a class="dblinks" href="/pmc/articles/PMC2963056/" ref="ordinalpos=4503&amp;ncbi_uid=3325227&amp;link_uid=PMC2963056">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4504.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3690525/figure/fig7/" ref="ordinalpos=4504&amp;ncbi_uid=5018961&amp;link_uid=PMC3690525" href="/pmc/articles/PMC3690525/"><img src="/pmc/articles/instance/3690525/bin/gr7.gif" src-large="/pmc/articles/instance/3690525/bin/gr7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3690525/" ref="ordinalpos=4504&amp;ncbi_uid=5018961&amp;link_uid=PMC3690525" image-link="/pmc/articles/PMC3690525/figure/fig7/" class="imagepopup">Figure 7.  From: ROS Production and NF-?B Activation Triggered by RAC1 Facilitate WNT-Driven Intestinal Stem Cell Proliferation and Colorectal Cancer Initiation. </a></p><div class="supp"><p class="details"><div>Model of RAC1-Mediated Control of Intestinal Tumor Initiation<br />(A) Normal small intestinal crypt with different cell populations outlined. LGR5 ISCs have high Wnt <span class="highlight" style="background-color:">signaling</span> leading to high RAC1-ROS-NF-κB <span class="highlight" style="background-color:">pathway</span> activation.<br />(B) Upon loss of <i>Apc</i>, RAC1-ROS-NF-κB <span class="highlight" style="background-color:">pathway</span> activation is increased throughout the crypt, permitting the expansion of the LGR5 ISC zone and progenitor cell hyperproliferation. The activation of the RAC1-ROS-NF-κB <span class="highlight" style="background-color:">pathway</span> throughout the crypt is critical for intestinal adenoma formation.</div></p></div><div class="aux"><div class="resc">Kevin B. Myant, et al. Cell Stem Cell. 2013 June 6;12(6):761-773.</div><p class="links"><a class="dblinks" href="/pubmed/23665120" ref="ordinalpos=4504&amp;ncbi_uid=5018961&amp;link_uid=23665120">Citation</a><a class="dblinks" href="/pmc/articles/PMC3690525/" ref="ordinalpos=4504&amp;ncbi_uid=5018961&amp;link_uid=PMC3690525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4505.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3411810/figure/pone-0042598-g006/" ref="ordinalpos=4505&amp;ncbi_uid=4168895&amp;link_uid=PMC3411810" href="/pmc/articles/PMC3411810/"><img src="/pmc/articles/instance/3411810/bin/pone.0042598.g006.gif" src-large="/pmc/articles/instance/3411810/bin/pone.0042598.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3411810/" ref="ordinalpos=4505&amp;ncbi_uid=4168895&amp;link_uid=PMC3411810" image-link="/pmc/articles/PMC3411810/figure/pone-0042598-g006/" class="imagepopup">Figure 6. Schematic representations of the cross-talk between MAPK and AKT pathways in cells with different genetic backgrounds..  From: BRAFV600E Negatively Regulates the AKT <span class="highlight" style="background-color:">Pathway</span> in Melanoma Cell Lines. </a></p><div class="supp"><p class="details"><div>(A) In cells with “wild-type” AKT <span class="highlight" style="background-color:">signaling</span>, the presence of BRAFV600E exerts a negative impact through mTORC2 on AKT phosphorylation and <span class="highlight" style="background-color:">pathway</span> activation. Suppression of AKT activity leads to a de-repression of AKT substrates PRAS40, FOXO and GSK3β due to the reduction of AKT-mediated phosphorylation. (B) In cells where AKT <span class="highlight" style="background-color:">signaling</span> is aberrantly activated (by PTEN loss, PI3K activation, or RTK amplification), the negative impact of BRAFV600E is countered by a dominant input upstream of AKT. Consequently, activated AKT phosphorylates substrates PRAS40, FOXO and GSK3β and suppresses their activity.</div></p></div><div class="aux"><div class="resc">Brenden Chen, et al. PLoS One. 2012;7(8):e42598.</div><p class="links"><a class="dblinks" href="/pubmed/22880048" ref="ordinalpos=4505&amp;ncbi_uid=4168895&amp;link_uid=22880048">Citation</a><a class="dblinks" href="/pmc/articles/PMC3411810/" ref="ordinalpos=4505&amp;ncbi_uid=4168895&amp;link_uid=PMC3411810">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4506.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2245955/figure/F5/" ref="ordinalpos=4506&amp;ncbi_uid=1297510&amp;link_uid=PMC2245955" href="/pmc/articles/PMC2245955/"><img src="/pmc/articles/instance/2245955/bin/1471-2202-8-100-5.gif" src-large="/pmc/articles/instance/2245955/bin/1471-2202-8-100-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2245955/" ref="ordinalpos=4506&amp;ncbi_uid=1297510&amp;link_uid=PMC2245955" image-link="/pmc/articles/PMC2245955/figure/F5/" class="imagepopup">Figure 5.  From: Analysis of gene expression during neurite outgrowth and regeneration. </a></p><div class="supp"><p class="details"><div><b>Gene expression changes observed in Sema3A <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Model of Sema3A <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, colored by GenMAPP to depict under-representation of genes changed during Sema3A exposure, as compared to the number changed during neurite outgrowth <i>in vitro</i>. Criteria for changed genes are the same as for the microarray analysis. Genes on the MAPP include both validated interaction partners from the literature as well as their homologs. Gene names are listed in Additional file 2.</div></p></div><div class="aux"><div class="resc">Moriah L Szpara, et al. BMC Neurosci. 2007;8:100-100.</div><p class="links"><a class="dblinks" href="/pubmed/18036227" ref="ordinalpos=4506&amp;ncbi_uid=1297510&amp;link_uid=18036227">Citation</a><a class="dblinks" href="/pmc/articles/PMC2245955/" ref="ordinalpos=4506&amp;ncbi_uid=1297510&amp;link_uid=PMC2245955">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4507.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4058311/figure/fig02/" ref="ordinalpos=4507&amp;ncbi_uid=6253748&amp;link_uid=PMC4058311" href="/pmc/articles/PMC4058311/"><img src="/pmc/articles/instance/4058311/bin/ajtr0006-0302-f2.gif" src-large="/pmc/articles/instance/4058311/bin/ajtr0006-0302-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4058311/" ref="ordinalpos=4507&amp;ncbi_uid=6253748&amp;link_uid=PMC4058311" image-link="/pmc/articles/PMC4058311/figure/fig02/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Pathway</span>-based analysis of breast cancer. </a></p><div class="supp"><p class="details"><div>mRNA expression and protein abundance of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes altered in breast cancer. The higher the <i>p</i> values, the stronger association.</div></p></div><div class="aux"><div class="resc">Dong Song, et al. Am J Transl Res. 2014;6(3):302-311.</div><p class="links"><a class="dblinks" href="/pubmed/24936222" ref="ordinalpos=4507&amp;ncbi_uid=6253748&amp;link_uid=24936222">Citation</a><a class="dblinks" href="/pmc/articles/PMC4058311/" ref="ordinalpos=4507&amp;ncbi_uid=6253748&amp;link_uid=PMC4058311">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4508.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2822470/figure/F8/" ref="ordinalpos=4508&amp;ncbi_uid=2243460&amp;link_uid=PMC2822470" href="/pmc/articles/PMC2822470/"><img src="/pmc/articles/instance/2822470/bin/nihms173999f8.gif" src-large="/pmc/articles/instance/2822470/bin/nihms173999f8.jpg" alt="FIGURE 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2822470/" ref="ordinalpos=4508&amp;ncbi_uid=2243460&amp;link_uid=PMC2822470" image-link="/pmc/articles/PMC2822470/figure/F8/" class="imagepopup">FIGURE 8.  From: Angiotensin II Inhibits the ROMK-like Small Conductance K Channel in Renal Cortical Collecting Duct during Dietary Potassium Restriction. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by which AngII inhibits the activity of ROMK channels in the CCD from LK-fed rats.</div></p></div><div class="aux"><div class="resc">Yuan Wei, et al. J Biol Chem. ;282(9):6455.</div><p class="links"><a class="dblinks" href="/pubmed/17194699" ref="ordinalpos=4508&amp;ncbi_uid=2243460&amp;link_uid=17194699">Citation</a><a class="dblinks" href="/pmc/articles/PMC2822470/" ref="ordinalpos=4508&amp;ncbi_uid=2243460&amp;link_uid=PMC2822470">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4509.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3997497/figure/pone-0095912-g012/" ref="ordinalpos=4509&amp;ncbi_uid=6068672&amp;link_uid=PMC3997497" href="/pmc/articles/PMC3997497/"><img src="/pmc/articles/instance/3997497/bin/pone.0095912.g012.gif" src-large="/pmc/articles/instance/3997497/bin/pone.0095912.g012.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3997497/" ref="ordinalpos=4509&amp;ncbi_uid=6068672&amp;link_uid=PMC3997497" image-link="/pmc/articles/PMC3997497/figure/pone-0095912-g012/" class="imagepopup">Figure 12. Schematic illustration of Notch-1 signal <span class="highlight" style="background-color:">pathway</span> to regulate gene expression..  From: Notch-1 <span class="highlight" style="background-color:">Signaling</span> Promotes the Malignant Features of Human Breast Cancer through NF-?B Activation. </a></p><div class="supp"><p class="details"><div>This model summarizes our findings that Notch-1 activation promotes the malignant features of human breast cancer MDA-MB-231 cells via cross-talking the NF-κB signal <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Li Li, et al. PLoS One. 2014;9(4):e95912.</div><p class="links"><a class="dblinks" href="/pubmed/24760075" ref="ordinalpos=4509&amp;ncbi_uid=6068672&amp;link_uid=24760075">Citation</a><a class="dblinks" href="/pmc/articles/PMC3997497/" ref="ordinalpos=4509&amp;ncbi_uid=6068672&amp;link_uid=PMC3997497">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4510.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2425463/figure/F7/" ref="ordinalpos=4510&amp;ncbi_uid=1487956&amp;link_uid=PMC2425463" href="/pmc/articles/PMC2425463/"><img src="/pmc/articles/instance/2425463/bin/gkn234f7.gif" src-large="/pmc/articles/instance/2425463/bin/gkn234f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2425463/" ref="ordinalpos=4510&amp;ncbi_uid=1487956&amp;link_uid=PMC2425463" image-link="/pmc/articles/PMC2425463/figure/F7/" class="imagepopup">Figure 7. From: Enhanced identification and biological validation of differential gene expression via Illumina whole-genome expression arrays through the use of the model-based background correction methodology. </a></p><div class="supp"><p class="details"><div>Western blot of proteins involved in the <i>Tnfr2</i> and <i>NF-kappaB</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The figures indicate upregulation of <i>Tnfr2</i> and <i>NF-kappaB</i>, and downregulation of <i>Ikk</i>. These results were consistent with gene-expression analysis using the MBCB data set and suggest active involvement of this <span class="highlight" style="background-color:">pathway</span> in mouse AML samples.</div></p></div><div class="aux"><div class="resc">Liang-Hao Ding, et al. Nucleic Acids Res. 2008 June;36(10):e58-e58.</div><p class="links"><a class="dblinks" href="/pubmed/18450815" ref="ordinalpos=4510&amp;ncbi_uid=1487956&amp;link_uid=18450815">Citation</a><a class="dblinks" href="/pmc/articles/PMC2425463/" ref="ordinalpos=4510&amp;ncbi_uid=1487956&amp;link_uid=PMC2425463">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4511.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3398478/figure/fig1/" ref="ordinalpos=4511&amp;ncbi_uid=4764265&amp;link_uid=PMC3398478" href="/pmc/articles/PMC3398478/"><img src="/pmc/articles/instance/3398478/bin/PC_096156_wc_f1.gif" src-large="/pmc/articles/instance/3398478/bin/PC_096156_wc_f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3398478/" ref="ordinalpos=4511&amp;ncbi_uid=4764265&amp;link_uid=PMC3398478" image-link="/pmc/articles/PMC3398478/figure/fig1/" class="imagepopup">Figure 1. From: Mitogen-Activated Protein Kinase <span class="highlight" style="background-color:">Signaling</span> in Plant-Interacting Fungi: Distinct Messages from Conserved Messengers[W]. </a></p><div class="supp"><p class="details"><div>MAPK Pathways in <i>S. cerevisiae</i>.<br />Yeast cells rely on four MAPK cascades to regulate mating, invasive growth, cell integrity, and high osmolarity. Smk1 may be part of a fifth, yet undefined, <span class="highlight" style="background-color:">pathway</span> that regulates ascospore formation. Activation of MAPK cascades depends on transmembrane receptors that perceive extracellular cues and translate information to intracellular <span class="highlight" style="background-color:">signaling</span> components, including GTPases, GEFs, and PKs. Upstream regulators may be shared between distinct <span class="highlight" style="background-color:">signaling</span> pathways, and timely activation of a specific cascade must be tightly regulated. In some cases, signal specificity is achieved through the use of scaffolding proteins promoting interaction between suitable MAPK <span class="highlight" style="background-color:">signaling</span> components. Following activation, MAPKs phosphorylate an array of substrates, including TFs that induce a transcriptional shift controlling output responses. See text for more details on each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In light of this study, the acronym “Sce” (for <i>S. cerevisiae</i>) was added to the standard nomenclature of yeast MAPKs.</div></p></div><div class="aux"><div class="resc">Louis-Philippe Hamel, et al. Plant Cell. 2012 April;24(4):1327-1351.</div><p class="links"><a class="dblinks" href="/pubmed/22517321" ref="ordinalpos=4511&amp;ncbi_uid=4764265&amp;link_uid=22517321">Citation</a><a class="dblinks" href="/pmc/articles/PMC3398478/" ref="ordinalpos=4511&amp;ncbi_uid=4764265&amp;link_uid=PMC3398478">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4512.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2845642/figure/F3/" ref="ordinalpos=4512&amp;ncbi_uid=2290272&amp;link_uid=PMC2845642" href="/pmc/articles/PMC2845642/"><img src="/pmc/articles/instance/2845642/bin/nihms186801f3.gif" src-large="/pmc/articles/instance/2845642/bin/nihms186801f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2845642/" ref="ordinalpos=4512&amp;ncbi_uid=2290272&amp;link_uid=PMC2845642" image-link="/pmc/articles/PMC2845642/figure/F3/" class="imagepopup">Figure 3. Putative CD80 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Involved in IFN-γ Production.  From: Molecular Mechanisms of T cell Receptor and Costimulatory Molecule Ligation/Blockade in Autoimmune Disease Therapy. </a></p><div class="supp"><p class="details"><div>Signals delivered by the engagement of the TCR (signal 1) and costimulatory molecules (CD28; signal 2) induce different <span class="highlight" style="background-color:">signaling</span> pathways that result in the activation of multiple transcription factors. Ligation of the TCR by peptide–MHC on an APC triggers the recruitment of the TCR and <span class="highlight" style="background-color:">signaling</span> elements, <i>e.g.</i> phospholipase C-1 (PLC-1) for the Ca<sup>2+</sup> influx–nuclear factor of activated T cells (NFAT) <span class="highlight" style="background-color:">pathway</span> and PKC- for the NF- B and AP-1 <span class="highlight" style="background-color:">pathway</span>, which control nuclear transcriptional and gene activation to lipid rafts. To induce differentiation to an IFN-γ producing Th1 cell, STAT4 activation coupled with NFAT activated by the TCR induces the expression of T-bet. In turn T-bet and NFAT induce the expression of IFN-γ. Current findings suggest that inhibition of PI3K, PKC, and PLC (in red) block CD80 induced increase in IFN-γ production. This supports the hypothesis that stimulation via CD80 on CD4<sup>+</sup> T cells positively regulates effector Th1 function. Cross-linking CD80 on T cells induces an increase in the tyrosine phosphorylation of multiple intracellular proteins, and increases the level of IFN-γ and T-bet transcription. Findings indicate that cross-linking CD80 on CD4<sup>+</sup> T cells activated in Th1-promoting conditions increases the level of IFN-γ produced by increasing the rate of IFN-γ transcription via the activation of a Ca<sup>2+</sup>-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Joseph R. Podojil, et al. Immunol Rev. ;229(1):337-355.</div><p class="links"><a class="dblinks" href="/pubmed/19426232" ref="ordinalpos=4512&amp;ncbi_uid=2290272&amp;link_uid=19426232">Citation</a><a class="dblinks" href="/pmc/articles/PMC2845642/" ref="ordinalpos=4512&amp;ncbi_uid=2290272&amp;link_uid=PMC2845642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4513.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4156304/figure/pone-0106006-g006/" ref="ordinalpos=4513&amp;ncbi_uid=6601466&amp;link_uid=PMC4156304" href="/pmc/articles/PMC4156304/"><img src="/pmc/articles/instance/4156304/bin/pone.0106006.g006.gif" src-large="/pmc/articles/instance/4156304/bin/pone.0106006.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4156304/" ref="ordinalpos=4513&amp;ncbi_uid=6601466&amp;link_uid=PMC4156304" image-link="/pmc/articles/PMC4156304/figure/pone-0106006-g006/" class="imagepopup">Figure 6. Working model of HIV-produced miRNA activation of bystander cells in HIV+ persons..  From: Novel HIV-1 MiRNAs Stimulate TNF? Release in Human Macrophages via TLR8 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>This figure provides a working model describing how HIV-produced miRNAs are encapsulated in exosomes and released from HIV-infected macrophages into the circulation. Host RNA polymerases transcribe HIV genomic RNA, which is then exported into the cytoplasm and packaged into infectious virions. However, some HIV transcripts can be processed in the host RNA interference <span class="highlight" style="background-color:">pathway</span> into mature viral miRNA. HIV miRNAs are packaged into multi-vesicular bodies and released by macrophages encapsulated in exosomes. Exosomes are disseminated either locally or systemically to be taken up by bystander macrophages, trafficking GU-rich vmiR88 and vmiR99 to the endosomal TLR8. Through this non-RNAi function of miRNAs that is distinct from the well-established role of miRNAs in RNA interference, vmiR88 and vmiR99 induce TLR8-mediated inflammatory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that leads to downstream release of TNFα leading to chronic immune activation.</div></p></div><div class="aux"><div class="resc">Mark A. Bernard, et al. PLoS One. 2014;9(9):e106006.</div><p class="links"><a class="dblinks" href="/pubmed/25191859" ref="ordinalpos=4513&amp;ncbi_uid=6601466&amp;link_uid=25191859">Citation</a><a class="dblinks" href="/pmc/articles/PMC4156304/" ref="ordinalpos=4513&amp;ncbi_uid=6601466&amp;link_uid=PMC4156304">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4514.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2648785/figure/F6/" ref="ordinalpos=4514&amp;ncbi_uid=1719252&amp;link_uid=PMC2648785" href="/pmc/articles/PMC2648785/"><img src="/pmc/articles/instance/2648785/bin/1471-2105-10-S1-S53-6.gif" src-large="/pmc/articles/instance/2648785/bin/1471-2105-10-S1-S53-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2648785/" ref="ordinalpos=4514&amp;ncbi_uid=1719252&amp;link_uid=PMC2648785" image-link="/pmc/articles/PMC2648785/figure/F6/" class="imagepopup">Figure 6.  From: Comparative analysis of the JAK/STAT <span class="highlight" style="background-color:">signaling</span> through erythropoietin receptor and thrombopoietin receptor using a systems approach. </a></p><div class="supp"><p class="details"><div><b>The JAK/STAT <span class="highlight" style="background-color:">pathway</span></b>. Each node indicates the component involving the transduction <span class="highlight" style="background-color:">signaling</span> or negative regulation of the <span class="highlight" style="background-color:">pathway</span> (R: receptor, JAK: JAK2, RJ: receptor-JAK2 complex, Ligand: Epo or Tpo, LRJ: ligand bound RJ, Pc: cytoplasmic phosphatase, Pn: nuclear phosphatase, SOCS: negative regulator). * and D denote a phosphorylation and a dimerization, respectively. According to the location in a cell, each molecule is tagged by c (cytoplasm) or n (nucleus). The system output was calculated by the level of nuclear phosphorylated STAT dimers (STAT*Dn).</div></p></div><div class="aux"><div class="resc">Hong-Hee Won, et al. BMC Bioinformatics. 2009;10(Suppl 1):S53-S53.</div><p class="links"><a class="dblinks" href="/pubmed/19208156" ref="ordinalpos=4514&amp;ncbi_uid=1719252&amp;link_uid=19208156">Citation</a><a class="dblinks" href="/pmc/articles/PMC2648785/" ref="ordinalpos=4514&amp;ncbi_uid=1719252&amp;link_uid=PMC2648785">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4515.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3121168/figure/F2/" ref="ordinalpos=4515&amp;ncbi_uid=3037917&amp;link_uid=PMC3121168" href="/pmc/articles/PMC3121168/"><img src="/pmc/articles/instance/3121168/bin/nihms-302889-f0003.gif" src-large="/pmc/articles/instance/3121168/bin/nihms-302889-f0003.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3121168/" ref="ordinalpos=4515&amp;ncbi_uid=3037917&amp;link_uid=PMC3121168" image-link="/pmc/articles/PMC3121168/figure/F2/" class="imagepopup">Figure 2. Summary of neural and limb patterning phenotypes in Hh <span class="highlight" style="background-color:">pathway</span> and cilia mutants.  From: The Primary Cilium: A <span class="highlight" style="background-color:">Signaling</span> Center During Vertebrate Development. </a></p><div class="supp"><p class="details"><div><b>a</b> | In wild-type (WT) embryos, ventral neural cell fates are specified by a gradient of Shh. The number and identity of digits in the limb is established by a Shh gradient from the posterior limb bud opposed by anterior Gli3 repressor (Gli3R). <b>b</b> | In the absence of Hh, (e.g. <i>Smo</i> −/−), ventral neural cell fates are lost. The limbs of Shh mutants lack digits. <b>c</b> | If the <span class="highlight" style="background-color:">pathway</span> is hyperactive (e.g. <i>Ptch</i> −/−), ventral cell types expand in the neural tube. Activation of the <span class="highlight" style="background-color:">pathway</span> within the limb, such as in Gli3 mutants (which lack Gli3R), causes the formation of extra digits. <b>d</b> | Anterograde IFT mutants (e.g. the IFT-B mutants <i>Ift88</i> or <i>Ift172</i>) lack cilia: Hh <span class="highlight" style="background-color:">signaling</span> is reduced, and the neural tube is dorsalized. This phenotype is milder than in (<b>b</b>) because cilia are also required for cilia Gli3R processing, and cell types that require low levels of Hh <span class="highlight" style="background-color:">signaling</span> are specified. Reduced Gli3R results in polydactyly. Dynein mutants display a similar phenotype, however they retain motor neurons in the caudal neural tube. <b>e</b> | IFT-A mutants (ie <i>Ift139</i>) exhibit phenotypes consistent with excess Hh <span class="highlight" style="background-color:">signaling</span>. <b>f</b> | Mutations that disrupt the kinesin <i>Kif7</i> cause a partial activation of the Hh <span class="highlight" style="background-color:">pathway</span>, with a modest expansion of cells that require intermediate levels of Hh.</div></p></div><div class="aux"><div class="resc">Sarah C. Goetz, et al. Nat Rev Genet. ;11(5):331-344.</div><p class="links"><a class="dblinks" href="/pubmed/20395968" ref="ordinalpos=4515&amp;ncbi_uid=3037917&amp;link_uid=20395968">Citation</a><a class="dblinks" href="/pmc/articles/PMC3121168/" ref="ordinalpos=4515&amp;ncbi_uid=3037917&amp;link_uid=PMC3121168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4516.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3696624/figure/F8/" ref="ordinalpos=4516&amp;ncbi_uid=5057192&amp;link_uid=PMC3696624" href="/pmc/articles/PMC3696624/"><img src="/pmc/articles/instance/3696624/bin/lar-29-113-g008.gif" src-large="/pmc/articles/instance/3696624/bin/lar-29-113-g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3696624/" ref="ordinalpos=4516&amp;ncbi_uid=5057192&amp;link_uid=PMC3696624" image-link="/pmc/articles/PMC3696624/figure/F8/" class="imagepopup">Figure 8.  From: In vitro and in vivo study of effects of fermented soybean product (chungkookjang) on NGF secretion ability and NGF receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Effects of SH-CKJ treatment on down-stream NGF receptor p75<sup>NTR</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via Western blot analysis. Total homogenates were prepared from brain tissue of Tg2576 mice treated with SH-CKJ for 8 weeks. Thirty micrograms of protein per sample was immunoblotted with antibody for each protein (A). The intensity of each band was determined using an imaging densitometer, and the relative level of each protein was calculated based on the intensity of actin (B). Three samples were assayed in triplicate via Western blot analysis. Values of the data represent the means±SD.</div></p></div><div class="aux"><div class="resc">Young-Ju Lee, et al. Lab Anim Res. 2013 June;29(2):113-126.</div><p class="links"><a class="dblinks" href="/pubmed/23825484" ref="ordinalpos=4516&amp;ncbi_uid=5057192&amp;link_uid=23825484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3696624/" ref="ordinalpos=4516&amp;ncbi_uid=5057192&amp;link_uid=PMC3696624">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4517.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3541279/figure/pone-0053230-g002/" ref="ordinalpos=4517&amp;ncbi_uid=4535042&amp;link_uid=PMC3541279" href="/pmc/articles/PMC3541279/"><img src="/pmc/articles/instance/3541279/bin/pone.0053230.g002.gif" src-large="/pmc/articles/instance/3541279/bin/pone.0053230.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3541279/" ref="ordinalpos=4517&amp;ncbi_uid=4535042&amp;link_uid=PMC3541279" image-link="/pmc/articles/PMC3541279/figure/pone-0053230-g002/" class="imagepopup">Figure 2. Common genes in cell death and stress response networks are affected by metyrapone or carbenoxolone..  From: <span class="highlight" style="background-color:">Pathway</span> Analysis Reveals Common Pro-Survival Mechanisms of Metyrapone and Carbenoxolone after Traumatic Brain Injury. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Apoptosis <span class="highlight" style="background-color:">signaling</span> at 24 h post-TBI. (<b>B</b>) Death receptor <span class="highlight" style="background-color:">signaling</span> at 4 h post-TBI. (<b>C</b>) Corticotropin-releasing hormone (CRH) <span class="highlight" style="background-color:">signaling</span> at 4 h post-TBI. (<b>D</b>) Amyloid processing <span class="highlight" style="background-color:">pathway</span> at 4 h post-TBI. Gene expression in all groups for pathways A–C is shown up or down-regulated relative to sham control levels with no fold cut-offs. Metyrapone and carbenoxolone attenuate expression of both pro-death and pro-survival genes in these pathways such as FLIP, which is known to protect against ischemic cell death. (See Fig. S15 for symbol key).</div></p></div><div class="aux"><div class="resc">Helen L. Hellmich, et al. PLoS One. 2013;8(1):e53230.</div><p class="links"><a class="dblinks" href="/pubmed/23326402" ref="ordinalpos=4517&amp;ncbi_uid=4535042&amp;link_uid=23326402">Citation</a><a class="dblinks" href="/pmc/articles/PMC3541279/" ref="ordinalpos=4517&amp;ncbi_uid=4535042&amp;link_uid=PMC3541279">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4518.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4031445/figure/F2/" ref="ordinalpos=4518&amp;ncbi_uid=6327540&amp;link_uid=PMC4031445" href="/pmc/articles/PMC4031445/"><img src="/pmc/articles/instance/4031445/bin/nihms-499360-f0002.gif" src-large="/pmc/articles/instance/4031445/bin/nihms-499360-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4031445/" ref="ordinalpos=4518&amp;ncbi_uid=6327540&amp;link_uid=PMC4031445" image-link="/pmc/articles/PMC4031445/figure/F2/" class="imagepopup">Figure 2.  From: Prostaglandin receptor EP2 in the crosshairs of anti-inflammation, anti-cancer, and neuroprotection. </a></p><div class="supp"><p class="details"><div>Signal transduction by prostaglandin receptor EP2. In response to PGE<sub>2</sub>, EP2 receptor mediates both G protein-dependent and -independent <span class="highlight" style="background-color:">signaling</span> pathways to conduct multiple beneficial and deleterious actions. We hypothesize that the EP2 receptor mediates cellular survival and neuroplasticity mainly via cAMP/PKA/CREB <span class="highlight" style="background-color:">pathway</span>, but inflammation and neurotoxicity via cAMP/Epac/Rap <span class="highlight" style="background-color:">signaling</span>, and cell proliferation and migration via β-arrestin. <span class="highlight" style="background-color:">Signaling</span> cross-talk occurs among these three EP2 downstream pathways, but only the major pathways and effects are indicated.</div></p></div><div class="aux"><div class="resc">Jianxiong Jiang and, et al. Trends Pharmacol Sci. ;34(7):413-423.</div><p class="links"><a class="dblinks" href="/pubmed/23796953" ref="ordinalpos=4518&amp;ncbi_uid=6327540&amp;link_uid=23796953">Citation</a><a class="dblinks" href="/pmc/articles/PMC4031445/" ref="ordinalpos=4518&amp;ncbi_uid=6327540&amp;link_uid=PMC4031445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4519.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3390788/figure/F4/" ref="ordinalpos=4519&amp;ncbi_uid=4100614&amp;link_uid=PMC3390788" href="/pmc/articles/PMC3390788/"><img src="/pmc/articles/instance/3390788/bin/nihms315743f4.gif" src-large="/pmc/articles/instance/3390788/bin/nihms315743f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3390788/" ref="ordinalpos=4519&amp;ncbi_uid=4100614&amp;link_uid=PMC3390788" image-link="/pmc/articles/PMC3390788/figure/F4/" class="imagepopup">Figure 4.  From: General Anesthesia and Altered States of Arousal: A Systems Neuroscience Analysis. </a></p><div class="supp"><p class="details"><div>NMDA receptor <span class="highlight" style="background-color:">signaling</span> during the awake state (<i>a</i>) and during ketamine administration (<i>b</i>). (<i>a</i>) Normal glutamatergic regulation of GABAergic interneuron <span class="highlight" style="background-color:">signaling</span> during wakefulness and normal pain <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. (<i>b</i>) Ketamine blockage of GABAergic interneuron inhibition and ketamine-induced mechanism of analgesia. Abbreviations: DRG, dorsal root ganglia; GABA, gamma aminobutyric acid; Glu, glutamate; PAF, peripheral afferent fiber; PN, projection neuron.</div></p></div><div class="aux"><div class="resc">Emery N. Brown, et al. Annu Rev Neurosci. ;34:601-628.</div><p class="links"><a class="dblinks" href="/pubmed/21513454" ref="ordinalpos=4519&amp;ncbi_uid=4100614&amp;link_uid=21513454">Citation</a><a class="dblinks" href="/pmc/articles/PMC3390788/" ref="ordinalpos=4519&amp;ncbi_uid=4100614&amp;link_uid=PMC3390788">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4520.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766837/figure/pone-0007728-g007/" ref="ordinalpos=4520&amp;ncbi_uid=2128573&amp;link_uid=PMC2766837" href="/pmc/articles/PMC2766837/"><img src="/pmc/articles/instance/2766837/bin/pone.0007728.g007.gif" src-large="/pmc/articles/instance/2766837/bin/pone.0007728.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766837/" ref="ordinalpos=4520&amp;ncbi_uid=2128573&amp;link_uid=PMC2766837" image-link="/pmc/articles/PMC2766837/figure/pone-0007728-g007/" class="imagepopup">Figure 7. Activation of the PPARγ-LXRα-ABC transcriptional cascade by ghrelin is dependent on the PI3-K/Akt <span class="highlight" style="background-color:">pathway</span>..  From: A Concerted Kinase Interplay Identifies PPAR? as a Molecular Target of Ghrelin <span class="highlight" style="background-color:">Signaling</span> in Macrophages. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Role of the PI3-K <span class="highlight" style="background-color:">pathway</span> on gene expression analysis from THP-1 cells treated with 100 nM ghrelin and 5 nM LY294002 for 48 h. Representative images and fold changes of mRNA expression are shown for the indicated genes. (<b>B</b>) Immunoblot analysis of THP-1 cells treated with ghrelin in presence or absence of 5 nM LY294002 for 48 h. Fold changes are shown relative to untreated cells. (<b>C</b>) A proposed model for the GHS-R1a-induced <span class="highlight" style="background-color:">signaling</span> to PPARγ in macrophages. Activation of GHS-R1a by ghrelin promotes the recruitment and activation of a Fyn/Dok-1 complex with the subsequent decrease in Erk1/2-mediated phosphorylation of PPARγ Ser-84, restraining its inhibitory potential. Ghrelin also activates the PI3-K/Akt <span class="highlight" style="background-color:">pathway</span> through a Gα<sub>q</sub>–dependent mechanism, which then promotes PPARγ AF-1 phosphorylation independently of Ser-84, resulting in receptor transcriptional activation and increase in the PPARγ-LXRα-ABCA1/G1 metabolic cascade in macrophages.</div></p></div><div class="aux"><div class="resc">Annie Demers, et al. PLoS One. 2009;4(11):e7728.</div><p class="links"><a class="dblinks" href="/pubmed/19888469" ref="ordinalpos=4520&amp;ncbi_uid=2128573&amp;link_uid=19888469">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766837/" ref="ordinalpos=4520&amp;ncbi_uid=2128573&amp;link_uid=PMC2766837">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4521.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2605563/figure/pone-0004093-g007/" ref="ordinalpos=4521&amp;ncbi_uid=1599772&amp;link_uid=PMC2605563" href="/pmc/articles/PMC2605563/"><img src="/pmc/articles/instance/2605563/bin/pone.0004093.g007.gif" src-large="/pmc/articles/instance/2605563/bin/pone.0004093.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2605563/" ref="ordinalpos=4521&amp;ncbi_uid=1599772&amp;link_uid=PMC2605563" image-link="/pmc/articles/PMC2605563/figure/pone-0004093-g007/" class="imagepopup">Figure 7. Proposed model for the involvement of NF-κB <span class="highlight" style="background-color:">signaling</span> in the regulation of cytokine and chemokine in Tat±morphine treated astrocytes..  From: Morphine Exacerbates HIV-1 Tat-Induced Cytokine Production in Astrocytes through Convergent Effects on [Ca2+]i, NF-?B Trafficking and Transcription. </a></p><div class="supp"><p class="details"><div>Arrows indicate activation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; the “┤” signs indicate <span class="highlight" style="background-color:">pathway</span> inhibition. Exposure to Tat alone increases IκB phosphorylation/degradation and NF-κB transcriptional activity, and additionally increases [Ca<sup>2+</sup>]<sub>i</sub> in astrocytes [48]. Although morphine treatment alone fails to significantly activate NF-κB in astrocytes, in combination with Tat it can potentiate Tat-induced increases in [Ca<sup>2+</sup>]<sub>i</sub> at sites upstream of NF-κB [37]. How [Ca<sup>2+</sup>]<sub>i</sub> affects IκB degradation in the present study is uncertain (?), but may involve increased oxidative stress [55], and decreased proteasome activity.</div></p></div><div class="aux"><div class="resc">Nazira El-Hage, et al. PLoS ONE. 2008;3(12):e4093.</div><p class="links"><a class="dblinks" href="/pubmed/19116667" ref="ordinalpos=4521&amp;ncbi_uid=1599772&amp;link_uid=19116667">Citation</a><a class="dblinks" href="/pmc/articles/PMC2605563/" ref="ordinalpos=4521&amp;ncbi_uid=1599772&amp;link_uid=PMC2605563">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4522.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1304358/figure/fig4/" ref="ordinalpos=4522&amp;ncbi_uid=662995&amp;link_uid=PMC1304358" href="/pmc/articles/PMC1304358/"><img src="/pmc/articles/instance/1304358/bin/biophysj00029884F04_HT.gif" src-large="/pmc/articles/instance/1304358/bin/biophysj00029884F04_HT.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1304358/" ref="ordinalpos=4522&amp;ncbi_uid=662995&amp;link_uid=PMC1304358" image-link="/pmc/articles/PMC1304358/figure/fig4/" class="imagepopup">FIGURE 4.  From: The JAK-STAT <span class="highlight" style="background-color:">Signaling</span> Network in the Human B-Cell: An Extreme <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Analysis. </a></p><div class="supp"><p class="details"><div>Cross talk in the JAK-STAT <span class="highlight" style="background-color:">signaling</span> network of the human B-cell. There are 10,731 pairwise combinations of the 147 extreme pathways. Each value is the percentage of all pairwise combinations that belong to the corresponding category.</div></p></div><div class="aux"><div class="resc">Jason A. Papin, et al. Biophys J. 2004 July;87(1):37-46.</div><p class="links"><a class="dblinks" href="/pubmed/15240442" ref="ordinalpos=4522&amp;ncbi_uid=662995&amp;link_uid=15240442">Citation</a><a class="dblinks" href="/pmc/articles/PMC1304358/" ref="ordinalpos=4522&amp;ncbi_uid=662995&amp;link_uid=PMC1304358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4523.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2890223/figure/F4/" ref="ordinalpos=4523&amp;ncbi_uid=2344497&amp;link_uid=PMC2890223" href="/pmc/articles/PMC2890223/"><img src="/pmc/articles/instance/2890223/bin/nihms-207759-f0004.gif" src-large="/pmc/articles/instance/2890223/bin/nihms-207759-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2890223/" ref="ordinalpos=4523&amp;ncbi_uid=2344497&amp;link_uid=PMC2890223" image-link="/pmc/articles/PMC2890223/figure/F4/" class="imagepopup">Figure 4.  From: NCX-4016, a nitro-derivative of aspirin, inhibits EGFR and STAT3 <span class="highlight" style="background-color:">signaling</span> and modulates Bcl-2 proteins in cisplatin-resistant human ovarian cancer cells and xenografts. </a></p><div class="supp"><p class="details"><div>Effect of NCX-4016 on EGFR/PI3K and STAT3 <span class="highlight" style="background-color:">signaling</span> in CR ovarian cancer cells. Representative immunoblot images showing the expression levels of pEGFR, pAkt and pSTAT3 and their phosphorylated counterparts are shown. The cells were treated with NCX-4016 (100 μM) for 12 or 24 h. The results show inhibition of EGFR and STAT3 activation, but no effect on MAPK <span class="highlight" style="background-color:">pathway</span>, in the CR cells treated with NCX-4016.</div></p></div><div class="aux"><div class="resc">Karuppaiyah Selvendiran, et al. Cell Cycle. ;7(1):81-88.</div><p class="links"><a class="dblinks" href="/pubmed/18196976" ref="ordinalpos=4523&amp;ncbi_uid=2344497&amp;link_uid=18196976">Citation</a><a class="dblinks" href="/pmc/articles/PMC2890223/" ref="ordinalpos=4523&amp;ncbi_uid=2344497&amp;link_uid=PMC2890223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4524.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412095/figure/fig5/" ref="ordinalpos=4524&amp;ncbi_uid=4177367&amp;link_uid=PMC3412095" href="/pmc/articles/PMC3412095/"><img src="/pmc/articles/instance/3412095/bin/JST2012-473410.005.gif" src-large="/pmc/articles/instance/3412095/bin/JST2012-473410.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412095/" ref="ordinalpos=4524&amp;ncbi_uid=4177367&amp;link_uid=PMC3412095" image-link="/pmc/articles/PMC3412095/figure/fig5/" class="imagepopup">Figure 5.  From: Contractile Activity Regulates Inducible Nitric Oxide Synthase Expression and NOi Production in Cardiomyocytes via a FAK-Dependent <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Rescue of BDM inhibition of iNOS expression by CD2-FAK and Myr-AKT. (a) NRVMs were pretreated with BDM (7.5 mM) for 24 h. To “rescue” the BDM inhibition of iNOS expression, some cells were then infected (48 h, 10 moi) with Adv-expressing ne<i>β</i>gal, WT-FAK, CD2-FAK, WT-AKT, or Myr-AKT with continued exposure to BDM. Cell extracts (50 <i>μ</i>g total protein per lane) were then separated by SDS-PAGE and Western blotting with antibodies specific for FAK, AKT, iNOS, and GAPDH. ((a) and (c)) The quantitative results of 5 experiments are depicted. *<i>P</i> &lt; 0.05 versus WT-FAK or WT-AKT cells. (d) NRVMs were treated with 7.5 mM BDM for 24 h. To “rescue” the BDM inhibition of NO<sub>i</sub> production, some cells were then infected (10 moi) with WT-AKT or Myr-AKT for 48 h with continued exposure to BDM. NO<sub>i</sub> was then measured using DAF-2 DA. The quantitative results of 5 experiments are depicted. *<i>P</i> &lt; 0.05 versus BDM, <sup>#</sup><i>P</i> &lt; 0.05 versus WT-AKT. (e) A multicomponent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that links mechanical load to AKT activation and downstream <span class="highlight" style="background-color:">signaling</span> to iNOS is depicted.</div></p></div><div class="aux"><div class="resc">Miensheng Chu, et al. J Signal Transduct. 2012;2012:473410.</div><p class="links"><a class="dblinks" href="/pubmed/22900166" ref="ordinalpos=4524&amp;ncbi_uid=4177367&amp;link_uid=22900166">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412095/" ref="ordinalpos=4524&amp;ncbi_uid=4177367&amp;link_uid=PMC3412095">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4525.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4262449/figure/pone-0115029-g001/" ref="ordinalpos=4525&amp;ncbi_uid=6960221&amp;link_uid=PMC4262449" href="/pmc/articles/PMC4262449/"><img src="/pmc/articles/instance/4262449/bin/pone.0115029.g001.gif" src-large="/pmc/articles/instance/4262449/bin/pone.0115029.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4262449/" ref="ordinalpos=4525&amp;ncbi_uid=6960221&amp;link_uid=PMC4262449" image-link="/pmc/articles/PMC4262449/figure/pone-0115029-g001/" class="imagepopup">Figure 1. Brain map showing ranked gene expression of SNCA and IFN-γ-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes..  From: Differential Co-Expression between α-Synuclein and IFN-γ <span class="highlight" style="background-color:">Signaling</span> Genes across Development and in Parkinson’s Disease. </a></p><div class="supp"><p class="details"><div>Expression ranges from high (red) to low (high), with the two maps showing disjoint ranges. Ranked expression was averaged across donors and subregions. Displayed expression for the interferon <span class="highlight" style="background-color:">pathway</span> gene set is the median across genes. Abbreviations for the brain regions: frontal lobe (FL); parietal lobe (PL); temporal lobe (TL); occipital lobe (OL); basal forebrain (BF); basal ganglia (BG); amygdala (AmG); hippocampal formation (HiF); epithalamus (EP); thalamus (TH); hypothalamus (Hy); mesencephalon (MES); metencephalon (MET) and myelencephalon (MY).</div></p></div><div class="aux"><div class="resc">Noa Liscovitch, et al. PLoS One. 2014;9(12):e115029.</div><p class="links"><a class="dblinks" href="/pubmed/25493648" ref="ordinalpos=4525&amp;ncbi_uid=6960221&amp;link_uid=25493648">Citation</a><a class="dblinks" href="/pmc/articles/PMC4262449/" ref="ordinalpos=4525&amp;ncbi_uid=6960221&amp;link_uid=PMC4262449">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4526.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3656108/figure/ppat-1003385-g007/" ref="ordinalpos=4526&amp;ncbi_uid=4910687&amp;link_uid=PMC3656108" href="/pmc/articles/PMC3656108/"><img src="/pmc/articles/instance/3656108/bin/ppat.1003385.g007.gif" src-large="/pmc/articles/instance/3656108/bin/ppat.1003385.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3656108/" ref="ordinalpos=4526&amp;ncbi_uid=4910687&amp;link_uid=PMC3656108" image-link="/pmc/articles/PMC3656108/figure/ppat-1003385-g007/" class="imagepopup">Figure 7. Scheme of PMT-stimulated <span class="highlight" style="background-color:">pathway</span> to impair osteoblastogenesis..  From: Pasteurella Multocida Toxin Prevents Osteoblast Differentiation by Transactivation of the MAP-Kinase Cascade via the G?q/11 - p63RhoGEF - RhoA Axis. </a></p><div class="supp"><p class="details"><div>Model of <span class="highlight" style="background-color:">signaling</span> pathways downstream of PMT-activated Gα<sub>q/11</sub>, inhibiting differentiation of osteoblasts. PMT permanently activates members of the Gα<sub>q/11</sub> family. This activation is independent of any GPCR interaction. In osteoblasts, Gα<sub>q/11</sub> stimulates RhoA via p63RhoGEF. RhoA-Rock stimulates in turn the MAPK <span class="highlight" style="background-color:">pathway</span> in a Ras dependent manner leading to altered gene expression. GPCR – G protein-coupled receptor; Rock – Rho kinase, ERK – extracellular signal-regulated kinase; RTK – receptor tyrosine kinase.</div></p></div><div class="aux"><div class="resc">Peter Siegert, et al. PLoS Pathog. 2013 May;9(5):e1003385.</div><p class="links"><a class="dblinks" href="/pubmed/23696743" ref="ordinalpos=4526&amp;ncbi_uid=4910687&amp;link_uid=23696743">Citation</a><a class="dblinks" href="/pmc/articles/PMC3656108/" ref="ordinalpos=4526&amp;ncbi_uid=4910687&amp;link_uid=PMC3656108">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4527.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4353714/figure/pone.0119539.g007/" ref="ordinalpos=4527&amp;ncbi_uid=7292262&amp;link_uid=PMC4353714" href="/pmc/articles/PMC4353714/"><img src="/pmc/articles/instance/4353714/bin/pone.0119539.g007.gif" src-large="/pmc/articles/instance/4353714/bin/pone.0119539.g007.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4353714/" ref="ordinalpos=4527&amp;ncbi_uid=7292262&amp;link_uid=PMC4353714" image-link="/pmc/articles/PMC4353714/figure/pone.0119539.g007/" class="imagepopup">Fig 7. Model depicting α3β1-dependent alternative polyadenylation of the MMP-9 mRNA..  From: Integrin α3β1 <span class="highlight" style="background-color:">Signaling</span> through MEK/ERK Determines Alternative Polyadenylation of the MMP-9 mRNA Transcript in Immortalized Mouse Keratinocytes. </a></p><div class="supp"><p class="details"><div>(A) Integrin α3β1 activates a MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that promotes selection of the proximal polyadenylation site (polyA #1) within the MMP-9 gene, thereby generating the short, more stable mRNA transcript and subsequent synthesis of MMP-9 protein. (B) In the absence of α3β1, MEK/ERK <span class="highlight" style="background-color:">signaling</span> is diminished and polyadenylation defaults to the distal polyadenylation site (polyA #2), thereby generating the long, ARE-containing transcript that is subject to mRNA degradation. Positions of the stop codon, AREs (yellow boxes), poly(A) signals, and poly(A+) tails are indicated.</div></p></div><div class="aux"><div class="resc">Dara S. Missan, et al. PLoS One. 2015;10(3):e0119539.</div><p class="links"><a class="dblinks" href="/pubmed/25751421" ref="ordinalpos=4527&amp;ncbi_uid=7292262&amp;link_uid=25751421">Citation</a><a class="dblinks" href="/pmc/articles/PMC4353714/" ref="ordinalpos=4527&amp;ncbi_uid=7292262&amp;link_uid=PMC4353714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4528.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4390554/figure/F4/" ref="ordinalpos=4528&amp;ncbi_uid=7400651&amp;link_uid=PMC4390554" href="/pmc/articles/PMC4390554/"><img src="/pmc/articles/instance/4390554/bin/nihms672759f4.gif" src-large="/pmc/articles/instance/4390554/bin/nihms672759f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4390554/" ref="ordinalpos=4528&amp;ncbi_uid=7400651&amp;link_uid=PMC4390554" image-link="/pmc/articles/PMC4390554/figure/F4/" class="imagepopup">Fig. 4. Feedback regulation of brassinosteroid biosynthesis and <span class="highlight" style="background-color:">signaling</span> by SPCH.  From: Direct roles of SPEECHLESS in the specification of stomatal self-renewing cells. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) ChIP-Seq profiles of select brassinosteroid (BR) <span class="highlight" style="background-color:">pathway</span> genes (labeled as in Fig. 2F). (<b>B</b>) RT-qPCR analysis of BR genes in 4-dpg SPCHpro:SPCH2-4A-YFP and Col seedlings. Values are means+/−SEM. (<b>C</b> &amp; <b>D</b>) Confocal images of 3-dpg adaxial cotyledons with propidium iodide-stained cell outlines (purple). Stomatal lineage-specific expression of hyperactive BIN2 (yellow) induces lineage proliferation (bracket) (C). Stomatal lineage expression pattern of BIM2pro:YFP-YFP (yellow) (D). (<b>E</b>) Alteration of stomatal density in gain-of-function BES1pro:<i>bes1-D</i> and <i>bes1</i>-RNAi knockdown. *: p&lt;0.05, ***: p&lt;0.001 (Wilcoxon ranksum test). (<b>F</b>) Model of SPCH-BR <span class="highlight" style="background-color:">pathway</span> interactions. SPCH, a target of BR <span class="highlight" style="background-color:">signaling</span>, feeds back (positively, red arrows or negatively, red T-bars) upon transcription of multiple <span class="highlight" style="background-color:">pathway</span> members.</div></p></div><div class="aux"><div class="resc">On Sun Lau, et al. Science. ;345(6204):1605-1609.</div><p class="links"><a class="dblinks" href="/pubmed/25190717" ref="ordinalpos=4528&amp;ncbi_uid=7400651&amp;link_uid=25190717">Citation</a><a class="dblinks" href="/pmc/articles/PMC4390554/" ref="ordinalpos=4528&amp;ncbi_uid=7400651&amp;link_uid=PMC4390554">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4529.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3424042/figure/F8/" ref="ordinalpos=4529&amp;ncbi_uid=5305588&amp;link_uid=PMC3424042" href="/pmc/articles/PMC3424042/"><img src="/pmc/articles/instance/3424042/bin/DEV079525F8.gif" src-large="/pmc/articles/instance/3424042/bin/DEV079525F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3424042/" ref="ordinalpos=4529&amp;ncbi_uid=5305588&amp;link_uid=PMC3424042" image-link="/pmc/articles/PMC3424042/figure/F8/" class="imagepopup">Fig. 8. From: Formation and specification of a Drosophila dopaminergic precursor cell. </a></p><div class="supp"><p class="details"><div><b>Major steps in MP3 and H-cell development.</b> A single segment is shown. (<b>A</b>) <i>sim</i> commits mesectodermal cells to an MP4 fate. (<b>B</b>) <i>wg</i> <span class="highlight" style="background-color:">signaling</span> activates <i>slp1/2</i> expression in the anterior region, which represses <i>en</i> and commits cells to an MP1 fate (pink). (<b>C</b>) <i>hh</i> <span class="highlight" style="background-color:">signaling</span> directs a group of anterior cells to an MP3 fate (green) by activating expression of <i>gsb</i>/<i>gsb-n</i>. (<b>D</b>) <i>Notch</i> <span class="highlight" style="background-color:">signaling</span> selects one cell in the MP3 cluster to become MP3. (<b>E</b>) H-cell and H-cell sib fates are determined by <i>Notch</i> <span class="highlight" style="background-color:">signaling</span> and asymmetric localization of Numb. <i>gsb</i>/<i>gsb-n</i> activate expression of <i>l(1)sc</i> and <i>tup</i> in the H-cell. (<b>F</b>) <i>l(1)sc</i> and <i>tup</i> control H-cell-specific gene expression and differentiation. Genes expressed in common between the H-cell and H-cell sib are regulated by a different <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Joseph D. Watson, et al. Development. 2012 September 15;139(18):3316-3325.</div><p class="links"><a class="dblinks" href="/pubmed/22874915" ref="ordinalpos=4529&amp;ncbi_uid=5305588&amp;link_uid=22874915">Citation</a><a class="dblinks" href="/pmc/articles/PMC3424042/" ref="ordinalpos=4529&amp;ncbi_uid=5305588&amp;link_uid=PMC3424042">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4530.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193436/figure/fig1/" ref="ordinalpos=4530&amp;ncbi_uid=3783263&amp;link_uid=PMC3193436" href="/pmc/articles/PMC3193436/"><img src="/pmc/articles/instance/3193436/bin/cr2010143f1.gif" src-large="/pmc/articles/instance/3193436/bin/cr2010143f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193436/" ref="ordinalpos=4530&amp;ncbi_uid=3783263&amp;link_uid=PMC3193436" image-link="/pmc/articles/PMC3193436/figure/fig1/" class="imagepopup">Figure 1.  From: The GTPase-activating protein Rap1GAP: A new player to modulate Ret <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Model describing the proposed role of Rap1GAP in the control of GDNF-induced Ret <span class="highlight" style="background-color:">signaling</span>. After activation of Ret by the heterocomplex GDNF/GFRα1, Ret becomes phosphorylated at different tyrosine sites. In particular, the Tyr1062 has been described to mediate the activation of Dok-4, which then triggers a sustained activation of Rap1-ERK1/2 <span class="highlight" style="background-color:">pathway</span> and neurite outgrowth. At the same time, Ret activation results in phosphorylation of the Tyr981 (Y981), which is the docking site involved in the recruitment of the negative <span class="highlight" style="background-color:">signaling</span> regulator Rap1GAP. Thus, Rap1GAP counterbalances the Dok-4/Rap1/ERK1/2 <span class="highlight" style="background-color:">pathway</span> and controls neurite outgrowth. The figure also shows a possible role of PKA in the inhibition of Rap1GAP activity (dashed line), and indicates the potential involvement of Rap1 in the modulation of neurite outgrowth through the concomitant activation of Vav2 and/or Tiam1, two GEFs for Rac1 and Cdc-42 (dashed arrow).</div></p></div><div class="aux"><div class="resc">Gustavo Paratcha, et al. Cell Res. 2011 February;21(2):217-219.</div><p class="links"><a class="dblinks" href="/pubmed/20956997" ref="ordinalpos=4530&amp;ncbi_uid=3783263&amp;link_uid=20956997">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193436/" ref="ordinalpos=4530&amp;ncbi_uid=3783263&amp;link_uid=PMC3193436">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4531.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726764/figure/pone-0069668-g001/" ref="ordinalpos=4531&amp;ncbi_uid=5170462&amp;link_uid=PMC3726764" href="/pmc/articles/PMC3726764/"><img src="/pmc/articles/instance/3726764/bin/pone.0069668.g001.gif" src-large="/pmc/articles/instance/3726764/bin/pone.0069668.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726764/" ref="ordinalpos=4531&amp;ncbi_uid=5170462&amp;link_uid=PMC3726764" image-link="/pmc/articles/PMC3726764/figure/pone-0069668-g001/" class="imagepopup">Figure 1. Strategy..  From: Modulation of the Akt <span class="highlight" style="background-color:">Pathway</span> Reveals a Novel Link with PERK/eIF2?, which Is Relevant during Hypoxia. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Chemical structure of the compounds targeting the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> used in this study. (<b>B</b>) Scheme of Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which regulates cell survival, showing the point of action of the drugs shown in <b>A</b>. Akt phosphorylation and activation result from its recruitment to PIP<sub>3</sub> at plasma membrane, after which it exerts cytoplasmic and nuclear functions. Accumulation of PIP<sub>3</sub> classically follows ligand (L) binding to tyrosin kinase cell-surface receptors (RTK), adapter proteins (AP) recruitment to RTK and finally, PI3K activation to phosphorylate PIP<sub>2</sub> to PIP<sub>3</sub>. While Akt can be activated by the UPR it is not known if Akt can also regulate the UPR.</div></p></div><div class="aux"><div class="resc">Matías Blaustein, et al. PLoS One. 2013;8(7):e69668.</div><p class="links"><a class="dblinks" href="/pubmed/23922774" ref="ordinalpos=4531&amp;ncbi_uid=5170462&amp;link_uid=23922774">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726764/" ref="ordinalpos=4531&amp;ncbi_uid=5170462&amp;link_uid=PMC3726764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4532.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2984446/figure/pone-0014027-g010/" ref="ordinalpos=4532&amp;ncbi_uid=2584380&amp;link_uid=PMC2984446" href="/pmc/articles/PMC2984446/"><img src="/pmc/articles/instance/2984446/bin/pone.0014027.g010.gif" src-large="/pmc/articles/instance/2984446/bin/pone.0014027.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2984446/" ref="ordinalpos=4532&amp;ncbi_uid=2584380&amp;link_uid=PMC2984446" image-link="/pmc/articles/PMC2984446/figure/pone-0014027-g010/" class="imagepopup">Figure 10. Model indicating a biphasic response of the homologous recombination <span class="highlight" style="background-color:">pathway</span> induced by the alkylating agent MMS..  From: PI 3 Kinase Related Kinases-Independent Proteolysis of BRCA1 Regulates Rad51 Recruitment during Genotoxic Stress in Human Cells. </a></p><div class="supp"><p class="details"><div>In response to MMS, human cells induce a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that culminates in BRCA1/BARD1 downregulation. This prevents the unwanted assembly of the HR machinery at the early stage of the MMS-induced DNA damage response. At the second stage, recovery and assembly of HR proteins ensure the repair of DSBs generated by replication blocks.</div></p></div><div class="aux"><div class="resc">Ian Hammond-Martel, et al. PLoS One. 2010;5(11):e14027.</div><p class="links"><a class="dblinks" href="/pubmed/21103343" ref="ordinalpos=4532&amp;ncbi_uid=2584380&amp;link_uid=21103343">Citation</a><a class="dblinks" href="/pmc/articles/PMC2984446/" ref="ordinalpos=4532&amp;ncbi_uid=2584380&amp;link_uid=PMC2984446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4533.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2933845/figure/Fig4/" ref="ordinalpos=4533&amp;ncbi_uid=2459612&amp;link_uid=PMC2933845" href="/pmc/articles/PMC2933845/"><img src="/pmc/articles/instance/2933845/bin/10555_2010_9238_Fig4_HTML.gif" src-large="/pmc/articles/instance/2933845/bin/10555_2010_9238_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2933845/" ref="ordinalpos=4533&amp;ncbi_uid=2459612&amp;link_uid=PMC2933845" image-link="/pmc/articles/PMC2933845/figure/Fig4/" class="imagepopup">Fig. 4.  From: Inhibition of cancer cell invasion and metastasis by genistein. </a></p><div class="supp"><p class="details"><div>Genistein inhibits critical pathways in cancer invasion. Our group has demonstrated that genistein can specifically target MEK4. This inhibition results in inactivation of the MEK4 <span class="highlight" style="background-color:">pathway</span>, decreased MMP-2 production, and decreased cell invasion. Genistein also activates Smad1, which is activated by the endoglin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and causes decreased cell invasion. Additionally, genistein inhibits FAK activation, resulting in increased cell adhesion. At this time, it is unclear whether the activation of Smad1 and FAK are due to genistein’s inhibition of MEK4 or via a different <span class="highlight" style="background-color:">signaling</span> mechanism</div></p></div><div class="aux"><div class="resc">Janet M. Pavese, et al. Cancer Metastasis Rev. 2010 September;29(3):465-482.</div><p class="links"><a class="dblinks" href="/pubmed/20730632" ref="ordinalpos=4533&amp;ncbi_uid=2459612&amp;link_uid=20730632">Citation</a><a class="dblinks" href="/pmc/articles/PMC2933845/" ref="ordinalpos=4533&amp;ncbi_uid=2459612&amp;link_uid=PMC2933845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4534.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC493279/figure/F3/" ref="ordinalpos=4534&amp;ncbi_uid=460533&amp;link_uid=PMC493279" href="/pmc/articles/PMC493279/"><img src="/pmc/articles/instance/493279/bin/1476-4598-3-20-3.gif" src-large="/pmc/articles/instance/493279/bin/1476-4598-3-20-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC493279/" ref="ordinalpos=4534&amp;ncbi_uid=460533&amp;link_uid=PMC493279" image-link="/pmc/articles/PMC493279/figure/F3/" class="imagepopup">Figure 3.  From: Molecular profiling of malignant peripheral nerve sheath tumors associated with neurofibromatosis type 1, based on large-scale real-time RT-PCR. </a></p><div class="supp"><p class="details"><div>Expression level of various genes in the dermal neurofibroma pool (white bars), the plexiform neurofibroma pool (gray bars) and the MPNST pool (black bars). A, <i>ERBB </i>family member genes; B, matrix metalloproteinase genes; C, inhibitor of metalloproteinase genes; D, chemokine genes; E, Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes; F, Hedgehog-Gli <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes; G, Gli-regulated genes.</div></p></div><div class="aux"><div class="resc">Pascale Lévy, et al. Mol Cancer. 2004;3:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/15255999" ref="ordinalpos=4534&amp;ncbi_uid=460533&amp;link_uid=15255999">Citation</a><a class="dblinks" href="/pmc/articles/PMC493279/" ref="ordinalpos=4534&amp;ncbi_uid=460533&amp;link_uid=PMC493279">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4535.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151358/figure/F5/" ref="ordinalpos=4535&amp;ncbi_uid=248726&amp;link_uid=PMC151358" href="/pmc/articles/PMC151358/"><img src="/pmc/articles/instance/151358/bin/pq0438011005.gif" src-large="/pmc/articles/instance/151358/bin/pq0438011005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151358/" ref="ordinalpos=4535&amp;ncbi_uid=248726&amp;link_uid=PMC151358" image-link="/pmc/articles/PMC151358/figure/F5/" class="imagepopup">Figure 5.  From: JNK phosphorylation of Bim-related members of the Bcl2 family induces Bax-dependent apoptosis. </a></p><div class="supp"><p class="details"><div>A JNK-dependent apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) Schematic illustration of MAPK phosphorylation of BimL and BimEL. Two groups of MAPK sites were identified. Three Ser-Pro sites are phosphorylated by the ERK group of MAPK within the BimEL-specific insert region. In addition, three JNK phosphorylation sites are located within the common region of BimL and BimEL that is absent from BimS. Phosphorylation of Bim on Thr-56 causes dissociation from DLC1. (<i>B</i>) Schematic illustration of a JNK-dependent apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is initiated by the phosphorylation of Bim and Bmf by JNK. In some cells, JNK may also contribute to Bim expression. The phosphorylation by JNK is proposed to cause the release of Bim and Bmf from dynein and myosin V motor complexes. The activated Bim and Bmf may directly activate Bax and Bak or may indirectly activate Bax and Bak by binding antiapoptotic Bcl2 family proteins (e.g., Bcl2 and Bcl-<span style="font-variant: small-caps" class="small-caps">X</span><sub><span style="font-variant: small-caps" class="small-caps">l</span></sub>). These mechanisms can lead to the engagement of the Bax/Bak-dependent mitochondrial <span class="highlight" style="background-color:">pathway</span> of apoptosis. Smac, second mitochondria-derived activator of caspase; HtrA2, second homology of the bacterial HtrA endoprotease; AIF, apoptosis-inducing factor; and EndoG, endonuclease G.</div></p></div><div class="aux"><div class="resc">Kui Lei, et al. Proc Natl Acad Sci U S A. 2003 March 4;100(5):2432-2437.</div><p class="links"><a class="dblinks" href="/pubmed/12591950" ref="ordinalpos=4535&amp;ncbi_uid=248726&amp;link_uid=12591950">Citation</a><a class="dblinks" href="/pmc/articles/PMC151358/" ref="ordinalpos=4535&amp;ncbi_uid=248726&amp;link_uid=PMC151358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4536.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281420/figure/f4-cancers-03-01639/" ref="ordinalpos=4536&amp;ncbi_uid=5273355&amp;link_uid=PMC3281420" href="/pmc/articles/PMC3281420/"><img src="/pmc/articles/instance/3757383/bin/cancers-03-01639f4.gif" src-large="/pmc/articles/instance/3757383/bin/cancers-03-01639f4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281420/" ref="ordinalpos=4536&amp;ncbi_uid=5273355&amp;link_uid=PMC3281420" image-link="/pmc/articles/PMC3281420/figure/f4-cancers-03-01639/" class="imagepopup">Figure 4. From: Targeting the Anti-Apoptotic Protein c-FLIP for Cancer Therapy. </a></p><div class="supp"><p class="details"><div>ATM kinase activity downregulates c-FLIP<sub>L</sub> and c-FLIP<sub>S</sub> at the protein level and connects DNA damage <span class="highlight" style="background-color:">signaling</span> to TRAIL-induced apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. DNA damaging agents induce ATM activation, which promotes c-FLIP<sub>L</sub> protein degradation and c-FLIP<sub>S</sub> downregulation through an unknown mechanism (Stagni <i>et al.</i> [122]).</div></p></div><div class="aux"><div class="resc">Ahmad R. Safa, et al. Cancers (Basel). 2011 June;3(2):1639-1671.</div><p class="links"><a class="dblinks" href="/pubmed/22348197" ref="ordinalpos=4536&amp;ncbi_uid=5273355&amp;link_uid=22348197">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281420/" ref="ordinalpos=4536&amp;ncbi_uid=5273355&amp;link_uid=PMC3281420">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4537.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4300045/figure/Fig3/" ref="ordinalpos=4537&amp;ncbi_uid=7080735&amp;link_uid=PMC4300045" href="/pmc/articles/PMC4300045/"><img src="/pmc/articles/instance/4300045/bin/13039_2014_95_Fig3_HTML.gif" src-large="/pmc/articles/instance/4300045/bin/13039_2014_95_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4300045/" ref="ordinalpos=4537&amp;ncbi_uid=7080735&amp;link_uid=PMC4300045" image-link="/pmc/articles/PMC4300045/figure/Fig3/" class="imagepopup">Figure 3.  From: Genetic changes observed in a case of adult pilocytic astrocytoma revealed by array CGH analysis. </a></p><div class="supp"><p class="details"><div> <b>Gene ontology (GO) analysis using the DAVID software.</b> Role of the affected genes within the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nives Pećina-Šlaus, et al. Mol Cytogenet. 2014;7(1):95.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4537&amp;ncbi_uid=7080735&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4300045/" ref="ordinalpos=4537&amp;ncbi_uid=7080735&amp;link_uid=PMC4300045">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4538.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4272520/figure/Fig3/" ref="ordinalpos=4538&amp;ncbi_uid=6984663&amp;link_uid=PMC4272520" href="/pmc/articles/PMC4272520/"><img src="/pmc/articles/instance/4272520/bin/12934_2014_174_Fig3_HTML.gif" src-large="/pmc/articles/instance/4272520/bin/12934_2014_174_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4272520/" ref="ordinalpos=4538&amp;ncbi_uid=6984663&amp;link_uid=PMC4272520" image-link="/pmc/articles/PMC4272520/figure/Fig3/" class="imagepopup">Figure 3.  From: Identification and characterization of the nitrate assimilation genes in the isolate of Streptomyces griseorubens JSD-1. </a></p><div class="supp"><p class="details"><div> <b>Enzymes involved in the nitrogen metabolic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from KEGG annotation.</b> Genes marked with grey represent the existence in this species.</div></p></div><div class="aux"><div class="resc">Haiwei Feng, et al. Microb Cell Fact. 2014;13(1):174.</div><p class="links"><a class="dblinks" href="/pubmed/25492123" ref="ordinalpos=4538&amp;ncbi_uid=6984663&amp;link_uid=25492123">Citation</a><a class="dblinks" href="/pmc/articles/PMC4272520/" ref="ordinalpos=4538&amp;ncbi_uid=6984663&amp;link_uid=PMC4272520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4539.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3103528/figure/pone-0020087-g005/" ref="ordinalpos=4539&amp;ncbi_uid=3001267&amp;link_uid=PMC3103528" href="/pmc/articles/PMC3103528/"><img src="/pmc/articles/instance/3103528/bin/pone.0020087.g005.gif" src-large="/pmc/articles/instance/3103528/bin/pone.0020087.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3103528/" ref="ordinalpos=4539&amp;ncbi_uid=3001267&amp;link_uid=PMC3103528" image-link="/pmc/articles/PMC3103528/figure/pone-0020087-g005/" class="imagepopup">Figure 5. TGF-beta <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (KEGG) showing rat P0-Ovary genes affected by AMH treatment..  From: Inhibitory Actions of Anti-M?llerian Hormone (AMH) on Ovarian Primordial Follicle Assembly. </a></p><div class="supp"><p class="details"><div>Red boxes – up-regulated, green and white boxes –not affected genes.</div></p></div><div class="aux"><div class="resc">Eric E. Nilsson, et al. PLoS One. 2011;6(5):e20087.</div><p class="links"><a class="dblinks" href="/pubmed/21637711" ref="ordinalpos=4539&amp;ncbi_uid=3001267&amp;link_uid=21637711">Citation</a><a class="dblinks" href="/pmc/articles/PMC3103528/" ref="ordinalpos=4539&amp;ncbi_uid=3001267&amp;link_uid=PMC3103528">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4540.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3070284/figure/F2/" ref="ordinalpos=4540&amp;ncbi_uid=2874056&amp;link_uid=PMC3070284" href="/pmc/articles/PMC3070284/"><img src="/pmc/articles/instance/3070284/bin/gcr0011000070002.gif" src-large="/pmc/articles/instance/3070284/bin/gcr0011000070002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3070284/" ref="ordinalpos=4540&amp;ncbi_uid=2874056&amp;link_uid=PMC3070284" image-link="/pmc/articles/PMC3070284/figure/F2/" class="imagepopup">Figure 2. From: Molecularly Targeted Therapy for Metastatic Colon Cancer: Proven Treatments and Promising New Agents. </a></p><div class="supp"><p class="details"><div>The endothelial growth factor receptor (EGFR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Majed El Zouhairi, et al. Gastrointest Cancer Res. 2011 Jan-Feb;4(1):15-21.</div><p class="links"><a class="dblinks" href="/pubmed/21464866" ref="ordinalpos=4540&amp;ncbi_uid=2874056&amp;link_uid=21464866">Citation</a><a class="dblinks" href="/pmc/articles/PMC3070284/" ref="ordinalpos=4540&amp;ncbi_uid=2874056&amp;link_uid=PMC3070284">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4541.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2944875/figure/pone-0012949-g002/" ref="ordinalpos=4541&amp;ncbi_uid=2494680&amp;link_uid=PMC2944875" href="/pmc/articles/PMC2944875/"><img src="/pmc/articles/instance/2944875/bin/pone.0012949.g002.gif" src-large="/pmc/articles/instance/2944875/bin/pone.0012949.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2944875/" ref="ordinalpos=4541&amp;ncbi_uid=2494680&amp;link_uid=PMC2944875" image-link="/pmc/articles/PMC2944875/figure/pone-0012949-g002/" class="imagepopup">Figure 2. GenMAPP pathways..  From: Gene Expression Study of Monocytes/Macrophages during Early Foreign Body Reaction and Identification of Potential Precursors of Myofibroblasts. </a></p><div class="supp"><p class="details"><div>Upregulated genes present in GenMAPP pathways of “TGF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” (A) and “cytokines and inflammatory response” (B).</div></p></div><div class="aux"><div class="resc">Lindsay Mesure, et al. PLoS One. 2010;5(9):e12949.</div><p class="links"><a class="dblinks" href="/pubmed/20886081" ref="ordinalpos=4541&amp;ncbi_uid=2494680&amp;link_uid=20886081">Citation</a><a class="dblinks" href="/pmc/articles/PMC2944875/" ref="ordinalpos=4541&amp;ncbi_uid=2494680&amp;link_uid=PMC2944875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4542.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3915939/figure/F4/" ref="ordinalpos=4542&amp;ncbi_uid=5793883&amp;link_uid=PMC3915939" href="/pmc/articles/PMC3915939/"><img src="/pmc/articles/instance/3915939/bin/nihms548887f4.gif" src-large="/pmc/articles/instance/3915939/bin/nihms548887f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3915939/" ref="ordinalpos=4542&amp;ncbi_uid=5793883&amp;link_uid=PMC3915939" image-link="/pmc/articles/PMC3915939/figure/F4/" class="imagepopup">Fig. 4.  From: EGFR(S) Inhibitors in the Treatment of Gastro-Intestinal Cancers: What's New?. </a></p><div class="supp"><p class="details"><div>Schematic representation of ligand induced activation of EGFR(s) signalling <span class="highlight" style="background-color:">pathway</span>. Ligand binding to the receptor results in dimerization and auto-/trans-phosphorylation that initiate the <span class="highlight" style="background-color:">signaling</span> cascade.</div></p></div><div class="aux"><div class="resc">Shailender Singh Kanwar, et al. Curr Drug Targets. ;11(6):682-698.</div><p class="links"><a class="dblinks" href="/pubmed/20298154" ref="ordinalpos=4542&amp;ncbi_uid=5793883&amp;link_uid=20298154">Citation</a><a class="dblinks" href="/pmc/articles/PMC3915939/" ref="ordinalpos=4542&amp;ncbi_uid=5793883&amp;link_uid=PMC3915939">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4543.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766677/figure/F7/" ref="ordinalpos=4543&amp;ncbi_uid=2525344&amp;link_uid=PMC2766677" href="/pmc/articles/PMC2766677/"><img src="/pmc/articles/instance/2766677/bin/zh89990943220007.gif" src-large="/pmc/articles/instance/2766677/bin/zh89990943220007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766677/" ref="ordinalpos=4543&amp;ncbi_uid=2525344&amp;link_uid=PMC2766677" image-link="/pmc/articles/PMC2766677/figure/F7/" class="imagepopup">Figure 7.  From: HIV-1 gp120-induced migration of dendritic cells is regulated by a novel kinase cascade involving Pyk2, p38 MAP kinase, and LSP1. </a></p><div class="supp"><p class="details"><div><b>Proposed scheme of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to immature dendritic cell chemotaxis upon exposure to M-tropic gp120</b>.</div></p></div><div class="aux"><div class="resc">Appakkudal R. Anand, et al. Blood. 2009 October 22;114(17):3588-3600.</div><p class="links"><a class="dblinks" href="/pubmed/19700666" ref="ordinalpos=4543&amp;ncbi_uid=2525344&amp;link_uid=19700666">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766677/" ref="ordinalpos=4543&amp;ncbi_uid=2525344&amp;link_uid=PMC2766677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4544.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2242778/figure/f5-mol14_5p0276/" ref="ordinalpos=4544&amp;ncbi_uid=1331656&amp;link_uid=PMC2242778" href="/pmc/articles/PMC2242778/"><img src="/pmc/articles/instance/2374727/bin/mol14_5p0276f5.gif" src-large="/pmc/articles/instance/2374727/bin/mol14_5p0276f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2242778/" ref="ordinalpos=4544&amp;ncbi_uid=1331656&amp;link_uid=PMC2242778" image-link="/pmc/articles/PMC2242778/figure/f5-mol14_5p0276/" class="imagepopup">Figure 5.  From: Molecular Pathways Involved in Loss of Kidney Graft Function with Tubular Atrophy and Interstitial Fibrosis. </a></p><div class="supp"><p class="details"><div>Canonical <span class="highlight" style="background-color:">pathway</span> analysis. Leukocyte transvasation <span class="highlight" style="background-color:">signaling</span>: genes differentially expressed in IF/TA samples. Red, overexpressed genes; green, underexpressed genes.</div></p></div><div class="aux"><div class="resc">Daniel G Maluf, et al. Mol Med. 2008 May-Jun;14(5-6):276-285.</div><p class="links"><a class="dblinks" href="/pubmed/18286166" ref="ordinalpos=4544&amp;ncbi_uid=1331656&amp;link_uid=18286166">Citation</a><a class="dblinks" href="/pmc/articles/PMC2242778/" ref="ordinalpos=4544&amp;ncbi_uid=1331656&amp;link_uid=PMC2242778">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4545.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3624394/figure/F4/" ref="ordinalpos=4545&amp;ncbi_uid=5352829&amp;link_uid=PMC3624394" href="/pmc/articles/PMC3624394/"><img src="/pmc/articles/instance/3624394/bin/zjv9990974730004.gif" src-large="/pmc/articles/instance/3624394/bin/zjv9990974730004.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3624394/" ref="ordinalpos=4545&amp;ncbi_uid=5352829&amp;link_uid=PMC3624394" image-link="/pmc/articles/PMC3624394/figure/F4/" class="imagepopup">Fig 4.  From: The Human Papillomavirus Type 8 E6 Protein Interferes with NOTCH Activation during Keratinocyte Differentiation. </a></p><div class="supp"><p class="details"><div>Comparative analysis of NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components in calcium-differentiated 8E6-iHFKs and GSI-treated C-HFKs by qRT-PCR. 8E6-iHFKs treated with DMSO and C-iHFKs treated with either DMSO or CpdE (2 μM) were grown for 6 days in calcium differentiation medium, and total RNA was collected and analyzed using the human NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> PCR array (SABiosciences). Genes corresponding to those identified in the KEGG NOTCH <span class="highlight" style="background-color:">pathway</span> are displayed, with relative mRNA levels compared to those for differentiated C-iHFKs treated with DMSO indicated using the color scheme shown at the bottom. Results represent averages from duplicate PCR arrays.</div></p></div><div class="aux"><div class="resc">Jordan M. Meyers, et al. J Virol. 2013 April;87(8):4762-4767.</div><p class="links"><a class="dblinks" href="/pubmed/23365452" ref="ordinalpos=4545&amp;ncbi_uid=5352829&amp;link_uid=23365452">Citation</a><a class="dblinks" href="/pmc/articles/PMC3624394/" ref="ordinalpos=4545&amp;ncbi_uid=5352829&amp;link_uid=PMC3624394">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4546.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3674667/figure/F7/" ref="ordinalpos=4546&amp;ncbi_uid=6220750&amp;link_uid=PMC3674667" href="/pmc/articles/PMC3674667/"><img src="/pmc/articles/instance/3674667/bin/4663f7.gif" src-large="/pmc/articles/instance/3674667/bin/4663f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3674667/" ref="ordinalpos=4546&amp;ncbi_uid=6220750&amp;link_uid=PMC3674667" image-link="/pmc/articles/PMC3674667/figure/F7/" class="imagepopup">Figure 7.  From: MicroRNAs activate natural killer cells through Toll-like receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Downregulation of the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components in NK cells from lymphoma patients.</b> (A,B) NK cells were isolated from PBMCs of both healthy donors and lymphoma patients as described in the Materials and Methods section. The purified NK cells were then immediately subjected to RNA extraction and cDNA synthesis. The expression levels of p65 (A) and IRAK1 (B) were determined by SYBR Green real-time RT-PCR assay.</div></p></div><div class="aux"><div class="resc">Shun He, et al. Blood. 2013 June 6;121(23):4663-4671.</div><p class="links"><a class="dblinks" href="/pubmed/23580661" ref="ordinalpos=4546&amp;ncbi_uid=6220750&amp;link_uid=23580661">Citation</a><a class="dblinks" href="/pmc/articles/PMC3674667/" ref="ordinalpos=4546&amp;ncbi_uid=6220750&amp;link_uid=PMC3674667">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4547.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3287237/figure/F1/" ref="ordinalpos=4547&amp;ncbi_uid=3836180&amp;link_uid=PMC3287237" href="/pmc/articles/PMC3287237/"><img src="/pmc/articles/instance/3287237/bin/JCI61498.f1.gif" src-large="/pmc/articles/instance/3287237/bin/JCI61498.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3287237/" ref="ordinalpos=4547&amp;ncbi_uid=3836180&amp;link_uid=PMC3287237" image-link="/pmc/articles/PMC3287237/figure/F1/" class="imagepopup">Figure 1. Altered TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the palates of Tgfbr2fl/fl;Wnt1-Cre mice. .  From: Modulation of noncanonical TGF-? <span class="highlight" style="background-color:">signaling</span> prevents cleft palate in Tgfbr2 mutant mice . </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Quantitative RT-PCR analyses of indicated genes in the palates of <i>Tgfbr2<sup>fl/fl</sup></i> (white bars) and <i>Tgfbr2<sup>fl/fl</sup>;Wnt1-Cre</i> (black bars) mice at E13.5 and E14.5. Three samples were analyzed for each experiment. Error bars represent SD. *<i>P</i> &lt; 0.05 as indicated by a 2-tailed Student’s <i>t</i> test. (<b>B</b>) Immunoblotting analysis of indicated molecules in E14.5 <i>Tgfbr2<sup>fl/fl</sup></i> and <i>Tgfbr2<sup>fl/fl</sup>;Wnt1-Cre</i> palates. </div></p></div><div class="aux"><div class="resc">Jun-ichi Iwata, et al. J Clin Invest. 2012 March 1;122(3):873-885.</div><p class="links"><a class="dblinks" href="/pubmed/22326956" ref="ordinalpos=4547&amp;ncbi_uid=3836180&amp;link_uid=22326956">Citation</a><a class="dblinks" href="/pmc/articles/PMC3287237/" ref="ordinalpos=4547&amp;ncbi_uid=3836180&amp;link_uid=PMC3287237">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4548.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3671288/figure/F2/" ref="ordinalpos=4548&amp;ncbi_uid=4983640&amp;link_uid=PMC3671288" href="/pmc/articles/PMC3671288/"><img src="/pmc/articles/instance/3671288/bin/fimmu-04-00130-g002.gif" src-large="/pmc/articles/instance/3671288/bin/fimmu-04-00130-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3671288/" ref="ordinalpos=4548&amp;ncbi_uid=4983640&amp;link_uid=PMC3671288" image-link="/pmc/articles/PMC3671288/figure/F2/" class="imagepopup">Figure 2.  From: Proinflammatory and Immunoregulatory Roles of Eicosanoids in T Cells. </a></p><div class="supp"><p class="details"><div><b>Inhibitory <span class="highlight" style="background-color:">pathway</span> of PGE2 in effector T cells</b>. PGE<sub>2</sub> mediates Treg inhibition of effector T cell function through a PKA-mediated <span class="highlight" style="background-color:">pathway</span>. <b>(A)</b> In response to continuous antigen exposure, for instance in cancer and HIV, adaptive regulatory T cells express COX-2 and produce PGE<sub>2</sub>, which stimulates FOXP3 expression in these cells. The Treg-derived PGE<sub>2</sub> can signal through the EP2 and EP4 receptors on effector T cells to inhibit the function of these cells through the <span class="highlight" style="background-color:">pathway</span> shown in <b>(B)</b>. Binding of PGE<sub>2</sub> to its receptors on effector T cells stimulates adenylyl cyclase activity, which increases intracellular cAMP levels and thus activates PKA. Aided by an Ezrin-EBP50-PAG scaffold, PKA phosphorylates Csk, which in turn phosphorylates Lck to inhibit its activity. Lck normally acts to promote TCR <span class="highlight" style="background-color:">signaling</span>; thus Lck inhibition through this PGE<sub>2</sub>-initiated <span class="highlight" style="background-color:">pathway</span> inhibits TCR <span class="highlight" style="background-color:">signaling</span> in effector T cells.</div></p></div><div class="aux"><div class="resc">Anna Mari Lone, et al. Front Immunol. 2013;4:130.</div><p class="links"><a class="dblinks" href="/pubmed/23760108" ref="ordinalpos=4548&amp;ncbi_uid=4983640&amp;link_uid=23760108">Citation</a><a class="dblinks" href="/pmc/articles/PMC3671288/" ref="ordinalpos=4548&amp;ncbi_uid=4983640&amp;link_uid=PMC3671288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4549.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4316787/figure/F4/" ref="ordinalpos=4549&amp;ncbi_uid=7192251&amp;link_uid=PMC4316787" href="/pmc/articles/PMC4316787/"><img src="/pmc/articles/instance/4316787/bin/fimmu-06-00023-g004.gif" src-large="/pmc/articles/instance/4316787/bin/fimmu-06-00023-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4316787/" ref="ordinalpos=4549&amp;ncbi_uid=7192251&amp;link_uid=PMC4316787" image-link="/pmc/articles/PMC4316787/figure/F4/" class="imagepopup">Figure 4.  From: Analysis of the Bovine Monocyte-Derived Macrophage Response to Mycobacterium avium Subspecies Paratuberculosis Infection Using RNA-seq. </a></p><div class="supp"><p class="details"><div><b>The fourth-ranked enriched canonical <span class="highlight" style="background-color:">pathway</span> identified using IPA at 2 hpi – the <i>CD40 <span class="highlight" style="background-color:">signaling</span></i> <span class="highlight" style="background-color:">pathway</span></b>. Red shading indicates increased expression in MAP-infected MDM relative to the non-infected control MDM. Green shading indicates decreased expression in MAP-infected MDM relative to the non-infected control MDM. White and gray shading indicates non-expression and non-differential expression, respectively.</div></p></div><div class="aux"><div class="resc">Maura E. Casey, et al. Front Immunol. 2015;6:23.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4549&amp;ncbi_uid=7192251&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4316787/" ref="ordinalpos=4549&amp;ncbi_uid=7192251&amp;link_uid=PMC4316787">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4550.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2259464/figure/fig3/" ref="ordinalpos=4550&amp;ncbi_uid=3044797&amp;link_uid=PMC2259464" href="/pmc/articles/PMC2259464/"><img src="/pmc/articles/instance/2259464/bin/gr3.gif" src-large="/pmc/articles/instance/2259464/bin/gr3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2259464/" ref="ordinalpos=4550&amp;ncbi_uid=3044797&amp;link_uid=PMC2259464" image-link="/pmc/articles/PMC2259464/figure/fig3/" class="imagepopup">Figure 3.  From: Molecular Genetics of Pancreatic Ductal Adenocarcinomas and Recent Implications for Translational Efforts. </a></p><div class="supp"><p class="details"><div>The Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves a series of inhibitory interactions between secreted ligands, cell surface receptors, and cytoplasmic proteins, culminating in the activation of transcription and up-regulation of Hh target genes by the DNA-binding transcription factor GLI. The Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by binding of Hh ligands—Sonic, Desert, or Indian hedgehog (SHH, DHH, IHH)—to the 12-transmembrane receptor Patched (PTCH). This interaction releases the inhibitory effects of PTC on the 7-transmembrane smoothened (SMO) receptor, and the conformational change with activation that SMO undergoes, in turn, facilitates its inhibition of the cytoplasmic protein Suppressor of Fused (SUFU). SUFU inhibition allows GLI to escape cytoplasmic sequestration, on which it undergoes nuclear translocation and activates the transcription of Hh target genes.</div></p></div><div class="aux"><div class="resc">Georg Feldmann, et al. J Mol Diagn. 2008 March;10(2):111-122.</div><p class="links"><a class="dblinks" href="/pubmed/18258927" ref="ordinalpos=4550&amp;ncbi_uid=3044797&amp;link_uid=18258927">Citation</a><a class="dblinks" href="/pmc/articles/PMC2259464/" ref="ordinalpos=4550&amp;ncbi_uid=3044797&amp;link_uid=PMC2259464">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4551.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3589936/figure/RRS098F1/" ref="ordinalpos=4551&amp;ncbi_uid=4682621&amp;link_uid=PMC3589936" href="/pmc/articles/PMC3589936/"><img src="/pmc/articles/instance/3589936/bin/rrs09801.gif" src-large="/pmc/articles/instance/3589936/bin/rrs09801.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3589936/" ref="ordinalpos=4551&amp;ncbi_uid=4682621&amp;link_uid=PMC3589936" image-link="/pmc/articles/PMC3589936/figure/RRS098F1/" class="imagepopup">Fig. 1. From: Anti-radiation damage effect of polyethylenimine as a toll-like receptor 5 targeted agonist. </a></p><div class="supp"><p class="details"><div>hTLR5 recombinant plasmid construction and cell-based screening model that activates hTLR5-mediated NF-kB <span class="highlight" style="background-color:">signaling</span>. After the hTLR5 gene was obtained from CaCo-2 cells, sub-cloning strategy was applied to clone the hTLR5 into pcDNA3.1-V5/His B vector. Following digestion and expression verification by western blot (WB), the recombinant plasmid was transfected into cells, and hTLR5-mediated NF-kB <span class="highlight" style="background-color:">signaling</span> was assayed by treatment of flagellin. (<b>A</b>) Flow diagram for construction of the recombinant plasmid of hTLR5. (<b>B</b>) hTLR5 construct sub-cloning and identification with restriction enzyme digestion. (<b>C</b>) hTLR5 protein expression identified by WB. (<b>D</b>) Cell-based screening model that activates hTLR5-mediated NF-kB <span class="highlight" style="background-color:">signaling</span>. Dual-luciferase reporter gene expression that can be activated by flagellin via the NF-κB intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by TLR5. *<i>P</i>&lt; 0.05 vs. control.</div></p></div><div class="aux"><div class="resc">Zhiqiang Hu, et al. J Radiat Res. 2013 March;54(2):243-250.</div><p class="links"><a class="dblinks" href="/pubmed/23104900" ref="ordinalpos=4551&amp;ncbi_uid=4682621&amp;link_uid=23104900">Citation</a><a class="dblinks" href="/pmc/articles/PMC3589936/" ref="ordinalpos=4551&amp;ncbi_uid=4682621&amp;link_uid=PMC3589936">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4552.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3852811/figure/F1/" ref="ordinalpos=4552&amp;ncbi_uid=5567876&amp;link_uid=PMC3852811" href="/pmc/articles/PMC3852811/"><img src="/pmc/articles/instance/3852811/bin/emss-52517-f0001.gif" src-large="/pmc/articles/instance/3852811/bin/emss-52517-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3852811/" ref="ordinalpos=4552&amp;ncbi_uid=5567876&amp;link_uid=PMC3852811" image-link="/pmc/articles/PMC3852811/figure/F1/" class="imagepopup">Figure 1. A metabolic network of aging.  From: Metabolic networks of longevity. </a></p><div class="supp"><p class="details"><div>A simplified scheme for metabolic proteins that are involved in longevity. There are complex interactions among individual metabolic <span class="highlight" style="background-color:">signaling</span> pathways, such as the insulin/IGF1, TOR, AMPK, sirtuin and FOXO pathways. These interactions impact healthspan and longevity. For instance, the TOR <span class="highlight" style="background-color:">pathway</span>, which is activated in situations of nutrient excess, is not only linked to other nutrient-responsive pathways, such as the insulin/IGF1 <span class="highlight" style="background-color:">pathway</span>, but also to the CR-induced <span class="highlight" style="background-color:">pathway</span> involving AMPK and Foxa activation. Longevity promoting proteins are in green, proteins that when mutated increase longevity are in red, other proteins are indicated in blue; mammalian protein names are used.</div></p></div><div class="aux"><div class="resc">Riekelt H. Houtkooper, et al. Cell. ;142(1):10.1016/j.cell.2010.06.029.</div><p class="links"><a class="dblinks" href="/pubmed/20603007" ref="ordinalpos=4552&amp;ncbi_uid=5567876&amp;link_uid=20603007">Citation</a><a class="dblinks" href="/pmc/articles/PMC3852811/" ref="ordinalpos=4552&amp;ncbi_uid=5567876&amp;link_uid=PMC3852811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4553.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3578760/figure/pgen-1003315-g008/" ref="ordinalpos=4553&amp;ncbi_uid=4645948&amp;link_uid=PMC3578760" href="/pmc/articles/PMC3578760/"><img src="/pmc/articles/instance/3578760/bin/pgen.1003315.g008.gif" src-large="/pmc/articles/instance/3578760/bin/pgen.1003315.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3578760/" ref="ordinalpos=4553&amp;ncbi_uid=4645948&amp;link_uid=PMC3578760" image-link="/pmc/articles/PMC3578760/figure/pgen-1003315-g008/" class="imagepopup">Figure 8. Proposed model for the KGB-1 <span class="highlight" style="background-color:">pathway</span> in stress response..  From: The Caenorhabditis elegans JNK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activates Expression of Stress Response Genes by Derepressing the Fos/HDAC Repressor Complex. </a></p><div class="supp"><p class="details"><div>In the absence of heavy metal stress, FOS-1 forms homodimers and binds to the TRE2 motif in the <i>kreg</i> target promoter. FOS-1 dimerization potentiates recruitment of HDA-1 to the promoter. The FOS-1/HDA-1 repressor complex represses transcription of <i>kreg</i> target genes (left panel). In the presence of heavy metal stress, the KGB-1 <span class="highlight" style="background-color:">pathway</span> is activated and FOS-1 is phosphorylated by KGB-1. FOS-1 phosphorylation leads to a switch from dimer to monomer, resulting in dissociation of the FOS-1/HDA-1 repressor complex from the target promoter that activates transcription of <i>kreg</i> genes (right panel).</div></p></div><div class="aux"><div class="resc">Ayuna Hattori, et al. PLoS Genet. 2013 February;9(2):e1003315.</div><p class="links"><a class="dblinks" href="/pubmed/23437011" ref="ordinalpos=4553&amp;ncbi_uid=4645948&amp;link_uid=23437011">Citation</a><a class="dblinks" href="/pmc/articles/PMC3578760/" ref="ordinalpos=4553&amp;ncbi_uid=4645948&amp;link_uid=PMC3578760">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4554.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4314091/figure/F1/" ref="ordinalpos=4554&amp;ncbi_uid=7120508&amp;link_uid=PMC4314091" href="/pmc/articles/PMC4314091/"><img src="/pmc/articles/instance/4314091/bin/nihms657287f1.gif" src-large="/pmc/articles/instance/4314091/bin/nihms657287f1.jpg" alt="Figure" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4314091/" ref="ordinalpos=4554&amp;ncbi_uid=7120508&amp;link_uid=PMC4314091" image-link="/pmc/articles/PMC4314091/figure/F1/" class="imagepopup">Figure. Blocking Wnt <span class="highlight" style="background-color:">signaling</span>.  From: The Unusual Case of Porcupine. </a></p><div class="supp"><p class="details"><div>Two chemical genetic strategies have identified small molecules that inhibit Wnt <span class="highlight" style="background-color:">signaling</span>. Certain cancers have mutations in constituents of the <span class="highlight" style="background-color:">pathway</span>. Apc, adenomatous polyposis coli; Tcf, T cell factor; Lef, lymphoid enhancer binding factor; Fzd, Frizzled; Lkb1, liver kinase B1; Rnf43, ring finger protein 43; Lrp5/6, LDL-related proteins 5/6; β-cat, β-catenin.</div></p></div><div class="aux"><div class="resc">Lawrence Lum, et al. Science. ;337(6097):922-923.</div><p class="links"><a class="dblinks" href="/pubmed/22923569" ref="ordinalpos=4554&amp;ncbi_uid=7120508&amp;link_uid=22923569">Citation</a><a class="dblinks" href="/pmc/articles/PMC4314091/" ref="ordinalpos=4554&amp;ncbi_uid=7120508&amp;link_uid=PMC4314091">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4555.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3210119/figure/pone-0025507-g002/" ref="ordinalpos=4555&amp;ncbi_uid=3403671&amp;link_uid=PMC3210119" href="/pmc/articles/PMC3210119/"><img src="/pmc/articles/instance/3210119/bin/pone.0025507.g002.gif" src-large="/pmc/articles/instance/3210119/bin/pone.0025507.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3210119/" ref="ordinalpos=4555&amp;ncbi_uid=3403671&amp;link_uid=PMC3210119" image-link="/pmc/articles/PMC3210119/figure/pone-0025507-g002/" class="imagepopup">Figure 2. C225-NP regulates EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in tumor cells but not in normal cells..  From: EGFR-Targeted Hybrid Plasmonic Magnetic Nanoparticles Synergistically Induce Autophagy and Apoptosis in Non-Small Cell Lung Cancer Cells. </a></p><div class="supp"><p class="details"><div>Inhibitory effects of C225-NP on phosphorylated EGFR and the downstream <span class="highlight" style="background-color:">signaling</span> molecules in human lung cancer and normal cells. C225-NP treatment reduced the expression of phosphorylated (p) forms of EGFR, AKT, p38 MAPK, and P44/42 MAPK in EGFR-positive tumor cells but not in normal cells. There was no effect on EGFR-null H520 tumor cells.</div></p></div><div class="aux"><div class="resc">Tomohisa Yokoyama, et al. PLoS One. 2011;6(11):e25507.</div><p class="links"><a class="dblinks" href="/pubmed/22087216" ref="ordinalpos=4555&amp;ncbi_uid=3403671&amp;link_uid=22087216">Citation</a><a class="dblinks" href="/pmc/articles/PMC3210119/" ref="ordinalpos=4555&amp;ncbi_uid=3403671&amp;link_uid=PMC3210119">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4556.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2447927/figure/F10/" ref="ordinalpos=4556&amp;ncbi_uid=1508681&amp;link_uid=PMC2447927" href="/pmc/articles/PMC2447927/"><img src="/pmc/articles/instance/2491375/bin/JCI0834277.f10.gif" src-large="/pmc/articles/instance/2491375/bin/JCI0834277.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2447927/" ref="ordinalpos=4556&amp;ncbi_uid=1508681&amp;link_uid=PMC2447927" image-link="/pmc/articles/PMC2447927/figure/F10/" class="imagepopup">Figure 10. Insulin <span class="highlight" style="background-color:">signaling</span> in the MBH is improved by adenoviral inhibition of the mTOR/S6K <span class="highlight" style="background-color:">pathway</span>..  From: Activation of hypothalamic S6 kinase mediates diet-induced hepatic insulin resistance in rats. </a></p><div class="supp"><p class="details"><div>Rats were fed NC or HFD for 1 d, and insulin <span class="highlight" style="background-color:">signaling</span> was analyzed 15 min after infusion of insulin (4 μU total) into the MBH. The tyrosine phosphorylation of IRS-1 (<b>A</b>) and phosphorylation of Akt at Thr308 (<b>B</b>) and at Ser473 (<b>C</b>) by insulin were enhanced in response to the overexpression of DN-S6K or Raptor/ΔCT. All graphs show the ratio of phosphoproteins to total proteins. *<i>P</i> &lt; 0.05; **<i>P</i> &lt; 0.01.    </div></p></div><div class="aux"><div class="resc">Hiraku Ono, et al. J Clin Invest. 2008 August 1;118(8):2959-2968.</div><p class="links"><a class="dblinks" href="/pubmed/18618016" ref="ordinalpos=4556&amp;ncbi_uid=1508681&amp;link_uid=18618016">Citation</a><a class="dblinks" href="/pmc/articles/PMC2447927/" ref="ordinalpos=4556&amp;ncbi_uid=1508681&amp;link_uid=PMC2447927">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4557.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2423294/figure/f8/" ref="ordinalpos=4557&amp;ncbi_uid=1478963&amp;link_uid=PMC2423294" href="/pmc/articles/PMC2423294/"><img src="/pmc/articles/instance/2423294/bin/zmb0110875290008.gif" src-large="/pmc/articles/instance/2423294/bin/zmb0110875290008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2423294/" ref="ordinalpos=4557&amp;ncbi_uid=1478963&amp;link_uid=PMC2423294" image-link="/pmc/articles/PMC2423294/figure/f8/" class="imagepopup">FIG. 8. From: Protein Kinase D1 Stimulates MEF2 Activity in Skeletal Muscle and Enhances Muscle Performance . </a></p><div class="supp"><p class="details"><div>Model for the PKD1/class II HDAC/MEF2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in skeletal muscle. Activation of PKD1 in skeletal muscle induces expression of the slow-twitch-fiber-specific contractile gene program and a subset of oxidative genes via phosphorylation of class II HDACs, which blocks their inhibitory influence on MEF2 as a consequence of their nuclear export. The calcineurin/NFAT <span class="highlight" style="background-color:">pathway</span> promotes the slow-twitch-fiber gene program and up-regulates a subset of oxidative metabolism genes (6, 9, 22, 31, 35). The CaMK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activates slow-twitch-fiber contractile genes and the oxidative metabolism program via activation of MEF2 and coactivator PGC-1 (19, 44).</div></p></div><div class="aux"><div class="resc">Mi-Sung Kim, et al. Mol Cell Biol. 2008 June;28(11):3600-3609.</div><p class="links"><a class="dblinks" href="/pubmed/18378694" ref="ordinalpos=4557&amp;ncbi_uid=1478963&amp;link_uid=18378694">Citation</a><a class="dblinks" href="/pmc/articles/PMC2423294/" ref="ordinalpos=4557&amp;ncbi_uid=1478963&amp;link_uid=PMC2423294">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4558.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3397482/figure/f2-ijms-13-06561/" ref="ordinalpos=4558&amp;ncbi_uid=4140515&amp;link_uid=PMC3397482" href="/pmc/articles/PMC3397482/"><img src="/pmc/articles/instance/3397482/bin/ijms-13-06561f2.gif" src-large="/pmc/articles/instance/3397482/bin/ijms-13-06561f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3397482/" ref="ordinalpos=4558&amp;ncbi_uid=4140515&amp;link_uid=PMC3397482" image-link="/pmc/articles/PMC3397482/figure/f2-ijms-13-06561/" class="imagepopup">Figure 2.  From: State of the Art in Silico Tools for the Study of <span class="highlight" style="background-color:">Signaling</span> Pathways in Cancer. </a></p><div class="supp"><p class="details"><div>A map of the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (as an example of an important target in cancer therapy) obtained from KEGG is shown [41]. The gene expression studies are used as an independent predictive method for the prognosis. In cancer genomics studies, tremendous effort has been devoted to <span class="highlight" style="background-color:">pathway</span>-based analysis. <span class="highlight" style="background-color:">Pathway</span> analysis is a promising tool to identify the mechanisms that underlie disease, the adaptive physiological compensatory responses and new avenues for investigation. Different pathways have different biological functions. Thus, studying each <span class="highlight" style="background-color:">pathway</span> separately is reasonable. Among the many pathways, only a few have been shown to have predictive power for the development of cancer. In this sense, KEGG could be a useful tool to identify genetic signatures [52].</div></p></div><div class="aux"><div class="resc">Vanessa Medina Villaamil, et al. Int J Mol Sci. 2012;13(6):6561-6581.</div><p class="links"><a class="dblinks" href="/pubmed/22837650" ref="ordinalpos=4558&amp;ncbi_uid=4140515&amp;link_uid=22837650">Citation</a><a class="dblinks" href="/pmc/articles/PMC3397482/" ref="ordinalpos=4558&amp;ncbi_uid=4140515&amp;link_uid=PMC3397482">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4559.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4324081/figure/F1/" ref="ordinalpos=4559&amp;ncbi_uid=7219023&amp;link_uid=PMC4324081" href="/pmc/articles/PMC4324081/"><img src="/pmc/articles/instance/4324081/bin/fonc-04-00389-g001.gif" src-large="/pmc/articles/instance/4324081/bin/fonc-04-00389-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4324081/" ref="ordinalpos=4559&amp;ncbi_uid=7219023&amp;link_uid=PMC4324081" image-link="/pmc/articles/PMC4324081/figure/F1/" class="imagepopup">Figure 1.  From: The Secret Role of microRNAs in Cancer Stem Cell Development and Potential Therapy: A Notch-<span class="highlight" style="background-color:">Pathway</span> Approach. </a></p><div class="supp"><p class="details"><div><b>Examples of miRNA re-expression therapy using miR-34, miR-200, and miR-199 family, in glioblastomas, pancreatic, and medulloblastomas cancer cells, all affecting the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Marianna Prokopi, et al. Front Oncol. 2014;4:389.</div><p class="links"><a class="dblinks" href="/pubmed/25717438" ref="ordinalpos=4559&amp;ncbi_uid=7219023&amp;link_uid=25717438">Citation</a><a class="dblinks" href="/pmc/articles/PMC4324081/" ref="ordinalpos=4559&amp;ncbi_uid=7219023&amp;link_uid=PMC4324081">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4560.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3603300/figure/fig5/" ref="ordinalpos=4560&amp;ncbi_uid=4740249&amp;link_uid=PMC3603300" href="/pmc/articles/PMC3603300/"><img src="/pmc/articles/instance/3603300/bin/MI2013-720689.005.gif" src-large="/pmc/articles/instance/3603300/bin/MI2013-720689.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3603300/" ref="ordinalpos=4560&amp;ncbi_uid=4740249&amp;link_uid=PMC3603300" image-link="/pmc/articles/PMC3603300/figure/fig5/" class="imagepopup">Figure 5.  From: Regulatory Role of GSK-3β on NF-κB, Nitric Oxide, and TNF-α in Group A Streptococcal Infection. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of group A streptococcus-induced GSK-3<i>β</i> activation, which leads to increased NF-<i>κ</i>B nuclear translocation and upregulation of TNF-<i>α</i> and NO production in macrophages.</div></p></div><div class="aux"><div class="resc">Yu-Tzu Chang, et al. Mediators Inflamm. 2013;2013:720689.</div><p class="links"><a class="dblinks" href="/pubmed/23533310" ref="ordinalpos=4560&amp;ncbi_uid=4740249&amp;link_uid=23533310">Citation</a><a class="dblinks" href="/pmc/articles/PMC3603300/" ref="ordinalpos=4560&amp;ncbi_uid=4740249&amp;link_uid=PMC3603300">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4561.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3705163/figure/fig8/" ref="ordinalpos=4561&amp;ncbi_uid=5084672&amp;link_uid=PMC3705163" href="/pmc/articles/PMC3705163/"><img src="/pmc/articles/instance/3705163/bin/onc2012334f8.gif" src-large="/pmc/articles/instance/3705163/bin/onc2012334f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3705163/" ref="ordinalpos=4561&amp;ncbi_uid=5084672&amp;link_uid=PMC3705163" image-link="/pmc/articles/PMC3705163/figure/fig8/" class="imagepopup">Figure 8.  From: Mutant p53 gain-of-function induces epithelial-mesenchymal transition through modulation of the miR-130b-ZEB1 axis. </a></p><div class="supp"><p class="details"><div>Schematic model indicating proposed mechanisms by which mutant p53 GOF induces EMT. Mutant p53 GOF induces EMT, through direct transrepression of miR-130b, an inhibitor of ZEB1, and subsequent activation of ZEB1-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">P Dong, et al. Oncogene. 2013 July 4;32(27):3286-3295.</div><p class="links"><a class="dblinks" href="/pubmed/22847613" ref="ordinalpos=4561&amp;ncbi_uid=5084672&amp;link_uid=22847613">Citation</a><a class="dblinks" href="/pmc/articles/PMC3705163/" ref="ordinalpos=4561&amp;ncbi_uid=5084672&amp;link_uid=PMC3705163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4562.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3070284/figure/F1/" ref="ordinalpos=4562&amp;ncbi_uid=2874057&amp;link_uid=PMC3070284" href="/pmc/articles/PMC3070284/"><img src="/pmc/articles/instance/3070284/bin/gcr0011000070001.gif" src-large="/pmc/articles/instance/3070284/bin/gcr0011000070001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3070284/" ref="ordinalpos=4562&amp;ncbi_uid=2874057&amp;link_uid=PMC3070284" image-link="/pmc/articles/PMC3070284/figure/F1/" class="imagepopup">Figure 1. From: Molecularly Targeted Therapy for Metastatic Colon Cancer: Proven Treatments and Promising New Agents. </a></p><div class="supp"><p class="details"><div>The vascular endothelial growth factor (VEGF) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Majed El Zouhairi, et al. Gastrointest Cancer Res. 2011 Jan-Feb;4(1):15-21.</div><p class="links"><a class="dblinks" href="/pubmed/21464866" ref="ordinalpos=4562&amp;ncbi_uid=2874057&amp;link_uid=21464866">Citation</a><a class="dblinks" href="/pmc/articles/PMC3070284/" ref="ordinalpos=4562&amp;ncbi_uid=2874057&amp;link_uid=PMC3070284">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4563.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2779493/figure/pone-0008064-g001/" ref="ordinalpos=4563&amp;ncbi_uid=2138671&amp;link_uid=PMC2779493" href="/pmc/articles/PMC2779493/"><img src="/pmc/articles/instance/2779493/bin/pone.0008064.g001.gif" src-large="/pmc/articles/instance/2779493/bin/pone.0008064.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2779493/" ref="ordinalpos=4563&amp;ncbi_uid=2138671&amp;link_uid=PMC2779493" image-link="/pmc/articles/PMC2779493/figure/pone-0008064-g001/" class="imagepopup">Figure 1. Correlation coefficients of four gene pairs from the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in two different classes..  From: Identifying Gene Interaction Enrichment for Gene Expression Data. </a></p><div class="aux"><div class="resc">Jigang Zhang, et al. PLoS One. 2009;4(11):e8064.</div><p class="links"><a class="dblinks" href="/pubmed/19956614" ref="ordinalpos=4563&amp;ncbi_uid=2138671&amp;link_uid=19956614">Citation</a><a class="dblinks" href="/pmc/articles/PMC2779493/" ref="ordinalpos=4563&amp;ncbi_uid=2138671&amp;link_uid=PMC2779493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4564.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3035824/figure/F5/" ref="ordinalpos=4564&amp;ncbi_uid=2759818&amp;link_uid=PMC3035824" href="/pmc/articles/PMC3035824/"><img src="/pmc/articles/instance/3035824/bin/nihms266995f5.gif" src-large="/pmc/articles/instance/3035824/bin/nihms266995f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3035824/" ref="ordinalpos=4564&amp;ncbi_uid=2759818&amp;link_uid=PMC3035824" image-link="/pmc/articles/PMC3035824/figure/F5/" class="imagepopup">Figure 5.  From: Genetically selected Marchigian Sardinian alcohol-preferring (msP) rats: an animal model to study the neurobiology of alcoholism. </a></p><div class="supp"><p class="details"><div>Mapk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from KEGG. Red shaded genes are differentially expressed in Marchigian Sardinian alcohol-preferring (msP) compared with Wistars</div></p></div><div class="aux"><div class="resc">Roberto Ciccocioppo, et al. Addict Biol. ;11(3-4):339-355.</div><p class="links"><a class="dblinks" href="/pubmed/16961763" ref="ordinalpos=4564&amp;ncbi_uid=2759818&amp;link_uid=16961763">Citation</a><a class="dblinks" href="/pmc/articles/PMC3035824/" ref="ordinalpos=4564&amp;ncbi_uid=2759818&amp;link_uid=PMC3035824">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4565.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4058194/figure/fig6/" ref="ordinalpos=4565&amp;ncbi_uid=6305738&amp;link_uid=PMC4058194" href="/pmc/articles/PMC4058194/"><img src="/pmc/articles/instance/4058194/bin/BMRI2014-375980.006.gif" src-large="/pmc/articles/instance/4058194/bin/BMRI2014-375980.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4058194/" ref="ordinalpos=4565&amp;ncbi_uid=6305738&amp;link_uid=PMC4058194" image-link="/pmc/articles/PMC4058194/figure/fig6/" class="imagepopup">Figure 6.  From: Simulated Annealing Based Algorithm for Identifying Mutated Driver Pathways in Cancer. </a></p><div class="supp"><p class="details"><div>In mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, there is the triplet of genes codes for proteins (orange nodes), and the pair (ATM and TP53) corresponds to interacting proteins in the cell cycle <span class="highlight" style="background-color:">pathway</span> (light blue nodes). These two pathways are reported in Ding et al. [9].</div></p></div><div class="aux"><div class="resc">Hai-Tao Li, et al. Biomed Res Int. 2014;2014:375980.</div><p class="links"><a class="dblinks" href="/pubmed/24982873" ref="ordinalpos=4565&amp;ncbi_uid=6305738&amp;link_uid=24982873">Citation</a><a class="dblinks" href="/pmc/articles/PMC4058194/" ref="ordinalpos=4565&amp;ncbi_uid=6305738&amp;link_uid=PMC4058194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4566.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3817619/figure/F7/" ref="ordinalpos=4566&amp;ncbi_uid=5453210&amp;link_uid=PMC3817619" href="/pmc/articles/PMC3817619/"><img src="/pmc/articles/instance/3817619/bin/nihms-151053-f0008.gif" src-large="/pmc/articles/instance/3817619/bin/nihms-151053-f0008.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3817619/" ref="ordinalpos=4566&amp;ncbi_uid=5453210&amp;link_uid=PMC3817619" image-link="/pmc/articles/PMC3817619/figure/F7/" class="imagepopup">Figure 7. Extrinsic and Intrinsic Apoptotic Pathways.  From: Infarction Induced Myocardial Apoptosis and ARC Activation. </a></p><div class="supp"><p class="details"><div>The extrinsic <span class="highlight" style="background-color:">pathway</span> is blocked by ARC’s ability to bind heterotrophically to Fas and FADD and prevent assembly of the death-inducing <span class="highlight" style="background-color:">signaling</span> complex (DISC). Subsequently, activation of caspase-8 is inhibited and apoptosis is prevented. ARC blocks the intrinsic <span class="highlight" style="background-color:">pathway</span> with its ability to bind to BAX and block BAX’s conformational activation and translocation to the mitochondria.</div></p></div><div class="aux"><div class="resc">Daryoush Ekhterae, et al. J Surg Res. 2011 March;166(1):10.1016/j.jss.2009.05.002.</div><p class="links"><a class="dblinks" href="/pubmed/19815236" ref="ordinalpos=4566&amp;ncbi_uid=5453210&amp;link_uid=19815236">Citation</a><a class="dblinks" href="/pmc/articles/PMC3817619/" ref="ordinalpos=4566&amp;ncbi_uid=5453210&amp;link_uid=PMC3817619">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4567.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2793058/figure/F10/" ref="ordinalpos=4567&amp;ncbi_uid=2598324&amp;link_uid=PMC2793058" href="/pmc/articles/PMC2793058/"><img src="/pmc/articles/instance/2793058/bin/zh00120961140010.gif" src-large="/pmc/articles/instance/2793058/bin/zh00120961140010.jpg" alt="Fig. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2793058/" ref="ordinalpos=4567&amp;ncbi_uid=2598324&amp;link_uid=PMC2793058" image-link="/pmc/articles/PMC2793058/figure/F10/" class="imagepopup">Fig. 10. From: IFN? regulates retinal pigment epithelial fluid transport. </a></p><div class="supp"><p class="details"><div>Schematic diagram of IFNγ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in retinal pigment epithelium. Major plasma membrane transporters and channels mediate fluid absorption across the RPE (Ref. 1) after activation of the canonical JAK-STAT <span class="highlight" style="background-color:">pathway</span>. GAS, γ-interferon activation site.</div></p></div><div class="aux"><div class="resc">Rong Li, et al. Am J Physiol Cell Physiol. 2009 December;297(6):C1452-C1465.</div><p class="links"><a class="dblinks" href="/pubmed/19794146" ref="ordinalpos=4567&amp;ncbi_uid=2598324&amp;link_uid=19794146">Citation</a><a class="dblinks" href="/pmc/articles/PMC2793058/" ref="ordinalpos=4567&amp;ncbi_uid=2598324&amp;link_uid=PMC2793058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4568.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2649397/figure/fig1/" ref="ordinalpos=4568&amp;ncbi_uid=1727957&amp;link_uid=PMC2649397" href="/pmc/articles/PMC2649397/"><img src="/pmc/articles/instance/2649397/bin/pp1491207f01.gif" src-large="/pmc/articles/instance/2649397/bin/pp1491207f01.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2649397/" ref="ordinalpos=4568&amp;ncbi_uid=1727957&amp;link_uid=PMC2649397" image-link="/pmc/articles/PMC2649397/figure/fig1/" class="imagepopup">Figure 1. From: Mitogen-Activated Protein Kinase Cascades and Ethylene: <span class="highlight" style="background-color:">Signaling</span>, Biosynthesis, or Both?. </a></p><div class="supp"><p class="details"><div>Proposed models for the involvement of MAPK modules in the ethylene <span class="highlight" style="background-color:">signaling</span> (A) or the ethylene biosynthetic <span class="highlight" style="background-color:">pathway</span> (B). A, Ethylene is perceived by endoplasmic reticulum (ER)-localized ethylene receptors. In the absence of ethylene, ETR1 and ERS1 interact with and activate the Raf-like MAPKKK CTR1. CTR1 represses ethylene <span class="highlight" style="background-color:">signaling</span> by inhibiting the activity of the potential membrane (M)-bound Nramp ion metal transporter-like protein EIN2 and inactivating the MKK9-MPK3/6 module. Additionally, CTR1-dependent <span class="highlight" style="background-color:">signaling</span>, possibly through another MAPK cascade, leads to the phosphorylation of the EIN3 transcription factor on T592 and, thus, to the degradation of EIN3 via the 26S proteasome. After the binding of ethylene, the <span class="highlight" style="background-color:">signaling</span> through the receptor/CTR1 module is inactivated and downstream <span class="highlight" style="background-color:">signaling</span> events are initiated. After translocation of MKK9 from the cytosol (C) to the nucleus (N), the MKK9-MPK3/6 module phosphorylates EIN3 on T174, causing the stabilization of the protein and protecting it from 26S proteasome-mediated degradation. Moreover, EIN2 positively regulates the ethylene-dependent transcriptional responses mediated by EIN3 through an unknown mechanism. B, Diverse abiotic and biotic stresses induce the activation of MAPK modules involving MKK4/5/9 and MPK3/6. The activated MPK3 and MPK6 are capable of phosphorylating two isoforms of the ACC synthase, namely ACS2 and ACS6, at their C terminus. By this phosphorylation, ACS2/6 become stabilized and preserved from degradation via the 26S proteasome. As ACC synthases convert <i>S</i>-adenosyl-<span style="font-variant: small-caps" class="small-caps">l</span>-Met (SAM) to ACC, a precursor of ethylene, the stabilization of the two ACS2/6 isoforms leads to an enhanced ethylene production and activation of the ethylene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Achim Hahn, et al. Plant Physiol. 2009 March;149(3):1207-1210.</div><p class="links"><a class="dblinks" href="/pubmed/19109412" ref="ordinalpos=4568&amp;ncbi_uid=1727957&amp;link_uid=19109412">Citation</a><a class="dblinks" href="/pmc/articles/PMC2649397/" ref="ordinalpos=4568&amp;ncbi_uid=1727957&amp;link_uid=PMC2649397">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4569.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3076924/figure/F2/" ref="ordinalpos=4569&amp;ncbi_uid=3486211&amp;link_uid=PMC3076924" href="/pmc/articles/PMC3076924/"><img src="/pmc/articles/instance/3076924/bin/nihms273335f2.gif" src-large="/pmc/articles/instance/3076924/bin/nihms273335f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3076924/" ref="ordinalpos=4569&amp;ncbi_uid=3486211&amp;link_uid=PMC3076924" image-link="/pmc/articles/PMC3076924/figure/F2/" class="imagepopup">Fig. 2. <span class="highlight" style="background-color:">Signaling</span> cascades of the Sig-1R.  From: Targeting ligand-operated chaperone sigma-1 receptors in the treatment of neuropsychiatric disorders. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways explored in the last 5 years are summarized. <span class="highlight" style="background-color:">Signaling</span> cascades of the Sig-1R fall into three categories: 1) the rapid action involving physical protein-protein interaction or <span class="highlight" style="background-color:">signaling</span> molecules (e.g., chaperone activation, Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span>, regulation of kinase activity or protein phosphorylation), 2) transcriptional regulation of gene expression downstream of the Sig-1R <span class="highlight" style="background-color:">signaling</span> (e.g., ROS-induced NFκB activation, CREB-induced transcription), 3) stress- or ligand-induced upregulation of Sig-1Rs. Overexpression or knockdown studies highlighted that the Sig-1R protein level per se, even in the absence of exogenous ligands, affects several cell death/survival signalings. It is not fully clarified how Sig-1Rs translocate from ER to plasmalemmal ER subdomains or to plasma membrane to physically associate with plasma membrane proteins such as ion channels and dopamine D1 receptors (see text “The unique subcellular localization of Sig-1Rs”). Note: data summarized are obtained from various cell types and brain regions. They are depicted together for simplicity. (+), pathways/molecules potentiated by Sig-1R expression or Sig-1R activation by agonists; (−), pathways/molecules inhibited by Sig-1R expression or Sig-1R activation by agonists. (--&gt;), activatory <span class="highlight" style="background-color:">pathway</span>; (--I) inhibitory <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hayashi Teruo, et al. Expert Opin Ther Targets. ;15(5):557-577.</div><p class="links"><a class="dblinks" href="/pubmed/21375464" ref="ordinalpos=4569&amp;ncbi_uid=3486211&amp;link_uid=21375464">Citation</a><a class="dblinks" href="/pmc/articles/PMC3076924/" ref="ordinalpos=4569&amp;ncbi_uid=3486211&amp;link_uid=PMC3076924">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4570.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3409292/figure/F3/" ref="ordinalpos=4570&amp;ncbi_uid=5248051&amp;link_uid=PMC3409292" href="/pmc/articles/PMC3409292/"><img src="/pmc/articles/instance/3409292/bin/nihms390720f3.gif" src-large="/pmc/articles/instance/3409292/bin/nihms390720f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3409292/" ref="ordinalpos=4570&amp;ncbi_uid=5248051&amp;link_uid=PMC3409292" image-link="/pmc/articles/PMC3409292/figure/F3/" class="imagepopup">Figure 3.  From: Genetic and Cellular Mechanisms Regulating Anterior Foregut and Esophageal Development. </a></p><div class="supp"><p class="details"><div>Conversion of simple columnar to stratified squamous epithelium in the esophagus involving dynamic Bmp <span class="highlight" style="background-color:">signaling</span> activities. (A) Immunostained cross sections of E11.0 and E12.5 trachea and esophagus. Krt8 is expressed in the epithelium of both the esophagus and trachea after their formation from the foregut. Sox2 remains highly expressed in the epithelium of the dorsal, foregut-derived esophagus at E11.0 and E12.5. Note the conversion from single to multi-layered epithelium in the esophagus from E11.0 to E12.5, shown by co-immunostaining with p63 and Sox2 antibodies. (B) Schematic of the stratification and differentiation of esophageal epithelium through a two-stage Bmp <span class="highlight" style="background-color:">signaling</span> pattern. Stratification of the epithelium from E11.0 to E14.5 correlates with Noggin-mediated suppression of Bmp <span class="highlight" style="background-color:">signaling</span>. Ectopic Bmp activity in <i>Noggin</i> null and <i>Shh-Cre; Rosa26<sup>caBmpr1a</sup></i> mutants inhibits the stratification process. Differentiation of the top layers of epithelium at E14.5-P9.0 requires activation of the Bmp <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> while basal progenitor cells remain negative for Bmp <span class="highlight" style="background-color:">signaling</span>. Deletion of <i>Bmpr1a</i> in <i>Shh-Cre; Bmpr1a<sup>loxp/loxp</sup></i> mutants inhibits the differentiation of suprabasal cells. Scale bar: 50μm. eso, esophagus; tra, trachea.</div></p></div><div class="aux"><div class="resc">Ian J. Jacobs, et al. Dev Biol. ;369(1):54-64.</div><p class="links"><a class="dblinks" href="/pubmed/22750256" ref="ordinalpos=4570&amp;ncbi_uid=5248051&amp;link_uid=22750256">Citation</a><a class="dblinks" href="/pmc/articles/PMC3409292/" ref="ordinalpos=4570&amp;ncbi_uid=5248051&amp;link_uid=PMC3409292">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4571.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407051/figure/pone-0041914-g006/" ref="ordinalpos=4571&amp;ncbi_uid=4150101&amp;link_uid=PMC3407051" href="/pmc/articles/PMC3407051/"><img src="/pmc/articles/instance/3407051/bin/pone.0041914.g006.gif" src-large="/pmc/articles/instance/3407051/bin/pone.0041914.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407051/" ref="ordinalpos=4571&amp;ncbi_uid=4150101&amp;link_uid=PMC3407051" image-link="/pmc/articles/PMC3407051/figure/pone-0041914-g006/" class="imagepopup">Figure 6. Attenuated activation of the wnt/beta-catenin and ERK <span class="highlight" style="background-color:">signaling</span> pathways in APCmin mice lacking MCH..  From: Reduced Intestinal Tumorigenesis in APCmin Mice Lacking Melanin-Concentrating Hormone. </a></p><div class="supp"><p class="details"><div>Paraffin sections of intestinal adenomas from WT and MCH KO mice, both of the APCmin background, were stained for (A) beta-catenin, a downstream effector of the wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (B) c-myc, a target of beta-catenin; and (C) p-erk. Quantification of differences between the groups (percentage of positive cells per adenoma) is shown in graphs on the right of each picture. Results are expressed as mean±sem. ***p&lt;0.005; ****p&lt;0.001.</div></p></div><div class="aux"><div class="resc">Jutta M. Nagel, et al. PLoS One. 2012;7(7):e41914.</div><p class="links"><a class="dblinks" href="/pubmed/22848656" ref="ordinalpos=4571&amp;ncbi_uid=4150101&amp;link_uid=22848656">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407051/" ref="ordinalpos=4571&amp;ncbi_uid=4150101&amp;link_uid=PMC3407051">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4572.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC415794/figure/f7/" ref="ordinalpos=4572&amp;ncbi_uid=569210&amp;link_uid=PMC415794" href="/pmc/articles/PMC415794/"><img src="/pmc/articles/instance/415794/bin/zjv0110446840007.gif" src-large="/pmc/articles/instance/415794/bin/zjv0110446840007.jpg" alt="FIG.7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC415794/" ref="ordinalpos=4572&amp;ncbi_uid=569210&amp;link_uid=PMC415794" image-link="/pmc/articles/PMC415794/figure/f7/" class="imagepopup">FIG.7. From: Human Papillomavirus Type 16 E6 Amino Acid 83 Variants Enhance E6-Mediated MAPK <span class="highlight" style="background-color:">Signaling</span> and Differentially Regulate Tumorigenesis by Notch <span class="highlight" style="background-color:">Signaling</span> and Oncogenic Ras. </a></p><div class="supp"><p class="details"><div>Activation of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mouse tumors and rafts. Microphotographs of a tumor generated at 2 weeks by HaCaT cells expressing the shown gene combinations (A) or rafts generated from HaCaT cells stably expressing Neo, E6, and S82DL83W (B) were immunostained by using antibodies against phospho-MAPK1/2 and phospho-Akt. Tumors in panel A were also stained with anti-HPV-16/18 E6 antibody to confirm the expression of E6. Microphotographs were taken under 20× magnification.</div></p></div><div class="aux"><div class="resc">Oishee Chakrabarti, et al. J Virol. 2004 June;78(11):5934-5945.</div><p class="links"><a class="dblinks" href="/pubmed/15140991" ref="ordinalpos=4572&amp;ncbi_uid=569210&amp;link_uid=15140991">Citation</a><a class="dblinks" href="/pmc/articles/PMC415794/" ref="ordinalpos=4572&amp;ncbi_uid=569210&amp;link_uid=PMC415794">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4573.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2533523/figure/Fig4/" ref="ordinalpos=4573&amp;ncbi_uid=1602117&amp;link_uid=PMC2533523" href="/pmc/articles/PMC2533523/"><img src="/pmc/articles/instance/2533523/bin/11693_2007_9007_Fig4_HTML.gif" src-large="/pmc/articles/instance/2533523/bin/11693_2007_9007_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2533523/" ref="ordinalpos=4573&amp;ncbi_uid=1602117&amp;link_uid=PMC2533523" image-link="/pmc/articles/PMC2533523/figure/Fig4/" class="imagepopup">Fig. 4.  From: Specificity of MAPK <span class="highlight" style="background-color:">signaling</span> towards FLO11 expression is established by crosstalk from cAMP <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><i>FLO11 expression in haploids and diploids</i>. Fractional dose response is shown with respect to upstream activator Ras2GTP. Curve ‘a’ represents the <i>FLO11</i> expression in diploids, where the total concentration of Ste12 is 5–10 fold less than haploids and cross-activation of MAPK <span class="highlight" style="background-color:">pathway</span> by cAMP <span class="highlight" style="background-color:">pathway</span> is not observed. Curve ‘b’ shows the case when the cross-activation is included in diploids. Curve ‘c’ represents the case for haploids, which shows complete expression</div></p></div><div class="aux"><div class="resc">P. K. Vinod, et al. Syst Synth Biol. 2007 April;1(2):99-108.</div><p class="links"><a class="dblinks" href="/pubmed/19003439" ref="ordinalpos=4573&amp;ncbi_uid=1602117&amp;link_uid=19003439">Citation</a><a class="dblinks" href="/pmc/articles/PMC2533523/" ref="ordinalpos=4573&amp;ncbi_uid=1602117&amp;link_uid=PMC2533523">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4574.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2907250/figure/F2/" ref="ordinalpos=4574&amp;ncbi_uid=2368728&amp;link_uid=PMC2907250" href="/pmc/articles/PMC2907250/"><img src="/pmc/articles/instance/2907250/bin/nihms-214831-f0002.gif" src-large="/pmc/articles/instance/2907250/bin/nihms-214831-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2907250/" ref="ordinalpos=4574&amp;ncbi_uid=2368728&amp;link_uid=PMC2907250" image-link="/pmc/articles/PMC2907250/figure/F2/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Pathway</span> analysis of breast cancer genome wide association study highlights three pathways and one canonical <span class="highlight" style="background-color:">signaling</span> cascade. </a></p><div class="supp"><p class="details"><div>Clustering analysis of <span class="highlight" style="background-color:">pathway</span> overlap. Pair-wise <span class="highlight" style="background-color:">pathway</span> overlap scores are depicted in a matrix color-coded in grayscale. (A) Clusters of highly significant related pathways are delineated by black triangles. (B) Details of the pathways included in cluster C1.</div></p></div><div class="aux"><div class="resc">Idan Menashe, et al. Cancer Res. ;70(11):4453-4459.</div><p class="links"><a class="dblinks" href="/pubmed/20460509" ref="ordinalpos=4574&amp;ncbi_uid=2368728&amp;link_uid=20460509">Citation</a><a class="dblinks" href="/pmc/articles/PMC2907250/" ref="ordinalpos=4574&amp;ncbi_uid=2368728&amp;link_uid=PMC2907250">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4575.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4076062/figure/F3/" ref="ordinalpos=4575&amp;ncbi_uid=6307866&amp;link_uid=PMC4076062" href="/pmc/articles/PMC4076062/"><img src="/pmc/articles/instance/4076062/bin/1471-2407-14-370-3.gif" src-large="/pmc/articles/instance/4076062/bin/1471-2407-14-370-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4076062/" ref="ordinalpos=4575&amp;ncbi_uid=6307866&amp;link_uid=PMC4076062" image-link="/pmc/articles/PMC4076062/figure/F3/" class="imagepopup">Figure 3.  From: Valproic acid sensitizes pancreatic cancer cells to natural killer cell-mediated lysis by upregulating MICA and MICB via the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>VPA activates the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in pancreatic cancer cells.</b> Pancreatic cancer cells were incubated with or without 1 mM VPA for 24 h. <b>(A)</b> Quantitative real-time RT-PCR analysis of <i>HER2, HER3, ATM</i>, <i>ATR</i>, and <i>PI3KCA</i> mRNA expression in PANC-1, MIA PaCa-2 and BxPC-3 cells. Data are mean ± SD of a single experiment performed in triplicate, all results were reproducible in three independent experiments. ns <i>P</i> &gt; =0.05; * <i>P</i> &lt; 0.05; ** <i>P</i> &lt; 0.01. <b>(B, C)</b> Western blotting analysis of the expression and phosphorylation of HER3 and Akt (ser 473).</div></p></div><div class="aux"><div class="resc">Pengfei Shi, et al. BMC Cancer. 2014;14:370-370.</div><p class="links"><a class="dblinks" href="/pubmed/24885711" ref="ordinalpos=4575&amp;ncbi_uid=6307866&amp;link_uid=24885711">Citation</a><a class="dblinks" href="/pmc/articles/PMC4076062/" ref="ordinalpos=4575&amp;ncbi_uid=6307866&amp;link_uid=PMC4076062">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4576.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3712288/figure/F5/" ref="ordinalpos=4576&amp;ncbi_uid=5353575&amp;link_uid=PMC3712288" href="/pmc/articles/PMC3712288/"><img src="/pmc/articles/instance/3712288/bin/nihms488125f5.gif" src-large="/pmc/articles/instance/3712288/bin/nihms488125f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3712288/" ref="ordinalpos=4576&amp;ncbi_uid=5353575&amp;link_uid=PMC3712288" image-link="/pmc/articles/PMC3712288/figure/F5/" class="imagepopup">Figure 5.  From: Targeting the IKK?/mTOR/VEGF <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> as a Potential Therapeutic Strategy for Obesity-Related Breast Cancer. </a></p><div class="supp"><p class="details"><div>TNFα-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promotes mammary tumor development. A, spontaneous mammary tumor development in MMTV-neu mice fed with either a HFD or normal diet. Each group contained 10 mice. The incidence of spontaneous tumor growth in MMTV-neu mice was based on tumors larger than 5 mm in diameter in the mammary gland. B, representative images of immunofluorescence staining of tumor tissues with the indicated antibodies. The relative fluorescence intensity from the images is shown on the right. The data represent the mean ± SD, (10 mice per group). ND, normal chow diet. *, <i>P</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Chun-Te Chen, et al. Mol Cancer Ther. ;11(10):2212-2221.</div><p class="links"><a class="dblinks" href="/pubmed/22826466" ref="ordinalpos=4576&amp;ncbi_uid=5353575&amp;link_uid=22826466">Citation</a><a class="dblinks" href="/pmc/articles/PMC3712288/" ref="ordinalpos=4576&amp;ncbi_uid=5353575&amp;link_uid=PMC3712288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4577.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2988058/figure/F3/" ref="ordinalpos=4577&amp;ncbi_uid=2582268&amp;link_uid=PMC2988058" href="/pmc/articles/PMC2988058/"><img src="/pmc/articles/instance/2988058/bin/1471-2172-11-53-3.gif" src-large="/pmc/articles/instance/2988058/bin/1471-2172-11-53-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2988058/" ref="ordinalpos=4577&amp;ncbi_uid=2582268&amp;link_uid=PMC2988058" image-link="/pmc/articles/PMC2988058/figure/F3/" class="imagepopup">Figure 3.  From: PAR1- and PAR2-induced innate immune markers are negatively regulated by PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oral keratinocytes. </a></p><div class="supp"><p class="details"><div><b>Expression of PAR1- and PAR2-induced innate immunity is enhanced when PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is blocked</b>. HOKs were incubated with inhibitors for PI3K (Wortmannin) at 25-400 nM (a-b) or for Akt (Akt inhibitor IV) at 0.5-2.5 uM (c-d) for 1 h and subsequently stimulated with thrombin (10 U/ml) or trypsin (10 nM) for 6 h. Cells were harvested and expression of the selected markers was assessed by QRT-PCR. Data from three separate donors set up in duplicates were normalized to GAPDH. Data are given as means of normalized samples to unstimulated control ± SEM. (*<i>p </i>&lt; 0.05 compared to the same condition without inhibitors).</div></p></div><div class="aux"><div class="resc">Maryam G Rohani, et al. BMC Immunol. 2010;11:53-53.</div><p class="links"><a class="dblinks" href="/pubmed/21029417" ref="ordinalpos=4577&amp;ncbi_uid=2582268&amp;link_uid=21029417">Citation</a><a class="dblinks" href="/pmc/articles/PMC2988058/" ref="ordinalpos=4577&amp;ncbi_uid=2582268&amp;link_uid=PMC2988058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4578.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2932971/figure/F1/" ref="ordinalpos=4578&amp;ncbi_uid=2800436&amp;link_uid=PMC2932971" href="/pmc/articles/PMC2932971/"><img src="/pmc/articles/instance/2932971/bin/1893fig1.gif" src-large="/pmc/articles/instance/2932971/bin/1893fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2932971/" ref="ordinalpos=4578&amp;ncbi_uid=2800436&amp;link_uid=PMC2932971" image-link="/pmc/articles/PMC2932971/figure/F1/" class="imagepopup">Figure 1. From: Functional microdomains in bacterial membranes. </a></p><div class="supp"><p class="details"><div> <i>yisP</i> affects the <span class="highlight" style="background-color:">pathway</span> to biofilm formation. (<i>A</i>) Schematic representation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to biofilm formation in <i>B. subtilis</i>. The <span class="highlight" style="background-color:">pathway</span> is triggered by activation of the master regulator Spo0A via phosphorylation by KinC. Dashed lines represent indirect activation. (<i>B</i>) Putative metabolic <span class="highlight" style="background-color:">pathway</span> to the formation of distinct polysioprenoids in <i>B. subtilis</i>. Enzymes discussed in the text are written in red, next to the reaction they catalyze. Dashed lines represent unknown steps. (<i>C</i>) Pellicle formation assay to test biofilm formation in different strains of <i>B. subtilis</i>. Positive and negative controls are represented by the wild-type strain (WT) and the matrix-deficient mutant (Δ<i>eps</i> Δ<i>yqxM</i>) (Branda et al. 2004, 2006).</div></p></div><div class="aux"><div class="resc">Daniel López, et al. Genes Dev. 2010 September 1;24(17):1893-1902.</div><p class="links"><a class="dblinks" href="/pubmed/20713508" ref="ordinalpos=4578&amp;ncbi_uid=2800436&amp;link_uid=20713508">Citation</a><a class="dblinks" href="/pmc/articles/PMC2932971/" ref="ordinalpos=4578&amp;ncbi_uid=2800436&amp;link_uid=PMC2932971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4579.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2820725/figure/F4/" ref="ordinalpos=4579&amp;ncbi_uid=2768182&amp;link_uid=PMC2820725" href="/pmc/articles/PMC2820725/"><img src="/pmc/articles/instance/2820725/bin/nihms171073f4.gif" src-large="/pmc/articles/instance/2820725/bin/nihms171073f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2820725/" ref="ordinalpos=4579&amp;ncbi_uid=2768182&amp;link_uid=PMC2820725" image-link="/pmc/articles/PMC2820725/figure/F4/" class="imagepopup">Figure 4.  From: Redox Remodeling as an Immunoregulatory Strategy. </a></p><div class="supp"><p class="details"><div>Redox <span class="highlight" style="background-color:">signaling</span> responses in DCs and T cells during T cell activation. (A) Redox <span class="highlight" style="background-color:">signaling</span> in DCs. The TCR-antigen•MHC complex interaction and the co-stimulatory signal results in an immediate early pro-oxidant response in DCs with ROS production e.g. by lipoxygenase and NADPH oxidase. Low levels of ROS act as <span class="highlight" style="background-color:">signaling</span> molecules to inhibit protein tyrosine phosphatases (PTPs) and activate protein kinases. ROS also activates the NF-κB <span class="highlight" style="background-color:">pathway</span>, which stimulates the expression of γ-glutamylcysteine ligase, thus increasing GSH synthesis. GSH activates the AP1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and initiates an antioxidant response. We postulate that system x<sub>c</sub><sup>-</sup>-dependent cystine uptake, GSH export and degradation into extracellular cysteine are stimulated as part of this response. (B) TCR <span class="highlight" style="background-color:">signaling</span> in T cells. Stimulation of T cells by DCs via the TCR results in phosphorylation and activation of ZAP-70 by leukocyte-specific protein tyrosine kinase (LCK). ZAP-70 directly phosphorylates the adaptor protein LAT and causes the assembly of multiprotein <span class="highlight" style="background-color:">signaling</span> complexes. Recruitment of the growth factor receptor bound protein 2 (GRB2) and phospholipase C γ1 (PLCγ1) to LAT leads to activation of downstream Ras and calcium <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Zhonghua Yan, et al. Biochemistry. ;49(6):1059.</div><p class="links"><a class="dblinks" href="/pubmed/20070126" ref="ordinalpos=4579&amp;ncbi_uid=2768182&amp;link_uid=20070126">Citation</a><a class="dblinks" href="/pmc/articles/PMC2820725/" ref="ordinalpos=4579&amp;ncbi_uid=2768182&amp;link_uid=PMC2820725">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4580.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3320973/figure/F8/" ref="ordinalpos=4580&amp;ncbi_uid=4782116&amp;link_uid=PMC3320973" href="/pmc/articles/PMC3320973/"><img src="/pmc/articles/instance/3320973/bin/zbc0171204250008.gif" src-large="/pmc/articles/instance/3320973/bin/zbc0171204250008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3320973/" ref="ordinalpos=4580&amp;ncbi_uid=4782116&amp;link_uid=PMC3320973" image-link="/pmc/articles/PMC3320973/figure/F8/" class="imagepopup">FIGURE 8. From: Roles of Heparan Sulfate Sulfation in Dentinogenesis. </a></p><div class="supp"><p class="details"><div><b>A model of Wnt <span class="highlight" style="background-color:">signaling</span> regulation induced by cell surface HSPG desulfation during dentinogenesis.</b> Desulfation by Sulf reduces the affinity of cell surface HS for Wnt10a as a means of facilitating Wnt10a binding to its receptor and hence potentiating the Wnt canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which consequently up-regulates <i>Dspp</i> expression in odontoblasts.</div></p></div><div class="aux"><div class="resc">Satoru Hayano, et al. J Biol Chem. 2012 April 6;287(15):12217-12229.</div><p class="links"><a class="dblinks" href="/pubmed/22351753" ref="ordinalpos=4580&amp;ncbi_uid=4782116&amp;link_uid=22351753">Citation</a><a class="dblinks" href="/pmc/articles/PMC3320973/" ref="ordinalpos=4580&amp;ncbi_uid=4782116&amp;link_uid=PMC3320973">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4581.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2684190/figure/F9-8031/" ref="ordinalpos=4581&amp;ncbi_uid=1804274&amp;link_uid=PMC2684190" href="/pmc/articles/PMC2684190/"><img src="/pmc/articles/instance/2684190/bin/zjh0060980310009.gif" src-large="/pmc/articles/instance/2684190/bin/zjh0060980310009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2684190/" ref="ordinalpos=4581&amp;ncbi_uid=1804274&amp;link_uid=PMC2684190" image-link="/pmc/articles/PMC2684190/figure/F9-8031/" class="imagepopup">Figure 9.  From: Platelet-Derived Hyaluronidase 2 Cleaves Hyaluronan into Fragments that Trigger Monocyte-Mediated Production of Proinflammatory Cytokines. </a></p><div class="supp"><p class="details"><div>Diagram of platelet HYAL2 generating <span class="highlight" style="background-color:">signaling</span> fragments of HA, and their interactions in the catabolic cascades associated with wound healing and inflammation. A diagram of interactions that occur between the <span class="highlight" style="background-color:">signaling</span> fragments of HA generated by the HYAL2 of platelets and their involvement in the earliest stages of wound healing and the subsequent inflammatory phase. Aberrations in this <span class="highlight" style="background-color:">pathway</span> can lead to chronicity of inflammation and possibly to autoimmune disorders.</div></p></div><div class="aux"><div class="resc">Carol de la Motte, et al. Am J Pathol. 2009 June;174(6):2254-2264.</div><p class="links"><a class="dblinks" href="/pubmed/19443707" ref="ordinalpos=4581&amp;ncbi_uid=1804274&amp;link_uid=19443707">Citation</a><a class="dblinks" href="/pmc/articles/PMC2684190/" ref="ordinalpos=4581&amp;ncbi_uid=1804274&amp;link_uid=PMC2684190">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4582.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2782733/figure/F2/" ref="ordinalpos=4582&amp;ncbi_uid=2165189&amp;link_uid=PMC2782733" href="/pmc/articles/PMC2782733/"><img src="/pmc/articles/instance/2782733/bin/nihms103661f2.gif" src-large="/pmc/articles/instance/2782733/bin/nihms103661f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2782733/" ref="ordinalpos=4582&amp;ncbi_uid=2165189&amp;link_uid=PMC2782733" image-link="/pmc/articles/PMC2782733/figure/F2/" class="imagepopup">Fig. 2.  From: Pharmacotherapy for Adults with Tumors of the Central Nervous System. </a></p><div class="supp"><p class="details"><div>EGFR <span class="highlight" style="background-color:">signaling</span> through the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. Erlotinib and gelfitinib interfere with autophosphorylation of EGFR. In tumors that exhibit upregulation of IGFR, <span class="highlight" style="background-color:">signaling</span> of that receptor through Ras/Raf can bypass the block and generate sufficient activated PI3K to signal through Akt; these tumors are therefore resistant to erlotinib and gelfitinib. Adjunctive treatment with farnesyl transferase inhibitors or nelfinavir can overcome this resistance.</div></p></div><div class="aux"><div class="resc">Nina F. Schor. Pharmacol Ther. ;121(3):253-264.</div><p class="links"><a class="dblinks" href="/pubmed/19091301" ref="ordinalpos=4582&amp;ncbi_uid=2165189&amp;link_uid=19091301">Citation</a><a class="dblinks" href="/pmc/articles/PMC2782733/" ref="ordinalpos=4582&amp;ncbi_uid=2165189&amp;link_uid=PMC2782733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4583.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3504715/figure/fig8/" ref="ordinalpos=4583&amp;ncbi_uid=5543492&amp;link_uid=PMC3504715" href="/pmc/articles/PMC3504715/"><img src="/pmc/articles/instance/3504715/bin/cdd201291f8.gif" src-large="/pmc/articles/instance/3504715/bin/cdd201291f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3504715/" ref="ordinalpos=4583&amp;ncbi_uid=5543492&amp;link_uid=PMC3504715" image-link="/pmc/articles/PMC3504715/figure/fig8/" class="imagepopup">Figure 8.  From: CITED2 functions as a molecular switch of cytokine-induced proliferation and quiescence. </a></p><div class="supp"><p class="details"><div>Cytokine-induced CITED2/MYC/E2F3/p21<sup>CIP1</sup> <span class="highlight" style="background-color:">signaling</span> in lung cancer. (<b>a</b>) Model of CITED2/MYC/E2F3/p21<sup>CIP1</sup> <span class="highlight" style="background-color:">signaling</span>. CITED2 expression is induced by TGF-<i>α</i>/EGFR through MYC. Elevated CITED2 levels further recruit p300 to the <i>E2F3</i> promoter to enhance MYC-mediated transactivation, leading to increased G1/S cell cycle transition (left). TGF-<i>β</i> downregulates CITED2, and loss of CITED2 prevents MYC from interacting with HDAC1, causing activation of the <i>p21</i><sup><i>CIP1</i></sup> promoter and cellular quiescence (right). (<b>b</b>) Model of cytokine-induced homeostasis and neoplastic development of lung epithelial cells. Homeostasis of normal lung epithelial cells is intricately balanced by TGF-<i>α</i>/EGFR proliferation and TGF-<i>β</i>/TGFBR quiescence <span class="highlight" style="background-color:">signaling</span>, both of which are fine-tuned by CITED2 expression. Activation of MYC/CITED2/E2F3/p21<sup>CIP1</sup> <span class="highlight" style="background-color:">pathway</span> by abnormal cytokine <span class="highlight" style="background-color:">signaling</span> such as EGFR mutation/amplification-activation and TGFBRII silencing contributes to neoplastic development of lung epithelial cells</div></p></div><div class="aux"><div class="resc">Y-T Chou, et al. Cell Death Differ. 2012 December;19(12):2015-2028.</div><p class="links"><a class="dblinks" href="/pubmed/22814619" ref="ordinalpos=4583&amp;ncbi_uid=5543492&amp;link_uid=22814619">Citation</a><a class="dblinks" href="/pmc/articles/PMC3504715/" ref="ordinalpos=4583&amp;ncbi_uid=5543492&amp;link_uid=PMC3504715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4584.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2948618/figure/F3/" ref="ordinalpos=4584&amp;ncbi_uid=3281038&amp;link_uid=PMC2948618" href="/pmc/articles/PMC2948618/"><img src="/pmc/articles/instance/2948618/bin/nihms230881f3.gif" src-large="/pmc/articles/instance/2948618/bin/nihms230881f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2948618/" ref="ordinalpos=4584&amp;ncbi_uid=3281038&amp;link_uid=PMC2948618" image-link="/pmc/articles/PMC2948618/figure/F3/" class="imagepopup">Figure 3. DDR <span class="highlight" style="background-color:">Signaling</span> Should influence Chemotherapy Regiments.  From: 3rd US-EU Workshop Systems level understanding of DNA damage responses. </a></p><div class="supp"><p class="details"><div>(A) In response to DNA damage Eukaryotic cells activate a complex <span class="highlight" style="background-color:">signaling</span> network to arrest cell cycle, initiate repair, or induce apoptosis. In addition to ATM/Chk2 and ATR/Chk1 a third DNA damage <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by p38 MAPK-activation of MAPKAP kinase 2 (MK2). Adapted and reprinted with permission from <i>Mol Cell. 2005 Jan 7;17(1):37-48</i>. (B) Detailed knowledge of DNA damage induced <span class="highlight" style="background-color:">signaling</span> pathways has great potential for personalized cancer treatments. Using Xengraft models Yaffe and colleagues have demonstrated that ATM inhibition can result in either pronounced chemoresistance or chemosensitization to DNA damaging agents. They have demonstrated how DNA damage <span class="highlight" style="background-color:">signaling</span> networks can be rewired and showed how <span class="highlight" style="background-color:">pathway</span> focused diagnostics can be used to successfully predict therapeutic outcomes in cancer treatment. Special thanks to H. Christian Reinhardt for creating the figure; adapted and reprinted with permission from <i>Cell Cycle. 2009 Oct 1;8(19):3112-9</i>).</div></p></div><div class="aux"><div class="resc">Miriam Sander, et al. Mutat Res. ;692(1-2):53-60.</div><p class="links"><a class="dblinks" href="/pubmed/20727903" ref="ordinalpos=4584&amp;ncbi_uid=3281038&amp;link_uid=20727903">Citation</a><a class="dblinks" href="/pmc/articles/PMC2948618/" ref="ordinalpos=4584&amp;ncbi_uid=3281038&amp;link_uid=PMC2948618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4585.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3696624/figure/F5/" ref="ordinalpos=4585&amp;ncbi_uid=5057198&amp;link_uid=PMC3696624" href="/pmc/articles/PMC3696624/"><img src="/pmc/articles/instance/3696624/bin/lar-29-113-g005.gif" src-large="/pmc/articles/instance/3696624/bin/lar-29-113-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3696624/" ref="ordinalpos=4585&amp;ncbi_uid=5057198&amp;link_uid=PMC3696624" image-link="/pmc/articles/PMC3696624/figure/F5/" class="imagepopup">Figure 5.  From: In vitro and in vivo study of effects of fermented soybean product (chungkookjang) on NGF secretion ability and NGF receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Effects of TG-CKJ and SH-CKJ treatment on down-stream NGF receptor p75<sup>NTR</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via Western blot analysis. Total cell lysates were prepared from PC12 cells treated with CM of vehicle, TG-CKJ, and SH-CKJ. Thirty micrograms of protein per sample was immunoblotted with antibody for each protein (A). The intensity of each band was determined using an imaging densitometer, and the relative level of each protein was calculated based on the intensity of actin (B). Three samples were assayed in triplicate via Western blot analysis. Values of the data represent the means±SD.</div></p></div><div class="aux"><div class="resc">Young-Ju Lee, et al. Lab Anim Res. 2013 June;29(2):113-126.</div><p class="links"><a class="dblinks" href="/pubmed/23825484" ref="ordinalpos=4585&amp;ncbi_uid=5057198&amp;link_uid=23825484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3696624/" ref="ordinalpos=4585&amp;ncbi_uid=5057198&amp;link_uid=PMC3696624">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4586.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2716072/figure/F4/" ref="ordinalpos=4586&amp;ncbi_uid=1977496&amp;link_uid=PMC2716072" href="/pmc/articles/PMC2716072/"><img src="/pmc/articles/instance/2716072/bin/nihms104454f4.gif" src-large="/pmc/articles/instance/2716072/bin/nihms104454f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2716072/" ref="ordinalpos=4586&amp;ncbi_uid=1977496&amp;link_uid=PMC2716072" image-link="/pmc/articles/PMC2716072/figure/F4/" class="imagepopup">Fig. 4.  From: Small Tumor Antigen of Polyomaviruses: Role in Viral Life Cycle and Cell Transformation. </a></p><div class="supp"><p class="details"><div>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> is stimulated by a number of extracellular stimuli, including nutrients, growth factors, stress and viral infection. <span class="highlight" style="background-color:">Signaling</span> generated by these stimuli is relayed to downstream effectors by the mTORC1 complex. The mTORC1 can be inhibited by rapamycin treatment. The upstream activators of the mTORC2 complex however are currently unknown and mTORC2 is insensitive to rapamycin treatment. Upon activation, the mTORC1 complex phosphorylates 4E-BPI and S6K resulting in their dissociation from the cap-binding protein, eIF4E, and initiation factor 3(eIF3)respectively. This signals to impact downstream events including ribosome biogenesis, translation and cell growth. The effector molecules of mTOR <span class="highlight" style="background-color:">pathway</span> are targeted and modified by many viruses to preferentially translate their own mRNA as explained in the text. The Sm t-Ag of polyomaviruses also targets this <span class="highlight" style="background-color:">pathway</span> at different levels as indicated.</div></p></div><div class="aux"><div class="resc">Kamel Khalili, et al. J Cell Physiol. ;215(2):309-319.</div><p class="links"><a class="dblinks" href="/pubmed/18022798" ref="ordinalpos=4586&amp;ncbi_uid=1977496&amp;link_uid=18022798">Citation</a><a class="dblinks" href="/pmc/articles/PMC2716072/" ref="ordinalpos=4586&amp;ncbi_uid=1977496&amp;link_uid=PMC2716072">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4587.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3234335/figure/F1/" ref="ordinalpos=4587&amp;ncbi_uid=4653982&amp;link_uid=PMC3234335" href="/pmc/articles/PMC3234335/"><img src="/pmc/articles/instance/3234335/bin/nihms318852f1.gif" src-large="/pmc/articles/instance/3234335/bin/nihms318852f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3234335/" ref="ordinalpos=4587&amp;ncbi_uid=4653982&amp;link_uid=PMC3234335" image-link="/pmc/articles/PMC3234335/figure/F1/" class="imagepopup">Figure 1. Model for the stimulation of hepcidin expression in hepatocytes by inflammation and by iron levels in the body.  From: Hereditary Hemochromatosis and Transferrin Receptor 2. </a></p><div class="supp"><p class="details"><div>Elevation of interleukin 6 (IL6) during inflammation stimulates hepcidin transcription through binding to the IL6-receptor (IL6R)-gp130 complex, which stimulates <span class="highlight" style="background-color:">signaling</span> through the phosphorylation of JAK leading to the phosphorylation of STAT3. The phosphorylated STAT3 dimer binds to two identified BMP responsive elements in the promoter region of hepcidin stimulating transcription. Elevation of iron levels in the body leads to the upregulation of BMP6 and Tf-saturation. BMP6 signals through binding to hemojuvelin (HJV), BMP receptors and neogenin (Neo) and signals through the SMAD4/SMAD1/5/8 <span class="highlight" style="background-color:">pathway</span> to stimulate hepcidin expression. Two BMP responsive elements in the promoter region of hepcidin have been identified. Less is known about the Tf-TfR2-HFE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (see text). Mutations in HFE result in a blunted pSMAD response. The p38 MAPK-ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is proposed to mediate this response.</div></p></div><div class="aux"><div class="resc">Juxing Chen, et al. Biochim Biophys Acta. ;1820(3):256-263.</div><p class="links"><a class="dblinks" href="/pubmed/21864651" ref="ordinalpos=4587&amp;ncbi_uid=4653982&amp;link_uid=21864651">Citation</a><a class="dblinks" href="/pmc/articles/PMC3234335/" ref="ordinalpos=4587&amp;ncbi_uid=4653982&amp;link_uid=PMC3234335">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4588.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2632088/figure/f1/" ref="ordinalpos=4588&amp;ncbi_uid=1682854&amp;link_uid=PMC2632088" href="/pmc/articles/PMC2632088/"><img src="/pmc/articles/instance/2632088/bin/zjb0030984240001.gif" src-large="/pmc/articles/instance/2632088/bin/zjb0030984240001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2632088/" ref="ordinalpos=4588&amp;ncbi_uid=1682854&amp;link_uid=PMC2632088" image-link="/pmc/articles/PMC2632088/figure/f1/" class="imagepopup">FIG. 1. From: Cyclic Di-GMP (c-Di-GMP) Goes into Host Cells--c-Di-GMP <span class="highlight" style="background-color:">Signaling</span> in the Obligate Intracellular Pathogen Anaplasma phagocytophilum  . </a></p><div class="supp"><p class="details"><div>Principal components of the c-di-GMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. C-di-GMP is synthesized by GGDEF domain proteins, which are di-guanylate cyclases. Degradation of c-di-GMP occurs through c-di-GMP-specific PDEs. These enzymatic activities are present in the unrelated EAL and HD-GYP domains and degrade c-di-GMP to pGpG and GMP, respectively. Subsequently, c-di-GMP binds to downstream proteins or RNA receptors (17, 20). c-di-GMP thereby stimulates sessility and inhibits motility and virulence properties required for acute infection.</div></p></div><div class="aux"><div class="resc">Ute Römling. J Bacteriol. 2009 February;191(3):683-686.</div><p class="links"><a class="dblinks" href="/pubmed/19047357" ref="ordinalpos=4588&amp;ncbi_uid=1682854&amp;link_uid=19047357">Citation</a><a class="dblinks" href="/pmc/articles/PMC2632088/" ref="ordinalpos=4588&amp;ncbi_uid=1682854&amp;link_uid=PMC2632088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4589.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2098746/figure/F2/" ref="ordinalpos=4589&amp;ncbi_uid=1113607&amp;link_uid=PMC2098746" href="/pmc/articles/PMC2098746/"><img src="/pmc/articles/instance/2098746/bin/nihms31785f2.gif" src-large="/pmc/articles/instance/2098746/bin/nihms31785f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2098746/" ref="ordinalpos=4589&amp;ncbi_uid=1113607&amp;link_uid=PMC2098746" image-link="/pmc/articles/PMC2098746/figure/F2/" class="imagepopup">Figure 2. ErbB <span class="highlight" style="background-color:">signaling</span> regulates activation of PI3K and Erk MAPK pathways in Xenopus DMZ explants.  From: PI3K and Erk MAPK mediate ErbB <span class="highlight" style="background-color:">signaling</span> in Xenopus gastrulation. </a></p><div class="supp"><p class="details"><div>DMZ explants from embryos overexpressing ErbB4 (2ng mRNA/embryo), ErbB4-MO (20ng MO/embryo), p110caax (1ng mRNA/embryo), Δp85 (1ng mRNA/embryo) or MEK* (1ng mRNA/embryo) were examined at gastrula stages for activation of PI3K (pAkt) or Erk MAPK (dpErk) pathways. Overexpression of ErbB4 enhanced, while depletion of ErbB4 reduced, the levels of both pAkt and dpErk. p110caax and MEK* stimulated pAkt and dpErk, respectively; but they also weakly increased phosphorylation of the other <span class="highlight" style="background-color:">pathway</span> component, suggesting a crosstalk between the two pathways.</div></p></div><div class="aux"><div class="resc">Shuyi Nie, et al. Mech Dev. ;124(9-10):657-667.</div><p class="links"><a class="dblinks" href="/pubmed/17716876" ref="ordinalpos=4589&amp;ncbi_uid=1113607&amp;link_uid=17716876">Citation</a><a class="dblinks" href="/pmc/articles/PMC2098746/" ref="ordinalpos=4589&amp;ncbi_uid=1113607&amp;link_uid=PMC2098746">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4590.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3830120/figure/F005/" ref="ordinalpos=4590&amp;ncbi_uid=5510482&amp;link_uid=PMC3830120" href="/pmc/articles/PMC3830120/"><img src="/pmc/articles/instance/3830120/bin/ooomb4-21-216-g005.gif" src-large="/pmc/articles/instance/3830120/bin/ooomb4-21-216-g005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3830120/" ref="ordinalpos=4590&amp;ncbi_uid=5510482&amp;link_uid=PMC3830120" image-link="/pmc/articles/PMC3830120/figure/F005/" class="imagepopup">Fig. 5.Effect of aromadendrin on LPS-induced MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RAW 264.7 macrophage cells. RAW 264.7 cells were stimulated with 200 ng/ml LPS in the absence or presence of aromadendrin. Western blot analysis was then performed to evaluate the activation of MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. LPSinduced phosphorylation of JNK was significantly attenuated with aromadendrin treatment, but phosphorylation of ERK and p38 was not decreased with aromadendrin treatment, suggesting that JNK <span class="highlight" style="background-color:">signaling</span> might play a key role in the LPS-induced overactivation of RAW 264.7 cells. β-Actin was used as an internal control. Images are representative of three independent experiments that shows reproducible results. ARD stands for aromadendrin..  From: Aromadendrin Inhibits Lipopolysaccharide-Induced Nuclear Translocation of NF-?B and Phosphorylation of JNK in RAW 264.7 Macrophage Cells. </a></p><div class="aux"><div class="resc">Jae-Won Lee, et al. Biomol Ther (Seoul). 2013 May 30;21(3):216-221.</div><p class="links"><a class="dblinks" href="/pubmed/24265867" ref="ordinalpos=4590&amp;ncbi_uid=5510482&amp;link_uid=24265867">Citation</a><a class="dblinks" href="/pmc/articles/PMC3830120/" ref="ordinalpos=4590&amp;ncbi_uid=5510482&amp;link_uid=PMC3830120">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4591.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2666387/figure/F7/" ref="ordinalpos=4591&amp;ncbi_uid=1820334&amp;link_uid=PMC2666387" href="/pmc/articles/PMC2666387/"><img src="/pmc/articles/instance/2666387/bin/nihms69896f7.gif" src-large="/pmc/articles/instance/2666387/bin/nihms69896f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2666387/" ref="ordinalpos=4591&amp;ncbi_uid=1820334&amp;link_uid=PMC2666387" image-link="/pmc/articles/PMC2666387/figure/F7/" class="imagepopup">Figure 7.  From: Src-Family Tyrosine Kinases in Wound- and Ligand-Induced Epidermal Growth Factor Receptor Activation in Human Corneal Epithelial Cells. </a></p><div class="supp"><p class="details"><div>Proposed Src involvement in HCEC wound healing. This scheme summarizes the <span class="highlight" style="background-color:">pathway</span> from wounding to HCEC wound closure and includes EGFR transactivation and its downstream <span class="highlight" style="background-color:">signaling</span>. Wounding activates Src, which leads to the release of HB-EGF. The soluble form of HB-EGF then binds EGFR, which transduces the signals into the intracellular <span class="highlight" style="background-color:">signaling</span> pathways, including the activation of Src. After EGFR activation, HCEC migration, proliferation, and wound healing are induced through PI3K, ERK, and other intracellular <span class="highlight" style="background-color:">signaling</span> pathways. Activation of PI3K, but not ERK, <span class="highlight" style="background-color:">pathway</span> in HCECs after EGFR (trans)activation requires Src activation. <i>Light solid arrows</i>: novel findings reported in the present paper. <i>Dash lines</i>: remain to be determined.</div></p></div><div class="aux"><div class="resc">Ke-Ping Xu, et al. Invest Ophthalmol Vis Sci. ;47(7):2832-2839.</div><p class="links"><a class="dblinks" href="/pubmed/16799022" ref="ordinalpos=4591&amp;ncbi_uid=1820334&amp;link_uid=16799022">Citation</a><a class="dblinks" href="/pmc/articles/PMC2666387/" ref="ordinalpos=4591&amp;ncbi_uid=1820334&amp;link_uid=PMC2666387">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4592.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3477499/figure/F6/" ref="ordinalpos=4592&amp;ncbi_uid=5953261&amp;link_uid=PMC3477499" href="/pmc/articles/PMC3477499/"><img src="/pmc/articles/instance/3477499/bin/nihms388492f6.gif" src-large="/pmc/articles/instance/3477499/bin/nihms388492f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3477499/" ref="ordinalpos=4592&amp;ncbi_uid=5953261&amp;link_uid=PMC3477499" image-link="/pmc/articles/PMC3477499/figure/F6/" class="imagepopup">Figure 6.  From: Cardiac O-GlcNAcylation blunts autophagic <span class="highlight" style="background-color:">signaling</span> in the diabetic heart. </a></p><div class="supp"><p class="details"><div>The role of the hexosamine biosynthesis <span class="highlight" style="background-color:">pathway</span> (HBP) in blunting autophagic <span class="highlight" style="background-color:">signaling</span>. Representative western blots showing Beclin-1 and LC3 protein in cardiomyocytes from non-diabetic (Con) and type 2 diabetic (<i>db/db</i>) under basal conditions and in response to glucose deprivation (A) and in Con cells subjected to glucose deprivation in the presence of the autophagy inhibitor 3-methyladenine (3-MA) or glucosamine (GlcN) (B). Inhibition of the HBP by 6-diazo-5-oxo-L-norleucine (DON) in glucose deprived <i>db/db</i> cardiomyocytes restores Beclin-1 and LC3 <span class="highlight" style="background-color:">signaling</span> (C). Calsequestrin (Csq) is an internal protein loading control.</div></p></div><div class="aux"><div class="resc">Susan A. Marsh, et al. Life Sci. 2013 March 28;92(11):648-656.</div><p class="links"><a class="dblinks" href="/pubmed/22728715" ref="ordinalpos=4592&amp;ncbi_uid=5953261&amp;link_uid=22728715">Citation</a><a class="dblinks" href="/pmc/articles/PMC3477499/" ref="ordinalpos=4592&amp;ncbi_uid=5953261&amp;link_uid=PMC3477499">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4593.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2737668/figure/F3/" ref="ordinalpos=4593&amp;ncbi_uid=2062603&amp;link_uid=PMC2737668" href="/pmc/articles/PMC2737668/"><img src="/pmc/articles/instance/2737668/bin/nihms127896f3.gif" src-large="/pmc/articles/instance/2737668/bin/nihms127896f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2737668/" ref="ordinalpos=4593&amp;ncbi_uid=2062603&amp;link_uid=PMC2737668" image-link="/pmc/articles/PMC2737668/figure/F3/" class="imagepopup">Figure 3.  From: Suberoyl bishydroxamic acid activates Notch1 <span class="highlight" style="background-color:">signaling</span> and suppresses tumor progression in an animal model of medullary thyroid carcinoma. </a></p><div class="supp"><p class="details"><div>SBHA activates the Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and decreases levels of chromogranin A (CgA). Total tumor tissue extracts were isolated from SBHA-treated or DMSO-treated mice and analyzed by Western blotting for expression of Notch1 intracellular domain (NICD), achaete-scute complex-like 1 (ASCL1) and CgA. SBHA treatment induced NICD protein, which led to a concomitant decrease in ASCL1, a downstream target of Notch1 <span class="highlight" style="background-color:">signaling</span>, as well as NE tumor marker CgA. G3PDH was used to confirm equal protein loading. The quantification of the bands was generated by comparison with G3PDH.</div></p></div><div class="aux"><div class="resc">Li Ning, et al. Ann Surg Oncol. ;15(9):2600-2605.</div><p class="links"><a class="dblinks" href="/pubmed/18563491" ref="ordinalpos=4593&amp;ncbi_uid=2062603&amp;link_uid=18563491">Citation</a><a class="dblinks" href="/pmc/articles/PMC2737668/" ref="ordinalpos=4593&amp;ncbi_uid=2062603&amp;link_uid=PMC2737668">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4594.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC514949/figure/f9/" ref="ordinalpos=4594&amp;ncbi_uid=483991&amp;link_uid=PMC514949" href="/pmc/articles/PMC514949/"><img src="/pmc/articles/instance/514949/bin/7600289f9.gif" src-large="/pmc/articles/instance/514949/bin/7600289f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC514949/" ref="ordinalpos=4594&amp;ncbi_uid=483991&amp;link_uid=PMC514949" image-link="/pmc/articles/PMC514949/figure/f9/" class="imagepopup">Figure 9.  From: VEGF-null cells require PDGFR ? <span class="highlight" style="background-color:">signaling</span>-mediated stromal fibroblast recruitment for tumorigenesis. </a></p><div class="supp"><p class="details"><div>Multiple pathways involved in tumorigenesis by VEGF-null tumor cells. VEGF-null tumor cells secrete PDGF AA (and perhaps CC) to recruit host stromal fibroblasts through PDGFR α <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The recruited host stromal fibroblasts in turn provide VEGF to simulate endothelial cell (EC) and to initiate angiogenesis. The stability of tumor vessels is dependent on the interaction between pericytes (PC) and endothelial cells (EC) via the PDGFR β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jianying Dong, et al. EMBO J. 2004 July 21;23(14):2800-2810.</div><p class="links"><a class="dblinks" href="/pubmed/15229650" ref="ordinalpos=4594&amp;ncbi_uid=483991&amp;link_uid=15229650">Citation</a><a class="dblinks" href="/pmc/articles/PMC514949/" ref="ordinalpos=4594&amp;ncbi_uid=483991&amp;link_uid=PMC514949">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4595.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3212439/figure/F3/" ref="ordinalpos=4595&amp;ncbi_uid=3514035&amp;link_uid=PMC3212439" href="/pmc/articles/PMC3212439/"><img src="/pmc/articles/instance/3212439/bin/nihms-296577-f0003.gif" src-large="/pmc/articles/instance/3212439/bin/nihms-296577-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3212439/" ref="ordinalpos=4595&amp;ncbi_uid=3514035&amp;link_uid=PMC3212439" image-link="/pmc/articles/PMC3212439/figure/F3/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> proteins present in LPS-treated thioglycollate-elicited macrophages derived from C57BL/6 WT and LMP7−/−/MECL-1−/− double knockout mice.  From: The Immunoproteasomes Regulate Lps-Induced Trif/Tram <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Murine Macrophages. </a></p><div class="supp"><p class="details"><div>Macrophages were cultured <i>in vitro</i> and treated with LPS (200 ng/ml) for 0, 0.5, 1, 2, 4 and 8 h. Cell lysates were prepared and the cytoplasmic <span class="highlight" style="background-color:">signaling</span> proteins were analyzed by Western Blotting. This experiment was repeated three times.</div></p></div><div class="aux"><div class="resc">Julia Reis, et al. Cell Biochem Biophys. ;60(1-2):119-126.</div><p class="links"><a class="dblinks" href="/pubmed/21455681" ref="ordinalpos=4595&amp;ncbi_uid=3514035&amp;link_uid=21455681">Citation</a><a class="dblinks" href="/pmc/articles/PMC3212439/" ref="ordinalpos=4595&amp;ncbi_uid=3514035&amp;link_uid=PMC3212439">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4596.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3903560/figure/F7/" ref="ordinalpos=4596&amp;ncbi_uid=5743623&amp;link_uid=PMC3903560" href="/pmc/articles/PMC3903560/"><img src="/pmc/articles/instance/3903560/bin/1479-5876-12-11-7.gif" src-large="/pmc/articles/instance/3903560/bin/1479-5876-12-11-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3903560/" ref="ordinalpos=4596&amp;ncbi_uid=5743623&amp;link_uid=PMC3903560" image-link="/pmc/articles/PMC3903560/figure/F7/" class="imagepopup">Figure 7.  From: Prediction of individual immune responsiveness to a candidate vaccine by a systems vaccinology approach. </a></p><div class="supp"><p class="details"><div><b>Ingenuity pathways analysis.</b> IFNγ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> uniquely activated in PBMCs of subject BE after 6 days stimulation with IGKV3-20.</div></p></div><div class="aux"><div class="resc">Annacarmen Petrizzo, et al. J Transl Med. 2014;12:11-11.</div><p class="links"><a class="dblinks" href="/pubmed/24428943" ref="ordinalpos=4596&amp;ncbi_uid=5743623&amp;link_uid=24428943">Citation</a><a class="dblinks" href="/pmc/articles/PMC3903560/" ref="ordinalpos=4596&amp;ncbi_uid=5743623&amp;link_uid=PMC3903560">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4597.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3628712/figure/pone-0059733-g011/" ref="ordinalpos=4597&amp;ncbi_uid=4820983&amp;link_uid=PMC3628712" href="/pmc/articles/PMC3628712/"><img src="/pmc/articles/instance/3628712/bin/pone.0059733.g011.gif" src-large="/pmc/articles/instance/3628712/bin/pone.0059733.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3628712/" ref="ordinalpos=4597&amp;ncbi_uid=4820983&amp;link_uid=PMC3628712" image-link="/pmc/articles/PMC3628712/figure/pone-0059733-g011/" class="imagepopup">Figure 11. Schematic representation of the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of adenosine-induced enhancement of barrier function in VVEC..  From: Adenosine A1 Receptors Promote Vasa Vasorum Endothelial Cell Barrier Integrity via Gi and Akt-Dependent Actin Cytoskeleton Remodeling. </a></p><div class="aux"><div class="resc">Nagavedi Siddaramappa Umapathy, et al. PLoS One. 2013; 2013;8(4):e59733.</div><p class="links"><a class="dblinks" href="/pubmed/23613714" ref="ordinalpos=4597&amp;ncbi_uid=4820983&amp;link_uid=23613714">Citation</a><a class="dblinks" href="/pmc/articles/PMC3628712/" ref="ordinalpos=4597&amp;ncbi_uid=4820983&amp;link_uid=PMC3628712">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4598.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507315/figure/fig3/" ref="ordinalpos=4598&amp;ncbi_uid=4461474&amp;link_uid=PMC3507315" href="/pmc/articles/PMC3507315/"><img src="/pmc/articles/instance/3507315/bin/MI2012-563709.003.gif" src-large="/pmc/articles/instance/3507315/bin/MI2012-563709.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507315/" ref="ordinalpos=4598&amp;ncbi_uid=4461474&amp;link_uid=PMC3507315" image-link="/pmc/articles/PMC3507315/figure/fig3/" class="imagepopup">Figure 3.  From: Clinical Application of Development of Nonantibiotic Macrolides That Correct Inflammation-Driven Immune Dysfunction in Inflammatory Skin Diseases. </a></p><div class="supp"><p class="details"><div>Fc epsilon receptor I <span class="highlight" style="background-color:">signaling</span> in mast cell <span class="highlight" style="background-color:">pathway</span> extracted from BioCarta database [9].</div></p></div><div class="aux"><div class="resc">Carmen Rodriguez-Cerdeira, et al. Mediators Inflamm. 2012;2012:563709.</div><p class="links"><a class="dblinks" href="/pubmed/23258954" ref="ordinalpos=4598&amp;ncbi_uid=4461474&amp;link_uid=23258954">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507315/" ref="ordinalpos=4598&amp;ncbi_uid=4461474&amp;link_uid=PMC3507315">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4599.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3359943/figure/fig1/" ref="ordinalpos=4599&amp;ncbi_uid=4758305&amp;link_uid=PMC3359943" href="/pmc/articles/PMC3359943/"><img src="/pmc/articles/instance/3359943/bin/AJRCMB464417fig1.gif" src-large="/pmc/articles/instance/3359943/bin/AJRCMB464417fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3359943/" ref="ordinalpos=4599&amp;ncbi_uid=4758305&amp;link_uid=PMC3359943" image-link="/pmc/articles/PMC3359943/figure/fig1/" class="imagepopup">Figure 1. From: Hypercapnia: A Nonpermissive Environment for the Lung. </a></p><div class="supp"><p class="details"><div>Schematic representation of elevated CO<sub>2</sub>–induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to rapid down-regulation of Na,K-ATPase in the alveolar epithelial plasma membrane and impaired alveolar fluid clearance.</div></p></div><div class="aux"><div class="resc">István Vadász, et al. Am J Respir Cell Mol Biol. 2012 April;46(4):417-421.</div><p class="links"><a class="dblinks" href="/pubmed/22246860" ref="ordinalpos=4599&amp;ncbi_uid=4758305&amp;link_uid=22246860">Citation</a><a class="dblinks" href="/pmc/articles/PMC3359943/" ref="ordinalpos=4599&amp;ncbi_uid=4758305&amp;link_uid=PMC3359943">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4600.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3093193/figure/F2/" ref="ordinalpos=4600&amp;ncbi_uid=3568202&amp;link_uid=PMC3093193" href="/pmc/articles/PMC3093193/"><img src="/pmc/articles/instance/3093193/bin/nihms289177f2.gif" src-large="/pmc/articles/instance/3093193/bin/nihms289177f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3093193/" ref="ordinalpos=4600&amp;ncbi_uid=3568202&amp;link_uid=PMC3093193" image-link="/pmc/articles/PMC3093193/figure/F2/" class="imagepopup">Figure 2. Selected factors and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating melanocyte function.  From: Update on the regulation of mammalian melanocyte function and skin pigmentation. </a></p><div class="supp"><p class="details"><div>Various factors that regulate melanocyte function in the skin are shown. Antagonists of receptor binding are shown in red.<br />Adapted from [20].</div></p></div><div class="aux"><div class="resc">Taisuke Kondo, et al. Expert Rev Dermatol. ;6(1):97-108.</div><p class="links"><a class="dblinks" href="/pubmed/21572549" ref="ordinalpos=4600&amp;ncbi_uid=3568202&amp;link_uid=21572549">Citation</a><a class="dblinks" href="/pmc/articles/PMC3093193/" ref="ordinalpos=4600&amp;ncbi_uid=3568202&amp;link_uid=PMC3093193">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="45" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="46" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="47" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:47:49-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB200E54312FF100000000009EB1E9;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB200E54312FF100000000009EB1E9" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>