
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="42" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A3662543128210000000000C4D034" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="41" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="42" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="43" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="42" /></div>
            <div><h2 class="result_count">Results: 4101 to 4200 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4101.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3062096/figure/fig1/" ref="ordinalpos=4101&amp;ncbi_uid=2843853&amp;link_uid=PMC3062096" href="/pmc/articles/PMC3062096/"><img src="/pmc/articles/instance/3062096/bin/PRI2011-219309.001.gif" src-large="/pmc/articles/instance/3062096/bin/PRI2011-219309.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3062096/" ref="ordinalpos=4101&amp;ncbi_uid=2843853&amp;link_uid=PMC3062096" image-link="/pmc/articles/PMC3062096/figure/fig1/" class="imagepopup">Figure 1.  From: Impact of KRAS Mutations on Management of Colorectal Carcinoma. </a></p><div class="supp"><p class="details"><div>EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [4]. (Reprinted with permission from  American Society of Clinical Oncology 2008.  All rights reserved.)</div></p></div><div class="aux"><div class="resc">Kevin M. Sullivan, et al. Patholog Res Int. 2011;2011:219309.</div><p class="links"><a class="dblinks" href="/pubmed/21437184" ref="ordinalpos=4101&amp;ncbi_uid=2843853&amp;link_uid=21437184">Citation</a><a class="dblinks" href="/pmc/articles/PMC3062096/" ref="ordinalpos=4101&amp;ncbi_uid=2843853&amp;link_uid=PMC3062096">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4102.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3774745/figure/pone-0073959-g006/" ref="ordinalpos=4102&amp;ncbi_uid=5329966&amp;link_uid=PMC3774745" href="/pmc/articles/PMC3774745/"><img src="/pmc/articles/instance/3774745/bin/pone.0073959.g006.gif" src-large="/pmc/articles/instance/3774745/bin/pone.0073959.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3774745/" ref="ordinalpos=4102&amp;ncbi_uid=5329966&amp;link_uid=PMC3774745" image-link="/pmc/articles/PMC3774745/figure/pone-0073959-g006/" class="imagepopup">Figure 6. <span class="highlight" style="background-color:">Pathway</span> analysis for genes frequently affected by copy number alterations in both adenomas and carcinomas..  From: Single Nucleotide Polymorphism Array Profiling of Adrenocortical Tumors - Evidence for an Adenoma Carcinoma Sequence?. </a></p><div class="supp"><p class="details"><div>A) List of the most significantly altered pathways according to the GeneGo analysis (Meta Core Analytical suite) including all the genes (n=696 annotated genes) with copy number alterations observed in at least 4 samples (recurrent CNA) in both adenomas and carcinomas. The P values are expresses as logarithmic scale. B) Graphical representation of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> map (gene ontology enrichment, <i>P=2.913</i><sup><i>e-4</i></sup>).<br />C) Graphical representation of the WNT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (gene ontology enrichment, <i>P=3.256</i><sup><i>e-3</i></sup>).<br />Genes with gains are underlined in red and genes with losses in blue. Detailed legend is available at <span style="text-decoration: underline" class="underline">http://pathwaymaps.com/pdf/MC_legend.pdf</span>.</div></p></div><div class="aux"><div class="resc">Cristina L. Ronchi, et al. PLoS One. 2013;8(9):e73959.</div><p class="links"><a class="dblinks" href="/pubmed/24066089" ref="ordinalpos=4102&amp;ncbi_uid=5329966&amp;link_uid=24066089">Citation</a><a class="dblinks" href="/pmc/articles/PMC3774745/" ref="ordinalpos=4102&amp;ncbi_uid=5329966&amp;link_uid=PMC3774745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4103.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3212955/figure/F7/" ref="ordinalpos=4103&amp;ncbi_uid=3352051&amp;link_uid=PMC3212955" href="/pmc/articles/PMC3212955/"><img src="/pmc/articles/instance/3212955/bin/1744-8069-7-49-7.gif" src-large="/pmc/articles/instance/3212955/bin/1744-8069-7-49-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3212955/" ref="ordinalpos=4103&amp;ncbi_uid=3352051&amp;link_uid=PMC3212955" image-link="/pmc/articles/PMC3212955/figure/F7/" class="imagepopup">Figure 7.  From: Resveratrol inhibits Cdk5 activity through regulation of p35 expression. </a></p><div class="supp"><p class="details"><div><b>Proposed resveratrol-mediated regulation of p35 expression and Cdk5 activity</b>. Resveratrol (RESV) reduces Cdk5 activity through inhibition of p35 expression. The inhibitory effects of resveratrol on p35 expression could be brought about at several levels, inhibition of ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> could affect Egr-1 expression, or activation of the p38 MAPK, JNK and NF-B <span class="highlight" style="background-color:">signaling</span> pathways could downregulate p35 promoter activity.</div></p></div><div class="aux"><div class="resc">Elias Utreras, et al. Mol Pain. 2011;7:49-49.</div><p class="links"><a class="dblinks" href="/pubmed/21736731" ref="ordinalpos=4103&amp;ncbi_uid=3352051&amp;link_uid=21736731">Citation</a><a class="dblinks" href="/pmc/articles/PMC3212955/" ref="ordinalpos=4103&amp;ncbi_uid=3352051&amp;link_uid=PMC3212955">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4104.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3026990/figure/fig5/" ref="ordinalpos=4104&amp;ncbi_uid=2762481&amp;link_uid=PMC3026990" href="/pmc/articles/PMC3026990/"><img src="/pmc/articles/instance/3026990/bin/PPAR2010-571783.005.gif" src-large="/pmc/articles/instance/3026990/bin/PPAR2010-571783.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3026990/" ref="ordinalpos=4104&amp;ncbi_uid=2762481&amp;link_uid=PMC3026990" image-link="/pmc/articles/PMC3026990/figure/fig5/" class="imagepopup">Figure 5.  From: Induction of Metastatic Gastric Cancer by Peroxisome Proliferator-Activated Receptor Activation. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> networks associated with PPAR<i></i> and inflammatory and invasive gene expression. Common <span class="highlight" style="background-color:">signaling</span> pathways were analyzed for six genes with the greatest changes in expression in gastric tumors (highlighted in <i>blue</i>) versus forestomach using Ariadne <span class="highlight" style="background-color:">Pathway</span> Studio 7.1.</div></p></div><div class="aux"><div class="resc">Claire B. Pollock, et al. PPAR Res. 2010;2010:571783.</div><p class="links"><a class="dblinks" href="/pubmed/21318167" ref="ordinalpos=4104&amp;ncbi_uid=2762481&amp;link_uid=21318167">Citation</a><a class="dblinks" href="/pmc/articles/PMC3026990/" ref="ordinalpos=4104&amp;ncbi_uid=2762481&amp;link_uid=PMC3026990">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4105.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3427885/figure/fig04/" ref="ordinalpos=4105&amp;ncbi_uid=4196745&amp;link_uid=PMC3427885" href="/pmc/articles/PMC3427885/"><img src="/pmc/articles/instance/3427885/bin/hep0052-0443-f4.gif" src-large="/pmc/articles/instance/3427885/bin/hep0052-0443-f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3427885/" ref="ordinalpos=4105&amp;ncbi_uid=4196745&amp;link_uid=PMC3427885" image-link="/pmc/articles/PMC3427885/figure/fig04/" class="imagepopup">Fig. 4.  From: Transcriptome Sequencing, Microarray, and Proteomic Analyses Reveal Cellular and Metabolic Impact of Hepatitis C Virus Infection In Vitro. </a></p><div class="supp"><p class="details"><div>Impact of HCV infection on ILK <span class="highlight" style="background-color:">signaling</span>. A schematic of the ILK <span class="highlight" style="background-color:">signaling</span> canonical <span class="highlight" style="background-color:">pathway</span> produced using IPA and overlaid with genes identified by RNA-Seq and Microarray analysis. Genes highlighted in red are up-regulated, those in green are down-regulated. Gray genes are not detected to be altered.</div></p></div><div class="aux"><div class="resc">Stephen D Woodhouse, et al. Hepatology. 2010 August;52(2):443-453.</div><p class="links"><a class="dblinks" href="/pubmed/20683944" ref="ordinalpos=4105&amp;ncbi_uid=4196745&amp;link_uid=20683944">Citation</a><a class="dblinks" href="/pmc/articles/PMC3427885/" ref="ordinalpos=4105&amp;ncbi_uid=4196745&amp;link_uid=PMC3427885">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4106.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3155249/figure/F5/" ref="ordinalpos=4106&amp;ncbi_uid=3156932&amp;link_uid=PMC3155249" href="/pmc/articles/PMC3155249/"><img src="/pmc/articles/instance/3155249/bin/nihms200111f5.gif" src-large="/pmc/articles/instance/3155249/bin/nihms200111f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3155249/" ref="ordinalpos=4106&amp;ncbi_uid=3156932&amp;link_uid=PMC3155249" image-link="/pmc/articles/PMC3155249/figure/F5/" class="imagepopup">Figure 5. Glucocorticoid (GC)/Insulin <span class="highlight" style="background-color:">signaling</span> strongly correlates with MSI in the CA region.  From: Aging-Related Gene Expression in Hippocampus Proper compared to Dentate Gyrus is Selectively Associated with Metabolic Syndrome Variables in Rhesus Monkeys. </a></p><div class="supp"><p class="details"><div>Six aging related genes associated with GC/stress and/or insulin/IGF1 <span class="highlight" style="background-color:">signaling</span> were used to construct a GC/insulin ratio that increases with elevated GC function and/or decreased insulin <span class="highlight" style="background-color:">signaling</span>. <b>Upper:</b> Expression values for the 6 genes were averaged by subject (note: Grif, a downregulated repressor in the glucocorticoid <span class="highlight" style="background-color:">pathway</span> and Akt3, a downregulated effector of the insulin <span class="highlight" style="background-color:">pathway</span> (*) were inverted prior to averaging). The resulting GC/insulin ratio values correlated strongly (r = 0.9; Pearson's test) with the MSI. <b>Lower:</b> names and aliases of populating genes, as well as protein names and basic functions are listed.</div></p></div><div class="aux"><div class="resc">E. M. Blalock, et al. J Neurosci. ;30(17):6058-6071.</div><p class="links"><a class="dblinks" href="/pubmed/20427664" ref="ordinalpos=4106&amp;ncbi_uid=3156932&amp;link_uid=20427664">Citation</a><a class="dblinks" href="/pmc/articles/PMC3155249/" ref="ordinalpos=4106&amp;ncbi_uid=3156932&amp;link_uid=PMC3155249">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4107.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2662567/figure/F2/" ref="ordinalpos=4107&amp;ncbi_uid=1763998&amp;link_uid=PMC2662567" href="/pmc/articles/PMC2662567/"><img src="/pmc/articles/instance/2662567/bin/JCI37934.f2.gif" src-large="/pmc/articles/instance/2662567/bin/JCI37934.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2662567/" ref="ordinalpos=4107&amp;ncbi_uid=1763998&amp;link_uid=PMC2662567" image-link="/pmc/articles/PMC2662567/figure/F2/" class="imagepopup">Figure 2. The MET RTK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and genes implicated in ASD risk..  From: The genetic and neurobiologic compass points toward common <span class="highlight" style="background-color:">signaling</span>                dysfunctions in autism spectrum disorders. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> of MET and other RTKs occurs via the PI3K or ERK1/2                   pathways. Rare mutations and CNVs (which are both designated by ) or                   associated common alleles (which are designated by *) have been identified in                   individuals with ASD in seven genes encoding proteins involved in these <span class="highlight" style="background-color:">signaling</span>                   pathways. Of note, an association between common MET variants and ASD has been                   reported for five independent family cohorts. PLAUR and SERPINE1 associations with                   ASD have been determined in single, large family cohorts (&gt;600 families).                   Ras disruption in Smith-Lemli-Opitz syndrome is due to alterations in cholesterol                   biosynthesis (which is designated by ). Also depicted are other                   proteins that interact with the MET <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, such as semaphorins,                   plexins, and other RTKs. MET can signal via the PI3K and the ERK <span class="highlight" style="background-color:">pathway</span>. RTKs,                   including MET, are involved in key neurodevelopmental processes, including axon                   guidance, synapse formation, and plasticity. Convergence of many different genetic                   etiologies suggests that risk via ERK/PI3K <span class="highlight" style="background-color:">signaling</span> may be common in ASD. Risk,                   severity of the pathophysiology (i.e., intellectual disability), and disorder                   heterogeneity may relate to differences in genetic and epigenetic points of entry                   to the pathways. Thus, the impact due to genetic risk, via regulators of ligand                   availability or RTKs such as MET, may be less severe than the more severe clinical                   impact (i.e., intellectual disability) from disruption downstream along the                   intracellular <span class="highlight" style="background-color:">signaling</span> pathways. c-cbl, E3 ubiquitin-protein ligase c-Cbl; rheb,                   Ras homolog enriched in brain; RSK, ribosomal S6 kinase; uPA, urokinase                   plasminogen activator.</div></p></div><div class="aux"><div class="resc">Pat Levitt, et al. J Clin Invest. 2009 April 1;119(4):747-754.</div><p class="links"><a class="dblinks" href="/pubmed/19339766" ref="ordinalpos=4107&amp;ncbi_uid=1763998&amp;link_uid=19339766">Citation</a><a class="dblinks" href="/pmc/articles/PMC2662567/" ref="ordinalpos=4107&amp;ncbi_uid=1763998&amp;link_uid=PMC2662567">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4108.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912738/figure/F5/" ref="ordinalpos=4108&amp;ncbi_uid=5823315&amp;link_uid=PMC3912738" href="/pmc/articles/PMC3912738/"><img src="/pmc/articles/instance/3912738/bin/fendo-05-00010-g005.gif" src-large="/pmc/articles/instance/3912738/bin/fendo-05-00010-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912738/" ref="ordinalpos=4108&amp;ncbi_uid=5823315&amp;link_uid=PMC3912738" image-link="/pmc/articles/PMC3912738/figure/F5/" class="imagepopup">Figure 5.  From: The Emerging Role of Insulin and Insulin-Like Growth Factor <span class="highlight" style="background-color:">Signaling</span> in Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div><b>Proposed model for the interaction between Shh and IGF <span class="highlight" style="background-color:">signaling</span></b>. Shh binds to patched family receptors (PTCH) to releases smoothened (SMO) signal transducer from Patched-dependent suppression. Activated SMO, in turn, induces SKT36 serine/threonine kinase to stabilize GLI family members for nuclear translocation. Shh <span class="highlight" style="background-color:">signaling</span> activates GLI dependent transcription of target genes including N-myc, D-cyclins, BMI1, GLI1, Bcl-2, and IRS-1, all of which have been implicated in cell cycle progression, proliferation, and stemness. IRS-1, in turn, mediates IGF-IR/PI3K <span class="highlight" style="background-color:">signaling</span> activation, which inhibits GSK3-mediated phosphorylation and consequent increased degradation of N-myc and D-cyclins. However, Shh may also activate IGF-IR/PI3K <span class="highlight" style="background-color:">pathway</span> via unknown intermediary molecules.</div></p></div><div class="aux"><div class="resc">Roberta Malaguarnera, et al. Front Endocrinol (Lausanne). 2014;5:10.</div><p class="links"><a class="dblinks" href="/pubmed/24550888" ref="ordinalpos=4108&amp;ncbi_uid=5823315&amp;link_uid=24550888">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912738/" ref="ordinalpos=4108&amp;ncbi_uid=5823315&amp;link_uid=PMC3912738">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4109.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3013016/figure/F8/" ref="ordinalpos=4109&amp;ncbi_uid=3446268&amp;link_uid=PMC3013016" href="/pmc/articles/PMC3013016/"><img src="/pmc/articles/instance/3013016/bin/zbc0071146750008.gif" src-large="/pmc/articles/instance/3013016/bin/zbc0071146750008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3013016/" ref="ordinalpos=4109&amp;ncbi_uid=3446268&amp;link_uid=PMC3013016" image-link="/pmc/articles/PMC3013016/figure/F8/" class="imagepopup">FIGURE 8. From: ERK1/2 Phosphorylate Raptor to Promote Ras-dependent Activation of mTOR Complex 1 (mTORC1). </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the molecular mechanisms employed by the Ras/MAPK <span class="highlight" style="background-color:">pathway</span> to regulate mTORC1 <span class="highlight" style="background-color:">signaling</span>.</b> Activation of the Ras/MAPK <span class="highlight" style="background-color:">pathway</span> stimulates mTORC1 activity through the coordinated action of the kinases ERK1/2 and RSK on the TSC protein complex upstream mTORC1, and on Raptor, an important partner of mTOR within mTORC1.</div></p></div><div class="aux"><div class="resc">Audrey Carriere, et al. J Biol Chem. 2011 January 7;286(1):567-577.</div><p class="links"><a class="dblinks" href="/pubmed/21071439" ref="ordinalpos=4109&amp;ncbi_uid=3446268&amp;link_uid=21071439">Citation</a><a class="dblinks" href="/pmc/articles/PMC3013016/" ref="ordinalpos=4109&amp;ncbi_uid=3446268&amp;link_uid=PMC3013016">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4110.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3979733/figure/pone-0093960-g007/" ref="ordinalpos=4110&amp;ncbi_uid=6018427&amp;link_uid=PMC3979733" href="/pmc/articles/PMC3979733/"><img src="/pmc/articles/instance/3979733/bin/pone.0093960.g007.gif" src-large="/pmc/articles/instance/3979733/bin/pone.0093960.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3979733/" ref="ordinalpos=4110&amp;ncbi_uid=6018427&amp;link_uid=PMC3979733" image-link="/pmc/articles/PMC3979733/figure/pone-0093960-g007/" class="imagepopup">Figure 7. Basic structure of the growth factor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, showing examples of serial and parallel combinations..  From: Synergistic and Antagonistic Drug Combinations Depend on Network Topology. </a></p><div class="aux"><div class="resc">Ning Yin, et al. PLoS One. 2014;9(4):e93960.</div><p class="links"><a class="dblinks" href="/pubmed/24713621" ref="ordinalpos=4110&amp;ncbi_uid=6018427&amp;link_uid=24713621">Citation</a><a class="dblinks" href="/pmc/articles/PMC3979733/" ref="ordinalpos=4110&amp;ncbi_uid=6018427&amp;link_uid=PMC3979733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4111.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775648/figure/F2/" ref="ordinalpos=4111&amp;ncbi_uid=6449589&amp;link_uid=PMC3775648" href="/pmc/articles/PMC3775648/"><img src="/pmc/articles/instance/3775648/bin/nihms495103f2.gif" src-large="/pmc/articles/instance/3775648/bin/nihms495103f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775648/" ref="ordinalpos=4111&amp;ncbi_uid=6449589&amp;link_uid=PMC3775648" image-link="/pmc/articles/PMC3775648/figure/F2/" class="imagepopup">Figure 2.  From: Next Generation Sequencing Analysis of Gene Regulation in the Rat Model of Retinopathy of Prematurity. </a></p><div class="supp"><p class="details"><div>Major elements of the noncanonical PCP Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Symbology as in figure 1.</div></p></div><div class="aux"><div class="resc">Rachel M. Griffith, et al. Doc Ophthalmol. 2013 August;127(1):13-31.</div><p class="links"><a class="dblinks" href="/pubmed/23775346" ref="ordinalpos=4111&amp;ncbi_uid=6449589&amp;link_uid=23775346">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775648/" ref="ordinalpos=4111&amp;ncbi_uid=6449589&amp;link_uid=PMC3775648">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4112.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3594407/figure/F8/" ref="ordinalpos=4112&amp;ncbi_uid=5865096&amp;link_uid=PMC3594407" href="/pmc/articles/PMC3594407/"><img src="/pmc/articles/instance/3594407/bin/nihms-415061-f0008.gif" src-large="/pmc/articles/instance/3594407/bin/nihms-415061-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3594407/" ref="ordinalpos=4112&amp;ncbi_uid=5865096&amp;link_uid=PMC3594407" image-link="/pmc/articles/PMC3594407/figure/F8/" class="imagepopup">Figure 8.  From: ?-CATENIN REGULATES INNATE AND ADAPTIVE IMMUNITY IN MOUSE LIVER ISCHEMIA-REPERFUSION INJURY. </a></p><div class="supp"><p class="details"><div>Schematic representation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by which -catenin may regulate intricate inflammatory responses in liver IRI. See text for details.</div></p></div><div class="aux"><div class="resc">Bibo Ke, et al. Hepatology. 2013 March;57(3):1203-1214.</div><p class="links"><a class="dblinks" href="/pubmed/23081841" ref="ordinalpos=4112&amp;ncbi_uid=5865096&amp;link_uid=23081841">Citation</a><a class="dblinks" href="/pmc/articles/PMC3594407/" ref="ordinalpos=4112&amp;ncbi_uid=5865096&amp;link_uid=PMC3594407">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4113.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4029595/figure/F7/" ref="ordinalpos=4113&amp;ncbi_uid=6157260&amp;link_uid=PMC4029595" href="/pmc/articles/PMC4029595/"><img src="/pmc/articles/instance/4029595/bin/nihms-582785-f0007.gif" src-large="/pmc/articles/instance/4029595/bin/nihms-582785-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4029595/" ref="ordinalpos=4113&amp;ncbi_uid=6157260&amp;link_uid=PMC4029595" image-link="/pmc/articles/PMC4029595/figure/F7/" class="imagepopup">Figure 7.  From: Exosomal Proteome Profiling: A Potential Multi-Marker Cellular Phenotyping Tool to Characterize Hypoxia-Induced Radiation Resistance in Breast Cancer. </a></p><div class="supp"><p class="details"><div>Proposed breast cancer <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> based on the KEGG network <span class="highlight" style="background-color:">pathway</span> analysis of hypoxia-activated tumor cell migration. In EGF-stimulated SKBR3 cells with decreased VPS4B expression, the assembly of F-actin and the formation of filopodia were promoted, suggesting that the dysfunction of the MVB <span class="highlight" style="background-color:">pathway</span> stimulates the migration of tumor cells under hypoxic conditions.</div></p></div><div class="aux"><div class="resc">Stefani N. Thomas, et al. Proteomes. ;1(2):87-108.</div><p class="links"><a class="dblinks" href="/pubmed/24860738" ref="ordinalpos=4113&amp;ncbi_uid=6157260&amp;link_uid=24860738">Citation</a><a class="dblinks" href="/pmc/articles/PMC4029595/" ref="ordinalpos=4113&amp;ncbi_uid=6157260&amp;link_uid=PMC4029595">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4114.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3272037/figure/pone-0030722-g010/" ref="ordinalpos=4114&amp;ncbi_uid=3796945&amp;link_uid=PMC3272037" href="/pmc/articles/PMC3272037/"><img src="/pmc/articles/instance/3272037/bin/pone.0030722.g010.gif" src-large="/pmc/articles/instance/3272037/bin/pone.0030722.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3272037/" ref="ordinalpos=4114&amp;ncbi_uid=3796945&amp;link_uid=PMC3272037" image-link="/pmc/articles/PMC3272037/figure/pone-0030722-g010/" class="imagepopup">Figure 10. Axon guidance and synapse growth are controlled by separate branches of the Tor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Diet and Energy-Sensing Inputs Affect TorC1-Mediated Axon Misrouting but Not TorC2-Directed Synapse Growth in a Drosophila Model of Tuberous Sclerosis. </a></p><div class="supp"><p class="details"><div>A schematic diagram illustrating the effects of Tor-<span class="highlight" style="background-color:">signaling</span> elements on axon guidance and synapse growth. <i>Rheb</i>-mediated axon misrouting defects are dependent on <span class="highlight" style="background-color:">signaling</span> through TorC1 and S6k, and can be rescued by diet restriction or AMPK expression. Axon guidance is largely insensitive to changes in Pi3K. Synapse overgrowth mediated by <i>Rheb</i> overexpression is dependent on <span class="highlight" style="background-color:">signaling</span> through TorC2, is not rescued by caloric restriction or AMPK activation, and is exacerbated by knockdown of <i>S6k</i>, suggesting the removal of a feedback inhibition. Synapse overgrowth is also induced by <i>Pi3K</i> overexpression, similarly to <i>Rheb</i>.</div></p></div><div class="aux"><div class="resc">Brian Dimitroff, et al. PLoS One. 2012;7(2):e30722.</div><p class="links"><a class="dblinks" href="/pubmed/22319582" ref="ordinalpos=4114&amp;ncbi_uid=3796945&amp;link_uid=22319582">Citation</a><a class="dblinks" href="/pmc/articles/PMC3272037/" ref="ordinalpos=4114&amp;ncbi_uid=3796945&amp;link_uid=PMC3272037">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4115.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163143/figure/fig3/" ref="ordinalpos=4115&amp;ncbi_uid=3190902&amp;link_uid=PMC3163143" href="/pmc/articles/PMC3163143/"><img src="/pmc/articles/instance/3163143/bin/JO2012-782020.003.gif" src-large="/pmc/articles/instance/3163143/bin/JO2012-782020.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163143/" ref="ordinalpos=4115&amp;ncbi_uid=3190902&amp;link_uid=PMC3163143" image-link="/pmc/articles/PMC3163143/figure/fig3/" class="imagepopup">Figure 3.  From: Targeting Angiogenesis for Controlling Neuroblastoma. </a></p><div class="supp"><p class="details"><div>Genomic and non-genomic actions of all-<i>trans</i> retinoic acid (ATRA). Genomic action of ATRA includes caspase-8 transcription, cell cycle arrest, and apoptosis. Non-genomic action of ATRA involves induction of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> for upregulation of secretase. ATRA activates retinoic acid receptor (RAR) and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for contribution to differentiation in neuroblastoma.</div></p></div><div class="aux"><div class="resc">Subhasree Roy Choudhury, et al. J Oncol. 2012;2012:782020.</div><p class="links"><a class="dblinks" href="/pubmed/21876694" ref="ordinalpos=4115&amp;ncbi_uid=3190902&amp;link_uid=21876694">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163143/" ref="ordinalpos=4115&amp;ncbi_uid=3190902&amp;link_uid=PMC3163143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4116.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2667898/figure/F6/" ref="ordinalpos=4116&amp;ncbi_uid=1843403&amp;link_uid=PMC2667898" href="/pmc/articles/PMC2667898/"><img src="/pmc/articles/instance/2667898/bin/nihms-106100-f0006.gif" src-large="/pmc/articles/instance/2667898/bin/nihms-106100-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2667898/" ref="ordinalpos=4116&amp;ncbi_uid=1843403&amp;link_uid=PMC2667898" image-link="/pmc/articles/PMC2667898/figure/F6/" class="imagepopup">Figure 6.  From: Cancer-Associated Transforming Growth Factor ? Type II Receptor Gene Mutant Causes Activation of Bone Morphogenic Protein-Smads and Invasive Phenotype. </a></p><div class="supp"><p class="details"><div>Proposed model of TR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal human keratinocytes and squamous carcinoma cells. In normal human keratinocytes (exemplified by HaCaT cells), TGF binds to the TRII and activates the classic Smad2/3 <span class="highlight" style="background-color:">pathway</span> via TRI (Alk-5). In addition, our studies indicate that TGF can activate BMP-Smads (Smad1 and Smad5) in a TRI (Alk-5)-dependent manner. In analogy with endothelial cells, we assume that one of the BMP type I receptors is involved in this process (Alk-x), with Alk-2 being the leading candidate. Moreover, one might speculate that the Smad2/3 arm of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates the homeostatic function of TGF, whereas the Smad1/5 arm is involved in the tissue injury response. In the case of some cancer-associated TRII mutants (exemplified by TRII<sup>E526Q</sup> in SqCC/Y1 cells), both arms of the TGF/Smad <span class="highlight" style="background-color:">pathway</span> are abrogated. In contrast, in the case of TRII<sup>R537P</sup>-mutant A253 cells, the TRII/Alk-5/Smad2/3 <span class="highlight" style="background-color:">pathway</span> is also abrogated, resulting in loss of TGF-mediated growth control. However, the Smad1/5 <span class="highlight" style="background-color:">pathway</span> is constitutively activated in an Alk-5-independent manner, apparently resulting in increased EMT, cell migration, and invasion. Thus, this type of mutant has a dual-function phenotype, associated with loss of one arm of the TGF/Smad <span class="highlight" style="background-color:">pathway</span> and activation of the second arm. Moreover, our studies indicate that this gain of function is inhibitable by the dual-receptor kinase inhibitor LY2109761, while it is resistant to selective TRI kinase inhibitors.</div></p></div><div class="aux"><div class="resc">Savita Bharathy, et al. Cancer Res. ;68(6):1656-1666.</div><p class="links"><a class="dblinks" href="/pubmed/18339844" ref="ordinalpos=4116&amp;ncbi_uid=1843403&amp;link_uid=18339844">Citation</a><a class="dblinks" href="/pmc/articles/PMC2667898/" ref="ordinalpos=4116&amp;ncbi_uid=1843403&amp;link_uid=PMC2667898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4117.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4237136/figure/F2/" ref="ordinalpos=4117&amp;ncbi_uid=6905009&amp;link_uid=PMC4237136" href="/pmc/articles/PMC4237136/"><img src="/pmc/articles/instance/4237136/bin/fgene-05-00361-g002.gif" src-large="/pmc/articles/instance/4237136/bin/fgene-05-00361-g002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4237136/" ref="ordinalpos=4117&amp;ncbi_uid=6905009&amp;link_uid=PMC4237136" image-link="/pmc/articles/PMC4237136/figure/F2/" class="imagepopup">FIGURE 2.  From: Pathogenic microRNAs in myeloid malignancies. </a></p><div class="supp"><p class="details"><div><b>Proposed mechanism for <i>miR-155</i>-mediated myeloid leukemogenesis.</b> Overexpression of <i>miR-155</i> leads to the activation of the PI3K-Akt <span class="highlight" style="background-color:">pathway</span> through negative regulation of Src Homology 2 domain-containing Inositol-5-Phosphatase (<i>SHIP1</i>). <i>SHIP1</i> is a phosphatase that mediates the conversion of phosphatidylinositol triphosphate (PIP3) to phosphatidylinositol diphosphate (PIP2). PIP3 normally acts as a docking site for <span class="highlight" style="background-color:">signaling</span> molecules in the PI3K-Akt <span class="highlight" style="background-color:">pathway</span> and helps relay the signal. Upon <i>miR-155</i> overexpression and thus, SHIP1 downregulation, PIP3 level is increased, which leads to the activation of the PI3K-Akt <span class="highlight" style="background-color:">pathway</span>. Green arrows indicate increased, and red arrows decreased activity upon <i>miR-155</i> overexpression.</div></p></div><div class="aux"><div class="resc">Mona Khalaj, et al. Front Genet. 2014;5:361.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4117&amp;ncbi_uid=6905009&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4237136/" ref="ordinalpos=4117&amp;ncbi_uid=6905009&amp;link_uid=PMC4237136">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4118.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3274430/figure/F3/" ref="ordinalpos=4118&amp;ncbi_uid=3793522&amp;link_uid=PMC3274430" href="/pmc/articles/PMC3274430/"><img src="/pmc/articles/instance/3274430/bin/1471-2407-12-21-3.gif" src-large="/pmc/articles/instance/3274430/bin/1471-2407-12-21-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3274430/" ref="ordinalpos=4118&amp;ncbi_uid=3793522&amp;link_uid=PMC3274430" image-link="/pmc/articles/PMC3274430/figure/F3/" class="imagepopup">Figure 3.  From: Gene expression in tumor cells and stroma in dsRed 4T1 tumors in eGFP-expressing mice with and without enhanced oxygenation. </a></p><div class="supp"><p class="details"><div><b>VEGF <span class="highlight" style="background-color:">pathway</span>, tumor</b>. Heatmap of genes in the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HSA04370). The table display gene names, systematic names, and fold change of gene expression compared to control of intermittent (I) and daily (D) hyperbaric oxygen (HBO), respectively. The VEGF <span class="highlight" style="background-color:">pathway</span> is significantly down-regulated by intermittent HBO treatment of 4T1 mammary tumor cells, but not by daily treatment. NS = not significant.</div></p></div><div class="aux"><div class="resc">Ingrid Moen, et al. BMC Cancer. 2012;12:21-21.</div><p class="links"><a class="dblinks" href="/pubmed/22251838" ref="ordinalpos=4118&amp;ncbi_uid=3793522&amp;link_uid=22251838">Citation</a><a class="dblinks" href="/pmc/articles/PMC3274430/" ref="ordinalpos=4118&amp;ncbi_uid=3793522&amp;link_uid=PMC3274430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4119.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC312639/figure/F9/" ref="ordinalpos=4119&amp;ncbi_uid=378395&amp;link_uid=PMC312639" href="/pmc/articles/PMC312639/"><img src="/pmc/articles/instance/312639/bin/10f9_L1TT.gif" src-large="/pmc/articles/instance/312639/bin/10f9_L1TT.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC312639/" ref="ordinalpos=4119&amp;ncbi_uid=378395&amp;link_uid=PMC312639" image-link="/pmc/articles/PMC312639/figure/F9/" class="imagepopup">Figure 9.  From: Regulation of gene expression by the small GTPase Rho through the ERK6 (p38?) MAP kinase <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed model for RhoA <span class="highlight" style="background-color:">signaling</span> to the c-<i>jun</i> promoter. Activated RhoA can initiate a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inducing ERK6, which in turn activates MEF2A and ATF2 transcription factors bound to the jAP1 and MEF2 sites on the c-<i>jun</i> promoter, thereby enhancing c-<i>jun</i> expression. Available data suggest that Rho A, PKN, MKK3/MKK6, and ERK6 (p38) are components of a novel signal transduction <span class="highlight" style="background-color:">pathway</span> involved in the regulation of gene expression and cellular transformation. Parallel pathways are depicted and described in the text. Unknown molecules are indicated by question marks.</div></p></div><div class="aux"><div class="resc">Maria Julia Marinissen, et al. Genes Dev. 2001 March 1;15(5):535-553.</div><p class="links"><a class="dblinks" href="/pubmed/11238375" ref="ordinalpos=4119&amp;ncbi_uid=378395&amp;link_uid=11238375">Citation</a><a class="dblinks" href="/pmc/articles/PMC312639/" ref="ordinalpos=4119&amp;ncbi_uid=378395&amp;link_uid=PMC312639">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4120.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2214701/figure/F3/" ref="ordinalpos=4120&amp;ncbi_uid=1149450&amp;link_uid=PMC2214701" href="/pmc/articles/PMC2214701/"><img src="/pmc/articles/instance/2214701/bin/JCI0833612.f3.gif" src-large="/pmc/articles/instance/2214701/bin/JCI0833612.f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2214701/" ref="ordinalpos=4120&amp;ncbi_uid=1149450&amp;link_uid=PMC2214701" image-link="/pmc/articles/PMC2214701/figure/F3/" class="imagepopup">Figure 3. A partial view of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Building bone to reverse osteoporosis and repair fractures. </a></p><div class="supp"><p class="details"><div>Wnts bind a receptor complex consisting of LRP5 or LRP6 and one of ten Fz proteins. This prevents phosphorylation of -catenin by GSK3 and other kinases and its subsequent degradation. Of note, mutating the residues that can be phosphorylated to alanine creates stable, gain-of-function -catenin proteins. Stabilized -catenin accumulates and translocates to the nucleus, where it interacts with Tcf7 and related transcription factors (Lef1, Tcf7L1, Tcf7L2) to regulate gene expression. Outside the cell, molecules that sequester either LRP5 (e.g., Dkk1 and sclerostin) or the Wnt ligand (e.g., Sfrp) negatively control the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Lithium chloride inhibits GSK3 inside the cell. APC, adenomatosis polyposis coli.</div></p></div><div class="aux"><div class="resc">Sundeep Khosla, et al. J Clin Invest. 2008 February 1;118(2):421-428.</div><p class="links"><a class="dblinks" href="/pubmed/18246192" ref="ordinalpos=4120&amp;ncbi_uid=1149450&amp;link_uid=18246192">Citation</a><a class="dblinks" href="/pmc/articles/PMC2214701/" ref="ordinalpos=4120&amp;ncbi_uid=1149450&amp;link_uid=PMC2214701">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4121.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1129099/figure/fig4/" ref="ordinalpos=4121&amp;ncbi_uid=462069&amp;link_uid=PMC1129099" href="/pmc/articles/PMC1129099/"><img src="/pmc/articles/instance/1129099/bin/zpq0190582160004.gif" src-large="/pmc/articles/instance/1129099/bin/zpq0190582160004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1129099/" ref="ordinalpos=4121&amp;ncbi_uid=462069&amp;link_uid=PMC1129099" image-link="/pmc/articles/PMC1129099/figure/fig4/" class="imagepopup">Fig. 4. From: The Toll <span class="highlight" style="background-color:">pathway</span> is important for an antiviral response in Drosophila. </a></p><div class="supp"><p class="details"><div>Model for <i>Drosophila</i> antiviral response. 1, Virus enters system and infects the cell. 2, Virus replication. 3, Lysis of infected cell, releasing internal cellular compounds and virus. 4, Released materials activate the IMD and Toll humoral immune pathways and cause local activation of the cellular response. 5, Global activation of hemocytes via Toll <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. 6, Activated hemocytes signal to the fat body enhancing Toll and IMD pathways activation. 7, Hemocytes recognize aberrant infected cells and engulf and eliminate these cells. Activated hemocytes then enforce stringent recognition of aberrant tissue and destroy infected cells more efficiently. Green arrows indicate <span class="highlight" style="background-color:">signaling</span> to activate the humoral response. Blue arrows indicate <span class="highlight" style="background-color:">signaling</span> to activate the cellular response.</div></p></div><div class="aux"><div class="resc">Robert A. Zambon, et al. Proc Natl Acad Sci U S A. 2005 May 17;102(20):7257-7262.</div><p class="links"><a class="dblinks" href="/pubmed/15878994" ref="ordinalpos=4121&amp;ncbi_uid=462069&amp;link_uid=15878994">Citation</a><a class="dblinks" href="/pmc/articles/PMC1129099/" ref="ordinalpos=4121&amp;ncbi_uid=462069&amp;link_uid=PMC1129099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4122.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3219506/figure/F2/" ref="ordinalpos=4122&amp;ncbi_uid=3423641&amp;link_uid=PMC3219506" href="/pmc/articles/PMC3219506/"><img src="/pmc/articles/instance/3219506/bin/nihms117527f2.gif" src-large="/pmc/articles/instance/3219506/bin/nihms117527f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3219506/" ref="ordinalpos=4122&amp;ncbi_uid=3423641&amp;link_uid=PMC3219506" image-link="/pmc/articles/PMC3219506/figure/F2/" class="imagepopup">Figure 2. Nonessential viral proteins that are needed to counteract the dsRNA-dependent PKR <span class="highlight" style="background-color:">pathway</span> that is activated upon viral infection.  From: Design and application of oncolytic HSV vectors for glioblastoma therapy. </a></p><div class="supp"><p class="details"><div>The schematic depicts the PKR <span class="highlight" style="background-color:">pathway</span> that is initiated following virus infection. Expression of PKR is activated by the interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that is initiated upon viral infection. PKR activation (phosphorylation) is induced in response to dsRNA that is produced in virally infected cells. Active PKR phosphorylates eIF2 and thereby inhibits protein synthesis and blocks the production of viral proteins. PKR also activates the transcription factor NF-B and induces an antiviral immune response. In order to counteract this <span class="highlight" style="background-color:">pathway</span>, 34.5 retargets protein phosphatase-1 to induce dephosphorylation of eIF2. US11 (if present with immediate-early gene kinetics) is able to inhibit PKR by a different mechanism, and the multifunctional ICP0 protein is also thought to play a role in inactivation of the interferon and PKR pathways. Activation of the Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibits PKR activity and complements for the lack of 34.5 (this example illustrates Ras activation after binding of EGF to its receptor).<br />HSV: Herpes simplex virus; NF: Nuclear factor; PKR: RNA-dependent protein kinase.</div></p></div><div class="aux"><div class="resc">Paola Grandi, et al. Expert Rev Neurother. ;9(4):505-517.</div><p class="links"><a class="dblinks" href="/pubmed/19344302" ref="ordinalpos=4122&amp;ncbi_uid=3423641&amp;link_uid=19344302">Citation</a><a class="dblinks" href="/pmc/articles/PMC3219506/" ref="ordinalpos=4122&amp;ncbi_uid=3423641&amp;link_uid=PMC3219506">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4123.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1971124/figure/pbio-0050248-g004/" ref="ordinalpos=4123&amp;ncbi_uid=1145887&amp;link_uid=PMC1971124" href="/pmc/articles/PMC1971124/"><img src="/pmc/articles/instance/2043018/bin/pbio.0050248.g004.gif" src-large="/pmc/articles/instance/2043018/bin/pbio.0050248.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1971124/" ref="ordinalpos=4123&amp;ncbi_uid=1145887&amp;link_uid=PMC1971124" image-link="/pmc/articles/PMC1971124/figure/pbio-0050248-g004/" class="imagepopup">Figure 4. Diagram of ITAM-Mediated <span class="highlight" style="background-color:">Signaling</span> Pathways in Human Leukocytes.  From: BDCA2/Fc?RI? Complex Signals through a Novel BCR-Like <span class="highlight" style="background-color:">Pathway</span> in Human Plasmacytoid Dendritic Cells. </a></p><div class="supp"><p class="details"><div>The schematic cascades that are involved in receptor proximal <span class="highlight" style="background-color:">signaling</span> (solid arrows) and those leading to NF-B activation (dotted lines) are shown.</div></p></div><div class="aux"><div class="resc">Wei Cao, et al. PLoS Biol. 2007 October;5(10):e248.</div><p class="links"><a class="dblinks" href="/pubmed/17850179" ref="ordinalpos=4123&amp;ncbi_uid=1145887&amp;link_uid=17850179">Citation</a><a class="dblinks" href="/pmc/articles/PMC1971124/" ref="ordinalpos=4123&amp;ncbi_uid=1145887&amp;link_uid=PMC1971124">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4124.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2603136/figure/F1/" ref="ordinalpos=4124&amp;ncbi_uid=1582269&amp;link_uid=PMC2603136" href="/pmc/articles/PMC2603136/"><img src="/pmc/articles/instance/2603136/bin/nihms-81890-f0005.gif" src-large="/pmc/articles/instance/2603136/bin/nihms-81890-f0005.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2603136/" ref="ordinalpos=4124&amp;ncbi_uid=1582269&amp;link_uid=PMC2603136" image-link="/pmc/articles/PMC2603136/figure/F1/" class="imagepopup">Figure 1.  From: Molecular Approaches towards the Inhibition of the Signal Transducer and Activator of Transcription 3 (Stat3) Protein. </a></p><div class="supp"><p class="details"><div>JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. STAT activity, arrowheaded lines; inhibitory events, barred lines. Abbreviations: Ub, ubiquitin; pY, pTyr; PTP, phosphotyrosine phosphatase; SOCS, suppressors of cytokine <span class="highlight" style="background-color:">signaling</span>; PIAS, protein inhibitors of activated STAT; SUMO, small ubiquitin-related modifier.</div></p></div><div class="aux"><div class="resc">Steven Fletcher, et al. ChemMedChem. ;3(8):1159-1168.</div><p class="links"><a class="dblinks" href="/pubmed/18683176" ref="ordinalpos=4124&amp;ncbi_uid=1582269&amp;link_uid=18683176">Citation</a><a class="dblinks" href="/pmc/articles/PMC2603136/" ref="ordinalpos=4124&amp;ncbi_uid=1582269&amp;link_uid=PMC2603136">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4125.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2929604/figure/fig1/" ref="ordinalpos=4125&amp;ncbi_uid=2449721&amp;link_uid=PMC2929604" href="/pmc/articles/PMC2929604/"><img src="/pmc/articles/instance/2929604/bin/GRP2010-537263.001.gif" src-large="/pmc/articles/instance/2929604/bin/GRP2010-537263.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2929604/" ref="ordinalpos=4125&amp;ncbi_uid=2449721&amp;link_uid=PMC2929604" image-link="/pmc/articles/PMC2929604/figure/fig1/" class="imagepopup">Figure 1.  From: Toll-Like Receptors in Hepatic Ischemia/Reperfusion and Transplantation. </a></p><div class="supp"><p class="details"><div>Role of TLR4 in hepatic I/R injury.  <span class="highlight" style="background-color:">Signaling</span> through the pattern recognition receptor TLR4 mediates multiple inflammatory pathways following hepatic I/R.  The activation of TLR4 <span class="highlight" style="background-color:">signaling</span> is dependent, in part, on circulating DAMPs.  Downstream <span class="highlight" style="background-color:">signaling</span> events include the TRIF-dependent activation of IRF3, the production of Type I IFN, and upregulation of CXCL10.  Additionally, TLR4 activation of IRF1 promotes release of the DAMP, HMGB1.  Lastly, TLR4 activation is thought to suppress the cytoprotective HO-1 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">John Evankovich, et al. Gastroenterol Res Pract. 2010;2010:537263.</div><p class="links"><a class="dblinks" href="/pubmed/20811615" ref="ordinalpos=4125&amp;ncbi_uid=2449721&amp;link_uid=20811615">Citation</a><a class="dblinks" href="/pmc/articles/PMC2929604/" ref="ordinalpos=4125&amp;ncbi_uid=2449721&amp;link_uid=PMC2929604">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4126.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2935335/figure/f3/" ref="ordinalpos=4126&amp;ncbi_uid=3059575&amp;link_uid=PMC2935335" href="/pmc/articles/PMC2935335/"><img src="/pmc/articles/instance/2935335/bin/fig-3.gif" src-large="/pmc/articles/instance/2935335/bin/fig-3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2935335/" ref="ordinalpos=4126&amp;ncbi_uid=3059575&amp;link_uid=PMC2935335" image-link="/pmc/articles/PMC2935335/figure/f3/" class="imagepopup">FIG. 3. From: Genetic Alterations in the Phosphatidylinositol-3 Kinase/Akt <span class="highlight" style="background-color:">Pathway</span> in Thyroid Cancer. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the PI3K/Akt and MAP kinase pathways driven by the increasing accumulation of genetic alterations in the progression from low-grade to high-grade thyroid cancers. In this model, primary genetic alterations in the MAP kinase <span class="highlight" style="background-color:">pathway</span>, such as the <i>BRAF</i> mutation, transform the thyroid cell into PTC. As the genetic alterations accumulate, particularly with those that can also activate the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, the cancer progresses into aggressive type, such as ATC. If primary genetic alterations, such as genetic or epigenetic silencing of <i>PTEN</i>, occur in the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, the thyroid cell is transformed into follicular thyroid tumors. As genetic alterations accumulate in this <span class="highlight" style="background-color:">pathway</span>, a follicular adenoma can progress into FTC when tumor invasion occurs, which can progress further into ATC when genetic alterations accumulate further, particularly if there are genetic alterations that can dually activate the PI3K/Akt and MAP kinase pathways, such as <i>Ras</i> mutations or genomic amplifications of the receptor tyrosine kinase genes. The increasing intensity of the gray tint in the circles represents increasing genetic alterations and the <span class="highlight" style="background-color:">signaling</span> pressure of the pathways in the progression from thyroid cells to benign thyroid tumors to differentiated thyroid cancers and to ATC. The figure is adapted from reference 19 with permission. FTC, follicular thyroid cancer.</div></p></div><div class="aux"><div class="resc">Mingzhao Xing. Thyroid. 2010 July;20(7):697-706.</div><p class="links"><a class="dblinks" href="/pubmed/20578891" ref="ordinalpos=4126&amp;ncbi_uid=3059575&amp;link_uid=20578891">Citation</a><a class="dblinks" href="/pmc/articles/PMC2935335/" ref="ordinalpos=4126&amp;ncbi_uid=3059575&amp;link_uid=PMC2935335">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4127.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2740573/figure/F5/" ref="ordinalpos=4127&amp;ncbi_uid=1980503&amp;link_uid=PMC2740573" href="/pmc/articles/PMC2740573/"><img src="/pmc/articles/instance/2740573/bin/zbc0340982740005.gif" src-large="/pmc/articles/instance/2740573/bin/zbc0340982740005.jpg" alt="FIGURE 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2740573/" ref="ordinalpos=4127&amp;ncbi_uid=1980503&amp;link_uid=PMC2740573" image-link="/pmc/articles/PMC2740573/figure/F5/" class="imagepopup">FIGURE 5. From: Proteolytic Cascade for the Activation of the Insect Toll <span class="highlight" style="background-color:">Pathway</span> Induced by the Fungal Cell Wall Component. </a></p><div class="supp"><p class="details"><div><b>A model of the serine proteases cascades regulating the Toll <span class="highlight" style="background-color:">pathway</span> in <i>Tenebrio</i>.</b> When the -1,3-glucan of fungi is exposed to the insect, <i>Tenebrio</i> GNBP3 binds to -1,3-glucan and then recruits pro-MSP. In the presence of Ca<sup>2+</sup>, the -1,3-glucan/GNBP3 complex induces activation of pro-MSP to activated MSP (<i>aMSP</i>). The aMSP converts pro-SAE to activated SAE (<i>aSAE</i>), which subsequently converts the pro-SPE to activated SPE (<i>aSPE</i>). The aSPE cleaves pro-Sptzle to processed Sptzle leading to the production of AMPs such as Tenecin 1 and Tenecin 2 via the Toll receptor. The Lys-type PG-dependent <i>Tenebrio</i> Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was previously reported by our group (26).</div></p></div><div class="aux"><div class="resc">Kyung-Baeg Roh, et al. J Biol Chem. 2009 July 17;284(29):19474-19481.</div><p class="links"><a class="dblinks" href="/pubmed/19473968" ref="ordinalpos=4127&amp;ncbi_uid=1980503&amp;link_uid=19473968">Citation</a><a class="dblinks" href="/pmc/articles/PMC2740573/" ref="ordinalpos=4127&amp;ncbi_uid=1980503&amp;link_uid=PMC2740573">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4128.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3275703/figure/F4/" ref="ordinalpos=4128&amp;ncbi_uid=3805747&amp;link_uid=PMC3275703" href="/pmc/articles/PMC3275703/"><img src="/pmc/articles/instance/3275703/bin/in-11-348-g004.gif" src-large="/pmc/articles/instance/3275703/bin/in-11-348-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3275703/" ref="ordinalpos=4128&amp;ncbi_uid=3805747&amp;link_uid=PMC3275703" image-link="/pmc/articles/PMC3275703/figure/F4/" class="imagepopup">Figure 4.  From: NDRG2 Promotes GATA-1 Expression through Regulation of the JAK2/STAT <span class="highlight" style="background-color:">Pathway</span> in PMA-stimulated U937 Cells. </a></p><div class="supp"><p class="details"><div>BMP-4/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by NDRG2 expression. U937-mock and U937-NDRG2 cells were treated with 0.1g/ml PMA for 48 h. (A) Equal amounts of whole cell lysates were subjected to electrophoresis on SDS-PAGE gels, and Western blot analysis was performed using specific antibodies. Actin was used as an internal loading control. (B, C) mRNA levels were measured by RT-PCR.</div></p></div><div class="aux"><div class="resc">Kyeongah Kang, et al. Immune Netw. 2011 December;11(6):348-357.</div><p class="links"><a class="dblinks" href="/pubmed/22346774" ref="ordinalpos=4128&amp;ncbi_uid=3805747&amp;link_uid=22346774">Citation</a><a class="dblinks" href="/pmc/articles/PMC3275703/" ref="ordinalpos=4128&amp;ncbi_uid=3805747&amp;link_uid=PMC3275703">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4129.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2729341/figure/fig7/" ref="ordinalpos=4129&amp;ncbi_uid=1970507&amp;link_uid=PMC2729341" href="/pmc/articles/PMC2729341/"><img src="/pmc/articles/instance/2729341/bin/DEV034603F7.gif" src-large="/pmc/articles/instance/2729341/bin/DEV034603F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2729341/" ref="ordinalpos=4129&amp;ncbi_uid=1970507&amp;link_uid=PMC2729341" image-link="/pmc/articles/PMC2729341/figure/fig7/" class="imagepopup">Fig. 7. From: MSP and GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> coordinately regulate actomyosin-dependent  cytoplasmic streaming and oocyte growth in C. elegans. </a></p><div class="supp"><p class="details"><div><b>MSP <span class="highlight" style="background-color:">signaling</span> triggers rMLC phosphorylation throughout the gonad  arm.</b> (<b>A-D</b>) Detection of cortical actin (red) and P-rMLC using  PSer19-MLC antibodies (green) in dissected gonads from hermaphrodites (A),  unmated <i>fog-2(oz40)</i> females (B), MSP-injected <i>fog-2(oz40)</i>  females 15 minutes post-injection (C) and <i>glp-1(tn777ts)</i>  hermaphrodites (D). Arrowheads indicate P-rMLC staining in ring channels of  the proximal oocytes. DNA is blue. Scale bar: 10 m. (<b>E</b>) A model for  the control of germline proliferation, oocyte growth and meiotic maturation by  GLP-1 and MSP <span class="highlight" style="background-color:">signaling</span> (see text for details). P-rMLC is shown in brackets to  indicate that MSP <span class="highlight" style="background-color:">signaling</span> is sufficient to promote rMLC phosphorylation in  the germline, whereas the <i>glp-1</i> <span class="highlight" style="background-color:">pathway</span> appears dispensable.</div></p></div><div class="aux"><div class="resc">Saravanapriah Nadarajan, et al. Development. 2009 July 1;136(13):2223-2234.</div><p class="links"><a class="dblinks" href="/pubmed/19502484" ref="ordinalpos=4129&amp;ncbi_uid=1970507&amp;link_uid=19502484">Citation</a><a class="dblinks" href="/pmc/articles/PMC2729341/" ref="ordinalpos=4129&amp;ncbi_uid=1970507&amp;link_uid=PMC2729341">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4130.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4116528/figure/F9/" ref="ordinalpos=4130&amp;ncbi_uid=6458029&amp;link_uid=PMC4116528" href="/pmc/articles/PMC4116528/"><img src="/pmc/articles/instance/4116528/bin/oncotarget-05-3880-g009.gif" src-large="/pmc/articles/instance/4116528/bin/oncotarget-05-3880-g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4116528/" ref="ordinalpos=4130&amp;ncbi_uid=6458029&amp;link_uid=PMC4116528" image-link="/pmc/articles/PMC4116528/figure/F9/" class="imagepopup">Figure 9. Diagram showing the induction of stem cell-like properties by Snail via the activation of the Smad1/Akt/GSK3 <span class="highlight" style="background-color:">pathway</span> and subsequent upregulation of Nanog expression.  From: Snail regulates Nanog status during the epithelial-mesenchymal transition via the Smad1/Akt/GSK3? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in non-small-cell lung cancer. </a></p><div class="aux"><div class="resc">Chen-Wei Liu, et al. Oncotarget. 2014 June;5(11):3880-3894.</div><p class="links"><a class="dblinks" href="/pubmed/25003810" ref="ordinalpos=4130&amp;ncbi_uid=6458029&amp;link_uid=25003810">Citation</a><a class="dblinks" href="/pmc/articles/PMC4116528/" ref="ordinalpos=4130&amp;ncbi_uid=6458029&amp;link_uid=PMC4116528">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4131.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3868569/figure/pgen-1004020-g010/" ref="ordinalpos=4131&amp;ncbi_uid=5645552&amp;link_uid=PMC3868569" href="/pmc/articles/PMC3868569/"><img src="/pmc/articles/instance/3868569/bin/pgen.1004020.g010.gif" src-large="/pmc/articles/instance/3868569/bin/pgen.1004020.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3868569/" ref="ordinalpos=4131&amp;ncbi_uid=5645552&amp;link_uid=PMC3868569" image-link="/pmc/articles/PMC3868569/figure/pgen-1004020-g010/" class="imagepopup">Figure 10. Model for the regulation of TATN-1 expression, tyrosine levels, and the resulting effects of tyrosine effects on cell <span class="highlight" style="background-color:">signaling</span> pathways..  From: TATN-1 Mutations Reveal a Novel Role for Tyrosine as a Metabolic Signal That Influences Developmental Decisions and Longevity in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>Shown is the <i>daf-2</i>/IGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and <i>eak</i> genes from Figure 1A with the addition, shown in blue, of the effects of tyrosine identified in this work and the control of TATN-1 expression by <i>daf-2</i>/IGFR <span class="highlight" style="background-color:">signaling</span> and by dietary and environmental cues as demonstrated in Figure 8. In summary, worms sense the available diet and environmental temperature and these factors both contribute to the dauer decision and the regulation of TATN-1 protein levels. Conditions which promote the dauer decision also tend to reduce the expression of TATN-1. The reductions in TATN-1 expression lead to reduced removal of tyrosine through degradation and increase free tyrosine levels in the animal. These increases in tyrosine activate <i>aak-2</i>/AMPK and disrupt the effects of normal <i>daf-2</i>/IGFR <span class="highlight" style="background-color:">signaling</span> through positive effects on <i>daf-16</i>/FOXO and perhaps inhibitory effects on <i>crh-1</i>/CREB. The effects of tyrosine are particularly pronounced when the <i>daf-2</i>/IGFR <span class="highlight" style="background-color:">pathway</span> is compromised, such as through mutations in the <i>daf-2</i>/IGFR gene or the <i>eak</i> genes which lie in a parallel <span class="highlight" style="background-color:">pathway</span>. This may be due to further reductions in TATN-1 expression or reductions in inhibitory regulators of <i>daf-16</i>/FOXO activity.</div></p></div><div class="aux"><div class="resc">Annabel A. Ferguson, et al. PLoS Genet. 2013 December;9(12):e1004020.</div><p class="links"><a class="dblinks" href="/pubmed/24385923" ref="ordinalpos=4131&amp;ncbi_uid=5645552&amp;link_uid=24385923">Citation</a><a class="dblinks" href="/pmc/articles/PMC3868569/" ref="ordinalpos=4131&amp;ncbi_uid=5645552&amp;link_uid=PMC3868569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4132.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3697962/figure/fig2/" ref="ordinalpos=4132&amp;ncbi_uid=6336399&amp;link_uid=PMC3697962" href="/pmc/articles/PMC3697962/"><img src="/pmc/articles/instance/3697962/bin/539fig2.gif" src-large="/pmc/articles/instance/3697962/bin/539fig2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3697962/" ref="ordinalpos=4132&amp;ncbi_uid=6336399&amp;link_uid=PMC3697962" image-link="/pmc/articles/PMC3697962/figure/fig2/" class="imagepopup">Figure 2.  From: To Grow or Not to Grow: Nutritional Control of Development During Caenorhabditis elegans L1 Arrest. </a></p><div class="supp"><p class="details"><div>Model of the <i>C. elegans</i> insulin-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. There are 40 putative insulin-like peptides in <i>C. elegans</i>, and their secretion is mediated by UNC-31/CAPS and the conserved ATPase ASNA-1. DAF-2/InsR is the sole insulin/IGF receptor in <i>C. elegans</i>, and it signals through a conserved PI3K <span class="highlight" style="background-color:">pathway</span> to antagonize function of DAF-16/FOXO. DAF-16/FOXO activates transcription of genes that promote L1 arrest, dauer formation, stress resistance, and lifespan extension. DAF-16/FOXO is not the only effector of insulin-like <span class="highlight" style="background-color:">signaling</span> (Tullet et al. 2008). DAF-16/FOXO is also regulated by additional factors and <span class="highlight" style="background-color:">signaling</span> pathways (Landis and Murphy 2010). DAF-18/PTEN functions independently of DAF-16/FOXO in the germline (Fukuyama et al. 2006).</div></p></div><div class="aux"><div class="resc">L. Ryan Baugh. Genetics. 2013 July;194(3):539-555.</div><p class="links"><a class="dblinks" href="/pubmed/23824969" ref="ordinalpos=4132&amp;ncbi_uid=6336399&amp;link_uid=23824969">Citation</a><a class="dblinks" href="/pmc/articles/PMC3697962/" ref="ordinalpos=4132&amp;ncbi_uid=6336399&amp;link_uid=PMC3697962">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4133.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4330034/figure/f5-ijn-10-1045/" ref="ordinalpos=4133&amp;ncbi_uid=7204924&amp;link_uid=PMC4330034" href="/pmc/articles/PMC4330034/"><img src="/pmc/articles/instance/4330034/bin/ijn-10-1045Fig5.gif" src-large="/pmc/articles/instance/4330034/bin/ijn-10-1045Fig5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4330034/" ref="ordinalpos=4133&amp;ncbi_uid=7204924&amp;link_uid=PMC4330034" image-link="/pmc/articles/PMC4330034/figure/f5-ijn-10-1045/" class="imagepopup">Figure 5.  From: A novel chemosynthetic peptide with -sheet motif efficiently kills Klebsiella pneumoniae in a mouse model. </a></p><div class="supp"><p class="details"><div>IK8L modulated <i>Kp</i>-induced inflammatory reaction via the IL-6/JAK/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.<br /><b>Notes:</b> (<b>A</b><b>C</b>) ELISA was used to measure IL-6, TNF- and IL-1 in BAL fluids of mice 8 hours and 24 hours post infection, respectively. (<b>D</b>) Cell <span class="highlight" style="background-color:">signaling</span> proteins relating to the JAK2/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were evaluated in lung tissue by Western blotting analysis. (<b>E</b>) Expression and phosphorylation of <span class="highlight" style="background-color:">signaling</span> proteins relating to NF-B transcription factor were evaluated in lung tissue by Western blotting. (<b>F</b>) Cell <span class="highlight" style="background-color:">signaling</span> proteins relating to the AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were evaluated by Western blot. -actin was used as a loading control. Data are shown as mean  SEM of n=3 mice per group and are representative of three independent experiments. *<i>P</i>&lt;0.05; **<i>P</i>&lt;0.01; ***<i>P</i>&lt;0.001; one-way ANOVA (Tukeys post hoc).<br /><b>Abbreviations:</b> ANOVA, analysis of variance; BAL, bronchoalveolar lavage; ELISA, enzyme-linked immunosorbent assay; IL, interleukin; <i>Kp</i>, <i>Klebsiella pneumoniae</i>; SEM, standard error of the mean; TNF-, tumor necrosis factor alpha; p, phosphorylation.</div></p></div><div class="aux"><div class="resc">Shirui Tan, et al. Int J Nanomedicine. 2015;10:1045-1059.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4133&amp;ncbi_uid=7204924&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4330034/" ref="ordinalpos=4133&amp;ncbi_uid=7204924&amp;link_uid=PMC4330034">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4134.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3948775/figure/pone-0091077-g009/" ref="ordinalpos=4134&amp;ncbi_uid=5912420&amp;link_uid=PMC3948775" href="/pmc/articles/PMC3948775/"><img src="/pmc/articles/instance/3948775/bin/pone.0091077.g009.gif" src-large="/pmc/articles/instance/3948775/bin/pone.0091077.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3948775/" ref="ordinalpos=4134&amp;ncbi_uid=5912420&amp;link_uid=PMC3948775" image-link="/pmc/articles/PMC3948775/figure/pone-0091077-g009/" class="imagepopup">Figure 9. Hypothetical network showing the metabolic pathways affecting steroidogenesis and androstenone biosynthesis..  From: <span class="highlight" style="background-color:">Pathway</span> Based Analysis of Genes and Interactions Influencing Porcine Testis Samples from Boars with Divergent Androstenone Content in Back Fat. </a></p><div class="supp"><p class="details"><div>Legend: Interactions inside each <span class="highlight" style="background-color:">pathway</span> shows the significant interactions from analysis with genes as nodes and significant KEGG <span class="highlight" style="background-color:">pathway</span> interactions as edges. Blue diamond nodes: pathways that might be involved in steroidogenesis, but not found in results. Cyan colored nodes: chemical compound or molecules synthesized in pathways. Purple node: external stimulus in the form of hormone <span class="highlight" style="background-color:">signaling</span>. Grey solid edges: hypothetical interactions based on information from literature. Very light blue circular nodes: genes involved in significant interactions. Red dashed edges: interactions positive and significant in HA samples, positive in LA samples. Orange solid edges: interactions positive in HA samples, negative and significant in LA samples. Orange dashed edges: interactions negative in HA samples, negative and significant in LA samples. Dark green solid edges: interactions positive and significant in LA samples, negative in HA samples. Dark green dashed edges: interactions positive and significant in LA samples, positive in HA samples. Light green solid edges: interactions positive in LA samples, negative and significant in HA samples. Light green dashed edges: interactions negative in LA samples, negative and significant in HA samples. In HA animals, steroid synthesis and androstenone synthesis might be activated by cAMP-PKC <span class="highlight" style="background-color:">signaling</span> and further boosted by de novo cholesterol synthesis by virtue of an active fatty acid metabolism <span class="highlight" style="background-color:">pathway</span> and anti lipid peroxidation activity of members of glutathione metabolism <span class="highlight" style="background-color:">pathway</span>. In LA animals steroid hormone synthesis and androstenone synthesis could have been affected by weak cAMP-PKA <span class="highlight" style="background-color:">signaling</span> and inhibition of steroidogenesis by ceramides synthesized from sphingolipid metabolic <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Sudeep Sahadevan, et al. PLoS One. 2014;9(3):e91077.</div><p class="links"><a class="dblinks" href="/pubmed/24614349" ref="ordinalpos=4134&amp;ncbi_uid=5912420&amp;link_uid=24614349">Citation</a><a class="dblinks" href="/pmc/articles/PMC3948775/" ref="ordinalpos=4134&amp;ncbi_uid=5912420&amp;link_uid=PMC3948775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4135.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3129933/figure/F11/" ref="ordinalpos=4135&amp;ncbi_uid=3562575&amp;link_uid=PMC3129933" href="/pmc/articles/PMC3129933/"><img src="/pmc/articles/instance/3129933/bin/zh30061159460011.gif" src-large="/pmc/articles/instance/3129933/bin/zh30061159460011.jpg" alt="Fig. 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3129933/" ref="ordinalpos=4135&amp;ncbi_uid=3562575&amp;link_uid=PMC3129933" image-link="/pmc/articles/PMC3129933/figure/F11/" class="imagepopup">Fig. 11. From: Dietary fiber enhances TGF-? <span class="highlight" style="background-color:">signaling</span> and growth inhibition in the gut. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the potential interaction of dietary fiber with the TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Upon binding to its receptors (TRs) on the gut epithelial cells, TGF- activates (or phosphorylates) Smad3 and inhibits Id2 expression. Butyrate, derived from dietary fiber in the gut, acts on the gut epithelial cells to increase Smad3 expression and subsequently to inhibit Id2 expression, thereby enhancing TGF--induced growth inhibition through cell cycle arrest and apoptosis induction.</div></p></div><div class="aux"><div class="resc">Yanna Cao, et al. Am J Physiol Gastrointest Liver Physiol. 2011 July;301(1):G156-G164.</div><p class="links"><a class="dblinks" href="/pubmed/21454444" ref="ordinalpos=4135&amp;ncbi_uid=3562575&amp;link_uid=21454444">Citation</a><a class="dblinks" href="/pmc/articles/PMC3129933/" ref="ordinalpos=4135&amp;ncbi_uid=3562575&amp;link_uid=PMC3129933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4136.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2633800/figure/F1/" ref="ordinalpos=4136&amp;ncbi_uid=1705799&amp;link_uid=PMC2633800" href="/pmc/articles/PMC2633800/"><img src="/pmc/articles/instance/2633800/bin/cib0101_0053_fig001.gif" src-large="/pmc/articles/instance/2633800/bin/cib0101_0053_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2633800/" ref="ordinalpos=4136&amp;ncbi_uid=1705799&amp;link_uid=PMC2633800" image-link="/pmc/articles/PMC2633800/figure/F1/" class="imagepopup">Figure 1.  From: POLTERGEIST and PLL1 are required for stem cell function with potential roles in cell asymmetry and auxin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>(At left) A model for CLV <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> at the shoot meristem. Apically-generated CLV3 signal primarily activates CLV1 on the apical face of L3 stem cells within the shoot meristem. CLV1 in turn negatively regulates POL/PLL1 primarily on the apical portion of the cell. Upon periclinal division, the asymmetric distribution of POL/PLL1 drives apical/basal fates, including the activation of WUS in the basal daughter.</div></p></div><div class="aux"><div class="resc">Jennifer M Gagne, et al. Commun Integr Biol. 2008 Jul-Sep;1(1):53-55.</div><p class="links"><a class="dblinks" href="/pubmed/19704791" ref="ordinalpos=4136&amp;ncbi_uid=1705799&amp;link_uid=19704791">Citation</a><a class="dblinks" href="/pmc/articles/PMC2633800/" ref="ordinalpos=4136&amp;ncbi_uid=1705799&amp;link_uid=PMC2633800">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4137.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3000872/figure/F4/" ref="ordinalpos=4137&amp;ncbi_uid=2867690&amp;link_uid=PMC3000872" href="/pmc/articles/PMC3000872/"><img src="/pmc/articles/instance/3000872/bin/nihms192852f4.gif" src-large="/pmc/articles/instance/3000872/bin/nihms192852f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3000872/" ref="ordinalpos=4137&amp;ncbi_uid=2867690&amp;link_uid=PMC3000872" image-link="/pmc/articles/PMC3000872/figure/F4/" class="imagepopup">Figure 4. A <span class="highlight" style="background-color:">signaling</span> scheme describing the relationship between laminar flow-mediated ERK5/MEF2/KLF2/eNOS <span class="highlight" style="background-color:">pathway</span> and H2O2 or AGE-mediated ERK5-SUMOylation.  From: SUMO -- A post-translational modification with therapeutic potential?. </a></p><div class="aux"><div class="resc">Chang-Hoon Woo, et al. Curr Opin Pharmacol. ;10(2):146-155.</div><p class="links"><a class="dblinks" href="/pubmed/20079693" ref="ordinalpos=4137&amp;ncbi_uid=2867690&amp;link_uid=20079693">Citation</a><a class="dblinks" href="/pmc/articles/PMC3000872/" ref="ordinalpos=4137&amp;ncbi_uid=2867690&amp;link_uid=PMC3000872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4138.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2834553/figure/F12/" ref="ordinalpos=4138&amp;ncbi_uid=2737569&amp;link_uid=PMC2834553" href="/pmc/articles/PMC2834553/"><img src="/pmc/articles/instance/2834553/bin/asn0021042760012.gif" src-large="/pmc/articles/instance/2834553/bin/asn0021042760012.jpg" alt="Figure 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2834553/" ref="ordinalpos=4138&amp;ncbi_uid=2737569&amp;link_uid=PMC2834553" image-link="/pmc/articles/PMC2834553/figure/F12/" class="imagepopup">Figure 12. From: Parathyroid Hormone-Related Protein Promotes Epithelial-Mesenchymal Transition. </a></p><div class="supp"><p class="details"><div>Proposed mechanism for ERK1/2 activation as a key <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> targeted by PTHrP(1-36) through interaction with EGFR, TGF-, and VEGF in renal tubuloepithelial cells.</div></p></div><div class="aux"><div class="resc">Juan Antonio Ardura, et al. J Am Soc Nephrol. 2010 February;21(2):237-248.</div><p class="links"><a class="dblinks" href="/pubmed/19959711" ref="ordinalpos=4138&amp;ncbi_uid=2737569&amp;link_uid=19959711">Citation</a><a class="dblinks" href="/pmc/articles/PMC2834553/" ref="ordinalpos=4138&amp;ncbi_uid=2737569&amp;link_uid=PMC2834553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4139.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4220839/figure/fig2/" ref="ordinalpos=4139&amp;ncbi_uid=6812833&amp;link_uid=PMC4220839" href="/pmc/articles/PMC4220839/"><img src="/pmc/articles/instance/4220839/bin/cmi201457f2.gif" src-large="/pmc/articles/instance/4220839/bin/cmi201457f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4220839/" ref="ordinalpos=4139&amp;ncbi_uid=6812833&amp;link_uid=PMC4220839" image-link="/pmc/articles/PMC4220839/figure/fig2/" class="imagepopup">Figure 2.  From: A new cytosolic DNA-recognition <span class="highlight" style="background-color:">pathway</span> for DNA-induced inflammatory responses. </a></p><div class="supp"><p class="details"><div>Cytosolic DNA recognition <span class="highlight" style="background-color:">pathway</span> for IL-1 production. Upon recognition of DNA, AIM2 interacts with ASC, a common adaptor of the inflammasome, leading to the assembly of an inflammasome and the cleavage of caspase-1, which induces the processing of pro-IL-1 and the secretion of IL-1. Roth <i>et al.</i><sup>4</sup> demonstrated a new cytosolic DNA sensing <span class="highlight" style="background-color:">pathway</span> that was able to induce IL-1. The presence of DNA or a DNA virus in the cytosol induces the formation of dsDNARad50CARD9 <span class="highlight" style="background-color:">signaling</span> complexes, which control NF-B activation and the transcription of the gene encoding pro-IL-1. AIM2, absent in melanoma 2; CARD9, caspase recruitment domain-containing protein 9.</div></p></div><div class="aux"><div class="resc">Jun Meng, et al. Cell Mol Immunol. 2014 November;11(6):506-509.</div><p class="links"><a class="dblinks" href="/pubmed/25027968" ref="ordinalpos=4139&amp;ncbi_uid=6812833&amp;link_uid=25027968">Citation</a><a class="dblinks" href="/pmc/articles/PMC4220839/" ref="ordinalpos=4139&amp;ncbi_uid=6812833&amp;link_uid=PMC4220839">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4140.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3906024/figure/pone-0079987-g001/" ref="ordinalpos=4140&amp;ncbi_uid=5774709&amp;link_uid=PMC3906024" href="/pmc/articles/PMC3906024/"><img src="/pmc/articles/instance/3906024/bin/pone.0079987.g001.gif" src-large="/pmc/articles/instance/3906024/bin/pone.0079987.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3906024/" ref="ordinalpos=4140&amp;ncbi_uid=5774709&amp;link_uid=PMC3906024" image-link="/pmc/articles/PMC3906024/figure/pone-0079987-g001/" class="imagepopup">Figure 1. Phosphoprofiling of the Proximal BCR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Uncovers High Variability in BCR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Behavior in CLL B cells..  From: Multidimensional Single-Cell Analysis of BCR <span class="highlight" style="background-color:">Signaling</span> Reveals Proximal Activation Defect As a Hallmark of Chronic Lymphocytic Leukemia B Cells. </a></p><div class="supp"><p class="details"><div>A. Simplified diagram of the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components investigated in this study. B. Representative histograms of phosphoresponses of B- cells for two CLL patients (a high and a low responder) and one healthy individual. CLL and healthy B cells were gated as described in the methods section. Stimulated samples (anti-IgM+H<sub>2</sub>O<sub>2</sub>) are indicated by a black line, unstimulated samples are shaded grey. C. Contour maps of the average B cell population: For each cohort, CLL and healthy, the cellular phosphoresponse fluorescence intensity values are averaged and viewed two dimensionally. Bimodality in the phosphoresponse of CLL B cells can be seen for the pairwise pPLC2 vs. pSYK) (see also Supp. Fig. S2 for other combinations of phosphoresponses). Healthy B cells show modest variability within the B-cell population, while the CLL B cells can be distinguished by their all-or-none response. D. Following IgM crosslinking, phosphoresponses are shown as the percentage of responding cells for each patient over an unstimulated matched control. This view of the wide, continuous range of each CLL patients' phosphoprofile highlights the high variability in the behavior of the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Only the pPLC2 response showed statistical significance when comparing healthy to CLL cells (***p&lt;0.0001).</div></p></div><div class="aux"><div class="resc">M. Lia Palomba, et al. PLoS One. 2014;9(1):e79987.</div><p class="links"><a class="dblinks" href="/pubmed/24489640" ref="ordinalpos=4140&amp;ncbi_uid=5774709&amp;link_uid=24489640">Citation</a><a class="dblinks" href="/pmc/articles/PMC3906024/" ref="ordinalpos=4140&amp;ncbi_uid=5774709&amp;link_uid=PMC3906024">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4141.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3273902/figure/F3/" ref="ordinalpos=4141&amp;ncbi_uid=3806128&amp;link_uid=PMC3273902" href="/pmc/articles/PMC3273902/"><img src="/pmc/articles/instance/3273902/bin/aging-03-1213-g003.gif" src-large="/pmc/articles/instance/3273902/bin/aging-03-1213-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3273902/" ref="ordinalpos=4141&amp;ncbi_uid=3806128&amp;link_uid=PMC3273902" image-link="/pmc/articles/PMC3273902/figure/F3/" class="imagepopup">Figure 3.  From: GSTT1 is upregulated by oxidative stress through p38-MK2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human granulosa cells: possible association with mitochondrial activity. </a></p><div class="supp"><p class="details"><div>Depletion of GSTT1 inactivates the p38MK2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. KGN cells (wildtype, Lamin GSTT1 or p38 cells) were treated with or without H<sub>2</sub>O<sub>2</sub> at 200 M for 24 h before investigation of the expression of GSTT1 and activation of p38 and MK2 by immunofluorescence analysis (<b>A</b>). The primary antibodies against GSTT1, phosphorylated p38 and phosphorylated MK2 were probed with anti-rabbit IgGAlexa488 (Green). Cells were counterstained with Hoechst 33342 at 10 M (Blue). Magnification: 200. Bar graphs represent the mean fluorescence intensity per cell  SEM (<b>B</b>). Oneway ANOVA: (<b>B</b>, GSTT1) P &lt; 0.001, (<b>B</b>, p-p38) P &lt; 0.05, (<b>B</b>, p-MK2) P &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Megumu Ito, et al. Aging (Albany NY). 2011 December;3(12):1213-1223.</div><p class="links"><a class="dblinks" href="/pubmed/22207314" ref="ordinalpos=4141&amp;ncbi_uid=3806128&amp;link_uid=22207314">Citation</a><a class="dblinks" href="/pmc/articles/PMC3273902/" ref="ordinalpos=4141&amp;ncbi_uid=3806128&amp;link_uid=PMC3273902">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4142.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2928642/figure/F5/" ref="ordinalpos=4142&amp;ncbi_uid=3130118&amp;link_uid=PMC2928642" href="/pmc/articles/PMC2928642/"><img src="/pmc/articles/instance/2928642/bin/zh00071063140005.gif" src-large="/pmc/articles/instance/2928642/bin/zh00071063140005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2928642/" ref="ordinalpos=4142&amp;ncbi_uid=3130118&amp;link_uid=PMC2928642" image-link="/pmc/articles/PMC2928642/figure/F5/" class="imagepopup">Fig. 5. From: Phosphatidic acid mediates activation of mTORC1 through the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>LPA, but not leucine, activates mTORC1 <span class="highlight" style="background-color:">signaling</span> through the MAP kinase <span class="highlight" style="background-color:">pathway</span>. Cells were incubated in medium containing 0.5% serum without leucine for 2 h. Leucine, LPA, PD-98059, and/or U-0126 was added to the medium, and S6K1 phosphorylation on Thr<sup>389</sup> and ERK1/2 phosphorylation on Thr<sup>202</sup>/Tyr<sup>204</sup> were assessed as described in Fig. 3 and 4 legends, respectively. <i>A</i> and <i>B</i>: means  SE of 5 experiments. In each experiment, 2 samples per condition were independently analyzed. <i>C</i> and <i>D</i>: means  SE of 3 experiments. In each experiment, 3 samples per condition were independently analyzed. Representative blots are shown. *<i>P</i> &lt; 0.02 vs. cells treated in medium containing 0.5% serum without leucine. <i>P</i> &lt; 0.011 vs. cells treated with LPA alone.</div></p></div><div class="aux"><div class="resc">Jeremiah N. Winter, et al. Am J Physiol Cell Physiol. 2010 August;299(2):C335-C344.</div><p class="links"><a class="dblinks" href="/pubmed/20427710" ref="ordinalpos=4142&amp;ncbi_uid=3130118&amp;link_uid=20427710">Citation</a><a class="dblinks" href="/pmc/articles/PMC2928642/" ref="ordinalpos=4142&amp;ncbi_uid=3130118&amp;link_uid=PMC2928642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4143.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3143653/figure/F6/" ref="ordinalpos=4143&amp;ncbi_uid=3629501&amp;link_uid=PMC3143653" href="/pmc/articles/PMC3143653/"><img src="/pmc/articles/instance/3143653/bin/zbc0351172370006.gif" src-large="/pmc/articles/instance/3143653/bin/zbc0351172370006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3143653/" ref="ordinalpos=4143&amp;ncbi_uid=3629501&amp;link_uid=PMC3143653" image-link="/pmc/articles/PMC3143653/figure/F6/" class="imagepopup">FIGURE 6. From: Receptor-independent Protein Kinase C? (PKC?) <span class="highlight" style="background-color:">Signaling</span> by Calpain-generated Free Catalytic Domains Induces HDAC5 Nuclear Export and Regulates Cardiac Transcription. </a></p><div class="supp"><p class="details"><div><b>Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis of PKC-CT regulated, MEF2-targetted cardiac mRNAs.</b> Ingenuity Pathways Analysis software was used to depict potential <span class="highlight" style="background-color:">signaling</span> pathways involving the 15 putative MEF2 targets among PKC-CT up-regulated mRNAs. Nine of 15 mRNAs were included in the network. <i>Lines with arrowheads</i>, molecule acts on a target; <i>lines without arrowheads</i>, binding only. <i>Solid lines</i>, direct interaction; <i>dotted lines</i>, indirect interaction. <i>Blue</i> background, regulated putative MEF2 targets; <i>white</i> background, other <span class="highlight" style="background-color:">signaling</span> network members.</div></p></div><div class="aux"><div class="resc">Yan Zhang, et al. J Biol Chem. 2011 July 29;286(30):26943-26951.</div><p class="links"><a class="dblinks" href="/pubmed/21642422" ref="ordinalpos=4143&amp;ncbi_uid=3629501&amp;link_uid=21642422">Citation</a><a class="dblinks" href="/pmc/articles/PMC3143653/" ref="ordinalpos=4143&amp;ncbi_uid=3629501&amp;link_uid=PMC3143653">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4144.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3535741/figure/fig1/" ref="ordinalpos=4144&amp;ncbi_uid=4529861&amp;link_uid=PMC3535741" href="/pmc/articles/PMC3535741/"><img src="/pmc/articles/instance/3535741/bin/DRT2012-752563.001.gif" src-large="/pmc/articles/instance/3535741/bin/DRT2012-752563.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3535741/" ref="ordinalpos=4144&amp;ncbi_uid=4529861&amp;link_uid=PMC3535741" image-link="/pmc/articles/PMC3535741/figure/fig1/" class="imagepopup">Figure 1.  From: Roles of PI3K/AKT/GSK3/mTOR <span class="highlight" style="background-color:">Pathway</span> in Cell <span class="highlight" style="background-color:">Signaling</span> of Mental Illnesses. </a></p><div class="supp"><p class="details"><div>Schematic representation of PI3K/AKT/GSK3/mTOR <span class="highlight" style="background-color:">signaling</span>. Examples of molecules known to act on the regulatory pathways are shown. Note that some critical pathways have been omitted for clarity.</div></p></div><div class="aux"><div class="resc">Yasuko Kitagishi, et al. Depress Res Treat. 2012;2012:752563.</div><p class="links"><a class="dblinks" href="/pubmed/23320155" ref="ordinalpos=4144&amp;ncbi_uid=4529861&amp;link_uid=23320155">Citation</a><a class="dblinks" href="/pmc/articles/PMC3535741/" ref="ordinalpos=4144&amp;ncbi_uid=4529861&amp;link_uid=PMC3535741">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4145.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3599951/figure/F2/" ref="ordinalpos=4145&amp;ncbi_uid=4716729&amp;link_uid=PMC3599951" href="/pmc/articles/PMC3599951/"><img src="/pmc/articles/instance/3599951/bin/1748-717X-8-46-2.gif" src-large="/pmc/articles/instance/3599951/bin/1748-717X-8-46-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3599951/" ref="ordinalpos=4145&amp;ncbi_uid=4716729&amp;link_uid=PMC3599951" image-link="/pmc/articles/PMC3599951/figure/F2/" class="imagepopup">Figure 2.  From: Molecular basis of 'hypoxic' breast cancer cell radio-sensitization: phytochemicals converge on radiation induced Rel <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Histograms showing attenuation and/or complete suppression of 59 IR-induced NFB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes with CUR, EF24, NLE, GEN, RES or RSE treatment in hypoxic human breast adenocarcinoma cells.</b> QPCR profiling was performed using human NFB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> profiler (Realtimeprimers.com, Elkins Park, PA) containing 88 genes and 8 internal controls. The <sup>ct</sup> values calculated by normalizing the gene expression levels to the expression of the housekeeping genes were compared between groups and expressed as fold change.</div></p></div><div class="aux"><div class="resc">Sheeja Aravindan, et al. Radiat Oncol. 2013;8:46-46.</div><p class="links"><a class="dblinks" href="/pubmed/23452621" ref="ordinalpos=4145&amp;ncbi_uid=4716729&amp;link_uid=23452621">Citation</a><a class="dblinks" href="/pmc/articles/PMC3599951/" ref="ordinalpos=4145&amp;ncbi_uid=4716729&amp;link_uid=PMC3599951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4146.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2688308/figure/F1/" ref="ordinalpos=4146&amp;ncbi_uid=1855021&amp;link_uid=PMC2688308" href="/pmc/articles/PMC2688308/"><img src="/pmc/articles/instance/2688308/bin/psb0406_0545_fig001.gif" src-large="/pmc/articles/instance/2688308/bin/psb0406_0545_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2688308/" ref="ordinalpos=4146&amp;ncbi_uid=1855021&amp;link_uid=PMC2688308" image-link="/pmc/articles/PMC2688308/figure/F1/" class="imagepopup">Figure 1.  From: Implication of chlorophyll biosynthesis on chloroplast-to-nucleus retrograde <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) Schematic <span class="highlight" style="background-color:">pathway</span> of tetrapyrrole biosynthesis in higher plants. The enzymes and <i>gun</i> mutants discussed in the text are marked in the <span class="highlight" style="background-color:">pathway</span>. Glu, glutamate; ALA, 5-aminolevulinic acid; Proto, protoporhyrin IX; MgProto, Mg-protoporhyrin; MgProtoME, Mg-protoporhyrin monomethyl ester; Pchlide, divinyl protochlorophyllide; HEMA1, glytamyl-tRNA reductase; MgCh, Mg-chelatase; MgMT, MgProto methyltransferase; Mg-Cy, Mg-protoporphyrin IX-monomethyl ester cyclase. (B) The tentative model for chloroplast retrograde <span class="highlight" style="background-color:">signaling</span> involving intermediates of chlorophyll biosynthesis. The porphyrin intermediates between protoporphyrin IX and protochlorophyllide may initiate the retrograde signal controlling the gene expression in nucleus. Alternatively, the intermediates may induce production of ROS that in turn activates the <span class="highlight" style="background-color:">signaling</span> cascade. For other details, see the text.</div></p></div><div class="aux"><div class="resc">Eevi Rintamki, et al. Plant Signal Behav. 2009 June;4(6):545-547.</div><p class="links"><a class="dblinks" href="/pubmed/19816147" ref="ordinalpos=4146&amp;ncbi_uid=1855021&amp;link_uid=19816147">Citation</a><a class="dblinks" href="/pmc/articles/PMC2688308/" ref="ordinalpos=4146&amp;ncbi_uid=1855021&amp;link_uid=PMC2688308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4147.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4148637/figure/F2/" ref="ordinalpos=4147&amp;ncbi_uid=6609799&amp;link_uid=PMC4148637" href="/pmc/articles/PMC4148637/"><img src="/pmc/articles/instance/4148637/bin/fnagi-06-00230-g002.gif" src-large="/pmc/articles/instance/4148637/bin/fnagi-06-00230-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4148637/" ref="ordinalpos=4147&amp;ncbi_uid=6609799&amp;link_uid=PMC4148637" image-link="/pmc/articles/PMC4148637/figure/F2/" class="imagepopup">Figure 2.  From: The Intriguing Regulators of Muscle Mass in Sarcopenia and Muscular Dystrophy. </a></p><div class="supp"><p class="details"><div><b>(A,B)</b> The adaptative changes of positive and negative regulators for muscle mass in sarcopenia and muscular dystrophy. Both sarcopenia and muscular dystrophy exhibit the marked defect of autophagy-dependent <span class="highlight" style="background-color:">signaling</span> possibly the latter due to hyperactivation of Akt/mTOR/p70S6K <span class="highlight" style="background-color:">pathway</span>. Lower activation of SRF-dependent <span class="highlight" style="background-color:">signaling</span> has been commonly recognized in these symptoms. Ubiquitinproteasome system (Atrogin-1 and MuRF-1) would not regulate muscle atrophy in the case of sarcopenia. It remains to be elucidated whether myostatinSmad <span class="highlight" style="background-color:">pathway</span> regulates to sarcopenic symptom and/or muscular dystrophy.</div></p></div><div class="aux"><div class="resc">Kunihiro Sakuma, et al. Front Aging Neurosci. 2014;6:230.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4147&amp;ncbi_uid=6609799&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4148637/" ref="ordinalpos=4147&amp;ncbi_uid=6609799&amp;link_uid=PMC4148637">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4148.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4086889/figure/pone-0101384-g004/" ref="ordinalpos=4148&amp;ncbi_uid=6373888&amp;link_uid=PMC4086889" href="/pmc/articles/PMC4086889/"><img src="/pmc/articles/instance/4086889/bin/pone.0101384.g004.gif" src-large="/pmc/articles/instance/4086889/bin/pone.0101384.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4086889/" ref="ordinalpos=4148&amp;ncbi_uid=6373888&amp;link_uid=PMC4086889" image-link="/pmc/articles/PMC4086889/figure/pone-0101384-g004/" class="imagepopup">Figure 4. PI3K/Akt <span class="highlight" style="background-color:">signaling</span> and differential gene expression analysis..  From: Genetic Dynamic Analysis of the Influenza A H5N1 NS1 Gene in China. </a></p><div class="supp"><p class="details"><div>(A) Schematic diagram for the regulation of PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) Overview of temporal differential gene expression in rhesus macaques infected with A/Anhui/2/2005 (H5N1) at different time points. A color scale indicating expression levels for the heat map is shown at the top right. Genes exhibited up-regulated expression pattern over time are highlighted in red.</div></p></div><div class="aux"><div class="resc">Kaifa Wei, et al. PLoS One. 2014;9(7):e101384.</div><p class="links"><a class="dblinks" href="/pubmed/25003973" ref="ordinalpos=4148&amp;ncbi_uid=6373888&amp;link_uid=25003973">Citation</a><a class="dblinks" href="/pmc/articles/PMC4086889/" ref="ordinalpos=4148&amp;ncbi_uid=6373888&amp;link_uid=PMC4086889">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4149.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4279680/figure/Fig7/" ref="ordinalpos=4149&amp;ncbi_uid=7005521&amp;link_uid=PMC4279680" href="/pmc/articles/PMC4279680/"><img src="/pmc/articles/instance/4279680/bin/13046_2014_102_Fig7_HTML.gif" src-large="/pmc/articles/instance/4279680/bin/13046_2014_102_Fig7_HTML.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4279680/" ref="ordinalpos=4149&amp;ncbi_uid=7005521&amp;link_uid=PMC4279680" image-link="/pmc/articles/PMC4279680/figure/Fig7/" class="imagepopup">Figure 7.  From: Pegylated arginine deiminase synergistically increases the cytotoxicity of gemcitabine in human pancreatic cancer. </a></p><div class="supp"><p class="details"><div> <b>Graphic representation of regulation of RRM2 by E2F-1.</b> Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of regulation of RRM2 by E2F-1 with subsequent upstream inhibition or stimulation by gemcitabine or arginine.</div></p></div><div class="aux"><div class="resc">Rouzbeh Daylami, et al. J Exp Clin Cancer Res. 2014;33(1):102.</div><p class="links"><a class="dblinks" href="/pubmed/25499121" ref="ordinalpos=4149&amp;ncbi_uid=7005521&amp;link_uid=25499121">Citation</a><a class="dblinks" href="/pmc/articles/PMC4279680/" ref="ordinalpos=4149&amp;ncbi_uid=7005521&amp;link_uid=PMC4279680">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4150.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2781329/figure/F4/" ref="ordinalpos=4150&amp;ncbi_uid=2545085&amp;link_uid=PMC2781329" href="/pmc/articles/PMC2781329/"><img src="/pmc/articles/instance/2781329/bin/zh20110956230004.gif" src-large="/pmc/articles/instance/2781329/bin/zh20110956230004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2781329/" ref="ordinalpos=4150&amp;ncbi_uid=2545085&amp;link_uid=PMC2781329" image-link="/pmc/articles/PMC2781329/figure/F4/" class="imagepopup">Fig. 4. From: Heme oxygenase: the key to renal function regulation. </a></p><div class="supp"><p class="details"><div>Schematic diagram for HO-1 modulation of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and adiponectin, EET levels, and vascular function. LKB1, serine/threonine kinase; C/EBPa, enhancer binding protein-.</div></p></div><div class="aux"><div class="resc">Nader G. Abraham, et al. Am J Physiol Renal Physiol. 2009 November;297(5):F1137-F1152.</div><p class="links"><a class="dblinks" href="/pubmed/19570878" ref="ordinalpos=4150&amp;ncbi_uid=2545085&amp;link_uid=19570878">Citation</a><a class="dblinks" href="/pmc/articles/PMC2781329/" ref="ordinalpos=4150&amp;ncbi_uid=2545085&amp;link_uid=PMC2781329">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4151.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC178345/figure/f5/" ref="ordinalpos=4151&amp;ncbi_uid=203067&amp;link_uid=PMC178345" href="/pmc/articles/PMC178345/"><img src="/pmc/articles/instance/178345/bin/ek0430035005.gif" src-large="/pmc/articles/instance/178345/bin/ek0430035005.jpg" alt="FIG. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC178345/" ref="ordinalpos=4151&amp;ncbi_uid=203067&amp;link_uid=PMC178345" image-link="/pmc/articles/PMC178345/figure/f5/" class="imagepopup">FIG. 5. From: A Yeast under Cover: the Capsule of Cryptococcus neoformans. </a></p><div class="supp"><p class="details"><div>cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the regulation of the capsule. See text for explanation. Red symbols, Pkr1p; black circles, Pka1p; blue-gray circles, cAMP.</div></p></div><div class="aux"><div class="resc">Indrani Bose, et al. Eukaryot Cell. 2003 August;2(4):655-663.</div><p class="links"><a class="dblinks" href="/pubmed/12912884" ref="ordinalpos=4151&amp;ncbi_uid=203067&amp;link_uid=12912884">Citation</a><a class="dblinks" href="/pmc/articles/PMC178345/" ref="ordinalpos=4151&amp;ncbi_uid=203067&amp;link_uid=PMC178345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4152.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3393825/figure/F2/" ref="ordinalpos=4152&amp;ncbi_uid=5764358&amp;link_uid=PMC3393825" href="/pmc/articles/PMC3393825/"><img src="/pmc/articles/instance/3393825/bin/nihms367853f2.gif" src-large="/pmc/articles/instance/3393825/bin/nihms367853f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3393825/" ref="ordinalpos=4152&amp;ncbi_uid=5764358&amp;link_uid=PMC3393825" image-link="/pmc/articles/PMC3393825/figure/F2/" class="imagepopup">Figure 2. Model of the C. elegans UPRmt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: <span class="highlight" style="background-color:">Signaling</span> the Mitochondrial Unfolded Protein Response. </a></p><div class="supp"><p class="details"><div>UPR<sup>mt</sup> <span class="highlight" style="background-color:">signaling</span> is initiated as unfolded proteins accumulate beyond the resident chaperone folding capacity. To prevent protein aggregation, unfolded or misfolded proteins are degraded to peptides by the quality control protease ClpP in the mitochondrial matrix [68]. HAF-1-mediated peptide efflux leads to activation of the bZip transcription factor ATFS-1 that accumulates in the nucleus. HAF-1-mediated peptide efflux is also required for the accumulation of the ubiquitin-like protein UBL-5 which complexes with the transcription factor DVE-1 [69]. ATFS-1 and DVE-1/UBL-5 then cooperatively induce the expression of mitochondrial chaperone genes including <i>Hsp60</i> and <i>mtHsp70</i> in order to restore protein homeostasis.</div></p></div><div class="aux"><div class="resc">Mark W. Pellegrino, et al. Biochim Biophys Acta. ;1833(2):410-416.</div><p class="links"><a class="dblinks" href="/pubmed/22445420" ref="ordinalpos=4152&amp;ncbi_uid=5764358&amp;link_uid=22445420">Citation</a><a class="dblinks" href="/pmc/articles/PMC3393825/" ref="ordinalpos=4152&amp;ncbi_uid=5764358&amp;link_uid=PMC3393825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4153.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3564524/figure/F10/" ref="ordinalpos=4153&amp;ncbi_uid=4805344&amp;link_uid=PMC3564524" href="/pmc/articles/PMC3564524/"><img src="/pmc/articles/instance/3564524/bin/210fig10.gif" src-large="/pmc/articles/instance/3564524/bin/210fig10.jpg" alt="FIGURE 10:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3564524/" ref="ordinalpos=4153&amp;ncbi_uid=4805344&amp;link_uid=PMC3564524" image-link="/pmc/articles/PMC3564524/figure/F10/" class="imagepopup">FIGURE 10:.  From: Apolipoprotein(a) stimulates nuclear translocation of ?-catenin: a novel pathogenic mechanism for lipoprotein(a). </a></p><div class="supp"><p class="details"><div>Novel <span class="highlight" style="background-color:">signaling</span> pathways involved in apo(a)-mediated effects on vascular endothelial cells. Apo(a), through its strong LBS in KIV<sub>10</sub>, binds to a putative receptor (possibly a complex of CD36 and TLR2), resulting in activation of a Src- and Rho/Rho kinasedependent <span class="highlight" style="background-color:">pathway</span> to increased VE-cadherin/-catenin complex dissociation, leading to an increase in vascular permeability in cultured EC. Apo(a) also induces a PI3K/Akt/GSK-3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, independent of the Rho/Rho kinase <span class="highlight" style="background-color:">pathway</span>, to increase -catenin nuclear translocation, and hence COX-2 expression, and PGE<sub>2</sub> secretion. Furthermore, apo(a) mediates loss of cellcell contact and stimulates -catenin translocation by disrupting the VE-cadherin/-catenin complex through inhibition of PTEN.</div></p></div><div class="aux"><div class="resc">Taewoo Cho, et al. Mol Biol Cell. 2013 February 1;24(3):210-221.</div><p class="links"><a class="dblinks" href="/pubmed/23243000" ref="ordinalpos=4153&amp;ncbi_uid=4805344&amp;link_uid=23243000">Citation</a><a class="dblinks" href="/pmc/articles/PMC3564524/" ref="ordinalpos=4153&amp;ncbi_uid=4805344&amp;link_uid=PMC3564524">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4154.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3936199/figure/F3/" ref="ordinalpos=4154&amp;ncbi_uid=5851973&amp;link_uid=PMC3936199" href="/pmc/articles/PMC3936199/"><img src="/pmc/articles/instance/3936199/bin/nihms-551241-f0003.gif" src-large="/pmc/articles/instance/3936199/bin/nihms-551241-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3936199/" ref="ordinalpos=4154&amp;ncbi_uid=5851973&amp;link_uid=PMC3936199" image-link="/pmc/articles/PMC3936199/figure/F3/" class="imagepopup">Figure 3. Regulation of hepcidin by BMP/SMAD, inflammatory and hypoxia/erythropoietic <span class="highlight" style="background-color:">signaling</span> in the liver.  From: Iron homeostasis in the liver. </a></p><div class="supp"><p class="details"><div>Three major pathways are critical for regulating basal and stimuli-induced hepcidin expression. Binding of iron containing Tf to Tfr1 causes a dissociation of Tfr1-High FE (HFE) complex and an interaction of HFE with Tfr2. Increased stabilization of Tfr2 increases BMP6 mediated phosphorylation of SMAD1/5/ 8 and recruitment of SMAD 1/5/8 and SMAD4 to the hepcidin proximal promoter. BMP/SMAD <span class="highlight" style="background-color:">signaling</span> is the major <span class="highlight" style="background-color:">pathway</span> by which hepcidin expression is coordinated to meet systemic iron requirements. Activation of hepcidin by inflammation is thought to act independently of the BMP/SMAD <span class="highlight" style="background-color:">pathway</span>. The best-studied mechanism is via the pro-inflammatory mediator IL-6. Binding of IL-6 to its receptor IL-6 receptor (IL-6R) initiates activation of the JAK-STAT3 <span class="highlight" style="background-color:">pathway</span>. STAT3 binds directly to the proximal promoter to increase hepcidin expression. Hypoxia and erythropoiesis are inhibitors of hepcidin expression and these are the least understood pathways by which hepcidin expression is regulated. Hypoxia and erythropoiesis have been shown to inhibit hepcidin expression via direct binding of HIF to the proximal promoter, an EPO-EPO receptor (EPOR) mediated decrease in C/EBP expression, and through increase in an unknown erythroid derived factor which signals through an undefined <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Erik R Anderson, et al. Compr Physiol. ;3(1):315-330.</div><p class="links"><a class="dblinks" href="/pubmed/23720289" ref="ordinalpos=4154&amp;ncbi_uid=5851973&amp;link_uid=23720289">Citation</a><a class="dblinks" href="/pmc/articles/PMC3936199/" ref="ordinalpos=4154&amp;ncbi_uid=5851973&amp;link_uid=PMC3936199">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4155.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2843459/figure/F11-8513/" ref="ordinalpos=4155&amp;ncbi_uid=2854201&amp;link_uid=PMC2843459" href="/pmc/articles/PMC2843459/"><img src="/pmc/articles/instance/2843459/bin/zjh0041085130011.gif" src-large="/pmc/articles/instance/2843459/bin/zjh0041085130011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2843459/" ref="ordinalpos=4155&amp;ncbi_uid=2854201&amp;link_uid=PMC2843459" image-link="/pmc/articles/PMC2843459/figure/F11-8513/" class="imagepopup">Figure 11.  From: Cooperation of TLR2 with MyD88, PI3K, and Rac1 in Lipoteichoic Acid-Induced cPLA2/COX-2-Dependent Airway Inflammatory Responses. </a></p><div class="supp"><p class="details"><div>Schematic diagram illustrating the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in LTA-induced cPLA<sub>2</sub>/COX-2dependent airway inflammation. LTA activates the TLR2/MyD88/PI3K/Rac1/Akt <span class="highlight" style="background-color:">pathway</span> to enhance ERK1/2, p38 MAPK, and JNK1/2 phosphorylation, which in turn initiates the activation of NF-B and ultimately induces cPLA<sub>2</sub>/COX-2dependent PGE<sub>2</sub> and IL-6 generation in HTSMCs.</div></p></div><div class="aux"><div class="resc">I-Ta Lee, et al. Am J Pathol. 2010 April;176(4):1671-1684.</div><p class="links"><a class="dblinks" href="/pubmed/20167866" ref="ordinalpos=4155&amp;ncbi_uid=2854201&amp;link_uid=20167866">Citation</a><a class="dblinks" href="/pmc/articles/PMC2843459/" ref="ordinalpos=4155&amp;ncbi_uid=2854201&amp;link_uid=PMC2843459">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4156.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4281466/figure/F1/" ref="ordinalpos=4156&amp;ncbi_uid=7010540&amp;link_uid=PMC4281466" href="/pmc/articles/PMC4281466/"><img src="/pmc/articles/instance/4281466/bin/nihms651538f1.gif" src-large="/pmc/articles/instance/4281466/bin/nihms651538f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4281466/" ref="ordinalpos=4156&amp;ncbi_uid=7010540&amp;link_uid=PMC4281466" image-link="/pmc/articles/PMC4281466/figure/F1/" class="imagepopup">Figure 1.  From: TGF-beta-dependent and -independent roles of STRAP in cancer. </a></p><div class="supp"><p class="details"><div>The TGF-beta <span class="highlight" style="background-color:">signaling</span> pathways. TGF-beta <span class="highlight" style="background-color:">signaling</span> can be propagated through Smad-dependent and Smad-independent pathways. In the Smad-dependent <span class="highlight" style="background-color:">pathway</span>, the activated TGF-beta receptor complex phosphorylates the R-Smads, Smad-2 and Smad-3. The phosphorylated R-Smads associate with the common Smad-4 and translocate to the nucleus where they function as transcriptional activators and repressors of gene expression. The inhibitory Smad, Smad-7, inhibits activation of the R-Smads by associating with TBRI. The WD40 domain protein, STRAP, also functions as an inhibitor of Smad-dependent <span class="highlight" style="background-color:">signaling</span> by associating with Smad-7 and TBRI. In addition to Smad activation, The TGF beta receptor complex can induce <span class="highlight" style="background-color:">signaling</span> through the Ras, RhoA, TAK1, and PI3K pathways. TGF-beta mediated activation of these non-Smad pathways has been associated with cellular transformation, proliferation, EMT, and migration.</div></p></div><div class="aux"><div class="resc">Jennifer Elisabeth Reiner, et al. Front Biosci (Landmark Ed). ;16:105-115.</div><p class="links"><a class="dblinks" href="/pubmed/21196161" ref="ordinalpos=4156&amp;ncbi_uid=7010540&amp;link_uid=21196161">Citation</a><a class="dblinks" href="/pmc/articles/PMC4281466/" ref="ordinalpos=4156&amp;ncbi_uid=7010540&amp;link_uid=PMC4281466">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4157.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3188786/figure/F1/" ref="ordinalpos=4157&amp;ncbi_uid=3287606&amp;link_uid=PMC3188786" href="/pmc/articles/PMC3188786/"><img src="/pmc/articles/instance/3188786/bin/fnana-05-00055-g001.gif" src-large="/pmc/articles/instance/3188786/bin/fnana-05-00055-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3188786/" ref="ordinalpos=4157&amp;ncbi_uid=3287606&amp;link_uid=PMC3188786" image-link="/pmc/articles/PMC3188786/figure/F1/" class="imagepopup">Figure 1.  From: Huntington's Disease and Striatal <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Altered striatal <span class="highlight" style="background-color:">signaling</span> pathways in HD</b>. Afferent corticostriatal and nigro-striatal projections modulate striatal <span class="highlight" style="background-color:">signaling</span>, which is impaired in Huntingtons Disease (HD). Neurotrophic factor BDNF release from cortical afferences is decreased in cortico-striatal synapses as a consequence of Exp-HTT-mediated down-regulation of <i>bdnf</i> transcription and axonal transport in cortical neurons. A shift from synaptic to extrasynaptic NMDAR-dependent <span class="highlight" style="background-color:">signaling</span> participates to striatal neurons to death. In a physiological condition, calcium influx through synaptic NMDAR promotes activation of the MAPkinase/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> along with its nuclear target, the MSK-1 protein, which phosphorylates the transcription factor CREB. MSK-1 activation promotes chromatin remodeling, which is crucial for CREB-mediated <i>pgc-1</i> transcription, a key gene involved in mitochondria biogenesis. Synaptic NMDARs also promote formation of non-toxic Exp-HTT inclusion via TRIC. In HD, localization and activity of extrasynaptic NMDAR are enhanced by Exp-HTT, which promotes neuronal cell injury and death. The toxic effect of extrasynaptic NMDAR is partly due to (i) the upregulation of Rhes expression that disaggregates the non-toxic Exp-HTT inclusions (ii) to an impairment of mitochondrial functions and (iii) to a decrease of the ERK/MSK-1/CREB <span class="highlight" style="background-color:">signaling</span> module to the <i>pgc-1</i> promoter. Dopamine release from nigro-striatal inputs promotes oxidative stress via the production of reactive oxygen species (ROS), which potentiates activation of the pro-apoptotic JNK <span class="highlight" style="background-color:">pathway</span> induced by Exp-HTT. In addition, Dopamine potentiates Exp-HTT-mediated striatal neurons death through activation of the Rho/ROCK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream D2R.</div></p></div><div class="aux"><div class="resc">Emmanuel Roze, et al. Front Neuroanat. 2011;5:55.</div><p class="links"><a class="dblinks" href="/pubmed/22007160" ref="ordinalpos=4157&amp;ncbi_uid=3287606&amp;link_uid=22007160">Citation</a><a class="dblinks" href="/pmc/articles/PMC3188786/" ref="ordinalpos=4157&amp;ncbi_uid=3287606&amp;link_uid=PMC3188786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4158.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3047783/figure/F11/" ref="ordinalpos=4158&amp;ncbi_uid=2885977&amp;link_uid=PMC3047783" href="/pmc/articles/PMC3047783/"><img src="/pmc/articles/instance/3047783/bin/self0103_0192_fig011.gif" src-large="/pmc/articles/instance/3047783/bin/self0103_0192_fig011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3047783/" ref="ordinalpos=4158&amp;ncbi_uid=2885977&amp;link_uid=PMC3047783" image-link="/pmc/articles/PMC3047783/figure/F11/" class="imagepopup">Figure 11.  From: The SCHOOL of nature. </a></p><div class="supp"><p class="details"><div>SCHOOL model of T-cell receptor (TCR) <span class="highlight" style="background-color:">signaling</span>. Interaction with multivalent ligand (not shown) clusters the receptors and pushes them to reorientate (I), to bring <span class="highlight" style="background-color:">signaling</span> subunits into a correct (permissive) relative orientation and in sufficient proximity in the formed receptor oligomer (for illustrative purposes, receptor dimer is shown), and thus to promote the trans-homointeractions between  molecules (II). Then, two alternative pathways can take a place depending on the nature of activating stimuli. First is going through a stage IV resulting in formation of <sub>2</sub> dimer (dimer of dimers) and phosphorylation of the  ITAM tyrosines, thus triggering the activation signal A. Then, the <span class="highlight" style="background-color:">signaling</span>  oligomers formed subsequently dissociate from the TCR-CD3 complex, resulting in internalization of the remaining engaged TCR-CD3 complexes (VII). This <span class="highlight" style="background-color:">pathway</span> leads to partial (or incomplete) T-cell activation. Alternatively, the intermediate complex formed at the stage II can undergo further rearrangements, starting trans-homointeractions between CD3 proteins (III) and resulting in formation of an oligomeric intermediate. The stages I, II and III can be reversible or irreversible depending on interreceptor proximity and relative orientation of the receptors in TCR dimers/oligomers as well as on time duration of the TCR-ligand contact and lifetime of the receptor in TCR dimers/oligomers that generally correlate with the nature of the stimulus and its specificity and affinity/avidity. Next, in the <span class="highlight" style="background-color:">signaling</span> oligomers formed (III), the ITAM tyrosines undergo phosphorylation by PTKs that leads to generation of the activation signal, dissociation of <span class="highlight" style="background-color:">signaling</span> oligomers and internalization of the remaining engaged TCR chains (VIII, XI). This <span class="highlight" style="background-color:">pathway</span> provides at least two different activation signals from the  and CD3 <span class="highlight" style="background-color:">signaling</span> oligomers (signals A and B), respectively, and results in full T-cell activation. The distinct <span class="highlight" style="background-color:">signaling</span> through . and CD3 oligomers (or through various combinations of <span class="highlight" style="background-color:">signaling</span> chains in CD3 oligomeric structures) might be also responsible for distinct functions such as T-cell proliferation, effector functions, T-cell survival, pathogen clearance, TCR anergy, etc. In addition, the <span class="highlight" style="background-color:">signaling</span> oligomers formed can sequentially interact with the <span class="highlight" style="background-color:">signaling</span> subunits of nonengaged TCRs resulting in formation of higher-order <span class="highlight" style="background-color:">signaling</span> oligomers, thus amplifying and propagating the activation signal (not shown). Also, this leads to the release and subsequent internalization of the remaining nonengaged TCR complexes and/or TCR chains (not shown). Immunoreceptor tyrosine-based activation motifs (ITAMs) are shown as green rectangles. TCR-CD3- components are represented as whole polypeptides and as a simplified axial view. Circular arrows indicate ligand-induced receptor reorientation. Black and magenta arrows indicate specific intersubunit hetero- and homointeractions between transmembrane and cytoplasmic domains, respectively. All interchain interactions in intermediate complexes are shown by dotted arrows reflecting their transition state. Phosphate groups are shown as filled gray circles. In an axial view, one solid small black line depicts one phosphorylated ITAM domain. PTK, protein tyrosine kinase; SCHOOL, <span class="highlight" style="background-color:">signaling</span> chain homooligomerization. Adapted from Sigalov AB. The SCHOOL of nature. I. Transmembrane <span class="highlight" style="background-color:">signaling</span>. Self/Nonself 2010; 1:439.</div></p></div><div class="aux"><div class="resc">Alexander B Sigalov. Self Nonself. 2010 Jul-Sep;1(3):192-224.</div><p class="links"><a class="dblinks" href="/pubmed/21487477" ref="ordinalpos=4158&amp;ncbi_uid=2885977&amp;link_uid=21487477">Citation</a><a class="dblinks" href="/pmc/articles/PMC3047783/" ref="ordinalpos=4158&amp;ncbi_uid=2885977&amp;link_uid=PMC3047783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4159.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3570103/figure/fig1/" ref="ordinalpos=4159&amp;ncbi_uid=5187142&amp;link_uid=PMC3570103" href="/pmc/articles/PMC3570103/"><img src="/pmc/articles/instance/3570103/bin/JEM_20121484_Fig1.gif" src-large="/pmc/articles/instance/3570103/bin/JEM_20121484_Fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3570103/" ref="ordinalpos=4159&amp;ncbi_uid=5187142&amp;link_uid=PMC3570103" image-link="/pmc/articles/PMC3570103/figure/fig1/" class="imagepopup">Figure 1. From: Notch <span class="highlight" style="background-color:">pathway</span> activation targets AML-initiating cell homeostasis and differentiation. </a></p><div class="supp"><p class="details"><div><b>The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is silenced in AML patients.</b> (A) Heat map showing relative expression levels of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> target genes in 187 AML and normal Lin<sup></sup>CD34<sup>+</sup>CD38<sup></sup> BM cells. (B) Box plot representing expression value of Notch receptors NOTCH1 and NOTCH2 in the same samples. Expression is normalized to GAPDH expression. (C) Heat map showing relative expression levels of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> target genes in Lin<sup></sup>CD34<sup>+</sup>CD38<sup></sup> and Lin<sup></sup>CD34<sup>+</sup>CD38<sup>+</sup> populations from 12 AML patients compared with normal Lin<sup></sup>CD34<sup>+</sup>CD38<sup></sup> BM cells. (D) qRT-PCR analysis for <i>HES1</i> gene in CD34<sup>+</sup> cord blood stem cells and four AML patient samples (left) and ChIP of H3K27me3 on the promoter of the same samples (right). (E) qRT-PCR analysis for <i>NRARP</i> gene in CD34<sup>+</sup> cord blood stem cells and four AML patient samples (left) and ChIP of H3K27me3 on the promoter of the same samples (right). (F) qRT-PCR analysis for <i>HES1</i> gene in THP1 and DND41 cell lines (left) and ChIP of H3K27me3 on the promoter of <i>HES1</i> gene in THP1 and DND41 cell lines (right). (B and DF) Error bars represent mean  SD. ***, P &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Camille Lobry, et al. J Exp Med. 2013 February 11;210(2):301-319.</div><p class="links"><a class="dblinks" href="/pubmed/23359070" ref="ordinalpos=4159&amp;ncbi_uid=5187142&amp;link_uid=23359070">Citation</a><a class="dblinks" href="/pmc/articles/PMC3570103/" ref="ordinalpos=4159&amp;ncbi_uid=5187142&amp;link_uid=PMC3570103">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4160.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4129812/figure/F2/" ref="ordinalpos=4160&amp;ncbi_uid=6606571&amp;link_uid=PMC4129812" href="/pmc/articles/PMC4129812/"><img src="/pmc/articles/instance/4129812/bin/WJEM-3-282-g003.gif" src-large="/pmc/articles/instance/4129812/bin/WJEM-3-282-g003.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4129812/" ref="ordinalpos=4160&amp;ncbi_uid=6606571&amp;link_uid=PMC4129812" image-link="/pmc/articles/PMC4129812/figure/F2/" class="imagepopup">Figure 2.  From: PKC? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves in TNF-?-induced IP3R1 expression in human mesangial cells. </a></p><div class="supp"><p class="details"><div>Effects of cell <span class="highlight" style="background-color:">signaling</span> inhibitors on TNF--induced IP<sub>3</sub>R1 protein expression.</div></p></div><div class="aux"><div class="resc">Yu-rong Wang, et al. World J Emerg Med. 2012;3(4):282-286.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4160&amp;ncbi_uid=6606571&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4129812/" ref="ordinalpos=4160&amp;ncbi_uid=6606571&amp;link_uid=PMC4129812">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4161.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3652760/figure/F1/" ref="ordinalpos=4161&amp;ncbi_uid=4892922&amp;link_uid=PMC3652760" href="/pmc/articles/PMC3652760/"><img src="/pmc/articles/instance/3652760/bin/1423-0127-20-26-1.gif" src-large="/pmc/articles/instance/3652760/bin/1423-0127-20-26-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3652760/" ref="ordinalpos=4161&amp;ncbi_uid=4892922&amp;link_uid=PMC3652760" image-link="/pmc/articles/PMC3652760/figure/F1/" class="imagepopup">Figure 1.  From: New insights of P2X7 receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in alveolar functions. </a></p><div class="supp"><p class="details"><div>Proposed pathophysiological role of P2X7 receptor <span class="highlight" style="background-color:">signaling</span> in the normal and injured alveolus.</div></p></div><div class="aux"><div class="resc">Amarjit Mishra. J Biomed Sci. 2013;20(1):26-26.</div><p class="links"><a class="dblinks" href="/pubmed/23634990" ref="ordinalpos=4161&amp;ncbi_uid=4892922&amp;link_uid=23634990">Citation</a><a class="dblinks" href="/pmc/articles/PMC3652760/" ref="ordinalpos=4161&amp;ncbi_uid=4892922&amp;link_uid=PMC3652760">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4162.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3112042/figure/F1/" ref="ordinalpos=4162&amp;ncbi_uid=3372490&amp;link_uid=PMC3112042" href="/pmc/articles/PMC3112042/"><img src="/pmc/articles/instance/3112042/bin/zh89991171800001.gif" src-large="/pmc/articles/instance/3112042/bin/zh89991171800001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3112042/" ref="ordinalpos=4162&amp;ncbi_uid=3372490&amp;link_uid=PMC3112042" image-link="/pmc/articles/PMC3112042/figure/F1/" class="imagepopup">Figure 1.  From: Induction of human fetal hemoglobin via the NRF2 antioxidant response <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Induction of -globin gene expression by activators of the NRF2 antioxidant response element <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> (A) The human -globin gene promoter contains an ARE consensus sequence. (B) Compounds that activate NRF2/ARE <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. (C) Effect of NRF2 activators on <i>NQO1</i> and -globin mRNA levels. K562 cells were treated with nontoxic doses of each compound for 12 hours before RNA isolation. (D) Time course of tBHQ, curcumin, and D3T induction of <i>NQO1</i> and -globin mRNA in K562 cells. Real-time PCR was performed in triplicate. Results are presented as relative mRNA expression normalized to untreated (C) or DMSO treated (D) samples. Error bars represent  SEM, *<i>P</i> &lt; .05, compared with DMSO.</div></p></div><div class="aux"><div class="resc">Elizabeth R. Macari, et al. Blood. 2011 June 2;117(22):5987-5997.</div><p class="links"><a class="dblinks" href="/pubmed/21464371" ref="ordinalpos=4162&amp;ncbi_uid=3372490&amp;link_uid=21464371">Citation</a><a class="dblinks" href="/pmc/articles/PMC3112042/" ref="ordinalpos=4162&amp;ncbi_uid=3372490&amp;link_uid=PMC3112042">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4163.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1850774/figure/f5/" ref="ordinalpos=4163&amp;ncbi_uid=919106&amp;link_uid=PMC1850774" href="/pmc/articles/PMC1850774/"><img src="/pmc/articles/instance/1850774/bin/jh1123398005.gif" src-large="/pmc/articles/instance/1850774/bin/jh1123398005.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1850774/" ref="ordinalpos=4163&amp;ncbi_uid=919106&amp;link_uid=PMC1850774" image-link="/pmc/articles/PMC1850774/figure/f5/" class="imagepopup">Figure 5. From: New Approaches to the Biology of Melanoma. </a></p><div class="supp"><p class="details"><div>Convergence of <span class="highlight" style="background-color:">signaling</span> pathways. CAM, integrin, and receptor tyrosine kinase <span class="highlight" style="background-color:">signaling</span> can converge in melanoma cells. Mutations in B-Raf can lead to constitutive activation of the ERK/MAP kinase <span class="highlight" style="background-color:">pathway</span>. AKT activation may occur through PTKs or integrins, and there appears to be a <span class="highlight" style="background-color:">signaling</span> feedback to CAM. <b>Broken lines</b> indicate that intermediates are not yet known.</div></p></div><div class="aux"><div class="resc">Meenhard Herlyn, et al. Am J Pathol. 2002 November;161(5):1949-1957.</div><p class="links"><a class="dblinks" href="/pubmed/12414540" ref="ordinalpos=4163&amp;ncbi_uid=919106&amp;link_uid=12414540">Citation</a><a class="dblinks" href="/pmc/articles/PMC1850774/" ref="ordinalpos=4163&amp;ncbi_uid=919106&amp;link_uid=PMC1850774">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4164.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4074261/figure/F4/" ref="ordinalpos=4164&amp;ncbi_uid=6404250&amp;link_uid=PMC4074261" href="/pmc/articles/PMC4074261/"><img src="/pmc/articles/instance/4074261/bin/DEV071746F4.gif" src-large="/pmc/articles/instance/4074261/bin/DEV071746F4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4074261/" ref="ordinalpos=4164&amp;ncbi_uid=6404250&amp;link_uid=PMC4074261" image-link="/pmc/articles/PMC4074261/figure/F4/" class="imagepopup">Fig. 4. From: Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> directly regulates Foxj1 expression and ciliogenesis in zebrafish Kupffer's vesicle. </a></p><div class="supp"><p class="details"><div><b>Wnt <span class="highlight" style="background-color:">signaling</span> functions downstream of the FGF <span class="highlight" style="background-color:">pathway</span> in regulation of</b> <b><i>foxj1a</i></b> <b>expression and cilia formation.</b> (<b>A-D</b>) Ectopic expression of Wnt ligands restores <i>foxj1a</i> levels in embryos with reduced FGF <span class="highlight" style="background-color:">signaling</span>. DFC<i><sup>fgfr1</sup></i> <sup>MO</sup> embryos exhibited downregulated <i>foxj1a</i> expression in DFCs (B; 17/18) compared with controls (A). Co-injection of <i>wnt8a</i> RNA with <i>fgfr1</i> MO restored <i>foxj1a</i> expression (C; 16/19). Ectopic expression of <i>wnt8a</i> RNA alone had no apparent effect on <i>foxj1a</i> expression (D). (<b>E-H</b>) Ectopic expression of FGF ligands fails to restore <i>foxj1a</i> expression in embryos with reduced Wnt <span class="highlight" style="background-color:">signaling</span>. Downregulated <i>foxj1a</i> expression in DFC<i><sup>fzd10</sup></i> <sup>MO</sup> embryos (F) was not restored by co-injection of <i>fgf8a</i> RNA (G). Ectopic expression of <i>fgf8a</i> RNA alone had no apparent effect on <i>foxj1a</i> expression (H). Shown are ventral views of 90-95% epiboly staged embryos. (<b>I</b>,<b>J</b>) Ectopic expression of Wnt ligands partially rescues cilia length defects in embryos with reduced FGF <span class="highlight" style="background-color:">signaling</span>. Shown are quantification of cilia length (I) and cilia number (J). Approximately 12-20 embryos were analyzed for each group. Data are represented as means.d. <sup>*</sup><i>P</i>&lt;0.01. NS, not significant (<i>P</i>&gt;0.05).</div></p></div><div class="aux"><div class="resc">Alissa Caron, et al. Development. 2012 February 1;139(3):514-524.</div><p class="links"><a class="dblinks" href="/pubmed/22190638" ref="ordinalpos=4164&amp;ncbi_uid=6404250&amp;link_uid=22190638">Citation</a><a class="dblinks" href="/pmc/articles/PMC4074261/" ref="ordinalpos=4164&amp;ncbi_uid=6404250&amp;link_uid=PMC4074261">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4165.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3249369/figure/fig6/" ref="ordinalpos=4165&amp;ncbi_uid=4483099&amp;link_uid=PMC3249369" href="/pmc/articles/PMC3249369/"><img src="/pmc/articles/instance/3249369/bin/23fig6.gif" src-large="/pmc/articles/instance/3249369/bin/23fig6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3249369/" ref="ordinalpos=4165&amp;ncbi_uid=4483099&amp;link_uid=PMC3249369" image-link="/pmc/articles/PMC3249369/figure/fig6/" class="imagepopup">Figure 6.  From: The Regulation of Filamentous Growth in Yeast. </a></p><div class="supp"><p class="details"><div>Mechanisms of signal integration among regulatory proteins and pathways that control filamentous growth. (A) Multiple <span class="highlight" style="background-color:">signaling</span> pathways converge on the <i>FLO11</i> promoter to modulate gene expression (Rupp et al. 1999). Both Snf1 and Rim101 are thought to function through the transcriptional repressors Nrg1 and Nrg2 (Kuchin et al. 2002; Lamb and Mitchell 2003). (B) Multiple <span class="highlight" style="background-color:">signaling</span> pathways regulate the activity of the filamentation MAPK <span class="highlight" style="background-color:">pathway</span>, adapted from (Chavel et al. 2010). Rtg refers to the retrograde mitochondrial <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Liu and Butow 2006). Other pathways also converge on <i>FLO11</i> that are not shown here (Bruckner and Mosch 2011).</div></p></div><div class="aux"><div class="resc">Paul J. Cullen, et al. Genetics. 2012 January;190(1):23-49.</div><p class="links"><a class="dblinks" href="/pubmed/22219507" ref="ordinalpos=4165&amp;ncbi_uid=4483099&amp;link_uid=22219507">Citation</a><a class="dblinks" href="/pmc/articles/PMC3249369/" ref="ordinalpos=4165&amp;ncbi_uid=4483099&amp;link_uid=PMC3249369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4166.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2769052/figure/F9/" ref="ordinalpos=4166&amp;ncbi_uid=2485063&amp;link_uid=PMC2769052" href="/pmc/articles/PMC2769052/"><img src="/pmc/articles/instance/2769052/bin/zmo0100959690009.gif" src-large="/pmc/articles/instance/2769052/bin/zmo0100959690009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2769052/" ref="ordinalpos=4166&amp;ncbi_uid=2485063&amp;link_uid=PMC2769052" image-link="/pmc/articles/PMC2769052/figure/F9/" class="imagepopup">Fig. 9. From: Signal Transducer and Activator of Transcription 3 Is Required for Abnormal Proliferation and Survival of TSC2-Deficient Cells: Relevance to Pulmonary Lymphangioleiomyomatosis. </a></p><div class="supp"><p class="details"><div>Schematic representation of a potential mechanism of IFN-- and STAT3-dependent cell growth modulated by TSC2/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Top, in normal unstimulated cells, tumor suppressor TSC2 inhibits mTOR/S6K1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and cell proliferation. Treatment with IFN- activates canonical STAT1 <span class="highlight" style="background-color:">signaling</span>, induces modest phosphorylation of STAT3, and attenuates cell proliferation. Bottom, loss of TSC2 function leads to the constitutive activation of mTOR/S6K1 and increased STAT3 activity, which are both required for abnormal cell proliferation. IFN- induces STAT1 activation and increases STAT3 phosphorylation. Re-expression of TSC2 or inhibition of mTOR activity inhibits IFN--dependent STAT3 phosphorylation and synergizes with IFN- in inhibition of cell proliferation.</div></p></div><div class="aux"><div class="resc">Elena A. Goncharova, et al. Mol Pharmacol. 2009 October;76(4):766-777.</div><p class="links"><a class="dblinks" href="/pubmed/19596836" ref="ordinalpos=4166&amp;ncbi_uid=2485063&amp;link_uid=19596836">Citation</a><a class="dblinks" href="/pmc/articles/PMC2769052/" ref="ordinalpos=4166&amp;ncbi_uid=2485063&amp;link_uid=PMC2769052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4167.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4404594/figure/Fig1/" ref="ordinalpos=4167&amp;ncbi_uid=7445784&amp;link_uid=PMC4404594" href="/pmc/articles/PMC4404594/"><img src="/pmc/articles/instance/4404594/bin/12885_2015_1300_Fig1_HTML.gif" src-large="/pmc/articles/instance/4404594/bin/12885_2015_1300_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4404594/" ref="ordinalpos=4167&amp;ncbi_uid=7445784&amp;link_uid=PMC4404594" image-link="/pmc/articles/PMC4404594/figure/Fig1/" class="imagepopup">Figure 1.  From: Cancerous inhibitor of protein phosphatase 2A (CIP2A) is an independent prognostic marker in wild-type KRAS metastatic colorectal cancer after colorectal liver metastasectomy. </a></p><div class="supp"><p class="details"><div>Model of CIP2A involvement in the EGFR-RAS <span class="highlight" style="background-color:">signaling</span> pathways. ETS1 mediates CIP2A overexpression in human cancers with increased EGFR-MEK-ERK <span class="highlight" style="background-color:">pathway</span> activity. A positive feedback loop of CIP2A and MEK/ERK <span class="highlight" style="background-color:">signaling</span> pathways is shown.</div></p></div><div class="aux"><div class="resc">Kuen-Feng Chen, et al. BMC Cancer. 2015;15:301.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4167&amp;ncbi_uid=7445784&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4404594/" ref="ordinalpos=4167&amp;ncbi_uid=7445784&amp;link_uid=PMC4404594">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4168.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3983496/figure/F2/" ref="ordinalpos=4168&amp;ncbi_uid=6058263&amp;link_uid=PMC3983496" href="/pmc/articles/PMC3983496/"><img src="/pmc/articles/instance/3983496/bin/fphys-05-00131-g0002.gif" src-large="/pmc/articles/instance/3983496/bin/fphys-05-00131-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3983496/" ref="ordinalpos=4168&amp;ncbi_uid=6058263&amp;link_uid=PMC3983496" image-link="/pmc/articles/PMC3983496/figure/F2/" class="imagepopup">Figure 2.  From: Connexin 43 hemichannels and intracellular <span class="highlight" style="background-color:">signaling</span> in bone cells. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the proposed intracellular <span class="highlight" style="background-color:">signaling</span> pathways regulated by Cx43 hemichannels in bone cells. (A)</b> Bisphosphonates bind to phosphatases present in the cell membrane. This induces Cx43 hemichannel opening, followed by activation of the kinases Src and MEK, and ERKs. ERKs activated downstream of Cx43 hemichannel opening are retained in the cytoplasm by a complex formed by arrestin and clathrins. This leads to the phosphorylation of the cytoplasmic targets p90<sup>RSK</sup>, BAD and C/EBP, which results in osteoblast and osteocyte survival. <b>(B)</b> Mechanical stimulation induces 51 integrin engagement and the association of the integrins with Cx43, by a mechanism that requires the protein 14-3-3. This results in hemichannel opening and the release of PGE2. PGE2, in turn, activates EP2/4 prostaglandin receptor by an autocrine/paracrine mechanism, leading to activation of the cAMP/PI3K <span class="highlight" style="background-color:">signaling</span> pathways, accumulation of catenin with the consequent activation of Wnt <span class="highlight" style="background-color:">signaling</span>, and inhibition of osteocyte apoptosis. <b>(C)</b> Parathyroid hormone (PTH), through binding to the PTH receptor, induces activation of the cAMP/PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cx43, by sequestering -arrestin away from the PTH receptor, facilitate cAMP/PKA-mediated downstream <span class="highlight" style="background-color:">signaling</span> and osteoblast survival.</div></p></div><div class="aux"><div class="resc">Lilian I. Plotkin. Front Physiol. 2014;5:131.</div><p class="links"><a class="dblinks" href="/pubmed/24772090" ref="ordinalpos=4168&amp;ncbi_uid=6058263&amp;link_uid=24772090">Citation</a><a class="dblinks" href="/pmc/articles/PMC3983496/" ref="ordinalpos=4168&amp;ncbi_uid=6058263&amp;link_uid=PMC3983496">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4169.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4135311/figure/F2/" ref="ordinalpos=4169&amp;ncbi_uid=6508736&amp;link_uid=PMC4135311" href="/pmc/articles/PMC4135311/"><img src="/pmc/articles/instance/4135311/bin/nihms613472f2.gif" src-large="/pmc/articles/instance/4135311/bin/nihms613472f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4135311/" ref="ordinalpos=4169&amp;ncbi_uid=6508736&amp;link_uid=PMC4135311" image-link="/pmc/articles/PMC4135311/figure/F2/" class="imagepopup">Figure 2.  From: Therapies in inborn errors of oxidative metabolism. </a></p><div class="supp"><p class="details"><div>The Sirt1/AMPK/PGC1/PPAR <span class="highlight" style="background-color:">signaling</span> network. A schematized and partial view of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> indicating its major downstream targets (bottom) and factors susceptible to interfere with <span class="highlight" style="background-color:">signaling</span> (green arrows). Abbreviations: AICAR, 5-aminoimidazole-4-carboxamide ribonucleoside; AMPK, AMP-activated protein kinase; FOXO, forkhead box O; OXPHOS, oxidative phosphorylation; PGC1, peroxisome proliferator-activated receptor  coactivator 1; PPAR, peroxisome proliferator-activated receptor; RXR, retinoid X receptor.</div></p></div><div class="aux"><div class="resc">Manuel Schiff, et al. Trends Endocrinol Metab. ;23(9):488-495.</div><p class="links"><a class="dblinks" href="/pubmed/22633959" ref="ordinalpos=4169&amp;ncbi_uid=6508736&amp;link_uid=22633959">Citation</a><a class="dblinks" href="/pmc/articles/PMC4135311/" ref="ordinalpos=4169&amp;ncbi_uid=6508736&amp;link_uid=PMC4135311">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4170.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1180390/figure/FG5/" ref="ordinalpos=4170&amp;ncbi_uid=546984&amp;link_uid=PMC1180390" href="/pmc/articles/PMC1180390/"><img src="/pmc/articles/instance/1180390/bin/AJHGv73p1240fg5.gif" src-large="/pmc/articles/instance/1180390/bin/AJHGv73p1240fg5.jpg" alt="Figure  5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1180390/" ref="ordinalpos=4170&amp;ncbi_uid=546984&amp;link_uid=PMC1180390" image-link="/pmc/articles/PMC1180390/figure/FG5/" class="imagepopup">Figure  5.  From: Capillary Malformation-Arteriovenous Malformation, a New Clinical and Genetic Disorder Caused by RASA1 Mutations. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways implicating p120-RasGAP. Growth-factor binding to RTKs on the cell surface activates the Ras/MAPK-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which mediates signals to transcription factors in the nucleus. Changes in gene expression lead to cellular responses, such as differentiation and proliferation. p120-RasGAP (RasGAP) downregulates Ras <span class="highlight" style="background-color:">signaling</span> by catalyzing intrinsic GTPase activity of Ras. p120-RasGAP also is implicated in <span class="highlight" style="background-color:">signaling</span> to the cytoskeleton by binding Rap1a and/or p190-RhoGAP (RhoGAP).</div></p></div><div class="aux"><div class="resc">Iiro Eerola, et al. Am J Hum Genet. 2003 December;73(6):1240-1249.</div><p class="links"><a class="dblinks" href="/pubmed/14639529" ref="ordinalpos=4170&amp;ncbi_uid=546984&amp;link_uid=14639529">Citation</a><a class="dblinks" href="/pmc/articles/PMC1180390/" ref="ordinalpos=4170&amp;ncbi_uid=546984&amp;link_uid=PMC1180390">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4171.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678348/figure/F6/" ref="ordinalpos=4171&amp;ncbi_uid=4983112&amp;link_uid=PMC3678348" href="/pmc/articles/PMC3678348/"><img src="/pmc/articles/instance/3678348/bin/nihms470119f6.gif" src-large="/pmc/articles/instance/3678348/bin/nihms470119f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678348/" ref="ordinalpos=4171&amp;ncbi_uid=4983112&amp;link_uid=PMC3678348" image-link="/pmc/articles/PMC3678348/figure/F6/" class="imagepopup">Figure 6.  From: Molecular Basis of NF-?B <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Structures of the Myddosome and other membrane-proximal interactions in the TLR/IL-1R <span class="highlight" style="background-color:">pathway</span>. (<i>a</i>) Domain organizations. (<i>b</i>) Crystal structure of the DD complex of the Myddosome in two orientations, showing a structure with 6 MyD88, 4 IRAK4, and 4 IRAK1 or IRAK2 molecules. (<i>c</i>) A structural model of the membrane-proximal events in TLR <span class="highlight" style="background-color:">signaling</span>. Abbreviations: DD, death domain; IL-1R, interleukin-1 receptor; IRAK, interleukin-1 receptor (IL-1R)-associated kinase; MyD88, myeloid differentiation primary-response gene 88; TLR, Toll-like receptor.</div></p></div><div class="aux"><div class="resc">Johanna Napetschnig, et al. Annu Rev Biophys. ;42:443-468.</div><p class="links"><a class="dblinks" href="/pubmed/23495970" ref="ordinalpos=4171&amp;ncbi_uid=4983112&amp;link_uid=23495970">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678348/" ref="ordinalpos=4171&amp;ncbi_uid=4983112&amp;link_uid=PMC3678348">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4172.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3358864/figure/fig01/" ref="ordinalpos=4172&amp;ncbi_uid=4018553&amp;link_uid=PMC3358864" href="/pmc/articles/PMC3358864/"><img src="/pmc/articles/instance/3358864/bin/pnas.1113666109fig01.gif" src-large="/pmc/articles/instance/3358864/bin/pnas.1113666109fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3358864/" ref="ordinalpos=4172&amp;ncbi_uid=4018553&amp;link_uid=PMC3358864" image-link="/pmc/articles/PMC3358864/figure/fig01/" class="imagepopup">Fig. 1. From: Mathematical modeling elucidates the role of transcriptional feedback in gibberellin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The gibberellin <span class="highlight" style="background-color:">signaling</span> network is composed of three modules. Perception: gibberellin (GA<sub>4</sub>) binds to the GID1 receptor, and this complex binds to DELLA proteins. The GA<sub>4</sub>GID1 complex then mediates the ubiquitination (indicated by Ub) of the DELLA proteins. Response: DELLA proteins mediate transcriptional activation of the <i>GID1</i>, <i>GA20ox</i>, and <i>GA3ox</i> genes and the repression of <i>DELLA</i> transcription. Biosynthesis: the enzyme GA20ox converts GA<sub>12</sub> to GA<sub>15</sub>, then to GA<sub>24</sub>, and finally, to GA<sub>9</sub>, which is subsequently converted to GA<sub>4</sub> by the enzyme GA3ox. Thus, activation of this <span class="highlight" style="background-color:">pathway</span> stimulates formation of GA<sub>4</sub> (provided that sufficient GA<sub>12</sub> is available).</div></p></div><div class="aux"><div class="resc">Alistair M. Middleton, et al. Proc Natl Acad Sci U S A. 2012 May 8;109(19):7571-7576.</div><p class="links"><a class="dblinks" href="/pubmed/22523240" ref="ordinalpos=4172&amp;ncbi_uid=4018553&amp;link_uid=22523240">Citation</a><a class="dblinks" href="/pmc/articles/PMC3358864/" ref="ordinalpos=4172&amp;ncbi_uid=4018553&amp;link_uid=PMC3358864">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4173.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3366670/figure/marinedrugs-10-00694-f004/" ref="ordinalpos=4173&amp;ncbi_uid=4053986&amp;link_uid=PMC3366670" href="/pmc/articles/PMC3366670/"><img src="/pmc/articles/instance/3366670/bin/marinedrugs-10-00694-g004.gif" src-large="/pmc/articles/instance/3366670/bin/marinedrugs-10-00694-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3366670/" ref="ordinalpos=4173&amp;ncbi_uid=4053986&amp;link_uid=PMC3366670" image-link="/pmc/articles/PMC3366670/figure/marinedrugs-10-00694-f004/" class="imagepopup">Figure 4.  From: A Marine Anthraquinone SZ-685C Overrides Adriamycin-Resistance in Breast Cancer Cells through Suppressing Akt <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Effect of SZ-685C on the Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in MCF-7/ADR and MCF-7/Akt cells. Cells were treated with different concentrations of ADR (<b>A</b>) and SZ-685C (<b>B</b>) for 48 h as noted. Western blotting analysis was performed using antibodies against phospho-Akt (Ser473 or Thr308), total Akt, phospho-Bad (Ser 136), total Bad, and Bcl-xL, with Actin used as a loading control.</div></p></div><div class="aux"><div class="resc">Xun Zhu, et al. Mar Drugs. 2012 April;10(4):694-711.</div><p class="links"><a class="dblinks" href="/pubmed/22690138" ref="ordinalpos=4173&amp;ncbi_uid=4053986&amp;link_uid=22690138">Citation</a><a class="dblinks" href="/pmc/articles/PMC3366670/" ref="ordinalpos=4173&amp;ncbi_uid=4053986&amp;link_uid=PMC3366670">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4174.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1876590/figure/F2/" ref="ordinalpos=4174&amp;ncbi_uid=929635&amp;link_uid=PMC1876590" href="/pmc/articles/PMC1876590/"><img src="/pmc/articles/instance/1876590/bin/zpq0190762950002.gif" src-large="/pmc/articles/instance/1876590/bin/zpq0190762950002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1876590/" ref="ordinalpos=4174&amp;ncbi_uid=929635&amp;link_uid=PMC1876590" image-link="/pmc/articles/PMC1876590/figure/F2/" class="imagepopup">Fig. 2. From: Nonclassical estrogen receptor ? <span class="highlight" style="background-color:">signaling</span> mediates negative feedback in the female mouse reproductive axis. </a></p><div class="supp"><p class="details"><div>Estrogen feedback. The ERE-independent ER <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is sufficient to convey estrogen negative but not positive feedback. (<i>A</i>) Serum LH from intact, OVX, OVX/estrogen-replaced females killed in the morning for negative feedback and afternoon for positive feedback (<i>n</i> = 516). (<i>B</i>) Serum estradiol from intact animals (<i>n</i> = 56). (<i>C</i>) Pituitary response to exogenous GnRH in OVX/estrogen-replaced females (<i>n</i> = 610).</div></p></div><div class="aux"><div class="resc">C. Glidewell-Kenney, et al. Proc Natl Acad Sci U S A. 2007 May 8;104(19):8173-8177.</div><p class="links"><a class="dblinks" href="/pubmed/17470805" ref="ordinalpos=4174&amp;ncbi_uid=929635&amp;link_uid=17470805">Citation</a><a class="dblinks" href="/pmc/articles/PMC1876590/" ref="ordinalpos=4174&amp;ncbi_uid=929635&amp;link_uid=PMC1876590">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4175.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2819049/figure/F3/" ref="ordinalpos=4175&amp;ncbi_uid=2213259&amp;link_uid=PMC2819049" href="/pmc/articles/PMC2819049/"><img src="/pmc/articles/instance/2819049/bin/nihms169163f3.gif" src-large="/pmc/articles/instance/2819049/bin/nihms169163f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2819049/" ref="ordinalpos=4175&amp;ncbi_uid=2213259&amp;link_uid=PMC2819049" image-link="/pmc/articles/PMC2819049/figure/F3/" class="imagepopup">Fig. 3.  From: How Telomeres Solve the End-Protection Problem. </a></p><div class="supp"><p class="details"><div>Different components of shelterin are dedicated to different aspects of the end-protection problem. TRF2 represses the ATM kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<b>A</b>), whereas POT1 ensures that the ATR kinase is not activated (<b>B</b>). In addition, TRF2 is the main repressor of NHEJ at telomeres, although POT1 contributes to the repression of NHEJ, especially after DNA replication. Both TRF2 and POT1 function to block HDR at telomeres (not shown). TRF2 is proposed to block NHEJ and ATM kinase <span class="highlight" style="background-color:">signaling</span> by forming the t-loop. In the t-loop structure (A), the telomere end is hidden from the DNA end sensor MRN that activates the ATM kinase <span class="highlight" style="background-color:">pathway</span> (MRN), and the Ku70/80 ring (which initiates NHEJ) will not be able to load onto the chromosome end. In (B), POT1 is proposed to block ATR kinase <span class="highlight" style="background-color:">signaling</span> by preventing the binding of RPA, the single-stranded DNA binding protein that activates the ATR kinase <span class="highlight" style="background-color:">pathway</span>. POT1 could block RPA from binding to the single-stranded telomeric DNA either when present at the telomere terminus (as shown) or when exposed in the D loop of the t-loop configuration.</div></p></div><div class="aux"><div class="resc">Titia de Lange. Science. ;326(5955):948.</div><p class="links"><a class="dblinks" href="/pubmed/19965504" ref="ordinalpos=4175&amp;ncbi_uid=2213259&amp;link_uid=19965504">Citation</a><a class="dblinks" href="/pmc/articles/PMC2819049/" ref="ordinalpos=4175&amp;ncbi_uid=2213259&amp;link_uid=PMC2819049">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4176.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2556850/figure/F2/" ref="ordinalpos=4176&amp;ncbi_uid=1534688&amp;link_uid=PMC2556850" href="/pmc/articles/PMC2556850/"><img src="/pmc/articles/instance/2556850/bin/nihms65948f2.gif" src-large="/pmc/articles/instance/2556850/bin/nihms65948f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2556850/" ref="ordinalpos=4176&amp;ncbi_uid=1534688&amp;link_uid=PMC2556850" image-link="/pmc/articles/PMC2556850/figure/F2/" class="imagepopup">Figure 2.  From: The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> plays an important role in lymphopoiesis and hematopoiesis. </a></p><div class="supp"><p class="details"><div>Overview of reports documenting Wnt <span class="highlight" style="background-color:">signaling</span> in T cell development and HSC biology. References with a green bullet provide evidence favoring a role, while those labeled with a red bullet provide evidence against a role for canonical Wnt <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Frank J. T. Staal, et al. Eur J Immunol. ;38(7):1788-1794.</div><p class="links"><a class="dblinks" href="/pubmed/18581335" ref="ordinalpos=4176&amp;ncbi_uid=1534688&amp;link_uid=18581335">Citation</a><a class="dblinks" href="/pmc/articles/PMC2556850/" ref="ordinalpos=4176&amp;ncbi_uid=1534688&amp;link_uid=PMC2556850">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4177.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2937064/figure/F1/" ref="ordinalpos=4177&amp;ncbi_uid=3131656&amp;link_uid=PMC2937064" href="/pmc/articles/PMC2937064/"><img src="/pmc/articles/instance/2937064/bin/nihms218132f1.gif" src-large="/pmc/articles/instance/2937064/bin/nihms218132f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2937064/" ref="ordinalpos=4177&amp;ncbi_uid=3131656&amp;link_uid=PMC2937064" image-link="/pmc/articles/PMC2937064/figure/F1/" class="imagepopup">Figure 1.  From: Metabolic Syndrome, Hormones, and Maintenance of T Cells during Aging. </a></p><div class="supp"><p class="details"><div>Leptin, IGFBP3, and rapamycin suppress the IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to preserve nave T cells and promote longevity. IGF-1 <span class="highlight" style="background-color:">signaling</span> through the IRS proteins, which bind to the p110 subunit of phosphatidylinositol 3-kinase (PI-3K), leads to the generation of phosphatidylinositol 3,4,5-triphosphate (PIP3) and phosphorylation of AKT by phosphoinositide dependent kinase (PDK1). Phosphorylation of AKT leads to subsequent activation of mTOR and phosphorylates or inactivates a family of apoptosis-inducing forkhead transcription factors, including forkhead box O1 (FOXO1 or FKHR), forkhead box O3 (FOXO3, or FKHRL1), and forkhead box O4 (MLLT-7, FOXO4 or AFX). Life-long enhanced IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is associated with shortening of lifespan. In contrast, inhibition of mTOR by dietary restriction or rapamycin has been shown to be associated with longevity in mammals. It has recently been shown that increased levels of factors such as leptin and IGFBP3 that can inhibit the IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, are also associated with preservation of nave CD8 T cells in human nonagenarians.</div></p></div><div class="aux"><div class="resc">Hui-Chen Hsu, et al. Curr Opin Immunol. ;22(4):541-548.</div><p class="links"><a class="dblinks" href="/pubmed/20591642" ref="ordinalpos=4177&amp;ncbi_uid=3131656&amp;link_uid=20591642">Citation</a><a class="dblinks" href="/pmc/articles/PMC2937064/" ref="ordinalpos=4177&amp;ncbi_uid=3131656&amp;link_uid=PMC2937064">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4178.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4337596/figure/f8-mmr-11-04-2504/" ref="ordinalpos=4178&amp;ncbi_uid=7236797&amp;link_uid=PMC4337596" href="/pmc/articles/PMC4337596/"><img src="/pmc/articles/instance/4337596/bin/MMR-11-04-2504-g07.gif" src-large="/pmc/articles/instance/4337596/bin/MMR-11-04-2504-g07.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4337596/" ref="ordinalpos=4178&amp;ncbi_uid=7236797&amp;link_uid=PMC4337596" image-link="/pmc/articles/PMC4337596/figure/f8-mmr-11-04-2504/" class="imagepopup">Figure 8.  From: Effects of miR-19b knockdown on the cardiac differentiation of P19 mouse embryonic carcinoma cells. </a></p><div class="supp"><p class="details"><div>Effect of microRNA-19b (miR-19b) on the Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) mRNA and (B and C) protein levels of molecules in the Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (WNT, -catenin and GSK3) at various time points during the stimulation of differentiation. Data are presented as the mean  standard error of the mean of three experiments (<sup>*</sup>P&lt;0.05; <sup>**</sup>P&lt;0.01).</div></p></div><div class="aux"><div class="resc">XUEHUA LIU, et al. Mol Med Rep. 2015 April;11(4):2504-2512.</div><p class="links"><a class="dblinks" href="/pubmed/25483911" ref="ordinalpos=4178&amp;ncbi_uid=7236797&amp;link_uid=25483911">Citation</a><a class="dblinks" href="/pmc/articles/PMC4337596/" ref="ordinalpos=4178&amp;ncbi_uid=7236797&amp;link_uid=PMC4337596">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4179.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3566487/figure/F8/" ref="ordinalpos=4179&amp;ncbi_uid=5814788&amp;link_uid=PMC3566487" href="/pmc/articles/PMC3566487/"><img src="/pmc/articles/instance/3566487/bin/zh40041306540008.gif" src-large="/pmc/articles/instance/3566487/bin/zh40041306540008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3566487/" ref="ordinalpos=4179&amp;ncbi_uid=5814788&amp;link_uid=PMC3566487" image-link="/pmc/articles/PMC3566487/figure/F8/" class="imagepopup">Fig. 8. From: Mesenchymal stem cells improve cardiac conduction by upregulation of connexin 43 through paracrine <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Canonical Wnt <span class="highlight" style="background-color:">signaling</span> and ERK1/2 <span class="highlight" style="background-color:">signaling</span> pathways affect Cx43 expression. Treatment of HL-1 monolayers with media obtained from Wnt3a overexpressing L1 cells significantly increased the  after 4 h (<i>A</i>; *<i>P</i> &lt; 0.05 compared with Ctrl). PD98058, an ERK1/2 inhibitor, attenuated the ConT-mediated increase of  in HL-1 monolayers (<i>B;</i> *<i>P</i> &lt; 0.05 compared with Ctrl; &amp;<i>P</i> &lt; 0.05 compared with Ctrl + PD98059; #<i>P</i> &lt; 0.05 compared with ConT). A schematic summary of the experimental results illustrates the proposed signal transduction <span class="highlight" style="background-color:">pathway</span> (<i>C)</i>. RTK, receptor tyrosine kinase.</div></p></div><div class="aux"><div class="resc">Shwetha Mureli, et al. Am J Physiol Heart Circ Physiol. 2013 February 15;304(4):H600-H609.</div><p class="links"><a class="dblinks" href="/pubmed/23241322" ref="ordinalpos=4179&amp;ncbi_uid=5814788&amp;link_uid=23241322">Citation</a><a class="dblinks" href="/pmc/articles/PMC3566487/" ref="ordinalpos=4179&amp;ncbi_uid=5814788&amp;link_uid=PMC3566487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4180.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3222216/figure/F7/" ref="ordinalpos=4180&amp;ncbi_uid=4447289&amp;link_uid=PMC3222216" href="/pmc/articles/PMC3222216/"><img src="/pmc/articles/instance/3222216/bin/DEV067280F7.gif" src-large="/pmc/articles/instance/3222216/bin/DEV067280F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3222216/" ref="ordinalpos=4180&amp;ncbi_uid=4447289&amp;link_uid=PMC3222216" image-link="/pmc/articles/PMC3222216/figure/F7/" class="imagepopup">Fig. 7. From: Neural crest specification by noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> and PAR-1. </a></p><div class="supp"><p class="details"><div><b>PAR-1 functions in the Wnt5/11 <span class="highlight" style="background-color:">pathway</span> upstream of <i>Pax3</i></b>. (<b>A</b>) <i>PAR-1</i> RNA promotes <i>Pax3</i> expression (stage 17). (<b>B</b>) PAR-1 is required for <i>Pax3</i> expression (stage 14/15). (<b>C</b>,<b>D</b>) PAR-1 rescues the <i>FoxD3</i> defect in embryos that were injected with <i>dnWnt5/11</i> RNA. Dorso-anterior view, except A (dorsal view). Arrows indicate altered gene expression. (<b>E</b>) Model for Wnt11R <span class="highlight" style="background-color:">signaling</span> during NC specification. Dvl and the associated PAR-1 protein kinase are essential regulators of NC-specific transcription in response to Wnt11R/Ror2 <span class="highlight" style="background-color:">signaling</span>. Dashed lines represent indirect effects via unknown intermediates.</div></p></div><div class="aux"><div class="resc">Olga Ossipova, et al. Development. 2011 December 15;138(24):5441-5450.</div><p class="links"><a class="dblinks" href="/pubmed/22110058" ref="ordinalpos=4180&amp;ncbi_uid=4447289&amp;link_uid=22110058">Citation</a><a class="dblinks" href="/pmc/articles/PMC3222216/" ref="ordinalpos=4180&amp;ncbi_uid=4447289&amp;link_uid=PMC3222216">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4181.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2965400/figure/f3-0030733/" ref="ordinalpos=4181&amp;ncbi_uid=3311071&amp;link_uid=PMC2965400" href="/pmc/articles/PMC2965400/"><img src="/pmc/articles/instance/2965400/bin/DMM000703F3.gif" src-large="/pmc/articles/instance/2965400/bin/DMM000703F3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2965400/" ref="ordinalpos=4181&amp;ncbi_uid=3311071&amp;link_uid=PMC2965400" image-link="/pmc/articles/PMC2965400/figure/f3-0030733/" class="imagepopup">Fig. 3. From: SHIP2, a factor associated with diet-induced obesity and insulin sensitivity, attenuates FGF <span class="highlight" style="background-color:">signaling</span> in vivo. </a></p><div class="supp"><p class="details"><div><b>Knockdown of maternal Ship2a alters BMP but not WNT <span class="highlight" style="background-color:">signaling</span> pathways.</b> (AD) Expression of <i>bmp7</i> is reduced in 3040% epiboly and 50% epiboly maternal <i>ship2a</i> morphants as compared with WT controls. (EJ) Gene expression that is dependent on the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not affected in maternal <i>ship2a</i>-morphant embryos. Expression of <i>vox</i> (E,F) or <i>vent</i> (G,H) in 40% epiboly embryos, or <i>bozozok</i> (<i>boz</i>) (IJ) in oblong stage (3.7 hpf) embryos, is unchanged by depletion of maternal Ship2a. AH, animal pole views with dorsal to the right; I and J, lateral views with animal pole up and dorsal to the right.</div></p></div><div class="aux"><div class="resc">Michael J. Jurynec, et al. Dis Model Mech. 2010 Nov-Dec;3(11-12):733-742.</div><p class="links"><a class="dblinks" href="/pubmed/20616095" ref="ordinalpos=4181&amp;ncbi_uid=3311071&amp;link_uid=20616095">Citation</a><a class="dblinks" href="/pmc/articles/PMC2965400/" ref="ordinalpos=4181&amp;ncbi_uid=3311071&amp;link_uid=PMC2965400">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4182.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3716572/figure/F3/" ref="ordinalpos=4182&amp;ncbi_uid=5107762&amp;link_uid=PMC3716572" href="/pmc/articles/PMC3716572/"><img src="/pmc/articles/instance/3716572/bin/1471-2407-13-307-3.gif" src-large="/pmc/articles/instance/3716572/bin/1471-2407-13-307-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3716572/" ref="ordinalpos=4182&amp;ncbi_uid=5107762&amp;link_uid=PMC3716572" image-link="/pmc/articles/PMC3716572/figure/F3/" class="imagepopup">Figure 3.  From: A link between premenopausal iron deficiency and breast cancer malignancy. </a></p><div class="supp"><p class="details"><div><b>Host iron deficiency stimulates Notch but not TGF-</b><b></b><b>, and Wnt <span class="highlight" style="background-color:">signaling</span>. (A)</b> Iron deficiency increased mRNA expression of Notch 2, 3, 4, their receptors Jagged 1, 2 and Hes1 by qPCR. <b>(B)</b> No significant changes for TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components, TGF-1, Smad 2, and Smad3. <b>(C)</b> No significant changes for Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components, LRP5, LRP6, and Wnt3a. *: Significantly different from the group of mice fed 350 ppm Fe diet.</div></p></div><div class="aux"><div class="resc">Jinlong Jian, et al. BMC Cancer. 2013;13:307-307.</div><p class="links"><a class="dblinks" href="/pubmed/23800380" ref="ordinalpos=4182&amp;ncbi_uid=5107762&amp;link_uid=23800380">Citation</a><a class="dblinks" href="/pmc/articles/PMC3716572/" ref="ordinalpos=4182&amp;ncbi_uid=5107762&amp;link_uid=PMC3716572">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4183.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3139487/figure/fig01/" ref="ordinalpos=4183&amp;ncbi_uid=3280541&amp;link_uid=PMC3139487" href="/pmc/articles/PMC3139487/"><img src="/pmc/articles/instance/3189825/bin/ajcr0001-0650-f1.gif" src-large="/pmc/articles/instance/3189825/bin/ajcr0001-0650-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3139487/" ref="ordinalpos=4183&amp;ncbi_uid=3280541&amp;link_uid=PMC3139487" image-link="/pmc/articles/PMC3139487/figure/fig01/" class="imagepopup">Figure 1.  From: Systematic review of pharmacogenetic testing for predicting clinical benefit to anti-EGFR therapy in metastatic colorectal cancer. </a></p><div class="supp"><p class="details"><div>Mutation frequencies and loss of protein expression in the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jennifers Lin, et al. Am J Cancer Res. 2011;1(5):650-662.</div><p class="links"><a class="dblinks" href="/pubmed/21779535" ref="ordinalpos=4183&amp;ncbi_uid=3280541&amp;link_uid=21779535">Citation</a><a class="dblinks" href="/pmc/articles/PMC3139487/" ref="ordinalpos=4183&amp;ncbi_uid=3280541&amp;link_uid=PMC3139487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4184.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1318476/figure/pbio-0040030-g001/" ref="ordinalpos=4184&amp;ncbi_uid=724808&amp;link_uid=PMC1318476" href="/pmc/articles/PMC1318476/"><img src="/pmc/articles/instance/1326285/bin/pbio.0040030.g001.gif" src-large="/pmc/articles/instance/1326285/bin/pbio.0040030.g001.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1318476/" ref="ordinalpos=4184&amp;ncbi_uid=724808&amp;link_uid=PMC1318476" image-link="/pmc/articles/PMC1318476/figure/pbio-0040030-g001/" class="imagepopup">A natural killer cell receptor called KIR2DL4 stimulates an immune <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from inside an intracellular vesicular structure (shown above).  From: Inside a Killer: Immune Signals May Promote Vascular Growth. </a></p><div class="aux"><div class="resc">PLoS Biol. 2006 January;4(1):e30.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4184&amp;ncbi_uid=724808&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC1318476/" ref="ordinalpos=4184&amp;ncbi_uid=724808&amp;link_uid=PMC1318476">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4185.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2949529/figure/F2/" ref="ordinalpos=4185&amp;ncbi_uid=3259129&amp;link_uid=PMC2949529" href="/pmc/articles/PMC2949529/"><img src="/pmc/articles/instance/2949529/bin/nihms206301f2.gif" src-large="/pmc/articles/instance/2949529/bin/nihms206301f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2949529/" ref="ordinalpos=4185&amp;ncbi_uid=3259129&amp;link_uid=PMC2949529" image-link="/pmc/articles/PMC2949529/figure/F2/" class="imagepopup">Figure 2. OGT and O-GlcNAc Regulate Insulin <span class="highlight" style="background-color:">Signaling</span>.  From: O-GlcNAc <span class="highlight" style="background-color:">Signaling</span>: A Metabolic Link Between Diabetes and Cancer?. </a></p><div class="supp"><p class="details"><div>During prolonged insulin <span class="highlight" style="background-color:">signaling</span> O-GlcNAcylation of key molecules have an affect on the attenuation of the insulin <span class="highlight" style="background-color:">signaling</span> cascade. As with any <span class="highlight" style="background-color:">signaling</span> cascade, there must be a mechanism in place to down-regulate the cascade after a prolonged period of activation. O-GlcNAcylation of key molecules can achieve this via the translocation of OGT in response to insulin.<br />A) During normal cellular conditions and early insulin mediated <span class="highlight" style="background-color:">signaling</span>, OGT (blue) is localized within the nucleus. Upon insulin stimulation, insulin (light green) binds the insulin receptor (IR, light blue) causing its autophosphorylation at tyrosine residues (P<sub>Y</sub>, red circles). Phosphorylation of the IR recruits IRS-1 (light red) to the membrane. IRS-1 is phosphorylated at multiple tyrosine residues, resulting in its binding to the p85 regulatory subunit (brown) of PI3K. PI3K then catalyzes the production of PIP3 (light blue) at the plasma membrane. PIP3 recruits PDK-1which increases the phosphorylation of AKT at threonine-308 (P<sub>T</sub>, red circle) causing its activation. Activated AKT then phosphorylates various substrates. Phosphorylation of GSK3 results in an increase in glycogen production. Phosphorylation of FOXO results in a decrease in transcription of gluconeogenic genes. Phosphorylation of AS160 results in increased GLUT4 translocation.<br />B) As a response to prolonged insulin <span class="highlight" style="background-color:">signaling</span>, a subset of OGT translocates from the nucleus to the plasma membrane and binds PIP3 through a PIP3 binding domain at its C-terminus. At the membrane, OGT is tyrosine phosphorylated by the IR leading to an increase in OGTs activity. Active OGT can O-GlcNAcylate key molecules namely p85/p110, PDK1, AKT, and FOXO to alter insulin <span class="highlight" style="background-color:">signaling</span>. O-GlcNAcylation (green circles) of IRS-1 can decrease its interaction with p85, decrease its activating phosphorylation at tyrosine-608 (thereby reducing its activity), while increasing its phosphorylation at serines-632 and 635. O-GlcNAcylation of AKT, results in a decrease in its activating phosphorylation of threonine-308. This causes a decrease in phosphorylation of GSK3, FOXO, and AS160. Decreasing phosphorylation of GSK3 results in decreased glycogen production. Decreasing phosphorylation of FOXO (along with increasing its O-GlcNAcylation) results in an increase in gluconeogenic genes. Decreasing phosphorylation of AS160 results in a decrease in GLUT4 translocation. Taken together O-GlcNAcylation of specific proteins within the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can result in a dampening effect of the insulin mediated signal.</div></p></div><div class="aux"><div class="resc">C. Slawson, et al. Trends Biochem Sci. ;35(10):547-555.</div><p class="links"><a class="dblinks" href="/pubmed/20466550" ref="ordinalpos=4185&amp;ncbi_uid=3259129&amp;link_uid=20466550">Citation</a><a class="dblinks" href="/pmc/articles/PMC2949529/" ref="ordinalpos=4185&amp;ncbi_uid=3259129&amp;link_uid=PMC2949529">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4186.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2712666/figure/F9/" ref="ordinalpos=4186&amp;ncbi_uid=1926935&amp;link_uid=PMC2712666" href="/pmc/articles/PMC2712666/"><img src="/pmc/articles/instance/2712666/bin/nihms112649f9.gif" src-large="/pmc/articles/instance/2712666/bin/nihms112649f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2712666/" ref="ordinalpos=4186&amp;ncbi_uid=1926935&amp;link_uid=PMC2712666" image-link="/pmc/articles/PMC2712666/figure/F9/" class="imagepopup">Figure 9. HVEM-LIGHT <span class="highlight" style="background-color:">pathway</span>.  From: TNF superfamily: costimulation and clinical applications. </a></p><div class="supp"><p class="details"><div>HVEM was originally identified as entry mediator of herpes virus. HVEM <span class="highlight" style="background-color:">signaling</span> is complex as it binds KIGHt as well as LT3 and is further complicated as LIGHT besides binding HVEM also binds LTR. This complex receptor-ligand interactions as well as HSV-HVEM interplay culminate in the expression of of array of <span class="highlight" style="background-color:">signaling</span> molecules, type I IFNs etc.</div></p></div><div class="aux"><div class="resc">Dass S Vinay, et al. Cell Biol Int. ;33(4):453-465.</div><p class="links"><a class="dblinks" href="/pubmed/19230849" ref="ordinalpos=4186&amp;ncbi_uid=1926935&amp;link_uid=19230849">Citation</a><a class="dblinks" href="/pmc/articles/PMC2712666/" ref="ordinalpos=4186&amp;ncbi_uid=1926935&amp;link_uid=PMC2712666">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4187.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3950678/figure/gkt1277-F6/" ref="ordinalpos=4187&amp;ncbi_uid=5907148&amp;link_uid=PMC3950678" href="/pmc/articles/PMC3950678/"><img src="/pmc/articles/instance/3950678/bin/gkt1277f6p.gif" src-large="/pmc/articles/instance/3950678/bin/gkt1277f6p.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3950678/" ref="ordinalpos=4187&amp;ncbi_uid=5907148&amp;link_uid=PMC3950678" image-link="/pmc/articles/PMC3950678/figure/gkt1277-F6/" class="imagepopup">Figure 6. From: Causal inference of gene regulation with subnetwork assembly from genetical genomics data. </a></p><div class="supp"><p class="details"><div>The causal gene network inferred from top discriminative modules at a significance level of 0.005. The major parts of the causal network that overlap with the KEGG maps are indicated. The network with green nodes involves metabolic pathways. The network with pink nodes provides a partial interface of the FAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which communicates with the regulation of actin cytoskeleton and the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The network with yellow nodes mostly connected by dotted lines partially overlaps with the PPAR singling <span class="highlight" style="background-color:">pathway</span>, and the network is centered on <i>RXRB</i>. This is the first study to suggest that the <i>RXRB</i>-causal network is involved in lymph node metastasis.</div></p></div><div class="aux"><div class="resc">Chien-Hua Peng, et al. Nucleic Acids Res. 2014 March;42(5):2803-2819.</div><p class="links"><a class="dblinks" href="/pubmed/24322297" ref="ordinalpos=4187&amp;ncbi_uid=5907148&amp;link_uid=24322297">Citation</a><a class="dblinks" href="/pmc/articles/PMC3950678/" ref="ordinalpos=4187&amp;ncbi_uid=5907148&amp;link_uid=PMC3950678">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4188.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3995118/figure/f1/" ref="ordinalpos=4188&amp;ncbi_uid=6104984&amp;link_uid=PMC3995118" href="/pmc/articles/PMC3995118/"><img src="/pmc/articles/instance/3995118/bin/fig-1.gif" src-large="/pmc/articles/instance/3995118/bin/fig-1.jpg" alt="&lt;b&gt;FIG. 1.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3995118/" ref="ordinalpos=4188&amp;ncbi_uid=6104984&amp;link_uid=PMC3995118" image-link="/pmc/articles/PMC3995118/figure/f1/" class="imagepopup">FIG. 1..  From: Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span> Contributes to Gastric Cancer Cell Growth and Proliferation. </a></p><div class="supp"><p class="details"><div>The Sonic Hedgehog (Shh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components are detected in primary gastric cancer cells. Immunofluorescent staining was manipulated to investigate the expression of SHH, PTCH, SMO, GLI1, and GLI2 in human primary gastric cancer cells (red), and the nuclei were counterstained with 4,6-diamidino-2-phenylindole (DAPI; blue). Scale bar, 100m.</div></p></div><div class="aux"><div class="resc">Jianhua Wan, et al. Biores Open Access. 2014 April 1;3(2):53-59.</div><p class="links"><a class="dblinks" href="/pubmed/24804165" ref="ordinalpos=4188&amp;ncbi_uid=6104984&amp;link_uid=24804165">Citation</a><a class="dblinks" href="/pmc/articles/PMC3995118/" ref="ordinalpos=4188&amp;ncbi_uid=6104984&amp;link_uid=PMC3995118">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4189.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3298526/figure/F1/" ref="ordinalpos=4189&amp;ncbi_uid=3851906&amp;link_uid=PMC3298526" href="/pmc/articles/PMC3298526/"><img src="/pmc/articles/instance/3298526/bin/1752-0509-6-3-1.gif" src-large="/pmc/articles/instance/3298526/bin/1752-0509-6-3-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3298526/" ref="ordinalpos=4189&amp;ncbi_uid=3851906&amp;link_uid=PMC3298526" image-link="/pmc/articles/PMC3298526/figure/F1/" class="imagepopup">Figure 1.  From: Biomarker robustness reveals the PDGF network as driving disease outcome in ovarian cancer patients in multiple studies. </a></p><div class="supp"><p class="details"><div><b>Venn diagrams demonstrate the overlap/lack of overlap between prognostic biomarkers</b>. (a) Blue circles stand for the genes identified as significant in stratifying patients into survival groups in TCGA. Red circles are genes identified in Duke set #1 and green circles are genes identified in Duke set #2. The shared colored circles are genes shared between datasets. While some genes are shared between two groups, none is shared among all three datasets. (b-d) Venn diagrams for the gene set signature analysis. In contrast, (e) shows the same analyses performed via <span class="highlight" style="background-color:">pathway</span> metrics. One <span class="highlight" style="background-color:">pathway</span> (PDGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) is shared among the three datasets and demonstrates the robustness of the <span class="highlight" style="background-color:">pathway</span> approach.</div></p></div><div class="aux"><div class="resc">Rotem Ben-Hamo, et al. BMC Syst Biol. 2012;6:3-3.</div><p class="links"><a class="dblinks" href="/pubmed/22236809" ref="ordinalpos=4189&amp;ncbi_uid=3851906&amp;link_uid=22236809">Citation</a><a class="dblinks" href="/pmc/articles/PMC3298526/" ref="ordinalpos=4189&amp;ncbi_uid=3851906&amp;link_uid=PMC3298526">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4190.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3043547/figure/F1/" ref="ordinalpos=4190&amp;ncbi_uid=2781062&amp;link_uid=PMC3043547" href="/pmc/articles/PMC3043547/"><img src="/pmc/articles/instance/3043547/bin/nihms-270242-f0001.gif" src-large="/pmc/articles/instance/3043547/bin/nihms-270242-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3043547/" ref="ordinalpos=4190&amp;ncbi_uid=2781062&amp;link_uid=PMC3043547" image-link="/pmc/articles/PMC3043547/figure/F1/" class="imagepopup">Figure 1. The NF-B activation pathways.  From: The NF-?B activation pathways, emerging molecular targets for cancer prevention and therapy. </a></p><div class="supp"><p class="details"><div>TNF- is shown as a representative inducer of the canonical <span class="highlight" style="background-color:">pathway</span>. In this <span class="highlight" style="background-color:">pathway</span>, TNF receptor 1 recruits and activates IKK to form a <span class="highlight" style="background-color:">signaling</span> complex through TNF receptor associated death domain (TRADD), RIP, and TRAF2. IKK then phosphorylates IB, which leads to IB ubiquitination and degradation in the proteasome. Subsequently, the NF-B complex (p65/p50) migrates to the nucleus and activates gene transcription. In the non-canonical <span class="highlight" style="background-color:">pathway</span> (represented by CD40L), NIK-mediated IKK activation triggers the processing of p100 to create p52. Then the NF-B complex (p52/RelB) moves to the nucleus to activate gene transcription. In the atypical <span class="highlight" style="background-color:">pathway</span> such as that activated by UV, IKK-independent mechanisms are involved in IB phosphorylation.</div></p></div><div class="aux"><div class="resc">Yong Lin, et al. Expert Opin Ther Targets. ;14(1):45-55.</div><p class="links"><a class="dblinks" href="/pubmed/20001209" ref="ordinalpos=4190&amp;ncbi_uid=2781062&amp;link_uid=20001209">Citation</a><a class="dblinks" href="/pmc/articles/PMC3043547/" ref="ordinalpos=4190&amp;ncbi_uid=2781062&amp;link_uid=PMC3043547">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4191.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2910246/figure/F8/" ref="ordinalpos=4191&amp;ncbi_uid=3194037&amp;link_uid=PMC2910246" href="/pmc/articles/PMC2910246/"><img src="/pmc/articles/instance/2910246/bin/nihms-208519-f0008.gif" src-large="/pmc/articles/instance/2910246/bin/nihms-208519-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2910246/" ref="ordinalpos=4191&amp;ncbi_uid=3194037&amp;link_uid=PMC2910246" image-link="/pmc/articles/PMC2910246/figure/F8/" class="imagepopup">Fig. 8. A conceptual view of the mechanism of IFN--induced apoptosis.  From: Induction of indoleamine 2,3-dioxygenase by interferon-gamma in human lens epithelial cells: Apoptosis through the formation of 3-hydroxykynurenine. </a></p><div class="supp"><p class="details"><div>IFN- induces IDO via JAK-STAT1 <span class="highlight" style="background-color:">signaling</span>, which catalyzes the first step in the kynurenine <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> produces 3OHKyn, which reacts and chemically modifies proteins and also generates ROS, resulting in apoptosis.</div></p></div><div class="aux"><div class="resc">Maneesh Mailankot, et al. Int J Biochem Cell Biol. ;42(9):1446-1454.</div><p class="links"><a class="dblinks" href="/pubmed/20435158" ref="ordinalpos=4191&amp;ncbi_uid=3194037&amp;link_uid=20435158">Citation</a><a class="dblinks" href="/pmc/articles/PMC2910246/" ref="ordinalpos=4191&amp;ncbi_uid=3194037&amp;link_uid=PMC2910246">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4192.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407147/figure/pone-0040773-g006/" ref="ordinalpos=4192&amp;ncbi_uid=4145296&amp;link_uid=PMC3407147" href="/pmc/articles/PMC3407147/"><img src="/pmc/articles/instance/3407147/bin/pone.0040773.g006.gif" src-large="/pmc/articles/instance/3407147/bin/pone.0040773.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407147/" ref="ordinalpos=4192&amp;ncbi_uid=4145296&amp;link_uid=PMC3407147" image-link="/pmc/articles/PMC3407147/figure/pone-0040773-g006/" class="imagepopup">Figure 6. EM-163 inhibits MyD88-mediated <span class="highlight" style="background-color:">signaling</span> and cytokine production..  From: Therapeutic Inhibition of Pro-Inflammatory <span class="highlight" style="background-color:">Signaling</span> and Toxicity to Staphylococcal Enterotoxin B by a Synthetic Dimeric BB-Loop Mimetic of MyD88. </a></p><div class="supp"><p class="details"><div>EM-163 inhibition was measured by monitoring LPS-induced SEAP activity via a MyD88-mediated NF-kB driven <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. HEK 293 stable transfected cell line (TLR4-MD2-NF-kB-SEAP) was activated with LPS (TLR-4 ligand) and treated with varying concentrations of EM-163 (1 mM to 50 M). Culture supernatants were tested for SEAP activity and compared to levels in the absence of EM-163. (<b>A</b>) Inhibition of SEAP reporter activity, (<b>B</b>) inhibition of TNF- and IL-1 production, <i>(</i><b>C</b>) Inhibition of TNF- and IL-1 production by EM-163 after SEB stimulation of HEK 293 cells transfected with HLA-DR (/) and FLAG-MyD88. Results represent three experiments.</div></p></div><div class="aux"><div class="resc">Teri L. Kissner, et al. PLoS One. 2012;7(7):e40773.</div><p class="links"><a class="dblinks" href="/pubmed/22848400" ref="ordinalpos=4192&amp;ncbi_uid=4145296&amp;link_uid=22848400">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407147/" ref="ordinalpos=4192&amp;ncbi_uid=4145296&amp;link_uid=PMC3407147">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4193.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3364790/figure/F4/" ref="ordinalpos=4193&amp;ncbi_uid=4854467&amp;link_uid=PMC3364790" href="/pmc/articles/PMC3364790/"><img src="/pmc/articles/instance/3364790/bin/cbt-13-542-g4.gif" src-large="/pmc/articles/instance/3364790/bin/cbt-13-542-g4.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3364790/" ref="ordinalpos=4193&amp;ncbi_uid=4854467&amp;link_uid=PMC3364790" image-link="/pmc/articles/PMC3364790/figure/F4/" class="imagepopup">Figure. From: Boswellic acid induces epigenetic alterations by modulating DNA methylation in colorectal cancer cells. </a></p><div class="supp"><p class="details"><div><b>Figure 4.</b> The <span class="highlight" style="background-color:">pathway</span> analysis revealed the most affected molecular <span class="highlight" style="background-color:">pathway</span> by AKBA. (A) The <span class="highlight" style="background-color:">pathway</span> analysis revealed that CRC metastasis <span class="highlight" style="background-color:">signaling</span> was among one of the top five canonical pathways that served as a target of AKBA-induced demethylation in CRC cells. (B) An illustration from Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) demonstrating the biological relationship between demethylated and overexpressed genes from the 3 genes clusters. Green bolded BM represents the genes that are described as biomarkers of CRC, Orange Fx represents the genes that have direct function in CRC, and CP represents the canonical <span class="highlight" style="background-color:">pathway</span> involved in CRC metastasis <span class="highlight" style="background-color:">signaling</span>. One of the genes, <i>SMPD3</i>, which was found within the most demethylated gene group cluster 1 in the methylation microarray analysis and subsequently confirmed in our validation analysis (expression by qRT-PCR and methylation by qMSP), is marked with * in red. Density of red color of icons and the number below the icons represents the fold change of gene expression induced by AKBA obtained in our genome-wide gene expression analysis.</div></p></div><div class="aux"><div class="resc">Yan Shen, et al. Cancer Biol Ther. 2012 May 1;13(7):542-552.</div><p class="links"><a class="dblinks" href="/pubmed/22415137" ref="ordinalpos=4193&amp;ncbi_uid=4854467&amp;link_uid=22415137">Citation</a><a class="dblinks" href="/pmc/articles/PMC3364790/" ref="ordinalpos=4193&amp;ncbi_uid=4854467&amp;link_uid=PMC3364790">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4194.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2842525/figure/F2/" ref="ordinalpos=4194&amp;ncbi_uid=2269163&amp;link_uid=PMC2842525" href="/pmc/articles/PMC2842525/"><img src="/pmc/articles/instance/2842525/bin/WJG-16-1337-g002.gif" src-large="/pmc/articles/instance/2842525/bin/WJG-16-1337-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2842525/" ref="ordinalpos=4194&amp;ncbi_uid=2269163&amp;link_uid=PMC2842525" image-link="/pmc/articles/PMC2842525/figure/F2/" class="imagepopup">Figure 2.  From: Impact of alcohol on hepatitis C virus replication and interferon <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>IFN- signal transduction. Binding of IFN- to its cognate receptor on the cell surface results in activation of the Janus activated kinase (JAK)/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> culminating in the production of interferon-stimulated genes (ISGs), many of which have anti-viral properties that act to limit HCV infection. Alcohol inhibits Y701, decreases interferon-stimulated response element (ISRE) activity, and subsequently dampens the ISG response.</div></p></div><div class="aux"><div class="resc">Erin M McCartney, et al. World J Gastroenterol. 2010 March 21;16(11):1337-1343.</div><p class="links"><a class="dblinks" href="/pubmed/20238400" ref="ordinalpos=4194&amp;ncbi_uid=2269163&amp;link_uid=20238400">Citation</a><a class="dblinks" href="/pmc/articles/PMC2842525/" ref="ordinalpos=4194&amp;ncbi_uid=2269163&amp;link_uid=PMC2842525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4195.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4145548/figure/fig1/" ref="ordinalpos=4195&amp;ncbi_uid=6551460&amp;link_uid=PMC4145548" href="/pmc/articles/PMC4145548/"><img src="/pmc/articles/instance/4145548/bin/BMRI2014-509149.001.gif" src-large="/pmc/articles/instance/4145548/bin/BMRI2014-509149.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4145548/" ref="ordinalpos=4195&amp;ncbi_uid=6551460&amp;link_uid=PMC4145548" image-link="/pmc/articles/PMC4145548/figure/fig1/" class="imagepopup">Figure 1.  From: FSim: A Novel Functional Similarity Search Algorithm and Tool for Discovering Functionally Related Gene Products. </a></p><div class="supp"><p class="details"><div>The distribution of FSim-calculated functional similarity scores for comparing the genes in the P53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (in red) with all the other genes (in green). The density plot and box-whisker plots of the two groups of functional similarity scores are shown in (a) and (b), separately. The figures show that the average functional similarity scores for genes within P53 <span class="highlight" style="background-color:">pathway</span> and the other genes which are not in P53 <span class="highlight" style="background-color:">pathway</span> are around 2 and 1, respectively.</div></p></div><div class="aux"><div class="resc">Qiang Hu, et al. Biomed Res Int. 2014;2014:509149.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4195&amp;ncbi_uid=6551460&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4145548/" ref="ordinalpos=4195&amp;ncbi_uid=6551460&amp;link_uid=PMC4145548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4196.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3274430/figure/F4/" ref="ordinalpos=4196&amp;ncbi_uid=3793521&amp;link_uid=PMC3274430" href="/pmc/articles/PMC3274430/"><img src="/pmc/articles/instance/3274430/bin/1471-2407-12-21-4.gif" src-large="/pmc/articles/instance/3274430/bin/1471-2407-12-21-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3274430/" ref="ordinalpos=4196&amp;ncbi_uid=3793521&amp;link_uid=PMC3274430" image-link="/pmc/articles/PMC3274430/figure/F4/" class="imagepopup">Figure 4.  From: Gene expression in tumor cells and stroma in dsRed 4T1 tumors in eGFP-expressing mice with and without enhanced oxygenation. </a></p><div class="supp"><p class="details"><div><b>MAPK <span class="highlight" style="background-color:">pathway</span>, tumor (a) and stroma (b)</b>. Heatmap of genes in the MAPK <span class="highlight" style="background-color:">pathway</span> (HSA04010). The table display gene names, systematic names, and fold change of gene expression compared to control of intermittent (I) and daily (D) hyperbaric oxygen (HBO), respectively. Intermittent and daily HBO treatment suppressed MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in both tumor cells and stromal cells compared to control. NS = not significant.</div></p></div><div class="aux"><div class="resc">Ingrid Moen, et al. BMC Cancer. 2012;12:21-21.</div><p class="links"><a class="dblinks" href="/pubmed/22251838" ref="ordinalpos=4196&amp;ncbi_uid=3793521&amp;link_uid=22251838">Citation</a><a class="dblinks" href="/pmc/articles/PMC3274430/" ref="ordinalpos=4196&amp;ncbi_uid=3793521&amp;link_uid=PMC3274430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4197.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3080176/figure/F1/" ref="ordinalpos=4197&amp;ncbi_uid=3246765&amp;link_uid=PMC3080176" href="/pmc/articles/PMC3080176/"><img src="/pmc/articles/instance/3080176/bin/z7g9991113290001.gif" src-large="/pmc/articles/instance/3080176/bin/z7g9991113290001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3080176/" ref="ordinalpos=4197&amp;ncbi_uid=3246765&amp;link_uid=PMC3080176" image-link="/pmc/articles/PMC3080176/figure/F1/" class="imagepopup">Figure 1. From: Microarray Analysis of XOPS-mCFP Zebrafish Retina Identifies Genes Associated with Rod Photoreceptor Degeneration and Regeneration. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of differentially expressed genes. (<b>A</b>) Top ten biological functions associated with gene expression changes between wild-type and XOPS-mCFP retinas. The <i>y</i>-axis represents the negative log of the p-value associated with the probability that each biological function was identified due to chance alone. (<b>B</b>) Top eleven canonical <span class="highlight" style="background-color:">signaling</span> pathways associated with differentially expressed genes. The blue bars represent the negative log of the p-value associated with the probability that each <span class="highlight" style="background-color:">pathway</span> was identified due to chance alone. The orange squares represent a ratio of the number of molecules from the microarray dataset that were associated with a given canonical <span class="highlight" style="background-color:">pathway</span> divided by the total number of molecules that map to that <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ann C. Morris, et al. Invest Ophthalmol Vis Sci. 2011 April;52(5):2255-2266.</div><p class="links"><a class="dblinks" href="/pubmed/21217106" ref="ordinalpos=4197&amp;ncbi_uid=3246765&amp;link_uid=21217106">Citation</a><a class="dblinks" href="/pmc/articles/PMC3080176/" ref="ordinalpos=4197&amp;ncbi_uid=3246765&amp;link_uid=PMC3080176">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4198.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3784453/figure/pone-0075462-g010/" ref="ordinalpos=4198&amp;ncbi_uid=5361871&amp;link_uid=PMC3784453" href="/pmc/articles/PMC3784453/"><img src="/pmc/articles/instance/3784453/bin/pone.0075462.g010.gif" src-large="/pmc/articles/instance/3784453/bin/pone.0075462.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3784453/" ref="ordinalpos=4198&amp;ncbi_uid=5361871&amp;link_uid=PMC3784453" image-link="/pmc/articles/PMC3784453/figure/pone-0075462-g010/" class="imagepopup">Figure 10. The proposed mechanism of MLPL in regulation of energy metabolism..  From: Leu452His Mutation in Lipoprotein Lipase Gene Transfer Associated with Hypertriglyceridemia in Mice in vivo. </a></p><div class="supp"><p class="details"><div>MLPL regulates energy metabolism via inhibition of the AMPK/SIRT1/PCG1- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Kaiyue Sun, et al. PLoS One. 2013;8(9):e75462.</div><p class="links"><a class="dblinks" href="/pubmed/24086538" ref="ordinalpos=4198&amp;ncbi_uid=5361871&amp;link_uid=24086538">Citation</a><a class="dblinks" href="/pmc/articles/PMC3784453/" ref="ordinalpos=4198&amp;ncbi_uid=5361871&amp;link_uid=PMC3784453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4199.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3416961/figure/F1/" ref="ordinalpos=4199&amp;ncbi_uid=5483962&amp;link_uid=PMC3416961" href="/pmc/articles/PMC3416961/"><img src="/pmc/articles/instance/3416961/bin/nihms368117f1.gif" src-large="/pmc/articles/instance/3416961/bin/nihms368117f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3416961/" ref="ordinalpos=4199&amp;ncbi_uid=5483962&amp;link_uid=PMC3416961" image-link="/pmc/articles/PMC3416961/figure/F1/" class="imagepopup">Figure 1.  From: Clinical and Proteomic Characterization of Acute Myeloid Leukemia with Mutated RAS. </a></p><div class="supp"><p class="details"><div>Simplified diagram depicting Ras <span class="highlight" style="background-color:">signaling</span> and crosstalk with PI3 kinase- Akt <span class="highlight" style="background-color:">pathway</span> in leukemia.</div></p></div><div class="aux"><div class="resc">Tapan M. Kadia, et al. Cancer. ;118(22):5550-5559.</div><p class="links"><a class="dblinks" href="/pubmed/22569880" ref="ordinalpos=4199&amp;ncbi_uid=5483962&amp;link_uid=22569880">Citation</a><a class="dblinks" href="/pmc/articles/PMC3416961/" ref="ordinalpos=4199&amp;ncbi_uid=5483962&amp;link_uid=PMC3416961">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4200.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3161999/figure/pone-0023802-g002/" ref="ordinalpos=4200&amp;ncbi_uid=3208008&amp;link_uid=PMC3161999" href="/pmc/articles/PMC3161999/"><img src="/pmc/articles/instance/3161999/bin/pone.0023802.g002.gif" src-large="/pmc/articles/instance/3161999/bin/pone.0023802.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3161999/" ref="ordinalpos=4200&amp;ncbi_uid=3208008&amp;link_uid=PMC3161999" image-link="/pmc/articles/PMC3161999/figure/pone-0023802-g002/" class="imagepopup">Figure 2. Heatmap illustrating the pairs of microRNAs and target mRNAs for the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in response to -irradiation in H1299 cells..  From: Coordinated Regulation of ATF2 by miR-26b in ?-Irradiated Lung Cancer Cells. </a></p><div class="aux"><div class="resc">Himanshu Arora, et al. PLoS One. 2011;6(8):e23802.</div><p class="links"><a class="dblinks" href="/pubmed/21901137" ref="ordinalpos=4200&amp;ncbi_uid=3208008&amp;link_uid=21901137">Citation</a><a class="dblinks" href="/pmc/articles/PMC3161999/" ref="ordinalpos=4200&amp;ncbi_uid=3208008&amp;link_uid=PMC3161999">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="41" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="42" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="43" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:46:03-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A3662543128210000000000C4D034;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A3662543128210000000000C4D034" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>