
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="5" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB244753FD3EB10000000000EEC8FD" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="4" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="5" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="6" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="5" /></div>
            <div><h2 class="result_count">Results: 401 to 500 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>401.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3832994/figure/F3/" ref="ordinalpos=401&amp;ncbi_uid=5551588&amp;link_uid=PMC3832994" href="/pmc/articles/PMC3832994/"><img src="/pmc/articles/instance/3832994/bin/nihms-412631-f0003.gif" src-large="/pmc/articles/instance/3832994/bin/nihms-412631-f0003.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3832994/" ref="ordinalpos=401&amp;ncbi_uid=5551588&amp;link_uid=PMC3832994" image-link="/pmc/articles/PMC3832994/figure/F3/" class="imagepopup">FIGURE 3.  From: Role of Epidermal Growth Factor-Triggered PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> in the Migration of Medulloblastoma-Derived Cells. </a></p><div class="supp"><p class="details"><div>Akt phosphorylation levels and localizations are mediated by EGF <span class="highlight" style="background-color:">pathway</span> stimulation and inhibition in MB-derived cells. Localization of phosphorylated Akt (pAkt) in non-stimulated cells (a), in cells stimulated with 1000 ng/mL EGF (b) cells in which EGF <span class="highlight" style="background-color:">signaling</span> was inhibited by 1 <i>μ</i>M Tarceva (c), and cells in which PI3K <span class="highlight" style="background-color:">signaling</span> was inhibited by 0.5 <i>μ</i>M Wortmannin (d). Localization of phosphorylated pERK in non-stimulated cells (e), in cells stimulated with 1000 ng/mL EGF (<i>F</i>), cells in which EGF <span class="highlight" style="background-color:">signaling</span> was inhibited by 1 <i>μ</i>M Tarceva (g), and cells in which PI3K <span class="highlight" style="background-color:">signaling</span> was inhibited by Wortmannin (h). Scale bars, 20 <i>μ</i>m. Quantification of pAkt (i) and pERK (j) in non-stimulated cells, cells stimulated with EGF, cells in which EGF <span class="highlight" style="background-color:">signaling</span> was inhibited with Tarceva and cells in which PI3K <span class="highlight" style="background-color:">signaling</span> was inhibited with Wortmannin. * <i>p</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Veronica Dudu, et al. Cell Mol Bioeng. ;5(4):502-413.</div><p class="links"><a class="dblinks" href="/pubmed/24273611" ref="ordinalpos=401&amp;ncbi_uid=5551588&amp;link_uid=24273611">Citation</a><a class="dblinks" href="/pmc/articles/PMC3832994/" ref="ordinalpos=401&amp;ncbi_uid=5551588&amp;link_uid=PMC3832994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>402.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F6/" ref="ordinalpos=402&amp;ncbi_uid=2525563&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-6.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=402&amp;ncbi_uid=2525563&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F6/" class="imagepopup">Figure 6.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Triangulated graph <i>D<sup>t </sup></i>obtained from BCR <span class="highlight" style="background-color:">pathway</span></b>. Triangulated graph <i>D<sup>t </sup></i>corresponding to the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=402&amp;ncbi_uid=2525563&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=402&amp;ncbi_uid=2525563&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>403.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F5/" ref="ordinalpos=403&amp;ncbi_uid=2525564&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-5.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=403&amp;ncbi_uid=2525564&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F5/" class="imagepopup">Figure 5.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Moral graph <i>D<sup>m </sup></i>obtained from BCR <span class="highlight" style="background-color:">pathway</span></b>. Moral graph <i>D<sup>m </sup></i>corresponding to the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=403&amp;ncbi_uid=2525564&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=403&amp;ncbi_uid=2525564&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>404.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3445776/figure/F1/" ref="ordinalpos=404&amp;ncbi_uid=5305730&amp;link_uid=PMC3445776" href="/pmc/articles/PMC3445776/"><img src="/pmc/articles/instance/3445776/bin/nihms-385597-f0001.gif" src-large="/pmc/articles/instance/3445776/bin/nihms-385597-f0001.jpg" alt="figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3445776/" ref="ordinalpos=404&amp;ncbi_uid=5305730&amp;link_uid=PMC3445776" image-link="/pmc/articles/PMC3445776/figure/F1/" class="imagepopup">figure 1.  From: Molecular Pathways: The Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Cancer. </a></p><div class="supp"><p class="details"><div>The Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Positive and negative regulatory components are depicted in green and red, respectively. A. In the absence of HH ligand, PTCH1 inhibits SMO allowing the GLI processing complex containing SUFU to generate GLIr transcriptional repressors. B. HH ligand binding to PTCH depresses SMO and generates GLIa nuclear factors that induce the expression of Hh target genes. Clinical and preclinical inhibitors of <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> are listed at their sites of <span class="highlight" style="background-color:">pathway</span> activity.</div></p></div><div class="aux"><div class="resc">Ross McMillan, et al. Clin Cancer Res. 2012 September 15;18(18):4883-4888.</div><p class="links"><a class="dblinks" href="/pubmed/22718857" ref="ordinalpos=404&amp;ncbi_uid=5305730&amp;link_uid=22718857">Citation</a><a class="dblinks" href="/pmc/articles/PMC3445776/" ref="ordinalpos=404&amp;ncbi_uid=5305730&amp;link_uid=PMC3445776">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>405.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2778684/figure/Fig1/" ref="ordinalpos=405&amp;ncbi_uid=2130364&amp;link_uid=PMC2778684" href="/pmc/articles/PMC2778684/"><img src="/pmc/articles/instance/2778684/bin/18_2008_Article_8042_Fig1.gif" src-large="/pmc/articles/instance/2778684/bin/18_2008_Article_8042_Fig1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2778684/" ref="ordinalpos=405&amp;ncbi_uid=2130364&amp;link_uid=PMC2778684" image-link="/pmc/articles/PMC2778684/figure/Fig1/" class="imagepopup">Figure 1.  From: Mesodermal fate decisions of a stem cell: the Wnt switch. </a></p><div class="supp"><p class="details"><div>Canonical and non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> pathways. There are at least three different intracellular Wnt <span class="highlight" style="background-color:">signaling</span> transduction pathways. These pathways are the Wnt/CatnB <span class="highlight" style="background-color:">pathway</span> (canonical <span class="highlight" style="background-color:">pathway</span>), the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> and the Wnt/planar cell polarity <span class="highlight" style="background-color:">pathway</span>. Both the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> and the Wnt/planar cell polarity <span class="highlight" style="background-color:">pathway</span> are combined as the non-canonical <span class="highlight" style="background-color:">pathway</span>, which involve the downstream members calmodulin kinase II (Cam-KII), protein kinase C (PKC) and c-jun kinase (JNK). See text for more details.</div></p></div><div class="aux"><div class="resc">L. A. Davis, et al. Cell Mol Life Sci. 2008 September;65(17):2658-2674.</div><p class="links"><a class="dblinks" href="/pubmed/18528633" ref="ordinalpos=405&amp;ncbi_uid=2130364&amp;link_uid=18528633">Citation</a><a class="dblinks" href="/pmc/articles/PMC2778684/" ref="ordinalpos=405&amp;ncbi_uid=2130364&amp;link_uid=PMC2778684">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>406.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3069900/figure/fig7/" ref="ordinalpos=406&amp;ncbi_uid=3571339&amp;link_uid=PMC3069900" href="/pmc/articles/PMC3069900/"><img src="/pmc/articles/instance/3069900/bin/gr7.gif" src-large="/pmc/articles/instance/3069900/bin/gr7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3069900/" ref="ordinalpos=406&amp;ncbi_uid=3571339&amp;link_uid=PMC3069900" image-link="/pmc/articles/PMC3069900/figure/fig7/" class="imagepopup">Figure 7.  From: Fibroblast Growth Factor-2 Promotes in Vitro Mitral Valve Interstitial Cell Repair through Transforming Growth Factor-?/Smad <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Summary of a putative <span class="highlight" style="background-color:">pathway</span> that regulates FGF-2–mediated wound repair through the TGF-β/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The TGF-β promotes the phosphorylation of Smad2/3, resulting in VIC wound repair. Betaglycan binds endogenous TGF-β, resulting in less available TGF-β to bind to its <span class="highlight" style="background-color:">signaling</span> receptors. The FGF-2 also binds betaglycan, which makes more TGF-β available for receptor <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Li Han, et al. Am J Pathol. 2011 January;178(1):119-127.</div><p class="links"><a class="dblinks" href="/pubmed/21224050" ref="ordinalpos=406&amp;ncbi_uid=3571339&amp;link_uid=21224050">Citation</a><a class="dblinks" href="/pmc/articles/PMC3069900/" ref="ordinalpos=406&amp;ncbi_uid=3571339&amp;link_uid=PMC3069900">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>407.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3328790/figure/F1/" ref="ordinalpos=407&amp;ncbi_uid=4478611&amp;link_uid=PMC3328790" href="/pmc/articles/PMC3328790/"><img src="/pmc/articles/instance/3328790/bin/nihms335394f1.gif" src-large="/pmc/articles/instance/3328790/bin/nihms335394f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3328790/" ref="ordinalpos=407&amp;ncbi_uid=4478611&amp;link_uid=PMC3328790" image-link="/pmc/articles/PMC3328790/figure/F1/" class="imagepopup">Fig. 1.  From: Transforming growth factor beta <span class="highlight" style="background-color:">signaling</span> in adult cardiovascular diseases and repair. </a></p><div class="supp"><p class="details"><div>TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its integration with other major pathways that is likely to be involved in the pathogenesis of cardiovascular diseases in elderly patients. The canonical TGFβ <span class="highlight" style="background-color:">pathway</span> involves SMADs. The non-canonical TGFβ pathways include several SMAD-independent <span class="highlight" style="background-color:">signaling</span> cascades such as calcium-calcineurin and MAPK pathways. The Ang II <span class="highlight" style="background-color:">pathway</span> can interact with TGFβ <span class="highlight" style="background-color:">signaling</span> through both SMAD-dependent and SMAD-independent pathways (see text for details).</div></p></div><div class="aux"><div class="resc">Thomas Doetschman, et al. Cell Tissue Res. ;347(1):203-223.</div><p class="links"><a class="dblinks" href="/pubmed/21953136" ref="ordinalpos=407&amp;ncbi_uid=4478611&amp;link_uid=21953136">Citation</a><a class="dblinks" href="/pmc/articles/PMC3328790/" ref="ordinalpos=407&amp;ncbi_uid=4478611&amp;link_uid=PMC3328790">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>408.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4322417/figure/F6/" ref="ordinalpos=408&amp;ncbi_uid=7163298&amp;link_uid=PMC4322417" href="/pmc/articles/PMC4322417/"><img src="/pmc/articles/instance/4322417/bin/nihms659902f6.gif" src-large="/pmc/articles/instance/4322417/bin/nihms659902f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4322417/" ref="ordinalpos=408&amp;ncbi_uid=7163298&amp;link_uid=PMC4322417" image-link="/pmc/articles/PMC4322417/figure/F6/" class="imagepopup">Figure 6. GPCR <span class="highlight" style="background-color:">signaling</span>.  From: G protein-coupled receptors — recent advances. </a></p><div class="supp"><p class="details"><div>In response to ligand binding a stimulation <span class="highlight" style="background-color:">signaling</span> can occur <i>via</i> G-protein-mediated <span class="highlight" style="background-color:">pathway</span> terminated by subsequent GRK/arrestin binding, or/and <i>via</i> β-arrestin-mediated <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Dorota Latek, et al. Acta Biochim Pol. ;59(4):515-529.</div><p class="links"><a class="dblinks" href="/pubmed/23251911" ref="ordinalpos=408&amp;ncbi_uid=7163298&amp;link_uid=23251911">Citation</a><a class="dblinks" href="/pmc/articles/PMC4322417/" ref="ordinalpos=408&amp;ncbi_uid=7163298&amp;link_uid=PMC4322417">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>409.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC95007/figure/F7/" ref="ordinalpos=409&amp;ncbi_uid=152016&amp;link_uid=PMC95007" href="/pmc/articles/PMC95007/"><img src="/pmc/articles/instance/95007/bin/jb0411265007.gif" src-large="/pmc/articles/instance/95007/bin/jb0411265007.jpg" alt="FIG. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC95007/" ref="ordinalpos=409&amp;ncbi_uid=152016&amp;link_uid=PMC95007" image-link="/pmc/articles/PMC95007/figure/F7/" class="imagepopup">FIG. 7.  From: New Family of Regulators in the Environmental <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Which Activates the General Stress Transcription Factor ?B of Bacillus subtilis. </a></p><div class="supp"><p class="details"><div>Cartoon summarizing the interactions among RsbR, its paralogs, and RsbS. As shown in Table 2, RsbR, YkoB, YojH, YqhA, and the fused YetI-YezB proteins interacted with each other and with the environmental <span class="highlight" style="background-color:">signaling</span> antagonist protein RsbS when paired in the yeast two-hybrid system. In contrast, YtvA interacted only with RsbR and YqhA. None of the newly characterized RsbR paralogs interacted with the downstream RsbT or RsbU regulator from the environmental <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, with RsbV or RsbW from the common part of the ς<sup>B</sup> regulatory <span class="highlight" style="background-color:">pathway</span>, or with SpoIIAA or SpoIIAB from the ς<sup>F</sup> regulatory <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Samina Akbar, et al. J Bacteriol. 2001 February;183(4):1329-1338.</div><p class="links"><a class="dblinks" href="/pubmed/11157946" ref="ordinalpos=409&amp;ncbi_uid=152016&amp;link_uid=11157946">Citation</a><a class="dblinks" href="/pmc/articles/PMC95007/" ref="ordinalpos=409&amp;ncbi_uid=152016&amp;link_uid=PMC95007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>410.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3297971/figure/F4/" ref="ordinalpos=410&amp;ncbi_uid=4665248&amp;link_uid=PMC3297971" href="/pmc/articles/PMC3297971/"><img src="/pmc/articles/instance/3297971/bin/nihms346928f4.gif" src-large="/pmc/articles/instance/3297971/bin/nihms346928f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3297971/" ref="ordinalpos=410&amp;ncbi_uid=4665248&amp;link_uid=PMC3297971" image-link="/pmc/articles/PMC3297971/figure/F4/" class="imagepopup">Figure 4.  From: Functional Protein <span class="highlight" style="background-color:">Pathway</span> Activation Mapping of the Progression of Normal Skin to Squamous Cell Carcinoma. </a></p><div class="supp"><p class="details"><div>Protein <span class="highlight" style="background-color:">Pathway</span> Activation Map of SCC. A Cancer Landscape (CScape) Protein <span class="highlight" style="background-color:">Pathway</span> Activation Map (4 Top left) is shown using population averages of RPMA data from patients with AK and SCC. Higher fold differences in the SCC group are shown in increasing shades of red while higher fold differences in <span class="highlight" style="background-color:">signaling</span> in the AK group are shown in green. Each balloon pin is placed over the protein measured. Magnified views of the AMPK-mTOR <span class="highlight" style="background-color:">pathway</span> (green box, middle top) and RAS-RAF-ERK <span class="highlight" style="background-color:">pathway</span> (red box, middle bottom) are shown to reveal <span class="highlight" style="background-color:">pathway</span> detail. Linked <span class="highlight" style="background-color:">pathway</span> members from each <span class="highlight" style="background-color:">pathway</span> are numbered and the individual protein data are plotted as scatter plots for the statistically significant endpoints.</div></p></div><div class="aux"><div class="resc">Janine G. Einspahr, et al. Cancer Prev Res (Phila). ;5(3):403-413.</div><p class="links"><a class="dblinks" href="/pubmed/22389437" ref="ordinalpos=410&amp;ncbi_uid=4665248&amp;link_uid=22389437">Citation</a><a class="dblinks" href="/pmc/articles/PMC3297971/" ref="ordinalpos=410&amp;ncbi_uid=4665248&amp;link_uid=PMC3297971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>411.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3629821/figure/F6/" ref="ordinalpos=411&amp;ncbi_uid=5989881&amp;link_uid=PMC3629821" href="/pmc/articles/PMC3629821/"><img src="/pmc/articles/instance/3629821/bin/nihms-451469-f0006.gif" src-large="/pmc/articles/instance/3629821/bin/nihms-451469-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3629821/" ref="ordinalpos=411&amp;ncbi_uid=5989881&amp;link_uid=PMC3629821" image-link="/pmc/articles/PMC3629821/figure/F6/" class="imagepopup">Figure 6. Model for TXNIP-mediated VEGFR2 <span class="highlight" style="background-color:">signaling</span> in EC angiogenesis.  From: Thioredoxin-Interacting Protein Mediates Sustained VEGFR2 <span class="highlight" style="background-color:">Signaling</span> in Endothelial Cells required for Angiogenesis. </a></p><div class="supp"><p class="details"><div>VEGF triggers autophosphorylation of VEGFR2 (Y1175), which is internalized inside endocytotic vesicles and leads to PLCγ1 phosphorylation in TXNIP-dependent. TXNIP sustains VEGFR2 <span class="highlight" style="background-color:">signaling</span> that required for EC tube formation and proliferation in angiogenesis. On the other hand, transient VEGFR2 <span class="highlight" style="background-color:">signaling</span> (p-AKT/p-eNOS <span class="highlight" style="background-color:">pathway</span>) is independent of TXNIP.</div></p></div><div class="aux"><div class="resc">Shin-Young Park, et al. Arterioscler Thromb Vasc Biol. 2013 April;33(4):737-743.</div><p class="links"><a class="dblinks" href="/pubmed/23393387" ref="ordinalpos=411&amp;ncbi_uid=5989881&amp;link_uid=23393387">Citation</a><a class="dblinks" href="/pmc/articles/PMC3629821/" ref="ordinalpos=411&amp;ncbi_uid=5989881&amp;link_uid=PMC3629821">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>412.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3795364/figure/cancers-05-00762-f006/" ref="ordinalpos=412&amp;ncbi_uid=5404760&amp;link_uid=PMC3795364" href="/pmc/articles/PMC3795364/"><img src="/pmc/articles/instance/3795364/bin/cancers-05-00762-g006.gif" src-large="/pmc/articles/instance/3795364/bin/cancers-05-00762-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3795364/" ref="ordinalpos=412&amp;ncbi_uid=5404760&amp;link_uid=PMC3795364" image-link="/pmc/articles/PMC3795364/figure/cancers-05-00762-f006/" class="imagepopup">Figure 6.  From: Influence of Levamisole and Other Angiogenesis Inhibitors on Angiogenesis and Endothelial Cell Morphology in Vitro. </a></p><div class="supp"><p class="details"><div>Schematic presentation of the influence of angiogenesis inhibitors on EC <span class="highlight" style="background-color:">signaling</span>. Important <span class="highlight" style="background-color:">signaling</span> pathways in ECs are the VEGF <span class="highlight" style="background-color:">pathway</span>, the ECM integrin <span class="highlight" style="background-color:">pathway</span> and the mTOR <span class="highlight" style="background-color:">pathway</span>. Blocking of the VEGF <span class="highlight" style="background-color:">pathway</span> by VEGF antibodies, SU5614, levamisole or PTPI IV inhibits differentiation. Blocking of ECM integrin <span class="highlight" style="background-color:">signaling</span> by TSP or mTOR <span class="highlight" style="background-color:">signaling</span> by rapamycin inhibits proliferation as does salubrinal and TNP-470. PF4, suramin, NSAIDs, clodronate, NSC87877, PTPI I, PTPI II inhibit EC proliferation in a morphologically similar way, while ibandronate gives mixed morphology, but the exact relationships between these with the major <span class="highlight" style="background-color:">signaling</span> pathways are unknown.</div></p></div><div class="aux"><div class="resc">Tina Friis, et al. Cancers (Basel). 2013 September;5(3):762-785.</div><p class="links"><a class="dblinks" href="/pubmed/24202320" ref="ordinalpos=412&amp;ncbi_uid=5404760&amp;link_uid=24202320">Citation</a><a class="dblinks" href="/pmc/articles/PMC3795364/" ref="ordinalpos=412&amp;ncbi_uid=5404760&amp;link_uid=PMC3795364">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>413.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3094853/figure/F1/" ref="ordinalpos=413&amp;ncbi_uid=2954034&amp;link_uid=PMC3094853" href="/pmc/articles/PMC3094853/"><img src="/pmc/articles/instance/3094853/bin/nihms254819f1.gif" src-large="/pmc/articles/instance/3094853/bin/nihms254819f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3094853/" ref="ordinalpos=413&amp;ncbi_uid=2954034&amp;link_uid=PMC3094853" image-link="/pmc/articles/PMC3094853/figure/F1/" class="imagepopup">Figure 1.  From: Unnatural Polyketide Analogues Selectively Target the HER <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div>Schematic representation of HER2-mediated PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activation of HER2 initiates the activation of the cytoplasmic tyrosine kinase domain and subsequent events to activate downstream of PI3K and Akt. The activation of Akt suppresses apoptosis and stimulates the cell survival <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Seok Joon Kwon, et al. Chembiochem. ;11(4):573-580.</div><p class="links"><a class="dblinks" href="/pubmed/20058253" ref="ordinalpos=413&amp;ncbi_uid=2954034&amp;link_uid=20058253">Citation</a><a class="dblinks" href="/pmc/articles/PMC3094853/" ref="ordinalpos=413&amp;ncbi_uid=2954034&amp;link_uid=PMC3094853">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>414.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3973863/figure/F7/" ref="ordinalpos=414&amp;ncbi_uid=5971858&amp;link_uid=PMC3973863" href="/pmc/articles/PMC3973863/"><img src="/pmc/articles/instance/3973863/bin/1476-4598-13-48-7.gif" src-large="/pmc/articles/instance/3973863/bin/1476-4598-13-48-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3973863/" ref="ordinalpos=414&amp;ncbi_uid=5971858&amp;link_uid=PMC3973863" image-link="/pmc/articles/PMC3973863/figure/F7/" class="imagepopup">Figure 7.  From: Chemoprevention of dietary digitoflavone on colitis-associated colon tumorigenesis through inducing Nrf2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and inhibition of inflammation. </a></p><div class="supp"><p class="details"><div><b>A proposed <span class="highlight" style="background-color:">pathway</span> for digitoflavone-induced Nrf2/ARE-mediated cytoprotective proteins via activation of p38 MAPK <span class="highlight" style="background-color:">signaling</span>.</b> Up-regulation of γ-GCSc, γ-GCSm explains the cytoprotective effects against oxidative stress in Caco-2 cells.</div></p></div><div class="aux"><div class="resc">Yang Yang, et al. Mol Cancer. 2014;13:48-48.</div><p class="links"><a class="dblinks" href="/pubmed/24602443" ref="ordinalpos=414&amp;ncbi_uid=5971858&amp;link_uid=24602443">Citation</a><a class="dblinks" href="/pmc/articles/PMC3973863/" ref="ordinalpos=414&amp;ncbi_uid=5971858&amp;link_uid=PMC3973863">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>415.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3440368/figure/pone-0044856-g007/" ref="ordinalpos=415&amp;ncbi_uid=4236419&amp;link_uid=PMC3440368" href="/pmc/articles/PMC3440368/"><img src="/pmc/articles/instance/3440368/bin/pone.0044856.g007.gif" src-large="/pmc/articles/instance/3440368/bin/pone.0044856.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3440368/" ref="ordinalpos=415&amp;ncbi_uid=4236419&amp;link_uid=PMC3440368" image-link="/pmc/articles/PMC3440368/figure/pone-0044856-g007/" class="imagepopup">Figure 7. A schematic representation of the relaxin <span class="highlight" style="background-color:">pathway</span> when disturbed by simazine..  From: Disturbed Relaxin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Testicular Dysfunction in Mouse Offspring upon Maternal Exposure to Simazine. </a></p><div class="supp"><p class="details"><div>Simazine inhibits testicular Rxfp1 expression, which consequently may limit the intracellular <span class="highlight" style="background-color:">signaling</span> of relaxin in the testis. The compromised relaxin-Rxfp1 <span class="highlight" style="background-color:">signaling</span> is further expected to downregulate the expression of relaxin target genes such as Nos2, Nos3, Vegf, and Mmp9, leading to the inhibition of NO release.</div></p></div><div class="aux"><div class="resc">Ho-Oak Park, et al. PLoS One. 2012;7(9):e44856.</div><p class="links"><a class="dblinks" href="/pubmed/22984576" ref="ordinalpos=415&amp;ncbi_uid=4236419&amp;link_uid=22984576">Citation</a><a class="dblinks" href="/pmc/articles/PMC3440368/" ref="ordinalpos=415&amp;ncbi_uid=4236419&amp;link_uid=PMC3440368">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>416.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4385844/figure/F7/" ref="ordinalpos=416&amp;ncbi_uid=7416934&amp;link_uid=PMC4385844" href="/pmc/articles/PMC4385844/"><img src="/pmc/articles/instance/4385844/bin/oncotarget-06-2180-g007.gif" src-large="/pmc/articles/instance/4385844/bin/oncotarget-06-2180-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4385844/" ref="ordinalpos=416&amp;ncbi_uid=7416934&amp;link_uid=PMC4385844" image-link="/pmc/articles/PMC4385844/figure/F7/" class="imagepopup">Figure 7. A model for γ-H2AX action in HCC angiogenesis via EGFR/HIF-1α/VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: γ-H2AX promotes hepatocellular carcinoma angiogenesis via EGFR/HIF-1α/VEGF pathways under hypoxic condition. </a></p><div class="supp"><p class="details"><div>This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was drawn by the <span class="highlight" style="background-color:">Pathway</span> Builder Tool 2.0. Hypoxia can induce the expression of γ-H2AX via cleavage of the DNA double-strand. The induced γ-H2AX could promote EGFR to translocate into the cell nucleus from the cell membrane through regulate COX2 expression. Nuclear localized EGFR was found to induce HIF-1α expression which resulted in VEGF expression and angiogenesis.</div></p></div><div class="aux"><div class="resc">Heng Xiao, et al. Oncotarget. 2015 February;6(4):2180-2192.</div><p class="links"><a class="dblinks" href="/pubmed/25537504" ref="ordinalpos=416&amp;ncbi_uid=7416934&amp;link_uid=25537504">Citation</a><a class="dblinks" href="/pmc/articles/PMC4385844/" ref="ordinalpos=416&amp;ncbi_uid=7416934&amp;link_uid=PMC4385844">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>417.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F7/" ref="ordinalpos=417&amp;ncbi_uid=2525562&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-7.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=417&amp;ncbi_uid=2525562&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F7/" class="imagepopup">Figure 7.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>DAG <i>D </i>obtained from ERBB <span class="highlight" style="background-color:">pathway</span></b>. DAG <i>D </i>corresponding to the chosen portion of ERBB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=417&amp;ncbi_uid=2525562&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=417&amp;ncbi_uid=2525562&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>418.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3642174/figure/pone-0062218-g001/" ref="ordinalpos=418&amp;ncbi_uid=4879562&amp;link_uid=PMC3642174" href="/pmc/articles/PMC3642174/"><img src="/pmc/articles/instance/3642174/bin/pone.0062218.g001.gif" src-large="/pmc/articles/instance/3642174/bin/pone.0062218.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3642174/" ref="ordinalpos=418&amp;ncbi_uid=4879562&amp;link_uid=PMC3642174" image-link="/pmc/articles/PMC3642174/figure/pone-0062218-g001/" class="imagepopup">Figure 1. Illustration of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> considered in this study..  From: Influences of Excluded Volume of Molecules on <span class="highlight" style="background-color:">Signaling</span> Processes on the Biomembrane. </a></p><div class="supp"><p class="details"><div>(1) The extracellular signal is transferred via activation of the receptor (Equation [Eq.] 1). (2) Autonomous inactivation of the receptor (Eq. 2). (3) Activation of the <span class="highlight" style="background-color:">signaling</span> protein by the active receptor (Eq. 3). (4) Autonomous inactivation of the <span class="highlight" style="background-color:">signaling</span> protein (Eq. 4). (5) Stochastic binding of the target protein from the cytoplasm to the membrane (Eq. 5). (6) Activation and unbinding of target proteins by active <span class="highlight" style="background-color:">signaling</span> proteins (Eq. 6). (7) Autonomous unbinding of the target protein (Eq. 7).</div></p></div><div class="aux"><div class="resc">Masashi Fujii, et al. PLoS One. 2013;8(5):e62218.</div><p class="links"><a class="dblinks" href="/pubmed/23658714" ref="ordinalpos=418&amp;ncbi_uid=4879562&amp;link_uid=23658714">Citation</a><a class="dblinks" href="/pmc/articles/PMC3642174/" ref="ordinalpos=418&amp;ncbi_uid=4879562&amp;link_uid=PMC3642174">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>419.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g005/" ref="ordinalpos=419&amp;ncbi_uid=4989292&amp;link_uid=PMC3677896" href="/pmc/articles/PMC3677896/"><img src="/pmc/articles/instance/3677896/bin/pone.0065032.g005.gif" src-large="/pmc/articles/instance/3677896/bin/pone.0065032.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3677896/" ref="ordinalpos=419&amp;ncbi_uid=4989292&amp;link_uid=PMC3677896" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g005/" class="imagepopup">Figure 5. Effects of SHH <span class="highlight" style="background-color:">signaling</span> agonists on dying cell induced tumor cell growth..  From: Sonic Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Supports Cancer Cell Growth during Cancer Radiotherapy. </a></p><div class="supp"><p class="details"><div>A. Activation of SHH <span class="highlight" style="background-color:">signaling</span> with SHH <span class="highlight" style="background-color:">signaling</span> agonist SAG enhances Panc1Fluc cell growth on irradiated dying Panc1 cells in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. *represents <i>P</i>&lt;0.05, **represents <i>P&lt;</i>0.01. B. Activation of SHH <span class="highlight" style="background-color:">signaling</span> with SAG enhances HT29Fluc cell growth on irradiated dying HT29 cells in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. **represents <i>P&lt;</i>0.01. C. Activation of SHH <span class="highlight" style="background-color:">signaling</span> with SHH <span class="highlight" style="background-color:">signaling</span> agonists, N-terminal fragment of Shh, enhances Panc1Fluc cell growth on irradiated dying Panc1 cells. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. *represents <i>P</i>&lt;0.05. D. Activation of SHH <span class="highlight" style="background-color:">signaling</span> with N-terminal fragment of Shh enhances HT29Fluc cell growth on irradiated dying HT29 cells. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. *represents <i>P</i>&lt;0.05.</div></p></div><div class="aux"><div class="resc">Jingjing Ma, et al. PLoS One. 2013;8(6):e65032.</div><p class="links"><a class="dblinks" href="/pubmed/23762282" ref="ordinalpos=419&amp;ncbi_uid=4989292&amp;link_uid=23762282">Citation</a><a class="dblinks" href="/pmc/articles/PMC3677896/" ref="ordinalpos=419&amp;ncbi_uid=4989292&amp;link_uid=PMC3677896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>420.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4158461/figure/fig7/" ref="ordinalpos=420&amp;ncbi_uid=6606918&amp;link_uid=PMC4158461" href="/pmc/articles/PMC4158461/"><img src="/pmc/articles/instance/4158461/bin/MI2014-246407.007.gif" src-large="/pmc/articles/instance/4158461/bin/MI2014-246407.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4158461/" ref="ordinalpos=420&amp;ncbi_uid=6606918&amp;link_uid=PMC4158461" image-link="/pmc/articles/PMC4158461/figure/fig7/" class="imagepopup">Figure 7.  From: The Protective Effects of the Supercritical-Carbon Dioxide Fluid Extract of Chrysanthemum indicum against Lipopolysaccharide-Induced Acute Lung Injury in Mice via Modulating Toll-Like Receptor 4 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The underlying mechanism of CFE on the TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The red dotted arrows indicate the inhibitory sites and effects of CEF.</div></p></div><div class="aux"><div class="resc">Xiao-Li Wu, et al. Mediators Inflamm. 2014;2014:246407.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=420&amp;ncbi_uid=6606918&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4158461/" ref="ordinalpos=420&amp;ncbi_uid=6606918&amp;link_uid=PMC4158461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>421.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4000238/figure/F2/" ref="ordinalpos=421&amp;ncbi_uid=6061067&amp;link_uid=PMC4000238" href="/pmc/articles/PMC4000238/"><img src="/pmc/articles/instance/4000238/bin/nihms568219f2.gif" src-large="/pmc/articles/instance/4000238/bin/nihms568219f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4000238/" ref="ordinalpos=421&amp;ncbi_uid=6061067&amp;link_uid=PMC4000238" image-link="/pmc/articles/PMC4000238/figure/F2/" class="imagepopup">Figure 2.  From: Wnt and the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in bone development and disease. </a></p><div class="supp"><p class="details"><div>The role of the canonical Wnt, Ihh, Bmp and PTH/PTHrP <span class="highlight" style="background-color:">signaling</span> pathways in regulating the differentiation of mesenchymal precursors. The Wnt canonical <span class="highlight" style="background-color:">pathway</span> and Ihh, Bmp, PTH/PTHrP pathways control the commitment of mesenchymal precursors and also the differentiation of osteoblasts/osteocytes. Though these processes, they regulate the osteoblastogenesis and bone remodeling.</div></p></div><div class="aux"><div class="resc">Yiping Wang, et al. Front Biosci (Landmark Ed). 2014 January 1;19:379-407.</div><p class="links"><a class="dblinks" href="/pubmed/24389191" ref="ordinalpos=421&amp;ncbi_uid=6061067&amp;link_uid=24389191">Citation</a><a class="dblinks" href="/pmc/articles/PMC4000238/" ref="ordinalpos=421&amp;ncbi_uid=6061067&amp;link_uid=PMC4000238">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>422.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3525723/figure/F2/" ref="ordinalpos=422&amp;ncbi_uid=5667127&amp;link_uid=PMC3525723" href="/pmc/articles/PMC3525723/"><img src="/pmc/articles/instance/3525723/bin/nihms408902f2.gif" src-large="/pmc/articles/instance/3525723/bin/nihms408902f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3525723/" ref="ordinalpos=422&amp;ncbi_uid=5667127&amp;link_uid=PMC3525723" image-link="/pmc/articles/PMC3525723/figure/F2/" class="imagepopup">Figure 2.  From: Metformin Alters The Insulin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> In Ischemic Cardiac Tissue In A Swine Model Of Metabolic Syndrome. </a></p><div class="supp"><p class="details"><div>Protein expression in ischemic myocardium. The ratio of phosphorylated to total protein expression calculated for proteins in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. *Bonferroni p&lt;0.001</div></p></div><div class="aux"><div class="resc">Nassrene Y. Elmadhun, et al. J Thorac Cardiovasc Surg. ;145(1):258-266.</div><p class="links"><a class="dblinks" href="/pubmed/23083540" ref="ordinalpos=422&amp;ncbi_uid=5667127&amp;link_uid=23083540">Citation</a><a class="dblinks" href="/pmc/articles/PMC3525723/" ref="ordinalpos=422&amp;ncbi_uid=5667127&amp;link_uid=PMC3525723">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>423.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136913/figure/F2/" ref="ordinalpos=423&amp;ncbi_uid=3090618&amp;link_uid=PMC3136913" href="/pmc/articles/PMC3136913/"><img src="/pmc/articles/instance/3136913/bin/sgtp0202_0104_fig002.gif" src-large="/pmc/articles/instance/3136913/bin/sgtp0202_0104_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136913/" ref="ordinalpos=423&amp;ncbi_uid=3090618&amp;link_uid=PMC3136913" image-link="/pmc/articles/PMC3136913/figure/F2/" class="imagepopup">Figure 2.  From: Translational connection of TGF? <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Targeting of eEF1A1 is a novel mechanism in TGFβ <span class="highlight" style="background-color:">signaling</span>. Canonical TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves activation of Smad2 and Smad3 proteins (Smad <span class="highlight" style="background-color:">pathway</span>). There are numbers of TGFβ <span class="highlight" style="background-color:">signaling</span> pathways that do not involve direct activation of Smads (non-Smad <span class="highlight" style="background-color:">pathway</span>). These non-Smad pathways include MAPK, Rho-like GTPase and PI3K/Akt, as examples. Activation of Smad and non-Smad <span class="highlight" style="background-color:">signaling</span> cascade ultimately leads to the transcription activation of target genes. Recently, our study uncovered that eEF1A1 is phosphorylated by type I TGFβ receptor kinase, which leads to the direct regulation of protein synthesis, without involvement of transcriptional regulation.</div></p></div><div class="aux"><div class="resc">Kah Wai Lin, et al. Small GTPases. 2011 Mar-Apr;2(2):104-108.</div><p class="links"><a class="dblinks" href="/pubmed/21776411" ref="ordinalpos=423&amp;ncbi_uid=3090618&amp;link_uid=21776411">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136913/" ref="ordinalpos=423&amp;ncbi_uid=3090618&amp;link_uid=PMC3136913">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>424.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2930641/figure/F2/" ref="ordinalpos=424&amp;ncbi_uid=2453779&amp;link_uid=PMC2930641" href="/pmc/articles/PMC2930641/"><img src="/pmc/articles/instance/2930641/bin/1748-7188-5-30-2.gif" src-large="/pmc/articles/instance/2930641/bin/1748-7188-5-30-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2930641/" ref="ordinalpos=424&amp;ncbi_uid=2453779&amp;link_uid=PMC2930641" image-link="/pmc/articles/PMC2930641/figure/F2/" class="imagepopup">Figure 2.  From: Survival associated <span class="highlight" style="background-color:">pathway</span> identification with group Lp penalized global AUC maximization. </a></p><div class="supp"><p class="details"><div><b>MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></b>. MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the associated genes. Genes in red are highly expressed in patients who died earlier and genes in yellow are highly expressed in patients who lived longer.</div></p></div><div class="aux"><div class="resc">Zhenqiu Liu, et al. Algorithms Mol Biol. 2010;5:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/20712896" ref="ordinalpos=424&amp;ncbi_uid=2453779&amp;link_uid=20712896">Citation</a><a class="dblinks" href="/pmc/articles/PMC2930641/" ref="ordinalpos=424&amp;ncbi_uid=2453779&amp;link_uid=PMC2930641">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>425.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g006/" ref="ordinalpos=425&amp;ncbi_uid=5583242&amp;link_uid=PMC3855681" href="/pmc/articles/PMC3855681/"><img src="/pmc/articles/instance/3855681/bin/pone.0080918.g006.gif" src-large="/pmc/articles/instance/3855681/bin/pone.0080918.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855681/" ref="ordinalpos=425&amp;ncbi_uid=5583242&amp;link_uid=PMC3855681" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g006/" class="imagepopup">Figure 6. Displaying the concept behind the construction of integrated <span class="highlight" style="background-color:">pathway</span> system that includes SAg expression regulatory <span class="highlight" style="background-color:">pathway</span> of S. aureus and TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of an infected host (H. sapiens)..  From: A Model of an Integrated Immune System <span class="highlight" style="background-color:">Pathway</span> in Homo sapiens and Its Interaction with Superantigen Producing Expression Regulatory <span class="highlight" style="background-color:">Pathway</span> in Staphylococcus aureus: Comparing Behavior of Pathogen Perturbed and Unperturbed <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="aux"><div class="resc">Namrata Tomar, et al. PLoS One. 2013;8(12):e80918.</div><p class="links"><a class="dblinks" href="/pubmed/24324645" ref="ordinalpos=425&amp;ncbi_uid=5583242&amp;link_uid=24324645">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855681/" ref="ordinalpos=425&amp;ncbi_uid=5583242&amp;link_uid=PMC3855681">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>426.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3528779/figure/pone-0052172-g006/" ref="ordinalpos=426&amp;ncbi_uid=4494395&amp;link_uid=PMC3528779" href="/pmc/articles/PMC3528779/"><img src="/pmc/articles/instance/3528779/bin/pone.0052172.g006.gif" src-large="/pmc/articles/instance/3528779/bin/pone.0052172.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3528779/" ref="ordinalpos=426&amp;ncbi_uid=4494395&amp;link_uid=PMC3528779" image-link="/pmc/articles/PMC3528779/figure/pone-0052172-g006/" class="imagepopup">Figure 6. Genes induced by INZ in p53 <span class="highlight" style="background-color:">pathway</span>..  From: Global Effect of Inauhzin on Human p53-Responsive Transcriptome. </a></p><div class="supp"><p class="details"><div>The p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> identified by <span class="highlight" style="background-color:">Pathway</span>-Express analysis. <span class="highlight" style="background-color:">Pathway</span>-Express analysis was performed on the INZ-up-regulated genes. Among the most significant KEGG pathways predicted to be relevant to NZ treatment was the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In red square are the genes that were up-regulated by INZ.</div></p></div><div class="aux"><div class="resc">Jun-Ming Liao, et al. PLoS One. 2012;7(12):e52172.</div><p class="links"><a class="dblinks" href="/pubmed/23284922" ref="ordinalpos=426&amp;ncbi_uid=4494395&amp;link_uid=23284922">Citation</a><a class="dblinks" href="/pmc/articles/PMC3528779/" ref="ordinalpos=426&amp;ncbi_uid=4494395&amp;link_uid=PMC3528779">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>427.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3375508/figure/F7/" ref="ordinalpos=427&amp;ncbi_uid=4986058&amp;link_uid=PMC3375508" href="/pmc/articles/PMC3375508/"><img src="/pmc/articles/instance/3375508/bin/zbc0261212630007.gif" src-large="/pmc/articles/instance/3375508/bin/zbc0261212630007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3375508/" ref="ordinalpos=427&amp;ncbi_uid=4986058&amp;link_uid=PMC3375508" image-link="/pmc/articles/PMC3375508/figure/F7/" class="imagepopup">FIGURE 7. From: Taurine Inhibits K+-Cl− Cotransporter KCC2 to Regulate Embryonic Cl− Homeostasis via With-no-lysine (WNK) Protein Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Hypothetical model of Cl<sup>−</sup> homeostasis regulated by taurine and WNK-SPAK/OSR1 <span class="highlight" style="background-color:">signaling</span> during perinatal periods.</b> To control the excitatory/inhibitory balance mediated by GABA, [Cl<sup>−</sup>]<i><sub>i</sub></i> is regulated by activation of the WNK-SPAK/OSR1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via KCC2 inhibition and possibly NKCC1 activation (54, 58, 59). In immature neurons, taurine is taken up into cells through TauT and activates WNK-SPAK/OSR1 <span class="highlight" style="background-color:">signaling</span> (<i>left</i>). <i>Red arrows</i> and <i>T-shaped bars</i> indicate activation and inactivation, respectively. Later (possibly a while after birth), this activation <span class="highlight" style="background-color:">pathway</span> induced by taurine diminishes, resulting in release of KCC/NKCC activity (<i>right</i>), whereas SPAK/OSR1 <span class="highlight" style="background-color:">signaling</span> recovers somewhat upon adulthood. Interestingly, in contrast to kinase <span class="highlight" style="background-color:">signaling</span> leading to KCC2 inhibition, other kinases are also known to facilitate KCC2 activity (see “Discussion”).</div></p></div><div class="aux"><div class="resc">Koichi Inoue, et al. J Biol Chem. 2012 June 15;287(25):20839-20850.</div><p class="links"><a class="dblinks" href="/pubmed/22544747" ref="ordinalpos=427&amp;ncbi_uid=4986058&amp;link_uid=22544747">Citation</a><a class="dblinks" href="/pmc/articles/PMC3375508/" ref="ordinalpos=427&amp;ncbi_uid=4986058&amp;link_uid=PMC3375508">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>428.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2826191/figure/F2/" ref="ordinalpos=428&amp;ncbi_uid=2276284&amp;link_uid=PMC2826191" href="/pmc/articles/PMC2826191/"><img src="/pmc/articles/instance/2826191/bin/nihms78785f2.gif" src-large="/pmc/articles/instance/2826191/bin/nihms78785f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2826191/" ref="ordinalpos=428&amp;ncbi_uid=2276284&amp;link_uid=PMC2826191" image-link="/pmc/articles/PMC2826191/figure/F2/" class="imagepopup">Figure 2.  From: Spatial Segregation of Ras <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Spatial segregation of <i>S. pombe</i> Ras1 pathways. The Ras1-Byr2 (MAP kinase) <span class="highlight" style="background-color:">pathway</span> signals from the PM to mediate cross-membrane <span class="highlight" style="background-color:">signaling</span> during mating. The Ras1-Scd1 (Cdc42) <span class="highlight" style="background-color:">pathway</span> regulates uncharacterized intracellular signals to control protein trafficking. Each Ras1 <span class="highlight" style="background-color:">pathway</span> is specifically activated by a GEF. The Ras1-Byr2 <span class="highlight" style="background-color:">pathway</span> is regulated by Ste6, while Efc25 selectively regulates the Ras1-Scd1 <span class="highlight" style="background-color:">pathway</span>. The farnesyl modification is indicated with a branched, unsaturated polyisoprene chain while palmitoylation is indicated with an acyl chain.</div></p></div><div class="aux"><div class="resc">Eric C. Chang, et al. Cell Cycle. ;5(17):1936-1939.</div><p class="links"><a class="dblinks" href="/pubmed/16931912" ref="ordinalpos=428&amp;ncbi_uid=2276284&amp;link_uid=16931912">Citation</a><a class="dblinks" href="/pmc/articles/PMC2826191/" ref="ordinalpos=428&amp;ncbi_uid=2276284&amp;link_uid=PMC2826191">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>429.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3133405/figure/F9/" ref="ordinalpos=429&amp;ncbi_uid=3515793&amp;link_uid=PMC3133405" href="/pmc/articles/PMC3133405/"><img src="/pmc/articles/instance/3133405/bin/zmb9991091330009.gif" src-large="/pmc/articles/instance/3133405/bin/zmb9991091330009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3133405/" ref="ordinalpos=429&amp;ncbi_uid=3515793&amp;link_uid=PMC3133405" image-link="/pmc/articles/PMC3133405/figure/F9/" class="imagepopup">Fig. 9. From: Spn1 Regulates the GNBP3-Dependent Toll <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div>Model of Spn1 regulation of the Toll-mediated innate immune response. Three separate side branches of the Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are activated either by microbial cell wall PRRs (GNBP3, for fungi and yeasts; GNBP1, PGRP-SA, and PGRP-SD for Gram-positive bacteria) or danger-<span class="highlight" style="background-color:">signaling</span> (virulence factors). These three side branches of the <span class="highlight" style="background-color:">pathway</span> converge on SPE, which activates Spz (the ligand of Toll). The intracellular Toll <span class="highlight" style="background-color:">pathway</span> signals trough Dif to activate the expression of <i>Drs</i> and other antimicrobial peptides in the fat body. The GNBP3 and danger-<span class="highlight" style="background-color:">signaling</span> branches of the Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are regulated independently by Spn1 and nec. Both of these serpins represent nonredundant genetic functions, and the lack activates <i>Drs</i> expression.</div></p></div><div class="aux"><div class="resc">Ane Fullaondo, et al. Mol Cell Biol. 2011 July;31(14):2960-2972.</div><p class="links"><a class="dblinks" href="/pubmed/21576362" ref="ordinalpos=429&amp;ncbi_uid=3515793&amp;link_uid=21576362">Citation</a><a class="dblinks" href="/pmc/articles/PMC3133405/" ref="ordinalpos=429&amp;ncbi_uid=3515793&amp;link_uid=PMC3133405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>430.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3043057/figure/pone-0016787-g006/" ref="ordinalpos=430&amp;ncbi_uid=2806308&amp;link_uid=PMC3043057" href="/pmc/articles/PMC3043057/"><img src="/pmc/articles/instance/3043057/bin/pone.0016787.g006.gif" src-large="/pmc/articles/instance/3043057/bin/pone.0016787.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3043057/" ref="ordinalpos=430&amp;ncbi_uid=2806308&amp;link_uid=PMC3043057" image-link="/pmc/articles/PMC3043057/figure/pone-0016787-g006/" class="imagepopup">Figure 6. Model of action of hSprouty2 on cyP-triggered Ras/Raf/MEK/ERK/Elk1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Sprouty2 and Spred1-2 Proteins Inhibit the Activation of the ERK <span class="highlight" style="background-color:">Pathway</span> Elicited by Cyclopentenone Prostanoids. </a></p><div class="supp"><p class="details"><div>PGA<sub>1</sub> binds to cysteine 118 of Ras and induces Ras·GTP loading and Raf/MEK/ERK/Elk1 <span class="highlight" style="background-color:">pathway</span> activation. Overexpression of Sprouty2 (Spry2) wt or Spry2 P59A P304A double mutant (unable to bind Grb2) block cyP and FGF-elicited ERK phosphorylation but not Ras activation, probably due to an effect on Raf <span class="highlight" style="background-color:">signaling</span>. However, Spry2 Y55F mutant does not inhibit cyP and FGF-induced ERK <span class="highlight" style="background-color:">pathway</span> activation, although treatment with PGA<sub>1</sub> was unable to promote tyrosine-phosphorylation of Spry2; in sharp contrast, RTK activation (mainly EGFR) provokes tyrosine-phosphorylation of Spry2 (at Y55 residue) increasing its binding affinity to the SH2 domain of c-Cbl, which leads to Spry2 ubiquitination.</div></p></div><div class="aux"><div class="resc">Carlota A. García-Domínguez, et al. PLoS One. 2011;6(2):e16787.</div><p class="links"><a class="dblinks" href="/pubmed/21364986" ref="ordinalpos=430&amp;ncbi_uid=2806308&amp;link_uid=21364986">Citation</a><a class="dblinks" href="/pmc/articles/PMC3043057/" ref="ordinalpos=430&amp;ncbi_uid=2806308&amp;link_uid=PMC3043057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>431.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3369892/figure/f9/" ref="ordinalpos=431&amp;ncbi_uid=4054683&amp;link_uid=PMC3369892" href="/pmc/articles/PMC3369892/"><img src="/pmc/articles/instance/3369892/bin/mv-v18-1339-f9.gif" src-large="/pmc/articles/instance/3369892/bin/mv-v18-1339-f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3369892/" ref="ordinalpos=431&amp;ncbi_uid=4054683&amp;link_uid=PMC3369892" image-link="/pmc/articles/PMC3369892/figure/f9/" class="imagepopup">Figure 9.  From: Evidence for a functional adrenomedullin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the mouse retina. </a></p><div class="supp"><p class="details"><div>Summary of the cellular localization of the ADM <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the retina. ADM was present near the outer segments (OS), in somata in the INL and the GCL, and in Müller cells. CRLR and RAMP2 were present near the OS, and in somata in the ONL, INL, and GCL. ADM, CRLR, and RAMP2 were colocalized near the OS, and in somata in the INL and the GCL.</div></p></div><div class="aux"><div class="resc">Jan Blom, et al. Mol Vis. 2012;18:1339-1353.</div><p class="links"><a class="dblinks" href="/pubmed/22690112" ref="ordinalpos=431&amp;ncbi_uid=4054683&amp;link_uid=22690112">Citation</a><a class="dblinks" href="/pmc/articles/PMC3369892/" ref="ordinalpos=431&amp;ncbi_uid=4054683&amp;link_uid=PMC3369892">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>432.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4068075/figure/F3/" ref="ordinalpos=432&amp;ncbi_uid=6307592&amp;link_uid=PMC4068075" href="/pmc/articles/PMC4068075/"><img src="/pmc/articles/instance/4068075/bin/1939-4551-7-13-3.gif" src-large="/pmc/articles/instance/4068075/bin/1939-4551-7-13-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4068075/" ref="ordinalpos=432&amp;ncbi_uid=6307592&amp;link_uid=PMC4068075" image-link="/pmc/articles/PMC4068075/figure/F3/" class="imagepopup">Figure 3.  From: Systems biology approaches to understanding Epithelial Mesenchymal Transition (EMT) in mucosal remodeling and <span class="highlight" style="background-color:">signaling</span> in asthma. </a></p><div class="supp"><p class="details"><div><b>Consequences of EMT on innate <span class="highlight" style="background-color:">signaling</span>.</b> The innate NFkB <span class="highlight" style="background-color:">pathway</span> is composed of the rapidly responding (canonical)- and the slower responding (noncanonical) pathways. The canonical <span class="highlight" style="background-color:">pathway</span> controls the inflammatory response, and is coupled to the noncanonical <span class="highlight" style="background-color:">pathway</span> through a feed-forward <span class="highlight" style="background-color:">pathway</span> by the synthesis of TRAF1 and NFκB2, two rate limiting factors in noncanonical <span class="highlight" style="background-color:">pathway</span> activation. Through epigenetic reprogramming and enhanced transcriptional elongation through the cyclin dependent kinase (CDK), the rate-limiting TRAF1/NFκB2 translation is reduced, resulting in enhanced <span class="highlight" style="background-color:">pathway</span> activation in EMT.</div></p></div><div class="aux"><div class="resc">Talha Ijaz, et al. World Allergy Organ J. 2014;7(1):13-13.</div><p class="links"><a class="dblinks" href="/pubmed/24982697" ref="ordinalpos=432&amp;ncbi_uid=6307592&amp;link_uid=24982697">Citation</a><a class="dblinks" href="/pmc/articles/PMC4068075/" ref="ordinalpos=432&amp;ncbi_uid=6307592&amp;link_uid=PMC4068075">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>433.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4326866/figure/fig06/" ref="ordinalpos=433&amp;ncbi_uid=7191221&amp;link_uid=PMC4326866" href="/pmc/articles/PMC4326866/"><img src="/pmc/articles/instance/4326866/bin/acel0013-0008-f6.gif" src-large="/pmc/articles/instance/4326866/bin/acel0013-0008-f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4326866/" ref="ordinalpos=433&amp;ncbi_uid=7191221&amp;link_uid=PMC4326866" image-link="/pmc/articles/PMC4326866/figure/fig06/" class="imagepopup">Figure 6.  From: A dual role of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during aging in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>Model for role of the Wnt <span class="highlight" style="background-color:">signaling</span> pathways in age-regulation. The activity of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> increases as worms age, leading to increased expression of <i>elt-5</i> GATA transcription factor. This GATA transcription factor represses the expression of <i>elt-3</i> in old worms. Changes in the <i>elt-3</i> GATA transcription factor activate a cascade of downstream effects, which causes aging. The <i>elt-3/elt-5</i> GATA transcriptional circuit could be one of a number of genetic programs that are under control of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during aging. In addition to developmental programs for aging, Wnt <span class="highlight" style="background-color:">signaling</span> might regulate genes responsible for cellular stress response, but not for oxidative damage.</div></p></div><div class="aux"><div class="resc">Marco Lezzerini, et al. Aging Cell. 2014 February;13(1):8-18.</div><p class="links"><a class="dblinks" href="/pubmed/23879250" ref="ordinalpos=433&amp;ncbi_uid=7191221&amp;link_uid=23879250">Citation</a><a class="dblinks" href="/pmc/articles/PMC4326866/" ref="ordinalpos=433&amp;ncbi_uid=7191221&amp;link_uid=PMC4326866">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>434.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3165056/figure/F4/" ref="ordinalpos=434&amp;ncbi_uid=3196010&amp;link_uid=PMC3165056" href="/pmc/articles/PMC3165056/"><img src="/pmc/articles/instance/3165056/bin/nihms132499f4.gif" src-large="/pmc/articles/instance/3165056/bin/nihms132499f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3165056/" ref="ordinalpos=434&amp;ncbi_uid=3196010&amp;link_uid=PMC3165056" image-link="/pmc/articles/PMC3165056/figure/F4/" class="imagepopup">Fig. 4. Viral countermeasures in the RNA sensing and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Functions of the cytoplasmic RNA sensors RIG-I and MDA-5: Key regulators of innate immunity. </a></p><div class="supp"><p class="details"><div>The steps leading from detection of RNA by RIG-I and MDA-5 through <span class="highlight" style="background-color:">signaling</span> and transcription of IFN genes are depicted. Viral gene products that interfere at different steps of this <span class="highlight" style="background-color:">pathway</span> are listed. An arrow indicates stimulation, and a bar indicates inhibition of the pathways</div></p></div><div class="aux"><div class="resc">Paola M Barral, et al. Pharmacol Ther. ;124(2):219-234.</div><p class="links"><a class="dblinks" href="/pubmed/19615405" ref="ordinalpos=434&amp;ncbi_uid=3196010&amp;link_uid=19615405">Citation</a><a class="dblinks" href="/pmc/articles/PMC3165056/" ref="ordinalpos=434&amp;ncbi_uid=3196010&amp;link_uid=PMC3165056">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>435.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3218473/figure/F1/" ref="ordinalpos=435&amp;ncbi_uid=3580017&amp;link_uid=PMC3218473" href="/pmc/articles/PMC3218473/"><img src="/pmc/articles/instance/3218473/bin/psb0606_0783_fig001.gif" src-large="/pmc/articles/instance/3218473/bin/psb0606_0783_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3218473/" ref="ordinalpos=435&amp;ncbi_uid=3580017&amp;link_uid=PMC3218473" image-link="/pmc/articles/PMC3218473/figure/F1/" class="imagepopup">Figure 1.  From: Glutathione as a <span class="highlight" style="background-color:">signaling</span> molecule. </a></p><div class="supp"><p class="details"><div>Position and involvement of GSH in the <span class="highlight" style="background-color:">signaling</span> web in plants. GSH has been known to participate in biotic stress tolerance by NPR1-dependent SA-mediated <span class="highlight" style="background-color:">pathway</span>. Only those signal transduction components relevant to this review has been shown. Continuous lines: established <span class="highlight" style="background-color:">signaling</span> pathways; broken lines: proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Srijani Ghanta, et al. Plant Signal Behav. 2011 June;6(6):783-788.</div><p class="links"><a class="dblinks" href="/pubmed/21969955" ref="ordinalpos=435&amp;ncbi_uid=3580017&amp;link_uid=21969955">Citation</a><a class="dblinks" href="/pmc/articles/PMC3218473/" ref="ordinalpos=435&amp;ncbi_uid=3580017&amp;link_uid=PMC3218473">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>436.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1502028/figure/F4/" ref="ordinalpos=436&amp;ncbi_uid=703400&amp;link_uid=PMC1502028" href="/pmc/articles/PMC1502028/"><img src="/pmc/articles/instance/1502028/bin/neo0710_0921_fig004.gif" src-large="/pmc/articles/instance/1502028/bin/neo0710_0921_fig004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1502028/" ref="ordinalpos=436&amp;ncbi_uid=703400&amp;link_uid=PMC1502028" image-link="/pmc/articles/PMC1502028/figure/F4/" class="imagepopup">Figure 4.  From: Disruption of Parallel and Converging <span class="highlight" style="background-color:">Signaling</span> Pathways Contributes to the Synergistic Antitumor Effects of Simultaneous mTOR and EGFR Inhibition in GBM Cells. </a></p><div class="supp"><p class="details"><div>Schematic of EGFR and mTOR <span class="highlight" style="background-color:">signaling</span> network, and <span class="highlight" style="background-color:">signaling</span> mediator effects associated with rapamycin and/or EKI-785 treatments. A notation of an “E” or “R,” in association with a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, indicates where the <span class="highlight" style="background-color:">pathway</span> is blocked by EKI-785 or rapamycin, respectively.</div></p></div><div class="aux"><div class="resc">Ravi D Rao, et al. Neoplasia. 2005 October;7(10):921-929.</div><p class="links"><a class="dblinks" href="/pubmed/16242075" ref="ordinalpos=436&amp;ncbi_uid=703400&amp;link_uid=16242075">Citation</a><a class="dblinks" href="/pmc/articles/PMC1502028/" ref="ordinalpos=436&amp;ncbi_uid=703400&amp;link_uid=PMC1502028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>437.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2752654/figure/F1/" ref="ordinalpos=437&amp;ncbi_uid=2111038&amp;link_uid=PMC2752654" href="/pmc/articles/PMC2752654/"><img src="/pmc/articles/instance/2752654/bin/nihms-117176-f0001.gif" src-large="/pmc/articles/instance/2752654/bin/nihms-117176-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2752654/" ref="ordinalpos=437&amp;ncbi_uid=2111038&amp;link_uid=PMC2752654" image-link="/pmc/articles/PMC2752654/figure/F1/" class="imagepopup">Fig. 1. Model of the σB <span class="highlight" style="background-color:">signaling</span> network.  From: Bypass suppression analysis maps the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> within a multidomain protein: the RsbP energy stress phosphatase 2C from Bacillus subtilis. </a></p><div class="supp"><p class="details"><div>Two independent <span class="highlight" style="background-color:">signaling</span> pathways converge on RsbV and RsbW, which comprise the partner switch that directly regulates σ<sup>B</sup> activity (see text). Each <span class="highlight" style="background-color:">pathway</span> conveys a particular class of stress, energy or environmental, and terminates with a PP2C phosphatase specific for RsbV-P. Dephosphorylated RsbV binds the RsbW anti-σ factor, forcing the release of σ<sup>B</sup>. In the energy <span class="highlight" style="background-color:">pathway</span> the RsbP phosphatase has an N-terminal PAS domain. In addition to RsbP activity, RsbQ α/β hydrolase activity is also required for energy <span class="highlight" style="background-color:">signaling</span>. Neither the parameter sensed by the PAS domain nor the substrate of the RsbQ hydrolase is known. In the environmental <span class="highlight" style="background-color:">pathway</span> RsbU has an N-terminal RsbU_N domain, which is required for the RsbT regulator to bind and activate the phosphatase by direct protein-protein interaction.</div></p></div><div class="aux"><div class="resc">Margaret S. Brody, et al. Mol Microbiol. ;72(5):1221-1234.</div><p class="links"><a class="dblinks" href="/pubmed/19432806" ref="ordinalpos=437&amp;ncbi_uid=2111038&amp;link_uid=19432806">Citation</a><a class="dblinks" href="/pmc/articles/PMC2752654/" ref="ordinalpos=437&amp;ncbi_uid=2111038&amp;link_uid=PMC2752654">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>438.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3189939/figure/pone-0025807-g003/" ref="ordinalpos=438&amp;ncbi_uid=3293758&amp;link_uid=PMC3189939" href="/pmc/articles/PMC3189939/"><img src="/pmc/articles/instance/3189939/bin/pone.0025807.g003.gif" src-large="/pmc/articles/instance/3189939/bin/pone.0025807.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3189939/" ref="ordinalpos=438&amp;ncbi_uid=3293758&amp;link_uid=PMC3189939" image-link="/pmc/articles/PMC3189939/figure/pone-0025807-g003/" class="imagepopup">Figure 3. Mapping gene expression changes on the Wnt <span class="highlight" style="background-color:">pathway</span> revealed strong upregulation of PCP <span class="highlight" style="background-color:">signaling</span> and downregulation of canonical <span class="highlight" style="background-color:">signaling</span>..  From: CAFET Algorithm Reveals Wnt/PCP Signature in Lung Squamous Cell Carcinoma. </a></p><div class="supp"><p class="details"><div>The three branches of Wnt <span class="highlight" style="background-color:">signaling</span> are shown –calcium (pink background), beta-catenin (green background), and PCP (blue background). Up/down arrows indicate overexpression and downregulation, respectively, in Group 1 samples. Genes whose expression change is consistent with canonical <span class="highlight" style="background-color:">pathway</span> inhibition and PCP <span class="highlight" style="background-color:">pathway</span> activation are colored red. Those genes promoting canonical <span class="highlight" style="background-color:">signaling</span> and inhibiting the PCP branch are colored green. Genes with no significant change in expression, or whose expression change has no selectivity between the canonical and non-canonical branches are colored white. The down-regulation of <i>CAMK2D</i> and <i>PRKCA</i> (dark blue) acts to inhibit Wnt Ca2+ <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Yue Hu, et al. PLoS One. 2011;6(10):e25807.</div><p class="links"><a class="dblinks" href="/pubmed/22016777" ref="ordinalpos=438&amp;ncbi_uid=3293758&amp;link_uid=22016777">Citation</a><a class="dblinks" href="/pmc/articles/PMC3189939/" ref="ordinalpos=438&amp;ncbi_uid=3293758&amp;link_uid=PMC3189939">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>439.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3465263/figure/pone-0046719-g010/" ref="ordinalpos=439&amp;ncbi_uid=4296681&amp;link_uid=PMC3465263" href="/pmc/articles/PMC3465263/"><img src="/pmc/articles/instance/3465263/bin/pone.0046719.g010.gif" src-large="/pmc/articles/instance/3465263/bin/pone.0046719.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3465263/" ref="ordinalpos=439&amp;ncbi_uid=4296681&amp;link_uid=PMC3465263" image-link="/pmc/articles/PMC3465263/figure/pone-0046719-g010/" class="imagepopup">Figure 10. Possible points of interaction between Pcmt1 and the upstream insulin-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Wortmannin Reduces Insulin <span class="highlight" style="background-color:">Signaling</span> and Death in Seizure-Prone Pcmt1−/− Mice. </a></p><div class="supp"><p class="details"><div>Wortmannin inhibits PI3K class kinases and has been shown to inhibit insulin <span class="highlight" style="background-color:">signaling</span>. The elevated insulin <span class="highlight" style="background-color:">signaling</span> in <i>Pcmt1<sup>−/−</sup></i> animals could arise from aberrant Akt activation (hypothesis 1 or 2), and/or aberrant interaction between Akt1 and mTOR (hypothesis 1 or 2) or activation of an alternate growth <span class="highlight" style="background-color:">pathway</span> (hypothesis 3).</div></p></div><div class="aux"><div class="resc">Kennen B. MacKay, et al. PLoS One. 2012;7(10):e46719.</div><p class="links"><a class="dblinks" href="/pubmed/23071621" ref="ordinalpos=439&amp;ncbi_uid=4296681&amp;link_uid=23071621">Citation</a><a class="dblinks" href="/pmc/articles/PMC3465263/" ref="ordinalpos=439&amp;ncbi_uid=4296681&amp;link_uid=PMC3465263">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>440.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285189/figure/pone-0031951-g007/" ref="ordinalpos=440&amp;ncbi_uid=3835377&amp;link_uid=PMC3285189" href="/pmc/articles/PMC3285189/"><img src="/pmc/articles/instance/3285189/bin/pone.0031951.g007.gif" src-large="/pmc/articles/instance/3285189/bin/pone.0031951.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285189/" ref="ordinalpos=440&amp;ncbi_uid=3835377&amp;link_uid=PMC3285189" image-link="/pmc/articles/PMC3285189/figure/pone-0031951-g007/" class="imagepopup">Figure 7. Stimulation of the H2 receptor inhibited the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Histamine Derived from Probiotic Lactobacillus reuteri Suppresses TNF via Modulation of PKA and ERK <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Immunoblot studies were performed to determine the effect of histamine on activation of the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ERK was activated by phosphorylation on Thr202/Tyr204 (P-ERK). Treatment of TLR-stimulated THP-1 cells with <i>L. reuteri</i> 6475, histamine, or U0126 suppressed activation of ERK1/2, an effect that was significantly blocked in the presence of ranitidine. The β-actin antibody served as the protein loading control.</div></p></div><div class="aux"><div class="resc">Carissa M. Thomas, et al. PLoS One. 2012;7(2):e31951.</div><p class="links"><a class="dblinks" href="/pubmed/22384111" ref="ordinalpos=440&amp;ncbi_uid=3835377&amp;link_uid=22384111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285189/" ref="ordinalpos=440&amp;ncbi_uid=3835377&amp;link_uid=PMC3285189">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>441.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4254673/figure/f7-or-33-01-0019/" ref="ordinalpos=441&amp;ncbi_uid=6913523&amp;link_uid=PMC4254673" href="/pmc/articles/PMC4254673/"><img src="/pmc/articles/instance/4254673/bin/OR-33-01-0019-g06.gif" src-large="/pmc/articles/instance/4254673/bin/OR-33-01-0019-g06.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4254673/" ref="ordinalpos=441&amp;ncbi_uid=6913523&amp;link_uid=PMC4254673" image-link="/pmc/articles/PMC4254673/figure/f7-or-33-01-0019/" class="imagepopup">Figure 7.  From: Activation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in breast cancer MCF-7 cells by a peptide derived from Porphyra yezoensis. </a></p><div class="supp"><p class="details"><div>Effect of PPY following mTOR siRNA-mediated knockdown. Western blot analysis of mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins after mTOR siRNA transfection.</div></p></div><div class="aux"><div class="resc">SU-JIN PARK, et al. Oncol Rep. 2015 January;33(1):19-24.</div><p class="links"><a class="dblinks" href="/pubmed/25333576" ref="ordinalpos=441&amp;ncbi_uid=6913523&amp;link_uid=25333576">Citation</a><a class="dblinks" href="/pmc/articles/PMC4254673/" ref="ordinalpos=441&amp;ncbi_uid=6913523&amp;link_uid=PMC4254673">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>442.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2128780/figure/F9/" ref="ordinalpos=442&amp;ncbi_uid=1159584&amp;link_uid=PMC2128780" href="/pmc/articles/PMC2128780/"><img src="/pmc/articles/instance/2128780/bin/nihms33729f9.gif" src-large="/pmc/articles/instance/2128780/bin/nihms33729f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2128780/" ref="ordinalpos=442&amp;ncbi_uid=1159584&amp;link_uid=PMC2128780" image-link="/pmc/articles/PMC2128780/figure/F9/" class="imagepopup">Fig. 9. Model of Egfr/Ras <span class="highlight" style="background-color:">pathway</span> function in early pupal vein cells.  From: EGFR/Ras <span class="highlight" style="background-color:">signaling</span> regulates DE-cadherin/Shotgun localization to control vein morphogenesis in the Drosophila wing. </a></p><div class="supp"><p class="details"><div>Rhomboid expression in presumptive vein cells activates Egfr ligands, leading to <span class="highlight" style="background-color:">signaling</span> through the Egfr/Ras <span class="highlight" style="background-color:">pathway</span>. Ras <span class="highlight" style="background-color:">signaling</span> directs vein cell identity through Dpp, and affects <i>shg</i> expression and protein localization (apical concentration) to alter vein cell shape in a Dpp-independent fashion.</div></p></div><div class="aux"><div class="resc">David D. O’Keefe, et al. Dev Biol. ;311(1):25-39.</div><p class="links"><a class="dblinks" href="/pubmed/17888420" ref="ordinalpos=442&amp;ncbi_uid=1159584&amp;link_uid=17888420">Citation</a><a class="dblinks" href="/pmc/articles/PMC2128780/" ref="ordinalpos=442&amp;ncbi_uid=1159584&amp;link_uid=PMC2128780">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>443.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC1501083/figure/F4/" ref="ordinalpos=443&amp;ncbi_uid=700881&amp;link_uid=PMC1501083" href="/pmc/articles/PMC1501083/"><img src="/pmc/articles/instance/1501083/bin/nihms10559f4a.gif" src-large="/pmc/articles/instance/1501083/bin/nihms10559f4a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC1501083/figure/F4/" ref="ordinalpos=443&amp;ncbi_uid=700881&amp;link_uid=PMC1501083" href="/pmc/articles/PMC1501083/"><img src="/pmc/articles/instance/1501083/bin/nihms10559f4b.gif" src-large="/pmc/articles/instance/1501083/bin/nihms10559f4b.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC1501083/figure/F4/" ref="ordinalpos=443&amp;ncbi_uid=700881&amp;link_uid=PMC1501083" href="/pmc/articles/PMC1501083/"><img src="/pmc/articles/instance/1501083/bin/nihms10559f4c.gif" src-large="/pmc/articles/instance/1501083/bin/nihms10559f4c.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1501083/" ref="ordinalpos=443&amp;ncbi_uid=700881&amp;link_uid=PMC1501083" image-link="/pmc/articles/PMC1501083/figure/F4/" class="imagepopup">Figure 4.  From: Parallel analysis of transcript and translation profiles: identification of metastasis-related signal pathways differentially regulated by drug and genetic modifications. </a></p><div class="supp"><p class="details"><div>Response profiles of the Akt, caspase-dependent apoptosis, and Stat <span class="highlight" style="background-color:">signaling</span> pathways to caffeine treatments and genetic variations. The expression of the proteins in the Akt <span class="highlight" style="background-color:">pathway</span> (A1 and A2), caspase-dependent apoptosis <span class="highlight" style="background-color:">pathway</span> (B1 and B2), and Stats <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (C1 and C2) were determined using Western blot. The signal intensity of the target proteins was normalized against the signal intensity of GAPDH. Significant effects of caffeine exposure and genetic variations were observed in all 3 pathways. The relative abundance of protein expression (Perturbed/Control) in each <span class="highlight" style="background-color:">pathway</span> was plotted to provide an overview in the similarity of genetic and drug modifications on each <span class="highlight" style="background-color:">pathway</span>. The profiles of all 4 groups are similar in Stats <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (C2), however, inconsistent in the Akt <span class="highlight" style="background-color:">pathway</span> (A2) and caspase-dependent apoptosis <span class="highlight" style="background-color:">pathway</span> (B2). The metastatic phenotype of each group is indicated as H (high) or L (low).</div></p></div><div class="aux"><div class="resc">Haiyan Yang, et al. J Proteome Res. ;5(7):1555-1567.</div><p class="links"><a class="dblinks" href="/pubmed/16823962" ref="ordinalpos=443&amp;ncbi_uid=700881&amp;link_uid=16823962">Citation</a><a class="dblinks" href="/pmc/articles/PMC1501083/" ref="ordinalpos=443&amp;ncbi_uid=700881&amp;link_uid=PMC1501083">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>444.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2649208/figure/F2/" ref="ordinalpos=444&amp;ncbi_uid=1699605&amp;link_uid=PMC2649208" href="/pmc/articles/PMC2649208/"><img src="/pmc/articles/instance/2657410/bin/zpq9990971880002.gif" src-large="/pmc/articles/instance/2657410/bin/zpq9990971880002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2649208/" ref="ordinalpos=444&amp;ncbi_uid=1699605&amp;link_uid=PMC2649208" image-link="/pmc/articles/PMC2649208/figure/F2/" class="imagepopup">Fig. 2. From: V600EBRAF is associated with disabled feedback inhibition of RAF-MEK <span class="highlight" style="background-color:">signaling</span> and elevated transcriptional output of the <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>A molecular interaction map displaying the functional interaction of genes in the MEK/ERK output profile with established roles in ERK <span class="highlight" style="background-color:">signaling</span>. A molecular interaction map was constructed for major constituents of the RAS/RAF/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and downstream effectors. The genes in the output profile that have established functional relationships to ERK <span class="highlight" style="background-color:">signaling</span> are indicated by gray shading, with the nature of their relationship (i.e., direct phosphorylation, transcriptional activation) indicated by using the symbols described by Pommier and colleagues (43) and shown in the legend.</div></p></div><div class="aux"><div class="resc">Christine A. Pratilas, et al. Proc Natl Acad Sci U S A. 2009 March 17;106(11):4519-4524.</div><p class="links"><a class="dblinks" href="/pubmed/19251651" ref="ordinalpos=444&amp;ncbi_uid=1699605&amp;link_uid=19251651">Citation</a><a class="dblinks" href="/pmc/articles/PMC2649208/" ref="ordinalpos=444&amp;ncbi_uid=1699605&amp;link_uid=PMC2649208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>445.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3474327/figure/F4/" ref="ordinalpos=445&amp;ncbi_uid=4782640&amp;link_uid=PMC3474327" href="/pmc/articles/PMC3474327/"><img src="/pmc/articles/instance/3474327/bin/nihms354873f4.gif" src-large="/pmc/articles/instance/3474327/bin/nihms354873f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3474327/" ref="ordinalpos=445&amp;ncbi_uid=4782640&amp;link_uid=PMC3474327" image-link="/pmc/articles/PMC3474327/figure/F4/" class="imagepopup">Figure 4. Smad-Dependent BMP <span class="highlight" style="background-color:">Signaling</span> Is Required for Lung Differentiation.  From: Generation of Multipotent Lung and Airway Progenitors from Mouse ESCs and Patient-Specific Cystic Fibrosis iPSCs. </a></p><div class="supp"><p class="details"><div>(A) NKX2.1 induction with 20 ng/ml FGF2 and 5 nM GSK3iXV only (control) or with additional BMP4 (20 ng/ml), BMP2 (20 ng/ml), BMP7 (20 ng/ml), BMP4 (20 ng/ml) + 10 μM Dorsomorphin, or BMP4 (20 ng/ml) + 1 μM PD98059. Scale bar, 100 μm.<br />(B) BMP <span class="highlight" style="background-color:">signaling</span> through SMAD-dependent pathways is required for lung Nkx2.1+ differentiation. In contrast, <span class="highlight" style="background-color:">signaling</span> through the MAP kinase <span class="highlight" style="background-color:">pathway</span> is not required for <i>Nkx2.1</i> expression. Dorsomorphin inhibits BMP <span class="highlight" style="background-color:">signaling</span> through the SMAD <span class="highlight" style="background-color:">pathway</span> while PD98059 inhibits MAPK <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Hongmei Mou, et al. Cell Stem Cell. 2012 April 6;10(4):385-397.</div><p class="links"><a class="dblinks" href="/pubmed/22482504" ref="ordinalpos=445&amp;ncbi_uid=4782640&amp;link_uid=22482504">Citation</a><a class="dblinks" href="/pmc/articles/PMC3474327/" ref="ordinalpos=445&amp;ncbi_uid=4782640&amp;link_uid=PMC3474327">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>446.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F8/" ref="ordinalpos=446&amp;ncbi_uid=2525561&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-8.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=446&amp;ncbi_uid=2525561&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F8/" class="imagepopup">Figure 8.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Moral graph <i>D<sup>m </sup></i>obtained from ERBB <span class="highlight" style="background-color:">pathway</span></b>. Moral graph <i>D<sup>m </sup></i>corresponding to the chosen portion of ERBB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=446&amp;ncbi_uid=2525561&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=446&amp;ncbi_uid=2525561&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>447.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151536/figure/f7/" ref="ordinalpos=447&amp;ncbi_uid=252155&amp;link_uid=PMC151536" href="/pmc/articles/PMC151536/"><img src="/pmc/articles/instance/151536/bin/mb0230960007.gif" src-large="/pmc/articles/instance/151536/bin/mb0230960007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151536/" ref="ordinalpos=447&amp;ncbi_uid=252155&amp;link_uid=PMC151536" image-link="/pmc/articles/PMC151536/figure/f7/" class="imagepopup">FIG. 7. From: The Ras/Raf/MEK/Extracellular Signal-Regulated Kinase <span class="highlight" style="background-color:">Pathway</span> Induces Autocrine-Paracrine Growth Inhibition via the Leukemia Inhibitory Factor/JAK/STAT <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Signal transduction pathways for Ras/Raf-mediated growth inhibition and differentiation. The potential growth-inhibitory <span class="highlight" style="background-color:">signaling</span> by Ras/Raf is depicted. The autocrine-paracrine <span class="highlight" style="background-color:">signaling</span> diverges from the intracellular <span class="highlight" style="background-color:">signaling</span> distal to MEK/ERK. The extracellular <span class="highlight" style="background-color:">pathway</span> mediates its effect through LIF expression and activation of JAK-STAT3.</div></p></div><div class="aux"><div class="resc">Jong-In Park, et al. Mol Cell Biol. 2003 January;23(2):543-554.</div><p class="links"><a class="dblinks" href="/pubmed/12509453" ref="ordinalpos=447&amp;ncbi_uid=252155&amp;link_uid=12509453">Citation</a><a class="dblinks" href="/pmc/articles/PMC151536/" ref="ordinalpos=447&amp;ncbi_uid=252155&amp;link_uid=PMC151536">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>448.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F9/" ref="ordinalpos=448&amp;ncbi_uid=2525560&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-9.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=448&amp;ncbi_uid=2525560&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F9/" class="imagepopup">Figure 9.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Triangulated graph <i>D<sup>t </sup></i>obtained from ERBB <span class="highlight" style="background-color:">pathway</span></b>. Triangulated graph <i>D<sup>t </sup></i>corresponding to the chosen portion of ERBB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=448&amp;ncbi_uid=2525560&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=448&amp;ncbi_uid=2525560&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>449.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3950212/figure/pone-0091332-g005/" ref="ordinalpos=449&amp;ncbi_uid=5906856&amp;link_uid=PMC3950212" href="/pmc/articles/PMC3950212/"><img src="/pmc/articles/instance/3950212/bin/pone.0091332.g005.gif" src-large="/pmc/articles/instance/3950212/bin/pone.0091332.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3950212/" ref="ordinalpos=449&amp;ncbi_uid=5906856&amp;link_uid=PMC3950212" image-link="/pmc/articles/PMC3950212/figure/pone-0091332-g005/" class="imagepopup">Figure 5. PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in the inhibitory effect of LFG-500 on NF-κB activation..  From: LFG-500 Inhibits the Invasion of Cancer Cells via Down-Regulation of PI3K/AKT/NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) LFG-500 has no influence on MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MDA-MB-231 cells were treated with various concentrations (2, 4, and 8 μM) of LFG-500 for 24 h. Total expression and phosphorylation of ERK, JNK, and p38 were analyzed by Western blotting. β-actin was used as a loading control. (<b>B</b>) LFG-500 inhibits the expression of PI3K p85α/γ and p-AKT. The expression of target proteins was determined by Western blotting. Densitometric analysis of the proteins studied (right). <i>*P</i>&lt;0.05 or <i>**P</i>&lt;0.01 represents significant difference from the control group.</div></p></div><div class="aux"><div class="resc">Chenglin Li, et al. PLoS One. 2014;9(3):e91332.</div><p class="links"><a class="dblinks" href="/pubmed/24618693" ref="ordinalpos=449&amp;ncbi_uid=5906856&amp;link_uid=24618693">Citation</a><a class="dblinks" href="/pmc/articles/PMC3950212/" ref="ordinalpos=449&amp;ncbi_uid=5906856&amp;link_uid=PMC3950212">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>450.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3896370/figure/pone-0085296-g006/" ref="ordinalpos=450&amp;ncbi_uid=5735664&amp;link_uid=PMC3896370" href="/pmc/articles/PMC3896370/"><img src="/pmc/articles/instance/3896370/bin/pone.0085296.g006.gif" src-large="/pmc/articles/instance/3896370/bin/pone.0085296.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3896370/" ref="ordinalpos=450&amp;ncbi_uid=5735664&amp;link_uid=PMC3896370" image-link="/pmc/articles/PMC3896370/figure/pone-0085296-g006/" class="imagepopup">Figure 6. Schematic diagram of the regulatory and <span class="highlight" style="background-color:">signaling</span> network of CDH17 in GC..  From: Targeting Cadherin-17 Inactivates Ras/Raf/MEK/ERK <span class="highlight" style="background-color:">Signaling</span> and Inhibits Cell Proliferation in Gastric Cancer. </a></p><div class="supp"><p class="details"><div>This schematic diagram demonstrates the inducing effect of CDH17 on Ras/Raf/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and illustrates the hypothetic involvement of integrins in GC. CDH17 indirectly affects integrins to stabilize their structure and activity. The up-regulation of cadherin-integrin <span class="highlight" style="background-color:">signaling</span> activates the Ras/Raf/MEK/ERK <span class="highlight" style="background-color:">pathway</span>. The activation of ERK regulates various nuclear and cytoplasmic substrates, including p53 and p21, which involve in diverse cellular responses, such as cell proliferation, migration, adhesion, colony formation, cell-cycle and apoptosis.</div></p></div><div class="aux"><div class="resc">Zhaohu Lin, et al. PLoS One. 2014;9(1):e85296.</div><p class="links"><a class="dblinks" href="/pubmed/24465527" ref="ordinalpos=450&amp;ncbi_uid=5735664&amp;link_uid=24465527">Citation</a><a class="dblinks" href="/pmc/articles/PMC3896370/" ref="ordinalpos=450&amp;ncbi_uid=5735664&amp;link_uid=PMC3896370">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>451.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2577137/figure/fig4/" ref="ordinalpos=451&amp;ncbi_uid=1770812&amp;link_uid=PMC2577137" href="/pmc/articles/PMC2577137/"><img src="/pmc/articles/instance/2639227/bin/carcinbgn186f04_ht.gif" src-large="/pmc/articles/instance/2639227/bin/carcinbgn186f04_ht.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2577137/" ref="ordinalpos=451&amp;ncbi_uid=1770812&amp;link_uid=PMC2577137" image-link="/pmc/articles/PMC2577137/figure/fig4/" class="imagepopup">Fig. 4. From: Synergistic effects of Pten loss and WNT/CTNNB1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in ovarian granulosa cell tumor development and progression. </a></p><div class="supp"><p class="details"><div>Altered PI3K/AKT <span class="highlight" style="background-color:">signaling</span> in <i>Pten</i><sup>flox/flox</sup>;<i>Amhr2</i><sup>cre/+</sup> GCT. Immunoblot analyses of the indicated PI3K/AKT <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> effectors in <i>Pten</i><sup>flox/flox</sup> granulosa cells (first two lanes) and <i>Pten</i><sup>flox/flox</sup>;<i>Amhr2</i><sup>cre/+</sup> granulosa cells tumors (final four lanes). Actin was used as a loading control.</div></p></div><div class="aux"><div class="resc">Marie-Noëlle Laguë, et al. Carcinogenesis. 2008 November;29(11):2062-2072.</div><p class="links"><a class="dblinks" href="/pubmed/18687666" ref="ordinalpos=451&amp;ncbi_uid=1770812&amp;link_uid=18687666">Citation</a><a class="dblinks" href="/pmc/articles/PMC2577137/" ref="ordinalpos=451&amp;ncbi_uid=1770812&amp;link_uid=PMC2577137">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>452.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1367180/figure/f2/" ref="ordinalpos=452&amp;ncbi_uid=630604&amp;link_uid=PMC1367180" href="/pmc/articles/PMC1367180/"><img src="/pmc/articles/instance/1367180/bin/zmb0040657430002.gif" src-large="/pmc/articles/instance/1367180/bin/zmb0040657430002.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1367180/" ref="ordinalpos=452&amp;ncbi_uid=630604&amp;link_uid=PMC1367180" image-link="/pmc/articles/PMC1367180/figure/f2/" class="imagepopup">FIG. 2. From: Tumorigenesis Suppressor Pdcd4 Down-Regulates Mitogen-Activated Protein Kinase Kinase Kinase Kinase 1 Expression To Suppress Colon Carcinoma Cell Invasion. </a></p><div class="supp"><p class="details"><div>Pdcd4 inhibits the JNK <span class="highlight" style="background-color:">pathway</span> but not the ERK activation <span class="highlight" style="background-color:">pathway</span>. (A) JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. JNK can be activated through a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from Rac/cdc42 or, alternatively, through a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from MAP4K1. (B) Pdcd4 inhibits phosphorylation of JNK and MKK-4 but not ERK. Western blotting was performed on cell lysates from the indicated cells with phospho-JNK, phospho-MKK-4, and phospho-ERK antibodies. The corresponding blots were stripped and reprobed with Pdcd4, JNK, MKK-4, and actin antibodies.</div></p></div><div class="aux"><div class="resc">Hsin-Sheng Yang, et al. Mol Cell Biol. 2006 February;26(4):1297-1306.</div><p class="links"><a class="dblinks" href="/pubmed/16449643" ref="ordinalpos=452&amp;ncbi_uid=630604&amp;link_uid=16449643">Citation</a><a class="dblinks" href="/pmc/articles/PMC1367180/" ref="ordinalpos=452&amp;ncbi_uid=630604&amp;link_uid=PMC1367180">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>453.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4246620/figure/f3-ol-09-01-0235/" ref="ordinalpos=453&amp;ncbi_uid=6891248&amp;link_uid=PMC4246620" href="/pmc/articles/PMC4246620/"><img src="/pmc/articles/instance/4246620/bin/OL-09-01-0235-g02.gif" src-large="/pmc/articles/instance/4246620/bin/OL-09-01-0235-g02.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4246620/" ref="ordinalpos=453&amp;ncbi_uid=6891248&amp;link_uid=PMC4246620" image-link="/pmc/articles/PMC4246620/figure/f3-ol-09-01-0235/" class="imagepopup">Figure 3.  From: The expression of Wnt-1 inducible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein-2 in astrocytoma: Correlation between pathological grade and clinical outcome. </a></p><div class="supp"><p class="details"><div>Expression levels of WISP-2 were found to significantly correlate with the survival of astrocytoma patients. Kaplan-Meier estimates of progression-free survival and overall survival are shown according to expression levels of WISP-2 for 154 astrocytomas patients. P-values were obtained using the log-rank test. WISP-2, Wnt-1 inducible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein-2.</div></p></div><div class="aux"><div class="resc">GELEI XIAO, et al. Oncol Lett. 2015 January;9(1):235-240.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=453&amp;ncbi_uid=6891248&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4246620/" ref="ordinalpos=453&amp;ncbi_uid=6891248&amp;link_uid=PMC4246620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>454.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2723940/figure/F3/" ref="ordinalpos=454&amp;ncbi_uid=1906451&amp;link_uid=PMC2723940" href="/pmc/articles/PMC2723940/"><img src="/pmc/articles/instance/2723940/bin/nihms98161f3.gif" src-large="/pmc/articles/instance/2723940/bin/nihms98161f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2723940/" ref="ordinalpos=454&amp;ncbi_uid=1906451&amp;link_uid=PMC2723940" image-link="/pmc/articles/PMC2723940/figure/F3/" class="imagepopup">Figure 3.  From: PTH and PTHrP <span class="highlight" style="background-color:">Signaling</span> in Osteoblasts. </a></p><div class="supp"><p class="details"><div>A schema for the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulated by NHERF.  Upper panel shows the dominant cAMP <span class="highlight" style="background-color:">signaling</span> pathways in most cells.  Lower panel illustrates how NHERF shift <span class="highlight" style="background-color:">signaling</span> from adenylate cyclase (AC) dominance to phospholipace C (PLC) dominance.</div></p></div><div class="aux"><div class="resc">Nabanita S. Datta, et al. Cell Signal. ;21(8):1245-1254.</div><p class="links"><a class="dblinks" href="/pubmed/19249350" ref="ordinalpos=454&amp;ncbi_uid=1906451&amp;link_uid=19249350">Citation</a><a class="dblinks" href="/pmc/articles/PMC2723940/" ref="ordinalpos=454&amp;ncbi_uid=1906451&amp;link_uid=PMC2723940">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>455.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4309023/figure/bau126-F1/" ref="ordinalpos=455&amp;ncbi_uid=7212789&amp;link_uid=PMC4309023" href="/pmc/articles/PMC4309023/"><img src="/pmc/articles/instance/4309023/bin/bau126f1p.gif" src-large="/pmc/articles/instance/4309023/bin/bau126f1p.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4309023/" ref="ordinalpos=455&amp;ncbi_uid=7212789&amp;link_uid=PMC4309023" image-link="/pmc/articles/PMC4309023/figure/bau126-F1/" class="imagepopup">Figure 1. From: Comparison of human cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> databases—evolution, drawbacks and challenges. </a></p><div class="supp"><p class="details"><div>Evolutionary tree of the existing and active <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> databases. This figure clearly depicts that KEGG, SPAD and STKE can be thought as the pioneers in the field of the development of human cell <span class="highlight" style="background-color:">signaling</span> databases. In the subsequent years, the evolution of human cell <span class="highlight" style="background-color:">signaling</span> databases is mainly led by the ‘Commercial Groups’ and several ‘Academic Groups’. The first commercial database, BIOCARTA and GENEGO were launched in 2000. In the subsequent years, several databases, like Cell <span class="highlight" style="background-color:">Signaling</span> TECHNOLOGY (2002), PROTEIN LOUNGE (2003), INVITROGEN (2005), Applied Biosystems (2007) and MILIPORE (2009) were also started to provide human cell <span class="highlight" style="background-color:">signaling</span> data freely to the users. By integrating the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components with corresponding antibodies, drugs and inhibitor molecules, these commercial databases are promoting the bench biologists to order and purchase those products more conveniently from their websites. On the other hand, the evolution of the other branch of the human cell <span class="highlight" style="background-color:">signaling</span> database is been continuously developed and monitored by several academic research groups across the world. Since 1995, almost each year, on an average one or two such databases are launched. The objectives of these databases are wider than the commercial databases and are not only restricted to the <span class="highlight" style="background-color:">pathway</span> data annotation and presentation but also to the analysis of cross talks of multiple pathways, drug target identification, <i>in silico</i> simulation, development of computer readable <span class="highlight" style="background-color:">pathway</span> data sharing process and <span class="highlight" style="background-color:">pathway</span> analysis, etc.</div></p></div><div class="aux"><div class="resc">Saikat Chowdhury, et al. Database (Oxford). 2015;2015:bau126.</div><p class="links"><a class="dblinks" href="/pubmed/25632107" ref="ordinalpos=455&amp;ncbi_uid=7212789&amp;link_uid=25632107">Citation</a><a class="dblinks" href="/pmc/articles/PMC4309023/" ref="ordinalpos=455&amp;ncbi_uid=7212789&amp;link_uid=PMC4309023">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>456.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3937030/figure/F1/" ref="ordinalpos=456&amp;ncbi_uid=5897631&amp;link_uid=PMC3937030" href="/pmc/articles/PMC3937030/"><img src="/pmc/articles/instance/3937030/bin/1472-6750-14-15-1.gif" src-large="/pmc/articles/instance/3937030/bin/1472-6750-14-15-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3937030/" ref="ordinalpos=456&amp;ncbi_uid=5897631&amp;link_uid=PMC3937030" image-link="/pmc/articles/PMC3937030/figure/F1/" class="imagepopup">Figure 1.  From: The relationship between mTOR signalling <span class="highlight" style="background-color:">pathway</span> and recombinant antibody productivity in CHO cell lines. </a></p><div class="supp"><p class="details"><div><b>The mTOR <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> This <span class="highlight" style="background-color:">pathway</span> involves communication between the upstream proteins that respond directly to such extracellular signals as nutrient availability, energy status, and physical stresses.</div></p></div><div class="aux"><div class="resc">Raihana Edros, et al. BMC Biotechnol. 2014;14:15-15.</div><p class="links"><a class="dblinks" href="/pubmed/24533650" ref="ordinalpos=456&amp;ncbi_uid=5897631&amp;link_uid=24533650">Citation</a><a class="dblinks" href="/pmc/articles/PMC3937030/" ref="ordinalpos=456&amp;ncbi_uid=5897631&amp;link_uid=PMC3937030">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>457.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3705708/figure/F4/" ref="ordinalpos=457&amp;ncbi_uid=5075366&amp;link_uid=PMC3705708" href="/pmc/articles/PMC3705708/"><img src="/pmc/articles/instance/3705708/bin/nihms-490798-f0004.gif" src-large="/pmc/articles/instance/3705708/bin/nihms-490798-f0004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3705708/" ref="ordinalpos=457&amp;ncbi_uid=5075366&amp;link_uid=PMC3705708" image-link="/pmc/articles/PMC3705708/figure/F4/" class="imagepopup">Fig. 4. The Hh <span class="highlight" style="background-color:">signaling</span> network.  From: The Hedgehog Signal Transduction Network. </a></p><div class="supp"><p class="details"><div>In vertebrates, Hh binding to Ptc modulates one or more of the following <span class="highlight" style="background-color:">signaling</span> modules: non-canonical type I Hh <span class="highlight" style="background-color:">signaling</span> exclusively through Ptc; canonical Hh <span class="highlight" style="background-color:">signaling</span> through Ptc, Smo, and Glis (mediated in some instances by G<sub>i</sub> proteins); and a noncanonical type II <span class="highlight" style="background-color:">signaling</span> through Smo and G<sub>i</sub> proteins. The cellular responses to each <span class="highlight" style="background-color:">signaling</span> module are illustrated at the bottom. The red crosses indicate inhibition of the <span class="highlight" style="background-color:">pathway</span> in the presence of Hh proteins.</div></p></div><div class="aux"><div class="resc">David J. Robbins, et al. Sci Signal. ;5(246):re6-re6.</div><p class="links"><a class="dblinks" href="/pubmed/23074268" ref="ordinalpos=457&amp;ncbi_uid=5075366&amp;link_uid=23074268">Citation</a><a class="dblinks" href="/pmc/articles/PMC3705708/" ref="ordinalpos=457&amp;ncbi_uid=5075366&amp;link_uid=PMC3705708">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>458.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3363004/figure/fig6/" ref="ordinalpos=458&amp;ncbi_uid=4047081&amp;link_uid=PMC3363004" href="/pmc/articles/PMC3363004/"><img src="/pmc/articles/instance/3363004/bin/JBB2012-357060.006.gif" src-large="/pmc/articles/instance/3363004/bin/JBB2012-357060.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3363004/" ref="ordinalpos=458&amp;ncbi_uid=4047081&amp;link_uid=PMC3363004" image-link="/pmc/articles/PMC3363004/figure/fig6/" class="imagepopup">Figure 6.  From: Multiple <span class="highlight" style="background-color:">Signaling</span> Pathways Are Involved in the Interleukine-4 Regulated Expression of DC-SIGN in THP-1 Cell Line. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways involved in IL-4-induced DC-SIGN expression on THP-1 cells. ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> plays a main role by transcription factor of Ets-1 to active DC-SIGN promoter. JAK-STAT and NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> pathways are also involved, which may be induced by IL-4/IL-4R directly or by the interactions with ERK <span class="highlight" style="background-color:">pathway</span> indirectly.</div></p></div><div class="aux"><div class="resc">Changzhong Jin, et al. J Biomed Biotechnol. 2012;2012:357060.</div><p class="links"><a class="dblinks" href="/pubmed/22675249" ref="ordinalpos=458&amp;ncbi_uid=4047081&amp;link_uid=22675249">Citation</a><a class="dblinks" href="/pmc/articles/PMC3363004/" ref="ordinalpos=458&amp;ncbi_uid=4047081&amp;link_uid=PMC3363004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>459.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3260728/figure/F2/" ref="ordinalpos=459&amp;ncbi_uid=3783563&amp;link_uid=PMC3260728" href="/pmc/articles/PMC3260728/"><img src="/pmc/articles/instance/3260728/bin/psb0608_1219_fig002.gif" src-large="/pmc/articles/instance/3260728/bin/psb0608_1219_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3260728/" ref="ordinalpos=459&amp;ncbi_uid=3783563&amp;link_uid=PMC3260728" image-link="/pmc/articles/PMC3260728/figure/F2/" class="imagepopup">Figure 2.  From: Regulation of NB-LRR-type UNI and its related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> crosstalk between UNI-triggered and ER-family-dependent <span class="highlight" style="background-color:">signaling</span> pathways modulates cytokinin-related <span class="highlight" style="background-color:">pathway</span>. UNI harbors a CC (coiled-coil) domain, an NB (nucleotide-binding-site) domain and an LRR (leucine-rich repeat) domain. A ligand or an elicitor for activation of UNI proteins have not yet been identified. The ERECTA (ER) family members encode LRR (leucine-rich-repeat) receptor-like kinases in a subfamily of transmembrane-type receptors. Information about co-receptors that might form receptor complexes with ER-family members in UNI-related <span class="highlight" style="background-color:">signaling</span> and also about ligands for such receptor complexes is still lacking. Activities of ER-family members in <i>UNI</i>-expressing cells are required for <i>uni-1D</i>/+ phenotypes and also for modulation of cytokinin-related <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Naoyuki Uchida, et al. Plant Signal Behav. 2011 August;6(8):1219-1222.</div><p class="links"><a class="dblinks" href="/pubmed/21758016" ref="ordinalpos=459&amp;ncbi_uid=3783563&amp;link_uid=21758016">Citation</a><a class="dblinks" href="/pmc/articles/PMC3260728/" ref="ordinalpos=459&amp;ncbi_uid=3783563&amp;link_uid=PMC3260728">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>460.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3240594/figure/pgen-1002415-g005/" ref="ordinalpos=460&amp;ncbi_uid=3726496&amp;link_uid=PMC3240594" href="/pmc/articles/PMC3240594/"><img src="/pmc/articles/instance/3240594/bin/pgen.1002415.g005.gif" src-large="/pmc/articles/instance/3240594/bin/pgen.1002415.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3240594/" ref="ordinalpos=460&amp;ncbi_uid=3726496&amp;link_uid=PMC3240594" image-link="/pmc/articles/PMC3240594/figure/pgen-1002415-g005/" class="imagepopup">Figure 5. Cotargeting of miRNAs by oncogenic <span class="highlight" style="background-color:">signaling</span> pathways..  From: A Densely Interconnected Genome-Wide Network of MicroRNAs and Oncogenic Pathways Revealed Using Gene Expression Signatures. </a></p><div class="supp"><p class="details"><div>A) Models of miRNA cotargeting and <span class="highlight" style="background-color:">pathway</span> cotargeting. (left) In miRNA co-targeting, multiple independently acting miRNAs can target the same <span class="highlight" style="background-color:">pathway</span> to modulate <span class="highlight" style="background-color:">pathway</span> activity (right) In <span class="highlight" style="background-color:">pathway</span> cotargeting, a <span class="highlight" style="background-color:">pathway</span> acts to target multiple independent miRNAs, which exert either similar or distinct downstream effects. B) miRNAs jointly correlated to a common <span class="highlight" style="background-color:">pathway</span> tend to be expressed from the same genomic cluster. In Scenario 1, a miRNA pair regulating a common <span class="highlight" style="background-color:">pathway</span> is associated with the same genomic location. In Scenario 2, that miRNA pair occupies different locations.(bottom graph) miRNAs connected to the same <span class="highlight" style="background-color:">pathway</span> are likely to be transcribed from the same genomic cluster rather than from different genomic clusters. C) miRNAs jointly correlated to a common <span class="highlight" style="background-color:">pathway</span> tend to exhibit significant overlaps in target mRNAs. Scenario 1 – miRNA pairs jointly correlated to the same <span class="highlight" style="background-color:">pathway</span> tend to exert similar downstream effects ie high overlap of target mRNAs (overlap ratio &gt;0.6). Scenario 2 – miRNAs pairs jointly correlated to the same <span class="highlight" style="background-color:">pathway</span> tend to exert distinct effects (overlap ratio &lt;0.4). (bottom graph) miRNAs with significant overlaps of target mRNAs (overlap ratio &gt;0.6) are more likely to be jointly connected to the same <span class="highlight" style="background-color:">pathway</span>. p-values were computed from a chi-square test.</div></p></div><div class="aux"><div class="resc">Chia Huey Ooi, et al. PLoS Genet. 2011 December;7(12):e1002415.</div><p class="links"><a class="dblinks" href="/pubmed/22194702" ref="ordinalpos=460&amp;ncbi_uid=3726496&amp;link_uid=22194702">Citation</a><a class="dblinks" href="/pmc/articles/PMC3240594/" ref="ordinalpos=460&amp;ncbi_uid=3726496&amp;link_uid=PMC3240594">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>461.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3368236/figure/F5/" ref="ordinalpos=461&amp;ncbi_uid=4051032&amp;link_uid=PMC3368236" href="/pmc/articles/PMC3368236/"><img src="/pmc/articles/instance/3368236/bin/fnmol-05-00071-g005.gif" src-large="/pmc/articles/instance/3368236/bin/fnmol-05-00071-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3368236/" ref="ordinalpos=461&amp;ncbi_uid=4051032&amp;link_uid=PMC3368236" image-link="/pmc/articles/PMC3368236/figure/F5/" class="imagepopup">Figure 5.  From: Semaphorin <span class="highlight" style="background-color:">Signaling</span> in Vertebrate Neural Circuit Assembly. </a></p><div class="supp"><p class="details"><div><b>Sema3E-PlexD1 <span class="highlight" style="background-color:">signaling</span> regulates <span class="highlight" style="background-color:">pathway</span>-specific synapse formation in the striatum</b>. Direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs are functionally and molecularly distinct. Direct <span class="highlight" style="background-color:">pathway</span> MSNs express type 1 dopamine receptors (Drd1) and indirect <span class="highlight" style="background-color:">pathway</span> MSNs express type 2 dopamine receptors (Drd2). <i>PlexD1</i> is exprsssed by Drd1<sup>on</sup>-direct <span class="highlight" style="background-color:">pathway</span> MSNs in the striatum, whereas <i>Sema3E</i> is expressed by subsets of neurons in the main thalamic nuclei that project to the striatum. Loss of Sema3E-PlexD1 <span class="highlight" style="background-color:">signaling</span> causes functional and anatomical rearrangement of thalamostriatal synapses specifically in direct <span class="highlight" style="background-color:">pathway</span> MSNs without effects on corticostriatal synapses.</div></p></div><div class="aux"><div class="resc">Yutaka Yoshida. Front Mol Neurosci. 2012;5:71.</div><p class="links"><a class="dblinks" href="/pubmed/22685427" ref="ordinalpos=461&amp;ncbi_uid=4051032&amp;link_uid=22685427">Citation</a><a class="dblinks" href="/pmc/articles/PMC3368236/" ref="ordinalpos=461&amp;ncbi_uid=4051032&amp;link_uid=PMC3368236">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>462.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3554769/figure/pone-0053663-g001/" ref="ordinalpos=462&amp;ncbi_uid=4565433&amp;link_uid=PMC3554769" href="/pmc/articles/PMC3554769/"><img src="/pmc/articles/instance/3554769/bin/pone.0053663.g001.gif" src-large="/pmc/articles/instance/3554769/bin/pone.0053663.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3554769/" ref="ordinalpos=462&amp;ncbi_uid=4565433&amp;link_uid=PMC3554769" image-link="/pmc/articles/PMC3554769/figure/pone-0053663-g001/" class="imagepopup">Figure 1. A <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for lung cancer..  From: A Mathematical Model for MicroRNA in Lung Cancer. </a></p><div class="supp"><p class="details"><div>A <span class="highlight" style="background-color:">pathway</span> from EGF-EGFR complex to MMP, which includes miR-9 and let-7, is given in (A) and a simplified <span class="highlight" style="background-color:">pathway</span> is shown in (B).</div></p></div><div class="aux"><div class="resc">Hye-Won Kang, et al. PLoS One. 2013;8(1):e53663.</div><p class="links"><a class="dblinks" href="/pubmed/23365639" ref="ordinalpos=462&amp;ncbi_uid=4565433&amp;link_uid=23365639">Citation</a><a class="dblinks" href="/pmc/articles/PMC3554769/" ref="ordinalpos=462&amp;ncbi_uid=4565433&amp;link_uid=PMC3554769">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>463.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3430251/figure/f5-ijms-13-08562/" ref="ordinalpos=463&amp;ncbi_uid=4207932&amp;link_uid=PMC3430251" href="/pmc/articles/PMC3430251/"><img src="/pmc/articles/instance/3430251/bin/ijms-13-08562f5.gif" src-large="/pmc/articles/instance/3430251/bin/ijms-13-08562f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3430251/" ref="ordinalpos=463&amp;ncbi_uid=4207932&amp;link_uid=PMC3430251" image-link="/pmc/articles/PMC3430251/figure/f5-ijms-13-08562/" class="imagepopup">Figure 5.  From: Fraction from Wax Apple [Syzygium samarangense (Blume) Merrill and Perry] Fruit Extract Ameliorates Insulin Resistance via Modulating Insulin <span class="highlight" style="background-color:">Signaling</span> and Inflammation <span class="highlight" style="background-color:">Pathway</span> in Tumor Necrosis Factor ?-Treated FL83B Mouse Hepatocytes. </a></p><div class="supp"><p class="details"><div>Postulated mechanism of FWFE on modulating insulin <span class="highlight" style="background-color:">signaling</span> and inflammation <span class="highlight" style="background-color:">pathway</span> in TNF-α-induced insulin resistant FL83B cells.</div></p></div><div class="aux"><div class="resc">Szu-Chuan Shen, et al. Int J Mol Sci. 2012;13(7):8562-8577.</div><p class="links"><a class="dblinks" href="/pubmed/22942720" ref="ordinalpos=463&amp;ncbi_uid=4207932&amp;link_uid=22942720">Citation</a><a class="dblinks" href="/pmc/articles/PMC3430251/" ref="ordinalpos=463&amp;ncbi_uid=4207932&amp;link_uid=PMC3430251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>464.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4011580/figure/F6/" ref="ordinalpos=464&amp;ncbi_uid=6110889&amp;link_uid=PMC4011580" href="/pmc/articles/PMC4011580/"><img src="/pmc/articles/instance/4011580/bin/oncotarget-05-1026-g006.gif" src-large="/pmc/articles/instance/4011580/bin/oncotarget-05-1026-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4011580/" ref="ordinalpos=464&amp;ncbi_uid=6110889&amp;link_uid=PMC4011580" image-link="/pmc/articles/PMC4011580/figure/F6/" class="imagepopup">Figure 6. A proposed model for how bypassing SCFβ-TRCP-mediated ubiquitination and degradation of the NEDD4 oncoprotein might promote tumorigenesis in part through elevating the Akt oncogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by downregulating the PTEN tumor suppressor.  From: SCFβ-TRCP-mediated degradation of NEDD4 inhibits tumorigenesis through modulating the PTEN/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="aux"><div class="resc">Jia Liu, et al. Oncotarget. 2014 April;5(4):1026-1037.</div><p class="links"><a class="dblinks" href="/pubmed/24657926" ref="ordinalpos=464&amp;ncbi_uid=6110889&amp;link_uid=24657926">Citation</a><a class="dblinks" href="/pmc/articles/PMC4011580/" ref="ordinalpos=464&amp;ncbi_uid=6110889&amp;link_uid=PMC4011580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>465.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2666999/figure/F5/" ref="ordinalpos=465&amp;ncbi_uid=1829939&amp;link_uid=PMC2666999" href="/pmc/articles/PMC2666999/"><img src="/pmc/articles/instance/2666999/bin/zpq9990973280005.gif" src-large="/pmc/articles/instance/2666999/bin/zpq9990973280005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2666999/" ref="ordinalpos=465&amp;ncbi_uid=1829939&amp;link_uid=PMC2666999" image-link="/pmc/articles/PMC2666999/figure/F5/" class="imagepopup">Fig. 5. From: An adrenal ?-arrestin 1-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> underlies angiotensin II-induced aldosterone production in vitro and in vivo. </a></p><div class="supp"><p class="details"><div>Schematic representation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of AngII-induced aldosterone production mediated by βarr1. For details, see main text. PIP<sub>2</sub>, Phosphatidylinositol 4′,5′-bisphosphate; IP<sub>3</sub>, Inositol 1′,4′,5′- trisphosphate; pERK, phospho-ERK.</div></p></div><div class="aux"><div class="resc">Anastasios Lymperopoulos, et al. Proc Natl Acad Sci U S A. 2009 April 7;106(14):5825-5830.</div><p class="links"><a class="dblinks" href="/pubmed/19289825" ref="ordinalpos=465&amp;ncbi_uid=1829939&amp;link_uid=19289825">Citation</a><a class="dblinks" href="/pmc/articles/PMC2666999/" ref="ordinalpos=465&amp;ncbi_uid=1829939&amp;link_uid=PMC2666999">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>466.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3606625/figure/F1/" ref="ordinalpos=466&amp;ncbi_uid=4736569&amp;link_uid=PMC3606625" href="/pmc/articles/PMC3606625/"><img src="/pmc/articles/instance/3606625/bin/nihms-357375-f0001.gif" src-large="/pmc/articles/instance/3606625/bin/nihms-357375-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3606625/" ref="ordinalpos=466&amp;ncbi_uid=4736569&amp;link_uid=PMC3606625" image-link="/pmc/articles/PMC3606625/figure/F1/" class="imagepopup">Fig. 1. The EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is of critical importance to human cancers and of a high complexity.  From: EGFR-Targeted Therapy in Malignant Glioma: Novel Aspects and Mechanisms of Drug Resistance. </a></p><div class="supp"><p class="details"><div>The EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> exerts its biological effects <i>via</i> two major modes of actions, namely, the cytoplasmic/ traditional (<b>a</b>) and the nuclear (<b>b</b>) <span class="highlight" style="background-color:">signaling</span> modes.<br /><b>a.</b> The cytoplasmic/traditional EGFR <span class="highlight" style="background-color:">pathway</span> is consisted of five major modules: PLC-γ-CaMK/PKC, Ras-Raf-MAPK, PI-3K-Akt-mTOR, JAK2/STAT3 and STAT3. Activation of these <span class="highlight" style="background-color:">signaling</span> modules often leads to tumorigenesis, tumor proliferation, metastasis, chemoresistance, and radio-resistance.<br /><b>b.</b> Nuclear EGFR has three key functions: (i) gene transactivation, (ii) tyrosine kinase, and (iii) protein-protein interaction.</div></p></div><div class="aux"><div class="resc">Hui-Wen Lo. Curr Mol Pharmacol. 2010 January;3(1):37-52.</div><p class="links"><a class="dblinks" href="/pubmed/20030624" ref="ordinalpos=466&amp;ncbi_uid=4736569&amp;link_uid=20030624">Citation</a><a class="dblinks" href="/pmc/articles/PMC3606625/" ref="ordinalpos=466&amp;ncbi_uid=4736569&amp;link_uid=PMC3606625">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>467.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3684083/figure/F4/" ref="ordinalpos=467&amp;ncbi_uid=5660033&amp;link_uid=PMC3684083" href="/pmc/articles/PMC3684083/"><img src="/pmc/articles/instance/3684083/bin/nihms459788f4.gif" src-large="/pmc/articles/instance/3684083/bin/nihms459788f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3684083/" ref="ordinalpos=467&amp;ncbi_uid=5660033&amp;link_uid=PMC3684083" image-link="/pmc/articles/PMC3684083/figure/F4/" class="imagepopup">Fig. 4. Overview of common signal <span class="highlight" style="background-color:">pathway</span> activation in GEP-NENs.  From: EGFR/TGF? and TGF?/CTGF <span class="highlight" style="background-color:">Signaling</span> in Neuroendocrine Neoplasia: Theoretical Therapeutic Targets. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> pathways activated either directly or due to cross-activation is MAPK and AKT/mTOR <span class="highlight" style="background-color:">signaling</span>. All ligands signal via these pathways to positively regulate proliferation and metastasis. Separately, an event that only occurs in a subset of GEP-NENs (Small-intestine NENs), TGFβ and CTGF activate fibrogenic pathways resulting in local (peritoneal) or distant (cardiac) disease. The common activation of RAS/RAF <span class="highlight" style="background-color:">signaling</span> suggests that therapeutic agents that target elements of this <span class="highlight" style="background-color:">pathway</span> may be potentially effective in NENs.</div></p></div><div class="aux"><div class="resc">M. Kidd, et al. Neuroendocrinology. ;97(1):35-44.</div><p class="links"><a class="dblinks" href="/pubmed/22710195" ref="ordinalpos=467&amp;ncbi_uid=5660033&amp;link_uid=22710195">Citation</a><a class="dblinks" href="/pmc/articles/PMC3684083/" ref="ordinalpos=467&amp;ncbi_uid=5660033&amp;link_uid=PMC3684083">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>468.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g005/" ref="ordinalpos=468&amp;ncbi_uid=4442588&amp;link_uid=PMC3517549" href="/pmc/articles/PMC3517549/"><img src="/pmc/articles/instance/3517549/bin/pone.0051657.g005.gif" src-large="/pmc/articles/instance/3517549/bin/pone.0051657.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517549/" ref="ordinalpos=468&amp;ncbi_uid=4442588&amp;link_uid=PMC3517549" image-link="/pmc/articles/PMC3517549/figure/pone-0051657-g005/" class="imagepopup">Figure 5. Summary of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> evolution..  From: The Evolution and Origin of Animal Toll-Like Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Revealed by Network-Level Molecular Evolutionary Analyses. </a></p><div class="supp"><p class="details"><div>Orthologs between different organisms are placed in the same relative position. If orthologs were not found, the ortholog place marked a question mark (?) with a circle. Divergence times of major metazoa groups are presented next to the relevant branches. Our result shows that the complete core <span class="highlight" style="background-color:">pathway</span> known from bilaterians is represented in the sponge <i>Amphimedon queenslandica</i>. The <span class="highlight" style="background-color:">pathway</span> is mostly absent in non-metazoans, <i>Saccharomyces cerevisiae</i> and <i>Monosiga brevicollis</i>. The nematode <i>Caenorhabditis elegans</i> and its relatives <i>Caenorhabditis Brigssae</i> are deficient in several of the <span class="highlight" style="background-color:">pathway</span> components. This may indicate of a functional loss of major components in nematodes.</div></p></div><div class="aux"><div class="resc">Xiaojun Song, et al. PLoS One. 2012;7(12):e51657.</div><p class="links"><a class="dblinks" href="/pubmed/23236523" ref="ordinalpos=468&amp;ncbi_uid=4442588&amp;link_uid=23236523">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517549/" ref="ordinalpos=468&amp;ncbi_uid=4442588&amp;link_uid=PMC3517549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>469.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3655553/figure/F1/" ref="ordinalpos=469&amp;ncbi_uid=4903353&amp;link_uid=PMC3655553" href="/pmc/articles/PMC3655553/"><img src="/pmc/articles/instance/3655553/bin/nihms264626f1.gif" src-large="/pmc/articles/instance/3655553/bin/nihms264626f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3655553/" ref="ordinalpos=469&amp;ncbi_uid=4903353&amp;link_uid=PMC3655553" image-link="/pmc/articles/PMC3655553/figure/F1/" class="imagepopup">Fig. 1. Schematic representation of β-AR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cardiac myocytes.  From: Modulation of ?-Adrenergic Receptor <span class="highlight" style="background-color:">Signaling</span> in Heart Failure and Longevity: Targeting Adenylyl Cyclase Type 5. </a></p><div class="supp"><p class="details"><div>Upon β-AR stimulation, the Gsα subunit-bound AC produces cAMP which activates target molecules, PKA and Epac. PKA regulates Ca<sup>2+</sup> homeostasis through SR and LTCC, increases contractility. Under conditions of excessive sympathetic stimulation, PKA-induced Ca<sup>2+</sup> leads to hypertrophy. On the other hand, activated PKA inhibits Raf, which results in the activation of the MEK/ERK <span class="highlight" style="background-color:">signaling</span>, and thus suppression of apoptosis. In contrast, Epac, the other target protein of cAMP than PKA, induces myocyte apoptosis via the P38MAPK <span class="highlight" style="background-color:">pathway</span>. Upon activation of β<sub>2</sub>-AR, the Giβγ subunits are released and activate the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. Activated Akt inhibits apoptosis, and accelerates hypertrophy</div></p></div><div class="aux"><div class="resc">David Ho, et al. Heart Fail Rev. 2010 September;15(5):495-512.</div><p class="links"><a class="dblinks" href="/pubmed/20658186" ref="ordinalpos=469&amp;ncbi_uid=4903353&amp;link_uid=20658186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3655553/" ref="ordinalpos=469&amp;ncbi_uid=4903353&amp;link_uid=PMC3655553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>470.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3103347/figure/F5/" ref="ordinalpos=470&amp;ncbi_uid=3364423&amp;link_uid=PMC3103347" href="/pmc/articles/PMC3103347/"><img src="/pmc/articles/instance/3103347/bin/zbc0281165510005.gif" src-large="/pmc/articles/instance/3103347/bin/zbc0281165510005.jpg" alt="FIGURE 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3103347/" ref="ordinalpos=470&amp;ncbi_uid=3364423&amp;link_uid=PMC3103347" image-link="/pmc/articles/PMC3103347/figure/F5/" class="imagepopup">FIGURE 5. From: Secretoglobin 3A2 Suppresses Bleomycin-induced Pulmonary Fibrosis by Transforming Growth Factor ? <span class="highlight" style="background-color:">Signaling</span> Down-regulation. </a></p><div class="supp"><p class="details"><div><b>Effect of SCGB3A2 on TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <span class="highlight" style="background-color:">Pathway</span> analysis of microarray data showed that the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was affected by administration of SCGB3A2. Median -fold differences from seven separate samples for each gene on which microarray analysis were run are color-coded based on the <i>scale</i> shown at the <i>bottom left</i>.</div></p></div><div class="aux"><div class="resc">Reiko Kurotani, et al. J Biol Chem. 2011 June 3;286(22):19682-19692.</div><p class="links"><a class="dblinks" href="/pubmed/21478551" ref="ordinalpos=470&amp;ncbi_uid=3364423&amp;link_uid=21478551">Citation</a><a class="dblinks" href="/pmc/articles/PMC3103347/" ref="ordinalpos=470&amp;ncbi_uid=3364423&amp;link_uid=PMC3103347">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>471.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC135652/figure/f8/" ref="ordinalpos=471&amp;ncbi_uid=298140&amp;link_uid=PMC135652" href="/pmc/articles/PMC135652/"><img src="/pmc/articles/instance/135652/bin/mb2120116008.gif" src-large="/pmc/articles/instance/135652/bin/mb2120116008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC135652/" ref="ordinalpos=471&amp;ncbi_uid=298140&amp;link_uid=PMC135652" image-link="/pmc/articles/PMC135652/figure/f8/" class="imagepopup">FIG. 8. From: Drosophila PDZ-GEF, a Guanine Nucleotide Exchange Factor for Rap1 GTPase, Reveals a Novel Upstream Regulatory Mechanism in the Mitogen-Activated Protein Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Model showing the role of dPDZ-GEF in the MAPK <span class="highlight" style="background-color:">pathway</span>. Various extracellular signals instigate the MAPK-related <span class="highlight" style="background-color:">signaling</span> cascade. Three GEFs, i.e., Sos, dC3G, and the newly discovered dPDZ-GEF, are involved in this <span class="highlight" style="background-color:">pathway</span>. dPDZ-GEF is the first genetically revealed GEF that is specific for Rap1. The dPDZ-GEF/Rap1 component shares the effector <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with the Sos/Ras component, and there are likely other effectors yet to be found. In this complex network, dPDZ-GEF controls cell fate and survival as an upstream regulatory component.</div></p></div><div class="aux"><div class="resc">Jun Hee Lee, et al. Mol Cell Biol. 2002 November;22(21):7658-7666.</div><p class="links"><a class="dblinks" href="/pubmed/12370312" ref="ordinalpos=471&amp;ncbi_uid=298140&amp;link_uid=12370312">Citation</a><a class="dblinks" href="/pmc/articles/PMC135652/" ref="ordinalpos=471&amp;ncbi_uid=298140&amp;link_uid=PMC135652">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>472.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2093951/figure/F9/" ref="ordinalpos=472&amp;ncbi_uid=1098163&amp;link_uid=PMC2093951" href="/pmc/articles/PMC2093951/"><img src="/pmc/articles/instance/2093951/bin/nihms31050f9.gif" src-large="/pmc/articles/instance/2093951/bin/nihms31050f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2093951/" ref="ordinalpos=472&amp;ncbi_uid=1098163&amp;link_uid=PMC2093951" image-link="/pmc/articles/PMC2093951/figure/F9/" class="imagepopup">Fig. 9. Model for the inhibitory effect of VIP on IL-12 <span class="highlight" style="background-color:">signaling</span>.  From: A NOVEL VIP <span class="highlight" style="background-color:">SIGNALING</span> <span class="highlight" style="background-color:">PATHWAY</span> IN T CELLS cAMP-&gt; Protein Tyrosine Phosphatase (SHP-2?)-&gt;JAK2/STAT4-&gt;Th1 differentiation. </a></p><div class="supp"><p class="details"><div>VIP acting through the VPAC1/2/ receptors activates adenylate cyclase resulting in the activation of PKA. PKA phosphorylates SHP-2, a protein tyrosine phosphatase which in turn acts on Jak2 and possibly directly on STAT4. Dephosphorylation of Jak2 and STAT4 interrupts the IL-12 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and prevents the expression of genes involved in Th1 differentiation and function.</div></p></div><div class="aux"><div class="resc">Li Liu, et al. Peptides. ;28(9):1814-1824.</div><p class="links"><a class="dblinks" href="/pubmed/17462790" ref="ordinalpos=472&amp;ncbi_uid=1098163&amp;link_uid=17462790">Citation</a><a class="dblinks" href="/pmc/articles/PMC2093951/" ref="ordinalpos=472&amp;ncbi_uid=1098163&amp;link_uid=PMC2093951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>473.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3245119/figure/gkr1208-F2/" ref="ordinalpos=473&amp;ncbi_uid=3747505&amp;link_uid=PMC3245119" href="/pmc/articles/PMC3245119/"><img src="/pmc/articles/instance/3245119/bin/gkr1208f2.gif" src-large="/pmc/articles/instance/3245119/bin/gkr1208f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3245119/" ref="ordinalpos=473&amp;ncbi_uid=3747505&amp;link_uid=PMC3245119" image-link="/pmc/articles/PMC3245119/figure/gkr1208-F2/" class="imagepopup">Figure 2. From: IPAVS: Integrated <span class="highlight" style="background-color:">Pathway</span> Resources, Analysis and Visualization System. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Single <span class="highlight" style="background-color:">pathway</span> view showing the non-canonical Wnt-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Protein CDC42 (highlighted) was clicked on the diagram [(A2) red rectangle], triggering the highlighting of the corresponding node on the tree (A1), the coloring of all its instances in the diagram (A2 red rectangle) and also the opening of the ‘Details’ panel (showing GO annotations) at the bottom (A4). Additionally, a context-popup-window displays the structure of CDC42 (A3). (<b>B</b>) Multi-<span class="highlight" style="background-color:">pathway</span> view showing the calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, with the view zoomed in to the Sarcoplasmic region. The <span class="highlight" style="background-color:">pathway</span> from IPAVS curated in the context of cardiac tissue (top) is compared with corresponding <span class="highlight" style="background-color:">pathway</span> from KEGG. The IPAVS <span class="highlight" style="background-color:">pathway</span> shows many more interactions specific to cardiac tissue regulating calcium homeostasis.</div></p></div><div class="aux"><div class="resc">Pradeep Kumar Sreenivasaiah, et al. Nucleic Acids Res. 2012 January;40(D1):D803-D808.</div><p class="links"><a class="dblinks" href="/pubmed/22140115" ref="ordinalpos=473&amp;ncbi_uid=3747505&amp;link_uid=22140115">Citation</a><a class="dblinks" href="/pmc/articles/PMC3245119/" ref="ordinalpos=473&amp;ncbi_uid=3747505&amp;link_uid=PMC3245119">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>474.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3590362/figure/Fig1/" ref="ordinalpos=474&amp;ncbi_uid=4707485&amp;link_uid=PMC3590362" href="/pmc/articles/PMC3590362/"><img src="/pmc/articles/instance/3590362/bin/12079_2012_181_Fig1_HTML.gif" src-large="/pmc/articles/instance/3590362/bin/12079_2012_181_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3590362/" ref="ordinalpos=474&amp;ncbi_uid=4707485&amp;link_uid=PMC3590362" image-link="/pmc/articles/PMC3590362/figure/Fig1/" class="imagepopup">Fig. 1.  From: A multicellular signal transduction network of AGE/RAGE <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of AGE/RAGE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in NetSlim- This map represents the NetSlim reactions of AGE/RAGE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in different cell types. The different types of reactions are distinguished by colors described in the legend. The map with reactions linked to their corresponding PubMed identifiers is available at http://www.netpath.org/netslim/age_<span class="highlight" style="background-color:">signaling</span>_pathways.html</div></p></div><div class="aux"><div class="resc">Sowmya Soman, et al. J Cell Commun Signal. 2013 March;7(1):19-23.</div><p class="links"><a class="dblinks" href="/pubmed/23161412" ref="ordinalpos=474&amp;ncbi_uid=4707485&amp;link_uid=23161412">Citation</a><a class="dblinks" href="/pmc/articles/PMC3590362/" ref="ordinalpos=474&amp;ncbi_uid=4707485&amp;link_uid=PMC3590362">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>475.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3857155/figure/F1/" ref="ordinalpos=475&amp;ncbi_uid=5984614&amp;link_uid=PMC3857155" href="/pmc/articles/PMC3857155/"><img src="/pmc/articles/instance/3857155/bin/nihms-523071-f0001.gif" src-large="/pmc/articles/instance/3857155/bin/nihms-523071-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3857155/" ref="ordinalpos=475&amp;ncbi_uid=5984614&amp;link_uid=PMC3857155" image-link="/pmc/articles/PMC3857155/figure/F1/" class="imagepopup">Figure 1.  From: New Connections Between Old Pathways: PDK1 <span class="highlight" style="background-color:">Signaling</span> Promotes Cellular Transformation through PLK1-Dependent MYC Stabilization. </a></p><div class="supp"><p class="details"><div>A basic depiction of the classical view of the PI3K <span class="highlight" style="background-color:">signaling</span> network (black arrows) as well as the novel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> revealed in the study by Tan et al. (5) (green arrows). Tumors with hyperactive PDK1 <span class="highlight" style="background-color:">signaling</span>, in addition to <span class="highlight" style="background-color:">signaling</span> through the classical downstream effector AKT, induce a novel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through phosphorylation of PLK1. Phosphorylation and activation of PLK1 results in phosphorylation of Serine 62 of MYC, leading to stabilization of MYC protein which promotes cellular transformation, in part through transcriptional upregulation of a cancer stem cell-like gene signature. As reported by Tan and colleagues (5), therapeutic targeting of this novel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with inhibitors of PDK1 (BX795 or BX912) or PLK1 (BI2536 or GW843682X) promotes programmed cell death of cancer cells that rely on MYC hyperactivation.</div></p></div><div class="aux"><div class="resc">John T. Cunningham, et al. Cancer Discov. ;3(10):10.1158/2159-8290.CD-13-0581.</div><p class="links"><a class="dblinks" href="/pubmed/24124229" ref="ordinalpos=475&amp;ncbi_uid=5984614&amp;link_uid=24124229">Citation</a><a class="dblinks" href="/pmc/articles/PMC3857155/" ref="ordinalpos=475&amp;ncbi_uid=5984614&amp;link_uid=PMC3857155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>476.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3606804/figure/fig1/" ref="ordinalpos=476&amp;ncbi_uid=4740307&amp;link_uid=PMC3606804" href="/pmc/articles/PMC3606804/"><img src="/pmc/articles/instance/3606804/bin/OXIMED2013-306073.001.gif" src-large="/pmc/articles/instance/3606804/bin/OXIMED2013-306073.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3606804/" ref="ordinalpos=476&amp;ncbi_uid=4740307&amp;link_uid=PMC3606804" image-link="/pmc/articles/PMC3606804/figure/fig1/" class="imagepopup">Figure 1.  From: Preventive and Therapeutic Effects of MG132 by Activating Nrf2-ARE <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> on Oxidative Stress-Induced Cardiovascular and Renal Injury. </a></p><div class="supp"><p class="details"><div>(a) Nrf2/Keap1-ARE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> under physical condition and (b) oxidative stress condition. ARE: antioxidant-responsive element; Cys: cysteine; Keap1: Kelch-like ECH-associated protein 1; Nrf2: E2-related factor 2; Ub: ubiquitin.</div></p></div><div class="aux"><div class="resc">Wenpeng Cui, et al. Oxid Med Cell Longev. 2013;2013:306073.</div><p class="links"><a class="dblinks" href="/pubmed/23533688" ref="ordinalpos=476&amp;ncbi_uid=4740307&amp;link_uid=23533688">Citation</a><a class="dblinks" href="/pmc/articles/PMC3606804/" ref="ordinalpos=476&amp;ncbi_uid=4740307&amp;link_uid=PMC3606804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>477.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3045689/figure/F9/" ref="ordinalpos=477&amp;ncbi_uid=2811748&amp;link_uid=PMC3045689" href="/pmc/articles/PMC3045689/"><img src="/pmc/articles/instance/3045689/bin/jcbn10-69f09.gif" src-large="/pmc/articles/instance/3045689/bin/jcbn10-69f09.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3045689/" ref="ordinalpos=477&amp;ncbi_uid=2811748&amp;link_uid=PMC3045689" image-link="/pmc/articles/PMC3045689/figure/F9/" class="imagepopup">Fig. 9.  From: Rebamipide suppresses TLR-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in regulating IRF3/7 and IFN-?/? reduction. </a></p><div class="supp"><p class="details"><div>Effect of rebamipide on NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Real-time RT-PCR of <i>NF-κB</i> in human colonic epithelial cells shows that LPS and poly(I:C) each increased <i>NF-κB</i> gene mRNA expression in colonic epithelial cells, and that adding rebamipide did not suppress either of these increases.</div></p></div><div class="aux"><div class="resc">Naotaka Ogasawara, et al. J Clin Biochem Nutr. 2011 March;48(2):154-160.</div><p class="links"><a class="dblinks" href="/pubmed/21373269" ref="ordinalpos=477&amp;ncbi_uid=2811748&amp;link_uid=21373269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3045689/" ref="ordinalpos=477&amp;ncbi_uid=2811748&amp;link_uid=PMC3045689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>478.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2042176/figure/F6/" ref="ordinalpos=478&amp;ncbi_uid=1142094&amp;link_uid=PMC2042176" href="/pmc/articles/PMC2042176/"><img src="/pmc/articles/instance/2042176/bin/zpq0410779130006.gif" src-large="/pmc/articles/instance/2042176/bin/zpq0410779130006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2042176/" ref="ordinalpos=478&amp;ncbi_uid=1142094&amp;link_uid=PMC2042176" image-link="/pmc/articles/PMC2042176/figure/F6/" class="imagepopup">Fig. 6. From: Kinetic insulation as an effective mechanism for achieving <span class="highlight" style="background-color:">pathway</span> specificity in intracellular <span class="highlight" style="background-color:">signaling</span> networks. </a></p><div class="supp"><p class="details"><div>Simultaneous stimulation of both pathways. (<i>A</i>) Application of a slowly increasing stimulus (<i>Left</i>, gray line) to <span class="highlight" style="background-color:">pathway</span> A and a square pulse lasting 45 min (<i>Left</i>, dashed gray line) to <span class="highlight" style="background-color:">pathway</span> B. Temporal profiles of the shared component C (<i>Left</i>, black line) and the terminal kinases KA (<i>Right</i>, solid line) and KB (<i>Right</i>, dashed line) are shown. (<i>B</i>) The same as in <i>A</i>, except the square pulse is applied to <span class="highlight" style="background-color:">pathway</span> B after a 1-h delay. (<i>C</i>) The same as in <i>A</i>, except the square pulse is applied after a 4-h delay. In this case, saturation of the common component C prevents <span class="highlight" style="background-color:">signaling</span> through <span class="highlight" style="background-color:">pathway</span> B.</div></p></div><div class="aux"><div class="resc">Marcelo Behar, et al. Proc Natl Acad Sci U S A. 2007 October 9;104(41):16146-16151.</div><p class="links"><a class="dblinks" href="/pubmed/17913886" ref="ordinalpos=478&amp;ncbi_uid=1142094&amp;link_uid=17913886">Citation</a><a class="dblinks" href="/pmc/articles/PMC2042176/" ref="ordinalpos=478&amp;ncbi_uid=1142094&amp;link_uid=PMC2042176">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>479.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041994/figure/Fig3/" ref="ordinalpos=479&amp;ncbi_uid=6228737&amp;link_uid=PMC4041994" href="/pmc/articles/PMC4041994/"><img src="/pmc/articles/instance/4041994/bin/12864_2014_6081_Fig3_HTML.gif" src-large="/pmc/articles/instance/4041994/bin/12864_2014_6081_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041994/" ref="ordinalpos=479&amp;ncbi_uid=6228737&amp;link_uid=PMC4041994" image-link="/pmc/articles/PMC4041994/figure/Fig3/" class="imagepopup">Figure 3.  From: In silico analyses reveal common cellular pathways affected by loss of heterozygosity (LOH) events in the lymphomagenesis of Non-Hodgkin's lymphoma (NHL). </a></p><div class="supp"><p class="details"><div> <b>Genes and pathways affected by LOH of</b> <b><i>PTPRJ</i></b> <b>. A)</b> Heatmap of genes differentially expressed between cases with retention (RET) and LOH calls. <b>B)</b> DNA polymerase (HSA03030), TP53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HSA04115) and Proteasome (HSA03050) were identified as the most enriched gene sets in cases with retention of <i>PTPRJ</i>. <b>C)</b> In cases with LOH, the gene sets VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HSA04370), Melanoma (HSA05218), ERBB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HSA04012), Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HSA04020), FC Epsilon RI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HSA04664) and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HSA04010) were the most enriched. Despite an FDR (false discovery rate) of 1 for all the enrichments, the results from this gene-set enrichment analysis were analysed based on the P values instead of the FDR scores. The high FDR values obtained in this analysis may be due to the low population size. The lists of all gene sets from this analysis are provided in Additional file 3 and Additional file 4.</div></p></div><div class="aux"><div class="resc">Carlos Aya-Bonilla, et al. BMC Genomics. 2014;15(1):390.</div><p class="links"><a class="dblinks" href="/pubmed/24885312" ref="ordinalpos=479&amp;ncbi_uid=6228737&amp;link_uid=24885312">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041994/" ref="ordinalpos=479&amp;ncbi_uid=6228737&amp;link_uid=PMC4041994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>480.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3546139/figure/F4/" ref="ordinalpos=480&amp;ncbi_uid=5661749&amp;link_uid=PMC3546139" href="/pmc/articles/PMC3546139/"><img src="/pmc/articles/instance/3546139/bin/nihms417389f4.gif" src-large="/pmc/articles/instance/3546139/bin/nihms417389f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3546139/" ref="ordinalpos=480&amp;ncbi_uid=5661749&amp;link_uid=PMC3546139" image-link="/pmc/articles/PMC3546139/figure/F4/" class="imagepopup">Figure 4. Selenocoxib-1-GSH inhibited Akt <span class="highlight" style="background-color:">signaling</span> to reduce the proliferative potential and promote apoptotic <span class="highlight" style="background-color:">signaling</span> in melanoma cells.  From: Simultaneous Targeting of COX-2 and AKT Using Selenocoxib-1-GSH to Inhibit Melanoma. </a></p><div class="supp"><p class="details"><div>4A. Selenocoxib-1-GSH inhibits the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. 4B. Selenocoxib-1-GSH activates the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. 4C. Selenocoxib-1-GSH decreased cyclin D1 protein levels indicating a reduction in cellular proliferation. 4D. Selenocoxib-1-GSH increased levels of cellular apoptosis.</div></p></div><div class="aux"><div class="resc">Raghavendra Gowda, et al. Mol Cancer Ther. ;12(1):3-15.</div><p class="links"><a class="dblinks" href="/pubmed/23112250" ref="ordinalpos=480&amp;ncbi_uid=5661749&amp;link_uid=23112250">Citation</a><a class="dblinks" href="/pmc/articles/PMC3546139/" ref="ordinalpos=480&amp;ncbi_uid=5661749&amp;link_uid=PMC3546139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>481.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3554743/figure/pone-0054111-g003/" ref="ordinalpos=481&amp;ncbi_uid=4562541&amp;link_uid=PMC3554743" href="/pmc/articles/PMC3554743/"><img src="/pmc/articles/instance/3554743/bin/pone.0054111.g003.gif" src-large="/pmc/articles/instance/3554743/bin/pone.0054111.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3554743/" ref="ordinalpos=481&amp;ncbi_uid=4562541&amp;link_uid=PMC3554743" image-link="/pmc/articles/PMC3554743/figure/pone-0054111-g003/" class="imagepopup">Figure 3. Differentially expressed miRNA-modulated putative targets in the part of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Genome-Wide Analysis of miRNA Signature Differentially Expressed in Doxorubicin-Resistant and Parental Human Hepatocellular Carcinoma Cell Lines. </a></p><div class="supp"><p class="details"><div>Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis was conducted to identify some of the putative targets for the most differentially expressed known and novel miRNAs in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Here only part of the <span class="highlight" style="background-color:">pathway</span> was displayed. miRNAs shown in green box are up-regulated while those in blue box were down-regulated.</div></p></div><div class="aux"><div class="resc">Jufeng Zhang, et al. PLoS One. 2013;8(1):e54111.</div><p class="links"><a class="dblinks" href="/pubmed/23359607" ref="ordinalpos=481&amp;ncbi_uid=4562541&amp;link_uid=23359607">Citation</a><a class="dblinks" href="/pmc/articles/PMC3554743/" ref="ordinalpos=481&amp;ncbi_uid=4562541&amp;link_uid=PMC3554743">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>482.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3128772/figure/f6/" ref="ordinalpos=482&amp;ncbi_uid=3565509&amp;link_uid=PMC3128772" href="/pmc/articles/PMC3128772/"><img src="/pmc/articles/instance/3128772/bin/fig-6.gif" src-large="/pmc/articles/instance/3128772/bin/fig-6.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3128772/" ref="ordinalpos=482&amp;ncbi_uid=3565509&amp;link_uid=PMC3128772" image-link="/pmc/articles/PMC3128772/figure/f6/" class="imagepopup">FIG. 6. From: Human Stem/Progenitor Cells from Bone Marrow Enhance Glial Differentiation of Rat Neural Stem Cells: A Role for Transforming Growth Factor ? and Notch <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic of cell <span class="highlight" style="background-color:">signaling</span> between rNSCs and hMSCs in coculture. Data obtained from multiple assays indicate activation of the classical Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in both the hMSCs and rNSCs. Notch <span class="highlight" style="background-color:">signaling</span> is thought to be unidirectional; therefore, the data suggest that Notch <span class="highlight" style="background-color:">signaling</span> is directed from hMSCs to rNSCs in some sub-populations of the cocultures and from rNSCs to hMSCs in other sub-populations.</div></p></div><div class="aux"><div class="resc">Andrew P. Robinson, et al. Stem Cells Dev. 2011 February;20(2):289-300.</div><p class="links"><a class="dblinks" href="/pubmed/20575640" ref="ordinalpos=482&amp;ncbi_uid=3565509&amp;link_uid=20575640">Citation</a><a class="dblinks" href="/pmc/articles/PMC3128772/" ref="ordinalpos=482&amp;ncbi_uid=3565509&amp;link_uid=PMC3128772">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>483.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3489884/figure/pone-0044561-g004/" ref="ordinalpos=483&amp;ncbi_uid=4361023&amp;link_uid=PMC3489884" href="/pmc/articles/PMC3489884/"><img src="/pmc/articles/instance/3489884/bin/pone.0044561.g004.gif" src-large="/pmc/articles/instance/3489884/bin/pone.0044561.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3489884/" ref="ordinalpos=483&amp;ncbi_uid=4361023&amp;link_uid=PMC3489884" image-link="/pmc/articles/PMC3489884/figure/pone-0044561-g004/" class="imagepopup">Figure 4. Gene expression of genes by KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Fetal Mesenchymal Stromal Cells Differentiating towards Chondrocytes Acquire a Gene Expression Profile Resembling Human Growth Plate Cartilage. </a></p><div class="supp"><p class="details"><div>KEGG <span class="highlight" style="background-color:">signaling</span> pathways significantly associated with chondrogenic differentiation of hfMSCs. For each <span class="highlight" style="background-color:">pathway</span>, genes showing the same distinct expression profile during 5 weeks of chondrogenic differentiation are depicted as groups.</div></p></div><div class="aux"><div class="resc">Sandy A. van Gool, et al. PLoS One. 2012;7(11):e44561.</div><p class="links"><a class="dblinks" href="/pubmed/23144774" ref="ordinalpos=483&amp;ncbi_uid=4361023&amp;link_uid=23144774">Citation</a><a class="dblinks" href="/pmc/articles/PMC3489884/" ref="ordinalpos=483&amp;ncbi_uid=4361023&amp;link_uid=PMC3489884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>484.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3331705/figure/A008052F1/" ref="ordinalpos=484&amp;ncbi_uid=6072827&amp;link_uid=PMC3331705" href="/pmc/articles/PMC3331705/"><img src="/pmc/articles/instance/3331705/bin/cshperspect-WNT-008052_F1.gif" src-large="/pmc/articles/instance/3331705/bin/cshperspect-WNT-008052_F1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3331705/" ref="ordinalpos=484&amp;ncbi_uid=6072827&amp;link_uid=PMC3331705" image-link="/pmc/articles/PMC3331705/figure/A008052F1/" class="imagepopup">Figure 1. From: Wnt <span class="highlight" style="background-color:">Signaling</span> in Cancer. </a></p><div class="supp"><p class="details"><div>Tumor suppressors and oncogenes in the Wnt <span class="highlight" style="background-color:">pathway</span>. Diagram of a basic Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in which oncogenes are depicted in green and tumor suppressors in red.</div></p></div><div class="aux"><div class="resc">Paul Polakis. Cold Spring Harb Perspect Biol. 2012 May;4(5):a008052.</div><p class="links"><a class="dblinks" href="/pubmed/22438566" ref="ordinalpos=484&amp;ncbi_uid=6072827&amp;link_uid=22438566">Citation</a><a class="dblinks" href="/pmc/articles/PMC3331705/" ref="ordinalpos=484&amp;ncbi_uid=6072827&amp;link_uid=PMC3331705">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>485.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4102778/figure/F7/" ref="ordinalpos=485&amp;ncbi_uid=6409523&amp;link_uid=PMC4102778" href="/pmc/articles/PMC4102778/"><img src="/pmc/articles/instance/4102778/bin/oncotarget-05-2881-g007.gif" src-large="/pmc/articles/instance/4102778/bin/oncotarget-05-2881-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4102778/" ref="ordinalpos=485&amp;ncbi_uid=6409523&amp;link_uid=PMC4102778" image-link="/pmc/articles/PMC4102778/figure/F7/" class="imagepopup">Figure 7. Overview of Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: GSK-3 as potential target for therapeutic intervention in cancer. </a></p><div class="supp"><p class="details"><div>The Hh <span class="highlight" style="background-color:">pathway</span> normally inhibits the membrane spanning PTCH1 GPCR receptor which is indicated by a squiggly red line crossing the membrane 14 times. PTCH1 normally inhibits SMO which is indicated by a purple squiggly line which crosses the membrane 7 times. SMO is then activated in the primary cilium and serves to inhibit SUFU which is indicated by a red octagon. SUFU normally inhibits the GLI1 and GLI2 transcriptions factors which are indicated in blue diamonds. The Rab23 GTPase may aid in the nuclear translocation of the transcription factors and is indicated by a purple oval. In contrast, GLI3 is activated by SUFU and may serve as a transcriptional repressor and is regulated by GSK-3. It is indicated in a black diamond. One of the genes that are regulated by GLI <span class="highlight" style="background-color:">signaling</span> is <i>IGF2</i>. Dysregulation of Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> occurs in certain cancers. Resistance to Hh <span class="highlight" style="background-color:">pathway</span> inhibitors may result from mutations in the pathways which lead to increased <span class="highlight" style="background-color:">pathway</span> expression and autocrine IGF2 expression. This figure is presented to give the reader an idea of the negative regulation of the Hh <span class="highlight" style="background-color:">pathway</span> by various components</div></p></div><div class="aux"><div class="resc">James A. McCubrey, et al. Oncotarget. 2014 May;5(10):2881-2911.</div><p class="links"><a class="dblinks" href="/pubmed/24931005" ref="ordinalpos=485&amp;ncbi_uid=6409523&amp;link_uid=24931005">Citation</a><a class="dblinks" href="/pmc/articles/PMC4102778/" ref="ordinalpos=485&amp;ncbi_uid=6409523&amp;link_uid=PMC4102778">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>486.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3923659/figure/pcbi-1003470-g002/" ref="ordinalpos=486&amp;ncbi_uid=5826775&amp;link_uid=PMC3923659" href="/pmc/articles/PMC3923659/"><img src="/pmc/articles/instance/3923659/bin/pcbi.1003470.g002.gif" src-large="/pmc/articles/instance/3923659/bin/pcbi.1003470.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3923659/" ref="ordinalpos=486&amp;ncbi_uid=5826775&amp;link_uid=PMC3923659" image-link="/pmc/articles/PMC3923659/figure/pcbi-1003470-g002/" class="imagepopup">Figure 2. IL-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reconstructed by combining related pathways and information from the literature..  From: The Structural <span class="highlight" style="background-color:">Pathway</span> of Interleukin 1 (IL-1) Initiated <span class="highlight" style="background-color:">Signaling</span> Reveals Mechanisms of Oncogenic Mutations and SNPs in Inflammation and Cancer. </a></p><div class="supp"><p class="details"><div> This detailed map of IL-1 <span class="highlight" style="background-color:">signaling</span> presents the protein-protein interactions and the resulting cellular events. The colored nodes represent proteins having experimentally identified 3D structures and the white nodes are the proteins without 3D structures. The edges represent protein-protein interactions (straight/dashed arrows relate to available/unavailable 3D structures of proteins) or associations leading to cellular events such as cell cycle or gene expression (dashed arrows beginning with circular heads).</div></p></div><div class="aux"><div class="resc">Saliha Ece Acuner Ozbabacan, et al. PLoS Comput Biol. 2014 February;10(2):e1003470.</div><p class="links"><a class="dblinks" href="/pubmed/24550720" ref="ordinalpos=486&amp;ncbi_uid=5826775&amp;link_uid=24550720">Citation</a><a class="dblinks" href="/pmc/articles/PMC3923659/" ref="ordinalpos=486&amp;ncbi_uid=5826775&amp;link_uid=PMC3923659">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>487.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2884164/figure/F9/" ref="ordinalpos=487&amp;ncbi_uid=2298053&amp;link_uid=PMC2884164" href="/pmc/articles/PMC2884164/"><img src="/pmc/articles/instance/2884164/bin/1755-8794-3-13-9.gif" src-large="/pmc/articles/instance/2884164/bin/1755-8794-3-13-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2884164/" ref="ordinalpos=487&amp;ncbi_uid=2298053&amp;link_uid=PMC2884164" image-link="/pmc/articles/PMC2884164/figure/F9/" class="imagepopup">Figure 9.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> networks mined from human pituitary adenoma proteomics data. </a></p><div class="supp"><p class="details"><div><b>ERK/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that is involved with pituitary adenoma comparative-proteomic data</b>. Red label = upregulated, green label = downregulated. The various shapes of nodes denote the different functions. A duplicated shape means this node contains multiple components. An arrow denotes the <span class="highlight" style="background-color:">pathway</span> direction. A line with a small circle denotes a biological result.</div></p></div><div class="aux"><div class="resc">Xianquan Zhan, et al. BMC Med Genomics. 2010;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/20426862" ref="ordinalpos=487&amp;ncbi_uid=2298053&amp;link_uid=20426862">Citation</a><a class="dblinks" href="/pmc/articles/PMC2884164/" ref="ordinalpos=487&amp;ncbi_uid=2298053&amp;link_uid=PMC2884164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>488.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3525024/figure/F7/" ref="ordinalpos=488&amp;ncbi_uid=5599694&amp;link_uid=PMC3525024" href="/pmc/articles/PMC3525024/"><img src="/pmc/articles/instance/3525024/bin/zh89991295950007.gif" src-large="/pmc/articles/instance/3525024/bin/zh89991295950007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3525024/" ref="ordinalpos=488&amp;ncbi_uid=5599694&amp;link_uid=PMC3525024" image-link="/pmc/articles/PMC3525024/figure/F7/" class="imagepopup">Figure 7.  From: Canonical and noncanonical Hedgehog <span class="highlight" style="background-color:">pathway</span> in the pathogenesis of multiple myeloma. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of canonical and noncanonical Hh <span class="highlight" style="background-color:">signaling</span> activation in MM.</b> The <span class="highlight" style="background-color:">signaling</span> is triggered by binding of ligand, produced by Hh <span class="highlight" style="background-color:">signaling</span> cells, such as MM cells or BMSCs, to Ptch1 on target cells (canonical or ligand-dependent/ receptor-induced <span class="highlight" style="background-color:">signaling</span>). This leads to inhibition of Ptch1 via cellular internalization and Smo localization on the cell surface. Smo activation leads to nuclear translocation of Gli transcription factors followed by expression of Gli target genes, such as c-myc, cyclin-D1, and Bcl-2. The biologic effect is cell proliferation, with deregulation contributing to tumorigenesis. Abnormal Hh <span class="highlight" style="background-color:">pathway</span> activation occurs also by the mechanisms of activation downstream to Smo (noncanonical or ligand-independent Hh <span class="highlight" style="background-color:">signaling</span>). Genetic alterations or ciliary protein overexpression leading to functional redundancy of Gli transcription factors, crosstalk between Hh <span class="highlight" style="background-color:">signaling</span>, and unrelated <span class="highlight" style="background-color:">pathway</span> are all causes of noncanonical Hh <span class="highlight" style="background-color:">signaling</span> activation. Canonical and noncanonical Hh pathways probably work in parallel, and a multitargeted approach directed against Hh <span class="highlight" style="background-color:">signaling</span> may be a more effective therapeutic strategy.</div></p></div><div class="aux"><div class="resc">Simona Blotta, et al. Blood. 2012 December 13;120(25):5002-5013.</div><p class="links"><a class="dblinks" href="/pubmed/22821765" ref="ordinalpos=488&amp;ncbi_uid=5599694&amp;link_uid=22821765">Citation</a><a class="dblinks" href="/pmc/articles/PMC3525024/" ref="ordinalpos=488&amp;ncbi_uid=5599694&amp;link_uid=PMC3525024">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>489.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4079089/figure/F1/" ref="ordinalpos=489&amp;ncbi_uid=6315568&amp;link_uid=PMC4079089" href="/pmc/articles/PMC4079089/"><img src="/pmc/articles/instance/4079089/bin/nihms582282f1.gif" src-large="/pmc/articles/instance/4079089/bin/nihms582282f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4079089/" ref="ordinalpos=489&amp;ncbi_uid=6315568&amp;link_uid=PMC4079089" image-link="/pmc/articles/PMC4079089/figure/F1/" class="imagepopup">Figure 1.  From: Growth arrest <span class="highlight" style="background-color:">signaling</span> of the Raf/MEK/ERK <span class="highlight" style="background-color:">pathway</span> in cancer. </a></p><div class="supp"><p class="details"><div>A conceptual model for tumor suppressive <span class="highlight" style="background-color:">signaling</span> of Raf/MEK/ERK. Oncogenic alterations leading to aberrant activation of Raf/MEK/ERK can direct the <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> to mediate growth arrest. Progression of tumorigenesis requires inactivation of this tumor suppressive response. In the absence of this mechanism, deregulated Raf/MEK/ERK activity is exploited to stimulate uncontrolled cell growth.</div></p></div><div class="aux"><div class="resc">Jong-In Park. Front Biol (Beijing). ;9(2):95-103.</div><p class="links"><a class="dblinks" href="/pubmed/24999356" ref="ordinalpos=489&amp;ncbi_uid=6315568&amp;link_uid=24999356">Citation</a><a class="dblinks" href="/pmc/articles/PMC4079089/" ref="ordinalpos=489&amp;ncbi_uid=6315568&amp;link_uid=PMC4079089">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>490.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4116523/figure/F2/" ref="ordinalpos=490&amp;ncbi_uid=6458773&amp;link_uid=PMC4116523" href="/pmc/articles/PMC4116523/"><img src="/pmc/articles/instance/4116523/bin/oncotarget-05-3823-g002.gif" src-large="/pmc/articles/instance/4116523/bin/oncotarget-05-3823-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4116523/" ref="ordinalpos=490&amp;ncbi_uid=6458773&amp;link_uid=PMC4116523" image-link="/pmc/articles/PMC4116523/figure/F2/" class="imagepopup">Figure 2. The effect of candidate peptides on VEGFR-3 phosphorylation and VEGFR-3-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Novel peptides suppress VEGFR-3 activity and antagonize VEGFR-3-mediated oncogenic effects. </a></p><div class="supp"><p class="details"><div>(<b>A</b> and <b>B</b>) Effects of candidate peptides on VEGFR-3-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A549 (<b>A</b>) and HEK293T/VEGFR-3 (<b>B</b>) cells were treated with rhVEGF-C in the absence or presence of candidate peptides or soluble VEGFR-3 (sR3; as the inhibitor for VEGF-C-induced VEGFR-3 activation). (<b>C</b> and <b>D</b>) Dose-dependent effects of P5 or P6 peptides on VEGFR-3-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A549 (<b>C</b>) and HEK293T/VEGFR-3 (<b>D</b>) cells were stimulated with rhVEGF-C with increased concentrations of P5 or P6 peptides. Cell lysates were harvested and subjected to SDS-PAGE followed by western blot analysis with anti–p-tyr1063/1068-VEGFR-3 and anti–p-src antibodies.</div></p></div><div class="aux"><div class="resc">Yi-Wen Chang, et al. Oncotarget. 2014 June;5(11):3823-3835.</div><p class="links"><a class="dblinks" href="/pubmed/25003617" ref="ordinalpos=490&amp;ncbi_uid=6458773&amp;link_uid=25003617">Citation</a><a class="dblinks" href="/pmc/articles/PMC4116523/" ref="ordinalpos=490&amp;ncbi_uid=6458773&amp;link_uid=PMC4116523">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>491.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4065255/figure/fig1/" ref="ordinalpos=491&amp;ncbi_uid=6284018&amp;link_uid=PMC4065255" href="/pmc/articles/PMC4065255/"><img src="/pmc/articles/instance/4065255/bin/1123f1.gif" src-large="/pmc/articles/instance/4065255/bin/1123f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4065255/" ref="ordinalpos=491&amp;ncbi_uid=6284018&amp;link_uid=PMC4065255" image-link="/pmc/articles/PMC4065255/figure/fig1/" class="imagepopup">Figure 1.  From: A Recent Global Selective Sweep on the age-1 Phosphatidylinositol 3-OH Kinase Regulator of the Insulin-Like <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Within Caenorhabditis remanei. </a></p><div class="supp"><p class="details"><div>Core components of the insulin-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i>. Genes examined in this study are indicated by a bold outline (clear <i>C. remanei</i> homologs of <i>akt-2</i> and <i>daf-18</i> could not be identified).</div></p></div><div class="aux"><div class="resc">Richard Jovelin, et al. G3 (Bethesda). 2014 June;4(6):1123-1133.</div><p class="links"><a class="dblinks" href="/pubmed/24727287" ref="ordinalpos=491&amp;ncbi_uid=6284018&amp;link_uid=24727287">Citation</a><a class="dblinks" href="/pmc/articles/PMC4065255/" ref="ordinalpos=491&amp;ncbi_uid=6284018&amp;link_uid=PMC4065255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>492.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2784807/figure/F4/" ref="ordinalpos=492&amp;ncbi_uid=2177807&amp;link_uid=PMC2784807" href="/pmc/articles/PMC2784807/"><img src="/pmc/articles/instance/2784807/bin/1471-2164-10-542-4.gif" src-large="/pmc/articles/instance/2784807/bin/1471-2164-10-542-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2784807/" ref="ordinalpos=492&amp;ncbi_uid=2177807&amp;link_uid=PMC2784807" image-link="/pmc/articles/PMC2784807/figure/F4/" class="imagepopup">Figure 4.  From: Gene network and <span class="highlight" style="background-color:">pathway</span> analysis of bovine mammary tissue challenged with Streptococcus uberis reveals induction of cell proliferation and inhibition of PPAR? <span class="highlight" style="background-color:">signaling</span> as potential mechanism for the negative relationships between immune response and lipid metabolism. </a></p><div class="supp"><p class="details"><div><b>PPARα/RXRα <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> expression among 2,102 DEG due to IMI with <i>Streptococcus uberis</i></b>. Red denotes up-regulation and green down-regulation of the gene.</div></p></div><div class="aux"><div class="resc">Kasey M Moyes, et al. BMC Genomics. 2009;10:542-542.</div><p class="links"><a class="dblinks" href="/pubmed/19925655" ref="ordinalpos=492&amp;ncbi_uid=2177807&amp;link_uid=19925655">Citation</a><a class="dblinks" href="/pmc/articles/PMC2784807/" ref="ordinalpos=492&amp;ncbi_uid=2177807&amp;link_uid=PMC2784807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>493.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4379025/figure/pone.0119512.g012/" ref="ordinalpos=493&amp;ncbi_uid=7401458&amp;link_uid=PMC4379025" href="/pmc/articles/PMC4379025/"><img src="/pmc/articles/instance/4379025/bin/pone.0119512.g012.gif" src-large="/pmc/articles/instance/4379025/bin/pone.0119512.g012.jpg" alt="Fig 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4379025/" ref="ordinalpos=493&amp;ncbi_uid=7401458&amp;link_uid=PMC4379025" image-link="/pmc/articles/PMC4379025/figure/pone.0119512.g012/" class="imagepopup">Fig 12. Schematic representation of intracellular <span class="highlight" style="background-color:">signaling</span> pathways leading to AFP expression and secretion..  From: Telomerase Inhibition Decreases Alpha-Fetoprotein Expression and Secretion by Hepatocellular Carcinoma Cell Lines: In Vitro and In Vivo Study. </a></p><div class="supp"><p class="details"><div>Our study demonstrates that the inhibition of telomerase, PKC and the PI3k/Akt/mTOR/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to decreases in AFP expression and secretion. In contrast, the inhibition of the MAPK/ERK <span class="highlight" style="background-color:">pathway</span> did not show any effect. However, the combination of a telomerase inhibitor with other <span class="highlight" style="background-color:">pathway</span> inhibitors (see results section) presented an additive effect with costunolide. The mechanism underlying the action of telomerase remains unclear. The modulation of AFP by telomerase may be either through the PI3k/Akt/mTOR/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> or through an independent <span class="highlight" style="background-color:">pathway</span>. Further investigations are needed to elucidate the modulation of AFP by telomerase.</div></p></div><div class="aux"><div class="resc">Roula Tahtouh, et al. PLoS One. 2015;10(3):e0119512.</div><p class="links"><a class="dblinks" href="/pubmed/25822740" ref="ordinalpos=493&amp;ncbi_uid=7401458&amp;link_uid=25822740">Citation</a><a class="dblinks" href="/pmc/articles/PMC4379025/" ref="ordinalpos=493&amp;ncbi_uid=7401458&amp;link_uid=PMC4379025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>494.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3883087/figure/F2/" ref="ordinalpos=494&amp;ncbi_uid=5684727&amp;link_uid=PMC3883087" href="/pmc/articles/PMC3883087/"><img src="/pmc/articles/instance/3883087/bin/nihms527962f2.gif" src-large="/pmc/articles/instance/3883087/bin/nihms527962f2.jpg" alt="Fig. (2)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3883087/" ref="ordinalpos=494&amp;ncbi_uid=5684727&amp;link_uid=PMC3883087" image-link="/pmc/articles/PMC3883087/figure/F2/" class="imagepopup">Fig. (2). Insulin/insulin-like growth factor 1 (IGF-1) <span class="highlight" style="background-color:">signaling</span> and cardiac aging in the mammalian system.  From: Cardiac Aging and Insulin Resistance: Could Insulin/Insulin-Like Growth Factor (IGF) <span class="highlight" style="background-color:">Signaling</span> be used as a Therapeutic Target?. </a></p><div class="supp"><p class="details"><div>A simplified insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cardiac cells and the effects associated with reduced phosphatidylinositol-4,5-bisphosphate 3-kinase (PI3K) <span class="highlight" style="background-color:">signaling</span> on markers of cardiac aging. IR: insulin receptor; IGF-1R: IGF-1 receptor; IRS: insulin receptor substrate.</div></p></div><div class="aux"><div class="resc">Sihem Boudina. Curr Pharm Des. ;19(32):5684-5694.</div><p class="links"><a class="dblinks" href="/pubmed/23448491" ref="ordinalpos=494&amp;ncbi_uid=5684727&amp;link_uid=23448491">Citation</a><a class="dblinks" href="/pmc/articles/PMC3883087/" ref="ordinalpos=494&amp;ncbi_uid=5684727&amp;link_uid=PMC3883087">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>495.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1331977/figure/pgen-0020006-g003/" ref="ordinalpos=495&amp;ncbi_uid=581258&amp;link_uid=PMC1331977" href="/pmc/articles/PMC1331977/"><img src="/pmc/articles/instance/1353280/bin/pgen.0020006.g003.gif" src-large="/pmc/articles/instance/1353280/bin/pgen.0020006.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1331977/" ref="ordinalpos=495&amp;ncbi_uid=581258&amp;link_uid=PMC1331977" image-link="/pmc/articles/PMC1331977/figure/pgen-0020006-g003/" class="imagepopup">Figure 3. An Expression Quantitative Trait Locus on Chromosome 2 Regulates Many GPCR Protein <span class="highlight" style="background-color:">Pathway</span> Genes.  From: Combined Expression Trait Correlations and Expression Quantitative Trait Locus Mapping. </a></p><div class="supp"><p class="details"><div>(A) Comapping of GPCR <span class="highlight" style="background-color:">pathway</span> traits to loci on Chromosome 2. The figure shows the peaks on Chromosome 2 versus LOD score for 174 GPCR protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> traits. The closed circles correspond to 38 seed traits.<br />(B) Comapping of GPCR <span class="highlight" style="background-color:">pathway</span> traits. The figure shows the magnitude (y-axis) versus location of LOD peaks (x-axis is scaled by cM) for 174 GPCR protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> traits. The closed circles correspond to the 38 seed traits. The interval mapping LOD profiles for all 38 seeds are included in Figure S2.</div></p></div><div class="aux"><div class="resc">Hong Lan, et al. PLoS Genet. 2006 January;2(1):e6.</div><p class="links"><a class="dblinks" href="/pubmed/16424919" ref="ordinalpos=495&amp;ncbi_uid=581258&amp;link_uid=16424919">Citation</a><a class="dblinks" href="/pmc/articles/PMC1331977/" ref="ordinalpos=495&amp;ncbi_uid=581258&amp;link_uid=PMC1331977">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>496.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3523725/figure/F7/" ref="ordinalpos=496&amp;ncbi_uid=4486478&amp;link_uid=PMC3523725" href="/pmc/articles/PMC3523725/"><img src="/pmc/articles/instance/3523725/bin/nihms425522f7.gif" src-large="/pmc/articles/instance/3523725/bin/nihms425522f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3523725/" ref="ordinalpos=496&amp;ncbi_uid=4486478&amp;link_uid=PMC3523725" image-link="/pmc/articles/PMC3523725/figure/F7/" class="imagepopup">Figure 7. A schematic diagram of LRP1-dependent vascular calcification.  From: PPAR? COUNTERACTS LRP1-INDUCED VASCULAR CALCIFICATION BY INHIBITING A WNT5A <span class="highlight" style="background-color:">SIGNALING</span> <span class="highlight" style="background-color:">PATHWAY</span>. </a></p><div class="supp"><p class="details"><div>When mice are fed an atherogenic diet, vascular calcification is induced if LRP1 is expressed in vSMCs. LRP1 positively regulates a Wnt5a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that activates chondrogenic-specific genes in the vascular wall. PPARγ and its target sFRP2, by counteracting the prochondrogenic Wnt5 <span class="highlight" style="background-color:">pathway</span>, protect against vascular calcification. Conversely, in the absence of LRP1, Wnt5a <span class="highlight" style="background-color:">signaling</span> is inhibited, an event that promotes intracellular cholesterol accumulation and foam cell development.</div></p></div><div class="aux"><div class="resc">Estelle Woldt, et al. Nat Commun. ;3:1077-1077.</div><p class="links"><a class="dblinks" href="/pubmed/23011131" ref="ordinalpos=496&amp;ncbi_uid=4486478&amp;link_uid=23011131">Citation</a><a class="dblinks" href="/pmc/articles/PMC3523725/" ref="ordinalpos=496&amp;ncbi_uid=4486478&amp;link_uid=PMC3523725">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>497.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2708980/figure/F8/" ref="ordinalpos=497&amp;ncbi_uid=1992579&amp;link_uid=PMC2708980" href="/pmc/articles/PMC2708980/"><img src="/pmc/articles/instance/2708980/bin/nihms109051f8.gif" src-large="/pmc/articles/instance/2708980/bin/nihms109051f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2708980/" ref="ordinalpos=497&amp;ncbi_uid=1992579&amp;link_uid=PMC2708980" image-link="/pmc/articles/PMC2708980/figure/F8/" class="imagepopup">Figure 8.  From: A proapoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving RasGRP, Erk, and Bim in B cells. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that links protein kinase C (PKC) and RasGRP1/3 to apoptosis in Toledo B-cell non-Hodgkins lymphoma (B-NHL) cells. Single-headed arrows denote positive regulatory influences while double-headed arrows denote reciprocal regulatory effects and physical associations between Bcl-2 family members. Points of action of the Mek inhibitor (U0126) and the pan-PKC inhibitor (Bis) are indicated.</div></p></div><div class="aux"><div class="resc">Stacey L. Stang, et al. Exp Hematol. ;37(1):122-134.</div><p class="links"><a class="dblinks" href="/pubmed/19100522" ref="ordinalpos=497&amp;ncbi_uid=1992579&amp;link_uid=19100522">Citation</a><a class="dblinks" href="/pmc/articles/PMC2708980/" ref="ordinalpos=497&amp;ncbi_uid=1992579&amp;link_uid=PMC2708980">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>498.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2822851/figure/pone-0009256-g001/" ref="ordinalpos=498&amp;ncbi_uid=2246452&amp;link_uid=PMC2822851" href="/pmc/articles/PMC2822851/"><img src="/pmc/articles/instance/2822851/bin/pone.0009256.g001.gif" src-large="/pmc/articles/instance/2822851/bin/pone.0009256.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2822851/" ref="ordinalpos=498&amp;ncbi_uid=2246452&amp;link_uid=PMC2822851" image-link="/pmc/articles/PMC2822851/figure/pone-0009256-g001/" class="imagepopup">Figure 1. Reduction of a <span class="highlight" style="background-color:">pathway</span> complexity using Atlas of <span class="highlight" style="background-color:">Signaling</span>..  From: Atlas of <span class="highlight" style="background-color:">Signaling</span> for Interpretation of Microarray Experiments. </a></p><div class="supp"><p class="details"><div>(A) Signal transduction <span class="highlight" style="background-color:">pathway</span> from integrin receptors to actin cytoskeleton built from individual proteins (red shapes) using relationships found in ResNet 6 database. Entrez official gene symbols are used for protein names. (B) Example of how a relation between two groups (functional classes) is formed in the Atlas of <span class="highlight" style="background-color:">Signaling</span>. (C) Same <span class="highlight" style="background-color:">pathway</span> as in (A) built with ontological groups: functional classes (orange hexagons) and cell process “actin-based cytoskeleton assembly” (yellow rectangle). The description of relationships between entities on the pathways is provided in the legend to Table 2.</div></p></div><div class="aux"><div class="resc">Ekaterina Kotelnikova, et al. PLoS One. 2010;5(2):e9256.</div><p class="links"><a class="dblinks" href="/pubmed/20174649" ref="ordinalpos=498&amp;ncbi_uid=2246452&amp;link_uid=20174649">Citation</a><a class="dblinks" href="/pmc/articles/PMC2822851/" ref="ordinalpos=498&amp;ncbi_uid=2246452&amp;link_uid=PMC2822851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>499.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3185015/figure/pone-0025641-g006/" ref="ordinalpos=499&amp;ncbi_uid=3275046&amp;link_uid=PMC3185015" href="/pmc/articles/PMC3185015/"><img src="/pmc/articles/instance/3185015/bin/pone.0025641.g006.gif" src-large="/pmc/articles/instance/3185015/bin/pone.0025641.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3185015/" ref="ordinalpos=499&amp;ncbi_uid=3275046&amp;link_uid=PMC3185015" image-link="/pmc/articles/PMC3185015/figure/pone-0025641-g006/" class="imagepopup">Figure 6. Rspo1 activates β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in XX germ cells..  From: RSPO1/?-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulates Oogonia Differentiation and Entry into Meiosis in the Mouse Fetal Ovary. </a></p><div class="supp"><p class="details"><div>A- Stabilization of β-catenin in XX germ cells is mediated by <i>Rspo1</i>. Active β-catenin (red) and MVH immunostaining (germ cells, green) in XY, XX <i>Rspo1<sup>+/−</sup></i> and <i>Rspo1<sup>−/−</sup></i> gonads at E14.5. DAPI (blue): nuclei. Arrowheads: nuclear β-catenin staining. B- Effectors of the WNT/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are expressed in XX germ cells. Quantitative RT-PCR analysis of <i>Axin2</i>, <i>Lef1</i>, <i>LRP6</i> expression in E13.5 germ cells (XX and XY <i>Oct4</i>-positive cells) and somatic cells (XX and XY <i>Oct4</i>-negative cells), using respectively <i>Hprt</i> as the normalization control. Bars represent mean+1 SEM, n = 3 individual experiments. C- Expression of <i>Axin2</i>, a target of β-catenin in XX germ cells. X-Gal staining (AXIN2) (blue) and MVH immunostaining (germ cells, red) in XY <i>Axin2<sup>+/LacZ</sup></i>, XX <i>Axin2<sup>+/LacZ</sup></i> and XX <i>Axin2<sup>+/LacZ</sup></i>; <i>Rspo1<sup>−/−</sup></i> gonads at E12.5. Arrowheads: germ cells. G: gonad. X-Gal staining (AXIN2) (blue) and MVH immunostaining (germ cells, green) on isolated germ cells from XY <i>Axin2<sup>+/LacZ</sup></i> and XX <i>Axin2<sup>+/LacZ</sup></i> gonads at E12.5. Arrowheads: germ cells, arrows: somatic cells.</div></p></div><div class="aux"><div class="resc">Anne-Amandine Chassot, et al. PLoS One. 2011;6(10):e25641.</div><p class="links"><a class="dblinks" href="/pubmed/21991325" ref="ordinalpos=499&amp;ncbi_uid=3275046&amp;link_uid=21991325">Citation</a><a class="dblinks" href="/pmc/articles/PMC3185015/" ref="ordinalpos=499&amp;ncbi_uid=3275046&amp;link_uid=PMC3185015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>500.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2787275/figure/F3/" ref="ordinalpos=500&amp;ncbi_uid=2204931&amp;link_uid=PMC2787275" href="/pmc/articles/PMC2787275/"><img src="/pmc/articles/instance/2787275/bin/ar2791-3.gif" src-large="/pmc/articles/instance/2787275/bin/ar2791-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2787275/" ref="ordinalpos=500&amp;ncbi_uid=2204931&amp;link_uid=PMC2787275" image-link="/pmc/articles/PMC2787275/figure/F3/" class="imagepopup">Figure 3.  From: Osteoarthritis associated with estrogen deficiency. </a></p><div class="supp"><p class="details"><div><b>Intracellular <span class="highlight" style="background-color:">signaling</span> pathways used to regulate the activity of estrogens, estrogen receptors, and selective estrogen receptor modulators on articular tissues</b>. <span class="highlight" style="background-color:">Pathway</span> 1: canonical estrogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (estrogen response element (ERE)-dependent) - ligand-activated estrogen receptors (ERs) bind specifically to EREs in the promoter of target genes. <span class="highlight" style="background-color:">Pathway</span> 2: non-ERE estrogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> - ligand-bound ERs interact with other transcription factors, such as activator protein (AP)-1, NF-κB and Sp1, forming complexes that mediate the transcription of genes whose promoters do not harbor EREs. Co-regulator molecules regulate the activity of the transcriptional complexes. <span class="highlight" style="background-color:">Pathway</span> 3: non-genomic estrogen <span class="highlight" style="background-color:">signaling</span> pathways - ERs and GP30 localized at or near the cell membrane might elicit the rapid response by activating the phosphatidylinositol-3/Akt (PI3K/Akt) and/or protein kinase C/mitogen activated protein kinase (PKC/MAPK) signal transduction pathways. <span class="highlight" style="background-color:">Pathway</span> 4: ligand-independent pathways - ERs can be stimulated by growth factors such as insulin-like growth factor (IGF)-1, transforming growth factor-β/mothers against decapentaplegic (TGF-β/SMAD), epidermal growth factor (EGF) and the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the absence of ligands, either by direct interaction or by MAP and PI3/Akt kinase-mediated phosphorylation. Since members of these <span class="highlight" style="background-color:">signaling</span> pathways are transcription factors, some of them, such as SMADs 3/4, can elicit estrogen responses by interacting with ER in the non-ERE-dependent genomic <span class="highlight" style="background-color:">pathway</span>. ERK, extracellular signal regulated kinase; GF, growth factor; GFR, growth factor receptor; MNAR, Modulator of nongenomic action of estrogen receptors; TF, transcription factor.</div></p></div><div class="aux"><div class="resc">Jorge A Roman-Blas, et al. Arthritis Res Ther. 2009;11(5):241-241.</div><p class="links"><a class="dblinks" href="/pubmed/19804619" ref="ordinalpos=500&amp;ncbi_uid=2204931&amp;link_uid=19804619">Citation</a><a class="dblinks" href="/pmc/articles/PMC2787275/" ref="ordinalpos=500&amp;ncbi_uid=2204931&amp;link_uid=PMC2787275">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="4" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="5" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="6" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T14:41:36-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB244753FD3EB10000000000EEC8FD;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB244753FD3EB10000000000EEC8FD" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>