
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="29" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC4769543100010000000000EB401B" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="28" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="29" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="30" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="29" /></div>
            <div><h2 class="result_count">Results: 2801 to 2900 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2801.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4014232/figure/fig02/" ref="ordinalpos=2801&amp;ncbi_uid=6114980&amp;link_uid=PMC4014232" href="/pmc/articles/PMC4014232/"><img src="/pmc/articles/instance/4014232/bin/ijcep0007-1524-f2.gif" src-large="/pmc/articles/instance/4014232/bin/ijcep0007-1524-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4014232/" ref="ordinalpos=2801&amp;ncbi_uid=6114980&amp;link_uid=PMC4014232" image-link="/pmc/articles/PMC4014232/figure/fig02/" class="imagepopup">Figure 2.  From: RNA-seq reveals determinants of sensitivity to chemotherapy drugs in esophageal carcinoma cells. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis (IPA) for those differentially expressed genes between Eca-109 and Eca-1 cells. A: The most significant canonical pathways in which the differentially expressed genes were enriched. The 348 differentially expressed genes (expression difference &gt;4-fold) were applied to Ingenuity <span class="highlight" style="background-color:">Pathway</span> analysis (IPA) software, and the most significant canonical pathways were shown. B: Gαi <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was predicted to be activated in multidrug-resistant Eca-1 cells.</div></p></div><div class="aux"><div class="resc">Li-Xin Yang, et al. Int J Clin Exp Pathol. 2014;7(4):1524-1533.</div><p class="links"><a class="dblinks" href="/pubmed/24817948" ref="ordinalpos=2801&amp;ncbi_uid=6114980&amp;link_uid=24817948">Citation</a><a class="dblinks" href="/pmc/articles/PMC4014232/" ref="ordinalpos=2801&amp;ncbi_uid=6114980&amp;link_uid=PMC4014232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2802.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4248487/figure/F7/" ref="ordinalpos=2802&amp;ncbi_uid=6913825&amp;link_uid=PMC4248487" href="/pmc/articles/PMC4248487/"><img src="/pmc/articles/instance/4248487/bin/IJD-59-531-g007.gif" src-large="/pmc/articles/instance/4248487/bin/IJD-59-531-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4248487/" ref="ordinalpos=2802&amp;ncbi_uid=6913825&amp;link_uid=PMC4248487" image-link="/pmc/articles/PMC4248487/figure/F7/" class="imagepopup">Figure 7.  From: Novel Medical Strategies Combating Nonmelanoma Skin Cancer. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and tumorigenesis</div></p></div><div class="aux"><div class="resc">Prasan R Bhandari, et al. Indian J Dermatol. 2014 Nov-Dec;59(6):531-546.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2802&amp;ncbi_uid=6913825&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4248487/" ref="ordinalpos=2802&amp;ncbi_uid=6913825&amp;link_uid=PMC4248487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2803.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3640078/figure/pone-0061901-g007/" ref="ordinalpos=2803&amp;ncbi_uid=4863253&amp;link_uid=PMC3640078" href="/pmc/articles/PMC3640078/"><img src="/pmc/articles/instance/3640078/bin/pone.0061901.g007.gif" src-large="/pmc/articles/instance/3640078/bin/pone.0061901.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3640078/" ref="ordinalpos=2803&amp;ncbi_uid=4863253&amp;link_uid=PMC3640078" image-link="/pmc/articles/PMC3640078/figure/pone-0061901-g007/" class="imagepopup">Figure 7. IL-6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in bladder cancer..  From: IL-6 Expression Regulates Tumorigenicity and Correlates with Prognosis in Bladder Cancer. </a></p><div class="aux"><div class="resc">Miao-Fen Chen, et al. PLoS One. 2013;8(4):e61901.</div><p class="links"><a class="dblinks" href="/pubmed/23637926" ref="ordinalpos=2803&amp;ncbi_uid=4863253&amp;link_uid=23637926">Citation</a><a class="dblinks" href="/pmc/articles/PMC3640078/" ref="ordinalpos=2803&amp;ncbi_uid=4863253&amp;link_uid=PMC3640078">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2804.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3676754/figure/f2-ijms-14-08740/" ref="ordinalpos=2804&amp;ncbi_uid=5061690&amp;link_uid=PMC3676754" href="/pmc/articles/PMC3676754/"><img src="/pmc/articles/instance/3676754/bin/ijms-14-08740f2a.gif" src-large="/pmc/articles/instance/3676754/bin/ijms-14-08740f2a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3676754/figure/f2-ijms-14-08740/" ref="ordinalpos=2804&amp;ncbi_uid=5061690&amp;link_uid=PMC3676754" href="/pmc/articles/PMC3676754/"><img src="/pmc/articles/instance/3676754/bin/ijms-14-08740f2b.gif" src-large="/pmc/articles/instance/3676754/bin/ijms-14-08740f2b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676754/" ref="ordinalpos=2804&amp;ncbi_uid=5061690&amp;link_uid=PMC3676754" image-link="/pmc/articles/PMC3676754/figure/f2-ijms-14-08740/" class="imagepopup">Figure 2.  From: The Brassinosteroid <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>--New Key Players and Interconnections with Other <span class="highlight" style="background-color:">Signaling</span> Networks Crucial for Plant Development and Stress Tolerance. </a></p><div class="supp"><p class="details"><div>Model of BR perception and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>a</b>) In the absence of BR, formation of the receptor complex is inhibited by the C-terminal domain (CTD) of BRI1 kinase and by the BKI1 protein. Positive regulators of BR <span class="highlight" style="background-color:">signaling</span> are inactive, whereas the cytoplasmic BIN2 kinase phosphorylates and inactivates transcription factors, what results in their cytoplasmic retention, export for nucleus and degradation. Expression of the BR-responsive genes is repressed; (<b>b</b>) After BR perception by the BRI1 kinase the transmembrane receptor complex is formed and phosphorylation/dephosphorylation cascade is initiated. BIN2 kinase is inhibited, what results in accumulation of active, dephosphorylated forms of transcription factors in the nucleus and stimulation of the expression of BR-responsive genes. The green arrows denote activation, whereas the red arrows denote inhibition; ‘P’ denotes phosphorylation/dephosphorylation, whereas ‘Me’ stands for methylation.</div></p></div><div class="aux"><div class="resc">Damian Gruszka. Int J Mol Sci. 2013 May;14(5):8740-8774.</div><p class="links"><a class="dblinks" href="/pubmed/23615468" ref="ordinalpos=2804&amp;ncbi_uid=5061690&amp;link_uid=23615468">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676754/" ref="ordinalpos=2804&amp;ncbi_uid=5061690&amp;link_uid=PMC3676754">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2805.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g003/" ref="ordinalpos=2805&amp;ncbi_uid=4989294&amp;link_uid=PMC3677896" href="/pmc/articles/PMC3677896/"><img src="/pmc/articles/instance/3677896/bin/pone.0065032.g003.gif" src-large="/pmc/articles/instance/3677896/bin/pone.0065032.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3677896/" ref="ordinalpos=2805&amp;ncbi_uid=4989294&amp;link_uid=PMC3677896" image-link="/pmc/articles/PMC3677896/figure/pone-0065032-g003/" class="imagepopup">Figure 3. Effects of SHH <span class="highlight" style="background-color:">signaling</span> antagonists on dying cell induced tumor cell repopulation..  From: Sonic Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Supports Cancer Cell Growth during Cancer Radiotherapy. </a></p><div class="supp"><p class="details"><div>A. GDC0449, a preclinical SHH <span class="highlight" style="background-color:">pathway</span> inhibitor, inhibits Panc1Fluc cell growth induced by dying Panc1 cell in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. *represents <i>P</i>&lt;0.05. B. Gant61 inhibits Panc1Fluc cells growth induced by dying Panc1 cells in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. **represents <i>P</i>&lt;0.01. C. Cyclopamin inhibits HT29Fluc cell growth induced by dying HT29 cells in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. *represents <i>P</i>&lt;0.05, **represents <i>P</i>&lt;0.01. D. Gant61 inhibits HT29Fluc cell growth induced by dying HT29 cells in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm. **represents <i>P</i>&lt;0.01. E. GDC0449 inhibits HT29 Fluc cell growth induced by dying HT29 cells in a dose-dependent manner. Top: signal intensity analysis from bioluminescence image; Bottom: representative bioluminescence images; scale bar represents 1 cm.</div></p></div><div class="aux"><div class="resc">Jingjing Ma, et al. PLoS One. 2013;8(6):e65032.</div><p class="links"><a class="dblinks" href="/pubmed/23762282" ref="ordinalpos=2805&amp;ncbi_uid=4989294&amp;link_uid=23762282">Citation</a><a class="dblinks" href="/pmc/articles/PMC3677896/" ref="ordinalpos=2805&amp;ncbi_uid=4989294&amp;link_uid=PMC3677896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2806.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3595309/figure/F2/" ref="ordinalpos=2806&amp;ncbi_uid=5868713&amp;link_uid=PMC3595309" href="/pmc/articles/PMC3595309/"><img src="/pmc/articles/instance/3595309/bin/cbt-14-262-g2.gif" src-large="/pmc/articles/instance/3595309/bin/cbt-14-262-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3595309/" ref="ordinalpos=2806&amp;ncbi_uid=5868713&amp;link_uid=PMC3595309" image-link="/pmc/articles/PMC3595309/figure/F2/" class="imagepopup">Figure. From: Cadherin-17 induces tumorigenesis and lymphatic metastasis in gastric cancer through activation of NF?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Inhibition of CDH17 repressed the malignant potency of MKN-45 cells and alleviated the activation of NFκB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(A)</b> Immunofluorescence staining demonstrated that CDH17 was significantly downregulated in miR-CDH17 cells, compared with the high expression in both MKN-45 and miR-neg cells (100μm). <b>(B)</b> Proliferation assay.<b> (C) </b>Adhesion assay. <b>(D)</b> Cell cycle analysis. <b>(E)</b> Invasion assay (100μm). <b>(F)</b> In western blot analysis, there was less expression of CDH17, nuclear p65, MMP-9 and VEGF-C, but more expression of IκB-α in miR-CDH17 group. There was no difference in total p65 among each group.</div></p></div><div class="aux"><div class="resc">Jin Wang, et al. Cancer Biol Ther. 2013 March 1;14(3):262-270.</div><p class="links"><a class="dblinks" href="/pubmed/23298905" ref="ordinalpos=2806&amp;ncbi_uid=5868713&amp;link_uid=23298905">Citation</a><a class="dblinks" href="/pmc/articles/PMC3595309/" ref="ordinalpos=2806&amp;ncbi_uid=5868713&amp;link_uid=PMC3595309">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2807.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2913378/figure/F3-8733/" ref="ordinalpos=2807&amp;ncbi_uid=3124850&amp;link_uid=PMC2913378" href="/pmc/articles/PMC2913378/"><img src="/pmc/articles/instance/2913378/bin/zjh0081087330003.gif" src-large="/pmc/articles/instance/2913378/bin/zjh0081087330003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2913378/" ref="ordinalpos=2807&amp;ncbi_uid=3124850&amp;link_uid=PMC2913378" image-link="/pmc/articles/PMC2913378/figure/F3-8733/" class="imagepopup">Figure 3.  From: Cytoskeletal Regulation of Epithelial Barrier Function During Inflammation. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways that mediate actomyosin-dependent disruption of the epithelial barrier in inflammation. Two <span class="highlight" style="background-color:">signaling</span> pathways that are activated by different inflammatory stimuli and result in disruption of the epithelial barrier are presented. The first <span class="highlight" style="background-color:">pathway</span> involves activation of MLCK resulting in a modest contraction of perijunctional actomyosin belt and reversible increase in paracellular permeability without gross alterations in AJC structure. The second <span class="highlight" style="background-color:">pathway</span> involves activation of the GEF-H1-Rho-ROCK <span class="highlight" style="background-color:">pathway</span> that leads to profound actomyosin contraction and AJC disassembly.</div></p></div><div class="aux"><div class="resc">Andrei I. Ivanov, et al. Am J Pathol. 2010 August;177(2):512-524.</div><p class="links"><a class="dblinks" href="/pubmed/20581053" ref="ordinalpos=2807&amp;ncbi_uid=3124850&amp;link_uid=20581053">Citation</a><a class="dblinks" href="/pmc/articles/PMC2913378/" ref="ordinalpos=2807&amp;ncbi_uid=3124850&amp;link_uid=PMC2913378">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2808.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4323367/figure/F7/" ref="ordinalpos=2808&amp;ncbi_uid=7169834&amp;link_uid=PMC4323367" href="/pmc/articles/PMC4323367/"><img src="/pmc/articles/instance/4323367/bin/ijbsv11p0274g007.gif" src-large="/pmc/articles/instance/4323367/bin/ijbsv11p0274g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4323367/" ref="ordinalpos=2808&amp;ncbi_uid=7169834&amp;link_uid=PMC4323367" image-link="/pmc/articles/PMC4323367/figure/F7/" class="imagepopup">Figure 7.  From: Protracted Upregulation of Leptin and IGF1 is Associated with Activation of PI3K/Akt and JAK2 <span class="highlight" style="background-color:">Pathway</span> in Mouse Intestine after Ionizing Radiation Exposure. </a></p><div class="supp"><p class="details"><div>Summary of radiation-induced persistent alterations of IGF1 and leptin <span class="highlight" style="background-color:">signaling</span> pathways observed in our study. Radiation exposure was associated with decreased IGFBP3, increased IGF1 and IGF1R, and activation of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> via IRS1. Radiation exposure also decreased adiponectin and Adipo-R, increased leptin and Ob-R, and activated JAK2 <span class="highlight" style="background-color:">pathway</span>. Adiponectin antagonizes IGF1 and leptin, and decreased adiponectin and its receptors are expected to further promote IGF1 and leptin <span class="highlight" style="background-color:">signaling</span> through reduced antagonism. There is also existence of crosstalk between JAK2 and PI3K/Akt via IRS1 and combined activation of these two pathways is contributing to intestinal cell proliferation with potential for intestinal disorder and colon cancer.</div></p></div><div class="aux"><div class="resc">Shubhankar Suman, et al. Int J Biol Sci. 2015;11(3):274-283.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2808&amp;ncbi_uid=7169834&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4323367/" ref="ordinalpos=2808&amp;ncbi_uid=7169834&amp;link_uid=PMC4323367">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2809.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3221743/figure/F5/" ref="ordinalpos=2809&amp;ncbi_uid=3670684&amp;link_uid=PMC3221743" href="/pmc/articles/PMC3221743/"><img src="/pmc/articles/instance/3221743/bin/nihms328658f5.gif" src-large="/pmc/articles/instance/3221743/bin/nihms328658f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3221743/" ref="ordinalpos=2809&amp;ncbi_uid=3670684&amp;link_uid=PMC3221743" image-link="/pmc/articles/PMC3221743/figure/F5/" class="imagepopup">Figure 5.  From: mTOR inhibition in macrophages of asymptomatic HIV+ persons reverses the decrease in TLR4-mediated TNF? release through prolongation of MAPK <span class="highlight" style="background-color:">pathway</span> activation. </a></p><div class="supp"><p class="details"><div>Working model describing how in HIV+ macrophages constitutively activated PI3K elicits blunting of lipid A-induced increases in TNFα release through sustained mTOR activation. PI3K-Akt-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is constitutively active in HIV+ macrophages and inhibits the expression of TNFα by 1) direct mTOR-dependent inhibition of MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. 2) IL-10 production through the activation of mTOR. 3) destabilization a pan specific MAP kinase phosphatase,(MKP-1) expression through declines in its phosphorylation status, Inhibition of MKP-1 elicited negative feedback on ERK1/2 and JNK1/2 phosphorylation prolongs ERK1/2 and JNK1/2 activation by declines in its dephosphorylation.</div></p></div><div class="aux"><div class="resc">Xin Li, et al. J Immunol. ;187(11):6052-6058.</div><p class="links"><a class="dblinks" href="/pubmed/22025552" ref="ordinalpos=2809&amp;ncbi_uid=3670684&amp;link_uid=22025552">Citation</a><a class="dblinks" href="/pmc/articles/PMC3221743/" ref="ordinalpos=2809&amp;ncbi_uid=3670684&amp;link_uid=PMC3221743">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2810.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3125314/figure/F3/" ref="ordinalpos=2810&amp;ncbi_uid=3044567&amp;link_uid=PMC3125314" href="/pmc/articles/PMC3125314/"><img src="/pmc/articles/instance/3125314/bin/1752-0509-5-74-3.gif" src-large="/pmc/articles/instance/3125314/bin/1752-0509-5-74-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3125314/" ref="ordinalpos=2810&amp;ncbi_uid=3044567&amp;link_uid=PMC3125314" image-link="/pmc/articles/PMC3125314/figure/F3/" class="imagepopup">Figure 3.  From: Structurally robust biological networks. </a></p><div class="supp"><p class="details"><div><b>Graphs associated with case studies</b>. A. The graph associated with the L-arabinose network, external inputs are represented as orange nodes. B. The graph associated with the cAMP <span class="highlight" style="background-color:">pathway</span>. C. The graph associated with the lac Operon network. D. The graph associated with the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Franco Blanchini, et al. BMC Syst Biol. 2011;5:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/21586168" ref="ordinalpos=2810&amp;ncbi_uid=3044567&amp;link_uid=21586168">Citation</a><a class="dblinks" href="/pmc/articles/PMC3125314/" ref="ordinalpos=2810&amp;ncbi_uid=3044567&amp;link_uid=PMC3125314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2811.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3101847/figure/A004440F1/" ref="ordinalpos=2811&amp;ncbi_uid=4838517&amp;link_uid=PMC3101847" href="/pmc/articles/PMC3101847/"><img src="/pmc/articles/instance/3101847/bin/cshperspect-PRT-004440_F1.gif" src-large="/pmc/articles/instance/3101847/bin/cshperspect-PRT-004440_F1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3101847/" ref="ordinalpos=2811&amp;ncbi_uid=4838517&amp;link_uid=PMC3101847" image-link="/pmc/articles/PMC3101847/figure/A004440F1/" class="imagepopup">Figure 1. From: Aging as an Event of Proteostasis Collapse. </a></p><div class="supp"><p class="details"><div>Pathways that regulate aging. The insulin-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and dietary restriction <span class="highlight" style="background-color:">pathway</span> induce longevity through mechanisms that are at least partially understood. Reduced mitochondrial function also increases lifespan, but the <span class="highlight" style="background-color:">signaling</span> components are not clear.</div></p></div><div class="aux"><div class="resc">Rebecca C. Taylor, et al. Cold Spring Harb Perspect Biol. 2011 May;3(5):a004440.</div><p class="links"><a class="dblinks" href="/pubmed/21441594" ref="ordinalpos=2811&amp;ncbi_uid=4838517&amp;link_uid=21441594">Citation</a><a class="dblinks" href="/pmc/articles/PMC3101847/" ref="ordinalpos=2811&amp;ncbi_uid=4838517&amp;link_uid=PMC3101847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2812.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405869/figure/A008078F4/" ref="ordinalpos=2812&amp;ncbi_uid=6431531&amp;link_uid=PMC3405869" href="/pmc/articles/PMC3405869/"><img src="/pmc/articles/instance/3405869/bin/cshperspect-WNT-008078_F4.gif" src-large="/pmc/articles/instance/3405869/bin/cshperspect-WNT-008078_F4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405869/" ref="ordinalpos=2812&amp;ncbi_uid=6431531&amp;link_uid=PMC3405869" image-link="/pmc/articles/PMC3405869/figure/A008078F4/" class="imagepopup">Figure 4. From: Wnt <span class="highlight" style="background-color:">Signaling</span> and Injury Repair. </a></p><div class="supp"><p class="details"><div>Does chronic injury lead to prolonged Wnt <span class="highlight" style="background-color:">signaling</span> and cancer? Injury results in a local increase in Wnt <span class="highlight" style="background-color:">signaling</span> and chronic injury leads to cancer. Many cancers also have up-regulated Wnt <span class="highlight" style="background-color:">signaling</span>. The Wnt <span class="highlight" style="background-color:">pathway</span> is best known to promote proliferation; an intriguing possibility is that chronic injury may induce a prolonged Wnt response leading to uncontrolled cell proliferation and cancer. In the lungs, cigarette smokers inflict chronic injury to the trachea and lung tissue, which may result in lung cancer. Relating to the skin, prolonged UV exposure leads to chronic inflammation of the dermis, which may result in skin cancer. In the liver, chronic alcohol consumption subjects the tissue to constant damage, which may lead to liver cancer.</div></p></div><div class="aux"><div class="resc">Jemima L. Whyte, et al. Cold Spring Harb Perspect Biol. 2012 August;4(8):a008078.</div><p class="links"><a class="dblinks" href="/pubmed/22723493" ref="ordinalpos=2812&amp;ncbi_uid=6431531&amp;link_uid=22723493">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405869/" ref="ordinalpos=2812&amp;ncbi_uid=6431531&amp;link_uid=PMC3405869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2813.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3490586/figure/fig10/" ref="ordinalpos=2813&amp;ncbi_uid=4352263&amp;link_uid=PMC3490586" href="/pmc/articles/PMC3490586/"><img src="/pmc/articles/instance/3490586/bin/PP_204503_lw_f10.gif" src-large="/pmc/articles/instance/3490586/bin/PP_204503_lw_f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3490586/" ref="ordinalpos=2813&amp;ncbi_uid=4352263&amp;link_uid=PMC3490586" image-link="/pmc/articles/PMC3490586/figure/fig10/" class="imagepopup">Figure 10. From: Diverse Roles of Strigolactone <span class="highlight" style="background-color:">Signaling</span> in Maize Architecture and the Uncoupling of a Branching-Specific Subnetwork1[C][W][OA]. </a></p><div class="supp"><p class="details"><div>Models for SL regulation of axillary bud outgrowth in eudicots, rice, and maize. In eudicots, SL promotes expression of <i>BRC1</i> and acts strictly through <i>BRC1</i> to suppress axillary bud outgrowth. In rice, SL <span class="highlight" style="background-color:">signaling</span> acts partially through <i>FC1</i> to inhibit outgrowth of axial buds. In maize, SL acts primarily on a downstream component of the <i>Tb1</i> <span class="highlight" style="background-color:">pathway</span> to regulate bud outgrowth. The open arrows indicate a major effect on the suppression of bud outgrowth.</div></p></div><div class="aux"><div class="resc">Jiahn Chou Guan, et al. Plant Physiol. 2012 November;160(3):1303-1317.</div><p class="links"><a class="dblinks" href="/pubmed/22961131" ref="ordinalpos=2813&amp;ncbi_uid=4352263&amp;link_uid=22961131">Citation</a><a class="dblinks" href="/pmc/articles/PMC3490586/" ref="ordinalpos=2813&amp;ncbi_uid=4352263&amp;link_uid=PMC3490586">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2814.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3104484/figure/F5/" ref="ordinalpos=2814&amp;ncbi_uid=2996061&amp;link_uid=PMC3104484" href="/pmc/articles/PMC3104484/"><img src="/pmc/articles/instance/3104484/bin/1749-8104-6-23-5.gif" src-large="/pmc/articles/instance/3104484/bin/1749-8104-6-23-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3104484/" ref="ordinalpos=2814&amp;ncbi_uid=2996061&amp;link_uid=PMC3104484" image-link="/pmc/articles/PMC3104484/figure/F5/" class="imagepopup">Figure 5.  From: En1 and Wnt <span class="highlight" style="background-color:">signaling</span> in midbrain dopaminergic neuronal development. </a></p><div class="supp"><p class="details"><div><b>Interplay between Wnt, Nurr1 and En1 <span class="highlight" style="background-color:">signaling</span> <i>in vitro </i>and <i>in vivo</i></b>. <b>(A) </b>Model adapted from Kitagawa <i>et al. </i>[156]: Wnt <span class="highlight" style="background-color:">signaling</span> via β-catenin enhances the transcriptional activity of Nurr1 in cells at Nurr1 responsive elements (NREs). In the absence of β-catenin, Nurr1 associates with T cell factor/lymphoid enhancer factor (TCF/LEF) in co-repressor complexes on NREs. After activation of Wnt <span class="highlight" style="background-color:">signaling</span>, β-catenin interacts with Nurr1 on NREs, competing with TCF/LEF for Nurr1 binding, resulting in the disruption of the co-repressors from the Nurr1 complex and the concomitant recruitment of coactivators. <b>(B) </b>Model adapted from Kitagawa <i>et al. </i>[156]: on the other hand, Nurr1 was observed to slightly modulate, in a negative way, the canonical Wnt <span class="highlight" style="background-color:">signaling</span> through association with the TCF/LEF region. After Wnt stimulation, β-catenin competed with Nurr1 for Lef binding on the TCF/LEF promoter site and disrupted Nurr1 binding, promoting Wnt-target gene transcription. <b>(C) </b>Several studies in <i>Drosophila </i>and chick embryos have described interactions between En1/engrailed (en) and the Wnt/wg <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> whereby engrailed expression is dependent on Wnt/wg <span class="highlight" style="background-color:">signaling</span> and <i>vice versa</i>. However, in <i>Drosophila</i>, engrailed expressing cells did not have active wg <span class="highlight" style="background-color:">signaling</span>. From mice studies it is known that Wnt <span class="highlight" style="background-color:">signaling</span> regulates En1 expression early in midbrain development. Whether the reverse happens in the mouse midbrain is not known. <b>(D) </b>In one cell culture study [216], it was observed that En1 can function as a negative regulator of β-catenin transcriptional activity in a post-translational manner (that is, by affecting β-catenin protein levels only). <b>(E) </b>Three questions remain currently unsolved: first, whether En1 cooperates with Nurr1 during mdDA development; second, whether Nurr1, En1 and canonical Wnt <span class="highlight" style="background-color:">signaling</span> cooperate in later stages of mdDA neuron development, such as in mdDA neuron specification; and third, whether Nurr1 and/or En1 regulate canonical Wnt <span class="highlight" style="background-color:">signaling</span> during mdDA neuron development.</div></p></div><div class="aux"><div class="resc">Maria TM Alves dos Santos, et al. Neural Dev. 2011;6:23-23.</div><p class="links"><a class="dblinks" href="/pubmed/21569278" ref="ordinalpos=2814&amp;ncbi_uid=2996061&amp;link_uid=21569278">Citation</a><a class="dblinks" href="/pmc/articles/PMC3104484/" ref="ordinalpos=2814&amp;ncbi_uid=2996061&amp;link_uid=PMC3104484">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2815.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2953617/figure/F2/" ref="ordinalpos=2815&amp;ncbi_uid=2514066&amp;link_uid=PMC2953617" href="/pmc/articles/PMC2953617/"><img src="/pmc/articles/instance/2953617/bin/nihms237584f2.gif" src-large="/pmc/articles/instance/2953617/bin/nihms237584f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2953617/" ref="ordinalpos=2815&amp;ncbi_uid=2514066&amp;link_uid=PMC2953617" image-link="/pmc/articles/PMC2953617/figure/F2/" class="imagepopup">Fig. 2.  From: The role of primary cilia in neuronal function. </a></p><div class="supp"><p class="details"><div>JS genes implicated in the Shh, Wnt, and PDGFα <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ARL13B and INPP5E localize to the cilium, whereas AHI1 and CEP290 localize to the basal body. CEP290, a centrosomal protein, is involved in Rab8 contained vesicle transport and ciliogenesis through its interaction with PCM-1. The disruption of ARL13B causes defects in ciliary structure and Shh <span class="highlight" style="background-color:">signaling</span>. ARL13B is required for body axis formation, renal function, and neural tube patterning. INPP5E plays an essential role in stabilizing cilia in response to serum stimulation, possibly by controlling PDGFα <span class="highlight" style="background-color:">signaling</span>. AHI1 interacts with and facilitates β-catenin (β-Cat) accumulation in the nucleus to positively modulate Wnt <span class="highlight" style="background-color:">signaling</span>. Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> via AHI1 is essential for maintaining renal homeostasis.</div></p></div><div class="aux"><div class="resc">Jeong Ho Lee, et al. Neurobiol Dis. ;38(2):167-172.</div><p class="links"><a class="dblinks" href="/pubmed/20097287" ref="ordinalpos=2815&amp;ncbi_uid=2514066&amp;link_uid=20097287">Citation</a><a class="dblinks" href="/pmc/articles/PMC2953617/" ref="ordinalpos=2815&amp;ncbi_uid=2514066&amp;link_uid=PMC2953617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2816.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3651907/figure/F2/" ref="ordinalpos=2816&amp;ncbi_uid=4894641&amp;link_uid=PMC3651907" href="/pmc/articles/PMC3651907/"><img src="/pmc/articles/instance/3651907/bin/nihms-398895-f0002.gif" src-large="/pmc/articles/instance/3651907/bin/nihms-398895-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3651907/" ref="ordinalpos=2816&amp;ncbi_uid=4894641&amp;link_uid=PMC3651907" image-link="/pmc/articles/PMC3651907/figure/F2/" class="imagepopup">Figure 2.  From: Learning from Jekyll to control Hyde: Hedgehog <span class="highlight" style="background-color:">Signaling</span> in Development and Cancer. </a></p><div class="supp"><p class="details"><div>This simplified view of the Hh <span class="highlight" style="background-color:">pathway</span> depicts only those core components most commonly mutated in cancer. The basic <span class="highlight" style="background-color:">signaling</span> cascade consists of a series of repressive interactions. Normally, in the absence of Shh, Gli proteins and target genes are actively repressed. When secreted Shh binds to its receptor Patched (Ptc/PTCH), the inhibition of Smoothened (Smo/SMOH), a G protein-coupled receptor that activates downstream intracellular components of the <span class="highlight" style="background-color:">pathway</span>, by Ptc is relieved.</div></p></div><div class="aux"><div class="resc">Monique T. Barakat, et al. Trends Mol Med. 2010 August;16(8):337-348.</div><p class="links"><a class="dblinks" href="/pubmed/20696410" ref="ordinalpos=2816&amp;ncbi_uid=4894641&amp;link_uid=20696410">Citation</a><a class="dblinks" href="/pmc/articles/PMC3651907/" ref="ordinalpos=2816&amp;ncbi_uid=4894641&amp;link_uid=PMC3651907">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2817.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2758297/figure/F3/" ref="ordinalpos=2817&amp;ncbi_uid=2478882&amp;link_uid=PMC2758297" href="/pmc/articles/PMC2758297/"><img src="/pmc/articles/instance/2758297/bin/nihms-133041-f0003.gif" src-large="/pmc/articles/instance/2758297/bin/nihms-133041-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758297/" ref="ordinalpos=2817&amp;ncbi_uid=2478882&amp;link_uid=PMC2758297" image-link="/pmc/articles/PMC2758297/figure/F3/" class="imagepopup">Figure 3. Genes associated with Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are regulated by LRAP.  From: LEUCINE-RICH AMELOGENIN PEPTIDE INDUCES OSTEOGENESIS BY ACTIVATION OF THE WNT <span class="highlight" style="background-color:">PATHWAY</span>. </a></p><div class="supp"><p class="details"><div>Clustergrams illustrating the magnitude of gene expression in LRAP-treated EBs at 1 hour (Min+L1h), comparing to EBs cultured in mineralization medium (Min 1h) and EBs cultured in the control medium (Control 1h); and LRAP-treated EBs for 4 hours (Min+L4h), comparing to EBs cultured in mineralization medium (Min 4h) and EBs cultured in the control medium (Control 4h).</div></p></div><div class="aux"><div class="resc">Rungnapa Warotayanont, et al. Biochem Biophys Res Commun. ;387(3):558-563.</div><p class="links"><a class="dblinks" href="/pubmed/19615979" ref="ordinalpos=2817&amp;ncbi_uid=2478882&amp;link_uid=19615979">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758297/" ref="ordinalpos=2817&amp;ncbi_uid=2478882&amp;link_uid=PMC2758297">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2818.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3856963/figure/f7-0981877/" ref="ordinalpos=2818&amp;ncbi_uid=5611027&amp;link_uid=PMC3856963" href="/pmc/articles/PMC3856963/"><img src="/pmc/articles/instance/3856963/bin/981877.fig7.gif" src-large="/pmc/articles/instance/3856963/bin/981877.fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3856963/" ref="ordinalpos=2818&amp;ncbi_uid=5611027&amp;link_uid=PMC3856963" image-link="/pmc/articles/PMC3856963/figure/f7-0981877/" class="imagepopup">Figure 7. From: Cytoplasmic localization of wild-type survivin is associated with constitutive activation of the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and represents a favorable prognostic factor in patients with acute myeloid leukemia. </a></p><div class="supp"><p class="details"><div>Outcomes of AML patients according to the PI3K/Akt/CytSurWT activation <span class="highlight" style="background-color:">pathway</span>. Overall survival was calculated with Kaplan-Meier curves for all 66 intensively treated patients and relapse-free survival was calculated for intensively treated patients achieving complete remission. (A) Kaplan-Meier curves for overall survival according to PI3K/Akt/CytSurWT activation <span class="highlight" style="background-color:">pathway</span> status. (B) Kaplan-Meier curves for relapse-free survival according to PI3K/Akt/CytSurWT activation <span class="highlight" style="background-color:">pathway</span> status.</div></p></div><div class="aux"><div class="resc">Juana Serrano-López, et al. Haematologica. 2013 December;98(12):1877-1885.</div><p class="links"><a class="dblinks" href="/pubmed/23812937" ref="ordinalpos=2818&amp;ncbi_uid=5611027&amp;link_uid=23812937">Citation</a><a class="dblinks" href="/pmc/articles/PMC3856963/" ref="ordinalpos=2818&amp;ncbi_uid=5611027&amp;link_uid=PMC3856963">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2819.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3380732/figure/F5/" ref="ordinalpos=2819&amp;ncbi_uid=4075450&amp;link_uid=PMC3380732" href="/pmc/articles/PMC3380732/"><img src="/pmc/articles/instance/3380732/bin/1477-5956-10-S1-S15-5.gif" src-large="/pmc/articles/instance/3380732/bin/1477-5956-10-S1-S15-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3380732/" ref="ordinalpos=2819&amp;ncbi_uid=4075450&amp;link_uid=PMC3380732" image-link="/pmc/articles/PMC3380732/figure/F5/" class="imagepopup">Figure 5.  From: Identifying transcription factors and microRNAs as key regulators of pathways using Bayesian inference on known <span class="highlight" style="background-color:">pathway</span> structures. </a></p><div class="supp"><p class="details"><div><b>Toy BN of 16 nodes and its error in approximating marginals using Gibbs sampler</b>. (a) Network with 16 nodes extracted from the extended MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The three nodes (in green) are the original nodes from the <span class="highlight" style="background-color:">pathway</span>. The TFs (squares) and microRNAs (triangles) that target them are also included. Due to the small size of the network the marginals were computed precisely. (b) Approximation of the marginal after different sample sizes of the Gibbs sampler. It can be seen that after 4,000 samples the error stays below 0.05.</div></p></div><div class="aux"><div class="resc">Damian Roqueiro, et al. Proteome Sci. 2012;10(Suppl 1):S15-S15.</div><p class="links"><a class="dblinks" href="/pubmed/22759573" ref="ordinalpos=2819&amp;ncbi_uid=4075450&amp;link_uid=22759573">Citation</a><a class="dblinks" href="/pmc/articles/PMC3380732/" ref="ordinalpos=2819&amp;ncbi_uid=4075450&amp;link_uid=PMC3380732">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2820.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3061506/figure/F1/" ref="ordinalpos=2820&amp;ncbi_uid=2927552&amp;link_uid=PMC3061506" href="/pmc/articles/PMC3061506/"><img src="/pmc/articles/instance/3061506/bin/z7g9990908530001.gif" src-large="/pmc/articles/instance/3061506/bin/z7g9990908530001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3061506/" ref="ordinalpos=2820&amp;ncbi_uid=2927552&amp;link_uid=PMC3061506" image-link="/pmc/articles/PMC3061506/figure/F1/" class="imagepopup">Figure 1. From: Glaucomatous Tissue Stress and the Regulation of Immune Response through Glial Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Integration of the identified proteins into the canonical <span class="highlight" style="background-color:">pathway</span> of TLR <span class="highlight" style="background-color:">signaling</span>. Using the bioinformatics analysis tools (Ingenuity Pathways Knowledge Base), the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was defined among the canonical pathways significantly associated with our high-throughput human proteomic data (right-tailed Fisher's exact test; <i>P</i> &lt; 0.05). Proteins shown in <i>blue</i> are those identified by the LC-MS/MS analysis of human retinal proteins, which exhibited an upregulation or were detectable only in glaucomatous samples (detailed information for abbreviated proteins is available at http://www.ncbi.nlm.nih.gov/protein).</div></p></div><div class="aux"><div class="resc">Cheng Luo, et al. Invest Ophthalmol Vis Sci. 2010 November;51(11):5697-5707.</div><p class="links"><a class="dblinks" href="/pubmed/20538986" ref="ordinalpos=2820&amp;ncbi_uid=2927552&amp;link_uid=20538986">Citation</a><a class="dblinks" href="/pmc/articles/PMC3061506/" ref="ordinalpos=2820&amp;ncbi_uid=2927552&amp;link_uid=PMC3061506">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2821.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285038/figure/F1/" ref="ordinalpos=2821&amp;ncbi_uid=3824499&amp;link_uid=PMC3285038" href="/pmc/articles/PMC3285038/"><img src="/pmc/articles/instance/3285038/bin/2045-3701-1-41-1.gif" src-large="/pmc/articles/instance/3285038/bin/2045-3701-1-41-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285038/" ref="ordinalpos=2821&amp;ncbi_uid=3824499&amp;link_uid=PMC3285038" image-link="/pmc/articles/PMC3285038/figure/F1/" class="imagepopup">Figure 1.  From: Roles of TGF? <span class="highlight" style="background-color:">signaling</span> Smads in squamous cell carcinoma. </a></p><div class="supp"><p class="details"><div><b>Schematic of Smads mediated TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. TGFβ ligand binds to TGFβRII/TGFβRI receptors leading to phosphorylation of Smad2/3. Phosphorylated Smad2/3 binds to Smad4 to form a protein complex that undergoes nuclear translocation and regulates the expression of TGFβ target genes through binding to the Smad-binding element (SBE).</div></p></div><div class="aux"><div class="resc">Gangwen Han, et al. Cell Biosci. 2011;1:41-41.</div><p class="links"><a class="dblinks" href="/pubmed/22204491" ref="ordinalpos=2821&amp;ncbi_uid=3824499&amp;link_uid=22204491">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285038/" ref="ordinalpos=2821&amp;ncbi_uid=3824499&amp;link_uid=PMC3285038">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2822.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2826877/figure/fig2/" ref="ordinalpos=2822&amp;ncbi_uid=2281319&amp;link_uid=PMC2826877" href="/pmc/articles/PMC2826877/"><img src="/pmc/articles/instance/2826877/bin/PPAR2009-818945.002.gif" src-large="/pmc/articles/instance/2826877/bin/PPAR2009-818945.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2826877/" ref="ordinalpos=2822&amp;ncbi_uid=2281319&amp;link_uid=PMC2826877" image-link="/pmc/articles/PMC2826877/figure/fig2/" class="imagepopup">Figure 2.  From: Cross-Talk between PPARγ and Insulin <span class="highlight" style="background-color:">Signaling</span> and Modulation of Insulin Sensitivity. </a></p><div class="supp"><p class="details"><div>Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in adipose cells. Binding of insulin to its tyrosine kinase receptor engages a cascade of intracellular phosphorylation events, including activation of phosphatidylinositol-3-kinase and ERK-1/2, that promote multiple biological responses, including glucose uptake, lipid metabolism, survival, differentiation, and modulation of gene transcription.</div></p></div><div class="aux"><div class="resc">Anna Leonardini, et al. PPAR Res. 2009;2009:818945.</div><p class="links"><a class="dblinks" href="/pubmed/20182551" ref="ordinalpos=2822&amp;ncbi_uid=2281319&amp;link_uid=20182551">Citation</a><a class="dblinks" href="/pmc/articles/PMC2826877/" ref="ordinalpos=2822&amp;ncbi_uid=2281319&amp;link_uid=PMC2826877">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2823.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2561291/figure/pone-0003430-g002/" ref="ordinalpos=2823&amp;ncbi_uid=1546538&amp;link_uid=PMC2561291" href="/pmc/articles/PMC2561291/"><img src="/pmc/articles/instance/2561291/bin/pone.0003430.g002.gif" src-large="/pmc/articles/instance/2561291/bin/pone.0003430.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2561291/" ref="ordinalpos=2823&amp;ncbi_uid=1546538&amp;link_uid=PMC2561291" image-link="/pmc/articles/PMC2561291/figure/pone-0003430-g002/" class="imagepopup">Figure 2. In silico simulations of TRAM-dependent <span class="highlight" style="background-color:">pathway</span> molecules upon TLR4 activation..  From: <span class="highlight" style="background-color:">Signaling</span> Flux Redistribution at Toll-Like Receptor <span class="highlight" style="background-color:">Pathway</span> Junctions. </a></p><div class="supp"><p class="details"><div>Simulation profiles (arbitrary units) of (A) TRAM activation, (B) IRF3 activation, and (C) <i>Cxcl10</i> induction in the wildtype (blue), knockout (red, dotted), and two-fold overexpression (green) of MyD88. (D) Schematic of <i>SFR</i>. (Top) Wildtype. Fluxes propagate through both the MyD88-dependent and TRAM-dependent pathways. (Middle) MyD88 KO. More fluxes propagate or <i>overflows</i> through the TRAM-dependent <span class="highlight" style="background-color:">pathway</span> resulting in increased <i>Cxcl10</i> induction. (Bottom) MyD88 overexpression by two-fold.</div></p></div><div class="aux"><div class="resc">Kumar Selvarajoo, et al. PLoS ONE. 2008;3(10):e3430.</div><p class="links"><a class="dblinks" href="/pubmed/18927610" ref="ordinalpos=2823&amp;ncbi_uid=1546538&amp;link_uid=18927610">Citation</a><a class="dblinks" href="/pmc/articles/PMC2561291/" ref="ordinalpos=2823&amp;ncbi_uid=1546538&amp;link_uid=PMC2561291">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2824.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1140591/figure/f7/" ref="ordinalpos=2824&amp;ncbi_uid=485247&amp;link_uid=PMC1140591" href="/pmc/articles/PMC1140591/"><img src="/pmc/articles/instance/1140591/bin/zmb0120550900007.gif" src-large="/pmc/articles/instance/1140591/bin/zmb0120550900007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1140591/" ref="ordinalpos=2824&amp;ncbi_uid=485247&amp;link_uid=PMC1140591" image-link="/pmc/articles/PMC1140591/figure/f7/" class="imagepopup">FIG. 7. From: Mitochondrial Reactive Oxygen Species Activation of p38 Mitogen-Activated Protein Kinase Is Required for Hypoxia <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A <span class="highlight" style="background-color:">signaling</span> model for hypoxic activation of HIF-1. Based on our current findings we propose that hypoxia stimulates oxidant production within mitochondria. These oxidants interact with an unknown protein “X” in the cytosol to activate MKK3/6 and p38α MAPK. The activation of the p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> results in the decrease in hydroxylation of the HIF-1α protein at proline and asparagine residues resulting in the activation of HIF-1. By contrast, anoxia or iron chelators (DFO) during normoxia inhibit hydroxylation directly by limiting the availability of oxygen as a substrate and iron as a cofactor, respectively. Thus, anoxia or DFO does not require <span class="highlight" style="background-color:">signaling</span> pathways for the activation of HIF-1. Hypoxia and anoxia have distinct <span class="highlight" style="background-color:">signaling</span> mechanisms to activation HIF-1.</div></p></div><div class="aux"><div class="resc">Brooke M. Emerling, et al. Mol Cell Biol. 2005 June;25(12):4853-4862.</div><p class="links"><a class="dblinks" href="/pubmed/15923604" ref="ordinalpos=2824&amp;ncbi_uid=485247&amp;link_uid=15923604">Citation</a><a class="dblinks" href="/pmc/articles/PMC1140591/" ref="ordinalpos=2824&amp;ncbi_uid=485247&amp;link_uid=PMC1140591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2825.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3854019/figure/pharmaceuticals-06-01429-f009/" ref="ordinalpos=2825&amp;ncbi_uid=5605104&amp;link_uid=PMC3854019" href="/pmc/articles/PMC3854019/"><img src="/pmc/articles/instance/3854019/bin/pharmaceuticals-06-01429-g009.gif" src-large="/pmc/articles/instance/3854019/bin/pharmaceuticals-06-01429-g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3854019/" ref="ordinalpos=2825&amp;ncbi_uid=5605104&amp;link_uid=PMC3854019" image-link="/pmc/articles/PMC3854019/figure/pharmaceuticals-06-01429-f009/" class="imagepopup">Figure 9.  From: Hedgehog <span class="highlight" style="background-color:">Pathway</span> Blockade Inhibits Melanoma Cell Growth in Vitro and in Vivo. </a></p><div class="supp"><p class="details"><div>Ingenuity Canonical <span class="highlight" style="background-color:">Pathway</span> analysis reveals downregulation of stem cell pluripotency <span class="highlight" style="background-color:">signaling</span> and cell cycle regulators. (<b>A</b>–<b>D</b>) Downregulated IPA (Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis) canonical pathways are shown for A375 and WM3248 cell lines treated with either 5 µM NVP-LDE-225 or SMO siRNA. The transcriptional profiles of the DMSO treated and non-targeting siRNA treated A375 and WM3248 cells were used as controls in the IPA analysis. The stem cell pathways that were downregulated by both SMO siRNA and NVP-LDE-225 in both cell lines are marked with an asterisk (<b>*</b>).</div></p></div><div class="aux"><div class="resc">Kathryn E. O’Reilly, et al. Pharmaceuticals (Basel). 2013 November;6(11):1429-1450.</div><p class="links"><a class="dblinks" href="/pubmed/24287465" ref="ordinalpos=2825&amp;ncbi_uid=5605104&amp;link_uid=24287465">Citation</a><a class="dblinks" href="/pmc/articles/PMC3854019/" ref="ordinalpos=2825&amp;ncbi_uid=5605104&amp;link_uid=PMC3854019">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2826.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC523892/figure/fig1/" ref="ordinalpos=2826&amp;ncbi_uid=544149&amp;link_uid=PMC523892" href="/pmc/articles/PMC523892/"><img src="/pmc/articles/instance/523892/bin/zpp0030483760001.gif" src-large="/pmc/articles/instance/523892/bin/zpp0030483760001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC523892/" ref="ordinalpos=2826&amp;ncbi_uid=544149&amp;link_uid=PMC523892" image-link="/pmc/articles/PMC523892/figure/fig1/" class="imagepopup">Figure 1. From: The SOS3 Family of Calcium Sensors and SOS2 Family of Protein Kinases in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Diagram showing that the SOS3-SOS2 <span class="highlight" style="background-color:">signaling</span> module functions in a salt stress-elicited Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which mediates salt tolerance. Similarly, various SCaBP-PKS complexes have been implicated in Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> pathways in response to abscisic acid (ABA), sugar, high pH, or drought and cold stresses.</div></p></div><div class="aux"><div class="resc">Deming Gong, et al. Plant Physiol. 2004 March;134(3):919-926.</div><p class="links"><a class="dblinks" href="/pubmed/15020756" ref="ordinalpos=2826&amp;ncbi_uid=544149&amp;link_uid=15020756">Citation</a><a class="dblinks" href="/pmc/articles/PMC523892/" ref="ordinalpos=2826&amp;ncbi_uid=544149&amp;link_uid=PMC523892">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2827.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3464048/figure/F1/" ref="ordinalpos=2827&amp;ncbi_uid=4853774&amp;link_uid=PMC3464048" href="/pmc/articles/PMC3464048/"><img src="/pmc/articles/instance/3464048/bin/nihms-406046-f0001.gif" src-large="/pmc/articles/instance/3464048/bin/nihms-406046-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3464048/" ref="ordinalpos=2827&amp;ncbi_uid=4853774&amp;link_uid=PMC3464048" image-link="/pmc/articles/PMC3464048/figure/F1/" class="imagepopup">Figure 1. Dysregulated survival pathways in large granular lymphocyte leukemia.  From: Large granular lymphocyte leukemia: from dysregulated pathways to therapeutic targets. </a></p><div class="supp"><p class="details"><div>The intact Fas–FasL-mediated apoptosis <span class="highlight" style="background-color:">pathway</span> is shown. Elevated levels of the soluble form of Fas or FasL contribute to apoptotic resistance in leukemic LGLs by acting as receptor or ligand decoys. MMP inhibitors are proposed for therapeutic intervention. Other contributing <span class="highlight" style="background-color:">signaling</span> pathways that are essential in LGL leukemia pathogenesis include the signals that are mediated by IL-15, PDGF and NF-κB. IL-15 promotes LGL survival either by activating the JAK–STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to upregulate gene expression of antiapoptotic factor or by targeting Bid to the proteasome for degradation. IL-15 <span class="highlight" style="background-color:">signaling</span> can be inhibited by a Mikβ1 monoclonal antibody targeting the IL-15R. PDGF functions through PI3K–Akt and Ras–MEK1–ERK <span class="highlight" style="background-color:">signaling</span> cascades, leading to cell proliferation and growth. PDGF <span class="highlight" style="background-color:">signaling</span> can be inhibited by RTK inhibitors such as imatinib. PI3K–Akt <span class="highlight" style="background-color:">signaling</span> can be blocked by using pan-PI3K inhibitors. Ras–MEK1–ERK <span class="highlight" style="background-color:">signaling</span> can be targeted using farnesyltransferase inhibitors to prevent farnesylation of Ras or by using MEK1 inhibitors such as PD98059 and U0216. NF-κB complex upregulates the transcription of survival genes resulting in survival of leukemic LGLs. Bortezomib, a proteasome inhibitor, can be used to prevent NF-κB <span class="highlight" style="background-color:">signaling</span>. FasL: Fas ligand; IκB: Inhibitor of NF-κB; IKK: IκB kinase; IL-15R: IL-15 receptor; LGL: Large granular lymphocyte; MMP: Matrix metalloproteinase; RTK: Receptor tyrosine kinase.</div></p></div><div class="aux"><div class="resc">Francis LeBlanc, et al. Future Oncol. 2012 July;8(7):787-801.</div><p class="links"><a class="dblinks" href="/pubmed/22830400" ref="ordinalpos=2827&amp;ncbi_uid=4853774&amp;link_uid=22830400">Citation</a><a class="dblinks" href="/pmc/articles/PMC3464048/" ref="ordinalpos=2827&amp;ncbi_uid=4853774&amp;link_uid=PMC3464048">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2828.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3441314/figure/F6/" ref="ordinalpos=2828&amp;ncbi_uid=5338107&amp;link_uid=PMC3441314" href="/pmc/articles/PMC3441314/"><img src="/pmc/articles/instance/3441314/bin/zgb0101257090006.gif" src-large="/pmc/articles/instance/3441314/bin/zgb0101257090006.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3441314/" ref="ordinalpos=2828&amp;ncbi_uid=5338107&amp;link_uid=PMC3441314" image-link="/pmc/articles/PMC3441314/figure/F6/" class="imagepopup">Figure 6.TLR4-induced macrophage activation regulated by NF-κB, PI3K/AKT, and MAPK <span class="highlight" style="background-color:">signaling</span> pathways..  From: The HIV-1 protease inhibitor nelfinavir activates PP2 and inhibits MAPK <span class="highlight" style="background-color:">signaling</span> in macrophages: a <span class="highlight" style="background-color:">pathway</span> to reduce inflammation. </a></p><div class="supp"><p class="details"><div>A model for potential mechanisms of the anti-inflammatory effects of NFV. The scavenger molecule CD14 in membrane-associated or sCD14 form binds LPS and localizes to a TLR4 <span class="highlight" style="background-color:">signaling</span> complex. A phospho-protein cascade, including IL-1R-associated kinase (IRAK)1 and -4 and TRAF6, serves as a <span class="highlight" style="background-color:">signaling</span> hub that leads into multiple downstream pathways, including NF-κB, PI3K/AKT, and MAPK. NF-κB <span class="highlight" style="background-color:">pathway</span> is initiated by TGF-β-activated kinase 1 (TAK1)-dependent phosphorylation of IKKα/β, which subsequently phosphorylates the inhibitory protein IκBα, triggering its ubiquitination and degradation. Degradation of IκBα releases p50/p65, which is phophorylated and translocated to the nucleus to initiate transcription of proinflammatory genes. PI3K/AKT <span class="highlight" style="background-color:">signaling</span> is triggered by receptor-interacting serine/threonine-protein kinase (RIP) and culminates with inhibition of proinflammatory <span class="highlight" style="background-color:">signaling</span>/transcription and/or enhancement of anti-inflammatory gene expression. MAPK <span class="highlight" style="background-color:">signaling</span> is triggered by activation of proximal MAP3K proteins, followed by downstream activation of Mek1/2 and Erk1/2 and induction of proinflammatory genes. NFV inhibits proteasome activity, leading to accumulation of misfolded proteins and activation of the UPR, including activation of PP1 and possibly PP2, which can be inhibited chemically with calyculin A (CA) and okadaic acid (OA), respectively.</div></p></div><div class="aux"><div class="resc">Mark A. Wallet, et al. J Leukoc Biol. 2012 October;92(4):795-805.</div><p class="links"><a class="dblinks" href="/pubmed/22786868" ref="ordinalpos=2828&amp;ncbi_uid=5338107&amp;link_uid=22786868">Citation</a><a class="dblinks" href="/pmc/articles/PMC3441314/" ref="ordinalpos=2828&amp;ncbi_uid=5338107&amp;link_uid=PMC3441314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2829.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4358479/figure/fig04/" ref="ordinalpos=2829&amp;ncbi_uid=7289341&amp;link_uid=PMC4358479" href="/pmc/articles/PMC4358479/"><img src="/pmc/articles/instance/4358479/bin/ijcem0008-0512-f4.gif" src-large="/pmc/articles/instance/4358479/bin/ijcem0008-0512-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4358479/" ref="ordinalpos=2829&amp;ncbi_uid=7289341&amp;link_uid=PMC4358479" image-link="/pmc/articles/PMC4358479/figure/fig04/" class="imagepopup">Figure 4.  From: Differential expression of 5-HT-related genes in symptomatic pulmonary embolism patients. </a></p><div class="supp"><p class="details"><div>The mRNA expression of genes related to receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showed that the expressions of DalDAG-GEF I, Tubby, EPAC, and PKA (factors in 5-HT receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) were down-regulated, and the expressions of PLCγ, PLCβ, SPA1, RIAM, RAPL, Talin, PKC and Pyk2 were up-regulated in PE patients (*P &lt; 0.05, **P &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Yun Jin, et al. Int J Clin Exp Med. 2015;8(1):512-518.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2829&amp;ncbi_uid=7289341&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4358479/" ref="ordinalpos=2829&amp;ncbi_uid=7289341&amp;link_uid=PMC4358479">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2830.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2913769/figure/F9/" ref="ordinalpos=2830&amp;ncbi_uid=3124998&amp;link_uid=PMC2913769" href="/pmc/articles/PMC2913769/"><img src="/pmc/articles/instance/2913769/bin/zpt0081090420009.gif" src-large="/pmc/articles/instance/2913769/bin/zpt0081090420009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2913769/" ref="ordinalpos=2830&amp;ncbi_uid=3124998&amp;link_uid=PMC2913769" image-link="/pmc/articles/PMC2913769/figure/F9/" class="imagepopup">Fig. 9. From: Estrogen Receptor ? <span class="highlight" style="background-color:">Signaling</span> through Phosphatase and Tensin Homolog/Phosphoinositide 3-Kinase/Akt/Glycogen Synthase Kinase 3 Down-Regulates Blood-Brain Barrier Breast Cancer Resistance Protein. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> pathways through which E2 reduces BCRP transport activity and protein expression in brain capillaries. <span class="highlight" style="background-color:">Pathway</span> A was characterized previously (Hartz et al., 2010). <span class="highlight" style="background-color:">Pathway</span> B is described in the present study. We speculate that the two pathways are linked in that transporter internalization must precede proteasomal degradation. It remains to be determined whether PTEN/PI3K/Akt/GSK3 <span class="highlight" style="background-color:">signaling</span> directs internalized BCRP to the proteasome or whether it stimulates ubiquitination and processing of the protein.</div></p></div><div class="aux"><div class="resc">A. M. S. Hartz, et al. J Pharmacol Exp Ther. 2010 August;334(2):467-476.</div><p class="links"><a class="dblinks" href="/pubmed/20460386" ref="ordinalpos=2830&amp;ncbi_uid=3124998&amp;link_uid=20460386">Citation</a><a class="dblinks" href="/pmc/articles/PMC2913769/" ref="ordinalpos=2830&amp;ncbi_uid=3124998&amp;link_uid=PMC2913769">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2831.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4028840/figure/F1/" ref="ordinalpos=2831&amp;ncbi_uid=6158050&amp;link_uid=PMC4028840" href="/pmc/articles/PMC4028840/"><img src="/pmc/articles/instance/4028840/bin/1750-9378-8-49-1.gif" src-large="/pmc/articles/instance/4028840/bin/1750-9378-8-49-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4028840/" ref="ordinalpos=2831&amp;ncbi_uid=6158050&amp;link_uid=PMC4028840" image-link="/pmc/articles/PMC4028840/figure/F1/" class="imagepopup">Figure 1.  From: Akt inhibitors: mechanism of action and implications for anticancer therapeutics. </a></p><div class="supp"><p class="details"><div><b>The PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [</b>[3]<b>].</b></div></p></div><div class="aux"><div class="resc">Jaikrit Bhutani, et al. Infect Agent Cancer. 2013;8:49-49.</div><p class="links"><a class="dblinks" href="/pubmed/24330834" ref="ordinalpos=2831&amp;ncbi_uid=6158050&amp;link_uid=24330834">Citation</a><a class="dblinks" href="/pmc/articles/PMC4028840/" ref="ordinalpos=2831&amp;ncbi_uid=6158050&amp;link_uid=PMC4028840">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2832.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204250/figure/F1/" ref="ordinalpos=2832&amp;ncbi_uid=3313893&amp;link_uid=PMC3204250" href="/pmc/articles/PMC3204250/"><img src="/pmc/articles/instance/3204250/bin/1756-0500-4-408-1.gif" src-large="/pmc/articles/instance/3204250/bin/1756-0500-4-408-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204250/" ref="ordinalpos=2832&amp;ncbi_uid=3313893&amp;link_uid=PMC3204250" image-link="/pmc/articles/PMC3204250/figure/F1/" class="imagepopup">Figure 1.  From: A comprehensive curated resource for follicle stimulating hormone <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>An overview of the FSH <span class="highlight" style="background-color:">pathway</span> page in NetPath</b>. The FSH <span class="highlight" style="background-color:">pathway</span> page in NetPath displays the statistics of data in the FSH <span class="highlight" style="background-color:">pathway</span> reactions including the number of proteins, protein-protein interactions, enzyme-substrate reactions, translocation events and differentially regulated genes in normal/primary human cells. The complete list of these reactions is accessible through the specific links provided for each reaction. Brief comments are provided for reactions curated under reactions such as PPI, enzyme-substrate reactions and transport. Each of these molecules is linked to NetPath molecule page which is further linked to Entrez gene, HPRD [30], OMIM [31] and Swiss-Prot [32] identifiers. A list of curators/reviewers is provided in the FSH <span class="highlight" style="background-color:">pathway</span> page with the details of the <span class="highlight" style="background-color:">Pathway</span> Authority. A 'comments' tab is provided in the <span class="highlight" style="background-color:">pathway</span> page to invite queries and suggestions from the community as a means to keep the <span class="highlight" style="background-color:">pathway</span> updated and as error-free as possible.</div></p></div><div class="aux"><div class="resc">Deepthi Telikicherla, et al. BMC Res Notes. 2011;4:408-408.</div><p class="links"><a class="dblinks" href="/pubmed/21996254" ref="ordinalpos=2832&amp;ncbi_uid=3313893&amp;link_uid=21996254">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204250/" ref="ordinalpos=2832&amp;ncbi_uid=3313893&amp;link_uid=PMC3204250">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2833.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3692130/figure/gkt451-F1/" ref="ordinalpos=2833&amp;ncbi_uid=5023455&amp;link_uid=PMC3692130" href="/pmc/articles/PMC3692130/"><img src="/pmc/articles/instance/3692130/bin/gkt451f1p.gif" src-large="/pmc/articles/instance/3692130/bin/gkt451f1p.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3692130/" ref="ordinalpos=2833&amp;ncbi_uid=5023455&amp;link_uid=PMC3692130" image-link="/pmc/articles/PMC3692130/figure/gkt451-F1/" class="imagepopup">Figure 1. From: Inferring the functional effect of gene expression changes in <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Graphical representation of Wnt <span class="highlight" style="background-color:">pathway</span> as described in the KEGG repository (1). This <span class="highlight" style="background-color:">pathway</span> has a canonical Wnt/β-catenin cascade and two non-canonical pathways, named Wnt/Planar cell polarity (Wnt/PCP) <span class="highlight" style="background-color:">pathway</span> and Wnt/Ca2+ <span class="highlight" style="background-color:">pathway</span>, which conform the complete Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<b>A</b>). Here, we represent the Wnt <span class="highlight" style="background-color:">pathway</span> in a gene expression microarray experiment in which colorectal cancer patients are compared with control subjects (14). In the diagrams, nodes represent proteins and edges represent the type of interactions among them. (A) The results provided by PATHiWAYS in which the stimulus–response circuits with significantly decreased activity in cancer patients with respect to healthy control subjects are highlighted in blue. Solid edges indicate activations and dotted edges indicate inhibitions. (<b>B</b>) Representation, using the cytoscape program, of the individual gene expression activations (in red) or deactivations (in blue) when comparing the cancer samples with respect to the normal control samples. Edges ended in an arrow indicate activations and edges ended in ‘T’ indicate inhibitions. Despite many genes in many circuits increase or decrease their activity (B), only a few circuits are really affected by these changes (A).</div></p></div><div class="aux"><div class="resc">Patricia Sebastián-León, et al. Nucleic Acids Res. 2013 July;41(Web Server issue):W213-W217.</div><p class="links"><a class="dblinks" href="/pubmed/23748960" ref="ordinalpos=2833&amp;ncbi_uid=5023455&amp;link_uid=23748960">Citation</a><a class="dblinks" href="/pmc/articles/PMC3692130/" ref="ordinalpos=2833&amp;ncbi_uid=5023455&amp;link_uid=PMC3692130">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2834.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3398381/figure/F2/" ref="ordinalpos=2834&amp;ncbi_uid=4211242&amp;link_uid=PMC3398381" href="/pmc/articles/PMC3398381/"><img src="/pmc/articles/instance/3398381/bin/nihms389957f2.gif" src-large="/pmc/articles/instance/3398381/bin/nihms389957f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3398381/" ref="ordinalpos=2834&amp;ncbi_uid=4211242&amp;link_uid=PMC3398381" image-link="/pmc/articles/PMC3398381/figure/F2/" class="imagepopup">Figure 2.  From: Estrogen <span class="highlight" style="background-color:">Signaling</span> and Cardiovascular Disease. </a></p><div class="supp"><p class="details"><div>Figure 2 A illustrated the interaction of the genomic and acute estrogen <span class="highlight" style="background-color:">signaling</span> pathways in regulating nitric oxide (NO) <span class="highlight" style="background-color:">signaling</span>. Estrogen (E2) can bind to ER resulting in dimerization and activation of gene transcription. Nitric oxide synthase (NOS) expression has been reported to be regulated by E2 and females have been shown to have higher levels of NOS than males. In addition, E2 active via rapid <span class="highlight" style="background-color:">signaling</span> pathways can activate the PI3K <span class="highlight" style="background-color:">pathway</span> regulating in phosphorylation and activation of NOS.<br />Figure 2B illustrates how this <span class="highlight" style="background-color:">signaling</span> might be altered with aging and disease and provides a possible explanation as to why HRT failed to protect in post-menopausal women. Tetrahydropterin (BH4) is a cofactor for NOS and in the absence of BH4, NOS now generates ROS rather than NO.</div></p></div><div class="aux"><div class="resc">Elizabeth Murphy. Circ Res. ;109(6):687-696.</div><p class="links"><a class="dblinks" href="/pubmed/21885836" ref="ordinalpos=2834&amp;ncbi_uid=4211242&amp;link_uid=21885836">Citation</a><a class="dblinks" href="/pmc/articles/PMC3398381/" ref="ordinalpos=2834&amp;ncbi_uid=4211242&amp;link_uid=PMC3398381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2835.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4379025/figure/pone.0119512.g007/" ref="ordinalpos=2835&amp;ncbi_uid=7401463&amp;link_uid=PMC4379025" href="/pmc/articles/PMC4379025/"><img src="/pmc/articles/instance/4379025/bin/pone.0119512.g007.gif" src-large="/pmc/articles/instance/4379025/bin/pone.0119512.g007.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4379025/" ref="ordinalpos=2835&amp;ncbi_uid=7401463&amp;link_uid=PMC4379025" image-link="/pmc/articles/PMC4379025/figure/pone.0119512.g007/" class="imagepopup">Fig 7. Effect of <span class="highlight" style="background-color:">pathway</span> inhibitors in the presence or absence of costunolide on AFP expression and secretion..  From: Telomerase Inhibition Decreases Alpha-Fetoprotein Expression and Secretion by Hepatocellular Carcinoma Cell Lines: In Vitro and In Vivo Study. </a></p><div class="supp"><p class="details"><div>The cells were seeded in a six-well plate at a density of 2x10<sup>5</sup> cells/well. At 80% confluence, the cells were treated with inhibitors of the PI3k/mTOR/Akt and MAPK/ERK <span class="highlight" style="background-color:">signaling</span> pathways and costunolide for 48 h. The supernatants were then collected and assessed by ELISA. (A) Effect of inhibitors of the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> on AFP secretion by HepG2/C3A cells. (B) Effect of inhibitors of the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span> on AFP secretion by PLC/PRF/5 cells. *P&lt;0.05 versus untreated cells. **P&lt;0.01 versus untreated cells.</div></p></div><div class="aux"><div class="resc">Roula Tahtouh, et al. PLoS One. 2015;10(3):e0119512.</div><p class="links"><a class="dblinks" href="/pubmed/25822740" ref="ordinalpos=2835&amp;ncbi_uid=7401463&amp;link_uid=25822740">Citation</a><a class="dblinks" href="/pmc/articles/PMC4379025/" ref="ordinalpos=2835&amp;ncbi_uid=7401463&amp;link_uid=PMC4379025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2836.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3307977/figure/fig6/" ref="ordinalpos=2836&amp;ncbi_uid=3883304&amp;link_uid=PMC3307977" href="/pmc/articles/PMC3307977/"><img src="/pmc/articles/instance/3307977/bin/cdd2011132f6.gif" src-large="/pmc/articles/instance/3307977/bin/cdd2011132f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3307977/" ref="ordinalpos=2836&amp;ncbi_uid=3883304&amp;link_uid=PMC3307977" image-link="/pmc/articles/PMC3307977/figure/fig6/" class="imagepopup">Figure 6.  From: FGF2 regulates melanocytes viability through the STAT3-transactivated PAX3 transcription. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the FGF2-STAT3-PAX3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">L Dong, et al. Cell Death Differ. 2012 April;19(4):616-622.</div><p class="links"><a class="dblinks" href="/pubmed/21997191" ref="ordinalpos=2836&amp;ncbi_uid=3883304&amp;link_uid=21997191">Citation</a><a class="dblinks" href="/pmc/articles/PMC3307977/" ref="ordinalpos=2836&amp;ncbi_uid=3883304&amp;link_uid=PMC3307977">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2837.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2682041/figure/f6/" ref="ordinalpos=2837&amp;ncbi_uid=1884156&amp;link_uid=PMC2682041" href="/pmc/articles/PMC2682041/"><img src="/pmc/articles/instance/2682041/bin/zmb0100980510006.gif" src-large="/pmc/articles/instance/2682041/bin/zmb0100980510006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2682041/" ref="ordinalpos=2837&amp;ncbi_uid=1884156&amp;link_uid=PMC2682041" image-link="/pmc/articles/PMC2682041/figure/f6/" class="imagepopup">FIG. 6. From: Normal Function of the Yeast TOR <span class="highlight" style="background-color:">Pathway</span> Requires the Type 2C Protein Phosphatase Ptc1 . </a></p><div class="supp"><p class="details"><div>Epistatic analysis of <i>ptc1</i> and mutations affecting the TOR <span class="highlight" style="background-color:">pathway</span>. (A) A simplified model of <span class="highlight" style="background-color:">signaling</span> through the TOR <span class="highlight" style="background-color:">pathway</span> (focused on the regulation of NCR genes) based on previous models (15, 29, 33). (B) Rapamycin and caffeine sensitivity of diverse mutants in genes relevant in the TOR <span class="highlight" style="background-color:">pathway</span> in the presence (+) or the absence of <i>PTC1</i> (−). Cultures were spotted on YPD plates containing the indicated concentrations of the drugs, and growth was monitored after 2 days. WT, wild type.</div></p></div><div class="aux"><div class="resc">Asier González, et al. Mol Cell Biol. 2009 May;29(10):2876-2888.</div><p class="links"><a class="dblinks" href="/pubmed/19273591" ref="ordinalpos=2837&amp;ncbi_uid=1884156&amp;link_uid=19273591">Citation</a><a class="dblinks" href="/pmc/articles/PMC2682041/" ref="ordinalpos=2837&amp;ncbi_uid=1884156&amp;link_uid=PMC2682041">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2838.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3834519/figure/F5/" ref="ordinalpos=2838&amp;ncbi_uid=5570971&amp;link_uid=PMC3834519" href="/pmc/articles/PMC3834519/"><img src="/pmc/articles/instance/3834519/bin/fgene-04-00236-g0005.gif" src-large="/pmc/articles/instance/3834519/bin/fgene-04-00236-g0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3834519/" ref="ordinalpos=2838&amp;ncbi_uid=5570971&amp;link_uid=PMC3834519" image-link="/pmc/articles/PMC3834519/figure/F5/" class="imagepopup">Figure 5.  From: Genome-wide multi-omics profiling of colorectal cancer identifies immune determinants strongly associated with relapse. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span> analysis of top variants from exome sequencing data</b>. Three pathways highly enriched with non-synonymous variants found exclusively in relapse patients are shown. <b>(A)</b> Antigen presentation <span class="highlight" style="background-color:">pathway</span> affected in 8 relapse cases shown. Variants with predicted activation effect were mapped predominantly to one branch of <span class="highlight" style="background-color:">pathway</span> related to activation of CD4+ Lymphocytes. <b>(B)</b> PKC-Theta <span class="highlight" style="background-color:">Signaling</span> in T-Lymphocytes affected in 14 relapse cases shown. <b>(C)</b> PTEN <span class="highlight" style="background-color:">Signaling</span> affected in 14 relapse cases shown.</div></p></div><div class="aux"><div class="resc">Subha Madhavan, et al. Front Genet. 2013;4:236.</div><p class="links"><a class="dblinks" href="/pubmed/24312117" ref="ordinalpos=2838&amp;ncbi_uid=5570971&amp;link_uid=24312117">Citation</a><a class="dblinks" href="/pmc/articles/PMC3834519/" ref="ordinalpos=2838&amp;ncbi_uid=5570971&amp;link_uid=PMC3834519">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2839.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2712453/figure/F7/" ref="ordinalpos=2839&amp;ncbi_uid=1925234&amp;link_uid=PMC2712453" href="/pmc/articles/PMC2712453/"><img src="/pmc/articles/instance/2712453/bin/1752-0509-3-67-7.gif" src-large="/pmc/articles/instance/2712453/bin/1752-0509-3-67-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2712453/" ref="ordinalpos=2839&amp;ncbi_uid=1925234&amp;link_uid=PMC2712453" image-link="/pmc/articles/PMC2712453/figure/F7/" class="imagepopup">Figure 7.  From: BowTieBuilder: modeling signal transduction pathways. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> inferred from a gene regulatory network</b>. This figure depicts a bow tie inferred from membrane-bound proteins and TFs that were predicted to be active in a gene regulatory network [21]. The proteins in the bottom are the TFs, the proteins at the top are the membrane-bound proteins and the remaining proteins constitute the inferred <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Different modules of this <span class="highlight" style="background-color:">pathway</span> are associated with different biological processes, where the core module is associated with exocytosis – a process through which cells direct secretory vesicles out of the cell.</div></p></div><div class="aux"><div class="resc">Jochen Supper, et al. BMC Syst Biol. 2009;3:67-67.</div><p class="links"><a class="dblinks" href="/pubmed/19566957" ref="ordinalpos=2839&amp;ncbi_uid=1925234&amp;link_uid=19566957">Citation</a><a class="dblinks" href="/pmc/articles/PMC2712453/" ref="ordinalpos=2839&amp;ncbi_uid=1925234&amp;link_uid=PMC2712453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2840.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2904940/figure/F5/" ref="ordinalpos=2840&amp;ncbi_uid=2710396&amp;link_uid=PMC2904940" href="/pmc/articles/PMC2904940/"><img src="/pmc/articles/instance/2904940/bin/1496fig5.gif" src-large="/pmc/articles/instance/2904940/bin/1496fig5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2904940/" ref="ordinalpos=2840&amp;ncbi_uid=2710396&amp;link_uid=PMC2904940" image-link="/pmc/articles/PMC2904940/figure/F5/" class="imagepopup">Figure 5. From: The CNK1 scaffold binds cytohesins and promotes insulin <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>WT-CNK1 and Q67L-Arf6 restore insulin-mediated <span class="highlight" style="background-color:">signaling</span> to the PI3K/AKT <span class="highlight" style="background-color:">pathway</span> in CNK1-depleted cells. (<i>A</i>) HeLa cells stably expressing either vector control or siRNA-resistant WT-CNK1 or ΔCBD-CNK1 were transfected with control or CNK1-siRNAs prior to serum starvation and insulin treatment. Lysates were prepared and examined by immunoblot analysis as indicated. (<i>B</i>) Serum-straved HeLa cells were left untreated or were treated overnight with the cytohesin inhibitor SecinH3 (20 μM) prior to insulin stimulation. Lysates were prepared and examined by immunoblot analysis. (<i>C</i>) HeLa cells stably expressing vector control or Q67L-Arf6 were transfected with control or CNK1-siRNAs prior to serum starvation and insulin treatment. Lysates were prepared and examined by immunoblot analysis. (<i>D</i>) HeLa cells transfected with control, Arf1, Arf6, or Arf1, and Arf6-siRNAs were serum-starved and then treated or not with insulin prior to lysis. Lysates were examined by immunoblot analysis.</div></p></div><div class="aux"><div class="resc">Junghwa Lim, et al. Genes Dev. 2010 July 15;24(14):1496-1506.</div><p class="links"><a class="dblinks" href="/pubmed/20634316" ref="ordinalpos=2840&amp;ncbi_uid=2710396&amp;link_uid=20634316">Citation</a><a class="dblinks" href="/pmc/articles/PMC2904940/" ref="ordinalpos=2840&amp;ncbi_uid=2710396&amp;link_uid=PMC2904940">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2841.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4159007/figure/evu158-F8/" ref="ordinalpos=2841&amp;ncbi_uid=6603326&amp;link_uid=PMC4159007" href="/pmc/articles/PMC4159007/"><img src="/pmc/articles/instance/4159007/bin/evu158f8p.gif" src-large="/pmc/articles/instance/4159007/bin/evu158f8p.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 8.—" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4159007/" ref="ordinalpos=2841&amp;ncbi_uid=6603326&amp;link_uid=PMC4159007" image-link="/pmc/articles/PMC4159007/figure/evu158-F8/" class="imagepopup">F ig . 8.—.  From: Epigenetics and the Evolution of Darwin's Finches. </a></p><div class="supp"><p class="details"><div>Epimutation-associated genes and correlated toll receptor <span class="highlight" style="background-color:">pathway</span>. The genes having associated epimutations in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> presented for the different species are identified as FUL (purple), SCA (green), PAR (blue), and CRA (red) colored boxed genes.</div></p></div><div class="aux"><div class="resc">Michael K. Skinner, et al. Genome Biol Evol. 2014 August;6(8):1972-1989.</div><p class="links"><a class="dblinks" href="/pubmed/25062919" ref="ordinalpos=2841&amp;ncbi_uid=6603326&amp;link_uid=25062919">Citation</a><a class="dblinks" href="/pmc/articles/PMC4159007/" ref="ordinalpos=2841&amp;ncbi_uid=6603326&amp;link_uid=PMC4159007">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2842.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3939323/figure/F3/" ref="ordinalpos=2842&amp;ncbi_uid=5909201&amp;link_uid=PMC3939323" href="/pmc/articles/PMC3939323/"><img src="/pmc/articles/instance/3939323/bin/jls0011432660003.gif" src-large="/pmc/articles/instance/3939323/bin/jls0011432660003.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3939323/" ref="ordinalpos=2842&amp;ncbi_uid=5909201&amp;link_uid=PMC3939323" image-link="/pmc/articles/PMC3939323/figure/F3/" class="imagepopup">Figure 3. From: Gene Expression Profiling of Human Kidneys Undergoing Laparoscopic Donor Nephrectomy. </a></p><div class="supp"><p class="details"><div>A, JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in interaction network. B, Working hypothesis, showing signal transduction process in <span class="highlight" style="background-color:">pathway</span>. The yellow boxes in A represent the DEGs.</div></p></div><div class="aux"><div class="resc">Ya-hong Xu, et al. JSLS. 2014 Jan-Mar;18(1):102-109.</div><p class="links"><a class="dblinks" href="/pubmed/24680151" ref="ordinalpos=2842&amp;ncbi_uid=5909201&amp;link_uid=24680151">Citation</a><a class="dblinks" href="/pmc/articles/PMC3939323/" ref="ordinalpos=2842&amp;ncbi_uid=5909201&amp;link_uid=PMC3939323">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2843.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3106420/figure/F1/" ref="ordinalpos=2843&amp;ncbi_uid=3371783&amp;link_uid=PMC3106420" href="/pmc/articles/PMC3106420/"><img src="/pmc/articles/instance/3106420/bin/nihms-289947-f0001.gif" src-large="/pmc/articles/instance/3106420/bin/nihms-289947-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3106420/" ref="ordinalpos=2843&amp;ncbi_uid=3371783&amp;link_uid=PMC3106420" image-link="/pmc/articles/PMC3106420/figure/F1/" class="imagepopup">Fig. 1.  From: Systems biology approach to Wilson's disease. </a></p><div class="supp"><p class="details"><div>The top 10 canonical <span class="highlight" style="background-color:">signaling</span> pathways down-regulated in LEC rats (<i>top</i>) and Atp7b<sup>−/−</sup> mice (<i>bottom</i>). Differentially expressed genes (greater than 1.5 fold) were derived from comparing 6-week old Atp7b<sup>−/−</sup> to wild-type littermates; or 26 week old LEC rats—to 4 week old LEC rats. Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (<i>IPA</i>) defined 218 genes from Atp7b<sup>−/−</sup> and 1208 from LEC rats eligible for <span class="highlight" style="background-color:">pathway</span> analysis. These genes were queried against the IPA <span class="highlight" style="background-color:">pathway</span> library to identify over-represented <span class="highlight" style="background-color:">signaling</span> pathways that were significantly down-regulated. The significance [–log (<i>P</i>-value)] of a given <span class="highlight" style="background-color:">pathway</span> is measured by the likelihood that the <span class="highlight" style="background-color:">pathway</span> is associated with the dataset by random chance using the Fischer’s exact test. <i>Striped bars</i> Pathways common to Atp7b<sup>−/−</sup> mice and LEC rats. <i>Black bars</i> Pathways unique to Atp7b<sup>−/−</sup> mice or LEC rats. *Pathways involved in lipid or steroid metabolism</div></p></div><div class="aux"><div class="resc">Jason L. Burkhead, et al. Biometals. ;24(3):455-466.</div><p class="links"><a class="dblinks" href="/pubmed/21380607" ref="ordinalpos=2843&amp;ncbi_uid=3371783&amp;link_uid=21380607">Citation</a><a class="dblinks" href="/pmc/articles/PMC3106420/" ref="ordinalpos=2843&amp;ncbi_uid=3371783&amp;link_uid=PMC3106420">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2844.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3234824/figure/F9/" ref="ordinalpos=2844&amp;ncbi_uid=4450465&amp;link_uid=PMC3234824" href="/pmc/articles/PMC3234824/"><img src="/pmc/articles/instance/3234824/bin/zbc0521189840009.gif" src-large="/pmc/articles/instance/3234824/bin/zbc0521189840009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3234824/" ref="ordinalpos=2844&amp;ncbi_uid=4450465&amp;link_uid=PMC3234824" image-link="/pmc/articles/PMC3234824/figure/F9/" class="imagepopup">FIGURE 9. From: Cysteine-rich 61-Connective Tissue Growth Factor-nephroblastoma-overexpressed 5 (CCN5)/Wnt-1-induced <span class="highlight" style="background-color:">Signaling</span> Protein-2 (WISP-2) Regulates MicroRNA-10b via Hypoxia-inducible Factor-1?-TWIST <span class="highlight" style="background-color:">Signaling</span> Networks in Human Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>Diagrammatic illustration of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in the regulation of miR-10b by CCN5 in breast cancer cells.</b> Under CCN5-free microenvironment, phosphorylates JNK resulting in activation of JNK <span class="highlight" style="background-color:">pathway</span> inducing HIF-1α synthesis or stability, which in turn up-regulates <i>TWIST1</i>, resulting in up-regulation of miR-10b from HOXD cluster (30), promoting migration of breast cancer cells. All the cascades are absent or minimal in CCN5-positive breast cancer cells or may be reversed if CCN5 is activated in CCN5-negative breast cancer cells. (+) indicates positive effect, and (−) indicates negative effect.</div></p></div><div class="aux"><div class="resc">Inamul Haque, et al. J Biol Chem. 2011 December 16;286(50):43475-43485.</div><p class="links"><a class="dblinks" href="/pubmed/22020939" ref="ordinalpos=2844&amp;ncbi_uid=4450465&amp;link_uid=22020939">Citation</a><a class="dblinks" href="/pmc/articles/PMC3234824/" ref="ordinalpos=2844&amp;ncbi_uid=4450465&amp;link_uid=PMC3234824">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2845.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2921074/figure/F1/" ref="ordinalpos=2845&amp;ncbi_uid=2414823&amp;link_uid=PMC2921074" href="/pmc/articles/PMC2921074/"><img src="/pmc/articles/instance/2921074/bin/nihms25473f1.gif" src-large="/pmc/articles/instance/2921074/bin/nihms25473f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2921074/" ref="ordinalpos=2845&amp;ncbi_uid=2414823&amp;link_uid=PMC2921074" image-link="/pmc/articles/PMC2921074/figure/F1/" class="imagepopup">Figure 1.  From: Neuro-immune interactions via the cholinergic anti-inflammatory <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Cholinergic mechanisms of modulating immune function and controlling inflammation (proposed mechanisms). Efferent vagus nerve cholinergic <span class="highlight" style="background-color:">signaling</span> inhibits TNF and other pro-inflammatory cytokine levels through α7nAChR-mediated mechanisms. Experimental evidence indicates that activation of α7nAChR on immune cells by nicotine can prevent NF-κB nuclear translocation. α7nAChR-mediated activation of the JAK2/STAT3 <span class="highlight" style="background-color:">pathway</span> has also been demonstrated in response to nicotine. These <span class="highlight" style="background-color:">signaling</span> pathways play a central role in transmitting vagal or nicotine-induced cholinergic anti-inflammatory <span class="highlight" style="background-color:">signaling</span> leading to inhibition of immune cell activation and suppression of TNF, HMGB1, macrophage inflammatory protein-2 (MIP-2) and IL-6 synthesis. Brain muscarinic acetylcholine receptor (mAChR) <span class="highlight" style="background-color:">signaling</span> mechanisms also modulate vagal immunoregulatory and anti-inflammatory output (See text for details).</div></p></div><div class="aux"><div class="resc">Margot Gallowitsch-Puerta, et al. Life Sci. ;80(24-25):2325-2329.</div><p class="links"><a class="dblinks" href="/pubmed/17289087" ref="ordinalpos=2845&amp;ncbi_uid=2414823&amp;link_uid=17289087">Citation</a><a class="dblinks" href="/pmc/articles/PMC2921074/" ref="ordinalpos=2845&amp;ncbi_uid=2414823&amp;link_uid=PMC2921074">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2846.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3750319/figure/F2/" ref="ordinalpos=2846&amp;ncbi_uid=5225784&amp;link_uid=PMC3750319" href="/pmc/articles/PMC3750319/"><img src="/pmc/articles/instance/3750319/bin/1476-4598-12-86-2.gif" src-large="/pmc/articles/instance/3750319/bin/1476-4598-12-86-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3750319/" ref="ordinalpos=2846&amp;ncbi_uid=5225784&amp;link_uid=PMC3750319" image-link="/pmc/articles/PMC3750319/figure/F2/" class="imagepopup">Figure 2.  From: The complexity of NF-?B <span class="highlight" style="background-color:">signaling</span> in inflammation and cancer. </a></p><div class="supp"><p class="details"><div><b>The canonical, non-canonical and the atypical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> In the canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> lipopolysaccharides (LPS), tumor necrosis factor α (TNFα) orinterleukin-1 (IL-1) activate Toll-like receptors (TLRs), tumor necrosis factor receptor (TNFR) and interleukin-1 receptor (IL-1R), respectively. Through a variety of adapter proteins and <span class="highlight" style="background-color:">signaling</span> kinases this leads to an activation of IKKβ in the IKK complex, which can then phosphorylate IκBα on Serine residues S32 and S36. This phosphorylation is a prerequisite for its subsequent polyubiquitination, which in turn results in proteasomal degradation of IκBα. NF-κB homo- or heterodimers can then translocate to nucleus and activate target gene transcription. <b>(B)</b> In the non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, activation of B-cell activation factor (BAFFR), CD40, receptor activator for nuclear factor kappa B (RANK) or lymphtoxin β-receptor (LTβR), leads to activation of IKKα by the NF-κB-inducing kinase (NIK). IKKα can the phosphorylate p100 on serine residues S866 and S870. This phosphorylation leads to polyubiquitination of p100 and its subsequent proteasomal processing to p52.p52-RelB heterodimers can then activate transcription of target genes. <b>(C)</b> In the atypical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, genotoxic stress leads to a translocation of NEMO to the nucleus where it is sumoylated and subsequently ubiquitinated. This process is mediated by the ataxia telangiectasia mutated (ATM) checkpoint kinase. NEMO and ATM can then return to the cytosol where they activate IKKβ.</div></p></div><div class="aux"><div class="resc">Bastian Hoesel, et al. Mol Cancer. 2013;12:86-86.</div><p class="links"><a class="dblinks" href="/pubmed/23915189" ref="ordinalpos=2846&amp;ncbi_uid=5225784&amp;link_uid=23915189">Citation</a><a class="dblinks" href="/pmc/articles/PMC3750319/" ref="ordinalpos=2846&amp;ncbi_uid=5225784&amp;link_uid=PMC3750319">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2847.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4204167/figure/f2-jcp-19-111/" ref="ordinalpos=2847&amp;ncbi_uid=6742794&amp;link_uid=PMC4204167" href="/pmc/articles/PMC4204167/"><img src="/pmc/articles/instance/4204167/bin/jcp-19-111f2.gif" src-large="/pmc/articles/instance/4204167/bin/jcp-19-111f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4204167/" ref="ordinalpos=2847&amp;ncbi_uid=6742794&amp;link_uid=PMC4204167" image-link="/pmc/articles/PMC4204167/figure/f2-jcp-19-111/" class="imagepopup">Figure 2. From: Targeting Nrf2 <span class="highlight" style="background-color:">Signaling</span> to Combat Chemoresistance. </a></p><div class="supp"><p class="details"><div>The Kelch-like ECH-associated protein 1 (Keap1)-nuclear factor E2-related factor 2 (Nrf2)-antioxidant response elements (ARE) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cul3, Cullin E3 ubiquitin ligase; Ubi, uniquitin; Maf, musculo-aponeurotic fibrosarcoma; HO-1, heme oxygenase-1; NQO1, NAD(P)H: quinone oxidoreductase 1; ABC, ATP binding cassette; MRP, multidrug resistance-associated protein.</div></p></div><div class="aux"><div class="resc">Jae Hong No, et al. J Cancer Prev. 2014 June;19(2):111-117.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2847&amp;ncbi_uid=6742794&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4204167/" ref="ordinalpos=2847&amp;ncbi_uid=6742794&amp;link_uid=PMC4204167">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2848.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678388/figure/F3/" ref="ordinalpos=2848&amp;ncbi_uid=4983341&amp;link_uid=PMC3678388" href="/pmc/articles/PMC3678388/"><img src="/pmc/articles/instance/3678388/bin/nihms362811f3.gif" src-large="/pmc/articles/instance/3678388/bin/nihms362811f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678388/" ref="ordinalpos=2848&amp;ncbi_uid=4983341&amp;link_uid=PMC3678388" image-link="/pmc/articles/PMC3678388/figure/F3/" class="imagepopup">Figure 3.  From: ATR <span class="highlight" style="background-color:">signaling</span>: more than meeting at the fork. </a></p><div class="supp"><p class="details"><div>ATR recruitment to some DNA base lesions proceeds through a mismatch repair-dependent mechanism.<br />Mismatch repair proteins may contribute directly and indirectly to ATR recognition of some times of DNA damage. Direct interactions between the ATR-ATRIP complex and MUTSα and MUTLα proteins may promote activation of ATR independently of RPA and single-stranded DNA. Alternatively, mismatch repair-dependent DNA processing may promote the RPA-dependent canonical ATR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Edward A. Nam, et al. Biochem J. ;436(3):527-536.</div><p class="links"><a class="dblinks" href="/pubmed/21615334" ref="ordinalpos=2848&amp;ncbi_uid=4983341&amp;link_uid=21615334">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678388/" ref="ordinalpos=2848&amp;ncbi_uid=4983341&amp;link_uid=PMC3678388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2849.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1415321/figure/fig12/" ref="ordinalpos=2849&amp;ncbi_uid=718553&amp;link_uid=PMC1415321" href="/pmc/articles/PMC1415321/"><img src="/pmc/articles/instance/1415321/bin/zmk0040675900012.gif" src-large="/pmc/articles/instance/1415321/bin/zmk0040675900012.jpg" alt="Figure 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1415321/" ref="ordinalpos=2849&amp;ncbi_uid=718553&amp;link_uid=PMC1415321" image-link="/pmc/articles/PMC1415321/figure/fig12/" class="imagepopup">Figure 12. From: Cell Surface Transglutaminase Promotes RhoA Activation via Integrin Clustering and Suppression of the Src-p190RhoGAP <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Summary of nonenzymatic effects of cell surface tTG on RhoA activity. Activation of RhoA by integrin-associated tTG is mediated by formation of integrin clusters and down-regulation of the Src-p190RhoGAP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The light, gray, and dark ovals, rectangles, and squares for Src, p190RhoGAP, and RhoA reflect their relative activities in the order of increase.</div></p></div><div class="aux"><div class="resc">Anna Janiak, et al. Mol Biol Cell. 2006 April;17(4):1606-1619.</div><p class="links"><a class="dblinks" href="/pubmed/16452636" ref="ordinalpos=2849&amp;ncbi_uid=718553&amp;link_uid=16452636">Citation</a><a class="dblinks" href="/pmc/articles/PMC1415321/" ref="ordinalpos=2849&amp;ncbi_uid=718553&amp;link_uid=PMC1415321">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2850.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3243033/figure/F1/" ref="ordinalpos=2850&amp;ncbi_uid=3736730&amp;link_uid=PMC3243033" href="/pmc/articles/PMC3243033/"><img src="/pmc/articles/instance/3243033/bin/org0703_0202_fig001.gif" src-large="/pmc/articles/instance/3243033/bin/org0703_0202_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3243033/" ref="ordinalpos=2850&amp;ncbi_uid=3736730&amp;link_uid=PMC3243033" image-link="/pmc/articles/PMC3243033/figure/F1/" class="imagepopup">Figure 1.  From: Planar cell polarity <span class="highlight" style="background-color:">pathway</span> in vertebrate epidermal development, homeostasis and repair. </a></p><div class="supp"><p class="details"><div>Regulation of epidermal homeostasis by PCP <span class="highlight" style="background-color:">signaling</span>. As a part of this <span class="highlight" style="background-color:">pathway</span>, GRHL3 is known to positively induce wound healing via activation of RhoGEF/RhoA axis. Furthermore, we hypothesize its involvement as a negative regulator of skin biology and pathology, including stem cell proliferation, inflammation and cancer.</div></p></div><div class="aux"><div class="resc">Sebastian Dworkin, et al. Organogenesis. 2011 Jul-Sep;7(3):202-208.</div><p class="links"><a class="dblinks" href="/pubmed/22041517" ref="ordinalpos=2850&amp;ncbi_uid=3736730&amp;link_uid=22041517">Citation</a><a class="dblinks" href="/pmc/articles/PMC3243033/" ref="ordinalpos=2850&amp;ncbi_uid=3736730&amp;link_uid=PMC3243033">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2851.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2674123/figure/F10/" ref="ordinalpos=2851&amp;ncbi_uid=1765646&amp;link_uid=PMC2674123" href="/pmc/articles/PMC2674123/"><img src="/pmc/articles/instance/2674123/bin/nihms-80058-f0011.gif" src-large="/pmc/articles/instance/2674123/bin/nihms-80058-f0011.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2674123/" ref="ordinalpos=2851&amp;ncbi_uid=1765646&amp;link_uid=PMC2674123" image-link="/pmc/articles/PMC2674123/figure/F10/" class="imagepopup">Figure 10.  From: Comparative Proteomic Study of Two Closely Related Ovarian Endometrioid Adenocarcinoma Cell Lines Using cIEF Fractionation and <span class="highlight" style="background-color:">Pathway</span> Analysis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> cascade of PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. Green nodes represent over-expression in MDAH-2774 and red nodes represent over-expression in TOV-112D. Plain nodes are imported from IPKB. This figure has been manually modified from integrative analysis by adding some proteins which were identified from only one cell line but did not meet the threshold of integrative analysis.</div></p></div><div class="aux"><div class="resc">Lan Dai, et al. Electrophoresis. ;30(7):1119-1131.</div><p class="links"><a class="dblinks" href="/pubmed/19288585" ref="ordinalpos=2851&amp;ncbi_uid=1765646&amp;link_uid=19288585">Citation</a><a class="dblinks" href="/pmc/articles/PMC2674123/" ref="ordinalpos=2851&amp;ncbi_uid=1765646&amp;link_uid=PMC2674123">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2852.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2894853/figure/pone-0011154-g002/" ref="ordinalpos=2852&amp;ncbi_uid=2379547&amp;link_uid=PMC2894853" href="/pmc/articles/PMC2894853/"><img src="/pmc/articles/instance/2894853/bin/pone.0011154.g002.gif" src-large="/pmc/articles/instance/2894853/bin/pone.0011154.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2894853/" ref="ordinalpos=2852&amp;ncbi_uid=2379547&amp;link_uid=PMC2894853" image-link="/pmc/articles/PMC2894853/figure/pone-0011154-g002/" class="imagepopup">Figure 2. Impact of disease-associated microRNAs on <span class="highlight" style="background-color:">signaling</span> pathways..  From: Tissue-Specific Target Analysis of Disease-Associated MicroRNAs in Human <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Enrichment for a particular disease and <span class="highlight" style="background-color:">pathway</span> was calculated by a LOD score. A positive score indicates an enrichment of microRNA targets for a disease-<span class="highlight" style="background-color:">pathway</span> interaction. Negative scores indicate depletion. <b>A</b>: Heatmap of microRNA target enrichment for a particular disease and <span class="highlight" style="background-color:">pathway</span>. Pathways and diseases are ordered by hierarchical clustering using Manhattan distance and ward clustering. <b>B</b>: Boxplot of disease-<span class="highlight" style="background-color:">pathway</span> associations ordered according to hierarchical clustering along the pathways. Red fields indicate an enrichments and blue a depletion. White fields indicate that no microRNA targets were found for this disease-<span class="highlight" style="background-color:">pathway</span> association.</div></p></div><div class="aux"><div class="resc">Andreas Kowarsch, et al. PLoS One. 2010;5(6):e11154.</div><p class="links"><a class="dblinks" href="/pubmed/20614023" ref="ordinalpos=2852&amp;ncbi_uid=2379547&amp;link_uid=20614023">Citation</a><a class="dblinks" href="/pmc/articles/PMC2894853/" ref="ordinalpos=2852&amp;ncbi_uid=2379547&amp;link_uid=PMC2894853">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2853.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3077116/figure/F2/" ref="ordinalpos=2853&amp;ncbi_uid=3521707&amp;link_uid=PMC3077116" href="/pmc/articles/PMC3077116/"><img src="/pmc/articles/instance/3077116/bin/nihms255742f2.gif" src-large="/pmc/articles/instance/3077116/bin/nihms255742f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3077116/" ref="ordinalpos=2853&amp;ncbi_uid=3521707&amp;link_uid=PMC3077116" image-link="/pmc/articles/PMC3077116/figure/F2/" class="imagepopup">Figure 2. Peroxisomal fatty acid oxidation and anaplerotic reactions drive the RTG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of cellular aging regulation.  From: Peroxisome Metabolism and Cellular Aging. </a></p><div class="supp"><p class="details"><div>Functionally compromised mitochondria trigger the RTG <span class="highlight" style="background-color:">pathway</span> in replicatively aging yeast. Gene transcription ensues, altering cell metabolism, stimulating peroxisome function, and maintaining cell viability. Please see text for details.</div></p></div><div class="aux"><div class="resc">Vladimir I. Titorenko, et al. Traffic. ;12(3):252-259.</div><p class="links"><a class="dblinks" href="/pubmed/21083858" ref="ordinalpos=2853&amp;ncbi_uid=3521707&amp;link_uid=21083858">Citation</a><a class="dblinks" href="/pmc/articles/PMC3077116/" ref="ordinalpos=2853&amp;ncbi_uid=3521707&amp;link_uid=PMC3077116">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2854.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3727099/figure/F2/" ref="ordinalpos=2854&amp;ncbi_uid=6335003&amp;link_uid=PMC3727099" href="/pmc/articles/PMC3727099/"><img src="/pmc/articles/instance/3727099/bin/zh40131307840002.gif" src-large="/pmc/articles/instance/3727099/bin/zh40131307840002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3727099/" ref="ordinalpos=2854&amp;ncbi_uid=6335003&amp;link_uid=PMC3727099" image-link="/pmc/articles/PMC3727099/figure/F2/" class="imagepopup">Fig. 2. From: Adenylyl cyclase type 5 in cardiac disease, metabolism, and aging. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> diagram for AC5 mediating cardiac stress and longevity. <i>Left</i>: cardiac dysfunction: <span class="highlight" style="background-color:">signaling</span> diagram for AC5 regulation of MnSOD transcriptionally through the sirtuin 1/forkhead box O3a (SIRT1/FoxO3a) <span class="highlight" style="background-color:">pathway</span> is shown. Imbalance between reactive oxygen species production and the intracellular antioxidant system results in the intolerance of AC5-Tg to stress (41). <i>Right</i>: longevity: disruption of AC5 activates the Raf/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The activation of ERK activates antioxidative stress and cell survival mechanism, which leads to longevity in AC5-KO mice (85). Arrows indicate the direction of <span class="highlight" style="background-color:">signaling</span>. Figures used are modified with permission from Lai et al. (41) and Yan et al. (85).</div></p></div><div class="aux"><div class="resc">Stephen F. Vatner, et al. Am J Physiol Heart Circ Physiol. 2013 July 1;305(1):H1-H8.</div><p class="links"><a class="dblinks" href="/pubmed/23624627" ref="ordinalpos=2854&amp;ncbi_uid=6335003&amp;link_uid=23624627">Citation</a><a class="dblinks" href="/pmc/articles/PMC3727099/" ref="ordinalpos=2854&amp;ncbi_uid=6335003&amp;link_uid=PMC3727099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2855.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1618798/figure/F3/" ref="ordinalpos=2855&amp;ncbi_uid=909896&amp;link_uid=PMC1618798" href="/pmc/articles/PMC1618798/"><img src="/pmc/articles/instance/1618798/bin/nihms-12625-0003.gif" src-large="/pmc/articles/instance/1618798/bin/nihms-12625-0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1618798/" ref="ordinalpos=2855&amp;ncbi_uid=909896&amp;link_uid=PMC1618798" image-link="/pmc/articles/PMC1618798/figure/F3/" class="imagepopup">Fig. 3.  From: The Paradoxical Pro- and Anti-apoptotic Actions of GSK3 in the Intrinsic and Extrinsic Apoptosis <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>The extrinsic apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Depiction of the extrinsic apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The four major apoptosis-inducing death receptors are activated when Fas ligand (FasL) activates Fas, TRAIL activates DR4 or DR5, or TNF activates TNF-R1 (left inset). The binding of the ligand to the death receptor induces trimerization of the receptor which produces a conformation that recruits FADD and procaspase-8 to the cytoplasmic tail of the receptor, altogether forming a protein complex known as the DISC. Within the DISC, caspase-8 is activated by auto-cleavage. In type I cells, sufficient active caspase-8 is generated to directly activate caspase-3 to carry out the apoptotic program. In type II cells, activation of caspase-8 leads to activation of caspase-3 indirectly through cleavage and activation of Bid, forming tBid, which activates the mitochondrial apoptotic mechanisms that are involved in the intrinsic apoptotic <span class="highlight" style="background-color:">pathway</span>. (B) GSK3 inhibits the extrinsic apoptotic <span class="highlight" style="background-color:">pathway</span> by impairing transduction of the signal from activated death receptors to the activation of caspase-8.</div></p></div><div class="aux"><div class="resc">Eléonore Beurel, et al. Prog Neurobiol. ;79(4):173-189.</div><p class="links"><a class="dblinks" href="/pubmed/16935409" ref="ordinalpos=2855&amp;ncbi_uid=909896&amp;link_uid=16935409">Citation</a><a class="dblinks" href="/pmc/articles/PMC1618798/" ref="ordinalpos=2855&amp;ncbi_uid=909896&amp;link_uid=PMC1618798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2856.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3690396/figure/F7/" ref="ordinalpos=2856&amp;ncbi_uid=5549264&amp;link_uid=PMC3690396" href="/pmc/articles/PMC3690396/"><img src="/pmc/articles/instance/3690396/bin/1223fig7.gif" src-large="/pmc/articles/instance/3690396/bin/1223fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3690396/" ref="ordinalpos=2856&amp;ncbi_uid=5549264&amp;link_uid=PMC3690396" image-link="/pmc/articles/PMC3690396/figure/F7/" class="imagepopup">Figure 7. From: Protein kinase A activates the Hippo <span class="highlight" style="background-color:">pathway</span> to modulate cell proliferation and differentiation. </a></p><div class="supp"><p class="details"><div>Regulation of the Hippo <span class="highlight" style="background-color:">pathway</span> by cAMP-PKA <span class="highlight" style="background-color:">signaling</span>. Upon stimulation of Gα<sub>s</sub>-coupled GPCR, activation of PKA by cAMP leads to inhibition of Rho GTPases, which indirectly inhibit Lats kinase activity. Stimulation of Gα<sub>12/13</sub>- or Gα<sub>q/11</sub>-coupled receptors antagonize the effect of cAMP or PKA on YAP phosphorylation by inducing Rho GTPases. Inhibition of YAP and TAZ mediates functions of cAMP and PKA on adipogenesis, cell proliferation, and apoptosis.</div></p></div><div class="aux"><div class="resc">Fa-Xing Yu, et al. Genes Dev. 2013 June 1;27(11):1223-1232.</div><p class="links"><a class="dblinks" href="/pubmed/23752589" ref="ordinalpos=2856&amp;ncbi_uid=5549264&amp;link_uid=23752589">Citation</a><a class="dblinks" href="/pmc/articles/PMC3690396/" ref="ordinalpos=2856&amp;ncbi_uid=5549264&amp;link_uid=PMC3690396">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2857.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4047873/figure/fig8/" ref="ordinalpos=2857&amp;ncbi_uid=6244382&amp;link_uid=PMC4047873" href="/pmc/articles/PMC4047873/"><img src="/pmc/articles/instance/4047873/bin/cddis2014229f8.gif" src-large="/pmc/articles/instance/4047873/bin/cddis2014229f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4047873/" ref="ordinalpos=2857&amp;ncbi_uid=6244382&amp;link_uid=PMC4047873" image-link="/pmc/articles/PMC4047873/figure/fig8/" class="imagepopup">Figure 8.  From: Sulfur dioxide inhibits vascular smooth muscle cell proliferation via suppressing the Erk/MAP kinase <span class="highlight" style="background-color:">pathway</span> mediated by cAMP/PKA <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>A schematic diagram of the inhibition of proliferation by SO<sub>2</sub> and the pathways. PDGF-BB can promote VSMC proliferation by activating Erk/MAPK <span class="highlight" style="background-color:">pathway</span> after combining with its specific RTKs. The increasing cAMP concentration in VSMCs induces PKA activation, which can disturb Erk/MAPK <span class="highlight" style="background-color:">signaling</span> transduction by inhibiting c-Raf activation. SO<sub>2</sub> derivatives can suppress the activities of Erk1/2, MEK1/2 and c-Raf, which does not result from separate targeting interferences of SO<sub>2</sub> derivatives on the three molecules but from the inhibition of upstream <span class="highlight" style="background-color:">signaling</span> by SO<sub>2</sub> derivatives. SO<sub>2</sub> derivatives might enhance adenylyl cyclase activity to elevate cAMP production from ATP and then activate PKA signal to block c-Raf activation, which finally contributes to the suppression of PDGF-BB-induced VSMC proliferation by SO<sub>2</sub> derivatives</div></p></div><div class="aux"><div class="resc">D Liu, et al. Cell Death Dis. 2014 May;5(5):e1251.</div><p class="links"><a class="dblinks" href="/pubmed/24853429" ref="ordinalpos=2857&amp;ncbi_uid=6244382&amp;link_uid=24853429">Citation</a><a class="dblinks" href="/pmc/articles/PMC4047873/" ref="ordinalpos=2857&amp;ncbi_uid=6244382&amp;link_uid=PMC4047873">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2858.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2743626/figure/F6/" ref="ordinalpos=2858&amp;ncbi_uid=2014886&amp;link_uid=PMC2743626" href="/pmc/articles/PMC2743626/"><img src="/pmc/articles/instance/2743626/bin/zmk0180991870006.gif" src-large="/pmc/articles/instance/2743626/bin/zmk0180991870006.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2743626/" ref="ordinalpos=2858&amp;ncbi_uid=2014886&amp;link_uid=PMC2743626" image-link="/pmc/articles/PMC2743626/figure/F6/" class="imagepopup">Figure 6. From: Dissection of a Hypoxia-induced, Nitric Oxide-mediated <span class="highlight" style="background-color:">Signaling</span> Cascade. </a></p><div class="supp"><p class="details"><div>Model for hypoxia-induced <span class="highlight" style="background-color:">signaling</span>. During normoxia (white area), activity of NOS presumably generates a pool of NO, possibly nitrite, which is stored in the mitochondria. During shortfalls of oxygen (hypoxia, shaded area), this pool gets converted back to NO. NO is present in the cytoplasm as GSNO. In the cytoplasm, generation of cGMP through sGC subsequently activates cGMP-dependent channels, resulting in mobilization of calcium through these channels. The two most novel steps in this proposed <span class="highlight" style="background-color:">pathway</span> are the production of NO by mitochondrial reduction and the activation of sGC by GSNO. The reductive production of NO by the mitochondrial has been documented in other contexts (Kozlov et al., 1999; Nohl et al., 2000). The activation of sGC by GSNO is suggested by our data and a recently recognized diversity in the mechanisms regulating sGCs leads us to suggest an activating input by transnitrosylation (see <i>Discussion</i>). Activation of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may allow adaptation to conditions of hypoxia. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as delineated in this article was defined by our data using S2 cells.</div></p></div><div class="aux"><div class="resc">Pascale F. Dijkers, et al. Mol Biol Cell. 2009 September 15;20(18):4083-4090.</div><p class="links"><a class="dblinks" href="/pubmed/19625446" ref="ordinalpos=2858&amp;ncbi_uid=2014886&amp;link_uid=19625446">Citation</a><a class="dblinks" href="/pmc/articles/PMC2743626/" ref="ordinalpos=2858&amp;ncbi_uid=2014886&amp;link_uid=PMC2743626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2859.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1618798/figure/F2/" ref="ordinalpos=2859&amp;ncbi_uid=909900&amp;link_uid=PMC1618798" href="/pmc/articles/PMC1618798/"><img src="/pmc/articles/instance/1618798/bin/nihms-12625-0002.gif" src-large="/pmc/articles/instance/1618798/bin/nihms-12625-0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1618798/" ref="ordinalpos=2859&amp;ncbi_uid=909900&amp;link_uid=PMC1618798" image-link="/pmc/articles/PMC1618798/figure/F2/" class="imagepopup">Fig. 2.  From: The Paradoxical Pro- and Anti-apoptotic Actions of GSK3 in the Intrinsic and Extrinsic Apoptosis <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>The intrinsic apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Depiction of the intrinsic apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In response to cell damaging insults, pro-apoptotic members of the bcl-2 family (e.g., Bax, Bim) are activated and they translocate to the mitochondria to neutralize anti-apoptotic proteins (e.g., bcl-2, Mcl-1, Bcl-xL). This results in disruption of mitochondria which triggers the release of pro-apoptotic molecules from the mitochondrial intermembrane space. Released cytochrome c clusters with APAF-1 and procaspase-9 in the presence of dATP to form the apoptosome to activate caspase-9. Activated caspase-9 cleaves and activates caspase-3, triggering a caspase cascade which ultimately results in the death of the cell. (B) GSK3 promotes the intrinsic apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> be regulating transcription factors that control the expression of pro- and anti-apoptotic proteins, by promoting microtubule disruption and cell structural changes that occur during apoptosis, and by promoting disruption of mitochondria.</div></p></div><div class="aux"><div class="resc">Eléonore Beurel, et al. Prog Neurobiol. ;79(4):173-189.</div><p class="links"><a class="dblinks" href="/pubmed/16935409" ref="ordinalpos=2859&amp;ncbi_uid=909900&amp;link_uid=16935409">Citation</a><a class="dblinks" href="/pmc/articles/PMC1618798/" ref="ordinalpos=2859&amp;ncbi_uid=909900&amp;link_uid=PMC1618798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2860.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3147278/figure/F3/" ref="ordinalpos=2860&amp;ncbi_uid=3458270&amp;link_uid=PMC3147278" href="/pmc/articles/PMC3147278/"><img src="/pmc/articles/instance/3147278/bin/nihms-308901-f0003.gif" src-large="/pmc/articles/instance/3147278/bin/nihms-308901-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3147278/" ref="ordinalpos=2860&amp;ncbi_uid=3458270&amp;link_uid=PMC3147278" image-link="/pmc/articles/PMC3147278/figure/F3/" class="imagepopup">Figure 3.  From: Interaction between Interleukin-1 Receptor 2 and Toll-like Receptor 4, and Cervical Cytokines. </a></p><div class="supp"><p class="details"><div>Cross-talk between TLR4 and IL-1 <span class="highlight" style="background-color:">signaling</span> pathways. Activation of TLR4 by lipopolysaccharide (LPS) initiates a signal through the Janus kinase signal transducer and activator of transcription 1 (JAK-STAT1) <span class="highlight" style="background-color:">pathway</span> as well as through tumor necrosis factor receptor-associated factor 6 (TRAF6). IL-1 signal transduction is initiated through the TRAF6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by the binding of either IL-1α or IL-1β to the IL-1R1/IL-1RAP complex. The IL-1R2/IL-1RAP complex competes for the binding of IL-1α and IL-1β and does not produce a signal when bound, therefore limiting the availability of IL-1α or IL-1β to bind to the IL-1R1/IL-1RAP complex. A dashed line indicates that other <span class="highlight" style="background-color:">signaling</span> molecules not shown are also involved in the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Kelli K. Ryckman, et al. J Reprod Immunol. ;90(2):220-226.</div><p class="links"><a class="dblinks" href="/pubmed/21704385" ref="ordinalpos=2860&amp;ncbi_uid=3458270&amp;link_uid=21704385">Citation</a><a class="dblinks" href="/pmc/articles/PMC3147278/" ref="ordinalpos=2860&amp;ncbi_uid=3458270&amp;link_uid=PMC3147278">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2861.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC551522/figure/F1/" ref="ordinalpos=2861&amp;ncbi_uid=497131&amp;link_uid=PMC551522" href="/pmc/articles/PMC551522/"><img src="/pmc/articles/instance/551522/bin/jbiol22-1.gif" src-large="/pmc/articles/instance/551522/bin/jbiol22-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC551522/" ref="ordinalpos=2861&amp;ncbi_uid=497131&amp;link_uid=PMC551522" image-link="/pmc/articles/PMC551522/figure/F1/" class="imagepopup">Figure 1.  From: Dishevelled and Wnt <span class="highlight" style="background-color:">signaling</span>: is the nucleus the final frontier?. </a></p><div class="supp"><p class="details"><div>A schematic representation of the Wnt signal transduction cascade. <b>(a) </b>For the canonical <span class="highlight" style="background-color:">pathway</span>, <span class="highlight" style="background-color:">signaling</span> through the Frizzled (Fz) and LRP5/6 receptor complex induces the stabilization of β-catenin via the DIX and PDZ domains of Dishevelled (Dsh) and a number of factors including Axin, glycogen synthase kinase 3 (GSK3) and casein kinase 1 (CK1). β-catenin translocates into the nucleus where it complexes with members of the LEF/TCF family of transcription factors to mediate transcriptional induction of target genes. β-catenin is then exported from the nucleus and degraded via the proteosomal machinery. <b>(b) </b>For non-canonical or planar cell polarity (PCP) <span class="highlight" style="background-color:">signaling</span>, Wnt <span class="highlight" style="background-color:">signaling</span> is transduced through Frizzled independent of LPR5/6. Utilizing the PDZ and DEP domains of Dsh, this <span class="highlight" style="background-color:">pathway</span> mediates cytoskeletal changes through activation of the small GTPases Rho and Rac. <b>(c) </b>For the Wnt-Ca<sup>2+ </sup><span class="highlight" style="background-color:">pathway</span>, Wnt <span class="highlight" style="background-color:">signaling</span> via Frizzled mediates activation of heterotrimeric G-proteins, which engage Dsh, phospholipase C (PLC; not shown), calcium-calmodulin kinase 2 (CamK2) and protein kinase C (PKC). This <span class="highlight" style="background-color:">pathway</span> also uses the PDZ and DEP domains of Dsh to modulate cell adhesion and motility. Note that for the PCP and Ca<sup>2+ </sup>pathways Dsh is proposed to function at the membrane, whereas for canonical <span class="highlight" style="background-color:">signaling</span> Dsh has been proposed to function in the cytoplasm; a recent study [10] implicates nuclear localization of Dsh in this <span class="highlight" style="background-color:">pathway</span>. See text for further details.</div></p></div><div class="aux"><div class="resc">Raymond Habas, et al. J Biol. 2005;4(1):2-2.</div><p class="links"><a class="dblinks" href="/pubmed/15720723" ref="ordinalpos=2861&amp;ncbi_uid=497131&amp;link_uid=15720723">Citation</a><a class="dblinks" href="/pmc/articles/PMC551522/" ref="ordinalpos=2861&amp;ncbi_uid=497131&amp;link_uid=PMC551522">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2862.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC150771/figure/fig8/" ref="ordinalpos=2862&amp;ncbi_uid=238063&amp;link_uid=PMC150771" href="/pmc/articles/PMC150771/"><img src="/pmc/articles/instance/150771/bin/003376f8.gif" src-large="/pmc/articles/instance/150771/bin/003376f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC150771/" ref="ordinalpos=2862&amp;ncbi_uid=238063&amp;link_uid=PMC150771" image-link="/pmc/articles/PMC150771/figure/fig8/" class="imagepopup">Figure 8. From: Molecular Dissection of the Gibberellin/Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> Pathways by Transiently Expressed RNA Interference in Barley Aleurone Cells. </a></p><div class="supp"><p class="details"><div>Important Steps Involved in the Hormonal Regulation of α-Amylase in Aleurone Cells.<br />A repressor, SLN1, and an activator, GAMyb, are essential components in the GA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to the induction of α-amylase. An alternative route from ABA to the site of interaction with the GA <span class="highlight" style="background-color:">signaling</span> cascade (represented by X) is proposed to indicate that the transcriptional induction of <i>PKABA1</i> is sufficient, but not necessary, to block α-amylase expression.</div></p></div><div class="aux"><div class="resc">Rodolfo Zentella, et al. Plant Cell. 2002 September;14(9):2289-2301.</div><p class="links"><a class="dblinks" href="/pubmed/12215521" ref="ordinalpos=2862&amp;ncbi_uid=238063&amp;link_uid=12215521">Citation</a><a class="dblinks" href="/pmc/articles/PMC150771/" ref="ordinalpos=2862&amp;ncbi_uid=238063&amp;link_uid=PMC150771">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2863.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3221273/figure/F7/" ref="ordinalpos=2863&amp;ncbi_uid=3735926&amp;link_uid=PMC3221273" href="/pmc/articles/PMC3221273/"><img src="/pmc/articles/instance/3221273/bin/cpb0027-0715-f07.gif" src-large="/pmc/articles/instance/3221273/bin/cpb0027-0715-f07.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3221273/" ref="ordinalpos=2863&amp;ncbi_uid=3735926&amp;link_uid=PMC3221273" image-link="/pmc/articles/PMC3221273/figure/F7/" class="imagepopup">Fig. 7.  From: Polycystin-1, 2, and STIM1 Interact with IP3R to Modulate ER Ca2+ Release through the PI3K/Akt <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed model. A. In the cell without PC1 expression, the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is not active, and PC2 is retained in the ER. Extracellular signals that activate the phospholipase C (PLC) <span class="highlight" style="background-color:">pathway</span> induce production of IP<sub>3</sub> to activate the IP<sub>3</sub>R. Activation of the IP<sub>3</sub>R releases Ca<sup>2+</sup> from within the ER to cytosol. PC2, which is retained in the ER interacts with IP<sub>3</sub>R and facilitates the release of Ca<sup>2+</sup>. Enhancement of Ca<sup>2+</sup> release by PC2-IP<sub>3</sub>R interaction eventually leads to Ca<sup>2+</sup> depletion in the ER and activation of the store-operated Ca<sup>2+</sup> influx (SOC). Blue arrow represents Ca<sup>2+</sup> movement, and orange arrow represents activation of inhibition (arrow with flat head) of cellular activities. B. In the cell with PC1 expression, the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is activated. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to an increase of STIM1-IP<sub>3</sub>R complex, reduction of the PC2-IP<sub>3</sub>R interaction in the ER, and probably translocation of PC2 from the ER to the plasma membrane. PC2 interacts with PC1 at the plasma membrane to serve as a Ca<sup>2+</sup> influx channel. Extracellular signals that activate the PLC <span class="highlight" style="background-color:">pathway</span> induce production of IP<sub>3</sub> to activate the IP<sub>3</sub>R. Activation of the IP<sub>3</sub>R will stimulate Ca<sup>2+</sup> release, however, reduction of the PC2-IP<sub>3</sub>R interaction and increased the association of STIM1-IP<sub>3</sub>R inhibit release of intracellular Ca<sup>2+</sup> that eventually keeps the Ca<sup>2+</sup> level within the ER high, and therefore inactivate the SOC.</div></p></div><div class="aux"><div class="resc">Netty G. Santoso, et al. Cell Physiol Biochem. 2011 June;27(6):715-726.</div><p class="links"><a class="dblinks" href="/pubmed/21691089" ref="ordinalpos=2863&amp;ncbi_uid=3735926&amp;link_uid=21691089">Citation</a><a class="dblinks" href="/pmc/articles/PMC3221273/" ref="ordinalpos=2863&amp;ncbi_uid=3735926&amp;link_uid=PMC3221273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2864.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3978385/figure/F3/" ref="ordinalpos=2864&amp;ncbi_uid=6006524&amp;link_uid=PMC3978385" href="/pmc/articles/PMC3978385/"><img src="/pmc/articles/instance/3978385/bin/nihms-570892-f0003.gif" src-large="/pmc/articles/instance/3978385/bin/nihms-570892-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3978385/" ref="ordinalpos=2864&amp;ncbi_uid=6006524&amp;link_uid=PMC3978385" image-link="/pmc/articles/PMC3978385/figure/F3/" class="imagepopup">Figure 3. Raf isoform switching in NRAS mutant melanoma.  From: NRAS mutant melanoma: biological behavior and future strategies for therapeutic management. </a></p><div class="supp"><p class="details"><div><b>A.</b> In normal melanocytes, cyclic AMP <span class="highlight" style="background-color:">pathway</span> activation promotes PKA-mediated inhibition of CRAF. Therefore MAPK <span class="highlight" style="background-color:">signaling</span> proceeds through BRAF rather than CRAF in these cells (left panel). <b>B.</b> In <i>NRAS</i> mutant melanoma, inhibition of PKA <span class="highlight" style="background-color:">signaling</span> due to enzymatic activity of phosphodiesterase (PDE) and negative feedback inhibition of BRAF due to activation of the MAPK <span class="highlight" style="background-color:">pathway</span> promotes CRAF-mediated MAPK <span class="highlight" style="background-color:">signaling</span> instead (right panel).</div></p></div><div class="aux"><div class="resc">Inna V. Fedorenko, et al. Oncogene. 2013 June 20;32(25):3009-3018.</div><p class="links"><a class="dblinks" href="/pubmed/23069660" ref="ordinalpos=2864&amp;ncbi_uid=6006524&amp;link_uid=23069660">Citation</a><a class="dblinks" href="/pmc/articles/PMC3978385/" ref="ordinalpos=2864&amp;ncbi_uid=6006524&amp;link_uid=PMC3978385">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2865.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3152260/figure/F10/" ref="ordinalpos=2865&amp;ncbi_uid=3150402&amp;link_uid=PMC3152260" href="/pmc/articles/PMC3152260/"><img src="/pmc/articles/instance/3152260/bin/nihms306819f10.gif" src-large="/pmc/articles/instance/3152260/bin/nihms306819f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3152260/" ref="ordinalpos=2865&amp;ncbi_uid=3150402&amp;link_uid=PMC3152260" image-link="/pmc/articles/PMC3152260/figure/F10/" class="imagepopup">Figure 10.  From: Opposing Actions of Insulin and Arsenite Converge on PKC? to Alter Keratinocyte Proliferative Potential and Differentiation. </a></p><div class="supp"><p class="details"><div>Model illustrating interrelations of insulin, PKCδ and arsenite. Activation of the insulin/IGF receptor <span class="highlight" style="background-color:">pathway</span> appears to suppress EGFR <span class="highlight" style="background-color:">signaling</span>, an action prevented by arsenite. Insulin/IGFR <span class="highlight" style="background-color:">signaling</span> also stimulates the PI3-kinase <span class="highlight" style="background-color:">pathway</span>, including mTOR <span class="highlight" style="background-color:">signaling</span>. These pathways converge on PKCδ, a critical determinant of cell state. Arsenite and PI3-kinase or mTOR inhibitor cotreatment thus has an additive response in maintaining proliferative potential and yields PKCδ levels equivalent to insulin deprivation. Arrows indicate activation, and bars indicate suppression. Chemical inhibitors are in italics.</div></p></div><div class="aux"><div class="resc">Tatiana V. Reznikova, et al. Mol Carcinog. ;49(4):398-409.</div><p class="links"><a class="dblinks" href="/pubmed/20082316" ref="ordinalpos=2865&amp;ncbi_uid=3150402&amp;link_uid=20082316">Citation</a><a class="dblinks" href="/pmc/articles/PMC3152260/" ref="ordinalpos=2865&amp;ncbi_uid=3150402&amp;link_uid=PMC3152260">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2866.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1821078/figure/F6/" ref="ordinalpos=2866&amp;ncbi_uid=992897&amp;link_uid=PMC1821078" href="/pmc/articles/PMC1821078/"><img src="/pmc/articles/instance/1821078/bin/nihms16116f6.gif" src-large="/pmc/articles/instance/1821078/bin/nihms16116f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1821078/" ref="ordinalpos=2866&amp;ncbi_uid=992897&amp;link_uid=PMC1821078" image-link="/pmc/articles/PMC1821078/figure/F6/" class="imagepopup">Fig. 6.  From: Changes in Wnt/?-Catenin <span class="highlight" style="background-color:">Pathway</span> During Regulated Growth in Rat Liver Regeneration. </a></p><div class="supp"><p class="details"><div>Meaningful variations in wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during liver regeneration and working hypotheses in regulated growth. (A) Graphic analysis of quantitative changes in wnt <span class="highlight" style="background-color:">pathway</span> components during liver regeneration is shown. There is an initial 2.5-fold increase in β-catenin protein, followed by an almost 3-fold decrease. There is also a 0.5-fold decrease in wnt-1 protein as the degradation <span class="highlight" style="background-color:">pathway</span> for β-catenin is activated. Following this, there is a 7-fold increase in APC protein apart from elevated serine-phosphorylated β-catenin and axin. TCF-4 levels remain mostly unaltered during these events. This graph also depicts an inverse quantitative relationship between E-cadherin and β-catenin proteins. An initial increase in β-catenin protein coincides with a decrease in E-cadherin levels, followed by an increase in E-cadherin protein as β-catenin decreases. (B) A model of regulated growth is hypothesized. Modulation of wnt <span class="highlight" style="background-color:">pathway</span> by altered β-catenin levels (decrease in the wnt-1 protein levels following an increase in the β-catenin protein) favors a possibility of a feedback-inhibitory loop in this <span class="highlight" style="background-color:">pathway</span>. Excess β-catenin protein [1] may activate this feedback loop, causing a decrease in wnt-1 protein levels [2]. This may lead to activation of the degradation <span class="highlight" style="background-color:">pathway</span> for β-catenin through ubiquitination, thus limiting and regulating availability of residual β-catenin for nuclear translocation [3] to control the target gene expression and cell proliferation. Aberrations in this feedback inhibition or checkpoint may lead to growth deregulation.</div></p></div><div class="aux"><div class="resc">Satdarshan P. S. Monga, et al. Hepatology. ;33(5):1098-1109.</div><p class="links"><a class="dblinks" href="/pubmed/11343237" ref="ordinalpos=2866&amp;ncbi_uid=992897&amp;link_uid=11343237">Citation</a><a class="dblinks" href="/pmc/articles/PMC1821078/" ref="ordinalpos=2866&amp;ncbi_uid=992897&amp;link_uid=PMC1821078">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2867.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2720436/figure/F1/" ref="ordinalpos=2867&amp;ncbi_uid=2038557&amp;link_uid=PMC2720436" href="/pmc/articles/PMC2720436/"><img src="/pmc/articles/instance/2720436/bin/nihms-76263-f0001.gif" src-large="/pmc/articles/instance/2720436/bin/nihms-76263-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2720436/" ref="ordinalpos=2867&amp;ncbi_uid=2038557&amp;link_uid=PMC2720436" image-link="/pmc/articles/PMC2720436/figure/F1/" class="imagepopup">Figure 1.  From: Expression of Yes-Associated Protein, YAP, in Common Solid Tumors. </a></p><div class="supp"><p class="details"><div>The Hippo <span class="highlight" style="background-color:">pathway</span>, a vital growth regulator of cell proliferation and apoptosis was initially identified by mosaic screens in <i>Drosophila melanogaster</i> (4,5). Information that is known about the Hippo <span class="highlight" style="background-color:">pathway</span> in <i>Drosophila</i> is likely directly applicable to mammalian systems as it has been shown that mammalian homologs are capable of rescuing <i>Drosophila</i> mutants defective in Hippo <span class="highlight" style="background-color:">signaling</span> (6-8). The components of this <span class="highlight" style="background-color:">pathway</span> which are highly conserved in mammals include the following: YAP (Yes-associated protein), Lats1/2, Mob, Mst1/2, Sav, Merlin, Expanded 1/2, and Fat 4 (Figure 1) (5). Hippo (Hpo) and Warts (Wts), two significant core <span class="highlight" style="background-color:">pathway</span> components, belong to the Sterile 20 (Ste20) and Nuclear Dbf2-related (NDR) serine/threonine protein kinase families, respectively (7,10-14). Salvador (Sav), a third major component of the Hippo <span class="highlight" style="background-color:">pathway</span>, is a scaffold protein with two WW domains and is homologous to the human ww45 protein (15,16). YAP, one of the highly conserved components in mammals, is considered to be a nuclear effector of the Hippo <span class="highlight" style="background-color:">pathway</span> (5,17). It has been noted that loss of critical components of the Hippo <span class="highlight" style="background-color:">pathway</span> can lead to uncontrolled cell growth, which implicates this as a tumor suppressor <span class="highlight" style="background-color:">pathway</span> (15).</div></p></div><div class="aux"><div class="resc">Angela A. Steinhardt, et al. Hum Pathol. ;39(11):1582-1589.</div><p class="links"><a class="dblinks" href="/pubmed/18703216" ref="ordinalpos=2867&amp;ncbi_uid=2038557&amp;link_uid=18703216">Citation</a><a class="dblinks" href="/pmc/articles/PMC2720436/" ref="ordinalpos=2867&amp;ncbi_uid=2038557&amp;link_uid=PMC2720436">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2868.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4051376/figure/F7/" ref="ordinalpos=2868&amp;ncbi_uid=6235882&amp;link_uid=PMC4051376" href="/pmc/articles/PMC4051376/"><img src="/pmc/articles/instance/4051376/bin/1471-2407-14-375-7.gif" src-large="/pmc/articles/instance/4051376/bin/1471-2407-14-375-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4051376/" ref="ordinalpos=2868&amp;ncbi_uid=6235882&amp;link_uid=PMC4051376" image-link="/pmc/articles/PMC4051376/figure/F7/" class="imagepopup">Figure 7.  From: Thrombomodulin expression regulates tumorigenesis in bladder cancer. </a></p><div class="supp"><p class="details"><div>TM <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in bladder cancer.</div></p></div><div class="aux"><div class="resc">Chun-Te Wu, et al. BMC Cancer. 2014;14:375-375.</div><p class="links"><a class="dblinks" href="/pubmed/24886404" ref="ordinalpos=2868&amp;ncbi_uid=6235882&amp;link_uid=24886404">Citation</a><a class="dblinks" href="/pmc/articles/PMC4051376/" ref="ordinalpos=2868&amp;ncbi_uid=6235882&amp;link_uid=PMC4051376">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2869.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3685413/figure/F2/" ref="ordinalpos=2869&amp;ncbi_uid=5004205&amp;link_uid=PMC3685413" href="/pmc/articles/PMC3685413/"><img src="/pmc/articles/instance/3685413/bin/nihms-349226-f0002.gif" src-large="/pmc/articles/instance/3685413/bin/nihms-349226-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3685413/" ref="ordinalpos=2869&amp;ncbi_uid=5004205&amp;link_uid=PMC3685413" image-link="/pmc/articles/PMC3685413/figure/F2/" class="imagepopup">Figure 2. Core <span class="highlight" style="background-color:">Signaling</span> Pathways in Chronic Lymphocytic Leukemia.  From: SF3B1 and Other Novel Cancer Genes in Chronic Lymphocytic Leukemia. </a></p><div class="supp"><p class="details"><div>The nine genes with significant mutation frequencies fall into five core <span class="highlight" style="background-color:">signaling</span> pathways, in which the genes play well-established roles: DNA damage repair and cell-cycle control (Panel A), Notch <span class="highlight" style="background-color:">signaling</span> (Panel B), inflammatory pathways (Panel C), Wnt <span class="highlight" style="background-color:">signaling</span> (Panel D), and RNA splicing and processing (Panel E). Genes with significant mutation frequencies are shown in red, and genes with mutations that are in a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related to chronic lymphocytic leukemia are shown in pink. A list of additional <span class="highlight" style="background-color:">pathway</span>-associated genes with mutations is provided in Table 6 in the Supplementary Appendix. Co-A denotes coenzyme A, TLR toll-like receptor, and TNFR tumor necrosis factor receptor.</div></p></div><div class="aux"><div class="resc">Lili Wang, et al. N Engl J Med. ;365(26):2497-2506.</div><p class="links"><a class="dblinks" href="/pubmed/22150006" ref="ordinalpos=2869&amp;ncbi_uid=5004205&amp;link_uid=22150006">Citation</a><a class="dblinks" href="/pmc/articles/PMC3685413/" ref="ordinalpos=2869&amp;ncbi_uid=5004205&amp;link_uid=PMC3685413">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2870.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3787764/figure/F8/" ref="ordinalpos=2870&amp;ncbi_uid=5410059&amp;link_uid=PMC3787764" href="/pmc/articles/PMC3787764/"><img src="/pmc/articles/instance/3787764/bin/DEV094102F8.gif" src-large="/pmc/articles/instance/3787764/bin/DEV094102F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3787764/" ref="ordinalpos=2870&amp;ncbi_uid=5410059&amp;link_uid=PMC3787764" image-link="/pmc/articles/PMC3787764/figure/F8/" class="imagepopup">Fig. 8. From: Myb promotes centriole amplification and later steps of the multiciliogenesis program. </a></p><div class="supp"><p class="details"><div><b>Proposed molecular <span class="highlight" style="background-color:">pathway</span> governing multiciliogenesis.</b> MYB and other transcriptional regulators associated with each step in the program are shown along with the regulatory relationships among them, together with the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that controls initiation of the program. The dashed line indicates that there is an auxiliary <span class="highlight" style="background-color:">pathway</span> that can partially bypass the requirement for MYB, at least in some MCCs.</div></p></div><div class="aux"><div class="resc">Fraser E. Tan, et al. Development. 2013 October 15;140(20):4277-4286.</div><p class="links"><a class="dblinks" href="/pubmed/24048590" ref="ordinalpos=2870&amp;ncbi_uid=5410059&amp;link_uid=24048590">Citation</a><a class="dblinks" href="/pmc/articles/PMC3787764/" ref="ordinalpos=2870&amp;ncbi_uid=5410059&amp;link_uid=PMC3787764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2871.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4218794/figure/pone-0111640-g004/" ref="ordinalpos=2871&amp;ncbi_uid=6804056&amp;link_uid=PMC4218794" href="/pmc/articles/PMC4218794/"><img src="/pmc/articles/instance/4218794/bin/pone.0111640.g004.gif" src-large="/pmc/articles/instance/4218794/bin/pone.0111640.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4218794/" ref="ordinalpos=2871&amp;ncbi_uid=6804056&amp;link_uid=PMC4218794" image-link="/pmc/articles/PMC4218794/figure/pone-0111640-g004/" class="imagepopup">Figure 4. Genes that were involved in the most significantly down-regulated pathways such as Natural Killer Cell <span class="highlight" style="background-color:">Signaling</span>, Crosstalk between Dendritic Cells and Natural Killer Cells, CD28 <span class="highlight" style="background-color:">Signaling</span> in T Helper Cells, PKCθ <span class="highlight" style="background-color:">Signaling</span> in T Lymphocytes..  From: Patient-Based Transcriptome-Wide Analysis Identify Interferon and Ubiquination Pathways as Potential Predictors of Influenza A Disease Severity. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">pathway</span> names were shared between different groups but the activated genes in each <span class="highlight" style="background-color:">pathway</span> were different. Differentially expressed genes (FDR &lt;0.05, fold change &gt;2) were highlighted in grey.</div></p></div><div class="aux"><div class="resc">Long Truong Hoang, et al. PLoS One. 2014;9(11):e111640.</div><p class="links"><a class="dblinks" href="/pubmed/25365328" ref="ordinalpos=2871&amp;ncbi_uid=6804056&amp;link_uid=25365328">Citation</a><a class="dblinks" href="/pmc/articles/PMC4218794/" ref="ordinalpos=2871&amp;ncbi_uid=6804056&amp;link_uid=PMC4218794">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2872.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4305955/figure/F1/" ref="ordinalpos=2872&amp;ncbi_uid=7109407&amp;link_uid=PMC4305955" href="/pmc/articles/PMC4305955/"><img src="/pmc/articles/instance/4305955/bin/WJG-11-1690-g001.gif" src-large="/pmc/articles/instance/4305955/bin/WJG-11-1690-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4305955/" ref="ordinalpos=2872&amp;ncbi_uid=7109407&amp;link_uid=PMC4305955" image-link="/pmc/articles/PMC4305955/figure/F1/" class="imagepopup">Figure 1.  From: HBVPathDB: A database of HBV infection-related molecular interaction network. </a></p><div class="supp"><p class="details"><div><i>Fibronectin</i>-related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> graph stored in HBVPathDB.</div></p></div><div class="aux"><div class="resc">Yi Zhang, et al. World J Gastroenterol. 2005 March 21;11(11):1690-1692.</div><p class="links"><a class="dblinks" href="/pubmed/15786551" ref="ordinalpos=2872&amp;ncbi_uid=7109407&amp;link_uid=15786551">Citation</a><a class="dblinks" href="/pmc/articles/PMC4305955/" ref="ordinalpos=2872&amp;ncbi_uid=7109407&amp;link_uid=PMC4305955">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2873.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3967813/figure/fig2/" ref="ordinalpos=2873&amp;ncbi_uid=6029316&amp;link_uid=PMC3967813" href="/pmc/articles/PMC3967813/"><img src="/pmc/articles/instance/3967813/bin/SCIENTIFICA2014-970315.002.gif" src-large="/pmc/articles/instance/3967813/bin/SCIENTIFICA2014-970315.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3967813/" ref="ordinalpos=2873&amp;ncbi_uid=6029316&amp;link_uid=PMC3967813" image-link="/pmc/articles/PMC3967813/figure/fig2/" class="imagepopup">Figure 2.  From: Human Interferon Alpha-2b: A Therapeutic Protein for Cancer Treatment. </a></p><div class="supp"><p class="details"><div>IFN<i>α</i>-2b <span class="highlight" style="background-color:">signaling</span> via JAK-STAT <span class="highlight" style="background-color:">pathway</span>, adapted from [13].</div></p></div><div class="aux"><div class="resc">Ratih Asmana Ningrum. Scientifica (Cairo). 2014;2014:970315.</div><p class="links"><a class="dblinks" href="/pubmed/24741445" ref="ordinalpos=2873&amp;ncbi_uid=6029316&amp;link_uid=24741445">Citation</a><a class="dblinks" href="/pmc/articles/PMC3967813/" ref="ordinalpos=2873&amp;ncbi_uid=6029316&amp;link_uid=PMC3967813">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2874.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3205325/figure/F2/" ref="ordinalpos=2874&amp;ncbi_uid=3505463&amp;link_uid=PMC3205325" href="/pmc/articles/PMC3205325/"><img src="/pmc/articles/instance/3205325/bin/nihms-312067-f0002.gif" src-large="/pmc/articles/instance/3205325/bin/nihms-312067-f0002.jpg" alt="Figure I" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3205325/" ref="ordinalpos=2874&amp;ncbi_uid=3505463&amp;link_uid=PMC3205325" image-link="/pmc/articles/PMC3205325/figure/F2/" class="imagepopup">Figure I.  From: Balancing acts: the role of TGF-? in the mucosal immune system. </a></p><div class="supp"><p class="details"><div>Canonical TGF-β <span class="highlight" style="background-color:">signaling</span>; Smad-dependent <span class="highlight" style="background-color:">pathway</span> downstream of TGF-β receptors.</div></p></div><div class="aux"><div class="resc">Joanne E. Konkel, et al. Trends Mol Med. ;17(11):668-676.</div><p class="links"><a class="dblinks" href="/pubmed/21890412" ref="ordinalpos=2874&amp;ncbi_uid=3505463&amp;link_uid=21890412">Citation</a><a class="dblinks" href="/pmc/articles/PMC3205325/" ref="ordinalpos=2874&amp;ncbi_uid=3505463&amp;link_uid=PMC3205325">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2875.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1885811/figure/F4/" ref="ordinalpos=2875&amp;ncbi_uid=960373&amp;link_uid=PMC1885811" href="/pmc/articles/PMC1885811/"><img src="/pmc/articles/instance/1885811/bin/1471-2105-8-168-4.gif" src-large="/pmc/articles/instance/1885811/bin/1471-2105-8-168-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1885811/" ref="ordinalpos=2875&amp;ncbi_uid=960373&amp;link_uid=PMC1885811" image-link="/pmc/articles/PMC1885811/figure/F4/" class="imagepopup">Figure 4.  From: BioPP: a tool for web-publication of biological networks. </a></p><div class="supp"><p class="details"><div>The web-accessible <span class="highlight" style="background-color:">pathway</span> navigator. A sample <span class="highlight" style="background-color:">pathway</span> is shown that represents part of the viral activated <span class="highlight" style="background-color:">signaling</span> pathways in human dendritic immune cells. The entities and links shown in the <span class="highlight" style="background-color:">pathway</span> are clickable to display the corresponding annotations. The content of the individual frames, which are indicated by roman numerals, are as follows: I – Help link. II – brief author provided description of the <span class="highlight" style="background-color:">pathway</span> and contact information for suggesting <span class="highlight" style="background-color:">pathway</span> revisions. III – link for the <span class="highlight" style="background-color:">pathway</span> legend (from CellDesigner), and links for collated lists of the annotations for interactions, protein, genes, and RNAs in the network. IV.1 navigating zoom rectangle. Dragging the zoom rectangle or clicking inside the index window (IV.2) displays the appropriate section of the <span class="highlight" style="background-color:">pathway</span> in IV.3. The arrow in IV.1 is a toggle button to hide/unhide the index window. The zoom facilitates navigating, particularly when the network map is large and fine-grain. V, VII, VIII, IX – sorted, hyperlinked lists of all the interactions, proteins, genes and RNAs, respectively. Clicking on a hyperlink displays the annotations in the annotation frame (VI). Links to the Entrez gene pages and the appropriate page of a public wiki-based discussion forum also launch new windows. VI – annotation frame which displays selected annotations. Clicking an interaction displays the annotations. Clicking a PMID opens a separate PubMed window for that citation.</div></p></div><div class="aux"><div class="resc">Ganesh A Viswanathan, et al. BMC Bioinformatics. 2007;8:168-168.</div><p class="links"><a class="dblinks" href="/pubmed/17519033" ref="ordinalpos=2875&amp;ncbi_uid=960373&amp;link_uid=17519033">Citation</a><a class="dblinks" href="/pmc/articles/PMC1885811/" ref="ordinalpos=2875&amp;ncbi_uid=960373&amp;link_uid=PMC1885811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2876.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4321714/figure/Fig6/" ref="ordinalpos=2876&amp;ncbi_uid=7158349&amp;link_uid=PMC4321714" href="/pmc/articles/PMC4321714/"><img src="/pmc/articles/instance/4321714/bin/13046_2014_108_Fig6_HTML.gif" src-large="/pmc/articles/instance/4321714/bin/13046_2014_108_Fig6_HTML.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4321714/" ref="ordinalpos=2876&amp;ncbi_uid=7158349&amp;link_uid=PMC4321714" image-link="/pmc/articles/PMC4321714/figure/Fig6/" class="imagepopup">Figure 6.  From: Histone acetyltransferase inhibitor II induces apoptosis in glioma cell lines via the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>Protein expression of factors related to the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in U251 cells treated with HATi II.</b> Western blot analysis of p53 and p21 <b>(A)</b> and factors downstream of the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(B, C)</b> in U251 cells treated with HATi II. <b>(D)</b> Western blot analysis confirming siRNA-mediated knockdown of Reprimo. <b>(E)</b> The morphological assay indicated that knockdown of Reprimo markedly reduced cell death in a dose-dependent manner in U251 cells treated with 10 or 20 μmol/l HATi II for 48 h. <b>(F)</b> The hoechst 33342 staining assay indicated that knockdown of Reprimo reduced apoptosis in U259 cells treated with 10 μmol/l HATi II for 48 h compared with control cells; **<i>P</i> &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Li-Xiao Xu, et al. J Exp Clin Cancer Res. 2014;33(1):108.</div><p class="links"><a class="dblinks" href="/pubmed/25523932" ref="ordinalpos=2876&amp;ncbi_uid=7158349&amp;link_uid=25523932">Citation</a><a class="dblinks" href="/pmc/articles/PMC4321714/" ref="ordinalpos=2876&amp;ncbi_uid=7158349&amp;link_uid=PMC4321714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2877.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2712453/figure/F4/" ref="ordinalpos=2877&amp;ncbi_uid=1925242&amp;link_uid=PMC2712453" href="/pmc/articles/PMC2712453/"><img src="/pmc/articles/instance/2712453/bin/1752-0509-3-67-4.gif" src-large="/pmc/articles/instance/2712453/bin/1752-0509-3-67-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2712453/" ref="ordinalpos=2877&amp;ncbi_uid=1925242&amp;link_uid=PMC2712453" image-link="/pmc/articles/PMC2712453/figure/F4/" class="imagepopup">Figure 4.  From: BowTieBuilder: modeling signal transduction pathways. </a></p><div class="supp"><p class="details"><div><b>MAKP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of yeast</b>. Depicted is the KEGG <span class="highlight" style="background-color:">pathway</span> and the <span class="highlight" style="background-color:">pathway</span> inferred by BowTieBuilder. The membrane-bound proteins are drawn at the top, and the TFs are drawn at the bottom. The inferred proteins are depicted in between. Proteins that do not overlap between the KEGG and BowTieBuilder pathways are depicted in gray. The core proteins are embedded in a gray box, where FUS3 is contained in all core structures. The recall and precision values are given in Table 1.</div></p></div><div class="aux"><div class="resc">Jochen Supper, et al. BMC Syst Biol. 2009;3:67-67.</div><p class="links"><a class="dblinks" href="/pubmed/19566957" ref="ordinalpos=2877&amp;ncbi_uid=1925242&amp;link_uid=19566957">Citation</a><a class="dblinks" href="/pmc/articles/PMC2712453/" ref="ordinalpos=2877&amp;ncbi_uid=1925242&amp;link_uid=PMC2712453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2878.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3770665/figure/pone-0074205-g005/" ref="ordinalpos=2878&amp;ncbi_uid=5304202&amp;link_uid=PMC3770665" href="/pmc/articles/PMC3770665/"><img src="/pmc/articles/instance/3770665/bin/pone.0074205.g005.gif" src-large="/pmc/articles/instance/3770665/bin/pone.0074205.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3770665/" ref="ordinalpos=2878&amp;ncbi_uid=5304202&amp;link_uid=PMC3770665" image-link="/pmc/articles/PMC3770665/figure/pone-0074205-g005/" class="imagepopup">Figure 5. Pathways analysis of transcriptomic perturbations implicates MAPK <span class="highlight" style="background-color:">signaling</span>..  From: Systems-Based Identification of Temporal Processing Pathways during Bone Cell Mechanotransduction. </a></p><div class="supp"><p class="details"><div>(A–B): Results from KEGG analysis for Groups A and B. Group A was significantly associated with MAPK <span class="highlight" style="background-color:">signaling</span> (an intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) as well as two pathways associated with extracellular signal transduction: focal adhesion and cytokine-cytokine receptor interactions. Group B was significantly associated with cytokine-cytokine receptor interactions only. *Bayes Factor &gt;6 (C–D): Results from SPEED analysis for Groups A and B. SPEED analysis revealed a significant overlap in Group A genes with the MAPK gene expression signature, and a significant overlap of Group B genes with the TNF-alpha <span class="highlight" style="background-color:">pathway</span>. *FDR &lt;0.05.</div></p></div><div class="aux"><div class="resc">Leah E. Worton, et al. PLoS One. 2013;8(9):e74205.</div><p class="links"><a class="dblinks" href="/pubmed/24040202" ref="ordinalpos=2878&amp;ncbi_uid=5304202&amp;link_uid=24040202">Citation</a><a class="dblinks" href="/pmc/articles/PMC3770665/" ref="ordinalpos=2878&amp;ncbi_uid=5304202&amp;link_uid=PMC3770665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2879.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2856900/figure/fig2/" ref="ordinalpos=2879&amp;ncbi_uid=2522097&amp;link_uid=PMC2856900" href="/pmc/articles/PMC2856900/"><img src="/pmc/articles/instance/2856900/bin/JCB_200912010_GS_Fig2.gif" src-large="/pmc/articles/instance/2856900/bin/JCB_200912010_GS_Fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2856900/" ref="ordinalpos=2879&amp;ncbi_uid=2522097&amp;link_uid=PMC2856900" image-link="/pmc/articles/PMC2856900/figure/fig2/" class="imagepopup">Figure 2. From: Rho1 regulates apoptosis via activation of the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at the plasma membrane. </a></p><div class="supp"><p class="details"><div><b>Ectopic Rho1 expression induces JNK <span class="highlight" style="background-color:">pathway</span> activation and apoptosis.</b> (A and B) Compared with wild-type discs (A), a reporter for <i>hid</i>, a proapoptotic gene and canonical apoptotic <span class="highlight" style="background-color:">pathway</span> component, is strongly up-regulated in cells that express a <i>UAS-Rho1<sup>+</sup></i> transgene (B). (C and D) <i>puc-lacZ</i>, a reporter for JNK <span class="highlight" style="background-color:">signaling</span>, is increased in <i>dpp-Gal4; UAS-Rho1<sup>+</sup></i>–expressing cells. (E and F) Mmp1, an additional downstream target of JNK <span class="highlight" style="background-color:">signaling</span>, is also up-regulated in cells expressing the <i>Rho1</i> transgene (compare E with F). (G and H) Apoptosis, caused by expression of <i>dpp-Gal4; UAS-Rho1<sup>+</sup></i> (<i>dpp&gt;Rho1</i>; G) is increased by reducing the dosage of <i>puc</i> (H) and thereby increasing JNK <span class="highlight" style="background-color:">signaling</span>. (I–K) Conversely, apoptosis induced by <i>dpp&gt;Rho1<sup>+</sup></i> is suppressed by blocking JNK <span class="highlight" style="background-color:">signaling</span> at the level of Bsk (<i>UAS-bsk<sup>DN</sup></i> [I] or <i>UAS-puc</i> [J]), the sole Jun kinase, or Hep (<i>UAS-hepRNAi</i>; K), a JNK kinase. (L) Knockdown of both Slpr and Tak1, two JNKKKs, together in <i>dpp&gt;Rho1<sup>+</sup></i>-expressing cells suppresses apoptosis to a large extent. For all genotypes, examples shown are representative of ≥20 imaginal discs examined. Bar, 25 µm.</div></p></div><div class="aux"><div class="resc">Amanda L. Neisch, et al. J Cell Biol. 2010 April 19;189(2):311-323.</div><p class="links"><a class="dblinks" href="/pubmed/20404112" ref="ordinalpos=2879&amp;ncbi_uid=2522097&amp;link_uid=20404112">Citation</a><a class="dblinks" href="/pmc/articles/PMC2856900/" ref="ordinalpos=2879&amp;ncbi_uid=2522097&amp;link_uid=PMC2856900">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2880.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2946881/figure/f1/" ref="ordinalpos=2880&amp;ncbi_uid=3132664&amp;link_uid=PMC2946881" href="/pmc/articles/PMC2946881/"><img src="/pmc/articles/instance/2946881/bin/fig-1.gif" src-large="/pmc/articles/instance/2946881/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2946881/" ref="ordinalpos=2880&amp;ncbi_uid=3132664&amp;link_uid=PMC2946881" image-link="/pmc/articles/PMC2946881/figure/f1/" class="imagepopup">FIG. 1. From: Adipose Tissue Engineering for Soft Tissue Regeneration. </a></p><div class="supp"><p class="details"><div>Regulation of adipogenesis by canonical Wnt <span class="highlight" style="background-color:">signaling</span> cascade. In the presence of Wnt <span class="highlight" style="background-color:">signaling</span>, adipogenesis is inhibited through β-catenin-mediated expression of Wnt target genes. Wnt <span class="highlight" style="background-color:">pathway</span> antagonists, including frizzled-related proteins (sFRPs), Wnt inhibitory factor (WIF-1), and Cerberus and Dickkopfs (Dkk), inhibit Wnt <span class="highlight" style="background-color:">signaling</span> and subsequently stimulate adipogenesis.</div></p></div><div class="aux"><div class="resc">Jennifer H. Choi, et al. Tissue Eng Part B Rev. 2010 August;16(4):413-426.</div><p class="links"><a class="dblinks" href="/pubmed/20166810" ref="ordinalpos=2880&amp;ncbi_uid=3132664&amp;link_uid=20166810">Citation</a><a class="dblinks" href="/pmc/articles/PMC2946881/" ref="ordinalpos=2880&amp;ncbi_uid=3132664&amp;link_uid=PMC2946881">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2881.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3131876/figure/fig7/" ref="ordinalpos=2881&amp;ncbi_uid=3475545&amp;link_uid=PMC3131876" href="/pmc/articles/PMC3131876/"><img src="/pmc/articles/instance/3131876/bin/cdd201072f7.gif" src-large="/pmc/articles/instance/3131876/bin/cdd201072f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3131876/" ref="ordinalpos=2881&amp;ncbi_uid=3475545&amp;link_uid=PMC3131876" image-link="/pmc/articles/PMC3131876/figure/fig7/" class="imagepopup">Figure 7.  From: zVAD-induced necroptosis in L929 cells depends on autocrine production of TNFα mediated by the PKC-MAPKs-AP-1 <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>TPA combining with caspase 8 inhibition induces necroptosis in L929 cells. (<b>a</b>) zVAD-fmk (zVAD) and BocD-fmk (BocD), but not QVD-oph (QVD), activate the PKC-MAPKs-AP-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. L929 cells were treated with zVAD (10 <i>μ</i>M), QVD (20 <i>μ</i>M), or BocD (10 <i>μ</i>M) for 6 h, and cell lysate was subjected to immunoblotting. (<b>b</b>) QVD, when combining with TPA, induces cell death. L929 cells were treated with zVAD (10 <i>μ</i>M), QVD (20 <i>μ</i>M), or DEVD-cho (DEVD, 100 <i>μ</i>M) with or without the presence of TPA (80 nM) for 24 h, and cell death was quantified as described in Figure 3b. (<b>c</b>) TPA induces cell death in caspase 8 knockdown cells. L929 cells with knockdown of caspase 8, were treated with TPA (80 nM) or zVAD (10 <i>μ</i>M) for 24 h, cell death was determined as described in Figure 3b. (<b>d</b>) Illustration of the <span class="highlight" style="background-color:">signaling</span> pathways for zVAD-induced necroptosis. Both promotion of autocrine production of TNF<i>α</i> by the PKC–MAPKs–AP-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and suppression of caspase activation to stabilize RIP1 are required for necroptosis induced by zVAD</div></p></div><div class="aux"><div class="resc">Y-T Wu, et al. Cell Death Differ. 2011 January;18(1):26-37.</div><p class="links"><a class="dblinks" href="/pubmed/20539307" ref="ordinalpos=2881&amp;ncbi_uid=3475545&amp;link_uid=20539307">Citation</a><a class="dblinks" href="/pmc/articles/PMC3131876/" ref="ordinalpos=2881&amp;ncbi_uid=3475545&amp;link_uid=PMC3131876">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2882.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3003620/figure/F1/" ref="ordinalpos=2882&amp;ncbi_uid=2639264&amp;link_uid=PMC3003620" href="/pmc/articles/PMC3003620/"><img src="/pmc/articles/instance/3003620/bin/1756-8722-3-45-1.gif" src-large="/pmc/articles/instance/3003620/bin/1756-8722-3-45-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3003620/" ref="ordinalpos=2882&amp;ncbi_uid=2639264&amp;link_uid=PMC3003620" image-link="/pmc/articles/PMC3003620/figure/F1/" class="imagepopup">Figure 1.  From: Targeted therapy in lymphoma. </a></p><div class="supp"><p class="details"><div><b>The PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Reprinted with permission from Altman JK, Platanias LC: Exploiting the mammalian target of rapamycin <span class="highlight" style="background-color:">pathway</span> in hematologic malignancies. <i>Curr Opin Hematol</i>. 2008, <b>15</b>:88-94.</div></p></div><div class="aux"><div class="resc">Patrick B Johnston, et al. J Hematol Oncol. 2010;3:45-45.</div><p class="links"><a class="dblinks" href="/pubmed/21092307" ref="ordinalpos=2882&amp;ncbi_uid=2639264&amp;link_uid=21092307">Citation</a><a class="dblinks" href="/pmc/articles/PMC3003620/" ref="ordinalpos=2882&amp;ncbi_uid=2639264&amp;link_uid=PMC3003620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2883.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1868478/figure/F6/" ref="ordinalpos=2883&amp;ncbi_uid=1054877&amp;link_uid=PMC1868478" href="/pmc/articles/PMC1868478/"><img src="/pmc/articles/instance/1868478/bin/nihms19660f6.gif" src-large="/pmc/articles/instance/1868478/bin/nihms19660f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1868478/" ref="ordinalpos=2883&amp;ncbi_uid=1054877&amp;link_uid=PMC1868478" image-link="/pmc/articles/PMC1868478/figure/F6/" class="imagepopup">Figure 6. Inhibition of TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in apoptotic and non-apoptotic populations in HSV-2 infected Jurkat cells.  From: Apoptosis and antigen receptor function in T and B cells following exposure to herpes simplex virus. </a></p><div class="supp"><p class="details"><div>Jurkat cells were infected with HSV-2 HG52 for 6 h prior to analysis. Cells were stained simultaneously with indo-1 and annexin V for detection of calcium flux in apoptotic and non-apoptotic populations. (A) Annexin V stain shows both apoptotic and non-apoptotic cells at 6 h p.i. (B) Apoptotic and non-apoptotic populations were gated individually and assessed for the ability mobilize calcium by flow cytometric analysis. (C) Both populations demonstrate inhibition of TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as shown by the small percentages of cells (upper right quadrants) able to mobilize calcium following OKT-3 stimulations (vertical arrows).</div></p></div><div class="aux"><div class="resc">Jin-Young Han, et al. Virology. ;359(2):253-263.</div><p class="links"><a class="dblinks" href="/pubmed/17067652" ref="ordinalpos=2883&amp;ncbi_uid=1054877&amp;link_uid=17067652">Citation</a><a class="dblinks" href="/pmc/articles/PMC1868478/" ref="ordinalpos=2883&amp;ncbi_uid=1054877&amp;link_uid=PMC1868478">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2884.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3981532/figure/F1/" ref="ordinalpos=2884&amp;ncbi_uid=6009879&amp;link_uid=PMC3981532" href="/pmc/articles/PMC3981532/"><img src="/pmc/articles/instance/3981532/bin/nihms127992f1.gif" src-large="/pmc/articles/instance/3981532/bin/nihms127992f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3981532/" ref="ordinalpos=2884&amp;ncbi_uid=6009879&amp;link_uid=PMC3981532" image-link="/pmc/articles/PMC3981532/figure/F1/" class="imagepopup">Figure 1. TOR <span class="highlight" style="background-color:">signaling</span> and Age-Associated Disease.  From: The TOR <span class="highlight" style="background-color:">pathway</span> comes of age. </a></p><div class="supp"><p class="details"><div>Schematic depicts age-related pathologies for which there is a clear link between TOR <span class="highlight" style="background-color:">signaling</span> and disease progression.</div></p></div><div class="aux"><div class="resc">Monique N. Stanfel, et al. Biochim Biophys Acta. 2009 October;1790(10):1067-1074.</div><p class="links"><a class="dblinks" href="/pubmed/19539012" ref="ordinalpos=2884&amp;ncbi_uid=6009879&amp;link_uid=19539012">Citation</a><a class="dblinks" href="/pmc/articles/PMC3981532/" ref="ordinalpos=2884&amp;ncbi_uid=6009879&amp;link_uid=PMC3981532">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2885.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4324947/figure/fig5/" ref="ordinalpos=2885&amp;ncbi_uid=7185923&amp;link_uid=PMC4324947" href="/pmc/articles/PMC4324947/"><img src="/pmc/articles/instance/4324947/bin/JDR2015-920280.005.gif" src-large="/pmc/articles/instance/4324947/bin/JDR2015-920280.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4324947/" ref="ordinalpos=2885&amp;ncbi_uid=7185923&amp;link_uid=PMC4324947" image-link="/pmc/articles/PMC4324947/figure/fig5/" class="imagepopup">Figure 5.  From: The PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Mediates the High Glucose-Induced Expression of Extracellular Matrix Molecules in Human Retinal Pigment Epithelial Cells. </a></p><div class="supp"><p class="details"><div>The PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated the mRNA expression of fibronectin, collagen IV, and laminin in RPE cells under high glucose conditions. (a, b, c) After pretreatment with 20 <i>μ</i>M LY294002 for 12 h, the mRNA expression of fibronectin, collagen IV, and laminin was significantly decreased when exposed to high glucose conditions for 24 h in RPE cells. <sup>*</sup><i>P</i> &lt; 0.01 versus NG or HG + LY. The data shown represent the mean ± SD of three independent experiments.</div></p></div><div class="aux"><div class="resc">Dong Qin, et al. J Diabetes Res. 2015;2015:920280.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2885&amp;ncbi_uid=7185923&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4324947/" ref="ordinalpos=2885&amp;ncbi_uid=7185923&amp;link_uid=PMC4324947">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2886.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3071979/figure/F2/" ref="ordinalpos=2886&amp;ncbi_uid=3485216&amp;link_uid=PMC3071979" href="/pmc/articles/PMC3071979/"><img src="/pmc/articles/instance/3071979/bin/nihms280847f2.gif" src-large="/pmc/articles/instance/3071979/bin/nihms280847f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3071979/" ref="ordinalpos=2886&amp;ncbi_uid=3485216&amp;link_uid=PMC3071979" image-link="/pmc/articles/PMC3071979/figure/F2/" class="imagepopup">Figure 2. Activation of NFKB <span class="highlight" style="background-color:">signaling</span> in the skin of Sharpincpdm/Sharpincpdm mutant mice.  From: Inhibition of NFKB <span class="highlight" style="background-color:">signaling</span> retards eosinophilic dermatitis in SHARPIN-deficient mice. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of transcriptome data in skin using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis<sup>®</sup> software suggested activation of both classical and alterative NFKB pathways. Most of molecules in NFKB pathways were upregulated, such as p65/RelA and NFKB p52/RelB in the mutant group from the pooled expression data of 2-, 4-, 6-, 8-, and 10-week old mice. <img src="/pmc/articles/instance/3071979/bin/nihms280847ig1.jpg" class="inline-graphic" style="border-style: none;" /> upregulation; <img src="/pmc/articles/instance/3071979/bin/nihms280847ig2.jpg" class="inline-graphic" style="border-style: none;" /> downregulation; □ no significant change</div></p></div><div class="aux"><div class="resc">Yanhua Liang, et al. J Invest Dermatol. ;131(1):141-149.</div><p class="links"><a class="dblinks" href="/pubmed/20811394" ref="ordinalpos=2886&amp;ncbi_uid=3485216&amp;link_uid=20811394">Citation</a><a class="dblinks" href="/pmc/articles/PMC3071979/" ref="ordinalpos=2886&amp;ncbi_uid=3485216&amp;link_uid=PMC3071979">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2887.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634111/figure/F1/" ref="ordinalpos=2887&amp;ncbi_uid=1708631&amp;link_uid=PMC2634111" href="/pmc/articles/PMC2634111/"><img src="/pmc/articles/instance/2634111/bin/psb0303_0175_fig001.gif" src-large="/pmc/articles/instance/2634111/bin/psb0303_0175_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634111/" ref="ordinalpos=2887&amp;ncbi_uid=1708631&amp;link_uid=PMC2634111" image-link="/pmc/articles/PMC2634111/figure/F1/" class="imagepopup">Figure 1.  From: Oxidative <span class="highlight" style="background-color:">signaling</span> in seed germination and dormancy. </a></p><div class="supp"><p class="details"><div>Hypothetical model proposing a central role of ROS in seed dormancy release and germination. In dormant state, high amount of ABA induces an active <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in dormancy maintenance. Lower ABA concentration in nondormant seeds would be associated with a higher level of ROS, presumably H<sub>2</sub>O<sub>2</sub>, which could in turn interfere with ABA and GA <span class="highlight" style="background-color:">signaling</span> pathways, modify redox status and induce protein carbonylation. NO may interfere with H<sub>2</sub>O<sub>2</sub> and activate GA <span class="highlight" style="background-color:">signaling</span>. The size of the letters is proportional to the relative amount of the compounds (ABA, GA, H<sub>2</sub>O<sub>2</sub>).</div></p></div><div class="aux"><div class="resc">Hayat El-Maarouf-Bouteau, et al. Plant Signal Behav. 2008 March;3(3):175-182.</div><p class="links"><a class="dblinks" href="/pubmed/19513212" ref="ordinalpos=2887&amp;ncbi_uid=1708631&amp;link_uid=19513212">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634111/" ref="ordinalpos=2887&amp;ncbi_uid=1708631&amp;link_uid=PMC2634111">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2888.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4353798/figure/f5-etm-09-04-1265/" ref="ordinalpos=2888&amp;ncbi_uid=7283571&amp;link_uid=PMC4353798" href="/pmc/articles/PMC4353798/"><img src="/pmc/articles/instance/4353798/bin/ETM-09-04-1265-g04.gif" src-large="/pmc/articles/instance/4353798/bin/ETM-09-04-1265-g04.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4353798/" ref="ordinalpos=2888&amp;ncbi_uid=7283571&amp;link_uid=PMC4353798" image-link="/pmc/articles/PMC4353798/figure/f5-etm-09-04-1265/" class="imagepopup">Figure 5.  From: Cbl-b-regulated extracellular signal-regulated kinase <span class="highlight" style="background-color:">signaling</span> is involved in the shikonin-induced apoptosis of lung cancer cells in vitro. </a></p><div class="supp"><p class="details"><div>Effects of shikonin (SK) on the extracellular signal-regulated kinase (ERK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in NCI-H460 cells. NCI-H460 cells were treated with 2.6 μM SK for the indicated times. The expression levels of phospho (p)-ERK and ERK proteins were then analyzed by western blotting.</div></p></div><div class="aux"><div class="resc">DAN QU, et al. Exp Ther Med. 2015 April;9(4):1265-1270.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2888&amp;ncbi_uid=7283571&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4353798/" ref="ordinalpos=2888&amp;ncbi_uid=7283571&amp;link_uid=PMC4353798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2889.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4286746/figure/f3-bt-23-26/" ref="ordinalpos=2889&amp;ncbi_uid=7067007&amp;link_uid=PMC4286746" href="/pmc/articles/PMC4286746/"><img src="/pmc/articles/instance/4286746/bin/bt-23-26f3.gif" src-large="/pmc/articles/instance/4286746/bin/bt-23-26f3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4286746/" ref="ordinalpos=2889&amp;ncbi_uid=7067007&amp;link_uid=PMC4286746" image-link="/pmc/articles/PMC4286746/figure/f3-bt-23-26/" class="imagepopup">Fig. 3. From: Atractylochromene Is a Repressor of Wnt/β-Catenin <span class="highlight" style="background-color:">Signaling</span> in Colon Cancer Cells. </a></p><div class="supp"><p class="details"><div>Atractylochromene inhibits β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in SW-480 cells. SW-480 cells were treated with 20 μg/ml of atractylochromene for the indicated time, and cells were harvested to prepare nuclear and cytosolic fractions (A), and whole cell lysates (B). Relative expression levels of target proteins were detected by western blot analysis.</div></p></div><div class="aux"><div class="resc">Ah-Ram Shim, et al. Biomol Ther (Seoul). 2015 January;23(1):26-30.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2889&amp;ncbi_uid=7067007&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4286746/" ref="ordinalpos=2889&amp;ncbi_uid=7067007&amp;link_uid=PMC4286746">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2890.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3205001/figure/pone-0027006-g006/" ref="ordinalpos=2890&amp;ncbi_uid=3339131&amp;link_uid=PMC3205001" href="/pmc/articles/PMC3205001/"><img src="/pmc/articles/instance/3205001/bin/pone.0027006.g006.gif" src-large="/pmc/articles/instance/3205001/bin/pone.0027006.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3205001/" ref="ordinalpos=2890&amp;ncbi_uid=3339131&amp;link_uid=PMC3205001" image-link="/pmc/articles/PMC3205001/figure/pone-0027006-g006/" class="imagepopup">Figure 6. Inhibition of NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by PL..  From: Amelioration of Experimental Autoimmune Encephalomyelitis by Plumbagin through Down-Regulation of JAK-STAT and NF-?B <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>CD4<sup>+</sup> T-cells from PL-treated mice, EAE mice and adjuvant control mice were cultured in the absence or presence of MOG peptide for 24 h, extracted proteins were subjected to electrophoresis and analyzed by immunoblotting for phosphorylated and total p65 and IκBα.</div></p></div><div class="aux"><div class="resc">Yan Jia, et al. PLoS One. 2011;6(10):e27006.</div><p class="links"><a class="dblinks" href="/pubmed/22066025" ref="ordinalpos=2890&amp;ncbi_uid=3339131&amp;link_uid=22066025">Citation</a><a class="dblinks" href="/pmc/articles/PMC3205001/" ref="ordinalpos=2890&amp;ncbi_uid=3339131&amp;link_uid=PMC3205001">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2891.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC304101/figure/fig2/" ref="ordinalpos=2891&amp;ncbi_uid=343901&amp;link_uid=PMC304101" href="/pmc/articles/PMC304101/"><img src="/pmc/articles/instance/304101/bin/pq1834202002.gif" src-large="/pmc/articles/instance/304101/bin/pq1834202002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC304101/" ref="ordinalpos=2891&amp;ncbi_uid=343901&amp;link_uid=PMC304101" image-link="/pmc/articles/PMC304101/figure/fig2/" class="imagepopup">Fig. 2. From: Indian hedgehog and ?-catenin <span class="highlight" style="background-color:">signaling</span>: Role in the sebaceous lineage of normal and neoplastic mammalian epidermis. </a></p><div class="supp"><p class="details"><div>Expression of proteins in the SHH-PTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human sebaceous tumors. Immunostaining was performed with antibodies to the proteins indicated. (<i>a</i>) PTCH1 C terminus. (<i>b</i>) PTCH1 N terminus. (<i>c</i>) SHH. (<i>d</i>) IHH. (<i>e</i>) DHH. (<i>f</i>) pan-HH. (<i>g</i>) GLI1. (<i>h</i>) GLI2. (Scale bar: 200 μm.)</div></p></div><div class="aux"><div class="resc">C. Niemann, et al. Proc Natl Acad Sci U S A. 2003 September 30;100(Suppl 1):11873-11880.</div><p class="links"><a class="dblinks" href="/pubmed/12917489" ref="ordinalpos=2891&amp;ncbi_uid=343901&amp;link_uid=12917489">Citation</a><a class="dblinks" href="/pmc/articles/PMC304101/" ref="ordinalpos=2891&amp;ncbi_uid=343901&amp;link_uid=PMC304101">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2892.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2919655/figure/F3/" ref="ordinalpos=2892&amp;ncbi_uid=3247652&amp;link_uid=PMC2919655" href="/pmc/articles/PMC2919655/"><img src="/pmc/articles/instance/2919655/bin/nihms195420f3.gif" src-large="/pmc/articles/instance/2919655/bin/nihms195420f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919655/" ref="ordinalpos=2892&amp;ncbi_uid=3247652&amp;link_uid=PMC2919655" image-link="/pmc/articles/PMC2919655/figure/F3/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> pathways leading to mRNA stabilization.  From: Diversity in Post-Transcriptional Control of Neutrophil Chemoattractant Cytokine Gene Expression. </a></p><div class="supp"><p class="details"><div>In myeloid cells, chemokine mRNA instability is dependent upon AUUUA-containing ARE motifs that are recognized by TTP while in non-myeloid cells, AUUUA-free sequences produce comparable instability through undefined RNA binding protein(s). TIR <span class="highlight" style="background-color:">signaling</span> couples to mRNA stabilization via the adaptor protein MyD88 with IRAK4 and IRAK1 kinases, TRAF6, TAK1, and the p38 MAP kinase cascade. IL-17 signals in non-myeloid cells through the adapter protein Act1 and couples with a second mRNA stabilization <span class="highlight" style="background-color:">pathway</span> that operates independently of TRAF6 and p38/MK2. This second <span class="highlight" style="background-color:">pathway</span> also appears to be utilized by IL-1 in non-myeloid cells and segregates from the NFκB activation <span class="highlight" style="background-color:">pathway</span> at IRAK1. These pathways link to an instability/stabilization mechanism that depends upon non-AUUUA containing sequence and does not require TTP. The details of this second <span class="highlight" style="background-color:">pathway</span>, including the specific target nucleotide sequence motifs, the RNA binding proteins, and the <span class="highlight" style="background-color:">signaling</span> events remain to be identified.</div></p></div><div class="aux"><div class="resc">Thomas Hamilton, et al. Cytokine. ;52(1-2):116-122.</div><p class="links"><a class="dblinks" href="/pubmed/20430641" ref="ordinalpos=2892&amp;ncbi_uid=3247652&amp;link_uid=20430641">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919655/" ref="ordinalpos=2892&amp;ncbi_uid=3247652&amp;link_uid=PMC2919655">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2893.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4324303/figure/F7/" ref="ordinalpos=2893&amp;ncbi_uid=7220108&amp;link_uid=PMC4324303" href="/pmc/articles/PMC4324303/"><img src="/pmc/articles/instance/4324303/bin/fmicb-06-00118-g0007.gif" src-large="/pmc/articles/instance/4324303/bin/fmicb-06-00118-g0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4324303/" ref="ordinalpos=2893&amp;ncbi_uid=7220108&amp;link_uid=PMC4324303" image-link="/pmc/articles/PMC4324303/figure/F7/" class="imagepopup">Figure 7.  From: The Pho4 transcription factor mediates the response to arsenate and arsenite in Candida albicans. </a></p><div class="supp"><p class="details"><div><b>Model of As (V) and As (III) <span class="highlight" style="background-color:">signaling</span></b>. A schematic representation of the MAPK pathways involved in arsenic <span class="highlight" style="background-color:">signaling</span> is depicted. As (V) triggers phosphorylation of both, HOG and CWI pathways. As (V) <span class="highlight" style="background-color:">signaling</span> requires Ssk1 suggesting that signal comes from outside the cell. Hog1 and Mkc1 phosphorylation may mediate transcription of genes involved in its detoxification or tolerance. As (III) triggers Hog1 phosphorylation and therefore, HOG <span class="highlight" style="background-color:">pathway</span> may control the transcription of genes involved in As (III) detoxification. As (III) <span class="highlight" style="background-color:">signaling</span> is independent of Ssk1 and others elements involves in Cek1 phosphorylation (Sho1, Opy2, and Msb2) but requires Ssk2 and Pbs2. The As (III) <span class="highlight" style="background-color:">signaling</span> may enter through an unknown element (represented as a question mark) or may enter the cells triggering Hog1 phosphorylation avoiding the three components system (Sln1-Ypd1-Ssk1).</div></p></div><div class="aux"><div class="resc">Verónica Urrialde, et al. Front Microbiol. 2015;6:118.</div><p class="links"><a class="dblinks" href="/pubmed/25717325" ref="ordinalpos=2893&amp;ncbi_uid=7220108&amp;link_uid=25717325">Citation</a><a class="dblinks" href="/pmc/articles/PMC4324303/" ref="ordinalpos=2893&amp;ncbi_uid=7220108&amp;link_uid=PMC4324303">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2894.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3737717/figure/F8/" ref="ordinalpos=2894&amp;ncbi_uid=6503140&amp;link_uid=PMC3737717" href="/pmc/articles/PMC3737717/"><img src="/pmc/articles/instance/3737717/bin/DEV089193F8.gif" src-large="/pmc/articles/instance/3737717/bin/DEV089193F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3737717/" ref="ordinalpos=2894&amp;ncbi_uid=6503140&amp;link_uid=PMC3737717" image-link="/pmc/articles/PMC3737717/figure/F8/" class="imagepopup">Fig. 8. From: The Pitx2:miR-200c/141:noggin <span class="highlight" style="background-color:">pathway</span> regulates Bmp <span class="highlight" style="background-color:">signaling</span> and ameloblast differentiation. </a></p><div class="supp"><p class="details"><div><b>A model for the role of miR-200c/141 in the adhesion of stem cells in the stellate reticulum to the outer enamel epithelium.</b> (<b>A</b>) Depiction of the incisor labial cervical loop and associated cells. Putative epithelial dental stem cells are located in the stellate reticulum and the outer enamel epithelium (indicated in green). (<b>B</b>) Model of the Pitx2:miR-200c/141:noggin regulatory <span class="highlight" style="background-color:">pathway</span> in the regulation of tooth development and renewal. Pitx2 directly activates miR-200c/141 and miR-203, which directly inhibit noggin and Bmper, which inhibit Bmp <span class="highlight" style="background-color:">signaling</span>. Bmp feeds back to activate miR-200c/141 and miR-203 expression. miR-200c/141 and miR-203 indirectly regulate amelogenin (Amel) and E-cadherin (E-cad) expression. These factors regulate tooth development and renewal.</div></p></div><div class="aux"><div class="resc">Huojun Cao, et al. Development. 2013 August 15;140(16):3348-3359.</div><p class="links"><a class="dblinks" href="/pubmed/23863486" ref="ordinalpos=2894&amp;ncbi_uid=6503140&amp;link_uid=23863486">Citation</a><a class="dblinks" href="/pmc/articles/PMC3737717/" ref="ordinalpos=2894&amp;ncbi_uid=6503140&amp;link_uid=PMC3737717">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2895.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3101100/figure/fig1/" ref="ordinalpos=2895&amp;ncbi_uid=3140539&amp;link_uid=PMC3101100" href="/pmc/articles/PMC3101100/"><img src="/pmc/articles/instance/3101100/bin/JCB_201004016_RGB_Fig1.gif" src-large="/pmc/articles/instance/3101100/bin/JCB_201004016_RGB_Fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3101100/" ref="ordinalpos=2895&amp;ncbi_uid=3140539&amp;link_uid=PMC3101100" image-link="/pmc/articles/PMC3101100/figure/fig1/" class="imagepopup">Figure 1. From: Modeling spinal muscular atrophy in Drosophila links Smn to FGF <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Multiple FGF <span class="highlight" style="background-color:">pathway</span> components modify <i>Smn</i>-dependent viability.</b> (A) Schematic diagram depicting the FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>Drosophila</i>. In <i>Drosophila</i>, <span class="highlight" style="background-color:">pathway</span> activation is mediated by the two known FGF receptor orthologues, <i>breathless</i> (<i>btl</i>) and <i>heartless</i> (<i>htl</i>) (Glazer and Shilo, 1991). <i>btl</i>, which functions in the tracheal system, is activated by its ligand <i>branchless</i> (<i>bnl</i>) (Sutherland et al., 1996), whereas <i>htl</i>, which functions in the mesoderm and muscles, is activated either by the <i>thisbe</i> (<i>ths</i>) (Kerr et al., 2003) or <i>pyramus</i> (<i>pyr</i>) (Stathopoulos et al., 2004) ligands. Both receptors act through Sos-Grb2 to activate Ras/Raf/MAP kinase <span class="highlight" style="background-color:">signaling</span>. Additional regulation of Ras/Raf/MAP kinase <span class="highlight" style="background-color:">signaling</span> occurs through <i>stumps</i> (Vincent et al., 1998), which regulates the phosphatase <i>corkscrew</i> (<i>csw</i>) (Petit et al., 2004). In turn, Csw negatively regulates <i>sprouty</i> (<i>sty</i>), itself a negative regulator of Raf, thereby leading to MAPK activation (Jarvis et al., 2006). (B) The lethal phenotype associated with mesoderm-specific <i>how24BGAL4</i>-directed expression of <i>UAS-Smn-RNAi</i><sup>C24</sup> is enhanced by the reduction of the FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components <i>htl</i>, <i>stumps</i>, and <i>breathless</i>. (C) <i>sprouty</i> alleles suppress the <i>how24BGAL4 UAS-Smn-RNAi</i> lethal phenotype. Significant differences are indicated (*, P &lt; 0.05; **, P &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Anindya Sen, et al. J Cell Biol. 2011 February 7;192(3):481-495.</div><p class="links"><a class="dblinks" href="/pubmed/21300852" ref="ordinalpos=2895&amp;ncbi_uid=3140539&amp;link_uid=21300852">Citation</a><a class="dblinks" href="/pmc/articles/PMC3101100/" ref="ordinalpos=2895&amp;ncbi_uid=3140539&amp;link_uid=PMC3101100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2896.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2814769/figure/F4/" ref="ordinalpos=2896&amp;ncbi_uid=2180893&amp;link_uid=PMC2814769" href="/pmc/articles/PMC2814769/"><img src="/pmc/articles/instance/2814769/bin/nihms168162f4.gif" src-large="/pmc/articles/instance/2814769/bin/nihms168162f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2814769/" ref="ordinalpos=2896&amp;ncbi_uid=2180893&amp;link_uid=PMC2814769" image-link="/pmc/articles/PMC2814769/figure/F4/" class="imagepopup">Figure 4. Hedgehog <span class="highlight" style="background-color:">signaling</span> and the cilium.  From: The Primary Cilium as a Complex <span class="highlight" style="background-color:">Signaling</span> Center. </a></p><div class="supp"><p class="details"><div>The Hedgehog (Hh) <span class="highlight" style="background-color:">pathway</span> in vertebrates utilizes the cilium as a <span class="highlight" style="background-color:">signaling</span> compartment. In the absence of the Hh ligand (A), the receptor Ptch resides in the cilium and through an unknown mechanism inhibits the transmembrane protein Smo. In the absence of ligand, the repressor forms of the Gli transcription factors inhibit Hh responsive gene transcription, while Gli activator is maintained in the cilia via Sufu binding.(B) Upon Hh binding, Ptch translocates out of the cilia membrane and its inhibition of Smo is alleviated. Activated Smo translocates into the cilia membrane and through unknown mechanisms the Gli activators translocate to the nucleus and activate Hh responsive gene expression.</div></p></div><div class="aux"><div class="resc">Nicolas F. Berbari, et al. Curr Biol. ;19(13):R526.</div><p class="links"><a class="dblinks" href="/pubmed/19602418" ref="ordinalpos=2896&amp;ncbi_uid=2180893&amp;link_uid=19602418">Citation</a><a class="dblinks" href="/pmc/articles/PMC2814769/" ref="ordinalpos=2896&amp;ncbi_uid=2180893&amp;link_uid=PMC2814769">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2897.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3544113/figure/F1/" ref="ordinalpos=2897&amp;ncbi_uid=5694190&amp;link_uid=PMC3544113" href="/pmc/articles/PMC3544113/"><img src="/pmc/articles/instance/3544113/bin/zh89991200730001.gif" src-large="/pmc/articles/instance/3544113/bin/zh89991200730001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3544113/" ref="ordinalpos=2897&amp;ncbi_uid=5694190&amp;link_uid=PMC3544113" image-link="/pmc/articles/PMC3544113/figure/F1/" class="imagepopup">Figure 1.  From: Outcome and pathologic classification of children and adolescents with mediastinal large B-cell lymphoma treated with FAB/LMB96 mature B-NHL therapy. </a></p><div class="supp"><p class="details"><div><b>Main deregulated <span class="highlight" style="background-color:">signaling</span> pathways in PMBCL.</b> Shown are the main activation cascades of JAK-STAT and NF-κB <span class="highlight" style="background-color:">signaling</span>. Alternative <span class="highlight" style="background-color:">pathway</span> activation exists. Known gene alterations leading to constitutive <span class="highlight" style="background-color:">pathway</span> activity in PMBCL are shown in color. Reprinted from Steidl and Gascoyne<sup>9</sup> by permission of the American Society of Hematology.</div></p></div><div class="aux"><div class="resc">Mary Gerrard, et al. Blood. 2013 January 10;121(2):278-285.</div><p class="links"><a class="dblinks" href="/pubmed/23149845" ref="ordinalpos=2897&amp;ncbi_uid=5694190&amp;link_uid=23149845">Citation</a><a class="dblinks" href="/pmc/articles/PMC3544113/" ref="ordinalpos=2897&amp;ncbi_uid=5694190&amp;link_uid=PMC3544113">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2898.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3387119/figure/fig06/" ref="ordinalpos=2898&amp;ncbi_uid=4101962&amp;link_uid=PMC3387119" href="/pmc/articles/PMC3387119/"><img src="/pmc/articles/instance/3387119/bin/pnas.1205210109fig06.gif" src-large="/pmc/articles/instance/3387119/bin/pnas.1205210109fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3387119/" ref="ordinalpos=2898&amp;ncbi_uid=4101962&amp;link_uid=PMC3387119" image-link="/pmc/articles/PMC3387119/figure/fig06/" class="imagepopup">Fig. 6. From: Perivascular-resident macrophage-like melanocytes in the inner ear are essential for the integrity of the intrastrial fluid-blood barrier. </a></p><div class="supp"><p class="details"><div>Working model of the PEDF/PEDFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating the expression of junction proteins. Capillary ECs are coupled by tight and adherens junctions that limit paracellular transport between adjacent cells. Circulating large molecules are extravasated from the capillary when PVM/Ms are absent. PVM/Ms control vascular permeability by affecting the expression of tight- and adherens-junction proteins through a PEDF/PEDFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Wenjing Zhang, et al. Proc Natl Acad Sci U S A. 2012 June 26;109(26):10388-10393.</div><p class="links"><a class="dblinks" href="/pubmed/22689949" ref="ordinalpos=2898&amp;ncbi_uid=4101962&amp;link_uid=22689949">Citation</a><a class="dblinks" href="/pmc/articles/PMC3387119/" ref="ordinalpos=2898&amp;ncbi_uid=4101962&amp;link_uid=PMC3387119">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2899.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2262896/figure/F4/" ref="ordinalpos=2899&amp;ncbi_uid=1260481&amp;link_uid=PMC2262896" href="/pmc/articles/PMC2262896/"><img src="/pmc/articles/instance/2262896/bin/1471-2164-9-71-4.gif" src-large="/pmc/articles/instance/2262896/bin/1471-2164-9-71-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2262896/" ref="ordinalpos=2899&amp;ncbi_uid=1260481&amp;link_uid=PMC2262896" image-link="/pmc/articles/PMC2262896/figure/F4/" class="imagepopup">Figure 4.  From: Activation of counter-regulatory mechanisms in a rat renal acute rejection model. </a></p><div class="supp"><p class="details"><div><b>TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Ingenuity Pathways Knowledge Base was used to display pathways containing multiple differently expressed genes. Significant up-regulation of genes in the canonical TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was observed after allogeneic transplantation. Up-regulated genes are highlighted in red the down-regulated genes are highlighted in green.</div></p></div><div class="aux"><div class="resc">Bayram Edemir, et al. BMC Genomics. 2008;9:71-71.</div><p class="links"><a class="dblinks" href="/pubmed/18261221" ref="ordinalpos=2899&amp;ncbi_uid=1260481&amp;link_uid=18261221">Citation</a><a class="dblinks" href="/pmc/articles/PMC2262896/" ref="ordinalpos=2899&amp;ncbi_uid=1260481&amp;link_uid=PMC2262896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2900.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC26920/figure/F3/" ref="ordinalpos=2900&amp;ncbi_uid=12471&amp;link_uid=PMC26920" href="/pmc/articles/PMC26920/"><img src="/pmc/articles/instance/26920/bin/pq1191208003.gif" src-large="/pmc/articles/instance/26920/bin/pq1191208003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC26920/" ref="ordinalpos=2900&amp;ncbi_uid=12471&amp;link_uid=PMC26920" image-link="/pmc/articles/PMC26920/figure/F3/" class="imagepopup">Figure 3.  From: Hydrogen peroxide is generated systemically in plant leaves by wounding and systemin via the octadecanoid <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The induction of H<sub>2</sub>O<sub>2</sub> production by wounding in leaves of (<i>A</i>) tomato plants transformed with a prosystemin antisense prosystemin gene and (<i>B</i>) a mutant tomato line (<i>def1</i>) that is compromised in the octadecanoid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. C, control; W, wounded; SYS, systemin; CHIT, chitosan. The plants were treated as in Fig. 1<i>C</i>.</div></p></div><div class="aux"><div class="resc">Martha Orozco-Cardenas, et al. Proc Natl Acad Sci U S A. 1999 May 25;96(11):6553-6557.</div><p class="links"><a class="dblinks" href="/pubmed/10339626" ref="ordinalpos=2900&amp;ncbi_uid=12471&amp;link_uid=10339626">Citation</a><a class="dblinks" href="/pmc/articles/PMC26920/" ref="ordinalpos=2900&amp;ncbi_uid=12471&amp;link_uid=PMC26920">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="28" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="29" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="30" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:36:56-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC4769543100010000000000EB401B;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC4769543100010000000000EB401B" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>