
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="23" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB75D35430F9910000000000ABCFF1" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="22" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="23" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="24" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="23" /></div>
            <div><h2 class="result_count">Results: 2201 to 2300 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2201.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4007526/figure/F4/" ref="ordinalpos=2201&amp;ncbi_uid=6103630&amp;link_uid=PMC4007526" href="/pmc/articles/PMC4007526/"><img src="/pmc/articles/instance/4007526/bin/jcav05p0382g004.gif" src-large="/pmc/articles/instance/4007526/bin/jcav05p0382g004.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4007526/" ref="ordinalpos=2201&amp;ncbi_uid=6103630&amp;link_uid=PMC4007526" image-link="/pmc/articles/PMC4007526/figure/F4/" class="imagepopup">Fig 4.  From: Carnosine Inhibits the Proliferation of Human Gastric Carcinoma Cells by Retarding Akt/mTOR/p70S6K <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Carnosine reduces the phosphorylation status of in AKT/mTOR <span class="highlight" style="background-color:">pathway</span>. The effect of carnosine on protein markers of AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was assessed by western blot after SGC-7901 (A) and MKN45 (B) cells were treatment with carnosine and rapamycin for 48 h. β-Actin was used as the loading control. The data are means ± SD of three independent experiments. *P&lt;0.05, **P&lt;0.01.</div></p></div><div class="aux"><div class="resc">Zhenwei Zhang, et al. J Cancer. 2014;5(5):382-389.</div><p class="links"><a class="dblinks" href="/pubmed/24799956" ref="ordinalpos=2201&amp;ncbi_uid=6103630&amp;link_uid=24799956">Citation</a><a class="dblinks" href="/pmc/articles/PMC4007526/" ref="ordinalpos=2201&amp;ncbi_uid=6103630&amp;link_uid=PMC4007526">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2202.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3320644/figure/pone-0034727-g007/" ref="ordinalpos=2202&amp;ncbi_uid=3899546&amp;link_uid=PMC3320644" href="/pmc/articles/PMC3320644/"><img src="/pmc/articles/instance/3320644/bin/pone.0034727.g007.gif" src-large="/pmc/articles/instance/3320644/bin/pone.0034727.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3320644/" ref="ordinalpos=2202&amp;ncbi_uid=3899546&amp;link_uid=PMC3320644" image-link="/pmc/articles/PMC3320644/figure/pone-0034727-g007/" class="imagepopup">Figure 7. The network of mating <span class="highlight" style="background-color:">pathway</span> in budding yeast deduced into minimal DoRA circuits..  From: Dose-Response Aligned Circuits in <span class="highlight" style="background-color:">Signaling</span> Systems. </a></p><div class="supp"><p class="details"><div>Left: the original mating <span class="highlight" style="background-color:">pathway</span> in budding yeast. Middle: the simplified <span class="highlight" style="background-color:">pathway</span> by coarse-graining. Right: the corresponding minimal DoRA circuits of two and three nodes in Figure 4.</div></p></div><div class="aux"><div class="resc">Long Yan, et al. PLoS One. 2012;7(4):e34727.</div><p class="links"><a class="dblinks" href="/pubmed/22496849" ref="ordinalpos=2202&amp;ncbi_uid=3899546&amp;link_uid=22496849">Citation</a><a class="dblinks" href="/pmc/articles/PMC3320644/" ref="ordinalpos=2202&amp;ncbi_uid=3899546&amp;link_uid=PMC3320644">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2203.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4065738/figure/fig3/" ref="ordinalpos=2203&amp;ncbi_uid=6313649&amp;link_uid=PMC4065738" href="/pmc/articles/PMC4065738/"><img src="/pmc/articles/instance/4065738/bin/BMRI2014-160692.003.gif" src-large="/pmc/articles/instance/4065738/bin/BMRI2014-160692.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4065738/" ref="ordinalpos=2203&amp;ncbi_uid=6313649&amp;link_uid=PMC4065738" image-link="/pmc/articles/PMC4065738/figure/fig3/" class="imagepopup">Figure 3.  From: The Role of Ubiquitination and Sumoylation in Diabetic Nephropathy. </a></p><div class="supp"><p class="details"><div>Regulation of NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by SUMO. Multiple signal transduction molecules of NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span>, such as I<i>κ</i>B<i>α</i>, NEMO, RelA, and P100, can be modified by SUMO. After SUMO-1 conjugated to NEMO, the IKK would be activated, similarly, SUMOylation I<i>κ</i>B<i>α</i> conjugated to the SUMO-1, and this would lead to the degradation of I<i>κ</i>B<i>α</i> and activation of the NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span>. And the SUMOylation of RelA and P100 was in accord with NEMO.</div></p></div><div class="aux"><div class="resc">Chenlin Gao, et al. Biomed Res Int. 2014;2014:160692.</div><p class="links"><a class="dblinks" href="/pubmed/24991536" ref="ordinalpos=2203&amp;ncbi_uid=6313649&amp;link_uid=24991536">Citation</a><a class="dblinks" href="/pmc/articles/PMC4065738/" ref="ordinalpos=2203&amp;ncbi_uid=6313649&amp;link_uid=PMC4065738">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2204.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4169577/figure/pone-0107950-g002/" ref="ordinalpos=2204&amp;ncbi_uid=6641978&amp;link_uid=PMC4169577" href="/pmc/articles/PMC4169577/"><img src="/pmc/articles/instance/4169577/bin/pone.0107950.g002.gif" src-large="/pmc/articles/instance/4169577/bin/pone.0107950.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4169577/" ref="ordinalpos=2204&amp;ncbi_uid=6641978&amp;link_uid=PMC4169577" image-link="/pmc/articles/PMC4169577/figure/pone-0107950-g002/" class="imagepopup">Figure 2. Infarct size expressed as a percentage of area at risk, as determined by TTC staining..  From: RISK and SAFE <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Involvement in Apolipoprotein A-I-Induced Cardioprotection. </a></p><div class="supp"><p class="details"><div>Results are expressed as mean±SEM. Wort: Wortmannin, an inhibitor of PI3 K/Akt <span class="highlight" style="background-color:">pathway</span>; U0126: an inhibitor of the MEK1/2-ERK1/2 <span class="highlight" style="background-color:">pathway</span>; AG490: an inhibitor of the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. *<i>p</i>&lt;0.05 <i>vs.</i> Control, n = 6–8 in each group.</div></p></div><div class="aux"><div class="resc">Hussein Kalakech, et al. PLoS One. 2014;9(9):e107950.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2204&amp;ncbi_uid=6641978&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4169577/" ref="ordinalpos=2204&amp;ncbi_uid=6641978&amp;link_uid=PMC4169577">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2205.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4362662/figure/Fig5/" ref="ordinalpos=2205&amp;ncbi_uid=7288731&amp;link_uid=PMC4362662" href="/pmc/articles/PMC4362662/"><img src="/pmc/articles/instance/4362662/bin/12885_2015_1119_Fig5_HTML.gif" src-large="/pmc/articles/instance/4362662/bin/12885_2015_1119_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4362662/" ref="ordinalpos=2205&amp;ncbi_uid=7288731&amp;link_uid=PMC4362662" image-link="/pmc/articles/PMC4362662/figure/Fig5/" class="imagepopup">Figure 5.  From: Resveratrol suppresses epithelial-to-mesenchymal transition in colorectal cancer through TGF-β1/Smads <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated Snail/E-cadherin expression. </a></p><div class="supp"><p class="details"><div> <b>Resveratrol repressed expressions of EMT-induced transcription factors and Smads <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Whole or cell extracts of different cellular compartments from LoVo cells were probed for <b>(A)</b> Slug Snail, ZEB1, Twist, and <b>(B)</b> p-Smad2/3, Smad2/3. **<i>P</i> &lt; 0.01, compared with group without treatment of TGF-β1 and resveratrol. <b>(C)</b> LoVo cells treated with TGF-β1 or (and) TGFβRI/II inhibitor (LY2109761) were analyzed for the Snail and E-cadherin mRNA expression by RT-PCR, GAPDH was chosen as a control. <b>(D)</b> Whole or cell extracts of different cellular compartments from LoVo cells were probed for p-Smad2/3, Smad2/3, Snail and E-cadherin expression. **<i>P</i> &lt; 0.01, compared with group without treatment of TGF-β1 and LY2109761.</div></p></div><div class="aux"><div class="resc">Qing Ji, et al. BMC Cancer. 2015;15:97.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2205&amp;ncbi_uid=7288731&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4362662/" ref="ordinalpos=2205&amp;ncbi_uid=7288731&amp;link_uid=PMC4362662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2206.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3343135/figure/fig1-1933719110393025/" ref="ordinalpos=2206&amp;ncbi_uid=4084206&amp;link_uid=PMC3343135" href="/pmc/articles/PMC3343135/"><img src="/pmc/articles/instance/3343135/bin/10.1177_1933719110393025-fig1.gif" src-large="/pmc/articles/instance/3343135/bin/10.1177_1933719110393025-fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3343135/" ref="ordinalpos=2206&amp;ncbi_uid=4084206&amp;link_uid=PMC3343135" image-link="/pmc/articles/PMC3343135/figure/fig1-1933719110393025/" class="imagepopup">Figure 1. From: Maternal Undernourished Fetal Kidneys Exhibit Differential Regulation of Nephrogenic Genes Including Downregulation of the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Protein expression of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> factors as determined by Western blotting. Five to six Control versus MUN kidney lysates; each protein probed with a specific primary antibody. β-actin represents a housekeeping protein that is not affected by maternal undernutrition (MUN) and was used for normalization of protein loading. Representative bands for the target protein and b-actin are depicted. *<i>P</i> ≤ .05, ** <i>P</i> ≤ .01.</div></p></div><div class="aux"><div class="resc">Thomas R. Magee, et al. Reprod Sci. 2011 June;18(6):563-576.</div><p class="links"><a class="dblinks" href="/pubmed/21273641" ref="ordinalpos=2206&amp;ncbi_uid=4084206&amp;link_uid=21273641">Citation</a><a class="dblinks" href="/pmc/articles/PMC3343135/" ref="ordinalpos=2206&amp;ncbi_uid=4084206&amp;link_uid=PMC3343135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2207.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3795306/figure/F2/" ref="ordinalpos=2207&amp;ncbi_uid=5389542&amp;link_uid=PMC3795306" href="/pmc/articles/PMC3795306/"><img src="/pmc/articles/instance/3795306/bin/fncir-07-00152-g002.gif" src-large="/pmc/articles/instance/3795306/bin/fncir-07-00152-g002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3795306/" ref="ordinalpos=2207&amp;ncbi_uid=5389542&amp;link_uid=PMC3795306" image-link="/pmc/articles/PMC3795306/figure/F2/" class="imagepopup">FIGURE 2.  From: Dopamine <span class="highlight" style="background-color:">Signaling</span> in reward-related behaviors. </a></p><div class="supp"><p class="details"><div><b>D2 receptor-mediated ERK activation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> D2 receptor-mediated ERK activation is dependent on Gα<sub>i</sub> protein coupling. It also appears that D2 receptor-mediated ERK activation requires the transactivation of receptor tyrosine kinase, which consequently activates downstream <span class="highlight" style="background-color:">signaling</span> involving matrix metalloproteinases (MMPs) with ectodomain shedding of EGFR ligand, for example, to finally activate ERK (Choi et al., 1999; Kim et al., 2004; Wang et al., 2005; ). The involvement of arrestin has also been suggested to contribute to D2 receptor-mediated ERK activation (Beom et al., 2004; Kim et al., 2004), which can activate MAPK <span class="highlight" style="background-color:">signaling</span> by mobilizing clathrin-mediated endocytosis in a β-arrestin/dynamin-dependent manner (Kim et al., 2004).</div></p></div><div class="aux"><div class="resc">Ja-Hyun Baik. Front Neural Circuits. 2013;7:152.</div><p class="links"><a class="dblinks" href="/pubmed/24130517" ref="ordinalpos=2207&amp;ncbi_uid=5389542&amp;link_uid=24130517">Citation</a><a class="dblinks" href="/pmc/articles/PMC3795306/" ref="ordinalpos=2207&amp;ncbi_uid=5389542&amp;link_uid=PMC3795306">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2208.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2943488/figure/F1/" ref="ordinalpos=2208&amp;ncbi_uid=3060512&amp;link_uid=PMC2943488" href="/pmc/articles/PMC2943488/"><img src="/pmc/articles/instance/2943488/bin/nihms-230584-f0001.gif" src-large="/pmc/articles/instance/2943488/bin/nihms-230584-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2943488/" ref="ordinalpos=2208&amp;ncbi_uid=3060512&amp;link_uid=PMC2943488" image-link="/pmc/articles/PMC2943488/figure/F1/" class="imagepopup">Figure 1.  From: Cytokine-Associated Drug Toxicity in Human Hepatocytes Is Associated with <span class="highlight" style="background-color:">Signaling</span> Network Dysregulation. </a></p><div class="supp"><p class="details"><div>A schematic of the drug- and cytokine-induced hepatocellular death <span class="highlight" style="background-color:">signaling</span> network, curated from literature references6-14, 19-21. Inflammatory cytokines bind to their cognate receptors and activate shared downstream phosphoprotein <span class="highlight" style="background-color:">signaling</span> pathways regulating cell death. Hepatotoxic drugs (hepatotoxicants) lead to various cellular stresses (<i>e.g.</i> ROS accumulation) that are integrated into stress-related phosphoprotein <span class="highlight" style="background-color:">signaling</span> (<i>e.g.</i> JNK <span class="highlight" style="background-color:">pathway</span>) and mitochondrial control of apoptosis. Phosphoproteins and cell death phenotypes experimentally assayed in this study are indicated.</div></p></div><div class="aux"><div class="resc">Benjamin D. Cosgrove, et al. Mol Biosyst. ;6(7):1195-1206.</div><p class="links"><a class="dblinks" href="/pubmed/20361094" ref="ordinalpos=2208&amp;ncbi_uid=3060512&amp;link_uid=20361094">Citation</a><a class="dblinks" href="/pmc/articles/PMC2943488/" ref="ordinalpos=2208&amp;ncbi_uid=3060512&amp;link_uid=PMC2943488">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2209.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3180925/figure/Fig2/" ref="ordinalpos=2209&amp;ncbi_uid=3287582&amp;link_uid=PMC3180925" href="/pmc/articles/PMC3180925/"><img src="/pmc/articles/instance/3188288/bin/11689_2011_9083_Fig2_HTML.gif" src-large="/pmc/articles/instance/3188288/bin/11689_2011_9083_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3180925/" ref="ordinalpos=2209&amp;ncbi_uid=3287582&amp;link_uid=PMC3180925" image-link="/pmc/articles/PMC3180925/figure/Fig2/" class="imagepopup">Fig. 2.  From: Synaptic Wnt <span class="highlight" style="background-color:">signaling</span>--a contributor to major psychiatric disorders?. </a></p><div class="supp"><p class="details"><div>Wnts exert both pre- and post-synaptic effects. Presynaptic Wnt <span class="highlight" style="background-color:">signaling</span> through Wnt3 and Wnt7 acts through a Dvl <span class="highlight" style="background-color:">pathway</span> to stabilize microtubules and also influences presynaptic clustering of acetylcholine receptors (<i>AChRs</i>) through an APC-mediated mechanism (<i>upper left</i>). Presynaptic β-catenin is also involved in synapse formation and maintenance through cell–cell adhesion mechanisms, though a relationship between this and Wnt <span class="highlight" style="background-color:">signaling</span> remains speculative. Wnt5a-activated <span class="highlight" style="background-color:">signaling</span> through the Ror kinase is also thought to contribute to an increase in presynaptic sites through a mechanism that has not yet been determined (<i>upper right</i>). Postsynaptic Wnt <span class="highlight" style="background-color:">signaling</span> driven by Wnt5a probably acts through a Dvl/Rac/JNK <span class="highlight" style="background-color:">pathway</span> to regulate the actin cytoskeleton (<i>lower right</i>). Application of Wnt3a, Wnt 7a, or Wnt7b is sufficient to induce β-catenin translocation to the nucleus and may well also influence synapses (not shown). As in the presynaptic terminal, β-catenin is present in the postsynaptic density of excitatory synapses where it is involved in cell–cell adhesion. Also not shown is a highly divergent postsynaptic <span class="highlight" style="background-color:">pathway</span> established at the <i>Drosophila</i> neuromuscular junction involving cleavage and translocation of Fzd receptors to the nucleus; there is evidence that a mechanistically similar <span class="highlight" style="background-color:">pathway</span> operates at some vertebrate synapses</div></p></div><div class="aux"><div class="resc">Nathan D. Okerlund, et al. J Neurodev Disord. 2011 June;3(2):162-174.</div><p class="links"><a class="dblinks" href="/pubmed/21533542" ref="ordinalpos=2209&amp;ncbi_uid=3287582&amp;link_uid=21533542">Citation</a><a class="dblinks" href="/pmc/articles/PMC3180925/" ref="ordinalpos=2209&amp;ncbi_uid=3287582&amp;link_uid=PMC3180925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2210.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2842040/figure/fig05/" ref="ordinalpos=2210&amp;ncbi_uid=2456830&amp;link_uid=PMC2842040" href="/pmc/articles/PMC2842040/"><img src="/pmc/articles/instance/2842040/bin/pnas.0906384107fig05.gif" src-large="/pmc/articles/instance/2842040/bin/pnas.0906384107fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2842040/" ref="ordinalpos=2210&amp;ncbi_uid=2456830&amp;link_uid=PMC2842040" image-link="/pmc/articles/PMC2842040/figure/fig05/" class="imagepopup">Fig. 5. From: Interactome analysis of longitudinal pharyngeal infection of cynomolgus macaques by group A Streptococcus . </a></p><div class="supp"><p class="details"><div>Relationship between increased production of IPP by GAS and <span class="highlight" style="background-color:">signaling</span> of γδ T cells. Up-regulation of mevalonic acid <span class="highlight" style="background-color:">pathway</span> genes (<i>mvaS.1</i>, <i>spy0686</i>, <i>mvaK1</i>, <i>mvaK2</i>, <i>mvaD</i>) results in increased production of IPP, which binds to the TCR of host γδ T cells present in the mucosal epithelium. Binding of IPP to the TCR stimulates recruitment of lipid rafts necessary for TCR <span class="highlight" style="background-color:">signaling</span>. Host genes involved in organization of lipid raft and receptor clustering, such as Agrin, Flotillin-2, and <i>NrCAM</i>, show increased expression levels to coordinate TCR and coreceptor <span class="highlight" style="background-color:">signaling</span>. The activated TCR initiates a signal transduction cascade leading to increased expression of host <i>MAP3KIP1</i> and <i>GADD45B</i>, both members of the p38 MAP kinase <span class="highlight" style="background-color:">pathway</span>, and <i>MAP4K1</i>, a hematopoietic-specific <span class="highlight" style="background-color:">signaling</span> kinase that activates the cellular stress activated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (SAPK1). Gamma-delta T cells are key defenders of the host mucosal surface and TCR <span class="highlight" style="background-color:">signaling</span> leading to increased proliferation or T-cell anergy is likely to influence the immune response to GAS.</div></p></div><div class="aux"><div class="resc">Patrick R. Shea, et al. Proc Natl Acad Sci U S A. 2010 March 9;107(10):4693-4698.</div><p class="links"><a class="dblinks" href="/pubmed/20179180" ref="ordinalpos=2210&amp;ncbi_uid=2456830&amp;link_uid=20179180">Citation</a><a class="dblinks" href="/pmc/articles/PMC2842040/" ref="ordinalpos=2210&amp;ncbi_uid=2456830&amp;link_uid=PMC2842040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2211.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2731733/figure/F5/" ref="ordinalpos=2211&amp;ncbi_uid=1998243&amp;link_uid=PMC2731733" href="/pmc/articles/PMC2731733/"><img src="/pmc/articles/instance/2731733/bin/1752-0509-3-80-5.gif" src-large="/pmc/articles/instance/2731733/bin/1752-0509-3-80-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2731733/" ref="ordinalpos=2211&amp;ncbi_uid=1998243&amp;link_uid=PMC2731733" image-link="/pmc/articles/PMC2731733/figure/F5/" class="imagepopup">Figure 5.  From: Knowledge based identification of essential <span class="highlight" style="background-color:">signaling</span> from genome-scale siRNA experiments. </a></p><div class="supp"><p class="details"><div><b>The Cytoskeletal Remodeling: FAK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> is enriched for HeLa and TOV edge hits using PIPA but not node hits using HGA enrichment</b>. Red indicators are HeLa siRNA hits and blue indicators are TOV hits. Both HeLa and TOV show significant enrichment of protein interactions connecting hits (p-value = 0.048 and p-value = 0.015 respectively). Different <span class="highlight" style="background-color:">signaling</span> branches in this <span class="highlight" style="background-color:">pathway</span> have lethal effects in the two different cell lines when perturbed, although both branches are annotated as associated with a cell growth phenotype. [GeneGo Metacore (2008)].</div></p></div><div class="aux"><div class="resc">Armand Bankhead, et al. BMC Syst Biol. 2009;3:80-80.</div><p class="links"><a class="dblinks" href="/pubmed/19653913" ref="ordinalpos=2211&amp;ncbi_uid=1998243&amp;link_uid=19653913">Citation</a><a class="dblinks" href="/pmc/articles/PMC2731733/" ref="ordinalpos=2211&amp;ncbi_uid=1998243&amp;link_uid=PMC2731733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2212.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2670105/figure/fig1/" ref="ordinalpos=2212&amp;ncbi_uid=1704628&amp;link_uid=PMC2670105" href="/pmc/articles/PMC2670105/"><img src="/pmc/articles/instance/2670105/bin/zbc0240967280001.gif" src-large="/pmc/articles/instance/2670105/bin/zbc0240967280001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2670105/" ref="ordinalpos=2212&amp;ncbi_uid=1704628&amp;link_uid=PMC2670105" image-link="/pmc/articles/PMC2670105/figure/fig1/" class="imagepopup">FIGURE 1. From: <span class="highlight" style="background-color:">Signaling</span> Threshold Regulation by the Ras Effector  IMP. </a></p><div class="supp"><p class="details"><div><b>IMP impedes Ras mitogenic <span class="highlight" style="background-color:">signaling</span> by limiting KSR1 complexes.</b> In  the absence of mitogen, IMP blocks KSR1 C-terminal homo-oligomerization  (<i>Step 1</i>), KSR1-c-Raf binding (<i>Step 2</i>), B-Raf-MEK binding  without affecting their association with KSR1 (<i>Step 3</i>), and  B-Raf-c-Raf hetero-oligomerization (<i>Step 4</i>). Upon stimulation,  Ras-GTP binds IMP and activates its autoubiquitination, thus relieving signal  inhibition. Concomitantly, Ras-GTP activates Raf and allows subsequent KSR1  complex formation and ERK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Sharon A. Matheny, et al. J Biol Chem. 2009 April 24;284(17):11007-11011.</div><p class="links"><a class="dblinks" href="/pubmed/19091743" ref="ordinalpos=2212&amp;ncbi_uid=1704628&amp;link_uid=19091743">Citation</a><a class="dblinks" href="/pmc/articles/PMC2670105/" ref="ordinalpos=2212&amp;ncbi_uid=1704628&amp;link_uid=PMC2670105">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2213.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3374974/figure/F2/" ref="ordinalpos=2213&amp;ncbi_uid=4581094&amp;link_uid=PMC3374974" href="/pmc/articles/PMC3374974/"><img src="/pmc/articles/instance/3374974/bin/nihms-339335-f0002.gif" src-large="/pmc/articles/instance/3374974/bin/nihms-339335-f0002.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3374974/figure/F2/" ref="ordinalpos=2213&amp;ncbi_uid=4581094&amp;link_uid=PMC3374974" href="/pmc/articles/PMC3374974/"><img src="/pmc/articles/instance/3374974/bin/nihms-339335-f0003.gif" src-large="/pmc/articles/instance/3374974/bin/nihms-339335-f0003.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3374974/figure/F2/" ref="ordinalpos=2213&amp;ncbi_uid=4581094&amp;link_uid=PMC3374974" href="/pmc/articles/PMC3374974/"><img src="/pmc/articles/instance/3374974/bin/nihms-339335-f0004.gif" src-large="/pmc/articles/instance/3374974/bin/nihms-339335-f0004.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3374974/" ref="ordinalpos=2213&amp;ncbi_uid=4581094&amp;link_uid=PMC3374974" image-link="/pmc/articles/PMC3374974/figure/F2/" class="imagepopup">Fig. 2.  From: EphA Receptor <span class="highlight" style="background-color:">Signaling</span> - Complexity and Emerging Themes. </a></p><div class="supp"><p class="details"><div>EphA2 <span class="highlight" style="background-color:">signaling</span> in normal epithelial cells and tumor cells. <b>A.</b> Under normal conditions, EphA2 and ephrin-A are properly expressed and engaged with each other. The <span class="highlight" style="background-color:">signaling</span> downstream from EphA2 activation by ligand engagement counteracts growth factor <span class="highlight" style="background-color:">signaling</span> by inhibiting activation of Ras/ERK and PI3K/Akt, which contributes to the maintenance of epithelial homeostasis. <b>B.</b> In EphA2-null epithelial cells, the negative regulator of growth factor <span class="highlight" style="background-color:">signaling</span> is disrupted, which renders epithelial cells more susceptible to carcinogen insults leading to tumor development. <b>C.</b> With tumor progression, EphA2 can be highly upregulated whereas ephrin-A is not or downregulated, which resulting in excess of unligated EphA2. Unligated EphA2 can serve as a substrate of Akt becoming an integral component of growth factor <span class="highlight" style="background-color:">pathway</span> to promote tumor cell migration and invasion. At the same time, unligated EphA2 is incapable of suppressing Ras/ERK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hui Miao, et al. Semin Cell Dev Biol. ;23(1):16-25.</div><p class="links"><a class="dblinks" href="/pubmed/22040915" ref="ordinalpos=2213&amp;ncbi_uid=4581094&amp;link_uid=22040915">Citation</a><a class="dblinks" href="/pmc/articles/PMC3374974/" ref="ordinalpos=2213&amp;ncbi_uid=4581094&amp;link_uid=PMC3374974">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2214.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3233625/figure/F3/" ref="ordinalpos=2214&amp;ncbi_uid=4265561&amp;link_uid=PMC3233625" href="/pmc/articles/PMC3233625/"><img src="/pmc/articles/instance/3233625/bin/cc1019_3274_fig003.gif" src-large="/pmc/articles/instance/3233625/bin/cc1019_3274_fig003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3233625/" ref="ordinalpos=2214&amp;ncbi_uid=4265561&amp;link_uid=PMC3233625" image-link="/pmc/articles/PMC3233625/figure/F3/" class="imagepopup">Figure 3.  From: Pleiotropic functions of EAPII/TTRAP/TDP2. </a></p><div class="supp"><p class="details"><div>Model of EAPII-mediated signal transduction: (1) EAPII negatively modulates TNFα <span class="highlight" style="background-color:">signaling</span>; (2) EAPII facilitates or represses the JNK-mediated apoptosis <span class="highlight" style="background-color:">pathway</span>, depending on the genotype of DJ-1 protein; (3) EAPII activates MAPK-ERK <span class="highlight" style="background-color:">signaling</span>; and (4) EAPII negatively modulates Nodal <span class="highlight" style="background-color:">signaling</span> through Smad3 association.</div></p></div><div class="aux"><div class="resc">Chunyang Li, et al. Cell Cycle. 2011 October 1;10(19):3274-3283.</div><p class="links"><a class="dblinks" href="/pubmed/21926483" ref="ordinalpos=2214&amp;ncbi_uid=4265561&amp;link_uid=21926483">Citation</a><a class="dblinks" href="/pmc/articles/PMC3233625/" ref="ordinalpos=2214&amp;ncbi_uid=4265561&amp;link_uid=PMC3233625">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2215.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013053/figure/pone-0096737-g004/" ref="ordinalpos=2215&amp;ncbi_uid=6117372&amp;link_uid=PMC4013053" href="/pmc/articles/PMC4013053/"><img src="/pmc/articles/instance/4013053/bin/pone.0096737.g004.gif" src-large="/pmc/articles/instance/4013053/bin/pone.0096737.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013053/" ref="ordinalpos=2215&amp;ncbi_uid=6117372&amp;link_uid=PMC4013053" image-link="/pmc/articles/PMC4013053/figure/pone-0096737-g004/" class="imagepopup">Figure 4. NOD1 <span class="highlight" style="background-color:">pathway</span> inhibitors block NF-κB and MAPK pathways..  From: Identification of Selective Small Molecule Inhibitors of the Nucleotide-Binding Oligomerization Domain 1 (NOD1) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Serum-starved 293/hNOD1 (A) or 293/hNOD2 (B) stable cells were pre-incubated with compounds at the indicated concentrations prior to stimulation for 1 hour with either Tri-DAP (50 µg/mL) or MDP (25 µg/mL), respectively. Compounds included the three selective NOD1 <span class="highlight" style="background-color:">pathway</span> inhibitors or previously identified inhibitors of NOD2 <span class="highlight" style="background-color:">signaling</span> (GSK669 and GSK400). Levels of total IκBα, as well as total and phosphorylated MAPKs (p38, JNK and ERK1/2) were determined by immunoblotting of whole cell lysates. A RIP2 inhibitor was used as a positive control with 293/hNOD2 cells in which the NOD1 <span class="highlight" style="background-color:">pathway</span> inhibitors had no effect on MDP responses. Results shown are representative of two separate experiments performed in both cell lines.</div></p></div><div class="aux"><div class="resc">David J. Rickard, et al. PLoS One. 2014;9(5):e96737.</div><p class="links"><a class="dblinks" href="/pubmed/24806487" ref="ordinalpos=2215&amp;ncbi_uid=6117372&amp;link_uid=24806487">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013053/" ref="ordinalpos=2215&amp;ncbi_uid=6117372&amp;link_uid=PMC4013053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2216.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3510657/figure/F1/" ref="ordinalpos=2216&amp;ncbi_uid=4400580&amp;link_uid=PMC3510657" href="/pmc/articles/PMC3510657/"><img src="/pmc/articles/instance/3510657/bin/nihms421042f1.gif" src-large="/pmc/articles/instance/3510657/bin/nihms421042f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3510657/" ref="ordinalpos=2216&amp;ncbi_uid=4400580&amp;link_uid=PMC3510657" image-link="/pmc/articles/PMC3510657/figure/F1/" class="imagepopup">Figure 1.  From: The Role of XBP-1 in Tumorigenicity. </a></p><div class="supp"><p class="details"><div>Schematic diagram of XBP-1 in the UPR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. XBP-1 is located at a critical junction in this <span class="highlight" style="background-color:">pathway</span> that controls pro-death and pro-survival signals.</div></p></div><div class="aux"><div class="resc">Ayesha N. Shajahan, et al. Drug News Perspect. ;22(5):241-246.</div><p class="links"><a class="dblinks" href="/pubmed/19609461" ref="ordinalpos=2216&amp;ncbi_uid=4400580&amp;link_uid=19609461">Citation</a><a class="dblinks" href="/pmc/articles/PMC3510657/" ref="ordinalpos=2216&amp;ncbi_uid=4400580&amp;link_uid=PMC3510657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2217.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678733/figure/F3/" ref="ordinalpos=2217&amp;ncbi_uid=6159683&amp;link_uid=PMC3678733" href="/pmc/articles/PMC3678733/"><img src="/pmc/articles/instance/3678733/bin/nihms462970f3.gif" src-large="/pmc/articles/instance/3678733/bin/nihms462970f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678733/" ref="ordinalpos=2217&amp;ncbi_uid=6159683&amp;link_uid=PMC3678733" image-link="/pmc/articles/PMC3678733/figure/F3/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Pathway</span> Databases: Making Chemical and Biological Sense of the Genomic Data Flood. </a></p><div class="supp"><p class="details"><div>Visualization of curated components of pathways and inferred interacting proteins and small molecules. A. A diagram of the Hippo/STK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as annotated in Reactome, using SBGN-like iconography to display proteins and complexes (rectangles) and small molecules (ovals), connected by reaction edges that distinguish inputs (plain end), outputs (arrowhead end) and catalysis (open-circle end) as well as reaction type (open box reaction node, chemical transformation; closed circle, binding). B. A portion of the manually annotated <span class="highlight" style="background-color:">pathway</span> is enlarged and overlaid with proteins found in high-throughput surveys (Wu et al. 2010) to interact with Hippo <span class="highlight" style="background-color:">pathway</span> components STK3 (MST2) or STK4 (MST1) or both. C) The manually annotated <span class="highlight" style="background-color:">pathway</span> is overlaid with small molecules from the ChEMBL database (Gaulton et al. 2012) that bind STK3 (MST2) or STK4 (MST1) or both.</div></p></div><div class="aux"><div class="resc">Peter D’Eustachio. Chem Biol. ;20(5):629-635.</div><p class="links"><a class="dblinks" href="/pubmed/23706629" ref="ordinalpos=2217&amp;ncbi_uid=6159683&amp;link_uid=23706629">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678733/" ref="ordinalpos=2217&amp;ncbi_uid=6159683&amp;link_uid=PMC3678733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2218.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1083689/figure/kvd014f1/" ref="ordinalpos=2218&amp;ncbi_uid=398637&amp;link_uid=PMC1083689" href="/pmc/articles/PMC1083689/"><img src="/pmc/articles/instance/1083689/bin/kvd01401.gif" src-large="/pmc/articles/instance/1083689/bin/kvd01401.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1083689/" ref="ordinalpos=2218&amp;ncbi_uid=398637&amp;link_uid=PMC1083689" image-link="/pmc/articles/PMC1083689/figure/kvd014f1/" class="imagepopup">Figure. From: The ins and outs of protein phosphorylation. </a></p><div class="supp"><p class="details"><div><b>Fig. 1.</b> Pathways controlling cell cycle progression and survival. Two <span class="highlight" style="background-color:">signaling</span> cascades emanate from the IRS molecules. One may signal survival, cell growth, and cell cycle progression through the PI3K branch and another, which constitutes a cooperative or overlapping <span class="highlight" style="background-color:">pathway</span>, operates through the MAPK cascade. The former can be down-regulated by the lipid phosphatase PTEN and acts independently of the MAPK <span class="highlight" style="background-color:">pathway</span>. Its effects on S6K1 regulate the translational machinery, influencing cell growth and cell division. The MAPK <span class="highlight" style="background-color:">pathway</span> may use different CREB kinases in a context-specific manner.</div></p></div><div class="aux"><div class="resc">George Thomas, et al. EMBO Rep. 2000 July 17;1(1):11-15.</div><p class="links"><a class="dblinks" href="/pubmed/11256615" ref="ordinalpos=2218&amp;ncbi_uid=398637&amp;link_uid=11256615">Citation</a><a class="dblinks" href="/pmc/articles/PMC1083689/" ref="ordinalpos=2218&amp;ncbi_uid=398637&amp;link_uid=PMC1083689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2219.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3593943/figure/F6/" ref="ordinalpos=2219&amp;ncbi_uid=6080555&amp;link_uid=PMC3593943" href="/pmc/articles/PMC3593943/"><img src="/pmc/articles/instance/3593943/bin/nihms424364f6.gif" src-large="/pmc/articles/instance/3593943/bin/nihms424364f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3593943/" ref="ordinalpos=2219&amp;ncbi_uid=6080555&amp;link_uid=PMC3593943" image-link="/pmc/articles/PMC3593943/figure/F6/" class="imagepopup">Fig. 6. AP-1 Complex mediates inflammatory <span class="highlight" style="background-color:">signaling</span> via TLR3 and MAP3K8.  From: Transcription Factor Complex AP-1 Mediates Inflammation Initiated by Chlamydia pneumoniae Infection. </a></p><div class="supp"><p class="details"><div>A. JUN and FOS knockdown by siRNA dramatically alters gene expression of TLR3 in <i>C. pneumoniae</i> infected HCAEC and THP-1 cells as measured by qRT-PCR.<br />B. Expression of AP-1 components correlates with MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during <i>C. pneumoniae</i> infection of HCAEC measured by both microarray and qRT-PCR.</div></p></div><div class="aux"><div class="resc">Anyou Wang, et al. Cell Microbiol. 2013 May;15(5):779-794.</div><p class="links"><a class="dblinks" href="/pubmed/23163821" ref="ordinalpos=2219&amp;ncbi_uid=6080555&amp;link_uid=23163821">Citation</a><a class="dblinks" href="/pmc/articles/PMC3593943/" ref="ordinalpos=2219&amp;ncbi_uid=6080555&amp;link_uid=PMC3593943">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2220.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109866/figure/F1/" ref="ordinalpos=2220&amp;ncbi_uid=3006878&amp;link_uid=PMC3109866" href="/pmc/articles/PMC3109866/"><img src="/pmc/articles/instance/3109866/bin/nihms296160f1.gif" src-large="/pmc/articles/instance/3109866/bin/nihms296160f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109866/" ref="ordinalpos=2220&amp;ncbi_uid=3006878&amp;link_uid=PMC3109866" image-link="/pmc/articles/PMC3109866/figure/F1/" class="imagepopup">Figure 1. Interactions between components of the insulin/IGF <span class="highlight" style="background-color:">signaling</span> and the TOR pathways.  From: Converging Pathways in Lifespan Regulation. </a></p><div class="supp"><p class="details"><div>The insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> (red ovals) and TOR (beige ovals) pathways may interact at multiple levels to ultimately regulate organismal lifespan. Dual coloring indicates proteins implicated in both pathways. White ovals are proteins that are not typically viewed as being in one <span class="highlight" style="background-color:">pathway</span> or the other. The proteins are labeled with the worm name followed by the mammalian name.</div></p></div><div class="aux"><div class="resc">Sri Devi Narasimhan, et al. Curr Biol. ;19(15):R657-R666.</div><p class="links"><a class="dblinks" href="/pubmed/19674551" ref="ordinalpos=2220&amp;ncbi_uid=3006878&amp;link_uid=19674551">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109866/" ref="ordinalpos=2220&amp;ncbi_uid=3006878&amp;link_uid=PMC3109866">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2221.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC187495/figure/F4/" ref="ordinalpos=2221&amp;ncbi_uid=220729&amp;link_uid=PMC187495" href="/pmc/articles/PMC187495/"><img src="/pmc/articles/instance/187495/bin/55671-10f4_L1TT.gif" src-large="/pmc/articles/instance/187495/bin/55671-10f4_L1TT.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC187495/" ref="ordinalpos=2221&amp;ncbi_uid=220729&amp;link_uid=PMC187495" image-link="/pmc/articles/PMC187495/figure/F4/" class="imagepopup">Figure 4.  From: The DAF-7 TGF-? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates chemosensory receptor gene expression in C. elegans. </a></p><div class="supp"><p class="details"><div>Model for regulation of chemoreceptor expression. (<i>A</i>) In the ASI neurons, DAF-7 TGF-β <span class="highlight" style="background-color:">signaling</span> maintains the expression of <i>sra-6</i>, promotes the expression of <i>str-3</i> and <i>srd-1</i>, and represses expression of <i>str-2</i>. High levels of dauer pheromone repress <i>daf-7</i> expression and promote dauer entry. Sub-dauer-inducing concentrations of pheromone (indicated by the dashed line) repress the expression of <i>sra-6</i>, <i>str-3, str-2</i>, and presumably <i>srd-1</i> (in box) via a DAF-7-independent <span class="highlight" style="background-color:">pathway</span>. In addition, the DAF-12 nuclear hormone receptor is required to promote <i>sra-6</i> expression and to repress <i>str-2</i> expression in the ASI neurons. <i>srd-1</i> expression also requires neuronal activity (Peckol et al. 2001). (<i>B</i>) In the ASH neurons, DAF-7 <span class="highlight" style="background-color:">signaling</span> via the DAF-3 inhibitory SMAD but not DAF-12 maintains <i>sra-6</i> expression.</div></p></div><div class="aux"><div class="resc">Katherine M. Nolan, et al. Genes Dev. 2002 December 1;16(23):3061-3073.</div><p class="links"><a class="dblinks" href="/pubmed/12464635" ref="ordinalpos=2221&amp;ncbi_uid=220729&amp;link_uid=12464635">Citation</a><a class="dblinks" href="/pmc/articles/PMC187495/" ref="ordinalpos=2221&amp;ncbi_uid=220729&amp;link_uid=PMC187495">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2222.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC126171/figure/cdf433f5/" ref="ordinalpos=2222&amp;ncbi_uid=202982&amp;link_uid=PMC126171" href="/pmc/articles/PMC126171/"><img src="/pmc/articles/instance/126171/bin/cdf433f5.gif" src-large="/pmc/articles/instance/126171/bin/cdf433f5.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC126171/" ref="ordinalpos=2222&amp;ncbi_uid=202982&amp;link_uid=PMC126171" image-link="/pmc/articles/PMC126171/figure/cdf433f5/" class="imagepopup">Figure. From: Different telomere damage <span class="highlight" style="background-color:">signaling</span> pathways in human and mouse cells. </a></p><div class="supp"><p class="details"><div><b>Fig. 5.</b> Schematic illustrating the difference in the telomere damage <span class="highlight" style="background-color:">signaling</span> pathways in human and mouse cells. Telomere damage created by telomere shortening or removal of TRF2 is proposed to activate the same <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The sensor(s) upstream of p16 and p53 have not been identified. In human cells, cell cycle arrest in response to telomere damage can be transduced through either the p53 or the p16/Rb pathways. In mouse cells, the telomere damage signal is transduced only through the p53 <span class="highlight" style="background-color:">pathway</span>. In the absence of p53, the p16/Rb <span class="highlight" style="background-color:">pathway</span> alone is insufficient to induce cell cycle arrest in response to telomere damage (shading). The arrow from p16 to p21 illustrates the ability of p16 to displace p21 from the CDK4/6 complex, resulting in a release of a latent pool of p21 that can inhibit cycE/A-dependent CDK2.</div></p></div><div class="aux"><div class="resc">Agata Smogorzewska, et al. EMBO J. 2002 August 15;21(16):4338-4348.</div><p class="links"><a class="dblinks" href="/pubmed/12169636" ref="ordinalpos=2222&amp;ncbi_uid=202982&amp;link_uid=12169636">Citation</a><a class="dblinks" href="/pmc/articles/PMC126171/" ref="ordinalpos=2222&amp;ncbi_uid=202982&amp;link_uid=PMC126171">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2223.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2700405/figure/F1/" ref="ordinalpos=2223&amp;ncbi_uid=1878952&amp;link_uid=PMC2700405" href="/pmc/articles/PMC2700405/"><img src="/pmc/articles/instance/2700405/bin/WJG-14-1823-g001.gif" src-large="/pmc/articles/instance/2700405/bin/WJG-14-1823-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2700405/" ref="ordinalpos=2223&amp;ncbi_uid=1878952&amp;link_uid=PMC2700405" image-link="/pmc/articles/PMC2700405/figure/F1/" class="imagepopup">Figure 1.  From: Crosstalk between tumor cells and microenvironment via Wnt <span class="highlight" style="background-color:">pathway</span> in colorectal cancer dissemination. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the canonical Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A: In the absence of Wnt ligands, destruction complex phosphylates β-catenin for ubiquitination and proteolytic degradation; B: In the presence of Wnt ligands, formation of destruction complex is not accomplished, resulting in nuclear translocation of β-catenin.</div></p></div><div class="aux"><div class="resc">Dan Huang, et al. World J Gastroenterol. 2008 March 28;14(12):1823-1827.</div><p class="links"><a class="dblinks" href="/pubmed/18350618" ref="ordinalpos=2223&amp;ncbi_uid=1878952&amp;link_uid=18350618">Citation</a><a class="dblinks" href="/pmc/articles/PMC2700405/" ref="ordinalpos=2223&amp;ncbi_uid=1878952&amp;link_uid=PMC2700405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2224.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3859655/figure/pone-0083067-g006/" ref="ordinalpos=2224&amp;ncbi_uid=5596329&amp;link_uid=PMC3859655" href="/pmc/articles/PMC3859655/"><img src="/pmc/articles/instance/3859655/bin/pone.0083067.g006.gif" src-large="/pmc/articles/instance/3859655/bin/pone.0083067.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3859655/" ref="ordinalpos=2224&amp;ncbi_uid=5596329&amp;link_uid=PMC3859655" image-link="/pmc/articles/PMC3859655/figure/pone-0083067-g006/" class="imagepopup">Figure 6. Schematic diagram of a negative regulatory loop of BMP-miR-17 family-BMPRII..  From: Role of miR-17 Family in the Negative Feedback Loop of Bone Morphogenetic Protein <span class="highlight" style="background-color:">Signaling</span> in Neuron. </a></p><div class="supp"><p class="details"><div>BMPs activate Smads via BMPRI-II complex, leading to the increase of miR-17 family, which represses BMPRII expression to prevent aberrant activation of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Qi Sun, et al. PLoS One. 2013;8(12):e83067.</div><p class="links"><a class="dblinks" href="/pubmed/24349434" ref="ordinalpos=2224&amp;ncbi_uid=5596329&amp;link_uid=24349434">Citation</a><a class="dblinks" href="/pmc/articles/PMC3859655/" ref="ordinalpos=2224&amp;ncbi_uid=5596329&amp;link_uid=PMC3859655">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2225.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC85950/figure/F8/" ref="ordinalpos=2225&amp;ncbi_uid=20376&amp;link_uid=PMC85950" href="/pmc/articles/PMC85950/"><img src="/pmc/articles/instance/85950/bin/mb1402005008.gif" src-large="/pmc/articles/instance/85950/bin/mb1402005008.jpg" alt="FIG. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC85950/" ref="ordinalpos=2225&amp;ncbi_uid=20376&amp;link_uid=PMC85950" image-link="/pmc/articles/PMC85950/figure/F8/" class="imagepopup">FIG. 8.  From: Transcriptional Regulation of the TATA-Binding Protein by Ras Cellular <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic model for Ras <span class="highlight" style="background-color:">pathway</span> regulation of hTBP promoter activity in primary rat hepatocytes. The model is based on the data in Fig. 4 to 7. Raf-, and RalGDS-MEK-dependent pathways and at least one MEK-independent <span class="highlight" style="background-color:">pathway</span> activate hTBP promoter activity. Three distinct sites within the TBP promoter further define these distinct Ras <span class="highlight" style="background-color:">signaling</span> events.</div></p></div><div class="aux"><div class="resc">Sandra A. S. Johnson, et al. Mol Cell Biol. 2000 July;20(14):5000-5009.</div><p class="links"><a class="dblinks" href="/pubmed/10866657" ref="ordinalpos=2225&amp;ncbi_uid=20376&amp;link_uid=10866657">Citation</a><a class="dblinks" href="/pmc/articles/PMC85950/" ref="ordinalpos=2225&amp;ncbi_uid=20376&amp;link_uid=PMC85950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2226.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2919000/figure/f8/" ref="ordinalpos=2226&amp;ncbi_uid=2798880&amp;link_uid=PMC2919000" href="/pmc/articles/PMC2919000/"><img src="/pmc/articles/instance/2919000/bin/zjv999093535008a.gif" src-large="/pmc/articles/instance/2919000/bin/zjv999093535008a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2919000/figure/f8/" ref="ordinalpos=2226&amp;ncbi_uid=2798880&amp;link_uid=PMC2919000" href="/pmc/articles/PMC2919000/"><img src="/pmc/articles/instance/2919000/bin/zjv999093535008b.gif" src-large="/pmc/articles/instance/2919000/bin/zjv999093535008b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919000/" ref="ordinalpos=2226&amp;ncbi_uid=2798880&amp;link_uid=PMC2919000" image-link="/pmc/articles/PMC2919000/figure/f8/" class="imagepopup">FIG. 8. From: Molecular Characterization of In Vivo Adjuvant Activity in Ferrets Vaccinated against Influenza Virus  . </a></p><div class="supp"><p class="details"><div>CpG ODN- and PEG-IFN-adjuvanted vaccine-regulated IFN <span class="highlight" style="background-color:">signaling</span> pathways. (A) IFN-responsive gene expression was determined by microarray analysis from the RNA of vaccinated ferrets at day 1 postvaccination (red, upregulation; blue, downregulation) and plotted by heat map. (B) Schematic created from the gene expression profiling data obtained at day 1 postimmunization by IPA. (Top) IPA canonical <span class="highlight" style="background-color:">pathway</span> of IFN <span class="highlight" style="background-color:">signaling</span> using Q-PCR data from the 2007 vaccine study and microarray data from the 2008 vaccine study from ferrets with CpG-mediated immunization. (Bottom) IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by PEG-IFN-adjuvanted vaccination by using the microarray data of the 2008 vaccine study (red, upregulation). (C) Gene expression heat map of molecules involved in Ras <span class="highlight" style="background-color:">signaling</span> at day 1 postvaccination (top) (red, upregulation) and IPA modeling of JAK/STAT <span class="highlight" style="background-color:">pathway</span>-mediated Ras <span class="highlight" style="background-color:">signaling</span> using the microarray data from the whole-blood RNA of ferrets given PEG-IFN-mediated immunization at day 1 postvaccination (bottom) (red, upregulation).</div></p></div><div class="aux"><div class="resc">Yuan Fang, et al. J Virol. 2010 September;84(17):8369-8388.</div><p class="links"><a class="dblinks" href="/pubmed/20534862" ref="ordinalpos=2226&amp;ncbi_uid=2798880&amp;link_uid=20534862">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919000/" ref="ordinalpos=2226&amp;ncbi_uid=2798880&amp;link_uid=PMC2919000">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2227.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3091413/figure/f7-cin-2011-093/" ref="ordinalpos=2227&amp;ncbi_uid=2952272&amp;link_uid=PMC3091413" href="/pmc/articles/PMC3091413/"><img src="/pmc/articles/instance/3091413/bin/cin-2011-093f7.gif" src-large="/pmc/articles/instance/3091413/bin/cin-2011-093f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3091413/" ref="ordinalpos=2227&amp;ncbi_uid=2952272&amp;link_uid=PMC3091413" image-link="/pmc/articles/PMC3091413/figure/f7-cin-2011-093/" class="imagepopup">Figure 7.  From: Associating GWAS Information with the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Using Transcription Profiling. </a></p><div class="supp"><p class="details"><div>Results of <span class="highlight" style="background-color:">pathway</span> prediction and network modeling showing interactions between genes containing SNPs associated with risk for breast cancer and genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> and other biological pathways. The results are based on genes containing SNPs with the largest effect size (<i>P</i> &lt; 10<sup>−5</sup>). Nodes represent genes and vertices represent interactions. The color code denotes the biological process in which the genes are involved as defined in Figure 3. The color codes in the vertices indicate the experimental techniques or a combination thereof used to confirm the relationship between the genes as determined by the experiments reported in the literature. Candidate genes containing SNPs associated with risk for breast cancer are shown in red, genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown in blue and novel genes are shown in black.</div></p></div><div class="aux"><div class="resc">Chindo Hicks, et al. Cancer Inform. 2011;10:93-108.</div><p class="links"><a class="dblinks" href="/pubmed/21584266" ref="ordinalpos=2227&amp;ncbi_uid=2952272&amp;link_uid=21584266">Citation</a><a class="dblinks" href="/pmc/articles/PMC3091413/" ref="ordinalpos=2227&amp;ncbi_uid=2952272&amp;link_uid=PMC3091413">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2228.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2685763/figure/F7/" ref="ordinalpos=2228&amp;ncbi_uid=1827505&amp;link_uid=PMC2685763" href="/pmc/articles/PMC2685763/"><img src="/pmc/articles/instance/2685763/bin/nihms66152f7.gif" src-large="/pmc/articles/instance/2685763/bin/nihms66152f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2685763/" ref="ordinalpos=2228&amp;ncbi_uid=1827505&amp;link_uid=PMC2685763" image-link="/pmc/articles/PMC2685763/figure/F7/" class="imagepopup">Figure 7. A model for the function of miR-126 in angiogenesis.  From: An Endothelial-specific microRNA Governs Vascular Integrity and Angiogenesis. </a></p><div class="supp"><p class="details"><div>Binding of VEGF and FGF to their receptors on ECs leads to activation of the MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which culminates in the nucleus to stimulate the transcription of genes involved in angiogenesis. miR-126 represses the expression of Spred-1, a negative regulator of Ras/MAP kinase <span class="highlight" style="background-color:">signaling</span>. Thus, loss of miR-126 function diminishes MAP kinase <span class="highlight" style="background-color:">signaling</span> in response to VEGF and FGF, whereas gain of miR-126 function enhances angiogenic <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Shusheng Wang, et al. Dev Cell. ;15(2):261-271.</div><p class="links"><a class="dblinks" href="/pubmed/18694565" ref="ordinalpos=2228&amp;ncbi_uid=1827505&amp;link_uid=18694565">Citation</a><a class="dblinks" href="/pmc/articles/PMC2685763/" ref="ordinalpos=2228&amp;ncbi_uid=1827505&amp;link_uid=PMC2685763">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2229.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2798676/figure/F10/" ref="ordinalpos=2229&amp;ncbi_uid=2132290&amp;link_uid=PMC2798676" href="/pmc/articles/PMC2798676/"><img src="/pmc/articles/instance/2798676/bin/JCI38012.f10.gif" src-large="/pmc/articles/instance/2798676/bin/JCI38012.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2798676/" ref="ordinalpos=2229&amp;ncbi_uid=2132290&amp;link_uid=PMC2798676" image-link="/pmc/articles/PMC2798676/figure/F10/" class="imagepopup">Figure 10. A hypothetical schematic of the contribution of SCARA5 to HCC via activation of the FAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Genetic and epigenetic silencing of SCARA5 may contribute to human hepatocellular carcinoma by activating FAK <span class="highlight" style="background-color:">signaling</span>             . </a></p><div class="supp"><p class="details"><div>The physiological function of SCARA5 is to inhibit the activity of the FAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by physically associating with FAK. In this way, downregulation of SCARA5 due to epigenetic and genetic events in HCC may contribute to tumorigenesis and progression, possibly by activating the FAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via initiation of the FAK-Src-Cas complex tyrosine phosphorylation cascade, along with increased MMP-9 activity. Based on the published data (52, 53, 67–71), the activating FAK-Src-Cas complex leads to tumor growth and metastasis, by promoting cell motility, invasion, cell cycle progression, survival, angiogenesis, and epithelial-to-mesenchymal transition, possibly through the secondary <span class="highlight" style="background-color:">signaling</span> pathways (indicated in bold). Cdc42, cell division cycle 42; CRK, v-crk sarcoma virus CT10 oncogene homolog (avian); DOCK180, dedicator of cytokinesis 1 (also known as DOCK1); Grb2, growth factor receptor-bound protein 2; PKL, paxillin kinase linker.    </div></p></div><div class="aux"><div class="resc">Jian Huang, et al. J Clin Invest. 2010 January 4;120(1):223-241.</div><p class="links"><a class="dblinks" href="/pubmed/20038795" ref="ordinalpos=2229&amp;ncbi_uid=2132290&amp;link_uid=20038795">Citation</a><a class="dblinks" href="/pmc/articles/PMC2798676/" ref="ordinalpos=2229&amp;ncbi_uid=2132290&amp;link_uid=PMC2798676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2230.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4007631/figure/F3/" ref="ordinalpos=2230&amp;ncbi_uid=6097616&amp;link_uid=PMC4007631" href="/pmc/articles/PMC4007631/"><img src="/pmc/articles/instance/4007631/bin/1755-8794-7-13-3.gif" src-large="/pmc/articles/instance/4007631/bin/1755-8794-7-13-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4007631/" ref="ordinalpos=2230&amp;ncbi_uid=6097616&amp;link_uid=PMC4007631" image-link="/pmc/articles/PMC4007631/figure/F3/" class="imagepopup">Figure 3.  From: Childhood abuse is associated with methylation of multiple loci in adult DNA. </a></p><div class="supp"><p class="details"><div><b>Differential methylation in the </b><b><i>WNT </i></b><b><span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The KEGG (http://www.genome.jp/kegg/mapper.html) depiction of the <i>WNT</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is shown with hypermethylated gene promoters (more methylated in the group with childhood abuse) colored red and the hypomethylated gene promoters colored blue.</div></p></div><div class="aux"><div class="resc">Matthew Suderman, et al. BMC Med Genomics. 2014;7:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/24618023" ref="ordinalpos=2230&amp;ncbi_uid=6097616&amp;link_uid=24618023">Citation</a><a class="dblinks" href="/pmc/articles/PMC4007631/" ref="ordinalpos=2230&amp;ncbi_uid=6097616&amp;link_uid=PMC4007631">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2231.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2398777/figure/pone-0002407-g005/" ref="ordinalpos=2231&amp;ncbi_uid=1433090&amp;link_uid=PMC2398777" href="/pmc/articles/PMC2398777/"><img src="/pmc/articles/instance/2398777/bin/pone.0002407.g005.gif" src-large="/pmc/articles/instance/2398777/bin/pone.0002407.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2398777/" ref="ordinalpos=2231&amp;ncbi_uid=1433090&amp;link_uid=PMC2398777" image-link="/pmc/articles/PMC2398777/figure/pone-0002407-g005/" class="imagepopup">Figure 5. Grem1-accelerated CL6 cardiomyogenesis through regulation of BMP- and Wnt/β-catenin-<span class="highlight" style="background-color:">signaling</span> pathways..  From: Gremlin Enhances the Determined Path to Cardiomyogenesis. </a></p><div class="supp"><p class="details"><div>CL6 embryonic cells start to differentiate into mesodermal cells through Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at the early stage (days 1–3), and mesodermal CL6 cells differentiate into mature cardiomyocytes by BMP <span class="highlight" style="background-color:">pathway</span> at the late stage (days 4–14). Grem1 accelerates DMSO-induced cardiomyogenesis through inhibition of the BMP-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Daisuke Kami, et al. PLoS ONE. 2008;3(6):e2407.</div><p class="links"><a class="dblinks" href="/pubmed/18545679" ref="ordinalpos=2231&amp;ncbi_uid=1433090&amp;link_uid=18545679">Citation</a><a class="dblinks" href="/pmc/articles/PMC2398777/" ref="ordinalpos=2231&amp;ncbi_uid=1433090&amp;link_uid=PMC2398777">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2232.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3541267/figure/F3/" ref="ordinalpos=2232&amp;ncbi_uid=4525810&amp;link_uid=PMC3541267" href="/pmc/articles/PMC3541267/"><img src="/pmc/articles/instance/3541267/bin/1755-1536-5-20-3.gif" src-large="/pmc/articles/instance/3541267/bin/1755-1536-5-20-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3541267/" ref="ordinalpos=2232&amp;ncbi_uid=4525810&amp;link_uid=PMC3541267" image-link="/pmc/articles/PMC3541267/figure/F3/" class="imagepopup">Figure 3.  From: Role of stem/progenitor cells in reparative disorders. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> pathways driving mesenchymal stem cells to differentiate into lineages found in reparative disorders.</b> While Wnt10b represses adipogenesis through the activity of B-catenin/TCF/Lef transcriptional complexes, activation of the non-canonical Wnt <span class="highlight" style="background-color:">pathway</span> by Wnt5b leads to repression of TCF/β-catenin transcriptional activity and yields the opposite results. Non-canonical TGFβ <span class="highlight" style="background-color:">signaling</span> participates in bone formation through activation of the osteogenesis regulator Runx2. On the other hand, canonical TGFβ <span class="highlight" style="background-color:">signaling</span> plays a central role in the regulation of the fibrogenic gene program. The BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> drives osteogenesis through SMADs 1, 5, 8 and shares the SMAD4 component with the TGFβ <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Thavaneetharajah Pretheeban, et al. Fibrogenesis Tissue Repair. 2012;5:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/23270300" ref="ordinalpos=2232&amp;ncbi_uid=4525810&amp;link_uid=23270300">Citation</a><a class="dblinks" href="/pmc/articles/PMC3541267/" ref="ordinalpos=2232&amp;ncbi_uid=4525810&amp;link_uid=PMC3541267">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2233.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3670265/figure/F1/" ref="ordinalpos=2233&amp;ncbi_uid=5319400&amp;link_uid=PMC3670265" href="/pmc/articles/PMC3670265/"><img src="/pmc/articles/instance/3670265/bin/jkst-2-e22925-g1.gif" src-large="/pmc/articles/instance/3670265/bin/jkst-2-e22925-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3670265/" ref="ordinalpos=2233&amp;ncbi_uid=5319400&amp;link_uid=PMC3670265" image-link="/pmc/articles/PMC3670265/figure/F1/" class="imagepopup">Figure. From: The role of JAK-STAT <span class="highlight" style="background-color:">signaling</span> within the CNS. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> JAK-STAT canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in CNS. Hormones, growth factors and cytokines can induce JAK phosphorylation and activation. Activated JAKs phosphorylate STATs which in turn homo or heterodimerize. STAT dimers are then translocated to the nucleus where they bind to DNA.</div></p></div><div class="aux"><div class="resc">Celine S. Nicolas, et al. JAKSTAT. 2013 January 1;2(1):e22925.</div><p class="links"><a class="dblinks" href="/pubmed/24058789" ref="ordinalpos=2233&amp;ncbi_uid=5319400&amp;link_uid=24058789">Citation</a><a class="dblinks" href="/pmc/articles/PMC3670265/" ref="ordinalpos=2233&amp;ncbi_uid=5319400&amp;link_uid=PMC3670265">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2234.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2880953/figure/F1/" ref="ordinalpos=2234&amp;ncbi_uid=2273558&amp;link_uid=PMC2880953" href="/pmc/articles/PMC2880953/"><img src="/pmc/articles/instance/2880953/bin/nihms195329f1.gif" src-large="/pmc/articles/instance/2880953/bin/nihms195329f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2880953/" ref="ordinalpos=2234&amp;ncbi_uid=2273558&amp;link_uid=PMC2880953" image-link="/pmc/articles/PMC2880953/figure/F1/" class="imagepopup">Fig. 1.  From: How degrading: Cyp26s in hindbrain development. </a></p><div class="supp"><p class="details"><div>Diagram of the RA <span class="highlight" style="background-color:">pathway</span>. An RA-producing cell (left) shows the biosynthetic <span class="highlight" style="background-color:">pathway</span> for RA and a responding cell (right) shows <span class="highlight" style="background-color:">signaling</span> and catabolism, although RA also likely acts in the cells that produce it. Retinyl esters and beta-carotene are important stores of RA precursor. Green dashed arrows point to components of the <span class="highlight" style="background-color:">pathway</span> induced by RA <span class="highlight" style="background-color:">signaling</span>. The red line denotes repression of aldh1as by RA. Once made, RA (yellow circles) enters cells and binds Crabps (blue ovals), which transport it either to RARs (blue and green ovals) in the nucleus (orange) for <span class="highlight" style="background-color:">signaling</span> or to Cyp26s (red hexagon) for degradation. ROH, retinol; RAL, retinal; adhs, alcohol dehydrogenases; aldh1as, aldehyde dehyrogenase 1as.</div></p></div><div class="aux"><div class="resc">Richard J. White, et al. Dev Dyn. ;237(10):2775-2790.</div><p class="links"><a class="dblinks" href="/pubmed/18816852" ref="ordinalpos=2234&amp;ncbi_uid=2273558&amp;link_uid=18816852">Citation</a><a class="dblinks" href="/pmc/articles/PMC2880953/" ref="ordinalpos=2234&amp;ncbi_uid=2273558&amp;link_uid=PMC2880953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2235.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2861642/figure/F2/" ref="ordinalpos=2235&amp;ncbi_uid=2307984&amp;link_uid=PMC2861642" href="/pmc/articles/PMC2861642/"><img src="/pmc/articles/instance/2861642/bin/1479-5876-8-30-2.gif" src-large="/pmc/articles/instance/2861642/bin/1479-5876-8-30-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2861642/" ref="ordinalpos=2235&amp;ncbi_uid=2307984&amp;link_uid=PMC2861642" image-link="/pmc/articles/PMC2861642/figure/F2/" class="imagepopup">Figure 2.  From: Epstein-Barr virus encoded latent membrane protein 1 regulates mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes which predict poor prognosis of nasopharyngeal carcinoma. </a></p><div class="supp"><p class="details"><div><b>Differentially expressed genes identified by microarray validated by Q-RT-PCR</b>. Five differentially expressed mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes identified by microarray were validated by Q-RT-PCR. The numerical value above each bar is the mean alteration of the gene by microarray analysis or Q-RT-PCR. AKT and VEGF were both up-regulated, and RPS6KB1, EIF4E and AMPK were all down-regulated in the HONE1-LMP1 cell line. Q-RT-PCR analysis of these genes was normalized to GAPDH, and repeated three times independently. The mean and standard deviation are shown.</div></p></div><div class="aux"><div class="resc">Jing Chen, et al. J Transl Med. 2010;8:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/20338061" ref="ordinalpos=2235&amp;ncbi_uid=2307984&amp;link_uid=20338061">Citation</a><a class="dblinks" href="/pmc/articles/PMC2861642/" ref="ordinalpos=2235&amp;ncbi_uid=2307984&amp;link_uid=PMC2861642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2236.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3893574/figure/F1/" ref="ordinalpos=2236&amp;ncbi_uid=5744198&amp;link_uid=PMC3893574" href="/pmc/articles/PMC3893574/"><img src="/pmc/articles/instance/3893574/bin/fphys-04-00408-g0001.gif" src-large="/pmc/articles/instance/3893574/bin/fphys-04-00408-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3893574/" ref="ordinalpos=2236&amp;ncbi_uid=5744198&amp;link_uid=PMC3893574" image-link="/pmc/articles/PMC3893574/figure/F1/" class="imagepopup">Figure 1.  From: Role of microRNAs in skeletal muscle hypertrophy. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of miRNAs involved in the regulation of the IGF-1/Akt <span class="highlight" style="background-color:">pathway</span> and myostatin <span class="highlight" style="background-color:">signaling</span></b>. IGF-1 increases the activity of Akt protein. This activation of Akt is attenuated by PTEN. The activated state of Akt stimulates protein synthesis through mTOR. Akt also inhibits protein degradation by suppressing the activity of GSK3β and FoxO1, which induces protein degradation by activating the expression of MuRF1 and Atrogin1. Therefore, the IGF-1/Akt <span class="highlight" style="background-color:">pathway</span> induces skeletal muscle hypertrophy. Recent studies have shown that miR-1, miR-133, miR-206, and miR-128a negatively regulate the IGF-1/Akt <span class="highlight" style="background-color:">pathway</span> by targeting positive regulators of the IGF-1/Akt <span class="highlight" style="background-color:">pathway</span> (IGF-1, IGF-1R, IRS1, HSP70, or p70S6K), while miR-29, miR-486, and miR-23a positively regulate this <span class="highlight" style="background-color:">pathway</span> by targeting negative regulators (PTEN, FoxO1, MuRF1, or Atrogin1). In contrast to the IGF-1/Akt <span class="highlight" style="background-color:">pathway</span>, myostatin <span class="highlight" style="background-color:">signaling</span> is a negative regulator of skeletal muscle mass. Myostatin activates Smad3 protein and may inhibit protein synthesis by repressing the transcription of miR-486 and miR-29, which target PTEN and FoxO1 proteins. Conversely, myostatin expression is regulated by miR-27a/b, miR-208a/b, and miR-499. Red lines indicate the inhibitory function of miRNAs identified in skeletal muscle, while red-dashed lines represent the inhibitory function of miRNAs identified in cardiac muscle and cultured cells.</div></p></div><div class="aux"><div class="resc">Keisuke Hitachi, et al. Front Physiol. 2013;4:408.</div><p class="links"><a class="dblinks" href="/pubmed/24474938" ref="ordinalpos=2236&amp;ncbi_uid=5744198&amp;link_uid=24474938">Citation</a><a class="dblinks" href="/pmc/articles/PMC3893574/" ref="ordinalpos=2236&amp;ncbi_uid=5744198&amp;link_uid=PMC3893574">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2237.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3438932/figure/F7/" ref="ordinalpos=2237&amp;ncbi_uid=5275299&amp;link_uid=PMC3438932" href="/pmc/articles/PMC3438932/"><img src="/pmc/articles/instance/3438932/bin/zbc0391223260007.gif" src-large="/pmc/articles/instance/3438932/bin/zbc0391223260007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3438932/" ref="ordinalpos=2237&amp;ncbi_uid=5275299&amp;link_uid=PMC3438932" image-link="/pmc/articles/PMC3438932/figure/F7/" class="imagepopup">FIGURE 7. From: Toll/Interleukin-1 Receptor Domain Dimers as the Platform for Activation and Enhanced Inhibition of Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model of the role of dimeric TIR platform for the inhibition and activation of MyD88 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>Left</i>, monomeric and dimeric MyD88 TIR domains inhibit TLR <span class="highlight" style="background-color:">signaling</span> by binding to cytoplasmic TIR dimer of activated TLRs, preventing binding of MyD88. Tethered TIR dimer induce MyD88 oligomerization, which triggers assembly of the Myddosome complex. <i>Right</i>, model of the assembly of Myddosome induced by TLR dimerization: sequential addition of MyD88 through interactions with TIR of TLRs and TIRs of MyD88 assembles the DD complex and triggers phosphorylation of IRAK kinases.</div></p></div><div class="aux"><div class="resc">Ota Fekonja, et al. J Biol Chem. 2012 September 7;287(37):30993-31002.</div><p class="links"><a class="dblinks" href="/pubmed/22829600" ref="ordinalpos=2237&amp;ncbi_uid=5275299&amp;link_uid=22829600">Citation</a><a class="dblinks" href="/pmc/articles/PMC3438932/" ref="ordinalpos=2237&amp;ncbi_uid=5275299&amp;link_uid=PMC3438932">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2238.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3216625/figure/f1/" ref="ordinalpos=2238&amp;ncbi_uid=3724767&amp;link_uid=PMC3216625" href="/pmc/articles/PMC3216625/"><img src="/pmc/articles/instance/3216625/bin/srep00144-f1.gif" src-large="/pmc/articles/instance/3216625/bin/srep00144-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3216625/" ref="ordinalpos=2238&amp;ncbi_uid=3724767&amp;link_uid=PMC3216625" image-link="/pmc/articles/PMC3216625/figure/f1/" class="imagepopup">Figure 1. TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and experimental activation profiles of <span class="highlight" style="background-color:">signaling</span>                         molecules..  From: Enhancing apoptosis in TRAIL-resistant cancer cells using fundamental             response rules. </a></p><div class="supp"><p class="details"><div>(A) Schematic topology of TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. See maintext for details.                         (B) Experimental activation profiles of p38, IκB, JNK, caspase-8 and -3 in                         wildtype, RIP1 KD*, FADD KD*, caspase-8 KD*, and TRAF2 KD in arbitrary units                         (a.u.) at <i>t</i> = 0, 10, 30, 60** and 120 min after TRAIL stimulation of                         HT1080 cells. The original source was obtained from Figure                             3A of ref. 22 and was processed                         through imageJ (see Methods). *data is unavailable for caspase-8 and -3, **                         available only for caspase-8 and -3. Note: interpolated dotted lines between                         experimental data points are inserted as a guide, they might not represent                         the actual temporal dynamics.</div></p></div><div class="aux"><div class="resc">Vincent Piras, et al. Sci Rep. 2011;1:144.</div><p class="links"><a class="dblinks" href="/pubmed/22355661" ref="ordinalpos=2238&amp;ncbi_uid=3724767&amp;link_uid=22355661">Citation</a><a class="dblinks" href="/pmc/articles/PMC3216625/" ref="ordinalpos=2238&amp;ncbi_uid=3724767&amp;link_uid=PMC3216625">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2239.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3062054/figure/F4/" ref="ordinalpos=2239&amp;ncbi_uid=2840783&amp;link_uid=PMC3062054" href="/pmc/articles/PMC3062054/"><img src="/pmc/articles/instance/3062054/bin/nihms-280598-f0004.gif" src-large="/pmc/articles/instance/3062054/bin/nihms-280598-f0004.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3062054/" ref="ordinalpos=2239&amp;ncbi_uid=2840783&amp;link_uid=PMC3062054" image-link="/pmc/articles/PMC3062054/figure/F4/" class="imagepopup">FIGURE 4. Therapeutic targeting of growth factor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in solid tumors.  From: Roles for Growth Factors in Cancer Progression. </a></p><div class="supp"><p class="details"><div>A scheme showing the major therapeutic strategies aimed at blocking growth factor <span class="highlight" style="background-color:">signaling</span> in cancer. The strategies include monoclonal antibodies (mAbs) against the GFs and their receptors, soluble receptors (TRAPs): small molecule tyrosine kinase inhibitors (TKIs), blockers of heat shock proteins (HSPs), antagonists of specific <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins, as well as inhibitors of histone deacetylases (HDACs) and methyl-transferase (MTase). Also represented are cytotoxic treatments. Examples of drugs are shown in yellow boxes; therapeutic agents that were approved for clinical use are highlighted in red, and targets are shown in parentheses. See details of <span class="highlight" style="background-color:">signaling</span> pathways in FIGURE 2.</div></p></div><div class="aux"><div class="resc">Esther Witsch, et al. Physiology (Bethesda). ;25(2):85-101.</div><p class="links"><a class="dblinks" href="/pubmed/20430953" ref="ordinalpos=2239&amp;ncbi_uid=2840783&amp;link_uid=20430953">Citation</a><a class="dblinks" href="/pmc/articles/PMC3062054/" ref="ordinalpos=2239&amp;ncbi_uid=2840783&amp;link_uid=PMC3062054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2240.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3690588/figure/F1/" ref="ordinalpos=2240&amp;ncbi_uid=5017175&amp;link_uid=PMC3690588" href="/pmc/articles/PMC3690588/"><img src="/pmc/articles/instance/3690588/bin/nihms478261f1.gif" src-large="/pmc/articles/instance/3690588/bin/nihms478261f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3690588/" ref="ordinalpos=2240&amp;ncbi_uid=5017175&amp;link_uid=PMC3690588" image-link="/pmc/articles/PMC3690588/figure/F1/" class="imagepopup">Fig. 1.  From: Wnt/?-catenin and its Diverse Physiological Cell <span class="highlight" style="background-color:">Signaling</span> Pathways in Neurodegenerative and Neuropsychiatric Disorders. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span>. A simplified diagram ofWnt β-catenin dependent and independent <span class="highlight" style="background-color:">signaling</span>: <b>a</b> β-catenin-dependent <span class="highlight" style="background-color:">pathway</span>: β-catenin dependent <span class="highlight" style="background-color:">signaling</span> is initiated by binding of Wnts to frizzled seven transmembrane receptors. This binding leads to a series of events culminating in deactivation of a multi-protein β-catenin destruction complex. Subsequently, hypophosphorylated β-catenin translocates to the nucleus where it binds TCF/LEF transcription factors to regulate gene expression. <b>b</b> Wnt β-catenin independent/calcium <span class="highlight" style="background-color:">pathway</span>. Wnts binding to frizzled lead to calcium influx, which activates calmodulin-dependent protein kinase II and NFAT. NFAT binds to its cognate target gene to regulate gene expression. <b>c</b> Wnt β-catenin independent/planar cell polarity <span class="highlight" style="background-color:">pathway</span>: Wnt binding to frizzled leads to Rac/Rho activation and subsequently to activation of kinases such as JNK that modulates cytoskeletal rearrangements. This figure is adapted from (Henderson and Al-Harthi 2011) and (Mulligan and Cheyette 2012)</div></p></div><div class="aux"><div class="resc">Lena Al-Harthi. J Neuroimmune Pharmacol. ;7(4):725-730.</div><p class="links"><a class="dblinks" href="/pubmed/23114888" ref="ordinalpos=2240&amp;ncbi_uid=5017175&amp;link_uid=23114888">Citation</a><a class="dblinks" href="/pmc/articles/PMC3690588/" ref="ordinalpos=2240&amp;ncbi_uid=5017175&amp;link_uid=PMC3690588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2241.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3641099/figure/pone-0062870-g001/" ref="ordinalpos=2241&amp;ncbi_uid=4874618&amp;link_uid=PMC3641099" href="/pmc/articles/PMC3641099/"><img src="/pmc/articles/instance/3641099/bin/pone.0062870.g001.gif" src-large="/pmc/articles/instance/3641099/bin/pone.0062870.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3641099/" ref="ordinalpos=2241&amp;ncbi_uid=4874618&amp;link_uid=PMC3641099" image-link="/pmc/articles/PMC3641099/figure/pone-0062870-g001/" class="imagepopup">Figure 1. Differentially expressed genes associated with Toll-Like Receptor <span class="highlight" style="background-color:">signaling</span> pathways..  From: Gene Expression Analysis of Peripheral Blood Cells Reveals Toll-Like Receptor <span class="highlight" style="background-color:">Pathway</span> Deregulation in Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">pathway</span> illustration was generated and linked with available experimental data in the MetaCore™ suite. Twelve genes (<i>TLR1, TLR2, TLR4, TLR6, TLR8, MYD88, MD-2, Beclin 1, TAB2, TAK1, IRAKM</i> and <i>IkB)</i> that up-regulated in the CRC patients are linked to Toll-Like Receptor <span class="highlight" style="background-color:">signaling</span> pathways and can be visualized on the map as thermometer-like figures. Up-ward thermometers have a red color and indicate up-regulated levels of gene expression in the CRC patients.</div></p></div><div class="aux"><div class="resc">Ye Xu, et al. PLoS One. 2013; 2013;8(5):e62870.</div><p class="links"><a class="dblinks" href="/pubmed/23650534" ref="ordinalpos=2241&amp;ncbi_uid=4874618&amp;link_uid=23650534">Citation</a><a class="dblinks" href="/pmc/articles/PMC3641099/" ref="ordinalpos=2241&amp;ncbi_uid=4874618&amp;link_uid=PMC3641099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2242.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3017505/figure/F1/" ref="ordinalpos=2242&amp;ncbi_uid=2704954&amp;link_uid=PMC3017505" href="/pmc/articles/PMC3017505/"><img src="/pmc/articles/instance/3017505/bin/nihms259372f1.gif" src-large="/pmc/articles/instance/3017505/bin/nihms259372f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3017505/" ref="ordinalpos=2242&amp;ncbi_uid=2704954&amp;link_uid=PMC3017505" image-link="/pmc/articles/PMC3017505/figure/F1/" class="imagepopup">Figure 1. A new branch of the yeast mating pheromone response <span class="highlight" style="background-color:">pathway</span>.  From: G-Protein <span class="highlight" style="background-color:">Signaling</span>: A New Branch in an Old <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>In the ‘classic’ <span class="highlight" style="background-color:">pathway</span>, pheromone binding to a cognate seven-transmembrane receptor leads to the activation of the Gα subunit of a receptor-coupled heterotrimeric G-protein. Active Gα releases Gβγ, which then binds to both the Ste20 protein kinase (PAK) and the Ste5 scaffold protein. Ste5 tugs Ste11 (MEKK), the first domino in the MAP kinase cascade, to the membrane, where it is phosphorylated and thereby activated by Ste20<sup>PAK</sup>. The new work by Slessareva <i>et al.</i> [2] provides evidence for the existence of a new branch of the <span class="highlight" style="background-color:">pathway</span> (surrounded by dotted frame) that modulates the efficiency of <span class="highlight" style="background-color:">signaling</span> through the classic branch. In the new branch, active Gα is proposed to move to the endosome, where it binds PI 3-kinase (Vps34) and to an associated protein kinase (Vps15) and stimulates the production of PI 3-phosphate at the endosome. The figure shows the PI 3-phosphate recruiting an unknown phospholipid-binding protein (represented by ?), perhaps Bem1, which reconnects to the classic <span class="highlight" style="background-color:">pathway</span> at an unknown point upstream of MAP kinase activation.</div></p></div><div class="aux"><div class="resc">Lee Bardwell. Curr Biol. ;16(19):R853-R855.</div><p class="links"><a class="dblinks" href="/pubmed/17027484" ref="ordinalpos=2242&amp;ncbi_uid=2704954&amp;link_uid=17027484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3017505/" ref="ordinalpos=2242&amp;ncbi_uid=2704954&amp;link_uid=PMC3017505">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2243.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399217/figure/F2/" ref="ordinalpos=2243&amp;ncbi_uid=4125432&amp;link_uid=PMC3399217" href="/pmc/articles/PMC3399217/"><img src="/pmc/articles/instance/3399217/bin/ijmsv09p0370g02.gif" src-large="/pmc/articles/instance/3399217/bin/ijmsv09p0370g02.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399217/" ref="ordinalpos=2243&amp;ncbi_uid=4125432&amp;link_uid=PMC3399217" image-link="/pmc/articles/PMC3399217/figure/F2/" class="imagepopup">Figure 2.  From: 18?-Glycyrrhetinic Acid Down-Regulates Expression of Type I and III Collagen via TGF-?1/Smad <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human and Rat Hepatic Stellate Cells. </a></p><div class="supp"><p class="details"><div>Effect of 18α-GA on mRNA expression of TGF-β1/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related proteins in human and rat HSCs. The mRNA expression of Smad3, COLl1A2, and COL3A1 was significantly decreased, while that of Smad7 dramatically increased, However, the mRNA expression of Smad2, Smad4, TβRI, TβRII remained unchanged after 18α-GA treatment. The mRNA expression of target genes was normalized to that of GAPDH and a value of 1.0 was assigned to the mRNA expression of target genes in the control group. *<i>P</i>&lt;0.05, **<i>P</i>&lt;0.01 vs control.</div></p></div><div class="aux"><div class="resc">Lei Zong, et al. Int J Med Sci. 2012;9(5):370-379.</div><p class="links"><a class="dblinks" href="/pubmed/22811611" ref="ordinalpos=2243&amp;ncbi_uid=4125432&amp;link_uid=22811611">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399217/" ref="ordinalpos=2243&amp;ncbi_uid=4125432&amp;link_uid=PMC3399217">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2244.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4394589/figure/Fig6/" ref="ordinalpos=2244&amp;ncbi_uid=7413378&amp;link_uid=PMC4394589" href="/pmc/articles/PMC4394589/"><img src="/pmc/articles/instance/4394589/bin/12864_2015_1475_Fig6_HTML.gif" src-large="/pmc/articles/instance/4394589/bin/12864_2015_1475_Fig6_HTML.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4394589/" ref="ordinalpos=2244&amp;ncbi_uid=7413378&amp;link_uid=PMC4394589" image-link="/pmc/articles/PMC4394589/figure/Fig6/" class="imagepopup">Figure 6.  From: The transcriptome of a complete episode of acute otitis media. </a></p><div class="supp"><p class="details"><div> <b>Regulation of genes in IL1/TLR <span class="highlight" style="background-color:">signaling</span> pathways.</b> Diagrammatic representation of responses for genes involved in IL-1 and TLR <span class="highlight" style="background-color:">signaling</span> during NTHi-induced AOM. Gene regulation indicated as in Figure 5. Pro-inflammatory genes within the direct <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and anti-inflammatory regulatory genes are expressed with similar kinetics.</div></p></div><div class="aux"><div class="resc">Michelle Hernandez, et al. BMC Genomics. 2015;16(1):259.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2244&amp;ncbi_uid=7413378&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4394589/" ref="ordinalpos=2244&amp;ncbi_uid=7413378&amp;link_uid=PMC4394589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2245.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2813230/figure/F4/" ref="ordinalpos=2245&amp;ncbi_uid=2167750&amp;link_uid=PMC2813230" href="/pmc/articles/PMC2813230/"><img src="/pmc/articles/instance/2813230/bin/1471-213X-9-74-4.gif" src-large="/pmc/articles/instance/2813230/bin/1471-213X-9-74-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2813230/" ref="ordinalpos=2245&amp;ncbi_uid=2167750&amp;link_uid=PMC2813230" image-link="/pmc/articles/PMC2813230/figure/F4/" class="imagepopup">Figure 4.  From: The roles of EGF and Wnt <span class="highlight" style="background-color:">signaling</span> during patterning of the C. elegans B?/? Equivalence Group. </a></p><div class="supp"><p class="details"><div><b>Patterning of equivalence groups in <i>C. elegans </i>(A) Model for EGF, Wnt and TGF-β <span class="highlight" style="background-color:">signaling</span> during Bγ/δ specification</b>. The EGF and TGF-β pathways specify Bγ fate by regulating the transcription of target genes such as <i>ceh-13/hox1</i>. Wnt controls the axis of division of Bγ, possibly by orienting the mitotic spindle. POPTOP expression suggests Wnt may play a role in Bγ fate specification. (B) A comparison of the HCG, VPCs, P11/12 and Bγ/δ groups. EGF and Wnt <span class="highlight" style="background-color:">signaling</span> have different requirements relative to each other during the patterning of each equivalence group. This difference may account for the specificity of fate by both pathways induced in each group. In addition, Wnt <span class="highlight" style="background-color:">signaling</span> is required for Bγ division along the correct axis. Such a role for Wnt <span class="highlight" style="background-color:">signaling</span> has not been observed in the other equivalence groups. Another factor that may contribute to fate specification in each equivalence group is the use of a third <span class="highlight" style="background-color:">pathway</span> during patterning. TGF-β <span class="highlight" style="background-color:">signaling</span> by <i>dbl-1/dpp </i>is required to specify Bγ fate and does not appear to act during VPC and P12 specification, equivalence groups in which EGF <span class="highlight" style="background-color:">signaling</span> is the major inductive signal. Finally, downstream of the EGF and Wnt pathways, a different Hox gene is expressed in each equivalence group and required to specify fate within that group. One exception is <i>ceh-13/Hox1 </i>for which a functional role in Bγ fate specification has not been identified.</div></p></div><div class="aux"><div class="resc">Adeline Seah, et al. BMC Dev Biol. 2009;9:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/20042118" ref="ordinalpos=2245&amp;ncbi_uid=2167750&amp;link_uid=20042118">Citation</a><a class="dblinks" href="/pmc/articles/PMC2813230/" ref="ordinalpos=2245&amp;ncbi_uid=2167750&amp;link_uid=PMC2813230">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2246.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3749383/figure/F1/" ref="ordinalpos=2246&amp;ncbi_uid=5226900&amp;link_uid=PMC3749383" href="/pmc/articles/PMC3749383/"><img src="/pmc/articles/instance/3749383/bin/mb-37-161-g001.gif" src-large="/pmc/articles/instance/3749383/bin/mb-37-161-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3749383/" ref="ordinalpos=2246&amp;ncbi_uid=5226900&amp;link_uid=PMC3749383" image-link="/pmc/articles/PMC3749383/figure/F1/" class="imagepopup">Fig. 1.  From: The Stress-Activated <span class="highlight" style="background-color:">Signaling</span> (SAS) Pathways of a Human Fungal Pathogen, Cryptococcus neoformans. </a></p><div class="supp"><p class="details"><div>The <i>C. neoformans</i> HOG <span class="highlight" style="background-color:">pathway</span>. The HOG <span class="highlight" style="background-color:">pathway</span> consists of multiple <span class="highlight" style="background-color:">signaling</span> components and is involved in a variety of stress responses, including osmotic, oxidative, and heavy metal stresses, breaches of cell wall integrity, and anti-fungal drug responses. Following exposure to an environmental cue, signals are transmitted to downstream genes of the HOG <span class="highlight" style="background-color:">pathway</span> via a two-component-like phosphorelay system and a HOG MAPK module composed of Ssk2 MAPK kinase kinase, Pbs2 MAPK kinase, and Hog1 MAPK.</div></p></div><div class="aux"><div class="resc">Kwang-Woo Jung, et al. Mycobiology. 2009 September;37(3):161-170.</div><p class="links"><a class="dblinks" href="/pubmed/23983528" ref="ordinalpos=2246&amp;ncbi_uid=5226900&amp;link_uid=23983528">Citation</a><a class="dblinks" href="/pmc/articles/PMC3749383/" ref="ordinalpos=2246&amp;ncbi_uid=5226900&amp;link_uid=PMC3749383">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2247.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3832598/figure/pone-0080806-g007/" ref="ordinalpos=2247&amp;ncbi_uid=5508460&amp;link_uid=PMC3832598" href="/pmc/articles/PMC3832598/"><img src="/pmc/articles/instance/3832598/bin/pone.0080806.g007.gif" src-large="/pmc/articles/instance/3832598/bin/pone.0080806.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3832598/" ref="ordinalpos=2247&amp;ncbi_uid=5508460&amp;link_uid=PMC3832598" image-link="/pmc/articles/PMC3832598/figure/pone-0080806-g007/" class="imagepopup">Figure 7. CD43 <span class="highlight" style="background-color:">signaling</span> cooperates with oncogenic signals to promote cell transformation..  From: CD43 Promotes Cells Transformation by Preventing Merlin-Mediated Contact Inhibition of Growth. </a></p><div class="supp"><p class="details"><div>In epithelial cells, upon interaction with putative ligand(s) present on the cell surface of neighboring cells or in the extracellular matrix, CD43 activates the PI3K/AKT <span class="highlight" style="background-color:">pathway</span> that results in the inhibition of the Hippo <span class="highlight" style="background-color:">pathway</span> by a mechanism involving Merlin phosphorylation and degradation, thus favoring cell survival and proliferation. Depending of the cellular context, <span class="highlight" style="background-color:">signaling</span> from intracellular CD43 located either on membrane vesicles or the nuclear membrane may also contribute to cellular transformation. Nonetheless, this might not be sufficient to overcome the anti-proliferative and death effects resulting from the activation of the ARF-p53 <span class="highlight" style="background-color:">pathway</span> also induced by CD43 [24]. However, in transformed cells with an impaired p53 <span class="highlight" style="background-color:">pathway</span> resulting from oncogenic signals like those provided by the E6 oncoprotein from HPV16, CD43 <span class="highlight" style="background-color:">signaling</span> promotes cells proliferation and tumor formation. </div></p></div><div class="aux"><div class="resc">Nohemi Camacho-Concha, et al. PLoS One. 2013;8(11):e80806.</div><p class="links"><a class="dblinks" href="/pubmed/24260485" ref="ordinalpos=2247&amp;ncbi_uid=5508460&amp;link_uid=24260485">Citation</a><a class="dblinks" href="/pmc/articles/PMC3832598/" ref="ordinalpos=2247&amp;ncbi_uid=5508460&amp;link_uid=PMC3832598">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2248.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2677410/figure/F1/" ref="ordinalpos=2248&amp;ncbi_uid=1813261&amp;link_uid=PMC2677410" href="/pmc/articles/PMC2677410/"><img src="/pmc/articles/instance/2677410/bin/nihms102225f1.gif" src-large="/pmc/articles/instance/2677410/bin/nihms102225f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2677410/" ref="ordinalpos=2248&amp;ncbi_uid=1813261&amp;link_uid=PMC2677410" image-link="/pmc/articles/PMC2677410/figure/F1/" class="imagepopup">Figure 1. Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a fundamental ‘rheostat’ for maintenance of stem cells.  From: A novel <span class="highlight" style="background-color:">signaling</span> network as a critical rheostat for the biology and maintenance of the normal stem cell and the cancer-initiating cell. </a></p><div class="supp"><p class="details"><div>In the context of normalcy holding this signal OFF/low maintains stem cells in a quiescent state, while the ON-signal induces active cycling in stem cells and their exhaustion (left panel). In the context of cancer, an aberrant surge in <span class="highlight" style="background-color:">signaling</span> output would evoke fail-safe mechanisms (e.g. cellular senescence, apoptosis or terminal differentiation) which would extinguish the pre-malignant stem cell clone. The cells can become a neoplasia only when they escape these mechanisms (right panel).</div></p></div><div class="aux"><div class="resc">Keisuke Ito, et al. Curr Opin Genet Dev. ;19(1):51-59.</div><p class="links"><a class="dblinks" href="/pubmed/19216069" ref="ordinalpos=2248&amp;ncbi_uid=1813261&amp;link_uid=19216069">Citation</a><a class="dblinks" href="/pmc/articles/PMC2677410/" ref="ordinalpos=2248&amp;ncbi_uid=1813261&amp;link_uid=PMC2677410">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2249.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3979852/figure/pone-0094445-g008/" ref="ordinalpos=2249&amp;ncbi_uid=6016441&amp;link_uid=PMC3979852" href="/pmc/articles/PMC3979852/"><img src="/pmc/articles/instance/3979852/bin/pone.0094445.g008.gif" src-large="/pmc/articles/instance/3979852/bin/pone.0094445.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3979852/" ref="ordinalpos=2249&amp;ncbi_uid=6016441&amp;link_uid=PMC3979852" image-link="/pmc/articles/PMC3979852/figure/pone-0094445-g008/" class="imagepopup">Figure 8. Scheme of TLR4 and CCR8 <span class="highlight" style="background-color:">signaling</span> cross talk in mouse PMφ..  From: Chemokine Receptor CCR8 Is Required for Lipopolysaccharide-Triggered Cytokine Production in Mouse Peritoneal Macrophages. </a></p><div class="supp"><p class="details"><div>LPS triggers TLR4 <span class="highlight" style="background-color:">signaling</span> and induces NF-κB and MAPK activation, possibly together with CCR8 activation, which results in the production of TNF-α, IL-6, and IL-10. CCR8 is required for full activation of both <span class="highlight" style="background-color:">signaling</span> pathways. Production of IL-6 and TNF-α involves both the NF-κB and MAPK pathways. IL-10 production is largely dependent on the JNK <span class="highlight" style="background-color:">pathway</span> and is mediated by CCR8 activation. In addition to the antagonistic effect on CCR8, possible sites of action of R243 are indicated.</div></p></div><div class="aux"><div class="resc">Tomoyuki Oshio, et al. PLoS One. 2014;9(4):e94445.</div><p class="links"><a class="dblinks" href="/pubmed/24714157" ref="ordinalpos=2249&amp;ncbi_uid=6016441&amp;link_uid=24714157">Citation</a><a class="dblinks" href="/pmc/articles/PMC3979852/" ref="ordinalpos=2249&amp;ncbi_uid=6016441&amp;link_uid=PMC3979852">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2250.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4260770/figure/f4-cin-suppl.2-2014-113/" ref="ordinalpos=2250&amp;ncbi_uid=6969888&amp;link_uid=PMC4260770" href="/pmc/articles/PMC4260770/"><img src="/pmc/articles/instance/4260770/bin/cin-suppl.2-2014-113f4.gif" src-large="/pmc/articles/instance/4260770/bin/cin-suppl.2-2014-113f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4260770/" ref="ordinalpos=2250&amp;ncbi_uid=6969888&amp;link_uid=PMC4260770" image-link="/pmc/articles/PMC4260770/figure/f4-cin-suppl.2-2014-113/" class="imagepopup">Figure 4.  From: Bayesian Joint Selection of Genes and Pathways: Applications in Multiple Myeloma Genomics. </a></p><div class="supp"><p class="details"><div>The 30 genes with highest posterior probabilities of regression coefficients being greater than <i>ϕ</i> = 0.5 in the absolute value. The line patterns correspond to the pathways to which the genes belong: dotted lines for genes in the galactose metabolism <span class="highlight" style="background-color:">pathway</span>; solid lines for genes in the cell cycle <span class="highlight" style="background-color:">pathway</span>; dashed lines for genes in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; dot-dash lines for genes in both the cell cycle and Wnt <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Lin Zhang, et al. Cancer Inform. 2014;13(Suppl 2):113-123.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2250&amp;ncbi_uid=6969888&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4260770/" ref="ordinalpos=2250&amp;ncbi_uid=6969888&amp;link_uid=PMC4260770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2251.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3171509/figure/F1/" ref="ordinalpos=2251&amp;ncbi_uid=3220210&amp;link_uid=PMC3171509" href="/pmc/articles/PMC3171509/"><img src="/pmc/articles/instance/3171509/bin/nihms123300f1.gif" src-large="/pmc/articles/instance/3171509/bin/nihms123300f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3171509/" ref="ordinalpos=2251&amp;ncbi_uid=3220210&amp;link_uid=PMC3171509" image-link="/pmc/articles/PMC3171509/figure/F1/" class="imagepopup">Fig. 1.  From: An Image Driven Systems Biology Approach for Neurodegenerative Disease Studies in the TSC-mTOR <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>This figure illustrates the core TSC-mTor <span class="highlight" style="background-color:">pathway</span>, as discussed in the text. The green <span class="highlight" style="background-color:">signaling</span> path relates to disease such as the Cowden syndrome, Bannayan Riley Rucvalcaba, Proteus syndrome and Lhermitte-Duclos. The purple <span class="highlight" style="background-color:">signaling</span> cascade relates to disease such as Familial cardiac hypertrophy, the Wolf-Parkinson-White syndrome and the Peutz–Jeghers syndrome. The blue <span class="highlight" style="background-color:">signaling</span> path relates to the tuberous sclerosis complex disease, affecting multi-organs and often leading to neurologic disorders. The red part is active in different types of cancers including ovarian, gastrointestinal, breast, sporadic and gall bladder and non-small cell carcinoma in the lung. A detailed overview about disease relating to this <span class="highlight" style="background-color:">pathway</span> is given in [6].</div></p></div><div class="aux"><div class="resc">Dominik Beck, et al. IEEE NIH Life Sci Syst Appl Workshop. :36-39.</div><p class="links"><a class="dblinks" href="/pubmed/21922078" ref="ordinalpos=2251&amp;ncbi_uid=3220210&amp;link_uid=21922078">Citation</a><a class="dblinks" href="/pmc/articles/PMC3171509/" ref="ordinalpos=2251&amp;ncbi_uid=3220210&amp;link_uid=PMC3171509">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2252.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3786863/figure/F3/" ref="ordinalpos=2252&amp;ncbi_uid=5356387&amp;link_uid=PMC3786863" href="/pmc/articles/PMC3786863/"><img src="/pmc/articles/instance/3786863/bin/nihms514492f3.gif" src-large="/pmc/articles/instance/3786863/bin/nihms514492f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3786863/" ref="ordinalpos=2252&amp;ncbi_uid=5356387&amp;link_uid=PMC3786863" image-link="/pmc/articles/PMC3786863/figure/F3/" class="imagepopup">Figure 3. Genetic control of the proliferation vs. differentiation decision.  From: Chapter 4. Stem cell proliferation versus meiotic fate decision in C. elegans. </a></p><div class="supp"><p class="details"><div>Core genetic <span class="highlight" style="background-color:">pathway</span> controlling the proliferation vs. differentiation decision (top) with factors that promote the proliferative fate and/or inhibit meiotic entry shown in green, while factors that promote meiotic entry and/or inhibit the proliferative fate shown in red. In this model, the LAG-2 and APX-1 ligands, which are expressed on the distal tip cell (DTC), interact with the GLP-1 Notch receptor, which is expressed on the germ cells. This causes the intracellular portion of GLP-1 to translocate to the nucleus and interact with the LAG-1 CSL transcription factor and SEL-8 Mastermind co-activator. These are all conserved components of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The GLP-1 <span class="highlight" style="background-color:">pathway</span> inhibits the activities of the redundant GLD-1 and GLD-2 genetic pathways, whose components all appear to regulate mRNA stability and/or translatability. An additional redundant <span class="highlight" style="background-color:">pathway</span> that is inhibited by GLP-1 <span class="highlight" style="background-color:">signaling</span> has been uncovered genetically (not shown), although genes that reside in this <span class="highlight" style="background-color:">pathway</span> have not been identified. The basic model of how <span class="highlight" style="background-color:">signaling</span> from the DTC regulates the proliferation vs. differentiation decision is shown (bottom). While germ cells are in the distal-most end of the gonad arm, close to the DTC (yellow), GLP-1 Notch <span class="highlight" style="background-color:">signaling</span> levels are high (green), keeping the activities of the GLD-1 and GLD-2 pathways low (red), resulting in cells remaining in the proliferative fate. As cells move proximally, GLP-1 Notch <span class="highlight" style="background-color:">signaling</span> levels decrease, causing the activities of the GLD-1 and GLD-2 pathways to increase, resulting in cells entering into meiotic prophase.</div></p></div><div class="aux"><div class="resc">Dave Hansen, et al. Adv Exp Med Biol. 2013;757:71-99.</div><p class="links"><a class="dblinks" href="/pubmed/22872475" ref="ordinalpos=2252&amp;ncbi_uid=5356387&amp;link_uid=22872475">Citation</a><a class="dblinks" href="/pmc/articles/PMC3786863/" ref="ordinalpos=2252&amp;ncbi_uid=5356387&amp;link_uid=PMC3786863">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2253.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4008004/figure/f1/" ref="ordinalpos=2253&amp;ncbi_uid=6146550&amp;link_uid=PMC4008004" href="/pmc/articles/PMC4008004/"><img src="/pmc/articles/instance/4008004/bin/jdi-1-134-g1.gif" src-large="/pmc/articles/instance/4008004/bin/jdi-1-134-g1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4008004/" ref="ordinalpos=2253&amp;ncbi_uid=6146550&amp;link_uid=PMC4008004" image-link="/pmc/articles/PMC4008004/figure/f1/" class="imagepopup">Figure 1.  From: Is there a place for inhibition of transforming growth factor-? and the polyol <span class="highlight" style="background-color:">pathway</span> in therapy for diabetic retinopathy?. </a></p><div class="supp"><p class="details"><div> Hyperglycemia alters the expression of multiple genes of the transforming growth factor‐β (TGF‐β) <span class="highlight" style="background-color:">pathway</span> through the polyol <span class="highlight" style="background-color:">pathway</span> in retinal vessels. Upregulated genes are shown in red and downregulated genes are shown in blue. The expected effect of overexpression of genes on Smad <span class="highlight" style="background-color:">signaling</span> is shown by arrowed lines (stimulation) or blunt lines (inhibition). The expected effect of downregulation of Hoxc8 is release of inhibition, and thus activation of <span class="highlight" style="background-color:">signaling</span> (dotted blunt line). BMP, bone morphogenetic proteins. The figure is a modification from the original in Gerhardinger <i>et al.</i><sup>3</sup> </div></p></div><div class="aux"><div class="resc">Nigishi Hotta. J Diabetes Investig. 2010 August 2;1(4):134-136.</div><p class="links"><a class="dblinks" href="/pubmed/24843421" ref="ordinalpos=2253&amp;ncbi_uid=6146550&amp;link_uid=24843421">Citation</a><a class="dblinks" href="/pmc/articles/PMC4008004/" ref="ordinalpos=2253&amp;ncbi_uid=6146550&amp;link_uid=PMC4008004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2254.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g004/" ref="ordinalpos=2254&amp;ncbi_uid=6143991&amp;link_uid=PMC4018296" href="/pmc/articles/PMC4018296/"><img src="/pmc/articles/instance/4018296/bin/pone.0096776.g004.gif" src-large="/pmc/articles/instance/4018296/bin/pone.0096776.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4018296/" ref="ordinalpos=2254&amp;ncbi_uid=6143991&amp;link_uid=PMC4018296" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g004/" class="imagepopup">Figure 4. EGF-induced migration <span class="highlight" style="background-color:">signaling</span> network in three cancer cell lines..  From: Chemical Genomic-Based <span class="highlight" style="background-color:">Pathway</span> Analyses for Epidermal Growth Factor-Mediated <span class="highlight" style="background-color:">Signaling</span> in Migrating Cancer Cells. </a></p><div class="supp"><p class="details"><div>EGF-induced migration <span class="highlight" style="background-color:">signaling</span> network in (<b>a</b>) A431 cells, (<b>b</b>) EC109 cells, and (<b>c</b>) TT cells. The threshold was set at ±50% from positive control. The edge color is decided based on the sign of edges; red indicates positive <span class="highlight" style="background-color:">signaling</span>, and blue indicates negative <span class="highlight" style="background-color:">signaling</span>. The size of the node indicates the number of connecting edges. The color of the node indicates the number of output pathways: a gradient color scale from green to red, interpolated over yellow.</div></p></div><div class="aux"><div class="resc">Shigeyuki Magi, et al. PLoS One. 2014;9(5):e96776.</div><p class="links"><a class="dblinks" href="/pubmed/24820097" ref="ordinalpos=2254&amp;ncbi_uid=6143991&amp;link_uid=24820097">Citation</a><a class="dblinks" href="/pmc/articles/PMC4018296/" ref="ordinalpos=2254&amp;ncbi_uid=6143991&amp;link_uid=PMC4018296">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2255.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3181423/figure/fig8/" ref="ordinalpos=2255&amp;ncbi_uid=3292970&amp;link_uid=PMC3181423" href="/pmc/articles/PMC3181423/"><img src="/pmc/articles/instance/3181423/bin/cddis201178f8.gif" src-large="/pmc/articles/instance/3181423/bin/cddis201178f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3181423/" ref="ordinalpos=2255&amp;ncbi_uid=3292970&amp;link_uid=PMC3181423" image-link="/pmc/articles/PMC3181423/figure/fig8/" class="imagepopup">Figure 8.  From: TNFR1 and TNFR2 regulate the extrinsic apoptotic <span class="highlight" style="background-color:">pathway</span> in myeloma cells by multiple mechanisms. </a></p><div class="supp"><p class="details"><div>Model of the differential regulation of CD95L- and TRAIL-induced apoptosis by TNFR1. TNFR1-induced activation of the classical NF<i>κ</i>B <span class="highlight" style="background-color:">pathway</span> results in concomitant upregulation of FLIP<sub>L/S</sub> and CD95. Although upregulation of FLIP<sub>L/S</sub> antagonizes TRAIL-induced apoptosis, this effect is overcompensated in context of CD95L-induced <span class="highlight" style="background-color:">signaling</span> by the strong increase in CD95 expression</div></p></div><div class="aux"><div class="resc">H Rauert, et al. Cell Death Dis. 2011 August;2(8):e194.</div><p class="links"><a class="dblinks" href="/pubmed/21850048" ref="ordinalpos=2255&amp;ncbi_uid=3292970&amp;link_uid=21850048">Citation</a><a class="dblinks" href="/pmc/articles/PMC3181423/" ref="ordinalpos=2255&amp;ncbi_uid=3292970&amp;link_uid=PMC3181423">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2256.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4266706/figure/fig04/" ref="ordinalpos=2256&amp;ncbi_uid=6970591&amp;link_uid=PMC4266706" href="/pmc/articles/PMC4266706/"><img src="/pmc/articles/instance/4266706/bin/ajcr0004-0710-f4.gif" src-large="/pmc/articles/instance/4266706/bin/ajcr0004-0710-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4266706/" ref="ordinalpos=2256&amp;ncbi_uid=6970591&amp;link_uid=PMC4266706" image-link="/pmc/articles/PMC4266706/figure/fig04/" class="imagepopup">Figure 4.  From: DACT2 is frequently methylated in human gastric cancer and methylation of DACT2 activated Wnt <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>DACT2 inhibits both Wnt/β-catenin and Wnt/c-Jun <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A. Results of TCF/LEF luciferase reporter assay. TopFlash: TCF/LEF-responsive reporter; pRL-TK: internal control; WT: wild-type; Mut: mutant. B. p-β-catenin, β-catenin, C-myc and p-c-Jun expression. p-β-catenin: phosphorylated β-catenin; p-c-Jun: phosphorylated c-Jun. Con siRNA: siRNA control; 8# siRNA: DACT2 siRNA #8.</div></p></div><div class="aux"><div class="resc">Yuanzi Yu, et al. Am J Cancer Res. 2014;4(6):710-724.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2256&amp;ncbi_uid=6970591&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4266706/" ref="ordinalpos=2256&amp;ncbi_uid=6970591&amp;link_uid=PMC4266706">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2257.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4124058/figure/F1/" ref="ordinalpos=2257&amp;ncbi_uid=6473102&amp;link_uid=PMC4124058" href="/pmc/articles/PMC4124058/"><img src="/pmc/articles/instance/4124058/bin/jkst-3-e29510-g1.gif" src-large="/pmc/articles/instance/4124058/bin/jkst-3-e29510-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4124058/" ref="ordinalpos=2257&amp;ncbi_uid=6473102&amp;link_uid=PMC4124058" image-link="/pmc/articles/PMC4124058/figure/F1/" class="imagepopup">Figure. From: Effects of JAK2-STAT3 <span class="highlight" style="background-color:">signaling</span> after cerebral insults. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Overview of JAK2-STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after cerebral insults. Traumatic brain injury (TBI), stroke, and status epilepticus (SE) injury models have shown that the JAK2-STAT3 <span class="highlight" style="background-color:">pathway</span> becomes activated which is indicated either by the increased phosphorylation of JAK2 and/or STAT3. 1 <sup>,</sup> 3 <sup>,</sup> 4 <sup>,</sup> 10 <sup>-</sup> 12 <sup>,</sup> 15 <sup>-</sup> 23 <sup>,</sup> 34 <sup>-</sup> 36  When JAK2-STAT3 becomes phosphorylated it results in the increased expression of STAT3 related genes involved in cell proliferation, 1 <sup>,</sup> 4 <sup>-</sup> 6  cellular differentiation, 1 <sup>,</sup> 4 <sup>-</sup> 6  cell survival, 1 <sup>,</sup> 3 <sup>-</sup> 6 <sup>,</sup> 15 <sup>,</sup> 20  and inhibitory neurotransmission. 1 <sup>,</sup> 10 <sup>,</sup> 34 </div></p></div><div class="aux"><div class="resc">Daniel J Raible, et al. JAKSTAT. 2014;3:e29510.</div><p class="links"><a class="dblinks" href="/pubmed/25105066" ref="ordinalpos=2257&amp;ncbi_uid=6473102&amp;link_uid=25105066">Citation</a><a class="dblinks" href="/pmc/articles/PMC4124058/" ref="ordinalpos=2257&amp;ncbi_uid=6473102&amp;link_uid=PMC4124058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2258.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3426292/figure/F1/" ref="ordinalpos=2258&amp;ncbi_uid=4192032&amp;link_uid=PMC3426292" href="/pmc/articles/PMC3426292/"><img src="/pmc/articles/instance/3426292/bin/nihms-398765-f0001.gif" src-large="/pmc/articles/instance/3426292/bin/nihms-398765-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3426292/" ref="ordinalpos=2258&amp;ncbi_uid=4192032&amp;link_uid=PMC3426292" image-link="/pmc/articles/PMC3426292/figure/F1/" class="imagepopup">Figure 1. Schematic model of the Hippo <span class="highlight" style="background-color:">signaling</span> network.  From: Hippo <span class="highlight" style="background-color:">signaling</span> in Drosophila: recent advances and insights. </a></p><div class="supp"><p class="details"><div>The model depicts most identified components of the <i>Drosophila</i> Hpo <span class="highlight" style="background-color:">pathway</span>, grouped into modules as discussed in the text. These include the Hpo kinase cassette (outlined in green), the Mer-ex complex (outlined in purple), the Fat branch of the <span class="highlight" style="background-color:">pathway</span> (outlined in orange), and transcription factors (within nucleus, shaded grey). Arrows depict key regulatory connections, pointed arrows show activating effects, block arrows show inhibiting effects.</div></p></div><div class="aux"><div class="resc">Binnaz Kucuk Staley, et al. Dev Dyn. ;241(1):3-15.</div><p class="links"><a class="dblinks" href="/pubmed/22174083" ref="ordinalpos=2258&amp;ncbi_uid=4192032&amp;link_uid=22174083">Citation</a><a class="dblinks" href="/pmc/articles/PMC3426292/" ref="ordinalpos=2258&amp;ncbi_uid=4192032&amp;link_uid=PMC3426292">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2259.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3543320/figure/pone-0054125-g008/" ref="ordinalpos=2259&amp;ncbi_uid=4537246&amp;link_uid=PMC3543320" href="/pmc/articles/PMC3543320/"><img src="/pmc/articles/instance/3543320/bin/pone.0054125.g008.gif" src-large="/pmc/articles/instance/3543320/bin/pone.0054125.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3543320/" ref="ordinalpos=2259&amp;ncbi_uid=4537246&amp;link_uid=PMC3543320" image-link="/pmc/articles/PMC3543320/figure/pone-0054125-g008/" class="imagepopup">Figure 8. Schematic diagram illustrating the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in ATPγS-induced COX-2 expression and PGE2 generation in A549 cells..  From: ATP Mediates NADPH Oxidase/ROS Generation and COX-2/PGE2 Expression in A549 Cells: Role of P2 Receptor-Dependent STAT3 Activation. </a></p><div class="supp"><p class="details"><div>ATPγS activates the P2 receptor/PKC/NADPH oxidase <span class="highlight" style="background-color:">pathway</span> to enhance ROS generation, which in turn initiates the activation of Jak2 and STAT3, and ultimately induces COX-2-dependent PGE<sub>2</sub> generation in A549 cells.</div></p></div><div class="aux"><div class="resc">Shin-Ei Cheng, et al. PLoS One. 2013;8(1):e54125.</div><p class="links"><a class="dblinks" href="/pubmed/23326583" ref="ordinalpos=2259&amp;ncbi_uid=4537246&amp;link_uid=23326583">Citation</a><a class="dblinks" href="/pmc/articles/PMC3543320/" ref="ordinalpos=2259&amp;ncbi_uid=4537246&amp;link_uid=PMC3543320">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2260.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4314132/figure/f6-molcell-38-1-58/" ref="ordinalpos=2260&amp;ncbi_uid=7162895&amp;link_uid=PMC4314132" href="/pmc/articles/PMC4314132/"><img src="/pmc/articles/instance/4314132/bin/molcell-38-1-58f6.gif" src-large="/pmc/articles/instance/4314132/bin/molcell-38-1-58f6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4314132/" ref="ordinalpos=2260&amp;ncbi_uid=7162895&amp;link_uid=PMC4314132" image-link="/pmc/articles/PMC4314132/figure/f6-molcell-38-1-58/" class="imagepopup">Fig. 6. From: TLR4 Mediates Pneumolysin-Induced ATF3 Expression through the JNK/p38 <span class="highlight" style="background-color:">Pathway</span> in Streptococcus pneumoniae-Infected RAW 264.7 Cells. </a></p><div class="supp"><p class="details"><div>A model for ATF3 action during <i>S. pneumoniae</i> infection. During pneumococcal infection, PLY seems to interact with TLR4 and activates ATF3 via the TLR4/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which stimulates formation of an ATF3 complex with c-Jun, and this complex binds to promoters of cytokines (TNF-α, IL-1β, IFN-γ), resulting in enhanced cytokine production.</div></p></div><div class="aux"><div class="resc">Cuong Thach Nguyen, et al. Mol Cells. 2015 January 31;38(1):58-64.</div><p class="links"><a class="dblinks" href="/pubmed/25518930" ref="ordinalpos=2260&amp;ncbi_uid=7162895&amp;link_uid=25518930">Citation</a><a class="dblinks" href="/pmc/articles/PMC4314132/" ref="ordinalpos=2260&amp;ncbi_uid=7162895&amp;link_uid=PMC4314132">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2261.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4037312/figure/f3-ad-5-3-203/" ref="ordinalpos=2261&amp;ncbi_uid=6197916&amp;link_uid=PMC4037312" href="/pmc/articles/PMC4037312/"><img src="/pmc/articles/instance/4037312/bin/AD-5-3-203-Figure-3.gif" src-large="/pmc/articles/instance/4037312/bin/AD-5-3-203-Figure-3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4037312/" ref="ordinalpos=2261&amp;ncbi_uid=6197916&amp;link_uid=PMC4037312" image-link="/pmc/articles/PMC4037312/figure/f3-ad-5-3-203/" class="imagepopup">Figure 3. From: Link between PI3K/AKT/PTEN <span class="highlight" style="background-color:">Pathway</span> and NOX Proteinin Diseases. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of PI3K/AKT/PTEN <span class="highlight" style="background-color:">signaling</span>, which modulate NOXs-derived ROS.</b> NOXs sense hypoxia and can help to mediate activation of angiogenic <span class="highlight" style="background-color:">pathway</span>, while maintaining unregulated cell growth and apoptotic cell death. Examples of molecules known to act on the regulatory pathways are shown. Note that some critical pathways have been omitted for clarity.</div></p></div><div class="aux"><div class="resc">Atsuko Nakanishi, et al. Aging Dis. 2014 June;5(3):203-211.</div><p class="links"><a class="dblinks" href="/pubmed/24900943" ref="ordinalpos=2261&amp;ncbi_uid=6197916&amp;link_uid=24900943">Citation</a><a class="dblinks" href="/pmc/articles/PMC4037312/" ref="ordinalpos=2261&amp;ncbi_uid=6197916&amp;link_uid=PMC4037312">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2262.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4201220/figure/f2-bt-22-371/" ref="ordinalpos=2262&amp;ncbi_uid=6860230&amp;link_uid=PMC4201220" href="/pmc/articles/PMC4201220/"><img src="/pmc/articles/instance/4201220/bin/bt-22-371f2.gif" src-large="/pmc/articles/instance/4201220/bin/bt-22-371f2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4201220/" ref="ordinalpos=2262&amp;ncbi_uid=6860230&amp;link_uid=PMC4201220" image-link="/pmc/articles/PMC4201220/figure/f2-bt-22-371/" class="imagepopup">Fig. 2. From: Caenorhabditis elegans: A Model System for Anti-Cancer Drug Discovery and Therapeutic Target Identification. </a></p><div class="supp"><p class="details"><div>Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> pathways and a strategy for the phenotype-based drug identification using <i>C. elegans</i> mutants. (A) Without <span class="highlight" style="background-color:">signaling</span> (absence of ligand), negative regulators (CKIα, GSK3β, Axin, and APC) phosphorylate β-catenin, and then ubiquitinated and degraded by the proteasome. (B) With <span class="highlight" style="background-color:">signaling</span> (presence of ligand), stabilized β-catenin in cytoplasm is translocated into nucleus and activate the expression of target genes. (C) Genetic <span class="highlight" style="background-color:">pathway</span> for control DTC fate specification (see text for explanation). (D) Wild-type hermaphrodite has two DTCs. lag-2::GFP reporter is strongly expressed in DTCs (see green). (E) Loss of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> affects DTC fate specification. (F) Activation of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> contributes to extra DTC formation. (G) Strategy for the phenotype-based identification of drugs that either inhibit or activate Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> using <i>C. elegans</i> mutants.</div></p></div><div class="aux"><div class="resc">Robert A. Kobet, et al. Biomol Ther (Seoul). 2014 September;22(5):371-383.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2262&amp;ncbi_uid=6860230&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4201220/" ref="ordinalpos=2262&amp;ncbi_uid=6860230&amp;link_uid=PMC4201220">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2263.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285457/figure/F6/" ref="ordinalpos=2263&amp;ncbi_uid=3823280&amp;link_uid=PMC3285457" href="/pmc/articles/PMC3285457/"><img src="/pmc/articles/instance/3285457/bin/nihms329107f6.gif" src-large="/pmc/articles/instance/3285457/bin/nihms329107f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285457/" ref="ordinalpos=2263&amp;ncbi_uid=3823280&amp;link_uid=PMC3285457" image-link="/pmc/articles/PMC3285457/figure/F6/" class="imagepopup">Figure 6. Signal transduction diagram depicting how Hic-5 affects TGF-β <span class="highlight" style="background-color:">signaling</span>.  From: Smad7 is Inactivated through a Direct Physical Interaction with the LIM Protein Hic-5/ARA55. </a></p><div class="supp"><p class="details"><div>Hic-5, which is induced by TGF-β through a mechanism that is yet to be resolved, binds to and inactivates both Smad3 and Smad7. Inactivation of Smad7 through its degradation relieves the inhibitory action of Smad7 on TGF-β receptors, which leads to phosphorylation and thus activation of Smads 2 and 3. Depletion of Smad7 frees up Hic-5 and is expected to enable further binding to and suppression of Smad3. We thus propose that the overall effect of Hic-5 on TGF-β <span class="highlight" style="background-color:">signaling</span> is inhibition of Smad3-dependent <span class="highlight" style="background-color:">pathway</span> and enhancement of Smad2-dependent or Smad-independent <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hui Wang, et al. Oncogene. ;27(54):6791-6805.</div><p class="links"><a class="dblinks" href="/pubmed/18762808" ref="ordinalpos=2263&amp;ncbi_uid=3823280&amp;link_uid=18762808">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285457/" ref="ordinalpos=2263&amp;ncbi_uid=3823280&amp;link_uid=PMC3285457">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2264.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3739055/figure/F4/" ref="ordinalpos=2264&amp;ncbi_uid=5185923&amp;link_uid=PMC3739055" href="/pmc/articles/PMC3739055/"><img src="/pmc/articles/instance/3739055/bin/nihms171049f4.gif" src-large="/pmc/articles/instance/3739055/bin/nihms171049f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3739055/" ref="ordinalpos=2264&amp;ncbi_uid=5185923&amp;link_uid=PMC3739055" image-link="/pmc/articles/PMC3739055/figure/F4/" class="imagepopup">Figure 4. <span class="highlight" style="background-color:">Signaling</span> pathways in Lkb1 mutant SCC.  From: LKB1 deficiency sensitizes mice to carcinogen-induced tumorigenesis. </a></p><div class="supp"><p class="details"><div>A) RAF pulldown assay showing levels of activated Ras-GTP in normal skin (lane 1), SCC from from Lkb1+/- mice (lanes 2 and 3), and papillomas (lanes 4 and 5). B) Western blot analysis of ERK and PI3K <span class="highlight" style="background-color:">pathway</span> components in papillomas (lanes 5 and 6) and Lkb1-mutant SCC (lanes 1-4). C) Western blot analysis of the AMPK/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in papillomas (lanes 5 and 6) and SCC (lanes 1-4).</div></p></div><div class="aux"><div class="resc">Sushma Gurumurthy, et al. Cancer Res. ;68(1):55-63.</div><p class="links"><a class="dblinks" href="/pubmed/18172296" ref="ordinalpos=2264&amp;ncbi_uid=5185923&amp;link_uid=18172296">Citation</a><a class="dblinks" href="/pmc/articles/PMC3739055/" ref="ordinalpos=2264&amp;ncbi_uid=5185923&amp;link_uid=PMC3739055">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2265.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3103347/figure/F7/" ref="ordinalpos=2265&amp;ncbi_uid=3364421&amp;link_uid=PMC3103347" href="/pmc/articles/PMC3103347/"><img src="/pmc/articles/instance/3103347/bin/zbc0281165510007.gif" src-large="/pmc/articles/instance/3103347/bin/zbc0281165510007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3103347/" ref="ordinalpos=2265&amp;ncbi_uid=3364421&amp;link_uid=PMC3103347" image-link="/pmc/articles/PMC3103347/figure/F7/" class="imagepopup">FIGURE 7. From: Secretoglobin 3A2 Suppresses Bleomycin-induced Pulmonary Fibrosis by Transforming Growth Factor ? <span class="highlight" style="background-color:">Signaling</span> Down-regulation. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram for the SCGB3A2-induced inhibition of the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> SCGB3A2 inhibits collagen production through STAT1 phosphorylation and increases expression of SMAD7, which inhibits TGFβ-mediated SMAD2/3 activation. A newly synthesized molecule <i>X</i> mediates the SCGB3A2-induced STAT1 phosphorylation.</div></p></div><div class="aux"><div class="resc">Reiko Kurotani, et al. J Biol Chem. 2011 June 3;286(22):19682-19692.</div><p class="links"><a class="dblinks" href="/pubmed/21478551" ref="ordinalpos=2265&amp;ncbi_uid=3364421&amp;link_uid=21478551">Citation</a><a class="dblinks" href="/pmc/articles/PMC3103347/" ref="ordinalpos=2265&amp;ncbi_uid=3364421&amp;link_uid=PMC3103347">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2266.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2195865/figure/F2/" ref="ordinalpos=2266&amp;ncbi_uid=1235448&amp;link_uid=PMC2195865" href="/pmc/articles/PMC2195865/"><img src="/pmc/articles/instance/2195865/bin/JEM001425.f2.gif" src-large="/pmc/articles/instance/2195865/bin/JEM001425.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2195865/" ref="ordinalpos=2266&amp;ncbi_uid=1235448&amp;link_uid=PMC2195865" image-link="/pmc/articles/PMC2195865/figure/F2/" class="imagepopup">Figure 2.  From: Cross-Talk in Cell Death <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>How apoptotic thresholding might influence experimental outcome. Many possible explanations may account for the differential sensitivity of cells from the ‘Northern hemisphere’ labs (e.g., Krammer/Peter) versus those from the ‘Southern hemisphere’ (e.g., Strasser) to CD95 stimulation (the Krammer/Peter labs are located in Germany and the Strasser lab is in Australia, hence the Northern and Southern hemisphere references.). Herein is illustrated one hypothesis. It assumes that most cell types have an apoptotic threshold that, when breached, results in apoptotic cell death. This apoptotic threshold (represented by the red plane) requires, say, 10 arbitrary apoptotic units (which could be, among other things, a critical mass of caspase catalytic activity). The ability of a cell to cross this threshold depends on the combined contributions of the extrinsic cell death <span class="highlight" style="background-color:">pathway</span> (gray) and the fraction of the intrinsic <span class="highlight" style="background-color:">pathway</span> (blue) that can be engaged through <span class="highlight" style="background-color:">pathway</span> cross-talk. Type I cells (first bar), in both the Northern and Southern hemispheres, harbor a very efficient DISC mechanism that alone can contribute the necessary apoptotic units to breach the threshold. Thus, regardless of the status of the intrinsic <span class="highlight" style="background-color:">pathway</span>, and even if it is completely blocked by the presence of Bcl-2, the type I cell can muster the necessary currency to engage the apoptotic machinery and die. Therefore, extrinsic and intrinsic pathways in type I cells appear independent (assuming intrinsic does not talk to extrinsic). In type II cells, DISC formation is weaker and the extrinsic <span class="highlight" style="background-color:">pathway</span> depends more on contributions from the intrinsic <span class="highlight" style="background-color:">pathway</span>. These cells behave differently in the Northern hemisphere versus the Southern hemisphere. In the Northern hemisphere, the combination of the extrinsic contribution and the fraction of the intrinsic <span class="highlight" style="background-color:">pathway</span> that it is able to engage is sufficient for apoptosis to proceed (second bar). Attenuation of the intrinsic component, by Bcl-2 overexpression for example, reduces the total number of apoptotic units to below the apoptotic threshold and the cells survive. The extrinsic <span class="highlight" style="background-color:">pathway</span>, therefore, appears to be modulated by Bcl-2. In the Southern hemisphere, the number of available apoptotic units that can be contributed by the intrinsic <span class="highlight" style="background-color:">pathway</span> may be lower. Thus, engagement of the extrinsic <span class="highlight" style="background-color:">pathway</span> with ligating CD95 (Fas/Apo1) antibody or trimerized ligand is not sufficient to mediate passage across the apoptotic threshold on its own (third bar). With a more efficient stimulus, mediated by multimerized CD95L, for example (fourth bar), a critical mass of the extrinsic <span class="highlight" style="background-color:">pathway</span> is enlisted to breach the apoptotic threshold, regardless of the contribution from the intrinsic <span class="highlight" style="background-color:">pathway</span>. Cell death in this scenario is therefore insensitive to Bcl-2 overexpression. The differences observed in the Northern and Southern hemispheres might therefore be explained by a combination of (a) the number of apoptotic units within the intrinsic <span class="highlight" style="background-color:">pathway</span> that can be recruited by cross-talk from the extrinsic <span class="highlight" style="background-color:">pathway</span> (lower in the South) and (b) the strength of the apoptotic stimulus (ligating antibodies and trimerized CD95L being equivalent but weaker than multimerized CD95L).</div></p></div><div class="aux"><div class="resc">Sophie Roy, et al. J Exp Med. 2000 October 16;192(8):f21-f26.</div><p class="links"><a class="dblinks" href="/pubmed/11034597" ref="ordinalpos=2266&amp;ncbi_uid=1235448&amp;link_uid=11034597">Citation</a><a class="dblinks" href="/pmc/articles/PMC2195865/" ref="ordinalpos=2266&amp;ncbi_uid=1235448&amp;link_uid=PMC2195865">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2267.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1995599/figure/F2/" ref="ordinalpos=2267&amp;ncbi_uid=1066285&amp;link_uid=PMC1995599" href="/pmc/articles/PMC1995599/"><img src="/pmc/articles/instance/1995599/bin/nihms25729f2.gif" src-large="/pmc/articles/instance/1995599/bin/nihms25729f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1995599/" ref="ordinalpos=2267&amp;ncbi_uid=1066285&amp;link_uid=PMC1995599" image-link="/pmc/articles/PMC1995599/figure/F2/" class="imagepopup">Figure 2. PTH-stimulated survival <span class="highlight" style="background-color:">signaling</span> in osteoblasts.  From: MOLECULAR AND CELLULAR MECHANISMS OF THE ANABOLIC EFFECT OF INTERMITTENT PTH. </a></p><div class="supp"><p class="details"><div>The upper panel depicts PTH-activated survival pathways, and the negative feedback on these pathways caused by stimulation of Runx2 degradation. The lower panel depicts the transient anti-apoptotic <span class="highlight" style="background-color:">signaling</span> and the changes in Runx2 that occur upon each injection of PTH. The net result is a temporary delay in osteoblast apoptosis. As injected PTH disappears from the circulation, Runx2 returns to basal levels and the cells are positioned to generate another episode of anti-apoptotic <span class="highlight" style="background-color:">signaling</span> in response to a subsequent injection. With sustained PTH elevation, on the other hand, the negative feedback <span class="highlight" style="background-color:">pathway</span> remains active, and further anti-apoptosis <span class="highlight" style="background-color:">signaling</span> is abrogated.</div></p></div><div class="aux"><div class="resc">Robert L. Jilka. Bone. ;40(6):1434-1446.</div><p class="links"><a class="dblinks" href="/pubmed/17517365" ref="ordinalpos=2267&amp;ncbi_uid=1066285&amp;link_uid=17517365">Citation</a><a class="dblinks" href="/pmc/articles/PMC1995599/" ref="ordinalpos=2267&amp;ncbi_uid=1066285&amp;link_uid=PMC1995599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2268.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3356063/figure/F1/" ref="ordinalpos=2268&amp;ncbi_uid=4039137&amp;link_uid=PMC3356063" href="/pmc/articles/PMC3356063/"><img src="/pmc/articles/instance/3356063/bin/fendo-02-00075-g001.gif" src-large="/pmc/articles/instance/3356063/bin/fendo-02-00075-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3356063/" ref="ordinalpos=2268&amp;ncbi_uid=4039137&amp;link_uid=PMC3356063" image-link="/pmc/articles/PMC3356063/figure/F1/" class="imagepopup">Figure 1.  From: Comparative Endocrinology of Aging and Longevity Regulation. </a></p><div class="supp"><p class="details"><div><b>Endocrine <span class="highlight" style="background-color:">signaling</span> pathways affecting longevity</b>. <b>(A)</b> Role of the conserved IIS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in longevity regulation. Insulin-like peptides bind to an insulin/IGF receptor on the plasma membrane. Ligand binding causes dimerization of the receptor, activating the intracellular tyrosine kinase domain. Phosphorylation by the receptor kinase domain activates the PI3 Kinase (PI3K). The <span class="highlight" style="background-color:">signaling</span> activity of PI3K, antagonized by the PTEN phosphatase, activates the Akt kinase. Akt phosphorylates FOXO/DAF-16 which is then sequestered in the cytosol. In the absence of IIS <span class="highlight" style="background-color:">signaling</span>, FOXO translocates to the nucleus and activates transcription of longevity promoting genes. AKT <span class="highlight" style="background-color:">signaling</span> also opposes longevity by activating the TOR <span class="highlight" style="background-color:">pathway</span>, which inhibits autophagy, a pro-longevity process. <b>(B)</b> Endocrine <span class="highlight" style="background-color:">signaling</span> in the <i>C. elegans</i> germline longevity <span class="highlight" style="background-color:">pathway</span>. The germ line stem cells send an unknown signal that allows other tissues to detect that they are present. When the germ line stem cells are ablated, the somatic gonad releases the steroid dafachronic acid that binds to the nuclear hormone receptor DAF-12. It is unclear exactly where DAF-12 acts to receive this signal but the intestine is a possible location. As a result of <span class="highlight" style="background-color:">signaling</span> via DAF-12, DAF-16/FOXO activity increases in the nuclei of intestinal cells. This in turn leads to <span class="highlight" style="background-color:">signaling</span> from the intestine to the other tissues that informs them about status of the DAF-16/FOXO in the intestine. INS-7 promotes IIS <span class="highlight" style="background-color:">signaling</span> which suppresses DAF-16/FOXO activity but DAF-16/FOXO in the intestine can down regulate the expression of INS-7 and this may be one mechanism by which intestinal DAF-16/FOXO controls other tissues. <b>(C)</b> Regulation of longevity by 20-hydroxyecdysone (20E) and juvenile hormone (JH) in insects. JH suppresses reproductive diapause which promotes longevity. JH may also negatively affect longevity in other ways. The receptor(s) to which JH binds is unknown. 20E activates the ecdysone receptor which may reduce longevity. IIS engages in cross talk with both JH and 20E and also reduces longevity via other <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">John B. Allard, et al. Front Endocrinol (Lausanne). 2011;2:75.</div><p class="links"><a class="dblinks" href="/pubmed/22654825" ref="ordinalpos=2268&amp;ncbi_uid=4039137&amp;link_uid=22654825">Citation</a><a class="dblinks" href="/pmc/articles/PMC3356063/" ref="ordinalpos=2268&amp;ncbi_uid=4039137&amp;link_uid=PMC3356063">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2269.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3997714/figure/f3-ol-07-05-1345/" ref="ordinalpos=2269&amp;ncbi_uid=6056255&amp;link_uid=PMC3997714" href="/pmc/articles/PMC3997714/"><img src="/pmc/articles/instance/3997714/bin/OL-07-05-1345-g02.gif" src-large="/pmc/articles/instance/3997714/bin/OL-07-05-1345-g02.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3997714/" ref="ordinalpos=2269&amp;ncbi_uid=6056255&amp;link_uid=PMC3997714" image-link="/pmc/articles/PMC3997714/figure/f3-ol-07-05-1345/" class="imagepopup">Figure 3.  From: Autophagy: A novel therapeutic target for hepatocarcinoma (Review). </a></p><div class="supp"><p class="details"><div>Sorafenib activates autophagy through different <span class="highlight" style="background-color:">signaling</span> pathways. Autophagic cell death is triggered by sorafenib by suppressing myeloid cell leukemia-1 (Mcl-1)-related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hepatocarcinoma (HCC) PLC5, SK-Hep1, HepG2 and Hep3B cells (53). Thus, inducers of autophagy may be used in combination with sorafenib to promote anti-cancer efficacy. Cytoprotective autophagy is triggered by sorafenib through endoplasmic reticulum (ER) or the Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in MHCC97-L and PLC/PRF/5 cells (47,48). Inhibitors of autophagy may therefore be used in combination with sorafenib to promote anti-cancer efficacy.</div></p></div><div class="aux"><div class="resc">ZHANGGUI WANG, et al. Oncol Lett. 2014 May;7(5):1345-1351.</div><p class="links"><a class="dblinks" href="/pubmed/24765136" ref="ordinalpos=2269&amp;ncbi_uid=6056255&amp;link_uid=24765136">Citation</a><a class="dblinks" href="/pmc/articles/PMC3997714/" ref="ordinalpos=2269&amp;ncbi_uid=6056255&amp;link_uid=PMC3997714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2270.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3292340/figure/F2/" ref="ordinalpos=2270&amp;ncbi_uid=3981916&amp;link_uid=PMC3292340" href="/pmc/articles/PMC3292340/"><img src="/pmc/articles/instance/3292340/bin/1632fig2.gif" src-large="/pmc/articles/instance/3292340/bin/1632fig2.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3292340/" ref="ordinalpos=2270&amp;ncbi_uid=3981916&amp;link_uid=PMC3292340" image-link="/pmc/articles/PMC3292340/figure/F2/" class="imagepopup">FIG. 2. From: Drosophila Insulin <span class="highlight" style="background-color:">Pathway</span> Mutants Affect Visual Physiology and Brain Function Besides Growth, Lipid, and Carbohydrate Metabolism. </a></p><div class="supp"><p class="details"><div>Insulin <span class="highlight" style="background-color:">signaling</span> is required for lipid and carbohydrate metabolism. <i>A</i>: Histogram shows the percentage increases in lipids of heteroallelic and homozygous mutant flies for the insulin <span class="highlight" style="background-color:">pathway</span> compared with paired heterozygous sibling controls. <i>B</i>: Histogram shows the percentage change in body carbohydrates of heteroallelic and homozygous mutant flies for the insulin <span class="highlight" style="background-color:">pathway</span> compared with paired heterozygous siblings (<i>n</i> = 8). *<i>P</i> &lt; 0.01. The error bars represent the SEM.</div></p></div><div class="aux"><div class="resc">Juan M. Murillo-Maldonado, et al. Diabetes. 2011 May;60(5):1632-1636.</div><p class="links"><a class="dblinks" href="/pubmed/21464442" ref="ordinalpos=2270&amp;ncbi_uid=3981916&amp;link_uid=21464442">Citation</a><a class="dblinks" href="/pmc/articles/PMC3292340/" ref="ordinalpos=2270&amp;ncbi_uid=3981916&amp;link_uid=PMC3292340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2271.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3360134/figure/F7/" ref="ordinalpos=2271&amp;ncbi_uid=5054821&amp;link_uid=PMC3360134" href="/pmc/articles/PMC3360134/"><img src="/pmc/articles/instance/3360134/bin/nihms349620f7.gif" src-large="/pmc/articles/instance/3360134/bin/nihms349620f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3360134/" ref="ordinalpos=2271&amp;ncbi_uid=5054821&amp;link_uid=PMC3360134" image-link="/pmc/articles/PMC3360134/figure/F7/" class="imagepopup">Figure 7.  From: Increased expression of NuSAP in recurrent prostate cancer is mediated by E2F1. </a></p><div class="supp"><p class="details"><div>Schematic representation of possible role of NuSAP in the Rb-E2F <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> including possible interactions with AR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Zulfiqar G Gulzar, et al. Oncogene. ;32(1):70-77.</div><p class="links"><a class="dblinks" href="/pubmed/22349817" ref="ordinalpos=2271&amp;ncbi_uid=5054821&amp;link_uid=22349817">Citation</a><a class="dblinks" href="/pmc/articles/PMC3360134/" ref="ordinalpos=2271&amp;ncbi_uid=5054821&amp;link_uid=PMC3360134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2272.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4218941/figure/f0005/" ref="ordinalpos=2272&amp;ncbi_uid=6818985&amp;link_uid=PMC4218941" href="/pmc/articles/PMC4218941/"><img src="/pmc/articles/instance/4218941/bin/gr1.gif" src-large="/pmc/articles/instance/4218941/bin/gr1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4218941/" ref="ordinalpos=2272&amp;ncbi_uid=6818985&amp;link_uid=PMC4218941" image-link="/pmc/articles/PMC4218941/figure/f0005/" class="imagepopup">Fig. 1.  From: Regulation of the effects of CYP2E1-induced oxidative stress by JNK <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Regulation and effects of JNK <span class="highlight" style="background-color:">signaling</span> in hepatocytes. Activation of JNK in hepatocytes occurs through phosphorylation of tyrosine and threonine residues that is triggered by extracellular stimuli including growth factors and cytokines, injurious mediators including pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPS) and nutritional and metabolic factors. This phosphorylation involves a multi-step protein kinase <span class="highlight" style="background-color:">signaling</span> cascade with MAPKKKs and the downstream mitogen-activated protein kinase kinases MKK4 and MKK7. JNK phosphorylation is negatively regulated by <span class="highlight" style="background-color:">signaling</span> through ERK1/2, PKC and PKD. JNK activity is also down regulated by phosphatases. JNK <span class="highlight" style="background-color:">signaling</span> effects are predominantly mediated through transcriptional activation of the c-Jun/AP-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, but also by the direct effects of JNK to lead to biological effects such as cell proliferation or death. Red lines show inhibitory pathways.</div></p></div><div class="aux"><div class="resc">Jörn M. Schattenberg, et al. Redox Biol. 2014;3:7-15.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2272&amp;ncbi_uid=6818985&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4218941/" ref="ordinalpos=2272&amp;ncbi_uid=6818985&amp;link_uid=PMC4218941">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2273.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3257793/figure/F1/" ref="ordinalpos=2273&amp;ncbi_uid=3763695&amp;link_uid=PMC3257793" href="/pmc/articles/PMC3257793/"><img src="/pmc/articles/instance/3257793/bin/psb0607_1049_fig001.gif" src-large="/pmc/articles/instance/3257793/bin/psb0607_1049_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3257793/" ref="ordinalpos=2273&amp;ncbi_uid=3763695&amp;link_uid=PMC3257793" image-link="/pmc/articles/PMC3257793/figure/F1/" class="imagepopup">Figure 1.  From: NO way!. </a></p><div class="supp"><p class="details"><div>Schematic representation of IKK/NκFB-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating NO levels in tomato seedlings. The sites of action of the pharmacological inhibitors as derived from existing mammalian models are indicated. The putative role of native SHR as a repressor of a NFκB-like molecule is also indicated. The mutation in <i>shr</i> presumably results in the removal of this negative regulator thereby resulting in the constitutive transcriptional activation of a NOS-like enzyme. The increase in NOS-like activity leads to increase in NO levels in the mutant resulting in inhibition of root elongation.</div></p></div><div class="aux"><div class="resc">Sangeeta Negi, et al. Plant Signal Behav. 2011 July;6(7):1049-1052.</div><p class="links"><a class="dblinks" href="/pubmed/21701257" ref="ordinalpos=2273&amp;ncbi_uid=3763695&amp;link_uid=21701257">Citation</a><a class="dblinks" href="/pmc/articles/PMC3257793/" ref="ordinalpos=2273&amp;ncbi_uid=3763695&amp;link_uid=PMC3257793">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2274.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2993798/figure/F2/" ref="ordinalpos=2274&amp;ncbi_uid=2613431&amp;link_uid=PMC2993798" href="/pmc/articles/PMC2993798/"><img src="/pmc/articles/instance/2993798/bin/aging-02-678-g002.gif" src-large="/pmc/articles/instance/2993798/bin/aging-02-678-g002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2993798/" ref="ordinalpos=2274&amp;ncbi_uid=2613431&amp;link_uid=PMC2993798" image-link="/pmc/articles/PMC2993798/figure/F2/" class="imagepopup">Figure 2.Elevated O-GlcNAc modification of proteins does not increase adult lifespan further in long-lived insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutant animals..  From: Intracellular protein glycosylation modulates insulin mediated lifespan in C. elegans. </a></p><div class="supp"><p class="details"><div>(<b>A</b> and <b>B</b>) Lifespan of <i>age-1(hx546)</i> and <i>sgk-1(ok538)</i> mutants is dependent on protein O-GlcNAc modification as seen in <i>age-1; ogt-1</i> (<b>A</b>) and <i>sgk-1; ogt-1</i> (<b>B</b>) double mutants. (<b>C</b> and <b>D</b>) The adult lifespan extension associated with the <i>oga-1(ok1207)</i> mutant is not synergistic or additive with mutations of the long-lived insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> kinase <i>age-1</i> (<b>C</b>) or sgk-1 (<b>D</b>). Lifespan curves (<b>A-D</b>) are based on pooled data from multiple replicates as described in Table 1. Control curves are re-plotted to facilitate comparisons in panels <b>A-D</b>.</div></p></div><div class="aux"><div class="resc">Mohammad M. Rahman, et al. Aging (Albany NY). 2010 October;2(10):678-690.</div><p class="links"><a class="dblinks" href="/pubmed/20952811" ref="ordinalpos=2274&amp;ncbi_uid=2613431&amp;link_uid=20952811">Citation</a><a class="dblinks" href="/pmc/articles/PMC2993798/" ref="ordinalpos=2274&amp;ncbi_uid=2613431&amp;link_uid=PMC2993798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2275.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3130517/figure/fig6/" ref="ordinalpos=2275&amp;ncbi_uid=3079316&amp;link_uid=PMC3130517" href="/pmc/articles/PMC3130517/"><img src="/pmc/articles/instance/3130517/bin/ECAM2011-939205.006.gif" src-large="/pmc/articles/instance/3130517/bin/ECAM2011-939205.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3130517/" ref="ordinalpos=2275&amp;ncbi_uid=3079316&amp;link_uid=PMC3130517" image-link="/pmc/articles/PMC3130517/figure/fig6/" class="imagepopup">Figure 6.  From: Ethanol Extract from Ampelopsis sinica Root Exerts Anti-Hepatitis B Virus Activity via Inhibition of p53 <span class="highlight" style="background-color:">Pathway</span> In Vitro. </a></p><div class="supp"><p class="details"><div>The effect of EASR on cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activity <i>in vitro</i>. The HepG2 cells were transfected with various plasmids provided by PathDetect Cis-/Trans-Reporting Systems. In the experimental group, cells were treated with 40 <i>μ</i>g mL<sup>−1</sup> EASR for 48 h. And the control group was cultured without EASR treatment. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activities were determined by luciferase reporter assay. EASR inhibited the activity of p53 <span class="highlight" style="background-color:">pathway</span> significantly (<i>P</i> &lt; .05). The data are presented as means ± SD (<i>n</i> = 3). *<i>P</i> &lt; .05 as compared with the control group.</div></p></div><div class="aux"><div class="resc">Ran Pang, et al. Evid Based Complement Alternat Med. 2011;2011:939205.</div><p class="links"><a class="dblinks" href="/pubmed/21738555" ref="ordinalpos=2275&amp;ncbi_uid=3079316&amp;link_uid=21738555">Citation</a><a class="dblinks" href="/pmc/articles/PMC3130517/" ref="ordinalpos=2275&amp;ncbi_uid=3079316&amp;link_uid=PMC3130517">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2276.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3871617/figure/pone-0082951-g005/" ref="ordinalpos=2276&amp;ncbi_uid=5635783&amp;link_uid=PMC3871617" href="/pmc/articles/PMC3871617/"><img src="/pmc/articles/instance/3871617/bin/pone.0082951.g005.gif" src-large="/pmc/articles/instance/3871617/bin/pone.0082951.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3871617/" ref="ordinalpos=2276&amp;ncbi_uid=5635783&amp;link_uid=PMC3871617" image-link="/pmc/articles/PMC3871617/figure/pone-0082951-g005/" class="imagepopup">Figure 5. Calcium <span class="highlight" style="background-color:">signaling</span> network remodeling in aging porcine left atrium..  From: NHE Isoform Switching and KChIP2 Upregulation in Aging Porcine Atria. </a></p><div class="supp"><p class="details"><div>Differentially expressed genes within a calcium <span class="highlight" style="background-color:">signaling</span> network in old versus young left atrium; network identified by <span class="highlight" style="background-color:">pathway</span> analysis (Ariadne). p = 0.009 for old versus young left atrial tissue with respect to the gene network shown (<i>n</i> = 3–5).</div></p></div><div class="aux"><div class="resc">Ritu Kant, et al. PLoS One. 2013;8(12):e82951.</div><p class="links"><a class="dblinks" href="/pubmed/24376615" ref="ordinalpos=2276&amp;ncbi_uid=5635783&amp;link_uid=24376615">Citation</a><a class="dblinks" href="/pmc/articles/PMC3871617/" ref="ordinalpos=2276&amp;ncbi_uid=5635783&amp;link_uid=PMC3871617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2277.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3986937/figure/F5/" ref="ordinalpos=2277&amp;ncbi_uid=6024741&amp;link_uid=PMC3986937" href="/pmc/articles/PMC3986937/"><img src="/pmc/articles/instance/3986937/bin/1742-2094-11-59-5.gif" src-large="/pmc/articles/instance/3986937/bin/1742-2094-11-59-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3986937/" ref="ordinalpos=2277&amp;ncbi_uid=6024741&amp;link_uid=PMC3986937" image-link="/pmc/articles/PMC3986937/figure/F5/" class="imagepopup">Figure 5.  From: Curcumin attenuates acute inflammatory injury by inhibiting the TLR4/MyD88/NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in experimental traumatic brain injury. </a></p><div class="supp"><p class="details"><div><b>Curcumin suppressed TLR4/MyD88/NF-</b>κ<b>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein expression </b><b><i>in vivo</i></b><b>. (A)</b> TLR4 and MyD88 protein expression in the injured tissue was significantly lower in curcumin-treated mice than in vehicle-treated mice at 24 hours post-trauma. <b>(B)</b> NF-κB p65 and p-IκB-α protein expression in the injured tissue was also significantly lower in curcumin-treated mice than in vehicle-treated mice at 24 hours post-trauma. In contrast, IκB-α protein expression was significantly higher in curcumin-treated mice than in vehicle-treated mice post-trauma. Values (mean ± SD) are representative of three independent experiments (n = 3 mice/group). *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Hai-tao Zhu, et al. J Neuroinflammation. 2014;11:59-59.</div><p class="links"><a class="dblinks" href="/pubmed/24669820" ref="ordinalpos=2277&amp;ncbi_uid=6024741&amp;link_uid=24669820">Citation</a><a class="dblinks" href="/pmc/articles/PMC3986937/" ref="ordinalpos=2277&amp;ncbi_uid=6024741&amp;link_uid=PMC3986937">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2278.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2675960/figure/F6/" ref="ordinalpos=2278&amp;ncbi_uid=1791229&amp;link_uid=PMC2675960" href="/pmc/articles/PMC2675960/"><img src="/pmc/articles/instance/2675960/bin/zmg0050944210006.gif" src-large="/pmc/articles/instance/2675960/bin/zmg0050944210006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2675960/" ref="ordinalpos=2278&amp;ncbi_uid=1791229&amp;link_uid=PMC2675960" image-link="/pmc/articles/PMC2675960/figure/F6/" class="imagepopup">Figure 6.  From: Dominant Activation of the Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in the Ovary Alters Theca Development and Prevents Ovulation. </a></p><div class="supp"><p class="details"><div>Expression of genes in the HH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in granulosa cells and residual ovarian tissue from <i>Amhr2</i><sup>+/+</sup><i>SmoM2</i> and <i>Amhr2</i><sup><i>cre</i>/+</sup><i>SmoM2</i> mice. Tissues were obtained from untreated mice (UT) or from eCG-primed mice before (0 h) or after injection of hCG. Total RNA was assayed by quantitative real-time RT-PCR. Data are mean ± <span style="font-variant: small-caps" class="small-caps">sem</span>; n = 3 granulosa cell and residual tissue preparations. <i>Bars</i> without <i>common superscripts</i> are significantly different (<i>P</i> &lt; 0.05).</div></p></div><div class="aux"><div class="resc">Yi Ren, et al. Mol Endocrinol. 2009 May;23(5):711-723.</div><p class="links"><a class="dblinks" href="/pubmed/19196835" ref="ordinalpos=2278&amp;ncbi_uid=1791229&amp;link_uid=19196835">Citation</a><a class="dblinks" href="/pmc/articles/PMC2675960/" ref="ordinalpos=2278&amp;ncbi_uid=1791229&amp;link_uid=PMC2675960">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2279.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3118704/figure/f3/" ref="ordinalpos=2279&amp;ncbi_uid=3376498&amp;link_uid=PMC3118704" href="/pmc/articles/PMC3118704/"><img src="/pmc/articles/instance/3118704/bin/fig-3.gif" src-large="/pmc/articles/instance/3118704/bin/fig-3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3118704/" ref="ordinalpos=2279&amp;ncbi_uid=3376498&amp;link_uid=PMC3118704" image-link="/pmc/articles/PMC3118704/figure/f3/" class="imagepopup">FIG. 3. From: Redox <span class="highlight" style="background-color:">Signaling</span> and the Innate Immune System in Alcoholic Liver Disease. </a></p><div class="supp"><p class="details"><div><b>Toll-like receptor-4 (TLR-4) <span class="highlight" style="background-color:">signaling</span></b> <b><i>via</i></b> <b>MyD-88-dependent pathways.</b> Chronic ethanol exposure increases lipopolysaccharide (LPS) in the circulation, and initiates a <span class="highlight" style="background-color:">signaling</span> cascade <i>via</i> TLR-4 on the surface of Kupffer cells. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activates mitogen-activated protein kinases (MAPKs), p38, c-Jun N-terminal kinase (JNK), and extracellular signal-regulated kinases 1 and 2 (ERK1/2), resulting in increased tumor necrosis factor (TNF)-α expression and stability.</div></p></div><div class="aux"><div class="resc">Jessica I. Cohen, et al. Antioxid Redox Signal. 2011 July 15;15(2):523-534.</div><p class="links"><a class="dblinks" href="/pubmed/21126203" ref="ordinalpos=2279&amp;ncbi_uid=3376498&amp;link_uid=21126203">Citation</a><a class="dblinks" href="/pmc/articles/PMC3118704/" ref="ordinalpos=2279&amp;ncbi_uid=3376498&amp;link_uid=PMC3118704">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2280.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3794382/figure/F5/" ref="ordinalpos=2280&amp;ncbi_uid=5390260&amp;link_uid=PMC3794382" href="/pmc/articles/PMC3794382/"><img src="/pmc/articles/instance/3794382/bin/fphys-04-00278-g0005.gif" src-large="/pmc/articles/instance/3794382/bin/fphys-04-00278-g0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3794382/" ref="ordinalpos=2280&amp;ncbi_uid=5390260&amp;link_uid=PMC3794382" image-link="/pmc/articles/PMC3794382/figure/F5/" class="imagepopup">Figure 5.  From: Methods and approaches in the topology-based analysis of biological pathways. </a></p><div class="supp"><p class="details"><div><b>Comparison of representative graph models for molecular interactions as used by different <span class="highlight" style="background-color:">pathway</span> databases.</b> In a KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(A)</b> nodes represent genes/gene products and edges represent regulatory signals such as activation, inhibition, phosphorylation, etc. (see http://www.genome.jp/kegg/document/help_<span class="highlight" style="background-color:">pathway</span>.html for details). In the chemical network representation of a KEGG metabolic <span class="highlight" style="background-color:">pathway</span> <b>(B)</b> the nodes represent biochemical compounds and edges represent chemical reactions. These chemical reactions are performed by enzymes which are proteins encoded by genes. Hence, in contrast with the <span class="highlight" style="background-color:">signaling</span> pathways in which genes are associated with nodes, in a metabolic pathways genes are associated with edges. This is the main reason most methods developed for <span class="highlight" style="background-color:">signaling</span> pathways cannot be applied directly to metabolic pathways. In an NCI-PID <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(C)</b> nodes fall in two categories: component nodes representing biomolecular components, or process nodes representing biochemical reactions or biological processes. Edges connect two biomolecular components through a biochemical reaction or a biological process. Process nodes can have 3 states: positive regulation, negative regulation, or “involved in.” (see http://pid.nci.nih.gov/userguide/network_maps.shtml for details). In a protein-protein interaction network <b>(D)</b> nodes represent proteins and the interactions among them represent physical binding. These interactions can be inferred from two-hybrid assays and they may be either undirected (top), or directed from the bait protein to the prey protein (bottom). In the Biological <span class="highlight" style="background-color:">Pathway</span> Exchange (BioPAX) <b>(E)</b> nodes are physical entities and edges are conversions. BioPAX entities can represent complexes, DNA, proteins, RNA, small molecules, DNA regions or RNA regions. Conversions can represent biochemical reactions complex assembly or degradation, transport or transport with biochemical reaction. This model is very generic and increasingly flexible. It provides a standard for <span class="highlight" style="background-color:">pathway</span> information to be available in machine readable format, therefore easy to use for <span class="highlight" style="background-color:">pathway</span> analysis and to exchange between <span class="highlight" style="background-color:">pathway</span> databases (see http://www.biopax.org/release/biopax-level3-documentation.pdf for details).</div></p></div><div class="aux"><div class="resc">Cristina Mitrea, et al. Front Physiol. 2013;4:278.</div><p class="links"><a class="dblinks" href="/pubmed/24133454" ref="ordinalpos=2280&amp;ncbi_uid=5390260&amp;link_uid=24133454">Citation</a><a class="dblinks" href="/pmc/articles/PMC3794382/" ref="ordinalpos=2280&amp;ncbi_uid=5390260&amp;link_uid=PMC3794382">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2281.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g004/" ref="ordinalpos=2281&amp;ncbi_uid=1718066&amp;link_uid=PMC2634968" href="/pmc/articles/PMC2634968/"><img src="/pmc/articles/instance/2634968/bin/pcbi.1000292.g004.gif" src-large="/pmc/articles/instance/2634968/bin/pcbi.1000292.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634968/" ref="ordinalpos=2281&amp;ncbi_uid=1718066&amp;link_uid=PMC2634968" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g004/" class="imagepopup">Figure 4. An overview of the discrete <span class="highlight" style="background-color:">signaling</span> (DIOS) pathways defined in the TLR network..  From: Identification of Potential <span class="highlight" style="background-color:">Pathway</span> Mediation Targets in Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>There were a total of ten pathways that signaled from input receptor signals to output transcription-level objectives. These ten pathways shared fourteen receptor signals and five output objectives. The most redundant objective was NF-kappa-B activation, which was the target for a majority of the pathways. Indeed, four of the pathways—RIP1, NOD1, NOD2, and RIP2/TRIP6/TRAF2—signaled only to NF-kappa-B. However, also note that IL-1 and a large subset of the TLRs signaled to multiple objectives through a variety of pathways such as PI3K, IL-1, and MyD88. Overall, this receptor-<span class="highlight" style="background-color:">pathway</span>-output format allowed for a better understanding of the TLR network and its input–output relationships, and also for the calculation of essential reactions as candidates for <span class="highlight" style="background-color:">signaling</span> mediation. Red: A summary of the eight critical network reactions identified through our analysis (see text). These control points were located within the ROS production, IL-1, and MyD88 pathways. Although some essential network reactions were identified for the other discrete <span class="highlight" style="background-color:">signaling</span> pathways, they were unsuitable for selective inhibition due either to their role in other <span class="highlight" style="background-color:">signaling</span> processes or their lack of specificity to a particular <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Fan Li, et al. PLoS Comput Biol. 2009 February;5(2):e1000292.</div><p class="links"><a class="dblinks" href="/pubmed/19229310" ref="ordinalpos=2281&amp;ncbi_uid=1718066&amp;link_uid=19229310">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634968/" ref="ordinalpos=2281&amp;ncbi_uid=1718066&amp;link_uid=PMC2634968">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2282.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3134295/figure/f1-cancers-03-01527/" ref="ordinalpos=2282&amp;ncbi_uid=5269010&amp;link_uid=PMC3134295" href="/pmc/articles/PMC3134295/"><img src="/pmc/articles/instance/3757376/bin/cancers-03-01527f1.gif" src-large="/pmc/articles/instance/3757376/bin/cancers-03-01527f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3134295/" ref="ordinalpos=2282&amp;ncbi_uid=5269010&amp;link_uid=PMC3134295" image-link="/pmc/articles/PMC3134295/figure/f1-cancers-03-01527/" class="imagepopup">Figure 1. From: Small Molecule Inhibitors of Bcl-2 Family Proteins for Pancreatic Cancer Therapy. </a></p><div class="supp"><p class="details"><div>The Apoptotic Process. There are two major apoptotic <span class="highlight" style="background-color:">signaling</span> pathways: the extrinsic <span class="highlight" style="background-color:">pathway</span> and the mitochondria (intrinsic) <span class="highlight" style="background-color:">pathway</span>. In addition, there is another <span class="highlight" style="background-color:">pathway</span> that involves T-cell mediated cytotoxicity and perforin-granzyme-dependent killing of the cell. The mitochondrial (intrinsic) <span class="highlight" style="background-color:">pathway</span> is regulated by Bcl-2 family and activated by mitochondrial disruption with subsequent cytochrome c release. Initiators of this <span class="highlight" style="background-color:">pathway</span> include UV irradiation and cytotoxic drugs. An ‘apoptosome’ is formed by the interaction of cytochrome c, Apaf-1, d-ATP/ ATP and procaspase-9 with subsequent initiation of the caspase cascade.</div></p></div><div class="aux"><div class="resc">Ashiq Masood, et al. Cancers (Basel). 2011 June;3(2):1527-1549.</div><p class="links"><a class="dblinks" href="/pubmed/21760983" ref="ordinalpos=2282&amp;ncbi_uid=5269010&amp;link_uid=21760983">Citation</a><a class="dblinks" href="/pmc/articles/PMC3134295/" ref="ordinalpos=2282&amp;ncbi_uid=5269010&amp;link_uid=PMC3134295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2283.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3118802/figure/pone-0020887-g003/" ref="ordinalpos=2283&amp;ncbi_uid=3037558&amp;link_uid=PMC3118802" href="/pmc/articles/PMC3118802/"><img src="/pmc/articles/instance/3118802/bin/pone.0020887.g003.gif" src-large="/pmc/articles/instance/3118802/bin/pone.0020887.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3118802/" ref="ordinalpos=2283&amp;ncbi_uid=3037558&amp;link_uid=PMC3118802" image-link="/pmc/articles/PMC3118802/figure/pone-0020887-g003/" class="imagepopup">Figure 3. A schematic figure illustrating progressive activation of a prototype toxicity <span class="highlight" style="background-color:">pathway</span>, with attendant discrete phenotypic transitions..  From: Toxicity Testing in the 21st Century: Defining New Risk Assessment Approaches Based on Perturbation of Intracellular Toxicity Pathways. </a></p><div class="supp"><p class="details"><div>Text on right of figure shows proposed approaches for characterizing discrete transitions through multiple cellular phenotypes, i.e., (1) basal function, (2) minimally perturbed cellular states, (3) upregulation of adaptive, homeostatic gene batteries, and finally (4) overtly adverse states with excessive <span class="highlight" style="background-color:">pathway</span> perturbations. The structure of the circuitry with various embedded, nonlinear feedback loops is schematized in the middle panels. In the context of <i>in vitro</i> assay design, varying free concentrations of a test compound in the test media lead to increasing activation of <span class="highlight" style="background-color:">pathway</span> component (<span class="highlight" style="background-color:">signaling</span> protein) A, thereby driving <span class="highlight" style="background-color:">pathway</span> perturbations of <span class="highlight" style="background-color:">signaling</span> components B and C, and further <span class="highlight" style="background-color:">signaling</span> events downstream of C. Each of these steps is expected to be associated with specific alterations in gene expression, phenotypic read-out and <span class="highlight" style="background-color:">pathway</span> activation, identified by CSBP modeling. As the concentration increases, various portions of the network would be sequentially activated until full activation was achieved (indicated by progressively darker shading of <span class="highlight" style="background-color:">pathway</span> components). Full <span class="highlight" style="background-color:">pathway</span> activation triggers a robust response throughout the <span class="highlight" style="background-color:">pathway</span> circuitry, leading to adverse outcomes measurable in the cellular assay.</div></p></div><div class="aux"><div class="resc">Sudin Bhattacharya, et al. PLoS One. 2011;6(6):e20887.</div><p class="links"><a class="dblinks" href="/pubmed/21701582" ref="ordinalpos=2283&amp;ncbi_uid=3037558&amp;link_uid=21701582">Citation</a><a class="dblinks" href="/pmc/articles/PMC3118802/" ref="ordinalpos=2283&amp;ncbi_uid=3037558&amp;link_uid=PMC3118802">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2284.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3060875/figure/pone-0017940-g003/" ref="ordinalpos=2284&amp;ncbi_uid=2843553&amp;link_uid=PMC3060875" href="/pmc/articles/PMC3060875/"><img src="/pmc/articles/instance/3060875/bin/pone.0017940.g003.gif" src-large="/pmc/articles/instance/3060875/bin/pone.0017940.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3060875/" ref="ordinalpos=2284&amp;ncbi_uid=2843553&amp;link_uid=PMC3060875" image-link="/pmc/articles/PMC3060875/figure/pone-0017940-g003/" class="imagepopup">Figure 3. Modulation of IL-17 <span class="highlight" style="background-color:">signaling</span> during acute thymic                                 atrophy..  From: Acute Endotoxin-Induced Thymic Atrophy Is Characterized By                     Intrathymic Inflammatory and Wound Healing Responses. </a></p><div class="supp"><p class="details"><div>The IL-17 <span class="highlight" style="background-color:">pathway</span> ranked as the third most statistically significant                                 <span class="highlight" style="background-color:">pathway</span>                                     (p-value = 1.45×10<sup>−13</sup>)                                 using Metacore <span class="highlight" style="background-color:">pathway</span> analysis. This <span class="highlight" style="background-color:">pathway</span> includes genes for                                 both secreted pro-inflammatory cytokines and the intracellular                                 response through IL-17R. Red genes represent significant increases                                 in mRNA levels and blue genes represent significant decreases in                                 mRNA levels in thymus tissue following LPS treatment. Mean fold                                 changes are given. Green arrows indicate positive influence. Arrows                                 may represent multiple steps. Plasma membrane is shown in gray.                                 Nuclear membrane is shown as dotted line. A detailed version of the                                 original Metacore <span class="highlight" style="background-color:">pathway</span> is in Figure                                     S4. A full legend of all GeneGo <span class="highlight" style="background-color:">pathway</span> map symbols is in                                     Figure S9 or at http://www.genego.com/pdf/MC_legend.pdf.</div></p></div><div class="aux"><div class="resc">Matthew J. Billard, et al. PLoS One. 2011;6(3):e17940.</div><p class="links"><a class="dblinks" href="/pubmed/21437240" ref="ordinalpos=2284&amp;ncbi_uid=2843553&amp;link_uid=21437240">Citation</a><a class="dblinks" href="/pmc/articles/PMC3060875/" ref="ordinalpos=2284&amp;ncbi_uid=2843553&amp;link_uid=PMC3060875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2285.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3785105/figure/F3/" ref="ordinalpos=2285&amp;ncbi_uid=5673385&amp;link_uid=PMC3785105" href="/pmc/articles/PMC3785105/"><img src="/pmc/articles/instance/3785105/bin/nihms458938f3.gif" src-large="/pmc/articles/instance/3785105/bin/nihms458938f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3785105/" ref="ordinalpos=2285&amp;ncbi_uid=5673385&amp;link_uid=PMC3785105" image-link="/pmc/articles/PMC3785105/figure/F3/" class="imagepopup">Figure 3.  From: Extended haplotype association study in Crohn's disease identifies a novel, Ashkenazi Jewish-specific missense mutation in the NF-κB <span class="highlight" style="background-color:">pathway</span> gene, HEATR3. </a></p><div class="supp"><p class="details"><div><i>HEATR3</i> is a positive component of the NOD2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. siRNA mediated knock down of <i>HEATR3</i> diminished NOD2-dependent NF-<i>κ</i>B induction (A) and IL-8 secretion (B) relative to non-targeting siRNA controls. Furthermore, over-expression of <i>HEATR3</i> increased NF-κB <span class="highlight" style="background-color:">signaling</span> (C) and IL-8 production (D). Cells were left unstimulated (black bars) or stimulated overnight with 20 ng/mL of MDP (grey bars). Together these results support a positive role for <i>HEATR3</i> in NOD2 <span class="highlight" style="background-color:">signaling</span>. * P&lt;0.05, ** P&lt;0.01, *** P&lt;0.001.</div></p></div><div class="aux"><div class="resc">Wei Zhang, et al. Genes Immun. 2013 Jul-Aug;14(5):310-316.</div><p class="links"><a class="dblinks" href="/pubmed/23615072" ref="ordinalpos=2285&amp;ncbi_uid=5673385&amp;link_uid=23615072">Citation</a><a class="dblinks" href="/pmc/articles/PMC3785105/" ref="ordinalpos=2285&amp;ncbi_uid=5673385&amp;link_uid=PMC3785105">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2286.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4314128/figure/f2-molcell-38-1-14/" ref="ordinalpos=2286&amp;ncbi_uid=7164160&amp;link_uid=PMC4314128" href="/pmc/articles/PMC4314128/"><img src="/pmc/articles/instance/4314128/bin/molcell-38-1-14f2.gif" src-large="/pmc/articles/instance/4314128/bin/molcell-38-1-14f2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4314128/" ref="ordinalpos=2286&amp;ncbi_uid=7164160&amp;link_uid=PMC4314128" image-link="/pmc/articles/PMC4314128/figure/f2-molcell-38-1-14/" class="imagepopup">Fig. 2. From: EphB/ephrinB <span class="highlight" style="background-color:">Signaling</span> in Cell Adhesion and Migration. </a></p><div class="supp"><p class="details"><div>Phosphorylation-independent ephrinB1 <span class="highlight" style="background-color:">signaling</span>. EphrinB1 promotes cell migration through the activation of Wnt/PCP <span class="highlight" style="background-color:">pathway</span> by interacting with CNK1 and Dsh. In addition, ephrinB1 interacts Par-6, and sequentially releases Cdc42-GTP from intact Par polarity complex, which has a major role in stabilizing TJs. Finally, accumulation of unphosphorylated ephrinB1 in TJs causes disruption of TJs and cell adhesion.</div></p></div><div class="aux"><div class="resc">Inji Park, et al. Mol Cells. 2015 January 31;38(1):14-19.</div><p class="links"><a class="dblinks" href="/pubmed/25475547" ref="ordinalpos=2286&amp;ncbi_uid=7164160&amp;link_uid=25475547">Citation</a><a class="dblinks" href="/pmc/articles/PMC4314128/" ref="ordinalpos=2286&amp;ncbi_uid=7164160&amp;link_uid=PMC4314128">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2287.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3034644/figure/F2/" ref="ordinalpos=2287&amp;ncbi_uid=2755792&amp;link_uid=PMC3034644" href="/pmc/articles/PMC3034644/"><img src="/pmc/articles/instance/3034644/bin/nihms267921f2.gif" src-large="/pmc/articles/instance/3034644/bin/nihms267921f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3034644/" ref="ordinalpos=2287&amp;ncbi_uid=2755792&amp;link_uid=PMC3034644" image-link="/pmc/articles/PMC3034644/figure/F2/" class="imagepopup">Figure 2.  From: Extracellular regulation of BMP <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Self-regulation in <i>Xenopus</i> dorsal-ventral patterning.<br />The <span class="highlight" style="background-color:">pathway</span> consists of a network of BMP-interacting proteins. The diagram indicates extracellular biochemical protein–protein interactions as black arrows, transcriptional regulation under the control of BMP <span class="highlight" style="background-color:">signaling</span> as green arrows, and the directional flow of Chordin–BMP complexes from the dorsal to the ventral side of the embryo as red arrows.</div></p></div><div class="aux"><div class="resc">Lise Zakin, et al. Curr Biol. ;20(3):R89-R92.</div><p class="links"><a class="dblinks" href="/pubmed/20144774" ref="ordinalpos=2287&amp;ncbi_uid=2755792&amp;link_uid=20144774">Citation</a><a class="dblinks" href="/pmc/articles/PMC3034644/" ref="ordinalpos=2287&amp;ncbi_uid=2755792&amp;link_uid=PMC3034644">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2288.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2638736/figure/F1/" ref="ordinalpos=2288&amp;ncbi_uid=1737863&amp;link_uid=PMC2638736" href="/pmc/articles/PMC2638736/"><img src="/pmc/articles/instance/2650170/bin/zpq9990967830001.gif" src-large="/pmc/articles/instance/2650170/bin/zpq9990967830001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2638736/" ref="ordinalpos=2288&amp;ncbi_uid=1737863&amp;link_uid=PMC2638736" image-link="/pmc/articles/PMC2638736/figure/F1/" class="imagepopup">Fig. 1. From: Identification of compounds that potentiate CREB <span class="highlight" style="background-color:">signaling</span> as possible enhancers of long-term memory. </a></p><div class="supp"><p class="details"><div>Scaffolds of enhancers of the CREB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) Ten of 30 scaffolds we identified are shown. Each scaffold is identified by a number. (<i>B</i>) Six scaffolds of compounds with known biological activities are shown. In parentheses are shown the number of active compounds found over the total number of compounds from that scaffold screened.</div></p></div><div class="aux"><div class="resc">Menghang Xia, et al. Proc Natl Acad Sci U S A. 2009 February 17;106(7):2412-2417.</div><p class="links"><a class="dblinks" href="/pubmed/19196967" ref="ordinalpos=2288&amp;ncbi_uid=1737863&amp;link_uid=19196967">Citation</a><a class="dblinks" href="/pmc/articles/PMC2638736/" ref="ordinalpos=2288&amp;ncbi_uid=1737863&amp;link_uid=PMC2638736">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2289.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2995894/figure/F6/" ref="ordinalpos=2289&amp;ncbi_uid=2677516&amp;link_uid=PMC2995894" href="/pmc/articles/PMC2995894/"><img src="/pmc/articles/instance/2995894/bin/nihms-204451-f0006.gif" src-large="/pmc/articles/instance/2995894/bin/nihms-204451-f0006.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2995894/" ref="ordinalpos=2289&amp;ncbi_uid=2677516&amp;link_uid=PMC2995894" image-link="/pmc/articles/PMC2995894/figure/F6/" class="imagepopup">Fig. 6. Classical RAS <span class="highlight" style="background-color:">pathway</span> and novel effects of (pro)renin mediated by (pro)renin receptor.  From: The renin angiotensin system and the metabolic syndrome. </a></p><div class="supp"><p class="details"><div>Prorenin and renin effects via PRR include Ang II-dependent and independent actions. Ang II-dependent actions involve increased catalytic activity of the PRR-bound prorenin to generate Ang I from AGT. Ang II-independent action involves intracellular <span class="highlight" style="background-color:">signaling</span> that triggers activation of an extracellular signal-related protein kinase (ERK1/2) <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chih-Hong Wang, et al. Open Hypertens J. ;3:1-13.</div><p class="links"><a class="dblinks" href="/pubmed/21132096" ref="ordinalpos=2289&amp;ncbi_uid=2677516&amp;link_uid=21132096">Citation</a><a class="dblinks" href="/pmc/articles/PMC2995894/" ref="ordinalpos=2289&amp;ncbi_uid=2677516&amp;link_uid=PMC2995894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2290.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2771724/figure/F3/" ref="ordinalpos=2290&amp;ncbi_uid=2172258&amp;link_uid=PMC2771724" href="/pmc/articles/PMC2771724/"><img src="/pmc/articles/instance/2771724/bin/nihms45325f3.gif" src-large="/pmc/articles/instance/2771724/bin/nihms45325f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2771724/" ref="ordinalpos=2290&amp;ncbi_uid=2172258&amp;link_uid=PMC2771724" image-link="/pmc/articles/PMC2771724/figure/F3/" class="imagepopup">Figure 3.  From: Strain differences in the gating-disruptive effects of apomorphine: Relationship to gene expression in nucleus accumbens <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Dopamine Receptor <span class="highlight" style="background-color:">Signaling</span>: Genes differentially expressed in the NAC of LE vs. SD rats are significantly over represented in the dopamine-receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In LE rats, <i>COMT, DRD1a</i> and <i>ADCY5</i> mRNA levels in the NAC (red-filled shapes) are higher than in SD rats. In contrast, <i>MAO(b), ADCY4, PDE10A, PKA (Prkag2)</i> and PP1 (<i>Ppp1r3c</i>) (green-filled shapes) exhibit reduced expression in LE rats compared to SD rats. Adapted from Ingenuity <sup>®</sup> System's database of canonical pathways for Dopamine Receptor <span class="highlight" style="background-color:">Signaling</span>. Where the biochemical annotation in the Ingenuity canonical <span class="highlight" style="background-color:">pathway</span> differs from the gene symbol used by Affymetrix (see Table 1), the Affymetrix gene symbol is included in parentheses.</div></p></div><div class="aux"><div class="resc">Paul D. Shilling, et al. Biol Psychiatry. ;63(8):748-758.</div><p class="links"><a class="dblinks" href="/pubmed/18083141" ref="ordinalpos=2290&amp;ncbi_uid=2172258&amp;link_uid=18083141">Citation</a><a class="dblinks" href="/pmc/articles/PMC2771724/" ref="ordinalpos=2290&amp;ncbi_uid=2172258&amp;link_uid=PMC2771724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2291.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3196116/figure/F10/" ref="ordinalpos=2291&amp;ncbi_uid=3529949&amp;link_uid=PMC3196116" href="/pmc/articles/PMC3196116/"><img src="/pmc/articles/instance/3196116/bin/zbc0481183470010.gif" src-large="/pmc/articles/instance/3196116/bin/zbc0481183470010.jpg" alt="FIGURE 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3196116/" ref="ordinalpos=2291&amp;ncbi_uid=3529949&amp;link_uid=PMC3196116" image-link="/pmc/articles/PMC3196116/figure/F10/" class="imagepopup">FIGURE 10. From: Hck Is a Key Regulator of Gene Expression in Alternatively Activated Human Monocytes. </a></p><div class="supp"><p class="details"><div><b>Proposed model shows the requirement of the IL-4Rα-Jak2-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for IL-13-induced M2 gene expression in human monocytes/macrophages.</b> IL-13 stimulates the Src kinase Hck via the IL-4Rα-Jak2-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to the activation of the Hck-MKK3/6-p38 MAPK <span class="highlight" style="background-color:">signaling</span> cascade. Consequently Stat1 and Stat3 are maximally activated, and gene expression of <i>15-LO</i>, <i>mannose receptor</i>, <i>MAO-A</i>, and <i>CD36</i> is induced.</div></p></div><div class="aux"><div class="resc">Ashish Bhattacharjee, et al. J Biol Chem. 2011 October 21;286(42):36709-36723.</div><p class="links"><a class="dblinks" href="/pubmed/21878628" ref="ordinalpos=2291&amp;ncbi_uid=3529949&amp;link_uid=21878628">Citation</a><a class="dblinks" href="/pmc/articles/PMC3196116/" ref="ordinalpos=2291&amp;ncbi_uid=3529949&amp;link_uid=PMC3196116">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2292.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4202134/figure/F7/" ref="ordinalpos=2292&amp;ncbi_uid=6742867&amp;link_uid=PMC4202134" href="/pmc/articles/PMC4202134/"><img src="/pmc/articles/instance/4202134/bin/oncotarget-05-7431-g007.gif" src-large="/pmc/articles/instance/4202134/bin/oncotarget-05-7431-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4202134/" ref="ordinalpos=2292&amp;ncbi_uid=6742867&amp;link_uid=PMC4202134" image-link="/pmc/articles/PMC4202134/figure/F7/" class="imagepopup">Figure 7. Proposed apoptosis-inducing <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> triggered by DSeA and TRAIL in A375 cells.  From: Selenocysteine derivative overcomes TRAIL resistance in melanoma cells: evidence for ROS-dependent synergism and <span class="highlight" style="background-color:">signaling</span> crosstalk. </a></p><div class="supp"><p class="details"><div>DSeA first indues intracellular ROS overproduction in a rapid time. The accumulation of intracellular ROS induces DNA damage-mediated p53 activation and dephosphorlation of Akt, which is negative regulated by PTEN. Inactivation of Akt and phosphorylation of p53 co-regulate the expression of Bcl-2 family proteins, resulting in depletion of <i>Δψ<sub>m</sub></i> and release of apoptogenic factors from the mitochondria into cytosol. Furthermore, the activation of p53 increases the expression of death receptors, which accompany with mitochondrial <span class="highlight" style="background-color:">pathway</span> to promote TRAIL-induced apoptosis through activation of caspase cascade and PARP cleavage.</div></p></div><div class="aux"><div class="resc">Wenqiang Cao, et al. Oncotarget. 2014 September;5(17):7431-7445.</div><p class="links"><a class="dblinks" href="/pubmed/25277183" ref="ordinalpos=2292&amp;ncbi_uid=6742867&amp;link_uid=25277183">Citation</a><a class="dblinks" href="/pmc/articles/PMC4202134/" ref="ordinalpos=2292&amp;ncbi_uid=6742867&amp;link_uid=PMC4202134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2293.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766768/figure/fig10/" ref="ordinalpos=2293&amp;ncbi_uid=2491903&amp;link_uid=PMC2766768" href="/pmc/articles/PMC2766768/"><img src="/pmc/articles/instance/2766768/bin/toxscikfp150f10_lw.gif" src-large="/pmc/articles/instance/2766768/bin/toxscikfp150f10_lw.jpg" alt="FIG. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766768/" ref="ordinalpos=2293&amp;ncbi_uid=2491903&amp;link_uid=PMC2766768" image-link="/pmc/articles/PMC2766768/figure/fig10/" class="imagepopup">FIG. 10. From: Induction of Suppressors of Cytokine <span class="highlight" style="background-color:">Signaling</span> by the Trichothecene Deoxynivalenol in the Mouse. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">pathway</span> for DON-induced SOCS expression and potential downstream effects. Impairment of cytokine <span class="highlight" style="background-color:">signaling</span> and GH-IGF1 has been demonstrated in other inflammatory models (Denson et al., 2003; Lang et al., 2005; O'Sullivan et al., 2007; Rico-Bautista et al., 2006).</div></p></div><div class="aux"><div class="resc">Chidozie J. Amuzie, et al. Toxicol Sci. 2009 October;111(2):277-287.</div><p class="links"><a class="dblinks" href="/pubmed/19625342" ref="ordinalpos=2293&amp;ncbi_uid=2491903&amp;link_uid=19625342">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766768/" ref="ordinalpos=2293&amp;ncbi_uid=2491903&amp;link_uid=PMC2766768">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2294.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4367644/figure/f1/" ref="ordinalpos=2294&amp;ncbi_uid=7321519&amp;link_uid=PMC4367644" href="/pmc/articles/PMC4367644/"><img src="/pmc/articles/instance/4367644/bin/tlcr-02-01-014-f1.gif" src-large="/pmc/articles/instance/4367644/bin/tlcr-02-01-014-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4367644/" ref="ordinalpos=2294&amp;ncbi_uid=7321519&amp;link_uid=PMC4367644" image-link="/pmc/articles/PMC4367644/figure/f1/" class="imagepopup">Figure 1.  From: Inhibition of insulin-like growth factor receptor: end of a targeted therapy?. </a></p><div class="supp"><p class="details"><div>IGF-1R <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. IGF-1 is regulated by binding to insulin growth factor binding protein (IGFBP). Free IGF-1 can bind to IGF-1R, and activate downstream <span class="highlight" style="background-color:">signaling</span> through the insulin receptor substrate (IRS) to promote cell growth and proliferation (Ras/Raf/MEK <span class="highlight" style="background-color:">pathway</span>) and cell survival and cell cycle progression (PI3K/AKT/mTOR <span class="highlight" style="background-color:">pathway</span>)</div></p></div><div class="aux"><div class="resc">Rathi N. Pillai, et al. Transl Lung Cancer Res. 2013 February;2(1):14-22.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2294&amp;ncbi_uid=7321519&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4367644/" ref="ordinalpos=2294&amp;ncbi_uid=7321519&amp;link_uid=PMC4367644">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2295.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2504275/figure/Fig1/" ref="ordinalpos=2295&amp;ncbi_uid=1455859&amp;link_uid=PMC2504275" href="/pmc/articles/PMC2504275/"><img src="/pmc/articles/instance/2504275/bin/11420_2007_9072_Fig1_HTML.gif" src-large="/pmc/articles/instance/2504275/bin/11420_2007_9072_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2504275/" ref="ordinalpos=2295&amp;ncbi_uid=1455859&amp;link_uid=PMC2504275" image-link="/pmc/articles/PMC2504275/figure/Fig1/" class="imagepopup">Fig. 1.  From: Role of Wnt <span class="highlight" style="background-color:">Signaling</span> in Bone Remodeling and Repair. </a></p><div class="supp"><p class="details"><div>The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Wnt proteins bind to a receptor complex composed of members of the frizzled gene family (<i>Fzd</i>) and low-density lipoprotein (<i>LDL</i>) receptor-related proteins (<i>LRP</i>). Receptor binding inhibits glycogen synthase kinase-3<i>β</i> (GSK-3). In the absence of Wnt <span class="highlight" style="background-color:">signaling</span>, GSK-3 phosphorylates <i>β</i>-catenin and targets it to a degradation <span class="highlight" style="background-color:">pathway</span> mediated by the APC protein (<i>APC</i>). In the presence of Wnt <span class="highlight" style="background-color:">signaling</span>, <i>β</i>-catenin escapes phosphorylation and degradation and accumulates in the cytoplasm. Excess <i>β</i>-catenin enters the nucleus where it associates with the lymphoid enhancing factor (<i>Lef</i>)/T cell factor (<i>Tcf</i>) transcription factors to regulate gene expression. Secreted frizzled-related proteins (<i>Sfrps</i>) contain the ligand binding domain of frizzled but lack the transmembrane region; they can bind the Wnt ligands but cannot transduce the Wnt signal, and thus, effectively block Wnt <span class="highlight" style="background-color:">signaling</span>. Dickkopf (<i>DKK</i>) proteins are secreted factors which also block Wnt <span class="highlight" style="background-color:">signaling</span> by binding to the LRP receptors</div></p></div><div class="aux"><div class="resc">Paul S. Issack, et al. HSS J. 2008 February;4(1):66-70.</div><p class="links"><a class="dblinks" href="/pubmed/18751865" ref="ordinalpos=2295&amp;ncbi_uid=1455859&amp;link_uid=18751865">Citation</a><a class="dblinks" href="/pmc/articles/PMC2504275/" ref="ordinalpos=2295&amp;ncbi_uid=1455859&amp;link_uid=PMC2504275">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2296.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3737593/figure/F4/" ref="ordinalpos=2296&amp;ncbi_uid=5176337&amp;link_uid=PMC3737593" href="/pmc/articles/PMC3737593/"><img src="/pmc/articles/instance/3737593/bin/nihms497341f4.gif" src-large="/pmc/articles/instance/3737593/bin/nihms497341f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3737593/" ref="ordinalpos=2296&amp;ncbi_uid=5176337&amp;link_uid=PMC3737593" image-link="/pmc/articles/PMC3737593/figure/F4/" class="imagepopup">Figure 4.  From: Genetics of Pulmonary Arterial Hypertension. </a></p><div class="supp"><p class="details"><div>A delicate balance between BMPR2 <span class="highlight" style="background-color:">pathway</span> and TGFβ <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> modulates proproliferative and apop-totic forces. A <i>BMPR2</i> mutation may perturb this balance, resulting in disease expression.</div></p></div><div class="aux"><div class="resc">Eric D. Austin, et al. Semin Respir Crit Care Med. ;30(4):386-398.</div><p class="links"><a class="dblinks" href="/pubmed/19634078" ref="ordinalpos=2296&amp;ncbi_uid=5176337&amp;link_uid=19634078">Citation</a><a class="dblinks" href="/pmc/articles/PMC3737593/" ref="ordinalpos=2296&amp;ncbi_uid=5176337&amp;link_uid=PMC3737593">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2297.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3758122/figure/F6/" ref="ordinalpos=2297&amp;ncbi_uid=5241856&amp;link_uid=PMC3758122" href="/pmc/articles/PMC3758122/"><img src="/pmc/articles/instance/3758122/bin/nihms504769f6.gif" src-large="/pmc/articles/instance/3758122/bin/nihms504769f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3758122/" ref="ordinalpos=2297&amp;ncbi_uid=5241856&amp;link_uid=PMC3758122" image-link="/pmc/articles/PMC3758122/figure/F6/" class="imagepopup">Fig. 6. PI-3K/Akt/JNK <span class="highlight" style="background-color:">pathway</span> is implicated in arsenite-mediated cell proliferation.  From: PI3K/Akt/JNK/c-Jun <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> is a Mediator for Arsenite-Induced Cyclin D1 Expression and Cell Growth in Human Bronchial Epithelial Cells. </a></p><div class="supp"><p class="details"><div><b>a</b> and <b>b</b>. Beas-2B cells transfected with control vector, <i>Δp85</i> or DN-Akt were treated with arsenite for 60 min and Western blot was then performed. The number was the relative blots density phosphor-JNK or c-Jun compared with total JNK or c-Jun. <b>c</b>. Beas-2B cells transfected with control vector, or DN-Akt or siCyclin D1 were exposed to various doses of arsenite for 48 h and ATP assay was performed. <b>d.</b> Beas-2B cells (1 × 10<sup>3</sup>) were pre-treated with JNK inhibitor (25 μM) or LY98059 (50 μM) for 30 min and then exposed to arsenite for 48 hrs. Subsequently, ATP assay was performed. <b>e.</b> illustration of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to cyclin D1 expression and cell proliferation upon arsenite exposure.</div></p></div><div class="aux"><div class="resc">Jin Ding, et al. Curr Cancer Drug Targets. ;9(4):500-509.</div><p class="links"><a class="dblinks" href="/pubmed/19519318" ref="ordinalpos=2297&amp;ncbi_uid=5241856&amp;link_uid=19519318">Citation</a><a class="dblinks" href="/pmc/articles/PMC3758122/" ref="ordinalpos=2297&amp;ncbi_uid=5241856&amp;link_uid=PMC3758122">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2298.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436188/figure/F8/" ref="ordinalpos=2298&amp;ncbi_uid=5218675&amp;link_uid=PMC3436188" href="/pmc/articles/PMC3436188/"><img src="/pmc/articles/instance/3436188/bin/zbc0371221010008.gif" src-large="/pmc/articles/instance/3436188/bin/zbc0371221010008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436188/" ref="ordinalpos=2298&amp;ncbi_uid=5218675&amp;link_uid=PMC3436188" image-link="/pmc/articles/PMC3436188/figure/F8/" class="imagepopup">FIGURE 8. From: The Wnt Receptor Ryk Plays a Role in Mammalian Planar Cell Polarity <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the interaction between Ryk and Wnt <span class="highlight" style="background-color:">signaling</span> pathways.</b> Ryk binds to Wnts via its WIF domain and modulates both Wnt/β-catenin and β-catenin-independent <span class="highlight" style="background-color:">signaling</span> (14, 18, 22, 24). As was shown in this study, Ryk interacts with the PCP <span class="highlight" style="background-color:">pathway</span> protein Vangl2 and can activate downstream RhoA <span class="highlight" style="background-color:">signaling</span> (<i>boxed</i>). <i>CRD,</i> cysteine-rich domain; <i>Fzd</i>, Frizzled; <i>LRP5/6</i>, low density lipoprotein-related protein 5 or 6; <i>PDZ</i>, PSD-95/Dlg/ZO-1 homology; <i>WIF</i>, Wnt inhibitory factor.</div></p></div><div class="aux"><div class="resc">Maria L. Macheda, et al. J Biol Chem. 2012 August 24;287(35):29312-29323.</div><p class="links"><a class="dblinks" href="/pubmed/22773843" ref="ordinalpos=2298&amp;ncbi_uid=5218675&amp;link_uid=22773843">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436188/" ref="ordinalpos=2298&amp;ncbi_uid=5218675&amp;link_uid=PMC3436188">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2299.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2259457/figure/F6/" ref="ordinalpos=2299&amp;ncbi_uid=1256064&amp;link_uid=PMC2259457" href="/pmc/articles/PMC2259457/"><img src="/pmc/articles/instance/2259457/bin/neo1003_0287_fig006.gif" src-large="/pmc/articles/instance/2259457/bin/neo1003_0287_fig006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2259457/" ref="ordinalpos=2299&amp;ncbi_uid=1256064&amp;link_uid=PMC2259457" image-link="/pmc/articles/PMC2259457/figure/F6/" class="imagepopup">Figure 6.  From: Inhibition of JAK1, 2/STAT3 <span class="highlight" style="background-color:">Signaling</span> Induces Apoptosis, Cell Cycle Arrest, and Reduces Tumor Cell Invasion in Colorectal Cancer Cells. </a></p><div class="supp"><p class="details"><div>Model of the possible contributions of JAK1, 2/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in CRC development. Proteasome-mediated proteolysis promotes dephosphorylation of the JAK2 kinase and, consequently, negatively regulates JAK/STAT3 <span class="highlight" style="background-color:">signaling</span> in CRC. Other unknown mechanisms may also have been involved in this process. Furthermore, JAK1, 2/STAT3 <span class="highlight" style="background-color:">signaling</span> is implicated in many areas of tumor progression, including cell growth, survival, invasion, and migration by regulation of gene expressions, such as Bcl-2, p16<sup>ink4a</sup>, p21<sup>waf1/cip1</sup>, and p27<sup>kip1</sup>, E-cadherin, VEGF, and MMPs. Additionally, FAK may be a component of the JAK <span class="highlight" style="background-color:">pathway</span> and a downstream of JAK, which may exert its oncogenic effects through interaction with other signal transduction pathways.</div></p></div><div class="aux"><div class="resc">Hua Xiong, et al. Neoplasia. 2008 March;10(3):287-297.</div><p class="links"><a class="dblinks" href="/pubmed/18320073" ref="ordinalpos=2299&amp;ncbi_uid=1256064&amp;link_uid=18320073">Citation</a><a class="dblinks" href="/pmc/articles/PMC2259457/" ref="ordinalpos=2299&amp;ncbi_uid=1256064&amp;link_uid=PMC2259457">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2300.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4414306/figure/Fig4/" ref="ordinalpos=2300&amp;ncbi_uid=7465480&amp;link_uid=PMC4414306" href="/pmc/articles/PMC4414306/"><img src="/pmc/articles/instance/4414306/bin/12864_2015_1364_Fig4_HTML.gif" src-large="/pmc/articles/instance/4414306/bin/12864_2015_1364_Fig4_HTML.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4414306/" ref="ordinalpos=2300&amp;ncbi_uid=7465480&amp;link_uid=PMC4414306" image-link="/pmc/articles/PMC4414306/figure/Fig4/" class="imagepopup">Figure 4.  From: Messenger RNA sequencing and <span class="highlight" style="background-color:">pathway</span> analysis provide novel insights into the biological basis of chickens’ feed efficiency. </a></p><div class="supp"><p class="details"><div> <b>Growth hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis using Ingenuity Molecule Activity Predictor (MAP).</b> MAP predicts the upstream and downstream effects of the mapped genes on growth hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and hypothesizes the overall states of this <span class="highlight" style="background-color:">pathway</span>. Red and green symbols indicate genes up- and down-regulated in high-FE chickens, respectively. Orange and blue nodes indicate genes predicted to be activated or inhibited in the high-FE chickens, respectively. The color intensity is proportional to the degree of fold change. Edges between the nodes are colored orange when leading to the activation of downstream genes, blue when inhibiting downstream genes. Yellow edges indicate that the states of downstream genes are inconsistent with the prediction based on previous findings.</div></p></div><div class="aux"><div class="resc">Nan Zhou, et al. BMC Genomics. 2015;16(1):195.</div><p class="links"><a class="dblinks" href="/pubmed/25886891" ref="ordinalpos=2300&amp;ncbi_uid=7465480&amp;link_uid=25886891">Citation</a><a class="dblinks" href="/pmc/articles/PMC4414306/" ref="ordinalpos=2300&amp;ncbi_uid=7465480&amp;link_uid=PMC4414306">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="22" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="23" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="24" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:33:31-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB75D35430F9910000000000ABCFF1;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB75D35430F9910000000000ABCFF1" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>