
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="17" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A6F8153FEC8110000000000D52037" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="16" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="17" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="18" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="17" /></div>
            <div><h2 class="result_count">Results: 1601 to 1700 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1601.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3210339/figure/fig2/" ref="ordinalpos=1601&amp;ncbi_uid=4330719&amp;link_uid=PMC3210339" href="/pmc/articles/PMC3210339/"><img src="/pmc/articles/instance/3210339/bin/jcbfm2011116f2.gif" src-large="/pmc/articles/instance/3210339/bin/jcbfm2011116f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3210339/" ref="ordinalpos=1601&amp;ncbi_uid=4330719&amp;link_uid=PMC3210339" image-link="/pmc/articles/PMC3210339/figure/fig2/" class="imagepopup">Figure 2.  From: Multipotent mesenchymal stromal cells increase tPA expression and concomitantly decrease PAI-1 expression in astrocytes through the sonic hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after stroke (in vitro study). </a></p><div class="supp"><p class="details"><div>MSCs stimulated the Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and thereby regulates tPA and PAI-1 expressions in cultured astrocytes. qRT-PCR and western blot show the (<b>A</b>) mRNA and (<b>B</b>) protein levels of tPA in cultured astrocytes. MSC treatment and addition of exogenous rm-Shh increased the tPA level in cultured astrocytes. The Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitor cyclopamine blocked the tPA increase in astrocytes with MSC coculture and rm-Shh treatment. (<b>C</b>) Western blot shows PAI-1 protein level in cultured astrocytes. MSC treatment and addition of exogenous rm-Shh decreased PAI-1 expression in cultured astrocytes. The Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitor cyclopamine increased PAI-1 expression in astrocytes with MSC coculture and rm-Shh treatment. <sup>*</sup><i>P</i>&lt;0.05, <sup>**</sup><i>P</i>&lt;0.01, compared with normal astrocytes; ^<i>P</i>&lt;0.05, ^^<i>P</i>&lt;0.01 compared with OGD astrocytes; <sup>#</sup><i>P</i>&lt;0.05, <sup>##</sup><i>P</i>&lt;0.01, compared with MSC treatment; <sup>++</sup><i>P</i>&lt;0.01, compared with rm-Shh treatment. A: astrocytes; AM: astrocytes cocultured with MSCs; AMC: astrocytes cocultured with MSCs and cyclopamine added; AS: astrocytes treated with rm-Shh; ASC: rm-Shh treated astrocytes with cyclopamine added. MSC, mesenchymal stromal cell; PAI-1, plasminogen activator inhibitor 1; qRT-PCR, quantitative real-rime PCR; rm-Shh, recombinant mouse Shh; Shh, sonic hedgehog; tPA, tissue plasminogen activator.</div></p></div><div class="aux"><div class="resc">Hongqi Xin, et al. J Cereb Blood Flow Metab. 2011 November;31(11):2181-2188.</div><p class="links"><a class="dblinks" href="/pubmed/21829213" ref="ordinalpos=1601&amp;ncbi_uid=4330719&amp;link_uid=21829213">Citation</a><a class="dblinks" href="/pmc/articles/PMC3210339/" ref="ordinalpos=1601&amp;ncbi_uid=4330719&amp;link_uid=PMC3210339">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1602.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2758611/figure/F6/" ref="ordinalpos=1602&amp;ncbi_uid=2040336&amp;link_uid=PMC2758611" href="/pmc/articles/PMC2758611/"><img src="/pmc/articles/instance/2758611/bin/nihms149166f6.gif" src-large="/pmc/articles/instance/2758611/bin/nihms149166f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758611/" ref="ordinalpos=1602&amp;ncbi_uid=2040336&amp;link_uid=PMC2758611" image-link="/pmc/articles/PMC2758611/figure/F6/" class="imagepopup">Figure 6.  From: Progressive Tumor Formation in Mice with Conditional Deletion of TGF-? <span class="highlight" style="background-color:">Signaling</span> in Head and Neck Epithelia Is Associated with Activation of the PI3k/Akt <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A schematic presentation of the proposed TGF- <span class="highlight" style="background-color:">signaling</span> alteration that promotes HNSCC in mice. In normal cells, TGF- inhibits cell proliferation through the Smad-dependent <span class="highlight" style="background-color:">pathway</span>. It also induces apoptosis by repressing the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> results in tumor suppression. Decreased Tgfbr1 expression in <i>Tgfbr1</i> cKO mice leads to increased cell proliferation and cell survival through PTEN independent activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. DMBA treatment, which causes <i>H-ras</i> mutation as well as other mechanisms, may also play an important role in Akt activation. Decreased TGFBR1 can also increase TGF-1 in tumor stroma by as yet unidentified mechanisms, which leads to increased invasion, angiogenesis, and inflammation, as well as immune suppression through the paracrine effect of TGF-. In summary, inactivation of TGF- <span class="highlight" style="background-color:">signaling</span>, in the context of <i>ras</i> mutations and aberrant activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and accompanied by increased paracrine effect of TGF-, switches TGF- <span class="highlight" style="background-color:">signaling</span> from tumor suppression in normal cells to tumor promotion in head and neck carcinogenesis of <i>Tgfbr1</i> cKO mice.</div></p></div><div class="aux"><div class="resc">Yansong Bian, et al. Cancer Res. ;69(14):5918-5926.</div><p class="links"><a class="dblinks" href="/pubmed/19584284" ref="ordinalpos=1602&amp;ncbi_uid=2040336&amp;link_uid=19584284">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758611/" ref="ordinalpos=1602&amp;ncbi_uid=2040336&amp;link_uid=PMC2758611">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1603.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3400838/figure/F8/" ref="ordinalpos=1603&amp;ncbi_uid=5143251&amp;link_uid=PMC3400838" href="/pmc/articles/PMC3400838/"><img src="/pmc/articles/instance/3400838/bin/zmo0081266080008.gif" src-large="/pmc/articles/instance/3400838/bin/zmo0081266080008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3400838/" ref="ordinalpos=1603&amp;ncbi_uid=5143251&amp;link_uid=PMC3400838" image-link="/pmc/articles/PMC3400838/figure/F8/" class="imagepopup">Fig. 8. From: Ligand Bias at Metabotropic Glutamate 1a Receptors: Molecular Determinants That Distinguish ?-Arrestin-Mediated from G Protein-Mediated <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Biased <span class="highlight" style="background-color:">signaling</span> of mGlu1a receptors. Upon agonist (Ago) stimulation, mGlu1a receptors activate two independent <span class="highlight" style="background-color:">signaling</span> pathways. Agonists such as quisqualate and DHPG activate G protein-mediated <span class="highlight" style="background-color:">signaling</span> leading to the activation of phospholipase C, phosphoinositide hydrolysis, and protein kinase C (PKC) and resulting in transient increases in ERK phosphorylation. This <span class="highlight" style="background-color:">signaling</span> is abolished with mutation of the Thr188 residue in the agonist binding pocket and can be disrupted by the PLC inhibitor U73122. In contrast, glutarate and succinate activate the -arrestin-1 (Arr1)-dependent <span class="highlight" style="background-color:">pathway</span>, which involves receptor internalization, leads to sustained ERK phosphorylation, and is responsible for the cytoprotective actions of mGlu1a receptors. This <span class="highlight" style="background-color:">signaling</span> is abolished with mutation of Arg323 or Lys409 residues in the agonist binding pocket and can be disrupted through shRNA-mediated inhibition of -arrestin-1 expression. Glutamate, aspartate, and cysteate are unbiased ligands that activate both pathways. Amino acid mutations that block one of the <span class="highlight" style="background-color:">signaling</span> pathways do not prevent the actions of agonists at the other <span class="highlight" style="background-color:">pathway</span>. MEK, mitogen-activated protein kinase kinase.</div></p></div><div class="aux"><div class="resc">Andrew C. Emery, et al. Mol Pharmacol. 2012 August;82(2):291-301.</div><p class="links"><a class="dblinks" href="/pubmed/22584219" ref="ordinalpos=1603&amp;ncbi_uid=5143251&amp;link_uid=22584219">Citation</a><a class="dblinks" href="/pmc/articles/PMC3400838/" ref="ordinalpos=1603&amp;ncbi_uid=5143251&amp;link_uid=PMC3400838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1604.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4294354/figure/F2/" ref="ordinalpos=1604&amp;ncbi_uid=7085091&amp;link_uid=PMC4294354" href="/pmc/articles/PMC4294354/"><img src="/pmc/articles/instance/4294354/bin/oncotarget-05-10969-g002.gif" src-large="/pmc/articles/instance/4294354/bin/oncotarget-05-10969-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4294354/" ref="ordinalpos=1604&amp;ncbi_uid=7085091&amp;link_uid=PMC4294354" image-link="/pmc/articles/PMC4294354/figure/F2/" class="imagepopup">Figure 2. Interactions of NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in pancreatic cancer (PC).  From: Critical role of NF-B in pancreatic cancer. </a></p><div class="supp"><p class="details"><div>As indicated in the diagram, both classical and non-classical NF-B pathways interact with a number of <span class="highlight" style="background-color:">signaling</span> pathways to remain constitutively activated in pancreatic cancer cells and promotes tumorigenesis and metastasis.</div></p></div><div class="aux"><div class="resc">Lakshmi Prabhu, et al. Oncotarget. 2014 November;5(22):10969-10975.</div><p class="links"><a class="dblinks" href="/pubmed/25473891" ref="ordinalpos=1604&amp;ncbi_uid=7085091&amp;link_uid=25473891">Citation</a><a class="dblinks" href="/pmc/articles/PMC4294354/" ref="ordinalpos=1604&amp;ncbi_uid=7085091&amp;link_uid=PMC4294354">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1605.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4050444/figure/F9/" ref="ordinalpos=1605&amp;ncbi_uid=6235471&amp;link_uid=PMC4050444" href="/pmc/articles/PMC4050444/"><img src="/pmc/articles/instance/4050444/bin/2045-3701-4-24-9.gif" src-large="/pmc/articles/instance/4050444/bin/2045-3701-4-24-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4050444/" ref="ordinalpos=1605&amp;ncbi_uid=6235471&amp;link_uid=PMC4050444" image-link="/pmc/articles/PMC4050444/figure/F9/" class="imagepopup">Figure 9.  From: Adipose-derived stem cells cooperate with fractional carbon dioxide laser in antagonizing photoaging: a potential role of Wnt and ?-catenin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Effect of intervened expression of wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins during ADSC-CM treated photoaging HDF.</b> Down regulated expression of wnt3a and -catenin inhibited the effect of ADSC-CM treated photoaging HDF.</div></p></div><div class="aux"><div class="resc">Xiao Xu, et al. Cell Biosci. 2014;4:24-24.</div><p class="links"><a class="dblinks" href="/pubmed/24917925" ref="ordinalpos=1605&amp;ncbi_uid=6235471&amp;link_uid=24917925">Citation</a><a class="dblinks" href="/pmc/articles/PMC4050444/" ref="ordinalpos=1605&amp;ncbi_uid=6235471&amp;link_uid=PMC4050444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1606.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3913901/figure/F4/" ref="ordinalpos=1606&amp;ncbi_uid=5825134&amp;link_uid=PMC3913901" href="/pmc/articles/PMC3913901/"><img src="/pmc/articles/instance/3913901/bin/fimmu-05-00034-g004.gif" src-large="/pmc/articles/instance/3913901/bin/fimmu-05-00034-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3913901/" ref="ordinalpos=1606&amp;ncbi_uid=5825134&amp;link_uid=PMC3913901" image-link="/pmc/articles/PMC3913901/figure/F4/" class="imagepopup">Figure 4.  From: Role of the TWEAK-Fn14-cIAP1-NF-?B <span class="highlight" style="background-color:">Signaling</span> Axis in the Regulation of Myogenesis and Muscle Homeostasis. </a></p><div class="supp"><p class="details"><div><b>Roles of TNF, TWEAK, cIAP1/2, and A20, in myogenesis in the regulation of switching from canonical to non-canonical NF-B <span class="highlight" style="background-color:">signaling</span></b>. The transition from myoblast to myotube has been known for several years to involve a switch from canonical to non-canonical NF-B <span class="highlight" style="background-color:">signaling</span>. More recently, several of the key players and the mechanisms of action for this switch have been identified. The promotion of TNF-mediated canonical NF-B <span class="highlight" style="background-color:">signaling</span> involves the positive regulation by the E3 ubiquitin ligase cIAP1/2 and the negative regulation by the deubiquitinase A20. The non-canonical <span class="highlight" style="background-color:">pathway</span> is oppositely controlled: it is negatively regulated by cIAP1/2 and positively regulated by A20. Interestingly, the cytokine TWEAK (a pathological factor in muscle atrophy) has been recently shown to stimulate non-canonical NF-B <span class="highlight" style="background-color:">signaling</span> by removing the negative regulation of cIAP1/2 on that NF-B <span class="highlight" style="background-color:">pathway</span>. Physiological levels of TWEAK, in fact, promote myogenesis through the enhancement of myoblast fusion amongst other things. The non-canonical NF-B <span class="highlight" style="background-color:">pathway</span> has been shown previously to regulate mitochondrial biogenesis and to promote myotube maintenance, and recently shown to do so through the NF-B-mediated induction of the mitochondrial regulator PPAR- co-activator 1 (PGC-1). Figure adapted from Ref. (65).</div></p></div><div class="aux"><div class="resc">Emeka K. Enwere, et al. Front Immunol. 2014;5:34.</div><p class="links"><a class="dblinks" href="/pubmed/24550918" ref="ordinalpos=1606&amp;ncbi_uid=5825134&amp;link_uid=24550918">Citation</a><a class="dblinks" href="/pmc/articles/PMC3913901/" ref="ordinalpos=1606&amp;ncbi_uid=5825134&amp;link_uid=PMC3913901">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1607.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3350954/figure/fig3/" ref="ordinalpos=1607&amp;ncbi_uid=4011837&amp;link_uid=PMC3350954" href="/pmc/articles/PMC3350954/"><img src="/pmc/articles/instance/3350954/bin/ECAM2012-195397.003.gif" src-large="/pmc/articles/instance/3350954/bin/ECAM2012-195397.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3350954/" ref="ordinalpos=1607&amp;ncbi_uid=4011837&amp;link_uid=PMC3350954" image-link="/pmc/articles/PMC3350954/figure/fig3/" class="imagepopup">Figure 3.  From: Electroacupuncture Pretreatment as a Novel Avenue to Protect Brain against Ischemia and Reperfusion Injury. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of endocannabinoid system in neuroprotection. This graph shows the endocannabinoid system and the post-receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in its neuroprotective effect. (a) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of CB1R in neuroprotection; (b) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of CB2R in neuroprotection. AEA: N-arach-idonoyl-ethanolamine-anandamide; 2-AG: 2-arachidonylglycerol; CB1R: cannabinoid receptor type 1; PLC: phospholipase c; IP<sub>3</sub>: inositol triphosphate; ER: endoplasmic reticulum; DAG: diacylglycerol; FAN: factor associated with neutral sphingomyelinase; AC: adenylatecyclase; cAMP: cyclic adenosine monophosphate; PI3K: phosphatidylinositol 3-kinase; MAPK: mitogen-activated protein kinase; <i></i>PKC: epsilon protein kinase C; PKA: protein kinase A; AKT: protein kinase B.</div></p></div><div class="aux"><div class="resc">Xin Li, et al. Evid Based Complement Alternat Med. 2012;2012:195397.</div><p class="links"><a class="dblinks" href="/pubmed/22611425" ref="ordinalpos=1607&amp;ncbi_uid=4011837&amp;link_uid=22611425">Citation</a><a class="dblinks" href="/pmc/articles/PMC3350954/" ref="ordinalpos=1607&amp;ncbi_uid=4011837&amp;link_uid=PMC3350954">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1608.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2324067/figure/F1/" ref="ordinalpos=1608&amp;ncbi_uid=1352387&amp;link_uid=PMC2324067" href="/pmc/articles/PMC2324067/"><img src="/pmc/articles/instance/2324067/bin/nihms40586f1.gif" src-large="/pmc/articles/instance/2324067/bin/nihms40586f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2324067/" ref="ordinalpos=1608&amp;ncbi_uid=1352387&amp;link_uid=PMC2324067" image-link="/pmc/articles/PMC2324067/figure/F1/" class="imagepopup">Fig. 1. CXCL12/CXCR4 <span class="highlight" style="background-color:">signaling</span> pathways and their regulation.  From: Multiple roles of chemokine CXCL12 in the central nervous system: A migration from immunology to neurobiology. </a></p><div class="supp"><p class="details"><div>Binding of CXCL12 to CXCR4 activates multiple G-protein-mediated <span class="highlight" style="background-color:">signaling</span> pathways (such as PI3K, MAPKs and NFB) and induces the release of intracellular Ca<sup>2+</sup>. These <span class="highlight" style="background-color:">signaling</span> pathways are involved in cell proliferation, migration and survival or apoptosis. Activation of CXCR4 leads to GRK/-arrestin-mediated receptor desensitization and prevents CXCR4 from further interacting with G proteins. GRK/-arrestin promotes clathin-dependent endocytosis that is coupled to receptor trafficking, degradation and resensitization. -arrestin2 may modulate CXCR4-mediated activation of p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Meizhang Li, et al. Prog Neurobiol. ;84(2):116-131.</div><p class="links"><a class="dblinks" href="/pubmed/18177992" ref="ordinalpos=1608&amp;ncbi_uid=1352387&amp;link_uid=18177992">Citation</a><a class="dblinks" href="/pmc/articles/PMC2324067/" ref="ordinalpos=1608&amp;ncbi_uid=1352387&amp;link_uid=PMC2324067">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1609.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2709391/figure/F8/" ref="ordinalpos=1609&amp;ncbi_uid=1874625&amp;link_uid=PMC2709391" href="/pmc/articles/PMC2709391/"><img src="/pmc/articles/instance/2709391/bin/zbc0300980340008.gif" src-large="/pmc/articles/instance/2709391/bin/zbc0300980340008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2709391/" ref="ordinalpos=1609&amp;ncbi_uid=1874625&amp;link_uid=PMC2709391" image-link="/pmc/articles/PMC2709391/figure/F8/" class="imagepopup">FIGURE 8. From: Loss of Cardiolipin Leads to Longevity Defects That Are Alleviated by Alterations in Stress Response <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Perturbation of PKC and HOG <span class="highlight" style="background-color:">signaling</span> in <i>pgs1</i>.</b> Under hypertonic or cold stress conditions, extracellular osmolarity is increased, which causes an efflux of water from the cell, reducing the turgor pressure on the cell wall. Activation of the HOG <span class="highlight" style="background-color:">pathway</span> by increased osmolarity leads to <i>increased</i> intracellular turgor pressure to counteract the extracellular osmolarity. In contrast, low extracellular osmolarity resulting from heat or hypotonic stress causes an influx of water into the cell, which increases the turgor pressure. To withstand the increased turgor pressure, the PKC-Slt2 <span class="highlight" style="background-color:">pathway</span> is activated to induce cell wall synthesis (30, 5558). Loss of PG/CL may generate a signal (as yet unidentified) that is detected by regulators or components of the PKC-Slt2 <span class="highlight" style="background-color:">pathway</span>, resulting in down-regulation of the <span class="highlight" style="background-color:">pathway</span>. In this case, activation of the HOG <span class="highlight" style="background-color:">pathway</span> will exacerbate the cell wall defects in <i>pgs1</i> cells by further increasing the turgor pressure.</div></p></div><div class="aux"><div class="resc">Jingming Zhou, et al. J Biol Chem. 2009 July 3;284(27):18106-18114.</div><p class="links"><a class="dblinks" href="/pubmed/19401462" ref="ordinalpos=1609&amp;ncbi_uid=1874625&amp;link_uid=19401462">Citation</a><a class="dblinks" href="/pmc/articles/PMC2709391/" ref="ordinalpos=1609&amp;ncbi_uid=1874625&amp;link_uid=PMC2709391">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1610.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4018400/figure/pone-0097471-g004/" ref="ordinalpos=1610&amp;ncbi_uid=6141374&amp;link_uid=PMC4018400" href="/pmc/articles/PMC4018400/"><img src="/pmc/articles/instance/4018400/bin/pone.0097471.g004.gif" src-large="/pmc/articles/instance/4018400/bin/pone.0097471.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4018400/" ref="ordinalpos=1610&amp;ncbi_uid=6141374&amp;link_uid=PMC4018400" image-link="/pmc/articles/PMC4018400/figure/pone-0097471-g004/" class="imagepopup">Figure 4. Putative Toll-like receptor and RIG-I-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: De Novo Characterization of the Spleen Transcriptome of the Large Yellow Croaker (Pseudosciaena crocea) and Analysis of the Immune Relevant Genes and Pathways Involved in the Antiviral Response. </a></p><div class="supp"><p class="details"><div>The putative Toll-like receptor and RIG-I-like receptor <span class="highlight" style="background-color:">signaling</span> pathways of the large yellow croaker were constructed based on the knowledge of TLR <span class="highlight" style="background-color:">signaling</span> in mammalian species. However, most interactions need to be confirmed experimentally.</div></p></div><div class="aux"><div class="resc">Yinnan Mu, et al. PLoS One. 2014;9(5):e97471.</div><p class="links"><a class="dblinks" href="/pubmed/24820969" ref="ordinalpos=1610&amp;ncbi_uid=6141374&amp;link_uid=24820969">Citation</a><a class="dblinks" href="/pmc/articles/PMC4018400/" ref="ordinalpos=1610&amp;ncbi_uid=6141374&amp;link_uid=PMC4018400">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1611.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3694486/figure/fig5/" ref="ordinalpos=1611&amp;ncbi_uid=5079528&amp;link_uid=PMC3694486" href="/pmc/articles/PMC3694486/"><img src="/pmc/articles/instance/3694486/bin/IJE2013-786574.005.gif" src-large="/pmc/articles/instance/3694486/bin/IJE2013-786574.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3694486/" ref="ordinalpos=1611&amp;ncbi_uid=5079528&amp;link_uid=PMC3694486" image-link="/pmc/articles/PMC3694486/figure/fig5/" class="imagepopup">Figure 5.  From: Puerarin Suppress Apoptosis of Human Osteoblasts via ERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and estrogen receptor mediated the antiapoptotic effects of puerarin in hOBs. Cells were incubated with PD98059 (10<i></i>M) and/or ICI 182780 (10<i></i>M) for 3h prior to treatment with 10<sup>8</sup>M puerarin for 48h. The bars represented the mean  SD (<i>n</i> = 6). *<i>P</i> &lt; 0.05, as compared with the puerarin-treated cells; **<i>P</i> &lt; 0.05, as compared with the serum-free treated cells.</div></p></div><div class="aux"><div class="resc">Ling-juan Liu, et al. Int J Endocrinol. 2013;2013:786574.</div><p class="links"><a class="dblinks" href="/pubmed/23843790" ref="ordinalpos=1611&amp;ncbi_uid=5079528&amp;link_uid=23843790">Citation</a><a class="dblinks" href="/pmc/articles/PMC3694486/" ref="ordinalpos=1611&amp;ncbi_uid=5079528&amp;link_uid=PMC3694486">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1612.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1449085/figure/fig5/" ref="ordinalpos=1612&amp;ncbi_uid=607064&amp;link_uid=PMC1449085" href="/pmc/articles/PMC1449085/"><img src="/pmc/articles/instance/1449085/bin/1286f5.gif" src-large="/pmc/articles/instance/1449085/bin/1286f5.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1449085/" ref="ordinalpos=1612&amp;ncbi_uid=607064&amp;link_uid=PMC1449085" image-link="/pmc/articles/PMC1449085/figure/fig5/" class="imagepopup">F igure  5..  From: Convergent, RIC-8-Dependent G? <span class="highlight" style="background-color:">Signaling</span> Pathways in the Caenorhabditis elegans Synaptic <span class="highlight" style="background-color:">Signaling</span> Network. </a></p><div class="supp"><p class="details"><div>The G<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> converges with the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> downstream of DAG production, and hyperactivation of the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> in the absence of a neuronal G<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> severely disrupts coordinated locomotion. (A) The effects on locomotion rate caused by activating the G<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> can be largely blocked by knocking out the G<sub>q</sub> synaptic vesicle priming <span class="highlight" style="background-color:">pathway</span>. Comparison of the mean locomotion rates of strains is shown in an <i>egl-30(+)</i> background (first set of bars) and an <i>egl-30(ad810)</i> null mutant background (second set of bars). Arrows point out the <i>egl-30</i> null single mutant and double mutants containing the <i>egl-30</i> null mutation in combination with mutations that activate <i>acy-1</i> or PKA. Note that the <i>acy-1</i> gain-of-function mutation causes a small, but significant, improvement to the locomotion rate of the <i>egl-30</i> null; however, this amounts to only 2% of the <i>acy-1</i> gain-of-function single mutant (shown in the left set of bars). In comparison, the locomotion rate of the <i>egl-30</i> null is completely unaffected by the <i>kin-2</i> mutation (which hyperactivates protein kinase A), yet the paralysis of this double mutant can still be completely rescued with phorbol esters (last bar; yellow). The values of the two bars with asterisks are 0.024 body bends/minute for <i>egl-30(ad810)</i> and 0.025 body bends/minute for <i>egl-30(ad810); kin-2(ce179)</i>. Error bars represent the standard error of the mean for populations of 10 early larval stage animals. (B) Mutations or conditions that activate the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> only partially suppress the paralysis caused by a block of the G<sub>s</sub> <span class="highlight" style="background-color:">pathway</span>. Comparison of the mean locomotion rates of strains in an <i>acy-1(+)</i> background (first set of bars) and an <i>acy-1(pk1279)</i> null mutant background (second set of bars) is shown. Arrows point out the <i>acy-1</i> null single mutant and double mutants containing the <i>acy-1</i> null mutation in combination with various mutations that activate the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> or in combination with phorbol esters, which is the last bar. Note that all of the mutations that activate the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> and even phorbol esters, which strongly suppress the more paralyzed <i>egl-30</i> null, really only partially suppress the <i>acy-1</i> null. The photograph shows the tightly knotted posture in which all of these double mutants (but the not the <i>acy-1</i> null single mutant) spend about two-thirds of their time. Error bars represent the standard error of the mean for populations of 10 early larval stage animals. (C) The neuronal G<sub>s</sub> <span class="highlight" style="background-color:">pathway</span> is required for the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> (and the DAG that it produces) to properly exert its effects. Comparison of the mean locomotion rates of strains in an <i>acy-1(+)</i> background (first set of bars) and a muscle-rescued <i>acy-1(pk1279)</i> null mutant background (second set of bars) is shown. These animals are null for <i>acy-1</i> in their nervous system, but they contain the <i>ceEx108</i> [<i>myo-3::acy-1(+)</i> cDNA] transgene that rescues <i>acy-1</i> in muscle cells and also partially rescues the larval lethality caused by the <i>acy-1</i> null mutation. Arrows point out the muscle-rescued <i>acy-1</i> null single mutant and double mutants containing the muscle-rescued <i>acy-1</i> null mutation in combination with a strong <i>egl-30(gf)</i> mutation that activates the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> or in combination with phorbol esters. Note that both of these conditions only partially suppress paralysis of the <i>acy-1</i> null. The photographs show the tightly knotted posture that dominates as a consequence of activating the G<sub>q</sub> <span class="highlight" style="background-color:">pathway</span> in the absence of a neuronal G<sub>s</sub> <span class="highlight" style="background-color:">pathway</span>. Error bars represent the standard error of the mean for populations of 10 early larval stage animals. See also supplemental QuickTime movies for Figure 5 at http://www.genetics.org/supplemental/.</div></p></div><div class="aux"><div class="resc">Nicole K. Reynolds, et al. Genetics. 2005 February;169(2):651-670.</div><p class="links"><a class="dblinks" href="/pubmed/15489511" ref="ordinalpos=1612&amp;ncbi_uid=607064&amp;link_uid=15489511">Citation</a><a class="dblinks" href="/pmc/articles/PMC1449085/" ref="ordinalpos=1612&amp;ncbi_uid=607064&amp;link_uid=PMC1449085">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1613.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3348440/figure/F1/" ref="ordinalpos=1613&amp;ncbi_uid=4847931&amp;link_uid=PMC3348440" href="/pmc/articles/PMC3348440/"><img src="/pmc/articles/instance/3348440/bin/nihms365922f1.gif" src-large="/pmc/articles/instance/3348440/bin/nihms365922f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3348440/" ref="ordinalpos=1613&amp;ncbi_uid=4847931&amp;link_uid=PMC3348440" image-link="/pmc/articles/PMC3348440/figure/F1/" class="imagepopup">Figure 1.  From: Endocytic trafficking of neurotrophins in neural development. </a></p><div class="supp"><p class="details"><div>Target-derived NGF mediates axon growth and synapse assembly in sympathetic neurons via endocytic <span class="highlight" style="background-color:">signaling</span> mechanisms. (a) Retrograde NGF <span class="highlight" style="background-color:">signaling</span>, likely via axonal transport of TrkA-<span class="highlight" style="background-color:">signaling</span> endosomes, activates transcription factors and the expression of downstream target genes essential for long-term axon growth and target innervation. (b) Local NGF-TrkA <span class="highlight" style="background-color:">signaling</span> from an endocytic platform in axons promotes axon growth. NGF-TrkA complexes are endocytosed via a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that includes Phospholipase C-, calcineurin and dynamin1. Internalization of TrkA in distal axons is required for short-term axon growth, in a manner independent of transcriptional responses. (c) NGF-TrkA <span class="highlight" style="background-color:">signaling</span> endosomes are retrogradely transported long-distance from axon terminals to the distal dendrites of sympathetic neurons. In dendrites, NGF-TrkA endosomal complexes signal via the MEK/MAPK <span class="highlight" style="background-color:">pathway</span> to regulate the clustering of acetylcholine receptors (nAChRs) and pre-existing postsynaptic density components including MAGUK, GKAP and Shank. Endosomal TrkA <span class="highlight" style="background-color:">signaling</span> modulates the assembly of postsynaptic components, in part, by restricting the anti-synaptic actions of p75 <span class="highlight" style="background-color:">signaling</span> in dendrites.</div></p></div><div class="aux"><div class="resc">Maria Ascano, et al. Trends Cell Biol. ;22(5):266-273.</div><p class="links"><a class="dblinks" href="/pubmed/22444728" ref="ordinalpos=1613&amp;ncbi_uid=4847931&amp;link_uid=22444728">Citation</a><a class="dblinks" href="/pmc/articles/PMC3348440/" ref="ordinalpos=1613&amp;ncbi_uid=4847931&amp;link_uid=PMC3348440">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1614.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4025875/figure/fig2/" ref="ordinalpos=1614&amp;ncbi_uid=6196147&amp;link_uid=PMC4025875" href="/pmc/articles/PMC4025875/"><img src="/pmc/articles/instance/4025875/bin/ml-2012-00192k_0002.gif" src-large="/pmc/articles/instance/4025875/bin/ml-2012-00192k_0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4025875/" ref="ordinalpos=1614&amp;ncbi_uid=6196147&amp;link_uid=PMC4025875" image-link="/pmc/articles/PMC4025875/figure/fig2/" class="imagepopup">Figure 2.  From: Structure-Activity Relationships for Side Chain Oxysterol Agonists of the Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Activation of the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The relative expression of GLI1 and PTCH was determined in M2-10B4 cells following treatment with the standard Hh activating OHCs (<b>1</b>:<b>2</b>) or <b>35</b> and the Hh-specific inhibitor Cyc. Cells were pretreated with Cyc (5 M) for 2 h and then treated with OHC (5 M). mRNA expression was determined following 24 h of incubation. Data are from a representative experiment performed in triplicate.</div></p></div><div class="aux"><div class="resc">Audrey Corman, et al. ACS Med Chem Lett. 2012 October 11;3(10):828-833.</div><p class="links"><a class="dblinks" href="/pubmed/24900386" ref="ordinalpos=1614&amp;ncbi_uid=6196147&amp;link_uid=24900386">Citation</a><a class="dblinks" href="/pmc/articles/PMC4025875/" ref="ordinalpos=1614&amp;ncbi_uid=6196147&amp;link_uid=PMC4025875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1615.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC99976/figure/F7/" ref="ordinalpos=1615&amp;ncbi_uid=66711&amp;link_uid=PMC99976" href="/pmc/articles/PMC99976/"><img src="/pmc/articles/instance/99976/bin/mb2310138007.gif" src-large="/pmc/articles/instance/99976/bin/mb2310138007.jpg" alt="FIG. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC99976/" ref="ordinalpos=1615&amp;ncbi_uid=66711&amp;link_uid=PMC99976" image-link="/pmc/articles/PMC99976/figure/F7/" class="imagepopup">FIG. 7.  From: The Caenorhabditis elegans EGL-15 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Implicates  a DOS-Like Multisubstrate Adaptor Protein in Fibroblast  Growth Factor Signal Transduction. </a></p><div class="supp"><p class="details"><div>Model of <i>C. elegans</i> EGL-15/FGFR <span class="highlight" style="background-color:">signaling</span>. EGL-15 activation leads to the phosphorylation of SOC-1 on tyrosine 408; this event triggers PTP-2 recruitment and/or activation. In addition, EGL-15 activates the canonical SEM-5/SOS-1/RAS/MAPK <span class="highlight" style="background-color:">pathway</span>. Potential targets of the SOC-1/PTP-2 cassette and/or its role in modulating activity of the RAS/MAPK cascade are indicated by dashed lines. Precisely how activation of EGL-15 triggers SOC-1 and SEM-5 to participate in EGL-15 signal transduction remains unresolved (indicated by dashed lines from the receptor).</div></p></div><div class="aux"><div class="resc">Jennifer L. Schutzman, et al. Mol Cell Biol. 2001 December;21(23):8104-8116.</div><p class="links"><a class="dblinks" href="/pubmed/11689700" ref="ordinalpos=1615&amp;ncbi_uid=66711&amp;link_uid=11689700">Citation</a><a class="dblinks" href="/pmc/articles/PMC99976/" ref="ordinalpos=1615&amp;ncbi_uid=66711&amp;link_uid=PMC99976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1616.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807379/figure/F8/" ref="ordinalpos=1616&amp;ncbi_uid=2213947&amp;link_uid=PMC2807379" href="/pmc/articles/PMC2807379/"><img src="/pmc/articles/instance/2807379/bin/nihms166563f8.gif" src-large="/pmc/articles/instance/2807379/bin/nihms166563f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807379/" ref="ordinalpos=1616&amp;ncbi_uid=2213947&amp;link_uid=PMC2807379" image-link="/pmc/articles/PMC2807379/figure/F8/" class="imagepopup">Figure 8. The hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may be involved in renal cystogenesis.  From: Nephronophthisis: Disease Mechanisms of a Ciliopathy. </a></p><div class="supp"><p class="details"><div>In the hedgehog <span class="highlight" style="background-color:">pathway</span> of vertebrates, upon binding of the Shh ligand to the patched (Ptch1) receptor, repression of the smoothened (Smo) receptor to be inserted into the cilium membrane is relieved, and posttranslational modification of the Gli transcription factors within the cilium induces both their activator (Gli<sup>Act</sup>) and repressor (Gli<sup>R</sup>) functions. Several studies in mice have demonstrated that ciliary proteins are needed for hedgehog <span class="highlight" style="background-color:">signaling</span>. Recently, the related transcription factor Gli-similar 2 (<i>GLIS2</i>) was found to be mutated in NPHP type 7 (from: Huangfu &amp; Anderson <i>Development</i>, 2005)141.</div></p></div><div class="aux"><div class="resc">Friedhelm Hildebrandt, et al. J Am Soc Nephrol. ;20(1):23-35.</div><p class="links"><a class="dblinks" href="/pubmed/19118152" ref="ordinalpos=1616&amp;ncbi_uid=2213947&amp;link_uid=19118152">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807379/" ref="ordinalpos=1616&amp;ncbi_uid=2213947&amp;link_uid=PMC2807379">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1617.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4003375/figure/fig1/" ref="ordinalpos=1617&amp;ncbi_uid=6511001&amp;link_uid=PMC4003375" href="/pmc/articles/PMC4003375/"><img src="/pmc/articles/instance/4003375/bin/cmi201353f1.gif" src-large="/pmc/articles/instance/4003375/bin/cmi201353f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4003375/" ref="ordinalpos=1617&amp;ncbi_uid=6511001&amp;link_uid=PMC4003375" image-link="/pmc/articles/PMC4003375/figure/fig1/" class="imagepopup">Figure 1.  From: Nck, a missing adaptor between the B-cell receptor complex and the BCAP/PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Nck proteins as the newly identified adaptors between the BCR complex and BCAP/PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. After BCR crosslinking, Nck proteins bind to phosphorylated Tyr204 at the tail of Ig <i>via</i> their SH2 domains to recruit BCAP through their SH3 domains. Nck may control BCR <span class="highlight" style="background-color:">signaling</span> by activating the BCAP/PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and cross-talking with other pathways. BCAP, B-cell adaptor for PI3K; BCR, B-cell antigen receptor; Ig, immunoglobulin- PI3K, phosphatidylinositol 3-kinase; SH3, Src-homology 3.</div></p></div><div class="aux"><div class="resc">Chuen-Miin Leu. Cell Mol Immunol. 2014 March;11(2):120-122.</div><p class="links"><a class="dblinks" href="/pubmed/24270472" ref="ordinalpos=1617&amp;ncbi_uid=6511001&amp;link_uid=24270472">Citation</a><a class="dblinks" href="/pmc/articles/PMC4003375/" ref="ordinalpos=1617&amp;ncbi_uid=6511001&amp;link_uid=PMC4003375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1618.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507493/figure/F9A/" ref="ordinalpos=1618&amp;ncbi_uid=5431132&amp;link_uid=PMC3507493" href="/pmc/articles/PMC3507493/"><img src="/pmc/articles/instance/3507493/bin/cc-11-3972-g9A.gif" src-large="/pmc/articles/instance/3507493/bin/cc-11-3972-g9A.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507493/" ref="ordinalpos=1618&amp;ncbi_uid=5431132&amp;link_uid=PMC3507493" image-link="/pmc/articles/PMC3507493/figure/F9A/" class="imagepopup">Figure. From: The milk protein ?-casein functions as a tumor suppressor via activation of STAT1 <span class="highlight" style="background-color:">signaling</span>, effectively preventing breast cancer tumor growth and metastasis. </a></p><div class="supp"><p class="details"><div><b>Figure 9A.</b> -casein transcriptional profiling. Results from <span class="highlight" style="background-color:">pathway</span> analysis with ASSESS. We performed <span class="highlight" style="background-color:">pathway</span> analysis using ASSESS (analysis of sample set enrichment scores). Note that -casein upregulates gene transcripts associated with inflammatory interferon/STAT1 <span class="highlight" style="background-color:">signaling</span> (A) and decreases gene transcripts associated with stemness (B).</div></p></div><div class="aux"><div class="resc">Gloria Bonuccelli, et al. Cell Cycle. 2012 November 1;11(21):3972-3982.</div><p class="links"><a class="dblinks" href="/pubmed/23047602" ref="ordinalpos=1618&amp;ncbi_uid=5431132&amp;link_uid=23047602">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507493/" ref="ordinalpos=1618&amp;ncbi_uid=5431132&amp;link_uid=PMC3507493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1619.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507493/figure/F9B/" ref="ordinalpos=1619&amp;ncbi_uid=5431130&amp;link_uid=PMC3507493" href="/pmc/articles/PMC3507493/"><img src="/pmc/articles/instance/3507493/bin/cc-11-3972-g9B.gif" src-large="/pmc/articles/instance/3507493/bin/cc-11-3972-g9B.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507493/" ref="ordinalpos=1619&amp;ncbi_uid=5431130&amp;link_uid=PMC3507493" image-link="/pmc/articles/PMC3507493/figure/F9B/" class="imagepopup">Figure. From: The milk protein ?-casein functions as a tumor suppressor via activation of STAT1 <span class="highlight" style="background-color:">signaling</span>, effectively preventing breast cancer tumor growth and metastasis. </a></p><div class="supp"><p class="details"><div><b>Figure 9B.</b> -casein transcriptional profiling. Results from <span class="highlight" style="background-color:">pathway</span> analysis with ASSESS. We performed <span class="highlight" style="background-color:">pathway</span> analysis using ASSESS (analysis of sample set enrichment scores). Note that -casein upregulates gene transcripts associated with inflammatory interferon/STAT1 <span class="highlight" style="background-color:">signaling</span> (A) and decreases gene transcripts associated with stemness (B).</div></p></div><div class="aux"><div class="resc">Gloria Bonuccelli, et al. Cell Cycle. 2012 November 1;11(21):3972-3982.</div><p class="links"><a class="dblinks" href="/pubmed/23047602" ref="ordinalpos=1619&amp;ncbi_uid=5431130&amp;link_uid=23047602">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507493/" ref="ordinalpos=1619&amp;ncbi_uid=5431130&amp;link_uid=PMC3507493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1620.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4035334/figure/pone-0098113-g003/" ref="ordinalpos=1620&amp;ncbi_uid=6186994&amp;link_uid=PMC4035334" href="/pmc/articles/PMC4035334/"><img src="/pmc/articles/instance/4035334/bin/pone.0098113.g003.gif" src-large="/pmc/articles/instance/4035334/bin/pone.0098113.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4035334/" ref="ordinalpos=1620&amp;ncbi_uid=6186994&amp;link_uid=PMC4035334" image-link="/pmc/articles/PMC4035334/figure/pone-0098113-g003/" class="imagepopup">Figure 3. Role of PTEN in CRP-induced downregulation of Akt/mTOR/p70S6K <span class="highlight" style="background-color:">pathway</span>..  From: C-Reactive Protein Inhibits Survivin Expression via Akt/mTOR <span class="highlight" style="background-color:">Pathway</span> Downregulation by PTEN Expression in Cardiac Myocytes. </a></p><div class="supp"><p class="details"><div>(A) H9c2 cardiac myocytes were treated with indicated concentration of CRP in 0.5% FBS for 24 hours. Expression levels of PTEN were determined by immunoblot analysis and RT-PCR, respectively. (B) H9c2 cells transfected with 20 nM of PTEN siRNA were incubated 50 g/ml of CRP in 0.5% FBS for 24 hours and then treated 10% FBS for 1 hour. Cells were harvested and analyzed for PTEN, Akt, mTOR and p70S6K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by immunoblot assay. (C) PTEN siRNA-transfected cells were incubated 50 g/ml of CRP in 0.5% FBS for 24 hours and then treated 10% FBS for 24 hours. Protein levels of PTEN or survivin were analyzed by immunoblot assay. (D) H9c2 cells pretreated with PTEN inhibitor (BpV, 5 M) for 1 hour were incubated with 50 g/ml of CRP in 0.5% FBS for 24 hours and then treated 10% FBS for 1 hour. Cells were harvested and analyzed for PTEN, Akt, mTOR and p70S6K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by immunoblot assay. (E) PTEN inhibitor-pretreated cells were incubated 50 g/ml of CRP in 0.5% FBS for 24 hours and then treated 10% FBS for 24 hours. Protein levels of PTEN or survivin were analyzed by immunoblot assay.</div></p></div><div class="aux"><div class="resc">Beom Seob Lee, et al. PLoS One. 2014;9(5):e98113.</div><p class="links"><a class="dblinks" href="/pubmed/24866016" ref="ordinalpos=1620&amp;ncbi_uid=6186994&amp;link_uid=24866016">Citation</a><a class="dblinks" href="/pmc/articles/PMC4035334/" ref="ordinalpos=1620&amp;ncbi_uid=6186994&amp;link_uid=PMC4035334">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1621.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193496/figure/fig2/" ref="ordinalpos=1621&amp;ncbi_uid=4090100&amp;link_uid=PMC3193496" href="/pmc/articles/PMC3193496/"><img src="/pmc/articles/instance/3193496/bin/cr201186f2.gif" src-large="/pmc/articles/instance/3193496/bin/cr201186f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193496/" ref="ordinalpos=1621&amp;ncbi_uid=4090100&amp;link_uid=PMC3193496" image-link="/pmc/articles/PMC3193496/figure/fig2/" class="imagepopup">Figure 2.  From: Wnt <span class="highlight" style="background-color:">signaling</span> through T-cell factor phosphorylation. </a></p><div class="supp"><p class="details"><div>HIPK2 and NLK/LIT-1 in Wnt <span class="highlight" style="background-color:">signaling</span>. The comparison of several properties of HIPK2 and NLK indicates that these kinases might function in the same <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during anteroposterior axis specification in vertebrate embryos. Casein kinases 1 and 2 as well as GSK3 are also involved in TCF phosphorylation (see text).</div></p></div><div class="aux"><div class="resc">Sergei Y Sokol. Cell Res. 2011 July;21(7):1002-1012.</div><p class="links"><a class="dblinks" href="/pubmed/21606952" ref="ordinalpos=1621&amp;ncbi_uid=4090100&amp;link_uid=21606952">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193496/" ref="ordinalpos=1621&amp;ncbi_uid=4090100&amp;link_uid=PMC3193496">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1622.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3521217/figure/F4/" ref="ordinalpos=1622&amp;ncbi_uid=4448656&amp;link_uid=PMC3521217" href="/pmc/articles/PMC3521217/"><img src="/pmc/articles/instance/3521217/bin/1471-2164-13-S7-S25-4.gif" src-large="/pmc/articles/instance/3521217/bin/1471-2164-13-S7-S25-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3521217/" ref="ordinalpos=1622&amp;ncbi_uid=4448656&amp;link_uid=PMC3521217" image-link="/pmc/articles/PMC3521217/figure/F4/" class="imagepopup">Figure 4.  From: Discovering <span class="highlight" style="background-color:">pathway</span> cross-talks based on functional relations between pathways. </a></p><div class="supp"><p class="details"><div><b>Interaction between S1P4 and RHOA <span class="highlight" style="background-color:">signaling</span> pathways</b>. The pathways are integrated and visualized via the <span class="highlight" style="background-color:">Pathway</span> Integration Tool (PINT) [36]. The oval nodes denote the proteins, and the octagon nodes denote the protein complexes. The edges between nodes represent the interactions or reactions. The green and blue nodes correspond to the components in S1P4 and ROHA <span class="highlight" style="background-color:">signaling</span> pathways, respectively. The yellow node is the common component between pathways.</div></p></div><div class="aux"><div class="resc">Chia-Lang Hsu, et al. BMC Genomics. 2012;13(Suppl 7):S25-S25.</div><p class="links"><a class="dblinks" href="/pubmed/23282018" ref="ordinalpos=1622&amp;ncbi_uid=4448656&amp;link_uid=23282018">Citation</a><a class="dblinks" href="/pmc/articles/PMC3521217/" ref="ordinalpos=1622&amp;ncbi_uid=4448656&amp;link_uid=PMC3521217">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1623.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507479/figure/F1/" ref="ordinalpos=1623&amp;ncbi_uid=4401060&amp;link_uid=PMC3507479" href="/pmc/articles/PMC3507479/"><img src="/pmc/articles/instance/3507479/bin/cc-11-3909-g1.gif" src-large="/pmc/articles/instance/3507479/bin/cc-11-3909-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507479/" ref="ordinalpos=1623&amp;ncbi_uid=4401060&amp;link_uid=PMC3507479" image-link="/pmc/articles/PMC3507479/figure/F1/" class="imagepopup">Figure. From: An expanded role of the tumor suppressor TSC1 in T cell tolerance. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> TSC1/2-mTOR <span class="highlight" style="background-color:">signaling</span> in T cell activation and tolerance. TCR engagement activates PLC1, which hydrolyzes PIP2 to generate DAG and IP3, two important second messengers that trigger the activation of multiple signal cascades. IP3 triggers Ca<sup>2+</sup> influx, which, in turn, induces the activation of the calcineurin-NFAT <span class="highlight" style="background-color:">pathway</span>. DAG associates and activates RasGRP1 and PKC, resulting in the activation of the Ras-Erk1/2-AP1 and IKK-NFB pathways, respectively. CD28 provides costimulation and enhances PI3K-Akt activation. The Ca<sup>2+</sup>-NFAT <span class="highlight" style="background-color:">pathway</span> alone induces T cell anergy by increasing expression of anergy-promoting molecules. This <span class="highlight" style="background-color:">pathway</span>, in concert with DAG-mediated pathways, induces T cell activation. DGKs convert DAG to PA and, thus, inhibit T cell activation. In T cells, the RasGRP1-Ras-Erk1/2 <span class="highlight" style="background-color:">pathway</span> as well as the PI3K-Akt <span class="highlight" style="background-color:">pathway</span>, is important for mTORC1 and mTORC2 <span class="highlight" style="background-color:">signaling</span>. TSC1 inhibits mTORC1, but promotes mTORC2 <span class="highlight" style="background-color:">signaling</span> and is important for T cell survival, quiescence and anergy.</div></p></div><div class="aux"><div class="resc">Xiao-Ping Zhong. Cell Cycle. 2012 November 1;11(21):3909-3910.</div><p class="links"><a class="dblinks" href="/pubmed/23032259" ref="ordinalpos=1623&amp;ncbi_uid=4401060&amp;link_uid=23032259">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507479/" ref="ordinalpos=1623&amp;ncbi_uid=4401060&amp;link_uid=PMC3507479">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1624.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2896193/figure/F3/" ref="ordinalpos=1624&amp;ncbi_uid=2389669&amp;link_uid=PMC2896193" href="/pmc/articles/PMC2896193/"><img src="/pmc/articles/instance/2896193/bin/gkq424f3.gif" src-large="/pmc/articles/instance/2896193/bin/gkq424f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2896193/" ref="ordinalpos=1624&amp;ncbi_uid=2389669&amp;link_uid=PMC2896193" image-link="/pmc/articles/PMC2896193/figure/F3/" class="imagepopup">Figure 3. From: Discovering causal <span class="highlight" style="background-color:">signaling</span> pathways through gene-expression patterns. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> crosstalk demonstrated by overlapping signature genes. The percent overlap between signature genes between all pairs of pathways is displayed as a heat map with higher overlap suggesting greater crosstalk between the respective <span class="highlight" style="background-color:">signaling</span> pathways. <span class="highlight" style="background-color:">Pathway</span> similarity is calculated as the negative log of the percent overlap and similar pathways are grouped using hierarchical clustering. Two major clusters are realized, separating pathways involved in immune response (JAK-STAT, TLR, IL-1 and TNF-) from pathways controlling cell cycle.</div></p></div><div class="aux"><div class="resc">Jignesh R. Parikh, et al. Nucleic Acids Res. 2010 July 1;38(Web Server issue):W109-W117.</div><p class="links"><a class="dblinks" href="/pubmed/20494976" ref="ordinalpos=1624&amp;ncbi_uid=2389669&amp;link_uid=20494976">Citation</a><a class="dblinks" href="/pmc/articles/PMC2896193/" ref="ordinalpos=1624&amp;ncbi_uid=2389669&amp;link_uid=PMC2896193">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1625.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412517/figure/F1/" ref="ordinalpos=1625&amp;ncbi_uid=4163805&amp;link_uid=PMC3412517" href="/pmc/articles/PMC3412517/"><img src="/pmc/articles/instance/3412517/bin/nihms390627f1.gif" src-large="/pmc/articles/instance/3412517/bin/nihms390627f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412517/" ref="ordinalpos=1625&amp;ncbi_uid=4163805&amp;link_uid=PMC3412517" image-link="/pmc/articles/PMC3412517/figure/F1/" class="imagepopup">Figure 1. Methods of Blocking the HGF/Met <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: Targeting the HGF/Met <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Cancer. </a></p><div class="supp"><p class="details"><div>Oncogenic <span class="highlight" style="background-color:">signaling</span> by cellular Met (in blue) and its natural ligands (in yellow) can be antagonized by several distinct inhibitor types (in violet). The intracellular tyrosine kinase (TK) domain and carboxyl terminal docking sites (Y1349 and Y1356) of Met are noted. <span class="highlight" style="background-color:">Pathway</span> inhibitors can be divided broadly into two subtypes (1) biological antagonists of HGF activation and HGF/Met binding, and (2) Met TK inhibitors (TKIs). Biological agents acting outside of the target cell (1) include anti-HGF and anti-Met mAbs, soluble Met ectodomain constructs and truncated HGF isoforms (all of which interfere with HGF/Met binding) and uncleavable forms of HGF (which competitively displace pro-HGF from its activators). Agents acting within the cell (2) include TKIs that competitively displace ATP from its TK domain binding site (TKI-1) and those which bind outside of the ATP binding pocket and inhibit Met TK activation allosterically (TKI-2).</div></p></div><div class="aux"><div class="resc">Fabiola Cecchi, et al. Eur J Cancer. ;46(7):1260-1270.</div><p class="links"><a class="dblinks" href="/pubmed/20303741" ref="ordinalpos=1625&amp;ncbi_uid=4163805&amp;link_uid=20303741">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412517/" ref="ordinalpos=1625&amp;ncbi_uid=4163805&amp;link_uid=PMC3412517">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1626.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3727419/figure/F3/" ref="ordinalpos=1626&amp;ncbi_uid=6445109&amp;link_uid=PMC3727419" href="/pmc/articles/PMC3727419/"><img src="/pmc/articles/instance/3727419/bin/nihms-486334-f0003.gif" src-large="/pmc/articles/instance/3727419/bin/nihms-486334-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3727419/" ref="ordinalpos=1626&amp;ncbi_uid=6445109&amp;link_uid=PMC3727419" image-link="/pmc/articles/PMC3727419/figure/F3/" class="imagepopup">Fig. 3. Down- and upregulated proteins in ARPE-19 cells after Rotenone or Methyl Viologen treatment.  From: Retinal Pigment Epithelium (RPE) exosomes contain <span class="highlight" style="background-color:">signaling</span> phosphoproteins affected by oxidative stress. </a></p><div class="supp"><p class="details"><div>Down- or up-regulated <span class="highlight" style="background-color:">signaling</span> proteins in treated ARPE-19 cells, compared to control (Mann-Whitney test, p  0.1). A. List of affected proteins; B. Proteins grouped in <span class="highlight" style="background-color:">signaling</span> pathways; C. <span class="highlight" style="background-color:">Pathway</span> description.</div></p></div><div class="aux"><div class="resc">Lucia Biasutto, et al. Exp Cell Res. ;319(13):2113-2123.</div><p class="links"><a class="dblinks" href="/pubmed/23669273" ref="ordinalpos=1626&amp;ncbi_uid=6445109&amp;link_uid=23669273">Citation</a><a class="dblinks" href="/pmc/articles/PMC3727419/" ref="ordinalpos=1626&amp;ncbi_uid=6445109&amp;link_uid=PMC3727419">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1627.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4260734/figure/fig7/" ref="ordinalpos=1627&amp;ncbi_uid=6963876&amp;link_uid=PMC4260734" href="/pmc/articles/PMC4260734/"><img src="/pmc/articles/instance/4260734/bin/cddis2014476f7.gif" src-large="/pmc/articles/instance/4260734/bin/cddis2014476f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4260734/" ref="ordinalpos=1627&amp;ncbi_uid=6963876&amp;link_uid=PMC4260734" image-link="/pmc/articles/PMC4260734/figure/fig7/" class="imagepopup">Figure 7.  From: The Hippo <span class="highlight" style="background-color:">pathway</span> is controlled by Angiotensin II <span class="highlight" style="background-color:">signaling</span> and its reactivation induces apoptosis in podocytes. </a></p><div class="supp"><p class="details"><div>Proposed model of the regulation and effects of hippo <span class="highlight" style="background-color:">pathway</span> in podocytes. Podocytes display a highly organized cytoskeleton, which is important for the complex structure of the cells. Actin dynamics or the disruption of the actin cytoskeleton impairs the complex structure of these cells, leading to a loss of physiological functions. The disruption of the actin cytoskeleton activates Hippo <span class="highlight" style="background-color:">signaling</span> by activating LATS kinase, which changes the predominant nuclear YAP signal to a more cytoplasmic distribution. Reactivation of Hippo <span class="highlight" style="background-color:">signaling</span> by activating LATS kinase is sufficient to induce apoptosis in podocytes, but not in HEK293T cells</div></p></div><div class="aux"><div class="resc">D O Wennmann, et al. Cell Death Dis. 2014 November;5(11):e1519.</div><p class="links"><a class="dblinks" href="/pubmed/25393475" ref="ordinalpos=1627&amp;ncbi_uid=6963876&amp;link_uid=25393475">Citation</a><a class="dblinks" href="/pmc/articles/PMC4260734/" ref="ordinalpos=1627&amp;ncbi_uid=6963876&amp;link_uid=PMC4260734">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1628.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2896552/figure/F2/" ref="ordinalpos=1628&amp;ncbi_uid=2682424&amp;link_uid=PMC2896552" href="/pmc/articles/PMC2896552/"><img src="/pmc/articles/instance/2896552/bin/ukmss-30602-f0002.gif" src-large="/pmc/articles/instance/2896552/bin/ukmss-30602-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2896552/" ref="ordinalpos=1628&amp;ncbi_uid=2682424&amp;link_uid=PMC2896552" image-link="/pmc/articles/PMC2896552/figure/F2/" class="imagepopup">Figure 2.  From: The Phosphoinositide 3-Kinase Inhibitor PI-103 Downregulates Choline Kinase ? Leading to Phosphocholine and Total Choline Decrease Detected by Magnetic Resonance Spectroscopy. </a></p><div class="supp"><p class="details"><div>Molecular effects of inhibition with PI-103 on the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PTEN null PC3 prostate and <i>PIK3CA</i> mutant HCT116 colon carcinoma cell lines. A, representative Western blots showing decreases in molecular biomarkers in the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PC3 prostate cancer cells following treatment with PI-103 at 1, 2.5, and 5 GI<sub>50</sub>, 24 h posttreatment. B, representative Western blots showing decreases in molecular biomarkers in the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PC3 prostate cancer cells at selected time points posttreatment with PI-103 (5 GI<sub>50</sub>). C, representative Western blots showing inhibition of molecular biomarkers in the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and changes in protein expression levels of enzymes involved in lipid metabolism in HCT116 colon cancer cells following treatment with PI-103 (5 GI<sub>50</sub>).</div></p></div><div class="aux"><div class="resc">Nada M. S. Al-Saffar, et al. Cancer Res. ;70(13):5507-5517.</div><p class="links"><a class="dblinks" href="/pubmed/20551061" ref="ordinalpos=1628&amp;ncbi_uid=2682424&amp;link_uid=20551061">Citation</a><a class="dblinks" href="/pmc/articles/PMC2896552/" ref="ordinalpos=1628&amp;ncbi_uid=2682424&amp;link_uid=PMC2896552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1629.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3661108/figure/F2/" ref="ordinalpos=1629&amp;ncbi_uid=5027630&amp;link_uid=PMC3661108" href="/pmc/articles/PMC3661108/"><img src="/pmc/articles/instance/3661108/bin/adip-2-61-g2.gif" src-large="/pmc/articles/instance/3661108/bin/adip-2-61-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3661108/" ref="ordinalpos=1629&amp;ncbi_uid=5027630&amp;link_uid=PMC3661108" image-link="/pmc/articles/PMC3661108/figure/F2/" class="imagepopup">Figure. From: The TLR family protein RP105/MD-1 complex. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Schematic diagram of TLR4/MD-2 and RP105/MD-1-mediated inflammatory mechanisms in adipose tissue inflammation. Black colored letters and arrows indicate the palmitate and TLR4/MD-2-mediated inflammatory <span class="highlight" style="background-color:">pathway</span>. Red colored letters and arrows indicate the RP105/MD-1-mediated inflammatory <span class="highlight" style="background-color:">pathway</span>. Precise RP105-mediated <span class="highlight" style="background-color:">signaling</span> in adipose tissue inflammation remains unclear, but <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> other than NFB and JNK may be responsible for RP105-dependnet adipose tissue inflammation.</div></p></div><div class="aux"><div class="resc">Yoshinori Nagai, et al. Adipocyte. 2013 April 1;2(2):61-66.</div><p class="links"><a class="dblinks" href="/pubmed/23805400" ref="ordinalpos=1629&amp;ncbi_uid=5027630&amp;link_uid=23805400">Citation</a><a class="dblinks" href="/pmc/articles/PMC3661108/" ref="ordinalpos=1629&amp;ncbi_uid=5027630&amp;link_uid=PMC3661108">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1630.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4312598/figure/Fig7/" ref="ordinalpos=1630&amp;ncbi_uid=7118387&amp;link_uid=PMC4312598" href="/pmc/articles/PMC4312598/"><img src="/pmc/articles/instance/4312598/bin/12870_2014_370_Fig7_HTML.gif" src-large="/pmc/articles/instance/4312598/bin/12870_2014_370_Fig7_HTML.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4312598/" ref="ordinalpos=1630&amp;ncbi_uid=7118387&amp;link_uid=PMC4312598" image-link="/pmc/articles/PMC4312598/figure/Fig7/" class="imagepopup">Figure 7.  From: Transcriptomic analysis of the late stages of grapevine (Vitis vinifera cv. Cabernet Sauvignon) berry ripening reveals significant induction of ethylene <span class="highlight" style="background-color:">signaling</span> and flavor pathways in the skin. </a></p><div class="supp"><p class="details"><div> <b>The transcript abundance of key components of the ethylene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Data are means  SE; n = 6.</div></p></div><div class="aux"><div class="resc">Grant R Cramer, et al. BMC Plant Biol. 2014;14(1):370.</div><p class="links"><a class="dblinks" href="/pubmed/25524329" ref="ordinalpos=1630&amp;ncbi_uid=7118387&amp;link_uid=25524329">Citation</a><a class="dblinks" href="/pmc/articles/PMC4312598/" ref="ordinalpos=1630&amp;ncbi_uid=7118387&amp;link_uid=PMC4312598">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1631.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4029079/figure/F3/" ref="ordinalpos=1631&amp;ncbi_uid=6162572&amp;link_uid=PMC4029079" href="/pmc/articles/PMC4029079/"><img src="/pmc/articles/instance/4029079/bin/1471-2164-15-182-3.gif" src-large="/pmc/articles/instance/4029079/bin/1471-2164-15-182-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4029079/" ref="ordinalpos=1631&amp;ncbi_uid=6162572&amp;link_uid=PMC4029079" image-link="/pmc/articles/PMC4029079/figure/F3/" class="imagepopup">Figure 3.  From: Transcriptomic analysis of differentially expressed genes in the Ras1CA-overexpressed and wildtype posterior silk glands. </a></p><div class="supp"><p class="details"><div><b>DEGs distribution in pathways in cancer, insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> Showing the number of DEGs. <b>(B) </b>Showing the common DEGs.</div></p></div><div class="aux"><div class="resc">Li Ma, et al. BMC Genomics. 2014;15:182-182.</div><p class="links"><a class="dblinks" href="/pubmed/24606580" ref="ordinalpos=1631&amp;ncbi_uid=6162572&amp;link_uid=24606580">Citation</a><a class="dblinks" href="/pmc/articles/PMC4029079/" ref="ordinalpos=1631&amp;ncbi_uid=6162572&amp;link_uid=PMC4029079">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1632.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2648682/figure/F1/" ref="ordinalpos=1632&amp;ncbi_uid=1717223&amp;link_uid=PMC2648682" href="/pmc/articles/PMC2648682/"><img src="/pmc/articles/instance/2648682/bin/JCI0938661.f1.gif" src-large="/pmc/articles/instance/2648682/bin/JCI0938661.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2648682/" ref="ordinalpos=1632&amp;ncbi_uid=1717223&amp;link_uid=PMC2648682" image-link="/pmc/articles/PMC2648682/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Pathway</span> for inhibition of IL-2 <span class="highlight" style="background-color:">signaling</span> in Tregs by PD-1..  From: PD-1 tempers Tregs in chronic HCV infection. </a></p><div class="supp"><p class="details"><div>PD-1 <span class="highlight" style="background-color:">signaling</span> leads to dephosphorylation of molecules in the T cell activation cascade previously phosphorylated in response to TCR ligation. In this issue of the <i>JCI</i>, Franceschini et al. show that PD-1 <span class="highlight" style="background-color:">signaling</span> leads to inhibition of IL-2 signals in Tregs (5). (i) IL-2 <span class="highlight" style="background-color:">signaling</span> via the IL-2 receptor (IL-2R) induces STAT-5 phosphorylation (pSTAT-5). (ii) Dimerization and nuclear translocation of phosphorylated STAT-5 leads to increased proliferation of Tregs but also to PD-1 upregulation. (iii) Increased surface PD-1 expression inhibits STAT-5 phosphorylation (5), possibly via SHP2. ITSM, immunoreceptor tyrosine-based switch motif.    </div></p></div><div class="aux"><div class="resc">Henry Radziewicz, et al. J Clin Invest. 2009 March 2;119(3):450-453.</div><p class="links"><a class="dblinks" href="/pubmed/19306502" ref="ordinalpos=1632&amp;ncbi_uid=1717223&amp;link_uid=19306502">Citation</a><a class="dblinks" href="/pmc/articles/PMC2648682/" ref="ordinalpos=1632&amp;ncbi_uid=1717223&amp;link_uid=PMC2648682">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1633.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4194559/figure/F1/" ref="ordinalpos=1633&amp;ncbi_uid=6724301&amp;link_uid=PMC4194559" href="/pmc/articles/PMC4194559/"><img src="/pmc/articles/instance/4194559/bin/WJG-20-13756-g001.gif" src-large="/pmc/articles/instance/4194559/bin/WJG-20-13756-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4194559/" ref="ordinalpos=1633&amp;ncbi_uid=6724301&amp;link_uid=PMC4194559" image-link="/pmc/articles/PMC4194559/figure/F1/" class="imagepopup">Figure 1.  From: Rho/ROCK <span class="highlight" style="background-color:">signaling</span> in motility and metastasis of gastric cancer. </a></p><div class="supp"><p class="details"><div>Presents an overview of the Rho/Rho-associated protein kinases <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MYPT1: Myosin phosphatase 1; MLC: Myosin light chain; ERM: Ezrin/radixin/moesin; ROCK: Rho-associated protein kinases; LIMK: LIM kinase.</div></p></div><div class="aux"><div class="resc">Tasuku Matsuoka, et al. World J Gastroenterol. 2014 October 14;20(38):13756-13766.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1633&amp;ncbi_uid=6724301&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4194559/" ref="ordinalpos=1633&amp;ncbi_uid=6724301&amp;link_uid=PMC4194559">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1634.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4129010/figure/fig03/" ref="ordinalpos=1634&amp;ncbi_uid=6489688&amp;link_uid=PMC4129010" href="/pmc/articles/PMC4129010/"><img src="/pmc/articles/instance/4129010/bin/ijcep0007-3984-f3.gif" src-large="/pmc/articles/instance/4129010/bin/ijcep0007-3984-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4129010/" ref="ordinalpos=1634&amp;ncbi_uid=6489688&amp;link_uid=PMC4129010" image-link="/pmc/articles/PMC4129010/figure/fig03/" class="imagepopup">Figure 3.  From: Knockdown of Aurora-B inhibits osteosarcoma cell invasion and migration via modulating PI3K/Akt/NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Inhibiting Aurora-B suppressed the expression of p-PI3K, PI3K, p-Akt and NF-B proteins in U-2 OS and HOS cells. The OS cells were infected with LV-AURKB or LV-NC for 8 h, cultured for 72 h, and then lysed. The protein was quantified, separated with 8% SDS-PAGE and assayed by Western-Blot. This is a representative image of the six experiments shown for each groups. It show that with the silencing of Aurora-B, the p-PI3K, PI3K, p-Akt and NF-B proteins were blockaded in varying degrees, which indicated that Aurora-B may down-regulate the PI3K/Akt/NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Liang Bo Zhu, et al. Int J Clin Exp Pathol. 2014;7(7):3984-3991.</div><p class="links"><a class="dblinks" href="/pubmed/25120775" ref="ordinalpos=1634&amp;ncbi_uid=6489688&amp;link_uid=25120775">Citation</a><a class="dblinks" href="/pmc/articles/PMC4129010/" ref="ordinalpos=1634&amp;ncbi_uid=6489688&amp;link_uid=PMC4129010">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1635.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2715000/figure/bire-79-05-01-f08/" ref="ordinalpos=1635&amp;ncbi_uid=1959634&amp;link_uid=PMC2715000" href="/pmc/articles/PMC2715000/"><img src="/pmc/articles/instance/2715000/bin/bire-79-05-01-f08.gif" src-large="/pmc/articles/instance/2715000/bin/bire-79-05-01-f08.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2715000/" ref="ordinalpos=1635&amp;ncbi_uid=1959634&amp;link_uid=PMC2715000" image-link="/pmc/articles/PMC2715000/figure/bire-79-05-01-f08/" class="imagepopup">FIG. 8. From: Role of Caspase 2 in Apoptotic <span class="highlight" style="background-color:">Signaling</span> in Primate and Murine Germ Cells. </a></p><div class="supp"><p class="details"><div>Summary of caspase 2-mediated <span class="highlight" style="background-color:">signaling</span> cascades in heat-induced germ cell apoptosis. Activation of caspase 2 after heat stress results in activation of MAPK14 and NOS2 induction. It is likely that activation of such <span class="highlight" style="background-color:">signaling</span> cascade perturbs the BAX/BCL2 rheostat in the mitochondria [8, 10, 11] and triggers the cytochrome c (Cyt.c)-mediated death <span class="highlight" style="background-color:">pathway</span>. Caspase 2-mediated <span class="highlight" style="background-color:">signaling</span> could also activate heat-induced germ cell apoptosis by targeting mitochondria directly, possibly through a decline in cellular metabolism (dotted line).</div></p></div><div class="aux"><div class="resc">Candace Johnson, et al. Biol Reprod. 2008 November;79(5):806-814.</div><p class="links"><a class="dblinks" href="/pubmed/18614702" ref="ordinalpos=1635&amp;ncbi_uid=1959634&amp;link_uid=18614702">Citation</a><a class="dblinks" href="/pmc/articles/PMC2715000/" ref="ordinalpos=1635&amp;ncbi_uid=1959634&amp;link_uid=PMC2715000">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1636.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4129369/figure/F1/" ref="ordinalpos=1636&amp;ncbi_uid=6486147&amp;link_uid=PMC4129369" href="/pmc/articles/PMC4129369/"><img src="/pmc/articles/instance/4129369/bin/nihms533662f1.gif" src-large="/pmc/articles/instance/4129369/bin/nihms533662f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4129369/" ref="ordinalpos=1636&amp;ncbi_uid=6486147&amp;link_uid=PMC4129369" image-link="/pmc/articles/PMC4129369/figure/F1/" class="imagepopup">Figure 1. Immunohistochemical detection of TGF- and BMP <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in pediatric germ cell tumors.  From: BMP <span class="highlight" style="background-color:">Signaling</span> Activity Distinguishes Histologic Subsets of Pediatric Germ Cell Tumors. </a></p><div class="supp"><p class="details"><div>Representative images from a tissue microarray containing 14 yolk sac tumors (AF) and 10 germinomatous tumors (seminomas, dysgerminomas or CNS germinomas) (GL). C, D, I, J: Phosphorylated-SMAD3 immunohistochemistry as a readout of TGF- <span class="highlight" style="background-color:">signaling</span> activity. E, F, K, L: Phosphorylated-SMAD1/5/8 immunohistochemistry as a readout of BMP <span class="highlight" style="background-color:">signaling</span> activity. Scoring results for individual tumors are summarized in Table 1. Scale bar: 100 microns.</div></p></div><div class="aux"><div class="resc">Nicholas Fustino, et al. Int J Androl. ;34(4 0 2):e218-e233.</div><p class="links"><a class="dblinks" href="/pubmed/21696393" ref="ordinalpos=1636&amp;ncbi_uid=6486147&amp;link_uid=21696393">Citation</a><a class="dblinks" href="/pmc/articles/PMC4129369/" ref="ordinalpos=1636&amp;ncbi_uid=6486147&amp;link_uid=PMC4129369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1637.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2247407/figure/F2/" ref="ordinalpos=1637&amp;ncbi_uid=1304280&amp;link_uid=PMC2247407" href="/pmc/articles/PMC2247407/"><img src="/pmc/articles/instance/2247407/bin/nihms39909f2.gif" src-large="/pmc/articles/instance/2247407/bin/nihms39909f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2247407/" ref="ordinalpos=1637&amp;ncbi_uid=1304280&amp;link_uid=PMC2247407" image-link="/pmc/articles/PMC2247407/figure/F2/" class="imagepopup">Fig. 2.  From: Winding through the WNT <span class="highlight" style="background-color:">pathway</span> during cellular development and demise. </a></p><div class="supp"><p class="details"><div>The Wnt-Frizzled <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> employs Akt to regulate an array of vital cellular functions that involve cellular protection, angiogenesis, cardiovascular remodeling, DNA repair, and the maintenance of membrane asymmetry in neuronal and vascular systems. The Wnt-Frizzled <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> employs Akt to impact upon the survival of neurons, endothelial cells, and cardiomyocytes to prevent cellular injury and promote angiogenesis. Through the activation of Akt, microglial activity is modulated to block harmful cytokine release and prevent the phagocytosis of cells tagged by cellular phosphatidylserine (PS) membrane exposure. The Wnt-Frizzled <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> function in cardiovascular remodeling that involves regulation of LVH (left ventricular hypertrophy) and reduction in MI (myocardial infarction) and HF (heart failure).</div></p></div><div class="aux"><div class="resc">F. Li, et al. Histol Histopathol. ;21(1):103-124.</div><p class="links"><a class="dblinks" href="/pubmed/16267791" ref="ordinalpos=1637&amp;ncbi_uid=1304280&amp;link_uid=16267791">Citation</a><a class="dblinks" href="/pmc/articles/PMC2247407/" ref="ordinalpos=1637&amp;ncbi_uid=1304280&amp;link_uid=PMC2247407">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1638.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2865148/figure/F1/" ref="ordinalpos=1638&amp;ncbi_uid=2796707&amp;link_uid=PMC2865148" href="/pmc/articles/PMC2865148/"><img src="/pmc/articles/instance/2865148/bin/nihms195527f1.gif" src-large="/pmc/articles/instance/2865148/bin/nihms195527f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2865148/" ref="ordinalpos=1638&amp;ncbi_uid=2796707&amp;link_uid=PMC2865148" image-link="/pmc/articles/PMC2865148/figure/F1/" class="imagepopup">Figure 1.  From: Molecular Mechanisms of System Control of NF-?B <span class="highlight" style="background-color:">Signaling</span> by I?B?. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The figure places emphasis on the role of IB, showing the different degradation pathways and transcriptional activation of new IB synthesis. The newly synthesized IB is either degraded, binds to an NF-B in the cytoplasm, or enters the nucleus and binds nuclear NF-B. This feedback part of the <span class="highlight" style="background-color:">pathway</span> is indicated by red arrows.</div></p></div><div class="aux"><div class="resc">Diego U. Ferreiro, et al. Biochemistry. ;49(8):1560-1567.</div><p class="links"><a class="dblinks" href="/pubmed/20055496" ref="ordinalpos=1638&amp;ncbi_uid=2796707&amp;link_uid=20055496">Citation</a><a class="dblinks" href="/pmc/articles/PMC2865148/" ref="ordinalpos=1638&amp;ncbi_uid=2796707&amp;link_uid=PMC2865148">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1639.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3012164/figure/F3/" ref="ordinalpos=1639&amp;ncbi_uid=3444773&amp;link_uid=PMC3012164" href="/pmc/articles/PMC3012164/"><img src="/pmc/articles/instance/3012164/bin/zdb0011164200003.gif" src-large="/pmc/articles/instance/3012164/bin/zdb0011164200003.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3012164/" ref="ordinalpos=1639&amp;ncbi_uid=3444773&amp;link_uid=PMC3012164" image-link="/pmc/articles/PMC3012164/figure/F3/" class="imagepopup">FIG. 3. From: Direct Recruitment of Insulin Receptor and ERK <span class="highlight" style="background-color:">Signaling</span> Cascade to Insulin-Inducible Gene Loci. </a></p><div class="supp"><p class="details"><div>MAPK Erk <span class="highlight" style="background-color:">pathway</span> at the <i>egr-1</i> locus in HTC-IR cells. <i>A</i>: Schematic of one arm of the IR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with MAPK <span class="highlight" style="background-color:">pathway</span> highlighted. <i>B</i>: Cells were treated as in Fig. 1<i>A</i>, cross-linked, and used in ChIP with antibodies to Raf-B, Raf-B phosphorylated at T598 and S601 (pRaf-B), MEK1/2, MEK1/2 phosphorylated at S217/S221 (pMEK), ERK 1, and ERK1/2 phosphorylated at T202/Y204 (pERK). ChIPed DNA was used in PCR as in Fig. 1<i>C</i>. Data represent mean  SEM for three independent experiments.</div></p></div><div class="aux"><div class="resc">Joel D. Nelson, et al. Diabetes. 2011 January;60(1):127-137.</div><p class="links"><a class="dblinks" href="/pubmed/20929976" ref="ordinalpos=1639&amp;ncbi_uid=3444773&amp;link_uid=20929976">Citation</a><a class="dblinks" href="/pmc/articles/PMC3012164/" ref="ordinalpos=1639&amp;ncbi_uid=3444773&amp;link_uid=PMC3012164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1640.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634110/figure/F1/" ref="ordinalpos=1640&amp;ncbi_uid=1708626&amp;link_uid=PMC2634110" href="/pmc/articles/PMC2634110/"><img src="/pmc/articles/instance/2634110/bin/psb0303_0166_fig001.gif" src-large="/pmc/articles/instance/2634110/bin/psb0303_0166_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634110/" ref="ordinalpos=1640&amp;ncbi_uid=1708626&amp;link_uid=PMC2634110" image-link="/pmc/articles/PMC2634110/figure/F1/" class="imagepopup">Figure 1.  From: The role of phytohormone <span class="highlight" style="background-color:">signaling</span> in ozone-induced cell death in plants. </a></p><div class="supp"><p class="details"><div>Proposed pathways for salicylic acid biosynthesis and <span class="highlight" style="background-color:">signaling</span> in ozone-exposed plants. There are two pathways for salicylic acid (SA) biosynthesis in plants. <i>Arabidopsis</i> mutant <i>sid2</i>, that have a defect in <i>ICS</i>, does not accumulate salicylic acid (SA) in response to ozone, implying that SA is synthesized from isochorismate <span class="highlight" style="background-color:">pathway</span> (down from Chorismate) in <i>Arabidopsis</i>. However, another evidences support synthesis of SA from phenylalanine (PAL) <span class="highlight" style="background-color:">pathway</span> (right from Chorismate) in plants except for <i>Arabidopsis</i>. Final product (SA) of this <span class="highlight" style="background-color:">pathway</span> inhibits <i>ICS</i> expression and/or ICS activity. NPR; nonexpressor of PR-1.</div></p></div><div class="aux"><div class="resc">Masanori Tamaoki. Plant Signal Behav. 2008 March;3(3):166-174.</div><p class="links"><a class="dblinks" href="/pubmed/19513211" ref="ordinalpos=1640&amp;ncbi_uid=1708626&amp;link_uid=19513211">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634110/" ref="ordinalpos=1640&amp;ncbi_uid=1708626&amp;link_uid=PMC2634110">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1641.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4352509/figure/fig2/" ref="ordinalpos=1641&amp;ncbi_uid=7302048&amp;link_uid=PMC4352509" href="/pmc/articles/PMC4352509/"><img src="/pmc/articles/instance/4352509/bin/IJE2015-505849.002.gif" src-large="/pmc/articles/instance/4352509/bin/IJE2015-505849.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4352509/" ref="ordinalpos=1641&amp;ncbi_uid=7302048&amp;link_uid=PMC4352509" image-link="/pmc/articles/PMC4352509/figure/fig2/" class="imagepopup">Figure 2.  From: Insulin Increases Sestrin 2 Content by Reducing Its Degradation through the PI3K/mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>PI<sub>3</sub>K/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is required for the upregulation of SESN2 level by insulin treatment. HepG2 cells were pretreated with indicated reagents at the indicated concentrations for 1h, followed by an 18h further treatment with 100nM of insulin. Cells were harvested for western blotting. (a) to (c) are representative images of at least three independent experiments.</div></p></div><div class="aux"><div class="resc">Dandan Chai, et al. Int J Endocrinol. 2015;2015:505849.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1641&amp;ncbi_uid=7302048&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4352509/" ref="ordinalpos=1641&amp;ncbi_uid=7302048&amp;link_uid=PMC4352509">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1642.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163147/figure/fig6/" ref="ordinalpos=1642&amp;ncbi_uid=3190914&amp;link_uid=PMC3163147" href="/pmc/articles/PMC3163147/"><img src="/pmc/articles/instance/3163147/bin/ECAM2012-792820.006.gif" src-large="/pmc/articles/instance/3163147/bin/ECAM2012-792820.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163147/" ref="ordinalpos=1642&amp;ncbi_uid=3190914&amp;link_uid=PMC3163147" image-link="/pmc/articles/PMC3163147/figure/fig6/" class="imagepopup">Figure 6.  From: Electroacupuncture at PC6 (Neiguan) Improves Extracellular Signal-Regulated Kinase <span class="highlight" style="background-color:">Signaling</span> Pathways Through the Regulation of Neuroendocrine Cytokines in Myocardial Hypertrophic Rats. </a></p><div class="supp"><p class="details"><div>EA improves ERK <span class="highlight" style="background-color:">signaling</span> pathways through the regulation of neuroendocrine cytokines in myocardial hypertrophic rats. The proposed hypothetic mechanism of acupuncture affecting the heart acupuncture indeed improves the cardiac function in MH, and this effect has a close relation with ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through regulating the role of neuroendocrine cytokines.</div></p></div><div class="aux"><div class="resc">Jia Li, et al. Evid Based Complement Alternat Med. 2012;2012:792820.</div><p class="links"><a class="dblinks" href="/pubmed/21876715" ref="ordinalpos=1642&amp;ncbi_uid=3190914&amp;link_uid=21876715">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163147/" ref="ordinalpos=1642&amp;ncbi_uid=3190914&amp;link_uid=PMC3163147">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1643.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3248275/figure/F7/" ref="ordinalpos=1643&amp;ncbi_uid=3739888&amp;link_uid=PMC3248275" href="/pmc/articles/PMC3248275/"><img src="/pmc/articles/instance/3248275/bin/JCI42204.f7.gif" src-large="/pmc/articles/instance/3248275/bin/JCI42204.f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3248275/" ref="ordinalpos=1643&amp;ncbi_uid=3739888&amp;link_uid=PMC3248275" image-link="/pmc/articles/PMC3248275/figure/F7/" class="imagepopup">Figure 7. Schematic diagram of IGF-1 signal transduction in mammary glands in vivo..  From: Developmental stage determines estrogen receptor alpha expression and non-genomic mechanisms that control IGF-1 <span class="highlight" style="background-color:">signaling</span> and mammary proliferation in mice. </a></p><div class="supp"><p class="details"><div>In prepubertal glands, when E<sub>2</sub> levels are low and ER levels are relatively high, IGF-1 stimulates the formation of ER/IRS-1 complexes and activation of IRS-1, which directs <span class="highlight" style="background-color:">signaling</span> via the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. Activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> induces cyclin D1 expression and mammary epithelial proliferation. Cyclin D1 levels are regulated at the level of protein stability, through inhibition of Thr286 phosphorylation and ubiquitin-mediated degradation. Activation of the Akt <span class="highlight" style="background-color:">pathway</span> also induces phosphorylation of c-Raf, which inhibits transduction via the MAPK <span class="highlight" style="background-color:">pathway</span>. In postpubertal glands, ER expression is downregulated in response to elevated E<sub>2</sub>. In postpubertal Tg glands, the combination of IGF-1 and E<sub>2</sub> further reduces ER levels. Due to this downregulation, ER/IRS-1 complexes do not form, and the IRS-1/PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is not activated. However, Shc activation is maintained, allowing signal transduction via the MAPK <span class="highlight" style="background-color:">pathway</span>, which does not result in increased cyclin D1 expression or epithelial proliferation.    </div></p></div><div class="aux"><div class="resc">Jie Tian, et al. J Clin Invest. 2012 January 3;122(1):192-204.</div><p class="links"><a class="dblinks" href="/pubmed/22182837" ref="ordinalpos=1643&amp;ncbi_uid=3739888&amp;link_uid=22182837">Citation</a><a class="dblinks" href="/pmc/articles/PMC3248275/" ref="ordinalpos=1643&amp;ncbi_uid=3739888&amp;link_uid=PMC3248275">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1644.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3080792/figure/F6/" ref="ordinalpos=1644&amp;ncbi_uid=2900749&amp;link_uid=PMC3080792" href="/pmc/articles/PMC3080792/"><img src="/pmc/articles/instance/3080792/bin/1477-5956-9-15-6.gif" src-large="/pmc/articles/instance/3080792/bin/1477-5956-9-15-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3080792/" ref="ordinalpos=1644&amp;ncbi_uid=2900749&amp;link_uid=PMC3080792" image-link="/pmc/articles/PMC3080792/figure/F6/" class="imagepopup">Figure 6.  From: Systems analysis of endothelial cell plasma membrane proteome of rat lung microvasculature. </a></p><div class="supp"><p class="details"><div><b>Expanded networks of transport and <span class="highlight" style="background-color:">signaling</span> pathways</b>. The relationships among the molecules involved in transport (a) and <span class="highlight" style="background-color:">signaling</span> pathways (b) are graphed as described in the text. Integrin-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> derived by the canonical <span class="highlight" style="background-color:">pathway</span> algorithms of IPA (c). The node colors and shapes are described in figure 5.</div></p></div><div class="aux"><div class="resc">Yan Li, et al. Proteome Sci. 2011;9:15-15.</div><p class="links"><a class="dblinks" href="/pubmed/21447187" ref="ordinalpos=1644&amp;ncbi_uid=2900749&amp;link_uid=21447187">Citation</a><a class="dblinks" href="/pmc/articles/PMC3080792/" ref="ordinalpos=1644&amp;ncbi_uid=2900749&amp;link_uid=PMC3080792">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1645.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3563492/figure/F3/" ref="ordinalpos=1645&amp;ncbi_uid=4636232&amp;link_uid=PMC3563492" href="/pmc/articles/PMC3563492/"><img src="/pmc/articles/instance/3563492/bin/1471-2148-13-19-3.gif" src-large="/pmc/articles/instance/3563492/bin/1471-2148-13-19-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3563492/" ref="ordinalpos=1645&amp;ncbi_uid=4636232&amp;link_uid=PMC3563492" image-link="/pmc/articles/PMC3563492/figure/F3/" class="imagepopup">Figure 3.  From: Impact of constitutional copy number variants on biological <span class="highlight" style="background-color:">pathway</span> evolution. </a></p><div class="supp"><p class="details"><div><b>CNV-gene frequency heatmaps </b><b><i>A.</i></b><i> CNV-gene frequency heatmap for Basal transcription factors <span class="highlight" style="background-color:">pathway</span> and example of the significant gene variant association for TAF4B gene from this <span class="highlight" style="background-color:">pathway</span> taken from YRI RNA-seq data.</i> In the heatmap, rows correspond to CNV-gene pairs and columns correspond to three Hapmap populations: YRI, CEU and ASN. The values of the heatmap are CNV polymorphism frequency (see Methods). Evidence for population differentiation is given below the box plots. The schematic diagram shows connection of the gene TFA4B with NF-kB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (see text for explanations). <b><i>B</i></b><i>. Example of the significant gene variant association for APOBEC3B gene taken from CEU RNA-seq data.</i> Evidence for population differentiation is given below the box plots. Schematic diagram shows connection of the gene APOBEC3B with Alu/L1 retrotransposition (see text for explanations). <b><i>C</i></b><i>. Examples of CNV-gene frequency heatmaps for four pathways</i>: <i>Apoptosis, Fructose and mannose metabolism, NF-kb <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and Androgen and estrogen metabolism.</i> Here flanking areas for CNV-gene pairs are 10 kb.</div></p></div><div class="aux"><div class="resc">Maria Poptsova, et al. BMC Evol Biol. 2013;13:19-19.</div><p class="links"><a class="dblinks" href="/pubmed/23342974" ref="ordinalpos=1645&amp;ncbi_uid=4636232&amp;link_uid=23342974">Citation</a><a class="dblinks" href="/pmc/articles/PMC3563492/" ref="ordinalpos=1645&amp;ncbi_uid=4636232&amp;link_uid=PMC3563492">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1646.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3069685/figure/F2/" ref="ordinalpos=1646&amp;ncbi_uid=2871839&amp;link_uid=PMC3069685" href="/pmc/articles/PMC3069685/"><img src="/pmc/articles/instance/3248118/bin/oncotarget-01-457-g002.gif" src-large="/pmc/articles/instance/3248118/bin/oncotarget-01-457-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3069685/" ref="ordinalpos=1646&amp;ncbi_uid=2871839&amp;link_uid=PMC3069685" image-link="/pmc/articles/PMC3069685/figure/F2/" class="imagepopup">Figure 2. An essential ATF5-mediated survival <span class="highlight" style="background-color:">pathway</span> in malignant glioma.  From: An Activating Transcription Factor 5-Mediated Survival <span class="highlight" style="background-color:">Pathway</span> as a Target for Cancer Therapy. </a></p><div class="supp"><p class="details"><div>Cell surface receptors (FGFR or EGFR) activate RAS/MAPK or PI3K <span class="highlight" style="background-color:">signaling</span> pathways through FGFR substrate 2 (FRS2) and culminate in the activation of CREB3L2, which in turn stimulates transcription of ATF5. ATF5 then antagonizes apoptosis by directly up-regulating expression of the anti-apoptotic factor MCL1. Commercially available pharmacological inhibitors of several components of the ATF5-mediated survival <span class="highlight" style="background-color:">pathway</span> are indicated in gray.</div></p></div><div class="aux"><div class="resc">Zhi Sheng, et al. Oncotarget. 2010 October;1(6):457-460.</div><p class="links"><a class="dblinks" href="/pubmed/21311102" ref="ordinalpos=1646&amp;ncbi_uid=2871839&amp;link_uid=21311102">Citation</a><a class="dblinks" href="/pmc/articles/PMC3069685/" ref="ordinalpos=1646&amp;ncbi_uid=2871839&amp;link_uid=PMC3069685">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1647.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2474534/figure/F2/" ref="ordinalpos=1647&amp;ncbi_uid=1461996&amp;link_uid=PMC2474534" href="/pmc/articles/PMC2474534/"><img src="/pmc/articles/instance/2474534/bin/zpq9990837830002.gif" src-large="/pmc/articles/instance/2474534/bin/zpq9990837830002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2474534/" ref="ordinalpos=1647&amp;ncbi_uid=1461996&amp;link_uid=PMC2474534" image-link="/pmc/articles/PMC2474534/figure/F2/" class="imagepopup">Fig. 2. From: Differential and chaotic calcium signatures in the symbiosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of legumes. </a></p><div class="supp"><p class="details"><div>Mycorrhizal-induced calcium oscillations require components of the Sym <span class="highlight" style="background-color:">pathway</span>. (<i>a</i> and <i>b</i>) Root hair cells on whole plants (<i>a</i>) and on root cultures (<i>b</i>) showed calcium oscillations when in close proximity to highly branched <i>G. intraradices</i> hyphae. (<i>b</i>) Plants mutated in <i>dmi1</i> or <i>dmi2</i> did not show any calcium changes in response to <i>G. intraradices</i>, whereas <i>dmi3</i> mutants did show mycorrhizal-induced calcium oscillations. (<i>c</i>) The symbiosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of <i>M. truncatula. DMI1</i>, <i>DMI2</i> and <i>DMI3</i> are necessary for both Nod factor and mycorrhizal factor <span class="highlight" style="background-color:">signaling</span>, whereas <i>NFP</i>, <i>NSP1</i>, <i>NSP2</i> and <i>ERN</i> are only required for Nod factor <span class="highlight" style="background-color:">signaling</span>. The <i>y</i> axis in <i>a</i> and <i>b</i> represents the ratio of YFP to CFP in arbitrary units. NF, Nod factor; MF, mycorrhizal factor; WT, wild type; YC2.1, cameleon. The numbers indicate numbers of cells with calcium ocillations/total numbers of cells analyzed.</div></p></div><div class="aux"><div class="resc">Sonja Kosuta, et al. Proc Natl Acad Sci U S A. 2008 July 15;105(28):9823-9828.</div><p class="links"><a class="dblinks" href="/pubmed/18606999" ref="ordinalpos=1647&amp;ncbi_uid=1461996&amp;link_uid=18606999">Citation</a><a class="dblinks" href="/pmc/articles/PMC2474534/" ref="ordinalpos=1647&amp;ncbi_uid=1461996&amp;link_uid=PMC2474534">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1648.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2929951/figure/pone-0012385-g008/" ref="ordinalpos=1648&amp;ncbi_uid=2449444&amp;link_uid=PMC2929951" href="/pmc/articles/PMC2929951/"><img src="/pmc/articles/instance/2929951/bin/pone.0012385.g008.gif" src-large="/pmc/articles/instance/2929951/bin/pone.0012385.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2929951/" ref="ordinalpos=1648&amp;ncbi_uid=2449444&amp;link_uid=PMC2929951" image-link="/pmc/articles/PMC2929951/figure/pone-0012385-g008/" class="imagepopup">Figure 8. Association between PCFs and toxicity associated pathways..  From: Mapping Drug Physico-Chemical Features to <span class="highlight" style="background-color:">Pathway</span> Activity Reveals Molecular Networks Linked to Toxicity Outcome. </a></p><div class="supp"><p class="details"><div>The figure represents the detailed relationship between pathways associated to chemical hits and pathways associated to toxicity. Pathways with membrane component were mostly associated to chemical hits whereas pathways with downstream signalling components were mostly associated to toxicity. This figure represents three possible links between pathways associated to chemical hits (<i>Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></i>, <i>Long-Term Depression</i> and <i>ErbB <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></i>) and toxicity (<i>Focal Adhesion</i>, <i>ALS</i> and <i>Pancreatic Cancer</i>) through shared genes between the pathways. Although each link presents a mechanism of action these were only implied by the <span class="highlight" style="background-color:">pathway</span> associated to toxicity. Genes found to be up or down regulated have been marked with a red or a blue arrow respectively.</div></p></div><div class="aux"><div class="resc">Philipp Antczak, et al. PLoS One. 2010;5(8):e12385.</div><p class="links"><a class="dblinks" href="/pubmed/20811577" ref="ordinalpos=1648&amp;ncbi_uid=2449444&amp;link_uid=20811577">Citation</a><a class="dblinks" href="/pmc/articles/PMC2929951/" ref="ordinalpos=1648&amp;ncbi_uid=2449444&amp;link_uid=PMC2929951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1649.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2172302/figure/fig6/" ref="ordinalpos=1649&amp;ncbi_uid=1121729&amp;link_uid=PMC2172302" href="/pmc/articles/PMC2172302/"><img src="/pmc/articles/instance/2172302/bin/200401034f6.gif" src-large="/pmc/articles/instance/2172302/bin/200401034f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2172302/" ref="ordinalpos=1649&amp;ncbi_uid=1121729&amp;link_uid=PMC2172302" image-link="/pmc/articles/PMC2172302/figure/fig6/" class="imagepopup">Figure 6. From: Novel <span class="highlight" style="background-color:">signaling</span> pathways mediating reciprocal control of keratinocyte migration and wound epithelialization through M3 and M4 muscarinic receptors. </a></p><div class="supp"><p class="details"><div><b>Hypothetical scheme of the <span class="highlight" style="background-color:">signaling</span> mechanisms linking M<sub>3</sub> and M<sub>4</sub> to regulation KC motility and integrin expression.</b> The <span class="highlight" style="background-color:">signaling</span> cascades originating at M<sub>3</sub> and M<sub>4</sub> include distinct stimulatory () and inhibitory () steps and converge at the common effector <span class="highlight" style="background-color:">pathway</span> involving Rho proteins.</div></p></div><div class="aux"><div class="resc">Alex I. Chernyavsky, et al. J Cell Biol. 2004 July 19;166(2):261-272.</div><p class="links"><a class="dblinks" href="/pubmed/15263021" ref="ordinalpos=1649&amp;ncbi_uid=1121729&amp;link_uid=15263021">Citation</a><a class="dblinks" href="/pmc/articles/PMC2172302/" ref="ordinalpos=1649&amp;ncbi_uid=1121729&amp;link_uid=PMC2172302">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1650.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3495857/figure/pone-0049275-g002/" ref="ordinalpos=1650&amp;ncbi_uid=4370807&amp;link_uid=PMC3495857" href="/pmc/articles/PMC3495857/"><img src="/pmc/articles/instance/3495857/bin/pone.0049275.g002.gif" src-large="/pmc/articles/instance/3495857/bin/pone.0049275.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3495857/" ref="ordinalpos=1650&amp;ncbi_uid=4370807&amp;link_uid=PMC3495857" image-link="/pmc/articles/PMC3495857/figure/pone-0049275-g002/" class="imagepopup">Figure 2. Involvement of the p38 MAPKHsp27 antiapoptotic <span class="highlight" style="background-color:">pathway</span> in drug resistance..  From: Quercetin Suppresses Drug-Resistant Spheres via the p38 MAPK-Hsp27 Apoptotic <span class="highlight" style="background-color:">Pathway</span> in Oral Cancer Cells. </a></p><div class="supp"><p class="details"><div>(A) The expression of p-Hsp27 was dose-dependently correlated with Cis treatment in OSCC cells. P-Hsp27 was correspondently overexpressed via the activation of upstream p38 MAPK <span class="highlight" style="background-color:">signaling</span> in DRSPs. (B) Knockdown of Hsp27 reversely induced the upregulation of CI-caspase 3 and CI-PARP, resulting in increased apoptosis in DRSPs. However, there was no alteration of upstream p38 MAPK. (C) Knockdown of p38 MAPK directly downregulated p-Hsp27, indicating the presence of the p38 MAPKHsp27 antiapoptotic <span class="highlight" style="background-color:">pathway</span> in DRSPs.</div></p></div><div class="aux"><div class="resc">Su-Feng Chen, et al. PLoS One. 2012;7(11):e49275.</div><p class="links"><a class="dblinks" href="/pubmed/23152886" ref="ordinalpos=1650&amp;ncbi_uid=4370807&amp;link_uid=23152886">Citation</a><a class="dblinks" href="/pmc/articles/PMC3495857/" ref="ordinalpos=1650&amp;ncbi_uid=4370807&amp;link_uid=PMC3495857">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1651.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4053611/figure/Fig4/" ref="ordinalpos=1651&amp;ncbi_uid=6254421&amp;link_uid=PMC4053611" href="/pmc/articles/PMC4053611/"><img src="/pmc/articles/instance/4053611/bin/11010_2014_2021_Fig4_HTML.gif" src-large="/pmc/articles/instance/4053611/bin/11010_2014_2021_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4053611/" ref="ordinalpos=1651&amp;ncbi_uid=6254421&amp;link_uid=PMC4053611" image-link="/pmc/articles/PMC4053611/figure/Fig4/" class="imagepopup">Fig. 4.  From: Estrogen improves the proliferation and differentiation of hBMSCs derived from postmenopausal osteoporosis through notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> enhanced the proliferation and differentiation of OP-hBMSCs. Growth curves (<b>a</b>) and plate colony formation assay (<b>b</b>) of OP-hBMSCs illustrate improved vitality in response to Notch activation.* <i>p</i> &lt; <i>0.05</i>, compared to the control. A significant increase in <i>Runx2</i>, <i>ALP</i>, and <i>Osterix</i> is observed in <i>NICD1</i> transgene positive OP-hBMSCs. * <i>p</i> &lt; <i>0.05</i>, compared to the control (<b>c</b>). ALP staining and Alizarin <i>red</i> staining demonstrated the elevated osteogenic differentiation and mineralization in response to Notch <span class="highlight" style="background-color:">signaling</span> (<b>d</b>)</div></p></div><div class="aux"><div class="resc">Jin-Zhu Fan, et al. Mol Cell Biochem. 2014;392(1-2):85-93.</div><p class="links"><a class="dblinks" href="/pubmed/24390089" ref="ordinalpos=1651&amp;ncbi_uid=6254421&amp;link_uid=24390089">Citation</a><a class="dblinks" href="/pmc/articles/PMC4053611/" ref="ordinalpos=1651&amp;ncbi_uid=6254421&amp;link_uid=PMC4053611">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1652.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1876599/figure/fig3/" ref="ordinalpos=1652&amp;ncbi_uid=929753&amp;link_uid=PMC1876599" href="/pmc/articles/PMC1876599/"><img src="/pmc/articles/instance/1876599/bin/nrs05004.f3.gif" src-large="/pmc/articles/instance/1876599/bin/nrs05004.f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1876599/" ref="ordinalpos=1652&amp;ncbi_uid=929753&amp;link_uid=PMC1876599" image-link="/pmc/articles/PMC1876599/figure/fig3/" class="imagepopup">Figure 3. Schematic representation of the current understanding of the PELP1/MNAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Functional and biological properties of the nuclear receptor coregulator PELP1/MNAR . </a></p><div class="supp"><p class="details"><div>PELP1 participation in multiple stages of nuclear receptor (genomic and nongenomic) <span class="highlight" style="background-color:">signaling</span> pathways suggests that PELP1 deregulation likely contributes to the development of hormonal independence, tumorigenesis and metastasis.</div></p></div><div class="aux"><div class="resc">Ratna K. Vadlamudi, et al. Nucl Recept Signal. 2007;5:e004.</div><p class="links"><a class="dblinks" href="/pubmed/17525794" ref="ordinalpos=1652&amp;ncbi_uid=929753&amp;link_uid=17525794">Citation</a><a class="dblinks" href="/pmc/articles/PMC1876599/" ref="ordinalpos=1652&amp;ncbi_uid=929753&amp;link_uid=PMC1876599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1653.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2174628/figure/f3/" ref="ordinalpos=1653&amp;ncbi_uid=1158095&amp;link_uid=PMC2174628" href="/pmc/articles/PMC2174628/"><img src="/pmc/articles/instance/2174628/bin/msb4100195-f3.gif" src-large="/pmc/articles/instance/2174628/bin/msb4100195-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2174628/" ref="ordinalpos=1653&amp;ncbi_uid=1158095&amp;link_uid=PMC2174628" image-link="/pmc/articles/PMC2174628/figure/f3/" class="imagepopup">Figure 3.  From: Evolvable <span class="highlight" style="background-color:">signaling</span> networks of receptor tyrosine kinases: relevance of robustness to malignancy and to cancer therapy. </a></p><div class="supp"><p class="details"><div>From a vertical RTK cascade to a <span class="highlight" style="background-color:">signaling</span> network. The boxed cascade (left) represents an invertebrate primordial <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> comprising a growth factor, an RTK and a <span class="highlight" style="background-color:">signaling</span> cascade culminating in regulation of gene expression (horizontal arrows). Two RTK expansion scenarios are presented. According to Option A, two events of RTK duplication generate four vertical cascades that gradually diverge, but they remain isolated from each other. According Option B, the four cascades richly interact and ultimately establish a layered <span class="highlight" style="background-color:">signaling</span> network. The lines of evidence we describe herein propose that the evolution of RTKs in vertebrates preferred Option B, because it imparts robustness and guarantees output reproducibility.</div></p></div><div class="aux"><div class="resc">Ido Amit, et al. Mol Syst Biol. 2007;3:151.</div><p class="links"><a class="dblinks" href="/pubmed/18059446" ref="ordinalpos=1653&amp;ncbi_uid=1158095&amp;link_uid=18059446">Citation</a><a class="dblinks" href="/pmc/articles/PMC2174628/" ref="ordinalpos=1653&amp;ncbi_uid=1158095&amp;link_uid=PMC2174628">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1654.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3274353/figure/F7/" ref="ordinalpos=1654&amp;ncbi_uid=4663157&amp;link_uid=PMC3274353" href="/pmc/articles/PMC3274353/"><img src="/pmc/articles/instance/3274353/bin/DEV071688F7.gif" src-large="/pmc/articles/instance/3274353/bin/DEV071688F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3274353/" ref="ordinalpos=1654&amp;ncbi_uid=4663157&amp;link_uid=PMC3274353" image-link="/pmc/articles/PMC3274353/figure/F7/" class="imagepopup">Fig. 7. From: The roles of FGF and MAP kinase <span class="highlight" style="background-color:">signaling</span> in the segregation of the epiblast and hypoblast cell lineages in bovine and human embryos. </a></p><div class="supp"><p class="details"><div><b>Model for the roles of FGF/MAP kinase <span class="highlight" style="background-color:">signaling</span> in the formation of NANOG-positive epiblast cells or GATA6-positive hypoblast/primitive endoderm cells.</b> In mouse, FGF <span class="highlight" style="background-color:">signaling</span> activates MAPK, which leads to the expression of GATA6 and repression of NANOG. In cattle, FGF <span class="highlight" style="background-color:">signaling</span> represses NANOG expression via MAPK. As a result of NANOG expression, expression of GATA6 is inhibited. GATA6 expression is also activated via an as yet unknown (question mark) FGF/MAPK-independent <span class="highlight" style="background-color:">pathway</span>. In human, activation of NANOG and GATA6 expression is coordinated via unknown (question mark) mechanisms.</div></p></div><div class="aux"><div class="resc">Ewart W. Kuijk, et al. Development. 2012 March 1;139(5):871-882.</div><p class="links"><a class="dblinks" href="/pubmed/22278923" ref="ordinalpos=1654&amp;ncbi_uid=4663157&amp;link_uid=22278923">Citation</a><a class="dblinks" href="/pmc/articles/PMC3274353/" ref="ordinalpos=1654&amp;ncbi_uid=4663157&amp;link_uid=PMC3274353">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1655.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2965074/figure/F3/" ref="ordinalpos=1655&amp;ncbi_uid=2535341&amp;link_uid=PMC2965074" href="/pmc/articles/PMC2965074/"><img src="/pmc/articles/instance/2965074/bin/nihms-246024-f0003.gif" src-large="/pmc/articles/instance/2965074/bin/nihms-246024-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2965074/" ref="ordinalpos=1655&amp;ncbi_uid=2535341&amp;link_uid=PMC2965074" image-link="/pmc/articles/PMC2965074/figure/F3/" class="imagepopup">Fig. 3.  From: The IB kinase complex: master regulator of NF-B <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Distinct modes of IKK complex activity transduce TNF and IL-1 <span class="highlight" style="background-color:">signaling</span> to NF-<i></i>B: Our findings verify that TNF-induced I<i></i>B<i></i> degradation and classical NF-<i></i>B activation is critically dependent upon NEMO and IKK<i></i> (<i>left</i>). Similarly IL-1 <span class="highlight" style="background-color:">signaling</span> absolutely requires intact NEMO (<i>right</i>). However, IL-1-induced I<i></i>B<i></i> degradation, NF-<i></i>B nuclear translocation and NF-<i></i>B transcriptional activity occurs in the absence of IKK<i></i> demonstrating that NEMO and IKK<i></i> can form a <span class="highlight" style="background-color:">signaling</span> complex capable of activating the classical NF-<i></i>B <span class="highlight" style="background-color:">pathway</span> in response to certain pro-inflammatory stimuli [83]</div></p></div><div class="aux"><div class="resc">Laura A. Solt, et al. Immunol Res. ;42(1-3):3-18.</div><p class="links"><a class="dblinks" href="/pubmed/18626576" ref="ordinalpos=1655&amp;ncbi_uid=2535341&amp;link_uid=18626576">Citation</a><a class="dblinks" href="/pmc/articles/PMC2965074/" ref="ordinalpos=1655&amp;ncbi_uid=2535341&amp;link_uid=PMC2965074">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1656.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2354921/figure/F5/" ref="ordinalpos=1656&amp;ncbi_uid=1410329&amp;link_uid=PMC2354921" href="/pmc/articles/PMC2354921/"><img src="/pmc/articles/instance/2354921/bin/nihms-43279-f0005.gif" src-large="/pmc/articles/instance/2354921/bin/nihms-43279-f0005.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2354921/" ref="ordinalpos=1656&amp;ncbi_uid=1410329&amp;link_uid=PMC2354921" image-link="/pmc/articles/PMC2354921/figure/F5/" class="imagepopup">Fig. 5. Model of BMP signal re-activation by Xolloid cleavage.  From: Molecular mechanisms of cell-cell <span class="highlight" style="background-color:">signaling</span> by the Spemann-Mangold organizer. </a></p><div class="supp"><p class="details"><div>BMP-4 binds to BMP receptors inducing the ventral <span class="highlight" style="background-color:">pathway</span>. Binding to Chordin blocks this <span class="highlight" style="background-color:">signaling</span>, whereas cleavage of Chordin by the Xolloid metalloprotease at two specific sites releases active BMP-4, re-establishing the ventralizing signal.</div></p></div><div class="aux"><div class="resc">EDDY M. DE ROBERTIS, et al. Int J Dev Biol. ;45(1):189-197.</div><p class="links"><a class="dblinks" href="/pubmed/11291846" ref="ordinalpos=1656&amp;ncbi_uid=1410329&amp;link_uid=11291846">Citation</a><a class="dblinks" href="/pmc/articles/PMC2354921/" ref="ordinalpos=1656&amp;ncbi_uid=1410329&amp;link_uid=PMC2354921">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1657.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3117351/figure/F7/" ref="ordinalpos=1657&amp;ncbi_uid=3027184&amp;link_uid=PMC3117351" href="/pmc/articles/PMC3117351/"><img src="/pmc/articles/instance/3117351/bin/btr203f7.gif" src-large="/pmc/articles/instance/3117351/bin/btr203f7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3117351/" ref="ordinalpos=1657&amp;ncbi_uid=3027184&amp;link_uid=PMC3117351" image-link="/pmc/articles/PMC3117351/figure/F7/" class="imagepopup">Fig. 7. From: RINQ: Reference-based Indexing for Network Queries. </a></p><div class="supp"><p class="details"><div>Sample queries from our experiments. (<b>a</b>) A subnetwork of ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of <i>Rattus Norvegicus</i> (rat). (<b>b</b>) A subnetwork of VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of <i>Mus Musculus</i> (mouse).</div></p></div><div class="aux"><div class="resc">Gnhan Glsoy, et al. Bioinformatics. 2011 July 1;27(13):i149-i158.</div><p class="links"><a class="dblinks" href="/pubmed/21685064" ref="ordinalpos=1657&amp;ncbi_uid=3027184&amp;link_uid=21685064">Citation</a><a class="dblinks" href="/pmc/articles/PMC3117351/" ref="ordinalpos=1657&amp;ncbi_uid=3027184&amp;link_uid=PMC3117351">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1658.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584891/figure/F4/" ref="ordinalpos=1658&amp;ncbi_uid=5262000&amp;link_uid=PMC3584891" href="/pmc/articles/PMC3584891/"><img src="/pmc/articles/instance/3584891/bin/zii9990900100004.gif" src-large="/pmc/articles/instance/3584891/bin/zii9990900100004.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584891/" ref="ordinalpos=1658&amp;ncbi_uid=5262000&amp;link_uid=PMC3584891" image-link="/pmc/articles/PMC3584891/figure/F4/" class="imagepopup">Fig 4.  From: Schistosome Egg Antigens Elicit a Proinflammatory Response by Trophoblast Cells of the Human Placenta. </a></p><div class="supp"><p class="details"><div>The ERK1/2 and NF-B <span class="highlight" style="background-color:">signaling</span> pathways are the dominant mediators of the proinflammatory cascade initiated by SEA in trophoblasts. Pharmacological inhibition of the JNK, p38, and ERK1/2 MAPK pathways as well as the NF-B <span class="highlight" style="background-color:">pathway</span> prior to SEA (25 g/ml) exposure for 24 h. (A, C, E, and G) IL-6 production after SEA with the addition of either vehicle (DMSO) or specific inhibitors. (B, D, F, and H) IL-8 production from the same experiments. (A and B) Inhibition of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> pathways (<i>n</i> = 6). (C and D) Inhibition of the p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>n</i> = 6). (E and F) Inhibition of the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>n</i> = 5). (G and H) Inhibition of the NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>n</i> = 6 individual trophoblast preparations from distinct placentas). All data are reported as means  SEM. Unique letters denote statistically significant differences at <i>P</i> values of &lt;0.05.</div></p></div><div class="aux"><div class="resc">Emily A. McDonald, et al. Infect Immun. 2013 March;81(3):704-712.</div><p class="links"><a class="dblinks" href="/pubmed/23250950" ref="ordinalpos=1658&amp;ncbi_uid=5262000&amp;link_uid=23250950">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584891/" ref="ordinalpos=1658&amp;ncbi_uid=5262000&amp;link_uid=PMC3584891">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1659.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766078/figure/F4/" ref="ordinalpos=1659&amp;ncbi_uid=2119696&amp;link_uid=PMC2766078" href="/pmc/articles/PMC2766078/"><img src="/pmc/articles/instance/2766078/bin/nihms126727f4.gif" src-large="/pmc/articles/instance/2766078/bin/nihms126727f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766078/" ref="ordinalpos=1659&amp;ncbi_uid=2119696&amp;link_uid=PMC2766078" image-link="/pmc/articles/PMC2766078/figure/F4/" class="imagepopup">Figure 4.  From: Endothelial Cells Provide Feedback Control for Vascular Remodeling Through a Mechanosensitive Autocrine TGF-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Regulation of perlecan requires autocrine TGF- <span class="highlight" style="background-color:">signaling</span>, p38 MAPK and ERK <span class="highlight" style="background-color:">signaling</span> pathways. <b>(a)</b> Mechanical strain induces increased TGF- production by human endothelial cells. Results shown are an ELISA assay after subtracting TGF- in growth media. This effect is blocked by inhibitors of the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (U0126) and p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (SB 293063). *<i>p</i> &lt; 0.05 versus all other groups; **<i>p</i> &lt; 0.05 significantly different from all groups except unloaded samples with SB293062; other comparisons not shown for simplicity. <b>(b)</b> The p38 and ERK <span class="highlight" style="background-color:">signaling</span> pathways are activated by 24 hrs of mechanical strain. Endothelial cells were treated with inhibitors one hour prior to exposure to 24 hrs of mechanical strain. Western blotting analysis revealed increased phosphorylation of smad 2, ERK1/2 and p38 MAPK in the absence of inhibitors. <b>(c)</b> Autocrine TGF- <span class="highlight" style="background-color:">signaling</span> is needed for maximal phosphorylation of p38, ERK1/2 and Smad 2. Endothelial cells were treated with a neutralizing antibody to TGF- for 1 hr prior to 24 hrs of cyclic mechanical strain. Western blotting analysis of intracellular <span class="highlight" style="background-color:">signaling</span> intermediates showed reduced activation of Smad 2, ERK1/2, and p38 by mechanical strain in the presence of a neutralizing antibody to TGF-. <b>(d)</b> Inhibitors to p38 or MEK block load-mediated increases in soluble perlecan. <b>(e)</b> A neutralizing antibody to TGF- blocks load-induced increase in soluble perlecan. <b>(f)</b> Addition of 100 pg/ml TGF- increased perlecan expression even in the absence of mechanical strain. An inhibitor to TGF- receptor I <span class="highlight" style="background-color:">signaling</span>, 200 nM of [3-(Pyridin-2-yl)-4-(4-quinonyl)] -1H-pyrazole, blocked perlecan upregulation by mechanical load.</div></p></div><div class="aux"><div class="resc">Aaron B. Baker, et al. Circ Res. ;103(3):289-297.</div><p class="links"><a class="dblinks" href="/pubmed/18583708" ref="ordinalpos=1659&amp;ncbi_uid=2119696&amp;link_uid=18583708">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766078/" ref="ordinalpos=1659&amp;ncbi_uid=2119696&amp;link_uid=PMC2766078">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1660.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3210157/figure/pone-0027318-g008/" ref="ordinalpos=1660&amp;ncbi_uid=3403850&amp;link_uid=PMC3210157" href="/pmc/articles/PMC3210157/"><img src="/pmc/articles/instance/3210157/bin/pone.0027318.g008.gif" src-large="/pmc/articles/instance/3210157/bin/pone.0027318.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3210157/" ref="ordinalpos=1660&amp;ncbi_uid=3403850&amp;link_uid=PMC3210157" image-link="/pmc/articles/PMC3210157/figure/pone-0027318-g008/" class="imagepopup">Figure 8. Hypothetical model of 16K hPRL effects on Notch <span class="highlight" style="background-color:">signaling</span> in endothelial/pericyte interactions..  From: The Antiangiogenic 16K Prolactin Impairs Functional Tumor Neovascularization by Inhibiting Vessel Maturation. </a></p><div class="supp"><p class="details"><div>When endothelial cells are cultured with pericytes, Dll4/Notch4 <span class="highlight" style="background-color:">pathway</span> is activated. 16K hPRL treatment (6 h) alters Dll4/Notch4 <span class="highlight" style="background-color:">signaling</span> by decreasing <i>DLL4</i>, <i>NOTCH4</i>, <i>HEY2</i> and <i>EPHRINB2</i> transcripts in endothelial cells. After 16 h of endothelial cell-pericyte cocultures, Notch3 and Dll1 are increased respectively in pericytes and endothelial cells. 16K hPRL inhibits the Dll1/Notch3 <span class="highlight" style="background-color:">signaling</span> by reducing <i>DLL1</i> transcripts in endothelial cells and <i>NOTCH3</i> and <i>aSMA</i> transcripts in pericytes.</div></p></div><div class="aux"><div class="resc">Ngoc-Quynh-Nhu Nguyen, et al. PLoS One. 2011;6(11):e27318.</div><p class="links"><a class="dblinks" href="/pubmed/22087289" ref="ordinalpos=1660&amp;ncbi_uid=3403850&amp;link_uid=22087289">Citation</a><a class="dblinks" href="/pmc/articles/PMC3210157/" ref="ordinalpos=1660&amp;ncbi_uid=3403850&amp;link_uid=PMC3210157">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1661.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2917731/figure/F1/" ref="ordinalpos=1661&amp;ncbi_uid=2986524&amp;link_uid=PMC2917731" href="/pmc/articles/PMC2917731/"><img src="/pmc/articles/instance/2917731/bin/iaa0152-0101-f01.gif" src-large="/pmc/articles/instance/2917731/bin/iaa0152-0101-f01.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2917731/" ref="ordinalpos=1661&amp;ncbi_uid=2986524&amp;link_uid=PMC2917731" image-link="/pmc/articles/PMC2917731/figure/F1/" class="imagepopup">Fig. 1.  From: FIP1L1/PDGFR?-Associated Systemic Mastocytosis. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> of FIP1L1/PDGFR+ (F/P) cells. FIP1L1/PDGFR+ (F/P) primary mouse eosinophils express up-regulated IL-5R and FIP1L1/PDGFR activates the JAK2/STAT5 <span class="highlight" style="background-color:">pathway</span>. The CCR3/ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may be amplified by FIP1L1/PDGFR expression [25]. Up-regulated expressions of <sub>4</sub> integrin and Siglec-F were observed in FIP1L1/PDGFR+murine eosinophils [8]. FIP1L1/PDGFR synergizes with SCF stimulation via c-kit to activate Akt <span class="highlight" style="background-color:">signaling</span> in mouse mast cells. Eosinophils and mast cells also express c-kit and IL-5R, respectively.</div></p></div><div class="aux"><div class="resc">Yoshiyuki Yamada, et al. Int Arch Allergy Immunol. 2010 June;152(Suppl 1):101-105.</div><p class="links"><a class="dblinks" href="/pubmed/20523072" ref="ordinalpos=1661&amp;ncbi_uid=2986524&amp;link_uid=20523072">Citation</a><a class="dblinks" href="/pmc/articles/PMC2917731/" ref="ordinalpos=1661&amp;ncbi_uid=2986524&amp;link_uid=PMC2917731">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1662.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2930854/figure/pone-0012486-g010/" ref="ordinalpos=1662&amp;ncbi_uid=2456532&amp;link_uid=PMC2930854" href="/pmc/articles/PMC2930854/"><img src="/pmc/articles/instance/2930854/bin/pone.0012486.g010.gif" src-large="/pmc/articles/instance/2930854/bin/pone.0012486.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2930854/" ref="ordinalpos=1662&amp;ncbi_uid=2456532&amp;link_uid=PMC2930854" image-link="/pmc/articles/PMC2930854/figure/pone-0012486-g010/" class="imagepopup">Figure 10. <span class="highlight" style="background-color:">Pathway</span> of EphA5 receptor activation of synaptogenesis and electrical activity..  From: Ephrin-A5 and EphA5 Interaction Induces Synaptogenesis during Early Hippocampal Development. </a></p><div class="supp"><p class="details"><div>Cell-cell contact activates ephrinA5-EphA5 receptor <span class="highlight" style="background-color:">signaling</span> during early development, prior to the onset of electrical activity. EphA5 receptor <span class="highlight" style="background-color:">signaling</span> induces the formation of synapses and spines with electrically active NMDA receptors. EphA5 may also activate cdc42 through an as-yet-unidentified GEF (not shown). See text for a presentation of the full <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yukio Akaneya, et al. PLoS One. 2010;5(8):e12486.</div><p class="links"><a class="dblinks" href="/pubmed/20824214" ref="ordinalpos=1662&amp;ncbi_uid=2456532&amp;link_uid=20824214">Citation</a><a class="dblinks" href="/pmc/articles/PMC2930854/" ref="ordinalpos=1662&amp;ncbi_uid=2456532&amp;link_uid=PMC2930854">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1663.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3431105/figure/fig1/" ref="ordinalpos=1663&amp;ncbi_uid=4223449&amp;link_uid=PMC3431105" href="/pmc/articles/PMC3431105/"><img src="/pmc/articles/instance/3431105/bin/OXIMED2012-608478.001.gif" src-large="/pmc/articles/instance/3431105/bin/OXIMED2012-608478.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3431105/" ref="ordinalpos=1663&amp;ncbi_uid=4223449&amp;link_uid=PMC3431105" image-link="/pmc/articles/PMC3431105/figure/fig1/" class="imagepopup">Figure 1.  From: ROS in Aging Caenorhabditis elegans: Damage or <span class="highlight" style="background-color:">Signaling</span>?. </a></p><div class="supp"><p class="details"><div>Schematic representation of <span class="highlight" style="background-color:">signaling</span> pathways regulated by ROS in <i>C. elegans</i>. PPP: pentose phosphate <span class="highlight" style="background-color:">pathway</span>; CAC: citric acid cycle; GAPDH: glyceraldehyde 3-phosphate dehydrogenase; Sp: spermatheca.</div></p></div><div class="aux"><div class="resc">Patricia Back, et al. Oxid Med Cell Longev. 2012;2012:608478.</div><p class="links"><a class="dblinks" href="/pubmed/22966416" ref="ordinalpos=1663&amp;ncbi_uid=4223449&amp;link_uid=22966416">Citation</a><a class="dblinks" href="/pmc/articles/PMC3431105/" ref="ordinalpos=1663&amp;ncbi_uid=4223449&amp;link_uid=PMC3431105">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1664.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3220493/figure/F6/" ref="ordinalpos=1664&amp;ncbi_uid=3675636&amp;link_uid=PMC3220493" href="/pmc/articles/PMC3220493/"><img src="/pmc/articles/instance/3220493/bin/zbc0511188570006.gif" src-large="/pmc/articles/instance/3220493/bin/zbc0511188570006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3220493/" ref="ordinalpos=1664&amp;ncbi_uid=3675636&amp;link_uid=PMC3220493" image-link="/pmc/articles/PMC3220493/figure/F6/" class="imagepopup">FIGURE 6. From: Fibroblast Growth Factor 2 Stimulation of Osteoblast Differentiation and Bone Formation Is Mediated by Modulation of the Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic model in which FGF2 stimulation of osteoblast differentiation is partially through modulation of the Wnt/-catenin <span class="highlight" style="background-color:">pathway</span>.</b> Loss of endogenous FGF2 results in reduced Wnt10b mRNA expression, reduced <i>Lrp5</i>/<i>Lrp6</i> mRNA expression, reduced -catenin mRNA and protein expression, and reduced inactive phosphorylated GSK3 protein expression, all of which lead to reduced Wnt/-catenin <span class="highlight" style="background-color:">signaling</span>. Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> directly regulates osteoblast transcription factors, including Runx2 and probably ATF4. The reduced Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> and reduction of Dkk2 mRNA expression (important for terminal osteoblast differentiation) probably contribute to decreased osteoblast differentiation and bone formation in <i>Fgf2</i><sup>/</sup> mice. However, exogenous FGF2 promoted -catenin nuclear accumulation and induced inhibited kinase GSK3, which could increase Wnt <span class="highlight" style="background-color:">signaling</span> in <i>Fgf2</i><sup>/</sup> BMSCs and probably contributes to rescue the decreased osteoblast differentiation in <i>Fgf2</i><sup>/</sup> BMSCs. <i>LEF</i>/<i>TCF</i>, lymphoid enhancer-binding factor 1/T cell-specific transcription factor.</div></p></div><div class="aux"><div class="resc">Yurong Fei, et al. J Biol Chem. 2011 November 25;286(47):40575-40583.</div><p class="links"><a class="dblinks" href="/pubmed/21987573" ref="ordinalpos=1664&amp;ncbi_uid=3675636&amp;link_uid=21987573">Citation</a><a class="dblinks" href="/pmc/articles/PMC3220493/" ref="ordinalpos=1664&amp;ncbi_uid=3675636&amp;link_uid=PMC3220493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1665.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2030456/figure/F5/" ref="ordinalpos=1665&amp;ncbi_uid=1171192&amp;link_uid=PMC2030456" href="/pmc/articles/PMC2030456/"><img src="/pmc/articles/instance/2048762/bin/JCI0731377.f5.gif" src-large="/pmc/articles/instance/2048762/bin/JCI0731377.f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2030456/" ref="ordinalpos=1665&amp;ncbi_uid=1171192&amp;link_uid=PMC2030456" image-link="/pmc/articles/PMC2030456/figure/F5/" class="imagepopup">Figure 5. Wnt5a signals through Jnk..  From: Derivation of sarcomas from mesenchymal stem cells via inactivation of the Wnt <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Relative activity of Wnt noncanonical <span class="highlight" style="background-color:">signaling</span> and its inhibitors: Wnt/Ca <span class="highlight" style="background-color:">pathway</span> (Ca and hispidin), Wnt/Rho <span class="highlight" style="background-color:">pathway</span> (Rho and Y-276632), and Wnt/Jnk (JNK and SP600125) in specified samples under the indicated conditions. (<b>B</b>) Relative activity of Wnt/Jnk in specified samples under the indicated conditions. All measurements (<b>A</b> and <b>B</b>) indicated are the average of 2 independent time series.                </div></p></div><div class="aux"><div class="resc">Igor Matushansky, et al. J Clin Invest. 2007 November 1;117(11):3248-3257.</div><p class="links"><a class="dblinks" href="/pubmed/17948129" ref="ordinalpos=1665&amp;ncbi_uid=1171192&amp;link_uid=17948129">Citation</a><a class="dblinks" href="/pmc/articles/PMC2030456/" ref="ordinalpos=1665&amp;ncbi_uid=1171192&amp;link_uid=PMC2030456">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1666.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3431883/figure/f3-ijms-13-10647/" ref="ordinalpos=1666&amp;ncbi_uid=4213160&amp;link_uid=PMC3431883" href="/pmc/articles/PMC3431883/"><img src="/pmc/articles/instance/3431883/bin/ijms-13-10647f3.gif" src-large="/pmc/articles/instance/3431883/bin/ijms-13-10647f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3431883/" ref="ordinalpos=1666&amp;ncbi_uid=4213160&amp;link_uid=PMC3431883" image-link="/pmc/articles/PMC3431883/figure/f3-ijms-13-10647/" class="imagepopup">Figure 3.  From: Molecular Mechanisms of Oligodendrocyte Injury in Multiple Sclerosis and Experimental Autoimmune Encephalomyelitis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> mechanisms of oligodendrocyte injury. (<b>A</b>) IFN-s STAT1/IRF-1 <span class="highlight" style="background-color:">signaling</span>; (<b>B</b>) T cell-mediated cytotoxicity and IRF-1/Caspase 1 <span class="highlight" style="background-color:">signaling</span>. IFN-s STAT1/IRF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> upregulates the expression of a number of immune and cell-death related genes, which in turn mediate cytotoxic cell death of oligodendrocytes. Experimental (transgenic) suppression of this particular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (STAT1, IRF-1), its downstream gene targets (TNF-R, Fas) and adoptive/associated functional molecules (FADD, caspases) specifically in oligodendrocytes results in protection against EAE. Involvement of Caspase 1 suggests possible occurrence of pyroptosis.</div></p></div><div class="aux"><div class="resc">Jilpa Patel, et al. Int J Mol Sci. 2012;13(8):10647-10659.</div><p class="links"><a class="dblinks" href="/pubmed/22949885" ref="ordinalpos=1666&amp;ncbi_uid=4213160&amp;link_uid=22949885">Citation</a><a class="dblinks" href="/pmc/articles/PMC3431883/" ref="ordinalpos=1666&amp;ncbi_uid=4213160&amp;link_uid=PMC3431883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1667.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2606015/figure/fig9/" ref="ordinalpos=1667&amp;ncbi_uid=1606683&amp;link_uid=PMC2606015" href="/pmc/articles/PMC2606015/"><img src="/pmc/articles/instance/2606015/bin/zbc0040961640009.gif" src-large="/pmc/articles/instance/2606015/bin/zbc0040961640009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2606015/" ref="ordinalpos=1667&amp;ncbi_uid=1606683&amp;link_uid=PMC2606015" image-link="/pmc/articles/PMC2606015/figure/fig9/" class="imagepopup">FIGURE 9. From: Mechanistic Analysis of the Amplification and Diversification Events  Induced by Vav Proteins in  B-lymphocytes. </a></p><div class="supp"><p class="details"><div><b>A schematic view of the Vav family-regulated <span class="highlight" style="background-color:">signaling</span> pathways of DT40  B-cells.</b> The main signal transduction molecules with positive roles in  downstream <span class="highlight" style="background-color:">signaling</span> are indicated in <i>black</i> (the ones analyzed in  this work) and <i>blue</i> (the ones not analyzed in our experiments).  Putative negative regulators of B-cell <span class="highlight" style="background-color:">signaling</span> that could be operating in  this <span class="highlight" style="background-color:">pathway</span> are shown in <i>red</i>. The main second messengers are  indicated in <i>green</i>. <i>Black thick arrows</i> highlight the  prevalence of a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> over the rest of the <span class="highlight" style="background-color:">signaling</span> connections  presented in the figure. <i>Gray arrows</i> indicate pathways that have not  been demonstrated yet in DT40 cells. See main text for further  information.</div></p></div><div class="aux"><div class="resc">Mara J. Caloca, et al. J Biol Chem. 2008 December 26;283(52):36454-36464.</div><p class="links"><a class="dblinks" href="/pubmed/18974050" ref="ordinalpos=1667&amp;ncbi_uid=1606683&amp;link_uid=18974050">Citation</a><a class="dblinks" href="/pmc/articles/PMC2606015/" ref="ordinalpos=1667&amp;ncbi_uid=1606683&amp;link_uid=PMC2606015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1668.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3350769/figure/Fig2/" ref="ordinalpos=1668&amp;ncbi_uid=4025916&amp;link_uid=PMC3350769" href="/pmc/articles/PMC3350769/"><img src="/pmc/articles/instance/3350769/bin/18_2011_898_Fig2_HTML.gif" src-large="/pmc/articles/instance/3350769/bin/18_2011_898_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3350769/" ref="ordinalpos=1668&amp;ncbi_uid=4025916&amp;link_uid=PMC3350769" image-link="/pmc/articles/PMC3350769/figure/Fig2/" class="imagepopup">Fig. 2.  From: MAPK <span class="highlight" style="background-color:">pathway</span> activation in pilocytic astrocytoma. </a></p><div class="supp"><p class="details"><div>Distribution of oncogenic hits in the MAPK/ERK <span class="highlight" style="background-color:">pathway</span> by tumor location. <b>a</b> Representation of the MAPK/ERK signal cascade. Aberrations activating the <span class="highlight" style="background-color:">pathway</span>, i.e. activating mutations (<i>star</i>), inactivating mutation of the repressor NF1 (<i>X</i>), fusions proteins and unknown alterations (<i>?</i>), are indicated with their frequencies in cerebellar and non-cerebellar PAs. Activation of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can induce various cellular responses like cell growth, differentiation, and oncogene-induced senescence (<i>OIS</i>). <b>b</b> The various MAPK <span class="highlight" style="background-color:">pathway</span> alterations are unevenly distributed in tumors of different locations. NF1 mutation is mainly seen in optic <span class="highlight" style="background-color:">pathway</span> gliomas but also occasionally in other locations. Tumors in other brain regions are dominated by RAF activation, with fusions occurring primarily in cerebellar tumors and mutations in supratentorial PAs. For the more infrequent hits, no prevalent locations can be given</div></p></div><div class="aux"><div class="resc">David T. W. Jones, et al. Cell Mol Life Sci. 2012 June;69(11):1799-1811.</div><p class="links"><a class="dblinks" href="/pubmed/22159586" ref="ordinalpos=1668&amp;ncbi_uid=4025916&amp;link_uid=22159586">Citation</a><a class="dblinks" href="/pmc/articles/PMC3350769/" ref="ordinalpos=1668&amp;ncbi_uid=4025916&amp;link_uid=PMC3350769">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1669.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3759173/figure/f4-cancers-03-02827/" ref="ordinalpos=1669&amp;ncbi_uid=5269501&amp;link_uid=PMC3759173" href="/pmc/articles/PMC3759173/"><img src="/pmc/articles/instance/3759173/bin/cancers-03-02827f4a.gif" src-large="/pmc/articles/instance/3759173/bin/cancers-03-02827f4a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3759173/figure/f4-cancers-03-02827/" ref="ordinalpos=1669&amp;ncbi_uid=5269501&amp;link_uid=PMC3759173" href="/pmc/articles/PMC3759173/"><img src="/pmc/articles/instance/3759173/bin/cancers-03-02827f4b.gif" src-large="/pmc/articles/instance/3759173/bin/cancers-03-02827f4b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759173/" ref="ordinalpos=1669&amp;ncbi_uid=5269501&amp;link_uid=PMC3759173" image-link="/pmc/articles/PMC3759173/figure/f4-cancers-03-02827/" class="imagepopup">Figure 4. From: Microarray Analysis in a Cell Death Resistant Glioma Cell Line to Identify <span class="highlight" style="background-color:">Signaling</span> Pathways and Novel Genes Controlling Resistance and Malignancy. </a></p><div class="supp"><p class="details"><div>The AKT/TGF- <span class="highlight" style="background-color:">pathway</span> and subcellular localization of differentially regulated genes connected with this <span class="highlight" style="background-color:">signaling</span> network: Within the AKT/TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> a total of 26 genes were differentially expressed in Group 1 (<b>a</b>); whereas 21 genes were shown to be regulated in Group 2 (<b>b</b>). (<i>Green marks</i> indicate upregulation, <i>red marks</i> down-regulation of the respective mRNA in 229R cells.)</div></p></div><div class="aux"><div class="resc">Janina Seznec, et al. Cancers (Basel). 2011 September;3(3):2827-2843.</div><p class="links"><a class="dblinks" href="/pubmed/24212935" ref="ordinalpos=1669&amp;ncbi_uid=5269501&amp;link_uid=24212935">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759173/" ref="ordinalpos=1669&amp;ncbi_uid=5269501&amp;link_uid=PMC3759173">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1670.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584589/figure/f3-ijo-40-04-1162/" ref="ordinalpos=1670&amp;ncbi_uid=4686865&amp;link_uid=PMC3584589" href="/pmc/articles/PMC3584589/"><img src="/pmc/articles/instance/3584589/bin/IJO-40-04-1162-g02.gif" src-large="/pmc/articles/instance/3584589/bin/IJO-40-04-1162-g02.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584589/" ref="ordinalpos=1670&amp;ncbi_uid=4686865&amp;link_uid=PMC3584589" image-link="/pmc/articles/PMC3584589/figure/f3-ijo-40-04-1162/" class="imagepopup">Figure 3.  From: MicroRNA-200a suppresses the Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by interacting with ?-catenin. </a></p><div class="supp"><p class="details"><div>Modulation of the Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by miR-200a inhibitor in SGC7901 and U251 cells. (A) Knockdown of miR-200a expression resulted in the induction of TopFlash luciferase activity, indicating the transcriptional activity of the -catenin/TCF complex. Knockdown of miR-200a expression had no effect on FopFlash luciferase activity. (B) Western blot analysis of -catenin and TCF-4 in cells treated with miR-200a inhibitor. (C) Lower levels of miR-200a cause nuclear accumulation of -catenin in both SGC7901 and U251 cells, while levels of TCF-4 increased in the nucleus compared to control vector.</div></p></div><div class="aux"><div class="resc">JUAN SU, et al. Int J Oncol. 2012 April;40(4):1162-1170.</div><p class="links"><a class="dblinks" href="/pubmed/22211245" ref="ordinalpos=1670&amp;ncbi_uid=4686865&amp;link_uid=22211245">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584589/" ref="ordinalpos=1670&amp;ncbi_uid=4686865&amp;link_uid=PMC3584589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1671.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC60047/figure/F1/" ref="ordinalpos=1671&amp;ncbi_uid=133689&amp;link_uid=PMC60047" href="/pmc/articles/PMC60047/"><img src="/pmc/articles/instance/60047/bin/pq2314813001.gif" src-large="/pmc/articles/instance/60047/bin/pq2314813001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC60047/" ref="ordinalpos=1671&amp;ncbi_uid=133689&amp;link_uid=PMC60047" image-link="/pmc/articles/PMC60047/figure/F1/" class="imagepopup">Figure 1.  From: Crosstalk of the group IIa and IIb metals calcium and zinc in cellular <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The effect of extracellular zinc on cellular calcium <span class="highlight" style="background-color:">signaling</span>. (<span class="highlight" style="background-color:">Pathway</span> A), Mechanism proposed by Hershfinkel <i>et al.</i> (5). (<span class="highlight" style="background-color:">Pathway</span> B), Alternative <span class="highlight" style="background-color:">pathway</span> through control of protein tyrosine kinases and phosphatases by zinc. Both pathways lead to phospholipase C activation, mobilization of intracellular calcium, and activation of the Na<sup>+</sup>/H<sup>+</sup> exchanger. Zinc exporters and importers control the transport of zinc between the extracellular space and the cytoplasm.</div></p></div><div class="aux"><div class="resc">Wolfgang Maret. Proc Natl Acad Sci U S A. 2001 October 23;98(22):12325-12327.</div><p class="links"><a class="dblinks" href="/pubmed/11675482" ref="ordinalpos=1671&amp;ncbi_uid=133689&amp;link_uid=11675482">Citation</a><a class="dblinks" href="/pmc/articles/PMC60047/" ref="ordinalpos=1671&amp;ncbi_uid=133689&amp;link_uid=PMC60047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1672.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4075464/figure/F6/" ref="ordinalpos=1672&amp;ncbi_uid=6308208&amp;link_uid=PMC4075464" href="/pmc/articles/PMC4075464/"><img src="/pmc/articles/instance/4075464/bin/nihms586796f6.gif" src-large="/pmc/articles/instance/4075464/bin/nihms586796f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4075464/" ref="ordinalpos=1672&amp;ncbi_uid=6308208&amp;link_uid=PMC4075464" image-link="/pmc/articles/PMC4075464/figure/F6/" class="imagepopup">Figure 6. Schematic presentation of relations between Rho and p38 MAPK <span class="highlight" style="background-color:">signaling</span> leading to barrier dysfunction in human lung microvascular and macrovascular endothelium.  From: Cell-type specific crosstalk between p38 MAPK and Rho <span class="highlight" style="background-color:">signaling</span> in lung micro- and macrovascular barrier dysfunction induced by Staphylococcus aureus-derived pathogens. </a></p><div class="supp"><p class="details"><div>Stimulation of both, microvascular and macrovascular EC, with LTA/PepG triggers Rho, p38 MAPK and NFB <span class="highlight" style="background-color:">signaling</span> leading to cytoskeletal remodeling, stress fiber formation and endothelial barrier dysfunction. In HPAEC, LTA/PepG-induced p38 MAPK activation is attenuated by inhibition of Rho kinase, while activation of Rho kinase is not affected by inhibition of p38 MAPK. In contrast, the Rho - p38 MAPK crosstalk in microvascular EC is bi-directional, as inhibition of either <span class="highlight" style="background-color:">pathway</span> affected its counterpart. Finally, both Rho and p38 MAPK pathways modulate LTA/PepG-induced activation of NFB <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tinghuai Wu, et al. Transl Res. ;162(1):45-55.</div><p class="links"><a class="dblinks" href="/pubmed/23571093" ref="ordinalpos=1672&amp;ncbi_uid=6308208&amp;link_uid=23571093">Citation</a><a class="dblinks" href="/pmc/articles/PMC4075464/" ref="ordinalpos=1672&amp;ncbi_uid=6308208&amp;link_uid=PMC4075464">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1673.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3445623/figure/pone-0044973-g002/" ref="ordinalpos=1673&amp;ncbi_uid=4271846&amp;link_uid=PMC3445623" href="/pmc/articles/PMC3445623/"><img src="/pmc/articles/instance/3445623/bin/pone.0044973.g002.gif" src-large="/pmc/articles/instance/3445623/bin/pone.0044973.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3445623/" ref="ordinalpos=1673&amp;ncbi_uid=4271846&amp;link_uid=PMC3445623" image-link="/pmc/articles/PMC3445623/figure/pone-0044973-g002/" class="imagepopup">Figure 2. The estrogen receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> identified by the Ingenuity software..  From: Gene Expression and Functional Annotation of the Human Ciliary Body Epithelia. </a></p><div class="supp"><p class="details"><div>This is one of the canonical pathways that contained statistically significantly more genes than expected by chance in the group of genes that are highly expressed in the non-pigmented epithelium (NPE) of the ciliary body. Gray fields indicate their presence among the highest expressed genes of the NPE; uncolored genes are added by the Ingenuity software to form the <span class="highlight" style="background-color:">pathway</span>. Solid lines between molecules indicate direct physical relationships between molecules (such as regulating and interacting protein dom;ains). Abbreviations of gene names are according to standard abbreviations used in Genbank. This <span class="highlight" style="background-color:">pathway</span> is an example of the endocrine <span class="highlight" style="background-color:">signaling</span> pathways found in the NPE.</div></p></div><div class="aux"><div class="resc">Sarah F. Janssen, et al. PLoS One. 2012;7(9):e44973.</div><p class="links"><a class="dblinks" href="/pubmed/23028713" ref="ordinalpos=1673&amp;ncbi_uid=4271846&amp;link_uid=23028713">Citation</a><a class="dblinks" href="/pmc/articles/PMC3445623/" ref="ordinalpos=1673&amp;ncbi_uid=4271846&amp;link_uid=PMC3445623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1674.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3180925/figure/Fig1/" ref="ordinalpos=1674&amp;ncbi_uid=3287583&amp;link_uid=PMC3180925" href="/pmc/articles/PMC3180925/"><img src="/pmc/articles/instance/3188288/bin/11689_2011_9083_Fig1_HTML.gif" src-large="/pmc/articles/instance/3188288/bin/11689_2011_9083_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3180925/" ref="ordinalpos=1674&amp;ncbi_uid=3287583&amp;link_uid=PMC3180925" image-link="/pmc/articles/PMC3180925/figure/Fig1/" class="imagepopup">Fig. 1.  From: Synaptic Wnt <span class="highlight" style="background-color:">signaling</span>--a contributor to major psychiatric disorders?. </a></p><div class="supp"><p class="details"><div>Wnts signal through a canonical <span class="highlight" style="background-color:">pathway</span> involving -catenin mediated transcriptional regulation (at <i>left</i>) and through non-canonical pathways involving calcium and Rac/JNK-mediated <span class="highlight" style="background-color:">signaling</span> (at <i>right</i>). In the absence of Wnt, the -catenin destruction complex, whose core members are the scaffold proteins axin and adenomatous polyposis coli (<i>APC</i>) and the protein kinase glycogen synthase kinase 3 (<i>GSK3</i>), binds and phosphorylates -catenin. Phospho--catenin is then ubiquitinated and destroyed in the proteosome. Wnt binding to Fzd and LRP5/6 activates Disheveled, which breaks up the -catenin destruction complex. This leads to accumulation of -catenin in the cytosol and translocation of -catenin into the nucleus, where it forms complexes with LEF/TCF transcription factors to promote transcription of selected gene targets. Wnt binding to Fzd can also trigger the Wnt/calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving calcium-dependent activation of CamK and PKC, or to the Wnt/PCP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> which leads to regulation of small GTPases and JNK. Wnt <span class="highlight" style="background-color:">signaling</span> is inhibited extracellularly by Dkk proteins, which specifically bind to LRP5/6 and thereby antagonize only Wnt/-catenin <span class="highlight" style="background-color:">signaling</span>, or by secreted Frizzled-related proteins (<i>SFRP</i>s) and Wnt inhibitory factor (<i>WIF</i>), which bind Wnts extracellularly to antagonize all Wnt-activated <span class="highlight" style="background-color:">signaling</span> pathways</div></p></div><div class="aux"><div class="resc">Nathan D. Okerlund, et al. J Neurodev Disord. 2011 June;3(2):162-174.</div><p class="links"><a class="dblinks" href="/pubmed/21533542" ref="ordinalpos=1674&amp;ncbi_uid=3287583&amp;link_uid=21533542">Citation</a><a class="dblinks" href="/pmc/articles/PMC3180925/" ref="ordinalpos=1674&amp;ncbi_uid=3287583&amp;link_uid=PMC3180925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1675.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3154890/figure/F8/" ref="ordinalpos=1675&amp;ncbi_uid=3306201&amp;link_uid=PMC3154890" href="/pmc/articles/PMC3154890/"><img src="/pmc/articles/instance/3154890/bin/2957fig8.gif" src-large="/pmc/articles/instance/3154890/bin/2957fig8.jpg" alt="FIGURE 8:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3154890/" ref="ordinalpos=1675&amp;ncbi_uid=3306201&amp;link_uid=PMC3154890" image-link="/pmc/articles/PMC3154890/figure/F8/" class="imagepopup">FIGURE 8:.  From: Dystroglycan is involved in skin morphogenesis downstream of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic model of Dg function in epidermal ectoderm differentiation. The data established that the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activates <i>X</i>-<i>dg</i> transcription in inner cell precursors of the epidermal ectoderm. As a result, the inner cell precursors set up the laminin in the ECM, allowing the access of ligands to their receptors. This activates a cascade of events that promotes the intercalation of the precursors of ciliated cells and the expression of P63, which are required for epidermal differentiation.</div></p></div><div class="aux"><div class="resc">Cathy Sirour, et al. Mol Biol Cell. 2011 August 15;22(16):2957-2969.</div><p class="links"><a class="dblinks" href="/pubmed/21680717" ref="ordinalpos=1675&amp;ncbi_uid=3306201&amp;link_uid=21680717">Citation</a><a class="dblinks" href="/pmc/articles/PMC3154890/" ref="ordinalpos=1675&amp;ncbi_uid=3306201&amp;link_uid=PMC3154890">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1676.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2013721/figure/fig4/" ref="ordinalpos=1676&amp;ncbi_uid=1087737&amp;link_uid=PMC2013721" href="/pmc/articles/PMC2013721/"><img src="/pmc/articles/instance/2013721/bin/GEN1771631f4.gif" src-large="/pmc/articles/instance/2013721/bin/GEN1771631f4.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2013721/" ref="ordinalpos=1676&amp;ncbi_uid=1087737&amp;link_uid=PMC2013721" image-link="/pmc/articles/PMC2013721/figure/fig4/" class="imagepopup">F igure  4..  From: Toward a Systems Biology of Mouse Inner Ear Organogenesis: Gene Expression Pathways, Patterns and Network Analysis. </a></p><div class="supp"><p class="details"><div>Examples of pathways deemed statistically significant by IPA within at least one expression pattern resulting from at least one analysis (see Table 2). To be deemed significant, the <span class="highlight" style="background-color:">pathway</span> is represented by at least two genes differentially expressed by 2-fold (for patterns of EML analysis) or 1.5-fold (for patterns of M and L analyses) with a Fisher's right-tailed exact test <i>P</i>-value of 0.05. The horizontal axis shows the percentage of genes within the <span class="highlight" style="background-color:">pathway</span> that varied in expression within a particular pattern of gene expression. For each <span class="highlight" style="background-color:">pathway</span> (AG), the total number of genes (listed by Ingenuity) in the <span class="highlight" style="background-color:">pathway</span> and the number of <span class="highlight" style="background-color:">pathway</span> genes actually on the gene chip are shown in parentheses next to the <span class="highlight" style="background-color:">pathway</span> name. For example, in Notch <span class="highlight" style="background-color:">signaling</span> (E) a total of 36 genes are listed by Ingenuity, of which 32 were on the gene chip. The M pattern of gene expression shows differential expression of 2 notch <span class="highlight" style="background-color:">pathway</span> genes (6.25% of 32 genes). The E pattern exhibits differential expression of only 1 notch <span class="highlight" style="background-color:">pathway</span> gene (3.1% of 32 genes) and is thus not deemed significantly enriched for this <span class="highlight" style="background-color:">pathway</span>. Refer to supplemental Figure 5 for a similar depiction of all 53 pathways identified in this study.</div></p></div><div class="aux"><div class="resc">Samin A. Sajan, et al. Genetics. 2007 September;177(1):631-653.</div><p class="links"><a class="dblinks" href="/pubmed/17660535" ref="ordinalpos=1676&amp;ncbi_uid=1087737&amp;link_uid=17660535">Citation</a><a class="dblinks" href="/pmc/articles/PMC2013721/" ref="ordinalpos=1676&amp;ncbi_uid=1087737&amp;link_uid=PMC2013721">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1677.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2781077/figure/F6/" ref="ordinalpos=1677&amp;ncbi_uid=2154468&amp;link_uid=PMC2781077" href="/pmc/articles/PMC2781077/"><img src="/pmc/articles/instance/2781077/bin/tlo0204_0258_fig006.gif" src-large="/pmc/articles/instance/2781077/bin/tlo0204_0258_fig006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2781077/" ref="ordinalpos=1677&amp;ncbi_uid=2154468&amp;link_uid=PMC2781077" image-link="/pmc/articles/PMC2781077/figure/F6/" class="imagepopup">Figure 6.  From: Renal Carcinogenesis After Uninephrectomy. </a></p><div class="supp"><p class="details"><div>Proposed IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in renal carcinogenesis induced by uninephrectomy. Down-regulation of IGFBP-3 in IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> contrasting with up-regulation of Akt/PKB and PKC expression may enhance cell proliferation, malignant transformation, cancer growth, and invasion.</div></p></div><div class="aux"><div class="resc">Yi Sui, et al. Transl Oncol. 2009 December 1;2(4):258-263.</div><p class="links"><a class="dblinks" href="/pubmed/19956387" ref="ordinalpos=1677&amp;ncbi_uid=2154468&amp;link_uid=19956387">Citation</a><a class="dblinks" href="/pmc/articles/PMC2781077/" ref="ordinalpos=1677&amp;ncbi_uid=2154468&amp;link_uid=PMC2781077">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1678.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4112623/figure/F1/" ref="ordinalpos=1678&amp;ncbi_uid=6463357&amp;link_uid=PMC4112623" href="/pmc/articles/PMC4112623/"><img src="/pmc/articles/instance/4112623/bin/2001-1326-3-25-1.gif" src-large="/pmc/articles/instance/4112623/bin/2001-1326-3-25-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4112623/" ref="ordinalpos=1678&amp;ncbi_uid=6463357&amp;link_uid=PMC4112623" image-link="/pmc/articles/PMC4112623/figure/F1/" class="imagepopup">Figure 1.  From: The role of the Hippo <span class="highlight" style="background-color:">pathway</span> in human disease and tumorigenesis. </a></p><div class="supp"><p class="details"><div><b>Schematic of the core Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in </b><b><i>Drosophila </i></b><b>and humans.</b> Human molecules are in bold, <i>Drosophila</i> molecules in parentheses. Regulators, which act to restrict YAP/TAZ/Yki activation, are blue, and downstream transcriptional effectors are red.</div></p></div><div class="aux"><div class="resc">Daniel A Barron, et al. Clin Transl Med. 2014;3:25-25.</div><p class="links"><a class="dblinks" href="/pubmed/25097728" ref="ordinalpos=1678&amp;ncbi_uid=6463357&amp;link_uid=25097728">Citation</a><a class="dblinks" href="/pmc/articles/PMC4112623/" ref="ordinalpos=1678&amp;ncbi_uid=6463357&amp;link_uid=PMC4112623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1679.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3086058/figure/F1/" ref="ordinalpos=1679&amp;ncbi_uid=2931600&amp;link_uid=PMC3086058" href="/pmc/articles/PMC3086058/"><img src="/pmc/articles/instance/3086058/bin/nihms286402f1.gif" src-large="/pmc/articles/instance/3086058/bin/nihms286402f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3086058/" ref="ordinalpos=1679&amp;ncbi_uid=2931600&amp;link_uid=PMC3086058" image-link="/pmc/articles/PMC3086058/figure/F1/" class="imagepopup">Fig. 1.  From: Vascular endothelial growth factor <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Representation of genes of the vascular endothelial growth factor (VEGF) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the sites at which bevacizumab, sorafenib, sunitinib, brivanib, and cilengitide are known to act.</div></p></div><div class="aux"><div class="resc">Michael L. Maitland, et al. Pharmacogenet Genomics. ;20(5):346-349.</div><p class="links"><a class="dblinks" href="/pubmed/20124951" ref="ordinalpos=1679&amp;ncbi_uid=2931600&amp;link_uid=20124951">Citation</a><a class="dblinks" href="/pmc/articles/PMC3086058/" ref="ordinalpos=1679&amp;ncbi_uid=2931600&amp;link_uid=PMC3086058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1680.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3569903/figure/f01/" ref="ordinalpos=1680&amp;ncbi_uid=4645119&amp;link_uid=PMC3569903" href="/pmc/articles/PMC3569903/"><img src="/pmc/articles/instance/3569903/bin/tre-02-51-249-1-g001.gif" src-large="/pmc/articles/instance/3569903/bin/tre-02-51-249-1-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3569903/" ref="ordinalpos=1680&amp;ncbi_uid=4645119&amp;link_uid=PMC3569903" image-link="/pmc/articles/PMC3569903/figure/f01/" class="imagepopup">Figure 1.Hypothetical mechanisms for LINGO-1 induced cerebellum Purkinje cell degeneration in ET. LINGO-1 might contribute to Purkinje cell degeneration in the cerebellum via regulation of the activity of the NgR1p75 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and EGFRPI3KAkt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>:.  From: LINGO-1 and Neurodegeneration: Pathophysiologic Clues for Essential Tremor. </a></p><div class="aux"><div class="resc">Zhi-dong Zhou, et al. Tremor Other Hyperkinet Mov (N Y). 2012;2:tre-02-51-249-1.</div><p class="links"><a class="dblinks" href="/pubmed/23439882" ref="ordinalpos=1680&amp;ncbi_uid=4645119&amp;link_uid=23439882">Citation</a><a class="dblinks" href="/pmc/articles/PMC3569903/" ref="ordinalpos=1680&amp;ncbi_uid=4645119&amp;link_uid=PMC3569903">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1681.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2881135/figure/fig4/" ref="ordinalpos=1681&amp;ncbi_uid=2605627&amp;link_uid=PMC2881135" href="/pmc/articles/PMC2881135/"><img src="/pmc/articles/instance/2881135/bin/GEN1852537f4.gif" src-large="/pmc/articles/instance/2881135/bin/GEN1852537f4.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2881135/" ref="ordinalpos=1681&amp;ncbi_uid=2605627&amp;link_uid=PMC2881135" image-link="/pmc/articles/PMC2881135/figure/fig4/" class="imagepopup">F igure  4..  From: Caenorhabditis elegans Fibroblast Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span> Can Occur Independently of the Multi-Substrate Adaptor FRS2. </a></p><div class="supp"><p class="details"><div>Comparison of FGFR and FRS2 <span class="highlight" style="background-color:">signaling</span> complexes in vertebrates and <i>C. elegans</i>. (A) The interaction between the vertebrate FGF receptor and Grb2 requires the <span class="highlight" style="background-color:">signaling</span> adaptor protein, FRS2. (B) The <i>C. elegans</i> FGF receptor EGL-15 can bind SEM-5 directly at Y<sup>1009</sup> and Y<sup>1087</sup>, which are required for SM chemoattraction. (C) The <i>C. elegans</i> FRS2 homolog, ROG-1, is involved in a RAS/MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that is required for germline progression, but does not involve EGL-15.</div></p></div><div class="aux"><div class="resc">Te-Wen Lo, et al. Genetics. 2010 June;185(2):537-547.</div><p class="links"><a class="dblinks" href="/pubmed/20308281" ref="ordinalpos=1681&amp;ncbi_uid=2605627&amp;link_uid=20308281">Citation</a><a class="dblinks" href="/pmc/articles/PMC2881135/" ref="ordinalpos=1681&amp;ncbi_uid=2605627&amp;link_uid=PMC2881135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1682.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3767831/figure/pone-0073341-g005/" ref="ordinalpos=1682&amp;ncbi_uid=5304133&amp;link_uid=PMC3767831" href="/pmc/articles/PMC3767831/"><img src="/pmc/articles/instance/3767831/bin/pone.0073341.g005.gif" src-large="/pmc/articles/instance/3767831/bin/pone.0073341.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3767831/" ref="ordinalpos=1682&amp;ncbi_uid=5304133&amp;link_uid=PMC3767831" image-link="/pmc/articles/PMC3767831/figure/pone-0073341-g005/" class="imagepopup">Figure 5. The comparative expression of different members of Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in parental cells and spheroid cells..  From: Epithelial-Mesenchymal Transition Associates with Maintenance of Stemness in Spheroid-Derived Stem-Like Colon Cancer Cells. </a></p><div class="supp"><p class="details"><div>The Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> is constitutively activated in spheroid cells, including up-regulated expression of Vimentin, Slug, Snail, and nuclear -catenin, but down-regulated expression of E-cadherin and GSK3, comparing with the corresponding parental cells.</div></p></div><div class="aux"><div class="resc">Xiao-Yan Han, et al. PLoS One. 2013;8(9):e73341.</div><p class="links"><a class="dblinks" href="/pubmed/24039918" ref="ordinalpos=1682&amp;ncbi_uid=5304133&amp;link_uid=24039918">Citation</a><a class="dblinks" href="/pmc/articles/PMC3767831/" ref="ordinalpos=1682&amp;ncbi_uid=5304133&amp;link_uid=PMC3767831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1683.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4325629/figure/f2-sccaa-8-027/" ref="ordinalpos=1683&amp;ncbi_uid=7184652&amp;link_uid=PMC4325629" href="/pmc/articles/PMC4325629/"><img src="/pmc/articles/instance/4325629/bin/sccaa-8-027Fig2.gif" src-large="/pmc/articles/instance/4325629/bin/sccaa-8-027Fig2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4325629/" ref="ordinalpos=1683&amp;ncbi_uid=7184652&amp;link_uid=PMC4325629" image-link="/pmc/articles/PMC4325629/figure/f2-sccaa-8-027/" class="imagepopup">Figure 2.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> in cancer stem cells: a focus on hematological cancers. </a></p><div class="supp"><p class="details"><div>Non-canonical Hh <span class="highlight" style="background-color:">signaling</span>.<br /><b>Note:</b> The Hh <span class="highlight" style="background-color:">pathway</span> can be activated directly through PTCH1 or SMO, or via alternative pathways including the PI3K/AKT and RAS/RAF/MEK/ERK <span class="highlight" style="background-color:">signaling</span> cascades.<br /><b>Abbreviations:</b> Hh, hedgehog; EMT, epithelial-mesenchymal transition; PTCH1, Patched 1; SMO, Smoothened.</div></p></div><div class="aux"><div class="resc">Victoria Campbell, et al. Stem Cells Cloning. 2015;8:27-38.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1683&amp;ncbi_uid=7184652&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4325629/" ref="ordinalpos=1683&amp;ncbi_uid=7184652&amp;link_uid=PMC4325629">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1684.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4229655/figure/F7/" ref="ordinalpos=1684&amp;ncbi_uid=6830546&amp;link_uid=PMC4229655" href="/pmc/articles/PMC4229655/"><img src="/pmc/articles/instance/4229655/bin/nihms573356f7.gif" src-large="/pmc/articles/instance/4229655/bin/nihms573356f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4229655/" ref="ordinalpos=1684&amp;ncbi_uid=6830546&amp;link_uid=PMC4229655" image-link="/pmc/articles/PMC4229655/figure/F7/" class="imagepopup">Figure 7.  From: Cell-to-cell <span class="highlight" style="background-color:">signaling</span> in E. coli and Salmonella. </a></p><div class="supp"><p class="details"><div>The LuxS/AI-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <i>lsrACDBFGE</i> genes are transcribed in an operon, whereas <i>lsrK</i> and <i>lsrR</i> are divergently transcribed. LsrB binds AI-2 which is then imported by the Lsr ABC transport system. Inside the cell, AI-2 is phosphorylated by LsrK and is thought to interact with LsrR, relieving the repression of the <i>lsr</i> operon.</div></p></div><div class="aux"><div class="resc">Melissa M. Kendall, et al. Ecosal Plus. ;2014:10.1128/ecosalplus.ESP-0002-2013.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1684&amp;ncbi_uid=6830546&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4229655/" ref="ordinalpos=1684&amp;ncbi_uid=6830546&amp;link_uid=PMC4229655">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1685.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3573771/figure/f5-ijmm-30-04-0734/" ref="ordinalpos=1685&amp;ncbi_uid=4615813&amp;link_uid=PMC3573771" href="/pmc/articles/PMC3573771/"><img src="/pmc/articles/instance/3573771/bin/IJMM-30-04-0734-g04.gif" src-large="/pmc/articles/instance/3573771/bin/IJMM-30-04-0734-g04.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3573771/" ref="ordinalpos=1685&amp;ncbi_uid=4615813&amp;link_uid=PMC3573771" image-link="/pmc/articles/PMC3573771/figure/f5-ijmm-30-04-0734/" class="imagepopup">Figure 5.  From: Induction of apoptosis by laminarin, regulating the insulin-like growth factor-IR <span class="highlight" style="background-color:">signaling</span> pathways in HT-29 human colon cells. </a></p><div class="supp"><p class="details"><div>Laminarin affects the protein expression of the IGF-IR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HT-29 cells. Cells were treated with laminarin (05 mg/ml) for 24 h. Proteins were visualized by western blotting.</div></p></div><div class="aux"><div class="resc">HEE-KYOUNG PARK, et al. Int J Mol Med. 2012 October;30(4):734-738.</div><p class="links"><a class="dblinks" href="/pubmed/22859258" ref="ordinalpos=1685&amp;ncbi_uid=4615813&amp;link_uid=22859258">Citation</a><a class="dblinks" href="/pmc/articles/PMC3573771/" ref="ordinalpos=1685&amp;ncbi_uid=4615813&amp;link_uid=PMC3573771">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1686.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3680918/figure/F6/" ref="ordinalpos=1686&amp;ncbi_uid=5024626&amp;link_uid=PMC3680918" href="/pmc/articles/PMC3680918/"><img src="/pmc/articles/instance/3680918/bin/bcr3232-6.gif" src-large="/pmc/articles/instance/3680918/bin/bcr3232-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3680918/" ref="ordinalpos=1686&amp;ncbi_uid=5024626&amp;link_uid=PMC3680918" image-link="/pmc/articles/PMC3680918/figure/F6/" class="imagepopup">Figure 6.  From: Prolactin-induced protein mediates cell invasion and regulates integrin <span class="highlight" style="background-color:">signaling</span> in estrogen receptor-negative breast cancer. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the PIP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ER-negative breast cancer</b>. Red arrow denotes stimulatory effect. ER, estrogen receptor; Fn, fibronectin; Fn-f, Fibronectin fragment; ITG-1, integrin-1.</div></p></div><div class="aux"><div class="resc">Ali Naderi, et al. Breast Cancer Res. 2012;14(4):R111-R111.</div><p class="links"><a class="dblinks" href="/pubmed/22817771" ref="ordinalpos=1686&amp;ncbi_uid=5024626&amp;link_uid=22817771">Citation</a><a class="dblinks" href="/pmc/articles/PMC3680918/" ref="ordinalpos=1686&amp;ncbi_uid=5024626&amp;link_uid=PMC3680918">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1687.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2810260/figure/F6/" ref="ordinalpos=1687&amp;ncbi_uid=2137044&amp;link_uid=PMC2810260" href="/pmc/articles/PMC2810260/"><img src="/pmc/articles/instance/2810260/bin/nihms-169743-f0006.gif" src-large="/pmc/articles/instance/2810260/bin/nihms-169743-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2810260/" ref="ordinalpos=1687&amp;ncbi_uid=2137044&amp;link_uid=PMC2810260" image-link="/pmc/articles/PMC2810260/figure/F6/" class="imagepopup">Figure 6.  From: Involvement of Notch1 <span class="highlight" style="background-color:">signaling</span> in neurogenesis in the subventricular zone of normal and ischemic rat brain in vivo. </a></p><div class="supp"><p class="details"><div>Schematic representation of Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in ischemic regulation of cell proliferation in adult rat brain SVZ.</div></p></div><div class="aux"><div class="resc">XiaoMei Wang, et al. J Cereb Blood Flow Metab. ;29(10):1644-1654.</div><p class="links"><a class="dblinks" href="/pubmed/19536070" ref="ordinalpos=1687&amp;ncbi_uid=2137044&amp;link_uid=19536070">Citation</a><a class="dblinks" href="/pmc/articles/PMC2810260/" ref="ordinalpos=1687&amp;ncbi_uid=2137044&amp;link_uid=PMC2810260">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1688.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3762983/figure/F2/" ref="ordinalpos=1688&amp;ncbi_uid=5274422&amp;link_uid=PMC3762983" href="/pmc/articles/PMC3762983/"><img src="/pmc/articles/instance/3762983/bin/nihms31675f2.gif" src-large="/pmc/articles/instance/3762983/bin/nihms31675f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3762983/" ref="ordinalpos=1688&amp;ncbi_uid=5274422&amp;link_uid=PMC3762983" image-link="/pmc/articles/PMC3762983/figure/F2/" class="imagepopup">Fig. 2.  From: The future of pancreatology: how to go where we have never been before. </a></p><div class="supp"><p class="details"><div>Example of a <span class="highlight" style="background-color:">pathway</span> reconstruction experiment. KLF11 targets in Panc1 cells were identified using <span class="highlight" style="background-color:">pathway</span>-specific arrays. Subsequently, a <span class="highlight" style="background-color:">pathway</span> was reconstructed using software that processes the data using semantic algorithms to search for available knowledge in literature databases (<span class="highlight" style="background-color:">Pathway</span> Assist; http://www.stratagene.com). The <span class="highlight" style="background-color:">pathway</span> depicted show interactions among proteins from a subset of proteins involved in transforming growth factor- <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Raul Urrutia. Am J Surg. 2007 October;194(4 0):S153-S157.</div><p class="links"><a class="dblinks" href="/pubmed/17903433" ref="ordinalpos=1688&amp;ncbi_uid=5274422&amp;link_uid=17903433">Citation</a><a class="dblinks" href="/pmc/articles/PMC3762983/" ref="ordinalpos=1688&amp;ncbi_uid=5274422&amp;link_uid=PMC3762983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1689.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3994297/figure/F8/" ref="ordinalpos=1689&amp;ncbi_uid=6045306&amp;link_uid=PMC3994297" href="/pmc/articles/PMC3994297/"><img src="/pmc/articles/instance/3994297/bin/kjpp-18-109-g008.gif" src-large="/pmc/articles/instance/3994297/bin/kjpp-18-109-g008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3994297/" ref="ordinalpos=1689&amp;ncbi_uid=6045306&amp;link_uid=PMC3994297" image-link="/pmc/articles/PMC3994297/figure/F8/" class="imagepopup">Fig. 8.  From: Multiple Effects of a Novel Epothilone Analog on Cellular Processes and <span class="highlight" style="background-color:">Signaling</span> Pathways Regulated by Rac1 GTPase in the Human Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div>A proposed model for <span class="highlight" style="background-color:">signaling</span> pathways affected by UTD2 in breast cancer cells. By targeting tubulin and promoting tubulin polymerization, UTD2 inhibits the Rac1/PAK1/p38/MMP-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is critical for the UTD2-induced actin cytoskeleton dysfunction. Meanwhile, the Rac1/PI3K/Akt/MMP-2 <span class="highlight" style="background-color:">pathway</span> may be also involved in UTD2-induced actin cytoskeleton dysfunction. In addition, represented by a thinner arrow in this model, UTD2 can moderately inhibit the Raf-MEK-ERK <span class="highlight" style="background-color:">pathway</span>. Together with the effect on survival factor Akt, these account for at least part of UTD2's antitumor activities.</div></p></div><div class="aux"><div class="resc">Hong Zhang, et al. Korean J Physiol Pharmacol. 2014 April;18(2):109-120.</div><p class="links"><a class="dblinks" href="/pubmed/24757372" ref="ordinalpos=1689&amp;ncbi_uid=6045306&amp;link_uid=24757372">Citation</a><a class="dblinks" href="/pmc/articles/PMC3994297/" ref="ordinalpos=1689&amp;ncbi_uid=6045306&amp;link_uid=PMC3994297">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1690.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2923187/figure/pone-0012222-g008/" ref="ordinalpos=1690&amp;ncbi_uid=2438094&amp;link_uid=PMC2923187" href="/pmc/articles/PMC2923187/"><img src="/pmc/articles/instance/2923187/bin/pone.0012222.g008.gif" src-large="/pmc/articles/instance/2923187/bin/pone.0012222.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2923187/" ref="ordinalpos=1690&amp;ncbi_uid=2438094&amp;link_uid=PMC2923187" image-link="/pmc/articles/PMC2923187/figure/pone-0012222-g008/" class="imagepopup">Figure 8. Functional <span class="highlight" style="background-color:">pathway</span> analysis of the 12 signature genes..  From: Hybrid Models Identified a 12-Gene Signature for Lung Cancer Prognosis and Chemoresponse Prediction. </a></p><div class="supp"><p class="details"><div>(A) Using core analysis from Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA), curated interactions were revealed among the identified signature genes and major lung cancer <span class="highlight" style="background-color:">signaling</span> pathways. (B) Six of the 12 signature genes also exhibited various curated interactions with eight prominent lung cancer hallmarks with <span class="highlight" style="background-color:">Pathway</span> Studio 7.0.</div></p></div><div class="aux"><div class="resc">Ying-Wooi Wan, et al. PLoS One. 2010;5(8):e12222.</div><p class="links"><a class="dblinks" href="/pubmed/20808922" ref="ordinalpos=1690&amp;ncbi_uid=2438094&amp;link_uid=20808922">Citation</a><a class="dblinks" href="/pmc/articles/PMC2923187/" ref="ordinalpos=1690&amp;ncbi_uid=2438094&amp;link_uid=PMC2923187">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1691.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4069059/figure/pone-0100127-g007/" ref="ordinalpos=1691&amp;ncbi_uid=6302286&amp;link_uid=PMC4069059" href="/pmc/articles/PMC4069059/"><img src="/pmc/articles/instance/4069059/bin/pone.0100127.g007.gif" src-large="/pmc/articles/instance/4069059/bin/pone.0100127.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4069059/" ref="ordinalpos=1691&amp;ncbi_uid=6302286&amp;link_uid=PMC4069059" image-link="/pmc/articles/PMC4069059/figure/pone-0100127-g007/" class="imagepopup">Figure 7. Illustration of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for cisplatin-induced necrosis..  From: Receptor Interactive Protein Kinase 3 Promotes Cisplatin-Triggered Necrosis in Apoptosis-Resistant Esophageal Squamous Cell Carcinoma Cells. </a></p><div class="supp"><p class="details"><div>Cisplatin-induced necrosis requires both innate suppression of the apoptotic <span class="highlight" style="background-color:">pathway</span> and necrosome assembly, including RIPK1, RIPK3 and MLKL, through autocrine production of TNF. The blue ellipses represent the downregulated proteins, and the orange ellipses represent the up-regulated proteins.</div></p></div><div class="aux"><div class="resc">Yang Xu, et al. PLoS One. 2014;9(6):e100127.</div><p class="links"><a class="dblinks" href="/pubmed/24959694" ref="ordinalpos=1691&amp;ncbi_uid=6302286&amp;link_uid=24959694">Citation</a><a class="dblinks" href="/pmc/articles/PMC4069059/" ref="ordinalpos=1691&amp;ncbi_uid=6302286&amp;link_uid=PMC4069059">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1692.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3759480/figure/fig03/" ref="ordinalpos=1692&amp;ncbi_uid=5294371&amp;link_uid=PMC3759480" href="/pmc/articles/PMC3759480/"><img src="/pmc/articles/instance/3759480/bin/ijcep0006-1734-f3.gif" src-large="/pmc/articles/instance/3759480/bin/ijcep0006-1734-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759480/" ref="ordinalpos=1692&amp;ncbi_uid=5294371&amp;link_uid=PMC3759480" image-link="/pmc/articles/PMC3759480/figure/fig03/" class="imagepopup">Figure 3.  From: Array analysis for potential biomarker of gemcitabine identification in non-small cell lung cancer cell lines. </a></p><div class="supp"><p class="details"><div>Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> was activated in gemcitabine resistant NSCLC cell lines. The 875 expression-altered genes were applied to IPA. IPA results showed that Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> was one of the most significantly activated <span class="highlight" style="background-color:">pathway</span> in gemcitabine resistant NSCLC cell lines. <i>P</i> value is 0.002 and 15 genes in the 875 gene-set were involved in this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hai-Hong Zhang, et al. Int J Clin Exp Pathol. 2013;6(9):1734-1746.</div><p class="links"><a class="dblinks" href="/pubmed/24040438" ref="ordinalpos=1692&amp;ncbi_uid=5294371&amp;link_uid=24040438">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759480/" ref="ordinalpos=1692&amp;ncbi_uid=5294371&amp;link_uid=PMC3759480">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1693.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4377510/figure/fig5/" ref="ordinalpos=1693&amp;ncbi_uid=7398083&amp;link_uid=PMC4377510" href="/pmc/articles/PMC4377510/"><img src="/pmc/articles/instance/4377510/bin/SARCOMA2015-839182.005.gif" src-large="/pmc/articles/instance/4377510/bin/SARCOMA2015-839182.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4377510/" ref="ordinalpos=1693&amp;ncbi_uid=7398083&amp;link_uid=PMC4377510" image-link="/pmc/articles/PMC4377510/figure/fig5/" class="imagepopup">Figure 5.  From: Molecular Analysis of a Recurrent Sarcoma Identifies a Mutation in FAF1. </a></p><div class="supp"><p class="details"><div>Possible transforming <span class="highlight" style="background-color:">pathway</span>. The point mutation in FAF1 is believed to cause a loss of function (crossed out in red). This may lead to an overactivity of the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Consistently, E2F1 (activated by LEF1) and<i> igf2bp1</i> (a transcriptional target of the WNT <span class="highlight" style="background-color:">pathway</span>) have been identified to be upregulated in the molecular analysis. In contrast to the negative regulator FAF1, IGF2BP1 is a positive regulator of NF-<i></i>B activity. The overactivity of the NF-<i></i>B <span class="highlight" style="background-color:">pathway</span> and the reduced efficacy of FAS <span class="highlight" style="background-color:">signaling</span> can be transforming.</div></p></div><div class="aux"><div class="resc">Georg F. Weber. Sarcoma. 2015;2015:839182.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1693&amp;ncbi_uid=7398083&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4377510/" ref="ordinalpos=1693&amp;ncbi_uid=7398083&amp;link_uid=PMC4377510">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1694.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3724517/figure/F4/" ref="ordinalpos=1694&amp;ncbi_uid=5131655&amp;link_uid=PMC3724517" href="/pmc/articles/PMC3724517/"><img src="/pmc/articles/instance/3724517/bin/nihms431868f4.gif" src-large="/pmc/articles/instance/3724517/bin/nihms431868f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3724517/" ref="ordinalpos=1694&amp;ncbi_uid=5131655&amp;link_uid=PMC3724517" image-link="/pmc/articles/PMC3724517/figure/F4/" class="imagepopup">Figure 4.  From: RAF265 Inhibits the Growth of Advanced Human Melanoma Tumors. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of differentially expressed genes in RAF265 responding versus nonresponding tumors. Ingenuity software was utilized to recognize pathways connecting genes that were differentially expressed in RAF265 responding versus nonresponding patients. A, <span class="highlight" style="background-color:">Pathway</span> analysis of genes involved in cell development and growth regulation that are differentially expressed in BRAFWT responding tumors to RAF265 versus those not responding to RAF265. B, <span class="highlight" style="background-color:">Pathway</span> analysis of genes involved in cell <span class="highlight" style="background-color:">signaling</span>, nucleotide metabolism, and small molecule biochemistry that are differentially expressed in BRAFWT tumors responding to RAF265 versus those not responding to RAF265. C, <span class="highlight" style="background-color:">Pathway</span> analysis of genes involved in cell <span class="highlight" style="background-color:">signaling</span> interactions that are differentially expressed in RAF265 responding versus nonresponding tumors, irrespective of BRAF mutation status.</div></p></div><div class="aux"><div class="resc">Yingjun Su, et al. Clin Cancer Res. ;18(8):2184-2198.</div><p class="links"><a class="dblinks" href="/pubmed/22351689" ref="ordinalpos=1694&amp;ncbi_uid=5131655&amp;link_uid=22351689">Citation</a><a class="dblinks" href="/pmc/articles/PMC3724517/" ref="ordinalpos=1694&amp;ncbi_uid=5131655&amp;link_uid=PMC3724517">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1695.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2795491/figure/F6/" ref="ordinalpos=1695&amp;ncbi_uid=2102586&amp;link_uid=PMC2795491" href="/pmc/articles/PMC2795491/"><img src="/pmc/articles/instance/2795491/bin/zpq9990901380006.gif" src-large="/pmc/articles/instance/2795491/bin/zpq9990901380006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2795491/" ref="ordinalpos=1695&amp;ncbi_uid=2102586&amp;link_uid=PMC2795491" image-link="/pmc/articles/PMC2795491/figure/F6/" class="imagepopup">Fig. 6. From: A protein kinase-phosphatase pair interacts with an ion channel to regulate ABA <span class="highlight" style="background-color:">signaling</span> in plant guard cells. </a></p><div class="supp"><p class="details"><div>A working model for an ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the regulation of stomatal movements. R, ABA receptors. P stands for phosphorylation. Details are described in <i>Concluding Remarks</i>.</div></p></div><div class="aux"><div class="resc">Sung Chul Lee, et al. Proc Natl Acad Sci U S A. 2009 December 15;106(50):21419-21424.</div><p class="links"><a class="dblinks" href="/pubmed/19955427" ref="ordinalpos=1695&amp;ncbi_uid=2102586&amp;link_uid=19955427">Citation</a><a class="dblinks" href="/pmc/articles/PMC2795491/" ref="ordinalpos=1695&amp;ncbi_uid=2102586&amp;link_uid=PMC2795491">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1696.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2862933/figure/F3/" ref="ordinalpos=1696&amp;ncbi_uid=2316747&amp;link_uid=PMC2862933" href="/pmc/articles/PMC2862933/"><img src="/pmc/articles/instance/2862933/bin/nem184f3.gif" src-large="/pmc/articles/instance/2862933/bin/nem184f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2862933/" ref="ordinalpos=1696&amp;ncbi_uid=2316747&amp;link_uid=PMC2862933" image-link="/pmc/articles/PMC2862933/figure/F3/" class="imagepopup">Figure 3. From: Genomic Analysis Highlights the Role of the JAK-STAT <span class="highlight" style="background-color:">Signaling</span> in the Anti-proliferative Effects of Dietary Flavonoid--'Ashwagandha' in Prostate Cancer Cells. </a></p><div class="supp"><p class="details"><div>Schematic representation of the genes regulated by Ashwagandha extract in JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The genes highlighted with an asterisk (*) are significantly regulated by Ashwagandha and are important in the regulation of cell proliferation, regulation of apoptosis and immunomodulation.</div></p></div><div class="aux"><div class="resc">Ravikumar Aalinkeel, et al. Evid Based Complement Alternat Med. 2010 June;7(2):177-187.</div><p class="links"><a class="dblinks" href="/pubmed/18955307" ref="ordinalpos=1696&amp;ncbi_uid=2316747&amp;link_uid=18955307">Citation</a><a class="dblinks" href="/pmc/articles/PMC2862933/" ref="ordinalpos=1696&amp;ncbi_uid=2316747&amp;link_uid=PMC2862933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1697.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1821127/figure/F1/" ref="ordinalpos=1697&amp;ncbi_uid=1010825&amp;link_uid=PMC1821127" href="/pmc/articles/PMC1821127/"><img src="/pmc/articles/instance/1829256/bin/zpq0100756500001.gif" src-large="/pmc/articles/instance/1829256/bin/zpq0100756500001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1821127/" ref="ordinalpos=1697&amp;ncbi_uid=1010825&amp;link_uid=PMC1821127" image-link="/pmc/articles/PMC1821127/figure/F1/" class="imagepopup">Fig. 1. From: A bile acid-like steroid modulates Caenorhabditis elegans lifespan through nuclear receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Dauer and germ-line longevity pathways. <sup>4</sup>-dafachronic acid inhibits longevity in the dauer pathways but promotes it in the germ-line longevity <span class="highlight" style="background-color:">pathway</span>, showing a dependence on <span class="highlight" style="background-color:">signaling</span> context. Germ-line <span class="highlight" style="background-color:">signaling</span> is DAF-16/FOXO-dependent, whereas dauer <span class="highlight" style="background-color:">signaling</span> can be rendered DAF-16/FOXO-independent. Both contexts require DAF-12/nuclear hormone receptor. (<i>A</i>) Dauer longevity. Liganded DAF-12 promotes reproductive development and short life, whereas unliganded DAF-12 together with corepressor DIN-1 promote stress resistance and longevity. (<i>B</i>) Germ-line longevity. In the absence of signals from the germ line, liganded DAF-12, together with nuclear localized DAF-16/FOXO, promote longevity.</div></p></div><div class="aux"><div class="resc">Birgit Gerisch, et al. Proc Natl Acad Sci U S A. 2007 March 20;104(12):5014-5019.</div><p class="links"><a class="dblinks" href="/pubmed/17360327" ref="ordinalpos=1697&amp;ncbi_uid=1010825&amp;link_uid=17360327">Citation</a><a class="dblinks" href="/pmc/articles/PMC1821127/" ref="ordinalpos=1697&amp;ncbi_uid=1010825&amp;link_uid=PMC1821127">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1698.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3117391/figure/F3/" ref="ordinalpos=1698&amp;ncbi_uid=3027332&amp;link_uid=PMC3117391" href="/pmc/articles/PMC3117391/"><img src="/pmc/articles/instance/3117391/bin/btr202f3.gif" src-large="/pmc/articles/instance/3117391/bin/btr202f3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3117391/" ref="ordinalpos=1698&amp;ncbi_uid=3027332&amp;link_uid=PMC3117391" image-link="/pmc/articles/PMC3117391/figure/F3/" class="imagepopup">Fig. 3. From: An enhanced Petri-net model to predict synergistic effects of pairwise drug combinations from gene microarray data. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">pathway</span> for getifinib and docetaxel to generate the synergistic effects on KRT8. The red interactions are for activation or phosphorylation between <span class="highlight" style="background-color:">signaling</span> molecules, while the green ones denote inhibition or dephosphorylation between <span class="highlight" style="background-color:">signaling</span> molecules. And the black interactions are for drugtarget relationships and the simulated <span class="highlight" style="background-color:">pathway</span> interactions from PPIs. The <span class="highlight" style="background-color:">pathway</span> was generated using IPA software (Ingenuity Systems, Inc, Redwood City, CA, USA).</div></p></div><div class="aux"><div class="resc">Guangxu Jin, et al. Bioinformatics. 2011 July 1;27(13):i310-i316.</div><p class="links"><a class="dblinks" href="/pubmed/21685086" ref="ordinalpos=1698&amp;ncbi_uid=3027332&amp;link_uid=21685086">Citation</a><a class="dblinks" href="/pmc/articles/PMC3117391/" ref="ordinalpos=1698&amp;ncbi_uid=3027332&amp;link_uid=PMC3117391">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1699.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405730/figure/F2/" ref="ordinalpos=1699&amp;ncbi_uid=4264568&amp;link_uid=PMC3405730" href="/pmc/articles/PMC3405730/"><img src="/pmc/articles/instance/3405730/bin/nihms389960f2.gif" src-large="/pmc/articles/instance/3405730/bin/nihms389960f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405730/" ref="ordinalpos=1699&amp;ncbi_uid=4264568&amp;link_uid=PMC3405730" image-link="/pmc/articles/PMC3405730/figure/F2/" class="imagepopup">Figure 2.  From: IL-2 Family Cytokines: New Insights into the Complex Roles of IL-2 as a Broad Regulator of T helper Cell Differentiation. </a></p><div class="supp"><p class="details"><div>IL-2 signals by activating three principal <span class="highlight" style="background-color:">signaling</span> pathways, the SHC/RAS/MAP kinase <span class="highlight" style="background-color:">pathway</span>, JAK1/JAK3/STAT5 <span class="highlight" style="background-color:">pathway</span>, and PI 3-K/AKT/p70 S6 kinase <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Wei Liao, et al. Curr Opin Immunol. ;23(5):598-604.</div><p class="links"><a class="dblinks" href="/pubmed/21889323" ref="ordinalpos=1699&amp;ncbi_uid=4264568&amp;link_uid=21889323">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405730/" ref="ordinalpos=1699&amp;ncbi_uid=4264568&amp;link_uid=PMC3405730">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1700.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3594115/figure/F1/" ref="ordinalpos=1700&amp;ncbi_uid=5863146&amp;link_uid=PMC3594115" href="/pmc/articles/PMC3594115/"><img src="/pmc/articles/instance/3594115/bin/nihms434903f1.gif" src-large="/pmc/articles/instance/3594115/bin/nihms434903f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3594115/" ref="ordinalpos=1700&amp;ncbi_uid=5863146&amp;link_uid=PMC3594115" image-link="/pmc/articles/PMC3594115/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Pathway</span>-specific impairment of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and endothelial dysfunction.  From: Role of Insulin Resistance in Endothelial Dysfunction. </a></p><div class="supp"><p class="details"><div>PI 3-kinase branch of insulin <span class="highlight" style="background-color:">signaling</span> regulates NO production and vasodilation in vascular endothelium. MAP-kinase branch of insulin <span class="highlight" style="background-color:">signaling</span> controls secretion of endothelin-1 (ET-1) and adhesion molecule expression in vascular endothelium. Glucotoxicity, lipotoxicity, and various cytokines activate <span class="highlight" style="background-color:">signaling</span> molecules that inhibit PI3K/Akt <span class="highlight" style="background-color:">signaling</span>. eNOS, endothelial nitric oxide synthase; IRS, insulin receptor substrate; MEK, MAPK kinase; PDK, phosphoinositide-dependent protein kinase; PKC, protein kinase C; IRS, insulin receptor substrate; ERK, extracellular signal-regulated kinase; JNK, C-Jun N-terminal kinase; p70S6K, p70 ribosomal S6 kinase; AP-1, activator protein-1; NO, nitric oxide; and ET-1, endothelin-1.</div></p></div><div class="aux"><div class="resc">Ranganath Muniyappa, et al. Rev Endocr Metab Disord. 2013 March;14(1):5-12.</div><p class="links"><a class="dblinks" href="/pubmed/23306778" ref="ordinalpos=1700&amp;ncbi_uid=5863146&amp;link_uid=23306778">Citation</a><a class="dblinks" href="/pmc/articles/PMC3594115/" ref="ordinalpos=1700&amp;ncbi_uid=5863146&amp;link_uid=PMC3594115">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="16" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="17" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="18" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:26:04-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A6F8153FEC8110000000000D52037;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A6F8153FEC8110000000000D52037" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>