
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="16" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A7DD053FEC9A100000000006490B6" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="15" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="16" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="17" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="16" /></div>
            <div><h2 class="result_count">Results: 1501 to 1600 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1501.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2777987/figure/F6/" ref="ordinalpos=1501&amp;ncbi_uid=2500476&amp;link_uid=PMC2777987" href="/pmc/articles/PMC2777987/"><img src="/pmc/articles/instance/2777987/bin/nihms144235f6.gif" src-large="/pmc/articles/instance/2777987/bin/nihms144235f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2777987/" ref="ordinalpos=1501&amp;ncbi_uid=2500476&amp;link_uid=PMC2777987" image-link="/pmc/articles/PMC2777987/figure/F6/" class="imagepopup">Figure 6. Regulation of Cellular Circadian Clock by Components in Insulin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: A Genome-Wide RNAi Screen for Modifiers of the Circadian Clock in Human Cells. </a></p><div class="supp"><p class="details"><div>(A) Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes that impact circadian function. Using the NIH David <span class="highlight" style="background-color:">Pathway</span> Analysis tool, we identified many components of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> represented in our screen hits. Genes that impact clock function and genes regulated by the clock are indicated with colored boxes.<br />(B) Effects of siRNAs against genes involved in insulin <span class="highlight" style="background-color:">signaling</span>. The representative results from the secondary screen using individual siRNA are presented. Data represent the mean ± SD (n = 2).<br />(C) Effects of chemical inhibitors against protein kinases involved in insulin <span class="highlight" style="background-color:">signaling</span>. Bioluminescence rhythms of <i>Bmal1-dLuc</i> cells were monitored in the presence of various concentrations of compounds (8 points of 3-fold dilution series; final 3 nM - 7 µM). Period or phase parameter was plotted against final concentrations of the compound. Data represent the mean ± SD (n = 4). The results for SP600215 and PMA are consistent with our previous observations (Hirota et al., 2008).</div></p></div><div class="aux"><div class="resc">Eric E. Zhang, et al. Cell. ;139(1):199-210.</div><p class="links"><a class="dblinks" href="/pubmed/19765810" ref="ordinalpos=1501&amp;ncbi_uid=2500476&amp;link_uid=19765810">Citation</a><a class="dblinks" href="/pmc/articles/PMC2777987/" ref="ordinalpos=1501&amp;ncbi_uid=2500476&amp;link_uid=PMC2777987">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1502.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938103/figure/F4/" ref="ordinalpos=1502&amp;ncbi_uid=3190175&amp;link_uid=PMC2938103" href="/pmc/articles/PMC2938103/"><img src="/pmc/articles/instance/2938103/bin/zjw0091037100004.gif" src-large="/pmc/articles/instance/2938103/bin/zjw0091037100004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938103/" ref="ordinalpos=1502&amp;ncbi_uid=3190175&amp;link_uid=PMC2938103" image-link="/pmc/articles/PMC2938103/figure/F4/" class="imagepopup">Fig. 4. From: Integrated Phosphoproteomics Analysis of a <span class="highlight" style="background-color:">Signaling</span> Network Governing Nutrient Response and Peroxisome Induction. </a></p><div class="supp"><p class="details"><div><b>Detection of known <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins.</b> Pathways identified in the literature were compared with proteins that were detected by mass spectrometry-based phosphoproteomics analysis.</div></p></div><div class="aux"><div class="resc">Ramsey A. Saleem, et al. Mol Cell Proteomics. 2010 September;9(9):2076-2088.</div><p class="links"><a class="dblinks" href="/pubmed/20395639" ref="ordinalpos=1502&amp;ncbi_uid=3190175&amp;link_uid=20395639">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938103/" ref="ordinalpos=1502&amp;ncbi_uid=3190175&amp;link_uid=PMC2938103">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1503.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3914352/figure/fig10/" ref="ordinalpos=1503&amp;ncbi_uid=5825666&amp;link_uid=PMC3914352" href="/pmc/articles/PMC3914352/"><img src="/pmc/articles/instance/3914352/bin/BMRI2014-686505.010.gif" src-large="/pmc/articles/instance/3914352/bin/BMRI2014-686505.010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3914352/" ref="ordinalpos=1503&amp;ncbi_uid=5825666&amp;link_uid=PMC3914352" image-link="/pmc/articles/PMC3914352/figure/fig10/" class="imagepopup">Figure 10.  From: Network Analysis of Neurodegenerative Disease Highlights a Role of Toll-Like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Flow diagram representing the molecular interactions in the Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (from KEGG database). The Toll-like receptors (TLRs) <span class="highlight" style="background-color:">pathway</span> is enriched with connector proteins, labeled in red. TLRs act as primary sensors that detect a wide variety of microbial components and elicit innate immune responses. The connector proteins found in our study (TRAF6, NF<i>κ</i>B, PI3K-AKT, ERK, IKK<i>ε</i>, and CASP8) are mediators in the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> which control the expression of an array of inflammatory cytokine genes.</div></p></div><div class="aux"><div class="resc">Thanh-Phuong Nguyen, et al. Biomed Res Int. 2014;2014:686505.</div><p class="links"><a class="dblinks" href="/pubmed/24551850" ref="ordinalpos=1503&amp;ncbi_uid=5825666&amp;link_uid=24551850">Citation</a><a class="dblinks" href="/pmc/articles/PMC3914352/" ref="ordinalpos=1503&amp;ncbi_uid=5825666&amp;link_uid=PMC3914352">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1504.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3304564/figure/fig02/" ref="ordinalpos=1504&amp;ncbi_uid=3893835&amp;link_uid=PMC3304564" href="/pmc/articles/PMC3304564/"><img src="/pmc/articles/instance/3304564/bin/ajcr0002-0178-f2.gif" src-large="/pmc/articles/instance/3304564/bin/ajcr0002-0178-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3304564/" ref="ordinalpos=1504&amp;ncbi_uid=3893835&amp;link_uid=PMC3304564" image-link="/pmc/articles/PMC3304564/figure/fig02/" class="imagepopup">Figure 2.  From: Therapeutic implications of targeting the PI3Kinase/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> module in melanoma therapy. </a></p><div class="supp"><p class="details"><div>Therapeutic implications of targeting the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. Schematic depiction of major components of the AKT <span class="highlight" style="background-color:">signaling</span> module and various pharmacological inhibitors (in various sages of clinical testing) that could potentially interfere with the activity of the AKT <span class="highlight" style="background-color:">pathway</span>. These compounds, alone or combined with various other modalities, may improve treatment efficacy.</div></p></div><div class="aux"><div class="resc">Ali R Jazirehi, et al. Am J Cancer Res. 2012;2(2):178-191.</div><p class="links"><a class="dblinks" href="/pubmed/22485197" ref="ordinalpos=1504&amp;ncbi_uid=3893835&amp;link_uid=22485197">Citation</a><a class="dblinks" href="/pmc/articles/PMC3304564/" ref="ordinalpos=1504&amp;ncbi_uid=3893835&amp;link_uid=PMC3304564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1505.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2490815/figure/fig2/" ref="ordinalpos=1505&amp;ncbi_uid=1507677&amp;link_uid=PMC2490815" href="/pmc/articles/PMC2490815/"><img src="/pmc/articles/instance/2490815/bin/PPAR2008-658520.002.gif" src-large="/pmc/articles/instance/2490815/bin/PPAR2008-658520.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2490815/" ref="ordinalpos=1505&amp;ncbi_uid=1507677&amp;link_uid=PMC2490815" image-link="/pmc/articles/PMC2490815/figure/fig2/" class="imagepopup">Figure 2.  From: PPAR Regulation of Inflammatory <span class="highlight" style="background-color:">Signaling</span> in CNS Diseases. </a></p><div class="supp"><p class="details"><div>Regulation of neuroinflammation by PPAR agonists in CNS diseases. CD40/TLR induce the activation of NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span> leading to expression of IL-12 family cytokines from APCs which in turn signal through Jak-Stat <span class="highlight" style="background-color:">pathway</span> in T cells leading to Th1/Th17 differentiation and development of CNS diseases. PPAR agonists modulate <span class="highlight" style="background-color:">signaling</span> and transcription in APC and T cells thereby preventing CNS diseases.</div></p></div><div class="aux"><div class="resc">John J. Bright, et al. PPAR Res. 2008;2008:658520.</div><p class="links"><a class="dblinks" href="/pubmed/18670616" ref="ordinalpos=1505&amp;ncbi_uid=1507677&amp;link_uid=18670616">Citation</a><a class="dblinks" href="/pmc/articles/PMC2490815/" ref="ordinalpos=1505&amp;ncbi_uid=1507677&amp;link_uid=PMC2490815">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1506.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3676658/figure/f3/" ref="ordinalpos=1506&amp;ncbi_uid=6226413&amp;link_uid=PMC3676658" href="/pmc/articles/PMC3676658/"><img src="/pmc/articles/instance/3676658/bin/fig-3.gif" src-large="/pmc/articles/instance/3676658/bin/fig-3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676658/" ref="ordinalpos=1506&amp;ncbi_uid=6226413&amp;link_uid=PMC3676658" image-link="/pmc/articles/PMC3676658/figure/f3/" class="imagepopup">Figure 3. From: Transforming Growth Factor Beta <span class="highlight" style="background-color:">Signaling</span> in Cutaneous Wound Healing: Lessons Learned from Animal Studies. </a></p><div class="supp"><p class="details"><div>Schematic diagram summarizing available information obtained using animal models to investigate the role of TGF-β <span class="highlight" style="background-color:">signaling</span> in wound healing. The basic components of the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> shown in Figure 1 are used here as a background to illustrate the phenotypic differences observed from manipulating the specific components in wound-healing studies. See text sections <i>Manipulation of TGF-β <span class="highlight" style="background-color:">signaling</span> in wound healing using exogenously added molecules</i> and <i>Genetic manipulation of TGF-β <span class="highlight" style="background-color:">signaling</span> in wound healing</i> for a detailed account of the <span class="highlight" style="background-color:">signaling</span> components manipulated and the specific effects on the wound-healing outcome. To see this illustration in color, the reader is referred to the web version of this article at www.liebertpub.com/wound</div></p></div><div class="aux"><div class="resc">Kenneth W. Finnson, et al. Adv Wound Care (New Rochelle). 2013 June;2(5):225-237.</div><p class="links"><a class="dblinks" href="/pubmed/24761336" ref="ordinalpos=1506&amp;ncbi_uid=6226413&amp;link_uid=24761336">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676658/" ref="ordinalpos=1506&amp;ncbi_uid=6226413&amp;link_uid=PMC3676658">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1507.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3977945/figure/F2/" ref="ordinalpos=1507&amp;ncbi_uid=6002757&amp;link_uid=PMC3977945" href="/pmc/articles/PMC3977945/"><img src="/pmc/articles/instance/3977945/bin/2045-3701-4-13-2.gif" src-large="/pmc/articles/instance/3977945/bin/2045-3701-4-13-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3977945/" ref="ordinalpos=1507&amp;ncbi_uid=6002757&amp;link_uid=PMC3977945" image-link="/pmc/articles/PMC3977945/figure/F2/" class="imagepopup">Figure 2.  From: Regulation of Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> by posttranslational modifications. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the simplified phosphorylation-mediated regulation of the core Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> components.</b> Phosphorylation of LRP6 at T1479 by CK1γ and at S1490 by GSK3 and Grk5/6 promotes Wnt <span class="highlight" style="background-color:">signaling</span>. Dvl phosphorylation mediated by RIPK4 and CK1ϵ is essential for Wnt <span class="highlight" style="background-color:">signaling</span>. Phosphorylation of Axin at S497/S500 by GSK3 is suppressed by Wnt ligand, resulting in reduced association with LRP6 and β-catenin. C-terminal phosphorylation of β-catenin by PKA inhibits its ubiquitination and thus promotes β-catenin <span class="highlight" style="background-color:">signaling</span> activity. TNIK phosphorylates TCF4 to activate its transcriptional activity. NLK and HIPK2 phosphorylate TCF/LEF factors to inhibit their interaction with DNA.</div></p></div><div class="aux"><div class="resc">Chenxi Gao, et al. Cell Biosci. 2014;4:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/24594309" ref="ordinalpos=1507&amp;ncbi_uid=6002757&amp;link_uid=24594309">Citation</a><a class="dblinks" href="/pmc/articles/PMC3977945/" ref="ordinalpos=1507&amp;ncbi_uid=6002757&amp;link_uid=PMC3977945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1508.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3601157/figure/F1/" ref="ordinalpos=1508&amp;ncbi_uid=4730878&amp;link_uid=PMC3601157" href="/pmc/articles/PMC3601157/"><img src="/pmc/articles/instance/3601157/bin/onci-2-e22745-g1.gif" src-large="/pmc/articles/instance/3601157/bin/onci-2-e22745-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3601157/" ref="ordinalpos=1508&amp;ncbi_uid=4730878&amp;link_uid=PMC3601157" image-link="/pmc/articles/PMC3601157/figure/F1/" class="imagepopup">Figure. From: Forging a link between oncogenic <span class="highlight" style="background-color:">signaling</span> and immunosuppression in melanoma. </a></p><div class="supp"><p class="details"><div><b>Figure 1. </b> Mechanisms of BRAF<sup>V600E</sup>-driven immunosuppression. The activation of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in melanoma cells by oncogenic BRAF<sup>V600E</sup> leads to the production of interleukin 1 (IL-1)α/β. Tumor-associated fibroblasts (TAFs) respond to IL-1 by upregulating an immunomodulatory transcriptional program resulting in the production of COX-2, PD-1 ligands and chemokines, as well as in the amplification of IL-1 <span class="highlight" style="background-color:">signaling</span>. Collectively, these factors and <span class="highlight" style="background-color:">signaling</span> circuitries suppress cytotoxic T lymphocyte (CTL) functions.</div></p></div><div class="aux"><div class="resc">Jahan S. Khalili, et al. Oncoimmunology. 2013 February 1;2(2):e22745.</div><p class="links"><a class="dblinks" href="/pubmed/23525189" ref="ordinalpos=1508&amp;ncbi_uid=4730878&amp;link_uid=23525189">Citation</a><a class="dblinks" href="/pmc/articles/PMC3601157/" ref="ordinalpos=1508&amp;ncbi_uid=4730878&amp;link_uid=PMC3601157">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1509.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3172156/figure/F2/" ref="ordinalpos=1509&amp;ncbi_uid=3220663&amp;link_uid=PMC3172156" href="/pmc/articles/PMC3172156/"><img src="/pmc/articles/instance/3172156/bin/nihms166282f2.gif" src-large="/pmc/articles/instance/3172156/bin/nihms166282f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3172156/" ref="ordinalpos=1509&amp;ncbi_uid=3220663&amp;link_uid=PMC3172156" image-link="/pmc/articles/PMC3172156/figure/F2/" class="imagepopup">Figure 2.  From: Recognition of viruses by cytoplasmic sensors. </a></p><div class="supp"><p class="details"><div>The RLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing RIG-I bound to ligand RNA and <span class="highlight" style="background-color:">signaling</span> downstream to IRF-3 and NF-kB to induce IFN-α/β production from a virus-infected cell. IFN-α/β is then shown <span class="highlight" style="background-color:">signaling</span> through the IFN-α/β receptor and the Jak-STAT <span class="highlight" style="background-color:">pathway</span> to drive ISG expression and an innate immune response. Details are described in the text.</div></p></div><div class="aux"><div class="resc">Courtney Wilkins, et al. Curr Opin Immunol. ;22(1):41-47.</div><p class="links"><a class="dblinks" href="/pubmed/20061127" ref="ordinalpos=1509&amp;ncbi_uid=3220663&amp;link_uid=20061127">Citation</a><a class="dblinks" href="/pmc/articles/PMC3172156/" ref="ordinalpos=1509&amp;ncbi_uid=3220663&amp;link_uid=PMC3172156">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1510.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2863204/figure/F8/" ref="ordinalpos=1510&amp;ncbi_uid=2936732&amp;link_uid=PMC2863204" href="/pmc/articles/PMC2863204/"><img src="/pmc/articles/instance/2863204/bin/zbc0221015640008.gif" src-large="/pmc/articles/instance/2863204/bin/zbc0221015640008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2863204/" ref="ordinalpos=1510&amp;ncbi_uid=2936732&amp;link_uid=PMC2863204" image-link="/pmc/articles/PMC2863204/figure/F8/" class="imagepopup">FIGURE 8. From: Low Density Lipoprotein Receptor-related Protein 1 (LRP1) Forms a <span class="highlight" style="background-color:">Signaling</span> Complex with Platelet-derived Growth Factor Receptor-? in Endosomes and Regulates Activation of the MAPK <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>LRP1 expression influences activation of the MAPK <span class="highlight" style="background-color:">pathway</span>.</b> Quantification of the results from Fig. 7 in LRP1<sup>+/−</sup>, LRP1<sup>−/−</sup>, and B41 clones is shown. Several <span class="highlight" style="background-color:">signaling</span> pathways affected by LRP1 expression are phosphorylation states of MAPK <span class="highlight" style="background-color:">pathway</span>-related proteins (<i>A</i>) and cell cycle-related proteins (<i>B</i>). *, mean values significantly larger than that of LRP1<sup>+/−</sup> or B41 cells (<i>p</i> &lt; 0.05); **, mean values significantly smaller than that of LRP1<sup>+/−</sup> or B41 cells (<i>p</i> &lt; 0.05).</div></p></div><div class="aux"><div class="resc">Selen Catania Muratoglu, et al. J Biol Chem. 2010 May 7;285(19):14308-14317.</div><p class="links"><a class="dblinks" href="/pubmed/20220145" ref="ordinalpos=1510&amp;ncbi_uid=2936732&amp;link_uid=20220145">Citation</a><a class="dblinks" href="/pmc/articles/PMC2863204/" ref="ordinalpos=1510&amp;ncbi_uid=2936732&amp;link_uid=PMC2863204">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1511.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3524355/figure/F2/" ref="ordinalpos=1511&amp;ncbi_uid=5535445&amp;link_uid=PMC3524355" href="/pmc/articles/PMC3524355/"><img src="/pmc/articles/instance/3524355/bin/nihms402297f2.gif" src-large="/pmc/articles/instance/3524355/bin/nihms402297f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3524355/" ref="ordinalpos=1511&amp;ncbi_uid=5535445&amp;link_uid=PMC3524355" image-link="/pmc/articles/PMC3524355/figure/F2/" class="imagepopup">Figure 2. Chromatin context affects DDR <span class="highlight" style="background-color:">signaling</span>.  From: DNA damage, chromatin, and transcription: the trinity of aging. </a></p><div class="supp"><p class="details"><div>Double-strand breaks occurring in eu- or hetero-chromatin are sensed by the DDR surveillance machinery and <span class="highlight" style="background-color:">signaling</span> is initiated. Chromatin remodeling complexes are recruited and relax chromatin structure (left <span class="highlight" style="background-color:">pathway</span>). This leads to amplification of DDR <span class="highlight" style="background-color:">signaling</span> and the initiation of downstream effector pathways such as DNA repair, apoptosis, and cell cycle arrest. In the absence of effective chromatin remodeling, and possibly in highly condensed heterochromatin, (right <span class="highlight" style="background-color:">pathway</span>), DDR <span class="highlight" style="background-color:">signaling</span> is restrained and downstream pathways are inhibited. As a result of DDR <span class="highlight" style="background-color:">signaling</span>, global gene expression programs can be altered through chromatin factor redistribution and DDR-induced local and global transcriptional changes.</div></p></div><div class="aux"><div class="resc">Rebecca C. Burgess, et al. Curr Opin Cell Biol. ;24(6):724-730.</div><p class="links"><a class="dblinks" href="/pubmed/22902297" ref="ordinalpos=1511&amp;ncbi_uid=5535445&amp;link_uid=22902297">Citation</a><a class="dblinks" href="/pmc/articles/PMC3524355/" ref="ordinalpos=1511&amp;ncbi_uid=5535445&amp;link_uid=PMC3524355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1512.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4090903/figure/F2/" ref="ordinalpos=1512&amp;ncbi_uid=6422000&amp;link_uid=PMC4090903" href="/pmc/articles/PMC4090903/"><img src="/pmc/articles/instance/4090903/bin/fimmu-05-00316-g002.gif" src-large="/pmc/articles/instance/4090903/bin/fimmu-05-00316-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4090903/" ref="ordinalpos=1512&amp;ncbi_uid=6422000&amp;link_uid=PMC4090903" image-link="/pmc/articles/PMC4090903/figure/F2/" class="imagepopup">Figure 2.  From: A Comparative Review of Toll-Like Receptor 4 Expression and Functionality in Different Animal Species. </a></p><div class="supp"><p class="details"><div><b>TLR4 intracellular <span class="highlight" style="background-color:">signaling</span> cascade</b>. The MyD88-dependent <span class="highlight" style="background-color:">pathway</span> induces NF-κB translocation and expression of pro-inflammatory cytokine genes. The MyD88-independent <span class="highlight" style="background-color:">pathway</span> induces NF-κB or IRF3 translocation, leading to expression of pro-inflammatory cytokine genes by NF-κB or IFN-β and TNF-α genes by IRF3. This canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is based on published mouse and human TLR4 knowledge.</div></p></div><div class="aux"><div class="resc">Céline Vaure, et al. Front Immunol. 2014;5:316.</div><p class="links"><a class="dblinks" href="/pubmed/25071777" ref="ordinalpos=1512&amp;ncbi_uid=6422000&amp;link_uid=25071777">Citation</a><a class="dblinks" href="/pmc/articles/PMC4090903/" ref="ordinalpos=1512&amp;ncbi_uid=6422000&amp;link_uid=PMC4090903">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1513.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4229124/figure/pone-0112014-g001/" ref="ordinalpos=1513&amp;ncbi_uid=6849245&amp;link_uid=PMC4229124" href="/pmc/articles/PMC4229124/"><img src="/pmc/articles/instance/4229124/bin/pone.0112014.g001.gif" src-large="/pmc/articles/instance/4229124/bin/pone.0112014.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4229124/" ref="ordinalpos=1513&amp;ncbi_uid=6849245&amp;link_uid=PMC4229124" image-link="/pmc/articles/PMC4229124/figure/pone-0112014-g001/" class="imagepopup">Figure 1. Verification of IFN inhibitors ability to block IFN induction or IFN <span class="highlight" style="background-color:">signaling</span>..  From: Inhibitors of the Interferon Response Enhance Virus Replication In Vitro. </a></p><div class="supp"><p class="details"><div>(a) Small molecules reported to inhibit the IKK2 (TPCA-1) and TBK1 (BX795, MRT6884, MRT6707) components of the IFN induction <span class="highlight" style="background-color:">pathway</span> were tested using the A549/pr(IFN-β).GFP reporter cell-line which contains an eGFP gene under control of the IFN-β promoter. The IFN induction <span class="highlight" style="background-color:">pathway</span> and hence GFP expression was activated by infection with a DI rich stock of PIV5VΔC. Effect of inhibitors at various concentrations was measured by monitoring fluorescence at 18 hours post-infection. (b) Small molecules reported to inhibit the JAK1 (Cyt387, AZD1480, Ruxolitinib, Tofacitinib) component of the IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were tested using the A549/pr(ISRE).GFP reporter cell-line which contains an eGFP gene under control of an ISRE promoter. The IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and hence GFP expression was activated by supplementing the cell supernatant with purified IFN. Effect of inhibitors at various concentrations was measured by monitoring fluorescence at 48 hours post IFN stimulation.</div></p></div><div class="aux"><div class="resc">Claire E. Stewart, et al. PLoS One. 2014;9(11):e112014.</div><p class="links"><a class="dblinks" href="/pubmed/25390891" ref="ordinalpos=1513&amp;ncbi_uid=6849245&amp;link_uid=25390891">Citation</a><a class="dblinks" href="/pmc/articles/PMC4229124/" ref="ordinalpos=1513&amp;ncbi_uid=6849245&amp;link_uid=PMC4229124">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1514.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4087218/figure/F2/" ref="ordinalpos=1514&amp;ncbi_uid=6361132&amp;link_uid=PMC4087218" href="/pmc/articles/PMC4087218/"><img src="/pmc/articles/instance/4087218/bin/jvs-15-179-g002.gif" src-large="/pmc/articles/instance/4087218/bin/jvs-15-179-g002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4087218/" ref="ordinalpos=1514&amp;ncbi_uid=6361132&amp;link_uid=PMC4087218" image-link="/pmc/articles/PMC4087218/figure/F2/" class="imagepopup">Fig. 2.  From: Effects of resveratrol on the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of obese mice. </a></p><div class="supp"><p class="details"><div>Restoration effects of resveratrol on protein phosphorylation in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of obese mice. (A) The phosphorylation levels of IRS-1 (Ser636, Ser639). (B) The levels of PI3-kinase p85-subunit α/γ (Tyr467/Tyr199) and PDK1 (Ser241) among the control, HF, and HFR groups were compared. Proteins extracted from each group (n = 12) were combined, and phosphorylation levels were analyzed using the antibody array.</div></p></div><div class="aux"><div class="resc">Hyun Ju Hong, et al. J Vet Sci. 2014 June;15(2):179-185.</div><p class="links"><a class="dblinks" href="/pubmed/24675832" ref="ordinalpos=1514&amp;ncbi_uid=6361132&amp;link_uid=24675832">Citation</a><a class="dblinks" href="/pmc/articles/PMC4087218/" ref="ordinalpos=1514&amp;ncbi_uid=6361132&amp;link_uid=PMC4087218">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1515.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3851046/figure/F7/" ref="ordinalpos=1515&amp;ncbi_uid=5570778&amp;link_uid=PMC3851046" href="/pmc/articles/PMC3851046/"><img src="/pmc/articles/instance/3851046/bin/1478-811X-11-74-7.gif" src-large="/pmc/articles/instance/3851046/bin/1478-811X-11-74-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3851046/" ref="ordinalpos=1515&amp;ncbi_uid=5570778&amp;link_uid=PMC3851046" image-link="/pmc/articles/PMC3851046/figure/F7/" class="imagepopup">Figure 7.  From: Zinc inhibits osteoclast differentiation by suppression of Ca2+-Calcineurin-NFATc1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>The proposed molecular mechanism for the inhibitory effects of zinc on RANKL-induced osteoclastogenesis.</b> Schematic models of the inhibitory effects of zinc on the Ca<sup>2+</sup>-Calcineurin-NFATc1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; <b>(A)</b> Zinc may inhibit calcineurin in the cytosol in the early period of osteoclastogenesis. <b>(B)</b> In the middle or late period, zinc could be suppressing calcium oscillations by blocking calcium influx from extracellular space. Cn A, Calcineurin A subunit; CaM, Calmodulin; P, phosphorylated.</div></p></div><div class="aux"><div class="resc">Kwang Hwan Park, et al. Cell Commun Signal. 2013;11:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/24088289" ref="ordinalpos=1515&amp;ncbi_uid=5570778&amp;link_uid=24088289">Citation</a><a class="dblinks" href="/pmc/articles/PMC3851046/" ref="ordinalpos=1515&amp;ncbi_uid=5570778&amp;link_uid=PMC3851046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1516.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3680532/figure/F6F__G/" ref="ordinalpos=1516&amp;ncbi_uid=6138933&amp;link_uid=PMC3680532" href="/pmc/articles/PMC3680532/"><img src="/pmc/articles/instance/3680532/bin/cc-12-1521-g6F__G.gif" src-large="/pmc/articles/instance/3680532/bin/cc-12-1521-g6F__G.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3680532/" ref="ordinalpos=1516&amp;ncbi_uid=6138933&amp;link_uid=PMC3680532" image-link="/pmc/articles/PMC3680532/figure/F6F__G/" class="imagepopup">Figure. From: PPAR? promotes oncogenic redirection of TGF-?1 <span class="highlight" style="background-color:">signaling</span> through the activation of the ABCA1-Cav1 <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 6F and G.</b> (<b>F</b>) Effect of TβR level on TGF-β1 regulation of cell proliferation. DU145 cells were transfected with siTβRs or WT-TβRs were treated with TGF-β1 (2 ng/ml, 24 h) and its effect on DNA replication was determined using [<sup>3</sup>H]thymidine incorporation assay (*, p &lt; 0.05; **, p &lt; 0.01). (<b>G</b>) Schematic representation of the oncogenic redirection of TGF-β1 <span class="highlight" style="background-color:">signaling</span> by the PPARδ-ABCA1-Cav1 <span class="highlight" style="background-color:">pathway</span> in human prostate carcinoma cells.</div></p></div><div class="aux"><div class="resc">Nam-Gu Her, et al. Cell Cycle. 2013 May 15;12(10):1521-1535.</div><p class="links"><a class="dblinks" href="/pubmed/23598720" ref="ordinalpos=1516&amp;ncbi_uid=6138933&amp;link_uid=23598720">Citation</a><a class="dblinks" href="/pmc/articles/PMC3680532/" ref="ordinalpos=1516&amp;ncbi_uid=6138933&amp;link_uid=PMC3680532">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1517.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2613484/figure/F7/" ref="ordinalpos=1517&amp;ncbi_uid=1656611&amp;link_uid=PMC2613484" href="/pmc/articles/PMC2613484/"><img src="/pmc/articles/instance/2613484/bin/nihms-82795-f0007.gif" src-large="/pmc/articles/instance/2613484/bin/nihms-82795-f0007.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2613484/" ref="ordinalpos=1517&amp;ncbi_uid=1656611&amp;link_uid=PMC2613484" image-link="/pmc/articles/PMC2613484/figure/F7/" class="imagepopup">Fig. 7. Integrated <span class="highlight" style="background-color:">signaling</span> by LTαβ-LTβR and HVEM-BTLA pathways regulates DC homeostasis.  From: Targeting lymphocyte activation through the lymphotoxin and LIGHT pathways. </a></p><div class="supp"><p class="details"><div>LTαβ specifically regulates the proliferation of CD8α<sup>-</sup> DC subsets in the spleen through a LTβR-NIK-RelB-dependent <span class="highlight" style="background-color:">pathway</span> during homeostasis. LIGHT expression in activated T cells can also increase DC proliferation through LTβR. <span class="highlight" style="background-color:">Signaling</span> through HVEM-BTLA provides inhibitory <span class="highlight" style="background-color:">signaling</span> that limits LTαβ-dependent proliferation of the CD8α<sup>-</sup> DC subsets. HVEM-BTLA expression in DCs and in stromal cells contributes to limiting DC proliferation (99).</div></p></div><div class="aux"><div class="resc">Carl F. Ware. Immunol Rev. ;223:186-201.</div><p class="links"><a class="dblinks" href="/pubmed/18613837" ref="ordinalpos=1517&amp;ncbi_uid=1656611&amp;link_uid=18613837">Citation</a><a class="dblinks" href="/pmc/articles/PMC2613484/" ref="ordinalpos=1517&amp;ncbi_uid=1656611&amp;link_uid=PMC2613484">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1518.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3491533/figure/fig3/" ref="ordinalpos=1518&amp;ncbi_uid=4352567&amp;link_uid=PMC3491533" href="/pmc/articles/PMC3491533/"><img src="/pmc/articles/instance/3491533/bin/ctg201217f3.gif" src-large="/pmc/articles/instance/3491533/bin/ctg201217f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3491533/" ref="ordinalpos=1518&amp;ncbi_uid=4352567&amp;link_uid=PMC3491533" image-link="/pmc/articles/PMC3491533/figure/fig3/" class="imagepopup">Figure 3.  From: Analysis of Notch and TGF-β <span class="highlight" style="background-color:">Signaling</span> Expression in Different Stages of Disease Progression During Hepatitis B Virus Infection. </a></p><div class="supp"><p class="details"><div>ABI custom array of 48 genes of TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was designed and quantitative real-time PCR analysis was performed with Syber Green.</div></p></div><div class="aux"><div class="resc">Nirupama Trehanpati, et al. Clin Transl Gastroenterol. 2012 October;3(10):e23.</div><p class="links"><a class="dblinks" href="/pubmed/23238065" ref="ordinalpos=1518&amp;ncbi_uid=4352567&amp;link_uid=23238065">Citation</a><a class="dblinks" href="/pmc/articles/PMC3491533/" ref="ordinalpos=1518&amp;ncbi_uid=4352567&amp;link_uid=PMC3491533">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1519.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3962152/figure/F0004/" ref="ordinalpos=1519&amp;ncbi_uid=5960237&amp;link_uid=PMC3962152" href="/pmc/articles/PMC3962152/"><img src="/pmc/articles/instance/3962152/bin/CSBJ-1-e201207005-g004.gif" src-large="/pmc/articles/instance/3962152/bin/CSBJ-1-e201207005-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3962152/" ref="ordinalpos=1519&amp;ncbi_uid=5960237&amp;link_uid=PMC3962152" image-link="/pmc/articles/PMC3962152/figure/F0004/" class="imagepopup">Figure 4.  From:  In silico evidence of <span class="highlight" style="background-color:">signaling</span> pathways of notch mediated networks in leukemia. </a></p><div class="supp"><p class="details"><div>KEGG <span class="highlight" style="background-color:">Pathway</span> database map showing Notch <span class="highlight" style="background-color:">signaling</span> including proteolytic cleavage and translocation of Notch followed by transcriptional activation of Notch target genes</div></p></div><div class="aux"><div class="resc">Kaiser Jamil, et al. Comput Struct Biotechnol J. 2012;1:e201207005.</div><p class="links"><a class="dblinks" href="/pubmed/24688641" ref="ordinalpos=1519&amp;ncbi_uid=5960237&amp;link_uid=24688641">Citation</a><a class="dblinks" href="/pmc/articles/PMC3962152/" ref="ordinalpos=1519&amp;ncbi_uid=5960237&amp;link_uid=PMC3962152">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1520.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3849276/figure/pone-0081807-g006/" ref="ordinalpos=1520&amp;ncbi_uid=5565272&amp;link_uid=PMC3849276" href="/pmc/articles/PMC3849276/"><img src="/pmc/articles/instance/3849276/bin/pone.0081807.g006.gif" src-large="/pmc/articles/instance/3849276/bin/pone.0081807.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3849276/" ref="ordinalpos=1520&amp;ncbi_uid=5565272&amp;link_uid=PMC3849276" image-link="/pmc/articles/PMC3849276/figure/pone-0081807-g006/" class="imagepopup">Figure 6. Model for PSGL-1–mediated <span class="highlight" style="background-color:">signaling</span>..  From: Lipid Raft Is Required for PSGL-1 Ligation Induced HL-60 Cell Adhesion on ICAM-1. </a></p><div class="supp"><p class="details"><div>A model of the signal transduction <span class="highlight" style="background-color:">pathway</span> induced by PSGL-1, which involves lipid rafts, Syk-Vav1 complex and other kinases. This <span class="highlight" style="background-color:">signaling</span> cascade leads to LFA-1integrin clustering that enables HL-60 cell adhere on ICAM-1.</div></p></div><div class="aux"><div class="resc">Tingshuang Xu, et al. PLoS One. 2013;8(12):e81807.</div><p class="links"><a class="dblinks" href="/pubmed/24312591" ref="ordinalpos=1520&amp;ncbi_uid=5565272&amp;link_uid=24312591">Citation</a><a class="dblinks" href="/pmc/articles/PMC3849276/" ref="ordinalpos=1520&amp;ncbi_uid=5565272&amp;link_uid=PMC3849276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1521.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136489/figure/pone-0021902-g010/" ref="ordinalpos=1521&amp;ncbi_uid=3090360&amp;link_uid=PMC3136489" href="/pmc/articles/PMC3136489/"><img src="/pmc/articles/instance/3136489/bin/pone.0021902.g010.gif" src-large="/pmc/articles/instance/3136489/bin/pone.0021902.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136489/" ref="ordinalpos=1521&amp;ncbi_uid=3090360&amp;link_uid=PMC3136489" image-link="/pmc/articles/PMC3136489/figure/pone-0021902-g010/" class="imagepopup">Figure 10. Modulation of the B Cell Antigen Receptor <span class="highlight" style="background-color:">Pathway</span> in Subgroup Y..  From: Pathways Activated during Human Asthma Exacerbation as Revealed by Gene Expression Patterns in Blood. </a></p><div class="supp"><p class="details"><div>Shown in red are B-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components that are up-regulated in Subgroup Y exacerbations.</div></p></div><div class="aux"><div class="resc">Unnur S. Bjornsdottir, et al. PLoS One. 2011;6(7):e21902.</div><p class="links"><a class="dblinks" href="/pubmed/21779351" ref="ordinalpos=1521&amp;ncbi_uid=3090360&amp;link_uid=21779351">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136489/" ref="ordinalpos=1521&amp;ncbi_uid=3090360&amp;link_uid=PMC3136489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1522.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2726720/figure/F3/" ref="ordinalpos=1522&amp;ncbi_uid=1938535&amp;link_uid=PMC2726720" href="/pmc/articles/PMC2726720/"><img src="/pmc/articles/instance/2726720/bin/nihms115902f3.gif" src-large="/pmc/articles/instance/2726720/bin/nihms115902f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2726720/" ref="ordinalpos=1522&amp;ncbi_uid=1938535&amp;link_uid=PMC2726720" image-link="/pmc/articles/PMC2726720/figure/F3/" class="imagepopup">Fig. 3.  From: Liver Stem Cells and Hepatocellular Carcinoma. </a></p><div class="supp"><p class="details"><div>Interplay of TGF-β, Wnt/β-catenin, Hedgehog, and Notch in stem cell differentiation and tumorigenesis. Schematic diagram of TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Note interplay between Wnt/β-catenin, Hedgehog, Notch, and TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> targets and expedited tumorigenesis.</div></p></div><div class="aux"><div class="resc">Lopa Mishra, et al. Hepatology. ;49(1):318-329.</div><p class="links"><a class="dblinks" href="/pubmed/19111019" ref="ordinalpos=1522&amp;ncbi_uid=1938535&amp;link_uid=19111019">Citation</a><a class="dblinks" href="/pmc/articles/PMC2726720/" ref="ordinalpos=1522&amp;ncbi_uid=1938535&amp;link_uid=PMC2726720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1523.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2868430/figure/F4/" ref="ordinalpos=1523&amp;ncbi_uid=2448220&amp;link_uid=PMC2868430" href="/pmc/articles/PMC2868430/"><img src="/pmc/articles/instance/2868430/bin/z7g9990902990004.gif" src-large="/pmc/articles/instance/2868430/bin/z7g9990902990004.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2868430/" ref="ordinalpos=1523&amp;ncbi_uid=2448220&amp;link_uid=PMC2868430" image-link="/pmc/articles/PMC2868430/figure/F4/" class="imagepopup">Figure 4. From: A Fungal pH-Responsive <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulating Aspergillus Adaptation and Invasion into the Cornea. </a></p><div class="supp"><p class="details"><div>Model of <i>Aspergillus</i> PacC activation. Receptors at the fungal cell membrane sense ambient pH conditions and signal an intracellular <span class="highlight" style="background-color:">pathway</span> to proteolytically cleave PacC into an active transcription factor that upregulates selected fungal genes. (Modified from Peñas MM, Hervas-Aguilar A, Munera-Huertas T, et al. Further characterization of the <span class="highlight" style="background-color:">signaling</span> proteolysis step in the Aspergillus nidulans pH signal transduction <span class="highlight" style="background-color:">pathway</span>. <i>Eukaryot Cell</i>. 2007;6:960–970, doi:10.1128/EC.00047-07. Amended with permission from the American Society for Microbiology.)</div></p></div><div class="aux"><div class="resc">Xia Hua, et al. Invest Ophthalmol Vis Sci. 2010 March;51(3):1517-1523.</div><p class="links"><a class="dblinks" href="/pubmed/19850840" ref="ordinalpos=1523&amp;ncbi_uid=2448220&amp;link_uid=19850840">Citation</a><a class="dblinks" href="/pmc/articles/PMC2868430/" ref="ordinalpos=1523&amp;ncbi_uid=2448220&amp;link_uid=PMC2868430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1524.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1553448/figure/F7/" ref="ordinalpos=1524&amp;ncbi_uid=779751&amp;link_uid=PMC1553448" href="/pmc/articles/PMC1553448/"><img src="/pmc/articles/instance/1553448/bin/1471-2164-7-92-7.gif" src-large="/pmc/articles/instance/1553448/bin/1471-2164-7-92-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1553448/" ref="ordinalpos=1524&amp;ncbi_uid=779751&amp;link_uid=PMC1553448" image-link="/pmc/articles/PMC1553448/figure/F7/" class="imagepopup">Figure 7.  From: Transcriptomes of human prostate cells. </a></p><div class="supp"><p class="details"><div><b>Transcript expression of BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members. </b>The diagram indicates cellular locations of differentially expressed genes in the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in prostate stromal cells. Decorin, a negative regulator of TGFβ <span class="highlight" style="background-color:">signaling</span>, is produced by stromal cells, which may result in autocrine regulation of the <span class="highlight" style="background-color:">pathway</span>. BMP4 and BMP5 are produced by stromal cells and may be responsible for paracrine regulation of Inhibitor of Differentiation (ID) gene expression in the prostate.</div></p></div><div class="aux"><div class="resc">Asa J Oudes, et al. BMC Genomics. 2006;7:92-92.</div><p class="links"><a class="dblinks" href="/pubmed/16638148" ref="ordinalpos=1524&amp;ncbi_uid=779751&amp;link_uid=16638148">Citation</a><a class="dblinks" href="/pmc/articles/PMC1553448/" ref="ordinalpos=1524&amp;ncbi_uid=779751&amp;link_uid=PMC1553448">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1525.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2913455/figure/F7/" ref="ordinalpos=1525&amp;ncbi_uid=3088780&amp;link_uid=PMC2913455" href="/pmc/articles/PMC2913455/"><img src="/pmc/articles/instance/2913455/bin/zh89991052990007.gif" src-large="/pmc/articles/instance/2913455/bin/zh89991052990007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2913455/" ref="ordinalpos=1525&amp;ncbi_uid=3088780&amp;link_uid=PMC2913455" image-link="/pmc/articles/PMC2913455/figure/F7/" class="imagepopup">Figure 7.  From: E-selectin engages PSGL-1 and CD44 through a common <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to induce integrin ?L?2-mediated slow leukocyte rolling. </a></p><div class="supp"><p class="details"><div><b>Model for E-selectin–mediated slow rolling</b>. The circled numbers represent new <span class="highlight" style="background-color:">signaling</span> components identified in this paper. Neutrophils rolling on E-selectin engage both CD44 and PSGL-1 to initiate <span class="highlight" style="background-color:">signaling</span> through a common <span class="highlight" style="background-color:">pathway</span> that requires lipid rafts, the cytoplasmic domain of PSGL-1, all 3 SFKs, the ITAM adaptors DAP12 and FcRγ, the Tec kinase Btk, and p38. This <span class="highlight" style="background-color:">signaling</span> cascade activates integrin leukocyte function-associated antigen 1 (LFA-1) to a conformation that enables slow rolling but not arrest on ICAM-1. PSGL-1 and CD44 may not be located in the same raft domains as depicted in the figure.</div></p></div><div class="aux"><div class="resc">Tadayuki Yago, et al. Blood. 2010 July 22;116(3):485-494.</div><p class="links"><a class="dblinks" href="/pubmed/20299514" ref="ordinalpos=1525&amp;ncbi_uid=3088780&amp;link_uid=20299514">Citation</a><a class="dblinks" href="/pmc/articles/PMC2913455/" ref="ordinalpos=1525&amp;ncbi_uid=3088780&amp;link_uid=PMC2913455">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1526.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3292697/figure/F3/" ref="ordinalpos=1526&amp;ncbi_uid=4659604&amp;link_uid=PMC3292697" href="/pmc/articles/PMC3292697/"><img src="/pmc/articles/instance/3292697/bin/nihms325679f3.gif" src-large="/pmc/articles/instance/3292697/bin/nihms325679f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3292697/" ref="ordinalpos=1526&amp;ncbi_uid=4659604&amp;link_uid=PMC3292697" image-link="/pmc/articles/PMC3292697/figure/F3/" class="imagepopup">Figure 3.  From: Inflammatory and protein metabolism <span class="highlight" style="background-color:">signaling</span> responses in human skeletal muscle following burn injury. </a></p><div class="supp"><p class="details"><div>Muscle protein levels of NFκB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Data presented as mean ± SEM. *Significantly different from control (p &lt; 0.05).</div></p></div><div class="aux"><div class="resc">Edward K Merritt, et al. J Burn Care Res. ;33(2):291-297.</div><p class="links"><a class="dblinks" href="/pubmed/22079905" ref="ordinalpos=1526&amp;ncbi_uid=4659604&amp;link_uid=22079905">Citation</a><a class="dblinks" href="/pmc/articles/PMC3292697/" ref="ordinalpos=1526&amp;ncbi_uid=4659604&amp;link_uid=PMC3292697">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1527.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3126319/figure/F7/" ref="ordinalpos=1527&amp;ncbi_uid=3527308&amp;link_uid=PMC3126319" href="/pmc/articles/PMC3126319/"><img src="/pmc/articles/instance/3126319/bin/zjv9990946520007.gif" src-large="/pmc/articles/instance/3126319/bin/zjv9990946520007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3126319/" ref="ordinalpos=1527&amp;ncbi_uid=3527308&amp;link_uid=PMC3126319" image-link="/pmc/articles/PMC3126319/figure/F7/" class="imagepopup">Fig. 7. From: Analysis of Interferon <span class="highlight" style="background-color:">Signaling</span> by Infectious Hepatitis C Virus Clones with Substitutions of Core Amino Acids 70 and 91. </a></p><div class="supp"><p class="details"><div>Schematic diagram of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in HCV core mutant infection and IFN resistance.</div></p></div><div class="aux"><div class="resc">Yusuke Funaoka, et al. J Virol. 2011 June;85(12):5986-5994.</div><p class="links"><a class="dblinks" href="/pubmed/21490101" ref="ordinalpos=1527&amp;ncbi_uid=3527308&amp;link_uid=21490101">Citation</a><a class="dblinks" href="/pmc/articles/PMC3126319/" ref="ordinalpos=1527&amp;ncbi_uid=3527308&amp;link_uid=PMC3126319">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1528.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3756920/figure/F10/" ref="ordinalpos=1528&amp;ncbi_uid=5490370&amp;link_uid=PMC3756920" href="/pmc/articles/PMC3756920/"><img src="/pmc/articles/instance/3756920/bin/2678fig10.gif" src-large="/pmc/articles/instance/3756920/bin/2678fig10.jpg" alt="FIGURE 10:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3756920/" ref="ordinalpos=1528&amp;ncbi_uid=5490370&amp;link_uid=PMC3756920" image-link="/pmc/articles/PMC3756920/figure/F10/" class="imagepopup">FIGURE 10:.  From: Rap-afadin axis in control of Rho <span class="highlight" style="background-color:">signaling</span> and endothelial barrier recovery. </a></p><div class="supp"><p class="details"><div>Role of Rap1 in EC barrier restoration after thrombin. In parallel with rapid activation of the Rho <span class="highlight" style="background-color:">pathway</span> leading to increased EC permeability, thrombin causes activation of Src kinase, which stimulates the Rap1-specific GEF C3G and, via Rap1-Tiam1, turns on the Rac1 <span class="highlight" style="background-color:">signaling</span>. Activation of Rap1-Rac1 <span class="highlight" style="background-color:">signaling</span> axis down-regulates the Rho <span class="highlight" style="background-color:">pathway</span> of barrier disruption and promotes reassembly of AJ complexes and endothelial monolayer barrier restoration.</div></p></div><div class="aux"><div class="resc">Anna A. Birukova, et al. Mol Biol Cell. 2013 September 1;24(17):2678-2688.</div><p class="links"><a class="dblinks" href="/pubmed/23864716" ref="ordinalpos=1528&amp;ncbi_uid=5490370&amp;link_uid=23864716">Citation</a><a class="dblinks" href="/pmc/articles/PMC3756920/" ref="ordinalpos=1528&amp;ncbi_uid=5490370&amp;link_uid=PMC3756920">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1529.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3883756/figure/F6/" ref="ordinalpos=1529&amp;ncbi_uid=5684991&amp;link_uid=PMC3883756" href="/pmc/articles/PMC3883756/"><img src="/pmc/articles/instance/3883756/bin/nihms439086f6.gif" src-large="/pmc/articles/instance/3883756/bin/nihms439086f6.jpg" alt="FIGURE 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3883756/" ref="ordinalpos=1529&amp;ncbi_uid=5684991&amp;link_uid=PMC3883756" image-link="/pmc/articles/PMC3883756/figure/F6/" class="imagepopup">FIGURE 6.  From: Anti-MHC Class I Antibody Activation of Proliferation and Survival <span class="highlight" style="background-color:">Signaling</span> in Murine Cardiac Allografts. </a></p><div class="supp"><p class="details"><div>Demonstration of interrelationships between proteins involved in the MHC class I-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in vivo. MDS analysis of protein phosphorylation data obtained by Western blot shows that S6K and Akt are closely linked and members of the mTOR <span class="highlight" style="background-color:">pathway</span>: mTOR, S6K, S6RP, are linked to Akt and related to each other. The great distance separating ERK from members of the mTOR <span class="highlight" style="background-color:">pathway</span> suggests multiple <span class="highlight" style="background-color:">signaling</span> inputs are required for ERK activation.</div></p></div><div class="aux"><div class="resc">Peter T. Jindra, et al. J Immunol. ;180(4):2214-2224.</div><p class="links"><a class="dblinks" href="/pubmed/18250428" ref="ordinalpos=1529&amp;ncbi_uid=5684991&amp;link_uid=18250428">Citation</a><a class="dblinks" href="/pmc/articles/PMC3883756/" ref="ordinalpos=1529&amp;ncbi_uid=5684991&amp;link_uid=PMC3883756">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1530.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3760732/figure/F8/" ref="ordinalpos=1530&amp;ncbi_uid=5265490&amp;link_uid=PMC3760732" href="/pmc/articles/PMC3760732/"><img src="/pmc/articles/instance/3760732/bin/nihms467206f8.gif" src-large="/pmc/articles/instance/3760732/bin/nihms467206f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3760732/" ref="ordinalpos=1530&amp;ncbi_uid=5265490&amp;link_uid=PMC3760732" image-link="/pmc/articles/PMC3760732/figure/F8/" class="imagepopup">Figure 8.  From: Activation of Notch-Mediated Protective <span class="highlight" style="background-color:">Signaling</span> in the Myocardium. </a></p><div class="supp"><p class="details"><div>Schematic representation of <span class="highlight" style="background-color:">pathway</span> interactions among HGF/c-Met, PI3K/Akt, and Notch <span class="highlight" style="background-color:">signaling</span> pathways. Activation of Notch receptor (red) by Delta/Jagged ligands (green) results in translocation of NICD (red oval) to the nucleus (purple), where Hes1 repressor (black) is upregulated, leading to downregulation of PTEN and c-Met transcription.<sup>25,31</sup> NICD also induces proliferative <span class="highlight" style="background-color:">signaling</span> (green box). Activation of c-Met receptor (brown) by ligand HGF (yellow oval) stimulates PI3K (pink) and pAkt (blue) <span class="highlight" style="background-color:">signaling</span>. Phospho-Akt may interact with NICD in the nucleus, potentially increasing expression of Delta/Jagged ligands.<sup>25</sup></div></p></div><div class="aux"><div class="resc">Natalie A. Gude, et al. Circ Res. 2008 May 9;102(9):1025-1035.</div><p class="links"><a class="dblinks" href="/pubmed/18369158" ref="ordinalpos=1530&amp;ncbi_uid=5265490&amp;link_uid=18369158">Citation</a><a class="dblinks" href="/pmc/articles/PMC3760732/" ref="ordinalpos=1530&amp;ncbi_uid=5265490&amp;link_uid=PMC3760732">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1531.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3390678/figure/F8/" ref="ordinalpos=1531&amp;ncbi_uid=5064386&amp;link_uid=PMC3390678" href="/pmc/articles/PMC3390678/"><img src="/pmc/articles/instance/3390678/bin/zbc0291214330008.gif" src-large="/pmc/articles/instance/3390678/bin/zbc0291214330008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3390678/" ref="ordinalpos=1531&amp;ncbi_uid=5064386&amp;link_uid=PMC3390678" image-link="/pmc/articles/PMC3390678/figure/F8/" class="imagepopup">FIGURE 8. From: Characterization of Arginylation Branch of N-end Rule <span class="highlight" style="background-color:">Pathway</span> in G-protein-mediated Proliferation and <span class="highlight" style="background-color:">Signaling</span> of Cardiomyocytes. </a></p><div class="supp"><p class="details"><div><b>Model in which ATE1 R-transferases regulate homeostasis of G-protein <span class="highlight" style="background-color:">signaling</span> in cardiomyocytes through arginylation of RGS4 (and RGS5 and RGS16 as well).</b> In this model, N-terminal Met is cotranslationally cleaved off by Met aminopeptidases (<i>Met Aps</i>), exposing the pro-N-degron Cys-2 at the N terminus. In normally growing embryos, the Cys-2 residue is oxidized into CysO<sub>2</sub>(H) or CysO<sub>3</sub>(H) and subsequently arginylated by ATE1 R-transferases, producing the N-degron Arg, which is recognized by N-recognins containing the UBR box. The degradation of RGS4 through the N-end rule <span class="highlight" style="background-color:">pathway</span> leads to the activation of G-protein <span class="highlight" style="background-color:">signaling</span> and thus cell proliferation. However, if the Cys-2 residue is not readily oxidized, for example in ischemia, RGS4 is accumulated and turns down G-protein <span class="highlight" style="background-color:">signaling</span>. This mechanism may represent a sensor of oxygen or its derivative in the cardiovascular system.</div></p></div><div class="aux"><div class="resc">Min Jae Lee, et al. J Biol Chem. 2012 July 6;287(28):24043-24052.</div><p class="links"><a class="dblinks" href="/pubmed/22577142" ref="ordinalpos=1531&amp;ncbi_uid=5064386&amp;link_uid=22577142">Citation</a><a class="dblinks" href="/pmc/articles/PMC3390678/" ref="ordinalpos=1531&amp;ncbi_uid=5064386&amp;link_uid=PMC3390678">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1532.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1636853/figure/f10/" ref="ordinalpos=1532&amp;ncbi_uid=761906&amp;link_uid=PMC1636853" href="/pmc/articles/PMC1636853/"><img src="/pmc/articles/instance/1636853/bin/zmb0200662920010.gif" src-large="/pmc/articles/instance/1636853/bin/zmb0200662920010.jpg" alt="FIG. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1636853/" ref="ordinalpos=1532&amp;ncbi_uid=761906&amp;link_uid=PMC1636853" image-link="/pmc/articles/PMC1636853/figure/f10/" class="imagepopup">FIG. 10. From: Different Routes of Bone Morphogenic Protein (BMP) Receptor Endocytosis Influence BMP <span class="highlight" style="background-color:">Signaling</span>  . </a></p><div class="supp"><p class="details"><div>Smad-dependent <span class="highlight" style="background-color:">signaling</span> and Smad-independent BMP <span class="highlight" style="background-color:">signaling</span> originate from distinct endocytic routes of the BMP receptors. Smad-dependent <span class="highlight" style="background-color:">signaling</span> is initiated upon BMP-2 binding to PFCs, which exist in an inactive state at the cell surface prior to ligand binding (45). Activation of this <span class="highlight" style="background-color:">signaling</span> cascade by BMP-2 binding results in the phosphorylation of Smad1/5 by the type I receptor. This step in Smad <span class="highlight" style="background-color:">signaling</span> occurs while the receptors reside at the plasma membrane; inhibition of endocytosis (treatment with chlorpromazine or expression of dominant negative dynamin) does not affect BMP-2-mediated Smad phosphorylation. Subsequent endocytosis of the receptors through CCPs, however, is needed to transmit the Smad signal into the nucleus to control the transcriptional activity of target genes. Treatment with lovastatin to perturb cholesterol-dependent plasma membrane regions does not affect Smad <span class="highlight" style="background-color:">signaling</span> but reduces BMP-2-mediated ALP induction through a Smad-independent cascade. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is initiated by the binding of BMP-2 to high-affinity BRI, and a subsequent recruitment of BRII into the <span class="highlight" style="background-color:">signaling</span> receptor complex BISC occurs (45). From the data presented here, we conclude that Smad-independent <span class="highlight" style="background-color:">signaling</span> resulting in ALP induction starts from receptors (BISCs) residing in cholesterol-enriched plasma membrane regions.</div></p></div><div class="aux"><div class="resc">Anke Hartung, et al. Mol Cell Biol. 2006 October;26(20):7791-7805.</div><p class="links"><a class="dblinks" href="/pubmed/16923969" ref="ordinalpos=1532&amp;ncbi_uid=761906&amp;link_uid=16923969">Citation</a><a class="dblinks" href="/pmc/articles/PMC1636853/" ref="ordinalpos=1532&amp;ncbi_uid=761906&amp;link_uid=PMC1636853">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1533.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3708286/figure/Fig2/" ref="ordinalpos=1533&amp;ncbi_uid=5091147&amp;link_uid=PMC3708286" href="/pmc/articles/PMC3708286/"><img src="/pmc/articles/instance/3708286/bin/18_2012_1188_Fig2_HTML.gif" src-large="/pmc/articles/instance/3708286/bin/18_2012_1188_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3708286/" ref="ordinalpos=1533&amp;ncbi_uid=5091147&amp;link_uid=PMC3708286" image-link="/pmc/articles/PMC3708286/figure/Fig2/" class="imagepopup">Fig. 2.  From: Polycystins and cellular Ca2+ <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways that relate disturbed polycystin-mediated Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> to cell proliferation. Defects in polycystin functions generally lead to a decrease in the cytosolic Ca<sup>2+</sup> concentration ([Ca<sup>2+</sup>]<sub>i</sub>). This results in an increase in cAMP concentration, probably via changes in Ca<sup>2+</sup>-dependent phosphodiesterase or Ca<sup>2+</sup>-dependent adenylate-cyclase activity (not shown). The Ca<sup>2+</sup>-cAMP link is represented by the <i>box</i>. In the presence of low [Ca<sup>2+</sup>]<sub>i</sub>, cAMP becomes pro-proliferative via activation of the Src/Ras/B-Raf/MEK/ERK <span class="highlight" style="background-color:">pathway</span>. Ca<sup>2+</sup> restriction causes decreased PI3-K/AKT <span class="highlight" style="background-color:">signaling</span>, which relieves the inhibition of B-Raf. Activation of growth-factor receptors with tyrosine-kinase activity (GFR) also contributes to the stimulation of MAPK/ERK <span class="highlight" style="background-color:">signaling</span> and cell proliferation. Polycystin-1 dysfunction thus upregulates the MAPK/ERK <span class="highlight" style="background-color:">pathway</span>, which results in inactivation of the tuberin complex and increased mTORC1. Cytoplasmic Ca<sup>2+</sup> also regulates the mTOR <span class="highlight" style="background-color:">pathway</span> via CaMKKß and AMPK (the <i>question mark</i> indicates that the occurrence of this mechanism was not yet explored in cystic cells). The deregulated Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> switches on canonical Wnt <span class="highlight" style="background-color:">signaling</span>, which activates mTOR via inhibiting GSK3 phosphorylation of tuberin. Another link to cell proliferation may depend on the activation of Ca<sup>2+</sup> oscillations and subsequent effects on gene transcription and cyclins via Ca<sup>2+</sup>-dependent NFAT or via protein kinase Cα (PKCα) <span class="highlight" style="background-color:">signaling</span>. <i>Boxed areas</i> indicate mechanisms not shown in detail. <i>Dotted arrows</i> indicate still-unresolved mechanisms. <i>GFR</i> growth-factor receptor</div></p></div><div class="aux"><div class="resc">D. Mekahli, et al. Cell Mol Life Sci. 2013 August;70(15):2697-2712.</div><p class="links"><a class="dblinks" href="/pubmed/23076254" ref="ordinalpos=1533&amp;ncbi_uid=5091147&amp;link_uid=23076254">Citation</a><a class="dblinks" href="/pmc/articles/PMC3708286/" ref="ordinalpos=1533&amp;ncbi_uid=5091147&amp;link_uid=PMC3708286">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1534.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3164671/figure/pone-0023691-g005/" ref="ordinalpos=1534&amp;ncbi_uid=3215801&amp;link_uid=PMC3164671" href="/pmc/articles/PMC3164671/"><img src="/pmc/articles/instance/3164671/bin/pone.0023691.g005.gif" src-large="/pmc/articles/instance/3164671/bin/pone.0023691.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3164671/" ref="ordinalpos=1534&amp;ncbi_uid=3215801&amp;link_uid=PMC3164671" image-link="/pmc/articles/PMC3164671/figure/pone-0023691-g005/" class="imagepopup">Figure 5. Schematic depiction of the neurotrophin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Genome-Wide Identification of Bcl11b Gene Targets Reveals Role in Brain-Derived Neurotrophic Factor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Those gene indicated in red represent genes upregulated in expression and those in green, downregulated in expression, in response to Bcl11b. Figure generated using <span class="highlight" style="background-color:">Pathway</span> Design by Ingenuity Systems Pathways Analysis. Abbreviations for the official gene symbols as follows: BDNF, <i>Bdnf</i>; NGF, <i>Ngfb</i>; PI3K, <i>Pik3r1</i>; GAB1, <i>Gabl1</i>; SHC, <i>Shc1</i>; PLCγ1, <i>Plcg2</i>; Ras, <i>Kras</i>; SOS, <i>Sos1</i>; SHP2, <i>Ptpn11</i>; MKK4/7, <i>Map2k7</i>; MEK, <i>Map2k1</i>; RSK, <i>Rps6ka1</i>; c-fos, <i>Fos</i>; Atf3, <i>Atf3</i>.</div></p></div><div class="aux"><div class="resc">Bin Tang, et al. PLoS One. 2011;6(9):e23691.</div><p class="links"><a class="dblinks" href="/pubmed/21912641" ref="ordinalpos=1534&amp;ncbi_uid=3215801&amp;link_uid=21912641">Citation</a><a class="dblinks" href="/pmc/articles/PMC3164671/" ref="ordinalpos=1534&amp;ncbi_uid=3215801&amp;link_uid=PMC3164671">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1535.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3797434/figure/f7/" ref="ordinalpos=1535&amp;ncbi_uid=5397089&amp;link_uid=PMC3797434" href="/pmc/articles/PMC3797434/"><img src="/pmc/articles/instance/3797434/bin/srep02960-f7.gif" src-large="/pmc/articles/instance/3797434/bin/srep02960-f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3797434/" ref="ordinalpos=1535&amp;ncbi_uid=5397089&amp;link_uid=PMC3797434" image-link="/pmc/articles/PMC3797434/figure/f7/" class="imagepopup">Figure 7. Schematic representation of the LPS-TLR4 and IL-32-PR3-PAR2 <span class="highlight" style="background-color:">signaling</span> pathways..  From: IL-32-PAR2 axis is an innate immunity sensor providing alternative <span class="highlight" style="background-color:">signaling</span> for LPS-TRIF axis. </a></p><div class="supp"><p class="details"><div>Once LPS from Gram-negative bacteria is recognized by TLR4, the TLR4-MyD88 <span class="highlight" style="background-color:">pathway</span> is triggered, and early-phase TNFα expression is initiated through the activation of NF-κB and MAPKs. Activation of PKCε results in activation of TRAM-TRIF, which is responsible for late-phase TNFα and type I interferon (i.e., IFN-α/β) expression. Concurrently, IL-32γ binds to PR3 and activates PAR2. PAR2-Ras and -TRIF pathways potentially induce TNFα expression, and the latter <span class="highlight" style="background-color:">pathway</span> in particular induces type I interferon expression via IRF3/7. At that time, TAG is synthesized to terminate TLR4-TRIF <span class="highlight" style="background-color:">signaling</span>, with TRIF preferentially recruited to PAR2 instead. Following LPS-induced TNFα expression mainly through TLR4, LPS-induced IL-32γ-PAR2 <span class="highlight" style="background-color:">signaling</span> gradually increases type I interferon expression, which potentially translates innate to adaptive immunity, or ceases a series of LPS-induced acute inflammation.</div></p></div><div class="aux"><div class="resc">Masanori Nakayama, et al. Sci Rep. 2013;3:2960.</div><p class="links"><a class="dblinks" href="/pubmed/24129891" ref="ordinalpos=1535&amp;ncbi_uid=5397089&amp;link_uid=24129891">Citation</a><a class="dblinks" href="/pmc/articles/PMC3797434/" ref="ordinalpos=1535&amp;ncbi_uid=5397089&amp;link_uid=PMC3797434">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1536.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3691869/figure/F6/" ref="ordinalpos=1536&amp;ncbi_uid=5023427&amp;link_uid=PMC3691869" href="/pmc/articles/PMC3691869/"><img src="/pmc/articles/instance/3691869/bin/nihms443298f6.gif" src-large="/pmc/articles/instance/3691869/bin/nihms443298f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3691869/" ref="ordinalpos=1536&amp;ncbi_uid=5023427&amp;link_uid=PMC3691869" image-link="/pmc/articles/PMC3691869/figure/F6/" class="imagepopup">Fig. 6.  From: SH3GL2 is frequently deleted in non-small cell lung cancer and downregulates tumor growth by modulating EGFR <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic outline of the regulation of <i>EGFR</i> internalization and mediated <span class="highlight" style="background-color:">signaling</span> by the <i>CIN85–SH3GL2–Cbl</i> complex. <b>a</b> Deletion or inactivation of <i>SH3GL2</i> (indicated by <i>X</i>) from this complex will allow tumor progression via continuous <span class="highlight" style="background-color:">signaling</span> through the <i>EGFR</i> <span class="highlight" style="background-color:">pathway</span> upon ligand (<i>EGF</i>) stimulation as depicted. <b>b</b> Presence of wild-type <i>SH3GL2</i> and the appropriately assembled <i>CIN85–SH3GL2– Cbl</i> complex on the other hand will trigger <i>EGFR</i> internalization and subsequent degradation thereby resulting in decreased tumor growth via downregulation of the <i>EGFR</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Santanu Dasgupta, et al. J Mol Med (Berl). ;91(3):381-393.</div><p class="links"><a class="dblinks" href="/pubmed/22968441" ref="ordinalpos=1536&amp;ncbi_uid=5023427&amp;link_uid=22968441">Citation</a><a class="dblinks" href="/pmc/articles/PMC3691869/" ref="ordinalpos=1536&amp;ncbi_uid=5023427&amp;link_uid=PMC3691869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1537.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3120887/figure/pone-0021495-g005/" ref="ordinalpos=1537&amp;ncbi_uid=3067904&amp;link_uid=PMC3120887" href="/pmc/articles/PMC3120887/"><img src="/pmc/articles/instance/3120887/bin/pone.0021495.g005.gif" src-large="/pmc/articles/instance/3120887/bin/pone.0021495.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3120887/" ref="ordinalpos=1537&amp;ncbi_uid=3067904&amp;link_uid=PMC3120887" image-link="/pmc/articles/PMC3120887/figure/pone-0021495-g005/" class="imagepopup">Figure 5. Predicted <span class="highlight" style="background-color:">signaling</span> pathways regulated by differentially expressed miRNA-families in frontal cortex and hippocampus..  From: MicroRNA Expression Profiling Reveals MiRNA Families Regulating Specific Biological Pathways in Mouse Frontal Cortex and Hippocampus. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of predicted target genes of differentially expressed miRNA families and clusters was carried out to assess their putative biological functions. Specific pathways that are predicted to be regulated by several miRNAs from the same family/cluster were identified (for a list of pathways see Table S2). (<b>a</b>) As an example, ceramide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> composed of miRNAs from miR-8 family and their predicted target genes is shown. (<b>b</b>) Dopamine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> including miR-34 family members and their predicted target genes.</div></p></div><div class="aux"><div class="resc">Juuso Juhila, et al. PLoS One. 2011;6(6):e21495.</div><p class="links"><a class="dblinks" href="/pubmed/21731767" ref="ordinalpos=1537&amp;ncbi_uid=3067904&amp;link_uid=21731767">Citation</a><a class="dblinks" href="/pmc/articles/PMC3120887/" ref="ordinalpos=1537&amp;ncbi_uid=3067904&amp;link_uid=PMC3120887">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1538.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3358558/figure/Fig6/" ref="ordinalpos=1538&amp;ncbi_uid=4033518&amp;link_uid=PMC3358558" href="/pmc/articles/PMC3358558/"><img src="/pmc/articles/instance/3358558/bin/11033_2012_1611_Fig6_HTML.gif" src-large="/pmc/articles/instance/3358558/bin/11033_2012_1611_Fig6_HTML.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3358558/" ref="ordinalpos=1538&amp;ncbi_uid=4033518&amp;link_uid=PMC3358558" image-link="/pmc/articles/PMC3358558/figure/Fig6/" class="imagepopup">Fig. 6.  From: Celecoxib inhibits growth of human autosomal dominant polycystic kidney cyst-lining epithelial cells through the VEGF/Raf/MAPK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Protein expression of cell cycle and apoptosis regulatory proteins. Effects of CXB (20 μM) on ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> phosphorylation (<b>a</b>), PCNA expression (<b>a</b>), Bax/Bcl-2 expression (<b>b</b>), caspase-3 cleavage (<b>b</b>), Cyclin A expression (<b>b</b>) and p21<sup>CIP/WAF1</sup> expression (<b>b</b>). GAPDH was used as a loading control. *<i>P</i> &lt; 0.05 vs. control; <i>n</i> = 8. GAPDH served as a loading control</div></p></div><div class="aux"><div class="resc">Tao Xu, et al. Mol Biol Rep. 2012 July;39(7):7743-7753.</div><p class="links"><a class="dblinks" href="/pubmed/22415852" ref="ordinalpos=1538&amp;ncbi_uid=4033518&amp;link_uid=22415852">Citation</a><a class="dblinks" href="/pmc/articles/PMC3358558/" ref="ordinalpos=1538&amp;ncbi_uid=4033518&amp;link_uid=PMC3358558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1539.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3382540/figure/AS-fig01/" ref="ordinalpos=1539&amp;ncbi_uid=4451705&amp;link_uid=PMC3382540" href="/pmc/articles/PMC3382540/"><img src="/pmc/articles/instance/3382540/bin/pnas.1117640S109fig01.gif" src-large="/pmc/articles/instance/3382540/bin/pnas.1117640S109fig01.jpg" alt="Fig. P1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3382540/" ref="ordinalpos=1539&amp;ncbi_uid=4451705&amp;link_uid=PMC3382540" image-link="/pmc/articles/PMC3382540/figure/AS-fig01/" class="imagepopup">Fig. P1. From: K-RAS GTPase- and B-RAF kinase-mediated T-cell tolerance defects in rheumatoid arthritis. </a></p><div class="supp"><p class="details"><div>The ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> controls T-cell activation and anergy. The diagram illustrates the activation of the ERK <span class="highlight" style="background-color:">pathway</span> after T-cell receptor activation. Increased cytoplasmic concentrations of K-RAS and B-RAF, as seen in T cells from patients with RA, increase the activation of the ERK <span class="highlight" style="background-color:">pathway</span> and alleviate the need for CD28-mediated co-stimulation. Increased ERK activity prevents the induction of anergy by preventing SHP-1 recruitment to the T-cell receptor <span class="highlight" style="background-color:">signaling</span> complex, sustains T-cell receptor <span class="highlight" style="background-color:">signaling</span>, and may contribute to the autoreactivity to citrullinated self-antigens characteristic of RA.</div></p></div><div class="aux"><div class="resc">Karnail Singh, et al. Proc Natl Acad Sci U S A. 2012 June 19;109(25):E1629-E1637.</div><p class="links"><a class="dblinks" href="/pubmed/22615393" ref="ordinalpos=1539&amp;ncbi_uid=4451705&amp;link_uid=22615393">Citation</a><a class="dblinks" href="/pmc/articles/PMC3382540/" ref="ordinalpos=1539&amp;ncbi_uid=4451705&amp;link_uid=PMC3382540">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1540.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4225461/figure/Fig1/" ref="ordinalpos=1540&amp;ncbi_uid=6833057&amp;link_uid=PMC4225461" href="/pmc/articles/PMC4225461/"><img src="/pmc/articles/instance/4225461/bin/13238_2014_98_Fig1_HTML.gif" src-large="/pmc/articles/instance/4225461/bin/13238_2014_98_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4225461/" ref="ordinalpos=1540&amp;ncbi_uid=6833057&amp;link_uid=PMC4225461" image-link="/pmc/articles/PMC4225461/figure/Fig1/" class="imagepopup">Figure 1.  From: A partial <span class="highlight" style="background-color:">pathway</span>- and network-based transformation reveals the synergistic mechanism of JA and UA against cerebral ischemia-reperfusion injury. </a></p><div class="supp"><p class="details"><div> <b>Statistically significant <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> networks between JU and vehicle groups</b>. Statistically significant <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> networks involving the differentially expressed genes between JU and vehicle groups. (A) Network 1; (B) Network 2; (C) Network 3; (D) Network 4. The red node denotes an up-regulated gene, and the green node denotes a down-regulated gene. The intensity of colours indicates the degree of up-regulation or down-regulation</div></p></div><div class="aux"><div class="resc">Shanshan Guo, et al. Protein Cell. 2014 November;5(11):873-877.</div><p class="links"><a class="dblinks" href="/pubmed/25217044" ref="ordinalpos=1540&amp;ncbi_uid=6833057&amp;link_uid=25217044">Citation</a><a class="dblinks" href="/pmc/articles/PMC4225461/" ref="ordinalpos=1540&amp;ncbi_uid=6833057&amp;link_uid=PMC4225461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1541.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694555/figure/F1/" ref="ordinalpos=1541&amp;ncbi_uid=1936009&amp;link_uid=PMC2694555" href="/pmc/articles/PMC2694555/"><img src="/pmc/articles/instance/2694555/bin/CG-9-349_F1.gif" src-large="/pmc/articles/instance/2694555/bin/CG-9-349_F1.jpg" alt="Fig. (1)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694555/" ref="ordinalpos=1541&amp;ncbi_uid=1936009&amp;link_uid=PMC2694555" image-link="/pmc/articles/PMC2694555/figure/F1/" class="imagepopup">Fig. (1).  From: Can Systems Biology Understand <span class="highlight" style="background-color:">Pathway</span> Activation? Gene Expression Signatures as Surrogate Markers for Understanding the Complexity of <span class="highlight" style="background-color:">Pathway</span> Activation. </a></p><div class="supp"><p class="details"><div>Methods to develop <span class="highlight" style="background-color:">pathway</span> signatures. Gene expression regulated by <span class="highlight" style="background-color:">signaling</span> pathways can be detected by comparing <span class="highlight" style="background-color:">pathway</span>-activated <i>vs</i>. quiescent cells or baseline-level <i>vs</i>. inactivated cells using gene expression microarrays (See text for detail).</div></p></div><div class="aux"><div class="resc">Hiraku Itadani, et al. Curr Genomics. 2008 August;9(5):349-360.</div><p class="links"><a class="dblinks" href="/pubmed/19517027" ref="ordinalpos=1541&amp;ncbi_uid=1936009&amp;link_uid=19517027">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694555/" ref="ordinalpos=1541&amp;ncbi_uid=1936009&amp;link_uid=PMC2694555">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1542.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3877642/figure/fig2/" ref="ordinalpos=1542&amp;ncbi_uid=5713246&amp;link_uid=PMC3877642" href="/pmc/articles/PMC3877642/"><img src="/pmc/articles/instance/3877642/bin/JDR2013-946432.002.gif" src-large="/pmc/articles/instance/3877642/bin/JDR2013-946432.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3877642/" ref="ordinalpos=1542&amp;ncbi_uid=5713246&amp;link_uid=PMC3877642" image-link="/pmc/articles/PMC3877642/figure/fig2/" class="imagepopup">Figure 2.  From: Effects of Acute Exercise and Chronic Exercise on the Liver Leptin-AMPK-ACC <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Rats with Type 2 Diabetes. </a></p><div class="supp"><p class="details"><div>Western blotting images of protein expression and protein phosphorylation related to the leptin-AMPK-ACC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of different groups of type 2 diabetes rats. (a) Leptin and its receptor. (b) AMPK<i>α</i>1/2 and p-AMPK<i>α</i>1/2. (c) AMPK<i>α</i>1 and p-AMPK<i>α</i>1. (d) AMPK<i>α</i>2 and p-AMPK<i>α</i>2. (e) ACC and p-ACC. (f) GADPH as an internal standard.</div></p></div><div class="aux"><div class="resc">Xuejie Yi, et al. J Diabetes Res. 2013;2013:946432.</div><p class="links"><a class="dblinks" href="/pubmed/24455748" ref="ordinalpos=1542&amp;ncbi_uid=5713246&amp;link_uid=24455748">Citation</a><a class="dblinks" href="/pmc/articles/PMC3877642/" ref="ordinalpos=1542&amp;ncbi_uid=5713246&amp;link_uid=PMC3877642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1543.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2990205/figure/F7/" ref="ordinalpos=1543&amp;ncbi_uid=3014931&amp;link_uid=PMC2990205" href="/pmc/articles/PMC2990205/"><img src="/pmc/articles/instance/2990205/bin/DEV060483F7.gif" src-large="/pmc/articles/instance/2990205/bin/DEV060483F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2990205/" ref="ordinalpos=1543&amp;ncbi_uid=3014931&amp;link_uid=PMC2990205" image-link="/pmc/articles/PMC2990205/figure/F7/" class="imagepopup">Fig. 7. From: The Hippo tumor suppressor <span class="highlight" style="background-color:">pathway</span> regulates intestinal stem cell regeneration. </a></p><div class="supp"><p class="details"><div><b>Hippo <span class="highlight" style="background-color:">signaling</span> restrains Yki-accelerated cell division in response to injury.</b> Schematic shows the transition of ISCs between normal homeostasis, which occurs in the absence of injury, and acute regeneration following injury. Upon damage, the Hippo <span class="highlight" style="background-color:">pathway</span> ligand, Ds, no longer activates Hippo <span class="highlight" style="background-color:">signaling</span> in ISCs, allowing Yki to activate downstream targets. One of these targets is the Upd/Os cytokine, which stimulates proliferation of ISCs/Ebs through the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. When ee and EC progeny are regenerated, Hippo <span class="highlight" style="background-color:">signaling</span> acts as a brake, cytokine production slows and the system returns to the homeostatic state.</div></p></div><div class="aux"><div class="resc">Phillip Karpowicz, et al. Development. 2010 December 15;137(24):4135-4145.</div><p class="links"><a class="dblinks" href="/pubmed/21098564" ref="ordinalpos=1543&amp;ncbi_uid=3014931&amp;link_uid=21098564">Citation</a><a class="dblinks" href="/pmc/articles/PMC2990205/" ref="ordinalpos=1543&amp;ncbi_uid=3014931&amp;link_uid=PMC2990205">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1544.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3845579/figure/cjc-32-08-427-g001/" ref="ordinalpos=1544&amp;ncbi_uid=5588561&amp;link_uid=PMC3845579" href="/pmc/articles/PMC3845579/"><img src="/pmc/articles/instance/3845579/bin/cjc-32-08-427-g001.gif" src-large="/pmc/articles/instance/3845579/bin/cjc-32-08-427-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3845579/" ref="ordinalpos=1544&amp;ncbi_uid=5588561&amp;link_uid=PMC3845579" image-link="/pmc/articles/PMC3845579/figure/cjc-32-08-427-g001/" class="imagepopup">Figure 1.The regulation of mammalian target of rapamycin (mTOR) by the LKB1/AMPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Dysregulation of mTOR activity through LKB1 inactivation. </a></p><div class="supp"><p class="details"><div>Environmental stress leads to the activation of AMP-activated protein kinase (AMPK) by liver kinase B1 (LKB1), which subsequently activates hamartin (TSC1)/tuberin (TSC2) complex. The activation of this <span class="highlight" style="background-color:">pathway</span> down-regulates mTOR activity, which leads to cell growth arrest. mTOR activity can also be regulated by growth factors through the PI3K/PTEN/AKT <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Wei Zhou, et al. Chin J Cancer. 2013 August;32(8):427-433.</div><p class="links"><a class="dblinks" href="/pubmed/23668926" ref="ordinalpos=1544&amp;ncbi_uid=5588561&amp;link_uid=23668926">Citation</a><a class="dblinks" href="/pmc/articles/PMC3845579/" ref="ordinalpos=1544&amp;ncbi_uid=5588561&amp;link_uid=PMC3845579">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1545.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3913963/figure/F5/" ref="ordinalpos=1545&amp;ncbi_uid=5788766&amp;link_uid=PMC3913963" href="/pmc/articles/PMC3913963/"><img src="/pmc/articles/instance/3913963/bin/1756-8722-7-13-5.gif" src-large="/pmc/articles/instance/3913963/bin/1756-8722-7-13-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3913963/" ref="ordinalpos=1545&amp;ncbi_uid=5788766&amp;link_uid=PMC3913963" image-link="/pmc/articles/PMC3913963/figure/F5/" class="imagepopup">Figure 5.  From: <span class="highlight" style="background-color:">Signaling</span> pathways in the development of infantile hemangioma. </a></p><div class="supp"><p class="details"><div><b>β-adrenergic <span class="highlight" style="background-color:">signaling</span> modulates multiple cellular processes in tumor progression and metastasis.</b> The ligation of β-ARs by epinephrine or norepinephrine triggers a G-protein coupled <span class="highlight" style="background-color:">signaling</span> cascade that stimulates cAMP synthesis. cAMP activates the PKA protein, which can mediate multiple signal pathways via the phosphorylation of various downstream signal proteins. In another major <span class="highlight" style="background-color:">pathway</span>, the cAMP activation of EPAC leads to the Rap1A-mediated activation of Raf/MAPK <span class="highlight" style="background-color:">signaling</span> pathways and downstream effects on diverse cellular processes.</div></p></div><div class="aux"><div class="resc">Yi Ji, et al. J Hematol Oncol. 2014;7:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/24479731" ref="ordinalpos=1545&amp;ncbi_uid=5788766&amp;link_uid=24479731">Citation</a><a class="dblinks" href="/pmc/articles/PMC3913963/" ref="ordinalpos=1545&amp;ncbi_uid=5788766&amp;link_uid=PMC3913963">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1546.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2528845/figure/F3/" ref="ordinalpos=1546&amp;ncbi_uid=1571035&amp;link_uid=PMC2528845" href="/pmc/articles/PMC2528845/"><img src="/pmc/articles/instance/2528845/bin/zef0020826790003.gif" src-large="/pmc/articles/instance/2528845/bin/zef0020826790003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2528845/" ref="ordinalpos=1546&amp;ncbi_uid=1571035&amp;link_uid=PMC2528845" image-link="/pmc/articles/PMC2528845/figure/F3/" class="imagepopup">Figure 3.  From: Cellular <span class="highlight" style="background-color:">Signaling</span> by Fibroblast Growth Factors (FGFs) and Their Receptors (FGFRs) in Male Reproduction. </a></p><div class="supp"><p class="details"><div>Regulation of FGF <span class="highlight" style="background-color:">signaling</span>. The stimulation of FGFR by FGF ligands results in the activation of specific downstream target genes. Among the targets of the FGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are several feedback inhibitors, which are involved in the attenuation of FGF <span class="highlight" style="background-color:">signaling</span> and are part of the FGF synexpression group. Spry acts at the level of Grb2 and/or the level of Raf. Sef and XFLRT3 are both located at the membrane and can interact directly with FGFR. Sef functions as a negative regulator, whereas XFLRT3 enhances FGF <span class="highlight" style="background-color:">signaling</span> resulting in the phosphorylation of the MAPK ERK. Sef has also been shown to affect the phosphorylation of the MAPK cascade, either at the level of ERK or by preventing the phosphorylation of ERK by MEK (99). MKP3 negatively regulates FGF <span class="highlight" style="background-color:">signaling</span> by dephosphorylation of activated ERK. P, Phosphorylation. [Adapted from Ref. 43 with permission from Elsevier.]</div></p></div><div class="aux"><div class="resc">Leanne M. Cotton, et al. Endocr Rev. 2008 April;29(2):193-216.</div><p class="links"><a class="dblinks" href="/pubmed/18216218" ref="ordinalpos=1546&amp;ncbi_uid=1571035&amp;link_uid=18216218">Citation</a><a class="dblinks" href="/pmc/articles/PMC2528845/" ref="ordinalpos=1546&amp;ncbi_uid=1571035&amp;link_uid=PMC2528845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1547.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3442865/figure/F1/" ref="ordinalpos=1547&amp;ncbi_uid=4414130&amp;link_uid=PMC3442865" href="/pmc/articles/PMC3442865/"><img src="/pmc/articles/instance/3442865/bin/psb-7-675-g1.gif" src-large="/pmc/articles/instance/3442865/bin/psb-7-675-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3442865/" ref="ordinalpos=1547&amp;ncbi_uid=4414130&amp;link_uid=PMC3442865" image-link="/pmc/articles/PMC3442865/figure/F1/" class="imagepopup">Figure. From: TOR <span class="highlight" style="background-color:">pathway</span> activation in Zea mays L. tissues. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Putative plant PI3K/TOR-S6K <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Pathway</span> activation by plant IGFs or insulin turns on a <span class="highlight" style="background-color:">signaling</span> cascade of reactions through the phosphorylation of kinases, mainly TOR and S6K, ultimately targeting ribosome synthesis. Arrowheads denote activation and truncated lines inhibition, interrogation marks denote no confirmed function (Modified from Sotelo et al.). 18 </div></p></div><div class="aux"><div class="resc">Verónica Garrocho-Villegas, et al. Plant Signal Behav. 2012 June 1;7(6):675-677.</div><p class="links"><a class="dblinks" href="/pubmed/22580703" ref="ordinalpos=1547&amp;ncbi_uid=4414130&amp;link_uid=22580703">Citation</a><a class="dblinks" href="/pmc/articles/PMC3442865/" ref="ordinalpos=1547&amp;ncbi_uid=4414130&amp;link_uid=PMC3442865">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1548.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3186766/figure/pone-0024371-g004/" ref="ordinalpos=1548&amp;ncbi_uid=3275313&amp;link_uid=PMC3186766" href="/pmc/articles/PMC3186766/"><img src="/pmc/articles/instance/3186766/bin/pone.0024371.g004.gif" src-large="/pmc/articles/instance/3186766/bin/pone.0024371.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3186766/" ref="ordinalpos=1548&amp;ncbi_uid=3275313&amp;link_uid=PMC3186766" image-link="/pmc/articles/PMC3186766/figure/pone-0024371-g004/" class="imagepopup">Figure 4. IL-15 Production <span class="highlight" style="background-color:">Pathway</span> and IL-22 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>..  From: Gene Expression Analysis in the Thalamus and Cerebrum of Horses Experimentally Infected with West Nile Virus. </a></p><div class="supp"><p class="details"><div>(A) The diagrams represent the different methods of IL-15 production. Green represents downregulation of pertinent molecules, red represents upregulation of pertinent molecules. WNV induces upregulation of the production of IL-15. (B) Red represents upregulation of gene expression, green represents down-regulation of gene expression. The JAK/STAT portion of the <span class="highlight" style="background-color:">pathway</span> is upregulated during viral infection, demonstrating an innate immune response. However, the SOCS3 molecule is also upregulated, indicating that infection with the virus may lead to subsequent suppression of the JAK/STAT <span class="highlight" style="background-color:">pathway</span> and evasion of the innate immune response. This network was generated through the use of IPA (Ingenuity Systems, www.ingenuity.com).</div></p></div><div class="aux"><div class="resc">Melissa A. Bourgeois, et al. PLoS One. 2011;6(10):e24371.</div><p class="links"><a class="dblinks" href="/pubmed/21991302" ref="ordinalpos=1548&amp;ncbi_uid=3275313&amp;link_uid=21991302">Citation</a><a class="dblinks" href="/pmc/articles/PMC3186766/" ref="ordinalpos=1548&amp;ncbi_uid=3275313&amp;link_uid=PMC3186766">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1549.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2447903/figure/fig8/" ref="ordinalpos=1549&amp;ncbi_uid=1436466&amp;link_uid=PMC2447903" href="/pmc/articles/PMC2447903/"><img src="/pmc/articles/instance/2447903/bin/jcb1820171f08.gif" src-large="/pmc/articles/instance/2447903/bin/jcb1820171f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2447903/" ref="ordinalpos=1549&amp;ncbi_uid=1436466&amp;link_uid=PMC2447903" image-link="/pmc/articles/PMC2447903/figure/fig8/" class="imagepopup">Figure 8. From: TWEAK-FN14 <span class="highlight" style="background-color:">signaling</span> induces lysosomal degradation of a cIAP1-TRAF2 complex to sensitize tumor cells to TNF?. </a></p><div class="supp"><p class="details"><div><b>Model for TWEAK killing.</b> TWEAK binding to FN14 induces the recruitment of a cIAP1–TRAF2 complex. This complex is then targeted for lysosomal degradation by the MVB <span class="highlight" style="background-color:">pathway</span>. The loss of cIAP1 or TRAF2 induces the stabilization of NIK and subsequent activation of noncanonical NF-κB. The canonical NF-κB <span class="highlight" style="background-color:">pathway</span> is also activated in response to TWEAK. Activation of NF-κB drives autocrine TNFα production in TWEAK-sensitive cell lines. The loss of cellular cIAP1 and TRAF2 by FN14 <span class="highlight" style="background-color:">signaling</span> prevents their recruitment to TNF-R1 upon autocrine TNFα binding, and this promotes the formation of a death-inducing complex and apoptotic <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">James E. Vince, et al. J Cell Biol. 2008 July 14;182(1):171-184.</div><p class="links"><a class="dblinks" href="/pubmed/18606850" ref="ordinalpos=1549&amp;ncbi_uid=1436466&amp;link_uid=18606850">Citation</a><a class="dblinks" href="/pmc/articles/PMC2447903/" ref="ordinalpos=1549&amp;ncbi_uid=1436466&amp;link_uid=PMC2447903">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1550.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3885266/figure/fig4/" ref="ordinalpos=1550&amp;ncbi_uid=5714384&amp;link_uid=PMC3885266" href="/pmc/articles/PMC3885266/"><img src="/pmc/articles/instance/3885266/bin/TSWJ2013-178708.004.gif" src-large="/pmc/articles/instance/3885266/bin/TSWJ2013-178708.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3885266/" ref="ordinalpos=1550&amp;ncbi_uid=5714384&amp;link_uid=PMC3885266" image-link="/pmc/articles/PMC3885266/figure/fig4/" class="imagepopup">Figure 4.  From: Blockage of Notch <span class="highlight" style="background-color:">Signaling</span> Inhibits the Migration and Proliferation of Retinal Pigment Epithelial Cells. </a></p><div class="supp"><p class="details"><div>Inhibition of Notch <span class="highlight" style="background-color:">signaling</span> reduced the expression of the Notch <span class="highlight" style="background-color:">signaling</span> target genes. The downregulation of four members of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (HES1, MYC, HEY2, and SOX9) was determined using real-time RT-PCR. Data are expressed as means ± SD. *<i>P</i> &lt; 0.05 versus vehicle and error bars represent at least two independent experiments.</div></p></div><div class="aux"><div class="resc">Weiwei Liu, et al. ScientificWorldJournal. 2013;2013:178708.</div><p class="links"><a class="dblinks" href="/pubmed/24453806" ref="ordinalpos=1550&amp;ncbi_uid=5714384&amp;link_uid=24453806">Citation</a><a class="dblinks" href="/pmc/articles/PMC3885266/" ref="ordinalpos=1550&amp;ncbi_uid=5714384&amp;link_uid=PMC3885266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1551.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137146/figure/fig1/" ref="ordinalpos=1551&amp;ncbi_uid=3095276&amp;link_uid=PMC3137146" href="/pmc/articles/PMC3137146/"><img src="/pmc/articles/instance/3137146/bin/PROCATS7122f1.gif" src-large="/pmc/articles/instance/3137146/bin/PROCATS7122f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137146/" ref="ordinalpos=1551&amp;ncbi_uid=3095276&amp;link_uid=PMC3137146" image-link="/pmc/articles/PMC3137146/figure/fig1/" class="imagepopup">Figure 1. From: Autophagy in Innate and Adaptive Immunity. </a></p><div class="supp"><p class="details"><div>Model of LPS-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for autophagy as predicted from our recent data (6) and from recent studies. LPSs induce autophagy by stimulation of the TLR4-TIR-domain-containing adapter-inducing interferon-β (TRIF)-receptor interacting protein (RIP) 11 <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">signaling</span> components directly downstream of RIP still need to be identified. Nevertheless, p38 mitogen-activated protein kinase is required for LPS-induced autophagy and for survival. This <span class="highlight" style="background-color:">pathway</span> seems to be in contrast to the c-Jun-N-terminal kinase (JNK)-mediated autophagic <span class="highlight" style="background-color:">pathway</span> previously described (25). Reproduced by permission from Reference 23.</div></p></div><div class="aux"><div class="resc">Yi Xu, et al. Proc Am Thorac Soc. 2010 February 15;7(1):22-28.</div><p class="links"><a class="dblinks" href="/pubmed/20160145" ref="ordinalpos=1551&amp;ncbi_uid=3095276&amp;link_uid=20160145">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137146/" ref="ordinalpos=1551&amp;ncbi_uid=3095276&amp;link_uid=PMC3137146">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1552.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3130749/figure/pone-0021504-g001/" ref="ordinalpos=1552&amp;ncbi_uid=3102710&amp;link_uid=PMC3130749" href="/pmc/articles/PMC3130749/"><img src="/pmc/articles/instance/3130749/bin/pone.0021504.g001.gif" src-large="/pmc/articles/instance/3130749/bin/pone.0021504.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3130749/" ref="ordinalpos=1552&amp;ncbi_uid=3102710&amp;link_uid=PMC3130749" image-link="/pmc/articles/PMC3130749/figure/pone-0021504-g001/" class="imagepopup">Figure 1. Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Components in Mosquitoes..  From: Comparative Genomic Analysis of Drosophila melanogaster and Vector Mosquito Developmental Genes. </a></p><div class="supp"><p class="details"><div>The canonical Wnt <span class="highlight" style="background-color:">pathway</span> is summarized in this figure. Wnt binds its receptor Fz, which activates Dsh, an inhibitor of GSK-3. GSK-3 normally prevents dissociation of β-catenin from APC; when GSK-3 is inhibited, β-catenin enters the nucleus and regulates transcription (reviewed in [28]). Analysis of <span class="highlight" style="background-color:">pathway</span> members uncovered mosquito lineage specific changes in the number of orthologous sequences for genes encoding various Wnt <span class="highlight" style="background-color:">pathway</span> components in mosquitoes. With respect to <i>D. melanogaster</i>, mosquito lineage specific gene absences (−), as well as increases in the number of orthologues (+) were noted for particular <span class="highlight" style="background-color:">pathway</span> members. Details are provided in the text.</div></p></div><div class="aux"><div class="resc">Susanta K. Behura, et al. PLoS One. 2011;6(7):e21504.</div><p class="links"><a class="dblinks" href="/pubmed/21754989" ref="ordinalpos=1552&amp;ncbi_uid=3102710&amp;link_uid=21754989">Citation</a><a class="dblinks" href="/pmc/articles/PMC3130749/" ref="ordinalpos=1552&amp;ncbi_uid=3102710&amp;link_uid=PMC3130749">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1553.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1766433/figure/F8/" ref="ordinalpos=1553&amp;ncbi_uid=799443&amp;link_uid=PMC1766433" href="/pmc/articles/PMC1766433/"><img src="/pmc/articles/instance/1766433/bin/zpq0510645340008.gif" src-large="/pmc/articles/instance/1766433/bin/zpq0510645340008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1766433/" ref="ordinalpos=1553&amp;ncbi_uid=799443&amp;link_uid=PMC1766433" image-link="/pmc/articles/PMC1766433/figure/F8/" class="imagepopup">Fig. 8. From: Protective effects of exercise and phosphoinositide 3-kinase(p110?) <span class="highlight" style="background-color:">signaling</span> in dilated and hypertrophic cardiomyopathy. </a></p><div class="supp"><p class="details"><div>Schematic illustrates physiological and pathological <span class="highlight" style="background-color:">signaling</span> cascades. A physiological stimulus acting via the IGF1-PI3K(p110α) <span class="highlight" style="background-color:">pathway</span> may inhibit <span class="highlight" style="background-color:">signaling</span> molecules downstream of GPCRs activated by pathological stimuli. Ang II, angiotensin II. Dotted line indicates it may not be a direct interaction. * indicates differential regulation of Akt by GPCRs vs. RTKs (31).</div></p></div><div class="aux"><div class="resc">Julie R. McMullen, et al. Proc Natl Acad Sci U S A. 2007 January 9;104(2):612-617.</div><p class="links"><a class="dblinks" href="/pubmed/17202264" ref="ordinalpos=1553&amp;ncbi_uid=799443&amp;link_uid=17202264">Citation</a><a class="dblinks" href="/pmc/articles/PMC1766433/" ref="ordinalpos=1553&amp;ncbi_uid=799443&amp;link_uid=PMC1766433">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1554.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3146004/figure/F4/" ref="ordinalpos=1554&amp;ncbi_uid=3147631&amp;link_uid=PMC3146004" href="/pmc/articles/PMC3146004/"><img src="/pmc/articles/instance/3146004/bin/lar-27-161-g004.gif" src-large="/pmc/articles/instance/3146004/bin/lar-27-161-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3146004/" ref="ordinalpos=1554&amp;ncbi_uid=3147631&amp;link_uid=PMC3146004" image-link="/pmc/articles/PMC3146004/figure/F4/" class="imagepopup">Figure 4.  From: Transcriptome Analysis of the Effects of Gomisin A on the Recovery of Carbon Tetrachloride-Induced Damage in Rat Liver. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> assignment of the important genes based on KEGG. The genes of up-regulation in cell cycle <span class="highlight" style="background-color:">pathway</span> (A) and the genes of down-regulation in TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (B) were determined by KEGG <span class="highlight" style="background-color:">pathway</span>. The genes found in our microarray analysis were labeled by star symbols.</div></p></div><div class="aux"><div class="resc">Young Mi Choi, et al. Lab Anim Res. 2011 June;27(2):161-169.</div><p class="links"><a class="dblinks" href="/pubmed/21826177" ref="ordinalpos=1554&amp;ncbi_uid=3147631&amp;link_uid=21826177">Citation</a><a class="dblinks" href="/pmc/articles/PMC3146004/" ref="ordinalpos=1554&amp;ncbi_uid=3147631&amp;link_uid=PMC3146004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1555.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3808465/figure/F4/" ref="ordinalpos=1555&amp;ncbi_uid=5651381&amp;link_uid=PMC3808465" href="/pmc/articles/PMC3808465/"><img src="/pmc/articles/instance/3808465/bin/nihms507169f4.gif" src-large="/pmc/articles/instance/3808465/bin/nihms507169f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3808465/" ref="ordinalpos=1555&amp;ncbi_uid=5651381&amp;link_uid=PMC3808465" image-link="/pmc/articles/PMC3808465/figure/F4/" class="imagepopup">Figure 4.  From: Dioxins: diagnostic and prognostic challenges arising from complex mechanisms. </a></p><div class="supp"><p class="details"><div>The AHR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Dioxins or endogenous ligands bind to the AHR in the cytosol, where it is translocated to the nucleus and dimerizes with the ARNT and directs transcription from dioxin response elements (DREs). Alternatively, dioxin <span class="highlight" style="background-color:">signaling</span> can be mediated through a ‘non-genomic’ <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> is initiated by ligand activated AHR, but lacks the requirement for nuclear translocation or ARNT. In this <span class="highlight" style="background-color:">pathway</span>, TCDD exposure can increase intracellular Ca<sup>2+</sup> elevations through an undefined mechanism.</div></p></div><div class="aux"><div class="resc">Noel M. Rysavy, et al. J Appl Toxicol. 2013 January;33(1):10.1002/jat.2759.</div><p class="links"><a class="dblinks" href="/pubmed/22610997" ref="ordinalpos=1555&amp;ncbi_uid=5651381&amp;link_uid=22610997">Citation</a><a class="dblinks" href="/pmc/articles/PMC3808465/" ref="ordinalpos=1555&amp;ncbi_uid=5651381&amp;link_uid=PMC3808465">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1556.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3095696/figure/F3/" ref="ordinalpos=1556&amp;ncbi_uid=3541677&amp;link_uid=PMC3095696" href="/pmc/articles/PMC3095696/"><img src="/pmc/articles/instance/3095696/bin/nihms270198f3.gif" src-large="/pmc/articles/instance/3095696/bin/nihms270198f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3095696/" ref="ordinalpos=1556&amp;ncbi_uid=3541677&amp;link_uid=PMC3095696" image-link="/pmc/articles/PMC3095696/figure/F3/" class="imagepopup">Figure 3.  From: Structure, function and regulation of the melanocortin receptors. </a></p><div class="supp"><p class="details"><div>Schematic representation of melanocortin receptor agonist-selective <span class="highlight" style="background-color:">signaling</span>. Agonist I and II exhibit different efficacies in <span class="highlight" style="background-color:">pathway</span> A and B. Agonist I is a full agonist for <span class="highlight" style="background-color:">pathway</span> A and B. Agonist II is a full agonist for only <span class="highlight" style="background-color:">pathway</span> B.</div></p></div><div class="aux"><div class="resc">Yingkui Yang. Eur J Pharmacol. ;660(1):125-130.</div><p class="links"><a class="dblinks" href="/pubmed/21208602" ref="ordinalpos=1556&amp;ncbi_uid=3541677&amp;link_uid=21208602">Citation</a><a class="dblinks" href="/pmc/articles/PMC3095696/" ref="ordinalpos=1556&amp;ncbi_uid=3541677&amp;link_uid=PMC3095696">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1557.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3491533/figure/fig8/" ref="ordinalpos=1557&amp;ncbi_uid=4352562&amp;link_uid=PMC3491533" href="/pmc/articles/PMC3491533/"><img src="/pmc/articles/instance/3491533/bin/ctg201217f8.gif" src-large="/pmc/articles/instance/3491533/bin/ctg201217f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3491533/" ref="ordinalpos=1557&amp;ncbi_uid=4352562&amp;link_uid=PMC3491533" image-link="/pmc/articles/PMC3491533/figure/fig8/" class="imagepopup">Figure 8.  From: Analysis of Notch and TGF-β <span class="highlight" style="background-color:">Signaling</span> Expression in Different Stages of Disease Progression During Hepatitis B Virus Infection. </a></p><div class="supp"><p class="details"><div>Diagrammatic presentation shows how Notch and TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved at each pathogenic stage of HBV infection. <span class="highlight" style="background-color:">Signaling</span> gene sets were activated in periphery of acute HBV (AVH-B)-infected patients, which were attenuated during the chronic stage of infection and again activated during cirrhosis and end stage of liver disease.</div></p></div><div class="aux"><div class="resc">Nirupama Trehanpati, et al. Clin Transl Gastroenterol. 2012 October;3(10):e23.</div><p class="links"><a class="dblinks" href="/pubmed/23238065" ref="ordinalpos=1557&amp;ncbi_uid=4352562&amp;link_uid=23238065">Citation</a><a class="dblinks" href="/pmc/articles/PMC3491533/" ref="ordinalpos=1557&amp;ncbi_uid=4352562&amp;link_uid=PMC3491533">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1558.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2966078/figure/F6/" ref="ordinalpos=1558&amp;ncbi_uid=3336434&amp;link_uid=PMC2966078" href="/pmc/articles/PMC2966078/"><img src="/pmc/articles/instance/2966078/bin/zbc0461036750006.gif" src-large="/pmc/articles/instance/2966078/bin/zbc0461036750006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2966078/" ref="ordinalpos=1558&amp;ncbi_uid=3336434&amp;link_uid=PMC2966078" image-link="/pmc/articles/PMC2966078/figure/F6/" class="imagepopup">FIGURE 6. From: p15RS Attenuates Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> by Disrupting ?-Catenin?TCF4 Interaction. </a></p><div class="supp"><p class="details"><div><b>A model to demonstrate the inhibitory role of p15RS on the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> In the quiescent state (without Wnt), p15RS interacts with TCF4 on the promoters of Wnt-target genes and inhibits transcription. When Wnt <span class="highlight" style="background-color:">signaling</span> is activated, β-catenin translocates into the nucleus to interact with p15RS thereafter removing p15RS from its association with TCF4, allowing β-catenin·TCF4 binding to the promoter to initiate gene transcription.</div></p></div><div class="aux"><div class="resc">Yinyuan Wu, et al. J Biol Chem. 2010 November 5;285(45):34621-34631.</div><p class="links"><a class="dblinks" href="/pubmed/20739273" ref="ordinalpos=1558&amp;ncbi_uid=3336434&amp;link_uid=20739273">Citation</a><a class="dblinks" href="/pmc/articles/PMC2966078/" ref="ordinalpos=1558&amp;ncbi_uid=3336434&amp;link_uid=PMC2966078">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1559.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC130620/figure/F5/" ref="ordinalpos=1559&amp;ncbi_uid=252088&amp;link_uid=PMC130620" href="/pmc/articles/PMC130620/"><img src="/pmc/articles/instance/130620/bin/pq2023557005.gif" src-large="/pmc/articles/instance/130620/bin/pq2023557005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC130620/" ref="ordinalpos=1559&amp;ncbi_uid=252088&amp;link_uid=PMC130620" image-link="/pmc/articles/PMC130620/figure/F5/" class="imagepopup">Figure 5.  From: Calpain as an effector of the Gq <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for inhibition of Wnt/?-catenin-regulated cell proliferation. </a></p><div class="supp"><p class="details"><div>Schematic representation of the interactions between the Gq and Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> pathways. As shown in the model, APC promotes β-catenin nuclear export and facilitates proteasome-mediated degradation of β-catenin after sequential phosphorylation by CKIα and GSK-3β on the Axin-based complex. Wnt binding to coreceptors LRP and Frizzled blocks proteasome-mediated degradation, which further leads to nuclear localization of β-catenin, gene expression, and cell proliferation. As an alternative degradation <span class="highlight" style="background-color:">pathway</span>, Ca<sup>2+</sup> activation by Gq <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promotes nuclear export of β-catenin into cytoplasm, where it is further degraded by calpain in a Ca<sup>2+</sup>-dependent manner. Consequently, β-catenin–TCF/LEF-dependent cell proliferation is inhibited. LRP, low-density-lipoprotein receptor-related <i>p</i>rotein; UPS, ubiquitin-proteasome system; PLCβ, phospholipase C β; PIP<sub>2</sub>, phosphatidylinositol 4,5-bisphosphate; IP<sub>3</sub>R, IP<sub>3</sub> receptor.</div></p></div><div class="aux"><div class="resc">Guangnan Li, et al. Proc Natl Acad Sci U S A. 2002 October 1;99(20):13254-13259.</div><p class="links"><a class="dblinks" href="/pubmed/12239346" ref="ordinalpos=1559&amp;ncbi_uid=252088&amp;link_uid=12239346">Citation</a><a class="dblinks" href="/pmc/articles/PMC130620/" ref="ordinalpos=1559&amp;ncbi_uid=252088&amp;link_uid=PMC130620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1560.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2553439/figure/F2/" ref="ordinalpos=1560&amp;ncbi_uid=1767304&amp;link_uid=PMC2553439" href="/pmc/articles/PMC2553439/"><img src="/pmc/articles/instance/2639041/bin/btn403f2.gif" src-large="/pmc/articles/instance/2639041/bin/btn403f2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2553439/" ref="ordinalpos=1560&amp;ncbi_uid=1767304&amp;link_uid=PMC2553439" image-link="/pmc/articles/PMC2553439/figure/F2/" class="imagepopup">Fig. 2. From: Predicting <span class="highlight" style="background-color:">pathway</span> membership via domain signatures. </a></p><div class="supp"><p class="details"><div>Prediction performance of the hierarchical classification model on an external validation set for the pruned KEGG hierarchy (<b>A</b>, 2760 genes) and for <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components (<b>B</b>, 458 genes).</div></p></div><div class="aux"><div class="resc">Holger Fröhlich, et al. Bioinformatics. 2008 October 1;24(19):2137-2142.</div><p class="links"><a class="dblinks" href="/pubmed/18676972" ref="ordinalpos=1560&amp;ncbi_uid=1767304&amp;link_uid=18676972">Citation</a><a class="dblinks" href="/pmc/articles/PMC2553439/" ref="ordinalpos=1560&amp;ncbi_uid=1767304&amp;link_uid=PMC2553439">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1561.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3628025/figure/F7/" ref="ordinalpos=1561&amp;ncbi_uid=6081204&amp;link_uid=PMC3628025" href="/pmc/articles/PMC3628025/"><img src="/pmc/articles/instance/3628025/bin/zee9991369840007.gif" src-large="/pmc/articles/instance/3628025/bin/zee9991369840007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3628025/" ref="ordinalpos=1561&amp;ncbi_uid=6081204&amp;link_uid=PMC3628025" image-link="/pmc/articles/PMC3628025/figure/F7/" class="imagepopup">Figure 7. From: TGF-? Effects on Prostate Cancer Cell Migration and Invasion Are Mediated by PGE2 through Activation of PI3K/AKT/mTOR <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A proposed model suggesting that TGF-β increases COX-2 levels and PGE<sub>2</sub> secretion in prostate cancer cells and both TGF-β and PGE<sub>2</sub> activate PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">BaoHan T. Vo, et al. Endocrinology. 2013 May;154(5):1768-1779.</div><p class="links"><a class="dblinks" href="/pubmed/23515290" ref="ordinalpos=1561&amp;ncbi_uid=6081204&amp;link_uid=23515290">Citation</a><a class="dblinks" href="/pmc/articles/PMC3628025/" ref="ordinalpos=1561&amp;ncbi_uid=6081204&amp;link_uid=PMC3628025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1562.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2515524/figure/fig8/" ref="ordinalpos=1562&amp;ncbi_uid=1477534&amp;link_uid=PMC2515524" href="/pmc/articles/PMC2515524/"><img src="/pmc/articles/instance/2515524/bin/JLR4991981f8.gif" src-large="/pmc/articles/instance/2515524/bin/JLR4991981f8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2515524/" ref="ordinalpos=1562&amp;ncbi_uid=1477534&amp;link_uid=PMC2515524" image-link="/pmc/articles/PMC2515524/figure/fig8/" class="imagepopup">Fig. 8. From: TGF?1, TNF?, and insulin <span class="highlight" style="background-color:">signaling</span> crosstalk in regulation of the rat cholesterol 7?-hydroxylase gene expression. </a></p><div class="supp"><p class="details"><div>A model of insulin/FoxO1, TGFβ1/Smad, and TNFα/cJun <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> crosstalk in coordinated regulation of bile acid synthesis in the cholestatic liver injury, fibrogenesis, and regeneration.</div></p></div><div class="aux"><div class="resc">Tiangang Li, et al. J Lipid Res. 2008 September;49(9):1981-1989.</div><p class="links"><a class="dblinks" href="/pubmed/18511845" ref="ordinalpos=1562&amp;ncbi_uid=1477534&amp;link_uid=18511845">Citation</a><a class="dblinks" href="/pmc/articles/PMC2515524/" ref="ordinalpos=1562&amp;ncbi_uid=1477534&amp;link_uid=PMC2515524">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1563.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3127159/figure/F2/" ref="ordinalpos=1563&amp;ncbi_uid=3492448&amp;link_uid=PMC3127159" href="/pmc/articles/PMC3127159/"><img src="/pmc/articles/instance/3127159/bin/cc1010_1582_fig002.gif" src-large="/pmc/articles/instance/3127159/bin/cc1010_1582_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3127159/" ref="ordinalpos=1563&amp;ncbi_uid=3492448&amp;link_uid=PMC3127159" image-link="/pmc/articles/PMC3127159/figure/F2/" class="imagepopup">Figure 2.  From: Thrombopoietin and hematopoietic stem cells. </a></p><div class="supp"><p class="details"><div>TPO <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In unstimulated cells (A), Mpl (in yellow) exists as a monomer, and <span class="highlight" style="background-color:">signaling</span> molecules, such as JAK2 and STATs, are inactive. Upon TPO binding (B), Mpl dimerizes. This conformational change results in phosphorylation of its associated JAK2 molecules, which leads to phosphorylation of Mpl and activation of downstream <span class="highlight" style="background-color:">signaling</span> pathways, including STATs, the MAPK <span class="highlight" style="background-color:">pathway</span> and the PI3K <span class="highlight" style="background-color:">pathway</span>. JAK2 can phosphorylate STAT3 and STAT5 in response to TPO <span class="highlight" style="background-color:">signaling</span>, which results in their dimerization and translocation to the nucleus. There transcription is initiated, and the integration of these pathways leads to the downstream effect of TPO. STATs also induce transcription of SOCS proteins, which are generally not expressed, resulting in negative feedback and downregulation of JAK2 and Mpl activation.</div></p></div><div class="aux"><div class="resc">Carolyn A de Graaf, et al. Cell Cycle. 2011 May 15;10(10):1582-1589.</div><p class="links"><a class="dblinks" href="/pubmed/21478671" ref="ordinalpos=1563&amp;ncbi_uid=3492448&amp;link_uid=21478671">Citation</a><a class="dblinks" href="/pmc/articles/PMC3127159/" ref="ordinalpos=1563&amp;ncbi_uid=3492448&amp;link_uid=PMC3127159">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1564.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713831/figure/pone-0006410-g006/" ref="ordinalpos=1564&amp;ncbi_uid=1944669&amp;link_uid=PMC2713831" href="/pmc/articles/PMC2713831/"><img src="/pmc/articles/instance/2713831/bin/pone.0006410.g006.gif" src-large="/pmc/articles/instance/2713831/bin/pone.0006410.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713831/" ref="ordinalpos=1564&amp;ncbi_uid=1944669&amp;link_uid=PMC2713831" image-link="/pmc/articles/PMC2713831/figure/pone-0006410-g006/" class="imagepopup">Figure 6. Distribution patterns of eight different <span class="highlight" style="background-color:">signaling</span> pathways..  From: Local Network Topology in Human Protein Interaction Data Predicts Functional Association. </a></p><div class="supp"><p class="details"><div>(<i>a</i>) The largest subcluster of 959 proteins is derived from the root of the whole 1729-member cluster. Each protein in this subcluster has a coordinate with respect to its order in the 959 members (from left to right); a <span class="highlight" style="background-color:">pathway</span> distribution is generated from the distribution of its members' coordinates under the bandwidth of 10 (R 2.25; IPA 5.5). (<i>b</i>) Distribution of the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>c</i>) Distributions of death-receptor and NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. (<i>d</i>) Distributions of B- and T-cell receptor <span class="highlight" style="background-color:">signaling</span> pathways. (<i>e</i>) Distributions of insulin receptor, Fc epsilon RI and natural killer cell <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Hua Li, et al. PLoS One. 2009;4(7):e6410.</div><p class="links"><a class="dblinks" href="/pubmed/19641626" ref="ordinalpos=1564&amp;ncbi_uid=1944669&amp;link_uid=19641626">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713831/" ref="ordinalpos=1564&amp;ncbi_uid=1944669&amp;link_uid=PMC2713831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1565.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2696359/figure/F1/" ref="ordinalpos=1565&amp;ncbi_uid=1823176&amp;link_uid=PMC2696359" href="/pmc/articles/PMC2696359/"><img src="/pmc/articles/instance/2696359/bin/nihms54315f1.gif" src-large="/pmc/articles/instance/2696359/bin/nihms54315f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2696359/" ref="ordinalpos=1565&amp;ncbi_uid=1823176&amp;link_uid=PMC2696359" image-link="/pmc/articles/PMC2696359/figure/F1/" class="imagepopup">Figure 1. Occurrence of Early and Late Oscillator Genes Within the TNF <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: TNF-Induced MAP Kinase Activation Oscillates in Time. </a></p><div class="supp"><p class="details"><div>These oscillating components were discovered by performing a KEGG <span class="highlight" style="background-color:">Pathway</span> Analysis using Genesifter software of the two pre-selected clusters comprising early and late oscillators (see Table 1).</div></p></div><div class="aux"><div class="resc">Jameel Iqbal, et al. Biochem Biophys Res Commun. ;371(4):906-911.</div><p class="links"><a class="dblinks" href="/pubmed/18384751" ref="ordinalpos=1565&amp;ncbi_uid=1823176&amp;link_uid=18384751">Citation</a><a class="dblinks" href="/pmc/articles/PMC2696359/" ref="ordinalpos=1565&amp;ncbi_uid=1823176&amp;link_uid=PMC2696359">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1566.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4373269/figure/F1/" ref="ordinalpos=1566&amp;ncbi_uid=7403514&amp;link_uid=PMC4373269" href="/pmc/articles/PMC4373269/"><img src="/pmc/articles/instance/4373269/bin/fpls-06-00170-g001.gif" src-large="/pmc/articles/instance/4373269/bin/fpls-06-00170-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4373269/" ref="ordinalpos=1566&amp;ncbi_uid=7403514&amp;link_uid=PMC4373269" image-link="/pmc/articles/PMC4373269/figure/F1/" class="imagepopup">FIGURE 1.  From: How salicylic acid takes transcriptional control over jasmonic acid <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Simplified model of the molecular machinery involved in the transcriptional regulation of the SA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (A), the JA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (B), or the antagonism of SA on the JA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (C).</b> By inducing reduction and monomerization of NPR1, SA activates NPR1 (star-shaped), which then triggers gene expression in the nucleus. JA-responsive genes are kept in check by JAZ repressors in the absence of JA. In the presence of JA, MYC or ERF transcription factors activate JA-responsive genes, but only if SA is absent. Activation of both the SA and JA <span class="highlight" style="background-color:">signaling</span> pathways leads to antagonism of JA-responsive gene expression by SA. There are indications for roles in SA/JA crosstalk for cytosolic NPR1, and nuclear localized TGAs, GRX480, and WRKYs. See text for details on the molecular processes underlying the transcriptional control, like redox <span class="highlight" style="background-color:">signaling</span>, sequestration, degradation, phosphorylation, and chromatin modification. Solid lines indicate established (in)activities and dashed lines hypothesized (in)activities, where black arrows specify activation and red blocks suppression. Red crosses indicate that gene transcription is hampered.</div></p></div><div class="aux"><div class="resc">Lotte Caarls, et al. Front Plant Sci. 2015;6:170.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1566&amp;ncbi_uid=7403514&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4373269/" ref="ordinalpos=1566&amp;ncbi_uid=7403514&amp;link_uid=PMC4373269">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1567.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3298845/figure/F2/" ref="ordinalpos=1567&amp;ncbi_uid=3856584&amp;link_uid=PMC3298845" href="/pmc/articles/PMC3298845/"><img src="/pmc/articles/instance/3298845/bin/nihms358828f2.gif" src-large="/pmc/articles/instance/3298845/bin/nihms358828f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3298845/" ref="ordinalpos=1567&amp;ncbi_uid=3856584&amp;link_uid=PMC3298845" image-link="/pmc/articles/PMC3298845/figure/F2/" class="imagepopup">Figure 2. The many roles of the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> in the liver.  From: The Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in liver biology and disease. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">pathway</span> plays a critical role in early liver development, postnatal growth and regeneration. It is also important for many functions in the adult liver and its aberrant function has been implicated in the development of tumors.</div></p></div><div class="aux"><div class="resc">Jaideep Behari. Expert Rev Gastroenterol Hepatol. ;4(6):745-756.</div><p class="links"><a class="dblinks" href="/pubmed/21108594" ref="ordinalpos=1567&amp;ncbi_uid=3856584&amp;link_uid=21108594">Citation</a><a class="dblinks" href="/pmc/articles/PMC3298845/" ref="ordinalpos=1567&amp;ncbi_uid=3856584&amp;link_uid=PMC3298845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1568.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3205231/figure/F3/" ref="ordinalpos=1568&amp;ncbi_uid=3494593&amp;link_uid=PMC3205231" href="/pmc/articles/PMC3205231/"><img src="/pmc/articles/instance/3205231/bin/nihms311476f3.gif" src-large="/pmc/articles/instance/3205231/bin/nihms311476f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3205231/" ref="ordinalpos=1568&amp;ncbi_uid=3494593&amp;link_uid=PMC3205231" image-link="/pmc/articles/PMC3205231/figure/F3/" class="imagepopup">Figure 3. Phy <span class="highlight" style="background-color:">signaling</span> pathways to turn on photomorphogenesis.  From: Phytochrome <span class="highlight" style="background-color:">signaling</span> mechanisms and the control of plant development. </a></p><div class="supp"><p class="details"><div>There are two major <span class="highlight" style="background-color:">signaling</span> pathways: first, in the “blue” <span class="highlight" style="background-color:">pathway</span>, phys de-repress the positively acting transcriptional regulators, including HY5, LAF1 and HFR1, by inhibiting the COP1 and DET1 containing CUL4 E3 ubiquitin ligases; Second, in the “orange” <span class="highlight" style="background-color:">pathway</span>, phys trigger the degradation of the negatively acting transcriptional regulators PIFs. The activity of PIFs could be enhanced by the COP1/DET1 through HFR1 repression; the protein stability of at least PIF3 is promoted by COP1 through an unknown mechanism. The recently identified HMR acts early in the pathways and is involved in the degradation of PIFs and phyA as well as the inhibition of DET1.</div></p></div><div class="aux"><div class="resc">Meng Chen, et al. Trends Cell Biol. ;21(11):664-671.</div><p class="links"><a class="dblinks" href="/pubmed/21852137" ref="ordinalpos=1568&amp;ncbi_uid=3494593&amp;link_uid=21852137">Citation</a><a class="dblinks" href="/pmc/articles/PMC3205231/" ref="ordinalpos=1568&amp;ncbi_uid=3494593&amp;link_uid=PMC3205231">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1569.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726753/figure/pone-0068985-g001/" ref="ordinalpos=1569&amp;ncbi_uid=5168262&amp;link_uid=PMC3726753" href="/pmc/articles/PMC3726753/"><img src="/pmc/articles/instance/3726753/bin/pone.0068985.g001.gif" src-large="/pmc/articles/instance/3726753/bin/pone.0068985.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726753/" ref="ordinalpos=1569&amp;ncbi_uid=5168262&amp;link_uid=PMC3726753" image-link="/pmc/articles/PMC3726753/figure/pone-0068985-g001/" class="imagepopup">Figure 1. Morphoproteomics and Immunohistochemistry Patients #1, 2..  From: Morphoproteomic Profiling of the Mammalian Target of Rapamycin (mTOR) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Desmoplastic Small Round Cell Tumor (EWS/WT1), Ewing's Sarcoma (EWS/FLI1) and Wilms' Tumor(WT1). </a></p><div class="supp"><p class="details"><div>In patient #1 (DSRCT) the PI3K/Akt/mammalian target of rapamycin (mTOR) <span class="highlight" style="background-color:">pathway</span> is constitutively activated by p-Akt (Ser 473) expression in the nuclear compartment of the tumor cells and p-mTOR phosphorylated on Ser 2448, suggesting mTORC2 (rictor+mTOR) as the dominant form. In patient #2 (Wilm’s tumor), the mTOR <span class="highlight" style="background-color:">pathway</span> is also activated with most tumor cells moderately expressing p-mTOR (Ser 2448) in plasmalemmal and cytoplasmic compartments. This coincides with the constitutive activation of one of the downstream effectors of the mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, namely p-p70S6K (Thr 389). There is constitutive activation of the Ras/Raf kinase/extracellular signal-regulated kinase (ERK) <span class="highlight" style="background-color:">pathway</span> p-ERK 1/2 (Thr202/Tyr204) expression in the Wilms tumor.</div></p></div><div class="aux"><div class="resc">Vivek Subbiah, et al. PLoS One. 2013;8(7):e68985.</div><p class="links"><a class="dblinks" href="/pubmed/23922674" ref="ordinalpos=1569&amp;ncbi_uid=5168262&amp;link_uid=23922674">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726753/" ref="ordinalpos=1569&amp;ncbi_uid=5168262&amp;link_uid=PMC3726753">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1570.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3099575/figure/F5/" ref="ordinalpos=1570&amp;ncbi_uid=3361050&amp;link_uid=PMC3099575" href="/pmc/articles/PMC3099575/"><img src="/pmc/articles/instance/3099575/bin/zh89991169630005.gif" src-large="/pmc/articles/instance/3099575/bin/zh89991169630005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3099575/" ref="ordinalpos=1570&amp;ncbi_uid=3361050&amp;link_uid=PMC3099575" image-link="/pmc/articles/PMC3099575/figure/F5/" class="imagepopup">Figure 5.  From: Tmprss6 is a genetic modifier of the Hfe-hemochromatosis phenotype in mice. </a></p><div class="supp"><p class="details"><div><b>Model of relationship among Hfe, Tmprss6, and the Bmp/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hepcidin regulation</b>. Our data are compatible with a model in which Hfe promotes Bmp/Smad <span class="highlight" style="background-color:">signaling</span> for hepcidin production by inhibiting the activity of Tmprss6 (A). However, we cannot exclude the possibility that Hfe promotes Bmp/Smad <span class="highlight" style="background-color:">signaling</span> by a mechanism that is independent of Tmprss6 but nevertheless cannot achieve a further elevation in Bmp/Smad target gene expression when Tmprss6 is absent. Hfe might, for example, promote Bmp/Smad <span class="highlight" style="background-color:">signaling</span> by Hjv (B) or promote Bmp/Smad <span class="highlight" style="background-color:">signaling</span> at a point functionally downstream of Tmprss6 (C).</div></p></div><div class="aux"><div class="resc">Karin E. Finberg, et al. Blood. 2011 April 28;117(17):4590-4599.</div><p class="links"><a class="dblinks" href="/pubmed/21355094" ref="ordinalpos=1570&amp;ncbi_uid=3361050&amp;link_uid=21355094">Citation</a><a class="dblinks" href="/pmc/articles/PMC3099575/" ref="ordinalpos=1570&amp;ncbi_uid=3361050&amp;link_uid=PMC3099575">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1571.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2925131/figure/F5/" ref="ordinalpos=1571&amp;ncbi_uid=3029390&amp;link_uid=PMC2925131" href="/pmc/articles/PMC2925131/"><img src="/pmc/articles/instance/2925131/bin/nihms197251f5.gif" src-large="/pmc/articles/instance/2925131/bin/nihms197251f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2925131/" ref="ordinalpos=1571&amp;ncbi_uid=3029390&amp;link_uid=PMC2925131" image-link="/pmc/articles/PMC2925131/figure/F5/" class="imagepopup">Fig. 5. <span class="highlight" style="background-color:">Pathway</span> analyses with the Gene Set Enrichment Analysis (GSEA) algorithm.  From: Genome-wide gene expression analysis identifies K-ras as a regulator of alcohol intake. </a></p><div class="supp"><p class="details"><div>To identify pathways regulated by alcohol in a K-ras dependent manner, the expression data set from the comparison of alcohol-treated K-ras<sup>+/−</sup> <i>vs.</i> Nf1<sup>+/−</sup> mice was ranked based on the signal-to-noise-ratio between the two genotypes and interrogated with gene sets representative of various molecular <span class="highlight" style="background-color:">pathway</span>. The top plots in each of the panels represent the enrichment score (<i>ES</i>). Asymmetrical distributions observed for <i>ES</i> (higher toward the left-hand side) indicate differential expression of genes in the <span class="highlight" style="background-color:">pathway</span> under study between K-ras<sup>+/−</sup> and Nf1<sup>+/−</sup> mice and their positive correlations with the Nf1<sup>+/−</sup> genotype, which is characterized by increased K-ras function (Costa et al., 2002). In the middle, are the corresponding heat maps and the “gene tags,” i.e., the location of genes from the gene sets representing the pathways under test. Ranked list metrics of gene expression levels are shown in the bottom plots. Differentially expressed pathways included: <b>A</b>) the phosphodiesterases (PDEs) <span class="highlight" style="background-color:">pathway</span>; <b>B</b>) the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which includes the phosphoinositide-3-kinase PI3K <span class="highlight" style="background-color:">pathway</span>; <b>C</b>) the Jak/Stat <span class="highlight" style="background-color:">pathway</span>; <b>D</b>) the nuclear factor kB (NF-κB) <span class="highlight" style="background-color:">pathway</span>; and <b>E</b>) the adipokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. These pathways were not found to be significantly correlated with one of the two genotypes in the ACC of saline treated K-ras<sup>+/−</sup> <i>vs</i>. Nf1<sup>+/−</sup>.</div></p></div><div class="aux"><div class="resc">Vez Repunte-Canonigo, et al. Brain Res. ;1339:1-10.</div><p class="links"><a class="dblinks" href="/pubmed/20388501" ref="ordinalpos=1571&amp;ncbi_uid=3029390&amp;link_uid=20388501">Citation</a><a class="dblinks" href="/pmc/articles/PMC2925131/" ref="ordinalpos=1571&amp;ncbi_uid=3029390&amp;link_uid=PMC2925131">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1572.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3202082/figure/F1/" ref="ordinalpos=1572&amp;ncbi_uid=3472087&amp;link_uid=PMC3202082" href="/pmc/articles/PMC3202082/"><img src="/pmc/articles/instance/3202082/bin/nihms314578f1.gif" src-large="/pmc/articles/instance/3202082/bin/nihms314578f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3202082/" ref="ordinalpos=1572&amp;ncbi_uid=3472087&amp;link_uid=PMC3202082" image-link="/pmc/articles/PMC3202082/figure/F1/" class="imagepopup">Figure 1.  From: Integration of BMP, Wnt, and Notch <span class="highlight" style="background-color:">signaling</span> pathways in osteoblast differentiation. </a></p><div class="supp"><p class="details"><div><b>BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>:</b> BMP binds heterodimeric receptor to activate Smad proteins, which transactivate osteoblastogenic genes either directly or via Runx2. Adapted from Kawabata et al 1998.</div></p></div><div class="aux"><div class="resc">Grace L. Lin, et al. J Cell Biochem. ;112(12):3491-3501.</div><p class="links"><a class="dblinks" href="/pubmed/21793042" ref="ordinalpos=1572&amp;ncbi_uid=3472087&amp;link_uid=21793042">Citation</a><a class="dblinks" href="/pmc/articles/PMC3202082/" ref="ordinalpos=1572&amp;ncbi_uid=3472087&amp;link_uid=PMC3202082">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1573.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3414724/figure/F4/" ref="ordinalpos=1573&amp;ncbi_uid=4183599&amp;link_uid=PMC3414724" href="/pmc/articles/PMC3414724/"><img src="/pmc/articles/instance/3414724/bin/fimmu-03-00224-g004.gif" src-large="/pmc/articles/instance/3414724/bin/fimmu-03-00224-g004.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3414724/" ref="ordinalpos=1573&amp;ncbi_uid=4183599&amp;link_uid=PMC3414724" image-link="/pmc/articles/PMC3414724/figure/F4/" class="imagepopup">FIGURE 4.  From: The phosphoinositide 3-kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal and malignant B cells: activation mechanisms, regulation and impact on cellular functions. </a></p><div class="supp"><p class="details"><div><b>Alterations in the PI3K <span class="highlight" style="background-color:">pathway</span> in malignant B cells</b>. In malignant B cells, several <span class="highlight" style="background-color:">pathway</span> players have been found to be over-expressed or hyper-phosphorylated. The figure illustrates <span class="highlight" style="background-color:">signaling</span> molecules whose expression, phosphorylation or activity are reported to be altered in malignant B cells (green indicates increased activity, red indicates reduced activity). Altered molecules discussed in the text include p110 subunits, Lyn, Syk, Btk, Akt, TC21, and the BCR <span class="highlight" style="background-color:">signaling</span> chain CD79b. The formation of a non-physiological complex between Lyn and Hsp90 in malignant B cells is indicated. Increased expression of microRNA species that target PTEN and SHIP are reported to reduce expression of these phosphatases. Moreover, the PTEN kinase CK2 is reported to be over-expressed which may diminish PTEN activity. ZAP-70 is expressed in some B cell malignancies, however the impact on the PI3K <span class="highlight" style="background-color:">pathway</span> is currently not clear.</div></p></div><div class="aux"><div class="resc">Samantha D. Pauls, et al. Front Immunol. 2012;3:224.</div><p class="links"><a class="dblinks" href="/pubmed/22908014" ref="ordinalpos=1573&amp;ncbi_uid=4183599&amp;link_uid=22908014">Citation</a><a class="dblinks" href="/pmc/articles/PMC3414724/" ref="ordinalpos=1573&amp;ncbi_uid=4183599&amp;link_uid=PMC3414724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1574.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2948497/figure/pone-0013054-g003/" ref="ordinalpos=1574&amp;ncbi_uid=2519666&amp;link_uid=PMC2948497" href="/pmc/articles/PMC2948497/"><img src="/pmc/articles/instance/2948497/bin/pone.0013054.g003.gif" src-large="/pmc/articles/instance/2948497/bin/pone.0013054.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2948497/" ref="ordinalpos=1574&amp;ncbi_uid=2519666&amp;link_uid=PMC2948497" image-link="/pmc/articles/PMC2948497/figure/pone-0013054-g003/" class="imagepopup">Figure 3. The top 15 canonical <span class="highlight" style="background-color:">signaling</span> pathways influenced by inhibition of GLI1/GLI2 function in HT29 and GC3/c1 cells..  From: cDNA Microarray Gene Expression Profiling of Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Inhibition in Human Colon Cancer Cells. </a></p><div class="supp"><p class="details"><div>The top 15 canonical <span class="highlight" style="background-color:">signaling</span> pathways, determined by IPA, that were significantly up-regulated or down-regulated by GANT61 treatment in HT29 and GC3/c1 cells, are shown. The 1,368 DEGs in HT29 and 1,002 DEGs in GC3/c1 were mapped to the IPA- defined network. The significance p-values that determine the probability that the association between the genes in the dataset and the canonical <span class="highlight" style="background-color:">pathway</span> is by chance alone were calculated by Fisher's exact test, and are expressed as –log (p-value). <b>A</b>. Pathways with enriched down-regulated genes. <b>B</b>. Pathways with enriched up-regulated genes. Blue: Pathways common to both HT29 and GC3/c1. Red: Pathways unique to either HT29 or GC3/c1. Yellow squares: Ratio of the number of DEGs that map to a specific canonical <span class="highlight" style="background-color:">pathway</span> divided by total number of genes that make up that <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ting Shi, et al. PLoS One. 2010;5(10):e13054.</div><p class="links"><a class="dblinks" href="/pubmed/20957031" ref="ordinalpos=1574&amp;ncbi_uid=2519666&amp;link_uid=20957031">Citation</a><a class="dblinks" href="/pmc/articles/PMC2948497/" ref="ordinalpos=1574&amp;ncbi_uid=2519666&amp;link_uid=PMC2948497">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1575.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4083453/figure/F5/" ref="ordinalpos=1575&amp;ncbi_uid=6421473&amp;link_uid=PMC4083453" href="/pmc/articles/PMC4083453/"><img src="/pmc/articles/instance/4083453/bin/fendo-05-00099-g005.gif" src-large="/pmc/articles/instance/4083453/bin/fendo-05-00099-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4083453/" ref="ordinalpos=1575&amp;ncbi_uid=6421473&amp;link_uid=PMC4083453" image-link="/pmc/articles/PMC4083453/figure/F5/" class="imagepopup">Figure 5.  From: Chromium-Insulin Reduces Insulin Clearance and Enhances Insulin <span class="highlight" style="background-color:">Signaling</span> by Suppressing Hepatic Insulin-Degrading Enzyme and Proteasome Protein Expression in KKAy Mice. </a></p><div class="supp"><p class="details"><div><b>The possible mechanisms of CRI enhancing insulin action and insulin <span class="highlight" style="background-color:">signaling</span> were shown in figure</b>. The CRI treatment may directly suppress IDE and ubiquitin-proteasome system (UPS) in the liver, and result in reduction of insulin clearance and degradation of proteins (such as IRS-1 and IRS-2) in the downstream of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The consequent of CRI treatment enhances insulin <span class="highlight" style="background-color:">signaling</span>, increases insulin sensitivity, and improves glucose metabolism in insulin-resistant and diabetic conditions.</div></p></div><div class="aux"><div class="resc">Zhong Q. Wang, et al. Front Endocrinol (Lausanne). 2014;5:99.</div><p class="links"><a class="dblinks" href="/pubmed/25071716" ref="ordinalpos=1575&amp;ncbi_uid=6421473&amp;link_uid=25071716">Citation</a><a class="dblinks" href="/pmc/articles/PMC4083453/" ref="ordinalpos=1575&amp;ncbi_uid=6421473&amp;link_uid=PMC4083453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1576.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2831143/figure/F2/" ref="ordinalpos=1576&amp;ncbi_uid=2737291&amp;link_uid=PMC2831143" href="/pmc/articles/PMC2831143/"><img src="/pmc/articles/instance/2831143/bin/nihms171876f2.gif" src-large="/pmc/articles/instance/2831143/bin/nihms171876f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2831143/" ref="ordinalpos=1576&amp;ncbi_uid=2737291&amp;link_uid=PMC2831143" image-link="/pmc/articles/PMC2831143/figure/F2/" class="imagepopup">Figure 2. Antiviral Innate Immune <span class="highlight" style="background-color:">Signaling</span> in Insects.  From: Innate antiviral immunity in Drosophila. </a></p><div class="supp"><p class="details"><div>Four pathways including three classical immune <span class="highlight" style="background-color:">signaling</span> pathways (Toll, Imd, Jak-STAT) are responsive to infection by different viruses (blue text) based on studies in both <i>Drosophila</i> and mosquitoes (italics). The Toll <span class="highlight" style="background-color:">pathway</span>, responsive to fungi and gram-positive bacteria, has been found to be antiviral in response to infection by Drosophila X and Dengue viruses. The Imd <span class="highlight" style="background-color:">pathway</span>, responsive to gram-negative bacteria, is antiviral in response to infection with Sindbis and Cricket Paralysis viruses. The Jak-STAT <span class="highlight" style="background-color:">pathway</span> restricts infection by Drosophila C and Dengue viruses. Some downstream effectors were induced by infection (red text, italicized for studies in mosquitoes). In addition, <i>Drosophila</i> recognize VSV via the glycoprotein VSV-G (the PAMP), through an unidentified PRR which leads to attenuation of nutrient <span class="highlight" style="background-color:">signaling</span>, likely at the level of PI3K. Repression of this <span class="highlight" style="background-color:">pathway</span> results in the induction of antiviral autophagy which attenuates VSV replication.</div></p></div><div class="aux"><div class="resc">Leah R. Sabin, et al. Curr Opin Immunol. ;22(1):4-9.</div><p class="links"><a class="dblinks" href="/pubmed/20137906" ref="ordinalpos=1576&amp;ncbi_uid=2737291&amp;link_uid=20137906">Citation</a><a class="dblinks" href="/pmc/articles/PMC2831143/" ref="ordinalpos=1576&amp;ncbi_uid=2737291&amp;link_uid=PMC2831143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1577.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3189326/figure/F2/" ref="ordinalpos=1577&amp;ncbi_uid=3476261&amp;link_uid=PMC3189326" href="/pmc/articles/PMC3189326/"><img src="/pmc/articles/instance/3189326/bin/nihms322195f2.gif" src-large="/pmc/articles/instance/3189326/bin/nihms322195f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3189326/" ref="ordinalpos=1577&amp;ncbi_uid=3476261&amp;link_uid=PMC3189326" image-link="/pmc/articles/PMC3189326/figure/F2/" class="imagepopup">Figure 2. Protein glycosylation can regulate <span class="highlight" style="background-color:">signaling</span> specificity.  From: Post-Translational Regulation of <span class="highlight" style="background-color:">Signaling</span> Mucins. </a></p><div class="supp"><p class="details"><div>A) Notch can bind to transmembrane ligands of the Delta and Serrate/Jagged families. Fringe is a glycosyltransferase that modifies Notch and enhances a Delta-specific response (green arrows) while inhibiting Serrate/Jagged-specific responses (red arrows). B) Two yeast MAPK pathways that share components. Nutrient limitation activates the filamentous growth <span class="highlight" style="background-color:">pathway</span> through Msb2 (green arrows). Osmotic stress activates the HOG <span class="highlight" style="background-color:">pathway</span> through Msb2 and Hkr1 (red arrows). Reduced glycosylation of Msb2 specifically triggers the filamentous growth <span class="highlight" style="background-color:">pathway</span> through a core module composed of proteins that are shared between the two pathways, shown in grey. The lighter dashed red arrows designate a function for Msb2 in the HOG <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Paul J. Cullen. Curr Opin Struct Biol. ;21(5):590-596.</div><p class="links"><a class="dblinks" href="/pubmed/21889329" ref="ordinalpos=1577&amp;ncbi_uid=3476261&amp;link_uid=21889329">Citation</a><a class="dblinks" href="/pmc/articles/PMC3189326/" ref="ordinalpos=1577&amp;ncbi_uid=3476261&amp;link_uid=PMC3189326">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1578.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3825415/figure/Fig3/" ref="ordinalpos=1578&amp;ncbi_uid=5511700&amp;link_uid=PMC3825415" href="/pmc/articles/PMC3825415/"><img src="/pmc/articles/instance/3825415/bin/10495_2013_886_Fig3_HTML.gif" src-large="/pmc/articles/instance/3825415/bin/10495_2013_886_Fig3_HTML.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3825415/" ref="ordinalpos=1578&amp;ncbi_uid=5511700&amp;link_uid=PMC3825415" image-link="/pmc/articles/PMC3825415/figure/Fig3/" class="imagepopup">Fig. 3.  From: Functional roles of fibroblast growth factor receptors (FGFRs) <span class="highlight" style="background-color:">signaling</span> in human cancers. </a></p><div class="supp"><p class="details"><div>FGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. FGFs induce FGFR-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by interacting with specific FGFRs and HSPGs. The macromolecular interactions mediate FGFRs dimerization or oligomerization and activate multiple signal transduction pathways, including those involving FRS2, RAS, p38 MAPKs, ERKs, JNKs, Src, PLCγ, Crk, PKC and PI3K. These pathways are negatively regulated in part by the activities of DUSPs, SPRY, SEF and CBL</div></p></div><div class="aux"><div class="resc">Kai Hung Tiong, et al. Apoptosis. 2013;18(12):1447-1468.</div><p class="links"><a class="dblinks" href="/pubmed/23900974" ref="ordinalpos=1578&amp;ncbi_uid=5511700&amp;link_uid=23900974">Citation</a><a class="dblinks" href="/pmc/articles/PMC3825415/" ref="ordinalpos=1578&amp;ncbi_uid=5511700&amp;link_uid=PMC3825415">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1579.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3068308/figure/fig4/" ref="ordinalpos=1579&amp;ncbi_uid=2899238&amp;link_uid=PMC3068308" href="/pmc/articles/PMC3068308/"><img src="/pmc/articles/instance/3068308/bin/PRI2011-620601.004.gif" src-large="/pmc/articles/instance/3068308/bin/PRI2011-620601.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3068308/" ref="ordinalpos=1579&amp;ncbi_uid=2899238&amp;link_uid=PMC3068308" image-link="/pmc/articles/PMC3068308/figure/fig4/" class="imagepopup">Figure 4.  From: Molecular Characteristics of Pancreatic Ductal Adenocarcinoma. </a></p><div class="supp"><p class="details"><div>MAPkinase signalling. Mitogen-activated phosphorylated kinase <span class="highlight" style="background-color:">signaling</span> occurs through a common <span class="highlight" style="background-color:">pathway</span>. A cellular stimulus results in phosphorylation of MAP3K, which in turn phosphorylates MAP2K. MAP2K subsequently phosphorylates MAPK, resulting in altered transcription of the MAPK target genes. The different components of the two MAPK <span class="highlight" style="background-color:">signaling</span> pathways often affected in PDAC, the ERK <span class="highlight" style="background-color:">pathway</span> and the JNK <span class="highlight" style="background-color:">pathway</span>, are depicted in the boxes on each side of the <span class="highlight" style="background-color:">signaling</span> cascade.</div></p></div><div class="aux"><div class="resc">Niki A. Ottenhof, et al. Patholog Res Int. 2011;2011:620601.</div><p class="links"><a class="dblinks" href="/pubmed/21512581" ref="ordinalpos=1579&amp;ncbi_uid=2899238&amp;link_uid=21512581">Citation</a><a class="dblinks" href="/pmc/articles/PMC3068308/" ref="ordinalpos=1579&amp;ncbi_uid=2899238&amp;link_uid=PMC3068308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1580.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193412/figure/fig8/" ref="ordinalpos=1580&amp;ncbi_uid=3770414&amp;link_uid=PMC3193412" href="/pmc/articles/PMC3193412/"><img src="/pmc/articles/instance/3193412/bin/cr2010161f8.gif" src-large="/pmc/articles/instance/3193412/bin/cr2010161f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193412/" ref="ordinalpos=1580&amp;ncbi_uid=3770414&amp;link_uid=PMC3193412" image-link="/pmc/articles/PMC3193412/figure/fig8/" class="imagepopup">Figure 8.  From: A single NF?B system for both canonical and non-canonical <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Wiring diagrams of the NFκB <span class="highlight" style="background-color:">signaling</span> system to chart the interdependence and functional overlap of <i>nfκb1</i> and <i>nfκb2</i> gene products. <b>(A)</b> The NFκB <span class="highlight" style="background-color:">signaling</span> system in <i>nfκb2<sup>−/−</sup></i> cells. Without its normal binding partner p52, RelB will bind p50 to form RelB:p50 dimers, which accumulate in the nucleus. As a result, constitutive RelB:p50 activity compensates for the loss of inducible RelB:p52 activity. <b>(B)</b> The NFκB <span class="highlight" style="background-color:">signaling</span> system in <i>nfκb1<sup>−/−</sup></i> cells. Without its normal binding partner p50, RelA will bind p52 causing an increase in constitutive p100 processing and thus a depletion of the IκBδ and p100 pool, which is required for RelB:p52 activation. Though the canonical <span class="highlight" style="background-color:">pathway</span> is largely preserved in <i>nfκb1</i><sup>−/−</sup> cells, compensation by p52 weakens the non-canonical <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Vincent Feng-Sheng Shih, et al. Cell Res. 2011 January;21(1):86-102.</div><p class="links"><a class="dblinks" href="/pubmed/21102550" ref="ordinalpos=1580&amp;ncbi_uid=3770414&amp;link_uid=21102550">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193412/" ref="ordinalpos=1580&amp;ncbi_uid=3770414&amp;link_uid=PMC3193412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1581.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137639/figure/F2/" ref="ordinalpos=1581&amp;ncbi_uid=3087802&amp;link_uid=PMC3137639" href="/pmc/articles/PMC3137639/"><img src="/pmc/articles/instance/3137639/bin/nihms306822f2.gif" src-large="/pmc/articles/instance/3137639/bin/nihms306822f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137639/" ref="ordinalpos=1581&amp;ncbi_uid=3087802&amp;link_uid=PMC3137639" image-link="/pmc/articles/PMC3137639/figure/F2/" class="imagepopup">Figure 2. Effects of RTKIs and <span class="highlight" style="background-color:">pathway</span> inhibitors on Akt-RSK-S6 kinase <span class="highlight" style="background-color:">signaling</span>.  From: Akt-RSK-S6-kinase <span class="highlight" style="background-color:">Signaling</span> Networks Activated by Oncogenic Receptor Tyrosine Kinases. </a></p><div class="supp"><p class="details"><div>(A) Strategy of probing <span class="highlight" style="background-color:">signaling</span> pathways with different inhibitors. Wort, wortmannin; Rapa, rapamycin. (B) Immunoblotting of three cancer cell lines treated with the indicated inhibitors. (C) Immunoblotting showing the effect of inhibitors on Akt, RSK, S6, and ERK phosphorylation. (D) Confirmation of MS/MS results by immunoblotting wih phosphospecific antibodies. Actin was used as loading control.</div></p></div><div class="aux"><div class="resc">Albrecht Moritz, et al. Sci Signal. ;3(136):ra64-ra64.</div><p class="links"><a class="dblinks" href="/pubmed/20736484" ref="ordinalpos=1581&amp;ncbi_uid=3087802&amp;link_uid=20736484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137639/" ref="ordinalpos=1581&amp;ncbi_uid=3087802&amp;link_uid=PMC3137639">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1582.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2742090/figure/A002014F4/" ref="ordinalpos=1582&amp;ncbi_uid=1994179&amp;link_uid=PMC2742090" href="/pmc/articles/PMC2742090/"><img src="/pmc/articles/instance/2742090/bin/cshperspect-GRA-a002014fig4.gif" src-large="/pmc/articles/instance/2742090/bin/cshperspect-GRA-a002014fig4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2742090/" ref="ordinalpos=1582&amp;ncbi_uid=1994179&amp;link_uid=PMC2742090" image-link="/pmc/articles/PMC2742090/figure/A002014F4/" class="imagepopup">Figure 4. From: Establishing and Interpreting Graded Sonic Hedgehog <span class="highlight" style="background-color:">Signaling</span> during Vertebrate Neural Tube Patterning: The Role of Negative Feedback. </a></p><div class="supp"><p class="details"><div>Negative feedback loops in the SHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> driven by SHH-binding proteins. The membrane proteins Gas1, Cdo, and Boc promote the activation of Smo and are transcriptionally inhibited by SHH <span class="highlight" style="background-color:">signaling</span>. Conversely, Ptc1 and Hhip1 are transcriptionally induced by Gli activation and inhibit the transduction of a SHH signal to Smo.</div></p></div><div class="aux"><div class="resc">Vanessa Ribes, et al. Cold Spring Harb Perspect Biol. 2009 August;1(2):a002014.</div><p class="links"><a class="dblinks" href="/pubmed/20066087" ref="ordinalpos=1582&amp;ncbi_uid=1994179&amp;link_uid=20066087">Citation</a><a class="dblinks" href="/pmc/articles/PMC2742090/" ref="ordinalpos=1582&amp;ncbi_uid=1994179&amp;link_uid=PMC2742090">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1583.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2577715/figure/f4/" ref="ordinalpos=1583&amp;ncbi_uid=2463791&amp;link_uid=PMC2577715" href="/pmc/articles/PMC2577715/"><img src="/pmc/articles/instance/2933166/bin/fig-4.gif" src-large="/pmc/articles/instance/2933166/bin/fig-4.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2577715/" ref="ordinalpos=1583&amp;ncbi_uid=2463791&amp;link_uid=PMC2577715" image-link="/pmc/articles/PMC2577715/figure/f4/" class="imagepopup">FIG. 4. From: Redox Regulation of Multidrug Resistance in Cancer Chemotherapy: Molecular Mechanisms and Therapeutic Opportunities. </a></p><div class="supp"><p class="details"><div><b>NF-κB <span class="highlight" style="background-color:">signaling</span> and <i>MDR</i> gene expression.</b> Activation of <i>mdr1b</i> expression by the carcinogens 2–AAF and DEN is shown in two different pathways but converges into NF-κB <span class="highlight" style="background-color:">signaling</span>. Induction by 2–AAF is through the activation of PI3K <span class="highlight" style="background-color:">signaling</span>, whereas that by DEN occurs through the Toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Membrane TLRs recognize external signals. When activated, the majority of TLRs induce activation of NF-κB and cytokine production in a MyD88–mediated manner. Also shown is the nonconventional transduction <span class="highlight" style="background-color:">pathway</span> induced by CD40 and EBV.</div></p></div><div class="aux"><div class="resc">Macus Tien Kuo. Antioxid Redox Signal. 2009 January;11(1):99-133.</div><p class="links"><a class="dblinks" href="/pubmed/18699730" ref="ordinalpos=1583&amp;ncbi_uid=2463791&amp;link_uid=18699730">Citation</a><a class="dblinks" href="/pmc/articles/PMC2577715/" ref="ordinalpos=1583&amp;ncbi_uid=2463791&amp;link_uid=PMC2577715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1584.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404671/figure/f6/" ref="ordinalpos=1584&amp;ncbi_uid=4137448&amp;link_uid=PMC3404671" href="/pmc/articles/PMC3404671/"><img src="/pmc/articles/instance/3404671/bin/ehp.1104833.g006.gif" src-large="/pmc/articles/instance/3404671/bin/ehp.1104833.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404671/" ref="ordinalpos=1584&amp;ncbi_uid=4137448&amp;link_uid=PMC3404671" image-link="/pmc/articles/PMC3404671/figure/f6/" class="imagepopup">Figure 6.  From: PCB-95 Modulates the Calcium-Dependent <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Responsible for Activity-Dependent Dendritic Growth. </a></p><div class="supp"><p class="details"><div>Schematic representation of RyR-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that mediates PCB-induced and activity-dependent dendritic growth. In the context of activity-dependent dendritic growth, RyR1 and RyR2 in the endoplasmic reticulum of the dendrite functionally link Ca<sup>2+</sup> influx via NMDA and AMPA receptors to Ca<sup>2+</sup> release from intracellular stores. Increased intracellular Ca<sup>2+</sup> as a consequence of RyR activation sequentially activates the CaMKK–CaMK–Ras–MEK/ERK–CREB–Wnt2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to promote dendritic growth. Abbreviations: AMPA, a-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid; βCat, β-catenin; Friz, frizzled, the cognate receptor for Wnt2; Ip<sub>3</sub>, inositol 1,4,5-triphosphate; mGlu, metabotropic glutamate; PMCA, plasma membrane calcium ATPase; R, receptor; VGCC, voltage-gated Ca<sup>2+</sup> channel.</div></p></div><div class="aux"><div class="resc">Gary A. Wayman, et al. Environ Health Perspect. 2012 July;120(7):1003-1009.</div><p class="links"><a class="dblinks" href="/pubmed/22534176" ref="ordinalpos=1584&amp;ncbi_uid=4137448&amp;link_uid=22534176">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404671/" ref="ordinalpos=1584&amp;ncbi_uid=4137448&amp;link_uid=PMC3404671">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1585.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3941707/figure/F8/" ref="ordinalpos=1585&amp;ncbi_uid=5880194&amp;link_uid=PMC3941707" href="/pmc/articles/PMC3941707/"><img src="/pmc/articles/instance/3941707/bin/nihms556521f8.gif" src-large="/pmc/articles/instance/3941707/bin/nihms556521f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3941707/" ref="ordinalpos=1585&amp;ncbi_uid=5880194&amp;link_uid=PMC3941707" image-link="/pmc/articles/PMC3941707/figure/F8/" class="imagepopup">Figure 8.  From: MG53-induced IRS-1 ubiquitination negatively regulates skeletal myogenesis and insulin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>MG53-mediated negative feedback regulation of skeletal myogenesis. MG53 transcription is initiated by an IGFR-IRS-1-Akt-MyoD <span class="highlight" style="background-color:">pathway</span> during skeletal myogenesis. The MG53 protein interacts with, ubiquitinates and degrades IRS-1, leading to the blockage of IGF and insulin <span class="highlight" style="background-color:">signaling</span> in fully-differentiated myotubes.</div></p></div><div class="aux"><div class="resc">Jae-Sung Yi, et al. Nat Commun. 2013;4:2354-2354.</div><p class="links"><a class="dblinks" href="/pubmed/23965929" ref="ordinalpos=1585&amp;ncbi_uid=5880194&amp;link_uid=23965929">Citation</a><a class="dblinks" href="/pmc/articles/PMC3941707/" ref="ordinalpos=1585&amp;ncbi_uid=5880194&amp;link_uid=PMC3941707">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1586.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2818196/figure/F5/" ref="ordinalpos=1586&amp;ncbi_uid=2764430&amp;link_uid=PMC2818196" href="/pmc/articles/PMC2818196/"><img src="/pmc/articles/instance/2818196/bin/JCS059659F5.gif" src-large="/pmc/articles/instance/2818196/bin/JCS059659F5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2818196/" ref="ordinalpos=1586&amp;ncbi_uid=2764430&amp;link_uid=PMC2818196" image-link="/pmc/articles/PMC2818196/figure/F5/" class="imagepopup">Fig. 5. From: RNAase-III enzyme Dicer maintains <span class="highlight" style="background-color:">signaling</span> pathways for differentiation and survival in mouse cortical neural stem cells. </a></p><div class="supp"><p class="details"><div><b>Unbalanced cell-survival and cell-death pathways in <i>Dicer</i>-deficient NSCs.</b> <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> analyses identified an increase of pro-cell-death proteins and a decrease of pro-survival proteins in <i>Dicer</i>-deficient NSCs.</div></p></div><div class="aux"><div class="resc">Yoko Kawase-Koga, et al. J Cell Sci. 2010 February 15;123(4):586-594.</div><p class="links"><a class="dblinks" href="/pubmed/20103535" ref="ordinalpos=1586&amp;ncbi_uid=2764430&amp;link_uid=20103535">Citation</a><a class="dblinks" href="/pmc/articles/PMC2818196/" ref="ordinalpos=1586&amp;ncbi_uid=2764430&amp;link_uid=PMC2818196">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1587.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2886325/figure/f3-ott-2-135/" ref="ordinalpos=1587&amp;ncbi_uid=2317386&amp;link_uid=PMC2886325" href="/pmc/articles/PMC2886325/"><img src="/pmc/articles/instance/2886325/bin/ott-2-135f3.gif" src-large="/pmc/articles/instance/2886325/bin/ott-2-135f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2886325/" ref="ordinalpos=1587&amp;ncbi_uid=2317386&amp;link_uid=PMC2886325" image-link="/pmc/articles/PMC2886325/figure/f3-ott-2-135/" class="imagepopup">Figure 3.  From: Multiple roles and therapeutic implications of Akt <span class="highlight" style="background-color:">signaling</span> in cancer. </a></p><div class="supp"><p class="details"><div>PI3K-Akt-mTOR <span class="highlight" style="background-color:">pathway</span> and cross-talk with other <span class="highlight" style="background-color:">signaling</span> cascades: (Ras/Raf/MAPK and BCR-ABL). PI3K-Akt and Ras/Raf/MAPK pathways are common routes that control key cellular responses. The large amount of cross-talk between these pathways is often responsible for treatment resistance.</div></p></div><div class="aux"><div class="resc">Emiliano Calvo, et al. Onco Targets Ther. 2009;2:135-150.</div><p class="links"><a class="dblinks" href="/pubmed/20616901" ref="ordinalpos=1587&amp;ncbi_uid=2317386&amp;link_uid=20616901">Citation</a><a class="dblinks" href="/pmc/articles/PMC2886325/" ref="ordinalpos=1587&amp;ncbi_uid=2317386&amp;link_uid=PMC2886325">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1588.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012780/figure/fig1/" ref="ordinalpos=1588&amp;ncbi_uid=6511125&amp;link_uid=PMC4012780" href="/pmc/articles/PMC4012780/"><img src="/pmc/articles/instance/4012780/bin/cmi201311f1.gif" src-large="/pmc/articles/instance/4012780/bin/cmi201311f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012780/" ref="ordinalpos=1588&amp;ncbi_uid=6511125&amp;link_uid=PMC4012780" image-link="/pmc/articles/PMC4012780/figure/fig1/" class="imagepopup">Figure 1.  From: New insights into IL-7 <span class="highlight" style="background-color:">signaling</span> pathways during early and late T cell development. </a></p><div class="supp"><p class="details"><div>Roles of IL-7 signals in pre-TCR development and the homeostasis of T cells. (<b>a</b>) The IL-7/Jak3/NFATc1 <span class="highlight" style="background-color:">pathway</span> functions as an alternative independent mechanism behind early T-cell development. Binding between IL-7 and the IL-7R induces the activation of Jak3, which phosphorylates Tyr371 of NFATc1, and facilitates NFATc1 translocation into the nucleus. This translocation produces downstream effectors that are involved in the function and differentiation of T cells. IL-7/Jak3/NFATc1 <span class="highlight" style="background-color:">signaling</span> cooperates with IL-7/Jak3/STAT5 <span class="highlight" style="background-color:">signaling</span> to synthesize Bcl-2, which is essential for early T-cell development. (<b>b</b>) IL-7 <span class="highlight" style="background-color:">signaling</span> regulates the homeostasis of CD8<sup>+</sup> T cells in a duration-dependent manner. Intermittent IL-7 <span class="highlight" style="background-color:">signaling</span> together with TCRs is critical for the quiescence and survival of T cells. Continuous IL-7 <span class="highlight" style="background-color:">signaling</span> leads to the production of IFN-γ, which triggers T-cell apoptosis, termed CICD. CICD, cytokine-induced cell death; IL-7R, IL-7 receptor; TCR, T-cell antigen receptor.</div></p></div><div class="aux"><div class="resc">Na Niu, et al. Cell Mol Immunol. 2013 May;10(3):187-189.</div><p class="links"><a class="dblinks" href="/pubmed/23584490" ref="ordinalpos=1588&amp;ncbi_uid=6511125&amp;link_uid=23584490">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012780/" ref="ordinalpos=1588&amp;ncbi_uid=6511125&amp;link_uid=PMC4012780">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1589.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2939368/figure/F7/" ref="ordinalpos=1589&amp;ncbi_uid=2825843&amp;link_uid=PMC2939368" href="/pmc/articles/PMC2939368/"><img src="/pmc/articles/instance/2939368/bin/2068fig7.gif" src-large="/pmc/articles/instance/2939368/bin/2068fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2939368/" ref="ordinalpos=1589&amp;ncbi_uid=2825843&amp;link_uid=PMC2939368" image-link="/pmc/articles/PMC2939368/figure/F7/" class="imagepopup">Figure 7. From: Ephrin-B1 forward <span class="highlight" style="background-color:">signaling</span> regulates craniofacial morphogenesis by controlling cell proliferation across Eph-ephrin boundaries. </a></p><div class="supp"><p class="details"><div>Ephrin-B1 forward <span class="highlight" style="background-color:">signaling</span> activates ERK/MAPK <span class="highlight" style="background-color:">signaling</span> in palatal cells. (<i>A</i>,<i>B</i>) Phosphorylated ERK1/ERK2 were detected by Western blot and quantified using ImageJ. (<i>A</i>) Time course of induction with 2 μg/mL preclustered ephrin-B1-Fc in media containing 10% FBS resulted in two phases of ERK1/ERK2 activation. (<i>B</i>) In starvation conditions (media containing 0.2% FBS), an activation of ERK1/ERK2 was observed at 2 min, followed by decay. (<i>C</i>,<i>D</i>) Real-time QRT–PCR quantification of immediate early transcriptional targets of ephrin-B1 forward <span class="highlight" style="background-color:">signaling</span> in MEP cells. (<i>C</i>) Regulation of expression of immediate early targets shadows the activation of the ERK/MAPK <span class="highlight" style="background-color:">signaling</span>. QRT–PCR over a time course of forward <span class="highlight" style="background-color:">signaling</span> induction shows initial up-regulation, down-regulation by 1 h, and increase in expression by 2 h (relative mean normalized to GAPDH ± SD) (<i>D</i>) Inhibition of Ras–MAPK activation eliminates the induction of IEG transcriptional targets. Real-time QRT–PCR over a time course of forward <span class="highlight" style="background-color:">signaling</span> induction in the presence of 10 μM U0126 (relative mean normalized to GAPDH ± SD). (<i>E</i>) BrdU incorporation after 3 h of treatment with 2 μg/mL preclustered ephrin-B1-Fc indicates that cell proliferation is induced by ephrin-B1 forward <span class="highlight" style="background-color:">signaling</span> in media containing 10% FBS (<i>n</i> = 9, mean ± SEM; <i>P</i> &lt; 0.008) or 0.2% FBS (<i>n</i> = 9, mean ± SEM; <i>P</i> &lt; 0.006); this effect is eliminated when Ras–MAPK <span class="highlight" style="background-color:">signaling</span> is blocked by treatment with 10 μM U0126 (<i>n</i> = 9, mean ± SEM). (<i>F</i>) Model of ephrin-B1 regulation of palatal morphogenesis. <i>Ephrin-B1<sup>+/−</sup></i> heterozygous females constitute mosaic expression of ephrin-B1 (red). Ephrin-B1 activates forward <span class="highlight" style="background-color:">signaling</span>, resulting in endocytosis and degradation of the EphB receptor. In ephrin-B1 mutant domains, relief of this feedback allows receptor up-regulation (green); several candidate mediators of this feedback were identified as phosphorylation targets of forward <span class="highlight" style="background-color:">signaling</span> (Rin1, Stam2, and Ap2β1). Activation of this forward signal also leads to the phosphorylation of several <span class="highlight" style="background-color:">signaling</span> adaptors (Shc1, Shb, and Dok1) that are capable of regulating ERK/MAPK. The transcriptional regulation of IEGs by activation of the ERK/MAPK may also play a role in feedback regulation of the <span class="highlight" style="background-color:">pathway</span>. Cell proliferation, which is critical for the outgrowth of the palatal shelves, is also dependent on <span class="highlight" style="background-color:">signaling</span> through ERK/MAPK.</div></p></div><div class="aux"><div class="resc">Jeffrey O. Bush, et al. Genes Dev. 2010 September 15;24(18):2068-2080.</div><p class="links"><a class="dblinks" href="/pubmed/20844017" ref="ordinalpos=1589&amp;ncbi_uid=2825843&amp;link_uid=20844017">Citation</a><a class="dblinks" href="/pmc/articles/PMC2939368/" ref="ordinalpos=1589&amp;ncbi_uid=2825843&amp;link_uid=PMC2939368">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1590.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2773873/figure/F4/" ref="ordinalpos=1590&amp;ncbi_uid=2079977&amp;link_uid=PMC2773873" href="/pmc/articles/PMC2773873/"><img src="/pmc/articles/instance/2773873/bin/WJG-14-6786-g004.gif" src-large="/pmc/articles/instance/2773873/bin/WJG-14-6786-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2773873/" ref="ordinalpos=1590&amp;ncbi_uid=2079977&amp;link_uid=PMC2773873" image-link="/pmc/articles/PMC2773873/figure/F4/" class="imagepopup">Figure 4.  From: Dynamic localization of hepatocellular transporters in health and disease. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways involved in the exocytic insertion of canalicular transporters promoted by cAMP and by TC and TUDC. cAMP effect involves elevation in cytosolic Ca<sup>2+</sup> and activation of the PI3K-dependent <span class="highlight" style="background-color:">pathway</span>. Formation of the CaM complex promotes apical insertion of transporters <i>via</i> unidentified mediators, and is counter-regulated by activation of cPKC. PI3K promotes exocytic insertion of canalicular transporters by activation of PKCδ and Erk-1 and Erk-2 of MAPK, <i>via</i> the Ras/Raf- MAPK kinase (MEK)-Erk-1/2 <span class="highlight" style="background-color:">pathway</span>. TC and TUDC also evoke the PI3K-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and promote insertion of canalicular transporters <i>via</i> the Ras/Raf-MEK-Erk-1/2 <span class="highlight" style="background-color:">pathway</span>. TUDC also stimulates canalicular carrier insertion by activation of MAPKs of the p38MAPK type, by an unknown mechanism.</div></p></div><div class="aux"><div class="resc">Marcelo G Roma, et al. World J Gastroenterol. 2008 November 28;14(44):6786-6801.</div><p class="links"><a class="dblinks" href="/pubmed/19058304" ref="ordinalpos=1590&amp;ncbi_uid=2079977&amp;link_uid=19058304">Citation</a><a class="dblinks" href="/pmc/articles/PMC2773873/" ref="ordinalpos=1590&amp;ncbi_uid=2079977&amp;link_uid=PMC2773873">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1591.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3326638/figure/F1/" ref="ordinalpos=1591&amp;ncbi_uid=4369399&amp;link_uid=PMC3326638" href="/pmc/articles/PMC3326638/"><img src="/pmc/articles/instance/3326638/bin/nihms325694f1.gif" src-large="/pmc/articles/instance/3326638/bin/nihms325694f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3326638/" ref="ordinalpos=1591&amp;ncbi_uid=4369399&amp;link_uid=PMC3326638" image-link="/pmc/articles/PMC3326638/figure/F1/" class="imagepopup">FIGURE 1. Mevalonate <span class="highlight" style="background-color:">pathway</span> and key downstream branches in isoprenoid biosynthesis.  From: Indirect Stimulation of Human V?2V?2 T cells Through Alterations in Isoprenoid Metabolism. </a></p><div class="supp"><p class="details"><div>HMG-CoA reductase is the rate-controlling enzyme in the mevalonate <span class="highlight" style="background-color:">pathway</span> and is subject to feedback regulation by downstream products. It is also inhibited by statins. FPP synthase converts IPP and DMAPP to GPP and FPP intermediates and is inhibited by aminobisphosphonates and alkylamines. Loss of FPP and GGPP leads to the loss of membrane anchoring of <span class="highlight" style="background-color:">signaling</span> proteins causing <span class="highlight" style="background-color:">signaling</span> defects and, in some cases, apoptosis.</div></p></div><div class="aux"><div class="resc">Hong Wang, et al. J Immunol. ;187(10):5099-5113.</div><p class="links"><a class="dblinks" href="/pubmed/22013129" ref="ordinalpos=1591&amp;ncbi_uid=4369399&amp;link_uid=22013129">Citation</a><a class="dblinks" href="/pmc/articles/PMC3326638/" ref="ordinalpos=1591&amp;ncbi_uid=4369399&amp;link_uid=PMC3326638">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1592.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041052/figure/F9/" ref="ordinalpos=1592&amp;ncbi_uid=6187486&amp;link_uid=PMC4041052" href="/pmc/articles/PMC4041052/"><img src="/pmc/articles/instance/4041052/bin/1477-5956-12-30-9.gif" src-large="/pmc/articles/instance/4041052/bin/1477-5956-12-30-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041052/" ref="ordinalpos=1592&amp;ncbi_uid=6187486&amp;link_uid=PMC4041052" image-link="/pmc/articles/PMC4041052/figure/F9/" class="imagepopup">Figure 9.  From: Pepsin-pancreatin protein hydrolysates from extruded amaranth inhibit markers of atherosclerosis in LPS-induced THP-1 macrophages-like human cells by reducing expression of proteins in LOX-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Potential mechanism of action of the EAH by reducing expression of proteins associated with LOX-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The red arrows indicate the modulation obtained in this study, for each marker, due to EAH treatment. Numbers 1, 2, 3, 4 and 5 indicate the stages of the atherosclerosis process.</div></p></div><div class="aux"><div class="resc">Alvaro Montoya-Rodríguez, et al. Proteome Sci. 2014;12:30-30.</div><p class="links"><a class="dblinks" href="/pubmed/24891839" ref="ordinalpos=1592&amp;ncbi_uid=6187486&amp;link_uid=24891839">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041052/" ref="ordinalpos=1592&amp;ncbi_uid=6187486&amp;link_uid=PMC4041052">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1593.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3094853/figure/F3/" ref="ordinalpos=1593&amp;ncbi_uid=2954032&amp;link_uid=PMC3094853" href="/pmc/articles/PMC3094853/"><img src="/pmc/articles/instance/3094853/bin/nihms254819f3.gif" src-large="/pmc/articles/instance/3094853/bin/nihms254819f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3094853/" ref="ordinalpos=1593&amp;ncbi_uid=2954032&amp;link_uid=PMC3094853" image-link="/pmc/articles/PMC3094853/figure/F3/" class="imagepopup">Figure 3.  From: Unnatural Polyketide Analogues Selectively Target the HER <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div>Effect of trifluoromethylcinnamic acid-based compounds (corresponding acids, SNAc esters, and their CHS reaction extracts) on HER2-mediated PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The BT-474 cells, which are confluent in 6-well plates (3 mL), were treated for 3 h with different trifluoromethylcinnamic acids (50 µ<span style="font-variant: small-caps" class="small-caps">m</span>), trifluoromethylcinnamoyl-SNAc esters (50 µ<span style="font-variant: small-caps" class="small-caps">m</span>), and their CHS reaction extracts (10 µL). Whole-cell lysates were subjected to SDS-PAGE analysis and probed with anti-phospho-HER2 (Tyr1221/1222), anti-HER2, anti-phospho-Akt (Ser473), anti-Akt antibody, anti-phospho-PI3K (Tyr508), and anti-actin, respectively.</div></p></div><div class="aux"><div class="resc">Seok Joon Kwon, et al. Chembiochem. ;11(4):573-580.</div><p class="links"><a class="dblinks" href="/pubmed/20058253" ref="ordinalpos=1593&amp;ncbi_uid=2954032&amp;link_uid=20058253">Citation</a><a class="dblinks" href="/pmc/articles/PMC3094853/" ref="ordinalpos=1593&amp;ncbi_uid=2954032&amp;link_uid=PMC3094853">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1594.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2905467/figure/F2/" ref="ordinalpos=1594&amp;ncbi_uid=2985165&amp;link_uid=PMC2905467" href="/pmc/articles/PMC2905467/"><img src="/pmc/articles/instance/2905467/bin/nihms191589f2.gif" src-large="/pmc/articles/instance/2905467/bin/nihms191589f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2905467/" ref="ordinalpos=1594&amp;ncbi_uid=2985165&amp;link_uid=PMC2905467" image-link="/pmc/articles/PMC2905467/figure/F2/" class="imagepopup">Figure 2.  From: ?-Adrenoceptor Inverse Agonists in Asthma. </a></p><div class="supp"><p class="details"><div>Alternative <span class="highlight" style="background-color:">signaling</span> by β<sub>2</sub>-ARs. Activation of β<sub>2</sub>-AR leads to both β-arrestin-dependent and G protein-dependent <span class="highlight" style="background-color:">signaling</span>. Acute β<sub>2</sub>-AR activation attenuates the asthma phenotype via the G protein-dependent <span class="highlight" style="background-color:">pathway</span> in airway smooth muscle cells. The asthma inflammation phenotype is significantly diminished when β<sub>2</sub>-AR or β-arrestin-2 is absent or when a β<sub>2</sub>-AR inverse agonist is chronically administered. The dotted lines represent a proposed <span class="highlight" style="background-color:">pathway</span> underlying the asthma phenotype that requires activation of a β<sub>2</sub>-AR-mediated β-arrestin-2-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (β<sub>2</sub>-AR, beta-2-adrenergic receptor; βarr2, β-arrestin-2; KO, knock-out; MAPK, mitogen-activatged protein kinase; AHR, airway hyperresponsiveness; ASM, airway smooth muscle).</div></p></div><div class="aux"><div class="resc">Burton F. Dickey, et al. Curr Opin Pharmacol. ;10(3):254-259.</div><p class="links"><a class="dblinks" href="/pubmed/20399707" ref="ordinalpos=1594&amp;ncbi_uid=2985165&amp;link_uid=20399707">Citation</a><a class="dblinks" href="/pmc/articles/PMC2905467/" ref="ordinalpos=1594&amp;ncbi_uid=2985165&amp;link_uid=PMC2905467">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1595.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3904612/figure/F8/" ref="ordinalpos=1595&amp;ncbi_uid=5782633&amp;link_uid=PMC3904612" href="/pmc/articles/PMC3904612/"><img src="/pmc/articles/instance/3904612/bin/JCI70198.f8.gif" src-large="/pmc/articles/instance/3904612/bin/JCI70198.f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3904612/" ref="ordinalpos=1595&amp;ncbi_uid=5782633&amp;link_uid=PMC3904612" image-link="/pmc/articles/PMC3904612/figure/F8/" class="imagepopup">Figure 8. Model of MNK1-mediated resistance to rapalogs..  From: MNK1 <span class="highlight" style="background-color:">pathway</span> activity maintains protein synthesis in rapalog-treated gliomas. </a></p><div class="supp"><p class="details"><div>Rapalogs abrogate mTORC1-mediated inhibition of 4EBP1 phosphorylation at Ser65 and Thr70. Activated by rapalogs, the MNK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> phosphorylates eIF4E and maintains 4EBP1 phosphorylation at Ser65. Activation of the MNK1/eIF4E <span class="highlight" style="background-color:">pathway</span> increases translation of cancer-promoting and antiapoptotic proteins, whereas 4EBP1 phosphorylation followed by eIF4E dissociation allows continued protein synthesis and, thus, cancer cell survival.</div></p></div><div class="aux"><div class="resc">Michal Grzmil, et al. J Clin Invest. 2014 February 3;124(2):742-754.</div><p class="links"><a class="dblinks" href="/pubmed/24401275" ref="ordinalpos=1595&amp;ncbi_uid=5782633&amp;link_uid=24401275">Citation</a><a class="dblinks" href="/pmc/articles/PMC3904612/" ref="ordinalpos=1595&amp;ncbi_uid=5782633&amp;link_uid=PMC3904612">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1596.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3397444/figure/marinedrugs-10-01345-f006/" ref="ordinalpos=1596&amp;ncbi_uid=4130336&amp;link_uid=PMC3397444" href="/pmc/articles/PMC3397444/"><img src="/pmc/articles/instance/3397444/bin/marinedrugs-10-01345-g006.gif" src-large="/pmc/articles/instance/3397444/bin/marinedrugs-10-01345-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3397444/" ref="ordinalpos=1596&amp;ncbi_uid=4130336&amp;link_uid=PMC3397444" image-link="/pmc/articles/PMC3397444/figure/marinedrugs-10-01345-f006/" class="imagepopup">Figure 6.  From: SD118-Xanthocillin X (1), a Novel Marine Agent Extracted from Penicillium commune, Induces Autophagy through the Inhibition of the MEK/ERK <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>SD118-Xanthocillin X (<b>1</b>) is involved in the autophagy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Effect of SD118-xanthocillin X (<b>1</b>) on the MEK/ERK/mTOR <span class="highlight" style="background-color:">pathway</span>. HepG2 cells were treated with 24.3 µM SD118-xanthocillin X (<b>1</b>), harvested and subjected to western blot analysis, which shows the time-dependent reduction of p-ERK1/2, p-mTOR and Bcl2. Protein contents were normalized with β-actin.</div></p></div><div class="aux"><div class="resc">Ying Zhao, et al. Mar Drugs. 2012 June;10(6):1345-1359.</div><p class="links"><a class="dblinks" href="/pubmed/22822377" ref="ordinalpos=1596&amp;ncbi_uid=4130336&amp;link_uid=22822377">Citation</a><a class="dblinks" href="/pmc/articles/PMC3397444/" ref="ordinalpos=1596&amp;ncbi_uid=4130336&amp;link_uid=PMC3397444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1597.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4263745/figure/pone-0115053-g011/" ref="ordinalpos=1597&amp;ncbi_uid=6982815&amp;link_uid=PMC4263745" href="/pmc/articles/PMC4263745/"><img src="/pmc/articles/instance/4263745/bin/pone.0115053.g011.gif" src-large="/pmc/articles/instance/4263745/bin/pone.0115053.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4263745/" ref="ordinalpos=1597&amp;ncbi_uid=6982815&amp;link_uid=PMC4263745" image-link="/pmc/articles/PMC4263745/figure/pone-0115053-g011/" class="imagepopup">Figure 11. Schematic depiction of the potential mechanism by which CSE-induced Caspase-12 activation dampens the defense response of oral mucosal epithelial cells to pathogenic microorganism through inhibition of the NOD1 <span class="highlight" style="background-color:">signaling</span> and hBDs production..  From: Caspase-12 Silencing Attenuates Inhibitory Effects of Cigarette Smoke Extract on NOD1 <span class="highlight" style="background-color:">Signaling</span> and hBDs Expression in Human Oral Mucosal Epithelial Cells. </a></p><div class="supp"><p class="details"><div>Mechanistically, CSE activated intracellular Caspase-12, which negatively regulated NOD1 <span class="highlight" style="background-color:">signaling</span> by suppressing NOD1 and NF-κB and inducing RIP2. As a part of downstream effectors of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, hBDs production was subsequently inhibited and the defense response to pathogens was dampened. CS: cigarette smoke; CSE: cigarette smoke extract.</div></p></div><div class="aux"><div class="resc">Xiang Wang, et al. PLoS One. 2014;9(12):e115053.</div><p class="links"><a class="dblinks" href="/pubmed/25503380" ref="ordinalpos=1597&amp;ncbi_uid=6982815&amp;link_uid=25503380">Citation</a><a class="dblinks" href="/pmc/articles/PMC4263745/" ref="ordinalpos=1597&amp;ncbi_uid=6982815&amp;link_uid=PMC4263745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1598.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3759162/figure/F7/" ref="ordinalpos=1598&amp;ncbi_uid=5264370&amp;link_uid=PMC3759162" href="/pmc/articles/PMC3759162/"><img src="/pmc/articles/instance/3759162/bin/nihms504925f7.gif" src-large="/pmc/articles/instance/3759162/bin/nihms504925f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759162/" ref="ordinalpos=1598&amp;ncbi_uid=5264370&amp;link_uid=PMC3759162" image-link="/pmc/articles/PMC3759162/figure/F7/" class="imagepopup">Fig. 7.  From: NSBP-1 mediates the effects of cholesterol on insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>Proposed model for the potential role of NSBP-1 in the insulin <span class="highlight" style="background-color:">pathway</span>. DAF-2 signals through the AGE-1 <span class="highlight" style="background-color:">signaling</span> cascade activate AKT-1 by phosphorylation. These kinases in turn regulate DAF-16 and NSBP-1 negatively. In the absence of insulin <span class="highlight" style="background-color:">signaling</span>, NSBP-1 binds to DAF-16 in the nucleus to regulate insulin <span class="highlight" style="background-color:">signaling</span>-mediated dauer formation, fat storage, and oxidative stress</div></p></div><div class="aux"><div class="resc">Mi Cheong Cheong, et al. Cell Mol Life Sci. 2013 May;70(9):1623-1636.</div><p class="links"><a class="dblinks" href="/pubmed/23255046" ref="ordinalpos=1598&amp;ncbi_uid=5264370&amp;link_uid=23255046">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759162/" ref="ordinalpos=1598&amp;ncbi_uid=5264370&amp;link_uid=PMC3759162">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1599.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC355851/figure/f8/" ref="ordinalpos=1599&amp;ncbi_uid=457872&amp;link_uid=PMC355851" href="/pmc/articles/PMC355851/"><img src="/pmc/articles/instance/355851/bin/zmb0060413650008.gif" src-large="/pmc/articles/instance/355851/bin/zmb0060413650008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC355851/" ref="ordinalpos=1599&amp;ncbi_uid=457872&amp;link_uid=PMC355851" image-link="/pmc/articles/PMC355851/figure/f8/" class="imagepopup">FIG. 8. From: Distinct Activities of the ?-Catenin Family, ?-Catulin and ?-Catenin, on ?-Catenin-Mediated <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed model for the observed inhibitory effects of α-catulin and α-catenin on the activation of cyclin D1 transcription. The cyclin D1 promoter is activated by serum via a Ras-mediated <span class="highlight" style="background-color:">pathway</span> which signals to the EtsB binding site and by a β-catenin-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (inducible by LiCl) which signals to the TCF-1 binding site. α-Catenin likely inhibits β-catenin-mediated activation through its direct interaction with β-catenin, as proposed by others (7). We propose that α-catulin may inhibit Ras-induced cyclin D1 promoter activation via attenuation of Rho-mediated <span class="highlight" style="background-color:">signaling</span> downstream of Ras.</div></p></div><div class="aux"><div class="resc">Keith D. Merdek, et al. Mol Cell Biol. 2004 March;24(6):2410-2422.</div><p class="links"><a class="dblinks" href="/pubmed/14993280" ref="ordinalpos=1599&amp;ncbi_uid=457872&amp;link_uid=14993280">Citation</a><a class="dblinks" href="/pmc/articles/PMC355851/" ref="ordinalpos=1599&amp;ncbi_uid=457872&amp;link_uid=PMC355851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1600.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3196148/figure/F9/" ref="ordinalpos=1600&amp;ncbi_uid=3477413&amp;link_uid=PMC3196148" href="/pmc/articles/PMC3196148/"><img src="/pmc/articles/instance/3196148/bin/zbc0471182530009.gif" src-large="/pmc/articles/instance/3196148/bin/zbc0471182530009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3196148/" ref="ordinalpos=1600&amp;ncbi_uid=3477413&amp;link_uid=PMC3196148" image-link="/pmc/articles/PMC3196148/figure/F9/" class="imagepopup">FIGURE 9. From: Auto-activation of c-JUN Gene by Amino Acid Deprivation of Hepatocellular Carcinoma Cells Reveals a Novel c-JUN-mediated <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Working model for the ATF4-dependent and c-JUN-dependent AA <span class="highlight" style="background-color:">signaling</span> pathways.</b> The GCN2-eIF2α-ATF4 <span class="highlight" style="background-color:">pathway</span> that culminates in ATF4 binding to CARE sites has been well documented (7). The novel information in this study illustrates the existence of a ERK-JNK-c-JUN/ATF2 <span class="highlight" style="background-color:">pathway</span> that functions independently of ATF4 by assembling c-JUN-c-JUN or c-JUN-ATF2 dimers at AP-1 sites. Although not indicated for clarity, it is known that ATF2 also participates in activating ATF4-dependent genes by catalyzing histone acetylation (16, 48). The <i>dashed lines</i> indicate that the mechanism is unknown.</div></p></div><div class="aux"><div class="resc">Lingchen Fu, et al. J Biol Chem. 2011 October 21;286(42):36724-36738.</div><p class="links"><a class="dblinks" href="/pubmed/21862593" ref="ordinalpos=1600&amp;ncbi_uid=3477413&amp;link_uid=21862593">Citation</a><a class="dblinks" href="/pmc/articles/PMC3196148/" ref="ordinalpos=1600&amp;ncbi_uid=3477413&amp;link_uid=PMC3196148">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="15" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="16" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="17" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:25:42-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A7DD053FEC9A100000000006490B6;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A7DD053FEC9A100000000006490B6" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>