
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="11" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC67DA53FEC0410000000000282B9D" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="10" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="11" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="12" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="11" /></div>
            <div><h2 class="result_count">Results: 1001 to 1100 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1001.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3723343/figure/F2/" ref="ordinalpos=1001&amp;ncbi_uid=5126285&amp;link_uid=PMC3723343" href="/pmc/articles/PMC3723343/"><img src="/pmc/articles/instance/3723343/bin/nihms357082f2.gif" src-large="/pmc/articles/instance/3723343/bin/nihms357082f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3723343/" ref="ordinalpos=1001&amp;ncbi_uid=5126285&amp;link_uid=PMC3723343" image-link="/pmc/articles/PMC3723343/figure/F2/" class="imagepopup">Figure 2. Model of Cripto modulation of activin and Nodal <span class="highlight" style="background-color:">signaling</span>.  From: Cripto/GRP78 modulation of the TGF-? <span class="highlight" style="background-color:">pathway</span> in development and oncogenesis. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) In the absence of Cripto, Nodal cannot signal and activin can signal at high levels leading to activation of genes responsive to both low and high levels of Smad2/3 <span class="highlight" style="background-color:">signaling</span>. (<b>B</b>) Cripto enables Nodal <span class="highlight" style="background-color:">signaling</span> and attenuates activin <span class="highlight" style="background-color:">signaling</span> and forms receptor complexes with these ligands that are structurally and functionally similar. The weaker <span class="highlight" style="background-color:">signaling</span> of Cripto-containing complexes precludes the regulation of genes that require high, sustained levels of Smad2/3 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Peter C. Gray, et al. FEBS Lett. ;586(14):1836-1845.</div><p class="links"><a class="dblinks" href="/pubmed/22306319" ref="ordinalpos=1001&amp;ncbi_uid=5126285&amp;link_uid=22306319">Citation</a><a class="dblinks" href="/pmc/articles/PMC3723343/" ref="ordinalpos=1001&amp;ncbi_uid=5126285&amp;link_uid=PMC3723343">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1002.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3045689/figure/F1/" ref="ordinalpos=1002&amp;ncbi_uid=2811756&amp;link_uid=PMC3045689" href="/pmc/articles/PMC3045689/"><img src="/pmc/articles/instance/3045689/bin/jcbn10-69f01.gif" src-large="/pmc/articles/instance/3045689/bin/jcbn10-69f01.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3045689/" ref="ordinalpos=1002&amp;ncbi_uid=2811756&amp;link_uid=PMC3045689" image-link="/pmc/articles/PMC3045689/figure/F1/" class="imagepopup">Fig. 1.  From: Rebamipide suppresses TLR-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in regulating IRF3/7 and IFN-?/? reduction. </a></p><div class="supp"><p class="details"><div>Inflammatory mucosa of UC patients over-expressed TBK1-IRF3/7 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human biopsy tissues. Real-time RT-PCR of <i>TBK1</i>, <i>NAP1</i>, <i>IRF3</i>, and <i>IRF7</i> confirmed the relationship between mRNA expression of these genes and ulcerative colitis in 10 patients. More <i>TBK1</i>, <i>NAP1</i>, <i>IRF3</i>, and <i>IRF7</i> mRNA was expressed in atypical, than in the normal mucosa of patients with UC.</div></p></div><div class="aux"><div class="resc">Naotaka Ogasawara, et al. J Clin Biochem Nutr. 2011 March;48(2):154-160.</div><p class="links"><a class="dblinks" href="/pubmed/21373269" ref="ordinalpos=1002&amp;ncbi_uid=2811756&amp;link_uid=21373269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3045689/" ref="ordinalpos=1002&amp;ncbi_uid=2811756&amp;link_uid=PMC3045689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1003.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g004/" ref="ordinalpos=1003&amp;ncbi_uid=5583245&amp;link_uid=PMC3855681" href="/pmc/articles/PMC3855681/"><img src="/pmc/articles/instance/3855681/bin/pone.0080918.g004.gif" src-large="/pmc/articles/instance/3855681/bin/pone.0080918.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855681/" ref="ordinalpos=1003&amp;ncbi_uid=5583245&amp;link_uid=PMC3855681" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g004/" class="imagepopup">Figure 4. TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of H. sapiens..  From: A Model of an Integrated Immune System <span class="highlight" style="background-color:">Pathway</span> in Homo sapiens and Its Interaction with Superantigen Producing Expression Regulatory <span class="highlight" style="background-color:">Pathway</span> in Staphylococcus aureus: Comparing Behavior of Pathogen Perturbed and Unperturbed <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>SEB refers to Staphylococcal Enterotoxin B. Sharp arrow head indicates activation and blocked arrow shows inhibition.</div></p></div><div class="aux"><div class="resc">Namrata Tomar, et al. PLoS One. 2013;8(12):e80918.</div><p class="links"><a class="dblinks" href="/pubmed/24324645" ref="ordinalpos=1003&amp;ncbi_uid=5583245&amp;link_uid=24324645">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855681/" ref="ordinalpos=1003&amp;ncbi_uid=5583245&amp;link_uid=PMC3855681">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1004.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4093200/figure/f5-ajas-27-5-733-15/" ref="ordinalpos=1004&amp;ncbi_uid=6399618&amp;link_uid=PMC4093200" href="/pmc/articles/PMC4093200/"><img src="/pmc/articles/instance/4093200/bin/ajas-27-5-733-15f5.gif" src-large="/pmc/articles/instance/4093200/bin/ajas-27-5-733-15f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4093200/" ref="ordinalpos=1004&amp;ncbi_uid=6399618&amp;link_uid=PMC4093200" image-link="/pmc/articles/PMC4093200/figure/f5-ajas-27-5-733-15/" class="imagepopup">Figure 5.  From: Effect of Glucagon-like Peptide 2 on Tight Junction in Jejunal Epithelium of Weaned Pigs though MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effect of GLP-2 (100 nmol/L) on distribution of tight junction proteins with MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocked by U0126 (10 μmol/L). (400-fold increase) (OD). GLP-2, glucagon-like peptide 2; MARK, mitogen-activated protein kinase; ZO-1, zonula occludens-1. <sup>a,b,c</sup> Treatments with different letters are different at p&lt;0.05.</div></p></div><div class="aux"><div class="resc">Changsong Yu, et al. Asian-Australas J Anim Sci. 2014 May;27(5):733-742.</div><p class="links"><a class="dblinks" href="/pubmed/25050009" ref="ordinalpos=1004&amp;ncbi_uid=6399618&amp;link_uid=25050009">Citation</a><a class="dblinks" href="/pmc/articles/PMC4093200/" ref="ordinalpos=1004&amp;ncbi_uid=6399618&amp;link_uid=PMC4093200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1005.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3817392/figure/f0005/" ref="ordinalpos=1005&amp;ncbi_uid=5461423&amp;link_uid=PMC3817392" href="/pmc/articles/PMC3817392/"><img src="/pmc/articles/instance/3817392/bin/gr1.gif" src-large="/pmc/articles/instance/3817392/bin/gr1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3817392/" ref="ordinalpos=1005&amp;ncbi_uid=5461423&amp;link_uid=PMC3817392" image-link="/pmc/articles/PMC3817392/figure/f0005/" class="imagepopup">Figure 1.  From: PI3K <span class="highlight" style="background-color:">signaling</span>: A key <span class="highlight" style="background-color:">pathway</span> in the control of sympathetic traffic and arterial pressure by leptin. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> pathways regulated by the leptin receptor (LepRb) in the central nervous system. Four primary intracellular <span class="highlight" style="background-color:">signaling</span> pathways emanates from LepRb: signal transducer and activator of transcription (STAT) 3 and STAT5 proteins, the extracellular signal-related kinase (ERK<b>)</b> and phosphoinositol-3 kinase (PI3K). Each <span class="highlight" style="background-color:">pathway</span> is activated by phosphorylation of specific tyrosine residues in ObRb. The four <span class="highlight" style="background-color:">signaling</span> pathways are implicated in leptin regulation of energy homeostasis. In contrast, leptin control of renal sympathetic nerve activity (SNA) and arterial pressure occur through PI3K <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Shannon M. Harlan, et al. Mol Metab. 2013 April;2(2):69-73.</div><p class="links"><a class="dblinks" href="/pubmed/24199153" ref="ordinalpos=1005&amp;ncbi_uid=5461423&amp;link_uid=24199153">Citation</a><a class="dblinks" href="/pmc/articles/PMC3817392/" ref="ordinalpos=1005&amp;ncbi_uid=5461423&amp;link_uid=PMC3817392">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1006.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3712061/figure/F7/" ref="ordinalpos=1006&amp;ncbi_uid=6342863&amp;link_uid=PMC3712061" href="/pmc/articles/PMC3712061/"><img src="/pmc/articles/instance/3712061/bin/2318fig7.gif" src-large="/pmc/articles/instance/3712061/bin/2318fig7.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3712061/" ref="ordinalpos=1006&amp;ncbi_uid=6342863&amp;link_uid=PMC3712061" image-link="/pmc/articles/PMC3712061/figure/F7/" class="imagepopup">FIG. 7. From: Hyperglycemia-Induced Protein Kinase C ?2 Activation Induces Diastolic Cardiac Dysfunction in Diabetic Rats by Impairing Caveolin-3 Expression and Akt/eNOS <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic depicting hyperglycemia-induced PKCβ2 activation effects upon Cav-3–modulated Akt/eNOS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cardiomyocyte caveolae are required for hyperglycemia-induced PKCβ<sub>2</sub> activation (translocation from cytosol to caveolae membrane). Excessive PKCβ<sub>2</sub> activation decreased Cav-3 expression, impairing the Akt/eNOS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Solid arrows depict stimulation, while transverse “T” shape indicates inhibition. PI3K, phosphatidylinositol 3-kinase.</div></p></div><div class="aux"><div class="resc">Shaoqing Lei, et al. Diabetes. 2013 July;62(7):2318-2328.</div><p class="links"><a class="dblinks" href="/pubmed/23474486" ref="ordinalpos=1006&amp;ncbi_uid=6342863&amp;link_uid=23474486">Citation</a><a class="dblinks" href="/pmc/articles/PMC3712061/" ref="ordinalpos=1006&amp;ncbi_uid=6342863&amp;link_uid=PMC3712061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1007.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3094802/figure/F4/" ref="ordinalpos=1007&amp;ncbi_uid=2954008&amp;link_uid=PMC3094802" href="/pmc/articles/PMC3094802/"><img src="/pmc/articles/instance/3094802/bin/nihms290166f4.gif" src-large="/pmc/articles/instance/3094802/bin/nihms290166f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3094802/" ref="ordinalpos=1007&amp;ncbi_uid=2954008&amp;link_uid=PMC3094802" image-link="/pmc/articles/PMC3094802/figure/F4/" class="imagepopup">Fig. 4.  From: Notch <span class="highlight" style="background-color:">Signaling</span> Proteins: Legitimate Targets for Cancer Therapy. </a></p><div class="supp"><p class="details"><div>The role of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the development and progression of cancer, and during the acquisition of EMT phenotype, and the formation of cancer stem cells. Natural agents and γ-secretase inhibitors could be useful for targeting Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins, which could enhance the efficacy of conventional cancer therapies.</div></p></div><div class="aux"><div class="resc">Zhiwei Wang, et al. Curr Protein Pept Sci. ;11(6):398-408.</div><p class="links"><a class="dblinks" href="/pubmed/20491628" ref="ordinalpos=1007&amp;ncbi_uid=2954008&amp;link_uid=20491628">Citation</a><a class="dblinks" href="/pmc/articles/PMC3094802/" ref="ordinalpos=1007&amp;ncbi_uid=2954008&amp;link_uid=PMC3094802">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1008.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2442829/figure/F2/" ref="ordinalpos=1008&amp;ncbi_uid=1408286&amp;link_uid=PMC2442829" href="/pmc/articles/PMC2442829/"><img src="/pmc/articles/instance/2442829/bin/nihms51320f2.gif" src-large="/pmc/articles/instance/2442829/bin/nihms51320f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2442829/" ref="ordinalpos=1008&amp;ncbi_uid=1408286&amp;link_uid=PMC2442829" image-link="/pmc/articles/PMC2442829/figure/F2/" class="imagepopup">Figure 2. Combinatorial approaches with inhibitors of the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span>.  From: Targeting the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span>: effective combinations and clinical considerations. </a></p><div class="supp"><p class="details"><div>Several approaches can be employed when combining <span class="highlight" style="background-color:">pathway</span> inhibitors with other targeted therapies. Inhibition of proximal <span class="highlight" style="background-color:">pathway</span> components, such as receptor tyrosine kinases (RTKs) and oncogenes, combined with distal inhibition of Akt or mTOR may be an effective approach to circumvent feedback activation that could occur with distal inhibition alone (left panel). Alternatively, dual inhibition of parallel <span class="highlight" style="background-color:">signaling</span> pathways prevents compensatory activation of redundant pro-survival pathways (middle panel). Finally, <span class="highlight" style="background-color:">pathway</span> inhibition can be combined with several other types of targeted therapies, including inhibition of histone deacetylase complexes (HDAC), the proteasome and cyclooxygenase-2 (COX-2) (right panel).</div></p></div><div class="aux"><div class="resc">Jaclyn LoPiccolo, et al. Drug Resist Updat. ;11(1-2):32-50.</div><p class="links"><a class="dblinks" href="/pubmed/18166498" ref="ordinalpos=1008&amp;ncbi_uid=1408286&amp;link_uid=18166498">Citation</a><a class="dblinks" href="/pmc/articles/PMC2442829/" ref="ordinalpos=1008&amp;ncbi_uid=1408286&amp;link_uid=PMC2442829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1009.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4035334/figure/pone-0098113-g002/" ref="ordinalpos=1009&amp;ncbi_uid=6186995&amp;link_uid=PMC4035334" href="/pmc/articles/PMC4035334/"><img src="/pmc/articles/instance/4035334/bin/pone.0098113.g002.gif" src-large="/pmc/articles/instance/4035334/bin/pone.0098113.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4035334/" ref="ordinalpos=1009&amp;ncbi_uid=6186995&amp;link_uid=PMC4035334" image-link="/pmc/articles/PMC4035334/figure/pone-0098113-g002/" class="imagepopup">Figure 2. Effect of CRP on the Akt/mTOR/p70S6K <span class="highlight" style="background-color:">pathway</span> in H9c2 cardiac myocytes..  From: C-Reactive Protein Inhibits Survivin Expression via Akt/mTOR <span class="highlight" style="background-color:">Pathway</span> Downregulation by PTEN Expression in Cardiac Myocytes. </a></p><div class="supp"><p class="details"><div>(A) After 24 hours of serum starvation, H9c2 cells were treated with 10% FBS for indicated time. Cells were harvested and analyzed for Akt/mTOR/p70S6K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by immunoblot assay. (B) H9c2 cells were pretreated for 24 hours with 50 µg/ml of CRP in 0.5% FBS and then treated 10% FBS for 1 hour. The protein levels were analyzed by immunoblot assay.</div></p></div><div class="aux"><div class="resc">Beom Seob Lee, et al. PLoS One. 2014;9(5):e98113.</div><p class="links"><a class="dblinks" href="/pubmed/24866016" ref="ordinalpos=1009&amp;ncbi_uid=6186995&amp;link_uid=24866016">Citation</a><a class="dblinks" href="/pmc/articles/PMC4035334/" ref="ordinalpos=1009&amp;ncbi_uid=6186995&amp;link_uid=PMC4035334">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1010.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4001531/figure/F4/" ref="ordinalpos=1010&amp;ncbi_uid=6099491&amp;link_uid=PMC4001531" href="/pmc/articles/PMC4001531/"><img src="/pmc/articles/instance/4001531/bin/JCI69611.f4.gif" src-large="/pmc/articles/instance/4001531/bin/JCI69611.f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4001531/" ref="ordinalpos=1010&amp;ncbi_uid=6099491&amp;link_uid=PMC4001531" image-link="/pmc/articles/PMC4001531/figure/F4/" class="imagepopup">Figure 4. Signal pathways affected by the L-mel signature..  From: Melanoma NOS1 expression promotes dysfunctional IFN <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> according to IPA, which highlighted JAK1, JAK2, IFNA1, and IFNA2 downregulation in L-mels (green). (<b>B</b>) G-coupled receptor <span class="highlight" style="background-color:">pathway</span> according to IPA with receptor families. Gi-coupled receptors were upregulated in L-mels (red). </div></p></div><div class="aux"><div class="resc">Qiuzhen Liu, et al. J Clin Invest. 2014 May 1;124(5):2147-2159.</div><p class="links"><a class="dblinks" href="/pubmed/24691438" ref="ordinalpos=1010&amp;ncbi_uid=6099491&amp;link_uid=24691438">Citation</a><a class="dblinks" href="/pmc/articles/PMC4001531/" ref="ordinalpos=1010&amp;ncbi_uid=6099491&amp;link_uid=PMC4001531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1011.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012154/figure/Fig1/" ref="ordinalpos=1011&amp;ncbi_uid=6109524&amp;link_uid=PMC4012154" href="/pmc/articles/PMC4012154/"><img src="/pmc/articles/instance/4012154/bin/12035_2013_8620_Fig1_HTML.gif" src-large="/pmc/articles/instance/4012154/bin/12035_2013_8620_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012154/" ref="ordinalpos=1011&amp;ncbi_uid=6109524&amp;link_uid=PMC4012154" image-link="/pmc/articles/PMC4012154/figure/Fig1/" class="imagepopup">Fig. 1.  From: Phagocytosis of Microglia in the Central Nervous System Diseases. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways involved in microglial phagocytosis. <i>A</i> Extracellular nucleotides, such as UDP and UTP, trigger microglial phagocytosis through P2Y6R/PLC/InsP3 <span class="highlight" style="background-color:">pathway</span>. <i>B</i> Apoptotic debris induces phagocytosis via TREM-2/DAP12/ERK/PKC <span class="highlight" style="background-color:">pathway</span>. <i>C</i> Endogenous or ectogenic detriments, such as LPS, viral nucleotides, α-synuclein, and f-Aβ, provoke phagocytosis by microglia via TLRs through activation of MyD88-dependent IRAK4/p38/scavenger receptors <span class="highlight" style="background-color:">pathway</span> or MyD88-independent actin-Cdc42/Rac <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Ruying Fu, et al. Mol Neurobiol. 2014;49(3):1422-1434.</div><p class="links"><a class="dblinks" href="/pubmed/24395130" ref="ordinalpos=1011&amp;ncbi_uid=6109524&amp;link_uid=24395130">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012154/" ref="ordinalpos=1011&amp;ncbi_uid=6109524&amp;link_uid=PMC4012154">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1012.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3130517/figure/fig7/" ref="ordinalpos=1012&amp;ncbi_uid=3079315&amp;link_uid=PMC3130517" href="/pmc/articles/PMC3130517/"><img src="/pmc/articles/instance/3130517/bin/ECAM2011-939205.007.gif" src-large="/pmc/articles/instance/3130517/bin/ECAM2011-939205.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3130517/" ref="ordinalpos=1012&amp;ncbi_uid=3079315&amp;link_uid=PMC3130517" image-link="/pmc/articles/PMC3130517/figure/fig7/" class="imagepopup">Figure 7.  From: Ethanol Extract from Ampelopsis sinica Root Exerts Anti-Hepatitis B Virus Activity via Inhibition of p53 <span class="highlight" style="background-color:">Pathway</span> In Vitro. </a></p><div class="supp"><p class="details"><div>EASR exerts anti-HBV effects via inhibition of HBV transcription and the p53-associated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ran Pang, et al. Evid Based Complement Alternat Med. 2011;2011:939205.</div><p class="links"><a class="dblinks" href="/pubmed/21738555" ref="ordinalpos=1012&amp;ncbi_uid=3079315&amp;link_uid=21738555">Citation</a><a class="dblinks" href="/pmc/articles/PMC3130517/" ref="ordinalpos=1012&amp;ncbi_uid=3079315&amp;link_uid=PMC3130517">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1013.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4366796/figure/F3/" ref="ordinalpos=1013&amp;ncbi_uid=7413578&amp;link_uid=PMC4366796" href="/pmc/articles/PMC4366796/"><img src="/pmc/articles/instance/4366796/bin/mol.114.095224f3.gif" src-large="/pmc/articles/instance/4366796/bin/mol.114.095224f3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4366796/" ref="ordinalpos=1013&amp;ncbi_uid=7413578&amp;link_uid=PMC4366796" image-link="/pmc/articles/PMC4366796/figure/F3/" class="imagepopup">Fig. 3. From: Delineation of a Conserved Arrestin-Biased <span class="highlight" style="background-color:">Signaling</span> Repertoire In Vivo. </a></p><div class="supp"><p class="details"><div>hPTH(1-34) and bPTH(7-34) stimulate distinct <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> profiles in multiple tissues. (A) Significantly populated canonical <span class="highlight" style="background-color:">signaling</span> pathways induced by bPTH(7-34) (red bars) or by hPTH(1-34) (black bars) in murine aorta. Each histogram bar represents the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> score (number of ligand upregulated <span class="highlight" style="background-color:">pathway</span>-populating transcripts minus the number of ligand downregulated <span class="highlight" style="background-color:">pathway</span>-populating transcripts. <span class="highlight" style="background-color:">Signaling</span> pathways within the yellow block represent the coherently regulated common pathways between the two ligands in the respective tissue. The associated Venn diagram indicates the functional <span class="highlight" style="background-color:">signaling</span> separation between bPTH(7-34) and hPTH(1-34) in the specific tissue experiment. (B) Bone, (C) heart, (D) kidney, (E) liver, and (F) lung all represent analogous data to (A).</div></p></div><div class="aux"><div class="resc">Stuart Maudsley, et al. Mol Pharmacol. 2015 April;87(4):706-717.</div><p class="links"><a class="dblinks" href="/pubmed/25637603" ref="ordinalpos=1013&amp;ncbi_uid=7413578&amp;link_uid=25637603">Citation</a><a class="dblinks" href="/pmc/articles/PMC4366796/" ref="ordinalpos=1013&amp;ncbi_uid=7413578&amp;link_uid=PMC4366796">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1014.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633528/figure/F1/" ref="ordinalpos=1014&amp;ncbi_uid=5916581&amp;link_uid=PMC3633528" href="/pmc/articles/PMC3633528/"><img src="/pmc/articles/instance/3633528/bin/nihms423991f1.gif" src-large="/pmc/articles/instance/3633528/bin/nihms423991f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633528/" ref="ordinalpos=1014&amp;ncbi_uid=5916581&amp;link_uid=PMC3633528" image-link="/pmc/articles/PMC3633528/figure/F1/" class="imagepopup">Fig. 1.  From: A cytokine axis regulates elastin formation and degradation. </a></p><div class="supp"><p class="details"><div>Cytokine <span class="highlight" style="background-color:">signaling</span> cascades that influence tropoelastin transcription. Cytokines and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> intermediates highlighted in <i>red</i> act to suppress tropoelastin transcription. Cytokines and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> intermediates highlighted in <i>green</i> act to promote tropoelastin transcription. RTK, receptor tyrosine kinases.</div></p></div><div class="aux"><div class="resc">Erin P. Sproul, et al. Matrix Biol. 2013 March 11;32(2):86-94.</div><p class="links"><a class="dblinks" href="/pubmed/23160093" ref="ordinalpos=1014&amp;ncbi_uid=5916581&amp;link_uid=23160093">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633528/" ref="ordinalpos=1014&amp;ncbi_uid=5916581&amp;link_uid=PMC3633528">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1015.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4161967/figure/f7/" ref="ordinalpos=1015&amp;ncbi_uid=6641167&amp;link_uid=PMC4161967" href="/pmc/articles/PMC4161967/"><img src="/pmc/articles/instance/4161967/bin/srep06344-f7.gif" src-large="/pmc/articles/instance/4161967/bin/srep06344-f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4161967/" ref="ordinalpos=1015&amp;ncbi_uid=6641167&amp;link_uid=PMC4161967" image-link="/pmc/articles/PMC4161967/figure/f7/" class="imagepopup">Figure 7. A model for the effects of PCA in Arg/N-end rule <span class="highlight" style="background-color:">pathway</span>-dependent GPCR <span class="highlight" style="background-color:">signaling</span> in the brain and its implication in neurodegeneration..  From: A Neurostimulant para-Chloroamphetamine Inhibits the Arginylation Branch of the N-end Rule <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>In normal conditions, the protein levels of RGS4 are maintained in low concentrations in the neuronal cells by the Arg/N-end rule <span class="highlight" style="background-color:">pathway</span>, which facilitate various signal transductions mediated by GPCR. Moreover, the Arg/N-end rule <span class="highlight" style="background-color:">pathway</span> appears to have a neuroprotective role in neurons for the clearance of various proteotoxic C-terminal protein fragments implicated in neurodegenerative diseases. When the Arg/N-end rule <span class="highlight" style="background-color:">pathway</span> is genetically or chemically inhibited, RGS4 levels are elevated to hydrolyze GTP of G protein's α subunit, which results in inactivation of GPCR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Yanxialei Jiang, et al. Sci Rep. 2014;4:6344.</div><p class="links"><a class="dblinks" href="/pubmed/25212999" ref="ordinalpos=1015&amp;ncbi_uid=6641167&amp;link_uid=25212999">Citation</a><a class="dblinks" href="/pmc/articles/PMC4161967/" ref="ordinalpos=1015&amp;ncbi_uid=6641167&amp;link_uid=PMC4161967">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1016.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2860309/figure/F1/" ref="ordinalpos=1016&amp;ncbi_uid=2294871&amp;link_uid=PMC2860309" href="/pmc/articles/PMC2860309/"><img src="/pmc/articles/instance/2860309/bin/nihms191392f1.gif" src-large="/pmc/articles/instance/2860309/bin/nihms191392f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2860309/" ref="ordinalpos=1016&amp;ncbi_uid=2294871&amp;link_uid=PMC2860309" image-link="/pmc/articles/PMC2860309/figure/F1/" class="imagepopup">Figure 1.  From: EDA <span class="highlight" style="background-color:">Signaling</span> and Skin Appendage Development. </a></p><div class="supp"><p class="details"><div>Schematic representation of <i>EDA</i> <span class="highlight" style="background-color:">signaling</span>. <i>EDA</i> is expressed in interfollicular cells; its receptor <i>EDAR</i> in follicular cells. Wnt/Lef1 and other putative signals (GATA, Nkx2 and ???) regulate transcription. The protein product ectodysplasin, is cleaved by furine-like proteases and bind to its receptor EDAR to initiate <i>EDA</i> <span class="highlight" style="background-color:">signaling</span>. EDAR recruits the cytoplasmic <span class="highlight" style="background-color:">signaling</span> adaptor protein EDRADD, and via TRAF6/TAK1/TAB2, the <span class="highlight" style="background-color:">signaling</span> activates the NEMO/IKK complex. That complex phosphorylates IkBs, leading to the entry of NFκBs into the nucleus and the activation of cognate gene transcription. Reported targets of <i>EDA</i> <span class="highlight" style="background-color:">signaling</span> include Wnt/Dkks, Shh, BMP/Sostdc1, LTβ and madcam1. <i>EDA</i> <span class="highlight" style="background-color:">signaling</span> may feedback regulate the Wnt <span class="highlight" style="background-color:">pathway</span> through Dkk4 and Dkk1. Asterisks indicate mutants that cause EDA-like phenotypes in patients and/or mouse models. Mutations in <i>TRAF6, NEMO, IkB</i>α or <i>NF</i>κ<i>Bs</i> also produce EDA-like phenotypes, but accompanied by immunodeficiency.</div></p></div><div class="aux"><div class="resc">Chang-Yi Cui, et al. Cell Cycle. ;5(21):2477-2483.</div><p class="links"><a class="dblinks" href="/pubmed/17102627" ref="ordinalpos=1016&amp;ncbi_uid=2294871&amp;link_uid=17102627">Citation</a><a class="dblinks" href="/pmc/articles/PMC2860309/" ref="ordinalpos=1016&amp;ncbi_uid=2294871&amp;link_uid=PMC2860309">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1017.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3945295/figure/pgen-1004183-g003/" ref="ordinalpos=1017&amp;ncbi_uid=5908464&amp;link_uid=PMC3945295" href="/pmc/articles/PMC3945295/"><img src="/pmc/articles/instance/3945295/bin/pgen.1004183.g003.gif" src-large="/pmc/articles/instance/3945295/bin/pgen.1004183.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3945295/" ref="ordinalpos=1017&amp;ncbi_uid=5908464&amp;link_uid=PMC3945295" image-link="/pmc/articles/PMC3945295/figure/pgen-1004183-g003/" class="imagepopup">Figure 3. Network connectivity of Sks1p kinase <span class="highlight" style="background-color:">signaling</span>..  From: The Yeast Sks1p Kinase <span class="highlight" style="background-color:">Signaling</span> Network Regulates Pseudohyphal Growth and Glucose Response. </a></p><div class="supp"><p class="details"><div>A) Network connectivity map constructed by expanding connections involving Sks1p-dependent phosphoproteins through genetic and physical interactions from KEGG, BioGrid, and GeneMania. Clusters of proteins enriched for the indicated KEGG <span class="highlight" style="background-color:">signaling</span> pathways are shown. Numbered proteins identify Sks1p and a subset of proteins exhibiting Sks1p-dependent phosphorylation by mass spectrometry (shaded red) within the context of the larger connected network. As indicated, the KEGG glycolysis/gluconeogenesis <span class="highlight" style="background-color:">pathway</span> was enriched in the subset of Sks1p-dependent phosphoproteins identified by mass spectrometry. Proteins within the network clustered around B) the KEGG glycolysis/gluconeogenesis <span class="highlight" style="background-color:">pathway</span> and C) the KEGG cell cycle and meiosis pathways are highlighted.</div></p></div><div class="aux"><div class="resc">Cole Johnson, et al. PLoS Genet. 2014 March;10(3):e1004183.</div><p class="links"><a class="dblinks" href="/pubmed/24603354" ref="ordinalpos=1017&amp;ncbi_uid=5908464&amp;link_uid=24603354">Citation</a><a class="dblinks" href="/pmc/articles/PMC3945295/" ref="ordinalpos=1017&amp;ncbi_uid=5908464&amp;link_uid=PMC3945295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1018.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3854659/figure/F1/" ref="ordinalpos=1018&amp;ncbi_uid=5680965&amp;link_uid=PMC3854659" href="/pmc/articles/PMC3854659/"><img src="/pmc/articles/instance/3854659/bin/CCGTM-7-9_F1.gif" src-large="/pmc/articles/instance/3854659/bin/CCGTM-7-9_F1.jpg" alt="Fig. (1)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3854659/" ref="ordinalpos=1018&amp;ncbi_uid=5680965&amp;link_uid=PMC3854659" image-link="/pmc/articles/PMC3854659/figure/F1/" class="imagepopup">Fig. (1).  From: Minireview: Targeting GPCR Activated ERK Pathways for Drug Discovery. </a></p><div class="supp"><p class="details"><div>Both G-protein and β-arrestin mediated <span class="highlight" style="background-color:">signaling</span> pathways can lead to ERK activation. However, the subcellular distributions of activated ERK1/2 are different. The phosphorylated ERK1/2 mediated by the β-arrestin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> largely remain in cytoplasm, while the phosphorylated ERK1/2 mediated by the G-protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> translocate to the nucleus.</div></p></div><div class="aux"><div class="resc">Haifeng Eishingdrelo, et al. Curr Chem Genomics Transl Med. 2013;7:9-15.</div><p class="links"><a class="dblinks" href="/pubmed/24396730" ref="ordinalpos=1018&amp;ncbi_uid=5680965&amp;link_uid=24396730">Citation</a><a class="dblinks" href="/pmc/articles/PMC3854659/" ref="ordinalpos=1018&amp;ncbi_uid=5680965&amp;link_uid=PMC3854659">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1019.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3589936/figure/RRS098F2/" ref="ordinalpos=1019&amp;ncbi_uid=4682619&amp;link_uid=PMC3589936" href="/pmc/articles/PMC3589936/"><img src="/pmc/articles/instance/3589936/bin/rrs09802.gif" src-large="/pmc/articles/instance/3589936/bin/rrs09802.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3589936/" ref="ordinalpos=1019&amp;ncbi_uid=4682619&amp;link_uid=PMC3589936" image-link="/pmc/articles/PMC3589936/figure/RRS098F2/" class="imagepopup">Fig. 2. From: Anti-radiation damage effect of polyethylenimine as a toll-like receptor 5 targeted agonist. </a></p><div class="supp"><p class="details"><div>PEI is a novel TLR5 agonist that activates the NF-κB intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by TLR5. (<b>A</b>) The cytotoxicity of PEI was measured by MTT assay. There was no significant cytotoxicity when PEI concentration was below 0.5 μmol/l. (<b>B</b>) PEI is a novel TLR5 agonist that activates the TLR5-mediated NF-κB intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Dual-luciferase reporter gene expression, showing PEI (0.2–1.0 μmol/l) can activate the TLR5-mediated NF-κB intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. *<i>P</i>&lt; 0.05, vs. control.</div></p></div><div class="aux"><div class="resc">Zhiqiang Hu, et al. J Radiat Res. 2013 March;54(2):243-250.</div><p class="links"><a class="dblinks" href="/pubmed/23104900" ref="ordinalpos=1019&amp;ncbi_uid=4682619&amp;link_uid=23104900">Citation</a><a class="dblinks" href="/pmc/articles/PMC3589936/" ref="ordinalpos=1019&amp;ncbi_uid=4682619&amp;link_uid=PMC3589936">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1020.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3913963/figure/F2/" ref="ordinalpos=1020&amp;ncbi_uid=5788769&amp;link_uid=PMC3913963" href="/pmc/articles/PMC3913963/"><img src="/pmc/articles/instance/3913963/bin/1756-8722-7-13-2.gif" src-large="/pmc/articles/instance/3913963/bin/1756-8722-7-13-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3913963/" ref="ordinalpos=1020&amp;ncbi_uid=5788769&amp;link_uid=PMC3913963" image-link="/pmc/articles/PMC3913963/figure/F2/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Signaling</span> pathways in the development of infantile hemangioma. </a></p><div class="supp"><p class="details"><div><b>The VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HemECs and HemSCs.</b> Upon ligand binding, VEGF receptors dimerize, leading to the phosphorylation of different tyrosine residues. Phosphorylation in turn elicits differential downstream <span class="highlight" style="background-color:">signaling</span> events.</div></p></div><div class="aux"><div class="resc">Yi Ji, et al. J Hematol Oncol. 2014;7:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/24479731" ref="ordinalpos=1020&amp;ncbi_uid=5788769&amp;link_uid=24479731">Citation</a><a class="dblinks" href="/pmc/articles/PMC3913963/" ref="ordinalpos=1020&amp;ncbi_uid=5788769&amp;link_uid=PMC3913963">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1021.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1895829/figure/fig5/" ref="ordinalpos=1021&amp;ncbi_uid=1020417&amp;link_uid=PMC1895829" href="/pmc/articles/PMC1895829/"><img src="/pmc/articles/instance/1895829/bin/zh80130697530005.gif" src-large="/pmc/articles/instance/1895829/bin/zh80130697530005.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1895829/" ref="ordinalpos=1021&amp;ncbi_uid=1020417&amp;link_uid=PMC1895829" image-link="/pmc/articles/PMC1895829/figure/fig5/" class="imagepopup">Figure 5. From: Zebrafish G protein ?2 is required for VEGF <span class="highlight" style="background-color:">signaling</span> during angiogenesis. </a></p><div class="supp"><p class="details"><div><b>A model of essential function of G protein in VEGF <span class="highlight" style="background-color:">signaling</span> during angiogenesis.</b> We hypothesize that the heterotrimeric G-protein, composed of α, β, and γ<sub>2</sub> subunits, is essential for the activation of PLCγ<sub>1</sub> and AKT in the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">TinChung Leung, et al. Blood. 2006 July 1;108(1):160-166.</div><p class="links"><a class="dblinks" href="/pubmed/16537812" ref="ordinalpos=1021&amp;ncbi_uid=1020417&amp;link_uid=16537812">Citation</a><a class="dblinks" href="/pmc/articles/PMC1895829/" ref="ordinalpos=1021&amp;ncbi_uid=1020417&amp;link_uid=PMC1895829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1022.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3789192/figure/fig6/" ref="ordinalpos=1022&amp;ncbi_uid=5399109&amp;link_uid=PMC3789192" href="/pmc/articles/PMC3789192/"><img src="/pmc/articles/instance/3789192/bin/cddis2013334f6.gif" src-large="/pmc/articles/instance/3789192/bin/cddis2013334f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3789192/" ref="ordinalpos=1022&amp;ncbi_uid=5399109&amp;link_uid=PMC3789192" image-link="/pmc/articles/PMC3789192/figure/fig6/" class="imagepopup">Figure 6.  From: Activation of TLR4 <span class="highlight" style="background-color:">signaling</span> promotes gastric cancer progression by inducing mitochondrial ROS production. </a></p><div class="supp"><p class="details"><div>Proposed model for the role of TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in GC cell proliferation. A drawing depicts the hypothetical role of TLR4 <span class="highlight" style="background-color:">signaling</span> in GC cell proliferation. We hypothesize that increased ROS generation, resulting from the activation of TLR4 <span class="highlight" style="background-color:">signaling</span>, mediates the activation of phosphorylated Akt, and then induces NF-<i>κ</i>B p65 nuclear translocation, which leads to GC cell proliferation</div></p></div><div class="aux"><div class="resc">X Yuan, et al. Cell Death Dis. 2013 September;4(9):e794.</div><p class="links"><a class="dblinks" href="/pubmed/24030146" ref="ordinalpos=1022&amp;ncbi_uid=5399109&amp;link_uid=24030146">Citation</a><a class="dblinks" href="/pmc/articles/PMC3789192/" ref="ordinalpos=1022&amp;ncbi_uid=5399109&amp;link_uid=PMC3789192">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1023.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3098250/figure/pone-0019776-g009/" ref="ordinalpos=1023&amp;ncbi_uid=2970394&amp;link_uid=PMC3098250" href="/pmc/articles/PMC3098250/"><img src="/pmc/articles/instance/3098250/bin/pone.0019776.g009.gif" src-large="/pmc/articles/instance/3098250/bin/pone.0019776.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3098250/" ref="ordinalpos=1023&amp;ncbi_uid=2970394&amp;link_uid=PMC3098250" image-link="/pmc/articles/PMC3098250/figure/pone-0019776-g009/" class="imagepopup">Figure 9. Chart explaining the function of HaCal in the cross talk between 20E and JH pathways..  From: The Participation of Calponin in the Cross Talk between 20-Hydroxyecdysone and Juvenile Hormone <span class="highlight" style="background-color:">Signaling</span> Pathways by Phosphorylation Variation. </a></p><div class="supp"><p class="details"><div>A, 20E <span class="highlight" style="background-color:">pathway</span>: 20E <span class="highlight" style="background-color:">signaling</span> leads HaCal protein phosphorylation by PKC via an unknown membrane <span class="highlight" style="background-color:">pathway</span> (<b>1</b>), HaCal is translocated into the nuclei (<b>2</b>), phosphorylated HaCal does not bind with phosphorylated USP1 in 20E <span class="highlight" style="background-color:">pathway</span> (<b>3</b>). EcR binds 20E and USP [52], [53] and other chaperone protein to form transcription complex, combine the 20E response element initiating the gene transcription [3], [52], [53], [54], [55], [56], [57]. B, JH <span class="highlight" style="background-color:">pathway</span>: Methoprene maintains HaCal non-phosphorylation and translocates HaCal into the nuclei (<b>1</b>), non-phosphorylated HaCal binds with non-phosphorylated USP1 (<b>2</b>). Met binds JH [20], [58], [59] and interacts with USP and other chaperone proteins [20], [58], and then this complex binds JH response element via Met to initiate JH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [59].</div></p></div><div class="aux"><div class="resc">Peng-Cheng Liu, et al. PLoS One. 2011;6(5):e19776.</div><p class="links"><a class="dblinks" href="/pubmed/21625546" ref="ordinalpos=1023&amp;ncbi_uid=2970394&amp;link_uid=21625546">Citation</a><a class="dblinks" href="/pmc/articles/PMC3098250/" ref="ordinalpos=1023&amp;ncbi_uid=2970394&amp;link_uid=PMC3098250">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1024.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2045624/figure/F1/" ref="ordinalpos=1024&amp;ncbi_uid=1165383&amp;link_uid=PMC2045624" href="/pmc/articles/PMC2045624/"><img src="/pmc/articles/instance/2045624/bin/JCI0733976.f1.gif" src-large="/pmc/articles/instance/2045624/bin/JCI0733976.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2045624/" ref="ordinalpos=1024&amp;ncbi_uid=1165383&amp;link_uid=PMC2045624" image-link="/pmc/articles/PMC2045624/figure/F1/" class="imagepopup">Figure 1. Crosstalk between OX40L-DCs and NKT cells..  From: OX40 <span class="highlight" style="background-color:">signaling</span> directly triggers the antitumor effects of NKT cells. </a></p><div class="supp"><p class="details"><div>It is hypothesized that OX40L-DCs first stimulate NKT cells (i) to proliferate and produce cytokines (ii). Subsequently, NKT cells activate DCs through cellular contacts (such as CD40/CD40L binding) and cytokine expression (iii), which induce the DCs to secret IL-12 and to upregulate costimulatory molecules such as CD80 and CD86. IL-12 stimulates the IL-12 receptor expressed by activated NKT cells (iv), which further potentiates NKT cell activation and forms a positive-feedback loop. Three lines of <span class="highlight" style="background-color:">signaling</span> transduction pathways are speculated to be activated in NKT cells: (a) TCR (Vα14 in mouse and Vα24 in human) <span class="highlight" style="background-color:">signaling</span> is assumed to be the first <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (b) other costimulatory <span class="highlight" style="background-color:">signaling</span> networks such as CD28 might be involved and interact with both the TCR <span class="highlight" style="background-color:">signaling</span> network and the OX40 <span class="highlight" style="background-color:">signaling</span> network; and (c) OX40 <span class="highlight" style="background-color:">signaling</span> components may control the fate of NKT cells through molecular interactions with the TCR <span class="highlight" style="background-color:">signaling</span> network. OX40 interacts with TNF receptor–associated factors and may lead to the regulation of transcription factors such as NF-AT, NF-κB, and T-bet.</div></p></div><div class="aux"><div class="resc">Dapeng Zhou. J Clin Invest. 2007 November 1;117(11):3169-3172.</div><p class="links"><a class="dblinks" href="/pubmed/17975660" ref="ordinalpos=1024&amp;ncbi_uid=1165383&amp;link_uid=17975660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2045624/" ref="ordinalpos=1024&amp;ncbi_uid=1165383&amp;link_uid=PMC2045624">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1025.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3454321/figure/pone-0045842-g005/" ref="ordinalpos=1025&amp;ncbi_uid=4273511&amp;link_uid=PMC3454321" href="/pmc/articles/PMC3454321/"><img src="/pmc/articles/instance/3454321/bin/pone.0045842.g005.gif" src-large="/pmc/articles/instance/3454321/bin/pone.0045842.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3454321/" ref="ordinalpos=1025&amp;ncbi_uid=4273511&amp;link_uid=PMC3454321" image-link="/pmc/articles/PMC3454321/figure/pone-0045842-g005/" class="imagepopup">Figure 5. Complement and IL-6 <span class="highlight" style="background-color:">signaling</span> in SARS-CoV infected-reinfected ferret lungs..  From: Lack of Innate Interferon Responses during SARS Coronavirus Infection in a Vaccination and Reinfection Ferret Model. </a></p><div class="supp"><p class="details"><div>(A) Complement and IL-6 <span class="highlight" style="background-color:">signaling</span> (C) genes selected by <span class="highlight" style="background-color:">pathway</span> analysis are shown in a one-way hierarchical cluster (red = upregulated, blue = downregulated). (B) IPA canonical complement system <span class="highlight" style="background-color:">pathway</span> analysis at 2 DPI. (D) IPA canonical IL-6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis at 5 DPI. All genes are significantly differently expressed (EDGE analysis: ≥2-fold change in at least one time point, p≤0.05, and q≤0.1).</div></p></div><div class="aux"><div class="resc">Mark J. Cameron, et al. PLoS One. 2012;7(9):e45842.</div><p class="links"><a class="dblinks" href="/pubmed/23029269" ref="ordinalpos=1025&amp;ncbi_uid=4273511&amp;link_uid=23029269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3454321/" ref="ordinalpos=1025&amp;ncbi_uid=4273511&amp;link_uid=PMC3454321">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1026.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3127908/figure/fig06/" ref="ordinalpos=1026&amp;ncbi_uid=3538971&amp;link_uid=PMC3127908" href="/pmc/articles/PMC3127908/"><img src="/pmc/articles/instance/3127908/bin/pnas.1104242108fig06.gif" src-large="/pmc/articles/instance/3127908/bin/pnas.1104242108fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3127908/" ref="ordinalpos=1026&amp;ncbi_uid=3538971&amp;link_uid=PMC3127908" image-link="/pmc/articles/PMC3127908/figure/fig06/" class="imagepopup">Fig. 6. From: Near-UV cyanobacteriochrome <span class="highlight" style="background-color:">signaling</span> system elicits negative phototaxis in the cyanobacterium Synechocystis sp. PCC 6803. </a></p><div class="supp"><p class="details"><div>Proposed model for the UirS/UirR/LsiR-based negative UV phototaxis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>Synechocystis</i>.</div></p></div><div class="aux"><div class="resc">Ji-Young Song, et al. Proc Natl Acad Sci U S A. 2011 June 28;108(26):10780-10785.</div><p class="links"><a class="dblinks" href="/pubmed/21670284" ref="ordinalpos=1026&amp;ncbi_uid=3538971&amp;link_uid=21670284">Citation</a><a class="dblinks" href="/pmc/articles/PMC3127908/" ref="ordinalpos=1026&amp;ncbi_uid=3538971&amp;link_uid=PMC3127908">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1027.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3486458/figure/F3/" ref="ordinalpos=1027&amp;ncbi_uid=4845503&amp;link_uid=PMC3486458" href="/pmc/articles/PMC3486458/"><img src="/pmc/articles/instance/3486458/bin/zjv9990967550003.gif" src-large="/pmc/articles/instance/3486458/bin/zjv9990967550003.jpg" alt="Fig 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3486458/" ref="ordinalpos=1027&amp;ncbi_uid=4845503&amp;link_uid=PMC3486458" image-link="/pmc/articles/PMC3486458/figure/F3/" class="imagepopup">Fig 3.  From: Porcine Circovirus Type 2 Induces Autophagy via the AMPK/ERK/TSC2/mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in PK-15 Cells. </a></p><div class="supp"><p class="details"><div>Downregulation of mTOR <span class="highlight" style="background-color:">signaling</span> is associated with ERK1/2 <span class="highlight" style="background-color:">pathway</span> activation after PCV2 infection. (A) PK-15 cells were treated as described in Fig. 2C, and whole-cell extracts were subjected to immunoblot analysis for LC3-II, p-mTOR, total mTOR, p-ERK1/2, and total ERK1/2. (B) Ratios of p-mTOR, p-ERK1/2, and LC3-II to β-actin were normalized to mock infection, set at 1.0. (C and D) PK-15 cells were treated as in Fig. 2G, and their ratios to β-actin were normalized to mock infection, set at 1.0.</div></p></div><div class="aux"><div class="resc">Binglin Zhu, et al. J Virol. 2012 November;86(22):12003-12012.</div><p class="links"><a class="dblinks" href="/pubmed/22915817" ref="ordinalpos=1027&amp;ncbi_uid=4845503&amp;link_uid=22915817">Citation</a><a class="dblinks" href="/pmc/articles/PMC3486458/" ref="ordinalpos=1027&amp;ncbi_uid=4845503&amp;link_uid=PMC3486458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1028.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3572677/figure/F1/" ref="ordinalpos=1028&amp;ncbi_uid=4621503&amp;link_uid=PMC3572677" href="/pmc/articles/PMC3572677/"><img src="/pmc/articles/instance/3572677/bin/fonc-03-00022-g001.gif" src-large="/pmc/articles/instance/3572677/bin/fonc-03-00022-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3572677/" ref="ordinalpos=1028&amp;ncbi_uid=4621503&amp;link_uid=PMC3572677" image-link="/pmc/articles/PMC3572677/figure/F1/" class="imagepopup">FIGURE 1.  From: How to target apoptosis <span class="highlight" style="background-color:">signaling</span> pathways for the treatment of pediatric cancers. </a></p><div class="supp"><p class="details"><div><b>Scheme of apoptosis pathways</b>. There are two key apoptosis <span class="highlight" style="background-color:">signaling</span> pathways, i.e., the death receptor (extrinsic) <span class="highlight" style="background-color:">pathway</span> and the mitochondrial (intrinsic) <span class="highlight" style="background-color:">pathway</span>. Stimulation of death receptors by their cognate ligands leads to activation of caspase-8, which either directly activates downstream caspases or initiates the mitochondrial <span class="highlight" style="background-color:">pathway</span> by cleaving Bid. Engagement of the intrinsic <span class="highlight" style="background-color:">pathway</span> results in the release of mitochondrial proteins such as cytochrome c or Smac into the cytosol, which in turn promotes activation of caspase-3 and apoptosis.</div></p></div><div class="aux"><div class="resc">Simone Fulda. Front Oncol. 2013;3:22.</div><p class="links"><a class="dblinks" href="/pubmed/23420072" ref="ordinalpos=1028&amp;ncbi_uid=4621503&amp;link_uid=23420072">Citation</a><a class="dblinks" href="/pmc/articles/PMC3572677/" ref="ordinalpos=1028&amp;ncbi_uid=4621503&amp;link_uid=PMC3572677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1029.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3696549/figure/F10/" ref="ordinalpos=1029&amp;ncbi_uid=5057401&amp;link_uid=PMC3696549" href="/pmc/articles/PMC3696549/"><img src="/pmc/articles/instance/3696549/bin/JCI65859.f10.gif" src-large="/pmc/articles/instance/3696549/bin/JCI65859.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3696549/" ref="ordinalpos=1029&amp;ncbi_uid=5057401&amp;link_uid=PMC3696549" image-link="/pmc/articles/PMC3696549/figure/F10/" class="imagepopup">Figure 10. A schematic representation of the proposed link between oncogenic TLR4/NANOG <span class="highlight" style="background-color:">signaling</span> and a defective TGF-β tumor suppressor <span class="highlight" style="background-color:">pathway</span> in generating TICs..  From: Reciprocal regulation by TLR4 and TGF-? in tumor-initiating stem-like cells. </a></p><div class="supp"><p class="details"><div>Ectopic upregulation of TLR4 and its activation by LPS induce the pluripotency factor NANOG, other stem cell genes, and self-renewal of TICs. NANOG induces IGF2BP3 and YAP1, which in turn inhibit TGF-β <span class="highlight" style="background-color:">signaling</span> at the level of SMAD3 phosphoactivation and p-SMAD3 nuclear translocation. The former effect is dependent on the IGF2BP3/AKT/mTOR <span class="highlight" style="background-color:">pathway</span>, while the latter is caused by p-YAP1/SMAD7/SMAD3 interactions, which are enhanced by IGF2BP3/AKT–mediated YAP1 phosphorylation. Restoration of the TGF-β tumor suppressor <span class="highlight" style="background-color:">pathway</span> by silencing <i>IGF2BP3</i> and <i>YAP1</i> downregulates TLR4 via SMAD3-mediated transcriptional repression and inhibits TLR4/NANOG–mediated TIC self-renewal and oncogenic activity while chemosensitizing TICs. A defective canonical TGF-β <span class="highlight" style="background-color:">pathway</span>, as in <i>Spnb<sup>+/–</sup></i> mice, upregulates TLR4 by transcriptional derepression and activates the TLR4/NANOG oncogenic <span class="highlight" style="background-color:">pathway</span> for liver tumorigenesis. </div></p></div><div class="aux"><div class="resc">Chia-Lin Chen, et al. J Clin Invest. 2013 July 1;123(7):2832-2849.</div><p class="links"><a class="dblinks" href="/pubmed/23921128" ref="ordinalpos=1029&amp;ncbi_uid=5057401&amp;link_uid=23921128">Citation</a><a class="dblinks" href="/pmc/articles/PMC3696549/" ref="ordinalpos=1029&amp;ncbi_uid=5057401&amp;link_uid=PMC3696549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1030.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2987401/figure/F1/" ref="ordinalpos=1030&amp;ncbi_uid=2609135&amp;link_uid=PMC2987401" href="/pmc/articles/PMC2987401/"><img src="/pmc/articles/instance/2987401/bin/nihms223381f1.gif" src-large="/pmc/articles/instance/2987401/bin/nihms223381f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2987401/" ref="ordinalpos=1030&amp;ncbi_uid=2609135&amp;link_uid=PMC2987401" image-link="/pmc/articles/PMC2987401/figure/F1/" class="imagepopup">Fig. 1. Model showing the possible involvement of NSMase-ceramide <span class="highlight" style="background-color:">pathway</span> in cell death <span class="highlight" style="background-color:">signaling</span> in oligodendrocytes in MS.  From: Sphingolipids in multiple sclerosis. </a></p><div class="supp"><p class="details"><div>Oxidative stress-inducing agents directly activate the NSMase-ceramide <span class="highlight" style="background-color:">pathway</span>. On the other hand, NGF requires p75NTR while TNFα and Ca<sup>2+</sup> use the phospholipase (PLA2) – arachidonic acid (AA) <span class="highlight" style="background-color:">pathway</span> to activate NSMase in oligodendrocytes.</div></p></div><div class="aux"><div class="resc">Arundhati Jana, et al. Neuromolecular Med. ;12(4):351-361.</div><p class="links"><a class="dblinks" href="/pubmed/20607622" ref="ordinalpos=1030&amp;ncbi_uid=2609135&amp;link_uid=20607622">Citation</a><a class="dblinks" href="/pmc/articles/PMC2987401/" ref="ordinalpos=1030&amp;ncbi_uid=2609135&amp;link_uid=PMC2987401">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1031.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3174986/figure/pone-0024963-g003/" ref="ordinalpos=1031&amp;ncbi_uid=3235410&amp;link_uid=PMC3174986" href="/pmc/articles/PMC3174986/"><img src="/pmc/articles/instance/3174986/bin/pone.0024963.g003.gif" src-large="/pmc/articles/instance/3174986/bin/pone.0024963.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3174986/" ref="ordinalpos=1031&amp;ncbi_uid=3235410&amp;link_uid=PMC3174986" image-link="/pmc/articles/PMC3174986/figure/pone-0024963-g003/" class="imagepopup">Figure 3. Type I interferon transcripts are induced by parasites on NK cells..  From: Plasmodium falciparum-Infected Erythrocytes and IL-12/IL-18 Induce Diverse Transcriptomes in Human NK Cells: IFN-?/? <span class="highlight" style="background-color:">Pathway</span> versus TREM <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Ingenuity Pathways Analysis identified the canonical <span class="highlight" style="background-color:">pathway</span> “Interferon <span class="highlight" style="background-color:">Signaling</span>” as highly associated with the 3D7-regulated genes on NK cells. Mainly IFN-αβ transcripts are induced. The differentially-regulated genes are marked in red (up-regulation) or green (down-regulation).</div></p></div><div class="aux"><div class="resc">Elisandra Grangeiro de Carvalho, et al. PLoS One. 2011;6(9):e24963.</div><p class="links"><a class="dblinks" href="/pubmed/21949811" ref="ordinalpos=1031&amp;ncbi_uid=3235410&amp;link_uid=21949811">Citation</a><a class="dblinks" href="/pmc/articles/PMC3174986/" ref="ordinalpos=1031&amp;ncbi_uid=3235410&amp;link_uid=PMC3174986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1032.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC145513/figure/cde176f2/" ref="ordinalpos=1032&amp;ncbi_uid=203828&amp;link_uid=PMC145513" href="/pmc/articles/PMC145513/"><img src="/pmc/articles/instance/145513/bin/cde176f2.gif" src-large="/pmc/articles/instance/145513/bin/cde176f2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC145513/" ref="ordinalpos=1032&amp;ncbi_uid=203828&amp;link_uid=PMC145513" image-link="/pmc/articles/PMC145513/figure/cde176f2/" class="imagepopup">Figure. From: Cpx <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> monitors biogenesis and affects assembly and expression of P pili. </a></p><div class="supp"><p class="details"><div><b>Fig. 2.</b> Short P pili are assembled without the Cpx <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> when expressed behind an inducible promoter. (<b>A</b>) Negative stain of wild-type cells expressing P pili behind the <i>trc</i> promoter (MC4100/pFJ29) at a magnification of 50 000. (<b>B</b>) Negative stain of cells with a <i>cpxR</i> null allele expressing P pili behind the <i>trc</i> promoter (TR51/pFJ29) at a magnification of 50 000. The bar represents 200 nm.</div></p></div><div class="aux"><div class="resc">Danielle L. Hung, et al. EMBO J. 2001 April 2;20(7):1508-1518.</div><p class="links"><a class="dblinks" href="/pubmed/11285215" ref="ordinalpos=1032&amp;ncbi_uid=203828&amp;link_uid=11285215">Citation</a><a class="dblinks" href="/pmc/articles/PMC145513/" ref="ordinalpos=1032&amp;ncbi_uid=203828&amp;link_uid=PMC145513">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1033.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3585591/figure/fig8/" ref="ordinalpos=1033&amp;ncbi_uid=5862976&amp;link_uid=PMC3585591" href="/pmc/articles/PMC3585591/"><img src="/pmc/articles/instance/3585591/bin/PP_211854_f8.gif" src-large="/pmc/articles/instance/3585591/bin/PP_211854_f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3585591/" ref="ordinalpos=1033&amp;ncbi_uid=5862976&amp;link_uid=PMC3585591" image-link="/pmc/articles/PMC3585591/figure/fig8/" class="imagepopup">Figure 8. From: Wheat Oxophytodienoate Reductase Gene TaOPR1 Confers Salinity Tolerance via Enhancement of Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> and Reactive Oxygen Species Scavenging1[C][W]. </a></p><div class="supp"><p class="details"><div>Hypothetical model of TaOPR1 action in Arabidopsis. TaOPR1 promotes ABA synthesis and <span class="highlight" style="background-color:">signaling</span> pathways, which elevates the activity occurring within the ABA-dependent abiotic stress-responsive <span class="highlight" style="background-color:">pathway</span> and increases the level of ROS-scavenging activity, thereby accounting for salinity tolerance. TaOPR1 can also directly improve downstream ABA <span class="highlight" style="background-color:">signaling</span> and the ABA-dependent stress-responsive <span class="highlight" style="background-color:">pathway</span> in an ABA synthesis-independent manner.</div></p></div><div class="aux"><div class="resc">Wei Dong, et al. Plant Physiol. 2013 March;161(3):1217-1228.</div><p class="links"><a class="dblinks" href="/pubmed/23321418" ref="ordinalpos=1033&amp;ncbi_uid=5862976&amp;link_uid=23321418">Citation</a><a class="dblinks" href="/pmc/articles/PMC3585591/" ref="ordinalpos=1033&amp;ncbi_uid=5862976&amp;link_uid=PMC3585591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1034.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3578810/figure/pone-0057231-g002/" ref="ordinalpos=1034&amp;ncbi_uid=4649219&amp;link_uid=PMC3578810" href="/pmc/articles/PMC3578810/"><img src="/pmc/articles/instance/3578810/bin/pone.0057231.g002.gif" src-large="/pmc/articles/instance/3578810/bin/pone.0057231.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3578810/" ref="ordinalpos=1034&amp;ncbi_uid=4649219&amp;link_uid=PMC3578810" image-link="/pmc/articles/PMC3578810/figure/pone-0057231-g002/" class="imagepopup">Figure 2. Apelin improves insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the hepatocytes treated by TNF-α..  From: Apelin Ameliorates TNF-?-Induced Reduction of Glycogen Synthesis in the Hepatocytes through G Protein-Coupled Receptor APJ. </a></p><div class="supp"><p class="details"><div>Apelin-13 improved insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (JNK-IRS1-AKT-GSK) in HepG2 cells (A) and mouse primary hepatocytes (B) treated with 10 ng/ml TNF-α for 24 h. Data represent the means ± S.E.M., n = 3 independent experiments. * p&lt;0.05; ** p&lt;0.01 and *** p&lt;0.001 by ANOVA test (v.s. control or TNF-α).</div></p></div><div class="aux"><div class="resc">Jiaojiao Chu, et al. PLoS One. 2013;8(2):e57231.</div><p class="links"><a class="dblinks" href="/pubmed/23437347" ref="ordinalpos=1034&amp;ncbi_uid=4649219&amp;link_uid=23437347">Citation</a><a class="dblinks" href="/pmc/articles/PMC3578810/" ref="ordinalpos=1034&amp;ncbi_uid=4649219&amp;link_uid=PMC3578810">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1035.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2064840/figure/fig9/" ref="ordinalpos=1035&amp;ncbi_uid=1201115&amp;link_uid=PMC2064840" href="/pmc/articles/PMC2064840/"><img src="/pmc/articles/instance/2064840/bin/jcb1780437f09.gif" src-large="/pmc/articles/instance/2064840/bin/jcb1780437f09.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2064840/" ref="ordinalpos=1035&amp;ncbi_uid=1201115&amp;link_uid=PMC2064840" image-link="/pmc/articles/PMC2064840/figure/fig9/" class="imagepopup">Figure 9. From: Cell size and invasion in TGF-?-induced epithelial to mesenchymal transition is regulated by activation of the mTOR <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Divergent TGF-β <span class="highlight" style="background-color:">signaling</span> pathways induce transcriptional regulation through Smads and translational regulation through the mTORC1 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Samy Lamouille, et al. J Cell Biol. 2007 July 30;178(3):437-451.</div><p class="links"><a class="dblinks" href="/pubmed/17646396" ref="ordinalpos=1035&amp;ncbi_uid=1201115&amp;link_uid=17646396">Citation</a><a class="dblinks" href="/pmc/articles/PMC2064840/" ref="ordinalpos=1035&amp;ncbi_uid=1201115&amp;link_uid=PMC2064840">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1036.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3971994/figure/F1/" ref="ordinalpos=1036&amp;ncbi_uid=5964473&amp;link_uid=PMC3971994" href="/pmc/articles/PMC3971994/"><img src="/pmc/articles/instance/3971994/bin/nihms566637f1.gif" src-large="/pmc/articles/instance/3971994/bin/nihms566637f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3971994/" ref="ordinalpos=1036&amp;ncbi_uid=5964473&amp;link_uid=PMC3971994" image-link="/pmc/articles/PMC3971994/figure/F1/" class="imagepopup">Fig. 1.  From: Mechanisms of HGF/Met <span class="highlight" style="background-color:">Signaling</span> to Brk and Sam68 in Breast Cancer Progression. </a></p><div class="supp"><p class="details"><div>HGF/Met receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Stimulation of Met receptor by its specific ligand HGF leads to activation of a variety of downstream <span class="highlight" style="background-color:">signaling</span> molecules, targeting cell proliferation, migration, and survival</div></p></div><div class="aux"><div class="resc">Alessia Locatelli, et al. Horm Cancer. 2012 April;3(0):14-25.</div><p class="links"><a class="dblinks" href="/pubmed/22124844" ref="ordinalpos=1036&amp;ncbi_uid=5964473&amp;link_uid=22124844">Citation</a><a class="dblinks" href="/pmc/articles/PMC3971994/" ref="ordinalpos=1036&amp;ncbi_uid=5964473&amp;link_uid=PMC3971994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1037.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3203698/figure/fig8/" ref="ordinalpos=1037&amp;ncbi_uid=4040524&amp;link_uid=PMC3203698" href="/pmc/articles/PMC3203698/"><img src="/pmc/articles/instance/3203698/bin/cr201127f8.gif" src-large="/pmc/articles/instance/3203698/bin/cr201127f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3203698/" ref="ordinalpos=1037&amp;ncbi_uid=4040524&amp;link_uid=PMC3203698" image-link="/pmc/articles/PMC3203698/figure/fig8/" class="imagepopup">Figure 8.  From: Cisplatin ototoxicity involves cytokines and STAT6 <span class="highlight" style="background-color:">signaling</span> network. </a></p><div class="supp"><p class="details"><div>Proposed mechanisms in cisplatin-induced auditory cell damage. Cisplatin induces ototoxicity through two different pathways. One is mediated by IL-4 and IL-13 cytokine <span class="highlight" style="background-color:">signaling</span> and subsequent STAT6, but NF-κB-independent <span class="highlight" style="background-color:">pathway</span>. Another is mediated primarily by TNF-α cytokine, which activates NF-κB <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Hyung-Jin Kim, et al. Cell Res. 2011 June;21(6):944-956.</div><p class="links"><a class="dblinks" href="/pubmed/21321603" ref="ordinalpos=1037&amp;ncbi_uid=4040524&amp;link_uid=21321603">Citation</a><a class="dblinks" href="/pmc/articles/PMC3203698/" ref="ordinalpos=1037&amp;ncbi_uid=4040524&amp;link_uid=PMC3203698">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1038.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3115623/figure/F5/" ref="ordinalpos=1038&amp;ncbi_uid=3019296&amp;link_uid=PMC3115623" href="/pmc/articles/PMC3115623/"><img src="/pmc/articles/instance/3115623/bin/nihms299705f5.gif" src-large="/pmc/articles/instance/3115623/bin/nihms299705f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3115623/" ref="ordinalpos=1038&amp;ncbi_uid=3019296&amp;link_uid=PMC3115623" image-link="/pmc/articles/PMC3115623/figure/F5/" class="imagepopup">Figure 5.  From: Role of CRF Receptor <span class="highlight" style="background-color:">Signaling</span> in Stress Vulnerability, Anxiety, and Depression. </a></p><div class="supp"><p class="details"><div>Major intracellular signal transduction pathways for CRF<sub>1</sub> and CRF<sub>2</sub> receptors. While the dominant mode of <span class="highlight" style="background-color:">signaling</span> for both CRF receptors <span class="highlight" style="background-color:">signaling</span> is Gs-coupled AC-PKA cascade, they may also signal via the PLC-PKC and ERK-MAPK cascades. CRF receptor AC-PKA <span class="highlight" style="background-color:">signaling</span> is stringently regulated by GRK- and arrestin-mediated homologous desensitization. Although the mechanisms regulating CRF receptor ERK-MAPK <span class="highlight" style="background-color:">signaling</span> are not yet fully understood, this <span class="highlight" style="background-color:">pathway</span> may potentiate BDNF-stimulated TrkB receptor function. (In color in <i>Annals</i> online.)</div></p></div><div class="aux"><div class="resc">Richard L. Hauger, et al. Ann N Y Acad Sci. ;1179:120-143.</div><p class="links"><a class="dblinks" href="/pubmed/19906236" ref="ordinalpos=1038&amp;ncbi_uid=3019296&amp;link_uid=19906236">Citation</a><a class="dblinks" href="/pmc/articles/PMC3115623/" ref="ordinalpos=1038&amp;ncbi_uid=3019296&amp;link_uid=PMC3115623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1039.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3144625/figure/F1/" ref="ordinalpos=1039&amp;ncbi_uid=3143097&amp;link_uid=PMC3144625" href="/pmc/articles/PMC3144625/"><img src="/pmc/articles/instance/3144625/bin/JCDR-2-91-g001.gif" src-large="/pmc/articles/instance/3144625/bin/JCDR-2-91-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3144625/" ref="ordinalpos=1039&amp;ncbi_uid=3143097&amp;link_uid=PMC3144625" image-link="/pmc/articles/PMC3144625/figure/F1/" class="imagepopup">Figure 1.  From: The role of endogenous aryl hydrocarbon receptor <span class="highlight" style="background-color:">signaling</span> in cardiovascular physiology. </a></p><div class="supp"><p class="details"><div>Aryl hydrocarbon receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Nan Zhang. J Cardiovasc Dis Res. 2011 Apr-Jun;2(2):91-95.</div><p class="links"><a class="dblinks" href="/pubmed/21814412" ref="ordinalpos=1039&amp;ncbi_uid=3143097&amp;link_uid=21814412">Citation</a><a class="dblinks" href="/pmc/articles/PMC3144625/" ref="ordinalpos=1039&amp;ncbi_uid=3143097&amp;link_uid=PMC3144625">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1040.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2593375/figure/f9/" ref="ordinalpos=1040&amp;ncbi_uid=1681899&amp;link_uid=PMC2593375" href="/pmc/articles/PMC2593375/"><img src="/pmc/articles/instance/2593375/bin/zmb0230878000009.gif" src-large="/pmc/articles/instance/2593375/bin/zmb0230878000009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2593375/" ref="ordinalpos=1040&amp;ncbi_uid=1681899&amp;link_uid=PMC2593375" image-link="/pmc/articles/PMC2593375/figure/f9/" class="imagepopup">FIG. 9. From: Role of the Caenorhabditis elegans Shc Adaptor Protein in the c-Jun N-Terminal Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> . </a></p><div class="supp"><p class="details"><div>Proposed model for SHC-1 function in the KGB-1 JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tomoaki Mizuno, et al. Mol Cell Biol. 2008 December;28(23):7041-7049.</div><p class="links"><a class="dblinks" href="/pubmed/18809575" ref="ordinalpos=1040&amp;ncbi_uid=1681899&amp;link_uid=18809575">Citation</a><a class="dblinks" href="/pmc/articles/PMC2593375/" ref="ordinalpos=1040&amp;ncbi_uid=1681899&amp;link_uid=PMC2593375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1041.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4148093/figure/F6/" ref="ordinalpos=1041&amp;ncbi_uid=6548716&amp;link_uid=PMC4148093" href="/pmc/articles/PMC4148093/"><img src="/pmc/articles/instance/4148093/bin/oncotarget-05-4709-g006.gif" src-large="/pmc/articles/instance/4148093/bin/oncotarget-05-4709-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4148093/" ref="ordinalpos=1041&amp;ncbi_uid=6548716&amp;link_uid=PMC4148093" image-link="/pmc/articles/PMC4148093/figure/F6/" class="imagepopup">Figure 6. HIF-VEGF-VEGFR <span class="highlight" style="background-color:">signaling</span> in CRC cells.  From: Intrinsic bevacizumab resistance is associated with prolonged activation of autocrine VEGF <span class="highlight" style="background-color:">signaling</span> and hypoxia tolerance in colorectal cancer cells and can be overcome by nintedanib, a small molecule angiokinase inhibitor. </a></p><div class="supp"><p class="details"><div>Hypoxia exposure activates HIF, thereby increasing the expression of VEGF, a HIF-transcriptional target. Subsequent binding of VEGF to its receptor on either CRC cells (autocrine <span class="highlight" style="background-color:">pathway</span>) or tumor-associated endothelial cells (paracrine <span class="highlight" style="background-color:">pathway</span>) leads to activation of VEGFR and its downstream <span class="highlight" style="background-color:">signaling</span> pathways. Receptor tyrosine kinases like VEGFR are able to activate and/or stabilize HIF and VEGF, thereby promoting a positive feed-back loop.</div></p></div><div class="aux"><div class="resc">Paul Mésange, et al. Oncotarget. 2014 July;5(13):4709-4721.</div><p class="links"><a class="dblinks" href="/pubmed/25015210" ref="ordinalpos=1041&amp;ncbi_uid=6548716&amp;link_uid=25015210">Citation</a><a class="dblinks" href="/pmc/articles/PMC4148093/" ref="ordinalpos=1041&amp;ncbi_uid=6548716&amp;link_uid=PMC4148093">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1042.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2597184/figure/F2/" ref="ordinalpos=1042&amp;ncbi_uid=1539609&amp;link_uid=PMC2597184" href="/pmc/articles/PMC2597184/"><img src="/pmc/articles/instance/2597184/bin/nihms-81185-f0002.gif" src-large="/pmc/articles/instance/2597184/bin/nihms-81185-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2597184/" ref="ordinalpos=1042&amp;ncbi_uid=1539609&amp;link_uid=PMC2597184" image-link="/pmc/articles/PMC2597184/figure/F2/" class="imagepopup">Figure 2. Schematic representation of two NF-κB <span class="highlight" style="background-color:">signaling</span> pathways.  From: Nuclear Factor-kappa B <span class="highlight" style="background-color:">Signaling</span> in Skeletal Muscle Atrophy. </a></p><div class="supp"><p class="details"><div>Activation of NF-κB can occur by classical or alternative <span class="highlight" style="background-color:">pathway</span>. Classical <span class="highlight" style="background-color:">pathway</span> involves the activation of IKKβ leading to phosphorylation and degradation of IκBs. The alternative <span class="highlight" style="background-color:">pathway</span> involves IKKα and leads to the phosphorylation of processing of p100, generating p52:relB heterodimers. NIK, NF-κB-inducing kinase; Ub, ubiquitin; MEKK3, mitogen-activated protein kinase kinase kinase 3.</div></p></div><div class="aux"><div class="resc">Hong Li, et al. J Mol Med (Berl). ;86(10):1113-1126.</div><p class="links"><a class="dblinks" href="/pubmed/18574572" ref="ordinalpos=1042&amp;ncbi_uid=1539609&amp;link_uid=18574572">Citation</a><a class="dblinks" href="/pmc/articles/PMC2597184/" ref="ordinalpos=1042&amp;ncbi_uid=1539609&amp;link_uid=PMC2597184">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1043.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013670/figure/f6-ijms-15-06925/" ref="ordinalpos=1043&amp;ncbi_uid=6110924&amp;link_uid=PMC4013670" href="/pmc/articles/PMC4013670/"><img src="/pmc/articles/instance/4013670/bin/ijms-15-06925f6.gif" src-large="/pmc/articles/instance/4013670/bin/ijms-15-06925f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013670/" ref="ordinalpos=1043&amp;ncbi_uid=6110924&amp;link_uid=PMC4013670" image-link="/pmc/articles/PMC4013670/figure/f6-ijms-15-06925/" class="imagepopup">Figure 6. From: Protective Effect of Resveratrol against IL-1?-Induced Inflammatory Response on Human Osteoarthritic Chondrocytes Partly via the TLR4/MyD88/NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: An "in Vitro Study". </a></p><div class="supp"><p class="details"><div>Schematic describing the proposed anti-inflammatory response resulting from resveratrol treatment and its effects on TLR4 following stimulation of chondrocytes with IL-1β. IL-1β-upregulated TLR4 expression results in the activation of the TLR4/MyD88/NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> causing translocation of NF-κB into the nucleus resulting in the production of proinflammatory cytokines such as IL-1β and TNFα. Resveratrol-mediated directly inhibition of TLR4 expression could block the TLR4/MyD88/NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (a1, dashed line), or resveratrol could affect TLR4 <span class="highlight" style="background-color:">pathway</span> processes in the nucleus through unknown regulatory mechanisms that regulate the gene expression (a2, dashed line); Direct inhibition of NF-κB by resveratrol, as described by Csaki [25] and Shakibaei [26] is shown in (b, solid line); Mechanisms involved in NF-κB suppression by resveratrol-activated SIRT1, as described by Lei [31] are shown in (c, solid line).</div></p></div><div class="aux"><div class="resc">Li Liu, et al. Int J Mol Sci. 2014 April;15(4):6925-6940.</div><p class="links"><a class="dblinks" href="/pubmed/24758933" ref="ordinalpos=1043&amp;ncbi_uid=6110924&amp;link_uid=24758933">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013670/" ref="ordinalpos=1043&amp;ncbi_uid=6110924&amp;link_uid=PMC4013670">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1044.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3102680/figure/pone-0020199-g004/" ref="ordinalpos=1044&amp;ncbi_uid=3000669&amp;link_uid=PMC3102680" href="/pmc/articles/PMC3102680/"><img src="/pmc/articles/instance/3102680/bin/pone.0020199.g004.gif" src-large="/pmc/articles/instance/3102680/bin/pone.0020199.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3102680/" ref="ordinalpos=1044&amp;ncbi_uid=3000669&amp;link_uid=PMC3102680" image-link="/pmc/articles/PMC3102680/figure/pone-0020199-g004/" class="imagepopup">Figure 4. Inferring <span class="highlight" style="background-color:">pathway</span> activity in Ras-transformed HBECs and NSCLC cells..  From: Phosphoproteomics Identifies Oncogenic Ras <span class="highlight" style="background-color:">Signaling</span> Targets and Their Involvement in Lung Adenocarcinomas. </a></p><div class="supp"><p class="details"><div>(A) The levels of regulated phosphoprotein signatures obtained from quantitative proteomics analysis were subjected to <span class="highlight" style="background-color:">pathway</span> activity analysis. Equations for summarizing the levels of phosphorylated proteins and for analyzing <span class="highlight" style="background-color:">pathway</span> activity are shown. <i>x<sub>i,j</sub></i> and <i>ρ<sub>i,j</sub></i> are the mean and standard deviation, respectively, of the value of the phosphoprotein <i>j</i> in class <i>i,</i> and <i>k</i> is the number of proteins in a given <span class="highlight" style="background-color:">pathway</span>. <i>a<sub>p</sub></i> is the activity score for <span class="highlight" style="background-color:">pathway</span> <i>p</i> between classes <i>c<sub>1</sub></i> and <i>c<sub>2</sub></i>, and <img src="/pmc/articles/instance/3102680/bin/pone.0020199.e003.jpg" class="inline-graphic" style="border-style: none;" /> and <img src="/pmc/articles/instance/3102680/bin/pone.0020199.e004.jpg" class="inline-graphic" style="border-style: none;" /> are the numbers of replicate experiments for the respective classes. In our case, <img src="/pmc/articles/instance/3102680/bin/pone.0020199.e005.jpg" class="inline-graphic" style="border-style: none;" /> and <img src="/pmc/articles/instance/3102680/bin/pone.0020199.e006.jpg" class="inline-graphic" style="border-style: none;" />are each 3. (B) Inferred activities of MAPK and Aurora B <span class="highlight" style="background-color:">signaling</span> pathways in Ras-transformed HBECs and NSCLC cells in comparison with HBEC-3KT are shown as relative <span class="highlight" style="background-color:">pathway</span> activity scores (t-scores). (C) The involvement of the Ras/Raf/MEK/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its cross-talk with basophilic kinases in NSCLC cells. Kinases identified as being upstream of the regulated phosphosites by NetworKIN analysis in this study are highlighted in bold font. Kinases targeting proline-directed and basophilic S/T sites are labeled in green and blue letters, respectively. The frequencies of the individual kinases identified in A549, H322, and H1299 cells in comparison with HBEC-3KT are shown in parentheses.</div></p></div><div class="aux"><div class="resc">Putty-Reddy Sudhir, et al. PLoS One. 2011;6(5):e20199.</div><p class="links"><a class="dblinks" href="/pubmed/21637843" ref="ordinalpos=1044&amp;ncbi_uid=3000669&amp;link_uid=21637843">Citation</a><a class="dblinks" href="/pmc/articles/PMC3102680/" ref="ordinalpos=1044&amp;ncbi_uid=3000669&amp;link_uid=PMC3102680">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1045.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2100073/figure/F5/" ref="ordinalpos=1045&amp;ncbi_uid=1122440&amp;link_uid=PMC2100073" href="/pmc/articles/PMC2100073/"><img src="/pmc/articles/instance/2100073/bin/1471-2105-8-335-5.gif" src-large="/pmc/articles/instance/2100073/bin/1471-2105-8-335-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2100073/" ref="ordinalpos=1045&amp;ncbi_uid=1122440&amp;link_uid=PMC2100073" image-link="/pmc/articles/PMC2100073/figure/F5/" class="imagepopup">Figure 5.  From: PathFinder: mining signal transduction <span class="highlight" style="background-color:">pathway</span> segments from protein-protein interaction networks. </a></p><div class="supp"><p class="details"><div><b>The filamentation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. (A) The main chain of the filamentation <span class="highlight" style="background-color:">pathway</span> (KEGG Database), (B) PathFinder output for the Sho1-Tec1 pair (75% recall, 17% precision). For (A), the dashed interactions indicate that these interactions do not exists in the database. For (B), the proteins that were not on the main chain of the <span class="highlight" style="background-color:">pathway</span> were not colored, whereas the proteins on the main chain are colored grey. Proteins that are part of other pathways are colored with different colors.</div></p></div><div class="aux"><div class="resc">Gurkan Bebek, et al. BMC Bioinformatics. 2007;8:335-335.</div><p class="links"><a class="dblinks" href="/pubmed/17854489" ref="ordinalpos=1045&amp;ncbi_uid=1122440&amp;link_uid=17854489">Citation</a><a class="dblinks" href="/pmc/articles/PMC2100073/" ref="ordinalpos=1045&amp;ncbi_uid=1122440&amp;link_uid=PMC2100073">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1046.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2214701/figure/F2/" ref="ordinalpos=1046&amp;ncbi_uid=1149454&amp;link_uid=PMC2214701" href="/pmc/articles/PMC2214701/"><img src="/pmc/articles/instance/2214701/bin/JCI0833612.f2.gif" src-large="/pmc/articles/instance/2214701/bin/JCI0833612.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2214701/" ref="ordinalpos=1046&amp;ncbi_uid=1149454&amp;link_uid=PMC2214701" image-link="/pmc/articles/PMC2214701/figure/F2/" class="imagepopup">Figure 2. The central role of canonical Wnt <span class="highlight" style="background-color:">signaling</span> in regulating osteoblast lineage specification, expansion, and terminal differentiation..  From: Building bone to reverse osteoporosis and repair fractures. </a></p><div class="supp"><p class="details"><div>Osteoblasts are derived from multipotent mesodermal or neural crest progenitors. Activation of the canonical (ca) Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, manifest through β-catenin stabilization, prevents the formation of cartilage (chondrogenesis). Wnt10b prevents adipogenesis (40). The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promotes survival of all cells of the osteoblast lineage and induces the proliferation of preosteoblasts. +, canonical Wnt <span class="highlight" style="background-color:">signaling</span> promotes the process; –, canonical Wnt <span class="highlight" style="background-color:">signaling</span> inhibits the process; AP, alkaline phosphatase; Cola1, collagen α1; DMP1, dentin matrix protein 1; OCN, osteocalcin; Osx, osterix; PTHR, receptor for PTH.                </div></p></div><div class="aux"><div class="resc">Sundeep Khosla, et al. J Clin Invest. 2008 February 1;118(2):421-428.</div><p class="links"><a class="dblinks" href="/pubmed/18246192" ref="ordinalpos=1046&amp;ncbi_uid=1149454&amp;link_uid=18246192">Citation</a><a class="dblinks" href="/pmc/articles/PMC2214701/" ref="ordinalpos=1046&amp;ncbi_uid=1149454&amp;link_uid=PMC2214701">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1047.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136489/figure/pone-0021902-g007/" ref="ordinalpos=1047&amp;ncbi_uid=3090363&amp;link_uid=PMC3136489" href="/pmc/articles/PMC3136489/"><img src="/pmc/articles/instance/3136489/bin/pone.0021902.g007.gif" src-large="/pmc/articles/instance/3136489/bin/pone.0021902.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136489/" ref="ordinalpos=1047&amp;ncbi_uid=3090363&amp;link_uid=PMC3136489" image-link="/pmc/articles/PMC3136489/figure/pone-0021902-g007/" class="imagepopup">Figure 7. Activation of the Interferon <span class="highlight" style="background-color:">Pathway</span> in Subgroup X..  From: Pathways Activated during Human Asthma Exacerbation as Revealed by Gene Expression Patterns in Blood. </a></p><div class="supp"><p class="details"><div>Type I and Type II interferon-induced genes including IRFs that are involved in the regulation of interferon response, and the JAK-STAT <span class="highlight" style="background-color:">signaling</span> components of the IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that are upregulated in Subgroup X exacerbations are shown in red.</div></p></div><div class="aux"><div class="resc">Unnur S. Bjornsdottir, et al. PLoS One. 2011;6(7):e21902.</div><p class="links"><a class="dblinks" href="/pubmed/21779351" ref="ordinalpos=1047&amp;ncbi_uid=3090363&amp;link_uid=21779351">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136489/" ref="ordinalpos=1047&amp;ncbi_uid=3090363&amp;link_uid=PMC3136489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1048.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3015011/figure/F6/" ref="ordinalpos=1048&amp;ncbi_uid=3605545&amp;link_uid=PMC3015011" href="/pmc/articles/PMC3015011/"><img src="/pmc/articles/instance/3015011/bin/nihms259696f6.gif" src-large="/pmc/articles/instance/3015011/bin/nihms259696f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3015011/" ref="ordinalpos=1048&amp;ncbi_uid=3605545&amp;link_uid=PMC3015011" image-link="/pmc/articles/PMC3015011/figure/F6/" class="imagepopup">Figure 6.  From: Novel Chemical Suppressors of Long QT Syndrome Identified by an in vivo Functional Screen. </a></p><div class="supp"><p class="details"><div>Flurandrenolide acts via the glucocorticoid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (a) Dose-response curve using steroids known to act through specific <span class="highlight" style="background-color:">signaling</span> pathways: dexamethasone (glucocorticoid <span class="highlight" style="background-color:">pathway</span>), testosterone (androgen <span class="highlight" style="background-color:">pathway</span>), and DOCA (mineralocorticoid <span class="highlight" style="background-color:">pathway</span>). (b) Knockdown of the glucocorticoid receptor blocks rescue of long QT by flurandrenolide and dexamethasone (n= 26–31).</div></p></div><div class="aux"><div class="resc">David S. Peal, et al. Circulation. ;123(1):23-30.</div><p class="links"><a class="dblinks" href="/pubmed/21098441" ref="ordinalpos=1048&amp;ncbi_uid=3605545&amp;link_uid=21098441">Citation</a><a class="dblinks" href="/pmc/articles/PMC3015011/" ref="ordinalpos=1048&amp;ncbi_uid=3605545&amp;link_uid=PMC3015011">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1049.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3938668/figure/pone-0090318-g005/" ref="ordinalpos=1049&amp;ncbi_uid=5881307&amp;link_uid=PMC3938668" href="/pmc/articles/PMC3938668/"><img src="/pmc/articles/instance/3938668/bin/pone.0090318.g005.gif" src-large="/pmc/articles/instance/3938668/bin/pone.0090318.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3938668/" ref="ordinalpos=1049&amp;ncbi_uid=5881307&amp;link_uid=PMC3938668" image-link="/pmc/articles/PMC3938668/figure/pone-0090318-g005/" class="imagepopup">Figure 5. Effect of baicalein on the p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Baicalein Reduces the Invasion of Glioma Cells via Reducing the Activity of p38 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) The protein levels of p38 and p-p38. (B) Phosphorylation density of p38 was digitally scanned. Values represent the means ± SD of three independent experiments performed in triplicate. *<i>p</i>&lt;0.05 and **<i>p</i>&lt;0.01 compared with the control group.</div></p></div><div class="aux"><div class="resc">Zhenni Zhang, et al. PLoS One. 2014;9(2):e90318.</div><p class="links"><a class="dblinks" href="/pubmed/24587321" ref="ordinalpos=1049&amp;ncbi_uid=5881307&amp;link_uid=24587321">Citation</a><a class="dblinks" href="/pmc/articles/PMC3938668/" ref="ordinalpos=1049&amp;ncbi_uid=5881307&amp;link_uid=PMC3938668">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1050.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3160520/figure/F2/" ref="ordinalpos=1050&amp;ncbi_uid=3579058&amp;link_uid=PMC3160520" href="/pmc/articles/PMC3160520/"><img src="/pmc/articles/instance/3160520/bin/nihms292596f2.gif" src-large="/pmc/articles/instance/3160520/bin/nihms292596f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3160520/" ref="ordinalpos=1050&amp;ncbi_uid=3579058&amp;link_uid=PMC3160520" image-link="/pmc/articles/PMC3160520/figure/F2/" class="imagepopup">Fig. 2.  From: Apolipoprotein E reduces food intake via PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the hypothalamus. </a></p><div class="supp"><p class="details"><div>Inhibition of apoE-induced activation of pAkt by LY294002 (50 μM), an inhibitor of the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, in cultured hypothalamic cells. A, representative immunoblots of pAkt and total Akt; B, quantitative analysis. Data represent mean ± SEM; n = 3. Bars that do not share the same letter are significantly different from each other.</div></p></div><div class="aux"><div class="resc">Ling Shen, et al. Physiol Behav. ;105(1):124-128.</div><p class="links"><a class="dblinks" href="/pubmed/21536059" ref="ordinalpos=1050&amp;ncbi_uid=3579058&amp;link_uid=21536059">Citation</a><a class="dblinks" href="/pmc/articles/PMC3160520/" ref="ordinalpos=1050&amp;ncbi_uid=3579058&amp;link_uid=PMC3160520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1051.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2862093/figure/F6/" ref="ordinalpos=1051&amp;ncbi_uid=2311562&amp;link_uid=PMC2862093" href="/pmc/articles/PMC2862093/"><img src="/pmc/articles/instance/2862093/bin/nihms191985f6.gif" src-large="/pmc/articles/instance/2862093/bin/nihms191985f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2862093/" ref="ordinalpos=1051&amp;ncbi_uid=2311562&amp;link_uid=PMC2862093" image-link="/pmc/articles/PMC2862093/figure/F6/" class="imagepopup">Figure 6. Proposed GLP-1R signal transduction scheme.  From: Glucagon-like Peptide-1 Receptor (GLP-1R) is present on human hepatocytes and has a direct role in decreasing hepatic steatosis in vitro by modulating elements of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>In our previous work we demonstrated that GLP-1 or Exendin-4 increased cAMP production. Here we propose that the GLP-1 action shares key downstream components of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, including PKCζ, which has been shown to be a key factor in NAFLD.</div></p></div><div class="aux"><div class="resc">Nitika Arora Gupta, et al. Hepatology. ;51(5):1584-1592.</div><p class="links"><a class="dblinks" href="/pubmed/20225248" ref="ordinalpos=1051&amp;ncbi_uid=2311562&amp;link_uid=20225248">Citation</a><a class="dblinks" href="/pmc/articles/PMC2862093/" ref="ordinalpos=1051&amp;ncbi_uid=2311562&amp;link_uid=PMC2862093">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1052.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2553439/figure/F1/" ref="ordinalpos=1052&amp;ncbi_uid=1767308&amp;link_uid=PMC2553439" href="/pmc/articles/PMC2553439/"><img src="/pmc/articles/instance/2639041/bin/btn403f1.gif" src-large="/pmc/articles/instance/2639041/bin/btn403f1.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2553439/" ref="ordinalpos=1052&amp;ncbi_uid=1767308&amp;link_uid=PMC2553439" image-link="/pmc/articles/PMC2553439/figure/F1/" class="imagepopup">Fig. 1. From: Predicting <span class="highlight" style="background-color:">pathway</span> membership via domain signatures. </a></p><div class="supp"><p class="details"><div>Prediction performance of our method (10×10-fold cross-validation). The accuracy measure uses the same loss function, which was used to train the classifier, and which takes into account the KEGG hierarchy. (<b>A</b>) <span class="highlight" style="background-color:">Pathway</span> prediction within pruned KEGG hierarchy (53 branches). (<b>B</b>) <span class="highlight" style="background-color:">Pathway</span> component prediction for <span class="highlight" style="background-color:">signaling</span> pathways (19 branches).</div></p></div><div class="aux"><div class="resc">Holger Fröhlich, et al. Bioinformatics. 2008 October 1;24(19):2137-2142.</div><p class="links"><a class="dblinks" href="/pubmed/18676972" ref="ordinalpos=1052&amp;ncbi_uid=1767308&amp;link_uid=18676972">Citation</a><a class="dblinks" href="/pmc/articles/PMC2553439/" ref="ordinalpos=1052&amp;ncbi_uid=1767308&amp;link_uid=PMC2553439">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1053.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3357552/figure/F1/" ref="ordinalpos=1053&amp;ncbi_uid=4034885&amp;link_uid=PMC3357552" href="/pmc/articles/PMC3357552/"><img src="/pmc/articles/instance/3357552/bin/fphys-03-00138-g001.gif" src-large="/pmc/articles/instance/3357552/bin/fphys-03-00138-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3357552/" ref="ordinalpos=1053&amp;ncbi_uid=4034885&amp;link_uid=PMC3357552" image-link="/pmc/articles/PMC3357552/figure/F1/" class="imagepopup">Figure 1.  From: Toll-Like Receptors in Liver Fibrosis: Cellular Crosstalk and Mechanisms. </a></p><div class="supp"><p class="details"><div><b>The overview of TLR <span class="highlight" style="background-color:">signaling</span></b>. TLR1, TLR2, TLR4, TLR5, and TLR6 are expressed on cell membrane. TLR3, TLR7/8, and TLR9 are expressed in endosome. All TLRs, expect for TLR3, activate MyD88-dependent <span class="highlight" style="background-color:">pathway</span> to induce NF-κB and p38/JNK activation. TLR2 and TLR4 <span class="highlight" style="background-color:">signaling</span> require TIRAP and MyD88. TLR3 requires IRIF to activate TBK1/IKKε. Subsequent to TLR4 internalization, TLR4 <span class="highlight" style="background-color:">signaling</span> activates TRAM/TRIF-dependent <span class="highlight" style="background-color:">pathway</span>. TLR3/4-dependent TRIF-dependent <span class="highlight" style="background-color:">signaling</span> induces IRF3 activation and IFN-β production. TLR7/8 and TLR9 induce IFN-α production through IRF7.</div></p></div><div class="aux"><div class="resc">Ling Yang, et al. Front Physiol. 2012;3:138.</div><p class="links"><a class="dblinks" href="/pubmed/22661952" ref="ordinalpos=1053&amp;ncbi_uid=4034885&amp;link_uid=22661952">Citation</a><a class="dblinks" href="/pmc/articles/PMC3357552/" ref="ordinalpos=1053&amp;ncbi_uid=4034885&amp;link_uid=PMC3357552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1054.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3172280/figure/pmed-1001093-g005/" ref="ordinalpos=1054&amp;ncbi_uid=3225638&amp;link_uid=PMC3172280" href="/pmc/articles/PMC3172280/"><img src="/pmc/articles/instance/3172280/bin/pmed.1001093.g005.gif" src-large="/pmc/articles/instance/3172280/bin/pmed.1001093.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3172280/" ref="ordinalpos=1054&amp;ncbi_uid=3225638&amp;link_uid=PMC3172280" image-link="/pmc/articles/PMC3172280/figure/pmed-1001093-g005/" class="imagepopup">Figure 5. Dynamic co-expression modules and their dominant trajectories across the five ocMOF subgroups..  From: Dissecting Inflammatory Complications in Critically Injured Patients by Within-Patient Gene Expression Changes: A Longitudinal Clinical Genomics Study. </a></p><div class="supp"><p class="details"><div>We used DAVID to obtain 54 functionally related gene sets from the 3,663 most significant probesets (10% FDR), which were then clustered into five modules according to the similarity of their dominant trajectories across the ocMOF subgroups. Modules A, B, C, D, and E contain 47, 37, 577, 231, and 364 probesets, respectively. We applied IPA to identify enriched pathways within each module. This IPA analysis shortlisted the following pathways as statistically significant (p-value&lt;0.002, after Bonferroni correction; see Table S5): Oxidative Phosphorylation (Module A); RAN, IL-10 and IL-6 <span class="highlight" style="background-color:">signaling</span>, and the Glycosphingolipid Biosynthesis-Lactoseries <span class="highlight" style="background-color:">Pathway</span> (Module C); Allograft Rejection <span class="highlight" style="background-color:">Signaling</span>, Antigen Presentation <span class="highlight" style="background-color:">Pathway</span>, Cytotoxic T Lymphocyte-mediated Apoptosis of Target Cells, OX40 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>, Nur77 <span class="highlight" style="background-color:">Signaling</span> in T Lymphocytes (Module D); and Protein Ubiquitination <span class="highlight" style="background-color:">Pathway</span>, Hypoxia <span class="highlight" style="background-color:">Signaling</span>, and Cleavage and Polyadenylation of Pre-mRNA in the Cardiovascular System (Module E). Note that Module A contains 47 probesets and one statistically significant <span class="highlight" style="background-color:">pathway</span>, and Module B contains 37 probesets and no statistically significant <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Keyur H. Desai, et al. PLoS Med. 2011 September;8(9):e1001093.</div><p class="links"><a class="dblinks" href="/pubmed/21931541" ref="ordinalpos=1054&amp;ncbi_uid=3225638&amp;link_uid=21931541">Citation</a><a class="dblinks" href="/pmc/articles/PMC3172280/" ref="ordinalpos=1054&amp;ncbi_uid=3225638&amp;link_uid=PMC3172280">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1055.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3357354/figure/F2/" ref="ordinalpos=1055&amp;ncbi_uid=4093001&amp;link_uid=PMC3357354" href="/pmc/articles/PMC3357354/"><img src="/pmc/articles/instance/3357354/bin/psb-7-135-g2.gif" src-large="/pmc/articles/instance/3357354/bin/psb-7-135-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3357354/" ref="ordinalpos=1055&amp;ncbi_uid=4093001&amp;link_uid=PMC3357354" image-link="/pmc/articles/PMC3357354/figure/F2/" class="imagepopup">Figure. From: KNAT1, KNAT2 and KNAT6 act downstream in the IDA-HAE/HSL2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to regulate floral organ abscission. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Model of IDA <span class="highlight" style="background-color:">signaling</span>. A mature posttranslational modified IDA peptide consisting of the EPIP, or part of the EPIP, domain may bind the extracellular LRRs of a HAE homodimer, a HSL2 homodimer, or a HAE-HSL2 heterodimer and cause autophosphorylation (P) of the kinase receptor domains. The signal from IDA-HAE/HSL2 is suggested to be transduced via a phosphorelay involving MKK4, MKK5, MPK3 and MPK6. Genetic evidence suggests that BP/KNAT1 acts as a negative downstream component of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by inhibiting cell separation and the expression of <i>KNAT2</i> and <i>KNAT6</i>. Upon activation of the IDA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the BP/KNAT1 restriction of <i>KNAT2</i> and <i>KNAT6</i> is elevated, and these transcription factors can act as positive regulators of floral organ separation by positively regulating genes promoting AZ cell separation. In addition, BP/KNAT1 is a positive regulator of EVR, which influences cell separation by a separate parallel <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Melinka A. Butenko, et al. Plant Signal Behav. 2012 January 1;7(1):135-138.</div><p class="links"><a class="dblinks" href="/pubmed/22301980" ref="ordinalpos=1055&amp;ncbi_uid=4093001&amp;link_uid=22301980">Citation</a><a class="dblinks" href="/pmc/articles/PMC3357354/" ref="ordinalpos=1055&amp;ncbi_uid=4093001&amp;link_uid=PMC3357354">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1056.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3121899/figure/F3/" ref="ordinalpos=1056&amp;ncbi_uid=3501865&amp;link_uid=PMC3121899" href="/pmc/articles/PMC3121899/"><img src="/pmc/articles/instance/3121899/bin/nihms273642f3.gif" src-large="/pmc/articles/instance/3121899/bin/nihms273642f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3121899/" ref="ordinalpos=1056&amp;ncbi_uid=3501865&amp;link_uid=PMC3121899" image-link="/pmc/articles/PMC3121899/figure/F3/" class="imagepopup">Fig. 3.  From: THE OCCUPANCY OF ENDOTHELIAL PROTEIN C RECEPTOR BY ITS LIGAND MODULATES THE PAR-1 DEPENDENT <span class="highlight" style="background-color:">SIGNALING</span> SPECIFICITY OF COAGULATION PROTEASES. </a></p><div class="supp"><p class="details"><div>Cartoons of PAR-1-dependent <span class="highlight" style="background-color:">signaling</span> by APC and thrombin in endothelial cells when EPCR is either free or occupied by protein C. (Left Panel) EPCR interacts with caveolin-1 (Cav-1) within lipid-rafts of endothelial cells when it is not occupied by the Gla-domain of protein C. Thrombin cleavage of PAR-1 activates RhoA, up-regulates the NF-κB <span class="highlight" style="background-color:">pathway</span> and elicits disruptive <span class="highlight" style="background-color:">signaling</span> responses by <span class="highlight" style="background-color:">signaling</span> through G<sub>12/13</sub> and/or G<sub>q</sub> proteins. (Right Panel) The occupancy of EPCR by protein C leads to dissociation of EPCR from caveolin-1 and a switch in the specificity of PAR-1, presumably by <span class="highlight" style="background-color:">signaling</span> via the G<sub>i</sub>-protein. Under these conditions, the thrombin cleavage of PAR-1 activates Rac1, inhibits the activation of the NF-κB <span class="highlight" style="background-color:">pathway</span> and initiates protective responses in endothelial cells. See the text for more detail. The figure is adopted from Ref. 39 with modifications.</div></p></div><div class="aux"><div class="resc">Alireza R. Rezaie. IUBMB Life. ;63(6):390-396.</div><p class="links"><a class="dblinks" href="/pubmed/21438119" ref="ordinalpos=1056&amp;ncbi_uid=3501865&amp;link_uid=21438119">Citation</a><a class="dblinks" href="/pmc/articles/PMC3121899/" ref="ordinalpos=1056&amp;ncbi_uid=3501865&amp;link_uid=PMC3121899">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1057.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3821642/figure/f8-ijms-14-20768/" ref="ordinalpos=1057&amp;ncbi_uid=5466706&amp;link_uid=PMC3821642" href="/pmc/articles/PMC3821642/"><img src="/pmc/articles/instance/3821642/bin/ijms-14-20768f8.gif" src-large="/pmc/articles/instance/3821642/bin/ijms-14-20768f8.jpg" alt="Scheme 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3821642/" ref="ordinalpos=1057&amp;ncbi_uid=5466706&amp;link_uid=PMC3821642" image-link="/pmc/articles/PMC3821642/figure/f8-ijms-14-20768/" class="imagepopup">Scheme 1.  From: Crosstalk between Beta-Catenin and Snail in the Induction of Epithelial to Mesenchymal Transition in Hepatocarcinoma: Role of the ERK1/2 <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A simplified summary of the <span class="highlight" style="background-color:">signaling</span> involved in response to TPA and leading to the process of EMT on HepG2 cells. This study highlights the formation of a complex network governed by ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, converging on the coregulation of Snail and the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> and responsible for the onset and the progression of EMT in hepatocellular carcinoma cells.</div></p></div><div class="aux"><div class="resc">Nathalie Zucchini-Pascal, et al. Int J Mol Sci. 2013 October;14(10):20768-20792.</div><p class="links"><a class="dblinks" href="/pubmed/24135872" ref="ordinalpos=1057&amp;ncbi_uid=5466706&amp;link_uid=24135872">Citation</a><a class="dblinks" href="/pmc/articles/PMC3821642/" ref="ordinalpos=1057&amp;ncbi_uid=5466706&amp;link_uid=PMC3821642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1058.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3487108/figure/F7/" ref="ordinalpos=1058&amp;ncbi_uid=4338369&amp;link_uid=PMC3487108" href="/pmc/articles/PMC3487108/"><img src="/pmc/articles/instance/3487108/bin/nihms346588f7.gif" src-large="/pmc/articles/instance/3487108/bin/nihms346588f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3487108/" ref="ordinalpos=1058&amp;ncbi_uid=4338369&amp;link_uid=PMC3487108" image-link="/pmc/articles/PMC3487108/figure/F7/" class="imagepopup">Figure 7. Platelet-Tumor Cell Contact and Platelet-Derived TGFβ1 Synergize to Promote an EMT-Like Transition and Metastasis.  From: Direct <span class="highlight" style="background-color:">Signaling</span> Between Platelets and Cancer Cells Induces an Epithelial-Mesenchymal-Like Transition and Promotes Metastasis. </a></p><div class="supp"><p class="details"><div>Platelets secrete TGFβ1, which activates the TGFβ/Smad <span class="highlight" style="background-color:">pathway</span> in tumor cells. Upon direct platelet-tumor cell contact, the NF-κB <span class="highlight" style="background-color:">pathway</span> is also activated in tumor cells and synergizes with TGFβ/Smad <span class="highlight" style="background-color:">signaling</span> to induce a rapid EMT, enhance invasiveness and promote metastasis. Activation of neither the TGFβ/Smad nor the NF-κB <span class="highlight" style="background-color:">pathway</span> alone is sufficient to promote metastasis. Thus, platelet-tumor cell contact triggers a synergistic interaction between TGFβ/Smad and NF-κB pathways that is necessary for efficient metastasis. The metastatic potential of tumor cells therefore continues to evolve outside of the primary tumor site in response to platelet-to-tumor cell <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Myriam Labelle, et al. Cancer Cell. 2011 November 15;20(5):576-590.</div><p class="links"><a class="dblinks" href="/pubmed/22094253" ref="ordinalpos=1058&amp;ncbi_uid=4338369&amp;link_uid=22094253">Citation</a><a class="dblinks" href="/pmc/articles/PMC3487108/" ref="ordinalpos=1058&amp;ncbi_uid=4338369&amp;link_uid=PMC3487108">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1059.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3212730/figure/F6/" ref="ordinalpos=1059&amp;ncbi_uid=3541551&amp;link_uid=PMC3212730" href="/pmc/articles/PMC3212730/"><img src="/pmc/articles/instance/3212730/bin/nihms331001f6.gif" src-large="/pmc/articles/instance/3212730/bin/nihms331001f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3212730/" ref="ordinalpos=1059&amp;ncbi_uid=3541551&amp;link_uid=PMC3212730" image-link="/pmc/articles/PMC3212730/figure/F6/" class="imagepopup">Figure 6. TGF-β <span class="highlight" style="background-color:">signaling</span> regulates genes bound by master transcription factors.  From: Master Transcription Factors Determine Cell-Type-Specific Responses to TGF-? <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) Schematic of analysis. Genes affected by loss of TGF-β <span class="highlight" style="background-color:">signaling</span> were identified by genome-wide microarray analysis. Next, the overlap of genes affected by TGF-β <span class="highlight" style="background-color:">signaling</span> and genes bound by each transcription factor (TF) were determined. The p-value was calculated using the hypogeometric distribution.<br />(B) Inhibition of the TGF-β <span class="highlight" style="background-color:">pathway</span> affects genes bound by the mES cell core regulatory transcription factors. Genome-wide expression was performed in mES cells that were cultured under standard conditions or in the presence of SB431542 for 24 hr. The association of genes bound by each transcription factor with genes affected by TGF-β <span class="highlight" style="background-color:">signaling</span> (significance of transcription factor occupancy at genes affected by TGF-β <span class="highlight" style="background-color:">signaling</span>, x-axis) was calculated for genes bound by Oct4, Sox2, Nanog, Smad3 and Zfx (y-axis). A factor was considered to bind genes affected by TGF-β <span class="highlight" style="background-color:">signaling</span> at a p value &lt; 1e-5 (gray line).<br />(C) TGF-β <span class="highlight" style="background-color:">signaling</span> leads to recruitment of p300 at genes co-occupied by Oct4 and Smad3. mES cells were treated with SB431542 for 24 hr before being washed and treated with fresh SB431542 or 10 ng/ml Activin for 1 hr. ChIP was performed for p300 and IgG, and qPCR was performed to quantify the fold enrichment of p300 relative to IgG (y-axis) at the indicated genes.<br />(D) TGF-β <span class="highlight" style="background-color:">signaling</span> regulates different genes in different cell types. Genome-wide expression analysis was performed after 24 hr treatment with SB431542 in mES cells and 12 hr after activation of TGF-β <span class="highlight" style="background-color:">signaling</span> in myotubes and pro-B cells. The fold change in expression for each affected gene is indicated by color (bottom) and is shown for mES cells (left), myotubes (center) and pro-B cells (right). All genes that change in only one cell type are shown. Statistical analysis was then performed (as described in A) to determine if there was an association between genes bound by each transcription factor and genes affected by TGF-β <span class="highlight" style="background-color:">signaling</span> for each cell type. Analysis was performed for genes affected by TGF-β <span class="highlight" style="background-color:">signaling</span> in mES cells (top right), myotubes (middle right) and pro-B cells (bottom right). See also Table S4.</div></p></div><div class="aux"><div class="resc">Alan C. Mullen, et al. Cell. ;147(3):565-576.</div><p class="links"><a class="dblinks" href="/pubmed/22036565" ref="ordinalpos=1059&amp;ncbi_uid=3541551&amp;link_uid=22036565">Citation</a><a class="dblinks" href="/pmc/articles/PMC3212730/" ref="ordinalpos=1059&amp;ncbi_uid=3541551&amp;link_uid=PMC3212730">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1060.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3027599/figure/F8/" ref="ordinalpos=1060&amp;ncbi_uid=2728064&amp;link_uid=PMC3027599" href="/pmc/articles/PMC3027599/"><img src="/pmc/articles/instance/3027599/bin/1471-2180-10-326-8.gif" src-large="/pmc/articles/instance/3027599/bin/1471-2180-10-326-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3027599/" ref="ordinalpos=1060&amp;ncbi_uid=2728064&amp;link_uid=PMC3027599" image-link="/pmc/articles/PMC3027599/figure/F8/" class="imagepopup">Figure 8.  From: Eukaryotic <span class="highlight" style="background-color:">signaling</span> pathways targeted by Salmonella effector protein AvrA in intestinal infection in vivo. </a></p><div class="supp"><p class="details"><div><b>Heat maps of <i>Salmonella</i>-responses to gene expression changes involved in four <span class="highlight" style="background-color:">signaling</span> transduction pathways</b>. A: mTOR <span class="highlight" style="background-color:">signaling</span>; B: NF-κB <span class="highlight" style="background-color:">pathway</span> C:SAPK/JNK <span class="highlight" style="background-color:">signaling</span>; D: Oxidative phosphorylation. Red denotes up-regulation; Green denotes down-regulated genes, black denotes unchanged or P-value &gt; 0.05 in three replicate experiments.</div></p></div><div class="aux"><div class="resc">Xingyin Liu, et al. BMC Microbiol. 2010;10:326-326.</div><p class="links"><a class="dblinks" href="/pubmed/21182782" ref="ordinalpos=1060&amp;ncbi_uid=2728064&amp;link_uid=21182782">Citation</a><a class="dblinks" href="/pmc/articles/PMC3027599/" ref="ordinalpos=1060&amp;ncbi_uid=2728064&amp;link_uid=PMC3027599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1061.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3361704/figure/f1/" ref="ordinalpos=1061&amp;ncbi_uid=4025032&amp;link_uid=PMC3361704" href="/pmc/articles/PMC3361704/"><img src="/pmc/articles/instance/3361704/bin/srep00428-f1.gif" src-large="/pmc/articles/instance/3361704/bin/srep00428-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3361704/" ref="ordinalpos=1061&amp;ncbi_uid=4025032&amp;link_uid=PMC3361704" image-link="/pmc/articles/PMC3361704/figure/f1/" class="imagepopup">Figure 1. siRNA on-targeting and off-targeting to genes in a hypothetical <span class="highlight" style="background-color:">pathway</span>..  From: siRNA off-target effects in genome-wide screens identify <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members. </a></p><div class="supp"><p class="details"><div>(a) On-target model correctly infers gene B as a <span class="highlight" style="background-color:">pathway</span> member due to on-target effects, depicted by the solid arrow from the siRNA (blue) to gene B. Extensive base-pairing between the siRNA and target gene B results in silencing. (b) A false-positive result incorrectly infers non-<span class="highlight" style="background-color:">pathway</span> gene C as a <span class="highlight" style="background-color:">pathway</span> member by neglecting off-targeting effects, depicted by dashed gray arrows from the siRNA (red) to <span class="highlight" style="background-color:">pathway</span> genes B and F. (c) Haystack explains screen results as a linear combination of the predicted off-targeting effects, depicted by dashed gray arrows from the siRNA (red) to <span class="highlight" style="background-color:">pathway</span> genes B and F. Imperfect base-pairing between siRNA (red) and 3′UTR region of off-target genes results in down-regulation.</div></p></div><div class="aux"><div class="resc">Eugen Buehler, et al. Sci Rep. 2012;2:428.</div><p class="links"><a class="dblinks" href="/pubmed/22645644" ref="ordinalpos=1061&amp;ncbi_uid=4025032&amp;link_uid=22645644">Citation</a><a class="dblinks" href="/pmc/articles/PMC3361704/" ref="ordinalpos=1061&amp;ncbi_uid=4025032&amp;link_uid=PMC3361704">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1062.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4323088/figure/F1/" ref="ordinalpos=1062&amp;ncbi_uid=7164390&amp;link_uid=PMC4323088" href="/pmc/articles/PMC4323088/"><img src="/pmc/articles/instance/4323088/bin/nihms641472f1.gif" src-large="/pmc/articles/instance/4323088/bin/nihms641472f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4323088/" ref="ordinalpos=1062&amp;ncbi_uid=7164390&amp;link_uid=PMC4323088" image-link="/pmc/articles/PMC4323088/figure/F1/" class="imagepopup">Figure 1.  From: Fibroblast growth factor (FGF) <span class="highlight" style="background-color:">signaling</span> in development and skeletal diseases. </a></p><div class="supp"><p class="details"><div>FGF-FGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">signaling</span> cascade commences upon the formation of an FGF binding complex, consisting of two FGF ligands, two heparin sulfate chains, and two FGFRs. Signal transduction largely follows one of three pathways. The RAS/MAP kinase <span class="highlight" style="background-color:">pathway</span>, initiated upon the formation of an FRS2 complex, controls cell proliferation and differentiation. The PI3/AKT <span class="highlight" style="background-color:">pathway</span> is also initiated by the formation of an FRS2 complex, and regulates cell survival and fate determination. Finally, upon binding of PLCγ to the activated FGFR, DAG and IP3 are formed, activating PKC. The PLCγ <span class="highlight" style="background-color:">pathway</span> influences cell morphology, migration, and adhesion.</div></p></div><div class="aux"><div class="resc">Chad M. Teven, et al. Genes Dis. ;1(2):199-213.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1062&amp;ncbi_uid=7164390&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4323088/" ref="ordinalpos=1062&amp;ncbi_uid=7164390&amp;link_uid=PMC4323088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1063.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3933272/figure/F4/" ref="ordinalpos=1063&amp;ncbi_uid=5898792&amp;link_uid=PMC3933272" href="/pmc/articles/PMC3933272/"><img src="/pmc/articles/instance/3933272/bin/1471-2164-14-592-4.gif" src-large="/pmc/articles/instance/3933272/bin/1471-2164-14-592-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3933272/" ref="ordinalpos=1063&amp;ncbi_uid=5898792&amp;link_uid=PMC3933272" image-link="/pmc/articles/PMC3933272/figure/F4/" class="imagepopup">Figure 4.  From: BayMiR: inferring evidence for endogenous miRNA-induced gene repression from mRNA expression profiles. </a></p><div class="supp"><p class="details"><div><b>WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: 32 targets of 5 miRNAs are involved in the <span class="highlight" style="background-color:">pathway</span> (red boxes). 14 mRNAs are targeted by the remaining miRNAs are colored in yellow; and 23 mRNAs involved in the <span class="highlight" style="background-color:">pathway</span> were excluded from the BayMiR target list since their expression variabilities across arrays were very low (white boxes). The miRNA family IDs: miR-518a-5p/520d-5p/524-5p,miR-556-3p,miR-4514/4692,miR-548aeajamx,miR-135ab/135a-5p;.</div></p></div><div class="aux"><div class="resc">Hossein Radfar, et al. BMC Genomics. 2013;14:592-592.</div><p class="links"><a class="dblinks" href="/pubmed/24001276" ref="ordinalpos=1063&amp;ncbi_uid=5898792&amp;link_uid=24001276">Citation</a><a class="dblinks" href="/pmc/articles/PMC3933272/" ref="ordinalpos=1063&amp;ncbi_uid=5898792&amp;link_uid=PMC3933272">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1064.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3877118/figure/pone-0083941-g008/" ref="ordinalpos=1064&amp;ncbi_uid=5676546&amp;link_uid=PMC3877118" href="/pmc/articles/PMC3877118/"><img src="/pmc/articles/instance/3877118/bin/pone.0083941.g008.gif" src-large="/pmc/articles/instance/3877118/bin/pone.0083941.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3877118/" ref="ordinalpos=1064&amp;ncbi_uid=5676546&amp;link_uid=PMC3877118" image-link="/pmc/articles/PMC3877118/figure/pone-0083941-g008/" class="imagepopup">Figure 8. Proposed Model of the SynGAP-Mediated Cellular <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> that Regulates Translation and Synaptic AMPAR Content..  From: SynGAP Regulates Protein Synthesis and Homeostatic Synaptic Plasticity in Developing Cortical Networks. </a></p><div class="supp"><p class="details"><div>Our data predict the following model: under conditions of basal neuronal activity, association of GluN2B and CaMKII links SynGAP to suppression of ERK <span class="highlight" style="background-color:">signaling</span>. ERK can activate Rheb, which, in turn, activates mTOR. Through suppression of mTOR function this mechanism inhibits protein synthesis and limits AMPAR contribution at developing cortical networks. Our data show that this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is critical for developmental synaptic plasticity as translation-dependent homeostatic synaptic plasticity is disrupted in the absence of GluN2B or SynGAP.</div></p></div><div class="aux"><div class="resc">Chih-Chieh Wang, et al. PLoS One. 2013;8(12):e83941.</div><p class="links"><a class="dblinks" href="/pubmed/24391850" ref="ordinalpos=1064&amp;ncbi_uid=5676546&amp;link_uid=24391850">Citation</a><a class="dblinks" href="/pmc/articles/PMC3877118/" ref="ordinalpos=1064&amp;ncbi_uid=5676546&amp;link_uid=PMC3877118">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1065.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3206195/figure/F3/" ref="ordinalpos=1065&amp;ncbi_uid=3559988&amp;link_uid=PMC3206195" href="/pmc/articles/PMC3206195/"><img src="/pmc/articles/instance/3206195/bin/nihms307754f3.gif" src-large="/pmc/articles/instance/3206195/bin/nihms307754f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3206195/" ref="ordinalpos=1065&amp;ncbi_uid=3559988&amp;link_uid=PMC3206195" image-link="/pmc/articles/PMC3206195/figure/F3/" class="imagepopup">Figure 3. BDNF – TrkB induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Fear Conditioning, Synaptic Plasticity, and the Amygdala: Implications for Posttraumatic Stress Disorder. </a></p><div class="supp"><p class="details"><div>BDNF binds to the TrkB receptor, resulting in the phosphorylation of two tyrosine sites (Y515 and Y816) on the intracellular domain of the TrkB receptor. Phosphorylation of the Y515 residue allows recruitment of Shc/FRS-2, which subsequently activates the Ras/MAPK and PI3K pathways. In contrast, phosphorylation of the Y816 residue allows recruitment of PLC, which activates the CAMK/CREB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Point mutations of the Y515 residue produce deficits in consolidation but not acquisition of fear conditioning <sup>72</sup>. Point mutations of the Y816 residue, on the other hand, produce deficits in acquisition <sup>72</sup>. Evidence exists for a role of BDNF <span class="highlight" style="background-color:">signaling</span> in the amygdala, hippocampus and PFC with respect to both the consolidation and extinction of fear conditioning.</div></p></div><div class="aux"><div class="resc">Amy L. Mahan, et al. Trends Neurosci. ;35(1):24-35.</div><p class="links"><a class="dblinks" href="/pubmed/21798604" ref="ordinalpos=1065&amp;ncbi_uid=3559988&amp;link_uid=21798604">Citation</a><a class="dblinks" href="/pmc/articles/PMC3206195/" ref="ordinalpos=1065&amp;ncbi_uid=3559988&amp;link_uid=PMC3206195">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1066.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2924313/figure/F5/" ref="ordinalpos=1066&amp;ncbi_uid=2424602&amp;link_uid=PMC2924313" href="/pmc/articles/PMC2924313/"><img src="/pmc/articles/instance/2924313/bin/1476-4598-9-212-5.gif" src-large="/pmc/articles/instance/2924313/bin/1476-4598-9-212-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2924313/" ref="ordinalpos=1066&amp;ncbi_uid=2424602&amp;link_uid=PMC2924313" image-link="/pmc/articles/PMC2924313/figure/F5/" class="imagepopup">Figure 5.  From: The Wnt/?-catenin <span class="highlight" style="background-color:">pathway</span> regulates growth and maintenance of colonospheres. </a></p><div class="supp"><p class="details"><div><b>Activation of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in spheroids/colonospheres</b>. <b>A</b>. Representative photomicrographs of HT-29 parental and colonospheres-derived cells depicting the membrane localization of β-catenin by immunofluorescence staining. <b>B</b>. The comparative expression of different members of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in parental cells (referred to as 'P') and colonospheres-derived cells (referred to as 'S'). The Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> is constitutively activated in colonosphere-derived cells, but not in parental cells as shown by the expression of various proteins: β-catenin, phosphorylated β-catenin, axin-1, phosphorylated GSK-3-β along with important downstream targets: c-myc and cyclin-D1. The numbers represent percent of corresponding control normalized to β-actin.</div></p></div><div class="aux"><div class="resc">Shailender S Kanwar, et al. Mol Cancer. 2010;9:212-212.</div><p class="links"><a class="dblinks" href="/pubmed/20691072" ref="ordinalpos=1066&amp;ncbi_uid=2424602&amp;link_uid=20691072">Citation</a><a class="dblinks" href="/pmc/articles/PMC2924313/" ref="ordinalpos=1066&amp;ncbi_uid=2424602&amp;link_uid=PMC2924313">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1067.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3862569/figure/pone-0081938-g006/" ref="ordinalpos=1067&amp;ncbi_uid=5609095&amp;link_uid=PMC3862569" href="/pmc/articles/PMC3862569/"><img src="/pmc/articles/instance/3862569/bin/pone.0081938.g006.gif" src-large="/pmc/articles/instance/3862569/bin/pone.0081938.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3862569/" ref="ordinalpos=1067&amp;ncbi_uid=5609095&amp;link_uid=PMC3862569" image-link="/pmc/articles/PMC3862569/figure/pone-0081938-g006/" class="imagepopup">Figure 6. Parallel comparison of BR biosynthetic and <span class="highlight" style="background-color:">signaling</span> pathways..  From: Brassinosteroids Regulate Plant Growth through Distinct <span class="highlight" style="background-color:">Signaling</span> Pathways in Selaginella and Arabidopsis. </a></p><div class="supp"><p class="details"><div>(A) <span class="highlight" style="background-color:">Pathway</span> showing the order in which Arabidopsis BR biosynthetic CYPs function. Selaginella CYP90s are shown to the right. (B) Model comparing Arabidopsis (left) with Selaginella (right) <span class="highlight" style="background-color:">signaling</span> pathways. Abbreviations are given beneath Table 2. P indicates phosphorylation. Solid lines indicate established <span class="highlight" style="background-color:">signaling</span> routes, whereas broken lines show possible routes. Flat-tipped and pointed arrows indicate inhibitory and positive actions, respectively. Question marks indicate steps that remain to be characterized.</div></p></div><div class="aux"><div class="resc">Jinyeong Cheon, et al. PLoS One. 2013;8(12):e81938.</div><p class="links"><a class="dblinks" href="/pubmed/24349155" ref="ordinalpos=1067&amp;ncbi_uid=5609095&amp;link_uid=24349155">Citation</a><a class="dblinks" href="/pmc/articles/PMC3862569/" ref="ordinalpos=1067&amp;ncbi_uid=5609095&amp;link_uid=PMC3862569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1068.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678388/figure/F1/" ref="ordinalpos=1068&amp;ncbi_uid=4983343&amp;link_uid=PMC3678388" href="/pmc/articles/PMC3678388/"><img src="/pmc/articles/instance/3678388/bin/nihms362811f1.gif" src-large="/pmc/articles/instance/3678388/bin/nihms362811f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678388/" ref="ordinalpos=1068&amp;ncbi_uid=4983343&amp;link_uid=PMC3678388" image-link="/pmc/articles/PMC3678388/figure/F1/" class="imagepopup">Figure 1.  From: ATR <span class="highlight" style="background-color:">signaling</span>: more than meeting at the fork. </a></p><div class="supp"><p class="details"><div>Canonical ATR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and functions of newly identified regulators.<br />(A) ATR <span class="highlight" style="background-color:">signaling</span> is activated in response to single-stranded DNA gaps in the genome. Independent recruitment of several checkpoint proteins leads to TOPBP1-dependent activation of the kinase and phosphorylation of numerous substrates including CHK1 to regulate cellular responses to DNA damage and replication stress. (B) The table lists newly identified regulators of this <span class="highlight" style="background-color:">pathway</span> and, when known, describes their position in the <span class="highlight" style="background-color:">pathway</span>. See text for details.</div></p></div><div class="aux"><div class="resc">Edward A. Nam, et al. Biochem J. ;436(3):527-536.</div><p class="links"><a class="dblinks" href="/pubmed/21615334" ref="ordinalpos=1068&amp;ncbi_uid=4983343&amp;link_uid=21615334">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678388/" ref="ordinalpos=1068&amp;ncbi_uid=4983343&amp;link_uid=PMC3678388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1069.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1291380/figure/F5/" ref="ordinalpos=1069&amp;ncbi_uid=618289&amp;link_uid=PMC1291380" href="/pmc/articles/PMC1291380/"><img src="/pmc/articles/instance/1291380/bin/1478-811X-3-12-5.gif" src-large="/pmc/articles/instance/1291380/bin/1478-811X-3-12-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1291380/" ref="ordinalpos=1069&amp;ncbi_uid=618289&amp;link_uid=PMC1291380" image-link="/pmc/articles/PMC1291380/figure/F5/" class="imagepopup">Figure 5.  From: Xenopus frizzled-4S, a splicing variant of Xfz4 is a context-dependent activator and inhibitor of Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Model for the modulation of Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> by extracellular domains of Frizzled receptors. We propose that a complex formed between Xfz4S and Wnt-3a/-8/-8b could be recognized by LRP and Wnt <span class="highlight" style="background-color:">signaling</span> could be activated in LRP dependent manner. Complexes between Xfz4S and Wnt-4/-5a/-11 would not be recognized by LRP and Xfz4S and Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> would not be activated by these Wnts.</div></p></div><div class="aux"><div class="resc">Rajeeb Kumar Swain, et al. Cell Commun Signal. 2005;3:12-12.</div><p class="links"><a class="dblinks" href="/pubmed/16236168" ref="ordinalpos=1069&amp;ncbi_uid=618289&amp;link_uid=16236168">Citation</a><a class="dblinks" href="/pmc/articles/PMC1291380/" ref="ordinalpos=1069&amp;ncbi_uid=618289&amp;link_uid=PMC1291380">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1070.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3590201/figure/pone-0057022-g003/" ref="ordinalpos=1070&amp;ncbi_uid=4696888&amp;link_uid=PMC3590201" href="/pmc/articles/PMC3590201/"><img src="/pmc/articles/instance/3590201/bin/pone.0057022.g003.gif" src-large="/pmc/articles/instance/3590201/bin/pone.0057022.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3590201/" ref="ordinalpos=1070&amp;ncbi_uid=4696888&amp;link_uid=PMC3590201" image-link="/pmc/articles/PMC3590201/figure/pone-0057022-g003/" class="imagepopup">Figure 3. KEGG <span class="highlight" style="background-color:">pathway</span> for calcium <span class="highlight" style="background-color:">signaling</span>..  From: The Identification of <span class="highlight" style="background-color:">Pathway</span> Markers in Intracranial Aneurysm Using Genome-Wide Association Data from Two Different Populations. </a></p><div class="supp"><p class="details"><div>The set of genes shown in blue includes genes that are found by EU GWAS; yellow includes genes that are found by JP GWAS; red includes genes that are found both by EU and JP GWAS.</div></p></div><div class="aux"><div class="resc">Burcu Bakir-Gungor, et al. PLoS One. 2013;8(3):e57022.</div><p class="links"><a class="dblinks" href="/pubmed/23483893" ref="ordinalpos=1070&amp;ncbi_uid=4696888&amp;link_uid=23483893">Citation</a><a class="dblinks" href="/pmc/articles/PMC3590201/" ref="ordinalpos=1070&amp;ncbi_uid=4696888&amp;link_uid=PMC3590201">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1071.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2504419/figure/F8/" ref="ordinalpos=1071&amp;ncbi_uid=1457283&amp;link_uid=PMC2504419" href="/pmc/articles/PMC2504419/"><img src="/pmc/articles/instance/2504419/bin/nihms56601f8.gif" src-large="/pmc/articles/instance/2504419/bin/nihms56601f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2504419/" ref="ordinalpos=1071&amp;ncbi_uid=1457283&amp;link_uid=PMC2504419" image-link="/pmc/articles/PMC2504419/figure/F8/" class="imagepopup">Fig. 8. Stimulation of splenic T cells activates transcription of the Ifi202 gene through the JNK/c-Jun <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Stimulation of T cells up-regulates expression of Ifi202, an interferon-inducible lupus susceptibility gene, through activation of JNK/c-Jun <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="aux"><div class="resc">Jianming Chen, et al. Immunol Lett. ;118(1):13-20.</div><p class="links"><a class="dblinks" href="/pubmed/18374989" ref="ordinalpos=1071&amp;ncbi_uid=1457283&amp;link_uid=18374989">Citation</a><a class="dblinks" href="/pmc/articles/PMC2504419/" ref="ordinalpos=1071&amp;ncbi_uid=1457283&amp;link_uid=PMC2504419">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1072.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3809843/figure/F9/" ref="ordinalpos=1072&amp;ncbi_uid=6441984&amp;link_uid=PMC3809843" href="/pmc/articles/PMC3809843/"><img src="/pmc/articles/instance/3809843/bin/nihms491332f9.gif" src-large="/pmc/articles/instance/3809843/bin/nihms491332f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3809843/" ref="ordinalpos=1072&amp;ncbi_uid=6441984&amp;link_uid=PMC3809843" image-link="/pmc/articles/PMC3809843/figure/F9/" class="imagepopup">Fig. 9.  From: Gremlin utilizes canonical and non-canonical TGF? <span class="highlight" style="background-color:">signaling</span> to induce lysyl oxidase (LOX) genes in human trabecular meshwork cells. </a></p><div class="supp"><p class="details"><div>Proposed mechanism of gremlin regulation of LOXs in TM. Gremlin blocks BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is unable to inhibit the endogenous TGF<sub>β</sub> <span class="highlight" style="background-color:">signaling</span> in TM cells. The endogenously active TGF<sub>β</sub> <span class="highlight" style="background-color:">pathway</span> can activate Smad, P38 and JNK pathways to induce different LOX genes and proteins in TM cells.</div></p></div><div class="aux"><div class="resc">Anirudh Sethi, et al. Exp Eye Res. 2013 August;113:10.1016/j.exer.2013.05.011.</div><p class="links"><a class="dblinks" href="/pubmed/23748100" ref="ordinalpos=1072&amp;ncbi_uid=6441984&amp;link_uid=23748100">Citation</a><a class="dblinks" href="/pmc/articles/PMC3809843/" ref="ordinalpos=1072&amp;ncbi_uid=6441984&amp;link_uid=PMC3809843">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1073.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3598224/figure/F4/" ref="ordinalpos=1073&amp;ncbi_uid=4718120&amp;link_uid=PMC3598224" href="/pmc/articles/PMC3598224/"><img src="/pmc/articles/instance/3598224/bin/1475-2891-12-24-4.gif" src-large="/pmc/articles/instance/3598224/bin/1475-2891-12-24-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3598224/" ref="ordinalpos=1073&amp;ncbi_uid=4718120&amp;link_uid=PMC3598224" image-link="/pmc/articles/PMC3598224/figure/F4/" class="imagepopup">Figure 4.  From: Associations between dietary patterns and gene expression profiles of healthy men and women: a cross-sectional study. </a></p><div class="supp"><p class="details"><div><b>The modified canonical pathways according to scores for the Western dietary pattern. Legend:</b> Gene expression differences (≥ ± 1.2 fold change) in canonical pathways comparing high and low scores for the Western dietary pattern <b>(A)</b> For the men <b>(B)</b> For the women. P-values for Functional Analysis of dataset by IPA (Fisher’s Exact Test) are presented. % of genes significantly up-regulated and down-regulated in each canonical <span class="highlight" style="background-color:">pathway</span> are presented (number of genes differently expressed/number of genes in the pathways*100). <b>(A)</b> Endothelin-1 <span class="highlight" style="background-color:">Signaling</span>: P = 0.01, 12 genes significantly different; PTEN <span class="highlight" style="background-color:">Signaling</span>: P = 0.01, 10 genes significantly different; iCOS-iCOSL <span class="highlight" style="background-color:">Signaling</span> in T Helper Cells: P = 0.008, 9 genes significantly different; Hypoxia <span class="highlight" style="background-color:">Signaling</span> in the Cardiovascular System: P = 0.006, 7 genes significantly different; Xenobiotic Metabolism <span class="highlight" style="background-color:">Signaling</span>: P = 0.005, 18 genes significantly different; NGF <span class="highlight" style="background-color:">Signaling</span>: P = 0.004, 10 genes significantly different; Neurotrophin/TRK <span class="highlight" style="background-color:">Signaling</span>: P = 0.00001, 11 genes significantly different. <b>(B)</b> IL-8 <span class="highlight" style="background-color:">Signaling</span>: P = 0.04, 15 genes significantly different; FXR/RXR Activation: P = 0.05, 9 genes significantly different; Atherosclerosis <span class="highlight" style="background-color:">Signaling</span>: P = 0.02, 12 genes significantly different; Role of Cytokines in Mediating Communication between Immune Cells: P = 0.02, 7 genes significantly different; Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>: P = 0.01, 7 genes significantly different; Role of NFAT in Regulation of the Immune Response: P = 0.01, 16 genes significantly different; Bladder Cancer <span class="highlight" style="background-color:">Signaling</span>: P = 0.002, 12 genes significantly different; Nf-κB <span class="highlight" style="background-color:">Signaling</span>: P = 0.002, 18 genes significantly different.</div></p></div><div class="aux"><div class="resc">Annie Bouchard-Mercier, et al. Nutr J. 2013;12:24-24.</div><p class="links"><a class="dblinks" href="/pubmed/23398686" ref="ordinalpos=1073&amp;ncbi_uid=4718120&amp;link_uid=23398686">Citation</a><a class="dblinks" href="/pmc/articles/PMC3598224/" ref="ordinalpos=1073&amp;ncbi_uid=4718120&amp;link_uid=PMC3598224">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1074.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2884124/figure/F1/" ref="ordinalpos=1074&amp;ncbi_uid=2297774&amp;link_uid=PMC2884124" href="/pmc/articles/PMC2884124/"><img src="/pmc/articles/instance/2884124/bin/psb0502_0160_fig001.gif" src-large="/pmc/articles/instance/2884124/bin/psb0502_0160_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2884124/" ref="ordinalpos=1074&amp;ncbi_uid=2297774&amp;link_uid=PMC2884124" image-link="/pmc/articles/PMC2884124/figure/F1/" class="imagepopup">Figure 1.  From: The PP2C-SnRK2 complex. </a></p><div class="supp"><p class="details"><div>Schematic representation of the major ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Under normal conditions, clade A PP2C constantly interacts with ABA-activated SnRK2 and inactivates its kinase activity by dephosphorylation. When the soluble ABA receptor recognizes the ABA molecule, it binds PP2C and inhibits its phosphatase activity. Then SnRK2 is activated and phosphorylates its targets, and the ABA signal is consequently turned on. In this ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the signal is gathered by 14 ABA receptors and transduced and integrated through nine PP2Cs and three SnRK2s. Presumably, in these steps, the signal information is processed by different interactions between components for correct cellular responses.</div></p></div><div class="aux"><div class="resc">Takashi Hirayama, et al. Plant Signal Behav. 2010 February;5(2):160-163.</div><p class="links"><a class="dblinks" href="/pubmed/20023393" ref="ordinalpos=1074&amp;ncbi_uid=2297774&amp;link_uid=20023393">Citation</a><a class="dblinks" href="/pmc/articles/PMC2884124/" ref="ordinalpos=1074&amp;ncbi_uid=2297774&amp;link_uid=PMC2884124">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1075.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3539509/figure/f2-medscimonit-17-4-ra100/" ref="ordinalpos=1075&amp;ncbi_uid=4824529&amp;link_uid=PMC3539509" href="/pmc/articles/PMC3539509/"><img src="/pmc/articles/instance/3539509/bin/medscimonit-17-4-RA100-g002.gif" src-large="/pmc/articles/instance/3539509/bin/medscimonit-17-4-RA100-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3539509/" ref="ordinalpos=1075&amp;ncbi_uid=4824529&amp;link_uid=PMC3539509" image-link="/pmc/articles/PMC3539509/figure/f2-medscimonit-17-4-ra100/" class="imagepopup">Figure 2.  From: Toll-like receptor and its roles in myocardial ischemic/reperfusion injury. </a></p><div class="supp"><p class="details"><div>TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: TLR4 is localized on cell surface for ligand recognition and activates both MyD88-dependent <span class="highlight" style="background-color:">pathway</span> and TRIF-dependent <span class="highlight" style="background-color:">pathway</span>. In MyD88-dependent <span class="highlight" style="background-color:">pathway</span>, TLR4 recruits MAL to link TIR domain of TLR4 with MyD88. MyD88 recruits the IRAK family of proteins and TRAF6. TRAF6 activates TAK1. The activated TAK1 not only activates the IKK complex activating NF-κB, also activates the MAPKs. In TRIF-dependent <span class="highlight" style="background-color:">pathway</span>, TLR4 recruits. TRAM to link TIR domain of TLR4 with TRIF. TRIF interacts with RIP1 and TRAF6, which could activate NF-κB and MAPKs; TRIF interacts with TRAF3 and activates TRAF3/IKKɛ which activates IRF3 and IRF7.</div></p></div><div class="aux"><div class="resc">Yu Fang, et al. Med Sci Monit. 2011;17(4):RA100-RA109.</div><p class="links"><a class="dblinks" href="/pubmed/21455117" ref="ordinalpos=1075&amp;ncbi_uid=4824529&amp;link_uid=21455117">Citation</a><a class="dblinks" href="/pmc/articles/PMC3539509/" ref="ordinalpos=1075&amp;ncbi_uid=4824529&amp;link_uid=PMC3539509">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1076.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2790753/figure/F1/" ref="ordinalpos=1076&amp;ncbi_uid=2233740&amp;link_uid=PMC2790753" href="/pmc/articles/PMC2790753/"><img src="/pmc/articles/instance/2790753/bin/jnn0001-0240-f01.gif" src-large="/pmc/articles/instance/2790753/bin/jnn0001-0240-f01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2790753/" ref="ordinalpos=1076&amp;ncbi_uid=2233740&amp;link_uid=PMC2790753" image-link="/pmc/articles/PMC2790753/figure/F1/" class="imagepopup">Fig. 1. From: Effect of Dietary Calcium and Dairy Proteins on the Adipose Tissue Gene Expression Profile in Diet-Induced Obesity. </a></p><div class="supp"><p class="details"><div>Microarray data for the genes in the <span class="highlight" style="background-color:">pathway</span> coined from the central genes in the KEGG pathways ‘insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>’ and ‘adipocytokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>’. The figure is modified from the <span class="highlight" style="background-color:">pathway</span> presented in KEGG [51] . Genes with no microarray data are shown in italics. Abstractions are presented with rounded grey shapes. ∗Genes with a ±1.2-fold change in between groups.</div></p></div><div class="aux"><div class="resc">Taru K. Pilvi, et al. J Nutrigenet Nutrigenomics. 2008 August;1(5):240-251.</div><p class="links"><a class="dblinks" href="/pubmed/19776631" ref="ordinalpos=1076&amp;ncbi_uid=2233740&amp;link_uid=19776631">Citation</a><a class="dblinks" href="/pmc/articles/PMC2790753/" ref="ordinalpos=1076&amp;ncbi_uid=2233740&amp;link_uid=PMC2790753">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1077.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2677440/figure/fig7/" ref="ordinalpos=1077&amp;ncbi_uid=1813496&amp;link_uid=PMC2677440" href="/pmc/articles/PMC2677440/"><img src="/pmc/articles/instance/2677440/bin/AJRCMB405610f7.gif" src-large="/pmc/articles/instance/2677440/bin/AJRCMB405610f7.jpg" alt="&lt;b&gt;Figure 7.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2677440/" ref="ordinalpos=1077&amp;ncbi_uid=1813496&amp;link_uid=PMC2677440" image-link="/pmc/articles/PMC2677440/figure/fig7/" class="imagepopup">Figure 7..  From: Rhinovirus-Induced Major Airway Mucin Production Involves a Novel TLR3-EGFR-Dependent <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> links RV infection and mucin expression. Question mark indicates that there may be other TRIF-independent pathways.</div></p></div><div class="aux"><div class="resc">Lingxiang Zhu, et al. Am J Respir Cell Mol Biol. 2009 May;40(5):610-619.</div><p class="links"><a class="dblinks" href="/pubmed/18978302" ref="ordinalpos=1077&amp;ncbi_uid=1813496&amp;link_uid=18978302">Citation</a><a class="dblinks" href="/pmc/articles/PMC2677440/" ref="ordinalpos=1077&amp;ncbi_uid=1813496&amp;link_uid=PMC2677440">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1078.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2275876/figure/Fig8/" ref="ordinalpos=1078&amp;ncbi_uid=1369689&amp;link_uid=PMC2275876" href="/pmc/articles/PMC2275876/"><img src="/pmc/articles/instance/2275876/bin/12079_2007_11_Fig8_HTML.gif" src-large="/pmc/articles/instance/2275876/bin/12079_2007_11_Fig8_HTML.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2275876/" ref="ordinalpos=1078&amp;ncbi_uid=1369689&amp;link_uid=PMC2275876" image-link="/pmc/articles/PMC2275876/figure/Fig8/" class="imagepopup">Fig. 8.  From: Role of the PI3-kinase/mTor <span class="highlight" style="background-color:">pathway</span> in the regulation of the stearoyl CoA desaturase (SCD1) gene expression by insulin in liver. </a></p><div class="supp"><p class="details"><div>Schematic representation of the regulation of SCD1 gene transcription in hepatic cells. Schematic representation of the regulation of SCD1 gene transcription in hepatic cells. The top part of the figure represents the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> implicated in the insulin regulation of SCD1 gene transcription. The <i>large grey arrow</i> represents the promoter where the different 5′-deletion constructs are indicated. The lower part represents the transcription factors implicated in the regulation of SCD1 gene transcription in liver in response to activation of the indicated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Daniel Mauvoisin, et al. J Cell Commun Signal. 2007 September;1(2):113-125.</div><p class="links"><a class="dblinks" href="/pubmed/18481202" ref="ordinalpos=1078&amp;ncbi_uid=1369689&amp;link_uid=18481202">Citation</a><a class="dblinks" href="/pmc/articles/PMC2275876/" ref="ordinalpos=1078&amp;ncbi_uid=1369689&amp;link_uid=PMC2275876">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1079.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3953715/figure/F1/" ref="ordinalpos=1079&amp;ncbi_uid=5946776&amp;link_uid=PMC3953715" href="/pmc/articles/PMC3953715/"><img src="/pmc/articles/instance/3953715/bin/fnmol-07-00018-g001.gif" src-large="/pmc/articles/instance/3953715/bin/fnmol-07-00018-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3953715/" ref="ordinalpos=1079&amp;ncbi_uid=5946776&amp;link_uid=PMC3953715" image-link="/pmc/articles/PMC3953715/figure/F1/" class="imagepopup">FIGURE 1.  From: Mechanisms regulating neuronal excitability and seizure development following mTOR <span class="highlight" style="background-color:">pathway</span> hyperactivation. </a></p><div class="supp"><p class="details"><div><b>PI3K/PTEN–mTOR <span class="highlight" style="background-color:">pathway</span>.</b> mTOR is activated by <span class="highlight" style="background-color:">signaling</span> through the PI3K–Akt <span class="highlight" style="background-color:">pathway</span>, regulating cell growth and proliferation. PTEN, TSC1, and TSC2 act as negative regulators of the mTOR <span class="highlight" style="background-color:">pathway</span>, and removal leads to hyperactivation of mTOR. mTOR complex 1 (mTORC1) <span class="highlight" style="background-color:">signaling</span> requires activation of the adaptor protein raptor, and mTOR complex 2 (mTORC2), which is largely insensitive to acute rapamycin treatment, requires activation of the rictor protein. Although regulation of mTORC2 is less clearly defined, it has been linked to cytoskeleton organization.</div></p></div><div class="aux"><div class="resc">Candi L. LaSarge, et al. Front Mol Neurosci. 2014;7:18.</div><p class="links"><a class="dblinks" href="/pubmed/24672426" ref="ordinalpos=1079&amp;ncbi_uid=5946776&amp;link_uid=24672426">Citation</a><a class="dblinks" href="/pmc/articles/PMC3953715/" ref="ordinalpos=1079&amp;ncbi_uid=5946776&amp;link_uid=PMC3953715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1080.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3069071/figure/pone-0018330-g007/" ref="ordinalpos=1080&amp;ncbi_uid=2883469&amp;link_uid=PMC3069071" href="/pmc/articles/PMC3069071/"><img src="/pmc/articles/instance/3069071/bin/pone.0018330.g007.gif" src-large="/pmc/articles/instance/3069071/bin/pone.0018330.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3069071/" ref="ordinalpos=1080&amp;ncbi_uid=2883469&amp;link_uid=PMC3069071" image-link="/pmc/articles/PMC3069071/figure/pone-0018330-g007/" class="imagepopup">Figure 7. SMO inhibition sensitizes cells to TRAIL-induced apoptosis by increasing DR4 and decreasing XIAP protein levels, thereby switching cells from Type II to Type I death receptor <span class="highlight" style="background-color:">signaling</span>..  From: Hedgehog Inhibition Promotes a Switch from Type II to Type I Cell Death Receptor <span class="highlight" style="background-color:">Signaling</span> in Cancer Cells. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the interaction of Hedgehog and TRAIL <span class="highlight" style="background-color:">signaling</span> in cholangiocarcinoma cells. Death <span class="highlight" style="background-color:">signaling</span> upon TRAIL binding to DR4 normally is dependent on mitochondrial dysfunction (induced by Bid/Bim as well as Bak/Bax activation) to induce cholangiocarcinoma cell death (Type II <span class="highlight" style="background-color:">pathway</span>), a <span class="highlight" style="background-color:">pathway</span> blocked by elevated Mcl-1 protein levels in cholangiocarcinoma cells. Alternately, following DR4 ligation, robust caspase 3 activation can induce Type I death <span class="highlight" style="background-color:">signaling</span>, a <span class="highlight" style="background-color:">pathway</span> inhibited by XIAP expression under normal conditions. Inhibition of SMO expression or activity sensitizes cells to TRAIL via increasing DR4 [19] and decreasing XIAP levels, thereby switching cells from Type II to Type I death receptor <span class="highlight" style="background-color:">signaling</span> and promoting apoptosis.</div></p></div><div class="aux"><div class="resc">Satoshi Kurita, et al. PLoS One. 2011;6(3):e18330.</div><p class="links"><a class="dblinks" href="/pubmed/21483830" ref="ordinalpos=1080&amp;ncbi_uid=2883469&amp;link_uid=21483830">Citation</a><a class="dblinks" href="/pmc/articles/PMC3069071/" ref="ordinalpos=1080&amp;ncbi_uid=2883469&amp;link_uid=PMC3069071">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1081.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3763569/figure/fig6/" ref="ordinalpos=1081&amp;ncbi_uid=5288103&amp;link_uid=PMC3763569" href="/pmc/articles/PMC3763569/"><img src="/pmc/articles/instance/3763569/bin/ECAM2013-423164.006.gif" src-large="/pmc/articles/instance/3763569/bin/ECAM2013-423164.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3763569/" ref="ordinalpos=1081&amp;ncbi_uid=5288103&amp;link_uid=PMC3763569" image-link="/pmc/articles/PMC3763569/figure/fig6/" class="imagepopup">Figure 6.  From: Berberine Reduces the Metastasis of Chondrosarcoma by Modulating the αvβ3 Integrin and the PKCδ, c-Src, and AP-1 <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Schematic presentation of the <span class="highlight" style="background-color:">signaling</span> pathways involved in berberine-inhibited metastasis of human chondrosarcoma. Berberine inhibits the migration and invasion of human chondrosarcoma cells by modulating the <i>α</i>v<i>β</i>3 integrin through PKC<i>δ</i>, c-Src, and AP-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chi-Ming Wu, et al. Evid Based Complement Alternat Med. 2013;2013:423164.</div><p class="links"><a class="dblinks" href="/pubmed/24027594" ref="ordinalpos=1081&amp;ncbi_uid=5288103&amp;link_uid=24027594">Citation</a><a class="dblinks" href="/pmc/articles/PMC3763569/" ref="ordinalpos=1081&amp;ncbi_uid=5288103&amp;link_uid=PMC3763569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1082.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2626396/figure/F6-7812/" ref="ordinalpos=1082&amp;ncbi_uid=1622258&amp;link_uid=PMC2626396" href="/pmc/articles/PMC2626396/"><img src="/pmc/articles/instance/2626396/bin/zjh0120878120006.gif" src-large="/pmc/articles/instance/2626396/bin/zjh0120878120006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2626396/" ref="ordinalpos=1082&amp;ncbi_uid=1622258&amp;link_uid=PMC2626396" image-link="/pmc/articles/PMC2626396/figure/F6-7812/" class="imagepopup">Figure 6.  From: Ras <span class="highlight" style="background-color:">Signaling</span> Influences Permissiveness of Malignant Peripheral Nerve Sheath Tumor Cells to Oncolytic Herpes. </a></p><div class="supp"><p class="details"><div>Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> influences the outcome of oncolytic virus G207 insult in MPNST cells. This diagram explains the influence of Ras <span class="highlight" style="background-color:">signaling</span> via its downstream effector pathways on permissiveness of MPNST cells to G207. In cells with low levels of Ras <span class="highlight" style="background-color:">signaling</span>, viral protein synthesis does not occur at an efficient level. However, exposure to virus does induce certain levels of apoptosis in these cells in part because of an inactive PI3K <span class="highlight" style="background-color:">pathway</span> failing to exert its anti-apoptotic effects. In cells with elevated levels of Ras <span class="highlight" style="background-color:">signaling</span>, however, efficient viral protein synthesis and activation of the PI3K <span class="highlight" style="background-color:">pathway</span> reduces the capability of cells to undergo apoptosis. The expected outcome of this scenario would be robust G207 replication and death of cells via lytic infection (necrosis).</div></p></div><div class="aux"><div class="resc">Faris Farassati, et al. Am J Pathol. 2008 December;173(6):1861-1872.</div><p class="links"><a class="dblinks" href="/pubmed/18988803" ref="ordinalpos=1082&amp;ncbi_uid=1622258&amp;link_uid=18988803">Citation</a><a class="dblinks" href="/pmc/articles/PMC2626396/" ref="ordinalpos=1082&amp;ncbi_uid=1622258&amp;link_uid=PMC2626396">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1083.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3718064/figure/F2/" ref="ordinalpos=1083&amp;ncbi_uid=6086653&amp;link_uid=PMC3718064" href="/pmc/articles/PMC3718064/"><img src="/pmc/articles/instance/3718064/bin/nihms490537f2.gif" src-large="/pmc/articles/instance/3718064/bin/nihms490537f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3718064/" ref="ordinalpos=1083&amp;ncbi_uid=6086653&amp;link_uid=PMC3718064" image-link="/pmc/articles/PMC3718064/figure/F2/" class="imagepopup">Figure 2. The Anaplastic Lymphoma Kinase (ALK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Membrane Phospholipids, EML4-ALK and Hsp90 as Novel Targets in Lung Cancer Treatment. </a></p><div class="supp"><p class="details"><div>The downstream <span class="highlight" style="background-color:">signaling</span> pathways include the Rat sarcoma (Ras), mitogen activated protein (MAP), extracellular signal regulated kinase (ERK), phosphoinositide 3-kinase (PI3K), Akt, Janus activated kinase (JAK3) and Signal Transducer and Activator of Transcription (STAT3) <span class="highlight" style="background-color:">signaling</span> pathways. In general, the RAS/MEK/ERK <span class="highlight" style="background-color:">pathway</span> drives cell proliferation, and the PI3K/Akt and JAK3-STAT3 pathways play pro-survival roles. Although different ALK fusions may differentially activate downstream <span class="highlight" style="background-color:">signaling</span> pathways, EML4-ALK signals through ERK and PI3K. Activation of the RAS/MEK/ERK and PI3K/Akt pathways play key roles in EML4-ALK–mediated cellular oncogenesis.</div></p></div><div class="aux"><div class="resc">Andrei Laszlo, et al. Cancer J. ;19(3):238-246.</div><p class="links"><a class="dblinks" href="/pubmed/23708071" ref="ordinalpos=1083&amp;ncbi_uid=6086653&amp;link_uid=23708071">Citation</a><a class="dblinks" href="/pmc/articles/PMC3718064/" ref="ordinalpos=1083&amp;ncbi_uid=6086653&amp;link_uid=PMC3718064">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1084.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041713/figure/F2/" ref="ordinalpos=1084&amp;ncbi_uid=6186549&amp;link_uid=PMC4041713" href="/pmc/articles/PMC4041713/"><img src="/pmc/articles/instance/4041713/bin/nihms589041f2.gif" src-large="/pmc/articles/instance/4041713/bin/nihms589041f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041713/" ref="ordinalpos=1084&amp;ncbi_uid=6186549&amp;link_uid=PMC4041713" image-link="/pmc/articles/PMC4041713/figure/F2/" class="imagepopup">Figure 2. DE-miRNAs identified in lung sarcoidosis tissues are predicted to target the TGFβ molecular <span class="highlight" style="background-color:">pathway</span>.  From: Differential Expression of MicroRNA and Predicted Targets in Pulmonary Sarcoidosis. </a></p><div class="supp"><p class="details"><div>DIANA-mirPath was used to predict the most likely molecular pathways targeted by PCR-validated DE-miRNAs in sarcoidosis lung tissue and identified the TGFβ <span class="highlight" style="background-color:">pathway</span> as the most likely candidate. The arrows represent activation, whereas the hammerheads represent inhibition. The molecular components of the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and their putative interactions were derived from the KEGG <span class="highlight" style="background-color:">pathway</span> database.</div></p></div><div class="aux"><div class="resc">Elliott D. Crouser, et al. Biochem Biophys Res Commun. ;417(2):886-891.</div><p class="links"><a class="dblinks" href="/pubmed/22209793" ref="ordinalpos=1084&amp;ncbi_uid=6186549&amp;link_uid=22209793">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041713/" ref="ordinalpos=1084&amp;ncbi_uid=6186549&amp;link_uid=PMC4041713">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1085.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3679038/figure/pone-0065929-g007/" ref="ordinalpos=1085&amp;ncbi_uid=5003310&amp;link_uid=PMC3679038" href="/pmc/articles/PMC3679038/"><img src="/pmc/articles/instance/3679038/bin/pone.0065929.g007.gif" src-large="/pmc/articles/instance/3679038/bin/pone.0065929.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3679038/" ref="ordinalpos=1085&amp;ncbi_uid=5003310&amp;link_uid=PMC3679038" image-link="/pmc/articles/PMC3679038/figure/pone-0065929-g007/" class="imagepopup">Figure 7. Schematic representation showing the involvement of ERK-Nrf-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in radioresistance of EL-4 cells..  From: Involvement of ERK-Nrf-2 <span class="highlight" style="background-color:">Signaling</span> in Ionizing Radiation Induced Cell Death in Normal and Tumor Cells. </a></p><div class="aux"><div class="resc">Raghavendra S. Patwardhan, et al. PLoS One. 2013;8(6):e65929.</div><p class="links"><a class="dblinks" href="/pubmed/23776571" ref="ordinalpos=1085&amp;ncbi_uid=5003310&amp;link_uid=23776571">Citation</a><a class="dblinks" href="/pmc/articles/PMC3679038/" ref="ordinalpos=1085&amp;ncbi_uid=5003310&amp;link_uid=PMC3679038">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1086.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2869094/figure/F1/" ref="ordinalpos=1086&amp;ncbi_uid=2603396&amp;link_uid=PMC2869094" href="/pmc/articles/PMC2869094/"><img src="/pmc/articles/instance/2869094/bin/nihms180614f1.gif" src-large="/pmc/articles/instance/2869094/bin/nihms180614f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2869094/" ref="ordinalpos=1086&amp;ncbi_uid=2603396&amp;link_uid=PMC2869094" image-link="/pmc/articles/PMC2869094/figure/F1/" class="imagepopup">Figure 1. Selected gene interactions and epigenetic dysregulation of the canonical WNT <span class="highlight" style="background-color:">pathway</span>.  From: Epigenetic regulation of WNT <span class="highlight" style="background-color:">signaling</span> in chronic lymphocytic leukemia. </a></p><div class="supp"><p class="details"><div>The genes shown in solid black have previously been reported to be dysregulated through DNA methylation, histone modifications, miRNA expression and combinations of these mechanisms. How they affect the overall <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not yet known.<br />LEF: Lymphoid enhancing factor; SFRP: Soluble frizzled protein.</div></p></div><div class="aux"><div class="resc">Lynda B Bennett, et al. Epigenomics. ;2(1):53-70.</div><p class="links"><a class="dblinks" href="/pubmed/20473358" ref="ordinalpos=1086&amp;ncbi_uid=2603396&amp;link_uid=20473358">Citation</a><a class="dblinks" href="/pmc/articles/PMC2869094/" ref="ordinalpos=1086&amp;ncbi_uid=2603396&amp;link_uid=PMC2869094">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1087.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3415586/figure/F1/" ref="ordinalpos=1087&amp;ncbi_uid=5143554&amp;link_uid=PMC3415586" href="/pmc/articles/PMC3415586/"><img src="/pmc/articles/instance/3415586/bin/nihms373163f1.gif" src-large="/pmc/articles/instance/3415586/bin/nihms373163f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3415586/" ref="ordinalpos=1087&amp;ncbi_uid=5143554&amp;link_uid=PMC3415586" image-link="/pmc/articles/PMC3415586/figure/F1/" class="imagepopup">Fig. 1. Hematopoietic progenitors of DC are differentially regulated by ERα <span class="highlight" style="background-color:">signaling</span> in Flt3L and GM-CSF mediated developmental pathways.  From: Estrogen receptors regulate an inflammatory <span class="highlight" style="background-color:">pathway</span> of dendritic cell differentiation: mechanisms and implications for immunity. </a></p><div class="supp"><p class="details"><div>Long-term repopulating hematopoietic progenitors (LSK-flt3<sup>−</sup>) progress through LSK-flt3<sup>+</sup> progenitors which branch into myeloid (MP) and lymphoid (LP) progenitors. For DC development, MP yield the macrophage-DC progenitor (MDP) and common DC progenitor (CDP). In Flt3L-mediated DC development during homeostasis, the CDP yields plasmacytoid DC (pDC) and pre-cDC, the immediate precursors of conventional DC (cDC) and some tissue DC (tDC). Estradiol (E)/ERα <span class="highlight" style="background-color:">signaling</span> decreases the number of DC that develop along this <span class="highlight" style="background-color:">pathway</span> (<i>blue dashed circle</i>). In the GM-CSF-driven <span class="highlight" style="background-color:">pathway</span>, MP give rise to MDP and monocytes that differentiate to populations of inflammatory DC and some tDC. Estradiol/ERα <span class="highlight" style="background-color:">signaling</span> promotes this developmental <span class="highlight" style="background-color:">pathway</span> (<i>red solid circle</i>). LP also yield some DC populations <i>in vivo</i> (not shown). This figure was adapted from (Schmid et al., 2010).</div></p></div><div class="aux"><div class="resc">Susan Kovats. Horm Behav. ;62(3):254-262.</div><p class="links"><a class="dblinks" href="/pubmed/22561458" ref="ordinalpos=1087&amp;ncbi_uid=5143554&amp;link_uid=22561458">Citation</a><a class="dblinks" href="/pmc/articles/PMC3415586/" ref="ordinalpos=1087&amp;ncbi_uid=5143554&amp;link_uid=PMC3415586">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1088.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2169297/figure/F1/" ref="ordinalpos=1088&amp;ncbi_uid=1097575&amp;link_uid=PMC2169297" href="/pmc/articles/PMC2169297/"><img src="/pmc/articles/instance/2169297/bin/nihms33167f1.gif" src-large="/pmc/articles/instance/2169297/bin/nihms33167f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2169297/" ref="ordinalpos=1088&amp;ncbi_uid=1097575&amp;link_uid=PMC2169297" image-link="/pmc/articles/PMC2169297/figure/F1/" class="imagepopup">Figure 1. Keratinocyte Beta2 Adrenergic Receptor <span class="highlight" style="background-color:">Signaling</span> Pathways.  From: Beta Adrenergic Receptors in Keratinocytes. </a></p><div class="supp"><p class="details"><div>(a) cAMP dependent <span class="highlight" style="background-color:">signaling</span>, (b) PP2A dependent <span class="highlight" style="background-color:">pathway</span>, (c) IP3 and PKC dependent <span class="highlight" style="background-color:">pathway</span>, and (d) <span class="highlight" style="background-color:">pathway</span> for beta2 adrenergic receptor blockade. The <span class="highlight" style="background-color:">signaling</span> depicted in the diagram is unique for keratinocytes. In other cell types, agonist activation of the receptor results in different downstream effector cascades. Dotted lines indicate <span class="highlight" style="background-color:">signaling</span> pathways where causal associations have been demonstrated but definitive mechanisms have not yet been defined (described in text).</div></p></div><div class="aux"><div class="resc">Raja K. Sivamani, et al. Dermatol Clin. ;25(4):643-x.</div><p class="links"><a class="dblinks" href="/pubmed/17903623" ref="ordinalpos=1088&amp;ncbi_uid=1097575&amp;link_uid=17903623">Citation</a><a class="dblinks" href="/pmc/articles/PMC2169297/" ref="ordinalpos=1088&amp;ncbi_uid=1097575&amp;link_uid=PMC2169297">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1089.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1435660/figure/f5/" ref="ordinalpos=1089&amp;ncbi_uid=575402&amp;link_uid=PMC1435660" href="/pmc/articles/PMC1435660/"><img src="/pmc/articles/instance/1435660/bin/73f5c.gif" src-large="/pmc/articles/instance/1435660/bin/73f5c.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1435660/" ref="ordinalpos=1089&amp;ncbi_uid=575402&amp;link_uid=PMC1435660" image-link="/pmc/articles/PMC1435660/figure/f5/" class="imagepopup">Figure 5. From: Hereditary Hemorrhagic Telangiectasia, a Vascular Dysplasia Affecting the TGF-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The balanced <span class="highlight" style="background-color:">signaling</span> through TGF-β/ALK1 and TGF-β/ALK5 in healthy endothelial cells is broken in HHT patients. In HHT1 patients (endoglin mutated), the positive cooperation between endoglin and ALK1 is impaired and the ALK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is partially abolished. The same is true when ALK1 is mutated in HHT2 patients. Thus, in a primary dysregulation of HHT endothelial cells, proliferation and migration would decrease and, consequently, ALK5 <span class="highlight" style="background-color:">signaling</span> and quiescence would prevail, compromising the angiogenesis process. However, ALK5 levels are decreased in these HHT endothelial cells as part of an adaptive response mechanism. Based on results by Fernández-L et al.75</div></p></div><div class="aux"><div class="resc">Africa Fernández-L, et al. Clin Med Res. 2006 March;4(1):66-78.</div><p class="links"><a class="dblinks" href="/pubmed/16595794" ref="ordinalpos=1089&amp;ncbi_uid=575402&amp;link_uid=16595794">Citation</a><a class="dblinks" href="/pmc/articles/PMC1435660/" ref="ordinalpos=1089&amp;ncbi_uid=575402&amp;link_uid=PMC1435660">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1090.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3836044/figure/F4/" ref="ordinalpos=1090&amp;ncbi_uid=5510990&amp;link_uid=PMC3836044" href="/pmc/articles/PMC3836044/"><img src="/pmc/articles/instance/3836044/bin/nihms528215f4.gif" src-large="/pmc/articles/instance/3836044/bin/nihms528215f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3836044/" ref="ordinalpos=1090&amp;ncbi_uid=5510990&amp;link_uid=PMC3836044" image-link="/pmc/articles/PMC3836044/figure/F4/" class="imagepopup">Fig. 4.  From: Role of ?-Catenin/TCF-4 <span class="highlight" style="background-color:">Signaling</span> in HIV Replication and Pathogenesis: Insights to Informing Novel Anti-HIV Molecular Therapeutics. </a></p><div class="supp"><p class="details"><div>Model of the inverse relationship between Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> and HIV replication. When the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> is active, HIV replication is inhibited in astrocytes, peripheral blood lymphocytes (<i>PBLs</i>), and monocytes/macrophages. When β-catenin <span class="highlight" style="background-color:">signaling</span> is inactive or suppressed, as occurs when monocytes differentiate into macrophages or astrocytes are exposed to IFNγ, HIV replication is enhanced</div></p></div><div class="aux"><div class="resc">Lisa J. Henderson, et al. J Neuroimmune Pharmacol. ;6(2):10.1007/s11481-011-9266-7.</div><p class="links"><a class="dblinks" href="/pubmed/21384147" ref="ordinalpos=1090&amp;ncbi_uid=5510990&amp;link_uid=21384147">Citation</a><a class="dblinks" href="/pmc/articles/PMC3836044/" ref="ordinalpos=1090&amp;ncbi_uid=5510990&amp;link_uid=PMC3836044">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1091.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3590201/figure/pone-0057022-g001/" ref="ordinalpos=1091&amp;ncbi_uid=4696890&amp;link_uid=PMC3590201" href="/pmc/articles/PMC3590201/"><img src="/pmc/articles/instance/3590201/bin/pone.0057022.g001.gif" src-large="/pmc/articles/instance/3590201/bin/pone.0057022.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3590201/" ref="ordinalpos=1091&amp;ncbi_uid=4696890&amp;link_uid=PMC3590201" image-link="/pmc/articles/PMC3590201/figure/pone-0057022-g001/" class="imagepopup">Figure 1. KEGG <span class="highlight" style="background-color:">pathway</span> for MAPK <span class="highlight" style="background-color:">signaling</span>..  From: The Identification of <span class="highlight" style="background-color:">Pathway</span> Markers in Intracranial Aneurysm Using Genome-Wide Association Data from Two Different Populations. </a></p><div class="supp"><p class="details"><div>The set of genes shown in blue includes genes that are found by EU GWAS; yellow includes genes that are found by JP GWAS; red includes genes that are found both by EU and JP GWAS.</div></p></div><div class="aux"><div class="resc">Burcu Bakir-Gungor, et al. PLoS One. 2013;8(3):e57022.</div><p class="links"><a class="dblinks" href="/pubmed/23483893" ref="ordinalpos=1091&amp;ncbi_uid=4696890&amp;link_uid=23483893">Citation</a><a class="dblinks" href="/pmc/articles/PMC3590201/" ref="ordinalpos=1091&amp;ncbi_uid=4696890&amp;link_uid=PMC3590201">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1092.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2233603/figure/F7/" ref="ordinalpos=1092&amp;ncbi_uid=1246097&amp;link_uid=PMC2233603" href="/pmc/articles/PMC2233603/"><img src="/pmc/articles/instance/2233603/bin/nihms37849f7.gif" src-large="/pmc/articles/instance/2233603/bin/nihms37849f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2233603/" ref="ordinalpos=1092&amp;ncbi_uid=1246097&amp;link_uid=PMC2233603" image-link="/pmc/articles/PMC2233603/figure/F7/" class="imagepopup">Figure 7. A schematic presentation of Dvl complex containing Dvl1, Dvl2 and Dvl3 in Frizzled-1/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Differential mediation of the Wnt canonical <span class="highlight" style="background-color:">pathway</span> by mammalian Dishevelleds-1, -2, and -3. </a></p><div class="supp"><p class="details"><div>*, denotes “activation” in response to Wnt3a.</div></p></div><div class="aux"><div class="resc">Yi-Nan Lee, et al. Cell Signal. ;20(2):443-452.</div><p class="links"><a class="dblinks" href="/pubmed/18093802" ref="ordinalpos=1092&amp;ncbi_uid=1246097&amp;link_uid=18093802">Citation</a><a class="dblinks" href="/pmc/articles/PMC2233603/" ref="ordinalpos=1092&amp;ncbi_uid=1246097&amp;link_uid=PMC2233603">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1093.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC115962/figure/F5/" ref="ordinalpos=1093&amp;ncbi_uid=303889&amp;link_uid=PMC115962" href="/pmc/articles/PMC115962/"><img src="/pmc/articles/instance/115962/bin/ii0591289005.gif" src-large="/pmc/articles/instance/115962/bin/ii0591289005.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC115962/" ref="ordinalpos=1093&amp;ncbi_uid=303889&amp;link_uid=PMC115962" image-link="/pmc/articles/PMC115962/figure/F5/" class="imagepopup">FIG. 5.  From: Replication of Toxoplasma gondii, but Not Trypanosoma cruzi, Is Regulated in Human Fibroblasts Activated with Gamma  Interferon: Requirement of a Functional  JAK/STAT <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>IFN-γ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing the defects in the human fibroblast mutants B3, B9, and B10.</div></p></div><div class="aux"><div class="resc">Isabela Penna Cerávolo, et al. Infect Immun. 1999 May;67(5):2233-2240.</div><p class="links"><a class="dblinks" href="/pubmed/10225879" ref="ordinalpos=1093&amp;ncbi_uid=303889&amp;link_uid=10225879">Citation</a><a class="dblinks" href="/pmc/articles/PMC115962/" ref="ordinalpos=1093&amp;ncbi_uid=303889&amp;link_uid=PMC115962">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1094.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4023363/figure/F1/" ref="ordinalpos=1094&amp;ncbi_uid=6160892&amp;link_uid=PMC4023363" href="/pmc/articles/PMC4023363/"><img src="/pmc/articles/instance/4023363/bin/AJA-16-378-g001.gif" src-large="/pmc/articles/instance/4023363/bin/AJA-16-378-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4023363/" ref="ordinalpos=1094&amp;ncbi_uid=6160892&amp;link_uid=PMC4023363" image-link="/pmc/articles/PMC4023363/figure/F1/" class="imagepopup">Figure 1.  From: PI3K-AKT-mTOR <span class="highlight" style="background-color:">signaling</span> in prostate cancer progression and androgen deprivation therapy resistance. </a></p><div class="supp"><p class="details"><div>The PI3K-AKT-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and therapeutic opportunities. (<b>a</b>) A simplified schematic of PI3K-AKT-mTOR <span class="highlight" style="background-color:">signaling</span> and therapeutic targets. (<b>b-d</b>) The molecular interplay between the PI3K and AR <span class="highlight" style="background-color:">signaling</span> axes. ARE: androgen response element; mTOR: mammalian target of rapamycin; PI3K: phosphoinositide 3-kinase; AR: androgen receptor; RTK: receptor tyrosine kinase; GPCR: G-protein coupled receptor.</div></p></div><div class="aux"><div class="resc">Merritt P Edlind, et al. Asian J Androl. 2014 May-Jun;16(3):378-386.</div><p class="links"><a class="dblinks" href="/pubmed/24759575" ref="ordinalpos=1094&amp;ncbi_uid=6160892&amp;link_uid=24759575">Citation</a><a class="dblinks" href="/pmc/articles/PMC4023363/" ref="ordinalpos=1094&amp;ncbi_uid=6160892&amp;link_uid=PMC4023363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1095.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3786678/figure/F7/" ref="ordinalpos=1095&amp;ncbi_uid=5358465&amp;link_uid=PMC3786678" href="/pmc/articles/PMC3786678/"><img src="/pmc/articles/instance/3786678/bin/nihms504943f7.gif" src-large="/pmc/articles/instance/3786678/bin/nihms504943f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3786678/" ref="ordinalpos=1095&amp;ncbi_uid=5358465&amp;link_uid=PMC3786678" image-link="/pmc/articles/PMC3786678/figure/F7/" class="imagepopup">Figure 7. Insulin, cGMP, and TGF-β <span class="highlight" style="background-color:">Signaling</span> Regulate Quiescence through PKG.  From: Insulin, cGMP, and TGF-? Signals Regulate Food Intake and Quiescence in C. elegans: A Model for Satiety. </a></p><div class="supp"><p class="details"><div>Quiescence after fasting and refeeding is controlled by a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in which PKG (EGL-4) is activated by insulin, cGMP, and TGF-β pathways.</div></p></div><div class="aux"><div class="resc">Young-jai You, et al. Cell Metab. 2008 March;7(3):249-257.</div><p class="links"><a class="dblinks" href="/pubmed/18316030" ref="ordinalpos=1095&amp;ncbi_uid=5358465&amp;link_uid=18316030">Citation</a><a class="dblinks" href="/pmc/articles/PMC3786678/" ref="ordinalpos=1095&amp;ncbi_uid=5358465&amp;link_uid=PMC3786678">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1096.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4249956/figure/pone-0113442-g005/" ref="ordinalpos=1096&amp;ncbi_uid=6918219&amp;link_uid=PMC4249956" href="/pmc/articles/PMC4249956/"><img src="/pmc/articles/instance/4249956/bin/pone.0113442.g005.gif" src-large="/pmc/articles/instance/4249956/bin/pone.0113442.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4249956/" ref="ordinalpos=1096&amp;ncbi_uid=6918219&amp;link_uid=PMC4249956" image-link="/pmc/articles/PMC4249956/figure/pone-0113442-g005/" class="imagepopup">Figure 5. The effect of engineered minibody on EGFR downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Effective Therapeutic Approach for Head and Neck Cancer by an Engineered Minibody Targeting the EGFR Receptor. </a></p><div class="supp"><p class="details"><div>(A) Phosphorylation of EGFR. The phosphorylated EGFR of frozen section of A431 tumor tissue was analyzed by immunohistochemistry staining using p-EGFR (Tyr 845) antibody. Scale bar = 200 µm (B) EFGR downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in tumor lysate was analyzed by western blot using each antibody as indicated. β-actin was used as a loading control.</div></p></div><div class="aux"><div class="resc">Young Pil Kim, et al. PLoS One. 2014;9(12):e113442.</div><p class="links"><a class="dblinks" href="/pubmed/25438047" ref="ordinalpos=1096&amp;ncbi_uid=6918219&amp;link_uid=25438047">Citation</a><a class="dblinks" href="/pmc/articles/PMC4249956/" ref="ordinalpos=1096&amp;ncbi_uid=6918219&amp;link_uid=PMC4249956">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1097.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3620498/figure/pone-0060028-g007/" ref="ordinalpos=1097&amp;ncbi_uid=4807866&amp;link_uid=PMC3620498" href="/pmc/articles/PMC3620498/"><img src="/pmc/articles/instance/3620498/bin/pone.0060028.g007.gif" src-large="/pmc/articles/instance/3620498/bin/pone.0060028.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3620498/" ref="ordinalpos=1097&amp;ncbi_uid=4807866&amp;link_uid=PMC3620498" image-link="/pmc/articles/PMC3620498/figure/pone-0060028-g007/" class="imagepopup">Figure 7. Willin orchestrates sciatic nerve fibroblasts..  From: Willin, an Upstream Component of the Hippo <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>, Orchestrates Mammalian Peripheral Nerve Fibroblasts. </a></p><div class="supp"><p class="details"><div>Ectopic expression of Willin activates the Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and YAP translocation from the nucleus to the cytoplasm. It also promotes wound closure by directional migration without increased cellular proliferation and induces down-regulation of ephrinB2 and EGFR expression. Knockdown of Willin expression further supports the aforementioned findings.</div></p></div><div class="aux"><div class="resc">Susana Moleirinho, et al. PLoS One. 2013;8(4):e60028.</div><p class="links"><a class="dblinks" href="/pubmed/23593160" ref="ordinalpos=1097&amp;ncbi_uid=4807866&amp;link_uid=23593160">Citation</a><a class="dblinks" href="/pmc/articles/PMC3620498/" ref="ordinalpos=1097&amp;ncbi_uid=4807866&amp;link_uid=PMC3620498">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1098.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3559476/figure/pone-0055615-g004/" ref="ordinalpos=1098&amp;ncbi_uid=4600262&amp;link_uid=PMC3559476" href="/pmc/articles/PMC3559476/"><img src="/pmc/articles/instance/3559476/bin/pone.0055615.g004.gif" src-large="/pmc/articles/instance/3559476/bin/pone.0055615.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3559476/" ref="ordinalpos=1098&amp;ncbi_uid=4600262&amp;link_uid=PMC3559476" image-link="/pmc/articles/PMC3559476/figure/pone-0055615-g004/" class="imagepopup">Figure 4. Regulation of Akt/GSK-3β/cyclin D1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by βKlotho..  From: ?Klotho Suppresses Tumor Growth in Hepatocellular Carcinoma by Regulating Akt/GSK-3?/Cyclin D1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Western blotting analysis of βKlotho, cyclin D1, phosphorylated Akt (p-Akt), Akt, phosphorylated GSK-3β (p-GSK-3β), GSK-3β and tubulin levels in the indicated hepatoma cell lines transfected with vector or βKlotho. The experiments were performed independently three times at least.</div></p></div><div class="aux"><div class="resc">Xiaoming Ye, et al. PLoS One. 2013;8(1):e55615.</div><p class="links"><a class="dblinks" href="/pubmed/23383245" ref="ordinalpos=1098&amp;ncbi_uid=4600262&amp;link_uid=23383245">Citation</a><a class="dblinks" href="/pmc/articles/PMC3559476/" ref="ordinalpos=1098&amp;ncbi_uid=4600262&amp;link_uid=PMC3559476">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1099.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2808458/figure/F2/" ref="ordinalpos=1099&amp;ncbi_uid=2690338&amp;link_uid=PMC2808458" href="/pmc/articles/PMC2808458/"><img src="/pmc/articles/instance/2808458/bin/nihms-147724-f0002.gif" src-large="/pmc/articles/instance/2808458/bin/nihms-147724-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2808458/" ref="ordinalpos=1099&amp;ncbi_uid=2690338&amp;link_uid=PMC2808458" image-link="/pmc/articles/PMC2808458/figure/F2/" class="imagepopup">Figure 2. PI3K <span class="highlight" style="background-color:">pathway</span> in TLR <span class="highlight" style="background-color:">signaling</span>.  From: Toll-like receptor <span class="highlight" style="background-color:">signaling</span> in cell proliferation and survival. </a></p><div class="supp"><p class="details"><div>PI3K physically interacts with multiple TLRs as well as MyD88. Activated PI3K regulates TLR <span class="highlight" style="background-color:">signaling</span> in both positive and negative ways. Inactivation of GSK3 by PI3K-Akt enhances the transcription of IL10 by allowing CREB DNA binding as well as CREB-CBP association. IL-12 transcription is inhibited due to decreased association of NF-kB with CBP. PI3K-Akt also phosphorylates mTOR which leads to the phosphorylation and activation of IRF-7 as well as enhanced production of type I IFNs. PI3K-Akt <span class="highlight" style="background-color:">pathway</span> is also involved in cell proliferation or survival induced by multiple TLR ligands through degradation of the cell cycle inhibitor p27.</div></p></div><div class="aux"><div class="resc">Xinyan Li, et al. Cytokine. ;49(1):1-9.</div><p class="links"><a class="dblinks" href="/pubmed/19775907" ref="ordinalpos=1099&amp;ncbi_uid=2690338&amp;link_uid=19775907">Citation</a><a class="dblinks" href="/pmc/articles/PMC2808458/" ref="ordinalpos=1099&amp;ncbi_uid=2690338&amp;link_uid=PMC2808458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1100.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC365712/figure/fig6/" ref="ordinalpos=1100&amp;ncbi_uid=470461&amp;link_uid=PMC365712" href="/pmc/articles/PMC365712/"><img src="/pmc/articles/instance/365712/bin/zpq0080438690006.gif" src-large="/pmc/articles/instance/365712/bin/zpq0080438690006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC365712/" ref="ordinalpos=1100&amp;ncbi_uid=470461&amp;link_uid=PMC365712" image-link="/pmc/articles/PMC365712/figure/fig6/" class="imagepopup">Fig. 6. From: A small molecular activator of cardiac hypertrophy uncovered in a chemical screen for modifiers of the calcineurin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Possible <span class="highlight" style="background-color:">signaling</span> pathways controlled by PAMH. The activity of PAMH is blocked by staurosporin and the tyrosine kinase inhibitor AG490, implicating serine, threonine kinases, and tyrosine kinases, respectively, in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> controlled by PAMH. PAMH drives nuclear export of HDAC5 and nuclear import of NFATc and stimulates the expression of MCIP1 and other genes involved in cardiac growth and remodeling. <span class="highlight" style="background-color:">Signaling</span> events that appear to play dominant roles in PAMH <span class="highlight" style="background-color:">signaling</span> are shown in bold. Those that are documented, but for which the relative importance remains to be established, are shown by thin lines, and those that remain hypothetical are shown by dashed lines. Prohypertrophic effectors are in green, and antihypertrophic effectors are in red.</div></p></div><div class="aux"><div class="resc">Erik Bush, et al. Proc Natl Acad Sci U S A. 2004 March 2;101(9):2870-2875.</div><p class="links"><a class="dblinks" href="/pubmed/14976250" ref="ordinalpos=1100&amp;ncbi_uid=470461&amp;link_uid=14976250">Citation</a><a class="dblinks" href="/pmc/articles/PMC365712/" ref="ordinalpos=1100&amp;ncbi_uid=470461&amp;link_uid=PMC365712">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="10" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="11" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="12" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:22:42-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC67DA53FEC0410000000000282B9D;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC67DA53FEC0410000000000282B9D" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>