
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="7" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB1BB753FE89D10000000000597A74" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="6" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="7" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="8" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="7" /></div>
            <div><h2 class="result_count">Results: 601 to 700 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>601.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4109886/figure/pntd-0002994-g005/" ref="ordinalpos=601&amp;ncbi_uid=6427571&amp;link_uid=PMC4109886" href="/pmc/articles/PMC4109886/"><img src="/pmc/articles/instance/4109886/bin/pntd.0002994.g005.gif" src-large="/pmc/articles/instance/4109886/bin/pntd.0002994.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4109886/" ref="ordinalpos=601&amp;ncbi_uid=6427571&amp;link_uid=PMC4109886" image-link="/pmc/articles/PMC4109886/figure/pntd-0002994-g005/" class="imagepopup">Figure 5. CHIKV inhibition of JAK/STAT, IMD and Toll <span class="highlight" style="background-color:">signaling</span> pathways..  From: Characterization of Aedes aegypti Innate-Immune Pathways that Limit Chikungunya Virus Replication. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> assays were performed as previously described [23]. Aag2 cells were co-transfected with reporter plasmids and <i>in vitro</i> transcribed CHIKVRep-SG-eGFP RNA, followed by stimulation of immune pathways by heat inactivated bacteria. <span class="highlight" style="background-color:">Signaling</span> activity was measured by activity of FFluc. Internal transfection control (RLuc) is shown to indicate the levels of gene expression. PBS, PBS added to cells containing only the <span class="highlight" style="background-color:">pathway</span> reporter and internal transfection control; <i>E. coli</i>, heat inactivated bacteria added to cells containing only the <span class="highlight" style="background-color:">pathway</span> reporter and internal transfection control; <i>B. subtilis</i>, heat inactivated bacteria added to cells containing only the <span class="highlight" style="background-color:">pathway</span> reporter and internal transfection control; CHIKV, PBS added to cells containing <span class="highlight" style="background-color:">pathway</span> reporter, internal transfection control and CHIKVRep-SG-eGFP replicon RNA; CHIKV+<i>E. coli</i>, heat inactivated <i>E. coli</i> added to cells containing <span class="highlight" style="background-color:">pathway</span> reporter, internal transfection control and CHIKVRep-SG-eGFP replicon; CHIKV+<i>B. subtilis</i>, heat inactivated <i>B. subtilis</i> added to cells containing <span class="highlight" style="background-color:">pathway</span> reporter, internal transfection control and CHIKVRep-SG-eGFP replicon. Data is shown after subtraction of background luciferase levels and relative to PBS control. (<b>A</b>) JAK/STAT <span class="highlight" style="background-color:">pathway</span> stimulation with heat inactivated <i>E. coli</i> in the presence or absence of CHIKV replicon. (<b>B</b>) IMD <span class="highlight" style="background-color:">pathway</span> stimulation with heat inactivated <i>E. coli</i> in the presence or absence of CHIKV replicon. (<b>C</b>) Toll <span class="highlight" style="background-color:">pathway</span> stimulation with heat inactivated <i>B. subtilis</i> in the presence or absence of CHIKV replicon. (<b>D</b>) Co-transfection of pACT-<i>Renilla</i> with either CHIKVRep-SG-eGFP replicon RNA or control FFLuc expressing RNA. (<b>E</b>) Co-transfection of control RNA in the presence and absence of CHIKVRep-SG-eGFP replicon RNA. Graphs represent the mean of three independent experiments performed in triplicate and error bars represent the standard error of mean. *, significantly different using a general linear mixed model.</div></p></div><div class="aux"><div class="resc">Melanie McFarlane, et al. PLoS Negl Trop Dis. 2014 July;8(7):e2994.</div><p class="links"><a class="dblinks" href="/pubmed/25058001" ref="ordinalpos=601&amp;ncbi_uid=6427571&amp;link_uid=25058001">Citation</a><a class="dblinks" href="/pmc/articles/PMC4109886/" ref="ordinalpos=601&amp;ncbi_uid=6427571&amp;link_uid=PMC4109886">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>602.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3935308/figure/bau007-F2/" ref="ordinalpos=602&amp;ncbi_uid=5849163&amp;link_uid=PMC3935308" href="/pmc/articles/PMC3935308/"><img src="/pmc/articles/instance/3935308/bin/bau007f2p.gif" src-large="/pmc/articles/instance/3935308/bin/bau007f2p.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3935308/" ref="ordinalpos=602&amp;ncbi_uid=5849163&amp;link_uid=PMC3935308" image-link="/pmc/articles/PMC3935308/figure/bau007-F2/" class="imagepopup">Figure 2. From: TSLP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> map: a platform for analysis of TSLP-mediated <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>A high-confidence reaction map of the TSLP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> reaction map contains 37 proteins involved in 9 molecular associations, 21 enzyme catalysis reactions and 7 translocation events. Apart from the reactions selected based on NetSlim criteria, we have also included 2 enzyme complexes namely MTORC1 and PI3-kinase complex based on the inhibitor assays that confirmed their role in TSLP <span class="highlight" style="background-color:">signaling</span> and our phosphoproteomics study (the nodes highlighted in green). The edges representing the relationships between nodes are explained in the legend.</div></p></div><div class="aux"><div class="resc">Jun Zhong, et al. Database (Oxford). 2014;2014:bau007.</div><p class="links"><a class="dblinks" href="/pubmed/24573880" ref="ordinalpos=602&amp;ncbi_uid=5849163&amp;link_uid=24573880">Citation</a><a class="dblinks" href="/pmc/articles/PMC3935308/" ref="ordinalpos=602&amp;ncbi_uid=5849163&amp;link_uid=PMC3935308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>603.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3241690/figure/pone-0029160-g007/" ref="ordinalpos=603&amp;ncbi_uid=3725349&amp;link_uid=PMC3241690" href="/pmc/articles/PMC3241690/"><img src="/pmc/articles/instance/3241690/bin/pone.0029160.g007.gif" src-large="/pmc/articles/instance/3241690/bin/pone.0029160.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3241690/" ref="ordinalpos=603&amp;ncbi_uid=3725349&amp;link_uid=PMC3241690" image-link="/pmc/articles/PMC3241690/figure/pone-0029160-g007/" class="imagepopup">Figure 7. A proposed model of the function of NFI-C and TGF-β1 during odontoblast differentiation and homeostasis..  From: Crosstalk between Nuclear Factor I-C and Transforming Growth Factor-?1 <span class="highlight" style="background-color:">Signaling</span> Regulates Odontoblast Differentiation and Homeostasis. </a></p><div class="supp"><p class="details"><div>During early odontoblast differentiation, p-Smad2/3 induced by TGF-β1 <span class="highlight" style="background-color:">signaling</span> increased binding to NFI-C in the cytoplasm. On the other hand, MAPK activation by TGF-β1 <span class="highlight" style="background-color:">signaling</span> increased the interaction of phosphorylated NFI-C and Smurf1/2. Collectively, TGF-β1 and MAPK activation enhanced the interaction and formation of Smad2/3-NFI-C-Smurf1/2 complex and resulted in the degradation of NFI-C. In contrast, during late odontoblast differentiation and mineralization, NFI-C <span class="highlight" style="background-color:">signaling</span> resulted in the dephosphorylation of p-Smad2/3. Consequently, TGF-β1 induces odontoblast differentiation through the Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in early odontoblast differentiation, while NFI-C <span class="highlight" style="background-color:">signaling</span> modulates late odontoblast differentiation and mineralization.</div></p></div><div class="aux"><div class="resc">Dong-Seol Lee, et al. PLoS One. 2011;6(12):e29160.</div><p class="links"><a class="dblinks" href="/pubmed/22195013" ref="ordinalpos=603&amp;ncbi_uid=3725349&amp;link_uid=22195013">Citation</a><a class="dblinks" href="/pmc/articles/PMC3241690/" ref="ordinalpos=603&amp;ncbi_uid=3725349&amp;link_uid=PMC3241690">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>604.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3527475/figure/pone-0050638-g004/" ref="ordinalpos=604&amp;ncbi_uid=4495137&amp;link_uid=PMC3527475" href="/pmc/articles/PMC3527475/"><img src="/pmc/articles/instance/3527475/bin/pone.0050638.g004.gif" src-large="/pmc/articles/instance/3527475/bin/pone.0050638.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3527475/" ref="ordinalpos=604&amp;ncbi_uid=4495137&amp;link_uid=PMC3527475" image-link="/pmc/articles/PMC3527475/figure/pone-0050638-g004/" class="imagepopup">Figure 4. Sal represses the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and increases intracellular calcium in vitro..  From: Salinomycin Inhibits Proliferation and Induces Apoptosis of Human Hepatocellular Carcinoma Cells In Vitro and In Vivo. </a></p><div class="supp"><p class="details"><div>A. Real-time PCR was performed to examine mRNA expression of the Wnt <span class="highlight" style="background-color:">pathway</span> (DKK1, DKK2, GSK-3α, GSK-3β, and β-catenin), the hedgehog <span class="highlight" style="background-color:">pathway</span> (Gli-1, Patched, Shh, and Smo), the TGF-β <span class="highlight" style="background-color:">pathway</span> (TGF-β, Smad1, Smad2, Smad3, Smad4, Smad5, and Smad8), and growth factor receptor pathways (EGFR, VEGFR, AKT, and MAPK). Compared to control, the expression of β-catenin is significantly down-regulated (*<i>p</i>&lt;0.05). B. Sal inhibits Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> in HepG2, SMMC-7721, and BEL-7402 cells. p-GSK-3β, GSK-3β, and β-catenin expression were evaluated by Western blot.</div></p></div><div class="aux"><div class="resc">Fan Wang, et al. PLoS One. 2012;7(12):e50638.</div><p class="links"><a class="dblinks" href="/pubmed/23284640" ref="ordinalpos=604&amp;ncbi_uid=4495137&amp;link_uid=23284640">Citation</a><a class="dblinks" href="/pmc/articles/PMC3527475/" ref="ordinalpos=604&amp;ncbi_uid=4495137&amp;link_uid=PMC3527475">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>605.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3615118/figure/pone-0060078-g002/" ref="ordinalpos=605&amp;ncbi_uid=4782975&amp;link_uid=PMC3615118" href="/pmc/articles/PMC3615118/"><img src="/pmc/articles/instance/3615118/bin/pone.0060078.g002.gif" src-large="/pmc/articles/instance/3615118/bin/pone.0060078.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3615118/" ref="ordinalpos=605&amp;ncbi_uid=4782975&amp;link_uid=PMC3615118" image-link="/pmc/articles/PMC3615118/figure/pone-0060078-g002/" class="imagepopup">Figure 2. LPS-liposomes induce the TRIF-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> but not the MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Liposomal Lipopolysaccharide Initiates TRIF-Dependent <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Independent of CD14. </a></p><div class="supp"><p class="details"><div>Thioglycollate-elicited peritoneal macrophages (1×10<sup>6</sup> cells) from WT mice were stimulated with LPS (100 ng/mL) or LPS-liposomes (100 ng/mL) for 0–120 min. The cells were then lysed and the extracts immunoblotted with anti-IRF3 (A), anti-IκBα antibodies (B), or anti-IRAK and anti-MAPK antibodies (C). Data are average of three independent experiments and band intensity was quantified with Image J 1.45. The values represent means ± S.E.M <b>*</b><i>P</i>&lt;0.05.</div></p></div><div class="aux"><div class="resc">Sachiko Watanabe, et al. PLoS One. 2013;8(4):e60078.</div><p class="links"><a class="dblinks" href="/pubmed/23565187" ref="ordinalpos=605&amp;ncbi_uid=4782975&amp;link_uid=23565187">Citation</a><a class="dblinks" href="/pmc/articles/PMC3615118/" ref="ordinalpos=605&amp;ncbi_uid=4782975&amp;link_uid=PMC3615118">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>606.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2649208/figure/F6/" ref="ordinalpos=606&amp;ncbi_uid=1699599&amp;link_uid=PMC2649208" href="/pmc/articles/PMC2649208/"><img src="/pmc/articles/instance/2657410/bin/zpq9990971880006.gif" src-large="/pmc/articles/instance/2657410/bin/zpq9990971880006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2649208/" ref="ordinalpos=606&amp;ncbi_uid=1699599&amp;link_uid=PMC2649208" image-link="/pmc/articles/PMC2649208/figure/F6/" class="imagepopup">Fig. 6. From: V600EBRAF is associated with disabled feedback inhibition of RAF-MEK <span class="highlight" style="background-color:">signaling</span> and elevated transcriptional output of the <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Models of RAF/MEK/ERK <span class="highlight" style="background-color:">signaling</span> in tumor cells with RTK activation and with <sup>V600E</sup>BRAF. (<i>A</i>) RTK-activated tumor cells: feedback inhibition occurs at multiple levels of the <span class="highlight" style="background-color:">pathway</span>, down-regulating both RAF/MEK activation and ERK phosphorylation levels. Steady-state levels of pERK are caused by low levels of both MEK activity and ERK dephosphorylation by DUSPs. Output of the <span class="highlight" style="background-color:">pathway</span> is low. (<i>B</i>) <sup>V600E</sup>BRAF cells: RAF kinase is active and insusceptible to negative feedback. Increased <span class="highlight" style="background-color:">pathway</span> output leads to increased DUSP6 expression. Steady-state levels of pERK are caused by high levels of MEK and DUSP6 dephosphorylation of ERK.</div></p></div><div class="aux"><div class="resc">Christine A. Pratilas, et al. Proc Natl Acad Sci U S A. 2009 March 17;106(11):4519-4524.</div><p class="links"><a class="dblinks" href="/pubmed/19251651" ref="ordinalpos=606&amp;ncbi_uid=1699599&amp;link_uid=19251651">Citation</a><a class="dblinks" href="/pmc/articles/PMC2649208/" ref="ordinalpos=606&amp;ncbi_uid=1699599&amp;link_uid=PMC2649208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>607.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109867/figure/F1/" ref="ordinalpos=607&amp;ncbi_uid=3006880&amp;link_uid=PMC3109867" href="/pmc/articles/PMC3109867/"><img src="/pmc/articles/instance/3109867/bin/nihms296159f1.gif" src-large="/pmc/articles/instance/3109867/bin/nihms296159f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109867/" ref="ordinalpos=607&amp;ncbi_uid=3006880&amp;link_uid=PMC3109867" image-link="/pmc/articles/PMC3109867/figure/F1/" class="imagepopup">Figure 1.  From: InAKTivation of insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> by dephosphorylation. </a></p><div class="supp"><p class="details"><div>the insulin/iGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is conserved across phylogeny.</div></p></div><div class="aux"><div class="resc">Sri Devi Narasimhan, et al. Cell Cycle. ;8(23):3878-3884.</div><p class="links"><a class="dblinks" href="/pubmed/19901535" ref="ordinalpos=607&amp;ncbi_uid=3006880&amp;link_uid=19901535">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109867/" ref="ordinalpos=607&amp;ncbi_uid=3006880&amp;link_uid=PMC3109867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>608.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3602678/figure/F7/" ref="ordinalpos=608&amp;ncbi_uid=5868100&amp;link_uid=PMC3602678" href="/pmc/articles/PMC3602678/"><img src="/pmc/articles/instance/3602678/bin/zh30051364090007.gif" src-large="/pmc/articles/instance/3602678/bin/zh30051364090007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3602678/" ref="ordinalpos=608&amp;ncbi_uid=5868100&amp;link_uid=PMC3602678" image-link="/pmc/articles/PMC3602678/figure/F7/" class="imagepopup">Fig. 7. From: Epidermal growth factor receptor expression and <span class="highlight" style="background-color:">signaling</span> are essential in glutamine's cytoprotective mechanism in heat-stressed intestinal epithelial-6 cells. </a></p><div class="supp"><p class="details"><div>Proposed working model. GLN is protective in the intestine by preventing EGFR degradation after HS injury, as well as by activating the protective EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. GLN phosphorylates ERK1/2 and inactivates p38MAPK via EGFR tyrosine kinase <span class="highlight" style="background-color:">signaling</span> to prevent apoptosis. However, GLN is protective by activating the PI3-K <span class="highlight" style="background-color:">pathway</span> independently from EGFR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Stefanie Niederlechner, et al. Am J Physiol Gastrointest Liver Physiol. 2013 March 1;304(5):G543-G552.</div><p class="links"><a class="dblinks" href="/pubmed/23275616" ref="ordinalpos=608&amp;ncbi_uid=5868100&amp;link_uid=23275616">Citation</a><a class="dblinks" href="/pmc/articles/PMC3602678/" ref="ordinalpos=608&amp;ncbi_uid=5868100&amp;link_uid=PMC3602678">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>609.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404972/figure/pone-0041596-g007/" ref="ordinalpos=609&amp;ncbi_uid=4147466&amp;link_uid=PMC3404972" href="/pmc/articles/PMC3404972/"><img src="/pmc/articles/instance/3404972/bin/pone.0041596.g007.gif" src-large="/pmc/articles/instance/3404972/bin/pone.0041596.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404972/" ref="ordinalpos=609&amp;ncbi_uid=4147466&amp;link_uid=PMC3404972" image-link="/pmc/articles/PMC3404972/figure/pone-0041596-g007/" class="imagepopup">Figure 7. LRP5 regulates lung vascular and alveolar development through the Ang-Tie2 <span class="highlight" style="background-color:">pathway</span>..  From: LRP5 Regulates Development of Lung Microvessels and Alveoli through the Angiopoietin-Tie2 <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Angiogenesis and alveolar formation in the lung can be controlled by LRP5-Tie2 <span class="highlight" style="background-color:">signaling</span> depending on the Ang1/Ang2 ratio. In normal postnatal lung development where Ang1 is dominant, LRP5, an essential component of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, increases Tie2 expression in ECs and stimulates (downward arrow symbols) neonatal lung vascular and alveolar development. However, LRP5-Tie2 <span class="highlight" style="background-color:">signaling</span> inhibits (T-bar symbol) vascular and alveolar development when the lungs are exposed to hyperoxia in which Ang2 is upregulated.</div></p></div><div class="aux"><div class="resc">Tadanori Mammoto, et al. PLoS One. 2012;7(7):e41596.</div><p class="links"><a class="dblinks" href="/pubmed/22848540" ref="ordinalpos=609&amp;ncbi_uid=4147466&amp;link_uid=22848540">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404972/" ref="ordinalpos=609&amp;ncbi_uid=4147466&amp;link_uid=PMC3404972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>610.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1681489/figure/f3/" ref="ordinalpos=610&amp;ncbi_uid=809485&amp;link_uid=PMC1681489" href="/pmc/articles/PMC1681489/"><img src="/pmc/articles/instance/1681489/bin/msb4100057-f3.gif" src-large="/pmc/articles/instance/1681489/bin/msb4100057-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1681489/" ref="ordinalpos=610&amp;ncbi_uid=809485&amp;link_uid=PMC1681489" image-link="/pmc/articles/PMC1681489/figure/f3/" class="imagepopup">Figure 3.  From: A comprehensive map of the toll-like receptor <span class="highlight" style="background-color:">signaling</span> network. </a></p><div class="supp"><p class="details"><div>The core of bow-tie networks may have ‘classifier hyperspace' where reactions to various inputs can be classified within subregions within the hyperspace that consists of activation levels of core elements of the bow-tie structure. (<b>A</b>) EGFR <span class="highlight" style="background-color:">signaling</span> and GPCR <span class="highlight" style="background-color:">signaling</span> network has a small numbers of core elements such as small GTPases, PIPs, and non-RTKs for EGFR <span class="highlight" style="background-color:">signaling</span>, and calcium, cAMP, and inositol phosphate for GPCR <span class="highlight" style="background-color:">signaling</span>. For example, a group of stimuli may all activate core elements in the way that can be classified into region C1, for example, and these stimuli triggers transcription of genes and physiological responses as denoted as C1. (<b>B</b>) TLR <span class="highlight" style="background-color:">signaling</span> network has a salient feature where possible bow-tie core is composed of a single element, MyD88, thus there is no classifier hyperspace created within the bow-tie network. Differences of responses against various stimuli are modulated by the activation of elements in collateral pathways, such as small GTPases, PIPs, and TICAM1 involved in MyD88-independent <span class="highlight" style="background-color:">pathway</span>, and MyD88 <span class="highlight" style="background-color:">pathway</span> essentially calibrates the whole network. Thus, for TLR <span class="highlight" style="background-color:">signaling</span> network, the classifier hyperspace has four dimensions.</div></p></div><div class="aux"><div class="resc">Kanae Oda, et al. Mol Syst Biol. 2006;2:2006.0015-2006.0015.</div><p class="links"><a class="dblinks" href="/pubmed/16738560" ref="ordinalpos=610&amp;ncbi_uid=809485&amp;link_uid=16738560">Citation</a><a class="dblinks" href="/pmc/articles/PMC1681489/" ref="ordinalpos=610&amp;ncbi_uid=809485&amp;link_uid=PMC1681489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>611.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405869/figure/A008078F5/" ref="ordinalpos=611&amp;ncbi_uid=6431530&amp;link_uid=PMC3405869" href="/pmc/articles/PMC3405869/"><img src="/pmc/articles/instance/3405869/bin/cshperspect-WNT-008078_F5.gif" src-large="/pmc/articles/instance/3405869/bin/cshperspect-WNT-008078_F5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405869/" ref="ordinalpos=611&amp;ncbi_uid=6431530&amp;link_uid=PMC3405869" image-link="/pmc/articles/PMC3405869/figure/A008078F5/" class="imagepopup">Figure 5. From: Wnt <span class="highlight" style="background-color:">Signaling</span> and Injury Repair. </a></p><div class="supp"><p class="details"><div>Harnessing Wnt <span class="highlight" style="background-color:">signaling</span> for therapeutic applications in injury repair. There are many biological ways to activate or repress the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; these may be therapeutic options for increasing a healing response or reducing uncontrolled proliferation. The use of small molecules, microRNAs, genetically modified cells, as well as ligands and inhibitors, are all potential future therapeutics for both activating and repressing the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jemima L. Whyte, et al. Cold Spring Harb Perspect Biol. 2012 August;4(8):a008078.</div><p class="links"><a class="dblinks" href="/pubmed/22723493" ref="ordinalpos=611&amp;ncbi_uid=6431530&amp;link_uid=22723493">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405869/" ref="ordinalpos=611&amp;ncbi_uid=6431530&amp;link_uid=PMC3405869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>612.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4404090/figure/Fig6/" ref="ordinalpos=612&amp;ncbi_uid=7433348&amp;link_uid=PMC4404090" href="/pmc/articles/PMC4404090/"><img src="/pmc/articles/instance/4404090/bin/12943_2015_332_Fig6_HTML.gif" src-large="/pmc/articles/instance/4404090/bin/12943_2015_332_Fig6_HTML.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4404090/" ref="ordinalpos=612&amp;ncbi_uid=7433348&amp;link_uid=PMC4404090" image-link="/pmc/articles/PMC4404090/figure/Fig6/" class="imagepopup">Figure 6.  From: PinX1 inhibits the invasion and metastasis of human breast cancer via suppressing NF-κB/MMP-9 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>A Model of PinX1 suppresses MMP-9 expression via NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> We speculate that PinX1 can suppress the expression of MMP-9 owing to the inhibition of transcriptional activity of NF-κB p65 proteins by direct protein-protein interaction with its G-patch domain which could combine with the C-terminus of the NF-κB-repression factor (NRF).</div></p></div><div class="aux"><div class="resc">Meilin Shi, et al. Mol Cancer. 2015;14:66.</div><p class="links"><a class="dblinks" href="/pubmed/25888829" ref="ordinalpos=612&amp;ncbi_uid=7433348&amp;link_uid=25888829">Citation</a><a class="dblinks" href="/pmc/articles/PMC4404090/" ref="ordinalpos=612&amp;ncbi_uid=7433348&amp;link_uid=PMC4404090">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>613.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4383408/figure/fig4/" ref="ordinalpos=613&amp;ncbi_uid=7412220&amp;link_uid=PMC4383408" href="/pmc/articles/PMC4383408/"><img src="/pmc/articles/instance/4383408/bin/BMRI2015-124721.004.gif" src-large="/pmc/articles/instance/4383408/bin/BMRI2015-124721.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4383408/" ref="ordinalpos=613&amp;ncbi_uid=7412220&amp;link_uid=PMC4383408" image-link="/pmc/articles/PMC4383408/figure/fig4/" class="imagepopup">Figure 4.  From: Protecting Intestinal Epithelial Cell Number 6 against Fission Neutron Irradiation through NF-κB <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The expressions of key molecules of NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in IEC-6 cell lines after neutron irradiation of 4Gy and using of LY294002 (C: control group; N: neutron irradiation group; LY: neutron irradiation of 4Gy with LY294002 treatment group).</div></p></div><div class="aux"><div class="resc">Gong-Min Chang, et al. Biomed Res Int. 2015;2015:124721.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=613&amp;ncbi_uid=7412220&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4383408/" ref="ordinalpos=613&amp;ncbi_uid=7412220&amp;link_uid=PMC4383408">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>614.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2417171/figure/fig6/" ref="ordinalpos=614&amp;ncbi_uid=1470538&amp;link_uid=PMC2417171" href="/pmc/articles/PMC2417171/"><img src="/pmc/articles/instance/2417171/bin/zbc0140830780006.gif" src-large="/pmc/articles/instance/2417171/bin/zbc0140830780006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2417171/" ref="ordinalpos=614&amp;ncbi_uid=1470538&amp;link_uid=PMC2417171" image-link="/pmc/articles/PMC2417171/figure/fig6/" class="imagepopup">FIGURE 6. From: Identification of a Ligand-induced Transient Refractory Period in Nuclear  Factor-?B  <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Refractory period in NF-</b>κ<b>B <span class="highlight" style="background-color:">signaling</span>.</b> Shown is a  schematic representation of the different ligand-induced autoregulatory  mechanisms that control responsiveness in the NF-κB <span class="highlight" style="background-color:">signaling</span>  <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Britney L. Moss, et al. J Biol Chem. 2008 March 28;283(13):8687-8698.</div><p class="links"><a class="dblinks" href="/pubmed/18203717" ref="ordinalpos=614&amp;ncbi_uid=1470538&amp;link_uid=18203717">Citation</a><a class="dblinks" href="/pmc/articles/PMC2417171/" ref="ordinalpos=614&amp;ncbi_uid=1470538&amp;link_uid=PMC2417171">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>615.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3935308/figure/bau007-F1/" ref="ordinalpos=615&amp;ncbi_uid=5849164&amp;link_uid=PMC3935308" href="/pmc/articles/PMC3935308/"><img src="/pmc/articles/instance/3935308/bin/bau007f1p.gif" src-large="/pmc/articles/instance/3935308/bin/bau007f1p.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3935308/" ref="ordinalpos=615&amp;ncbi_uid=5849164&amp;link_uid=PMC3935308" image-link="/pmc/articles/PMC3935308/figure/bau007-F1/" class="imagepopup">Figure 1. From: TSLP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> map: a platform for analysis of TSLP-mediated <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>An overview of the TSLP <span class="highlight" style="background-color:">pathway</span> page in NetPath. The TSLP <span class="highlight" style="background-color:">pathway</span> page in NetPath provides statistics pertaining to the number of molecules curated, links to TSLP <span class="highlight" style="background-color:">pathway</span> reactions and a list of genes that are differentially regulated by TSLP. Each molecule in the <span class="highlight" style="background-color:">pathway</span> page is linked to the corresponding NetPath molecule page, which is further linked to Entrez Gene, HPRD, OMIM and Swiss-Prot identifiers. The reaction page of the TSLP <span class="highlight" style="background-color:">pathway</span> contains a list of each type of reaction such as physical interactions, enzyme catalysis or transport with a brief description of the reactions including their PTM dependence or interacting regions/domains/motifs whenever available in literature. The list of curators and reviewers are provided in the TSLP <span class="highlight" style="background-color:">pathway</span> page with the details of the <span class="highlight" style="background-color:">pathway</span> authority. A comments tab is also provided.</div></p></div><div class="aux"><div class="resc">Jun Zhong, et al. Database (Oxford). 2014;2014:bau007.</div><p class="links"><a class="dblinks" href="/pubmed/24573880" ref="ordinalpos=615&amp;ncbi_uid=5849164&amp;link_uid=24573880">Citation</a><a class="dblinks" href="/pmc/articles/PMC3935308/" ref="ordinalpos=615&amp;ncbi_uid=5849164&amp;link_uid=PMC3935308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>616.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2425463/figure/F6/" ref="ordinalpos=616&amp;ncbi_uid=1487959&amp;link_uid=PMC2425463" href="/pmc/articles/PMC2425463/"><img src="/pmc/articles/instance/2425463/bin/gkn234f6.gif" src-large="/pmc/articles/instance/2425463/bin/gkn234f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2425463/" ref="ordinalpos=616&amp;ncbi_uid=1487959&amp;link_uid=PMC2425463" image-link="/pmc/articles/PMC2425463/figure/F6/" class="imagepopup">Figure 6. From: Enhanced identification and biological validation of differential gene expression via Illumina whole-genome expression arrays through the use of the model-based background correction methodology. </a></p><div class="supp"><p class="details"><div>Comparison of <span class="highlight" style="background-color:">pathway</span> analysis on the data set using the MBCB, RMA or RAW background corrections. (<b>A</b>) IPA canonical <span class="highlight" style="background-color:">pathway</span> analysis depicting the top 9 <span class="highlight" style="background-color:">signaling</span> pathways that changed in AML tissues. The MBCB methodology identified more genes than either methodology in most categories. It also identified the death receptor <span class="highlight" style="background-color:">pathway</span> as the <span class="highlight" style="background-color:">pathway</span> that showed the most differences between the MBCB, RMA and RAW methods. (<b>B</b>) Scheme of death receptor <span class="highlight" style="background-color:">pathway</span> overlaid with the expression ratio of the MBCB data set. (<b>C</b>) Scheme of death receptor <span class="highlight" style="background-color:">pathway</span> overlaid with the expression ratio of the RMA data set. (<b>D</b>) Scheme of the death receptor <span class="highlight" style="background-color:">pathway</span> overlaid with the expression ratios from the RAW data set. Red, over-expressed in AML; green, under-expressed in AML; white, no significant change between AML and normal tissues. The cartoons show activation of <i>Tnfr2</i> and <i>NF-kappaB</i> <span class="highlight" style="background-color:">signaling</span> in the MBCB data set and the failure of detection of the same <span class="highlight" style="background-color:">pathway</span> by the RMA and RAW data sets.</div></p></div><div class="aux"><div class="resc">Liang-Hao Ding, et al. Nucleic Acids Res. 2008 June;36(10):e58-e58.</div><p class="links"><a class="dblinks" href="/pubmed/18450815" ref="ordinalpos=616&amp;ncbi_uid=1487959&amp;link_uid=18450815">Citation</a><a class="dblinks" href="/pmc/articles/PMC2425463/" ref="ordinalpos=616&amp;ncbi_uid=1487959&amp;link_uid=PMC2425463">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>617.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3813433/figure/F1/" ref="ordinalpos=617&amp;ncbi_uid=5489070&amp;link_uid=PMC3813433" href="/pmc/articles/PMC3813433/"><img src="/pmc/articles/instance/3813433/bin/nihms396676f1.gif" src-large="/pmc/articles/instance/3813433/bin/nihms396676f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3813433/" ref="ordinalpos=617&amp;ncbi_uid=5489070&amp;link_uid=PMC3813433" image-link="/pmc/articles/PMC3813433/figure/F1/" class="imagepopup">Figure 1. Graphic representation of the effect of fisetin on PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span>.  From: Dietary flavonoid fisetin: A novel dual inhibitor of PI3K/Akt and mTOR for prostate cancer management. </a></p><div class="supp"><p class="details"><div>Simultaneously targeting both PI3K/Akt and mTOR has the potential to inhibit both upstream and downstream <span class="highlight" style="background-color:">signaling</span> in the <span class="highlight" style="background-color:">pathway</span>. Fisetin inhibits the mTOR <span class="highlight" style="background-color:">pathway</span> and keeps the feedback loop in check by also inhibiting the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and inhibits cell survival and growth.</div></p></div><div class="aux"><div class="resc">Vaqar Mustafa Adhami, et al. Biochem Pharmacol. 2012 November 15;84(10):10.1016/j.bcp.2012.07.012.</div><p class="links"><a class="dblinks" href="/pubmed/22842629" ref="ordinalpos=617&amp;ncbi_uid=5489070&amp;link_uid=22842629">Citation</a><a class="dblinks" href="/pmc/articles/PMC3813433/" ref="ordinalpos=617&amp;ncbi_uid=5489070&amp;link_uid=PMC3813433">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>618.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3020247/figure/F4/" ref="ordinalpos=618&amp;ncbi_uid=2992137&amp;link_uid=PMC3020247" href="/pmc/articles/PMC3020247/"><img src="/pmc/articles/instance/3020247/bin/nihms261268f4.gif" src-large="/pmc/articles/instance/3020247/bin/nihms261268f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3020247/" ref="ordinalpos=618&amp;ncbi_uid=2992137&amp;link_uid=PMC3020247" image-link="/pmc/articles/PMC3020247/figure/F4/" class="imagepopup">Figure 4.  From: Pneumococcal Peptidoglycan-Polysaccharides Regulate Toll-Like Receptor 2 in the Mouse Middle Ear Epithelial Cells. </a></p><div class="supp"><p class="details"><div>TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in pneumococcal otitis media of rats. All genes involved in the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were evaluated by GeneSifter using <i>Z</i>-score as a parameter. Pn6A <i>vs</i>. PBS control had <i>Z</i>-scores (2.41 for up-regulated genes and −1.85 for down-regulated genes) of TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>A</i>, <i>black bar</i>, up-regulated; <i>gray bar</i>, down-regulated). Similar changes were observed in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>Dotted line</i> in <i>A</i> indicates a normal range for <i>Z</i>-scores (<i>A</i>). Microarray heat maps demonstrated that the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes and related genes were up-regulated (IL-1<i>β</i>, IL-6, MIP-1<i>α</i>, TLR4, TLR2, p38MAPK, jun, I<i>κ</i>B<i>α</i>) or down-regulated (MAPK14 and PI3 K p85<i>β</i>) (<i>B</i>) in the rat middle ear mucosa after Pn6A challenge for 3–14 d. <i>Upward</i> and <i>downward rows</i> indicate up-regulated and down-regulated genes, respectively.</div></p></div><div class="aux"><div class="resc">MASAHIRO KOMORI, et al. Pediatr Res. ;69(2):101-105.</div><p class="links"><a class="dblinks" href="/pubmed/21076367" ref="ordinalpos=618&amp;ncbi_uid=2992137&amp;link_uid=21076367">Citation</a><a class="dblinks" href="/pmc/articles/PMC3020247/" ref="ordinalpos=618&amp;ncbi_uid=2992137&amp;link_uid=PMC3020247">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>619.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2893887/figure/F7/" ref="ordinalpos=619&amp;ncbi_uid=2949831&amp;link_uid=PMC2893887" href="/pmc/articles/PMC2893887/"><img src="/pmc/articles/instance/2893887/bin/nihms197252f7.gif" src-large="/pmc/articles/instance/2893887/bin/nihms197252f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2893887/" ref="ordinalpos=619&amp;ncbi_uid=2949831&amp;link_uid=PMC2893887" image-link="/pmc/articles/PMC2893887/figure/F7/" class="imagepopup">Figure 7. Regulation of RasC <span class="highlight" style="background-color:">signaling</span> during chemotaxis.  From: A Ras <span class="highlight" style="background-color:">signaling</span> complex controls the RasC-TORC2 <span class="highlight" style="background-color:">pathway</span> and directed cell migration. </a></p><div class="supp"><p class="details"><div>Chemoattractant stimulation promotes the F-actin-dependent recruitment of the Sca1 complex to the plasma membrane and the subsequent activation of the RasC-TORC2-PKB/PKBR1 <span class="highlight" style="background-color:">pathway</span> at the leading edge of chemotaxing cells. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> controls cAMP production and leads to modulation of the F-actin cytoskeleton, thereby regulating signal relay and cell motility, respectively. The RasG-PI3K <span class="highlight" style="background-color:">pathway</span> appears to regulate F-actin, ACA (via CRAC), and PKB independently of the RasC-TORC2 <span class="highlight" style="background-color:">pathway</span>, and other regulators and/or Ras proteins control TORC2 activity in addition to RasC. It is also possible that RasC has other effectors. TORC2 and PKB/PKBR1 regulate RasC activity in a negative feedback fashion that involves regulation of the Sca1 complex’s localization through phosphorylation of Sca1. PP2A seems to be necessary for the function of the Sca1 complex, possibly for the dynamic regulation of the complex by phosphorylation.</div></p></div><div class="aux"><div class="resc">Pascale G. Charest, et al. Dev Cell. ;18(5):737-749.</div><p class="links"><a class="dblinks" href="/pubmed/20493808" ref="ordinalpos=619&amp;ncbi_uid=2949831&amp;link_uid=20493808">Citation</a><a class="dblinks" href="/pmc/articles/PMC2893887/" ref="ordinalpos=619&amp;ncbi_uid=2949831&amp;link_uid=PMC2893887">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>620.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012725/figure/F1/" ref="ordinalpos=620&amp;ncbi_uid=6121171&amp;link_uid=PMC4012725" href="/pmc/articles/PMC4012725/"><img src="/pmc/articles/instance/4012725/bin/oncotarget-05-1116-g001.gif" src-large="/pmc/articles/instance/4012725/bin/oncotarget-05-1116-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012725/" ref="ordinalpos=620&amp;ncbi_uid=6121171&amp;link_uid=PMC4012725" image-link="/pmc/articles/PMC4012725/figure/F1/" class="imagepopup">Figure 1. Targeting synergistic dependency on MAPK and dopamine <span class="highlight" style="background-color:">signaling</span> in glioblastoma.  From: Dopamine <span class="highlight" style="background-color:">signaling</span>: target in glioblastoma. </a></p><div class="supp"><p class="details"><div>Canonical pro-survival and mitogenic MAPK <span class="highlight" style="background-color:">signaling</span>, commonly deregulated in glioblastoma, is depicted as a simplified <span class="highlight" style="background-color:">signaling</span> cascade of EGFR, Ras-GTP and MAPK kinases Erk1 and Erk2 (Erk1/2). At the Ras node, MAPK <span class="highlight" style="background-color:">pathway</span> is positively modulated by dopamine <span class="highlight" style="background-color:">signaling</span>. Upon ligand binding, the DRD2 receptor holds inactive small GTPase Rap1 through activation of heterotrimeric G protein alpha i2 (alpha/beta/gamma) and the GTPase-activating protein RAP (RAP-GAPII), which promote GDP-bound (inactive) Rap1. Rap1 is a Ras antagonist and its inactivation thus results in amplification of MAPK <span class="highlight" style="background-color:">signaling</span>. Therefore, combined targeting of both pathways can offer a promising strategy for glioblastoma therapy.</div></p></div><div class="aux"><div class="resc">Jiri Bartek, et al. Oncotarget. 2014 May;5(5):1116-1117.</div><p class="links"><a class="dblinks" href="/pubmed/24657925" ref="ordinalpos=620&amp;ncbi_uid=6121171&amp;link_uid=24657925">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012725/" ref="ordinalpos=620&amp;ncbi_uid=6121171&amp;link_uid=PMC4012725">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>621.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4298363/figure/F7/" ref="ordinalpos=621&amp;ncbi_uid=7106599&amp;link_uid=PMC4298363" href="/pmc/articles/PMC4298363/"><img src="/pmc/articles/instance/4298363/bin/aging-06-1019-g007.gif" src-large="/pmc/articles/instance/4298363/bin/aging-06-1019-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4298363/" ref="ordinalpos=621&amp;ncbi_uid=7106599&amp;link_uid=PMC4298363" image-link="/pmc/articles/PMC4298363/figure/F7/" class="imagepopup">Figure 7. Schematic diagram of GHS-R mediated thermogenic regulation in brown adipocytes.  From: The suppression of ghrelin <span class="highlight" style="background-color:">signaling</span> mitigates age-associated thermogenic impairment. </a></p><div class="supp"><p class="details"><div>Our data suggest that ghrelin <span class="highlight" style="background-color:">signaling</span> may regulate thermogenesis in BAT via the following 4 independent and interconnected <span class="highlight" style="background-color:">signaling</span> pathways: 1) Ablation of GHS-R stimulates SNS-mediated NE release, which in turn induces β3-AR expression, subsequently activating thermogenic <span class="highlight" style="background-color:">signaling</span> cascades in BAT. This involves activation of thermogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> PKA-CREB-UCP1 and lipolytic <span class="highlight" style="background-color:">pathway</span> PKA-HSL-UCP1. 2) Ablation of GHS-R enhances insulin <span class="highlight" style="background-color:">signaling</span> in BAT, which improves insulin sensitivity of BAT and activates key thermogenic regulator PKA. 3) Ablation of GHS-R enhances AMPK activity in BAT, which increases DNA and protein synthesis of mitochondria, thus increasing mitochondrial biogenesis. 4) Ablation of GHS-R augments mitochondrial dynamics, enhancing both mitochondrial fission and fussion; this restores mitochondrial architecture and improves mitochondrial homeostasis. Improved mitochondrial homeostasis enhances the sensitivity of mitochondria to FFA to further promote mitochondrial uncoupling. Collectively, GHS-R ablation increases thermogenesis in BAT by activating thermogenic <span class="highlight" style="background-color:">signaling</span>, sensitizing insulin <span class="highlight" style="background-color:">signaling</span>, increasing mitochondrial biogenesis, and enhancing mitochondrial dynamics.</div></p></div><div class="aux"><div class="resc">Ligen Lin, et al. Aging (Albany NY). 2014 December;6(12):1019-1032.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=621&amp;ncbi_uid=7106599&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4298363/" ref="ordinalpos=621&amp;ncbi_uid=7106599&amp;link_uid=PMC4298363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>622.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2794543/figure/pone-0008509-g008/" ref="ordinalpos=622&amp;ncbi_uid=2096756&amp;link_uid=PMC2794543" href="/pmc/articles/PMC2794543/"><img src="/pmc/articles/instance/2794543/bin/pone.0008509.g008.gif" src-large="/pmc/articles/instance/2794543/bin/pone.0008509.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2794543/" ref="ordinalpos=622&amp;ncbi_uid=2096756&amp;link_uid=PMC2794543" image-link="/pmc/articles/PMC2794543/figure/pone-0008509-g008/" class="imagepopup">Figure 8. Hypothesis for an interplay between insulin and BIO <span class="highlight" style="background-color:">signaling</span> in contracting myotubes..  From: Activation of Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> Increases Insulin Sensitivity through a Reciprocal Regulation of Wnt10b and SREBP-1c in Skeletal Muscle Cells. </a></p><div class="supp"><p class="details"><div>Wnt10b (<b>a</b>) and BIO (<b>b</b>) activate the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> through the inactivation of GSK-3β<sup>Y216</sup> phosphorylation, which results in the nuclear translocation of active β-catenin, stimulation of myogenic genes transcription such as myoD, and inhibition of Srebp-1c transcription (<b>f</b>). Insulin-induced Srebp-1c transcription is mediated by the MAPK <span class="highlight" style="background-color:">pathway</span> in muscle cells. BIO inhibits the MAPK <span class="highlight" style="background-color:">pathway</span>, which could explain the down-regulation of Srebp-1c gene expression (<b>c</b>). In parallel, inactivation of GSK-3β<sup>Y216</sup> is followed by autophosphorylation of PDK1<sup>S241</sup> which phosphorylates Akt1<sup>T308</sup> (but not Akt2<sup>S473</sup>), then the subsequent phosphorylation of AS160<sup>S588</sup> induces GLUT4 translocation (<b>e</b>). In contrast, insulin stimulates GLUT4 translocation through the PI3K/Akt2<sup>S473</sup>/AS160 <span class="highlight" style="background-color:">pathway</span> (<b>d</b>). BIO activates the AMP kinase <span class="highlight" style="background-color:">pathway</span> by phosphorylating AMPK-α1<sup>S485</sup>, which also induces GLUT4 translocation (<b>e</b>). These results strongly suggest that Wnt <span class="highlight" style="background-color:">signaling</span>, in contrast to insulin <span class="highlight" style="background-color:">signaling</span>, increases glucose transport in both insulin-sensitive and insulin-resistant myotubes through the activation of AMPK-α1 and Akt2/AS160 pathways.</div></p></div><div class="aux"><div class="resc">Mounira Abiola, et al. PLoS One. 2009;4(12):e8509.</div><p class="links"><a class="dblinks" href="/pubmed/20041157" ref="ordinalpos=622&amp;ncbi_uid=2096756&amp;link_uid=20041157">Citation</a><a class="dblinks" href="/pmc/articles/PMC2794543/" ref="ordinalpos=622&amp;ncbi_uid=2096756&amp;link_uid=PMC2794543">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>623.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3671203/figure/F1/" ref="ordinalpos=623&amp;ncbi_uid=4983552&amp;link_uid=PMC3671203" href="/pmc/articles/PMC3671203/"><img src="/pmc/articles/instance/3671203/bin/fphys-04-00124-g001.gif" src-large="/pmc/articles/instance/3671203/bin/fphys-04-00124-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3671203/" ref="ordinalpos=623&amp;ncbi_uid=4983552&amp;link_uid=PMC3671203" image-link="/pmc/articles/PMC3671203/figure/F1/" class="imagepopup">Figure 1.  From: Integrative Analysis of Cancer-Related <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div><b>Interconnected <span class="highlight" style="background-color:">signaling</span> network of cancer-related genes and modules</b>. <b>(A)</b> Representation of the cancer relevant <span class="highlight" style="background-color:">signaling</span> network including all modules and contributing genes. <b>(B)</b> Exemplified schematic layout of the module grouping strategy; all genes in the network were associated with functional modules, e.g., genes encoding ligands, RTKs, or adaptor proteins of a given <span class="highlight" style="background-color:">pathway</span> were grouped into the corresponding ligand, RTK, or adaptor modules. In living cells, these modules contribute to activity of the pathways upstream <span class="highlight" style="background-color:">signaling</span>, here represented by a <span class="highlight" style="background-color:">pathway</span> upstream module. Functionally, the <span class="highlight" style="background-color:">pathway</span> upstream module of a given <span class="highlight" style="background-color:">pathway</span> activates or inhibits the activity of distinct downstream modules, here exemplified for activation of a MAPK and a PI3K/AKT module. These modules themselves consist of associated genes, e.g., encoding for ERK, RAS, or RAF isoforms in case of MAPK <span class="highlight" style="background-color:">signaling</span> or PI3K catalytic and regulatory isoforms as well as AKT, PTEN, and others. In our network, these downstream modules are again linked to events acting further downstream in the <span class="highlight" style="background-color:">pathway</span>, like activation of an AP1 or MYC transcription factor module.</div></p></div><div class="aux"><div class="resc">Thomas Kessler, et al. Front Physiol. 2013;4:124.</div><p class="links"><a class="dblinks" href="/pubmed/23760067" ref="ordinalpos=623&amp;ncbi_uid=4983552&amp;link_uid=23760067">Citation</a><a class="dblinks" href="/pmc/articles/PMC3671203/" ref="ordinalpos=623&amp;ncbi_uid=4983552&amp;link_uid=PMC3671203">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>624.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC59496/figure/F2/" ref="ordinalpos=624&amp;ncbi_uid=128320&amp;link_uid=PMC59496" href="/pmc/articles/PMC59496/"><img src="/pmc/articles/instance/59496/bin/pp1295804002.gif" src-large="/pmc/articles/instance/59496/bin/pp1295804002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC59496/" ref="ordinalpos=624&amp;ncbi_uid=128320&amp;link_uid=PMC59496" image-link="/pmc/articles/PMC59496/figure/F2/" class="imagepopup">Figure 2.  From: Changes in Cytosolic pH within Arabidopsis Root Columella Cells Play a Key Role in the Early <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> for Root Gravitropism. </a></p><div class="supp"><p class="details"><div>Outline of the experimental protocol designed to test the effects of various pH modifiers on the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for root gravitropism. At <i>T</i> = −30 min, vertical roots were rotated through 135° (Stimulus 1). At <i>T</i> = −5 min, 0.1 μL of control or pH-modiying solutions was applied solely to the root cap. Roots were returned to vertical at <i>T</i> = 0 (Stimulus 2). Three possible resultant morphologies are shown: I, Disrupted <span class="highlight" style="background-color:">signaling</span>; II, normal <span class="highlight" style="background-color:">signaling</span>; and III, enhanced <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Amie Caroline Scott, et al. Plant Physiol. 1999 December;121(4):1291-1298.</div><p class="links"><a class="dblinks" href="/pubmed/10594116" ref="ordinalpos=624&amp;ncbi_uid=128320&amp;link_uid=10594116">Citation</a><a class="dblinks" href="/pmc/articles/PMC59496/" ref="ordinalpos=624&amp;ncbi_uid=128320&amp;link_uid=PMC59496">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>625.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3469843/figure/F2/" ref="ordinalpos=625&amp;ncbi_uid=4309865&amp;link_uid=PMC3469843" href="/pmc/articles/PMC3469843/"><img src="/pmc/articles/instance/3469843/bin/lar-28-155-g002.gif" src-large="/pmc/articles/instance/3469843/bin/lar-28-155-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3469843/" ref="ordinalpos=625&amp;ncbi_uid=4309865&amp;link_uid=PMC3469843" image-link="/pmc/articles/PMC3469843/figure/F2/" class="imagepopup">Figure 2.  From: Effects of Red Liriope platyphylla on NGF secretion ability, NGF receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and ?-secretase components in NSE/hAPPsw transgenic mice expressing Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>Effects of RLP on downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of NGF receptor, including (A) TrkA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and (B) p75<sup>NTR</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Total cell lysates were prepared from brain cortex of NSE/hAPPsw Tg mice treated with vehicle or RLP as described in the Materials and Methods. Fifty micrograms of protein per sample was immunoblotted with antibodies for each protein. Three samples were assayed in triplicate by Western blotting. Data are reported as the mean±SD. <sup>*</sup><i>P</i>&lt;0.05 is the significance level compared to cells treated with vehicle.</div></p></div><div class="aux"><div class="resc">Sun-Il Choi, et al. Lab Anim Res. 2012 September;28(3):155-163.</div><p class="links"><a class="dblinks" href="/pubmed/23091515" ref="ordinalpos=625&amp;ncbi_uid=4309865&amp;link_uid=23091515">Citation</a><a class="dblinks" href="/pmc/articles/PMC3469843/" ref="ordinalpos=625&amp;ncbi_uid=4309865&amp;link_uid=PMC3469843">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>626.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3124552/figure/pone-0021638-g008/" ref="ordinalpos=626&amp;ncbi_uid=3076889&amp;link_uid=PMC3124552" href="/pmc/articles/PMC3124552/"><img src="/pmc/articles/instance/3124552/bin/pone.0021638.g008.gif" src-large="/pmc/articles/instance/3124552/bin/pone.0021638.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3124552/" ref="ordinalpos=626&amp;ncbi_uid=3076889&amp;link_uid=PMC3124552" image-link="/pmc/articles/PMC3124552/figure/pone-0021638-g008/" class="imagepopup">Figure 8. Canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis of group cluster composition..  From: Multiple Oxygen Tension Environments Reveal Diverse Patterns of Transcriptional Regulation in Primary Astrocytes. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Four way Venn diagram analysis of significantly populated canonical <span class="highlight" style="background-color:">signaling</span> pathways populated by group clusters <i>ONE</i>, <i>TWO</i>, <i>THREE</i> and <i>FOUR</i>. (<b>B</b>–<b>E</b>) Proportional pie charts created from multiple associated canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <i>hybrid</i> scores generated by the group cluster-unique (<b>B</b>-<i>ONE</i> unique; <b>C</b>-<i>TWO</i> unique; <b>D</b>-<i>THREE</i> unique; <b>E</b>-<i>FOUR</i> unique) <span class="highlight" style="background-color:">signaling</span> pathways populated by at least two genes with a co-probability of at least ≤0.05. Associated percentage values indicate the relative distribution of the respective <span class="highlight" style="background-color:">pathway</span> scores for each unique group cluster.</div></p></div><div class="aux"><div class="resc">Wayne Chadwick, et al. PLoS One. 2011;6(6):e21638.</div><p class="links"><a class="dblinks" href="/pubmed/21738745" ref="ordinalpos=626&amp;ncbi_uid=3076889&amp;link_uid=21738745">Citation</a><a class="dblinks" href="/pmc/articles/PMC3124552/" ref="ordinalpos=626&amp;ncbi_uid=3076889&amp;link_uid=PMC3124552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>627.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3377784/figure/F6/" ref="ordinalpos=627&amp;ncbi_uid=5045867&amp;link_uid=PMC3377784" href="/pmc/articles/PMC3377784/"><img src="/pmc/articles/instance/3377784/bin/nihms376161f6.gif" src-large="/pmc/articles/instance/3377784/bin/nihms376161f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3377784/" ref="ordinalpos=627&amp;ncbi_uid=5045867&amp;link_uid=PMC3377784" image-link="/pmc/articles/PMC3377784/figure/F6/" class="imagepopup">Fig. 6. Potential mechanisms for SOD2-mediated TSCC metastasis in association with Snail <span class="highlight" style="background-color:">signaling</span>.  From: Manganese superoxide dismutase induces migration and invasion of tongue squamous cell carcinoma via H2O2-dependent Snail <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Sod2 is activated by a variety of stimuli leading to an increase in the steady-state production of H<sub>2</sub>O<sub>2</sub>. H<sub>2</sub>O<sub>2</sub> activates the Snail-ERK1/2-MMP1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in TSCC metastasis.</div></p></div><div class="aux"><div class="resc">Zhonghua Liu, et al. Free Radic Biol Med. ;53(1):44-50.</div><p class="links"><a class="dblinks" href="/pubmed/22580338" ref="ordinalpos=627&amp;ncbi_uid=5045867&amp;link_uid=22580338">Citation</a><a class="dblinks" href="/pmc/articles/PMC3377784/" ref="ordinalpos=627&amp;ncbi_uid=5045867&amp;link_uid=PMC3377784">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>628.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2683622/figure/F9/" ref="ordinalpos=628&amp;ncbi_uid=1795814&amp;link_uid=PMC2683622" href="/pmc/articles/PMC2683622/"><img src="/pmc/articles/instance/2683622/bin/nihms105613f9.gif" src-large="/pmc/articles/instance/2683622/bin/nihms105613f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2683622/" ref="ordinalpos=628&amp;ncbi_uid=1795814&amp;link_uid=PMC2683622" image-link="/pmc/articles/PMC2683622/figure/F9/" class="imagepopup">Figure 9. Hypothetical model for PKD1 involvement in TLR/MyD88 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Protein kinase D1 is essential for MyD88-dependent TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Ligand-bound TLR recruits MyD88. TLR-bound MyD88 interacts with several proteins including IRFs, TRAF3, IRAK4 and IRAK1. On one hand, activated IRFs and TRAF3 initiates <span class="highlight" style="background-color:">signaling</span> cascades that lead to production of type I IFNs. One the other hand, IRAK1 is activated by IRAK4 in the TLR/MyD88/IRAK receptor complex. Subsequently, PKD1 is recruited to the TLR/MyD88 receptor complex by interacting with IRAK4 and IRAK1 and then activated by IRAKs. IRAK1-bound activated PKD1 leaves the TLR/MyD88 receptor complex and binds to TRAF6, which allows TRAF6 to interact with IRAK1, and leads to the ubiquitination of TRAF6. Ubiquitinated TRAF6 leads to the activation of TAK1, which in turn activates <span class="highlight" style="background-color:">signaling</span> cascades that lead to activation of MAPKs and NF-κB, and subsequent production of pro- inflammatory cytokines and chemokines.</div></p></div><div class="aux"><div class="resc">Jeoung-Eun Park, et al. J Immunol. ;182(10):6316-6327.</div><p class="links"><a class="dblinks" href="/pubmed/19414785" ref="ordinalpos=628&amp;ncbi_uid=1795814&amp;link_uid=19414785">Citation</a><a class="dblinks" href="/pmc/articles/PMC2683622/" ref="ordinalpos=628&amp;ncbi_uid=1795814&amp;link_uid=PMC2683622">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>629.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3747267/figure/pone-0071349-g005/" ref="ordinalpos=629&amp;ncbi_uid=5219882&amp;link_uid=PMC3747267" href="/pmc/articles/PMC3747267/"><img src="/pmc/articles/instance/3747267/bin/pone.0071349.g005.gif" src-large="/pmc/articles/instance/3747267/bin/pone.0071349.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3747267/" ref="ordinalpos=629&amp;ncbi_uid=5219882&amp;link_uid=PMC3747267" image-link="/pmc/articles/PMC3747267/figure/pone-0071349-g005/" class="imagepopup">Figure 5. Depletion of TBCK influences mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: TBCK Influences Cell Proliferation, Cell Size and mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>HEK293 cells were infected with the indicated lentivirus and subjected to Western blotting (<b>A</b>) and immunofluorescence analysis (<b>B</b>–<b>C</b>) with the antibodies as shown. GFP-positive signals indicate the cells infected by lentivirus. DNA was visualized by DAPI. Bar, 10 µm.</div></p></div><div class="aux"><div class="resc">Yueli Liu, et al. PLoS One. 2013;8(8):e71349.</div><p class="links"><a class="dblinks" href="/pubmed/23977024" ref="ordinalpos=629&amp;ncbi_uid=5219882&amp;link_uid=23977024">Citation</a><a class="dblinks" href="/pmc/articles/PMC3747267/" ref="ordinalpos=629&amp;ncbi_uid=5219882&amp;link_uid=PMC3747267">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>630.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2639789/figure/F9/" ref="ordinalpos=630&amp;ncbi_uid=1776221&amp;link_uid=PMC2639789" href="/pmc/articles/PMC2639789/"><img src="/pmc/articles/instance/2639789/bin/nihms85988f9.gif" src-large="/pmc/articles/instance/2639789/bin/nihms85988f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2639789/" ref="ordinalpos=630&amp;ncbi_uid=1776221&amp;link_uid=PMC2639789" image-link="/pmc/articles/PMC2639789/figure/F9/" class="imagepopup">Figure 9.  From: Role of Dickkopf-1 (Dkk1), an Antagonist of the Wnt-?-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>, in Estrogen-Induced Neuroprotection and Attenuation of Tau Phosphorylation. </a></p><div class="supp"><p class="details"><div>Summary diagram of E2 regulation of Wnt-β-Catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> following cerebral ischemia and proposed role in E2 ischemic neuroprotection. E2 acts to inhibit cerebral ischemia activation of JNK/Jun/Dkk1 <span class="highlight" style="background-color:">signaling</span>, which prevents subsequent phosphorylation of tau and β-Catenin and cell death, while it acts to enhances expression of Wnt3, leading to accumulation of nuclear β-Catenin and enhanced expression of survivin, which inhibits apoptosis and enhances neuronal cell survival.</div></p></div><div class="aux"><div class="resc">Quan-Guang Zhang, et al. J Neurosci. ;28(34):8430-8441.</div><p class="links"><a class="dblinks" href="/pubmed/18716201" ref="ordinalpos=630&amp;ncbi_uid=1776221&amp;link_uid=18716201">Citation</a><a class="dblinks" href="/pmc/articles/PMC2639789/" ref="ordinalpos=630&amp;ncbi_uid=1776221&amp;link_uid=PMC2639789">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>631.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2602983/figure/pone-0003917-g005/" ref="ordinalpos=631&amp;ncbi_uid=1580602&amp;link_uid=PMC2602983" href="/pmc/articles/PMC2602983/"><img src="/pmc/articles/instance/2602983/bin/pone.0003917.g005.gif" src-large="/pmc/articles/instance/2602983/bin/pone.0003917.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2602983/" ref="ordinalpos=631&amp;ncbi_uid=1580602&amp;link_uid=PMC2602983" image-link="/pmc/articles/PMC2602983/figure/pone-0003917-g005/" class="imagepopup">Figure 5. Dkk1 inhibits Wnt <span class="highlight" style="background-color:">signaling</span> in HB1.F3..  From: Olig2-Induced Neural Stem Cell Differentiation Involves Downregulation of Wnt <span class="highlight" style="background-color:">Signaling</span> and Induction of Dickkopf-1 Expression. </a></p><div class="supp"><p class="details"><div>A. TOPflash/FOPflash reporter assay shows that Dkk1 represses Wnt <span class="highlight" style="background-color:">signaling</span> in HB1.F3 in a dosage-dependent manner. Data are means±S.D. of triplicate samples. B. Dkk1 represses the expression of c-myc, one of the target genes of Wnt <span class="highlight" style="background-color:">pathway</span>, in HB1.F3 in a dosage-dependent manner. C. Dkk1 treatment induces the expression of oligodendrocytes markers (Olig2 and CNPase) in HB1.F3.</div></p></div><div class="aux"><div class="resc">Sung-Min Ahn, et al. PLoS ONE. 2008;3(12):e3917.</div><p class="links"><a class="dblinks" href="/pubmed/19093005" ref="ordinalpos=631&amp;ncbi_uid=1580602&amp;link_uid=19093005">Citation</a><a class="dblinks" href="/pmc/articles/PMC2602983/" ref="ordinalpos=631&amp;ncbi_uid=1580602&amp;link_uid=PMC2602983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>632.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2669196/figure/f1/" ref="ordinalpos=632&amp;ncbi_uid=1863455&amp;link_uid=PMC2669196" href="/pmc/articles/PMC2669196/"><img src="/pmc/articles/instance/2669196/bin/zek0040933030001.gif" src-large="/pmc/articles/instance/2669196/bin/zek0040933030001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2669196/" ref="ordinalpos=632&amp;ncbi_uid=1863455&amp;link_uid=PMC2669196" image-link="/pmc/articles/PMC2669196/figure/f1/" class="imagepopup">FIG. 1. From: Hog1 Mitogen-Activated Protein Kinase (MAPK) Interrupts Signal Transduction between the Kss1 MAPK and the Tec1 Transcription Factor To Maintain <span class="highlight" style="background-color:">Pathway</span> Specificity . </a></p><div class="supp"><p class="details"><div>FG and HOG MAPK <span class="highlight" style="background-color:">signaling</span> pathways. Components that are shared between the FG and HOG pathways are depicted in solid green. Components of the FG <span class="highlight" style="background-color:">pathway</span> are outlined in yellow, and components of the HOG <span class="highlight" style="background-color:">pathway</span> are outlined in blue. For simplicity, some interactions for which there exist evidence are not depicted such as those between Sho1 and Ste50 and Sho1 and Ste11. Also, the interaction between Ste50 and Cdc42 is not depicted in the HOG <span class="highlight" style="background-color:">pathway</span>, and only one of several possible transcription factors is depicted in the HOG <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Teresa R. Shock, et al. Eukaryot Cell. 2009 April;8(4):606-616.</div><p class="links"><a class="dblinks" href="/pubmed/19218425" ref="ordinalpos=632&amp;ncbi_uid=1863455&amp;link_uid=19218425">Citation</a><a class="dblinks" href="/pmc/articles/PMC2669196/" ref="ordinalpos=632&amp;ncbi_uid=1863455&amp;link_uid=PMC2669196">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>633.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3128635/figure/fig1-1947601911408079/" ref="ordinalpos=633&amp;ncbi_uid=3078623&amp;link_uid=PMC3128635" href="/pmc/articles/PMC3128635/"><img src="/pmc/articles/instance/3128635/bin/10.1177_1947601911408079-fig1.gif" src-large="/pmc/articles/instance/3128635/bin/10.1177_1947601911408079-fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3128635/" ref="ordinalpos=633&amp;ncbi_uid=3078623&amp;link_uid=PMC3128635" image-link="/pmc/articles/PMC3128635/figure/fig1-1947601911408079/" class="imagepopup">Figure 1. From: RAS Interaction with PI3K. </a></p><div class="supp"><p class="details"><div>RAS <span class="highlight" style="background-color:">signaling</span> to the PI3K <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Esther Castellano, et al. Genes Cancer. 2011 March;2(3):261-274.</div><p class="links"><a class="dblinks" href="/pubmed/21779497" ref="ordinalpos=633&amp;ncbi_uid=3078623&amp;link_uid=21779497">Citation</a><a class="dblinks" href="/pmc/articles/PMC3128635/" ref="ordinalpos=633&amp;ncbi_uid=3078623&amp;link_uid=PMC3128635">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>634.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4033595/figure/F4/" ref="ordinalpos=634&amp;ncbi_uid=6167705&amp;link_uid=PMC4033595" href="/pmc/articles/PMC4033595/"><img src="/pmc/articles/instance/4033595/bin/nihms382768f4.gif" src-large="/pmc/articles/instance/4033595/bin/nihms382768f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4033595/" ref="ordinalpos=634&amp;ncbi_uid=6167705&amp;link_uid=PMC4033595" image-link="/pmc/articles/PMC4033595/figure/F4/" class="imagepopup">Fig. 4.  From: Phosphorylation of serine 212 confers novel activity to human estrogen receptor ?. </a></p><div class="supp"><p class="details"><div>Differential gene regulation by ERα S212A and ERα S212D. (A) Venn diagram illustrating the number of specific and common genes regulated by the two ERα mutants. (B) Gene ontology analysis (IPA) for significant canonical pathways. A, arylhydrocarbon receptor <span class="highlight" style="background-color:">signaling</span>; B, tumoricidal function of hepatic natural killer cells; C, LPS/IL-1 mediated inhibition of RXR function; D, IL-6 <span class="highlight" style="background-color:">signaling</span>; E, dendritic cell maturation; F, communication between innate and adaptive immune cells; G, OX40 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; H, regulation of IL-2 expression in activated and anergic T lymphocytes; I, CTLA4 <span class="highlight" style="background-color:">signaling</span> in cytotoxic T lymphocytes; J, T helper cell differentiation; K, IL-9 <span class="highlight" style="background-color:">signaling</span>; L, CD28 <span class="highlight" style="background-color:">signaling</span> in T helper cells; M, ICOs-ICOSL <span class="highlight" style="background-color:">signaling</span> in T helper cells; N, Crosstalk between dendritic cells and natural killer cells; O, PKCδ <span class="highlight" style="background-color:">signaling</span> in T lymphocytes; P, MSP-ROH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; Q, antigen presentation <span class="highlight" style="background-color:">pathway</span>. Bars in darker blue and in lighter blue represent <span class="highlight" style="background-color:">signaling</span> pathways that were regulated by ERα S121A and ERα S212D, respectively.</div></p></div><div class="aux"><div class="resc">Sawako Shindo, et al. Steroids. ;77(5):448-453.</div><p class="links"><a class="dblinks" href="/pubmed/22266331" ref="ordinalpos=634&amp;ncbi_uid=6167705&amp;link_uid=22266331">Citation</a><a class="dblinks" href="/pmc/articles/PMC4033595/" ref="ordinalpos=634&amp;ncbi_uid=6167705&amp;link_uid=PMC4033595">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>635.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2775047/figure/F1/" ref="ordinalpos=635&amp;ncbi_uid=2091805&amp;link_uid=PMC2775047" href="/pmc/articles/PMC2775047/"><img src="/pmc/articles/instance/2775047/bin/nihms44974f1.gif" src-large="/pmc/articles/instance/2775047/bin/nihms44974f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2775047/" ref="ordinalpos=635&amp;ncbi_uid=2091805&amp;link_uid=PMC2775047" image-link="/pmc/articles/PMC2775047/figure/F1/" class="imagepopup">Figure 1. TLR4-initiated <span class="highlight" style="background-color:">signaling</span> pathways in BECs.  From: TLR Mediated Immune Responses in the Urinary Tract. </a></p><div class="supp"><p class="details"><div>Upon ligation by LPS, TLR4 initiates the classical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving NF-kB as well as a second chain of reactions where cAMP is a major substrate. The latter <span class="highlight" style="background-color:">pathway</span> results in a markedly faster cytokine response than the former <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jeongmin Song, et al. Curr Opin Microbiol. ;11(1):66-73.</div><p class="links"><a class="dblinks" href="/pubmed/18243043" ref="ordinalpos=635&amp;ncbi_uid=2091805&amp;link_uid=18243043">Citation</a><a class="dblinks" href="/pmc/articles/PMC2775047/" ref="ordinalpos=635&amp;ncbi_uid=2091805&amp;link_uid=PMC2775047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>636.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3476325/figure/F9/" ref="ordinalpos=636&amp;ncbi_uid=5400062&amp;link_uid=PMC3476325" href="/pmc/articles/PMC3476325/"><img src="/pmc/articles/instance/3476325/bin/zbc0441226970009.gif" src-large="/pmc/articles/instance/3476325/bin/zbc0441226970009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3476325/" ref="ordinalpos=636&amp;ncbi_uid=5400062&amp;link_uid=PMC3476325" image-link="/pmc/articles/PMC3476325/figure/F9/" class="imagepopup">FIGURE 9. From: CCL2/CCR2 Chemokine <span class="highlight" style="background-color:">Signaling</span> Coordinates Survival and Motility of Breast Cancer Cells through Smad3 Protein- and p42/44 Mitogen-activated Protein Kinase (MAPK)-dependent Mechanisms. </a></p><div class="supp"><p class="details"><div><b>Proposed model for how CCL2 <span class="highlight" style="background-color:">signaling</span> through CCR2 <span class="highlight" style="background-color:">signaling</span> regulates survival and motility of breast cancer cells.</b> CCL2 binds to CCR2 to stimulate Smad3 and p42/44MAPK pathways. In one <span class="highlight" style="background-color:">pathway</span>, CCL2 induces MEK <span class="highlight" style="background-color:">signaling</span> through p42/44MAPK independently of Smad3 to promote cell survival. In addition, CCL2 activates Smad3, which cooperates with the MEK-p42/44MAPK <span class="highlight" style="background-color:">pathway</span> to regulate cell motility and survival through RhoA-dependent mechanisms.</div></p></div><div class="aux"><div class="resc">Wei Bin Fang, et al. J Biol Chem. 2012 October 19;287(43):36593-36608.</div><p class="links"><a class="dblinks" href="/pubmed/22927430" ref="ordinalpos=636&amp;ncbi_uid=5400062&amp;link_uid=22927430">Citation</a><a class="dblinks" href="/pmc/articles/PMC3476325/" ref="ordinalpos=636&amp;ncbi_uid=5400062&amp;link_uid=PMC3476325">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>637.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3565105/figure/F1/" ref="ordinalpos=637&amp;ncbi_uid=5754738&amp;link_uid=PMC3565105" href="/pmc/articles/PMC3565105/"><img src="/pmc/articles/instance/3565105/bin/zef0061228240001.gif" src-large="/pmc/articles/instance/3565105/bin/zef0061228240001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3565105/" ref="ordinalpos=637&amp;ncbi_uid=5754738&amp;link_uid=PMC3565105" image-link="/pmc/articles/PMC3565105/figure/F1/" class="imagepopup">Figure 1. From: Derailed Estrogen <span class="highlight" style="background-color:">Signaling</span> and Breast Cancer: An Authentic Couple. </a></p><div class="supp"><p class="details"><div>Hypothetical model illustrates the E2-ERα <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving both genomic and extranuclear <span class="highlight" style="background-color:">signaling</span> pathways. <i>Extranuclear <span class="highlight" style="background-color:">signaling</span> of E2/ERα:</i> Several <span class="highlight" style="background-color:">signaling</span> proteins like PI3K, Src, HPIP, MTA1s, <i>etc</i>., interact with ERα in the cytoplasm to activate E2 extranuclear <span class="highlight" style="background-color:">signaling</span>. Rapid E2 <span class="highlight" style="background-color:">signaling</span> promotes interaction of ERα with caveolin-1 in the caveoli and activates the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. Sequestration of ERα by HPIP in the cytoplasm through a microtubule scaffolding mechanism facilitates PI3K/Src recruitment, and activation of AKT/MAPK pathways occurs in response to rapid E2 <span class="highlight" style="background-color:">signaling</span>. MTA1s also activate E2 rapid <span class="highlight" style="background-color:">signaling</span> through cytoplasmic sequestration of the receptor. In response to rapid E2/ERα <span class="highlight" style="background-color:">signaling</span>, PELP1 could activate AKT and MAPK pathways by interaction with PI3K and Src kinase, respectively. Activation of downstream <span class="highlight" style="background-color:">signaling</span> kinases such as AKT, MAPK, PAK1, <i>etc.</i>, by growth factor <span class="highlight" style="background-color:">signaling</span> led to phosphorylation of ERα, which could further impact its nuclear activity. Phosphorylation of ERα by GSK3 also enhances ERα transcriptional activity. Methylation of ERα at arginine 260 by PRMT1 involves activation of FAK <span class="highlight" style="background-color:">signaling</span> in response to E2 rapid <span class="highlight" style="background-color:">signaling</span>. <i>Genomic <span class="highlight" style="background-color:">signaling</span> of E2/ERα:</i> Ligand binding to ERα ensures heat shock protein (HSP) dissociation and the receptor's nuclear entry. Upon nuclear translocation, ligand-bound receptor binds to its target genes to activate the transcription. If the HDAC complex is recruited to ERα chromatin, ERα-dependent transcription is repressed, whereas HAT complex recruitment activates ERα-dependent transcription. This model also illustrates that ERα can regulate the gene expression by extensive chromatin looping to bring genes together for coordinated transcriptional regulation. PM, Plasma membrane; NM, nuclear membrane; P, phosphorylation; M, methylation; GF, growth factor; MT, microtubules; CoA, coactivator; CoR, corepressor; ER, estrogen receptor α; HDAC, histone deacetyl transferase; RNA Pol II, RNA polymerase II; TFB, transcription factor binding proteins; FAK, focal adhesion kinase.</div></p></div><div class="aux"><div class="resc">Bramanandam Manavathi, et al. Endocr Rev. 2013 February;34(1):1-32.</div><p class="links"><a class="dblinks" href="/pubmed/22947396" ref="ordinalpos=637&amp;ncbi_uid=5754738&amp;link_uid=22947396">Citation</a><a class="dblinks" href="/pmc/articles/PMC3565105/" ref="ordinalpos=637&amp;ncbi_uid=5754738&amp;link_uid=PMC3565105">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>638.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4121322/figure/pone-0104148-g005/" ref="ordinalpos=638&amp;ncbi_uid=6463698&amp;link_uid=PMC4121322" href="/pmc/articles/PMC4121322/"><img src="/pmc/articles/instance/4121322/bin/pone.0104148.g005.gif" src-large="/pmc/articles/instance/4121322/bin/pone.0104148.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4121322/" ref="ordinalpos=638&amp;ncbi_uid=6463698&amp;link_uid=PMC4121322" image-link="/pmc/articles/PMC4121322/figure/pone-0104148-g005/" class="imagepopup">Figure 5. Involvement of MARCO in the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: The Scavenger Receptor MARCO Modulates TLR-Induced Responses in Dendritic Cells. </a></p><div class="supp"><p class="details"><div>Expression of differentially expressed genes in our dataset that are known to be regulated through by SMAD transcription factors within the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was measured by RT-PCR in resting WT and MARCO<sup>−/−</sup> DC. *<i>P</i>&lt;.05, **<i>P</i>&lt;.01. Data show the mean ± SD of 3 WT and 3 MARCO<sup>−/−</sup> samples where each sample represents a pool of 3 splenocyte preparations.</div></p></div><div class="aux"><div class="resc">Haydn T. Kissick, et al. PLoS One. 2014;9(8):e104148.</div><p class="links"><a class="dblinks" href="/pubmed/25089703" ref="ordinalpos=638&amp;ncbi_uid=6463698&amp;link_uid=25089703">Citation</a><a class="dblinks" href="/pmc/articles/PMC4121322/" ref="ordinalpos=638&amp;ncbi_uid=6463698&amp;link_uid=PMC4121322">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>639.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3894178/figure/pgen-1004098-g006/" ref="ordinalpos=639&amp;ncbi_uid=5721150&amp;link_uid=PMC3894178" href="/pmc/articles/PMC3894178/"><img src="/pmc/articles/instance/3894178/bin/pgen.1004098.g006.gif" src-large="/pmc/articles/instance/3894178/bin/pgen.1004098.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3894178/" ref="ordinalpos=639&amp;ncbi_uid=5721150&amp;link_uid=PMC3894178" image-link="/pmc/articles/PMC3894178/figure/pgen-1004098-g006/" class="imagepopup">Figure 6. Schematic view of developmental link between nutrition and sex via JH <span class="highlight" style="background-color:">signaling</span> for sex-specific exaggerated trait development in the stag beetle..  From: Developmental Link between Sex and Nutrition; doublesex Regulates Sex-Specific Mandible Growth via Juvenile Hormone <span class="highlight" style="background-color:">Signaling</span> in Stag Beetles. </a></p><div class="supp"><p class="details"><div>In addition to JH and Dsx, other possible factors were described which might be involved in integration of spatio-temporal information. Nutrition information is mediated by JH <span class="highlight" style="background-color:">signaling</span> and promote mandible enlargement in males. DsxM (CmDsxA and CmDsxB) might play a promoting role for JH-dependent mandible enlargement or recruiting other <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> such as insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (ISS). In females, DsxF (CmDsxC and CmDsxD) inhibit JH effect by reducing the JH sensitivity of mandible cell.</div></p></div><div class="aux"><div class="resc">Hiroki Gotoh, et al. PLoS Genet. 2014 January;10(1):e1004098.</div><p class="links"><a class="dblinks" href="/pubmed/24453990" ref="ordinalpos=639&amp;ncbi_uid=5721150&amp;link_uid=24453990">Citation</a><a class="dblinks" href="/pmc/articles/PMC3894178/" ref="ordinalpos=639&amp;ncbi_uid=5721150&amp;link_uid=PMC3894178">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>640.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407377/figure/F2/" ref="ordinalpos=640&amp;ncbi_uid=4842847&amp;link_uid=PMC3407377" href="/pmc/articles/PMC3407377/"><img src="/pmc/articles/instance/3407377/bin/nihms388183f2.gif" src-large="/pmc/articles/instance/3407377/bin/nihms388183f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407377/" ref="ordinalpos=640&amp;ncbi_uid=4842847&amp;link_uid=PMC3407377" image-link="/pmc/articles/PMC3407377/figure/F2/" class="imagepopup">Fig. 2. Schema of TAK1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: TGF-? <span class="highlight" style="background-color:">signaling</span> via TAK1 <span class="highlight" style="background-color:">pathway</span>: Role in kidney fibrosis. </a></p><div class="supp"><p class="details"><div>Under unstimulated condition, TAK1 associates with TβRI through complex formation with TAB2 and TRAF6. Upon TGF-β1 stimulation and formation of TβRI-TβRII hetero-tetrameric complexes, autopolyubiqitination of TRAF6 leads to polyubiquitination (Ub) of TAK1, triggering release of TAK1 from the receptor complexes. TAK1 interacts with TAB1, which in turn induces autophosphorylation of TAK1. The activated TAK1 transmits TGF-β1 signal to downstream <span class="highlight" style="background-color:">signaling</span> pathways such as the MKK3-p38 or MKK4-JNK cascade, or is rapidly deactivated by phosphatase PP2A. TGF-β1-induced TAK1 activation occurs independent of TβRI kinase activity.</div></p></div><div class="aux"><div class="resc">Mary E. Choi, et al. Semin Nephrol. ;32(3):244-252.</div><p class="links"><a class="dblinks" href="/pubmed/22835455" ref="ordinalpos=640&amp;ncbi_uid=4842847&amp;link_uid=22835455">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407377/" ref="ordinalpos=640&amp;ncbi_uid=4842847&amp;link_uid=PMC3407377">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>641.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3657461/figure/fig3/" ref="ordinalpos=641&amp;ncbi_uid=4964525&amp;link_uid=PMC3657461" href="/pmc/articles/PMC3657461/"><img src="/pmc/articles/instance/3657461/bin/PC2013-920612.003.gif" src-large="/pmc/articles/instance/3657461/bin/PC2013-920612.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3657461/" ref="ordinalpos=641&amp;ncbi_uid=4964525&amp;link_uid=PMC3657461" image-link="/pmc/articles/PMC3657461/figure/fig3/" class="imagepopup">Figure 3.  From: Dissecting Major <span class="highlight" style="background-color:">Signaling</span> Pathways throughout the Development of Prostate Cancer. </a></p><div class="supp"><p class="details"><div>The PI3K/AKT <span class="highlight" style="background-color:">signaling</span> in prostate cancer. PI3K/AKT can induce enhanced activation of cancer cells by direct downstream effects. At the same time, this <span class="highlight" style="background-color:">pathway</span> (and downstream target genes) might affect the action of ERKs, which could lead to inhibition of AR-dependent activation, thus favoring an AR-independent growth. Conversely, AR <span class="highlight" style="background-color:">pathway</span> target genes can limit the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>, favoring an AR-dependent tumor growth. A deregulated PI3K <span class="highlight" style="background-color:">pathway</span> (usually due to mutated or null PTEN) can also inhibit the Ras/MEK/ERK <span class="highlight" style="background-color:">pathway</span>, through enhanced activation of AKT. PI3K/AKT can also enhance the presence of stable metalloproteinase receptors (MT1-MMP), which favors invasive and metastatic phenotypes for these tumor cells. TGF<i>β</i> <span class="highlight" style="background-color:">signaling</span> (through TGF<i>β</i>3 ligation) can have a dual role in PI3K/AKT in PCa cells; for instance, benign cell lines enhance the expression of AKT and subsequent activation of this <span class="highlight" style="background-color:">pathway</span> following TGF<i>β</i>3 engagement; malignant cell lines enhance PTEN expression in response to TGF<i>β</i>3 engagement. Finally, N-cadherin enhanced expression in PCa cells leads to enhanced production of CCL2, which avoids autophagy in part through PI3K/AKT <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Henrique B. da Silva, et al. Prostate Cancer. 2013;2013:920612.</div><p class="links"><a class="dblinks" href="/pubmed/23738079" ref="ordinalpos=641&amp;ncbi_uid=4964525&amp;link_uid=23738079">Citation</a><a class="dblinks" href="/pmc/articles/PMC3657461/" ref="ordinalpos=641&amp;ncbi_uid=4964525&amp;link_uid=PMC3657461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>642.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3046919/figure/F7/" ref="ordinalpos=642&amp;ncbi_uid=2808984&amp;link_uid=PMC3046919" href="/pmc/articles/PMC3046919/"><img src="/pmc/articles/instance/3046919/bin/nihms255077f7.gif" src-large="/pmc/articles/instance/3046919/bin/nihms255077f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3046919/" ref="ordinalpos=642&amp;ncbi_uid=2808984&amp;link_uid=PMC3046919" image-link="/pmc/articles/PMC3046919/figure/F7/" class="imagepopup">Figure 7.  From: Tumor Suppressor A20 Protects against Cardiac Hypertrophy and Fibrosis through Blocking TAK1-Dependent <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed model of the process of A20. In the present study we demonstrated that the inhibitory effects of A20 are achieved by blocking 4 proposed <span class="highlight" style="background-color:">signaling</span> pathways. First, activated TAK1 results in activation of p38 and JNK <span class="highlight" style="background-color:">signaling</span> pathways, enhances hypertrophic markers expression, and subsequently leads to cardiac hypertrophy. Second, TAK1 is shown to promote the Smad <span class="highlight" style="background-color:">signaling</span>, increase expression of fibrotic markers, and result in collagen synthesis and fibrosis. Third, activation of TAK1 leads to <span class="highlight" style="background-color:">signaling</span> through the IKKβ/IκBα/NF-κB <span class="highlight" style="background-color:">pathway</span>, promoting proinflammatory cytokine expression and leading to inflammation. Fourth, hypertrophic stimuli also activate apoptotic <span class="highlight" style="background-color:">signaling</span>, ultimately leading to apoptosis. A20 blocks these proposed TAK1-dependent <span class="highlight" style="background-color:">signaling</span> pathways and apoptotic <span class="highlight" style="background-color:">signaling</span> and then protects against cardiac hypertrophy, fibrosis, inflammation, and apoptosis, finally preventing the progression of cardiac remodeling and heart failure.</div></p></div><div class="aux"><div class="resc">He Huang, et al. Hypertension. ;56(2):232-239.</div><p class="links"><a class="dblinks" href="/pubmed/20585109" ref="ordinalpos=642&amp;ncbi_uid=2808984&amp;link_uid=20585109">Citation</a><a class="dblinks" href="/pmc/articles/PMC3046919/" ref="ordinalpos=642&amp;ncbi_uid=2808984&amp;link_uid=PMC3046919">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>643.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3343135/figure/fig2-1933719110393025/" ref="ordinalpos=643&amp;ncbi_uid=4084205&amp;link_uid=PMC3343135" href="/pmc/articles/PMC3343135/"><img src="/pmc/articles/instance/3343135/bin/10.1177_1933719110393025-fig2.gif" src-large="/pmc/articles/instance/3343135/bin/10.1177_1933719110393025-fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3343135/" ref="ordinalpos=643&amp;ncbi_uid=4084205&amp;link_uid=PMC3343135" image-link="/pmc/articles/PMC3343135/figure/fig2-1933719110393025/" class="imagepopup">Figure 2. From: Maternal Undernourished Fetal Kidneys Exhibit Differential Regulation of Nephrogenic Genes Including Downregulation of the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Differential expression of mRNA and proteins encoding factors in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Arrow direction indicates up or downregulation detected by DNA microarray (open arrows) or by Western blotting (filled arrows). Diagram is a modification of the Kyoto Encyclopedia of Genes and Genome (KEGG) based Notch <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Thomas R. Magee, et al. Reprod Sci. 2011 June;18(6):563-576.</div><p class="links"><a class="dblinks" href="/pubmed/21273641" ref="ordinalpos=643&amp;ncbi_uid=4084205&amp;link_uid=21273641">Citation</a><a class="dblinks" href="/pmc/articles/PMC3343135/" ref="ordinalpos=643&amp;ncbi_uid=4084205&amp;link_uid=PMC3343135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>644.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1386711/figure/F4/" ref="ordinalpos=644&amp;ncbi_uid=669156&amp;link_uid=PMC1386711" href="/pmc/articles/PMC1386711/"><img src="/pmc/articles/instance/1386711/bin/1471-2105-7-41-4.gif" src-large="/pmc/articles/instance/1386711/bin/1471-2105-7-41-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1386711/" ref="ordinalpos=644&amp;ncbi_uid=669156&amp;link_uid=PMC1386711" image-link="/pmc/articles/PMC1386711/figure/F4/" class="imagepopup">Figure 4.  From: Discovering semantic features in the literature: a foundation for building functional associations. </a></p><div class="supp"><p class="details"><div><b>MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mapping</b>. Subset of genes in cluster K (colored in pink) mapped onto the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> diagram for <i>S. cerevisiae </i>(04010sce <span class="highlight" style="background-color:">pathway</span>), as provided in the KEGG <span class="highlight" style="background-color:">PATHWAY</span> database [37].</div></p></div><div class="aux"><div class="resc">Monica Chagoyen, et al. BMC Bioinformatics. 2006;7:41-41.</div><p class="links"><a class="dblinks" href="/pubmed/16438716" ref="ordinalpos=644&amp;ncbi_uid=669156&amp;link_uid=16438716">Citation</a><a class="dblinks" href="/pmc/articles/PMC1386711/" ref="ordinalpos=644&amp;ncbi_uid=669156&amp;link_uid=PMC1386711">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>645.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1764827/figure/F10/" ref="ordinalpos=645&amp;ncbi_uid=794438&amp;link_uid=PMC1764827" href="/pmc/articles/PMC1764827/"><img src="/pmc/articles/instance/1764827/bin/neo0811_0964_fig010.gif" src-large="/pmc/articles/instance/1764827/bin/neo0811_0964_fig010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1764827/" ref="ordinalpos=645&amp;ncbi_uid=794438&amp;link_uid=PMC1764827" image-link="/pmc/articles/PMC1764827/figure/F10/" class="imagepopup">Figure 10.  From: <span class="highlight" style="background-color:">Signaling</span> from p53 to NF-?B Determines the Chemotherapy Responsiveness of Neuroblastoma. </a></p><div class="supp"><p class="details"><div>Dox and VP16 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in NB.</div></p></div><div class="aux"><div class="resc">Michael B Armstrong, et al. Neoplasia. 2006 November;8(11):964-974.</div><p class="links"><a class="dblinks" href="/pubmed/17215959" ref="ordinalpos=645&amp;ncbi_uid=794438&amp;link_uid=17215959">Citation</a><a class="dblinks" href="/pmc/articles/PMC1764827/" ref="ordinalpos=645&amp;ncbi_uid=794438&amp;link_uid=PMC1764827">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>646.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3143377/figure/F3/" ref="ordinalpos=646&amp;ncbi_uid=3636675&amp;link_uid=PMC3143377" href="/pmc/articles/PMC3143377/"><img src="/pmc/articles/instance/3143377/bin/nihms-295616-f0003.gif" src-large="/pmc/articles/instance/3143377/bin/nihms-295616-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3143377/" ref="ordinalpos=646&amp;ncbi_uid=3636675&amp;link_uid=PMC3143377" image-link="/pmc/articles/PMC3143377/figure/F3/" class="imagepopup">Figure 3. Targeting the B-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Molecular Targeted Approaches in Mantle Cell lymphoma.. </a></p><div class="supp"><p class="details"><div>The B-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is initiated through phosphorylation of co-receptors Igα (CD79a) and Igβ (CD79b), recruiting the tyrosine kinase SYK. In turn, SYK phosphorylates several downstream kinases including BTK and PI3Kδ. Activation of PI3Kδ and phospholipase C (PLC) generates PIP3 and diacylglycerol (DAG), respectively. DAG is necessary for activation of protein kinase C (PKC). Inhibitors targeting the <span class="highlight" style="background-color:">signaling</span> cascade downstream of the BCR in clinical development are indicated.</div></p></div><div class="aux"><div class="resc">Marc A. Weniger, et al. Semin Hematol. ;48(3):214-226.</div><p class="links"><a class="dblinks" href="/pubmed/21782064" ref="ordinalpos=646&amp;ncbi_uid=3636675&amp;link_uid=21782064">Citation</a><a class="dblinks" href="/pmc/articles/PMC3143377/" ref="ordinalpos=646&amp;ncbi_uid=3636675&amp;link_uid=PMC3143377">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>647.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726370/figure/F7/" ref="ordinalpos=647&amp;ncbi_uid=5133113&amp;link_uid=PMC3726370" href="/pmc/articles/PMC3726370/"><img src="/pmc/articles/instance/3726370/bin/1479-5876-11-173-7.gif" src-large="/pmc/articles/instance/3726370/bin/1479-5876-11-173-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726370/" ref="ordinalpos=647&amp;ncbi_uid=5133113&amp;link_uid=PMC3726370" image-link="/pmc/articles/PMC3726370/figure/F7/" class="imagepopup">Figure 7.  From: Metallothionein 2A inhibits NF-?B <span class="highlight" style="background-color:">pathway</span> activation and predicts clinical outcome segregated with TNM stage in gastric cancer patients following radical resection. </a></p><div class="supp"><p class="details"><div><b>The potential <span class="highlight" style="background-color:">pathway</span> of MT2A involved in NF-κB <span class="highlight" style="background-color:">signaling</span>.</b> MT2A mediated IκB-α up-regulation to suppress the NF-κB <span class="highlight" style="background-color:">pathway</span> activation. Ectopic expression of MT2A could induce upregulation of IκB-α in mRNA and protein levels, as well as down-regulation of p-IκB-α and cyclin D1expression. Moreover, MT2A could be one of the transcriptional factors to induce the activity of IκB-α promoter, which indicates that MT2A is the negative regulator in NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yuanming Pan, et al. J Transl Med. 2013;11:173-173.</div><p class="links"><a class="dblinks" href="/pubmed/23870553" ref="ordinalpos=647&amp;ncbi_uid=5133113&amp;link_uid=23870553">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726370/" ref="ordinalpos=647&amp;ncbi_uid=5133113&amp;link_uid=PMC3726370">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>648.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2645475/figure/F1/" ref="ordinalpos=648&amp;ncbi_uid=1675279&amp;link_uid=PMC2645475" href="/pmc/articles/PMC2645475/"><img src="/pmc/articles/instance/2645475/bin/nihms72752f1.gif" src-large="/pmc/articles/instance/2645475/bin/nihms72752f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2645475/" ref="ordinalpos=648&amp;ncbi_uid=1675279&amp;link_uid=PMC2645475" image-link="/pmc/articles/PMC2645475/figure/F1/" class="imagepopup">Figure 1. Classical TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Transforming growth factor-beta <span class="highlight" style="background-color:">signaling</span> in thoracic aortic aneurysm development: a paradox in pathogenesis. </a></p><div class="supp"><p class="details"><div>The classical profibrotic TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is initiated upon binding of ligand to a homodimer of the type II TGF-β receptor (TGF-βRII) (<b>1</b>.). The type II receptor is autophosphorylated (<b>2.</b>), which then recruits and transphosphorylates a type I receptor (TGF-βRI) (<b>3.</b>). The activated type I receptor in turn phosphorylates and activates a receptor-Smad (R-Smad) (<b>4.</b>). The R-Smad then binds the common co-Smad (<b>5.</b>) and translocates to the nucleus (<b>6.</b>). Once in the nucleus, it binds transcriptional co-factors and forms an activated transcriptional complex capable of inducing transcription of profibrotic genes (<b>7.</b>).</div></p></div><div class="aux"><div class="resc">Jeffrey A. Jones, et al. J Vasc Res. ;46(2):119-137.</div><p class="links"><a class="dblinks" href="/pubmed/18765947" ref="ordinalpos=648&amp;ncbi_uid=1675279&amp;link_uid=18765947">Citation</a><a class="dblinks" href="/pmc/articles/PMC2645475/" ref="ordinalpos=648&amp;ncbi_uid=1675279&amp;link_uid=PMC2645475">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>649.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3440872/figure/F4/" ref="ordinalpos=649&amp;ncbi_uid=4234267&amp;link_uid=PMC3440872" href="/pmc/articles/PMC3440872/"><img src="/pmc/articles/instance/3440872/bin/nihms347997f4.gif" src-large="/pmc/articles/instance/3440872/bin/nihms347997f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3440872/" ref="ordinalpos=649&amp;ncbi_uid=4234267&amp;link_uid=PMC3440872" image-link="/pmc/articles/PMC3440872/figure/F4/" class="imagepopup">Fig. 4.  From: Regulation of Insulin-Like Growth Factor <span class="highlight" style="background-color:">Signaling</span> by Yap Governs Cardiomyocyte Proliferation and Embryonic Heart Size. </a></p><div class="supp"><p class="details"><div>Model showing a role for Yap in governing cardiac growth and survival by interlinking the Hippo, IGF, and Wnt pathways. Yap promotes cardiomyocyte proliferation through activation of IGF <span class="highlight" style="background-color:">signaling</span> and stabilization of the active form of β-catenin. Inhibition of IGF <span class="highlight" style="background-color:">signaling</span> or β-catenin blocks Yap-induced cardiomyocyte proliferation. Yap interlinks the Hippo <span class="highlight" style="background-color:">pathway</span> (MST1/2 and LATS1/2), the Wnt <span class="highlight" style="background-color:">pathway</span> [Wnt, Disheveled (Dvl), Axin, APC, and GSK-3β], and the IGF <span class="highlight" style="background-color:">pathway</span> (IGF, IGF1R, PI3K, Akt, and GSK-3β) to control β-catenin <span class="highlight" style="background-color:">signaling</span> and regulate cardiac growth and development. Each of the three pathways influenced by Yap is demarcated by a box.</div></p></div><div class="aux"><div class="resc">Mei Xin, et al. Sci Signal. 2011 October 25;4(196):ra70-ra70.</div><p class="links"><a class="dblinks" href="/pubmed/22028467" ref="ordinalpos=649&amp;ncbi_uid=4234267&amp;link_uid=22028467">Citation</a><a class="dblinks" href="/pmc/articles/PMC3440872/" ref="ordinalpos=649&amp;ncbi_uid=4234267&amp;link_uid=PMC3440872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>650.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3402285/figure/fig7/" ref="ordinalpos=650&amp;ncbi_uid=5142170&amp;link_uid=PMC3402285" href="/pmc/articles/PMC3402285/"><img src="/pmc/articles/instance/3402285/bin/ASN.2011111123f7.gif" src-large="/pmc/articles/instance/3402285/bin/ASN.2011111123f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3402285/" ref="ordinalpos=650&amp;ncbi_uid=5142170&amp;link_uid=PMC3402285" image-link="/pmc/articles/PMC3402285/figure/fig7/" class="imagepopup">Figure 7. From: Tissue Plasminogen Activator Activates NF-?B through a <span class="highlight" style="background-color:">Pathway</span> Involving Annexin A2/CD11b and Integrin-Linked Kinase. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span> in tPA-induced NF-κB activation. tPA promotes the interaction and aggregation of annexin A2 and CD11b, inducing the clustering and outside-in <span class="highlight" style="background-color:">signaling</span> of CD11b integrin, which leads to the activation of the downstream ILK, phosphorylation of IκB, and subsequent activation of NF-κB <span class="highlight" style="background-color:">pathway</span>. (A–C) Blockade of any step within this <span class="highlight" style="background-color:">signaling</span> cascade by various strategies eliminates tPA-induced NF-κB activation. (D) Ectopic expression of wild-type ILK mimics tPA effect.</div></p></div><div class="aux"><div class="resc">Ling Lin, et al. J Am Soc Nephrol. 2012 August;23(8):1329-1338.</div><p class="links"><a class="dblinks" href="/pubmed/22677557" ref="ordinalpos=650&amp;ncbi_uid=5142170&amp;link_uid=22677557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3402285/" ref="ordinalpos=650&amp;ncbi_uid=5142170&amp;link_uid=PMC3402285">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>651.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3294424/figure/F6/" ref="ordinalpos=651&amp;ncbi_uid=4663561&amp;link_uid=PMC3294424" href="/pmc/articles/PMC3294424/"><img src="/pmc/articles/instance/3294424/bin/zmr9990922860006.gif" src-large="/pmc/articles/instance/3294424/bin/zmr9990922860006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3294424/" ref="ordinalpos=651&amp;ncbi_uid=4663561&amp;link_uid=PMC3294424" image-link="/pmc/articles/PMC3294424/figure/F6/" class="imagepopup">Fig 6.  From: The Multiple <span class="highlight" style="background-color:">Signaling</span> Systems Regulating Virulence in Pseudomonas aeruginosa. </a></p><div class="supp"><p class="details"><div>PVD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in P. aeruginosa. In the absence of Fe-PVD (left), the <span class="highlight" style="background-color:">signaling</span> system is inactive. Binding of Fe-PVD to the PVD receptor FpvA (right) initiates a <span class="highlight" style="background-color:">signaling</span> cascade that requires TonB1 and FpvR and stimulates the production of FpvA, PVD, ToxA, and the PrpL protease.</div></p></div><div class="aux"><div class="resc">Pol Nadal Jimenez, et al. Microbiol Mol Biol Rev. 2012 March;76(1):46-65.</div><p class="links"><a class="dblinks" href="/pubmed/22390972" ref="ordinalpos=651&amp;ncbi_uid=4663561&amp;link_uid=22390972">Citation</a><a class="dblinks" href="/pmc/articles/PMC3294424/" ref="ordinalpos=651&amp;ncbi_uid=4663561&amp;link_uid=PMC3294424">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>652.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1828071/figure/F2/" ref="ordinalpos=652&amp;ncbi_uid=996329&amp;link_uid=PMC1828071" href="/pmc/articles/PMC1828071/"><img src="/pmc/articles/instance/1828071/bin/nihms16392f2.gif" src-large="/pmc/articles/instance/1828071/bin/nihms16392f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1828071/" ref="ordinalpos=652&amp;ncbi_uid=996329&amp;link_uid=PMC1828071" image-link="/pmc/articles/PMC1828071/figure/F2/" class="imagepopup">Fig. 2.  From: The Role of Intracellular <span class="highlight" style="background-color:">Signaling</span> in Insulin-mediated Regulation of Drug Metabolizing Enzyme Gene and Protein Expression. </a></p><div class="supp"><p class="details"><div>Insulin-mediated activation of PI3K <span class="highlight" style="background-color:">signaling</span> pathways. A diagrammatic representation of insulin/insulin receptor-mediated <span class="highlight" style="background-color:">signaling</span> through the PI3K, PDK1/2 and Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing downstream effects on gene transcription and mRNA translation.</div></p></div><div class="aux"><div class="resc">Sang K. Kim, et al. Pharmacol Ther. ;113(1):88-120.</div><p class="links"><a class="dblinks" href="/pubmed/17097148" ref="ordinalpos=652&amp;ncbi_uid=996329&amp;link_uid=17097148">Citation</a><a class="dblinks" href="/pmc/articles/PMC1828071/" ref="ordinalpos=652&amp;ncbi_uid=996329&amp;link_uid=PMC1828071">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>653.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2422835/figure/F1/" ref="ordinalpos=653&amp;ncbi_uid=1471980&amp;link_uid=PMC2422835" href="/pmc/articles/PMC2422835/"><img src="/pmc/articles/instance/2422835/bin/nihms-50556-f0001.gif" src-large="/pmc/articles/instance/2422835/bin/nihms-50556-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2422835/" ref="ordinalpos=653&amp;ncbi_uid=1471980&amp;link_uid=PMC2422835" image-link="/pmc/articles/PMC2422835/figure/F1/" class="imagepopup">Figure 1. Ceramide synthesis.  From: The sphingolipid salvage <span class="highlight" style="background-color:">pathway</span> in ceramide metabolism and <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The scheme shows metabolic pathways for ceramide synthesis composed of the sphingomyelinase <span class="highlight" style="background-color:">pathway</span>, the <i>de novo</i> <span class="highlight" style="background-color:">pathway</span>, the exogenous ceramide-recycling <span class="highlight" style="background-color:">pathway</span>, and the salvage <span class="highlight" style="background-color:">pathway</span>. Dotted lines indicate the <span class="highlight" style="background-color:">pathway</span> of ceramide synthesis resulting from recycling/salvaging sphingosine.</div></p></div><div class="aux"><div class="resc">Kazuyuki Kitatani, et al. Cell Signal. ;20(6):1010-1018.</div><p class="links"><a class="dblinks" href="/pubmed/18191382" ref="ordinalpos=653&amp;ncbi_uid=1471980&amp;link_uid=18191382">Citation</a><a class="dblinks" href="/pmc/articles/PMC2422835/" ref="ordinalpos=653&amp;ncbi_uid=1471980&amp;link_uid=PMC2422835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>654.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726753/figure/pone-0068985-g002/" ref="ordinalpos=654&amp;ncbi_uid=5168261&amp;link_uid=PMC3726753" href="/pmc/articles/PMC3726753/"><img src="/pmc/articles/instance/3726753/bin/pone.0068985.g002.gif" src-large="/pmc/articles/instance/3726753/bin/pone.0068985.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726753/" ref="ordinalpos=654&amp;ncbi_uid=5168261&amp;link_uid=PMC3726753" image-link="/pmc/articles/PMC3726753/figure/pone-0068985-g002/" class="imagepopup">Figure 2. Simplified mTORC 1 and 2 <span class="highlight" style="background-color:">pathway</span> and the relationship with WT1, EWS-FLI-1 and EWS-WT1..  From: Morphoproteomic Profiling of the Mammalian Target of Rapamycin (mTOR) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Desmoplastic Small Round Cell Tumor (EWS/WT1), Ewing's Sarcoma (EWS/FLI1) and Wilms' Tumor(WT1). </a></p><div class="supp"><p class="details"><div>Morphoproteomic analyses shows a similarity among DSRCT, ES and WT in the constitutive activation of the mTOR <span class="highlight" style="background-color:">pathway</span>, evidenced by the expression of p-mTOR and p-p70S6K. We also found that mTORC2 is the dominant form in both ES and DSRCT, whereas mTORC1 is activated in WT. Our results also suggest a mechanism that may be used to predict response to IGF1R/mTOR inhibitor therapy, which is to compare Akt/mTOR <span class="highlight" style="background-color:">pathway</span> and Ras/Raf/ERK <span class="highlight" style="background-color:">pathway</span> activation. If the IGF1R/mTOR <span class="highlight" style="background-color:">pathway</span> is the dominant <span class="highlight" style="background-color:">pathway</span>, the patient may respond to IGF1R-related therapy.</div></p></div><div class="aux"><div class="resc">Vivek Subbiah, et al. PLoS One. 2013;8(7):e68985.</div><p class="links"><a class="dblinks" href="/pubmed/23922674" ref="ordinalpos=654&amp;ncbi_uid=5168261&amp;link_uid=23922674">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726753/" ref="ordinalpos=654&amp;ncbi_uid=5168261&amp;link_uid=PMC3726753">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>655.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4253421/figure/F7/" ref="ordinalpos=655&amp;ncbi_uid=6900928&amp;link_uid=PMC4253421" href="/pmc/articles/PMC4253421/"><img src="/pmc/articles/instance/4253421/bin/oncotarget-05-9092-g007.gif" src-large="/pmc/articles/instance/4253421/bin/oncotarget-05-9092-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4253421/" ref="ordinalpos=655&amp;ncbi_uid=6900928&amp;link_uid=PMC4253421" image-link="/pmc/articles/PMC4253421/figure/F7/" class="imagepopup">Figure 7. Proposed model of the LGR5-mediated promotion of cervical cancer growth via the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: LGR5 promotes the proliferation and tumor formation of cervical cancer cells through the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Once activated, the LGR5 protein recruits the LRP-Frizzled receptor complex, which binds to Wnt ligands and reinforces Wnt <span class="highlight" style="background-color:">signaling</span> following the phosphorylation of LRP. A series of steps ensue, including the accumulation of β-catenin, which is translocated to the nucleus and, together with the TCF/LEF family of transcription factors, increases related gene expression, ultimately leading to the progression of cervical cancer.</div></p></div><div class="aux"><div class="resc">Qing Chen, et al. Oncotarget. 2014 October;5(19):9092-9105.</div><p class="links"><a class="dblinks" href="/pubmed/25193857" ref="ordinalpos=655&amp;ncbi_uid=6900928&amp;link_uid=25193857">Citation</a><a class="dblinks" href="/pmc/articles/PMC4253421/" ref="ordinalpos=655&amp;ncbi_uid=6900928&amp;link_uid=PMC4253421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>656.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3741305/figure/pone-0070427-g004/" ref="ordinalpos=656&amp;ncbi_uid=5201531&amp;link_uid=PMC3741305" href="/pmc/articles/PMC3741305/"><img src="/pmc/articles/instance/3741305/bin/pone.0070427.g004.gif" src-large="/pmc/articles/instance/3741305/bin/pone.0070427.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3741305/" ref="ordinalpos=656&amp;ncbi_uid=5201531&amp;link_uid=PMC3741305" image-link="/pmc/articles/PMC3741305/figure/pone-0070427-g004/" class="imagepopup">Figure 4. Proposed model for the contribution of EGFR/Ras/MEK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to TGF-β1-induced stemness and tumorigenicity..  From: EMT-Induced Stemness and Tumorigenicity Are Fueled by the EGFR/Ras <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The activation of TGF-β <span class="highlight" style="background-color:">pathway</span> triggers <i>Runx3<sup>−/−</sup>p53<sup>−/−</sup></i> GIF-14 gastric epithelial cells into EMT, concurrently inducing a stem cell-like and tumorigenic state. The data presented in this report show that TGF-β1 induces an EGFR/Ras gene expression signature marked by increased Egfr expression, which sensitized GIF-14 cells to EGF. The activation of EGFR/Ras <span class="highlight" style="background-color:">pathway</span> promotes stemness and tumorigenicity in GIF-14 cells in a MEK1/2-dependent manner that did not involve increased EMT.</div></p></div><div class="aux"><div class="resc">Dominic Chih-Cheng Voon, et al. PLoS One. 2013;8(8):e70427.</div><p class="links"><a class="dblinks" href="/pubmed/23950932" ref="ordinalpos=656&amp;ncbi_uid=5201531&amp;link_uid=23950932">Citation</a><a class="dblinks" href="/pmc/articles/PMC3741305/" ref="ordinalpos=656&amp;ncbi_uid=5201531&amp;link_uid=PMC3741305">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>657.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2759504/figure/pone-0007489-g010/" ref="ordinalpos=657&amp;ncbi_uid=2048671&amp;link_uid=PMC2759504" href="/pmc/articles/PMC2759504/"><img src="/pmc/articles/instance/2759504/bin/pone.0007489.g010.gif" src-large="/pmc/articles/instance/2759504/bin/pone.0007489.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2759504/" ref="ordinalpos=657&amp;ncbi_uid=2048671&amp;link_uid=PMC2759504" image-link="/pmc/articles/PMC2759504/figure/pone-0007489-g010/" class="imagepopup">Figure 10. Model of <span class="highlight" style="background-color:">signaling</span> pathways involved in myometrial ERK activation..  From: Stretch Activates Human Myometrium via ERK, Caldesmon and Focal Adhesion <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>In addition to classical G-protein coupled receptors (GPCR) <span class="highlight" style="background-color:">pathway</span> described by others, data presented here suggest that the activation of ERK by focal adhesion <span class="highlight" style="background-color:">signaling</span> also promotes myometrial contraction and contributes to the initiation of labor. p: phosphorylation.</div></p></div><div class="aux"><div class="resc">Yunping Li, et al. PLoS One. 2009;4(10):e7489.</div><p class="links"><a class="dblinks" href="/pubmed/19834610" ref="ordinalpos=657&amp;ncbi_uid=2048671&amp;link_uid=19834610">Citation</a><a class="dblinks" href="/pmc/articles/PMC2759504/" ref="ordinalpos=657&amp;ncbi_uid=2048671&amp;link_uid=PMC2759504">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>658.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2864807/figure/f2-09_173_.de_re/" ref="ordinalpos=658&amp;ncbi_uid=2328104&amp;link_uid=PMC2864807" href="/pmc/articles/PMC2864807/"><img src="/pmc/articles/instance/2864807/bin/09_173_.de_ref2.gif" src-large="/pmc/articles/instance/2864807/bin/09_173_.de_ref2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2864807/" ref="ordinalpos=658&amp;ncbi_uid=2328104&amp;link_uid=PMC2864807" image-link="/pmc/articles/PMC2864807/figure/f2-09_173_.de_re/" class="imagepopup">Figure 2.  From: PPAR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Cancer-Related Proteins Are Involved in Celiac Disease-Associated Tissue Damage. </a></p><div class="supp"><p class="details"><div>A simplified view of differentially expressed proteins according to PPAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting from <span class="highlight" style="background-color:">Pathway</span> Express tool analysis. All the illustrated proteins were downregulated in Group B (CD patients with Marsh II–III) with respect to Group A (Controls and CD patients with Marsh 0–I). Protein abbreviations correspond to Entrez Gene official symbols.</div></p></div><div class="aux"><div class="resc">Maria Paola Simula, et al. Mol Med. 2010 May-Jun;16(5-6):199-209.</div><p class="links"><a class="dblinks" href="/pubmed/20454521" ref="ordinalpos=658&amp;ncbi_uid=2328104&amp;link_uid=20454521">Citation</a><a class="dblinks" href="/pmc/articles/PMC2864807/" ref="ordinalpos=658&amp;ncbi_uid=2328104&amp;link_uid=PMC2864807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>659.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4134797/figure/fig3/" ref="ordinalpos=659&amp;ncbi_uid=6528907&amp;link_uid=PMC4134797" href="/pmc/articles/PMC4134797/"><img src="/pmc/articles/instance/4134797/bin/JIR2014-197548.003.gif" src-large="/pmc/articles/instance/4134797/bin/JIR2014-197548.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4134797/" ref="ordinalpos=659&amp;ncbi_uid=6528907&amp;link_uid=PMC4134797" image-link="/pmc/articles/PMC4134797/figure/fig3/" class="imagepopup">Figure 3.  From: Cellular <span class="highlight" style="background-color:">Signaling</span> and Production of Galactose-Deficient IgA1 in IgA Nephropathy, an Autoimmune Disease. </a></p><div class="supp"><p class="details"><div>Examples of potential <span class="highlight" style="background-color:">signaling</span> mechanisms involved in the<i> O</i>-glycosylation of IgA1. Several <span class="highlight" style="background-color:">signaling</span> mechanisms likely regulate<i> O</i>-glycosylation patterns of IgA1 in the Golgi apparatus. In this scheme, we have highlighted examples of three types of <span class="highlight" style="background-color:">signaling</span> systems that are potential players in the production of galactose-deficient IgA1 (Gd-IgA1). BAFF <span class="highlight" style="background-color:">signaling</span> through the BAFF receptor(s) (TACI, BAFF-R, and BCMA/TRAF) prevents degradation of NF-<i>κ</i>B and enhances its subsequent translocation to the nucleus. This mechanism represents an example of a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related to survival and proliferation of cells producing Gd-IgA1. Cytokine LIF, that binds the LIF-R/gp130 receptor complex, signals in B cells through STAT1 phosphorylation by JAK, followed by dimerization and translocation of the active STAT dimers to the nucleus. IL-6 activates the STAT3 <span class="highlight" style="background-color:">pathway</span> in B cells via JAK activation at the IL-6/gp130 complex. LIF and IL-6 <span class="highlight" style="background-color:">signaling</span> pathways are examples of a mechanism by which environmental factors may alter expression of specific glycosyltransferases. Activation of the MAPK pathways through various intraextracellular mechanisms can lead directly to the Golgi apparatus. MAPK <span class="highlight" style="background-color:">signaling</span> in B cells is another example of a <span class="highlight" style="background-color:">pathway</span> for an environment-mediated influence to change expression of specific glycosyltransferases, to signal directly to the Golgi apparatus, or both.</div></p></div><div class="aux"><div class="resc">Colin Reily, et al. J Immunol Res. 2014;2014:197548.</div><p class="links"><a class="dblinks" href="/pubmed/25152896" ref="ordinalpos=659&amp;ncbi_uid=6528907&amp;link_uid=25152896">Citation</a><a class="dblinks" href="/pmc/articles/PMC4134797/" ref="ordinalpos=659&amp;ncbi_uid=6528907&amp;link_uid=PMC4134797">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>660.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3458271/figure/F2/" ref="ordinalpos=660&amp;ncbi_uid=4286268&amp;link_uid=PMC3458271" href="/pmc/articles/PMC3458271/"><img src="/pmc/articles/instance/3458271/bin/fphys-03-00376-g0002.gif" src-large="/pmc/articles/instance/3458271/bin/fphys-03-00376-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3458271/" ref="ordinalpos=660&amp;ncbi_uid=4286268&amp;link_uid=PMC3458271" image-link="/pmc/articles/PMC3458271/figure/F2/" class="imagepopup">Figure 2.  From: Ephrin regulation of palate development. </a></p><div class="supp"><p class="details"><div><b>Two proposed models of ephrin-B reverse <span class="highlight" style="background-color:">signaling</span> in palatal EMT based on current evidence.</b> <span class="highlight" style="background-color:">Pathway</span> 1 places ephrin <span class="highlight" style="background-color:">signaling</span> downstream of Tgfß3 <span class="highlight" style="background-color:">signaling</span> such that Tgfß3 stimulates expression of EphBs and/or ephrin-Bs, leading to activation of PI3K <span class="highlight" style="background-color:">signaling</span>. In <span class="highlight" style="background-color:">pathway</span> 2, Tgfß3 and ephrin-B <span class="highlight" style="background-color:">signaling</span> act in parallel to stimulate PI3K together. Pathways 1 and 2 are not mutually exclusive.</div></p></div><div class="aux"><div class="resc">M. Douglas Benson, et al. Front Physiol. 2012;3:376.</div><p class="links"><a class="dblinks" href="/pubmed/23055980" ref="ordinalpos=660&amp;ncbi_uid=4286268&amp;link_uid=23055980">Citation</a><a class="dblinks" href="/pmc/articles/PMC3458271/" ref="ordinalpos=660&amp;ncbi_uid=4286268&amp;link_uid=PMC3458271">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>661.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2928279/figure/pone-0012405-g005/" ref="ordinalpos=661&amp;ncbi_uid=2449689&amp;link_uid=PMC2928279" href="/pmc/articles/PMC2928279/"><img src="/pmc/articles/instance/2928279/bin/pone.0012405.g005.gif" src-large="/pmc/articles/instance/2928279/bin/pone.0012405.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2928279/" ref="ordinalpos=661&amp;ncbi_uid=2449689&amp;link_uid=PMC2928279" image-link="/pmc/articles/PMC2928279/figure/pone-0012405-g005/" class="imagepopup">Figure 5. Correlations between Jak/Stat and PI3K <span class="highlight" style="background-color:">pathway</span> activity with apoptotic responses..  From: Distinct Patterns of DNA Damage Response and Apoptosis Correlate with Jak/Stat and PI3Kinase Response Profiles in Human Acute Myelogenous Leukemia. </a></p><div class="supp"><p class="details"><div>Scatter plots were generated to determine the correlations between <span class="highlight" style="background-color:">pathway</span> activity (PCA analysis of modulated Jak/Stat and PI3K <span class="highlight" style="background-color:">signaling</span>) with apoptosis responses to etoposide and staurosporine. Filled blue circles denote samples from CR patients and unfilled red squares denote samples from NR patients. (A) Scatter plot to show correlations between response to etoposide and staurosporine with exemplary 2D flow plots (B) showing Jak/Stat <span class="highlight" style="background-color:">pathway</span> responses after treatment with a specific modulator. (C) Scatter plots showing correlations between PCA of Jak/Stat <span class="highlight" style="background-color:">pathway</span> activity with either etoposide or staurosporine. (D) Pearson and Spearman correlation coefficients and probabilities show a general inverse correlation between Jak/Stat <span class="highlight" style="background-color:">pathway</span> activity with apoptotic responsiveness to staurosporine but not to etoposide with outliers shown in the plot. (E) Scatter plot to show correlations between response to etoposide and staurosporine with exemplary 2D flow plots (F) showing PI3K <span class="highlight" style="background-color:">pathway</span> responses after treatment with a specific modulator. (G) Scatter plots showing correlations between PCA of PI3K <span class="highlight" style="background-color:">pathway</span> activity with either etoposide or staurosporine. (H) Pearson and Spearman correlation coefficients and probabilities show a general inverse correlation between PI3K <span class="highlight" style="background-color:">pathway</span> activity with apoptotic responsiveness to etoposide and staurosporine with outliers shown in the plot.</div></p></div><div class="aux"><div class="resc">David B. Rosen, et al. PLoS One. 2010;5(8):e12405.</div><p class="links"><a class="dblinks" href="/pubmed/20811632" ref="ordinalpos=661&amp;ncbi_uid=2449689&amp;link_uid=20811632">Citation</a><a class="dblinks" href="/pmc/articles/PMC2928279/" ref="ordinalpos=661&amp;ncbi_uid=2449689&amp;link_uid=PMC2928279">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>662.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4139812/figure/ijms-15-11760-f005/" ref="ordinalpos=662&amp;ncbi_uid=6521938&amp;link_uid=PMC4139812" href="/pmc/articles/PMC4139812/"><img src="/pmc/articles/instance/4139812/bin/ijms-15-11760-g005.gif" src-large="/pmc/articles/instance/4139812/bin/ijms-15-11760-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4139812/" ref="ordinalpos=662&amp;ncbi_uid=6521938&amp;link_uid=PMC4139812" image-link="/pmc/articles/PMC4139812/figure/ijms-15-11760-f005/" class="imagepopup">Figure 5.  From: Paeonol Suppresses Chondrosarcoma Metastasis through Up-Regulation of miR-141 by Modulating PKC? and c-Src <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>c-Src <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in paeonol response of chondrosarcoma. Cells were incubated with various paeonol concentrations for 24 h, c-Src kinase activity and phosphorylation examined by western blot (<i>n</i> = 5) and c-Src kinase activity kit (<i>n</i> = 5). Results are expressed as mean ± S.E.M. <b>*</b>, <i>p</i> &lt; 0.05 compared with control.</div></p></div><div class="aux"><div class="resc">Chi-Ting Horng, et al. Int J Mol Sci. 2014 July;15(7):11760-11772.</div><p class="links"><a class="dblinks" href="/pubmed/24992595" ref="ordinalpos=662&amp;ncbi_uid=6521938&amp;link_uid=24992595">Citation</a><a class="dblinks" href="/pmc/articles/PMC4139812/" ref="ordinalpos=662&amp;ncbi_uid=6521938&amp;link_uid=PMC4139812">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>663.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2675720/figure/fig8/" ref="ordinalpos=663&amp;ncbi_uid=1787737&amp;link_uid=PMC2675720" href="/pmc/articles/PMC2675720/"><img src="/pmc/articles/instance/2675720/bin/pp1500114f08.gif" src-large="/pmc/articles/instance/2675720/bin/pp1500114f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2675720/" ref="ordinalpos=663&amp;ncbi_uid=1787737&amp;link_uid=PMC2675720" image-link="/pmc/articles/PMC2675720/figure/fig8/" class="imagepopup">Figure 8. From: A <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Linking Nitric Oxide Production to Heterotrimeric G Protein and Hydrogen Peroxide Regulates Extracellular Calmodulin Induction of Stomatal Closure in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Model showing the possible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for ExtCaM-induced stomatal closure. ExtCaM activates G protein to induce H<sub>2</sub>O<sub>2</sub> generation by NADPH oxidases, which increases NO production depending on AtNOA1; NO subsequently arouses stomatal closure.</div></p></div><div class="aux"><div class="resc">Jian-Hua Li, et al. Plant Physiol. 2009 May;150(1):114-124.</div><p class="links"><a class="dblinks" href="/pubmed/19321706" ref="ordinalpos=663&amp;ncbi_uid=1787737&amp;link_uid=19321706">Citation</a><a class="dblinks" href="/pmc/articles/PMC2675720/" ref="ordinalpos=663&amp;ncbi_uid=1787737&amp;link_uid=PMC2675720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>664.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2258797/figure/f7/" ref="ordinalpos=664&amp;ncbi_uid=1244260&amp;link_uid=PMC2258797" href="/pmc/articles/PMC2258797/"><img src="/pmc/articles/instance/2258797/bin/zmb0050873260007.gif" src-large="/pmc/articles/instance/2258797/bin/zmb0050873260007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2258797/" ref="ordinalpos=664&amp;ncbi_uid=1244260&amp;link_uid=PMC2258797" image-link="/pmc/articles/PMC2258797/figure/f7/" class="imagepopup">FIG. 7. From: Up-Regulation of P-TEFb by the MEK1-Extracellular Signal-Regulated Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Contributes to Stimulated Transcription Elongation of Immediate Early Genes in Neuroendocrine Cells  . </a></p><div class="supp"><p class="details"><div>Transcription elongation controlled by P-TEFb, NELF, and DSIF through the MEK1-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TRH stimulation activates the MEK1-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, resulting in up-regulation of nuclear CDK9 and cyclin T1/CDK9 complex (i.e., P-TEFb). This up-regulation may be caused by stabilization or translocation of CDK9. The up-regulated P-TEFb is subsequently recruited on IEGs (c-<i>fos</i>, <i>junB</i>, <i>MKP-1</i>, etc.), probably through transcription activators or chromatin-associated proteins. On IEGs, P-TEFb phosphorylates CTR of Spt5, NELF, and CTD Ser-2 of Pol II, favoring transcription elongation and maturation of mRNA by the Pol II complex associated with P-TEFb and DSIF.</div></p></div><div class="aux"><div class="resc">Toshitsugu Fujita, et al. Mol Cell Biol. 2008 March;28(5):1630-1643.</div><p class="links"><a class="dblinks" href="/pubmed/18086894" ref="ordinalpos=664&amp;ncbi_uid=1244260&amp;link_uid=18086894">Citation</a><a class="dblinks" href="/pmc/articles/PMC2258797/" ref="ordinalpos=664&amp;ncbi_uid=1244260&amp;link_uid=PMC2258797">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>665.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2699768/figure/F3/" ref="ordinalpos=665&amp;ncbi_uid=1869708&amp;link_uid=PMC2699768" href="/pmc/articles/PMC2699768/"><img src="/pmc/articles/instance/2699768/bin/nihms98138f3.gif" src-large="/pmc/articles/instance/2699768/bin/nihms98138f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2699768/" ref="ordinalpos=665&amp;ncbi_uid=1869708&amp;link_uid=PMC2699768" image-link="/pmc/articles/PMC2699768/figure/F3/" class="imagepopup">Figure 3. The AhR <span class="highlight" style="background-color:">pathway</span> can alter matrix metabolism and deposition through interactions with other <span class="highlight" style="background-color:">signaling</span> pathways.  From: The aryl hydrocarbon receptor (AhR) <span class="highlight" style="background-color:">pathway</span> as a regulatory <span class="highlight" style="background-color:">pathway</span> for cell adhesion and matrix metabolism. </a></p><div class="supp"><p class="details"><div> Depicted are some of the processes known to take place between the AhR <span class="highlight" style="background-color:">pathway</span>; NFκB, Estrogen Receptor (ER), Retinoic Acid Receptor (RAR) and Transforming Growth Factor β (TGFβ) pathways.</div></p></div><div class="aux"><div class="resc">Tiffany Kung, et al. Biochem Pharmacol. ;77(4):536-546.</div><p class="links"><a class="dblinks" href="/pubmed/18940186" ref="ordinalpos=665&amp;ncbi_uid=1869708&amp;link_uid=18940186">Citation</a><a class="dblinks" href="/pmc/articles/PMC2699768/" ref="ordinalpos=665&amp;ncbi_uid=1869708&amp;link_uid=PMC2699768">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>666.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3757404/figure/f1-cancers-03-02050/" ref="ordinalpos=666&amp;ncbi_uid=5271249&amp;link_uid=PMC3757404" href="/pmc/articles/PMC3757404/"><img src="/pmc/articles/instance/3757404/bin/cancers-03-02050f1.gif" src-large="/pmc/articles/instance/3757404/bin/cancers-03-02050f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3757404/" ref="ordinalpos=666&amp;ncbi_uid=5271249&amp;link_uid=PMC3757404" image-link="/pmc/articles/PMC3757404/figure/f1-cancers-03-02050/" class="imagepopup">Figure 1. From: Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> in Normal and Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div>Schematic Diagram of Wnt/β-catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.</div></p></div><div class="aux"><div class="resc">Kenneth C. Valkenburg, et al. Cancers (Basel). 2011 June;3(2):2050-2079.</div><p class="links"><a class="dblinks" href="/pubmed/24212796" ref="ordinalpos=666&amp;ncbi_uid=5271249&amp;link_uid=24212796">Citation</a><a class="dblinks" href="/pmc/articles/PMC3757404/" ref="ordinalpos=666&amp;ncbi_uid=5271249&amp;link_uid=PMC3757404">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>667.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2878172/figure/F1/" ref="ordinalpos=667&amp;ncbi_uid=2253155&amp;link_uid=PMC2878172" href="/pmc/articles/PMC2878172/"><img src="/pmc/articles/instance/2878172/bin/ijbsv06p0268g01.gif" src-large="/pmc/articles/instance/2878172/bin/ijbsv06p0268g01.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2878172/" ref="ordinalpos=667&amp;ncbi_uid=2253155&amp;link_uid=PMC2878172" image-link="/pmc/articles/PMC2878172/figure/F1/" class="imagepopup">Figure 1.  From: Notch and Wnt <span class="highlight" style="background-color:">Signaling</span>, Physiological Stimuli and Postnatal Myogenesis. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Susan Tsivitse. Int J Biol Sci. 2010;6(3):268-281.</div><p class="links"><a class="dblinks" href="/pubmed/20567496" ref="ordinalpos=667&amp;ncbi_uid=2253155&amp;link_uid=20567496">Citation</a><a class="dblinks" href="/pmc/articles/PMC2878172/" ref="ordinalpos=667&amp;ncbi_uid=2253155&amp;link_uid=PMC2878172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>668.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634073/figure/pone-0061457-g004/" ref="ordinalpos=668&amp;ncbi_uid=4832431&amp;link_uid=PMC3634073" href="/pmc/articles/PMC3634073/"><img src="/pmc/articles/instance/3634073/bin/pone.0061457.g004.gif" src-large="/pmc/articles/instance/3634073/bin/pone.0061457.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634073/" ref="ordinalpos=668&amp;ncbi_uid=4832431&amp;link_uid=PMC3634073" image-link="/pmc/articles/PMC3634073/figure/pone-0061457-g004/" class="imagepopup">Figure 4. Lithium chloride represses Hh <span class="highlight" style="background-color:">signaling</span> activity..  From: Lithium Inhibits Tumorigenic Potential of PDA Cells through Targeting Hedgehog-GLI <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) Real-time PCR analysis of mRNA expression levels of components of the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PANC-1 cells treated with different concentrations of LiCl for 24 h. Data (Mean ± SEM) were representative of three independent experiments. <i>P</i> values are indicated by asterisks (relative to control): *<i>P</i>&lt;0.05, **<i>P</i>&lt;0.01, ***<i>P</i>&lt;0.001. (B) GLI-luciferase activities in AsPC-1 cells transfected with GLI-luciferase reporter and treated with different concentrations of LiCl for 18 h. Data (Mean ± SEM) were representative of three independent experiments. <i>P</i> values are indicated by asterisks (relative to control): *<i>P</i>&lt;0.05, **<i>P</i>&lt;0.01, ***<i>P</i>&lt;0.001.</div></p></div><div class="aux"><div class="resc">Zhonglu Peng, et al. PLoS One. 2013;8(4):e61457.</div><p class="links"><a class="dblinks" href="/pubmed/23626687" ref="ordinalpos=668&amp;ncbi_uid=4832431&amp;link_uid=23626687">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634073/" ref="ordinalpos=668&amp;ncbi_uid=4832431&amp;link_uid=PMC3634073">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>669.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2613669/figure/fig8/" ref="ordinalpos=669&amp;ncbi_uid=1660122&amp;link_uid=PMC2613669" href="/pmc/articles/PMC2613669/"><img src="/pmc/articles/instance/2613669/bin/tpc2002989f08.gif" src-large="/pmc/articles/instance/2613669/bin/tpc2002989f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2613669/" ref="ordinalpos=669&amp;ncbi_uid=1660122&amp;link_uid=PMC2613669" image-link="/pmc/articles/PMC2613669/figure/fig8/" class="imagepopup">Figure 8. From: Arbuscular Mycorrhiza-Specific <span class="highlight" style="background-color:">Signaling</span> in Rice Transcends the Common Symbiosis <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>AM-Specific <span class="highlight" style="background-color:">Signaling</span> Pathways Including Bifurcation at the Ca<sup>2+</sup> Response.<br />The common SYM <span class="highlight" style="background-color:">pathway</span> displaying the components analyzed within this study consists of CASTOR and POLLUX upstream and CCAMK and CYCLOPS downstream of Ca<sup>2+</sup> spiking. Expression of <i>AM11</i> is induced early and relies on an intact SYM <span class="highlight" style="background-color:">pathway</span>. Induction of systemically (S; gray arrows) induced genes, <i>AM3</i> and <i>AM34</i>, and the late (L; black arrows) induced gene, <i>AM14</i>, relies on an intact Ca<sup>2+</sup> response and thus requires CASTOR, POLLUX, and CCAMK but is independent of CYCLOPS. Expression of <i>PT11</i> is dependent on arbuscule formation. The dependence of <i>AM10</i> and <i>AM15</i> on SYM <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> or on arbuscule formation is not known (dashed arrows with question marks). <i>AM1</i> and <i>AM2</i> are induced early but do not require functional CASTOR, POLLUX, CCAMK, or CYCLOPS, demonstrating the existence of an alternative AM <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (AP).</div></p></div><div class="aux"><div class="resc">Caroline Gutjahr, et al. Plant Cell. 2008 November;20(11):2989-3005.</div><p class="links"><a class="dblinks" href="/pubmed/19033527" ref="ordinalpos=669&amp;ncbi_uid=1660122&amp;link_uid=19033527">Citation</a><a class="dblinks" href="/pmc/articles/PMC2613669/" ref="ordinalpos=669&amp;ncbi_uid=1660122&amp;link_uid=PMC2613669">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>670.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3956974/figure/Fig3/" ref="ordinalpos=670&amp;ncbi_uid=5936284&amp;link_uid=PMC3956974" href="/pmc/articles/PMC3956974/"><img src="/pmc/articles/instance/3973051/bin/13238_2013_13_Fig3_HTML.gif" src-large="/pmc/articles/instance/3973051/bin/13238_2013_13_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3956974/" ref="ordinalpos=670&amp;ncbi_uid=5936284&amp;link_uid=PMC3956974" image-link="/pmc/articles/PMC3956974/figure/Fig3/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Signaling</span> control of the constitutive androstane receptor (CAR). </a></p><div class="supp"><p class="details"><div> <b><span class="highlight" style="background-color:">Signaling</span> control of CAR activation</b>. Chemicals illustrating activation or deactivation of a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are denoted in blue and red, respectively</div></p></div><div class="aux"><div class="resc">Hui Yang, et al. Protein Cell. 2014 February;5(2):113-123.</div><p class="links"><a class="dblinks" href="/pubmed/24474196" ref="ordinalpos=670&amp;ncbi_uid=5936284&amp;link_uid=24474196">Citation</a><a class="dblinks" href="/pmc/articles/PMC3956974/" ref="ordinalpos=670&amp;ncbi_uid=5936284&amp;link_uid=PMC3956974">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>671.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4288643/figure/Fig6/" ref="ordinalpos=671&amp;ncbi_uid=7054229&amp;link_uid=PMC4288643" href="/pmc/articles/PMC4288643/"><img src="/pmc/articles/instance/4288643/bin/12906_2014_2017_Fig6_HTML.gif" src-large="/pmc/articles/instance/4288643/bin/12906_2014_2017_Fig6_HTML.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4288643/" ref="ordinalpos=671&amp;ncbi_uid=7054229&amp;link_uid=PMC4288643" image-link="/pmc/articles/PMC4288643/figure/Fig6/" class="imagepopup">Figure 6.  From: Zuo Jin Wan reverses P-gp-mediated drug-resistance by inhibiting activation of the PI3K/Akt/NF-κB <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>Schematic summary of the inhibition mechanisms of ZJW on the PI3K/Akt/NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Activation of PI3K/Akt/NF-κB <span class="highlight" style="background-color:">pathway</span> could reduce MDR phenotype. ZJW suppress phosphorylation of Akt (Ser473) and NF-κB.</div></p></div><div class="aux"><div class="resc">Hua Sui, et al. BMC Complement Altern Med. 2014;14(1):279.</div><p class="links"><a class="dblinks" href="/pubmed/25085593" ref="ordinalpos=671&amp;ncbi_uid=7054229&amp;link_uid=25085593">Citation</a><a class="dblinks" href="/pmc/articles/PMC4288643/" ref="ordinalpos=671&amp;ncbi_uid=7054229&amp;link_uid=PMC4288643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>672.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3268901/figure/F1/" ref="ordinalpos=672&amp;ncbi_uid=4486775&amp;link_uid=PMC3268901" href="/pmc/articles/PMC3268901/"><img src="/pmc/articles/instance/3268901/bin/nihms-343797-f0001.gif" src-large="/pmc/articles/instance/3268901/bin/nihms-343797-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3268901/" ref="ordinalpos=672&amp;ncbi_uid=4486775&amp;link_uid=PMC3268901" image-link="/pmc/articles/PMC3268901/figure/F1/" class="imagepopup">Figure 1.  From: Exercise modulates microRNAs that affect the PTEN/mTOR <span class="highlight" style="background-color:">pathway</span> in rats after spinal cord injury. </a></p><div class="supp"><p class="details"><div>Schematic of the PTEN/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Gang Liu, et al. Exp Neurol. ;233(1):447-456.</div><p class="links"><a class="dblinks" href="/pubmed/22123082" ref="ordinalpos=672&amp;ncbi_uid=4486775&amp;link_uid=22123082">Citation</a><a class="dblinks" href="/pmc/articles/PMC3268901/" ref="ordinalpos=672&amp;ncbi_uid=4486775&amp;link_uid=PMC3268901">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>673.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4105941/figure/pcbi-1003573-g001/" ref="ordinalpos=673&amp;ncbi_uid=6409196&amp;link_uid=PMC4105941" href="/pmc/articles/PMC4105941/"><img src="/pmc/articles/instance/4105941/bin/pcbi.1003573.g001.gif" src-large="/pmc/articles/instance/4105941/bin/pcbi.1003573.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4105941/" ref="ordinalpos=673&amp;ncbi_uid=6409196&amp;link_uid=PMC4105941" image-link="/pmc/articles/PMC4105941/figure/pcbi-1003573-g001/" class="imagepopup">Figure 1. The <span class="highlight" style="background-color:">pathway</span> diagram of Smad <span class="highlight" style="background-color:">signaling</span> (using symbols from BioCarta)..  From: The Self-Limiting Dynamics of TGF-? <span class="highlight" style="background-color:">Signaling</span> In Silico and In Vitro, with Negative Feedback through PPM1A Upregulation. </a></p><div class="supp"><p class="details"><div>The dashed arrows indicate those reactions which are modeled in black box. The red arrows indicate the negative regulatory effects: (1) PPM1A dephosphorylating phospho-R-Smad; (2) Smurf2 induced proteasome degradation of phospho-R-Smad; (3) I-Smad induced receptor inhibition; (4) PPM1A upregulation by Smad <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Junjie Wang, et al. PLoS Comput Biol. 2014 June;10(6):e1003573.</div><p class="links"><a class="dblinks" href="/pubmed/24901250" ref="ordinalpos=673&amp;ncbi_uid=6409196&amp;link_uid=24901250">Citation</a><a class="dblinks" href="/pmc/articles/PMC4105941/" ref="ordinalpos=673&amp;ncbi_uid=6409196&amp;link_uid=PMC4105941">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>674.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4120774/figure/F4/" ref="ordinalpos=674&amp;ncbi_uid=6460828&amp;link_uid=PMC4120774" href="/pmc/articles/PMC4120774/"><img src="/pmc/articles/instance/4120774/bin/nihms-314392-f0004.gif" src-large="/pmc/articles/instance/4120774/bin/nihms-314392-f0004.jpg" alt="Figure - 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4120774/" ref="ordinalpos=674&amp;ncbi_uid=6460828&amp;link_uid=PMC4120774" image-link="/pmc/articles/PMC4120774/figure/F4/" class="imagepopup">Figure - 4. Effect of DIM on the MAPK <span class="highlight" style="background-color:">signaling</span> mechanisms.  From: ATTENUATION OF MULTI-TARGETED PROLIFERATION-LINKED <span class="highlight" style="background-color:">SIGNALING</span> BY 3,3?-DIINDOLYLMETHANE (DIM): FROM BENCH TO CLINIC. </a></p><div class="supp"><p class="details"><div>This <span class="highlight" style="background-color:">pathway</span> has been shown to be inhibited by DIM; these kinases transduce the signal by phosphorylating downstream protein kinases. Some of the phosphorylated <span class="highlight" style="background-color:">signaling</span> molecules translocate to the nucleus to activate transcription factors (such as AP-1) which regulate the expression of select set of genes.</div></p></div><div class="aux"><div class="resc">Sanjeev Banerjee, et al. Mutat Res. ;728(0):47-66.</div><p class="links"><a class="dblinks" href="/pubmed/21703360" ref="ordinalpos=674&amp;ncbi_uid=6460828&amp;link_uid=21703360">Citation</a><a class="dblinks" href="/pmc/articles/PMC4120774/" ref="ordinalpos=674&amp;ncbi_uid=6460828&amp;link_uid=PMC4120774">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>675.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3928074/figure/F1/" ref="ordinalpos=675&amp;ncbi_uid=5824990&amp;link_uid=PMC3928074" href="/pmc/articles/PMC3928074/"><img src="/pmc/articles/instance/3928074/bin/nihms550827f1.gif" src-large="/pmc/articles/instance/3928074/bin/nihms550827f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3928074/" ref="ordinalpos=675&amp;ncbi_uid=5824990&amp;link_uid=PMC3928074" image-link="/pmc/articles/PMC3928074/figure/F1/" class="imagepopup">Figure 1. VEGF <span class="highlight" style="background-color:">Pathway</span> in CLL.  From: Phase II Trials of Single Agent Anti-VEGF Therapy for Patients with Chronic Lymphocytic Leukemia (CLL). </a></p><div class="supp"><p class="details"><div>Figure shows scheme of VEGF receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in CLL as well as aspects of <span class="highlight" style="background-color:">pathway</span> targeted by bevacizumab, AZD2171, and sunitinib malate.</div></p></div><div class="aux"><div class="resc">Tait Shanafelt, et al. Leuk Lymphoma. ;51(12):2222-2229.</div><p class="links"><a class="dblinks" href="/pubmed/21054149" ref="ordinalpos=675&amp;ncbi_uid=5824990&amp;link_uid=21054149">Citation</a><a class="dblinks" href="/pmc/articles/PMC3928074/" ref="ordinalpos=675&amp;ncbi_uid=5824990&amp;link_uid=PMC3928074">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>676.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2562168/figure/F2/" ref="ordinalpos=676&amp;ncbi_uid=1551013&amp;link_uid=PMC2562168" href="/pmc/articles/PMC2562168/"><img src="/pmc/articles/instance/2562168/bin/nihms67756f2.gif" src-large="/pmc/articles/instance/2562168/bin/nihms67756f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2562168/" ref="ordinalpos=676&amp;ncbi_uid=1551013&amp;link_uid=PMC2562168" image-link="/pmc/articles/PMC2562168/figure/F2/" class="imagepopup">Figure 2. Crosstalk <span class="highlight" style="background-color:">signaling</span> is switch-like.  From: Membrane localization of scaffold proteins promotes graded <span class="highlight" style="background-color:">signaling</span> in the yeast MAP kinase cascade. </a></p><div class="supp"><p class="details"><div>A. Crosstalk between the mating and the high osmolarity glycerol (HOG) <span class="highlight" style="background-color:">pathway</span>. Although these pathways are normally separate, high osmolarity stimuli will trigger activation of the mating <span class="highlight" style="background-color:">pathway</span> when HOG <span class="highlight" style="background-color:">pathway</span> output is disrupted (e.g., in <i>hog1</i>Δ cells). <span class="highlight" style="background-color:">Pathway</span> components necessary for this phenomenon are shown in red.<br />B. Crosstalk <span class="highlight" style="background-color:">signaling</span> is switch-like. <i>P<sub>FUS1</sub>-GFP</i> expression was analyzed in <i>hog1</i>Δ <i>ste5</i>Δ cells (PPY1372) treated with 0–1 M sorbitol.<br />C. Stimulus identity influences <span class="highlight" style="background-color:">signaling</span> behavior. Top: <i>P<sub>FUS1</sub>-GFP</i> FACS profiles of <i>hog1</i>Δ <i>STE5</i> cells (PPY1825) treated with α factor or sorbitol. Bottom: responses in the same strains were averaged over the entire cell population, using a <i>FUS1-lacZ</i> reporter gene and β-galactosidase assays (mean ± SD; n=3).</div></p></div><div class="aux"><div class="resc">Satoe Takahashi, et al. Curr Biol. ;18(16):1184-1191.</div><p class="links"><a class="dblinks" href="/pubmed/18722124" ref="ordinalpos=676&amp;ncbi_uid=1551013&amp;link_uid=18722124">Citation</a><a class="dblinks" href="/pmc/articles/PMC2562168/" ref="ordinalpos=676&amp;ncbi_uid=1551013&amp;link_uid=PMC2562168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>677.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1609106/figure/F8/" ref="ordinalpos=677&amp;ncbi_uid=883184&amp;link_uid=PMC1609106" href="/pmc/articles/PMC1609106/"><img src="/pmc/articles/instance/1609106/bin/1471-213X-6-45-8.gif" src-large="/pmc/articles/instance/1609106/bin/1471-213X-6-45-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1609106/" ref="ordinalpos=677&amp;ncbi_uid=883184&amp;link_uid=PMC1609106" image-link="/pmc/articles/PMC1609106/figure/F8/" class="imagepopup">Figure 8.  From: Activated AKT/PKB <span class="highlight" style="background-color:">signaling</span> in C. elegans uncouples temporally distinct outputs of DAF-2/insulin-like <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model for regulation of insulin <span class="highlight" style="background-color:">pathway</span> phenotypes by AKT-1 and PDK-1 activation</b>. Activated <span class="highlight" style="background-color:">signaling</span> components in the absence of AKT-1 <span class="highlight" style="background-color:">signaling</span> in <i>age-1(mg109) </i>mutant background. See text for details.</div></p></div><div class="aux"><div class="resc">Minaxi S Gami, et al. BMC Dev Biol. 2006;6:45-45.</div><p class="links"><a class="dblinks" href="/pubmed/17020605" ref="ordinalpos=677&amp;ncbi_uid=883184&amp;link_uid=17020605">Citation</a><a class="dblinks" href="/pmc/articles/PMC1609106/" ref="ordinalpos=677&amp;ncbi_uid=883184&amp;link_uid=PMC1609106">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>678.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3182966/figure/F3/" ref="ordinalpos=678&amp;ncbi_uid=3253723&amp;link_uid=PMC3182966" href="/pmc/articles/PMC3182966/"><img src="/pmc/articles/instance/3182966/bin/1477-5956-9-53-3.gif" src-large="/pmc/articles/instance/3182966/bin/1477-5956-9-53-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3182966/" ref="ordinalpos=678&amp;ncbi_uid=3253723&amp;link_uid=PMC3182966" image-link="/pmc/articles/PMC3182966/figure/F3/" class="imagepopup">Figure 3.  From: Improved protein arrays for quantitative systems analysis of the dynamics of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> interactions. </a></p><div class="supp"><p class="details"><div><b>Specificity of Qdot-RPPA</b>. <b>A</b>. The regulation of AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Calyculin A is a phosphotase inhibitor leading to phospho-AKT accumulation. LY294002 inhibits PI3K phosphorylation therefore blocks the activation of downstream AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>B</b>. Cells were treated with inhibitors or serum as indicated to alter AKT activity. RPPA probed with either total AKT or phospho-AKT antibodies. All the samples were spotted in triplicates with 5 serial dilutions. <b>C</b>. Corresponding Western blots using the same lysates in <b>B</b>.</div></p></div><div class="aux"><div class="resc">Xiaoyu Wang, et al. Proteome Sci. 2011;9:53-53.</div><p class="links"><a class="dblinks" href="/pubmed/21917185" ref="ordinalpos=678&amp;ncbi_uid=3253723&amp;link_uid=21917185">Citation</a><a class="dblinks" href="/pmc/articles/PMC3182966/" ref="ordinalpos=678&amp;ncbi_uid=3253723&amp;link_uid=PMC3182966">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>679.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2890502/figure/F4/" ref="ordinalpos=679&amp;ncbi_uid=2346504&amp;link_uid=PMC2890502" href="/pmc/articles/PMC2890502/"><img src="/pmc/articles/instance/2890502/bin/1752-0509-4-72-4.gif" src-large="/pmc/articles/instance/2890502/bin/1752-0509-4-72-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2890502/" ref="ordinalpos=679&amp;ncbi_uid=2346504&amp;link_uid=PMC2890502" image-link="/pmc/articles/PMC2890502/figure/F4/" class="imagepopup">Figure 4.  From: An information-flow-based model with dissipation, saturation and direction for active <span class="highlight" style="background-color:">pathway</span> inference. </a></p><div class="supp"><p class="details"><div><b>Pathways between Gpa1 to Prp39 predicted by our method</b>. Two pathways are inferred. <span class="highlight" style="background-color:">Pathway</span> 1 is the pheromone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> identified by both Tu <i>et al.</i>'s method and the information flow model. <span class="highlight" style="background-color:">Pathway</span> 2 is identified only by the information flow model. It regulates the recovery from the pheromone arrest.</div></p></div><div class="aux"><div class="resc">Xianwen Ren, et al. BMC Syst Biol. 2010;4:72-72.</div><p class="links"><a class="dblinks" href="/pubmed/20504374" ref="ordinalpos=679&amp;ncbi_uid=2346504&amp;link_uid=20504374">Citation</a><a class="dblinks" href="/pmc/articles/PMC2890502/" ref="ordinalpos=679&amp;ncbi_uid=2346504&amp;link_uid=PMC2890502">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>680.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3749067/figure/F1/" ref="ordinalpos=680&amp;ncbi_uid=5550519&amp;link_uid=PMC3749067" href="/pmc/articles/PMC3749067/"><img src="/pmc/articles/instance/3749067/bin/nihms497727f1.gif" src-large="/pmc/articles/instance/3749067/bin/nihms497727f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3749067/" ref="ordinalpos=680&amp;ncbi_uid=5550519&amp;link_uid=PMC3749067" image-link="/pmc/articles/PMC3749067/figure/F1/" class="imagepopup">Fig. 1.  From: A Role for WNT/?-Catenin <span class="highlight" style="background-color:">Signaling</span> in the Neural Mechanisms of Behavior. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Kimberly A. Maguschak, et al. J Neuroimmune Pharmacol. ;7(4):763-773.</div><p class="links"><a class="dblinks" href="/pubmed/22415718" ref="ordinalpos=680&amp;ncbi_uid=5550519&amp;link_uid=22415718">Citation</a><a class="dblinks" href="/pmc/articles/PMC3749067/" ref="ordinalpos=680&amp;ncbi_uid=5550519&amp;link_uid=PMC3749067">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>681.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2288618/figure/F7/" ref="ordinalpos=681&amp;ncbi_uid=1271220&amp;link_uid=PMC2288618" href="/pmc/articles/PMC2288618/"><img src="/pmc/articles/instance/2288618/bin/nihms43701f7.gif" src-large="/pmc/articles/instance/2288618/bin/nihms43701f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2288618/" ref="ordinalpos=681&amp;ncbi_uid=1271220&amp;link_uid=PMC2288618" image-link="/pmc/articles/PMC2288618/figure/F7/" class="imagepopup">Fig. 7. Model of NOD2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: TAK1 is a central mediator of NOD2 <span class="highlight" style="background-color:">signaling</span> in epidermal cells. </a></p><div class="aux"><div class="resc">Jae Young Kim, et al. J Biol Chem. ;283(1):137-144.</div><p class="links"><a class="dblinks" href="/pubmed/17965022" ref="ordinalpos=681&amp;ncbi_uid=1271220&amp;link_uid=17965022">Citation</a><a class="dblinks" href="/pmc/articles/PMC2288618/" ref="ordinalpos=681&amp;ncbi_uid=1271220&amp;link_uid=PMC2288618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>682.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4063740/figure/pntd-0002949-g008/" ref="ordinalpos=682&amp;ncbi_uid=6289477&amp;link_uid=PMC4063740" href="/pmc/articles/PMC4063740/"><img src="/pmc/articles/instance/4063740/bin/pntd.0002949.g008.gif" src-large="/pmc/articles/instance/4063740/bin/pntd.0002949.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4063740/" ref="ordinalpos=682&amp;ncbi_uid=6289477&amp;link_uid=PMC4063740" image-link="/pmc/articles/PMC4063740/figure/pntd-0002949-g008/" class="imagepopup">Figure 8. Hypothetical S. mansoni MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Regulation of Schistosoma mansoni Development and Reproduction by the Mitogen-Activated Protein Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) EGF activates the Ras/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. EGFR transmits the signal to the intracellular environment through the activation of Ras and sequential phosphorylation of SmRAF (ePK of TKL group and raf family), SmSTE7 (ePK of STE group and STE7family) and SmERK (ePK of CMGC group, MAPK family and ERK1/2 subfamily). Activated ERK translocates to the nucleus and inhibits the formation of the elk1/SRF complex and, in this case, oviposition remains constant. In B and C, SmRas or SmERK activity is interfered with (via RNAi) and the signal is not transmitted. Elk1 forms a complex with SRF which binds to the c-Fos promoter and this initiates c-Fos transcription that subsequently prevents the egg laying.</div></p></div><div class="aux"><div class="resc">Luiza Freire de Andrade, et al. PLoS Negl Trop Dis. 2014 June;8(6):e2949.</div><p class="links"><a class="dblinks" href="/pubmed/24945272" ref="ordinalpos=682&amp;ncbi_uid=6289477&amp;link_uid=24945272">Citation</a><a class="dblinks" href="/pmc/articles/PMC4063740/" ref="ordinalpos=682&amp;ncbi_uid=6289477&amp;link_uid=PMC4063740">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>683.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2649571/figure/pone-0004690-g007/" ref="ordinalpos=683&amp;ncbi_uid=1729774&amp;link_uid=PMC2649571" href="/pmc/articles/PMC2649571/"><img src="/pmc/articles/instance/2649571/bin/pone.0004690.g007.gif" src-large="/pmc/articles/instance/2649571/bin/pone.0004690.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2649571/" ref="ordinalpos=683&amp;ncbi_uid=1729774&amp;link_uid=PMC2649571" image-link="/pmc/articles/PMC2649571/figure/pone-0004690-g007/" class="imagepopup">Figure 7. One possible model of the <span class="highlight" style="background-color:">signaling</span> pathways mediating axon guidance in response to Wnt..  From: A ?-Catenin-Dependent Wnt <span class="highlight" style="background-color:">Pathway</span> Mediates Anteroposterior Axon Guidance in C. elegans Motor Neurons. </a></p><div class="supp"><p class="details"><div>Because <i>lin-17</i> mutants have both underextension and overextension defects, we propose that LIN-17/Fz can activate both a canonical <span class="highlight" style="background-color:">pathway</span> that attracts axons (or prevents axon termination) (left) and a yet uncharacterized <span class="highlight" style="background-color:">pathway</span> that repels axons (or triggers axon termination) (right). The existence of this additional <span class="highlight" style="background-color:">pathway</span> and its interaction with the canonical <span class="highlight" style="background-color:">pathway</span> are supported by the analysis of <i>lin-17</i>; <i>bar-1</i> double mutants (see Figure 5). According to this model and consistent with our data, both the activation of the uncharacterized <span class="highlight" style="background-color:">pathway</span> and the inhibition of the canonical <span class="highlight" style="background-color:">pathway</span> by LIN-44/Wnt are required for the axons to stop in their correct position.</div></p></div><div class="aux"><div class="resc">Géraldine S. Maro, et al. PLoS ONE. 2009;4(3):e4690.</div><p class="links"><a class="dblinks" href="/pubmed/19259273" ref="ordinalpos=683&amp;ncbi_uid=1729774&amp;link_uid=19259273">Citation</a><a class="dblinks" href="/pmc/articles/PMC2649571/" ref="ordinalpos=683&amp;ncbi_uid=1729774&amp;link_uid=PMC2649571">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>684.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1287678/figure/f9/" ref="ordinalpos=684&amp;ncbi_uid=606184&amp;link_uid=PMC1287678" href="/pmc/articles/PMC1287678/"><img src="/pmc/articles/instance/1287678/bin/zam0110560540009.gif" src-large="/pmc/articles/instance/1287678/bin/zam0110560540009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1287678/" ref="ordinalpos=684&amp;ncbi_uid=606184&amp;link_uid=PMC1287678" image-link="/pmc/articles/PMC1287678/figure/f9/" class="imagepopup">FIG. 9. From: Identifying Components of the NF-?B <span class="highlight" style="background-color:">Pathway</span> in the Beneficial Euprymna scolopes-Vibrio fischeri Light Organ Symbiosis . </a></p><div class="supp"><p class="details"><div>Putative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of the evolutionarily conserved Toll/NF-κB <span class="highlight" style="background-color:">pathway</span> utilizing the <i>E. scolopes</i> transcripts identified in the present study. Black shading indicates identification of full-length transcripts, and gray shading indicates identification of partial length transcripts (see Tables 1 and 2). Dashed lines indicate a <span class="highlight" style="background-color:">pathway</span> in which not all of the <span class="highlight" style="background-color:">signaling</span> intermediates have been found in <i>E. scolopes</i>.</div></p></div><div class="aux"><div class="resc">Michael S. Goodson, et al. Appl Environ Microbiol. 2005 November;71(11):6934-6946.</div><p class="links"><a class="dblinks" href="/pubmed/16269728" ref="ordinalpos=684&amp;ncbi_uid=606184&amp;link_uid=16269728">Citation</a><a class="dblinks" href="/pmc/articles/PMC1287678/" ref="ordinalpos=684&amp;ncbi_uid=606184&amp;link_uid=PMC1287678">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>685.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052372/figure/F1/" ref="ordinalpos=685&amp;ncbi_uid=6237508&amp;link_uid=PMC4052372" href="/pmc/articles/PMC4052372/"><img src="/pmc/articles/instance/4052372/bin/nihms582555f1.gif" src-large="/pmc/articles/instance/4052372/bin/nihms582555f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052372/" ref="ordinalpos=685&amp;ncbi_uid=6237508&amp;link_uid=PMC4052372" image-link="/pmc/articles/PMC4052372/figure/F1/" class="imagepopup">Fig. 1.  From: Wnt <span class="highlight" style="background-color:">Signaling</span> in Angiogenesis. </a></p><div class="supp"><p class="details"><div>Schematic of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that responds to Norrin interaction with Frizzled4.</div></p></div><div class="aux"><div class="resc">Nancy L. Parmalee, et al. Curr Drug Targets. ;9(7):558-564.</div><p class="links"><a class="dblinks" href="/pubmed/18673241" ref="ordinalpos=685&amp;ncbi_uid=6237508&amp;link_uid=18673241">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052372/" ref="ordinalpos=685&amp;ncbi_uid=6237508&amp;link_uid=PMC4052372">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>686.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3854132/figure/pcbi-1003358-g001/" ref="ordinalpos=686&amp;ncbi_uid=5593907&amp;link_uid=PMC3854132" href="/pmc/articles/PMC3854132/"><img src="/pmc/articles/instance/3854132/bin/pcbi.1003358.g001.gif" src-large="/pmc/articles/instance/3854132/bin/pcbi.1003358.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3854132/" ref="ordinalpos=686&amp;ncbi_uid=5593907&amp;link_uid=PMC3854132" image-link="/pmc/articles/PMC3854132/figure/pcbi-1003358-g001/" class="imagepopup">Figure 1. BAD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of apoptosis regulation in prostate cancer cells, mediated by psychological stress..  From: Systems Modeling of Anti-apoptotic Pathways in Prostate Cancer: Psychological Stress Triggers a Synergism Pattern Switch in Drug Combination Therapy. </a></p><div class="supp"><p class="details"><div>Phosphorylated BAD is critical for the anti-apoptotic effect of multiple <span class="highlight" style="background-color:">signaling</span> pathways, while dephosphorylated BAD has a pro-apoptotic effect. Stress, EGF, and PI3K can activate independent <span class="highlight" style="background-color:">signaling</span> pathways that phosphorylate BAD. Stress, represented by epinephrine (or VIP) can promote the phosphorylation of BAD at Ser<sup>112</sup> via activation of cAMP and PKA. EGF phosphorylates BAD at Ser<sup>112</sup> through the Ras/Raf-MEK/ERK-KinaseX <span class="highlight" style="background-color:">pathway</span> and at Ser<sup>136</sup> through the Rac-PAK <span class="highlight" style="background-color:">pathway</span>. PI3K transmits signals to Ser<sup>136</sup> through AKT activation, and AKT can partially regulate Ser<sup>112</sup>. The anti-apoptotic role of the activation of CREB and Mcl-1 induced by stress was determined by comparing the predictive power of different models. Multiple drugs targeting different <span class="highlight" style="background-color:">signaling</span> pathways were integrated into the model.</div></p></div><div class="aux"><div class="resc">Xiaoqiang Sun, et al. PLoS Comput Biol. 2013 December;9(12):e1003358.</div><p class="links"><a class="dblinks" href="/pubmed/24339759" ref="ordinalpos=686&amp;ncbi_uid=5593907&amp;link_uid=24339759">Citation</a><a class="dblinks" href="/pmc/articles/PMC3854132/" ref="ordinalpos=686&amp;ncbi_uid=5593907&amp;link_uid=PMC3854132">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>687.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281071/figure/pone-0031229-g004/" ref="ordinalpos=687&amp;ncbi_uid=3816276&amp;link_uid=PMC3281071" href="/pmc/articles/PMC3281071/"><img src="/pmc/articles/instance/3281071/bin/pone.0031229.g004.gif" src-large="/pmc/articles/instance/3281071/bin/pone.0031229.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281071/" ref="ordinalpos=687&amp;ncbi_uid=3816276&amp;link_uid=PMC3281071" image-link="/pmc/articles/PMC3281071/figure/pone-0031229-g004/" class="imagepopup">Figure 4. Display in the Thrombin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of those differentially expressed genes from thrombin treated HMVEC..  From: RNA-seq Reveals Novel Transcriptome of Genes and Their Isoforms in Human Pulmonary Microvascular Endothelial Cells Treated with Thrombin. </a></p><div class="supp"><p class="details"><div>Thrombin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> was constructed by Ingenuity. Those with 1.3 fold up- or down-regulated genes in our data were uploaded and displayed into this <span class="highlight" style="background-color:">pathway</span>. Red color indicates up-regulation of gene expression and green down-regulated genes while white indicate no change of gene expression. The detailed fold changes and full name of each gene or isoform were presented in Table S4.</div></p></div><div class="aux"><div class="resc">Li Qin Zhang, et al. PLoS One. 2012;7(2):e31229.</div><p class="links"><a class="dblinks" href="/pubmed/22359579" ref="ordinalpos=687&amp;ncbi_uid=3816276&amp;link_uid=22359579">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281071/" ref="ordinalpos=687&amp;ncbi_uid=3816276&amp;link_uid=PMC3281071">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>688.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2366876/figure/F7/" ref="ordinalpos=688&amp;ncbi_uid=1452356&amp;link_uid=PMC2366876" href="/pmc/articles/PMC2366876/"><img src="/pmc/articles/instance/2366876/bin/zmk0050885110007.gif" src-large="/pmc/articles/instance/2366876/bin/zmk0050885110007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2366876/" ref="ordinalpos=688&amp;ncbi_uid=1452356&amp;link_uid=PMC2366876" image-link="/pmc/articles/PMC2366876/figure/F7/" class="imagepopup">Figure 7. From: The Rim101 <span class="highlight" style="background-color:">Pathway</span> Is Involved in Rsb1 Expression Induced by Altered Lipid Asymmetry. </a></p><div class="supp"><p class="details"><div>Model of the lipid asymmetry <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A signal from moderate increases in pH induces expression of the pH-responsive genes through activation of the Rim101 <span class="highlight" style="background-color:">pathway</span>. The same Rim101 <span class="highlight" style="background-color:">pathway</span> transduces a signal resulting in Rsb1 expression, but other pathways, such as that involving Mot3 and Mck1, are also required. Changes in lipid asymmetry, caused either by a flip or flop mutation, induce activation of the Rim101 <span class="highlight" style="background-color:">pathway</span> and probably of the Mot3/Mck1-related pathways as well. However, these pathways are activated differently by the flip or flop mutation. The Rim101 <span class="highlight" style="background-color:">pathway</span> is activated more strongly by the flip mutation, whereas other pathways seem to be activated more predominantly by the flop mutation. Pdr1 may act as a basic transcription factor, since little Rsb1 expression was observed in <i>pdr1</i>Δ cells.</div></p></div><div class="aux"><div class="resc">Mika Ikeda, et al. Mol Biol Cell. 2008 May;19(5):1922-1931.</div><p class="links"><a class="dblinks" href="/pubmed/18287536" ref="ordinalpos=688&amp;ncbi_uid=1452356&amp;link_uid=18287536">Citation</a><a class="dblinks" href="/pmc/articles/PMC2366876/" ref="ordinalpos=688&amp;ncbi_uid=1452356&amp;link_uid=PMC2366876">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>689.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3522553/figure/F1/" ref="ordinalpos=689&amp;ncbi_uid=4462246&amp;link_uid=PMC3522553" href="/pmc/articles/PMC3522553/"><img src="/pmc/articles/instance/3522553/bin/1471-2105-13-186-1.gif" src-large="/pmc/articles/instance/3522553/bin/1471-2105-13-186-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3522553/" ref="ordinalpos=689&amp;ncbi_uid=4462246&amp;link_uid=PMC3522553" image-link="/pmc/articles/PMC3522553/figure/F1/" class="imagepopup">Figure 1.  From: Integrated QSAR study for inhibitors of hedgehog signal <span class="highlight" style="background-color:">pathway</span> against multiple cell lines:a collaborative filtering method. </a></p><div class="supp"><p class="details"><div>Components of the Hedgehog (Hh) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and molecular sites targeted by Hh <span class="highlight" style="background-color:">pathway</span> inhibitors.</div></p></div><div class="aux"><div class="resc">Jun Gao, et al. BMC Bioinformatics. 2012;13:186-186.</div><p class="links"><a class="dblinks" href="/pubmed/22849868" ref="ordinalpos=689&amp;ncbi_uid=4462246&amp;link_uid=22849868">Citation</a><a class="dblinks" href="/pmc/articles/PMC3522553/" ref="ordinalpos=689&amp;ncbi_uid=4462246&amp;link_uid=PMC3522553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>690.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3662220/figure/F1/" ref="ordinalpos=690&amp;ncbi_uid=4921205&amp;link_uid=PMC3662220" href="/pmc/articles/PMC3662220/"><img src="/pmc/articles/instance/3662220/bin/nihms345409f1.gif" src-large="/pmc/articles/instance/3662220/bin/nihms345409f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3662220/" ref="ordinalpos=690&amp;ncbi_uid=4921205&amp;link_uid=PMC3662220" image-link="/pmc/articles/PMC3662220/figure/F1/" class="imagepopup">Figure 1. MAPK and PI3K <span class="highlight" style="background-color:">Signaling</span> Pathways.  From: Novel zebrafish model reveals a critical role for mapk in lymphangiogenesis. </a></p><div class="supp"><p class="details"><div>VEGFR2 and R3 can stimulate MAPK and PI3K <span class="highlight" style="background-color:">signaling</span> pathways in vitro. Inhibitors CI-1040, BEZ235 (BEZ), and PTK787/ZK2222584 (PTK) were employed to target the MAPK <span class="highlight" style="background-color:">pathway</span>, the PI3K <span class="highlight" style="background-color:">pathway</span>, or both pathways, respectively.</div></p></div><div class="aux"><div class="resc">R. Dawn Fevurly, et al. J Pediatr Surg. ;47(1):177-182.</div><p class="links"><a class="dblinks" href="/pubmed/22244413" ref="ordinalpos=690&amp;ncbi_uid=4921205&amp;link_uid=22244413">Citation</a><a class="dblinks" href="/pmc/articles/PMC3662220/" ref="ordinalpos=690&amp;ncbi_uid=4921205&amp;link_uid=PMC3662220">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>691.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1892100/figure/F1/" ref="ordinalpos=691&amp;ncbi_uid=987250&amp;link_uid=PMC1892100" href="/pmc/articles/PMC1892100/"><img src="/pmc/articles/instance/1892100/bin/1471-2105-8-S3-S3-1.gif" src-large="/pmc/articles/instance/1892100/bin/1471-2105-8-S3-S3-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1892100/" ref="ordinalpos=691&amp;ncbi_uid=987250&amp;link_uid=PMC1892100" image-link="/pmc/articles/PMC1892100/figure/F1/" class="imagepopup">Figure 1.  From: e-Science and biological <span class="highlight" style="background-color:">pathway</span> semantics. </a></p><div class="supp"><p class="details"><div>Image showing the human insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from BioCarta.</div></p></div><div class="aux"><div class="resc">Joanne S Luciano, et al. BMC Bioinformatics. 2007;8(Suppl 3):S3-S3.</div><p class="links"><a class="dblinks" href="/pubmed/17493286" ref="ordinalpos=691&amp;ncbi_uid=987250&amp;link_uid=17493286">Citation</a><a class="dblinks" href="/pmc/articles/PMC1892100/" ref="ordinalpos=691&amp;ncbi_uid=987250&amp;link_uid=PMC1892100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>692.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3302445/figure/F7/" ref="ordinalpos=692&amp;ncbi_uid=4260222&amp;link_uid=PMC3302445" href="/pmc/articles/PMC3302445/"><img src="/pmc/articles/instance/3302445/bin/zmb9991094310007.gif" src-large="/pmc/articles/instance/3302445/bin/zmb9991094310007.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3302445/" ref="ordinalpos=692&amp;ncbi_uid=4260222&amp;link_uid=PMC3302445" image-link="/pmc/articles/PMC3302445/figure/F7/" class="imagepopup">Fig 7.  From: The Chromatin-Remodeling Enzymes BRG1 and CHD4 Antagonistically Regulate Vascular Wnt <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Model for how BRG1 and CHD4 antagonistically influence Wnt <span class="highlight" style="background-color:">signaling</span> during yolk sac vascular development. BRG1 activates the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in two ways: through transcriptional regulation of multiple <i>Fzd</i> receptor genes upstream of β-catenin and through coregulation of Wnt target genes downstream of β-catenin (12). CHD4 represses the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream of β-catenin through transcriptional regulation of the Wnt-responsive transcription factor <i>Tcf7</i> and select Wnt target genes.</div></p></div><div class="aux"><div class="resc">Carol D. Curtis, et al. Mol Cell Biol. 2012 April;32(7):1312-1320.</div><p class="links"><a class="dblinks" href="/pubmed/22290435" ref="ordinalpos=692&amp;ncbi_uid=4260222&amp;link_uid=22290435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3302445/" ref="ordinalpos=692&amp;ncbi_uid=4260222&amp;link_uid=PMC3302445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>693.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC186276/figure/F3/" ref="ordinalpos=693&amp;ncbi_uid=212221&amp;link_uid=PMC186276" href="/pmc/articles/PMC186276/"><img src="/pmc/articles/instance/186276/bin/46891-2f3_L1TT.gif" src-large="/pmc/articles/instance/186276/bin/46891-2f3_L1TT.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC186276/" ref="ordinalpos=693&amp;ncbi_uid=212221&amp;link_uid=PMC186276" image-link="/pmc/articles/PMC186276/figure/F3/" class="imagepopup">Figure 3.  From: Activation of Wnt <span class="highlight" style="background-color:">signaling</span> bypasses the requirement for RTK/Ras <span class="highlight" style="background-color:">signaling</span> during C. elegans vulval induction. </a></p><div class="supp"><p class="details"><div>Model. We propose that Wnt <span class="highlight" style="background-color:">signaling</span> acts to maintain <i>lin-39</i> expression in all VPCs to prevent them from adopting the Fused fate. PRY-1 Axin and APR-1 APC act negatively to regulate the Wnt <span class="highlight" style="background-color:">pathway</span>, and BAR-1 β-catenin and POP-1 TCF act positively. In a cell in which the Ras <span class="highlight" style="background-color:">pathway</span> is inactive like P4.p, LIN-39 is required to adopt the 3° uninduced cell fate. In a cell in which the Ras <span class="highlight" style="background-color:">pathway</span> is active such as P6.p, we propose that the activity of both pathways leads to higher levels of LIN-39 expression, as has been reported by Maloof and Kenyon (1998). We propose that this higher level of LIN-39 and/or other Wnt <span class="highlight" style="background-color:">pathway</span> targets is instructive in causing the VPC to adopt an induced cell fate. In a cell in which the Wnt <span class="highlight" style="background-color:">pathway</span> has been hyperactivated (e.g., <i>pry-1</i> loss-of-function), higher levels of LIN-39 and/or other Wnt targets allows adoption of an induced cell fate in the absence of input from the Ras <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Julie E. Gleason, et al. Genes Dev. 2002 May 15;16(10):1281-1290.</div><p class="links"><a class="dblinks" href="/pubmed/12023306" ref="ordinalpos=693&amp;ncbi_uid=212221&amp;link_uid=12023306">Citation</a><a class="dblinks" href="/pmc/articles/PMC186276/" ref="ordinalpos=693&amp;ncbi_uid=212221&amp;link_uid=PMC186276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>694.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678880/figure/f4-ol-05-05-1719/" ref="ordinalpos=694&amp;ncbi_uid=4982775&amp;link_uid=PMC3678880" href="/pmc/articles/PMC3678880/"><img src="/pmc/articles/instance/3678880/bin/OL-05-05-1719-g03.gif" src-large="/pmc/articles/instance/3678880/bin/OL-05-05-1719-g03.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678880/" ref="ordinalpos=694&amp;ncbi_uid=4982775&amp;link_uid=PMC3678880" image-link="/pmc/articles/PMC3678880/figure/f4-ol-05-05-1719/" class="imagepopup">Figure 4.  From: Inhibition of the Wnt palmitoyltransferase porcupine suppresses cell growth and downregulates the Wnt/?-catenin <span class="highlight" style="background-color:">pathway</span> in gastric cancer. </a></p><div class="supp"><p class="details"><div>PPN inhibitor downregulated the activity of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Transcriptional activity of two transcription factors (TCF and LEF). FOP was used as the negative control. (B–E) Real-time PCR analysis of downstream Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> target genes (AXIN2, C-MYC, CCND1 and BIRC5, respectively). (F) Western blot for key regulators of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The cells were treated with IWP-2 for 3 days. <sup>*</sup>P&lt;0.05, <sup>**</sup>P&lt;0.01 and <sup>***</sup>P&lt;0.001. PPN, porcupine; IWP, inhibitors of Wnt production.</div></p></div><div class="aux"><div class="resc">MIN-LI MO, et al. Oncol Lett. 2013 May;5(5):1719-1723.</div><p class="links"><a class="dblinks" href="/pubmed/23761839" ref="ordinalpos=694&amp;ncbi_uid=4982775&amp;link_uid=23761839">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678880/" ref="ordinalpos=694&amp;ncbi_uid=4982775&amp;link_uid=PMC3678880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>695.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2755566/figure/F8/" ref="ordinalpos=695&amp;ncbi_uid=2006392&amp;link_uid=PMC2755566" href="/pmc/articles/PMC2755566/"><img src="/pmc/articles/instance/2755566/bin/nihms-119602-f0008.gif" src-large="/pmc/articles/instance/2755566/bin/nihms-119602-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2755566/" ref="ordinalpos=695&amp;ncbi_uid=2006392&amp;link_uid=PMC2755566" image-link="/pmc/articles/PMC2755566/figure/F8/" class="imagepopup">Figure 8. Proposed mechanism of intracrine PTHrP regulation of apoptosis, involving integrin α6β4 and the PI3-K/Akt <span class="highlight" style="background-color:">pathway</span>.  From: Parathyroid Hormone-related Protein Regulates Cell Survival Pathways via Integrin ?6?4-mediated Activation of PI3-K/Akt <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>PTHrP increases cell surface levels of integrin α6β4, which signals synergistically with growth factor receptors to activate the PI3-K/Akt <span class="highlight" style="background-color:">pathway</span>. This results in phosphorylation, and therefore inactivation of GSK-3. Since GSK-3 is known to phosphorylate c-myc, thereby targeting it to the ubiquitin-proteasome <span class="highlight" style="background-color:">pathway</span>, inactivation of GSK-3 results in increased c-myc levels and therefore decreased apoptosis. The PI3-K/Akt <span class="highlight" style="background-color:">pathway</span> also regulates the Bcl-2 family of proteins, resulting in decreased apoptosis. Thus, the net effect of intracrine PTHrP <span class="highlight" style="background-color:">signaling</span> in C4-2 cells is increased cell survival.</div></p></div><div class="aux"><div class="resc">Vandanajay Bhatia, et al. Mol Cancer Res. ;7(7):1119-1131.</div><p class="links"><a class="dblinks" href="/pubmed/19584267" ref="ordinalpos=695&amp;ncbi_uid=2006392&amp;link_uid=19584267">Citation</a><a class="dblinks" href="/pmc/articles/PMC2755566/" ref="ordinalpos=695&amp;ncbi_uid=2006392&amp;link_uid=PMC2755566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>696.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3116825/figure/pone-0020580-g004/" ref="ordinalpos=696&amp;ncbi_uid=3035229&amp;link_uid=PMC3116825" href="/pmc/articles/PMC3116825/"><img src="/pmc/articles/instance/3116825/bin/pone.0020580.g004.gif" src-large="/pmc/articles/instance/3116825/bin/pone.0020580.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3116825/" ref="ordinalpos=696&amp;ncbi_uid=3035229&amp;link_uid=PMC3116825" image-link="/pmc/articles/PMC3116825/figure/pone-0020580-g004/" class="imagepopup">Figure 4. Dcdc2 affects ciliary <span class="highlight" style="background-color:">signaling</span>..  From: Increased Expression of the Dyslexia Candidate Gene DCDC2 Affects Length and <span class="highlight" style="background-color:">Signaling</span> of Primary Cilia in Neurons. </a></p><div class="supp"><p class="details"><div>Overexpression of DCDC2 activates Shh <span class="highlight" style="background-color:">signaling</span> (A). DCDC2 overexpressing neurons had more Shh <span class="highlight" style="background-color:">pathway</span> activation compared to neurons transfected with DCDC2 deletion constructs, Dcdc2-shRNA or control vectors (ANOVA followed by T-test, * p = 7,3×10<sup>−5</sup>). Dcdc2 knockdown by shRNA activates canonical Wnt <span class="highlight" style="background-color:">signaling</span> (B) as measured by a reporter luciferase assay (ANOVA, followed by T-test * p = 3,7×10<sup>−5</sup>). The error bars indicate SEM.</div></p></div><div class="aux"><div class="resc">Satu Massinen, et al. PLoS One. 2011;6(6):e20580.</div><p class="links"><a class="dblinks" href="/pubmed/21698230" ref="ordinalpos=696&amp;ncbi_uid=3035229&amp;link_uid=21698230">Citation</a><a class="dblinks" href="/pmc/articles/PMC3116825/" ref="ordinalpos=696&amp;ncbi_uid=3035229&amp;link_uid=PMC3116825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>697.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3539953/figure/F5/" ref="ordinalpos=697&amp;ncbi_uid=4521008&amp;link_uid=PMC3539953" href="/pmc/articles/PMC3539953/"><img src="/pmc/articles/instance/3539953/bin/1471-2164-13-279-5.gif" src-large="/pmc/articles/instance/3539953/bin/1471-2164-13-279-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3539953/" ref="ordinalpos=697&amp;ncbi_uid=4521008&amp;link_uid=PMC3539953" image-link="/pmc/articles/PMC3539953/figure/F5/" class="imagepopup">Figure 5.  From: Expression sequence tag library derived from peripheral blood mononuclear cells of the chlorocebus sabaeus. </a></p><div class="supp"><p class="details"><div><b>Representation of the “Interferon <span class="highlight" style="background-color:">Signaling</span>” and “Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>” pathways.</b> (<b>A</b>) Representation of the “Interferon <span class="highlight" style="background-color:">Signaling</span>” <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) Representation of the “Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>” <span class="highlight" style="background-color:">pathway</span>. Genes present in the EST library are shown in gray.</div></p></div><div class="aux"><div class="resc">Nicolas Tchitchek, et al. BMC Genomics. 2012;13:279-279.</div><p class="links"><a class="dblinks" href="/pubmed/22726727" ref="ordinalpos=697&amp;ncbi_uid=4521008&amp;link_uid=22726727">Citation</a><a class="dblinks" href="/pmc/articles/PMC3539953/" ref="ordinalpos=697&amp;ncbi_uid=4521008&amp;link_uid=PMC3539953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>698.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3789883/figure/pgen-1003770-g001/" ref="ordinalpos=698&amp;ncbi_uid=5370224&amp;link_uid=PMC3789883" href="/pmc/articles/PMC3789883/"><img src="/pmc/articles/instance/3789883/bin/pgen.1003770.g001.gif" src-large="/pmc/articles/instance/3789883/bin/pgen.1003770.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3789883/" ref="ordinalpos=698&amp;ncbi_uid=5370224&amp;link_uid=PMC3789883" image-link="/pmc/articles/PMC3789883/figure/pgen-1003770-g001/" class="imagepopup">Figure 1. Diseases show a wide range of support for enrichment of disease associations in pathways..  From: Integrated Enrichment Analysis of Variants and Pathways in Genome-Wide Association Studies Indicates Central Role for IL-2 <span class="highlight" style="background-color:">Signaling</span> Genes in Type 1 Diabetes, and Cytokine <span class="highlight" style="background-color:">Signaling</span> Genes in Crohn's Disease. </a></p><div class="supp"><p class="details"><div>Each row shows the <span class="highlight" style="background-color:">pathway</span> with the largest BF for enrichment of disease associations among 3158 candidate gene sets. Columns left to right: (1) disease; (2) enriched <span class="highlight" style="background-color:">pathway</span>; (3) <span class="highlight" style="background-color:">pathway</span> database, and repository where <span class="highlight" style="background-color:">pathway</span> is retrieved if different from database; (4) BF for hypothesis that disease associations are enriched among SNPs assigned to <span class="highlight" style="background-color:">pathway</span>; (5) posterior probability of enrichment hypothesis; (6) number of genes assigned to <span class="highlight" style="background-color:">pathway</span>; (7) number of SNPs near these genes. Abbreviations used in figure: PID = NCI Nature <span class="highlight" style="background-color:">Pathway</span> Interaction Database [163], BS = NCBI BioSystems [164], PC = <span class="highlight" style="background-color:">Pathway</span> Commons [165]. Databases and database identifiers for pathways listed here: “Transport of connexons to the plasma membrane” (Reactome 11050, PC); “Tumor suppressor Arf inhibits ribosomal biogenesis” (BioCarta); “Cytokine <span class="highlight" style="background-color:">signaling</span> in immune system” (Reactome 75790, BS 366171); “Alanine biosynthesis” (PANTHER P02724); “Measles” (KEGG hsa05162, BS 213306); “IL2-mediated <span class="highlight" style="background-color:">signaling</span> events” (PID il2_1pathway, BS 137976); “Incretin synthesis, secretion, and inactivation” (Reactome 23974, PC). *Null and enrichment hypotheses for RA and T1D include enrichment of disease associations in MHC, in which SNPs within MHC are enriched at a different level than non-MHC SNPs in <span class="highlight" style="background-color:">pathway</span>; <img src="/pmc/articles/instance/3789883/bin/pgen.1003770.e049.jpg" class="inline-graphic" style="border-style: none;" /> and 4.6 for RA and T1D, respectively. Number of genes/SNPs for RA and T1D count only non-MHC genes assigned to <span class="highlight" style="background-color:">pathway</span>. **Illustrative posterior probability assuming a “conservative” prior (see text).</div></p></div><div class="aux"><div class="resc">Peter Carbonetto, et al. PLoS Genet. 2013 October;9(10):e1003770.</div><p class="links"><a class="dblinks" href="/pubmed/24098138" ref="ordinalpos=698&amp;ncbi_uid=5370224&amp;link_uid=24098138">Citation</a><a class="dblinks" href="/pmc/articles/PMC3789883/" ref="ordinalpos=698&amp;ncbi_uid=5370224&amp;link_uid=PMC3789883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>699.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3774835/figure/F4/" ref="ordinalpos=699&amp;ncbi_uid=5395848&amp;link_uid=PMC3774835" href="/pmc/articles/PMC3774835/"><img src="/pmc/articles/instance/3774835/bin/nihms471147f4.gif" src-large="/pmc/articles/instance/3774835/bin/nihms471147f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3774835/" ref="ordinalpos=699&amp;ncbi_uid=5395848&amp;link_uid=PMC3774835" image-link="/pmc/articles/PMC3774835/figure/F4/" class="imagepopup">Figure 4. mTORC1 is activated by ERK2/RSK1 <span class="highlight" style="background-color:">signaling</span> to TSC2 in LEC.219 but not in BEC.219.  From: A unique herpesviral transcriptional program in KSHV-infected lymphatic endothelial cells leads to mTORC1 activation and rapamycin sensitivity. </a></p><div class="supp"><p class="details"><div>The phosphorylation status of relevant <span class="highlight" style="background-color:">signaling</span> pathways was determined by immunoblotting with indicated antibodies. (<b>A</b>) Western blots of the substrates of mTORC1 <span class="highlight" style="background-color:">signaling</span>, p70 S6K (that phosphorylates S6) and 4EBP1, in mock and stably infected (inf) LEC and BEC were performed. (<b>B</b>) The MAPK <span class="highlight" style="background-color:">pathway</span> (MEK/ERK/RSK) and PI3K/AKT <span class="highlight" style="background-color:">pathway</span> converge at TSC2 to regulate mTORC1 <span class="highlight" style="background-color:">signaling</span>. (<b>C</b>) PI3K/AKT <span class="highlight" style="background-color:">signaling</span> to TSC2 T1462 shows no contribution to the mTORC1 activation in LEC.219. (<b>D</b>) ERK2/RSK1 signals to TSC2 S1798 in LEC.219 without any contribution from upstream MEK1/2. See also Figure S3.</div></p></div><div class="aux"><div class="resc">Henry H Chang, et al. Cell Host Microbe. 2013 April 17;13(4):429-440.</div><p class="links"><a class="dblinks" href="/pubmed/23601105" ref="ordinalpos=699&amp;ncbi_uid=5395848&amp;link_uid=23601105">Citation</a><a class="dblinks" href="/pmc/articles/PMC3774835/" ref="ordinalpos=699&amp;ncbi_uid=5395848&amp;link_uid=PMC3774835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>700.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2700400/figure/fig10/" ref="ordinalpos=700&amp;ncbi_uid=1878891&amp;link_uid=PMC2700400" href="/pmc/articles/PMC2700400/"><img src="/pmc/articles/instance/2700400/bin/JCB_200809151_RGB_Fig10.gif" src-large="/pmc/articles/instance/2700400/bin/JCB_200809151_RGB_Fig10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2700400/" ref="ordinalpos=700&amp;ncbi_uid=1878891&amp;link_uid=PMC2700400" image-link="/pmc/articles/PMC2700400/figure/fig10/" class="imagepopup">Figure 10. From: Synaptic activity prompts ?-secretase-mediated cleavage of EphA4 and dendritic spine formation. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of EphA4 <span class="highlight" style="background-color:">signaling</span> at dendritic spines.</b> (A) Effect of familial Alzheimer's disease–linked mutation of PS1 on the processing of EphA4. PS double knockout cells were cotransfected with EphA4 CTF-HA and the indicated PS1 mutants. The purified membrane fractions were incubated for 4 h. The generated ICD was purified by immunoprecipitation using the anti-HA antibody, followed by Western blotting. (B) Schematic diagram of EphA4 <span class="highlight" style="background-color:">signaling</span> at dendritic spines. (B, top) Ephrin-dependent retraction of dendritic spines. Activation of EphA4 by Ephrin retracts dendritic spines by activating the RhoA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, by inhibiting the integrin <span class="highlight" style="background-color:">signaling</span> pathways, and, possibly, by inhibiting the Rac <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B, bottom) γ-secretase–dependent enhancement of the formation of dendritic spines. Activity-induced processing of EphA4 by γ-secretase enhances the formation of dendritic spines by activating the Rac <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. EICD, EphA4 ICD.</div></p></div><div class="aux"><div class="resc">Eiji Inoue, et al. J Cell Biol. 2009 May 4;185(3):551-564.</div><p class="links"><a class="dblinks" href="/pubmed/19414612" ref="ordinalpos=700&amp;ncbi_uid=1878891&amp;link_uid=19414612">Citation</a><a class="dblinks" href="/pmc/articles/PMC2700400/" ref="ordinalpos=700&amp;ncbi_uid=1878891&amp;link_uid=PMC2700400">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="6" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="7" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="8" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:08:41-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB1BB753FE89D10000000000597A74;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB1BB753FE89D10000000000597A74" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>