
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="50" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A0A0E543141A10000000000673442" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="49" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="50" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="51" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="50" /></div>
            <div><h2 class="result_count">Results: 4901 to 5000 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4901.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4106844/figure/pone-0102974-g003/" ref="ordinalpos=4901&amp;ncbi_uid=6406307&amp;link_uid=PMC4106844" href="/pmc/articles/PMC4106844/"><img src="/pmc/articles/instance/4106844/bin/pone.0102974.g003.gif" src-large="/pmc/articles/instance/4106844/bin/pone.0102974.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4106844/" ref="ordinalpos=4901&amp;ncbi_uid=6406307&amp;link_uid=PMC4106844" image-link="/pmc/articles/PMC4106844/figure/pone-0102974-g003/" class="imagepopup">Figure 3. FnIII-1c activates NF-κB in parallel with the p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Regulation of the Innate Immune Response by Fibronectin: Synergism between the III-1 and EDA Domains. </a></p><div class="supp"><p class="details"><div>Monolayers of human fibroblasts were serum-starved overnight and treated with the indicated concentration of p38 inhibitor (SB203580), MK-2 inhibitor (MK-2 inhibitor III), or NFκB inhibitors (PS-1145, Bay11-7082) for 1 h. Cells were then stimulated with 20 µM FnIII-1c for 1vh. Cells were lysed and the nuclear fraction isolated and analyzed by Western blot for the presence of NF-κB protein p65/rel A (A). Blots were quantified by densitometry and normalized to total lamin A/C. Values are mean ± S.D. of 3 separate experiments (B). Cytoplasmic fractions were analyzed by Western blot for p-p38 (C). Membranes were stripped and reprobed with an antibody to nuclear Lamin A/C or FAK, which served as loading control. Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (D).</div></p></div><div class="aux"><div class="resc">Rhiannon Kelsh, et al. PLoS One. 2014;9(7):e102974.</div><p class="links"><a class="dblinks" href="/pubmed/25051083" ref="ordinalpos=4901&amp;ncbi_uid=6406307&amp;link_uid=25051083">Citation</a><a class="dblinks" href="/pmc/articles/PMC4106844/" ref="ordinalpos=4901&amp;ncbi_uid=6406307&amp;link_uid=PMC4106844">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4902.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC118015/figure/f6/" ref="ordinalpos=4902&amp;ncbi_uid=326556&amp;link_uid=PMC118015" href="/pmc/articles/PMC118015/"><img src="/pmc/articles/instance/118015/bin/ek0320088006.gif" src-large="/pmc/articles/instance/118015/bin/ek0320088006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC118015/" ref="ordinalpos=4902&amp;ncbi_uid=326556&amp;link_uid=PMC118015" image-link="/pmc/articles/PMC118015/figure/f6/" class="imagepopup">FIG. 6. From: G-Protein <span class="highlight" style="background-color:">Signaling</span> Mediates Asexual Development at 25?C but Has No Effect on Yeast-Like Growth at 37?C in the Dimorphic Fungus Penicillium marneffei. </a></p><div class="supp"><p class="details"><div>Proposed model for signal transduction pathways regulating development and growth in <i>P. marneffei</i>. GasA <span class="highlight" style="background-color:">signaling</span> at 25°C involves a cAMP-PKA <span class="highlight" style="background-color:">pathway</span> in the regulation of asexual development and the expression of the core regulatory factors BrlA and AbaA. GasA has no apparent role during yeast cell morphogenesis at 37°C. Therefore, we hypothesize the presence of a <i>brlA</i> repressor (black box) as well as an <i>abaA</i> activator (white box) which are independent of GasA <span class="highlight" style="background-color:">signaling</span> at this temperature.</div></p></div><div class="aux"><div class="resc">Sophie Zuber, et al. Eukaryot Cell. 2002 June;1(3):440-447.</div><p class="links"><a class="dblinks" href="/pubmed/12455992" ref="ordinalpos=4902&amp;ncbi_uid=326556&amp;link_uid=12455992">Citation</a><a class="dblinks" href="/pmc/articles/PMC118015/" ref="ordinalpos=4902&amp;ncbi_uid=326556&amp;link_uid=PMC118015">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4903.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3566170/figure/pone-0054374-g008/" ref="ordinalpos=4903&amp;ncbi_uid=4632623&amp;link_uid=PMC3566170" href="/pmc/articles/PMC3566170/"><img src="/pmc/articles/instance/3566170/bin/pone.0054374.g008.gif" src-large="/pmc/articles/instance/3566170/bin/pone.0054374.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3566170/" ref="ordinalpos=4903&amp;ncbi_uid=4632623&amp;link_uid=PMC3566170" image-link="/pmc/articles/PMC3566170/figure/pone-0054374-g008/" class="imagepopup">Figure 8. Schematic diagram of the <span class="highlight" style="background-color:">signaling</span> pathways involved in Cd-induced pancreatic β-cell apoptosis..  From: Cadmium Induces Apoptosis in Pancreatic ?-Cells through a Mitochondria-Dependent <span class="highlight" style="background-color:">Pathway</span>: The Role of Oxidative Stress-Mediated c-Jun N-Terminal Kinase Activation. </a></p><div class="supp"><p class="details"><div>Proposed models representing that Cd induces pancreatic β-cell apoptosis through ROS triggering JNK activation-regulated mitochondria-dependent apoptotic <span class="highlight" style="background-color:">signaling</span> cascades are illustrated in this schematic diagram.</div></p></div><div class="aux"><div class="resc">Kai-Chih Chang, et al. PLoS One. 2013;8(2):e54374.</div><p class="links"><a class="dblinks" href="/pubmed/23405080" ref="ordinalpos=4903&amp;ncbi_uid=4632623&amp;link_uid=23405080">Citation</a><a class="dblinks" href="/pmc/articles/PMC3566170/" ref="ordinalpos=4903&amp;ncbi_uid=4632623&amp;link_uid=PMC3566170">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4904.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3439510/figure/F4/" ref="ordinalpos=4904&amp;ncbi_uid=4236539&amp;link_uid=PMC3439510" href="/pmc/articles/PMC3439510/"><img src="/pmc/articles/instance/3439510/bin/nihms311505f4.gif" src-large="/pmc/articles/instance/3439510/bin/nihms311505f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3439510/" ref="ordinalpos=4904&amp;ncbi_uid=4236539&amp;link_uid=PMC3439510" image-link="/pmc/articles/PMC3439510/figure/F4/" class="imagepopup">Fig. 4.  From: Cadmium induces autophagy through ROS-dependent activation of the LKB1-AMPK <span class="highlight" style="background-color:">signaling</span> in skin epidermal cells. </a></p><div class="supp"><p class="details"><div>mTOR <span class="highlight" style="background-color:">signaling</span> is downstream of LKB1-AMPK <span class="highlight" style="background-color:">pathway</span> in cadmium-exposed cells. (A) Cells were incubated with various cadmium concentrations (0–10 μM) for 24 h. (B) Cells were also exposed various times (0–24 h) at 10 μM cadmium. The proteins levels were analyzed by Western blot. The phosphorylated mTOR and total mTOR levels were analyzed after transfection with siRNA specific for LKB1(C) or AMPKα (D) to the cells. β-actin was used as loading control.</div></p></div><div class="aux"><div class="resc">Young-Ok Son, et al. Toxicol Appl Pharmacol. 2011 September 15;255(3):287-296.</div><p class="links"><a class="dblinks" href="/pubmed/21767558" ref="ordinalpos=4904&amp;ncbi_uid=4236539&amp;link_uid=21767558">Citation</a><a class="dblinks" href="/pmc/articles/PMC3439510/" ref="ordinalpos=4904&amp;ncbi_uid=4236539&amp;link_uid=PMC3439510">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4905.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3169155/figure/fig05/" ref="ordinalpos=4905&amp;ncbi_uid=3500876&amp;link_uid=PMC3169155" href="/pmc/articles/PMC3169155/"><img src="/pmc/articles/instance/3169155/bin/pnas.1107411108fig05.gif" src-large="/pmc/articles/instance/3169155/bin/pnas.1107411108fig05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3169155/" ref="ordinalpos=4905&amp;ncbi_uid=3500876&amp;link_uid=PMC3169155" image-link="/pmc/articles/PMC3169155/figure/fig05/" class="imagepopup">Fig. 5. From: Serotonin <span class="highlight" style="background-color:">signaling</span> is associated with lower amyloid-? levels and plaques in transgenic mice and humans. </a></p><div class="supp"><p class="details"><div>Model of serotonergic regulation of Aβ metabolism. Serotonin receptors are activated on the cell surface, which initiates a <span class="highlight" style="background-color:">signaling</span> cascade that leads to ERK phosphorylation and activation. ERK appears to increase α-secretase cleavage and may reduce γ-secretase cleavage of APP. The particular serotonin receptor subtypes responsible for ERK activation in this paradigm are unknown. Not all receptors or ligands that activate ERK will alter APP processing, however similar to serotonin, NMDA receptors can also modulate APP processing through the ERK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">John R. Cirrito, et al. Proc Natl Acad Sci U S A. 2011 September 6;108(36):14968-14973.</div><p class="links"><a class="dblinks" href="/pubmed/21873225" ref="ordinalpos=4905&amp;ncbi_uid=3500876&amp;link_uid=21873225">Citation</a><a class="dblinks" href="/pmc/articles/PMC3169155/" ref="ordinalpos=4905&amp;ncbi_uid=3500876&amp;link_uid=PMC3169155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4906.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4338284/figure/pone.0118132.g004/" ref="ordinalpos=4906&amp;ncbi_uid=7232347&amp;link_uid=PMC4338284" href="/pmc/articles/PMC4338284/"><img src="/pmc/articles/instance/4338284/bin/pone.0118132.g004.gif" src-large="/pmc/articles/instance/4338284/bin/pone.0118132.g004.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4338284/" ref="ordinalpos=4906&amp;ncbi_uid=7232347&amp;link_uid=PMC4338284" image-link="/pmc/articles/PMC4338284/figure/pone.0118132.g004/" class="imagepopup">Fig 4. ERK/MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>..  From: Molecular Characterization of the Peripheral Airway Field of Cancerization in Lung Adenocarcinoma. </a></p><div class="supp"><p class="details"><div>IPA was used to generate a network of molecules from mRNA and miRNA that were significantly different in peripheral airway field of cancerization in lung cancer patients compared to smoker controls. This composite network merge three top networks based on “network score,” which converges on the ERK/MAPk <span class="highlight" style="background-color:">pathway</span>: specifically merging on Extracellular signal-related kinases 1 and 2 (ERK1/2), c-Jun N-terminal kinases (JNKs), and p38 kinases subfamily. <b>Red</b> is up-regulated and <b>Green</b> is down-regulated.</div></p></div><div class="aux"><div class="resc">Jun-Chieh J. Tsay, et al. PLoS One. 2015;10(2):e0118132.</div><p class="links"><a class="dblinks" href="/pubmed/25705890" ref="ordinalpos=4906&amp;ncbi_uid=7232347&amp;link_uid=25705890">Citation</a><a class="dblinks" href="/pmc/articles/PMC4338284/" ref="ordinalpos=4906&amp;ncbi_uid=7232347&amp;link_uid=PMC4338284">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4907.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC515345/figure/fig8/" ref="ordinalpos=4907&amp;ncbi_uid=490770&amp;link_uid=PMC515345" href="/pmc/articles/PMC515345/"><img src="/pmc/articles/instance/515345/bin/zmk0090428070008.gif" src-large="/pmc/articles/instance/515345/bin/zmk0090428070008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC515345/" ref="ordinalpos=4907&amp;ncbi_uid=490770&amp;link_uid=PMC515345" image-link="/pmc/articles/PMC515345/figure/fig8/" class="imagepopup">Figure 8. From: Polarized Membrane Traffic and Cell Polarity Development Is Dependent on Dihydroceramide Synthase-Regulated Sphinganine Turnover. </a></p><div class="supp"><p class="details"><div>Working model that describes the relationship between sphinganine metabolism, polarized membrane traffic from SAC, and apical PM biogenesis in response to cAMP/PKA <span class="highlight" style="background-color:">signaling</span> in HepG2 cells. First, cAMP/PKA <span class="highlight" style="background-color:">signaling</span> reorganizes polarized membrane traffic from the SAC, which includes the rerouting of a fluorescent lipid probe from a basolateral direction into the transcytotic <span class="highlight" style="background-color:">pathway</span>, leading to the apical surface. Activation of this <span class="highlight" style="background-color:">pathway</span> is crucial for cAMP/PKA-stimulated polarity development. cAMP/PKA <span class="highlight" style="background-color:">signaling</span> also increases the activity of dihydroceramide synthase, thereby stimulating the turnover of sphinganine. The inability to turn over sphinganine and the consequent accumulation of free sphinganine inhibits cAMP/PKA-induced activation of the apical PM directed traffic route from SAC and, hence, polarity development.</div></p></div><div class="aux"><div class="resc">Sven C.D. van IJzendoorn, et al. Mol Biol Cell. 2004 September;15(9):4115-4124.</div><p class="links"><a class="dblinks" href="/pubmed/15229289" ref="ordinalpos=4907&amp;ncbi_uid=490770&amp;link_uid=15229289">Citation</a><a class="dblinks" href="/pmc/articles/PMC515345/" ref="ordinalpos=4907&amp;ncbi_uid=490770&amp;link_uid=PMC515345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4908.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3160087/figure/F8/" ref="ordinalpos=4908&amp;ncbi_uid=3548414&amp;link_uid=PMC3160087" href="/pmc/articles/PMC3160087/"><img src="/pmc/articles/instance/3160087/bin/DEV065656F7.gif" src-large="/pmc/articles/instance/3160087/bin/DEV065656F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3160087/" ref="ordinalpos=4908&amp;ncbi_uid=3548414&amp;link_uid=PMC3160087" image-link="/pmc/articles/PMC3160087/figure/F8/" class="imagepopup">Fig. 7. From: Regulation of extra-embryonic endoderm stem cell differentiation by Nodal and Cripto <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model of the role of the Nodal and Cripto <span class="highlight" style="background-color:">signaling</span> pathways in mouse XEN cell differentiation.</b> (<b>A</b>) Nodal <span class="highlight" style="background-color:">signaling</span> through Alk4 is Cryptic dependent and leads to Smad2 phosphorylation. This canonical <span class="highlight" style="background-color:">pathway</span> eventually leads to visceral endoderm and AVE differentiation (blue dashed arrow). (<b>B</b>) In the absence of Nodal, Cripto can signal through Alk4 to promote Smad2 phosphorylation, which may lead to visceral endoderm and AVE differentiation; this Cripto activity could conceivably be mediated through Alk4 binding by an unknown TGFβ ligand that is distinct from Nodal. However, the ability of Cripto to promote visceral endoderm and AVE differentiation is at least partially independent of Alk4 kinase activity and Smad2 phosphorylation, as it is not blocked by SB431542. Thus, Cripto can also signal in a non-canonical <span class="highlight" style="background-color:">pathway</span> (red dashed lines), either through Alk4 in a kinase-independent manner through an unknown signal transducer `X' or through an unknown cell-surface receptor `Y' that can also mediate visceral endoderm and AVE differentiation. VE, visceral endoderm; AVE, anterior visceral endoderm.</div></p></div><div class="aux"><div class="resc">Marianna Kruithof-de Julio, et al. Development. 2011 September 15;138(18):3885-3895.</div><p class="links"><a class="dblinks" href="/pubmed/21862554" ref="ordinalpos=4908&amp;ncbi_uid=3548414&amp;link_uid=21862554">Citation</a><a class="dblinks" href="/pmc/articles/PMC3160087/" ref="ordinalpos=4908&amp;ncbi_uid=3548414&amp;link_uid=PMC3160087">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4909.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4162460/figure/fig8/" ref="ordinalpos=4909&amp;ncbi_uid=6638399&amp;link_uid=PMC4162460" href="/pmc/articles/PMC4162460/"><img src="/pmc/articles/instance/4162460/bin/onc201432f8.gif" src-large="/pmc/articles/instance/4162460/bin/onc201432f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4162460/" ref="ordinalpos=4909&amp;ncbi_uid=6638399&amp;link_uid=PMC4162460" image-link="/pmc/articles/PMC4162460/figure/fig8/" class="imagepopup">Figure 8.  From: Targeting Epstein-Barr virus oncoprotein LMP1-mediated glycolysis sensitizes nasopharyngeal carcinoma to radiation therapy. </a></p><div class="supp"><p class="details"><div>A schematic for reprogramming of EBV-induced glucose metabolism. The PI3-K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> likely has critical roles in EBV-mediated glucose metabolism reprogramming, which acts mainly by disrupting the stabilization of c-Myc. When the PI3-K/Akt-GSK3beta-FBW7-c-Myc <span class="highlight" style="background-color:">signaling</span> axis is impeded, c-Myc transactivates the transcription of HK2, causing the upregulation of glycolysis and confers resistance against mitochondrial-dependent apoptosis, which correlates with the poor overall survival of NPC patients.</div></p></div><div class="aux"><div class="resc">L Xiao, et al. Oncogene. 2014 September 11;33(37):4568-4578.</div><p class="links"><a class="dblinks" href="/pubmed/24662831" ref="ordinalpos=4909&amp;ncbi_uid=6638399&amp;link_uid=24662831">Citation</a><a class="dblinks" href="/pmc/articles/PMC4162460/" ref="ordinalpos=4909&amp;ncbi_uid=6638399&amp;link_uid=PMC4162460">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4910.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052347/figure/F5/" ref="ordinalpos=4910&amp;ncbi_uid=6238698&amp;link_uid=PMC4052347" href="/pmc/articles/PMC4052347/"><img src="/pmc/articles/instance/4052347/bin/1752-0509-8-57-5.gif" src-large="/pmc/articles/instance/4052347/bin/1752-0509-8-57-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052347/" ref="ordinalpos=4910&amp;ncbi_uid=6238698&amp;link_uid=PMC4052347" image-link="/pmc/articles/PMC4052347/figure/F5/" class="imagepopup">Figure 5.  From: Interactomics profiling of the negative regulatory function of carbon monoxide on RANKL-treated RAW 264.7 cells during osteoclastogenesis. </a></p><div class="supp"><p class="details"><div><b>The PPI network and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis. </b><b>(A)</b> The PPI network, as shown in the interaction view, generated by the STRING database. Black squares indicate the protein clusters, and the red square the controlling hub, MAP3K4. <b>(B)</b> The computed predicted <span class="highlight" style="background-color:">signaling</span> pathways that are activated in RANKL-treated RAW 264.7 cells.</div></p></div><div class="aux"><div class="resc">Feng-Jen Tseng, et al. BMC Syst Biol. 2014;8:57-57.</div><p class="links"><a class="dblinks" href="/pubmed/24886323" ref="ordinalpos=4910&amp;ncbi_uid=6238698&amp;link_uid=24886323">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052347/" ref="ordinalpos=4910&amp;ncbi_uid=6238698&amp;link_uid=PMC4052347">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4911.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3718996/figure/Fig8/" ref="ordinalpos=4911&amp;ncbi_uid=5124022&amp;link_uid=PMC3718996" href="/pmc/articles/PMC3718996/"><img src="/pmc/articles/instance/3718996/bin/12035_2013_8429_Fig8_HTML.gif" src-large="/pmc/articles/instance/3718996/bin/12035_2013_8429_Fig8_HTML.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3718996/" ref="ordinalpos=4911&amp;ncbi_uid=5124022&amp;link_uid=PMC3718996" image-link="/pmc/articles/PMC3718996/figure/Fig8/" class="imagepopup">Fig. 8.  From: Dissecting the Inter-Substrate Navigation of Migrating Glioblastoma Cells with the Stripe Assay Reveals a Causative Role of ROCK. </a></p><div class="supp"><p class="details"><div>Graphs showing the different regulation of glioma cell <span class="highlight" style="background-color:">signaling</span> when cells were cultured on biomatrix, myelin and unmyelinated tissue (retina). <i>Arrows</i> indicate the up- and downregulation of intracellular <span class="highlight" style="background-color:">signaling</span> members of the Rho kinase <span class="highlight" style="background-color:">pathway</span>. These differences likely explain that alternative pathways exist for migration on different substrates</div></p></div><div class="aux"><div class="resc">Sonja Mertsch, et al. Mol Neurobiol. 2013 August;48(1):169-179.</div><p class="links"><a class="dblinks" href="/pubmed/23436115" ref="ordinalpos=4911&amp;ncbi_uid=5124022&amp;link_uid=23436115">Citation</a><a class="dblinks" href="/pmc/articles/PMC3718996/" ref="ordinalpos=4911&amp;ncbi_uid=5124022&amp;link_uid=PMC3718996">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4912.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1201250/figure/f7/" ref="ordinalpos=4912&amp;ncbi_uid=616024&amp;link_uid=PMC1201250" href="/pmc/articles/PMC1201250/"><img src="/pmc/articles/instance/1201250/bin/zii0080550480007.gif" src-large="/pmc/articles/instance/1201250/bin/zii0080550480007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1201250/" ref="ordinalpos=4912&amp;ncbi_uid=616024&amp;link_uid=PMC1201250" image-link="/pmc/articles/PMC1201250/figure/f7/" class="imagepopup">FIG. 7. From: Toll-Like Receptor 4 Mediates Tolerance in Macrophages Stimulated with Toxoplasma gondii-Derived Heat Shock Protein 70 . </a></p><div class="supp"><p class="details"><div>Mechanism of TgHSP70 tolerance in PMs from WT mice. TgHSP70 stimulation leads to TLR2 activation and the <span class="highlight" style="background-color:">pathway</span> of TLR2-MyD88-IRAK4-TRAF6-IKK-mediated phosphorylation of I-κB. These <span class="highlight" style="background-color:">signaling</span> cascades produce NO. Simultaneously, TgHSP70-stimulated TLR4 mediates IFN-β activation that suppresses NF-κB-inducing kinase (NIK) (18). Inhibition of these <span class="highlight" style="background-color:">signaling</span> cascades by IFN-β is possibly mediated by SOCS-1 (7) and limits the production of NO.</div></p></div><div class="aux"><div class="resc">Hye-Seong Mun, et al. Infect Immun. 2005 August;73(8):4634-4642.</div><p class="links"><a class="dblinks" href="/pubmed/16040976" ref="ordinalpos=4912&amp;ncbi_uid=616024&amp;link_uid=16040976">Citation</a><a class="dblinks" href="/pmc/articles/PMC1201250/" ref="ordinalpos=4912&amp;ncbi_uid=616024&amp;link_uid=PMC1201250">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4913.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4101960/figure/fig1/" ref="ordinalpos=4913&amp;ncbi_uid=6468397&amp;link_uid=PMC4101960" href="/pmc/articles/PMC4101960/"><img src="/pmc/articles/instance/4101960/bin/BMRI2014-124902.001.gif" src-large="/pmc/articles/instance/4101960/bin/BMRI2014-124902.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4101960/" ref="ordinalpos=4913&amp;ncbi_uid=6468397&amp;link_uid=PMC4101960" image-link="/pmc/articles/PMC4101960/figure/fig1/" class="imagepopup">Figure 1.  From: Effect of Angiotensin II and Small GTPase Ras <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Inhibition on Early Renal Changes in a Murine Model of Obstructive Nephropathy. </a></p><div class="supp"><p class="details"><div>Effect of systemically administrated angiotensin II (Ang II) on Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Ras activation was evaluated as Ras GTP by ELISA (a). Phosphorylated (p)-ERK1/2 and p-Akt protein expression were evaluated as the ratio p/total protein by Western blot ((b) and (c), resp.). Bars represent the mean ± SEM of the optical density measured in kidney samples of control saline group (Ctrl; <i>n</i> = 3) and angiotensin II-treated animals (Ang II, 0.8 mg/kg; <i>n</i> = 3–5/per time point). *<i>P</i> &lt; 0.05 versus control group.</div></p></div><div class="aux"><div class="resc">Ana B. Rodríguez-Peña, et al. Biomed Res Int. 2014;2014:124902.</div><p class="links"><a class="dblinks" href="/pubmed/25101263" ref="ordinalpos=4913&amp;ncbi_uid=6468397&amp;link_uid=25101263">Citation</a><a class="dblinks" href="/pmc/articles/PMC4101960/" ref="ordinalpos=4913&amp;ncbi_uid=6468397&amp;link_uid=PMC4101960">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4914.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3892125/figure/F1/" ref="ordinalpos=4914&amp;ncbi_uid=5707555&amp;link_uid=PMC3892125" href="/pmc/articles/PMC3892125/"><img src="/pmc/articles/instance/3892125/bin/1744-8069-9-64-1.gif" src-large="/pmc/articles/instance/3892125/bin/1744-8069-9-64-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3892125/" ref="ordinalpos=4914&amp;ncbi_uid=5707555&amp;link_uid=PMC3892125" image-link="/pmc/articles/PMC3892125/figure/F1/" class="imagepopup">Figure 1.  From: The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates pain-related synaptic plasticity in rat entorhinal-hippocampal pathways. </a></p><div class="supp"><p class="details"><div><b>Activation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the hippocampus by persistent peripheral nociception, but the effect was reversed by rapamycin (RAPA). (A)</b> Western blot analysis of p-mTOR, mTOR, p-S6K, and S6K expression in hippocampal lysates at various times after bee venom (BV) treatment. <b>(B)</b> Western analysis of p-mTOR, mTOR, p-S6K, and S6K in hippocampus tissues extracts from saline control (sal-control), Sal + BV, and bupivacaine (Bup) + BV groups. <b>(C)</b> Western blot analysis of p-mTOR, mTOR, p-S6K, and S6K in hippocampal lysates from Sal-control, 2%DMSO + BV, and RAPA + BV groups. Relative expresions of p-mTOR, mTOR, p-S6K, and S6K immunolabeling are shown relative to the control or sal-control (100%). β-tubulin was used as an internal control. The data were showed as means ± SD (n = 3). <sup>*</sup><i>P</i> &lt; 0.05 <i>vs.</i> sal-control group; <sup>#</sup><i>P</i> &lt; 0.05, <sup>##</sup><i>P</i> &lt; 0.01, bupivacaine + BV group <i>vs.</i> Sal + BV; <sup>Δ</sup><i>P</i> &lt; 0.05, <sup>ΔΔ</sup><i>P</i> &lt; 0.01, RAPA + BV group <i>vs.</i> 2%DMSO + BV group.</div></p></div><div class="aux"><div class="resc">Dan Lyu, et al. Mol Pain. 2013;9:64-64.</div><p class="links"><a class="dblinks" href="/pubmed/24313960" ref="ordinalpos=4914&amp;ncbi_uid=5707555&amp;link_uid=24313960">Citation</a><a class="dblinks" href="/pmc/articles/PMC3892125/" ref="ordinalpos=4914&amp;ncbi_uid=5707555&amp;link_uid=PMC3892125">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4915.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3737553/figure/gkt467-F6/" ref="ordinalpos=4915&amp;ncbi_uid=5178069&amp;link_uid=PMC3737553" href="/pmc/articles/PMC3737553/"><img src="/pmc/articles/instance/3737553/bin/gkt467f6p.gif" src-large="/pmc/articles/instance/3737553/bin/gkt467f6p.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3737553/" ref="ordinalpos=4915&amp;ncbi_uid=5178069&amp;link_uid=PMC3737553" image-link="/pmc/articles/PMC3737553/figure/gkt467-F6/" class="imagepopup">Figure 6. From: A novel interplay between the Fanconi anemia core complex and ATR-ATRIP kinase during DNA cross-link repair. </a></p><div class="supp"><p class="details"><div>Our proposed model for two distinct pathways in ICL-induced ATR <span class="highlight" style="background-color:">signaling</span>. RPA complex accumulates on exposed ssDNA region surrounding the ICL (middle). RPA-ATRIP binding, which is facilitated by the FA core complex, promotes ATR recruitment. In the left side of the <span class="highlight" style="background-color:">pathway</span>, which is dependent on the FA core complex but not on the RAD17-TOPBP1 <span class="highlight" style="background-color:">pathway</span>, FANCI is phosphorylated, triggering FANCD2 monoubiquitination. In the right side, CHK1 is phosphorylated depending on the RAD17-TOPBP1 <span class="highlight" style="background-color:">pathway</span>, but the FA core complex is largely dispensable for the CHK1 activation.</div></p></div><div class="aux"><div class="resc">Junya Tomida, et al. Nucleic Acids Res. 2013 August;41(14):6930-6941.</div><p class="links"><a class="dblinks" href="/pubmed/23723247" ref="ordinalpos=4915&amp;ncbi_uid=5178069&amp;link_uid=23723247">Citation</a><a class="dblinks" href="/pmc/articles/PMC3737553/" ref="ordinalpos=4915&amp;ncbi_uid=5178069&amp;link_uid=PMC3737553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4916.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2443685/figure/F12/" ref="ordinalpos=4916&amp;ncbi_uid=1415005&amp;link_uid=PMC2443685" href="/pmc/articles/PMC2443685/"><img src="/pmc/articles/instance/2443685/bin/nihms54637f12.gif" src-large="/pmc/articles/instance/2443685/bin/nihms54637f12.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2443685/" ref="ordinalpos=4916&amp;ncbi_uid=1415005&amp;link_uid=PMC2443685" image-link="/pmc/articles/PMC2443685/figure/F12/" class="imagepopup">Figure 12. Schematic representation of a <span class="highlight" style="background-color:">pathway</span> for protection of cardiomyocytes by EET via the PI3K/Akt pro-survival <span class="highlight" style="background-color:">pathway</span>.  From: Multiple anti-apoptotic targets of the PI3K-Akt survival <span class="highlight" style="background-color:">pathway</span> are activated by epoxyeicosatrienoic acids to protect cardiomyocytes from hypoxia/anoxia. </a></p><div class="supp"><p class="details"><div>We have reported that pretreatment with EET increases (i) PI3K activity (ii) phosphorylation of Akt (iii) phosphorylation of BAD (iv) intracellular XIAP and (v) inhibition of activity of caspases-9. Each of these events decreases activity of caspase 3, a late effector, to protect the cardiomyocyte from apoptosis by HR. These targets are known to align in a <span class="highlight" style="background-color:">signaling</span> cascade as illustrated in the schematic. It should be noted that the results do not implicate absence of other pro-survival or mitotic pathways that may also be targeted by EETs.</div></p></div><div class="aux"><div class="resc">Anuradha Dhanasekaran, et al. Am J Physiol Heart Circ Physiol. ;294(2):H724-H735.</div><p class="links"><a class="dblinks" href="/pubmed/18055514" ref="ordinalpos=4916&amp;ncbi_uid=1415005&amp;link_uid=18055514">Citation</a><a class="dblinks" href="/pmc/articles/PMC2443685/" ref="ordinalpos=4916&amp;ncbi_uid=1415005&amp;link_uid=PMC2443685">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4917.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1398071/figure/f2/" ref="ordinalpos=4917&amp;ncbi_uid=690632&amp;link_uid=PMC1398071" href="/pmc/articles/PMC1398071/"><img src="/pmc/articles/instance/1398071/bin/zek0030626420002.gif" src-large="/pmc/articles/instance/1398071/bin/zek0030626420002.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1398071/" ref="ordinalpos=4917&amp;ncbi_uid=690632&amp;link_uid=PMC1398071" image-link="/pmc/articles/PMC1398071/figure/f2/" class="imagepopup">FIG. 2. From: The RIM101 <span class="highlight" style="background-color:">Pathway</span> Contributes to Yeast Cell Wall Assembly and Its Function Becomes Essential in the Absence of Mitogen-Activated Protein Kinase Slt2p. </a></p><div class="supp"><p class="details"><div>Activation of the PKC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> under different conditions. (A) Time course of Slt2p phosphorylation after calcofluor treatment. Samples were collected at the indicated times after the addition of calcofluor. The levels of total Slt2p are shown as a loading control. (B) Slt2p phosphorylation in wild-type and <i>fks1Δ</i> strains growing logarithmically or after calcofluor treatment. (C) Comparative levels of Slt2p phosphorylation in wild-type and <i>rim21Δ</i> strains after caffeine, calcofluor, or thermal stress. WT, wild type.</div></p></div><div class="aux"><div class="resc">F. Castrejon, et al. Eukaryot Cell. 2006 March;5(3):507-517.</div><p class="links"><a class="dblinks" href="/pubmed/16524906" ref="ordinalpos=4917&amp;ncbi_uid=690632&amp;link_uid=16524906">Citation</a><a class="dblinks" href="/pmc/articles/PMC1398071/" ref="ordinalpos=4917&amp;ncbi_uid=690632&amp;link_uid=PMC1398071">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4918.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3424514/figure/F6/" ref="ordinalpos=4918&amp;ncbi_uid=4186578&amp;link_uid=PMC3424514" href="/pmc/articles/PMC3424514/"><img src="/pmc/articles/instance/3424514/bin/nihms396413f6.gif" src-large="/pmc/articles/instance/3424514/bin/nihms396413f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3424514/" ref="ordinalpos=4918&amp;ncbi_uid=4186578&amp;link_uid=PMC3424514" image-link="/pmc/articles/PMC3424514/figure/F6/" class="imagepopup">Figure 6.  From: The rearranged during transfection/papillary thyroid carcinoma tyrosine kinase is an estrogen-dependent gene required for the growth of estrogen receptor positive breast cancer cells. </a></p><div class="supp"><p class="details"><div>Effect of estrogen upregulation and siRNA knockdown of RET/PTC kinase expression on downstream kinase <span class="highlight" style="background-color:">signaling</span> pathways. Hormone-depleted MCF-7 cells were transfected with siRNA against RET/PTC kinase or siRNA against luciferase for 36h. Then cells were switched into serum free IMEM with E2 or vehicle for 24h. Before harvesting, cells were treated with physiological level (1ng/ml) of the RET/PTC kinase ligand glial cell line-derived neurotrophic factor (GDNF) for 10min to stimulate the RET/PTC kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Western blot analysis of both phosphorylated and unphosphorylated kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components is shown.</div></p></div><div class="aux"><div class="resc">Chunyu Wang, et al. Breast Cancer Res Treat. ;133(2):487-500.</div><p class="links"><a class="dblinks" href="/pubmed/21947652" ref="ordinalpos=4918&amp;ncbi_uid=4186578&amp;link_uid=21947652">Citation</a><a class="dblinks" href="/pmc/articles/PMC3424514/" ref="ordinalpos=4918&amp;ncbi_uid=4186578&amp;link_uid=PMC3424514">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4919.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3148700/figure/F4/" ref="ordinalpos=4919&amp;ncbi_uid=3449436&amp;link_uid=PMC3148700" href="/pmc/articles/PMC3148700/"><img src="/pmc/articles/instance/3148700/bin/asn0081147830004.gif" src-large="/pmc/articles/instance/3148700/bin/asn0081147830004.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3148700/" ref="ordinalpos=4919&amp;ncbi_uid=3449436&amp;link_uid=PMC3148700" image-link="/pmc/articles/PMC3148700/figure/F4/" class="imagepopup">Figure 4. From: Calcium Mediates Glomerular Filtration through Calcineurin and mTORC2/Akt <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Protamine sulfate activates <span class="highlight" style="background-color:">signaling</span> through the calcineurin <span class="highlight" style="background-color:">pathway</span>. (A) Western blots show treatment with 600 μg/ml PS reduces the amount of full-length (110 kD) Synpo present in glomerular lysates. Pretreatment with 10 μM E64 prevents this reduction in full-length Synpo. (B and C) Pretreatment with 24 μg/ml FK506 (B) or 50 μg/ml cyclosporine A (C) reduces PS-mediated glomerular injury. (D) Treatment with 10 μM CatL inhibitor E64 partially blocks albumin release upon PS treatment.</div></p></div><div class="aux"><div class="resc">John Vassiliadis, et al. J Am Soc Nephrol. 2011 August;22(8):1453-1461.</div><p class="links"><a class="dblinks" href="/pubmed/21784900" ref="ordinalpos=4919&amp;ncbi_uid=3449436&amp;link_uid=21784900">Citation</a><a class="dblinks" href="/pmc/articles/PMC3148700/" ref="ordinalpos=4919&amp;ncbi_uid=3449436&amp;link_uid=PMC3148700">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4920.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3151103/figure/F9/" ref="ordinalpos=4920&amp;ncbi_uid=3653444&amp;link_uid=PMC3151103" href="/pmc/articles/PMC3151103/"><img src="/pmc/articles/instance/3151103/bin/zbc0381174950009.gif" src-large="/pmc/articles/instance/3151103/bin/zbc0381174950009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3151103/" ref="ordinalpos=4920&amp;ncbi_uid=3653444&amp;link_uid=PMC3151103" image-link="/pmc/articles/PMC3151103/figure/F9/" class="imagepopup">FIGURE 9. From: Nicotinamide Phosphoribosyltransferase Is Essential for Interleukin-1?-mediated Dedifferentiation of Articular Chondrocytes via SIRT1 and Extracellular Signal-regulated Kinase (ERK) Complex <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic summary of a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> underlying IL-1β-induced dedifferentiation of primary cultured articular chondrocytes.</b> <i>A</i>, schematic summary of SIRT1 function either as downstream of NAMPT <span class="highlight" style="background-color:">signaling</span> or as upstream of ERK and p38 kinase. <i>B</i>, 0.25 μg of GST, 0.5 μg of GST-ERK2, or 2.0 μg of GST-pERK fusion proteins were initially bound with glutathione-agarose beads for 2 h, and then each complex was incubated with the active form of 0.5 μg of His-SIRT1 fusion proteins for 12 h. The protein complexes were detected by Western blotting with anti-GST (<i>top</i>) or anti-His antibody (<i>bottom</i>).</div></p></div><div class="aux"><div class="resc">Eun-Hee Hong, et al. J Biol Chem. 2011 August 12;286(32):28619-28631.</div><p class="links"><a class="dblinks" href="/pubmed/21697093" ref="ordinalpos=4920&amp;ncbi_uid=3653444&amp;link_uid=21697093">Citation</a><a class="dblinks" href="/pmc/articles/PMC3151103/" ref="ordinalpos=4920&amp;ncbi_uid=3653444&amp;link_uid=PMC3151103">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4921.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3425953/figure/F3/" ref="ordinalpos=4921&amp;ncbi_uid=4190611&amp;link_uid=PMC3425953" href="/pmc/articles/PMC3425953/"><img src="/pmc/articles/instance/3425953/bin/nihms398364f3.gif" src-large="/pmc/articles/instance/3425953/bin/nihms398364f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3425953/" ref="ordinalpos=4921&amp;ncbi_uid=4190611&amp;link_uid=PMC3425953" image-link="/pmc/articles/PMC3425953/figure/F3/" class="imagepopup">Figure 3.  From: High-Throughput Screening Strategies for Targeted Identification Small-Molecule Compounds Towards Activated Pathways in Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>The activity of PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal and tumor cells. Left panel represents the activity of the <span class="highlight" style="background-color:">pathway</span> in the normal intestinal epithelial cells. Right panel shows three models of hyperactivation of the PI3K <span class="highlight" style="background-color:">pathway</span> in the colorectal cancer cells caused by EGFR copy number increased, PI3K and PTEN mutations, respectively. Red star – activation; red crossed lines – inactivation; dashed lines – irrelevant signal.</div></p></div><div class="aux"><div class="resc">Agnieszka B. Bialkowska, et al. Future Oncol. ;8(3):259-272.</div><p class="links"><a class="dblinks" href="/pubmed/22409463" ref="ordinalpos=4921&amp;ncbi_uid=4190611&amp;link_uid=22409463">Citation</a><a class="dblinks" href="/pmc/articles/PMC3425953/" ref="ordinalpos=4921&amp;ncbi_uid=4190611&amp;link_uid=PMC3425953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4922.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2002614/figure/fig9/" ref="ordinalpos=4922&amp;ncbi_uid=1084992&amp;link_uid=PMC2002614" href="/pmc/articles/PMC2002614/"><img src="/pmc/articles/instance/2002614/bin/tpc1902454f09.gif" src-large="/pmc/articles/instance/2002614/bin/tpc1902454f09.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2002614/" ref="ordinalpos=4922&amp;ncbi_uid=1084992&amp;link_uid=PMC2002614" image-link="/pmc/articles/PMC2002614/figure/fig9/" class="imagepopup">Figure 9. From: N-Acylethanolamine Metabolism Interacts with Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> in Arabidopsis thaliana Seedlings. </a></p><div class="supp"><p class="details"><div>Diagram Showing the Hypothetical Action of Elevated Levels of NAE on Seedling Growth and Its Relationship to ABA Action.<br />NAE arrests seedling growth via a <span class="highlight" style="background-color:">pathway</span> dependent upon <i>ABI1</i>, <i>ABI2</i>, <i>ABI3</i>, and <i>ABI5</i>, and this is associated with upregulation of ABA-responsive genes, such as At <i>HVA22b</i>, <i>CRA1</i>, <i>RAB18</i>, <i>EXGT-A4</i>, <i>ATEM1</i>, and <i>LEAM17</i>. This suppression of seedling growth is reminiscent of the ABA-induced secondary dormancy program discovered by Chua and coworkers (Lopez-Molina et al., 2002), which is operable in young seedlings. We propose that NAE12:0 intersects the ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream from ABA and that <i>ABI3</i> expression is key to the suppression of growth induced by NAE metabolism. Independent of ABA <span class="highlight" style="background-color:">signaling</span>, NAE12:0 can influence gene expression in seeds and seedlings and suppress seedling growth. Ectopic overexpression of FAAH can reverse the growth suppression by NAE, but the interaction with ABA <span class="highlight" style="background-color:">signaling</span> in FAAH overexpressors is complex since FAAH overexpressors are hypersensitive to ABA. By contrast, the situation with <i>abi</i> mutants is more straightforward because they exhibited partial tolerance to NAE12:0 in addition to their ABA-tolerant phenotypes.</div></p></div><div class="aux"><div class="resc">Neal D. Teaster, et al. Plant Cell. 2007 August;19(8):2454-2469.</div><p class="links"><a class="dblinks" href="/pubmed/17766402" ref="ordinalpos=4922&amp;ncbi_uid=1084992&amp;link_uid=17766402">Citation</a><a class="dblinks" href="/pmc/articles/PMC2002614/" ref="ordinalpos=4922&amp;ncbi_uid=1084992&amp;link_uid=PMC2002614">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4923.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1383554/figure/f10/" ref="ordinalpos=4923&amp;ncbi_uid=660587&amp;link_uid=PMC1383554" href="/pmc/articles/PMC1383554/"><img src="/pmc/articles/instance/1383554/bin/7600973f10.gif" src-large="/pmc/articles/instance/1383554/bin/7600973f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1383554/" ref="ordinalpos=4923&amp;ncbi_uid=660587&amp;link_uid=PMC1383554" image-link="/pmc/articles/PMC1383554/figure/f10/" class="imagepopup">Figure 10.  From: MAPKAPK-2-mediated LIM-kinase activation is critical for VEGF-induced actin remodeling and cell migration. </a></p><div class="supp"><p class="details"><div>A proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for VEGF-induced LIMK1 activation and cofilin phosphorylation. VEGF induces activation of MK2 via the MKK6 and p38 MAPK <span class="highlight" style="background-color:">signaling</span> cascade. MK2 activates LIMK1 by phosphorylation of Ser-323, which in turn stimulates phosphorylation of cofilin. Together with MK2-mediated Hsp27 phosphorylation, cofilin phosphorylation stimulates stress fiber formation, cell migration, and tube formation.</div></p></div><div class="aux"><div class="resc">Miho Kobayashi, et al. EMBO J. 2006 February 22;25(4):713-726.</div><p class="links"><a class="dblinks" href="/pubmed/16456544" ref="ordinalpos=4923&amp;ncbi_uid=660587&amp;link_uid=16456544">Citation</a><a class="dblinks" href="/pmc/articles/PMC1383554/" ref="ordinalpos=4923&amp;ncbi_uid=660587&amp;link_uid=PMC1383554">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4924.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3294277/figure/F8/" ref="ordinalpos=4924&amp;ncbi_uid=3841632&amp;link_uid=PMC3294277" href="/pmc/articles/PMC3294277/"><img src="/pmc/articles/instance/3294277/bin/nihms358061f8.gif" src-large="/pmc/articles/instance/3294277/bin/nihms358061f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3294277/" ref="ordinalpos=4924&amp;ncbi_uid=3841632&amp;link_uid=PMC3294277" image-link="/pmc/articles/PMC3294277/figure/F8/" class="imagepopup">Figure 8.  From: Formation and <span class="highlight" style="background-color:">Signaling</span> Actions of Electrophilic Lipids. </a></p><div class="supp"><p class="details"><div>Regulation of insulin sensitivity by electrophiles through adduction and inhibition of PTP1B. Phosphatases play a critical role in maintaining cellular homeostasis by regulating and limiting phosphorylation and downstream <span class="highlight" style="background-color:">signaling</span>. In particular, the insulin receptor and its <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are tightly regulated by phosphorylation and dephosphorylation reactions. Activation of the insulin receptor leads to its phosphorylation and triggers the phosphorylation of the IRS protein family, resulting in downstream activation of kinases and modulation of metabolic enzymes and pathways. PTP1B exerts a tight control of these phosphorylation events. Adduction of PTP1B by electrophiles at the catalytic cysteine results in catalytic inhibition, prolonging the effects and intensity of regulated pathways, thus impacting glucose homeostasis and insulin sensitivity. Electrophiles modulate this <span class="highlight" style="background-color:">pathway</span> both directly (through covalent binding to PTP1B) and indirectly (through activation of the general metabolic regulator PPARγ).</div></p></div><div class="aux"><div class="resc">Francisco J. Schopfer, et al. Chem Rev. ;111(10):5997-6021.</div><p class="links"><a class="dblinks" href="/pubmed/21928855" ref="ordinalpos=4924&amp;ncbi_uid=3841632&amp;link_uid=21928855">Citation</a><a class="dblinks" href="/pmc/articles/PMC3294277/" ref="ordinalpos=4924&amp;ncbi_uid=3841632&amp;link_uid=PMC3294277">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4925.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3929081/figure/F2/" ref="ordinalpos=4925&amp;ncbi_uid=5829307&amp;link_uid=PMC3929081" href="/pmc/articles/PMC3929081/"><img src="/pmc/articles/instance/3929081/bin/nihms519779f2.gif" src-large="/pmc/articles/instance/3929081/bin/nihms519779f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3929081/" ref="ordinalpos=4925&amp;ncbi_uid=5829307&amp;link_uid=PMC3929081" image-link="/pmc/articles/PMC3929081/figure/F2/" class="imagepopup">Figure 2.  From: Differential Tumorigenic Potential and Matriptase Activation between PDGF B versus PDGF D in Prostate Cancer. </a></p><div class="supp"><p class="details"><div>PDGF D–specific <span class="highlight" style="background-color:">signaling</span> activates the JNK <span class="highlight" style="background-color:">pathway</span>. PDGF B- and PDGF D–overexpressing mPrEC (A) and BPH-1 (B) cells were serum starved for 48 hours and whole-cell lysate harvested to assess the MAPK <span class="highlight" style="background-color:">signaling</span> cascade, as indicated. BPH-1 (C) and mPrE cells (D) were used to further characterize upstream and downstream components of the JNK <span class="highlight" style="background-color:">pathway</span> as indicated. E, PDGF D mPrEC or BPH-1 cells were transduced with scrambled or β-PDGFR (shβR) shRNA. β-PDGFR knockdown was assessed via RT-PCR and its effects on Erk and JNK activation.</div></p></div><div class="aux"><div class="resc">Abdo J. Najy, et al. Mol Cancer Res. ;10(8):1087-1097.</div><p class="links"><a class="dblinks" href="/pubmed/22689130" ref="ordinalpos=4925&amp;ncbi_uid=5829307&amp;link_uid=22689130">Citation</a><a class="dblinks" href="/pmc/articles/PMC3929081/" ref="ordinalpos=4925&amp;ncbi_uid=5829307&amp;link_uid=PMC3929081">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4926.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3221730/figure/F2/" ref="ordinalpos=4926&amp;ncbi_uid=3671501&amp;link_uid=PMC3221730" href="/pmc/articles/PMC3221730/"><img src="/pmc/articles/instance/3221730/bin/nihms324864f2.gif" src-large="/pmc/articles/instance/3221730/bin/nihms324864f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3221730/" ref="ordinalpos=4926&amp;ncbi_uid=3671501&amp;link_uid=PMC3221730" image-link="/pmc/articles/PMC3221730/figure/F2/" class="imagepopup">Fig. 2.  From: 14-3-3 Proteins: Diverse Functions in Cell Proliferation and Cancer Progression. </a></p><div class="supp"><p class="details"><div>14-3-3 negatively regulates Hippo <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. (A) In the canonical Hippo <span class="highlight" style="background-color:">pathway</span>, Lats generates a phosphorylation-dependent 14-3-3 binding site on YAP/TAZ under conditions of high cell density. The binding of 14-3-3 sequesters YAP/TAZ in the cytosol. NPHP4 can complete with YAP/TAZ for Lats binding, thus blocking Lats-mediated phosphorylation of YAP/TAZ and 14-3-3 binding. (B) In non-canonical Hippo <span class="highlight" style="background-color:">signaling</span>, α-catenin forms a complex with 14-3-3-bound YAP under high cell density, thus stabilizing the 14-3-3/YAP interaction. Under low cell density conditions, PP2A dephosphorylates the 14-3-3 docking site on YAP, resulting in its nuclear import.</div></p></div><div class="aux"><div class="resc">Alyson K. Freeman, et al. Semin Cell Dev Biol. ;22(7):681-687.</div><p class="links"><a class="dblinks" href="/pubmed/21884813" ref="ordinalpos=4926&amp;ncbi_uid=3671501&amp;link_uid=21884813">Citation</a><a class="dblinks" href="/pmc/articles/PMC3221730/" ref="ordinalpos=4926&amp;ncbi_uid=3671501&amp;link_uid=PMC3221730">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4927.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3023525/figure/F8/" ref="ordinalpos=4927&amp;ncbi_uid=3610509&amp;link_uid=PMC3023525" href="/pmc/articles/PMC3023525/"><img src="/pmc/articles/instance/3023525/bin/zbc0061146680008.gif" src-large="/pmc/articles/instance/3023525/bin/zbc0061146680008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3023525/" ref="ordinalpos=4927&amp;ncbi_uid=3610509&amp;link_uid=PMC3023525" image-link="/pmc/articles/PMC3023525/figure/F8/" class="imagepopup">FIGURE 8. From: Fractalkine Attenuates Excito-neurotoxicity via Microglial Clearance of Damaged Neurons and Antioxidant Enzyme Heme Oxygenase-1 Expression. </a></p><div class="supp"><p class="details"><div><b>Model of the role of sFKN in microglial phagocytosis and neuroprotection.</b> sFKN, which is secreted from neurons that are damaged by glutamate, promotes microglial phagocytosis of neuronal debris through the release of MFG-E8. sFKN also induces the expression of the antioxidant enzyme HO-1 in microglia via Nrf2 recruitment and activation of the JNK MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The neuroprotective effects of sFKN are also mediated in part by activation of ERK MAPK, although the downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has not yet been elucidated. Therefore, sFKN may be an intrinsic neuroprotectant for damaged yet surviving neurons.</div></p></div><div class="aux"><div class="resc">Mariko Noda, et al. J Biol Chem. 2011 January 21;286(3):2308-2319.</div><p class="links"><a class="dblinks" href="/pubmed/21071446" ref="ordinalpos=4927&amp;ncbi_uid=3610509&amp;link_uid=21071446">Citation</a><a class="dblinks" href="/pmc/articles/PMC3023525/" ref="ordinalpos=4927&amp;ncbi_uid=3610509&amp;link_uid=PMC3023525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4928.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4302448/figure/F1/" ref="ordinalpos=4928&amp;ncbi_uid=7101599&amp;link_uid=PMC4302448" href="/pmc/articles/PMC4302448/"><img src="/pmc/articles/instance/4302448/bin/NRR-7-1688-g001.gif" src-large="/pmc/articles/instance/4302448/bin/NRR-7-1688-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4302448/" ref="ordinalpos=4928&amp;ncbi_uid=7101599&amp;link_uid=PMC4302448" image-link="/pmc/articles/PMC4302448/figure/F1/" class="imagepopup">Figure 1.  From: Changes in expression and secretion patterns of fibroblast growth factor 8 and Sonic Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecules during murine neural stem/progenitor cell differentiation in vitro☆. </a></p><div class="supp"><p class="details"><div>Dynamic expression of FGF8, FGFRs and Sonic Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecule mRNA during neural stem/progenitor cells differentiation <i>in vitro</i> was measured on day, 10 and 20 by reverse transcription-PCR.<br /><sup>a</sup><i>P</i> &lt; 0.05, <i>vs.</i> expression of the previous differentiation stage. The results were expressed as absorbance ratio of mRNA expression on day 10 or 20 to that of the neural stem/progenitor cell stage, which was assigned a value of 1 (mean ± SD, <i>n</i> = 4).<br />FGF: Fibroblast growth factor; FGFR: fibroblast growth factor receptor; NSCs/NPCs: neural stem/progenitor cells.</div></p></div><div class="aux"><div class="resc">Jiang Lu, et al. Neural Regen Res. 2012 August 5;7(22):1688-1694.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4928&amp;ncbi_uid=7101599&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4302448/" ref="ordinalpos=4928&amp;ncbi_uid=7101599&amp;link_uid=PMC4302448">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4929.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4007181/figure/fig3/" ref="ordinalpos=4929&amp;ncbi_uid=6623501&amp;link_uid=PMC4007181" href="/pmc/articles/PMC4007181/"><img src="/pmc/articles/instance/4007181/bin/aps2009119f3.gif" src-large="/pmc/articles/instance/4007181/bin/aps2009119f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4007181/" ref="ordinalpos=4929&amp;ncbi_uid=6623501&amp;link_uid=PMC4007181" image-link="/pmc/articles/PMC4007181/figure/fig3/" class="imagepopup">Figure 3.  From: Rosiglitazone sensitizes hepatocellular carcinoma cell lines to 5-fluorouracil antitumor activity through activation of the PPAR? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The effect of rosiglitazone on PTEN and COX-2 protein expression. Cellular protein was isolated from BEL-7402 cells (A) or Huh 7 cells (C) that were cultured with rosiglitazone (30 μmol/L) in the presence or absence of 5-FU (10 μmol/L) for 24 h. Then, western blotting analysis was used to detect the PTEN and COX-2 protein expression. (B) The PPARγ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was inactivated by PPARγ siRNA method in BEL-7402 cells (B) or Huh 7 cells (D). Western blotting analysis was used to detect the PTEN and COX-2 protein expression of BEL-7402 cells treated with rosiglitazone (30 μmol/L) with or without 5-FU (10 μmol/L) for 24 h. Results are representative of three independent experiments. Con, cells treated with 0.1% DMSO; Con siRNA, non-specific siRNA, as a negative control.</div></p></div><div class="aux"><div class="resc">Liang-qi Cao, et al. Acta Pharmacol Sin. 2009 September;30(9):1316-1322.</div><p class="links"><a class="dblinks" href="/pubmed/19684609" ref="ordinalpos=4929&amp;ncbi_uid=6623501&amp;link_uid=19684609">Citation</a><a class="dblinks" href="/pmc/articles/PMC4007181/" ref="ordinalpos=4929&amp;ncbi_uid=6623501&amp;link_uid=PMC4007181">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4930.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3814639/figure/F2/" ref="ordinalpos=4930&amp;ncbi_uid=6032993&amp;link_uid=PMC3814639" href="/pmc/articles/PMC3814639/"><img src="/pmc/articles/instance/3814639/bin/2179fig2.gif" src-large="/pmc/articles/instance/3814639/bin/2179fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3814639/" ref="ordinalpos=4930&amp;ncbi_uid=6032993&amp;link_uid=PMC3814639" image-link="/pmc/articles/PMC3814639/figure/F2/" class="imagepopup">Figure 2. From: The emerging role of the Nrf2-Keap1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancer. </a></p><div class="supp"><p class="details"><div>Schematic model of the Nrf2–Keap1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Under basal conditions, Keap1 binds to the ETGE and DLG motifs on Nrf2 and brings Nrf2 into Keap1–Cul3–E3 ubiquitin ligase complex, leading to ubiquitination and subsequent degradation of Nrf2. Oxidative stress or electrophiles can cause a conformational change in the Keap1–Cul3–E3 ubiquitin ligase by acting on specific cysteine residues in Keap1. These changes disrupt Nrf2–Keap1 binding at the DLG domain. Nrf2 is stabilized, and free Nrf2 translocates to the nucleus, where it dimerizes with members of the small Maf family and binds to AREs (5′-RTGABNNNGCR-3′) within regulatory regions of a wide variety of cell defense genes, including NQO1, GCLM, HO-1, and MRP1. (E) ETGE; (D) DLG.</div></p></div><div class="aux"><div class="resc">Melba C. Jaramillo, et al. Genes Dev. 2013 October 15;27(20):2179-2191.</div><p class="links"><a class="dblinks" href="/pubmed/24142871" ref="ordinalpos=4930&amp;ncbi_uid=6032993&amp;link_uid=24142871">Citation</a><a class="dblinks" href="/pmc/articles/PMC3814639/" ref="ordinalpos=4930&amp;ncbi_uid=6032993&amp;link_uid=PMC3814639">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4931.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2869326/figure/ppat-1000889-g008/" ref="ordinalpos=4931&amp;ncbi_uid=2356894&amp;link_uid=PMC2869326" href="/pmc/articles/PMC2869326/"><img src="/pmc/articles/instance/2869326/bin/ppat.1000889.g008.gif" src-large="/pmc/articles/instance/2869326/bin/ppat.1000889.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2869326/" ref="ordinalpos=4931&amp;ncbi_uid=2356894&amp;link_uid=PMC2869326" image-link="/pmc/articles/PMC2869326/figure/ppat-1000889-g008/" class="imagepopup">Figure 8. set3Δ/Δ cells are a hyperreactive to cAMP/PKA induction by GlcNAc..  From: The Set3/Hos2 Histone Deacetylase Complex Attenuates cAMP/PKA <span class="highlight" style="background-color:">Signaling</span> to Regulate Morphogenesis and Virulence of Candida albicans. </a></p><div class="supp"><p class="details"><div>(A) Colony morphologies of wild type and <i>set3</i>Δ/Δ cells grown on YPD at 30°C for 3 days in the presence of the indicated amounts of N-acetylglucosamine (GlcNAc). Scale bar corresponds to 2 mm. (B) The “threshold shift” model for Set3C function in triggering morphogenesis. The cAMP/PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> transmits environmental information, thereby shaping the morphogenetic change. In wild type cells (left panel), the sensitivity of the <span class="highlight" style="background-color:">pathway</span> to adequate signals is antagonized by the SetC. If the Set3C is disrupted or impaired (right panel), the threshold for morphogenetic conversion is shifted, and the <span class="highlight" style="background-color:">pathway</span> responds to milder inducing stimuli by triggering filamentation. In addition, metabolites such as adenine also modulates the activity of the <span class="highlight" style="background-color:">pathway</span> through as yet undisclosed mechanisms.</div></p></div><div class="aux"><div class="resc">Denes Hnisz, et al. PLoS Pathog. 2010 May;6(5):e1000889.</div><p class="links"><a class="dblinks" href="/pubmed/20485517" ref="ordinalpos=4931&amp;ncbi_uid=2356894&amp;link_uid=20485517">Citation</a><a class="dblinks" href="/pmc/articles/PMC2869326/" ref="ordinalpos=4931&amp;ncbi_uid=2356894&amp;link_uid=PMC2869326">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4932.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3533556/figure/F7/" ref="ordinalpos=4932&amp;ncbi_uid=4493061&amp;link_uid=PMC3533556" href="/pmc/articles/PMC3533556/"><img src="/pmc/articles/instance/3533556/bin/JCI64896.f7.gif" src-large="/pmc/articles/instance/3533556/bin/JCI64896.f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3533556/" ref="ordinalpos=4932&amp;ncbi_uid=4493061&amp;link_uid=PMC3533556" image-link="/pmc/articles/PMC3533556/figure/F7/" class="imagepopup">Figure 7. Scheme for CLCA1/MAPK13 <span class="highlight" style="background-color:">signaling</span> to mucus production..  From: IL-13-induced airway mucus production is attenuated by MAPK13 inhibition. </a></p><div class="supp"><p class="details"><div>The CLCA1/MAPK13 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is initiated when environmental stimuli (e.g., allergen or virus) stimulate IL-13 production and subsequent activation of IL-13 receptor on the surface of mucous precursor cells. IL-13 receptor activation leads to STAT6 activation and consequent induction of <i>CLCA1</i> gene expression. The CLCA1 protein is secreted as N-terminal and C-terminal CLCA1 (N-CLCA1 and C-CLCA1) peptide fragments that might activate a putative CLCA1 receptor and in turn MAPK13 to drive a consequent increase in mucin (especially <i>MUC5AC</i>) gene expression and mucus production in airway mucous cells. </div></p></div><div class="aux"><div class="resc">Yael G. Alevy, et al. J Clin Invest. 2012 December 3;122(12):4555-4568.</div><p class="links"><a class="dblinks" href="/pubmed/23187130" ref="ordinalpos=4932&amp;ncbi_uid=4493061&amp;link_uid=23187130">Citation</a><a class="dblinks" href="/pmc/articles/PMC3533556/" ref="ordinalpos=4932&amp;ncbi_uid=4493061&amp;link_uid=PMC3533556">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4933.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4100156/figure/ijms-15-10365-f002/" ref="ordinalpos=4933&amp;ncbi_uid=6382272&amp;link_uid=PMC4100156" href="/pmc/articles/PMC4100156/"><img src="/pmc/articles/instance/4100156/bin/ijms-15-10365-g002.gif" src-large="/pmc/articles/instance/4100156/bin/ijms-15-10365-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4100156/" ref="ordinalpos=4933&amp;ncbi_uid=6382272&amp;link_uid=PMC4100156" image-link="/pmc/articles/PMC4100156/figure/ijms-15-10365-f002/" class="imagepopup">Figure 2.  From: Up-Regulation of Rhoa/Rho Kinase <span class="highlight" style="background-color:">Pathway</span> by Translationally Controlled Tumor Protein in Vascular Smooth Muscle Cells. </a></p><div class="supp"><p class="details"><div>Role of TCTP in the RhoA-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in TCTP-transgenic (TCTP-TG) mice. (<b>a</b>) Representative photomicrographs of immunohistochemistry (IHC) detecting TCTP and p-MLC in the aorta from wild-type (WT) and TCTP-TG mice. Immunostaining of aortas from WT and TCTP-TG were performed with antibodies detecting TCTP and phosphorylated MLC (p-MLC). Nuclei were counterstained blue with hematoxylin reagent (<i>N</i> = 3); (<b>b</b>) Western blot analysis of aortic tissues was done as described in Materials and Methods using anti-TCTP, -RhoA, -p-MLC, -MLC, and -β-actin-specific antibodies.</div></p></div><div class="aux"><div class="resc">Jeehye Maeng, et al. Int J Mol Sci. 2014 June;15(6):10365-10376.</div><p class="links"><a class="dblinks" href="/pubmed/24918292" ref="ordinalpos=4933&amp;ncbi_uid=6382272&amp;link_uid=24918292">Citation</a><a class="dblinks" href="/pmc/articles/PMC4100156/" ref="ordinalpos=4933&amp;ncbi_uid=6382272&amp;link_uid=PMC4100156">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4934.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137255/figure/cjc-30-01-013-g001/" ref="ordinalpos=4934&amp;ncbi_uid=6137744&amp;link_uid=PMC3137255" href="/pmc/articles/PMC3137255/"><img src="/pmc/articles/instance/4012260/bin/cjc-30-01-013-g001.gif" src-large="/pmc/articles/instance/4012260/bin/cjc-30-01-013-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137255/" ref="ordinalpos=4934&amp;ncbi_uid=6137744&amp;link_uid=PMC3137255" image-link="/pmc/articles/PMC3137255/figure/cjc-30-01-013-g001/" class="imagepopup">Figure 1.Major milestones in the studies of hedgehog <span class="highlight" style="background-color:">signaling</span> as related to human diseases, particularly cancer. For all references, please see the text for details..  From: Clinical implications of hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors. </a></p><div class="aux"><div class="resc">Hailan Liu, et al. Chin J Cancer. 2011 January;30(1):13-26.</div><p class="links"><a class="dblinks" href="/pubmed/21192841" ref="ordinalpos=4934&amp;ncbi_uid=6137744&amp;link_uid=21192841">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137255/" ref="ordinalpos=4934&amp;ncbi_uid=6137744&amp;link_uid=PMC3137255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4935.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633809/figure/F12/" ref="ordinalpos=4935&amp;ncbi_uid=6091953&amp;link_uid=PMC3633809" href="/pmc/articles/PMC3633809/"><img src="/pmc/articles/instance/3633809/bin/z380051391980012.gif" src-large="/pmc/articles/instance/3633809/bin/z380051391980012.jpg" alt="Figure 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633809/" ref="ordinalpos=4935&amp;ncbi_uid=6091953&amp;link_uid=PMC3633809" image-link="/pmc/articles/PMC3633809/figure/F12/" class="imagepopup">Figure 12. From: LPA1-induced cytoskeleton reorganization drives fibrosis through CTGF-dependent fibroblast proliferation. </a></p><div class="supp"><p class="details"><div>Proposed profibrotic collaboration between mesothelial cells and fibroblasts in peritoneal fibrosis. LPA-LPA<sub>1</sub> <span class="highlight" style="background-color:">signaling</span> drives mesothelial cell CTGF expression through a Gα<sub>12/13</sub>-RhoA-ROCK-MRTF-SRF <span class="highlight" style="background-color:">pathway</span>, and this mesothelial CTGF, in turn, drives fibroblast proliferation.</div></p></div><div class="aux"><div class="resc">Norihiko Sakai, et al. FASEB J. 2013 May;27(5):1830-1846.</div><p class="links"><a class="dblinks" href="/pubmed/23322166" ref="ordinalpos=4935&amp;ncbi_uid=6091953&amp;link_uid=23322166">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633809/" ref="ordinalpos=4935&amp;ncbi_uid=6091953&amp;link_uid=PMC3633809">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4936.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163143/figure/fig2/" ref="ordinalpos=4936&amp;ncbi_uid=3190903&amp;link_uid=PMC3163143" href="/pmc/articles/PMC3163143/"><img src="/pmc/articles/instance/3163143/bin/JO2012-782020.002.gif" src-large="/pmc/articles/instance/3163143/bin/JO2012-782020.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163143/" ref="ordinalpos=4936&amp;ncbi_uid=3190903&amp;link_uid=PMC3163143" image-link="/pmc/articles/PMC3163143/figure/fig2/" class="imagepopup">Figure 2.  From: Targeting Angiogenesis for Controlling Neuroblastoma. </a></p><div class="supp"><p class="details"><div>Schematic presentation of molecular changes associated with N-Myc amplification in neuroblastoma. Amplified N-Myc after heterodimerization with Max augments synthesis of several genes and through PI3K/Akt-mediated <span class="highlight" style="background-color:">pathway</span> regulates VEGF <span class="highlight" style="background-color:">signaling</span> resulting in neuroblastoma tumorigenesis.</div></p></div><div class="aux"><div class="resc">Subhasree Roy Choudhury, et al. J Oncol. 2012;2012:782020.</div><p class="links"><a class="dblinks" href="/pubmed/21876694" ref="ordinalpos=4936&amp;ncbi_uid=3190903&amp;link_uid=21876694">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163143/" ref="ordinalpos=4936&amp;ncbi_uid=3190903&amp;link_uid=PMC3163143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4937.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2841319/figure/F1/" ref="ordinalpos=4937&amp;ncbi_uid=2439656&amp;link_uid=PMC2841319" href="/pmc/articles/PMC2841319/"><img src="/pmc/articles/instance/2841319/bin/nihms184270f1.gif" src-large="/pmc/articles/instance/2841319/bin/nihms184270f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2841319/" ref="ordinalpos=4937&amp;ncbi_uid=2439656&amp;link_uid=PMC2841319" image-link="/pmc/articles/PMC2841319/figure/F1/" class="imagepopup">Figure 1. A schematic showing TGF beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its interaction with other molecules like PDGF, CD44 and cyclins through transcription factor Egr-1.  From: Pathogenesis of kidney disease in systemic lupus erythematosus. </a></p><div class="supp"><p class="details"><div>PDGF, platelet-derived growth factor.</div></p></div><div class="aux"><div class="resc">Harini Bagavant, et al. Curr Opin Rheumatol. ;21(5):489-494.</div><p class="links"><a class="dblinks" href="/pubmed/19584729" ref="ordinalpos=4937&amp;ncbi_uid=2439656&amp;link_uid=19584729">Citation</a><a class="dblinks" href="/pmc/articles/PMC2841319/" ref="ordinalpos=4937&amp;ncbi_uid=2439656&amp;link_uid=PMC2841319">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4938.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3720582/figure/pone-0069132-g001/" ref="ordinalpos=4938&amp;ncbi_uid=5183321&amp;link_uid=PMC3720582" href="/pmc/articles/PMC3720582/"><img src="/pmc/articles/instance/3720582/bin/pone.0069132.g001.gif" src-large="/pmc/articles/instance/3720582/bin/pone.0069132.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3720582/" ref="ordinalpos=4938&amp;ncbi_uid=5183321&amp;link_uid=PMC3720582" image-link="/pmc/articles/PMC3720582/figure/pone-0069132-g001/" class="imagepopup">Figure 1. Reconstructed Hedgehog <span class="highlight" style="background-color:">Pathway</span>..  From: Structural and Logical Analysis of a Comprehensive Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> to Identify Alternative Drug Targets for Glioma, Colon and Pancreatic Cancer. </a></p><div class="supp"><p class="details"><div>Total 57 proteins included in this <span class="highlight" style="background-color:">pathway</span> figure. The green and red arrows are indicating Activation/Production and Inhibition process respectively. The black arrows indicate the nuclear translocation process. All the proteins of this network are allocated into four main regions with different color codes: Extracellular and Membrane (Blue); Cytoplasm (Red); Nucleus (Green); and Output (Yellow). The output proteins are linked with various cellular responses (cross talk with other pathways or phenotypic expressions) with black dotted arrow.</div></p></div><div class="aux"><div class="resc">Saikat Chowdhury, et al. PLoS One. 2013;8(7):e69132.</div><p class="links"><a class="dblinks" href="/pubmed/23935937" ref="ordinalpos=4938&amp;ncbi_uid=5183321&amp;link_uid=23935937">Citation</a><a class="dblinks" href="/pmc/articles/PMC3720582/" ref="ordinalpos=4938&amp;ncbi_uid=5183321&amp;link_uid=PMC3720582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4939.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1851929/figure/F8/" ref="ordinalpos=4939&amp;ncbi_uid=934471&amp;link_uid=PMC1851929" href="/pmc/articles/PMC1851929/"><img src="/pmc/articles/instance/1851929/bin/nihms18962f8.gif" src-large="/pmc/articles/instance/1851929/bin/nihms18962f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1851929/" ref="ordinalpos=4939&amp;ncbi_uid=934471&amp;link_uid=PMC1851929" image-link="/pmc/articles/PMC1851929/figure/F8/" class="imagepopup">Fig. 8.  From: Murine coronavirus-induced oligodendrocyte apoptosis is mediated through the activation of the Fas <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed model for the activation of oligodendrocyte apoptosis by MHV infection. (a) Interaction between the viral S protein and MHV receptor (MHVR). (b) Fusion between viral envelope and cell membrane. (c) Integration of viral envelope proteins (E, M, S, S2) into the cell membrane after fusion. (d) Activation of the Fas signal <span class="highlight" style="background-color:">pathway</span> by virus–cell fusion and the post-fusion complexes. This is likely the major <span class="highlight" style="background-color:">pathway</span> leading to activation of the caspase-8 and mitochondrial apoptotic pathways. (e) An alternative <span class="highlight" style="background-color:">pathway</span> of the activation of the Fas <span class="highlight" style="background-color:">signaling</span>. Solid lines with arrows indicate the steps (pathways) that are supported by experimental data, whereas dashed lines with arrows denote the hypothetic pathways with little or no experimental data. The following blocking reagents were used in the experiments to establish the individual steps of the pathways: the CC1 (anti-MHVR antibody), αS (anti-S protein antibody), Fas/Fc (chimeric soluble receptor), C8IN (caspase-8 inhibitor), C9IN (caspase-9 inhibitor) and Bcl-2 or Bcl-xL.</div></p></div><div class="aux"><div class="resc">Yin Liu, et al. Virology. ;360(2):364-375.</div><p class="links"><a class="dblinks" href="/pubmed/17156812" ref="ordinalpos=4939&amp;ncbi_uid=934471&amp;link_uid=17156812">Citation</a><a class="dblinks" href="/pmc/articles/PMC1851929/" ref="ordinalpos=4939&amp;ncbi_uid=934471&amp;link_uid=PMC1851929">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4940.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1183594/figure/fig3/" ref="ordinalpos=4940&amp;ncbi_uid=573068&amp;link_uid=PMC1183594" href="/pmc/articles/PMC1183594/"><img src="/pmc/articles/instance/1183594/bin/zpq0310590760003.gif" src-large="/pmc/articles/instance/1183594/bin/zpq0310590760003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1183594/" ref="ordinalpos=4940&amp;ncbi_uid=573068&amp;link_uid=PMC1183594" image-link="/pmc/articles/PMC1183594/figure/fig3/" class="imagepopup">Fig. 3. From: The doublesex-related gene, XDmrt4, is required for neurogenesis in the olfactory system. </a></p><div class="supp"><p class="details"><div>Caudalizing factors restrict <i>XDmrt4</i> expression domain. Activation of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by injection of Wnt-3a blocks <i>XDmrt4</i>, <i>XBF1</i>, and <i>Six1</i> expression at the neurula stage. Conversely, inhibition of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by injection of β-catenin morpholino (β-cat-mo) or Xdkk1 mRNA expands posteriorly <i>XDmrt4</i>, <i>XBF1</i>, and <i>Six1</i> placodal expression domains. At the tail bud stage, the same manipulations of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulted in a reduction or an expansion of <i>Xebf2</i> and <i>XEmx2</i> expression in the nasal placode. In most cases, Xdkk1 injection leads to an expansion of these markers on both sides of the injected embryos presumably because of Xdkk1 protein diffusion. Overexpression of Fgf8 completely blocks <i>XDmrt4</i>, <i>XBF1</i>, and <i>Six1</i> placodal expression at the neurula stage, as well as the nasal placode expression of <i>Xebf2</i> and <i>XEmx2</i> at the tail bud stage. Anterior views are shown, dorsal to the top. RNA encoding the lineage tracer β-gal was coinjected in all cases to identify the injected side (red staining, left, arrows). (Magnifications: ×8.)</div></p></div><div class="aux"><div class="resc">Xiao Huang, et al. Proc Natl Acad Sci U S A. 2005 August 9;102(32):11349-11354.</div><p class="links"><a class="dblinks" href="/pubmed/16061812" ref="ordinalpos=4940&aamp;ncbi_uid=573068&amp;link_uid=16061812">Citation</a><a class="dblinks" href="/pmc/articles/PMC1183594/" ref="ordinalpos=4940&amp;ncbi_uid=573068&amp;link_uid=PMC1183594">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4941.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3434350/figure/fig1/" ref="ordinalpos=4941&amp;ncbi_uid=5247705&amp;link_uid=PMC3434350" href="/pmc/articles/PMC3434350/"><img src="/pmc/articles/instance/3434350/bin/cr201285f1.gif" src-large="/pmc/articles/instance/3434350/bin/cr201285f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3434350/" ref="ordinalpos=4941&amp;ncbi_uid=5247705&amp;link_uid=PMC3434350" image-link="/pmc/articles/PMC3434350/figure/fig1/" class="imagepopup">Figure 1.  From: Found in translation of mTOR <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Translational impact of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Multiple cellular pathways are regulated by <span class="highlight" style="background-color:">signaling</span> from mTOR complexes, including regulation of autophagy, lipid metabolism, and control of actin cytoskeleton in addition to the regulation of translation. Translational control by mTOR is regulated predominantly by <span class="highlight" style="background-color:">signaling</span> from mTORC1, and mediated by phosphorylation of 4E-BPs and S6K, which directly influence the function of eIF4E and RpS6, respectively. Now, two papers by Hsieh <i>et al</i>. and Thoreen <i>et al</i>. demonstrate that mTOR <span class="highlight" style="background-color:">signaling</span> to the ribosome can regulate a specific subset of mRNA transcripts, the majority of which contain a 5′ TOP, a 5′ TOP-like or a novel PRTE element in their 5′ UTR regions. While Thoreen <i>et al</i>. now establish an important mTOR-sensitive regulation of eIF4G1 downstream of 4E-BPs, Hsieh <i>et al</i>. focus on a subset of mRNA transcripts (YB-1, CD44, MTA1 and vimentin) strongly linked to regulation of metastasis, and propose that they play a role in the progression of <i>Pten</i>-loss-driven prostate cancer through hyperactivation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. While, the ATP site inhibitor INK128 was shown to inhibit tumor progression in mice, it should be noted that such inhibitors also block all activities of mTORC1 and 2, and while they can successfully block the translational up-regulation of the 4-gene metastasis signature described by Hsieh <i>et al</i>., they also block other metastatic pathways regulated by independently controlled mTOR pathways.</div></p></div><div class="aux"><div class="resc">John G Clohessy, et al. Cell Res. 2012 September;22(9):1315-1318.</div><p class="links"><a class="dblinks" href="/pubmed/22641373" ref="ordinalpos=4941&amp;ncbi_uid=5247705&amp;link_uid=22641373">Citation</a><a class="dblinks" href="/pmc/articles/PMC3434350/" ref="ordinalpos=4941&amp;ncbi_uid=5247705&amp;link_uid=PMC3434350">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4942.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3682584/figure/F9/" ref="ordinalpos=4942&amp;ncbi_uid=6246341&amp;link_uid=PMC3682584" href="/pmc/articles/PMC3682584/"><img src="/pmc/articles/instance/3682584/bin/zbc0291352880009.gif" src-large="/pmc/articles/instance/3682584/bin/zbc0291352880009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3682584/" ref="ordinalpos=4942&amp;ncbi_uid=6246341&amp;link_uid=PMC3682584" image-link="/pmc/articles/PMC3682584/figure/F9/" class="imagepopup">FIGURE 9. From: Fibroblast Growth Factor Receptor Like-1 (FGFRL1) Interacts with SHP-1 Phosphatase at Insulin Secretory Granules and Induces Beta-cell ERK1/2 Protein Activation. </a></p><div class="supp"><p class="details"><div><b>Proposed mechanisms of FGFRL1-directed <span class="highlight" style="background-color:">signaling</span> via SHP-1.</b> Canonical FGF receptors (<i>i.e.</i> FGFR1) activate downstream MAPK <span class="highlight" style="background-color:">signaling</span> events via ligand-induced receptor dimerization and <i>trans</i>-autophosphorylation. Conversely, association of SHP-1 with FGFRL1 at insulin secretory granules may localize the phosphatase, enabling up-regulation of the MAPK <span class="highlight" style="background-color:">pathway</span> (<i>i.e.</i> enhanced ERK1/2 phosphorylation). Alternatively, FGFRL1 at the plasma membrane may bind extracellular FGF ligand to elevate ERK1/2 phosphorylation by means of a MEK-independent <span class="highlight" style="background-color:">signaling</span> cascade.</div></p></div><div class="aux"><div class="resc">Pamuditha N. Silva, et al. J Biol Chem. 2013 June 14;288(24):17859-17870.</div><p class="links"><a class="dblinks" href="/pubmed/23640895" ref="ordinalpos=4942&amp;ncbi_uid=6246341&amp;link_uid=23640895">Citation</a><a class="dblinks" href="/pmc/articles/PMC3682584/" ref="ordinalpos=4942&amp;ncbi_uid=6246341&amp;link_uid=PMC3682584">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4943.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g001/" ref="ordinalpos=4943&amp;ncbi_uid=6232470&amp;link_uid=PMC4041745" href="/pmc/articles/PMC4041745/"><img src="/pmc/articles/instance/4041745/bin/pone.0098229.g001.gif" src-large="/pmc/articles/instance/4041745/bin/pone.0098229.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041745/" ref="ordinalpos=4943&amp;ncbi_uid=6232470&amp;link_uid=PMC4041745" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g001/" class="imagepopup">Figure 1. Flow chart of the <span class="highlight" style="background-color:">pathway</span> enrichment analysis of the GWAS on BC survival..  From: Systematic <span class="highlight" style="background-color:">Pathway</span> Enrichment Analysis of a Genome-Wide Association Study on Breast Cancer Survival Reveals an Influence of Genes Involved in Cell Adhesion and Calcium <span class="highlight" style="background-color:">Signaling</span> on the Patients' Clinical Outcome. </a></p><div class="aux"><div class="resc">Andrea Woltmann, et al. PLoS One. 2014;9(6):e98229.</div><p class="links"><a class="dblinks" href="/pubmed/24886783" ref="ordinalpos=4943&amp;ncbi_uid=6232470&amp;link_uid=24886783">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041745/" ref="ordinalpos=4943&amp;ncbi_uid=6232470&amp;link_uid=PMC4041745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4944.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1998874/figure/Fig4/" ref="ordinalpos=4944&amp;ncbi_uid=1071775&amp;link_uid=PMC1998874" href="/pmc/articles/PMC1998874/"><img src="/pmc/articles/instance/1998874/bin/335_2007_9040_Fig4_HTML.gif" src-large="/pmc/articles/instance/1998874/bin/335_2007_9040_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1998874/" ref="ordinalpos=4944&amp;ncbi_uid=1071775&amp;link_uid=PMC1998874" image-link="/pmc/articles/PMC1998874/figure/Fig4/" class="imagepopup">Fig. 4.  From: Moving toward a system genetics view of disease. </a></p><div class="supp"><p class="details"><div>Genes comprising simple linearly ordered pathways operate in a network context. <b>A </b>The classic view of TGF-β <span class="highlight" style="background-color:">signaling</span> (Alberts 2002) involves Tgfbr2 as a key component. <i>Tgfbr2</i> was recently identified and validated as an obesity susceptibility gene. <b>B </b>The genes comprising the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are correlated with hundreds of other genes in the liver network (Schadt et al. 2005) so that components of this <span class="highlight" style="background-color:">pathway</span> affect and are affected by many different parts of the network</div></p></div><div class="aux"><div class="resc">Solveig K. Sieberts, et al. Mamm Genome. 2007 July;18(6-7):389-401.</div><p class="links"><a class="dblinks" href="/pubmed/17653589" ref="ordinalpos=4944&amp;ncbi_uid=1071775&amp;link_uid=17653589">Citation</a><a class="dblinks" href="/pmc/articles/PMC1998874/" ref="ordinalpos=4944&amp;ncbi_uid=1071775&amp;link_uid=PMC1998874">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4945.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2875818/figure/F8/" ref="ordinalpos=4945&amp;ncbi_uid=2978019&amp;link_uid=PMC2875818" href="/pmc/articles/PMC2875818/"><img src="/pmc/articles/instance/2875818/bin/zee0061053620008.gif" src-large="/pmc/articles/instance/2875818/bin/zee0061053620008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2875818/" ref="ordinalpos=4945&amp;ncbi_uid=2978019&amp;link_uid=PMC2875818" image-link="/pmc/articles/PMC2875818/figure/F8/" class="imagepopup">Figure 8.  From: Luteinizing Hormone Stimulates Mammalian Target of Rapamycin <span class="highlight" style="background-color:">Signaling</span> in Bovine Luteal Cells via Pathways Independent of AKT and Mitogen-Activated Protein Kinase: Modulation of Glycogen Synthase Kinase 3 and AMP-Activated Protein Kinase. </a></p><div class="supp"><p class="details"><div>Proposed model of LH-mediated regulation of MTOR <span class="highlight" style="background-color:">signaling</span> in luteal cells. The results described are consistent with the model in which LH activates the LH receptor and adenylyl cyclase, which leads to an increase in cAMP. As a result, there is an increase in the phosphorylation and inactivation of GSK3B. At the same time, there is a reduction in the activation of AMPK. These events reduce the activity of the TSC, allowing MTOR to phosphorylate components of the translational apparatus, S6K1 and 4EBP1. Treatment with rapamycin inhibits MTOR activation without altering progesterone secretion. Treatment with AICAR activates AMPK, inhibits MTOR phosphorylation of S6K1, and inhibits progesterone secretion. The results suggest that LH uses a unique <span class="highlight" style="background-color:">pathway</span> to activate MTOR <span class="highlight" style="background-color:">signaling</span> in bovine luteal cells, but the rapamycin-sensitive MTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> does not contribute to corpus luteum progesterone synthesis.</div></p></div><div class="aux"><div class="resc">Xiaoying Hou, et al. Endocrinology. 2010 June;151(6):2846-2857.</div><p class="links"><a class="dblinks" href="/pubmed/20351317" ref="ordinalpos=4945&amp;ncbi_uid=2978019&amp;link_uid=20351317">Citation</a><a class="dblinks" href="/pmc/articles/PMC2875818/" ref="ordinalpos=4945&amp;ncbi_uid=2978019&amp;link_uid=PMC2875818">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4946.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4058521/figure/fig1/" ref="ordinalpos=4946&amp;ncbi_uid=6306333&amp;link_uid=PMC4058521" href="/pmc/articles/PMC4058521/"><img src="/pmc/articles/instance/4058521/bin/BMRI2014-504808.001.gif" src-large="/pmc/articles/instance/4058521/bin/BMRI2014-504808.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4058521/" ref="ordinalpos=4946&amp;ncbi_uid=6306333&amp;link_uid=PMC4058521" image-link="/pmc/articles/PMC4058521/figure/fig1/" class="imagepopup">Figure 1.  From: Roles of Renal Proximal Tubule Transport in Acid/Base Balance and Blood Pressure Regulation. </a></p><div class="supp"><p class="details"><div>Ang II <span class="highlight" style="background-color:">signaling</span> in mouse and human PTs. In mouse PTs, low concentrations of Ang II induce transport stimulation via either PKC activation or decrease in intracellular cAMP resulting in ERK activation, while high concentrations of Ang II induce transport inhibition via NO/cGMP/cGKII <span class="highlight" style="background-color:">pathway</span>. In human PTs, by contrast, Ang II induces dose-dependent transport stimulation via NO/cGMP/ERK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Motonobu Nakamura, et al. Biomed Res Int. 2014;2014:504808.</div><p class="links"><a class="dblinks" href="/pubmed/24982885" ref="ordinalpos=4946&amp;ncbi_uid=6306333&amp;link_uid=24982885">Citation</a><a class="dblinks" href="/pmc/articles/PMC4058521/" ref="ordinalpos=4946&amp;ncbi_uid=6306333&amp;link_uid=PMC4058521">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4947.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4128657/figure/pone-0104202-g003/" ref="ordinalpos=4947&amp;ncbi_uid=6491098&amp;link_uid=PMC4128657" href="/pmc/articles/PMC4128657/"><img src="/pmc/articles/instance/4128657/bin/pone.0104202.g003.gif" src-large="/pmc/articles/instance/4128657/bin/pone.0104202.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4128657/" ref="ordinalpos=4947&amp;ncbi_uid=6491098&amp;link_uid=PMC4128657" image-link="/pmc/articles/PMC4128657/figure/pone-0104202-g003/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> networks of genes that are associated with IFN responsiveness..  From: An Interferon Response Gene Signature Is Associated with the Therapeutic Response of Hepatitis C Patients. </a></p><div class="supp"><p class="details"><div>The gene network was generated using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software.</div></p></div><div class="aux"><div class="resc">Lawrence M. Pfeffer, et al. PLoS One. 2014;9(8):e104202.</div><p class="links"><a class="dblinks" href="/pubmed/25111807" ref="ordinalpos=4947&amp;ncbi_uid=6491098&amp;link_uid=25111807">Citation</a><a class="dblinks" href="/pmc/articles/PMC4128657/" ref="ordinalpos=4947&amp;ncbi_uid=6491098&amp;link_uid=PMC4128657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4948.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3866415/figure/F14/" ref="ordinalpos=4948&amp;ncbi_uid=6058147&amp;link_uid=PMC3866415" href="/pmc/articles/PMC3866415/"><img src="/pmc/articles/instance/3866415/bin/zns9991346540014.gif" src-large="/pmc/articles/instance/3866415/bin/zns9991346540014.jpg" alt="Figure 14." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3866415/" ref="ordinalpos=4948&amp;ncbi_uid=6058147&amp;link_uid=PMC3866415" image-link="/pmc/articles/PMC3866415/figure/F14/" class="imagepopup">Figure 14. From: Soluble Guanylate Cyclase Generation of cGMP Regulates Migration of MGE Neurons. </a></p><div class="supp"><p class="details"><div>Schematic diagram proposing the mechanism of cGMP (generated by sGC) mediated neurite extension via inhibition of the Rho <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in MGE neurons.</div></p></div><div class="aux"><div class="resc">Shyamali Mandal, et al. J Neurosci. 2013 October 23;33(43):16897-16914.</div><p class="links"><a class="dblinks" href="/pubmed/24155296" ref="ordinalpos=4948&amp;ncbi_uid=6058147&amp;link_uid=24155296">Citation</a><a class="dblinks" href="/pmc/articles/PMC3866415/" ref="ordinalpos=4948&amp;ncbi_uid=6058147&amp;link_uid=PMC3866415">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4949.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3562341/figure/pone-0050695-g004/" ref="ordinalpos=4949&amp;ncbi_uid=4595009&amp;link_uid=PMC3562341" href="/pmc/articles/PMC3562341/"><img src="/pmc/articles/instance/3562341/bin/pone.0050695.g004.gif" src-large="/pmc/articles/instance/3562341/bin/pone.0050695.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3562341/" ref="ordinalpos=4949&amp;ncbi_uid=4595009&amp;link_uid=PMC3562341" image-link="/pmc/articles/PMC3562341/figure/pone-0050695-g004/" class="imagepopup">Figure 4. T cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Accumulation of Splice Variants and Transcripts in Response to PI3K Inhibition in T Cells. </a></p><div class="supp"><p class="details"><div>Green indicates a down-regulated gene, red indicates an up-regulated gene. Spliced genes are marked in grey.</div></p></div><div class="aux"><div class="resc">Alice Riedel, et al. PLoS One. 2013;8(2):e50695.</div><p class="links"><a class="dblinks" href="/pubmed/23383294" ref="ordinalpos=4949&amp;ncbi_uid=4595009&amp;link_uid=23383294">Citation</a><a class="dblinks" href="/pmc/articles/PMC3562341/" ref="ordinalpos=4949&amp;ncbi_uid=4595009&amp;link_uid=PMC3562341">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4950.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3237086/figure/F1/" ref="ordinalpos=4950&amp;ncbi_uid=4406527&amp;link_uid=PMC3237086" href="/pmc/articles/PMC3237086/"><img src="/pmc/articles/instance/3237086/bin/zjw0121140060001.gif" src-large="/pmc/articles/instance/3237086/bin/zjw0121140060001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3237086/" ref="ordinalpos=4950&amp;ncbi_uid=4406527&amp;link_uid=PMC3237086" image-link="/pmc/articles/PMC3237086/figure/F1/" class="imagepopup">Fig. 1. From: Quantitative Chemical Proteomics Reveals New Potential Drug Targets in Head and Neck Cancer. </a></p><div class="supp"><p class="details"><div><b>Quantitative chemical proteomics strategy for the identification of kinase drug targets or <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> biomarkers.</b></div></p></div><div class="aux"><div class="resc">Zhixiang Wu, et al. Mol Cell Proteomics. 2011 December;10(12):M111.011635.</div><p class="links"><a class="dblinks" href="/pubmed/21955398" ref="ordinalpos=4950&amp;ncbi_uid=4406527&amp;link_uid=21955398">Citation</a><a class="dblinks" href="/pmc/articles/PMC3237086/" ref="ordinalpos=4950&amp;ncbi_uid=4406527&amp;link_uid=PMC3237086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4951.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2715837/figure/F1/" ref="ordinalpos=4951&amp;ncbi_uid=1973305&amp;link_uid=PMC2715837" href="/pmc/articles/PMC2715837/"><img src="/pmc/articles/instance/2715837/bin/nihms124211f1.gif" src-large="/pmc/articles/instance/2715837/bin/nihms124211f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2715837/" ref="ordinalpos=4951&amp;ncbi_uid=1973305&amp;link_uid=PMC2715837" image-link="/pmc/articles/PMC2715837/figure/F1/" class="imagepopup">Figure 1.  From: G-Protein-Coupled Receptor Kinase 4 Polymorphisms and Blood Pressure Response to Metoprolol Among African Americans: Sex-Specificity and Interactions. </a></p><div class="supp"><p class="details"><div>Schematic of <i>GRK4</i> <span class="highlight" style="background-color:">pathway</span>. <i>GRK4</i> phosphorylates G-protein-coupled receptor proteins (GPCRs), such as β-adrenoreceptors, resulting in subsequent binding of β-arrestin and uncoupling of GPCRs mediated intracellular <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Vibha Bhatnagar, et al. Am J Hypertens. ;22(3):332-338.</div><p class="links"><a class="dblinks" href="/pubmed/19119263" ref="ordinalpos=4951&amp;ncbi_uid=1973305&amp;link_uid=19119263">Citation</a><a class="dblinks" href="/pmc/articles/PMC2715837/" ref="ordinalpos=4951&amp;ncbi_uid=1973305&amp;link_uid=PMC2715837">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4952.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1208922/figure/F1/" ref="ordinalpos=4952&amp;ncbi_uid=623907&amp;link_uid=PMC1208922" href="/pmc/articles/PMC1208922/"><img src="/pmc/articles/instance/1208922/bin/1477-3163-4-16-1.gif" src-large="/pmc/articles/instance/1208922/bin/1477-3163-4-16-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1208922/" ref="ordinalpos=4952&amp;ncbi_uid=623907&amp;link_uid=PMC1208922" image-link="/pmc/articles/PMC1208922/figure/F1/" class="imagepopup">Figure 1.  From: Sulindac metabolites inhibit epidermal growth factor receptor activation and expression. </a></p><div class="supp"><p class="details"><div><b>The EGFR/MAPK/ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. We have previously shown that sulindac metabolites inhibit both MEK1/2 and ERK1/2 activation. The goal of this study was to determine if this inhibition is due to downregulation of the EGFR.</div></p></div><div class="aux"><div class="resc">Heather A Pangburn, et al. J Carcinog. 2005;4:16-16.</div><p class="links"><a class="dblinks" href="/pubmed/16138927" ref="ordinalpos=4952&amp;ncbi_uid=623907&amp;link_uid=16138927">Citation</a><a class="dblinks" href="/pmc/articles/PMC1208922/" ref="ordinalpos=4952&amp;ncbi_uid=623907&amp;link_uid=PMC1208922">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4953.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3268919/figure/F8/" ref="ordinalpos=4953&amp;ncbi_uid=4568295&amp;link_uid=PMC3268919" href="/pmc/articles/PMC3268919/"><img src="/pmc/articles/instance/3268919/bin/nihms341577f8.gif" src-large="/pmc/articles/instance/3268919/bin/nihms341577f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3268919/" ref="ordinalpos=4953&amp;ncbi_uid=4568295&amp;link_uid=PMC3268919" image-link="/pmc/articles/PMC3268919/figure/F8/" class="imagepopup">Fig. 8. Schematic diagram showing the mechanism by which uPA/uPAR shRNA inhibits angiogenesis in glioblastoma cells and xenografts.  From: uPA and uPAR shRNA Inhibit Angiogenesis via Enhanced Secretion of SVEGFR1 Independent of GM-CSF but Dependent on TIMP-1 in Endothelial and Glioblastoma Cells. </a></p><div class="supp"><p class="details"><div>Panel (A) Binding of uPA to uPAR coordinates with GM-CSF receptor to activate JAK2/STAT5 mediated angiogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. VEGF/VEGFR <span class="highlight" style="background-color:">signaling</span> is also well known to initiate angiogenesis. Panel (B) Knockdown of uPA and uPAR inhibits the angiogenesis <span class="highlight" style="background-color:">signaling</span> induced by both GM-CSF and VEGF. uPA/uPAR down regulation decreased GM-CSF secretion resulting in inhibition of JAK2/STAT5 activation. Further knock down of uPA/uPAR increased sVEGFR1 secretion which binds to free bound VEGF and acts as a scavenger inhibiting VEGF mediated <span class="highlight" style="background-color:">signaling</span>. sVEGFR1 secretion was mediated by TIMP-1 and ectodomain shedding of VEGFR1 regulated by phosphorylation at the Tyr1213 of the VEGFR1 receptor.</div></p></div><div class="aux"><div class="resc">Hari Raghu, et al. Mol Oncol. ;6(1):33-47.</div><p class="links"><a class="dblinks" href="/pubmed/22177802" ref="ordinalpos=4953&amp;ncbi_uid=4568295&amp;link_uid=22177802">Citation</a><a class="dblinks" href="/pmc/articles/PMC3268919/" ref="ordinalpos=4953&amp;ncbi_uid=4568295&amp;link_uid=PMC3268919">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4954.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3863418/figure/f05/" ref="ordinalpos=4954&amp;ncbi_uid=5606112&amp;link_uid=PMC3863418" href="/pmc/articles/PMC3863418/"><img src="/pmc/articles/instance/3863418/bin/bio-02-12-1336-f05.gif" src-large="/pmc/articles/instance/3863418/bin/bio-02-12-1336-f05.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3863418/" ref="ordinalpos=4954&amp;ncbi_uid=5606112&amp;link_uid=PMC3863418" image-link="/pmc/articles/PMC3863418/figure/f05/" class="imagepopup">Fig. 5.Impaired Notch <span class="highlight" style="background-color:">signaling</span> in K5-Ets1 BT skin..  From: Aberrant epidermal differentiation and disrupted ?Np63/Notch regulatory axis in Ets1 transgenic mice. </a></p><div class="supp"><p class="details"><div>(A) Quantitative real-time PCR to measure mRNA levels of genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Data are represented as mean ± SEM. *<i>P</i>&lt;0.05, Student's <i>t</i>-test. (B–E) Immunofluorescent staining for Notch1 in the skin of E14.5, E16.5 and E18.5 wild-type and E18.5 K5-Ets1 BT embryos (all green). (F–K) Immunofluorescent staining for Notch2, Notch3 and Hes1 in the skin of E18.5 wild-type and K5-Ets1 BT embryos (all green). White arrows point to membrane-associated staining of Notch2 and Notch3. Each section was co-stained with TOPRO-3 to mark nuclei (blue). Scale bars in all cases are 37.5 µm.</div></p></div><div class="aux"><div class="resc">Shu Shien Chin, et al. Biol Open. 2013 December 15;2(12):1336-1345.</div><p class="links"><a class="dblinks" href="/pubmed/24337118" ref="ordinalpos=4954&amp;ncbi_uid=5606112&amp;link_uid=24337118">Citation</a><a class="dblinks" href="/pmc/articles/PMC3863418/" ref="ordinalpos=4954&amp;ncbi_uid=5606112&amp;link_uid=PMC3863418">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4955.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432117/figure/pone-0044570-g002/" ref="ordinalpos=4955&amp;ncbi_uid=4217912&amp;link_uid=PMC3432117" href="/pmc/articles/PMC3432117/"><img src="/pmc/articles/instance/3432117/bin/pone.0044570.g002.gif" src-large="/pmc/articles/instance/3432117/bin/pone.0044570.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432117/" ref="ordinalpos=4955&amp;ncbi_uid=4217912&amp;link_uid=PMC3432117" image-link="/pmc/articles/PMC3432117/figure/pone-0044570-g002/" class="imagepopup">Figure 2. BMS309403 stimulates glucose uptake in C2C12 myotubes and AMPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in a dose dependent manner..  From: BMS309403 Stimulates Glucose Uptake in Myotubes through Activation of AMP-Activated Protein Kinase. </a></p><div class="supp"><p class="details"><div>C2C12 myotubes were starved for 2 hours in serum free high glucose DMEM containing 0.2%BSA followed by incubation for 2 h with the indicated concentrations of BMS309403. (A) 2-deoxy-D[1-<sup>3</sup>H]-glucose was added for 15 min at 37°C, followed by glucose uptake measurement. (B) Western blotting analysis of AMPK and Akt <span class="highlight" style="background-color:">signaling</span> molecules.</div></p></div><div class="aux"><div class="resc">Wanhua Lin, et al. PLoS One. 2012;7(8):e44570.</div><p class="links"><a class="dblinks" href="/pubmed/22952994" ref="ordinalpos=4955&amp;ncbi_uid=4217912&amp;link_uid=22952994">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432117/" ref="ordinalpos=4955&amp;ncbi_uid=4217912&amp;link_uid=PMC3432117">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4956.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4168054/figure/Fig4/" ref="ordinalpos=4956&amp;ncbi_uid=6633714&amp;link_uid=PMC4168054" href="/pmc/articles/PMC4168054/"><img src="/pmc/articles/instance/4168054/bin/12985_2014_2488_Fig4_HTML.gif" src-large="/pmc/articles/instance/4168054/bin/12985_2014_2488_Fig4_HTML.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4168054/" ref="ordinalpos=4956&amp;ncbi_uid=6633714&amp;link_uid=PMC4168054" image-link="/pmc/articles/PMC4168054/figure/Fig4/" class="imagepopup">Figure 4.  From: Activation of Pro-survival CaMK4?/CREB and Pro-death MST1 <span class="highlight" style="background-color:">signaling</span> at early and late times during a mouse model of prion disease. </a></p><div class="supp"><p class="details"><div> <b>Identification of two <span class="highlight" style="background-color:">signaling</span> pathways of potential interest involved in neuronal survival and death. (A)</b> Hierarchical clustering the normalized and log<sub>2</sub> transformed densitometric data of the expression levels of 109 protein kinases detected in primary multiplex Western blots of brainstem-cerebellums of scrapie-infected mice at 70, 90, 110, 130 dpi or at terminal stage of disease (TER). Red, higher expression level; green, lower expression level; grey, no data (protein kinases that were not resolved, not detected, or not quantitated due to transfer or blotting artifacts). Cluster (<i>i</i>) consists of protein kinases involved in the NMDAR-regulated CaMK4β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Cluster (<i>ii</i>) consists of protein kinases involved in the MST1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The protein kinases highlighted in Figure 2, CaMK4β and DLK, are indicated by (●), and MKK7 is indicated by (◄). <b>(B)</b> Expression levels of protein kinases involved in the CaMK4β and MST1 <span class="highlight" style="background-color:">signaling</span> pathways, and included in the primary multiplex Western blots, at 70 dpi (top panel) and 130 dpi (bottom panel), respectively. Color bars indicate the levels in each of the three scrapie-infected mice normalized to those in mock-infected mice.</div></p></div><div class="aux"><div class="resc">Rory H Shott, et al. Virol J. 2014;11(1):160.</div><p class="links"><a class="dblinks" href="/pubmed/25183307" ref="ordinalpos=4956&amp;ncbi_uid=6633714&amp;link_uid=25183307">Citation</a><a class="dblinks" href="/pmc/articles/PMC4168054/" ref="ordinalpos=4956&amp;ncbi_uid=6633714&amp;link_uid=PMC4168054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4957.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3754252/figure/F1/" ref="ordinalpos=4957&amp;ncbi_uid=5264238&amp;link_uid=PMC3754252" href="/pmc/articles/PMC3754252/"><img src="/pmc/articles/instance/3754252/bin/JCI67364.f1.gif" src-large="/pmc/articles/instance/3754252/bin/JCI67364.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3754252/" ref="ordinalpos=4957&amp;ncbi_uid=5264238&amp;link_uid=PMC3754252" image-link="/pmc/articles/PMC3754252/figure/F1/" class="imagepopup">Figure 1. FA <span class="highlight" style="background-color:">signaling</span> is essential for the SAC..  From: Fanconi anemia <span class="highlight" style="background-color:">signaling</span> network regulates the spindle assembly checkpoint. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) RNAi screen schematic. (<b>B</b>) Microscopy-based screen readout. HeLa<sup>GFP-H2B/mCherry-αtubulin</sup> cells arrest in prometaphase upon taxol exposure. CDC25A knockdown results in premitotic arrest (25). <i>MAD2</i> siRNA induces multinucleation due to SAC failure. Original magnification, ×200 (BD <span class="highlight" style="background-color:">Pathway</span> 855). (<b>C</b>) Representative images of cells transfected with indicated siRNAs and treated with taxol. Note multinucleation induced by <i>FANCA</i> and <i>FANCC</i> siRNAs and <i>MAD2</i> siRNA (positive control). Original magnification, ×200 (BD <span class="highlight" style="background-color:">Pathway</span> 855). (<b>D</b>) Quantification of screen results. *<i>P</i> &lt; 0.0001 (1-way ANOVA); <i>n</i> = 9 counts per siRNA. All bars represent mean values ± SEM. </div></p></div><div class="aux"><div class="resc">Grzegorz Nalepa, et al. J Clin Invest. 2013 September 3;123(9):3839-3847.</div><p class="links"><a class="dblinks" href="/pubmed/23934222" ref="ordinalpos=4957&amp;ncbi_uid=5264238&amp;link_uid=23934222">Citation</a><a class="dblinks" href="/pmc/articles/PMC3754252/" ref="ordinalpos=4957&amp;ncbi_uid=5264238&amp;link_uid=PMC3754252">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4958.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3049838/figure/F3/" ref="ordinalpos=4958&amp;ncbi_uid=3653032&amp;link_uid=PMC3049838" href="/pmc/articles/PMC3049838/"><img src="/pmc/articles/instance/3049838/bin/nihms254703f3.gif" src-large="/pmc/articles/instance/3049838/bin/nihms254703f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3049838/" ref="ordinalpos=4958&amp;ncbi_uid=3653032&amp;link_uid=PMC3049838" image-link="/pmc/articles/PMC3049838/figure/F3/" class="imagepopup">Figure 3.  From: How does cAMP/Protein kinase A <span class="highlight" style="background-color:">signaling</span> lead to tumors in the adrenal cortex and other tissues?. </a></p><div class="supp"><p class="details"><div>Schematic representation of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in R1α deficient tumor cells. <i>Wnt</i> <span class="highlight" style="background-color:">signaling</span> enrichment was demonstrated in human and mouse tumors associated with cAMP/PKA activation. Up-regulation of WNT3, WNT3A, WNT7 and WISP2 induces the activation of the β-catenin-dependent <span class="highlight" style="background-color:">signaling</span> (canonical <span class="highlight" style="background-color:">pathway</span>). Binding of Wnt ligands to the frizzled, LRP5, and LRP6 receptors inhibits the degradation of β-catenin cytoplasmic complex and leads to nuclear accumulation of β-catenin. <i>CTNNB1</i> mutations in adrenal tumors also promote β-catenin stabilization. miR-449 is one of the highest down-regulated microRNAs in PPNAD and regulates <i>WISP2</i> expression. WNT, Wingless-type MMTV integration site family; DKK, Dickkopf; WISP2, WNT1-inducible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein 2; LRP5, low-density lipoprotein receptor-related protein 5; DVL, Dishevelled; APC, activated protein C; GSK3β, glycogen synthase kinase 3 beta; LEF/TCF, lymphoid enhancer factor/T-cell factor; PKA, protein kinase A; PPNAD, primary pigmented nodular adrenocortical disease.</div></p></div><div class="aux"><div class="resc">Madson Q. Almeida, et al. Mol Cell Endocrinol. ;336(1-2):162-168.</div><p class="links"><a class="dblinks" href="/pubmed/21111774" ref="ordinalpos=4958&amp;ncbi_uid=3653032&amp;link_uid=21111774">Citation</a><a class="dblinks" href="/pmc/articles/PMC3049838/" ref="ordinalpos=4958&amp;ncbi_uid=3653032&amp;link_uid=PMC3049838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4959.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2651576/figure/pone-0004771-g006/" ref="ordinalpos=4959&amp;ncbi_uid=1756056&amp;link_uid=PMC2651576" href="/pmc/articles/PMC2651576/"><img src="/pmc/articles/instance/2651576/bin/pone.0004771.g006.gif" src-large="/pmc/articles/instance/2651576/bin/pone.0004771.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2651576/" ref="ordinalpos=4959&amp;ncbi_uid=1756056&amp;link_uid=PMC2651576" image-link="/pmc/articles/PMC2651576/figure/pone-0004771-g006/" class="imagepopup">Figure 6. A model of the metabolic stress response regulation by oncogenic BRAF in melanoma cells..  From: Uncoupling of the LKB1-AMPK? Energy Sensor <span class="highlight" style="background-color:">Pathway</span> by Growth Factors and Oncogenic BRAFV600E. </a></p><div class="supp"><p class="details"><div>Resistance to stress conditions is essential for melanoma cells survival. We propose that oncogenic BRAF<sup>V600E</sup> <span class="highlight" style="background-color:">signaling</span> (left panel) protects to apoptosis by regulating BH3-family members and confers resistance to low energy conditions promoting the uncoupling of LKB1 and AMPK through Erk1/2 and p90<sup>Rsk</sup>. Under this condition BRAF mutant cells have a limited response to low energy conditions. On the right panel the inhibition of BRAF <span class="highlight" style="background-color:">signaling</span> allows the formation of the LKB1-AMPK complexes restoring the energy stress <span class="highlight" style="background-color:">pathway</span> and promoting the down-regulation of anti-apoptotic proteins such as Mcl1. The activation of AMPK by metabolic stress conditions and the inhibition of BRAF <span class="highlight" style="background-color:">signaling</span> would have synergistic effects promoting apoptosis.</div></p></div><div class="aux"><div class="resc">Rosaura Esteve-Puig, et al. PLoS ONE. 2009;4(3):e4771.</div><p class="links"><a class="dblinks" href="/pubmed/19274086" ref="ordinalpos=4959&amp;ncbi_uid=1756056&amp;link_uid=19274086">Citation</a><a class="dblinks" href="/pmc/articles/PMC2651576/" ref="ordinalpos=4959&amp;ncbi_uid=1756056&amp;link_uid=PMC2651576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4960.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3650620/figure/F4/" ref="ordinalpos=4960&amp;ncbi_uid=4899342&amp;link_uid=PMC3650620" href="/pmc/articles/PMC3650620/"><img src="/pmc/articles/instance/3650620/bin/fendo-04-00052-g004.gif" src-large="/pmc/articles/instance/3650620/bin/fendo-04-00052-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3650620/" ref="ordinalpos=4960&amp;ncbi_uid=4899342&amp;link_uid=PMC3650620" image-link="/pmc/articles/PMC3650620/figure/F4/" class="imagepopup">Figure 4.  From: Mechanisms of Obesity-Induced Inflammation and Insulin Resistance: Insights into the Emerging Role of Nutritional Strategies. </a></p><div class="supp"><p class="details"><div><b>Cross-talk between insulin and inflammatory <span class="highlight" style="background-color:">signaling</span> pathways</b>. Inflammatory <span class="highlight" style="background-color:">signaling</span> pathways activated by SFA or by pro-inflammatory cytokines IL-1β, IL-6, and TNF-α initiate a cascade of events that promote the release of further inflammatory mediators. These <span class="highlight" style="background-color:">signaling</span> events converge at the NF-κB and MAPK pathways, resulting in the translocation of transcription factors to the nucleus, transcriptional activation, and cytokine production. The inflammasome is activated through a two-hit process; with obesity the first hit occurs when TLR4 is activated by SFAs and this results in pro-IL-1β production, ATP or ceramides then provide the second hit. The NLRP3 inflammasome acts on pro-caspase-1 causing the release of caspase-1; caspase-1 then acts upon pro-IL-1β cleaving this precursor to the active IL-1β form. Insulin <span class="highlight" style="background-color:">signaling</span> promotes glucose uptake by promoting the translocation of GLUT4 to the cell surface plasma membrane. Inflammatory <span class="highlight" style="background-color:">signaling</span> pathways can alter the phosphorylation status of IRS-1. IRS-1 is crucial in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, tyrosine phosphorylation is associated with an insulin sensitive state. IKKβ and JNK can promote serine phosphorylation of IRS-1 and this phosphorylation state is linked to insulin resistance and reduced glucose uptake.</div></p></div><div class="aux"><div class="resc">Maeve A. McArdle, et al. Front Endocrinol (Lausanne). 2013;4:52.</div><p class="links"><a class="dblinks" href="/pubmed/23675368" ref="ordinalpos=4960&amp;ncbi_uid=4899342&amp;link_uid=23675368">Citation</a><a class="dblinks" href="/pmc/articles/PMC3650620/" ref="ordinalpos=4960&amp;ncbi_uid=4899342&amp;link_uid=PMC3650620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4961.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2884164/figure/F8/" ref="ordinalpos=4961&amp;ncbi_uid=2298055&amp;link_uid=PMC2884164" href="/pmc/articles/PMC2884164/"><img src="/pmc/articles/instance/2884164/bin/1755-8794-3-13-8.gif" src-large="/pmc/articles/instance/2884164/bin/1755-8794-3-13-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2884164/" ref="ordinalpos=4961&amp;ncbi_uid=2298055&amp;link_uid=PMC2884164" image-link="/pmc/articles/PMC2884164/figure/F8/" class="imagepopup">Figure 8.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> networks mined from human pituitary adenoma proteomics data. </a></p><div class="supp"><p class="details"><div><b>Mitochondrial dysfunctional <span class="highlight" style="background-color:">pathway</span> that is involved with pituitary adenoma comparative-proteomic data</b>. Red label = upregulated, green label = down regulated. The various shapes of nodes denote the different functions. A duplicated shape means this node contains multiple components. An arrow denotes the <span class="highlight" style="background-color:">pathway</span> direction. A line with a small circle denotes a biological result.</div></p></div><div class="aux"><div class="resc">Xianquan Zhan, et al. BMC Med Genomics. 2010;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/20426862" ref="ordinalpos=4961&amp;ncbi_uid=2298055&amp;link_uid=20426862">Citation</a><a class="dblinks" href="/pmc/articles/PMC2884164/" ref="ordinalpos=4961&amp;ncbi_uid=2298055&amp;link_uid=PMC2884164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4962.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3660501/figure/pone-0062235-g004/" ref="ordinalpos=4962&amp;ncbi_uid=4919670&amp;link_uid=PMC3660501" href="/pmc/articles/PMC3660501/"><img src="/pmc/articles/instance/3660501/bin/pone.0062235.g004.gif" src-large="/pmc/articles/instance/3660501/bin/pone.0062235.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3660501/" ref="ordinalpos=4962&amp;ncbi_uid=4919670&amp;link_uid=PMC3660501" image-link="/pmc/articles/PMC3660501/figure/pone-0062235-g004/" class="imagepopup">Figure 4. Role of neurotrophins was assessed using gene silencing technique for TrkA and TrkB receptors..  From: Enriched Environment Prevents Hypobaric Hypoxia Induced Memory Impairment and Neurodegeneration: Role of BDNF/PI3K/GSK3? <span class="highlight" style="background-color:">Pathway</span> Coupled with CREB Activation. </a></p><div class="supp"><p class="details"><div>Results showed that knockdown of TrkB significantly increased caspase3 activity even in presence of enriched environment whereas blocking of TrkA has diminutive effect on caspase3 activity in comparison to TrkB inhibition. To further explore the <span class="highlight" style="background-color:">signaling</span> cascade, ERK and PI3K <span class="highlight" style="background-color:">pathway</span> was blocked with U0216 and Wortmannin respectively and it has been observed that inhibition of PI3K <span class="highlight" style="background-color:">pathway</span> significantly increases caspase3 activity (B) whereas inhibiting ERK <span class="highlight" style="background-color:">pathway</span> have no significant effect in group exposed to hypobaric hypoxia in enriched cage (C). Data represents Mean ± SEM. In “*” and “#” represents p&lt;0.001 when compared to control and hypoxic group while “?”represents p&lt;0.05 when compared to control group.</div></p></div><div class="aux"><div class="resc">Vishal Jain, et al. PLoS One. 2013;8(5):e62235.</div><p class="links"><a class="dblinks" href="/pubmed/23704876" ref="ordinalpos=4962&amp;ncbi_uid=4919670&amp;link_uid=23704876">Citation</a><a class="dblinks" href="/pmc/articles/PMC3660501/" ref="ordinalpos=4962&amp;ncbi_uid=4919670&amp;link_uid=PMC3660501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4963.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3871244/figure/F5/" ref="ordinalpos=4963&amp;ncbi_uid=5683244&amp;link_uid=PMC3871244" href="/pmc/articles/PMC3871244/"><img src="/pmc/articles/instance/3871244/bin/JCI71180.f5.gif" src-large="/pmc/articles/instance/3871244/bin/JCI71180.f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3871244/" ref="ordinalpos=4963&amp;ncbi_uid=5683244&amp;link_uid=PMC3871244" image-link="/pmc/articles/PMC3871244/figure/F5/" class="imagepopup">Figure 5. WNT <span class="highlight" style="background-color:">pathway</span> deregulation in subgroup III tumors..  From: MYC-driven accumulation of 2-hydroxyglutarate is associated with breast cancer prognosis. </a></p><div class="supp"><p class="details"><div>Box plots show relative gene expression levels, and bee swarm plots show DNA methylation M-values, of genes encoding various inhibitors of the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, including Wnt inhibitory factor 1 (<i>WIF1</i>), secreted frizzled-related protein 1 (<i>SFRP1</i>), <i>SFRP2</i>, <i>SFRP4</i>, Dickkopf-related protein 2 (<i>DKK2</i>), and <i>DKK3</i>, in subgroup I (<i>n</i> = 19) and subgroup III (<i>n</i> = 23). Expression of WNT <span class="highlight" style="background-color:">pathway</span> inhibitors was significantly reduced in subgroup III tumors (Welch test). Data were collected using GeneChip Human Gene 1.0 ST arrays and Human Methylation 450 BeadChips. </div></p></div><div class="aux"><div class="resc">Atsushi Terunuma, et al. J Clin Invest. 2014 January 2;124(1):398-412.</div><p class="links"><a class="dblinks" href="/pubmed/24316975" ref="ordinalpos=4963&amp;ncbi_uid=5683244&amp;link_uid=24316975">Citation</a><a class="dblinks" href="/pmc/articles/PMC3871244/" ref="ordinalpos=4963&amp;ncbi_uid=5683244&amp;link_uid=PMC3871244">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4964.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4098730/figure/fig09/" ref="ordinalpos=4964&amp;ncbi_uid=6469624&amp;link_uid=PMC4098730" href="/pmc/articles/PMC4098730/"><img src="/pmc/articles/instance/4098730/bin/phy2-2-e00284-g9.gif" src-large="/pmc/articles/instance/4098730/bin/phy2-2-e00284-g9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4098730/" ref="ordinalpos=4964&amp;ncbi_uid=6469624&amp;link_uid=PMC4098730" image-link="/pmc/articles/PMC4098730/figure/fig09/" class="imagepopup">Figure 9. From: Chronic unpredictable stress regulates visceral adipocyte-mediated glucose metabolism and inflammatory circuits in male rats. </a></p><div class="supp"><p class="details"><div>Stress‐induced intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in rat epididymal adipocytes. Adipocytes were isolated from epididymal fat depots after collagenase digestion and total protein was collected and subjected to multiplex immunoassay using a panel of 13 intracellular <span class="highlight" style="background-color:">signaling</span> phosphor‐kinases. (A) Activation of JNK, PKCs as well as (B) the NF‐<i>κ</i>B <span class="highlight" style="background-color:">pathway</span> and (C) MAPK pathways are activated by stress (the networks for JNK and ERK are also identified as among the top activated in our network analyses, sides of A and B) have been shown to adversely affect insulin <span class="highlight" style="background-color:">signaling</span> in different tissues. (Mann–Whitney, **<i>P </i>&lt;<i></i>0.01, ***<i>P </i>&lt;<i></i>0.001, <i>n </i>=<i></i>10).</div></p></div><div class="aux"><div class="resc">Iordanes Karagiannides, et al. Physiol Rep. 2014 May 1;2(5):e00284.</div><p class="links"><a class="dblinks" href="/pubmed/24819750" ref="ordinalpos=4964&amp;ncbi_uid=6469624&amp;link_uid=24819750">Citation</a><a class="dblinks" href="/pmc/articles/PMC4098730/" ref="ordinalpos=4964&amp;ncbi_uid=6469624&amp;link_uid=PMC4098730">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4965.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3983002/figure/F6/" ref="ordinalpos=4965&amp;ncbi_uid=6337089&amp;link_uid=PMC3983002" href="/pmc/articles/PMC3983002/"><img src="/pmc/articles/instance/3983002/bin/1384fig6.gif" src-large="/pmc/articles/instance/3983002/bin/1384fig6.jpg" alt="FIGURE 6:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3983002/" ref="ordinalpos=4965&amp;ncbi_uid=6337089&amp;link_uid=PMC3983002" image-link="/pmc/articles/PMC3983002/figure/F6/" class="imagepopup">FIGURE 6:.  From: Differential regulation of myosin heavy chains defines new muscle domains in zebrafish. </a></p><div class="supp"><p class="details"><div>Retinoic acid and wnt <span class="highlight" style="background-color:">signaling</span> determines <i>fmyhc</i> expression domains. (A) <i>fmyhc1.3</i> in situ hybridization in <i>fmyhc1.2</i>:GFP embryos treated with DMSO, 0.2 μM RA, 200 μM DEAB, or 100 μM IWR-1 from shield stage to 30 hpf. (B) qPCR analysis of genes in the wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on 30-hpf embryos treated with RA from shield stage. (C) qPCR analysis of genes in the wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on 30-hpf embryos treated with RA from 28 hpf. (D) Top, <i>fmyhc1.2</i>:GFP embryos treated with DMSO, 100 μM IWR-1 in combination with 0.2 μM RA, or 200 μM DEAB from 18 ss to 36 hpf. Bottom, in situ hybridization showing mRNA expression of <i>fmyhc1.3.</i> Scale bar, 50 μm.</div></p></div><div class="aux"><div class="resc">Hanna Nord, et al. Mol Biol Cell. 2014 April 15;25(8):1384-1395.</div><p class="links"><a class="dblinks" href="/pubmed/24523292" ref="ordinalpos=4965&amp;ncbi_uid=6337089&amp;link_uid=24523292">Citation</a><a class="dblinks" href="/pmc/articles/PMC3983002/" ref="ordinalpos=4965&amp;ncbi_uid=6337089&amp;link_uid=PMC3983002">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4966.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4294653/figure/f8-dddt-9-349/" ref="ordinalpos=4966&amp;ncbi_uid=7087776&amp;link_uid=PMC4294653" href="/pmc/articles/PMC4294653/"><img src="/pmc/articles/instance/4294653/bin/dddt-9-349Fig8.gif" src-large="/pmc/articles/instance/4294653/bin/dddt-9-349Fig8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4294653/" ref="ordinalpos=4966&amp;ncbi_uid=7087776&amp;link_uid=PMC4294653" image-link="/pmc/articles/PMC4294653/figure/f8-dddt-9-349/" class="imagepopup">Figure 8.  From: Plumbagin elicits differential proteomic responses mainly involving cell cycle, apoptosis, autophagy, and epithelial-to-mesenchymal transition pathways in human prostate cancer PC-3 and DU145 cells. </a></p><div class="supp"><p class="details"><div>Proteomic analysis revealed a network of <span class="highlight" style="background-color:">signaling</span> pathways regulated by PLB in PC-3 cells.<br /><b>Notes:</b> A network of <span class="highlight" style="background-color:">signaling</span> pathways was analyzed by IPA according to the 1,225 molecules and 341 related pathways which were regulated by PLB in PC-3 cells.<br /><b>Abbreviations:</b> IPA, Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis; PLB, plumbagin; TCA, tricarboxylic acid cycle.</div></p></div><div class="aux"><div class="resc">Jia-Xuan Qiu, et al. Drug Des Devel Ther. 2015;9:349-417.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4966&amp;ncbi_uid=7087776&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4294653/" ref="ordinalpos=4966&amp;ncbi_uid=7087776&amp;link_uid=PMC4294653">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4967.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2828612/figure/F1/" ref="ordinalpos=4967&amp;ncbi_uid=2293365&amp;link_uid=PMC2828612" href="/pmc/articles/PMC2828612/"><img src="/pmc/articles/instance/2828612/bin/nihms-178826-f0001.gif" src-large="/pmc/articles/instance/2828612/bin/nihms-178826-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2828612/" ref="ordinalpos=4967&amp;ncbi_uid=2293365&amp;link_uid=PMC2828612" image-link="/pmc/articles/PMC2828612/figure/F1/" class="imagepopup">Figure 1.  From: A Critical Evaluation of the Ubiquitin-Proteasome System in Parkinson's Disease. </a></p><div class="supp"><p class="details"><div>A putative mechanism by which the UPS acts downstream of BDNF/TrkB <span class="highlight" style="background-color:">signaling</span> to ultimately regulate α-synuclein aggregation. According to this hypothetical model, synuclein-mediated inhibition of the UPS interferes with stimulatory effects of BDNF on synaptic activity, as well as the inhibitory influence of BDNF <span class="highlight" style="background-color:">signaling</span> on parkin cleavage and inactivation by the Fas/FADD death receptor <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Casey Cook, et al. Biochim Biophys Acta. ;1792(7):664-675.</div><p class="links"><a class="dblinks" href="/pubmed/19419700" ref="ordinalpos=4967&amp;ncbi_uid=2293365&amp;link_uid=19419700">Citation</a><a class="dblinks" href="/pmc/articles/PMC2828612/" ref="ordinalpos=4967&amp;ncbi_uid=2293365&amp;link_uid=PMC2828612">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4968.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2801368/figure/F1/" ref="ordinalpos=4968&amp;ncbi_uid=2154986&amp;link_uid=PMC2801368" href="/pmc/articles/PMC2801368/"><img src="/pmc/articles/instance/2801368/bin/psb0410_0983_fig001.gif" src-large="/pmc/articles/instance/2801368/bin/psb0410_0983_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2801368/" ref="ordinalpos=4968&amp;ncbi_uid=2154986&amp;link_uid=PMC2801368" image-link="/pmc/articles/PMC2801368/figure/F1/" class="imagepopup">Figure 1.  From: Capping invertase activity by its inhibitor. </a></p><div class="supp"><p class="details"><div>A model on sugar sensing and <span class="highlight" style="background-color:">signaling</span> from extra- to intra-cellular compartments mediated by cell wall invertase, its inhibitor and G-proteins. Glucose generated by interaction between invertase and its inhibitor in apoplasm may be sensed by AtRGS1. The GTP-bound active form of Gα can transmit glucose signal into the cytoplasm via direct interaction with AtRGS1 and THF1, and induce sugar responses. The active form of Gα can also regulate chloroplast development and photosynthesis in a THF1-independent <span class="highlight" style="background-color:">pathway</span> via controlling nuclear gene expression. A Golgi-localized hexose transporter (SGB1) has been genetically identified to mediate Gβγ <span class="highlight" style="background-color:">signaling</span>. CW; cell wall. PM; plasma membrane. N; nucleus.</div></p></div><div class="aux"><div class="resc">Yong-Ling Ruan, et al. Plant Signal Behav. 2009 October;4(10):983-985.</div><p class="links"><a class="dblinks" href="/pubmed/19826227" ref="ordinalpos=4968&amp;ncbi_uid=2154986&amp;link_uid=19826227">Citation</a><a class="dblinks" href="/pmc/articles/PMC2801368/" ref="ordinalpos=4968&amp;ncbi_uid=2154986&amp;link_uid=PMC2801368">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4969.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3500853/figure/F3/" ref="ordinalpos=4969&amp;ncbi_uid=4378180&amp;link_uid=PMC3500853" href="/pmc/articles/PMC3500853/"><img src="/pmc/articles/instance/3500853/bin/nihms416992f3.gif" src-large="/pmc/articles/instance/3500853/bin/nihms416992f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3500853/" ref="ordinalpos=4969&amp;ncbi_uid=4378180&amp;link_uid=PMC3500853" image-link="/pmc/articles/PMC3500853/figure/F3/" class="imagepopup">Fig. 3.  From: Essential Roles of EphB Receptors and EphrinB Ligands in Endothelial Cell Function and Angiogenesis. </a></p><div class="supp"><p class="details"><div>Mechanisms of EphrinB reverse <span class="highlight" style="background-color:">signaling</span>. (A) EphrinB activation by EphB receptor leads to the recruitment of Src family kinases (SFKs) that phosphorylate EphrinB intracellular domain at tyrosine residues. The adaptor molecule Grb4, which contains a Src-homology-2 (SH2) domain, is recruited to the phosphorylated EphrinB and initiates a number of <span class="highlight" style="background-color:">signaling</span> events that regulate cytoskeleton dynamics and focal adhesions. The phosphorylated EphrinB can also recruit the Jak2/STAT3 complex; the phosphorylated STAT3 translocates to the nucleus regulating expression of target genes. (B) Activated EphrinB regulates the internalization of VEGFR2 and VEGFR3 through PDZ-mediated binding of yet unknown protein(s). After VEGF binding, VEGFR is phosphorylated; <span class="highlight" style="background-color:">signaling</span> from activated VEGFR requires internalization, which is positively regulated by EphrinB <span class="highlight" style="background-color:">signaling</span>. (C) RGS3 is a PDZ-containing protein that constitutively binds to EphrinB and links <span class="highlight" style="background-color:">signaling</span> from the G-protein-coupled receptor CXCR4 to EphrinB. CXCR4 <span class="highlight" style="background-color:">signaling</span> in response to the ligand SDF1 is induced by the dissociation of Gβγ and GTP-Gα subunits. PDZ-RGS3 can inhibit CXCR4 <span class="highlight" style="background-color:">signaling</span> by enhancing the GTPase activity of the Gα subunit, resulting in the reformation of the inactive heterodimeric CXCR4 receptor. (D) The scaffold protein Dishevelled binds EphrinB through its DEP domain and mediates <span class="highlight" style="background-color:">signaling</span> via the Rho small GTPase <span class="highlight" style="background-color:">pathway</span>. (E) The scaffold protein Par6 associates with EphrinB resulting in the loss of tight junctions. PAR6 forms a complex with activated apical protein kinase C (aPKC) and Cdc42-GTP; the complex localizes to the apical cell junctions where it regulates tight junctions. EphrinB1 can compete with Cdc42 for binding to Par6 and thus reduce tight junctions. (F) Cross-regulation between EphrinB phosphorylation- and PDZ-dependent <span class="highlight" style="background-color:">signaling</span> pathways. The phosphatase PTB-BL, recruited through its PDZ domain to EphrinB cytoplasmic domain, inactivates Src and dephosphorylates EphrinB.</div></p></div><div class="aux"><div class="resc">Ombretta Salvucci, et al. Adv Cancer Res. 2012;114:21-57.</div><p class="links"><a class="dblinks" href="/pubmed/22588055" ref="ordinalpos=4969&amp;ncbi_uid=4378180&amp;link_uid=22588055">Citation</a><a class="dblinks" href="/pmc/articles/PMC3500853/" ref="ordinalpos=4969&amp;ncbi_uid=4378180&amp;link_uid=PMC3500853">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4970.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2873688/figure/F1/" ref="ordinalpos=4970&amp;ncbi_uid=2381395&amp;link_uid=PMC2873688" href="/pmc/articles/PMC2873688/"><img src="/pmc/articles/instance/2873688/bin/nihms201344f1.gif" src-large="/pmc/articles/instance/2873688/bin/nihms201344f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2873688/" ref="ordinalpos=4970&amp;ncbi_uid=2381395&amp;link_uid=PMC2873688" image-link="/pmc/articles/PMC2873688/figure/F1/" class="imagepopup">Fig. 1.  From: Insulin/IGF-1 and ROS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> cross-talk in aging and longevity determination. </a></p><div class="supp"><p class="details"><div>The interactions of the insulin receptor linked-redox <span class="highlight" style="background-color:">signaling</span> pathways. This scheme shows the major interactions of insulin <span class="highlight" style="background-color:">signaling</span> involving the activation of the receptor tyrosine kinase cascade. This leads to the initial redox-mediated priming of the receptor tyrosine phosphorylation and its IRS proteins. The scheme also links the ligand activation bypass of insulin <span class="highlight" style="background-color:">signaling</span> mediated by reactive oxygen species. The steady-state level of tyrosine phosphorylation of these proteins is regulated by PTPases, a family of enzymes that constitutes one of the key targets of regulation by reactive oxygen species.</div></p></div><div class="aux"><div class="resc">John Papaconstantinou. Mol Cell Endocrinol. ;299(1):89-100.</div><p class="links"><a class="dblinks" href="/pubmed/19103250" ref="ordinalpos=4970&amp;ncbi_uid=2381395&amp;link_uid=19103250">Citation</a><a class="dblinks" href="/pmc/articles/PMC2873688/" ref="ordinalpos=4970&amp;ncbi_uid=2381395&amp;link_uid=PMC2873688">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4971.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2866141/figure/F4/" ref="ordinalpos=4971&amp;ncbi_uid=2916997&amp;link_uid=PMC2866141" href="/pmc/articles/PMC2866141/"><img src="/pmc/articles/instance/2866141/bin/nihms192885f4.gif" src-large="/pmc/articles/instance/2866141/bin/nihms192885f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2866141/" ref="ordinalpos=4971&amp;ncbi_uid=2916997&amp;link_uid=PMC2866141" image-link="/pmc/articles/PMC2866141/figure/F4/" class="imagepopup">Figure 4. Schematic of proposed mechanism by which GM-CSF regulates LPS responses in macrophages.  From: The Molecular Basis of Pulmonary Alveolar Proteinosis. </a></p><div class="supp"><p class="details"><div>GM-CSF initially binds to the low-affinity GM-CSF receptor α chain (α), followed by association with the affinity-enhancing β chain (β) and constitutively associated Janus kinase 2 (JAK 2) resulting in assembly of a dodecameric structure <span class="highlight" style="background-color:">signaling</span> complex.<br />At low concentrations of GM-CSF (&lt;300 pg/ml), as are normally present in normal healthy tissues, <span class="highlight" style="background-color:">signaling</span> occurs through Ser585 of the β chain and results in myeloid cell survival and GM-CSF stimulated expression of some GM-CSF-dependent TLR-4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components, CD14, RP105, and IRAK-M (dotted lines); other components are not affected by GM-CSF. Both GM-CSF-dependent and GM-CSF-independent components are required for normal responses to LPS. At high concentrations of GM-CSF (&gt;300 pg/ml), <span class="highlight" style="background-color:">signaling</span> via Ser585 is extinguished and <span class="highlight" style="background-color:">signaling</span> occurs exclusively via residue Tyr577, activating STAT5 <span class="highlight" style="background-color:">signaling</span>, which enhances NFκB activity and the LPS response. Adapted from reference [15].</div></p></div><div class="aux"><div class="resc">Brenna Carey, et al. Clin Immunol. ;135(2):223-235.</div><p class="links"><a class="dblinks" href="/pubmed/20338813" ref="ordinalpos=4971&amp;ncbi_uid=2916997&amp;link_uid=20338813">Citation</a><a class="dblinks" href="/pmc/articles/PMC2866141/" ref="ordinalpos=4971&amp;ncbi_uid=2916997&amp;link_uid=PMC2866141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4972.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4051675/figure/pone-0099343-g001/" ref="ordinalpos=4972&amp;ncbi_uid=6264909&amp;link_uid=PMC4051675" href="/pmc/articles/PMC4051675/"><img src="/pmc/articles/instance/4051675/bin/pone.0099343.g001.gif" src-large="/pmc/articles/instance/4051675/bin/pone.0099343.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4051675/" ref="ordinalpos=4972&amp;ncbi_uid=6264909&amp;link_uid=PMC4051675" image-link="/pmc/articles/PMC4051675/figure/pone-0099343-g001/" class="imagepopup">Figure 1. JA- and SA-dependent <span class="highlight" style="background-color:">signaling</span> pathways in the lif2-1 mutant..  From: The hnRNP-Q Protein LIF2 Participates in the Plant Immune Response. </a></p><div class="supp"><p class="details"><div>(A) Expression levels of genes involved in the JA-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>LOX3, AOS, AOC, OPR3, COI1</i>, and JAR1) and of a JA-responsive marker gene (<i>PDF1.2)</i> in the rosette leaves of seven-week-old wild-type (WT) and <i>lif2-1</i> plants. (B) Expression levels of genes of the SA-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (<i>EDS1</i>, <i>PAD4</i>, and <i>ICS1</i>) and of SA-responsive marker genes (<i>NPR1</i> and <i>PR1</i>) in seven-week-old wild-type (WT) and <i>lif2-1</i> rosette leaves. CC-NB-LRR and TIR-NB-LRR are disease resistance proteins with coiled-coil (CC), nucleotide-binding (NB), leucine-rich repeat (LRR), or Toll-Interleukin Receptor (TIR) domains. (C) Quantification of phytohormone contents in the rosette leaves of three-week-old plants using HPLC-electrospray-MS/MS. The amount of JA was expressed as a ratio of peak areas (209&gt;62/214&gt;62) per fresh weight (FW). The amount of other hormones was expressed in ng/FW. The bars represent standard deviation.</div></p></div><div class="aux"><div class="resc">Clémentine Le Roux, et al. PLoS One. 2014;9(6):e99343.</div><p class="links"><a class="dblinks" href="/pubmed/24914891" ref="ordinalpos=4972&amp;ncbi_uid=6264909&amp;link_uid=24914891">Citation</a><a class="dblinks" href="/pmc/articles/PMC4051675/" ref="ordinalpos=4972&amp;ncbi_uid=6264909&amp;link_uid=PMC4051675">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4973.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3426662/figure/F4/" ref="ordinalpos=4973&amp;ncbi_uid=5146683&amp;link_uid=PMC3426662" href="/pmc/articles/PMC3426662/"><img src="/pmc/articles/instance/3426662/bin/nihms371575f4.gif" src-large="/pmc/articles/instance/3426662/bin/nihms371575f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3426662/" ref="ordinalpos=4973&amp;ncbi_uid=5146683&amp;link_uid=PMC3426662" image-link="/pmc/articles/PMC3426662/figure/F4/" class="imagepopup">Figure 4. Two forms of Notch activity in CNS longitudinal axon growth in Drosophila.  From: Notch <span class="highlight" style="background-color:">signaling</span> and neural connectivity. </a></p><div class="supp"><p class="details"><div>A) Notch plays two parallel, but separate, roles to promote growth of longitudinal axons in the <i>Drosophila</i> CNS [25]. Canonical, nuclear Notch <span class="highlight" style="background-color:">signaling</span> in glial cells (green) promotes morphogenesis of the substratum for approaching pioneer axons (dMP2, vMP2; black). “Non-canonical”, cytoplasmic Notch <span class="highlight" style="background-color:">signaling</span> in the growth cones of the pioneer axons drives their extension alongside the glia. Ligand for both Notch functions is provided by the axons of commissural interneurons that bear Delta protein (pink), and that cross the longitudinal <span class="highlight" style="background-color:">pathway</span>. It is likely that there is also some level of canonical Notch <span class="highlight" style="background-color:">signaling</span> occurring in the pioneer neurons (light gray nuclei). CNS midline is indicated to the right; Delta<sup>+</sup> commissural interneurons are located laterally in the nerve cord (left).</div></p></div><div class="aux"><div class="resc">Edward Giniger. Curr Opin Genet Dev. ;22(4):339-346.</div><p class="links"><a class="dblinks" href="/pubmed/22608692" ref="ordinalpos=4973&amp;ncbi_uid=5146683&amp;link_uid=22608692">Citation</a><a class="dblinks" href="/pmc/articles/PMC3426662/" ref="ordinalpos=4973&amp;ncbi_uid=5146683&amp;link_uid=PMC3426662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4974.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2559936/figure/f8/" ref="ordinalpos=4974&amp;ncbi_uid=1540738&amp;link_uid=PMC2559936" href="/pmc/articles/PMC2559936/"><img src="/pmc/articles/instance/2559936/bin/zjw0100832490008.gif" src-large="/pmc/articles/instance/2559936/bin/zjw0100832490008.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2559936/" ref="ordinalpos=4974&amp;ncbi_uid=1540738&amp;link_uid=PMC2559936" image-link="/pmc/articles/PMC2559936/figure/f8/" class="imagepopup">F ig . 8. From: A Portrait of Tissue Phosphoprotein Stability in the Clinical Tissue Procurement Process. </a></p><div class="supp"><p class="details"><div><b>Protein fluctuations over time by cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Leiomyoma tissue replicates incubated at room temperature and analyzed by RPPA reveal <span class="highlight" style="background-color:">pathway</span>- and protein-specific increases and decreases over time (<i>n</i> = 3 for 44 and 60 min; <i>n</i> = 4 for 119, 146, and 191 min; <i>n</i> = 5 for 949 min; ±S.D. as a percentage of time 0 sample). Cell <span class="highlight" style="background-color:">signaling</span> pathways represented are: apoptosis (<i>A</i>), adhesion (<i>B</i>), hypoxia (<i>C</i>), glucose/insulin <span class="highlight" style="background-color:">signaling</span> (<i>D</i>), proliferation/survival (<i>E</i>), cell cycle (<i>F</i>), stress/inflammation (<i>G</i>), and transcription factors (<i>H</i>). <i>FAK</i>, focal adhesion kinase; <i>AcCoA</i>, acetyl-CoA carboxylase.</div></p></div><div class="aux"><div class="resc">Virginia Espina, et al. Mol Cell Proteomics. 2008 October;7(10):1998-2018.</div><p class="links"><a class="dblinks" href="/pubmed/18667411" ref="ordinalpos=4974&amp;ncbi_uid=1540738&amp;link_uid=18667411">Citation</a><a class="dblinks" href="/pmc/articles/PMC2559936/" ref="ordinalpos=4974&amp;ncbi_uid=1540738&amp;link_uid=PMC2559936">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4975.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3926869/figure/F1/" ref="ordinalpos=4975&amp;ncbi_uid=5847360&amp;link_uid=PMC3926869" href="/pmc/articles/PMC3926869/"><img src="/pmc/articles/instance/3926869/bin/onci-2-e26961-g1.gif" src-large="/pmc/articles/instance/3926869/bin/onci-2-e26961-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3926869/" ref="ordinalpos=4975&amp;ncbi_uid=5847360&amp;link_uid=PMC3926869" image-link="/pmc/articles/PMC3926869/figure/F1/" class="imagepopup">Figure. From: Immunosuppressants in cancer prevention and therapy. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> The mTOR <span class="highlight" style="background-color:">pathway</span> is involved in both cancer and senescence. Growth factors, cytokines, insulin, nutrients, and oncoproteins activate the phosphoinositide-3-kinase/mammalian target of rapamycin (PI3K/mTOR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which promotes cellular growth and cell cycle progression. Mutations that results in constitutive activation of the mTOR <span class="highlight" style="background-color:">pathway</span> are involved in oncogenic transformation when intrinsic controls on cell cycle progression are disabled. Conversely, when mTOR is activated and the cell cycle is arrested, cells enter a senescent state. Rapamycin and other rapalogs (e.g., everolimus) inhibit the mTOR complex 1 (mTORC1) and suppress such a geroconversion.</div></p></div><div class="aux"><div class="resc">Mikhail V Blagosklonny. Oncoimmunology. 2013 December 1;2(12):e26961.</div><p class="links"><a class="dblinks" href="/pubmed/24575379" ref="ordinalpos=4975&amp;ncbi_uid=5847360&amp;link_uid=24575379">Citation</a><a class="dblinks" href="/pmc/articles/PMC3926869/" ref="ordinalpos=4975&amp;ncbi_uid=5847360&amp;link_uid=PMC3926869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4976.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4116751/figure/F2/" ref="ordinalpos=4976&amp;ncbi_uid=6430934&amp;link_uid=PMC4116751" href="/pmc/articles/PMC4116751/"><img src="/pmc/articles/instance/4116751/bin/nihms605701f2.gif" src-large="/pmc/articles/instance/4116751/bin/nihms605701f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4116751/" ref="ordinalpos=4976&amp;ncbi_uid=6430934&amp;link_uid=PMC4116751" image-link="/pmc/articles/PMC4116751/figure/F2/" class="imagepopup">Figure 2. Possible genetic interactions that can enhance the phenotype of a hypormorphic allele.  From: Dissection of Genetic Pathways in C. elegans. </a></p><div class="supp"><p class="details"><div>A hypomorphic mutation (X) causes partial loss of gene function, which only slightly reduces the activity of a corresponding <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This reduction can be dramatically enhanced by loss or reduction in the activity of a gene (Y) that encodes a physical interacting partner (i), acts upstream (ii), downstream (iii) of the same <span class="highlight" style="background-color:">pathway</span>, or functions in a parallel <span class="highlight" style="background-color:">pathway</span> that functionally compensate for each other (iv).</div></p></div><div class="aux"><div class="resc">Zheng Wang, et al. Methods Cell Biol. ;106:113-157.</div><p class="links"><a class="dblinks" href="/pubmed/22118276" ref="ordinalpos=4976&amp;ncbi_uid=6430934&amp;link_uid=22118276">Citation</a><a class="dblinks" href="/pmc/articles/PMC4116751/" ref="ordinalpos=4976&amp;ncbi_uid=6430934&amp;link_uid=PMC4116751">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4977.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1347434/figure/fig2/" ref="ordinalpos=4977&amp;ncbi_uid=567297&amp;link_uid=PMC1347434" href="/pmc/articles/PMC1347434/"><img src="/pmc/articles/instance/1347434/bin/gkj071f2.gif" src-large="/pmc/articles/instance/1347434/bin/gkj071f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1347434/" ref="ordinalpos=4977&amp;ncbi_uid=567297&amp;link_uid=PMC1347434" image-link="/pmc/articles/PMC1347434/figure/fig2/" class="imagepopup">Figure 2.  From: LEGER: knowledge database and visualization tool for comparative genomics of pathogenic and non-pathogenic Listeria species. </a></p><div class="supp"><p class="details"><div>The integrated KEGG <span class="highlight" style="background-color:">Pathway</span> Viewer facilitates the generation of functional hypotheses. A separate table summarizes affected pathways. Submitted genes or proteins are marked along metabolic or <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> maps. For example, Lmo1986 and Lmo2006, two enzymes out of seventeen that are annotated within the ‘Pantothenate and CoA biosynthesis’ <span class="highlight" style="background-color:">pathway</span> were found to be co-regulated and mediate the first steps of this synthesis (see partial view of the corresponding map).</div></p></div><div class="aux"><div class="resc">Guido Dieterich, et al. Nucleic Acids Res. 2006 January 1;34(Database issue):D402-D406.</div><p class="links"><a class="dblinks" href="/pubmed/16381897" ref="ordinalpos=4977&amp;ncbi_uid=567297&amp;link_uid=16381897">Citation</a><a class="dblinks" href="/pmc/articles/PMC1347434/" ref="ordinalpos=4977&amp;ncbi_uid=567297&amp;link_uid=PMC1347434">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4978.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2291030/figure/F3/" ref="ordinalpos=4978&amp;ncbi_uid=1296348&amp;link_uid=PMC2291030" href="/pmc/articles/PMC2291030/"><img src="/pmc/articles/instance/2291030/bin/1752-0509-2-19-3.gif" src-large="/pmc/articles/instance/2291030/bin/1752-0509-2-19-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2291030/" ref="ordinalpos=4978&amp;ncbi_uid=1296348&amp;link_uid=PMC2291030" image-link="/pmc/articles/PMC2291030/figure/F3/" class="imagepopup">Figure 3.  From: Comprehensive proteomic analysis of bovine spermatozoa of varying fertility rates and identification of biomarkers associated with fertility. </a></p><div class="supp"><p class="details"><div><b>EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> generated by the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) software.</b> EGF and PDGF <span class="highlight" style="background-color:">signaling</span> pathways were the top two pathways in the top 10 functions/diseases associated with the high fertility spermatozoa (Table 2). Each node represents a protein; proteins in shaded nodes were found in the high fertility spermatozoa dataset (see additional file 1) while proteins in clear nodes were not found in the high fertility spermatozoa dataset.</div></p></div><div class="aux"><div class="resc">Divyaswetha Peddinti, et al. BMC Syst Biol. 2008;2:19-19.</div><p class="links"><a class="dblinks" href="/pubmed/18294385" ref="ordinalpos=4978&amp;ncbi_uid=1296348&amp;link_uid=18294385">Citation</a><a class="dblinks" href="/pmc/articles/PMC2291030/" ref="ordinalpos=4978&amp;ncbi_uid=1296348&amp;link_uid=PMC2291030">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4979.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3913887/figure/F9/" ref="ordinalpos=4979&amp;ncbi_uid=5823796&amp;link_uid=PMC3913887" href="/pmc/articles/PMC3913887/"><img src="/pmc/articles/instance/3913887/bin/fphys-05-00019-g0009.gif" src-large="/pmc/articles/instance/3913887/bin/fphys-05-00019-g0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3913887/" ref="ordinalpos=4979&amp;ncbi_uid=5823796&amp;link_uid=PMC3913887" image-link="/pmc/articles/PMC3913887/figure/F9/" class="imagepopup">Figure 9.  From: Insulin <span class="highlight" style="background-color:">signaling</span> pathways in lepidopteran ecdysone secretion. </a></p><div class="supp"><p class="details"><div><b>PTTH and insulin <span class="highlight" style="background-color:">signaling</span> pathways in lepidopteran ecdysone secretion</b>. PTTH: In both <i>Bombyx</i> and <i>Manduca</i>, PTTH has been shown to stimulate tyrosine kinase activity. We suggest that this leads to the phosphorylation of a postulated receptor substrate (modeled as Torso Receptor Substrate, TRS), capable of interacting with PLC (phospholipase C). PTTH is known to increase intracellular calcium, activate calcium-sensitive adenylyl cyclase (not shown), and to elevate cyclic AMP. TRS is postulated to recruit the adapter protein GRB2 and SOS, enhancing the activity of RAS and RAF, leading to known activation of MEK, ERK, and RSK. Cyclic AMP is also suggested to act via an effect on the MAPkinase <span class="highlight" style="background-color:">pathway</span> (dotted line). Insulin/Bombyxin: In both Bombyx and Manduca, insulin or bombyxin activate PI3K and Akt, which is presumed to increase the activity of TOR (target of rapamycin) through proteins not shown. TOR enhances translation by suppressing the translation inhibitor 4E-binding protein (4EBP), and activating p70 S6 kinase, which targets ribosomal protein S6. The primary difference seen between the prothoracic glands of Manduca and Bombyx is the ability of both insulin and PTTH to activate the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Bombyx (PI3K, Akt, TOR, 4EBP) and by this <span class="highlight" style="background-color:">pathway</span> to stimulate ecdysone secretion (Gray curved arrow). By contrast, in Manduca, although PTTH stimulates the phosphorylation of 4EBP suggesting activation of PI3K (dotted line), the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is neither sufficient, nor required, for ecdysone synthesis. Instead, secretion is mediated by PTTH-stimulated activation of MAPkinases (Black curved arrow).</div></p></div><div class="aux"><div class="resc">Wendy A. Smith, et al. Front Physiol. 2014;5:19.</div><p class="links"><a class="dblinks" href="/pubmed/24550835" ref="ordinalpos=4979&amp;ncbi_uid=5823796&amp;link_uid=24550835">Citation</a><a class="dblinks" href="/pmc/articles/PMC3913887/" ref="ordinalpos=4979&amp;ncbi_uid=5823796&amp;link_uid=PMC3913887">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4980.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2826402/figure/pone-0009370-g002/" ref="ordinalpos=4980&amp;ncbi_uid=2278431&amp;link_uid=PMC2826402" href="/pmc/articles/PMC2826402/"><img src="/pmc/articles/instance/2826402/bin/pone.0009370.g002.gif" src-large="/pmc/articles/instance/2826402/bin/pone.0009370.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2826402/" ref="ordinalpos=4980&amp;ncbi_uid=2278431&amp;link_uid=PMC2826402" image-link="/pmc/articles/PMC2826402/figure/pone-0009370-g002/" class="imagepopup">Figure 2. Luminescent and drug-resistant reporters for the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Lentiviral Vectors to Probe and Manipulate the Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A) HEK293T cells were infected with the 7xTcf-FFluc (“7TF”) virus and responded to purified Wnt3A protein in a dose-dependent manner. B) HEK293A cells were infected with the 7xTcf-FFluc//SV40-mCherry (“7TFC”) virus and responded similarly to purified Wnt3A protein (left). Cells expressed mCherry in a constitutive manner (right). C) HEK293A cells were infected with the 7xTcf-FFluc//SV40-Puro<sup>R</sup> (“7TFP”) virus and also responded to purified Wnt3A protein in a dose-dependent manner (left). Selection of 7TFP-infected HEK293T cells with puromycin (1 µg/ml) allowed for enrichment of Wnt-responding cells, resulting in increased luciferase signal upon addition of conditioned medium (right, compare “Puro” with “−”). D) Adipose-derived mesenchymal cells (“ADMCs”) were infected with the 7xTcf-Puro<sup>R</sup> (7TP) lentivirus. Addition of purified Wnt3A protein (500 ng/ml final, “Wnt3A”) allowed selection of Wnt-responding cells with puromycin (1 µg/ml), while cells incubated in absence of Wnt3A (“vhc”) were all killed by the drug treatment. For all graphs, activity is relative to vehicle- or mock-treated cells.</div></p></div><div class="aux"><div class="resc">Christophe Fuerer, et al. PLoS One. 2010;5(2):e9370.</div><p class="links"><a class="dblinks" href="/pubmed/20186325" ref="ordinalpos=4980&amp;ncbi_uid=2278431&amp;link_uid=20186325">Citation</a><a class="dblinks" href="/pmc/articles/PMC2826402/" ref="ordinalpos=4980&amp;ncbi_uid=2278431&amp;link_uid=PMC2826402">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4981.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3266628/figure/F6/" ref="ordinalpos=4981&amp;ncbi_uid=4569042&amp;link_uid=PMC3266628" href="/pmc/articles/PMC3266628/"><img src="/pmc/articles/instance/3266628/bin/nihms-341944-f0006.gif" src-large="/pmc/articles/instance/3266628/bin/nihms-341944-f0006.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3266628/" ref="ordinalpos=4981&amp;ncbi_uid=4569042&amp;link_uid=PMC3266628" image-link="/pmc/articles/PMC3266628/figure/F6/" class="imagepopup">Fig. 6.  From: Notch <span class="highlight" style="background-color:">pathway</span> regulation of neural crest cell development in vivo. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> promotes cell proliferation of neural crest derivatives. (A-C) The DRG (arrows) are NCC-derived structures, as evident by X-Gal staining of <i>Wnt1Cre;R26R, Wnt1Cre;R26R;Rosa<sup>Notch</sup></i>, and <i>Wnt1Cre;R26R;RBP-J<sup>f/f</sup></i> E10.5 sections. (n=8 for each genotype) (D-G) Notch activation results in increased NCC proliferation, while Notch <span class="highlight" style="background-color:">pathway</span> inactivation results in significant reduction of NCC proliferation in the DRG, as determined by BrdU incorporation (arrows), compared to <i>Wnt1Cre</i>-negative control littermates. (n=6 for each genotype) * p ≤ 0.01</div></p></div><div class="aux"><div class="resc">Timothy J. Mead, et al. Dev Dyn. ;241(2):376-389.</div><p class="links"><a class="dblinks" href="/pubmed/22275227" ref="ordinalpos=4981&amp;ncbi_uid=4569042&amp;link_uid=22275227">Citation</a><a class="dblinks" href="/pmc/articles/PMC3266628/" ref="ordinalpos=4981&amp;ncbi_uid=4569042&amp;link_uid=PMC3266628">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4982.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4151153/figure/F2/" ref="ordinalpos=4982&amp;ncbi_uid=6615477&amp;link_uid=PMC4151153" href="/pmc/articles/PMC4151153/"><img src="/pmc/articles/instance/4151153/bin/fimmu-05-00421-g002.gif" src-large="/pmc/articles/instance/4151153/bin/fimmu-05-00421-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4151153/" ref="ordinalpos=4982&amp;ncbi_uid=6615477&amp;link_uid=PMC4151153" image-link="/pmc/articles/PMC4151153/figure/F2/" class="imagepopup">Figure 2.  From: Regulation of Toll-Like Receptors-Mediated Inflammation by Immunobiotics in Bovine Intestinal Epitheliocytes: Role of <span class="highlight" style="background-color:">Signaling</span> Pathways and Negative Regulators. </a></p><div class="supp"><p class="details"><div><b>Modulation of toll-like receptor 4 (TLR4) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by <i>Lactobacillus casei</i> OLL2768, <i>Bifidobacterium adolescentis</i> MCC-75, and <i>Bifidobacterium adolescentis</i> ATCC15705 in bovine intestinal epithelial epitheliocytes (BIE cells)</b>. Anti-inflammatory immunobiotic strains up-regulate the expression of TLR negative regulators, reduce the activation phosphatidylinositol 3-kinase (PI3K), mitogen-activated protein kinase (MAPK), and nuclear factor κB (NF-κB) pathways; and diminish the production of inflammatory mediators by BIE cells.</div></p></div><div class="aux"><div class="resc">Julio Villena, et al. Front Immunol. 2014;5:421.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4982&amp;ncbi_uid=6615477&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4151153/" ref="ordinalpos=4982&amp;ncbi_uid=6615477&amp;link_uid=PMC4151153">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4983.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3737295/figure/F1/" ref="ordinalpos=4983&amp;ncbi_uid=5216488&amp;link_uid=PMC3737295" href="/pmc/articles/PMC3737295/"><img src="/pmc/articles/instance/3737295/bin/cc-12-1989-g1.gif" src-large="/pmc/articles/instance/3737295/bin/cc-12-1989-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3737295/" ref="ordinalpos=4983&amp;ncbi_uid=5216488&amp;link_uid=PMC3737295" image-link="/pmc/articles/PMC3737295/figure/F1/" class="imagepopup">Figure. From: Moving beyond translation: Glucose-TOR <span class="highlight" style="background-color:">signaling</span> in the transcriptional control of cell cycle. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Model of glucose-TOR <span class="highlight" style="background-color:">signaling</span> in transcriptional regulation of cell cycle. Leaf photosynthesis-derived glucose is indispensable for activating TOR kinase via glycolysis and mitochondrial bioenergetic relay to control a host of effectors and transcription factors for the concerted activation of the proliferation of progenitor and stem cells in meristems. ETC, electron transport chain; <i>RGF</i>, root growth factors; <i>UPB1</i>, UPBEAT1 transcription factor; GSH, the glutathione synthesis <span class="highlight" style="background-color:">pathway</span>; TFs, transcription factors.</div></p></div><div class="aux"><div class="resc">Yan Xiong, et al. Cell Cycle. 2013 July 1;12(13):1989-1990.</div><p class="links"><a class="dblinks" href="/pubmed/23759578" ref="ordinalpos=4983&amp;ncbi_uid=5216488&amp;link_uid=23759578">Citation</a><a class="dblinks" href="/pmc/articles/PMC3737295/" ref="ordinalpos=4983&amp;ncbi_uid=5216488&amp;link_uid=PMC3737295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4984.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3292657/figure/F4/" ref="ordinalpos=4984&amp;ncbi_uid=4753575&amp;link_uid=PMC3292657" href="/pmc/articles/PMC3292657/"><img src="/pmc/articles/instance/3292657/bin/nihms338168f4.gif" src-large="/pmc/articles/instance/3292657/bin/nihms338168f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3292657/" ref="ordinalpos=4984&amp;ncbi_uid=4753575&amp;link_uid=PMC3292657" image-link="/pmc/articles/PMC3292657/figure/F4/" class="imagepopup">Figure 4. ERK1/2 <span class="highlight" style="background-color:">signaling</span> affects osteogenesis via regulating WNT/β-catenin <span class="highlight" style="background-color:">pathway</span>.  From: Basic Fibroblast Growth Factor Inhibits Osteogenic Differentiation of SHED through ERK <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Western blot analysis showed that bFGF reduced expression level of total β-catenin and active- β-catenin. β-actin was used as an internal control. Three independent assays showed similar results (*<i>P</i>&lt;0.05). <b>(B)</b> bFGF treatment decreased β-catenin expression level and ERK1/2 inhibitor treatment increased β-catenin levels. β-actin was used as an internal control. Three independent assays showed similar results (*<i>P</i>&lt;0.05). (<b>C</b>) bFGF activates ERK1/2 through RAS and MEK. Phosphorylated ERK1/2 inhibited β-catenin expression and then bone formation was reduced.</div></p></div><div class="aux"><div class="resc">Bei Li, et al. Oral Dis. ;18(3):285-292.</div><p class="links"><a class="dblinks" href="/pubmed/22151351" ref="ordinalpos=4984&amp;ncbi_uid=4753575&amp;link_uid=22151351">Citation</a><a class="dblinks" href="/pmc/articles/PMC3292657/" ref="ordinalpos=4984&amp;ncbi_uid=4753575&amp;link_uid=PMC3292657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4985.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3018566/figure/F3/" ref="ordinalpos=4985&amp;ncbi_uid=3609556&amp;link_uid=PMC3018566" href="/pmc/articles/PMC3018566/"><img src="/pmc/articles/instance/3018566/bin/nihms247147f3.gif" src-large="/pmc/articles/instance/3018566/bin/nihms247147f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3018566/" ref="ordinalpos=4985&amp;ncbi_uid=3609556&amp;link_uid=PMC3018566" image-link="/pmc/articles/PMC3018566/figure/F3/" class="imagepopup">Figure 3. Effect of leptin receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on macrophage content of epididymal fat tissue (immunohistochemical staining with macrophage specific Mac-3 antibody).  From: Leptin Receptor-induced STAT3-Independent <span class="highlight" style="background-color:">Signaling</span> Pathways are Protective against Atherosclerosis in a Murine Model of Obesity and Hyperlipidemia. </a></p><div class="supp"><p class="details"><div>A) <i>Lepr<sup>+/+</sup>,ApoE<sup>−/−</sup></i>. B) <i>Lepr<sup>s/s</sup>,ApoE<sup>−/−</sup></i>. C) <i>Lepr<sup>db/db</sup>,ApoE<sup>−/−</sup></i>. D) % Mac-3 stained cells. E) Plasma Mcp-1 level. F) Plasma FABP4 level. N=6–8 per group. *<i>P</i>&lt;0.05. Arrows point to Mac-3 positive stained cells (brown). Magnification is 40×, arrows showing stained cells.</div></p></div><div class="aux"><div class="resc">Wei Luo, et al. Atherosclerosis. ;214(1):81-85.</div><p class="links"><a class="dblinks" href="/pubmed/21067751" ref="ordinalpos=4985&amp;ncbi_uid=3609556&amp;link_uid=21067751">Citation</a><a class="dblinks" href="/pmc/articles/PMC3018566/" ref="ordinalpos=4985&amp;ncbi_uid=3609556&amp;link_uid=PMC3018566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4986.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2763556/figure/F3/" ref="ordinalpos=4986&amp;ncbi_uid=2431738&amp;link_uid=PMC2763556" href="/pmc/articles/PMC2763556/"><img src="/pmc/articles/instance/2763556/bin/nihms124335f3.gif" src-large="/pmc/articles/instance/2763556/bin/nihms124335f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2763556/" ref="ordinalpos=4986&amp;ncbi_uid=2431738&amp;link_uid=PMC2763556" image-link="/pmc/articles/PMC2763556/figure/F3/" class="imagepopup">Figure 3. In vivo effects of alterations in the Fas Ag <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Human and mouse colon cancer utilizes CD95 <span class="highlight" style="background-color:">signaling</span> for local growth and metastatic spread to liver. </a></p><div class="supp"><p class="details"><div>A) MC38 or M38/FLIP/L cells were injected subcutaneously or intravenously in C57BL/6 mice. At 4 weeks local and metastatic tumor growth was evaluated. B) Tumor characteristics.<br />N=10 for all groups except MC38 where n=30, MC38/FLIP where n=9. All tumors were evaluated at 4 weeks except * = 3 weeks due to large tumor size).</div></p></div><div class="aux"><div class="resc">Hanchen Li, et al. Gastroenterology. ;137(3):934-944.e4.</div><p class="links"><a class="dblinks" href="/pubmed/19524576" ref="ordinalpos=4986&amp;ncbi_uid=2431738&amp;link_uid=19524576">Citation</a><a class="dblinks" href="/pmc/articles/PMC2763556/" ref="ordinalpos=4986&amp;ncbi_uid=2431738&amp;link_uid=PMC2763556">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4987.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2245850/figure/fig7/" ref="ordinalpos=4987&amp;ncbi_uid=1295785&amp;link_uid=PMC2245850" href="/pmc/articles/PMC2245850/"><img src="/pmc/articles/instance/2245850/bin/pp1460576f07.gif" src-large="/pmc/articles/instance/2245850/bin/pp1460576f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2245850/" ref="ordinalpos=4987&amp;ncbi_uid=1295785&amp;link_uid=PMC2245850" image-link="/pmc/articles/PMC2245850/figure/fig7/" class="imagepopup">Figure 7. From: UV-B <span class="highlight" style="background-color:">Signaling</span> Pathways with Different Fluence-Rate Response Profiles Are Distinguished in Mature Arabidopsis Leaf Tissue by Requirement for UVR8, HY5, and HYH. </a></p><div class="supp"><p class="details"><div>Model showing the UVR8-dependent and UVR8-independent UV-B <span class="highlight" style="background-color:">signaling</span> pathways and representative genes they regulate in mature Arabidopsis leaf tissue. The broken arrow leading from HYH to the <i>CHS</i>, <i>ELIP1</i>, and <i>CRYD</i> genes indicates that HYH cannot completely compensate for the absence of HY5 in this <span class="highlight" style="background-color:">pathway</span>. The large arrow leading from UV-B to UVR8 overlaps the low fluence and high fluence range, indicating that both high and low UV-B fluence rates can activate UVR8.</div></p></div><div class="aux"><div class="resc">Bobby A. Brown, et al. Plant Physiol. 2008 February;146(2):576-588.</div><p class="links"><a class="dblinks" href="/pubmed/18055587" ref="ordinalpos=4987&amp;ncbi_uid=1295785&amp;link_uid=18055587">Citation</a><a class="dblinks" href="/pmc/articles/PMC2245850/" ref="ordinalpos=4987&amp;ncbi_uid=1295785&amp;link_uid=PMC2245850">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4988.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1469776/figure/F1/" ref="ordinalpos=4988&amp;ncbi_uid=673875&amp;link_uid=PMC1469776" href="/pmc/articles/PMC1469776/"><img src="/pmc/articles/instance/1469776/bin/JCI0624652.f1.gif" src-large="/pmc/articles/instance/1469776/bin/JCI0624652.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1469776/" ref="ordinalpos=4988&amp;ncbi_uid=673875&amp;link_uid=PMC1469776" image-link="/pmc/articles/PMC1469776/figure/F1/" class="imagepopup">Figure 1. EMT-specific upregulation of PDGFR <span class="highlight" style="background-color:">pathway</span> genes generates an autocrine PDGF/PDGFR loop..  From: Autocrine PDGFR <span class="highlight" style="background-color:">signaling</span> promotes mammary cancer metastasis. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Regulation of PDGFR <span class="highlight" style="background-color:">pathway</span> genes in various cell pairs based on EpH4 mouse mammary epithelial cells that undergo TGF-β–induced alterations in epithelial plasticity (see ref. 29). Two cell pairs underwent EMT in response to TGF-β (yellow), 2 underwent reversible scattering (green), and 2 cell pairs served as oncogene controls (black). Shown are the oncogenes cooperating with TGF-β, the predominant <span class="highlight" style="background-color:">signaling</span> pathways activated, and the fold regulation of 5 PDGF <span class="highlight" style="background-color:">pathway</span> genes (red: upregulation; pink: no significant regulation, blue: downregulation). (<b>B</b>) Concentrated, serum-free supernatants from EpRas and EpRasXT cells were tested on huPDGFR-A/FDCP-1 cells for mitogenic activity ([<sup>3</sup>H]thymidine incorporation). Specificity of the mitogenic response for PDGF was verified by addition of saturating amounts of a neutralizing α–PDGF-A/B antibody.                </div></p></div><div class="aux"><div class="resc">Martin Jechlinger, et al. J Clin Invest. 2006 June 1;116(6):1561-1570.</div><p class="links"><a class="dblinks" href="/pubmed/16741576" ref="ordinalpos=4988&amp;ncbi_uid=673875&amp;link_uid=16741576">Citation</a><a class="dblinks" href="/pmc/articles/PMC1469776/" ref="ordinalpos=4988&amp;ncbi_uid=673875&amp;link_uid=PMC1469776">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4989.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4278318/figure/F1/" ref="ordinalpos=4989&amp;ncbi_uid=7065125&amp;link_uid=PMC4278318" href="/pmc/articles/PMC4278318/"><img src="/pmc/articles/instance/4278318/bin/oncoscience-01-0476-g001.gif" src-large="/pmc/articles/instance/4278318/bin/oncoscience-01-0476-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4278318/" ref="ordinalpos=4989&amp;ncbi_uid=7065125&amp;link_uid=PMC4278318" image-link="/pmc/articles/PMC4278318/figure/F1/" class="imagepopup">Figure 1. Schematic representation of negative impact of NMI on TGFβ <span class="highlight" style="background-color:">signaling</span>.  From: NMI and EMT. </a></p><div class="supp"><p class="details"><div>IFN-γ – interferon gamma; JAK1 – Janus kinase-1; STAT1/5 – signal transducer and activator of transcription-1/5; NMI – N-myc interactor; TGF-B/ SMAD – TGF-β/SMAD <span class="highlight" style="background-color:">pathway</span>;? represents possible other mechanisms.477</div></p></div><div class="aux"><div class="resc">Rajeev S. Samant, et al. Oncoscience. 2014;1(7):476-477.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4989&amp;ncbi_uid=7065125&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4278318/" ref="ordinalpos=4989&amp;ncbi_uid=7065125&amp;link_uid=PMC4278318">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4990.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3335425/figure/fig5/" ref="ordinalpos=4990&amp;ncbi_uid=3990412&amp;link_uid=PMC3335425" href="/pmc/articles/PMC3335425/"><img src="/pmc/articles/instance/3335425/bin/JBB2012-305964.005.gif" src-large="/pmc/articles/instance/3335425/bin/JBB2012-305964.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3335425/" ref="ordinalpos=4990&amp;ncbi_uid=3990412&amp;link_uid=PMC3335425" image-link="/pmc/articles/PMC3335425/figure/fig5/" class="imagepopup">Figure 5.  From: Quantitative Proteomic (iTRAQ) Analysis of 1st Trimester Maternal Plasma Samples in Pregnancies at Risk for Preeclampsia. </a></p><div class="supp"><p class="details"><div>PANTHER analysis for <span class="highlight" style="background-color:">pathway</span>. In total, 10 proteins were identified as elevated. Shown above is the different <span class="highlight" style="background-color:">signaling</span> pathways hits by these protein.</div></p></div><div class="aux"><div class="resc">Varaprasad Kolla, et al. J Biomed Biotechnol. 2012;2012:305964.</div><p class="links"><a class="dblinks" href="/pubmed/22570525" ref="ordinalpos=4990&amp;ncbi_uid=3990412&amp;link_uid=22570525">Citation</a><a class="dblinks" href="/pmc/articles/PMC3335425/" ref="ordinalpos=4990&amp;ncbi_uid=3990412&amp;link_uid=PMC3335425">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4991.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2584886/figure/f9/" ref="ordinalpos=4991&amp;ncbi_uid=1612343&amp;link_uid=PMC2584886" href="/pmc/articles/PMC2584886/"><img src="/pmc/articles/instance/2584886/bin/zh50110853070009.gif" src-large="/pmc/articles/instance/2584886/bin/zh50110853070009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2584886/" ref="ordinalpos=4991&amp;ncbi_uid=1612343&amp;link_uid=PMC2584886" image-link="/pmc/articles/PMC2584886/figure/f9/" class="imagepopup">Fig. 9. From: CRTH2 antagonism significantly ameliorates airway hyperreactivity and downregulates inflammation-induced genes in a mouse model of airway inflammation. </a></p><div class="supp"><p class="details"><div>Heatmap of select genes downregulated in CRA-challenged mice treated with Compound A. In this clustergram, genes are grouped into arachidonic acid <span class="highlight" style="background-color:">pathway</span>, <span class="highlight" style="background-color:">signaling</span> molecules, and transcription factors.</div></p></div><div class="aux"><div class="resc">Nicholas W. Lukacs, et al. Am J Physiol Lung Cell Mol Physiol. 2008 November;295(5):L767-L779.</div><p class="links"><a class="dblinks" href="/pubmed/18757520" ref="ordinalpos=4991&amp;ncbi_uid=1612343&amp;link_uid=18757520">Citation</a><a class="dblinks" href="/pmc/articles/PMC2584886/" ref="ordinalpos=4991&amp;ncbi_uid=1612343&amp;link_uid=PMC2584886">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4992.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2975945/figure/F2/" ref="ordinalpos=4992&amp;ncbi_uid=2569121&amp;link_uid=PMC2975945" href="/pmc/articles/PMC2975945/"><img src="/pmc/articles/instance/2975945/bin/nihms-240452-f0003.gif" src-large="/pmc/articles/instance/2975945/bin/nihms-240452-f0003.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2975945/" ref="ordinalpos=4992&amp;ncbi_uid=2569121&amp;link_uid=PMC2975945" image-link="/pmc/articles/PMC2975945/figure/F2/" class="imagepopup">Figure 2. Proteomic coverage of the JAK/Stat <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as determined by Ingenuity Pathways Analysis.  From: Characterization of the Human Pancreatic Islet Proteome by Two-Dimensional LC/MS/MS. </a></p><div class="supp"><p class="details"><div>Those proteins identified in this work by two ore more peptides are shaded in grey.</div></p></div><div class="aux"><div class="resc">Thomas O. Metz, et al. J Proteome Res. ;5(12):3345-3354.</div><p class="links"><a class="dblinks" href="/pubmed/17137336" ref="ordinalpos=4992&amp;ncbi_uid=2569121&amp;link_uid=17137336">Citation</a><a class="dblinks" href="/pmc/articles/PMC2975945/" ref="ordinalpos=4992&amp;ncbi_uid=2569121&amp;link_uid=PMC2975945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4993.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2975945/figure/F4/" ref="ordinalpos=4993&amp;ncbi_uid=2569119&amp;link_uid=PMC2975945" href="/pmc/articles/PMC2975945/"><img src="/pmc/articles/instance/2975945/bin/nihms-240452-f0005.gif" src-large="/pmc/articles/instance/2975945/bin/nihms-240452-f0005.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2975945/" ref="ordinalpos=4993&amp;ncbi_uid=2569119&amp;link_uid=PMC2975945" image-link="/pmc/articles/PMC2975945/figure/F4/" class="imagepopup">Figure 4. Proteomic coverage of the NF-κβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as determined by Ingenuity Pathways Analysis.  From: Characterization of the Human Pancreatic Islet Proteome by Two-Dimensional LC/MS/MS. </a></p><div class="supp"><p class="details"><div>Those proteins identified in this work by two ore more peptides are shaded in grey.</div></p></div><div class="aux"><div class="resc">Thomas O. Metz, et al. J Proteome Res. ;5(12):3345-3354.</div><p class="links"><a class="dblinks" href="/pubmed/17137336" ref="ordinalpos=4993&amp;ncbi_uid=2569119&amp;link_uid=17137336">Citation</a><a class="dblinks" href="/pmc/articles/PMC2975945/" ref="ordinalpos=4993&amp;ncbi_uid=2569119&amp;link_uid=PMC2975945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4994.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3533466/figure/F5/" ref="ordinalpos=4994&amp;ncbi_uid=5658780&amp;link_uid=PMC3533466" href="/pmc/articles/PMC3533466/"><img src="/pmc/articles/instance/3533466/bin/mol.112.081497f5.gif" src-large="/pmc/articles/instance/3533466/bin/mol.112.081497f5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3533466/" ref="ordinalpos=4994&amp;ncbi_uid=5658780&amp;link_uid=PMC3533466" image-link="/pmc/articles/PMC3533466/figure/F5/" class="imagepopup">Fig. 5. From: Identification and Characterization of Poly(I:C)-induced Molecular Responses Attenuated by Nicotine in Mouse Macrophages. </a></p><div class="supp"><p class="details"><div>Nicotine’s effect on multiple signal transducers in the downstream <span class="highlight" style="background-color:">pathway</span> of calcium. Using IPA software, a calcium downstream <span class="highlight" style="background-color:">pathway</span> containing four <span class="highlight" style="background-color:">signaling</span> branches was generated: CAMK, PYK2, EGFR, and PLC. Significantly downregulated genes are shown in green. For a detailed summary of changes in each phospho-site, please see Table 3.</div></p></div><div class="aux"><div class="resc">Wen-Yan Cui, et al. Mol Pharmacol. 2013 January;83(1):61-72.</div><p class="links"><a class="dblinks" href="/pubmed/23028093" ref="ordinalpos=4994&amp;ncbi_uid=5658780&amp;link_uid=23028093">Citation</a><a class="dblinks" href="/pmc/articles/PMC3533466/" ref="ordinalpos=4994&amp;ncbi_uid=5658780&amp;link_uid=PMC3533466">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4995.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC261890/figure/pbio.0000053-g008/" ref="ordinalpos=4995&amp;ncbi_uid=296931&amp;link_uid=PMC261890" href="/pmc/articles/PMC261890/"><img src="/pmc/articles/instance/261890/bin/pbio.0000053.g008.gif" src-large="/pmc/articles/instance/261890/bin/pbio.0000053.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC261890/" ref="ordinalpos=4995&amp;ncbi_uid=296931&amp;link_uid=PMC261890" image-link="/pmc/articles/PMC261890/figure/pbio.0000053-g008/" class="imagepopup">Figure 8. Model of the Constitutive <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> That Provides a Basal Repression of RAG Gene Transcription.  From: T Cell Receptor-Independent Basal <span class="highlight" style="background-color:">Signaling</span> via Erk and Abl Kinases Suppresses RAG Gene Expression. </a></p><div class="supp"><p class="details"><div>Constitutive <span class="highlight" style="background-color:">signaling</span> in resting thymoytes and our model Jurkat T cell line represses <i>RAG</i> gene expression. Chemical inhibition of <span class="highlight" style="background-color:">signaling</span> molecules, genetic modifications, or induced expression of genes that resulted in elevated <i>RAG</i> expression are summarized in the gray-shaded boxes. The tonic signal and therefore normal expression of <i>RAG</i> genes rely strongly on <span class="highlight" style="background-color:">signaling</span> through the adapter LAT. The effects of basal phosphorylation of LAT are twofold: (1) recruitment and activation of PLCγ generates low levels of second messengers that signal through calcium and Ras pathways, culminating in Erk kinase activity; and (2) phosphorylation of tyrosine residue 6 of LAT, likely establishing a LAT–PLCγ–Abl complex, is required for low-level kinase activity of Abl. Abl and Erk kinase activities deliver unique repressive signals to control <i>RAG</i> gene expression.</div></p></div><div class="aux"><div class="resc">Jeroen P Roose, et al. PLoS Biol. 2003 November;1(2):e53.</div><p class="links"><a class="dblinks" href="/pubmed/14624253" ref="ordinalpos=4995&amp;ncbi_uid=296931&amp;link_uid=14624253">Citation</a><a class="dblinks" href="/pmc/articles/PMC261890/" ref="ordinalpos=4995&amp;ncbi_uid=296931&amp;link_uid=PMC261890">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4996.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3759360/figure/F2/" ref="ordinalpos=4996&amp;ncbi_uid=5661182&amp;link_uid=PMC3759360" href="/pmc/articles/PMC3759360/"><img src="/pmc/articles/instance/3759360/bin/nihms505366f2.gif" src-large="/pmc/articles/instance/3759360/bin/nihms505366f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3759360/" ref="ordinalpos=4996&amp;ncbi_uid=5661182&amp;link_uid=PMC3759360" image-link="/pmc/articles/PMC3759360/figure/F2/" class="imagepopup">Figure 2. NF-κB activation <span class="highlight" style="background-color:">signaling</span> pathways.  From: When Ubiquitin Meets NF-?B: A trove for anti-cancer drug development. </a></p><div class="supp"><p class="details"><div><b>(A)</b> A variety of extracellular stimuli activate NF-κB <span class="highlight" style="background-color:">signaling</span> via the canonical or non-canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The canonical <span class="highlight" style="background-color:">pathway</span> is activated in response to engagement of membrane-bound receptors of various cytokines or pathogens, such as TNFα, IL-1β and LPS. One characteristics of the canonical <span class="highlight" style="background-color:">pathway</span> is the activation of NEMO-containing IKK complex which can directly phosphorylate NF-κB-bound IκB proteins, resulting in their β-TrCP-SCF ligase –dependent polyubiquitination and subsequent proteasomal degradation. The activation of IKK is mediated by collaboration of polyubiquitin chains with a variety of linkages, including K63-, K11- and linear/M1-linkages. Degradation of IκBα releases NF-κB which then translocates into the nucleus and regulates gene transcription. The non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated by a smaller group of stimuli such as lymphotoxin-β and CD40 ligand. The engagement of these stimuli with their receptors leads to ubiquitination and proteasomal degradation of RING ligase TRAF3, and consequently inhibits TRAF3-dependent proteasomal degradation of NIK. The NIK is constitutively active and stabilization of NIK leads to activation of IKKα homodimer, which in turn phosphorylates p100. Phosphorylated p100 then undergoes proteasome-mediated partial processing, yielding the p52-RelB dimer. Subsequently, P52-RelB translocates into the nucleus to regulate gene transcription. <b>(B)</b> Genotoxic agents can induce nuclear DNA damage which initiates a genotoxic NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the nucleus. DNA damage may induce formation of nuclear PIDDosome and activation of PARP1, which both facilitate E3 PIASy-dependent sumoylation of NEMO. NEMO sumoylation enriches NEMO in the nucleus where it can associate with, and be phosphorylated by, the apical DNA damage responsive kinase ATM. Phosphorylation of NEMO promotes its monoubiquitination, which is required for NEMO-ATM complex export from the nucleus. In the cytoplasm, NEMO-ATM complex further recruits ELKS and facilitates K63-linked and M1-linked polyubiquitination on ELKS and NEMO, respectively, resulting in subsequent activation of IKK and NF-κB. Alternatively, ATM may be independently exported into the cytoplasm, where it associates with TRAF6 and promotes TRAF6 K63-linked polyubiquitination, also leading to activation of IKK and NF-κB.</div></p></div><div class="aux"><div class="resc">Zhao-Hui Wu, et al. Curr Pharm Des. 2013;19(18):3263-3275.</div><p class="links"><a class="dblinks" href="/pubmed/23151140" ref="ordinalpos=4996&amp;ncbi_uid=5661182&amp;link_uid=23151140">Citation</a><a class="dblinks" href="/pmc/articles/PMC3759360/" ref="ordinalpos=4996&amp;ncbi_uid=5661182&amp;link_uid=PMC3759360">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4997.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3937033/figure/F8/" ref="ordinalpos=4997&amp;ncbi_uid=5899092&amp;link_uid=PMC3937033" href="/pmc/articles/PMC3937033/"><img src="/pmc/articles/instance/3937033/bin/1752-0509-8-17-8.gif" src-large="/pmc/articles/instance/3937033/bin/1752-0509-8-17-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3937033/" ref="ordinalpos=4997&amp;ncbi_uid=5899092&amp;link_uid=PMC3937033" image-link="/pmc/articles/PMC3937033/figure/F8/" class="imagepopup">Figure 8.  From: Identification of core T cell network based on immunome interactome. </a></p><div class="supp"><p class="details"><div><b>T cell PPI Network mapped to TCR, with main JAK-STAT and MAPK <span class="highlight" style="background-color:">signaling</span> pathways.</b> The figure shows the TCR and the main JAK-STAT and MAPK <span class="highlight" style="background-color:">signaling</span> networks that are connected to the TCR-dependent T cell response. The <span class="highlight" style="background-color:">pathway</span> information is adapted from the KEGG Pathways resource. The TPPIN proteins are represented by red-colored boxes with protein names in white text. The <span class="highlight" style="background-color:">signaling</span> network demonstrates the effectiveness of the filtering approach as many of the central proteins in T cell response are left intact after filtering.</div></p></div><div class="aux"><div class="resc">Gabriel N Teku, et al. BMC Syst Biol. 2014;8:17-17.</div><p class="links"><a class="dblinks" href="/pubmed/24528953" ref="ordinalpos=4997&amp;ncbi_uid=5899092&amp;link_uid=24528953">Citation</a><a class="dblinks" href="/pmc/articles/PMC3937033/" ref="ordinalpos=4997&amp;ncbi_uid=5899092&amp;link_uid=PMC3937033">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4998.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3247990/figure/F8/" ref="ordinalpos=4998&amp;ncbi_uid=4469474&amp;link_uid=PMC3247990" href="/pmc/articles/PMC3247990/"><img src="/pmc/articles/instance/3247990/bin/zbc0021291680008.gif" src-large="/pmc/articles/instance/3247990/bin/zbc0021291680008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3247990/" ref="ordinalpos=4998&amp;ncbi_uid=4469474&amp;link_uid=PMC3247990" image-link="/pmc/articles/PMC3247990/figure/F8/" class="imagepopup">FIGURE 8. From: Nuclear Factor ?B Subunits RelB and cRel Negatively Regulate Toll-like Receptor 3-mediated ?-Interferon Production via Induction of Transcriptional Repressor Protein YY1. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of post-induction repression of IFN-β expression by YY1 in TLR3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> TLR3 is engaged by a dsRNA ligand (<i>e.g.</i> poly(I:C)) promoting activation of the TRIF/TBK1/inducible IκB kinase <span class="highlight" style="background-color:">pathway</span> leading to the binding of IRF3 and IRF7 to the PRD I/III domain of the IFN-β promoter. This results in increased transcriptional activation of the <i>IFN</i>-β gene. TLR3 is a poor activator of the classical NF-κB <span class="highlight" style="background-color:">pathway</span> and especially p65 but can strongly stimulate the nuclear translocation of the Rel subunits RelB and c-Rel followed by induction of YY1. The latter translocates to the nucleus in response to TLR3 <span class="highlight" style="background-color:">signaling</span>, where it displaces IRF7 from the IFN-β promoter resulting in post-induction repression of IFN-β expression.</div></p></div><div class="aux"><div class="resc">Jakub Siednienko, et al. J Biol Chem. 2011 December 30;286(52):44750-44763.</div><p class="links"><a class="dblinks" href="/pubmed/22065573" ref="ordinalpos=4998&amp;ncbi_uid=4469474&amp;link_uid=22065573">Citation</a><a class="dblinks" href="/pmc/articles/PMC3247990/" ref="ordinalpos=4998&amp;ncbi_uid=4469474&amp;link_uid=PMC3247990">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4999.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3906049/figure/pone-0086183-g007/" ref="ordinalpos=4999&amp;ncbi_uid=5776915&amp;link_uid=PMC3906049" href="/pmc/articles/PMC3906049/"><img src="/pmc/articles/instance/3906049/bin/pone.0086183.g007.gif" src-large="/pmc/articles/instance/3906049/bin/pone.0086183.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3906049/" ref="ordinalpos=4999&amp;ncbi_uid=5776915&amp;link_uid=PMC3906049" image-link="/pmc/articles/PMC3906049/figure/pone-0086183-g007/" class="imagepopup">Figure 7. Starvation effects on insulin <span class="highlight" style="background-color:">signaling</span> components and JH synthesis in the CA of mosquitoes..  From: Starvation Increases Insulin Sensitivity and Reduces Juvenile Hormone Synthesis in Mosquitoes. </a></p><div class="supp"><p class="details"><div>This scheme summarizes starvation-related changes of insulin/TOR <span class="highlight" style="background-color:">pathway</span> components and JH synthesis. Molecules in red color are down-regulated (↓), while those in green are upregulated (↑). Previous experiments established the involvement of phosphoinositide 3-kinase (PI3K) and TOR in the transduction of insulin <span class="highlight" style="background-color:">signaling</span> in the CA [13]. Astarvation-dependent decrease of insulin results in an increase of FOXO <span class="highlight" style="background-color:">signaling</span> that promotes activation of transcription of insulin receptor (INSr) and 4E-binding protein (4EBP). Transcripts levels for FOXO increase and mRNAs for JHAMT and TOR decrease. JH synthesis decrease, while increases of 4EBP inhibit translation and increases of INSr enhance insulin sensitivity.</div></p></div><div class="aux"><div class="resc">Meritxell Perez-Hedo, et al. PLoS One. 2014;9(1):e86183.</div><p class="links"><a class="dblinks" href="/pubmed/24489697" ref="ordinalpos=4999&amp;ncbi_uid=5776915&amp;link_uid=24489697">Citation</a><a class="dblinks" href="/pmc/articles/PMC3906049/" ref="ordinalpos=4999&amp;ncbi_uid=5776915&amp;link_uid=PMC3906049">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>5000.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3685283/figure/F5/" ref="ordinalpos=5000&amp;ncbi_uid=5258462&amp;link_uid=PMC3685283" href="/pmc/articles/PMC3685283/"><img src="/pmc/articles/instance/3685283/bin/nihms398418f5.gif" src-large="/pmc/articles/instance/3685283/bin/nihms398418f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3685283/" ref="ordinalpos=5000&amp;ncbi_uid=5258462&amp;link_uid=PMC3685283" image-link="/pmc/articles/PMC3685283/figure/F5/" class="imagepopup">Figure 5.  From: INHIBITION OF THE HEDGEHOG <span class="highlight" style="background-color:">PATHWAY</span> TARGETS THE TUMOR-ASSOCIATED STROMA IN PANCREATIC CANCER. </a></p><div class="supp"><p class="details"><div>Hh <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in PDAC <i>in vitro</i> and <i>in vivo</i>. (A) Quantitative PCR for Gli1 expression. HPSC-GFP cells were cultured with or without Bxpc3 for 96 hours and sorted by flow cytometry. ** p&lt;0.0001. Immunohistochemistry for Ki67 (B) and CD31 (C) was performed on mouse pancreatic tumors with tumor/stroma ratio 1:3 treated with AZD8542 or control. Total magnification 100X. Data is shown as means ± SEM;* p&lt;0.005 and ** p&lt;0.0001.</div></p></div><div class="aux"><div class="resc">Rosa F. Hwang, et al. Mol Cancer Res. ;10(9):1147-1157.</div><p class="links"><a class="dblinks" href="/pubmed/22859707" ref="ordinalpos=5000&amp;ncbi_uid=5258462&amp;link_uid=22859707">Citation</a><a class="dblinks" href="/pmc/articles/PMC3685283/" ref="ordinalpos=5000&amp;ncbi_uid=5258462&amp;link_uid=PMC3685283">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="49" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="50" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="51" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:51:43-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A0A0E543141A10000000000673442;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A0A0E543141A10000000000673442" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>