
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="38" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A163554311D510000000000EA0B91" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="37" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="38" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="39" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="38" /></div>
            <div><h2 class="result_count">Results: 3701 to 3800 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3701.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507315/figure/fig6/" ref="ordinalpos=3701&amp;ncbi_uid=4461471&amp;link_uid=PMC3507315" href="/pmc/articles/PMC3507315/"><img src="/pmc/articles/instance/3507315/bin/MI2012-563709.006.gif" src-large="/pmc/articles/instance/3507315/bin/MI2012-563709.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507315/" ref="ordinalpos=3701&amp;ncbi_uid=4461471&amp;link_uid=PMC3507315" image-link="/pmc/articles/PMC3507315/figure/fig6/" class="imagepopup">Figure 6.  From: Clinical Application of Development of Nonantibiotic Macrolides That Correct Inflammation-Driven Immune Dysfunction in Inflammatory Skin Diseases. </a></p><div class="supp"><p class="details"><div>Nitric oxide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> extracted from BioCarta database [9].</div></p></div><div class="aux"><div class="resc">Carmen Rodriguez-Cerdeira, et al. Mediators Inflamm. 2012;2012:563709.</div><p class="links"><a class="dblinks" href="/pubmed/23258954" ref="ordinalpos=3701&amp;ncbi_uid=4461471&amp;link_uid=23258954">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507315/" ref="ordinalpos=3701&amp;ncbi_uid=4461471&amp;link_uid=PMC3507315">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3702.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3335081/figure/pone-0035712-g009/" ref="ordinalpos=3702&amp;ncbi_uid=3968276&amp;link_uid=PMC3335081" href="/pmc/articles/PMC3335081/"><img src="/pmc/articles/instance/3335081/bin/pone.0035712.g009.gif" src-large="/pmc/articles/instance/3335081/bin/pone.0035712.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3335081/" ref="ordinalpos=3702&amp;ncbi_uid=3968276&amp;link_uid=PMC3335081" image-link="/pmc/articles/PMC3335081/figure/pone-0035712-g009/" class="imagepopup">Figure 9. The cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for osteogenic/adipogenic differentiation in MSCs by resveratrol..  From: Resveratrol Mediated Modulation of Sirt-1/Runx2 Promotes Osteogenic Differentiation of Mesenchymal Stem Cells: Potential Role of Runx2 Deacetylation. </a></p><div class="aux"><div class="resc">Mehdi Shakibaei, et al. PLoS One. 2012;7(4):e35712.</div><p class="links"><a class="dblinks" href="/pubmed/22539994" ref="ordinalpos=3702&amp;ncbi_uid=3968276&amp;link_uid=22539994">Citation</a><a class="dblinks" href="/pmc/articles/PMC3335081/" ref="ordinalpos=3702&amp;ncbi_uid=3968276&amp;link_uid=PMC3335081">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3703.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262585/figure/fig1/" ref="ordinalpos=3703&amp;ncbi_uid=3781893&amp;link_uid=PMC3262585" href="/pmc/articles/PMC3262585/"><img src="/pmc/articles/instance/3262585/bin/IJBC2011-923250.001.gif" src-large="/pmc/articles/instance/3262585/bin/IJBC2011-923250.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262585/" ref="ordinalpos=3703&amp;ncbi_uid=3781893&amp;link_uid=PMC3262585" image-link="/pmc/articles/PMC3262585/figure/fig1/" class="imagepopup">Figure 1.  From: The Role of Aryl Hydrocarbon Receptor and Crosstalk with Estrogen Receptor in Response of Breast Cancer Cells to the Novel Antitumor Agents Benzothiazoles and Aminoflavone. </a></p><div class="supp"><p class="details"><div>The aryl hydrocarbon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mariana A. Callero, et al. Int J Breast Cancer. 2011;2011:923250.</div><p class="links"><a class="dblinks" href="/pubmed/22295239" ref="ordinalpos=3703&amp;ncbi_uid=3781893&amp;link_uid=22295239">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262585/" ref="ordinalpos=3703&amp;ncbi_uid=3781893&amp;link_uid=PMC3262585">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3704.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1440308/figure/F5/" ref="ordinalpos=3704&amp;ncbi_uid=579804&amp;link_uid=PMC1440308" href="/pmc/articles/PMC1440308/"><img src="/pmc/articles/instance/1440308/bin/1742-4682-3-13-5.gif" src-large="/pmc/articles/instance/1440308/bin/1742-4682-3-13-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1440308/" ref="ordinalpos=3704&amp;ncbi_uid=579804&amp;link_uid=PMC1440308" image-link="/pmc/articles/PMC1440308/figure/F5/" class="imagepopup">Figure 5.  From: A method for the generation of standardized qualitative dynamical systems of regulatory networks. </a></p><div class="supp"><p class="details"><div><b>Alternative Th network</b>. T helper <span class="highlight" style="background-color:">pathway</span> published in [70], reinterpreted as a <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Luis Mendoza, et al. Theor Biol Med Model. 2006;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/16542429" ref="ordinalpos=3704&amp;ncbi_uid=579804&amp;link_uid=16542429">Citation</a><a class="dblinks" href="/pmc/articles/PMC1440308/" ref="ordinalpos=3704&amp;ncbi_uid=579804&amp;link_uid=PMC1440308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3705.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1440308/figure/F7/" ref="ordinalpos=3705&amp;ncbi_uid=579798&amp;link_uid=PMC1440308" href="/pmc/articles/PMC1440308/"><img src="/pmc/articles/instance/1440308/bin/1742-4682-3-13-7.gif" src-large="/pmc/articles/instance/1440308/bin/1742-4682-3-13-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1440308/" ref="ordinalpos=3705&amp;ncbi_uid=579798&amp;link_uid=PMC1440308" image-link="/pmc/articles/PMC1440308/figure/F7/" class="imagepopup">Figure 7.  From: A method for the generation of standardized qualitative dynamical systems of regulatory networks. </a></p><div class="supp"><p class="details"><div><b>Alternative Th network</b>. T helper <span class="highlight" style="background-color:">pathway</span> published in [71], reinterpreted as a <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Luis Mendoza, et al. Theor Biol Med Model. 2006;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/16542429" ref="ordinalpos=3705&amp;ncbi_uid=579798&amp;link_uid=16542429">Citation</a><a class="dblinks" href="/pmc/articles/PMC1440308/" ref="ordinalpos=3705&amp;ncbi_uid=579798&amp;link_uid=PMC1440308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3706.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC533963/figure/f11/" ref="ordinalpos=3706&amp;ncbi_uid=406577&amp;link_uid=PMC533963" href="/pmc/articles/PMC533963/"><img src="/pmc/articles/instance/533963/bin/zmb0240446070011.gif" src-large="/pmc/articles/instance/533963/bin/zmb0240446070011.jpg" alt="FIG. 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC533963/" ref="ordinalpos=3706&amp;ncbi_uid=406577&amp;link_uid=PMC533963" image-link="/pmc/articles/PMC533963/figure/f11/" class="imagepopup">FIG. 11. From: Brk Activates Rac1 and Promotes Cell Migration and Invasion by Phosphorylating Paxillin. </a></p><div class="supp"><p class="details"><div>Model illustrating the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through which Brk participates in EGF-induced cell migration and invasion.</div></p></div><div class="aux"><div class="resc">Hsin-Yi Chen, et al. Mol Cell Biol. 2004 December;24(24):10558-10572.</div><p class="links"><a class="dblinks" href="/pubmed/15572663" ref="ordinalpos=3706&amp;ncbi_uid=406577&amp;link_uid=15572663">Citation</a><a class="dblinks" href="/pmc/articles/PMC533963/" ref="ordinalpos=3706&amp;ncbi_uid=406577&amp;link_uid=PMC533963">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3707.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3925772/figure/F1/" ref="ordinalpos=3707&amp;ncbi_uid=6215737&amp;link_uid=PMC3925772" href="/pmc/articles/PMC3925772/"><img src="/pmc/articles/instance/3925772/bin/nihms548108f1.gif" src-large="/pmc/articles/instance/3925772/bin/nihms548108f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3925772/" ref="ordinalpos=3707&amp;ncbi_uid=6215737&amp;link_uid=PMC3925772" image-link="/pmc/articles/PMC3925772/figure/F1/" class="imagepopup">Figure 1. Cellular Optogenetics: Approaches for Dissecting Complex <span class="highlight" style="background-color:">Signaling</span> Networks.  From: Using Optogenetics to Interrogate the Dynamic Control of Signal Transmission by the Ras/Erk Module. </a></p><div class="supp"><p class="details"><div>(A) Optogenetic inputs can be used to stimulate a single intracellular node to isolate subnetworks within the full physiological <span class="highlight" style="background-color:">signaling</span> response network.<br />(B) By applying time-varying light inputs, it is possible to dissect how dynamics are transmitted and drive specific responses.<br />(C) These approaches can be used to understand how a shared <span class="highlight" style="background-color:">signaling</span> node can yield distinct responses when reused in multiple physiological pathways. Specificity can be encoded by distinct combinations of activated pathways (left panel) or by the dynamics of activation (e.g., duration or amplitude) of a single <span class="highlight" style="background-color:">pathway</span> (right panel).<br />(D) Direct optogenetic activation of shared nodes provides a powerful approach to dissect these mechanisms of encoding.</div></p></div><div class="aux"><div class="resc">Jared E. Toettcher, et al. Cell. ;155(6):1422-1434.</div><p class="links"><a class="dblinks" href="/pubmed/24315106" ref="ordinalpos=3707&amp;ncbi_uid=6215737&amp;link_uid=24315106">Citation</a><a class="dblinks" href="/pmc/articles/PMC3925772/" ref="ordinalpos=3707&amp;ncbi_uid=6215737&amp;link_uid=PMC3925772">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3708.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3688827/figure/pone-0065128-g006/" ref="ordinalpos=3708&amp;ncbi_uid=5058703&amp;link_uid=PMC3688827" href="/pmc/articles/PMC3688827/"><img src="/pmc/articles/instance/3688827/bin/pone.0065128.g006.gif" src-large="/pmc/articles/instance/3688827/bin/pone.0065128.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3688827/" ref="ordinalpos=3708&amp;ncbi_uid=5058703&amp;link_uid=PMC3688827" image-link="/pmc/articles/PMC3688827/figure/pone-0065128-g006/" class="imagepopup">Figure 6. Diagram of integrin (ITGA5 and ITGAV) and SRC <span class="highlight" style="background-color:">signaling</span> pathways..  From: Transcriptome Analysis Reveals Novel Entry Mechanisms and a Central Role of SRC in Host Defense during High Multiplicity Mycobacterial Infection. </a></p><div class="supp"><p class="details"><div>Shown are AhR <span class="highlight" style="background-color:">signaling</span>, the NRF2-mediated oxidative stress response, and the NLRP3 inflammasome activation <span class="highlight" style="background-color:">pathway</span> regulated by ITGA5/ITGAV and SRC axis. Up-regulated genes downstream of ITGA5/ITGAV and SRC were linked to these three identified <span class="highlight" style="background-color:">signaling</span> pathways with data only displayed from BCG infection. For details on gene abbreviations and fold changes compared with control, see Table S3. The network is graphically represented as nodes (gene products) and edges (the biological relationship between nodes). Red-shaded nodes represent up-regulated genes.</div></p></div><div class="aux"><div class="resc">Jay Zhang. PLoS One. 2013;8(6):e65128.</div><p class="links"><a class="dblinks" href="/pubmed/23824656" ref="ordinalpos=3708&amp;ncbi_uid=5058703&amp;link_uid=23824656">Citation</a><a class="dblinks" href="/pmc/articles/PMC3688827/" ref="ordinalpos=3708&amp;ncbi_uid=5058703&amp;link_uid=PMC3688827">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3709.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013623/figure/f3-ijms-15-06184/" ref="ordinalpos=3709&amp;ncbi_uid=6111772&amp;link_uid=PMC4013623" href="/pmc/articles/PMC4013623/"><img src="/pmc/articles/instance/4013623/bin/ijms-15-06184f3.gif" src-large="/pmc/articles/instance/4013623/bin/ijms-15-06184f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013623/" ref="ordinalpos=3709&amp;ncbi_uid=6111772&amp;link_uid=PMC4013623" image-link="/pmc/articles/PMC4013623/figure/f3-ijms-15-06184/" class="imagepopup">Figure 3. From: Obesity and Its Metabolic Complications: The Role of Adipokines and the Relationship between Obesity, Inflammation, Insulin Resistance, Dyslipidemia and Nonalcoholic Fatty Liver Disease. </a></p><div class="supp"><p class="details"><div>Schematic view of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in adipose tissue. Binding of insulin to its receptor on adipocytes triggers the phosphorylation and activation of insulin receptor substrate, which forms a docking site for phosphatidylinositol 3-kinase (PI3K) at the membrane. When docked, PI3K converts phosphatidylinositol 4,5-bisphosphate to phosphatidylinositol 3,4,5-trisphosphate, a second messenger that activates phosphoinositide-dependent protein kinase 1 and recruits Akt (also known as protein kinase B, PKB) to the cell membrane. Consequently, PI3K-AKT/PKB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates metabolic processes. The red arrows indicate up-regulation (when pointing upward) or down-regulation (when pointing downward) in response to PI3K-AKT/PKB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Ras-mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> leads to the activation of genes which are involved in cell growth, thereby promoting inflammation and atherogenesis. IRS-1, insulin receptor substrate; MAPK, mitogen-activated protein kinase; PDK, phosphoinositide-dependent protein kinase 1; PI3K, phosphatidylinositol 3-kinase; PIP2, phosphatidylinositol 4,5-bisphosphate; PIP3, phosphatidylinositol 3,4,5-trisphosphate; PKB, protein kinase B.</div></p></div><div class="aux"><div class="resc">Un Ju Jung, et al. Int J Mol Sci. 2014 April;15(4):6184-6223.</div><p class="links"><a class="dblinks" href="/pubmed/24733068" ref="ordinalpos=3709&amp;ncbi_uid=6111772&amp;link_uid=24733068">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013623/" ref="ordinalpos=3709&amp;ncbi_uid=6111772&amp;link_uid=PMC4013623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3710.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3931013/figure/F3/" ref="ordinalpos=3710&amp;ncbi_uid=5836107&amp;link_uid=PMC3931013" href="/pmc/articles/PMC3931013/"><img src="/pmc/articles/instance/3931013/bin/nihms552613f3.gif" src-large="/pmc/articles/instance/3931013/bin/nihms552613f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3931013/" ref="ordinalpos=3710&amp;ncbi_uid=5836107&amp;link_uid=PMC3931013" image-link="/pmc/articles/PMC3931013/figure/F3/" class="imagepopup">Figure 3.  From: Identification of Predictive Markers of Response to the MEK1/2 Inhibitor Selumetinib (AZD6244) in K-ras-Mutated Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> represented by KEGG analysis. Red boxes represent core genes in the Wnt <span class="highlight" style="background-color:">pathway</span> with increased expression in K-<i>ras</i> mutant CRC cells resistant to AZD6244.</div></p></div><div class="aux"><div class="resc">John J. Tentler, et al. Mol Cancer Ther. ;9(12):3351-3362.</div><p class="links"><a class="dblinks" href="/pubmed/20923857" ref="ordinalpos=3710&amp;ncbi_uid=5836107&amp;link_uid=20923857">Citation</a><a class="dblinks" href="/pmc/articles/PMC3931013/" ref="ordinalpos=3710&amp;ncbi_uid=5836107&amp;link_uid=PMC3931013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3711.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2330018/figure/F2/" ref="ordinalpos=3711&amp;ncbi_uid=1370747&amp;link_uid=PMC2330018" href="/pmc/articles/PMC2330018/"><img src="/pmc/articles/instance/2330018/bin/nihms-44092-f0002.gif" src-large="/pmc/articles/instance/2330018/bin/nihms-44092-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2330018/" ref="ordinalpos=3711&amp;ncbi_uid=1370747&amp;link_uid=PMC2330018" image-link="/pmc/articles/PMC2330018/figure/F2/" class="imagepopup">Figure 2.  From: Characterization of Wnt <span class="highlight" style="background-color:">Signaling</span> during Photoreceptor Degeneration. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">pathway</span> activation protects primary retinal cultures from oxidative stress. (<b>A</b>) Viability of the retinal cultures was reduced by 50% by 0.1 mM H<sub>2</sub>O<sub>2</sub>. Wnt <span class="highlight" style="background-color:">signaling</span> activator SB216763 (40 <i></i>M) significantly increased the viability of the retinal cultures, whereas the vehicle control DMSO had no effect. Viability was measured by XTT assay and normalized to the untreated cultures. (<b>B</b>) TUNEL assay confirmed that photoreceptors in the cultures were rescued from H<sub>2</sub>O<sub>2</sub> toxicity. Recombinant Wnt3a ligand (150 ng/mL) was used to activate the Wnt <span class="highlight" style="background-color:">pathway</span>, and rhodopsin+/TUNEL+ cells were counted. (<b>C</b>) Recombinant Wnt3a increased the viability of retinal cultures exposed to H<sub>2</sub>O<sub>2</sub>, measured by XTT assay. Two concentrations of Wnt3a were tested. The no Wnt cells were treated with H<sub>2</sub>O<sub>2</sub> alone. (<b>D</b>, <b>E</b>) The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was activated in the retinal cultures. Cultures were transfected with the Wnt reporter plasmid and a LacZ plasmid and then were incubated with the Wnt activators SB216763 and Wnt3a. Wnt <span class="highlight" style="background-color:">signaling</span> was measured by the luciferase reporter assay using <i></i>-galactosidase activity for normalization (values given in thousands).</div></p></div><div class="aux"><div class="resc">Hyun Yi, et al. Invest Ophthalmol Vis Sci. ;48(12):5733-5741.</div><p class="links"><a class="dblinks" href="/pubmed/18055826" ref="ordinalpos=3711&amp;ncbi_uid=1370747&amp;link_uid=18055826">Citation</a><a class="dblinks" href="/pmc/articles/PMC2330018/" ref="ordinalpos=3711&amp;ncbi_uid=1370747&amp;link_uid=PMC2330018">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3712.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938100/figure/F4/" ref="ordinalpos=3712&amp;ncbi_uid=3059937&amp;link_uid=PMC2938100" href="/pmc/articles/PMC2938100/"><img src="/pmc/articles/instance/2938100/bin/zjw0071036350004.gif" src-large="/pmc/articles/instance/2938100/bin/zjw0071036350004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938100/" ref="ordinalpos=3712&amp;ncbi_uid=3059937&amp;link_uid=PMC2938100" image-link="/pmc/articles/PMC2938100/figure/F4/" class="imagepopup">Fig. 4. From: Interactome Mapping of the Phosphatidylinositol 3-Kinase-Mammalian Target of Rapamycin <span class="highlight" style="background-color:">Pathway</span> Identifies Deformed Epidermal Autoregulatory Factor-1 as a New Glycogen Synthase Kinase-3 Interactor. </a></p><div class="supp"><p class="details"><div><b>Connections of literature-completed interactome of PI3K-mTOR <span class="highlight" style="background-color:">pathway</span> with other <span class="highlight" style="background-color:">signaling</span> pathways.</b> PI3K-mTOR <span class="highlight" style="background-color:">pathway</span> components as depicted as <i>triangles</i>, and interactors are depicted as <i>circles. Pink links</i> represent interactions found in this study, and <i>gray links</i> are manually curated interactions from the literature. The molecular function of each protein is described by a <i>color code</i>, and interactors are grouped according to their belonging to specific <span class="highlight" style="background-color:">signaling</span> pathways. <i>GAP</i>, GTPase-activating protein; <i>TGF</i>, transforming growth factor; <i>PKC</i>, protein kinase C; <i>MAP</i>, mitogen-activated protein; <i>NFKB</i>, nuclear factor B; <i>JAK</i>, Janus kinase; <i>STAT</i>, signal transducer and activator of transcription; <i>CaM</i>, calmodulin.</div></p></div><div class="aux"><div class="resc">Fanny Pilot-Storck, et al. Mol Cell Proteomics. 2010 July;9(7):1578-1593.</div><p class="links"><a class="dblinks" href="/pubmed/20368287" ref="ordinalpos=3712&amp;ncbi_uid=3059937&amp;link_uid=20368287">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938100/" ref="ordinalpos=3712&amp;ncbi_uid=3059937&amp;link_uid=PMC2938100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3713.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2900623/figure/F3/" ref="ordinalpos=3713&amp;ncbi_uid=2861846&amp;link_uid=PMC2900623" href="/pmc/articles/PMC2900623/"><img src="/pmc/articles/instance/2900623/bin/cam0402_0255_fig003.gif" src-large="/pmc/articles/instance/2900623/bin/cam0402_0255_fig003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2900623/" ref="ordinalpos=3713&amp;ncbi_uid=2861846&amp;link_uid=PMC2900623" image-link="/pmc/articles/PMC2900623/figure/F3/" class="imagepopup">Figure 3.  From: New therapeutic strategies targeting transmembrane signal transduction in the immune system. </a></p><div class="supp"><p class="details"><div>SCHOOL model of T-cell receptor <span class="highlight" style="background-color:">signaling</span>. A schematic representation of the SCHOOL-based molecular mechanisms of T-cell receptor (TCR) <span class="highlight" style="background-color:">signaling</span>. Immunoreceptor tyrosine-based activation motifs (ITAMs) are shown as gray rectangles. TCR-CD3- components are represented as whole polypeptides and as a simplified axial view. All interchain interactions in intermediate complexes are shown by dotted arrows reflecting their transition state. Circular arrow indicates ligand-induced receptor reorientation. Interaction with multivalent ligand (not shown) clusters the receptors and pushes them to reorientate (I) and bring <span class="highlight" style="background-color:">signaling</span> subunits into a correct relative orientation and in sufficient proximity in the formed receptor oligomer (for illustrative purposes, receptor dimer is shown), thus starting the trans-homointeractions between  molecules (II). Then, two alternative pathways can take a place depending on the nature of activating stimuli. First is going through a stage IV resulting in formation of 2 dimer (dimer of dimers) and phosphorylation of the  ITAM tyrosines, thus triggering downstream <span class="highlight" style="background-color:">signaling</span> events. Then, the <span class="highlight" style="background-color:">signaling</span>  oligomers formed subsequently dissociate from the TCR-CD3 complex, resulting in internalization of the remaining engaged TCR-CD3 complexes (VII). This <span class="highlight" style="background-color:">pathway</span> leads to partial (or incomplete) T cell activation. Alternatively, the intermediate complex formed at the stage II can undergo further rearrangements, starting trans-homointeractions between CD3 proteins (III) and resulting in formation of an oligomeric intermediate. Again, the stages I, II and III can be reversible or irreversible depending on interreceptor proximity and relative orientation of the receptors in TCR dimers/oligomers as well as on time duration of the TCR-ligand contact and lifetime of the receptor in TCR dimers/oligomers that generally correlate with the nature of the stimulus and its specificity and affinity/avidity. Next, in the <span class="highlight" style="background-color:">signaling</span> oligomers formed (III), the ITAM tyrosines undergo phosphorylation by PTKs that leads to generation of the activation signal, dissociation of <span class="highlight" style="background-color:">signaling</span> oligomers and internalization of the remaining engaged TCR chains (VIII, XI). This <span class="highlight" style="background-color:">pathway</span> provides at least two different activation signals from the  and CD3 <span class="highlight" style="background-color:">signaling</span> oligomers (signals A and B), respectively, and results in full T-cell activation. The distinct <span class="highlight" style="background-color:">signaling</span> through  and CD3 oligomers (or through various combinations of <span class="highlight" style="background-color:">signaling</span> chains in CD3 oligomeric structures) might be also responsible for distinct functions such as T-cell proliferation, effector functions, T-cell survival, pathogen clearance, TCR anergy, etc. In addition, the <span class="highlight" style="background-color:">signaling</span> oligomers formed can sequentially interact with the <span class="highlight" style="background-color:">signaling</span> subunits of nonengaged TCRs resulting in formation of higher-order <span class="highlight" style="background-color:">signaling</span> oligomers, thus amplifying and propagating the activation signal (not shown). Also, this leads to the release and subsequent internalization of the remaining nonengaged TCR complexes and/or TCR chains (not shown). Abbreviations: PTK, protein tyrosine kinase. Phosphate groups are shown as filled gray circles.</div></p></div><div class="aux"><div class="resc">Alexander B Sigalov. Cell Adh Migr. 2010 Apr-Jun;4(2):255-267.</div><p class="links"><a class="dblinks" href="/pubmed/20519929" ref="ordinalpos=3713&amp;ncbi_uid=2861846&amp;link_uid=20519929">Citation</a><a class="dblinks" href="/pmc/articles/PMC2900623/" ref="ordinalpos=3713&amp;ncbi_uid=2861846&amp;link_uid=PMC2900623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3714.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2147684/figure/F4/" ref="ordinalpos=3714&amp;ncbi_uid=1215828&amp;link_uid=PMC2147684" href="/pmc/articles/PMC2147684/"><img src="/pmc/articles/instance/2147684/bin/nihms-35362-f0004.gif" src-large="/pmc/articles/instance/2147684/bin/nihms-35362-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2147684/" ref="ordinalpos=3714&amp;ncbi_uid=1215828&amp;link_uid=PMC2147684" image-link="/pmc/articles/PMC2147684/figure/F4/" class="imagepopup">Figure 4. Summary of IGF-I activated <span class="highlight" style="background-color:">signaling</span> pathways in pancreatic cancer cells and their impact on COX-2 expression.  From: Regulation of cyclooxygenase-2 (COX-2) expression in human pancreatic carcinoma cells by the insulin-like growth factor-I receptor (IGF-IR) system. </a></p><div class="supp"><p class="details"><div>IGF-I leads to induction of multiple <span class="highlight" style="background-color:">signaling</span> intermediates down-stream of IGF-IR and IRS-1 (  ). For investigating <span class="highlight" style="background-color:">signaling</span> pathways involved in IGF-I mediated COX-2 expression, inhibitors to MAPK/Erk1/2, PI-3K/Akt, P38, or SAPK were studied. The results show that MAPK/Erk is the predominant <span class="highlight" style="background-color:">pathway</span> for mediating IGF-I-induced COX-2 expression in pancreatic cancer cells (bold arrow). HIF-1 is downstream of the IGF-I/IGF-IR <span class="highlight" style="background-color:">signaling</span> cascade, as previously reported (dotted arrow) [7]. However, selective inhibition of HIF-1 using RNAi, decreased constitutive COX-2 expression, but had no effect on inducible COX-2 up-regulation suggesting that other transcription factors are involved in IGF-IR medicated Cox-2 induction. In contrast, PI-3K and SAPK appear to function as negative regulatory pathways for COX-2 expression since selective inhibition leads to increases in constitutive COX-2. The P38 <span class="highlight" style="background-color:">pathway</span> is activated upon stimulation with IGF-I, but inhibition of this protein had no effect of COX-2 expression in pancreatic cancer cells.</div></p></div><div class="aux"><div class="resc">Oliver Stoeltzing, et al. Cancer Lett. ;258(2):291-300.</div><p class="links"><a class="dblinks" href="/pubmed/17950526" ref="ordinalpos=3714&amp;ncbi_uid=1215828&amp;link_uid=17950526">Citation</a><a class="dblinks" href="/pmc/articles/PMC2147684/" ref="ordinalpos=3714&amp;ncbi_uid=1215828&amp;link_uid=PMC2147684">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3715.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4219656/figure/pbio-1001985-g005/" ref="ordinalpos=3715&amp;ncbi_uid=6824272&amp;link_uid=PMC4219656" href="/pmc/articles/PMC4219656/"><img src="/pmc/articles/instance/4219656/bin/pbio.1001985.g005.gif" src-large="/pmc/articles/instance/4219656/bin/pbio.1001985.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4219656/" ref="ordinalpos=3715&amp;ncbi_uid=6824272&amp;link_uid=PMC4219656" image-link="/pmc/articles/PMC4219656/figure/pbio-1001985-g005/" class="imagepopup">Figure 5. Injury-induced activation of Stat92E is downstream of the Draper <span class="highlight" style="background-color:">signaling</span> cascade..  From: PI3K <span class="highlight" style="background-color:">Signaling</span> and Stat92E Converge to Modulate Glial Responsiveness to Axonal Injury. </a></p><div class="supp"><p class="details"><div>(AC) Single slice confocal images; (A) Glial-specific knockdown of components of the Draper <span class="highlight" style="background-color:">pathway</span> in a <i>10XStat92E-dGFP</i> reporter background one day after antennal ORN axotomy; control (<i>10XStat92E-dGFP/+</i>) <i>draper</i><sup><i>5</i><i>5</i></sup> (<i>10XStat92E-dGFP/+, draper</i><sup><i>5</i></sup><i>/draper</i><sup><i>5</i></sup>), <i>shark<sup>RNAi</sup></i> (<i>shark<sup>RNAi</sup></i>/<i>10XStat92E-dGFP; repo-Gal4</i>/+), <i>rac1<sup>N17</sup></i>(<i>10XStat92E-dGFP/+; repo-Gal4</i>/<i>rac1<sup>N17</sup></i>), <i>src42a<sup>RNAi</sup></i> (<i>src42a<sup>RNAi</sup></i>/<i>10XStat92E-dGFP; repo-Gal4</i>/+), and <i>dCed6<sup>RNAi</sup></i> (<i>dCed6<sup>RNAi</sup></i>/<i>10XStat92E-dGFP; repo-Gal4</i>/+) genotypes. (B) <i>dee7-Gal4</i> activity in control (<i>dee7-Gal4,UAS-mCD8::GFP</i>/+) and <i>draper</i><sup><i>5/</i><i>5</i></sup> (<i>dee7-Gal4,UAS-mCD8::GFP</i>/+, <i>draper</i><sup><i>5</i></sup><i>/draper</i><sup><i>5</i></sup>) backgrounds. Uninjured, one day after antennal injury, and one day after maxillary palp injury are shown. Dotted circles indicate antennal lobes. (C) Glial-specific activation of components of the <i>draper</i> <span class="highlight" style="background-color:">pathway</span> in a <i>10XStat92E-dGFP</i> reporter background; <i>rac1<sup>V12</sup></i> (<i>10XStat92E-dGFP/+; repo-Gal4</i>/<i>rac1<sup>v12</sup></i>), and <i>src42a<sup>ca</sup></i> (<i>10XStat92E-dGFP/+; repo-Gal4</i>/<i>src42a<sup>ca</sup></i>).</div></p></div><div class="aux"><div class="resc">Johnna Doherty, et al. PLoS Biol. 2014 November;12(11):e1001985.</div><p class="links"><a class="dblinks" href="/pubmed/25369313" ref="ordinalpos=3715&amp;ncbi_uid=6824272&amp;link_uid=25369313">Citation</a><a class="dblinks" href="/pmc/articles/PMC4219656/" ref="ordinalpos=3715&amp;ncbi_uid=6824272&amp;link_uid=PMC4219656">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3716.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3694713/figure/fig10/" ref="ordinalpos=3716&amp;ncbi_uid=6078544&amp;link_uid=PMC3694713" href="/pmc/articles/PMC3694713/"><img src="/pmc/articles/instance/3694713/bin/pc_112102_f10.gif" src-large="/pmc/articles/instance/3694713/bin/pc_112102_f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3694713/" ref="ordinalpos=3716&amp;ncbi_uid=6078544&amp;link_uid=PMC3694713" image-link="/pmc/articles/PMC3694713/figure/fig10/" class="imagepopup">Figure 10. From: Deletion of a Tandem Gene Family in Arabidopsis: Increased MEKK2 Abundance Triggers Autoimmunity when the MEKK1-MKK1/2-MPK4 <span class="highlight" style="background-color:">Signaling</span> Cascade Is Disrupted[C][W]. </a></p><div class="supp"><p class="details"><div>Model for the Role of MEKK2 Regulation in Autoimmunity.<br />MPK4 acts downstream of MEKK1 and MKK1/2 as a negative regulator of MEKK2 RNA abundance. Activated MPK4 is required for the repression of MEKK2 expression. In a wild-type plant, MEKK2 is expressed at a basal level. When the MEKK1-MKK1/2-MPK4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is disrupted, MEKK2 abundance increases, which in turn triggers the autoimmune response.</div></p></div><div class="aux"><div class="resc">Shih-Heng Su, et al. Plant Cell. 2013 May;25(5):1895-1910.</div><p class="links"><a class="dblinks" href="/pubmed/23695980" ref="ordinalpos=3716&amp;ncbi_uid=6078544&amp;link_uid=23695980">Citation</a><a class="dblinks" href="/pmc/articles/PMC3694713/" ref="ordinalpos=3716&amp;ncbi_uid=6078544&amp;link_uid=PMC3694713">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3717.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2890293/figure/F1/" ref="ordinalpos=3717&amp;ncbi_uid=2345346&amp;link_uid=PMC2890293" href="/pmc/articles/PMC2890293/"><img src="/pmc/articles/instance/2890293/bin/nihms-213082-f0001.gif" src-large="/pmc/articles/instance/2890293/bin/nihms-213082-f0001.jpg" alt="Figure I" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2890293/" ref="ordinalpos=3717&amp;ncbi_uid=2345346&amp;link_uid=PMC2890293" image-link="/pmc/articles/PMC2890293/figure/F1/" class="imagepopup">Figure I.  From: Bacterial chemoreceptors: high-performance <span class="highlight" style="background-color:">signaling</span> in networked arrays. </a></p><div class="supp"><p class="details"><div>The chemoreceptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>E. coli</i>. Components and reactions in red promote counter clockwise (CCW) flagellar rotation; those in green promote clockwise (CW) flagellar rotation. Components in gray represent inactive forms. Solid lines represent enzymatic reactions; broken lines indicate binding interactions. CheA-derived phosphoryl groups are shown as blue spheres. Receptor modification sites are shown as white (unmethylated) and black (methylated) circles.</div></p></div><div class="aux"><div class="resc">Gerald L. Hazelbauer, et al. Trends Biochem Sci. ;33(1):9-19.</div><p class="links"><a class="dblinks" href="/pubmed/18165013" ref="ordinalpos=3717&amp;ncbi_uid=2345346&amp;link_uid=18165013">Citation</a><a class="dblinks" href="/pmc/articles/PMC2890293/" ref="ordinalpos=3717&amp;ncbi_uid=2345346&amp;link_uid=PMC2890293">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3718.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1964795/figure/F5/" ref="ordinalpos=3718&amp;ncbi_uid=1104750&amp;link_uid=PMC1964795" href="/pmc/articles/PMC1964795/"><img src="/pmc/articles/instance/1964795/bin/nihms26012f5.gif" src-large="/pmc/articles/instance/1964795/bin/nihms26012f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1964795/" ref="ordinalpos=3718&amp;ncbi_uid=1104750&amp;link_uid=PMC1964795" image-link="/pmc/articles/PMC1964795/figure/F5/" class="imagepopup">Figure 5. Anti-apoptotic mechanisms of Akt.  From: INHIBITION OF MAPK AND STIMULATION OF AKT KINASE <span class="highlight" style="background-color:">SIGNALING</span> PATHWAYS: TWO APPROACHES WITH THERAPEUTIC POTENTIAL IN THE TREATMENT OF NEURODEGENERATIVE DISEASE. </a></p><div class="supp"><p class="details"><div>Akt has been described to inhibit apoptosis at multiple levels: <span class="highlight" style="background-color:">signaling</span> pathways, transcription factors, and, in the intrinsic <span class="highlight" style="background-color:">pathway</span> of PCD, at pre-mitochondrial, mitochondrial, and post-mitochondrial levels. Detailed mechanisms for some of these pathways are presented in the text. For more complete reviews, see (Datta et al., 1999; Brunet et al., 2001; Downward, 2004).</div></p></div><div class="aux"><div class="resc">Robert E. Burke. Pharmacol Ther. ;114(3):261-277.</div><p class="links"><a class="dblinks" href="/pubmed/17399794" ref="ordinalpos=3718&amp;ncbi_uid=1104750&amp;link_uid=17399794">Citation</a><a class="dblinks" href="/pmc/articles/PMC1964795/" ref="ordinalpos=3718&amp;ncbi_uid=1104750&amp;link_uid=PMC1964795">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3719.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4359317/figure/F4/" ref="ordinalpos=3719&amp;ncbi_uid=7328691&amp;link_uid=PMC4359317" href="/pmc/articles/PMC4359317/"><img src="/pmc/articles/instance/4359317/bin/oncotarget-06-1594-g004.gif" src-large="/pmc/articles/instance/4359317/bin/oncotarget-06-1594-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4359317/" ref="ordinalpos=3719&amp;ncbi_uid=7328691&amp;link_uid=PMC4359317" image-link="/pmc/articles/PMC4359317/figure/F4/" class="imagepopup">Figure 4. Predicted gene targets in the GATA3 <span class="highlight" style="background-color:">pathway</span> display correlation with hsa-miR-532 in the ER- group and not in the ER+ group.  From: MicroRNA regulation of molecular pathways as a generic mechanism and as a core disease phenotype. </a></p><div class="supp"><p class="details"><div>Six genes in the Gata3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were found to have a possible binding site with hsa-miR-532. The graphs presented here show the correlation between miR-532 and the six genes in the two groups tested (ER- and ER+) in the TCGA dataset. As can be seen here, a significant correlation was found in the ER- patients while a no correlation was observed in the ER+ group.</div></p></div><div class="aux"><div class="resc">Rotem Ben-Hamo, et al. Oncotarget. 2015 January;6(3):1594-1604.</div><p class="links"><a class="dblinks" href="/pubmed/25593195" ref="ordinalpos=3719&amp;ncbi_uid=7328691&amp;link_uid=25593195">Citation</a><a class="dblinks" href="/pmc/articles/PMC4359317/" ref="ordinalpos=3719&amp;ncbi_uid=7328691&amp;link_uid=PMC4359317">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3720.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2555978/figure/F6/" ref="ordinalpos=3720&amp;ncbi_uid=1522404&amp;link_uid=PMC2555978" href="/pmc/articles/PMC2555978/"><img src="/pmc/articles/instance/2555978/bin/ukmss-2187-f0006.gif" src-large="/pmc/articles/instance/2555978/bin/ukmss-2187-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2555978/" ref="ordinalpos=3720&amp;ncbi_uid=1522404&amp;link_uid=PMC2555978" image-link="/pmc/articles/PMC2555978/figure/F6/" class="imagepopup">Figure 6. A model to explain the phenotypes obtained when FGF <span class="highlight" style="background-color:">signaling</span> is progressively reduced in mes/r1.  From: Specific regions within the embryonic midbrain and cerebellum require different levels of FGF <span class="highlight" style="background-color:">signaling</span> during development. </a></p><div class="supp"><p class="details"><div>Schematic diagrams illustrate dorsal views of mes/r1 in embryos of the genotypes indicated at the somite stages denoted. The intensity of the blue color in the triangle to the left of a diagram illustrates the level of FGF <span class="highlight" style="background-color:">signaling</span>. (<b>A</b>) The regions in which cells are dying or have already died are indicated by red stippling. (<b>B</b>) The regions that contain the progenitors (p) of the superior colliculus (SC), inferior colliculus (IC), cerebellar vermis (CbV) and cerebellar hemispheres (CbH) are indicated. The phenotypes observed in mutants that attain progressively lower levels of FGF <span class="highlight" style="background-color:">signaling</span> are illustrated. In mes/r1-S2GOF embryos, the anterior mes has been lost, and the surviving posterior cells are specified to an anterior (SC) fate. Consequently the IC does not develop. In mes/r1-S2GOF;F8 embryos, the effects on the mes are similar, and in addition, the vermis fails to develop because roof plate (RP) expansion in r1 results in a loss of CbV progenitors. (<b>C</b>) In the wild-type embryo, FGF <span class="highlight" style="background-color:">signaling</span> induces/maintains the expression of <i>Grem1</i>, which functions to inhibit BMP <span class="highlight" style="background-color:">signaling</span> and expression of the BMP effector MSX1. This <span class="highlight" style="background-color:">pathway</span> maintains the normal balance of vermis and roof plate development. In the mes/r1-S2GOF;F8 embryo, the reduction in the level of FGF <span class="highlight" style="background-color:">signaling</span> results in a severe reduction in <i>Grem1</i> expression. In the absence of this antagonist, the level of BMP <span class="highlight" style="background-color:">signaling</span> and therefore MSX1 expression increases and an expanded roof plate develops where the vermis progenitors normal reside.</div></p></div><div class="aux"><div class="resc">M. Albert Basson, et al. Development. ;135(5):889-898.</div><p class="links"><a class="dblinks" href="/pubmed/18216176" ref="ordinalpos=3720&amp;ncbi_uid=1522404&amp;link_uid=18216176">Citation</a><a class="dblinks" href="/pmc/articles/PMC2555978/" ref="ordinalpos=3720&amp;ncbi_uid=1522404&amp;link_uid=PMC2555978">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3721.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2174972/figure/pbio-0060008-g005/" ref="ordinalpos=3721&amp;ncbi_uid=1151550&amp;link_uid=PMC2174972" href="/pmc/articles/PMC2174972/"><img src="/pmc/articles/instance/2214806/bin/pbio.0060008.g005.gif" src-large="/pmc/articles/instance/2214806/bin/pbio.0060008.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2174972/" ref="ordinalpos=3721&amp;ncbi_uid=1151550&amp;link_uid=PMC2174972" image-link="/pmc/articles/PMC2174972/figure/pbio-0060008-g005/" class="imagepopup">Figure 5. LMP1s TRADD-Binding Domain (LTB) Induces LMP1-Type <span class="highlight" style="background-color:">Signaling</span> but Does Not Activate Apoptosis in the Context of the TNFR1 <span class="highlight" style="background-color:">Signaling</span> Domain.  From: The Viral Oncoprotein LMP1 Exploits TRADD for <span class="highlight" style="background-color:">Signaling</span> by Masking Its Apoptotic Activity. </a></p><div class="supp"><p class="details"><div>(A) The LTB is functional in the context of the TNFR1 <span class="highlight" style="background-color:">signaling</span> domain. NF-B reporter assays in HEK293 cells. Data are mean values of three independent experiments  standard deviation.<br />(B) The transferred LTB determines TRAF6-dependent NF-B activation. NF-B reporter assays in TRAF6/ MEFs are shown. Where indicated (black bars), TRAF6 was co-transfected. Data are mean values of four independent experiments  standard deviation; statistics: two-tailed Student's <i>t</i>-test.<br />(C) Lack of apoptosis induction by LTB in the context of the TNFR1 <span class="highlight" style="background-color:">signaling</span> domain. Transient cell death assays in human BJAB B lymphocytes are shown. The NF-B <span class="highlight" style="background-color:">pathway</span> was blocked by co-expression of domaint-negative I-B(S32/36A). Where indicated, the cells were incubated in the presence of 2 M zVAD-fmk (zVAD), a pan-caspase inhibitor. PI, propidium iodide. Data are mean values of three independent experiments  standard deviation.<br />(D) Apoptosis induction by LMP1-TNFR1 is dependent on TRADD. Cell death assays in DG75 <i>TRADD</i>+/+ and DG75 <i>TRADD</i>/ cells are shown. Data are mean values of three independent experiments  standard deviation; statistics: two-tailed Student's <i>t</i>-Test.</div></p></div><div class="aux"><div class="resc">Frank Schneider, et al. PLoS Biol. 2008 January;6(1):e8.</div><p class="links"><a class="dblinks" href="/pubmed/18198944" ref="ordinalpos=3721&amp;ncbi_uid=1151550&amp;link_uid=18198944">Citation</a><a class="dblinks" href="/pmc/articles/PMC2174972/" ref="ordinalpos=3721&amp;ncbi_uid=1151550&amp;link_uid=PMC2174972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3722.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3009859/figure/F5/" ref="ordinalpos=3722&amp;ncbi_uid=3607245&amp;link_uid=PMC3009859" href="/pmc/articles/PMC3009859/"><img src="/pmc/articles/instance/3009859/bin/zbc0061145640005.gif" src-large="/pmc/articles/instance/3009859/bin/zbc0061145640005.jpg" alt="FIGURE 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3009859/" ref="ordinalpos=3722&amp;ncbi_uid=3607245&amp;link_uid=PMC3009859" image-link="/pmc/articles/PMC3009859/figure/F5/" class="imagepopup">FIGURE 5. From: Phosphatidylinositol 3-Kinase (PI3K) <span class="highlight" style="background-color:">Signaling</span> via Glycogen Synthase Kinase-3 (Gsk-3) Regulates DNA Methylation of Imprinted Loci. </a></p><div class="supp"><p class="details"><div><b>Activation of components of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reduces DNA methylation.</b> <i>A</i>, Western blots showing that WT ESCs stably transfected with p110* constitutively activate insulin <span class="highlight" style="background-color:">signaling</span>. <i>p-Akt</i>, phospho-Akt; <i>p-Gsk-3</i>, phospho-glycogen synthase kinase-3. <i>B</i>, bisulfite sequencing analysis of the <i>Igf2/H19</i> DMD in p110* ESCs when compared with WT ESCs (<i>p</i> &lt; 0.01, two-tailed <i>t</i> test). <i>C</i>, bisulfite sequencing analysis of the <i>Igf2r</i> DMR2 in p110* ESCs when compared with WT ESCs. <i>Open circles</i> represent unmethylated CpG dinucleotides, whereas <i>filled circles</i> represent methylated CpG dinucleotides.</div></p></div><div class="aux"><div class="resc">Anthony P. Popkie, et al. J Biol Chem. 2010 December 31;285(53):41337-41347.</div><p class="links"><a class="dblinks" href="/pubmed/21047779" ref="ordinalpos=3722&amp;ncbi_uid=3607245&amp;link_uid=21047779">Citation</a><a class="dblinks" href="/pmc/articles/PMC3009859/" ref="ordinalpos=3722&amp;ncbi_uid=3607245&amp;link_uid=PMC3009859">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3723.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2721821/figure/F6/" ref="ordinalpos=3723&amp;ncbi_uid=1966531&amp;link_uid=PMC2721821" href="/pmc/articles/PMC2721821/"><img src="/pmc/articles/instance/2729032/bin/zpq9990991120006.gif" src-large="/pmc/articles/instance/2729032/bin/zpq9990991120006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2721821/" ref="ordinalpos=3723&amp;ncbi_uid=1966531&amp;link_uid=PMC2721821" image-link="/pmc/articles/PMC2721821/figure/F6/" class="imagepopup">Fig. 6. From: Small-molecule inhibitors reveal multiple strategies for Hedgehog <span class="highlight" style="background-color:">pathway</span> blockade. </a></p><div class="supp"><p class="details"><div>Graphical representation of the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in its activated state and possible sites of HPI action. HPI-1 inhibits both endogenous (solid arrows) and exogenous (dashed arrow) Gli1/Gli2 activity, suggesting that it acts independently of the primary cilium. HPI-2 and HPI-3 appear to block the conversion of full-length Gli2 proteins into transcriptional activators, and HPI-4 disrupts ciliogenesis and therefore ciliary processes required for Gli function.</div></p></div><div class="aux"><div class="resc">Joel M. Hyman, et al. Proc Natl Acad Sci U S A. 2009 August 18;106(33):14132-14137.</div><p class="links"><a class="dblinks" href="/pubmed/19666565" ref="ordinalpos=3723&amp;ncbi_uid=1966531&amp;link_uid=19666565">Citation</a><a class="dblinks" href="/pmc/articles/PMC2721821/" ref="ordinalpos=3723&amp;ncbi_uid=1966531&amp;link_uid=PMC2721821">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3724.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3569742/figure/F6/" ref="ordinalpos=3724&amp;ncbi_uid=5794680&amp;link_uid=PMC3569742" href="/pmc/articles/PMC3569742/"><img src="/pmc/articles/instance/3569742/bin/nihms-435607-f0006.gif" src-large="/pmc/articles/instance/3569742/bin/nihms-435607-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3569742/" ref="ordinalpos=3724&amp;ncbi_uid=5794680&amp;link_uid=PMC3569742" image-link="/pmc/articles/PMC3569742/figure/F6/" class="imagepopup">Figure 6. Schematic model depicting the role of Tmem64-mediated Ca2+ oscillation in production of mitochondrial ROS during OC differentiation.  From: Tmem64 modulates calcium <span class="highlight" style="background-color:">signaling</span> during RANKL-mediated osteoclast differentiation. </a></p><div class="supp"><p class="details"><div>Shown are mechanisms that involve the RANKL-stimulated Ca<sup>2+</sup> signal <span class="highlight" style="background-color:">pathway</span>. Association between Tmem64 and SERCA2 in the ER leads to cytosolic Ca<sup>2+</sup> spiking for activation of NFATc1 and production of mitochondrial ROS, thereby triggering a Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> cascades that promote osteoclast differentiation and activation.</div></p></div><div class="aux"><div class="resc">Hyunsoo Kim, et al. Cell Metab. ;17(2):249-260.</div><p class="links"><a class="dblinks" href="/pubmed/23395171" ref="ordinalpos=3724&amp;ncbi_uid=5794680&amp;link_uid=23395171">Citation</a><a class="dblinks" href="/pmc/articles/PMC3569742/" ref="ordinalpos=3724&amp;ncbi_uid=5794680&amp;link_uid=PMC3569742">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3725.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC149122/figure/fig7/" ref="ordinalpos=3725&amp;ncbi_uid=211448&amp;link_uid=PMC149122" href="/pmc/articles/PMC149122/"><img src="/pmc/articles/instance/149122/bin/00-0152f7.gif" src-large="/pmc/articles/instance/149122/bin/00-0152f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC149122/" ref="ordinalpos=3725&amp;ncbi_uid=211448&amp;link_uid=PMC149122" image-link="/pmc/articles/PMC149122/figure/fig7/" class="imagepopup">Figure 7. From: Fumonisin B1-Induced Cell Death in Arabidopsis Protoplasts Requires Jasmonate-, Ethylene-, and Salicylate-Dependent <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>A Model for Cell Death <span class="highlight" style="background-color:">Signaling</span> in Arabidopsis Protoplasts.<br />Weak and strong SA signals are denoted by smaller and larger letters, respectively. As alternatives to the light-dependent PAL-mediated generation of SA shown in the model, SA might be produced from SA -glucoside through a <span class="highlight" style="background-color:">pathway</span> independent of PAL and light, and the primary role of light in PCD may be to generate ROS rather than to activate the transcription of <i>PAL</i> genes. See text for details.</div></p></div><div class="aux"><div class="resc">Tsuneaki Asai, et al. Plant Cell. 2000 October;12(10):1823-1836.</div><p class="links"><a class="dblinks" href="/pubmed/11041879" ref="ordinalpos=3725&amp;ncbi_uid=211448&amp;link_uid=11041879">Citation</a><a class="dblinks" href="/pmc/articles/PMC149122/" ref="ordinalpos=3725&amp;ncbi_uid=211448&amp;link_uid=PMC149122">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3726.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3655322/figure/F3/" ref="ordinalpos=3726&amp;ncbi_uid=4934434&amp;link_uid=PMC3655322" href="/pmc/articles/PMC3655322/"><img src="/pmc/articles/instance/3655322/bin/fonc-03-00123-g003.gif" src-large="/pmc/articles/instance/3655322/bin/fonc-03-00123-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3655322/" ref="ordinalpos=3726&amp;ncbi_uid=4934434&amp;link_uid=PMC3655322" image-link="/pmc/articles/PMC3655322/figure/F3/" class="imagepopup">Figure 3.  From: The Role of the Hippo <span class="highlight" style="background-color:">Pathway</span> in Melanocytes and Melanoma. </a></p><div class="supp"><p class="details"><div><b>Diagram showing some of the upstream elements of the proposed Hippo <span class="highlight" style="background-color:">pathway</span></b>. Melanocytes are strongly polarized and receive signals on the one hand from the epidermis through adherens junctions with keratinocytes, and on the other hand from the basement membrane through integrins. They also receive signals through receptor tyrosine kinases such as Axl and through G-protein coupled receptors (GPCRs). All are likely to signal through kinases to Lats1/2; arrows indicate <span class="highlight" style="background-color:">signaling</span> events.</div></p></div><div class="aux"><div class="resc">Ji Eun Kim, et al. Front Oncol. 2013;3:123.</div><p class="links"><a class="dblinks" href="/pubmed/23720711" ref="ordinalpos=3726&amp;ncbi_uid=4934434&amp;link_uid=23720711">Citation</a><a class="dblinks" href="/pmc/articles/PMC3655322/" ref="ordinalpos=3726&amp;ncbi_uid=4934434&amp;link_uid=PMC3655322">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3727.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1531665/figure/F4/" ref="ordinalpos=3727&amp;ncbi_uid=742841&amp;link_uid=PMC1531665" href="/pmc/articles/PMC1531665/"><img src="/pmc/articles/instance/1531665/bin/neo0605_0603_fig004.gif" src-large="/pmc/articles/instance/1531665/bin/neo0605_0603_fig004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1531665/" ref="ordinalpos=3727&amp;ncbi_uid=742841&amp;link_uid=PMC1531665" image-link="/pmc/articles/PMC1531665/figure/F4/" class="imagepopup">Figure 4.  From: Activation of the Erk <span class="highlight" style="background-color:">Pathway</span> Is Required for TGF-?1-Induced EMT In Vitro. </a></p><div class="supp"><p class="details"><div>Model of TGF-1-induced Erk activation and EMT being blocked by U0126. TGF-1 may activate the Erk <span class="highlight" style="background-color:">pathway</span> through upstream factors, such as Ras/Raf, or directly through more downstream factors, such as MEK or Erk. To investigate the role of Erk <span class="highlight" style="background-color:">signaling</span> in TGF-1-mediated EMT, the MEK1/2 inhibitor, U0126, was used to block Erk phosphorylation in NMuMG and MCT cells. In these cells, TGF-1 induction of Erk activity and EMT was blocked by U0126. However, TGF-1 may activate other pathways, such as p38, JNK, PI3-kinase, and RhoA, to induce EMT in other cell types.</div></p></div><div class="aux"><div class="resc">Lu Xie, et al. Neoplasia. 2004 September;6(5):603-610.</div><p class="links"><a class="dblinks" href="/pubmed/15548370" ref="ordinalpos=3727&amp;ncbi_uid=742841&amp;link_uid=15548370">Citation</a><a class="dblinks" href="/pmc/articles/PMC1531665/" ref="ordinalpos=3727&amp;ncbi_uid=742841&amp;link_uid=PMC1531665">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3728.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3464266/figure/pone-0046887-g002/" ref="ordinalpos=3728&amp;ncbi_uid=4283688&amp;link_uid=PMC3464266" href="/pmc/articles/PMC3464266/"><img src="/pmc/articles/instance/3464266/bin/pone.0046887.g002.gif" src-large="/pmc/articles/instance/3464266/bin/pone.0046887.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3464266/" ref="ordinalpos=3728&amp;ncbi_uid=4283688&amp;link_uid=PMC3464266" image-link="/pmc/articles/PMC3464266/figure/pone-0046887-g002/" class="imagepopup">Figure 2. Genes significantly associated with pancreatic cancer in (A) the Fc epsilon RI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (B) long-term depression; (C) MODY; (D) olfactory transduction; (E) neuroactive ligand-receptor interaction <span class="highlight" style="background-color:">pathway</span>; and (F) vascular smooth muscle contraction pathways..  From: Insights into Pancreatic Cancer Etiology from <span class="highlight" style="background-color:">Pathway</span> Analysis of Genome-Wide Association Study Data. </a></p><div class="supp"><p class="details"><div>Genes with log10(<i>P</i>) &lt;1 were not included in the plots. For clarity, not all genes are labeled. For details, see Tables S1 and S3.</div></p></div><div class="aux"><div class="resc">Peng Wei, et al. PLoS One. 2012;7(10):e46887.</div><p class="links"><a class="dblinks" href="/pubmed/23056513" ref="ordinalpos=3728&amp;ncbi_uid=4283688&amp;link_uid=23056513">Citation</a><a class="dblinks" href="/pmc/articles/PMC3464266/" ref="ordinalpos=3728&amp;ncbi_uid=4283688&amp;link_uid=PMC3464266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3729.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3917548/figure/f1-br-01-06-0837/" ref="ordinalpos=3729&amp;ncbi_uid=5927748&amp;link_uid=PMC3917548" href="/pmc/articles/PMC3917548/"><img src="/pmc/articles/instance/3917548/bin/BR-01-06-0837-g00.gif" src-large="/pmc/articles/instance/3917548/bin/BR-01-06-0837-g00.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3917548/" ref="ordinalpos=3729&amp;ncbi_uid=5927748&amp;link_uid=PMC3917548" image-link="/pmc/articles/PMC3917548/figure/f1-br-01-06-0837/" class="imagepopup">Figure 1.  From: The emerging role of interleukin-21 in allergic diseases (Review). </a></p><div class="supp"><p class="details"><div>Schematic representation of the interleukin (IL)-21 receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IL-21R/<sub>c</sub>-driven <span class="highlight" style="background-color:">signaling</span> may activate three pathways, namely the JAK/STAT, PI3K/Akt and MAPK pathways. JAK, Janus-activated kinase; STAT, signal transducer and activator of transcription; PI3K, phosphatidylinositol 3-kinase; MAPK, mitogen-activated protein kinase.</div></p></div><div class="aux"><div class="resc">FANG GONG, et al. Biomed Rep. 2013 November;1(6):837-839.</div><p class="links"><a class="dblinks" href="/pubmed/24649038" ref="ordinalpos=3729&amp;ncbi_uid=5927748&amp;link_uid=24649038">Citation</a><a class="dblinks" href="/pmc/articles/PMC3917548/" ref="ordinalpos=3729&amp;ncbi_uid=5927748&amp;link_uid=PMC3917548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3730.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2730119/figure/F3/" ref="ordinalpos=3730&amp;ncbi_uid=1980787&amp;link_uid=PMC2730119" href="/pmc/articles/PMC2730119/"><img src="/pmc/articles/instance/2730119/bin/nihms122211f3.gif" src-large="/pmc/articles/instance/2730119/bin/nihms122211f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2730119/" ref="ordinalpos=3730&amp;ncbi_uid=1980787&amp;link_uid=PMC2730119" image-link="/pmc/articles/PMC2730119/figure/F3/" class="imagepopup">Fig. 3.  From: Regulation of potassium (K) handling in the renal collecting duct. </a></p><div class="supp"><p class="details"><div>A scheme showing the role of different <span class="highlight" style="background-color:">signaling</span> pathways in mediating the effect of high K intake on ROMK and BK channels in the CCD. The circle with x indicates the inhibition of a particular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Wen-Hui Wang, et al. Pflugers Arch. ;458(1):157-168.</div><p class="links"><a class="dblinks" href="/pubmed/18839206" ref="ordinalpos=3730&amp;ncbi_uid=1980787&amp;link_uid=18839206">Citation</a><a class="dblinks" href="/pmc/articles/PMC2730119/" ref="ordinalpos=3730&amp;ncbi_uid=1980787&amp;link_uid=PMC2730119">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3731.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3506267/figure/F3/" ref="ordinalpos=3731&amp;ncbi_uid=4401106&amp;link_uid=PMC3506267" href="/pmc/articles/PMC3506267/"><img src="/pmc/articles/instance/3506267/bin/2050-6511-13-3-3.gif" src-large="/pmc/articles/instance/3506267/bin/2050-6511-13-3-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3506267/" ref="ordinalpos=3731&amp;ncbi_uid=4401106&amp;link_uid=PMC3506267" image-link="/pmc/articles/PMC3506267/figure/F3/" class="imagepopup">Figure 3.  From: The potential for selective pharmacological therapies through biased receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Impact of agonist-directed stimulus on drug screening.</b> Canonical strategies for high throughput (HTS) screening pass the most active molecules from the HTS (represented by the dextral tail of the Boltzman distribution representing the best responses in the HTS) into secondary assays and animal models for further testing. To detect bias in <span class="highlight" style="background-color:">signaling</span>, the active molecules from the HTS are all tested in another functional assay for another <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the most active molecules from that assay pooled with the actives from the HTS for further testing. The most active molecules in the secondary biased assay often are not the most active in the HTS thus a spectrum of agonists of differing stimulus bias is tested in animal models.</div></p></div><div class="aux"><div class="resc">Terry Kenakin. BMC Pharmacol Toxicol. 2012;13:3-3.</div><p class="links"><a class="dblinks" href="/pubmed/22947056" ref="ordinalpos=3731&amp;ncbi_uid=4401106&amp;link_uid=22947056">Citation</a><a class="dblinks" href="/pmc/articles/PMC3506267/" ref="ordinalpos=3731&amp;ncbi_uid=4401106&amp;link_uid=PMC3506267">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3732.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3019858/figure/f1/" ref="ordinalpos=3732&amp;ncbi_uid=3063512&amp;link_uid=PMC3019858" href="/pmc/articles/PMC3019858/"><img src="/pmc/articles/instance/3019858/bin/zmb9991088860001.gif" src-large="/pmc/articles/instance/3019858/bin/zmb9991088860001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3019858/" ref="ordinalpos=3732&amp;ncbi_uid=3063512&amp;link_uid=PMC3019858" image-link="/pmc/articles/PMC3019858/figure/f1/" class="imagepopup">FIG. 1. From: Mitogen-Activated Protein Kinases Promote WNT/?-Catenin <span class="highlight" style="background-color:">Signaling</span> via Phosphorylation of LRP6  . </a></p><div class="supp"><p class="details"><div>A whole-kinome siRNA screen identifies MAPKs as regulators of WNT <span class="highlight" style="background-color:">signaling</span>. (A) A whole-kinome siRNA screen using a luminescent readout for WNT activity revealed many potential regulators of WNT <span class="highlight" style="background-color:">signaling</span>. GSK3, a known <span class="highlight" style="background-color:">pathway</span> inhibitor, served as an internal control. (B) A secondary screen using some of the strongest hits from the original library was performed for confirmation. (C) List of components of the MAPK cascade identified in the screen. Shades of gray indicate the association with the specific MAPK cascade (ERK5, JNK1, and p38).</div></p></div><div class="aux"><div class="resc">Igor ervenka, et al. Mol Cell Biol. 2011 January;31(1):179-189.</div><p class="links"><a class="dblinks" href="/pubmed/20974802" ref="ordinalpos=3732&amp;ncbi_uid=3063512&amp;link_uid=20974802">Citation</a><a class="dblinks" href="/pmc/articles/PMC3019858/" ref="ordinalpos=3732&amp;ncbi_uid=3063512&amp;link_uid=PMC3019858">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3733.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2838378/figure/F1/" ref="ordinalpos=3733&amp;ncbi_uid=2737761&amp;link_uid=PMC2838378" href="/pmc/articles/PMC2838378/"><img src="/pmc/articles/instance/2838378/bin/nihms180949f1.gif" src-large="/pmc/articles/instance/2838378/bin/nihms180949f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2838378/" ref="ordinalpos=3733&amp;ncbi_uid=2737761&amp;link_uid=PMC2838378" image-link="/pmc/articles/PMC2838378/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Signaling</span> Pathways Critical for AIlergic Airway Inflammation. </a></p><div class="supp"><p class="details"><div>Simplified schematic of the IgE receptor (FcRI) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. FcRI engagement results in phosphorylation of receptor ITAMs by Lyn and Fyn, thus generating docking sites for theses proteins as well as for Syk. Binding to ITAMs results in the upregulation of Fyn, Lyn and Syk kinase activity and causes phosphorylation of downstream <span class="highlight" style="background-color:">signaling</span> molecules. As a consequence of these events multi-protein complexes containing various <span class="highlight" style="background-color:">signaling</span> molecules are formed that the induce activation of MAPK pathways and the generation of second messenger molecules, leading to the induction of cell activation. Sphks - sphingosine kinases.</div></p></div><div class="aux"><div class="resc">John D. Colgan, et al. Curr Opin Allergy Clin Immunol. ;10(1):42-47.</div><p class="links"><a class="dblinks" href="/pubmed/19996739" ref="ordinalpos=3733&amp;ncbi_uid=2737761&amp;link_uid=19996739">Citation</a><a class="dblinks" href="/pmc/articles/PMC2838378/" ref="ordinalpos=3733&amp;ncbi_uid=2737761&amp;link_uid=PMC2838378">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3734.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4204446/figure/F7/" ref="ordinalpos=3734&amp;ncbi_uid=6744498&amp;link_uid=PMC4204446" href="/pmc/articles/PMC4204446/"><img src="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g007.gif" src-large="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4204446/" ref="ordinalpos=3734&amp;ncbi_uid=6744498&amp;link_uid=PMC4204446" image-link="/pmc/articles/PMC4204446/figure/F7/" class="imagepopup">Figure 7.  From: Updates on Psoriasis and Cutaneous Oncology. </a></p><div class="supp"><p class="details"><div>In the absence of the hedgehog ligand, PTCH inhibits SMO and the hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is suppressed. </div></p></div><div class="aux"><div class="resc">Neal Bhatia, et al. J Clin Aesthet Dermatol. 2014 July;7(7 Suppl SC2S5-S22):S5-S22.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3734&amp;ncbi_uid=6744498&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4204446/" ref="ordinalpos=3734&amp;ncbi_uid=6744498&amp;link_uid=PMC4204446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3735.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3197696/figure/F9/" ref="ordinalpos=3735&amp;ncbi_uid=3472535&amp;link_uid=PMC3197696" href="/pmc/articles/PMC3197696/"><img src="/pmc/articles/instance/3197696/bin/nihms312528f9.gif" src-large="/pmc/articles/instance/3197696/bin/nihms312528f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3197696/" ref="ordinalpos=3735&amp;ncbi_uid=3472535&amp;link_uid=PMC3197696" image-link="/pmc/articles/PMC3197696/figure/F9/" class="imagepopup">Figure 9. Schematic of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> depicting possible sites of action by PD-1/PD-L1.  From: HIV mediated PI3K/Akt activation in antigen presenting cells leads to PD-1 ligand upregulation and suppression of HIV specific CD8 T-cells. </a></p><div class="aux"><div class="resc">Karuppiah Muthumani, et al. J Immunol. ;187(6):2932-2943.</div><p class="links"><a class="dblinks" href="/pubmed/21856939" ref="ordinalpos=3735&amp;ncbi_uid=3472535&amp;link_uid=21856939">Citation</a><a class="dblinks" href="/pmc/articles/PMC3197696/" ref="ordinalpos=3735&amp;ncbi_uid=3472535&amp;link_uid=PMC3197696">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3736.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3565434/figure/F8/" ref="ordinalpos=3736&amp;ncbi_uid=4602940&amp;link_uid=PMC3565434" href="/pmc/articles/PMC3565434/"><img src="/pmc/articles/instance/3565434/bin/nihms-273633-f0008.gif" src-large="/pmc/articles/instance/3565434/bin/nihms-273633-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3565434/" ref="ordinalpos=3736&amp;ncbi_uid=4602940&amp;link_uid=PMC3565434" image-link="/pmc/articles/PMC3565434/figure/F8/" class="imagepopup">Figure 8.  From: Inhibition of the Plasma Membrane Ca2+ Pump by CD44 Receptor Activation of Tyrosine Kinases Increases the Action Potential Afterhyperpolarization in Sensory Neurons. </a></p><div class="supp"><p class="details"><div>Schematic diagram of proposed CD44-regulated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Biswarup Ghosh, et al. J Neurosci. ;31(7):2361-2370.</div><p class="links"><a class="dblinks" href="/pubmed/21325503" ref="ordinalpos=3736&amp;ncbi_uid=4602940&amp;link_uid=21325503">Citation</a><a class="dblinks" href="/pmc/articles/PMC3565434/" ref="ordinalpos=3736&amp;ncbi_uid=4602940&amp;link_uid=PMC3565434">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3737.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3013804/figure/F3/" ref="ordinalpos=3737&amp;ncbi_uid=2693722&amp;link_uid=PMC3013804" href="/pmc/articles/PMC3013804/"><img src="/pmc/articles/instance/3013804/bin/gkq970f3.gif" src-large="/pmc/articles/instance/3013804/bin/gkq970f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3013804/" ref="ordinalpos=3737&amp;ncbi_uid=2693722&amp;link_uid=PMC3013804" image-link="/pmc/articles/PMC3013804/figure/F3/" class="imagepopup">Figure 3. From: RegPhos: a system to explore the protein kinase-substrate phosphorylation network in humans. </a></p><div class="supp"><p class="details"><div>Case study of the discovered phosphorylation networks associated with insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tzong-Yi Lee, et al. Nucleic Acids Res. 2011 January;39(Database issue):D777-D787.</div><p class="links"><a class="dblinks" href="/pubmed/21037261" ref="ordinalpos=3737&amp;ncbi_uid=2693722&amp;link_uid=21037261">Citation</a><a class="dblinks" href="/pmc/articles/PMC3013804/" ref="ordinalpos=3737&amp;ncbi_uid=2693722&amp;link_uid=PMC3013804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3738.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3611002/figure/F3/" ref="ordinalpos=3738&amp;ncbi_uid=5954374&amp;link_uid=PMC3611002" href="/pmc/articles/PMC3611002/"><img src="/pmc/articles/instance/3611002/bin/zbc0171344690003.gif" src-large="/pmc/articles/instance/3611002/bin/zbc0171344690003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3611002/" ref="ordinalpos=3738&amp;ncbi_uid=5954374&amp;link_uid=PMC3611002" image-link="/pmc/articles/PMC3611002/figure/F3/" class="imagepopup">FIGURE 3. From: Leucine Facilitates Insulin <span class="highlight" style="background-color:">Signaling</span> through a G?i Protein-dependent <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Hepatocytes. </a></p><div class="supp"><p class="details"><div><b>Leucine facilitates insulin <span class="highlight" style="background-color:">signaling</span> through a <span class="highlight" style="background-color:">pathway</span> independent of mTORC2.</b> Hepa1c1c7 hepatocytes were treated with leucine, insulin, or insulin plus leucine for 30 min, followed by evaluations of phosphorylated mTOR (<i>P-mTOR</i>), total mTOR (<i>T-mTOR</i>), and -actin by immunoblotting and then quantified. <i>B</i>, Hepa1c1c7 cells were transfected with siRNAs against Rictor or scrambled siRNA for 36 h and were then treated with leucine or insulin alone or insulin plus leucine for 30 min, followed by evaluations of phosphorylated Akt (at serine 473), total Akt, total Rictor, and -actin by using immunoblotting with specific antibodies. <i>C</i>, Hepa1c1c7 cell were transfected with siRNAs against Raptor or scrambled (<i>Scr</i>) siRNA for 36 h and were then treated with leucine (0.6 m<span style="font-variant: small-caps" class="small-caps">m</span>) or insulin (10 n<span style="font-variant: small-caps" class="small-caps">m</span>) alone or insulin plus leucine for 30 min, followed by evaluations of phosphorylated Akt (at serine 473), total Akt, total Raptor, and -actin by using immunoblotting. Results were presented as the mean  S.E. of four independent experiments, each in duplicate. <i>Bars</i> not sharing a letter are statistically different (<i>p</i> &lt; 0.05).</div></p></div><div class="aux"><div class="resc">Xuefeng Yang, et al. J Biol Chem. 2013 March 29;288(13):9313-9320.</div><p class="links"><a class="dblinks" href="/pubmed/23404499" ref="ordinalpos=3738&amp;ncbi_uid=5954374&amp;link_uid=23404499">Citation</a><a class="dblinks" href="/pmc/articles/PMC3611002/" ref="ordinalpos=3738&amp;ncbi_uid=5954374&amp;link_uid=PMC3611002">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3739.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1189072/figure/pgen-0010019-g006/" ref="ordinalpos=3739&amp;ncbi_uid=593185&amp;link_uid=PMC1189072" href="/pmc/articles/PMC1189072/"><img src="/pmc/articles/instance/1193528/bin/pgen.0010019.g006.gif" src-large="/pmc/articles/instance/1193528/bin/pgen.0010019.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1189072/" ref="ordinalpos=3739&amp;ncbi_uid=593185&amp;link_uid=PMC1189072" image-link="/pmc/articles/PMC1189072/figure/pgen-0010019-g006/" class="imagepopup">Figure 6. Premature Stop Codons Were Identified in the uki and lep Mutant in Hip and Ptc2, Respectively.  From: The Zebrafish Mutants dre, uki, and lep Encode Negative Regulators of the Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> . </a></p><div class="supp"><p class="details"><div>(A) Schematic representation of the genomic organization of the Hip gene. All three alleles of the <i>uki</i> mutation contain premature stop codons positioned in exon 5 and exon 7.<br />(B) Representation of the protein structure of Ptc2 shows that the identified nonsense mutation is positioned after the sixth transmembrane domain, probably resulting in a malfunctioning protein.<br />(C) ISH experiments show that Ptc1 expression is increased in <i>uki</i> and <i>lep</i> (arrow), confirming the aberrant activation of the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Marco J Koudijs, et al. PLoS Genet. 2005 August;1(2):e19.</div><p class="links"><a class="dblinks" href="/pubmed/16121254" ref="ordinalpos=3739&amp;ncbi_uid=593185&amp;link_uid=16121254">Citation</a><a class="dblinks" href="/pmc/articles/PMC1189072/" ref="ordinalpos=3739&amp;ncbi_uid=593185&amp;link_uid=PMC1189072">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3740.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2064281/figure/fig6/" ref="ordinalpos=3740&amp;ncbi_uid=1191664&amp;link_uid=PMC2064281" href="/pmc/articles/PMC2064281/"><img src="/pmc/articles/instance/2064281/bin/jcb1770809f06.gif" src-large="/pmc/articles/instance/2064281/bin/jcb1770809f06.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2064281/" ref="ordinalpos=3740&amp;ncbi_uid=1191664&amp;link_uid=PMC2064281" image-link="/pmc/articles/PMC2064281/figure/fig6/" class="imagepopup">Figure 6. From: Essential role of PI3-kinase and phospholipase A2 in Dictyostelium discoideum chemotaxis. </a></p><div class="supp"><p class="details"><div><b>Model of <span class="highlight" style="background-color:">signaling</span> cascades leading to <i>D. discoideum</i> chemotaxis.</b> cAMP activates multiple pathways. The PI3K and PLA2 <span class="highlight" style="background-color:">pathway</span> are parallel mediators of chemotaxis: either one can mediate chemotaxis, and chemotaxis is blocked nearly completely when both pathways are inhibited. PIP<sub>3</sub> is the likely mediator of the PI3K <span class="highlight" style="background-color:">pathway</span>, by recruiting PH-containing proteins modulating the actin cytoskeleton. The messenger of the PLA2 <span class="highlight" style="background-color:">pathway</span> controlling chemotaxis is unknown. The PI3K <span class="highlight" style="background-color:">pathway</span> appears to be controlled by the PLC <span class="highlight" style="background-color:">pathway</span>, presumably at the level of PIP<sub>2</sub> degradation, leading to a reduction of membrane-associated PTEN that degrades PIP<sub>3</sub>. The PLA2 <span class="highlight" style="background-color:">pathway</span> is dependent on cytosolic Ca<sup>2+</sup>, but it is unknown whether this occurs at the level of PLA2 activation or the action of downstream messengers on the chemotaxis system. Cytosolic Ca<sup>2+</sup> is regulated by Ca<sup>2+</sup> uptake from the medium (which is both G protein dependent and independent), free fatty acid (FFA)mediated Ca<sup>2+</sup> release from acidic stores, and the IP<sub>3</sub> receptormediated Ca<sup>2+</sup> release from the endoplasmic reticulum.</div></p></div><div class="aux"><div class="resc">Peter J.M. van Haastert, et al. J Cell Biol. 2007 June 4;177(5):809-816.</div><p class="links"><a class="dblinks" href="/pubmed/17535967" ref="ordinalpos=3740&amp;ncbi_uid=1191664&amp;link_uid=17535967">Citation</a><a class="dblinks" href="/pmc/articles/PMC2064281/" ref="ordinalpos=3740&amp;ncbi_uid=1191664&amp;link_uid=PMC2064281">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3741.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4033045/figure/F1/" ref="ordinalpos=3741&amp;ncbi_uid=6201767&amp;link_uid=PMC4033045" href="/pmc/articles/PMC4033045/"><img src="/pmc/articles/instance/4033045/bin/fnmol-07-00046-g001.gif" src-large="/pmc/articles/instance/4033045/bin/fnmol-07-00046-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4033045/" ref="ordinalpos=3741&amp;ncbi_uid=6201767&amp;link_uid=PMC4033045" image-link="/pmc/articles/PMC4033045/figure/F1/" class="imagepopup">FIGURE 1.  From: GSK-3?, a pivotal kinase in Alzheimer disease. </a></p><div class="supp"><p class="details"><div><b>Regulation of GSK-3 activity</b>. GSK-3 is constitutively active in most tissues and most commonly regulated by inhibitory phosphorylation on Ser9. The most relevant extracellular <span class="highlight" style="background-color:">signaling</span> pathways known to negatively regulate GSK-3 activity are those of insulin/insulin-like growth factor I and Wnt. In the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Wnt stabilizes levels of -catenin. Subsequently, stabilized -catenin initiates the transcription of target genes. GSK-3 phosphorylates several components of this transduction <span class="highlight" style="background-color:">pathway</span>, -catenin being the most widely characterized. Phosphorylated -catenin is recognized by ubiquitin and targeted for proteasomal degradation (Wu and He, 2006). In addition, Akt phosphorylate Ser9 of GSK-3 in the context of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Sutherland et al., 1993). Consequently, signals that modify GSK-3 activity are expected to alter -catenin levels (Forde and Dale, 2007).</div></p></div><div class="aux"><div class="resc">Mara Llorens-Martin, et al. Front Mol Neurosci. 2014;7:46.</div><p class="links"><a class="dblinks" href="/pubmed/24904272" ref="ordinalpos=3741&amp;ncbi_uid=6201767&amp;link_uid=24904272">Citation</a><a class="dblinks" href="/pmc/articles/PMC4033045/" ref="ordinalpos=3741&amp;ncbi_uid=6201767&amp;link_uid=PMC4033045">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3742.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4368090/figure/f4-mmr-11-05-3219/" ref="ordinalpos=3742&amp;ncbi_uid=7325724&amp;link_uid=PMC4368090" href="/pmc/articles/PMC4368090/"><img src="/pmc/articles/instance/4368090/bin/MMR-11-05-3219-g03.gif" src-large="/pmc/articles/instance/4368090/bin/MMR-11-05-3219-g03.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4368090/" ref="ordinalpos=3742&amp;ncbi_uid=7325724&amp;link_uid=PMC4368090" image-link="/pmc/articles/PMC4368090/figure/f4-mmr-11-05-3219/" class="imagepopup">Figure 4.  From: Exploring the association between interleukin-1 and its interacting proteins in Alzheimers disease. </a></p><div class="supp"><p class="details"><div>Functional mechanisms of the pathology of Alzheimers disease are associated with IL-1. (A) Interaction map describing the <span class="highlight" style="background-color:">signaling</span> pathways triggered by IL-1. (B) Interaction map describing the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that induces changes to IL-1 in Alzheimers disease. IL, interleukin.</div></p></div><div class="aux"><div class="resc">LUSHUANG XIE, et al. Mol Med Rep. 2015 May;11(5):3219-3228.</div><p class="links"><a class="dblinks" href="/pubmed/25585621" ref="ordinalpos=3742&amp;ncbi_uid=7325724&amp;link_uid=25585621">Citation</a><a class="dblinks" href="/pmc/articles/PMC4368090/" ref="ordinalpos=3742&amp;ncbi_uid=7325724&amp;link_uid=PMC4368090">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3743.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1804097/figure/F7/" ref="ordinalpos=3743&amp;ncbi_uid=937691&amp;link_uid=PMC1804097" href="/pmc/articles/PMC1804097/"><img src="/pmc/articles/instance/1804097/bin/nihms14446f7.gif" src-large="/pmc/articles/instance/1804097/bin/nihms14446f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1804097/" ref="ordinalpos=3743&amp;ncbi_uid=937691&amp;link_uid=PMC1804097" image-link="/pmc/articles/PMC1804097/figure/F7/" class="imagepopup">Figure 7.  From: Bad as a converging <span class="highlight" style="background-color:">signaling</span> molecule between survival PI3-K/Akt and death JNK in neurons after transient focal cerebral ischemia in rats. </a></p><div class="supp"><p class="details"><div>Schematic depicting the effect of the PI3-K/Akt <span class="highlight" style="background-color:">pathway</span> and the JNK <span class="highlight" style="background-color:">pathway</span> on Bad activation in the peripheral area after brain ischemia/reperfusion injury and pharmacological manipulation. The PI3-K/Akt <span class="highlight" style="background-color:">pathway</span> promotes phosphorylation of Bad and the association of Bad with 14-3-3, which mediates cell survival. In contrast, the JNK <span class="highlight" style="background-color:">pathway</span> encourages the dissociation of Bad from 14-3-3, the dephosphorylation of Bad, and the dimerization of Bad with Bcl-X<sub>L</sub> or Bcl-2, followed by cell death. After ischemia/reperfusion injury, the signal balance between the PI3-K/Akt <span class="highlight" style="background-color:">pathway</span> and the JNK <span class="highlight" style="background-color:">pathway</span> regulates Bad activation. Bad might play a crucial role as a determinant of cell fate after cerebral ischemia. MAPKKK, mitogen-activated protein kinase kinase kinase; MAPKK, mitogen-activated protein kinase kinase; P, phosphate group.</div></p></div><div class="aux"><div class="resc">Hiroshi Kamada, et al. J Cereb Blood Flow Metab. ;27(3):521-533.</div><p class="links"><a class="dblinks" href="/pubmed/16820799" ref="ordinalpos=3743&amp;ncbi_uid=937691&amp;link_uid=16820799">Citation</a><a class="dblinks" href="/pmc/articles/PMC1804097/" ref="ordinalpos=3743&amp;ncbi_uid=937691&amp;link_uid=PMC1804097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3744.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3736507/figure/fig1/" ref="ordinalpos=3744&amp;ncbi_uid=5216142&amp;link_uid=PMC3736507" href="/pmc/articles/PMC3736507/"><img src="/pmc/articles/instance/3736507/bin/BMRI2013-916530.001.gif" src-large="/pmc/articles/instance/3736507/bin/BMRI2013-916530.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3736507/" ref="ordinalpos=3744&amp;ncbi_uid=5216142&amp;link_uid=PMC3736507" image-link="/pmc/articles/PMC3736507/figure/fig1/" class="imagepopup">Figure 1.  From: The Age-Related Changes in Cartilage and Osteoarthritis. </a></p><div class="supp"><p class="details"><div>Schematic diagram showing the key role of ROS in age-related chondrocyte changes. Excessive levels of ROS inhibited matrix synthesis (aggrecan, type II collagen) by suppressing the IRS-1-PI-3 kinase-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> or by activating the ERK MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Sustained activation of ERK can induce cell senescence. In addition, extracellular ROS could also contribute to the inhibition of the Akt <span class="highlight" style="background-color:">pathway</span> through oxidized low-density lipoprotein (LDL). The binding of oxidized LDL to cell surface receptor LOX-1 was found to induce chondrocyte senescence (blue arrow).</div></p></div><div class="aux"><div class="resc">YongPing Li, et al. Biomed Res Int. 2013;2013:916530.</div><p class="links"><a class="dblinks" href="/pubmed/23971049" ref="ordinalpos=3744&amp;ncbi_uid=5216142&amp;link_uid=23971049">Citation</a><a class="dblinks" href="/pmc/articles/PMC3736507/" ref="ordinalpos=3744&amp;ncbi_uid=5216142&amp;link_uid=PMC3736507">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3745.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2891946/figure/F7/" ref="ordinalpos=3745&amp;ncbi_uid=2920812&amp;link_uid=PMC2891946" href="/pmc/articles/PMC2891946/"><img src="/pmc/articles/instance/2891946/bin/nihms182803f7.gif" src-large="/pmc/articles/instance/2891946/bin/nihms182803f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2891946/" ref="ordinalpos=3745&amp;ncbi_uid=2920812&amp;link_uid=PMC2891946" image-link="/pmc/articles/PMC2891946/figure/F7/" class="imagepopup">Figure 7.  From: Thrombin Up-regulates the Angiopoietin/Tie2 Axis: EPCR occupancy prevents the thrombin mobilization of angiopoietin2 and P-selectin from Weibel-Palade bodies. </a></p><div class="supp"><p class="details"><div>Hypothetical model of crosstalks between EPCR, PAR-1, S1P<sub>1</sub>, and Tie2. EPCR is associated with caveolin-1 (Cav-1) in lipid-rafts of endothelial cells when the receptor is not occupied by Gla-domain of protein C/APC. Thrombin cleavage of PAR-1 elicits disruptive responses through <span class="highlight" style="background-color:">signaling</span> via G<sub>q</sub> and/or G<sub>12/13</sub>, thereby activating the NF-B <span class="highlight" style="background-color:">pathway</span>. However, the occupancy of EPCR by protein C (PC) results in the dissociation of EPCR from caveolin-1, thereby switching the specificity of PAR-1 <span class="highlight" style="background-color:">signaling</span> by coupling it to the G<sub>i</sub>-protein and/or transactivation of the G<sub>i</sub>-protein coupled receptor, S1P<sub>1</sub>, and mediating the phosphorylation of the receptor by the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. The EPCR-dependent PAR-1 cleavage by thrombin also increases the expression levels of Ang1 and Tie2, thus initiating/amplifying the PI3K/Akt-dependent protective <span class="highlight" style="background-color:">pathway</span> through the Ang1-mediated phosphorylation of Tie2.</div></p></div><div class="aux"><div class="resc">Jong-Sup Bae, et al. J Thromb Haemost. ;8(5):1107-1115.</div><p class="links"><a class="dblinks" href="/pubmed/20180904" ref="ordinalpos=3745&amp;ncbi_uid=2920812&amp;link_uid=20180904">Citation</a><a class="dblinks" href="/pmc/articles/PMC2891946/" ref="ordinalpos=3745&amp;ncbi_uid=2920812&amp;link_uid=PMC2891946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3746.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1609183/figure/F7/" ref="ordinalpos=3746&amp;ncbi_uid=883721&amp;link_uid=PMC1609183" href="/pmc/articles/PMC1609183/"><img src="/pmc/articles/instance/1609183/bin/1471-2407-6-236-7.gif" src-large="/pmc/articles/instance/1609183/bin/1471-2407-6-236-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1609183/" ref="ordinalpos=3746&amp;ncbi_uid=883721&amp;link_uid=PMC1609183" image-link="/pmc/articles/PMC1609183/figure/F7/" class="imagepopup">Figure 7.  From: Fusion of the BCL9 HD2 domain to E1A increases the cytopathic effect of an oncolytic adenovirus that targets colon cancer cells. </a></p><div class="supp"><p class="details"><div><b>Three possible examples of tumor-specific targeting using E1A fusion proteins</b>. In each case E1A is fused to a protein domain (X) that binds to a protein normally found at the promoter in increased amounts in tumor cells. Notch <span class="highlight" style="background-color:">Pathway</span>: in the absence of a signal, CSL recruits co-repressors; upon <span class="highlight" style="background-color:">signaling</span>, NotchIC is released, binds to CSL and activates transcription. Hedgehog <span class="highlight" style="background-color:">pathway</span>: in the absence of a signal, Cubitus Interruptus is cleaved to its repressor form; upon <span class="highlight" style="background-color:">signaling</span>, full-length Ci is produced and activates transcription. Hypoxia <span class="highlight" style="background-color:">pathway</span>: in normoxia, HIF1 is degraded; in hypoxia, HIF1 is stabilized, dimerizes with ARNT, binds to DNA and activates transcription.</div></p></div><div class="aux"><div class="resc">Christophe Fuerer, et al. BMC Cancer. 2006;6:236-236.</div><p class="links"><a class="dblinks" href="/pubmed/17020613" ref="ordinalpos=3746&amp;ncbi_uid=883721&amp;link_uid=17020613">Citation</a><a class="dblinks" href="/pmc/articles/PMC1609183/" ref="ordinalpos=3746&amp;ncbi_uid=883721&amp;link_uid=PMC1609183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3747.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3094054/figure/F8/" ref="ordinalpos=3747&amp;ncbi_uid=3481358&amp;link_uid=PMC3094054" href="/pmc/articles/PMC3094054/"><img src="/pmc/articles/instance/3094054/bin/zh20051162300008.gif" src-large="/pmc/articles/instance/3094054/bin/zh20051162300008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3094054/" ref="ordinalpos=3747&amp;ncbi_uid=3481358&amp;link_uid=PMC3094054" image-link="/pmc/articles/PMC3094054/figure/F8/" class="imagepopup">Fig. 8. From: Cilia movement regulates expression of the Raf-1 kinase inhibitor protein. </a></p><div class="supp"><p class="details"><div>Schematic of proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal kidney cells. Cilium bending in response to tubular flow allows for calcium entry into the cell. This results in the translocation of PKC to the cell membrane, where it leads to an increase in RKIP. RKIP then suppresses <span class="highlight" style="background-color:">signaling</span> through the MAPK/ERK <span class="highlight" style="background-color:">pathway</span>, leading to control of cellular proliferation and maintenance of cell differentiation.</div></p></div><div class="aux"><div class="resc">Kelli M. Sas, et al. Am J Physiol Renal Physiol. 2011 May;300(5):F1163-F1170.</div><p class="links"><a class="dblinks" href="/pubmed/21345975" ref="ordinalpos=3747&amp;ncbi_uid=3481358&amp;link_uid=21345975">Citation</a><a class="dblinks" href="/pmc/articles/PMC3094054/" ref="ordinalpos=3747&amp;ncbi_uid=3481358&amp;link_uid=PMC3094054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3748.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4254673/figure/f8-or-33-01-0019/" ref="ordinalpos=3748&amp;ncbi_uid=6913522&amp;link_uid=PMC4254673" href="/pmc/articles/PMC4254673/"><img src="/pmc/articles/instance/4254673/bin/OR-33-01-0019-g07.gif" src-large="/pmc/articles/instance/4254673/bin/OR-33-01-0019-g07.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4254673/" ref="ordinalpos=3748&amp;ncbi_uid=6913522&amp;link_uid=PMC4254673" image-link="/pmc/articles/PMC4254673/figure/f8-or-33-01-0019/" class="imagepopup">Figure 8.  From: Activation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in breast cancer MCF-7 cells by a peptide derived from Porphyra yezoensis. </a></p><div class="supp"><p class="details"><div>Proposed model for autophagy and metastasis mediated by mTOR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">SU-JIN PARK, et al. Oncol Rep. 2015 January;33(1):19-24.</div><p class="links"><a class="dblinks" href="/pubmed/25333576" ref="ordinalpos=3748&amp;ncbi_uid=6913522&amp;link_uid=25333576">Citation</a><a class="dblinks" href="/pmc/articles/PMC4254673/" ref="ordinalpos=3748&amp;ncbi_uid=6913522&amp;link_uid=PMC4254673">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3749.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3138690/figure/F6/" ref="ordinalpos=3749&amp;ncbi_uid=3637209&amp;link_uid=PMC3138690" href="/pmc/articles/PMC3138690/"><img src="/pmc/articles/instance/3138690/bin/zh89991174130006.gif" src-large="/pmc/articles/instance/3138690/bin/zh89991174130006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3138690/" ref="ordinalpos=3749&amp;ncbi_uid=3637209&amp;link_uid=PMC3138690" image-link="/pmc/articles/PMC3138690/figure/F6/" class="imagepopup">Figure 6.  From: A tumor suppressor function of the Msr1 gene in leukemia stem cells of chronic myeloid leukemia. </a></p><div class="supp"><p class="details"><div><b><i>Msr1</i> affects CML development by regulating PI3K-AKT-GSK-3 <span class="highlight" style="background-color:">pathway</span> and -Catenin.</b> (A) <i>Msr1</i> regulates <span class="highlight" style="background-color:">signaling</span> pathways involved in the regulation of LSC functions. Spleens of recipients of <i>BCR-ABL</i>transduced WT or <i>Msr1</i><sup>/</sup> donor bone marrow cells (n = 4) at 14 days after transplantation were collected and protein lysates were prepared for expression of a group of <span class="highlight" style="background-color:">signaling</span> molecules. Loss of <i>Msr1</i> markedly activated the PI3K-AKT <span class="highlight" style="background-color:">pathway</span> and increased expression of -Catenin. (B) Quantification of relative protein expression levels in spleens of recipients of <i>BCR-ABL</i>-transduced WT or <i>Msr1</i><sup>/</sup> donor bone marrow cells. Mean value ( SD) for each group (n = 4) is shown (<i>P</i> &lt; .05, **<i>P</i> &lt; .01). (C) PMA inhibits PI3K-AKT <span class="highlight" style="background-color:">signaling</span> pathways and -Catenin expression in human CML cells. K562 cells were treated with PMA, and protein lysates were collected for expression of a group of <span class="highlight" style="background-color:">signaling</span> molecules. PMA markedly inhibits activity of PI3K-AKT and reduced expression of -Catenin. (D) Over-expression of <i>Msr</i>1 inhibits the PI3K-AKT <span class="highlight" style="background-color:">signaling</span> pathways and reduces -Catenin expression in human CML cells. Protein lysates from K562 transduced with <i>pMSCV-Msr1-GFP</i> or <i>pMSCV-GFP</i> retrovirus were analyzed by Western blotting for expression of a group of <span class="highlight" style="background-color:">signaling</span> molecules indicated. (E) <i>Msr1</i> suppresses the proliferation of human leukemic cells. The same number of K562 cells transduced with <i>pMSCV-GFP</i> or <i>pMSCV-Msr1-GFP</i> was cultured in 24-well plates. Live cells were counted at 24h, 48h and 96h (*<i>P</i> &lt; .05, **<i>P</i> &lt; .01). (F) A summary of <i>Msr1</i> <span class="highlight" style="background-color:">pathway</span> in LSCs of CML.</div></p></div><div class="aux"><div class="resc">Yaoyu Chen, et al. Blood. 2011 July 14;118(2):390-400.</div><p class="links"><a class="dblinks" href="/pubmed/21596859" ref="ordinalpos=3749&amp;ncbi_uid=3637209&amp;link_uid=21596859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3138690/" ref="ordinalpos=3749&amp;ncbi_uid=3637209&amp;link_uid=PMC3138690">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3750.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3835791/figure/pone-0079065-g001/" ref="ordinalpos=3750&amp;ncbi_uid=5519605&amp;link_uid=PMC3835791" href="/pmc/articles/PMC3835791/"><img src="/pmc/articles/instance/3835791/bin/pone.0079065.g001.gif" src-large="/pmc/articles/instance/3835791/bin/pone.0079065.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3835791/" ref="ordinalpos=3750&amp;ncbi_uid=5519605&amp;link_uid=PMC3835791" image-link="/pmc/articles/PMC3835791/figure/pone-0079065-g001/" class="imagepopup">Figure 1. The ATX/LPA <span class="highlight" style="background-color:">pathway</span>. ATX cleaves the lipid LPC to generate LPA. LPA binding to GPCRs stimulates several <span class="highlight" style="background-color:">signaling</span> pathways..  From: Non-Invasive Imaging of Tumors by Monitoring Autotaxin Activity Using an Enzyme-Activated Near-Infrared Fluorogenic Substrate. </a></p><div class="aux"><div class="resc">Damian Madan, et al. PLoS One. 2013;8(11):e79065.</div><p class="links"><a class="dblinks" href="/pubmed/24278115" ref="ordinalpos=3750&amp;ncbi_uid=5519605&amp;link_uid=24278115">Citation</a><a class="dblinks" href="/pmc/articles/PMC3835791/" ref="ordinalpos=3750&amp;ncbi_uid=5519605&amp;link_uid=PMC3835791">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3751.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3566969/figure/F7/" ref="ordinalpos=3751&amp;ncbi_uid=4630712&amp;link_uid=PMC3566969" href="/pmc/articles/PMC3566969/"><img src="/pmc/articles/instance/3566969/bin/1471-2164-14-37-7.gif" src-large="/pmc/articles/instance/3566969/bin/1471-2164-14-37-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3566969/" ref="ordinalpos=3751&amp;ncbi_uid=4630712&amp;link_uid=PMC3566969" image-link="/pmc/articles/PMC3566969/figure/F7/" class="imagepopup">Figure 7.  From: Transcriptional profiling of the Arabidopsis abscission mutant hae hsl2 by RNA-Seq. </a></p><div class="supp"><p class="details"><div><b>HAE HSL2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and a summary of </b><b><i>HAE HSL2</i></b><b>-dependent and </b><b><i>HAE HSL2</i></b><b>-independent gene expression changes during abscission.</b></div></p></div><div class="aux"><div class="resc">Chad E Niederhuth, et al. BMC Genomics. 2013;14:37-37.</div><p class="links"><a class="dblinks" href="/pubmed/23327667" ref="ordinalpos=3751&amp;ncbi_uid=4630712&amp;link_uid=23327667">Citation</a><a class="dblinks" href="/pmc/articles/PMC3566969/" ref="ordinalpos=3751&amp;ncbi_uid=4630712&amp;link_uid=PMC3566969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3752.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2631405/figure/F1/" ref="ordinalpos=3752&amp;ncbi_uid=1675500&amp;link_uid=PMC2631405" href="/pmc/articles/PMC2631405/"><img src="/pmc/articles/instance/2631405/bin/nihms84804f1.gif" src-large="/pmc/articles/instance/2631405/bin/nihms84804f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2631405/" ref="ordinalpos=3752&amp;ncbi_uid=1675500&amp;link_uid=PMC2631405" image-link="/pmc/articles/PMC2631405/figure/F1/" class="imagepopup">Figure 1. Comparison of GH/IGF-1/insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in C. elegans, Drosophila and mammals.  From: Role of the GH/IGF-1 axis in lifespan and healthspan: lessons from animal models. </a></p><div class="supp"><p class="details"><div>Schematic adapted from two recent reviews[191,192]</div></p></div><div class="aux"><div class="resc">Darlene E. Berryman, et al. Growth Horm IGF Res. ;18(6):455-471.</div><p class="links"><a class="dblinks" href="/pubmed/18710818" ref="ordinalpos=3752&amp;ncbi_uid=1675500&amp;link_uid=18710818">Citation</a><a class="dblinks" href="/pmc/articles/PMC2631405/" ref="ordinalpos=3752&amp;ncbi_uid=1675500&amp;link_uid=PMC2631405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3753.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3295195/figure/F10/" ref="ordinalpos=3753&amp;ncbi_uid=4209491&amp;link_uid=PMC3295195" href="/pmc/articles/PMC3295195/"><img src="/pmc/articles/instance/3295195/bin/zmb9991094060010.gif" src-large="/pmc/articles/instance/3295195/bin/zmb9991094060010.jpg" alt="Fig 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3295195/" ref="ordinalpos=3753&amp;ncbi_uid=4209491&amp;link_uid=PMC3295195" image-link="/pmc/articles/PMC3295195/figure/F10/" class="imagepopup">Fig 10.  From: Angiotensin II Induces Epithelial-to-Mesenchymal Transition in Renal Epithelial Cells through Reactive Oxygen Species/Src/Caveolin-Mediated Activation of an Epidermal Growth Factor Receptor-Extracellular Signal-Regulated Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A novel mechanism linking ROS/Src/caveolin-mediated activation of the EGFR-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to chronic Ang II exposure-induced EMT in cultured renal tubular epithelial cells. Our studies utilizing the renal proximal tubular epithelial cells expressing functioning AT1 receptors suggest that binding of Ang II to the seven-transmembrane G-protein-coupled AT1 receptor activates metalloproteinase (Mpase)-dependent release of soluble HB-EGF, a natural EGFR ligand that binds and transactivates EGFR, as we have recently documented (5), resulting in acute <span class="highlight" style="background-color:">signaling</span> leading to ERK activation; such ligand-activated ERK <span class="highlight" style="background-color:">signaling</span> diminishes within 1 h (Fig. 5, 6, and 8C). The transient nature of this EGFR-ERK <span class="highlight" style="background-color:">signaling</span> is presumably due to the quick endocytosis of the ligand (HB-EGF)-activated EGFR through clathrin-coated pits, followed by degradation through the endosomal/lysosomal <span class="highlight" style="background-color:">pathway</span> (9, 39). Meanwhile, Ang II activation of AT1 receptors also leads to transactivation of EGFR through ROS-dependent Src kinase activation, which phosphorylates both caveolin-1 (Cav) at Y14 and EGFR at Y845, thus inducing association of EGFR with Cav and the adaptor proteins SHC and GRB2, resulting in prolonged EGFR-ERK <span class="highlight" style="background-color:">signaling</span>. In the case of chronic Ang II exposure, such prolonged <span class="highlight" style="background-color:">signaling</span> activity persists in the epithelial cells, leading to alterations in gene expression and consequently eliciting a phenotypic change to EMT.</div></p></div><div class="aux"><div class="resc">Jianchun Chen, et al. Mol Cell Biol. 2012 March;32(5):981-991.</div><p class="links"><a class="dblinks" href="/pubmed/22215616" ref="ordinalpos=3753&amp;ncbi_uid=4209491&amp;link_uid=22215616">Citation</a><a class="dblinks" href="/pmc/articles/PMC3295195/" ref="ordinalpos=3753&amp;ncbi_uid=4209491&amp;link_uid=PMC3295195">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3754.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3506182/figure/F2/" ref="ordinalpos=3754&amp;ncbi_uid=5306549&amp;link_uid=PMC3506182" href="/pmc/articles/PMC3506182/"><img src="/pmc/articles/instance/3506182/bin/nihms-396828-f0002.gif" src-large="/pmc/articles/instance/3506182/bin/nihms-396828-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3506182/" ref="ordinalpos=3754&amp;ncbi_uid=5306549&amp;link_uid=PMC3506182" image-link="/pmc/articles/PMC3506182/figure/F2/" class="imagepopup">Figure 2. CCN6 regulates BMP4-mediated TAK1 and p38 activation.  From: CCN6 modulates BMP <span class="highlight" style="background-color:">signaling</span> via the Smad-independent TAK1/p38 <span class="highlight" style="background-color:">pathway</span>, acting to suppress metastasis of breast cancer. </a></p><div class="supp"><p class="details"><div><b>A.</b> Quantitative real time PCR array of TGF-/BMP <span class="highlight" style="background-color:">pathway</span> genes reveal that CCN6 KD principally leads to upregulation of BMP4 and BMP-regulated genes in HME cells. No significant effect was observed on the mRNA levels of TGF-1, TGF-2, BMP1, 2, 3, 5 and 6 genes, or co-receptor Smads between HME CCN6 KD and controls. <b>B.</b> Immunnoblots of HME and MDA-MB-231 cells show that CCN6 mainly regulates BMP4 protein and its non-Smad <span class="highlight" style="background-color:">signaling</span> proteins P-TAK1 and P-p38 MAPK.</div></p></div><div class="aux"><div class="resc">Anupama Pal, et al. Cancer Res. ;72(18):4818-4828.</div><p class="links"><a class="dblinks" href="/pubmed/22805309" ref="ordinalpos=3754&amp;ncbi_uid=5306549&amp;link_uid=22805309">Citation</a><a class="dblinks" href="/pmc/articles/PMC3506182/" ref="ordinalpos=3754&amp;ncbi_uid=5306549&amp;link_uid=PMC3506182">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3755.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4279394/figure/F5/" ref="ordinalpos=3755&amp;ncbi_uid=7040918&amp;link_uid=PMC4279394" href="/pmc/articles/PMC4279394/"><img src="/pmc/articles/instance/4279394/bin/oncotarget-05-10571-g005.gif" src-large="/pmc/articles/instance/4279394/bin/oncotarget-05-10571-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4279394/" ref="ordinalpos=3755&amp;ncbi_uid=7040918&amp;link_uid=PMC4279394" image-link="/pmc/articles/PMC4279394/figure/F5/" class="imagepopup">Figure 5. SOX10 suppresses Wnt <span class="highlight" style="background-color:">signaling</span> via compete with TCF4 to bind -catenin.  From: SOX10, a novel HMG-box-containing tumor suppressor, inhibits growth and metastasis of digestive cancers by suppressing the Wnt/-catenin <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Inhibition of TCF activity (A) and <i>CCND1</i>, <i>c-Myc</i> and <i>MMP7</i> promoter reporter activities (B) in SOX10-expressing tumor cells in a dose-dependent manner. *: <i>p</i>&lt;0.05, **: <i>p</i>&lt;0.01. qRT-PCR (C) and western blot (D) showed ectopic expression of SOX10 disrupted Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (E) Co-immunoprecipitation (Co-IP) assays showing that exogenous V5-SOX10 binds endogenous -catenin and vice versa. IP antibody or IgG negative control was used as indicated. (F) GST pull-down assay was performed with GST--catenin fusion protein or GST and V5-SOX10 as indicated.</div></p></div><div class="aux"><div class="resc">Xin Tong, et al. Oncotarget. 2014 November;5(21):10571-10583.</div><p class="links"><a class="dblinks" href="/pubmed/25301735" ref="ordinalpos=3755&amp;ncbi_uid=7040918&amp;link_uid=25301735">Citation</a><a class="dblinks" href="/pmc/articles/PMC4279394/" ref="ordinalpos=3755&amp;ncbi_uid=7040918&amp;link_uid=PMC4279394">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3756.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4339951/figure/F1/" ref="ordinalpos=3756&amp;ncbi_uid=7217794&amp;link_uid=PMC4339951" href="/pmc/articles/PMC4339951/"><img src="/pmc/articles/instance/4339951/bin/nihms658496f1.gif" src-large="/pmc/articles/instance/4339951/bin/nihms658496f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4339951/" ref="ordinalpos=3756&amp;ncbi_uid=7217794&amp;link_uid=PMC4339951" image-link="/pmc/articles/PMC4339951/figure/F1/" class="imagepopup">Fig. 1.  From: Genetic Modifiers for Neuromuscular Diseases. </a></p><div class="supp"><p class="details"><div>The TGF <span class="highlight" style="background-color:">pathway</span> in muscular dystrophy. <i>SPP1</i>, encoding osteopontin (OPN), and TGF are co-regulated. A) When myofibers are injured, osteopontin and TGF <span class="highlight" style="background-color:">signaling</span> pathways are activated, leading to ECM accumulation and fibrosis of the muscle. B) TGF activates the <i>SPP1</i> promoter through SMAD <span class="highlight" style="background-color:">signaling</span>. A SNP in the <i>SPP1</i> promoter is associated with a more severe form of DMD. A SNP in <i>TGFBR2</i> results in increased <i>SPP1</i> expression. The VTTT allele of <i>LTBP4</i> leads to increased TGF <span class="highlight" style="background-color:">signaling</span> and is associated with a more severe form of DMD. Treatment of myoblasts with TGF results in increased osteopontin expression. The <i>mdx</i> mouse deficient for <i>SPP1</i>/osteopontin has reduced TGF <span class="highlight" style="background-color:">signaling</span> and reduced fibrosis.</div></p></div><div class="aux"><div class="resc">Kay-Marie Lamar, et al. J Neuromuscul Dis. ;1(1):3-13.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3756&amp;ncbi_uid=7217794&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4339951/" ref="ordinalpos=3756&amp;ncbi_uid=7217794&amp;link_uid=PMC4339951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3757.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3052972/figure/F1/" ref="ordinalpos=3757&amp;ncbi_uid=3646866&amp;link_uid=PMC3052972" href="/pmc/articles/PMC3052972/"><img src="/pmc/articles/instance/3052972/bin/nihms240750f1.gif" src-large="/pmc/articles/instance/3052972/bin/nihms240750f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3052972/" ref="ordinalpos=3757&amp;ncbi_uid=3646866&amp;link_uid=PMC3052972" image-link="/pmc/articles/PMC3052972/figure/F1/" class="imagepopup">Figure 1. Peg-IFN decreases -catenin activity in vitro and does so independent of JAK/STAT <span class="highlight" style="background-color:">signaling</span>.  From: PEGYLATED INTERFERON ALPHA TARGETS WNT <span class="highlight" style="background-color:">SIGNALING</span> BY INDUCING NUCLEAR EXPORT OF ?-CATENIN. </a></p><div class="supp"><p class="details"><div><b>A.</b> Relative luciferase activity following transfection with either TOPFlash or FOPFlash reporter plasmid and treatment with three or four doses of Peg-IFN in HepG2 cells. Experiments were done in triplicate and averaged. (*p&lt;0.05)<br /><b>B.</b> Relative luciferase activity following transfection with either TOPFlash or FOPFlash reporter plasmid and one treatment with Peg-IFN or with one treatment with lithium followed by one dose treatment of Peg-IFN in Hep3Bcells. Experiments were done in triplicate and averaged. (**p&lt;0.005)<br /><b>C.</b> Inhibition of JAK/Stat <span class="highlight" style="background-color:">pathway</span> with AG490 does not reverse the effect of Peg-IFN on Topflash indicating that this effect is independent of the JAK/Stat <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Michael D. Thompson, et al. J Hepatol. ;54(3):506-512.</div><p class="links"><a class="dblinks" href="/pubmed/21093092" ref="ordinalpos=3757&amp;ncbi_uid=3646866&amp;link_uid=21093092">Citation</a><a class="dblinks" href="/pmc/articles/PMC3052972/" ref="ordinalpos=3757&amp;ncbi_uid=3646866&amp;link_uid=PMC3052972">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3758.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1693893/figure/fig7/" ref="ordinalpos=3758&amp;ncbi_uid=819059&amp;link_uid=PMC1693893" href="/pmc/articles/PMC1693893/"><img src="/pmc/articles/instance/1693893/bin/gkl834f7.gif" src-large="/pmc/articles/instance/1693893/bin/gkl834f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1693893/" ref="ordinalpos=3758&amp;ncbi_uid=819059&amp;link_uid=PMC1693893" image-link="/pmc/articles/PMC1693893/figure/fig7/" class="imagepopup">Figure 7.  From: NSPc1 is a cell growth regulator that acts as a transcriptional repressor of p21Waf1/Cip1 via the RARE element. </a></p><div class="supp"><p class="details"><div>A regulation model of NSPc1/RA receptors in the p21 <span class="highlight" style="background-color:">pathway</span>. (<b>a</b>) Regulatory model: NSPc1 represses p21 expression via RARE element. Note, NSPc1 needs co-binding factor (CoB) to facilitate its localization on RARE element. (<b>b</b>) When the RA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated, RXRs/RARs compete with NSPc1 for binding RARE element on the p21 promoter, and then transcription is activated. (<b>c</b>) Signal <span class="highlight" style="background-color:">pathway</span> model shows connections between NSPc1, RA and p21 pathways.</div></p></div><div class="aux"><div class="resc">Yanhua Gong, et al. Nucleic Acids Res. 2006 December;34(21):6158-6169.</div><p class="links"><a class="dblinks" href="/pubmed/17088287" ref="ordinalpos=3758&amp;ncbi_uid=819059&amp;link_uid=17088287">Citation</a><a class="dblinks" href="/pmc/articles/PMC1693893/" ref="ordinalpos=3758&amp;ncbi_uid=819059&amp;link_uid=PMC1693893">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3759.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3227525/figure/F3/" ref="ordinalpos=3759&amp;ncbi_uid=3877156&amp;link_uid=PMC3227525" href="/pmc/articles/PMC3227525/"><img src="/pmc/articles/instance/3227525/bin/nihms328325f3.gif" src-large="/pmc/articles/instance/3227525/bin/nihms328325f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3227525/" ref="ordinalpos=3759&amp;ncbi_uid=3877156&amp;link_uid=PMC3227525" image-link="/pmc/articles/PMC3227525/figure/F3/" class="imagepopup">Figure 3. Regulation of Wnt <span class="highlight" style="background-color:">pathway</span> genes by Sox17 in OPCs.  From: Sox17 regulates the Wnt/beta-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oligodendrocyte progenitor cells. </a></p><div class="supp"><p class="details"><div>(A) Purified OPCs were transfected with Sox17 siRNAs and allowed to recover for 24 hours in fresh medium supplemented with PDGF. Total cell protein was extracted and the expression of Cdc42, phosphorylated-ERK1/2, C-myc and p27 was detected by Western blotting. (B) OPCs were cultured in medium containing 1 % FBS, in the absence (N1) and presence of 30 ng/ml of recombinant human Wnt3a for 3 days. After culture, total protein was extracted and the expression of RAC/Cdc42/PAKs <span class="highlight" style="background-color:">pathway</span> proteins were detected by Western blotting. (C) Schematic representation of <span class="highlight" style="background-color:">signaling</span> relationships between gene products found regulated by Sox17 siRNA, either at the RNA or protein levels. Upregulated gene products are represented in red, and downregulated products in blue. Solid arrows indicate stimulation, while bars indicate repression of activity.</div></p></div><div class="aux"><div class="resc">Li-Jin Chew, et al. J Neurosci. ;31(39):13921-13935.</div><p class="links"><a class="dblinks" href="/pubmed/21957254" ref="ordinalpos=3759&amp;ncbi_uid=3877156&amp;link_uid=21957254">Citation</a><a class="dblinks" href="/pmc/articles/PMC3227525/" ref="ordinalpos=3759&amp;ncbi_uid=3877156&amp;link_uid=PMC3227525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3760.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2953807/figure/F5/" ref="ordinalpos=3760&amp;ncbi_uid=2513296&amp;link_uid=PMC2953807" href="/pmc/articles/PMC2953807/"><img src="/pmc/articles/instance/3157716/bin/oncotarget-01-210-g005.gif" src-large="/pmc/articles/instance/3157716/bin/oncotarget-01-210-g005.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2953807/" ref="ordinalpos=3760&amp;ncbi_uid=2513296&amp;link_uid=PMC2953807" image-link="/pmc/articles/PMC2953807/figure/F5/" class="imagepopup">Fig. 5. Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> supports Jagged1 expression in ovarian cancer cells..  From: Jagged1 Expression Regulated by Notch3 and Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> Pathways in Ovarian Cancer. </a></p><div class="supp"><p class="details"><div>Quantitative real time PCR shows a significant decrease in Jagged1 expression level after knockdown of either -catenin (left panel) or TCF4 (right panel) in OVCAR3 cells. Well-known WNT targets including CCND1, SOX9, and CD44 are used as positive controls.</div></p></div><div class="aux"><div class="resc">Xu Chen, et al. Oncotarget. 2010 July;1(3):210-218.</div><p class="links"><a class="dblinks" href="/pubmed/20953350" ref="ordinalpos=3760&amp;ncbi_uid=2513296&amp;link_uid=20953350">Citation</a><a class="dblinks" href="/pmc/articles/PMC2953807/" ref="ordinalpos=3760&amp;ncbi_uid=2513296&amp;link_uid=PMC2953807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3761.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC262421/figure/f3/" ref="ordinalpos=3761&amp;ncbi_uid=300074&amp;link_uid=PMC262421" href="/pmc/articles/PMC262421/"><img src="/pmc/articles/instance/262421/bin/mb2231072003.gif" src-large="/pmc/articles/instance/262421/bin/mb2231072003.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC262421/" ref="ordinalpos=3761&amp;ncbi_uid=300074&amp;link_uid=PMC262421" image-link="/pmc/articles/PMC262421/figure/f3/" class="imagepopup">FIG. 3. From: Discrete Functions of TRAF1 and TRAF2 in Drosophila melanogaster Mediated by c-Jun N-Terminal Kinase and NF-?B-Dependent <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Genetic interactions between DTRAF1 and the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components. Scanning electron micrographs of adult eyes are shown. (A) <i>gmr</i>-GAL4, EP(2)0578/+. (B) <i>gmr</i>-GAL4, EP(2)0578/UAS-<i>bsk</i>. (C) <i>gmr</i>-GAL4, EP(2)0578/UAS-<i>hep</i>. (D) <i>hep<sup>1</sup></i>/Y; <i>gmr</i>-GAL4, EP(2)0578/+. (E) EP(X)1516/Y; <i>gmr</i>-GAL4/+. (F) EP(X)1516/Y; <i>gmr</i>-GAL4/UAS-<i>bsk</i>. (G) EP(X)1516/Y; <i>gmr</i>-GAL4/UAS-<i>hep</i>. (H) EP(X)1516/Y; <i>gmr</i>-GAL4, EP(2)0578/+. (I) <i>gmr</i>-GAL4/UAS-<i>DTAK1</i>. (J) <i>gmr</i>-GAL4, EP(2)0578/UAS-<i>DTAK1</i>. (K) <i>DTAK1<sup>1</sup></i>/Y; <i>gmr</i>-GAL4, EP(2)0578/+. (L) <i>DTAK1<sup>1</sup></i>/Y. Anterior to the left and dorsal to the top. Magnification: 200.</div></p></div><div class="aux"><div class="resc">Guang-Ho Cha, et al. Mol Cell Biol. 2003 November;23(22):7982-7991.</div><p class="links"><a class="dblinks" href="/pubmed/14585960" ref="ordinalpos=3761&amp;ncbi_uid=300074&amp;link_uid=14585960">Citation</a><a class="dblinks" href="/pmc/articles/PMC262421/" ref="ordinalpos=3761&amp;ncbi_uid=300074&amp;link_uid=PMC262421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3762.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052545/figure/f1-pharmaceuticals-04-00117/" ref="ordinalpos=3762&amp;ncbi_uid=6238949&amp;link_uid=PMC4052545" href="/pmc/articles/PMC4052545/"><img src="/pmc/articles/instance/4052545/bin/pharmaceuticals-04-00117f1.gif" src-large="/pmc/articles/instance/4052545/bin/pharmaceuticals-04-00117f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052545/" ref="ordinalpos=3762&amp;ncbi_uid=6238949&amp;link_uid=PMC4052545" image-link="/pmc/articles/PMC4052545/figure/f1-pharmaceuticals-04-00117/" class="imagepopup">Figure 1.  From: Sphingosine-1-Phosphate-Specific G Protein-Coupled Receptors as Novel Therapeutic Targets for Atherosclerosis. </a></p><div class="supp"><p class="details"><div>S1P receptor <span class="highlight" style="background-color:">signaling</span>. S1P<sub>1</sub> couples exclusively to G<sub>i</sub>, whereas S1P<sub>2</sub> and S1P<sub>3</sub> couple to multiple G proteins. The G proteins preferred by S1P<sub>2</sub> and S1P<sub>3</sub> are G<sub>12/13</sub> and G<sub>q</sub>, respectively. G<sub>i</sub> couples to stimulation of phosphoinositide 3-kinase (PI3K)-Akt/Rac <span class="highlight" style="background-color:">pathway</span> and Ras-mitogen activated protein kinase <span class="highlight" style="background-color:">pathway</span>, G<sub>12/13</sub> couples to activation of Rho <span class="highlight" style="background-color:">pathway</span>, mediating Rac inhibition, NF-B activation and PTEN stimulation. G<sub>q</sub> mediates stimulation of phospholipase C (PLC) <span class="highlight" style="background-color:">pathway</span>. Nitric oxide synthase (eNOS) in ECs is fully activated by G<sub>i</sub>- and Akt-mediated phosphorylation of eNOS in concert with a G<sub>q</sub>-mediated, Ca<sup>2+</sup>/calmodulin-dependent activation. The straight and dotted lines show the stimulatory and inhibitory regulations, respectively.</div></p></div><div class="aux"><div class="resc">Yasuo Okamoto, et al. Pharmaceuticals (Basel). 2011 January;4(1):117-137.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3762&amp;ncbi_uid=6238949&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052545/" ref="ordinalpos=3762&amp;ncbi_uid=6238949&amp;link_uid=PMC4052545">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3763.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2814769/figure/F5/" ref="ordinalpos=3763&amp;ncbi_uid=2180890&amp;link_uid=PMC2814769" href="/pmc/articles/PMC2814769/"><img src="/pmc/articles/instance/2814769/bin/nihms168162f5.gif" src-large="/pmc/articles/instance/2814769/bin/nihms168162f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2814769/" ref="ordinalpos=3763&amp;ncbi_uid=2180890&amp;link_uid=PMC2814769" image-link="/pmc/articles/PMC2814769/figure/F5/" class="imagepopup">Figure 5. Wnt <span class="highlight" style="background-color:">signaling</span> and the cilium.  From: The Primary Cilium as a Complex <span class="highlight" style="background-color:">Signaling</span> Center. </a></p><div class="supp"><p class="details"><div>The cilium/basal body may function as a regulatory switch to control the balance between the canonical and noncanonical Wnt pathways. In the canonical <span class="highlight" style="background-color:">pathway</span> (A, left), a Wnt ligand binds to the co-receptors Frizzled and LRP. This inhibits the activity of the -catenin destruction complex possibly through the Dishevelled (Dvl) protein and leads to stabilization of -catenin., which accumulates in the nucleus and with LEF and TCF activates target genes. In the noncanonical <span class="highlight" style="background-color:">pathway</span> (A, right), Wnt binds to a frizzled receptor, independent of LRP. This activates a membrane form of Dvl which regulates downstream targets. The noncanonical Wnt signal activates Inversin, which resides in multiple locations in the cell including the cilium or at the base of the cilium. Inversin induces the degradation of cytoplasmic but not the membrane form of Dvl. In a ciliated cell (A), both the canonical and noncanonical pathways are operative and the strength of the canonical <span class="highlight" style="background-color:">pathway</span> (A right) is thought to be influenced by the noncanonical Wnt <span class="highlight" style="background-color:">pathway</span> (A, left). In the absence of cilia (B), the model would suggest that noncanonical <span class="highlight" style="background-color:">pathway</span> is unable to efficiently antagonize the activity of the canonical.</div></p></div><div class="aux"><div class="resc">Nicolas F. Berbari, et al. Curr Biol. ;19(13):R526.</div><p class="links"><a class="dblinks" href="/pubmed/19602418" ref="ordinalpos=3763&amp;ncbi_uid=2180890&amp;link_uid=19602418">Citation</a><a class="dblinks" href="/pmc/articles/PMC2814769/" ref="ordinalpos=3763&amp;ncbi_uid=2180890&amp;link_uid=PMC2814769">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3764.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2711664/figure/f1/" ref="ordinalpos=3764&amp;ncbi_uid=1911496&amp;link_uid=PMC2711664" href="/pmc/articles/PMC2711664/"><img src="/pmc/articles/instance/2711664/bin/zh10070957180001.gif" src-large="/pmc/articles/instance/2711664/bin/zh10070957180001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2711664/" ref="ordinalpos=3764&amp;ncbi_uid=1911496&amp;link_uid=PMC2711664" image-link="/pmc/articles/PMC2711664/figure/f1/" class="imagepopup">Fig. 1. From: The action of leptin in the ventral tegmental area to decrease food intake is dependent on Jak-2 <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Signal transduction mechanisms implicated in leptin action. Leptin binding activates Janus kinase (Jak-2) that leads to the binding of and tyrosine phosphorylation of signal transducer and activator of transcription (STAT3). In addition, leptin receptor <span class="highlight" style="background-color:">signaling</span> can activate the insulin receptor substrate (IRS)-phosphatidylinositol 3-kinase [PI 3-kinase (PI3K)] <span class="highlight" style="background-color:">pathway</span> and mammalian target of rapamycin (mTOR) <span class="highlight" style="background-color:">signaling</span> via protein kinase B (Akt).</div></p></div><div class="aux"><div class="resc">Gregory J. Morton, et al. Am J Physiol Endocrinol Metab. 2009 July;297(1):E202-E210.</div><p class="links"><a class="dblinks" href="/pubmed/19435852" ref="ordinalpos=3764&amp;ncbi_uid=1911496&amp;link_uid=19435852">Citation</a><a class="dblinks" href="/pmc/articles/PMC2711664/" ref="ordinalpos=3764&amp;ncbi_uid=1911496&amp;link_uid=PMC2711664">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3765.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2573088/figure/fig11/" ref="ordinalpos=3765&amp;ncbi_uid=1496447&amp;link_uid=PMC2573088" href="/pmc/articles/PMC2573088/"><img src="/pmc/articles/instance/2573088/bin/zbc0470855210011.gif" src-large="/pmc/articles/instance/2573088/bin/zbc0470855210011.jpg" alt="FIGURE 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2573088/" ref="ordinalpos=3765&amp;ncbi_uid=1496447&amp;link_uid=PMC2573088" image-link="/pmc/articles/PMC2573088/figure/fig11/" class="imagepopup">FIGURE 11. From: Up-regulating Sphingosine 1-Phosphate Receptor-2 <span class="highlight" style="background-color:">Signaling</span> Impairs  Chemotactic, Wound-healing, and Morphogenetic Responses in Senescent  Endothelial  Cells. </a></p><div class="supp"><p class="details"><div><b>Model for S1P<sub>2</sub> up-regulation impairs the activation of  integrin</b> <b><sub>v</sub></b><b><sub>3</sub>, chemotactic,  wound-healing, and morphogenetic responses in senescent ECs.</b>  S1P<sub>1</sub>-mediated Akt/Rac <span class="highlight" style="background-color:">signaling</span> stimulates integrin  <sub>v</sub><sub>3</sub> activation, chemotactic, wound-healing,  and morphogenetic responses in young ECs. In contrast, both S1P<sub>1</sub>  and S1P<sub>2</sub> subtypes of S1P family receptors are up-regulated in  senescent ECs. The increased expression of S1P<sub>2</sub> antagonizes the  stimulatory <span class="highlight" style="background-color:">signaling</span> mediated by S1P/S1P<sub>1</sub> <span class="highlight" style="background-color:">pathway</span>  via elevating cellular PTEN phosphatase activity. The S1P<sub>2</sub>-mediated  PTEN activation results in the inactivation of Rac GTPase, which ultimately  inhibits the integrin <sub>v</sub><sub>3</sub> activation,  migratory, wound-healing, and morphogenetic capabilities in senescent ECs.</div></p></div><div class="aux"><div class="resc">Rosendo Estrada, et al. J Biol Chem. 2008 October 31;283(44):30363-30375.</div><p class="links"><a class="dblinks" href="/pubmed/18765664" ref="ordinalpos=3765&amp;ncbi_uid=1496447&amp;link_uid=18765664">Citation</a><a class="dblinks" href="/pmc/articles/PMC2573088/" ref="ordinalpos=3765&amp;ncbi_uid=1496447&amp;link_uid=PMC2573088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3766.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3648641/figure/F1/" ref="ordinalpos=3766&amp;ncbi_uid=4883520&amp;link_uid=PMC3648641" href="/pmc/articles/PMC3648641/"><img src="/pmc/articles/instance/3648641/bin/nihms465394f1.gif" src-large="/pmc/articles/instance/3648641/bin/nihms465394f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3648641/" ref="ordinalpos=3766&amp;ncbi_uid=4883520&amp;link_uid=PMC3648641" image-link="/pmc/articles/PMC3648641/figure/F1/" class="imagepopup">Figure 1. Regulation of hepcidin by iron.  From: Unraveling Mechanisms Regulating Systemic Iron Homeostasis. </a></p><div class="supp"><p class="details"><div>Binding of BMP6 to receptors (BMPRs) at the hepatocyte plasma membrane results in phosphorylation (P) of receptor-associated SMAD proteins (R-SMADs), which complex with the common mediator SMAD4. This SMAD4 complex translocates to the nucleus to bind specific regulatory elements in the promoter of the hepcidin gene, increasing hepcidin transcription. The GPI-linked protein HJV acts as a BMP co-receptor that promotes hepcidin <span class="highlight" style="background-color:">signaling</span> through this <span class="highlight" style="background-color:">pathway</span>. Neogenin may interact with HJV and enhance BMP <span class="highlight" style="background-color:">signaling</span>. In contrast, the transmembrane serine protease TMPRSS6 inhibits hepcidin <span class="highlight" style="background-color:">signaling</span> through the BMP <span class="highlight" style="background-color:">pathway</span> by cleaving HJV from the plasma membrane. Hepatic BMP6 production increases in response to chronic iron loading, suggesting an autocrine or paracrine mechanism for regulating hepcidin synthesis to limit the progression of systemic iron loading. Binding of holo-transferrin (HoloTf) to TFR1 displaces HFE, which may interact with TFR2 to promote hepcidin expression. While evidence suggests that HFE modulates hepcidin <span class="highlight" style="background-color:">signaling</span> through the BMP <span class="highlight" style="background-color:">pathway</span>, how HFE and its interactions with TFR1 and TFR2 may relate functionally to the BMP <span class="highlight" style="background-color:">pathway</span> is not yet known.</div></p></div><div class="aux"><div class="resc">Karin E. Finberg. Hematology Am Soc Hematol Educ Program. 2011;2011:532-537.</div><p class="links"><a class="dblinks" href="/pubmed/22160085" ref="ordinalpos=3766&amp;ncbi_uid=4883520&amp;link_uid=22160085">Citation</a><a class="dblinks" href="/pmc/articles/PMC3648641/" ref="ordinalpos=3766&amp;ncbi_uid=4883520&amp;link_uid=PMC3648641">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3767.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3245021/figure/gkr1127-F3/" ref="ordinalpos=3767&amp;ncbi_uid=3747495&amp;link_uid=PMC3245021" href="/pmc/articles/PMC3245021/"><img src="/pmc/articles/instance/3245021/bin/gkr1127f3.gif" src-large="/pmc/articles/instance/3245021/bin/gkr1127f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3245021/" ref="ordinalpos=3767&amp;ncbi_uid=3747495&amp;link_uid=PMC3245021" image-link="/pmc/articles/PMC3245021/figure/gkr1127-F3/" class="imagepopup">Figure 3. From: hiPathDB: a human-integrated <span class="highlight" style="background-color:">pathway</span> database with facile visualization. </a></p><div class="supp"><p class="details"><div>Reformatted modeling of the BioCarta apoptotic <span class="highlight" style="background-color:">signaling</span> in response to DNA damage <span class="highlight" style="background-color:">pathway</span>. Note that+andnext to the gene circles indicate the active and inactive forms, respectively. Small red ovals indicate translocation, and the new location of protein is provided next to the oval (yellow box, biological processes; blue circle, gene or protein; gray circle, pseudo node; red circle, complex node).</div></p></div><div class="aux"><div class="resc">Namhee Yu, et al. Nucleic Acids Res. 2012 January;40(D1):D797-D802.</div><p class="links"><a class="dblinks" href="/pubmed/22123737" ref="ordinalpos=3767&amp;ncbi_uid=3747495&amp;link_uid=22123737">Citation</a><a class="dblinks" href="/pmc/articles/PMC3245021/" ref="ordinalpos=3767&amp;ncbi_uid=3747495&amp;link_uid=PMC3245021">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3768.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3004666/figure/f1-cmr-1-079/" ref="ordinalpos=3768&amp;ncbi_uid=2645631&amp;link_uid=PMC3004666" href="/pmc/articles/PMC3004666/"><img src="/pmc/articles/instance/3004666/bin/cmr-1-079f1.gif" src-large="/pmc/articles/instance/3004666/bin/cmr-1-079f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3004666/" ref="ordinalpos=3768&amp;ncbi_uid=2645631&amp;link_uid=PMC3004666" image-link="/pmc/articles/PMC3004666/figure/f1-cmr-1-079/" class="imagepopup">Figure 1.  From: Evolving role of cetuximab in the treatment of colorectal cancer. </a></p><div class="supp"><p class="details"><div>Simplified EGFR-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Ligands are EGF, TGF-, ER, and AR. The two main pathways are the PI3K/AKT/mTOR and RAS/RAF/MAPK cascades. The IGF1 receptor can dimerize with EGFR and activate EGFR <span class="highlight" style="background-color:">signaling</span>. Alternatively, downstream crosstalk between <span class="highlight" style="background-color:">signaling</span> cascades may occur. Alterations of downstream effectors known to interfere with EGFR <span class="highlight" style="background-color:">signaling</span> are marked in red.<br /><b>Abbreviations:</b> AR, amphiregulin; EGF, epidermal growth factor; EGFR, epidermal growth factor receptor; ER, epiregulin; IGF1, insulin-like growth factor-1; TGF-, transforming growth factor-.</div></p></div><div class="aux"><div class="resc">Gunter Schuch, et al. Cancer Manag Res. 2009;1:79-88.</div><p class="links"><a class="dblinks" href="/pubmed/21188126" ref="ordinalpos=3768&amp;ncbi_uid=2645631&amp;link_uid=21188126">Citation</a><a class="dblinks" href="/pmc/articles/PMC3004666/" ref="ordinalpos=3768&amp;ncbi_uid=2645631&amp;link_uid=PMC3004666">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3769.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2258762/figure/f7/" ref="ordinalpos=3769&amp;ncbi_uid=1243405&amp;link_uid=PMC2258762" href="/pmc/articles/PMC2258762/"><img src="/pmc/articles/instance/2258762/bin/zmb0050873380007.gif" src-large="/pmc/articles/instance/2258762/bin/zmb0050873380007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2258762/" ref="ordinalpos=3769&amp;ncbi_uid=1243405&amp;link_uid=PMC2258762" image-link="/pmc/articles/PMC2258762/figure/f7/" class="imagepopup">FIG. 7. From: Wingless <span class="highlight" style="background-color:">Signaling</span> Induces Widespread Chromatin Remodeling of Target Loci . </a></p><div class="supp"><p class="details"><div>The <i>Notum</i> locus is widely acetylated in response to Wg <span class="highlight" style="background-color:">signaling</span> in vivo. Da-Gal4 was used to drive expression of Wg (Da/Wg) or GPI-dFz2 (Da/GPI-dFz2) to ubiquitously activate or inhibit the <span class="highlight" style="background-color:">pathway</span> in embryos, respectively. (A) ChIP analysis indicates that TCF binds to the upstream <i>Notum</i> WRE region when the <span class="highlight" style="background-color:">pathway</span> is activated (black line) and inhibited (dashed line) but not the region containing the intronic WRE. (B) AcH3 ChIP analysis shows widespread increased acetylation across the <i>Notum</i> locus in response to Wg <span class="highlight" style="background-color:">signaling</span> (compare dashed and black lines). Each data point represents the mean of two independent ChIP samples, with the error bars indicating standard deviations. All experiments were performed multiple times, and the data shown are representative.</div></p></div><div class="aux"><div class="resc">David S. Parker, et al. Mol Cell Biol. 2008 March;28(5):1815-1828.</div><p class="links"><a class="dblinks" href="/pubmed/18160704" ref="ordinalpos=3769&amp;ncbi_uid=1243405&amp;link_uid=18160704">Citation</a><a class="dblinks" href="/pmc/articles/PMC2258762/" ref="ordinalpos=3769&amp;ncbi_uid=1243405&amp;link_uid=PMC2258762">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3770.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC309682/figure/f6/" ref="ordinalpos=3770&amp;ncbi_uid=362738&amp;link_uid=PMC309682" href="/pmc/articles/PMC309682/"><img src="/pmc/articles/instance/309682/bin/mb2430944006.gif" src-large="/pmc/articles/instance/309682/bin/mb2430944006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC309682/" ref="ordinalpos=3770&amp;ncbi_uid=362738&amp;link_uid=PMC309682" image-link="/pmc/articles/PMC309682/figure/f6/" class="imagepopup">FIG. 6. From: <span class="highlight" style="background-color:">Signaling</span> from Akt to FRAP/TOR Targets both 4E-BP and S6K in Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div>Models of the insulin-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>Drosophila</i> and regulation of d4E-BP by phosphorylation. (A) A common <span class="highlight" style="background-color:">pathway</span> regulates 4E-BP and S6K. The regulation of 4E-BP is dependent on <span class="highlight" style="background-color:">signaling</span> from Inr-PI3K-Akt-TSC and TOR. The regulation of S6K is also effected by PDK1 and the same upstream elements. The different size of the Akt typeface is to illustrate that the intensity of <span class="highlight" style="background-color:">signaling</span> toward TSC affects 4E-BP and S6K differentially. (B) The pattern of phosphorylation of d4E-BP as observed by IEF-SDS-PAGE may result from one of three possible models. (i) In the primed model, d4E-BP is already phosphorylated at Thr37 and is subsequently phosphorylated on one additional site, Thr46, after insulin stimulation. (ii) In the sequential model, d4E-BP is phosphorylated on Thr37 and then Thr46 (or vice versa). (iii) In the coordinated model, d4E-BP is phosphorylated coordinately on Thr37 and Thr46.</div></p></div><div class="aux"><div class="resc">Mathieu Miron, et al. Mol Cell Biol. 2003 December;23(24):9117-9126.</div><p class="links"><a class="dblinks" href="/pubmed/14645523" ref="ordinalpos=3770&amp;ncbi_uid=362738&amp;link_uid=14645523">Citation</a><a class="dblinks" href="/pmc/articles/PMC309682/" ref="ordinalpos=3770&amp;ncbi_uid=362738&amp;link_uid=PMC309682">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3771.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3057274/figure/F7/" ref="ordinalpos=3771&amp;ncbi_uid=3416898&amp;link_uid=PMC3057274" href="/pmc/articles/PMC3057274/"><img src="/pmc/articles/instance/3057274/bin/nihms269294f7.gif" src-large="/pmc/articles/instance/3057274/bin/nihms269294f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3057274/" ref="ordinalpos=3771&amp;ncbi_uid=3416898&amp;link_uid=PMC3057274" image-link="/pmc/articles/PMC3057274/figure/F7/" class="imagepopup">Figure 7. SmoM2 expressing cells have active Hh <span class="highlight" style="background-color:">signaling</span> and downregulation of the Notch <span class="highlight" style="background-color:">pathway</span>.  From: Altered differentiation and paracrine stimulation of mammary epithelial cell proliferation by conditionally activated Smoothened. </a></p><div class="supp"><p class="details"><div>Representative FACS plot illustrating EGFP expressing (recombined) SmoM2;Cre MECs and Tomato Red expressing (un-recombined) SmoM2;Cre MEC populations (A). Quantitative RT-PCR illustrating the 13-fold increased expression of SmoM2 (p=.03) in EGFP expressing (recombined) SmoM2;Cre MECs as compared to Tomato Red expressing (un-recombined) SmoM2;Cre MECs (B). Hh <span class="highlight" style="background-color:">pathway</span> gene expression in sorted cell populations illustrating induction of Ptch2 (p=0.04), Gli-1 (p=0.04), and Gli-2 (p=0.03) in EGFP expressing (recombined) SmoM2;Cre MECs as compared to wildtype SmoM2;+ MECs (C). Notch <span class="highlight" style="background-color:">pathway</span> gene expression in sorted cell populations illustrating the 23-fold increased expression of Dll-1 (p=0.02), the 5-fold increased expression of Jag2 (p&lt;0.01), and the 18-fold downregulation of Notch4 (p&lt;0.0001), the 14-fold downregulation of Dll-4 (p=0.03) and the 7-fold downregulation of Hes6 (p=0.03) in EGFP expressing (recombined) SmoM2;Cre MECs as compared to wildtype SmoM2;+ MECs.</div></p></div><div class="aux"><div class="resc">Adriana P. Visbal, et al. Dev Biol. ;352(1):116-127.</div><p class="links"><a class="dblinks" href="/pubmed/21276786" ref="ordinalpos=3771&amp;ncbi_uid=3416898&amp;link_uid=21276786">Citation</a><a class="dblinks" href="/pmc/articles/PMC3057274/" ref="ordinalpos=3771&amp;ncbi_uid=3416898&amp;link_uid=PMC3057274">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3772.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2795794/figure/F6/" ref="ordinalpos=3772&amp;ncbi_uid=2105042&amp;link_uid=PMC2795794" href="/pmc/articles/PMC2795794/"><img src="/pmc/articles/instance/2795794/bin/nihms139264f6.gif" src-large="/pmc/articles/instance/2795794/bin/nihms139264f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2795794/" ref="ordinalpos=3772&amp;ncbi_uid=2105042&amp;link_uid=PMC2795794" image-link="/pmc/articles/PMC2795794/figure/F6/" class="imagepopup">Figure 6. Analysis of reactive stroma and hedgehog <span class="highlight" style="background-color:">signaling</span> in human prostate tissue.  From: The Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span> Stimulates Prostate Tumor Growth by Paracrine <span class="highlight" style="background-color:">Signaling</span> and Recaptures Embryonic Gene Expression in Tumor Myofibroblasts. </a></p><div class="supp"><p class="details"><div>The relationship of reactive stroma and Hh <span class="highlight" style="background-color:">signaling</span> was analyzed in prostate using Pearson product-moment correlation. The samples were divided into 4 groups according to benign/tumor and reactive/non-reactive. The numbers listed are correlation p values. Values less than 0.05 are highlighted in red. All correlations in red are positive correlations (data not shown).</div></p></div><div class="aux"><div class="resc">Aubie Shaw, et al. Oncogene. ;28(50):4480-4490.</div><p class="links"><a class="dblinks" href="/pubmed/19784071" ref="ordinalpos=3772&amp;ncbi_uid=2105042&amp;link_uid=19784071">Citation</a><a class="dblinks" href="/pmc/articles/PMC2795794/" ref="ordinalpos=3772&amp;ncbi_uid=2105042&amp;link_uid=PMC2795794">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3773.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2615406/figure/F9/" ref="ordinalpos=3773&amp;ncbi_uid=1686745&amp;link_uid=PMC2615406" href="/pmc/articles/PMC2615406/"><img src="/pmc/articles/instance/2615406/bin/nihms71041f9.gif" src-large="/pmc/articles/instance/2615406/bin/nihms71041f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2615406/" ref="ordinalpos=3773&amp;ncbi_uid=1686745&amp;link_uid=PMC2615406" image-link="/pmc/articles/PMC2615406/figure/F9/" class="imagepopup">Fig. 9. Scheme for the cell <span class="highlight" style="background-color:">signaling</span> response to ischemia.  From: CAVEOLAE ARE AN ESSENTIAL COMPONENT OF THE <span class="highlight" style="background-color:">PATHWAY</span> FOR ENDOTHELIAL CELL <span class="highlight" style="background-color:">SIGNALING</span> ASSOCIATED WITH ABRUPT REDUCTION OF SHEAR STRESS. </a></p><div class="supp"><p class="details"><div>Altered shear stress is sensed by caveolae resulting in K<sub>ATP</sub> channel closure and cell membrane depolarization. Subsequent activation of NADPH oxidase (NOX2) and downstream <span class="highlight" style="background-color:">signaling</span> pathways results in cell proliferation which may be a compensatory mechanism to restore perfusion.</div></p></div><div class="aux"><div class="resc">Tatyana Milovanova, et al. Biochim Biophys Acta. ;1783(10):1866-1875.</div><p class="links"><a class="dblinks" href="/pubmed/18573285" ref="ordinalpos=3773&amp;ncbi_uid=1686745&amp;link_uid=18573285">Citation</a><a class="dblinks" href="/pmc/articles/PMC2615406/" ref="ordinalpos=3773&amp;ncbi_uid=1686745&amp;link_uid=PMC2615406">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3774.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2629278/figure/F1/" ref="ordinalpos=3774&amp;ncbi_uid=1647308&amp;link_uid=PMC2629278" href="/pmc/articles/PMC2629278/"><img src="/pmc/articles/instance/2629278/bin/zpq9990963490001.gif" src-large="/pmc/articles/instance/2629278/bin/zpq9990963490001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2629278/" ref="ordinalpos=3774&amp;ncbi_uid=1647308&amp;link_uid=PMC2629278" image-link="/pmc/articles/PMC2629278/figure/F1/" class="imagepopup">Fig. 1. From: How CYCLOPS keeps an eye on plant symbiosis. </a></p><div class="supp"><p class="details"><div>Two alternative models for the role of <i>CYCLOPS</i> in symbiosis <span class="highlight" style="background-color:">signaling</span>. (<i>A</i>) The model of the Sym <span class="highlight" style="background-color:">pathway</span> according to Yano <i>et al.</i> (1). Symbiotic signals are perceived by appropriate receptor complexes, activation of which will lead to a <span class="highlight" style="background-color:">signaling</span> cascade within the Sym <span class="highlight" style="background-color:">pathway</span>. Because <i>cyclops</i> mutants show spontaneous nodulation it is interpreted that <i>CYCLOPS</i> acts as a branching point in the <span class="highlight" style="background-color:">pathway</span>, where fungal and bacterial infection are regulated in a <i>CYCLOPS</i>-dependent manner and nodule organogenesis is regulated in a <i>CYCLOPS</i>-independent manner. This finding suggests that <i>NSP1</i>, <i>NSP2</i>, and <i>ERN1</i> transcription factors have dual roles in both the bacterial infection branch and the nodule organogenesis branch as mutants in these genes are impaired in both. (<i>B</i>) An alternative model for the Sym <span class="highlight" style="background-color:">pathway</span>. Because <i>cyclops</i> mutants show greatly reduced levels of spontaneous nodulation it is interpreted that <i>CYCLOPS</i> is required for nodule organogenesis, but that the <i>cyclops</i> mutants have low penetrance. Low penetrance allows a more parsimonious Sym <span class="highlight" style="background-color:">pathway</span> in which <i>CYCLOPS</i> has analogous functions to all other members of the Sym <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> bifurcates into nodulation-specific and mycorrhization-specific branches, each governed by their own set of transcription factors. NF, Nod factor. Pictures of a root nodule on <i>L. japonicus</i> and an arbuscule of <i>Glomus versiforme</i> in an <i>M. truncatula</i> root cortical cell stained with WGA Alexa Fluor 488 are shown at right.</div></p></div><div class="aux"><div class="resc">Ward Capoen, et al. Proc Natl Acad Sci U S A. 2008 December 23;105(51):20053-20054.</div><p class="links"><a class="dblinks" href="/pubmed/19091956" ref="ordinalpos=3774&amp;ncbi_uid=1647308&amp;link_uid=19091956">Citation</a><a class="dblinks" href="/pmc/articles/PMC2629278/" ref="ordinalpos=3774&amp;ncbi_uid=1647308&amp;link_uid=PMC2629278">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3775.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4166520/figure/F1/" ref="ordinalpos=3775&amp;ncbi_uid=6618916&amp;link_uid=PMC4166520" href="/pmc/articles/PMC4166520/"><img src="/pmc/articles/instance/4166520/bin/nihms621484f1.gif" src-large="/pmc/articles/instance/4166520/bin/nihms621484f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4166520/" ref="ordinalpos=3775&amp;ncbi_uid=6618916&amp;link_uid=PMC4166520" image-link="/pmc/articles/PMC4166520/figure/F1/" class="imagepopup">Figure 1. Synthetic pathways for LPA and major LPA <span class="highlight" style="background-color:">signaling</span> pathways connecting inflammation and cancer.  From: ATX-LPA Receptor Axis in Inflammation and Cancer. </a></p><div class="supp"><p class="details"><div>ATX hydrolyzes lysophospholipids, in particular lysophosphatidylcholine (LPC), to produce bioactive LPA. Newly produced LPA acts on its own GPCRs via at least three distinct classes of heterotrimeric G proteins  Gq, Gi and G12/13 activating multiple downstream pathways and evoke its biological effects, including RAS-ERK <span class="highlight" style="background-color:">pathway</span> through Gi and Gq; PI3K-AKT <span class="highlight" style="background-color:">pathway</span> through Gi; PKC GSK30  -catenin <span class="highlight" style="background-color:">pathway</span> through Gq (and /or Gi); Rho-CDC42 <span class="highlight" style="background-color:">pathway</span> through G12/13; Src-Stat <span class="highlight" style="background-color:">pathway</span> through Gi, which induce expression and activation of multiple transcriptors including STAT3, NF-B, ATF-2, which induce cell proliferation and production of cytokines. The cytokines bind to their receptors inducing Stat 3 and 5 production and activation.</div></p></div><div class="aux"><div class="resc">Shuying Liu, et al. Cell Cycle. ;8(22):3695-3701.</div><p class="links"><a class="dblinks" href="/pubmed/19855166" ref="ordinalpos=3775&amp;ncbi_uid=6618916&amp;link_uid=19855166">Citation</a><a class="dblinks" href="/pmc/articles/PMC4166520/" ref="ordinalpos=3775&amp;ncbi_uid=6618916&amp;link_uid=PMC4166520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3776.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4046017/figure/F4/" ref="ordinalpos=3776&amp;ncbi_uid=6201833&amp;link_uid=PMC4046017" href="/pmc/articles/PMC4046017/"><img src="/pmc/articles/instance/4046017/bin/1297-9716-45-54-2.gif" src-large="/pmc/articles/instance/4046017/bin/1297-9716-45-54-2.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4046017/" ref="ordinalpos=3776&amp;ncbi_uid=6201833&amp;link_uid=PMC4046017" image-link="/pmc/articles/PMC4046017/figure/F4/" class="imagepopup">Figure 4.  From: From mouth to macrophage: mechanisms of innate immune subversion by Mycobacterium avium subsp. paratuberculosis. </a></p><div class="supp"><p class="details"><div><b>Inhibition of IFNg <span class="highlight" style="background-color:">signaling</span> by MAP.</b> MAP infection inhibits JAK-STAT <span class="highlight" style="background-color:">signaling</span> via increased expression of negative regulators of the IFN receptor, SOCS1 and SOCS3, as well as decreased expression of IFN receptor chains 1 and 2. These modifications of cellular responsiveness occurred in temporal fashion with early induction of SOCS1 and 3 with subsequent repression of the IFN receptor. MAP inhibition of IFN responsiveness may be mediated by MAP effector molecule PtpB. PtpB may disrupt the normal IFN receptor immune <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by dephosphorylating a key <span class="highlight" style="background-color:">signaling</span> intermediate, or perhaps the receptor itself.</div></p></div><div class="aux"><div class="resc">Ryan J Arsenault, et al. Vet Res. 2014;45(1):54-54.</div><p class="links"><a class="dblinks" href="/pubmed/24885748" ref="ordinalpos=3776&amp;ncbi_uid=6201833&amp;link_uid=24885748">Citation</a><a class="dblinks" href="/pmc/articles/PMC4046017/" ref="ordinalpos=3776&amp;ncbi_uid=6201833&amp;link_uid=PMC4046017">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3777.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC37350/figure/F10/" ref="ordinalpos=3777&amp;ncbi_uid=91111&amp;link_uid=PMC37350" href="/pmc/articles/PMC37350/"><img src="/pmc/articles/instance/37350/bin/mk0611533010.gif" src-large="/pmc/articles/instance/37350/bin/mk0611533010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC37350/" ref="ordinalpos=3777&amp;ncbi_uid=91111&amp;link_uid=PMC37350" image-link="/pmc/articles/PMC37350/figure/F10/" class="imagepopup">Figure 10.  From: Regulation of Epidermal Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span> by  Endocytosis and Intracellular Trafficking. </a></p><div class="supp"><p class="details"><div>Summary of trafficking and <span class="highlight" style="background-color:">signaling</span> of EGFR in  human mammary epithelial cells. Shown is a simplified diagram of the  endocytic <span class="highlight" style="background-color:">pathway</span>. Listed below is the time period during which the  EGFR traverses the different compartments after EGF addition as  determined by colocalization studies. The association of different  substrates with the EGFR is based on results from both 184A1 and HB2  cells.</div></p></div><div class="aux"><div class="resc">Patrick Burke, et al. Mol Biol Cell. 2001 June;12(6):1897-1910.</div><p class="links"><a class="dblinks" href="/pubmed/11408594" ref="ordinalpos=3777&amp;ncbi_uid=91111&amp;link_uid=11408594">Citation</a><a class="dblinks" href="/pmc/articles/PMC37350/" ref="ordinalpos=3777&amp;ncbi_uid=91111&amp;link_uid=PMC37350">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3778.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3897622/figure/pone-0086103-g006/" ref="ordinalpos=3778&amp;ncbi_uid=5738480&amp;link_uid=PMC3897622" href="/pmc/articles/PMC3897622/"><img src="/pmc/articles/instance/3897622/bin/pone.0086103.g006.gif" src-large="/pmc/articles/instance/3897622/bin/pone.0086103.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3897622/" ref="ordinalpos=3778&amp;ncbi_uid=5738480&amp;link_uid=PMC3897622" image-link="/pmc/articles/PMC3897622/figure/pone-0086103-g006/" class="imagepopup">Figure 6. Summary of findings and proposed mechanism..  From: p120RasGAP Is a Mediator of Rho <span class="highlight" style="background-color:">Pathway</span> Activation and Tumorigenicity in the DLD1 Colorectal Cancer Cell Line. </a></p><div class="supp"><p class="details"><div>In DLD1 cells, active KRAS stabilizes RasGAP expression, which in turn binds to and stabilizes RhoGAP phosphorylation. This complex then activates Rho <span class="highlight" style="background-color:">pathway</span> activation, either by sequestration of RhoGAP away from Rho, or by increasing Rho turnover. In DKO4 cells, RasGAP is not expressed, due in part to a truncating mutation and in part to lack of expression downstream of active KRAS. In this situation, RhoGAP is not phosphorylated, and so Rho <span class="highlight" style="background-color:">pathway</span> phenotypes are inactive. When RasGAP is overexpressed in DKO4, RhoGAP is once again phosphorylated and Rho <span class="highlight" style="background-color:">pathway</span> is active- however, lack of stabilization and/or contributing <span class="highlight" style="background-color:">signaling</span> pathways from active RAS means that tumorigenicity does not attain the same level as DLD1. The bottom of the figure summarizes the main characteristics of RhoGAP and RasGAP in these cell lines.</div></p></div><div class="aux"><div class="resc">Shawna L. Organ, et al. PLoS One. 2014;9(1):e86103.</div><p class="links"><a class="dblinks" href="/pubmed/24465899" ref="ordinalpos=3778&amp;ncbi_uid=5738480&amp;link_uid=24465899">Citation</a><a class="dblinks" href="/pmc/articles/PMC3897622/" ref="ordinalpos=3778&amp;ncbi_uid=5738480&amp;link_uid=PMC3897622">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3779.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2890584/figure/pone-0011278-g006/" ref="ordinalpos=3779&amp;ncbi_uid=2347939&amp;link_uid=PMC2890584" href="/pmc/articles/PMC2890584/"><img src="/pmc/articles/instance/2890584/bin/pone.0011278.g006.gif" src-large="/pmc/articles/instance/2890584/bin/pone.0011278.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2890584/" ref="ordinalpos=3779&amp;ncbi_uid=2347939&amp;link_uid=PMC2890584" image-link="/pmc/articles/PMC2890584/figure/pone-0011278-g006/" class="imagepopup">Figure 6. A diagram summarizing the proposed connection between MOR and NMDAR and the role of the Akt-nNOS <span class="highlight" style="background-color:">pathway</span> in cross-regulation..  From: Mu-Opioid Receptors Transiently Activate the Akt-nNOS <span class="highlight" style="background-color:">Pathway</span> to Produce Sustained Potentiation of PKC-Mediated NMDAR-CaMKII <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>See Discussion for details.</div></p></div><div class="aux"><div class="resc">Pilar Snchez-Blzquez, et al. PLoS One. 2010;5(6):e11278.</div><p class="links"><a class="dblinks" href="/pubmed/20585660" ref="ordinalpos=3779&amp;ncbi_uid=2347939&amp;link_uid=20585660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2890584/" ref="ordinalpos=3779&amp;ncbi_uid=2347939&amp;link_uid=PMC2890584">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3780.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3534691/figure/pone-0053444-g008/" ref="ordinalpos=3780&amp;ncbi_uid=4509988&amp;link_uid=PMC3534691" href="/pmc/articles/PMC3534691/"><img src="/pmc/articles/instance/3534691/bin/pone.0053444.g008.gif" src-large="/pmc/articles/instance/3534691/bin/pone.0053444.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3534691/" ref="ordinalpos=3780&amp;ncbi_uid=4509988&amp;link_uid=PMC3534691" image-link="/pmc/articles/PMC3534691/figure/pone-0053444-g008/" class="imagepopup">Figure 8. Schematic diagram showing the NRG-erbB-FAK <span class="highlight" style="background-color:">signaling</span> in the repairing process following peripheral nerve injury (PNI)..  From: Neuregulin Facilitates Nerve Regeneration by Speeding Schwann Cell Migration via ErbB2/3-Dependent FAK <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Following PNI [as created by end-to-side neurorraphy (ESN)], NRG would activate the erbB2-FAK <span class="highlight" style="background-color:">pathway</span> that promotes the Schwann cells to migrate to the distal stump for successful axonal guidance and nerve regeneration. Un: ulnar nerve; Mc: musculocutaneous nerve; ECM: extracellular matrix.</div></p></div><div class="aux"><div class="resc">Hung-Ming Chang, et al. PLoS One. 2013;8(1):e53444.</div><p class="links"><a class="dblinks" href="/pubmed/23301073" ref="ordinalpos=3780&amp;ncbi_uid=4509988&amp;link_uid=23301073">Citation</a><a class="dblinks" href="/pmc/articles/PMC3534691/" ref="ordinalpos=3780&amp;ncbi_uid=4509988&amp;link_uid=PMC3534691">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3781.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3653345/figure/fig07/" ref="ordinalpos=3781&amp;ncbi_uid=6083011&amp;link_uid=PMC3653345" href="/pmc/articles/PMC3653345/"><img src="/pmc/articles/instance/3653345/bin/chest_143_5_1252_fig07.gif" src-large="/pmc/articles/instance/3653345/bin/chest_143_5_1252_fig07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3653345/" ref="ordinalpos=3781&amp;ncbi_uid=6083011&amp;link_uid=PMC3653345" image-link="/pmc/articles/PMC3653345/figure/fig07/" class="imagepopup">Figure 7. From: Lumican Regulates Ventilation-Induced Epithelial-Mesenchymal Transition Through Extracelluar Signal-Regulated Kinase <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic figure illustrating the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation with high tidal volume mechanical ventilation. Mechanical stretch-induced cytokine production and EMT were attenuated with lum<sup>/</sup> mice and pharmacologic inhibition of ERK1/2 activity by PD98059. EMT = epithelial-mesenchymal transition; VILI = ventilator-induced lung injury. See Figure 1 and 6 legends for expansion of other abbreviations.</div></p></div><div class="aux"><div class="resc">Li-Fu Li, et al. Chest. 2013 May;143(5):1252-1260.</div><p class="links"><a class="dblinks" href="/pubmed/23154825" ref="ordinalpos=3781&amp;ncbi_uid=6083011&amp;link_uid=23154825">Citation</a><a class="dblinks" href="/pmc/articles/PMC3653345/" ref="ordinalpos=3781&amp;ncbi_uid=6083011&amp;link_uid=PMC3653345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3782.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4303155/figure/fig02/" ref="ordinalpos=3782&amp;ncbi_uid=7091827&amp;link_uid=PMC4303155" href="/pmc/articles/PMC4303155/"><img src="/pmc/articles/instance/4303155/bin/cam40003-0878-f2.gif" src-large="/pmc/articles/instance/4303155/bin/cam40003-0878-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4303155/" ref="ordinalpos=3782&amp;ncbi_uid=7091827&amp;link_uid=PMC4303155" image-link="/pmc/articles/PMC4303155/figure/fig02/" class="imagepopup">Figure 2.  From: A systematic review of microRNA expression profiling studies in human gastric cancer. </a></p><div class="supp"><p class="details"><div>MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing the target genes of microRNAs.</div></p></div><div class="aux"><div class="resc">Sirjana Shrestha, et al. Cancer Med. 2014 August;3(4):878-888.</div><p class="links"><a class="dblinks" href="/pubmed/24902858" ref="ordinalpos=3782&amp;ncbi_uid=7091827&amp;link_uid=24902858">Citation</a><a class="dblinks" href="/pmc/articles/PMC4303155/" ref="ordinalpos=3782&amp;ncbi_uid=7091827&amp;link_uid=PMC4303155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3783.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4158168/figure/fig6/" ref="ordinalpos=3783&amp;ncbi_uid=6606564&amp;link_uid=PMC4158168" href="/pmc/articles/PMC4158168/"><img src="/pmc/articles/instance/4158168/bin/MI2014-973986.006.gif" src-large="/pmc/articles/instance/4158168/bin/MI2014-973986.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4158168/" ref="ordinalpos=3783&amp;ncbi_uid=6606564&amp;link_uid=PMC4158168" image-link="/pmc/articles/PMC4158168/figure/fig6/" class="imagepopup">Figure 6.  From: Metformin Attenuates Experimental Autoimmune Arthritis through Reciprocal Regulation of Th17/Treg Balance and Osteoclastogenesis. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that metformin uses to regulate T cell and osteoclasts in autoimmune arthritis.</div></p></div><div class="aux"><div class="resc">Hye-Jin Son, et al. Mediators Inflamm. 2014;2014:973986.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3783&amp;ncbi_uid=6606564&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4158168/" ref="ordinalpos=3783&amp;ncbi_uid=6606564&amp;link_uid=PMC4158168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3784.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3105031/figure/pone-0020362-g006/" ref="ordinalpos=3784&amp;ncbi_uid=3005440&amp;link_uid=PMC3105031" href="/pmc/articles/PMC3105031/"><img src="/pmc/articles/instance/3105031/bin/pone.0020362.g006.gif" src-large="/pmc/articles/instance/3105031/bin/pone.0020362.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3105031/" ref="ordinalpos=3784&amp;ncbi_uid=3005440&amp;link_uid=PMC3105031" image-link="/pmc/articles/PMC3105031/figure/pone-0020362-g006/" class="imagepopup">Figure 6. Proposed model of the p53miR-34E2F1h-eag1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Transcriptional and Post-Transcriptional Mechanisms for Oncogenic Overexpression of Ether  Go-Go K+ Channel. </a></p><div class="supp"><p class="details"><div>RA: retinoic acid, which has been shown to enhance <i>miR-34</i> expression; E2F1/3: E2F1 and E2F3.</div></p></div><div class="aux"><div class="resc">Huixian Lin, et al. PLoS One. 2011;6(5):e20362.</div><p class="links"><a class="dblinks" href="/pubmed/21655246" ref="ordinalpos=3784&amp;ncbi_uid=3005440&amp;link_uid=21655246">Citation</a><a class="dblinks" href="/pmc/articles/PMC3105031/" ref="ordinalpos=3784&amp;ncbi_uid=3005440&amp;link_uid=PMC3105031">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3785.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2888945/figure/F3/" ref="ordinalpos=3785&amp;ncbi_uid=2336144&amp;link_uid=PMC2888945" href="/pmc/articles/PMC2888945/"><img src="/pmc/articles/instance/2888945/bin/nihms136601f3.gif" src-large="/pmc/articles/instance/2888945/bin/nihms136601f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2888945/" ref="ordinalpos=3785&amp;ncbi_uid=2336144&amp;link_uid=PMC2888945" image-link="/pmc/articles/PMC2888945/figure/F3/" class="imagepopup">Figure 3.  From: Antitumor Agents 268. Design, Synthesis, and Mechanistic Studies of New 9-Substituted Phenanthrene-based Tylophorine Analogs as Potent Cytotoxic Agents. </a></p><div class="supp"><p class="details"><div>Inhibitory effects of PBT analogues on NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in A549 and CL1-0 cells</div></p></div><div class="aux"><div class="resc">Xiaoming Yang, et al. J Med Chem. ;52(16):5262-5268.</div><p class="links"><a class="dblinks" href="/pubmed/19645447" ref="ordinalpos=3785&amp;ncbi_uid=2336144&amp;link_uid=19645447">Citation</a><a class="dblinks" href="/pmc/articles/PMC2888945/" ref="ordinalpos=3785&amp;ncbi_uid=2336144&amp;link_uid=PMC2888945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3786.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2681376/figure/f7/" ref="ordinalpos=3786&amp;ncbi_uid=1872606&amp;link_uid=PMC2681376" href="/pmc/articles/PMC2681376/"><img src="/pmc/articles/instance/2681376/bin/zh00050959230007.gif" src-large="/pmc/articles/instance/2681376/bin/zh00050959230007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2681376/" ref="ordinalpos=3786&amp;ncbi_uid=1872606&amp;link_uid=PMC2681376" image-link="/pmc/articles/PMC2681376/figure/f7/" class="imagepopup">Fig. 7. From: Regulation of cardiac fibroblast collagen synthesis by adenosine: roles for Epac and PI3K. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for adenosine-mediated inhibition of collagen production. See text for more information.</div></p></div><div class="aux"><div class="resc">Francisco Villarreal, et al. Am J Physiol Cell Physiol. 2009 May;296(5):C1178-C1184.</div><p class="links"><a class="dblinks" href="/pubmed/19279233" ref="ordinalpos=3786&amp;ncbi_uid=1872606&amp;link_uid=19279233">Citation</a><a class="dblinks" href="/pmc/articles/PMC2681376/" ref="ordinalpos=3786&amp;ncbi_uid=1872606&amp;link_uid=PMC2681376">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3787.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2645926/figure/F11/" ref="ordinalpos=3787&amp;ncbi_uid=1681278&amp;link_uid=PMC2645926" href="/pmc/articles/PMC2645926/"><img src="/pmc/articles/instance/2645926/bin/nihms86471f11.gif" src-large="/pmc/articles/instance/2645926/bin/nihms86471f11.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2645926/" ref="ordinalpos=3787&amp;ncbi_uid=1681278&amp;link_uid=PMC2645926" image-link="/pmc/articles/PMC2645926/figure/F11/" class="imagepopup">Figure 11.  From: Phorbol-12-myristate 13-acetate acting through protein kinase C induces translocator protein (18-kDa) Tspo gene expression. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> underlying PMA-induced activation of the <i>Tspo</i> promoter.</div></p></div><div class="aux"><div class="resc">Amani Batarseh, et al. Biochemistry. ;47(48):12886-12899.</div><p class="links"><a class="dblinks" href="/pubmed/18975922" ref="ordinalpos=3787&amp;ncbi_uid=1681278&amp;link_uid=18975922">Citation</a><a class="dblinks" href="/pmc/articles/PMC2645926/" ref="ordinalpos=3787&amp;ncbi_uid=1681278&amp;link_uid=PMC2645926">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3788.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1828115/figure/F7/" ref="ordinalpos=3788&amp;ncbi_uid=996783&amp;link_uid=PMC1828115" href="/pmc/articles/PMC1828115/"><img src="/pmc/articles/instance/1828115/bin/nihms16533f7.gif" src-large="/pmc/articles/instance/1828115/bin/nihms16533f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1828115/" ref="ordinalpos=3788&amp;ncbi_uid=996783&amp;link_uid=PMC1828115" image-link="/pmc/articles/PMC1828115/figure/F7/" class="imagepopup">Fig. 7.  From: The role of Axin2 in calvarial morphogenesis and craniosynostosis. </a></p><div class="supp"><p class="details"><div>Activation of  -catenin and LEF/TCF <span class="highlight" style="background-color:">signaling</span> in skull development. Mice carrying the TOPGAL transgene, capable of responding to  -catenin and LEF/TCF <span class="highlight" style="background-color:">signaling</span>, were used to monitor activation of this <span class="highlight" style="background-color:">pathway</span> by  -gal staining analyses in cranial skulls at E16.5 (C,E) and P1 (D,F). Whole-mount staining shows activation of the canonical Wnt <span class="highlight" style="background-color:">pathway</span> during early skull formation (CF). Arrowheads indicate the coronal sutures and arrows indicate the activation of  -catenin and LEF/TCF <span class="highlight" style="background-color:">signaling</span> in medial parts of the nasal bones. The non-transgenic mice, with no background staining, indicate the specificity of the assay (A,B). Scale bars: 2 mm in AD; 500  m in E,F.</div></p></div><div class="aux"><div class="resc">Hsiao-Man Ivy Yu, et al. Development. ;132(8):1995-2005.</div><p class="links"><a class="dblinks" href="/pubmed/15790973" ref="ordinalpos=3788&amp;ncbi_uid=996783&amp;link_uid=15790973">Citation</a><a class="dblinks" href="/pmc/articles/PMC1828115/" ref="ordinalpos=3788&amp;ncbi_uid=996783&amp;link_uid=PMC1828115">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3789.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC199181/figure/F3/" ref="ordinalpos=3789&amp;ncbi_uid=246197&amp;link_uid=PMC199181" href="/pmc/articles/PMC199181/"><img src="/pmc/articles/instance/199181/bin/JCI0111918.f3.gif" src-large="/pmc/articles/instance/199181/bin/JCI0111918.f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC199181/" ref="ordinalpos=3789&amp;ncbi_uid=246197&amp;link_uid=PMC199181" image-link="/pmc/articles/PMC199181/figure/F3/" class="imagepopup">Figure 3.  From: Hostile takeovers: viral appropriation of the NF-kB <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Modulation of the NF-B <span class="highlight" style="background-color:">pathway</span> by viruses. Mechanisms of activation of the NF-B <span class="highlight" style="background-color:">pathway</span> by HTLV-1Tax protein include direct association of Tax and IKK- or latent NF-B/IB complexes. EBV LMP1mediated <span class="highlight" style="background-color:">signaling</span> results in the recruitment of TRADD, which interacts with TRAF2 and RIP kinase to activate NF-B. TRAF2 potentially activates NIK and MEKK1, which phosphorylate and activate the IKK complex. Other mechanisms of activation of the NF-B <span class="highlight" style="background-color:">pathway</span> by viruses include direct activation of MEKK1 and PKC by HBx protein from HBV, persistent degradation of IB by SV, ER overload by influenza and adenovirus, and generation of reactive oxygen intermediates by influenza virus. NF-B activation is inhibited at either the IKK activation step by E1A of adenovirus or at the IB degradation step by MV and HCV core C protein. Lines in green with a + indicate activation of NF-B, while lines in red with a  indicate inhibition of NF-B.</div></p></div><div class="aux"><div class="resc">John Hiscott, et al. J Clin Invest. 2001 January 15;107(2):143-151.</div><p class="links"><a class="dblinks" href="/pubmed/11160127" ref="ordinalpos=3789&amp;ncbi_uid=246197&amp;link_uid=11160127">Citation</a><a class="dblinks" href="/pmc/articles/PMC199181/" ref="ordinalpos=3789&amp;ncbi_uid=246197&amp;link_uid=PMC199181">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3790.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3794710/figure/F7/" ref="ordinalpos=3790&amp;ncbi_uid=5378765&amp;link_uid=PMC3794710" href="/pmc/articles/PMC3794710/"><img src="/pmc/articles/instance/3794710/bin/nihms515952f7.gif" src-large="/pmc/articles/instance/3794710/bin/nihms515952f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3794710/" ref="ordinalpos=3790&amp;ncbi_uid=5378765&amp;link_uid=PMC3794710" image-link="/pmc/articles/PMC3794710/figure/F7/" class="imagepopup">Figure 7. Delineation of the intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> whereby DDR2 leads to Snail1 protein stabilization and tumor cell invasion/migration.  From: The Collagen Receptor Discoidin Domain Receptor 2 Stabilizes Snail1 Protein to Facilitate Breast Cancer Metastasis. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) MDA-MB-231 cells shRNAi-depleted of DDR2 or control (SCR) were added to collagen I coated (+) or uncoated plates (). Western blot with the indicated antibodies was performed on cell extracts. The fold change in phospho-S9-GSK3 relative to unstimulated control cells was calculated from densitometry. (<b>b</b>) MDA-MB-231 cells shRNAi-depleted of GSK3 or control (SCR) were added to collagen I coated (+) or uncoated plates (). Western blots with the indicated antibodies were performed on cell extracts. The fold change in the level of Snail1 relative to unstimulated control cells was calculated from densitometry. (<b>c</b>) MDA-MB-231 cells shRNAi-depleted of ERK1 or ERK2 or control (SCR) were added to collagen I coated (+) or uncoated plates (). Western blots with the indicated antibodies were performed on cell extracts. The fold change in the level of Snail1 protein relative to unstimulated control cells was calculated from densitometry. (<b>d</b>) Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from collagen activated DDR2 to Snail1 stabilization and MT1-MMP and collagen I production. (<b>e</b>) MDA-MB-231 cells were shRNAi-depleted of the indicated proteins then added to collagen I coated (+) or uncoated plates (). Cell extracts were Western blotted with the indicated antibodies.</div></p></div><div class="aux"><div class="resc">Kun Zhang, et al. Nat Cell Biol. 2013 June;15(6):677-687.</div><p class="links"><a class="dblinks" href="/pubmed/23644467" ref="ordinalpos=3790&amp;ncbi_uid=5378765&amp;link_uid=23644467">Citation</a><a class="dblinks" href="/pmc/articles/PMC3794710/" ref="ordinalpos=3790&amp;ncbi_uid=5378765&amp;link_uid=PMC3794710">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3791.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3663255/figure/Fig1/" ref="ordinalpos=3791&amp;ncbi_uid=4923783&amp;link_uid=PMC3663255" href="/pmc/articles/PMC3663255/"><img src="/pmc/articles/instance/3663255/bin/18_2012_1151_Fig1_HTML.gif" src-large="/pmc/articles/instance/3663255/bin/18_2012_1151_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3663255/" ref="ordinalpos=3791&amp;ncbi_uid=4923783&amp;link_uid=PMC3663255" image-link="/pmc/articles/PMC3663255/figure/Fig1/" class="imagepopup">Fig. 1.  From: Genetic visualization of notch <span class="highlight" style="background-color:">signaling</span> in mammalian neurogenesis. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The proneural genes <i>Mash1</i> and <i>Ngn2</i> induce expression of notch ligands such as Dll1, which activate notch <span class="highlight" style="background-color:">signaling</span> in neighboring cells. Upon activation, the notch intracellular domain (NICD) is released from the transmembrane region and transferred into the nucleus, where NICD forms a complex with RBPj and induces <i>Hes1</i> and <i>Hes5</i> expression. Hes1 and Hes5 repress proneural gene expression. During maturation and trafficking to the cell surface, notch receptors undergo furin processing and glycosylation, which can impact their responsiveness to their ligands. The ligand-induced activation of notch <span class="highlight" style="background-color:">signaling</span> is dynamically regulated by endocytic trafficking, which can be modulated by the different ubiquitin ligases, such as Mind bom and Neuralized</div></p></div><div class="aux"><div class="resc">Itaru Imayoshi, et al. Cell Mol Life Sci. 2013 June;70(12):2045-2057.</div><p class="links"><a class="dblinks" href="/pubmed/22971775" ref="ordinalpos=3791&amp;ncbi_uid=4923783&amp;link_uid=22971775">Citation</a><a class="dblinks" href="/pmc/articles/PMC3663255/" ref="ordinalpos=3791&amp;ncbi_uid=4923783&amp;link_uid=PMC3663255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3792.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3279592/figure/F1/" ref="ordinalpos=3792&amp;ncbi_uid=4660986&amp;link_uid=PMC3279592" href="/pmc/articles/PMC3279592/"><img src="/pmc/articles/instance/3279592/bin/nihms344673f1.gif" src-large="/pmc/articles/instance/3279592/bin/nihms344673f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3279592/" ref="ordinalpos=3792&amp;ncbi_uid=4660986&amp;link_uid=PMC3279592" image-link="/pmc/articles/PMC3279592/figure/F1/" class="imagepopup">Figure 1. Effect of Smad 3 gene silencing on expression of TGF- induced mRNA in MN.  From: Inhibition of M. tuberculosis induced <span class="highlight" style="background-color:">signaling</span> by transforming growth factor-? in human mononuclear phagocytes. </a></p><div class="supp"><p class="details"><div>MN were transfected by siRNA for Smad3 (black) or control construct (gray). MN cultures received recombinant (r) TGF- (10 ng/ml) after 4 hrs. Expression of uPAR, uPA, PA-1 and TGF- mRNA were assessed in cell lysates after 24 hr. Expression of TNF a, not known to be under Smad3 <span class="highlight" style="background-color:">pathway</span> of TGF- <span class="highlight" style="background-color:">signaling</span>, was also assessed as control gene (last panel). Inhibition of Smad3 reduced mRNA for all genes under this classical <span class="highlight" style="background-color:">pathway</span> of cell <span class="highlight" style="background-color:">signaling</span>. Induction of uPAR and its inhibition by Smad3 siRNA was most dramatic.</div></p></div><div class="aux"><div class="resc">Mianda Wu, et al. Scand J Immunol. ;75(3):301-304.</div><p class="links"><a class="dblinks" href="/pubmed/22150316" ref="ordinalpos=3792&amp;ncbi_uid=4660986&amp;link_uid=22150316">Citation</a><a class="dblinks" href="/pmc/articles/PMC3279592/" ref="ordinalpos=3792&amp;ncbi_uid=4660986&amp;link_uid=PMC3279592">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3793.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2696053/figure/F1/" ref="ordinalpos=3793&amp;ncbi_uid=1817654&amp;link_uid=PMC2696053" href="/pmc/articles/PMC2696053/"><img src="/pmc/articles/instance/2696053/bin/nihms106693f1.gif" src-large="/pmc/articles/instance/2696053/bin/nihms106693f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2696053/" ref="ordinalpos=3793&amp;ncbi_uid=1817654&amp;link_uid=PMC2696053" image-link="/pmc/articles/PMC2696053/figure/F1/" class="imagepopup">Figure 1.  From: The molecular logic of Notch <span class="highlight" style="background-color:">signaling</span>: a structural and biochemical perspective. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and domain organization of Notch receptors and DSL ligands. (A) Model for the major events in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Signals initiated by the engagement of ligand (1) lead to metalloprotease cleavage (MP) at site S2 (2). This proteolytic step allows the cleavage of Notch by the -secretase complex at site S3 within the transmembrane domain (3), and release of intracellular notch (ICN) from the membrane (4). ICN translocates to the nucleus where it enters into a transcriptional activation complex with CSL and Mastermind (MAM;5). (B, C) The domain organization of Notch receptors (B) and DSL-family ligands (C) from fly, human and worm.</div></p></div><div class="aux"><div class="resc">Wendy R. Gordon, et al. J Cell Sci. ;121(Pt 19):3109-3119.</div><p class="links"><a class="dblinks" href="/pubmed/18799787" ref="ordinalpos=3793&amp;ncbi_uid=1817654&amp;link_uid=18799787">Citation</a><a class="dblinks" href="/pmc/articles/PMC2696053/" ref="ordinalpos=3793&amp;ncbi_uid=1817654&amp;link_uid=PMC2696053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3794.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3986667/figure/F4/" ref="ordinalpos=3794&amp;ncbi_uid=6095154&amp;link_uid=PMC3986667" href="/pmc/articles/PMC3986667/"><img src="/pmc/articles/instance/3986667/bin/1752-0509-8-39-4.gif" src-large="/pmc/articles/instance/3986667/bin/1752-0509-8-39-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3986667/" ref="ordinalpos=3794&amp;ncbi_uid=6095154&amp;link_uid=PMC3986667" image-link="/pmc/articles/PMC3986667/figure/F4/" class="imagepopup">Figure 4.  From: The protein-protein interaction network of eyestalk, Y-organ and hepatopancreas in Chinese mitten crab Eriocheir sinensis. </a></p><div class="supp"><p class="details"><div><b>Evolutionary origin of protein interactions in seven <span class="highlight" style="background-color:">signaling</span> pathways in study.</b> Protein interactions in the seven <span class="highlight" style="background-color:">signaling</span> pathways in study were divided into Evo-groups according to the origins of the corresponding components. A blank in the right side represents that there are no protein originating in this evo-group. For each <span class="highlight" style="background-color:">pathway</span>, the proportion of interactions in each evolution stage to all the interactions in this <span class="highlight" style="background-color:">pathway</span> is shown in different shades of green. A darker green colour stands for a larger proportion.</div></p></div><div class="aux"><div class="resc">Tong Hao, et al. BMC Syst Biol. 2014;8:39-39.</div><p class="links"><a class="dblinks" href="/pubmed/24674293" ref="ordinalpos=3794&amp;ncbi_uid=6095154&amp;link_uid=24674293">Citation</a><a class="dblinks" href="/pmc/articles/PMC3986667/" ref="ordinalpos=3794&amp;ncbi_uid=6095154&amp;link_uid=PMC3986667">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3795.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2896660/figure/fig2/" ref="ordinalpos=3795&amp;ncbi_uid=2393057&amp;link_uid=PMC2896660" href="/pmc/articles/PMC2896660/"><img src="/pmc/articles/instance/2896660/bin/JBB2010-236528.002.gif" src-large="/pmc/articles/instance/2896660/bin/JBB2010-236528.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2896660/" ref="ordinalpos=3795&amp;ncbi_uid=2393057&amp;link_uid=PMC2896660" image-link="/pmc/articles/PMC2896660/figure/fig2/" class="imagepopup">Figure 2.  From: Comparative Pathogenesis and Systems Biology for  Biodefense Virus Vaccine Development. </a></p><div class="supp"><p class="details"><div>Placing datasets into a functional context using <span class="highlight" style="background-color:">pathway</span> analysis. Example transcriptional expression data were uploaded to the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis application (http://www.ingenuity.com/), and its knowledgebase was used to construct <span class="highlight" style="background-color:">signaling</span> networks on the basis of known interactions from the literature. These approaches can allow the visualization of networks associated with viral infection and identify <span class="highlight" style="background-color:">signaling</span> hubs which may act as master switches of the host response.</div></p></div><div class="aux"><div class="resc">Gavin C. Bowick, et al. J Biomed Biotechnol. 2010;2010:236528.</div><p class="links"><a class="dblinks" href="/pubmed/20617142" ref="ordinalpos=3795&amp;ncbi_uid=2393057&amp;link_uid=20617142">Citation</a><a class="dblinks" href="/pmc/articles/PMC2896660/" ref="ordinalpos=3795&amp;ncbi_uid=2393057&amp;link_uid=PMC2896660">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3796.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4129812/figure/F3/" ref="ordinalpos=3796&amp;ncbi_uid=6606570&amp;link_uid=PMC4129812" href="/pmc/articles/PMC4129812/"><img src="/pmc/articles/instance/4129812/bin/WJEM-3-282-g006.gif" src-large="/pmc/articles/instance/4129812/bin/WJEM-3-282-g006.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4129812/" ref="ordinalpos=3796&amp;ncbi_uid=6606570&amp;link_uid=PMC4129812" image-link="/pmc/articles/PMC4129812/figure/F3/" class="imagepopup">Figure 3.  From: PKC? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves in TNF-?-induced IP3R1 expression in human mesangial cells. </a></p><div class="supp"><p class="details"><div>Effects of TNF- and <span class="highlight" style="background-color:">signaling</span> inhibitors on intraocular PKC activity.</div></p></div><div class="aux"><div class="resc">Yu-rong Wang, et al. World J Emerg Med. 2012;3(4):282-286.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3796&amp;ncbi_uid=6606570&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4129812/" ref="ordinalpos=3796&amp;ncbi_uid=6606570&amp;link_uid=PMC4129812">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3797.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3442952/figure/F2/" ref="ordinalpos=3797&amp;ncbi_uid=5340043&amp;link_uid=PMC3442952" href="/pmc/articles/PMC3442952/"><img src="/pmc/articles/instance/3442952/bin/nihms-381091-f0002.gif" src-large="/pmc/articles/instance/3442952/bin/nihms-381091-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3442952/" ref="ordinalpos=3797&amp;ncbi_uid=5340043&amp;link_uid=PMC3442952" image-link="/pmc/articles/PMC3442952/figure/F2/" class="imagepopup">Figure 2. Sustained activation of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in rat macrophages and BMM cells following A. actinomycetemcomitans stimulation.  From: Sustained MAPK Activation with Aggregatibacter actinomycetemcomitans Causes Inflammatory Bone Loss. </a></p><div class="supp"><p class="details"><div>Sustained activation of intracellular <span class="highlight" style="background-color:">signaling</span> NFB and MAPK/MKP pathways is observed post stimulation with <i>A. actinomycetemcomitans</i> in NR8383 rat macrophage cells (<i>A</i>) and BMMs (<i>B</i>). Western blot data representative of 3 separate experiments for panel A and 2 separate experiments in panel B.</div></p></div><div class="aux"><div class="resc">Joni Dunmyer, et al. Mol Oral Microbiol. 2012 October;27(5):397-407.</div><p class="links"><a class="dblinks" href="/pubmed/22958388" ref="ordinalpos=3797&amp;ncbi_uid=5340043&amp;link_uid=22958388">Citation</a><a class="dblinks" href="/pmc/articles/PMC3442952/" ref="ordinalpos=3797&amp;ncbi_uid=5340043&amp;link_uid=PMC3442952">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3798.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3529395/figure/F6/" ref="ordinalpos=3798&amp;ncbi_uid=4463535&amp;link_uid=PMC3529395" href="/pmc/articles/PMC3529395/"><img src="/pmc/articles/instance/3529395/bin/nihms425483f6.gif" src-large="/pmc/articles/instance/3529395/bin/nihms425483f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3529395/" ref="ordinalpos=3798&amp;ncbi_uid=4463535&amp;link_uid=PMC3529395" image-link="/pmc/articles/PMC3529395/figure/F6/" class="imagepopup">Figure 6. Reelin <span class="highlight" style="background-color:">signaling</span> through Apoer2 potentiates NMDAR-mediated Ca2+ influx at excitatory synapses.  From: LIPOPROTEIN RECEPTORS - AN EVOLUTIONARILY ANCIENT MULTIFUNCTIONAL RECEPTOR FAMILY. </a></p><div class="supp"><p class="details"><div>Ligation of Apoer2 by Reelin in the dendrite activates the SFK Fyn, which phosphorylates the NMDAR on NR2 subunits. Increased NMDAR phosphorylation stabilizes NMDAR at the plasma membrane (Snyder et al. 2005), and can also increase Ca<sup>2+</sup> conductance. In this manner, the Reelin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> participates directly in the mechanisms that regulate synaptic plasticity, memory and learning.</div></p></div><div class="aux"><div class="resc">Marco Dieckmann, et al. Biol Chem. ;391(11):1341-1363.</div><p class="links"><a class="dblinks" href="/pubmed/20868222" ref="ordinalpos=3798&amp;ncbi_uid=4463535&amp;link_uid=20868222">Citation</a><a class="dblinks" href="/pmc/articles/PMC3529395/" ref="ordinalpos=3798&amp;ncbi_uid=4463535&amp;link_uid=PMC3529395">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3799.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2000811/figure/F5/" ref="ordinalpos=3799&amp;ncbi_uid=1164937&amp;link_uid=PMC2000811" href="/pmc/articles/PMC2000811/"><img src="/pmc/articles/instance/2045604/bin/JCI0732084.f5.gif" src-large="/pmc/articles/instance/2045604/bin/JCI0732084.f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2000811/" ref="ordinalpos=3799&amp;ncbi_uid=1164937&amp;link_uid=PMC2000811" image-link="/pmc/articles/PMC2000811/figure/F5/" class="imagepopup">Figure 5. Activation of the HRI/eIF2P <span class="highlight" style="background-color:">pathway</span> in BMDMs by LPS..  From: The function of heme-regulated eIF2? kinase in murine iron homeostasis and macrophage maturation. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Time course of the activation of HRI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> upon LPS treatment. Cells were harvested after LPS treatment at the time points indicated, and equal amounts of protein extracts were analyzed for expression of HRI, eIF2P, Chop, and total eIF2 by Western blot analyses. (<b>B</b>) ROS production in macrophages upon LPS treatment. DCF fluorescence in macrophages was measured by FACS analysis after 6-hour LPS treatment with or without -ME (<i>n</i> = 3). (<b>C</b>) Prevention of the activation of HRI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by LPS with -ME. After 6-hour LPS treatment in the presence or absence of -ME, expression of eIF2P, Chop, and total eIF2 proteins was analyzed. The ratios of eIF2P to eIF2 and Chop to eIF2 were calculated. The ratio of each at time 0 was defined as 1. The normalized ratios are shown above the autoradiograms.                </div></p></div><div class="aux"><div class="resc">Sijin Liu, et al. J Clin Invest. 2007 November 1;117(11):3296-3305.</div><p class="links"><a class="dblinks" href="/pubmed/17932563" ref="ordinalpos=3799&amp;ncbi_uid=1164937&amp;link_uid=17932563">Citation</a><a class="dblinks" href="/pmc/articles/PMC2000811/" ref="ordinalpos=3799&amp;ncbi_uid=1164937&amp;link_uid=PMC2000811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3800.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4204446/figure/F8/" ref="ordinalpos=3800&amp;ncbi_uid=6744497&amp;link_uid=PMC4204446" href="/pmc/articles/PMC4204446/"><img src="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g008.gif" src-large="/pmc/articles/instance/4204446/bin/jcad_7_7_suppl1-g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4204446/" ref="ordinalpos=3800&amp;ncbi_uid=6744497&amp;link_uid=PMC4204446" image-link="/pmc/articles/PMC4204446/figure/F8/" class="imagepopup">Figure 8.  From: Updates on Psoriasis and Cutaneous Oncology. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> overview; target gene expression is initiated by hedgehog ligand binding to patch. </div></p></div><div class="aux"><div class="resc">Neal Bhatia, et al. J Clin Aesthet Dermatol. 2014 July;7(7 Suppl SC2S5-S22):S5-S22.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3800&amp;ncbi_uid=6744497&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4204446/" ref="ordinalpos=3800&amp;ncbi_uid=6744497&amp;link_uid=PMC4204446">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="37" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="38" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="39" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:43:44-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A163554311D510000000000EA0B91;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A163554311D510000000000EA0B91" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>