
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="4" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="8A1A393946D25221_0091SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC496C53EBEF210000000000A5A525" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene">Gene</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=12" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="3" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="4" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="5" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="4" /></div>
            <div><h2 class="result_count">Results: 301 to 400 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>301.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2893997/figure/F7/" ref="ordinalpos=301&amp;ncbi_uid=2467627&amp;link_uid=PMC2893997" href="/pmc/articles/PMC2893997/"><img src="/pmc/articles/instance/2893997/bin/zmk0131094950007.gif" src-large="/pmc/articles/instance/2893997/bin/zmk0131094950007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2893997/" ref="ordinalpos=301&amp;ncbi_uid=2467627&amp;link_uid=PMC2893997" image-link="/pmc/articles/PMC2893997/figure/F7/" class="imagepopup">Figure 7. From: Unregulated ARF6 Activation in Epithelial Cysts Generates Hyperactive <span class="highlight" style="background-color:">Signaling</span> Endosomes and Disrupts Morphogenesis. </a></p><div class="supp"><p class="details"><div>Working model for ARF6-GTP-induced <span class="highlight" style="background-color:">signaling</span> during cyst development. Increased ARF6-GTP levels lead to enhanced PLD activation, and potentially, the recruitment of nm23H1, to facilitate growth factor internalization into <span class="highlight" style="background-color:">signaling</span> endosomes. <span class="highlight" style="background-color:">Signaling</span> endosomes serve as platforms for constitutive receptor <span class="highlight" style="background-color:">signaling</span>. PA enrichment on these vesicular compartments could be important for recruitment of downstream <span class="highlight" style="background-color:">signaling</span> molecules. The type of <span class="highlight" style="background-color:">signaling</span> response elicited is probably dependent on the cell type. In MDCK cysts, elevated ARF6-GTP leads to activation of the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jogender S. Tushir, et al. Mol Biol Cell. 2010 July 1;21(13):2355-2366.</div><p class="links"><a class="dblinks" href="/pubmed/20462959" ref="ordinalpos=301&amp;ncbi_uid=2467627&amp;link_uid=20462959">Citation</a><a class="dblinks" href="/pmc/articles/PMC2893997/" ref="ordinalpos=301&amp;ncbi_uid=2467627&amp;link_uid=PMC2893997">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>302.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3952558/figure/F2/" ref="ordinalpos=302&amp;ncbi_uid=5911491&amp;link_uid=PMC3952558" href="/pmc/articles/PMC3952558/"><img src="/pmc/articles/instance/3952558/bin/nihms-556997-f0002.gif" src-large="/pmc/articles/instance/3952558/bin/nihms-556997-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3952558/" ref="ordinalpos=302&amp;ncbi_uid=5911491&amp;link_uid=PMC3952558" image-link="/pmc/articles/PMC3952558/figure/F2/" class="imagepopup">Figure 2.  From: Topographical changes in extracellular matrix: Activation of TLR4 <span class="highlight" style="background-color:">signaling</span> and solid tumor progression. </a></p><div class="supp"><p class="details"><div>TLR4 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. LPS-induced activation of TLR4 is mediated by CD14. CD14 delivers LPS to TLR4/MD-2 complex and downstream <span class="highlight" style="background-color:">signaling</span> is divided into MyD88-dependent and independent <span class="highlight" style="background-color:">signaling</span>. MyD88-dependent <span class="highlight" style="background-color:">signaling</span> results in the rapid activation of NF-κB and the induction of pro-inflammatory cytokines, while MyD88-independent <span class="highlight" style="background-color:">signaling</span> results in late activation of NF-κB and induction of type I interferons.</div></p></div><div class="aux"><div class="resc">Rhiannon M. Kelsh, et al. Trends Cancer Res. 2013 January 1;9:1-13.</div><p class="links"><a class="dblinks" href="/pubmed/24634571" ref="ordinalpos=302&amp;ncbi_uid=5911491&amp;link_uid=24634571">Citation</a><a class="dblinks" href="/pmc/articles/PMC3952558/" ref="ordinalpos=302&amp;ncbi_uid=5911491&amp;link_uid=PMC3952558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>303.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2972700/figure/F1/" ref="ordinalpos=303&amp;ncbi_uid=2562486&amp;link_uid=PMC2972700" href="/pmc/articles/PMC2972700/"><img src="/pmc/articles/instance/2972700/bin/nihms243195f1.gif" src-large="/pmc/articles/instance/2972700/bin/nihms243195f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2972700/" ref="ordinalpos=303&amp;ncbi_uid=2562486&amp;link_uid=PMC2972700" image-link="/pmc/articles/PMC2972700/figure/F1/" class="imagepopup">Figure 1. Wnts initiate multiple <span class="highlight" style="background-color:">signaling</span> pathways.  From: Wnt <span class="highlight" style="background-color:">Signaling</span> During Fracture Repair. </a></p><div class="supp"><p class="details"><div>The “canonical <span class="highlight" style="background-color:">pathway</span>” (center) is stimulated when Wnts bind to Fzd receptors and Lrp5/6 co-receptors. A cytoplasmic complex composed of GSK3β, Axin, and APC is inhibited, allowing cytoplasmic levels of β-catenin levels to rise. Some β-catenin translocates to the nucleus where it associates with Tcf/Lef transcription factors to regulate gene expression. During non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> (left), Wnts bind to a Fzd receptor, Ryk or Ror2, and activate downstream <span class="highlight" style="background-color:">signaling</span> events that do not involve GSK3β or β-catenin. Wnt <span class="highlight" style="background-color:">signaling</span> pathways are inhibited by several mechanisms (right). Secreted frizzled-related proteins (Sfrps) antagonize canonical Wnt <span class="highlight" style="background-color:">signaling</span> by binding the ligands and preventing their association with Fzd receptors. Dkk1 suppresses Wnt <span class="highlight" style="background-color:">signaling</span> by forming a ternary complex with Lrp5/6 and Krm1/2. Sclerostin (Scl) also binds to Lrp5/6, but not Krm1/2, to antagonize canonical Wnt <span class="highlight" style="background-color:">signaling</span>. In the absence of Wnt/Lrp <span class="highlight" style="background-color:">signaling</span>, GSK3β phosphorylates (asterisks) β-catenin, which marks it for ubiquitination and proteosomal degradation.</div></p></div><div class="aux"><div class="resc">Frank J. Secreto, et al. Curr Osteoporos Rep. ;7(2):64-69.</div><p class="links"><a class="dblinks" href="/pubmed/19631031" ref="ordinalpos=303&amp;ncbi_uid=2562486&amp;link_uid=19631031">Citation</a><a class="dblinks" href="/pmc/articles/PMC2972700/" ref="ordinalpos=303&amp;ncbi_uid=2562486&amp;link_uid=PMC2972700">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>304.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1890306/figure/pone-0000547-g004/" ref="ordinalpos=304&amp;ncbi_uid=972506&amp;link_uid=PMC1890306" href="/pmc/articles/PMC1890306/"><img src="/pmc/articles/instance/1890306/bin/pone.0000547.g004.gif" src-large="/pmc/articles/instance/1890306/bin/pone.0000547.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1890306/" ref="ordinalpos=304&amp;ncbi_uid=972506&amp;link_uid=PMC1890306" image-link="/pmc/articles/PMC1890306/figure/pone-0000547-g004/" class="imagepopup">Figure 4. A <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with four steps in series and with no parallel connections..  From: Selective Control of the Apoptosis <span class="highlight" style="background-color:">Signaling</span> Network in Heterogeneous Cell Populations. </a></p><div class="supp"><p class="details"><div>If the number of steps in the <span class="highlight" style="background-color:">pathway</span> is large, the ouput activity <i>a<sub>out</sub></i> obeys log-normal statistics (see text).</div></p></div><div class="aux"><div class="resc">Diego Calzolari, et al. PLoS ONE. 2007;2(6):e547.</div><p class="links"><a class="dblinks" href="/pubmed/17579719" ref="ordinalpos=304&amp;ncbi_uid=972506&amp;link_uid=17579719">Citation</a><a class="dblinks" href="/pmc/articles/PMC1890306/" ref="ordinalpos=304&amp;ncbi_uid=972506&amp;link_uid=PMC1890306">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>305.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4256027/figure/f2-bt-22-491/" ref="ordinalpos=305&amp;ncbi_uid=6942074&amp;link_uid=PMC4256027" href="/pmc/articles/PMC4256027/"><img src="/pmc/articles/instance/4256027/bin/bt-22-491f2.gif" src-large="/pmc/articles/instance/4256027/bin/bt-22-491f2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4256027/" ref="ordinalpos=305&amp;ncbi_uid=6942074&amp;link_uid=PMC4256027" image-link="/pmc/articles/PMC4256027/figure/f2-bt-22-491/" class="imagepopup">Fig. 2. From: Lightening up Light Therapy: Activation of Retrograde <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by Photobiomodulation. </a></p><div class="supp"><p class="details"><div>Integration of intracellular mitochondrial function rely on both anterograde and retrograde <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The anterograde <span class="highlight" style="background-color:">signaling</span> system transduces signals from environments (external and/or internal) to nucleus, activating genomic program to adjust mitochondrial functioning. The retrograde <span class="highlight" style="background-color:">signaling</span> system monitors the function of mitochondria and transduce signals back to the nucleus. Studies on cytoplasmic transducers are actively pursued. See text for details.</div></p></div><div class="aux"><div class="resc">Hong Pyo Kim. Biomol Ther (Seoul). 2014 November;22(6):491-496.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=305&amp;ncbi_uid=6942074&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4256027/" ref="ordinalpos=305&amp;ncbi_uid=6942074&amp;link_uid=PMC4256027">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>306.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634369/figure/F1/" ref="ordinalpos=306&amp;ncbi_uid=1710849&amp;link_uid=PMC2634369" href="/pmc/articles/PMC2634369/"><img src="/pmc/articles/instance/2634369/bin/psb0310_0761_fig001.gif" src-large="/pmc/articles/instance/2634369/bin/psb0310_0761_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634369/" ref="ordinalpos=306&amp;ncbi_uid=1710849&amp;link_uid=PMC2634369" image-link="/pmc/articles/PMC2634369/figure/F1/" class="imagepopup">Figure 1.  From: Ethylene <span class="highlight" style="background-color:">signaling</span> regulates salt stress response. </a></p><div class="supp"><p class="details"><div>Ethylene <span class="highlight" style="background-color:">signaling</span> and salt stress response. Black arrows and lines indicate active receptor <span class="highlight" style="background-color:">signaling</span>, which leads to temporary sensitive response under salt stress through suppression of AtNAC2 and activation of AtERF4. Red arrows and lines indicate active ethylene <span class="highlight" style="background-color:">signaling</span>, which inhibits receptor <span class="highlight" style="background-color:">signaling</span> and leads to tolerant response under salt stress. Green lines and bars indicate the <span class="highlight" style="background-color:">pathway</span> regulating the EIN3 stability. Blue arrows and lines indicate new <span class="highlight" style="background-color:">pathway</span> bypassing EIN2 and regulating ethylene response and salt stress response.25</div></p></div><div class="aux"><div class="resc">Yang-Rong Cao, et al. Plant Signal Behav. 2008 October;3(10):761-763.</div><p class="links"><a class="dblinks" href="/pubmed/19513226" ref="ordinalpos=306&amp;ncbi_uid=1710849&amp;link_uid=19513226">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634369/" ref="ordinalpos=306&amp;ncbi_uid=1710849&amp;link_uid=PMC2634369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>307.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC230332/figure/f7/" ref="ordinalpos=307&amp;ncbi_uid=289338&amp;link_uid=PMC230332" href="/pmc/articles/PMC230332/"><img src="/pmc/articles/instance/230332/bin/mb2031580007.gif" src-large="/pmc/articles/instance/230332/bin/mb2031580007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC230332/" ref="ordinalpos=307&amp;ncbi_uid=289338&amp;link_uid=PMC230332" image-link="/pmc/articles/PMC230332/figure/f7/" class="imagepopup">FIG. 7. From: Myostatin Signals through a Transforming Growth Factor ?-Like <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> To Block Adipogenesis. </a></p><div class="supp"><p class="details"><div>Model of extracellular and intracellular antagonism of BMP <span class="highlight" style="background-color:">signaling</span> by myostatin and TGF-β. Myostatin signals through ActRIIB and either ALK4 or ALK5 to activate a TGF-β-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Myostatin potently antagonizes BMP7 but not BMP2 by competing for BMP7 binding to the ActRIIB type II receptor.</div></p></div><div class="aux"><div class="resc">A. Rebbapragada, et al. Mol Cell Biol. 2003 October;23(20):7230-7242.</div><p class="links"><a class="dblinks" href="/pubmed/14517293" ref="ordinalpos=307&amp;ncbi_uid=289338&amp;link_uid=14517293">Citation</a><a class="dblinks" href="/pmc/articles/PMC230332/" ref="ordinalpos=307&amp;ncbi_uid=289338&amp;link_uid=PMC230332">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>308.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4276980/figure/nutrients-06-05500-f002/" ref="ordinalpos=308&amp;ncbi_uid=7068192&amp;link_uid=PMC4276980" href="/pmc/articles/PMC4276980/"><img src="/pmc/articles/instance/4276980/bin/nutrients-06-05500-g002.gif" src-large="/pmc/articles/instance/4276980/bin/nutrients-06-05500-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4276980/" ref="ordinalpos=308&amp;ncbi_uid=7068192&amp;link_uid=PMC4276980" image-link="/pmc/articles/PMC4276980/figure/nutrients-06-05500-f002/" class="imagepopup">Figure 2.  From: Bee Pollen Improves Muscle Protein and Energy Metabolism in Malnourished Old Rats through Interfering with the Mtor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Mitochondrial Activity. </a></p><div class="supp"><p class="details"><div>Regulation of the mTOR/p70S6k/4EBP1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after dietary restriction and refeeding with fresh pollen-containing diets in old rats.</div></p></div><div class="aux"><div class="resc">Jérôme Salles, et al. Nutrients. 2014 December;6(12):5500-5516.</div><p class="links"><a class="dblinks" href="/pubmed/25470375" ref="ordinalpos=308&amp;ncbi_uid=7068192&amp;link_uid=25470375">Citation</a><a class="dblinks" href="/pmc/articles/PMC4276980/" ref="ordinalpos=308&amp;ncbi_uid=7068192&amp;link_uid=PMC4276980">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>309.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3480403/figure/pone-0047773-g002/" ref="ordinalpos=309&amp;ncbi_uid=4321315&amp;link_uid=PMC3480403" href="/pmc/articles/PMC3480403/"><img src="/pmc/articles/instance/3480403/bin/pone.0047773.g002.gif" src-large="/pmc/articles/instance/3480403/bin/pone.0047773.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3480403/" ref="ordinalpos=309&amp;ncbi_uid=4321315&amp;link_uid=PMC3480403" image-link="/pmc/articles/PMC3480403/figure/pone-0047773-g002/" class="imagepopup">Figure 2. Classification and regression tree analysis of polymorphisms of estrogen metabolic and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes..  From: Epistasis Analysis for Estrogen Metabolic and <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Genes on Young Ischemic Stroke Patients. </a></p><div class="aux"><div class="resc">Yi-Chen Hsieh, et al. PLoS One. 2012;7(10):e47773.</div><p class="links"><a class="dblinks" href="/pubmed/23112845" ref="ordinalpos=309&amp;ncbi_uid=4321315&amp;link_uid=23112845">Citation</a><a class="dblinks" href="/pmc/articles/PMC3480403/" ref="ordinalpos=309&amp;ncbi_uid=4321315&amp;link_uid=PMC3480403">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>310.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855388/figure/pone-0082160-g004/" ref="ordinalpos=310&amp;ncbi_uid=5589446&amp;link_uid=PMC3855388" href="/pmc/articles/PMC3855388/"><img src="/pmc/articles/instance/3855388/bin/pone.0082160.g004.gif" src-large="/pmc/articles/instance/3855388/bin/pone.0082160.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855388/" ref="ordinalpos=310&amp;ncbi_uid=5589446&amp;link_uid=PMC3855388" image-link="/pmc/articles/PMC3855388/figure/pone-0082160-g004/" class="imagepopup">Figure 4. Meta network of the ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> containing miRNA, TF and drug regulation..  From: CyTargetLinker: A Cytoscape App to Integrate Regulatory Interactions in Network Analysis. </a></p><div class="supp"><p class="details"><div>The ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (http://wikipathways.org/index.php/<span class="highlight" style="background-color:">Pathway</span>:WP673) from WikiPathways was extended with TFs, miRNAs and drugs regulating the elements in the <span class="highlight" style="background-color:">pathway</span>. The nodes from the original <span class="highlight" style="background-color:">pathway</span> are coloured in grey, TFs in green, miRNAs in yellow and drugs in purple. 258 TF-gene interactions from ENCODE, 162 miRNA-target interactions from the two validated miRNA-target RegINs (miRTarBase and miRecords) and 138 drug-target interactions from DrugBank were added to the network. The edges are coloured based on the RegIN source, purple and orange for ENCODE (proximal and distal), blue and red for miRTarBase and miRecords, and green for DrugBank.</div></p></div><div class="aux"><div class="resc">Martina Kutmon, et al. PLoS One. 2013;8(12):e82160.</div><p class="links"><a class="dblinks" href="/pubmed/24340000" ref="ordinalpos=310&amp;ncbi_uid=5589446&amp;link_uid=24340000">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855388/" ref="ordinalpos=310&amp;ncbi_uid=5589446&amp;link_uid=PMC3855388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>311.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3976145/figure/F1/" ref="ordinalpos=311&amp;ncbi_uid=5995249&amp;link_uid=PMC3976145" href="/pmc/articles/PMC3976145/"><img src="/pmc/articles/instance/3976145/bin/nihms567403f1.gif" src-large="/pmc/articles/instance/3976145/bin/nihms567403f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3976145/" ref="ordinalpos=311&amp;ncbi_uid=5995249&amp;link_uid=PMC3976145" image-link="/pmc/articles/PMC3976145/figure/F1/" class="imagepopup">Figure 1.  From: Epigenetic Regulation of WNT <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Genes in Inflammatory Bowel Disease (IBD) Associated Neoplasia. </a></p><div class="supp"><p class="details"><div>Overview of the genes involved in the WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. When the <span class="highlight" style="background-color:">pathway</span> is turned on by binding of WNT, a cascade of events leads to translocation of β-catenin to the nucleus, which in turn upregulates the expression of many cancer-related genes.</div></p></div><div class="aux"><div class="resc">Mashaal Dhir, et al. J Gastrointest Surg. 2008 October;12(10):1745-1753.</div><p class="links"><a class="dblinks" href="/pubmed/18716850" ref="ordinalpos=311&amp;ncbi_uid=5995249&amp;link_uid=18716850">Citation</a><a class="dblinks" href="/pmc/articles/PMC3976145/" ref="ordinalpos=311&amp;ncbi_uid=5995249&amp;link_uid=PMC3976145">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>312.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3527976/figure/F2/" ref="ordinalpos=312&amp;ncbi_uid=5617531&amp;link_uid=PMC3527976" href="/pmc/articles/PMC3527976/"><img src="/pmc/articles/instance/3527976/bin/zbc0021334140002.gif" src-large="/pmc/articles/instance/3527976/bin/zbc0021334140002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3527976/" ref="ordinalpos=312&amp;ncbi_uid=5617531&amp;link_uid=PMC3527976" image-link="/pmc/articles/PMC3527976/figure/F2/" class="imagepopup">FIGURE 2. From: Mammalian Target of Rapamycin (mTOR) <span class="highlight" style="background-color:">Signaling</span> Network in Skeletal Myogenesis. </a></p><div class="supp"><p class="details"><div><b>Two rapamycin-sensitive myogenic mTOR <span class="highlight" style="background-color:">signaling</span> pathways.</b> A kinase-independent mTOR <span class="highlight" style="background-color:">pathway</span> controls IGF2 expression through transcriptional regulation at a muscle-specific enhancer, as well as through suppression of miR-125b, which targets <i>Igf2</i>. IGF2 regulates myogenesis by activating the IGF1 receptor (<i>IGF-IR</i>) and PI3K/Akt <span class="highlight" style="background-color:">signaling</span>. A kinase-dependent mTOR <span class="highlight" style="background-color:">pathway</span> regulates myocyte fusion and myotube/myofiber maturation by controlling follistatin expression via a MyoD/miR-1/HDAC4 <span class="highlight" style="background-color:">pathway</span>. Amino acid signals are most likely upstream of the IGF2 <span class="highlight" style="background-color:">pathway</span>, but their role in the follistatin <span class="highlight" style="background-color:">pathway</span> is not clear.</div></p></div><div class="aux"><div class="resc">Yejing Ge, et al. J Biol Chem. 2012 December 21;287(52):43928-43935.</div><p class="links"><a class="dblinks" href="/pubmed/23115234" ref="ordinalpos=312&amp;ncbi_uid=5617531&amp;link_uid=23115234">Citation</a><a class="dblinks" href="/pmc/articles/PMC3527976/" ref="ordinalpos=312&amp;ncbi_uid=5617531&amp;link_uid=PMC3527976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>313.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3141626/figure/F5/" ref="ordinalpos=313&amp;ncbi_uid=3107299&amp;link_uid=PMC3141626" href="/pmc/articles/PMC3141626/"><img src="/pmc/articles/instance/3141626/bin/1477-5956-9-35-5.gif" src-large="/pmc/articles/instance/3141626/bin/1477-5956-9-35-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3141626/" ref="ordinalpos=313&amp;ncbi_uid=3107299&amp;link_uid=PMC3141626" image-link="/pmc/articles/PMC3141626/figure/F5/" class="imagepopup">Figure 5.  From: Analysis of EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in nasopharyngeal carcinoma cells by quantitative phosphoproteomics. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> network of EGFR-regulated phosphoproteins in NPC cells</b>. Proteins identified in Table 1 were imported into <span class="highlight" style="background-color:">Pathway</span> Studio, and an interaction map was generated with information from ResnetCore of Ariadne Genomics, and KEGG <span class="highlight" style="background-color:">Pathway</span> database. Legend of the interaction network is summarized on the right of the figure. Phosphoproteins, which are identified as phosphorylated targets of EGF <span class="highlight" style="background-color:">signaling</span>, were highlighted on the map (blue).</div></p></div><div class="aux"><div class="resc">Lin Ruan, et al. Proteome Sci. 2011;9:35-35.</div><p class="links"><a class="dblinks" href="/pubmed/21711528" ref="ordinalpos=313&amp;ncbi_uid=3107299&amp;link_uid=21711528">Citation</a><a class="dblinks" href="/pmc/articles/PMC3141626/" ref="ordinalpos=313&amp;ncbi_uid=3107299&amp;link_uid=PMC3141626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>314.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4029079/figure/F5/" ref="ordinalpos=314&amp;ncbi_uid=6162570&amp;link_uid=PMC4029079" href="/pmc/articles/PMC4029079/"><img src="/pmc/articles/instance/4029079/bin/1471-2164-15-182-5.gif" src-large="/pmc/articles/instance/4029079/bin/1471-2164-15-182-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4029079/" ref="ordinalpos=314&amp;ncbi_uid=6162570&amp;link_uid=PMC4029079" image-link="/pmc/articles/PMC4029079/figure/F5/" class="imagepopup">Figure 5.  From: Transcriptomic analysis of differentially expressed genes in the Ras1CA-overexpressed and wildtype posterior silk glands. </a></p><div class="supp"><p class="details"><div><b>Verification of transcriptomic results of DEGs only in “pathways in cancer”, “insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>”, or “MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” by qPCR.</b> DEGs only in “pathways in cancer” <b>(A)</b>, “insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” <b>(B)</b>, “MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” <b>(C)</b>. The transcriptional results of DEGs (FPKM) are marked with black. The q-PCR data of DEGs upregulated and downregulated by Ras1 are marked with red and green. The mis-matched DEGs are marked with gray. WT: wildtype; Ras1: Fil-GAL4 &gt; UAS-Ras1<sup>CA</sup>.</div></p></div><div class="aux"><div class="resc">Li Ma, et al. BMC Genomics. 2014;15:182-182.</div><p class="links"><a class="dblinks" href="/pubmed/24606580" ref="ordinalpos=314&amp;ncbi_uid=6162570&amp;link_uid=24606580">Citation</a><a class="dblinks" href="/pmc/articles/PMC4029079/" ref="ordinalpos=314&amp;ncbi_uid=6162570&amp;link_uid=PMC4029079">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>315.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3289858/figure/fig2/" ref="ordinalpos=315&amp;ncbi_uid=3874833&amp;link_uid=PMC3289858" href="/pmc/articles/PMC3289858/"><img src="/pmc/articles/instance/3289858/bin/TSWJ2012-912147.002.gif" src-large="/pmc/articles/instance/3289858/bin/TSWJ2012-912147.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3289858/" ref="ordinalpos=315&amp;ncbi_uid=3874833&amp;link_uid=PMC3289858" image-link="/pmc/articles/PMC3289858/figure/fig2/" class="imagepopup">Figure 2.  From: Yeast as a Tool to Study <span class="highlight" style="background-color:">Signaling</span> Pathways in Mitochondrial Stress Response and Cytoprotection. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways possibly involved in the interplay between cell death and cytoprotection processes in yeast cells in response to acetic acid stress. Acetic acid causes intracellular acidification, mitochondrial dysfunction, and cell death. TOR <span class="highlight" style="background-color:">pathway</span> is involved in the <span class="highlight" style="background-color:">signaling</span> of AA-PCD; rapamycin inhibition of TOR can trigger autophagy and RTG target gene expression. Dotted lines indicate the hypothetical <span class="highlight" style="background-color:">signaling</span> pathways in AA-PCD regulation in response to mitochondrial dysfunction, on the basis of indirect experimental evidence (see text for details). Intracellular acidification stimulates RAS-cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, causing mitochondrial dysfunction, which may lead to AA-PCD or activate retrograde response inducing AA-PCD resistance. RAS <span class="highlight" style="background-color:">pathway</span> negatively regulates Mks1p, a negative regulator of RTG <span class="highlight" style="background-color:">pathway</span>. Positive and negative regulators of RTG <span class="highlight" style="background-color:">pathway</span> are shown in blue and red, respectively. TOR <span class="highlight" style="background-color:">signaling</span> is found at the crossroad of RTG, AA-PCD, and autophagy pathways.</div></p></div><div class="aux"><div class="resc">Maša Ždralević, et al. ScientificWorldJournal. 2012;2012:912147.</div><p class="links"><a class="dblinks" href="/pubmed/22454613" ref="ordinalpos=315&amp;ncbi_uid=3874833&amp;link_uid=22454613">Citation</a><a class="dblinks" href="/pmc/articles/PMC3289858/" ref="ordinalpos=315&amp;ncbi_uid=3874833&amp;link_uid=PMC3289858">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>316.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3255869/figure/F1/" ref="ordinalpos=316&amp;ncbi_uid=4091064&amp;link_uid=PMC3255869" href="/pmc/articles/PMC3255869/"><img src="/pmc/articles/instance/3255869/bin/zjv9990954570001.gif" src-large="/pmc/articles/instance/3255869/bin/zjv9990954570001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3255869/" ref="ordinalpos=316&amp;ncbi_uid=4091064&amp;link_uid=PMC3255869" image-link="/pmc/articles/PMC3255869/figure/F1/" class="imagepopup">Fig 1.  From: The TLR4-TRIF <span class="highlight" style="background-color:">Pathway</span> Protects against H5N1 Influenza Virus Infection. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MD-2 and CD14 (green) are essential for both arms of the TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, whereas MyD88 (purple) and TRIF (blue) are essential for <span class="highlight" style="background-color:">signaling</span> that leads to cytokine/chemokine or type I IFN production, respectively.</div></p></div><div class="aux"><div class="resc">Kyoko Shinya, et al. J Virol. 2012 January;86(1):19-24.</div><p class="links"><a class="dblinks" href="/pubmed/22031950" ref="ordinalpos=316&amp;ncbi_uid=4091064&amp;link_uid=22031950">Citation</a><a class="dblinks" href="/pmc/articles/PMC3255869/" ref="ordinalpos=316&amp;ncbi_uid=4091064&amp;link_uid=PMC3255869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>317.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3291562/figure/pone-0032391-g004/" ref="ordinalpos=317&amp;ncbi_uid=3841479&amp;link_uid=PMC3291562" href="/pmc/articles/PMC3291562/"><img src="/pmc/articles/instance/3291562/bin/pone.0032391.g004.gif" src-large="/pmc/articles/instance/3291562/bin/pone.0032391.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3291562/" ref="ordinalpos=317&amp;ncbi_uid=3841479&amp;link_uid=PMC3291562" image-link="/pmc/articles/PMC3291562/figure/pone-0032391-g004/" class="imagepopup">Figure 4. Mapping of differentially expressed genes on the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Function-Based Discovery of Significant Transcriptional Temporal Patterns in Insulin Stimulated Muscle Cells. </a></p><div class="supp"><p class="details"><div>62 genes (in red) belonging to the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (GenMapp), selected as differentially expressed and associated with significant temporal expression patterns.</div></p></div><div class="aux"><div class="resc">Barbara Di Camillo, et al. PLoS One. 2012;7(3):e32391.</div><p class="links"><a class="dblinks" href="/pubmed/22396763" ref="ordinalpos=317&amp;ncbi_uid=3841479&amp;link_uid=22396763">Citation</a><a class="dblinks" href="/pmc/articles/PMC3291562/" ref="ordinalpos=317&amp;ncbi_uid=3841479&amp;link_uid=PMC3291562">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>318.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC355855/figure/f10/" ref="ordinalpos=318&amp;ncbi_uid=457984&amp;link_uid=PMC355855" href="/pmc/articles/PMC355855/"><img src="/pmc/articles/instance/355855/bin/zmb0060414880010.gif" src-large="/pmc/articles/instance/355855/bin/zmb0060414880010.jpg" alt="FIG. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC355855/" ref="ordinalpos=318&amp;ncbi_uid=457984&amp;link_uid=PMC355855" image-link="/pmc/articles/PMC355855/figure/f10/" class="imagepopup">FIG. 10. From: Jab1/CSN5, a Component of the COP9 Signalosome, Regulates Transforming Growth Factor ? <span class="highlight" style="background-color:">Signaling</span> by Binding to Smad7 and Promoting Its Degradation. </a></p><div class="supp"><p class="details"><div>Models for repression of TGF-β <span class="highlight" style="background-color:">signaling</span> by the Smurf-Smad7 complex and reactivation of TGF-β <span class="highlight" style="background-color:">signaling</span> by the Jab1/CSN5-Smad7 complex. (A) Smad7 inhibits TGF-β <span class="highlight" style="background-color:">signaling</span> by preventing access of Smad2 or Smad3 to the TGF-β receptor. (B) Smad7 inhibits TGF-β1 <span class="highlight" style="background-color:">signaling</span> by binding directly to Smurfs, bringing the Smurf/Smad7 complex to TβRI, and inducing the turnover of TβRI. (C) Jab1/CSN5 enhances TGF-β <span class="highlight" style="background-color:">signaling</span> by binding to Smad7 and inducing degradation of Smad7 through the COP9-proteosome <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Byung-Chul Kim, et al. Mol Cell Biol. 2004 March;24(6):2251-2262.</div><p class="links"><a class="dblinks" href="/pubmed/14993265" ref="ordinalpos=318&amp;ncbi_uid=457984&amp;link_uid=14993265">Citation</a><a class="dblinks" href="/pmc/articles/PMC355855/" ref="ordinalpos=318&amp;ncbi_uid=457984&amp;link_uid=PMC355855">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>319.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3081368/figure/F8/" ref="ordinalpos=319&amp;ncbi_uid=3391733&amp;link_uid=PMC3081368" href="/pmc/articles/PMC3081368/"><img src="/pmc/articles/instance/3081368/bin/nihms-271519-f0008.gif" src-large="/pmc/articles/instance/3081368/bin/nihms-271519-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3081368/" ref="ordinalpos=319&amp;ncbi_uid=3391733&amp;link_uid=PMC3081368" image-link="/pmc/articles/PMC3081368/figure/F8/" class="imagepopup">Fig. 8. A model of function of Hmgb1 and Hmgb2 during mouse forelimb development.  From: HMGB factors are required for posterior digit development through integrating <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activities. </a></p><div class="supp"><p class="details"><div>Combined function of <i>Hmgb1</i> and <i>Hmgb2</i> integrates the Hox-Shh <span class="highlight" style="background-color:">pathway</span>, Wnt/ß-catenin <span class="highlight" style="background-color:">signaling</span> and BMP <span class="highlight" style="background-color:">signaling</span> for proper development of digit 5 in the forelimb bud. See detail in the text.</div></p></div><div class="aux"><div class="resc">Junji Itou, et al. Dev Dyn. ;240(5):1151-1162.</div><p class="links"><a class="dblinks" href="/pubmed/21384471" ref="ordinalpos=319&amp;ncbi_uid=3391733&amp;link_uid=21384471">Citation</a><a class="dblinks" href="/pmc/articles/PMC3081368/" ref="ordinalpos=319&amp;ncbi_uid=3391733&amp;link_uid=PMC3081368">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>320.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2447663/figure/fig9/" ref="ordinalpos=320&amp;ncbi_uid=1434058&amp;link_uid=PMC2447663" href="/pmc/articles/PMC2447663/"><img src="/pmc/articles/instance/2447663/bin/zbc0190834310009.gif" src-large="/pmc/articles/instance/2447663/bin/zbc0190834310009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2447663/" ref="ordinalpos=320&amp;ncbi_uid=1434058&amp;link_uid=PMC2447663" image-link="/pmc/articles/PMC2447663/figure/fig9/" class="imagepopup">FIGURE 9. From: MEK <span class="highlight" style="background-color:">Signaling</span> Is Required for Phosphorylation of eIF2? following  Amino Acid Limitation of HepG2 Human Hepatoma  Cells. </a></p><div class="supp"><p class="details"><div><b>There is an interdependence between the MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and  the amino acid response <span class="highlight" style="background-color:">pathway</span>.</b> The <i>diagram</i> depicts activation  of the MEK-ERK <span class="highlight" style="background-color:">pathway</span> by amino acid limitation and the function of the  MEK/ERK <span class="highlight" style="background-color:">pathway</span> to permit or enhance the GCN2-dependent phosphorylation of  eIF2α.</div></p></div><div class="aux"><div class="resc">Michelle M. Thiaville, et al. J Biol Chem. 2008 April 18;283(16):10848-10857.</div><p class="links"><a class="dblinks" href="/pubmed/18287093" ref="ordinalpos=320&amp;ncbi_uid=1434058&amp;link_uid=18287093">Citation</a><a class="dblinks" href="/pmc/articles/PMC2447663/" ref="ordinalpos=320&amp;ncbi_uid=1434058&amp;link_uid=PMC2447663">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>321.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2873688/figure/F4/" ref="ordinalpos=321&amp;ncbi_uid=2381387&amp;link_uid=PMC2873688" href="/pmc/articles/PMC2873688/"><img src="/pmc/articles/instance/2873688/bin/nihms201344f4.gif" src-large="/pmc/articles/instance/2873688/bin/nihms201344f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2873688/" ref="ordinalpos=321&amp;ncbi_uid=2381387&amp;link_uid=PMC2873688" image-link="/pmc/articles/PMC2873688/figure/F4/" class="imagepopup">Fig. 4.  From: Insulin/IGF-1 and ROS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> cross-talk in aging and longevity determination. </a></p><div class="supp"><p class="details"><div>Molecular mechanism and <span class="highlight" style="background-color:">pathway</span> of JNK activation by p85. The full-length p85 regulatory subunit activates JNK through the insulin/cdc42/MKK <span class="highlight" style="background-color:">pathway</span>. Activation of JNK results in negative feedback on the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Thus, the loss of p85 improves insulin sensitivity, in part by diminishing this JNK-mediated negative regulation. (This figure is a modification from Taniguchi et al., 2007.)</div></p></div><div class="aux"><div class="resc">John Papaconstantinou. Mol Cell Endocrinol. ;299(1):89-100.</div><p class="links"><a class="dblinks" href="/pubmed/19103250" ref="ordinalpos=321&amp;ncbi_uid=2381387&amp;link_uid=19103250">Citation</a><a class="dblinks" href="/pmc/articles/PMC2873688/" ref="ordinalpos=321&amp;ncbi_uid=2381387&amp;link_uid=PMC2873688">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>322.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3660694/figure/Fig1/" ref="ordinalpos=322&amp;ncbi_uid=4962454&amp;link_uid=PMC3660694" href="/pmc/articles/PMC3660694/"><img src="/pmc/articles/instance/3660694/bin/12079_2012_186_Fig1_HTML.gif" src-large="/pmc/articles/instance/3660694/bin/12079_2012_186_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3660694/" ref="ordinalpos=322&amp;ncbi_uid=4962454&amp;link_uid=PMC3660694" image-link="/pmc/articles/PMC3660694/figure/Fig1/" class="imagepopup">Fig. 1.  From: <span class="highlight" style="background-color:">Signaling</span> network of Oncostatin M <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of Oncostatin M <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>- NetSlim (http://www.netpath.org/netslim/NetSlim_114) map generated to graphically represent the different reactions induced by OSM in various cell types</div></p></div><div class="aux"><div class="resc">Gourav Dey, et al. J Cell Commun Signal. 2013 June;7(2):103-108.</div><p class="links"><a class="dblinks" href="/pubmed/23255051" ref="ordinalpos=322&amp;ncbi_uid=4962454&amp;link_uid=23255051">Citation</a><a class="dblinks" href="/pmc/articles/PMC3660694/" ref="ordinalpos=322&amp;ncbi_uid=4962454&amp;link_uid=PMC3660694">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>323.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3645653/figure/f1-ijms-14-06542/" ref="ordinalpos=323&amp;ncbi_uid=4894470&amp;link_uid=PMC3645653" href="/pmc/articles/PMC3645653/"><img src="/pmc/articles/instance/3645653/bin/ijms-14-06542f1.gif" src-large="/pmc/articles/instance/3645653/bin/ijms-14-06542f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3645653/" ref="ordinalpos=323&amp;ncbi_uid=4894470&amp;link_uid=PMC3645653" image-link="/pmc/articles/PMC3645653/figure/f1-ijms-14-06542/" class="imagepopup">Figure 1.  From: The Multiple Faces of Prostaglandin E2 G-Protein Coupled Receptor <span class="highlight" style="background-color:">Signaling</span> during the Dendritic Cell Life Cycle. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways activated by PGE2 stimulation of the human EP2 and EP4 prostanoid receptors. Phosphorylation of GSK-3α via either PKA or PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibits the kinase activity of GSK-3α. Inhibition of GSK-3α stabilizes β-catenin that results in a decrease in its degradation and promotes β-catenin nuclear translocation and transcriptional activity of Tcf/Lef-regulated genes. Activation of either PKA or PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to phosphorylation of the transcription factor CREB regulating cAMP responsive gene expression. Activation of the PI3K/ERK <span class="highlight" style="background-color:">pathway</span> induces functional expression of EGR-1, known to regulate PGE2 synthase. In addition, PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibits the activity of PKA.</div></p></div><div class="aux"><div class="resc">Sandra De Keijzer, et al. Int J Mol Sci. 2013 April;14(4):6542-6555.</div><p class="links"><a class="dblinks" href="/pubmed/23528886" ref="ordinalpos=323&amp;ncbi_uid=4894470&amp;link_uid=23528886">Citation</a><a class="dblinks" href="/pmc/articles/PMC3645653/" ref="ordinalpos=323&amp;ncbi_uid=4894470&amp;link_uid=PMC3645653">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>324.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2869087/figure/F1/" ref="ordinalpos=324&amp;ncbi_uid=2355665&amp;link_uid=PMC2869087" href="/pmc/articles/PMC2869087/"><img src="/pmc/articles/instance/2869087/bin/nihms201649f1.gif" src-large="/pmc/articles/instance/2869087/bin/nihms201649f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2869087/" ref="ordinalpos=324&amp;ncbi_uid=2355665&amp;link_uid=PMC2869087" image-link="/pmc/articles/PMC2869087/figure/F1/" class="imagepopup">Figure 1. Hedgehog proangiogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Hedgehog takes a new "RHOad" to angiogenesis. </a></p><div class="supp"><p class="details"><div>Schematic representation of this novel proangiogenic Hedgehog <span class="highlight" style="background-color:">signaling</span> cascade in endothelial cells. Hedgehog promotes tube formation through a SMOOTHENED-Gi-RAC1-RHOA <span class="highlight" style="background-color:">pathway</span>. This cellular event is GLI independent and is accompanied by ligand dependent inhibition of PATCHED pro-apoptotic functions in endothelial cells.</div></p></div><div class="aux"><div class="resc">Martin E. Fernandez-Zapico. Cell Cycle. ;9(4):646-647.</div><p class="links"><a class="dblinks" href="/pubmed/20190577" ref="ordinalpos=324&amp;ncbi_uid=2355665&amp;link_uid=20190577">Citation</a><a class="dblinks" href="/pmc/articles/PMC2869087/" ref="ordinalpos=324&amp;ncbi_uid=2355665&amp;link_uid=PMC2869087">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>325.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3950223/figure/pone-0091580-g005/" ref="ordinalpos=325&amp;ncbi_uid=5906025&amp;link_uid=PMC3950223" href="/pmc/articles/PMC3950223/"><img src="/pmc/articles/instance/3950223/bin/pone.0091580.g005.gif" src-large="/pmc/articles/instance/3950223/bin/pone.0091580.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3950223/" ref="ordinalpos=325&amp;ncbi_uid=5906025&amp;link_uid=PMC3950223" image-link="/pmc/articles/PMC3950223/figure/pone-0091580-g005/" class="imagepopup">Figure 5. Schematic drawing outlining the <span class="highlight" style="background-color:">pathway</span> of the Rho-GDP dissociation inhibitor alpha, GTP-Rho, Rock and p-cofilin that mediates the enhancement of migration and cytoskeletal reorganization by cyclic strain treatment..  From: Cyclic Strain-Induced Cytoskeletal Rearrangement of Human Periodontal Ligament Cells via the Rho <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Inhibitors and activators of each of the molecules in the <span class="highlight" style="background-color:">pathway</span> regulate the effect of cyclic strain on the member(s) downstream, but not upstream. These studies allow the determination of the hierarchical relationships among the <span class="highlight" style="background-color:">signaling</span> molecules shown in this diagram.</div></p></div><div class="aux"><div class="resc">Jinsong Pan, et al. PLoS One. 2014;9(3):e91580.</div><p class="links"><a class="dblinks" href="/pubmed/24619024" ref="ordinalpos=325&amp;ncbi_uid=5906025&amp;link_uid=24619024">Citation</a><a class="dblinks" href="/pmc/articles/PMC3950223/" ref="ordinalpos=325&amp;ncbi_uid=5906025&amp;link_uid=PMC3950223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>326.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3417165/figure/F1/" ref="ordinalpos=326&amp;ncbi_uid=4180959&amp;link_uid=PMC3417165" href="/pmc/articles/PMC3417165/"><img src="/pmc/articles/instance/3417165/bin/fimmu-03-00245-g001.gif" src-large="/pmc/articles/instance/3417165/bin/fimmu-03-00245-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3417165/" ref="ordinalpos=326&amp;ncbi_uid=4180959&amp;link_uid=PMC3417165" image-link="/pmc/articles/PMC3417165/figure/F1/" class="imagepopup">Figure 1.  From: The Role of the PI3K <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in CD4+ T Cell Differentiation and Function. </a></p><div class="supp"><p class="details"><div><b>The role of PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in T<sub>regs</sub></b>. Arrow indicates activating phosphorylation; line with a perpendicular line at the end indicates inhibitory phosphorylation; line with a circle at the end indicates dephosphorylation; dashed line indicates resulting outcome of <span class="highlight" style="background-color:">signaling</span>. Green represents components of the PI3K/AKT <span class="highlight" style="background-color:">pathway</span> which have been shown to be beneficial for T<sub>reg</sub> function and/or development. On the contrary, red indicates molecules with activity thought to be inhibitory for T<sub>reg</sub> function and/or development.</div></p></div><div class="aux"><div class="resc">Jonathan M. Han, et al. Front Immunol. 2012;3:245.</div><p class="links"><a class="dblinks" href="/pubmed/22905034" ref="ordinalpos=326&amp;ncbi_uid=4180959&amp;link_uid=22905034">Citation</a><a class="dblinks" href="/pmc/articles/PMC3417165/" ref="ordinalpos=326&amp;ncbi_uid=4180959&amp;link_uid=PMC3417165">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>327.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2441458/figure/F2/" ref="ordinalpos=327&amp;ncbi_uid=1388138&amp;link_uid=PMC2441458" href="/pmc/articles/PMC2441458/"><img src="/pmc/articles/instance/2441458/bin/gb-2008-9-5-224-2.gif" src-large="/pmc/articles/instance/2441458/bin/gb-2008-9-5-224-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2441458/" ref="ordinalpos=327&amp;ncbi_uid=1388138&amp;link_uid=PMC2441458" image-link="/pmc/articles/PMC2441458/figure/F2/" class="imagepopup">Figure 2.  From: Glypicans. </a></p><div class="supp"><p class="details"><div>Positive and negative effects of GPC3 on cell <span class="highlight" style="background-color:">signaling</span>. In the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (left), GPC3 exerts a positive effect. Wnt binds to the receptor Frizzled to induce <span class="highlight" style="background-color:">signaling</span> (green arrow). GPC3 facilitates and/or stabilizes the interaction between Wnt and Frizzled with the consequent increment on <span class="highlight" style="background-color:">signaling</span>. In the Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (right), GPC3 exerts an inhibitory effect. The binding of Hh to the receptor Patched (Ptc) triggers the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by blocking the inhibitory activity of Ptc on Smoothened. GPC3 competes with Ptc for Hh binding. The interaction of Hh with GPC3 triggers the endocytosis and degradation of the complex with the consequent reduction of Hh available for binding to Ptc.</div></p></div><div class="aux"><div class="resc">Jorge Filmus, et al. Genome Biol. 2008;9(5):224-224.</div><p class="links"><a class="dblinks" href="/pubmed/18505598" ref="ordinalpos=327&amp;ncbi_uid=1388138&amp;link_uid=18505598">Citation</a><a class="dblinks" href="/pmc/articles/PMC2441458/" ref="ordinalpos=327&amp;ncbi_uid=1388138&amp;link_uid=PMC2441458">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>328.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4138095/figure/pone-0104282-g006/" ref="ordinalpos=328&amp;ncbi_uid=6517709&amp;link_uid=PMC4138095" href="/pmc/articles/PMC4138095/"><img src="/pmc/articles/instance/4138095/bin/pone.0104282.g006.gif" src-large="/pmc/articles/instance/4138095/bin/pone.0104282.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4138095/" ref="ordinalpos=328&amp;ncbi_uid=6517709&amp;link_uid=PMC4138095" image-link="/pmc/articles/PMC4138095/figure/pone-0104282-g006/" class="imagepopup">Figure 6. Integrative pathways from different cross-talking <span class="highlight" style="background-color:">pathway</span> pairs..  From: Integrating Multi-Omics for Uncovering the Architecture of Cross-Talking Pathways in Breast Cancer. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) “Complement and coagulation cascades” and “<span class="highlight" style="background-color:">Pathway</span> in cancer_1”, “Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>” and “<span class="highlight" style="background-color:">Pathway</span> in cancer_2”; (<b>B</b>) “Axon guidance” and “MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>”. Each ellipse represents a gene in KEGG <span class="highlight" style="background-color:">pathway</span> or ppi network. For a given gene, the ellipse is divided into four parts (from left to right) corresponding to different molecular level with abnormal states (red) in mRNA expression, (green) DNA methylation, (yellow) copy number and blue (somatic mutation), respectively.</div></p></div><div class="aux"><div class="resc">Li Wang, et al. PLoS One. 2014;9(8):e104282.</div><p class="links"><a class="dblinks" href="/pubmed/25137136" ref="ordinalpos=328&amp;ncbi_uid=6517709&amp;link_uid=25137136">Citation</a><a class="dblinks" href="/pmc/articles/PMC4138095/" ref="ordinalpos=328&amp;ncbi_uid=6517709&amp;link_uid=PMC4138095">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>329.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2533523/figure/Fig2/" ref="ordinalpos=329&amp;ncbi_uid=1602124&amp;link_uid=PMC2533523" href="/pmc/articles/PMC2533523/"><img src="/pmc/articles/instance/2533523/bin/11693_2007_9007_Fig2_HTML.gif" src-large="/pmc/articles/instance/2533523/bin/11693_2007_9007_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2533523/" ref="ordinalpos=329&amp;ncbi_uid=1602124&amp;link_uid=PMC2533523" image-link="/pmc/articles/PMC2533523/figure/Fig2/" class="imagepopup">Fig. 2.  From: Specificity of MAPK <span class="highlight" style="background-color:">signaling</span> towards FLO11 expression is established by crosstalk from cAMP <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><i>Effect of cross-activation of MAPK <span class="highlight" style="background-color:">pathway</span> by cAMP <span class="highlight" style="background-color:">pathway</span></i>. (<b>a</b>) Fractional dose response is shown with respect to Kss1p of MAPK <span class="highlight" style="background-color:">pathway</span>, with cross-activation of MAPK <span class="highlight" style="background-color:">pathway</span> by Tpks of cAMP <span class="highlight" style="background-color:">pathway</span> included. Curve ‘a’ shows the case without cross-activation. The strength of cross activation was varied by varing the ‘Km’ value of Tpks for the transcriptional activator Ste12Tec1 complex, b (1500 nM), c (600 nM), d (450 nM), e (375 nM), f (330 nM) and g (300 nM). (<b>b</b>) <i>FLO11</i> expression with respect to upstream activator Ras2GTP. Curve ‘a’ represents the case without cross-activation of MAPK <span class="highlight" style="background-color:">pathway</span>, whereas curve ‘b’ represents the case with cross-activation (Km = 450 nM). The requirement of Ras2GTP decreases with cross-activation. (<b>c</b>) Fractional dose response with respect to upstream activator of cAMP <span class="highlight" style="background-color:">pathway</span> (Gpa2p) and MAPK <span class="highlight" style="background-color:">pathway</span> (Ras2GTP), when the fractional activation of either <span class="highlight" style="background-color:">pathway</span> was fixed at 20% while the other was varied in presence of cross-activation. Curve ‘a’ represents dose response when fractional activation of cAMP <span class="highlight" style="background-color:">pathway</span> was restricted to 20%. Curve ‘b’ represents dose response when fractional activation of MAPK <span class="highlight" style="background-color:">pathway</span> was restricted to 20%</div></p></div><div class="aux"><div class="resc">P. K. Vinod, et al. Syst Synth Biol. 2007 April;1(2):99-108.</div><p class="links"><a class="dblinks" href="/pubmed/19003439" ref="ordinalpos=329&amp;ncbi_uid=1602124&amp;link_uid=19003439">Citation</a><a class="dblinks" href="/pmc/articles/PMC2533523/" ref="ordinalpos=329&amp;ncbi_uid=1602124&amp;link_uid=PMC2533523">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>330.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2398769/figure/F4/" ref="ordinalpos=330&amp;ncbi_uid=1432898&amp;link_uid=PMC2398769" href="/pmc/articles/PMC2398769/"><img src="/pmc/articles/instance/2398769/bin/nihms47946f4.gif" src-large="/pmc/articles/instance/2398769/bin/nihms47946f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2398769/" ref="ordinalpos=330&amp;ncbi_uid=1432898&amp;link_uid=PMC2398769" image-link="/pmc/articles/PMC2398769/figure/F4/" class="imagepopup">Fig. 4.  From: Progesterone Receptors: Form and Function in Brain. </a></p><div class="supp"><p class="details"><div>Model of progesterone-induced neuroprotective <span class="highlight" style="background-color:">signaling</span>. Progestogen prevents synaptic dysfunction associated with aging and neurodegeneration through (1) ligand activation of a progesterone receptor, (2) which initiates second messenger <span class="highlight" style="background-color:">signaling</span> cascades (3) to promote neuronal survival. These <span class="highlight" style="background-color:">signaling</span> cascades converge on mitochondrial function to protect against against toxic insults and include the passive prevention <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the active protection <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the passive prevention <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, both ERK/CREB/Bcl-2 and Akt pathways are simultaneously activated, which (4) enhances mitochondrial function and enables neurons to better withstand neurodegenerative insults. The active protection <span class="highlight" style="background-color:">pathway</span> acts to block Aβ-induced JNK activation and mitochondrial dysfunction.</div></p></div><div class="aux"><div class="resc">Roberta Diaz Brinton, et al. Front Neuroendocrinol. ;29(2):313-339.</div><p class="links"><a class="dblinks" href="/pubmed/18374402" ref="ordinalpos=330&amp;ncbi_uid=1432898&amp;link_uid=18374402">Citation</a><a class="dblinks" href="/pmc/articles/PMC2398769/" ref="ordinalpos=330&amp;ncbi_uid=1432898&amp;link_uid=PMC2398769">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>331.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4259472/figure/pone-0113830-g006/" ref="ordinalpos=331&amp;ncbi_uid=6961704&amp;link_uid=PMC4259472" href="/pmc/articles/PMC4259472/"><img src="/pmc/articles/instance/4259472/bin/pone.0113830.g006.gif" src-large="/pmc/articles/instance/4259472/bin/pone.0113830.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4259472/" ref="ordinalpos=331&amp;ncbi_uid=6961704&amp;link_uid=PMC4259472" image-link="/pmc/articles/PMC4259472/figure/pone-0113830-g006/" class="imagepopup">Figure 6. Notch <span class="highlight" style="background-color:">pathway</span> was regulated in Oridonin in HUVECs..  From: Oridonin Inhibits Tumor Growth and Metastasis through Anti-Angiogenesis by Blocking the Notch <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) DAPT blocked Notch gene transcription and Notch activity. (B) and (C) Oridonin remarkably inhibited expression of key ligands, receptors and downstream genes (Fig. 6B, Q-PCR) and proteins (Fig. 6C, Western blot) involved in the Jagged-Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. (D) Schematic diagram of Oridonin inhibiting tumor growth and tumor metastasis by regulating the Jagged-Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HUVECs.</div></p></div><div class="aux"><div class="resc">Yanmin Dong, et al. PLoS One. 2014;9(12):e113830.</div><p class="links"><a class="dblinks" href="/pubmed/25485753" ref="ordinalpos=331&amp;ncbi_uid=6961704&amp;link_uid=25485753">Citation</a><a class="dblinks" href="/pmc/articles/PMC4259472/" ref="ordinalpos=331&amp;ncbi_uid=6961704&amp;link_uid=PMC4259472">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>332.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3732195/figure/F2/" ref="ordinalpos=332&amp;ncbi_uid=5550401&amp;link_uid=PMC3732195" href="/pmc/articles/PMC3732195/"><img src="/pmc/articles/instance/3732195/bin/nihms-430191-f0002.gif" src-large="/pmc/articles/instance/3732195/bin/nihms-430191-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3732195/" ref="ordinalpos=332&amp;ncbi_uid=5550401&amp;link_uid=PMC3732195" image-link="/pmc/articles/PMC3732195/figure/F2/" class="imagepopup">Figure 2. Activation of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in trastuzumab resistant cell lines.  From: Expression of Wnt3 activates Wnt/?-catenin <span class="highlight" style="background-color:">pathway</span> and promotes EMT-like phenotype in trastuzumab resistant HER2-overexpressing breast cancer cells. </a></p><div class="supp"><p class="details"><div>A, Bar graph representing significantly up-regulated (open bars) or down-regulated (darkc bars) mRNA of Wnt <span class="highlight" style="background-color:">pathway</span> genes in SKBR3/100-8 compared to SKBR3 analyzed by microarray analysis. B, Bar graph of regulated Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> in SKBR3/100-8 (dark bars) and BT474/100-2 (open bars) compared to SKBR3 and BT474 analyzed by specific Wnt/β-catenin mRNA array. The relative level of each indicated gene was adjusted with GAPDH and each bar indicates mean fold change and SD from four determinations.</div></p></div><div class="aux"><div class="resc">Yanyuan Wu, et al. Mol Cancer Res. ;10(12):1597-1606.</div><p class="links"><a class="dblinks" href="/pubmed/23071104" ref="ordinalpos=332&amp;ncbi_uid=5550401&amp;link_uid=23071104">Citation</a><a class="dblinks" href="/pmc/articles/PMC3732195/" ref="ordinalpos=332&amp;ncbi_uid=5550401&amp;link_uid=PMC3732195">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>333.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3998120/figure/F4/" ref="ordinalpos=333&amp;ncbi_uid=6054793&amp;link_uid=PMC3998120" href="/pmc/articles/PMC3998120/"><img src="/pmc/articles/instance/3998120/bin/nihms571268f4.gif" src-large="/pmc/articles/instance/3998120/bin/nihms571268f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3998120/" ref="ordinalpos=333&amp;ncbi_uid=6054793&amp;link_uid=PMC3998120" image-link="/pmc/articles/PMC3998120/figure/F4/" class="imagepopup">Figure 4. Amplification and Expansion of Oncogenic Pathways as Metastatic Traits.  From: Origins of Metastatic Traits. </a></p><div class="supp"><p class="details"><div>(A) Metastatic traits acquired by a quantitative gain in <span class="highlight" style="background-color:">pathway</span> output. Amplification of the <span class="highlight" style="background-color:">signaling</span> capacity of cell survival and proliferation pathways provides survival and stemness advantages to disseminated cancer cells. In this case, the level of metastatic fitness is proportional to the robustness of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In primary tumors with a relatively high abundance of growth and survival signals, the <span class="highlight" style="background-color:">signaling</span> capacity of oncogenic pathways such as PI3K, Notch, and Wnt, is sufficient to support tumor growth. However, for disseminated cancer cells reaching distant tissue microenvironment where <span class="highlight" style="background-color:">pathway</span>-activating signals are scarce, the <span class="highlight" style="background-color:">signaling</span> capacity of these pathways is not sufficient for survival. The traits selected under such pressure include the expression of components that amplify the <span class="highlight" style="background-color:">signaling</span> capacity of the <span class="highlight" style="background-color:">pathway</span> in response to limiting levels of <span class="highlight" style="background-color:">pathway</span> activators in the host microenvironment. In triple-negative breast cancer examples include VCAM-1 and SRC as amplifiers of the PI3K <span class="highlight" style="background-color:">pathway</span> in cancer cells reaching the lungs or the bone marrow, respectively, whereas Tenascin C and Periostin act as amplifiers of the WNT and NOTCH pathways in cancer cells reaching the lungs.<br />(B) Metastatic traits acquired by a qualitative expansion of <span class="highlight" style="background-color:">pathway</span> output. Tumor-initiating pathways may additionally provide prometastatic traits by gaining access to subsets of target genes that enhance the homing of disseminated cancer cells to sites of survival. In this case, the level of meta-static fitness is not linearly proportional to the <span class="highlight" style="background-color:">signaling</span> strength of the <span class="highlight" style="background-color:">pathway</span> but depends of the <span class="highlight" style="background-color:">pathway</span> activating an additional set of effectors. One example is provided by the expansion of target genes that the HIF <span class="highlight" style="background-color:">pathway</span> activates in renal cell carcinoma as a result of epigenetic modifications that open the gene promoters to access by activated HIF.</div></p></div><div class="aux"><div class="resc">Sakari Vanharanta, et al. Cancer Cell. 2013 October 14;24(4):410-421.</div><p class="links"><a class="dblinks" href="/pubmed/24135279" ref="ordinalpos=333&amp;ncbi_uid=6054793&amp;link_uid=24135279">Citation</a><a class="dblinks" href="/pmc/articles/PMC3998120/" ref="ordinalpos=333&amp;ncbi_uid=6054793&amp;link_uid=PMC3998120">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>334.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4026686/figure/F1/" ref="ordinalpos=334&amp;ncbi_uid=6165612&amp;link_uid=PMC4026686" href="/pmc/articles/PMC4026686/"><img src="/pmc/articles/instance/4026686/bin/fendo-05-00071-g001.gif" src-large="/pmc/articles/instance/4026686/bin/fendo-05-00071-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4026686/" ref="ordinalpos=334&amp;ncbi_uid=6165612&amp;link_uid=PMC4026686" image-link="/pmc/articles/PMC4026686/figure/F1/" class="imagepopup">Figure 1.  From: Diversity and Bias through Receptor-Receptor Interactions in GPCR Heteroreceptor Complexes. Focus on Examples from Dopamine D2 Receptor Heteromerization. </a></p><div class="supp"><p class="details"><div><b>Illustration of changes in cAMP–pCREB and MAPK–SRE <span class="highlight" style="background-color:">signaling</span> pathways induced by agonist actions at NTR1–D2R heterodimers and at NTR1 receptor homodimers and D2 receptor homodimers</b>. NT produces via an antagonistic allosteric NTR1–D2 receptor–receptor interaction a reduction in the D2R agonist-induced activation of Gi/o leading to a marked reduction of activity in the cAMP–pCREB <span class="highlight" style="background-color:">pathway</span>. Instead, β-arrestin-mediated internalization and <span class="highlight" style="background-color:">signaling</span> may be favored due to a biased modulation of the D2R protomer via the allosteric receptor–receptor interaction in the heterodimer (see left part). The coagonist treatment also results in a facilitatory interaction in the activation of the MAPK–SRE <span class="highlight" style="background-color:">pathway</span>, which may take place in this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at the level of PKC. Thus, the NTR1 agonist can activate the NTR1 protomer, which activates the MAPK–SRE <span class="highlight" style="background-color:">pathway</span> via Gq/11–PLC–PKC. The D2R agonist can target the D2R protomer and produce its activation of the MAPK–SRE <span class="highlight" style="background-color:">pathway</span> via several mechanisms, including β-arrestin–PI3k–PKC or β-arrestin–Scr–PKC (see left part). It should also be noted that this facilitatory interaction can also involve the D2R receptor homodimers and the NTR1 receptor homodimers (see right part). However, the facilitatory interaction may become stronger in the NTR1–D2R heteromer <span class="highlight" style="background-color:">signaling</span> since the NT protomer-induced antagonism of D2R protomer-mediated Gi/o <span class="highlight" style="background-color:">signaling</span> may favor the switch toward β-arrestin recruitment and <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Kjell Fuxe, et al. Front Endocrinol (Lausanne). 2014;5:71.</div><p class="links"><a class="dblinks" href="/pubmed/24860548" ref="ordinalpos=334&amp;ncbi_uid=6165612&amp;link_uid=24860548">Citation</a><a class="dblinks" href="/pmc/articles/PMC4026686/" ref="ordinalpos=334&amp;ncbi_uid=6165612&amp;link_uid=PMC4026686">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>335.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3314445/figure/F9/" ref="ordinalpos=335&amp;ncbi_uid=3883796&amp;link_uid=PMC3314445" href="/pmc/articles/PMC3314445/"><img src="/pmc/articles/instance/3314445/bin/JCI57256.f9.gif" src-large="/pmc/articles/instance/3314445/bin/JCI57256.f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3314445/" ref="ordinalpos=335&amp;ncbi_uid=3883796&amp;link_uid=PMC3314445" image-link="/pmc/articles/PMC3314445/figure/F9/" class="imagepopup">Figure 9. Proposed mechanism underlying disrupted brain insulin <span class="highlight" style="background-color:">signaling</span> in AD..  From: An anti-diabetes agent protects the mouse brain from defective insulin <span class="highlight" style="background-color:">signaling</span> caused by Alzheimer's disease-associated A? oligomers. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) AβOs stimulate TNF-α <span class="highlight" style="background-color:">signaling</span>, which activates the JNK <span class="highlight" style="background-color:">pathway</span> and, possibly, PKR and IKK pathways. Activation of these stress-sensitive kinases, which can also be triggered by endoplasmic reticulum stress (36), results in serine phosphorylation of IRS-1, blocking downstream insulin <span class="highlight" style="background-color:">signaling</span>. (<b>B</b>) Stimulation of insulin and GLP1 receptors blocks AβO-induced defects in insulin <span class="highlight" style="background-color:">signaling</span>. Binding of exendin-4 to GLP1 receptors and of insulin to IRs prevented activation of JNK, allowing physiological tyrosine phosphorylation of IRS-1 and stimulating downstream insulin <span class="highlight" style="background-color:">signaling</span>. In both panels, red arrows indicate inhibitory pathways and green arrows indicate stimulatory pathways of insulin <span class="highlight" style="background-color:">signaling</span>. I, insulin. </div></p></div><div class="aux"><div class="resc">Theresa R. Bomfim, et al. J Clin Invest. 2012 April 2;122(4):1339-1353.</div><p class="links"><a class="dblinks" href="/pubmed/22476196" ref="ordinalpos=335&amp;ncbi_uid=3883796&amp;link_uid=22476196">Citation</a><a class="dblinks" href="/pmc/articles/PMC3314445/" ref="ordinalpos=335&amp;ncbi_uid=3883796&amp;link_uid=PMC3314445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>336.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2886346/figure/fig2/" ref="ordinalpos=336&amp;ncbi_uid=2630826&amp;link_uid=PMC2886346" href="/pmc/articles/PMC2886346/"><img src="/pmc/articles/instance/2886346/bin/JCB_200912082_RGB_Fig2.gif" src-large="/pmc/articles/instance/2886346/bin/JCB_200912082_RGB_Fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2886346/" ref="ordinalpos=336&amp;ncbi_uid=2630826&amp;link_uid=PMC2886346" image-link="/pmc/articles/PMC2886346/figure/fig2/" class="imagepopup">Figure 2. From: MAPK <span class="highlight" style="background-color:">signaling</span> to the early secretory <span class="highlight" style="background-color:">pathway</span> revealed by kinase/phosphatase functional screening. </a></p><div class="supp"><p class="details"><div><b>Network of kinase hits indicates a role for MAPK <span class="highlight" style="background-color:">signaling</span> in the regulation of ER export.</b> (A) All ER-class hits were assembled into a network using the STRING database. (B) Assembly of all of the Golgi-class hits into a network using the STRING database. Edges in both networks indicate an interaction that is experimentally verified or an association into a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (C) ER-, mixed-, and Golgi-class hits specifically enriched in <span class="highlight" style="background-color:">signaling</span> pathways deduced from the DAVID database.</div></p></div><div class="aux"><div class="resc">Hesso Farhan, et al. J Cell Biol. 2010 June 14;189(6):997-1011.</div><p class="links"><a class="dblinks" href="/pubmed/20548102" ref="ordinalpos=336&amp;ncbi_uid=2630826&amp;link_uid=20548102">Citation</a><a class="dblinks" href="/pmc/articles/PMC2886346/" ref="ordinalpos=336&amp;ncbi_uid=2630826&amp;link_uid=PMC2886346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>337.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3351275/figure/F3/" ref="ordinalpos=337&amp;ncbi_uid=4762203&amp;link_uid=PMC3351275" href="/pmc/articles/PMC3351275/"><img src="/pmc/articles/instance/3351275/bin/nihms362590f3.gif" src-large="/pmc/articles/instance/3351275/bin/nihms362590f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3351275/" ref="ordinalpos=337&amp;ncbi_uid=4762203&amp;link_uid=PMC3351275" image-link="/pmc/articles/PMC3351275/figure/F3/" class="imagepopup">Figure 3. Reciprocal feedback regulation of PI3K/AKT and AR <span class="highlight" style="background-color:">signaling</span>.  From: Negative feedback and adaptive resistance to the targeted therapy of cancer. </a></p><div class="supp"><p class="details"><div>On the left panel is a model depicting crosstalk between the PI3K/AKT <span class="highlight" style="background-color:">pathway</span> and the androgen receptor (AR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with each <span class="highlight" style="background-color:">pathway</span> negatively regulating the other. In the middle panel is the consequence of inhibiting AR <span class="highlight" style="background-color:">signaling</span> with downregulation of AR causing lower levels of FKBP5 and thus impairing the function of the AKT phosphatase PHLPP. The result is that AR inhibition causes an upregulation in AKT activity. In the right panel is the consequence of inhibiting PI3K/AKT <span class="highlight" style="background-color:">signaling</span> with downregulation of AKT causing an induction of RTKs such as HER3 and this causing an induction of AR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Sarat Chandarlapaty. Cancer Discov. ;2(4):311-319.</div><p class="links"><a class="dblinks" href="/pubmed/22576208" ref="ordinalpos=337&amp;ncbi_uid=4762203&amp;link_uid=22576208">Citation</a><a class="dblinks" href="/pmc/articles/PMC3351275/" ref="ordinalpos=337&amp;ncbi_uid=4762203&amp;link_uid=PMC3351275">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>338.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2100073/figure/F4/" ref="ordinalpos=338&amp;ncbi_uid=1122443&amp;link_uid=PMC2100073" href="/pmc/articles/PMC2100073/"><img src="/pmc/articles/instance/2100073/bin/1471-2105-8-335-4.gif" src-large="/pmc/articles/instance/2100073/bin/1471-2105-8-335-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2100073/" ref="ordinalpos=338&amp;ncbi_uid=1122443&amp;link_uid=PMC2100073" image-link="/pmc/articles/PMC2100073/figure/F4/" class="imagepopup">Figure 4.  From: PathFinder: mining signal transduction <span class="highlight" style="background-color:">pathway</span> segments from protein-protein interaction networks. </a></p><div class="supp"><p class="details"><div><b>The pheromone response <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. (A) The main chain of the pheromone <span class="highlight" style="background-color:">pathway</span> downloaded from KEGG, (B) the output of our PathFinder implementation, (C) The color-coding algorithm output for the pheromone <span class="highlight" style="background-color:">pathway</span> [20], (D) the NetSearch program prediction for the pheromone <span class="highlight" style="background-color:">pathway</span> [18]. The interactions that do not exist in the PPIN are shown with dashed edges in (A). Notice that, for (B),(C) and (D) the proteins that were not on the main chain of the <span class="highlight" style="background-color:">pathway</span>, as shown in (A), were not colored.</div></p></div><div class="aux"><div class="resc">Gurkan Bebek, et al. BMC Bioinformatics. 2007;8:335-335.</div><p class="links"><a class="dblinks" href="/pubmed/17854489" ref="ordinalpos=338&amp;ncbi_uid=1122443&amp;link_uid=17854489">Citation</a><a class="dblinks" href="/pmc/articles/PMC2100073/" ref="ordinalpos=338&amp;ncbi_uid=1122443&amp;link_uid=PMC2100073">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>339.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872883/figure/F1/" ref="ordinalpos=339&amp;ncbi_uid=2374795&amp;link_uid=PMC2872883" href="/pmc/articles/PMC2872883/"><img src="/pmc/articles/instance/2872883/bin/gb-2010-11-2-r23-1.gif" src-large="/pmc/articles/instance/2872883/bin/gb-2010-11-2-r23-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872883/" ref="ordinalpos=339&amp;ncbi_uid=2374795&amp;link_uid=PMC2872883" image-link="/pmc/articles/PMC2872883/figure/F1/" class="imagepopup">Figure 1.  From: Identification of functional modules that correlate with phenotypic difference: the influence of network topology. </a></p><div class="supp"><p class="details"><div><b>Pathways adapted from KEGG</b>. <b>(a) </b>Renal cell carcinoma. <b>(b) </b>MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(c) </b>Axon guidance. <b>(d) </b>Amyotrophic lateral sclerosis. <b>(e) </b>Fcε RI <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(f) </b>Gonadotropin-releasing hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(g) </b>Jak-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(h) </b>Basal cell carcinoma. Red indicates an abnormality.</div></p></div><div class="aux"><div class="resc">Jui-Hung Hung, et al. Genome Biol. 2010;11(2):R23-R23.</div><p class="links"><a class="dblinks" href="/pubmed/20187943" ref="ordinalpos=339&amp;ncbi_uid=2374795&amp;link_uid=20187943">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872883/" ref="ordinalpos=339&amp;ncbi_uid=2374795&amp;link_uid=PMC2872883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>340.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4169862/figure/F5/" ref="ordinalpos=340&amp;ncbi_uid=6642193&amp;link_uid=PMC4169862" href="/pmc/articles/PMC4169862/"><img src="/pmc/articles/instance/4169862/bin/aging-06-690-g005.gif" src-large="/pmc/articles/instance/4169862/bin/aging-06-690-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4169862/" ref="ordinalpos=340&amp;ncbi_uid=6642193&amp;link_uid=PMC4169862" image-link="/pmc/articles/PMC4169862/figure/F5/" class="imagepopup">Figure 5. hNAG-1 transgenic mice have reduced IGF-1/Insulin/mTOR <span class="highlight" style="background-color:">signaling</span>.  From: hNAG-1 increases lifespan by regulating energy metabolism and insulin/IGF-1/mTOR <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Phosphorylation of proteins of IGF-1/insulin/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was analyzed by Western blot (n=3) in abdominal WAT. Both samples from 30 wk and 95 wk old animals were analyzed.</div></p></div><div class="aux"><div class="resc">Xingya Wang, et al. Aging (Albany NY). 2014 August;6(8):690-704.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=340&amp;ncbi_uid=6642193&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4169862/" ref="ordinalpos=340&amp;ncbi_uid=6642193&amp;link_uid=PMC4169862">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>341.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3331679/figure/F2/" ref="ordinalpos=341&amp;ncbi_uid=4796890&amp;link_uid=PMC3331679" href="/pmc/articles/PMC3331679/"><img src="/pmc/articles/instance/3331679/bin/nihms369262f2.gif" src-large="/pmc/articles/instance/3331679/bin/nihms369262f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3331679/" ref="ordinalpos=341&amp;ncbi_uid=4796890&amp;link_uid=PMC3331679" image-link="/pmc/articles/PMC3331679/figure/F2/" class="imagepopup">Figure 2. The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: mTOR <span class="highlight" style="background-color:">signaling</span> in growth control and disease. </a></p><div class="supp"><p class="details"><div>(A) The key <span class="highlight" style="background-color:">signaling</span> nodes that regulate mTORC1 and mTORC2.<br />(B) The key outputs of the mTORC1 and mTORC2 pathways.<br />See text for details. Refer to the text and abbreviation list for details about the complete name of proteins.</div></p></div><div class="aux"><div class="resc">Mathieu Laplante, et al. Cell. ;149(2):274-293.</div><p class="links"><a class="dblinks" href="/pubmed/22500797" ref="ordinalpos=341&amp;ncbi_uid=4796890&amp;link_uid=22500797">Citation</a><a class="dblinks" href="/pmc/articles/PMC3331679/" ref="ordinalpos=341&amp;ncbi_uid=4796890&amp;link_uid=PMC3331679">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>342.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3269610/figure/F2/" ref="ordinalpos=342&amp;ncbi_uid=3784976&amp;link_uid=PMC3269610" href="/pmc/articles/PMC3269610/"><img src="/pmc/articles/instance/3269610/bin/ijbsv08p0272g02.gif" src-large="/pmc/articles/instance/3269610/bin/ijbsv08p0272g02.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3269610/" ref="ordinalpos=342&amp;ncbi_uid=3784976&amp;link_uid=PMC3269610" image-link="/pmc/articles/PMC3269610/figure/F2/" class="imagepopup">Figure 2.  From: TGF-? and BMP <span class="highlight" style="background-color:">Signaling</span> in Osteoblast Differentiation and Bone Formation. </a></p><div class="supp"><p class="details"><div>BMP <span class="highlight" style="background-color:">signaling</span> and negative regulation in bone formation. Smad-dependent-BMP <span class="highlight" style="background-color:">signaling</span> binds to receptor type II (R-II) and receptor type I (R-I) and then the <span class="highlight" style="background-color:">signaling</span> transduces to their Smads. Activated Smads form a complex with Smad4 and then translocate into the nucleus where they interact with other transcription factors to trigger target gene expression. Neogenin regulates BMP receptor association and Smad1/5/8 <span class="highlight" style="background-color:">signaling</span>. Activated Smads regulate expression of transcriptional factors and transcriptional coactivators important in osteoblasts (Dlx5, Runx2 and Osx). Smad6 binds type I BMP receptor and prevents Smad1/5/8 to be activated. Non-Smad-dependent TAK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> also regulates bone formation. The interplay between BMPs and Wnt <span class="highlight" style="background-color:">signaling</span> affects bone formation 99. BMPRIA <span class="highlight" style="background-color:">signaling</span> upregulates Sost expression primarily through Smad-dependent <span class="highlight" style="background-color:">signaling</span>, while it upregulates DKK1 through Smad-dependent and non-Smad-dependent <span class="highlight" style="background-color:">signaling</span>. Both Sost and DKK1 inhibit canonical Wnt <span class="highlight" style="background-color:">signaling</span>, leading to a decrease in bone mass. P: phosphorylation; Ub: ubiquitination.</div></p></div><div class="aux"><div class="resc">Guiqian Chen, et al. Int J Biol Sci. 2012;8(2):272-288.</div><p class="links"><a class="dblinks" href="/pubmed/22298955" ref="ordinalpos=342&amp;ncbi_uid=3784976&amp;link_uid=22298955">Citation</a><a class="dblinks" href="/pmc/articles/PMC3269610/" ref="ordinalpos=342&amp;ncbi_uid=3784976&amp;link_uid=PMC3269610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>343.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4091705/figure/pgen-1004481-g008/" ref="ordinalpos=343&amp;ncbi_uid=6395227&amp;link_uid=PMC4091705" href="/pmc/articles/PMC4091705/"><img src="/pmc/articles/instance/4091705/bin/pgen.1004481.g008.gif" src-large="/pmc/articles/instance/4091705/bin/pgen.1004481.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4091705/" ref="ordinalpos=343&amp;ncbi_uid=6395227&amp;link_uid=PMC4091705" image-link="/pmc/articles/PMC4091705/figure/pgen-1004481-g008/" class="imagepopup">Figure 8. Summary of RPM-1 <span class="highlight" style="background-color:">signaling</span> through the ANC-1/BAR-1 <span class="highlight" style="background-color:">pathway</span>..  From: The Nesprin Family Member ANC-1 Regulates Synapse Formation and Axon Termination by Functioning in a <span class="highlight" style="background-color:">Pathway</span> with RPM-1 and ?-Catenin. </a></p><div class="supp"><p class="details"><div>Canonical Wnt <span class="highlight" style="background-color:">signaling</span> through Disheveled (Dis) and APR-1 (APC ortholog) regulates the β-catenin BAR-1. In the absence of Wnt, APR-1 is active and inhibits BAR-1. In the presence of Wnt, APR-1 is inhibited and BAR-1 activity is increased. Higher levels of BAR-1 lead to increased nuclear import and activation of the transcription factor POP-1 (TCF/LEF). We have shown that RPM-1 binds to ANC-1, and RPM-1 and ANC-1 function in the same <span class="highlight" style="background-color:">pathway</span> to positively regulate BAR-1 activity. RPM-1 and ANC-1 are likely to function in a protein complex at the nuclear envelope to regulate BAR-1 nuclear levels by inhibition of EMR-1. Canonical Wnt <span class="highlight" style="background-color:">signaling</span> is likely to act coordinately with RPM-1 and ANC-1 to regulate BAR-1 activity.</div></p></div><div class="aux"><div class="resc">Erik D. Tulgren, et al. PLoS Genet. 2014 July;10(7):e1004481.</div><p class="links"><a class="dblinks" href="/pubmed/25010424" ref="ordinalpos=343&amp;ncbi_uid=6395227&amp;link_uid=25010424">Citation</a><a class="dblinks" href="/pmc/articles/PMC4091705/" ref="ordinalpos=343&amp;ncbi_uid=6395227&amp;link_uid=PMC4091705">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>344.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4152229/figure/F1/" ref="ordinalpos=344&amp;ncbi_uid=6551019&amp;link_uid=PMC4152229" href="/pmc/articles/PMC4152229/"><img src="/pmc/articles/instance/4152229/bin/nihms606900f1.gif" src-large="/pmc/articles/instance/4152229/bin/nihms606900f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4152229/" ref="ordinalpos=344&amp;ncbi_uid=6551019&amp;link_uid=PMC4152229" image-link="/pmc/articles/PMC4152229/figure/F1/" class="imagepopup">Figure 1. The apoptosis <span class="highlight" style="background-color:">pathway</span>.  From: Targeting the Apoptosis <span class="highlight" style="background-color:">Pathway</span> in Hematologic Malignancies. </a></p><div class="supp"><p class="details"><div>There are two apoptosis pathways, intrinsic and extrinsic. The intrinsic <span class="highlight" style="background-color:">pathway</span> is activated by cellular stress. When activated, the BH3-only proteins inhibit antiapoptotic BCL-2 proteins and induce BAK/BAX oligomerization and permeabilization of the mitochondria and release of cytochrome c and SMAC/DIABLO. Cytochrome c forms a complex with caspase-9 and APAF-1, leading to activation of caspase-9. Caspase-9 activates the executioner caspases (caspase-3, −6, and −7) and induces cell death. The extrinsic <span class="highlight" style="background-color:">pathway</span> is activated by death signals mediated by death ligands. The death ligand activates the death receptor, which forms a complex called the death-inducing <span class="highlight" style="background-color:">signaling</span> complex (DISC) and sends a signal to activate caspase-8 and −10. Caspase-8 and −10 activate caspase-3. In some cells, the extrinsic <span class="highlight" style="background-color:">pathway</span> crosstalks with the intrinsic <span class="highlight" style="background-color:">pathway</span> through caspase-8–mediated truncation of BID to tBID. tBID activates BAK/BAX oligomerization and induces apoptosis through the mitochondrial <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Shadia Zaman, et al. Leuk Lymphoma. ;55(9):1980-1992.</div><p class="links"><a class="dblinks" href="/pubmed/24295132" ref="ordinalpos=344&amp;ncbi_uid=6551019&amp;link_uid=24295132">Citation</a><a class="dblinks" href="/pmc/articles/PMC4152229/" ref="ordinalpos=344&amp;ncbi_uid=6551019&amp;link_uid=PMC4152229">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>345.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3811338/figure/F2/" ref="ordinalpos=345&amp;ncbi_uid=5449036&amp;link_uid=PMC3811338" href="/pmc/articles/PMC3811338/"><img src="/pmc/articles/instance/3811338/bin/zek9990941640002.gif" src-large="/pmc/articles/instance/3811338/bin/zek9990941640002.jpg" alt="Fig 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3811338/" ref="ordinalpos=345&amp;ncbi_uid=5449036&amp;link_uid=PMC3811338" image-link="/pmc/articles/PMC3811338/figure/F2/" class="imagepopup">Fig 2.  From: Ras <span class="highlight" style="background-color:">Signaling</span> Gets Fine-Tuned: Regulation of Multiple Pathogenic Traits of Candida albicans. </a></p><div class="supp"><p class="details"><div>Diagram of the central Ras/cAMP/PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components and examples of signals to which the Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> responds. TFs, transcription factors.</div></p></div><div class="aux"><div class="resc">Diane O. Inglis, et al. Eukaryot Cell. 2013 October;12(10):1316-1325.</div><p class="links"><a class="dblinks" href="/pubmed/23913542" ref="ordinalpos=345&amp;ncbi_uid=5449036&amp;link_uid=23913542">Citation</a><a class="dblinks" href="/pmc/articles/PMC3811338/" ref="ordinalpos=345&amp;ncbi_uid=5449036&amp;link_uid=PMC3811338">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>346.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285629/figure/pone-0032087-g008/" ref="ordinalpos=346&amp;ncbi_uid=3835202&amp;link_uid=PMC3285629" href="/pmc/articles/PMC3285629/"><img src="/pmc/articles/instance/3285629/bin/pone.0032087.g008.gif" src-large="/pmc/articles/instance/3285629/bin/pone.0032087.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285629/" ref="ordinalpos=346&amp;ncbi_uid=3835202&amp;link_uid=PMC3285629" image-link="/pmc/articles/PMC3285629/figure/pone-0032087-g008/" class="imagepopup">Figure 8. Model describing the role of Sprouty4 in NGF-induced TrkA receptor <span class="highlight" style="background-color:">signaling</span> and biology..  From: Sprouty4 Is an Endogenous Negative Modulator of TrkA <span class="highlight" style="background-color:">Signaling</span> and Neuronal Differentiation Induced by NGF. </a></p><div class="supp"><p class="details"><div>NGF-TrkA <span class="highlight" style="background-color:">signaling</span> induces Sprouty4 expression via MAPK <span class="highlight" style="background-color:">pathway</span>. Then, Sprouty4 acting through a negative feedback loop specifically inhibits MAPK <span class="highlight" style="background-color:">pathway</span> and Rac1 activation, thereby preventing neuronal differentiation of PC12 and DRG neurons.</div></p></div><div class="aux"><div class="resc">Fernando C. Alsina, et al. PLoS One. 2012;7(2):e32087.</div><p class="links"><a class="dblinks" href="/pubmed/22384148" ref="ordinalpos=346&amp;ncbi_uid=3835202&amp;link_uid=22384148">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285629/" ref="ordinalpos=346&amp;ncbi_uid=3835202&amp;link_uid=PMC3285629">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>347.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3344687/figure/F7/" ref="ordinalpos=347&amp;ncbi_uid=3986353&amp;link_uid=PMC3344687" href="/pmc/articles/PMC3344687/"><img src="/pmc/articles/instance/3344687/bin/1742-2094-9-46-7.gif" src-large="/pmc/articles/instance/3344687/bin/1742-2094-9-46-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3344687/" ref="ordinalpos=347&amp;ncbi_uid=3986353&amp;link_uid=PMC3344687" image-link="/pmc/articles/PMC3344687/figure/F7/" class="imagepopup">Figure 7.  From: Heme activates TLR4-mediated inflammatory injury via MyD88/TRIF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in intracerebral hemorrhage. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the Heme/TLR4/MyD88 and/or TRIF hypothesis in ICH</b>. Heme activates TLR4-mediated inflammatory injury <i>via </i>the MyD88/TRIF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in intracerebral hemorrhage mouse brain.</div></p></div><div class="aux"><div class="resc">Sen Lin, et al. J Neuroinflammation. 2012;9:46-46.</div><p class="links"><a class="dblinks" href="/pubmed/22394415" ref="ordinalpos=347&amp;ncbi_uid=3986353&amp;link_uid=22394415">Citation</a><a class="dblinks" href="/pmc/articles/PMC3344687/" ref="ordinalpos=347&amp;ncbi_uid=3986353&amp;link_uid=PMC3344687">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>348.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4039177/figure/F1/" ref="ordinalpos=348&amp;ncbi_uid=6181521&amp;link_uid=PMC4039177" href="/pmc/articles/PMC4039177/"><img src="/pmc/articles/instance/4039177/bin/nihms199097f1.gif" src-large="/pmc/articles/instance/4039177/bin/nihms199097f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4039177/" ref="ordinalpos=348&amp;ncbi_uid=6181521&amp;link_uid=PMC4039177" image-link="/pmc/articles/PMC4039177/figure/F1/" class="imagepopup">Figure 1. Itraconazole inhibits Hh <span class="highlight" style="background-color:">signaling</span>.  From: Itraconazole, a commonly used anti-fungal that inhibits Hedgehog <span class="highlight" style="background-color:">pathway</span> activity and cancer growth. </a></p><div class="supp"><p class="details"><div>(A) A schematic view of the Hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the absence of Hh, Patched (Ptch) suppresses Smoothened (Smo) function. Hh, when present, binds to and inhibits Ptch, permitting Smo accumulation in the primary cilium (not shown) and causing activation of the <span class="highlight" style="background-color:">pathway</span> via the Gli family of transcription factors. <i>Ptch</i> and <i>Gli1</i> are themselves transcriptional targets of the <span class="highlight" style="background-color:">pathway</span>. Oxysterols (dashed green bracket) act between Ptch and Smo, as <span class="highlight" style="background-color:">pathway</span> activators, whereas statins (dashed red bracket) act downstream of Ptch and at or upstream of Smo, as <span class="highlight" style="background-color:">pathway</span> inhibitors. SAG and cyclopamine activate and inhibit the <span class="highlight" style="background-color:">pathway</span>, respectively, by binding to the transmembrane domain of Smo. Activators and inhibitors of the <span class="highlight" style="background-color:">pathway</span> are labeled in green and red, respectively. (B) Hh <span class="highlight" style="background-color:">signaling</span> assays. Luciferase reporter activity under the control of an 8×-Gli binding site in the Shh-Light2 reporter cell line was measured upon stimulation with ShhN-containing medium. Itraconazole blocked Hh <span class="highlight" style="background-color:">pathway</span> activity (IC<sub>50</sub>≈800 nM). (C) Schematic view of mammalian cholesterol biosynthesis from 3-hydroxy-3-methyl-glutaryl-CoA (HMG-CoA). Statins inhibit HMG-CoA reductase whereas azole antifungal drugs inhibit 14α-lanosterol demethylase (14LDM), as indicated. Lathosterol and desmosterol are cholesterol precursors downstream of 14LDM. (D) Among the azole antifungals, itraconazole was the most potent inhibitor of Hh <span class="highlight" style="background-color:">pathway</span> activity. (E) Hydroxy-itraconazole, the major metabolite of itraconazole in mammals, also inhibited the Hh <span class="highlight" style="background-color:">pathway</span> (IC<sub>50</sub> ≈1.2 µM). All <span class="highlight" style="background-color:">signaling</span> assays were performed with Shh-Light2 cells in 0.5% serum media and data are shown as the mean of triplicates ± s.d.<br />See also Figure S1 and Table S1.</div></p></div><div class="aux"><div class="resc">James Kim, et al. Cancer Cell. ;17(4):388-399.</div><p class="links"><a class="dblinks" href="/pubmed/20385363" ref="ordinalpos=348&amp;ncbi_uid=6181521&amp;link_uid=20385363">Citation</a><a class="dblinks" href="/pmc/articles/PMC4039177/" ref="ordinalpos=348&amp;ncbi_uid=6181521&amp;link_uid=PMC4039177">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>349.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3335938/figure/F1/" ref="ordinalpos=349&amp;ncbi_uid=4447977&amp;link_uid=PMC3335938" href="/pmc/articles/PMC3335938/"><img src="/pmc/articles/instance/3335938/bin/cbt1212_1032_fig001.gif" src-large="/pmc/articles/instance/3335938/bin/cbt1212_1032_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3335938/" ref="ordinalpos=349&amp;ncbi_uid=4447977&amp;link_uid=PMC3335938" image-link="/pmc/articles/PMC3335938/figure/F1/" class="imagepopup">Figure 1.  From: The Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of key therapeutic targets in the PI3K/Akt and MAPK pathways together with pharmacological agents inhibiting members of these <span class="highlight" style="background-color:">signaling</span> cascades. Shown are the key therapeutic targets regulating melanoma development. Whereas PI3K/Akt3 <span class="highlight" style="background-color:">pathway</span> involved in cell survival and apoptosis regulation, MAPK <span class="highlight" style="background-color:">pathway</span> is implicated in cell proliferation. Targeting members of these <span class="highlight" style="background-color:">signaling</span> cascades using pharmacological agents has been shown to inhibit melanoma tumor development. Pharmacological agents inhibiting PI3K/Akt <span class="highlight" style="background-color:">signaling</span>; perifosine, 17-AAG, CCI-779 and RAD-001 have been evaluated in clinical trials. Vemurafenib and sorafenib targeting MAPK are also being evaluated in various clinical trials for inhibiting melanomas.</div></p></div><div class="aux"><div class="resc">SubbaRao V Madhunapantula, et al. Cancer Biol Ther. 2011 December 15;12(12):1032-1049.</div><p class="links"><a class="dblinks" href="/pubmed/22157148" ref="ordinalpos=349&amp;ncbi_uid=4447977&amp;link_uid=22157148">Citation</a><a class="dblinks" href="/pmc/articles/PMC3335938/" ref="ordinalpos=349&amp;ncbi_uid=4447977&amp;link_uid=PMC3335938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>350.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2793289/figure/F6/" ref="ordinalpos=350&amp;ncbi_uid=2088973&amp;link_uid=PMC2793289" href="/pmc/articles/PMC2793289/"><img src="/pmc/articles/instance/2793289/bin/zmk0240992820006.gif" src-large="/pmc/articles/instance/2793289/bin/zmk0240992820006.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2793289/" ref="ordinalpos=350&amp;ncbi_uid=2088973&amp;link_uid=PMC2793289" image-link="/pmc/articles/PMC2793289/figure/F6/" class="imagepopup">Figure 6. From: Binding the Atypical RA Domain of Ste50p to the Unfolded Opy2p Cytoplasmic Tail Is Essential for the High-Osmolarity Glycerol <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Mapping Ste50p-RA domain surface residues for Opy2p interaction and their function in HOG <span class="highlight" style="background-color:">pathway</span> regulation described in Table 1 onto the structure of the Ste50p-RA domain. (A and B) Residues most affected in NMR experiments by binding of the s<sup>OPY2p</sup> and f<sup>OPY2p</sup> peptides, respectively. The most affected residues are in red, moderately affected in orange. (C) Residues mutated and functionally characterized as summarized in Table 1 are displayed as gray (mutation with no defect in the HOG <span class="highlight" style="background-color:">pathway</span> as compared with the wt Ste50p); red (mutations with defect in the HOG <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> specifically through the Opy2p-FID (=ΔSID); blue (mutations with defect in the HOG <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> specifically through the Opy2p-SID (=ΔFID); and violet (mutations defective in the HOG <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> even with the wt Opy2p).</div></p></div><div class="aux"><div class="resc">Irena Ekiel, et al. Mol Biol Cell. 2009 December 15;20(24):5117-5126.</div><p class="links"><a class="dblinks" href="/pubmed/19846660" ref="ordinalpos=350&amp;ncbi_uid=2088973&amp;link_uid=19846660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2793289/" ref="ordinalpos=350&amp;ncbi_uid=2088973&amp;link_uid=PMC2793289">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>351.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1783401/figure/F6/" ref="ordinalpos=351&amp;ncbi_uid=874228&amp;link_uid=PMC1783401" href="/pmc/articles/PMC1783401/"><img src="/pmc/articles/instance/1783401/bin/zpq0510645370006.gif" src-large="/pmc/articles/instance/1783401/bin/zpq0510645370006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1783401/" ref="ordinalpos=351&amp;ncbi_uid=874228&amp;link_uid=PMC1783401" image-link="/pmc/articles/PMC1783401/figure/F6/" class="imagepopup">Fig. 6. From: The SCL transcriptional network and BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> interact to regulate RUNX1 activity. </a></p><div class="supp"><p class="details"><div>Integration of Runx1 and the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> into the SCL transcriptional network. A schematic diagram of an emerging hematopoietic transcriptional network operating within the embryo during the establishment of definitive hematopoiesis in the AGM.</div></p></div><div class="aux"><div class="resc">John E. Pimanda, et al. Proc Natl Acad Sci U S A. 2007 January 16;104(3):840-845.</div><p class="links"><a class="dblinks" href="/pubmed/17213321" ref="ordinalpos=351&amp;ncbi_uid=874228&amp;link_uid=17213321">Citation</a><a class="dblinks" href="/pmc/articles/PMC1783401/" ref="ordinalpos=351&amp;ncbi_uid=874228&amp;link_uid=PMC1783401">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>352.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2997573/figure/F1/" ref="ordinalpos=352&amp;ncbi_uid=2804812&amp;link_uid=PMC2997573" href="/pmc/articles/PMC2997573/"><img src="/pmc/articles/instance/2997573/bin/nihms-158810-f0001.gif" src-large="/pmc/articles/instance/2997573/bin/nihms-158810-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2997573/" ref="ordinalpos=352&amp;ncbi_uid=2804812&amp;link_uid=PMC2997573" image-link="/pmc/articles/PMC2997573/figure/F1/" class="imagepopup">Figure 1.  From: PELP1: A novel therapeutic target for hormonal cancers. </a></p><div class="supp"><p class="details"><div>Schematic representation of the current understanding of PELP1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. PELP1 appears to function as a scaffolding protein that facilitates formation of various <span class="highlight" style="background-color:">signaling</span> complexes and its expression is deregulated in hormonal cancers. PELP1 regulates multiple <span class="highlight" style="background-color:">signaling</span> pathways including ER-genomic <span class="highlight" style="background-color:">signaling</span> by facilitating epigenetic changes, ER-nongenomic <span class="highlight" style="background-color:">signaling</span> by activating Src-MAPK <span class="highlight" style="background-color:">pathway</span>, local estrogen synthesis via ERRα - aromatase <span class="highlight" style="background-color:">signaling</span> and cell cycle progression via pRb/E2F <span class="highlight" style="background-color:">pathway</span>. PELP1 deregulation in hormonal tumors is likely to contribute to the development of tumorigenesis, metastasis and hormonal independence by activating multiple <span class="highlight" style="background-color:">signaling</span> pathways and by facilitating ER <span class="highlight" style="background-color:">signaling</span> cross talk. Insert in the middle shows PELP1 expression in advanced breast tumors as analyzed by IHC.</div></p></div><div class="aux"><div class="resc">Dimple Chakravarty, et al. IUBMB Life. ;62(3):162-169.</div><p class="links"><a class="dblinks" href="/pubmed/20014005" ref="ordinalpos=352&amp;ncbi_uid=2804812&amp;link_uid=20014005">Citation</a><a class="dblinks" href="/pmc/articles/PMC2997573/" ref="ordinalpos=352&amp;ncbi_uid=2804812&amp;link_uid=PMC2997573">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>353.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3890531/figure/F5/" ref="ordinalpos=353&amp;ncbi_uid=5704229&amp;link_uid=PMC3890531" href="/pmc/articles/PMC3890531/"><img src="/pmc/articles/instance/3890531/bin/1471-2164-14-904-5.gif" src-large="/pmc/articles/instance/3890531/bin/1471-2164-14-904-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3890531/" ref="ordinalpos=353&amp;ncbi_uid=5704229&amp;link_uid=PMC3890531" image-link="/pmc/articles/PMC3890531/figure/F5/" class="imagepopup">Figure 5.  From: An overview of gene expression dynamics during early ovarian folliculogenesis: specificity of follicular compartments and bi-directional dialog. </a></p><div class="supp"><p class="details"><div><b>VEGF and NOTCH <span class="highlight" style="background-color:">signaling</span> pathways.</b> Gene expression involved in the VEGF and NOTCH <span class="highlight" style="background-color:">signaling</span> pathways. Genes in red are over expressed in oocytes. Genes in green are over expressed in GCs. Relative expression of VEGF and FLT1 from RNAseq data. The Y axis corresponds to RNA-seq normalized read counts. <b>A</b>: Oocyte-GC communications in the NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>B</b>: GC-GC communications in the NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>C</b>: Oocyte-GC communications in the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Agnes Bonnet, et al. BMC Genomics. 2013;14:904-904.</div><p class="links"><a class="dblinks" href="/pubmed/24350644" ref="ordinalpos=353&amp;ncbi_uid=5704229&amp;link_uid=24350644">Citation</a><a class="dblinks" href="/pmc/articles/PMC3890531/" ref="ordinalpos=353&amp;ncbi_uid=5704229&amp;link_uid=PMC3890531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>354.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F1/" ref="ordinalpos=354&amp;ncbi_uid=2525568&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-1.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=354&amp;ncbi_uid=2525568&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F1/" class="imagepopup">Figure 1.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. B cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human taken from KEGG [9].</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=354&amp;ncbi_uid=2525568&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=354&amp;ncbi_uid=2525568&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>355.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3521148/figure/F4/" ref="ordinalpos=355&amp;ncbi_uid=4455285&amp;link_uid=PMC3521148" href="/pmc/articles/PMC3521148/"><img src="/pmc/articles/instance/3521148/bin/fendo-03-00162-g004.gif" src-large="/pmc/articles/instance/3521148/bin/fendo-03-00162-g004.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3521148/" ref="ordinalpos=355&amp;ncbi_uid=4455285&amp;link_uid=PMC3521148" image-link="/pmc/articles/PMC3521148/figure/F4/" class="imagepopup">FIGURE 4.  From: Mechanisms underlying the tissue-specific and regulated activity of the Gnrhr promoter in mammals. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> pathways in the homologous up-regulation of the mouse GnRHR promoter by GnRH in gonadotrope and non-gonadotrope cells</b>. In αT3−1 cells, GnRH-induced activation of the PKC-dependent <span class="highlight" style="background-color:">pathway</span> stimulates promoter activity through the MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, involving ERK1 and ERK2. Receptor-independent activation of the PKA-dependent <span class="highlight" style="background-color:">pathway</span> inhibits GnRH-stimulated promoter activity. In GGH3 cells, GnRH induces the simultaneous activation of the PKC and PKA-dependent pathways for stimulating promoter activity. Constitutive activation of the MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, involving ERK1 and ERK2, inhibits basal and GnRH-stimulated promoter activity. The three elements, SURG−1, SURG−2, and CRE therefore participate in GnRH up-regulation in both αT3−1 and GGH3 cells. In αT3−1 cells, the positive involvement of SURG−1 and SURG−2 have been demonstrated whereas that of the CRE element is only suspected to be inhibitory. In GGH3 cells, the CRE element is required for promoter stimulation and appears as the ultimate target of the PKA-dependent <span class="highlight" style="background-color:">pathway</span>. AP1, also referred to as SURG−2 is only potentially involved essentially because GnRH also activates the PKC-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in these cells.</div></p></div><div class="aux"><div class="resc">Anne-Laure Schang, et al. Front Endocrinol (Lausanne). 2012;3:162.</div><p class="links"><a class="dblinks" href="/pubmed/23248618" ref="ordinalpos=355&amp;ncbi_uid=4455285&amp;link_uid=23248618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3521148/" ref="ordinalpos=355&amp;ncbi_uid=4455285&amp;link_uid=PMC3521148">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>356.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3994550/figure/F1/" ref="ordinalpos=356&amp;ncbi_uid=6048010&amp;link_uid=PMC3994550" href="/pmc/articles/PMC3994550/"><img src="/pmc/articles/instance/3994550/bin/1471-2121-15-9-1.gif" src-large="/pmc/articles/instance/3994550/bin/1471-2121-15-9-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3994550/" ref="ordinalpos=356&amp;ncbi_uid=6048010&amp;link_uid=PMC3994550" image-link="/pmc/articles/PMC3994550/figure/F1/" class="imagepopup">Figure 1.  From: Leucine modulates dynamic phosphorylation events in insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and enhances insulin-dependent glycogen synthesis in human skeletal muscle cells. </a></p><div class="supp"><p class="details"><div><b>Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The scheme represents the current knowledge on insulin <span class="highlight" style="background-color:">signaling</span> downstream of the insulin receptor (IR). Proteins that were measured by our experiment are circled and marked with the corresponding phosphorylation sites.</div></p></div><div class="aux"><div class="resc">Barbara Di Camillo, et al. BMC Cell Biol. 2014;15:9-9.</div><p class="links"><a class="dblinks" href="/pubmed/24646332" ref="ordinalpos=356&amp;ncbi_uid=6048010&amp;link_uid=24646332">Citation</a><a class="dblinks" href="/pmc/articles/PMC3994550/" ref="ordinalpos=356&amp;ncbi_uid=6048010&amp;link_uid=PMC3994550">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>357.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3612391/figure/F2/" ref="ordinalpos=357&amp;ncbi_uid=5862355&amp;link_uid=PMC3612391" href="/pmc/articles/PMC3612391/"><img src="/pmc/articles/instance/3612391/bin/nihms-431589-f0002.gif" src-large="/pmc/articles/instance/3612391/bin/nihms-431589-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3612391/" ref="ordinalpos=357&amp;ncbi_uid=5862355&amp;link_uid=PMC3612391" image-link="/pmc/articles/PMC3612391/figure/F2/" class="imagepopup">Fig. 2. Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> in regulating vascular patterning via its interaction with VEGF <span class="highlight" style="background-color:">signaling</span>.  From: The role and mechanism-of-action of Sema3E and Plexin-D1 in vascular and neural development. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Tip cell dynamics are regulated by Sema3E-Plexin-D1 modulating Dll4-Notch <span class="highlight" style="background-color:">signaling</span> via a VEGF-induced negative feedback mechanism [15]. VEGF directly controls the expression of Plexin-D1 selectively in endothelial cells at the front of actively sprouting blood vessels. In turn, Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> inhibits the activity of VEGF-induced Dll4-Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which controls the balance between tip and stalk cells, thereby the retinal vascular network topology.<br /><b>(B)</b> Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> contributes to proper vasculature formation after pathological condition such as ischemic retinopathy [16]. Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> counteracts VEGF-mediated <span class="highlight" style="background-color:">signaling</span> and the small GTPase RhoJ might be involved in modulating the two pathways. Under the ischemic retinopathy condition, abnormal extraretinal vascular projections are suppressed by enhancing Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span>. <b>(C)</b> Plexin-D1 <span class="highlight" style="background-color:">signaling</span> induces soluble flt-1 expression to negatively control VEGF <span class="highlight" style="background-color:">signaling</span> in zebra fish truck [17]. During zebrafish segmental artery development, Plexin-D1-mediated <span class="highlight" style="background-color:">signaling</span> limits aorta’s angiogenic potential by inhibiting VEGF <span class="highlight" style="background-color:">signaling</span> via soluble flt-1. The specific member of Sema3 subfamily that mediates this event in the zebrafish is yet to be identified.</div></p></div><div class="aux"><div class="resc">Won-Jong Oh, et al. Semin Cell Dev Biol. 2013 March;24(3):156-162.</div><p class="links"><a class="dblinks" href="/pubmed/23270617" ref="ordinalpos=357&amp;ncbi_uid=5862355&amp;link_uid=23270617">Citation</a><a class="dblinks" href="/pmc/articles/PMC3612391/" ref="ordinalpos=357&amp;ncbi_uid=5862355&amp;link_uid=PMC3612391">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>358.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3092528/figure/F6/" ref="ordinalpos=358&amp;ncbi_uid=3317839&amp;link_uid=PMC3092528" href="/pmc/articles/PMC3092528/"><img src="/pmc/articles/instance/3092528/bin/nihms289056f6.gif" src-large="/pmc/articles/instance/3092528/bin/nihms289056f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3092528/" ref="ordinalpos=358&amp;ncbi_uid=3317839&amp;link_uid=PMC3092528" image-link="/pmc/articles/PMC3092528/figure/F6/" class="imagepopup">Figure 6. Schematic model of TROY-Pyk2-Rac1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in GBM cells.  From: TROY (TNFRSF19) is Overexpressed in Advanced Glial Tumors and Promotes Glioblastoma Cell Invasion via Pyk2-Rac1 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Activation of the TROY receptor by possible interaction with NgR1-LINGO complex elicits a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to recruit Pyk2 to the TROY receptor complex. Pyk2 functions to induce Rac1 activation potentially by influencing a Rac guanine nucleotide exchange factor (GEF) to drive glioma cell migration and invasion.</div></p></div><div class="aux"><div class="resc">Vincent M. Paulino, et al. Mol Cancer Res. ;8(11):1558-1567.</div><p class="links"><a class="dblinks" href="/pubmed/20881009" ref="ordinalpos=358&amp;ncbi_uid=3317839&amp;link_uid=20881009">Citation</a><a class="dblinks" href="/pmc/articles/PMC3092528/" ref="ordinalpos=358&amp;ncbi_uid=3317839&amp;link_uid=PMC3092528">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>359.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g005/" ref="ordinalpos=359&amp;ncbi_uid=3758044&amp;link_uid=PMC3256223" href="/pmc/articles/PMC3256223/"><img src="/pmc/articles/instance/3256223/bin/pone.0030140.g005.gif" src-large="/pmc/articles/instance/3256223/bin/pone.0030140.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3256223/" ref="ordinalpos=359&amp;ncbi_uid=3758044&amp;link_uid=PMC3256223" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g005/" class="imagepopup">Figure 5. Modeling the individual effect of 100 anti-miRNAs..  From: Modeling of miRNA and Drug Action in the EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A and B</b>: Each anti-miRNA is activated when the corresponding miRNA is over-expressed individually (an anti-mir-combination effect is omitted). Each row in this heatmap represents the predicted anti-miRNA effect and the more columns in the same row appears red or orange, the stronger is the predicted effect that this particular miRNA inhibitor can exert on the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In this manner, we can examine the impact of each anti-miRNA on this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jian Li, et al. PLoS One. 2012;7(1):e30140.</div><p class="links"><a class="dblinks" href="/pubmed/22253908" ref="ordinalpos=359&amp;ncbi_uid=3758044&amp;link_uid=22253908">Citation</a><a class="dblinks" href="/pmc/articles/PMC3256223/" ref="ordinalpos=359&amp;ncbi_uid=3758044&amp;link_uid=PMC3256223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>360.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3092909/figure/Fig5/" ref="ordinalpos=360&amp;ncbi_uid=3034663&amp;link_uid=PMC3092909" href="/pmc/articles/PMC3092909/"><img src="/pmc/articles/instance/3092909/bin/12263_2010_179_Fig5_HTML.gif" src-large="/pmc/articles/instance/3092909/bin/12263_2010_179_Fig5_HTML.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3092909/" ref="ordinalpos=360&amp;ncbi_uid=3034663&amp;link_uid=PMC3092909" image-link="/pmc/articles/PMC3092909/figure/Fig5/" class="imagepopup">Fig. 5.  From: Curcumin synergizes with resveratrol to stimulate the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human articular chondrocytes in vitro. </a></p><div class="supp"><p class="details"><div>Molecular model outlining the proposed mechanism for the stimulatory effects of resveratrol and curcumin on IL-1β- or U0126-induced MAPK <span class="highlight" style="background-color:">pathway</span> inhibition and apoptosis in primary chondrocytes in vitro<i>.</i> Signal transduction through β1-integrins leads to the activation of the Ras-MAP Kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activation of the MAPK <span class="highlight" style="background-color:">pathway</span> may be one of the mechanisms by which β1-integrins regulate chondrocyte gene expression. Previous studies have shown that activation of the MAPK <span class="highlight" style="background-color:">pathway</span> regulates the activity of a number of intracellular <span class="highlight" style="background-color:">signaling</span> proteins through phosphorylation. Erk1/Erk2, a downstream kinase of the MAPK <span class="highlight" style="background-color:">pathway</span>, regulates the expression and activity of various transcription factors. Specific inhibition of Erk1/Erk2 by U0126 or IL-1β results in cleavage of caspase-3 in human chondrocytes in vitro. Since activation of caspase-3 and DNA fragmentation are common features of apoptosis, the specific inhibition of the Ras-mitogen-activated kinase leads to chondrocyte apoptosis. Resveratrol and curcumin both block the inhibition effect of U0126 or IL-1β on the MAPK <span class="highlight" style="background-color:">pathway</span> in chondrocytes</div></p></div><div class="aux"><div class="resc">Mehdi Shakibaei, et al. Genes Nutr. 2011 May;6(2):171-179.</div><p class="links"><a class="dblinks" href="/pubmed/21484156" ref="ordinalpos=360&amp;ncbi_uid=3034663&amp;link_uid=21484156">Citation</a><a class="dblinks" href="/pmc/articles/PMC3092909/" ref="ordinalpos=360&amp;ncbi_uid=3034663&amp;link_uid=PMC3092909">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>361.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4368090/figure/f3-mmr-11-05-3219/" ref="ordinalpos=361&amp;ncbi_uid=7325725&amp;link_uid=PMC4368090" href="/pmc/articles/PMC4368090/"><img src="/pmc/articles/instance/4368090/bin/MMR-11-05-3219-g02.gif" src-large="/pmc/articles/instance/4368090/bin/MMR-11-05-3219-g02.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4368090/" ref="ordinalpos=361&amp;ncbi_uid=7325725&amp;link_uid=PMC4368090" image-link="/pmc/articles/PMC4368090/figure/f3-mmr-11-05-3219/" class="imagepopup">Figure 3.  From: Exploring the association between interleukin-1β and its interacting proteins in Alzheimer’s disease. </a></p><div class="supp"><p class="details"><div>Kyoto Encyclopedia of Genes and Genomes <span class="highlight" style="background-color:">pathway</span> created using ‘IL-1β’ as a key word. (A) Molecular map describing the responses triggered in the NF-κB <span class="highlight" style="background-color:">pathway</span>. (B) Molecular map describing the responses triggered in the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (C) Molecular map describing the responses triggered in the Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (D) Molecular map describing the responses triggered in the apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (E) Molecular map describing the responses triggered in the pertussis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (F) Molecular map describing the responses triggered in the legionellosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IL, interleukin, NF-κB, nuclear factor kappa B; MAPK, mitogen-activated protein kinase. Activation is reflected as a straight arrow and inhibition is reflected as an inverse ‘T’ shape. Indirect activation is reflected as a dotted arrow and indirect inhibition is reflected as a dotted inverted ‘T’ shape. Dissociation is reflected as equal lines and a missing interaction is reflected as a half arrow. The round-edged rectangle represents a membrane receptor and diamond shapes represent phosphorylation. The ‘V’ shape represents similarities with factors in Table II. The hexagon represents factors found in Fig. 1B, the ellipse with white color represents factors found in Fig. 1C and the octagon represents factors found in both Fig. 1B and C. Round-edged rectangles with white color indicate that both membrane receptor and be found in Fig. 1C and dotted round-edged rectangles indicated that both membrane receptor and be found in Table II. White dotted round-edged rectangles indicate that both membrane receptor and is found in Fig. 1B. Overlapping shapes represent a complex.</div></p></div><div class="aux"><div class="resc">LUSHUANG XIE, et al. Mol Med Rep. 2015 May;11(5):3219-3228.</div><p class="links"><a class="dblinks" href="/pubmed/25585621" ref="ordinalpos=361&amp;ncbi_uid=7325725&amp;link_uid=25585621">Citation</a><a class="dblinks" href="/pmc/articles/PMC4368090/" ref="ordinalpos=361&amp;ncbi_uid=7325725&amp;link_uid=PMC4368090">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>362.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3728205/figure/pone-0070456-g006/" ref="ordinalpos=362&amp;ncbi_uid=5181540&amp;link_uid=PMC3728205" href="/pmc/articles/PMC3728205/"><img src="/pmc/articles/instance/3728205/bin/pone.0070456.g006.gif" src-large="/pmc/articles/instance/3728205/bin/pone.0070456.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3728205/" ref="ordinalpos=362&amp;ncbi_uid=5181540&amp;link_uid=PMC3728205" image-link="/pmc/articles/PMC3728205/figure/pone-0070456-g006/" class="imagepopup">Figure 6. Apoptosis promoted by innexons may involve the inactivation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: A Role for Innexin2 and Innexin3 Proteins from Spodoptera litura in Apoptosis. </a></p><div class="supp"><p class="details"><div>P-Akt has been detected by using p-Akt antibody, Bm12 cell as a control of Hi5 cells (A) and the inactivation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> presented in apoptosis cells in Sf9 and spli221 (B). Assays were repeated at least three times.</div></p></div><div class="aux"><div class="resc">Tian Liu, et al. PLoS One. 2013;8(7):e70456.</div><p class="links"><a class="dblinks" href="/pubmed/23936205" ref="ordinalpos=362&amp;ncbi_uid=5181540&amp;link_uid=23936205">Citation</a><a class="dblinks" href="/pmc/articles/PMC3728205/" ref="ordinalpos=362&amp;ncbi_uid=5181540&amp;link_uid=PMC3728205">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>363.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3945187/figure/pgen-1004193-g005/" ref="ordinalpos=363&amp;ncbi_uid=5909275&amp;link_uid=PMC3945187" href="/pmc/articles/PMC3945187/"><img src="/pmc/articles/instance/3945187/bin/pgen.1004193.g005.gif" src-large="/pmc/articles/instance/3945187/bin/pgen.1004193.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3945187/" ref="ordinalpos=363&amp;ncbi_uid=5909275&amp;link_uid=PMC3945187" image-link="/pmc/articles/PMC3945187/figure/pgen-1004193-g005/" class="imagepopup">Figure 5. Relationship of intracellular MAPK <span class="highlight" style="background-color:">pathway</span> to RET receptor tyrosine kinase <span class="highlight" style="background-color:">signaling</span>..  From: Mitogen-Activated Protein Kinase (MAPK) <span class="highlight" style="background-color:">Pathway</span> Regulates Branching by Remodeling Epithelial Cell Adhesion. </a></p><div class="supp"><p class="details"><div><i>In situ</i> hybridization of the RET <span class="highlight" style="background-color:">signaling</span> target <i>Cxcr4</i> in E13.5 (A) control kidney shows expression in UB tips (arrow) as well as in kidney mesenchyme, while expression is lost in (B) UB tip (arrow) of dko kidney. (C–D) Another RET <span class="highlight" style="background-color:">signaling</span> target <i>Dusp6</i>, which is a negative regulator of MAPK activity, shows no changes (arrows) in the absence of MAPK <span class="highlight" style="background-color:">pathway</span> activity at E13.5. (E) Table summarizing the expression results of RET <span class="highlight" style="background-color:">signaling</span> targets in UB of dko kidneys. Scale bar: A–D 500 µm.</div></p></div><div class="aux"><div class="resc">Anneliis Ihermann-Hella, et al. PLoS Genet. 2014 March;10(3):e1004193.</div><p class="links"><a class="dblinks" href="/pubmed/24603431" ref="ordinalpos=363&amp;ncbi_uid=5909275&amp;link_uid=24603431">Citation</a><a class="dblinks" href="/pmc/articles/PMC3945187/" ref="ordinalpos=363&amp;ncbi_uid=5909275&amp;link_uid=PMC3945187">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>364.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g004/" ref="ordinalpos=364&amp;ncbi_uid=3758045&amp;link_uid=PMC3256223" href="/pmc/articles/PMC3256223/"><img src="/pmc/articles/instance/3256223/bin/pone.0030140.g004.gif" src-large="/pmc/articles/instance/3256223/bin/pone.0030140.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3256223/" ref="ordinalpos=364&amp;ncbi_uid=3758045&amp;link_uid=PMC3256223" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g004/" class="imagepopup">Figure 4. Modeling of anti-mir-489 and anti-mir-34a effects on the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Modeling of miRNA and Drug Action in the EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Heatmaps <b>A</b> and <b>B</b> are the results of anti-mir-489 simulations. <b>C</b> and <b>D</b> are the results of anti-mir-34a simulations. The first experiment within each heatmap (from bottom to top) is the ‘control’ state; the miRNA effect and anti-miRNA effect of the different experiments is always compared versus the ‘control’ state. <b>A</b>: Quantitative changes of mir-489 direct target-proteins due to different amounts of anti-mir-489 per experiment. The inhibition effects of mir-489 is inversely correlated with the anti-mir-489 concentration. <b>B</b>: Quantitative changes of the EGFR downstream activated proteins according to different amounts of anti-mir-489 per experiment. The concentrations of many downstream activated proteins of the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> correlate with the concentration of anti-mir-489. <b>C</b>: Quantitative changes of mir-34a direct target-proteins due to different amounts of anti-mir-34a per experiment. The inhibition effects of mir-34a inversely correlate with the anti-mir-34a concentration. <b>D</b>: Quantitative changes of the EGFR downstream activated proteins according to different amounts of anti-mir-34a per experiment. The concentrations of many downstream activated proteins correlate with the anti-mir-34a concentration.</div></p></div><div class="aux"><div class="resc">Jian Li, et al. PLoS One. 2012;7(1):e30140.</div><p class="links"><a class="dblinks" href="/pubmed/22253908" ref="ordinalpos=364&amp;ncbi_uid=3758045&amp;link_uid=22253908">Citation</a><a class="dblinks" href="/pmc/articles/PMC3256223/" ref="ordinalpos=364&amp;ncbi_uid=3758045&amp;link_uid=PMC3256223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>365.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3520155/figure/F8/" ref="ordinalpos=365&amp;ncbi_uid=4915303&amp;link_uid=PMC3520155" href="/pmc/articles/PMC3520155/"><img src="/pmc/articles/instance/3520155/bin/nihms425970f8.gif" src-large="/pmc/articles/instance/3520155/bin/nihms425970f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3520155/" ref="ordinalpos=365&amp;ncbi_uid=4915303&amp;link_uid=PMC3520155" image-link="/pmc/articles/PMC3520155/figure/F8/" class="imagepopup">Figure 8. The Control of Semaphorin-1a Reverse <span class="highlight" style="background-color:">Signaling</span> by Pebble and RhoGAPp190.  From: The Control of Semaphorin-1a-mediated Reverse <span class="highlight" style="background-color:">Signaling</span> by Opposing Pebble and RhoGAPp190 Functions in Drosophila. </a></p><div class="supp"><p class="details"><div>Repulsive guidance signals specified by Sema-1a bidirectional <span class="highlight" style="background-color:">signaling</span> play an important role in sculpting <i>Drosophila</i> neuromuscular connectivity. In our model, Sema-1a forward <span class="highlight" style="background-color:">signaling</span> largely contributes to CNS axon guidance, whereas both forward and reverse <span class="highlight" style="background-color:">signaling</span> are required to induce axon-axon repulsion at specific <span class="highlight" style="background-color:">pathway</span> choice points in the PNS. In the PNS, two direct but opposing regulators of Rho family small GTPases, <i>pbl</i> and <i>p190</i>, are required for attractive target recognition processes that guide motor axons before reaching, or after leaving, guidance choice points. Furthermore, these proteins mediate Sema-1a reverse <span class="highlight" style="background-color:">signaling</span> through the control of Rho1 activity at guidance choice points and likely participate in the recognition of these choice points by axonal growth cones, an essential prerequisite for selective axon-axon defasciculation mediated by repulsive <span class="highlight" style="background-color:">signaling</span> pathways. We propose that Pbl links choice-point recognition <span class="highlight" style="background-color:">signaling</span> and Sema-1a reverse <span class="highlight" style="background-color:">signaling</span>, and further, that Sema-1a reverse <span class="highlight" style="background-color:">signaling</span> collaborates in parallel with receptor functions of PlexA to induce selective axon-axon repulsion. In addition, Pbl not only competes, but also collaborates, with p190 to regulate Sema-1a reverse <span class="highlight" style="background-color:">signaling</span> in a context dependent manner. Sema-1a/PlexA–mediated repulsive <span class="highlight" style="background-color:">signaling</span> and target recognition <span class="highlight" style="background-color:">signaling</span> are likely integrated and coordinated by <i>pbl</i> and <i>p190</i>, both spatially and temporally, so as to establish precise connections between motoneurons and their muscle targets.</div></p></div><div class="aux"><div class="resc">Sangyun Jeong, et al. Neuron. ;76(4):721-734.</div><p class="links"><a class="dblinks" href="/pubmed/23177958" ref="ordinalpos=365&amp;ncbi_uid=4915303&amp;link_uid=23177958">Citation</a><a class="dblinks" href="/pmc/articles/PMC3520155/" ref="ordinalpos=365&amp;ncbi_uid=4915303&amp;link_uid=PMC3520155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>366.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4333971/figure/F1/" ref="ordinalpos=366&amp;ncbi_uid=7207675&amp;link_uid=PMC4333971" href="/pmc/articles/PMC4333971/"><img src="/pmc/articles/instance/4333971/bin/BJBMS-14-191-g001.gif" src-large="/pmc/articles/instance/4333971/bin/BJBMS-14-191-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4333971/" ref="ordinalpos=366&amp;ncbi_uid=7207675&amp;link_uid=PMC4333971" image-link="/pmc/articles/PMC4333971/figure/F1/" class="imagepopup">FIGURE 1.  From: Epigenetic alterations of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancer: a mini review. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members: Central to this <span class="highlight" style="background-color:">pathway</span> is a multiprotein scaffold composed of adenomatous polyposis coli (APC), glycogen synthase kinase (GSK)–3β, axin and the transcriptional cofactor β-catenin. Fz – Frizzled receptor.</div></p></div><div class="aux"><div class="resc">Ljiljana Serman, et al. Bosn J Basic Med Sci. 2014 November;14(4):191-194.</div><p class="links"><a class="dblinks" href="/pubmed/25428669" ref="ordinalpos=366&amp;ncbi_uid=7207675&amp;link_uid=25428669">Citation</a><a class="dblinks" href="/pmc/articles/PMC4333971/" ref="ordinalpos=366&amp;ncbi_uid=7207675&amp;link_uid=PMC4333971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>367.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4238421/figure/F1/" ref="ordinalpos=367&amp;ncbi_uid=6907271&amp;link_uid=PMC4238421" href="/pmc/articles/PMC4238421/"><img src="/pmc/articles/instance/4238421/bin/fpls-05-00648-g001.gif" src-large="/pmc/articles/instance/4238421/bin/fpls-05-00648-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4238421/" ref="ordinalpos=367&amp;ncbi_uid=6907271&amp;link_uid=PMC4238421" image-link="/pmc/articles/PMC4238421/figure/F1/" class="imagepopup">FIGURE 1.  From: An alternate route of ethylene receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>A model of the ethylene receptor <span class="highlight" style="background-color:">signaling</span> via the C- and N-termini of ethylene receptor. (A)</b> Ethylene receptor C-terminal <span class="highlight" style="background-color:">signaling</span>, here represented by ETHYLENE RESPONSE1 (ETR1), is mediated via the histidine kinase domain to CTR1 to phosphorylate EIN2 residues S645 and S924, and EIN2 is retained at the endoplasmic reticulum for degradation mediated by the F-box proteins EIN2 TARGETING PROTEIN1 (ETP1) and ETP2. Ethylene <span class="highlight" style="background-color:">signaling</span> mediated by EIN2 to the nuclear EIN3/EIL1 is inhibited, and EIN3 and EIL1 undergo degradation mediated by the F-box proteins EIN3 BINDING F-BOX PROTEIN1 (EBF1) and EBF2. <b>(B)</b> “Default” ethylene <span class="highlight" style="background-color:">signaling</span> by EIN2. Without the receptors (in the receptor quintuple mutant), EIN2-mediated ethylene <span class="highlight" style="background-color:">signaling</span> fully occurs and the mutants produce an extremely strong constitutive ethylene response phenotype. With the receptors, a portion of EIN2 could be underphosphorylated, presumably because of interconversion between the phosphorylated and underphosphorylated state or incomplete phosphorylation by CTR1, and cleaved to potentially activate ethylene <span class="highlight" style="background-color:">signaling</span>. <b>(C,D)</b> Ethylene <span class="highlight" style="background-color:">signaling</span> suppression by the receptor N-terminus: <b>(C)</b> With the receptors, the “default” ethylene <span class="highlight" style="background-color:">signaling</span> mediated by EIN2 can be repressed in part by the receptor N-terminal <span class="highlight" style="background-color:">signaling</span> (red line), regardless of CTR1 docking. <b>(D)</b> Ethylene binding or docking of a kinase-inactive ctr1 at the receptors inhibits the receptor C-terminal <span class="highlight" style="background-color:">signaling</span>, and the cleaved EIN2 C-terminus is translocated to the nucleus to induce EIN3/EIL1-directed expression of ethylene response genes. The N-terminal <span class="highlight" style="background-color:">signaling</span> is also inhibited (red dotted line), and the inhibition of EIN2-mediated <span class="highlight" style="background-color:">signaling</span> is largely alleviated. Components involved in the receptor N-terminal <span class="highlight" style="background-color:">signaling</span> remain to be identified. AHP1 and EIN2 are proposed to be involved in the CTR1-independent <span class="highlight" style="background-color:">pathway</span> because of their interaction with ETR1, and AHP1 also mediates cytokinin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Jingyi Zhang, et al. Front Plant Sci. 2014;5:648.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=367&amp;ncbi_uid=6907271&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4238421/" ref="ordinalpos=367&amp;ncbi_uid=6907271&amp;link_uid=PMC4238421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>368.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2790539/figure/F1/" ref="ordinalpos=368&amp;ncbi_uid=2678235&amp;link_uid=PMC2790539" href="/pmc/articles/PMC2790539/"><img src="/pmc/articles/instance/2790539/bin/nihms-149936-f0001.gif" src-large="/pmc/articles/instance/2790539/bin/nihms-149936-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2790539/" ref="ordinalpos=368&amp;ncbi_uid=2678235&amp;link_uid=PMC2790539" image-link="/pmc/articles/PMC2790539/figure/F1/" class="imagepopup">Fig. 1.  From: Mitochondrial and apoptotic neuronal death <span class="highlight" style="background-color:">signaling</span> pathways in cerebral ischemia. </a></p><div class="supp"><p class="details"><div>Mitochondria-dependent pathways of apoptosis in cerebral ischemia and reperfusion. After cerebral ischemia, various pathways, such as the death receptor <span class="highlight" style="background-color:">pathway</span>, p53 <span class="highlight" style="background-color:">pathway</span>, c-Jun N-terminal kinase (JNK) <span class="highlight" style="background-color:">pathway</span>, PI3-K <span class="highlight" style="background-color:">pathway</span>, and the MAPK <span class="highlight" style="background-color:">pathway</span> are activated. Most <span class="highlight" style="background-color:">signaling</span> pathways induce apoptosis with the help of pro-apoptotic proteins, such as cytochrome <i>c</i>, Endo G, AIF and Smac, which are stored in mitochondria.</div></p></div><div class="aux"><div class="resc">Kuniyasu Niizuma, et al. Biochim Biophys Acta. ;1802(1):92-99.</div><p class="links"><a class="dblinks" href="/pubmed/19751828" ref="ordinalpos=368&amp;ncbi_uid=2678235&amp;link_uid=19751828">Citation</a><a class="dblinks" href="/pmc/articles/PMC2790539/" ref="ordinalpos=368&amp;ncbi_uid=2678235&amp;link_uid=PMC2790539">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>369.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4032575/figure/fig4/" ref="ordinalpos=369&amp;ncbi_uid=6183402&amp;link_uid=PMC4032575" href="/pmc/articles/PMC4032575/"><img src="/pmc/articles/instance/4032575/bin/gr4.gif" src-large="/pmc/articles/instance/4032575/bin/gr4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4032575/" ref="ordinalpos=369&amp;ncbi_uid=6183402&amp;link_uid=PMC4032575" image-link="/pmc/articles/PMC4032575/figure/fig4/" class="imagepopup">Figure 4.  From: Compensatory Flux Changes within an Endocytic Trafficking Network Maintain Thermal Robustness of Notch <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Distinct Requirements for Notch <span class="highlight" style="background-color:">Signaling</span> Initiated by Different Mechanisms<br />(A) Basal N <span class="highlight" style="background-color:">signaling</span> in S2 cells is reduced by cholesterol depletion, but Dx-induced N <span class="highlight" style="background-color:">signaling</span> is unaffected.<br />(B) Basal and Dx-induced N signals are reduced by RNAi of Mam, Dyn, and Rab5, but only Dx-induced <span class="highlight" style="background-color:">signaling</span> is reduced by RNAi of Rab7 and Dor. Basal, but not Dx, signal depends on Kuz.<br />(C) N <span class="highlight" style="background-color:">signaling</span> in cells exposed to Dl is reduced by cholesterol depletion, but the fold change after ligand-induction is increased. <span class="highlight" style="background-color:">Signaling</span> by the N<sup>D505A</sup> construct is removed by cholesterol depletion.<br />(D) RNAi knockdown of components of the GSL synthesis <span class="highlight" style="background-color:">pathway</span> preferentially reduces the basal N signal compared to Dx and ligand-induced signal. chol., cholesterol.<br />(E) In cholesterol-depleted cells, ligand-induced N <span class="highlight" style="background-color:">signaling</span> is reduced by RNAi of Kuz, Dyn, and Rab5 but insensitive to Rab7 or Dor RNAi.<br />(F) Rab5DN reduces basal, ligand, and Dx-induced <span class="highlight" style="background-color:">signaling</span>, but only Dx <span class="highlight" style="background-color:">signaling</span> is reduced by Rab7DN.<br />Data are displayed as means ± SEM (minimum n = 3), p &lt; 0.05 for differences stated in the legend (Student t test). See also Figure S4.</div></p></div><div class="aux"><div class="resc">Hideyuki Shimizu, et al. Cell. 2014 May 22;157(5):1160-1174.</div><p class="links"><a class="dblinks" href="/pubmed/24855951" ref="ordinalpos=369&amp;ncbi_uid=6183402&amp;link_uid=24855951">Citation</a><a class="dblinks" href="/pmc/articles/PMC4032575/" ref="ordinalpos=369&amp;ncbi_uid=6183402&amp;link_uid=PMC4032575">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>370.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2810618/figure/F8/" ref="ordinalpos=370&amp;ncbi_uid=2141694&amp;link_uid=PMC2810618" href="/pmc/articles/PMC2810618/"><img src="/pmc/articles/instance/2810618/bin/nihms-168399-f0008.gif" src-large="/pmc/articles/instance/2810618/bin/nihms-168399-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2810618/" ref="ordinalpos=370&amp;ncbi_uid=2141694&amp;link_uid=PMC2810618" image-link="/pmc/articles/PMC2810618/figure/F8/" class="imagepopup">Figure 8.  From: Heart Valve Development. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> network model for heart valve development and remodeling. In the <span class="highlight" style="background-color:">signaling</span> network model for cardiac cushion development, numerous <span class="highlight" style="background-color:">signaling</span> pathways and transcriptional regulators act to coordinately regulate the process of heart valve formation. Each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is a simplified schema of the <span class="highlight" style="background-color:">signaling</span> events that occur; see Figures 2 through 7 for the details of these <span class="highlight" style="background-color:">signaling</span> pathways. Red arrows denote positive/synergistic interactions between pathways. Blunt red arrows denote inhibitory effects between pathways.</div></p></div><div class="aux"><div class="resc">Ehrin J. Armstrong, et al. Circ Res. ;95(5):459-470.</div><p class="links"><a class="dblinks" href="/pubmed/15345668" ref="ordinalpos=370&amp;ncbi_uid=2141694&amp;link_uid=15345668">Citation</a><a class="dblinks" href="/pmc/articles/PMC2810618/" ref="ordinalpos=370&amp;ncbi_uid=2141694&amp;link_uid=PMC2810618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>371.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3045689/figure/F5/" ref="ordinalpos=371&amp;ncbi_uid=2811752&amp;link_uid=PMC3045689" href="/pmc/articles/PMC3045689/"><img src="/pmc/articles/instance/3045689/bin/jcbn10-69f05.gif" src-large="/pmc/articles/instance/3045689/bin/jcbn10-69f05.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3045689/" ref="ordinalpos=371&amp;ncbi_uid=2811752&amp;link_uid=PMC3045689" image-link="/pmc/articles/PMC3045689/figure/F5/" class="imagepopup">Fig. 5.  From: Rebamipide suppresses TLR-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in regulating IRF3/7 and IFN-?/? reduction. </a></p><div class="supp"><p class="details"><div>Rebamipide suppressed TLR3-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human colonic epithelial cells. Effect of rebamipide on TLR3-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human colonic epithelial cells examined by both real-time RT-PCR of <i>TBK1</i>, <i>IRF3</i> and <i>IRF7</i> and by Western blotting of TBK1. Poly(I:C) (TLR3 ligand) was added to colonic epithelial cell line, CaCo2 with or without rebamipide. Poly(I:C) alone increased, whereas poly(I:C) plus rebamipide suppressed the mRNA expression of all of these genes to control levels.</div></p></div><div class="aux"><div class="resc">Naotaka Ogasawara, et al. J Clin Biochem Nutr. 2011 March;48(2):154-160.</div><p class="links"><a class="dblinks" href="/pubmed/21373269" ref="ordinalpos=371&amp;ncbi_uid=2811752&amp;link_uid=21373269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3045689/" ref="ordinalpos=371&amp;ncbi_uid=2811752&amp;link_uid=PMC3045689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>372.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3398840/figure/F6/" ref="ordinalpos=372&amp;ncbi_uid=4122754&amp;link_uid=PMC3398840" href="/pmc/articles/PMC3398840/"><img src="/pmc/articles/instance/3398840/bin/nihms383023f6.gif" src-large="/pmc/articles/instance/3398840/bin/nihms383023f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3398840/" ref="ordinalpos=372&amp;ncbi_uid=4122754&amp;link_uid=PMC3398840" image-link="/pmc/articles/PMC3398840/figure/F6/" class="imagepopup">Figure 6.  From: Inhibition of Carcinoma Cell Motility by Epoxyeicosatrienoic Acid (EET) Antagonists. </a></p><div class="supp"><p class="details"><div>A proposed diagram of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for EET-mediated prostate carcinoma cell motility. EET = epoxyeicosatrienoic acids; EETR = putative EET receptor, EEZE; EET antagonists; EGFR = epidermal growth factor receptor, HB-EGF = Heparin-binding EGF-like growth factor; MMPs = matrix metalloproteinases. PI3K/Akt = phosphatidylinositol 3-kinase/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Pro = Pro-HB-EGF. In this proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, EETs bind to a putative EETR to activate MMPs and cleave pro-HB-EGF to HB-EGF. Then, HB-EGF activates EGFR<sup>(30,31)</sup> and initiate <span class="highlight" style="background-color:">signaling</span> cascade to activate PI3K/Akt <span class="highlight" style="background-color:">pathway</span> to increase cell motility. An unidentified mechanism of EET-activated EETR to activate PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span><sup>(32)</sup> cannot be ruled out.</div></p></div><div class="aux"><div class="resc">Kasem Nithipatikom, et al. Cancer Sci. ;101(12):2629-2636.</div><p class="links"><a class="dblinks" href="/pubmed/20804500" ref="ordinalpos=372&amp;ncbi_uid=4122754&amp;link_uid=20804500">Citation</a><a class="dblinks" href="/pmc/articles/PMC3398840/" ref="ordinalpos=372&amp;ncbi_uid=4122754&amp;link_uid=PMC3398840">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>373.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2417166/figure/fig8/" ref="ordinalpos=373&amp;ncbi_uid=1470420&amp;link_uid=PMC2417166" href="/pmc/articles/PMC2417166/"><img src="/pmc/articles/instance/2417166/bin/zbc0150831300008.gif" src-large="/pmc/articles/instance/2417166/bin/zbc0150831300008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2417166/" ref="ordinalpos=373&amp;ncbi_uid=1470420&amp;link_uid=PMC2417166" image-link="/pmc/articles/PMC2417166/figure/fig8/" class="imagepopup">FIGURE 8. From: Glucagon-like Peptide-1 Activation of TCF7L2-dependent Wnt <span class="highlight" style="background-color:">Signaling</span>  Enhances Pancreatic Beta Cell  Proliferation. </a></p><div class="supp"><p class="details"><div><b>Schematic model of Wnt <span class="highlight" style="background-color:">signaling</span> pathways in INS-1 cells.</b>  <i>A,</i> basal endogenous <span class="highlight" style="background-color:">signaling</span> in the absence of GLP-1 or other  exogenous agonists. <i>B,</i> <span class="highlight" style="background-color:">signaling</span> in response to GLP-1 agonists such  as Exd4. The <i>thicker arrow</i> indicates a major <span class="highlight" style="background-color:">pathway</span> of GLP-1/GLP-1R  to cAMP/PKA phosphorylating Ser-675 on β-catenin and thereby stabilizing  against degradation. The <i>thin arrows</i> with <i>Xs</i> indicate that  the <span class="highlight" style="background-color:">pathway</span> designated by the <i>arrow</i> is not involved in  GLP-1-activated Wnt <span class="highlight" style="background-color:">signaling</span> at the level of activating gene transcription  via β-catenin/TCF7L2. The <i>cross-bar</i> indicates the inhibition of  GSK3 activity by Gao and disheveled (<i>DVL</i>), an upstream <span class="highlight" style="background-color:">pathway</span> of  canonical Wnt <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Zhengyu Liu, et al. J Biol Chem. 2008 March 28;283(13):8723-8735.</div><p class="links"><a class="dblinks" href="/pubmed/18216022" ref="ordinalpos=373&amp;ncbi_uid=1470420&amp;link_uid=18216022">Citation</a><a class="dblinks" href="/pmc/articles/PMC2417166/" ref="ordinalpos=373&amp;ncbi_uid=1470420&amp;link_uid=PMC2417166">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>374.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2928279/figure/pone-0012405-g002/" ref="ordinalpos=374&amp;ncbi_uid=2449692&amp;link_uid=PMC2928279" href="/pmc/articles/PMC2928279/"><img src="/pmc/articles/instance/2928279/bin/pone.0012405.g002.gif" src-large="/pmc/articles/instance/2928279/bin/pone.0012405.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2928279/" ref="ordinalpos=374&amp;ncbi_uid=2449692&amp;link_uid=PMC2928279" image-link="/pmc/articles/PMC2928279/figure/pone-0012405-g002/" class="imagepopup">Figure 2. Induced Jak/Stat and PI3K/S6 <span class="highlight" style="background-color:">signaling</span> varies among AML samples..  From: Distinct Patterns of DNA Damage Response and Apoptosis Correlate with Jak/Stat and PI3Kinase Response Profiles in Human Acute Myelogenous Leukemia. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> activity is represented by heat-maps. Blue: Low <span class="highlight" style="background-color:">Signaling</span>, Red: High <span class="highlight" style="background-color:">Signaling</span>, scaled for an individual readout/metric. (A) Jak/Stat <span class="highlight" style="background-color:">pathway</span> activity was measured after treatment of AML samples with IL-27, IL-6, and G-CSF. Heat map of Jak/Stat <span class="highlight" style="background-color:">signaling</span> responses, where each row represents a specific readout and metric such as: p-Stat1 | Basal, IL-27 → p-Stat1 | Total. The samples have been ordered according to IL-27 → p-Stat3 | Total, since this node had the highest AUC and p-value for stratifying CRs from NRs [28]. Cytogenetic risk and FAB categories are noted for each sample. (B) Exemplary 2D flow plots showing <span class="highlight" style="background-color:">signaling</span> responses to G-CSF or IL-27 treatment. (C) PI3K <span class="highlight" style="background-color:">pathway</span> activity was measured after treatment of AML samples with FLT3L, SCF and SDF-1α. Heat map of PI3K <span class="highlight" style="background-color:">signaling</span> responses, where each row represents a specific readout and metric. Patient samples have been ordered according to FLT3L → p-Akt | Fold responses. (D) Exemplary 2D Flow plots showing <span class="highlight" style="background-color:">signaling</span> responses to FLT3L or SCF treatment.</div></p></div><div class="aux"><div class="resc">David B. Rosen, et al. PLoS One. 2010;5(8):e12405.</div><p class="links"><a class="dblinks" href="/pubmed/20811632" ref="ordinalpos=374&amp;ncbi_uid=2449692&amp;link_uid=20811632">Citation</a><a class="dblinks" href="/pmc/articles/PMC2928279/" ref="ordinalpos=374&amp;ncbi_uid=2449692&amp;link_uid=PMC2928279">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>375.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2686461/figure/F1/" ref="ordinalpos=375&amp;ncbi_uid=1835884&amp;link_uid=PMC2686461" href="/pmc/articles/PMC2686461/"><img src="/pmc/articles/instance/2686461/bin/gkn653f1.gif" src-large="/pmc/articles/instance/2686461/bin/gkn653f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2686461/" ref="ordinalpos=375&amp;ncbi_uid=1835884&amp;link_uid=PMC2686461" image-link="/pmc/articles/PMC2686461/figure/F1/" class="imagepopup">Figure 1. From: PID: the <span class="highlight" style="background-color:">Pathway</span> Interaction Database. </a></p><div class="supp"><p class="details"><div>BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> header information includes the date of the latest revision; the data curation or import source; the curator; the reviewers; the stable <span class="highlight" style="background-color:">pathway</span> identifier; links to a <span class="highlight" style="background-color:">pathway</span>-specific molecule list and a <span class="highlight" style="background-color:">pathway</span>-specific references list; and links to <span class="highlight" style="background-color:">pathway</span> graphic and text data exchange format options.</div></p></div><div class="aux"><div class="resc">Carl F. Schaefer, et al. Nucleic Acids Res. 2009 January;37(Database issue):D674-D679.</div><p class="links"><a class="dblinks" href="/pubmed/18832364" ref="ordinalpos=375&amp;ncbi_uid=1835884&amp;link_uid=18832364">Citation</a><a class="dblinks" href="/pmc/articles/PMC2686461/" ref="ordinalpos=375&amp;ncbi_uid=1835884&amp;link_uid=PMC2686461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>376.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2815088/figure/F3/" ref="ordinalpos=376&amp;ncbi_uid=2732960&amp;link_uid=PMC2815088" href="/pmc/articles/PMC2815088/"><img src="/pmc/articles/instance/2815088/bin/nihms-136354-f0003.gif" src-large="/pmc/articles/instance/2815088/bin/nihms-136354-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2815088/" ref="ordinalpos=376&amp;ncbi_uid=2732960&amp;link_uid=PMC2815088" image-link="/pmc/articles/PMC2815088/figure/F3/" class="imagepopup">Figure 3. The highly conserved hexosamine biosynthetic <span class="highlight" style="background-color:">pathway</span> leading to O-GlcNAc cycling.  From: The Hexosamine <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: O-GlcNAc cycling in feast or famine. </a></p><div class="supp"><p class="details"><div>The enzymes involved in the synthesis and utilization of UDP-GlcNAc from nutrient precursors are shown. The role of O-GlcNAc cycling is emphasized, but UDP-GlcNAc is also the precursor for glycolipids, O-linked GalNAc and N-linked Glycoproteins in the endomembrane system as indicated by <i>other</i> on each side of the UDP-HexNAcs (see text). The mammalian enzymes are listed with the <i>C. elegans</i> homologs in <i>italics.</i> The nutrient-derived precursors glucose, glutamine, acetyl-CoA, UTP are integrated by the <span class="highlight" style="background-color:">pathway</span> and UDP-GlcNAc levels reflect the nutrient status. O-GlcNAc cycling is the terminal step in the hexosamine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">John A. Hanover, et al. Biochim Biophys Acta. ;1800(2):80.</div><p class="links"><a class="dblinks" href="/pubmed/19647043" ref="ordinalpos=376&amp;ncbi_uid=2732960&amp;link_uid=19647043">Citation</a><a class="dblinks" href="/pmc/articles/PMC2815088/" ref="ordinalpos=376&amp;ncbi_uid=2732960&amp;link_uid=PMC2815088">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>377.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945950/figure/F3/" ref="ordinalpos=377&amp;ncbi_uid=2525566&amp;link_uid=PMC2945950" href="/pmc/articles/PMC2945950/"><img src="/pmc/articles/instance/2945950/bin/1752-0509-4-121-3.gif" src-large="/pmc/articles/instance/2945950/bin/1752-0509-4-121-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945950/" ref="ordinalpos=377&amp;ncbi_uid=2525566&amp;link_uid=PMC2945950" image-link="/pmc/articles/PMC2945950/figure/F3/" class="imagepopup">Figure 3.  From: Gene set analysis exploiting the topology of a <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>DAG <i>D </i>obtained from BCR <span class="highlight" style="background-color:">pathway</span></b>. DAG <i>D </i>corresponding to the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Maria Sofia Massa, et al. BMC Syst Biol. 2010;4:121-121.</div><p class="links"><a class="dblinks" href="/pubmed/20809931" ref="ordinalpos=377&amp;ncbi_uid=2525566&amp;link_uid=20809931">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945950/" ref="ordinalpos=377&amp;ncbi_uid=2525566&amp;link_uid=PMC2945950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>378.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3499976/figure/F1/" ref="ordinalpos=378&amp;ncbi_uid=4373552&amp;link_uid=PMC3499976" href="/pmc/articles/PMC3499976/"><img src="/pmc/articles/instance/3499976/bin/nihms384184f1.gif" src-large="/pmc/articles/instance/3499976/bin/nihms384184f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3499976/" ref="ordinalpos=378&amp;ncbi_uid=4373552&amp;link_uid=PMC3499976" image-link="/pmc/articles/PMC3499976/figure/F1/" class="imagepopup">Figure 1.  From: Wnts: up-and-coming at the synapse. </a></p><div class="supp"><p class="details"><div>Wnt transduction pathways and their functions during synapse development, function and plasticity. <b>(a)</b> Summary of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In this <span class="highlight" style="background-color:">pathway</span>, secreted Wnts bind to Fz receptors in the receiving cell. This binding activates the scaffolding PDZ protein DVL, which disrupts a so-called ‘destruction complex’ containing GSK-3β, axin and APC. In the absence of Wnt <span class="highlight" style="background-color:">signaling</span>, this complex constitutively phosphorylates β-catenin, leading to its degradation by ubiquitin-dependent proteasomal degradation. Wnt binding to Fz receptors disrupts the destruction complex, resulting in cytoplasmic stabilization of β-catenin and its import into the nucleus. In the nucleus, β-catenin associates with members of the LEF/TCF transcription factor family to regulate transcription. Also depicted are a Wnt antagonist FRP and an inhibitor of GSK-3, frequently rearranged in advanced T-cell lymphomas–GSK-3-binding protein (Frat–GBP), which can also bind to DVL. <b>(b)</b> The Wnt–Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In this <span class="highlight" style="background-color:">pathway</span>, Fz activation by Wnts leads to increased intracellular Ca<sup>2+</sup> and nuclear import of the transcription factor NF-AT. <b>(c)</b> The Frizzled nuclear import (FNI) <span class="highlight" style="background-color:">pathway</span>. In this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Wnts binds to Fz2 receptors in the receiving cell. Fz is endocytosed and back-transported towards the nucleus in a process that depends on its binding to the 7-PDZ protein GRIP. At some point during this trafficking, the C-terminal region of Fz2 is cleaved, and the cleaved fragment imported into the nucleus. <b>(d)</b> The divergent canonical Wnt <span class="highlight" style="background-color:">pathway</span>. In this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Wnt binding to Fz inhibits the destruction complex, and prevents the phosphorylation of microtubule-associated proteins such as MAP1B, Tau and MAP2, thereby regulating microtubule stability. <b>(e)</b> The PCP <span class="highlight" style="background-color:">pathway</span>. In this transduction <span class="highlight" style="background-color:">pathway</span>, Fz acts through JNK to regulate the cytoskeleton. Abbreviations: CAMKII, calcium–calmodulin-dependent protein kinase II; LRP, low-density lipoprotein receptor; PKC, protein kinase C.</div></p></div><div class="aux"><div class="resc">Sean D. Speese, et al. Trends Neurosci. 2007 June;30(6):268-275.</div><p class="links"><a class="dblinks" href="/pubmed/17467065" ref="ordinalpos=378&amp;ncbi_uid=4373552&amp;link_uid=17467065">Citation</a><a class="dblinks" href="/pmc/articles/PMC3499976/" ref="ordinalpos=378&amp;ncbi_uid=4373552&amp;link_uid=PMC3499976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>379.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193034/figure/F7/" ref="ordinalpos=379&amp;ncbi_uid=3288464&amp;link_uid=PMC3193034" href="/pmc/articles/PMC3193034/"><img src="/pmc/articles/instance/3193034/bin/1471-2229-11-129-7.gif" src-large="/pmc/articles/instance/3193034/bin/1471-2229-11-129-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193034/" ref="ordinalpos=379&amp;ncbi_uid=3288464&amp;link_uid=PMC3193034" image-link="/pmc/articles/PMC3193034/figure/F7/" class="imagepopup">Figure 7.  From: The NAC domain-containing protein, GmNAC6, is a downstream component of the ER stress- and osmotic stress-induced NRP-mediated cell-death <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>The osmotic-stress and ER-stress signal-integrating <span class="highlight" style="background-color:">pathway</span></b>. ER stress and osmotic stress activate two independent <span class="highlight" style="background-color:">signaling</span> pathways (1 and 2), which converge on NRP-A and NRP-B expression to activate an osmotic- and ER-stress integrating <span class="highlight" style="background-color:">pathway</span>, also called the integrated <span class="highlight" style="background-color:">pathway</span>. The enhanced accumulation of membrane-associated NRPs activates a cascade to induce the expression of the nuclear transactivator, NAC6, which, in turn, promotes cell death.</div></p></div><div class="aux"><div class="resc">Jerusa AQA Faria, et al. BMC Plant Biol. 2011;11:129-129.</div><p class="links"><a class="dblinks" href="/pubmed/21943253" ref="ordinalpos=379&amp;ncbi_uid=3288464&amp;link_uid=21943253">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193034/" ref="ordinalpos=379&amp;ncbi_uid=3288464&amp;link_uid=PMC3193034">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>380.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3640382/figure/marinedrugs-11-00316-f006/" ref="ordinalpos=380&amp;ncbi_uid=4866079&amp;link_uid=PMC3640382" href="/pmc/articles/PMC3640382/"><img src="/pmc/articles/instance/3640382/bin/marinedrugs-11-00316-g006.gif" src-large="/pmc/articles/instance/3640382/bin/marinedrugs-11-00316-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3640382/" ref="ordinalpos=380&amp;ncbi_uid=4866079&amp;link_uid=PMC3640382" image-link="/pmc/articles/PMC3640382/figure/marinedrugs-11-00316-f006/" class="imagepopup">Figure 6.  From: Asperolide A, a Marine-Derived Tetranorditerpenoid, Induces G2/M Arrest in Human NCI-H460 Lung Carcinoma Cells, Is Mediated by p53-p21 Stabilization and Modulated by Ras/Raf/MEK/ERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) The probable effect of asperolide A on the Ras/Raf/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (<b>B</b>) Mechanisms of asperolide A-induced G2/M cell cycle arrest.</div></p></div><div class="aux"><div class="resc">Cuiting Lv, et al. Mar Drugs. 2013 February;11(2):316-331.</div><p class="links"><a class="dblinks" href="/pubmed/23434831" ref="ordinalpos=380&amp;ncbi_uid=4866079&amp;link_uid=23434831">Citation</a><a class="dblinks" href="/pmc/articles/PMC3640382/" ref="ordinalpos=380&amp;ncbi_uid=4866079&amp;link_uid=PMC3640382">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>381.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4147239/figure/Fig1/" ref="ordinalpos=381&amp;ncbi_uid=6541629&amp;link_uid=PMC4147239" href="/pmc/articles/PMC4147239/"><img src="/pmc/articles/instance/4147239/bin/11864_2014_294_Fig1_HTML.gif" src-large="/pmc/articles/instance/4147239/bin/11864_2014_294_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4147239/" ref="ordinalpos=381&amp;ncbi_uid=6541629&amp;link_uid=PMC4147239" image-link="/pmc/articles/PMC4147239/figure/Fig1/" class="imagepopup">Fig. 1.  From: Targeting the mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Neuroendocrine Tumors. </a></p><div class="supp"><p class="details"><div>The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Simplified representation of key components of the mTOR <span class="highlight" style="background-color:">signaling</span> network. The mTOR <span class="highlight" style="background-color:">pathway</span> plays an important role in mediating growth factor signals that stimulate cell growth and proliferation and regulate angiogenesis and cell metabolism. Arrows represent activation; bars represent inhibition. Adapted from Yao et al, 2013 [66].</div></p></div><div class="aux"><div class="resc">Jennifer Chan, et al. Curr Treat Options Oncol. 2014;15(3):365-379.</div><p class="links"><a class="dblinks" href="/pubmed/25092520" ref="ordinalpos=381&amp;ncbi_uid=6541629&amp;link_uid=25092520">Citation</a><a class="dblinks" href="/pmc/articles/PMC4147239/" ref="ordinalpos=381&amp;ncbi_uid=6541629&amp;link_uid=PMC4147239">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>382.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2576563/figure/fig7/" ref="ordinalpos=382&amp;ncbi_uid=1539104&amp;link_uid=PMC2576563" href="/pmc/articles/PMC2576563/"><img src="/pmc/articles/instance/2576563/bin/zbc0480855990007.gif" src-large="/pmc/articles/instance/2576563/bin/zbc0480855990007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2576563/" ref="ordinalpos=382&amp;ncbi_uid=1539104&amp;link_uid=PMC2576563" image-link="/pmc/articles/PMC2576563/figure/fig7/" class="imagepopup">FIGURE 7. From: MKP1 Regulates the Induction of MUC5AC Mucin by Streptococcus  pneumoniae Pneumolysin by Inhibiting the PAK4-JNK <span class="highlight" style="background-color:">Signaling</span>  <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of tight regulation of MUC5AC mucin induction  by pneumolysin via PAK4-JNK and MKP1 <span class="highlight" style="background-color:">signaling</span> pathways.</b> As indicated, the  PAK4-JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> negatively regulates MUC5AC induction by  recombinant pneumolysin in a TLR4-dependent manner. Interestingly induction of  MKP1 appears to be crucial for inhibiting JNK activation. Our data thereby  unveil a complex <span class="highlight" style="background-color:">signaling</span> mechanism underlying tight regulation of MUC5AC  mucin by pneumococcal pneumolysin.</div></p></div><div class="aux"><div class="resc">Un-Hwan Ha, et al. J Biol Chem. 2008 November 7;283(45):30624-30631.</div><p class="links"><a class="dblinks" href="/pubmed/18782768" ref="ordinalpos=382&amp;ncbi_uid=1539104&amp;link_uid=18782768">Citation</a><a class="dblinks" href="/pmc/articles/PMC2576563/" ref="ordinalpos=382&amp;ncbi_uid=1539104&amp;link_uid=PMC2576563">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>383.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412625/figure/F6/" ref="ordinalpos=383&amp;ncbi_uid=5144615&amp;link_uid=PMC3412625" href="/pmc/articles/PMC3412625/"><img src="/pmc/articles/instance/3412625/bin/nihms392558f6.gif" src-large="/pmc/articles/instance/3412625/bin/nihms392558f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412625/" ref="ordinalpos=383&amp;ncbi_uid=5144615&amp;link_uid=PMC3412625" image-link="/pmc/articles/PMC3412625/figure/F6/" class="imagepopup">Figure 6. Schematic representation of potential effects of VELCADE on TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Combination therapy with VELCADE and tPA is neuroprotective in aged rats after stroke and targets miRNA-146a and the toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Stroke induces the activation of vascular TLR signal, which leads to upregulation of proinflammatory genes via the activation of NF-κB. VELCADE inactivates NF-kB through either directly blocking stroke-activated TLR2 and TLR4 or indirectly repressing IRAK1 by VELCADE upregulated miR-146a, which consequently downregulates proinflammatory gene expression. Thus, VELCADE acts on a multi-level inhibition of TLR <span class="highlight" style="background-color:">signaling</span> in ischemic cerebral vessels.</div></p></div><div class="aux"><div class="resc">Li Zhang, et al. Arterioscler Thromb Vasc Biol. ;32(8):1856-1864.</div><p class="links"><a class="dblinks" href="/pubmed/22723435" ref="ordinalpos=383&amp;ncbi_uid=5144615&amp;link_uid=22723435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412625/" ref="ordinalpos=383&amp;ncbi_uid=5144615&amp;link_uid=PMC3412625">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>384.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3118552/figure/F7/" ref="ordinalpos=384&amp;ncbi_uid=3030169&amp;link_uid=PMC3118552" href="/pmc/articles/PMC3118552/"><img src="/pmc/articles/instance/3118552/bin/nihms293665f7.gif" src-large="/pmc/articles/instance/3118552/bin/nihms293665f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3118552/" ref="ordinalpos=384&amp;ncbi_uid=3030169&amp;link_uid=PMC3118552" image-link="/pmc/articles/PMC3118552/figure/F7/" class="imagepopup">Figure 7. Schematic of heat stress inhibition of STAT1 and iNOS <span class="highlight" style="background-color:">signaling</span> pathways.  From: Activation of the Stress Protein Response Inhibits the STAT1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and iNOS function in Alveolar Macrophages: Role of Hsp90 and Hsp70. </a></p><div class="supp"><p class="details"><div>Heat stress or treatment of macrophages with 17-AAG results in the disruption of the STAT1-Hsp90 complex preventing STAT1 phosphorylation and translocation to the nucleus. Post-heat stress when the STAT1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is responsive to IFN-γ stimulation, nitric oxide production from iNOS is inhibited due to the inducible form of Hsp70.</div></p></div><div class="aux"><div class="resc">Marybeth Howard, et al. Thorax. ;65(4):346-353.</div><p class="links"><a class="dblinks" href="/pubmed/20388761" ref="ordinalpos=384&amp;ncbi_uid=3030169&amp;link_uid=20388761">Citation</a><a class="dblinks" href="/pmc/articles/PMC3118552/" ref="ordinalpos=384&amp;ncbi_uid=3030169&amp;link_uid=PMC3118552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>385.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2675004/figure/F1/" ref="ordinalpos=385&amp;ncbi_uid=1780066&amp;link_uid=PMC2675004" href="/pmc/articles/PMC2675004/"><img src="/pmc/articles/instance/2675004/bin/nihms-107317-f0003.gif" src-large="/pmc/articles/instance/2675004/bin/nihms-107317-f0003.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2675004/" ref="ordinalpos=385&amp;ncbi_uid=1780066&amp;link_uid=PMC2675004" image-link="/pmc/articles/PMC2675004/figure/F1/" class="imagepopup">Fig. 1. The TNF receptor super-family and the NF-κB <span class="highlight" style="background-color:">signaling</span> system.  From: Crosstalk via the NF-?B <span class="highlight" style="background-color:">Signaling</span> System. </a></p><div class="supp"><p class="details"><div>Various members of the TNF receptor super-family utilize the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to effect diverse physiological responses through NF-κB dependent gene expression program. The stimulus-specificity of the cellular response and the functional interaction of distinct stimuli <i>via</i> the NF-κB <span class="highlight" style="background-color:">signaling</span> system is the focus of this review.</div></p></div><div class="aux"><div class="resc">Soumen Basak, et al. Cytokine Growth Factor Rev. ;19(3-4):187-197.</div><p class="links"><a class="dblinks" href="/pubmed/18515173" ref="ordinalpos=385&amp;ncbi_uid=1780066&amp;link_uid=18515173">Citation</a><a class="dblinks" href="/pmc/articles/PMC2675004/" ref="ordinalpos=385&amp;ncbi_uid=1780066&amp;link_uid=PMC2675004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>386.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2846156/figure/FIG3/" ref="ordinalpos=386&amp;ncbi_uid=2294716&amp;link_uid=PMC2846156" href="/pmc/articles/PMC2846156/"><img src="/pmc/articles/instance/2846156/bin/zpg0020922640003.gif" src-large="/pmc/articles/instance/2846156/bin/zpg0020922640003.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2846156/" ref="ordinalpos=386&amp;ncbi_uid=2294716&amp;link_uid=PMC2846156" image-link="/pmc/articles/PMC2846156/figure/FIG3/" class="imagepopup">F ig . 3. From: Therapeutic Targeting of Toll-Like Receptors for Infectious and Inflammatory           Diseases and Cancer. </a></p><div class="supp"><p class="details"><div>Drugs targeting the TLR4/MD-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activation of TLR4 recruits the                 adapter pairs Mal/MyD88 and TRAM/TRIF. <span class="highlight" style="background-color:">Signaling</span> through Mal/Myd88 recruits IRAK1,                 IRAK4, Traf6, and TAK1-binding protein (TAB) to activate the NFκB <span class="highlight" style="background-color:">signaling</span>                 <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Signaling</span> through TRAM/TRIF activates NFκB through TRAF6, but also                 activates <span class="highlight" style="background-color:">signaling</span> through IRF1 and IRF3 through TRAF3. Several drugs now target                 the TLR4/MD-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Eritoran and the AGP compounds bind to the                 TLR4/MD-2 lipid IA binding site, monoclonal antibodies (for example the neutralizing                 antibody from NovImmune, Geneva, Switzerland) bind to TLR4. Soluble peptides, such                 as the BB loop peptides, target the BB loop on the TIR domain, the region of the                 protein important in receptor dimerization. The small molecular inhibitor TAK-242                 targets the <span class="highlight" style="background-color:">signaling</span> domain of the TIR.</div></p></div><div class="aux"><div class="resc">Luke A. J. O'Neill, et al. Pharmacol Rev. 2009 June;61(2):177-197.</div><p class="links"><a class="dblinks" href="/pubmed/19474110" ref="ordinalpos=386&amp;ncbi_uid=2294716&amp;link_uid=19474110">Citation</a><a class="dblinks" href="/pmc/articles/PMC2846156/" ref="ordinalpos=386&amp;ncbi_uid=2294716&amp;link_uid=PMC2846156">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>387.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2553439/figure/F3/" ref="ordinalpos=387&amp;ncbi_uid=1767301&amp;link_uid=PMC2553439" href="/pmc/articles/PMC2553439/"><img src="/pmc/articles/instance/2639041/bin/btn403f3.gif" src-large="/pmc/articles/instance/2639041/bin/btn403f3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2553439/" ref="ordinalpos=387&amp;ncbi_uid=1767301&amp;link_uid=PMC2553439" image-link="/pmc/articles/PMC2553439/figure/F3/" class="imagepopup">Fig. 3. From: Predicting <span class="highlight" style="background-color:">pathway</span> membership via domain signatures. </a></p><div class="supp"><p class="details"><div>Predicted <span class="highlight" style="background-color:">pathway</span> component (shaded) for <i>PLCH2</i> in the Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Holger Fröhlich, et al. Bioinformatics. 2008 October 1;24(19):2137-2142.</div><p class="links"><a class="dblinks" href="/pubmed/18676972" ref="ordinalpos=387&amp;ncbi_uid=1767301&amp;link_uid=18676972">Citation</a><a class="dblinks" href="/pmc/articles/PMC2553439/" ref="ordinalpos=387&amp;ncbi_uid=1767301&amp;link_uid=PMC2553439">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>388.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4001295/figure/fig7/" ref="ordinalpos=388&amp;ncbi_uid=6066192&amp;link_uid=PMC4001295" href="/pmc/articles/PMC4001295/"><img src="/pmc/articles/instance/4001295/bin/cddis2014101f7.gif" src-large="/pmc/articles/instance/4001295/bin/cddis2014101f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4001295/" ref="ordinalpos=388&amp;ncbi_uid=6066192&amp;link_uid=PMC4001295" image-link="/pmc/articles/PMC4001295/figure/fig7/" class="imagepopup">Figure 7.  From: Opposing TNF-α/IL-1β- and BMP-2-activated MAPK <span class="highlight" style="background-color:">signaling</span> pathways converge on Runx2 to regulate BMP-2-induced osteoblastic differentiation. </a></p><div class="supp"><p class="details"><div>Opposing TNF-<i>α</i>/IL-1<i>β</i>- and BMP-2-activated MAPK <span class="highlight" style="background-color:">signaling</span> pathways converge on Runx2 to regulate BMP-2-induced osteoblastic differentiation. (<b>a</b>) BMP-2 activates the canonical BMP/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and non-canonical MAPK pathways to regulate osteoblastic differentiation of osteoprogenitor cells. In the activated MAPK pathways, only p38 and ERK1/2 <span class="highlight" style="background-color:">signaling</span> are required to regulate the expression and activation of Runx2. (<b>b</b>) TNF-<i>α</i> and IL-1<i>β</i> also activate p38, ERK1/2, and JNK1/2 <span class="highlight" style="background-color:">signaling</span>, but they act to oppose the effects of BMP-2 on Runx2 expression and osteoblastic differentiation. (<b>c</b>) Without the positive influence of BMP/Smad <span class="highlight" style="background-color:">signaling</span>, both ERK1/2 and JNK1/2 <span class="highlight" style="background-color:">signaling</span> are activated, but BMP-2-induced Runx2 activity is inhibited, resulting in the suppression of osteoblastic differentiation</div></p></div><div class="aux"><div class="resc">R-L Huang, et al. Cell Death Dis. 2014 April;5(4):e1187.</div><p class="links"><a class="dblinks" href="/pubmed/24743742" ref="ordinalpos=388&amp;ncbi_uid=6066192&amp;link_uid=24743742">Citation</a><a class="dblinks" href="/pmc/articles/PMC4001295/" ref="ordinalpos=388&amp;ncbi_uid=6066192&amp;link_uid=PMC4001295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>389.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4102805/figure/F3/" ref="ordinalpos=389&amp;ncbi_uid=6408864&amp;link_uid=PMC4102805" href="/pmc/articles/PMC4102805/"><img src="/pmc/articles/instance/4102805/bin/oncotarget-05-3220-g003.gif" src-large="/pmc/articles/instance/4102805/bin/oncotarget-05-3220-g003.jpg" alt="Fig 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4102805/" ref="ordinalpos=389&amp;ncbi_uid=6408864&amp;link_uid=PMC4102805" image-link="/pmc/articles/PMC4102805/figure/F3/" class="imagepopup">Fig 3. S100A4-RAGE interaction hyperactivates MAPK/ERK and hypoxia <span class="highlight" style="background-color:">signaling</span> pathways.  From: RAGE mediates S100A4-induced cell motility via MAPK/ERK and hypoxia <span class="highlight" style="background-color:">signaling</span> and is a prognostic biomarker for human colorectal cancer metastasis. </a></p><div class="supp"><p class="details"><div>A) Relative luciferase activity of HCT116/vector cells, transfected with <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> specific reporter constructs. Results were normalized to the internal control. Data represent mean values ± SD. B) Relative luciferase activity of HCT116/vector (grey bars) and HCT116/RAGE cells (black bars), transfected with <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> specific reporter constructs. Cells were incubated for 24 h with BSA or rS100A4. Results were normalized to the respective untreated HCT116/vector cells. Data represent mean values ± SD.</div></p></div><div class="aux"><div class="resc">Mathias Dahlmann, et al. Oncotarget. 2014 May;5(10):3220-3233.</div><p class="links"><a class="dblinks" href="/pubmed/24952599" ref="ordinalpos=389&amp;ncbi_uid=6408864&amp;link_uid=24952599">Citation</a><a class="dblinks" href="/pmc/articles/PMC4102805/" ref="ordinalpos=389&amp;ncbi_uid=6408864&amp;link_uid=PMC4102805">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>390.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3741273/figure/pone-0071826-g002/" ref="ordinalpos=390&amp;ncbi_uid=5201143&amp;link_uid=PMC3741273" href="/pmc/articles/PMC3741273/"><img src="/pmc/articles/instance/3741273/bin/pone.0071826.g002.gif" src-large="/pmc/articles/instance/3741273/bin/pone.0071826.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3741273/" ref="ordinalpos=390&amp;ncbi_uid=5201143&amp;link_uid=PMC3741273" image-link="/pmc/articles/PMC3741273/figure/pone-0071826-g002/" class="imagepopup">Figure 2. Q-PCR validation of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> alterations in Iqgap2−/− livers..  From: Transcript Profiling Identifies Iqgap2−/− Mouse as a Model for Advanced Human Hepatocellular Carcinoma. </a></p><div class="supp"><p class="details"><div>The same liver RNA samples used for microarray (24-month-old WT, N  =  3, and 24-month-old KO, N  =  3) were analyzed using the Mouse Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> RT<sup>2</sup> Profiler PCR Array. 11 genes belonging to the <span class="highlight" style="background-color:">pathway</span> and found to be differentially expressed in <i>Iqgap2<sup>−/−</sup></i> livers are shown.</div></p></div><div class="aux"><div class="resc">Dmitri V. Gnatenko, et al. PLoS One. 2013;8(8):e71826.</div><p class="links"><a class="dblinks" href="/pubmed/23951254" ref="ordinalpos=390&amp;ncbi_uid=5201143&amp;link_uid=23951254">Citation</a><a class="dblinks" href="/pmc/articles/PMC3741273/" ref="ordinalpos=390&amp;ncbi_uid=5201143&amp;link_uid=PMC3741273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>391.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889201/figure/F4/" ref="ordinalpos=391&amp;ncbi_uid=5701883&amp;link_uid=PMC3889201" href="/pmc/articles/PMC3889201/"><img src="/pmc/articles/instance/3889201/bin/nihms538110f4.gif" src-large="/pmc/articles/instance/3889201/bin/nihms538110f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889201/" ref="ordinalpos=391&amp;ncbi_uid=5701883&amp;link_uid=PMC3889201" image-link="/pmc/articles/PMC3889201/figure/F4/" class="imagepopup">Figure 4.  From: The Human Sulfatase 2 Inhibitor 2,4-Disulfonylphenyl-tert-Butylnitrone (OKN-007) has an Antitumor Effect in Hepatocellular Carcinoma Mediated via Suppression of TGFB1/SMAD2 and Hedgehog/GLI1 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>OKN-007 inhibits TGFB1/SMAD2 and Hedgehog/GLI1 <span class="highlight" style="background-color:">signaling</span> in Huh7 cells. (A) OKN-007 treatment was shown to repress the TGFB1/SMAD2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Huh7 cells by both SBE reporter luciferase activity assay and immunoblotting for phospho-SMAD2. (B) OKN-007 treatment was shown to suppress the Hh/GLI1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Huh7 cells by both a GLI reporter luciferase activity assay and immunoblotting for GLI1 protein.</div></p></div><div class="aux"><div class="resc">Xin Zheng, et al. Genes Chromosomes Cancer. ;52(3):225-236.</div><p class="links"><a class="dblinks" href="/pubmed/23109092" ref="ordinalpos=391&amp;ncbi_uid=5701883&amp;link_uid=23109092">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889201/" ref="ordinalpos=391&amp;ncbi_uid=5701883&amp;link_uid=PMC3889201">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>392.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3087179/figure/F5/" ref="ordinalpos=392&amp;ncbi_uid=3381968&amp;link_uid=PMC3087179" href="/pmc/articles/PMC3087179/"><img src="/pmc/articles/instance/3087179/bin/nihms280124f5.gif" src-large="/pmc/articles/instance/3087179/bin/nihms280124f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3087179/" ref="ordinalpos=392&amp;ncbi_uid=3381968&amp;link_uid=PMC3087179" image-link="/pmc/articles/PMC3087179/figure/F5/" class="imagepopup">Figure 5. The model of IL-4R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in RMS.  From: IL-4R Drives Dedifferentiation, Mitogenesis and Metastasis in Rhabdomyosarcoma. </a></p><div class="supp"><p class="details"><div>In embryonal rhabdomyosarcoma (ERMS), IL-4R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is mediated by the type II IL-4R complex (IL-4Rα/IL-13Rα1). Both IL-4 and IL-13 stimulate the type II heterodimeric receptor, then intracellular STAT6 protein is phosphorylated to induce tumor progression. In alveolar rhabdomyosarcoma (ARMS), both the type I (IL-4Rα/common γ chain) and type II receptor complex are highly expressed. IL-4 and/or IL-13 activate IL-4R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ARMS mediated by these receptor complexes, resulting in tumor progression including lymph node and pulmonary metastasis.</div></p></div><div class="aux"><div class="resc">Tohru Hosoyama, et al. Clin Cancer Res. ;17(9):2757-2766.</div><p class="links"><a class="dblinks" href="/pubmed/21536546" ref="ordinalpos=392&amp;ncbi_uid=3381968&amp;link_uid=21536546">Citation</a><a class="dblinks" href="/pmc/articles/PMC3087179/" ref="ordinalpos=392&amp;ncbi_uid=3381968&amp;link_uid=PMC3087179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>393.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1451179/figure/fig6/" ref="ordinalpos=393&amp;ncbi_uid=621631&amp;link_uid=PMC1451179" href="/pmc/articles/PMC1451179/"><img src="/pmc/articles/instance/1451179/bin/6772f6.gif" src-large="/pmc/articles/instance/1451179/bin/6772f6.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  6.—" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1451179/" ref="ordinalpos=393&amp;ncbi_uid=621631&amp;link_uid=PMC1451179" image-link="/pmc/articles/PMC1451179/figure/fig6/" class="imagepopup">F igure  6.—.  From: A Mitogen-Activated Protein Kinase <span class="highlight" style="background-color:">Pathway</span> Essential for Mating and Contributing to Vegetative Growth in Neurospora crassa. </a></p><div class="supp"><p class="details"><div>Model for MAP kinase <span class="highlight" style="background-color:">pathway</span> regulation of gene expression and development. (A) The cAMP-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may regulate the <i>mak-2</i>-related MAP kinase <span class="highlight" style="background-color:">pathway</span> to control aerial development or the two pathways may act independently. (B) Conidiophore development is repressed by the cAMP <span class="highlight" style="background-color:">pathway</span> directly or by its effect on the MAK-2 <span class="highlight" style="background-color:">pathway</span>. On the basis of microarray and enhanced conidiation of the mutants, MAK-2 exerts a stronger effect than PP-1 and MAK-2 may act through PP-1 and another (?) factor to repress conidiation. (C) Protoperithecial development and ascospore lethality depend on the MAK-2 <span class="highlight" style="background-color:">pathway</span> and is independent of cAMP <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Dan Li, et al. Genetics. 2005 July;170(3):1091-1104.</div><p class="links"><a class="dblinks" href="/pubmed/15802524" ref="ordinalpos=393&amp;ncbi_uid=621631&amp;link_uid=15802524">Citation</a><a class="dblinks" href="/pmc/articles/PMC1451179/" ref="ordinalpos=393&amp;ncbi_uid=621631&amp;link_uid=PMC1451179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>394.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2965086/figure/pone-0013543-g006/" ref="ordinalpos=394&amp;ncbi_uid=2560274&amp;link_uid=PMC2965086" href="/pmc/articles/PMC2965086/"><img src="/pmc/articles/instance/2965086/bin/pone.0013543.g006.gif" src-large="/pmc/articles/instance/2965086/bin/pone.0013543.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2965086/" ref="ordinalpos=394&amp;ncbi_uid=2560274&amp;link_uid=PMC2965086" image-link="/pmc/articles/PMC2965086/figure/pone-0013543-g006/" class="imagepopup">Figure 6. Overview of <span class="highlight" style="background-color:">pathway</span> differences observed between FLT3-ITD and FLT3-WT AML samples across the two studies..  From: Functional Characterization of FLT3 Receptor <span class="highlight" style="background-color:">Signaling</span> Deregulation in Acute Myeloid Leukemia by Single Cell Network Profiling (SCNP). </a></p><div class="supp"><p class="details"><div>Illustration of <span class="highlight" style="background-color:">pathway</span> differences between FLT3-ITD and FLT3-WT samples. Red coloring of readouts and bold lines indicate increased <span class="highlight" style="background-color:">pathway</span> activity. White coloring and non-bold lines indicate decreased <span class="highlight" style="background-color:">pathway</span> activity. Dotted line from FLT3L to p-Stat5 in FLT3-WT samples indicates connectivity in a subset of patients. Red “X” indicates DNA Damage. Faded modulators (FLT3L, IL-27) in FLT3-ITD indicate decreased modulator induced <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">David B. Rosen, et al. PLoS One. 2010;5(10):e13543.</div><p class="links"><a class="dblinks" href="/pubmed/21048955" ref="ordinalpos=394&amp;ncbi_uid=2560274&amp;link_uid=21048955">Citation</a><a class="dblinks" href="/pmc/articles/PMC2965086/" ref="ordinalpos=394&amp;ncbi_uid=2560274&amp;link_uid=PMC2965086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>395.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3559548/figure/pone-0055487-g009/" ref="ordinalpos=395&amp;ncbi_uid=4600495&amp;link_uid=PMC3559548" href="/pmc/articles/PMC3559548/"><img src="/pmc/articles/instance/3559548/bin/pone.0055487.g009.gif" src-large="/pmc/articles/instance/3559548/bin/pone.0055487.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3559548/" ref="ordinalpos=395&amp;ncbi_uid=4600495&amp;link_uid=PMC3559548" image-link="/pmc/articles/PMC3559548/figure/pone-0055487-g009/" class="imagepopup">Figure 9. Newly proposed IL-10 feedback loop..  From: Novel Mechanism of Inhibition of Dendritic Cells Maturation by Mesenchymal Stem Cells via Interleukin-10 and the JAK1/STAT3 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The figure illustrates the interplay between IL-10 and the JAK1/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in mechanism inhibition of DCs maturation by MSCs. The MSCs facilitate the combination of the cytokine IL-10 with its IL-10 receptor. This complex activates the JAK1/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that suppresses the DCs maturation. This inhibition of maturation causes a decrease in the expression of IL-10. Thus, the levels of the cytokine IL-10 command this positive feedback loop.</div></p></div><div class="aux"><div class="resc">Wen-hua Liu, et al. PLoS One. 2013;8(1):e55487.</div><p class="links"><a class="dblinks" href="/pubmed/23383203" ref="ordinalpos=395&amp;ncbi_uid=4600495&amp;link_uid=23383203">Citation</a><a class="dblinks" href="/pmc/articles/PMC3559548/" ref="ordinalpos=395&amp;ncbi_uid=4600495&amp;link_uid=PMC3559548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>396.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3697781/figure/bat047-F3/" ref="ordinalpos=396&amp;ncbi_uid=5050673&amp;link_uid=PMC3697781" href="/pmc/articles/PMC3697781/"><img src="/pmc/articles/instance/3697781/bin/bat047f3p.gif" src-large="/pmc/articles/instance/3697781/bin/bat047f3p.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3697781/" ref="ordinalpos=396&amp;ncbi_uid=5050673&amp;link_uid=PMC3697781" image-link="/pmc/articles/PMC3697781/figure/bat047-F3/" class="imagepopup">Figure 3. From: CHD@ZJU: a knowledgebase providing network-based research platform on coronary heart disease. </a></p><div class="supp"><p class="details"><div>A merged <span class="highlight" style="background-color:">pathway</span> interactome network with MAPK, ErbB and calcium <span class="highlight" style="background-color:">signaling</span> pathways in Homo sapiens. Red nodes were genes involved in MAPK <span class="highlight" style="background-color:">signaling</span> pathways, blue nodes were genes involved in calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and green nodes were genes involved in ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Gray nodes were involved in multiple pathways.</div></p></div><div class="aux"><div class="resc">Leihong Wu, et al. Database (Oxford). 2013;2013:bat047.</div><p class="links"><a class="dblinks" href="/pubmed/23818526" ref="ordinalpos=396&amp;ncbi_uid=5050673&amp;link_uid=23818526">Citation</a><a class="dblinks" href="/pmc/articles/PMC3697781/" ref="ordinalpos=396&amp;ncbi_uid=5050673&amp;link_uid=PMC3697781">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>397.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3624419/figure/F8/" ref="ordinalpos=397&amp;ncbi_uid=6015414&amp;link_uid=PMC3624419" href="/pmc/articles/PMC3624419/"><img src="/pmc/articles/instance/3624419/bin/zbc0181345670008.gif" src-large="/pmc/articles/instance/3624419/bin/zbc0181345670008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3624419/" ref="ordinalpos=397&amp;ncbi_uid=6015414&amp;link_uid=PMC3624419" image-link="/pmc/articles/PMC3624419/figure/F8/" class="imagepopup">FIGURE 8. From: MicroRNA 375 Mediates the <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> of Corticotropin-releasing Factor (CRF) Regulating Pro-opiomelanocortin (POMC) Expression by Targeting Mitogen-activated Protein Kinase 8. </a></p><div class="supp"><p class="details"><div><b>The function of miR-375 in the CRF-enhanced POMC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> CRF increases the intracellular cAMP through binding CRF receptor 1, which leads to the activation of PKA, which in turn enhances ERK1/2 phosphorylation levels. The phosphorylation of ERK1/2 activates NGFI-B and further promotes the transcription of POMC. Our results have shown that MAP3K8 mediates the CRF signal <span class="highlight" style="background-color:">pathway</span> through regulating ERK1/2 phosphorylation and POMC expression. However, miR-375 functions as a negative regulator in this <span class="highlight" style="background-color:">pathway</span> by down-regulating MAP3K8 protein levels in AtT-20 cells. CRF also inhibits the expression of miR-375 through PKA. This <span class="highlight" style="background-color:">pathway</span> indicates that miR-375 is a downstream molecule of PKA, which targets MAP3K8, and subsequently down-regulates ERK1/2 phosphorylation and the expression of NGFI-B. This is a novel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that mediates the effect of CRF, which regulates the expression of POMC, thus regulating the POMC-related hormone secretion in the pituitary gland. <i>Arrows</i> indicate stimulation; <i>T bars</i> indicate inhibition.</div></p></div><div class="aux"><div class="resc">Nan Zhang, et al. J Biol Chem. 2013 April 12;288(15):10361-10373.</div><p class="links"><a class="dblinks" href="/pubmed/23430746" ref="ordinalpos=397&amp;ncbi_uid=6015414&amp;link_uid=23430746">Citation</a><a class="dblinks" href="/pmc/articles/PMC3624419/" ref="ordinalpos=397&amp;ncbi_uid=6015414&amp;link_uid=PMC3624419">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>398.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC533862/figure/F1/" ref="ordinalpos=398&amp;ncbi_uid=404763&amp;link_uid=PMC533862" href="/pmc/articles/PMC533862/"><img src="/pmc/articles/instance/533862/bin/1477-7827-2-75-1.gif" src-large="/pmc/articles/instance/533862/bin/1477-7827-2-75-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC533862/" ref="ordinalpos=398&amp;ncbi_uid=404763&amp;link_uid=PMC533862" image-link="/pmc/articles/PMC533862/figure/F1/" class="imagepopup">Figure 1.  From: Role of tyrosine phosphorylation in sperm capacitation / acrosome reaction. </a></p><div class="supp"><p class="details"><div><b>Heuristic model showing the tyrosine phosphorylation <span class="highlight" style="background-color:">signaling</span> pathways in sperm cell involved in capacitation</b>. There seems to be three major <span class="highlight" style="background-color:">signaling</span> pathways operating in sperm cell, namely cAMP/PKA-dependent <span class="highlight" style="background-color:">pathway</span> (<span class="highlight" style="background-color:">pathway</span> I), receptor tyrosine kinase <span class="highlight" style="background-color:">pathway</span> (<span class="highlight" style="background-color:">pathway</span> II), and non-receptor protein tyrosine kinase <span class="highlight" style="background-color:">pathway</span> (<span class="highlight" style="background-color:">pathway</span> III). The <span class="highlight" style="background-color:">pathway</span> I, that is exclusive only to sperm cells, seems to be the major <span class="highlight" style="background-color:">pathway</span> among the three pathways. These cascades may not be mutually exclusive and include cross-talk among several molecules. Many key molecules and receptors are still to be identified to completely elucidate the molecular mechanism and signal transduction cascade involved in capacitation. G-protein coupled receptor <span class="highlight" style="background-color:">pathway</span> has not been included in this model.</div></p></div><div class="aux"><div class="resc">Rajesh K Naz, et al. Reprod Biol Endocrinol. 2004;2:75-75.</div><p class="links"><a class="dblinks" href="/pubmed/15535886" ref="ordinalpos=398&amp;ncbi_uid=404763&amp;link_uid=15535886">Citation</a><a class="dblinks" href="/pmc/articles/PMC533862/" ref="ordinalpos=398&amp;ncbi_uid=404763&amp;link_uid=PMC533862">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>399.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3561537/figure/F6/" ref="ordinalpos=399&amp;ncbi_uid=5763419&amp;link_uid=PMC3561537" href="/pmc/articles/PMC3561537/"><img src="/pmc/articles/instance/3561537/bin/zbc0071337540006.gif" src-large="/pmc/articles/instance/3561537/bin/zbc0071337540006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3561537/" ref="ordinalpos=399&amp;ncbi_uid=5763419&amp;link_uid=PMC3561537" image-link="/pmc/articles/PMC3561537/figure/F6/" class="imagepopup">FIGURE 6. From: NEU3 Sialidase Is Activated under Hypoxia and Protects Skeletal Muscle Cells from Apoptosis through the Activation of the Epidermal Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and the Hypoxia-inducible Factor (HIF)-1?. </a></p><div class="supp"><p class="details"><div><b>Scheme of EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by NEU3-induced GM3 reduction.</b> <i>A</i>, NEU3 removing sialic acid from gangliosides exposed on the outer membrane of adjacent cells. <i>B</i>, mechanism of NEU3 activation of pro-survival <span class="highlight" style="background-color:">signaling</span> cascades, mediated by GM3 removal and consequent activation of EGFR <span class="highlight" style="background-color:">signaling</span> cascade, ultimately up-regulating HIF-1α.</div></p></div><div class="aux"><div class="resc">Raffaella Scaringi, et al. J Biol Chem. 2013 February 1;288(5):3153-3162.</div><p class="links"><a class="dblinks" href="/pubmed/23209287" ref="ordinalpos=399&amp;ncbi_uid=5763419&amp;link_uid=23209287">Citation</a><a class="dblinks" href="/pmc/articles/PMC3561537/" ref="ordinalpos=399&amp;ncbi_uid=5763419&amp;link_uid=PMC3561537">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>400.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3975838/figure/F1/" ref="ordinalpos=400&amp;ncbi_uid=5992104&amp;link_uid=PMC3975838" href="/pmc/articles/PMC3975838/"><img src="/pmc/articles/instance/3975838/bin/1756-8722-7-18-1.gif" src-large="/pmc/articles/instance/3975838/bin/1756-8722-7-18-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3975838/" ref="ordinalpos=400&amp;ncbi_uid=5992104&amp;link_uid=PMC3975838" image-link="/pmc/articles/PMC3975838/figure/F1/" class="imagepopup">Figure 1.  From: Targeting hedgehog <span class="highlight" style="background-color:">signaling</span> in myelofibrosis and other hematologic malignancies. </a></p><div class="supp"><p class="details"><div><b>Janus kinase 2 (JAK2)/signal transducer and activation of transcription (STAT) and hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> pathways in normal development (A) and mechanisms of Hh <span class="highlight" style="background-color:">signaling</span> in cancer (B). (A)</b> JAK/STAT <span class="highlight" style="background-color:">signaling</span>: the JAK2/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated upon binding of a cytokine to its receptor, causing phosphorylation and activation of JAK2, which then recruits and phosphorylates STATs. STATs dimerize, translocate to the nucleus, and activate target gene transcription. Hh <span class="highlight" style="background-color:">signaling</span>: in the absence of Hh ligand, patched (PTCH) inhibits smoothened (SMO). Glioma-associated oncogene homolog 1/2 (GLI1/2) transcription factors are sequestered in the cytoplasm by a repressor complex containing suppressor of fused (SUFU) and degraded. GLI3 is released from SUFU, processed into a repressor form (GLI3<sup>R</sup>), and translocates to the nucleus to inhibit transcription of Hh <span class="highlight" style="background-color:">pathway</span> target genes. Hh <span class="highlight" style="background-color:">signaling</span> is activated upon binding of Hh to PTCH. PTCH-mediated inhibition of SMO is relieved, and SMO activates release of GLIs from the SUFU complex. Activated GLIs (GLI<sup>A</sup>) then translocate to the nucleus to regulate target gene transcription. <b>(B)</b> Several mechanisms of Hh <span class="highlight" style="background-color:">pathway</span> activation in cancer have been proposed, including ligand independent (mutation driven) and ligand dependent (autocrine or paracrine) <span class="highlight" style="background-color:">signaling</span>. During autocrine <span class="highlight" style="background-color:">signaling</span>, Hh ligands produced in the tumor activate Hh <span class="highlight" style="background-color:">signaling</span> in the same tumor cells. Paracrine <span class="highlight" style="background-color:">signaling</span> can involve tumor-to-stroma <span class="highlight" style="background-color:">signaling</span> or stroma-to-tumor <span class="highlight" style="background-color:">signaling</span> (reverse paracrine). During tumor-to-stroma <span class="highlight" style="background-color:">signaling</span>, Hh ligands produced in the tumor activate Hh <span class="highlight" style="background-color:">signaling</span> in surrounding stromal cells, which release growth hormones that in turn feed tumor growth. In the reverse model (stroma-to-tumor), which has been observed in hematologic malignancies (lymphoma, myeloid neoplasms, and multiple myeloma), Hh ligands produced in stromal cells activate Hh <span class="highlight" style="background-color:">signaling</span> in the tumor. BCL2, B-cell CLL/lymphoma 2; BCL2L1, BCL2-like 1; BMP, bone morphogenetic protein; CCND1, cyclin D1; MCL1, myeloid cell leukemia sequence 1; PIM1, pim-1 oncogene.</div></p></div><div class="aux"><div class="resc">Raoul Tibes, et al. J Hematol Oncol. 2014;7:18-18.</div><p class="links"><a class="dblinks" href="/pubmed/24598114" ref="ordinalpos=400&amp;ncbi_uid=5992104&amp;link_uid=24598114">Citation</a><a class="dblinks" href="/pmc/articles/PMC3975838/" ref="ordinalpos=400&amp;ncbi_uid=5992104&amp;link_uid=PMC3975838">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="3" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="4" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="5" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553ebca2cbe05d491bc324d4">signaling pathway <span class="number">(471856)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebc89437716036a8a124a">disease pathway <span class="number">(392428)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553ebc72437716036a8a0f36">differentiation pathway <span class="number">(234089)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553ebc53cbe05d491bc316a6">metabolic pathway <span class="number">(214028)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553ebc19cbe05d491bc30c66">cancer pathways <span class="number">(277189)</span></a>
          <div class="tertiary">PMC</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-27T18:59:52-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC496C53EBEF210000000000A5A525;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC496C53EBEF210000000000A5A525" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="12" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="8A1A393946D25221_0091SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- 8A1A393946D25221_0091SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r451781 Wed, Nov 12 2014 02:48:02 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er451781+Wed%2C+Nov+12+2014+02%3A48%3A02'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>